<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN""http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html><head><title>Patent US5582989 - Multiplex genomic DNA amplification for deletion detection - Google Patents</title><script>(function(){(function(){function e(a){this.t={};this.tick=function(a,c,b){var d=void 0!=b?b:(new Date).getTime();this.t[a]=[d,c];if(void 0==b)try{window.console.timeStamp("CSI/"+a)}catch(e){}};this.tick("start",null,a)}var a;window.performance&&(a=window.performance.timing);var f=a?new e(a.responseStart):new e;window.jstiming={Timer:e,load:f};if(a){var c=a.navigationStart,d=a.responseStart;0<c&&d>=c&&(window.jstiming.srt=d-c)}if(a){var b=window.jstiming.load;0<c&&d>=c&&(b.tick("_wtsrt",void 0,c),b.tick("wtsrt_",
"_wtsrt",d),b.tick("tbsd_","wtsrt_"))}try{a=null,window.chrome&&window.chrome.csi&&(a=Math.floor(window.chrome.csi().pageT),b&&0<c&&(b.tick("_tbnd",void 0,window.chrome.csi().startE),b.tick("tbnd_","_tbnd",c))),null==a&&window.gtbExternal&&(a=window.gtbExternal.pageT()),null==a&&window.external&&(a=window.external.pageT,b&&0<c&&(b.tick("_tbnd",void 0,window.external.startE),b.tick("tbnd_","_tbnd",c))),a&&(window.jstiming.pt=a)}catch(g){}})();})();
</script><link rel="stylesheet" href="/patents/css/_8a2b04e7bf975d5171d8e4c0b6365c7a/kl_intl_patents_bundle.css" type="text/css" /><script src="/books/javascript/atb_8a2b04e7bf975d5171d8e4c0b6365c7a__en.js"></script><script>function googleTranslateElementInit() {new google.translate.TranslateElement({pageLanguage: "en",gaTrack: true,gaId: "UA-27188110-1",multilanguagePage: true});}</script><script src="//translate.google.com/translate_a/element.js?cb=googleTranslateElementInit"></script><meta name="DC.type" content="Patent"><meta name="DC.title" content="Multiplex genomic DNA amplification for deletion detection"><meta name="DC.contributor" content="Charles T. Caskey" scheme="inventor"><meta name="DC.contributor" content="Jeffrey S. Chamberlain" scheme="inventor"><meta name="DC.contributor" content="Richard A. L. Gibbs" scheme="inventor"><meta name="DC.contributor" content="Joel E. Ranier" scheme="inventor"><meta name="DC.contributor" content="Phi N. Nguyen" scheme="inventor"><meta name="DC.contributor" content="Baylor College Of Medicine" scheme="assignee"><meta name="DC.date" content="1994-9-30" scheme="dateSubmitted"><meta name="DC.description" content="The present invention relates to a method for detecting multiple DNA sequences simultaneously. The method involves amplification of multiple sequences simultaneously by annealing a plurality of paired oligonucleotide primers to single stranded DNA. One member of each pair is complementary to the sense strand of a sequences and the other member is complementary to a different segment of the anti-sense strand of the same sequence. The amplification occurs by alternately annealing and extending the primers. The invention also includes oligonucleotide primer sequences helpful in detecting genetic diseases and/or exogenous DNA sequences."><meta name="DC.date" content="1996-12-10" scheme="issued"><meta name="DC.relation" content="EP:0237362:A1" scheme="references"><meta name="DC.relation" content="EP:0256630:A1" scheme="references"><meta name="DC.relation" content="EP:0364255:A2" scheme="references"><meta name="citation_reference" content="Byrne, B. C., et al, Detection of HIV 1 RNA Sequences by In Vitro DNA Amplification, Nucleic Acids Res. 16 (9), 1988, p. 4165."><meta name="citation_reference" content="Byrne, B. C., et al, Detection of HIV-1 RNA Sequences by In-Vitro DNA Amplification, Nucleic Acids Res. 16 (9), 1988, p. 4165."><meta name="citation_reference" content="Cai, S. P., et al, A Simple Approach to Prenatal Diagnosis of Beta Thalassemia in a Geographic Area where Multiple Mutations Occur, Blood 71 (5), 1988, pp. 1357 1360."><meta name="citation_reference" content="Cai, S. P., et al, A Simple Approach to Prenatal Diagnosis of Beta Thalassemia in a Geographic Area where Multiple Mutations Occur, Blood 71 (5), 1988, pp. 1357-1360."><meta name="citation_reference" content="Chamberlain, J. R., et al, Deletion Screening of the Duchenne Muscular Dystrophy Locus Via Multiplex DNA Amplification, Nucleic Acids Res. 16(23), 1988, pp. 11141 11156."><meta name="citation_reference" content="Chamberlain, J. R., et al, Deletion Screening of the Duchenne Muscular Dystrophy Locus Via Multiplex DNA Amplification, Nucleic Acids Res. 16(23), 1988, pp. 11141-11156."><meta name="citation_reference" content="Chamberlain, J. S., et al, Analysis of Duchenne Muscular Dystrophy Gene Mutations in Mice and Humans, Cellualr and Molecular Biology of Muscle Development, vol. 93, (Stockdale, F. and Kedes, L., eds), New York, Alan R. Liss Press, 1989, pp. 951 962."><meta name="citation_reference" content="Chamberlain, J. S., et al, Analysis of Duchenne Muscular Dystrophy Gene Mutations in Mice and Humans, Cellualr and Molecular Biology of Muscle Development, vol. 93, (Stockdale, F. and Kedes, L., eds), New York, Alan R. Liss Press, 1989, pp. 951-962."><meta name="citation_reference" content="Chamberlain, J. S., et al, Expression of the Murine Duchenne Muscular Dystrophy Gene in Muscle and Brain, Science 239, 1988, pp. 1416 1418."><meta name="citation_reference" content="Chamberlain, J. S., et al, Expression of the Murine Duchenne Muscular Dystrophy Gene in Muscle and Brain, Science 239, 1988, pp. 1416-1418."><meta name="citation_reference" content="Chamberlain, J. S., et al, Expression of the Murine Duchenne Muscular Dystrophy Gene in the Muscle and Brain of Normal and Mutant MDX Mice, J. Cell. Biol. 12C:319, 1988."><meta name="citation_reference" content="Chamberlain, J. S., et al, Rapid detection of deletions at the Duchenne muscular dystropy locus via multiplex genomic DNA amplification, Amer. Journal of Human Genetics, 1988, vol. 43, p. A178, (0711) 3.2."><meta name="citation_reference" content="Chamberlain, J. S., et al., &quot;Rapid detection of deletions at the Duchenne muscular dystrophy locus via multiplex genomic DNA amplification&quot; American Journal of Human Genetics 43 (3 sup) (1988) Abstract 0711."><meta name="citation_reference" content="Chamberlain, J. S., et al., Rapid detection of deletions at the Duchenne muscular dystrophy locus via multiplex genomic DNA amplification American Journal of Human Genetics 43 (3 sup) (1988) Abstract 0711."><meta name="citation_reference" content="Chehab, F. F., et al, Detection of Sickle Cell Anaemia and Thalassaemias, Nature 329: 293, 1987."><meta name="citation_reference" content="Chelly, J., et al, Transcription of the Dystrophin Gene in Muscle and Non muscle Tissues, Nature (London) 333, 1988, pp. 858 860."><meta name="citation_reference" content="Chelly, J., et al, Transcription of the Dystrophin Gene in Muscle and Non-muscle Tissues, Nature (London) 333, 1988, pp. 858-860."><meta name="citation_reference" content="Darras, B. T., et al, Direct Method for Prenatal Diagnosis and Carrier Detection in Duchenne/Becker Muscular Dystrophy Using the Entire Dystrophin cDNA, Am. J. Med. Genet. 29, 1988, pp. 713 726."><meta name="citation_reference" content="Darras, B. T., et al, Direct Method for Prenatal Diagnosis and Carrier Detection in Duchenne/Becker Muscular Dystrophy Using the Entire Dystrophin cDNA, Am. J. Med. Genet. 29, 1988, pp. 713-726."><meta name="citation_reference" content="Dilella, A. G., et al, Screening for Phenylketonuria Mutations by DNA Amplification With the Polymerase Chain Reaction, Lancet 1 (8584), 1988, pp. 497 499."><meta name="citation_reference" content="Dilella, A. G., et al, Screening for Phenylketonuria Mutations by DNA Amplification With the Polymerase Chain Reaction, Lancet 1 (8584), 1988, pp. 497-499."><meta name="citation_reference" content="Duggan, et al, HTLV I Induced Lymphoma Mimicking Hodgkin s Disease. Diagnosis by Polymerase Chain Reaction Amplification of Specific HTLV I Sequences in Tumor DNA, Blood 71 (4), 1988, pp. 1027 1032."><meta name="citation_reference" content="Duggan, et al, HTLV-I-Induced Lymphoma Mimicking Hodgkin&#39;s Disease. Diagnosis by Polymerase Chain Reaction Amplification of Specific HTLV-I Sequences in Tumor DNA, Blood 71 (4), 1988, pp. 1027-1032."><meta name="citation_reference" content="Erlich, H. A., et al, Genetic Analysis Using Enzymatic Amplification of Specific Genomic Sequences, Current Communications in Molecular Biology: DNA Probes: Applications in Genetic and Infectious Disease and Cancer, Conference, Cold Spring Harbor Apr. 20 23, 1986, Illus Paper ISBN 0 87969 196 4, 1986, pp. 107 112."><meta name="citation_reference" content="Erlich, H. A., et al, Genetic Analysis Using Enzymatic Amplification of Specific Genomic Sequences, Current Communications in Molecular Biology: DNA Probes: Applications in Genetic and Infectious Disease and Cancer, Conference, Cold Spring Harbor Apr. 20-23, 1986, Illus Paper ISBN 0-87969-196-4, 1986, pp. 107-112."><meta name="citation_reference" content="Farzadegan, H., et al, Loss of Human Immunodeficiency Virus Type 1 (HIV 1) Antibodies with Evidence of Viral Infection in Asymptomatic Homosexual Men: A Report from the Multicenter AIDS Cohort Study, Ann Intern Med. 108:6, Jun. 1988, pp. 785 790."><meta name="citation_reference" content="Farzadegan, H., et al, Loss of Human Immunodeficiency Virus Type 1 (HIV-1) Antibodies with Evidence of Viral-Infection in Asymptomatic Homosexual Men: A Report from the Multicenter AIDS Cohort Study, Ann-Intern-Med. 108:6, Jun. 1988, pp. 785-790."><meta name="citation_reference" content="Heiling, R., et al, A 230kb Cosmid Walk in the Duchenne Muscular Dystrophy Gene: Detection of a Conserved Sequence and of a Possible Deletion Prone Region, Nucl. Acids Res 15 (22), 1987, pp. 9129 9142."><meta name="citation_reference" content="Heiling, R., et al, A 230kb Cosmid Walk in the Duchenne Muscular Dystrophy Gene: Detection of a Conserved Sequence and of a Possible Deletion Prone Region, Nucl. Acids Res 15 (22), 1987, pp. 9129-9142."><meta name="citation_reference" content="Hejtmancik, J. F., et al, Carrier Diagnosis of Duchenne Muscular Dystrophy Using Restriction Fragment Length Polymorphisms, Neurology 36, 1986, pp. 1553 1562."><meta name="citation_reference" content="Hejtmancik, J. F., et al, Carrier Diagnosis of Duchenne Muscular Dystrophy Using Restriction Fragment Length Polymorphisms, Neurology 36, 1986, pp. 1553-1562."><meta name="citation_reference" content="Higuchi, R., et al, DNA Typing from Single Hairs, Nature (Lond) 332 (6164), 1988, pp. 543 546."><meta name="citation_reference" content="Higuchi, R., et al, DNA Typing from Single Hairs, Nature (Lond) 332 (6164), 1988, pp. 543-546."><meta name="citation_reference" content="Impriam, C. C., et al, Analysis of DNA Extracted from Formalin Fixed, Paraffin Embedded Tissues by Enzymatic Amplification and Hybridization with Sequence Specific Oligonucleotides, Biochem Biophys Res Commun 142 (3), 1987, pp. 710 716."><meta name="citation_reference" content="Impriam, C. C., et al, Analysis of DNA Extracted from Formalin-Fixed, Paraffin-Embedded Tissues by Enzymatic Amplification and Hybridization with Sequence-Specific Oligonucleotides, Biochem Biophys Res Commun 142 (3), 1987, pp. 710-716."><meta name="citation_reference" content="Jeffrey, A. J., et al, Amplification of Human Minisatellites by the Polymerase Chain Reaction: Towards DNA Fingerprinting of Single Cells, Nucleic Acids Res. 16(23), 1988, pp. 10953 10971."><meta name="citation_reference" content="Jeffrey, A. J., et al, Amplification of Human Minisatellites by the Polymerase Chain Reaction: Towards DNA Fingerprinting of Single Cells, Nucleic Acids Res. 16(23), 1988, pp. 10953-10971."><meta name="citation_reference" content="Kawasaki, E. S., et al, Diagnosis of Chronic Myeloid and Acute Lymphocytic Leukemias by Detection of Leukemia Specific Messenger RNA Sequences Amplified in Intro, Proc. Natl Acad Sci U S A 85 (15), 1988, pp. 5698 5702."><meta name="citation_reference" content="Kawasaki, E. S., et al, Diagnosis of Chronic Myeloid and Acute Lymphocytic Leukemias by Detection of Leukemia-Specific Messenger RNA Sequences Amplified in-Intro, Proc. Natl Acad Sci U S A 85 (15), 1988, pp. 5698-5702."><meta name="citation_reference" content="Koenig, M., et al, Complete Cloning of the Duchenne Muscular Dystrophy (DMD) cDNA and Preliminary Genomic Organization of the DMD Gene in Normal and Affected Individuals, Cell 50, 1987, pp. 507 517."><meta name="citation_reference" content="Koenig, M., et al, Complete Cloning of the Duchenne Muscular Dystrophy (DMD) cDNA and Preliminary Genomic Organization of the DMD Gene in Normal and Affected Individuals, Cell 50, 1987, pp. 507-517."><meta name="citation_reference" content="Koenig, M., et al, The Complete Sequence of Dystrophin Predicts a Rod Shaped Cytoskeletal Protein, Cell 53, 1988, pp. 219 288."><meta name="citation_reference" content="Koenig, M., et al, The Complete Sequence of Dystrophin Predicts a Rod-Shaped Cytoskeletal Protein, Cell 53, 1988, pp. 219-288."><meta name="citation_reference" content="Kogan et al. (1987) The New England Journal of Medicine, vol. 317 (16) pp. 985 990."><meta name="citation_reference" content="Kogan et al. (1987) The New England Journal of Medicine, vol. 317 (16) pp. 985-990."><meta name="citation_reference" content="Kogan, S. C., An Improved Method for Prenatal Diagnosis of Genetic Diseases by Analysis of Amplified DNA Sequences, N. Engl. J. Med. 317, 1987, pp. 985 990."><meta name="citation_reference" content="Kogan, S. C., An Improved Method for Prenatal Diagnosis of Genetic Diseases by Analysis of Amplified DNA Sequences, N. Engl. J. Med. 317, 1987, pp. 985-990."><meta name="citation_reference" content="Kunkel, L. M., Analysis of Deletions in DNA From Patients with Becker and Duchenne Muscular Dystrophy, Nature (London) 322, 1986, pp.73 77."><meta name="citation_reference" content="Kunkel, L. M., Analysis of Deletions in DNA From Patients with Becker and Duchenne Muscular Dystrophy, Nature (London) 322, 1986, pp.73-77."><meta name="citation_reference" content="Laure, F., et al, Detection of HIV 1 DNA in Infants and Children by Means of the Polymerase Chain Reaction, Lancet 2 (8610) 1988, pp. 538 541."><meta name="citation_reference" content="Laure, F., et al, Detection of HIV 1 DNA in Infants and Children by Means of the Polymerase Chain Reaction, Lancet 2 (8610) 1988, pp. 538-541."><meta name="citation_reference" content="Li, H., et al Amplification and Analysis of DNA Sequences in Single Human Sperm and Diploid Cells, Nature (Lond) 335 (6189), 1988, pp. 414 417."><meta name="citation_reference" content="Li, H., et al Amplification and Analysis of DNA Sequences in Single Human Sperm and Diploid Cells, Nature (Lond) 335 (6189), 1988, pp. 414-417."><meta name="citation_reference" content="Mullis, K., et al, Specific Enzymatic Amplification of DNA In Vitro: The Polymerase Chain Reaction, Dold Spring Harbor Laboratory, vol. 51, (parts 1 and 2), Molecular Biology of Homo Sapiens, Jun. 1986, XXV 702P, (Part 1), pp. 263 273."><meta name="citation_reference" content="Mullis, K., et al, Specific Enzymatic Amplification of DNA In-Vitro: The Polymerase Chain Reaction, Dold Spring Harbor Laboratory, vol. 51, (parts 1 and 2), Molecular Biology of Homo Sapiens, Jun. 1986, XXV+702P, (Part 1), pp. 263-273."><meta name="citation_reference" content="Murakawa, G. J., et al, Direct Detection of HIV 1 RNA from AIDS and ARC Patient Samples, DNA (N Y) 7 (4), 1988, pp. 287 295."><meta name="citation_reference" content="Murakawa, G. J., et al, Direct Detection of HIV-1 RNA from AIDS and ARC Patient Samples, DNA (N Y) 7 (4), 1988, pp. 287-295."><meta name="citation_reference" content="Newton, C. R., et al, Diagnosis of Alpha 1 Antitrypsin Deficiency by Enzymatic Amplification of Human Genomic DNA and Direct Sequencing of Polymerase Chain Reaction Products, Nucleic Acids Res. 16 (17), 1988, pp. 8233 8243."><meta name="citation_reference" content="Newton, C. R., et al, Diagnosis of Alpha-1 Antitrypsin Deficiency by Enzymatic Amplification of Human Genomic DNA and Direct Sequencing of Polymerase Chain Reaction Products, Nucleic Acids Res. 16 (17), 1988, pp. 8233-8243."><meta name="citation_reference" content="Ou, C Y., et al, DNA Amplification for Direct Detection of HIV 1 in DNA of Peripheral Blood Mononuclear Cells, Science (Wash D C) 239 (4837), 1988, pp. 295 297."><meta name="citation_reference" content="Ou, C-Y., et al, DNA Amplification for Direct Detection of HIV-1 in DNA of Peripheral Blood Mononuclear Cells, Science (Wash D C) 239 (4837), 1988, pp. 295-297."><meta name="citation_reference" content="Saiki, R. K., et al, Analysis of Enzymatically Amplified B globin and HLA DQa DNA with Allele Specific Oligonucleotide Probes, Nature (London) 324, 1986, pp. 163 166."><meta name="citation_reference" content="Saiki, R. K., et al, Analysis of Enzymatically Amplified B-globin and HLA-DQa-DNA with Allele-Specific Oligonucleotide Probes, Nature (London) 324, 1986, pp. 163-166."><meta name="citation_reference" content="Saiki, R. K., et al, Diagnosis of Sickle Cell Anemia and B Thalasemia with Enzymatically Amplified DNA and Non Radioactive Allele Specific Oligonucleotide Probes, N. Engl. J. Med. 319, 1988, pp. 537 541."><meta name="citation_reference" content="Saiki, R. K., et al, Diagnosis of Sickle Cell Anemia and B Thalasemia with Enzymatically Amplified DNA and Non-Radioactive Allele-Specific Oligonucleotide Probes, N. Engl. J. Med. 319, 1988, pp. 537-541."><meta name="citation_reference" content="Saiki, R. K., et al, Primer Directed Enzymatic Amplification of DNA with Thermostable DNA Polymerase, Science (Wash D C) 239 (4839), 1988, pp. 487 491."><meta name="citation_reference" content="Saiki, R. K., et al, Primer-Directed Enzymatic Amplification of DNA with Thermostable DNA Polymerase, Science (Wash D C) 239 (4839), 1988, pp. 487-491."><meta name="citation_reference" content="Scharf, S. J., et al, Direct Cloning and Sequence Analysis of Enzymatically Amplified Genomic Sequences, Science (Wash D C) 233 (4768), 1986, pp. 1076 1078."><meta name="citation_reference" content="Scharf, S. J., et al, Direct Cloning and Sequence Analysis of Enzymatically Amplified Genomic Sequences, Science (Wash D C) 233 (4768), 1986, pp. 1076-1078."><meta name="citation_reference" content="Stoflet, E. S., et al., &quot;Genomic Amplificaion with Transcript Sequencing&quot; Science 239:491-494 (1988)."><meta name="citation_reference" content="Stoflet, E. S., et al., Genomic Amplificaion with Transcript Sequencing Science 239:491 494 (1988)."><meta name="citation_reference" content="Wong, C., et al, Characterization of Beta Thalassemia Mutations Using Direct Genomic Sequences of Amplified Single Copy DNA, Nature (Lond) 330 (6146), 1987, pp. 384 386."><meta name="citation_reference" content="Wong, C., et al, Characterization of Beta-Thalassemia Mutations Using Direct Genomic Sequences of Amplified Single Copy DNA, Nature (Lond) 330 (6146), 1987, pp. 384-386."><meta name="citation_patent_number" content="US:5582989"><meta name="citation_patent_application_number" content="US:08/315,673"><link rel="canonical" href="http://www.google.com/patents/US5582989"/><meta property="og:url" content="http://www.google.com/patents/US5582989"/><meta name="title" content="Patent US5582989 - Multiplex genomic DNA amplification for deletion detection"/><meta name="description" content="The present invention relates to a method for detecting multiple DNA sequences simultaneously. The method involves amplification of multiple sequences simultaneously by annealing a plurality of paired oligonucleotide primers to single stranded DNA. One member of each pair is complementary to the sense strand of a sequences and the other member is complementary to a different segment of the anti-sense strand of the same sequence. The amplification occurs by alternately annealing and extending the primers. The invention also includes oligonucleotide primer sequences helpful in detecting genetic diseases and/or exogenous DNA sequences."/><meta property="og:title" content="Patent US5582989 - Multiplex genomic DNA amplification for deletion detection"/><meta property="og:type" content="book"/><meta property="og:site_name" content="Google Books"/><meta property="og:image" content="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><link rel="image_src" href="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><script>(function(){try{var aa=function(a,b,c,d){d=d||{};d._sn=["cfg",b,c].join(".");window.gbar.logger.ml(a,d)};var g=window.gbar=window.gbar||{},l=window.gbar.i=window.gbar.i||{},m={},n;function _tvn(a,b){var c=parseInt(a,10);return isNaN(c)?b:c}function _tvf(a,b){var c=parseFloat(a);return isNaN(c)?b:c}function _tvv(a){return!!a}function p(a,b,c){(c||g)[a]=b}g.bv={n:_tvn("2",0),r:"",f:".67.",e:"0",m:_tvn("0",1)};
function q(a,b,c){var d="on"+b;if(a.addEventListener)a.addEventListener(b,c,!1);else if(a.attachEvent)a.attachEvent(d,c);else{var f=a[d];a[d]=function(){var a=f.apply(this,arguments),b=c.apply(this,arguments);return void 0==a?b:void 0==b?a:b&&a}}}var s=function(a){return function(){return g.bv.m==a}},ba=s(1),ca=s(2);p("sb",ba);p("kn",ca);l.a=_tvv;l.b=_tvf;l.c=_tvn;l.i=aa;var da=window.gbar.i.i;var t,u,v,w;function ea(a){v=a}function fa(a){var b;if(b=v&&window.encodeURIComponent)b=a.href,b=!b.match(/^http[s]?:\/\/accounts\.google\.[^/]*\/ClearSID/i)&&!b.match(/^http[s]?:\/\/[^/]*\/accounts\/ClearSID/i);if(b=b&&encodeURIComponent(v()))a.href=a.href.replace(/([?&]continue=)[^&]*/,"$1"+b)}function ga(a){window.gApplication&&(a.href=window.gApplication.getTabUrl(a.href))}
function ha(a){var b=document.forms[0].q,c=window.encodeURIComponent&&b&&b.value,b=b&&b.placeholder;c&&c!=b&&(a.href=a.href.replace(/([?&])q=[^&]*|$/,function(a,b){return(b||"&")+"q="+encodeURIComponent(c)}))}n=l.a("")?ga:ha;
function x(a,b,c,d,f,e){var h=document.getElementById(a);if(h){var k=h.style;k.left=d?"auto":b+"px";k.right=d?b+"px":"auto";k.top=c+"px";k.visibility=u?"hidden":"visible";f&&e?(k.width=f+"px",k.height=e+"px"):(x(t,b,c,d,h.offsetWidth,h.offsetHeight),u=u?"":a)}}
var y=[],ia=function(a,b){y.push(b)},ja=function(a){a=a||window.event;var b=a.target||a.srcElement;a.cancelBubble=!0;null==t&&(a=document.createElement(Array.every||window.createPopup?"iframe":"div"),a.frameBorder="0",t=a.id="gbs",a.src="javascript:''",b.parentNode.appendChild(a),q(document,"click",z));var c=b,b=0;"gb3"!=c.className&&(c=c.parentNode);a=c.getAttribute("aria-owns")||"gbi";var d=c.offsetWidth,f=20<c.offsetTop?46:24;document.getElementById("tphdr")&&(f-=3);var e=!1;do b+=c.offsetLeft||
0;while(c=c.offsetParent);var c=(document.documentElement.clientWidth||document.body.clientWidth)-b-d,h,d=document.body,k=document.defaultView;k&&k.getComputedStyle?(d=k.getComputedStyle(d,""))&&(h=d.direction):h=d.currentStyle?d.currentStyle.direction:d.style.direction;h="rtl"==h;if("gbi"==a){for(d=0;k=y[d++];)k();A(null,window.navExtra);h&&(b=c,e=!0)}else h||(b=c,e=!0);u!=a&&z();x(a,b,f,e)},z=function(){u&&x(u,0,0)},A=function(a,b){var c,d=document.getElementById("gbi"),f=a;f||(f=d.firstChild);
for(;b&&(c=b.pop());){var e=d,h=c,k=f;w||(w="gb2");e.insertBefore(h,k).className=w}},ka=function(a,b,c){if((b=document.getElementById(b))&&a){a.className="gb4";var d=document.createElement("span");d.appendChild(a);d.appendChild(document.createTextNode(" | "));d.id=c;b.appendChild(d)}},la=function(){return document.getElementById("gb_70")},ma=function(){return!!u};p("qs",n);p("setContinueCb",ea);p("pc",fa);p("tg",ja);p("close",z);p("addLink",ka);p("almm",A);p("si",la);p("adh",ia);p("op",ma);var B=function(){},C=function(){},F=function(a){var b=new Image,c=D;b.onerror=b.onload=b.onabort=function(){try{delete E[c]}catch(a){}};E[c]=b;b.src=a;D=c+1},E=[],D=0;p("logger",{il:C,ml:B,log:F});var G=window.gbar.logger;var H={},na={},I=[],oa=l.b("0.1",.1),pa=l.a("1",!0),qa=function(a,b){I.push([a,b])},ra=function(a,b){H[a]=b},sa=function(a){return a in H},J={},K=function(a,b){J[a]||(J[a]=[]);J[a].push(b)},ta=function(a){K("m",a)},L=function(a,b){var c=document.createElement("script");c.src=a;c.async=pa;Math.random()<oa&&(c.onerror=function(){c.onerror=null;B(Error("Bundle load failed: name="+(b||"UNK")+" url="+a))});(document.getElementById("xjsc")||document.getElementsByTagName("body")[0]||
document.getElementsByTagName("head")[0]).appendChild(c)},N=function(a){for(var b=0,c;(c=I[b])&&c[0]!=a;++b);!c||c[1].l||c[1].s||(c[1].s=!0,M(2,a),c[1].url&&L(c[1].url,a),c[1].libs&&m.d&&m.d(c[1].libs))},O=function(a){K("gc",a)},P=null,ua=function(a){P=a},M=function(a,b,c){if(P){a={t:a,b:b};if(c)for(var d in c)a[d]=c[d];try{P(a)}catch(f){}}};p("mdc",H);p("mdi",na);p("bnc",I);p("qGC",O);p("qm",ta);p("qd",J);p("lb",N);p("mcf",ra);p("bcf",qa);p("aq",K);p("mdd","");p("has",sa);
p("trh",ua);p("tev",M);var Q=l.b("0.1",.001),R=0;
function _mlToken(a,b){try{if(1>R){R++;var c,d=a,f=b||{},e=encodeURIComponent,h=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&jexpid=",e("17483"),"&srcpg=",e("prop=22"),"&jsr=",Math.round(1/Q),"&ogev=",e("ZqPtU5fUNJLFoAScy4DACQ"),"&ogf=",g.bv.f,"&ogrp=",e("1"),"&ogv=",e("1407723702.0"),"&oggv="+e("es_plusone_gc_20140723.0_p0"),"&ogd=",e("com"),"&ogc=",e("NZL"),"&ogl=",e("en")];f._sn&&(f._sn="og."+
f._sn);for(var k in f)h.push("&"),h.push(e(k)),h.push("="),h.push(e(f[k]));h.push("&emsg=");h.push(e(d.name+":"+d.message));var r=h.join("");S(r)&&(r=r.substr(0,2E3));c=r;var Aa=window.gbar.logger._aem(a,c);F(Aa)}}catch(Na){}}var S=function(a){return 2E3<=a.length},va=function(a,b){return b};function T(a){B=a;p("_itl",S,G);p("_aem",va,G);p("ml",B,G);a={};H.er=a}l.a("")?T(function(a){throw a;}):l.a("1")&&Math.random()<Q&&T(_mlToken);I.push(["m",{url:"//ssl.gstatic.com/gb/js/scm_7385cc5883250b43a39405734c1bea59.js"}]);g.mcf("c",{});g.sg={c:""};if(l.a("1")){var wa=l.a("");I.push(["gc",{auto:wa,url:"//ssl.gstatic.com/gb/js/abc/gci_91f30755d6a6b787dcc2a4062e6e9824.js",libs:"googleapis.client:plusone:gapi.iframes"}]);var xa={version:"gci_91f30755d6a6b787dcc2a4062e6e9824.js",index:"",lang:"en"};H.gc=xa;var U=function(a){window.googleapis&&window.iframes?a&&a():(a&&O(a),N("gc"))};p("lGC",U);l.a("1")&&p("lPWF",U)};window.__PVT="";if(l.a("1")&&l.a("1")){var V=function(a){U(function(){K("pw",a);N("pw")})};p("lPW",V);I.push(["pw",{url:"//ssl.gstatic.com/gb/js/abc/pwm_45f73e4df07a0e388b0fa1f3d30e7280.js"}]);var W=[],ya=function(a){W[0]=a},za=function(a,b){var c=b||{};c._sn="pw";B(a,c)},Ba={signed:W,elog:za,base:"https://plusone.google.com/u/0",loadTime:(new Date).getTime()};H.pw=Ba;var X=function(a,b){for(var c=b.split("."),d=function(){var b=arguments;a(function(){for(var a=g,d=0,e=c.length-1;d<e;++d)a=a[c[d]];a[c[d]].apply(a,b)})},f=g,e=0,h=c.length-1;e<h;++e)f=
f[c[e]]=f[c[e]]||{};return f[c[e]]=d};X(V,"pw.clk");X(V,"pw.hvr");p("su",ya,g.pw)};function Ca(){function a(){for(var b;(b=e[h++])&&"m"!=b[0]&&!b[1].auto;);b&&(M(2,b[0]),b[1].url&&L(b[1].url,b[0]),b[1].libs&&m.d&&m.d(b[1].libs));h<e.length&&setTimeout(a,0)}function b(){0<f--?setTimeout(b,0):a()}var c=l.a("1"),d=l.a(""),f=3,e=I,h=0,k=window.gbarOnReady;if(k)try{k()}catch(r){da(r,"ml","or")}d?p("ldb",a):c?q(window,"load",b):b()}p("rdl",Ca);var Da={D:1,H:2,da:3,p:4,W:5,M:6,F:7,g:8,ha:9,U:10,L:11,T:12,S:13,N:14,Q:15,P:16,fa:17,w:18,O:19,ga:20,ea:21,u:22,G:23,ja:24,ka:25,ia:26,A:27,j:28,o:29,k:30,ca:31,Z:32,$:33,J:34,K:35,ba:36,aa:37,Y:38,B:39,R:40,v:41,X:42,V:43,h:48,C:49,I:500},Y=[1,2,3,4,5,6,9,10,11,13,14,28,29,30,34,35,37,38,39,40,41,42,43,48,49,500];var Z=l.b("0.001",1E-4),Ea=l.b("1",1),Fa=!1,Ga=!1;if(l.a("1")){var Ha=Math.random();Ha<=Z&&(Fa=!0);Ha<=Ea&&(Ga=!0)}var Ia=Da,$=null;function Ja(){var a=0,b=function(b,d){l.a(d)&&(a|=b)};b(1,"");b(2,"");b(4,"");b(8,"");return a}
function Ka(a,b){var c=Z,d=Fa,f;f=a;if(!$){$={};for(var e=0;e<Y.length;e++){var h=Y[e];$[h]=!0}}if(f=!!$[f])c=Ea,d=Ga;if(d){d=encodeURIComponent;g.rp?(f=g.rp(),f="-1"!=f?f:"1"):f="1";c=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&oge=",a,"&ogex=",d("17483"),"&ogev=",d("ZqPtU5fUNJLFoAScy4DACQ"),"&ogf=",g.bv.f,"&ogp=",d("22"),"&ogrp=",d(f),"&ogsr=",Math.round(1/c),"&ogv=",d("1407723702.0"),"&oggv="+
d("es_plusone_gc_20140723.0_p0"),"&ogd=",d("com"),"&ogl=",d("en"),"&ogc=",d("NZL"),"&ogus=",Ja()];if(b){"ogw"in b&&(c.push("&ogw="+b.ogw),delete b.ogw);var k;f=b;e=[];for(k in f)0!=e.length&&e.push(","),e.push(La(k)),e.push("."),e.push(La(f[k]));k=e.join("");""!=k&&(c.push("&ogad="),c.push(d(k)))}F(c.join(""))}}function La(a){"number"==typeof a&&(a+="");return"string"==typeof a?a.replace(".","%2E").replace(",","%2C"):a}C=Ka;p("il",C,G);var Ma={};H.il=Ma;setTimeout(function(){C(Ia.g)},0);}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var b=window.gbar.i.i;var c=window.gbar;var f=function(d){try{var a=document.getElementById("gbom");a&&d.appendChild(a.cloneNode(!0))}catch(e){b(e,"omas","aomc")}};c.aomc=f;}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var a=window.gbar;a.mcf("pm",{p:""});}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{window.gbar.rdl();}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
if (window['_OC_timingAction']) {window['_OC_timingAction']('patents_refpage');}</script><style>#gbar,#guser{font-size:13px;padding-top:1px !important;}#gbar{float:left;height:22px}#guser{padding-bottom:7px !important;text-align:right}.gbh,.gbd{border-top:1px solid #c9d7f1;font-size:1px}.gbh{height:0;position:absolute;top:24px;width:100%}#gbs,.gbm{background:#fff;left:0;position:absolute;text-align:left;visibility:hidden;z-index:1000}.gbm{border:1px solid;border-color:#c9d7f1 #36c #36c #a2bae7;z-index:1001}.gb1{margin-right:.5em}.gb1,.gb3{zoom:1}.gb2{display:block;padding:.2em .5em}.gb2,.gb3{text-decoration:none !important;border-bottom:none}a.gb1,a.gb4{text-decoration:underline !important}a.gb1,a.gb2,a.gb3,a.gb4{color:#00c !important}.gbi .gb3,.gbi .gb2,.gbi .gb4{color:#dd8e27 !important}.gbf .gb3,.gbf .gb2,.gbf .gb4{color:#900 !important}a.gb2:hover{background:#36c;color:#fff !important}#gbar .gbz0l{color:#000 !important;cursor:default;font-weight:bold;text-decoration:none !important}
#gbar { padding:.3em .6em !important;}</style></head><body  topmargin="3" marginheight="3"><div id=gbar><nobr><a onclick=gbar.qs(this);gbar.logger.il(1,{t:1}); class=gb1 id=gb_1 href="https://www.google.com/search?sa=N&tab=tw">Search</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:2}); class=gb1 id=gb_2 href="http://www.google.com/search?hl=en&tbm=isch&source=og&sa=N&tab=ti">Images</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:8}); class=gb1 id=gb_8 href="http://maps.google.com/maps?hl=en&sa=N&tab=tl">Maps</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:78}); class=gb1 id=gb_78 href="https://play.google.com/?hl=en&sa=N&tab=t8">Play</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:36}); class=gb1 id=gb_36 href="http://www.youtube.com/results?sa=N&tab=t1">YouTube</a> <a onclick=gbar.logger.il(1,{t:5}); class=gb1 id=gb_5 href="http://news.google.com/nwshp?hl=en&tab=tn">News</a> <a onclick=gbar.logger.il(1,{t:23}); class=gb1 id=gb_23 href="https://mail.google.com/mail/?tab=tm">Gmail</a> <a onclick=gbar.logger.il(1,{t:25}); class=gb1 id=gb_25 href="https://drive.google.com/?tab=to">Drive</a> <a class=gb3 href="http://www.google.com/intl/en/options/" onclick="this.blur();gbar.tg(event);return !1" aria-haspopup=true><u>More</u> <small>&#9660;</small></a><div class=gbm id=gbi><a onclick=gbar.logger.il(1,{t:24}); class=gb2 id=gb_24 href="https://www.google.com/calendar?tab=tc">Calendar</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:51}); class=gb2 id=gb_51 href="http://translate.google.com/?hl=en&sa=N&tab=tT">Translate</a><a onclick=gbar.logger.il(1,{t:17}); class=gb2 id=gb_17 href="http://www.google.com/mobile/?hl=en&tab=tD">Mobile</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:10}); class=gb2 id=gb_10 href="http://www.google.com/search?hl=en&tbo=u&tbm=bks&source=og&sa=N&tab=tp">Books</a><a onclick=gbar.logger.il(1,{t:212}); class=gb2 id=gb_212 href="https://wallet.google.com/manage/?tab=ta">Wallet</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:6}); class=gb2 id=gb_6 href="http://www.google.com/search?hl=en&tbo=u&tbm=shop&source=og&sa=N&tab=tf">Shopping</a><a onclick=gbar.logger.il(1,{t:30}); class=gb2 id=gb_30 href="http://www.blogger.com/?tab=tj">Blogger</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:27}); class=gb2 id=gb_27 href="http://www.google.com/finance?sa=N&tab=te">Finance</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:31}); class=gb2 id=gb_31 href="https://plus.google.com/photos?sa=N&tab=tq">Photos</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:12}); class=gb2 id=gb_12 href="http://www.google.com/search?hl=en&tbo=u&tbm=vid&source=og&sa=N&tab=tv">Videos</a><div class=gb2><div class=gbd></div></div><a onclick=gbar.logger.il(1,{t:66}); href="http://www.google.com/intl/en/options/" class=gb2>Even more &raquo;</a></div></nobr></div><div id=guser width=100%><nobr><span id=gbn class=gbi></span><span id=gbf class=gbf></span><span id=gbe></span><a target=_top id=gb_70 href="https://www.google.com/accounts/Login?service=&continue=http://www.google.com/patents%3Fhl%3Den&hl=en" class=gb4>Sign in</a><div style="display: none"><div class=gbm id=gbd5 aria-owner=gbg5><div class=gbmc><ol id=gbom class=gbmcc></ol></div></div></div></nobr></div><div class=gbh style=left:0></div><div class=gbh style=right:0></div><div role="alert" style="position: absolute; left: 0; right: 0;"><a href="http://www.google.com/patents/us5582989?hl=en&amp;output=html_text" title="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."><img border="0" src="http://www.google.com/images/cleardot.gif"alt="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."></a></div><div id="guser"><nobr></nobr></div><div style="clear:both;"></div><div id="gb-top-search-box" class="gb-top-search-box-small gb-reset"><table><tr><td class="logo"><a href="http://www.google.com/patents" class="logo-link"><img class="logo-img" src="/intl/en/images/logos/google_logo_41.png" alt="Go to Google Books Home" height="41"/></a></td><td><form action="http://www.google.com/search" name="f" id="vheadf" method="get"><span id="hf"></span><input type="hidden" name="tbm" value="pts"/><input type="hidden" name="tbo" value="1"/><input type="hidden" name="hl" value="en"/><table><tr><td><div class="inputs"><table><tr><td><div class="text-input"><input type="text" name="q" id="vheadq" class="text" maxlength="2048" size="31" value="" title="Search Patents" accesskey="s" autocomplete="off"/><script>window._OC_autoDir &&window._OC_autoDir('vheadq', 'tia-vheadq');</script></div></td><td><div class="submit-input"><input name="btnG" class="submit" type="submit" value=""/></div></td></tr></table></div></td><td class="col-ext-links"><div class="ext-links"><a href="http://www.google.com/advanced_patent_search">&lt;nobr&gt;Advanced Patent Search&lt;/nobr&gt;</a></div></td></tr></table></form></td></tr></table></div><div class="kd-appbar"><h2 class="kd-appname"><a href="/patents">Patents</a></h2><div class="kd-buttonbar left" id="left-toolbar-buttons"><a id="appbar-write-review-link" href=""></a><a id="appbar-view-print-sample-link" href=""></a><a id="appbar-view-ebook-sample-link" href=""></a><a id="appbar-patents-prior-art-finder-link" href="https://www.google.com/patents/related/US5582989"></a><a id="appbar-patents-discuss-this-link" href="http://www.google.com/url?id=hfdABAABERAJ&amp;q=http://patents.stackexchange.com/redirect/google-patents%3Fpatent%3DUS5582989&amp;usg=AFQjCNEHISQm4nLbade3RC-GgF8JLAFWRw" data-is-grant="true"></a><a id="appbar-read-patent-link" href="//docs.google.com/viewer?url=patentimages.storage.googleapis.com/pdfs/US5582989.pdf"></a><a id="appbar-download-pdf-link" href="//patentimages.storage.googleapis.com/pdfs/US5582989.pdf"></a></div><div class="kd-buttonbar right" id="right-toolbar-buttons"></div></div><div id="books-microdata" itemscope=""itemtype="http://schema.org/Book"itemid="http://www.google.com/patents/US5582989" style="display:none"><span itemprop="description">The present invention relates to a method for detecting multiple DNA sequences simultaneously. The method involves amplification of multiple sequences simultaneously by annealing a plurality of paired oligonucleotide primers to single stranded DNA. One member of each pair is complementary to the sense...</span><span itemprop="url">http://www.google.com/patents/US5582989?utm_source=gb-gplus-share</span><span class="main-title" itemprop="name">Patent US5582989 - Multiplex genomic DNA amplification for deletion detection</span><img itemprop="image" src="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"alt="Patent US5582989 - Multiplex genomic DNA amplification for deletion detection" title="Patent US5582989 - Multiplex genomic DNA amplification for deletion detection"></div><div style="display: none"><ol id="ofe-gear-menu-contents" class="gbmcc"><li class="gbe gbmtc"><a class="gbmt goog-menuitem-content" id="" href="http://www.google.com/advanced_patent_search">Advanced Patent Search</a></li></ol></div><table id="viewport_table" cellpadding="0" style="clear:both" cellspacing="0"><tr><td id="viewport_td"><div class=vertical_module_list_row><div id=intl_patents class=about_content><div id=intl_patents_v><table class="patent-bibdata"><tr><td class="patent-bibdata-heading">Publication number</td><td class="single-patent-bibdata">US5582989 A</td></tr><tr><td class="patent-bibdata-heading">Publication type</td><td class="single-patent-bibdata">Grant</td></tr><tr><td class="patent-bibdata-heading">Application number</td><td class="single-patent-bibdata">US 08/315,673</td></tr><tr><td class="patent-bibdata-heading">Publication date</td><td class="single-patent-bibdata">Dec 10, 1996</td></tr><tr><td class="patent-bibdata-heading">Filing date</td><td class="single-patent-bibdata">Sep 30, 1994</td></tr><tr><td class="patent-bibdata-heading">Priority date<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The priority date is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the date listed."></span></td><td class="single-patent-bibdata">Oct 12, 1988</td></tr><tr><td class="patent-bibdata-heading">Fee status<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The fee status is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the status or dates listed."></span></td><td class="single-patent-bibdata">Paid</td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Also published as</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/CA1339731C">CA1339731C</a>, </span><span class="patent-bibdata-value"><a href="/patents/DE68929299D1">DE68929299D1</a>, </span><span class="patent-bibdata-value"><a href="/patents/DE68929299T2">DE68929299T2</a>, </span><span class="patent-bibdata-value"><a href="/patents/DE68929299T3">DE68929299T3</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP0364255A2">EP0364255A2</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP0364255A3">EP0364255A3</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP0364255B1">EP0364255B1</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP0364255B2">EP0364255B2</a></span></span></td></tr><tr class="patent-bibdata-list-row alternate-patent-number"><td class="patent-bibdata-heading">Publication number</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value">08315673, </span><span class="patent-bibdata-value">315673, </span><span class="patent-bibdata-value">US 5582989 A, </span><span class="patent-bibdata-value">US 5582989A, </span><span class="patent-bibdata-value">US-A-5582989, </span><span class="patent-bibdata-value">US5582989 A, </span><span class="patent-bibdata-value">US5582989A</span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Inventors</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Charles+T.+Caskey%22">Charles T. Caskey</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Jeffrey+S.+Chamberlain%22">Jeffrey S. Chamberlain</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Richard+A.+L.+Gibbs%22">Richard A. L. Gibbs</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Joel+E.+Ranier%22">Joel E. Ranier</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Phi+N.+Nguyen%22">Phi N. Nguyen</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Original Assignee</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=inassignee:%22Baylor+College+Of+Medicine%22">Baylor College Of Medicine</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Export Citation</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/US5582989.bibtex">BiBTeX</a>, </span><span class="patent-bibdata-value"><a href="/patents/US5582989.enw">EndNote</a>, </span><span class="patent-bibdata-value"><a href="/patents/US5582989.ris">RefMan</a></span></span></td></tr><tr class="patent-internal-links"><td colspan=2><span class="patent-bibdata-value"><a href="#backward-citations">Patent Citations</a> (3),</span> <span class="patent-bibdata-value"><a href="#npl-citations">Non-Patent Citations</a> (73),</span> <span class="patent-bibdata-value"><a href="#forward-citations">Referenced by</a> (81),</span> <span class="patent-bibdata-value"><a href="#classifications">Classifications</a> (22),</span> <span class="patent-bibdata-value"><a href="#legal-events">Legal Events</a> (10)</span> </td></tr><tr><td colspan=2 class="patent-bibdata-external-link-spacer-top"></td></tr><tr class="patent-bibdata-external-link-spacer-bottom"></tr><tr><td colspan=2><span class="patent-bibdata-heading">External Links:&nbsp;</span><span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=hfdABAABERAJ&q=http://patft.uspto.gov/netacgi/nph-Parser%3FSect2%3DPTO1%26Sect2%3DHITOFF%26p%3D1%26u%3D/netahtml/PTO/search-bool.html%26r%3D1%26f%3DG%26l%3D50%26d%3DPALL%26RefSrch%3Dyes%26Query%3DPN/5582989&usg=AFQjCNHJ8vmGjzsCPfii4Dvk6v1JRZBDqQ">USPTO</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=hfdABAABERAJ&q=http://assignments.uspto.gov/assignments/q%3Fdb%3Dpat%26pat%3D5582989&usg=AFQjCNHbz-5Om2Zy6bDir2Bpz1afeJImfg">USPTO Assignment</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=hfdABAABERAJ&q=http://worldwide.espacenet.com/publicationDetails/biblio%3FCC%3DUS%26NR%3D5582989A%26KC%3DA%26FT%3DD&usg=AFQjCNEp3HuDgzPp0vNNG0IL-aKWk4ZaRg">Espacenet</a></span></span></td></tr><tr class="patent-bibdata-group-spacer"></tr></table><div class="number-and-title"><span class="patent-title"><invention-title mxw-id="PT54111850" lang="EN" load-source="patent-office">Multiplex genomic DNA amplification for deletion detection</invention-title></span><br><span class="patent-number">US 5582989 A</span></div><div class="patent-section patent-abstract-section"><div class="patent-section-header"><span class="patent-section-title">Abstract</span></div><div class="patent-text"><abstract mxw-id="PA37593290" lang="EN" load-source="patent-office"> <div class="abstract">The present invention relates to a method for detecting multiple DNA sequences simultaneously. The method involves amplification of multiple sequences simultaneously by annealing a plurality of paired oligonucleotide primers to single stranded DNA. One member of each pair is complementary to the sense strand of a sequences and the other member is complementary to a different segment of the anti-sense strand of the same sequence. The amplification occurs by alternately annealing and extending the primers. The invention also includes oligonucleotide primer sequences helpful in detecting genetic diseases and/or exogenous DNA sequences.</div>
  </abstract></div></div><div class="patent-section patent-drawings-section"><div class="patent-section-header"><span class="patent-section-title">Images<span class="patent-section-count">(6)</span></span></div><div class="patent-drawings-body"><div class="patent-drawings-carousel"><div class="drawings"><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5582989-2.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5582989-2.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5582989-3.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5582989-3.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5582989-4.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5582989-4.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5582989-5.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5582989-5.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5582989-6.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5582989-6.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5582989-7.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5582989-7.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div></div></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img" alt="Previous page"src="/googlebooks/images/kennedy/page_left.png"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img" alt="Next page"src="/googlebooks/images/kennedy/page_right.png"width="21" height="21" /></div></div></div><div class="patent-post-drawings"></div><div class="patent-section patent-claims-section"><div class="patent-section-header"><span class="patent-section-title">Claims<span class="patent-section-count">(15)</span></span></div><div class="patent-text"><div mxw-id="PCLM59355439" lang="EN" load-source="patent-office" class="claims">
    <claim-statement>What is claimed is:</claim-statement> <div class="claim"> <div num="1" class="claim">
      <div class="claim-text">1. A method for simultaneously detecting known deletions from at least three DNA sequences, comprising the steps of:<div class="claim-text">treating said DNA to form single-stranded complementary strands;</div> <div class="claim-text">adding at least three pairs of oligonucleotide primers, each pair specific for a different sequence, one primer of each pair substantially complementary to a part of the sequence in the sense-strand and the other primer of each pair substantially complementary to a different part of the same sequence in the complementary anti-sense strand;</div> <div class="claim-text">annealing the at least three pairs of primers to their complementary sequences, all primers being subjected to the same reaction conditions;</div> <div class="claim-text">simultaneously extending said at least three pairs of annealed primers from each primer's 3' terminus to synthesize an extension product complementary to the strands annealed to each primer, said extension products, after separation from their complement, being capable of serving as templates for the synthesis of an extension product from the other primer of each pair;</div> <div class="claim-text">separating said extension products from said templates to produce single-stranded molecules;</div> <div class="claim-text">amplifying said single stranded molecules by repeating, at least once, said annealing, extending and separating steps; and</div> <div class="claim-text">identifying said amplified extension products from each different sequence.</div> </div>
    </div>
    </div> <div class="claim-dependent"> <div num="2" class="claim">
      <div class="claim-text">2. The method of claim 1 for detecting deletions from at least genomic DNA sequences, wherein said sequences are selected from the group of known sequences on the X and Y chromosomes.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="3" class="claim">
      <div class="claim-text">3. The method of claim 2 for the detection of X-linked disease, wherein said genomic DNA sequences contain a deletion that causes a genetic disease.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="4" class="claim">
      <div class="claim-text">4. The method of claim 3 for the detection of said X-linked genetic diseases selected from the group consisting of ornithine transcarbamylase deficiency, hypoxanthine phosphoribosyltransferfase deficiency, steroid sulfatase deficiency and X-linked muscular dystrophy.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="5" class="claim">
      <div class="claim-text">5. The method of claim 4 for the detection of X-linked muscular dystrophy, wherein each pair of said at least three pairs of primers are complementary to different sequences within the gene coding for the dystrophin protein.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="6" class="claim">
      <div class="claim-text">6. The method of claim 5, wherein the at least three pairs of primers is selected from the group consisting of:<div class="claim-text">(1) 5'-GACTTTCGATGTTGAGATTACTTTCCC-3'</div> <div class="claim-text">(2) 5'-AAGCTTGAGATGCTCTCACCTTTTCC-3',</div> <div class="claim-text">(1) 5'-GTCCTTTACACACTTTACCTGTTGAG-3'</div> <div class="claim-text">(2) 5'-GGCCTCATTCTCATGTTCTAATTAG-3',</div> <div class="claim-text">(1) 5'-AAACATGGAACATCCTTGTGGGGAC-3'</div> <div class="claim-text">(2) 5'-CATTCCTATTAGATCTGTCGCCCTAC-3',</div> <div class="claim-text">(1) 5'-GATAGTGGGCTTTACTTACATCCTTC-3'</div> <div class="claim-text">(2) 5'-GAAAGCACGCAACATAAGATACACCT-3',</div> <div class="claim-text">(1) 5'-CTTGATCCATATGCTTTTACCTGCA-3'</div> <div class="claim-text">(2) 5'-TCCATCACCCTTCAGAACCTGATCT-3',</div> <div class="claim-text">(1) 5'-TTGAATACATTGGTTAAATCCCAACATG-3'</div> <div class="claim-text">(2) 5'-CCTGAATAAAGTCTTCCTTACCACAC-3', and</div> <div class="claim-text">(1) 5'-TTCTACCACATCCCATTTTCTTCCA-3'</div> <div class="claim-text">(2) 5'-GATGGCAAAAGTGTTGAGAAAAAGTC-3'.</div> </div>
    </div>
    </div> <div class="claim-dependent"> <div num="7" class="claim">
      <div class="claim-text">7. The method of claim 3, wherein said genomic DNA is from fetal tissue.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="8" class="claim">
      <div class="claim-text">8. The method of claim 1 for detecting deletions from at least three genomic DNA sequences, wherein the at least three pairs of primers is selected from the group consisting of:<div class="claim-text">(1) 5'-GACTTTCGATGTTGAGATTACTTTCCC-3'</div> <div class="claim-text">(2) 5'-AAGCTTGAGATGCTCTCACCTTTTCC-3',</div> <div class="claim-text">(1) 5'-GTCCTTTACACACTTTACCTGTTGAG-3'</div> <div class="claim-text">(2) 5'-GGCCTCATTCTCATGTTCTAATTAG-3',</div> <div class="claim-text">(1) 5'-AAACATGGAACATCCTTGTGGGGAC-3'</div> <div class="claim-text">(2) 5'-CATTCCTATTAGATCTGTCGCCCTAC-3',</div> <div class="claim-text">(1) 5'-GATAGTGGGCTTTACTTACATCCTTC-3',</div> <div class="claim-text">(2) 5'-GAAAGCACGCAACATAAGATACACCT-3',</div> <div class="claim-text">(1) 5'-CTTGATCCATATGCTTTTACCTGCA-3'</div> <div class="claim-text">(2) 5'-TCCATCACCCTTCAGAACCTGATCT-3',</div> <div class="claim-text">(1) 5'-TTGAATACATTGGTTAAATCCCAACATG-3'</div> <div class="claim-text">(2) 5'-CCTGAATAAAGTCTTCCTTACCACAC-3',</div> <div class="claim-text">(1) 5'-TTCTACCACATCCCATTTTCTTCCA-3'</div> <div class="claim-text">(2) 5'-GATGGCAAAAGTGTTGAGAAAAAGTC-3',</div> <div class="claim-text">(1) 5'-TGGTCTCCTTAAACCTGTCTT-3'</div> <div class="claim-text">(2) 5'-ACACAACTGTGTTCACTAG-3',</div> <div class="claim-text">(1) 5'-ACGTGGAGTGACGATGCTCTTCCC-3'</div> <div class="claim-text">(2) 5'-GTGGGATTCACCACTTTTCCC-3', and</div> <div class="claim-text">(1) 5'-GAAGTCAAGGACACCGAGGAA-3'</div> <div class="claim-text">(2) 5'-AGCCCTCTGGCCAGTCCTAGTG-3'.</div> </div>
    </div>
    </div> <div class="claim-dependent"> <div num="9" class="claim">
      <div class="claim-text">9. The method of claim 1, wherein said at least three pairs of primers have Tms such that the lowest Tm and the highest Tm vary by no more than 8.3° C.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="10" class="claim">
      <div class="claim-text">10. The method of claim 1, wherein said at least three pairs of primers have Tms such that the lower Tm of each pair varies from the lower Tm of each other pair by no more than 4.4° C.</div>
    </div>
    </div> <div class="claim"> <div num="11" class="claim">
      <div class="claim-text">11. A method for simultaneously detecting at least three DNA sequences, comprising the steps of:<div class="claim-text">adding to a common reaction vessel containing a sample mixture of at least three distinct, target sequences in single-stranded form, at least three pairs of oligonucleotide primers, each pair specific for a different sequence, one primer of each pair substantially complementary to a part of the sequence in the sense-strand and the other primer of each pair substantially complementary to a different part of the same sequence in the complementary anti-sense strand;</div> <div class="claim-text">annealing the at least three pairs of primers to their complementary sequences, all primers being subject to the same reaction conditions;</div> <div class="claim-text">simultaneously extending said at least three pairs of annealed primers from each primer's 3' terminus to synthesize an extension product complementary to the strands annealed to each primer, said extension products, after separation from their complement, being capable of serving as templates for the synthesis of an extension product from the other primer of each pair;</div> <div class="claim-text">separating said extension products from said templates to produce single-stranded molecules;</div> <div class="claim-text">amplifying said single stranded target sequences by repeating, at least once, said annealing, extending and separating steps; and</div> <div class="claim-text">identifying whether amplified extension products have been synthesized from each different sequence, as a result of the presence or absence of each target sequence.</div> </div>
    </div>
    </div> <div class="claim"> <div num="12" class="claim">
      <div class="claim-text">12. A method for simultaneously detecting known deletions from at least three DNA sequences, comprising the steps of:<div class="claim-text">treating said DNA to form single-stranded complementary strands;</div> <div class="claim-text">adding at least three pairs of oligonucleotide primers, each pair specific for a different sequence, one primer of each pair substantially complementary to a part of the sequence in the sense-strand and the other primer of each pair substantially complementary to a different part of the same sequence in the complementary anti-sense strand and each primer having a Tm such that the lowest Tm and highest Tm of all added primers varies by no more than 8.3° C.;</div> <div class="claim-text">annealing the at least three pairs of primers to their complementary sequences, all primers being subjected to the same reaction conditions;</div> <div class="claim-text">simultaneously extending said at least three pairs of annealed primers from each primer's 3' terminus to synthesize an extension product complementary to the strands annealed to each primer, said extension products, after separation from their complement, being capable of serving as templates for the synthesis of an extension product from the other primer of each pair;</div> <div class="claim-text">separating said extension products from said templates to produce single-stranded molecules;</div> <div class="claim-text">amplifying said single stranded molecules by repeating, at least once, said annealing, extending and separating steps; and</div> <div class="claim-text">identifying said amplified extension products from each different sequence; and</div> <div class="claim-text">analyzing said amplified extension products to detect known deletions.</div> </div>
    </div>
    </div> <div class="claim"> <div num="13" class="claim">
      <div class="claim-text">13. A method for simultaneously detecting a presence or absence of at least three target DNA sequences, comprising the steps of:<div class="claim-text">adding to a common reaction vessel containing a sample mixture of at least three distinct, target sequences in single-stranded form, at least three pairs of oligonucleotide primers, each pair specific for a different sequence, one primer of each pair substantially complementary to a part of the sequence in the sense-strand and the other primer of each pair substantially complementary to a different part of the same sequence in the complementary anti-sense strand and each primer having a Tm such that the lowest Tm and highest Tm of all added primers varies by no more than 8.3° C.;</div> <div class="claim-text">annealing the at least three pairs of primers to their complementary sequences, all primers being subject to the same reaction conditions;</div> <div class="claim-text">simultaneously extending said at least three pairs of annealed primers from each primer's 3' terminus to synthesize an extension product complementary to the strands annealed to each primer, said extension products, after separation from their complement, being capable of serving as templates for the synthesis of an extension product from the other primer of each pair;</div> <div class="claim-text">separating said extension products from said templates to produce single-stranded molecules;</div> <div class="claim-text">amplifying said single stranded target sequences by repeating, at least once, said annealing, extending and separating steps; and</div> <div class="claim-text">identifying whether amplified extension products have been synthesized from each different target sequence, wherein a presence of an extension product indicates the presence of a target sequence and an absence of an extension product indicates the absence of a target sequence.</div> </div>
    </div>
    </div> <div class="claim"> <div num="14" class="claim">
      <div class="claim-text">14. A method for simultaneously detecting known deletions from at least three DNA sequences, comprising the steps of:<div class="claim-text">treating said DNA to form single-stranded complementary strands;</div> <div class="claim-text">adding at least three pairs of oligonucleotide primers, each pair specific for a different sequence, one primer of each pair substantially complementary to a part of the sequence in the sense-strand and the other primer of each pair substantially complementary to a different part of the same sequence in the complementary anti-sense strand and each primer having a Tm such that the lowest Tm and highest Tm of all added primers varies by no more than 4.4° C.;</div> <div class="claim-text">annealing the at least three pairs of primers to their complementary sequences, all primers being subjected to the same reaction conditions;</div> <div class="claim-text">simultaneously extending said at least three pairs of annealed primers from each primer's 3' terminus to synthesize an extension product complementary to the strands annealed to each primer, said extension products, after separation from their complement, being capable of serving as templates for the synthesis of an extension product from the other primer of each pair;</div> <div class="claim-text">separating said extension products from said templates to produce single-stranded molecules;</div> <div class="claim-text">amplifying said single stranded molecules by repeating, at least once, said annealing, extending and separating steps; and</div> <div class="claim-text">identifying said amplified extension products from each different sequence; and</div> <div class="claim-text">analyzing said amplified extension products to detect known deletions.</div> </div>
    </div>
    </div> <div class="claim"> <div num="15" class="claim">
      <div class="claim-text">15. A method for simultaneously detecting at least three target DNA sequences, comprising the steps of:<div class="claim-text">adding to a common reaction vessel containing a sample mixture of at least three distinct, target sequences in single-stranded form, at least three pairs of oligonucleotide primers, each pair specific for a different sequence, one primer of each pair substantially complementary to a part of the sequence in the sense-strand and the other primer of each pair substantially complementary to a different part of the same sequence in the complementary anti-sense strand and each primer having a Tm such that the lowest Tm and highest Tm of all added primers varies by no more than 4.4° C.;</div> <div class="claim-text">annealing the at least three pairs of primers to their complementary sequences, all primers being subject to the same reaction conditions;</div> <div class="claim-text">simultaneously extending said at least three pairs of annealed primers from each primer's 3' terminus to synthesize an extension product complementary to the strands annealed to each primer, said extension products, after separation from their complement, being capable of serving as templates for the synthesis of an extension product from the other primer of each pair;</div> <div class="claim-text">separating said extension products from said templates to produce single-stranded molecules;</div> <div class="claim-text">amplifying said single stranded target sequences by repeating, at least once, said annealing, extending and separating steps; and</div> <div class="claim-text">identifying whether amplified extension products have been synthesized from each different target sequence, wherein a presence of an extension product indicates the presence of a target sequence and an absence of an extension product indicates the absence of a target sequence.</div> </div>
    </div>
  </div> </div></div></div><div class="patent-section patent-description-section"><div class="patent-section-header"><span class="patent-section-title">Description</span></div><div class="patent-text"><div mxw-id="PDES66857217" lang="EN" load-source="patent-office" class="description">
    <p>This application is a continuation of application Ser. No. 08/060,463, filed May 12, 1993, which was a continuation of application Ser. No. 07/770,742, filed Oct. 2, 1991, now abandoned, which was a continuation of application Ser. No. 07/256,689, filed Oct. 12, 1988, all now abandoned.</p>
    <heading>FIELD OF THE INVENTION</heading> <p>This invention relates to the field of simultaneous detection of deletions in genomic DNA sequences by the process of amplification of multiple sequences within the hemizygous or homozygous genome. The nucleic acid sequences are amplified by the process of simultaneous multiple repetitive reactions. This method of deletion detection is useful in a variety of areas including screening for genetic disease, and animal husbandry. Multiplex DNA amplification is also applicable to the simultaneous analysis of multiple genomic sequences and is useful in forensic medicine, disease screening, and in the development of recombinant or transgenic organisms.</p>
    <heading>BACKGROUND</heading> <p>This invention is an improvement on currently established procedures for the detection of genetic diseases resulting from mutations and deletions in genomic DNA sequences. Prenatal diagnosis and carrier detection of many X-linked diseases are available via Southern analysis using full length cDNA clones. Unfortunately, there are several major limitations that prevent widespread and routine use of Southern analysis for diagnosis of genetic disease. In many of the X-linked diseases, the defective sequences are unknown and probes are unavailable. In other diseases, such as X-linked muscular dystrophy, there are multiple exons, at least 60, scattered over a large area of genomic DNA, approximately 2.4 million bases. The introns average 35 Kb in length. In the case of muscular dystrophy, at least 7-9 separate cDNA subclones are necessary for Southern blot analysis to resolve each exon-containing restriction fragment for hyplotype assignment or diagnosis of genomic alterations. Furthermore, Southern analysis is an expensive, tedious, and time-consuming technique that requires the use of radioisotopes, making it unsuitable for routine use in clinical laboratories.</p>
    <p>An alternative to Southern analysis for mutation and deletion detection is the polymerase chain reaction (PCR) described by Mullis et al. in U.S. Pat. No. 4,683,195 which issued on Jul. 28, 1987 and by Mullis in U.S. Pat. No. 4,683,202 which issued on Jul. 28, 1987. With PCR, specific regions of a gene can be amplified up to a million-fold from nanogram quantities of genomic DNA. After amplification the nucleic acid sequences can be analyzed for the presence of mutant alleles either by direct DNA sequencing or by hybridization with allele-specific oligonucleotide probes. The PCR technique has proven useful in the diagnosis of several diseases including β-thalassemia, hemophilia A, sickle cell anemia and phenylketonuria. Routine screening for genetic diseases and exogenous DNA sequences, such as virus, with PCR, has been limited by the ability to conduct tests for only a single sequence at a time. Screening for a plurality of possible DNA sequences requires a cumbersomely large number of separate assays, thus increasing the time, expense, and tedium of performing such assays. For example, in some diseases, such as Duchenne muscular dystrophy (DMD), PCR diagnosis has been limited since point mutations leading to DMD have not been identified. Approximately 60% of the cases of DMD are due to deletions. The other 40% are unknown at present, but probably involve mutations of the intron-exon splice sites or the creation of premature stop codons. Thus a large gene like the DMD gene must be screened with multiple assays.</p>
    <p>In both U.S. Pat. Nos. 4,683,195 and 4,683,202, procedures are described for amplification of specific sequences. Both patents describe procedures for detecting the presence or absence of at least one specific nucleic acid sequence in a sample containing a mixture of sequences. Although the patents claim at least one sequence and state that multiple sequences can be detected, they do not provide an effective procedure for amplifying multiple sequences at the same time. In the examples, single sequences are amplified or multiple sequences are amplified sequentially. Adding primers for a second sequence is usually possible, but when primers for more than two sequences are added the procedure falls apart. The present application is an improvement on the PCR method and solves the problems encountered when primers for multiple sequences are reacted simultaneously. The present invention describes a procedure for simultaneous amplification of multiple sequences, and for the application of this multiplex amplification procedure in order to detect a plurality of deletions within the same gene or within multiple genes.</p>
    <p>The procedures of the present application provide improved methods for the detection of deletions in hemizygous genes on the X and Y chromosomes. The procedures are effective in detecting genetic diseases caused by deletions on the X or Y chromosome, for example, DMD. They are also effective in detecting homozyous deletions and may be used to simultaneously screen for many possible homozygous or hemizygous deletions as long as parts of the appropriate genetic sequences are known. The procedure for multiplex amplification also enables simultaneous analysis of multiple genetic loci regardless of the presence or absence of deletions.</p>
    <heading>SUMMARY OF THE INVENTION</heading> <p>An object of the present invention is a method for simultaneously detecting deletions at a plurality of genomic DNA sequences.</p>
    <p>An additional object of the present invention is to detect X-linked genetic diseases.</p>
    <p>A further object of the present invention is the diagnosis of DMD.</p>
    <p>A further object of the present invention is to simultaneously analyze multiple genetic loci for polymorphisms and/or non-deletion mutations.</p>
    <p>Thus, in accomplishing the foregoing objects there is provided in accordance with one aspect of the present invention, a method for simultaneously detecting deletions at a plurality of genomic DNA sequences, comprising the steps of:</p>
    <p>Treating said genomic DNA to form single stranded complementary strands;</p>
    <p>Adding a plurality of paired oligonucleotide primers, each pair specific for a different sequence, one primer of each pair substantially complementary to a part of the sequence in the sense strand and the other primer of each pair substantially complementary to a different part of the same sequence in the complementary anti-sense strand;</p>
    <p>Annealing the plurality of primers to their complementary sequences;</p>
    <p>Simultaneously extending said plurality of annealed primers from each primer's 3' terminus to synthesize an extension product complementary to the strands annealed to each primer, said extension products, after separating from their complement, serving as templates for the synthesis of an extension product from the other primer of each pair;</p>
    <p>Separating said extension products from said templates to produce single-stranded molecules;</p>
    <p>Amplifying said single stranded molecules by repeating at least once, said annealing, extending and separating steps; and</p>
    <p>Identifying said amplified extension products from each different sequence.</p>
    <p>Additional embodiments include detection of deletions at a plurality of genomic DNA sequences on the X and Y chromosomes or on autosomal chromosomes when the deletions are homozygous. A variety of X-linked diseases can be detected including ornithine transcarbamylase deficiency, hypoxanthine phosphoribosyltransferfase deficiency, steroid sulfatase deficiency and X-linked muscular dystrophy.</p>
    <p>In another embodiment, X-linked muscular dystrophy is detected using a plurality of paired primers which are complementary to different sequences within the gene coding for the protein dystrophin. Other embodiments include multiple oligonucleotide primers useful in detecting X-linked genetic disease.</p>
    <p>Other and further objects, features and advantages will be apparent from the following description of the presently preferred embodiments of the invention given for the purpose of disclosure when taken in conjunction with the accompanying drawings.</p>
    <heading>BRIEF DESCRIPTION OF THE DRAWINGS</heading> <p>The invention will be more readily understood from a reading of the following specification and by references to the accompanying drawings, forming a part thereof:</p>
    <p>FIG. 1 is a schematic representation of the DMD gene illustrating the approximate size of the locus, the position of the amplified fragments and the location of the genomic regions that have been cloned and sequenced.</p>
    <p>FIG. 2 is an example of a PCR reaction used to detect a deletion in fetal DNA for prenatal diagnosis.</p>
    <p>FIG. 3A and 3B represents the multiplex DNA amplification of lymphoblast DNA from unrelated male DMD patients. 3A. and 3B. show two sets of ten samples. Each DRL # refers to the R. J. Kleberg Center for Human Genetics Diagnostic Research Laboratory family number. MW: Hae III digested φX174 DNA. (-): no template DNA added to the reaction. The relationship between the amplified region and the region on the gene is indicated to the right of A. The letters correspond to those on FIG. 1.</p>
    <p>FIG. 4 represents Multiplex DNA amplification for prenatal diagnosis of DMD. Shown are the results of amplification using DNA from an affected male (AM; lymphoblast DNA) and a male fetus (MF; cultured amniotic fluid cell DNA) from six different families. Both the affected male and the fetal DNAs of DRL #s 521 and 531 display a deletion of region f (FIG. 1); diagnosing these fetuses as affected. In DRL #43C the affected male is deleted for all regions except f, while the fetus is unaffected. The affected male in DRL #483 is deleted for region a, while the male fetus is unaffected. Neither of the samples from DRL #s 485 or 469 displayed a deletion with this technique.</p>
    <p>FIG. 5 represents Multiplex DNA amplification from chorionic villus specimen (CVS) DNA. Both the affected male (AM; lymphoblast DNA) and the male fetus (MF; CVS DNA) from DRL #92 display a deletion of regions e and f (FIG. 1), diagnosing the fetus as affected. CVS DNA from DRL #120 did not display a deletion with this technique.</p>
    <p>FIG. 6 shows amplification of seven exon regions of the DMD locus.</p>
    <p>The drawings are not necessarily to scale and certain features of the invention may be exaggerated in scale or shown in schematic form in the interests of clarity and conciseness.</p>
    <heading>DETAILED DESCRIPTION</heading> <p>It will be readily apparent to one skilled in the art that various substitutions and modifications may be made to the invention disclosed herein, without departing from the scope and spirit of the invention.</p>
    <p>The term "oligonucleotide primers" as used herein defines a molecule comprised of more than three deoxyribonucleotides or ribonucleotides. Its exact length will depend on many factors relating to the ultimate function and use of the oligonucleotide primer, including temperature, source of the primer and use of the method. The oligonucleotide primer can occur naturally, as in a purified restriction digest, or be produced synthetically. The oligonucleotide primer is capable of acting as an initiation point for synthesis when placed under conditions which induce synthesis of a primer extension product complementary to a nucleic acid strand. The conditions can include the presence of nucleotides and an inducing agent such as a DNA polymerase at a suitable temperature and pH. In the preferred embodiment, the primer is a single-stranded oligodeoxyribonucleotide of sufficient length to prime the synthesis of an extension product from a specific sequence in the presence of an inducing agent. In the deletion detection procedure, the oligonucleotides are usually at least greater than 12 mers in length. In the preferred embodiment, the oligonucleotide primers are about 18 to 29 mers in length. Sensitivity and specificity of the oligonucleotide primers are determined by the primer length and uniqueness of sequence within a given sample of template DNA. Primers which are too short, for example, less than about 12 mer may show non-specific binding to a wide variety of sequences in the genomic DNA and thus are not very helpful. In the preferred embodiment, the oligonucleotide primer is usually selected for its ability to anneal to intron sequences in the proximity of the 5' or 3' end of the exon or to anneal to a sequence at the intron-exon junction. Since the known deletion defects resulting in genetic diseases result from deletions that include the exons or intron-splice site regions, it is preferable to have primers complementary to intron sequences.</p>
    <p>Each primer pair herein was selected to be substantially complementary to the different strands of each specific sequence to be amplified. Thus, one primer of each pair is sufficiently complementary to hybridize with a part of the sequence in the sense strand and the other primer of each pair is sufficiently complementary to hybridize with a different part of the same sequence in the anti-sense strand. Thus, although the primer sequence need not reflect the exact sequence of the template, the more closely it does reflect the exact sequence the better the binding during the annealing stage.</p>
    <p>Within a primer pair, each primer preferably binds at a site on the sequence of interest distant from the other primer. In the preferred embodiment the distance between the primers should be sufficient to allow the synthesis of an extension product between the two binding sites, yet close enough so that the extension product of each primer, when separated from its template, can serve as a template for the other primer. The extension products from the two paired primers are complementary to each other and can serve as templates for further synthesis. The further apart the binding sites, the more genomic DNA there is which can be screened. However, if the distance is too great the extension products will not efficiently overlap with the primers and thus amplification will not occur.</p>
    <p>As used herein the term "extension product" refers to the nucleotide sequence which is synthesized from the 3' end of the oligonucleotide primer and which is complementary to the strand to which the oligonucleotide primer is bound.</p>
    <p>As used herein the term "differentially labeled" shall indicate that each extension product can be distinguished from all the others because it has a different label attached or is of a different size or binds to a specifically labelled oligonucleotide. One skilled in the art will recognize that a variety of labels are available. For example, these can include radioisotopes, fluorescers, chemiluminescers, enzymes and antibodies. Various factors affect the choice of the label. These include the effect of the label on the rate of hybridization and binding of the primer to the DNA, the sensitivity of the label, the ease of making the labeled primer, probe or extension products, the ability to automate, available instrumentation, convenience and the like. For example, a different radioisotope could be used such as <sup>32</sup> p, <sup>3</sup> H, or <sup>14</sup> C; a different fluorescer such as fluorescein, tetramethylrhodamine, Texas Red or 4-chloro-7- nitrobenzo-2-oxa-1-diazole (NBD); or a mixture of different labels such as radioisotopes, fluorescers and chemiluminescers. Alternatively, the primers can be selected such that the amplified extension products for each sequence are of different lengths and thus can be separated by a variety of methods known in the art. Similarily, the extension products could include a restriction fragment length polymorphism which could be used to distinguish different extension products. In these examples, each primer or its extension product can be differentiated from all the other primers when they are in a mixture. Alternatively, probes which bind to the amplified extension products could be labeled and sets of probes which distinguish alleles of a single sequence within a multiplex DNA amplification reaction may be used whether or not labelled.</p>
    <p>Each specific, different DNA sequence, which is to be detected herein, can derive from genomic DNA of the organism or exogenous DNA such as virus, bacteria or parasites. The source of genomic DNA from the organism to be tested can be blood, hair or tissue (including chorionic villi, amniotic cells, fibroblasts and biopsies). The source of DNA may be freshly obtained or have been suitably stored for extended periods of time. The DNA must be of sufficient quality to permit amplification. The genomic DNA can be prepared by a variety of techniques known to one skilled in the art.</p>
    <p>As used herein, the term "deletion" refers to those genomic DNA sequences in which one or more nucleic acid base has been deleted from the sequence and thus is no longer present in the gene. The size of the deletion can affect the sensitivity of the amplification procedure. Generally, the larger the deletion the larger the sensitivity.</p>
    <p>Any specific known nucleic acid sequence can be detected by the present method. Preferably, at least part of the sequence is deleted from the genome. It is only necessary that a sufficient number of bases at both ends of the sequence be known in sufficient detail to prepare oligonucleotide primers which will hybridize to the different strands of the desired sequence at relative positions along the sequence.</p>
    <p>The oligonucleotide primers may be prepared using any suitable method, for example, phosphotriester and phosphodiester methods or automated embodiments thereof, the synthesis of oligonucleotides on a modified solid support, the isolation from a biological source (restriction endonuclease digestion), and the generation by enzymatically directed copying of a DNA or RNA template.</p>
    <p>One embodiment of the present invention is a method for simultaneously detecting deletions at a plurality of DNA sequences, comprising the steps of: treating said DNA to form single stranded complementary strands; adding a plurality of paired oligonucleotide primers, each pair specific for a different sequence, one primer of each pair substantially complementary to a part of the sequence in the sense-strand and the other primer of each pair substantially complementary to a different part of the same sequence in the complementary anti-sense strand; annealing the plurality of primers to their complementary sequences; simultaneously extending said plurality of annealed primers from each primer's 3' terminus to synthesize an extension product complementary to the strands annealed to each primer, said extension products, after separation from the complement, serving as templates for the synthesis of an extension product from the other primer of each pair; separating said extension products from said templates to produce single-stranded molecules; amplifying said single-stranded molecules by repeating, at least once, said annealing, extending and separating steps; and identifying said amplified extension product from each different sequence.</p>
    <p>One preferred embodiment of the present invention is a method for detecting deletions at a plurality of genomic DNA sequences, wherein said sequences are selected from a group of sequences on the X and Y chromosomes. It is preferrable to detect hemizygous genes on the X and Y chromosomes, since this increases the level of sensitivity. When the procedure is used to detect the heterozygous state, it requires quantitative measurement, and thus is much less efficient than detecting the presence or absence of sequences as is done for hemizygous genes. For example, if part of an exon has been deleted the multiplex amplification method of the present invention will detect this by either failing to produce an oligonucleotide sequence or by production of an oligonucleotide sequence of a different size. Furthermore multiple exons can be screened at the same time. Thus, it is easy to detect the presence of a deletion. However, in looking at heterozygous states, where the chromosomes have one normal gene and one deleted gene, the normal gene will produce a normal product, and thus there is the necessity to measure the quantitative difference in the production of extension products.</p>
    <p>A second embodiment of the present invention is to permit simultaneous amplification of multiple, possibly unrelated sequences for the purpose of their simultaneous analysis. Such analysis may simply involve the determination of whether exogenous sequences (virus, bacteria or other parasites) are present within a sample of DNA, or might involve the detection of polymorphisms or mutations within a plurality of sequences. The polymorphisms or mutations can be detected by a variety of methods well known to those skilled in the art. The methods include, but are not limited to, direct DNA sequencing, allele-specific oligonucleotide hybridization, and competitive oligonucleotide priming.</p>
    <p>The multiplex genomic DNA amplification method is preferably used to detect X-linked diseases resulting from deletions in the genomic DNA sequence. Genetic diseases can be caused by a variety of mechanisms including mutations and deletions. The procedure described herein was developed for detection of genetic diseases which result from deletions within the genome. Examples of some X-linked diseases which are candidates for the use of multiplex genomic DNA amplification are ornithine transcarbamylase deficiency, hypoxanthine phosphoribosyltransferase deficiency, steroid sulfatase deficiency and X-linked muscular dystrophy. Other disorders on the X chromosome or genes on the Y chromosome can also be easily detected. The procedure is also applicable to the detection of any set of known point mutations within a set of genomic sequences. The procedure is also applicable to the simultaneous detection of any set of exogenous DNA sequences in a given DNA sample. The procedure is also applicable to the simultaneous detection of any set of polymorphic or variable tandemly repetitive sequences within a genone.</p>
    <p>The advantages of the multiplex amplification system are that numerous diseases or specific DNA sequence alterations can be detected in the same assay. For example, primers to hypoxanthine phosphoribosyltransferfase deficiency, steroid sulfatase deficiency, X-linked muscular dystrophy, ornithine transcarbamylase deficiency and other X-linked diseases can all be run simultaneously on the same sample. Furthermore, the multiplex amplification procedure is useful for very large genes with multiple exons, such as the dystrophin gene. Because of the large size of the dystrophin locus, Mullis type PCR amplification is not able to scan the whole gene in one assay. Thus, it is necessary for multiple site amplification within the gene to detect all possible deletions which could result in disease. Deletions at the DMD locus can encompass any of the approximately 60 plus exons which are distributed over more than 2 million bases of DNA. Virtually all of these exons are separated by large introns and so up to 60 separate reactions could be required for complete analysis of DMD deletions. To simplify this task, the present invention of a multiplex genomic DNA amplification for deletion detection can be employed to perform simultaneous examination of multiple exons. For example, oligonucleotide primers flanking separate DMD gene exons can be synthesized and combined and used for multiplex DNA applications. At present, up to at least 7 different DMD gene sequences have been examined simultaneously. The entire procedure for the multiplex amplification from start-up to photography of the results takes less than 5 hours. The relative locations of the amplified regions do not affect the results and exons have been amplified which have been separated by at least 1000 kb. The PCR amplification technique of Mullis is adequate for one and possibly two pair of primers, but when greater than two pairs of primers are used the procedure will not adequately amplify all the appropriate sequences.</p>
    <p>One skilled in the art readily appreciates that as more exon gene sequences become available the applicability of this test will expand to examine for deletions in multiple genes at the same time or examine multiple sites within the same gene at the same time. The later example is important for genes such as dystrophin which are so large that primers annealed to the ends of the gene will not traverse the whole gene sequence. Thus the necessity of doing multiple analysis to detect deletions in different regions of the gene. In addition, as specific mutations within multiple unrelated genes become known, multiplex DNA amplification can be applied to simultaneously assay for the presence of any of these mutations.</p>
    <p>Furthermore, as specific or highly variable DNA sequence polymorphisms become known in various genetic Loci, multiplex DNA amplification can be used to simultaneously analyze these polymorphisms to determine the haplotype or to determine the identity or source of DNA (genetic footprinting).</p>
    <p>The number of analyses which can be run simultaneously is unlimited, however, the upper limit is probably about 20 and is dependent on the size differences required for resolution and/or the number of labels or methods which are available to resolve the extension products. The ability to simultaneously amplify only 9 exons would allow the detection of greater than 90% of all known DMD deletions in a single reaction. The ability to simultaneously amplify even as few as 10 exons allows the rapid and simple diagnosis of DMD deletions using only a few separate reactions. Assuming that there are about 60 exons in the DMD gene and that the exons are widely separated such that primers are needed for every exon, a maximum of 6 separate assays is needed to detect all deletions in this gene. Under the same assumptions the Mullis PCR method would require 60 separate reactions to detect the deletions in this gene. Thus, as the size of the gene increases and the number of exons which cannot be detected together increases the advantages of this method are greatly enhanced. Furthermore, use of an automatic PCR apparatus (such as that produced by Perkin-Elmer/Cetus) and DNA sequencing machines will facilitate resolution and detection of amplified DNA fragments, will help automate the assay and will lead to the method being applied routinely in clinical laboratories without the need for highly trained research personnel.</p>
    <p>The following examples are offered by way of illustration and are not intended to limit the invention in any manner. In the examples all percentages are by weight, if for solids and by volume if for liquids, and all temperatures are in degrees Celsius unless otherwise noted.</p>
    <heading>EXAMPLE 1</heading> <p>The following conditions are currently in use to perform simultaneous amplification of a plurality of separate genomic regions within the human DMD gene. These conditions may need to be slightly modified depending on the particular regions to be amplified, the number and length of sequences to be amplified, and the choice of oligonucleotide primers. The time of reaction is highly dependent on the overall sequence length. Thus, as the number of amplified sequences increase and/or the length of amplified sequences increases, the time must be increased. The temperature is dependent on the length, the uniqueness of the primer sequence and the relative percentage of GC bases. The longer the primers, the higher the temperature needed. The more unique the sequence, the lower the temperature needed to amplify. GC rich primers need higher temperatures to prevent cross hybridization and to allow unique amplification. However, as the AT percentage increases, higher temperatures cause these primers to melt. Thus, these primers must be lengthened for the reaction to work.</p>
    <p>Template DNA was prepared from the tissue chosen for analysis using a variety of well-established methods known to those skilled in the art. Typically, 100 μl reaction volumes were utilized. Approximately 500 ng of DNA was added to a solution comprised of the following: 67 mM Tris-HCL [pH 8.8 at 25°]; 6.7 mM magnesium chloride; 16.6 mM ammonium sulfate; 10 mM β-mercaptoethanol; 6.7 μM ethylene diamine tetra-acetic acid (EDTA); and 170 μg/mL bovine serum albumin. This solution can be prepared beforehand and appears to be stable for very long periods of storage at -70° . The enzyme, Taq polymerase, was added to achieve a final concentration of 100 units/mL. This reaction mixture was gently mixed. The reaction mixture was overlaid with about 50 μL of paraffin oil, and the reaction vessel (preferably a 0.5 ml microcentrifuge tube) was centrifuged at 14,000×g for 10 sec. Amplification was performed either by manually transferring the reaction vessels between glycerol filled heat blocks at the appropriate temperatures, or automatically transferring the reaction vessels with a Perkin-Elmer/Cetus thermocycler using the `step-cycle` functions. The reaction was controlled by regulated and repetitive temperature changes of various duration. Initially the reaction was heated to 94° for 7 minutes. Subsequently 25 cycles of the following temperature durations were applied: 94° for 1 minute, then 55° for 45 seconds, then 65° for 3 1/2 minutes. Following completion of the final cycle the reaction was incubated at 65° for an additional 7 minutes. Reactions were then stored at 4° until analysis.</p>
    <p>Genomic DNA deletions and/or exogenous DNA sequences were determined by examining the amplification products. For example, the lack of an expected amplification product indicates a deletion. Many methods for this determination are known to those skilled in the art. The preferred method involves electrophoresis of about one-twentieth of the reaction on a 1.4% (weight/vol) agarose gel in the following buffer: 40 mM tris-HCl; 20 mM sodium acetate, 1 mM EDTA (adjusted to pH 7.2 with glacial acetic acid), and 0.5 μg/μl. of ethidium bromide. Electrophoresis was performed at 3.7 volts/cM for 100 minutes per 14 cM of agarose gel length. Analysis was completed by examining the electrophoresed reaction products on an ultraviolet radiation transilluminator, and the results were photographed for permanent records.</p>
    <p>When the analysis requires determination of DNA sequence polymorphisms or mutations within individual amplification products the agarose gel is transferred to an appropriate DNA binding medium such as nitrocellulose using well-established procedures, for example, Southern blotting. Individual DNA sequences within the amplified DNA fragments can be determined by a variety of techniques including allele-specific oligonucleotide hybridization. Alternatively, reaction products may be further analyzed prior to electrophoresis on agarose gel by competitive oligonucleotide primer amplification, using separate allele-specific primers for each amplified DNA sequence of the multiplex amplification reaction products.</p>
    <p>A third method for determining DNA sequence differences within individual amplification products does not require electrophoresis. In this method, aliquots of the multiplex amplification reaction are sequentially applied to an appropriate DNA binding membrane such as nitrocellulose, and then each aliquot is analyzed via hybridization with individual members of sets of allele-specific oligonucleotide (ASO) probes, each separate aliquot being hybridized with one member of a pair of ASO probes specific for one member of the multiply amplified DNA sequences.</p>
    <heading>EXAMPLE 2</heading> <p>FIG. 1 is a schematic representation of the DMD locus. The relative location of the exons used in the DMD gene amplification examples are illustrated.</p>
    <p>For detection of DMD, a variety of probes can be used either in individual PCR reactions or in combinations in multiplex PCR reactions. These probes are shown in Table 1.</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">                                  TABLE 1__________________________________________________________________________Summary of DMD gene multiplex amplification primer sets.                                             Tm*Exon and Size       Primer Sequence          Amplified                                       Deleted                                             °C.__________________________________________________________________________a. Exon 8   F-GTCCTTTACACACTTTACCTGTTGAG                                360 bp 11.3% 73.0   (182 bp) R-GGCCTCATTCTCATGTTCTAATTAG           73.0b. Exon 17  F-GACTTTCGATGTTGAGATTACTTTCCC                                416 bp  9.4% 77.4   (178 bp) R-AAGCTTGAGATGCTCTCACCTTTTCC          79.9c. Exon 19  F-TTCTACCACATCCCATTTTCTTCCA                                459 bp 10.3% 78.1   (88 bp)  R-GATGGCAAAAGTGTTGAGAAAAAGTC          77.0d. 4.1 Kb   F-CTTGATCCATATGCTTTTACCTGCA                                268 bp  4.0% 76.9   Hind III R-TCCATCACCCTTCAGAACCTGATCT           79.3   (148 bp)e. 0.5 Kb   F-AAACATGGAACATCCTTGTGGGGAC                                547 bp  8.4% 81.3   Hind III R-CATTCCTATTAGATCTGTCGCCCTAC          76.3   (176 bp)f. 1.2/3.8 Kb       F-TTGAATACATTGGTTAAATCCCAACATG                                506 bp 18.2% 78.8   Hind III R-CCTGAATAAAGTCTTCCTTACCACAC          74.3   (159 bp)g. Exon 12  F-GATAGTGGGCTTTACTTACATCCTTC                                337 bp  9.6% 73.7   (151 bp) R-GAAAGCACGCAACATAAGATACACCT          77.4                                Total:   38%__________________________________________________________________________ *Tm = melting temperature, i.e., that temperature at which 50% of strands are dissociated at 1M monovalent cation concentration, calculated by the nearest neighbor method.</pre>
    
    <p>In Table 1 each exon is designated a, b, c, d, e, f, or g and corresponds to the same letter in FIG. 1. When the exon number is known it is listed. If the exon number is not known, the size of the genomic Hind III fragment containing that exon is listed. Also shown is the size of the exon in base pairs (bp). The PCR primer sequences are shown in 5'--3' orientation. The forward primer (F), hybridizes 5' of the exon, and the reverse primer (R), hybridizes 3' of the exon. The size of the amplified fragment obtained with each primer set is also shown.</p>
    <p>The percentage of analyzed DMD patients that are deleted for each indicated exon is shown in column four. This total number is less than the sum of the individual exon deletion frequencies because many deletions encompass multiple exons.</p>
    <p>In Table 2 are the exon and flanking intron sequences for Exon 17. The exon is from 227 to 402. The primer sequences used to amplify this sequence are 7 to 33 and 396 to 421.</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">                                  TABLE 2__________________________________________________________________________5'10      20        30        40        50TAAATTGACT     TTCGATGTTG               AGATTACTTT                         CCCTTGCTAT                                   TTCAGTGAAC60        70        80        90        100CAAACTTAAG     TCAGATAAAA               CAATTTTATT                         TGGCTTCAAT                                   ATGGTGCTAT110       120       130       140       150TTTGATCTGA     AGGTCAATCT               ACCAACAAGC                         AAGAACAGTT                                   TCTCATTATT160       170       180       190       200TTCCTTTGCC     ACTCCAAGCA               GTCTTTACTG                         AAGTCTTTCG                                   AGCAATGTCT210       220       230       240       250GACCTCTGTT     TCAATACTTC               TCACAGATTT                         CACAGGCTGT                                   CACCACCACT260       270       280       290       300CAGCCATCAC     TAACACAGAC               AACTGTAATG                         GAAACAGTAA                                   CTACGGTGAC310       320       330       340       350CACAAGGGAA     CAGATCCTGG               TAAAGCATGC                         TCAAGAGGAA                                   CTTCCACCAC360       370       380       390       400CACCTCCCCA     AAAGAAGAGG               CAGATTACTG                         TGGATTCTGA                                   AATTAGGAAA410       420       430       440       450AGGTGAGAGC     ATCTCAAGCT               TTTATCTGCA                         AATGAAGTGG                                   AGAAAACTCA460       470       480       490       500TTTACAGCAG     TTTTGTTGGT               GGTGTTTTCA                         CTTCAGCAAT                                   ATTTCCAGAA510       520       530       540       550TCCTCGGGTA     CCTGTAATGT               CAGTTAATGT                         AGTGAGAAAA                                   ATTATGAAGT560       570       580       590       600ACATTTTAAA     ACTTTCACAA               GAAATCACTA                         TCGCAACAGA                                   AACTAAATGC610       620       630       640       650TTAATGGAAA     TGGTGTTTTC               TGGGGTGAAA                         GAAGAAACTA                                   TAGAAACTAT660       670       680       690       700AGGTGATAAA     CTACTGTGGT               AGCATTTTAA                         TCCTAAAAGT                                   TTCTTTCTTT710       720       730       740       750CTTTTTTTTT     TTTCTTCCTT               ATAAAGGGCC                         TGCTTGTTGA                                   GTCCCTAGTT760       770       780       790       800TTGCATTAAA     TGTCTTTTTT               TTCCAGTAAC                         GGAAAGTGCA                                   TTTTCATGAA810       820       830       840       850GAAGTACACC     TATAATAGAT               GGGATCCATC                         CTGGTAGTTT                                   ACGAGAACAT860       870       880       890       900GATGTCTCAG     TCTGCGCATC               CTAAATCAGG                         AGTAATTACA                                   GAACACATTT910       920       930       940       950CCTGTTCTTT     GATATTTATA               AAGTCTTATC                         TTGAAGGTGT                                   TAGAATTTTT960       970       980       990       1000AACTGATCTT     TTTGTGACTA               TTCAGAATTA                         TGCATTTTAG                                   ATAAGATTAG1010      1020      1030      1040GTATTATGTA     AATCAGTGGA               TATATTAAAT                         GATGGCAATA                                   A-3'__________________________________________________________________________</pre>
    
    <p>In Table 3 is the exon and flanking intron sequences for Exon d of Table 1 [or, the exon located on a 4.1 Kb Hind III fragment]. The exon is from 295 to 442. The primer sequences used to amplify this sequence are 269 to 293 and 512 to 536.</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">                                  TABLE 3__________________________________________________________________________5'10      20        30        40        50TGTCCAAAAT     AGTTGACTTT               CTTTCTTTAA                         TCAATAAATA                                   TATTACTTTA60        70        80        90        100AAGGGAAAAA     TTGCAACCTT               CCATTTAAAA                         TCAGCTTTAT                                   ATTGAGTATT110       120       130       140       150TTTTTAAAAT     GTTGTGTGTA               CATGCTAGGT                         GTGTATATTA                                   ATTTTTATTT160       170       180       190       200GTTACTTGAA     ACTAAACTCT               GCAAATGCAG                         GAAACTATCA                                   GAGTGATATC210       220       230       240       250TTTGTCAGTA     TAACCAAAAA               ATATACGCTA                         TATCTCTATA                                   ATCTGTTTTA260       270       280       290       300CATAATCCAT     CTATTTTTCT               TGATCCATAT                         GCTTTTACCT                                   GCAGGCGATT310       320       330       340       350TGACAGATCT     GTTGAGAAAT               GGCGGCGTTT                         TCATTATGAT                                   ATAAAGATAT360       370       380       390       400TTAATCAGTG     GCTAACAGAA               GCTGAACAGT                         TTCTCAGAAA                                   GACACAAATT410       420       430       440       450CCTGAGAATT     GGGAACATGC               TAAATACAAA                         TGGTATCTTA                                   AGGTAAGTCT460       470       480       490       500TTGATTTGTT     TTTTCGAAAT               TGTATTTATC                         TTCAGCACAT                                   CTGGACTCTT510       520       530       540       550TAACTTCTTA     AAGATCAGGT               TCTGAAGGGT                         GATGGAAATT                                   ACTTTTGACT560       570       580GTTGTTGTCA     TCATTATATT               ACTAGAAAGA                         AAA-3'__________________________________________________________________________</pre>
    
    <p>In Table 4 is the exon and flanking intron sequences for Exon e of Table 1 [0.5 Kb Hind III fragment exon]. The exon is from 396 to 571. The primer sequences used to amplify this sequence are 51 to 75 and 572 to 597.</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">                                  TABLE 4__________________________________________________________________________5'10      20        30        40        50ACCCAAATAC     TTTGTTCATG               TTTAAATTTT                         ACAACATTTC                                   ATAGACTATT60        70        80        90        100AAACATGGAA     CATCCTTGTG               GGGACAAGAA                         ATCGAATTTG                                   CTCTTGAAAA110       120       130       140       150GGTTTCCAAC     TAATTGATTT               GTAGGACATT                         ATAACATCCT                                   CTAGCTGACA160       170       180       190       200AGCTTACAAA     AATAAAAACT               GGAGCTAACC                         GAGAGGGTGC                                   TTTTTTCCCT210       220       230       240       250GACACATAAA     AGGTGTCTTT               CTGTCTTGTA                         TCCTTTGGAT                                   ATGGGCATGT260       270       280       290       300CAGTTTCATA     GGGAAATTTT               CACATGGAGC                         TTTTGTATTT                                   CTTTCTTTGC310       320       330       340       350CAGTACAACT     GCATGTGGTA               GCACACTGTT                         TAATCTTTTC                                   TCAAATAAAA360       370       380       390       400AGACATGGGG     CTTCATTTTT               GTTTTGCCTT                         TTTGGTATCT                                   TACAGGAACT410       420       430       440       450CCAGGATGGC     ATTGGGCAGC               GGCAAACTGT                         TGTCAGAACA                                   TTGAATGCAA460       470       480       490       500CTGGGGAAGA     AATAATTCAG               CAATCCTCAA                         AAACAGATGC                                   CAGTATTCTA510       520       530       540       550CAGGAAAAAT     TGGGAAGCCT               GAATCTGCGG                         TGGCAGGAGG                                   TCTGCAAACA560       570       580       590       600GCTGTCAGAC     AGAAAAAAGA               GGTAGGGCGA                         CAGATCTAAT                                   AGGAATGAAA610       620ACATTTTAGC     AGACTTTTTA               AGCTT-3'__________________________________________________________________________</pre>
    
    <p>In Table 5 is the exon and flanking intron sequences for Exon f, Table 1 [overlaps the 1.2 Kb and 3.8 Kb Hind III fragments]. The exon is from 221 to 406. The primer sequences used to amplify this sequence are 26 to 53 and 516 to 541.</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">                                  TABLE 5__________________________________________________________________________5'10      20        30        40        50TTTTGTAGAC     GGTTAATGAA               TAATTTTGAA                         TACATTGGTT                                   AAATCCCAAC60        70        80        90        100ATGTAATATA     TGTAAATAAT               CAATATTATG                         CTGCTAAAAT                                   AACACAAATC110       120       130       140       150AGTAAGATTC     TGTAATATTT               CATGATAAAT                         AACTTTTGAA                                   AATATATTTT160       170       180       190       200TAAACATTTT     GCTTATGCCT               TGAGAATTAT                         TTACCTTTTT                                   AAAATGTATT210       220       230       240       250TTCCTTTCAG     GTTTCCAGAG               CTTTACCTGA                         GAAACAAGGA                                   GAAATTGAAG260       270       280       290       300CTCAAATAAA     AGACCTTGGG               CAGCTTGAAA                         AAAAGCTTGA                                   AGACCTTGAA310       320       330       340       350GAGCAGTTAA     ATCATCTGCT               GCTGTGGTTA                         TCTCCTATTA                                   GGAATCAGTT360       370       380       390       400GGAAATTTAT     AACCAACCAA               ACCAAGAAGG                         ACCATTTGAC                                   GTTAAGGTAG410       420       430       440       450GGGAACTTTT     TGCTTTAATA               TTTTTGTCTT                         TTTTAAGAAA                                   AATGGCAATA460       470       480       490       500TCACTGAATT     TTCTCATTTG               GTATCATTAT                         TAAAGACAAA                                   ATATTACTTG510       520       530       540       550TTAAAGTGTG     GTAAGGAAGA               CTTTATTCAG                         GATAACCACA                                   ATAGGCACAG560       570       580       590       600GGACCACTGC     AATGGAGTAT               TACAGGAGGT                         TGGATAGAGA                                   GAGATTGGGC610       620       630       640       650TCAACTCTAA     ATACAGCACA               GTGGAAGTAG                         GAATTTATAG                                   C-3'__________________________________________________________________________</pre>
    
    <p>In Table 6 is the exon and flanking intron sequences for Exon 12. The exon is from 180 to 329. The primer sequences used to amplify this sequence are 27 to 52 and 332 to 357.</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">                                  TABLE 6__________________________________________________________________________5'10      20        30        40        50TGAGAAATAA     TAGTTCCGGG               GTGACTGATA                         GTGGGCTTTA                                   CTTACATCCT60        70        80        90        100TCTCAATGTC     CAATAGATGC               CCCCAAATGC                         GAACATTCCA                                   TATATTATAA110       120       130       140       150ATTCTATTGT     TTTACATTGT               GATGTTCAGT                         AATAAGTTGC                                   TTTCAAAGAG160       170       180       190       200GTCATAATAG     GCTTCTTTCA               AATTTTCAGT                         TTACATAGAG                                   TTTTAATGGA210       220       230       240       250TCTCCAGAAT     CAGAAACTGA               AAGAGTTGAA                         TGACTGGCTA                                   ACAAAACAGA260       270       280       290       300AGAAAGAACA     AGGAAAATGG               AGGAAGAGCC                         TCTTGGACCT                                   GATCTTGAAG310       320       330       340       350ACCTAAAACG     CCAAGTACAA               CAACATAAGG                         TAGGTGTATC                                   TTATGTTGCG360       370       380       390       400TGCTTTCTAC     TAGAAAGCAA               ACTCTGTGTA                         TAGTACCTAT                                   ACACAGTAAC410       420       430       440       450ACAGATGACA     TGGTTGATGG               GAGAGAATTA                         AAACTTAAAG                                   TCAGCCATAT460       470       480       490       500TTTAAAAATT     ATTTTTACCT               AATTGTTTTT                         GCAATCTTTG                                   TTGCCAATGG510       520       530       540       550CCTTGAATAA     GTCCCCTCCA               AAATTCAGGT                         GATTGTATTA                                   GGAGATGGAA560       570       580       590       600TATTTAAGGG     TGAATAATCC               ATCAGGGCTC                         CTCCCTTAAG                                   AATAGGATCA610       620       630       640       650AGTCCCATAT     AAAAGAGGCT               TCACACAGTG                         TTCTCCTATC                                   TCTTGACCCT660       670       680       690       700CCACCATGCA     CCACCATGTG               AAAACTCTGT                         GAAAAGGCCC                                   TCACCAGATG710       720       730       740       750CTAACATCTT     GATCTTGGAT               TTCCCAAACT                         CGAGAACTGT                                   GAAAAAATAA760       770       780       790       800AGGTACATTC     TTCCTAAATT               ACCTCATTCT                         CATTTAAACA                                   CACAAAGTGC810ACACATAGCT     G-3'__________________________________________________________________________</pre>
    
    <p>In Table 7 is the exon and flanking intron sequences for the Exon located on a 10 Kb Hind III fragment. The exon is from 1 to 150.</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">                                  TABLE 7__________________________________________________________________________5'10      20        30        40        50TTACTGGTGG     AAGAGTTGCC               CCTGCGCCAG                         GGAATTCTCA                                   AACAATTAAA60        70        80        90        100TGAAACTGGA     GGACCCGTGC               TTGTAAGTGC                         TCCCATAAGC                                   CCAGAAGAGC110       120       130       140       150AAGATAAACT     TGAAAATAAG               CTCAAGCAGA                         CAAATCTCCA                                   GTGGATAAAG160       170       180       190       200GTTAGACATT     AACCATCTCT               TCCGTCACAT                         GTGTTAAATG                                   TTGCAAGTAT210       220       230       240       250TTGTATGTAT     TTTGTTTCCT               GGGTGCTTCA                         TTGGTCGGGG                                   AGGAGGCTGG260       270       280TATGTGGATT     GTTGTTTTGT               TTTGTTTTTT-3'__________________________________________________________________________</pre>
    
    <p>In Table 8 is the exon and flanking intron sequences for the Exon located on a 1.6 Kb Hind III fragment from 512 to 622.</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">                                  TABLE 8__________________________________________________________________________5'10      20        30        40        50AAGCTTTGAT     ACTGTGCTTT               AAGTGTTTAC                         CCTTTGGAAA                                   GAAAATAATT60        70        80        90        100TTGACAGTGA     TGTAGAAATA               ATTATTTGAT                         ATTTATTTCA                                   AAACAAAATT110       120       130       140       150TATATCCAAT     ACTAAACACA               GAATTTTGTA                         AAACAATAAG                                   TGTATAAAGT160       170       180       190       200AAAATGAACA     TTAGGATTAT               TGAGATTATT                         GTAGCTAAAA                                   CTAGTGTTTA210       220       230       240       250TTCATATAAA     TTATGTTAAT               AAATTGTATT                         GTCATTATTG                                   CATTTTACTT260       270       280       290       300TTTTGAAAAG     TAGTTAATGC               CTGTGTTTCT                         ATATGAGTAT                                   TATATAATTC310       320       330       340       350AAGAAGATAT     TGGATGAATT               TTTTTTTTAA                         GTTTAATGTG                                   TTTCACATCT360       370       380       390       400CTGTTTCTTT     TCTCTGCACC               AAAAGTCACA                         TTTTTGTGCC                                   CTTATGTACC410       420       430       440       450AGGCAGAAAT     TGATCTGCAA               TACATGTGGA                         GTCTCCAAGG                                   GTATATTTAA460       470       480       490       500ATTTAGTAAT     TTTATTGCTA               ACTGTGAAGT                         TAATCTGCAC                                   TATATGGGTT510       520       530       540       550CTTTTCCCCA     GGAAACTGAA               ATAGCAGTTC                         AAGCTAAACA                                   ACCGGATGTG560       570       580       590       600GAAGAGATTT     TGTCTAAAGG               GCAGCATTTG                         TACAAGGAAA                                   AACCAGCCAC610       620       630       640       650TCAGCCAGTG     AAGGTAATGA               AGCAACCTCT                         AGCAATATCC                                   ATTACCTCAT660       670       680       690       700AATGGGTTAT     GCTTCGCCTG               TTGTACATTT                         GCCATTGACG                                   TGGACTATTT710       720       730       740       750ATAATCAGTG     AAATAACTTG               TAAGGAAATA                         CTGGCCATAC                                   TGTAATAGCA760       770       780       790       800GAGGCAAAGC     TGTCTTTTTG               ATCAGCATAT                         CCTATTTATA                                   TATTGTGATC810       820       830       840TTAAGGCTAT     TAACGAGTCA               TTGCTTTAAA                         GGACTCATTT                                   CTGTC-3'__________________________________________________________________________</pre>
    
    <p>In Table 9 is the exon and flanking intron sequences for the Exon located on a 3.1 Kb Hind III fragment, The exon is from 519 to 751.</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">                                  TABLE 9__________________________________________________________________________5'103     113       123       133       143CCCATCTTGT     TTTGCCTTTG               TTTTTTCTTG                         AATAAAAAAA                                   AAATAAGTAA153       163       173       183       193AATTTATTTC     CCTGGCAAGG               TCTGAAAACT                         TTTGTTTTCT                                   TTACCACTTC203       213       223       233       243CACAATGTAT     ATGATTGTTA               CTGAGAAGGC                         TTATTTAACT                                   TAAGTTACTT253       263       273       283       293GTCCAGGCAT     GAGAATGAGC               AAAATCGTTT                         TTTAAAAAAT                                   TGTTAAATGT303       313       323       333       343ATATTAATGA     AAAGGTTGAA               TCTTTTCATT                         TTCTACCATG                                   TATTGCTAAA353       363       373       383       393CAAAGTATCC     ACATTGTTAG               AAAAAGATAT                         ATAATGTCAT                                   GAATAAGAGT403       413       423       433       443TTGGCTCAAA     TTGTTACTCT               TCAATTAAAT                         TTGACTTATT                                   GTTATTGAAA453       463       473       483       493TTGGCTCTTT     AGCTTGTGTT               TCTAATTTTT                         CTTTTTCTTC                                   TTTTTTCCTT503       513       523       533       543TTTGCAAAAA     CCCAAAATAT               TTTAGCTCCT                         ACTCAGACTG                                   TTACTCTGGT553       563       573       583       593GACACAACCT     GTGGTTACTA               AGGAAACTGC                         CATCTCCAAA                                   CTAGAAATGC603       613       623       633       643CATCTTCCTT     GATGTTGGAG               GTACCTGCTC                         TGGCAGATTT                                   CAACCGGGCT653       663       673       683       693TGGACAGAAC     TTACCGACTG               GCTTTCTCTG                         CTTGATCAAG                                   TTATAAAATC703       713       723       733       743ACAGAGGGTG     ATGGTGGGTG               ACCTTGAGGA                         TATCAACGAG                                   ATGATCATCA753       763       773       783       793AGCAGAAGGT     ATGAGAAAAA               ATGATAAAAG                         TTGGCAGAAG                                   TTTTTCTTTA803       813       823       833       843AAATGAAGAT     TTTCCACCAA               TCACTTTACT                         CTCCTAGACC                                   ATTTCCCACC853       863       873       883       893AGTTCTTAGG     CAACTGTTTC               TCTCTCAGCA                         AACACATTAC                                   TCTCACTATT903       913       923       933       943CAGCCTAAGT     ATAATCAGGT               ATAAATTAAT                         GCAAATAACA                                   AAAGTAGCCA953       963       973       983       993TACATTAAAA     AGGAAAATAT               ACAAAAAAAA                         AAAAAAAAAA                                   AAGCCAGAAA1003      1013CCTACAGAAT     AGTGCTCTAG               TAATTAC-3'__________________________________________________________________________</pre>
    
    <p>In Table 10 is the exon and flanking intron sequences for the Exon located on a 1.5 Kb Hind III fragment. The exon is from 190 to 337.</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">                                  TABLE 10__________________________________________________________________________5'10      20        30        40        50ATCTCTATCA     TTAGAGATCT               GAATATGAAA                         TACTTGTCAA                                   AGTGAATGAA60        70        80        90        100AATTTNNTAA     ATTATGTATG               GTTAACATCT                         TTAAATTGCT                                   TATTTTTAAA110       120       130       140       150TTGCCATGTT     TGTGTCCCAG               TTTGCATTAA                         CAAATAGTTT                                   GAGAACTATG160       170       180       190       200TTGGAAAAAA     AAATAACAAT               TTTATTCTTC                         TTTCTCCAGG                                   CTAGAAGAAC210       220       230       240       250AAAAGAATAT     CTTGTCAGAA               TTTCAAAGAG                         ATTTAAATGA                                   ATTTGTTTTA260       270       280       290       300TGGTTGGAGG     AAGCAGATAA               CATTGCTAGT                         ATCCCACTTG                                   AACCTGGAAA310       320       330       340       350AGAGCAGCAA     CTAAAAGAAA               AGCTTGAGCA                         AGTCAAGGTA                                   ATTTTATTTT360       370       380       390       400CTCAAATCCC     CCAGGGCCTG               CTTGCATAAA                         GAAGTATATG                                   AATCTATTTT410       420       430       440       450TTAATTCAAT     CATTGGTTTT               CTGCCCATTA                         GGTTATTCAT                                   AGTTCCTTGC460       470       480       490       500TAAAGTGTTT     TTCTCACAAC               TTTATTTCTT                         CTTAACCCTG                                   CAGTTCTGAA510       520       530       540       550CCAGTGCACA     TAAGAACATA               TGTATATATG                         TGTGTGTGTG                                   TATTTATATA560       570       580       590       600TACACACACA     CATATTGCAT               CTATACATCT                         ACACATATAG                                   ATGTATAGAT610       620       630       640       650TCAATATGTC     TAAAAATGTA               TATAATTCAC                         AGTTTTTATC                                   TTTGATTTGA660       670       680ATATTTAAGG     GACTGAGACT               CACACTCATA                         TACTTTT-3'__________________________________________________________________________</pre>
    
    <heading>EXAMPLE 3</heading> <heading>Prenatal Diagnosis and Detection of DMD Using PCR</heading> <p>An example of prenatal diagnosis with PCR deletion detection is demonstrated using synthesized oligonucleotide primers (set b, Table 1). This primer set corresponds to the intron sequences flanking Exon 17 of the human DMD gene, a region which has been isolated and sequenced (Table 2).</p>
    <p>The results of this analysis are shown in FIG. 2. The PCR products (one-twentieth of the total reaction) were obtained with template DNA isolated from a control male □, the male fetus being diagnosed Λ, the DMD carrier mother (O) and an affected male brother of the fetus ▪. Also shown is a DNA molecular weight standard (MW; Hae III digested φX174 DNA). The results demonstrate that the affected male carries a deletion of Exon 17, which was not amplified, but that the fetus does not carry the deletion and is therefore unaffected. These results indicate that PCR is useful in the diagnosis of DMD cases containing a deletion involving this exon.</p>
    <heading>EXAMPLE 4</heading> <heading>Multiplex Detection</heading> <p>An example of multiplex detection is shown in FIGS. 3A and 3B.</p>
    <p>This analysis was done using six primer pairs (sets a-f, Table 1) and the conditions described in Example 1. Automatic rather than manual amplification was performed. These oligonucleotide primers represent the flanking regions of six separate DMD gene exons. They were combined into a reaction vial and used for multiplex genomic DNA amplifications. Template DNA was isolated from lymphoblasts (from blood sample). Analysis was by agarose gel electrophoresis.</p>
    <p>When non-deleted DNA was used as a template, the six dispersed regions of the DMD gene were simultaneously and specificially amplified (FIG. 3A, Sample #534). Discrete deletions, which were detected with this method, are shown in FIGS. 3A and 3B. Several DNA samples containing normal, partial or total DMD gene deletions are shown. FIGS. 3A and 3B also show a DNA molecular weight standard (MW: Hae III digested φX174 DNA), and a negative control (-) where no template DNA was added to the reactions. FIG. 3A also indicates which amplified DNA fragment corresponds to which exon (a-f) of FIG. 1.</p>
    <heading>EXAMPLE 5</heading> <heading>Prenatal Diagnosis</heading> <p>Multiplex PCR has been used successfully in several prenatal diagnoses. The conditions are as described above in Example 1. FIG. 4 shows Multiplex DNA amplification for prenatal diagnosis of DMD. Shown are the results of amplification using DNA from affected males (AM; lymphoblast DNA) and male fetuses (MF; cultured amniotic fluid cell DNA) from six different families. Analysis was as described in Example 1. Both the affected male and the fetal DNA of DRL #s 521 and 531 display a deletion of region f (FIG. 1). Thus these fetuses were diagnosed as affected. In DRL #43C the affected male is deleted for all regions except f, while the fetus is unaffected. The affected male in DRL #483 is deleted for region a, while the male fetus is unaffected. Neither of the samples from DRL #s 485 or 469 displayed a deletion with this technique. Thus, if a deletion defect causes DMD in these families it occurred in an untested exon.</p>
    <heading>EXAMPLE 6</heading> <heading>Prenatal Diagnosis Using Multiplex DNA Amplification of Chorionic Villus Specimen (CVS) DNA</heading> <p>FIG. 5 demonstrates Multiplex DNA amplification from CVS DNA. Both the affected male (AM; lymphoblast DNA) and the male fetus (MF; CVS DNA) from DRL #92 display a deletion of regions e and f (FIG. 1). Thus the fetus was diagnosed as affected. CVS DNA from DRL #120 did not display a deletion with this technique. Samples were analyzed as described in Example 1. These results demonstrate that the multiplex amplification technique works well for prenatal diagnosis when CVS DNA is used as the template for amplification.</p>
    <heading>EXAMPLE 7</heading> <heading>Multiplex Amplification of Seven Separate Exons of the DMD Gene</heading> <p>This example demonstrates that seven separate DNA sequences can be simultaneously amplified using the multiplex amplification technique. Conditions were as described in Example 1. Primer sets a-g (Table 1) were added to the reaction. Thus seven exon regions of the DMD gene (FIG. 1) were amplified (FIG. 6).</p>
    <heading>EXAMPLE 8</heading> <heading>Multiplex DNA Amplification for the Simultaneous Detection of Mutations Leading to Multiple Common Genetic Diseases</heading> <p>This example describes how the multiplex amplification technique can be used to simultaneously screen a newborn male for any of the most common mutations leading to DMD, sickle-cell anemia and α<sub>1</sub> -antitrypsin deficiency. In this assay any or all of the primers sets listed in Table 1 can be used for multiplex DNA amplification to diagnose the majority of possible DMD gene deletions. Additionally, primer sets can be added to the amplification reaction to identify mutations leading to additional genetic diseases. Other primer sets include:</p>
    <p>A. 5'-TGGTCTCCTTAAACCTGTCTT-3'5'-ACACAACTGTGTTCACTAG-3'</p>
    <p>These oligonucleotides amplify a 167 bp segment of the human β-globin gene, containing the DNA base that is mutated in β<sup>s</sup> (sickle-cell) hemoglobinopathy. The presence or absence of the mutant β<sup>s</sup> sequence is then determined either by separate dot blot or Southern blot hybridization of the multiplex amplification reaction with each of two labelled allele-specific oligonucleotide (ASO) probes specific for the normal or β<sup>s</sup> sequence. The sequence of these two ASO probes is:</p>
    <p>1) Normal: 5'-CTCCTGAGGAGA-3'</p>
    <p>2) β<sup>s</sup> : 5'-CTCCTGTGGAGA-3'</p>
    <p>If dot blot hybridization is used, a separate application of DNA from the multiplex amplification reaction to a DNA membrane, such as nitrocellulose, is required for each probe that will be used in the hybridization. Hybridization of each labelled probe, whether the probes are complementary to individual alleles of a given gene or to separate genes, must be performed individually. Alternatively and preferably, two aliquots of the amplification reaction are separately electrophoresed on agarose gels and transferred to nitrocellulose or a similar membrane using Southern analysis. Each of the two Southern blots are then hybridized with one member of each labelled set of specific ASO primers. Thus each known mutant or normal allele of each DNA fragment amplified in the multiplex reaction can be determined.</p>
    <p>In addition to the above described primer sets the following oligonucleotide primers can also be added to the amplification procedure:</p>
    <p>B. 5'-ACGTGGAGTGACGATGCTCTTCCC-3'5'-GTGGGATTCACCACTTTTCCC-3'</p>
    <p>These primers produce a 450 bp DNA fragment containing the DNA base change that produces the Z allele of the α<sub>1</sub> -antitrypsin gene and leads to α<sub>1</sub> -antitrypsin deficiency. The Z allele and the normal M allele are distinguished from other alleles in the multiplex amplification reaction by hybridization with the ASO probes:</p>
    <p>1) Normal (M)allele:5'-ATCGACGAGAAA-3'</p>
    <p>2) Mutant (Z)allele:5'-ATCGACAAGAAA-3'</p>
    <p>Hybridization analysis is performed in parallel with the β-globin probes as described above.</p>
    <p>In addition, the oligonucleotides</p>
    <p>C. 5'-GAAGTCAAGGACACCGAGGAA-3'5'-AGCCCTCTGGCCAGTCCTAGTG-3'</p>
    <p>can also be added to the multiplex reaction to produce a 340 bp DNA region of the α<sub>1</sub> -antitrypsin gene that contains the DNA base change that produces the S allele and leads to α<sub>1</sub> -antitrypsin deficiency. The S allele is distinguished from other alleles in the multiplex amplification as described above for the β<sup>s</sup> and Z alleles by using the two ASO probes specific for the M and S allele:</p>
    <p>Normal (M)allele 5'-ACCTGGAAAATG-3'</p>
    <p>Mutant (S)allele 5'-ACCTGGTAAATG-3'</p>
    <p>Using the primers described in Table 1 and in A, B and C of this example, the common mutations leading to DMD, sickle cell anemia and α<sub>1</sub> -antitrypsin deficiency can be simultaneously determined.</p>
    <p>One skilled in the art will readily appreciate that the present invention is well adapted to carry out the objects and obtain the ends and advantages mentioned, as well, those inherent therein. The methods, procedures and techniques described herein are presently representative of the preferred embodiments, are intended to be exemplary, and are not intended as limitations on the scope. Changes therein and other uses will occur to those skilled in the art which are encompassed within the spirit of the invention or defined by the scope of the appended claims.</p>
    </div></div></div><div class="patent-section patent-tabular-section"><a id="backward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Cited Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP0237362A1?cl=en">EP0237362A1</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Mar 13, 1987</td><td class="patent-data-table-td patent-date-value">Sep 16, 1987</td><td class="patent-data-table-td ">F. Hoffmann-La Roche Ag</td><td class="patent-data-table-td ">Process for detecting specific nucleotide variations and genetic polymorphisms present in nucleic acids and kits therefor</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP0256630A1?cl=en">EP0256630A1</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jun 9, 1987</td><td class="patent-data-table-td patent-date-value">Feb 24, 1988</td><td class="patent-data-table-td ">Howard Hughes Medical Institute</td><td class="patent-data-table-td ">Specific probes for the identification of mutations in the human phenylalanine hydroxylase gene</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP0364255A2?cl=en">EP0364255A2</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Oct 11, 1989</td><td class="patent-data-table-td patent-date-value">Apr 18, 1990</td><td class="patent-data-table-td ">Baylor College Of Medicine</td><td class="patent-data-table-td ">Multiplex genomic DNA amplification for deletion detection</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="npl-citations"></a><div class="patent-section-header"><span class="patent-section-title">Non-Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th colspan="3"class="patent-data-table-th">Reference</th></tr></thead><tr><td class="patent-data-table-td ">1</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Byrne, B. C., et al, Detection of HIV 1 RNA Sequences by In Vitro DNA Amplification, Nucleic Acids Res. 16 (9), 1988, p. 4165.</td></tr><tr><td class="patent-data-table-td ">2</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Byrne, B. C., et al, Detection of HIV-1 RNA Sequences by In-Vitro DNA Amplification, Nucleic Acids Res. 16 (9), 1988, p. 4165.</td></tr><tr><td class="patent-data-table-td ">3</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Cai, S. P., et al, A Simple Approach to Prenatal Diagnosis of Beta Thalassemia in a Geographic Area where Multiple Mutations Occur, Blood 71 (5), 1988, pp. 1357 1360.</td></tr><tr><td class="patent-data-table-td ">4</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Cai, S. P., et al, A Simple Approach to Prenatal Diagnosis of Beta Thalassemia in a Geographic Area where Multiple Mutations Occur, Blood 71 (5), 1988, pp. 1357-1360.</td></tr><tr><td class="patent-data-table-td ">5</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Chamberlain, J. R., et al, Deletion Screening of the Duchenne Muscular Dystrophy Locus Via Multiplex DNA Amplification, Nucleic Acids Res. 16(23), 1988, pp. 11141 11156.</td></tr><tr><td class="patent-data-table-td ">6</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Chamberlain, J. R., et al, Deletion Screening of the Duchenne Muscular Dystrophy Locus Via Multiplex DNA Amplification, Nucleic Acids Res. 16(23), 1988, pp. 11141-11156.</td></tr><tr><td class="patent-data-table-td ">7</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Chamberlain, J. S., et al, Analysis of Duchenne Muscular Dystrophy Gene Mutations in Mice and Humans, Cellualr and Molecular Biology of Muscle Development, vol. 93, (Stockdale, F. and Kedes, L., eds), New York, Alan R. Liss Press, 1989, pp. 951 962.</td></tr><tr><td class="patent-data-table-td ">8</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Chamberlain, J. S., et al, Analysis of Duchenne Muscular Dystrophy Gene Mutations in Mice and Humans, Cellualr and Molecular Biology of Muscle Development, vol. 93, (Stockdale, F. and Kedes, L., eds), New York, Alan R. Liss Press, 1989, pp. 951-962.</td></tr><tr><td class="patent-data-table-td ">9</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Chamberlain, J. S., et al, Expression of the Murine Duchenne Muscular Dystrophy Gene in Muscle and Brain, Science 239, 1988, pp. 1416 1418.</td></tr><tr><td class="patent-data-table-td ">10</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Chamberlain, J. S., et al, Expression of the Murine Duchenne Muscular Dystrophy Gene in Muscle and Brain, Science 239, 1988, pp. 1416-1418.</td></tr><tr><td class="patent-data-table-td ">11</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Chamberlain, J. S., et al, Expression of the Murine Duchenne Muscular Dystrophy Gene in the Muscle and Brain of Normal and Mutant MDX Mice, J. Cell. Biol. 12C:319, 1988.</td></tr><tr><td class="patent-data-table-td ">12</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Chamberlain, J. S., et al, Rapid detection of deletions at the Duchenne muscular dystropy locus via multiplex genomic DNA amplification, Amer. Journal of Human Genetics, 1988, vol. 43, p. A178, (0711) 3.2.</td></tr><tr><td class="patent-data-table-td ">13</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Chamberlain, J. S., et al., "<a href='http://scholar.google.com/scholar?q="Rapid+detection+of+deletions+at+the+Duchenne+muscular+dystrophy+locus+via+multiplex+genomic+DNA+amplification"'>Rapid detection of deletions at the Duchenne muscular dystrophy locus via multiplex genomic DNA amplification</a>" American Journal of Human Genetics 43 (3 sup) (1988) Abstract 0711.</td></tr><tr><td class="patent-data-table-td ">14</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Chamberlain, J. S., et al., Rapid detection of deletions at the Duchenne muscular dystrophy locus via multiplex genomic DNA amplification American Journal of Human Genetics 43 (3 sup) (1988) Abstract 0711.</td></tr><tr><td class="patent-data-table-td ">15</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Chehab, F. F., et al, Detection of Sickle Cell Anaemia and Thalassaemias, Nature 329: 293, 1987.</td></tr><tr><td class="patent-data-table-td ">16</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Chelly, J., et al, Transcription of the Dystrophin Gene in Muscle and Non muscle Tissues, Nature (London) 333, 1988, pp. 858 860.</td></tr><tr><td class="patent-data-table-td ">17</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Chelly, J., et al, Transcription of the Dystrophin Gene in Muscle and Non-muscle Tissues, Nature (London) 333, 1988, pp. 858-860.</td></tr><tr><td class="patent-data-table-td ">18</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Darras, B. T., et al, Direct Method for Prenatal Diagnosis and Carrier Detection in Duchenne/Becker Muscular Dystrophy Using the Entire Dystrophin cDNA, Am. J. Med. Genet. 29, 1988, pp. 713 726.</td></tr><tr><td class="patent-data-table-td ">19</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Darras, B. T., et al, Direct Method for Prenatal Diagnosis and Carrier Detection in Duchenne/Becker Muscular Dystrophy Using the Entire Dystrophin cDNA, Am. J. Med. Genet. 29, 1988, pp. 713-726.</td></tr><tr><td class="patent-data-table-td ">20</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Dilella, A. G., et al, Screening for Phenylketonuria Mutations by DNA Amplification With the Polymerase Chain Reaction, Lancet 1 (8584), 1988, pp. 497 499.</td></tr><tr><td class="patent-data-table-td ">21</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Dilella, A. G., et al, Screening for Phenylketonuria Mutations by DNA Amplification With the Polymerase Chain Reaction, Lancet 1 (8584), 1988, pp. 497-499.</td></tr><tr><td class="patent-data-table-td ">22</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Duggan, et al, HTLV I Induced Lymphoma Mimicking Hodgkin s Disease. Diagnosis by Polymerase Chain Reaction Amplification of Specific HTLV I Sequences in Tumor DNA, Blood 71 (4), 1988, pp. 1027 1032.</td></tr><tr><td class="patent-data-table-td ">23</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Duggan, et al, HTLV-I-Induced Lymphoma Mimicking Hodgkin's Disease. Diagnosis by Polymerase Chain Reaction Amplification of Specific HTLV-I Sequences in Tumor DNA, Blood 71 (4), 1988, pp. 1027-1032.</td></tr><tr><td class="patent-data-table-td ">24</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Erlich, H. A., et al, Genetic Analysis Using Enzymatic Amplification of Specific Genomic Sequences, Current Communications in Molecular Biology: DNA Probes: Applications in Genetic and Infectious Disease and Cancer, Conference, Cold Spring Harbor Apr. 20 23, 1986, Illus Paper ISBN 0 87969 196 4, 1986, pp. 107 112.</td></tr><tr><td class="patent-data-table-td ">25</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Erlich, H. A., et al, Genetic Analysis Using Enzymatic Amplification of Specific Genomic Sequences, Current Communications in Molecular Biology: DNA Probes: Applications in Genetic and Infectious Disease and Cancer, Conference, Cold Spring Harbor Apr. 20-23, 1986, Illus Paper ISBN 0-87969-196-4, 1986, pp. 107-112.</td></tr><tr><td class="patent-data-table-td ">26</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Farzadegan, H., et al, Loss of Human Immunodeficiency Virus Type 1 (HIV 1) Antibodies with Evidence of Viral Infection in Asymptomatic Homosexual Men: A Report from the Multicenter AIDS Cohort Study, Ann Intern Med. 108:6, Jun. 1988, pp. 785 790.</td></tr><tr><td class="patent-data-table-td ">27</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Farzadegan, H., et al, Loss of Human Immunodeficiency Virus Type 1 (HIV-1) Antibodies with Evidence of Viral-Infection in Asymptomatic Homosexual Men: A Report from the Multicenter AIDS Cohort Study, Ann-Intern-Med. 108:6, Jun. 1988, pp. 785-790.</td></tr><tr><td class="patent-data-table-td ">28</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Heiling, R., et al, A 230kb Cosmid Walk in the Duchenne Muscular Dystrophy Gene: Detection of a Conserved Sequence and of a Possible Deletion Prone Region, Nucl. Acids Res 15 (22), 1987, pp. 9129 9142.</td></tr><tr><td class="patent-data-table-td ">29</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Heiling, R., et al, A 230kb Cosmid Walk in the Duchenne Muscular Dystrophy Gene: Detection of a Conserved Sequence and of a Possible Deletion Prone Region, Nucl. Acids Res 15 (22), 1987, pp. 9129-9142.</td></tr><tr><td class="patent-data-table-td ">30</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Hejtmancik, J. F., et al, Carrier Diagnosis of Duchenne Muscular Dystrophy Using Restriction Fragment Length Polymorphisms, Neurology 36, 1986, pp. 1553 1562.</td></tr><tr><td class="patent-data-table-td ">31</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Hejtmancik, J. F., et al, Carrier Diagnosis of Duchenne Muscular Dystrophy Using Restriction Fragment Length Polymorphisms, Neurology 36, 1986, pp. 1553-1562.</td></tr><tr><td class="patent-data-table-td ">32</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Higuchi, R., et al, DNA Typing from Single Hairs, Nature (Lond) 332 (6164), 1988, pp. 543 546.</td></tr><tr><td class="patent-data-table-td ">33</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Higuchi, R., et al, DNA Typing from Single Hairs, Nature (Lond) 332 (6164), 1988, pp. 543-546.</td></tr><tr><td class="patent-data-table-td ">34</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Impriam, C. C., et al, Analysis of DNA Extracted from Formalin Fixed, Paraffin Embedded Tissues by Enzymatic Amplification and Hybridization with Sequence Specific Oligonucleotides, Biochem Biophys Res Commun 142 (3), 1987, pp. 710 716.</td></tr><tr><td class="patent-data-table-td ">35</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Impriam, C. C., et al, Analysis of DNA Extracted from Formalin-Fixed, Paraffin-Embedded Tissues by Enzymatic Amplification and Hybridization with Sequence-Specific Oligonucleotides, Biochem Biophys Res Commun 142 (3), 1987, pp. 710-716.</td></tr><tr><td class="patent-data-table-td ">36</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Jeffrey, A. J., et al, Amplification of Human Minisatellites by the Polymerase Chain Reaction: Towards DNA Fingerprinting of Single Cells, Nucleic Acids Res. 16(23), 1988, pp. 10953 10971.</td></tr><tr><td class="patent-data-table-td ">37</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Jeffrey, A. J., et al, Amplification of Human Minisatellites by the Polymerase Chain Reaction: Towards DNA Fingerprinting of Single Cells, Nucleic Acids Res. 16(23), 1988, pp. 10953-10971.</td></tr><tr><td class="patent-data-table-td ">38</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Kawasaki, E. S., et al, Diagnosis of Chronic Myeloid and Acute Lymphocytic Leukemias by Detection of Leukemia Specific Messenger RNA Sequences Amplified in Intro, Proc. Natl Acad Sci U S A 85 (15), 1988, pp. 5698 5702.</td></tr><tr><td class="patent-data-table-td ">39</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Kawasaki, E. S., et al, Diagnosis of Chronic Myeloid and Acute Lymphocytic Leukemias by Detection of Leukemia-Specific Messenger RNA Sequences Amplified in-Intro, Proc. Natl Acad Sci U S A 85 (15), 1988, pp. 5698-5702.</td></tr><tr><td class="patent-data-table-td ">40</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Koenig, M., et al, Complete Cloning of the Duchenne Muscular Dystrophy (DMD) cDNA and Preliminary Genomic Organization of the DMD Gene in Normal and Affected Individuals, Cell 50, 1987, pp. 507 517.</td></tr><tr><td class="patent-data-table-td ">41</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Koenig, M., et al, Complete Cloning of the Duchenne Muscular Dystrophy (DMD) cDNA and Preliminary Genomic Organization of the DMD Gene in Normal and Affected Individuals, Cell 50, 1987, pp. 507-517.</td></tr><tr><td class="patent-data-table-td ">42</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Koenig, M., et al, The Complete Sequence of Dystrophin Predicts a Rod Shaped Cytoskeletal Protein, Cell 53, 1988, pp. 219 288.</td></tr><tr><td class="patent-data-table-td ">43</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Koenig, M., et al, The Complete Sequence of Dystrophin Predicts a Rod-Shaped Cytoskeletal Protein, Cell 53, 1988, pp. 219-288.</td></tr><tr><td class="patent-data-table-td ">44</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Kogan et al. (1987) The New England Journal of Medicine, vol. 317 (16) pp. 985 990.</td></tr><tr><td class="patent-data-table-td ">45</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Kogan et al. (1987) The New England Journal of Medicine, vol. 317 (16) pp. 985-990.</td></tr><tr><td class="patent-data-table-td ">46</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Kogan, S. C., An Improved Method for Prenatal Diagnosis of Genetic Diseases by Analysis of Amplified DNA Sequences, N. Engl. J. Med. 317, 1987, pp. 985 990.</td></tr><tr><td class="patent-data-table-td ">47</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Kogan, S. C., An Improved Method for Prenatal Diagnosis of Genetic Diseases by Analysis of Amplified DNA Sequences, N. Engl. J. Med. 317, 1987, pp. 985-990.</td></tr><tr><td class="patent-data-table-td ">48</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Kunkel, L. M., Analysis of Deletions in DNA From Patients with Becker and Duchenne Muscular Dystrophy, Nature (London) 322, 1986, pp.73 77.</td></tr><tr><td class="patent-data-table-td ">49</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Kunkel, L. M., Analysis of Deletions in DNA From Patients with Becker and Duchenne Muscular Dystrophy, Nature (London) 322, 1986, pp.73-77.</td></tr><tr><td class="patent-data-table-td ">50</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Laure, F., et al, Detection of HIV 1 DNA in Infants and Children by Means of the Polymerase Chain Reaction, Lancet 2 (8610) 1988, pp. 538 541.</td></tr><tr><td class="patent-data-table-td ">51</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Laure, F., et al, Detection of HIV 1 DNA in Infants and Children by Means of the Polymerase Chain Reaction, Lancet 2 (8610) 1988, pp. 538-541.</td></tr><tr><td class="patent-data-table-td ">52</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Li, H., et al Amplification and Analysis of DNA Sequences in Single Human Sperm and Diploid Cells, Nature (Lond) 335 (6189), 1988, pp. 414 417.</td></tr><tr><td class="patent-data-table-td ">53</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Li, H., et al Amplification and Analysis of DNA Sequences in Single Human Sperm and Diploid Cells, Nature (Lond) 335 (6189), 1988, pp. 414-417.</td></tr><tr><td class="patent-data-table-td ">54</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Mullis, K., et al, Specific Enzymatic Amplification of DNA In Vitro: The Polymerase Chain Reaction, Dold Spring Harbor Laboratory, vol. 51, (parts 1 and 2), Molecular Biology of Homo Sapiens, Jun. 1986, XXV 702P, (Part 1), pp. 263 273.</td></tr><tr><td class="patent-data-table-td ">55</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Mullis, K., et al, Specific Enzymatic Amplification of DNA In-Vitro: The Polymerase Chain Reaction, Dold Spring Harbor Laboratory, vol. 51, (parts 1 and 2), Molecular Biology of Homo Sapiens, Jun. 1986, XXV+702P, (Part 1), pp. 263-273.</td></tr><tr><td class="patent-data-table-td ">56</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Murakawa, G. J., et al, Direct Detection of HIV 1 RNA from AIDS and ARC Patient Samples, DNA (N Y) 7 (4), 1988, pp. 287 295.</td></tr><tr><td class="patent-data-table-td ">57</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Murakawa, G. J., et al, Direct Detection of HIV-1 RNA from AIDS and ARC Patient Samples, DNA (N Y) 7 (4), 1988, pp. 287-295.</td></tr><tr><td class="patent-data-table-td ">58</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Newton, C. R., et al, Diagnosis of Alpha 1 Antitrypsin Deficiency by Enzymatic Amplification of Human Genomic DNA and Direct Sequencing of Polymerase Chain Reaction Products, Nucleic Acids Res. 16 (17), 1988, pp. 8233 8243.</td></tr><tr><td class="patent-data-table-td ">59</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Newton, C. R., et al, Diagnosis of Alpha-1 Antitrypsin Deficiency by Enzymatic Amplification of Human Genomic DNA and Direct Sequencing of Polymerase Chain Reaction Products, Nucleic Acids Res. 16 (17), 1988, pp. 8233-8243.</td></tr><tr><td class="patent-data-table-td ">60</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Ou, C Y., et al, DNA Amplification for Direct Detection of HIV 1 in DNA of Peripheral Blood Mononuclear Cells, Science (Wash D C) 239 (4837), 1988, pp. 295 297.</td></tr><tr><td class="patent-data-table-td ">61</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Ou, C-Y., et al, DNA Amplification for Direct Detection of HIV-1 in DNA of Peripheral Blood Mononuclear Cells, Science (Wash D C) 239 (4837), 1988, pp. 295-297.</td></tr><tr><td class="patent-data-table-td ">62</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Saiki, R. K., et al, Analysis of Enzymatically Amplified B globin and HLA DQa DNA with Allele Specific Oligonucleotide Probes, Nature (London) 324, 1986, pp. 163 166.</td></tr><tr><td class="patent-data-table-td ">63</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Saiki, R. K., et al, Analysis of Enzymatically Amplified B-globin and HLA-DQa-DNA with Allele-Specific Oligonucleotide Probes, Nature (London) 324, 1986, pp. 163-166.</td></tr><tr><td class="patent-data-table-td ">64</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Saiki, R. K., et al, Diagnosis of Sickle Cell Anemia and B Thalasemia with Enzymatically Amplified DNA and Non Radioactive Allele Specific Oligonucleotide Probes, N. Engl. J. Med. 319, 1988, pp. 537 541.</td></tr><tr><td class="patent-data-table-td ">65</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Saiki, R. K., et al, Diagnosis of Sickle Cell Anemia and B Thalasemia with Enzymatically Amplified DNA and Non-Radioactive Allele-Specific Oligonucleotide Probes, N. Engl. J. Med. 319, 1988, pp. 537-541.</td></tr><tr><td class="patent-data-table-td ">66</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Saiki, R. K., et al, Primer Directed Enzymatic Amplification of DNA with Thermostable DNA Polymerase, Science (Wash D C) 239 (4839), 1988, pp. 487 491.</td></tr><tr><td class="patent-data-table-td ">67</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Saiki, R. K., et al, Primer-Directed Enzymatic Amplification of DNA with Thermostable DNA Polymerase, Science (Wash D C) 239 (4839), 1988, pp. 487-491.</td></tr><tr><td class="patent-data-table-td ">68</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Scharf, S. J., et al, Direct Cloning and Sequence Analysis of Enzymatically Amplified Genomic Sequences, Science (Wash D C) 233 (4768), 1986, pp. 1076 1078.</td></tr><tr><td class="patent-data-table-td ">69</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Scharf, S. J., et al, Direct Cloning and Sequence Analysis of Enzymatically Amplified Genomic Sequences, Science (Wash D C) 233 (4768), 1986, pp. 1076-1078.</td></tr><tr><td class="patent-data-table-td ">70</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Stoflet, E. S., et al., "<a href='http://scholar.google.com/scholar?q="Genomic+Amplificaion+with+Transcript+Sequencing"'>Genomic Amplificaion with Transcript Sequencing</a>" Science 239:491-494 (1988).</td></tr><tr><td class="patent-data-table-td ">71</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Stoflet, E. S., et al., Genomic Amplificaion with Transcript Sequencing Science 239:491 494 (1988).</td></tr><tr><td class="patent-data-table-td ">72</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Wong, C., et al, Characterization of Beta Thalassemia Mutations Using Direct Genomic Sequences of Amplified Single Copy DNA, Nature (Lond) 330 (6146), 1987, pp. 384 386.</td></tr><tr><td class="patent-data-table-td ">73</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Wong, C., et al, Characterization of Beta-Thalassemia Mutations Using Direct Genomic Sequences of Amplified Single Copy DNA, Nature (Lond) 330 (6146), 1987, pp. 384-386.</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="forward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Referenced by</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Citing Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6048689">US6048689</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Mar 28, 1997</td><td class="patent-data-table-td patent-date-value">Apr 11, 2000</td><td class="patent-data-table-td ">Gene Logic, Inc.</td><td class="patent-data-table-td ">Screening for presence or absence of sequence polymorphism in gene sample; performing allele preferential hybridization, if no sequence variation found, then perform sequence location assay, if no sequence polymorphism found sequence gene</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6063567">US6063567</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jan 7, 1997</td><td class="patent-data-table-td patent-date-value">May 16, 2000</td><td class="patent-data-table-td ">Visible Genetics Inc.</td><td class="patent-data-table-td ">Method, reagents and kit for diagnosis and targeted screening for retinoblastoma</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6124092">US6124092</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jun 12, 1997</td><td class="patent-data-table-td patent-date-value">Sep 26, 2000</td><td class="patent-data-table-td ">The Perkin-Elmer Corporation</td><td class="patent-data-table-td ">Generating a nucleotide sequencing ladders; mixing recoverable primers with nucleotide sequencing templates, binding nucleotide sequencing ladders to recovery tag binding compounds, recovering nucleotide sequence ladders</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6197510">US6197510</a></td><td class="patent-data-table-td patent-date-value">Oct 1, 1998</td><td class="patent-data-table-td patent-date-value">Mar 6, 2001</td><td class="patent-data-table-td ">Bio-Id Diagnostic Inc.</td><td class="patent-data-table-td ">Sequencing by using mixture of two nucleic acids, adding labeled sequencing primers of 2 different lengths, running polymerase chain reaction with chain stopping nucleotide, differentiating products based on weight</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6428964">US6428964</a></td><td class="patent-data-table-td patent-date-value">Mar 15, 2001</td><td class="patent-data-table-td patent-date-value">Aug 6, 2002</td><td class="patent-data-table-td ">Exact Sciences Corporation</td><td class="patent-data-table-td ">Adding to a biological sample suspected to contain a target nucleic acid, single-stranded nucleic acids that hybridize to a region of target nucleic acid if this region is unaltered, adding an agent to degrade single stranded nucleic acids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6472185">US6472185</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">May 25, 2000</td><td class="patent-data-table-td patent-date-value">Oct 29, 2002</td><td class="patent-data-table-td ">Pioneer Hi-Bred International, Inc.</td><td class="patent-data-table-td ">Use of selective DNA fragment amplification products for hybridization-based genetic fingerprinting, marker assisted selection, and high-throughput screening</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6489146">US6489146</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Feb 5, 1999</td><td class="patent-data-table-td patent-date-value">Dec 3, 2002</td><td class="patent-data-table-td ">Glaxo Group Limited</td><td class="patent-data-table-td ">End-complementary polymerase reaction</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6492109">US6492109</a></td><td class="patent-data-table-td patent-date-value">Sep 21, 1998</td><td class="patent-data-table-td patent-date-value">Dec 10, 2002</td><td class="patent-data-table-td ">Gene Logic, Inc.</td><td class="patent-data-table-td ">Isolated oligonucleotide that specifically hybridizes to a brca2 nucleotide sequence having a specified deletion; gene which predisposes individuals to breast and ovarian cancer</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6514699">US6514699</a></td><td class="patent-data-table-td patent-date-value">Jun 13, 2000</td><td class="patent-data-table-td patent-date-value">Feb 4, 2003</td><td class="patent-data-table-td ">Pe Corporation (Ny)</td><td class="patent-data-table-td ">Simultaneous generation of multiple polynucleotide extension products (sequencing ladders or PCR amplification products)from recoverable primers and polynucleotide templates.</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6582920">US6582920</a></td><td class="patent-data-table-td patent-date-value">Aug 31, 2001</td><td class="patent-data-table-td patent-date-value">Jun 24, 2003</td><td class="patent-data-table-td ">Gen-Probe Incorporated</td><td class="patent-data-table-td ">Amplification of HIV-1 RT sequences for detection of sequences associated with drug-resistance mutations</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6605451">US6605451</a></td><td class="patent-data-table-td patent-date-value">Jun 6, 2000</td><td class="patent-data-table-td patent-date-value">Aug 12, 2003</td><td class="patent-data-table-td ">Xtrana, Inc.</td><td class="patent-data-table-td ">Methods and devices for multiplexing amplification reactions</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6623920">US6623920</a></td><td class="patent-data-table-td patent-date-value">Jul 7, 2000</td><td class="patent-data-table-td patent-date-value">Sep 23, 2003</td><td class="patent-data-table-td ">Gen-Probe Incorporated</td><td class="patent-data-table-td ">Human immunodeficiency virus (HIV); contacting with oligomer that hybridizes to long terminal repeat sequence; separating complex; replicating the sequences or a cDNA made therefrom; probes</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6686163">US6686163</a></td><td class="patent-data-table-td patent-date-value">Dec 20, 2001</td><td class="patent-data-table-td patent-date-value">Feb 3, 2004</td><td class="patent-data-table-td ">Gene Logic Inc.</td><td class="patent-data-table-td ">Coding sequence haplotype of the human BRCA1 gene</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6703228">US6703228</a></td><td class="patent-data-table-td patent-date-value">Sep 24, 1999</td><td class="patent-data-table-td patent-date-value">Mar 9, 2004</td><td class="patent-data-table-td ">Massachusetts Institute Of Technology</td><td class="patent-data-table-td ">Methods and products related to genotyping and DNA analysis</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6750020">US6750020</a></td><td class="patent-data-table-td patent-date-value">Nov 20, 2001</td><td class="patent-data-table-td patent-date-value">Jun 15, 2004</td><td class="patent-data-table-td ">Exact Sciences Corporation</td><td class="patent-data-table-td ">Method for alteration detection</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6838256">US6838256</a></td><td class="patent-data-table-td patent-date-value">Dec 13, 2000</td><td class="patent-data-table-td patent-date-value">Jan 4, 2005</td><td class="patent-data-table-td ">Gene Logic Inc.</td><td class="patent-data-table-td ">Coding sequences of the human BRCA1 gene</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6946254">US6946254</a></td><td class="patent-data-table-td patent-date-value">Apr 28, 2003</td><td class="patent-data-table-td patent-date-value">Sep 20, 2005</td><td class="patent-data-table-td ">Gen-Probe Incorporated</td><td class="patent-data-table-td ">Amplification of HIV-1 gag sequences for detection of sequences associated with drug-resistance mutations</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6977178">US6977178</a></td><td class="patent-data-table-td patent-date-value">Sep 3, 2002</td><td class="patent-data-table-td patent-date-value">Dec 20, 2005</td><td class="patent-data-table-td ">Transnetyx, Inc.</td><td class="patent-data-table-td ">System and method for transgenic and targeted mutagenesis screening</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7011943">US7011943</a></td><td class="patent-data-table-td patent-date-value">Sep 4, 2001</td><td class="patent-data-table-td patent-date-value">Mar 14, 2006</td><td class="patent-data-table-td ">Transnetyx, Inc.</td><td class="patent-data-table-td ">Detecting preferential nucleotide sequences in pool of sequences; obtain nucleotide sequences, deposit on substrate, incubate with probe, detect signal, which indicates, presencec of preferential sequences</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7056705">US7056705</a></td><td class="patent-data-table-td patent-date-value">Oct 7, 2002</td><td class="patent-data-table-td patent-date-value">Jun 6, 2006</td><td class="patent-data-table-td ">Samsung Electronics, Co. Ltd.</td><td class="patent-data-table-td ">Amplification with a high specificity, a high speed, a high sensitivity and a low cost through a multiplex PCR to detect a maturity-onset of diabetes of the young (MODY) 2 associated gene</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7087414">US7087414</a></td><td class="patent-data-table-td patent-date-value">May 19, 2003</td><td class="patent-data-table-td patent-date-value">Aug 8, 2006</td><td class="patent-data-table-td ">Applera Corporation</td><td class="patent-data-table-td ">Discriminately identifying nucleic acid sequences of coliform bacteria; high throughput assay</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7090975">US7090975</a></td><td class="patent-data-table-td patent-date-value">Aug 7, 2001</td><td class="patent-data-table-td patent-date-value">Aug 15, 2006</td><td class="patent-data-table-td ">Promega Corporation</td><td class="patent-data-table-td ">Pyrophosphorolysis and incorporation of nucleotide method for nucleic acid detection</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7097979">US7097979</a></td><td class="patent-data-table-td patent-date-value">Aug 1, 2003</td><td class="patent-data-table-td patent-date-value">Aug 29, 2006</td><td class="patent-data-table-td ">Gen-Probe Incorporated</td><td class="patent-data-table-td ">Detection of HIV-1 by nucleic acid amplification</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7108976">US7108976</a></td><td class="patent-data-table-td patent-date-value">Oct 14, 2002</td><td class="patent-data-table-td patent-date-value">Sep 19, 2006</td><td class="patent-data-table-td ">Affymetrix, Inc.</td><td class="patent-data-table-td ">Complexity management of genomic DNA by locus specific amplification</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7208272">US7208272</a></td><td class="patent-data-table-td patent-date-value">Dec 16, 2002</td><td class="patent-data-table-td patent-date-value">Apr 24, 2007</td><td class="patent-data-table-td ">Samsung Electronics Co., Ltd.</td><td class="patent-data-table-td ">Multiplex PCR primer set for human HNF-1α gene amplification</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7208274">US7208274</a></td><td class="patent-data-table-td patent-date-value">Feb 28, 2003</td><td class="patent-data-table-td patent-date-value">Apr 24, 2007</td><td class="patent-data-table-td ">Ravgen, Inc.</td><td class="patent-data-table-td ">Using oligonucleotide template to detect and determine sequence of restriction fragment length polymorphisms on single or multiple chromosomes; genotyping</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7208295">US7208295</a></td><td class="patent-data-table-td patent-date-value">Nov 19, 2002</td><td class="patent-data-table-td patent-date-value">Apr 24, 2007</td><td class="patent-data-table-td ">Affymetrix, Inc.</td><td class="patent-data-table-td ">Joining nucleic acid to oligonucleotide sequences; obtain oligonucleotide, incubate with primer, ligate, denature, amplify, recover chain extended oligonucleotide</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7250252">US7250252</a></td><td class="patent-data-table-td patent-date-value">Dec 21, 2000</td><td class="patent-data-table-td patent-date-value">Jul 31, 2007</td><td class="patent-data-table-td ">David Aaron Katz</td><td class="patent-data-table-td ">Amplification based polymorphism detection</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7282361">US7282361</a></td><td class="patent-data-table-td patent-date-value">Mar 8, 2005</td><td class="patent-data-table-td patent-date-value">Oct 16, 2007</td><td class="patent-data-table-td ">Transnetyx, Inc.</td><td class="patent-data-table-td ">Treating mammalian tissue with a lysis buffer of proteinase K to obtain a lysate, binding genomic DNA to magnetic particles in the presence of the lysate, and disassociating the DNA from the magnetic particles</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7332277">US7332277</a></td><td class="patent-data-table-td patent-date-value">Sep 11, 2003</td><td class="patent-data-table-td patent-date-value">Feb 19, 2008</td><td class="patent-data-table-td ">Ravgen, Inc.</td><td class="patent-data-table-td ">Methods for detection of genetic disorders</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7374877">US7374877</a></td><td class="patent-data-table-td patent-date-value">Jun 3, 2005</td><td class="patent-data-table-td patent-date-value">May 20, 2008</td><td class="patent-data-table-td ">Gen-Probe Incorporated</td><td class="patent-data-table-td ">Amplification of HIV-1 gag sequences for detection of sequences associated with drug-resistance mutations</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7405044">US7405044</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Oct 7, 2005</td><td class="patent-data-table-td patent-date-value">Jul 29, 2008</td><td class="patent-data-table-td ">Reliagene Technologies Inc.</td><td class="patent-data-table-td ">Multiplex PCR for simultaneous quantitation of human nuclear, mitochondrial, and male Y-chromosome DNA</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7442506">US7442506</a></td><td class="patent-data-table-td patent-date-value">Aug 26, 2005</td><td class="patent-data-table-td patent-date-value">Oct 28, 2008</td><td class="patent-data-table-td ">Ravgen, Inc.</td><td class="patent-data-table-td ">Methods for detection of genetic disorders</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7452671">US7452671</a></td><td class="patent-data-table-td patent-date-value">May 1, 2006</td><td class="patent-data-table-td patent-date-value">Nov 18, 2008</td><td class="patent-data-table-td ">Affymetrix, Inc.</td><td class="patent-data-table-td ">the complexity of a nucleic acid sample to interrogate a collection of target sequences; fragmenting the nucleic acid sample with a restriction enzyme that has at least one variable position in the recognition sequence; predictable and reproducible</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7459273">US7459273</a></td><td class="patent-data-table-td patent-date-value">Aug 5, 2004</td><td class="patent-data-table-td patent-date-value">Dec 2, 2008</td><td class="patent-data-table-td ">Affymetrix, Inc.</td><td class="patent-data-table-td ">Using a locus specific primer complementary to a region near a selected polymorphism; synthesizing pools of locus specific primers that incorporate some degenerate positions; amplifying extended capture probes;hybridization; analyzing hybridization pattern to determine alleles present for polymorphisms</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7479537">US7479537</a></td><td class="patent-data-table-td patent-date-value">May 11, 2005</td><td class="patent-data-table-td patent-date-value">Jan 20, 2009</td><td class="patent-data-table-td ">Abbott Laboratories Inc.</td><td class="patent-data-table-td ">Reagents and methods useful for detecting diseases of the breast</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7494817">US7494817</a></td><td class="patent-data-table-td patent-date-value">Jun 30, 2005</td><td class="patent-data-table-td patent-date-value">Feb 24, 2009</td><td class="patent-data-table-td ">Transnet Yx, Inc.</td><td class="patent-data-table-td ">Mixing cellular debris and magnetically-responsive particles (having glass microfiber coating) to facilitate reversible immobilization of genomic nucleic acid; bioassay</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7510829">US7510829</a></td><td class="patent-data-table-td patent-date-value">Nov 19, 2002</td><td class="patent-data-table-td patent-date-value">Mar 31, 2009</td><td class="patent-data-table-td ">Affymetrix, Inc.</td><td class="patent-data-table-td ">Primers for use in the amplification of preferential nucleotide sequences</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7531328">US7531328</a></td><td class="patent-data-table-td patent-date-value">Jul 7, 2005</td><td class="patent-data-table-td patent-date-value">May 12, 2009</td><td class="patent-data-table-td ">Applied Biosystem, Llc</td><td class="patent-data-table-td ">Methods and devices for multiplexing amplification reactions</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7598060">US7598060</a></td><td class="patent-data-table-td patent-date-value">Apr 15, 2005</td><td class="patent-data-table-td patent-date-value">Oct 6, 2009</td><td class="patent-data-table-td ">Ravgen, Inc.</td><td class="patent-data-table-td ">Using oligonucleotide template to detect and determine sequence of restriction fragment length polymorphisms on single or multiple chromosomes; genotyping</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7718370">US7718370</a></td><td class="patent-data-table-td patent-date-value">Dec 28, 2006</td><td class="patent-data-table-td patent-date-value">May 18, 2010</td><td class="patent-data-table-td ">Ravgen, Inc.</td><td class="patent-data-table-td ">Determining the sequence of alleles of a locus of interest, and quantitating a ratio for the alleles at the locus of interest, wherein the ratio indicates the presence or absence of a chromosomal abnormality; noninvasive; determining the sequence of DNA from a fetus</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7723040">US7723040</a></td><td class="patent-data-table-td patent-date-value">Sep 4, 2008</td><td class="patent-data-table-td patent-date-value">May 25, 2010</td><td class="patent-data-table-td ">Gen-Probe Incorporated</td><td class="patent-data-table-td ">Detection of HIV-1 by nucleic acid amplification</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7727720">US7727720</a></td><td class="patent-data-table-td patent-date-value">Aug 26, 2005</td><td class="patent-data-table-td patent-date-value">Jun 1, 2010</td><td class="patent-data-table-td ">Ravgen, Inc.</td><td class="patent-data-table-td ">Determining sequence of alleles at locus; quantitating ratio for preferential alleles; screen for allelic variants; corellate allelic variant with presence or absence of a chromosomal abnormality</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7741027">US7741027</a></td><td class="patent-data-table-td patent-date-value">Apr 9, 2008</td><td class="patent-data-table-td patent-date-value">Jun 22, 2010</td><td class="patent-data-table-td ">Gen-Probe Incorporated</td><td class="patent-data-table-td ">Nucleic acid oligomers; protease, reverse transcriptase; labeled oligonucleotide that specifically hybridizes; determine appropriate treatment</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7745119">US7745119</a></td><td class="patent-data-table-td patent-date-value">Nov 21, 2005</td><td class="patent-data-table-td patent-date-value">Jun 29, 2010</td><td class="patent-data-table-td ">Investigen, Inc.</td><td class="patent-data-table-td ">detecting the presence or amount of a target polynucleotide; detecting the presence of a disease state; detecting infection of a host by a pathogen, genetic variation, genetically modified organism, strain of an organism</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7754451">US7754451</a></td><td class="patent-data-table-td patent-date-value">Apr 24, 2007</td><td class="patent-data-table-td patent-date-value">Jul 13, 2010</td><td class="patent-data-table-td ">Affymetrix, Inc.</td><td class="patent-data-table-td ">Multiplex oligonucleotide addition and target amplification</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7771947">US7771947</a></td><td class="patent-data-table-td patent-date-value">Feb 20, 2008</td><td class="patent-data-table-td patent-date-value">Aug 10, 2010</td><td class="patent-data-table-td ">Investigen, Inc.</td><td class="patent-data-table-td ">Using substrate comprising immobilized nucleic acid analogues as tool in detecting and isolating preferential polynucleotides</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7785786">US7785786</a></td><td class="patent-data-table-td patent-date-value">Jan 23, 2006</td><td class="patent-data-table-td patent-date-value">Aug 31, 2010</td><td class="patent-data-table-td ">Quest Diagnostics Investments Incorporated</td><td class="patent-data-table-td ">Methods for detecting nucleic acids using multiple signals</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7862999">US7862999</a></td><td class="patent-data-table-td patent-date-value">Jan 17, 2008</td><td class="patent-data-table-td patent-date-value">Jan 4, 2011</td><td class="patent-data-table-td ">Affymetrix, Inc.</td><td class="patent-data-table-td ">Multiplex targeted amplification using flap nuclease</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8021839">US8021839</a></td><td class="patent-data-table-td patent-date-value">Feb 23, 2007</td><td class="patent-data-table-td patent-date-value">Sep 20, 2011</td><td class="patent-data-table-td ">Investigen, Inc.</td><td class="patent-data-table-td ">Quantitative analysis; optical properties; kits</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8039216">US8039216</a></td><td class="patent-data-table-td patent-date-value">Jul 28, 2010</td><td class="patent-data-table-td patent-date-value">Oct 18, 2011</td><td class="patent-data-table-td ">Quest Diagnostics Investments Incorporated</td><td class="patent-data-table-td ">Methods for detecting nucleic acids using multiple signals</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8114978">US8114978</a></td><td class="patent-data-table-td patent-date-value">Dec 2, 2008</td><td class="patent-data-table-td patent-date-value">Feb 14, 2012</td><td class="patent-data-table-td ">Affymetrix, Inc.</td><td class="patent-data-table-td ">Methods for genotyping selected polymorphism</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8119352">US8119352</a></td><td class="patent-data-table-td patent-date-value">Jun 19, 2007</td><td class="patent-data-table-td patent-date-value">Feb 21, 2012</td><td class="patent-data-table-td ">Cepheld</td><td class="patent-data-table-td ">Multi-stage amplification reactions by control of sequence replication times</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8133667">US8133667</a></td><td class="patent-data-table-td patent-date-value">Nov 17, 2008</td><td class="patent-data-table-td patent-date-value">Mar 13, 2012</td><td class="patent-data-table-td ">Affymetrix, Inc.</td><td class="patent-data-table-td ">Methods for genotyping with selective adaptor ligation</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8153372">US8153372</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Dec 19, 2007</td><td class="patent-data-table-td patent-date-value">Apr 10, 2012</td><td class="patent-data-table-td ">The Board Of Regents For Oklahoma State University</td><td class="patent-data-table-td ">Method for simultaneously determining in a single multiplex reaction gender of donors and quantities of genomic DNA and ratios thereof, presence and extent of DNA degradation, and PCR inhibition within a human DNA sample</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8198027">US8198027</a></td><td class="patent-data-table-td patent-date-value">Dec 20, 2007</td><td class="patent-data-table-td patent-date-value">Jun 12, 2012</td><td class="patent-data-table-td ">Gen-Probe Incorporated</td><td class="patent-data-table-td ">Methods and compositions for nucleic acid amplification</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8222023">US8222023</a></td><td class="patent-data-table-td patent-date-value">Sep 10, 2008</td><td class="patent-data-table-td patent-date-value">Jul 17, 2012</td><td class="patent-data-table-td ">Micronics, Inc.</td><td class="patent-data-table-td ">Integrated nucleic acid assays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8304184">US8304184</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jun 25, 2009</td><td class="patent-data-table-td patent-date-value">Nov 6, 2012</td><td class="patent-data-table-td ">Baback Gharizadeh</td><td class="patent-data-table-td ">Genotyping using multiple variant-specific primer pools</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8304214">US8304214</a></td><td class="patent-data-table-td patent-date-value">Nov 21, 2007</td><td class="patent-data-table-td patent-date-value">Nov 6, 2012</td><td class="patent-data-table-td ">Applied Biosystems, Llc</td><td class="patent-data-table-td ">Methods for multiplexing amplification reactions</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8323897">US8323897</a></td><td class="patent-data-table-td patent-date-value">Nov 26, 2003</td><td class="patent-data-table-td patent-date-value">Dec 4, 2012</td><td class="patent-data-table-td ">Applied Biosystems, Llc</td><td class="patent-data-table-td ">Multiplex amplification of polynucleotides</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8425861">US8425861</a></td><td class="patent-data-table-td patent-date-value">Apr 4, 2008</td><td class="patent-data-table-td patent-date-value">Apr 23, 2013</td><td class="patent-data-table-td ">Netbio, Inc.</td><td class="patent-data-table-td ">Methods for rapid multiplexed amplification of target nucleic acids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8512955">US8512955</a></td><td class="patent-data-table-td patent-date-value">Jul 1, 2010</td><td class="patent-data-table-td patent-date-value">Aug 20, 2013</td><td class="patent-data-table-td ">Gen-Probe Incorporated</td><td class="patent-data-table-td ">Methods and compositions for nucleic acid amplification</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8642268">US8642268</a></td><td class="patent-data-table-td patent-date-value">Apr 30, 2012</td><td class="patent-data-table-td patent-date-value">Feb 4, 2014</td><td class="patent-data-table-td ">Gen-Probe Incorporated</td><td class="patent-data-table-td ">Methods and compositions for nucleic acid amplification</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8673560">US8673560</a></td><td class="patent-data-table-td patent-date-value">Sep 14, 2012</td><td class="patent-data-table-td patent-date-value">Mar 18, 2014</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and compositions for multiplex PCR</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8728728">US8728728</a></td><td class="patent-data-table-td patent-date-value">Sep 14, 2012</td><td class="patent-data-table-td patent-date-value">May 20, 2014</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and compositions for multiplex PCR</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8728736">US8728736</a></td><td class="patent-data-table-td patent-date-value">Sep 14, 2012</td><td class="patent-data-table-td patent-date-value">May 20, 2014</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and compositions for multiplex PCR</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8772017">US8772017</a></td><td class="patent-data-table-td patent-date-value">Jun 8, 2012</td><td class="patent-data-table-td patent-date-value">Jul 8, 2014</td><td class="patent-data-table-td ">Micronics, Inc.</td><td class="patent-data-table-td ">Integrated nucleic acid assays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8785130">US8785130</a></td><td class="patent-data-table-td patent-date-value">Jul 6, 2006</td><td class="patent-data-table-td patent-date-value">Jul 22, 2014</td><td class="patent-data-table-td ">Bio-Id Diagnostic Inc.</td><td class="patent-data-table-td ">Use of markers including nucleotide sequence based codes to monitor methods of detection and identification of genetic material</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/CN1303216C?cl=en">CN1303216C</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Nov 1, 2002</td><td class="patent-data-table-td patent-date-value">Mar 7, 2007</td><td class="patent-data-table-td ">三星电子株式会社</td><td class="patent-data-table-td ">Multiplex PCR primer set for human glucokinase gene amplification</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP1321531A2?cl=en">EP1321531A2</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Dec 18, 2002</td><td class="patent-data-table-td patent-date-value">Jun 25, 2003</td><td class="patent-data-table-td ">SAMSUNG ELECTRONICS Co. Ltd.</td><td class="patent-data-table-td ">Multiplex pcr primer set for human hnf-1alpha gene amplification</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP1380654A1?cl=en">EP1380654A1</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Oct 30, 2002</td><td class="patent-data-table-td patent-date-value">Jan 14, 2004</td><td class="patent-data-table-td ">Samsung Electronics Co., Ltd.</td><td class="patent-data-table-td ">Multiplex PCR primer set for human glucokinase gene amplification</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP1559794A2?cl=en">EP1559794A2</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Dec 18, 2002</td><td class="patent-data-table-td patent-date-value">Aug 3, 2005</td><td class="patent-data-table-td ">Samsung Electronics Co., Ltd.</td><td class="patent-data-table-td ">Multiplex PCR primer set for human HNF-1alpha gene amplification</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2295603A1?cl=en">EP2295603A1</a></td><td class="patent-data-table-td patent-date-value">Jun 1, 2001</td><td class="patent-data-table-td patent-date-value">Mar 16, 2011</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Method and devices for multiplexing amplification reactions</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2351853A1?cl=en">EP2351853A1</a></td><td class="patent-data-table-td patent-date-value">Jun 1, 2001</td><td class="patent-data-table-td patent-date-value">Aug 3, 2011</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Method and devices for multiplexing amplification reactions</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2505591A1?cl=en">EP2505591A1</a></td><td class="patent-data-table-td patent-date-value">Feb 13, 2006</td><td class="patent-data-table-td patent-date-value">Oct 3, 2012</td><td class="patent-data-table-td ">Memorial Sloan-Kettering Cancer Center</td><td class="patent-data-table-td ">Methods and compositions for detecting a drug resistant EGFR mutant</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1998048052A2?cl=en">WO1998048052A2</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Apr 17, 1998</td><td class="patent-data-table-td patent-date-value">Oct 29, 1998</td><td class="patent-data-table-td ">Abbott Lab</td><td class="patent-data-table-td ">Amplification based mutation detection</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1999015701A1?cl=en">WO1999015701A1</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Sep 23, 1998</td><td class="patent-data-table-td patent-date-value">Apr 1, 1999</td><td class="patent-data-table-td ">Oncormed Inc</td><td class="patent-data-table-td ">SUSCEPTIBILITY MUTATION 6495delGC OF BRCA2</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2001036442A1?cl=en">WO2001036442A1</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Nov 17, 2000</td><td class="patent-data-table-td patent-date-value">May 25, 2001</td><td class="patent-data-table-td ">Xiuli Chen</td><td class="patent-data-table-td ">Simultaneous detection of hbv, hcv and hiv in plasma samples using a multiplex capture assay</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2001088174A1?cl=en">WO2001088174A1</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">May 17, 2001</td><td class="patent-data-table-td patent-date-value">Nov 22, 2001</td><td class="patent-data-table-td ">Natalia Broude</td><td class="patent-data-table-td ">Novel compositions and methods for carrying out multiple pcr reactions on a single sample</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2001094634A2?cl=en">WO2001094634A2</a></td><td class="patent-data-table-td patent-date-value">Jun 1, 2001</td><td class="patent-data-table-td patent-date-value">Dec 13, 2001</td><td class="patent-data-table-td ">Biopool Int Inc</td><td class="patent-data-table-td ">Methods and devices for multiplexing amplification reactions</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2011041611A1?cl=en">WO2011041611A1</a></td><td class="patent-data-table-td patent-date-value">Sep 30, 2010</td><td class="patent-data-table-td patent-date-value">Apr 7, 2011</td><td class="patent-data-table-td ">The Regents Of The University Of California</td><td class="patent-data-table-td ">Cofactors and methods for use for individuals</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="classifications"></a><div class="patent-section-header"><span class="patent-section-title">Classifications</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th"> </th><th class="patent-data-table-th"> </th></tr></thead><tr><td class="patent-data-table-td ">U.S. Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=hfdABAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S006180">435/6.18</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=hfdABAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S091100">435/91.1</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=hfdABAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S023100">536/23.1</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=hfdABAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S091200">435/91.2</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=hfdABAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S024300">536/24.3</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=hfdABAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S024330">536/24.33</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=hfdABAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S024100">536/24.1</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=hfdABAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S022100">536/22.1</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=hfdABAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S024320">536/24.32</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=hfdABAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S024310">536/24.31</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=hfdABAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S006100">435/6.1</a></span></td></tr><tr><td class="patent-data-table-td ">International Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=hfdABAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0015090000">C12N15/09</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=hfdABAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12Q0001680000">C12Q1/68</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=hfdABAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C07K0014470000">C07K14/47</a></span></td></tr><tr><td class="patent-data-table-td ">Cooperative Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=hfdABAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12Q1/6858">C12Q1/6858</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=hfdABAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C07K14/4708">C07K14/4708</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=hfdABAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12Q1/686">C12Q1/686</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=hfdABAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12Q1/6883">C12Q1/6883</a></span></td></tr><tr><td class="patent-data-table-td ">European Classification</td><td class="patent-data-table-td "><span class="nested-value">C12Q1/68D2G</span>, <span class="nested-value">C12Q1/68M6</span>, <span class="nested-value">C12Q1/68D4</span>, <span class="nested-value">C07K14/47A2A</span></td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="legal-events"></a><div class="patent-section-header"><span class="patent-section-title">Legal Events</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Date</th><th class="patent-data-table-th">Code</th><th class="patent-data-table-th">Event</th><th class="patent-data-table-th">Description</th></tr></thead><tr><td class="patent-data-table-td patent-date-value">Dec 4, 2012</td><td class="patent-data-table-td ">B2</td><td class="patent-data-table-td ">Reexamination certificate second reexamination</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">THE PATENTABILITY OF CLAIMS 12-15 IS CONFIRMED.CLAIMS 1-5, 7 AND 11 ARE CANCELLED.CLAIMS 6 AND 8-10ARE DETERMINED TO BE PATENTABLE AS AMENDED.</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Dec 7, 2010</td><td class="patent-data-table-td ">RR</td><td class="patent-data-table-td ">Request for reexamination filed</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20100902</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Sep 1, 2009</td><td class="patent-data-table-td ">RR</td><td class="patent-data-table-td ">Request for reexamination filed</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20090618</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Mar 3, 2009</td><td class="patent-data-table-td ">B1</td><td class="patent-data-table-td ">Reexamination certificate first reexamination</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">THE PATENTABILITY OF CLAIMS 1-5, 7 AND 9-15 IS CONFIRMED. CLAIMS 6 AND 8 WERE NOT REEXAMINED.</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">May 30, 2008</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">12</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Jan 29, 2008</td><td class="patent-data-table-td ">RR</td><td class="patent-data-table-td ">Request for reexamination filed</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20071204</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Jun 2, 2004</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">8</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">May 18, 2004</td><td class="patent-data-table-td ">CC</td><td class="patent-data-table-td ">Certificate of correction</td><td class="patent-data-table-td "></td></tr><tr><td class="patent-data-table-td patent-date-value">Dec 30, 2003</td><td class="patent-data-table-td ">CC</td><td class="patent-data-table-td ">Certificate of correction</td><td class="patent-data-table-td "></td></tr><tr><td class="patent-data-table-td patent-date-value">Jun 1, 2000</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">4</span></div></td></tr></table><div class="patent-section-footer"></div></div><div class="modal-dialog" id="patent-images-lightbox"><div class="patent-lightbox-controls"><div class="patent-lightbox-rotate-controls"><div class="patent-lightbox-rotation-text">Rotate</div><div class="rotate-icon rotate-ccw-icon"></div><div class="rotate-icon rotate-cw-icon"></div></div><div class="patent-lightbox-index-counter"></div><a class="patent-lightbox-fullsize-link" target="_blank">Original Image</a><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_right.png" alt="Next page"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_left.png" alt="Previous page"width="21" height="21" /></div></div><div class="modal-dialog-content"><div class="patent-lightbox-image-holder"><div class="patent-lightbox-placeholder"></div></div></div></div><script>_OC_initPatentsAtb({image_not_available_html: " Image not available"});</script></div></div></div></td></tr></table><script>(function() {var href = window.location.href;if (href.indexOf('?') !== -1) {var parameters = href.split('?')[1].split('&');for (var i = 0; i < parameters.length; i++) {var param = parameters[i].split('=');if (param[0] == 'focus') {var elem = document.getElementById(param[1]);if (elem) {elem.focus();}}}}})();</script><script>_OC_addFlags({LockSrc:"/books/javascript/lock_8a2b04e7bf975d5171d8e4c0b6365c7a.js", Host:"http://www.google.com/", IsBooksRentalEnabled:1, IsWebstoreDisplayCaseEnabled:1, IsObfuscationEnabled:1, IsBrowsingHistoryEnabled:1, IsWebReaderSvgEnabled:0, IsGeoLayerEnabled:1, IsImageModeNotesEnabled:1, IsCopyMenuItemEnabled:1, IsGiftingEnabled:0, IsWebReaderUniversalPaginatorEnabled:0, IsOfflineBubbleEnabled:1, IsReaderEnabledForPlayRequests:1, IsFutureOnSaleVolumesEnabled:1, IsOfflineRestrictedCopyEnabled:1, IsBooksUnifiedLeftNavEnabled:1, IsRestrictedCopyEnabled:1, IsZipitFolderCollectionEnabled:1, IsEndOfSampleRecommendationsEnabled:1, IsRatingsOnBookcardsEnabled:1, IsAdsDisabled:0, IsIframePageDisplayEnabled:0, IsEmbeddedMediaEnabled:1, IsImageModeAnnotationsEnabled:1, IsMyLibraryGooglePlusEnabled:1, IsImagePageProviderEnabled:0, IsBookcardListPriceSmall:0, IsInternalUser:0, IsBooksShareButtonEnabled:0, IsPreOrdersEnabled:0, IsDisabledRandomBookshelves:0, WebstoreDisplayCasePosition:3});_OC_Run({"enable_p13n":false,"add_vol_to_collection_base_url":"http://www.google.com/patents?op=add\u0026sig=ACfU3U3idRhYD_N4sRRpC_P5e9naVtHY-A\u0026id=hfdABAABERAJ","remove_vol_from_collection_base_url":"http://www.google.com/patents?op=remove\u0026sig=ACfU3U3EMBGfh0zsJ4CGxu_tCgr5KoMmAQ\u0026id=hfdABAABERAJ","logged_in":false,"p13n_save_user_settings_url":"http://www.google.com/patents?op=edit_user_settings\u0026sig=ACfU3U04CIaOjro9mQZhYDsS1zvNg4ohNg","is_cobrand":false,"sign_in_url":"https://www.google.com/accounts/Login?service=\u0026continue=http://www.google.com/patents%3Fhl%3Den\u0026hl=en","is_play_enabled":true}, {"volume_id":"","is_ebook":true,"volumeresult":{"has_flowing_text":false,"has_scanned_text":true,"can_download_pdf":false,"can_download_epub":false,"is_pdf_drm_enabled":false,"is_epub_drm_enabled":false,"download_pdf_url":"http://www.google.com/patents/download/Multiplex_genomic_DNA_amplification_for.pdf?id=hfdABAABERAJ\u0026output=pdf\u0026sig=ACfU3U18in49fRNlYQrrSmvObP4F2TtMeQ"},"sample_url":"http://www.google.com/patents/reader?id=hfdABAABERAJ\u0026printsec=frontcover\u0026output=reader\u0026source=gbs_atb_hover","is_browsable":true,"is_public_domain":true}, {});</script><div id="footer_table" style="font-size:83%;text-align:center;position:relative;top:20px;height:4.5em;margin-top:2em"><div style="margin-bottom:8px"><a href=http://www.google.com/><nobr>Google&nbsp;Home</nobr></a> - <a href=//www.google.com/patents/sitemap/><nobr>Sitemap</nobr></a> - <a href=http://www.google.com/googlebooks/uspto.html><nobr>USPTO Bulk Downloads</nobr></a> - <a href=/intl/en/privacy/><nobr>Privacy Policy</nobr></a> - <a href=/intl/en/policies/terms/><nobr>Terms of Service</nobr></a> - <a href=https://support.google.com/faqs/answer/2539193?hl=en><nobr>About Google Patents</nobr></a> - <a href="http://www.google.com/tools/feedback/intl/en/error.html" onclick="try{_OC_startFeedback({productId: '72792',locale: 'en'});return false;}catch(e){}"><nobr>Send Feedback</nobr></a></div><span>Data provided by IFI CLAIMS Patent Services</span><br><span >&copy;2012 Google</span></div> <script type="text/javascript">var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));</script><script type="text/javascript">var pageTracker = _gat._getTracker("UA-27188110-1");pageTracker._setCookiePath("/patents/");pageTracker._trackPageview();</script> </body></html>