<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN""http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html><head><title>Patent US5612179 - Intron sequence analysis method for detection of adjacent and remote locus ... - Google Patents</title><script>(function(){(function(){function e(a){this.t={};this.tick=function(a,c,b){var d=void 0!=b?b:(new Date).getTime();this.t[a]=[d,c];if(void 0==b)try{window.console.timeStamp("CSI/"+a)}catch(e){}};this.tick("start",null,a)}var a;window.performance&&(a=window.performance.timing);var f=a?new e(a.responseStart):new e;window.jstiming={Timer:e,load:f};if(a){var c=a.navigationStart,d=a.responseStart;0<c&&d>=c&&(window.jstiming.srt=d-c)}if(a){var b=window.jstiming.load;0<c&&d>=c&&(b.tick("_wtsrt",void 0,c),b.tick("wtsrt_",
"_wtsrt",d),b.tick("tbsd_","wtsrt_"))}try{a=null,window.chrome&&window.chrome.csi&&(a=Math.floor(window.chrome.csi().pageT),b&&0<c&&(b.tick("_tbnd",void 0,window.chrome.csi().startE),b.tick("tbnd_","_tbnd",c))),null==a&&window.gtbExternal&&(a=window.gtbExternal.pageT()),null==a&&window.external&&(a=window.external.pageT,b&&0<c&&(b.tick("_tbnd",void 0,window.external.startE),b.tick("tbnd_","_tbnd",c))),a&&(window.jstiming.pt=a)}catch(g){}})();})();
</script><link rel="stylesheet" href="/patents/css/_8a2b04e7bf975d5171d8e4c0b6365c7a/kl_intl_patents_bundle.css" type="text/css" /><script src="/books/javascript/atb_8a2b04e7bf975d5171d8e4c0b6365c7a__en.js"></script><script>function googleTranslateElementInit() {new google.translate.TranslateElement({pageLanguage: "en",gaTrack: true,gaId: "UA-27188110-1",multilanguagePage: true});}</script><script src="//translate.google.com/translate_a/element.js?cb=googleTranslateElementInit"></script><meta name="DC.type" content="Patent"><meta name="DC.title" content="Intron sequence analysis method for detection of adjacent and remote locus alleles as haplotypes"><meta name="DC.contributor" content="Malcolm J. Simons" scheme="inventor"><meta name="DC.contributor" content="Genetype A.G." scheme="assignee"><meta name="DC.date" content="1992-9-23" scheme="dateSubmitted"><meta name="DC.description" content="The present invention provides a method for detection of at least one allele of a genetic locus and can be used to provide direct determination of the haplotype. The method comprises amplifying genomic DNA with a primer pair that spans an intron sequence and defines a DNA sequence in genetic linkage with an allele to be detected. The primer-defined DNA sequence contains a sufficient number of intron sequence nucleotides to characterize the allele. Genomic DNA is amplified to produce an amplified DNA sequence characteristic of the allele. The amplified DNA sequence is analyzed to detect the presence of a genetic variation in the amplified DNA sequence such as a change in the length of the sequence, gain or loss of a restriction site or substitution of a nucleotide. The variation is characteristic of the allele to be detected and can be used to detect remote alleles. Kits comprising one or more of the reagents used in the method are also described."><meta name="DC.date" content="1997-3-18" scheme="issued"><meta name="DC.relation" content="EP:0256630:A1" scheme="references"><meta name="DC.relation" content="EP:0269260:A2" scheme="references"><meta name="DC.relation" content="EP:0364255:A2" scheme="references"><meta name="DC.relation" content="US:4582788" scheme="references"><meta name="DC.relation" content="US:4683194" scheme="references"><meta name="DC.relation" content="US:4683195" scheme="references"><meta name="DC.relation" content="US:5192659" scheme="references"><meta name="DC.relation" content="WO:1989011547:A1" scheme="references"><meta name="DC.relation" content="WO:1989012697:A1" scheme="references"><meta name="citation_reference" content="Abraham, et al., &quot;Sequence differences between HLA-B and TNF distinguish different MHC ancestral haplotypes,&quot; Antigens, vol 39, No. 3, Mar. 1992, pp. 117-121."><meta name="citation_reference" content="Abraham, et al., Sequence differences between HLA B and TNF distinguish different MHC ancestral haplotypes, Antigens , vol 39, No. 3, Mar. 1992, pp. 117 121."><meta name="citation_reference" content="Allen et al., Biotechniques , vol. 7, No. 7, pp. 736 744 (1989)."><meta name="citation_reference" content="Allen et al., Biotechniques, vol. 7, No. 7, pp. 736-744 (1989)."><meta name="citation_reference" content="Boehnke et al., &quot;Fine-Structure Genetic Mapping of Human Chromosomes Using the Polymerase Chain Reaction on Single Sperm: Experimental Design Considerations,&quot; The American Journal of Human Genetics, Volumr 45, No. 1, Jul. 1989, pp. 21-32."><meta name="citation_reference" content="Boehnke et al., Fine Structure Genetic Mapping of Human Chromosomes Using the Polymerase Chain Reaction on Single Sperm: Experimental Design Considerations, The American Journal of Human Genetics , Volumr 45, No. 1, Jul. 1989, pp. 21 32."><meta name="citation_reference" content="Clark, &quot;Interference of Haplotypes from PCR-amplified Samples of Diploid Populations&quot;, Mol. Biol. Evol., vol. 7, No. 2, 1990, pp. 111-122."><meta name="citation_reference" content="Clark, Interference of Haplotypes from PCR amplified Samples of Diploid Populations , Mol. Biol. Evol., vol. 7, No. 2, 1990, pp. 111 122."><meta name="citation_reference" content="Deng, &quot;A sensitive non-radioactive PCR-RFLP analysis for detecting point mutations at 12th condon oncogene c-Ha-ras in DNAs of gastic cancer,&quot; Nucleic Acids Research, vol. 16, No. 13, 1988, p. 6231."><meta name="citation_reference" content="Deng, A sensitive non radioactive PCR RFLP analysis for detecting point mutations at 12th condon oncogene c Ha ras in DNAs of gastic cancer, Nucleic Acids Research , vol. 16, No. 13, 1988, p. 6231."><meta name="citation_reference" content="Dicker, et al., &quot;Sequence Analysis of a Human Gene Responsible for Drug Resistance: A Rapid Method for Manual and Automated Direct Sequencing of Products Generated by the Polymerase Chain Reaction,&quot; BioTechniques, vol. 7, No. 8, 1989, pp. 830-837."><meta name="citation_reference" content="Dicker, et al., Sequence Analysis of a Human Gene Responsible for Drug Resistance: A Rapid Method for Manual and Automated Direct Sequencing of Products Generated by the Polymerase Chain Reaction, BioTechniques , vol. 7, No. 8, 1989, pp. 830 837."><meta name="citation_reference" content="DiLella, et al., &quot;Screening for Phenylketonuria Mutations by DNA Amplification with the Polymerase Chain Reaction,&quot; The Lancet, Mar. 5, 1988, pp.. 497-499."><meta name="citation_reference" content="DiLella, et al., &quot;Tight linkage between a splicing mutation and a specific DNA haplotype in phenylketonuria,&quot; Nature, vol. 322, No. 28, Aug. 1986, pp. 799-803."><meta name="citation_reference" content="DiLella, et al., Screening for Phenylketonuria Mutations by DNA Amplification with the Polymerase Chain Reaction, The Lancet , Mar. 5, 1988, pp.. 497 499."><meta name="citation_reference" content="DiLella, et al., Tight linkage between a splicing mutation and a specific DNA haplotype in phenylketonuria, Nature , vol. 322, No. 28, Aug. 1986, pp. 799 803."><meta name="citation_reference" content="Erlich et al., &quot;Reliability of the HLA-DQα PCR-Based Olignucleotide Typing System,&quot; Journal of Forensic Sciences, Sep. 1990, pp. 1017-1018."><meta name="citation_reference" content="Erlich et al., Reliability of the HLA DQ PCR Based Olignucleotide Typing System, Journal of Forensic Sciences , Sep. 1990, pp. 1017 1018."><meta name="citation_reference" content="Funke et al., &quot;Detection of a New Msp I Restriction Fragment Length Polymorphism in the Apolipoprotein A-I Gene,&quot; J. Clin. Chem. Clin. Biochem., vol. 25, No. 3, 1987, pp. 131-134."><meta name="citation_reference" content="Funke et al., Detection of a New Msp I Restriction Fragment Length Polymorphism in the Apolipoprotein A I Gene, J. Clin. Chem. Clin. Biochem. , vol. 25, No. 3, 1987, pp. 131 134."><meta name="citation_reference" content="Geisel et al., &quot;A New APA LI Restriction Fragment Length Polymrophism in the Low Density Lipoprotein Receptor Gene,&quot; Journal of Clinical Chemistry and Clinical Biochemistry, vol. 26, No. 7, 1988, pp. 429-434."><meta name="citation_reference" content="Geisel et al., A New APA LI Restriction Fragment Length Polymrophism in the Low Density Lipoprotein Receptor Gene, Journal of Clinical Chemistry and Clinical Biochemistry , vol. 26, No. 7, 1988, pp. 429 434."><meta name="citation_reference" content="Graham et al., &quot;Application of Molecular Genetics to Prenatal Diagnosis and Carrier Detection in the Hemophilias: Some Limitaitons,&quot; Blood, vol. 66, No. 4, Oct. 1985, pp. 759-764."><meta name="citation_reference" content="Graham et al., Application of Molecular Genetics to Prenatal Diagnosis and Carrier Detection in the Hemophilias: Some Limitaitons, Blood , vol. 66, No. 4, Oct. 1985, pp. 759 764."><meta name="citation_reference" content="Kerem et al., &quot;Identification of the Cystic Fibrosis Gene: Genetic Analysis,&quot;, Science, vol. 245, 1989, pp. 1073-1080."><meta name="citation_reference" content="Kerem et al., Identification of the Cystic Fibrosis Gene: Genetic Analysis, , Science , vol. 245, 1989, pp. 1073 1080."><meta name="citation_reference" content="Kogan et al., &quot;An Improved Method for Prenatal Diagnosis of Genetic Diseases by Analysis of Amplifed DNA Sequences--Application to Hemophilia A,&quot; The New England Journal of Medicine, vol. 317, No. 16, Oct. 15, 1987, pp. 985-990."><meta name="citation_reference" content="Kogan et al., An Improved Method for Prenatal Diagnosis of Genetic Diseases by Analysis of Amplifed DNA Sequences Application to Hemophilia A, The New England Journal of Medicine, vol. 317, No. 16, Oct. 15, 1987, pp. 985 990."><meta name="citation_reference" content="Mardis et al., &quot;Automated Methods for Single-Stranded DNA Isolation and Dideoxynucleotide DNA Sequencing Reactions on a Robotic Workstation,&quot; BioTechniques, vol. 7, No. 8, 1989, pp. 840-850."><meta name="citation_reference" content="Mardis et al., Automated Methods for Single Stranded DNA Isolation and Dideoxynucleotide DNA Sequencing Reactions on a Robotic Workstation, BioTechniques , vol. 7, No. 8, 1989, pp. 840 850."><meta name="citation_reference" content="Marx, &quot;Disecting the Complex Diseases,&quot; Science, vol. 247, Mar. 30, 1990, pp. 1540-1542."><meta name="citation_reference" content="Marx, Disecting the Complex Diseases, Science , vol. 247, Mar. 30, 1990, pp. 1540 1542."><meta name="citation_reference" content="Molecular Biology of The Cell , 2nd Edition, New York, 1989, Chapter 9: RNA Synthesis and RNA Processing pp. 531 535 and Chapter 10: Control of Gene Expression pp. 602 604."><meta name="citation_reference" content="Molecular Biology of The Cell, 2nd Edition, New York, 1989, Chapter 9: &quot;RNA Synthesis and RNA Processing&quot; pp. 531-535 and Chapter 10: Control of Gene Expression pp. 602-604."><meta name="citation_reference" content="Molecular Biology of the Gene , 4th Edition, California, 1987, Chapter 20: Conserved Sequences at Exon Intron Boundaries in Pre mRNA, pp. 639 640."><meta name="citation_reference" content="Molecular Biology of the Gene, 4th Edition, California, 1987, Chapter 20: &quot;Conserved Sequences at Exon-Intron Boundaries in Pre-mRNA,&quot; pp. 639-640."><meta name="citation_reference" content="Padgett et al. Amer. Rev. Biochem. 1986. 55:1119 150."><meta name="citation_reference" content="Padgett et al. Amer. Rev. Biochem. 1986. 55:1119-150."><meta name="citation_reference" content="Riess, et al., &quot;Hypervariability of intronic simple (gt)n (ga)m repeats in HLA-DRB genes,&quot; Immunogenetics, vol. 32, No. 2, Aug. 1990, pp. 110-116."><meta name="citation_reference" content="Riess, et al., Hypervariability of intronic simple (gt) n (ga) m repeats in HLA DRB genes, Immunogenetics , vol. 32, No. 2, Aug. 1990, pp. 110 116."><meta name="citation_reference" content="Riordan et al., &quot;Identification of the Cystic Fibrosis Gene: Cloning and Characterization of Complementary DNA,&quot;, Science, vol. 245, 1989, pp. 1066-1073."><meta name="citation_reference" content="Riordan et al., Identification of the Cystic Fibrosis Gene: Cloning and Characterization of Complementary DNA, , Science , vol. 245, 1989, pp. 1066 1073."><meta name="citation_reference" content="Rommens et al., &quot;Identification of the Cystic Fibrosis Gene: Chromosome Walking and Jumping,&quot;, Science, vol. 245, 1989, pp. 1059-1065."><meta name="citation_reference" content="Rommens et al., Identification of the Cystic Fibrosis Gene: Chromosome Walking and Jumping, , Science , vol. 245, 1989, pp. 1059 1065."><meta name="citation_reference" content="Saiki et al., &quot;Enzymatic Amplification of β-Globin Genomic Sequences and Restriction Site Analysis for Diagnosis of Sickle Cell Anemia,&quot;, Science, vol. 230, 1985, pp. 1350-1354."><meta name="citation_reference" content="Saiki et al., Enzymatic Amplification of Globin Genomic Sequences and Restriction Site Analysis for Diagnosis of Sickle Cell Anemia, , Science , vol. 230, 1985, pp. 1350 1354."><meta name="citation_reference" content="Simon and Erlich, pp. 952 958 In: Immunology of HLA , vol. 1: Springer Verlag, New York (1989)."><meta name="citation_reference" content="Simon and Erlich, pp. 952-958 In: Immunology of HLA, vol. 1: Springer-Verlag, New York (1989)."><meta name="citation_reference" content="Simons et al., pp. 952 958 In: Immunology of HLA , vol. 1:Springer Verlag, New York (1989)."><meta name="citation_reference" content="Simons et al., pp. 952-958 In: Immunology of HLA, vol. 1:Springer-Verlag, New York (1989)."><meta name="citation_reference" content="Simons, et al., pp. 959 1023 In: Immunology of HLA , vol. 1: Springer Verlag, New York (1989)."><meta name="citation_reference" content="Simons, et al., pp. 959-1023 In: Immunology of HLA, vol. 1: Springer-Verlag, New York (1989)."><meta name="citation_reference" content="Stephens et al., &quot;Theoretical Underpinning of the Single-Molecule-Dilution (SMD) Method of Direct Haplotype Resolution,&quot; Am. J. Hum. Genet, vol. 46, 1990, pp. 1149-1155."><meta name="citation_reference" content="Stephens et al., Theoretical Underpinning of the Single Molecule Dilution (SMD) Method of Direct Haplotype Resolution, Am. J. Hum. Genet , vol. 46, 1990, pp. 1149 1155."><meta name="citation_reference" content="Waikan et al. P.N.A.S, U.S.A., vol. 75, No. 11, pp. 5631 5635, Nov. 1978."><meta name="citation_reference" content="Waikan et al. P.N.A.S, U.S.A., vol. 75, No. 11, pp. 5631-5635, Nov. 1978."><meta name="citation_patent_number" content="US:5612179"><meta name="citation_patent_application_number" content="US:07/949,652"><link rel="canonical" href="http://www.google.com/patents/US5612179"/><meta property="og:url" content="http://www.google.com/patents/US5612179"/><meta name="title" content="Patent US5612179 - Intron sequence analysis method for detection of adjacent and remote locus alleles as haplotypes"/><meta name="description" content="The present invention provides a method for detection of at least one allele of a genetic locus and can be used to provide direct determination of the haplotype. The method comprises amplifying genomic DNA with a primer pair that spans an intron sequence and defines a DNA sequence in genetic linkage with an allele to be detected. The primer-defined DNA sequence contains a sufficient number of intron sequence nucleotides to characterize the allele. Genomic DNA is amplified to produce an amplified DNA sequence characteristic of the allele. The amplified DNA sequence is analyzed to detect the presence of a genetic variation in the amplified DNA sequence such as a change in the length of the sequence, gain or loss of a restriction site or substitution of a nucleotide. The variation is characteristic of the allele to be detected and can be used to detect remote alleles. Kits comprising one or more of the reagents used in the method are also described."/><meta property="og:title" content="Patent US5612179 - Intron sequence analysis method for detection of adjacent and remote locus alleles as haplotypes"/><meta property="og:type" content="book"/><meta property="og:site_name" content="Google Books"/><meta property="og:image" content="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><link rel="image_src" href="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><script>(function(){try{var aa=function(a,b,c,d){d=d||{};d._sn=["cfg",b,c].join(".");window.gbar.logger.ml(a,d)};var g=window.gbar=window.gbar||{},l=window.gbar.i=window.gbar.i||{},m={},n;function _tvn(a,b){var c=parseInt(a,10);return isNaN(c)?b:c}function _tvf(a,b){var c=parseFloat(a);return isNaN(c)?b:c}function _tvv(a){return!!a}function p(a,b,c){(c||g)[a]=b}g.bv={n:_tvn("2",0),r:"",f:".67.",e:"0",m:_tvn("0",1)};
function q(a,b,c){var d="on"+b;if(a.addEventListener)a.addEventListener(b,c,!1);else if(a.attachEvent)a.attachEvent(d,c);else{var f=a[d];a[d]=function(){var a=f.apply(this,arguments),b=c.apply(this,arguments);return void 0==a?b:void 0==b?a:b&&a}}}var s=function(a){return function(){return g.bv.m==a}},ba=s(1),ca=s(2);p("sb",ba);p("kn",ca);l.a=_tvv;l.b=_tvf;l.c=_tvn;l.i=aa;var da=window.gbar.i.i;var t,u,v,w;function ea(a){v=a}function fa(a){var b;if(b=v&&window.encodeURIComponent)b=a.href,b=!b.match(/^http[s]?:\/\/accounts\.google\.[^/]*\/ClearSID/i)&&!b.match(/^http[s]?:\/\/[^/]*\/accounts\/ClearSID/i);if(b=b&&encodeURIComponent(v()))a.href=a.href.replace(/([?&]continue=)[^&]*/,"$1"+b)}function ga(a){window.gApplication&&(a.href=window.gApplication.getTabUrl(a.href))}
function ha(a){var b=document.forms[0].q,c=window.encodeURIComponent&&b&&b.value,b=b&&b.placeholder;c&&c!=b&&(a.href=a.href.replace(/([?&])q=[^&]*|$/,function(a,b){return(b||"&")+"q="+encodeURIComponent(c)}))}n=l.a("")?ga:ha;
function x(a,b,c,d,f,e){var h=document.getElementById(a);if(h){var k=h.style;k.left=d?"auto":b+"px";k.right=d?b+"px":"auto";k.top=c+"px";k.visibility=u?"hidden":"visible";f&&e?(k.width=f+"px",k.height=e+"px"):(x(t,b,c,d,h.offsetWidth,h.offsetHeight),u=u?"":a)}}
var y=[],ia=function(a,b){y.push(b)},ja=function(a){a=a||window.event;var b=a.target||a.srcElement;a.cancelBubble=!0;null==t&&(a=document.createElement(Array.every||window.createPopup?"iframe":"div"),a.frameBorder="0",t=a.id="gbs",a.src="javascript:''",b.parentNode.appendChild(a),q(document,"click",z));var c=b,b=0;"gb3"!=c.className&&(c=c.parentNode);a=c.getAttribute("aria-owns")||"gbi";var d=c.offsetWidth,f=20<c.offsetTop?46:24;document.getElementById("tphdr")&&(f-=3);var e=!1;do b+=c.offsetLeft||
0;while(c=c.offsetParent);var c=(document.documentElement.clientWidth||document.body.clientWidth)-b-d,h,d=document.body,k=document.defaultView;k&&k.getComputedStyle?(d=k.getComputedStyle(d,""))&&(h=d.direction):h=d.currentStyle?d.currentStyle.direction:d.style.direction;h="rtl"==h;if("gbi"==a){for(d=0;k=y[d++];)k();A(null,window.navExtra);h&&(b=c,e=!0)}else h||(b=c,e=!0);u!=a&&z();x(a,b,f,e)},z=function(){u&&x(u,0,0)},A=function(a,b){var c,d=document.getElementById("gbi"),f=a;f||(f=d.firstChild);
for(;b&&(c=b.pop());){var e=d,h=c,k=f;w||(w="gb2");e.insertBefore(h,k).className=w}},ka=function(a,b,c){if((b=document.getElementById(b))&&a){a.className="gb4";var d=document.createElement("span");d.appendChild(a);d.appendChild(document.createTextNode(" | "));d.id=c;b.appendChild(d)}},la=function(){return document.getElementById("gb_70")},ma=function(){return!!u};p("qs",n);p("setContinueCb",ea);p("pc",fa);p("tg",ja);p("close",z);p("addLink",ka);p("almm",A);p("si",la);p("adh",ia);p("op",ma);var B=function(){},C=function(){},F=function(a){var b=new Image,c=D;b.onerror=b.onload=b.onabort=function(){try{delete E[c]}catch(a){}};E[c]=b;b.src=a;D=c+1},E=[],D=0;p("logger",{il:C,ml:B,log:F});var G=window.gbar.logger;var H={},na={},I=[],oa=l.b("0.1",.1),pa=l.a("1",!0),qa=function(a,b){I.push([a,b])},ra=function(a,b){H[a]=b},sa=function(a){return a in H},J={},K=function(a,b){J[a]||(J[a]=[]);J[a].push(b)},ta=function(a){K("m",a)},L=function(a,b){var c=document.createElement("script");c.src=a;c.async=pa;Math.random()<oa&&(c.onerror=function(){c.onerror=null;B(Error("Bundle load failed: name="+(b||"UNK")+" url="+a))});(document.getElementById("xjsc")||document.getElementsByTagName("body")[0]||
document.getElementsByTagName("head")[0]).appendChild(c)},N=function(a){for(var b=0,c;(c=I[b])&&c[0]!=a;++b);!c||c[1].l||c[1].s||(c[1].s=!0,M(2,a),c[1].url&&L(c[1].url,a),c[1].libs&&m.d&&m.d(c[1].libs))},O=function(a){K("gc",a)},P=null,ua=function(a){P=a},M=function(a,b,c){if(P){a={t:a,b:b};if(c)for(var d in c)a[d]=c[d];try{P(a)}catch(f){}}};p("mdc",H);p("mdi",na);p("bnc",I);p("qGC",O);p("qm",ta);p("qd",J);p("lb",N);p("mcf",ra);p("bcf",qa);p("aq",K);p("mdd","");p("has",sa);
p("trh",ua);p("tev",M);var Q=l.b("0.1",.001),R=0;
function _mlToken(a,b){try{if(1>R){R++;var c,d=a,f=b||{},e=encodeURIComponent,h=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&jexpid=",e("17483"),"&srcpg=",e("prop=22"),"&jsr=",Math.round(1/Q),"&ogev=",e("Z5ztU_yYONGgogThgoCoAg"),"&ogf=",g.bv.f,"&ogrp=",e("1"),"&ogv=",e("1407723702.0"),"&oggv="+e("es_plusone_gc_20140723.0_p0"),"&ogd=",e("com"),"&ogc=",e("JPN"),"&ogl=",e("en")];f._sn&&(f._sn="og."+
f._sn);for(var k in f)h.push("&"),h.push(e(k)),h.push("="),h.push(e(f[k]));h.push("&emsg=");h.push(e(d.name+":"+d.message));var r=h.join("");S(r)&&(r=r.substr(0,2E3));c=r;var Aa=window.gbar.logger._aem(a,c);F(Aa)}}catch(Na){}}var S=function(a){return 2E3<=a.length},va=function(a,b){return b};function T(a){B=a;p("_itl",S,G);p("_aem",va,G);p("ml",B,G);a={};H.er=a}l.a("")?T(function(a){throw a;}):l.a("1")&&Math.random()<Q&&T(_mlToken);I.push(["m",{url:"//ssl.gstatic.com/gb/js/scm_7385cc5883250b43a39405734c1bea59.js"}]);g.mcf("c",{});g.sg={c:""};if(l.a("1")){var wa=l.a("");I.push(["gc",{auto:wa,url:"//ssl.gstatic.com/gb/js/abc/gci_91f30755d6a6b787dcc2a4062e6e9824.js",libs:"googleapis.client:plusone:gapi.iframes"}]);var xa={version:"gci_91f30755d6a6b787dcc2a4062e6e9824.js",index:"",lang:"en"};H.gc=xa;var U=function(a){window.googleapis&&window.iframes?a&&a():(a&&O(a),N("gc"))};p("lGC",U);l.a("1")&&p("lPWF",U)};window.__PVT="";if(l.a("1")&&l.a("1")){var V=function(a){U(function(){K("pw",a);N("pw")})};p("lPW",V);I.push(["pw",{url:"//ssl.gstatic.com/gb/js/abc/pwm_45f73e4df07a0e388b0fa1f3d30e7280.js"}]);var W=[],ya=function(a){W[0]=a},za=function(a,b){var c=b||{};c._sn="pw";B(a,c)},Ba={signed:W,elog:za,base:"https://plusone.google.com/u/0",loadTime:(new Date).getTime()};H.pw=Ba;var X=function(a,b){for(var c=b.split("."),d=function(){var b=arguments;a(function(){for(var a=g,d=0,e=c.length-1;d<e;++d)a=a[c[d]];a[c[d]].apply(a,b)})},f=g,e=0,h=c.length-1;e<h;++e)f=
f[c[e]]=f[c[e]]||{};return f[c[e]]=d};X(V,"pw.clk");X(V,"pw.hvr");p("su",ya,g.pw)};function Ca(){function a(){for(var b;(b=e[h++])&&"m"!=b[0]&&!b[1].auto;);b&&(M(2,b[0]),b[1].url&&L(b[1].url,b[0]),b[1].libs&&m.d&&m.d(b[1].libs));h<e.length&&setTimeout(a,0)}function b(){0<f--?setTimeout(b,0):a()}var c=l.a("1"),d=l.a(""),f=3,e=I,h=0,k=window.gbarOnReady;if(k)try{k()}catch(r){da(r,"ml","or")}d?p("ldb",a):c?q(window,"load",b):b()}p("rdl",Ca);var Da={D:1,H:2,da:3,p:4,W:5,M:6,F:7,g:8,ha:9,U:10,L:11,T:12,S:13,N:14,Q:15,P:16,fa:17,w:18,O:19,ga:20,ea:21,u:22,G:23,ja:24,ka:25,ia:26,A:27,j:28,o:29,k:30,ca:31,Z:32,$:33,J:34,K:35,ba:36,aa:37,Y:38,B:39,R:40,v:41,X:42,V:43,h:48,C:49,I:500},Y=[1,2,3,4,5,6,9,10,11,13,14,28,29,30,34,35,37,38,39,40,41,42,43,48,49,500];var Z=l.b("0.001",1E-4),Ea=l.b("1",1),Fa=!1,Ga=!1;if(l.a("1")){var Ha=Math.random();Ha<=Z&&(Fa=!0);Ha<=Ea&&(Ga=!0)}var Ia=Da,$=null;function Ja(){var a=0,b=function(b,d){l.a(d)&&(a|=b)};b(1,"");b(2,"");b(4,"");b(8,"");return a}
function Ka(a,b){var c=Z,d=Fa,f;f=a;if(!$){$={};for(var e=0;e<Y.length;e++){var h=Y[e];$[h]=!0}}if(f=!!$[f])c=Ea,d=Ga;if(d){d=encodeURIComponent;g.rp?(f=g.rp(),f="-1"!=f?f:"1"):f="1";c=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&oge=",a,"&ogex=",d("17483"),"&ogev=",d("Z5ztU_yYONGgogThgoCoAg"),"&ogf=",g.bv.f,"&ogp=",d("22"),"&ogrp=",d(f),"&ogsr=",Math.round(1/c),"&ogv=",d("1407723702.0"),"&oggv="+
d("es_plusone_gc_20140723.0_p0"),"&ogd=",d("com"),"&ogl=",d("en"),"&ogc=",d("JPN"),"&ogus=",Ja()];if(b){"ogw"in b&&(c.push("&ogw="+b.ogw),delete b.ogw);var k;f=b;e=[];for(k in f)0!=e.length&&e.push(","),e.push(La(k)),e.push("."),e.push(La(f[k]));k=e.join("");""!=k&&(c.push("&ogad="),c.push(d(k)))}F(c.join(""))}}function La(a){"number"==typeof a&&(a+="");return"string"==typeof a?a.replace(".","%2E").replace(",","%2C"):a}C=Ka;p("il",C,G);var Ma={};H.il=Ma;setTimeout(function(){C(Ia.g)},0);}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var b=window.gbar.i.i;var c=window.gbar;var f=function(d){try{var a=document.getElementById("gbom");a&&d.appendChild(a.cloneNode(!0))}catch(e){b(e,"omas","aomc")}};c.aomc=f;}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var a=window.gbar;a.mcf("pm",{p:""});}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{window.gbar.rdl();}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
if (window['_OC_timingAction']) {window['_OC_timingAction']('patents_refpage');}</script><style>#gbar,#guser{font-size:13px;padding-top:1px !important;}#gbar{float:left;height:22px}#guser{padding-bottom:7px !important;text-align:right}.gbh,.gbd{border-top:1px solid #c9d7f1;font-size:1px}.gbh{height:0;position:absolute;top:24px;width:100%}#gbs,.gbm{background:#fff;left:0;position:absolute;text-align:left;visibility:hidden;z-index:1000}.gbm{border:1px solid;border-color:#c9d7f1 #36c #36c #a2bae7;z-index:1001}.gb1{margin-right:.5em}.gb1,.gb3{zoom:1}.gb2{display:block;padding:.2em .5em}.gb2,.gb3{text-decoration:none !important;border-bottom:none}a.gb1,a.gb4{text-decoration:underline !important}a.gb1,a.gb2,a.gb3,a.gb4{color:#00c !important}.gbi .gb3,.gbi .gb2,.gbi .gb4{color:#dd8e27 !important}.gbf .gb3,.gbf .gb2,.gbf .gb4{color:#900 !important}a.gb2:hover{background:#36c;color:#fff !important}#gbar .gbz0l{color:#000 !important;cursor:default;font-weight:bold;text-decoration:none !important}
#gbar { padding:.3em .6em !important;}</style></head><body  topmargin="3" marginheight="3"><div id=gbar><nobr><a onclick=gbar.qs(this);gbar.logger.il(1,{t:1}); class=gb1 id=gb_1 href="https://www.google.com/search?sa=N&tab=tw">Search</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:2}); class=gb1 id=gb_2 href="http://www.google.com/search?hl=en&tbm=isch&source=og&sa=N&tab=ti">Images</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:8}); class=gb1 id=gb_8 href="http://maps.google.com/maps?hl=en&sa=N&tab=tl">Maps</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:78}); class=gb1 id=gb_78 href="https://play.google.com/?hl=en&sa=N&tab=t8">Play</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:36}); class=gb1 id=gb_36 href="http://www.youtube.com/results?sa=N&tab=t1">YouTube</a> <a onclick=gbar.logger.il(1,{t:5}); class=gb1 id=gb_5 href="http://news.google.com/nwshp?hl=en&tab=tn">News</a> <a onclick=gbar.logger.il(1,{t:23}); class=gb1 id=gb_23 href="https://mail.google.com/mail/?tab=tm">Gmail</a> <a onclick=gbar.logger.il(1,{t:25}); class=gb1 id=gb_25 href="https://drive.google.com/?tab=to">Drive</a> <a class=gb3 href="http://www.google.com/intl/en/options/" onclick="this.blur();gbar.tg(event);return !1" aria-haspopup=true><u>More</u> <small>&#9660;</small></a><div class=gbm id=gbi><a onclick=gbar.logger.il(1,{t:24}); class=gb2 id=gb_24 href="https://www.google.com/calendar?tab=tc">Calendar</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:51}); class=gb2 id=gb_51 href="http://translate.google.com/?hl=en&sa=N&tab=tT">Translate</a><a onclick=gbar.logger.il(1,{t:17}); class=gb2 id=gb_17 href="http://www.google.com/mobile/?hl=en&tab=tD">Mobile</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:10}); class=gb2 id=gb_10 href="http://www.google.com/search?hl=en&tbo=u&tbm=bks&source=og&sa=N&tab=tp">Books</a><a onclick=gbar.logger.il(1,{t:212}); class=gb2 id=gb_212 href="https://wallet.google.com/manage/?tab=ta">Wallet</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:6}); class=gb2 id=gb_6 href="http://www.google.com/search?hl=en&tbo=u&tbm=shop&source=og&sa=N&tab=tf">Shopping</a><a onclick=gbar.logger.il(1,{t:30}); class=gb2 id=gb_30 href="http://www.blogger.com/?tab=tj">Blogger</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:27}); class=gb2 id=gb_27 href="http://www.google.com/finance?sa=N&tab=te">Finance</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:31}); class=gb2 id=gb_31 href="https://plus.google.com/photos?sa=N&tab=tq">Photos</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:12}); class=gb2 id=gb_12 href="http://www.google.com/search?hl=en&tbo=u&tbm=vid&source=og&sa=N&tab=tv">Videos</a><div class=gb2><div class=gbd></div></div><a onclick=gbar.logger.il(1,{t:66}); href="http://www.google.com/intl/en/options/" class=gb2>Even more &raquo;</a></div></nobr></div><div id=guser width=100%><nobr><span id=gbn class=gbi></span><span id=gbf class=gbf></span><span id=gbe></span><a target=_top id=gb_70 href="https://www.google.com/accounts/Login?service=&continue=http://www.google.com/patents%3Fhl%3Den&hl=en" class=gb4>Sign in</a><div style="display: none"><div class=gbm id=gbd5 aria-owner=gbg5><div class=gbmc><ol id=gbom class=gbmcc></ol></div></div></div></nobr></div><div class=gbh style=left:0></div><div class=gbh style=right:0></div><div role="alert" style="position: absolute; left: 0; right: 0;"><a href="http://www.google.com/patents/us5612179?hl=en&amp;output=html_text" title="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."><img border="0" src="http://www.google.com/images/cleardot.gif"alt="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."></a></div><div id="guser"><nobr></nobr></div><div style="clear:both;"></div><div id="gb-top-search-box" class="gb-top-search-box-small gb-reset"><table><tr><td class="logo"><a href="http://www.google.com/patents" class="logo-link"><img class="logo-img" src="/intl/en/images/logos/google_logo_41.png" alt="Go to Google Books Home" height="41"/></a></td><td><form action="http://www.google.com/search" name="f" id="vheadf" method="get"><span id="hf"></span><input type="hidden" name="tbm" value="pts"/><input type="hidden" name="tbo" value="1"/><input type="hidden" name="hl" value="en"/><table><tr><td><div class="inputs"><table><tr><td><div class="text-input"><input type="text" name="q" id="vheadq" class="text" maxlength="2048" size="31" value="" title="Search Patents" accesskey="s" autocomplete="off"/><script>window._OC_autoDir &&window._OC_autoDir('vheadq', 'tia-vheadq');</script></div></td><td><div class="submit-input"><input name="btnG" class="submit" type="submit" value=""/></div></td></tr></table></div></td><td class="col-ext-links"><div class="ext-links"><a href="http://www.google.com/advanced_patent_search">&lt;nobr&gt;Advanced Patent Search&lt;/nobr&gt;</a></div></td></tr></table></form></td></tr></table></div><div class="kd-appbar"><h2 class="kd-appname"><a href="/patents">Patents</a></h2><div class="kd-buttonbar left" id="left-toolbar-buttons"><a id="appbar-write-review-link" href=""></a><a id="appbar-view-print-sample-link" href=""></a><a id="appbar-view-ebook-sample-link" href=""></a><a id="appbar-patents-prior-art-finder-link" href="https://www.google.com/patents/related/US5612179"></a><a id="appbar-patents-discuss-this-link" href="http://www.google.com/url?id=bhJCBAABERAJ&amp;q=http://patents.stackexchange.com/redirect/google-patents%3Fpatent%3DUS5612179&amp;usg=AFQjCNE8oAr3WFld6OXlzd4Ue_HGFuV25g" data-is-grant="true"></a><a id="appbar-read-patent-link" href="//docs.google.com/viewer?url=patentimages.storage.googleapis.com/pdfs/US5612179.pdf"></a><a id="appbar-download-pdf-link" href="//patentimages.storage.googleapis.com/pdfs/US5612179.pdf"></a></div><div class="kd-buttonbar right" id="right-toolbar-buttons"></div></div><div id="books-microdata" itemscope=""itemtype="http://schema.org/Book"itemid="http://www.google.com/patents/US5612179" style="display:none"><span itemprop="description">The present invention provides a method for detection of at least one allele of a genetic locus and can be used to provide direct determination of the haplotype. The method comprises amplifying genomic DNA with a primer pair that spans an intron sequence and defines a DNA sequence in genetic linkage...</span><span itemprop="url">http://www.google.com/patents/US5612179?utm_source=gb-gplus-share</span><span class="main-title" itemprop="name">Patent US5612179 - Intron sequence analysis method for detection of adjacent and remote locus alleles as haplotypes</span><img itemprop="image" src="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"alt="Patent US5612179 - Intron sequence analysis method for detection of adjacent and remote locus alleles as haplotypes" title="Patent US5612179 - Intron sequence analysis method for detection of adjacent and remote locus alleles as haplotypes"></div><div style="display: none"><ol id="ofe-gear-menu-contents" class="gbmcc"><li class="gbe gbmtc"><a class="gbmt goog-menuitem-content" id="" href="http://www.google.com/advanced_patent_search">Advanced Patent Search</a></li></ol></div><table id="viewport_table" cellpadding="0" style="clear:both" cellspacing="0"><tr><td id="viewport_td"><div class=vertical_module_list_row><div id=intl_patents class=about_content><div id=intl_patents_v><table class="patent-bibdata"><tr><td class="patent-bibdata-heading">Publication number</td><td class="single-patent-bibdata">US5612179 A</td></tr><tr><td class="patent-bibdata-heading">Publication type</td><td class="single-patent-bibdata">Grant</td></tr><tr><td class="patent-bibdata-heading">Application number</td><td class="single-patent-bibdata">US 07/949,652</td></tr><tr><td class="patent-bibdata-heading">Publication date</td><td class="single-patent-bibdata">Mar 18, 1997</td></tr><tr><td class="patent-bibdata-heading">Filing date</td><td class="single-patent-bibdata">Sep 23, 1992</td></tr><tr><td class="patent-bibdata-heading">Priority date<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The priority date is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the date listed."></span></td><td class="single-patent-bibdata">Aug 25, 1989</td></tr><tr><td class="patent-bibdata-heading">Fee status<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The fee status is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the status or dates listed."></span></td><td class="single-patent-bibdata">Paid</td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Also published as</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/CA2023888A1">CA2023888A1</a>, </span><span class="patent-bibdata-value"><a href="/patents/CA2023888C">CA2023888C</a>, </span><span class="patent-bibdata-value"><a href="/patents/DE69029018D1">DE69029018D1</a>, </span><span class="patent-bibdata-value"><a href="/patents/DE69029018T2">DE69029018T2</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP0414469A2">EP0414469A2</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP0414469A3">EP0414469A3</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP0414469B1">EP0414469B1</a>, </span><span class="patent-bibdata-value"><a href="/patents/US5192659">US5192659</a>, </span><span class="patent-bibdata-value"><a href="/patents/US5789568">US5789568</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20030119003">US20030119003</a></span></span></td></tr><tr class="patent-bibdata-list-row alternate-patent-number"><td class="patent-bibdata-heading">Publication number</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value">07949652, </span><span class="patent-bibdata-value">949652, </span><span class="patent-bibdata-value">US 5612179 A, </span><span class="patent-bibdata-value">US 5612179A, </span><span class="patent-bibdata-value">US-A-5612179, </span><span class="patent-bibdata-value">US5612179 A, </span><span class="patent-bibdata-value">US5612179A</span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Inventors</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Malcolm+J.+Simons%22">Malcolm J. Simons</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Original Assignee</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=inassignee:%22Genetype+A.G.%22">Genetype A.G.</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Export Citation</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/US5612179.bibtex">BiBTeX</a>, </span><span class="patent-bibdata-value"><a href="/patents/US5612179.enw">EndNote</a>, </span><span class="patent-bibdata-value"><a href="/patents/US5612179.ris">RefMan</a></span></span></td></tr><tr class="patent-internal-links"><td colspan=2><span class="patent-bibdata-value"><a href="#backward-citations">Patent Citations</a> (9),</span> <span class="patent-bibdata-value"><a href="#npl-citations">Non-Patent Citations</a> (56),</span> <span class="patent-bibdata-value"><a href="#forward-citations">Referenced by</a> (70),</span> <span class="patent-bibdata-value"><a href="#classifications">Classifications</a> (20),</span> <span class="patent-bibdata-value"><a href="#legal-events">Legal Events</a> (14)</span> </td></tr><tr><td colspan=2 class="patent-bibdata-external-link-spacer-top"></td></tr><tr class="patent-bibdata-external-link-spacer-bottom"></tr><tr><td colspan=2><span class="patent-bibdata-heading">External Links:&nbsp;</span><span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=bhJCBAABERAJ&q=http://patft.uspto.gov/netacgi/nph-Parser%3FSect2%3DPTO1%26Sect2%3DHITOFF%26p%3D1%26u%3D/netahtml/PTO/search-bool.html%26r%3D1%26f%3DG%26l%3D50%26d%3DPALL%26RefSrch%3Dyes%26Query%3DPN/5612179&usg=AFQjCNFyKsKObFU5AD6DIZ5yj8uxLzf4Xg">USPTO</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=bhJCBAABERAJ&q=http://assignments.uspto.gov/assignments/q%3Fdb%3Dpat%26pat%3D5612179&usg=AFQjCNHtPR71e8L1ZWE4pIZpiRn1RrlIAQ">USPTO Assignment</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=bhJCBAABERAJ&q=http://worldwide.espacenet.com/publicationDetails/biblio%3FCC%3DUS%26NR%3D5612179A%26KC%3DA%26FT%3DD&usg=AFQjCNG2B2wlbicLm_AZZKerkXEYfrnO1A">Espacenet</a></span></span></td></tr><tr class="patent-bibdata-group-spacer"></tr></table><div class="number-and-title"><span class="patent-title"><invention-title mxw-id="PT54151220" lang="EN" load-source="patent-office">Intron sequence analysis method for detection of adjacent and remote locus alleles as haplotypes</invention-title></span><br><span class="patent-number">US 5612179 A</span></div><div class="patent-section patent-abstract-section"><div class="patent-section-header"><span class="patent-section-title">Abstract</span></div><div class="patent-text"><abstract mxw-id="PA37622543" lang="EN" load-source="patent-office"> <div class="abstract">The present invention provides a method for detection of at least one allele of a genetic locus and can be used to provide direct determination of the haplotype. The method comprises amplifying genomic DNA with a primer pair that spans an intron sequence and defines a DNA sequence in genetic linkage with an allele to be detected. The primer-defined DNA sequence contains a sufficient number of intron sequence nucleotides to characterize the allele. Genomic DNA is amplified to produce an amplified DNA sequence characteristic of the allele. The amplified DNA sequence is analyzed to detect the presence of a genetic variation in the amplified DNA sequence such as a change in the length of the sequence, gain or loss of a restriction site or substitution of a nucleotide. The variation is characteristic of the allele to be detected and can be used to detect remote alleles. Kits comprising one or more of the reagents used in the method are also described.</div>
  </abstract></div></div><div class="patent-section patent-drawings-section"><div class="patent-section-header"><span class="patent-section-title">Images<span class="patent-section-count">(41)</span></span></div><div class="patent-drawings-body"><div class="patent-drawings-carousel"><div class="drawings"><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5612179-0.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5612179-0.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5612179-1.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5612179-1.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5612179-2.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5612179-2.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5612179-3.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5612179-3.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5612179-4.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5612179-4.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5612179-5.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5612179-5.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5612179-6.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5612179-6.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5612179-7.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5612179-7.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5612179-8.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5612179-8.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5612179-9.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5612179-9.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5612179-10.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5612179-10.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5612179-11.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5612179-11.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5612179-12.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5612179-12.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5612179-13.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5612179-13.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5612179-14.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5612179-14.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5612179-15.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5612179-15.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5612179-16.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5612179-16.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5612179-17.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5612179-17.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5612179-18.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5612179-18.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5612179-19.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5612179-19.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5612179-20.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5612179-20.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5612179-21.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5612179-21.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5612179-22.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5612179-22.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5612179-23.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5612179-23.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5612179-24.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5612179-24.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5612179-25.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5612179-25.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5612179-26.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5612179-26.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5612179-27.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5612179-27.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5612179-28.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5612179-28.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5612179-29.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5612179-29.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5612179-30.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5612179-30.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5612179-31.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5612179-31.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5612179-32.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5612179-32.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5612179-33.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5612179-33.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5612179-34.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5612179-34.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5612179-35.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5612179-35.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5612179-36.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5612179-36.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5612179-37.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5612179-37.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5612179-38.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5612179-38.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5612179-39.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5612179-39.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5612179-40.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5612179-40.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div></div></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img" alt="Previous page"src="/googlebooks/images/kennedy/page_left.png"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img" alt="Next page"src="/googlebooks/images/kennedy/page_right.png"width="21" height="21" /></div></div></div><div class="patent-post-drawings"></div><div class="patent-section patent-claims-section"><div class="patent-section-header"><span class="patent-section-title">Claims<span class="patent-section-count">(36)</span></span></div><div class="patent-text"><div mxw-id="PCLM5071176" lang="EN" load-source="patent-office" class="claims">
    <claim-statement>What is claimed is:</claim-statement> <div class="claim"> <div num="1" class="claim">
      <div class="claim-text">1. A method for detection of at least one coding region allele of a multi-allelic genetic locus comprising:<div class="claim-text">a) amplifying genomic DNA with a primer pair that spans a non-coding region sequence, said primer pair defining a DNA sequence which is in genetic linkage with said genetic locus and contains a sufficient number of non-coding region sequence nucleotides to produce an amplified DNA sequence characteristic of said allele; and</div> <div class="claim-text">b) analyzing the amplified DNA sequence to detect the allele.</div> </div>
    </div>
    </div> <div class="claim-dependent"> <div num="2" class="claim">
      <div class="claim-text">2. The method of claim 1 wherein said amplified DNA sequence includes at least about 300 nucleotides corresponding to non-coding region sequences.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="3" class="claim">
      <div class="claim-text">3. The method of claim 1 wherein said non-coding region sequence is adjacent to an exon encoding said allele.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="4" class="claim">
      <div class="claim-text">4. The method of claim 1 wherein said amplified DNA sequence is characteristic of at least one nonadjacent allele.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="5" class="claim">
      <div class="claim-text">5. The method of claim 1 wherein said amplified DNA sequence is characteristic of at least one adjacent allele and at least one nonadjacent allele.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="6" class="claim">
      <div class="claim-text">6. The method of claim 5 wherein said amplified DNA sequence includes at least about 1,000 nucleotides corresponding to non-coding region sequences.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="7" class="claim">
      <div class="claim-text">7. The method of claim 1 wherein said genetic locus has at least four alleles.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="8" class="claim">
      <div class="claim-text">8. The method of claim 1 wherein said genetic locus has at least eight alleles.</div>
    </div>
    </div> <div class="claim"> <div num="9" class="claim">
      <div class="claim-text">9. A method for detection of at least one allele of a multi-allelic genetic locus comprising:<div class="claim-text">a) amplifying genomic DNA with a primer pair that spans a non-coding region sequence, said primer pair defining a DNA sequence which is in genetic linkage with said allele and contains a sufficient number of non-coding region sequence nucleotides to produce an amplified DNA sequence characteristic of said allele; and</div> <div class="claim-text">b) analyzing said amplified DNA sequence to determine the presence of a genetic variation in said amplified sequence to detect the allele.</div> </div>
    </div>
    </div> <div class="claim-dependent"> <div num="10" class="claim">
      <div class="claim-text">10. The method of claim 9 wherein said variation in said amplified DNA sequence is a variation in the length of the primer-defined amplified DNA sequence.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="11" class="claim">
      <div class="claim-text">11. The method of claim 9 wherein said variation in said amplified DNA sequence is a change in the presence of at least one restriction site in the primer-defined amplified DNA sequence.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="12" class="claim">
      <div class="claim-text">12. The method of claim 9 wherein said variation in said amplified DNA sequence is a change in the location of at least one restriction site in the primer-defined amplified DNA sequence.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="13" class="claim">
      <div class="claim-text">13. The method of claim 9 wherein said variation in said amplified DNA sequence is a substitution of at least one nucleotide in the primer-defined amplified DNA sequence.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="14" class="claim">
      <div class="claim-text">14. The method of claim 9 wherein said genetic locus is a major histocompatibility locus.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="15" class="claim">
      <div class="claim-text">15. The method of claim 9 wherein said allele is associated with a monogenic disease.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="16" class="claim">
      <div class="claim-text">16. The method of claim 15 wherein said monogenic disease is cystic fibrosis.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="17" class="claim">
      <div class="claim-text">17. The method of claim 9 wherein at least about 70% of said primer-defined amplified DNA sequence corresponds to non-coding region sequences.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="18" class="claim">
      <div class="claim-text">18. The method of claim 9 wherein said primer-defined amplified DNA sequence is from 300 to 500 nucleotides in length.</div>
    </div>
    </div> <div class="claim"> <div num="19" class="claim">
      <div class="claim-text">19. A method for producing RFLP patterns for an HLA locus of an individual comprising the steps of:<div class="claim-text">a) amplifying HLA DNA from said individual with a primer pair specific for said HLA locus under conditions suitable to produce an amplified DNA sequence, primer sites for said primers being located in intervening sequence I and in intervening sequence III when said HLA locus is a Class I locus and in intervening sequence I and in intervening sequence II when said locus is a Class II locus;</div> <div class="claim-text">b) producing a digest by combining said amplified DNA sequence with at least one endonuclease that cleaves said amplified DNA sequence to yield a set of fragments having distinctive fragment lengths; and</div> <div class="claim-text">c) producing RFLP patterns from said digest.</div> </div>
    </div>
    </div> <div class="claim-dependent"> <div num="20" class="claim">
      <div class="claim-text">20. The method of claim 19 wherein said amplification comprises:<div class="claim-text">i) combining said primer pair with HLA DNA from said individual under hybridizing conditions for a period of time sufficient for each primer in said primer pair to produce an extension product which, when separated from its complement, can serve as a template for synthesis of the extension product of the other primer to produce a mixture;</div> <div class="claim-text">ii) treating said mixture under denaturing conditions to separate the primers from their extension products;</div> <div class="claim-text">iii) treating said mixture with said primer pair such that a primer extension product is synthesized using each of the templates produced in step ii as a template, resulting in amplification of the HLA DNA; and</div> <div class="claim-text">iv) repeating steps ii and iii to produce an amplified DNA sequence.</div> </div>
    </div>
    </div> <div class="claim-dependent"> <div num="21" class="claim">
      <div class="claim-text">21. The method of claim 19 wherein a second primer pair specific for said HLA locus is also used to amplify said HLA DNA.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="22" class="claim">
      <div class="claim-text">22. The method of claim 19 wherein producing said RFLP fragment pattern comprises:<div class="claim-text">i). combining said amplified DNA sequence with at least one endonuclease that cleaves said amplified DNA sequence to yield a set of fragments having distinctive fragment lengths;</div> <div class="claim-text">ii). separating said fragments based on the length of the fragments to produce separated fragments; and</div> <div class="claim-text">iii). visualizing said separated fragments to produce RFLP fragment patterns.</div> </div>
    </div>
    </div> <div class="claim-dependent"> <div num="23" class="claim">
      <div class="claim-text">23. The method of claim 22 wherein said fragments are separated using gel electrophoresis and visualized using a nucleotide-specific stain.</div>
    </div>
    </div> <div class="claim"> <div num="24" class="claim">
      <div class="claim-text">24. A method for determining whether DNA in a sample is from a particular individual comprising the steps of:<div class="claim-text">a) amplifying DNA from said individual and DNA from said sample with a primer pair specific for an HLA locus under suitable conditions to produce an amplified DNA sequence from said individual and from said sample, said primers being located in intervening sequences I and III for an HLA Class I locus and in intervening sequences I and II for a Class II locus;</div> <div class="claim-text">b) combining said amplified DNA sequence from said individual and said amplified sample DNA from said sample with at least one endonuclease that cleaves said amplified DNA sequence into a plurality of cleaved sequences of sufficiently different lengths to distinguish between alleles of said HLA locus for a period of time sufficient for digestion of said amplified DNA to produce a digest; and</div> <div class="claim-text">c) comparing restriction fragment length polymorphic patterns produced by said digest from said individual and from said sample to determine whether DNA in the sample is from the individual.</div> </div>
    </div>
    </div> <div class="claim"> <div num="25" class="claim">
      <div class="claim-text">25. A method for determining whether an individual is the father of a child comprising the steps of:<div class="claim-text">a) amplifying DNA from said individual, DNA from said child and DNA from said child's mother with a pair of primers specific for an HLA locus under suitable conditions to produce amplified DNA sequences, said primers being located in intervening sequences I and III for an HLA Class I locus and in intervening sequences I and II for a Class II locus;</div> <div class="claim-text">b) combining said amplified DNA sequence from said individual, said amplified DNA sequence from said child's mother, and said amplified sample DNA from said child with at least one endonuclease that cleaves said amplified DNA sequence into a plurality of cleaved sequences of sufficiently different lengths to distinguish between alleles of said HLA locus to produce a digest; and</div> <div class="claim-text">c) comparing restriction fragment length polymorphic patterns produced by said digest from said individual, from said child's mother and from said child.</div> </div>
    </div>
    </div> <div class="claim"> <div num="26" class="claim">
      <div class="claim-text">26. A DNA analysis method for determining coding region alleles of a multi-allelic genetic locus comprising identifying sequence polymorphisms characteristic of the alleles, wherein said sequence polymorphisms characteristic of the alleles are present in a non-coding region sequence, said non-coding region sequence being not more than about two kilobases in length.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="27" class="claim">
      <div class="claim-text">27. The method of claim 26 wherein said sequence polymorphisms characteristic of the alleles are within five kilobases of a variable exon of the genetic locus.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="28" class="claim">
      <div class="claim-text">28. The method of claim 26 wherein said sequence polymorphisms characteristic of the alleles are within two kilobases of a variable exon of the genetic locus.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="29" class="claim">
      <div class="claim-text">29. The method of claim 26 wherein said sequence polymorphism characteristic of said coding region allele is present in an intervening sequence adjacent to a variable exon of the locus.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="30" class="claim">
      <div class="claim-text">30. The method of claim 29 wherein the genetic locus is an HLA Class I locus and the intervening sequence is intervening sequence I, II or III.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="31" class="claim">
      <div class="claim-text">31. The method of claim 29 wherein the genetic locus is an HLA Class II locus and the intervening sequence is intervening sequence I or II.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="32" class="claim">
      <div class="claim-text">32. The method of claim 26 wherein said non-coding region sequence is not more than about one kilobase in length.</div>
    </div>
    </div> <div class="claim"> <div num="33" class="claim">
      <div class="claim-text">33. A method for producing RFLP fragments characteristic of alleles of an HLA locus of an individual comprising the steps of:<div class="claim-text">a) amplifying genomic HLA DNA from said individual with a primer pair specific for said HLA locus under conditions suitable to produce an amplified DNA sequence; and</div> <div class="claim-text">b) producing a digest by combining said amplified DNA sequence with at least one endonuclease that cleaves said amplified DNA sequence to yield a set of fragments having distinctive fragment lengths.</div> </div>
    </div>
    </div> <div class="claim-dependent"> <div num="34" class="claim">
      <div class="claim-text">34. The method of claim 33 additionally comprising the step of producing RFLP patterns from said digest.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="35" class="claim">
      <div class="claim-text">35. The method of claim 33 wherein said primers define a DNA sequence that contains all exons that encode allelic variability associated with said HLA locus.</div>
    </div>
    </div> <div class="claim"> <div num="36" class="claim">
      <div class="claim-text">36. A method for producing RFLP fragments for an HLA locus of an individual comprising the steps of:<div class="claim-text">a) amplifying genomic HLA DNA from said individual with a primer pair specific for said HLA locus under conditions suitable to produce an amplified DNA sequence, said primers defining a DNA sequence that contains all exons that encode allelic variability associated with said HLA locus; and</div> <div class="claim-text">b) producing a digest by combining said amplified DNA sequence with at least one endonuclease that cleaves said amplified DNA sequence to yield a set of fragments having distinctive fragment lengths.</div> </div>
    </div>
  </div> </div></div></div><div class="patent-section patent-description-section"><div class="patent-section-header"><span class="patent-section-title">Description</span></div><div class="patent-text"><div mxw-id="PDES66959593" lang="EN" load-source="patent-office" class="description">
    <heading>CROSS REFERENCE TO RELATED APPLICATION</heading> <p>This application is a continuation of application Ser. No. 07/551,239, filed Jul. 11, 1990. This application is a continuation of U.S. Pat. No. 5,192,659, issued Mar. 9, 1993, (U.S. Ser. No. 07/551,239, filed Jul. 11, 1990), which is a continuation-in-part of U.S. Ser. No. 07/465,863, filed Jan. 16, 1990, which is a continuation in part of U.S. Ser. No. 07/405,499, filed Sep. 11, 1989 (now abandoned), which is a continuation in part of U.S. application Ser. No. 07/398,217, filed Aug. 25, 1989 (now abandoned).</p>
    <heading>FIELD OF THE INVENTION</heading> <p>The present invention relates to a method for detection of alleles and haplotypes and reagents therefor.</p>
    <heading>BACKGROUND OF THE INVENTION</heading> <p>Due in part to a number of new analytical techniques, there has been a significant increase in knowledge about genetic information, particularly in humans. Allelic variants of genetic loci have been correlated to malignant and non-malignant monogenic and multigenic diseases. For example, monogenic diseases for which the defective gene has been identified include DuChenne muscular dystrophy, sickle-cell anemia, Lesch Nyhan syndrome, hemophilia, beta-thalassemia, cystic fibrosis, polycystic kidney disease, ADA deficiency, α-1-antitrypsin deficiency, Wilm's tumor and retinoblastoma. Other diseases which are believed to be monogenic for which the gene has not been identified include fragile X mental retardation and Huntington's chorea.</p>
    <p>Genes associated with multigenic diseases such as diabetes, colon cancer and premature coronary atherosclerosis have also been identified.</p>
    <p>In addition to identifying individuals at risk for or carriers of genetic diseases, detection of allelic variants of a genetic locus has been used for organ transplantation, forensics, disputed paternity and a variety of other purposes in humans. In commercially important plants and animals, genes have not only been analyzed but genetically engineered and transmitted into other organisms.</p>
    <p>A number of techniques have been employed to detect allelic variants of genetic loci including analysis of restriction fragment length polymorphic (RFLP) patterns, use of oligonucleotide probes, and DNA amplification methods. One of the most complicated groups of allelic variants, the major histocompatibility complex (MHC), has been extensively studied. The problems encountered in attempting to determine the HLA type of an individual are exemplary of problems encountered in characterizing other genetic loci.</p>
    <p>The major histocompatibility complex is a cluster of genes that occupy a region on the short arm of chromosome 6. This complex, denoted the human leukocyte antigen (HLA) complex, includes at least 50 loci. For the purposes of HLA tissue typing, two main classes of loci are recognized. The Class I loci encode transplantation antigens and are designated A, B and C. The Class II loci (DRA, DRB, DQA1, DQB, DPA and DPB) encode products that control immune responsiveness. Of the Class II loci, all the loci are polymorphic with the exception of the DRA locus. That is, the DRA antigen polypeptide sequence is invariant.</p>
    <p>HLA determinations are used in paternity determinations, transplant compatibility testing, forensics, blood component therapy, anthropological studies, and in disease association correlations to diagnose disease or predict disease susceptibility. Due power of HLA to distinguish individuals and the need to match HLA type for transplantation, analytical methods to unambiguously characterize the alleles of the genetic loci associated with the complex have been sought. At present, DNA typing using RFLP and oligonucleotide probes has been used to type Class II locus alleles. Alleles of Class I loci and Class II DR and DQ loci are typically determined by serological methods. The alleles of the Class II DP locus are determined by primed lymphocyte typing (PLT).</p>
    <p>Each of the HLA analysis methods has drawbacks. Serological methods require standard sera that are not widely available and must be continuously replenished. Additionally, serotyping is based on the reaction of the HLA gene products in the sample with the antibodies in the reagent sera. The antibodies recognize the expression products of the HLA genes on the surface of nucleated cells. The determination of fetal HLA type by serological methods may be difficult due to lack of maturation of expression of the antigens in fetal blood cells.</p>
    <p>Oligonucleotide probe typing can be performed in two days and has been further improved by the recent use of polymerase chain reaction (PCR) amplification. PCR-based oligoprobe typing has been performed on Class II loci. Primed lymphocyte typing requires 5 to 10 days to complete and involves cell culture with its difficulties and inherent variability.</p>
    <p>RFLP analysis is time consuming, requiring about 5 to 7 days to complete. Analysis of the fragment patterns is complex. Additionally, the technique requires the use of labelled probes. The most commonly used label, <sup>32</sup> P, presents well known drawbacks associated with the use of radionuclides.</p>
    <p>A fast, reliable method of genetic locus analysis is highly desirable.</p>
    <heading>DESCRIPTION OF THE PRIOR ART</heading> <p>U.S. Pat. No. 4,683,195 (to Mullis et al, issued Jul. 28, 1987) describes a process for amplifying, detecting and/or cloning nucleic acid sequences. The method involves treating separate complementary strands of DNA with two oligonucleotide primers, extending the primers to form complementary extension products that act as templates for synthesizing the desired nucleic acid sequence and detecting the amplified sequence. The method is commonly referred to as the polymerase chain reaction sequence amplification method or PCR. Variations of the method are described in U.S. Pat. No. 4,683,194 (to Saiki et al, issued Jul. 28, 1987). The polymerase chain reaction sequence amplification method is also described by Saiki et al, Science, 230:1350-1354 (1985) and Scharf et al, Science, 324:163-166 (1986).</p>
    <p>U.S. Pat. No. 4,582,788 (to Erlich, issued Apr. 15, 1986) describes an HLA typing method based on restriction length polymorphism (RFLP) and cDNA probes used therewith. The method is carried out by digesting an individual's HLA DNA with a restriction endonuclease that produces a polymorphic digestion pattern, subjecting the digest to genomic blotting using a labelled cDNA probe that is complementary to an HLA DNA sequence involved in the polymorphism, and comparing the resulting genomic blotting pattern with a standard. Locus-specific probes for Class II loci (DQ) are also described.</p>
    <p>Kogan et al, New Engl. J. Med, 317:985-990 (1987) describes an improved PCR sequence amplification method that uses a heat-stable polymerase (Taq polymerase) and high temperature amplification. The stringent conditions used in the method provide sufficient fidelity of replication to permit analysis of the amplified DNA by determining DNA sequence lengths by visual inspection of an ethidium bromide-stained gel. The method was used to analyze DNA associated with hemophilia A in which additional tandem repeats of a DNA sequence are associated with the disease and the amplified sequences were significantly longer than sequences that are not associated with the disease.</p>
    <p>Simons and Erlich, pp 952-958 In: Immunology of HLA Vol. 1: Springer-Verlag, New York (1989) summarized RFLP-sequence interrelations at the DPA and DPB loci. RFLP fragment patterns analyzed with probes by Southern blotting provided distinctive patterns for DPw1-5 alleles and the corresponding DPB1 allele sequences, characterized two subtypic patterns for DPw2 and DPw4, and identified new DPw alleles.</p>
    <p>Simons et al, pp 959-1023 In: Immunology of HLA Vol. 1: Springer-Verlag, New York (1989) summarized restriction length polymorphisms of HLA sequences for class II loci as determined by the 10th International Workshop Southern Blot Analysis. Southern blot analysis was shown to be suitable for typing of the major classes of HLA loci.</p>
    <p>A series of three articles [Rommens et al, Science 245:1059-1065 (1989), Riordan et al, Science 245:1066-1072 (1989) and Kerem et al, Science 245:1073-1079 (1989) report a new gene analysis method called "jumping" used to identify the location of the CF gene, the sequence of the CF gene, and the defect in the gene and its percentage in the disease population, respectively.</p>
    <p>DiLelia et al, The Lancet i:497-499 (1988) describes a screening method for detecting the two major alleles responsible for phenylketonuria in caucasians of Northern European descent. The mutations, located at about the center of exon 12 and at the exon 12 junction with intervening sequence 12 are detected by PCR amplification of a 245 bp region of exon 12 and flanking intervening sequences. The amplified sequence encompasses both mutations and is analyzed using probes specific for each of the alleles (without prior electrophoretic separation).</p>
    <p>Dicker et al, BioTechniques 7:830-837 (1989) and Mardis et al, BioTechniques 7:840-850 (1989) report on automated techniques for sequencing of DNA sequences, particularly PCR-generated sequences.</p>
    <p>Each of the above-described references is incorporated herein by reference in its entirety.</p>
    <heading>SUMMARY OF THE INVENTION</heading> <p>The present invention provides a method for detection of at least one allele of a genetic locus and can be used to provide direct determination of the haplotype. The method comprises amplifying genomic DNA with a primer pair that spans an intron sequence and defines a DNA sequence in genetic linkage with an allele to be detected. The primer-defined DNA sequence contains a sufficient number of intron sequence nucleotides to characterize the allele. Genomic DNA is amplified to produce an amplified DNA sequence characteristic of the allele. The amplified DNA sequence is analyzed to detect the presence of a genetic variation in the amplified DNA sequence such as a change in the length of the sequence, gain or loss of a restriction site or substitution of a nucleotide. The variation is characteristic of the allele to be detected.</p>
    <p>The present invention is based on the finding that intron sequences contain genetic variations that are characteristic of adjacent and remote alleles on the same chromosome. In particular, DNA sequences that include a sufficient number of intron sequence nucleotides can be used for direct determination of haplotype.</p>
    <p>The method can be used to detect alleles of genetic loci for any eukaryotic organism. Of particular interest are loci associated with malignant and nonmalignant monogenic and multigenic diseases, and identification of individual organisms or species in both plants and animals. In a preferred embodiment, the method is used to determine HLA allele type and haplotype.</p>
    <p>Kits comprising one or more of the reagents used in the method are also described.</p>
    <heading>DETAILED DESCRIPTION OF THE INVENTION</heading> <p>The present invention provides a method for detection of alleles and haplotypes through analysis of intron sequence variation. The present invention is based on the discovery that amplification of intron sequences that exhibit linkage disequilibrium with adjacent and remote loci can be used to detect alleles of those loci. The present method reads haplotypes as the direct output of the intron typing analysis when a single, individual organism is tested. The method is particularly useful in humans but is generally applicable to all eukaryotes, and is preferably used to analyze plant and animal species.</p>
    <p>The method comprises amplifying genomic DNA with a primer pair that spans an intron sequence and defines a DNA sequence in genetic linkage with an allele to be detected. Primer sites are located in conserved regions in the introns or exons bordering the intron sequence to be amplified. The primer-defined DNA sequence contains a sufficient number of intron sequence nucleotides to characterize the allele. The amplified DNA sequence is analyzed to detect the presence of a genetic variation such as a change in the length of the sequence, gain or loss of a restriction site or substitution of a nucleotide.</p>
    <p>The intron sequences provide genetic variations that, in addition to those found in exon sequences, further distinguish sample DNA, providing additional information about the individual organism. This information is particularly valuable for identification of individuals such as in paternity determinations and in forensic applications. The information is also valuable in any other application where heterozygotes (two different alleles) are to be distinguished from homozygotes (two copies of one allele).</p>
    <p>More specifically, the present invention provides information regarding intron variation. Using the methods and reagents of this invention, two types of intron variation associated with genetic loci have been found. The first is allele-associated intron variation. That is, the intron variation pattern associates with the allele type at an adjacent locus. The second type of variation is associated with remote alleles (haplotypes). That is, the variation is present in individual organisms with the same genotype at the primary locus. Differences may occur between sequences of the same adjacent and remote locus types. However, individual-limited variation is uncommon.</p>
    <p>Furthermore, an amplified DNA sequence that contains sufficient intron sequences will vary depending on the allele present in the sample. That is, the introns contain genetic variations (e.g. length polymorphisms due to insertions and/or deletions and changes in the number or location of restriction sites) which are associated with the particular allele of the locus and with the alleles at remote loci.</p>
    <p>The reagents used in carrying out the methods of this invention are also described. The reagents can be provided in kit form comprising one or more of the reagents used in the method.</p>
    <heading>Definitions</heading> <p>The term "allele", as used herein, means a genetic variation associated with a coding region; that is, an alternative form of the gene.</p>
    <p>The term "linkage", as used herein, refers to the degree to which regions of genomic DNA are inherited together. Regions on different chromosomes do not exhibit linkage and are inherited together 50% of the time. Adjacent genes that are always inherited together exhibit 100% linkage.</p>
    <p>The term "linkage disequilibrium", as used herein, refers to the co-occurrence of two alleles at linked loci such that the frequency of the co-occurrence of the alleles is greater than would be expected from the separate frequencies of occurrence of each allele. Alleles that co-occur with frequencies expected from their separate frequencies are said to be in "linkage equilibrium".</p>
    <p>As used herein, "haplotype" is a region of genomic DNA on a chromosome which is bounded by recombination sites such that genetic loci within a haplotypic region are usually inherited as a unit. However, occasionally, genetic rearrangements may occur within a haplotype. Thus, the term haplotype is an operational term that refers to the occurrence on a chromosome of linked loci.</p>
    <p>As used herein, the term "intron" refers to untranslated DNA sequences between exons, together with 5' and 3' untranslated regions associated with a genetic locus. In addition, the term is used to refer to the spacing sequences between genetic loci (intergenic spacing sequences) which are not associated with a coding region and are colloquially referred to as "junk". While the art traditionally uses the term "intron" to refer only to untranslated sequences between exons, this expanded definition was necessitated by the lack of any art recognized term which encompasses all non-exon sequences.</p>
    <p>As used herein, an "intervening sequence" is an intron which is located between two exons within a gene. The term does not encompass upstream and downstream noncoding sequences associated with the genetic locus.</p>
    <p>As used herein, the term "amplified DNA sequence" refers to DNA sequences which are copies of a portion of a DNA sequence and its complementary sequence, which copies correspond in nucleotide sequence to the original DNA sequence and its complementary sequence.</p>
    <p>The term "complement", as used herein, refers to a DNA sequence that is complementary to a specified DNA sequence.</p>
    <p>The term "primer site", as used herein, refers to the area of the target DNA to which a primer hybridizes.</p>
    <p>The term "primer pair", as used herein, means a set of primers including a 5' upstream primer that hybridizes with the 5' end of the DNA sequence to be amplified and a 3' downstream primer that hybridizes with the complement of the 3' end of the sequence to be amplified.</p>
    <p>The term "exon-limited primers", as used herein, means a primer pair having primers located within or just outside of an exon in a conserved portion of the intron, which primers amplify a DNA sequence which includes an exon or a portion thereof and not more than a small, para-exon region of the adjacent intron(s).</p>
    <p>The term "intron-spanning primers", as used herein, means a primer pair that amplifies at least a portion of one intron, which amplified intron region includes sequences which are not conserved. The intron-spanning primers can be located in conserved regions of the introns or in adjacent, upstream and/or downstream exon sequences.</p>
    <p>The term "genetic locus", as used herein, means the region of the genomic DNA that includes the gene that encodes a protein including any upstream or downstream transcribed noncoding regions and associated regulatory regions. Therefore, an HLA locus is the region of the genomic DNA that includes the gene that encodes an HLA gene product.</p>
    <p>As used herein, the term "adjacent locus" refers to either (1) the locus in which a DNA sequence is located or (2) the nearest upstream or downstream genetic locus for intron DNA sequences not associated with a genetic locus.</p>
    <p>As used herein, the term "remote locus" refers to either (1) a locus which is upstream or downstream from the locus in which a DNA sequence is located or (2) for intron sequences not associated with a genetic locus, a locus which is upstream or downstream from the nearest upstream or downstream genetic locus to the intron sequence.</p>
    <p>The term "locus-specific primer", as used herein, means a primer that specifically hybridizes with a portion of the stated gene locus or its complementary strand, at least for one allele of the locus, and does not hybridize with other DNA sequences under the conditions used in the amplification method.</p>
    <p>As used herein, the terms "endonuclease" and "restriction endonuclease" refer to an enzyme that cuts double-stranded DNA having a particular nucleotide sequence. The specificities of numerous endonucleases are well known and can be found in a variety of publications, e.g. Molecular Cloning: A Laboratory Manual by Maniatis et al, Cold Spring Harbor Laboratory 1982. That manual is incorporated herein by reference in its entirety.</p>
    <p>The term "restriction fragment length polymorphism" (or RFLP), as used herein, refers to differences in DNA nucleotide sequences that produce fragments of different lengths when cleaved by a restriction endonuclease.</p>
    <p>The term "primer-defined length polymorphisms" (or PDLP), as used herein, refers to differences in the lengths of amplified DNA sequences due to insertions or deletions in the intron region of the locus included in the amplified DNA sequence.</p>
    <p>The term "HLA DNA", as used herein, means DNA that includes the genes that encode HLA antigens. HLA DNA is found in all nucleated human cells.</p>
    <heading>Primers</heading> <p>The method of this invention is based on amplification of selected intron regions of genomic DNA. The methodology is facilitated by the use of primers that selectively amplify DNA associated with one or more alleles of a genetic locus of interest and not with other genetic loci.</p>
    <p>A locus-specific primer pair contains a 5' upstream primer that defines the 5' end of the amplified sequence by hybridizing with the 5' end of the target sequence to be amplified and a 3' downstream primer that defines the 3' end of the amplified sequence by hybridizing with the complement of the 3' end of the DNA sequence to be amplified. The primers in the primer pair do not hybridize with DNA of other genetic loci under the conditions used in the present invention.</p>
    <p>For each primer of the locus-specific primer pair, the primer hybridizes to at least one allele of the DNA locus to be amplified or to its complement. A primer pair can be prepared for each allele of a selected locus, which primer pair amplifies only DNA for the selected locus. In this way combinations of primer pairs can be used to amplify genomic DNA of a particular locus, irrespective of which allele is present in a sample. Preferably, the primer pair amplifies DNA of at least two, more preferably more than two, alleles of a locus. In a most preferred embodiment, the primer sites are conserved, and thus amplify all haplotypes. However, primer pairs or combinations thereof that specifically bind with the most common alleles present in a particular population group are also contemplated.</p>
    <p>The amplified DNA sequence that is defined by the primers contains a sufficient number of intron sequence nucleotides to distinguish between at least two alleles of an adjacent locus, and preferably, to identify the allele of the locus which is present in the sample. For some purposes, the sequence can also be selected to contain sufficient genetic variations to distinguish between individual organisms with the same allele or to distinguish between haplotypes.</p>
    <heading>Length of sequence</heading> <p>The length of the amplified sequence which is required to include sufficient genetic variability to enable discrimination between all alleles of a locus bears a direct relation to the extent of the polymorphism of the locus (the number of alleles). That is, as the number of alleles of the tested locus increases, the size of an amplified sequence which contains sufficient genetic variations to identify each allele increases. For a particular population group, one or more of the recognized alleles for any given locus may be absent from that group and need not be considered in determining a sequence which includes sufficient variability for that group. Conveniently, however, the primer pairs are selected to amplify a DNA sequence which is sufficient to distinguish between all recognized alleles of the tested locus. The same considerations apply when a haplotype is determined.</p>
    <p>For example, the least polymorphic HLA locus is DPA which currently has four recognized alleles. For that locus, a primer pair which amplifies only a portion of the variable exon encoding the allelic variation contains sufficient genetic variability to distinguish between the alleles when the primer sites are located in an appropriate region of the variable exon. Exon-limited primers can be used to produce an amplified sequence that includes as few as about 200 nucleotides (nt). However, as the number of alleles of the locus increases, the number of genetic variations in the sequence must increase to distinguish all alleles. Addition of invariant exon sequences provides no additional genetic variation. When about eight or more alleles are to be distinguished, as for the DQA1 locus and more variable loci, amplified sequences should extend into at least one intron in the locus, preferably an intron adjacent to the variable exon.</p>
    <p>Additionally, where alleles of the locus exist which differ by a single basepair in the variable exon, intron sequences are included in amplified sequences to provide sufficient variability to distinguish alleles. For example, for the DQA1 locus (with eight currently recognized alleles) and the DPB locus (with 24 alleles), the DQA1.1/1.2 (now referred to as DQA1 0101/0102) and DPB2.1/4.2 (now referred to as DPB0201/0402) alleles differ by a single basepair. To distinguish those alleles, amplified sequences which include an intron sequence region are required. About 300 to 500 nucleotides is sufficient, depending on the location of the sequence. That is, 300 to 500 nucleotides comprised primarily of intron sequence nucleotides sufficiently close to the variable exon are sufficient.</p>
    <p>For loci with-more extensive polymorphisms (such as DQB with 14 currently recognized alleles, DPB with 24 currently recognized alleles, DRB with 34 currently recognized alleles and for each of the Class I loci), the amplified sequences need to be larger to provide sufficient variability to distinguish between all the alleles. An amplified sequence that includes at least about 0.5 kilobases (Kb), preferably at least about 1.0 Kb, more preferably at least about 1.5 Kb generally provides a sufficient number of restriction sites for loci with extensive polymorphisms. The amplified sequences used to characterize highly polymorphic loci are generally between about 800 to about 2,000 nucleotides (nt), preferably between about 1000 to about 1800 nucleotides in length.</p>
    <p>When haplotype information regarding remote alleles is desired, the sequences are generally between about 1,000 to about 2,000 nt in length. Longer sequences are required when the amplified sequence encompasses highly conserved regions such as exons or highly conserved intron regions, e.g., promoters, operators and other DNA regulatory regions. Longer amplified sequences (including more intron nucleotide sequences) are also required as the distance between the amplified sequences and the allele to be detected increases.</p>
    <p>Highly conserved regions included in the amplified DNA sequence, such as exon sequences or highly conserved intron sequences (e.g. promoters, enhancers, or other regulatory regions) may provide little or no genetic variation. Therefore, such regions do not contribute, or contribute only minimally, to the genetic variations present in the amplified DNA sequence. When such regions are included in the amplified DNA sequence, additional nucleotides may be required to encompass sufficient genetic variations to distinguish alleles, in comparison to an amplified DNA sequence of the same length including only intron sequences.</p>
    <heading>Location of the amplified DNA sequence</heading> <p>The amplified DNA sequence is located in a region of genomic DNA that contains genetic variation which is in genetic linkage with the allele to be detected. Preferably, the sequence is located in an intron sequence adjacent to an exon of the genetic locus. More preferably, the amplified sequence includes an intervening sequence adjacent to an exon that encodes the allelic variability associated with the locus (a variable exon). The sequence preferably includes at least a portion of one of the introns adjacent to a variable exon and can include a portion of the variable exon. When additional sequence information is required, the amplified DNA sequence preferably encompasses a variable exon and all or a portion of both adjacent intron sequences.</p>
    <p>Alternatively, the amplified sequence can be in an intron which does not border an exon of the genetic locus. Such introns are located in the downstream or upstream gene flanking regions or even in an intervening sequence in another genetic locus which is in linkage disequilibrium with the allele to be detected.</p>
    <p>For some genetic loci, genomic DNA sequences may not be available. When only cDNA sequences are available and intron locations within the sequence are not identified, primers are selected at intervals of about 200 nt and used to amplify genomic DNA. If the amplified sequence contains about 200 nt, the location of the first primer is moved about 200 nt to one side of the second primer location and the amplification is repeated until either (1) an amplified DNA sequence that is larger than expected is produced or (2) no amplified DNA sequence is produced. In either case, the location of an intron sequence has been determined. The same methodology can be used when only the sequence of a marker site that is highly linked to the genetic locus is available, as is the case for many genes associated with inherited diseases.</p>
    <p>When the amplified DNA sequence does not include all or a portion of an intron adjacent to the variable exon(s), the sequence must also satisfy a second requirement. The amplified sequence must be sufficiently close to the variable exon(s) to exclude recombination and loss of linkage disequilibrium between the amplified sequence and the variable exon(s). This requirement is satisfied if the regions of the genomic DNA are within about 5 Kb, preferably within about 4 Kb, most preferably within 2 Kb of the variable exon(s). The amplified sequence can be outside of the genetic locus but is preferably within the genetic locus.</p>
    <p>Preferably, for each primer pair, the amplified DNA sequence defined by the primers includes at least 200 nucleotides, and more preferably at least 400 nucleotides, of an intervening sequence adjacent to the variable exon(s). Although the variable exon usually provides fewer variations in a given number of nucleotides than an adjacent intervening sequence, each of those variations provides allele-relevant information. Therefore, inclusion of the variable exon provides an advantage.</p>
    <p>Since PCR methodology can be used to amplify sequences of several Kb, the primers can be located so that additional exons or intervening sequences are included in the amplified sequence. Of course, the increased size of the amplified DNA sequence increases the chance of replication error, so addition of invariant regions provides some disadvantages. However, those disadvantages are not as likely to affect an analysis based on the length of the sequence or the RFLP fragment patterns as one based on sequencing the amplification product. For particular alleles, especially those with highly similar exon sequences, amplified sequences of greater than about 1 or 1.5 Kb may be necessary to discriminate between all alleles of a particular locus.</p>
    <p>The ends of the amplified DNA sequence are defined by the primer pair used in the amplification. Each primer sequence must correspond to a conserved region of the genomic DNA sequence. Therefore, the location of the amplified sequence will, to some extent, be dictated by the need to locate the primers in conserved regions. When sufficient intron sequence information to determine conserved intron regions is not available, the primers can be located in conserved portions of the exons and used to amplify intron sequences between those exons.</p>
    <p>When appropriately-located, conserved sequences are not unique to the genetic locus, a second primer located within the amplified sequence produced by the first primer pair can be used to provide an amplified DNA sequence specific for the genetic locus. At least one of the primers of the second primer pair is located in a conserved region of the amplified DNA sequence defined by the first primer pair. The second primer pair is used following amplification with the first primer pair to amplify a portion of the amplified DNA sequence produced by the first primer pair.</p>
    <p>There are three major types of genetic variations that can be detected and used to identify an allele. Those variations, in order of ease of detection, are (1) a change in the length of the sequence, (2) a change in the presence or location of at least one restriction site and (3) the substitution of one or a few nucleotides that does not result in a change in a restriction site. Other variations within the amplified DNA sequence are also detectable.</p>
    <p>There are three types of techniques which can be used to detect the variations. The first is sequencing the amplified DNA sequence. Sequencing is the most time consuming and also the most revealing analytical method, since it detects any type of genetic variation in the amplified sequence. The second analytical method uses allele-specific oligonucleotide or sequence-specific oligonucleotides probes (ASO or SSO probes). Probes can detect single nucleotide changes which result in any of the types of genetic variations, so long as the exact sequence of the variable site is known. A third type of analytical method detects sequences of different lengths (e.g., due to an insertion or deletion or a change in the location of a restriction site) and/or different numbers of sequences (due to either gain or loss of restriction sites). A preferred detection method is by gel or capillary electrophoresis. To detect changes in the lengths of fragments or the number of fragments due to changes in restriction sites, the amplified sequence must be digested with an appropriate restriction endonuclease prior to analysis of fragment length patterns.</p>
    <p>The first genetic variation is a difference in the length of the primer-defined amplified DNA sequence, referred to herein as a primer-defined length polymorphism (PDLP), which difference in length distinguishes between at least two alleles of the genetic locus. The PDLPs result from insertions or deletions of large stretches (in comparison to the total length of the amplified DNA sequence) of DNA in the portion of the intron sequence defined by the primer pair. To detect PDLPs, the amplified DNA sequence is located in a region containing insertions or deletions of a size that is detectable by the chosen method. The amplified DNA sequence should have a length which provides optimal resolution of length differences. For electrophoresis, DNA sequences of about 300 to 500 bases in length provide optimal resolution of length differences. Nucleotide sequences which differ in length by as few as 3 nt, preferably 25 to 50 nt, can be distinguished. However, sequences as long as 800 to 2,000 nt which differ by at least about 50 nt are also readily distinguishable. Gel electrophoresis and capillary electrophoresis have similar limits of resolution. Preferably the length differences between amplified DNA sequences will be at least 10, more preferably 20, most preferably 50 or more, nt between the alleles. Preferably, the amplified DNA sequence is between 300 to 1,000 nt and encompasses length differences of at least 3, preferably 10 or more nt.</p>
    <p>Preferably, the amplified sequence is located in an area which provides PDLP sequences that distinguish most or all of the alleles of a locus. An example of PDLP-based identification of five of the eight DQA1 alleles is described in detail in the examples.</p>
    <p>When the variation to be detected is a change in a restriction site, the amplified DNA sequence necessarily contains at least one restriction site which (1) is present in one allele and not in another, (2) is apparently located in a different position in the sequence of at least two alleles, or (3) combinations thereof. The amplified sequence will preferably be located such that restriction endonuclease cleavage produces fragments of detectably different lengths, rather than two or more fragments of approximately the same length.</p>
    <p>For allelic differences detected by ASO or SSO probes, the amplified DNA sequence includes a region of from about 200 to about 400 nt which is present in one or more alleles and not present in one or more other alleles. In a most preferred embodiment, the sequence contains a region detectable by a probe that is present in only one allele of the genetic locus. However, combinations of probes which react with some alleles and not others can be used to characterize the alleles.</p>
    <p>For the method described herein, it is contemplated that use of more than one amplified DNA sequence and/or use of more than one analytical method per amplified DNA sequence may be required for highly polymorphic loci, particularly for loci where alleles differ by single nucleotide substitutions that are not unique to the allele or when information regarding remote alleles (haplotypes) is desired. More particularly, it may be necessary to combine a PDLP analysis with an RFLP analysis, to use two or more amplified DNA sequences located in different positions or to digest a single amplified DNA sequence with a plurality of endonucleases to distinguish all the alleles of some loci. These combinations are intended to be included within the scope of this invention.</p>
    <p>For example, the analysis of the haplotypes of DQA1 locus described in the examples uses PDLPs and RFLP analysis using three different enzyme digests to distinguish the eight alleles and 20 of the 32 haplotypes of the locus.</p>
    <heading>Length and sequence homology of primers</heading> <p>Each locus-specific primer includes a number of nucleotides which, under the conditions used in the hybridization, are sufficient to hybridize with an allele of the locus to be amplified and to be free from hybridization with alleles of other loci. The specificity of the primer increases with the number of nucleotides in its sequence under conditions that provide the same stringency. Therefore, longer primers are desirable. Sequences with fewer than 15 nucleotides are less certain to be specific for a particular locus. That is, sequences with fewer than 15 nucleotides are more likely to be present in a portion of the DNA associated with other genetic loci, particularly loci of other common origin or evolutionarily closely related origin, in inverse proportion to the length of the nucleotide sequence.</p>
    <p>Each primer preferably includes at least about 15 nucleotides, more preferably at least about 20 nucleotides. The primer preferably does not exceed about 30 nucleotides, more preferably about 25 nucleotides. Most preferably, the primers have between about 20 and about 25 nucleotides.</p>
    <p>A number of preferred primers are described herein. Each of those primers hybridizes with at least about 15 consecutive nucleotides of the designated region of the allele sequence. For many of the primers, the sequence is not identical for all of the other alleles of the locus. For each of the primers, additional preferred primers have sequences which correspond to the sequences of the homologous region of other alleles of the locus or to their complements.</p>
    <p>When two sets of primer pairs are used sequentially, with the second primer pair amplifying the product of the first primer pair, the primers can be the same size as those used for the first amplification. However, smaller primers can be used in the second amplification and provide the requisite specificity. These smaller primers can be selected to be allele-specific, if desired. The primers of the second primer pair can have 15 or fewer, preferably 8 to 12, more preferably 8 to 10 nucleotides. When two sets of primer pairs are used to produce two amplified sequences, the second amplified DNA sequence is used in the subsequent analysis of genetic variation and must meet the requirements discussed previously for the amplified DNA sequence.</p>
    <p>The primers preferably have a nucleotide sequence that is identical to a portion of the DNA sequence to be amplified or its complement. However, a primer having two nucleotides that differ from the target DNA sequence or its complement also can be used. Any nucleotides that are not identical to the sequence or its complement are preferably not located at the 3' end of the primer. The 3' end of the primer preferably has at least two, preferably three or more, nucleotides that are complementary to the sequence to which the primer binds. Any nucleotides that are not identical to the sequence to be amplified or its complement will preferably not be adjacent in the primer sequence. More preferably, noncomplementary nucleotides in the primer sequence will be separated by at least three, more preferably at least five, nucleotides. The primers should have a melting temperature (T<sub>m</sub>) from about 55° to 75° C. Preferably the T<sub>m</sub> is from about 60° C. to about 65° C. to facilitate stringent amplification conditions.</p>
    <p>The primers can be prepared using a number of methods, such as, for example, the phosphotriester and phosphodiester methods or automated embodiments thereof. The phosphodiester and phosphotriester methods are described in Cruthers, Science 230:281-285 (1985); Brown et al, Meth. Enzymol., 68:109 (1979); and Nrang et al, Meth. Enzymol., 68:90 (1979). In one automated method, diethylphosphoramidites which can be synthesized as described by Beaucage et al, Tetrahedron letters, 22:1859-1962 (1981) are used as starting materials. A method for synthesizing primer oligonucleotide sequences on a modified solid support is described in U.S. Pat. No. 4,458,066. Each of the above references is incorporated herein by reference in its entirety.</p>
    <p>Exemplary primer sequences for analysis of Class I and Class II HLA loci; bovine leukocyte antigens, and cystic fibrosis are described herein.</p>
    <heading>Amplification</heading> <p>The locus-specific primers are used in an amplification process to produce a sufficient amount of DNA for the analysis method. For production of RFLP fragment patterns or PDLP patterns which are analyzed by electrophoresis, about 1 to about 500 ng of DNA is required. A preferred amplification method is the polymerase chain reaction (PCR). PCR amplification methods are described in U.S. Pat. No. 4,683,195 (to Mullis et al, issued Jul. 28, 1987); U.S Pat. No. 4,683,194 (to Saiki et al, issued Jul. 28, 1987); Saiki et al, Science, 230:1350-1354 (1985); Scharf et al, Science, 324:163-166 (1986); Kogan et al, New Engl. J. Med, 317:985-990 (1987) and Saiki, Gyllensten and Erlich, The Polymerase Chain Reaction in Genome Analysis: A Practical Approach, ed. Davies pp. 141-152, (1988) I.R.L. Press, Oxford. Each of the above references is incorporated herein by reference in its entirety.</p>
    <p>Prior to amplification, a sample of the individual organism's DNA is obtained. All nucleated cells contain genomic DNA and, therefore, are potential sources of the required DNA. For higher animals, peripheral blood cells are typically used rather than tissue samples. As little as 0.01 to 0.05 cc of peripheral blood provides sufficient DNA for amplification. Hair, semen and tissue can also be used as samples. In the case of fetal analyses, placental cells or fetal cells present in amniotic fluid can be used. The DNA is isolated from nucleated cells under conditions that minimize DNA degradation. Typically, the isolation involves digesting the cells with a protease that does not attack DNA at a temperature and pH that reduces the likelihood of DNase activity. For peripheral blood cells, lysing the cells with a hypotonic solution (water) is sufficient to release the DNA.</p>
    <p>DNA isolation from nucleated cells is described by Kan et al, N. Engl. J. Med. 297:1080-1084 (1977); Kan et al, Nature 251:392-392 (1974); and Kan et al, PNAS 75:5631-5635 (1978). Each of the above references is incorporated herein by reference in its entirety. Extraction procedures for samples such as blood, semen, hair follicles, semen, mucous membrane epithelium and other sources of genomic DNA are well known. For plant cells, digestion of the cells with cellulase releases DNA. Thereafter DNA is purified as described above.</p>
    <p>The extracted DNA can be purified by dialysis, chromatography, or other known methods for purifying polynucleotides prior to amplification. Typically, the DNA is not purified prior to amplification.</p>
    <p>The amplified DNA sequence is produced by using the portion of the DNA and its complement bounded by the primer pair as a template. As a first step in the method, the DNA strands are separated into single stranded DNA. This strand separation can be accomplished by a number of methods including physical or chemical means. A preferred method is the physical method of separating the strands by heating the DNA until it is substantially (approximately 93%) denatured. Heat denaturation involves temperatures ranging from about 80° to 105° C. for times ranging from about 15 to 30 seconds. Typically, heating the DNA to a temperature of from 90° to 93° C. for about 30 seconds to about 1 minute is sufficient.</p>
    <p>The primer extension product(s) produced are complementary to the primer-defined region of the DNA and hybridize therewith to form a duplex of equal length strands. The duplexes of the extension products and their templates are then separated into single-stranded DNA. When the complementary strands of the duplexes are separated, the strands are ready to be used as a template for the next cycle of synthesis of additional DNA strands.</p>
    <p>Each of the synthesis steps can be performed using conditions suitable for DNA amplification. Generally, the amplification step is performed in a buffered aqueous solution, preferably at a pH of about 7 to about 9, more preferably about pH 8. A suitable amplification buffer contains Tris-HCl as a buffering agent in the range of about 10 to 100 mM. The buffer also includes a monovalent salt, preferably at a concentration of at least about 10 mM and not greater than about 60 mM. Preferred monovalent salts are KCl, NaCl and (NH<sub>4</sub>)<sub>2</sub> SO<sub>4</sub>. The buffer also contains MgCl<sub>2</sub> at about 5 to 50 mM. Other buffering systems such as hepes or glycine-NaOH and potassium phosphate buffers can be used. Typically, the total volume of the amplification reaction mixture is about 50 to 100 μl.</p>
    <p>Preferably, for genomic DNA, a molar excess of about 10<sup>6</sup> :1 primer:template of the primer pair is added to the buffer containing the separated DNA template strands. A large molar excess of the primers improves the efficiency of the amplification process. In general, about 100 to 150 ng of each primer is added.</p>
    <p>The deoxyribonucleotide triphosphates dATP, dCTP, dGTP and dTTP are also added to the amplification mixture in amounts sufficient to produce the amplified DNA sequences. Preferably, the dNTPs are present at a concentration of about 0.75 to about 4.0 mM, more preferably about 2.0 mM. The resulting solution is heated to about 90° to 93° C. for from about 30 seconds to about 1 minute to separate the strands of the DNA. After this heating period the solution is cooled to the amplification temperature.</p>
    <p>Following separation of the DNA strands, the primers are allowed to anneal to the strands. The annealing temperature varies with the length and GC content of the primers. Those variables are reflected in the T<sub>m</sub> of each primer. Exemplary HLA DQA1 primers of this invention, described below, require temperatures of about 55° C. The exemplary HLA Class I primers of this invention require slightly higher temperatures of about 62° to about 68° C. The extension reaction step is performed following annealing of the primers to the genomic DNA.</p>
    <p>An appropriate agent for inducing or catalyzing the primer extension reaction is added to the amplification mixture either before or after the strand separation (denaturation) step, depending on the stability of the agent under the denaturation conditions. The DNA synthesis reaction is allowed to occur under conditions which are well known in the art. This synthesis reaction (primer extension) can occur at from room temperature up to a temperature above which the polymerase no longer functions efficiently. Elevating the amplification temperature enhances the stringency of the reaction. As stated previously, stringent conditions are necessary to ensure that the amplified sequence and the DNA template sequence contain the same nucleotide sequence, since substitution of nucleotides can alter the restriction sites of probe binding sites in the amplified sequence.</p>
    <p>The inducing agent may be any compound or system which facilitates synthesis of primer extension products, preferably enzymes. Suitable enzymes for this purpose include DNA polymerases (such as, for example, E. coli DNA polymerase I, Klenow fragment of E. coli DNA polymerase I, T4 DNA polymerase), reverse transcriptase, and other enzymes (including heat-stable polymerases) which facilitate combination of the nucleotides in the proper manner to form the primer extension products. Most preferred is Taq polymerase or other heat-stable polymerases which facilitate DNA synthesis at elevated temperatures (about 60° to 90° C.). Taq polymerase is described, e.g., by Chien et al, J. Bacteriol., 127:1550-1557 (1976). That article is incorporated herein by reference in its entirety. When the extension step is performed at about 72° C., about 1 minute is required for every 1000 bases of target DNA to be amplified.</p>
    <p>The synthesis of the amplified sequence is initiated at the 3' end of each primer and proceeds toward the 5' end of the template along the template DNA strand, until synthesis terminates, producing DNA sequences of different lengths. The newly synthesized strand and its complementary strand form a double-stranded molecule which is used in the succeeding steps of the process. In the next step, the strands of the double-stranded molecule are separated (denatured) as described above to provide single-stranded molecules.</p>
    <p>New DNA is synthesized on the single-stranded template molecules. Additional polymerase, nucleotides and primers can be added if necessary for the reaction to proceed under the conditions described above. After this step, half of the extension product consists of the amplified sequence bounded by the two primers. The steps of strand separation and extension product synthesis can be repeated as many times as needed to produce the desired quantity of the amplified DNA sequence. The amount of the amplified sequence produced accumulates exponentially. Typically, about 25 to 30 cycles are sufficient to produce a suitable amount of the amplified DNA sequence for analysis.</p>
    <p>The amplification method can be performed in a step-wise fashion where after each step new reagents are added, or simultaneously, where all reagents are added at the initial step, or partially step-wise and partially simultaneously, where fresh reagent is added after a given number of steps. The amplification reaction mixture can contain, in addition to the sample genomic DNA, the four nucleotides, the primer pair in molar excess, and the inducing agent, e.g., Taq polymerase.</p>
    <p>Each step of the process occurs sequentially notwithstanding the initial presence of all the reagents. Additional materials may be added as necessary. Typically, the polymerase is not replenished when using a heat-stable polymerase. After the appropriate number of cycles to produce the desired amount of the amplified sequence, the reaction may be halted by inactivating the enzymes, separating the components of the reaction or stopping the thermal cycling.</p>
    <p>In a preferred embodiment of the method, the amplification includes the use of a second primer pair to perform a second amplification following the first amplification. The second primer pair defines a DNA sequence which is a portion of the first amplified sequence. That is, at least one of the primers of the second primer pair defines one end of the second amplified sequence which is within the ends of the first amplified sequence. In this way, the use of the second primer pair helps to ensure that any amplified sequence produced in the second amplification reaction is specific for the tested locus. That is, non-target sequences which may be copied by a locus-specific pair are unlikely to contain sequences that hybridize with a second locus-specific primer pair located within the first amplified sequence.</p>
    <p>In another embodiment, the second primer pair is specific for one allele of the locus. In this way, detection of the presence of a second amplified sequence indicates that the allele is present in the sample. The presence of a second amplified sequence can be determined by quantitating the amount of DNA at the start and the end of the second amplification reaction. Methods for quantitating DNA are well known and include determining the optical density at 260 (OD<sub>260</sub>), and preferably additionally determining the ratio of the optical density at 260 to the optical density at 280 (OD<sub>260</sub> /OD<sub>280</sub>) to determine the amount of DNA in comparison to protein in the sample.</p>
    <p>Preferably, the first amplification will contain sufficient primer for only a limited number of primer extension cycles, e.g. less than 15, preferably about 10 to 12 cycles, so that the amount of amplified sequence produced by the process is sufficient for the second amplification but does not interfere with a determination of whether amplification occurred with the second primer pair. Alternatively, the amplification reaction can be continued for additional cycles and aliquoted to provide appropriate amounts of DNA for one or more second amplification reactions. Approximately 100 to 150 ng of each primer of the second primer pair is added to the amplification reaction mixture. The second set of primers is preferably added following the initial cycles with the first primer pair. The amount of the first primer pair can be limited in comparison to the second primer pair so that, following addition of the second pair, substantially all of the amplified sequences will be produced by the second pair.</p>
    <p>As stated previously, the DNA can be quantitated to determine whether an amplified sequence was produced in the second amplification. If protein in the reaction mixture interferes with the quantitation (usually due to the presence of the polymerase), the reaction mixture can be purified, as by using a 100,000 MW cut off filter. Such filters are commercially available from Millipore and from Centricon.</p>
    <heading>Analysis of the Amplified DNA Sequence</heading> <p>As discussed previously, the method used to analyze the amplified DNA sequence to characterize the allele(s) present in the sample DNA depends on the genetic variation in the sequence. When distinctions between alleles include primer-defined length polymorphisms, the amplified sequences are separated based on length, preferably using gel or capillary electrophoresis. When using probe hybridization for analysis, the amplified sequences are reacted with labeled probes. When the analysis is based on RFLP fragment patterns, the amplified sequences are digested with one or more restriction endonucleases to produce a digest and the resultant fragments are separated based on length, preferably using gel or capillary electrophoresis. When the only variation encompassed by the amplified sequence is a sequence variation that does not result in a change in length or a change in a restriction site and is unsuitable for detection by a probe, the amplified DNA sequences are sequenced.</p>
    <p>Procedures for each step of the various analytical methods are well known and are described below.</p>
    <heading>Production of RFLP Fragment Patterns</heading> <heading>Restriction endonucleases</heading> <p>A restriction endonuclease is an enzyme that cleaves or cuts DNA hydrolytically at a specific nucleotide sequence called a restriction site. Endonucleases that produce blunt end DNA fragments (hydrolysis of the phosphodiester bonds on both DNA strands occur at the same site) as well as endonucleases that produce sticky ended fragments (the hydrolysis sites on the strands are separated by a few nucleotides from each other) can be used.</p>
    <p>Restriction enzymes are available commercially from a number of sources including Sigma Pharmaceuticals, Bethesda Research Labs, Boehringer-Manheim and Pharmacia. As stated previously, a restriction endonuclease used in the present invention cleaves an amplified DNA sequence of this invention to produce a digest comprising a set of fragments having distinctive fragment lengths. In particular, the fragments for one allele of a locus differ in size from the fragments for other alleles of the locus. The patterns produced by separation and visualization of the fragments of a plurality of digests are sufficient to distinguish each allele of the locus. More particularly, the endonucleases are chosen so that by using a plurality of digests of the amplified sequence, preferably fewer than five, more preferably two or three digests, the alleles of a locus can be distinguished.</p>
    <p>In selecting an endonuclease, the important consideration is the number of fragments produced for amplified sequences of the various alleles of a locus. More particularly, a sufficient number of fragments must be produced to distinguish between the alleles and, if required, to provide for individuality determinations. However, the number of fragments must not be so large or so similar in size that a pattern that is not distinguishable from those of other haplotypes by the particular detection method is produced. Preferably, the fragments are of distinctive sizes for each allele. That is, for each endonuclease digest of a particular amplified sequence, the fragments for an allele preferably differ from the fragments for every other allele of the locus by at least 10, preferably 20, more preferably 30, most preferably 50 or more nucleotides.</p>
    <p>One of ordinary skill can readily determine whether an endonuclease produces RFLP fragments having distinctive fragment lengths. The determination can be made experimentally by cleaving an amplified sequence for each allele with the designated endonuclease in the invention method. The fragment patterns can then be analyzed. Distinguishable patterns will be readily recognized by determining whether comparison of two or more digest patterns is sufficient to demonstrate characteristic differences between the patterns of the alleles.</p>
    <p>The number of digests that need to be prepared for any particular analysis will depend on the desired information and the particular sample to be analyzed. Since HLA analyses are used for a variety of purposes ranging from individuality determinations for forensics and paternity to tissue typing for transplantation, the HLA complex will be used as exemplary.</p>
    <p>A single digest may be sufficient to determine that an individual cannot be the person whose blood was found at a crime scene. In general, however, where the DNA samples do not differ, the use of two to three digests for each of two to three HLA loci will be sufficient for matching applications (forensics, paternity). For complete HLA typing, each locus needs to be determined.</p>
    <p>In a preferred embodiment, sample HLA DNA sequences are divided into aliquots containing similar amounts of DNA per aliquot and are amplified with primer pairs (or combinations of primer pairs) to produce amplified DNA sequences for a number of HLA loci. Each amplification mixture contains only primer pairs for one HLA locus. The amplified sequences are preferably processed concurrently, so that a number of digest RFLP fragment patterns can be produced from one sample. In this way, the HLA type for a number of alleles can be determined'simultaneously.</p>
    <p>Alternatively, preparation of a number of RFLP fragment patterns provides additional comparisons of patterns to distinguish samples for forensic and paternity analyses where analysis of one locus frequently fails to provide sufficient information for the determination when the sample DNA has the same allele as the DNA to which it is compared.</p>
    <heading>Production of RFLP fragments</heading> <p>Following amplification, the amplified DNA sequence is combined with an endonuclease that cleaves or cuts the amplified DNA sequence hydrolytically at a specific restriction site. The combination of the endonuclease with the amplified DNA sequence produces a digest containing a set of fragments having distinctive fragment lengths. U.S. Pat. No. 4,582,788 (to Erlich, issued Apr. 15, 1986) describes an HLA typing method based on restriction length polymorphism (RFLP). That patent is incorporated herein by reference in its entirety.</p>
    <p>In a preferred embodiment, two or more aliquots of the amplification reaction mixture having approximately equal amounts of DNA per aliquot are prepared. Conveniently about 5 to about 10 μl of a 100 μl reaction mixture is used for each aliquot. Each aliquot is combined with a different endonuclease to produce a plurality of digests. In this way, by using a number of endonucleases for a particular amplified DNA sequence, locus-specific combinations of endonucleases that distinguish a plurality of alleles of a particular locus can be readily determined. Following preparation of the digests, each of the digests can be used to form RFLP patterns. Preferably, two or more digests can be pooled prior to pattern formation.</p>
    <p>Alternatively, two or more restriction endonucleases can be used to produce a single digest. The digest differs from one where each enzyme is used separately and the resultant fragments are pooled since fragments produced by one enzyme may include one or more restriction sites recognized by another enzyme in the digest. Patterns produced by simultaneous digestion by two or more enzymes will include more fragments than pooled products of separate digestions using those enzymes and will be more complex to analyze.</p>
    <p>Furthermore, one or more restriction endonucleases can be used to digest two or more amplified DNA sequences. That is, for more complete resolution of all the alleles of a locus, it may be desirable to produce amplified DNA sequences encompassing two different regions. The amplified DNA sequences can be combined and digested with at least one restriction endonuclease to produce RFLP patterns.</p>
    <p>The digestion of the amplified DNA sequence with the endonuclease can be carried out in an aqueous solution under conditions favoring endonuclease activity. Typically the solution is buffered to a pH of about 6.5 to 8.0. Mild temperatures, preferably about 20° C. to about 45° C., more preferably physiological temperatures (25° to 40° C.), are employed. Restriction endonucleases normally require magnesium ions and, in some instances, cofactors (ATP and S-adenosyl methionine) or other agents for their activity. Therefore, a source of such ions, for instance inorganic magnesium salts, and other agents, when required, are present in the digestion mixture. Suitable conditions are described by the manufacturer of the endonuclease and generally vary as to whether the endonuclease requires high, medium or low salt conditions for optimal activity.</p>
    <p>The amount of DNA in the digestion mixture is typically in the range of 1% to 20% by weight. In most instances 5 to 20 μg of total DNA digested to completion provides an adequate sample for production of RFLP fragments. Excess endonuclease, preferably one to five units/μg DNA, is used.</p>
    <p>The set of fragments in the digest is preferably further processed to produce RFLP patterns which are analyzed. If desired, the digest can be purified by precipitation and resuspension as described by Kan et al, PNAS 75:5631-5635 (1978), prior to additional processing. That article is incorporated herein by reference in its entirety.</p>
    <p>Once produced, the fragments are analyzed by well known methods. Preferably, the fragments are analyzed using electrophoresis. Gel electrophoresis methods are described in detail hereinafter. Capillary electrophoresis methods can be automated (as by using Model 207A analytical capillary electrophoresis system from Applied Biosystems of Foster City, Calif.) and are described in Chin et al, American Biotechnology Laboratory News Edition, December, 1989.</p>
    <heading>Electrophoretic Separation of DNA Fragments</heading> <p>Electrophoresis is the separation of DNA sequence fragments contained in a supporting medium by size and charge under the influence of an applied electric field. Gel sheets or slabs, e.g. agarose, agarose-acrylamide or polyacrylamide, are typically used for nucleotide sizing gels. The electrophoresis conditions affect the desired degree of resolution of the fragments. A degree of resolution that separates fragments that differ in size from one another by as little as 10 nucleotides is usually sufficient. Preferably, the gels will be capable of resolving fragments which differ by 3 to 5 nucleotides. However, for some purposes (where the differences in sequence length are large), discrimination of sequence differences of at least 100 nt may be sufficiently sensitive for the analysis.</p>
    <p>Preparation and staining of analytical gels is well known. For example, a 3% Nusieve 1% agarose gel which is stained using ethidium bromide is described in Boerwinkle et al, PNAS, 86:212-216 (1989). Detection of DNA in polyacrylamide gels using silver stain is described in Goldman et al, Electrophoresis, 3:24-26 (1982); Marshall, Electrophoresis, 4:269-272 (1983); Tegelstrom, Electrophoresis, 7:226-229 (1987); and Allen et al, BioTechniques 7:736-744 (1989). The method described by Allen et al, using large-pore size ultrathin-layer, rehydratable polyacrylamide gels stained with silver is preferred. Each of those articles is incorporated herein by reference in its entirety.</p>
    <p>Size markers can be run on the same gel to permit estimation of the size of the restriction fragments. Comparison to one or more control sample(s) can be made in addition to or in place of the use of size markers. The size markers or control samples are usually run in one or both the lanes at the edge of the gel, and preferably, also in at least one central lane. In carrying out the electrophoresis, the DNA fragments are loaded onto one end of the gel slab (commonly called the "origin") and the fragments separate by electrically facilitated transport through the gel, with the shortest fragment electrophoresing from the origin towards the other (anode) end of the slab at the fastest rate. An agarose slab gel is typically electrophoresed using about 100 volts for 30 to 45 minutes. A polyacrylamide slab gel is typically electrophoresed using about 200 to 1,200 volts for 45 to 60 minutes.</p>
    <p>After electrophoresis, the gel is readied for visualization. The DNA fragments can be visualized by staining the gel with a nucleic acid-specific stain such as ethidium bromide or, preferably, with silver stain, which is not specific for DNA. Ethidium bromide staining is described in Boerwinkle et al, supra. Silver staining is described in Goldman et al, supra, Marshall, supra, Tegelstrom, supra, and Allen et al, supra.</p>
    <heading>Probes</heading> <p>Allele-specific oligonucleotides or probes are used to identify DNA sequences which have regions that hybridize with the probe sequence. The amplified DNA sequences defined by a locus-specific primer pair can be used as probes in RFLP analyses using genomic DNA. U.S. Pat. No. 4,582,788 (to Erlich, issued Apr. 15, 1986) describes an exemplary HLA typing method based on analysis of RFLP patterns produced by genomic DNA. The analysis uses cDNA probes to analyze separated DNA fragments in a Southern blot type of analysis. As stated in the patent "[C]omplementary DNA probes that are specific to one (locus-specific) or more (multilocus) particular HLA DNA sequences involved in the polymorphism are essential components of the hybridization step of the typing method" (col. 6, l. 3-7).</p>
    <p>The amplified DNA sequences of the present method can be used as probes in the method described in that patent or in the present method to detect the presence of an amplified DNA sequence of a particular allele. More specifically, an amplified DNA sequence having a known allele can be produced and used as a probe to detect the presence of the allele in sample DNA which is amplified by the present method.</p>
    <p>Preferably, however, when a probe is used to distinguish alleles in the amplified DNA sequences of the present invention, the probe has a relatively short sequence (in comparison to the length of the amplified DNA sequence) which minimizes the sequence homology of other alleles of the locus with the probe sequence. That is, the probes will correspond to a region of the amplified DNA sequence which has the largest number of nucleotide differences from the amplified DNA sequences of other alleles produced using that primer pair.</p>
    <p>The probes can be labelled with a detectable atom, radical or ligand using known labeling techniques. Radiolabels, usually <sup>32</sup> P, are typically used. The probes can be labeled with <sup>32</sup> P by nick translation with an α-<sup>32</sup> P-dNTP (Rigby et al, J. Mol. Biol., 113:237 (1977)) or other available procedures to make the locus-specific probes for use in the methods described in the patent. The probes are preferably labeled with an enzyme, such as hydrogen peroxidase. Coupling enzyme labels to nucleotide sequences are well known. Each of the above references is incorporated herein by reference in its entirety.</p>
    <p>The analysis method known as "Southern blotting" that is described by Southern, J. Mol. Biol., 98:503-517 (1975) is an analysis method that relies on the use of probes. In Southern blotting the DNA fragments are electrophoresed, transferred and affixed to a support that binds nucleic acid, and hybridized with an appropriately labeled cDNA probe. Labeled hybrids are detected by autoradiography, or preferably, use of enzyme labels.</p>
    <p>Reagents and conditions for blotting are described by Southern, supra; Wahl et al, PNAS 6:3683-3687 (1979); Kan et al, PNAS, supra, U.S. Pat. No. 4,302,204 and Molecular Cloning: A Laboratory Manual by Maniatis et al, Cold Spring Harbor Laboratory 1982. After the transfer is complete the paper is separated from the gel and is dried. Hybridization (annealing) of the resolved single stranded DNA on the paper to an probe is effected by incubating the paper with the probe under hybridizing conditions. See Southern, supra; Kan et al, PNAS, supra and U.S. Pat. No. 4,302,204, col 5, line 8 et seq. Complementary DNA probes specific for one allele, one locus (locus-specific) or more are essential components of the hybridization step of the typing method. Locus-specific probes can be made by the amplification method for locus-specific amplified sequences, described above. The probes are made detectable by labeling as described above.</p>
    <p>The final step in the Southern blotting method is identifying labeled hybrids on the paper (or gel in the solution hybridization embodiment). Autoradiography can be used to detect radiolabel-containing hybrids. Enzyme labels are detected by use of a color development system specific for the enzyme. In general, the enzyme cleaves a substrate, which cleavage either causes the substrate to develop or change color. The color can be visually perceptible in natural light or a fluorochrome which is excited by a known wavelength of light.</p>
    <heading>Sequencing</heading> <p>Genetic variations in amplified DNA sequences which reflect allelic difference in the sample DNA can also be detected by sequencing the amplified DNA sequences. Methods for sequencing oligonucleotide sequences are well known and are described in, for example, Molecular Cloning: A Laboratory Manual by Maniatis et al, Cold Spring Harbor Laboratory 1982. Currently, sequencing can be automated using a number of commercially available instruments.</p>
    <p>Due to the amount of time currently required to obtain sequencing information, other analysis methods, such as gel electrophoresis of the amplified DNA sequences or a restriction endonuclease digest thereof are preferred for clinical analyses.</p>
    <heading>Kits</heading> <p>As stated previously, the kits of this invention comprise one or more of the reagents used in the above described methods. In one embodiment, a kit comprises at least one genetic locus-specific primer pair in a suitable container. Preferably the kit contains two or more locus-specific primer pairs. In one embodiment, the primer pairs are for different loci and are in separate containers. In another embodiment, the primer pairs are specific for the same locus. In that embodiment, the primer pairs will preferably be in the same container when specific for different alleles of the same genetic locus and in different containers when specific for different portions of the same allele sequence. Sets of primer pairs which are used sequentially can be provided in separate containers in one kit. The primers of each pair can be in separate containers, particularly when one primer is used in each set of primer pairs. However, each pair is preferably provided at a concentration which facilitates use of the primers at the concentrations required for all amplifications in which it will be used.</p>
    <p>The primers can be provided in a small volume (e.g. 100 μl ) of a suitable solution such as sterile water or Tris buffer and can be frozen. Alternatively, the primers can be air dried.</p>
    <p>In another embodiment, a kit comprises, in separate containers, two or more endonucleases useful in the methods of this invention. The kit will preferably contain a locus-specific combination of endonucleases. The endonucleases can be provided in a suitable solution such as normal saline or physiologic buffer with 50% glycerol (at about -20° C.) to maintain enzymatic activity.</p>
    <p>The kit can contain one or more locus-specific primer pairs together with locus-specific combinations of endonucleases and may additionally include a control. The control can be an amplified DNA sequence defined by a locus-specific primer pair or DNA having a known HLA type for a locus of interest.</p>
    <p>Additional reagents such as amplification buffer, digestion buffer, a DNA polymerase and nucleotide triphosphates can be provided separately or in the kit. The kit may additionally contain gel preparation and staining reagents or preformed gels.</p>
    <p>Analyses of exemplary genetic loci are described below.</p>
    <heading>Analysis of HLA Type</heading> <p>The present method of analysis of genetic variation in an amplified DNA sequence to determine allelic difference in sample DNA can be used to determine HLA type. Primer pairs that specifically amplify genomic DNA associated with one HLA locus are described in detail hereinafter. In a preferred embodiment, the primers define a DNA sequence that contains all exons that encode allelic variability associated with the HLA locus together with at least a portion of one of the adjacent intron sequences. For Class I loci, the variable exons are the second and third exons. For Class II loci, the variable exon is the second exon. The primers are preferably located so that a substantial portion of the amplified sequence corresponds to intron sequences.</p>
    <p>The intron sequences provide restriction sites that, in comparison to cDNA sequences, provide additional information about the individual; e.g., the haplotype. Inclusion of exons within the amplified DNA sequences does not provide as many genetic variations that enable distinction between alleles as an intron sequence of the same length, particularly for constant exons. This additional intron sequence information is particularly valuable in paternity determinations and in forensic applications. It is also valuable in typing for transplant matching in that the variable lengths of intron sequences included in the amplified sequence produced by the primers enables a distinction to be made between certain heterozygotes (two different alleles) and homozygotes (two copies of one allele).</p>
    <p>Allelic differences in the DNA sequences of HLA loci are illustrated below. The tables illustrate the sequence homology of various alleles and indicate exemplary primer binding sites. Table 1 is an illustration of the alignment of the nucleotides of the Class I A2, A3, Ax, A24 (formerly referred to as A9), B27, B58 (formerly referred to as B17), C1, C2 and C3 allele sequences in intervening sequence (IVS) I and III. (The gene sequences and their numbering that are used in the tables and throughout the specification can be found in the Genbank and/or European Molecular Biology Laboratories (EMBL) sequence databanks. Those sequences are incorporated herein by reference in their entirety.) Underlined nucleotides represent the regions of the sequence to which exemplary locus-specific or Class I-specific primers bind.</p>
    <p>Table 2 illustrates the alignment of the nucleotides in IVS I and II of the DQA3 (now DQA1 0301), DQA1.2 (now DQA1 0102) and DQA4.1 (now DQA1 0501) alleles of the DQA1 locus (formerly referred to as the DR4, DR6 and DR3 alleles of the DQA1 locus, respectively). Underlined nucleotides represent the regions of the sequence to which exemplary DQA1 locus-specific primers bind.</p>
    <p>Table 3 illustrates the alignment of the nucleotides in IVS I, exon 2 and IVS II of two individuals having the DQw1<sub>V</sub> allele (designated hereinafter as DQw1<sub>V</sub> a and DQw1<sub>V</sub> b for the upper and lower sequences in the table, respectively), the DQw2 and DQw8 alleles of the DQB1 locus. Nucleotides indicated in the DQw1<sub>V</sub> b, DQw2 and DQw8 allele sequences are those which differ from the DQw1<sub>V</sub> a sequence. Exon 2 begins and ends at nt 599 and nt 870 of the DQw1<sub>V</sub> a allele sequence, respectively. Underlined nucleotides represent the regions of the sequence to which exemplary DQB1 locus-specific primers bind.</p>
    <p>Table 4 illustrates the alignment of the nucleotides in IVS I, exon 2 and IVS II of the DPB4.1, DPB9, New and DPw3 alleles of the DPB1 locus. Nucleotides indicated in the DPB9, New and DPw3 allele sequences are those which differ from the DPB4.1 sequence. Exon 2 begins and ends at nt 7644 and nt 7907 of the DPB4.1 allele sequence, respectively. Underlined nucleotides represent the regions of the sequence to which exemplary DPB1 locus-specific primers bind.</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">                                  TABLE 1__________________________________________________________________________Class I Seq ##STR1## ##STR2## ##STR3## ##STR4## ##STR5## ##STR6## ##STR7## ##STR8## ##STR9## ##STR10## ##STR11## ##STR12## ##STR13## ##STR14## ##STR15## ##STR16## ##STR17## ##STR18## ##STR19## ##STR20## ##STR21## ##STR22## ##STR23## ##STR24## ##STR25## ##STR26## ##STR27## ##STR28## ##STR29## ##STR30## ##STR31##__________________________________________________________________________</pre>
    
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">                                  TABLE 2__________________________________________________________________________DQA1 Seq ##STR32## ##STR33## ##STR34## ##STR35## ##STR36## ##STR37## ##STR38## ##STR39## ##STR40## ##STR41## ##STR42## ##STR43## ##STR44## ##STR45## ##STR46##__________________________________________________________________________</pre>
    
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">                                  TABLE 3__________________________________________________________________________DQB1Seq ##STR47## ##STR48## ##STR49## ##STR50## ##STR51## ##STR52## ##STR53## ##STR54## ##STR55## ##STR56## ##STR57## ##STR58## ##STR59## ##STR60## ##STR61## ##STR62## ##STR63## ##STR64## ##STR65## ##STR66## ##STR67## ##STR68## ##STR69## ##STR70## ##STR71## ##STR72##__________________________________________________________________________</pre>
    
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">                                  TABLE 4__________________________________________________________________________ ##STR73## ##STR74## ##STR75## ##STR76## ##STR77## ##STR78##DPB4.1 7934 CCCAGGGCAGCCCCGCGGGCCCGTGCCCAG__________________________________________________________________________</pre>
    
    <heading>Primers for HLA loci</heading> <p>Exemplary HLA locus-specific primers are listed below. Each of the primers hybridizes with at least about 15 consecutive nucleotides of the designated region of the allele sequence. The designation of an exemplary preferred primer together with its sequence is also shown. For many of the primers, the sequence is not identical for all of the other alleles of the locus. For each of the following preferred primers, additional preferred primers have sequences which correspond to the sequences of the homologous region of other alleles of the locus or to their complements.</p>
    <p>In one embodiment, Class I loci are amplified by using an A, B or C locus-specific primer together with a Class I locus-specific primer. The Class I primer preferably hybridizes with IVS III sequences (or their complements) or, more preferably, with IVS I sequences (or their complements). The term "Class I-specific primer", as used herein, means that the primer hybridizes with an allele sequence (or its complement) for at least two different Class I loci and does not hybridize with Class II locus allele sequences under the conditions used. Preferably, the Class I primer hybridizes with at least one allele of each of the A, B and C loci. More preferably, the Class I primer hybridizes with a plurality of, most preferably all of, the Class I allele loci or their complements. Exemplary Class I locus-specific primers are also listed below.</p>
    <heading>HLA Primers</heading> <p>A locus-specific primers</p>
    <p>allelic location: nt 1735-1757 of A3 designation: SGD009.AIVS3.R2NP sequence: CATGTGGCCATCTTGAGAATGGA</p>
    <p>allelic location: nt 1541-1564 of A2 designation: SGD006.AIVS3.R1NP sequence: GCCCGGGAGATCTACAGGCGATCA</p>
    <p>allelic location: nt 1533-1553 of A2 designation: A2.1 sequence: CGCCTCCCTGATCGCCTGTAG</p>
    <p>allelic location: nt 1667-1685 of A2 designation: A2.2 sequence: CCAGAGAGTGACTCTGAGG</p>
    <p>allelic location: nt 1704-1717 of A2 designation: A2.3 sequence: CACAATTAAGGGAT</p>
    <p>B locus-specific primers</p>
    <p>allelic location: nt 1108-1131 of B17 designation: SGD007.BIVS3.R1NP sequence: TCCCCGGCGACCTATAGGAGATGG</p>
    <p>allelic location: nt 1582-1604 of B17 designation: SGD010.BIVS3.R2NP sequence: CTAGGACCACCCATGTGACCAGC</p>
    <p>allelic location: nt 500-528 of B27 designation: B2.1 sequence: ATCTCCTCAGACGCCGAGATGCGTCAC</p>
    <p>allelic location: nt 545-566 of B27 designation: B2.2 sequence: CTCCTGCTGCTCTGGGGGGCAG</p>
    <p>allelic location: nt 1852-1876 of B27 designation: B2.3 sequence: ACTTTACCTCCACTCAGATCAGGAG</p>
    <p>allelic location: nt 1945-1976 of B27 designation: B2.4 sequence: CGTCCAGGCTGGTGTCTGGGTTCTGTGCCCCT</p>
    <p>allelic location: nt 2009-2031 of B27 designation: B2.5 sequence: CTGGTCACATGGGTGGTCCTAGG</p>
    <p>allelic location: nt 2054-2079 of B27 designation: B2.6 sequence: CGCCTGAATTTTCTGACTCTTCCCAT</p>
    <p>C locus-specific primers</p>
    <p>allelic location: nt 1182-1204 of C3 designation: SGD008.CIVS3.R1NP sequence: ATCCCGGGAGATCTACAGGAGATG</p>
    <p>allelic location: nt 1665-1687 of C3 designation: SGD011.CIVS3.R2NP sequence: AACAGCGCCCATGTGACCATCCT</p>
    <p>allelic location: nt 499-525 of C1 designation: C2.1 sequence: CTGGGGAGGCGCCGCGTTGAGGATTCT</p>
    <p>allelic location: nt 642-674 of C1 designation: C2.2 sequence: CGTCTCCGCAGTCCCGGTTCTAAAGTTCCCAGT</p>
    <p>allelic location: nt 738-755 of C1 designation: C2.3 sequence: ATCCTCGTGCTCTCGGGA</p>
    <p>allelic location: nt 1970-1987 of C1 designation: C2.4 sequence: TGTGGTCAGGCTGCTGAC</p>
    <p>allelic location: nt 2032-2051 of C1 designation: C2.5 sequence: AAGGTTTGATTCCAGCTT</p>
    <p>allelic location: nt 2180-2217 of C1 designation: C2.6 sequence: CCCCTTCCCCACCCCAGGTGTTCCTGTCCATTCTTCAGGA</p>
    <p>allelic location: nt 2222-2245 of C1 designation: C2.7 sequence: CACATGGGCGCTGTTGGAGTGTCG</p>
    <p>Class I loci-specific primers</p>
    <p>allelic location: nt 599-620 of A2 designation: SGD005.IIVS1.LNP sequence: GTGAGTGCGGGGTCGGGAGGGA</p>
    <p>allelic location: nt 489-506 of A2 designation: 1.1 sequence: CACCCACCGGGACTCAGA</p>
    <p>allelic location: nt 574-595 of A2 designation: 1.2 sequence: TGGCCCTGACCCAGACCTGGGC</p>
    <p>allelic location: nt 691-711 of A2 designation: 1.3 sequence: GAGGGTCGGGCGGGTCTCAGC</p>
    <p>allelic location: nt 1816-1831 of A2 designation: 1.4 sequence: CTCTCAGGCCTTGTTC</p>
    <p>allelic location: nt 1980-1923 of A2 designation: 1.5 sequence: CAGAAGTCGCTGTTCC</p>
    <p>DQA1 locus-specific primers</p>
    <p>allelic location: nt 23-41 of DQA3 designation: SGD001.DQA1.LNP sequence: TTCTGAGCCAGTCCTGAGA</p>
    <p>allelic location: nt 45-64 of DQA3 designation: DQA3 E1a sequence: TTGCCCTGACCACCGTGATG</p>
    <p>allelic location: nt 444-463 of DQA3 designation: DQA3 E1b sequence: CTTCCTGCTTGTCATCTTCA</p>
    <p>allelic location: nt 536-553 of DQA3 designation: DQA3 E1c sequence: CCATGAATTTGATGGAGA</p>
    <p>allelic location: nt 705-723 of DQA3 designation: DQA3 E1d sequence: ACCGCTGCTACCAATGGTA</p>
    <p>allelic location: nt 789-806 of DQA3 designation: SGD003.DQA1.RNP sequence: CCAAGAGGTCCCCAGATC</p>
    <p>DRA locus-specific primers allelic location: nt 49-68 of DRA HUMMHDRAM (1183 nt sequence, Accession No. K01171) designation: DRA E1 sequence: TCATCATAGCTGTGCTGATG</p>
    <p>allelic location: nt 98-118 of DRA HUMMHDRAM (1183 nt sequence, Accession No. K01171) designation: DRA 5'E2 (5' indicates the primer is used as the 5' primer) sequence: AGAACATGTGATCATCCAGGC</p>
    <p>allelic location: nt 319-341 of DRA HUMMHDRAM (1183 nt sequence, Accession No. K01171) designation: DRA 3'E2 sequence: CCAACTATACTCCGATCACCAAT</p>
    <p>DRB locus-specific primers</p>
    <p>allelic location: nt 79-101 of DRB HUMMHDRC (1153 nt sequence, Accession No. K01171) designation: DRB E1 sequence: TGACAGTGACACTGATGGTGCTG</p>
    <p>allelic location: nt 123-143 of DRB HUMMHDRC (1153 nt sequence, Accession No. K01171) designation: DRB 5'E2 sequence: GGGGACACCCGACCACGTTTC</p>
    <p>allelic location: nt 357-378 of DRB HUMMHDRC (1153 nt sequence, Accession No. K01171) designation: DRB 3'E2 sequence: TGCAGACACAACTACGGGGTTG</p>
    <p>DQB1 locus-specific primers</p>
    <p>allelic location: nt 509-532 DQB1 DQw1<sub>V</sub> a designation: DQB E1 sequence: TGGCTGAGGGCAGAGACTCTCCC</p>
    <p>allelic location: nt 628-647 of DQB1 DQw1<sub>V</sub> a designation: DQB 5'E2 sequence: TGCTACTTCACCAACGGGAC</p>
    <p>allelic location: nt 816-834 of DQB1 DQw1<sub>V</sub> a designation: DQB 3'E2 sequence: GGTGTGCACACACAACTAC</p>
    <p>allelic location: nt 124-152 of DQB1 DQw1<sub>V</sub> a designation: DQB 5'IVS1a sequence: AGGTATTTTACCCAGGGACCAAGAGAT</p>
    <p>allelic location: nt 314-340 of DQB1 DQw1<sub>V</sub> a designation: DQB 5'IVS1b sequence: ATGTAAAATCAGCCCGACTGCCTCTTC</p>
    <p>allelic location: nt 1140-1166 of DQB1 DQw1<sub>V</sub> a designation: DQB 3'IVS2 sequence: GCCTCGTGCCTTATGCGTTTGCCTCCT</p>
    <p>DPB1 locus-specific primers</p>
    <p>allelic location: nt 6116-6136 of DPB14.1 designation: DPB E1 sequence: TGAGGTTAATAAACTGGAGAA</p>
    <p>allelic location: nt 7604-7624 of DPB14.1 designation: DPB 5'IVS1 sequence: GAGAGTGGCGCCTCCGCTCAT</p>
    <p>allelic location: nt 7910-7929 of DPB14.1 designation: DPB 3'IVS2 sequence: GAGTGAGGGCTTTGGGCCGG</p>
    <heading>Primer pairs for HLA analyses</heading> <p>It is well understood that for each primer pair, the 5' upstream primer hybridizes with the 5' end of the sequence to be amplified and the 3' downstream primer hybridizes with the complement of the 3' end of the sequence. The primers amplify a sequence between the regions of the DNA to which the primers bind and its complementary sequence including the regions to which the primers bind. Therefore, for each of the primers described above, whether the primer binds to the HLA-encoding strand or its complement depends on whether the primer functions as the 5' upstream primer or the 3' downstream primer for that particular primer pair.</p>
    <p>In one embodiment, a Class I locus-specific primer pair includes a Class I locus-specific primer and an A, B or C locus-specific primer. Preferably, the Class I locus-specific primer is the 5' upstream primer and hybridizes with a portion of the complement of IVS I. In that case, the locus-specific primer is preferably the 3' downstream primer and hybridizes with IVS III. The primer pairs amplify a sequence of about 1.0 to about 1.5 Kb.</p>
    <p>In another embodiment, the primer pair comprises two locus-specific primers that amplify a DNA sequence that does not include the variable exon(s). In one example of that embodiment, the 3' downstream primer and the 5' upstream primer are Class I locus-specific primers that hybridize with IVS III and its complement, respectively. In that case a sequence of about 0.5 Kb corresponding to the intron sequence is amplified.</p>
    <p>Preferably, locus-specific primers for the particular locus, rather than for the HLA class, are used for each primer of the primer pair. Due to differences in the Class II gene sequences, locus-specific primers which are specific for only one locus participate in amplifying the DRB,-DQA1, DQB and DPB loci. Therefore, for each of the preferred Class II locus primer pairs, each primer of the pair participates in amplifying only the designated locus and no other Class II loci.</p>
    <heading>Analytical methods</heading> <p>In one embodiment, the amplified sequence includes sufficient intron sequences to encompass length polymorphisms. The primer-defined length polymorphisms (PDLPs) are indicative of the HLA locus allele in the sample. For some HLA loci, use of a single primer pair produces primer-defined length polymorphisms that distinguish between some of the alleles of the locus. For other loci, two or more pairs of primers are used in separate amplifications to distinguish the alleles. For other loci, the amplified DNA sequence is cleaved with one or more restriction endonucleases to distinguish the alleles. The primer-defined length polymorphisms are particularly useful in screening processes.</p>
    <p>In anther embodiment, the invention provides an improved method that uses PCR amplification of a genomic HLA DNA sequence of one HLA locus. Following amplification, the amplified DNA sequence is combined with at least one endonuclease to produce a digest. The endonuclease cleaves the amplified DNA sequence to yield a set of fragments having distinctive fragment lengths. Usually the amplified sequence is divided, and two or more endonuclease digests are produced. The digests can be used, either separately or combined, to produce RFLP patterns that can distinguish between individuals. Additional digests can be prepared to provide enhanced specificity to distinguish between even closely related individuals with the same HLA type.</p>
    <p>In a preferred embodiment, the presence of a particular allele can be verified by performing a two step amplification procedure in which an amplified sequence produced by a first primer pair is amplified by a second primer pair which binds to and defines a sequence within the first amplified sequence. The first primer pair can be specific for one or more alleles of the HLA locus. The second primer pair is preferably specific for one allele of the HLA locus, rather than a plurality of alleles. The presence of an amplified sequence indicates the presence of the allele, which is confirmed by production of characteristic RFLP patterns.</p>
    <p>To analyze RFLP patterns, fragments in the digest are separated by size and then visualized. In the case of typing for a particular HLA locus, the analysis is directed to detecting the two DNA allele sequences that uniquely characterize that locus in each individual. Usually this is performed by comparing the sample digest RFLP patterns to a pattern produced by a control sample of known HLA allele type. However, when the method is used for paternity testing or forensics, the analysis need not involve identifying a particular locus or loci but can be done by comparing single or multiple RFLP patterns of one individual with that of another individual using the same restriction endonuclease and primers to determine similarities and differences between the patterns.</p>
    <p>The number of digests that need to be prepared for any particular analysis will depend on the desired information and the particular sample to be analyzed. For example, one digest may be sufficient to determine that an individual cannot be the person whose blood was found at a crime scene. In general, the use of two to three digests for each of two to three HLA loci will be sufficient for matching applications (forensics, paternity). For complete HLA haplotyping; e.g., for transplantation, additional loci may need to be analyzed.</p>
    <p>As described previously, combinations of primer pairs can be used in the amplification method to amplify a particular HLA DNA locus irrespective of the allele present in the sample. In a preferred embodiment, samples of HLA DNA are divided into aliquots containing similar amounts of DNA per aliquot and are amplified with primer pairs (or combinations of primer pairs) to produce amplified DNA sequences for additional HLA loci. Each amplification mixture contains only primer pairs for one HLA locus. The amplified sequences are preferably processed concurrently, so that a number of digest RFLP fragment patterns can be produced from one sample. In this way, the HLA type for a number of alleles can be determined simultaneously.</p>
    <p>Alternatively, preparation of a number of RFLP fragment patterns provides additional comparisons of patterns to distinguish samples for forensic and paternity analyses where analysis of one locus frequently fails to provide sufficient information for the determination when the sample DNA has the same allele as the DNA to which it is compared.</p>
    <p>The use of HLA types in paternity tests or transplantation testing and in disease diagnosis and prognosis is described in Basic &amp; Clinical Immunology, 3rd Ed (1980) Lange Medical Publications, pp 187-190, which is incorporated herein by reference in its entirety. HLA determinations fall into two general categories. The first involves matching of DNA from an individual and a sample. This category involves forensic determinations and paternity testing. For category 1 analysis, the particular HLA type is not as important as whether the DNA from the individuals is related. The second category is in tissue typing such as for use in transplantation. In this case, rejection of the donated blood or tissue will depend on whether the recipient and the donor express the same or different antigens. This is in contrast to first category analyses where differences in the HLA DNA in either the introns or exons is determinative.</p>
    <p>For forensic applications, analysis of the sample DNA of the suspected perpetrator of the crime and DNA found at the crime scene are analyzed concurrently and compared to determine whether the DNA is from the same individual. The determination preferably includes analysis of at least three digests of amplified DNA of the DQA1 locus and preferably also of the A locus. More preferably, the determination also includes analysis of at least three digests of amplified DNA of an additional locus, e.g. the DPB locus. In this way, the probability that differences between the DNA samples can be discriminated is sufficient.</p>
    <p>For paternity testing, the analysis involves comparison of DNA of the child, the mother and the putative father to determine the probability that the child inherited the obligate haplotype DNA from the putative father. That is, any DNA sequence in the child that is not present in the mother's DNA must be consistent with being provided by the putative father. Analysis of two to three digests for the DQA1 and preferably also for the A locus is usually sufficient. More preferably, the determination also includes analysis of digests of an additional locus, e.g. the DPB locus.</p>
    <p>For tissue typing determinations for transplantation matching, analysis of three loci (HLA A, B, and DR) is often sufficient-. Preferably, the final analysis involves comparison of additional loci including DQ and DP.</p>
    <heading>Production of RFLP fragment patterns</heading> <p>The following table of exemplary fragment pattern lengths demonstrates distinctive patterns. For example, as shown in the table, BsrI cleaves A2, A3 and A9 allele amplified sequences defined by primers SGD005.IIVS1.LNP and SGD009.AIVS3.R2NP into sets of fragments with the following numbers of nucleotides (740, 691), (809, 335, 283) and (619, 462, 256, 93), respectively. The fragment patterns clearly indicate which of the three A alleles is present. The following table illustrates a number of exemplary endonucleases that produce distinctive RFLP fragment patterns for exemplary A allele sequences.</p>
    <p>Table 2 illustrates the set of RFLP fragments produced by use of the designated endonucleases for analysis of three A locus alleles. For each endonuclease, the number of nucleotides of each of the fragments in a set produced by the endonuclease is listed. The first portion of the table illustrates RFLP fragment lengths using the primers designated SGD009.AIVS3.R2NP and SGD005.IIVS1.LNP which produce the longer of the two exemplary sequences. The second portion of the table illustrates RFLP fragment lengths using the primers designated SGD006.AIVS3.R1NP and SGD005.IIVS1.LNP which produce the shorter of the sequences. The third portion of the table illustrates the lengths of fragments of a DQA1 locus-specific amplified sequence defined by the primers designated SGD001.DQA1.LNP and SGD003.DQA1.RNP.</p>
    <p>As shown in the Table, each of the endonucleases produces a characteristic RFLP fragment pattern which can readily distinguish which of the three A alleles is present in a sample.</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">                                  TABLE 5__________________________________________________________________________RFLP FRAGMENT PATTERNS__________________________________________________________________________A -- LongBsrI   A2     740     691  A3  809                      335     283  A9                619     462           256       93Cfr101 A2  1055          399     245  A3             473                    399     247  A9     786        399DraII  A2  698                      251        138  A3                369     315                               251     247  A9     596     427           251               80FokI   A2     728        248                151  A3             515        225                               213     151  A9  1004                             151GsuI   A2     868        547                   36  A3  904                   523  A9             638           419     373HphI   A2  1040                     239               72  A3             419                    375                218                                          163  A9     643     419                    373MboII  A2     1011               165                               143     132  A3             893                    194        143        115  A9  1349                                       51PpumI  A2  698                      295     251       138  A3                369     364        251                                          242  A9     676     503                   251PssI   A2  695                      295     251       138  A3                366     315        251                                          242  A9     596     427                   251A -- ShortBsrI   A2  691                      254  A3             345                    335     283  A9     619                   256     93Cfr101 A2  A3  A9DraII  A2             295                    251     210                               138  A3     315        251     210  A9  427           251     210FokI   A2     293     248           151     143                                          129    51  A3                225     213                               151     143                                          129    51  A9  539                      151     146                                          129GsuI   A2     868                   61      36  A3  904                      59  A9             414                    373     178HphI   A2  554           339  A3     411     375        177  A9     414     373        178MboII  A2  A3  A9PpumI  A2             295                    257        212        69  A3     364                251                               210     72 66  A9  503                   251                               211PssI   A2             295                    251     219        72  A3     315        251        207     72 66  A9  427           251        208     72__________________________________________________________________________DQA1AluIDQA3     449   335DQA4.1      338          332               122DQA1.2         335             287            123             52CvijIDQA3                 271      187                            122   99      64  34        7DQA4.1               277                   219         102   79     55                                              36        17                                                        7DQA1.2                     201      101                                  99 80 76  55                                              36                                                35      7DdeIDQA3  587                                 88   65    30      11                                                        3DQA4.1      388               194         89   64  36        11                                                        3DQA1.2      395               165         88   65                                            41                                              36        11                                                        3MboIIDQA3        366               184                            172           62DQA4.1   407       353                                    32DQA1.2      330          316                        89       32                                                30MnlIDQA3                    214   176                            172         72  43                                              36                                                23      21                                                        17                                                        10DQA4.1            294         179      149       40                                              36                                                33      21DQA1.2                  216         136                                  123                                     73 54                                          44                                            40                                              36                                                24      21                                                        15                                                        10                                                        5NlaIIIDQA3     458         266                     60DQA4.1            300                263                   229DQA1.2                  223                      190         124                                     116                                        75    39                                                30TthIIIIIDQA3     417            226         141DQA4.1 426   371DQA1.2 428                        148                               141      75__________________________________________________________________________</pre>
    
    <heading>Screening Analysis for Genetic Disease</heading> <p>Carriers of genetic diseases and those affected by the disease can be identified by use of the present method. Depending on the disease, the screening analysis can be used to detect the presence of one or more alleles associated with the disease or the presence of haplotypes associated with the disease. Furthermore, by analyzing haplotypes, the method can detect genetic diseases that are not associated with coding region variations but are found in regulatory or other untranslated regions of the genetic locus. The screening method is exemplified below by analysis of cystic fibrosis (CF).</p>
    <p>Cystic fibrosis is an autosomal recessive disease, requiring the presence of a mutant gene on each chromosome. CF is the most common genetic disease in Caucasians, occurring once in 2,000 live births. It is estimated that one in forty Caucasians are carriers for the disease.</p>
    <p>Recently a specific deletion of three adjacent basepairs in the open reading frame of the putative CF gene leading to the loss of a phenylalanine residue at position 508 of the predicted 1480 amino acid polypeptide was reported [Kerem et al, Science 245:1073-1080 (1989)]. Based on haplotype analysis, the deletion may account for most CF mutations in Northern European populations (about 68%). A second mutation is reportedly prevalent in some Southern European populations. Additional data indicate that several other mutations may cause the disease.</p>
    <p>Studies of haplotypes of parents of CF patients (who necessarily have one normal and one disease-associated haplotype) indicated that there are at least 178 haplotypes associated with the CF locus. Of those haplotypes, 90 are associated only with the disease; 78 are found only in normals; and 10 are associated with both the disease and with normals (Kerem et al, supra). The disease apparently is caused by several different mutations, some in very low frequency in the population. As demonstrated by the haplotype information, there are more haplotypes associated with the locus than there are mutant alleles responsible for the disease.</p>
    <p>A genetic screening program (based on amplification of exon regions and analysis of the resultant amplified DNA sequence with probes specific for each of the mutations or with enzymes producing RFLP patterns characteristic of each mutation) may take years to develop. Such tests would depend on detection and characterization of each of the mutations, or at least of mutations causing about 90 to 95% or more of the cases of the disease. The alternative is to detect only 70 to 80% of the CF-associated genes. That alternative is generally considered unacceptable and is the cause of much concern in the scientific community.</p>
    <p>The present method directly determines haplotypes associated with the locus and can detect haplotypes among the 178 currently recognized haplotypes associated with the disease locus. Additional haplotypes associated with the disease are readily determined through the rapid analysis of DNA of numerous CF patients by the methods of this invention. Furthermore, any mutations which may be associated with noncoding regulatory regions can also be detected by the method and will be identified by the screening process.</p>
    <p>Rather than attempting to determine and then detect each defect in a coding region that causes the disease, the present method amplifies intron sequences associated with the locus to determine allelic and sub-allelic patterns. In contrast to use of mutation-specific probes where only known sequence defects can be detected, new PDLP and RFLP patterns produced by intron sequences indicate the presence of a previously unrecognized haplotype.</p>
    <p>The same analysis can be performed for phenylalanine hydroxylase locus nutations that cause phenylketonuria and for beta-globin mutations that cause beta-thalassemia and sickle cell disease and for other loci known to be associated with a genetic disease. Furthermore, neither the mutation site nor the location for a disease gene is required to determine haplotypes associated with the disease. Amplified intron sequences in the regions of closely flanking RFLP markers, such as are known for Huntington's disease and many other inherited diseases, can provide sufficient information to screen for haplotypes associated with the disease.</p>
    <p>Muscular dystrophy (MD) is a sex-linked disease. The disease-associated gene comprises a 2.3 million basepair sequence that encodes 3,685 amino acid protein, dystrophin. A map of mutations for 128 of 34 patients with Becker's muscular dystrophy and 160 patients with Duchenne muscular dystrophy identified 115 deletions and 13 duplications in the coding region sequence [Den Dunnen et al, Am. J. Hum. Genet. 45:835-847 (1989)]. Although the disease is associated with a large number of mutations that vary widely, the mutations have a non-random distribution in the sequence and are localized to two major mutation hot spots, Den Dunnen et al, supra. Further, a recombination hot spot within the gene sequence has been identified [Grimm et al, Am. J. Hum. Genet. 45:368-372 (1989)].</p>
    <p>For analysis of MD, haplotypes on each side of the recombination hot spot are preferably determined. Primer pairs defining amplified DNA sequences are preferably located near, within about 1 to 10 Kbp of the hot spot on either side of the hot spot. In addition, due to the large size of the gene, primer pairs defining amplified DNA sequences are preferably located near each end of the gene sequence and most preferably also in an intermediate location on each side of the hot spot. In this way, haplotypes associated with the disease can be identified.</p>
    <p>Other diseases, particularly malignancies, have been shown to be the result of an inherited recessive gene together with a somatic mutation of the normal gene. One malignancy that is due to such "loss of heterogeneity" is retinoblastoma, a childhood cancer. The loss of the normal gene through mutation has been demonstrated by detection of the presence of one mutation in all somatic cells (indicating germ cell origin) and detection of a second mutation in some somatic cells [Scheffer et al, Am. J. Hum. Genet. 45:252-260 (1989)]. The disease can be detected by amplifying somatic cell, genomic DNA sequences that encompass sufficient intron sequence nucleotides. The amplified DNA sequences preferably encompass intron sequences locate near one or more of the markers described by Scheffer et al, supra. Preferably, an amplified DNA sequence located near an intragenic marker and an amplified DNA sequence located near a flanking marker are used.</p>
    <p>An exemplary analysis for CF is described in detail in the examples. Analysis of genetic loci for other monogenic and multigenic genetic diseases can be performed in a similar manner.</p>
    <p>As the foregoing description indicates, the present method of analysis of intron sequences is generally applicable to detection of any type of genetic trait. Other monogenic and multigenic traits can be readily analyzed by the methods of the present invention. Furthermore, the analysis methods of the present method are applicable to all eukaryotic cells, and are preferably used on those of plants and animals. Examples of analysis of BoLA (bovine MHC determinants) further demonstrates the general applicability of the methods of this invention.</p>
    <p>This invention is further illustrated by the following specific but non-limiting examples. Procedures that are constructively reduced to practice are described in the present tense, and procedures that have been carried out in the laboratory are set forth in the past tense.</p>
    <heading>EXAMPLE 1</heading> <heading>Forensic Testing</heading> <p>DNA extracted from peripheral blood of the suspected perpetrator of a crime and DNA from blood found at the crime scene are analyzed to determine whether the two samples of DNA are from the same individual or from different individuals.</p>
    <p>The extracted DNA from each sample is used to form two replicate aliquots per sample, each aliquot having 1 μg of sample DNA. Each replicate is combined in a total volume of 100 μl with a primer pair (1 μg of each primer), dNTPs (2.5 mM each) and 2.5 units of Taq polymerase in amplification buffer (50 mM KCl; 10 mM Tris-HCl, pH 8.0; 2.5 mM MgCl<sub>2</sub> ; 100 μg/ml gelatin) to form four amplification reaction mixtures. The first primer pair contains the primers designated SGD005.IIVS1.LNP and SGD009.AIVS3.R2NP (A locus-specific). The second primer pair contains the primers designated SGD001.DQA1.LNP and SGD003.DQA1.RNP (DQA locus-specific). Each primer is synthesized using an Applied Biosystems model 308A DNA synthesizer. The amplification reaction mixtures are designated SA (suspect's DNA, A locus-specific primers), SD (suspect's DNA, DQA1 locus-specific primers), CA (crime scene DNA, A locus-specific primers) and CD (crime scene DNA, DQA1 locus-specific primers).</p>
    <p>Each amplification reaction mixture is heated to 94° C. for 30 seconds. The primers are annealed to the sample DNA by cooling the reaction mixtures to 65° C. for each of the A locus-specific amplification mixtures and to 55° C. for each of the DQA1 locus-specific amplification mixtures and maintaining the respective temperatures for one minute. The primer extension step is performed by heating each of the amplification mixtures to 72° C. for one minute. The denaturation, annealing and extension cycle is repeated 30 times for each amplification mixture.</p>
    <p>Each amplification mixture is aliquoted to prepare three restriction endonuclease digestion mixtures per amplification mixture. The A locus reaction mixtures are combined with the endonucleases BsrI, Cfr101 and DraII. The DQA1 reaction mixtures are combined with AluI, CvijI and DdeI.</p>
    <p>To produce each digestion mixture, each of three replicate aliquots of 10 μl of each amplification mixture is combined with 5 units of the respective enzyme for 60 minutes at 37° C. under conditions recommended by the manufacturer of each endonuclease.</p>
    <p>Following digestion, the three digestion mixtures for each of the samples (SA, SD, CA and CD) are pooled and electrophoresed on a 6.5% polyacrylamide gel for 45 minutes at 100 volts. Following electrophoresis, the gel is stained with ethidium bromide.</p>
    <p>The samples contain fragments of the following lengths:</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="equation">SA: 786, 619, 596, 462, 427, 399, 256, 251, 93, 80 CA: 809, 786, 619, 596, 473, 462, 427, 399, 369, 335, 315, 283, 256, 251, 247, 93, 80</pre>
    
    <p>
      </p> <pre xml:space="preserve" listing-type="equation">SD: 388, 338, 332, 277, 219, 194, 122, 102, 89, 79, 64, 55 CD: 587, 449, 388, 338, 335, 332, 277, 271, 219, 194, 187, 122, 102, 99, 89, 88, 79, 65, 64, 55</pre>
    
    <p>The analysis demonstrates that the blood from the crime scene and from the suspected perpetrator are not from the same individual. The blood from the crime scene and from the suspected perpetrator are, respectively, A3, A9, DQA1 0501, DQA1 0301 and A9, A9, DQA1 0501, DQA1 0501.</p>
    <heading>EXAMPLE 2</heading> <heading>Paternity Testing</heading> <p>Chorionic villus tissue was obtained by transcervical biopsy from a 7-week old conceptus (fetus). Blood samples were obtained by venepuncture from the mother (M), and from the alleged father (AF). DNA was extracted from the chorionic villus biopsy, and from the blood samples. DNA was extracted from the sample from M by use of nonionic detergent (Tween 20) and proteinase K. DNA was extracted from the sample from F by hypotonic lyuis. More specifically, 100 μl of blood was diluted to 1.5 ml in PBS and centrifuged to remove buffy coat. Following two hypotonic lysis treatments involving resuspension of buffy coat cells in water, the pellets were washed until redness disappeared. Colorless pellets were resuspended in water and boiled for 20 minutes. Five 10 mm chorionic villus fronds were received. One frond was immersed in 200 μl water. NaOH was added to 0.05 M. The sample was boiled for 20 minutes and then neutralized with HCl. No further purification was performed for any of the samples.</p>
    <p>The extracted DNA was submitted to PCR for amplification of sequences associated with the HLA loci, DQA1 and DPB1. The primers used were: (1) as a 5' primer for the DQA1 locus, the primer designated SGD001.DQA1.LNP (DQA 5'IVS1) (corresponding to nt 23-39 of the DQA1 0301 allele sequence) and as the 3' primer for the DQA1 locus, the primer designated SGD003.DQA1.RNP (DQA 3'IVS2 corresponding to nt 789-806 of the DQA1 0301 sequence; (2) as the DPB primers, the primers designated 5'IVS1 nt 7604-7624 and 3'IVS2 7910-7929. The amplification reaction mixtures were: 150 ng of each primer; 25 μ of test DNA; 10 mM Tris HCl, pH 8.3; 50 mM KCl; 1.5 mM MgCl<sub>2</sub> ; 0.01% (w/v) gelatin; 200 μM dNTPs; water to 100 μl and 2.5 U Taq polymerase.</p>
    <p>The amplification was performed by heating the amplification reaction mixture to 94° C. for 10 minutes prior to addition of Taq polymerase. For DQA1, the amplification was performed at 94° C. for 30 seconds, then 55° C. for 30 seconds, then 72° C. for 1 minute for 30 cycles, finishing with 72° C. for 10 minutes. For DPB, the amplification was performed at 96° C. for 30 seconds, then 65° C. for 30 seconds, finishing with 65° C. for 10 minutes.</p>
    <p>Amplification was shown to be technically satisfactory by test gel electrophoresis which demonstrated the presence of double stranded DNA of the anticipated size in the amplification reaction mixture. The test gel was 2% agarose in TBE (tris borate EDTA) buffer, loaded with 15 μl of the amplification reaction mixture per lane and electrophoresed at 200 v for about 2 hours until the tracker dye migrated between 6 to 7 cm into the 10 cm gel.</p>
    <p>The amplified DQA1 and DPB1 sequences were subjected to restriction endonuclease digestion using DdeI and MboII (8 and 12 units, respectively at 37° C. for 3 hours) for DQA1, and RsaI and FokI (8 and 11 units, respectively at 37° C. overnight) for DPB1 in 0.5 to 2.0 μl of enzyme buffers recommended by the supplier, Pharmacia together with 16-18 μl of the amplified product. The digested DNA was fragment size-length separated on gel electrophoresis (3% Nusieve). The RFLP patterns were examined under ultraviolet light after staining the gel with ethidium bromide.</p>
    <p>Fragment pattern analysis is performed by allele assignment of the non-maternal alleles using expected fragment sizes based on the sequences of known endonuclease restriction sites. The fragment pattern analysis revealed the obligate paternal DQA1 allele to be DQA1 0102 and DPB to be DPwl. The fragment patterns were consistent with AF being the biological father.</p>
    <p>To calculate the probability of true paternity, HLA types were assigned. Maternal and AF DQA1 types were consistent with those predicted from the HLA Class II gene types determined by serological testing using lymphocytotoxic antisera.</p>
    <p>Considering alleles of the two HLA loci as being in linkage equilibrium, the combined probability of non-paternity was given by:</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="equation">0.042×0.314-0.013</pre>
    
    <p>i.e. the probability of paternity is (1-0.013) or 98.7%.</p>
    <p>The relative chance of paternity is thus 74:75, i.e. the chance that the AF is not the biological father is approximately 1 in 75. The parties to the dispute chose to regard these results as confirming the paternity of the fetus by the alleged father.</p>
    <heading>EXAMPLE 3</heading> <heading>Analysis of the HLA DQA1 Locus</heading> <p>The three haplotypes of the HLA DQA1 0102 locus were analyzed as described below. Those haplotypes are DQA1 0102 DR15 Dw2; DQA1 0102 DR16 Dw21; and DQA1 0102 DR13 Dw19. The distinction between the haplotypes is particularly difficult because there is a one basepair difference between the 0102 alleles and the 0101 and 0103 alleles, which difference is not unique in DQA1 allele sequences.</p>
    <p>The procedure used for the amplification is the same as that described in Example 1, except that the amplification used thirty cycles of 94° C. for 30 seconds, 60° C. for 30 seconds, and 72° C. for 60 seconds. The sequences of the primers were:</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">______________________________________SGD 001 -- 5' TTCTGAGCCAGTCCTGAGA 3'; andSGD 003 -- 5' GATCTGGGGACCTCTTGG 3'.______________________________________</pre>
    
    <p>These primers hybridize to sequences about 500 bp upstream from the 5' end of the second exon and 50 bp downstream from the second exon and produce amplified DNA sequences in the 700 to 800 bp range.</p>
    <p>Following amplification, the amplified DNA sequences were electrophoresed on a 4% polyacrylamide gel to determine the PDLP type. In this case, amplified DNA sequences for 0102 comigrate with (are the same length as) 0101 alleles and subsequent enzyme digestion is necessary to distinguish them.</p>
    <p>The amplified DNA sequences were digested using the restriction enzyme AluI (Bethesda Research Laboratories) which cleaves DNA at the sequence AGCT. The digestion was performed by mixing 5 units (1 μl) of enzyme with 10 μl of the amplified DNA sequence (between about 0.5 and 1 μg of DNA) in the enzyme buffer provided by the manufacturer according to the manufacturer's directions to form a digest. The digest was then incubated for 2 hours at 37° C. for complete enzymatic digestion.</p>
    <p>The products of the digestion reaction are mixed with approximately 0.1 μg of "ladder" nucleotide sequences (nucleotide control sequences beginning at 123 bp in length and increasing in length by 123 bp to a final size of about 5,000 bp; available commercially from Bethesda Research Laboratories, Bethesda Md.) and were electrophoresed using a 4% horizontal ultra-thin polyacrylamide gel (E-C Apparatus, Clearwater Fla.). The bands in the gel were visualized (stained) using silver stain technique [Allen et al, BioTechniques 7:736-744 (1989)].</p>
    <p>Three distinctive fragment patterns which correspond to the three haplotypes were produced using AluI. The patterns (in base pair sized fragments) were:</p>
    <p>1. DR15 DQ6 Dw2: 120, 350, 370, 480</p>
    <p>2. DR13 DQ6 Dw19: 120, 330, 350, 480</p>
    <p>3. DR16 DQ6 Dw21: 120, 330, 350</p>
    <p>The procedure was repeated using a 6.5% vertical polyacrylamide gel and ethidium bromide stain and provided the same results. However, the fragment patterns were more readily distinguishable using the ultrathin gels and silver stain.</p>
    <p>This exemplifies analysis according to the method of this invention. Using the same procedure, 20 of the other 32 DR/DQ haplotypes for DQA1 were identified using the same primer pair and two additional enzymes (DdeI and MboII). PDLP groups and fragment patterns for each of the DQA1 haplotypes with the three endonucleases are illustrated in Table 6.</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">  - AluI  PDLP DR Dw 480 410 405 400 390 370 360 350 340 330 310 300 270 240 120 110 100  0101  1  1 | | | | | | | | | | + | + | | + |  1 20 | | | | | | | | | | + | + | | + |   14  9 | | | | | | | | | | + | + | | + |  0102 15  2 + | | | | + | + | | | | | | + | |   16 21 | | | | | |  | + | + | | | | + | |   13 19 + | | | | | | + | + | | | | + | |  0103 13 18,24 | | | | | | | | | | | | | | | | |     13 18,25 | | | | | | | | | | + | + | | + |    8  8.3 | | | | | | + | + | | | | | + | +   11 DB2 | | | | | .vertline . | | | | | | | | | | |   15 12 | | | | | |  | | | | | | | | | | |  0201  7 DB1 | | | + | | | | | | | + | | | | |    7 17 | | | + | | | | | | | + | | | | |    7 11 | | | | + | | | + | | | | | | | |  0301  4  4(7) | | | | | | | | | | | | | | | | |  4 13.2(7) | | | | | | | | | | | | | | | | |      4  4(8) | | | + | | |  + | | | | | | | | |    4 10(8) | | | + | | | + | | | | | | | | |    4 13.1(8) | | | | | | | | | | | | | | | | |  4 14(8) | | | + | | | + | | | | | | | | |  4 KT2 | | | | | | | | | | | | | | | | |  4 15 | | | | | | | | | | | | | | | | |    9 23 | | + | | | + | | | | | | | | | |  0401  8  8.1 | (+) | | | | | | | | + | | + + + |    8  8.2 | (+) | | | | | | | | + | | + + + |  3 RSH | | | | | | | | | | | | | | | | |  0501  3  3,24 | (+) | | | | | | | | + | | | | + |    3  3,25 | (+) | | | | | | | | + | | | | + |   11  5 | (+) | | | | | | | | + | | | | + |   11  5(9104) | | + | | | | | | | + | | | | + |   14 16 | (+) | | | | | | | | + | | | | + |   12 DB6 | (+) | | | | | | | | + | | | | + |   16 22 | | | | | |  | | | | | | | | | | |  0601  8  8.3 | (+) | | | | | | | | + | | | | + |  PDLP DR Dw 480 410 405 400 390 370 360 350 340 330 310 300 270 240 120 110 100  DdeI  PDLP DR Dw 650 520 450 420 410 400 390 300 200 190 150 90 80 60 50  0101  1  1 | | | + | | | | | | + | + + |1 20 | | | + | | | | | | + | + + |    14  9 | | | + | | | | | | + | + + |   0102 15  2 | + | + | | | | | | + | + + |   16 21 | | | + | | | | | | + | + + |    13 19 | | | | + | | | | | + | + + |   0103 13 18,24 | | | | | | | | | | | | | | |   13 18,25 | | | | | | | | | | | | | | |    8  8.3 | | + | | | | | | | + | + | +    11 DB2 | | | | | | | | | | | | | | |   15 12 | | | | | |  | | | | | | | | |  0201  7 DB1 | | | | | + | | | | + | + | +  7 17 | | | | | | | | | | + | + | +7 11 | | | | | + | | | | + | + | +   0301  4  4(7) | | | | | | | | | | | | | | |    4 13.2(7) | | | | | | | | | | | | | | |    4  4(8) + | | | | | | | | | | + | | +    4 10(8) + | | | | | | | | | | + | | +    4 13.1(8) | | | | | | | | | | | | | | |    4 14(8) + | | | | | | | | | | + | | +  4 KT2 | | | | | | | | | | | | | | |  4 15 | | | | | | | | | | | | | | |    9 23 + | | | | | | | | | | + | | +  0401  8  8.1 | | | | | | | + + | | + + | +       8  8.2 | | | | | | | + + | | + + | +       3 RSH | | | | | | | | | | | | | | |  0501  3  3,24 | | | | | + | | + | | | + | +3  3,25 | | | | | + | | + | | | + | +    11  5 | | | | | + | | + | | | + | +    11  5(9104) | | | | | + | | | + | | + | +    14 16 | | | | | .vertline . + | + | | | + | +      12 DB6 | | | | | + | | + | | | + | +   16 22 | | | | | |  | | | | | | | | |  0601  8  8.3 | | | | | | | + + | | + + | +  PDLP DR Dw 650 520 450 420 410 400 390 300 200 190 150 90 80 60 50  MboII  PDLP DR Dw 390 385 380 370 365 360 350 340 335 330 305 300 250 190 180 170 140 130  0101  1  1 | | | | | | + | | | | + | | | | | |  1 20 | | | | | | + | | | | + | | | | | |   14  9 | | | | | | + | | | + | | | | | | |  0102 15  2 | | | + | | | | | | | + | | | | | |   16 21 | | | + | | | | | | | + | | | | | |   13 19 | | | + | | | | | | | + | | | | | |  0103 13 18,24 | | | | | | | | | | | | | | | | | |   13 18,25 | | | | | + | | | | | | | + | | + |    8  8.3 | | | | | + | | | | | | | + | | + |   11 DB2 | | | | | .vertline . | | | | | | | | | | | |     15 12 | | | | | | | | | | | | | | | | | |  0201  7 DB1 | | | | | | + | | | | | | | | + | +  7 17 | | | | | | + | | | | | | | | + | +  7 11 | | | | | | + | | | | | | | | + | +  0301  4  4(7) | | | | | | | | | | | | | | | | | |    4 13.2(7) | | | | | | | | | | | | | | | | | |    4  4(8) | | | | | | | | | | | | + | + + | |    4 10(8) | | | | | | | | | | | | + | + + | |    4 13.1(8) | | | | | | | | | | | | | | | | | |    4 14(8) | | | | | | | | | | | | + | + + | |  4 KT2 | | | | | | | | | | | | | | | | | |      4 15 | | | | | | | | | | | | | | | | | |  9 23 | | | | | | | | | | | | + | + + | |  0401  8  8.1 | | | | | + | | | + | | | | | | | |    8  8.2 | | | | + | | | + | | | | | | | | |  3 RSH | | | | | | | | | | | | | | | | | |    0501  3  3,24 + | + | | | | + | | | | | | | | | |    3  3,25 + | + | | | | + | | | | | | | | | |   11  5 + | + | | | | + | | | | | | | | | |   11  5(9104) | | | | | | | + | | | | | | | | + +   14 16 | + | | | | | | | | + | | | | | | |   12 DB6 + | + | | | | + | | | | | | | | | |   16 22 | | | | | |  | | | | | | | | | | | |    0601  8  8.3 | | | | | | | | | | + | | | | | | |  PDLP DR Dw 390 385 380 370 365 360 350 340 335 330 305 300 250 190 180 170 140 130</pre>
    
    <p>This example illustrates the ability of the method of this invention to distinguish the alleles and haplotypes of a genetic locus. Specifically, the example shows that PDLP analysis stratifies five of the eight alleles. These three restriction endonuclease digests distinguish each of the eight alleles and many of the 35 known haplotypes of the locus. The use of additional endonuclease digests for this amplified DNA sequence can be expected to distinguish all of the known haplotypes and to potentially identify other previously unrecognized haplotypes. Alternatively, use of the same or other endonuclease digests for another amplified DNA sequence in this locus can be expected to distinguish the haplotypes.</p>
    <p>In addition, analysis of amplified DNA sequences at the DRA locus in the telomeric direction and DQB in the centromeric direction, preferably together with analysis of a central locus, can readily distinguish all of the haplotypes for the region.</p>
    <p>The same methods are readily applied to other loci.</p>
    <heading>EXAMPLE 4</heading> <heading>Analysis of the HLA DQA1 Locus</heading> <p>The DNA of an individual is analyzed to determine which of the three haplotypes of the HLA DQA1 0102 locus are present. Genomic DNA is amplified as described in Example 3. Each of the amplified DNA sequences is sequenced to identify the haplotypes of the individual. The individual is shown to have the haplotypes DR15 DQ6 Dw2; DR13 DQ6 Dw19.</p>
    <p>The procedure is repeated as described in Example 3 through the production of the AluI digest. Each of the digest fragments is sequenced. The individual is shown to have the haplotypes DR15 DQ6 Dw2; DR13 DQ6 Dw19.</p>
    <heading>EXAMPLE 5</heading> <heading>DQA1 Allele-Specific Amplification</heading> <p>Primers were synthesized that specifically bind the 0102 and 0301 alleles of the DQA1 locus. The 5' primer was the SGD 001 primer used in Example 3. The sequences of the 3' primers are listed below.</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">______________________________________0102 5' TTGCTGAACTCAGGCCACC 3'0301 5' TGCGGAACAGAGGCAACTG 3'______________________________________</pre>
    
    <p>The amplification was performed as described in Example 3 using 30 cycles of a standard (94° C., 60° C., 72° C.) PCR reaction. The template DNAs for each of the 0101, 0301 and 0501 alleles were amplified separately. As determined by gel electrophoresis, the 0102-allele-specific primer amplified only template 0102 DNA and the 0301-allele-specific primer amplified only template 0301 DNA. Thus, each of the primers was allele-specific.</p>
    <heading>EXAMPLE 6</heading> <heading>Detection of Cystic Fibrosis</heading> <p>The procedure used for the amplification described in Example 3 is-repeated. The sequences of the primers are illustrated below. The first two primers are upstream primers, and the third is a downstream primer. The primers amplify a DNA sequence that encompasses all of intervening sequence 1</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">______________________________________5' CAG AGG TCG CCT CTG GA 3';5' AAG GCC AGC GTT GTC TCC A 3'; and3' CCT CAA AAT TGG TCT GGT 5'.______________________________________</pre>
    
    <p>These primers hybridize to the complement of sequences located from nt 136-152 and nt 154-172, and to nt 187-207. [The nucleotide numbers are found in Riordan et al, Science 245:1066-1072 (1989).]</p>
    <p>Following amplification, the amplified DNA sequences are electrophoresed on a 4% polyacrylamide gel to determine the PDLP type. The amplified DNA sequences are separately digested using each of the restriction enzymes AluI, MnlI and RsaI (Bethesda Research Laboratories). The digestion is performed as described in Example 3. The products of the digestion reaction are electrophoresed and visualized using a 4% horizontal ultra-thin polyacrylamide gel and silver stain as described in Example 3.</p>
    <p>Distinctive fragment patterns which correspond to disease-associated and normal haplotypes are produced.</p>
    <heading>EXAMPLE 7</heading> <heading>Analysis of Bovine Leukocyte Antigen Class I</heading> <p>Bovine Leukocyte Antigen (BOLA) Class I alleles and haplotypes are analyzed in the same manner as described in Example 3. The primers are listed below.</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">______________________________________Bovine Primers (Class I HLA homolog)                         T<sub>m</sub>______________________________________5' primer: 5' TCC TGG TCC TGA CCG AGA 3'                         (62°)3' primer: 1)    3' A TGT GCC TTT GGA GGG TCT 5'                             (62°)    (for .sup.˜ 600 bp product)2)       3' GCC AAC AT GAT CCG CAT 5'                             (62°)    (for ˜900 bp product)______________________________________</pre>
    
    <p>For the approximately 900 bp sequence PDLP analysis is sufficient to distinguish alleles 1 and 3 (893 and 911 bp, respectively). Digests are prepared as described in Example 3 using AluI and DdeI. The following patterns are produced for the 900 bp sequence.</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="equation">Allele 1, AluI digest: 712, 181 Allele 3, AluI digest: 430, 300, 181</pre>
    
    <p>
      </p> <pre xml:space="preserve" listing-type="equation">Allele 1, DdeI digest: 445, 201, 182, 28 Allele 3, DdeI digest: 406, 185, 182, 28, 16</pre>
    
    <p>The 600 bp sequence also produces distinguishable fragment patterns for those alleles. However, those patterns are not as dramatically different as the patterns produced by the 600 bp sequence digests.</p>
    <heading>EXAMPLE 8</heading> <heading>Preparation of Primers</heading> <p>Each of the following primers is synthesized using an Applied Biosystems model 308A DNA synthesizer.</p>
    <heading>HLA locus primers</heading> <p>A locus-specific primers</p>
    <p>SGD009.AIVS3.R2NP CATGTGGCCATCTTGAGAATGGA SGD006.AIVS3.R1NP GCCCGGGAGATCTACAGGCGATCA A2.1 CGCCTCCCTGATCGCCTGTAG A2.2 CCAGAGAGTGACTCTGAGG A2.3 CACAATTAAGGGAT</p>
    <p>B locus-specific primers</p>
    <p>SGD007.BIVS3.R1NP TCCCCGGCGACCTATAGGAGATGG SGD010.BIVS3.R2NP CTAGGACCACCCATGTGACCAGC B2.1 ATCTCCTCAGACGCCGAGATGCGTCAC B2.2 CTCCTGCTGCTCTGGGGGGCAG B2.3 ACTTTACCTCCACTCAGATCAGGAG B2.4 CGTCCAGGCTGGTGTCTGGGTTCTGTGCCCCT B2.5 CTGGTCACATGGGTGGTCCTAGG B2.6 CGCCTGAATTTTCTGACTCTTCCCAT</p>
    <p>C locus-specific primers</p>
    <p>SGD008.CIVS3.R1NP ATCCCGGGAGATCTACAGGAGATG SGD011.CIVS3.R2NP AACAGCGCCCATGTGACCATCCT C2.1 CTGGGGAGGCGCCGCGTTGAGGATTCT C2.2 CGTCTCCGCAGTCCCGGTTCTAAAGTTCCCAGT C2.3 ATCCTCGTGCTCTCGGGA C2.4 TGTGGTCAGGCTGCTGAC C2.5 AAGGTTTGATTCCAGCTT C2.6 CCCCTTCCCCACCCCAGGTGTTCCTGTCCATTCTTCAGGA C2.7 CACATGGGCGCTGTTGGAGTGTCG</p>
    <p>Class I loci-specific primers</p>
    <p>SGD005.IIVS1.LNP GTGAGTGCGGGGTCGGGAGGGA 1.1 CACCCACCGGGACTCAGA 1.2 TGGCCCTGACCCAGACCTGGGC 1.3 GAGGGTCGGGCGGGTCTCAGC 1.4 CTCTCAGGCCTTGTTC 1.5 CAGAAGTCGCTGTTCC</p>
    <p>DQA1 locus-specific primers</p>
    <p>SGD001.DQA1.LNP TTCTGAGCCAGTCCTGAGA DQA3 E1a TTGCCCTGACCACCGTGATG DQA3 E1b CTTCCTGCTTGTCATCTTCA DQA3 E1c CCATGAATTTGATGGAGA DQA3 E1d ACCGCTGCTACCAATGGTA SGD003.DQA1.RNP CCAAGAGGTCCCCAGATC</p>
    <p>DRA locus-specific primers</p>
    <p>DRA E1 TCATCATAGCTGTGCTGATG DRA 5'E2 AGAACATGTGATCATCCAGGC DRA 3'E2 CCAACTATACTCCGATCACCAAT</p>
    <p>DRB locus-specific primers</p>
    <p>DRB E1 TGACAGTGACACTGATGGTGCTG DRB 5'E2 GGGGACACCCGACCACGTTTC DRB 3'E2 TGCAGACACAACTACGGGGTTG</p>
    <p>DQB1 locus-specific primers</p>
    <p>DQB E1 TGGCTGAGGGCAGAGACTCTCCC DQB 5'E2 TGCTACTTCACCAACGGGAC DQB 3'E2 GGTGTGCACACACAACTAC DQB 5'IVS1a AGGTATTTTACCCAGGGACCAAGAGAT DQB 5'IVS1b ATGTAAAATCAGCCCGACTGCCTCTTC DQB 3'IVS2 GCCTCGTGCCTTATGCGTTTGCCTCCT</p>
    <p>DPB1 locus'specific primers</p>
    <p>DPB E1 TGAGGTTAATAAACTGGAGAA DPB 5'IVS1 GAGAGTGGCGCCTCCGCTCAT DPB 3'IVS2 GAGTGAGGGCTTTGGGCCGG</p>
    </div></div></div><div class="patent-section patent-tabular-section"><a id="backward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Cited Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4582788">US4582788</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jan 7, 1983</td><td class="patent-data-table-td patent-date-value">Apr 15, 1986</td><td class="patent-data-table-td ">Cetus Corporation</td><td class="patent-data-table-td ">Digestion human leukocyte antigen-dna endonuclease to give pattern, then calibration to standard</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4683194">US4683194</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Mar 28, 1985</td><td class="patent-data-table-td patent-date-value">Jul 28, 1987</td><td class="patent-data-table-td ">Cetus Corporation</td><td class="patent-data-table-td ">Method for detection of polymorphic restriction sites and nucleic acid sequences</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4683195">US4683195</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Feb 7, 1986</td><td class="patent-data-table-td patent-date-value">Jul 28, 1987</td><td class="patent-data-table-td ">Cetus Corporation</td><td class="patent-data-table-td ">One of the first Polymerase Chain Reaction (PCR) patents</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5192659">US5192659</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jul 11, 1990</td><td class="patent-data-table-td patent-date-value">Mar 9, 1993</td><td class="patent-data-table-td ">Genetype Ag</td><td class="patent-data-table-td ">DNA and primers for genetic engineering</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP0256630A1?cl=en">EP0256630A1</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jun 9, 1987</td><td class="patent-data-table-td patent-date-value">Feb 24, 1988</td><td class="patent-data-table-td ">Howard Hughes Medical Institute</td><td class="patent-data-table-td ">Specific probes for the identification of mutations in the human phenylalanine hydroxylase gene</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP0269260A2?cl=en">EP0269260A2</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Oct 27, 1987</td><td class="patent-data-table-td patent-date-value">Jun 1, 1988</td><td class="patent-data-table-td ">Biotechnology Research Partners, Ltd.</td><td class="patent-data-table-td ">apoAI-CIII-AIV, apoAII apoB, apoCI, and LDL receptor polymorphisms for genetic fingerprinting and predictive of atherosclerosis</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP0364255A2?cl=en">EP0364255A2</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Oct 11, 1989</td><td class="patent-data-table-td patent-date-value">Apr 18, 1990</td><td class="patent-data-table-td ">Baylor College Of Medicine</td><td class="patent-data-table-td ">Multiplex genomic DNA amplification for deletion detection</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1989011547A1?cl=en">WO1989011547A1</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">May 18, 1989</td><td class="patent-data-table-td patent-date-value">Nov 30, 1989</td><td class="patent-data-table-td ">Cetus Corp</td><td class="patent-data-table-td ">Method for hla dp typing</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1989012697A1?cl=en">WO1989012697A1</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jun 21, 1989</td><td class="patent-data-table-td patent-date-value">Dec 28, 1989</td><td class="patent-data-table-td ">Univ Washington</td><td class="patent-data-table-td ">Method for detecting abnormal genes</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="npl-citations"></a><div class="patent-section-header"><span class="patent-section-title">Non-Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th colspan="3"class="patent-data-table-th">Reference</th></tr></thead><tr><td class="patent-data-table-td ">1</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Abraham, et al., "<a href='http://scholar.google.com/scholar?q="Sequence+differences+between+HLA-B+and+TNF+distinguish+different+MHC+ancestral+haplotypes%2C"'>Sequence differences between HLA-B and TNF distinguish different MHC ancestral haplotypes,</a>" Antigens, vol 39, No. 3, Mar. 1992, pp. 117-121.</td></tr><tr><td class="patent-data-table-td ">2</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Abraham, et al., Sequence differences between HLA B and TNF distinguish different MHC ancestral haplotypes, Antigens , vol 39, No. 3, Mar. 1992, pp. 117 121.</td></tr><tr><td class="patent-data-table-td ">3</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Allen et al., Biotechniques , vol. 7, No. 7, pp. 736 744 (1989).</td></tr><tr><td class="patent-data-table-td ">4</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Allen et al., Biotechniques, vol. 7, No. 7, pp. 736-744 (1989).</td></tr><tr><td class="patent-data-table-td ">5</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Boehnke et al., "<a href='http://scholar.google.com/scholar?q="Fine-Structure+Genetic+Mapping+of+Human+Chromosomes+Using+the+Polymerase+Chain+Reaction+on+Single+Sperm%3A+Experimental+Design+Considerations%2C"'>Fine-Structure Genetic Mapping of Human Chromosomes Using the Polymerase Chain Reaction on Single Sperm: Experimental Design Considerations,</a>" The American Journal of Human Genetics, Volumr 45, No. 1, Jul. 1989, pp. 21-32.</td></tr><tr><td class="patent-data-table-td ">6</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Boehnke et al., Fine Structure Genetic Mapping of Human Chromosomes Using the Polymerase Chain Reaction on Single Sperm: Experimental Design Considerations, The American Journal of Human Genetics , Volumr 45, No. 1, Jul. 1989, pp. 21 32.</td></tr><tr><td class="patent-data-table-td ">7</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Clark, "<a href='http://scholar.google.com/scholar?q="Interference+of+Haplotypes+from+PCR-amplified+Samples+of+Diploid+Populations"'>Interference of Haplotypes from PCR-amplified Samples of Diploid Populations</a>", Mol. Biol. Evol., vol. 7, No. 2, 1990, pp. 111-122.</td></tr><tr><td class="patent-data-table-td ">8</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Clark, Interference of Haplotypes from PCR amplified Samples of Diploid Populations , Mol. Biol. Evol., vol. 7, No. 2, 1990, pp. 111 122.</td></tr><tr><td class="patent-data-table-td ">9</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Deng, "<a href='http://scholar.google.com/scholar?q="A+sensitive+non-radioactive+PCR-RFLP+analysis+for+detecting+point+mutations+at+12th+condon+oncogene+c-Ha-ras+in+DNAs+of+gastic+cancer%2C"'>A sensitive non-radioactive PCR-RFLP analysis for detecting point mutations at 12th condon oncogene c-Ha-ras in DNAs of gastic cancer,</a>" Nucleic Acids Research, vol. 16, No. 13, 1988, p. 6231.</td></tr><tr><td class="patent-data-table-td ">10</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Deng, A sensitive non radioactive PCR RFLP analysis for detecting point mutations at 12th condon oncogene c Ha ras in DNAs of gastic cancer, Nucleic Acids Research , vol. 16, No. 13, 1988, p. 6231.</td></tr><tr><td class="patent-data-table-td ">11</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Dicker, et al., "<a href='http://scholar.google.com/scholar?q="Sequence+Analysis+of+a+Human+Gene+Responsible+for+Drug+Resistance%3A+A+Rapid+Method+for+Manual+and+Automated+Direct+Sequencing+of+Products+Generated+by+the+Polymerase+Chain+Reaction%2C"'>Sequence Analysis of a Human Gene Responsible for Drug Resistance: A Rapid Method for Manual and Automated Direct Sequencing of Products Generated by the Polymerase Chain Reaction,</a>" BioTechniques, vol. 7, No. 8, 1989, pp. 830-837.</td></tr><tr><td class="patent-data-table-td ">12</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Dicker, et al., Sequence Analysis of a Human Gene Responsible for Drug Resistance: A Rapid Method for Manual and Automated Direct Sequencing of Products Generated by the Polymerase Chain Reaction, BioTechniques , vol. 7, No. 8, 1989, pp. 830 837.</td></tr><tr><td class="patent-data-table-td ">13</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">DiLella, et al., "<a href='http://scholar.google.com/scholar?q="Screening+for+Phenylketonuria+Mutations+by+DNA+Amplification+with+the+Polymerase+Chain+Reaction%2C"'>Screening for Phenylketonuria Mutations by DNA Amplification with the Polymerase Chain Reaction,</a>" The Lancet, Mar. 5, 1988, pp.. 497-499.</td></tr><tr><td class="patent-data-table-td ">14</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">DiLella, et al., "<a href='http://scholar.google.com/scholar?q="Tight+linkage+between+a+splicing+mutation+and+a+specific+DNA+haplotype+in+phenylketonuria%2C"'>Tight linkage between a splicing mutation and a specific DNA haplotype in phenylketonuria,</a>" Nature, vol. 322, No. 28, Aug. 1986, pp. 799-803.</td></tr><tr><td class="patent-data-table-td ">15</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">DiLella, et al., Screening for Phenylketonuria Mutations by DNA Amplification with the Polymerase Chain Reaction, The Lancet , Mar. 5, 1988, pp.. 497 499.</td></tr><tr><td class="patent-data-table-td ">16</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">DiLella, et al., Tight linkage between a splicing mutation and a specific DNA haplotype in phenylketonuria, Nature , vol. 322, No. 28, Aug. 1986, pp. 799 803.</td></tr><tr><td class="patent-data-table-td ">17</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Erlich et al., "<a href='http://scholar.google.com/scholar?q="Reliability+of+the+HLA-DQ%CE%B1+PCR-Based+Olignucleotide+Typing+System%2C"'>Reliability of the HLA-DQα PCR-Based Olignucleotide Typing System,</a>" Journal of Forensic Sciences, Sep. 1990, pp. 1017-1018.</td></tr><tr><td class="patent-data-table-td ">18</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Erlich et al., Reliability of the HLA DQ PCR Based Olignucleotide Typing System, Journal of Forensic Sciences , Sep. 1990, pp. 1017 1018.</td></tr><tr><td class="patent-data-table-td ">19</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Funke et al., "<a href='http://scholar.google.com/scholar?q="Detection+of+a+New+Msp+I+Restriction+Fragment+Length+Polymorphism+in+the+Apolipoprotein+A-I+Gene%2C"'>Detection of a New Msp I Restriction Fragment Length Polymorphism in the Apolipoprotein A-I Gene,</a>" J. Clin. Chem. Clin. Biochem., vol. 25, No. 3, 1987, pp. 131-134.</td></tr><tr><td class="patent-data-table-td ">20</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Funke et al., Detection of a New Msp I Restriction Fragment Length Polymorphism in the Apolipoprotein A I Gene, J. Clin. Chem. Clin. Biochem. , vol. 25, No. 3, 1987, pp. 131 134.</td></tr><tr><td class="patent-data-table-td ">21</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Geisel et al., "<a href='http://scholar.google.com/scholar?q="A+New+APA+LI+Restriction+Fragment+Length+Polymrophism+in+the+Low+Density+Lipoprotein+Receptor+Gene%2C"'>A New APA LI Restriction Fragment Length Polymrophism in the Low Density Lipoprotein Receptor Gene,</a>" Journal of Clinical Chemistry and Clinical Biochemistry, vol. 26, No. 7, 1988, pp. 429-434.</td></tr><tr><td class="patent-data-table-td ">22</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Geisel et al., A New APA LI Restriction Fragment Length Polymrophism in the Low Density Lipoprotein Receptor Gene, Journal of Clinical Chemistry and Clinical Biochemistry , vol. 26, No. 7, 1988, pp. 429 434.</td></tr><tr><td class="patent-data-table-td ">23</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Graham et al., "<a href='http://scholar.google.com/scholar?q="Application+of+Molecular+Genetics+to+Prenatal+Diagnosis+and+Carrier+Detection+in+the+Hemophilias%3A+Some+Limitaitons%2C"'>Application of Molecular Genetics to Prenatal Diagnosis and Carrier Detection in the Hemophilias: Some Limitaitons,</a>" Blood, vol. 66, No. 4, Oct. 1985, pp. 759-764.</td></tr><tr><td class="patent-data-table-td ">24</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Graham et al., Application of Molecular Genetics to Prenatal Diagnosis and Carrier Detection in the Hemophilias: Some Limitaitons, Blood , vol. 66, No. 4, Oct. 1985, pp. 759 764.</td></tr><tr><td class="patent-data-table-td ">25</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Kerem et al., "<a href='http://scholar.google.com/scholar?q="Identification+of+the+Cystic+Fibrosis+Gene%3A+Genetic+Analysis%2C"'>Identification of the Cystic Fibrosis Gene: Genetic Analysis,</a>", Science, vol. 245, 1989, pp. 1073-1080.</td></tr><tr><td class="patent-data-table-td ">26</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Kerem et al., Identification of the Cystic Fibrosis Gene: Genetic Analysis, , Science , vol. 245, 1989, pp. 1073 1080.</td></tr><tr><td class="patent-data-table-td ">27</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Kogan et al., "<a href='http://scholar.google.com/scholar?q="An+Improved+Method+for+Prenatal+Diagnosis+of+Genetic+Diseases+by+Analysis+of+Amplifed+DNA+Sequences--Application+to+Hemophilia+A%2C"'>An Improved Method for Prenatal Diagnosis of Genetic Diseases by Analysis of Amplifed DNA Sequences--Application to Hemophilia A,</a>" The New England Journal of Medicine, vol. 317, No. 16, Oct. 15, 1987, pp. 985-990.</td></tr><tr><td class="patent-data-table-td ">28</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Kogan et al., An Improved Method for Prenatal Diagnosis of Genetic Diseases by Analysis of Amplifed DNA Sequences Application to Hemophilia A, The New England Journal of Medicine, vol. 317, No. 16, Oct. 15, 1987, pp. 985 990.</td></tr><tr><td class="patent-data-table-td ">29</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Mardis et al., "<a href='http://scholar.google.com/scholar?q="Automated+Methods+for+Single-Stranded+DNA+Isolation+and+Dideoxynucleotide+DNA+Sequencing+Reactions+on+a+Robotic+Workstation%2C"'>Automated Methods for Single-Stranded DNA Isolation and Dideoxynucleotide DNA Sequencing Reactions on a Robotic Workstation,</a>" BioTechniques, vol. 7, No. 8, 1989, pp. 840-850.</td></tr><tr><td class="patent-data-table-td ">30</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Mardis et al., Automated Methods for Single Stranded DNA Isolation and Dideoxynucleotide DNA Sequencing Reactions on a Robotic Workstation, BioTechniques , vol. 7, No. 8, 1989, pp. 840 850.</td></tr><tr><td class="patent-data-table-td ">31</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Marx, "<a href='http://scholar.google.com/scholar?q="Disecting+the+Complex+Diseases%2C"'>Disecting the Complex Diseases,</a>" Science, vol. 247, Mar. 30, 1990, pp. 1540-1542.</td></tr><tr><td class="patent-data-table-td ">32</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Marx, Disecting the Complex Diseases, Science , vol. 247, Mar. 30, 1990, pp. 1540 1542.</td></tr><tr><td class="patent-data-table-td ">33</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Molecular Biology of The Cell , 2nd Edition, New York, 1989, Chapter 9: RNA Synthesis and RNA Processing pp. 531 535 and Chapter 10: Control of Gene Expression pp. 602 604.</td></tr><tr><td class="patent-data-table-td ">34</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Molecular Biology of The Cell, 2nd Edition, New York, 1989, Chapter 9: "<a href='http://scholar.google.com/scholar?q="RNA+Synthesis+and+RNA+Processing"'>RNA Synthesis and RNA Processing</a>" pp. 531-535 and Chapter 10: Control of Gene Expression pp. 602-604.</td></tr><tr><td class="patent-data-table-td ">35</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Molecular Biology of the Gene , 4th Edition, California, 1987, Chapter 20: Conserved Sequences at Exon Intron Boundaries in Pre mRNA, pp. 639 640.</td></tr><tr><td class="patent-data-table-td ">36</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Molecular Biology of the Gene, 4th Edition, California, 1987, Chapter 20: "<a href='http://scholar.google.com/scholar?q="Conserved+Sequences+at+Exon-Intron+Boundaries+in+Pre-mRNA%2C"'>Conserved Sequences at Exon-Intron Boundaries in Pre-mRNA,</a>" pp. 639-640.</td></tr><tr><td class="patent-data-table-td ">37</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Padgett et al. Amer. Rev. Biochem. 1986. 55:1119 150.</td></tr><tr><td class="patent-data-table-td ">38</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Padgett et al. Amer. Rev. Biochem. 1986. 55:1119-150.</td></tr><tr><td class="patent-data-table-td ">39</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Riess, et al., "<a href='http://scholar.google.com/scholar?q="Hypervariability+of+intronic+simple+%28gt%29n+%28ga%29m+repeats+in+HLA-DRB+genes%2C"'>Hypervariability of intronic simple (gt)n (ga)m repeats in HLA-DRB genes,</a>" Immunogenetics, vol. 32, No. 2, Aug. 1990, pp. 110-116.</td></tr><tr><td class="patent-data-table-td ">40</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Riess, et al., Hypervariability of intronic simple (gt) n (ga) m repeats in HLA DRB genes, Immunogenetics , vol. 32, No. 2, Aug. 1990, pp. 110 116.</td></tr><tr><td class="patent-data-table-td ">41</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Riordan et al., "<a href='http://scholar.google.com/scholar?q="Identification+of+the+Cystic+Fibrosis+Gene%3A+Cloning+and+Characterization+of+Complementary+DNA%2C"'>Identification of the Cystic Fibrosis Gene: Cloning and Characterization of Complementary DNA,</a>", Science, vol. 245, 1989, pp. 1066-1073.</td></tr><tr><td class="patent-data-table-td ">42</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Riordan et al., Identification of the Cystic Fibrosis Gene: Cloning and Characterization of Complementary DNA, , Science , vol. 245, 1989, pp. 1066 1073.</td></tr><tr><td class="patent-data-table-td ">43</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Rommens et al., "<a href='http://scholar.google.com/scholar?q="Identification+of+the+Cystic+Fibrosis+Gene%3A+Chromosome+Walking+and+Jumping%2C"'>Identification of the Cystic Fibrosis Gene: Chromosome Walking and Jumping,</a>", Science, vol. 245, 1989, pp. 1059-1065.</td></tr><tr><td class="patent-data-table-td ">44</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Rommens et al., Identification of the Cystic Fibrosis Gene: Chromosome Walking and Jumping, , Science , vol. 245, 1989, pp. 1059 1065.</td></tr><tr><td class="patent-data-table-td ">45</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Saiki et al., "<a href='http://scholar.google.com/scholar?q="Enzymatic+Amplification+of+%CE%B2-Globin+Genomic+Sequences+and+Restriction+Site+Analysis+for+Diagnosis+of+Sickle+Cell+Anemia%2C"'>Enzymatic Amplification of β-Globin Genomic Sequences and Restriction Site Analysis for Diagnosis of Sickle Cell Anemia,</a>", Science, vol. 230, 1985, pp. 1350-1354.</td></tr><tr><td class="patent-data-table-td ">46</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Saiki et al., Enzymatic Amplification of Globin Genomic Sequences and Restriction Site Analysis for Diagnosis of Sickle Cell Anemia, , Science , vol. 230, 1985, pp. 1350 1354.</td></tr><tr><td class="patent-data-table-td ">47</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Simon and Erlich, pp. 952 958 In: Immunology of HLA , vol. 1: Springer Verlag, New York (1989).</td></tr><tr><td class="patent-data-table-td ">48</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Simon and Erlich, pp. 952-958 In: Immunology of HLA, vol. 1: Springer-Verlag, New York (1989).</td></tr><tr><td class="patent-data-table-td ">49</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Simons et al., pp. 952 958 In: Immunology of HLA , vol. 1:Springer Verlag, New York (1989).</td></tr><tr><td class="patent-data-table-td ">50</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Simons et al., pp. 952-958 In: Immunology of HLA, vol. 1:Springer-Verlag, New York (1989).</td></tr><tr><td class="patent-data-table-td ">51</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Simons, et al., pp. 959 1023 In: Immunology of HLA , vol. 1: Springer Verlag, New York (1989).</td></tr><tr><td class="patent-data-table-td ">52</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Simons, et al., pp. 959-1023 In: Immunology of HLA, vol. 1: Springer-Verlag, New York (1989).</td></tr><tr><td class="patent-data-table-td ">53</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Stephens et al., "<a href='http://scholar.google.com/scholar?q="Theoretical+Underpinning+of+the+Single-Molecule-Dilution+%28SMD%29+Method+of+Direct+Haplotype+Resolution%2C"'>Theoretical Underpinning of the Single-Molecule-Dilution (SMD) Method of Direct Haplotype Resolution,</a>" Am. J. Hum. Genet, vol. 46, 1990, pp. 1149-1155.</td></tr><tr><td class="patent-data-table-td ">54</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Stephens et al., Theoretical Underpinning of the Single Molecule Dilution (SMD) Method of Direct Haplotype Resolution, Am. J. Hum. Genet , vol. 46, 1990, pp. 1149 1155.</td></tr><tr><td class="patent-data-table-td ">55</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Waikan et al. P.N.A.S, U.S.A., vol. 75, No. 11, pp. 5631 5635, Nov. 1978.</td></tr><tr><td class="patent-data-table-td ">56</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Waikan et al. P.N.A.S, U.S.A., vol. 75, No. 11, pp. 5631-5635, Nov. 1978.</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="forward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Referenced by</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Citing Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5972604">US5972604</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Aug 23, 1996</td><td class="patent-data-table-td patent-date-value">Oct 26, 1999</td><td class="patent-data-table-td ">Regents Of The University Of Minnesota</td><td class="patent-data-table-td ">DNA sequence-based HLA typing method</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6383751">US6383751</a></td><td class="patent-data-table-td patent-date-value">Oct 23, 1998</td><td class="patent-data-table-td patent-date-value">May 7, 2002</td><td class="patent-data-table-td ">William John Barendse</td><td class="patent-data-table-td ">Assessing marbling score of cattle by testing animal for presence or absense of allele of 5&#39; untranslated region of gene encoding thyroglobin; presence of allele 3 indicates high score, homozygositiy for allele 2 indicates low score</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6627067">US6627067</a></td><td class="patent-data-table-td patent-date-value">Jun 22, 2000</td><td class="patent-data-table-td patent-date-value">Sep 30, 2003</td><td class="patent-data-table-td ">President And Fellows Of Harvard College</td><td class="patent-data-table-td ">Rapid, reliable and inexpensive characterization of such as nucleic acids; increased temperature, bias voltage operating range and chemical environment accommodation; reduced noise levels; durable</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6703228">US6703228</a></td><td class="patent-data-table-td patent-date-value">Sep 24, 1999</td><td class="patent-data-table-td patent-date-value">Mar 9, 2004</td><td class="patent-data-table-td ">Massachusetts Institute Of Technology</td><td class="patent-data-table-td ">Methods and products related to genotyping and DNA analysis</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6759192">US6759192</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jun 4, 1999</td><td class="patent-data-table-td patent-date-value">Jul 6, 2004</td><td class="patent-data-table-td ">Genset S.A.</td><td class="patent-data-table-td ">Polymorphic markers of prostate carcinoma tumor antigen-1(PCTA-1)</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6844154">US6844154</a></td><td class="patent-data-table-td patent-date-value">Mar 30, 2001</td><td class="patent-data-table-td patent-date-value">Jan 18, 2005</td><td class="patent-data-table-td ">Polygenyx, Inc.</td><td class="patent-data-table-td ">High throughput methods for haplotyping</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6931326">US6931326</a></td><td class="patent-data-table-td patent-date-value">Dec 21, 2001</td><td class="patent-data-table-td patent-date-value">Aug 16, 2005</td><td class="patent-data-table-td ">Genaissance Pharmaceuticals, Inc.</td><td class="patent-data-table-td ">Methods for obtaining and using haplotype data</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7058517">US7058517</a></td><td class="patent-data-table-td patent-date-value">Jun 26, 2000</td><td class="patent-data-table-td patent-date-value">Jun 6, 2006</td><td class="patent-data-table-td ">Genaissance Pharmaceuticals, Inc.</td><td class="patent-data-table-td ">Methods for obtaining and using haplotype data</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7189503">US7189503</a></td><td class="patent-data-table-td patent-date-value">Dec 18, 2003</td><td class="patent-data-table-td patent-date-value">Mar 13, 2007</td><td class="patent-data-table-td ">President And Fellows Of Harvard College</td><td class="patent-data-table-td ">Analysis system comprising structure of an ion-permeable passage connecting first AND second pools; molecular motor of processive nucleic acid translocation is not bacteriophage receptor; differentiation between hybridized nucleotide monomers before entering channel</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7238485">US7238485</a></td><td class="patent-data-table-td patent-date-value">Mar 23, 2005</td><td class="patent-data-table-td patent-date-value">Jul 3, 2007</td><td class="patent-data-table-td ">President And Fellows Of Harvard College</td><td class="patent-data-table-td ">Methods and apparatus for characterizing polynucleotides</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7435541">US7435541</a></td><td class="patent-data-table-td patent-date-value">Apr 4, 2002</td><td class="patent-data-table-td patent-date-value">Oct 14, 2008</td><td class="patent-data-table-td ">Sequenom, Inc.</td><td class="patent-data-table-td ">Restriction enzyme genotyping</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7468248">US7468248</a></td><td class="patent-data-table-td patent-date-value">Dec 31, 2003</td><td class="patent-data-table-td patent-date-value">Dec 23, 2008</td><td class="patent-data-table-td ">Cargill, Incorporated</td><td class="patent-data-table-td ">identifying in the nucleic acid sample, a nucleotide occurrence of a single nucleotide polymorphism (SNP) at position 300 of SEQ ID NO:21645, thereby inferring the trait, wherein the trait is marbling, tenderness, fat thickness, red meat yield, or average daily weight gain</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7511127">US7511127</a></td><td class="patent-data-table-td patent-date-value">Dec 31, 2003</td><td class="patent-data-table-td patent-date-value">Mar 31, 2009</td><td class="patent-data-table-td ">Cargill, Incorporated</td><td class="patent-data-table-td ">Compositions, methods and systems for inferring bovine breed</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7547771">US7547771</a></td><td class="patent-data-table-td patent-date-value">May 27, 2004</td><td class="patent-data-table-td patent-date-value">Jun 16, 2009</td><td class="patent-data-table-td ">Serono Genetics Institute S.A.</td><td class="patent-data-table-td ">Biallelic tumor markers for use as diagnostic/prognostic indicators of cell proliferative disorders</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7585626">US7585626</a></td><td class="patent-data-table-td patent-date-value">Aug 19, 2004</td><td class="patent-data-table-td patent-date-value">Sep 8, 2009</td><td class="patent-data-table-td ">Quest Diagnostics Investments Incorporated</td><td class="patent-data-table-td ">Use of at least one primer which includes a complementary region that hybridizes adjacent to the target segment of nucleotides and further includes an extender region that has a nucleotide sequence that is different from the complementary region and does not hybridize to the target segment of nucleotides</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7625706">US7625706</a></td><td class="patent-data-table-td patent-date-value">Jul 3, 2007</td><td class="patent-data-table-td patent-date-value">Dec 1, 2009</td><td class="patent-data-table-td ">Agilent Technologies, Inc.</td><td class="patent-data-table-td ">Using nanopore apparatus comprising immobilized polymerase as replicative tool in monitoring gene expression analysis</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7659054">US7659054</a></td><td class="patent-data-table-td patent-date-value">Oct 25, 2000</td><td class="patent-data-table-td patent-date-value">Feb 9, 2010</td><td class="patent-data-table-td ">Nuvelo, Inc.</td><td class="patent-data-table-td ">Methods for genetic analysis of DNA to detect sequence variances</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7666588">US7666588</a></td><td class="patent-data-table-td patent-date-value">Sep 12, 2003</td><td class="patent-data-table-td patent-date-value">Feb 23, 2010</td><td class="patent-data-table-td ">Ibis Biosciences, Inc.</td><td class="patent-data-table-td ">Methods for rapid forensic analysis of mitochondrial DNA and characterization of mitochondrial DNA heteroplasmy</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7666592">US7666592</a></td><td class="patent-data-table-td patent-date-value">Feb 17, 2005</td><td class="patent-data-table-td patent-date-value">Feb 23, 2010</td><td class="patent-data-table-td ">Ibis Biosciences, Inc.</td><td class="patent-data-table-td ">Methods for concurrent identification and quantification of an unknown bioagent</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7709206">US7709206</a></td><td class="patent-data-table-td patent-date-value">Feb 13, 2008</td><td class="patent-data-table-td patent-date-value">May 4, 2010</td><td class="patent-data-table-td ">Metamorphix, Inc.</td><td class="patent-data-table-td ">Using presence of single nucleotide polymorphism as evaluative tool in determining animal health, meat quality and edibility</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7714275">US7714275</a></td><td class="patent-data-table-td patent-date-value">May 24, 2005</td><td class="patent-data-table-td patent-date-value">May 11, 2010</td><td class="patent-data-table-td ">Ibis Biosciences, Inc.</td><td class="patent-data-table-td ">Mass spectrometry with selective ion filtration by digital thresholding</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7718354">US7718354</a></td><td class="patent-data-table-td patent-date-value">Dec 5, 2003</td><td class="patent-data-table-td patent-date-value">May 18, 2010</td><td class="patent-data-table-td ">Ibis Biosciences, Inc.</td><td class="patent-data-table-td ">Methods for rapid identification of pathogens in humans and animals</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7741036">US7741036</a></td><td class="patent-data-table-td patent-date-value">Jan 13, 2006</td><td class="patent-data-table-td patent-date-value">Jun 22, 2010</td><td class="patent-data-table-td ">Ibis Biosciences, Inc.</td><td class="patent-data-table-td ">by a combination of nucleic acid amplification and molecular weight determination using primers which hybridize to conserved sequence regions of nucleic acids derived from a bioagent and which bracket variable sequence regions that uniquely identify the bioagent</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7781162">US7781162</a></td><td class="patent-data-table-td patent-date-value">Sep 11, 2003</td><td class="patent-data-table-td patent-date-value">Aug 24, 2010</td><td class="patent-data-table-td ">Ibis Biosciences, Inc.</td><td class="patent-data-table-td ">Using replication of intelligent primers (ribosomal RNA or house keeping oligonucleotides) to identify, classify and diagnose infection</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7811753">US7811753</a></td><td class="patent-data-table-td patent-date-value">Jul 14, 2004</td><td class="patent-data-table-td patent-date-value">Oct 12, 2010</td><td class="patent-data-table-td ">Ibis Biosciences, Inc.</td><td class="patent-data-table-td ">Methods for repairing degraded DNA</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7846738">US7846738</a></td><td class="patent-data-table-td patent-date-value">Aug 16, 2004</td><td class="patent-data-table-td patent-date-value">Dec 7, 2010</td><td class="patent-data-table-td ">President And Fellows Of Harvard College</td><td class="patent-data-table-td ">determining the conformational distribution of a plurality of biological polymers and to detect binding or denaturation events; employ a nanopore which the polymer, e.g a nucleic acid, traverses; as the polymer traverses the nanopore, measurements of transport properties of the nanopore yield data</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7947454">US7947454</a></td><td class="patent-data-table-td patent-date-value">Nov 25, 2009</td><td class="patent-data-table-td patent-date-value">May 24, 2011</td><td class="patent-data-table-td ">President And Fellows Of Harvard College</td><td class="patent-data-table-td ">Methods and apparatus for characterizing polynucleotides</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7956175">US7956175</a></td><td class="patent-data-table-td patent-date-value">Mar 7, 2007</td><td class="patent-data-table-td patent-date-value">Jun 7, 2011</td><td class="patent-data-table-td ">Ibis Biosciences, Inc.</td><td class="patent-data-table-td ">Compositions for use in identification of bacteria</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7964343">US7964343</a></td><td class="patent-data-table-td patent-date-value">May 12, 2004</td><td class="patent-data-table-td patent-date-value">Jun 21, 2011</td><td class="patent-data-table-td ">Ibis Biosciences, Inc.</td><td class="patent-data-table-td ">Using column chromatography to isolate preferential polynucleotides from solution; electrospray mass spectrometry</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8013142">US8013142</a></td><td class="patent-data-table-td patent-date-value">Mar 13, 2007</td><td class="patent-data-table-td patent-date-value">Sep 6, 2011</td><td class="patent-data-table-td ">Ibis Biosciences, Inc.</td><td class="patent-data-table-td ">Compositions for use in identification of bacteria</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8017322">US8017322</a></td><td class="patent-data-table-td patent-date-value">Sep 21, 2005</td><td class="patent-data-table-td patent-date-value">Sep 13, 2011</td><td class="patent-data-table-td ">Ibis Biosciences, Inc.</td><td class="patent-data-table-td ">Method for rapid detection and identification of bioagents</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8017358">US8017358</a></td><td class="patent-data-table-td patent-date-value">Jan 13, 2006</td><td class="patent-data-table-td patent-date-value">Sep 13, 2011</td><td class="patent-data-table-td ">Ibis Biosciences, Inc.</td><td class="patent-data-table-td ">Method for rapid detection and identification of bioagents</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8017743">US8017743</a></td><td class="patent-data-table-td patent-date-value">Oct 30, 2007</td><td class="patent-data-table-td patent-date-value">Sep 13, 2011</td><td class="patent-data-table-td ">Ibis Bioscience, Inc.</td><td class="patent-data-table-td ">Method for rapid detection and identification of bioagents</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8026064">US8026064</a></td><td class="patent-data-table-td patent-date-value">Apr 18, 2008</td><td class="patent-data-table-td patent-date-value">Sep 27, 2011</td><td class="patent-data-table-td ">Metamorphix, Inc.</td><td class="patent-data-table-td ">Compositions, methods and systems for inferring bovine breed</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8026084">US8026084</a></td><td class="patent-data-table-td patent-date-value">Jul 21, 2006</td><td class="patent-data-table-td patent-date-value">Sep 27, 2011</td><td class="patent-data-table-td ">Ibis Biosciences, Inc.</td><td class="patent-data-table-td ">Methods for rapid identification and quantitation of nucleic acid variants</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8046171">US8046171</a></td><td class="patent-data-table-td patent-date-value">Apr 18, 2003</td><td class="patent-data-table-td patent-date-value">Oct 25, 2011</td><td class="patent-data-table-td ">Ibis Biosciences, Inc.</td><td class="patent-data-table-td ">Methods and apparatus for genetic evaluation</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8057993">US8057993</a></td><td class="patent-data-table-td patent-date-value">Apr 22, 2004</td><td class="patent-data-table-td patent-date-value">Nov 15, 2011</td><td class="patent-data-table-td ">Ibis Biosciences, Inc.</td><td class="patent-data-table-td ">Methods for identification of coronaviruses</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8071309">US8071309</a></td><td class="patent-data-table-td patent-date-value">Oct 30, 2007</td><td class="patent-data-table-td patent-date-value">Dec 6, 2011</td><td class="patent-data-table-td ">Ibis Biosciences, Inc.</td><td class="patent-data-table-td ">Methods for rapid identification of pathogens in humans and animals</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8073627">US8073627</a></td><td class="patent-data-table-td patent-date-value">Jan 9, 2004</td><td class="patent-data-table-td patent-date-value">Dec 6, 2011</td><td class="patent-data-table-td ">Ibis Biosciences, Inc.</td><td class="patent-data-table-td ">System for indentification of pathogens</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8084207">US8084207</a></td><td class="patent-data-table-td patent-date-value">Oct 1, 2009</td><td class="patent-data-table-td patent-date-value">Dec 27, 2011</td><td class="patent-data-table-td ">Ibis Bioscience, Inc.</td><td class="patent-data-table-td ">Compositions for use in identification of papillomavirus</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8097416">US8097416</a></td><td class="patent-data-table-td patent-date-value">May 25, 2007</td><td class="patent-data-table-td patent-date-value">Jan 17, 2012</td><td class="patent-data-table-td ">Ibis Biosciences, Inc.</td><td class="patent-data-table-td ">Methods for identification of sepsis-causing bacteria</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8119336">US8119336</a></td><td class="patent-data-table-td patent-date-value">Mar 2, 2005</td><td class="patent-data-table-td patent-date-value">Feb 21, 2012</td><td class="patent-data-table-td ">Ibis Biosciences, Inc.</td><td class="patent-data-table-td ">Compositions for use in identification of alphaviruses</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8148163">US8148163</a></td><td class="patent-data-table-td patent-date-value">Sep 16, 2009</td><td class="patent-data-table-td patent-date-value">Apr 3, 2012</td><td class="patent-data-table-td ">Ibis Biosciences, Inc.</td><td class="patent-data-table-td ">Sample processing units, systems, and related methods</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8158354">US8158354</a></td><td class="patent-data-table-td patent-date-value">Sep 17, 2004</td><td class="patent-data-table-td patent-date-value">Apr 17, 2012</td><td class="patent-data-table-td ">Ibis Biosciences, Inc.</td><td class="patent-data-table-td ">Methods for rapid purification of nucleic acids for subsequent analysis by mass spectrometry by solution capture</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8158936">US8158936</a></td><td class="patent-data-table-td patent-date-value">Feb 12, 2010</td><td class="patent-data-table-td patent-date-value">Apr 17, 2012</td><td class="patent-data-table-td ">Ibis Biosciences, Inc.</td><td class="patent-data-table-td ">Ionization probe assemblies</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8163895">US8163895</a></td><td class="patent-data-table-td patent-date-value">Aug 24, 2005</td><td class="patent-data-table-td patent-date-value">Apr 24, 2012</td><td class="patent-data-table-td ">Ibis Biosciences, Inc.</td><td class="patent-data-table-td ">Compositions for use in identification of orthopoxviruses</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8173957">US8173957</a></td><td class="patent-data-table-td patent-date-value">Jan 26, 2010</td><td class="patent-data-table-td patent-date-value">May 8, 2012</td><td class="patent-data-table-td ">Ibis Biosciences, Inc.</td><td class="patent-data-table-td ">Mass spectrometry with selective ion filtration by digital thresholding</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8182992">US8182992</a></td><td class="patent-data-table-td patent-date-value">Mar 3, 2006</td><td class="patent-data-table-td patent-date-value">May 22, 2012</td><td class="patent-data-table-td ">Ibis Biosciences, Inc.</td><td class="patent-data-table-td ">Compositions for use in identification of adventitious viruses</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8185323">US8185323</a></td><td class="patent-data-table-td patent-date-value">Jan 6, 2008</td><td class="patent-data-table-td patent-date-value">May 22, 2012</td><td class="patent-data-table-td ">Degen Zhuo</td><td class="patent-data-table-td ">Method of identifying exons and introns and diagnostic uses thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8187814">US8187814</a></td><td class="patent-data-table-td patent-date-value">Jan 8, 2010</td><td class="patent-data-table-td patent-date-value">May 29, 2012</td><td class="patent-data-table-td ">Ibis Biosciences, Inc.</td><td class="patent-data-table-td ">Methods for concurrent identification and quantification of an unknown bioagent</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8214154">US8214154</a></td><td class="patent-data-table-td patent-date-value">Oct 30, 2007</td><td class="patent-data-table-td patent-date-value">Jul 3, 2012</td><td class="patent-data-table-td ">Ibis Biosciences, Inc.</td><td class="patent-data-table-td ">Systems for rapid identification of pathogens in humans and animals</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8252599">US8252599</a></td><td class="patent-data-table-td patent-date-value">Aug 30, 2011</td><td class="patent-data-table-td patent-date-value">Aug 28, 2012</td><td class="patent-data-table-td ">Ibis Biosciences, Inc.</td><td class="patent-data-table-td ">Sample processing units, systems, and related methods</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8265878">US8265878</a></td><td class="patent-data-table-td patent-date-value">Oct 30, 2007</td><td class="patent-data-table-td patent-date-value">Sep 11, 2012</td><td class="patent-data-table-td ">Ibis Bioscience, Inc.</td><td class="patent-data-table-td ">Method for rapid detection and identification of bioagents</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8268565">US8268565</a></td><td class="patent-data-table-td patent-date-value">Dec 2, 2008</td><td class="patent-data-table-td patent-date-value">Sep 18, 2012</td><td class="patent-data-table-td ">Ibis Biosciences, Inc.</td><td class="patent-data-table-td ">Methods for identifying bioagents</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8298760">US8298760</a></td><td class="patent-data-table-td patent-date-value">Feb 13, 2007</td><td class="patent-data-table-td patent-date-value">Oct 30, 2012</td><td class="patent-data-table-td ">Ibis Bioscience, Inc.</td><td class="patent-data-table-td ">Secondary structure defining database and methods for determining identity and geographic origin of an unknown bioagent thereby</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8380442">US8380442</a></td><td class="patent-data-table-td patent-date-value">Oct 31, 2007</td><td class="patent-data-table-td patent-date-value">Feb 19, 2013</td><td class="patent-data-table-td ">Ibis Bioscience, Inc.</td><td class="patent-data-table-td ">Secondary structure defining database and methods for determining identity and geographic origin of an unknown bioagent thereby</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8394640">US8394640</a></td><td class="patent-data-table-td patent-date-value">Dec 6, 2010</td><td class="patent-data-table-td patent-date-value">Mar 12, 2013</td><td class="patent-data-table-td ">President And Fellows Of Harvard College</td><td class="patent-data-table-td ">Study of polymer molecules and conformations with a nanopore</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8407010">US8407010</a></td><td class="patent-data-table-td patent-date-value">Mar 17, 2008</td><td class="patent-data-table-td patent-date-value">Mar 26, 2013</td><td class="patent-data-table-td ">Ibis Biosciences, Inc.</td><td class="patent-data-table-td ">Methods for rapid forensic analysis of mitochondrial DNA</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8450064">US8450064</a></td><td class="patent-data-table-td patent-date-value">Mar 8, 2010</td><td class="patent-data-table-td patent-date-value">May 28, 2013</td><td class="patent-data-table-td ">Cargill Incorporated</td><td class="patent-data-table-td ">Methods and systems for inferring bovine traits</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8476415">US8476415</a></td><td class="patent-data-table-td patent-date-value">Apr 16, 2012</td><td class="patent-data-table-td patent-date-value">Jul 2, 2013</td><td class="patent-data-table-td ">Ibis Biosciences, Inc.</td><td class="patent-data-table-td ">Methods for rapid purification of nucleic acids for subsequent analysis by mass spectrometry by solution capture</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8534447">US8534447</a></td><td class="patent-data-table-td patent-date-value">Sep 16, 2009</td><td class="patent-data-table-td patent-date-value">Sep 17, 2013</td><td class="patent-data-table-td ">Ibis Biosciences, Inc.</td><td class="patent-data-table-td ">Microplate handling systems and related computer program products and methods</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8546082">US8546082</a></td><td class="patent-data-table-td patent-date-value">May 25, 2007</td><td class="patent-data-table-td patent-date-value">Oct 1, 2013</td><td class="patent-data-table-td ">Ibis Biosciences, Inc.</td><td class="patent-data-table-td ">Methods for identification of sepsis-causing bacteria</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8550694">US8550694</a></td><td class="patent-data-table-td patent-date-value">Sep 16, 2009</td><td class="patent-data-table-td patent-date-value">Oct 8, 2013</td><td class="patent-data-table-td ">Ibis Biosciences, Inc.</td><td class="patent-data-table-td ">Mixing cartridges, mixing stations, and related kits, systems, and methods</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8551738">US8551738</a></td><td class="patent-data-table-td patent-date-value">Nov 11, 2009</td><td class="patent-data-table-td patent-date-value">Oct 8, 2013</td><td class="patent-data-table-td ">Ibis Biosciences, Inc.</td><td class="patent-data-table-td ">Systems and methods for rapid identification of nucleic acid variants</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8563250">US8563250</a></td><td class="patent-data-table-td patent-date-value">Mar 5, 2007</td><td class="patent-data-table-td patent-date-value">Oct 22, 2013</td><td class="patent-data-table-td ">Ibis Biosciences, Inc.</td><td class="patent-data-table-td ">Methods for identifying bioagents</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8609430">US8609430</a></td><td class="patent-data-table-td patent-date-value">Aug 27, 2012</td><td class="patent-data-table-td patent-date-value">Dec 17, 2013</td><td class="patent-data-table-td ">Ibis Biosciences, Inc.</td><td class="patent-data-table-td ">Sample processing units, systems, and related methods</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8669056">US8669056</a></td><td class="patent-data-table-td patent-date-value">Aug 29, 2011</td><td class="patent-data-table-td patent-date-value">Mar 11, 2014</td><td class="patent-data-table-td ">Cargill Incorporated</td><td class="patent-data-table-td ">Compositions, methods, and systems for inferring bovine breed</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8673556">US8673556</a></td><td class="patent-data-table-td patent-date-value">May 18, 2011</td><td class="patent-data-table-td patent-date-value">Mar 18, 2014</td><td class="patent-data-table-td ">President And Fellows Of Harvard College</td><td class="patent-data-table-td ">Methods and apparatus for characterizing polynucleotides</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2031063A2?cl=en">EP2031063A2</a></td><td class="patent-data-table-td patent-date-value">Jun 20, 2005</td><td class="patent-data-table-td patent-date-value">Mar 4, 2009</td><td class="patent-data-table-td ">Duke University</td><td class="patent-data-table-td ">Modulators of odorant receptors</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1999023248A1?cl=en">WO1999023248A1</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Oct 23, 1998</td><td class="patent-data-table-td patent-date-value">May 14, 1999</td><td class="patent-data-table-td ">William John Barendse</td><td class="patent-data-table-td ">Assessing lipid metabolism</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="classifications"></a><div class="patent-section-header"><span class="patent-section-title">Classifications</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th"> </th><th class="patent-data-table-th"> </th></tr></thead><tr><td class="patent-data-table-td ">U.S. Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=bhJCBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S006110">435/6.11</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=bhJCBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S023100">536/23.1</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=bhJCBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S024310">536/24.31</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=bhJCBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S091200">435/91.2</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=bhJCBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S024330">536/24.33</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=bhJCBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S024300">536/24.3</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=bhJCBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S091100">435/91.1</a></span></td></tr><tr><td class="patent-data-table-td ">International Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=bhJCBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=A61K0048000000">A61K48/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=bhJCBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0015090000">C12N15/09</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=bhJCBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12Q0001680000">C12Q1/68</a></span></td></tr><tr><td class="patent-data-table-td ">Cooperative Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=bhJCBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12Q1/6881">C12Q1/6881</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=bhJCBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12Q1/683">C12Q1/683</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=bhJCBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12Q1/6858">C12Q1/6858</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=bhJCBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12Q1/6827">C12Q1/6827</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=bhJCBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12Q1/6883">C12Q1/6883</a></span></td></tr><tr><td class="patent-data-table-td ">European Classification</td><td class="patent-data-table-td "><span class="nested-value">C12Q1/68M6</span>, <span class="nested-value">C12Q1/68B6</span>, <span class="nested-value">C12Q1/68D2G</span>, <span class="nested-value">C12Q1/68B6A</span>, <span class="nested-value">C12Q1/68M4</span></td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="legal-events"></a><div class="patent-section-header"><span class="patent-section-title">Legal Events</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Date</th><th class="patent-data-table-th">Code</th><th class="patent-data-table-th">Event</th><th class="patent-data-table-th">Description</th></tr></thead><tr><td class="patent-data-table-td patent-date-value">Feb 18, 2014</td><td class="patent-data-table-td ">B3</td><td class="patent-data-table-td ">Reexamination certificate third or further reexamination</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">THE PATENTABILITY OF CLAIMS 1-15, 17, 18, 26-29 AND 32 IS CONFIRMED.CLAIMS 16, 19-25, 30-31 AND 33-36 WERE NOT REEXAMINED.</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Oct 29, 2013</td><td class="patent-data-table-td ">RR</td><td class="patent-data-table-td ">Request for reexamination filed</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20130905</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Sep 24, 2013</td><td class="patent-data-table-td ">B3</td><td class="patent-data-table-td ">Reexamination certificate third or further reexamination</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">THE PATENTABILITY OF CLAIMS 1-18 AND 26-32 IS CONFIRMED.CLAIMS 19-25 AND 33-36 WERE NOT REEXAMINED.</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Apr 9, 2013</td><td class="patent-data-table-td ">RR</td><td class="patent-data-table-td ">Request for reexamination filed</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20130301</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Apr 2, 2013</td><td class="patent-data-table-td ">B2</td><td class="patent-data-table-td ">Reexamination certificate second reexamination</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">THE PATENTABILITY OF CLAIMS 1-18 AND 26-32 IS CONFIRMED.CLAIMS 19-25 AND 33-36 WERE NOT REEXAMINED.</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Jun 19, 2012</td><td class="patent-data-table-td ">RR</td><td class="patent-data-table-td ">Request for reexamination filed</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20120510</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">May 4, 2010</td><td class="patent-data-table-td ">B1</td><td class="patent-data-table-td ">Reexamination certificate first reexamination</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">THE PATENTABILITY OF CLAIMS 26-32 IS CONFIRMED. CLAIMS 1-25 AND 33-36 WERE NOT REEXAMINED.</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Dec 2, 2008</td><td class="patent-data-table-td ">RR</td><td class="patent-data-table-td ">Request for reexamination filed</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20081015</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Aug 19, 2008</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">12</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Aug 25, 2004</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">8</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Nov 13, 2002</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">GENETIC TECHNOLOGIES LIMITED, AUSTRALIA</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNOR:GENETYPE A.G.;REEL/FRAME:013484/0039</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20021031</span></div><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">GENETIC TECHNOLOGIES LIMITED 60-66 HANOVER STREETF</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNOR:GENETYPE A.G. /AR;REEL/FRAME:013484/0039</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Feb 15, 2001</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">4</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Feb 15, 2001</td><td class="patent-data-table-td ">SULP</td><td class="patent-data-table-td ">Surcharge for late payment</td><td class="patent-data-table-td "></td></tr><tr><td class="patent-data-table-td patent-date-value">Oct 10, 2000</td><td class="patent-data-table-td ">REMI</td><td class="patent-data-table-td ">Maintenance fee reminder mailed</td><td class="patent-data-table-td "></td></tr></table><div class="patent-section-footer"></div></div><div class="modal-dialog" id="patent-images-lightbox"><div class="patent-lightbox-controls"><div class="patent-lightbox-rotate-controls"><div class="patent-lightbox-rotation-text">Rotate</div><div class="rotate-icon rotate-ccw-icon"></div><div class="rotate-icon rotate-cw-icon"></div></div><div class="patent-lightbox-index-counter"></div><a class="patent-lightbox-fullsize-link" target="_blank">Original Image</a><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_right.png" alt="Next page"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_left.png" alt="Previous page"width="21" height="21" /></div></div><div class="modal-dialog-content"><div class="patent-lightbox-image-holder"><div class="patent-lightbox-placeholder"></div></div></div></div><script>_OC_initPatentsAtb({image_not_available_html: " Image not available"});</script></div></div></div></td></tr></table><script>(function() {var href = window.location.href;if (href.indexOf('?') !== -1) {var parameters = href.split('?')[1].split('&');for (var i = 0; i < parameters.length; i++) {var param = parameters[i].split('=');if (param[0] == 'focus') {var elem = document.getElementById(param[1]);if (elem) {elem.focus();}}}}})();</script><script>_OC_addFlags({LockSrc:"/books/javascript/lock_8a2b04e7bf975d5171d8e4c0b6365c7a.js", Host:"http://www.google.com/", IsBooksRentalEnabled:1, IsWebstoreDisplayCaseEnabled:1, IsObfuscationEnabled:1, IsBrowsingHistoryEnabled:1, IsWebReaderSvgEnabled:0, IsGeoLayerEnabled:1, IsImageModeNotesEnabled:1, IsCopyMenuItemEnabled:1, IsGiftingEnabled:0, IsWebReaderUniversalPaginatorEnabled:0, IsOfflineBubbleEnabled:1, IsReaderEnabledForPlayRequests:1, IsFutureOnSaleVolumesEnabled:1, IsOfflineRestrictedCopyEnabled:1, IsBooksUnifiedLeftNavEnabled:1, IsRestrictedCopyEnabled:1, IsZipitFolderCollectionEnabled:1, IsEndOfSampleRecommendationsEnabled:1, IsRatingsOnBookcardsEnabled:1, IsAdsDisabled:0, IsIframePageDisplayEnabled:0, IsEmbeddedMediaEnabled:1, IsImageModeAnnotationsEnabled:1, IsMyLibraryGooglePlusEnabled:1, IsImagePageProviderEnabled:0, IsBookcardListPriceSmall:0, IsInternalUser:0, IsBooksShareButtonEnabled:0, IsPreOrdersEnabled:0, IsDisabledRandomBookshelves:0, WebstoreDisplayCasePosition:3});_OC_Run({"enable_p13n":false,"add_vol_to_collection_base_url":"http://www.google.com/patents?op=add\u0026sig=ACfU3U1H4rHHnNdhgj4qyzxGQoVCPY-4Kg\u0026id=bhJCBAABERAJ","remove_vol_from_collection_base_url":"http://www.google.com/patents?op=remove\u0026sig=ACfU3U2r8B43FOhbqwpVF-qxD3ZEVWwT7A\u0026id=bhJCBAABERAJ","logged_in":false,"p13n_save_user_settings_url":"http://www.google.com/patents?op=edit_user_settings\u0026sig=ACfU3U02kY6S0dqJZy0iO32syHzaMKeMpg","is_cobrand":false,"sign_in_url":"https://www.google.com/accounts/Login?service=\u0026continue=http://www.google.com/patents%3Fhl%3Den\u0026hl=en","is_play_enabled":true}, {"volume_id":"","is_ebook":true,"volumeresult":{"has_flowing_text":false,"has_scanned_text":true,"can_download_pdf":false,"can_download_epub":false,"is_pdf_drm_enabled":false,"is_epub_drm_enabled":false,"download_pdf_url":"http://www.google.com/patents/download/Intron_sequence_analysis_method_for_dete.pdf?id=bhJCBAABERAJ\u0026output=pdf\u0026sig=ACfU3U0hBVwt1jM_ffq6oUWYlHimcEI9gA"},"sample_url":"http://www.google.com/patents/reader?id=bhJCBAABERAJ\u0026printsec=frontcover\u0026output=reader\u0026source=gbs_atb_hover","is_browsable":true,"is_public_domain":true}, {});</script><div id="footer_table" style="font-size:83%;text-align:center;position:relative;top:20px;height:4.5em;margin-top:2em"><div style="margin-bottom:8px"><a href=http://www.google.com/><nobr>Google&nbsp;Home</nobr></a> - <a href=//www.google.com/patents/sitemap/><nobr>Sitemap</nobr></a> - <a href=http://www.google.com/googlebooks/uspto.html><nobr>USPTO Bulk Downloads</nobr></a> - <a href=/intl/en/privacy/><nobr>Privacy Policy</nobr></a> - <a href=/intl/en/policies/terms/><nobr>Terms of Service</nobr></a> - <a href=https://support.google.com/faqs/answer/2539193?hl=en><nobr>About Google Patents</nobr></a> - <a href="http://www.google.com/tools/feedback/intl/en/error.html" onclick="try{_OC_startFeedback({productId: '72792',locale: 'en'});return false;}catch(e){}"><nobr>Send Feedback</nobr></a></div><span>Data provided by IFI CLAIMS Patent Services</span><br><span >&copy;2012 Google</span></div> <script type="text/javascript">var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));</script><script type="text/javascript">var pageTracker = _gat._getTracker("UA-27188110-1");pageTracker._setCookiePath("/patents/");pageTracker._trackPageview();</script> </body></html>