<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN""http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html><head><title>Patent US5624803 - In vivo oligonucleotide generator, and methods of testing the binding ... - Google Patents</title><script>(function(){(function(){function e(a){this.t={};this.tick=function(a,c,b){var d=void 0!=b?b:(new Date).getTime();this.t[a]=[d,c];if(void 0==b)try{window.console.timeStamp("CSI/"+a)}catch(e){}};this.tick("start",null,a)}var a;window.performance&&(a=window.performance.timing);var f=a?new e(a.responseStart):new e;window.jstiming={Timer:e,load:f};if(a){var c=a.navigationStart,d=a.responseStart;0<c&&d>=c&&(window.jstiming.srt=d-c)}if(a){var b=window.jstiming.load;0<c&&d>=c&&(b.tick("_wtsrt",void 0,c),b.tick("wtsrt_",
"_wtsrt",d),b.tick("tbsd_","wtsrt_"))}try{a=null,window.chrome&&window.chrome.csi&&(a=Math.floor(window.chrome.csi().pageT),b&&0<c&&(b.tick("_tbnd",void 0,window.chrome.csi().startE),b.tick("tbnd_","_tbnd",c))),null==a&&window.gtbExternal&&(a=window.gtbExternal.pageT()),null==a&&window.external&&(a=window.external.pageT,b&&0<c&&(b.tick("_tbnd",void 0,window.external.startE),b.tick("tbnd_","_tbnd",c))),a&&(window.jstiming.pt=a)}catch(g){}})();})();
</script><link rel="stylesheet" href="/patents/css/_8a2b04e7bf975d5171d8e4c0b6365c7a/kl_intl_patents_bundle.css" type="text/css" /><script src="/books/javascript/atb_8a2b04e7bf975d5171d8e4c0b6365c7a__en.js"></script><script>function googleTranslateElementInit() {new google.translate.TranslateElement({pageLanguage: "en",gaTrack: true,gaId: "UA-27188110-1",multilanguagePage: true});}</script><script src="//translate.google.com/translate_a/element.js?cb=googleTranslateElementInit"></script><meta name="DC.type" content="Patent"><meta name="DC.title" content="In vivo oligonucleotide generator, and methods of testing the binding affinity of triplex forming oligonucleotides derived therefrom"><meta name="DC.contributor" content="Sarah B. Noonberg" scheme="inventor"><meta name="DC.contributor" content="C. Anthony Hunt" scheme="inventor"><meta name="DC.contributor" content="The Regents Of The University Of California" scheme="assignee"><meta name="DC.date" content="1994-10-13" scheme="dateSubmitted"><meta name="DC.description" content="The present invention encompasses improved methods and materials for the delivering of antisense, triplex, and/or ribozyme oligonucleotides intracellularly, and RNA polymerase III-based constructs termed &quot;oligonucleotide generators&quot; to accomplish the delivery of oligonucleotides. Also encompassed by the present invention are methods for screening oligonucleotide sequences that are candidates for triplex formation."><meta name="DC.date" content="1997-4-29" scheme="issued"><meta name="DC.relation" content="EP:0601585:A2" scheme="references"><meta name="DC.relation" content="US:5190931" scheme="references"><meta name="DC.relation" content="US:5272065" scheme="references"><meta name="DC.relation" content="US:5316930" scheme="references"><meta name="DC.relation" content="WO:1995006718:A2" scheme="references"><meta name="DC.relation" content="WO:1995006744:A2" scheme="references"><meta name="citation_reference" content="Case et al., &quot;The unusual stability of the IS10 anti-sense RNA is critical for its function and is determined by the structure of its stem-domain&quot; EMBO J. (1989) 8:4297-4305."><meta name="citation_reference" content="Case et al., The unusual stability of the IS10 anti sense RNA is critical for its function and is determined by the structure of its stem domain EMBO J. (1989) 8:4297 4305."><meta name="citation_reference" content="Chrisey et al., &quot;Antisense technology: Principles and prospects for therapeutic development&quot; BioPharm (1991) pp. 36-42."><meta name="citation_reference" content="Chrisey et al., Antisense technology: Principles and prospects for therapeutic development BioPharm (1991) pp. 36 42."><meta name="citation_reference" content="Cooney et al., &quot;Site-specific oligonucleotide binding represses transcription of the human c-myc gene in vitro&quot; Science (1988) 241:456-459."><meta name="citation_reference" content="Cooney et al., Site specific oligonucleotide binding represses transcription of the human c myc gene in vitro Science (1988) 241:456 459."><meta name="citation_reference" content="Das et al., &quot;Upstream regulatory elements are necessary and sufficient for transcription of a U6 RNA gene by RNA polymerase III&quot; EMBO J. (1988) 7:503-512."><meta name="citation_reference" content="Das et al., Upstream regulatory elements are necessary and sufficient for transcription of a U6 RNA gene by RNA polymerase III EMBO J. (1988) 7:503 512."><meta name="citation_reference" content="Durland et al., &quot;Binding of triple helix forming oligonucleotides to sites in gene promoters&quot; Biochem. (1991) 30:9246-9255."><meta name="citation_reference" content="Durland et al., Binding of triple helix forming oligonucleotides to sites in gene promoters Biochem. (1991) 30:9246 9255."><meta name="citation_reference" content="Duval Valentin et al., Specific inhibition of transcription by triple helix forming oligonucleotides Proc. Natl. Acad. Sci. USA (1992) 89:504 508."><meta name="citation_reference" content="Duval-Valentin et al., &quot;Specific inhibition of transcription by triple helix-forming oligonucleotides&quot; Proc. Natl. Acad. Sci. USA (1992) 89:504-508."><meta name="citation_reference" content="H e l e ne, The anti gene strategy: control of gene expression by triplex forming oligonucleotides Anti Cancer Drug Design (1991) 6:569 584."><meta name="citation_reference" content="H el ene, &quot;The anti-gene strategy: control of gene expression by triplex-forming-oligonucleotides&quot; Anti-Cancer Drug Design (1991) 6:569-584."><meta name="citation_reference" content="Hannon et al., &quot;Multiple cis-acting elements are required for RNA polymerase III transcription of the gene encoding H1 RNA, the RNA component of human RNase P&quot; J. Biol. Chem. (1991) 266:22796-22799."><meta name="citation_reference" content="Hannon et al., Multiple cis acting elements are required for RNA polymerase III transcription of the gene encoding H1 RNA, the RNA component of human RNase P J. Biol. Chem. (1991) 266:22796 22799."><meta name="citation_reference" content="Izant, &quot;Chimeric antisense RNAs&quot; Gene Regulation: Biology of Antisense RNA and DNA (1992) Erickson, R.P. et al., eds., Raven Press, New York, pp. 183-195."><meta name="citation_reference" content="Izant, Chimeric antisense RNAs Gene Regulation: Biology of Antisense RNA and DNA (1992) Erickson, R.P. et al., eds., Raven Press, New York, pp. 183 195."><meta name="citation_reference" content="Jennings et al., &quot;Inhibition of SV40 replicon function by engineered antisense RNA transcribed by RNA polymerase III&quot; EMBO J. (1987) 6:3043-3047."><meta name="citation_reference" content="Jennings et al., Inhibition of SV40 replicon function by engineered antisense RNA transcribed by RNA polymerase III EMBO J. (1987) 6:3043 3047."><meta name="citation_reference" content="Junker et al., &quot;Reduction in replication of the human immunodeficiency virus type 1 in human cell lines by polymerase III-driven transcription of chimeric tRNA-antisense RNA genes&quot; Antisense Res. &amp; Develop. (1994) 4:165-172."><meta name="citation_reference" content="Junker et al., Reduction in replication of the human immunodeficiency virus type 1 in human cell lines by polymerase III driven transcription of chimeric tRNA antisense RNA genes Antisense Res. &amp; Develop. (1994) 4:165 172."><meta name="citation_reference" content="Kunkel et al., &quot;Transcription of a human U6 small nuclear RNA gene in vivo withstands deletion of intragenic sequences but not for an upstream TATATA box&quot; Nucleic Acids Res. (1989) 17:7371-7379."><meta name="citation_reference" content="Kunkel et al., &quot;U6 small nuclear RNA is transcribed by RNA polymerase III&quot; Proc. Natl. Acad. USA (1986) 83:8575-8579."><meta name="citation_reference" content="Kunkel et al., Transcription of a human U6 small nuclear RNA gene in vivo withstands deletion of intragenic sequences but not for an upstream TATATA box Nucleic Acids Res. (1989) 17:7371 7379."><meta name="citation_reference" content="Kunkel et al., U6 small nuclear RNA is transcribed by RNA polymerase III Proc. Natl. Acad. USA (1986) 83:8575 8579."><meta name="citation_reference" content="Lyamichev et al, Nucleic Acids Res. 16: 2165 (1988)."><meta name="citation_reference" content="Marshallsay et al., &quot;Characterization of the U3 and U6 snRNA genes from wheat: U3snRNA genes in monocot plants are transcribed by RNA polymerase III&quot; Plant Mol. Biol. (1992) 19:973-983."><meta name="citation_reference" content="Marshallsay et al., Characterization of the U3 and U6 snRNA genes from wheat: U3snRNA genes in monocot plants are transcribed by RNA polymerase III Plant Mol. Biol. (1992) 19:973 983."><meta name="citation_reference" content="Moffat, &quot;Making sense of antisense&quot; Science (1991) 253:510-511."><meta name="citation_reference" content="Moffat, Making sense of antisense Science (1991) 253:510 511."><meta name="citation_reference" content="Murphy et al., &quot;The in vitro transcription of the 7SK RNA gene by RNA polymerase III is dependent only on the presence of an upstream promoter&quot; Cell (1987) 51:81-87."><meta name="citation_reference" content="Murphy et al., The in vitro transcription of the 7SK RNA gene by RNA polymerase III is dependent only on the presence of an upstream promoter Cell (1987) 51:81 87."><meta name="citation_reference" content="Noonberg et al., &quot;Detection of triplex-forming RNA oligonucleotides by triplex blotting&quot; BioTechniques (1994) 16:1070-1072."><meta name="citation_reference" content="Noonberg et al., &quot;In vivo generation of highly abundant sequence-specific oligonucleotides for antisense and triplex gene regulation&quot; Nucleic Acids Res. (1994) 22:2830-2836."><meta name="citation_reference" content="Noonberg et al., Detection of triplex forming RNA oligonucleotides by triplex blotting BioTechniques (1994) 16:1070 1072."><meta name="citation_reference" content="Noonberg et al., In vivo generation of highly abundant sequence specific oligonucleotides for antisense and triplex gene regulation Nucleic Acids Res. (1994) 22:2830 2836."><meta name="citation_reference" content="Sullenger et al., &quot;Expression of chimeric tRNA-driven antisense transcripts renders NIH 3T3 cells highly resistant to Moloney murine leukemia virus replication&quot; Mol. &amp; Cell. Biol. (1990) 10:6512-6523."><meta name="citation_reference" content="Sullenger et al., Expression of chimeric tRNA driven antisense transcripts renders NIH 3T3 cells highly resistant to Moloney murine leukemia virus replication Mol. &amp; Cell. Biol. (1990) 10:6512 6523."><meta name="citation_reference" content="Terns et al., &quot;Multiple cis-acting signals for export of pre-U1 snRNA from the nucleus&quot; Genes &amp; Development (1993) 7:1898-1908."><meta name="citation_reference" content="Terns et al., Multiple cis acting signals for export of pre U1 snRNA from the nucleus Genes &amp; Development (1993) 7:1898 1908."><meta name="citation_reference" content="Williard et al., &quot;Paradoxical production of target protein using antisense RNA expression vectors&quot; Gene (1994) 149:21-24."><meta name="citation_reference" content="Williard et al., Paradoxical production of target protein using antisense RNA expression vectors Gene (1994) 149:21 24."><meta name="citation_reference" content="Willis, &quot;RNA polymerase III. Genes, factors and transcriptional specificity&quot; Eur. J. Biochem. (1993) 212:1-11."><meta name="citation_reference" content="Willis, RNA polymerase III. Genes, factors and transcriptional specificity Eur. J. Biochem. (1993) 212:1 11."><meta name="citation_reference" content="Yuan et al., &quot;5&#39; flanking sequences of human MRP/7-2 RNA gene are required and sufficient for the transcription by RNA polymerase III&quot; Biochim. Biophys. Acta (1991) 1089:33-39."><meta name="citation_reference" content="Yuan et al., 5 flanking sequences of human MRP/7 2 RNA gene are required and sufficient for the transcription by RNA polymerase III Biochim. Biophys. Acta (1991) 1089:33 39."><meta name="citation_patent_number" content="US:5624803"><meta name="citation_patent_application_number" content="US:08/324,001"><link rel="canonical" href="http://www.google.com/patents/US5624803"/><meta property="og:url" content="http://www.google.com/patents/US5624803"/><meta name="title" content="Patent US5624803 - In vivo oligonucleotide generator, and methods of testing the binding affinity of triplex forming oligonucleotides derived therefrom"/><meta name="description" content="The present invention encompasses improved methods and materials for the delivering of antisense, triplex, and/or ribozyme oligonucleotides intracellularly, and RNA polymerase III-based constructs termed &quot;oligonucleotide generators&quot; to accomplish the delivery of oligonucleotides. Also encompassed by the present invention are methods for screening oligonucleotide sequences that are candidates for triplex formation."/><meta property="og:title" content="Patent US5624803 - In vivo oligonucleotide generator, and methods of testing the binding affinity of triplex forming oligonucleotides derived therefrom"/><meta property="og:type" content="book"/><meta property="og:site_name" content="Google Books"/><meta property="og:image" content="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><link rel="image_src" href="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><script>(function(){try{var aa=function(a,b,c,d){d=d||{};d._sn=["cfg",b,c].join(".");window.gbar.logger.ml(a,d)};var g=window.gbar=window.gbar||{},l=window.gbar.i=window.gbar.i||{},m={},n;function _tvn(a,b){var c=parseInt(a,10);return isNaN(c)?b:c}function _tvf(a,b){var c=parseFloat(a);return isNaN(c)?b:c}function _tvv(a){return!!a}function p(a,b,c){(c||g)[a]=b}g.bv={n:_tvn("2",0),r:"",f:".67.",e:"0",m:_tvn("0",1)};
function q(a,b,c){var d="on"+b;if(a.addEventListener)a.addEventListener(b,c,!1);else if(a.attachEvent)a.attachEvent(d,c);else{var f=a[d];a[d]=function(){var a=f.apply(this,arguments),b=c.apply(this,arguments);return void 0==a?b:void 0==b?a:b&&a}}}var s=function(a){return function(){return g.bv.m==a}},ba=s(1),ca=s(2);p("sb",ba);p("kn",ca);l.a=_tvv;l.b=_tvf;l.c=_tvn;l.i=aa;var da=window.gbar.i.i;var t,u,v,w;function ea(a){v=a}function fa(a){var b;if(b=v&&window.encodeURIComponent)b=a.href,b=!b.match(/^http[s]?:\/\/accounts\.google\.[^/]*\/ClearSID/i)&&!b.match(/^http[s]?:\/\/[^/]*\/accounts\/ClearSID/i);if(b=b&&encodeURIComponent(v()))a.href=a.href.replace(/([?&]continue=)[^&]*/,"$1"+b)}function ga(a){window.gApplication&&(a.href=window.gApplication.getTabUrl(a.href))}
function ha(a){var b=document.forms[0].q,c=window.encodeURIComponent&&b&&b.value,b=b&&b.placeholder;c&&c!=b&&(a.href=a.href.replace(/([?&])q=[^&]*|$/,function(a,b){return(b||"&")+"q="+encodeURIComponent(c)}))}n=l.a("")?ga:ha;
function x(a,b,c,d,f,e){var h=document.getElementById(a);if(h){var k=h.style;k.left=d?"auto":b+"px";k.right=d?b+"px":"auto";k.top=c+"px";k.visibility=u?"hidden":"visible";f&&e?(k.width=f+"px",k.height=e+"px"):(x(t,b,c,d,h.offsetWidth,h.offsetHeight),u=u?"":a)}}
var y=[],ia=function(a,b){y.push(b)},ja=function(a){a=a||window.event;var b=a.target||a.srcElement;a.cancelBubble=!0;null==t&&(a=document.createElement(Array.every||window.createPopup?"iframe":"div"),a.frameBorder="0",t=a.id="gbs",a.src="javascript:''",b.parentNode.appendChild(a),q(document,"click",z));var c=b,b=0;"gb3"!=c.className&&(c=c.parentNode);a=c.getAttribute("aria-owns")||"gbi";var d=c.offsetWidth,f=20<c.offsetTop?46:24;document.getElementById("tphdr")&&(f-=3);var e=!1;do b+=c.offsetLeft||
0;while(c=c.offsetParent);var c=(document.documentElement.clientWidth||document.body.clientWidth)-b-d,h,d=document.body,k=document.defaultView;k&&k.getComputedStyle?(d=k.getComputedStyle(d,""))&&(h=d.direction):h=d.currentStyle?d.currentStyle.direction:d.style.direction;h="rtl"==h;if("gbi"==a){for(d=0;k=y[d++];)k();A(null,window.navExtra);h&&(b=c,e=!0)}else h||(b=c,e=!0);u!=a&&z();x(a,b,f,e)},z=function(){u&&x(u,0,0)},A=function(a,b){var c,d=document.getElementById("gbi"),f=a;f||(f=d.firstChild);
for(;b&&(c=b.pop());){var e=d,h=c,k=f;w||(w="gb2");e.insertBefore(h,k).className=w}},ka=function(a,b,c){if((b=document.getElementById(b))&&a){a.className="gb4";var d=document.createElement("span");d.appendChild(a);d.appendChild(document.createTextNode(" | "));d.id=c;b.appendChild(d)}},la=function(){return document.getElementById("gb_70")},ma=function(){return!!u};p("qs",n);p("setContinueCb",ea);p("pc",fa);p("tg",ja);p("close",z);p("addLink",ka);p("almm",A);p("si",la);p("adh",ia);p("op",ma);var B=function(){},C=function(){},F=function(a){var b=new Image,c=D;b.onerror=b.onload=b.onabort=function(){try{delete E[c]}catch(a){}};E[c]=b;b.src=a;D=c+1},E=[],D=0;p("logger",{il:C,ml:B,log:F});var G=window.gbar.logger;var H={},na={},I=[],oa=l.b("0.1",.1),pa=l.a("1",!0),qa=function(a,b){I.push([a,b])},ra=function(a,b){H[a]=b},sa=function(a){return a in H},J={},K=function(a,b){J[a]||(J[a]=[]);J[a].push(b)},ta=function(a){K("m",a)},L=function(a,b){var c=document.createElement("script");c.src=a;c.async=pa;Math.random()<oa&&(c.onerror=function(){c.onerror=null;B(Error("Bundle load failed: name="+(b||"UNK")+" url="+a))});(document.getElementById("xjsc")||document.getElementsByTagName("body")[0]||
document.getElementsByTagName("head")[0]).appendChild(c)},N=function(a){for(var b=0,c;(c=I[b])&&c[0]!=a;++b);!c||c[1].l||c[1].s||(c[1].s=!0,M(2,a),c[1].url&&L(c[1].url,a),c[1].libs&&m.d&&m.d(c[1].libs))},O=function(a){K("gc",a)},P=null,ua=function(a){P=a},M=function(a,b,c){if(P){a={t:a,b:b};if(c)for(var d in c)a[d]=c[d];try{P(a)}catch(f){}}};p("mdc",H);p("mdi",na);p("bnc",I);p("qGC",O);p("qm",ta);p("qd",J);p("lb",N);p("mcf",ra);p("bcf",qa);p("aq",K);p("mdd","");p("has",sa);
p("trh",ua);p("tev",M);var Q=l.b("0.1",.001),R=0;
function _mlToken(a,b){try{if(1>R){R++;var c,d=a,f=b||{},e=encodeURIComponent,h=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&jexpid=",e("17483"),"&srcpg=",e("prop=22"),"&jsr=",Math.round(1/Q),"&ogev=",e("OinuU-zBO-a-sQT3u4CwAw"),"&ogf=",g.bv.f,"&ogrp=",e("1"),"&ogv=",e("1407723702.0"),"&oggv="+e("es_plusone_gc_20140723.0_p0"),"&ogd=",e("com"),"&ogc=",e("PRT"),"&ogl=",e("en")];f._sn&&(f._sn="og."+
f._sn);for(var k in f)h.push("&"),h.push(e(k)),h.push("="),h.push(e(f[k]));h.push("&emsg=");h.push(e(d.name+":"+d.message));var r=h.join("");S(r)&&(r=r.substr(0,2E3));c=r;var Aa=window.gbar.logger._aem(a,c);F(Aa)}}catch(Na){}}var S=function(a){return 2E3<=a.length},va=function(a,b){return b};function T(a){B=a;p("_itl",S,G);p("_aem",va,G);p("ml",B,G);a={};H.er=a}l.a("")?T(function(a){throw a;}):l.a("1")&&Math.random()<Q&&T(_mlToken);I.push(["m",{url:"//ssl.gstatic.com/gb/js/scm_7385cc5883250b43a39405734c1bea59.js"}]);g.mcf("c",{});g.sg={c:""};if(l.a("1")){var wa=l.a("");I.push(["gc",{auto:wa,url:"//ssl.gstatic.com/gb/js/abc/gci_91f30755d6a6b787dcc2a4062e6e9824.js",libs:"googleapis.client:plusone:gapi.iframes"}]);var xa={version:"gci_91f30755d6a6b787dcc2a4062e6e9824.js",index:"",lang:"en"};H.gc=xa;var U=function(a){window.googleapis&&window.iframes?a&&a():(a&&O(a),N("gc"))};p("lGC",U);l.a("1")&&p("lPWF",U)};window.__PVT="";if(l.a("1")&&l.a("1")){var V=function(a){U(function(){K("pw",a);N("pw")})};p("lPW",V);I.push(["pw",{url:"//ssl.gstatic.com/gb/js/abc/pwm_45f73e4df07a0e388b0fa1f3d30e7280.js"}]);var W=[],ya=function(a){W[0]=a},za=function(a,b){var c=b||{};c._sn="pw";B(a,c)},Ba={signed:W,elog:za,base:"https://plusone.google.com/u/0",loadTime:(new Date).getTime()};H.pw=Ba;var X=function(a,b){for(var c=b.split("."),d=function(){var b=arguments;a(function(){for(var a=g,d=0,e=c.length-1;d<e;++d)a=a[c[d]];a[c[d]].apply(a,b)})},f=g,e=0,h=c.length-1;e<h;++e)f=
f[c[e]]=f[c[e]]||{};return f[c[e]]=d};X(V,"pw.clk");X(V,"pw.hvr");p("su",ya,g.pw)};function Ca(){function a(){for(var b;(b=e[h++])&&"m"!=b[0]&&!b[1].auto;);b&&(M(2,b[0]),b[1].url&&L(b[1].url,b[0]),b[1].libs&&m.d&&m.d(b[1].libs));h<e.length&&setTimeout(a,0)}function b(){0<f--?setTimeout(b,0):a()}var c=l.a("1"),d=l.a(""),f=3,e=I,h=0,k=window.gbarOnReady;if(k)try{k()}catch(r){da(r,"ml","or")}d?p("ldb",a):c?q(window,"load",b):b()}p("rdl",Ca);var Da={D:1,H:2,da:3,p:4,W:5,M:6,F:7,g:8,ha:9,U:10,L:11,T:12,S:13,N:14,Q:15,P:16,fa:17,w:18,O:19,ga:20,ea:21,u:22,G:23,ja:24,ka:25,ia:26,A:27,j:28,o:29,k:30,ca:31,Z:32,$:33,J:34,K:35,ba:36,aa:37,Y:38,B:39,R:40,v:41,X:42,V:43,h:48,C:49,I:500},Y=[1,2,3,4,5,6,9,10,11,13,14,28,29,30,34,35,37,38,39,40,41,42,43,48,49,500];var Z=l.b("0.001",1E-4),Ea=l.b("1",1),Fa=!1,Ga=!1;if(l.a("1")){var Ha=Math.random();Ha<=Z&&(Fa=!0);Ha<=Ea&&(Ga=!0)}var Ia=Da,$=null;function Ja(){var a=0,b=function(b,d){l.a(d)&&(a|=b)};b(1,"");b(2,"");b(4,"");b(8,"");return a}
function Ka(a,b){var c=Z,d=Fa,f;f=a;if(!$){$={};for(var e=0;e<Y.length;e++){var h=Y[e];$[h]=!0}}if(f=!!$[f])c=Ea,d=Ga;if(d){d=encodeURIComponent;g.rp?(f=g.rp(),f="-1"!=f?f:"1"):f="1";c=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&oge=",a,"&ogex=",d("17483"),"&ogev=",d("OinuU-zBO-a-sQT3u4CwAw"),"&ogf=",g.bv.f,"&ogp=",d("22"),"&ogrp=",d(f),"&ogsr=",Math.round(1/c),"&ogv=",d("1407723702.0"),"&oggv="+
d("es_plusone_gc_20140723.0_p0"),"&ogd=",d("com"),"&ogl=",d("en"),"&ogc=",d("PRT"),"&ogus=",Ja()];if(b){"ogw"in b&&(c.push("&ogw="+b.ogw),delete b.ogw);var k;f=b;e=[];for(k in f)0!=e.length&&e.push(","),e.push(La(k)),e.push("."),e.push(La(f[k]));k=e.join("");""!=k&&(c.push("&ogad="),c.push(d(k)))}F(c.join(""))}}function La(a){"number"==typeof a&&(a+="");return"string"==typeof a?a.replace(".","%2E").replace(",","%2C"):a}C=Ka;p("il",C,G);var Ma={};H.il=Ma;setTimeout(function(){C(Ia.g)},0);}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var b=window.gbar.i.i;var c=window.gbar;var f=function(d){try{var a=document.getElementById("gbom");a&&d.appendChild(a.cloneNode(!0))}catch(e){b(e,"omas","aomc")}};c.aomc=f;}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var a=window.gbar;a.mcf("pm",{p:""});}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{window.gbar.rdl();}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
if (window['_OC_timingAction']) {window['_OC_timingAction']('patents_refpage');}</script><style>#gbar,#guser{font-size:13px;padding-top:1px !important;}#gbar{float:left;height:22px}#guser{padding-bottom:7px !important;text-align:right}.gbh,.gbd{border-top:1px solid #c9d7f1;font-size:1px}.gbh{height:0;position:absolute;top:24px;width:100%}#gbs,.gbm{background:#fff;left:0;position:absolute;text-align:left;visibility:hidden;z-index:1000}.gbm{border:1px solid;border-color:#c9d7f1 #36c #36c #a2bae7;z-index:1001}.gb1{margin-right:.5em}.gb1,.gb3{zoom:1}.gb2{display:block;padding:.2em .5em}.gb2,.gb3{text-decoration:none !important;border-bottom:none}a.gb1,a.gb4{text-decoration:underline !important}a.gb1,a.gb2,a.gb3,a.gb4{color:#00c !important}.gbi .gb3,.gbi .gb2,.gbi .gb4{color:#dd8e27 !important}.gbf .gb3,.gbf .gb2,.gbf .gb4{color:#900 !important}a.gb2:hover{background:#36c;color:#fff !important}#gbar .gbz0l{color:#000 !important;cursor:default;font-weight:bold;text-decoration:none !important}
#gbar { padding:.3em .6em !important;}</style></head><body  topmargin="3" marginheight="3"><div id=gbar><nobr><a onclick=gbar.qs(this);gbar.logger.il(1,{t:1}); class=gb1 id=gb_1 href="https://www.google.com/search?sa=N&tab=tw">Search</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:2}); class=gb1 id=gb_2 href="http://www.google.com/search?hl=en&tbm=isch&source=og&sa=N&tab=ti">Images</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:8}); class=gb1 id=gb_8 href="http://maps.google.com/maps?hl=en&sa=N&tab=tl">Maps</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:78}); class=gb1 id=gb_78 href="https://play.google.com/?hl=en&sa=N&tab=t8">Play</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:36}); class=gb1 id=gb_36 href="http://www.youtube.com/results?sa=N&tab=t1">YouTube</a> <a onclick=gbar.logger.il(1,{t:5}); class=gb1 id=gb_5 href="http://news.google.com/nwshp?hl=en&tab=tn">News</a> <a onclick=gbar.logger.il(1,{t:23}); class=gb1 id=gb_23 href="https://mail.google.com/mail/?tab=tm">Gmail</a> <a onclick=gbar.logger.il(1,{t:25}); class=gb1 id=gb_25 href="https://drive.google.com/?tab=to">Drive</a> <a class=gb3 href="http://www.google.com/intl/en/options/" onclick="this.blur();gbar.tg(event);return !1" aria-haspopup=true><u>More</u> <small>&#9660;</small></a><div class=gbm id=gbi><a onclick=gbar.logger.il(1,{t:24}); class=gb2 id=gb_24 href="https://www.google.com/calendar?tab=tc">Calendar</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:51}); class=gb2 id=gb_51 href="http://translate.google.com/?hl=en&sa=N&tab=tT">Translate</a><a onclick=gbar.logger.il(1,{t:17}); class=gb2 id=gb_17 href="http://www.google.com/mobile/?hl=en&tab=tD">Mobile</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:10}); class=gb2 id=gb_10 href="http://www.google.com/search?hl=en&tbo=u&tbm=bks&source=og&sa=N&tab=tp">Books</a><a onclick=gbar.logger.il(1,{t:212}); class=gb2 id=gb_212 href="https://wallet.google.com/manage/?tab=ta">Wallet</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:6}); class=gb2 id=gb_6 href="http://www.google.com/search?hl=en&tbo=u&tbm=shop&source=og&sa=N&tab=tf">Shopping</a><a onclick=gbar.logger.il(1,{t:30}); class=gb2 id=gb_30 href="http://www.blogger.com/?tab=tj">Blogger</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:27}); class=gb2 id=gb_27 href="http://www.google.com/finance?sa=N&tab=te">Finance</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:31}); class=gb2 id=gb_31 href="https://plus.google.com/photos?sa=N&tab=tq">Photos</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:12}); class=gb2 id=gb_12 href="http://www.google.com/search?hl=en&tbo=u&tbm=vid&source=og&sa=N&tab=tv">Videos</a><div class=gb2><div class=gbd></div></div><a onclick=gbar.logger.il(1,{t:66}); href="http://www.google.com/intl/en/options/" class=gb2>Even more &raquo;</a></div></nobr></div><div id=guser width=100%><nobr><span id=gbn class=gbi></span><span id=gbf class=gbf></span><span id=gbe></span><a target=_top id=gb_70 href="https://www.google.com/accounts/Login?service=&continue=http://www.google.com/patents%3Fhl%3Den&hl=en" class=gb4>Sign in</a><div style="display: none"><div class=gbm id=gbd5 aria-owner=gbg5><div class=gbmc><ol id=gbom class=gbmcc></ol></div></div></div></nobr></div><div class=gbh style=left:0></div><div class=gbh style=right:0></div><div role="alert" style="position: absolute; left: 0; right: 0;"><a href="http://www.google.com/patents/us5624803?hl=en&amp;output=html_text" title="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."><img border="0" src="http://www.google.com/images/cleardot.gif"alt="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."></a></div><div id="guser"><nobr></nobr></div><div style="clear:both;"></div><div id="gb-top-search-box" class="gb-top-search-box-small gb-reset"><table><tr><td class="logo"><a href="http://www.google.com/patents" class="logo-link"><img class="logo-img" src="/intl/en/images/logos/google_logo_41.png" alt="Go to Google Books Home" height="41"/></a></td><td><form action="http://www.google.com/search" name="f" id="vheadf" method="get"><span id="hf"></span><input type="hidden" name="tbm" value="pts"/><input type="hidden" name="tbo" value="1"/><input type="hidden" name="hl" value="en"/><table><tr><td><div class="inputs"><table><tr><td><div class="text-input"><input type="text" name="q" id="vheadq" class="text" maxlength="2048" size="31" value="" title="Search Patents" accesskey="s" autocomplete="off"/><script>window._OC_autoDir &&window._OC_autoDir('vheadq', 'tia-vheadq');</script></div></td><td><div class="submit-input"><input name="btnG" class="submit" type="submit" value=""/></div></td></tr></table></div></td><td class="col-ext-links"><div class="ext-links"><a href="http://www.google.com/advanced_patent_search">&lt;nobr&gt;Advanced Patent Search&lt;/nobr&gt;</a></div></td></tr></table></form></td></tr></table></div><div class="kd-appbar"><h2 class="kd-appname"><a href="/patents">Patents</a></h2><div class="kd-buttonbar left" id="left-toolbar-buttons"><a id="appbar-write-review-link" href=""></a><a id="appbar-view-print-sample-link" href=""></a><a id="appbar-view-ebook-sample-link" href=""></a><a id="appbar-patents-prior-art-finder-link" href="https://www.google.com/patents/related/US5624803"></a><a id="appbar-patents-discuss-this-link" href="http://www.google.com/url?id=tWZCBAABERAJ&amp;q=http://patents.stackexchange.com/redirect/google-patents%3Fpatent%3DUS5624803&amp;usg=AFQjCNHMMEPoRGEIMjPtn6x72CnS23K9Lw" data-is-grant="true"></a><a id="appbar-read-patent-link" href="//docs.google.com/viewer?url=patentimages.storage.googleapis.com/pdfs/US5624803.pdf"></a><a id="appbar-download-pdf-link" href="//patentimages.storage.googleapis.com/pdfs/US5624803.pdf"></a></div><div class="kd-buttonbar right" id="right-toolbar-buttons"></div></div><div id="books-microdata" itemscope=""itemtype="http://schema.org/Book"itemid="http://www.google.com/patents/US5624803" style="display:none"><span itemprop="description">The present invention encompasses improved methods and materials for the delivering of antisense, triplex, and/or ribozyme oligonucleotides intracellularly, and RNA polymerase III-based constructs termed &quot;oligonucleotide generators&quot; to accomplish the delivery of oligonucleotides. Also encompassed by...</span><span itemprop="url">http://www.google.com/patents/US5624803?utm_source=gb-gplus-share</span><span class="main-title" itemprop="name">Patent US5624803 - In vivo oligonucleotide generator, and methods of testing the binding affinity of triplex forming oligonucleotides derived therefrom</span><img itemprop="image" src="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"alt="Patent US5624803 - In vivo oligonucleotide generator, and methods of testing the binding affinity of triplex forming oligonucleotides derived therefrom" title="Patent US5624803 - In vivo oligonucleotide generator, and methods of testing the binding affinity of triplex forming oligonucleotides derived therefrom"></div><div style="display: none"><ol id="ofe-gear-menu-contents" class="gbmcc"><li class="gbe gbmtc"><a class="gbmt goog-menuitem-content" id="" href="http://www.google.com/advanced_patent_search">Advanced Patent Search</a></li></ol></div><table id="viewport_table" cellpadding="0" style="clear:both" cellspacing="0"><tr><td id="viewport_td"><div class=vertical_module_list_row><div id=intl_patents class=about_content><div id=intl_patents_v><table class="patent-bibdata"><tr><td class="patent-bibdata-heading">Publication number</td><td class="single-patent-bibdata">US5624803 A</td></tr><tr><td class="patent-bibdata-heading">Publication type</td><td class="single-patent-bibdata">Grant</td></tr><tr><td class="patent-bibdata-heading">Application number</td><td class="single-patent-bibdata">US 08/324,001</td></tr><tr><td class="patent-bibdata-heading">Publication date</td><td class="single-patent-bibdata">Apr 29, 1997</td></tr><tr><td class="patent-bibdata-heading">Filing date</td><td class="single-patent-bibdata">Oct 13, 1994</td></tr><tr><td class="patent-bibdata-heading">Priority date<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The priority date is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the date listed."></span></td><td class="single-patent-bibdata">Oct 14, 1993</td></tr><tr><td class="patent-bibdata-heading">Fee status<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The fee status is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the status or dates listed."></span></td><td class="single-patent-bibdata">Paid</td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Also published as</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/CA2173361A1">CA2173361A1</a>, </span><span class="patent-bibdata-value"><a href="/patents/CA2173361C">CA2173361C</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP0804563A1">EP0804563A1</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP0804563A4">EP0804563A4</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP1650298A1">EP1650298A1</a>, </span><span class="patent-bibdata-value"><a href="/patents/WO1995010607A1">WO1995010607A1</a></span></span></td></tr><tr class="patent-bibdata-list-row alternate-patent-number"><td class="patent-bibdata-heading">Publication number</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value">08324001, </span><span class="patent-bibdata-value">324001, </span><span class="patent-bibdata-value">US 5624803 A, </span><span class="patent-bibdata-value">US 5624803A, </span><span class="patent-bibdata-value">US-A-5624803, </span><span class="patent-bibdata-value">US5624803 A, </span><span class="patent-bibdata-value">US5624803A</span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Inventors</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Sarah+B.+Noonberg%22">Sarah B. Noonberg</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22C.+Anthony+Hunt%22">C. Anthony Hunt</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Original Assignee</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=inassignee:%22The+Regents+Of+The+University+Of+California%22">The Regents Of The University Of California</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Export Citation</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/US5624803.bibtex">BiBTeX</a>, </span><span class="patent-bibdata-value"><a href="/patents/US5624803.enw">EndNote</a>, </span><span class="patent-bibdata-value"><a href="/patents/US5624803.ris">RefMan</a></span></span></td></tr><tr class="patent-internal-links"><td colspan=2><span class="patent-bibdata-value"><a href="#backward-citations">Patent Citations</a> (6),</span> <span class="patent-bibdata-value"><a href="#npl-citations">Non-Patent Citations</a> (47),</span> <span class="patent-bibdata-value"><a href="#forward-citations">Referenced by</a> (192),</span> <span class="patent-bibdata-value"><a href="#classifications">Classifications</a> (25),</span> <span class="patent-bibdata-value"><a href="#legal-events">Legal Events</a> (7)</span> </td></tr><tr><td colspan=2 class="patent-bibdata-external-link-spacer-top"></td></tr><tr class="patent-bibdata-external-link-spacer-bottom"></tr><tr><td colspan=2><span class="patent-bibdata-heading">External Links:&nbsp;</span><span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=tWZCBAABERAJ&q=http://patft.uspto.gov/netacgi/nph-Parser%3FSect2%3DPTO1%26Sect2%3DHITOFF%26p%3D1%26u%3D/netahtml/PTO/search-bool.html%26r%3D1%26f%3DG%26l%3D50%26d%3DPALL%26RefSrch%3Dyes%26Query%3DPN/5624803&usg=AFQjCNGpxKQ1D3F3sm13_G5O6q-RHnYwRQ">USPTO</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=tWZCBAABERAJ&q=http://assignments.uspto.gov/assignments/q%3Fdb%3Dpat%26pat%3D5624803&usg=AFQjCNGMJP0s5w7Pa2rXyW3Ctvf_xEkdZw">USPTO Assignment</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=tWZCBAABERAJ&q=http://worldwide.espacenet.com/publicationDetails/biblio%3FCC%3DUS%26NR%3D5624803A%26KC%3DA%26FT%3DD&usg=AFQjCNHkE6VxRFnEaqEn8rMiGG0Q1ciEXQ">Espacenet</a></span></span></td></tr><tr class="patent-bibdata-group-spacer"></tr></table><div class="number-and-title"><span class="patent-title"><invention-title mxw-id="PT54161703" lang="EN" load-source="patent-office">In vivo oligonucleotide generator, and methods of testing the binding affinity of triplex forming oligonucleotides derived therefrom</invention-title></span><br><span class="patent-number">US 5624803 A</span></div><div class="patent-section patent-abstract-section"><div class="patent-section-header"><span class="patent-section-title">Abstract</span></div><div class="patent-text"><abstract mxw-id="PA37635548" lang="EN" load-source="patent-office"> <div class="abstract">The present invention encompasses improved methods and materials for the delivering of antisense, triplex, and/or ribozyme oligonucleotides intracellularly, and RNA polymerase III-based constructs termed "oligonucleotide generators" to accomplish the delivery of oligonucleotides. Also encompassed by the present invention are methods for screening oligonucleotide sequences that are candidates for triplex formation.</div>
  </abstract></div></div><div class="patent-section patent-drawings-section"><div class="patent-section-header"><span class="patent-section-title">Images<span class="patent-section-count">(32)</span></span></div><div class="patent-drawings-body"><div class="patent-drawings-carousel"><div class="drawings"><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5624803-1.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5624803-1.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5624803-2.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5624803-2.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5624803-3.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5624803-3.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5624803-4.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5624803-4.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5624803-5.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5624803-5.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5624803-6.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5624803-6.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5624803-7.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5624803-7.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5624803-8.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5624803-8.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5624803-9.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5624803-9.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5624803-10.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5624803-10.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5624803-11.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5624803-11.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5624803-12.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5624803-12.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5624803-13.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5624803-13.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5624803-14.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5624803-14.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5624803-15.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5624803-15.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5624803-16.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5624803-16.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5624803-17.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5624803-17.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5624803-18.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5624803-18.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5624803-19.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5624803-19.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5624803-20.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5624803-20.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5624803-21.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5624803-21.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5624803-22.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5624803-22.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5624803-23.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5624803-23.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5624803-24.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5624803-24.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5624803-25.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5624803-25.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5624803-26.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5624803-26.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5624803-27.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5624803-27.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5624803-28.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5624803-28.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5624803-29.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5624803-29.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5624803-30.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5624803-30.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5624803-31.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5624803-31.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5624803-32.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5624803-32.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div></div></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img" alt="Previous page"src="/googlebooks/images/kennedy/page_left.png"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img" alt="Next page"src="/googlebooks/images/kennedy/page_right.png"width="21" height="21" /></div></div></div><div class="patent-post-drawings"></div><div class="patent-section patent-claims-section"><div class="patent-section-header"><span class="patent-section-title">Claims<span class="patent-section-count">(28)</span></span></div><div class="patent-text"><div mxw-id="PCLM5084661" lang="EN" load-source="patent-office" class="claims">
    <claim-statement>We claim:</claim-statement> <div class="claim"> <div num="1" class="claim">
      <div class="claim-text">1. A construct for generating a specific oligonucleotide within a cell, which construct comprises a nucleotide sequence from which the transcript is the specific oligonucleotide, said nucleotide sequence being flanked in the 5' direction by a stabilizing region and in the 3' direction by a termination sequence, and a promoter, which initiates transcription by RNA polymerase III, and which promoter is in the 5' direction from the stabilizing region.</div>
    </div>
    </div> <div class="claim"> <div num="2" class="claim">
      <div class="claim-text">2. An oligonucleotide generator, comprising from 5' to 3':<div class="claim-text">(a) an U6-type RNA polymerase III promoter;</div> <div class="claim-text">(b) a specific nucleotide sequence from which a specific oligonucleotide can be transcribed; and</div> <div class="claim-text">(c) a termination sequence; wherein the components of the oligonucleotide generator are operably linked; and wherein the oligonucleotide generator is capable of being transcribed by RNA polymerase III to produce a transcript comprising the specific oligonucleotide.</div> </div>
    </div>
    </div> <div class="claim-dependent"> <div num="3" class="claim">
      <div class="claim-text">3. The oligonucleotide generator of claim 2, further comprising:<div class="claim-text">a stabilizing region from which a first hairpin-forming sequence can be transcribed; wherein the stabilizing region is operably linked and positioned between the U6-type RNA polymerase III promoter and the specific nucleotide sequence; and wherein the oligonucleotide generator is capable of being transcribed by RNA polymerase III to produce a transcript comprising from 5' to 3' the first hairpin-forming sequence and the specific oligonucleotide.</div> </div>
    </div>
    </div> <div class="claim-dependent"> <div num="4" class="claim">
      <div class="claim-text">4. The oligonucleotide generator of claim 3, further comprising<div class="claim-text">a capping segment; wherein the capping segment is operably linked and positioned between the stabilizing region and the specific nucleotide sequence; and wherein the oligonucleotide generator is capable of being transcribed by RNA polymerase III to produce a transcript comprising from 5' to 3' the first hairpin-forming sequence, a capping segment of AUAUCC or AUAUAC, and the specific oligonucleotide.</div> </div>
    </div>
    </div> <div class="claim-dependent"> <div num="5" class="claim">
      <div class="claim-text">5. The oligonucleotide generator of claim 4, wherein the first hairpin-forming sequence of the transcript consist of the first 20 nucleotides of the naturally-occurring human U6 transcript.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="6" class="claim">
      <div class="claim-text">6. The oligonucleotide generator of claim 4, wherein the capping segment of the RNA transcript is AUAUAC.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="7" class="claim">
      <div class="claim-text">7. The oligonucleotide generator of claim 2, wherein the U6-type RNA polymerase III promoter is selected from the group consisting of the U6 promoter, the 7SK promoter, the H1 RNA gene promoter, the plant U3 snRNA gene promoter, the MRP gene promoter, and recombinant promoters thereof, which recombinant promoters are capable of initiating transcription by RNA polymerase III from a position upstream of the transcribed DNA.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="8" class="claim">
      <div class="claim-text">8. The oligonucleotide generator of claim 7, wherein the U6-type RNA polymerase III promoter is the human U6 promoter.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="9" class="claim">
      <div class="claim-text">9. The oligonucleotide generator of claim 2, wherein the specific oligonucleotide is selected from the group consisting of antisense, triplex-forming, ribozyme oligonucleotides, and combinations thereof.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="10" class="claim">
      <div class="claim-text">10. The oligonucleotide generator of claim 9, wherein the specific oligonucleotide is an antisense oligonucleotide.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="11" class="claim">
      <div class="claim-text">11. The oligonucleotide generator of claim 9, wherein the specific oligonucleotide is a triplex-forming oligonucleotide.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="12" class="claim">
      <div class="claim-text">12. The oligonucleotide generator of claim 9, wherein the specific oligonucleotide is a ribozyme oligonucleotide.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="13" class="claim">
      <div class="claim-text">13. The oligonucleotide generator of claim 3, further comprising:<div class="claim-text">a 3' tail from which a second hairpin-forming sequence can be transcribed;</div> <div class="claim-text">wherein the 3' tail is operably linked and positioned between the specific nucleotide sequence and the termination sequence; and</div> <div class="claim-text">wherein the oligonucleotide generator is capable of being transcribed by RNA polymerase III to produce a transcript comprising from 5' to 3' the first hairpin-forming sequence, the specific oligonucleotide, and the second hairpin-forming sequence.</div> </div>
    </div>
    </div> <div class="claim-dependent"> <div num="14" class="claim">
      <div class="claim-text">14. The oligonucleotide generator of claim 3, further comprising:<div class="claim-text">a 3' tail from which a lariat-forming sequence can be transcribed;</div> <div class="claim-text">wherein the 3' tail is operably linked and positioned between the specific nucleotide sequence and the termination sequence;</div> <div class="claim-text">wherein the oligonucleotide generator is capable of being transcribed by RNA polymerase III to produce a transcript comprising from 5' to 3' the first hairpin-forming sequence, the specific oligonucleotide, and the lariat-forming sequence; and</div> <div class="claim-text">wherein the transcript is predicted to from a stable lariat structure by Watson-Crick base pairing between the nucleotides of the first hairpin-forming region and the lariat-forming region.</div> </div>
    </div>
    </div> <div class="claim-dependent"> <div num="15" class="claim">
      <div class="claim-text">15. The oligonucleotide generator of claim 2, further comprising:<div class="claim-text">a 5' tail from which a first lariat-forming sequence can be transcribed; and</div> <div class="claim-text">a 3' tail from which a second lariat-forming sequence can be transcribed;<div class="claim-text">wherein the 5' tail is operably linked and positioned between the U6-type RNA polymerase III promoter and the specific nucleotide sequence;</div> <div class="claim-text">wherein the 3' tail is operably linked and positioned between the specific nucleotide sequence and the termination sequence;</div> <div class="claim-text">wherein the oligonucleotide generator is capable of being transcribed by RNA polymerase III to produce a transcript comprising from 5' to 3' the first lariat-forming sequence, the specific oligonucleotide, and the second lariat-forming sequence; and</div> <div class="claim-text">wherein the transcript is predicted to from a stable lariat structure by Watson-Crick base pairing between the nucleotides of the first lariat-forming region and the second lariat-forming region.</div> </div> </div>
    </div>
    </div> <div class="claim-dependent"> <div num="16" class="claim">
      <div class="claim-text">16. The oligonucleotide generator of claim 4, further comprising two oligonucleotides operably linked and position of either side of the termination sequence such that the first 20 nucleotides downstream of the specific nucleotide sequence on the sense strand is identical to the final 20 nucleotide of the 3' portion of the human U6 gene:<div class="claim-text">5' GTCCTAGGCTTTTGCACTTTT 3'(SEQ ID NO:8);</div> <div class="claim-text">wherein the U6-type RNA polymerase III promoter is the U6 promoter; and</div> <div class="claim-text">wherein the first hairpin-forming sequence and the capping segment of the transcript consist of the first 25 nucleotides of the naturally-occurring human U6 transcript.</div> </div>
    </div>
    </div> <div class="claim-dependent"> <div num="17" class="claim">
      <div class="claim-text">17. The oligonucleotide generator of claim 16, wherein the sense strand of the specific nucleotide sequence is U6ON:<div class="claim-text">5' TCGACTCCTCTTCCTCCTCCACCTCCTCCTCCCATGCA 3'(SEQ ID NO:1).</div> </div>
    </div>
    </div> <div class="claim-dependent"> <div num="18" class="claim">
      <div class="claim-text">18. The oligonucleotide generator of claim 2, further comprising a viral vector capable of inducing integration of the oligonucleotide into a chromosome of a target cell.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="19" class="claim">
      <div class="claim-text">19. A method for generating oligonucleotides intracellularly, comprising administering an oligonucleotide generator of claim 2, in a form that permits entry of the oligonucleotides into a target cell.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="20" class="claim">
      <div class="claim-text">20. A method for continuously generating oligonucleotides intracellularly, comprising administering an oligonucleotide generator of claim 18 in a form that permits entry of the oligonucleotides into a cell.</div>
    </div>
    </div> <div class="claim"> <div num="21" class="claim">
      <div class="claim-text">21. A generator vector, comprising from 5' to 3':<div class="claim-text">(a) a U6-type promoter;</div> <div class="claim-text">(b) a stabilizing region from which a hairpin-forming sequence can be transcribed; and</div> <div class="claim-text">(c) a termination sequence; wherein the components of the generator vector are operably linked.</div> </div>
    </div>
    </div> <div class="claim-dependent"> <div num="22" class="claim">
      <div class="claim-text">22. The generator vector of claim 21, further comprising from 5' to 3':<div class="claim-text">a first restriction enzyme site; and</div> <div class="claim-text">a second restriction enzyme site; wherein the first and second restriction enzyme sites are operably linked and positioned between the stabilizing region and the termination sequence.</div> </div>
    </div>
    </div> <div class="claim-dependent"> <div num="23" class="claim">
      <div class="claim-text">23. The generator vector of claim 22, further comprising<div class="claim-text">a capping segment selected from the group consisting of ATATCC and ATATAC;<div class="claim-text">wherein the capping segment is operably linked and positioned between the stabilizing region and the first restriction enzyme site.</div> </div> </div>
    </div>
    </div> <div class="claim-dependent"> <div num="24" class="claim">
      <div class="claim-text">24. The generator vector of claim 23, wherein the U6-type promoter is the human U6 promoter; wherein the hairpin-forming sequence of the transcript consist of the first 20 nucleotides of the naturally-occurring human U6 transcript; and wherein the capping segment is ATATCC.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="25" class="claim">
      <div class="claim-text">25. The generator vector of claim 24,<div class="claim-text">wherein the first restriction enzyme site is an XhoI site; and</div> <div class="claim-text">wherein the second restriction enzyme site is an NsiI site.</div> </div>
    </div>
    </div> <div class="claim-dependent"> <div num="26" class="claim">
      <div class="claim-text">26. The generator vector of claim 24, further comprising two oligonucleotides operably linked and position of either side of the termination sequence such that the first 20 nucleotides downstream of the second restriction enzyme site on the sense strand is identical to the final 20 nucleotide of the 3' portion of the human U6 gene:<div class="claim-text">5' GTCCTAGGCTTTTGCACTTTT 3'(SEQ ID NO:7).</div> </div>
    </div>
    </div> <div class="claim"> <div num="27" class="claim">
      <div class="claim-text">27. A method of measuring triplex formation, which method comprises:<div class="claim-text">(a) attaching a single-stranded nucleic acid to a solid support;</div> <div class="claim-text">(b) contacting the solid support with a fluid comprising a labeled double-stranded probe;</div> <div class="claim-text">(c) separating the unbound probe from the solid support; and</div> <div class="claim-text">(d) quantifying the amount of labeled double-stranded probe bound to the solid support.</div> </div>
    </div>
    </div> <div class="claim"> <div num="28" class="claim">
      <div class="claim-text">28. A method of measuring triplex blotting, which method comprises:<div class="claim-text">(a) attaching a double-stranded nucleic acid to a solid support;</div> <div class="claim-text">(b) contacting the solid support with a fluid comprising a labeled single-stranded probe;</div> <div class="claim-text">(c) separating the unbound probe from the solid support; and</div> <div class="claim-text">(d) quantifying the amount of labeled single-stranded probe bound to the solid support.</div> </div>
    </div>
  </div> </div></div></div><div class="patent-section patent-description-section"><div class="patent-section-header"><span class="patent-section-title">Description</span></div><div class="patent-text"><div mxw-id="PDES66973077" lang="EN" load-source="patent-office" class="description">
    <heading>CROSS-REFERENCE TO RELATED APPLICATION</heading> <p>This application is a continuation-in-part of U.S. application Ser. No. 08/138,666, filed Oct. 14, 1993, now abandoned. The disclosure of the above-referenced patent application is hereby incorporated in its entirety herein by reference.</p>
    <heading>TECHNICAL FIELD</heading> <p>This invention relates to gene regulation technologies, gene therapies, and methods of measuring triplex binding. Specifically the invention relates to the use of U6-type RNA polymerase III promoters in constructs that produce intracellular oligonucleotides, particularly antisense, triplex-binding, and/or ribozyme RNA transcripts, which are relatively free of disruptive RNA secondary structure in their binding domains. Specifically the invention also relates to a method for screening oligonucleotide sequences that are candidates for triplex formation with a double-stranded DNA target site.</p>
    <heading>BACKGROUND</heading> <p>There are three main strategies for using oligonucleotides to affect gene regulation. These strategies involve the use of antisense oligonucleotides, triplex (or antigene) oligonucleotides, and ribozymes. Each strategy has its inherent advantages and limitations, and each has a growing base of experimental successes.</p>
    <p>Antisense Oligonucleotides</p>
    <p>Antisense oligonucleotides are the earliest examples of oligonucleotide-based approaches to gene regulation. Not only were they the first to be tested for biological activity in the laboratory and the clinic, but also the antisense concept may have first arisen in prokaryotes. Naturally occurring antisense RNA has been detected in E. coli bacteria as well as in colE1 and IS10 plasmids. In these strains, regions of transcribed RNA in the antisense orientation serve to regulate the translation of RNA in the sense orientation (Inouye, M. (1988) Gene 72:25-34; and Simons, R. W. and Kleckner, N. (1988) Annual Rev. Gen. 22:567-600). Similar naturally occurring antisense regulation strategies have now been identified in several eukaryotic genes as well (Bentley, D. L. and Groudine, M. (1986) Nature 321:702-706; Kimelman, D. Gene regulation: Biology of Antisense RNA and DNA, R. P. Erickson, J. G. Izant, eds. (Raven Press, New York) pp. 1-10). Given that the normal messenger RNA (mRNA) transcribed from DNA is referred to as the "sense" RNA strand, oppositely oriented RNA are termed antisense RNA. Antisense oligonucleotides, then, refer to specific sequences of DNA or RNA which can bind in a Watson-Crick fashion to a sequence on a target mRNA.</p>
    <p>In forming a double-stranded region on the mRNA, subsequent steps of protein synthesis may be interrupted by any of a variety of mechanisms. Interruption may occur by sterically blocking ribosome assembly or progression, sterically blocking intron/exon junctions and splice-sites needed for the processing of premature mRNA, or by invoking the cellular enzyme RNAse H that specifically cleaves mRNA in mRNA/DNA hybrids. The potential of antisense oligonucleotides to enact specific inhibition of protein synthesis is reflected in the tremendous number of publications which have appeared in the nearly 20 years since the earliest report of an antisense effect against the Rous Sarcoma Virus (Zamecnik, P. C. and Stephenson, M. L. (1978) Proc. Natl. Acad. Sci. USA 75:280-284; and for general reference on antisense oligonucleotides see: Moffat, A. S. (1991) Science 253:510-511; Chrisey, L. A. and Hawkins, J. W. (1991) Biopharm. 36-42; Oligonucleotides: Antisense Inhibitors of Gene Expression (1989) J. S. Cohen, ed. (MacMillan Press, London); and Stein, C. A. and Cheng, Y. C. (1993) Science 261:1004-1012). Antisense RNA and DNA has been demonstrated to lead as much as 95% inhibition of specific mRNA translation. In some cases, the antisense strategy has led to upregulated levels of the corresponding protein (Williard, R. L. et al. (1994) Gene (in press)).</p>
    <p>A single gene encoded in DNA can, and most often will, be transcribed multiple times, giving rise to many copies of mRNA. In turn, a single mRNA molecule can, and most often will, be translated multiple times giving rise to many copies of the corresponding protein. Therefore, amplification can take place both at the DNA-&gt;mRNA level, and also at the mRNA-&gt;protein level. Antisense oligonucleotides can be very effective at blocking the translation of RNA and reducing the amount of protein synthesized, but they must be present within the cell in sufficient numbers to account for the previous DNA-&gt;mRNA amplification step. If a cell contains a compensatory response to lower levels of a given protein, feedback may upregulate the transcription of the corresponding RNA, increase mRNA stability, or signal for an increase in ribosomal assembly--all of which may serve to diminish the efficacy of an antisense approach. Given this constraint, it is unlikely that 100% inhibition of target gene expression could be achieved. To achieve this maximal level of inhibition, one must backstep and posit a strategy in which mRNA transcription (and thus the first level of amplification) is prevented. Such a strategy can be theoretically achieved by triple helix-forming (triplex) oligonucleotides.</p>
    <p>Triplex Oligonucleotides</p>
    <p>The development of triplex oligonucleotides as inhibitors of gene expression is receiving considerable, albeit delayed, attention. In 1957, Felsenfeld, Davies, and Rich described a surprisingly stable structure composed of three homopolymeric nucleic acid strands (Felsenfeld, G. et al. (1957) J. Am. Chem. Soc. 79:2023-2024). Two of these strands formed normal Watson-Crick hydrogen bonds, while the third strand was associated by what are now called Hoogsteen hydrogen bonds. Later studies confirmed and elaborated upon these findings, and determined that a pyrimidine-rich third strand could reside in the major groove of homopurine/homopyrimidine double-stranded DNA or RNA without perturbation of underlying Watson-Crick base-pairs. The binding was found to be pH-dependent (when cytosine residues were involved), and oriented parallel to the corresponding purine strand of the double-stranded helix (Lyamichev, V. I. et al. (1986) J. Biomol. Struct. and Dynam. 3:667-669; and Moser, H. E. and Dervan, P. B. (1987) Science 238:645-650). Nearly three decades passed before several laboratories began to use triplex oligonucleotides both to create a new class of sequence-specific DNA cleaving tools (Moser, H. E. and Dervan, P. B. (1987) Science 238:645-650; Strobel, S. A. and Dervan, P. B. (1990) Science 249:73-73; Strobel, S. A. and Dervan, P. B. (1991) Nature 350:172-174; and Dervan, P. B. Oligonucleotides: Antisense Inhibitors of Gene Expression, J. S. Cohen, ed. (Macmillan Press, London) pp. 197-210) as well as to mediate specific gene regulation at the level of the promoter (Durland, R. H. et al. (1991) Biochem. 30:9246-9255; Duval-Valentin, G. et al. (1992) Proc. Natl. Acad. Sci. USA 89:504-508; and Maher, L. J. et al. (1989) Science 245:725-730).</p>
    <p>Subsequently, a number of reports emerged citing the ability of a sequence-specific pyrimidine-rich oligonucleotide to inhibit the binding of DNA binding proteins and in vitro gene transcription and elongation (Maher, L. J. et al. (1992) Biochem. 31:70-81; Young SL et al. (1991) Proc. Natl. Acad. Sci. USA 88:10023-10026; and Cooney, M. et al. (1988) Science 241:456-459). In addition to the pyrimidine-rich third strand binding motif previously described, a second triplex binding motif was identified involving a purine-rich third strand bound in a Mg++-dependent, pH-independent fashion (Chamberlin, M. J. and Patterson D. L. (1965) J. Mol. Biol. 12:410-428; Cooney, M. et al. (1988) Science 241:456-459; and Beal, P. A. and Dervan, P. B. (1991) Science 251:1360-1363). Orientation was shown to be antiparallel when the third strand was composed of G and A residues and dependent upon GpA steps in the target sequence when the third strand was composed of G and T residues (Sun, J. S. et al. (1991) C. R. Acad. Sci. Paris Serial III 313:585-590). A growing number of reports of in vivo inhibition of gene expression have been cited using triplex oligonucleotides of this second binding motif, while only triplex oligonucleotides conjugated to crosslinking agents have shown similar capabilities when using the first binding motif (Postel, E. H. et al. (1991) Proc. Natl. Acad. Sci. USA 88:8227-8231; McShan, W. M. et al. (1992) J. Biol. Chem. 267:5712-5721; Ing, N. H. et al. (1993) Nuc. Acids Res. 21:2789-2796; Roy, C. (1993) Nuc. Acids Res. 21:2845-2852); and Grigoriev, M. et al. (1993) Proc. Natl. Acad. Sci. USA 90:3501-3505).</p>
    <p>While the majority of reports which cite triplex-based repression of transcription both in vitro and in vivo have utilized DNA triplex oligonucleotides on DNA double-stranded targets, recent research indicates that the thermodynamics of binding may favor an RNA third strand over a DNA third strand should a pyrimidine-rich binding motif be chosen (Roberts, R. W. and Crothers, D. M. (1992) Science 258:1463-1467; and Escude, C. et al. (1993) Nuc. Acids Res. 21:5547-5553). However, RNA purine-rich oligonucleotides have not been shown to form triplex structures with DNA double-stranded targets under any known conditions (Escude, C. et al. (1993) Nuc. Acids Res. 21:5547-5553; and Noonberg, S. B. et al. (1994) BioTechniques 16:1070-1073), but have been shown to form triplex structures with RNA homopurine/hompyrimidine duplexes and DNA/RNA hybrid duplexes (Chamberlin, M. J. and Patterson, D. L. (1965) J. Mol. Biol. 12:410-428). Such results underscore the importance of differing sugar backbones on triplex formation and stability.</p>
    <p>A great deal of research effort and resources are being devoted to triplex oligonucleotides due to their potential to abolish completely the expression of a specific protein, especially when conjugated to intercalating agents which can crosslink the underlying DNA at a given triplex site (Grigoriev, M. et al. (1993) Proc. Natl. Acad. Sci. USA 90:3501-3505; and Helene, C. and Toulme, J. J. in Oligonucleotides: Antisense Inhibitors of Gene Expression, J. S. Cohen, ed. (Macmillan Press, London) pp. 137-172). Whether the triplex structure is permanent or in a binding equilibrium, triplex formation may inhibit the expression of a given gene in a variety of mechanisms: steric interference with transcription factor binding and assembly on promoter targets, alteration of duplex rigidity and inability for nonadjacent DNA binding proteins to associate, alteration of major and minor grooves to prevent neighboring transcription factor binding, and inhibition of elongation of the transcription complex producing truncated mRNA transcripts. In addition, a covalent complex may induce DNA repair elements to cleave out the linkage and thus inactivate a regulatory region of a gene.</p>
    <p>Ribozymes</p>
    <p>First identified within the RNA of Tetrahymenae (Kruger, K. (1982) Cell 31:147-157), ribozymes are able to perform self-cleavage reactions without additional protein enzyme or catalytic elements. This feature suggests that ribozymes may represent the first self-replicating entity. Ribozymes have now been isolated (primarily from viral sources) in various species of plants, fungi, and animals, all sharing the same capacity for catalytic activity without additional protein co-factors (Foster, A. C. and Symons, R. H. (1987) Cell 49:211-220). Their possible role in selective gene regulation stems from the ability to couple the catalytic RNA center domain to flanking RNA sequences designed to target an mRNA molecule by Watson-Crick base-pairing. Thus, the ribozyme can bind and cleave a target mRNA without self-impairment. Like antisense oligonucleotides, ribozymes also act on mRNA as opposed to DNA, and therefore do not affect the initial amplification step of transcription. And like antisense oligonucleotides, it may be near impossible to achieve 100% inhibition of target gene expression using a ribozyme strategy. However, unlike antisense oligonucleotides, ribozymes have the potential to quickly, permanently, and repetitively inactivate substrate mRNA. These differences may allow for the detection of significant biological activity at far lower concentrations intracellularly.</p>
    <p>As with antisense and triplex oligonucleotides, a growing number of reports of ribozyme-mediated suppression of protein synthesis are appearing (Cotten, M. and Birnstiel, M. L.(1989) EMBO J. 8:3861-3868; Sarver, M. et al. (1990) Science 247:1222-1224; Cameron, F. H. and Jennings, P. A. (1989) Proc Natl Acad Sci USA 86:9139-9143; and Haseloff, J. and Gerlach, W. L. Nature 334: 585-591).</p>
    <p>Increasing attention has been drawn to employing antisense, ribozyme, and triplex forming oligonucleotides in strategies for specific gene regulation. Oligonucleotides may specifically bind to viral DNA and RNA sequences involved in viral replication and pathogenicity, cellular oncogenic DNA and RNA sequences involved in neoplastic cell proliferation and differentiation, and various protein coding DNA and RNA sequences involved in disease pathophysiology. Oligonucleotides are thought to have potential utility in antiviral, anticancer, and antiprotein therapeutics.</p>
    <p>Historically, oligonucleotides have been introduced into cells in one of three ways: by addition to the extracellular media, by invasive techniques such as electroporation or microinjection, or by integration of antisense mRNA vectors into the host chromosome. Each of these methods has its advantages and limitations. The first method, the addition of oligonucleotides to the media bathing the cells, induces little cellular damage and delivers short nucleic acid sequences, but the mechanisms of oligonucleotide uptake have poor efficiency and the cost of oligonucleotide synthesis is significant.</p>
    <p>The second class of techniques, microinjection and electroporation, can greatly increase the efficiency of uptake of small oligonucleotides into the cell, but may cause significant cellular injury and disruption of normal cellular function. The third class of techniques, integration of the target gene and RNA polymerase II promotor in an antisense orientation via plasmids or retroviral vectors, can provide stable transcripts in healthy cells, but these transcripts retain their full length and can be expected to display considerable secondary structure, masking key antisense regions. In addition, RNA polymerase II transcripts are not continually produced, and transcriptional frequency is quite low in comparison to RNA products from polymerases I and III.</p>
    <p>Interest in the biological activity of triplex-forming oligonucleotides has been steadily increasing, owing in part to their potential as artificial repressors of gene expression (Helene, C. (1991) Anticancer Drug Design 6:569-584; Maher III, L. J. et al. (1989) Science 245:725-730; Postel, E. H. et al. (1991) Proc. Natl. Acad. Sci. USA 88:8227-8231; and Ing, N. H. et al. (1993) Nucleic Acids Res. 21:2789-2796) as well as mediators of site-specific DNA cleavage (Moser, H. E. and Dervan, P. B. (1987) Science 238:645-650; and Strobel, S. A. and Dervan, P. B. (1991) Nature 350:172-174).</p>
    <p>The potential of triplex and antisense oligonucleotides to inhibit selectively protein synthesis from a specified target gene has generated significant enthusiasm for their development as experimental therapeutics. Inhibition of expression of virally-derived proteins (Zamecnik &amp; Stephenson (1978) Proc. Natl. Acad. Sci. USA 75: 280-284; Agrawal, S. (1991) In Prospects for Antisense Nucleic Acid Therapy of Cancer and AIDS. Eric Wickstrom, ed. (Wiley-Liss, New York), pp. 143-158; and McShan, W. M. et al. (1992) J. Biol. Chem. 267: 5712-5721) or endogenously activated oncogenes that contribute to cancer induction and/or progression (Helene, C. (1991) Anticancer Drug Design 6: 569-584; Postel, E. H. et al. (1991) Proc. Natl. Acad. Sci. USA 88: 8227-8231; and Calabretta, B. (1991) Cancer Research 51: 4505-4510) represent two particularly active areas of applied research, although the technology is also a powerful basic science research tool for the functional assessment of specific genes in cellular growth and differentiation (Simons, M. et al. (1992) Nature 359: 67-70).</p>
    <p>While sufficient evidence indicates that oligonucleotides can cross the multiple cellular membrane barriers needed to reach their intracellular targets (Loke, S. L. et al. (1989) Proc. Natl. Acad. Sci. USA 86: 3474-3478; Yakubov, L. et al. (1989) Proc. Natl. Acad. Sci. USA 86: 6454-6458; Wu-Pong, S. et al. (1992) Pharmaceutical Research 9: 1010-1017; and Noonberg, S. B. et al. (1993) J. Invest. Dermatol. 101: 727-731), a growing number of reports suggest that this uptake process is highly inefficient and may exhibit cell-type specificity and heterogeneity (Noonberg, S. B. et al. (1993) J. Invest. Dermatol. 101: 727-731; Krieg, A. M. et al. (1991) Antisense Research and Development 1: 161-171; and Iverson, P. L. et al. (1992) Antisense Research and Development 2: 211-222). In addition, imaging studies demonstrate that the typical pattern of oligonucleotide uptake results in oligonucleotide compartmentalization within punctate vesicles believed to be of endosomal origin (Loke, S. L. et al. (1989) Proc. Natl. Acad. Sci. USA 86: 3474-3478; and Yakubov, L. et al. (1989) Proc. Natl. Acad. Sci. USA 86: 6454-6458), sequestered from their DNA or RNA targets, and subject to eventual lysosomal fusion and nuclease degradation. Rapid extracellular degradation has also been noted (Wickstrom, E. (1986) J. Biochem. Biophys. Meth. 13: 97-102). Biological activity is thought to arise from the small fraction of full-length oligonucleotide that either escapes from endosomes and rapidly accumulates in the nucleus, or enters the cytoplasm by another process and similarly accumulates intranuclearly. Oligonucleotide/nucleic acid target interactions can thus occur en route to or within the nucleus.</p>
    <p>Current strategies for the delivery of nucleic acid for antisense gene regulation fall into 1 of 2 major classes: direct extracellular addition of short DNA oligonucleotides (or analogues thereof) to cell culture media (as described in the previous chapter), or cellular gene transfection which is transcribed by RNA polymerase II into a long antisense transcript (or mRNA with antisense insert). Strategies for the delivery of nucleic acids for triplex gene regulation fall entirely into the first class, while strategies for the delivery of nucleic acid for ribozyme gene regulation fall entirely into the second class. Both classes have clear advantages, but both are also handicapped by limitations which have yet to be adequately circumvented (Table I).</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">              TABLE I______________________________________Advantages and Limitations to Current NucleicAcid Delivery Strategies    Advantages  Limitations______________________________________Oligodeoxyribo-      Easy introduction                    Uptake can benucleotides      to cells.     inefficient,                    heterogeneous.      Minimal       Rapid intra/extracellular      secondary     degradation.      structure concerns.      No transfection                    Possible toxicity.      required.      Accommodates  Noncontinuous      chemical      administration.      analogues      Can be used for                    High cost of synthesis      antisense or triplex                    and purification.      strategies.Polymerase II      Intracellular Variable expression.transcripts      expression.      Can be inducible.                    Rapid intracellular                    degradation.      Can be made   Long transcripts with      permanent.    variable start and stop                    positions.      Inexpensive.  Considerable secondary                    structure can mask                    binding regions.______________________________________</pre>
    
    <p>Perhaps the biggest advantage of extracellular addition of oligonucleotides translates into the biggest disadvantage of intracellular generation of polymerase II transcripts--namely the length of the nucleic acid delivered. Oligonucleotides are, by their very name, short fragments of nucleic acid, and are thus generally capable of weak or predictable secondary and/or intermolecular structures. As such, their binding regions are expected to display increased accessibility to an intracellular nucleic acid target (whose accessibility may be difficult to predict). In contrast, polymerase II transcripts are generally very long (&gt;1 kilobase) with highly variable trailing 3' sequences. These long and variable transcripts inevitably lead to complex secondary and tertiary structures which may mask key binding sequences. In most cases, a structural prediction of these long transcripts cannot be accurately determined.</p>
    <p>Analogously, perhaps the biggest advantage of intracellular generation of polymerase II transcripts translates into the biggest disadvantage of extracellular addition of oligonucleotides--namely the site of nucleic acid delivery. Polymerase II transcripts are, by their very nature, generated intracellularly and thus do not need to cross the cell membrane barrier. For nuclear targets, the transcript is already in the correct subcellular compartment, while for cytoplasmic targets, the transcript must be actively or passively transported across the nuclear membrane. In contrast, extracellularly-added oligonucleotides must first cross the cell membrane, a process which has been shown to be inefficient, heterogeneous, cell-type specific, and to often lead to endosomal sequestration and subsequent lysosomal degradation. In addition, nucleic acid from both strategies are also susceptible to rapid degradation by cellular endonucleases and exonucleases, and both strategies may give rise to low or variable intracellular nucleic acid concentrations.</p>
    <p>A logical question then becomes, can one design a system which combines the major advantages of both strategies while eliminating the major disadvantages? In positing such a system, one can put forth the following design criteria:</p>
    <p>1. The system must be capable of delivering short nucleic acids of a pre-specified sequence and length to allow for adequate secondary structure prediction.</p>
    <p>2. The system must be capable of delivering nucleic acid intracellularly.</p>
    <p>3. The system must be capable of delivering nucleic acids of sufficient intracellular stability against nuclease degradation.</p>
    <p>4. The system must be capable of delivering nucleic acids in high yield without cell-type specificity.</p>
    <heading>SUMMARY OF THE INVENTION</heading> <p>We have developed a system, here termed an "oligonucleotide generator", or an "in vivo oligonucleotide generator" for intracellular generation of short sequence-specific oligonucleotides in extremely high yield for the purposes of gene regulation. The invention provides for a continuous and abundant supply of short genetic fragments for use in any of a variety of gene regulation strategies.</p>
    <p>According to the oligonucleotide generator invention, RNA polymerase III based promoter and terminator genetic sequences are used with one or more antisense, ribozyme, or triplex forming oligonucleotides as the coding regions. The constructs in some embodiments of the invention are provided with self-complementary ends to enhance stability.</p>
    <p>We have developed a method, here termed "triplex blotting", for detection of triplex-forming RNA using radiolabeled double-stranded DNA probes within a background of total cellular RNA. Triplex blotting provides for detection of single cellular or in vitro generated RNA species, electrophoretically separated and immobilized on filters with radiolabeled triplex-forming double-stranded DNA probes.</p>
    <p>The triplex blotting invention provides for comparison of relative binding affinities of various triplex-forming RNAs, for screening potential RNA sequences for triplex formation with double-stranded DNA targets, and for confirming the specificity of triplex formation of a DNA target probe within total cellular RNA. The benefits of triplex blotting include: sensitive and specific detection of homopurine/homopyrimidine RNA sequences; rapid screening of duplex DNA target sequences against triplex forming RNAs, with direct comparison of relative binding affinities; and confirmation of specificity of triplex formation amidst a background of total cellular RNA.</p>
    <p>The present invention encompasses, constructs for generating a specific oligonucleotide within a cell, which construct comprises a nucleotide sequence from which the transcript is the specific oligonucleotide, said nucleotide sequence being flanked in the 5' direction by a stabilizing region and in the 3' direction by a termination sequence, and a promoter, which initiates transcription by RNA polymerase III, and which promoter is in the 5' direction from the stabilizing region.</p>
    <p>The present invention further encompasses oligonucleotide generators, comprising from 5' to 3': (a) a U6-type RNA polymerase III promoter; (b) a specific nucleotide sequence from which a specific oligonucleotide can be transcribed; and (c) a termination sequence; wherein the components of the oligonucleotide generator are operably linked; and wherein the oligonucleotide generator is capable of being transcribed by RNA polymerass III to produce a transcript comprising the specific oligonucleotide.</p>
    <p>The present invention encompasses methods for generating oligonucleotides intracellularly, comprising administering an oligonucleotide generator of the invention, in a form that permits entry of the oligonucleotides into a target cell.</p>
    <p>The present invention encompasses generator vectors, comprising from 5' to 3': (a) a U6-type promoter; (b) a stabilizing region from which a hairpin-forming sequence can be transcribed; and (c) a termination sequence; wherein the components of the generator vector are operably linked.</p>
    <p>The present invention encompasses methods of measuring triplex formation, which method comprises: (a) attaching a single-stranded nucleic acid to a solid support; (b) contacting the solid support with a fluid comprising a labeled double-stranded probe; (c) separating the unbound probe from the solid support; and (d) quantifying the amount of labeled double-stranded probe bound to the solid support.</p>
    <p>The present invention encompasses methods of measuring triplex blotting, which method comprises: (a) attaching a double-stranded nucleic acid to a solid support; (b) contacting the solid support with a fluid comprising a labeled single-stranded probe; (c) separating the unbound probe from the solid support; and (d) quantifying the amount of labeled single-stranded probe bound to the solid support.</p>
    <heading>BRIEF DESCRIPTION OF THE DRAWINGS</heading> <p>FIG. 1 is a diagram presenting an overview of the oligonucleotide generator system according to the invention, and provides an example of its use.</p>
    <p>FIG. 2 is a diagram illustrating in further detail the invention shown in overview in FIG. 1, demonstrating exemplary transcript configurations and promoter and terminator sequences (SEQ ID NO:8 and SEQ ID NO:9) according to the invention.</p>
    <p>FIG. 3 (SEQ ID NO:10 through SEQ ID NO:13) is a diagram showing a double-stranded DNA probe and triplex forming RNAs. The 50 base-pair double-stranded probe corresponds to bases -68 to -19 on the HER2/c-erb B2/neu proto-oncogene. The region in bold refers to the homopurine/homopyrimidine tract involved in triplex formation. The CU-rich and GA-rich RNA sequences correspond to pyrimidine (parallel) or purine (antiparallel) third strand triplex binding motifs, respectively.</p>
    <p>FIG. 4 is a second diagram presenting an overview of the oligonucleotide generator system according to the invention. The structure of the U6 snRNA gene, the chimeric U6ON gene and the resulting U6ON transcript are shown. FIG. 4A. shows the U6 gene has three critical promoter elements necessary for efficient transcription, a 5' self-complementary hairpin sequence sufficient for capping, and a string of 5 thymidine residues necessary for termination. FIG. 4B demonstrates the elements which were retained in the construction of the chimeric gene, while its internal sequence was mutated to produce two unique restriction sites for inserting oligonucleotide sequences (bold line). FIG. 4C shows the resulting oligonucleotide which may retain the 5' hairpin, followed by the oligonucleotide (bold line) and the native U6 uridine-rich sequence.</p>
    <p>FIG. 5 is a half-tone reproduction of Northern blots which demonstrate the production and nuclear localization of the U6ON transcript. For FIG. 5A, MDA453 and 293 cells were transfected with either 10 μg of the chimeric U6ON gene or 10 μg of promoterless plasmid DNA. Total cellular RNA was isolated 48 hours later followed by Northern blotting with both U6 and U6ON radiolabeled probes. For FIG. 5B, MDA453 cells were transfected with increasing quantities of the chimeric gene followed by RNA isolation at 48 hours and Northern blotting as described above. All transfections contained 40 μg total DNA, with promoterless plasmid DNA supplementing the chimeric gene as necessary. For FIG. 5C, MDA453 cells were transfected with 10 μg of the chimeric gene and after 48 hours, RNA was separated into nuclear and cytoplasmic fractions. The nuclear fraction shown above contained the U6ON transcript along with the native U6 snRNA. All Northern blots were generated from 10 μg of RNA loaded/well.</p>
    <p>FIG. 6 is a graph and a half-tone reproduction of a Northern blot which demonstrates the kinetics of U6ON expression. For FIG. 6A, 293 cells were transfected with 20 μg of the chimeric gene and total cellular RNA was isolated at 0, 3, 6, 9, and 12 hours time points. After Northern blotting (10 mg RNA added/well) and autoradiography with a U6ON radiolabeled probe, the U6ON bands were cut from the filter and scintillation counted. For FIG. 6B, MDA453 cells were transfected with 20 mg of the chimeric gene and total cellular RNA was isolated at 48, 72, 96, 120, 144, and 168 hour time points. Northern blotting followed with 20 μg of RNA added/well.</p>
    <p>FIG. 7 is a graph and a half-tone reproduction of a Northern blot which demonstrates the intracellular stabilities of the chimeric gene and the U6ON transcript. For FIG. 7A, cell counting in parallel with the RNA isolations of FIG. 7B allowed the U6ON band densities to be normalized to account for the dilutional effects of cell division. Normalized band densities were plotted as a function of time to determine the rate of chimeric gene degradation (or inactivation). For FIG. 7B, 293 cells were transfected with 5 μg of the chimeric gene and after 48 hours, cellular transcription was halted by a 10 μg/ml treatment of Actinomycin D. At 0, 0.5, 1, 2, and 4 hour time points, RNA was isolated. Northern blotting (20 μg RNA/well) with U6 and U6ON radiolabeled probes, followed by densitometry of the U6ON bands, allowed for the determination of U6ON half-life.</p>
    <p>FIG. 8 is a half-tone reproduction of a Northern blot demonstrating the immunoprecipitation of the U6ON transcript with a 5'γ-monomethyl phosphate cap. Total cellular RNA samples isolated after a transfection with 20 μg of U6ON in 293 cells were immunoprecipitated with 0.5 mg of a U6 cap-specific antibody as previously described (Gupta et al. (1990), J. Biol. Chem. 265:19137-19142. Eric Wickstrom, ed. (Wiley-Liss, New York), pp. 143-158). Each immunoprecipitation required 20 μg of initial total cellular RNA. Immunoprecipitated RNA was used for Northern blotting with U6 and U6ON radiolabeled probes.</p>
    <p>FIG. 9 (SEQ ID NO:14 and SEQ ID NO:15) is diagram of two predicted secondary structures for two RNA transcripts and a half-tone reproduction of a Northern blot, demonstrating the effects of the insert sequence on secondary structure and intracellular transcript levels. For FIG. 9A, RNA secondary structures and associated energies were predicted for two different constructs, U6CTcon and U6AS, using the program RNAFOLD (Martinez, H. (1990) Methods in Enzymology, 183, 306-317). Upper case letters refer to base-pairings, lower case letters refer to mismatches and colons refer to bulged regions. The energies of U6ON and mU6, which were found to have similar structural profiles to U6CTcon and U6AS, respectively, are given in parentheses. For FIG. 9B, RNA secondary structure was predicted for 4 different oligonucleotide transcripts and the corresponding chimeric genes were constructed. 20 μg of the chimeric genes were transfected into MDA453 cells, followed by Northern blotting (20 μg RNA added/well) 48 hours later with a U6 probe and a probe for each of the possible RNA transcripts.</p>
    <p>FIG. 10 (SEQ ID NO:16 through SEQ ID NO:18) is a diagram of the HER2 proto-oncogene promoter map with a triplex RNA oligonucleotide.</p>
    <p>FIG. 11 is a graph representing the effects of the U6ON and U6CTcon oligonucleotide generators on CAT activity in cells expressing CAT from an HER2 promoter. FIG. 11A illustrates the CAT activity arising from the minimal 125 bp HER2 promoter and the stronger 500 bp HER2 promoter in MDA453 and MCF-7 cells 48 hours after co-transfection with 20 μg of either U6ON or U6CTcon. Promoterless plasmid DNA was used to equalize DNA concentrations. Both U6ON and U6CTcon RNA oligonucleotides demonstrated virtually 100% downregulation of the minimal promoter and roughly 65 to 70% downregulation of the stronger 500 bp promoter. FIG. 11B illustrates the HER2/CAT downregulation of the stronger 500 bp promoter by triplex and antisense RNA oligonucleotides after co-transfection with 10 μg of either U6ON or U6AUG.</p>
    <p>FIG. 12 is two graphs and two half-tone reproductions of Northern blots, which demonstrate the effect of the U6ON generator on endogenous HER2 mRNA and cell growth in MDA 453 cells. FIG. 12A is a Northern blot, which compares the levels GAPDH and HER2 RNA from 48 to 96 hours after transfection with the U6ON oligonucleotide generator, as compared with control DNA. FIG. 12B is a graph demonstrating the HER2 RNA values from FIG. 12A after normalization for the GAPDH levels. FIG. 12C demonstrates the temporal pattern of U6ON expression after transient transfection. FIG. 12D demonstrates the cell growth rate in U6ON transfected cells versus control cells between 48 hours and 96 hours.</p>
    <p>FIG. 13 is a diagram detailing the promoter, termination, and capping factors of the chimeric gene (oligonucleotide generator). This drawing illustrates schematically a possible configurations of the needed transcriptional factors of both the U6 gene and the chimeric RNA-oligonucleotide producing gene.</p>
    <p>FIG. 14 is a half-tone reproduction of a Northern blot and a graph which demonstrate the dose dependence of gene transfection on U6 and U6ON in 293 cells. Cells were transfected with 40 μg of DNA which contained 5 to 40 μg of the chimeric gene or promoterless plasmid DNA. RNA was isolated 48 hours after transfection. After Northern blotting with U6 and U6ON probes (FIG. 14A), densitometry was performed to quantitate the relative downregulation of U6 with concurrent upregulation of U6ON and the data was presented in FIG. 14B. Error bars arise from standard deviation of densitometry data and slight inequalities in RNA loading, normalized by ethidium bromide staining.</p>
    <p>FIG. 15 is a half-tone reproduction of a Northern blot, demonstrating that U6 levels vary as a function of RNA oligonucleotide stability. 293 cells were treated with 20 μg of the gene for U6AS (lanes 1-4) or U6ON (lanes 5 to 8) followed by RNA isolation at 48 hour, 72 hour, 96 hour, and 120 hours time points. U6AS has previously been shown to be an unstable RNA oligonucleotide while U6ON has been shown to be a stable and 5' capped RNA oligonucleotide.</p>
    <p>FIG. 16 is a half-tone reproduction of a Northern blot, demonstrating that U6ON can downregulate U6 stability. 293 cells were transfected with 20 μg of the chimeric gene for U6ON followed 48 hours later by 10 μg/ml treatment with Actinomycin D to mediate total transcription arrest. RNA isolations followed at 0 hour, 0.5 hour, 1 hour, 2 hour, 4 hour, and 8 hour time points followed by Northern blotting.</p>
    <p>FIG. 17 is a graph demonstrating that U6 stability is titratable by U6ON levels. The experiment presented in FIG. 16 was repeated using 10 μg and 5 μg chimeric gene transfection doses. After performing individual Northern blots, data was amassed and quantitated by densitometry.</p>
    <p>FIG. 18 is a series of three graphs, illustrating the results of dynamic simulation of U6 and U6ON expression in 293. The dynamic modelling program Stella was used to solve the differential equations described in the Examples section. Data was then output to Cricket Graph software and graphically displayed FIG. 18A shows steady-state levels of U6 expression. FIG. 18B shows transient and steady-state levels of U6ON expression. FIG. 18C shows transient and steady-state levels of expression of U6 after reduction of transcript stability by U6ON production.</p>
    <p>FIG. 19 is a graph demonstrating the dose dependence of U6ON on 7SK, U1, and U3 RNA levels. The nylon filter used to generate FIG. 14A for 293 cells was stripped at 70° C. in prehybridization buffer for 40 minutes followed by reprobing with 7SK, U1 and U3 probes. Quantitative data was obtained by densitometry analysis.</p>
    <p>FIG. 20 is a series of three graphs, demonstrating the stabilities of 7SK (FIG. 20A), U1 (FIG. 20B), and U3 (FIG. 20C) RNA in the presence of U6ON. The nylon filters used to generate FIG. 17 for 20 μg and 5 μg gene transfections were stripped and reprobed as described for the previous figure. Quantitative data was obtained by densitometry analysis.</p>
    <p>FIG. 21 is a graph demonstrating the effect of RNA oligonucleotide stability on 7SK and U1 RNA levels. 20 μg of the gene for U6ON, U6CTcon, U6AS, or promoterless plasmid control DNA was transfected in MDA453 cells followed by RNA isolation and Northern blotting at 48 hours with 7SK and U1 probes. U60N and U6CTcon are stable RNA oligonucleotides while U6AS is unstable. Quantitative data was obtained by densitometry analysis.</p>
    <p>FIG. 22 is a graph demonstrating the effect of U6ON on GAPDH RNA levels. MDA453 cells were transfected with 20 μg of U6ON, U6AS or promoterless plasmid control DNA followed by RNA isolations at 48 hour, 72 hour, and 96 hour time points. Agarose gel Northern blotting (as opposed to polyacrylamide Northern blotting) with a GAPDH probe and densitometry followed.</p>
    <p>FIG. 23 is a set of two graphs, demonstrating the effect of U6ON on co-transfected β-HCG. MDA453 cells were co-transfected with 20 μg of a chimeric oligonucleotide-producing gene or promoterless plasmid DNA followed by β-HCG quantitation by an immunoradiometric assay described in the Examples below. Two representative experiments are provided to demonstrate the variability in β-HCG expression, but lack of correlation with RNA oligonucleotide presence, levels, lengths, or stabilities.</p>
    <p>FIG. 24 (SEQ ID NO:19 through SEQ ID NO:23) is a diagram illustrating a double-stranded DNA probe and triplex forming RNAs. The 43-bp double-stranded DNA probe corresponds to bases -76 to -34 on the HER2/c-erb B2/neu proto-oncogene promoter. The region in bold refers to the homopurine/homopyrimidine tract involved in triplex formation. The CU-rich RNA sequence corresponds to the pyrimidine-rich (parallel) third strand triplex binding motif.</p>
    <p>FIG. 25 is a half-tone reproduction of an autoradiogram, demonstrating triplex blotting with in vitro generated RNA. CU-rich, GA-rich, or control RNA strands were generated in vitro by T3 or T7 RNA polymerases from approximately 0.2 pmoles of linearized plasmid. Equal aliquots from these reactions were added to formamide loading buffer, fractionated by electrophoresis and Triplex blotted as described in Example 18.</p>
    <p>FIG. 26 is a set of two half-tone reproductions of autoradiograms, demonstrating triplex blotting versus Northern blotting with SKBR3 total cellular RNA. Cells were transfected with either 0 μg, 10 μg or 20 μg of the modified U6 plasmid which generates an 82 nucleotide triplex-forming CU-rich RNA oligonucleotide (lanes 2 and 3) or 20 μg of promoterless plasmid DNA (lane 1). FIG. 26A shows equal amounts of total cellular RNA (11 μg per lane) which were added to formamide loading buffer, fractionated by electrophoresis and Triplex blotted as described in Example 17. FIG. 26B shows the same filter after stripping at pH 7.5 and reprobing by Northern blot with radiolabeled single-stranded DNA generated by random priming.</p>
    <p>FIG. 27 (SEQ ID NO:24 and SEQ ID NO:25) is a set of two diagrams displaying (FIG. 27A) the sequence of the mU6 parent vector from which oligonucleotide-producing genes are made, and (FIG. 27B) the sequence of the U6ON generator.</p>
    <heading>DETAILED DESCRIPTION OF THE INVENTION</heading> <p>An improved method for delivery of oligonucleotides, preferably antisense or triplex oligonucleotides, was developed in order to circumvent the obstacles of extracellular degradation, cellular uptake, and intracellular sequestration. This new method is sufficiently general to provide for ribozyme delivery as well. The strategy was designed with the following criteria in mind: (a) oligonucleotides should be generated in high yield within the cell nucleus without significant cell type specificity; (b) they should be sufficiently stable; (c) they should contain minimal secondary structure that could mask binding regions; and (d) they should be of a pre-determined and well-defined sequence and length.</p>
    <p>These criteria were satisfied by constructing one of the preferred embodiments of the oligonucleotide generator invention, containing regulatory regions from the human U6 small nuclear RNA (snRNA) gene and a synthetic double-stranded insert bearing the oligonucleotide to be generated. U6 snRNA, which normally functions in conjunction with several small nuclear riboproteins (snRNPs) in the splicing of premature messenger RNA (Manniatis &amp; Reed (1987) Nature 325: 673-678), is transcribed in high yield by RNA polymerase III, requires only upstream promoter sequences for initiation, and terminates cleanly upon reaching a string of 4-6 thymine residues (Kunkel, G. et al. (1986) Proc. Natl. Acad. Sci. USA 83: 8575-8579; Reddy, R. et al. (1987) J. Biol. Chem. 262: 75-81; Kunkel &amp; Pederson (1989) Nucleic Acids Res. 17: 7371-7379; and Willis, I. M. (1993) European J. of Biochem. 212: 1-11.) Transcript stability is strongly enhanced by 5' γ-monomethyl phosphate capping (Singh &amp; Reddy (1989) Proc. Natl. Acad. Sci. USA 86: 8280-8283; and Singh, R. et al. (1990) Mol. Cell Biol. 10: 939-946) which is directed by a 5' self-complementary hairpin followed by a conserved hexameric AUAUAC sequence (Shumyatsky, G. et al. (1993) Nucleic Acids Res. 21: 4756-4761). Within this description of the present invention the abundant production, intranuclear localization, kinetics of expression, capping, and insert-specific stability of transcripts generated from an oligonucleotide generator of the invention were characterized.</p>
    <p>In a general aspect, the oligonucleotide generators of the invention encompass a construct for producing a specific oligonucleotide within a cell, which construct comprises (a) an U6-type RNA polymerase III promoter; (b) a specific nucleotide sequence from which the specific oligonucleotide is transcribed; and (c) a termination sequence; wherein the components of the construct are operably linked and positioned from 5' to 3' in the order of (a), (b), and (c).</p>
    <p>The terms "U6-type RNA polymerase III promoter" and "U6-type promoter" are used interchangeably herein to refer to a promoter which is able to initiate transcription by RNA polymerase III from a position upstream of the transcribed DNA. "U6-type promoters" have been referred to in the literature, in at least one instance, as RNA polymerase III, type III promoters (Willis, I. (1993) FEBS 212: 1-11). The "U6-type promoter" contains regulatory elements which are necessary and sufficient to facilitate transcription by RNA polymerase III, but these regulatory elements are not themselves transcribed. Thus, U6-type RNA polymerase III promoters include the following promoters: naturally-occurring U6 from higher order eukaryotes (Das et al. (1988) EMBO J. 7(2): 503-512), 7SK (Murphy et al. Cell 51:81-87), H1 RNA gene (Hannon, G. et al. (1991) J. Biol. Chem. 266(34): 22796-22799), U3 snRNA genes in plants (Marshallsay C. et al. (1992) Plant Molecular Biology 19(6): 973-983), and MRP gene (Yuan, Y. and Reddy, R. Biochem. et Biophys. Acta 1089(1): 33-39), as well as any recombinant promoter sequence which is able to initiate transcription by RNA polymerase III without itself being transcribed. Preferably, naturally-occurring U6 promoter is used as the U6-type RNA polymerase III promoter.</p>
    <p>Recombinant U6-type promoters for use in the oligonucleotide generators of the invention will usually have the distal sequence enhancer, proximal sequence element and TATA box as displayed in FIG. 4B. The proximal sequence element and TATA box are required for Polymerase III transcription, and the relative position of these elements to each other and the start site are relatively inflexible. The distal sequence enhancer, however, can be deleted in part or completely, as well as moved closer to or farther away from the proximal sequence element, particularly if a reduction in the level of transcription is desired.</p>
    <p>The oligonucleotide generators of the present invention can be used to facilitate delivery of oligonucleotides to any type of eukaryotic cell. The choice of a specific U6-type promoter is made on the basis of the target cell (cell to be transfected with the generator). U6-type promoters derived from a species that is closely related or at least has a similar promoter sequence to that of the target cell type are preferably used if maximal transcription of the specific oligonucleotide is desired. For example, human U6 promoters, when integrated into an oligonucleotide generator of the invention, would provide higher levels of transcription in human cells, but would be less effective in plant cells. Conversely, a plant U6 promoter would provide higher levels of oligonucleotide production in a plant than a mammalian system.</p>
    <p>In a general aspect, the invention also features a construct for generating a specific oligonucleotide within a cell, which construct includes a nucleotide sequence from which the transcript is the specific oligonucleotide, flanked on the 5' end by a stabilizing region and on the 3' end by a termination sequence, and a promoter at the 5' end of the stabilizing region.</p>
    <p>In preferred embodiments, the promoter and the stabilizing region are derived from a RNA polymerase III gene, and in particular embodiments it is derived from a human U6 small nuclear RNA gene. In particularly preferred embodiments the stabilizing region comprises the first approximately 25 nucleotides of the human U6 small nuclear RNA gene, as this includes a portion that forms a stable hairpin in the product and makes a "cap" that prevents degradation of the product at the 5' end. Alternatively, any hairpin-forming sequence can be used, and needn't be derived from a native source.</p>
    <p>Generally, the optional "stabilizing region" of the oligonucleotide generator can be of any length, so long as the resulting RNA transcript of the oligonucleotide generator is predicted to form a hairpin structure by a computer program that models and predicts secondary structure of RNA. These computer programs include, for example, the algorithm described in Example 8. While longer stabilizing regions can be used, they are generally between about 16 and about 50 nucleotides in length. Most preferably the segments of the stabilizing region that are predicted to form base pairs in the resulting RNA transcript are continuous and perfectly complimentary, containing no mismatches. However, such mismatches are tolerated within the segments of the stabilizing region that are predicted to form base pairs, so long as they do not completely disrupt the hairpin structure as predicted by the computer modeling program. Usually the mismatched bases are less than about 1 in 5 of the nucleotides in the complimentary regions of the predicted hairpin structure, and almost always less than about 1 in 4.</p>
    <p>A "stabilizing region" when present is preferably able to reduce the rate of intracellular degradation for the resulting RNA transcript as compared with an identical RNA transcript that does not contain the region transcribed from the stabilizing region. Methods of measuring the intracellular degradation rates of RNA are known in the art, and include the methods described in Example 11.</p>
    <p>As described above there is a portion of the 5' end of the U6 gene that "makes a `cap` that prevents degradation of the product at the 5' end", and is included in some preferred embodiments of the oligonucleotide generators of the invention. The stability of the resulting RNA transcripts is strongly enhanced by this 5' γ-monomethyl phosphate capping, which is directed by a 5' self-complementary hairpin followed by a conserved hexameric AUAUAC or AUAUCC sequence, preferably AUAUAC sequence, in the RNA transcript. Thus, when present in a oligonucleotide generator or generator vector of the invention the sequence on the coding strand for the capping segment is ATATCC or ATATAC. This optional "capping segment" in the oligonucleotide generators of the invention is usually only operable when it is immediately downstream of the hairpin structure of a stabilizing region, although the first two nucleotides of the capping segment may form part of the hairpin structure. When the "capping segment" is present, the hairpin structure of the stabilizing region is preferably about 20 to about 30 nucleotides in length, more preferably about 20 nucleotides in length, most preferably the hairpin structure from the 5' end of the U6 transcript.</p>
    <p>The "specific oligonucleotide" may be any oligonucleotide that is desired to be transcribed within the cell and includes, for example, a triplex-forming oligonucleotide, an antisense oligonucleotide, a ribozyme, or a combination of these. Thus, the oligonucleotide generators of the present invention can be used to deliver RNA oligonucleotides for any purpose. For example, the oligonucleotide generators of the invention may be used to deliver tumor suppressing RNAs (Rastinejad, F. et al. (1993) Cell 75: 1107-1117; and Wickens, M. and Takayama, K. (1994) Nature 367: 17-18). Particularly favorable results are obtained using the construct of the invention for intracellular production of oligonucleotides in the size range between about 10 and about 60 nucleotides (and more particularly in the range between about 20 and 50 nucleotides), although the success of the invention is not strictly dependent upon the length of the oligonucleotide product. Oligonucleotide products are usually less than about 500 nucleotides in length.</p>
    <p>In preferred embodiments, the termination region includes, in addition to a termination sequence (e.g.. TTTT), a sequence between the termination sequence and the 3' end of the oligonucleotide sequence, to provide a 3' tail on the product; this aids in protecting the product from degradation at the 3' end; and the tail may be constructed to form a hairpin or other protective structure. Thus, the termination region includes at a minimum a transcription termination sequence recognized by RNA polymerase III, i.e. a stretch of four to six thymine nucleotides on the coding strand of the oligonucleotide generator.</p>
    <p>The optional region of the oligonucleotide generator that provides the "3' tail" on the RNA transcript may be designed such that it forms a hairpin of any size, generally between about 16 and about 50 nucleotides in length. Also this region may be designed to form a lariat structure by base pairing with the nucleotides transcribed from a stabilizing region or a region of the transcript that is upstream of the specific oligonucleotide. Preferably, when a lariat forming 3' tail is used, the oligonucleotide generator also provides for the transcription of a hairpin structure in the 5' stabilizing region immediately followed by a capping segment. Thus, some of the RNA transcript may be capped with 5' γ-monomethyl phosphate prior to the formation of the more thermodynamically stable lariat structure.</p>
    <p>The oligonucleotide generators of the invention can optionally be designed such that it produces a transcript with a predicted lariat conformation, wherein the stem of the lariat formed by the Watson-Crick base pairing of a 3' tail in the termination region and a portion of the transcript which is upstream of the specific oligonucleotide. The stem of the lariat structure may be of any length as long as a stable lariat structure is predicted by a computer program that models and predicts secondary structure of RNA. These computer programs include, for example, the algorithm described in Example 9. While longer stem regions can be used, the lariat stem is generally between about 8 and about 30 nucleotides in length. Most preferably, the stem of the lariat is predicted to form continuous base pairs for the stems entire length, containing no mismatches. However, such mismatches are tolerated within the segments of the lariat stem that are predicted to form base pairs, so long as they do not completely disrupt the lariat structure as predicted by the computer modeling program. Usually the mismatched bases are less than about 1 in 5 of the nucleotides in the lariat stem, and almost always less than about 1 in 4 of the nucleotides in the lariat stem.</p>
    <p>Thus, the present invention encompasses oligonucleotide generators, comprising from 5' to 3': (a) an U6-type RNA polymerase III promoter; (b) a specific nucleotide sequence from which a specific oligonucleotide can be transcribed; and (c) a termination sequence; wherein the components of the oligonucleotide generator are operably linked; and wherein the oligonucleotide generator is capable of being transcribed by RNA polymerase III to produce a transcript comprising the specific oligonucleotide; and further comprising: a 5' tail from which a first lariat-forming sequence can be transcribed; and a 3' tail from which a second lariat-forming sequence can be transcribed; wherein the 5' tail is operably linked and positioned between the U6-type RNA polymerase III promoter and the specific nucleotide sequence; wherein the 3' tail is operably linked and positioned between the specific nucleotide sequence and the termination sequence; wherein the oligonucleotide generator is capable of being transcribed by RNA polymerase III to produce a transcript comprising from 5' to 3' the first lariat-forming sequence, the specific oligonucleotide, and the second lariat-forming sequence; and wherein the transcript is predicted to from a stable lariat structure by Watson-Crick base pairing between the nucleotides of the first lariat-forming region and the second lariat-forming region.</p>
    <p>In particularly preferred embodiments, the stabilizing and termination portions of the construct are derived from the same source, and the construct is made by providing a vector containing, in sequence but not necessarily contiguous sequence, a promoter of the U6-type, a stabilizing 5' portion of the source gene (which may preferably be a part of a type III gene such as the first -25 nucleotides of the human U6 gene), a XhoI site, a NsiI site, and a 3' portion of the source gene including at least a termination sequence (which may preferably be a part of the same type III gene such as a ˜20 nucleotide 3' portion of the human U6 gene). Of course, the XhoI and NsiI restriction sites can be replaced with any first and second unique restriction enzyme sites to facilitate insertion of the specific nucleotide sequence.</p>
    <p>Thus, in another general aspect, the present invention encompasses vectors which comprise any of the oligonucleotide generators described herein with the specific nucleotide sequence removed. These "generator vectors" may be used to construct an oligonucleotide generator of the invention. Preferably these generator vectors comprise two unique restriction enzyme sites, one on each side of the position of the generator vector into which the specific nucleotide sequence must be inserted to form an oligonucleotide generator of the invention. These unique restriction enzyme sites, when present, serve to facilitate insertion of the specific nucleotide sequence into the generator vector. More preferably the two restriction enzyme sites are not recognized by the same enzyme.</p>
    <p>The U6 gene was chosen to provide the regulatory components for some of the preferred embodiments of the invention, as it is transcribed in nearly all mammalian cells in high yield; is transcribed constitutively; requires only upstream promoter sequences for transcription; and initiates and terminates cleanly and precisely (Kunkel, G. et al. (1986) Proc. Natl. Acad. Sci. USA 83: 8575-8579; Reddy, R. et al. (1987) J. Biol. Chem. 262: 75-81; Kunkel &amp; Pederson (1989) Nucleic Acids Res. 17: 7371-7379; and Willis, I. M. (1993) European J. of Biochem. 212: 1-11.) In addition, the U6 gene contains a sequence-specific signal that directs the 5' capping of transcripts by a γ-monomethyl phosphate which greatly augments transcript stability (Singh &amp; Reddy (1989) Proc. Natl. Acad. Sci. USA 86: 8280-8283; Singh, R. et al. (1990) Mol. Cell Biol. 10: 939-946; and Shumyatsky, G. et al. (1993) Nucleic Acids Res. 21: 756-4761). The output of this gene results in the abundant intracellular production of short, sequence-specific RNA oligonucleotides containing a 5' nuclease-resistant γ-monomethyl phosphate cap.</p>
    <p>In another general aspect, the invention features a method for intracellularly generating an oligonucleotide of interest in a subject or in cells or tissues derived from a subject, including administering to the subject the construct according to the invention in a form that permits entry of the construct into the subject's cells. The administration may be carried out by any of various techniques known, for example, in the art of gene therapy (see, for example, J. W. Larrick and Kathy L. Burck, Gene Therapy, Application of Molecular Biology, Elsevier, Holland, 1991); or by administration in immunoliposomes, according to techniques known in the immunotherapeutic art, wherein the immunoliposome may be targeted to, and its contents delivered into, a particular cell type (such as, for example, a breast cancer cell); or by localized injection at a site for treatment; or by way of mucosally-lined passages; or via the airways, for example; all depending upon the particular treatment.</p>
    <p>Advantages of the oligonucleotide generator of the invention include high yield and continuous production of oligonucleotides within the cell, and minimal secondary structure within binding regions on the oligonucleotide products, except where the secondary structure is an important functional part of a ribozyme contained in the specific oligonucleotide.</p>
    <p>Viral integration into the chromosome of the cell can, according to the invention, confer permanence to the oligonucleotide-based antiviral, anticancer, or antiprotein gene regulation.</p>
    <p>The boxed regions in FIG. 2 (i.e. A box, TATA box) represent natural invariant promotor and terminator sequences with their lengths in base pairs identified below. The straight line regions represent areas where sequence-specific oligonucleotide portions can be inserted. These oligonucleotide portions may be any oligonucleotide to be transcribed, for example, triplex forming oligonucleotides, antisense oligonucleotides, ribozymes, or a combination thereof. The oligonucleotides can be designed for binding to different regions of different DNA or RNA targets, to different regions of the same DNA or RNA target, or to the same region of the same DNA or RNA target. Decisions as to vector design would be based upon whether the experimenter wanted to hit multiple targets broadly or a single target intensely. In addition, as shown in FIG. 2, self complementary ends can be generated on the oligonucleotide which may form small double-stranded hairpin loops. Such double-stranded ends will protect against exonuclease activity, prolong oligonucleotide half-life within the cell, and prevent other oligonucleotide secondary structures from masking key binding regions.</p>
    <p>Once the promotor and oligonucleotide sequences have been attached in the correct orientation, they can be inserted into a viral vector (such as an adenovirus or retrovirus) and integrated into the chromosomes of the cells of interest as, for example, the methods described by Sullenger et al. (1990) Molecular and Cellular Biology 10(12): 6512-6523. The result of the integration would produce a continual and large supply of short length oligonucleotides generated by the cell's own transcriptional machinery based upon natural RNA polymerase III promotion initiation, and termination sequences. By varying the multiplicity of viral vector infection, the degree of gene regulation can be modulated.</p>
    <p>The advantages of such a RNA polymerase III system for generating antisense and triplex forming oligonucleotides are manifold. RNA polymerase III transcribes at a nearly constant rate and high frequency in almost all mammalian cell types, in marked contrast to the widely used RNA polymerase II based systems which transcribe at lower frequencies and are highly variable with time and cell type. The RNA polymerase III transcription initiation and termination processes are also highly efficient allowing for clean transcription start and stop sites, usually within 1-2 nucleotides. This feature too stands in contrast with RNA polymerase II approaches which generate widely varying transcript lengths with long polyadenylated tails and often hundreds to thousands of trailing 31 nucleotides. In addition, since RNA polymerase III normally generates transcripts that are both selectively transported to the cytoplasm (rRNA, tRNA) and maintained in the nucleus (various snRNA), it may be possible to utilize similar sequences to keep the designed oligonucleotide sequences primarily in the cell compartment of interest.</p>
    <p>Several reports have emerged citing the ability of a transfer RNA (tRNA) gene to be used as a carrier for an antisense oligonucleotide (Izant, J. G. (1992) In Gene Regulation: Biological Activity of Antisense RNA and DNA. R. P. Erickson, J. G. Izant, eds. Raven Press (New York), pp. 183-196; and Sullenger, B. et al. (1990) Mol. Cell. Biol. 10: 6512-6523). In one case the oligonucleotide was placed within one of the hairpin loops in the internal region of the gene (Izant, J. G. (1992) In Gene Regulation: Biological Activity of Antisense RNA and DNA. R. P. Erickson, J. G. Izant, eds. Raven Press (New York), pp. 183-196), in another instance, the oligonucleotide was placed on the 3' tailing region of the tRNA (Sullenger, B. et al. (1990) Mol. Cell. Biol. 10: 6512-6523). Both reports demonstrate an antisense effect against a target mRNA using these RNA transcripts, but did not result in the RNase H-mediated cleavage of the target. As with the U6 system, these approaches utilize polymerase III for oligonucleotide production. However, in both of these systems, the oligonucleotide is within a much larger sequence containing key promoter elements. Consequently, the antisense binding sequence is constrained to regions where the tRNA can be transcribed normally, and is likely masked by structure of the much larger tRNA sequence, or interfere with intragenic promoter function. In addition, these tRNA strategies may result in oligonucleotides which are close enough in structure to native tRNA molecules to disrupt translation of mRNA into proteins or result in the transport of the chimeric oligonucleotide out of the nucleus.</p>
    <p>By contrast, U6 and 7SK -based systems require no internal promoter as opposed to the tRNA chimeric genes, and thus can be composed almost entirely of the oligonucleotide of interest. This feature is important in determining and designing the secondary structure of the transcript to maximize the oligonucleotide binding sequence while also maximizing stabilizing flanking sequences. In one preferred embodiment, a 5' U6 flanking sequence was maintained to invoke the enzyme(s) which recognize and cap the transcript. Another technique which may provide both oligonucleotide accessibility and stability is to build in a 5' and 3' self-complementary hairpin, creating a lariat-like structure with the oligonucleotide within the loop. As the RNA is small, secondary structure prediction algorithms have utility in the design of these transcripts. The U6 system also has nearly all of the native U6 deleted from the resulting chimeric gene. This feature eliminates the possible sequestering of oligonucleotide within the active spliceosome, as well as eliminates possible toxicity from dysfunctional chimeric U6 RNA that can carry out only a subset of the functions of the native U6 RNA.</p>
    <p>There is another chimeric gene that has been developed by others, which utilizes the promoter of U2 snRNA gene (Izant, J. G. (1992) In Gene Regulation: Biological Activity of Antisense RNA and DNA. R. P. Erickson, J. G. Izant, eds. Raven Press (New York), pp. 183-196). Like the U6-based chimeric gene, the promoter is entirely upstream, but unlike the U6-based chimeric gene, it is transcribed by RNA polymerase II. These chimeric genes also retain nearly all of the native U2 gene which is considerably longer and more complex in structure than is the tRNA gene. Thus, accurate predictions of structure and accessibility of these oligonucleotides are significantly more difficult to obtain a priori. However, the kinetics of expression, stability, intracellular localization, and absolute RNA transcript levels have not been characterized in these chimeric genes.</p>
    <p>In the oligonucleotide generators of the invention, the optional use of viral integration allows for the modulation of infection with the oligonucleotide generator, and thus the amount of transcripts generated. Such a feature will be invaluable in generating dose-response curves to a given target. Viral integration will also generate cells that are healthy and otherwise functioning and replicating normally, while producing the constant supply of oligonucleotides. As opposed to plasmid transfection techniques, the integration is permanent and does not require continuous selective pressure. Once the producer lines of virally integrated cells are generated, a variety of primary cells and cell lines can be infected, a variety of combination and concentrations of infection can be tried, and the effect of various strategies of gene regulation can be compared (i.e. antisense versus triplex).</p>
    <p>An additional advantage arises from the capability to hit multiple targets by the use of more than one oligonucleotide sequence within a transcript. Multiple sequences could target the start codon, a splice site, and the ribosomal binding sequence within a single mRNA, thus increasing the likelihood of blocking subsequent translation of that protein. Alternatively, multiple mRNA's could be used as targets leading to downregulation of multiple proteins along a common pathway.</p>
    <p>We have designed, constructed, and tested a system for the intracellular generation of short sequence-specific oligonucleotides in extremely high yield for the purposes of gene regulation. The system, when transfected into cells by electroporation, produces the desired oligonucleotide in high quantity. We have analyzed the production of our transcript by both Northern and RNAse Protection assays, and both analyses show that the quantity of oligonucleotide produced is at least several orders of magnitude higher than the quantity of a typical messenger RNA. However, should lower concentrations be warranted, smaller doses of our system result in a lower quantity of the oligonucleotide. Experimentation has also shown that the oligonucleotide is sufficiently stable (i.e., it is composed of only one strong band; no degradation products are observable), and is of an exact "user-definable" length.</p>
    <p>This system has the advantage of user flexibility in oligonucleotide sequence, length, and yield not capable with current intracellular nucleic acid delivery techniques. Its intracellular location and seemingly constant production bypass uptake and sequestration concerns which limit the utility of current extracellular nucleic acid delivery techniques. This system is also compatible with viral integration for conferring permanence in oligonucleotide generation. We expect our system to result in enhanced capacity for gene regulation by antisense, triplex, or any other application of oligonucleotides in cells.</p>
    <p>The in vivo oligonucleotide generator according to the invention can be used for permanently downregulating or upregulating cellular proteins of interest. Examples of effective antisense, triplex-forming, and rybozyme oligonucleotides, as well as strategies for developing new efficacious oligonucleotides are known in the art and are thus not described exhaustively herein. Examples of proteins for which downregulation might be indicated include transplantation antigens (e.g. ICAM, MHC classes I and II antigens), hormones, cellular adhesion molecules, clotting proteins, oncogene products, and proteins of viruses which form latent infections in human, including but not limited to Herpes, HIV, CMV, and human papilloma viruses. Information concerning the use of oligonucleotides to downregulate transplantation antigens is found in PCT/US 93/00797. Examples of proteins for which upregulation might be indicated include underexpressed proteins in deficiency states.</p>
    <p>The in vivo oligonucleotide generator according to the invention can be used for control of viral, parasitic or mycotic replication, by targeting key replication and protein sequences (such as for example the T antigen in SV40) and to prevent viral infection by downregulating necessary attachment proteins.</p>
    <p>The in vivo oligonucleotide generator according to the invention can be used to block the proliferative effect of cellular or viral oncogenes such as myc, myb, or fos and therefore reduce or prevent neoplasm growth and tissue invasion.</p>
    <p>The in vivo oligonucleotide generator according to the invention can be used to create new genetically altered organisms, cells and tissues, useful for example in agriculture and in human and veterinary medicine.</p>
    <p>In the Examples section below we demonstrate that the levels of production of the RNA transcripts, e.g. U6ON, produced by the oligonucleotide generators of the invention can rival and even exceed those of the native U6 snRNA (5×10<sup>4</sup> to 5×10<sup>6</sup> copies/cell) and, like native U6 RNA, these RNA oligonucleotides can be capped and remain intranuclear in controlled concentrations which may range from 160 μM to 16 mM. U6ON production occurs rapidly upon transfection and can still be detected up to one week after transfection, as 50% inactivation of the parent plasmid from steady-state requires approximately 4 days. This long-lived production suggests that in slowly growing cell populations, longer time points (i.e., longer than the typical 48 to 72 hours) may be used for measuring a biological response from a transient transfection of the oligonucleotide generator. In addition, this long-lived production may allow for the detection of biological effects of antisense or triplex oligonucleotides after transient transfection, even when the target mRNA and/or protein is fairly stable. For example the half-life of U6ON transcripts is estimated to be 1 hour; however, transcript stability is dependent upon the sequence of the oligonucleotide insert.</p>
    <p>The sequence of the insert is able to affect the ability of the transcript to retain the initial 5' hairpin structure, and thus the ability to obtain a 5' cap. Correlations have been observed between RNA secondary structure predictions and experimental determinations of transcript levels containing different oligonucleotide insert sequences. When the algorithm predicted the loss of the 5' hairpin, dramatic decreases in transcript levels were seen experimentally after electroporation and Northern blotting. Low transcript levels reflect either a decrease in production or an increase in degradation. As the U6 gene has consistently been shown to require only upstream promoter sequences for transcription (Kunkel &amp; Pederson (1989) Nucleic Acids Res. 17: 7371-7379; and Willis, I. M. (1993) European J. of Biochem. 212: 1-11), the low transcript levels seen with some oligonucleotide insert sequences cannot be due to a decrease in production, and must therefore are due to an increase in degradation. U6 stability has been attributed primarily to its 5' cap and its extensive hybridization with self+U4 (Terns, M. P. et al. (1993) Genes and Development 7: 1898-1908). As all of the chimeric oligonucleotide generator genes have the U6/U4+self hybridization regions deleted, it becomes more likely that differences in stability are due to the presence or absence of a 5' cap. Finally, correlations between modeling studies and experimental data point toward the retention of a 5' hairpin structure as a means of linking oligonucleotide sequence, retention of the 5' cap, transcript stability, and thus, absolute transcript levels. Consequently, in the design of an oligonucleotide insert, the overall secondary structure of the RNA transcript has importance in optimizing transcript stability and steady-state transcript levels.</p>
    <p>There are a variety of potential applications for a system which generates sequence-specific short RNA oligonucleotides in high yield within the cell nucleus. For example, antisense oligonucleotides can be generated intracellularly in levels several orders of magnitude greater than typical sense mRNA molecules and far greater than antisense mRNA generated by more traditional vectors that rely on RNA polymerase II for transcription. In addition, the ability to produce short transcripts minimizes the chances that the binding region for a targeted biological effect is masked by secondary structure which can occur with much larger antisense mRNA transcripts that do not have pre-determined length or sequence.</p>
    <p>The present invention also provides a means for the generation of intranuclear triplex RNA oligonucleotides. Pyrimidine-rich triplex RNA oligonucleotides which bind in a parallel fashion with respect to the corresponding purine strand of a homopurine/homopyrimidine duplex (Felsenfeld, G. et al. (1957) J. Am. Chem. Soc. 79:2023-2024), have a greatly increased binding affinity over their triplex DNA oligonucleotide counterparts (Roberts &amp; Crothers (1992) Science 258: 1463-1467; and Escude, C. et al. (1993) Nucleic Acids Res. 21: 5547-5553). The high concentration of a triplex RNA oligonucleotide which is both generated and retained in the nucleus in vast excess over its DNA duplex target may drive triplex binding to a critical element on a gene promoter, and block subsequent gene expression.</p>
    <p>In addition to antisense and triplex oligonucleotides, the vectors of the present invention may be used in generating longer length ribozyme transcripts for use in binding and cleaving target mRNA. Combinations of triplex and antisense oligonucleotides as well as ribozymes targeted to a single gene may also yield synergistic approaches to the selective repression of gene expression.</p>
    <p>Other potential uses of the present invention include the quenching of specific single-stranded nucleic acid binding proteins by short RNA sequences, or generating self-complementary RNA hairpins that can mimic known DNA binding consensus sequences, thus quenching specific DNA binding transcription factors. These embodiments of the present invention rely on the nuclear localization of abundant and sufficiently stable oligonucleotides with short and fully defined sequences to allow for reasonable approximation of secondary structure.</p>
    <p>Triplex Blotting</p>
    <p>In another aspect, the invention features a method for screening oligonucleotide sequences that are candidates for triplex formation with a double-stranded DNA target site, including steps of identifying a sequence corresponding to the target site, producing and isolating oligonucleotide sequences that appear likely to bind to the target site, prehybridizing and hybridizing the isolated oligonucleotide sequences with a labeled double-stranded probe having the sequence of the target site, removing unbound probe from the oligonucleotide sequences, and detecting the label to identify probe-bound oligonucleotides.</p>
    <p>In its most general aspects the triplex blotting invention described herein embodies two different methods of measuring triplex formation and the strength of triplex binding. The first method comprises (a) attaching a single-stranded nucleic acid to a solid support; (b) contacting the solid support with a fluid comprising a labeled double-stranded probe; (c) separating the unbound probe from the solid support; and (d) quantifying the amount of labeled double-stranded probe bound to the solid support. The second method comprises (a) attaching a double-stranded nucleic acid to a solid support; (b) contacting the solid support with a fluid comprising a labeled single-stranded probe; (c) separating the unbound probe from the solid support; and (d) quantifying the amount of labeled single-stranded probe bound to the solid support.</p>
    <p>The attachment of single and double-stranded nucleic acids to solid supports can be accomplished by any method known in the art including, but not limited to, Northern blotting transfer techniques, slot or dot blotting techniques, or by the nucleic acid chip technology of Affymax. The single-stranded nucleic acid used in the triplex blotting method of the invention can be either DNA or RNA. The double-stranded nucleic acid used in the triplex blotting method of the invention can be DNA:DNA, DNA:RNA, or RNA:RNA duplexes. In all cases modified nucleotides, and nucleotide linking groups (groups to replace internucleotide phosphodiester linkages) can be used.</p>
    <p>The label on the labeled single and double-stranded probes can be any nucleic acid label known in the art including, but not limited to, radionuclides (e.g. <sup>32</sup> P and <sup>35</sup> S), fluorescent dyes, and biotin.</p>
    <p>The separation of the unbound probe from the solid support can be accomplished by any washing technique known in the art for use with Northern, Southern and slot blotting. Such washing techniques may include multiple washes at increasing stringencies while monitoring the ratio of specifically bound labeled probe to background labeled probe, for example with a geiger counter for probes labeled with <sup>32</sup> P as is typically done with Northern blotting. Increasing stringency can be accomplished by reducing the concentration of salts generally, and Na<sup>+</sup> particularly, by increasing the temperature, or by increasing the pH.</p>
    <p>The quantification of the amount of labeled single- or double-stranded probe bound to the solid support may be preformed by any method known in the art for the quantification of the particular label used. For example in the case of a radionuclide the quantification can be performed by autoradiography followed by densitometry or scintillation counting.</p>
    <p>In one preferred embodiment of the triplex blotting invention described herein, is for use in drug screening assays. For such assays, the single-stranded nucleic acid is attached to the solid support and the double-stranded nucleic acid is labeled and used as a probe. In this case the probe is designed to mimic a particular target site for triplex binding. A large number of single-stranded nucleic acids with different sequences are attached to one or more solid supports in an ordered and cataloged manner. The solid support is contacted with a fluid comprising the labeled double-stranded probe/target site. After the unbound probe is separated from the solid support, the amount of labeled double-stranded probe bound to each different single-stranded nucleic acid is quantified. The single-stranded nucleic acid with the highest amount of labeled double-stranded probe bound to its position on the solid support is the best candidate for a triplex-forming oligonucleotide.</p>
    <p>The practice of the present invention will employ, unless otherwise indicated, conventional techniques of protein chemistry, molecular biology, microbiology and recombinant DNA technology, which are known to one of ordinary skill in the art. Such techniques are explained fully in the literature. See, e.g., R. K. Scopes, Protein Purification Principles and Practice, 2nd ed. (Springer-Verlag 1987); S. Colowick and N. Kaplan, Eds., Methods in Enzymology (Academic Press, Inc.); Sambrook, Fritsch &amp; Maniatis, Molecular Cloning: A Laboratory Manual, 2nd ed. (Cold Spring Harbor Laboratory Press 1989); M. J. Gait, Ed., Oligonucleotide Synthesis (1984); and D. M. Weir and C. C. Blackwell, Eds., Handbook of Experimental Immunology, Vols. I-IV (Blackwell Scientific Publications 1986).</p>
    <p>All patents, patent applications and publications mentioned herein, whether supra or infra, are hereby incorporated herein by reference in their entirety.</p>
    <p>Described below are examples of the present invention which are provided for illustrative purposes, and not to limit the scope of the present invention. In light of the disclosure, numerous embodiments within the scope of the claims will be apparent to those of ordinary skill in the art.</p>
    <heading>EXAMPLES</heading> <heading>Example 1</heading> <p>Initial Characterization of In Vivo Oligonucleotide Generator</p>
    <p>Expression and Lack of Toxicity</p>
    <p>Tests of the system using several different cell populations (breast cancer MDA453 cells; human embryonic kidney 293 cells) and have produced comparable results. Normal U6 small nuclear RNA which utilizes identical transcriptional assemblies were unaffected by transfections of up to 20 μg of the chimeric ON-producing gene; Cells producing the oligonucleotide grow normally and demonstrate no morphologic changes at these levels of transfection.</p>
    <p>Kinetics</p>
    <p>We have examined how fast the oligonucleotide generator can produce RNA oligonucleotides after cell transfection with the parent plasmid. We electroplated cells and examined total RNA at 12 hours, 24 hours, and 48 hours post-transfection. Using a system substantially as described above, the oligonucleotide generator reaches a near steady-state level at about 12 hours post-transfection. These data indicate that the oligonucleotide is formed rapidly, and therefore may have immediate activity once the system is introduced into cells.</p>
    <p>Dose-response</p>
    <p>We have examined the effect of increasing the amount of the oligonucleotide generator in cells with respect to RNA oligonucleotide levels in cells 48 hours after transfection. To summarize, when cells are transfected with 0, 5, 10, 20, or 40 micrograms of the generator using a system substantially as described above, the amount of RNA oligonucleotide isolated from cells increases linearly. Thus, it is possible to drive cells to produce enormous levels of a single RNA oligonucleotide. At 40 microgram transfections, we estimate that we can produce on the order of 1-5 million copies per cell. This level is very far in excess of that achievable with typical mRNA species.</p>
    <p>Effect of the Oligonucleotide Generator on the Cells</p>
    <p>As the oligonucleotide generator produces such high levels of RNA oligonucleotides in the cell, we examined the effect of the oligonucleotide generator on two parameters, namely, cell growth and normal U6 RNA levels. After transfection of 10<sup>7</sup> cells with 0, 5, or 10 micrograms of an oligonucleotide generator substantially as described above, no effect on the ability of cells to divide and grow normally was detected, and no significant effect on native U6 levels, which might have indicated competition for transcription factors, was detected. However, at levels of transfection ≧20 μg of the U60N generator, cellular levels of U6 RNA decreased. Further analysis indicated that this decrease was due to a lessened stability of the cellular U6 RNA.</p>
    <heading>Example 2</heading> <p>Targeting the HER2 Proto-Oncogene Promoter with Triple Helix Forming Oligonucleotide</p>
    <p>Oligonucleotides designed to form local triple helices with the HER2 promoter and interfere with transcription factor binding may potentially inhibit HER2 receptor expression and growth of HER2 overexpressing breast cancers.</p>
    <p>We have identified, using promoter deletion mapping, a 125 base-pair core proximal promoter region capable of conferring greater than 30 fold variation in HER2 transcriptional activity. Within this region are several putative protein binding sites (including CAAT, TATA, and a GAGGAA ets-related response element), essential for promotor activity, which both surround and overlap a 41 base-pair purine-rich sequence. Gel mobility shift experiments demonstrate that this 41 base-pair sequence is compatible with triple helix formation by both parallel (pyrimidine-rich) and antiparallel (purine-rich) triplex binding motifs.</p>
    <p>We tested oligonucleotides ranging in length from 26 to 41 nucleotides. Increasing oligonucleotide length corresponds with increasing gel retardation, however triplex formation is relatively unaffected by length, demonstrating only sequence-specificity. Antiparallel triplex formation is strongly dependent upon Mg<sup>++</sup> (0.1 to 10mM), while parallel triplex formation has an additional sensitivity to pH, demonstrating significant loss of hybridization from pH 7.0 to 8.0. These triple helix forming oligonucleotides can inhibit binding of ets-related proteins such as PU.1 to the HER2 promoter.</p>
    <p>In addition, transfection of synthetic genes capable of constitutively generating short triple helix forming oligonucleotide transcripts is testing the ability of these oligonucleotides to downregulate both native HER2 receptor levels as well as a co-transfected HER2-driven chloramphenicol acetyltransferase reporter gene. Triple helix forming oligonucleotides and effect of HER2 downregulation on proliferation and tumorigenicity of HER2-positive breast cancer cells have a therapeutic potential.</p>
    <heading>Example 3</heading> <p>Construction of the Chimeric U6ON Gene</p>
    <p>The human U6 gene cloned within the SmaI site of pGeml (Promega, Madison, Wis.), along with a mutant human U6 gene with bases +25 to +55 replaced by an XhoI restriction site (with A/C substitution at base 24) were generously provided by G. Kunkel and T. Pederson (Kunkel &amp; Pederson (1989) Nucleic Acids Res. 17: 7371-7379). The mutant U6 gene was recloned into a pBluescript (Stratagene, La Jolla, Calif.) vector to produce single-stranded phage and to allow two site-directed mutations at bases +86 and +88 (T to G and G to A, respectively) to create a unique NsiI restriction site. This plasmid, mU6, was then recloned back into pGEM1, cut with XhoI and NsiI, and religated with a synthetic 38 bp duplex fragment bearing 5' XhoI and 3' NsiI compatible overhanging ends (Keystone Laboratories, Menlo Park, Calif.). Incorporation of the synthetic oligonucleotide was verified by Maxam and Gilbert dideoxy DNA sequencing. The resulting transcript arising from this vector, U6ON, is 25 nucleotides shorter than native U6 RNA (82 vs. 107). Sequences of the upper strand of U6ON and other various inserts are as listed:</p>
    <p>U6ON:(SEQ ID NO:1)</p>
    <p>5' TCGACTCCTCTTCCTCCTCCACCTCCTCCTCCCATGCA 3'</p>
    <p>U6CTcon(SEQ ID NO:2):</p>
    <p>5' TCGACCTCCCTTCCCTTCCCTTCCCCTTCCTCCATGCA 3'</p>
    <p>U6AS(SEQ ID NO:3):</p>
    <p>5' TCGACATGAGCATTCATCAGGCGGGCAAGAATGTGATGCA 3'</p>
    <p>MU6(SEQ ID NO:4):</p>
    <p>5' TCGAGCATGGCCCCTGCGCAAGGATGACACGCAAATGCA 3'</p>
    <p>FIG. 4 illustrates schematically the structure of the native U6 snRNA gene, the modifications involved in generating the chimeric gene, and the resulting RNA oligonucleotide transcript, U60N. As shown, the upstream promoter and enhancer regulatory regions, the initial 25 bp (with A/C substitution at base 24), and the terminal 19 bp of the native U6 gene were retained in the chimeric gene. Mutagenesis and restriction digest removed the remaining native U6 internal sequence in order to create a hybrid of native and synthetic sequences in a gene designed to express any oligonucleotide of interest. As shown in FIG. 4C, the transcribed RNA oligonucleotide was designed to retain the original 5' hairpin, in order to obtain the 5' γ-monomethyl phosphate cap. This initial sequence is followed by the sequence-specific oligonucleotide and the native U6 uridine-rich 3' terminus. The total length of the resulting transcript is a function of the synthetic oligonucleotide inserted, a 38 bp duplex was inserted yielding a U6ON of 82 nucleotides.</p>
    <heading>Example 4</heading> <p>Cell Culture and Gene Transfection</p>
    <p>The human embryonic kidney cell line, 293, and the human breast cancer cell line, MDA453 (ATCC, Rockville, Md.), were transfected by electroporation (250 V, 960 μF) with 5 μg to 40 μg of the chimeric gene or promoterless plasmid DNA. Cell viability after transfection ranged from 40-60% (with 293 cells showing slightly higher tolerance to electroporation than MDA453 cells) and was unaltered by increasing gene transfection dosage up to 40 μg/10<sup>7</sup> cells. 293 cells were cultured in minimal essential media with Earle's basic salt solution, 10% fetal calf serum supplemented with 100 U/ml penicillin and streptomycin in 5% CO<sub>2</sub> incubators. MDA453 cells were cultured in Leibovitz L-15 media with 10% fetal calf serum supplemented with 100 U/ml penicillin and streptomycin in the absence of CO<sub>2</sub>. Where indicated, cell counts were obtained by Coulter counting.</p>
    <heading>Example 5</heading> <p>RNA Isolation and Northern Blotting</p>
    <p>Total cellular RNA was isolated 48 hours after transfection by the guanidinium isothiocyanate/cesium chloride centrifugation technique (Glisin, V. R. et al. (1974) Biochemistry 13: 2633-2643). RNA (10-20 μg, as indicated in Brief Description of the Drawings) was electrophoresed in 8% polyacrylamide/7 M urea gels, electroblotted onto nylon filters (Amersham, Arlington Heights, Ill.) in 8 mM Na<sub>2</sub> HPO<sub>4</sub> /17 mM NaH<sub>2</sub> PO<sub>4</sub> buffer for 3 hours at 350 mA, and then UV cross-linked onto the filters for 2 min. Probes to detect native U6 and the generated RNA oligonucleotide were radiolabeled by random-priming from an 800 bp BamHI/EcoRI fragment taken from the original U6 gene or the chimeric gene within pGem1. After membrane hybridization and autoradiography, bands were either quantitated by scanning densitometry or cut from the filter for scintillation counting.</p>
    <p>To prepare nuclear and cytoplasmic RNA fractions, transfected cells were electroporated with 10 μg of the chimeric gene and after 48 hours, cells were washed twice in phosphate buffered saline (PBS) without calcium or magnesium, and the nuclei extracted by gentle hypotonic lysis (Maniatis, T. (1989) Molecular Cloning: a laboratory manual. T. Maniatis, E. Fritsch, J. Sambrook, eds. Cold Spring Harbor Press (New York: Cold Spring Harbor)). After 15 seconds of vortexing and 5 min at 4° C., nuclei were pelleted and rewashed in PBS. RNA from the nuclear pellets and the aqueous cytoplasmic fraction was separately extracted in 4 M guanidinium isothiocyanate/cesium chloride.</p>
    <heading>Example 6</heading> <p>Cellular Transcription Arrest</p>
    <p>Intracellular stabilities of U6ON and normal U6 were assessed by halting cellular transcription with 10 μg/ml of Actinomycin D (Sigma, St. Louis, Mo.) administered to cell cultures 48 hours after transfection. Cells were harvested and total cellular RNA was isolated at 0, 0.5, 1, 2, and 4 hour time points after Actinomycin D treatment. Northern blotting was performed to quantitate U6 and U6ON transcript levels.</p>
    <heading>Example 7</heading> <p>RNA immunoprecipitation</p>
    <p>293 cells were transfected with 20 μg of the chimeric gene or promoterless plasmid DNA, and after 48 hours, total cellular RNA was isolated. 20 μg of this RNA was used for immunoprecipitation with 0.5 mg of a 5' γ-monomethyl phosphate cap-specific antibody, generously provided by R. Reddy. Incubation and precipitation conditions were followed as previously described for this antibody (Gupta, S. et al. (1990) J. Biol. Chem. 265: 19137-19142).</p>
    <heading>Example 8</heading> <p>RNA Secondary Structure Prediction</p>
    <p>Proposed secondary structures of the RNA oligonucleotides were obtained using the Martinez algorithm RNAFOLD (Martinez, H. (1990) Meth. in Enzymol. 183: 306-317). In all models, loop destabilization was allowed and a maximum "bulge" size of 30 nucleotides was permitted.</p>
    <heading>Example 9</heading> <p>Intracellular Generation of RNA Oligonucleotides</p>
    <p>Generation in Two Cell Types</p>
    <p>MDA453 and 293 cells were transfected with either 10 μg of the chimeric U6ON gene or 10 μg of promoterless plasmid DNA as described in Example 5. Total cellular RNA was isolated 48 hours later followed by Northern blotting with both U6 and U6ON radiolabeled probes as described in Example 6. In FIG. 5A, the intracellular generation of this sequence-specific RNA oligonucleotide, U6ON, is shown in two different human cell lines, MDA453 and 293, following transfection with 10 μg of the chimeric gene. In this experiment and in other replications of this experiment with other human cell types, no cell-type specificity in production has been observed in any of 5 different human cell lines tested. However, transcript levels varied in accordance with the amount of chimeric gene transfected within the range of 5 to 40 μg plasmid DNA per 10<sup>7</sup> cells.</p>
    <p>Linear Correlation Between Transcription Rate and Transfection Dose</p>
    <p>MDA453 cells were transfected with increasing quantities of the chimeric gene followed by RNA isolation at 48 hours and Northern blotting as described above in Examples 5 and 6. All transfections contained 40 μg total DNA, with promoterless plasmid DNA supplementing the chimeric gene as necessary. FIG. 5B illustrates that within this 8-fold range of transfected gene dosage, a near 100-fold linear variation in U6ON transcript levels is observed. Using native U6 RNA levels (known to be present at roughly 0.5×10<sup>6</sup> copies per cell (Sauterer, R. et al. (1988) Exptl. Cell Research 176: 344-359)) as a marker, densitometry was performed to compare the U6ON bands at each transfection level with the U6 band at the 5 μg gene transfection level. Previous results have shown that native U6 transcript levels do not vary upon transfection with 0, 5, or 10 μg U6ON gene transfection levels, but do show transient decreases in transcript levels after transfection with 20 to 40 μg of the U6ON gene). This analysis yields estimates of steady-state intracellular U6ON transcript levels to range from 5×10<sup>4</sup> to 5×10<sup>6</sup> copies/cell at 48 hours post-transfection, depending upon quantity of gene transfected. If nuclei are assumed to be spherical and to have an average diameter of 10 μm, these values correspond to an intranuclear (see FIG. 5C) concentration ranging from 160 μM to 16 mM. These calculations assume an even distribution of the U6ON gene throughout the electroporated cell population, as is found for the U6 gene. Thus, errors resulting from this assumption may lead to higher actual intracellular transcript concentrations.</p>
    <p>Localization to the Nuclear Fraction</p>
    <p>MDA453 cells were transfected with 10 μg of the chimeric gene and after 48 hours, RNA was separated into nuclear and cytoplasmic fractions as described in Example 6. The nuclear fraction shown above contained the U6ON transcript along with the native U6 snRNA. All Northern blots were generated as described in Example 6 from 10 μg of RNA loaded/well. As shown in FIG. 5C, when RNA from gene transfected or mock transfected MDA453 cells is separated into nuclear and cytoplasmic fractions, U6ON is found predominantly in the nuclear fraction, along with native U6. U6ON could not be detected to any significant extent in the cytoplasmic fraction. Moreover, the relative ratio of U6 to U6ON found in the nuclear fraction mirrors the ratio found in total cellular RNA samples.</p>
    <heading>Example 10</heading> <p>Kinetic Analysis of U60N Expression</p>
    <p>FIG. 6 illustrates the rapid production, steady-state levels, and decaying expression of U6ON in 293 cells and MDA453 cells.</p>
    <p>Rapid Production and Steady State Levels</p>
    <p>293 cells were transfected with 20 μg of the chimeric gene and total cellular RNA was isolated at 0, 3, 6, 9, and 12 hours time points. After Northern blotting (10 μg RNA added/well) and autoradiography with a U6ON radiolabeled probe, the U6ON bands were cut from the filter and scintillation counted. As shown in FIG. 6A, analysis over the first 48 hours post-transfection shows that U6ON expression from the chimeric gene begins within 3 hours post-transfection, and reaches steady-state levels in less than 10 hours. Between 12 hours and 48 hours post-transfection, steady-state U6ON levels are constant.</p>
    <p>Decaying Expression</p>
    <p>MDA453 cells were transfected with 20 μg of the chimeric gene and total cellular RNA was isolated at 48, 72, 96, 120, 144, and 168 hour time points. Northern blotting followed with 20 μg of RNA added/well. The Northern blot shown in FIG. 6B demonstrates the decline in U6ON transcript levels out to 168 hours post-transfection where production is diminished but still readily detectable.</p>
    <heading>Example 11</heading> <p>Intracellular Stabilities of the Chimeric Gene and the U6ON Transcript</p>
    <p>Estimation of U6ON Degradation Rate</p>
    <p>Cell counting in parallel with the RNA isolations of FIG. 6B allowed the U6ON band densities to be normalized to account for the dilutional effects of cell division. Normalized band densities were plotted as a function of time to determine the rate of chimeric gene degradation (or inactivation).</p>
    <p>Specifically, intracellular stability of the transfected gene was estimated under the assumption that the observed decline in U6ON transcript levels between 48 hours and 168 hours (FIG. 6B) arises predominantly from two major causes: plasmid degradation (or functional inactivation) and the dilutional effect of cell division (given the equal amount of RNA loaded per lane). The dilutional effect of cell division was accounted for by cell counting in parallel with RNA isolation from 48 hours to 168 hours, and normalizing the Northern blot density values by these cell counts. (Normalized band density=[absolute cell count/cell count at 48 hours]* raw band density) The rate of plasmid degradation (or inactivation) was then estimated as the amount of time required for transcript levels to diminish by 50% from steady-state (48 hours) levels. Cell counting revealed a 38 hour average doubling time from 48 hours to 120 hours, after which time, cell confluence was reached. From 120 hours to 168 hours, absolute cell counts declined slightly. As shown in FIG. 7A, there was a relatively constant plasmid degradation (or inactivation) rate after this normalization procedure, suggesting a zero-order decay process with plasmid half-life (τ.sub. 1/2) determinations dependent on the initial (48 hours) plasmid levels. Thus, given the above assumptions and constraints, after a 20 μg transfection of the chimeric gene into 10<sup>7</sup> cells, approximately 50% of the chimeric gene remains functional after 96 hours (4 days). Variations of this estimate in different cell types would be expected.</p>
    <p>Determination of U6ON half-life</p>
    <p>293 cells were transfected with 5 μg of the chimeric gene and after 48 hours, cellular transcription was halted by a 10 μg/ml treatment of Actinomycin D, as described in Examples 5 and 7. At 0, 0.5, 1, 2, and 4 hours time points, RNA was isolated as described in Example 6. Northern blotting (20 μg RNA/well) with U6 and U6ON radiolabeled probes, followed by densitometry of the U6ON bands, allowed for the determination of U6ON half-life, as described in Example 6. Thus, the intracellular τ.sub. 1/2  of the U6ON transcript was directly measured by halting cellular transcription with Actinomycin D treatment 48 hours after transfection with 5 μg of the chimeric gene, and monitoring the decay of intensity from the U6ON band in Northern blots. The native U6 band was also monitored as a control since its intracellular τ.sub. 1/2  is known to be 16-24 hours (Sauterer, R. et al. (1988) Exptl. Cell Research 176: 344-359; and Terns, M. P. et al. (1993) Genes and Development 7: 1898-1908). FIG. 7B demonstrates the decline in U6ON band intensity between 0 hours and 4 hours following transcription arrest, indicating an intracellular τ.sub. 1/2  of approximately 1 hour after quantitation by densitometry. This analysis also confirms the prolonged stability of native U6. Similar U6ON τ.sub. 1/2  values were obtained when the experiment was repeated with 10 μg and 20 μg gene transfections. Such results indicate that increases in absolute U6ON steady-state transcript levels do not affect U6ON τ.sub. 1/2  determinations, consistent with a first-order process of transcript degradation.</p>
    <heading>Example 12</heading> <p>Measurement of 5' γ-Monomethyl Phosphate Capping for U6ON</p>
    <p>To determine whether the retention of the capping signal of native U6 in the U6ON gene allowed for the production of capped U6ON transcripts, RNA immunoprecipitations with a 5' γ-monomethyl phosphate cap-specific antibody were performed. This antibody has previously been shown to be specific for U6, 7SK, and several other unidentified transcripts which contain this unique 5' cap.</p>
    <p>Total cellular RNA samples isolated after a transfection with 20 μg of U6ON in 293 cells were immunoprecipitated with 0.5 mg of a U6 cap-specific antibody as previously described (Gupta, S. et al. (1990) J. Biol. Chem. 265: 19137-19142) and as reviewed in brief in Example 8. Each immunoprecipitation required 20 μg of initial total cellular RNA. Immunoprecipitated RNA was used for Northern blotting with U6 and U6ON radiolabeled probes as described above in Example 6. As shown in FIG. 8, the U6ON transcript is specifically recognized and immunoprecipitated by this antibody after a 20 μg gene transfection in 293 cells, despite an A/C substitution at base 24. The relative decline in native U6 transcripts immunoprecipitated in the presence of U6ON may be attributed to limiting levels of the antibody, a transient decrease in U6 transcript levels at this higher transfection dose, or competition with U6 RNA for capping enzyme(s) and/or substrates.</p>
    <heading>Example 13</heading> <p>Insert Sequence-Specific Effects on Transcript Secondary Structure and Intracellular Transcript Levels</p>
    <p>As seen in FIG. 7, immunoprecipitating total cellular RNA with a U6 cap-specific antibody confirmed that the U6ON obtains the 5' γ-monomethyl phosphate cap structure found on native U6 RNA. However, insert sequences which favor disruption of the initial 5' hairpin for a longer and more stable stem-loop secondary structure may reduce overall transcript stability, and thus steady-state transcript levels.</p>
    <p>Transcript Secondary Structure</p>
    <p>As shown in FIG. 9A, the conformational output and associated energies of the RNA secondary structure prediction algorithm RNAFOLD (Martinez, H. (1990) Meth. in Enzymol. 183: 306-317), given two different oligonucleotide insert sequences, U6CTcon and U6AS. Despite the same initial nucleotide sequence derived from native U6 in both transcripts, the expected ability to retain the 5' initial hairpin within this sequence differs as a result of the downstream insert sequence. The overall structure and energy values obtained for U6ON and mU6, mirror U6CTcon and U6AS, respectively. Using this structure prediction program, a variety of chimeric genes which generate transcripts that are predicted to prefer one conformation over the other were then designed and constructed.</p>
    <p>Predicted Secondary Structure Affects Intracellular Transcript Levels</p>
    <p>RNA secondary structure was predicted for 4 different oligonucleotide transcripts and the corresponding chimeric genes were constructed. 20 μg of the chimeric genes were transfected into MDA453 cells, followed by Northern blotting (20 μg RNA added/well) 48 hours later with a U6 probe and a probe for each of the possible RNA transcripts.</p>
    <p>As shown in FIG. 9B, when the algorithm predicts that the 5' hairpin is disrupted by downstream secondary structure (as in U6AS and mU6), steady-state transcript levels are drastically reduced. Only at very long film exposures (6 days) can the bands corresponding to mU6 and U6AS be observed.</p>
    <p>Ten chimeric gene constructs have been created to test the hypothesis that the insert sequence can affect intracellular transcript stability and thus steady-state transcript levels by interfering with the formation of the initial 5' hairpin (6 of which the algorithm predicts to retain the initial 5' hairpin, and 4 of which the algorithm predicts to disrupt the initial 5' hairpin). Of these 10 constructs, 8 conform to the pattern of expression and stability shown in FIG. 9B in both MDA453 cells and 293 cells. The two constructs which did not conform were designed to generate stable RNA transcripts, but upon transfection and Northern blotting were found to generate unstable transcripts. All constructs designed to generate unstable transcripts gave rise to unstable transcripts. These disparities may arise from limitations in predicting a preferred RNA state from two competing states in vivo.</p>
    <heading>Example 14</heading> <p>Targeting the HER2 proto-Oncogene Promoter With Triplex RNA Oligonucleotides</p>
    <p>Over the past two decades, multidisciplinary contributions from genetics, biology, and clinical medicine have led to a greatly improved understanding of the molecular mechanisms which underlie cancer induction and progression. Central to this improved understanding is the recognition of the role of proto-oncogenes in the control of cell growth and differentiation (Varmus, H. E. (1984) Ann. Rev. Gen. 18:553-612; Bishop, J. M. (1987) Science 235:305-308; Park, M. and Vande Woude, G. (1989) Cancer: Principles and Practice of Oncology, 3rd ed., In DeVita, V. T., Hellman, S, and Rosenberg, S. A. (eds), pp. 45-66; and Greenberg, M. E. and Ziff, E. B. (1984) Nature 311:433-436). There is considerable data linking the amplification and overexpression of a proto-oncogene with altered cell growth phenotypes (Erisman, M. D. et al. (1985) Mol. Cell. Biol. 5:1969-1976; and Brodeur, G. M. et al. (1984) Science 224:1121-1124).</p>
    <p>In support of this link is evidence that several known proto-oncogenes encode growth factor receptors which couple extracellular protein stimuli (through receptor ligands) with intracellular signal cascades that lead to the synthesis of RNA and DNA necessary for increased cell growth and proliferation. One such proto-oncogene is known as HER2/c-erb B2/neu (hereafter referred to as HER2) which encodes a 185 kD, 1255 amino acid transmembrane receptor (Akiyama, T. et al. (19??) Science 232:1644-1646). The amplification and/or overexpression of HER2 is detected in 11% to 41% of human breast carcinomas (Gullick, W. A. et al. (1991) Brit. J. Cancer 63:434-438), and correlates strongly with increased breast tumor aggressiveness and reduced patient survival (Slamon, D.J. et al. (1987) Science 235:177-182).</p>
    <p>The detection of HER2 amplification and/or overexpression in a significant percentage of breast carcinomas as well as the ability of HER2 to act as a potent prognostic factor of both overall and disease-free patient survival, are two compelling arguments in support of continued research on this proto-oncogene. An additional argument stems from recent pre-clinical and clinical evidence that downregulation of HER2 receptor levels by anti-HER2 monoclonal antibodies may have therapeutic value in managing HER2-positive breast cancer patients (Park, J. W. et al. (1991) Genes, Oncogenes, and Hormones: Advances in Cellular and Molecular Biology of Breast Cancer, Dickson, R. B. and Lippman (eds), pp. 193-211). Apart from the limited efficacy that may be anticipated from an antibody-based therapy, recent identification of HER2-positive tumor cells resistant to downregulation by anti-HER2 antibodies (Scott, G. K. et al. (1993) Mol. Cell. Biol. 13:2247-2257) demonstrates the need for a strategy which targets HER2 at the level of the gene rather than at the level of the mature protein.</p>
    <p>The HER2 Promoter and Triplex RNA Oligonucleotides</p>
    <p>A triplex approach to HER2 downregulation was chosen because of the unique HER2 promoter structure and because of a lack of information on suitable mRNA target sequences. Previous attempts at HER2 downregulation by antisense oligonucleotides were only partially, if at all, successful (Christopher C. Benz &amp; Debu Tripathy, personal communication), and the secondary structure of HER2 has not been sufficiently well characterized to predict accessible mRNA sequences. Therefore, production of an anti-HER2 antisense RNA oligonucleotide was ruled out. Production of an anti-HER2 ribozyme was similarly ruled out both because of a lack of a priori knowledge of susceptible target sequences, as well as because of a necessity of producing a much longer RNA oligonucleotide containing the entire ribozyme.</p>
    <p>However, both the sequence and structural/functional elements of the HER2 promoter have been well characterized (Scott, G. K. et al. (1994) J. Biol. Chem. 269: 19848-19858; Ishii, S. et al. (1987) Proc. Natl. Acad. Sci. USA 84: 4374-4378; and Hollywood, D. P. and Hurst, H. C. (1993) EMBO J. 12: 2369-2376) and were uniquely suited for a triplex approach to gene regulation. The critical region of the HER2 promoter has been localized to a 125 base-pair sequence which is capable of conferring a &gt;30-fold variation in HER2 transcriptional activity (Scott, G. K. et al. (1994) J. Biol. Chem. 269: 19848-19858). Within this proximal promoter illustrated in FIG. 10 were several putative protein binding sites (including CAAT, TATA, and a GAGGAA ets-related response element) which surround a 28 base-pair homopurine/homopyrimidine sequence (with 1 central A/T inversion).</p>
    <p>Given the structure of the HER2 promoter, should triplex formation occur in vivo, the net result of transcriptional repression could occur through one of several mechanisms. The RNA oligonucleotide could compete directly for binding on the promoter with one or more essential DNA binding transcription factors; the non-triplex-forming 5' or 3' tails of the RNA oligonucleotide could sterically interfere with protein/DNA binding or protein/protein associations; the RNA oligonucleotide could alter duplex rigidity, prevent DNA looping and inhibit association of CAAT and TATA binding proteins; and finally, triplex formation could alter the dimensions of the major and minor grooves and prevent or decrease protein binding on neighboring sites.</p>
    <p>While the consensus binding sites illustrated in FIG. 10 did not directly overlap the triplex binding sequence, DNA methylation interference assays on the GAGGAA ets-related response element have demonstrated partial DMS methylation protection upstream to the last few guanines of the triplex target (Scott, G. K. et al. (1994) J. Biol. Chem. 269: 19848-19858). In addition, DNase I protection assays have shown clear footprinting within the triplex binding site (Hollywood, D. P. and Hurst, H. C. (1993) EMBO J. 12: 2369-2376), and a hypersensitivity site within the GAGGAA response element. Moreover, initial experiments have demonstrated that DNA oligonucleotide-directed triplex formation on the HER2 promoter using a GT purine-rich binding motif could inhibit transcription factor binding to the ets-response element, and that this inhibition was enhanced in the presence of both nucleotide and non-nucleotide 5' "tails" attached to the triplex oligonucleotide. These "tails," were not found to confer increased inhibition through an increase in triplex binding affinity, but more likely through an increase in steric interference at the adjacent ets binding site. Such similar effects could also occur with the RNA 5' and 3' "tails" of U6ON. Finally, a recent report has demonstrated that triplex formation with a DNA oligonucleotide could inhibit transcription from the HER2 promoter in vitro (Ebbinghaus, S. W. et al. (19??) J. Clin. Invest. 92:2433-2439). These findings support the hypothesis that triplex formation on this promoter could mediate HER2 transcriptional repression in vivo.</p>
    <p>The U6ON was tested for its ability to downregulate HER2 promoter activity. In the first set of experiments, the activity of the promoter was monitored by expression of a co-transfected reporter chloramphenicol acetyl transferase (CAT) gene to which different HER2 promoters was fused. This bacterial CAT gene is not found within the human genome; therefore, its expression in transfected cells was indicative of a functioning promoter. Two different promoter constructs were tested: the first was the minimal promoter extending 125 base-pairs from the transcriptional start site, and the second was a larger and stronger promoter extending 500 base-pairs from the transcriptional start site. Experiments were performed in both HER2-positive cells (MDA453) and HER2-negative cells (MCF-7). The effect of the anti-HER2 triplex RNA oligonucleotide, U6ON, was then compared with an anti-CAT antisense RNA oligonucleotide, U6AUG. These experiments provided an initial comparison of triplex and antisense data to a common gene.</p>
    <p>In the second set of experiments, the activity of the promoter was monitored by examination of endogenous levels of HER2 RNA from 48 to 96 hours after transfection with the chimeric gene for U6ON or control DNA. As HER2 encodes a transmembrane growth factor receptor, RNA isolations were performed in parallel with cell counting to monitor the effect of U6ON on cell growth, and to correlate HER2 mRNA downregulation with growth rate alterations.</p>
    <p>A variety of breast cancer cell lines exist which overexpress HER2 to varying extents. The HER2 overexpression can arise from either increased HER2 transcriptional activity or HER2 gene amplification. These phenomena may in fact be causally related, although the exact mechanism through which this might occur has not been clearly delineated. MDA453 is a prototypical HER2-positive cell line which demonstrates both HER2 overexpression and HER2 gene amplification. These cells are easily transfected by a variety of methods, are relatively homogenous in size and shape, and have previously been demonstrated to be capable of driving transcription from a reporter gene fused to HER2 (Scott, G. K. et al. (1994) J. Biol. Chem. 269:19848-19858). For these reasons they were chosen as a model HER2-positive cell line for testing the effects of RNA oligonucleotides on HER2 promoter activity. MCF-7 cells are also easily transfected, are widely used in breast cancer research, and have previously been shown to be incapable of driving significant transcription from a reporter gene fused to HER2 (Scott, G. K. et al. (1994) J. Biol. Chem. 269:19848-19858). For these reasons they were chosen as a model HER2-negative cell line for testing the effects of RNA oligonucleotides on HER2 promoter activity.</p>
    <p>Materials and Methods</p>
    <p>Unless otherwise noted the material and methods used in Example 14 are described in the preceding Examples.</p>
    <p>Construction of U6AUG</p>
    <p>The chimeric gene producing an RNA oligonucleotide referred to as U6AUG was designed as an antisense CAT oligonucleotide to a particularly GC-rich sequence just upstream from the CAT gene stop site. The ATG start site could not be used as an antisense RNA oligonucleotide as it is surrounded by runs of T residues. However, that sequence was used to generate U6mini. The Martinez RNA secondary structure prediction algorithm was implemented to ensure that the GC-rich antisense oligonucleotide did not interfere with the formation of the initial 5' hairpin. Northern blotting with 10 to 20 μg gene transfections in MDA453 cells then confirmed that the RNA oligonucleotide, U6AUG, was indeed abundantly produced, stable, and gave rise to an expression profile similar to that described for U6ON. The sequence of the upper strand of the inserted duplex fragment was:</p>
    <p>5' TCGACCGCCCCGCCCTGCCACTCATCGCAGTACATGCA 3'(SEQ ID NO:5)</p>
    <p>The digestion, ligation, and sequencing of this gene was as described in the Examples above.</p>
    <p>MCF-7 cell culture</p>
    <p>MCF-7 cells were maintained before and after electroporation in DME medium with 1 g/L glucose, supplemented with 10% fetal calf serum, 100 U/ml of penicillin/streptomycin, and 10 μg/ml insulin at 37° C., 5% CO<sub>2</sub>.</p>
    <p>β-HCG quatitation</p>
    <p>Determination of cellular β-HCG was performed 48 hours after MDA453 cell electroporation with 2 μg of the β-HCG gene added to 20 μg of the chimeric oligonucleotide producing gene or promoterless control DNA using the Tandem R total β-HCG immunoradiometric assay (Hybritech, San Diego, Calif.). In this kit, anti-β-HCG antibodies are coupled to plastic beads which are then incubated with the supernatant from transfected cells. A secondary I<sup>125</sup> radiolabeled anti-β-HCG antibody is then added, and after a 1 hour incubation at 37° C., beads are washed in 0.01% sodium azide and quantitated by a gamma counting.</p>
    <p>CAT Assays</p>
    <p>Each CAT assay used 10 to 20 μg of the chimeric gene co-transfected by electroporation with 20 μg of the CAT gene and 2 μg of the β-HCG gene per approximately 3×10<sup>6</sup> cells. Total DNA concentrations were constant for all electroporations by supplementing where necessary with promoterless plasmid DNA. Cells were harvested 48 hours after transfection by freeze thaw lysis in 100 mM Tris, pH 7.6, 2% Triton X. Cell lysates were then collected and microcentrifuged (12,000 xg) for 10 min at 4° C., and supernatants were heated to 65° C. for 10 min to inactivate cellular proteases. Each sample was then overlaid with 100 mM Tris, pH 7.6, 1 mM chloramphenicol (Sigma), and 1 μCi of [<sup>3</sup> H]-acetyl coenzyme A (Dupont/NEN Research Products), followed by the addition of 5 ml of Econofluor water immiscible scintillation fluid (Dupont/NEN Research Products). Samples were immediately scintillation counted for 1 min to obtain baseline values, followed by incubation at 37° C. and counting at regular intervals (20-30 min) for 2 to 3 hours. CAT activity values were obtained by normalizing the slope of counts vs. time for each sample with corresponding β-HCG values obtained as described above.</p>
    <p>Effect of RNA oligonucleotides on HER2-driven CAT Expression</p>
    <p>FIG. 11A illustrates the CAT activity arising from the minimal 125 bp HER2 promoter and the stronger 500 bp HER2 promoter in MDA453 and MCF-7 cells 48 hours after co-transfection with 20 μg of either U6ON or U6CTcon. Promoterless plasmid DNA was used to equalize DNA concentrations. Both U60N and U6CTcon RNA oligonucleotides demonstrated virtually 100% downregulation of the minimal promoter and roughly 65 to 70% downregulation of the stronger 500 bp promoter. As expected, no CAT activity was found in MCF-7 cells in either the presence or absence of RNA oligonucleotides. Interestingly, β-HCG levels, which were used to normalize cell populations for transfection efficiency were not similarly downregulated. Thus, the HER2/CAT downregulation cannot be attributed to a general downregulation of protein synthesis. Successive repetitions of this CAT assay using these two chimeric gene constructs with the 500 bp promoter have produced similar patterns of downregulation (ranging from roughly 45% to 75%).</p>
    <p>FIG. 11B compares HER2/CAT downregulation of the stronger 500 bp promoter by triplex and antisense RNA oligonucleotides after co-transfection with 10 μg of either U6ON or U6AUG. The antisense RNA oligonucleotide produced 57% downregulation whereas the U6ON produced 49% downregulation. Successive repetitions of this CAT assay have demonstrated similar patterns of downregulation. While absolute levels vary, in each case the antisense RNA oligonucleotide continually demonstrates 5 to 15% greater downregulation than the triplex RNA oligonucleotide. As proper controls are still lacking, the significance of these observations has yet to be determined.</p>
    <p>FIG. 12 illustrates the effect of U6ON on endogenous levels of HER2 mRNA and cell growth in MDA453 cells. As seen from the Northern blot (FIG. 12A), GAPDH levels are not significantly altered by the presence of the RNA oligonucleotide from 48 to 96 hours after gene transfection. Note also that no indication of unspliced GAPDH or HER2 could be detected despite a 20 μg gene transfection, suggesting that U6 was present and functioning within the spliceosome. However, after normalization with GAPDH levels, a relative decrease in HER2 levels was seen in the U6ON transfected cells when compared to cells transfected with equal amounts of promoterless plasmid DNA (FIG. 12B). This decrease in HER2 RNA was most prominent at 48 hours (47.5% downregulation) and 72 hours (50.8% downregulation), but declines in significance at 96 hours (24.1% downregulation). Given the long t.sub. 1/2  of intracellular HER2 mRNA (roughly 24 hours), complete promoter shut down immediately upon transfection could only lead to a maximum of 75% downregulation. Thus, the nearly 50% downregulation at 48 hours indicated that initially, the promoter was operating at only one-third of its normal rate.</p>
    <p>This pattern of high HER2 downregulation at 48 hours and 72 hours and low HER2 downregulation at 96 hours HER2 was reflective of the corresponding decline in intracellular U6ON levels at 48 h, 72 h, and 96 hours shown in the Northern blot of FIG. 12C. As demonstrated in FIG. 6B, a 20 μg U6ON transfection in MDA453 cells lead to nearly equal levels of U6 and U6ON at 48 hours post-transfection, with U6 slightly reduced from normal levels. Earlier rough calculations estimated an intracellular U6ON concentration of 5×10<sup>5</sup> copies of U6ON/cell leading to an intranuclear U6ON concentration of 1.6 mM. That such mass accumulation of an RNA oligonucleotide within the nucleus appears to affect the production of HER2 without affecting the production of GAPDH demonstrates a degree of specificity of U6ON. Cell counts obtained in parallel with RNA isolations (FIG. 12D) showed a corresponding decrease in cell growth in U6ON transfected cells versus control cells between 48 hours and 72 hours (11.7% versus 28.8%), but a rebound in growth rate between 72 hours and 96 hours (85% versus 65%).</p>
    <heading>Example 15</heading> <p>Effect of Intracellular RNA Oligonucleotides on Native U6 Levels: Evidence that U6 Transcription and Capping are Uncoupled In Vivo</p>
    <p>The effect of producing high concentrations (160 μM to 16 mM) of intranuclear RNA oligonucleotides generated from a plasmid bearing the regulatory promoter, capping, and termination sequences of the human U6 gene on native U6 RNA levels was studied. Native U6 RNA levels were unaffected by transfections of up to 20 μg of the chimeric oligonucleotide-producing gene, but U6 RNA levels decreased substantially after transfections of 20 μg to 40 μg of the chimeric gene. These effects were seen within 24 hours after gene transfection and persisted unchanged up to 48 hours after gene transfection. U6 RNA levels returned to native levels after 72 to 96 hours. These decreases in U6 RNA were determined to result primarily from a decrease in U6 RNA stability rather than a decrease in U6 RNA production. U6 RNA stability was found to be titratable in the presence of increasing levels of stable RNA oligonucleotides bearing a 5' γ-monomethyl phosphate cap, but were normal and unchanged for unstable RNA oligonucleotides whose secondary structure may inhibit capping. These results on U6 RNA differential stability in the presence of capped RNA oligonucleotides strongly suggest that in vivo transcription and capping are uncoupled for U6 RNA.</p>
    <p>Short, sequence-specific RNA oligonucleotides can be generated in high yield within the cell nucleus using an expression system derived from the U6 gene. As the U6 gene requires only upstream elements for initiation and elongation by RNA Polymerase III, and stops cleanly upon reaching a string of 4 or more thymine residues (Kunkel G, et al. (1986) Proc. Natl. Acad. Sci. 83:8575-8579; Reddy R, et al. (1987) J. Biol Chem. 262:75-81; Kunkel G and Pederson T. (1989) Nuc. Acids Res. 17:7371-7379), its internal sequence can be removed and be replaced by an oligonucleotide sequence designed to target cellular DNA or RNA. In addition, the retention of an initial native U6 hairpin and the majority of an adjacent hexameric sequence will allow for efficient 5' γ-monomethyl phosphate capping (Singh, R. and Reddy, R. (1989) Proc. Natl. Acad. Sci. USA 86:8280'8283; Singh, R. et al. (1990) Mol. Cell. Biol. 10:939-946). As all of the regulatory elements of U6 are retained in the chimeric oligonucleotide-producing gene, one could expect that as the quantity of gene transfected is increased, one or more of the regulatory elements of U6 would come under limiting supply. The experiments of this section seek to determine which of these elements is most limiting, and at what transfected gene dose this element becomes limiting.</p>
    <p>U6 is a unique mammalian gene as it appears to represent an intermediate between standard class II genes and class III genes (Dahlberg, J. E. and Lund, E. (1992) Science 254:1462-1463). U6 is transcribed by RNA polymerase III and stops at runs of thymines; however, unlike almost all other class III genes, it contains no functional internal control region. While there is a region within the gene with homology to the "box A" sequence of other class III genes, it is either vestigial or coincidental since it can be deleted without loss of wild type expression (Kunkel, G. and Pederson, T. (1989) Nuc. Acids Res. 17:7371-7379). In addition, like standard class II genes, the U6 promoter contains a TATA box (located at position -30) which is essential for promoter activity. Interestingly, the exact location of this TATA box is slightly altered with respect to the position of class II gene TATA boxes, and alteration of this U6 TATA box converts the gene from a class III gene to a class II gene (Lobo, S. M. and Hernandez, N. (1989) Cell 58:55-67). As in class II and class III genes, as well as class I genes, U6 transcription requires a TATA box binding factor (TBP) (Rigby, P. W. (1993) Cell 72:7-10). This factor is a subunit within the transcription factor TFIIIB also essential for initiation.</p>
    <p>In addition to RNA polymerase III and TFIIIB, U6 transcription requires an as yet unpurified factor referred to as the proximal sequence element binding factor (PBP) which binds strongly to a highly conserved element adjacent to the TATA box (located at -60). This element has been shown to confer species specificity to the U6 gene, as well as to ensure initiation at the proper starting nucleotide (Simmen, K. A. et al. (1992) J. Mol. Biol. 223:873-884; and Goomer, R. S. and Kunkel, G. R. (1992) Nuc. Acids Res. 20:4903-4912). The last known promoter element necessary for efficient U6 transcription is referred to as the distal control region (DCR) which resides at roughly -244 to -149 and contains binding sites for the transcription activator Oct1 (Danzeiser, D. A. et al. (1993) Mol. Cell. Biol. 13:4670-4678). Two other transcription factors, TFIID and TFIIA have been shown to stimulate U6 expression in vitro, but their roles in in vivo U6 expression have not been elucidated (Simmen, K. A. et al. (1991) EMBO J. 10:1853-1862). All of these U6 promoter elements are also found in the promoter of the chimetic oligonucleotide-producing gene, and all of these factors are expected to play identical roles in the initiation and elongation of the RNA oligonucleotide.</p>
    <p>The chimeric gene also shares the initial 25 nucleotides and adjacent hexameric sequence (with 1 C to A mismatch at base position 24) and potential for 5' capping with the U6 gene. Thus, both the recently purified capping enzyme (Shimba, S. and Reddy, R. (1994) J. Biol. Chem. 269:12419-12423) and the capping substrate(s) are expected to function identically for both genes. A final shared element is the Lupus-associated (La) antigen (Rinke, J. and Steitz, J. (1982) Cell 49:149-159). This 55 kD protein has been shown to bind transiently to the uridine-rich tail of U6. It may function to protect the 3' end of nascent transcripts from nuclease degradation, or to participate in 3' post-transcriptional modifications such as the addition of uridine residues and a terminal 2'-3' cyclic phosphate (Lund, E. and Dahlberg, J. E. Science 255:327-330). Regardless of activity, it may be expected to function in both nascent U6 as well as nascent RNA oligonucleotides. A summary of the factors shared by both genes and/or transcripts is provided in FIG. 13.</p>
    <p>As the entire U6 sequence from +26 to +87 was deleted in creating the chimeric oligonucleotide-producing gene, putative catalytic elements and U4 and U2 hybridization regions within this deleted sequence are not found within the RNA oligonucleotide (Datta, B. and Weiner, A. M. Nature 352:821-824; and Sontheimer, E. J. and Steitz, J. A. (1993) Science 262:1989-1996). As these elements are essential for U6 splicing activity within the splicing apparatus, neither splicing nor association with U2 or U4 are expected with the RNA oligonucleotide.</p>
    <p>By monitoring U6 RNA levels and U6 RNA stability in the presence and absence of both the capped (U6ON) and the presumably non-capped (U6AS) RNA oligonucleotides, information was obtained on the dose-dependent effects of RNA oligonucleotide production. At ≧20 μg of transfected chimeric U6ON gene, a dose-dependent decrease in total U6 RNA levels was observed. This decrease was attributed primarily to a decrease in U6 RNA stability rather than a decrease in U6 production as no decrease in U6 RNA levels was found upon transfection with ≧20 μg of the chimeric U6AS gene. U6 stability has previously been described as resulting from the nuclease-resistant 5' cap and the extensive 3' hybridizations (Terns, M. P. et al. (1993) Genes and Devt. 7:1898-1908). Since U6ON can be 5' capped but cannot hybridize with itself or with, competition for capping enzymes and/or cofactors is assumed, and the 5' capping apparatus is deemed the factor in most limiting supply.</p>
    <p>The most likely explanation--that the capping apparatus was saturated and thus RNA stability is reduced at high levels of U6ON gene transfection, but not at high levels of U6AS gene transfection--was tested by examining the half-life, t.sub. 1/2, of U6 RNA in the presence of U6ON. FIG. 16 illustrates the dramatic reduction in t.sub. 1/2  seen 48 hours after a 20 μg U6ON gene transfection followed by transcription arrest by Actinomycin D in 293 cells. The normal t.sub. 1/2  value for U6 has previously been determined to range from 16 to 24 hours. The t.sub. 1/2 value estimated after scanning densitometry of this Northern blot was roughly 3 hours. A similar dramatic decrease in U6 stability (9-fold) has previously been reported in characterizing capped vs. non-capped in vitro generated U6 RNA after microinjection into Xenopus oocytes (Shumyatsky, G. et al. (1993) Nucleic Acids Res 21: 4756-4761).</p>
    <p>Materials and Methods</p>
    <p>All relevant details on cells and cell culture, electroporation, transcription arrest, RNA isolation, Northern blotting, and secondary structure prediction are described in the Examples above. The capped intracellular RNA oligonucleotide consists of the same CU-rich sequence referred to as U6ON. The non-capped intracellular RNA oligonucleotide consists of the same mixed nucleotide sequence as that used in FIG. 9 and is similarly referred to as U6AS. The following Method is pertinent to Example 15 in particular.</p>
    <p>Dynamical Modeling of U6 and U60N</p>
    <p>Simple linear first and second order differential equations were derived to describe the expression of both U6 and U6ON, and dynamical simulations were performed using the modelling program STELLA (High Performance Systems, Hanover, N.H.). In these models, steady-state values of RNA levels were assumed to be a function of only two variables, production (i.e., from transcription) and decay (i.e., from nuclease degradation). Production, P, was assumed to be a function of the initiation rate constant, K<sub>i</sub>, and gene copy number, gcn, and the decay rate constant K<sub>d</sub>, was determined from the experimentally derived values of t.sub. 1/2, assuming a first order model of degradation.</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="equation">d[U6]/dt=K<sub>i</sub> *gcn-K<sub>d</sub> *[U6]=P-K<sub>d</sub> *[U6]</pre>
    
    <p>
      </p> <pre xml:space="preserve" listing-type="equation">d[U6ON]/dt=K<sub>i</sub> *gcn-K<sub>d</sub> *[U6ON]</pre>
    
    <p>For both U6 and U6ON, the initiation rate constant, K<sub>i</sub> was assumed to be constant and to represent constitutive transcription. The rate of elongation was eliminated from these models as the process of initiation was assumed to be the rate limiting step in transcription. Thus, the rate of initiation was assumed to represent the rate of production of a single complete U6 or U6ON RNA molecule from its genomic template. For U6, the gene copy number, and thus P was held constant, whereas for U6ON, the gene copy number was assumed to be directly coupled and vary linearly with gene transfection dose. Initial values of U6ON were set to 0 and initial values of U6 were set to the normal value of 0.5×10<sup>6</sup> copies/cell. U6 gene expression was assumed to be equally present in 100% of the cell population and to be unaffected by the physical process of electroporation. Electroporation was assumed to result in U6ON gene expression with a transfection efficiency of 70% of the cell population. Differential equations were solved by Euler's method.</p>
    <p>Results</p>
    <p>FIG. 14A and 14B demonstrate the effect of increasing quantities of the transfected gene for U6ON in human 293 cells. Results demonstrate that the levels of U6 RNA remain relatively constant up to 10 to 20 μg transfection doses, but cause a decrease in U6 RNA levels at transfection doses of 20 to 40 μg. The RNA in each case was isolated 48 hours after electroporation; however, isolation at 24 hours also produced identical trends of U6ON upregulation with concurrent U6 downregulation.</p>
    <p>Example 13 presented evidence that the sequence of the inserted oligonucleotide can effect the stability of the resulting intracellular RNA transcript by interfering with formation of an initial hairpin and disrupting the necessary signal for capping (FIGS. 9A and 9B). Several genes were designed to contain this interfering oligonucleotide sequence, the prototype of which was termed U6AS. The generation of this unstable U6AS transcript was used to determine whether this dose-dependent decrease in total U6 RNA levels was a result of a decrease in production (i.e., saturation of a requisite transcription factor or polymerase) or an increase in degradation (i.e., greater accessibility to exonucleases and/or endonucleases).</p>
    <p>FIG. 15 illustrates the differences in U6 RNA profiles 48 hours to 120 hours after transfection with 20 μg of either the U6AS gene or the U6ON gene in 293 cells. (The pattern was also verified with another stable RNA oligonucleotide, U6CTcon.) Whereas U6 RNA levels remained relatively unaffected by the presence of the unstable and presumed non-capped U6AS, U6 RNA levels were drastically reduced and then rebound in the presence of the stable capped U6ON. Given that U6AS and U6ON differ only in the 28 nucleotide internal oligonucleotide sequence and their ability to retain the initial 5' hairpin needed for capping, not in the upstream promoter or termination signals, saturation of transcription factors, RNA polymerase III, the La antigen, or the poorly characterized post-transcriptional modification apparatus were unlikely as an explanations for the disparity in U6 levels.</p>
    <p>FIG. 17 demonstrates graphically the profiles of U6 degradation as U6ON gene transfection doses were reduced from 20 μg down to 5 μg. As the dose was lowered, the U6 RNA stability increased to its native level, consistent with the trends presented in FIG. 14. The stability of U6ON after a 20 μg gene transfection dose was shown in the graph, but similar profiles were seen after 10 μg and 5 μg gene transfections. No effect on normal U6 RNA stability was seen 48 hours after a 20 μg U6AS gene transfection. Transcription arrests were not carried out for time points longer than 8 hours to avoid significant artifacts due to cellular dysfunction (such as reduced levels of cellular nucleases) as a result of the Actinomycin D treatment.</p>
    <p>FIG. 18 demonstrates the simulated output of U6 and U6ON given different parameters of decay rate constant (derived from different t.sub. 1/2 values) and gene copy number (derived from transfection dose). While these models were highly simplified, they corroborate previously determined experimental parameters of U6ON expression; they provided broad estimates as to the gene copy number of both the human U6 gene and the electroporated U6ON gene, and their rates of transcription initiation; and they gave schematic representation to the effect of large quantities of U6ON upon U6.</p>
    <p>In FIG. 18A, the steady-state production of U6 is displayed. This RNA has previously been demonstrated to be present at roughly 0.5×10<sup>6</sup> copies/cell with a t.sub. 1/2  value of roughly 24 hours. Using these values, a production flow rate, P, of 4 transcripts/second was derived. The exact copy number of the human U6 gene was not known explicitly due to the presence of many pseudogenes, but was estimated to range between 2 and 10 based on comparisons with other species (Saluz, H. et al. (1988) Nuc. Acids Res. 16: 3582; and Das, G. et al. (1987) J. Biol. Chem. 262: 1187-1193). This lead to an estimate of transcription initiation rate, K<sub>i</sub>, of 0.4 to 2 initiations/second/gene copy. As the contribution to transcript production by elongation was assumed negligible, K<sub>i</sub> estimations referred to the rate of production of a completed full length transcript from a single gene copy. These K<sub>i</sub> estimations were reasonably consistent with those previously made for the U1 and U2 genes, present in similar intranuclear concentrations with similar t.sub. 1/2 values (Dahlberg, J. E. and Lund, E. (1988) Structure and function of major and minor small nuclear ribonucleoprotein particles, Springer-verlag, Berlin).</p>
    <p>In FIG. 18B, the profile of U6ON expression after a 20 μg gene transfection was depicted using K<sub>d</sub> and K<sub>i</sub> *gcn values experimentally derived in Example 14 from production rates (FIG. 7A) and t.sub. 1/2  values (FIG. 7B), respectively. Implementing these parameters and simulating production for 48 hours produces a steady-state production levels very similar to that shown in Example 10 (FIG. 5B) i.e. slightly above U6 levels. Here, the K<sub>i</sub> *gcn value was roughly 100. If constitutive expression is assumed, K<sub>i</sub> was identical for U6 and U6ON, and therefore 0.4 to 2 initiations/second/gene copy. This lead to an estimation of 50 to 250 functional U6ON-producing genes/cell after transfection in the electropotation conditions described above.</p>
    <p>In FIG. 18C, the profile of U6 expression following transfection with 20 μg of the U6ON-producing gene was plotted. The effect of production of U6ON on U6 RNA was modeled as a first order exponential smoothed step function of degradation from an initial value derived from a 24 hour t.sub. 1/2  to a final value derived from a 3 hour t.sub. 1/2. As expected, upon transfection, U6 RNA levels dropped by a factor of 8, and the subsequent U6 expression profile demonstrated a second order appearance (illustrated by slight overshoot at 12 h). In addition, the experimental observation that this new steady-state value was reached by 24 hours is consistent with this model. The reduction in U6 RNA levels modeled solely by the impact of reduced RNA stability, in comparison with the experimentally-derived observations presented in this and previous Examples, support further the notion that reduced U6 RNA stems primarily from reduced stability rather than reduced production.</p>
    <p>From the results presented above, several important conclusions can be drawn. Firstly, upon transfection with ≧20 μg of a gene for stable (capped) RNA oligonucleotides, total U6 RNA levels were decreased. Secondly, this decrease in U6 RNA could be attributed primarily to a decrease in RNA stability rather than a decrease in RNA production. Thirdly, a saturation of capping enzymes and/or cofactors with subsequent production of uncapped U6 RNA was the most likely explanation for the decrease in U6 RNA stability. Finally, these conclusions taken together support the inference that transcription and capping are uncoupled in the U6 gene.</p>
    <p>The observation of decreased levels of U6 RNA 24 to 48 hours after transfection with ≧20 μg of the chimeric U6ON-producing gene placed a clear upper limit on the gene dosage above which cells were affected by the production of the stable RNA oligonucleotide. This 20 μg value was estimated in two unrelated cell lines, 293 and MDA453, suggesting that the type of cell transfected probably did not play a large role in defining this upper limit. However, the conditions of electroporation (i.e., the voltage and capacitance that sets the dimensions of the electrical pulse) would be expected to play a large role. In these experiments, a setting of 250 V, 960 μF led to a cell survival rate post-electroporation of roughly 40 to 60%. At higher voltage settings, the survival percentage would be reduced, but those surviving might contain higher doses of transfected gene, leading to possible reductions of U6 RNA levels at lower than 20 μg doses.</p>
    <p>Several lines of evidence lead to the conclusion that the reduction in U6 RNA was a result of decreased RNA stability rather than decreased RNA production. In strongest support is the finding of dramatically reduced U6 RNA t.sub. 1/2  in the presence of a high concentrations of U6ON. Thus, a decrease in U6 RNA must play a role in overall decreased levels, but does it play the primary role? The observation that U6 RNA levels were not reduced in the presence of similar transfection doses with a gene for an unstable RNA oligonucleotide lends additional support to the above hypothesis. Given all previous evidence of needing only upstream U6 promoter sequences and downstream termination sequences for proper initiation and termination of transcription, one can assume that the unstable RNA oligonucleotide (U6AS) was generated equally as often and used the same transcription factors as the stable RNA oligonucleotide (U6ON). Given equal gene copy numbers of U6AS and U6ON, each should have bound to the same amount of transcription factors if there existed no system of negative feedback control for U6. To date, no such regulatory system has been described.</p>
    <p>Saturation of the capping apparatus appears the most likely cause of decreased total U6 RNA in the presence of abundant levels of capped U6ON RNA. U6 RNA stability has previously been attributed primarily to its 5' cap and its substantial 3' hybridizations. Since U60N contains no regions amenable to self or U4 hybridizations, but does obtain the 5' cap, the dramatic decrease in U6 stability should therefore resulted from the loss of the 5' cap. In further support, evidence of an 8- to 9-fold loss of U6 RNA stability has previously been reported for intracellular non-capped U6 by two independent laboratories (Terns, M. P. et al. (1993) Genes and Devt. 7: 1898-1908; and Shumyatsky, G. et al. (1993) Nuc. Acids Res. 21:4756-4761). This reduction corresponded well to the 8-fold decrease in U6 RNA t.sub. 1/2  seen upon production of high concentrations of U6ON. Therefore, if the above two properties are indeed primarily responsible for the 24 hour t.sub. 1/2  of U6 RNA, the contribution of extensive hybridization with self and U4 RNA to the stability of native U6 RNA can be estimated as roughly 3-fold.</p>
    <p>To date, all other known capped RNA (including mRNA and other small nuclear U-rich RNA such as U1, U2, etc.) are thought to have transcription coupled to capping (Reddy R. et al. (1992) Pharm Ther. 54: 249-267). If this U6 RNA with reduced stability does represent non-capped U6, this finding demonstrates for the first time in vivo (and corroborates observations in vitro (Gupta et al. (1990) J. Biol. Chem. 265: 9491-9495) that transcription and capping are uniquely uncoupled for the U6 gene.</p>
    <heading>Example 16</heading> <p>Effect of Intracellular RNA Oligonucleotides On General Properties of Transcription and Translation</p>
    <p>The effect of U6ON expression on 7SK, U1, U3, and GAPDH RNA levels and β-HCG protein levels was studied to make an initial assessment of possible toxic effects of the chimeric oligonucleotide-producing gene on cellular transcription and translation. Whereas levels of U6 RNA are significantly decreased upon transfection with ≧20 μg of a chimeric gene producing a stable, capped RNA oligonucleotide, no such effects were seen in any of the above RNA species. Relative levels were equally unaffected by the inherent stability of the intracellularly-generated oligonucleotide. Transcription arrests with Actinomycin D demonstrated that the presence of an RNA oligonucleotide did not perceptively alter the &gt;24 hour t.sub. 1/2  of the small nuclear RNA. No increases in GAPDH RNA levels were seen at successive time points 48 hours to 96 hours after transfection as the oligonucleotide-producing gene was becoming increasingly diluted within cells. Expression of the oligonucleotide-producing gene also did not demonstrate any effect on cellular translation as measured by immunoradiometric protein quantitation of a co-transfected exogenous gene for β-HCG. Taken together, these results suggest that neither stable nor unstable intracellularly generated RNA oligonucleotides significantly alter the cellular properties of transcription or translation.</p>
    <p>Constitutive production of abundant intracellular RNA oligonucleotides from an exogenous gene source may be expected to utilize a significant quantity of cellular energy and resources. A variety of cellular factors, both general and specific, could, in theory, be overwhelmed by the increased demands brought about by high doses of the oligonucleotide-producing gene. Evidence for saturation by U6ON of factors involved in the 5' capping of U6, with subsequent decrease in RNA stability, is one example of how a cell might be adversely affected by the RNA oligonucleotides. This adverse effect was found to diminish as transfection dose, and thus intracellular U6ON levels, were decreased, in Example 15 above.</p>
    <p>To extend this investigation of possible cellular toxicity to include the more general processes of cellular transcription and translation, the expression of four other genes were monitored. Two of these genes, 7SK and U1, have basal promoter structures similar to U6, and thus the oligonucleotide-producing gene. The other two genes, U3 and GAPDH, share relatively few promoter features with U6 and the oligonucleotide-producing gene. The study of the RNA expression from these four genes provided a means for monitoring not only polymerase and transcription factor availability, but also nuclease and nucleotide saturation.</p>
    <p>The structural features of the 7SK promoter has previously been described as nearly identical to those found in U6 (Wasserman DA and Steitz, JA. (1991) Molecular and Cellular Biology 11: 3432-3445; and Murphy S, et al. (1987) Cell 51: 81-87). Both contain a TATA box, a PSE, as well as the upstream DCR. Both are transcribed by RNA polymerase III and terminated by a series of thymine residues. In addition, both receive the unique 5' non-nucleotide γ-monomethyl phosphate cap. More recent studies, however, suggest that despite the high degree of homology in promoter structure and polymerase-specificity, the critical TBP is complexed differently in the two genes and are not mutually exchangeable (Surig, D. et al. (1993) Gene Expression 3: 175-185). Aside from the U6 gene itself, 7SK represents the gene with the greatest degree of promoter similarity and transcriptional requirements to that of the oligonucleotide-producing gene. The function of this RNA gene product is still unknown.</p>
    <p>The spliceosomal U1 gene is transcribed by RNA polymerase II, rather than RNA polymerase III, and does not contain a TATA box; however, it does share the same requirement for the PSE and the DCR. Interestingly, the same PBP can activate both promoters, but, as was found for 7SK, the TBP resides in different complexes--TFIIB vs TFIIIB (Bernues J, et al. (1993) EMBO J. 12: 3573-3585). This difference is thought to provide the means for differential polymerase selectivity. Both U1 and 7SK are nuclear (but nonnucleolar), abundant, constitutively expressed, and stable (t 1/2≧24 h) as is U6. In addition, U1 is functionally related to U6 in the formation of an active spliceosome. Thus, the structural and functional similarities of 7SK and U1 to U6 and the oligonucleotide-producing gene make them important genes to monitor for signs of transcriptional dysfunction.</p>
    <p>U3 is also a small nuclear RNA gene, but does not function within the mRNA spliceosome, and is confined in location to the nucleolus (Carmo-Fonseca, M., et al. (1991) EMBO J. 10: 195-206). It is believed to play an important role in the splicing of ribosomal RNA. Glyceraldehyde-3-phosphate dehydrogenase (GAPDH) is a translated gene encoding an enzyme within the glycolytic pathway of glucose metabolism. Both of these genes are transcribed by RNA polymerase II and share only few common transcriptional factors with U6 or the oligonucleotide-producing gene. In addition, neither RNA reside primarily in the same subcellular component as U6 or the RNA oligonucleotide. The RNA from these genes allow monitoring of availability of more general transcription factors or substrates, which would lead to RNA downregulation, and nucleases, which would lead to RNA upregulation.</p>
    <p>To test for interference with the translational apparatus, a co-transfected, non-native gene was introduced. Monitoring of protein levels from this gene provided unambiguous information about the ability to synthesize a protein from a genomic template. Artifacts based upon differential or extended stabilities of endogenous proteins or mRNA cannot arise with this approach. In addition, the rapid expression of the RNA oligonucleotide upon transfection allowed for an examination of protein induction in the presence of virtually steady-state levels of the RNA oligonucleotide.</p>
    <p>Using the above described gene products as potential markers of RNA oligonucleotide-induced cellular toxicity, indications of RNA downregulation, upregulation or altered stability were examined. Neither general nor specific effects of RNA oligonucleotide production on cellular transcription could be detected in any of the above RNA, even at concentrations previously found detrimental to U6. Protein synthesis was similarly unaffected by the expression of the RNA oligonucleotides (stable oligonucleotides or unstable oligonucleotides). These findings suggest that, upon initial investigation, only U6, the gene most closely related to the oligonucleotide-producing gene, was adversely affected by RNA oligonucleotide expression. Other more distantly related cellular RNA appear to retain normal levels of expression and degradation in the presence of RNA oligonucleotide expression, and the protein synthesis pathway appears to function normally.</p>
    <p>Materials and Methods</p>
    <p>See the Examples above for details on construction of the chimeric genes, cells and cell culture, electroporation, transcription arrest, RNA isolation and polyacrylamide gel Northern blotting. The following Materials and Methods are relevant to this Example.</p>
    <p>Agarose gel Northern blotting</p>
    <p>A 1% agarose gel was prepared in 1× 3-(n-morpholino)propanesulfuric acid (MOPS) with formaldehyde and diethylpyrocarbonate (DEPC) treated water. Total cellular RNA was prepared for electrophoresis with the addition of 1× MOPS, 13.2% formaldehyde, 36.8% formamide, and 5% bromophenol blue/xylene cyanol loading buffer. Samples were heated for 10 min at 65° C. followed by the addition of 0.045 μg/ml ethidium bromide. The gel was electrophoresed in 1× MOPS at a constant 100 V, washed in DEPC-treated water and photographed. RNA was transferred to Hybond N nylon filters (Amersham, Arlington Heights, Ill.) by passive capillary transfer in 10× standard sodium citrate (SSC) overnight, followed by UV crosslinking for 2 min. Hybridization and washing conditions were as described in Example 6.</p>
    <p>Construction of U6mini</p>
    <p>The chimeric gene producing an RNA oligonucleotide referred to as U6mini was designed to contain strings of 4 thymines within the oligonucleotide insert region (at base positions 31-34 and 40-43) to terminate the transcript prematurely. The sequence of the upper strand of the inserted duplex fragment was:</p>
    <p>5' TCGACTTTTCTCCATTTTAGCTTCCTTAGCTCCTGATGCA 3'(SEQ ID NO:6)</p>
    <p>The digestion, ligation, and sequencing of this gene was as described in the previous chapter, Materials and Methods, "Construction of the chimeric gene."</p>
    <p>β-HCG quatitation</p>
    <p>Determination of cellular β-HCG was performed 48 hours after MDA453 cell electroporation with 2 μg of the β-HCG gene added to 20 μg of the chimeric oligonucleotide producing gene or promoterless control DNA using the Tandem R total β-HCG immunoradiometric assay (Hybritech, San Diego, Calif.). In this kit, anti-β-HCG antibodies were coupled to plastic beads which are then incubated with the supernatant from transfected cells. A secondary I<sup>125</sup> radiolabeled anti-β-HCG antibody was then added, and after a 1 hour incubation at 37° C., beads are washed in 0.01% sodium azide and quantitated by a gamma counting.</p>
    <p>Results</p>
    <p>FIG. 19 illustrates the lack of perturbation in 7SK, U1, or U3 RNA levels following transfection with increasing doses (5 to 40 μg DNA/10<sup>7</sup> cells) of the chimeric gene for U6ON in 293 cells. This gene has been demonstrated to generate RNA oligonucleotide levels ranging from 5×10<sup>4</sup> to 5×10<sup>6</sup> copies/cell and at higher levels was demonstrated to lead to a downregulation of U6 RNA. The absence of any detectable small nuclear RNA downregulation argues strongly that neither polymerases nor TBP or PBP transcription factors were in limiting supply. The absence of detectable RNA upregulation argues analogously that cellular nucleases were not saturated by the presence of additional RNA oligonucleotides.</p>
    <p>FIG. 20 illustrates a similar lack of perturbation in 7SK, U1, or U3 RNA stability. The nearly constant levels of these small nuclear RNA following transcription arrest 48 hours after electroporation with either 5 μg or 20 μg of the chimeric U6ON-producing gene demonstrated that these RNA retained their normal ≧24 hour t.sub. 1/2. This finding was not surprising for U1 or U3 RNA which obtain a trimethyl guanine 5' cap which has proven to contribute substantially to RNA stability. This cap is structurally distinct from that found on U6 and U6ON, and has been believed to be coupled to the process of transcription (Reddy R. et al. (1992) Pharm Ther. 54: 249-267). However, 7SK RNA is believed to obtain the same 5' γ-monomethyl phosphate cap found on U6 and U6ON. By the same reasoning used to explain U6 downregulation, one might expect that the saturation of the capping apparatus should similarly reduce 7SK stability and therefore overall 7SK levels. However, certain factors could explain this disparity. Most importantly, the increase in RNA stability afforded by the presence of the 5' cap has been shown to be far greater for U6 than for 7SK (&gt;3-fold difference in cap dependence at 8 h) (Shumyatsky G et al. (1993) Nucleic Acids Res 21: 4756-4761). Secondly, the absolute t.sub. 1/2  value has not been adequately determined for 7SK. If it is substantially larger than that of U6, one may need to carry out the transcription arrests for longer than 8 hours to see an effect. Finally the 330 nucleotide length of 7SK may involve more secondary structure or protein binding sites than the 108 nucleotide length of U6, protecting the RNA from more substantial degradation.</p>
    <p>The absence of an effect on the inherent stability of the generated RNA on 7SK and U1 RNA levels is demonstrated in FIG. 21. In the Examples above U6ON and U6CTcon were both found to be stable and abundant RNA oligonucleotides, whereas U6AS was found to be highly unstable and present at only barely detectable levels. Neither 7SK nor U1 RNA levels were altered perceptively by the different RNA oligonucleotide sequences, stabilities, or overall levels in MDA453 cells. Cellular GAPDH RNA levels were similarly unaffected by a 20 μg transfections of the chimeric gene for U6ON or U6AS monitored 48 to 96 hours post-transfection as seen in FIG. 22.</p>
    <p>FIG. 23 illustrates two representative examples of β-HCG protein quantitation 48 hours after co-transfection with 20 μg of various chimeric oligonucleotide-producing genes or promoterless control DNA in MDA453 cells. The presence of substantial levels of β-HCG attests to the presence of an intact protein synthesis pathway after transfection with different chimeric genes and subsequent production of RNA oligonucleotides. U6mini refers to construct which terminates at 34 nucleotides. Absolute levels of the protein were highly variable, but did not demonstrate any detectable trend correlating with oligonucleotide levels or oligonucleotide stability. Variations were assumed to result primarily from differences in transfection efficiency, cell survival rate post-electroporation, and experimental error. As expected, β-HCG levels were found to be positively affected by total co-transfected DNA concentrations; however, supplementing DNA concentrations with U6ON or promoterless control DNA gave rise to similar increases in total β-HCG levels (data not shown).</p>
    <p>A first analysis of the possible cellular toxicity arising from intracellular production of RNA oligonucleotides in high concentrations was undertaken in Examples 16 and 17. Upon transient transfections with increasing doses of a variety of chimeric genes, adverse effects were seen only in the U6 gene. This adverse effect took the form of a decrease in U6 stability leading to a decrease in overall U6 RNA levels; however, it was present only after transfection doses of ≧20 μg/10<sup>7</sup> cells and only with chimeric genes giving rise to stable RNA oligonucleotides. Lowering the transfection dose relieved both the decrease in stability as well as the decrease in overall U6 RNA levels. Interestingly, even in total cellular RNA samples demonstrating a significant decrease in U6 RNA from normal levels, no longer length non-spliced RNA from either the GAPDH or the human HER2 gene (as described in Example 14) could be detected. Thus, it is possible that lowered U6 levels would not negatively impact the general health of a cell. The finding that U6 is normally in 2-3 fold excess over U4 and U5 (found with U6 in bi-and tri-small nuclear riboprotein complexes) is consistent with this possibility (Sauterer R, et al. (1988) Exptl Cell Research 176:344-359).</p>
    <p>The conditions upon which U6 stability was decreased, along with the amount by which it was decreased, suggest strongly that the 5' capping apparatus can be saturated by large amounts of stable RNA oligonucleotides giving rise to a population of uncapped U6 RNA. This suggestion, while not yet definitively proven, leads to in vivo evidence that in U6, capping and transcription are uncoupled. Such evidence is consistent with a previous report demonstrating in vitro evidence of uncoupling (Gupta et al. (1990) J. Biol. Chem. 265: 9491-9495), as well as with a previous report that the activity of a purified capping factor on labeled U6 RNA could be quenched by excess unlabeled U6 RNA (Shimba S. and Reddy R. (1994) J. Biol. Chem. 269:12419-12423).</p>
    <p>No other adverse effects of RNA oligonucleotide production could be detected in several other small nuclear RNA with differing transcription factor, polymerase, and capping requirements. U1, U3, and 7SK demonstrated no apparent upregulation or downregulation upon transfection with increasing doses on a variety of chimeric genes. Levels of the "house-keeping" RNA, GAPDH, were similarly unaffected by oligonucleotide production as were protein levels arising from a non-natural co-transfected gene for β-HCG. These results imply that, upon first analysis, transient transfection with subsequent production of high concentrations of an RNA oligonucleotide does not significantly impair the general properties of cellular transcription or translation. Further experimentation with cells stably transfected with the chimeric gene will provide much needed additional information on toxicity.</p>
    <heading>Example 17</heading> <p>Specificity of Triplex Formation with Intracellularly Generated RNA Oleonucleotides</p>
    <p>Triplex formation with RNA oligonucleotides and double-stranded DNA provides a means of controlling gene expression from specific promoters and/or creating more selective DNA cleaving agents. The development of a novel technique, Triplex blotting, designed to detect RNA species capable of triplex formation with radiolabeled double-stranded DNA probes within a background of total cellular RNA is described. Triplex blotting offers a new approach for screening potential RNA sequences for triplex formation with double-stranded DNA targets, for comparing relative binding affinities of various triplex-forming RNAs, and for confirming the specificity of triplex formation of a DNA target probe within total cellular RNA. In addition, the technique allows for repeated probing of the same filter while varying critical hybridization conditions such as pH, temperature, or ionic strength.</p>
    <p>Interest in oligonucleotides designed to form triple helices on double-stranded DNA has been steadily increasing, primarily due to their potential as artificial repressors of gene expression (Helene, C. (1991) Anticancer Drug Design 6:569-584; Ing, N. H. et al. (1993) Nuc. Acids Res. 21:2789-2796; Maher III, L. J. et al (1989) Science 245:725-730; and Postel, E. H. et al. (1991) Proc. Natl. Acad. Sci. USA 88:8227-8231) and their potential as mediators of site-specific DNA cleavage (Moser, H. E. and Dervan, P. B. (1987) Science 238:645-650; and Strobel, S. A. and Dervan, P. B. (1992) Met. Enzymol. 216:309-321). While the majority of reports cite activity from DNA oligonucleotides, recent reports demonstrate that triplex formation with RNA oligonucleotides may result in a marked increase of free energy of formation over their DNA oligonucleotide counterparts (Roberts, R. W. and Crothers, D. M. (1992) Science 258:1463-1467).</p>
    <p>A general technique is described in this Example for the detection of specific RNA molecules capable of triplex formation by a method referred to as Triplex blotting. Similar in nature to standard Northern and Southern blotting techniques, Triplex blotting allows for the detection of a single species of cellular or in vitro generated RNA by triplex formation on radiolabeled double-stranded DNA probes. The benefits of triplex blotting include the sensitive and specific detection of homopurine or homopyrimidine RNA sequences, the rapid screening of duplex DNA target sequences against potential triplex forming RNAs (with direct comparison of relative binding affinities), and the confirmation of specificity of triplex formation amidst a background of total cellular RNA. Moreover, a single blot can be probed multiple times allowing for direct comparison of triplex binding in a variety of hybridization conditions.</p>
    <p>Materials and Methods</p>
    <p>In vitro generation of triplex RNA</p>
    <p>CU-rich RNA was generated in vitro from a pBluescript (Stratagene, La Jolla, Calif.) derivative in which the multicloning site was replaced with a 28 base-pair synthetic oligonucleotide duplex. When linearized with SstI and transcribed by T3 RNA polymerase (GIBCO BRL/Life Technologies, Gaithersberg, Md.) the transcript 5' UCC UCU UCC UCC UCC CCC UCC UCC UCCC 3' was generated. When linearized with KpnI and transcribed by T7 RNA polymerase (GIBCO BRL) in the opposite orientation, the corresponding antisense GA-rich strand was generated. Both transcripts also contained approximately 12 nucleotides of flanking RNA derived from cloning and the polymerase start sequences. Control RNA was generated by linearizing pBluescript with EcoR1 in the center of its multicloning site and transcribing with T3 RNA polymerase to give similar full length transcripts. All restriction enzymes were purchased from GIBCO/BRL.</p>
    <p>In vivo generation of triplex RNA oligonucleotides</p>
    <p>Sequence-specific CU-rich RNA oligonucleotides were generated intracellularly after electroporation with 10 or 20 μg of the chimeric gene producing the RNA oligonucleotide U6ON per 10<sup>7</sup> cells. Details on the construction of this chimeric gene is found in the Examples above. Control SKBR3 cell populations were electroporated with 20 μg of promoterless plasmid DNA. RNA was isolated 48 hours after transfection by the guanidine isothiocyanate/CsCl method (Glisin, V. R. et al. (1974) Biochem. 13:2633-2643).</p>
    <p>Cells and cell culture</p>
    <p>Intracellular triplex RNA oligonucleotides were generated in the human breast cell line, SKBR3 (ATCC, Rockville, Md.). These cells were grown in McCoy's medium without Tricine supplemented with 10% fetal calf serum and 100 U/ml of penicillin/streptomycin, and were maintained before and after transfection at 5% CO<sub>2</sub>. Transfection occurred by electroporation (Biorad, Cambridge, Mass.) using settings as previously described in Example 5.</p>
    <p>Radiolabeling the duplex probe</p>
    <p>The double-stranded probe for Triplex blotting was generated by annealing a 43-mer oligonucleotide with a short 10-mer primer and extending with a<sup>32</sup> P-labeled dCTP, 0.5 mM dATP, dTTP, and dGTP and Exo Klenow fragment (Stratagene, La Jolla, Calif.) for 25 minutes followed by a 10 minute chase with 0.5 mM cold dCTP. The radiolabel was constrained to the strand unable to form a Watson-Crick duplex with the generated RNA oligonucleotide. The 43 base-pair duplex corresponds to -76 to -34 of the human HER2 promoter within which is a 28 base-pair triplex target (FIG. 24).</p>
    <p>Triplex blotting</p>
    <p>Triplex blots were obtained by first electrophoretically fractionating in vivo or in vitro generated RNA on 7 M urea, 6% polyacrylamide gels, transferring to nylon Hybond-N filter (Amersham, Arlington Heights, Ill.) by electroblotting in a buffer composed of 17 mM NaH<sub>2</sub> PO<sub>4</sub> /8 mM Na<sub>2</sub> HPO<sub>4</sub>, followed by UV crosslinking for 2 min. Filters were prehybridized for 1 hour at 20° C. in a solution containing 500 mM sodium acetate, 5 mM EDTA brought to pH 5.5 with glacial acetic acid (5× NAE), 5× Denhardt's solution, 1% sodium dodecyl sulfate (SDS), and 20 μg/ml of salmon sperm DNA. Prehybridization was followed by overnight hybridization with 10 pmoles of the radiolabeled probe in the same solution at room temperature with gentle shaking. Filters were subsequently washed twice with shaking in 2× NAE pH 5.5, 0.1% SDS for 20 minutes at room temperature.</p>
    <p>FIG. 24 illustrates the double-stranded triplex target within the HER2 proximal promoter and the CU-rich, GA-rich, and control RNA sequences. Note the presence of a single mismatch within the homopurine/homopyrimidine region. CU-rich triplex RNA and DNA oligonucleotides have been shown to bind tightly to double-stranded DNA in a pH-dependent fashion, oriented parallel to the purine strand in the major groove (Felsenfeld, G. et al. (1957) J. Am. Chem. Soc. 79:2023-2024; and Moser, H. E. and Dervan, P. B. (1987) Science 238:645-650). GA-rich triplex DNA oligonucleotides have been demonstrated to bind in a Mg<sup>++</sup> dependent fashion, oriented antiparallel to the purine strand in the major groove (Beal, P. A. and Dervan, P. B. (1991) Science 251:1360-1363). Neither GA-rich nor GU-rich triplex RNA oligonucleotides have been documented to form triple helices with double-stranded DNA under any known conditions (Skoog, J. U. and Maher, L. J. (1993) Nuc. Acids Res. 21:2131-2138).</p>
    <p>Given the orientational preferences and the homopurine/pyrimidine nature of the triplex oligonucleotides, both GA-rich and CU-rich triplex RNAs could be generated in vitro from a single plasmid with oppositely directed bacterial promoters. The control RNA was generated by run-off transcription from pBluescript linearized at the center of the multicloning site. After in vitro transcription, RNA samples were electrophoretically fractionated, transferred and crosslinked to a nylon filter, and hybridized with the <sup>32</sup> P-labeled double-stranded DNA probe bearing the triplex target sequence. FIG. 25 demonstrates the sensitive and precise detection of triplex-forming CU-rich RNA by triplex blotting. GA-rich and mixed-sequence control RNA are not detected as they are unable to form triplex structures. Parallel experiments using <sup>32</sup> P-labeled UTP or GTP in the in vitro transcription reaction confirmed that the transcripts' mobility corresponded to those of the triplex RNA oligonucleotides.</p>
    <p>This Triplex blotting technique was tested on total cellular RNA samples after triplex RNA transcripts were generated intracellularly through a plasmid bearing the U6 gene with its internal 25-87 nucleotide sequence replaced by the CU-rich triplex-forming RNA sequence shown in FIG. 24. The U6 gene has previously been shown to be transcribed by RNA polymerase III, however it does not contain any intragenic control regions characteristic of other class III genes (Kunkel, G. R. et al. (1987) Proc. Natl. Acad. Sci. USA 83:8575-8579; and Kunkel, G. R. and Pederson, T. (1989) Nuc. Acids Res. 18:7371-7379). Therefore, the internal U6 sequence can be replaced by a triplex-forming (or antisense) sequence and hybrid RNA sequences can be generated intracellularly. The generated triplex RNA oligonucleotide has previously been referred to as U6ON.</p>
    <p>In FIG. 26a, SKBR3 cells were transfected with either 10 or 20 μg (lanes 2 and 3) of the modified U6 triplex-forming RNA generating plasmid, and after 48 hours RNA was isolated, fractionated and blotted as performed with the in vitro generated RNA. Control SKBR3 cell populations were transfected with 20 μg of promoterless plasmid DNA (lane 1). FIG. 26a demonstrates that the double-stranded probe cleanly detects the specific triplex-forming RNA species, U6ON, amidst the background of total cellular RNA. This filter was then stripped in 0.5× NAE, pH 7.5 at 70° C. for 1 hour and reprobed by a standard Northern blot to verify transcript generation and mobility (FIG. 28b).</p>
    <p>To determine optimal binding conditions or to investigate stringency of triplex interactions, ionic strength or pH can be altered in the hybridization buffer. For instance, hybridization and washing at pH 7.5 eliminated virtually all triplex signal from both the in vitro generated RNA filter of FIG. 25 as well as the in vivo generated RNA filter of FIG. 26.</p>
    <p>These results clearly demonstrate the feasibility, sensitivity, and specificity of Triplex blotting to detect triplex RNA and establishes this technique as a new tool to aid researchers investigating the potential biological applications of triplex technology.</p>
    <p>The results presented in this Example accomplish several major goals. The ability of CU-rich RNA sequences but not GA-rich RNA sequences to form triplexes with a double-stranded homopurine/homopyrimidine DNA target is confirmed. This target sequence (corresponding to a 28 bp region of the HER2/c-erbB2/neu proto-oncogene promoter) and CU-rich triplex RNA are thus justified for use in further investigation of the in vivo effect of an intracellularly generated triplex RNA oligonucleotide.</p>
    <p>The technique of Triplex blotting also confirms the specificity of triplex binding to the target sequence. Within a background of total cellular RNA, only the CU-rich in vivo generated RNA is detected by the duplex probe. Thus, probability is reduced that another cellular RNA transcript will interfere by triplex interactions with the target sequence. This technique also offers a means of testing the optimal pH, temperature, and ionic strength conditions for RNA triplex oligonucleotides as has been previously tested for DNA triplex oligonucleotides. The distinct advantage of this new method over those currently in use (such as gel retardation assays and DNase I footprinting) is the ability to repeatedly change hybridization conditions on the same filter instead of needing to repeat the entire experiment with each new set of conditions, thus reducing experimental error. Finally, Triplex blotting offers a means to test not only the specificity of triplex formation, but also the stringency of triplex formation. By creating in vivo generated CU-rich triplex RNA oligonucleotides with various mismatches, the tolerance to these mismatches can be studied in a variety of hybridization conditions.</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">__________________________________________________________________________SEQUENCE LISTING(1) GENERAL INFORMATION:(iii) NUMBER OF SEQUENCES: 25(2) INFORMATION FOR SEQ ID NO:1:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 38 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:1:TCGACTCCTCTTCCTCCTCCACCTCCTCCTCCCATGCA38(2) INFORMATION FOR SEQ ID NO:2:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 38 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:2:TCGACCTCCCTTCCCTTCCCTTCCCCTTCCTCCATGCA38(2) INFORMATION FOR SEQ ID NO:3:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 40 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:3:TCGACATGAGCATTCATCAGGCGGGCAAGAATGTGATGCA40(2) INFORMATION FOR SEQ ID NO:4:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 39 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:4:TCGAGCATGGCCCCTGCGCAAGGATGACACGCAAATGCA39(2) INFORMATION FOR SEQ ID NO:5:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 38 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:5:TCGACCGCCCCGCCCTGCCACTCATCGCAGTACATGCA38(2) INFORMATION FOR SEQ ID NO:6:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 40 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:6:TCGACTTTTCTCCATTTTAGCTTCCTTAGCTCCTGATGCA40(2) INFORMATION FOR SEQ ID NO:7:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 21 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:7:GTCCTAGGCTTTTGCACTTTT21(2) INFORMATION FOR SEQ ID NO:8:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 21 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: double(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:8:GTCCTAGGCTTTTGCACTTTT21(2) INFORMATION FOR SEQ ID NO:9:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 21 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: double(D) TOPOLOGY: linear(iv) ANTI-SENSE: YES(xi) SEQUENCE DESCRIPTION: SEQ ID NO:9:AAAAGTCCAAAAGCCTAGGAC21(2) INFORMATION FOR SEQ ID NO:10:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 50 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: double(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:10:ATTCTTATACTTCCTCAAGCAGCCCTCCTCCTCCACCTCCTCCTTCTCCT50(2) INFORMATION FOR SEQ ID NO:11:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 50 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: double(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:11:AGGAGAAGGAGGAGGTGGAGGAGGAGGGCTGCTTGAGGAAGTATAAGAAT50(2) INFORMATION FOR SEQ ID NO:12:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 28 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:12:UCCUCUUCCUCCUCCCCCUCCUCCUCCC28(2) INFORMATION FOR SEQ ID NO:13:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 28 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:13:GGGAGGAGGAGGGGGAGGAGGAAGAGGA28(2) INFORMATION FOR SEQ ID NO:14:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 82 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: both(xi) SEQUENCE DESCRIPTION: SEQ ID NO:14:GUGCUCGCUUCGGCAGCACAUAUCCUCGACCUCCCUUCCCUUCCCUUCCCCUUCCUCCAU60GCAUGAAGCGUUCCAUAUUUUU82(2) INFORMATION FOR SEQ ID NO:15:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 84 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: both(xi) SEQUENCE DESCRIPTION: SEQ ID NO:15:GUGCUCGCUUCGGCAGCACAUAUCCUCGACAUGAGCAUUCAUCAGGCGGGCAAGAAUGUG60AUGCAUGAAGCGUUCCAUAUUUUU84(2) INFORMATION FOR SEQ ID NO:16:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 28 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:16:UCCUCUUCCUCCUCCCCCUCCUCCUCCC28(2) INFORMATION FOR SEQ ID NO:17:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 58 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: double(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:17:CCCAATCACAGGAGAAGGAGGAGGTGGAGGAGGAGGGCTGCTTGAGGAAGTATAAGAA58(2) INFORMATION FOR SEQ ID NO:18:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 58 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: double(D) TOPOLOGY: linear(iv) ANTI-SENSE: YES(xi) SEQUENCE DESCRIPTION: SEQ ID NO:18:TTCTTATACTTCCTCAAGCAGCCCTCCTCCTCCACCTCCTCCTTCTCCTGTGATTGGG58(2) INFORMATION FOR SEQ ID NO:19:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 43 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: double(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:19:CAAGCAGCCCTCCTCCTCCACCTCCTCCTTCTCCTGTGATTGG43(2) INFORMATION FOR SEQ ID NO:20:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 43 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: double(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:20:CCAATCACAGGAGAAGGAGGAGGTGGAGGAGGAGGGCTGCTTG43(2) INFORMATION FOR SEQ ID NO:21:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 28 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:21:UCCUCUUCCUCCUCCCCCUCCUCCUCCC28(2) INFORMATION FOR SEQ ID NO:22:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 28 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:22:GGGAGGAGGAGGGGGAGGAGGAAGAGGA28(2) INFORMATION FOR SEQ ID NO:23:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 28 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:23:GGGCCCCCCCUCGAGGUCGACGGUAUCG28(2) INFORMATION FOR SEQ ID NO:24:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 323 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:24:TTCCCATGATTCCTTCATATTTGCATATACGATACAAGGCTGTTAGAGAGATAATTAGAA60TTAATTTGACTGTAAACACAAAGATATTAGTACAAAATACGTGACGTAGAAAGTAATAAT120TTCTTGGGTAGTTTGCAGTTTTTAAAATTATGTTTTAAAATGGACTATCATATGCTTACC180GTAACTTGAAAGTATTTCGATTTCTTGGCTTTATATATCTTGTGGAAAGGACGAAACACC240GTGCTCGCTTCGGCAGCACATATCCTCGAGCATGGCCCCTGCGCAAGGATGACACGCAAA300TGCATGAAGCGTTCCATATTTTT323(2) INFORMATION FOR SEQ ID NO:25:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 322 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:25:TTCCCATGATTCCTTCATATTTGCATATACGATACAAGGCTGTTAGAGAGATAATTAGAA60TTAATTTGACTGTAAACACAAAGATATTAGTACAAAATACGTGACGTAGAAAGTAATAAT120TTCTTGGGTAGTTTGCAGTTTTTAAAATTATGTTTTAAAATGGACTATCATATGCTTACC180GTAACTTGAAAGTATTTCGATTTCTTGGCTTTATATATCTTGTGGAAAGGACGAAACACC240GTGCTCGCTTCGGCAGCACATATCCTCGACTCCTCTTCCTCCTCCACCTCCTCCTCCCAT300GCATGAAGCGTTCCATATTTTT322__________________________________________________________________________</pre>
    
    </div></div></div><div class="patent-section patent-tabular-section"><a id="backward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Cited Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5190931">US5190931</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Nov 15, 1989</td><td class="patent-data-table-td patent-date-value">Mar 2, 1993</td><td class="patent-data-table-td ">The Research Foundation Of State University Of New York</td><td class="patent-data-table-td ">Regulation of gene expression by employing translational inhibition of MRNA utilizing interfering complementary MRNA</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5272065">US5272065</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jun 21, 1990</td><td class="patent-data-table-td patent-date-value">Dec 21, 1993</td><td class="patent-data-table-td ">Research Foundation Of State University Of New York</td><td class="patent-data-table-td ">Regulation of gene expression by employing translational inhibition of MRNA utilizing interfering complementary MRNA</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5316930">US5316930</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jan 5, 1989</td><td class="patent-data-table-td patent-date-value">May 31, 1994</td><td class="patent-data-table-td ">Pioneer Hi-Bred International, Inc.</td><td class="patent-data-table-td ">DNA molecules and plant or bacteria cells</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP0601585A2?cl=en">EP0601585A2</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Dec 9, 1993</td><td class="patent-data-table-td patent-date-value">Jun 15, 1994</td><td class="patent-data-table-td ">Enzo Therapeutics, Inc.</td><td class="patent-data-table-td ">Nucleic acid construct for inhibiting or regulating the functions of an immuno response gene, method utilizing the same and immunologically compatible system containing the same</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1995006718A2?cl=en">WO1995006718A2</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Sep 2, 1994</td><td class="patent-data-table-td patent-date-value">Mar 9, 1995</td><td class="patent-data-table-td ">Viagene Inc</td><td class="patent-data-table-td ">Methods of suppressing autoimmune response</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1995006744A2?cl=en">WO1995006744A2</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Sep 2, 1994</td><td class="patent-data-table-td patent-date-value">Mar 9, 1995</td><td class="patent-data-table-td ">Viagene Inc</td><td class="patent-data-table-td ">Methods of suppressing immune response by gene therapy</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="npl-citations"></a><div class="patent-section-header"><span class="patent-section-title">Non-Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th colspan="3"class="patent-data-table-th">Reference</th></tr></thead><tr><td class="patent-data-table-td ">1</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Case et al., "<a href='http://scholar.google.com/scholar?q="The+unusual+stability+of+the+IS10+anti-sense+RNA+is+critical+for+its+function+and+is+determined+by+the+structure+of+its+stem-domain"'>The unusual stability of the IS10 anti-sense RNA is critical for its function and is determined by the structure of its stem-domain</a>" EMBO J. (1989) 8:4297-4305.</td></tr><tr><td class="patent-data-table-td ">2</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Case et al., The unusual stability of the IS10 anti sense RNA is critical for its function and is determined by the structure of its stem domain EMBO J. (1989) 8:4297 4305.</td></tr><tr><td class="patent-data-table-td ">3</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Chrisey et al., "<a href='http://scholar.google.com/scholar?q="Antisense+technology%3A+Principles+and+prospects+for+therapeutic+development"'>Antisense technology: Principles and prospects for therapeutic development</a>" BioPharm (1991) pp. 36-42.</td></tr><tr><td class="patent-data-table-td ">4</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Chrisey et al., Antisense technology: Principles and prospects for therapeutic development BioPharm (1991) pp. 36 42.</td></tr><tr><td class="patent-data-table-td ">5</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Cooney et al., "<a href='http://scholar.google.com/scholar?q="Site-specific+oligonucleotide+binding+represses+transcription+of+the+human+c-myc+gene+in+vitro"'>Site-specific oligonucleotide binding represses transcription of the human c-myc gene in vitro</a>" Science (1988) 241:456-459.</td></tr><tr><td class="patent-data-table-td ">6</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Cooney et al., Site specific oligonucleotide binding represses transcription of the human c myc gene in vitro Science (1988) 241:456 459.</td></tr><tr><td class="patent-data-table-td ">7</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Das et al., "<a href='http://scholar.google.com/scholar?q="Upstream+regulatory+elements+are+necessary+and+sufficient+for+transcription+of+a+U6+RNA+gene+by+RNA+polymerase+III"'>Upstream regulatory elements are necessary and sufficient for transcription of a U6 RNA gene by RNA polymerase III</a>" EMBO J. (1988) 7:503-512.</td></tr><tr><td class="patent-data-table-td ">8</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Das et al., Upstream regulatory elements are necessary and sufficient for transcription of a U6 RNA gene by RNA polymerase III EMBO J. (1988) 7:503 512.</td></tr><tr><td class="patent-data-table-td ">9</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Durland et al., "<a href='http://scholar.google.com/scholar?q="Binding+of+triple+helix+forming+oligonucleotides+to+sites+in+gene+promoters"'>Binding of triple helix forming oligonucleotides to sites in gene promoters</a>" Biochem. (1991) 30:9246-9255.</td></tr><tr><td class="patent-data-table-td ">10</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Durland et al., Binding of triple helix forming oligonucleotides to sites in gene promoters Biochem. (1991) 30:9246 9255.</td></tr><tr><td class="patent-data-table-td ">11</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Duval Valentin et al., Specific inhibition of transcription by triple helix forming oligonucleotides Proc. Natl. Acad. Sci. USA (1992) 89:504 508.</td></tr><tr><td class="patent-data-table-td ">12</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Duval-Valentin et al., "<a href='http://scholar.google.com/scholar?q="Specific+inhibition+of+transcription+by+triple+helix-forming+oligonucleotides"'>Specific inhibition of transcription by triple helix-forming oligonucleotides</a>" Proc. Natl. Acad. Sci. USA (1992) 89:504-508.</td></tr><tr><td class="patent-data-table-td ">13</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">H e l e ne, The anti gene strategy: control of gene expression by triplex forming oligonucleotides Anti Cancer Drug Design (1991) 6:569 584.</td></tr><tr><td class="patent-data-table-td ">14</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">H el ene, "<a href='http://scholar.google.com/scholar?q="The+anti-gene+strategy%3A+control+of+gene+expression+by+triplex-forming-oligonucleotides"'>The anti-gene strategy: control of gene expression by triplex-forming-oligonucleotides</a>" Anti-Cancer Drug Design (1991) 6:569-584.</td></tr><tr><td class="patent-data-table-td ">15</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Hannon et al., "<a href='http://scholar.google.com/scholar?q="Multiple+cis-acting+elements+are+required+for+RNA+polymerase+III+transcription+of+the+gene+encoding+H1+RNA%2C+the+RNA+component+of+human+RNase+P"'>Multiple cis-acting elements are required for RNA polymerase III transcription of the gene encoding H1 RNA, the RNA component of human RNase P</a>" J. Biol. Chem. (1991) 266:22796-22799.</td></tr><tr><td class="patent-data-table-td ">16</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Hannon et al., Multiple cis acting elements are required for RNA polymerase III transcription of the gene encoding H1 RNA, the RNA component of human RNase P J. Biol. Chem. (1991) 266:22796 22799.</td></tr><tr><td class="patent-data-table-td ">17</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Izant, "<a href='http://scholar.google.com/scholar?q="Chimeric+antisense+RNAs"'>Chimeric antisense RNAs</a>" Gene Regulation: Biology of Antisense RNA and DNA (1992) Erickson, R.P. et al., eds., Raven Press, New York, pp. 183-195.</td></tr><tr><td class="patent-data-table-td ">18</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Izant, Chimeric antisense RNAs Gene Regulation: Biology of Antisense RNA and DNA (1992) Erickson, R.P. et al., eds., Raven Press, New York, pp. 183 195.</td></tr><tr><td class="patent-data-table-td ">19</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Jennings et al., "<a href='http://scholar.google.com/scholar?q="Inhibition+of+SV40+replicon+function+by+engineered+antisense+RNA+transcribed+by+RNA+polymerase+III"'>Inhibition of SV40 replicon function by engineered antisense RNA transcribed by RNA polymerase III</a>" EMBO J. (1987) 6:3043-3047.</td></tr><tr><td class="patent-data-table-td ">20</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Jennings et al., Inhibition of SV40 replicon function by engineered antisense RNA transcribed by RNA polymerase III EMBO J. (1987) 6:3043 3047.</td></tr><tr><td class="patent-data-table-td ">21</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Junker et al., "<a href='http://scholar.google.com/scholar?q="Reduction+in+replication+of+the+human+immunodeficiency+virus+type+1+in+human+cell+lines+by+polymerase+III-driven+transcription+of+chimeric+tRNA-antisense+RNA+genes"'>Reduction in replication of the human immunodeficiency virus type 1 in human cell lines by polymerase III-driven transcription of chimeric tRNA-antisense RNA genes</a>" Antisense Res. &amp; Develop. (1994) 4:165-172.</td></tr><tr><td class="patent-data-table-td ">22</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Junker et al., Reduction in replication of the human immunodeficiency virus type 1 in human cell lines by polymerase III driven transcription of chimeric tRNA antisense RNA genes Antisense Res. &amp; Develop. (1994) 4:165 172.</td></tr><tr><td class="patent-data-table-td ">23</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Kunkel et al., "<a href='http://scholar.google.com/scholar?q="Transcription+of+a+human+U6+small+nuclear+RNA+gene+in+vivo+withstands+deletion+of+intragenic+sequences+but+not+for+an+upstream+TATATA+box"'>Transcription of a human U6 small nuclear RNA gene in vivo withstands deletion of intragenic sequences but not for an upstream TATATA box</a>" Nucleic Acids Res. (1989) 17:7371-7379.</td></tr><tr><td class="patent-data-table-td ">24</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Kunkel et al., "<a href='http://scholar.google.com/scholar?q="U6+small+nuclear+RNA+is+transcribed+by+RNA+polymerase+III"'>U6 small nuclear RNA is transcribed by RNA polymerase III</a>" Proc. Natl. Acad. USA (1986) 83:8575-8579.</td></tr><tr><td class="patent-data-table-td ">25</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Kunkel et al., Transcription of a human U6 small nuclear RNA gene in vivo withstands deletion of intragenic sequences but not for an upstream TATATA box Nucleic Acids Res. (1989) 17:7371 7379.</td></tr><tr><td class="patent-data-table-td ">26</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Kunkel et al., U6 small nuclear RNA is transcribed by RNA polymerase III Proc. Natl. Acad. USA (1986) 83:8575 8579.</td></tr><tr><td class="patent-data-table-td ">27</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Lyamichev et al, Nucleic Acids Res. 16: 2165 (1988).</td></tr><tr><td class="patent-data-table-td ">28</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Marshallsay et al., "<a href='http://scholar.google.com/scholar?q="Characterization+of+the+U3+and+U6+snRNA+genes+from+wheat%3A+U3snRNA+genes+in+monocot+plants+are+transcribed+by+RNA+polymerase+III"'>Characterization of the U3 and U6 snRNA genes from wheat: U3snRNA genes in monocot plants are transcribed by RNA polymerase III</a>" Plant Mol. Biol. (1992) 19:973-983.</td></tr><tr><td class="patent-data-table-td ">29</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Marshallsay et al., Characterization of the U3 and U6 snRNA genes from wheat: U3snRNA genes in monocot plants are transcribed by RNA polymerase III Plant Mol. Biol. (1992) 19:973 983.</td></tr><tr><td class="patent-data-table-td ">30</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Moffat, "<a href='http://scholar.google.com/scholar?q="Making+sense+of+antisense"'>Making sense of antisense</a>" Science (1991) 253:510-511.</td></tr><tr><td class="patent-data-table-td ">31</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Moffat, Making sense of antisense Science (1991) 253:510 511.</td></tr><tr><td class="patent-data-table-td ">32</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Murphy et al., "<a href='http://scholar.google.com/scholar?q="The+in+vitro+transcription+of+the+7SK+RNA+gene+by+RNA+polymerase+III+is+dependent+only+on+the+presence+of+an+upstream+promoter"'>The in vitro transcription of the 7SK RNA gene by RNA polymerase III is dependent only on the presence of an upstream promoter</a>" Cell (1987) 51:81-87.</td></tr><tr><td class="patent-data-table-td ">33</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Murphy et al., The in vitro transcription of the 7SK RNA gene by RNA polymerase III is dependent only on the presence of an upstream promoter Cell (1987) 51:81 87.</td></tr><tr><td class="patent-data-table-td ">34</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Noonberg et al., "<a href='http://scholar.google.com/scholar?q="Detection+of+triplex-forming+RNA+oligonucleotides+by+triplex+blotting"'>Detection of triplex-forming RNA oligonucleotides by triplex blotting</a>" BioTechniques (1994) 16:1070-1072.</td></tr><tr><td class="patent-data-table-td ">35</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Noonberg et al., "<a href='http://scholar.google.com/scholar?q="In+vivo+generation+of+highly+abundant+sequence-specific+oligonucleotides+for+antisense+and+triplex+gene+regulation"'>In vivo generation of highly abundant sequence-specific oligonucleotides for antisense and triplex gene regulation</a>" Nucleic Acids Res. (1994) 22:2830-2836.</td></tr><tr><td class="patent-data-table-td ">36</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Noonberg et al., Detection of triplex forming RNA oligonucleotides by triplex blotting BioTechniques (1994) 16:1070 1072.</td></tr><tr><td class="patent-data-table-td ">37</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Noonberg et al., In vivo generation of highly abundant sequence specific oligonucleotides for antisense and triplex gene regulation Nucleic Acids Res. (1994) 22:2830 2836.</td></tr><tr><td class="patent-data-table-td ">38</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Sullenger et al., "<a href='http://scholar.google.com/scholar?q="Expression+of+chimeric+tRNA-driven+antisense+transcripts+renders+NIH+3T3+cells+highly+resistant+to+Moloney+murine+leukemia+virus+replication"'>Expression of chimeric tRNA-driven antisense transcripts renders NIH 3T3 cells highly resistant to Moloney murine leukemia virus replication</a>" Mol. &amp; Cell. Biol. (1990) 10:6512-6523.</td></tr><tr><td class="patent-data-table-td ">39</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Sullenger et al., Expression of chimeric tRNA driven antisense transcripts renders NIH 3T3 cells highly resistant to Moloney murine leukemia virus replication Mol. &amp; Cell. Biol. (1990) 10:6512 6523.</td></tr><tr><td class="patent-data-table-td ">40</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Terns et al., "<a href='http://scholar.google.com/scholar?q="Multiple+cis-acting+signals+for+export+of+pre-U1+snRNA+from+the+nucleus"'>Multiple cis-acting signals for export of pre-U1 snRNA from the nucleus</a>" Genes &amp; Development (1993) 7:1898-1908.</td></tr><tr><td class="patent-data-table-td ">41</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Terns et al., Multiple cis acting signals for export of pre U1 snRNA from the nucleus Genes &amp; Development (1993) 7:1898 1908.</td></tr><tr><td class="patent-data-table-td ">42</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Williard et al., "<a href='http://scholar.google.com/scholar?q="Paradoxical+production+of+target+protein+using+antisense+RNA+expression+vectors"'>Paradoxical production of target protein using antisense RNA expression vectors</a>" Gene (1994) 149:21-24.</td></tr><tr><td class="patent-data-table-td ">43</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Williard et al., Paradoxical production of target protein using antisense RNA expression vectors Gene (1994) 149:21 24.</td></tr><tr><td class="patent-data-table-td ">44</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Willis, "<a href='http://scholar.google.com/scholar?q="RNA+polymerase+III.+Genes%2C+factors+and+transcriptional+specificity"'>RNA polymerase III. Genes, factors and transcriptional specificity</a>" Eur. J. Biochem. (1993) 212:1-11.</td></tr><tr><td class="patent-data-table-td ">45</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Willis, RNA polymerase III. Genes, factors and transcriptional specificity Eur. J. Biochem. (1993) 212:1 11.</td></tr><tr><td class="patent-data-table-td ">46</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Yuan et al., "<a href='http://scholar.google.com/scholar?q="5%27+flanking+sequences+of+human+MRP%2F7-2+RNA+gene+are+required+and+sufficient+for+the+transcription+by+RNA+polymerase+III"'>5' flanking sequences of human MRP/7-2 RNA gene are required and sufficient for the transcription by RNA polymerase III</a>" Biochim. Biophys. Acta (1991) 1089:33-39.</td></tr><tr><td class="patent-data-table-td ">47</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Yuan et al., 5 flanking sequences of human MRP/7 2 RNA gene are required and sufficient for the transcription by RNA polymerase III Biochim. Biophys. Acta (1991) 1089:33 39.</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="forward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Referenced by</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Citing Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5814500">US5814500</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Oct 31, 1996</td><td class="patent-data-table-td patent-date-value">Sep 29, 1998</td><td class="patent-data-table-td ">The Johns Hopkins University School Of Medicine</td><td class="patent-data-table-td ">Delivery construct for antisense nucleic acids and methods of use</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5914230">US5914230</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Dec 20, 1996</td><td class="patent-data-table-td patent-date-value">Jun 22, 1999</td><td class="patent-data-table-td ">Dade Behring Inc.</td><td class="patent-data-table-td ">Homogeneous amplification and detection of nucleic acids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5948897">US5948897</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jun 13, 1997</td><td class="patent-data-table-td patent-date-value">Sep 7, 1999</td><td class="patent-data-table-td ">Simon Fraser University</td><td class="patent-data-table-td ">Method of binding two or more DNA double helices and products formed</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6127173">US6127173</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Sep 22, 1998</td><td class="patent-data-table-td patent-date-value">Oct 3, 2000</td><td class="patent-data-table-td ">Ribozyme Pharmaceuticals, Inc.</td><td class="patent-data-table-td ">Nucleotide sequence for targeted cleavage of nucleic acids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6133024">US6133024</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">May 7, 1998</td><td class="patent-data-table-td patent-date-value">Oct 17, 2000</td><td class="patent-data-table-td ">Aventis Pharma S.A.</td><td class="patent-data-table-td ">Recombinant vector which codes for an rna capable of forming a triple helix with target nucleic acid; for transcription of the rna in target cells</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6146886">US6146886</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Aug 7, 1995</td><td class="patent-data-table-td patent-date-value">Nov 14, 2000</td><td class="patent-data-table-td ">Ribozyme Pharmaceuticals, Inc.</td><td class="patent-data-table-td ">RNA polymerase III-based expression of therapeutic RNAs</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6183959">US6183959</a></td><td class="patent-data-table-td patent-date-value">Jul 8, 1998</td><td class="patent-data-table-td patent-date-value">Feb 6, 2001</td><td class="patent-data-table-td ">Ribozyme Pharmaceuticals, Inc.</td><td class="patent-data-table-td ">Identifying nucleic acid catalyst which modifies a biological system by introducing random library of catalysts into system then determining sequence of portion of substrate binding domain from system which has been modified</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6221628">US6221628</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Sep 8, 1997</td><td class="patent-data-table-td patent-date-value">Apr 24, 2001</td><td class="patent-data-table-td ">Brigham &amp; Women&#39;s Hospital</td><td class="patent-data-table-td ">Nucleotide sequences coding preferential polypeptide for use in diagnosis and treatment atherosclerosis, ischemia, hypertension, restenosis and arterial inflammation</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6346416">US6346416</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Aug 29, 2000</td><td class="patent-data-table-td patent-date-value">Feb 12, 2002</td><td class="patent-data-table-td ">Isis Pharmaceuticals, Inc.</td><td class="patent-data-table-td ">Antisense inhibition of HPK/GCK-like kinase expression</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6387699">US6387699</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Sep 8, 2000</td><td class="patent-data-table-td patent-date-value">May 14, 2002</td><td class="patent-data-table-td ">Isis Pharmaceuticals, Inc.</td><td class="patent-data-table-td ">Antisense inhibition of A20 expression</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6448009">US6448009</a></td><td class="patent-data-table-td patent-date-value">Sep 29, 2000</td><td class="patent-data-table-td patent-date-value">Sep 10, 2002</td><td class="patent-data-table-td ">Ribozyme Pharmaceuticals, Inc.</td><td class="patent-data-table-td ">Method for target site selection and discovery</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6492109">US6492109</a></td><td class="patent-data-table-td patent-date-value">Sep 21, 1998</td><td class="patent-data-table-td patent-date-value">Dec 10, 2002</td><td class="patent-data-table-td ">Gene Logic, Inc.</td><td class="patent-data-table-td ">Isolated oligonucleotide that specifically hybridizes to a brca2 nucleotide sequence having a specified deletion; gene which predisposes individuals to breast and ovarian cancer</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6492126">US6492126</a></td><td class="patent-data-table-td patent-date-value">Aug 11, 1999</td><td class="patent-data-table-td patent-date-value">Dec 10, 2002</td><td class="patent-data-table-td ">Millennium Pharmaceuticals, Inc.</td><td class="patent-data-table-td ">Compositions and methods for the treatment and diagnosis of cardiovascular disease</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6573054">US6573054</a></td><td class="patent-data-table-td patent-date-value">Jan 10, 2002</td><td class="patent-data-table-td patent-date-value">Jun 3, 2003</td><td class="patent-data-table-td ">Dade Behring Inc.</td><td class="patent-data-table-td ">Providing target polynucleotide, reference polynucleotides, oliognucleotide probes, subjecting to isothermal conditions for amplifying to form non- dissociative termolecular complexes; evaluation</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6656731">US6656731</a></td><td class="patent-data-table-td patent-date-value">Jan 7, 2000</td><td class="patent-data-table-td patent-date-value">Dec 2, 2003</td><td class="patent-data-table-td ">Max Planck Gesellschaft Zur Forderung Der Wissenschaften E.V.</td><td class="patent-data-table-td ">Nucleic acid catalysts with endonuclease activity</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6673611">US6673611</a></td><td class="patent-data-table-td patent-date-value">May 25, 2001</td><td class="patent-data-table-td patent-date-value">Jan 6, 2004</td><td class="patent-data-table-td ">Sirna Therapeutics, Inc.</td><td class="patent-data-table-td ">Nucleic acid molecules with novel chemical compositions capable of modulating gene expression</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6686163">US6686163</a></td><td class="patent-data-table-td patent-date-value">Dec 20, 2001</td><td class="patent-data-table-td patent-date-value">Feb 3, 2004</td><td class="patent-data-table-td ">Gene Logic Inc.</td><td class="patent-data-table-td ">Coding sequence haplotype of the human BRCA1 gene</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6692910">US6692910</a></td><td class="patent-data-table-td patent-date-value">Jul 27, 1998</td><td class="patent-data-table-td patent-date-value">Feb 17, 2004</td><td class="patent-data-table-td ">The University Of Connecticut</td><td class="patent-data-table-td ">Controlling, reducing gene expression; genetic engineering; viricides; anticancer agents; genetic disorders</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6831171">US6831171</a></td><td class="patent-data-table-td patent-date-value">Feb 8, 2001</td><td class="patent-data-table-td patent-date-value">Dec 14, 2004</td><td class="patent-data-table-td ">Yale University</td><td class="patent-data-table-td ">Nucleic acid catalysts with endonuclease activity</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6838256">US6838256</a></td><td class="patent-data-table-td patent-date-value">Dec 13, 2000</td><td class="patent-data-table-td patent-date-value">Jan 4, 2005</td><td class="patent-data-table-td ">Gene Logic Inc.</td><td class="patent-data-table-td ">Coding sequences of the human BRCA1 gene</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6852535">US6852535</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Aug 2, 2000</td><td class="patent-data-table-td patent-date-value">Feb 8, 2005</td><td class="patent-data-table-td ">Sirna Therapeutics, Inc.</td><td class="patent-data-table-td ">A transcribed non-naturally occurring RNA molecule comprising a desired RNA molecule, wherein the 3&#39; region of the RNA is able to base-pair with at least 8 bases at the 5&#39; terminus of the same RNA molecule</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6924096">US6924096</a></td><td class="patent-data-table-td patent-date-value">Sep 29, 1998</td><td class="patent-data-table-td patent-date-value">Aug 2, 2005</td><td class="patent-data-table-td ">The Johns Hopkins University School Of Medicine</td><td class="patent-data-table-td ">Includes intact stem loop structures and an antisense nucleic acid; useful for inhibition of selected genes in a cell; allele-specific targeting is also useful in combination replacement gene therapy</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6936461">US6936461</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jul 31, 2002</td><td class="patent-data-table-td patent-date-value">Aug 30, 2005</td><td class="patent-data-table-td ">New York Institute Of Technology</td><td class="patent-data-table-td ">Method for immobilizing multistranded nucleic acid molecules by modifying more than one strand thereof, and binding each strand to a solid support</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7022828">US7022828</a></td><td class="patent-data-table-td patent-date-value">May 28, 2002</td><td class="patent-data-table-td patent-date-value">Apr 4, 2006</td><td class="patent-data-table-td ">Sirna Theraputics, Inc.</td><td class="patent-data-table-td ">Anticancer agents, antiinflammatory agents, autoimmune disease; modulation of gene expression; nuclear factor kappa B</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7105657">US7105657</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Nov 8, 2002</td><td class="patent-data-table-td patent-date-value">Sep 12, 2006</td><td class="patent-data-table-td ">Board Of Regents Of The University Of Nebraska</td><td class="patent-data-table-td ">Compositions and methods for inhibiting pancreatic cancer metastasis</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7176304">US7176304</a></td><td class="patent-data-table-td patent-date-value">May 11, 2004</td><td class="patent-data-table-td patent-date-value">Feb 13, 2007</td><td class="patent-data-table-td ">Mcswiggen James</td><td class="patent-data-table-td ">RNA interference mediated inhibition of vascular endothelial growth factor and vascular endothelial growth factor receptor gene expression using short interfering nucleic acid (siNA)</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7241618">US7241618</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Dec 19, 2002</td><td class="patent-data-table-td patent-date-value">Jul 10, 2007</td><td class="patent-data-table-td ">Cancer Research Technology Limited</td><td class="patent-data-table-td ">A polynucleotide comprising a RNA polymerase promoter, able encoding a siRNA; treating cancer and genetic disorders; genetic engineering, inhibit gene expression of a target gene</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7273854">US7273854</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">May 26, 2000</td><td class="patent-data-table-td patent-date-value">Sep 25, 2007</td><td class="patent-data-table-td ">Sanofi-Aventis</td><td class="patent-data-table-td ">Recombinant adenovirus, adeno-associated virus or herpes virus (especially a replication defective virus) containing a nucleic acid of a specific sequence encoding an antisense RNA that inhibits expression of p53.</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7294504">US7294504</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jul 23, 2002</td><td class="patent-data-table-td patent-date-value">Nov 13, 2007</td><td class="patent-data-table-td ">Allele Biotechnology &amp; Pharmaceuticals, Inc.</td><td class="patent-data-table-td ">For suppressing the expression of a gene targets and for degrading a target RNA using DNA as a vehicle</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7422896">US7422896</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Aug 12, 2002</td><td class="patent-data-table-td patent-date-value">Sep 9, 2008</td><td class="patent-data-table-td ">Allele Biotechnology &amp; Pharmaceuticals, Inc.</td><td class="patent-data-table-td ">Compositions for DNA mediated gene silencing</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7517864">US7517864</a></td><td class="patent-data-table-td patent-date-value">Dec 9, 2005</td><td class="patent-data-table-td patent-date-value">Apr 14, 2009</td><td class="patent-data-table-td ">Sirna Therapeutics, Inc.</td><td class="patent-data-table-td ">Study and interfererance, diagnosis and treatment of polyglutamine using short interfering nucleic acid for treatment of dementia or siezures or muscular dystrophy</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7521431">US7521431</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Oct 31, 2003</td><td class="patent-data-table-td patent-date-value">Apr 21, 2009</td><td class="patent-data-table-td ">The Trustees Of The University Of Pennsylvania</td><td class="patent-data-table-td ">Compositions and methods for siRNA inhibition of HIF-1 alpha</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7528116">US7528116</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">May 29, 2003</td><td class="patent-data-table-td patent-date-value">May 5, 2009</td><td class="patent-data-table-td ">Memorial Sloan Kettering Cancer Center</td><td class="patent-data-table-td ">Kinase suppressor of Ras inactivation for therapy of Ras mediated tumorigenesis</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7538095">US7538095</a></td><td class="patent-data-table-td patent-date-value">Oct 30, 2002</td><td class="patent-data-table-td patent-date-value">May 26, 2009</td><td class="patent-data-table-td ">The Carnegie Institution Of Washington</td><td class="patent-data-table-td ">Genetic inhibition by double-stranded RNA</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7560438">US7560438</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Oct 30, 2002</td><td class="patent-data-table-td patent-date-value">Jul 14, 2009</td><td class="patent-data-table-td ">The Carnegie Institution Of Washington</td><td class="patent-data-table-td ">Genetic inhibition by double-stranded RNA</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7588915">US7588915</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Aug 18, 2003</td><td class="patent-data-table-td patent-date-value">Sep 15, 2009</td><td class="patent-data-table-td ">Ryogen Llc</td><td class="patent-data-table-td ">Isolated genomic polynucleotide fragments from chromosome 7</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7605249">US7605249</a></td><td class="patent-data-table-td patent-date-value">Nov 25, 2003</td><td class="patent-data-table-td patent-date-value">Oct 20, 2009</td><td class="patent-data-table-td ">Medtronic, Inc.</td><td class="patent-data-table-td ">To deliver specifically tailored small interfering RNA as therapeutic agents for treatment of Parkinson&#39;s disease</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7611723">US7611723</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Mar 20, 2003</td><td class="patent-data-table-td patent-date-value">Nov 3, 2009</td><td class="patent-data-table-td ">Nanocyte Inc.</td><td class="patent-data-table-td ">Stinging cells expressing an exogenous polynucleotide encoding a therapeutic, diagnostic or a cosmetic agent and methods compositions and devices utilizing such stinging cells or capsules derived therefrom for delivering the therapeutic, diagnostic or cosmetic agent into a tissue</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7612195">US7612195</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Oct 21, 2004</td><td class="patent-data-table-td patent-date-value">Nov 3, 2009</td><td class="patent-data-table-td ">Aventis Pharmaceuticals Inc.</td><td class="patent-data-table-td ">For carrying a target gene specific insert into a cell in order to modulate the expression of a target gene; modified Lentivirus vector</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7618948">US7618948</a></td><td class="patent-data-table-td patent-date-value">Oct 19, 2005</td><td class="patent-data-table-td patent-date-value">Nov 17, 2009</td><td class="patent-data-table-td ">Medtronic, Inc.</td><td class="patent-data-table-td ">Brain delivery of small interfering RNA or vectors containing the DNA encoding for small interfering RNA; beneficial effects on improving memory and/or cognitive function in a subject.</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7622633">US7622633</a></td><td class="patent-data-table-td patent-date-value">Oct 30, 2002</td><td class="patent-data-table-td patent-date-value">Nov 24, 2009</td><td class="patent-data-table-td ">Carnegie Institution Of Washington</td><td class="patent-data-table-td ">Introducing ribonucleic acid (RNA) into cell in an amount sufficient to inhibit expression of target gene, wherein RNA comprises a double-stranded structure with an identical nucleotide sequence as compared to portion of target gene</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7625750">US7625750</a></td><td class="patent-data-table-td patent-date-value">Dec 26, 2002</td><td class="patent-data-table-td patent-date-value">Dec 1, 2009</td><td class="patent-data-table-td ">Allele Biotechnology &amp; Pharmaceuticals, Inc.</td><td class="patent-data-table-td ">Compositions for DNA mediated gene silencing</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7635571">US7635571</a></td><td class="patent-data-table-td patent-date-value">Dec 7, 2000</td><td class="patent-data-table-td patent-date-value">Dec 22, 2009</td><td class="patent-data-table-td ">Siemens Healthcare Diagnostics Products Gmbh</td><td class="patent-data-table-td ">Amplified signal in binding assays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7638497">US7638497</a></td><td class="patent-data-table-td patent-date-value">Jun 30, 2006</td><td class="patent-data-table-td patent-date-value">Dec 29, 2009</td><td class="patent-data-table-td ">Aegera Therapeutics, Inc.</td><td class="patent-data-table-td ">For enhancing apoptosis and treating proliferative disorders; antitumor agents</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7638620">US7638620</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jun 8, 2006</td><td class="patent-data-table-td patent-date-value">Dec 29, 2009</td><td class="patent-data-table-td ">Aegera Therapeutics, Inc.</td><td class="patent-data-table-td ">Use to enhance apoptosis; antiproliferative agents</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7645744">US7645744</a></td><td class="patent-data-table-td patent-date-value">Jan 26, 2009</td><td class="patent-data-table-td patent-date-value">Jan 12, 2010</td><td class="patent-data-table-td ">The Trustees Of The University Of Pennsylvania</td><td class="patent-data-table-td ">Compositions and methods for siRNA inhibition of HIF-1 alpha</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7700280">US7700280</a></td><td class="patent-data-table-td patent-date-value">Dec 30, 2004</td><td class="patent-data-table-td patent-date-value">Apr 20, 2010</td><td class="patent-data-table-td ">The Penn State Research Foundation</td><td class="patent-data-table-td ">Methods for assessing cisplatin resistance, disease progression, and treatment efficacy in ovarian cancer</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7704688">US7704688</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Mar 22, 2004</td><td class="patent-data-table-td patent-date-value">Apr 27, 2010</td><td class="patent-data-table-td ">Plant Bioscience Limited</td><td class="patent-data-table-td ">Using RNA interference to prevent and treat viral diseases</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7732591">US7732591</a></td><td class="patent-data-table-td patent-date-value">Aug 8, 2006</td><td class="patent-data-table-td patent-date-value">Jun 8, 2010</td><td class="patent-data-table-td ">Medtronic, Inc.</td><td class="patent-data-table-td ">for treating a neurodegenerative disorder comprising the steps of surgically implanting a catheter; by brain infusion of small interfering RNA ; without impairment of cell endoplasmic reticulum, spontaneous motor activity, or locomotor activity</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7741469">US7741469</a></td><td class="patent-data-table-td patent-date-value">Apr 4, 2007</td><td class="patent-data-table-td patent-date-value">Jun 22, 2010</td><td class="patent-data-table-td ">University Of Iowa Research Foundation</td><td class="patent-data-table-td ">using small nucleic acid molecules to modulate the expression of genes encoding proteins involved in deafness caused by dominant negative mechanism of action</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7754697">US7754697</a></td><td class="patent-data-table-td patent-date-value">Aug 22, 2003</td><td class="patent-data-table-td patent-date-value">Jul 13, 2010</td><td class="patent-data-table-td ">Commonwealth Scientific And Industrial Research Organisation</td><td class="patent-data-table-td ">Control of gene expression</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7759320">US7759320</a></td><td class="patent-data-table-td patent-date-value">Mar 25, 2008</td><td class="patent-data-table-td patent-date-value">Jul 20, 2010</td><td class="patent-data-table-td ">Alnylam Pharmaceuticals, Inc.</td><td class="patent-data-table-td ">Double-stranded ribonucleic acid havng a sense strand and an antisense strand with specific sequences; mediating RNA interference; decreasing virus titre and sustaining platelet count and lymphocyte count</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7759478">US7759478</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Aug 29, 2003</td><td class="patent-data-table-td patent-date-value">Jul 20, 2010</td><td class="patent-data-table-td ">Rosetta Genomics Ltd.</td><td class="patent-data-table-td ">viral genomic address messenger (VGAM); selectively inhibit translation of known host target genes, and are believed to represent a novel pervasive viral attack mechanism</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7772200">US7772200</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jul 21, 2006</td><td class="patent-data-table-td patent-date-value">Aug 10, 2010</td><td class="patent-data-table-td ">Alnylam Pharmaceuticals, Inc.</td><td class="patent-data-table-td ">iRNA agents targeted to the Rho-A gene</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7776552">US7776552</a></td><td class="patent-data-table-td patent-date-value">Aug 3, 2006</td><td class="patent-data-table-td patent-date-value">Aug 17, 2010</td><td class="patent-data-table-td ">University Of Ottawa</td><td class="patent-data-table-td ">Drug screening; for inhibiting apoptosis in mammalian cells; zinc finger domains; consensus sequences; genetic engineering; for treating AIDS, alzheimer&#39;s disease, parkinson&#39;s disease, amyotrophic lateral sclerosis (ALS), retinitis pigmentosa, autoimmune diseases</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7790150">US7790150</a></td><td class="patent-data-table-td patent-date-value">Sep 4, 2004</td><td class="patent-data-table-td patent-date-value">Sep 7, 2010</td><td class="patent-data-table-td ">The General Hospital Corporation</td><td class="patent-data-table-td ">acetal resin or ketone polymer conjugate; drug covalently attached to polymer via succinamide linkages; the succinamidoester is oriented such that an ester is formed at the carrier side while the carboxyl forms amide bond with an amine-containing drug; glycine link; drug delivery; camptothecin CPT, Taxol</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7794931">US7794931</a></td><td class="patent-data-table-td patent-date-value">Sep 22, 2003</td><td class="patent-data-table-td patent-date-value">Sep 14, 2010</td><td class="patent-data-table-td ">Yale University</td><td class="patent-data-table-td ">mRNAs serve as metabolite-sensitive genetic switches for regulating gene expression; nonimmunogenic aptamer domains are swapped to create new recognition domain which causes genetic modulation with effector compounds; drug screening; gene therapy</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7819842">US7819842</a></td><td class="patent-data-table-td patent-date-value">Nov 21, 2006</td><td class="patent-data-table-td patent-date-value">Oct 26, 2010</td><td class="patent-data-table-td ">Medtronic, Inc.</td><td class="patent-data-table-td ">Chronically implantable guide tube for repeated intermittent delivery of materials or fluids to targeted tissue sites</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7820632">US7820632</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Feb 13, 2003</td><td class="patent-data-table-td patent-date-value">Oct 26, 2010</td><td class="patent-data-table-td ">City Of Hope</td><td class="patent-data-table-td ">For inhibiting expression of human immunodeficiency virus (HIV); for use in vitro/vivo</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7829694">US7829694</a></td><td class="patent-data-table-td patent-date-value">May 25, 2004</td><td class="patent-data-table-td patent-date-value">Nov 9, 2010</td><td class="patent-data-table-td ">Medtronic, Inc.</td><td class="patent-data-table-td ">surgerical insertion of catheter; supplying protein antagonistl Alzheimer&#39;s disease; Parkinson&#39;s disease; Huntington&#39;s disease; Spinocerebella atropy</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7838619">US7838619</a></td><td class="patent-data-table-td patent-date-value">Jan 14, 2003</td><td class="patent-data-table-td patent-date-value">Nov 23, 2010</td><td class="patent-data-table-td ">The General Hospital Corporation</td><td class="patent-data-table-td ">Biodegradable polyketal polymers and methods for their formation and use</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7851452">US7851452</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Mar 22, 2005</td><td class="patent-data-table-td patent-date-value">Dec 14, 2010</td><td class="patent-data-table-td ">The Trustees Of The University Of Pennsylvania</td><td class="patent-data-table-td ">Methods of use of bcl-6-derived nucleotides to induce apoptosis</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7872118">US7872118</a></td><td class="patent-data-table-td patent-date-value">Sep 6, 2007</td><td class="patent-data-table-td patent-date-value">Jan 18, 2011</td><td class="patent-data-table-td ">Opko Ophthalmics, Llc</td><td class="patent-data-table-td ">siRNA and methods of manufacture</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7884200">US7884200</a></td><td class="patent-data-table-td patent-date-value">Oct 7, 2009</td><td class="patent-data-table-td patent-date-value">Feb 8, 2011</td><td class="patent-data-table-td ">Aventis Pharmaceuticals Inc.</td><td class="patent-data-table-td ">For carrying a target gene specific insert into a cell in order to modulate the expression of a target gene; modified Lentivirus vector; used to down-regulate the expression of a particular target gene within a cell</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7888325">US7888325</a></td><td class="patent-data-table-td patent-date-value">Dec 12, 2006</td><td class="patent-data-table-td patent-date-value">Feb 15, 2011</td><td class="patent-data-table-td ">Medical College Of Georgia Research Institute, Inc.</td><td class="patent-data-table-td ">Composition and method for in vivo and in vitro attenuation of gene expression using double stranded RNA</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7893036">US7893036</a></td><td class="patent-data-table-td patent-date-value">Mar 26, 2008</td><td class="patent-data-table-td patent-date-value">Feb 22, 2011</td><td class="patent-data-table-td ">University Of Massachusetts</td><td class="patent-data-table-td ">In vivo production of small interfering RNAs that mediate gene silencing</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7902352">US7902352</a></td><td class="patent-data-table-td patent-date-value">Aug 9, 2006</td><td class="patent-data-table-td patent-date-value">Mar 8, 2011</td><td class="patent-data-table-td ">Medtronic, Inc.</td><td class="patent-data-table-td ">Isolated nucleic acid duplex for reducing huntington gene expression</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7939503">US7939503</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jun 4, 2004</td><td class="patent-data-table-td patent-date-value">May 10, 2011</td><td class="patent-data-table-td ">Agensys, Inc.</td><td class="patent-data-table-td ">Nucleotide sequences coding six transmembrane epithelial antigen of the prostate (STEAP)-1) for use as tool in diagnosis, prevention and treatment of cell proliferative disorders; RNA intereference</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7951898">US7951898</a></td><td class="patent-data-table-td patent-date-value">Nov 18, 2009</td><td class="patent-data-table-td patent-date-value">May 31, 2011</td><td class="patent-data-table-td ">The General Hospital Corporation</td><td class="patent-data-table-td ">Biodegradable polyketal polymers and methods for their formation and use</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7964714">US7964714</a></td><td class="patent-data-table-td patent-date-value">May 12, 2003</td><td class="patent-data-table-td patent-date-value">Jun 21, 2011</td><td class="patent-data-table-td ">Potomac Pharmaceuticals Inc.</td><td class="patent-data-table-td ">Gene expression suppression agents</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7973020">US7973020</a></td><td class="patent-data-table-td patent-date-value">May 27, 2010</td><td class="patent-data-table-td patent-date-value">Jul 5, 2011</td><td class="patent-data-table-td ">Alnylam Pharmaceuticals, Inc.</td><td class="patent-data-table-td ">Compositions and methods for inhibiting expression of a gene from the ebola virus</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7988668">US7988668</a></td><td class="patent-data-table-td patent-date-value">Nov 21, 2006</td><td class="patent-data-table-td patent-date-value">Aug 2, 2011</td><td class="patent-data-table-td ">Medtronic, Inc.</td><td class="patent-data-table-td ">Microsyringe for pre-packaged delivery of pharmaceuticals</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8012944">US8012944</a></td><td class="patent-data-table-td patent-date-value">Oct 28, 2004</td><td class="patent-data-table-td patent-date-value">Sep 6, 2011</td><td class="patent-data-table-td ">Pharmascience Inc.</td><td class="patent-data-table-td ">Method for treating cancer using IAP antisense oligomer and chemotherapeutic agent</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8030459">US8030459</a></td><td class="patent-data-table-td patent-date-value">Jul 18, 2003</td><td class="patent-data-table-td patent-date-value">Oct 4, 2011</td><td class="patent-data-table-td ">The General Hospital Corporation</td><td class="patent-data-table-td ">Oxime conjugates and methods for their formation and use</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8044032">US8044032</a></td><td class="patent-data-table-td patent-date-value">Oct 2, 2006</td><td class="patent-data-table-td patent-date-value">Oct 25, 2011</td><td class="patent-data-table-td ">Yissum Research Development Company Of The Hebrew University Of Jerusalem</td><td class="patent-data-table-td ">Selective killing of cells by activation of double-stranded RNA dependent protein kinase-PKR</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8048670">US8048670</a></td><td class="patent-data-table-td patent-date-value">Jan 15, 2004</td><td class="patent-data-table-td patent-date-value">Nov 1, 2011</td><td class="patent-data-table-td ">Commonwealth Scientific And Industrial Research Organisation</td><td class="patent-data-table-td ">Expression vector comprising promoter elements and which codes complimentary nucleotide sequences of preferential genes for use as therapeutic tool in targeted tissue/cell therapy; gene expression inhibition</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8053419">US8053419</a></td><td class="patent-data-table-td patent-date-value">Apr 8, 2004</td><td class="patent-data-table-td patent-date-value">Nov 8, 2011</td><td class="patent-data-table-td ">Commonwealth Scientific And Industrial Research Organisation</td><td class="patent-data-table-td ">Using RNA interference as therapeutic tool in treatment of metabolic disorders; tissue directed therapy</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8058251">US8058251</a></td><td class="patent-data-table-td patent-date-value">Oct 31, 2007</td><td class="patent-data-table-td patent-date-value">Nov 15, 2011</td><td class="patent-data-table-td ">Kaemmerer William F</td><td class="patent-data-table-td ">Devices, systems and methods for improving memory and/or cognitive function through brain delivery of siRNA</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8058255">US8058255</a></td><td class="patent-data-table-td patent-date-value">Jun 15, 2009</td><td class="patent-data-table-td patent-date-value">Nov 15, 2011</td><td class="patent-data-table-td ">Applied Biosystems, Llc</td><td class="patent-data-table-td ">Methods and compositions concerning siRNA&#39;s as mediators of RNA interference</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8067383">US8067383</a></td><td class="patent-data-table-td patent-date-value">Jan 17, 2003</td><td class="patent-data-table-td patent-date-value">Nov 29, 2011</td><td class="patent-data-table-td ">Commonwealth Scientific And Industrial Research Organisation</td><td class="patent-data-table-td ">Synthetic genes and genetic constructs comprising same I</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8076071">US8076071</a></td><td class="patent-data-table-td patent-date-value">Sep 14, 2010</td><td class="patent-data-table-td patent-date-value">Dec 13, 2011</td><td class="patent-data-table-td ">City Of Hope</td><td class="patent-data-table-td ">Methods for producing interfering RNA molecules in mammalian cells and therapeutic uses for such molecules</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8084599">US8084599</a></td><td class="patent-data-table-td patent-date-value">Mar 15, 2005</td><td class="patent-data-table-td patent-date-value">Dec 27, 2011</td><td class="patent-data-table-td ">City Of Hope</td><td class="patent-data-table-td ">Methods and compositions for the specific inhibition of gene expression by double-stranded RNA</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8101164">US8101164</a></td><td class="patent-data-table-td patent-date-value">Aug 19, 2010</td><td class="patent-data-table-td patent-date-value">Jan 24, 2012</td><td class="patent-data-table-td ">The General Hospital Corporation</td><td class="patent-data-table-td ">Dual phase drug release system</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8101584">US8101584</a></td><td class="patent-data-table-td patent-date-value">Mar 6, 2003</td><td class="patent-data-table-td patent-date-value">Jan 24, 2012</td><td class="patent-data-table-td ">Alnylam Pharmaceuticals, Inc.</td><td class="patent-data-table-td ">Method and medicament for inhibiting the expression of a given gene</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8101742">US8101742</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Oct 31, 2007</td><td class="patent-data-table-td patent-date-value">Jan 24, 2012</td><td class="patent-data-table-td ">Alnylam Pharmaceuticals, Inc.</td><td class="patent-data-table-td ">Method and medicament for inhibiting the expression of a given gene</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8106098">US8106098</a></td><td class="patent-data-table-td patent-date-value">Jan 9, 2007</td><td class="patent-data-table-td patent-date-value">Jan 31, 2012</td><td class="patent-data-table-td ">The General Hospital Corporation</td><td class="patent-data-table-td ">Protein conjugates with a water-soluble biocompatible, biodegradable polymer</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8106179">US8106179</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jul 2, 2004</td><td class="patent-data-table-td patent-date-value">Jan 31, 2012</td><td class="patent-data-table-td ">Commissariat A L&#39;energie Atomique</td><td class="patent-data-table-td ">Small interfering RNA specific to sub-units α, α′and β of the Kinase Protein ck2, and the applications of the same</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8106181">US8106181</a></td><td class="patent-data-table-td patent-date-value">Feb 22, 2010</td><td class="patent-data-table-td patent-date-value">Jan 31, 2012</td><td class="patent-data-table-td ">City Of Hope</td><td class="patent-data-table-td ">Methods for producing interfering RNA molecules in mammalian cells and therapeutic uses for such molecules</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8114851">US8114851</a></td><td class="patent-data-table-td patent-date-value">Oct 31, 2007</td><td class="patent-data-table-td patent-date-value">Feb 14, 2012</td><td class="patent-data-table-td ">Alnylam Pharmaceuticals, Inc.</td><td class="patent-data-table-td ">Method and medicament for inhibiting the expression of a given gene</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8114981">US8114981</a></td><td class="patent-data-table-td patent-date-value">Oct 31, 2007</td><td class="patent-data-table-td patent-date-value">Feb 14, 2012</td><td class="patent-data-table-td ">Alnylam Pharmaceuticals, Inc.</td><td class="patent-data-table-td ">Method and medicament for inhibiting the expression of a given gene</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8119608">US8119608</a></td><td class="patent-data-table-td patent-date-value">Mar 6, 2003</td><td class="patent-data-table-td patent-date-value">Feb 21, 2012</td><td class="patent-data-table-td ">Alnylam Pharmaceuticals, Inc.</td><td class="patent-data-table-td ">Specific inhibition of expression of target gene in mammals using short double stranded RNA; dsRNA less than 49 nucleotides in length, has nucleotide sequence complementary to part of target gene; treating cancer, Alzheimer&#39;s disease</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8119611">US8119611</a></td><td class="patent-data-table-td patent-date-value">Aug 27, 2009</td><td class="patent-data-table-td patent-date-value">Feb 21, 2012</td><td class="patent-data-table-td ">Medtronic, Inc.</td><td class="patent-data-table-td ">Treatment of neurodegenerative disease through intracranial delivery of SIRNA</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8148345">US8148345</a></td><td class="patent-data-table-td patent-date-value">Apr 13, 2009</td><td class="patent-data-table-td patent-date-value">Apr 3, 2012</td><td class="patent-data-table-td ">Georgia Health Sciences University Research Institute, Inc.</td><td class="patent-data-table-td ">Composition and method for in vivo and in vitro attenuation of gene expression using double stranded RNA</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8153776">US8153776</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Aug 20, 2007</td><td class="patent-data-table-td patent-date-value">Apr 10, 2012</td><td class="patent-data-table-td ">Cold Spring Harbor Laboratory</td><td class="patent-data-table-td ">Using targeted gene expression inhibition to treat and prevent viral infection</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8168774">US8168774</a></td><td class="patent-data-table-td patent-date-value">Sep 2, 2005</td><td class="patent-data-table-td patent-date-value">May 1, 2012</td><td class="patent-data-table-td ">Commonwealth Scientific And Industrial Research Organisation</td><td class="patent-data-table-td ">Control of gene expression</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8168776">US8168776</a></td><td class="patent-data-table-td patent-date-value">Mar 6, 2003</td><td class="patent-data-table-td patent-date-value">May 1, 2012</td><td class="patent-data-table-td ">Alnylam Pharmaceuticals, Inc.</td><td class="patent-data-table-td ">Comprises double-stranded RNA; extracted from HeLa cells</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8178662">US8178662</a></td><td class="patent-data-table-td patent-date-value">Jul 31, 2009</td><td class="patent-data-table-td patent-date-value">May 15, 2012</td><td class="patent-data-table-td ">Ryogen Llc</td><td class="patent-data-table-td ">Isolated AEBP1 genomic polynucleotide fragments from chromosome 7 and their uses</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8183217">US8183217</a></td><td class="patent-data-table-td patent-date-value">Jul 13, 2005</td><td class="patent-data-table-td patent-date-value">May 22, 2012</td><td class="patent-data-table-td ">Commonwealth Scientific And Industrial Research Organisation</td><td class="patent-data-table-td ">Methods and means for obtaining modified phenotypes</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8183362">US8183362</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Oct 31, 2007</td><td class="patent-data-table-td patent-date-value">May 22, 2012</td><td class="patent-data-table-td ">Alnylam Pharmaceuticals, Inc.</td><td class="patent-data-table-td ">Method and medicament for inhibiting the expression of a given gene</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8192935">US8192935</a></td><td class="patent-data-table-td patent-date-value">Feb 23, 2010</td><td class="patent-data-table-td patent-date-value">Jun 5, 2012</td><td class="patent-data-table-td ">The Penn State Research Foundation</td><td class="patent-data-table-td ">Methods for assessing cisplatin resistance, disease progression, and treatment efficacy in ovarian cancer as related to MetAP2 expression</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8202845">US8202845</a></td><td class="patent-data-table-td patent-date-value">Apr 16, 2003</td><td class="patent-data-table-td patent-date-value">Jun 19, 2012</td><td class="patent-data-table-td ">Acuity Pharmaceuticals, Inc.</td><td class="patent-data-table-td ">Means and methods for the specific modulation of target genes in the CNS and the eye and methods for their identification</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8202846">US8202846</a></td><td class="patent-data-table-td patent-date-value">Nov 23, 2004</td><td class="patent-data-table-td patent-date-value">Jun 19, 2012</td><td class="patent-data-table-td ">Cold Spring Harbor Laboratory</td><td class="patent-data-table-td ">Methods and compositions for RNA interference</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8202980">US8202980</a></td><td class="patent-data-table-td patent-date-value">Jul 2, 2003</td><td class="patent-data-table-td patent-date-value">Jun 19, 2012</td><td class="patent-data-table-td ">Alnylam Pharmaceuticals, Inc.</td><td class="patent-data-table-td ">Antisense oligonucleotide for use in prevention and treatment of viral and tumor disorders; gene expression inhibition; RNA interference</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8232260">US8232260</a></td><td class="patent-data-table-td patent-date-value">Feb 21, 2011</td><td class="patent-data-table-td patent-date-value">Jul 31, 2012</td><td class="patent-data-table-td ">University Of Massachusetts</td><td class="patent-data-table-td ">In vivo production of small interfering RNAs that mediate gene silencing</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8236775">US8236775</a></td><td class="patent-data-table-td patent-date-value">Dec 3, 2009</td><td class="patent-data-table-td patent-date-value">Aug 7, 2012</td><td class="patent-data-table-td ">The Trustees Of The University Of Pennsylvania</td><td class="patent-data-table-td ">Compositions and methods for siRNA inhibition of HIF-1 α</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8247383">US8247383</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Nov 19, 2004</td><td class="patent-data-table-td patent-date-value">Aug 21, 2012</td><td class="patent-data-table-td ">The General Hospital Corporation</td><td class="patent-data-table-td ">Drug-carrier complexes and methods of use thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8247427">US8247427</a></td><td class="patent-data-table-td patent-date-value">Jun 3, 2011</td><td class="patent-data-table-td patent-date-value">Aug 21, 2012</td><td class="patent-data-table-td ">The General Hospital Corporation</td><td class="patent-data-table-td ">Dual phase drug release system</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8258108">US8258108</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Oct 12, 2007</td><td class="patent-data-table-td patent-date-value">Sep 4, 2012</td><td class="patent-data-table-td ">Memorial Sloan-Kettering Cancer Center</td><td class="patent-data-table-td ">Gene expression inhibition by contacting cells which express kinase suppressor of Ras (KSR) with a nucleic acid which is complementary to a portion of the mRNA encoding KSR; antitumor, anticarcinogenic, antimetastasis and antiproliferative agents</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8258112">US8258112</a></td><td class="patent-data-table-td patent-date-value">Sep 10, 2009</td><td class="patent-data-table-td patent-date-value">Sep 4, 2012</td><td class="patent-data-table-td ">Medtronic, Inc</td><td class="patent-data-table-td ">Methods and sequences to suppress primate huntington gene Expression</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8258285">US8258285</a></td><td class="patent-data-table-td patent-date-value">Dec 17, 2004</td><td class="patent-data-table-td patent-date-value">Sep 4, 2012</td><td class="patent-data-table-td ">Plant Bioscience Limited</td><td class="patent-data-table-td ">RNA molecules and vectors for gene silencing</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8263569">US8263569</a></td><td class="patent-data-table-td patent-date-value">May 30, 2008</td><td class="patent-data-table-td patent-date-value">Sep 11, 2012</td><td class="patent-data-table-td ">Plant Biosciences Limited</td><td class="patent-data-table-td ">Gene silencing</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8283329">US8283329</a></td><td class="patent-data-table-td patent-date-value">Sep 28, 2007</td><td class="patent-data-table-td patent-date-value">Oct 9, 2012</td><td class="patent-data-table-td ">The Carnegie Institution Of Washington</td><td class="patent-data-table-td ">Genetic inhibition of double-stranded RNA</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8299235">US8299235</a></td><td class="patent-data-table-td patent-date-value">Dec 17, 2004</td><td class="patent-data-table-td patent-date-value">Oct 30, 2012</td><td class="patent-data-table-td ">Plant Bioscience Limited</td><td class="patent-data-table-td ">RNA molecules and vectors for gene silencing</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8313899">US8313899</a></td><td class="patent-data-table-td patent-date-value">Jul 31, 2009</td><td class="patent-data-table-td patent-date-value">Nov 20, 2012</td><td class="patent-data-table-td ">Ryogen Llc</td><td class="patent-data-table-td ">Isolated snare YKT6 genomic polynucleotide fragments from chomosome 7 and their uses</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8313900">US8313900</a></td><td class="patent-data-table-td patent-date-value">Jul 31, 2009</td><td class="patent-data-table-td patent-date-value">Nov 20, 2012</td><td class="patent-data-table-td ">Ryogen Llc</td><td class="patent-data-table-td ">Isolated DNA directed 50kD regulatory subunit (POLD2) genomic polynucleotide fragments from chomosome 7 and their uses</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8313901">US8313901</a></td><td class="patent-data-table-td patent-date-value">Dec 21, 2006</td><td class="patent-data-table-td patent-date-value">Nov 20, 2012</td><td class="patent-data-table-td ">Yale University</td><td class="patent-data-table-td ">Methods and compositions related to the modulation of riboswitches</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8323884">US8323884</a></td><td class="patent-data-table-td patent-date-value">Jul 31, 2009</td><td class="patent-data-table-td patent-date-value">Dec 4, 2012</td><td class="patent-data-table-td ">Ryogen Llc</td><td class="patent-data-table-td ">Isolated SNARE YKT6 genomic polynucleotide fragments from chromosome 7 and their uses</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8324367">US8324367</a></td><td class="patent-data-table-td patent-date-value">Oct 16, 2009</td><td class="patent-data-table-td patent-date-value">Dec 4, 2012</td><td class="patent-data-table-td ">Medtronic, Inc.</td><td class="patent-data-table-td ">Compositions and methods for making therapies delivered by viral vectors reversible for safety and allele-specificity</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8329463">US8329463</a></td><td class="patent-data-table-td patent-date-value">Jul 19, 2010</td><td class="patent-data-table-td patent-date-value">Dec 11, 2012</td><td class="patent-data-table-td ">MAX-PLANCK-Gesellschaft zur Förderung der Wissenschaften e.V.</td><td class="patent-data-table-td ">RNA interference mediating small RNA molecules</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8334374">US8334374</a></td><td class="patent-data-table-td patent-date-value">May 17, 2012</td><td class="patent-data-table-td patent-date-value">Dec 18, 2012</td><td class="patent-data-table-td ">Commonwealth Scientific And Industrial Research Organisation</td><td class="patent-data-table-td ">Methods and means for obtaining modified phenotypes</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8338091">US8338091</a></td><td class="patent-data-table-td patent-date-value">Feb 21, 2012</td><td class="patent-data-table-td patent-date-value">Dec 25, 2012</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and compositions for seamless cloning of nucleic acid molecules</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8349607">US8349607</a></td><td class="patent-data-table-td patent-date-value">Jul 23, 2009</td><td class="patent-data-table-td patent-date-value">Jan 8, 2013</td><td class="patent-data-table-td ">Plant Bioscience Limited</td><td class="patent-data-table-td ">Gene silencing</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8354390">US8354390</a></td><td class="patent-data-table-td patent-date-value">Apr 25, 2011</td><td class="patent-data-table-td patent-date-value">Jan 15, 2013</td><td class="patent-data-table-td ">Alnylam Pharmaceuticals, Inc.</td><td class="patent-data-table-td ">Compositions and methods for inhibiting expression of a gene from the ebola virus</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8361442">US8361442</a></td><td class="patent-data-table-td patent-date-value">Jul 28, 2011</td><td class="patent-data-table-td patent-date-value">Jan 29, 2013</td><td class="patent-data-table-td ">The General Hospital Corporation</td><td class="patent-data-table-td ">Oxime conjugates and methods for their formation and use</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8362231">US8362231</a></td><td class="patent-data-table-td patent-date-value">Jan 6, 2010</td><td class="patent-data-table-td patent-date-value">Jan 29, 2013</td><td class="patent-data-table-td ">Max-Planck-Gesellschaft zur Föderung der Wissenschaften E.V.</td><td class="patent-data-table-td ">RNA interference mediating small RNA molecules</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8372968">US8372968</a></td><td class="patent-data-table-td patent-date-value">Aug 7, 2009</td><td class="patent-data-table-td patent-date-value">Feb 12, 2013</td><td class="patent-data-table-td ">MAX-PLANCK-Gesellschaft zur Förderung der Wissenschaften e.V.</td><td class="patent-data-table-td ">RNA interference mediating small RNA molecules</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8383599">US8383599</a></td><td class="patent-data-table-td patent-date-value">May 16, 2008</td><td class="patent-data-table-td patent-date-value">Feb 26, 2013</td><td class="patent-data-table-td ">Cold Spring Harbor Laboratory</td><td class="patent-data-table-td ">Methods and compositions for RNA interference</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8383758">US8383758</a></td><td class="patent-data-table-td patent-date-value">Apr 26, 2011</td><td class="patent-data-table-td patent-date-value">Feb 26, 2013</td><td class="patent-data-table-td ">The General Hospital Corporation</td><td class="patent-data-table-td ">Biodegradable polyketal polymers and methods for their formation and use</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8389244">US8389244</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jul 31, 2003</td><td class="patent-data-table-td patent-date-value">Mar 5, 2013</td><td class="patent-data-table-td ">City Of Hope</td><td class="patent-data-table-td ">Methods and kits for synthesis of siRNA expression cassettes</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8394628">US8394628</a></td><td class="patent-data-table-td patent-date-value">Oct 4, 2010</td><td class="patent-data-table-td patent-date-value">Mar 12, 2013</td><td class="patent-data-table-td ">University Of Massachusetts</td><td class="patent-data-table-td ">RNA sequence-specific mediators of RNA interference</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8415319">US8415319</a></td><td class="patent-data-table-td patent-date-value">Sep 23, 2011</td><td class="patent-data-table-td patent-date-value">Apr 9, 2013</td><td class="patent-data-table-td ">Medtronic, Inc.</td><td class="patent-data-table-td ">Devices, systems and methods for improving memory and/or cognitive function through brain delivery of siRNA</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8420391">US8420391</a></td><td class="patent-data-table-td patent-date-value">Oct 4, 2010</td><td class="patent-data-table-td patent-date-value">Apr 16, 2013</td><td class="patent-data-table-td ">University Of Massachusetts</td><td class="patent-data-table-td ">RNA sequence-specific mediators of RNA interference</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8431119">US8431119</a></td><td class="patent-data-table-td patent-date-value">Aug 22, 2007</td><td class="patent-data-table-td patent-date-value">Apr 30, 2013</td><td class="patent-data-table-td ">Aventis Pharma S.A.</td><td class="patent-data-table-td ">Pharmaceutical compositions and utilization thereof particularly for the treatment of neurodegenerative diseases</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8431547">US8431547</a></td><td class="patent-data-table-td patent-date-value">Nov 7, 2011</td><td class="patent-data-table-td patent-date-value">Apr 30, 2013</td><td class="patent-data-table-td ">Commonwealth Scientific And Industrial Research Organisation</td><td class="patent-data-table-td ">Synthetic genes and genetic constructs</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8440810">US8440810</a></td><td class="patent-data-table-td patent-date-value">Feb 23, 2011</td><td class="patent-data-table-td patent-date-value">May 14, 2013</td><td class="patent-data-table-td ">Yale University</td><td class="patent-data-table-td ">Riboswitches, methods for their use, and compositions for use with riboswitches</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8445237">US8445237</a></td><td class="patent-data-table-td patent-date-value">Jul 12, 2010</td><td class="patent-data-table-td patent-date-value">May 21, 2013</td><td class="patent-data-table-td ">MAX-PLANCK-Gesellschaft zur Förderung der Wissenschaften e.V.</td><td class="patent-data-table-td ">RNA interference mediating small RNA molecules</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8470792">US8470792</a></td><td class="patent-data-table-td patent-date-value">Dec 4, 2009</td><td class="patent-data-table-td patent-date-value">Jun 25, 2013</td><td class="patent-data-table-td ">Opko Pharmaceuticals, Llc.</td><td class="patent-data-table-td ">Compositions and methods for selective inhibition of VEGF</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8491880">US8491880</a></td><td class="patent-data-table-td patent-date-value">Dec 10, 2009</td><td class="patent-data-table-td patent-date-value">Jul 23, 2013</td><td class="patent-data-table-td ">Mersana Therapeutics, Inc.</td><td class="patent-data-table-td ">Pharmaceutical formulations of biodegradable biocompatible camptothecin-polymer conjugates</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8524680">US8524680</a></td><td class="patent-data-table-td patent-date-value">Aug 31, 2009</td><td class="patent-data-table-td patent-date-value">Sep 3, 2013</td><td class="patent-data-table-td ">Applied Biosystems, Llc</td><td class="patent-data-table-td ">High potency siRNAS for reducing the expression of target genes</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8530438">US8530438</a></td><td class="patent-data-table-td patent-date-value">Mar 19, 2010</td><td class="patent-data-table-td patent-date-value">Sep 10, 2013</td><td class="patent-data-table-td ">University Of Massachusetts</td><td class="patent-data-table-td ">Vivo production of small interfering RNAs that mediate gene silencing</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8541384">US8541384</a></td><td class="patent-data-table-td patent-date-value">Jun 8, 2006</td><td class="patent-data-table-td patent-date-value">Sep 24, 2013</td><td class="patent-data-table-td ">The Trustees Of The University Of Pennsylvania</td><td class="patent-data-table-td ">Gene expression inhibition using small interfering RNAs which are specific for the vascular endothelial growth factor (VEGF) gene and the VEGF receptor genes Flt-1 and Flk-1/KDR; diabetic retinopathy, age related macular degeneration and many types of cancer</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8546345">US8546345</a></td><td class="patent-data-table-td patent-date-value">Jun 8, 2006</td><td class="patent-data-table-td patent-date-value">Oct 1, 2013</td><td class="patent-data-table-td ">The Trustees Of The University Of Pennsylvania</td><td class="patent-data-table-td ">Gene expression inhibition using small interfering RNAs which are specific for the vascular endothelial growth factor (VEGF) gene and the VEGF receptor genes Flt-1 and Flk-1/KDR; diabetic retinopathy, age related macular degeneration and many types of cancer</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8546419">US8546419</a></td><td class="patent-data-table-td patent-date-value">Jul 19, 2012</td><td class="patent-data-table-td patent-date-value">Oct 1, 2013</td><td class="patent-data-table-td ">The General Hospital Corporation</td><td class="patent-data-table-td ">Dual phase drug release system</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8552171">US8552171</a></td><td class="patent-data-table-td patent-date-value">Oct 4, 2010</td><td class="patent-data-table-td patent-date-value">Oct 8, 2013</td><td class="patent-data-table-td ">University Of Massachusetts</td><td class="patent-data-table-td ">RNA sequence-specific mediators of RNA interference</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8557785">US8557785</a></td><td class="patent-data-table-td patent-date-value">Mar 23, 2010</td><td class="patent-data-table-td patent-date-value">Oct 15, 2013</td><td class="patent-data-table-td ">University Of Massachusetts</td><td class="patent-data-table-td ">In vivo production of small interfering RNAS that mediate gene silencing</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8580754">US8580754</a></td><td class="patent-data-table-td patent-date-value">Oct 1, 2007</td><td class="patent-data-table-td patent-date-value">Nov 12, 2013</td><td class="patent-data-table-td ">Carnegie Institution Of Washington</td><td class="patent-data-table-td ">Genetic inhibition by double-stranded RNA</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8598332">US8598332</a></td><td class="patent-data-table-td patent-date-value">Apr 7, 1999</td><td class="patent-data-table-td patent-date-value">Dec 3, 2013</td><td class="patent-data-table-td ">Bayer Cropscience N.V.</td><td class="patent-data-table-td ">Methods and means for obtaining modified phenotypes</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8618069">US8618069</a></td><td class="patent-data-table-td patent-date-value">Oct 8, 2009</td><td class="patent-data-table-td patent-date-value">Dec 31, 2013</td><td class="patent-data-table-td ">Medtronic, Inc.</td><td class="patent-data-table-td ">Devices, systems and methods for improving memory and/or cognitive function through brain delivery of siRNA</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8632997">US8632997</a></td><td class="patent-data-table-td patent-date-value">Oct 4, 2010</td><td class="patent-data-table-td patent-date-value">Jan 21, 2014</td><td class="patent-data-table-td ">University Of Massachusetts</td><td class="patent-data-table-td ">RNA sequence-specific mediators of RNA interference</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8658356">US8658356</a></td><td class="patent-data-table-td patent-date-value">Jun 3, 2011</td><td class="patent-data-table-td patent-date-value">Feb 25, 2014</td><td class="patent-data-table-td ">City Of Hope</td><td class="patent-data-table-td ">Methods and compositions for the specific inhibition of gene expression by double-stranded RNA</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8664376">US8664376</a></td><td class="patent-data-table-td patent-date-value">Jun 8, 2012</td><td class="patent-data-table-td patent-date-value">Mar 4, 2014</td><td class="patent-data-table-td ">Nitto Denko Corporation</td><td class="patent-data-table-td ">Retinoid-liposomes for enhancing modulation of HSP47 expression</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8691786">US8691786</a></td><td class="patent-data-table-td patent-date-value">Jan 24, 2013</td><td class="patent-data-table-td patent-date-value">Apr 8, 2014</td><td class="patent-data-table-td ">City Of Hope</td><td class="patent-data-table-td ">Methods and compositions for the specific inhibition of gene expression by double-stranded RNA</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8710104">US8710104</a></td><td class="patent-data-table-td patent-date-value">Nov 6, 2009</td><td class="patent-data-table-td patent-date-value">Apr 29, 2014</td><td class="patent-data-table-td ">Triact Therapeutics, Inc.</td><td class="patent-data-table-td ">Catecholic butanes and use thereof for cancer therapy</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8710209">US8710209</a></td><td class="patent-data-table-td patent-date-value">Dec 8, 2010</td><td class="patent-data-table-td patent-date-value">Apr 29, 2014</td><td class="patent-data-table-td ">Nitto Denko Corporation</td><td class="patent-data-table-td ">Modulation of HSP47 expression</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8729037">US8729037</a></td><td class="patent-data-table-td patent-date-value">Oct 31, 2007</td><td class="patent-data-table-td patent-date-value">May 20, 2014</td><td class="patent-data-table-td ">Alnylam Pharmaceuticals, Inc.</td><td class="patent-data-table-td ">Method and medicament for inhibiting the expression of a given gene</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8735064">US8735064</a></td><td class="patent-data-table-td patent-date-value">Dec 23, 2008</td><td class="patent-data-table-td patent-date-value">May 27, 2014</td><td class="patent-data-table-td ">Bergenbio As</td><td class="patent-data-table-td ">Methods for creating and identifying functional RNA interference elements</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8741867">US8741867</a></td><td class="patent-data-table-td patent-date-value">Jun 8, 2012</td><td class="patent-data-table-td patent-date-value">Jun 3, 2014</td><td class="patent-data-table-td ">Nitto Denko Corporation</td><td class="patent-data-table-td ">Retinoid-liposomes for treating fibrosis</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8742092">US8742092</a></td><td class="patent-data-table-td patent-date-value">Oct 4, 2010</td><td class="patent-data-table-td patent-date-value">Jun 3, 2014</td><td class="patent-data-table-td ">University Of Massachusetts</td><td class="patent-data-table-td ">RNA sequence-specific mediators of RNA interference</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8759102">US8759102</a></td><td class="patent-data-table-td patent-date-value">Dec 17, 2004</td><td class="patent-data-table-td patent-date-value">Jun 24, 2014</td><td class="patent-data-table-td ">Plant Bioscience Limited</td><td class="patent-data-table-td ">Short RNA producing gene silencing in cells</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8765930">US8765930</a></td><td class="patent-data-table-td patent-date-value">Jun 4, 2010</td><td class="patent-data-table-td patent-date-value">Jul 1, 2014</td><td class="patent-data-table-td ">MAX-PLANCK-Gesellschaft zur Förderung der Wissenschaften e.V.</td><td class="patent-data-table-td ">RNA interference mediating small RNA molecules</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8778902">US8778902</a></td><td class="patent-data-table-td patent-date-value">Jul 13, 2010</td><td class="patent-data-table-td patent-date-value">Jul 15, 2014</td><td class="patent-data-table-td ">MAX-PLANCK-Gesellschaft zur Förderung der Wissenschaften e.V.</td><td class="patent-data-table-td ">RNA interference mediating small RNA molecules</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8779236">US8779236</a></td><td class="patent-data-table-td patent-date-value">Jul 23, 2009</td><td class="patent-data-table-td patent-date-value">Jul 15, 2014</td><td class="patent-data-table-td ">Plant Bioscience Limited</td><td class="patent-data-table-td ">Gene silencing</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20090228998">US20090228998</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Mar 30, 2009</td><td class="patent-data-table-td patent-date-value">Sep 10, 2009</td><td class="patent-data-table-td ">Academisch Ziekenhuis Leiden</td><td class="patent-data-table-td ">Induction of exon skipping in eukaryotic cells</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20100292099">US20100292099</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Aug 22, 2008</td><td class="patent-data-table-td patent-date-value">Nov 18, 2010</td><td class="patent-data-table-td ">Keren Pharmaceutical, Inc.</td><td class="patent-data-table-td ">Targeting of rna with external guide sequences</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20110231955">US20110231955</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jan 3, 2011</td><td class="patent-data-table-td patent-date-value">Sep 22, 2011</td><td class="patent-data-table-td ">Commonwealth Scientific And Industrial Research Organisation</td><td class="patent-data-table-td ">EFFICIENT GENE SILENCING IN PLANTS USING SHORT dsRNA SEQUENCES</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2145628A1?cl=en">EP2145628A1</a></td><td class="patent-data-table-td patent-date-value">Nov 26, 2003</td><td class="patent-data-table-td patent-date-value">Jan 20, 2010</td><td class="patent-data-table-td ">Medtronic, Inc.</td><td class="patent-data-table-td ">Treatment of neurodegenerative disease through intracranial delivery of short interfering RNA (siRNA)</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2169072A1?cl=en">EP2169072A1</a></td><td class="patent-data-table-td patent-date-value">Aug 23, 2005</td><td class="patent-data-table-td patent-date-value">Mar 31, 2010</td><td class="patent-data-table-td ">Alnylam Pharmaceuticals, Inc</td><td class="patent-data-table-td ">Multiple RNA polymerase III promoter expression constructs</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2270161A2?cl=en">EP2270161A2</a></td><td class="patent-data-table-td patent-date-value">Aug 23, 2005</td><td class="patent-data-table-td patent-date-value">Jan 5, 2011</td><td class="patent-data-table-td ">Alnylam Pharmaceuticals, Inc.</td><td class="patent-data-table-td ">Multiple RNA polymerase III promoter expression constructs</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2278004A1?cl=en">EP2278004A1</a></td><td class="patent-data-table-td patent-date-value">Feb 20, 2003</td><td class="patent-data-table-td patent-date-value">Jan 26, 2011</td><td class="patent-data-table-td ">Sirna Therapeutics, Inc.</td><td class="patent-data-table-td ">RNA interference mediated inhibition of gene expression using short interfering nucleic acid (siNA)</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2287305A1?cl=en">EP2287305A1</a></td><td class="patent-data-table-td patent-date-value">Feb 20, 2003</td><td class="patent-data-table-td patent-date-value">Feb 23, 2011</td><td class="patent-data-table-td ">Sirna Therapeutics, Inc.</td><td class="patent-data-table-td ">RNA interference mediated inhibition of gene expression using short interfering nucleic acid (siNA)</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2287306A1?cl=en">EP2287306A1</a></td><td class="patent-data-table-td patent-date-value">Feb 20, 2003</td><td class="patent-data-table-td patent-date-value">Feb 23, 2011</td><td class="patent-data-table-td ">Sirna Therapeutics, Inc.</td><td class="patent-data-table-td ">RNA interference mediated inhibition of gene expression using short interfering nucleic acid (siNA)</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2298896A1?cl=en">EP2298896A1</a></td><td class="patent-data-table-td patent-date-value">Jan 6, 2005</td><td class="patent-data-table-td patent-date-value">Mar 23, 2011</td><td class="patent-data-table-td ">The Board of Trustees of the University of Illionis</td><td class="patent-data-table-td ">Methods of inhibiting tumor cell proliferation with FOXM1 siRNA</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2371959A2?cl=en">EP2371959A2</a></td><td class="patent-data-table-td patent-date-value">May 24, 2005</td><td class="patent-data-table-td patent-date-value">Oct 5, 2011</td><td class="patent-data-table-td ">Medtronic, Inc.</td><td class="patent-data-table-td ">Treatment of neurodegenerative disease through intracranial delivery of siRNA</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2395112A1?cl=en">EP2395112A1</a></td><td class="patent-data-table-td patent-date-value">Dec 30, 2004</td><td class="patent-data-table-td patent-date-value">Dec 14, 2011</td><td class="patent-data-table-td ">The Penn State Research Foundation</td><td class="patent-data-table-td ">Methods for predicting and overcoming resistance to chemotherapy in ovarian cancer and for predicting colon cancer occurrence</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2005066371A2?cl=en">WO2005066371A2</a></td><td class="patent-data-table-td patent-date-value">Dec 30, 2004</td><td class="patent-data-table-td patent-date-value">Jul 21, 2005</td><td class="patent-data-table-td ">Penn State Res Found</td><td class="patent-data-table-td ">Methods for predicting and overcoming resistance to chemotherapy in ovarian cancer and for predicting colon cancer occurrence</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2008021149A2?cl=en">WO2008021149A2</a></td><td class="patent-data-table-td patent-date-value">Aug 8, 2007</td><td class="patent-data-table-td patent-date-value">Feb 21, 2008</td><td class="patent-data-table-td ">Medtronic Inc</td><td class="patent-data-table-td ">Compositions, devices and methods for treatment of huntington&#39;s disease through intracranial delivery of sirna</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2010107952A2?cl=en">WO2010107952A2</a></td><td class="patent-data-table-td patent-date-value">Mar 17, 2010</td><td class="patent-data-table-td patent-date-value">Sep 23, 2010</td><td class="patent-data-table-td ">Merck Sharp &amp; Dohme Corp.</td><td class="patent-data-table-td ">RNA INTERFERENCE MEDIATED INHIBITION OF CONNECTIVE TISSUE GROWTH FACTOR (CTGF) GENE EXPRESSION USING SHORT INTERFERING NUCLEIC ACID (siNA)</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2010107955A2?cl=en">WO2010107955A2</a></td><td class="patent-data-table-td patent-date-value">Mar 17, 2010</td><td class="patent-data-table-td patent-date-value">Sep 23, 2010</td><td class="patent-data-table-td ">Merck Sharp &amp; Dohme Corp.</td><td class="patent-data-table-td ">RNA INTERFERENCE MEDIATED INHIBITION OF BTB AND CNC HOMOLOGY 1, BASIC LEUCINE ZIPPER TRANSCRIPTION FACTOR 1 (BACH 1) GENE EXPRESSION USING SHORT INTERFERING NUCLEIC ACID (siNA) SEQUENCE LISTING</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2010107957A2?cl=en">WO2010107957A2</a></td><td class="patent-data-table-td patent-date-value">Mar 17, 2010</td><td class="patent-data-table-td patent-date-value">Sep 23, 2010</td><td class="patent-data-table-td ">Merck Sharp &amp; Dohme Corp.</td><td class="patent-data-table-td ">RNA INTERFERENCE MEDIATED INHIBITION OF GATA BINDING PROTEIN 3 (GATA3) GENE EXPRESSION USING SHORT INTERFERING NUCLEIC ACID (siNA)</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2010107958A1?cl=en">WO2010107958A1</a></td><td class="patent-data-table-td patent-date-value">Mar 17, 2010</td><td class="patent-data-table-td patent-date-value">Sep 23, 2010</td><td class="patent-data-table-td ">Merck Sharp &amp; Dohme Corp.</td><td class="patent-data-table-td ">RNA INTERFERENCE MEDIATED INHIBITION OF SIGNAL TRANSDUCER AND ACTIVATOR OF TRANSCRIPTION 6 (STAT6) GENE EXPRESSION USING SHORT INTERFERING NUCLEIC ACID (siNA)</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2010111464A1?cl=en">WO2010111464A1</a></td><td class="patent-data-table-td patent-date-value">Mar 25, 2010</td><td class="patent-data-table-td patent-date-value">Sep 30, 2010</td><td class="patent-data-table-td ">Merck Sharp &amp; Dohme Corp.</td><td class="patent-data-table-td ">RNA INTERFERENCE MEDIATED INHIBITION OF APOPTOSIS SIGNAL-REGULATING KINASE 1 (ASK1) GENE EXPRESSION USING SHORT INTERFERING NUCLEIC ACID (siNA)</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2010111468A2?cl=en">WO2010111468A2</a></td><td class="patent-data-table-td patent-date-value">Mar 25, 2010</td><td class="patent-data-table-td patent-date-value">Sep 30, 2010</td><td class="patent-data-table-td ">Merck Sharp &amp; Dohme Corp.</td><td class="patent-data-table-td ">RNA INTERFERENCE MEDIATED INHIBITION OF THE NERVE GROWTH FACTOR BETA CHAIN (NGFß) GENE EXPRESSION USING SHORT INTERFERING NUCLEIC ACID (SINA)</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2010111471A2?cl=en">WO2010111471A2</a></td><td class="patent-data-table-td patent-date-value">Mar 25, 2010</td><td class="patent-data-table-td patent-date-value">Sep 30, 2010</td><td class="patent-data-table-td ">Merck Sharp &amp; Dohme Corp.</td><td class="patent-data-table-td ">RNA INTERFERENCE MEDIATED INHIBITION OF SIGNAL TRANSDUCER AND ACTIVATOR OF TRANSCRIPTION 1 (STAT1) GENE EXPRESSION USING SHORT INTERFERING NUCLEIC ACID (siNA)</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2010111490A2?cl=en">WO2010111490A2</a></td><td class="patent-data-table-td patent-date-value">Mar 25, 2010</td><td class="patent-data-table-td patent-date-value">Sep 30, 2010</td><td class="patent-data-table-td ">Merck Sharp &amp; Dohme Corp.</td><td class="patent-data-table-td ">RNA INTERFERENCE MEDIATED INHIBITION OF THE THYMIC STROMAL LYMPHOPOIETIN (TSLP) GENE EXPRESSION USING SHORT INTERFERING NUCLEIC ACID (siNA)</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2010111497A2?cl=en">WO2010111497A2</a></td><td class="patent-data-table-td patent-date-value">Mar 25, 2010</td><td class="patent-data-table-td patent-date-value">Sep 30, 2010</td><td class="patent-data-table-td ">Merck Sharp &amp; Dohme Corp.</td><td class="patent-data-table-td ">RNA INTERFERENCE MEDIATED INHIBITION OF THE INTERCELLULAR ADHESION MOLECULE 1 (ICAM-1)GENE EXPRESSION USING SHORT INTERFERING NUCLEIC ACID (siNA)</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2011146938A1?cl=en">WO2011146938A1</a></td><td class="patent-data-table-td patent-date-value">May 23, 2011</td><td class="patent-data-table-td patent-date-value">Nov 24, 2011</td><td class="patent-data-table-td ">NanoOncology, Inc.</td><td class="patent-data-table-td ">Reagents and methods for treating cancer</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2011163436A1?cl=en">WO2011163436A1</a></td><td class="patent-data-table-td patent-date-value">Jun 23, 2011</td><td class="patent-data-table-td patent-date-value">Dec 29, 2011</td><td class="patent-data-table-td ">Quark Pharmaceuticals, Inc.</td><td class="patent-data-table-td ">Double stranded rna compounds to rhoa and use thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2012061443A2?cl=en">WO2012061443A2</a></td><td class="patent-data-table-td patent-date-value">Nov 1, 2011</td><td class="patent-data-table-td patent-date-value">May 10, 2012</td><td class="patent-data-table-td ">NanoOncology, Inc.</td><td class="patent-data-table-td ">Compositions of a peptide-based system for cell-specific targeting</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2012118910A2?cl=en">WO2012118910A2</a></td><td class="patent-data-table-td patent-date-value">Mar 1, 2012</td><td class="patent-data-table-td patent-date-value">Sep 7, 2012</td><td class="patent-data-table-td ">Quark Pharmaceuticals, Inc.</td><td class="patent-data-table-td ">Compositions and methods for treating lung disease and injury</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2012170957A2?cl=en">WO2012170957A2</a></td><td class="patent-data-table-td patent-date-value">Jun 8, 2012</td><td class="patent-data-table-td patent-date-value">Dec 13, 2012</td><td class="patent-data-table-td ">Nitto Denko Corporation</td><td class="patent-data-table-td ">Retinoid-liposomes for enhancing modulation of hsp47 expression</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2013106494A1?cl=en">WO2013106494A1</a></td><td class="patent-data-table-td patent-date-value">Jan 10, 2013</td><td class="patent-data-table-td patent-date-value">Jul 18, 2013</td><td class="patent-data-table-td ">Quark Pharmaceuticals, Inc.</td><td class="patent-data-table-td ">Combination therapy for treating hearing and balance disorders</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2014043292A1?cl=en">WO2014043292A1</a></td><td class="patent-data-table-td patent-date-value">Sep 12, 2013</td><td class="patent-data-table-td patent-date-value">Mar 20, 2014</td><td class="patent-data-table-td ">Quark Pharmaceuticals, Inc.</td><td class="patent-data-table-td ">Double-stranded oligonucleotide molecules to p53 and methods of use thereof</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="classifications"></a><div class="patent-section-header"><span class="patent-section-title">Classifications</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th"> </th><th class="patent-data-table-th"> </th></tr></thead><tr><td class="patent-data-table-td ">U.S. Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=tWZCBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S006160">435/6.16</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=tWZCBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S320100">435/320.1</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=tWZCBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S023100">536/23.1</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=tWZCBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S091100">435/91.1</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=tWZCBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S024100">536/24.1</a></span></td></tr><tr><td class="patent-data-table-td ">International Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=tWZCBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0015113000">C12N15/113</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=tWZCBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0015110000">C12N15/11</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=tWZCBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12Q0001680000">C12Q1/68</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=tWZCBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0015850000">C12N15/85</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=tWZCBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12P0019340000">C12P19/34</a></span></td></tr><tr><td class="patent-data-table-td ">Cooperative Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=tWZCBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12Q1/6834">C12Q1/6834</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=tWZCBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N2310/11">C12N2310/11</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=tWZCBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N2320/51">C12N2320/51</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=tWZCBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N15/1138">C12N15/1138</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=tWZCBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12P19/34">C12P19/34</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=tWZCBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N2330/30">C12N2330/30</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=tWZCBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12Q1/6839">C12Q1/6839</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=tWZCBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N2310/111">C12N2310/111</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=tWZCBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N2310/15">C12N2310/15</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=tWZCBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N15/111">C12N15/111</a></span></td></tr><tr><td class="patent-data-table-td ">European Classification</td><td class="patent-data-table-td "><span class="nested-value">C12Q1/68B12</span>, <span class="nested-value">C12Q1/68B10</span>, <span class="nested-value">C12N15/113E</span>, <span class="nested-value">C12N15/11M</span>, <span class="nested-value">C12P19/34</span></td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="legal-events"></a><div class="patent-section-header"><span class="patent-section-title">Legal Events</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Date</th><th class="patent-data-table-th">Code</th><th class="patent-data-table-th">Event</th><th class="patent-data-table-th">Description</th></tr></thead><tr><td class="patent-data-table-td patent-date-value">Oct 29, 2008</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">12</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Nov 14, 2006</td><td class="patent-data-table-td ">B1</td><td class="patent-data-table-td ">Reexamination certificate first reexamination</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">CLAIMS 27 AND 28 WERE PREVIOUSLY DISCLAIMED. CLAIMS 1-10, 16, 21-23 AND 26 ARE CANCELLED. CLAIMS 11-15, 17-19 AND 24 ARE DETERMINED TO BE PATENTABLE AS AMENDED. CLAIMS 20 AND 25, DEPENDENT ON AN AMENDED CLAIM, ARE DETERMINED TO BE PATENTABLE. NEW CLAIMS 29-39 ARE ADDED AND DETERMINED TO BE PATENTABLE.</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Oct 29, 2004</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">8</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Sep 7, 2004</td><td class="patent-data-table-td ">RR</td><td class="patent-data-table-td ">Request for reexamination filed</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20040727</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Aug 17, 2004</td><td class="patent-data-table-td ">DC</td><td class="patent-data-table-td ">Disclaimer filed</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20040603</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Oct 27, 2000</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">4</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Feb 12, 1996</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">REGENTS OF THE UNIVERSITY OF CALIFORNIA, THE, CALI</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNORS:NOONBERG, SARAH B.;HUNT, C. ANTHONY;REEL/FRAME:007818/0921;SIGNING DATES FROM 19950901 TO 19950905</span></div></td></tr></table><div class="patent-section-footer"></div></div><div class="modal-dialog" id="patent-images-lightbox"><div class="patent-lightbox-controls"><div class="patent-lightbox-rotate-controls"><div class="patent-lightbox-rotation-text">Rotate</div><div class="rotate-icon rotate-ccw-icon"></div><div class="rotate-icon rotate-cw-icon"></div></div><div class="patent-lightbox-index-counter"></div><a class="patent-lightbox-fullsize-link" target="_blank">Original Image</a><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_right.png" alt="Next page"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_left.png" alt="Previous page"width="21" height="21" /></div></div><div class="modal-dialog-content"><div class="patent-lightbox-image-holder"><div class="patent-lightbox-placeholder"></div></div></div></div><script>_OC_initPatentsAtb({image_not_available_html: " Image not available"});</script></div></div></div></td></tr></table><script>(function() {var href = window.location.href;if (href.indexOf('?') !== -1) {var parameters = href.split('?')[1].split('&');for (var i = 0; i < parameters.length; i++) {var param = parameters[i].split('=');if (param[0] == 'focus') {var elem = document.getElementById(param[1]);if (elem) {elem.focus();}}}}})();</script><script>_OC_addFlags({LockSrc:"/books/javascript/lock_8a2b04e7bf975d5171d8e4c0b6365c7a.js", Host:"http://www.google.com/", IsBooksRentalEnabled:1, IsWebstoreDisplayCaseEnabled:1, IsObfuscationEnabled:1, IsBrowsingHistoryEnabled:1, IsWebReaderSvgEnabled:0, IsGeoLayerEnabled:1, IsImageModeNotesEnabled:1, IsCopyMenuItemEnabled:1, IsGiftingEnabled:0, IsWebReaderUniversalPaginatorEnabled:0, IsOfflineBubbleEnabled:1, IsReaderEnabledForPlayRequests:1, IsFutureOnSaleVolumesEnabled:1, IsOfflineRestrictedCopyEnabled:1, IsBooksUnifiedLeftNavEnabled:1, IsRestrictedCopyEnabled:1, IsZipitFolderCollectionEnabled:1, IsEndOfSampleRecommendationsEnabled:1, IsRatingsOnBookcardsEnabled:1, IsAdsDisabled:0, IsIframePageDisplayEnabled:0, IsEmbeddedMediaEnabled:1, IsImageModeAnnotationsEnabled:1, IsMyLibraryGooglePlusEnabled:1, IsImagePageProviderEnabled:0, IsBookcardListPriceSmall:0, IsInternalUser:0, IsBooksShareButtonEnabled:0, IsPreOrdersEnabled:0, IsDisabledRandomBookshelves:0, WebstoreDisplayCasePosition:3});_OC_Run({"enable_p13n":false,"add_vol_to_collection_base_url":"http://www.google.com/patents?op=add\u0026sig=ACfU3U0uUf5-vOJb48DR_hkLeGM6xKd6ZQ\u0026id=tWZCBAABERAJ","remove_vol_from_collection_base_url":"http://www.google.com/patents?op=remove\u0026sig=ACfU3U2GNnMU4ViuuBM-zoMlrLNLc7qPkw\u0026id=tWZCBAABERAJ","logged_in":false,"p13n_save_user_settings_url":"http://www.google.com/patents?op=edit_user_settings\u0026sig=ACfU3U1e5wEkk70OIWMfzhix1Xyi4g16og","is_cobrand":false,"sign_in_url":"https://www.google.com/accounts/Login?service=\u0026continue=http://www.google.com/patents%3Fhl%3Den\u0026hl=en","is_play_enabled":true}, {"volume_id":"","is_ebook":true,"volumeresult":{"has_flowing_text":false,"has_scanned_text":true,"can_download_pdf":false,"can_download_epub":false,"is_pdf_drm_enabled":false,"is_epub_drm_enabled":false,"download_pdf_url":"http://www.google.com/patents/download/In_vivo_oligonucleotide_generator_and_me.pdf?id=tWZCBAABERAJ\u0026output=pdf\u0026sig=ACfU3U1RnaDv3iqyPDl3QTPdDswDcMTdhA"},"sample_url":"http://www.google.com/patents/reader?id=tWZCBAABERAJ\u0026printsec=frontcover\u0026output=reader\u0026source=gbs_atb_hover","is_browsable":true,"is_public_domain":true}, {});</script><div id="footer_table" style="font-size:83%;text-align:center;position:relative;top:20px;height:4.5em;margin-top:2em"><div style="margin-bottom:8px"><a href=http://www.google.com/><nobr>Google&nbsp;Home</nobr></a> - <a href=//www.google.com/patents/sitemap/><nobr>Sitemap</nobr></a> - <a href=http://www.google.com/googlebooks/uspto.html><nobr>USPTO Bulk Downloads</nobr></a> - <a href=/intl/en/privacy/><nobr>Privacy Policy</nobr></a> - <a href=/intl/en/policies/terms/><nobr>Terms of Service</nobr></a> - <a href=https://support.google.com/faqs/answer/2539193?hl=en><nobr>About Google Patents</nobr></a> - <a href="http://www.google.com/tools/feedback/intl/en/error.html" onclick="try{_OC_startFeedback({productId: '72792',locale: 'en'});return false;}catch(e){}"><nobr>Send Feedback</nobr></a></div><span>Data provided by IFI CLAIMS Patent Services</span><br><span >&copy;2012 Google</span></div> <script type="text/javascript">var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));</script><script type="text/javascript">var pageTracker = _gat._getTracker("UA-27188110-1");pageTracker._setCookiePath("/patents/");pageTracker._trackPageview();</script> </body></html>