<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN""http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html><head><title>Patent US5665540 - Multicolor in situ hybridization methods for genetic testing - Google Patents</title><script>(function(){(function(){function e(a){this.t={};this.tick=function(a,c,b){var d=void 0!=b?b:(new Date).getTime();this.t[a]=[d,c];if(void 0==b)try{window.console.timeStamp("CSI/"+a)}catch(e){}};this.tick("start",null,a)}var a;window.performance&&(a=window.performance.timing);var f=a?new e(a.responseStart):new e;window.jstiming={Timer:e,load:f};if(a){var c=a.navigationStart,d=a.responseStart;0<c&&d>=c&&(window.jstiming.srt=d-c)}if(a){var b=window.jstiming.load;0<c&&d>=c&&(b.tick("_wtsrt",void 0,c),b.tick("wtsrt_",
"_wtsrt",d),b.tick("tbsd_","wtsrt_"))}try{a=null,window.chrome&&window.chrome.csi&&(a=Math.floor(window.chrome.csi().pageT),b&&0<c&&(b.tick("_tbnd",void 0,window.chrome.csi().startE),b.tick("tbnd_","_tbnd",c))),null==a&&window.gtbExternal&&(a=window.gtbExternal.pageT()),null==a&&window.external&&(a=window.external.pageT,b&&0<c&&(b.tick("_tbnd",void 0,window.external.startE),b.tick("tbnd_","_tbnd",c))),a&&(window.jstiming.pt=a)}catch(g){}})();})();
</script><link rel="stylesheet" href="/patents/css/_8a2b04e7bf975d5171d8e4c0b6365c7a/kl_intl_patents_bundle.css" type="text/css" /><script src="/books/javascript/atb_8a2b04e7bf975d5171d8e4c0b6365c7a__en.js"></script><script>function googleTranslateElementInit() {new google.translate.TranslateElement({pageLanguage: "en",gaTrack: true,gaId: "UA-27188110-1",multilanguagePage: true});}</script><script src="//translate.google.com/translate_a/element.js?cb=googleTranslateElementInit"></script><meta name="DC.type" content="Patent"><meta name="DC.title" content="Multicolor in situ hybridization methods for genetic testing"><meta name="DC.contributor" content="Roger V. Lebo" scheme="inventor"><meta name="DC.contributor" content="The Regents Of The University Of California" scheme="assignee"><meta name="DC.date" content="1993-11-29" scheme="dateSubmitted"><meta name="DC.description" content="This invention relates to novel methods of optimally analyzing commonly obtained prenatal cell samples by in situ hybridization. In addition, this method diagnoses gene deletion and gene multiplication using multicolor in situ hybridization. A method is also provided to use multicolor in situ hybridization to identify chromosomal haplotypes co-segregating with disease-related genetic alterations and with normal genes. This haplotype in situ protocol simplifies haplotype segregation analysis in pedigrees."><meta name="DC.date" content="1997-9-9" scheme="issued"><meta name="DC.relation" content="US:4888278" scheme="references"><meta name="citation_reference" content="Ballabio et al. PNAS 84:4519 4523, 1987."><meta name="citation_reference" content="Ballabio et al. PNAS 84:4519-4523, 1987."><meta name="citation_reference" content="Ballabio et al., &quot;Isolation and characterization of a steroid sulfatase cDNA clone: Genomic deletions in patients with X-chromosome-linked ichthyosis&quot;, Proc. Natl. Acad. Sci., 84:4519-4523 (Jul. 1987)."><meta name="citation_reference" content="Ballabio et al., Isolation and characterization of a steroid sulfatase cDNA clone: Genomic deletions in patients with X chromosome linked ichthyosis , Proc. Natl. Acad. Sci., 84:4519 4523 (Jul. 1987)."><meta name="citation_reference" content="Cong et al., &quot;Assignment of Human Tracheobronchial Mucin Gene(s) to 11p15 and a Tracheobronchial Mucin-related Sequence to Chromosome 13&quot; Hum. Genet. 86:167-172 (1990)."><meta name="citation_reference" content="Cong et al., Assignment of Human Tracheobronchial Mucin Gene(s) to 11p15 and a Tracheobronchial Mucin related Sequence to Chromosome 13 Hum. Genet. 86:167 172 (1990)."><meta name="citation_reference" content="Greenberg et al., &quot;Molecular Analysis of the Smith-Magenis Syndrome: A Possible Contiguous-Gene Syndrome Associated with del(17) (pll.2)&quot; Am. J. Hum. Genet. 49:1207-1218 (1991)."><meta name="citation_reference" content="Greenberg et al., &quot;Molecular Analysis of the Smith-Magenis Syndrome: A Possible Contiguous-Gene Syndrome Associated with del(17)(pll.2)&quot; Am. J. Hum. Genet. 49:1207-1218 (1991)."><meta name="citation_reference" content="Greenberg et al., Molecular Analysis of the Smith Magenis Syndrome: A Possible Contiguous Gene Syndrome Associated with del(17) (pll.2) Am. J. Hum. Genet. 49:1207 1218 (1991)."><meta name="citation_reference" content="Greenberg et al., Molecular Analysis of the Smith Magenis Syndrome: A Possible Contiguous Gene Syndrome Associated with del(17)(pll.2) Am. J. Hum. Genet. 49:1207 1218 (1991)."><meta name="citation_reference" content="Grundy et al., &quot;The Polymorphic FcT Receptor II Gene Maps to Human Chromosome 1q&quot; Immunogenetics 29:331-339 (1989)."><meta name="citation_reference" content="Grundy et al., The Polymorphic FcT Receptor II Gene Maps to Human Chromosome 1q Immunogenetics 29:331 339 (1989)."><meta name="citation_reference" content="Grundy, et al.., Immunogenetics, 29:331 339 (1989)."><meta name="citation_reference" content="Grundy, et al.., Immunogenetics, 29:331-339 (1989)."><meta name="citation_reference" content="Harper et al., &quot;Localization of Single Copy DNA Sequences on G-Banded Human Chromosomes by in situ Hybridization&quot; Chromosoma 83:431-439 (1981)."><meta name="citation_reference" content="Harper et al., Localization of Single Copy DNA Sequences on G Banded Human Chromosomes by in situ Hybridization Chromosoma 83:431 439 (1981)."><meta name="citation_reference" content="Harper, et al., Chromosoma, 83:431 439 (1981)."><meta name="citation_reference" content="Harper, et al., Chromosoma, 83:431-439 (1981)."><meta name="citation_reference" content="Koenig et al. Cell 50:509 517, 1987."><meta name="citation_reference" content="Koenig et al. Cell 50:509-517, 1987."><meta name="citation_reference" content="Koenig et al., &quot;Complete Cloning of the Duchenne Muscular Dystrophy (DMD) cDNA and Preliminary Genomic Organization of the DMA Gene in Normal and Affected Individuals&quot;, Cell 50:509-517 (Jul. 31, 1987)."><meta name="citation_reference" content="Koenig et al., Complete Cloning of the Duchenne Muscular Dystrophy (DMD) cDNA and Preliminary Genomic Organization of the DMA Gene in Normal and Affected Individuals , Cell 50:509 517 (Jul. 31, 1987)."><meta name="citation_reference" content="Kwiatkowski et al. Am. J. Human Genet. 46:559 567, 1990."><meta name="citation_reference" content="Kwiatkowski et al. Am. J. Human Genet. 46:559-567, 1990."><meta name="citation_reference" content="Kwiatkowski et al., &quot;Identification of the functional prolifin gene, its localization to chromosome subband 17p13.3, and demonstration of its deletion in some patients with Miller-Dieker syndrome&quot;, Am. J. Hum. Genet. 46:559-567 (1990)."><meta name="citation_reference" content="Kwiatkowski et al., Identification of the functional prolifin gene, its localization to chromosome subband 17p13.3, and demonstration of its deletion in some patients with Miller Dieker syndrome , Am. J. Hum. Genet. 46:559 567 (1990)."><meta name="citation_reference" content="Langer Safer et al., Immunological Method for Mapping Genes on Drosophila Polytene Chromosomes Proc. Natl. Acad. Sci. USA 79:4381 4385 (Jul. 1982)."><meta name="citation_reference" content="Langer Safer, et al., Proc. Natl. Acad. Sci. USA, 79:4381 4385 (Jul., 1982)."><meta name="citation_reference" content="Langer-Safer et al., &quot;Immunological Method for Mapping Genes on Drosophila Polytene Chromosomes&quot; Proc. Natl. Acad. Sci. USA 79:4381-4385 (Jul. 1982)."><meta name="citation_reference" content="Langer-Safer, et al., Proc. Natl. Acad. Sci. USA, 79:4381-4385 (Jul., 1982)."><meta name="citation_reference" content="Lathrop et al., &quot;Strategies for Multilocus Linkage Analysis in Humans&quot; Proc. Natl. Acad. Sci. USA 81:3443-3446 (Jun. 1984)."><meta name="citation_reference" content="Lathrop et al., Strategies for Multilocus Linkage Analysis in Humans Proc. Natl. Acad. Sci. USA 81:3443 3446 (Jun. 1984)."><meta name="citation_reference" content="Lathrop, et al., Proc. Natl. Acad. Sci. USA, 81:3443 3446 (Jun., 1984)."><meta name="citation_reference" content="Lathrop, et al., Proc. Natl. Acad. Sci. USA, 81:3443-3446 (Jun., 1984)."><meta name="citation_reference" content="Lebo et al , &quot;Chromosome 1 Charcot-Marie-Tooth Disease (CMT1B) Locus in the FcT Receptor Gene Region&quot; Hum. Genet. 88:1-12 (1991)."><meta name="citation_reference" content="Lebo et al , Chromosome 1 Charcot Marie Tooth Disease (CMT1B) Locus in the FcT Receptor Gene Region Hum. Genet. 88:1 12 (1991)."><meta name="citation_reference" content="Lebo et al., &quot;Defining DNA Diagnostic Tests Appropriate for Standard Clinical Care&quot; Am. J. Hum. Genet. 47:583-590 (1990)."><meta name="citation_reference" content="Lebo et al., &quot;Multicolor Fluorescence in situ Hybridization and Pulsed Field Electrophoresis Dissect CMT1B Gene Region&quot; Genetics Springer-Verlag pp. 1-8 (1991)."><meta name="citation_reference" content="Lebo et al., &quot;Multicolor Fluorescence in situ Hybridization and Pulsed Field Electrophoresis Dissect CMT1B Gene Region&quot; Hum. Genet. 88:13-20 (1991)."><meta name="citation_reference" content="Lebo et al., &quot;Multicolor in situ Hybridization Prenatally Diagnoses X-Linked Ichthyosis&quot; Am. CytoGenet. Conf. Abstract (1991)."><meta name="citation_reference" content="Lebo et al., &quot;Multicolor in-situ Hybridization and Linkage Analysis Order Charcot-Marie-Toot Type 1 (CMTIA) Gene-Region Markers&quot; Am. J. Hum. Genet. 50:42-55 (1992)."><meta name="citation_reference" content="Lebo et al., &quot;Rare McArdle Disease Locus Polymorphic site on 11q13 contains CpG Sequence&quot; Hum. Genet. 86:17-24 (1990)."><meta name="citation_reference" content="Lebo et al., Defining DNA Diagnostic Tests Appropriate for Standard Clinical Care Am. J. Hum. Genet. 47:583 590 (1990)."><meta name="citation_reference" content="Lebo et al., Multicolor Fluorescence in situ Hybridization and Pulsed Field Electrophoresis Dissect CMT1B Gene Region Genetics Springer Verlag pp. 1 8 (1991)."><meta name="citation_reference" content="Lebo et al., Multicolor Fluorescence in situ Hybridization and Pulsed Field Electrophoresis Dissect CMT1B Gene Region Hum. Genet. 88:13 20 (1991)."><meta name="citation_reference" content="Lebo et al., Multicolor in situ Hybridization and Linkage Analysis Order Charcot Marie Toot Type 1 (CMTIA) Gene Region Markers Am. J. Hum. Genet. 50:42 55 (1992)."><meta name="citation_reference" content="Lebo et al., Multicolor in situ Hybridization Prenatally Diagnoses X Linked Ichthyosis Am. CytoGenet. Conf. Abstract (1991)."><meta name="citation_reference" content="Lebo et al., Rare McArdle Disease Locus Polymorphic site on 11q13 contains CpG Sequence Hum. Genet. 86:17 24 (1990)."><meta name="citation_reference" content="Lebo, et al., Am. CytoGenet. Conf., Abstract (1991)."><meta name="citation_reference" content="Lebo, et al., Am. J. Hum. Genet., 47:583 590 (1990)."><meta name="citation_reference" content="Lebo, et al., Am. J. Hum. Genet., 47:583-590 (1990)."><meta name="citation_reference" content="Lebo, et al., Am. J. Hum. Genet., 50:42 55 (1992)."><meta name="citation_reference" content="Lebo, et al., Am. J. Hum. Genet., 50:42-55 (1992)."><meta name="citation_reference" content="Lebo, et al., Hum. Genet., 86:17 24 (1990)."><meta name="citation_reference" content="Lebo, et al., Hum. Genet., 86:17-24 (1990)."><meta name="citation_reference" content="Lebo, et al., Hum. Genet., 88:13 (1991)."><meta name="citation_reference" content="Lebo, et al., Hum. Genet., 88:13 20 (1991)."><meta name="citation_reference" content="Lebo, et al., Hum. Genet., 88:13-20 (1991)."><meta name="citation_reference" content="Li et al., &quot;Characterization of a low copy repetitive element S232 Involved in the Generation of Frequent Deletions of the Distal Short Arm of the Human X Chromosome&quot; Nuc. Acids Res. 20:1117-1122 (1992)."><meta name="citation_reference" content="Li et al., Characterization of a low copy repetitive element S232 Involved in the Generation of Frequent Deletions of the Distal Short Arm of the Human X Chromosome Nuc. Acids Res. 20:1117 1122 (1992)."><meta name="citation_reference" content="Lichter et al., &quot;Is non-isotopic in situ hybridization finally coming of age&quot; Nature 345:93-94 (1990)."><meta name="citation_reference" content="Lichter et al., Is non isotopic in situ hybridization finally coming of age Nature 345:93 94 (1990)."><meta name="citation_reference" content="Lupinski, et al., Cell, 68:219 232 (1991)."><meta name="citation_reference" content="Lupinski, et al., Cell, 68:219-232 (1991)."><meta name="citation_reference" content="Lupski et al. &quot;DNA Duplication Associated with Charcot-Marie-Tooth Disease Type 1A&quot;0 Cell 68:219-232 (Jul. 26, 1991)."><meta name="citation_reference" content="Lupski et al. DNA Duplication Associated with Charcot Marie Tooth Disease Type 1A 0 Cell 68:219 232 (Jul. 26, 1991)."><meta name="citation_reference" content="Lux et al., &quot;Hereditary spherocytosis associated with deletion of human erythrocyte ankyrin gene on chromosome&quot; Nature 345:736-739 (1990)."><meta name="citation_reference" content="Lux et al., Hereditary spherocytosis associated with deletion of human erythrocyte ankyrin gene on chromosome Nature 345:736 739 (1990)."><meta name="citation_reference" content="Lux, et al., Nature, 345:736 739 (1990)."><meta name="citation_reference" content="Lux, et al., Nature, 345:736-739 (1990)."><meta name="citation_reference" content="Mathews Analy Biochem 169:1 25 (1988)."><meta name="citation_reference" content="Mathews Analy Biochem 169:1-25 (1988)."><meta name="citation_reference" content="Nederlof et al., &quot;Three-Color Fluorescence in situ Hybridization for the Simultaneous Detection of Multiple Nucleic Acid Sequences&quot; Cytometry 10:20-27 (1989)."><meta name="citation_reference" content="Nederlof et al., Three Color Fluorescence in situ Hybridization for the Simultaneous Detection of Multiple Nucleic Acid Sequences Cytometry 10:20 27 (1989)."><meta name="citation_reference" content="Nederlof, et al., Cytometry, 10:20 27 (1989)."><meta name="citation_reference" content="Nederlof, et al., Cytometry, 10:20-27 (1989)."><meta name="citation_reference" content="Orkin et al. Nature 286:538 540, 1980."><meta name="citation_reference" content="Orkin et al. Nature 286:538-540, 1980."><meta name="citation_reference" content="Orkin et al., &quot;Linage of β-thalassaemia Mutations &amp; β-globin gene polymorphisms with DNA Polymorphisms in Human β-globin Gene Cluster&quot; Nature 296:627-631 (Apr. 15, 1982)."><meta name="citation_reference" content="Orkin et al., &quot;Partial deletion of the α-globin structural gene in human α-thalassaemia&quot;, Nature, 286:538-540 (Jul. 31, 1980)."><meta name="citation_reference" content="Orkin et al., Linage of thalassaemia Mutations &amp; globin gene polymorphisms with DNA Polymorphisms in Human globin Gene Cluster Nature 296:627 631 (Apr. 15, 1982)."><meta name="citation_reference" content="Orkin et al., Partial deletion of the globin structural gene in human thalassaemia , Nature, 286:538 540 (Jul. 31, 1980)."><meta name="citation_reference" content="Orkin et el., &quot;Linage of β-thalassaemia Mutations and βglobin gene polymorphisms with DNA Polymorphisms in Human β-globin Gene Cluster&quot; Nature 296:627-631 (Apr. 15, 1982)."><meta name="citation_reference" content="Orkin et el., Linage of thalassaemia Mutations and globin gene polymorphisms with DNA Polymorphisms in Human globin Gene Cluster Nature 296:627 631 (Apr. 15, 1982)."><meta name="citation_reference" content="Patel et al. Am. J. Human Genet. 46:801 809, 1990."><meta name="citation_reference" content="Patel et al. Am. J. Human Genet. 46:801-809, 1990."><meta name="citation_reference" content="Patel, et al., &quot;Genetic Mapping of Autosomal Dominant Charcot-Marie-Tooth Disease in a Large French-Acadian Kindred: Identification of New Linked Markers on Chromosome 17&quot;, Am J Hum Genet., 46:801-809, 1990."><meta name="citation_reference" content="Patel, et al., Genetic Mapping of Autosomal Dominant Charcot Marie Tooth Disease in a Large French Acadian Kindred: Identification of New Linked Markers on Chromosome 17 , Am J Hum Genet., 46:801 809, 1990."><meta name="citation_reference" content="Raeymaekers et al. &quot;Duplication in Chromosome 17p11.2 in Charcot-Marie-Tooth Neuropathy Type 1a (CMT 1a)&quot; Neuromuscular Dis. 1(2):93-97 (1991)."><meta name="citation_reference" content="Raeymaekers et al. Duplication in Chromosome 17p11.2 in Charcot Marie Tooth Neuropathy Type 1a (CMT 1a) Neuromuscular Dis. 1(2):93 97 (1991)."><meta name="citation_reference" content="Raeymaekers, et al., Neuromuscular Dis., 1(2):93 97 (1991)."><meta name="citation_reference" content="Raeymaekers, et al., Neuromuscular Dis., 1(2):93-97 (1991)."><meta name="citation_reference" content="Trask et al., &quot;Mapping of Human Chromosome Xq28 by Two Color Fluorescence in situ Hybridization of DNA Sequences to Interphase Cell Nuclei&quot; Am. J. Hum. Genet. 48:1-15 (1991)."><meta name="citation_reference" content="Trask et al., &quot;The Proximity of DNA Sequences in Interphase Cell Nuclei is Correlated to Genomic Distance and Permits Ordering of Cosmids Spanning 250 Kilobase Pairs&quot; Genomics 5:710-717 (1989)."><meta name="citation_reference" content="Trask et al., Mapping of Human Chromosome Xq28 by Two Color Fluorescence in situ Hybridization of DNA Sequences to Interphase Cell Nuclei Am. J. Hum. Genet. 48:1 15 (1991)."><meta name="citation_reference" content="Trask et al., The Proximity of DNA Sequences in Interphase Cell Nuclei is Correlated to Genomic Distance and Permits Ordering of Cosmids Spanning 250 Kilobase Pairs Genomics 5:710 717 (1989)."><meta name="citation_reference" content="Trask, et al., Am. J. Hum. Genet., 48:1 15 (1991)."><meta name="citation_reference" content="Trask, et al., Am. J. Hum. Genet., 48:1-15 (1991)."><meta name="citation_reference" content="Trask, et al., Genomics, 5:710 717 (1989)."><meta name="citation_reference" content="Trask, et al., Genomics, 5:710-717 (1989)."><meta name="citation_reference" content="Tsipouras et al. &quot;A Centromere-Based Linkage Group on Long Arm of Human Chromosome 17&quot; Cytogenet Cell Genet 47:109-110 (1988)."><meta name="citation_reference" content="Tsipouras et al. A Centromere Based Linkage Group on Long Arm of Human Chromosome 17 Cytogenet Cell Genet 47:109 110 (1988)."><meta name="citation_reference" content="Tsipouras et al., &quot;A Centromere-Based Linkage Group on the Long Arm of Human Chromosome 17&quot; Cytogenet Cell Genet 47:109-110 (1988)."><meta name="citation_reference" content="Tsipouras et al., A Centromere Based Linkage Group on the Long Arm of Human Chromosome 17 Cytogenet Cell Genet 47:109 110 (1988)."><meta name="citation_reference" content="Weissenbach et al., &quot;A Second-Generation Linkage Map of the Human Genome&quot; Nature 359:794-801 (Oct. 29, 1992)."><meta name="citation_reference" content="Weissenbach et al., A Second Generation Linkage Map of the Human Genome Nature 359:794 801 (Oct. 29, 1992)."><meta name="citation_reference" content="Wijmenga et al., &quot;Mapping of Facioscapulohumeral Muscular Dystrophy Gene to Chromosome 4q35-qter by Multipoint Linkage Analysis and in situ Hybridization&quot; Genomics 9:570-575 (1991)."><meta name="citation_reference" content="Wijmenga et al., Mapping of Facioscapulohumeral Muscular Dystrophy Gene to Chromosome 4q35 qter by Multipoint Linkage Analysis and in situ Hybridization Genomics 9:570 575 (1991)."><meta name="citation_reference" content="Wiljmenga, et al, Genomics, 9:570 575 (1991)."><meta name="citation_reference" content="Wiljmenga, et al, Genomics, 9:570-575 (1991)."><meta name="citation_patent_number" content="US:5665540"><meta name="citation_patent_application_number" content="US:08/159,070"><link rel="canonical" href="http://www.google.com/patents/US5665540"/><meta property="og:url" content="http://www.google.com/patents/US5665540"/><meta name="title" content="Patent US5665540 - Multicolor in situ hybridization methods for genetic testing"/><meta name="description" content="This invention relates to novel methods of optimally analyzing commonly obtained prenatal cell samples by in situ hybridization. In addition, this method diagnoses gene deletion and gene multiplication using multicolor in situ hybridization. A method is also provided to use multicolor in situ hybridization to identify chromosomal haplotypes co-segregating with disease-related genetic alterations and with normal genes. This haplotype in situ protocol simplifies haplotype segregation analysis in pedigrees."/><meta property="og:title" content="Patent US5665540 - Multicolor in situ hybridization methods for genetic testing"/><meta property="og:type" content="book"/><meta property="og:site_name" content="Google Books"/><meta property="og:image" content="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><link rel="image_src" href="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><script>(function(){try{var aa=function(a,b,c,d){d=d||{};d._sn=["cfg",b,c].join(".");window.gbar.logger.ml(a,d)};var g=window.gbar=window.gbar||{},l=window.gbar.i=window.gbar.i||{},m={},n;function _tvn(a,b){var c=parseInt(a,10);return isNaN(c)?b:c}function _tvf(a,b){var c=parseFloat(a);return isNaN(c)?b:c}function _tvv(a){return!!a}function p(a,b,c){(c||g)[a]=b}g.bv={n:_tvn("2",0),r:"",f:".67.",e:"0",m:_tvn("0",1)};
function q(a,b,c){var d="on"+b;if(a.addEventListener)a.addEventListener(b,c,!1);else if(a.attachEvent)a.attachEvent(d,c);else{var f=a[d];a[d]=function(){var a=f.apply(this,arguments),b=c.apply(this,arguments);return void 0==a?b:void 0==b?a:b&&a}}}var s=function(a){return function(){return g.bv.m==a}},ba=s(1),ca=s(2);p("sb",ba);p("kn",ca);l.a=_tvv;l.b=_tvf;l.c=_tvn;l.i=aa;var da=window.gbar.i.i;var t,u,v,w;function ea(a){v=a}function fa(a){var b;if(b=v&&window.encodeURIComponent)b=a.href,b=!b.match(/^http[s]?:\/\/accounts\.google\.[^/]*\/ClearSID/i)&&!b.match(/^http[s]?:\/\/[^/]*\/accounts\/ClearSID/i);if(b=b&&encodeURIComponent(v()))a.href=a.href.replace(/([?&]continue=)[^&]*/,"$1"+b)}function ga(a){window.gApplication&&(a.href=window.gApplication.getTabUrl(a.href))}
function ha(a){var b=document.forms[0].q,c=window.encodeURIComponent&&b&&b.value,b=b&&b.placeholder;c&&c!=b&&(a.href=a.href.replace(/([?&])q=[^&]*|$/,function(a,b){return(b||"&")+"q="+encodeURIComponent(c)}))}n=l.a("")?ga:ha;
function x(a,b,c,d,f,e){var h=document.getElementById(a);if(h){var k=h.style;k.left=d?"auto":b+"px";k.right=d?b+"px":"auto";k.top=c+"px";k.visibility=u?"hidden":"visible";f&&e?(k.width=f+"px",k.height=e+"px"):(x(t,b,c,d,h.offsetWidth,h.offsetHeight),u=u?"":a)}}
var y=[],ia=function(a,b){y.push(b)},ja=function(a){a=a||window.event;var b=a.target||a.srcElement;a.cancelBubble=!0;null==t&&(a=document.createElement(Array.every||window.createPopup?"iframe":"div"),a.frameBorder="0",t=a.id="gbs",a.src="javascript:''",b.parentNode.appendChild(a),q(document,"click",z));var c=b,b=0;"gb3"!=c.className&&(c=c.parentNode);a=c.getAttribute("aria-owns")||"gbi";var d=c.offsetWidth,f=20<c.offsetTop?46:24;document.getElementById("tphdr")&&(f-=3);var e=!1;do b+=c.offsetLeft||
0;while(c=c.offsetParent);var c=(document.documentElement.clientWidth||document.body.clientWidth)-b-d,h,d=document.body,k=document.defaultView;k&&k.getComputedStyle?(d=k.getComputedStyle(d,""))&&(h=d.direction):h=d.currentStyle?d.currentStyle.direction:d.style.direction;h="rtl"==h;if("gbi"==a){for(d=0;k=y[d++];)k();A(null,window.navExtra);h&&(b=c,e=!0)}else h||(b=c,e=!0);u!=a&&z();x(a,b,f,e)},z=function(){u&&x(u,0,0)},A=function(a,b){var c,d=document.getElementById("gbi"),f=a;f||(f=d.firstChild);
for(;b&&(c=b.pop());){var e=d,h=c,k=f;w||(w="gb2");e.insertBefore(h,k).className=w}},ka=function(a,b,c){if((b=document.getElementById(b))&&a){a.className="gb4";var d=document.createElement("span");d.appendChild(a);d.appendChild(document.createTextNode(" | "));d.id=c;b.appendChild(d)}},la=function(){return document.getElementById("gb_70")},ma=function(){return!!u};p("qs",n);p("setContinueCb",ea);p("pc",fa);p("tg",ja);p("close",z);p("addLink",ka);p("almm",A);p("si",la);p("adh",ia);p("op",ma);var B=function(){},C=function(){},F=function(a){var b=new Image,c=D;b.onerror=b.onload=b.onabort=function(){try{delete E[c]}catch(a){}};E[c]=b;b.src=a;D=c+1},E=[],D=0;p("logger",{il:C,ml:B,log:F});var G=window.gbar.logger;var H={},na={},I=[],oa=l.b("0.1",.1),pa=l.a("1",!0),qa=function(a,b){I.push([a,b])},ra=function(a,b){H[a]=b},sa=function(a){return a in H},J={},K=function(a,b){J[a]||(J[a]=[]);J[a].push(b)},ta=function(a){K("m",a)},L=function(a,b){var c=document.createElement("script");c.src=a;c.async=pa;Math.random()<oa&&(c.onerror=function(){c.onerror=null;B(Error("Bundle load failed: name="+(b||"UNK")+" url="+a))});(document.getElementById("xjsc")||document.getElementsByTagName("body")[0]||
document.getElementsByTagName("head")[0]).appendChild(c)},N=function(a){for(var b=0,c;(c=I[b])&&c[0]!=a;++b);!c||c[1].l||c[1].s||(c[1].s=!0,M(2,a),c[1].url&&L(c[1].url,a),c[1].libs&&m.d&&m.d(c[1].libs))},O=function(a){K("gc",a)},P=null,ua=function(a){P=a},M=function(a,b,c){if(P){a={t:a,b:b};if(c)for(var d in c)a[d]=c[d];try{P(a)}catch(f){}}};p("mdc",H);p("mdi",na);p("bnc",I);p("qGC",O);p("qm",ta);p("qd",J);p("lb",N);p("mcf",ra);p("bcf",qa);p("aq",K);p("mdd","");p("has",sa);
p("trh",ua);p("tev",M);var Q=l.b("0.1",.001),R=0;
function _mlToken(a,b){try{if(1>R){R++;var c,d=a,f=b||{},e=encodeURIComponent,h=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&jexpid=",e("17483"),"&srcpg=",e("prop=22"),"&jsr=",Math.round(1/Q),"&ogev=",e("oy7uU_CwAeymsATIl4D4DQ"),"&ogf=",g.bv.f,"&ogrp=",e("1"),"&ogv=",e("1407723702.0"),"&oggv="+e("es_plusone_gc_20140723.0_p0"),"&ogd=",e("com"),"&ogc=",e("DEU"),"&ogl=",e("en")];f._sn&&(f._sn="og."+
f._sn);for(var k in f)h.push("&"),h.push(e(k)),h.push("="),h.push(e(f[k]));h.push("&emsg=");h.push(e(d.name+":"+d.message));var r=h.join("");S(r)&&(r=r.substr(0,2E3));c=r;var Aa=window.gbar.logger._aem(a,c);F(Aa)}}catch(Na){}}var S=function(a){return 2E3<=a.length},va=function(a,b){return b};function T(a){B=a;p("_itl",S,G);p("_aem",va,G);p("ml",B,G);a={};H.er=a}l.a("")?T(function(a){throw a;}):l.a("1")&&Math.random()<Q&&T(_mlToken);I.push(["m",{url:"//ssl.gstatic.com/gb/js/scm_7385cc5883250b43a39405734c1bea59.js"}]);g.mcf("c",{});g.sg={c:""};if(l.a("1")){var wa=l.a("");I.push(["gc",{auto:wa,url:"//ssl.gstatic.com/gb/js/abc/gci_91f30755d6a6b787dcc2a4062e6e9824.js",libs:"googleapis.client:plusone:gapi.iframes"}]);var xa={version:"gci_91f30755d6a6b787dcc2a4062e6e9824.js",index:"",lang:"en"};H.gc=xa;var U=function(a){window.googleapis&&window.iframes?a&&a():(a&&O(a),N("gc"))};p("lGC",U);l.a("1")&&p("lPWF",U)};window.__PVT="";if(l.a("1")&&l.a("1")){var V=function(a){U(function(){K("pw",a);N("pw")})};p("lPW",V);I.push(["pw",{url:"//ssl.gstatic.com/gb/js/abc/pwm_45f73e4df07a0e388b0fa1f3d30e7280.js"}]);var W=[],ya=function(a){W[0]=a},za=function(a,b){var c=b||{};c._sn="pw";B(a,c)},Ba={signed:W,elog:za,base:"https://plusone.google.com/u/0",loadTime:(new Date).getTime()};H.pw=Ba;var X=function(a,b){for(var c=b.split("."),d=function(){var b=arguments;a(function(){for(var a=g,d=0,e=c.length-1;d<e;++d)a=a[c[d]];a[c[d]].apply(a,b)})},f=g,e=0,h=c.length-1;e<h;++e)f=
f[c[e]]=f[c[e]]||{};return f[c[e]]=d};X(V,"pw.clk");X(V,"pw.hvr");p("su",ya,g.pw)};function Ca(){function a(){for(var b;(b=e[h++])&&"m"!=b[0]&&!b[1].auto;);b&&(M(2,b[0]),b[1].url&&L(b[1].url,b[0]),b[1].libs&&m.d&&m.d(b[1].libs));h<e.length&&setTimeout(a,0)}function b(){0<f--?setTimeout(b,0):a()}var c=l.a("1"),d=l.a(""),f=3,e=I,h=0,k=window.gbarOnReady;if(k)try{k()}catch(r){da(r,"ml","or")}d?p("ldb",a):c?q(window,"load",b):b()}p("rdl",Ca);var Da={D:1,H:2,da:3,p:4,W:5,M:6,F:7,g:8,ha:9,U:10,L:11,T:12,S:13,N:14,Q:15,P:16,fa:17,w:18,O:19,ga:20,ea:21,u:22,G:23,ja:24,ka:25,ia:26,A:27,j:28,o:29,k:30,ca:31,Z:32,$:33,J:34,K:35,ba:36,aa:37,Y:38,B:39,R:40,v:41,X:42,V:43,h:48,C:49,I:500},Y=[1,2,3,4,5,6,9,10,11,13,14,28,29,30,34,35,37,38,39,40,41,42,43,48,49,500];var Z=l.b("0.001",1E-4),Ea=l.b("1",1),Fa=!1,Ga=!1;if(l.a("1")){var Ha=Math.random();Ha<=Z&&(Fa=!0);Ha<=Ea&&(Ga=!0)}var Ia=Da,$=null;function Ja(){var a=0,b=function(b,d){l.a(d)&&(a|=b)};b(1,"");b(2,"");b(4,"");b(8,"");return a}
function Ka(a,b){var c=Z,d=Fa,f;f=a;if(!$){$={};for(var e=0;e<Y.length;e++){var h=Y[e];$[h]=!0}}if(f=!!$[f])c=Ea,d=Ga;if(d){d=encodeURIComponent;g.rp?(f=g.rp(),f="-1"!=f?f:"1"):f="1";c=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&oge=",a,"&ogex=",d("17483"),"&ogev=",d("oy7uU_CwAeymsATIl4D4DQ"),"&ogf=",g.bv.f,"&ogp=",d("22"),"&ogrp=",d(f),"&ogsr=",Math.round(1/c),"&ogv=",d("1407723702.0"),"&oggv="+
d("es_plusone_gc_20140723.0_p0"),"&ogd=",d("com"),"&ogl=",d("en"),"&ogc=",d("DEU"),"&ogus=",Ja()];if(b){"ogw"in b&&(c.push("&ogw="+b.ogw),delete b.ogw);var k;f=b;e=[];for(k in f)0!=e.length&&e.push(","),e.push(La(k)),e.push("."),e.push(La(f[k]));k=e.join("");""!=k&&(c.push("&ogad="),c.push(d(k)))}F(c.join(""))}}function La(a){"number"==typeof a&&(a+="");return"string"==typeof a?a.replace(".","%2E").replace(",","%2C"):a}C=Ka;p("il",C,G);var Ma={};H.il=Ma;setTimeout(function(){C(Ia.g)},0);}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var b=window.gbar.i.i;var c=window.gbar;var f=function(d){try{var a=document.getElementById("gbom");a&&d.appendChild(a.cloneNode(!0))}catch(e){b(e,"omas","aomc")}};c.aomc=f;}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var a=window.gbar;a.mcf("pm",{p:""});}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{window.gbar.rdl();}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
if (window['_OC_timingAction']) {window['_OC_timingAction']('patents_refpage');}</script><style>#gbar,#guser{font-size:13px;padding-top:1px !important;}#gbar{float:left;height:22px}#guser{padding-bottom:7px !important;text-align:right}.gbh,.gbd{border-top:1px solid #c9d7f1;font-size:1px}.gbh{height:0;position:absolute;top:24px;width:100%}#gbs,.gbm{background:#fff;left:0;position:absolute;text-align:left;visibility:hidden;z-index:1000}.gbm{border:1px solid;border-color:#c9d7f1 #36c #36c #a2bae7;z-index:1001}.gb1{margin-right:.5em}.gb1,.gb3{zoom:1}.gb2{display:block;padding:.2em .5em}.gb2,.gb3{text-decoration:none !important;border-bottom:none}a.gb1,a.gb4{text-decoration:underline !important}a.gb1,a.gb2,a.gb3,a.gb4{color:#00c !important}.gbi .gb3,.gbi .gb2,.gbi .gb4{color:#dd8e27 !important}.gbf .gb3,.gbf .gb2,.gbf .gb4{color:#900 !important}a.gb2:hover{background:#36c;color:#fff !important}#gbar .gbz0l{color:#000 !important;cursor:default;font-weight:bold;text-decoration:none !important}
#gbar { padding:.3em .6em !important;}</style></head><body  topmargin="3" marginheight="3"><div id=gbar><nobr><a onclick=gbar.qs(this);gbar.logger.il(1,{t:1}); class=gb1 id=gb_1 href="https://www.google.com/search?sa=N&tab=tw">Search</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:2}); class=gb1 id=gb_2 href="http://www.google.com/search?hl=en&tbm=isch&source=og&sa=N&tab=ti">Images</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:8}); class=gb1 id=gb_8 href="http://maps.google.com/maps?hl=en&sa=N&tab=tl">Maps</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:78}); class=gb1 id=gb_78 href="https://play.google.com/?hl=en&sa=N&tab=t8">Play</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:36}); class=gb1 id=gb_36 href="http://www.youtube.com/results?sa=N&tab=t1">YouTube</a> <a onclick=gbar.logger.il(1,{t:5}); class=gb1 id=gb_5 href="http://news.google.com/nwshp?hl=en&tab=tn">News</a> <a onclick=gbar.logger.il(1,{t:23}); class=gb1 id=gb_23 href="https://mail.google.com/mail/?tab=tm">Gmail</a> <a onclick=gbar.logger.il(1,{t:25}); class=gb1 id=gb_25 href="https://drive.google.com/?tab=to">Drive</a> <a class=gb3 href="http://www.google.com/intl/en/options/" onclick="this.blur();gbar.tg(event);return !1" aria-haspopup=true><u>More</u> <small>&#9660;</small></a><div class=gbm id=gbi><a onclick=gbar.logger.il(1,{t:24}); class=gb2 id=gb_24 href="https://www.google.com/calendar?tab=tc">Calendar</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:51}); class=gb2 id=gb_51 href="http://translate.google.com/?hl=en&sa=N&tab=tT">Translate</a><a onclick=gbar.logger.il(1,{t:17}); class=gb2 id=gb_17 href="http://www.google.com/mobile/?hl=en&tab=tD">Mobile</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:10}); class=gb2 id=gb_10 href="http://www.google.com/search?hl=en&tbo=u&tbm=bks&source=og&sa=N&tab=tp">Books</a><a onclick=gbar.logger.il(1,{t:212}); class=gb2 id=gb_212 href="https://wallet.google.com/manage/?tab=ta">Wallet</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:6}); class=gb2 id=gb_6 href="http://www.google.com/search?hl=en&tbo=u&tbm=shop&source=og&sa=N&tab=tf">Shopping</a><a onclick=gbar.logger.il(1,{t:30}); class=gb2 id=gb_30 href="http://www.blogger.com/?tab=tj">Blogger</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:27}); class=gb2 id=gb_27 href="http://www.google.com/finance?sa=N&tab=te">Finance</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:31}); class=gb2 id=gb_31 href="https://plus.google.com/photos?sa=N&tab=tq">Photos</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:12}); class=gb2 id=gb_12 href="http://www.google.com/search?hl=en&tbo=u&tbm=vid&source=og&sa=N&tab=tv">Videos</a><div class=gb2><div class=gbd></div></div><a onclick=gbar.logger.il(1,{t:66}); href="http://www.google.com/intl/en/options/" class=gb2>Even more &raquo;</a></div></nobr></div><div id=guser width=100%><nobr><span id=gbn class=gbi></span><span id=gbf class=gbf></span><span id=gbe></span><a target=_top id=gb_70 href="https://www.google.com/accounts/Login?service=&continue=http://www.google.com/patents%3Fhl%3Den&hl=en" class=gb4>Sign in</a><div style="display: none"><div class=gbm id=gbd5 aria-owner=gbg5><div class=gbmc><ol id=gbom class=gbmcc></ol></div></div></div></nobr></div><div class=gbh style=left:0></div><div class=gbh style=right:0></div><div role="alert" style="position: absolute; left: 0; right: 0;"><a href="http://www.google.com/patents/us5665540?hl=en&amp;output=html_text" title="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."><img border="0" src="http://www.google.com/images/cleardot.gif"alt="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."></a></div><div id="guser"><nobr></nobr></div><div style="clear:both;"></div><div id="gb-top-search-box" class="gb-top-search-box-small gb-reset"><table><tr><td class="logo"><a href="http://www.google.com/patents" class="logo-link"><img class="logo-img" src="/intl/en/images/logos/google_logo_41.png" alt="Go to Google Books Home" height="41"/></a></td><td><form action="http://www.google.com/search" name="f" id="vheadf" method="get"><span id="hf"></span><input type="hidden" name="tbm" value="pts"/><input type="hidden" name="tbo" value="1"/><input type="hidden" name="hl" value="en"/><table><tr><td><div class="inputs"><table><tr><td><div class="text-input"><input type="text" name="q" id="vheadq" class="text" maxlength="2048" size="31" value="" title="Search Patents" accesskey="s" autocomplete="off"/><script>window._OC_autoDir &&window._OC_autoDir('vheadq', 'tia-vheadq');</script></div></td><td><div class="submit-input"><input name="btnG" class="submit" type="submit" value=""/></div></td></tr></table></div></td><td class="col-ext-links"><div class="ext-links"><a href="http://www.google.com/advanced_patent_search">&lt;nobr&gt;Advanced Patent Search&lt;/nobr&gt;</a></div></td></tr></table></form></td></tr></table></div><div class="kd-appbar"><h2 class="kd-appname"><a href="/patents">Patents</a></h2><div class="kd-buttonbar left" id="left-toolbar-buttons"><a id="appbar-write-review-link" href=""></a><a id="appbar-view-print-sample-link" href=""></a><a id="appbar-view-ebook-sample-link" href=""></a><a id="appbar-patents-prior-art-finder-link" href="https://www.google.com/patents/related/US5665540"></a><a id="appbar-patents-discuss-this-link" href="http://www.google.com/url?id=cdFDBAABERAJ&amp;q=http://patents.stackexchange.com/redirect/google-patents%3Fpatent%3DUS5665540&amp;usg=AFQjCNGpGQK6JQj54LNOTfi8j92H1ydgmw" data-is-grant="true"></a><a id="appbar-read-patent-link" href="//docs.google.com/viewer?url=patentimages.storage.googleapis.com/pdfs/US5665540.pdf"></a><a id="appbar-download-pdf-link" href="//patentimages.storage.googleapis.com/pdfs/US5665540.pdf"></a></div><div class="kd-buttonbar right" id="right-toolbar-buttons"></div></div><div id="books-microdata" itemscope=""itemtype="http://schema.org/Book"itemid="http://www.google.com/patents/US5665540" style="display:none"><span itemprop="description">This invention relates to novel methods of optimally analyzing commonly obtained prenatal cell samples by in situ hybridization. In addition, this method diagnoses gene deletion and gene multiplication using multicolor in situ hybridization. A method is also provided to use multicolor in situ hybridization...</span><span itemprop="url">http://www.google.com/patents/US5665540?utm_source=gb-gplus-share</span><span class="main-title" itemprop="name">Patent US5665540 - Multicolor in situ hybridization methods for genetic testing</span><img itemprop="image" src="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"alt="Patent US5665540 - Multicolor in situ hybridization methods for genetic testing" title="Patent US5665540 - Multicolor in situ hybridization methods for genetic testing"></div><div style="display: none"><ol id="ofe-gear-menu-contents" class="gbmcc"><li class="gbe gbmtc"><a class="gbmt goog-menuitem-content" id="" href="http://www.google.com/advanced_patent_search">Advanced Patent Search</a></li></ol></div><table id="viewport_table" cellpadding="0" style="clear:both" cellspacing="0"><tr><td id="viewport_td"><div class=vertical_module_list_row><div id=intl_patents class=about_content><div id=intl_patents_v><table class="patent-bibdata"><tr><td class="patent-bibdata-heading">Publication number</td><td class="single-patent-bibdata">US5665540 A</td></tr><tr><td class="patent-bibdata-heading">Publication type</td><td class="single-patent-bibdata">Grant</td></tr><tr><td class="patent-bibdata-heading">Application number</td><td class="single-patent-bibdata">US 08/159,070</td></tr><tr><td class="patent-bibdata-heading">Publication date</td><td class="single-patent-bibdata">Sep 9, 1997</td></tr><tr><td class="patent-bibdata-heading">Filing date</td><td class="single-patent-bibdata">Nov 29, 1993</td></tr><tr><td class="patent-bibdata-heading">Priority date<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The priority date is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the date listed."></span></td><td class="single-patent-bibdata">Apr 21, 1992</td></tr><tr><td class="patent-bibdata-heading">Fee status<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The fee status is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the status or dates listed."></span></td><td class="single-patent-bibdata">Paid</td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Also published as</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/US5654148">US5654148</a>, </span><span class="patent-bibdata-value"><a href="/patents/WO1993021345A1">WO1993021345A1</a></span></span></td></tr><tr class="patent-bibdata-list-row alternate-patent-number"><td class="patent-bibdata-heading">Publication number</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value">08159070, </span><span class="patent-bibdata-value">159070, </span><span class="patent-bibdata-value">US 5665540 A, </span><span class="patent-bibdata-value">US 5665540A, </span><span class="patent-bibdata-value">US-A-5665540, </span><span class="patent-bibdata-value">US5665540 A, </span><span class="patent-bibdata-value">US5665540A</span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Inventors</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Roger+V.+Lebo%22">Roger V. Lebo</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Original Assignee</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=inassignee:%22The+Regents+Of+The+University+Of+California%22">The Regents Of The University Of California</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Export Citation</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/US5665540.bibtex">BiBTeX</a>, </span><span class="patent-bibdata-value"><a href="/patents/US5665540.enw">EndNote</a>, </span><span class="patent-bibdata-value"><a href="/patents/US5665540.ris">RefMan</a></span></span></td></tr><tr class="patent-internal-links"><td colspan=2><span class="patent-bibdata-value"><a href="#backward-citations">Patent Citations</a> (1),</span> <span class="patent-bibdata-value"><a href="#npl-citations">Non-Patent Citations</a> (110),</span> <span class="patent-bibdata-value"><a href="#forward-citations">Referenced by</a> (12),</span> <span class="patent-bibdata-value"><a href="#classifications">Classifications</a> (8),</span> <span class="patent-bibdata-value"><a href="#legal-events">Legal Events</a> (5)</span> </td></tr><tr><td colspan=2 class="patent-bibdata-external-link-spacer-top"></td></tr><tr class="patent-bibdata-external-link-spacer-bottom"></tr><tr><td colspan=2><span class="patent-bibdata-heading">External Links:&nbsp;</span><span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=cdFDBAABERAJ&q=http://patft.uspto.gov/netacgi/nph-Parser%3FSect2%3DPTO1%26Sect2%3DHITOFF%26p%3D1%26u%3D/netahtml/PTO/search-bool.html%26r%3D1%26f%3DG%26l%3D50%26d%3DPALL%26RefSrch%3Dyes%26Query%3DPN/5665540&usg=AFQjCNH7vz9nHTcNXuHy9uprQ0xfoG2grA">USPTO</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=cdFDBAABERAJ&q=http://assignments.uspto.gov/assignments/q%3Fdb%3Dpat%26pat%3D5665540&usg=AFQjCNFOMjM8EbQQkEFpxUEhmAfwmhrBvg">USPTO Assignment</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=cdFDBAABERAJ&q=http://worldwide.espacenet.com/publicationDetails/biblio%3FCC%3DUS%26NR%3D5665540A%26KC%3DA%26FT%3DD&usg=AFQjCNFi16bmetuB3ByoA3ozmt-7fdi-wQ">Espacenet</a></span></span></td></tr><tr class="patent-bibdata-group-spacer"></tr></table><div class="number-and-title"><span class="patent-title"><invention-title mxw-id="PT54210986" lang="EN" load-source="patent-office">Multicolor in situ hybridization methods for genetic testing</invention-title></span><br><span class="patent-number">US 5665540 A</span></div><div class="patent-section patent-abstract-section"><div class="patent-section-header"><span class="patent-section-title">Abstract</span></div><div class="patent-text"><abstract mxw-id="PA37676329" lang="EN" load-source="patent-office"> <div class="abstract">This invention relates to novel methods of optimally analyzing commonly obtained prenatal cell samples by in situ hybridization. In addition, this method diagnoses gene deletion and gene multiplication using multicolor in situ hybridization. A method is also provided to use multicolor in situ hybridization to identify chromosomal haplotypes co-segregating with disease-related genetic alterations and with normal genes. This haplotype in situ protocol simplifies haplotype segregation analysis in pedigrees.</div>
  </abstract></div></div><div class="patent-section patent-drawings-section"><div class="patent-section-header"><span class="patent-section-title">Images<span class="patent-section-count">(2)</span></span></div><div class="patent-drawings-body"><div class="patent-drawings-carousel"><div class="drawings"><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5665540-2.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5665540-2.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5665540-3.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5665540-3.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div></div></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img" alt="Previous page"src="/googlebooks/images/kennedy/page_left.png"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img" alt="Next page"src="/googlebooks/images/kennedy/page_right.png"width="21" height="21" /></div></div></div><div class="patent-post-drawings"></div><div class="patent-section patent-claims-section"><div class="patent-section-header"><span class="patent-section-title">Claims<span class="patent-section-count">(21)</span></span></div><div class="patent-text"><div mxw-id="PCLM5129749" lang="EN" load-source="patent-office" class="claims">
    <claim-statement>What is claimed is:</claim-statement> <div class="claim"> <div num="1" class="claim">
      <div class="claim-text">1. A method to identify an abnormal number of a gene in a chromosome complement from cells of a patient, comprising:<div class="claim-text">obtaining a sample of cells from the patient comprising said chromosome complement;</div> <div class="claim-text">hybridizing a first probe labeled with a first fluorescent color to said chromosome complement, wherein said first labeled probe hybridizes specifically to a chromosome region adjacent to said gene;</div> <div class="claim-text">hybridizing a second labeled probe to said chromosome complement, wherein said second labeled probe hybridizes specifically to said gene and the label of said second labeled probe has a fluorescent color distinguishable from said first fluorescent color;</div> <div class="claim-text">detecting signals from said first and second probes;</div> <div class="claim-text">detecting an abnormal gene number of said gene by identifying said chromosome complement having a quantity of signal from said first probe that is not equal to the quantity of signal from said second probe, wherein said signals from first and second probes are spaced in a predetermined geometric pattern.</div> </div>
    </div>
    </div> <div class="claim-dependent"> <div num="2" class="claim">
      <div class="claim-text">2. The method of claim 1, wherein the abnormal number of the gene results from a deletion.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="3" class="claim">
      <div class="claim-text">3. The method of claim 2, wherein said first labeled probe hybridizes to an X chromosome.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="4" class="claim">
      <div class="claim-text">4. The method of claim 2, wherein said gene is asteroid sulfatase gene.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="5" class="claim">
      <div class="claim-text">5. The method of claim 4, wherein said second labeled probe is cosmid CY437.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="6" class="claim">
      <div class="claim-text">6. The method of claim 2, wherein said gene is a alpha-thalassemia gene.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="7" class="claim">
      <div class="claim-text">7. The method of claim 2, wherein said gene is a Duchenne muscular dystrophy gene.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="8" class="claim">
      <div class="claim-text">8. The method of claim 2, wherein said gene is a Miller-Deiker gene.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="9" class="claim">
      <div class="claim-text">9. The method of claim 2, wherein said gene is a Smith-Magenis gene.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="10" class="claim">
      <div class="claim-text">10. The method of claim 1, wherein the abnormal number of the gene results from a multiplication.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="11" class="claim">
      <div class="claim-text">11. The method of claim 10, wherein said gene is located in chromosome 17.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="12" class="claim">
      <div class="claim-text">12. The method of claim 11, wherein said gene is the Charcot-Marie-Tooth Disease Type 1A gene.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="13" class="claim">
      <div class="claim-text">13. The method of claim 12, wherein said first labeled probe is cosmid centromeric probe D17Z1.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="14" class="claim">
      <div class="claim-text">14. The method of claim 12, wherein said second labeled probe hybridizes to a duplicated Charcot-Marie-Tooth Disease Type 1A gene region.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="15" class="claim">
      <div class="claim-text">15. The method of claim 1, wherein said first labeled probe and said second labeled probe are each labeled with a different compound from a group including Texas red, fluorescein, phycocrytherin, rhodamine, phycocyanin, allophycocyanin, horseradish peroxidase, and alkaline phosphatase.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="16" class="claim">
      <div class="claim-text">16. The method of claim 1, wherein said chromosome complement is from a prenatal sample.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="17" class="claim">
      <div class="claim-text">17. The method of claim 16, wherein the prenatal sample comprises amniocytes, chorionic villi, fetal fibroblasts, or preimplantation embryo cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="18" class="claim">
      <div class="claim-text">18. The method of claim 17, wherein the sample comprises amniocytes and the method further comprises suspending the amniocytes in a 1:1::0.2% KCl1:0.2% NaCitrate solution.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="19" class="claim">
      <div class="claim-text">19. The method of claim 17, wherein the sample comprises chorionic villi, and the method further comprises suspending the chorionic villi in a 1:1::0.2% NaCitrate:0.2% KCl solution.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="20" class="claim">
      <div class="claim-text">20. The method of claim 17, wherein the sample comprises fetal fibroblasts, and the method further comprises suspending the fetal fibroblasts in a 75 mM KCl hypotonic solution.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="21" class="claim">
      <div class="claim-text">21. The method of claim 17, wherein the sample comprises preimplantation embryo cells and the method further comprises placing the cells in a 1% sodium citrate solution.</div>
    </div>
  </div> </div></div></div><div class="patent-section patent-description-section"><div class="patent-section-header"><span class="patent-section-title">Description</span></div><div class="patent-text"><div mxw-id="PDES67018165" lang="EN" load-source="patent-office" class="description">
    <p>This invention was made with Government support under Grant No. NS-25541-01A1, awarded by the National Institute of Health. The Government has certain rights in this invention.</p>
    <p>This is a continuation of application Ser. No. 07/871,967, filed Apr. 21, 1992, now abandoned.</p>
    <heading>BACKGROUND OF THE INVENTION</heading> <p>1. Field of the Invention</p>
    <p>This invention relates generally to multicolor in situ hybridization methods that detect genetic abnormalities. In particular, this invention relates to novel methods to prepare cells for multicolor in situ hybridization and to detect genetic deletions, multiplications, and chromosomal haplotypes associated with genetically related diseases.</p>
    <p>2. Brief Description of Background Art</p>
    <p>Over 2000 human diseases result from DNA alterations including deletions, multiplications and nucleotide substitutions. Finding genetic disease alterations in individuals and following these alterations in families provides a means to confirm clinical diagnoses and to diagnose disease in carriers, preclinical and subclinical affected individuals, affected unborn fetuses, and preimplantation embryos. Counselling based upon accurate diagnoses allows patients to make informed decisions about potential parenting, ongoing pregnancy, and early intervention in affected individuals.</p>
    <p>Disease associated deletions, multiplications, and substitutions may be large or small. Diseases such as steroid sulfatase deficiency, alpha-thalassemia, and Duchenne muscular dystrophy usually result from large deletions. Steroid sulfatase deficiency and alpha-thalassemia patients have common deletion breakpoints while Duchenne muscular dystrophy patients have many deletion and duplication breakpoints. Diseases such as Charcot-Marie-Tooth Disease Type 1A (CMT 1A) have multiple duplication breakpoints that cause the same disease by multiplying the same specific gene(s).</p>
    <p>Even in the absence of deletions or multiplications, nucleotide substitutions can result in genetic disease. Multiple alleles within genes are common. Most alleles result from neutral mutations that produce indistinguishable, normally active gene products or express normally variable characteristics like eye color. In contrast, some alleles are associated with clinical disease like sickle cell anemia. Many disease associated alleles co-segregate with specific haplotypes reflected by a unique sequence of neutral nucleotide changes in that gene region. Identifying and following the segregation of affected haplotypes in families provides a means to identify a disease-related gene even when the specific gene mutation is unknown.</p>
    <p>Previously diagnosis and confirmation of genetic disease and carrier states often relied upon enzyme activity testing, statistical analysis, or invasive diagnostic procedures. For example, painful nerve conduction tests have been necessary to detect preclinical and subclinical cases of CMT1A, and amniocyte steroid sulfatase assays have been used to prenatally diagnose steroid sulfatase deficiency.</p>
    <p>DNA polymorphisms (RFLPs; Restriction fragment length polymorphisms) have been used to diagnose more than 20 genetic diseases reliably (See Lebo et al, Am. J. Hum. Genet. 47:583-590, 1990a, for list). The DNA Committee of the Pacific Southwest Regional Genetics Network proposed that a prenatal clinical test must be informative in at least 70% of fetuses and must be at least 95% reliable (Ibid.). All subsequent communications have agreed with these criteria. The percent informative matings are calculated according to Chakravarti and Buetow, Am. J. Hum. Genet. 37:984-997 (1985), with different formulas for autosomal recessive, autosomal dominant, and X-linked genetic disease. Not all matings are informative because parents may be homozygous for neutral DNA polymorphisms (FIG. 1). The proportion of informative matings depend upon 1) the number of different alleles at each gene locus, 2) the relative frequency of each allele in the population (the most informative have more than one common allele), and 3) whether alleles are distributed randomly throughout the population. Finding enough informative polymorphisms can be very laborious when few or uncommon polymorphisms are found at a disease locus [Lebo et al, Am. J. Hum. Genet. 47:583-590 (1990]. Using characterized polymorphisms may be laborious since several often need to be tested. See, e.g., Lebo et al, Am. J. Med. Genet. 37:187-190, (1990). Even then a proportion of uninformative results in some pedigrees are anticipated. For example, although a substantial number of polymorphic loci have been reported in the CMT1A gene region, multiple polymorphic tests must be used to identify carriers and affecteds. Further complicating accurate diagnosis is the low but real recombination frequency between the reported flanking polymorphisms and the CMT1A locus. Thus for diseases like CMT1A a direct DNA test would provide a more rapid, reliable, and tolerable diagnostic test.</p>
    <p>In situ nucleic acid hybridization (in situ hybridization) is the most direct means to map unique chromosome-specific sequences. Originally in situ hybridization used radiolabeled probes to detect target genes [Harper and Saunders, Chromosoma, 83:431-439 (1981)]. While radiolabeled probes detected unique nucleic acid sequences on metaphase chromosomes, detection lacked both speed and resolution. Fluorescent labels conjugated to DNA probes localized targets quickly with high resolution [Langer-Safer et al., Proc. Natl. Acad. Sci. USA, 79:4381-4385 (1982)] to identify single copy genes in metaphase chromosomes [Landegant et al., Nature, 317:175-177 (1985)].</p>
    <p>Simultaneously using different colored nucleic acid probes developed for repetitive sequences [Nederlof et al., Cytometry, 10:20-27 (1989)] ordered cosmid clones on a region of uniformly stained chromosome [Lichter et al., Science, 247:64-69 (1990)] and resolved closely spaced DNA clones in interphase nuclei [Trask et al., Genomics, 5:710-717 (1989)]. Different colored probes have been used to demonstrate chromosome translocation in chronic myelogenous leukemia in interphase nuclei [Tikachuk et al., Science, 250:559-562 (1990)].</p>
    <p>While substantial nonspecific signal previously limited diagnostic utility, recent studies demonstrated that a 40kb myophosphorylase cosmid probe gave two unique hybridized signals on every interphase nucleus observed [Lebo et al., Hum. Genet. 86:17-24 (1990)]. This result demonstrated that in situ hybridization to interphase nuclei was sufficiently specific to count gene target number. This specificity has been demonstrated using a Texas red labeled 2kb keratin cDNA hybridized simultaneously with a yellow-green fluorescein labeled 3kb flanking genomic sequence that each identify two and only two targets in blue DAPI stained normal interphase nuclei (hereinafter referred to as multicolor in situ hybridization).</p>
    <p>Recently protocols have been improved to order 2kb DNA targets with either Texas red label or yellow-green fluorescein label on blue DAPI banded metaphase chromosomes [Lebo et al., Hum. Genet. 88:13-20 (1991); Lebo et al., Am. J. Hum. Genet. 50:42-55, 1992]. This improved sensitivity and specificity makes it possible to map small gene probes to morphologically distinguishable chromosome regions. Closely spaced clones within 250kb have recently been ordered on banded chromosomes. Thus both copies of duplicated disease genes separated by about 250kb are anticipated to be resolved on early metaphase chromosomes to test for duplicated disease genes directly.</p>
    <p>DNA replication that generates a double hybridization signal is difficult to distinguish from true intrachromosomal duplication [Lupsky et al. Cell, 68:219-232 (1991)]. Efforts to minimize cells with replicated gene targets have included using confluent amniocyte and fibroblast cultures that have reached stationary phase as well as testing direct amniocytes and chorionic villus cells.</p>
    <p>Protocols allowing different cells to be tested by multicolor in situ hybridization would provide extended clinical applicability of this protocol. False negative in situ hybridization results must be minimized by optimally processing and fixing cells to slides according to cell type and slide age. Optimal proteinase K digestion removes histone and non-histone cellular and chromatin-bound proteins to expose nearly all gene sequences to probe without removing the remaining protein cytoskeleton that holds the nuclear DNA in place. Excessive and under digestion each result in failure to detect gene sequences (false negatives). The same multicolor in situ hybridization protocol that optimally detects lymphocyte genes will detect no genes in cultured fibroblasts. Likewise protocols that work on fresh preparations will detect no sequences on old slides because proteolytic enzyme digestion must be increased to expose unique DNA sequences.</p>
    <p>Although in situ hybridization has not been applied to prenatal diagnosis previously, scoring each hybridized cell individually can distinguish common fetal cells from contaminating maternal cells in prenatal samples to give an unambiguous gene deletion or multiplication diagnosis. In contrast, contaminating DNA from nucleated maternal blood cells mixed with direct amniocytes or maternal decidual cells in improperly dissected chorionic villi can be amplified along with fetal DNA by the polymerase chain reaction (PCR) to give false gene signals. Currently restriction enzyme analysis is used to obtain reliable results on DNA extracted from direct amniotic fluid samples from at-risk alpha-thalassemia fetuses so that minor contaminating maternal bands which are significantly lighter do not interfere with diagnosis [Lebo, Hum. Genet. 85:293-299, (1990)]. Culturing all tested amniocytes removes maternal blood cells but increases diagnostic time 2-3 weeks. Excellent dissection of chorionic villi will remove nearly all maternal decidual cells, but is technically difficult and requires specialized training. PCR results cannot be trusted with suboptimally dissected chorionic villi because maternal decidual cells grow along with fetal chorionic villus cells. Thus multiple samples may have to be taken when fetal and maternal DNA results do not differ. In contrast, testing cells directly by in situ hybridization can distinguish a small contaminating maternal cell population among the fetal cells because each cell is scored individually. In addition smaller maternal lymphocyte nuclei can be distinguished visually from the larger direct amniocyte nuclei. Thus multicolor in situ hybridization eliminates culture time and false results from maternal cellular DNA amplified indiscriminately by the polymerase chain reaction.</p>
    <p>Prior to the present invention, identifying the specific haplotype of a chromosome was cumbersome and time consuming. Chromosomal haplotypes contain many mutations and probing a sufficient number for familial segregation analysis is labor intensive. Despite the many mutations present in a haplotype, the variability between individuals is expected to be small because the races are considered to have diverged about 200,000 years ago based upon no differences in the alpha-globin gene exons in the three races. The intron and gene flanking sequences are expected to have greater variability. Recently HLA intron sequences have been analyzed and homology on the order of 95% has been found. The FCG2 genes which share 98% exon homology have been distinguished from the FCG3 genes which share 98% homologous sequences but differ from the FCG2 genes by about 5% [Lebo et al., Hum. Genet. 88:13-20, (1991)]. Prior hybridization protocols were unable to distinguish this level of homology. Haplotype analysis by in situ hybridization is easy and direct even in prenatal samples where maternal contamination can confuse the results.</p>
    <p>The present invention overcomes these and other shortcomings by providing a novel combination of multicolor in situ hybridization and analysis of geometric fluorescence patterns to increase the proportion of nuclei correctly scored for genetic aberrations. Methods of multicolor in situ hybridization are provided to allow precise and rapid genetic haplotype analysis. Novel methods of cellular preparation expand the applicability of the claimed methods to prenatal diagnosis.</p>
    <heading>SUMMARY OF THE INVENTION</heading> <p>Fixation methods for in situ hybridization to amniocytes, chorionic villus cells, fibroblasts, and blastocysts are included for diagnostic and research applications. Using in situ hybridization to uncultured amniocytes and chorionic villus cells provides accurate results quickly from small amniotic fluid volumes or a few villi without expensive, time consuming cell culture.</p>
    <p>This novel method also detects abnormal gene number in amniocytes, chorionic villus cells, fibroblasts, and blastocysts. The method comprises obtaining a sample from a patient or fetus, hybridizing a first labeled probe specifically to a chromosomal region adjacent to the gene, simultaneously hybridizing a second labeled probe specifically to said gene, and ascertaining the relative number and positions of the first and second labels in the chromosome complement. Gene deletion is detected in interphase nuclei by scoring interphase nuclei with two flanking gene targets reflecting two intact chromosome regions that have not undergone DNA duplication during cell growth. Then the number of adjacent gene copies are counted. Nuclei with an autosomal gene adjacent to both control spots are normal (FIG. 2C). A nucleus with an autosomal gene adjacent to only one control spot has the second gene deleted (FIG. 2E). Nuclei with two or more genes adjacent to one or both control spots have extra gene copies (FIG. 2D). When scoring metaphase chromosome complements all targets are found on each replicated chromatid and each metaphase chromosome has two chromatids. Thus a single copy gene will have two metaphase chromosome spots on each homologous chromosome and a duplicated gene on that chromosome will have four spots (FIGS. 2A, B).</p>
    <p>Improved in situ hybridization methods in the present invention identify genetic disease-associated haplotypes. The method comprises obtaining a sample from all appropriate family members including affected patients and at-risk fetuses, hybridizing at least two fluorescently labeled probes specific for different haplotypes of said gene to chromosomes of each sample, wherein each different probe is labeled with a fluorescent label of a different color, observing the color hybridizing specifically to 0, 1, or 2 gene targets per interphase nucleus (0, 2 or 4 targets per metaphase karyotype), and correlating the colors specific for individual haplotypes with segregation of the disease haplotype in the family. As in the most simple application, a flanking probe labeled in one color and a probe specific for the mutation-carrying haplotype in a second color will distinguish 0, 1, or 2 mutant allele haplotypes per cell (FIGS. 2F, G).</p>
    <p>Kits are also provided to practice the present invention conveniently. These kits contain probes homologous to genes and adjacent DNA regions with different fluorescent labels.</p>
    <heading>BRIEF DESCRIPTION OF THE DRAWINGS</heading> <p>FIG. 1 illustrates segregation of polymorphic markers at three loci at an autosomal dominant genetic disease locus. The polymorphisms in locus 1 are not informative, locus 2 partially informative, and locus 3 fully informative. When a fluorochrome labeled probe specific for locus 3 allele 2 is used, all individuals can be diagnosed correctly by in situ hybridization with 0, 1, or 2 haplotype-specific loci. Additional flanking loci can serve as controls as can different fluorochrome labels for the other alleles.</p>
    <p>FIG. 2A illustrates a normal chromosome complement with two different fluorescent probes hybridized to a gene (open circles) and an adjacent region (solid circles).</p>
    <p>FIG. 2B illustrates a chromosome complement with a duplicated gene with one fluorescent probe hybridized to two genes (open circles) and the other different colored fluorescent probe hybridized to an adjacent region (solid circles). The order is unimportant.</p>
    <p>FIG. 2C illustrates a normal interphase nucleus with two different fluorescent probes hybridized to each normal autosomal gene (solid circle) and an adjacent region (open circle).</p>
    <p>FIG. 2D illustrates the appearance of an interphase nucleus containing the chromosome in 2B with the gene duplicated (left) and the normal chromosome (right).</p>
    <p>FIG. 2E illustrates an interphase nucleus with an autosomal gene deletion (left) and a normal chromosome (right).</p>
    <p>FIG. 2F illustrates an adjacent probe (open circle) hybridized to a chromosome having a haplotype homologous to a haplotype probe (solid circle, left) and a chromosome that does not hybridize to the haplotype-specific probe but does hybridize to the flanking probe (open circle, right).</p>
    <p>FIG. 2G illustrates an adjacent probe (open circle) hybridized to a chromosome having a haplotype homologous to a haplotype probe (solid circle, left) and a chromosome that does not hybridize to the first haplotype-specific probe but does hybridize to the second haplotype specific probe (dotted circle, right) with a flanking probe (open circle, right).</p>
    <p>FIG. 3 illustrates the segregation of informative alleles for an autosomal recessive genetic disease. A single fluorochrome identifying both alleles in the affected patient can test carrier status directly in other family members as in FIG. 2F.</p>
    <heading>DESCRIPTION OF THE PREFERRED EMBODIMENT</heading> <p>Definitions</p>
    <p>As used herein, the term "gene" refers to any nucleic acid sequence within a chromosome with a functional role in encoding or transcribing a protein or regulating other gene expression of introns or a flanking genomic sequence which co-segregates with high fidelity. The "gene" may consist of all of the functional nucleic acids responsible for encoding a protein or only a portion of the nucleic acids responsible for encoding or expressing a protein. The nucleic acid subsequence may be within exons, introns, initiation, termination, or promoter sequences or other regulatory sequences. The nucleic acid sequence that results in an altered protein product or altered protein concentration may be of any length. When the term "gene" is used with a disease name, e.g. Charcot-Marie-Tooth Type 1A (CMT1A) gene, the gene is the nucleic acid sequence within the chromosome complement which is multiplied, deleted or otherwise mutated to result in the disease phenotype.</p>
    <p>As used herein, the term "adjacent" refers to a flanking gene region on the same chromosome. The adjacent region may be either 3' to 5' to the gene. Although this distance may vary, the flanking sequence must be sufficiently distance so as not to be deleted, multiplied or mutated along with the respective gene in carriers or affecteds. The distance must also be sufficiently small so that fragmentation of interphase nuclei during slide preparation and hybridization will not disrupt hybridization of probes directed to both the gene and the adjacent region in a statistically significant fashion. Fragmentation of metaphase chromosome complements must be sufficiently rare to permit scoring flanking and gene fluorescence of intact banded chromosome complements. In a disease such as CMT 1A in which a gene is multiplied different distances in both directions, the adjacent region will be sufficiently distant in either direction so that it is not multiplied with the gene. Typically minimal distance to "adjacent" sequences refers to a range just distal to the common deletion breakpoint for diseases like alpha-thalassemia to sequences megabases from highly variable, large duplicated chromosome regions as CMT1A. When two different colored adjacent fluorescent probes are visualized in the same location, the composite color is scored (i.e. red+green=yellow).</p>
    <p>As used herein, the term "sample" refers to a collection of biological material from an organism containing nucleated cells. This biological material may be solid tissue as from a fresh or preserved organ or tissue sample or biopsy; blood or any blood constituents; bodily fluids such as amniotic fluid, peritoneal fluid, or interstitial fluid; cells from any time in gestation including prefertilizied or fertilized embryo or preimplantation blastocysts, or any other sample with intact interphase nuclei or metaphase cells no matter how many chromosomes are present. The "sample" may contain compounds which are not naturally intermixed with the biological material such as preservatives anticoagulants, buffers, fixatives, nutrients, antibiotics, or the like.</p>
    <p>The biological material must contain substantially intact chromosome complements. As used herein, the term "chromosome" refers to substantially intact chromosomes and the term "chromosome complement" refers to a substantially complete, intact chromosome set. A fragmented chromosome is considered substantially intact provided that the deleted or multiplied gene and the adjacent regions of the chromosome are intact and a sufficient amount of the chromosome is linked to the intact deleted or multiplied gene and adjacent regions to allow identification of the specific chromosome region copy number (e.g. the X chromosome or Y chromosome in the steroid sulfatase gene (STS) assay). A chromosome complement is substantially intact provided that all unknown chromosome regions of interest are intact, accessible, and hybridize to all probes to give the correct number of control flanking chromosome spots and unknown gene copies in the statistically expected frequency. Visualizing multiple gene copies on any chromosome may only occur in a statistically significant fraction of interphase nuclei (i.e. 15-60% for CMT1A) because the duplicated genes are quite close. Chromosome identification may be by means of evenly stained chromosomal DNA in metaphase spreads, traditional metaphase or prometaphase chromosome banding, hybridization to control chromosome-specific sequences with a spectrally unique labeled probe, or other means.</p>
    <p>As used herein, the term "predetermined pattern" refers to a previously identified geometrical relationship between probe target position and number on a normal chromosome complement and modified patterns on abnormal chromosome complements. When appropriate, a "predetermined pattern" includes not only the spatial relationship and quantity of different probe targets on interphase nuclei but also on metaphase chromosome complements that are stained uniformly or banded by any chromosome banding procedure. Generally, single adjacent probe targets on normal chromosomes carrying each normal gene type of interest will reflect the "predetermined normal pattern" in the method of the present invention.</p>
    <p>As used herein, the term "deletion" refers to the absence of a "gene" in a chromosome complement which results in a specific genetically related disease and perhaps in additional genetic diseases resulting from adjacent gene deletions. At least one additional "gene" is present in chromosome complements of subjects who do not exhibit the disease.</p>
    <p>As used herein, the terms "multiplication" or "duplication" refer to the presence of one or more extra "gene" copies in an affected subject's chromosome complement resulting in a specific genetic disease compared to the normal number of "genes" in chromosome complements of subjects without the disease.</p>
    <p>As used herein, the term "haplotype" refers to an ordered combination of alleles and/or neutral mutations in a defined chromosome region or chromosome complement. These sequences may be within a gene, between genes, or in an adjacent gene(s) or chromosome region that cosegregates with high fidelity.</p>
    <p>As used herein, the term "allele" refers to one of several alternative forms of a "gene" occupying a given chromosome locus.</p>
    <p>Fixation methods specifically for in situ hybridization are provided for amniocytes, chorionic villus cells, fibroblasts, and blastocysts. These methods can be used routinely for diagnostic and research samples. Amniocytes from small volumes of direct amniotic fluid and nuclei from a few chorionic villi provide accurate prenatal results without expensive, time consuming cell culture.</p>
    <p>In addition a novel method is provided for multicolor in situ hybridization. The method comprises obtaining a sample containing the unknown chromosome complement, using a first labeled probe that hybridizes specifically to a chromosomal region adjacent to the gene, using a second labeled probe which hybridizes specifically to said gene on the chromosome, and ascertaining the relative positions of the first and second labels in the chromosome complement. These relative locations have a predetermined normal geometric pattern in the chromosome complement. Deletion of an autosomal gene in amniocytes, chorionic villus cells, fibroblasts, or blastocysts is detected by identifying more first labeled flanking control DNA targets than the number of second labeled probe gene targets. Multiplication of an autosomal gene is detected by identifying fewer first control labeled probe targets than the number of second labeled probe targets. Additional probes and colors may be added to refine and extend this general procedure to include more loci per haplotype, more tested gene loci or flanking loci per hybridization procedure or sex chromosome testing.</p>
    <p>The present invention also provides a novel method for identifying a gene haplotype by multicolor in situ hybridization. This method comprises obtaining a sample with intact patient cell nuclei, hybridizing a first labeled probe specifically to a chromosomal region adjacent to the gene, hybridizing a second labeled probe specifically to a unique haplotype of said gene, and ascertaining the relative position and number of the first and second labels on the chromosome complement. This improved in situ hybridization method provides a means to identify haplotypes associated with genetic diseases. This method comprises obtaining a patient sample including nucleated cells, hybridizing these to at least two fluorescently labeled probes specific for different haplotypes of said gene wherein each different probe is tagged with a different color fluorescent label, observing the color of each hybridizing locus, and correlating probe color, number, and position to the patient's genetic status.</p>
    <p>Kits are also provided to practice the present invention conveniently. These kits contain probes homologous to genes and adjacent DNA regions with different fluorescent labels.</p>
    <p>Multicolor fluorescence in situ hybridization involves hybridizing two or more fluorescently labeled nucleic acid probes specifically to nucleic acids in a cytological preparation with total DNA labeled using a different colored stain. Probes are labeled with different fluorescent compounds each with a distinguishable color. When the nucleic acid probes are hybridized to cellular nucleic acid sequences, hybridization sites are detected by a unique colored fluorescence emission. Probe signal spacing and order determines the relative chromosomal position and number of homologous nucleic acid sequences.</p>
    <p>Chosen probes may hybridizes to homologous cellular nucleic acid sequences that are not located in the chromosomal region of interest or are not perfectly homologous to the probes. Also, if the homologous nucleic acid sequence in the cell is not exposed, the probe may not be able to hybridize.</p>
    <p>The present invention provides specific methods to use multicolor fluorescence in situ hybridization to detect gene multiplication and deletion in chromosome complements. Target probes which hybridize to nucleic acid sequences within the multiplied or deleted gene segments are used to detect the presence or absence of a gene and its copy number. Location probes which hybridize to nucleic acid sequences flanking the multiplied or deleted gene sequences are used to mark the proper chromosomal region. Detecting a location probe adjacent to the gene provides a means to exclude nonspecific fluorescence signals outside the gene region. Since location probes are adjacent to genes, intact chromosomes with deleted genes will exhibit a location probe without an adjacent target probe while fragmented or missing chromosome regions will not hybridize to either target or location probe and will exhibit no fluorescence.</p>
    <p>The in situ hybridization method in the present invention also provides a rapid, reliable, and highly informative means to identify chromosomal haplotypes associated with diseases. Many diseases associated with small mutations (e.g. one nucleotide substitution in sickle cell anemia) co-segregate with a specific chromosomal haplotype. Although the disease-related mutation may be too small to be identified by in situ hybridization, a chromosomal region defining a haplotype can be detected because it is much larger. Identifying a haplotype co-segregating with a disease-related mutation can delineate disease status. Prior methods of in situ hybridization could not distinguish between haplotypes because most haplotypes vary only by about 5%. The present method allows small haplotype differences to be distinguished.</p>
    <p>Probes to different haplotypes are used simultaneously. Each probe is labeled with a compound having a different fluorescent color. If a chromosomal region has more haplotypes than can be distinguished by available labels, the different probes can be hybridized to the cells in groups and the different labels bound to only one probe in each group. Following hybridization, the cells are scored. Correlation of labels to probe identifies the chromosomal haplotype. Cells are only scored when a labeled probe flanking the haplotype region hybridizes to both chromosomes in the chromosome complement. This flanking probe signal identifies haplotype chromosome regions that are sufficiently intact and available for hybridization.</p>
    <p>Probes are chosen to be within common deleted or duplicated gene regions including exons, introns, and unique adjacent regions. Probes may be either RNA or DNA oligonucleotides or polynucleotides and may contain not only naturally occurring nucleotides but their analogs like digoxygenin dCTP, biotin dcTP 7-azaguanosine, azidothymidine, inosine, or uridine. The advantages of analogs include greater stability, resistance to nuclease activity, ease of signal attachment, increased protection from extraneous contamination and an increased number of probe-specific colors.</p>
    <p>Probes have sufficient complementarity to their target polynucleotides so that stable and specific binding occurs between the chromosome and the probe. The degree of homology required for stable hybridization varies with the stringency of the hybridization medium and/or wash medium. Preferably completely homologous probes are employed in the present invention, but persons of skill in the art will readily appreciate that probes exhibiting lesser but sufficient homology can be used in the present invention.</p>
    <p>Probes may be generated and chosen by several means including mapping by in situ hybridization [Landegent et al, Nature 317:175-177, 1985], somatic cell hybrid panels [Ruddle &amp; Creagan, Ann. Rev. Genet. 9:431, 1981], or spot blots of sorted chromosomes [Lebo et al, Science 225:57-59, 1984]; chromosomal linkage analysis [Ott, Analysis of Human Genetic Linkage, Johns Hopkins Univ Press, pp. 1-197, 1985]; or cloned and isolated from sorted chromosome libraries from human cell lines or somatic cell hybrids with human chromosomes [Deaven et al, Cold Spring Harbor Symp. LI:159-168, 1986; Lebo et al. Cold Spring Harbor Symp. LI:169-176], radiation somatic cell hybrids [Cox et al, Am. J. Hum. Genet. 43:A141, 1988], microdissection of a Chromosome region [Claussen et al, Cytometry 11:suppl 4 p. 12, 1990], (all of which are incorporated by reference), or from yeast artificial chromosomes (YACs) identified by PCR primers specific for a unique chromosome locus (sequence tagged site or STS) or other suitable means like an adjacent YAC clone. Probes may be genomic DNA, cDNA, or viral DNA cloned in a plasmid, phage, cosmid, YAC, or any other suitable vector. A majority of cosmid length probes are excellent because the signal is sufficiently intense and the domain is sufficiently small to identify easily scored domains on interphase nuclei or metaphase chromosomes. Probes may be cloned or synthesized chemically. When cloned, the isolated probe nucleic acid fragments are typically inserted into a replication vector, such as lambda phage, pBR322, M13, pJB8, c2RB, pcos1EMBL, or vectors containing the SP6 or T7 promoter and cloned as a library in a bacterial host. General probe cloning procedures are described in Arrand J. E., Nucleic Acid Hybridization A Practical Approach, Hames B. D., Higgins, S. J., Eds., IRL Press 1985, pp. 17-45 and Sambrook, J., Fritsch, E. F., Maniatis, T., Molecular Cloning A Laboratory Manual, Cold Spring Harbor Press, 1989, pp. 2.1-3.58, both of which are incorporated herein by reference.</p>
    <p>Briefly, the DNA sequences to be cloned are partially or completely digested by cloning-site specific restriction enzyme digestion of selected nucleic acids. The ends of the DNA sequences to be cloned may be modified to minimize ligation to other DNA fragments but still permit ligation to the vector. Other modifications may be made prior to ligation to the vector like blunt ending or addition of homopolymer tails or oligonucleotide linkers. Vector nucleic acids must be cleaved with appropriate restriction enzymes to specifically open the cloning site. The resulting DNA fragments are recovered and tested for ligating efficiency. Acceptable DNA fragments to be cloned are ligated to the acceptable vector. The resulting ligated DNA sequences may then be processed appropriately for the chosen vector. Generally, ligated plasmids are transformed into bacteria, or packaged into a bacteriophage and transfected into bacteria, or introduced into yeast. The bacteria are plated and screened for clones carrying the sequences of interest or pools of yeast clones are grown and the DNA extracted. Additional screening steps including PCR amplification for an STS are used to identify a pool of yeast clones with the DNA sequence of interest. Then clones are plated in an array, grown, and screened with sequence-specific probe. Selected clones are grown and probes extracted by cell lysis and nucleic acid extraction followed by gel electrophoresis, high pressure liquid chromatography, or low or high speed column or gradient centrifugation. Pulsed field gel electrophoresis may be preferred to isolate a pure YAC clone.</p>
    <p>Alternatively, oligonucleotide probes may be synthesized chemically with or without fluorochromes, chemically active groups on nucleotides, or labeling enzymes using commercially available methods and devices like the Model 380B DNA synthesizer from Applied Biosystems, Foster City, Calif., using reagents supplied by the same company. Other methods may also be used to synthesize oligonucleotide probes, e.g., the solid phase phosphoramidite method that produces probes of about 15-250 bases. Methods are detailed in Caruthers et al., Cold Spring Harbor Symp. Quant. Biol., 47:411-418, 1982, and Adams, et al., J. Am. Chem. Soc., 105:661, 1983, both of which are incorporated herein by reference.</p>
    <p>Polymerase chain reaction can also be used to obtain large quantities of isolated probes. The method is outlined in U.S. Pat. No. 4,683,202, incorporated herein by reference.</p>
    <p>When synthesizing a probe for a specific target, the choice of nucleic acid sequence will determine target specificity. For example, probes may be selected by using commercially available computer programs to compare known DNA sequences from genes sequences found in libraries like Genebank and EMBL. The programs identify unique nucleotide sequences within the gene of interest. One such program is Eugene. Oligonucleotide sequences for PCR of a unique chromosome region can be chosen optimally by choosing sequences according to previously established protocols or by computer programs that choose the degree of homology desired along with the length of the probe. Sequences are chosen to avoid technical problems like primer dimers resulting from amplification of hybridized primers.</p>
    <p>Probes may be labeled with several fluorophors or enzymes that generate colored products. The probes used in the assay must be labeled with more than one distinguishable fluorescent or pigment color. These color differences provide a means to identify the hybridization positions of specific probes. Probes may be labeled with Texas red, rhodamine and its derivatives, fluorescein and its derivatives, dansyl, umbelliferone and the like or with horse radish peroxidase, alkaline phosphatase, or the like. Probes that are not separated spatially can be identified by a different color light or pigment resulting from mixing two other colors (light red+green=yellow) pigment (blue+yellow=green) or by using well chosen filter sets that pass only one color at a time.</p>
    <p>Probes can be labeled directly or indirectly. The common indirect labeling schemes covalently bind a ligand to the nucleotide and prepare labeled probe by incorporating this using random priming or nick translation. The ligand then binds an anti-ligand which is covalently bound to a fluorescent label. Ligands and anti-ligands vary widely. When a ligand has an anti-ligand, e.g., biotin, thyroxine, or cortisol, the ligand may be used in conjunction with the labelled naturally-occurring anti-ligand. Alternatively, a hapten or antigen may be used in combination with an antibody.</p>
    <p>One preferred indirect DNA probe label is bio-11-dUTP (ENZO Diagnostics, New York, N.Y.). First the probes are purified by cesium chloride gradient ultracentrifugation and glass bead extraction with a Gene Clean kit (Bio 101, La Jolla, Calif.). Clean probe (1 microgram) is then mixed with bio-11-dUTP or other DNA probe label along with <sup>3</sup> HdGTP tracer in the presence of 2 ng DNase per 50 microliter reaction mixture. After nick translation or random priming in the presence of polymerase, nuclease, and unlabeled nucleotides, labeled probe is separated from unincorporated nucleotides with G-50 Sephadex spin columns equilibrated in 150 mM NaCl, 15 mM sodium citrate, 0.1% sodium dodecyl sulfate (pH 7.0). Labeled probes are precipitated with a 50-fold excess of salmon sperm carrier DNA and yeast tRNA. Probes having repetitive sequences are further mixed with 100- to 500-fold excess depurinated human placental DNA. The probes are ethanol precipitated, washed twice in 70% ethanol at 0° C. and dried for 10 minutes at 23° C. in a Savant speed vac. The pellets are redissolved to 8 ng/microliter in 100% deionized formamide for 30 minutes at 37° C. The solution is mixed with an equal volume of 2×SSC, 20% dextran sulfate, 100 mM NaPO4 and denatured for 5 minutes at 90° C. Denatured unique probes are chilled on ice. Probes with repetitive sequences are prehybridized to placental human DNA for 4 hours at 37° C. Fluorescent labels are bound to the probes following hybridization. The same protocol may be followed to incorporate the hapten digoxygenin-11-dUTP (Boehringer, Mannheim, Germany) into probe.</p>
    <p>Preferably chromosomal DNA is stained with a spectrally distinguishable DNA specific stain such as DAPI. The chromosome staining provides a background to facilitate the identification of chromosomes and aid in distinguishing the colors and positions of the labels. Typically the chromosome is stained for three minutes with DAPI (Sigma) in McIlvane's buffer. The slide is rinsed twice in McIlvane's buffer for two minutes and drained well. Other DNA-specific stains like propidium iodide that uniformly stain chromosomes may be used with at least two other spectrally distinguishable stains or at lower concentrations so as not to obscure one stain color.</p>
    <p>The method of the present invention may be practiced on many cell types. Prenatal genetic diagnosis is dependent upon collection of cells of fetal origin. The present invention is particularly useful for prenatal diagnosis of genetic diseases in amniocytes, chorionic villus cells and blastocysts, as well as direct lymphocytes from any fetus or other subject or purified fetal cells from maternal circulation. The safest and most convenient methods of fetal cell collection are amniocentesis and chorionic villus sampling. Because amniocentesis taps fluid which is external to the fetus, the most abundant fetal cells in amniotic fluid are amniocytes. Smaller volumes of relatively pure amniotic fluid samples provide more amniocytes than cells such as lymphocytes which are commonly employed in other genetic analyses. Since viable amniocytes occur in small numbers in the amniotic fluid, larger fluid volumes are needed to provide a sufficient number of cells for restriction enzyme analysis but small volumes are diagnostic by direct multicolor in situ hybridization. Because culture is not required prior to assay with smaller fluid samples, this method is faster and safer. Chorionic villus sampling followed by careful, expert dissection from the maternal decidua provides many high quality cells which are ideal for in situ hybridization to interphase nuclei. Even though culturing is necessary for multiple karyotype analysis, direct in situ hybridization to chorionic villus cells provides an earlier answer than direct in situ hybridization to amniocytes for gene deletions, multiplications, or haplotype analysis. In fact, 95% of the significant chromosome abnormalities in newborns can be detected within 48 hours by in situ hybridization for aneuploidy in 5 chromosomes [Lebo et al., Am. J. Med. Genet.].</p>
    <p>Following collection, cells must be prepared for hybridization. Different cell types must be prepared differently. Lymphocyte and solid tissue preparation is described in Pardue, M. L., Nucleic Acid Hybridisation, A Practical Approach, Hames, B. D., Higgins, S. J., Eds., IRL Press, 1985, PP. 188-190, incorporated herein by reference. Briefly, lymphocytes are prepared from an aminopterin-synchronized culture. Colcemid is incubated with the culture and removed. Cells are collected by centrifugation and resuspended in hypotonic solution (1:1::0.4% NaCitrate:0.4% KCl) for 15 min. After a repeated centrifugation the pellet is resuspended in about 1 ml supernatant and the suspension fixed slowly in Carnoy's fixative (3:1::methanol:glacial acetic acid) for 15 min. After repeated fixation for at least 30 min the cell suspension is dropped on wet, ethanol cleaned slides for hybridization.</p>
    <p>Solid tissue is dissected into small pieces and fixed in Carnoy's fix (3:1::methanol:glacial acetic acid). Tissue in the fixative is added to acetic acid on a coverslip. The tissue is minced on the coverslip to release cells. A slide is placed on the coverslip and pressed to squash the cells. The slide is frozen on dry ice and the coverslip is removed. The slide is washed in ethanol and dried for hybridization.</p>
    <p>Standard protocols to prepare solid tissue for in situ hybridization are not effective for amniocytes, blastocysts, or chorionic villus cells (either direct or cultured). Generally preparation of these cells includes collection of the cells in amniotic fluid (direct amniocytes), mincing direct chorionic villi and dissociating with 85 U/ml pronase, or detachment from a culture plate by pronase or trypsin digestion (cultured cells). The fluid containing the cells is placed in a 15 ml polypropylene tube. The tube is centrifuged at 200 g for 7 minutes, the supernatant is removed and the cell pellet is gently resuspended by tapping. A hypotonic solution is added dropwise to a total volume of 1-2 ml while gently mixing. The hypotonic solution varies with the cell type. For direct amniocytes and direct chorionic villus cells the hypotonic solution is 1:1::0.2% KCl:0.2% NaCitrate. For cultured amniocytes the hypotonic solution is 1:1::0.4% sodium citrate:0.4% KCl. For cultured chorionic villus cells the hypotonic solution is 1:3::Hanks PBS:H<sub>2</sub> O.</p>
    <p>The solution is incubated in a 37° C. waterbath for 8 minutes and then immediately centrifuged at 200 g for 7 minutes. The supernatant is removed and the cell pellet is dissociated and fixed in 1-2 ml fix (3:1::methanol:glacial acetic acid) added dropwise with gentle tapping. After 15 minutes the solution is centrifuged at 200 g for 7 minutes. The supernatant is removed and the cell pellet dissociated and fixed in 1-2 ml fix added dropwise with gentle tapping. Precleaned cells stored in 100% methanol are chilled in an ice slurry and fixed cells are added dropwise to the slides. Slides are then air dried and examined by phase contrast microscopy to assure an adequate number of attached cells. If an unusually high level of cytoplasm is noted, additional slides can be made and plunged into glacial acetic acid 5 to 60 seconds to remove excess cytoplasm. Suitable slides are baked for 2 hours at 60° C. on a slide warmer.</p>
    <p>Blastocysts are derived from preimplantation embryos. A preferred method of cellular preparation and fixation is as follows.</p>
    <p>Embryo manipulation is best done under a dissecting microscope at 50X magnification. Embryos may be grown in Ham F10 medium using maternal serum in tissue culture dishes. The embryo is transferred into Ham F10 medium without serum. Then the embryo is transferred into Ham F10 with 1% BSA titrated to pH 2-2.15. After all of visible zona pellucida is dissolved, the embryo is transferred into phosphate buffered saline without calcium or magnesium (Hanks). Then the embryo is transferred into 1% sodium citrate hypotonic solution, transferred to the slide to be tested and fixed to the slide by adding dropwise Carnoy fix (3:1::methanol:glacial acetic acid). The position of the embryo is marked on the bottom of the slide. Dropwise fixation continues until the embryo disappears. Then the slide is fixed overnight in Carnoy's fixative. Subsequent preparation and in situ hybridization is completed according to cultured amniocyte protocols except that fixed cells are treated with approximately 6-fold more proteinase K to remove extra protein. Proteinase K treatment varies according to the number of blastocysts treated, how much zona pellucida remains, the size of the blastocysts which is usually dependent upon the number of divisions completed in the embryo, and the health of the cells tested.</p>
    <p>Just prior to hybridization the slides are rehydrated in phosphate buffered saline and treated with proteinase K (100 pg\ml; 100 microliters/slide [Stock solution: 10 mg/ml aged at 37° C. for 3 hours and frozen in aliquots] for 8 minutes at 37° C., then washed for 5 minutes at 23° C. in PBS plus 50 mM MgCl<sub>2</sub>. The slides are then fixed for 10 minutes at 23° C. in 1% formaldehyde in PBS plus 50 mM MgCl<sub>2</sub>. The slides are washed twice for 3 minutes at 23° C. in PBS, dehydrated in an ethanol series, then air dried. When dry the slide bound DNA is denatured for 5 minutes at 70° C. in 2×SSC, 70% formamide, 50 mM NaPO4 (pH 7.0). The slides are then chilled in 70% ethanol at 0° C., dehydrated in ethanol, and air dried.</p>
    <p>In situ hybridization is carried out on cells fixed to slides. In situ hybridization may be performed by several protocols such as those described in Pardue, M. L., Nucleic Acid Hybridisation, A Practical Approach, Hames, B. D., Higgins, S. J., Eds., IRL Press, 1985, pp. 190-198, and John, et al., Nature, 223:582-587 (1969), both of which are incorporated herein by reference.</p>
    <p>Various degrees of hybridization stringency can be employed. As the hybridization conditions become more stringent, a greater degree of complementarity is required between the probe and target to form and maintain a stable duplex. Stringency is increased by raising temperature, lowering salt concentration, or raising formamide concentration. Adding dextran sulfate or raising its concentration can increase the effective concentration of labeled probe to increase the rate of hybridization and ultimate signal intensity. In addition, ultrasonic treatment of the reaction vessel in a commercially available sonication bath may accelerate hybridization.</p>
    <p>After hybridization, slides are washed in a solution typically containing reagents similar to those found in the hybridization solution. Ideally the hybridization and initial washing solutions have similar stringencies. The washing time may vary from minutes to hours but longer washes typically lower nonspecific background in proportion to the length of total washing time in all solutions.</p>
    <p>A preferred technique follows: Slides are prepared as above and then aged by warming at 60° C. for 2 hours. Aged slides are treated with essentially protease free RNase (Sigma, Cat. No. R5503; 6.5 Kurnitz units/ml) for 60 min in 2×SSC at 37° C. The slides are washed three times 5 min each at 23° C. in 2×SSC, dehydrated in a graded ethanol series, and air dried. The slides were rehydrated 5 min in phosphate-buffered saline [PBS: 100 mM NaPO<sub>4</sub> ; 185 mM NaCl; (pH7.2)], then digested with proteinase K [Boehringer, Mannheim, Germany, Cat. No. 745723; Stock: 200 U/ml incubated at 37° C. for 3 hr and stored in frozen aliquots at -20° C. and diluted 1/100,000 in PBS]. After 8 minutes proteinase K digestion at 37° C., slides were washed 5 min in PBS and then the proteinase K was inactivated in 1% formaldehyde in PBS plus 50 mM MgCl<sub>2</sub>. The slide was washed twice for 3 min in PBS, dehydrated in an ethanol series, and air dried. Chromosomal DNA bound to the slide was denatured 5 min at 70° C. in 2×SSC, 70% formamide, and 50 mM NaPO<sub>4</sub> (pH 7.0), chilled in 70% ethanol at 0° C., further dehydrated in ethanol, and air dried.</p>
    <p>Labeled denatured probes (2-4 ng/microliter) were added to coverslips and the slides inverted on the coverslips remain inverted in a slide rack during incubation. The probes and slides were hybridized overnight at 37° C. in a moist dark chamber. Following hybridization, the slides were washed 2×15 min in 50% formamide in 2×SSC at 37° C., and then twice in 2×SSC for 30 min prior to washing at high stringency 3×5 min in 0.1×SSC at 60° C. in a shaking waterbath. The slides were blocked with 200 microliters/slide of 4×SSC and 10% fish skin gelatin (Cat. No. G-7765, Sigma, St. Louis, Mo.) for 30 minutes and drained (blocking step). The slides were then incubated with 100 microliters/slide avidin-Texas red (Vector Labs, Burlingame, Calif.) diluted 1:1000 in 4×SSC plus 2% fish skin gelatin for 30 minutes at 37° C. in a moist dark chamber followed by an additional 30 min incubation at 37° C. with biotinylated goat anti-avidin (Vector) diluted 1:100 as before from 1 mg/ml stock (amplification step). Slides were washed 3×5 min and blocked as before. The slides were then incubated with 100 microliters/slide avidin-Texas red diluted 1:1000 in 4×SSC plus 2% fish skin gelatin (FSG) for 30 minutes at 37° C. Slides are washed and blocked as before followed by an additional incubation for 30 minutes at 37° C. with biotinylated goat anti-avidin (Vector) diluted 1:100 in 4×SSC plus 2% FSG. Slides were washed in 4×SSC and 0.1% Tween 40 three times, blocked with fish skin gelatin and drained. The slides were then incubated with 100 microliters/slide avidin-Texas red diluted 1:1000 in 4×SSC plus 2% fish skin gelatin for 30 minutes at 37° C. The slides were stained with DAPI (Sigma) in McIlvane's buffer, rinsed and drained as suggested by the manufacturer. The slides were mounted in anti-fade.</p>
    <p>For probes containing digoxygenin, the signal may be amplified with mouse anti-digoxygenin (Sigma). During hybridization with both biotin-bound probes and digoxygenin bound probes the amplification can occur simultaneously. The slides are washed three times in 4×SSC and 0.1% Tween 40, blocked with fish skin gelatin and drained. Then the digoxygenin signal is fluorescently labeled with 100 microliters/slide rabbit anti-mouse FITC (Sigma). The slides are washed again three times in 4×SSC and 0.1% Tween 40, blocked with fish skin gelatin and drained. The slides are further amplified with goat anti-rabbit FITC (Sigma). The slides are washed again in 4×SSC and 0.1% Tween 40 three times and drained. The slides are stained with DAPI (Sigma) in McIlvane's buffer, rinsed, and drained. The slides are mounted in anti-fade.</p>
    <p>The slides are analyzed by standard techniques of fluorescent microscopy such as described in Lebo et al, Hum. Genet. 88:13-20, 1991, incorporated herein by reference. Preferably photomicrographs are taken of entire chromosome complements including complete karyotypes and whole interphase nuclei. Briefly, each slide is observed in a microscope equipped with appropriate excitation filters, dichroic, and barrier filters. Filters are chosen based on the excitation and emission spectra of the fluorochromes used. Ektachrome 400 color slide film is exposed different lengths of time depending upon the label, signal intensity, and filter chosen. Typically Texas red signal is photographed 30 times longer than blue DAPI signal. Multiple single color or dual color exposures using the proper filters for the chosen labels are performed with the slides left in place on the microscope stage when the filters are changed.</p>
    <p>Gene multiplications are identified by observing a pattern of hybridized probes on a chromosome complement. The probe which is homologous to a region adjacent (adjacent probe) to the multiplied gene provides a marker for the proper gene position. In a normal chromosome the signal for the probe which is homologous to the gene (gene probe) is close to the signal for the adjacent probe. See FIGS. 2A and 2C. If the gene is multiplied, there will be more than one signal for the gene probe close to the signal for the adjacent probe. See FIGS. 2B and 2D. If the gene is not accessible for hybridization because of chromosome breakage, chromatin-bound proteins, or other factors, neither probe will hybridize. Non-specific binding of the gene probe will be identified by gene probe signal without adjacent probe signal.</p>
    <p>Gene deletions are detected in a similar manner. Signals of the adjacent probe identify chromosome complements able to bind the gene probe when the gene is present. A normal Chromosome is identified by an adjacent probe signal lying close to a gene probe signal. See FIG. 2C. If the gene is deleted, the adjacent probe signal is observed alone without a close gene probe signal. See FIG. 2E. If the gene is not accessible for hybridization, neither the adjacent probe nor the gene probe is detected, controlling for false positives.</p>
    <p>Chromosome haplotypes are identified by the correlation of detected labels with the respective probes. Preferably an adjacent probe is hybridized to the chromosome to identify the haplotype region. The presence or absence of the adjacent probe provides a control for false negatives and positives as above. Once a hybridized haplotype probe has been detected, the color of the label is correlated with the probe and the haplotype is identified (FIGS. 2F and 2G).</p>
    <heading>EXAMPLES</heading> <heading>Example I</heading> <p>Preparation of Direct Amniocytes on Slides for In Situ Hybridization:</p>
    <p>Five milliliters of amniotic fluid was collected in a 15 ml polypropylene tube. The tube was centrifuged at 200 g for 7 minutes, the supernatant was removed and the cell pellet was gently resuspended by tapping. A hypotonic solution (1:1::0.2% KCl:0.2% NaCitrate) was added dropwise to a total volume of 1-2 ml while gently mixing. The solution was incubated in a 37° C. waterbath for 8 minutes and then immediately centrifuged at 200 g for 7 minutes. The supernatant was removed and the cell pellet was dissociated and fixed in 1-2 ml Carnoy's solution (3:1::methanol: glacial acetic acid) added dropwise with gentle tapping. After 15 minutes the solution was centrifuged at 200 g for 7 minutes. The supernatant was removed and the cell pellet was dissociated and fixed in 1-2 ml Carnoy's solution (3:1::methanol: glacial acetic acid) added dropwise with gentle tapping. Precleaned slides stored in 100% methanol were chilled in an ice slurry and fixed cells were added dropwise to the slides. Slides were then air-dried and examined under a phase contrast microscope to assure an adequate number of cells.</p>
    <p>Suitable slides were baked for 2 hours at 60° C. on a slide warmer. Just prior to hybridization the slides were rehydrated in proteinase K buffer (2 mM tris-HCl (pH 7.5) plus 0.2 mM CaCl<sub>2</sub>), treated with proteinase K (100 pg\ml in proteinase K buffer prepared as before; 100 microliters/slide) for 8 minutes at 37° C., washed for 5 minutes at 23° C. in PBS plus 50 mM MgCl<sub>2</sub>. The slides were then fixed for 10 minutes at 23° C. in 1% formaldehyde in PBS plus 50 mM MgCl<sub>2</sub>. The slides were washed twice for 3 minutes at 23° C. in PBS and dehydrated in an ethanol series. The slides were then air dried, denatured for 5 minutes at 70° C. in 2×SSC, 70% formamide, 50 mM NaPO4 (pH 7.0) chilled in 70% ethanol at 0° C. dehydrated in ethanol, and air dried.</p>
    <heading>Example II</heading> <p>Preparation of Direct Chorionic Villus Cells on Slides for In Situ Hybridization.</p>
    <p>Chorionic villi were dissected from maternal decidua and minced with a scalpel. Cells in minced tissue were dissociated with pronase (85 U/ml) at 37° C. for 20 minutes. The dissociated cells were then centrifuged at 200 g for seven minutes. The supernatant was removed and the cells were prepared according to the protocol of Example I using 1:1::0.2% sodium citrate:0.2% KCl as the hypotonic solution.</p>
    <heading>Example III</heading> <p>Preparation of Cultured Amniocytes on Slides for In Situ Hybridization.</p>
    <p>Cultured amniocytes were separated from culture dishes by 0.5 g/L trypsin, 0.2 g/L EDTA, 1.0 g/L glucose, 0.58 g/L NaHCO<sub>3</sub> digestion at 23° C. for 1-5 minutes. Trypsin is neutralized and the cells removed with culture medium plus 10% fetal calf serum. Harvested cells were pelleted by centrifugation at 200 g for 7 minutes. The supernatant was removed and the cells were prepared according to protocol of Example I using 1:1::0.4% sodium citrate:0.4% KCl as the hypotonic solution.</p>
    <heading>Example IV</heading> <p>Preparation of Cultured Chorionic Villus Cells on Slides for in Situ Hybridization.</p>
    <p>Cultured chorionic villus cells were harvested from culture dishes by trypsin as in Example III. Harvested cells were pelleted by centrifugation at 200 g for 7 minutes. The supernatant was removed and the cells were prepared according to the protocol of Example I using 1:3::Hanks PBS:H<sub>2</sub> O as the hypotonic solution.</p>
    <heading>Example V</heading> <p>Preparation of Preimplantation Blastocysts on Slides for In Situ Hybridization.</p>
    <p>All embryo manipulation is under a dissecting microscope at 50X magnification. Embryos are grown in Ham F10 medium using maternal serum in 10 cm tissue culture dishes. The embryo is transferred in a siliconized, flame-pulled pasteur pipette with a cotton plug into a 2 well siliconized slide into Ham F10 medium without serum. Then the embryo is transferred into Ham F10 with 1% BSA titrated to pH 2-2.5 by adding 10N HCl. After all of visible zona pellucida is dissolved, the embryo is transferred into phosphate buffered without calcium or magnesium (Hanks). Then the embryo is transferred into 1% sodium citrate hypotonic solution for 2 min, transferred to the slide to be tested with teflon-surrounded wells, and fixed to the slide by adding dropwise Carnoy fix (3:1::methanol:glacial acetic acid). The position of the embryo is marked on the bottom of the slide with a diamond tip pencil. Dropwise fixation continues until the embryo disappears. Then the slide is fixed overnight in Carnoy's fixative in a capped 50 ml Corning polypropylene tube. Subsequent preparation and in situ hybridization is completed according to cultured amniocyte protocols except that fixed cells are treated with approximately 6-fold more proteinase K to remove extra protein. Proteinase K treatment varies according to the number of blastocysts treated, how much zona pellucida remains, the size of the blastocysts which is usually dependent upon the number of divisions completed in the embryo, and the health of the cells tested.</p>
    <heading>Example VI</heading> <p>Detection of Gene Deletion Resulting in Steroid Sulfatase Deficiency.</p>
    <p>Steroid sulfatase deficiency is an X-linked genetic disorder resulting form deletion of the steroid sulfatase gene on the X chromosome. Multicolor in situ hybridization was performed to identify normal, carrier and disease states.</p>
    <p>Cosmid CY437 carries a 40kb insert homologous to a DNA sequence located about 100kb 5' to the steroid sulfatase gene on the X chromosome. Only X chromosomes having the steroid sulfatase gene hybridize to cosmid CY437. X chromosomes lacking the steroid sulfatase gene (i.e. from carrier or disease states) do not hybridize to cosmid CY437.</p>
    <p>Cosmid CS12 carries a 32kb Y chromosome derived flanking insert homologous to a nucleic acid sequences on both the X and Y chromosomes. The homologous X chromosome-specific sequence flanking the steroid sulfatase gene is not deleted in X-linked ichthyosis patients when the steroid sulfatase gene has been deleted. Hybridization of CS12 accurately identifies all X and Y chromosomes and facilitates detection of specific cosmid CY437 signal.</p>
    <p>Cosmid CY437 was labeled with bio-11-dUTP (ENZO Diagnostics, New York, N.Y.). Cosmid CS12 was labeled with digoxygenin 11-dUTP (Boehringer-Mannheim, Mannheim, Germany). Both probes were labeled by nick translation along with 3H-dGTP tracer with DNAse in a concentration of 2 ng/microliter hybridization mix. Labeled probes were separated from unlabeled DNA with G-50 Sephadex spin columns equilibrated with 150 mM NaCl, 15 mM NaCitrate, and 0.1% SDS (pH 7.0). The labeled probes averaged 200-500 base pairs long and had 30%-60% 3H-dGTP carrier incorporated. Labeled probes with unique and repetitive DNA sequences were precipitated with a fifty-fold excess of salmon sperm carrier DNA and of yeast tRNA, and 100 to 500-fold excess of depurinated human placental DNA (Sigma, St. Louis, Mo.). The precipitated DNA was washed in 70% ethanol and dried in a Savant speed vac. The dried DNA pellets were redissolved to a concentration of 4 ng/ml in 100% deionized formamide and mixed to a final concentration of 1×SSC in 10% dextran sulfate and 50 mM NaPO<sub>4</sub> (pH 7.0). The resulting dissolved DNA was denatured at 95° C. in a heat block and prehybridized for 4 hours at 65° C. prior to adding to the slide.</p>
    <p>Amniocytes were prepared and fixed to slides as in Example 1 above. Labeled probes were added to the slides at a concentration of 4 ng/microliter and the slides inverted. The probes and samples were hybridized overnight at 37° C. Following overnight hybridization, the slides were washed in 50% formamide in 2×SSC and then in 2×SSC prior to washing at high stringency in 0.1×SSC at 60° C. The slides were blocked with 200 microliters/slide of 4×SSC and 10% fish skin gelatin (Cat. No. G-7765, Sigma, St. Louis, Mo.) for 30 minutes and drained (blocking step). The slides were then incubated with 100 microliters/slide avidin-Texas red diluted 1:1000 in 4×SSC plus 2% fish skin gelatin for 30 minutes at 37° C., washed, blocked, and incubated for 30 minutes at 37° C. with biotinylated goat anti-avidin (Vector) diluted 1:100 in 4×SSC; 2% FSG (amplification step). Slides were washed in 4×SSC and 0.1% Tween 40 three times, blocked with fish skin gelatin and drained (washing step). The blocking step, amplification step, and washing step were repeated.</p>
    <p>A Zeiss photomicroscope was used to take photomicrographs with Kodacolor 400 ASA film (Eastman-Kodak, Rochester, N.Y.). DAPI stained interphase nuclei and banded metaphase chromosomes were photographed through a Hoechst filter set [excitation 360 WB50 #9008, dichroic 470 #8902, emission 470 DF65 #8902]. Fluorescein and rhodamine signals were photographed either separately through label specific filter sets or simultaneously through a fluorescein-Texas red filter set or a rhodamine-fluorescein filter set from Omega Optical, Battleboro, Vt. The FITC filter set specifications were excitation 480 DF30 #8902, dichroic 505DRLP #8902, emission 535 DK45 #8902. The Texas red filter set specifications were excitation 560 DF40 #8902, dichroic 595 DRLP #8902, emission 635 DF60 #8902.</p>
    <p>Eighty to ninety percent of tested samples revealed two fluorescein-labeled CS12 hybridization spots. Cells exhibiting two green (CS12) spots (indicating hybridization of both sex chromosomes) were scored for the number of red (CY437) spots to detect the steroid sulfatase gene. It was possible to distinguish affected males from normal males, carrier females, and normal females.</p>
    <p>All control males and normal males revealed one red (CY437) spot. Three of 43 cells from affected males showed weak red (CY437) spots. When compared to normal males and females the dim character of the red (CY437) spots in the affected males was clearly differentiated. Four of forty four carrier female cells showed two red (CY437) spots. All other female cells showed 1 red (CY437) spots. Two of 20 normal female cells had one red (CY437) spot. The remaining females had 2 red (CY437) spots.</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">              TABLE 1______________________________________       RED: CY437 (X)       No. ScoredSLIDE DESIGNATION         0      1     2    CLINICAL STATUS______________________________________Control       0      20    0    normal maleUnknown #1    20     2     0    affected maleUnknown #2    0      2     20   normal femaleUnknown #3    20     1     0    affected maleUnknown #4    0      20    0    carrier femaleUnknown #5    0      20    4    carrier femaleUnknown #6    0      20    0    normal male______________________________________</pre>
    
    <heading>Example VII</heading> <p>Detection of Gene Multiplication Resulting in Charcot-Marie-Tooth Disease Type 1A.</p>
    <p>CMT 1A is a disease associated with duplication of a gene on the short arm of chromosome 17. Multicolor in situ hybridization was performed prenatally to identify carrier and affected individuals. Cells were fixed on slides according to the procedures of Examples I-IV. Cosmid probe 6G1 mapped to the most commonly duplicated CMT1A gene region is the "gene" probe and centromeric probe D17Z1 was the flanking control probe.</p>
    <p>Centromeric probe D17Z1 was labeled with bio-11-dUTP (ENZO Diagnostics, New York, N.Y.). Cosmid 6G1 probe was labeled with digoxygenin 11-dUTP (Boehringer-Mannheim, Mannheim, Germany). Both probes were labeled by nick translation along with 3H-dGTP tracer with DNAse in a concentration of 2 ng/50 microliter reaction mix. Labeled probes were separated from unlabeled DNA with G-50 Sephadex spin columns equilibrated with 150 mM NaCl, 15 mM NaCitrate, and 0.1% SDS (pH 7.0). The labeled probes averaged 200-500 base pairs long and had 30%-60% 3H-dGTP content. Labeled probes which contained unique DNA sequences were precipitated with a fifty-fold excess of salmon sperm carrier DNA and yeast tRNA, and to probes like whole cosmids with some unique and some repetitive sequences an additional 100 to 500-fold excess of depurinated human placental DNA (Sigma, St. Louis, Mo.) was added. The precipitated DNA was washed in 70% ethanol and dried in a Savant speed vac. The dried DNA pellets were redissolved to a concentration of 4 ng/microliter in 100% deionized formamide and mixed to a final concentration of 1×SSC in 10% dextran sulfate and 50 mM NaPO4 (pH 7.0). The resulting dissolved DNA was denatured in hybridization mix and probes with unique and repetitive sequences were prehybridized 4 hours at 65° C. to preanneal the repetitive labeled sequences to unlabeled placental DNA repeats. Prehybridization was skipped for moderately repetitive probes like chromosome-specific repetitive sequences.</p>
    <p>Preannealed probe in hybridization solution was added to coverslips. Inverted slides were added to the coverslips, placed on slide racks, hybridized overnight at 37° C. Following overnight hybridization, the slides were washed in 50% formamide in 2×SSC and then in 2×SSC prior to washing at high stringency in 0.1×SSC at 60° C. The slides were blocked with 200 microliters/slide of 4×SSC and 10% fish skin gelatin (Cat. No. G-7765, Sigma, St. Louis, Mo.) for 30 minutes and drained (blocking step). The slides were then incubated with 100 microliters/slide avidin-Texas red diluted 1:1000 in 4×SSC plus 2% fish skin gelatin for 30 minutes at 37° C., washed, blocked, and incubated for 30 minutes at 37° C. with biotinylated goat anti-avidin (Vector) diluted 1:100 in 4×SSC; 2% FSG (amplification step). Slides were washed three times in 4×SSC and 0.1% Tween 40, blocked with fish skin gelatin and drained (washing step). The blocking step, amplification step, and washing step were repeated.</p>
    <p>A Zeiss photomicroscope was used to take photomicrographs with Kodacolor 400 ASA film (Eastman-Kodak, Rochester, N.Y.). DAPI stained interphase nuclei and banded metaphase chromosomes were photographed through a Hoechst filter set [excitation 360 WB50 #9008, dichroic 470 #8902, emission 470 DF65 #8902]. Fluorescein and rhodamine signals were photographed either separately through label specific filter sets or simultaneously through a fluorescein-Texas red filter set or a rhodamine-fluorescein filter set from Omega Optical, Battleboro, Vt. The FITC filter set specifications were excitation 480 DF30 #8902, dichroic 505DRLP #8902, emission 535 DK45 #8902. The Texas red filter set specifications were excitation 560 DF40 #8902, dichroic 595 DRLP #8902, emission 635 DF60 #8902.</p>
    <p>A pregnant woman was shown to have CMT1A gene duplication by RFLP analysis. Fetal cells were tested for CMT1A gene duplication by RLLP analysis. The fetal DNA was uninformative for four RFLP's with three polymorphic DNA probes. Multicolor in situ hybridization clearly demonstrated a duplication of the CMT1A gene only 48 hours after receiving the fetal cells.</p>
    <heading>Example VIII</heading> <p>Prenatal Identification of Chromosomal Haplotypes</p>
    <p>Differences of 5% sequence homology between the FCG2 and FCG3 genes can be distinguished by high stringency multicolor in situ hybridization (Lebo et al, 1991). Differences in HLA allele introns differ by 5% (Genetype Corp., International HLA Meeting, Bombay, 1991). These data anticipate that in situ hybridization to the dozen or so specific HLA haplotypes with haplotype-specific probes will distinguish different alleles. Whether introns at other standard gene loci differ by 5% is unknown because selection may require more heterogeneity at the HLA locus. Nevertheless the number of haplotypes at the beta-globin locus is less than the anticipated number of randomly mixed polymorphic haplotypes so that all beta-thalassemia mutations will have arisen on one of these subset of existing haplotypes [Orkin et al., Nature 296:627-631, (1982)]. Nevertheless, flanking gene regions are anticipated to be under less selection pressure so that sequences within 0.3 Mb that combine with an allele less than 1% per meiosis could be tested to follow disease allele segregation in any family. For diseases like Huntington's disease which for which more than 95% probably arose from a single founder gene mutation, a single disease-specific probe will be sufficient to diagnose nearly all at-risk patients. For diseases with a few founder mutations like sickle cell anemia, a few allele-specific probes will distinguish most mutant alleles. Other diseases resulted from multiple severe mutations at the same locus: &gt;120 cystic fibrosis mutations, &gt;80 hemophilia A mutations, one-third of all Duchenne muscular dystrophy mutations are new mutations. For these diseases a limited number of probes against the small subset of founder haplotypes like those found at the HLA and beta-globin loci are expected to distinguish nearly all alleles in all families. Thus in situ hybridization is a unique way to test multiple neutral mutation sites instead of testing one RFLP site at a time with PCR amplification and digestion or restriction enzyme analysis. Which alleles are in each subject can be determined by multiple color labels attached one to each different allele and tested on enough slides to test each haplotype. The more different colored, sensitive, distinguishable fluorochrome schemes become available, the fewer the required slide number to identify affected haplotypes. Segregation of parental haplotypes ware followed by color labeling as demonstrated in FIG. 1 for autosomal dominant disease, and FIG. 3 for autosomal recessive genetic disease.</p>
    </div></div></div><div class="patent-section patent-tabular-section"><a id="backward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Cited Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4888278">US4888278</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Oct 13, 1988</td><td class="patent-data-table-td patent-date-value">Dec 19, 1989</td><td class="patent-data-table-td ">University Of Massachusetts Medical Center</td><td class="patent-data-table-td ">Contacting with non-homopolymer labelled probe of nucleic acid fragments</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="npl-citations"></a><div class="patent-section-header"><span class="patent-section-title">Non-Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th colspan="3"class="patent-data-table-th">Reference</th></tr></thead><tr><td class="patent-data-table-td ">1</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Ballabio et al. PNAS 84:4519 4523, 1987.</td></tr><tr><td class="patent-data-table-td ">2</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Ballabio et al. PNAS 84:4519-4523, 1987.</td></tr><tr><td class="patent-data-table-td ">3</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Ballabio et al., "<a href='http://scholar.google.com/scholar?q="Isolation+and+characterization+of+a+steroid+sulfatase+cDNA+clone%3A+Genomic+deletions+in+patients+with+X-chromosome-linked+ichthyosis"'>Isolation and characterization of a steroid sulfatase cDNA clone: Genomic deletions in patients with X-chromosome-linked ichthyosis</a>", Proc. Natl. Acad. Sci., 84:4519-4523 (Jul. 1987).</td></tr><tr><td class="patent-data-table-td ">4</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Ballabio et al., Isolation and characterization of a steroid sulfatase cDNA clone: Genomic deletions in patients with X chromosome linked ichthyosis , Proc. Natl. Acad. Sci., 84:4519 4523 (Jul. 1987).</td></tr><tr><td class="patent-data-table-td ">5</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Cong et al., "<a href='http://scholar.google.com/scholar?q="Assignment+of+Human+Tracheobronchial+Mucin+Gene%28s%29+to+11p15+and+a+Tracheobronchial+Mucin-related+Sequence+to+Chromosome+13"'>Assignment of Human Tracheobronchial Mucin Gene(s) to 11p15 and a Tracheobronchial Mucin-related Sequence to Chromosome 13</a>" Hum. Genet. 86:167-172 (1990).</td></tr><tr><td class="patent-data-table-td ">6</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Cong et al., Assignment of Human Tracheobronchial Mucin Gene(s) to 11p15 and a Tracheobronchial Mucin related Sequence to Chromosome 13 Hum. Genet. 86:167 172 (1990).</td></tr><tr><td class="patent-data-table-td ">7</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Greenberg et al., "<a href='http://scholar.google.com/scholar?q="Molecular+Analysis+of+the+Smith-Magenis+Syndrome%3A+A+Possible+Contiguous-Gene+Syndrome+Associated+with+del%2817%29+%28pll.2%29"'>Molecular Analysis of the Smith-Magenis Syndrome: A Possible Contiguous-Gene Syndrome Associated with del(17) (pll.2)</a>" Am. J. Hum. Genet. 49:1207-1218 (1991).</td></tr><tr><td class="patent-data-table-td ">8</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Greenberg et al., "<a href='http://scholar.google.com/scholar?q="Molecular+Analysis+of+the+Smith-Magenis+Syndrome%3A+A+Possible+Contiguous-Gene+Syndrome+Associated+with+del%2817%29%28pll.2%29"'>Molecular Analysis of the Smith-Magenis Syndrome: A Possible Contiguous-Gene Syndrome Associated with del(17)(pll.2)</a>" Am. J. Hum. Genet. 49:1207-1218 (1991).</td></tr><tr><td class="patent-data-table-td ">9</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Greenberg et al., Molecular Analysis of the Smith Magenis Syndrome: A Possible Contiguous Gene Syndrome Associated with del(17) (pll.2) Am. J. Hum. Genet. 49:1207 1218 (1991).</td></tr><tr><td class="patent-data-table-td ">10</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Greenberg et al., Molecular Analysis of the Smith Magenis Syndrome: A Possible Contiguous Gene Syndrome Associated with del(17)(pll.2) Am. J. Hum. Genet. 49:1207 1218 (1991).</td></tr><tr><td class="patent-data-table-td ">11</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Grundy et al., "<a href='http://scholar.google.com/scholar?q="The+Polymorphic+FcT+Receptor+II+Gene+Maps+to+Human+Chromosome+1q"'>The Polymorphic FcT Receptor II Gene Maps to Human Chromosome 1q</a>" Immunogenetics 29:331-339 (1989).</td></tr><tr><td class="patent-data-table-td ">12</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Grundy et al., The Polymorphic FcT Receptor II Gene Maps to Human Chromosome 1q Immunogenetics 29:331 339 (1989).</td></tr><tr><td class="patent-data-table-td ">13</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Grundy, et al.., Immunogenetics, 29:331 339 (1989).</td></tr><tr><td class="patent-data-table-td ">14</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Grundy, et al.., Immunogenetics, 29:331-339 (1989).</td></tr><tr><td class="patent-data-table-td ">15</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Harper et al., "<a href='http://scholar.google.com/scholar?q="Localization+of+Single+Copy+DNA+Sequences+on+G-Banded+Human+Chromosomes+by+in+situ+Hybridization"'>Localization of Single Copy DNA Sequences on G-Banded Human Chromosomes by in situ Hybridization</a>" Chromosoma 83:431-439 (1981).</td></tr><tr><td class="patent-data-table-td ">16</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Harper et al., Localization of Single Copy DNA Sequences on G Banded Human Chromosomes by in situ Hybridization Chromosoma 83:431 439 (1981).</td></tr><tr><td class="patent-data-table-td ">17</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Harper, et al., Chromosoma, 83:431 439 (1981).</td></tr><tr><td class="patent-data-table-td ">18</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Harper, et al., Chromosoma, 83:431-439 (1981).</td></tr><tr><td class="patent-data-table-td ">19</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Koenig et al. Cell 50:509 517, 1987.</td></tr><tr><td class="patent-data-table-td ">20</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Koenig et al. Cell 50:509-517, 1987.</td></tr><tr><td class="patent-data-table-td ">21</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Koenig et al., "<a href='http://scholar.google.com/scholar?q="Complete+Cloning+of+the+Duchenne+Muscular+Dystrophy+%28DMD%29+cDNA+and+Preliminary+Genomic+Organization+of+the+DMA+Gene+in+Normal+and+Affected+Individuals"'>Complete Cloning of the Duchenne Muscular Dystrophy (DMD) cDNA and Preliminary Genomic Organization of the DMA Gene in Normal and Affected Individuals</a>", Cell 50:509-517 (Jul. 31, 1987).</td></tr><tr><td class="patent-data-table-td ">22</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Koenig et al., Complete Cloning of the Duchenne Muscular Dystrophy (DMD) cDNA and Preliminary Genomic Organization of the DMA Gene in Normal and Affected Individuals , Cell 50:509 517 (Jul. 31, 1987).</td></tr><tr><td class="patent-data-table-td ">23</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Kwiatkowski et al. Am. J. Human Genet. 46:559 567, 1990.</td></tr><tr><td class="patent-data-table-td ">24</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Kwiatkowski et al. Am. J. Human Genet. 46:559-567, 1990.</td></tr><tr><td class="patent-data-table-td ">25</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Kwiatkowski et al., "<a href='http://scholar.google.com/scholar?q="Identification+of+the+functional+prolifin+gene%2C+its+localization+to+chromosome+subband+17p13.3%2C+and+demonstration+of+its+deletion+in+some+patients+with+Miller-Dieker+syndrome"'>Identification of the functional prolifin gene, its localization to chromosome subband 17p13.3, and demonstration of its deletion in some patients with Miller-Dieker syndrome</a>", Am. J. Hum. Genet. 46:559-567 (1990).</td></tr><tr><td class="patent-data-table-td ">26</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Kwiatkowski et al., Identification of the functional prolifin gene, its localization to chromosome subband 17p13.3, and demonstration of its deletion in some patients with Miller Dieker syndrome , Am. J. Hum. Genet. 46:559 567 (1990).</td></tr><tr><td class="patent-data-table-td ">27</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Langer Safer et al., Immunological Method for Mapping Genes on Drosophila Polytene Chromosomes Proc. Natl. Acad. Sci. USA 79:4381 4385 (Jul. 1982).</td></tr><tr><td class="patent-data-table-td ">28</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Langer Safer, et al., Proc. Natl. Acad. Sci. USA, 79:4381 4385 (Jul., 1982).</td></tr><tr><td class="patent-data-table-td ">29</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Langer-Safer et al., "<a href='http://scholar.google.com/scholar?q="Immunological+Method+for+Mapping+Genes+on+Drosophila+Polytene+Chromosomes"'>Immunological Method for Mapping Genes on Drosophila Polytene Chromosomes</a>" Proc. Natl. Acad. Sci. USA 79:4381-4385 (Jul. 1982).</td></tr><tr><td class="patent-data-table-td ">30</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Langer-Safer, et al., Proc. Natl. Acad. Sci. USA, 79:4381-4385 (Jul., 1982).</td></tr><tr><td class="patent-data-table-td ">31</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Lathrop et al., "<a href='http://scholar.google.com/scholar?q="Strategies+for+Multilocus+Linkage+Analysis+in+Humans"'>Strategies for Multilocus Linkage Analysis in Humans</a>" Proc. Natl. Acad. Sci. USA 81:3443-3446 (Jun. 1984).</td></tr><tr><td class="patent-data-table-td ">32</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Lathrop et al., Strategies for Multilocus Linkage Analysis in Humans Proc. Natl. Acad. Sci. USA 81:3443 3446 (Jun. 1984).</td></tr><tr><td class="patent-data-table-td ">33</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Lathrop, et al., Proc. Natl. Acad. Sci. USA, 81:3443 3446 (Jun., 1984).</td></tr><tr><td class="patent-data-table-td ">34</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Lathrop, et al., Proc. Natl. Acad. Sci. USA, 81:3443-3446 (Jun., 1984).</td></tr><tr><td class="patent-data-table-td ">35</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Lebo et al , "<a href='http://scholar.google.com/scholar?q="Chromosome+1+Charcot-Marie-Tooth+Disease+%28CMT1B%29+Locus+in+the+FcT+Receptor+Gene+Region"'>Chromosome 1 Charcot-Marie-Tooth Disease (CMT1B) Locus in the FcT Receptor Gene Region</a>" Hum. Genet. 88:1-12 (1991).</td></tr><tr><td class="patent-data-table-td ">36</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Lebo et al , Chromosome 1 Charcot Marie Tooth Disease (CMT1B) Locus in the FcT Receptor Gene Region Hum. Genet. 88:1 12 (1991).</td></tr><tr><td class="patent-data-table-td ">37</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Lebo et al., "<a href='http://scholar.google.com/scholar?q="Defining+DNA+Diagnostic+Tests+Appropriate+for+Standard+Clinical+Care"'>Defining DNA Diagnostic Tests Appropriate for Standard Clinical Care</a>" Am. J. Hum. Genet. 47:583-590 (1990).</td></tr><tr><td class="patent-data-table-td ">38</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Lebo et al., "<a href='http://scholar.google.com/scholar?q="Multicolor+Fluorescence+in+situ+Hybridization+and+Pulsed+Field+Electrophoresis+Dissect+CMT1B+Gene+Region"'>Multicolor Fluorescence in situ Hybridization and Pulsed Field Electrophoresis Dissect CMT1B Gene Region</a>" Genetics Springer-Verlag pp. 1-8 (1991).</td></tr><tr><td class="patent-data-table-td ">39</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Lebo et al., "<a href='http://scholar.google.com/scholar?q="Multicolor+Fluorescence+in+situ+Hybridization+and+Pulsed+Field+Electrophoresis+Dissect+CMT1B+Gene+Region"'>Multicolor Fluorescence in situ Hybridization and Pulsed Field Electrophoresis Dissect CMT1B Gene Region</a>" Hum. Genet. 88:13-20 (1991).</td></tr><tr><td class="patent-data-table-td ">40</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Lebo et al., "<a href='http://scholar.google.com/scholar?q="Multicolor+in+situ+Hybridization+Prenatally+Diagnoses+X-Linked+Ichthyosis"'>Multicolor in situ Hybridization Prenatally Diagnoses X-Linked Ichthyosis</a>" Am. CytoGenet. Conf. Abstract (1991).</td></tr><tr><td class="patent-data-table-td ">41</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Lebo et al., "<a href='http://scholar.google.com/scholar?q="Multicolor+in-situ+Hybridization+and+Linkage+Analysis+Order+Charcot-Marie-Toot+Type+1+%28CMTIA%29+Gene-Region+Markers"'>Multicolor in-situ Hybridization and Linkage Analysis Order Charcot-Marie-Toot Type 1 (CMTIA) Gene-Region Markers</a>" Am. J. Hum. Genet. 50:42-55 (1992).</td></tr><tr><td class="patent-data-table-td ">42</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Lebo et al., "<a href='http://scholar.google.com/scholar?q="Rare+McArdle+Disease+Locus+Polymorphic+site+on+11q13+contains+CpG+Sequence"'>Rare McArdle Disease Locus Polymorphic site on 11q13 contains CpG Sequence</a>" Hum. Genet. 86:17-24 (1990).</td></tr><tr><td class="patent-data-table-td ">43</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Lebo et al., Defining DNA Diagnostic Tests Appropriate for Standard Clinical Care Am. J. Hum. Genet. 47:583 590 (1990).</td></tr><tr><td class="patent-data-table-td ">44</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Lebo et al., Multicolor Fluorescence in situ Hybridization and Pulsed Field Electrophoresis Dissect CMT1B Gene Region Genetics Springer Verlag pp. 1 8 (1991).</td></tr><tr><td class="patent-data-table-td ">45</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Lebo et al., Multicolor Fluorescence in situ Hybridization and Pulsed Field Electrophoresis Dissect CMT1B Gene Region Hum. Genet. 88:13 20 (1991).</td></tr><tr><td class="patent-data-table-td ">46</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Lebo et al., Multicolor in situ Hybridization and Linkage Analysis Order Charcot Marie Toot Type 1 (CMTIA) Gene Region Markers Am. J. Hum. Genet. 50:42 55 (1992).</td></tr><tr><td class="patent-data-table-td ">47</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Lebo et al., Multicolor in situ Hybridization Prenatally Diagnoses X Linked Ichthyosis Am. CytoGenet. Conf. Abstract (1991).</td></tr><tr><td class="patent-data-table-td ">48</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Lebo et al., Rare McArdle Disease Locus Polymorphic site on 11q13 contains CpG Sequence Hum. Genet. 86:17 24 (1990).</td></tr><tr><td class="patent-data-table-td ">49</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Lebo, et al., Am. CytoGenet. Conf., Abstract (1991).</td></tr><tr><td class="patent-data-table-td ">50</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Lebo, et al., Am. J. Hum. Genet., 47:583 590 (1990).</td></tr><tr><td class="patent-data-table-td ">51</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Lebo, et al., Am. J. Hum. Genet., 47:583-590 (1990).</td></tr><tr><td class="patent-data-table-td ">52</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Lebo, et al., Am. J. Hum. Genet., 50:42 55 (1992).</td></tr><tr><td class="patent-data-table-td ">53</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Lebo, et al., Am. J. Hum. Genet., 50:42-55 (1992).</td></tr><tr><td class="patent-data-table-td ">54</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Lebo, et al., Hum. Genet., 86:17 24 (1990).</td></tr><tr><td class="patent-data-table-td ">55</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Lebo, et al., Hum. Genet., 86:17-24 (1990).</td></tr><tr><td class="patent-data-table-td ">56</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Lebo, et al., Hum. Genet., 88:13 (1991).</td></tr><tr><td class="patent-data-table-td ">57</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Lebo, et al., Hum. Genet., 88:13 20 (1991).</td></tr><tr><td class="patent-data-table-td ">58</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Lebo, et al., Hum. Genet., 88:13-20 (1991).</td></tr><tr><td class="patent-data-table-td ">59</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Li et al., "<a href='http://scholar.google.com/scholar?q="Characterization+of+a+low+copy+repetitive+element+S232+Involved+in+the+Generation+of+Frequent+Deletions+of+the+Distal+Short+Arm+of+the+Human+X+Chromosome"'>Characterization of a low copy repetitive element S232 Involved in the Generation of Frequent Deletions of the Distal Short Arm of the Human X Chromosome</a>" Nuc. Acids Res. 20:1117-1122 (1992).</td></tr><tr><td class="patent-data-table-td ">60</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Li et al., Characterization of a low copy repetitive element S232 Involved in the Generation of Frequent Deletions of the Distal Short Arm of the Human X Chromosome Nuc. Acids Res. 20:1117 1122 (1992).</td></tr><tr><td class="patent-data-table-td ">61</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Lichter et al., "<a href='http://scholar.google.com/scholar?q="Is+non-isotopic+in+situ+hybridization+finally+coming+of+age"'>Is non-isotopic in situ hybridization finally coming of age</a>" Nature 345:93-94 (1990).</td></tr><tr><td class="patent-data-table-td ">62</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Lichter et al., Is non isotopic in situ hybridization finally coming of age Nature 345:93 94 (1990).</td></tr><tr><td class="patent-data-table-td ">63</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Lupinski, et al., Cell, 68:219 232 (1991).</td></tr><tr><td class="patent-data-table-td ">64</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Lupinski, et al., Cell, 68:219-232 (1991).</td></tr><tr><td class="patent-data-table-td ">65</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Lupski et al. "<a href='http://scholar.google.com/scholar?q="DNA+Duplication+Associated+with+Charcot-Marie-Tooth+Disease+Type+1A"'>DNA Duplication Associated with Charcot-Marie-Tooth Disease Type 1A</a>"0 Cell 68:219-232 (Jul. 26, 1991).</td></tr><tr><td class="patent-data-table-td ">66</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Lupski et al. DNA Duplication Associated with Charcot Marie Tooth Disease Type 1A 0 Cell 68:219 232 (Jul. 26, 1991).</td></tr><tr><td class="patent-data-table-td ">67</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Lux et al., "<a href='http://scholar.google.com/scholar?q="Hereditary+spherocytosis+associated+with+deletion+of+human+erythrocyte+ankyrin+gene+on+chromosome"'>Hereditary spherocytosis associated with deletion of human erythrocyte ankyrin gene on chromosome</a>" Nature 345:736-739 (1990).</td></tr><tr><td class="patent-data-table-td ">68</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Lux et al., Hereditary spherocytosis associated with deletion of human erythrocyte ankyrin gene on chromosome Nature 345:736 739 (1990).</td></tr><tr><td class="patent-data-table-td ">69</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Lux, et al., Nature, 345:736 739 (1990).</td></tr><tr><td class="patent-data-table-td ">70</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Lux, et al., Nature, 345:736-739 (1990).</td></tr><tr><td class="patent-data-table-td ">71</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Mathews Analy Biochem 169:1 25 (1988).</td></tr><tr><td class="patent-data-table-td ">72</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Mathews Analy Biochem 169:1-25 (1988).</td></tr><tr><td class="patent-data-table-td ">73</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Nederlof et al., "<a href='http://scholar.google.com/scholar?q="Three-Color+Fluorescence+in+situ+Hybridization+for+the+Simultaneous+Detection+of+Multiple+Nucleic+Acid+Sequences"'>Three-Color Fluorescence in situ Hybridization for the Simultaneous Detection of Multiple Nucleic Acid Sequences</a>" Cytometry 10:20-27 (1989).</td></tr><tr><td class="patent-data-table-td ">74</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Nederlof et al., Three Color Fluorescence in situ Hybridization for the Simultaneous Detection of Multiple Nucleic Acid Sequences Cytometry 10:20 27 (1989).</td></tr><tr><td class="patent-data-table-td ">75</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Nederlof, et al., Cytometry, 10:20 27 (1989).</td></tr><tr><td class="patent-data-table-td ">76</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Nederlof, et al., Cytometry, 10:20-27 (1989).</td></tr><tr><td class="patent-data-table-td ">77</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Orkin et al. Nature 286:538 540, 1980.</td></tr><tr><td class="patent-data-table-td ">78</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Orkin et al. Nature 286:538-540, 1980.</td></tr><tr><td class="patent-data-table-td ">79</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Orkin et al., "<a href='http://scholar.google.com/scholar?q="Linage+of+%CE%B2-thalassaemia+Mutations+%26amp%3B+%CE%B2-globin+gene+polymorphisms+with+DNA+Polymorphisms+in+Human+%CE%B2-globin+Gene+Cluster"'>Linage of β-thalassaemia Mutations &amp; β-globin gene polymorphisms with DNA Polymorphisms in Human β-globin Gene Cluster</a>" Nature 296:627-631 (Apr. 15, 1982).</td></tr><tr><td class="patent-data-table-td ">80</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Orkin et al., "<a href='http://scholar.google.com/scholar?q="Partial+deletion+of+the+%CE%B1-globin+structural+gene+in+human+%CE%B1-thalassaemia"'>Partial deletion of the α-globin structural gene in human α-thalassaemia</a>", Nature, 286:538-540 (Jul. 31, 1980).</td></tr><tr><td class="patent-data-table-td ">81</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Orkin et al., Linage of thalassaemia Mutations &amp; globin gene polymorphisms with DNA Polymorphisms in Human globin Gene Cluster Nature 296:627 631 (Apr. 15, 1982).</td></tr><tr><td class="patent-data-table-td ">82</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Orkin et al., Partial deletion of the globin structural gene in human thalassaemia , Nature, 286:538 540 (Jul. 31, 1980).</td></tr><tr><td class="patent-data-table-td ">83</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Orkin et el., "<a href='http://scholar.google.com/scholar?q="Linage+of+%CE%B2-thalassaemia+Mutations+and+%CE%B2globin+gene+polymorphisms+with+DNA+Polymorphisms+in+Human+%CE%B2-globin+Gene+Cluster"'>Linage of β-thalassaemia Mutations and βglobin gene polymorphisms with DNA Polymorphisms in Human β-globin Gene Cluster</a>" Nature 296:627-631 (Apr. 15, 1982).</td></tr><tr><td class="patent-data-table-td ">84</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Orkin et el., Linage of thalassaemia Mutations and globin gene polymorphisms with DNA Polymorphisms in Human globin Gene Cluster Nature 296:627 631 (Apr. 15, 1982).</td></tr><tr><td class="patent-data-table-td ">85</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Patel et al. Am. J. Human Genet. 46:801 809, 1990.</td></tr><tr><td class="patent-data-table-td ">86</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Patel et al. Am. J. Human Genet. 46:801-809, 1990.</td></tr><tr><td class="patent-data-table-td ">87</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Patel, et al., "<a href='http://scholar.google.com/scholar?q="Genetic+Mapping+of+Autosomal+Dominant+Charcot-Marie-Tooth+Disease+in+a+Large+French-Acadian+Kindred%3A+Identification+of+New+Linked+Markers+on+Chromosome+17"'>Genetic Mapping of Autosomal Dominant Charcot-Marie-Tooth Disease in a Large French-Acadian Kindred: Identification of New Linked Markers on Chromosome 17</a>", Am J Hum Genet., 46:801-809, 1990.</td></tr><tr><td class="patent-data-table-td ">88</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Patel, et al., Genetic Mapping of Autosomal Dominant Charcot Marie Tooth Disease in a Large French Acadian Kindred: Identification of New Linked Markers on Chromosome 17 , Am J Hum Genet., 46:801 809, 1990.</td></tr><tr><td class="patent-data-table-td ">89</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Raeymaekers et al. "<a href='http://scholar.google.com/scholar?q="Duplication+in+Chromosome+17p11.2+in+Charcot-Marie-Tooth+Neuropathy+Type+1a+%28CMT+1a%29"'>Duplication in Chromosome 17p11.2 in Charcot-Marie-Tooth Neuropathy Type 1a (CMT 1a)</a>" Neuromuscular Dis. 1(2):93-97 (1991).</td></tr><tr><td class="patent-data-table-td ">90</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Raeymaekers et al. Duplication in Chromosome 17p11.2 in Charcot Marie Tooth Neuropathy Type 1a (CMT 1a) Neuromuscular Dis. 1(2):93 97 (1991).</td></tr><tr><td class="patent-data-table-td ">91</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Raeymaekers, et al., Neuromuscular Dis., 1(2):93 97 (1991).</td></tr><tr><td class="patent-data-table-td ">92</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Raeymaekers, et al., Neuromuscular Dis., 1(2):93-97 (1991).</td></tr><tr><td class="patent-data-table-td ">93</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Trask et al., "<a href='http://scholar.google.com/scholar?q="Mapping+of+Human+Chromosome+Xq28+by+Two+Color+Fluorescence+in+situ+Hybridization+of+DNA+Sequences+to+Interphase+Cell+Nuclei"'>Mapping of Human Chromosome Xq28 by Two Color Fluorescence in situ Hybridization of DNA Sequences to Interphase Cell Nuclei</a>" Am. J. Hum. Genet. 48:1-15 (1991).</td></tr><tr><td class="patent-data-table-td ">94</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Trask et al., "<a href='http://scholar.google.com/scholar?q="The+Proximity+of+DNA+Sequences+in+Interphase+Cell+Nuclei+is+Correlated+to+Genomic+Distance+and+Permits+Ordering+of+Cosmids+Spanning+250+Kilobase+Pairs"'>The Proximity of DNA Sequences in Interphase Cell Nuclei is Correlated to Genomic Distance and Permits Ordering of Cosmids Spanning 250 Kilobase Pairs</a>" Genomics 5:710-717 (1989).</td></tr><tr><td class="patent-data-table-td ">95</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Trask et al., Mapping of Human Chromosome Xq28 by Two Color Fluorescence in situ Hybridization of DNA Sequences to Interphase Cell Nuclei Am. J. Hum. Genet. 48:1 15 (1991).</td></tr><tr><td class="patent-data-table-td ">96</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Trask et al., The Proximity of DNA Sequences in Interphase Cell Nuclei is Correlated to Genomic Distance and Permits Ordering of Cosmids Spanning 250 Kilobase Pairs Genomics 5:710 717 (1989).</td></tr><tr><td class="patent-data-table-td ">97</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Trask, et al., Am. J. Hum. Genet., 48:1 15 (1991).</td></tr><tr><td class="patent-data-table-td ">98</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Trask, et al., Am. J. Hum. Genet., 48:1-15 (1991).</td></tr><tr><td class="patent-data-table-td ">99</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Trask, et al., Genomics, 5:710 717 (1989).</td></tr><tr><td class="patent-data-table-td ">100</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Trask, et al., Genomics, 5:710-717 (1989).</td></tr><tr><td class="patent-data-table-td ">101</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Tsipouras et al. "<a href='http://scholar.google.com/scholar?q="A+Centromere-Based+Linkage+Group+on+Long+Arm+of+Human+Chromosome+17"'>A Centromere-Based Linkage Group on Long Arm of Human Chromosome 17</a>" Cytogenet Cell Genet 47:109-110 (1988).</td></tr><tr><td class="patent-data-table-td ">102</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Tsipouras et al. A Centromere Based Linkage Group on Long Arm of Human Chromosome 17 Cytogenet Cell Genet 47:109 110 (1988).</td></tr><tr><td class="patent-data-table-td ">103</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Tsipouras et al., "<a href='http://scholar.google.com/scholar?q="A+Centromere-Based+Linkage+Group+on+the+Long+Arm+of+Human+Chromosome+17"'>A Centromere-Based Linkage Group on the Long Arm of Human Chromosome 17</a>" Cytogenet Cell Genet 47:109-110 (1988).</td></tr><tr><td class="patent-data-table-td ">104</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Tsipouras et al., A Centromere Based Linkage Group on the Long Arm of Human Chromosome 17 Cytogenet Cell Genet 47:109 110 (1988).</td></tr><tr><td class="patent-data-table-td ">105</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Weissenbach et al., "<a href='http://scholar.google.com/scholar?q="A+Second-Generation+Linkage+Map+of+the+Human+Genome"'>A Second-Generation Linkage Map of the Human Genome</a>" Nature 359:794-801 (Oct. 29, 1992).</td></tr><tr><td class="patent-data-table-td ">106</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Weissenbach et al., A Second Generation Linkage Map of the Human Genome Nature 359:794 801 (Oct. 29, 1992).</td></tr><tr><td class="patent-data-table-td ">107</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Wijmenga et al., "<a href='http://scholar.google.com/scholar?q="Mapping+of+Facioscapulohumeral+Muscular+Dystrophy+Gene+to+Chromosome+4q35-qter+by+Multipoint+Linkage+Analysis+and+in+situ+Hybridization"'>Mapping of Facioscapulohumeral Muscular Dystrophy Gene to Chromosome 4q35-qter by Multipoint Linkage Analysis and in situ Hybridization</a>" Genomics 9:570-575 (1991).</td></tr><tr><td class="patent-data-table-td ">108</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Wijmenga et al., Mapping of Facioscapulohumeral Muscular Dystrophy Gene to Chromosome 4q35 qter by Multipoint Linkage Analysis and in situ Hybridization Genomics 9:570 575 (1991).</td></tr><tr><td class="patent-data-table-td ">109</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Wiljmenga, et al, Genomics, 9:570 575 (1991).</td></tr><tr><td class="patent-data-table-td ">110</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Wiljmenga, et al, Genomics, 9:570-575 (1991).</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="forward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Referenced by</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Citing Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5972654">US5972654</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jan 20, 1998</td><td class="patent-data-table-td patent-date-value">Oct 26, 1999</td><td class="patent-data-table-td ">Incyte Pharmaceuticals, Inc.</td><td class="patent-data-table-td ">The use of these sequences in the diagnosis, treatment, and prevention of developmental, reproductive, muscle, immunological, and neoplastic disorders.</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6037163">US6037163</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Aug 4, 1999</td><td class="patent-data-table-td patent-date-value">Mar 14, 2000</td><td class="patent-data-table-td ">Incyte Pharmaceuticals, Inc.</td><td class="patent-data-table-td ">Human microfibril-associated glycoprotein 4 splice variant</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6309637">US6309637</a></td><td class="patent-data-table-td patent-date-value">Dec 21, 1999</td><td class="patent-data-table-td patent-date-value">Oct 30, 2001</td><td class="patent-data-table-td ">Incyte Genomics, Inc.</td><td class="patent-data-table-td ">For treating or preventing disorders associated with expression of mag4v.</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6696246">US6696246</a></td><td class="patent-data-table-td patent-date-value">Aug 23, 1999</td><td class="patent-data-table-td patent-date-value">Feb 24, 2004</td><td class="patent-data-table-td ">Naxcor, Inc.</td><td class="patent-data-table-td ">Allowing stringent wash conditions reducing background signals without loss of hybrids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6913879">US6913879</a></td><td class="patent-data-table-td patent-date-value">Jul 10, 2000</td><td class="patent-data-table-td patent-date-value">Jul 5, 2005</td><td class="patent-data-table-td ">Telechem International Inc.</td><td class="patent-data-table-td ">Concurrent genetic mapping of preferential genomic fragments; neonatal diagnostic screening</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7189509">US7189509</a></td><td class="patent-data-table-td patent-date-value">Aug 16, 2002</td><td class="patent-data-table-td patent-date-value">Mar 13, 2007</td><td class="patent-data-table-td ">Zhifeng Shao</td><td class="patent-data-table-td ">Analysis of gene expression profiles using sequential hybridization</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8021614">US8021614</a></td><td class="patent-data-table-td patent-date-value">Jun 8, 2006</td><td class="patent-data-table-td patent-date-value">Sep 20, 2011</td><td class="patent-data-table-td ">The General Hospital Corporation</td><td class="patent-data-table-td ">Devices and methods for enrichment and alteration of cells and other particles</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8304230">US8304230</a></td><td class="patent-data-table-td patent-date-value">Mar 21, 2007</td><td class="patent-data-table-td patent-date-value">Nov 6, 2012</td><td class="patent-data-table-td ">The General Hospital Corporation</td><td class="patent-data-table-td ">Microfluidic device for cell separation and uses thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8372579">US8372579</a></td><td class="patent-data-table-td patent-date-value">May 8, 2007</td><td class="patent-data-table-td patent-date-value">Feb 12, 2013</td><td class="patent-data-table-td ">The General Hospital Corporation</td><td class="patent-data-table-td ">Microfluidic device for cell separation and uses thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8548747">US8548747</a></td><td class="patent-data-table-td patent-date-value">Oct 6, 2005</td><td class="patent-data-table-td patent-date-value">Oct 1, 2013</td><td class="patent-data-table-td ">Children&#39;s Hospital Medical Center Of Akron</td><td class="patent-data-table-td ">Optimizing genome-wide mutation analysis of chromosomes and genes</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8585971">US8585971</a></td><td class="patent-data-table-td patent-date-value">Apr 20, 2012</td><td class="patent-data-table-td patent-date-value">Nov 19, 2013</td><td class="patent-data-table-td ">The General Hospital Corporation</td><td class="patent-data-table-td ">Devices and method for enrichment and alteration of cells and other particles</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20130071843">US20130071843</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Mar 15, 2011</td><td class="patent-data-table-td patent-date-value">Mar 21, 2013</td><td class="patent-data-table-td ">Jeremy A. Squire</td><td class="patent-data-table-td ">Methods, probe sets, and kits for detection of deletion of tumor suppressor genes by fluorescence in situ hybridization</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="classifications"></a><div class="patent-section-header"><span class="patent-section-title">Classifications</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th"> </th><th class="patent-data-table-th"> </th></tr></thead><tr><td class="patent-data-table-td ">U.S. Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=cdFDBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S006110">435/6.11</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=cdFDBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc436/defs436.htm&usg=AFQjCNGx2wdEO-lMZYH5-JBwc-X_dDDKIg#C436S811000">436/811</a></span></td></tr><tr><td class="patent-data-table-td ">International Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=cdFDBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12Q0001680000">C12Q1/68</a></span></td></tr><tr><td class="patent-data-table-td ">Cooperative Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=cdFDBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12Q1/6883">C12Q1/6883</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=cdFDBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12Q1/6841">C12Q1/6841</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=cdFDBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=Y10S436/811">Y10S436/811</a></span></td></tr><tr><td class="patent-data-table-td ">European Classification</td><td class="patent-data-table-td "><span class="nested-value">C12Q1/68M6</span>, <span class="nested-value">C12Q1/68B14</span></td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="legal-events"></a><div class="patent-section-header"><span class="patent-section-title">Legal Events</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Date</th><th class="patent-data-table-th">Code</th><th class="patent-data-table-th">Event</th><th class="patent-data-table-th">Description</th></tr></thead><tr><td class="patent-data-table-td patent-date-value">Mar 9, 2009</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">12</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Apr 11, 2006</td><td class="patent-data-table-td ">B1</td><td class="patent-data-table-td ">Reexamination certificate first reexamination</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">CLAIMS 1, 3, 5, 8, 13, 14 AND 15 ARE DETERMINED TO BE PATENTABLE AS AMENDED. CLAIMS 2, 4, 6, 7, 9-12 AND 16-21, DEPENDENT ON AN AMENDED CLAIM, ARE DETERMINED TO BE PATENTABLE.</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Mar 9, 2005</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">8</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Mar 8, 2001</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">4</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Apr 18, 2000</td><td class="patent-data-table-td ">RR</td><td class="patent-data-table-td ">Request for reexamination filed</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20000306</span></div></td></tr></table><div class="patent-section-footer"></div></div><div class="modal-dialog" id="patent-images-lightbox"><div class="patent-lightbox-controls"><div class="patent-lightbox-rotate-controls"><div class="patent-lightbox-rotation-text">Rotate</div><div class="rotate-icon rotate-ccw-icon"></div><div class="rotate-icon rotate-cw-icon"></div></div><div class="patent-lightbox-index-counter"></div><a class="patent-lightbox-fullsize-link" target="_blank">Original Image</a><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_right.png" alt="Next page"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_left.png" alt="Previous page"width="21" height="21" /></div></div><div class="modal-dialog-content"><div class="patent-lightbox-image-holder"><div class="patent-lightbox-placeholder"></div></div></div></div><script>_OC_initPatentsAtb({image_not_available_html: " Image not available"});</script></div></div></div></td></tr></table><script>(function() {var href = window.location.href;if (href.indexOf('?') !== -1) {var parameters = href.split('?')[1].split('&');for (var i = 0; i < parameters.length; i++) {var param = parameters[i].split('=');if (param[0] == 'focus') {var elem = document.getElementById(param[1]);if (elem) {elem.focus();}}}}})();</script><script>_OC_addFlags({LockSrc:"/books/javascript/lock_8a2b04e7bf975d5171d8e4c0b6365c7a.js", Host:"http://www.google.com/", IsBooksRentalEnabled:1, IsWebstoreDisplayCaseEnabled:1, IsObfuscationEnabled:1, IsBrowsingHistoryEnabled:1, IsWebReaderSvgEnabled:0, IsGeoLayerEnabled:1, IsImageModeNotesEnabled:1, IsCopyMenuItemEnabled:1, IsGiftingEnabled:0, IsWebReaderUniversalPaginatorEnabled:0, IsOfflineBubbleEnabled:1, IsReaderEnabledForPlayRequests:1, IsFutureOnSaleVolumesEnabled:1, IsOfflineRestrictedCopyEnabled:1, IsBooksUnifiedLeftNavEnabled:1, IsRestrictedCopyEnabled:1, IsZipitFolderCollectionEnabled:1, IsEndOfSampleRecommendationsEnabled:1, IsRatingsOnBookcardsEnabled:1, IsAdsDisabled:0, IsIframePageDisplayEnabled:0, IsEmbeddedMediaEnabled:1, IsImageModeAnnotationsEnabled:1, IsMyLibraryGooglePlusEnabled:1, IsImagePageProviderEnabled:0, IsBookcardListPriceSmall:0, IsInternalUser:0, IsBooksShareButtonEnabled:0, IsPreOrdersEnabled:0, IsDisabledRandomBookshelves:0, WebstoreDisplayCasePosition:3});_OC_Run({"enable_p13n":false,"add_vol_to_collection_base_url":"http://www.google.com/patents?op=add\u0026sig=ACfU3U0eVwcLYhUVu-cp13kGVyZwuCENBA\u0026id=cdFDBAABERAJ","remove_vol_from_collection_base_url":"http://www.google.com/patents?op=remove\u0026sig=ACfU3U1sIb8MkC3CuqoeH6jhIuZVaslkWg\u0026id=cdFDBAABERAJ","logged_in":false,"p13n_save_user_settings_url":"http://www.google.com/patents?op=edit_user_settings\u0026sig=ACfU3U1ftiPjsdHXbJtX25NNmuHJnkY08Q","is_cobrand":false,"sign_in_url":"https://www.google.com/accounts/Login?service=\u0026continue=http://www.google.com/patents%3Fhl%3Den\u0026hl=en","is_play_enabled":true}, {"volume_id":"","is_ebook":true,"volumeresult":{"has_flowing_text":false,"has_scanned_text":true,"can_download_pdf":false,"can_download_epub":false,"is_pdf_drm_enabled":false,"is_epub_drm_enabled":false,"download_pdf_url":"http://www.google.com/patents/download/Multicolor_in_situ_hybridization_methods.pdf?id=cdFDBAABERAJ\u0026output=pdf\u0026sig=ACfU3U1_D9RUc3dXFVyMfvwN4DAsGDOEgQ"},"sample_url":"http://www.google.com/patents/reader?id=cdFDBAABERAJ\u0026printsec=frontcover\u0026output=reader\u0026source=gbs_atb_hover","is_browsable":true,"is_public_domain":true}, {});</script><div id="footer_table" style="font-size:83%;text-align:center;position:relative;top:20px;height:4.5em;margin-top:2em"><div style="margin-bottom:8px"><a href=http://www.google.com/><nobr>Google&nbsp;Home</nobr></a> - <a href=//www.google.com/patents/sitemap/><nobr>Sitemap</nobr></a> - <a href=http://www.google.com/googlebooks/uspto.html><nobr>USPTO Bulk Downloads</nobr></a> - <a href=/intl/en/privacy/><nobr>Privacy Policy</nobr></a> - <a href=/intl/en/policies/terms/><nobr>Terms of Service</nobr></a> - <a href=https://support.google.com/faqs/answer/2539193?hl=en><nobr>About Google Patents</nobr></a> - <a href="http://www.google.com/tools/feedback/intl/en/error.html" onclick="try{_OC_startFeedback({productId: '72792',locale: 'en'});return false;}catch(e){}"><nobr>Send Feedback</nobr></a></div><span>Data provided by IFI CLAIMS Patent Services</span><br><span >&copy;2012 Google</span></div> <script type="text/javascript">var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));</script><script type="text/javascript">var pageTracker = _gat._getTracker("UA-27188110-1");pageTracker._setCookiePath("/patents/");pageTracker._trackPageview();</script> </body></html>