<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN""http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html><head><title>Patent US5688688 - Vector for expression of a polypeptide in a mammalian cell - Google Patents</title><script>(function(){(function(){function e(a){this.t={};this.tick=function(a,c,b){var d=void 0!=b?b:(new Date).getTime();this.t[a]=[d,c];if(void 0==b)try{window.console.timeStamp("CSI/"+a)}catch(e){}};this.tick("start",null,a)}var a;window.performance&&(a=window.performance.timing);var f=a?new e(a.responseStart):new e;window.jstiming={Timer:e,load:f};if(a){var c=a.navigationStart,d=a.responseStart;0<c&&d>=c&&(window.jstiming.srt=d-c)}if(a){var b=window.jstiming.load;0<c&&d>=c&&(b.tick("_wtsrt",void 0,c),b.tick("wtsrt_",
"_wtsrt",d),b.tick("tbsd_","wtsrt_"))}try{a=null,window.chrome&&window.chrome.csi&&(a=Math.floor(window.chrome.csi().pageT),b&&0<c&&(b.tick("_tbnd",void 0,window.chrome.csi().startE),b.tick("tbnd_","_tbnd",c))),null==a&&window.gtbExternal&&(a=window.gtbExternal.pageT()),null==a&&window.external&&(a=window.external.pageT,b&&0<c&&(b.tick("_tbnd",void 0,window.external.startE),b.tick("tbnd_","_tbnd",c))),a&&(window.jstiming.pt=a)}catch(g){}})();})();
</script><link rel="stylesheet" href="/patents/css/_8a2b04e7bf975d5171d8e4c0b6365c7a/kl_intl_patents_bundle.css" type="text/css" /><script src="/books/javascript/atb_8a2b04e7bf975d5171d8e4c0b6365c7a__en.js"></script><script>function googleTranslateElementInit() {new google.translate.TranslateElement({pageLanguage: "en",gaTrack: true,gaId: "UA-27188110-1",multilanguagePage: true});}</script><script src="//translate.google.com/translate_a/element.js?cb=googleTranslateElementInit"></script><meta name="DC.type" content="Patent"><meta name="DC.title" content="Vector for expression of a polypeptide in a mammalian cell"><meta name="DC.contributor" content="Paul A. Luciw" scheme="inventor"><meta name="DC.contributor" content="Dino Dina" scheme="inventor"><meta name="DC.contributor" content="Steven Rosenberg" scheme="inventor"><meta name="DC.contributor" content="Barbara S. Chapman" scheme="inventor"><meta name="DC.contributor" content="Richard M. Thayer" scheme="inventor"><meta name="DC.contributor" content="Nancy L. Haigwood" scheme="inventor"><meta name="DC.contributor" content="Chiron Corporation" scheme="assignee"><meta name="DC.date" content="1994-8-10" scheme="dateSubmitted"><meta name="DC.description" content="Polynucleotide sequences are provided for the diagnosis of the presence of retroviral infection in a human host associated with lymphadenopathy syndrome and/or acquired immune deficiency syndrome, for expression of polypeptides and use of the polypeptides to prepare antibodies, where both the polypeptides and antibodies may be employed as diagnostic reagents or in therapy, e.g., vaccines and passive immunization. The sequences provide detection of the viral infectious agents associated with the indicated syndromes and can be used for expression of antigenic polypeptides."><meta name="DC.date" content="1997-11-18" scheme="issued"><meta name="DC.relation" content="EP:0173177" scheme="references"><meta name="DC.relation" content="EP:0260148:A2" scheme="references"><meta name="DC.relation" content="EP:0323997:A1" scheme="references"><meta name="DC.relation" content="US:4708818" scheme="references"><meta name="DC.relation" content="US:4740461" scheme="references"><meta name="DC.relation" content="US:4981790" scheme="references"><meta name="DC.relation" content="US:4988624" scheme="references"><meta name="DC.relation" content="US:5024939" scheme="references"><meta name="DC.relation" content="US:5156949" scheme="references"><meta name="DC.relation" content="US:5168062" scheme="references"><meta name="citation_reference" content="Boshart M; Weber F; Jahn G; Dorsch Hasler K; Fleckenstein B; Schaffner W, A very strong enhancer is located upstream of an immediate early gene of human cytomegalovirus, Cell, (Jun. 1985) 41 (2) 521 30."><meta name="citation_reference" content="Boshart M; Weber F; Jahn G; Dorsch-Hasler K; Fleckenstein B; Schaffner W, &quot;A very strong enhancer is located upstream of an immediate early gene of human cytomegalovirus,&quot; Cell, (Jun. 1985) 41 (2) 521-30."><meta name="citation_reference" content="Burke et al., J. of Biol. Chem. (1986) &quot;The Functional Domains of Coagulation Factor VIII:C&quot; 261:12574-78."><meta name="citation_reference" content="Burke et al., J. of Biol. Chem. (1986) The Functional Domains of Coagulation Factor VIII:C 261:12574 78."><meta name="citation_reference" content="Chapman et al., Nucleic Acids Research (1991) &quot;Effect of intron A from human cytomegalovirus (Towne) immediate-early gene on heterlogous expression in mammalian cells&quot; 19(14):3979-3986."><meta name="citation_reference" content="Chapman et al., Nucleic Acids Research (1991) Effect of intron A from human cytomegalovirus (Towne) immediate early gene on heterlogous expression in mammalian cells 19(14):3979 3986."><meta name="citation_reference" content="Copeland et al., &quot;Complete Amino Acid Seq . . . ,&quot; FEBS Lett 162:390-5 (1983)."><meta name="citation_reference" content="Copeland et al., Complete Amino Acid Seq . . . , FEBS Lett 162:390 5 (1983)."><meta name="citation_reference" content="Foecking M K; Hofstetter H, &quot;Powerful and versatile enhancer-promoter unit for mammalian expression vectors,&quot; Gene, (1986) 45 (1) 101-5."><meta name="citation_reference" content="Foecking M K; Hofstetter H, Powerful and versatile enhancer promoter unit for mammalian expression vectors, Gene, (1986) 45 (1) 101 5."><meta name="citation_reference" content="Gorman C M; Moffat L F; Howard B H, &quot;Recombinant genomes which express chloramphenicol acetyltransferase in mammalian cells,&quot; Mol Cell Biol, (Sep. 1982) 2 (9) 1044-51."><meta name="citation_reference" content="Gorman C M; Moffat L F; Howard B H, Recombinant genomes which express chloramphenicol acetyltransferase in mammalian cells, Mol Cell Biol, (Sep. 1982) 2 (9) 1044 51."><meta name="citation_reference" content="LeFemina et al. (1980) in Fields, et al. (eds.) Animal Virus Genetics , Structural Organization of the DNA Molecules from Human Cytomegalovirus Acad. Press, N.Y., 18:39 55."><meta name="citation_reference" content="LeFemina et al. (1980) in Fields, et al. (eds.) Animal Virus Genetics, &quot;Structural Organization of the DNA Molecules from Human Cytomegalovirus&quot; Acad. Press, N.Y., 18:39-55."><meta name="citation_reference" content="Lusky et al., Cell (1984) &quot;Characterization of the Bovine Papilloma Virus Plasmid Maintenance Sequences&quot; 36:391-401."><meta name="citation_reference" content="Lusky et al., Cell (1984) Characterization of the Bovine Papilloma Virus Plasmid Maintenance Sequences 36:391 401."><meta name="citation_reference" content="Myers et al., Cell (1981) &quot;SV40 Gene Expression is Modulated by the Cooperative Binding of T Antigen to DNA&quot; 25:373-384."><meta name="citation_reference" content="Myers et al., Cell (1981) SV40 Gene Expression is Modulated by the Cooperative Binding of T Antigen to DNA 25:373 384."><meta name="citation_reference" content="Reitz et al., &quot;Characterization and Distribution of Nucleic . . . ,&quot; Proc Natl. Acad Sci (USA) 78:1887-1891 (1981)."><meta name="citation_reference" content="Reitz et al., Characterization and Distribution of Nucleic . . . , Proc Natl. Acad Sci (USA) 78:1887 1891 (1981)."><meta name="citation_reference" content="Rio et al., Cell (1983) &quot;SV40 T Antigen Binding Site Mutations That Affect Autoregulation&quot; 32:1227-1240."><meta name="citation_reference" content="Rio et al., Cell (1983) SV40 T Antigen Binding Site Mutations That Affect Autoregulation 32:1227 1240."><meta name="citation_reference" content="Seiki et al., &quot;Human Adult T-Cell Leukemia . . . &quot;, Proc Natl Acad Sci 80:3618-3622 (1983)."><meta name="citation_reference" content="Seiki et al., Human Adult T Cell Leukemia . . . , Proc Natl Acad Sci 80:3618 3622 (1983)."><meta name="citation_reference" content="Stenberg R M; Thomsen D R; Stinski M F, &quot;Structural analysis of the major immediate early gene of human cytomegalovirus,&quot; J Virol, (Jan. 1984) 49 (1) 190-9."><meta name="citation_reference" content="Stenberg R M; Thomsen D R; Stinski M F, Structural analysis of the major immediate early gene of human cytomegalovirus, J Virol, (Jan. 1984) 49 (1) 190 9."><meta name="citation_reference" content="Stern, &quot;Predicting Antigenic Sites on Proteins,&quot; Tibtech 9:163-168 (1991)."><meta name="citation_reference" content="Stern, Predicting Antigenic Sites on Proteins, Tibtech 9:163 168 (1991)."><meta name="citation_reference" content="Stinski M F; Thomsen D R; Stenberg R M; Goldstein L C, &quot;Organization and expression of the immediate early genes of human cytomegalovirus,&quot; J Virol, (Apr. 1983) 46 (1) 1-14."><meta name="citation_reference" content="Stinski M F; Thomsen D R; Stenberg R M; Goldstein L C, Organization and expression of the immediate early genes of human cytomegalovirus, J Virol, (Apr. 1983) 46 (1) 1 14."><meta name="citation_reference" content="Subramani et al., Molecular and Cellular Biology (1981) &quot;Expression of the Mouse Dihydrofolate Reductase Complementary Deoxyribonucleic Acid in Simian Virus 40 Vectors&quot; 1:854-864."><meta name="citation_reference" content="Subramani et al., Molecular and Cellular Biology (1981) Expression of the Mouse Dihydrofolate Reductase Complementary Deoxyribonucleic Acid in Simian Virus 40 Vectors 1:854 864."><meta name="citation_reference" content="Thomsen D R; Stenberg R M; Goins W F; Stinski M F, &quot;Promoter-regulatory region of the major immediate early gene of human cytomegalovirus,&quot; Proc Natl Acad Sci U S A, (Feb. 1984) 81 (3) 659-63."><meta name="citation_reference" content="Thomsen D R; Stenberg R M; Goins W F; Stinski M F, Promoter regulatory region of the major immediate early gene of human cytomegalovirus, Proc Natl Acad Sci U S A, (Feb. 1984) 81 (3) 659 63."><meta name="citation_reference" content="Ulmer et al., Science (1993) &quot;Heterologous Protection Against Influenza by Injection of DNA Encoding a Viral Protein&quot; 259:1745-49."><meta name="citation_reference" content="Ulmer et al., Science (1993) Heterologous Protection Against Influenza by Injection of DNA Encoding a Viral Protein 259:1745 49."><meta name="citation_reference" content="van Zonneveld A J; Veerman H; Pannekoek H, &quot;Autonomous functions of structural domains on human tissue-type plasminogen activator,&quot; Proc Natl Acad Sci U S A, (Jul., 1986) 83 (13) 4670-4."><meta name="citation_reference" content="van Zonneveld A J; Veerman H; Pannekoek H, Autonomous functions of structural domains on human tissue type plasminogen activator, Proc Natl Acad Sci U S A, (Jul., 1986) 83 (13) 4670 4."><meta name="citation_patent_number" content="US:5688688"><meta name="citation_patent_application_number" content="US:08/288,336"><link rel="canonical" href="http://www.google.com/patents/US5688688"/><meta property="og:url" content="http://www.google.com/patents/US5688688"/><meta name="title" content="Patent US5688688 - Vector for expression of a polypeptide in a mammalian cell"/><meta name="description" content="Polynucleotide sequences are provided for the diagnosis of the presence of retroviral infection in a human host associated with lymphadenopathy syndrome and/or acquired immune deficiency syndrome, for expression of polypeptides and use of the polypeptides to prepare antibodies, where both the polypeptides and antibodies may be employed as diagnostic reagents or in therapy, e.g., vaccines and passive immunization. The sequences provide detection of the viral infectious agents associated with the indicated syndromes and can be used for expression of antigenic polypeptides."/><meta property="og:title" content="Patent US5688688 - Vector for expression of a polypeptide in a mammalian cell"/><meta property="og:type" content="book"/><meta property="og:site_name" content="Google Books"/><meta property="og:image" content="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><link rel="image_src" href="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><script>(function(){try{var aa=function(a,b,c,d){d=d||{};d._sn=["cfg",b,c].join(".");window.gbar.logger.ml(a,d)};var g=window.gbar=window.gbar||{},l=window.gbar.i=window.gbar.i||{},m={},n;function _tvn(a,b){var c=parseInt(a,10);return isNaN(c)?b:c}function _tvf(a,b){var c=parseFloat(a);return isNaN(c)?b:c}function _tvv(a){return!!a}function p(a,b,c){(c||g)[a]=b}g.bv={n:_tvn("2",0),r:"",f:".67.",e:"0",m:_tvn("0",1)};
function q(a,b,c){var d="on"+b;if(a.addEventListener)a.addEventListener(b,c,!1);else if(a.attachEvent)a.attachEvent(d,c);else{var f=a[d];a[d]=function(){var a=f.apply(this,arguments),b=c.apply(this,arguments);return void 0==a?b:void 0==b?a:b&&a}}}var s=function(a){return function(){return g.bv.m==a}},ba=s(1),ca=s(2);p("sb",ba);p("kn",ca);l.a=_tvv;l.b=_tvf;l.c=_tvn;l.i=aa;var da=window.gbar.i.i;var t,u,v,w;function ea(a){v=a}function fa(a){var b;if(b=v&&window.encodeURIComponent)b=a.href,b=!b.match(/^http[s]?:\/\/accounts\.google\.[^/]*\/ClearSID/i)&&!b.match(/^http[s]?:\/\/[^/]*\/accounts\/ClearSID/i);if(b=b&&encodeURIComponent(v()))a.href=a.href.replace(/([?&]continue=)[^&]*/,"$1"+b)}function ga(a){window.gApplication&&(a.href=window.gApplication.getTabUrl(a.href))}
function ha(a){var b=document.forms[0].q,c=window.encodeURIComponent&&b&&b.value,b=b&&b.placeholder;c&&c!=b&&(a.href=a.href.replace(/([?&])q=[^&]*|$/,function(a,b){return(b||"&")+"q="+encodeURIComponent(c)}))}n=l.a("")?ga:ha;
function x(a,b,c,d,f,e){var h=document.getElementById(a);if(h){var k=h.style;k.left=d?"auto":b+"px";k.right=d?b+"px":"auto";k.top=c+"px";k.visibility=u?"hidden":"visible";f&&e?(k.width=f+"px",k.height=e+"px"):(x(t,b,c,d,h.offsetWidth,h.offsetHeight),u=u?"":a)}}
var y=[],ia=function(a,b){y.push(b)},ja=function(a){a=a||window.event;var b=a.target||a.srcElement;a.cancelBubble=!0;null==t&&(a=document.createElement(Array.every||window.createPopup?"iframe":"div"),a.frameBorder="0",t=a.id="gbs",a.src="javascript:''",b.parentNode.appendChild(a),q(document,"click",z));var c=b,b=0;"gb3"!=c.className&&(c=c.parentNode);a=c.getAttribute("aria-owns")||"gbi";var d=c.offsetWidth,f=20<c.offsetTop?46:24;document.getElementById("tphdr")&&(f-=3);var e=!1;do b+=c.offsetLeft||
0;while(c=c.offsetParent);var c=(document.documentElement.clientWidth||document.body.clientWidth)-b-d,h,d=document.body,k=document.defaultView;k&&k.getComputedStyle?(d=k.getComputedStyle(d,""))&&(h=d.direction):h=d.currentStyle?d.currentStyle.direction:d.style.direction;h="rtl"==h;if("gbi"==a){for(d=0;k=y[d++];)k();A(null,window.navExtra);h&&(b=c,e=!0)}else h||(b=c,e=!0);u!=a&&z();x(a,b,f,e)},z=function(){u&&x(u,0,0)},A=function(a,b){var c,d=document.getElementById("gbi"),f=a;f||(f=d.firstChild);
for(;b&&(c=b.pop());){var e=d,h=c,k=f;w||(w="gb2");e.insertBefore(h,k).className=w}},ka=function(a,b,c){if((b=document.getElementById(b))&&a){a.className="gb4";var d=document.createElement("span");d.appendChild(a);d.appendChild(document.createTextNode(" | "));d.id=c;b.appendChild(d)}},la=function(){return document.getElementById("gb_70")},ma=function(){return!!u};p("qs",n);p("setContinueCb",ea);p("pc",fa);p("tg",ja);p("close",z);p("addLink",ka);p("almm",A);p("si",la);p("adh",ia);p("op",ma);var B=function(){},C=function(){},F=function(a){var b=new Image,c=D;b.onerror=b.onload=b.onabort=function(){try{delete E[c]}catch(a){}};E[c]=b;b.src=a;D=c+1},E=[],D=0;p("logger",{il:C,ml:B,log:F});var G=window.gbar.logger;var H={},na={},I=[],oa=l.b("0.1",.1),pa=l.a("1",!0),qa=function(a,b){I.push([a,b])},ra=function(a,b){H[a]=b},sa=function(a){return a in H},J={},K=function(a,b){J[a]||(J[a]=[]);J[a].push(b)},ta=function(a){K("m",a)},L=function(a,b){var c=document.createElement("script");c.src=a;c.async=pa;Math.random()<oa&&(c.onerror=function(){c.onerror=null;B(Error("Bundle load failed: name="+(b||"UNK")+" url="+a))});(document.getElementById("xjsc")||document.getElementsByTagName("body")[0]||
document.getElementsByTagName("head")[0]).appendChild(c)},N=function(a){for(var b=0,c;(c=I[b])&&c[0]!=a;++b);!c||c[1].l||c[1].s||(c[1].s=!0,M(2,a),c[1].url&&L(c[1].url,a),c[1].libs&&m.d&&m.d(c[1].libs))},O=function(a){K("gc",a)},P=null,ua=function(a){P=a},M=function(a,b,c){if(P){a={t:a,b:b};if(c)for(var d in c)a[d]=c[d];try{P(a)}catch(f){}}};p("mdc",H);p("mdi",na);p("bnc",I);p("qGC",O);p("qm",ta);p("qd",J);p("lb",N);p("mcf",ra);p("bcf",qa);p("aq",K);p("mdd","");p("has",sa);
p("trh",ua);p("tev",M);var Q=l.b("0.1",.001),R=0;
function _mlToken(a,b){try{if(1>R){R++;var c,d=a,f=b||{},e=encodeURIComponent,h=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&jexpid=",e("17483"),"&srcpg=",e("prop=22"),"&jsr=",Math.round(1/Q),"&ogev=",e("CJ7tU42GLYa-sATc-ICADA"),"&ogf=",g.bv.f,"&ogrp=",e("1"),"&ogv=",e("1407723702.0"),"&oggv="+e("es_plusone_gc_20140723.0_p0"),"&ogd=",e("com"),"&ogc=",e("LUX"),"&ogl=",e("en")];f._sn&&(f._sn="og."+
f._sn);for(var k in f)h.push("&"),h.push(e(k)),h.push("="),h.push(e(f[k]));h.push("&emsg=");h.push(e(d.name+":"+d.message));var r=h.join("");S(r)&&(r=r.substr(0,2E3));c=r;var Aa=window.gbar.logger._aem(a,c);F(Aa)}}catch(Na){}}var S=function(a){return 2E3<=a.length},va=function(a,b){return b};function T(a){B=a;p("_itl",S,G);p("_aem",va,G);p("ml",B,G);a={};H.er=a}l.a("")?T(function(a){throw a;}):l.a("1")&&Math.random()<Q&&T(_mlToken);I.push(["m",{url:"//ssl.gstatic.com/gb/js/scm_7385cc5883250b43a39405734c1bea59.js"}]);g.mcf("c",{});g.sg={c:""};if(l.a("1")){var wa=l.a("");I.push(["gc",{auto:wa,url:"//ssl.gstatic.com/gb/js/abc/gci_91f30755d6a6b787dcc2a4062e6e9824.js",libs:"googleapis.client:plusone:gapi.iframes"}]);var xa={version:"gci_91f30755d6a6b787dcc2a4062e6e9824.js",index:"",lang:"en"};H.gc=xa;var U=function(a){window.googleapis&&window.iframes?a&&a():(a&&O(a),N("gc"))};p("lGC",U);l.a("1")&&p("lPWF",U)};window.__PVT="";if(l.a("1")&&l.a("1")){var V=function(a){U(function(){K("pw",a);N("pw")})};p("lPW",V);I.push(["pw",{url:"//ssl.gstatic.com/gb/js/abc/pwm_45f73e4df07a0e388b0fa1f3d30e7280.js"}]);var W=[],ya=function(a){W[0]=a},za=function(a,b){var c=b||{};c._sn="pw";B(a,c)},Ba={signed:W,elog:za,base:"https://plusone.google.com/u/0",loadTime:(new Date).getTime()};H.pw=Ba;var X=function(a,b){for(var c=b.split("."),d=function(){var b=arguments;a(function(){for(var a=g,d=0,e=c.length-1;d<e;++d)a=a[c[d]];a[c[d]].apply(a,b)})},f=g,e=0,h=c.length-1;e<h;++e)f=
f[c[e]]=f[c[e]]||{};return f[c[e]]=d};X(V,"pw.clk");X(V,"pw.hvr");p("su",ya,g.pw)};function Ca(){function a(){for(var b;(b=e[h++])&&"m"!=b[0]&&!b[1].auto;);b&&(M(2,b[0]),b[1].url&&L(b[1].url,b[0]),b[1].libs&&m.d&&m.d(b[1].libs));h<e.length&&setTimeout(a,0)}function b(){0<f--?setTimeout(b,0):a()}var c=l.a("1"),d=l.a(""),f=3,e=I,h=0,k=window.gbarOnReady;if(k)try{k()}catch(r){da(r,"ml","or")}d?p("ldb",a):c?q(window,"load",b):b()}p("rdl",Ca);var Da={D:1,H:2,da:3,p:4,W:5,M:6,F:7,g:8,ha:9,U:10,L:11,T:12,S:13,N:14,Q:15,P:16,fa:17,w:18,O:19,ga:20,ea:21,u:22,G:23,ja:24,ka:25,ia:26,A:27,j:28,o:29,k:30,ca:31,Z:32,$:33,J:34,K:35,ba:36,aa:37,Y:38,B:39,R:40,v:41,X:42,V:43,h:48,C:49,I:500},Y=[1,2,3,4,5,6,9,10,11,13,14,28,29,30,34,35,37,38,39,40,41,42,43,48,49,500];var Z=l.b("0.001",1E-4),Ea=l.b("1",1),Fa=!1,Ga=!1;if(l.a("1")){var Ha=Math.random();Ha<=Z&&(Fa=!0);Ha<=Ea&&(Ga=!0)}var Ia=Da,$=null;function Ja(){var a=0,b=function(b,d){l.a(d)&&(a|=b)};b(1,"");b(2,"");b(4,"");b(8,"");return a}
function Ka(a,b){var c=Z,d=Fa,f;f=a;if(!$){$={};for(var e=0;e<Y.length;e++){var h=Y[e];$[h]=!0}}if(f=!!$[f])c=Ea,d=Ga;if(d){d=encodeURIComponent;g.rp?(f=g.rp(),f="-1"!=f?f:"1"):f="1";c=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&oge=",a,"&ogex=",d("17483"),"&ogev=",d("CJ7tU42GLYa-sATc-ICADA"),"&ogf=",g.bv.f,"&ogp=",d("22"),"&ogrp=",d(f),"&ogsr=",Math.round(1/c),"&ogv=",d("1407723702.0"),"&oggv="+
d("es_plusone_gc_20140723.0_p0"),"&ogd=",d("com"),"&ogl=",d("en"),"&ogc=",d("LUX"),"&ogus=",Ja()];if(b){"ogw"in b&&(c.push("&ogw="+b.ogw),delete b.ogw);var k;f=b;e=[];for(k in f)0!=e.length&&e.push(","),e.push(La(k)),e.push("."),e.push(La(f[k]));k=e.join("");""!=k&&(c.push("&ogad="),c.push(d(k)))}F(c.join(""))}}function La(a){"number"==typeof a&&(a+="");return"string"==typeof a?a.replace(".","%2E").replace(",","%2C"):a}C=Ka;p("il",C,G);var Ma={};H.il=Ma;setTimeout(function(){C(Ia.g)},0);}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var b=window.gbar.i.i;var c=window.gbar;var f=function(d){try{var a=document.getElementById("gbom");a&&d.appendChild(a.cloneNode(!0))}catch(e){b(e,"omas","aomc")}};c.aomc=f;}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var a=window.gbar;a.mcf("pm",{p:""});}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{window.gbar.rdl();}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
if (window['_OC_timingAction']) {window['_OC_timingAction']('patents_refpage');}</script><style>#gbar,#guser{font-size:13px;padding-top:1px !important;}#gbar{float:left;height:22px}#guser{padding-bottom:7px !important;text-align:right}.gbh,.gbd{border-top:1px solid #c9d7f1;font-size:1px}.gbh{height:0;position:absolute;top:24px;width:100%}#gbs,.gbm{background:#fff;left:0;position:absolute;text-align:left;visibility:hidden;z-index:1000}.gbm{border:1px solid;border-color:#c9d7f1 #36c #36c #a2bae7;z-index:1001}.gb1{margin-right:.5em}.gb1,.gb3{zoom:1}.gb2{display:block;padding:.2em .5em}.gb2,.gb3{text-decoration:none !important;border-bottom:none}a.gb1,a.gb4{text-decoration:underline !important}a.gb1,a.gb2,a.gb3,a.gb4{color:#00c !important}.gbi .gb3,.gbi .gb2,.gbi .gb4{color:#dd8e27 !important}.gbf .gb3,.gbf .gb2,.gbf .gb4{color:#900 !important}a.gb2:hover{background:#36c;color:#fff !important}#gbar .gbz0l{color:#000 !important;cursor:default;font-weight:bold;text-decoration:none !important}
#gbar { padding:.3em .6em !important;}</style></head><body  topmargin="3" marginheight="3"><div id=gbar><nobr><a onclick=gbar.qs(this);gbar.logger.il(1,{t:1}); class=gb1 id=gb_1 href="https://www.google.com/search?sa=N&tab=tw">Search</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:2}); class=gb1 id=gb_2 href="http://www.google.com/search?hl=en&tbm=isch&source=og&sa=N&tab=ti">Images</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:8}); class=gb1 id=gb_8 href="http://maps.google.com/maps?hl=en&sa=N&tab=tl">Maps</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:78}); class=gb1 id=gb_78 href="https://play.google.com/?hl=en&sa=N&tab=t8">Play</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:36}); class=gb1 id=gb_36 href="http://www.youtube.com/results?sa=N&tab=t1">YouTube</a> <a onclick=gbar.logger.il(1,{t:5}); class=gb1 id=gb_5 href="http://news.google.com/nwshp?hl=en&tab=tn">News</a> <a onclick=gbar.logger.il(1,{t:23}); class=gb1 id=gb_23 href="https://mail.google.com/mail/?tab=tm">Gmail</a> <a onclick=gbar.logger.il(1,{t:25}); class=gb1 id=gb_25 href="https://drive.google.com/?tab=to">Drive</a> <a class=gb3 href="http://www.google.com/intl/en/options/" onclick="this.blur();gbar.tg(event);return !1" aria-haspopup=true><u>More</u> <small>&#9660;</small></a><div class=gbm id=gbi><a onclick=gbar.logger.il(1,{t:24}); class=gb2 id=gb_24 href="https://www.google.com/calendar?tab=tc">Calendar</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:51}); class=gb2 id=gb_51 href="http://translate.google.com/?hl=en&sa=N&tab=tT">Translate</a><a onclick=gbar.logger.il(1,{t:17}); class=gb2 id=gb_17 href="http://www.google.com/mobile/?hl=en&tab=tD">Mobile</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:10}); class=gb2 id=gb_10 href="http://www.google.com/search?hl=en&tbo=u&tbm=bks&source=og&sa=N&tab=tp">Books</a><a onclick=gbar.logger.il(1,{t:212}); class=gb2 id=gb_212 href="https://wallet.google.com/manage/?tab=ta">Wallet</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:6}); class=gb2 id=gb_6 href="http://www.google.com/search?hl=en&tbo=u&tbm=shop&source=og&sa=N&tab=tf">Shopping</a><a onclick=gbar.logger.il(1,{t:30}); class=gb2 id=gb_30 href="http://www.blogger.com/?tab=tj">Blogger</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:27}); class=gb2 id=gb_27 href="http://www.google.com/finance?sa=N&tab=te">Finance</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:31}); class=gb2 id=gb_31 href="https://plus.google.com/photos?sa=N&tab=tq">Photos</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:12}); class=gb2 id=gb_12 href="http://www.google.com/search?hl=en&tbo=u&tbm=vid&source=og&sa=N&tab=tv">Videos</a><div class=gb2><div class=gbd></div></div><a onclick=gbar.logger.il(1,{t:66}); href="http://www.google.com/intl/en/options/" class=gb2>Even more &raquo;</a></div></nobr></div><div id=guser width=100%><nobr><span id=gbn class=gbi></span><span id=gbf class=gbf></span><span id=gbe></span><a target=_top id=gb_70 href="https://www.google.com/accounts/Login?service=&continue=http://www.google.com/patents%3Fhl%3Den&hl=en" class=gb4>Sign in</a><div style="display: none"><div class=gbm id=gbd5 aria-owner=gbg5><div class=gbmc><ol id=gbom class=gbmcc></ol></div></div></div></nobr></div><div class=gbh style=left:0></div><div class=gbh style=right:0></div><div role="alert" style="position: absolute; left: 0; right: 0;"><a href="http://www.google.com/patents/us5688688?hl=en&amp;output=html_text" title="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."><img border="0" src="http://www.google.com/images/cleardot.gif"alt="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."></a></div><div id="guser"><nobr></nobr></div><div style="clear:both;"></div><div id="gb-top-search-box" class="gb-top-search-box-small gb-reset"><table><tr><td class="logo"><a href="http://www.google.com/patents" class="logo-link"><img class="logo-img" src="/intl/en/images/logos/google_logo_41.png" alt="Go to Google Books Home" height="41"/></a></td><td><form action="http://www.google.com/search" name="f" id="vheadf" method="get"><span id="hf"></span><input type="hidden" name="tbm" value="pts"/><input type="hidden" name="tbo" value="1"/><input type="hidden" name="hl" value="en"/><table><tr><td><div class="inputs"><table><tr><td><div class="text-input"><input type="text" name="q" id="vheadq" class="text" maxlength="2048" size="31" value="" title="Search Patents" accesskey="s" autocomplete="off"/><script>window._OC_autoDir &&window._OC_autoDir('vheadq', 'tia-vheadq');</script></div></td><td><div class="submit-input"><input name="btnG" class="submit" type="submit" value=""/></div></td></tr></table></div></td><td class="col-ext-links"><div class="ext-links"><a href="http://www.google.com/advanced_patent_search">&lt;nobr&gt;Advanced Patent Search&lt;/nobr&gt;</a></div></td></tr></table></form></td></tr></table></div><div class="kd-appbar"><h2 class="kd-appname"><a href="/patents">Patents</a></h2><div class="kd-buttonbar left" id="left-toolbar-buttons"><a id="appbar-write-review-link" href=""></a><a id="appbar-view-print-sample-link" href=""></a><a id="appbar-view-ebook-sample-link" href=""></a><a id="appbar-patents-prior-art-finder-link" href="https://www.google.com/patents/related/US5688688"></a><a id="appbar-patents-discuss-this-link" href="http://www.google.com/url?id=QIBEBAABERAJ&amp;q=http://patents.stackexchange.com/redirect/google-patents%3Fpatent%3DUS5688688&amp;usg=AFQjCNHD84T-DxNKj7KaiDVquRrFlskU4g" data-is-grant="true"></a><a id="appbar-read-patent-link" href="//docs.google.com/viewer?url=patentimages.storage.googleapis.com/pdfs/US5688688.pdf"></a><a id="appbar-download-pdf-link" href="//patentimages.storage.googleapis.com/pdfs/US5688688.pdf"></a></div><div class="kd-buttonbar right" id="right-toolbar-buttons"></div></div><div id="books-microdata" itemscope=""itemtype="http://schema.org/Book"itemid="http://www.google.com/patents/US5688688" style="display:none"><span itemprop="description">Polynucleotide sequences are provided for the diagnosis of the presence of retroviral infection in a human host associated with lymphadenopathy syndrome and/or acquired immune deficiency syndrome, for expression of polypeptides and use of the polypeptides to prepare antibodies, where both the polypeptides...</span><span itemprop="url">http://www.google.com/patents/US5688688?utm_source=gb-gplus-share</span><span class="main-title" itemprop="name">Patent US5688688 - Vector for expression of a polypeptide in a mammalian cell</span><img itemprop="image" src="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"alt="Patent US5688688 - Vector for expression of a polypeptide in a mammalian cell" title="Patent US5688688 - Vector for expression of a polypeptide in a mammalian cell"></div><div style="display: none"><ol id="ofe-gear-menu-contents" class="gbmcc"><li class="gbe gbmtc"><a class="gbmt goog-menuitem-content" id="" href="http://www.google.com/advanced_patent_search">Advanced Patent Search</a></li></ol></div><table id="viewport_table" cellpadding="0" style="clear:both" cellspacing="0"><tr><td id="viewport_td"><div class=vertical_module_list_row><div id=intl_patents class=about_content><div id=intl_patents_v><table class="patent-bibdata"><tr><td class="patent-bibdata-heading">Publication number</td><td class="single-patent-bibdata">US5688688 A</td></tr><tr><td class="patent-bibdata-heading">Publication type</td><td class="single-patent-bibdata">Grant</td></tr><tr><td class="patent-bibdata-heading">Application number</td><td class="single-patent-bibdata">US 08/288,336</td></tr><tr><td class="patent-bibdata-heading">Publication date</td><td class="single-patent-bibdata">Nov 18, 1997</td></tr><tr><td class="patent-bibdata-heading">Filing date</td><td class="single-patent-bibdata">Aug 10, 1994</td></tr><tr><td class="patent-bibdata-heading">Priority date<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The priority date is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the date listed."></span></td><td class="single-patent-bibdata">Oct 31, 1984</td></tr><tr><td class="patent-bibdata-heading">Fee status<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The fee status is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the status or dates listed."></span></td><td class="single-patent-bibdata">Paid</td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Also published as</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/CA1341423C">CA1341423C</a>, </span><span class="patent-bibdata-value"><a href="/patents/CA1341482C">CA1341482C</a>, </span><span class="patent-bibdata-value"><a href="/patents/DE1245678T1">DE1245678T1</a>, </span><span class="patent-bibdata-value"><a href="/patents/DE3587394D1">DE3587394D1</a>, </span><span class="patent-bibdata-value"><a href="/patents/DE3587394T2">DE3587394T2</a>, </span><span class="patent-bibdata-value"><a href="/patents/DE3587394T3">DE3587394T3</a>, </span><span class="patent-bibdata-value"><a href="/patents/DE3588252D1">DE3588252D1</a>, </span><span class="patent-bibdata-value"><a href="/patents/DE3588252T2">DE3588252T2</a>, </span><span class="patent-bibdata-value"><a href="/patents/DE3588253D1">DE3588253D1</a>, </span><span class="patent-bibdata-value"><a href="/patents/DE3588253T2">DE3588253T2</a>, </span><span class="patent-bibdata-value"><a href="/patents/DE10399037I1">DE10399037I1</a>, </span><span class="patent-bibdata-value"><a href="/patents/DE10399037I2">DE10399037I2</a>, </span><span class="patent-bibdata-value"><a href="/patents/DE122005000031I1">DE122005000031I1</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP0181150A1">EP0181150A1</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP0181150B1">EP0181150B1</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP0181150B2">EP0181150B2</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP0518443A1">EP0518443A1</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP0518443B1">EP0518443B1</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP1245678A2">EP1245678A2</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP1245678A3">EP1245678A3</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP1245678B1">EP1245678B1</a>, </span><span class="patent-bibdata-value"><a href="/patents/US5156949">US5156949</a>, </span><span class="patent-bibdata-value"><a href="/patents/US6458527">US6458527</a>, </span><span class="patent-bibdata-value"><a href="/patents/US6531276">US6531276</a></span></span></td></tr><tr class="patent-bibdata-list-row alternate-patent-number"><td class="patent-bibdata-heading">Publication number</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value">08288336, </span><span class="patent-bibdata-value">288336, </span><span class="patent-bibdata-value">US 5688688 A, </span><span class="patent-bibdata-value">US 5688688A, </span><span class="patent-bibdata-value">US-A-5688688, </span><span class="patent-bibdata-value">US5688688 A, </span><span class="patent-bibdata-value">US5688688A</span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Inventors</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Paul+A.+Luciw%22">Paul A. Luciw</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Dino+Dina%22">Dino Dina</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Steven+Rosenberg%22">Steven Rosenberg</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Barbara+S.+Chapman%22">Barbara S. Chapman</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Richard+M.+Thayer%22">Richard M. Thayer</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Nancy+L.+Haigwood%22">Nancy L. Haigwood</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Original Assignee</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=inassignee:%22Chiron+Corporation%22">Chiron Corporation</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Export Citation</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/US5688688.bibtex">BiBTeX</a>, </span><span class="patent-bibdata-value"><a href="/patents/US5688688.enw">EndNote</a>, </span><span class="patent-bibdata-value"><a href="/patents/US5688688.ris">RefMan</a></span></span></td></tr><tr class="patent-internal-links"><td colspan=2><span class="patent-bibdata-value"><a href="#backward-citations">Patent Citations</a> (10),</span> <span class="patent-bibdata-value"><a href="#npl-citations">Non-Patent Citations</a> (38),</span> <span class="patent-bibdata-value"><a href="#forward-citations">Referenced by</a> (24),</span> <span class="patent-bibdata-value"><a href="#classifications">Classifications</a> (75),</span> <span class="patent-bibdata-value"><a href="#legal-events">Legal Events</a> (7)</span> </td></tr><tr><td colspan=2 class="patent-bibdata-external-link-spacer-top"></td></tr><tr class="patent-bibdata-external-link-spacer-bottom"></tr><tr><td colspan=2><span class="patent-bibdata-heading">External Links:&nbsp;</span><span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=QIBEBAABERAJ&q=http://patft.uspto.gov/netacgi/nph-Parser%3FSect2%3DPTO1%26Sect2%3DHITOFF%26p%3D1%26u%3D/netahtml/PTO/search-bool.html%26r%3D1%26f%3DG%26l%3D50%26d%3DPALL%26RefSrch%3Dyes%26Query%3DPN/5688688&usg=AFQjCNGt54dVFFJOt8i453lahMzqqaGLbg">USPTO</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=QIBEBAABERAJ&q=http://assignments.uspto.gov/assignments/q%3Fdb%3Dpat%26pat%3D5688688&usg=AFQjCNG0KfS9LNl5RCe_4g39jLapC_i1xA">USPTO Assignment</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=QIBEBAABERAJ&q=http://worldwide.espacenet.com/publicationDetails/biblio%3FCC%3DUS%26NR%3D5688688A%26KC%3DA%26FT%3DD&usg=AFQjCNHAzU_x6dN_RxMh_Y9ynQP8eIZxlg">Espacenet</a></span></span></td></tr><tr class="patent-bibdata-group-spacer"></tr></table><div class="number-and-title"><span class="patent-title"><invention-title mxw-id="PT54233717" lang="EN" load-source="patent-office">Vector for expression of a polypeptide in a mammalian cell</invention-title></span><br><span class="patent-number">US 5688688 A</span></div><div class="patent-section patent-abstract-section"><div class="patent-section-header"><span class="patent-section-title">Abstract</span></div><div class="patent-text"><abstract mxw-id="PA37699539" lang="EN" load-source="patent-office"> <div class="abstract">Polynucleotide sequences are provided for the diagnosis of the presence of retroviral infection in a human host associated with lymphadenopathy syndrome and/or acquired immune deficiency syndrome, for expression of polypeptides and use of the polypeptides to prepare antibodies, where both the polypeptides and antibodies may be employed as diagnostic reagents or in therapy, e.g., vaccines and passive immunization. The sequences provide detection of the viral infectious agents associated with the indicated syndromes and can be used for expression of antigenic polypeptides.</div>
  </abstract></div></div><div class="patent-section patent-drawings-section"><div class="patent-section-header"><span class="patent-section-title">Images<span class="patent-section-count">(59)</span></span></div><div class="patent-drawings-body"><div class="patent-drawings-carousel"><div class="drawings"><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5688688-1.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5688688-1.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5688688-2.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5688688-2.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5688688-3.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5688688-3.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5688688-4.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5688688-4.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5688688-5.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5688688-5.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5688688-6.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5688688-6.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5688688-7.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5688688-7.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5688688-8.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5688688-8.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5688688-9.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5688688-9.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5688688-10.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5688688-10.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5688688-11.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5688688-11.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5688688-12.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5688688-12.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5688688-13.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5688688-13.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5688688-14.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5688688-14.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5688688-15.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5688688-15.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5688688-16.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5688688-16.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5688688-17.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5688688-17.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5688688-18.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5688688-18.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5688688-19.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5688688-19.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5688688-20.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5688688-20.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5688688-21.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5688688-21.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5688688-22.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5688688-22.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5688688-23.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5688688-23.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5688688-24.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5688688-24.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5688688-25.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5688688-25.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5688688-26.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5688688-26.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5688688-27.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5688688-27.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5688688-28.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5688688-28.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5688688-29.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5688688-29.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5688688-30.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5688688-30.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5688688-31.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5688688-31.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5688688-32.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5688688-32.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5688688-33.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5688688-33.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5688688-34.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5688688-34.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5688688-35.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5688688-35.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5688688-36.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5688688-36.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5688688-37.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5688688-37.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5688688-38.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5688688-38.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5688688-39.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5688688-39.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5688688-40.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5688688-40.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5688688-41.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5688688-41.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5688688-42.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5688688-42.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5688688-43.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5688688-43.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5688688-44.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5688688-44.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5688688-45.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5688688-45.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5688688-46.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5688688-46.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5688688-47.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5688688-47.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5688688-48.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5688688-48.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5688688-49.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5688688-49.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5688688-50.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5688688-50.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5688688-51.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5688688-51.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5688688-52.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5688688-52.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5688688-53.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5688688-53.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5688688-54.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5688688-54.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5688688-55.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5688688-55.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5688688-56.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5688688-56.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5688688-57.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5688688-57.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5688688-58.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5688688-58.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5688688-59.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5688688-59.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div></div></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img" alt="Previous page"src="/googlebooks/images/kennedy/page_left.png"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img" alt="Next page"src="/googlebooks/images/kennedy/page_right.png"width="21" height="21" /></div></div></div><div class="patent-post-drawings"></div><div class="patent-section patent-claims-section"><div class="patent-section-header"><span class="patent-section-title">Claims<span class="patent-section-count">(24)</span></span></div><div class="patent-text"><div mxw-id="PCLM5155936" lang="EN" load-source="patent-office" class="claims">
    <claim-statement>We claim:</claim-statement> <div class="claim"> <div num="1" class="claim">
      <div class="claim-text">1. A vector for expression of a polypeptide in a mammalian cell comprising a first polynucleotide sequence that comprises:<div class="claim-text">a) an upstream SV40 origin of replication;</div> <div class="claim-text">b) a downstream SV40 polyadenylation region; and</div> <div class="claim-text">c) a transcription regulatory region from human cytomegalovirus immediate early region HCMV IE1, wherein the transcription regulatory region includes the first HCMV IE1 intron proximal to the 3' end of the HCMV IE1 promoter, is interposed between the SV40 origin of replication and the SV40 polyadenylation region, and is capable of directing the transcription of a polypeptide coding sequence operably linked downstream from the transcription regulatory region.</div> </div>
    </div>
    </div> <div class="claim-dependent"> <div num="2" class="claim">
      <div class="claim-text">2. The vector of claim 1, wherein the polynucleotide sequence further comprises a linker that comprises a restriction site for insertion of the coding region of a polypeptide.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="3" class="claim">
      <div class="claim-text">3. The vector of claim 2, wherein the restriction site is a SalI site.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="4" class="claim">
      <div class="claim-text">4. The vector of claim 1, wherein the SV40 polyadenylation region comprises the SV40 polyadenylation sequence present in plasmid pSV7d.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="5" class="claim">
      <div class="claim-text">5. The vector of claim 1, wherein the SV40 origin of replication comprises the SV40 origin of replication sequence present in plasmid pSVT2.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="6" class="claim">
      <div class="claim-text">6. The vector of claim 1, further comprising a selectable marker.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="7" class="claim">
      <div class="claim-text">7. The vector of claim 5, wherein the selectable marker is a polynucleotide sequence that encodes ampicillin resistance.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="8" class="claim">
      <div class="claim-text">8. The vector of claim 1, further comprising a bacterial origin of replication.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="9" class="claim">
      <div class="claim-text">9. The vector of claim 1, wherein the polynucleotide sequence comprises the HCMV sequences present in plasmid pCMV6ARV120tpa, ATCC Accession No. 68249.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="10" class="claim">
      <div class="claim-text">10. The vector of claim 2, further comprising a coding region that encodes a polypeptide, inserted at the restriction site.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="11" class="claim">
      <div class="claim-text">11. The vector of claim 10, further comprising a region encoding a signal sequence effective in directing the secretion of the polypeptide encoded by the coding region, positioned upstream from the coding region.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="12" class="claim">
      <div class="claim-text">12. The vector of claim 11, wherein the signal sequence is derived from the human tissue plasminogen activator leader sequence.</div>
    </div>
    </div> <div class="claim"> <div num="13" class="claim">
      <div class="claim-text">13. A vector produced by the process comprising linking together in an operative manner:<div class="claim-text">a) a SV40 origin of replication;</div> <div class="claim-text">b) a SV40 polyadenylation region; and</div> <div class="claim-text">c) a transcription regulatory region from human cytomegalovirus immediate early region HCMV IE1, wherein said regulatory region includes the first HCMV IE1 intron proximal to the 3' end of the HCMV IE1 promoter and is capable of directing the transcription of a polypeptide coding sequence operably linked downstream therefrom.</div> </div>
    </div>
    </div> <div class="claim-dependent"> <div num="14" class="claim">
      <div class="claim-text">14. The vector of claim 13, wherein the vector is arranged in the same manner as plasmid pCMV6a.</div>
    </div>
    </div> <div class="claim"> <div num="15" class="claim">
      <div class="claim-text">15. A method for producing a vector for expression of a polypeptide in a mammalian cell comprising:<div class="claim-text">a) providing a first polynucleotide molecule that comprises a SV40 origin of replication;</div> <div class="claim-text">b) providing a second polynucleotide molecule that comprises a SV40 polyadenylation region;</div> <div class="claim-text">c) providing a third polynucleotide molecule that comprises a transcription regulatory region from human cytomegalovirus immediate early region HCMV IE1, wherein said regulatory region includes the first HCMV IE1 intron proximal to the 3' end of the HCMV IE1 promoter; and</div> <div class="claim-text">d) linking the SV40 origin of replication, the SV40 polyadenylation region and the regulatory region from HCMV IE1 together to form a vector that is capable of effecting the transcription of a polypeptide coding sequence operatively linked downstream from the regulatory region.</div> </div>
    </div>
    </div> <div class="claim-dependent"> <div num="16" class="claim">
      <div class="claim-text">16. A method for producing the vector of claim 1, comprising introducing the vector into a host cell and allowing the host cell to generate a plurality of said vectors.</div>
    </div>
    </div> <div class="claim"> <div num="17" class="claim">
      <div class="claim-text">17. An isolated nucleic acid molecule comprising an enhanced promoter, wherein the enhanced promoter comprises the human cytomegalovirus immediate early region HCMV IE1 promoter and the first intron proximate to the 3' end of the HCMV IE1 promoter.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="18" class="claim">
      <div class="claim-text">18. The nucleic acid molecule of claim 17, wherein the promoter region is derived from a subclone of human cytomegalovirus (Towne strain).</div>
    </div>
    </div> <div class="claim-dependent"> <div num="19" class="claim">
      <div class="claim-text">19. A vector for expression of a polypeptide in a mammalian cell, comprising the nucleic acid molecule of claims 17, wherein the nucleic acid molecule is capable of directing the transcription of a polypeptide coding sequence operably linked downstream of the nucleic acid molecule.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="20" class="claim">
      <div class="claim-text">20. The vector of claim 19, further comprising an origin of replication operably linked upstream of the nucleic acid molecule.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="21" class="claim">
      <div class="claim-text">21. The vector of claim 19, further comprising a polyadenylation region operably linked downstream of the nucleic acid molecule.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="22" class="claim">
      <div class="claim-text">22. A vector for expression of a polypeptide in a mammalian cell, comprising:<div class="claim-text">a) an upstream origin of replication;</div> <div class="claim-text">b) a downstream polyadenylation region; and</div> <div class="claim-text">c) the nucleic acid molecule of claim 17 interposed between the origin of replication and the polyadenylation region, wherein the enhanced promoter region is capable of directing the transcription of a polypeptide coding sequence operably linked downstream from the promoter region.</div> </div>
    </div>
    </div> <div class="claim-dependent"> <div num="23" class="claim">
      <div class="claim-text">23. A method for constructing the vector of claim 19, comprising operatively linking together the nucleic acid molecule and the polypeptide coding sequence.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="24" class="claim">
      <div class="claim-text">24. A method for producing the vector constructed in claim 23, comprising introducing the vector into a host cell that is capable of replicating the vector and allowing the host cell to replicate the vector.</div>
    </div>
  </div> </div></div></div><div class="patent-section patent-description-section"><div class="patent-section-header"><span class="patent-section-title">Description</span></div><div class="patent-text"><div mxw-id="PDES67044351" lang="EN" load-source="patent-office" class="description">
    <heading>CROSS-REFERENCE TO RELATED APPLICATIONS</heading> <p>This application is a divisional of application Ser. No. 08/107,377, filed Aug. 17, 1993, which is a divisional of application Ser. No. 08/083,391, filed Jun. 28, 1993, which is a continuation of application Ser. No. 07/931,191, filed Aug. 17, 1992, which is a divisional of application Ser. No. 07/138,894, filed Dec. 24, 1987, now U.S. Pat. No. 5,156,949, which is a continuation-in-part of U.S. patent application Ser. No. 773,447, filed 6 Sep. 1985, now abandoned, which is a continuation-in-part of U.S patent application Ser. No. 696,534, filed 30 Jan. 1985, now abandoned, which is a continuation-in-part of U.S. patent application Ser. No. 667,501, filed 31 Oct. 1984, now abandoned. The disclosures of the above application are incorporated herein by reference.</p>
    <heading>TECHNICAL FIELD</heading> <p>The present invention is directed to nucleotide sequences, such as DNA, encoding human immunodeficiency virus polypeptides, the use of such nucleotide sequences in diagnostic procedures and in the production of recombinant protein, as well as the use of such proteins in diagnostic, prophylactic, and therapeutic applications.</p>
    <heading>BACKGROUND OF THE INVENTION</heading> <p>Acquired immune deficiency syndrome (AIDS) is now recognized as one of the greatest health threats facing modern medicine. There is, as yet, no cure for this almost invariably fatal disease. This state of affairs has made the prevention of the disease an extremely high priority in the medical community. An individual who is infected with human immunodeficiency virus (HIV), the etiologic agent of AIDS, can transmit the disease, and yet remain asymptomatic for many years. The ability to accurately screen large numbers of asymptomatic individuals (e.g., healthy appearing blood donors) for HIV infection is of great importance. Furthermore, the development of a vaccine would be particularly desirable, since it would afford some protection against transmission of AIDS by individuals who either are not detected by a diagnostic test, or evade such a test.</p>
    <p>In 1983-1984, three groups independently identified the suspected etiological agent of AIDS. See, e.g., Barre-Sinoussi et al. (1983) Science 220: 868-871; Montagnier et al., in Human T-Cell Leukemia Viruses (Gallo, Essex &amp; Gross, eds., 1984); Vilmer et al. (1984) The Lancet 1: 753; Popovic et al. (1984) Science 224: 497-500; Levy et al. (1984) Science 225: 840-842. These isolates were variously called lymphadenopathy-associated virus (LAV), human T-cell lymphotropic virus type III (HTLV-III), or AIDS-associated retrovirus (ARV). All of these isolates are strains of the same virus, and were later collectively named human immunodeficiency virus (HIV). With the isolation of a related AIDS-causing virus, the strains originally called HIV are now termed HIV-1 and the related virus is called HIV-2. See, e.g., Guyader et al. (1987) Nature 326: 662-669; Brun-Vezinet et al. (1986) Science 233: 343-346; Clavel et al. (1986) Nature 324: 691-695.</p>
    <p>Initially, HIV was propagated in culture in human mitogen-activated T cells. This method, however, could not produce the large quantities of virus required for serology assays on the scale required to protect public health and safety. It was not until immortalized cell lines capable of becoming chronically infected in vitro were discovered that HIV could be produced in any substantial quantities. See, e.g., Montagnier et al. (1984) Science 225: 63-66; Levy et al., supra; Popovic et al., supra. The ability to grow the virus in culture led to the development of immunoassays for the detection of anti-HIV antibodies in the blood of patients suspected of having been infected, as well as for screening blood donors. See, e.g., Schupbach et al. (1984) Science 224: 503-505; Sarngadharan et al. (1984) Science 224: 506-508; Feorino et al. (1984) Science 225: 69-72; Kalyanaraman et al. (1984) Science 225: 321-323; Culliton et al. (1984) Science 226: 1128-1131; Groopman et al. (1984) Science 226: 447-449; Ho et al. (1984) Science 226: 451-453; U.S. Pat. No. 4,520,113.</p>
    <p>Due to the great hazard of cultivating HIV in vitro, the number of facilities and individuals capable of working with the virus is necessarily limited. Furthermore, while tissue culture may provide viral polypeptides suitable for use in diagnostic assays, it is highly undesirable to employ polypeptides produced by tissue culture in vaccine compositions due to the risk of infectivity posed by live, intact virus.</p>
    <p>While production of vital polypeptides by recombinant means could be considered to be a solution to the problems described above, the production of recombinant proteins was not possible prior to the present invention. For example, HIV nucleotide sequences were not available and sequenced so as to enable the production of recombinant proteins. Even more importantly, it was unknown whether recombinantly produced viral protein would be sufficiently similar in antigenic properties to native HIV polypeptides so as to be generally useful in diagnostic assays or vaccine production. In addition, homology between the genome of HIV and human T-cell leukemia virus type I and type II (HTLV-I and -II) had been reported. See, e.g., Arya et al. (1984) Science 225: 927-930. Thus, it was unclear that sufficiently unique epitopes of HIV could be produced by recombinant means to distinguish HIV from HTLV-I or HTLV-II. Furthermore, it was unclear prior to the present invention whether the various HIV isolates possessed sufficiently related epitopes so that a recombinant polypeptide based on one isolate could be useful in a general diagnostic assay or vaccine composition.</p>
    <p>Prior to the present invention, therefore, recombinant HIV polypeptides could not be produced and it was not clear that such polypeptides would be generally useful in diagnostic, prophylactic, or therapeutic methods or products.</p>
    <heading>SUMMARY OF THE INVENTION</heading> <p>Nucleotide sequences and expression of nucleotide sequences are provided for detecting the presence of complementary sequences associated with a retroviral etiologic agent (HIV, e.g., HIV-1 or -2) for lymphadenopathy syndrome (LAS), acquired immune deficiency syndrome (AIDS) or AIDS-related complex (ARC), and for producing polypeptides. The single-stranded sequences are at least 20, more usually of at least about 50 nucleotides in length, and may find use as probes. The double-stranded sequences may find use as genes coding for expression of polypeptides, either fragments or complete polypeptides expressed by the virus or fused proteins, for use in diagnosis of HIV infection or evaluating stage of infection, the production of antibodies to HIV, and the production of vaccines. Based on the nucleotide sequences, synthetic peptides may also be prepared.</p>
    <p>Specific aspects of the invention include:</p>
    <p>1. A DNA construct comprising a replication system recognized by a unicellular microorganism and a DNA sequence coding for at least 20 bp of a human immunodeficiency virus (HIV) genome, said replication system being a non-HIV replication system;</p>
    <p>2. A DNA construct comprising a replication system recognized by a unicellular microorganism and a DNA sequence of at least about 21 bp having an open reading frame and having a sequence substantially complementary to a sequence found in the gag, env, or pol region of an HIV, coding for a polypeptide which is immunologically non-cross-reactive with HTLV-I and HTVL-II, and reactive with an HIV;</p>
    <p>3. A restriction endonuclease fragment of at least about 1.5 kbp derived from restriction enzyme digestion by at least one restriction endonuclease of a DNA sequence coding for an HIV of the class HIV-1;</p>
    <p>4. A DNA sequence comprising a fragment of at least about 20 bp, wherein the strands are complementary to a restriction endonuclease fragment described in 3 above, said sequence duplexing with am HIV nucleic acid sequence and not duplexing with HTLV-I or HTLV-II under comparable selective hybridization conditions;</p>
    <p>5. A method for detecting the presence of an HIV nucleic acid sequence present in a nucleic acid sample obtained from a physiological sample, which comprises:</p>
    <p>(a) combining said nucleic acid sample with a single-stranded nucleic acid sequence of at least about 20 bases complementary to a sequence in said HIV and non-cross-reactive with HTLV-I and -II under conditions of predetermined stringency for hybridization; and</p>
    <p>(b) detecting duplex formation between said DNA sequence and nucleic acid present in said sample;</p>
    <p>6. A method for cloning DNA specific for an HIV, which comprises growing a unicellular microorganism containing the above-described DNA construct, whereby said DNA sequence is replicated;</p>
    <p>7. A method for producing an expression product of HIV which comprises:</p>
    <p>(a) transforming a unicellular microorganism host with a DNA construct having transcriptional and translational initiation and termination regulatory signals functional in said host and an HIV DNA sequence of at least 21 bp having an open reading frame and under the regulatory control of said signals; and</p>
    <p>(b) growing said host in a nutrient medium, whereby said expression product is produced;</p>
    <p>8. A method for producing an expression product of HIV which comprises growing mammalian host cells having a DNA construct comprising transcriptional and translational initiation and termination regulatory signals functional in said host cells and a DNA sequence of at least 21 bp and less than the whole HIV genome, said sequence having an open reading frame and an initiation codon at its 5'-terminus and under the transcriptional and translational control of said regulatory signals, whereby a polypeptide encoded by said sequence is expressed;</p>
    <p>9. A method of detecting antibodies to HIV in a sample suspected of containing said antibodies comprising:</p>
    <p>(a) providing a support with at least one antigenic recombinant HIV polypeptide bound thereto;</p>
    <p>(b) contacting said sample with said support-bound polypeptide;</p>
    <p>(c) washing the support;</p>
    <p>(d) contacting the support with labeled antibody to human immunoglobulin; and</p>
    <p>(e) detecting the presence of said antibodies to HIV on said support via said label;</p>
    <p>10. Recombinant HIV polypeptides including, but not limited to:</p>
    <p>(a) p16gag;</p>
    <p>(b) p25gag;</p>
    <p>(c) an env polypeptide;</p>
    <p>(d) p31pol;</p>
    <p>(e) a fusion protein of p16gag and p25gag;</p>
    <p>(f) a fusion protein of a gag polypeptide and an env polypeptide;</p>
    <p>(g) a fusion protein comprising an env polypeptide;</p>
    <p>(h) a fusion protein comprising p31pol;</p>
    <p>(i) gp120env;</p>
    <p>(j) gp41env;</p>
    <p>(k) A fusion protein comprising env-5b; and</p>
    <p>(l) reverse transcriptase.</p>
    <p>11. An article of manufacture for use in an assay for anti-HIV antibodies comprising at least one of the above-described HIV polypeptides bound to a solid support.</p>
    <p>12. A vaccine composition, and a method of producing antibodies in a mammal comprising administering to said mammal said vaccine composition wherein the vaccine composition comprises an antigenically effective amount of a recombinant HIV polypeptide.</p>
    <p>Other embodiments will also be apparent from the description below.</p>
    <heading>BRIEF DESCRIPTION OF THE DRAWINGS</heading> <p>FIG. 1 is a restriction map of proviral DNA from HIV strain ARV-2.</p>
    <p>FIGS. 2 and 3 are restriction maps of recombinant  phages containing ARV-2 sequences.</p>
    <p>FIGS. 4A-4D is a complete nucleotide sequence of ARV-2, derived from partial sequences of several ARV clones. Corresponding amino acid sequences are indicated for the open reading frames of the individual genes.</p>
    <p>FIGS. 5A-5H is the nucleotide sequence of ARV-2(9B). The amino acid sequences for the products of the gag, pol, and env genes are indicated. The U3, R, and U5 regions of the LTRs are also designated. The cap site is position +1. The nucleotides at the beginning of each line are numbered, and the amino acids at the end of each line are indicated. FIG. 5 herein shows the same sequence as that in FIG. 5 of both U.S. Ser. No. 773,447 (filed 6 Sep. 1985) and U.S. Ser. No. 696,534 (filed 30 Jan. 1985), the nucleotides in the figure of the earlier applications being numbered from the beginning of the integrated sequences.</p>
    <p>FIG. 6 is a flow diagram showing the procedure for making the plasmid of pSV-7c/env, an expression vector for ARV-2 env gene.</p>
    <p>FIG. 7 is a flow diagram showing the procedures for making the plasmids pGAG25-10 and pGAG41-10.</p>
    <p>FIG. 8 is the nucleotide sequence of the p25gag gene cloned in plasmid pGAG25-10 and the amino acid sequence encoded by that gene.</p>
    <p>FIGS. 9A-9C is the coding strand of the nucleotide sequence cloned in pGAG41-10 for producing the fusion protein p41gag and the corresponding amino acid.</p>
    <p>FIG. 10 is a nucleotide sequence coding for p16gag protein that was cloned into plasmid ptac5 to make an expression plasmid for producing p16gag in bacteria.</p>
    <p>FIGS. 11A and 11B is a nucleotide sequence that encodes ARV-2 env protein that was used to prepare plasmid pDPC303.</p>
    <p>FIGS. 12A-12B is a nucleotide sequence that encodes ARV-2 p31 protein and is contained in plasmid pTP31.</p>
    <p>FIG. 13 is a map of the ARV env gene showing the regions env-1, env-2, env-3, env-4, and env-5.</p>
    <p>FIG. 14 is a restriction map of plasmid pDM15, which was used to construct S. cerevisiae strain JSC302.</p>
    <p>FIG. 15 is the synthetic nucleotide sequence env-5b, which encodes the amino acid sequence of the ARV env-5 region.</p>
    <p>FIGS. 16A and 16B is the results of an indirect ELISA in which an AIDS patient's serum () was titrated against microtiter plates coated with recombinant polypeptides from env regions. A pool of serum samples from random blood donors was used as a control (o). Panel A shows the results for purified, recombinant env-2. Panel B shows the results with purified, recombinant env-5b. The insert in each panel shows a Coomassie-stained gel (lane 1) and an immunoblot with the AIDS patient's serum (lane 2) of the purified antigens used in these ELISAs.</p>
    <p>FIG. 17 shows the results of an ELISA, employing recombinant env-2 (top panel) and env-5b (bottom panel) polypeptides, run on seronegative blood donors.</p>
    <p>FIG. 18 shows the results of an ELISA, employing recombinant env-2 (top panel) and env-5b (bottom panel) polypeptides, run on HIV seropositive patients, including those diagnosed as having AIDS or AIDS-related complex (ARC), as well as those having contacts with AIDS patients.</p>
    <p>FIG. 19 shows the results of ELISAs used to measure antibody titers in the AIDS seropositive patients of FIG. 18.</p>
    <p>FIG. 20 is a flow diagram shoving the procedure for making plasmid pAB24/RT4, an expression vector for HIV reverse transcriptase.</p>
    <p>FIG. 21 is a flow diagram showing the construction of pC1/1-p25-ADH-GAP, a yeast expression vector for p25gag.</p>
    <p>FIGS. 22A and 22B shows the DNA and amino acid sequences of the p25gag structural region in pC1/1-p25-ADH-GAP.</p>
    <p>FIG. 23 is a flow diagram showing the construction of pC1/1-pSP31-ADH-GAP (pC1/1-pSP31-GAP-ADH2), a yeast expression vector for a SOD/p31pol fusion protein.</p>
    <p>FIGS. 24A and 24B shows the DNA and amino acid sequences of the SOD/p31pol structural region in pC1/1-pSP31-ADH-GAP.</p>
    <p>FIG. 25 is a flow diagram showing the construction of pSOD/env5b from pSODCF2 and a synthetic env-5b sequence.</p>
    <p>FIGS. 26A and 26B shows the nucleotide sequence and putative amino acid sequence of the SOD/env-4 fusion construct in pBS24/SOD-SFenv4.</p>
    <p>FIG. 27 is a restriction map for yeast shuttle vector pXB24.</p>
    <p>FIG. 28 is a restriction map for yeast expression vector pAB-GAP-env2.</p>
    <p>FIG. 29 is a restriction map of pCMV6a.</p>
    <p>FIG. 30 is an immunoblot performed with AIDS patient serum on env-1 (lanes A, B), env-2 (lanes C, D) and env-3 (lanes E, F). Lanes A, C and E are immunoblots with normal sera, while lanes B, D and F are immunoblots with serum from an AIDS patient.</p>
    <p>FIGS. 31A and 32B shows an ELISA survey for p31 antibodies. Panel (a) shows the results for random, normal blood donors. Panel (b) shows the results for virus-seropositive individuals. The shaded bars are for sera that scored negative in the virus immunoblot assays.</p>
    <heading>MODES FOR CARRYING OUT THE INVENTION</heading> <p>Nucleotide sequences are provided which are at least in part specific for sequences present in HIV retroviruses, which are the etiological agent of AIDS. HIV is an art-recognized family of viruses, e.g., HIV-1 and HIV-2. The original isolates of these viruses were variably referred to as lymphadenopathy virus (LAV)  Barre-Sinoussi et al. (1983) Science 220: 868-871!, human T-cell 19mphotropic virus-III (HTLV-III)  Popovic et al. (1984) Science 224: 497! and AIDS-associated retrovirus (ARV)  Levy et al. (1984) Science 225: 840-842!. Applicants originally termed these isolates "human T-cell lymphotropic retrovirus (hTLR)". Subsequently, the name HIV has been given to these retroviruses by an international committee. Thus, HIV (and particularly HIV-1) shall be used herein as an equivalent to hTLR. Examples of HIV-1 were previously called LAV, ARV and HTLV-III. Among the identifying characteristics of HIV retroviruses are (i) being an etiologic of AIDS, (ii) being cytopathic in vitro, (iii) having a tropism for CD4-bearing cells, and (iv) having elements trans-activating the expression of viral genes acting at the LTR level.</p>
    <p>New HIVs may be shown to be of the same class by being similar in their morphology, serology, reverse transcriptase optima, cytopathology, amino acid sequence, and nucleotide sequence as known HIV strains. Coffin et al. (1986) Nature 321: 10. Within different HIV-1 isolates, for example, the gag and pol proteins shows about 90-95% homology at the amino acid level, and the env precursor shows about 65-85% homology (most of the variations being confined to certain "hypervariable" regions), with all 23 env cysteines being conserved. Alizon et al. (1986) Cell 46: 63-74. HIV-2, however, is a new class of the HIV family that is not a strain of HIV-1 according to the recommended criteria of the international taxonomy committee. See, e.g., Guyader et al. (1987) Nature 326: 662-669. HIV-1 and HIV-2 show an overall approximate amino acid homology of about 42%, with about 60% amino acid homology for the gag and pol proteins, and about 40% for the env precursor.</p>
    <p>The nucleotide sequences of this invention may be the entire sequence of the retrovirus and/or the provirus or may be fragments thereof based on restriction enzyme digestion of HIV (provirus and/or other dsDNA homologous to retrovirus RNA), which fragments may be all or part of the LTR, gag, pol, env, and/or other open reading frames, such as Q (or sor), R, tat, and art (or trs) (sometimes referred to by the designation "orf" herein), untranslated regions intermediate coding regions, and fragments and combinations thereof. The minimum size single-stranded fragment will be at least 20 bases and usually at least 50 bases and may be 100 bases or more, where the entire HIV is about 9.5 kb. The sequence may be obtained as a fragment from the HIV or be synthesized.</p>
    <p>The fragments can be used in a wide variety of ways, depending upon their size, their natural function, the use for which they are desired, and the degree to which they can be manipulated to modify their function. Thus, sequences of at least 20 bases, more usually at least 50 bases, and usually not exceeding about 1000 bases, more usually not exceeding about 500 bases, may serve as probes for detection of the presence of HIV in a host cell, including the genome, or in a physiological fluid, such as blood, lymph, saliva, spinal fluid, or the like. These sequences may include coding and/or non-coding sequences. The coding sequences may involve the gag, pol, env or other open reading frames, either in whole or in part. Where splicing occurs between, for example, a region in the LTR sequence and a coding sequence in another region, the joined DNA from the provirus, linked by in vitro manipulation, or from cDNA or cloned cDNA, may be employed.</p>
    <p>It is found that HIV is highly polymorphic. Therefore, not only may DNA prepared from various isolates vary by one or more point mutations, but even the passage of a single isolate may result in variation in the progeny. Thus, where the nucleotide sequences are used for duplex formation, hybridization, or annealing, for example, for diagnosis or monitoring of the presence of the virus in vivo or in vitro, complete base pairing will not be required. One or more mismatches are permissible. To ensure that the presence of one or a few, usually not more than three, mismatches still allows for stable duplexes under the predetermined stringency of hybridizing or annealing conditions, probes will normally be greater than 20 bases, preferably at least about 50 bases or more.</p>
    <p>The method of detection will involve duplex formation by annealing or hybridization of the oligonucleotide probe, either labeled or unlabeled, depending upon the nature of the detection system, with the DNA or RNA of a host suspected of harboring the provirus or virus. A physiological sample may include tissue, blood, saliva, serum, etc. Particularly, blood samples will be taken, more particularly blood samples containing peripheral mononuclear cells, which may be lysed and the DNA or RNA isolated in accordance with known techniques. Cells may be cultured to amplify virus in vitro, or treated to stimulate PBLs, thereby producing more virus. Conveniently, the cells are treated with a detergent, nucleic acids are extracted with organic solvents and precipitated in an appropriately buffered medium, and the DNA or RNA isolated. Depending upon the particular protocol, the DNA may be fragmented by mechanical shearing or restriction endonuclease digestion.</p>
    <p>The sample polynucleotide mixture obtained from the human host can be bound to a support or may be used in solution depending upon the nature of the protocol. The well-established Southern technique  (1975) J. Mol. Biol. 98: 503! may be employed with denatured DNA, by binding the single-stranded fragments to a nitrocellulose filter. Alternatively, RNA can be blotted on nitrocellulose following the procedure described by Thomas, (1980) Proc. Natl. Acad. Sci. (USA) 77: 5201. Desirably, the fragments will be electrophoresed prior to binding to a support, so as to be able to select for various sized fractions. Other techniques may also be used such as described in Meinkoth &amp; Wahl, (1984) Anal. Biochem. 138: 267-284.</p>
    <p>The oligonucleotide probe may be DNA or RNA, usually DNA. The oligonucleotide sequence may be prepared synthetically or in vivo by cloning, where the complementary sequence may then be excised from the cloning vehicle or retained with the cloning vehicle. Various cloning vehicles are available, such as pBR322, M13, Charon 4A, or the like, desirably a single-stranded vehicle, such as M13.</p>
    <p>As indicated, the oligonucleotide probe may be labeled or unlabeled. A wide variety of techniques exist for labeling DNA and RNA. As illustrative of such techniques, is radiolabeling using nick translation, tailing with terminal deoxytransferase, or the like, where the bases which are employed carry radioactive <sup>32</sup> P. Alternatively, radioactive nucleotides can be employed where carbon, nitrogen or other radioactive atoms may be part of the nucleoside structure. Other labels which may be used include fluorophores, enzymes, enzyme substrates, enzyme cofactors, enzyme inhibitors, or the like. Alternatively, instead of having a label which provides for a detectable signal by itself or in conjunction with other reactive agents, ligands can be used to which receptors bind, where the receptors are labeled such as with the above-indicated labels, which labels provide detectable signals by themselves or in conjunction with other reagents. See, e.g., Leary et al. (1983) Proc. Natl. Acad. Sci. (USA) 80: 4045-4049; Cosstick et al. (1984) Nucleic Acids Res. 12: 1791-1810; PCT Pub. No. WO 83/02277.</p>
    <p>The oligonucleotide probes are hybridized with the denatured human host nucleic acid, substantially intact or fragmented, or fractions thereof, under conditions of predetermined stringency. The stringency will depend upon the size and composition of the probe, the degree of mismatching, the desired cross reactivity with other strains of the subject HIV, and the like. Usually, an organic solvent such as formamide will be present in from about 30 to 60 vol percent, more usually from about 40 to 50 vol percent, with salt concentration from 0.5 to 1M. Temperatures will generally range from about 30 C. to 65 C., more usually from about 35 C. to 50 C. The times for duplex formation may be varied widely, although minimum times will Usually be at least about one hour and not more than about 72 hours, the time being selected in accordance with the amount of DNA or RNA available, the proportion of DNA or RNA as compared to total DNA or RNA, or the like. Stringency may also be modified by ionic strength and temperature. The hybridization and annealing can be carried out in two stages: a first stage in a hybridization medium; and, a second stage, involving washings at a higher stringency, by varying either or both-temperature and ionic strength.</p>
    <p>As understood in the art, the term "stringent hybridization conditions" as used herein refers to hybridization conditions which allow for closely related nucleic acid sequences to duplex (e.g., greater than about 90% homology), but not unrelated sequences. The appropriate conditions can be established by routine procedures, such as running Southern hybridization at increasing stringency until only related species are resolved and the background and/or control hybridization has disappeared (i.e., selective hybridization).</p>
    <p>The oligonucleotide probe may be obtained in a variety of ways. Viral RNA from HIV may be isolated from the supernatant of cells infected (e.g., HIV-1 or HIV-2) in culture, and the high molecular weight materials precipitated and the DNA removed, for example, employing DNase. The residual RNA may then be divided into molecular weight fractions, where the fraction associated with the molecular weight of the retrovirus is isolated. This fraction will be from about 8 to 10 kb viral RNA. The vital RNA may be further purified by conventional techniques, such as electrophoresis, chromatography, or the like.</p>
    <p>Nucleotide probes may be prepared employing reverse transcriptase using primers, e.g., random primers or specific primers. The cDNA may be prepared employing a radioactive label, e.g., <sup>32</sup> P, present with one or more of the dNTPs. Reverse transcription will provide various sized fragments depending on the primers, the efficiency of transcription, the integrity of the RNA, and the like. The resulting cDNA sequences may be cloned, separated and used for detection of the presence of a provirus in the human genome or for isolation of pure retroviral RNA.</p>
    <p>Using specific primers of 10 to 20 bases, or more, HIV may be reverse transcribed and the resulting ss DNA used as a probe specific for the region which hybridized to the primer. By employing one or more radionucleotide-labeled bases, the probes will be radiolabeled to provide a detectable signal. Alternatively, modified bases may be employed which will be randomly incorporated into the probe and may be used to provide for a detectable signal. For example, biotin-modified bases may be employed. The resulting biotin-containing probe may then be used in conjunction with labeled avidin to provide for a detectable signal upon hybridization and duplex formation.</p>
    <p>Of particular interest is employing the region containing the gag or env genes, where fragments may be employed to screen proviral DNA in infected cells, to determine the identity of retroviruses associated with AIDS or LAS obtained from different human hosts. Probes providing for the desired degree of cross-reactivity or absence of cross-reactivity may then be prepared in a form, either labeled or unlabeled, useful for diagnostic assays employing hybridization and annealing.</p>
    <p>The double-stranded DNA sequences, either isolated and cloned from proviral DNA or cDNA or synthesized, may be used for expression of polypeptides which may be a precursor protein subject to further manipulation by cleavage, or a complete mature protein or fragment thereof. The smallest sequence of interest, so as to encode an amino acid sequence capable of specific binding, for example, to a receptor or an immunoglobulin, will be 21 bp, usually at least 45 bp, exclusive of the initiation codon. The sequence may code for any greater portion of or the complete polypeptide, or may include flanking regions of a precursor polypeptide, so as to include portions of sequences or entire sequences coding for two or more different mature polypeptides. The sequence will usually be less than about 5 kbp, more usually less than about 3 kbp.</p>
    <p>The sequences having open reading frames as numbered in FIG. 4 are the genes beginning at nucleotide (nt) 838 to 2298 (gag); 2347 to 2825 (small polypeptide between gag and pol regions); 2965 to 5103 (pol); and 6236 to 8800 (env). It is to be understood that the above sequences may be spliced to other sequences present in the retrovirus, so that the 5'-end of the sequence may not code for the N-terminal amino acid of the expression product. The splice site may be at the 5'-terminus of the open reading frame or internal to the open reading frame. The initiation codon for the protein may not be the first codon for methionine, but may be the second or third methionine, so that employing the entire sequence indicated above may result in an extended protein. However, for the gag and env genes there will be proteolytic processing in mammalian cells, which processing may include the removal of extra amino acids.</p>
    <p>In isolating the different domains the provirus may be digested with restriction endonucleases, the fragments electrophoresed and fragments having the proper size and duplexing with a probe, when available, are isolated, cloned in a cloning vector, and excised from the vector. The fragments may then be manipulated for expression. Superfluous nucleotides may be removed from one or both termini using Ba131 digestion. By restriction mapping, convenient restriction sites may be located external or internal to the coding region. Primer repair or in vitro mutagenesis may be employed for defining a terminus, for insertions, deletion, point or multiple mutations, or the like, where codons may be changed, either cryptic or changing the amino acid, restriction sites introduced or removed, or the like. Where the gene has been truncated, the lost nucleotides may be replaced using an adaptor. Adaptors are particularly useful for joining coding regions to ensure the proper reading frame.</p>
    <p>The env domain of HIV can be obtained by digestion of the provirus with EcoRI and KpnI and purification of a 3300 base pair (bp) fragment, which fragment contains about 400 bp of 5' non-coding and about 200 bp of 3' non-coding region. Three different methionines coded for by the sequence in the 5' end of the open reading frame may serve as translational initiation sites.</p>
    <p>The open reading frame of the env gene of ARV-2 has a coding capacity of 863 amino acids. Portions of the env gene coding for the polypeptides shown in FIG. 5 were produced in S. cerevisiae using yeast expression vectors. See FIG. 13. Env-2, encompassing amino acid residues 26 to 510, corresponds to the major portion of the mature envelope glycoprotein, gp120, that is external to viral and infected cell membranes. Env-1 includes amino acid residues 26 to 276 and represents approximately the amino-terminal half of the gp120 polypeptide. Env-3, stretching between amino acid residues 529 to 855, corresponds to the portion of the env gene which encodes gp41, the viral glycoprotein that spans membranes and serves as an anchor for the envelope glycoprotein complex. Env-4, amino acid residues 272 to 509, correspond to the carboxyl terminal half of gp120. Env-5b, encompassing amino acid residues 557 to 677, corresponds to the region of gp41 stretching between the two hydrophobic domains. These various recombinant portions of the env domain are valuable in diagnostic assays for HIV infections, particularly env-2 and env-5b.</p>
    <p>Digestion of proviral sequences with SacI and EcoRV provides a fragment of about 2300 bp which contains the gag domain and a second small open reading frame towards the 3' end of the gag region. The gag domain is about 1500 bp and codes for a large precursor protein which is processed to yield proteins of about 25,000 (p25), 16,000 (p16) and 12,000 (p12) daltons. Digestion with SacI and BglII may also be used to obtain exclusively the gag domain with p12, p25 and partial p16 regions.</p>
    <p>Digestion of the previous with KpnI and SstI provides a fragment containing the portion of the pol domain that encodes p31. Native HIV reverse transcriptase (RT) is purified from virions in p66 and p51 forms. Both of these forms have identical N-termini, apparently differing at the C-termini. RT is encoded within a domain of the viral pol gene. The mature enzyme is derived by proteolytic processing from a large precursor polypeptide whose cleavage is thought to be mediated by a viral protease. This protease, by analogy with other retroviruses, also cleaves the gag gene precursor. For direct expression of the RT domain in yeast, the N- and C-termini of the mature protein were estimated by drawing on homology comparisons with the amino acid sequences of pol gene products of other retroviruses. Precise amino acid choices for termini were based on the target specificities of retroviral proteases, including the AIDS virus protease, from known gag subunit sequences. Accordingly, the Phe-Pro at positions 155 and 156 of the ARV-2 pol open reading frame and the Val-Pro at positions 163 and 164 were selected as likely N-termini. A likely C terminal processing site was estimated at the Val-Pro of positions 691 and 692. See FIG. 5. Recombinant RT is valuable in diagnostic assays for HIV infections.</p>
    <p>The polypeptides which are expressed by the above DNA sequences may find use in a variety of ways. The polypeptides or immunologically active fragments thereof, may find use as diagnostic reagents, being used in labeled or unlabeled form or immobilized (i.e., bound to a solid surface), as vaccines, in the production of monoclonal antibodies, e.g., inhibiting antibodies, or the like.</p>
    <p>The DNA sequences may be joined with other sequences, such as viruses, e.g., vaccinia virus or adenovirus, to be used for vaccination. Particularly, the DNA sequence of the viral antigen may be inserted into the vaccinia virus at a site where it can be expressed, so as to provide an antigen of HIV recognized as an immunogen by the host. The gag, pol, or env genes or fragments thereof that encode immunogens could be used.</p>
    <p>Another alternative is to join the gag, env, or pol regions or portions thereof to HBsAg gene or pre-S HBsAg gene or immunogenic portions thereof, which portion is capable of forming particles in a unicellular microorganism host, e.g., yeast or mammalian cells. Thus, particles are formed which will present the immunogen to the host in immunogenic form, when the host is vaccinated with assembled particles.</p>
    <p>As vaccines, the various forms of the immunogen can be administered in a variety of ways, orally, parenterally, intravenously, intra-arterially, subcutaneously, intramuscularly, or the like. Usually, these will be provided in a physiologically acceptable vehicle, generally distilled water, phosphate-buffered saline, physiological saline, buffers containing SDS or EDTA, and the like. Various adjuvants may be included, such as aluminum hydroxide, MTP in saline and Tween 80, and the dosages, number of times of administration and manner of administration determined empirically.</p>
    <p>In order to obtain the HIV sequence (e.g., HIV-1 or HIV-2), virus can be pelleted from the supernatant of infected host cells. A 9 kbRNA species is purified by electrophoresis of the viral RNA in low-melting agarose gels, followed by phenol extraction. The purified RNA may then be used as a template with random primers in a reverse transcriptase reaction. The resulting cDNA is then screened for hybridization to polyA+ RNA from infected and uninfected cells, or to one of  vectors containing HIV DNA disclosed herein. For the polyA+ RNA, hybridization occurring from infected, but not uninfected cells, is related to HIV.</p>
    <p>Genomic DNA from infected cells can be digested with restriction enzymes and used to prepare a bacteriophage library. Based upon restriction analysis of the previously obtained fragments of the retrovirus, the viral genome can be partially digested with EcoRI and 9 kb-15 kb DNA fragments isolated and employed to prepare the library. The resulting recombinant phage may be screened using a double-lift screening method employing the viral cDNA probe, followed by further purification, e.g., plaque-purification and propagation in large liquid cultures. From the library, the complete sequence of the virus can be obtained and detected with the previously described probe.</p>
    <p>HIV DNA (either provirus or cDNA) may be cloned in any convenient vector. Constructs can be prepared, either circular or linear, where the HIV DNA, either the entire HIV or fragments thereof, may be ligated to a replication system functional in a microorganism host, either prokaryotic or eukaryotic cells (mammalian, yeast, arthropod, plant). Micro-organism hosts include E. coli B. subtilis, P. aerugenosa, S. cerevisiae, N. crassa, etc. Replication systems may be derived from ColEl, 2 m plasmid, , SV40, bovine papilloma virus, or the like, that is, both plasmids and viruses. Besides the replication system and the HIV DNA, the construct will usually also include one or more markers, which allow for selection of transformed or transfected hosts. Markers may include biocide resistance, e.g., resistance to antibiotics, heavy metals, etc., complementation in an auxotrophic host to provide prototrophy, and the like.</p>
    <p>To produce recombinant polypeptides, expression vectors will be employed. For expression in microorganisms, the expression vector may differ from the cloning vector in having transcriptional and translational initiation and termination regulatory signal sequences and may or may not include a replication system which is functional in the expression host. The coding sequence is inserted between the initiation and termination regulatory signals so as to be under their regulatory control. Expression vectors may also include the use of regulable promoters, e.g., temperature-sensitive or inducible by chemicals, or genes which will allow for integration and amplification of the vector and HIV DNA such as tk, dhfr, metallothionein, or the like.</p>
    <p>The expression vector is introduced into an appropriate host where the regulatory signals are functional in such host. The expression host is grown in an appropriate nutrient medium, whereby the desired polypeptide is produced and isolated from cells or from the medium when the polypeptide is secreted.</p>
    <p>Where a host is employed in which the HIV transcriptional and translational regulatory signals are functional, then the HIV DNA sequence may be manipulated to provide for expression of the desired polypeptide in proper juxtaposition to the regulatory signals.</p>
    <p>The polypeptide products can be obtained in substantially pure form, particularly free of debris from human cells, which debris may include such contaminants as proteins, polysaccharides, lipids, nucleic acids, viruses, bacteria, fungi, etc., and combinations thereof. Generally, the polypeptide products will have less than about 10-15 weight percent, preferably less than about 5 weight percent, of contaminating materials from the expression host. Depending upon whether the desired polypeptide is produced in the cytoplasm or secreted, the manner of isolation will vary. Where the product is in the cytoplasm, the cells are harvested, lysed, the product extracted and purified, using solvent extraction, chromatography, gel exclusion, electrophoresis, or the like. Where secreted, the desired product will be extracted from the nutrient medium and purified in accordance with the methods described above.</p>
    <p>In many cases it will be desirable to express the recombinant HIV polypeptide as a fusion protein. This is particularly true with polypeptides such as p31pol and the transmembrane region of gp41env (env-5), to obtain improved levels of expression. The fusion proteins approach allows the addition of a signal sequence to the HIV polypeptide so that the product is secreted by the expression host. Generally, the DNA sequence for the HIV polypeptide is in the C-terminal portion of the fused gene, the heterologous sequence making up the N-terminal. The choice of the appropriate heterologous sequence for fusion to the HIV sequence is a matter of choice within the skill of the art. Preferred heterologous sequences include the N-termini of -galactosidase and human superoxide dismutase. It is usually preferable that the heterologous sequence be non-immunogenic to humans. In one embodiment, however, two HIV sequences from different immunogenic domains of the virus, such as gag and env, are fused together. This produces a single fusion protein with the immunogenic potential of the two parent polypeptides.</p>
    <p>The expression products of the env, gag, and pol genes and immunogenic fragments thereof having immunogenic sites may be used for screening antisera from patients' blood to determine whether antibodies are present which bind to HIV antigens. One or more of the antigens may be used in the assay. Preferred modes of the assay employ a combination of gag and env antigens or pol and env antigens. A combination of p25gag, p16gag, or p31pol and env antigens is particularly preferred. A wide variety of assay techniques can be employed, involving labeled or unlabeled antigens or immobilized antigens. The label may be fluorescers, radionuclides, enzymes, chemiluminescers, magnetic particles, enzyme substrates, cofactors or inhibitors, ligands, or the like.</p>
    <p>A particularly convenient technique is to bind the antigen to a support that will bind proteins, such as the surface of an assay tube, a well of an assay plate, or a strip of material like nitrocellulose or nylon, and then contact the sample with the immobilized antigen. After washing the support to remove non-specifically bound antisera, labeled antibodies to human Ig are added and specifically bound label determined.</p>
    <p>ELISA and "dot-blot" assays are particularly useful for screening blood or serum samples for anti-HIV antibodies. The ELISA assay uses microtiter trays having wells that have been coated with the antigenic HIV polypeptides(s). The wells are also typically post-coated with a nonantigenic protein to avoid nonspecific binding of antibodies in the sample to the well surface. The sample is deposited in the wells and incubated therein for a suitable period under conditions favorable to antigen-antibody binding.</p>
    <p>Anti-HIV antibodies present in the sample will bind to the antigen(s) on the well wall. The sample is then removed and the wells are washed to remove any residual, unbound sample. A reagent containing enzyme-labeled antibodies to human immunoglobulin is then deposited in the wells and incubated therein to permit binding between the labeled anti-human Ig antibodies and HIV antigen-human antibody complexes bound to the well wall. Upon completion of the incubation, the reagent is removed and the wells washed to remove unbound labeled reagent. A substrate reagent is then added to the wells and incubated therein. Enzymatic activity on the substrate is determined visually or spectrophotometrically and is an indication of the presence and amount of anti-HIV antibody-containing immune complex bound to the well surface.</p>
    <p>The "dot-blot" procedure involves using HIV antigen(s) immobilized on a piece or strip of bibulous support material, such as nitrocellulose filter paper or nylon membrane, rather than antigen-coated microtiter trays. The support will also be treated subsequently with a nonantigenic protein to eliminate nonspecific binding of antibody .to the support. The antigen-carrying support is contacted with (e.g., dipped into) the sample and allowed to incubate therein. Again, any anti-HIV antibodies in the sample will bind to the antigen(s) immobilized on the support. After a suitable incubation period the support is withdrawn from the sample and washed in buffer to remove any unbound sample from the paper. The support is then incubated with the enzyme-labeled antibody to human Ig reagent for a suitable incubation period. Following treatment with the labeled reagent the support is washed in buffer, followed by incubation in the substrate solution. Enzymatic activity, indicating the presence of anti-HIV antibody-containing complexes on the support, causes color changes on the support which may be detected optically.</p>
    <p>Either of these techniques may be modified to employ labels other than enzymes, or to detect non-human anti-HIV antibodies (e.g., primate). The reading or detection phases will be altered accordingly.</p>
    <p>The antigenic HIV polypeptide may also be used as immunogens by themselves or joined to other antigens for the production of antisera or monoclonal antibodies which may be used for therapy or diagnosis. When used as immunogens, the HIV polypeptides can be prepared as vaccine compositions, as is known in the art. The immunoglobulins may be from any mammalian source, e.g., rodent, such as rat or mouse, primate, such as baboon, monkey or human, or the like. For diagnosis, the antibodies can be used in conventional ways to detect HIV in a clinical sample.</p>
    <heading>EXAMPLES</heading> <p>The following examples are offered for illustrative purposes only, and are not intended to limit the scope of the claims in any way. The examples are organized as follows:</p>
    <p>1. Isolation, Cloning and Characterization of HIV</p>
    <p>1.1. Purification and preparation of viral RNA</p>
    <p>1.2. Synthesis of labeled viral probe</p>
    <p>1.3. Detection of HIV RNA and DNA in mammalian cells</p>
    <p>1.4. Cloning of proviral HIV DNA</p>
    <p>1.5. Polymorphism of HIV</p>
    <p>1.6. Sequencing of proviral DNA</p>
    <p>1.7. Amino acid sequences of native HIV proteins</p>
    <p>2. Expression of HIV Polypeptides in Mammalian Cells</p>
    <p>2.1. Expression of env peptide</p>
    <p>2.1.1. Mammalian expression vector pSV-7c</p>
    <p>2.1.2. Transformation of cells with env sequences</p>
    <p>2.1.3. Detecting expression via immunofluorescence</p>
    <p>2.2 Expression of gp160env</p>
    <p>2.2.1. Mammalian expression vector pSV7d</p>
    <p>2.2.2. Expression of tPA/gp160</p>
    <p>2.3 Expression of gp120env</p>
    <p>2.3.1. Expression of tPA/gp120 in pSV7d</p>
    <p>2.3.2. Expression of gp120 using CMV IE-1 promoter</p>
    <p>2.4. Expression of gag peptides</p>
    <p>2.5. Expression of gag-env fusion protein</p>
    <p>3. Expression of HIV Polypeptides in Bacteria</p>
    <p>3.1. Expression of p25gag</p>
    <p>3.1.1. Host-vector system</p>
    <p>3.1.2. Construction of pGAG25-10</p>
    <p>3.1.3. Transformation and expression of gag peptide</p>
    <p>3.1.4. Fermentation process</p>
    <p>3.1.4.1. Preparation of master seed stock</p>
    <p>3.1.4.2. Fermenter inoculum</p>
    <p>3.1.4.3. Fermentation and harvest</p>
    <p>3.1.5. Purification of p25gag</p>
    <p>3.1.6. Characterization of recombinant p25gag</p>
    <p>3.1.6.1. Protein gel electrophoresis</p>
    <p>3.1.6.2. Western analysis</p>
    <p>3.1.6.3. ELISA comparison of recombinant and native p25gag</p>
    <p>3.2. Expression of p16gag</p>
    <p>3.3. Expression of fusion protein p41gag</p>
    <p>3.4. Expression of p31pol</p>
    <p>3.4.1. Host-vector system</p>
    <p>3.4.2. Construction of pTP31.2</p>
    <p>3.4.2.1. Construction of M13 template 01100484</p>
    <p>3.4.2.2. In vitro mutagenesis of 01100484</p>
    <p>3.4.2.3. Isolation of DNA fragments containing p31</p>
    <p>3.4.2.4. Cloning of p31 into plot 7</p>
    <p>3.4.2.5. Construction of pTP31</p>
    <p>3.4.3. Screening of transformants</p>
    <p>3.4.4. Characterization of recombinant protein</p>
    <p>3.5. Expression of SOD-p31 fusion protein</p>
    <p>3.6. Expression of SOD-env5b fusion protein</p>
    <p>3.6.1. Host-Vector System</p>
    <p>3.6.2. Construction of pSOD/env5b</p>
    <p>3.6.3. Expression of SOD/env5b fusion protein</p>
    <p>3.6.4. Protein Purification</p>
    <p>3.7. Expression of -gal-env fusion proteins</p>
    <p>3.7.1. Host-vector system</p>
    <p>3.7.2. Construction of pII-3</p>
    <p>3.7.3. Expression and characterization of fusion protein</p>
    <p>4. Expression of HIV Polypeptides in Yeast</p>
    <p>4.1. Expression of envH peptide</p>
    <p>4.1.1. Host-vector system</p>
    <p>4.1.2. Construction of pDPC303</p>
    <p>4.1.3. Transformation and expression</p>
    <p>4.1.4. Purification of envH protein</p>
    <p>4.1.4.1. Cell breakage</p>
    <p>4.1.4.2. SDS extraction of insoluble material</p>
    <p>4.1.4.3. Selective precipitation and gel filtration</p>
    <p>4.1.5. Characterization of recombinant envH</p>
    <p>4.2. Expression of env subregion polypeptides</p>
    <p>4.2.1. Env-1</p>
    <p>4.2.1.1. GAP promoter</p>
    <p>4.2.2. Env-2</p>
    <p>4.2.2.1. Construction of pAB24-GAP-env2</p>
    <p>4.2.2.2. Transformation and expression</p>
    <p>4.2.2.3. Purification of env-2 protein</p>
    <p>4.2.2.4. Immunogenicity</p>
    <p>4.2.3. Env-3</p>
    <p>4.2.3.1. GAP promoter</p>
    <p>4.2.3.2. ADH-2/GAPDH promoter</p>
    <p>4.2.4. Env-4</p>
    <p>4.2.4.1. pBS24/SF2env4/GAP</p>
    <p>4.2.4.2. pBS24</p>
    <p>4.2.4.3. pBS24/SOD-SF2env4</p>
    <p>4.2.5. ySOD/env-5b fusion protein</p>
    <p>4.2.6. Env 4-5</p>
    <p>4.2.6.1. Construction of pBS24.1/SOD-SF2env 4-5</p>
    <p>4.2.6.2. Transformation and expression</p>
    <p>4.2.6.3. Protein purification</p>
    <p>4.3. p31pol</p>
    <p>4.3.1. GAP/ADH2 promoter</p>
    <p>4.3.2. GAP promoter</p>
    <p>4.3.3. SOD-p31 fusion protein</p>
    <p>4.3.3.1. pC1/1-pSP31-GAP-ADH2</p>
    <p>4.3.3.2. Transformation and expression</p>
    <p>4.3.3.3. Purification and characterization</p>
    <p>4.4 Reverse transcriptase (RT)</p>
    <p>4.4.1. pAB24/RT4 expression vector</p>
    <p>4.4.2. Transformation and expression</p>
    <p>4.4.3. Purification</p>
    <p>4.4.4. Electrophoresis and immunoblotting</p>
    <p>4.4.5. RT activity assay</p>
    <p>4.5. p25gag</p>
    <p>4.5.1. Host-vector system</p>
    <p>4.5.2. Saccharomyces cerevisiae AB110</p>
    <p>4.5.3. pC1/1-p25-ADH-GAP</p>
    <p>4.5.4. Transformation and expression</p>
    <p>4.5.5. Protein Purification</p>
    <p>4.6. p53gag</p>
    <p>4.6.1. Construction of pC1/1-GAP-p53</p>
    <p>4.6.2. Transformation and expression</p>
    <p>4.6.3. Protein purification</p>
    <p>5. Immunoassays for Anti-HIV Abs Using Recombinant HIV Polypeptides</p>
    <p>5.1. ELISA-A</p>
    <p>5.1.1. Assay Protocol</p>
    <p>5.1.2. Results</p>
    <p>5.2. ELISA-B</p>
    <p>5.2.1. Assay Protocol</p>
    <p>5.2.2. Results</p>
    <p>5.3. Dot Blot Assay</p>
    <p>5.4. Immunoblot Strip ELISA Assay</p>
    <p>6. Serology Studies with Recombinant HIV Polypeptides</p>
    <p>6.1. Immunoblot with Env Polypeptides</p>
    <p>6.2. ELISA with Env Polypeptides</p>
    <p>6.3. ELISA with Gag Polypeptides</p>
    <p>6.4. Western and ELISA with Pol Polypeptides</p>
    <p>7. HIV Immunization</p>
    <p>7.1. Immunization of mice</p>
    <p>7.2. Immunization of guinea pigs</p>
    <p>7. Deposits of Biological Materials</p>
    <p>1. Isolation, Cloning and Characterization of HIV</p>
    <p>The DNA and RNA sequences of HIV are provided, as well as fragments thereof, which find extensive use in the detection of the presence of HIV, for the expression of protein specific for HIV and the use of such proteins for the production of monoclonal antibodies for in vitro and in vivo use, in diagnostics, therapy, or the like. In addition, due to the observed polymorphism of HIV, probes are indicated which can be used to detect the presence of HIV or a particular polymorphism thereof. The probes are at least about 20 bases and will usually not be more than about 500 bases and may be in the gag, env or LTR region. Furthermore, a strategy is provided for analyzing the various polymorphisms, using restriction enzyme analysis, whereby different isolates can be related in accordance with different families.</p>
    <p>1.1 Purification and preparation of viral RNA</p>
    <p>HUT-78 cells infected with ARV-2 (ATCC Accession No. CRL 8597, deposited on Aug. 7, 1984) were obtained from Dr. Jay Levy, University of California, San Francisco. Cultures were grown for two weeks in RPMI medium with 10% fetal calf serum. Cells were removed by low-speed centrifugation (1,000 g for 10 min), and the resulting supernatants centrifuged at 2 Krpm for 1 h at 4 C. using a SW-28 rotor. The pellet, containing the virus, was resuspended in 10 mM Tris-HCl, pH 7.5 on ice. The resuspended pellet was treated with 10 g of DNase (Boehringer-Mannhein) and was layered onto a linear sucrose gradient (15-50% in 10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 20 mM NaCl). The gradient was spun at 34 Krpm for 4 h at 4 C., in SW-41 rotor. Five 2.5 ml fractions were collected and an aliquot of each was electrophoresed in a 1% agarose, 5 mM methyl mercury hydroxide gel  Bailey et al. (1976) Anal. Biochem. 70: 75-85! to determine which contained the 9 kb viral RNA. The fraction containing the viral RNA (identified by gel analysis) was diluted to 10 ml in 10 mM Tris-HCl, pH 7.5, 1 mM EDTA and was centrifuged at 34 Krpm for 2 h at 4 C. The pellet was resuspended in 20 mM Tris-HCl, pH 7.6, 10 mM EDTA, 0.1% SDS, and 200 g/ml proteinase K. Incubation was carried out for 15 min at room temperature. The mixture was extracted with phenol and the aqueous phase was made 400 mM NaCl and precipitated with ethanol. The pellet was resuspended in water and stored at -70 C.</p>
    <p>To purify the viral RNA from the nucleic acid pellet obtained as described above, a sample was electrophoresed in a low-melting 1% agarose gel containing 5 mM Methyl mercury hydroxide. After electrophoresis, the gel was stained with 0.1% ethidium bromide and nucleic acid bands were visualized under UV light. The region corresponding to 9 kb was cut from the gel and the agarose was melted at 70 C. for 2 to 3 min in three volumes of 0.3M NaCl, 10 mM Tris, pH 7.5, 1 mM EDTA. The mixture was extracted with an equal volume of phenol. The aqueous phase was reextracted with phenol and was precipitated with ethanol. The pellet was washed with cold 95% ethanol, air dried, resuspended in water and stored at -70 C. until use. One hundred ml of culture medium yielded 0.5 to 1 g of purified RNA.</p>
    <p>1.2. Synthesis of labeled viral probe</p>
    <p>A <sup>32</sup> P-labeled cDNA was made to the gel purified viral RNA using random primers (calf thymus primers) prepared as described in Maniatis et al., Molecular Cloning: A Laboratory Manual (Cold Spring Harbor Laboratories 1982). The reaction mixture contained 2 of 0.5M MgCl<sub>2</sub> ; 5 l of 0.1M dithiothreitol; 2.5 l each of 10 mM dATP, 10 mM dGTP and 10 mM dTTP; 2.5 l calf thymus primer (100A<sub>260</sub> /ml); 0.5 g viral RNA; 5 l of actinomycin D (200 g/ml); 10 l of <sup>32</sup> P-dCTP (&gt;3000 Ci/mmole, 1 mCi/ml) and 1 l of AMV reverse transcriptase (17 units/l) in a 50 l reaction volume. The reaction was incubated for 1 h at 37 C. The probe was purified away from free nucleotides by gel filtration using a Sephadex G50 column. The void volume was pooled, NaCl was added to a final concentration of 400 mM and carrier single-stranded DNA to 100 g/ml, and the cDNA was precipitated with ethanol. The pellet was resuspended in water and incorporated <sup>32</sup> P counts were determined.</p>
    <p>1.3. Detection of HIV RNA and DNA in mammalian cells</p>
    <p>PolyA+ RNA was prepared from HUT-78 cells infected with ARV-2, ARV-3 or ARV-4 (three different isolates from three different AIDS patients) and from uninfected HUT-78 cells. The polyA+ RNA was electrophoresed on 1% agarose gels containing 5 mM methyl mercury hydroxide (Bailey et al. supra), was transferred to nitrocellulose filters, and hybridized with the homologous probe prepared as described in Section 1.2. Hybridizations were carried out in 50% formamide, 3 SSC at 42 C. Washes were at 50 C. in 0.2 SSC. A 9 kbp band was present in all three samples of infected HUT-78 cells. This band was absent in polyA+ from uninfected cells.</p>
    <p>High molecular weight DNA (chromosomal) was prepared from cultures of HUT-78 cells infected with ARV-2 and from non-infected HUT-78 cells following the procedure of Luciw et al. (1984) Molec. &amp; Cell Biol. 4: 1260-1269. The DNA was digested with restriction enzyme(s), electrophoresed in 1% agarose gels and blotted onto nitrocellulose following the procedure described by Southern, (1975), supra. Blots were hybridized with the <sup>32</sup> P-labeled probe (10<sup>6</sup> cpm/blot) in a mixture containing 50% formamide, 3 SSC, 10 mM Hepes, pH 7.0, 100 g/ml denatured carrier-DNA, 100 g/ml yeast RNA and 1 Denhardt's for 36 h at 42 C. Filters were washed once at room temperature in 2 SSC and twice at 42 C. in 0.2 SSC, 0.1% SDS. Filters were air dried and exposed to X-Omar film using an intensifying screen.</p>
    <p>The homologous <sup>32</sup> P-probe to ARV-2 hybridized specifically to two bands in the DNA from infected cells restricted with SacI. These bands were absent when DNA of non-infected cells was used, indicating that the probe is hybridizing specifically to infected cells presumably to the provirus integrated in the chromosomal DNA. The molecular weight of the bands is approximately 5 kb and 3 kb.</p>
    <p>In order to determine if different enzymes would cut the proviral sequence, several other restriction digestions of the cell DNA were carried out using EcoRI, SphI or KpnI or double digestions using two of them. Southern results show specific bands hybridizing when DNA of infected cells is used. FIG. 1 shows a schematic map of the positions of restriction enzyme sites in the proviral sequence, and indicates fragment sites.</p>
    <p>1.4. Cloning of proviral HIV DNA</p>
    <p>High molecular weight cell. DNA from infected HUT-78 cells was prepared following the procedure of Luciw et al., supra. The DNA was digested with EcoRI, which cuts once in the provirus, centrifuged in a sucrose gradient and fractions corresponding to 8-15 kb were pooled, dialyzed and concentrated by ethanol precipitation. The bacteriophage  derivative cloning vector, EMBL-4.  Karn et al. (1983) Methods Enzymol. 101: 3-19! was digested to completion with a mixture of EcoRI, BamHI and Sail restriction enzymes and the DNA then deproteinized by phenol-chloroform extraction, precipitated with cold ethanol and resuspended in ligation buffer. The EMBL-4 phage DNA and EcoRI digest of cellular DNA were mixed and ligated and the resultant recombinant phage genomes packaged in vitro. After phage infection of -sensitive E. coli (DP50supF), about 500,000 phage plaques were transferred onto nitrocellulose filters, DNA was fixed and the filters were screened with a homologous <sup>32</sup> P-probe prepared as described in Section 1.2. Eleven recombinant phage out of 500,000 phage annealed in the initial double-lift screening method (Maniatis et al., supra) to viral cDNA probe, and these were further plaque-purified and propagated in large liquid cultures for preparation of recombinant DNA. Plaque-purified phage containing ARV DNA were propagated in liquid culture in E. coli DP50supF; phage particles were harvested and banded in CsCl gradients and recombinant phage DNA was prepared by phenol extraction followed by ethanol precipitation (Maniatis et al., supra). One microgram of purified phage DNA was digested with restriction enzymes, electrophoresed on 1% agarose gels, and visualized with ethidium bromide under ultraviolet light. The DNA from these gels was transferred to nitrocellulose and annealed with viral cDNA probe.</p>
    <p>Described below is the analysis of the 11 recombinant phage DNA molecules utilizing restriction enzymes and viral cDNA probe. Two examples were selected for detailed description: -9B contained an insertion of full-length proviral DNA along with flanking cell sequences, and -7A harbored a full-length viral genome permuted with respect to the EcoRI site. Digestion of -9B DNA with SacI yielded viral DNA fragments of 3.8 kb and 5.7 kb (FIG. 2). EcoRI digestion of -9B produced virus containing DNA species at 6.4 kb and 8.0 kb; a double digest of SacI and EcoRI gave viral DNA fragments at 3.8 kb and 5.4 kb (FIG. 2). This pattern is consistent with that of a provirus linked to cell DNA. The patterns of the digestion with KpnI and with a mixture of KpnI and EcORI support this conclusion for -9B. In particular, digestion of -9B DNA with KpnI showed a 5.2 kb species that represents the internal fragment seen in proviral DNA (FIGS. 1 and 2). Bacteriophage ARV-2(9B) was deposited at the ATCC on 25 Jan. 1985 and given Accession No. 40158. A second recombinant phage, -10C, also contained a full-length proviral DNA insertion.</p>
    <p>When the DNA of -7A phage was treated with SacI, two viral DNA fragments were observed at 3.8 kb and 6.6 kb (FIG. 3). EcoRI digestion of -7A DNA produced a 9.4 kb viral species, and a digestion with a mixture of EcoRI and SacI yielded two viral DNA fragments at 3.8 kb and 5.4 kb (FIG. 3). Thus, -7A should represent a recombinant phage clone containing a full-length linear HIV genome that is permuted with respect to the EcoRI site. Analyses with KpnI support this model. After digestion with KpnI a DNA fragment at 4.2 kb was observed as well as other DNA species. See FIG. 3. The data indicated that the 4.2 kb DNA fragment represents the circle junction and the others represent HIV DNA linked to cell DNA and bacteriophage vector DNA. The double digestion with KpnI and EcoRI left the 4.2 kb DNA intact and produced two fragments at 1.6 kb and 3.6 kb (FIG. 3); these three DNA species added up to 9.4 kb and constituted the HIV genome predicted from a permuted configuration.</p>
    <p>In addition to the two types of recombinant phage containing HIV DNA described above, phage was obtained that (1) possessed the left half of the viral genome from the EcoRI site in viral DNA extending into flanking cell DNA  AARV-2(SA)! and (2) phage that had the right half of the viral genome  XARV-2(7D)! from the EcoRI site in viral DNA extending into flanking cell DNA. The four types of recombinant phage DNA structures are predicted from the analysis of viral DNA observed in infected cells. Maps of restriction enzyme sites are shown in FIGS. 1-3. The data for these maps were compiled from studies of proviral DNA in infected cells, from characterization of recombinant phage DNA, and from preliminary DNA sequence information. Bacteriophages ARV-2(7D) (right) and ARV-2(8A) (left) were deposited at the ATCC on Oct. 26, 1984 and given Accession Nos. 40143 and 40144, respectively.</p>
    <p>To validate the cloned HIV DNA, a radioactive probe was prepared from two regions of the cloned HIV genome that represent about 70% of the genome and this probe was used to detect HIV DNA in restriction enzyme digests of DNA from infected cells. Whole-cell DNA was prepared from HUT-78 cell cultures infected with HIV-1 strains ARV-2, ARV-3 and ARV-4, and analyzed by digestion with restriction enzymes as described in Section 1.3. A probe to cloned ARV-2 DNA was prepared as follows: DNA from recombinant phage -7A (FIG. 3) was digested with SacI or with a mixture of SacI and KpnI and digestion products were electrophoresed in 1% agarose gels (low-melting agarose); the 3.8 kb DNA fragment from the digestion with SacI and the 3.1 kb DNA fragment from the digestion with SacI and KpnI (FIG. 3) were eluted, pooled, denatured by boiling in water for 2 min, and used as templates with random DNA primers (calf thymus) with reverse transcriptase.</p>
    <p>DNA from HUT-78 cells infected with ARV-2, ARV-3 or ARV-4 were digested with SacI, PstI or HindIII. Digested DNA was electrophoresed on 1% agarose gels, blotted onto nitrocellulose filters and annealed with probe to cloned ARV-2 DNA or to homologous cDNA probe prepared as described in Section 1.2. Results show that SacI, PstI, and, HindIII fragments detected by both probes are identical in all isolates.</p>
    <p>1.5. Polymorphism of HIV</p>
    <p>To measure the relatedness of independent HIV isolates, restriction enzyme digests Of DNA from HUT-78 cells infected with ARV-3 and ARV-4 were analyzed with the probe made from cloned ARV-2 DNA. The SacI digest of ARV-3 DNA was similar to that of ARV-2 whereas the HindIII digests displayed different patterns. The SacI digest and the PstI digest of ARV-4 DNA differed from the corresponding digests of ARV-2 DNA. The intensity of the annealing signals obtained with ARV-3 and ARV-4 samples was much lower (about 10-fold less) than that for ARV-2 DNA, probably as a result of the fact that fewer cells were infected in the ARV-3 and ARV-4 cultures. The viral-specific DNA fragments produced by SacI treatment of ARV-3 and ARV-4 DNA totaled 9.0-9.5 kbp, a value similar to that of ARV-2. and in consonance with the RNA genome sizes.</p>
    <p>1.6. Sequencing of proviral DNA</p>
    <p>Fragments or subfragments of ARV-2 DNA were prepared from the recombinant phages and cloned into M13 according to conventional procedures (Maniatis et al., supra). Sequencing was performed according to Sanger et al.  (1977) Proc. Natl. Acad. Sci. USA 74: 5463!, using the universal M13 primer or chemically synthesized primers complementary to ARV-2 sequence. The sequence is shown in FIG. 4. Also indicated in this figure are the restriction sites present in the DNA and the open reading frames encoded by the sequence.</p>
    <p>The sequence of the HIV-1 DNA from  phage 9B was sequenced in a similar manner. This sequence is shown in FIG. 5. There are several differences between the sequences shown in FIGS. 4 and 5 which reflect the polymorphism in HIV and the fact that the sequence of FIG. 4 was derived as a composite from sequence data on several HIV isolates, whereas the sequence of FIG. 5 is from a single isolate. The main difference affecting polypeptide sequence is that the small open reading frame between the gag and pol genes in FIG. 4 is not independent, but is part of the pol gene in FIG. 5. This merger of these reading frames was the result of three base changes. The region of fusion and sequence change occurs roughly between nucleotides 2853 and 2941 in FIG. 5.</p>
    <p>Furthermore, open reading frames in FIG. 4 were translated into amino acids beginning with the first methionine in the open reading frame, whereas in FIG. 5 translation into amino acids was begun immediately with the codon following the stop codon.</p>
    <p>1.7. Amino acid sequences of native HIV proteins</p>
    <p>ARV-2 was prepared and purified as described in Section 1.1. The viral proteins were electrophoresed on an acrylamide gel, and the band corresponding to a 24,000 dalton or 16,000 dalton protein was excised from the gel and used for sequencing. Micro-sequence analysis was performed using Applied Biosystems model 470A protein sequencer similar to that described by Hewick et al. (1981) J. Biol. Chem. 256: 7990-7997. Phenylthiohydantoin amino acids were identified by HPLC using a Beckman Ultrasphere ODS column and a trifluoroacetic acid-acetonitrile buffer system as reported by Hawke et al. (1982) Anal. Biochem. 120: 308-311. Table 1 shows the first 20 amino acids from the amino terminus determined for p25-gag protein and Table 2 shows the first 30 amino acids for p16-gag protein.</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">              TABLE 1______________________________________Amino-terminal sequence of p25gag  Position        Amino acid______________________________________   1    Pro   2    Ile   3    Val   4    Gln   5    Asn   6    Leu   7    Gln   8    Gly   9    Gln  10    Met  11    Val  12    (His)  13    Gln  14    Ala  15    Ile  16    (Ser)  17    Pro  18    (Arg, Lys)  19    Thr  20    (Leu)______________________________________</pre>
    
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">              TABLE 2______________________________________Amino-terminal sequence of p16gag  Position        Amino acid______________________________________   1    (Met)   2    Gln   3    Arg   4    Gly   5    Asn   6    Phe   7    Arg   8    Asn   9    Gln  10    Arg  11    Lys  12    Thr  13    Val  14    Lys  15    --(Cys)  16    Phe  17    Asn  18    --(Cys)  19    Gly  20    Lys  21    Glu  22    Gly  23    (His)  24    Ile  25    Ala  26    (Lys)  27    Asn  28    (Gly)  29    (Arg)  30    (Ala, Leu)______________________________________</pre>
    
    <p>The amino acid sequence of Table 1 is predicted by nucleotides 1195 to 1255, and from Table 2 by nucleotides 1930 to 2120, from the ARV-2 DNA sequence (indicated by bars in FIG. 4). Therefore, these results confirm that the indicated gag open reading frame is in fact being translated and identifies the N-termini of p25 and p16.</p>
    <p>2. Expression of HIV Polypeptides in Mammalian Cells</p>
    <p>2.1. Expression of env peptide</p>
    <p>2.1.1. Mammalian expression vector pSV-7c</p>
    <p>Plasmid pSV-7c contains the SV40 early promoter, origin of replication and polyA processing signal, and was constructed as described below.</p>
    <p>A 400 bp BamHI-HindIII fragment containing the SV40 origin of replication and early promoter was obtained by digestion of plasmid pSVgtl (P. Berg, Stanford University, Palo Alto, Calif.) and purification through gel electrophoresis. A second 240 bp SV40 fragment containing SV40 polyA addition sites was obtained by digestion of pSV2/dhfr  Subramani et al. (1981) J. Mol. Cell. Biol. 1: 854-864! with BclI and BamHI, and gel purification. Both fragments were fused together through a linker which provided for a HindIII overhang in the 5'-end, a BclI overhang in the 3'-end, three general restriction sites and three successive stop codons in all three reading frames. The sequence of the polylinker was the following: ##STR1## The about 670 bp fragment, containing SV40 origin of replication, SV40 early promoter, polylinker with stop codons and SV40 polyadenylation site, was cloned into the BamHI site of pML  Lusky &amp; Botchan, (1984) Cell 36: 391!, a pBR322 derivative with an about 2.5 kbp deletion, to yield pSV-6. To eliminate the EcoRI and EcoRV sites present in pML sequences of pSV-6, this plasmid was digested with EcoRI and EcoRV, treated with Ba131 nuclease to remove about 200 bp per end, religated and cloned to yield pSV-7a. The Ba131 resection also eliminated one BamHI restriction site flanking the SV40 region approximately 200 bp away from the EcoRV site. To further eliminate the other BamHI site flanking the SV40 region, pSV-7a was digested with NruI, which cuts in the SV40 sequence, and PvuII, which cuts in the pML sequence upstream from the origin of replication. The plasmid was recircularized by blunt end ligation and cloned to yield pSV-7b. To increase the number of cloning sites of pSV-7b, a new polylinker was cloned into the plasmid to replace the previous one but the stop codons in the three frames were still retained. For this purpose, pSV-7b was digested with StuI and XbaI, and a linker of the following sequence was lighted to the vector to yield pSV-7c. ##STR2## 2.1.2. Transformation of cells with env sequences</p>
    <p>The HIV DNA coding for the env region was excised from a recombinant phage ARV-2(7D)  7D! which contains the right half of the provirus by digestion with KpnI and EcoRI and gel purification of about 3300 bp fragment. This fragment was cloned into plasmid pSV-7c as shown in FIG. 6. The recombinant vector (pSV7c/env) was used to transfect Cos cells  Mellon et al. (1981) Cell 27: 279! following the procedure of van der Eb &amp; Graham  (1980) Methods in Enzymology 65: 826-839! as modified by Parker and Stark  (1979) J. Virol. 31: 360369!, in 60 mm plastic dishes (510<sup>5</sup> cells per dish). Thirty-six hours later the cells were transferred to glass microscope slides and cultured for another 24 h (510<sup>4</sup> cells per chamber in a 4-chamber Titer-tek slide). The cell sheet was rinsed in PBS and fixed by dipping in cold acetone for 5 sec.</p>
    <p>2.1.3. Detecting expression via immunoflorescence</p>
    <p>The fixed-cell monolayers in each chamber were incubated with AIDS patient sera (Reference serum EW511 obtained from Dr. Paul Feorino, Center for Disease Control) and with control normal human sera. Both sera were diluted 1/100 in PBS and 50 microliters were applied per chamber, incubated for 1 h at 37 C., and rinsed by dipping in PBS. Fluoresceinated goat anti-human antisera (Cappel Labs) was diluted 1/100 in PBS and 50 microliters were applied to each chamber and incubated for 30 min at 37 C. After washing in PBS, 75% glycerol in PBS was added to the cell sheets, a coverslip was placed on top, and the cells were observed in a fluorescence microscope. Abut five cells in 100 showed bright immunofluorescence with the AIDS sera. No cells stained with control normal human sera. Cells transfected with a control plasmid containing only SV40 vector sequences did not show any immunofluorescence with any antisera. Thus, the EcoRI-KpnI ARV-2 DNA fragment encodes a gene that can be expressed in mammalian cells and the expression product detected specifically by AIDS patient sera.</p>
    <p>2.2. Expression of gp160env</p>
    <p>2.2.1. Mammalian expression vector pSV7d</p>
    <p>pSV7c and pSV7d represent successive polylinker replacements. The polylinker in pSV7c was digested with BgIII and XbaI, and then ligated with the following linker to yield pSV7d: ##STR3## 2.2.2. Expression of tPA/gp160</p>
    <p>In order to achieve optimal secretion of gp160 from mammalian tissue culture cells, the 5' end of the coding sequence was modified to accept a heterologous signal sequence known to direct efficient secretion of both the homologous gene (human tissue plasminogen activator) and deletion variants of this gene. van Zonnefeld et al. (1986) Proc. Natl. Acad. Sci. USA 83: 4670.</p>
    <p>A portion of the HIV env gene was excised with SacI and StuI (positions 5555 and 6395) and was subcloned into the vector M13mp19  Yanisch-Perron et al. (1985) Gene 33: 103-109! between SacI and SmaI. Oligonucleotide-directed mutagenesis  Zoller et al. (1983) Methods in Enzymology 100: 468-500! was used to create an NheI site at the junction of the natural signal peptide and the mature envelope polypeptide using the following oligonucleotide:</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="equation">5'-GATGATCTGTTCAGCTAGCGAAAAATTGTGG-3'</pre>
    
    <p>This mutagenesis changes cytosine-5867 to guanine and adenine-5868 to cytosine, thereby creating an NheI site and altering the codon for threonine-30 to code for serine.</p>
    <p>In parallel, the tPA gene was likewise mutagenized in M13 to place an NheI site near the carboxyl end of the tPA signal peptide. The following sequences show the 5' UT sequences and signal for wild type tPA leader and for the NheI variant.</p>
    <p>Wild-type sequence of the tPA signal: ##STR4##</p>
    <p>The NheI variant of the tPA signal: ##STR5##</p>
    <p>Following mutagenesis and sequence verification, a 174 bp fragment containing 99 bp of 5' untranslated sequence and the signal sequence from tPA was excised from the tPA-containing M13 clone using SalI and NheI and fused to the 559 bp fragment containing the 5' end of the any gene which was excised from the env-containing M13 clone with NheI and HindIII (contributed by the M13 polylinker) and these fragments were subcloned into M13mp18 between SalI and HindIII. This plasmid is called M13tpaS.NheIenv.</p>
    <p>The DNA and amino acid sequences of the tPA signal fused to the 5' end of gp160 is: ##STR6##</p>
    <p>From this M13 subclone, M13tpaS.NheIenv, the DNA fragment containing the heterologous signal and 5' untranslated sequences fused to the 5' end of the env gene was excised using the XbaI site from the M13 polylinker and the unique NdeI site found at position 5954 in the env gene. This 268 bp fragment was ligated with a 2506 bp NdeI-XhoI fragment, encoding the remainder of the any gene, from NdeI at position 5954 through the stop codon at position 8344, and 116 additional bases to the XhoI site at position 8460 and at the same time with vector pSV7d cleaved with XbaI and SalI in order to generate a fully functional gp160 gene in the proper orientation for expression from the SV40 early promoter. This expression plasmid, called pSV7dARV160tpa, was used to transfect COS 7 cells as described by Sompayrac et al. (1981) Proc. Natl. Acad. Sci. USA 78: 7575. Transfected cells were assayed for gp160 expression by immunofluorescence as described in Section 2.1.3 above. Approximately 5% of the cells showed bright fluorescence when exposed to the HIV infected human sera, and no cells stained with normal human serum. Positive sera employed were obtained from the Interstate Blood Bank.</p>
    <p>Permanent cell lines were obtained by transfection of CHO cells lacking dihydrofolate reductase (dhfr). CHO dhfr<sup>-</sup> cells  Urlaub et al. (1980) Proc. Natl. Acad. Sci. USA 77: 4216! were plated at a density of 510<sup>5</sup> to 10<sup>6</sup> cells per 10 cm dish the day prior to transfection in nutrient medium (F12 supplemented with 1.18 mg/ml Na<sub>2</sub> COhd 3, 292 ug/ml glutamine, 110 ug/ml sodium pyruvate, 100 U/ml penicillin, 100 U/ml streptomycin, 150 ug/ml proline and 10% fetal calf serum). Cells were transfected using the calcium phosphate coprecipitation method described by Graham et al. (1973) Virology 52: 456-467 with the gp160 expression plasmid, pSV7dARV160tpa; a plasmid bearing as a selectable marker, a dhfr gene driven by the adenovirus major late promoter, pAD-dhfr; and a plasmid bearing the human tissue plasminogen activator gene driven by the SV40 early promoter, pSV7tpa3, to serve as an easily screened gene marker. Plasmid pSV7tpa3 is a mammalian cell expression vector containing the full length tpa cDNA (with 99 bp 5' untranslated sequences) cloned into the SalI site of pSV7d, with the orientation such that the proper reading frame is transcribed from the SV40 early promoter.</p>
    <p>Samples were added to the dishes of cells fed with fresh medium and allowed to settle for 6 h in an incubator (5% CO<sub>2</sub>, 95% air) at 37 C. Six hours later, the supernatants were aspirated, the cells rinsed gently with calcium and magnesium-free phosphate buffered saline (PBS-CMF), and the dishes exposed to 15% glycerol in HEPES-buffered saline as an adjuvant for 3.5 to 4 min. After rinsing, the medium on the cells was replaced with fresh F12 as described. Forty-eight hours after the addition of DNA to the cells, the cells were trypsinized and split 1:20 into selective medium, Dulbecco's Modified Eagle Medium (DMEM) supplemented with 4.5 mg/ml glucose, 3.7 mg/ml Na<sub>2</sub> CO<sub>3</sub>, 292 ug/ml glutamine, 110 ug/ml sodium pyruvate, 100 U/ml penicillin, 100 U/ml streptomycin, 150 ug/ml proline, and 10% dialyzed fetal calf serum. After growth in selective medium for 1-2 weeks, colonies appeared. These colonies were assayed for production of tPA by the casein-plasminogen-agar overlay assay. Granelli-Piperno et al. (1978) J. Exp. Med. 148: 223-234. Individual clones were transferred to microwell plates and expanded to T75 flasks. Twenty-four cell lines that were the highest tPA producers were screened by immunofluorescence for gp160 expression as described for COS cells above and in Section 2.1.3. Of these, seven lines fluoresced brightly when reacted with HIV-positive serum. These seven gp160-positive lines were pooled by mixing equal numbers of each cell line and plating in 10 cm dishes at a cell density of 110<sup>5</sup> cells per dish. Replicate dishes were fed with DMEM supplemented as described above with the addition of varying amounts of methotrexate (amethopterin, Sigma): 0.01, 0.02, 0.05, 0.1 and 0.2 micromolar. After growth in the amplification medium containing methotrexate for 2 to 3 weeks, colonies appeared on the 0.1 and 0.2 micromolar methotrexate dishes. These colonies were transferred to microwell plates and expanded to T75 flasks.</p>
    <p>Clones were screened and scored for increased tPA production. Lysates of the cells were prepared in order to examine any gp160 expressed and found in the cell membranes, as is found in HIV infected human lymphocytic lines. Lysates were prepared from cell monolayers by rinsing in cold PBS-CMF and adding lysis buffer consisting of 100 mM NaCl, 20 mM Tris pH 7.5, 1 mM EDTA, 0.5% NP40, 0.5% sodium deoxycholate, 1% BSA, 1 ug/ml pepstatin, 1 mM PMSF, and 17 ug/ml aprotinin. Lysates were collected, incubated on ice for 10 min with occasional vortexing, spun 5 min at 14000 g in a microcentrifuge, and supernatants transferred to new tubes for storage at -70 C. These samples were assayed by an ELISA specific for envH derived from yeast (see Sections 4.1.2 and 4.1.5).</p>
    <p>Immunoblot analysis was done by standard Western transfer of proteins from 8% denaturing gels  Laemmli (1970) Nature 227: 680-685! and analysis with human immune sera or mouse monoclonal antibodies raised against yeast-expressed envH and env-2 (see Sections 4.1.2 and 4.1.5). The blocking solution is 0.3% Tween-20, 2% normal goat serum in phosphate buffered saline, and all incubations and washes were done in this solution. The human imune sera (Interstate Blood Bank) were diluted 1:100, and Tago goat anti-human horse radish peroxidase (HRP) conjugate was diluted 1:250. Development was with 4-chloro-1-napthol (BioRad). Mouse monoclonal antibody 98B3  Stephans et al., in Viruses and Human Cancer, pp. 29-41 (R. C. Gallo et al., eds., 1987)! was used undiluted as ascites fluid or diluted to 50 ug/ml from a stock of ammonium sulfate purified 1 g. Goat anti-mouse HRP conjugate (Tago) was diluted 1:200.</p>
    <p>ELISA analysis was done using the cellular lysates. Plates were coated with mouse monoclonal IgM antibody 95C9  Stephans et al., supra! at a concentration of 250 ng per 100 microliters per well in PBS. After 2 h at 37 C. or overnight at 4 C., the plates were washed 6 times in wash buffer (0.137M NaCl, 0.05% Triton X-100) and the samples added to the wells in casein diluent (100 mM sodium phosphate, 0.1% casein, 1 mM EDTA, 1% Triton X-100, 0.5M NaCl, 0.01% thimerosol pH7.5), and plates were incubated overnight at 4 C. and 4 h at 37. Samples were aspirated, then plates were washed 6 times as above. Rabbit-anti-env-2 polyclonal serum  Barr et al. (1987) Vaccine 5: 90-101! is diluted 1:100 in the diluent above and added to the plates (100 microliters per well). Plates were incubated for 1 h at 37 C., and then aspirated and washed as described above. Goat anti-rabbit HRP conjugate (Tago) was diluted 1:1000 in the diluent buffer, and 100 microliters were added to each well. Plates were incubated 1 h at 37. Samples were aspirated and the plates were washed as described above. Plates were developed with ABTS color reagent and H<sub>2</sub> O<sub>2</sub> for 30 min at 37. ABTS was dissolved in citrate buffer and 100 microliters per well were added to the plate. The reaction was stopped by adding 50 ul per well 10% SDS, and the plates were read at 415 nm, 600 nm reference beam. Env-2 was used as a standard using serial 2-fold dilutions starting with 100 ng/ml to 11 places.</p>
    <p>Six CHO clones positive by the ELISA assay were amplified in a pool in methotrexate as described above, except that after isolation of colonies from the first round of amplification, cells were again pooled and set up for a consecutive round of amplification. For the first round, methotrexate levels were 0.25, 0.5, 0.75, 1.0, 1.25, and 1.75 micromolar. Colonies from the 0.5 and 0.75 micromolar dishes were pooled and set up in 0.75, 1.0, 2.0, 3.0, 4.0, and 5 micromolar. Clones were isolated from 3, 4, and 5 micromolar levels, expanded and assayed for gp160 production by the ELISA. Increases in expression were observed for these clones as compared to the clones grown in 0.1 and 0.2 micromolar methotrexate, and this expression level represents an improvement over the levels of gp160 in HIV-infected HUT cells.</p>
    <p>2.3 Expression of gp120env</p>
    <p>2.3.1. Expression of engineered gp120 in pS7Vd</p>
    <p>In order to express a secreted form of the envelope polypeptide in mammalian cells, the env gene was modified by in vitro mutagenesis to eliminate any potential transmembrane domains and to provide a stop codon following the processing site between the gp120 and gp41 domains of the gp160 protein. This mutagenesis was accomplished by subcloning the fragment which encodes the env gene from clone pSV7c/env (Section 2.1.2.) by excising with HindIII and XhoI (positions 5582 to 8460) and inserting the 2.8 kb fragment into M13 mp19 previously digested with HindIII and SalI. A 37 bp oligonucleotide of the following sequence was used to alter the sequence at the gp120/gp41 processing site at position 7306 to encode 2 stop codons and two restriction endonuclease sites.</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="equation">5'-GAACATAGCTGTCGACAAGCTTCATCATCTTTTTTCT-3'</pre>
    
    <p>The sequence of the wild-type gene and the mutant are shown below: ##STR7##</p>
    <p>Following mutagenesis, the gene was engineered for optimal secretion into the medium utilizing the same procedure that was used for gp160. The 268 bp XbaI-NdeI fragment containing the heterologous 5' untranslated sequences and signal sequences from human tPA fused to the 5' end of env was excised from the M13 clone M13tpaS.NheIenv described in Section 2.2.2. This fragment was ligated with a 1363 bp NdeI-SalI fragment encoding the remainder of the gp120 coding region which was isolated from the gp120 mutant (positions 5954 and 7317) described above, and both fragments were inserted into the vector pSV7d previously digested with XbaI and SalI.</p>
    <p>This plasmid, pSV7dARV120tpa, has a gene encoding gp120 that ends precisely at the processing site between gp120 and gp41, Arginine-509, expressed from the SV40 early promoter. Plasmid pSV7dARV120 was used to transient COS7 cells as described in Section 2.2.2. Transfected cells were assayed for expression and secretion of gp120 into the culture medium by immunoblot analysis of conditioned cell medium from the transfected cells. Immunoblot analysis was done using both HIV-positive human serum and mouse monoclonal antibodies described in Section 2.2.2. Using both antibody sources, the gp120 transfected cells secreted into the medium a polypeptide of the size of gp120 that reacted with these immunological reagents, while mock transfected or pSV7dARV160tpa transfected COS cell supernatants showed no such polypeptide. These results are consistent with the production of authentic gp120 in COS cells.</p>
    <p>2.3.2. Expression of gp120env using CMV IE-1 promoter</p>
    <p>In an effort to improve the expression of gp120 in COS and other mammalian cell types, the gp120 coding region was excised from pSV7dARV120tpa using SalI and inserted into the unique SalI site of the CMV IE-1 expression vector pCMV6a (described below) and screened for insertion in the proper orientation for expression from the CMV IE-1 promoter. The resulting plasmid, pCMV6ARV120tpa, was used to transfect COS 7 cells as described in Sections 2.2.2. and 2.3.1. The conditioned cell medium from the transfected cells was assayed by immunoblot and by ELISA, as described in Sections 2.2.2. and 2.3.1. The CMV6ARV120tpa-transfected cells expressed gp120 at levels of 60-250 ng/ml and the pSV7ARV120tpa-transfected cells expressed less than 0.5 ng/ml. These results indicate a significant increase in expression of the gp120 polypeptide using the CMV expression plasmid relative to the SV40 expression plasmid described in Section 2.3.1, 50 to 100-fold in magnitude as measured by the ELISA.</p>
    <p>Plasmid pCMV6a is a mammalian cell expression vector which contains the transcriptional regulatory region from human cytomegalovirus immediate early region, HCMV IE1. The plasmid contains the SV40 polyadenylation region derived from pSV7d (Section 2.2.1.) as a 700 bp. PvuI-SalI fragment; the SV40 origin of replication derived for pSVT2  Meyers et al. (1981) Cell 25: 373-384; Rio et al. (1983) Cell 32: 1227-1240! as a 1.4 kbPvuI-EcoRI (filled in with Klenow fragment); and the HCMV IE1 promoter as a 1.7 kbp SspI-SalI fragment derived from a subclone of the human cytomegalovirus (Towne strain). The HCMV IE1 promoter region contains the region encoding the first exon (5' untranslated), the first intron and the start of the second exon (where the SalI site was created by in vitro mutagenesis). The map of pCMV6a is shown in FIG. 29.</p>
    <p>2.4. Expression of gag peptides</p>
    <p>The ARV-2 DNA coding for the gag region was excised from a recombinant phage (7A; FIG. 3) by digestion with KpnI and SacI and gel purification of a fragment of about 3200 bp. This gag containing fragment was cloned into KpnI and SacI digested pSV-7c/envES (described below) to yield pSV-7c/gag. The recombinant vector was used to transfect Cos cells. Expression of gag protein was detected as previously described for the env protein (Section 2.1). About 5 percent of the cells transfected with SV40 vector containing the ARV-2 gag fragment showed bright immunofluorescence when incubated with the AIDS sera and a second fluoresceinated goat anti-human antisera. Cells transfected with a control plasmid containing only SV40 vector sequences did not show any immunofluorescence with any antisera.</p>
    <p>Plasmid pSV-7c/envAES was constructed as follows: Plasmid pSV-7c/env (described in section 2.1.2) was digested with EcoRI and SacI. A linker of the following sequence, which regenerates both the EcoRI and SacI sites and provides for a BamHI site between them was ligated to the linearized vector. ##STR8## This vector was digested with KpnI and SacI, the fragment corresponding to linearized plasmid was purified by gel electrophoresis and ligated to the 3200 bp KpnI-SacI fragment containing the gag region.</p>
    <p>2.5 Expression of gag-env fusion protein</p>
    <p>A mammalian cell expression vector containing</p>
    <p>a fused sequence of nt 225 to nt 1650 encoding for a gag region and nt 5957 to nt 8582 for an env region (FIG. 5) was constructed as follows. Plasmid pSV-7c/gag (described in Section 2.4) was digested With BglII. The overhangs were filled in with reverse transcriptase, and the resulting two fragments were subsequently digested with SacI. The 1390 SacI-BglII bp fragment coding for gag was purified by gel electrophoresis and cloned into pSV-7c/env (Section 2.1.2) prepared as follows.</p>
    <p>Vector pSV-7c/env was digested with NdeI which cuts 59 codons downstream from the N-terminal ATG of env. Ends were filled in with reverse transcriptase and the linearized vector was digested with SacI. The SacI-NdeI vector fragment was purified by gel electrophoresis and ligated to the gag fragment obtained as indicated above. The ligation mixture was used to transform HB101, and plasmid pSV-7c/gagenv was thus obtained.</p>
    <p>Plasmid pSV-7c/gagenv was used to transform Cos cells using the procedure described in Section 2.1.2 above. Expression was detected by immunofluorescence using an AIDS patient serum and following the procedure described previously (Section 2.1.3). About 5 cells in 100 showed bright immunofluorescence with the immune serum. No cells stained with control normal human serum.</p>
    <p>3. Expression of HIV Polypeptides in Bacteria</p>
    <p>3.1. Expression of p25gag</p>
    <p>3.1.1. Host-vector system</p>
    <p>The p25gag protein is synthesized by E. coli strain D1210 transformed with plasmid pGAG25-10. Plasmid pGAG25-10 is a pBR322 derivative which contains the sequence coding for p25gag under transcriptional control of a hybrid tac promoter  De Boer et al. (1983) Proc. Natl. Acad. Sci. USA 80: 21-25! derived from sequences of the trp and the lac UV5 promoters. Expression of p25gag is induced in bacterial transformants with isopropylthiogalactoside (IPTG).</p>
    <p>E. coli D1210, a lac-repressor overproducing strain, carries the lacIq and. lacY<sup>+</sup> alleles on the chromosome but otherwise is identical to E. coli HB101 (F<sup>-</sup>, lacI<sup>+</sup>, lacO<sup>+</sup>, lacZ<sup>+</sup>, lacY<sup>-</sup>, gal<sup>-</sup>, pro<sup>-</sup>, leu<sup>-</sup>, thi<sup>-</sup>, end<sup>-</sup>, hsm<sup>-</sup>, hsr<sup>-</sup>, recA<sup>-</sup>, rpsL<sup>-</sup>) from which it was derived.</p>
    <p>3.1.2. Construction of pGAG25-10.</p>
    <p>PlasmidpGAG25-10 was constructed by cloning a 699 bp DNA fragment coding for p25gag into plasmid ptac5, according to the scheme shown in FIG. 7. The vector ptac5 is a pBR322 derivative which contains the tac promoter, Shine Delgarno sequences, and a polylinker as a substitution of the original pBR322 sequences comprised between the EcoRI and PvuII restriction sites.</p>
    <p>The 699 bp DNA fragment codes for the complete p25gag protein (amino acid residues 136 to 365 as numbered in FIG. 5), the only difference being that a methionine was added as the first amino acid in pGAG25-10 to allow for translational initiation. This change, as well as other changes in nucleotide sequence as indicated below, were achieved by using chemical synthesis of parts of the DNA fragment. The DNA fragment also includes two stop codons at the 3' end of the sequence.</p>
    <p>FIG. 8 shows the nucleotide sequence cloned in pGAG25-10 and the amino acid sequence derived from it. DNA sequences that are not underlined in the figure were derived directly from the AARV-2(9B) DNA. All other sequences were chemically synthesized or derived from vector ptac5. Changes were introduced in this DNA sequence, with respect to the original DNA, to create or delete restriction sites, to add a methionine prior to the isoleucine (second residue of p25) or to include stop codons after the last codon of p25gag. However, as previously indicated, all changes in the DNA sequence, except those in the first codon, do not alter the amino acid sequence of p25gag.</p>
    <p>3.1.3. Transformation and expression of gag peptide</p>
    <p>E. coli D1210 cells are made competent for transformation following a standard protocol  Cohen et al. (1982) Proc. Natl. Acad. Sci. USA 69: 2110!. Transformation is performed as indicated in the protocol with 25-50 ng of pGAG25-10. The transformation mix is plated on agar plates made in L-Broth containing 100 g/ml ampicillin. Plates are incubated for 12 h at 37 C.</p>
    <p>Single ampicillin resistant colonies are transferred into 1 ml L-Broth containing 100 g/ml ampicillin and grown at 37 C. Expression of p25gag protein is induced by adding 10 l of 100 mM IPTG (Sigma) to a final concentration of 1 mM followed by incubation at 37 C. for 2 h.</p>
    <p>Cells from 1 ml of induced cultures are pelleted and resuspended in 100 l Laemmli sample buffer. After 3 cycles of boiling and freezing, portions of resultant lysates are analyzed on standard denaturing acrylamide gels. Proteins are visualized by staining with Coomassie blue.</p>
    <p>The extent of expression is initially determined by appearance of new protein bands for induced candidate samples compared with control. Proteins of molecular weights expected for the genes expressed comprised 2%-5% of total cell protein in the highest expressing recombinants as determined by visual inspection with reference to a standard protein of known amount.</p>
    <p>Authenticity of the expressed proteins is determined by standard Western transfer of proteins to nitrocellulose and analysis with appropriate human or rabbit immune sera or mouse monoclonal antibodies or by comparing ELISA assays of purified recombinant and natural proteins using human immune sera from AIDS patients (see Section 3.1.6).</p>
    <p>3.1.4. Fermentation process</p>
    <p>3.1.4.1. Preparation of master seed stack</p>
    <p>Transformant cells from a culture expressing high levels (3%) of p25gag are streaked onto an L-Broth plate containing 100 g/ml ampicillin and the plate is incubated overnight at 37 C. A single colony is inoculated into 10 ml of L-Broth, 100 g/ml ampicillin and grown overnight at 37 C. An aliguot is used to verify plasmid structure by restriction mapping with SalI and PstI. A second aliguot is used to induce expression of p25gag and the rest of the culture is made 15% glycerol by adding 1/4 volume of 75% sterile glycerol. Glycerol cell stocks are aliquoted in 1 ml and quickly frozen in liquid nitrogen or dry-ice ethanol bath. These master seed stocks are stored at -70 C.</p>
    <p>3.1.4.2. Fermenter inoculum</p>
    <p>The master seed stock is scraped with a sterile applicator which is used to streak an L-Broth plate containing 100 g/ml ampicillin. Single colonies from this plate are used to inoculate 20-50 ml of L-Broth/amp, which is incubated at 37 C. overnight.</p>
    <p>An aliquot of the overnight culture is used to inoculate larger volumes (1-6 liters) of L-Broth/amp. Cells are incubated at 37 C. overnight and reach an O.D.<sub>650</sub> of approximately 5 prior to use as inoculum for the fermenter run.</p>
    <p>3.1.4.3. Fermentation and harvest</p>
    <p>100 showed bright immunofluorescence with the immune</p>
    <p>Fermenters (capacity: 16 liters) containing 10 l of L-Broth and 1 ml of antifoam are inoculated with 100-500 ml from the inoculum culture. Cells are grown at 37 C. to an O.D. of about 1. Expression of p25gag is induced by addition of 100 ml of an IPTG Solution (100 mM) to yield a 1 mM final concentration in the fermenter. Cells are grown for 3 additional hours and subsequently harvested using continuous flow centrifugation. At this step cells may be frozen and kept at -20 C. until purification of p25gag proceeds. Alternatively, 250 l fermenters are inoculated with 1-5 l from the inoculum culture. Growth, induction, and harvest are as indicated before.</p>
    <p>3.1.5. Purification of p25gag</p>
    <p>Frozen E. coli cells are thawed and suspended in 2.5 volumes of lysis buffer (0.1M sodium phosphate (NaPi), pH 7.5, 1 mM EDTA, 0.1M NaCl). Cells are broken in a non-continuous system using a 300 ml glass unit of a Dyno-mill at 3000 rpm and 140 ml of acid-washed glass beads for 15 min. The jacketed chamber is kept cool by a -20 C. ethylene glycol solution. Broken cells are centrifuged at 27,000 g for 25 minutes to remove debris and glass beads. The supernatant is recovered and kept at 4 C.</p>
    <p>The cell extract is made 30% (NH<sub>4</sub>)<sub>2</sub> SO<sub>4</sub> by slowly adding the ammonium sulfate at 4 C. The extract is stirred for 10 min after the final concentration is achieved, followed by centrifugation at 27,000 g for 20 min. The pellet is resuspended in 1M NaCl, 1 mM EDTA, 1% Triton X-100, and 5% SDS, and then boiled for 5 min.</p>
    <p>The fraction obtained by selective precipitation is submitted to gel filtration using a G50 Sephadex column equilibrated in 0.03M NaPi, pH 6.8. Chromatography is developed in the same solution. Fractions are collected and absorbance at 280 nm is determined. Protein-containing fractions are pooled and characterized by protein gel electrophoresis, Western analysis, and ELISA.</p>
    <p>3.1.6. Characterization of recombinant p25gag</p>
    <p>3.1.6.1. Protein gel electrophoresis</p>
    <p>SDS-polyacrylamide gel analysis (10%-20% gradient gels) of proteins from pGAG25-containing cells and control cells indicated that varying levels of a protein of a molecular weight of about 25,000 were specifically induced in cells containing p25gag expression plasmids after derepression of the tacI promoter with IPTG. Identity of the p25gag gene product was confirmed by both an enzyme-linked immunosorbent assay (ELISA; see Section 3.1.6.3) and Western immunoblot analysis (see Section 3.1.6.2) using both AIDS patient serum and a monoclonal antibody to viral p25gag.</p>
    <p>3.1.6.2. Western analysis</p>
    <p>Samples were electrophoresed under denaturing conditions on a 10%-20% polyacrylamide gradient gel. Samples were electroblotted onto nitrocellulose. The nitrocellulose paper was washed with a 1:250 dilution of AIDS patient reference serum (EW5111, obtained from P. Feorino; Centers for Disease Control, Atlanta, Ga.) and then with a 1:500 dilution of PIRP-conjugated goat antiserum to human immunoglobulin (Cappel, No. 3201-0081). Alternatively, the nitrocellulose was washed with undiluted culture supernatant from 76C, a murine monoclonal antibody to HIV-1 p25gag, and then with a 1:500 dilution of HRP-conjugated goat antiserum to mouse immunoglobulin (TAGO, No. 6450). The substrate for immunoblots was HRP color development reagent containing 4-chloro-1-naphthol.</p>
    <p>The p25gag protein reacted with both AIDS patient reference serum and with the monoclonal antibody, while it shows no reactivity with the non-immune serum.</p>
    <p>3.1.6.3. ELISA comparison of recombinant and native p25gag</p>
    <p>The reactivity of sera with the purified p25gag (recombinant and native) was assayed by coating wells of microtiter plates with 0.25 g/ml of purified antigen, adding dilutions of test sera, followed by a 1:1000 dilution of HRP-conjugated goat antiserum to human immunoglobulin. Finally, the wells received substrate solution (150 g/ml 2,2'-azino-di- 3-ethylbenzyl-thiazoline sulfonic acid!, 0.001% H<sub>2</sub> O<sub>2</sub>, 0.1M citrate pH 4). The reaction was stopped after incubation for 30 min at 37 C. by the addition of 50 l/well of 10% SDS. The absorbance was read on a Flow Titertech ELISA reader at 414 nm. Samples were assayed in duplicate beginning at a dilution of 1:100 and by serial 2-fold dilutions thereafter.</p>
    <p>As an example, a single sample of HIV antibody-positive serum and normal human serum were each titrated on both recombinant and native p25gag. The antibody-positive serum reacted equally with viral and recombinant antigen. There was no reaction seen with serum from a normal individual.</p>
    <p>The reactivity of purified recombinant p25gag to various sera was then compared to that of natural p25gag protein purified by preparative polyacrylamide gel electrophoresis in an ELISA assay. For comparisons, assays were also done using disrupted gradient purified virus (5 g/ml) as antigen. The table below summarizes the results of assays on 8 AIDS sera that scored positive in the assay with disrupted virus and 6 normal sera that were negative in the disrupted virus assay.</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">______________________________________ELISA ASSAY TITER<sup>a</sup>SERUMNUMBER Disrupted Virus                Recomb. p25gag                            Viral p25gag______________________________________Group I: Sera Scoring As Positive in Virus ELISA<sup>b</sup>1      51,200        3,125       3,1255      12,800         25          256      12,800        625         6257      12,800        3,125       3,1258      25,600        15,625      15,6259      12,800        625         62513       800         125         12518      3,200        625         625GroupII: Sera Scoring Negative in Virus ELISA<sup>b</sup>15      -c           -           -16     -             -           -19     -             -           -21     -             -           -26     -             -           -33     -             -           -______________________________________ <sup>a</sup> Reported as the reciprocal of the serum dilution that gave a signa equivalent to 50% of the maximum. <sup>b</sup> Results were confirmed by immunofluorescence and immunoblotting as described previously. <sup>c</sup> No detectable signal at a 1:25 serum dilution.</pre>
    
    <p>These results show that p25gag purified from bacteria behaves identically to similarly purified p25gag from AIDS virus in an ELISA of the eight AIDS patient sera. The results of the ELISA show that there is a wide variation in the levels of anti-p25gag antibodies and suggests that antibodies to some virus-encoded proteins may not be detected using conventional virus-based assay systems.</p>
    <p>3.2. Expression of p16gag</p>
    <p>The sequence shown in FIG. 10 and coding for the p16gag protein was chemically synthesized  Warner et al. (1984) DNA 3: 401! using yeast-preferred codons. The blunt-end to SalI fragment (381 bp) was cloned into pvuII-SalI digested and gel-isolated ptac5 (see Section 3.1). The resulting plasmid, pGAG16, was used to transform E. coli D1210 cells, as in Section 3.1.3. Expression was induced with IPTG, and proteins were analyzed by polyacrylamide gel electrophoresis and Western analysis. A band of about 16,000 daltons was induced by IPTG in the transformed cells. This protein showed reactivity in Western blots with immune sera from AIDS patients. No reactivity was observed with sera from normal individuals.</p>
    <p>3.3 Expression of fusion protein p41gag</p>
    <p>A fusion protein of the p25gag and p16gag proteins of ARV-2, designated p41gag, was synthesized in E. coli strain D1210 transformed with plasmid pGAG41-10. pGAG41-10 was constructed from plasmidpGAG25-10 (see Section 3.1.2) as shown in FIG. 7 by inserting an SphI-HpaI fragment from the ARV-2 genome containing the sequences from the C-terminal p16gag portion of the p53gag precursor polyprotein and part of the p25gag protein between the SphI and BamHI sites of pGAG25-10. The coding strand of the DNA sequence cloned in pGAG41-10 is shown in FIG. 9. Transformation and induction of expression were effected by the procedures described above. The cells were treated and the p41gag protein was visualized on Coomassie-stained gel as described above. The approximate molecular weight of the observed protein was 41,000 daltons. The protein reacted with AIDS sera and monoclonal antibody to p25gag in Western and ELISA analyses carried out as above.</p>
    <p>3.4. Expression of p31pol</p>
    <p>3.4.1. Host-vector system</p>
    <p>The C-terminal region of the polymerase gene (p31pol). is synthesized by E. coli strain D1210 transformed with plasmid pTP31.2. Plasmid pTP31.2 is a pBR322 derivation which contains the sequence coding for p31 under transcriptional control of the hybrid tac promoter (described in Section 3.1). Expression of p31 is induced in bacterial transformants by IPTG.</p>
    <p>3.4.2. Construction of pTP31.2</p>
    <p>3.4.2.1. Construction of M13 template 01100484</p>
    <p>A 4.87 kb DNA fragment was isolated from a KpnI digest of ARV-2 (9b) containing the 3' end of the pol gene, orf-1, env and the 5' end of off-2, that had been run on a 1% low melting point agarose (Sea-Pack) gel and extracted with phenol at 65 C., precipitated with 100% ethanol and resuspended in TE. Eight l of this material were further digested with SstI for 1 h at 37 C. in a final volume of 10 l. After heat inactivation of the enzyme, 1.25 l of this digest were ligated to 20 ng of M13mp19 previously cut with KpnI and SstI, in the presence of ATP and in a final volume of 20 l. The reaction was allowed to proceed for 2 h at room temperature. Five l of this mixture were used to transform competent E. coli JM101. Clear plaques were grown and single-stranded DNA was prepared as described in Messing &amp; Vieira, (1982) Gene 19: 269-276. One of these plaques containing the 1848 bp fragment was designated 01100484.</p>
    <p>3.4.2.2. In vitro mutagenesis of 01100484</p>
    <p>The DNA sequence in 01100484 was altered by site-specific mutagenesis to generate a restriction site recognized by NcoI (CCATGG). An oligodeoxynucleotide that substitutes the A for a C at position 3845 (FIG. 5) and changes a T for an A at position 3851 (FIG. 5) was synthesized using solid phase phosphoramidite chemistry. Both of these changes are silent in terms of the amino acid sequence, and the second one was introduced to decrease the stability of the heterologous molecules. The oligomer was named ARV-216 and has the sequence: 5'-TTAAAATCACTTGCCATGGCTCTCCAATTACTG and corresponds to the noncoding strand since the M13 derivative template 01100484 is single-stranded and contains the coding strand. The 5' phosphorylated oligomer was annealed to the 01100484 M13 template at 55 C. in the presence of M13 sequencing primer, 50 mM Tris-HCl pH 8, 20 mM KCl, 7 mM MgCl<sub>2</sub> and 0.1 mM EDTA. The polymerization reaction was done in 100 l containing 50 ng/l DNA duplex, 150 mM dNTPs, 1 mM ATP, 33 mM Tris-acetate pH 7.8., 66 mM potassium acetate, 10 mM magnesium acetate, 5 mM DTT, 12.5 units of T4 polymerase, 100 g/ml T4 gene 32 protein and 5 units of T4 DNA ligase. The reaction was incubated at 30 C. for 30 min and was stopped by the addition of EDTA and SDS (10 mM and 0.2% respectively, final concentration).</p>
    <p>Competent JM101 E. coli cells were transformed with 1, 2, and 4 l of a 1:10 dilution of the polymerization product and plated into YT plates. Plaques were lifted by adsorption to nitrocellulose filters and denatured in 0.2N NaOH, 1.5M NaCl, followed by neutralization in 0.5M Tris-HCl pH 7.3, 3M NaCl and equilibrated in 6 SSC. The filters were blotted dry, baked at 80 C. for 2 h and preannealed at 37 C. in 0.2% SDS, 10 Denhardt's, 6 SSC. After 1 h, 7.5 million CPM of labeled ARV-216 were added to the filters and incubated for 2 additional h at 37 C. The filters were washed in 6 SSC at 42 C. for 20 min, blot-dried and used to expose film at -70 C. for 1 h using an intensifying screen. Strong hybridizing plaques were grown and single-stranded DNA was prepared from them and used as templates for sequencing. Sequencing showed that template 01021785 contains the NcoI site as well as the second substitution mentioned above.</p>
    <p>A second oligomer was synthesized to insert sites for SalI and EcoRI immediately after the termination codon of the pol gene (position 4647, FIG. 5). This oligomer was called ARV-248 and has the sequence: 5'-GGTGTTTTACTAAAGAATTCCGTCGACTAATCCTCATCC. Using the template 01020785, site specific mutagenesis was carried out as described above except that the filter wash after the hybridization was done at 65 C. As above, 8 strong hybridizing plaques were grown and single-stranded DNA was sequenced. The sequence of template 01031985 shows that it contains the restriction sites for NcoI, SalI, and EcoRI as intended.</p>
    <p>3.4.2.3. Isolation of DNA fragments containing p31</p>
    <p>Replicative form (RF) of the 01031985 template was prepared by growing. 6 clear plaques, each in 1.5 ml of 2 YT at 37 C. for 5 h. Double-stranded DNA was obtained as described by Maniatis et al., supra, pooled and resuspended in 100 l final volume. Ten l of RF were digested with NcoI and EcoRI in a final volume of 20 l. This fragment was used for direct p31 expression in bacteria. An additional 20 l of RF were cut with NcoI and SalI in 40 l. This fragment was used for p31 expression in yeast. The samples were run on a 1% low melting point agarose (Sea-Pack) gel and the DNAs were visualized by fluorescence with ethidium bromide. The 800 bp bands were cut and the DNAs were extracted from the gel as mentioned above and resuspended in 10 l of TE. The fragments were called ARV248NR#2 and ARV248NL, respectively.</p>
    <p>3.4.2.4. Cloning of p31 into plot 7</p>
    <p>The vector plot #7 (3 g)  Hallewell et al. (1985) Nucl. Acid Res. 13: 2017-2034! was cut with NcoI and EcoRI in 40 l final volume and the enzymes were heat-inactivated after 3 h. Two l of this digest were mixed with 2 l of ARV248NR#2 and ligated in 20 l in the presence of ATP and T4 DNA ligase at 14 C. overnight, and 10 l of this mixture were used to transform competent D1210 cells. Colonies resistant to 2 mM IPTG and 100 g/ml ampicillin were selected and supercoiled DNA was extracted from each of them. The DNAs were then restricted with NcoI and EcoRI and analyzed by agarose gel electrophoresis. Clones with the appropriate 800 bp insert were selected for further use. They are designated pRSP248 numbers 3 and 4.</p>
    <p>3.4.2.5. Construction of pTP31</p>
    <p>The NcoI site introduced into 01100485 is 52 bp downstream from the putative start of p31. Three oligomers were synthesized as above that code for the first 17 amino acids of p31 and generate a cohesive NcoI end at the 3' end of the molecule. The 5' end of the molecule has been extended beyond the initiation codon to include a ribsome binding site. The oligomers that were synthesized have the sequences: ##STR9##</p>
    <p>One hundred fifty picomoles each of dephosphorylated ARV-221, phosphorylated ARV-222 and ARV-223 were ligated to 20 g of pRSP248 previously cut with NcoI, at 14 C. for 18 h in a final volume of 62 l. After phenol extraction and ethanol precipitation, the DNA was resuspended in 40 l H<sub>2</sub> O and incubated with 15 units of Klenow fragment in the presence of 0.5 mM dNTPs for 1 h at room temperature. The sample was phenol extracted, ethanol precipitated, resuspended in 40 l H<sub>2</sub> O, and digested with EcoRI. The DNA was then run on a low melting point agarose gel and the fragment of about 820 bp was extracted as described above and resuspended in a final volume of 20 l of H<sub>2</sub> O. After phosphorylating the ends, 5 l of the sample were incubated for 18 h at 14 C. with 150 ng of plot7 that had been cut with PvuII and EcoRI and its ends dephosphorylated, in the presence of T4 DNA ligase, ATP and in a final volume of 31 l. Five l of ligation product were used to transform RR1M15. Clones resistant to 100 g/ml of ampicillin were selected and supercoiled DNA was extracted from them. The DNAs were digested with NcoI and EcoRI and resolved on a 1% agarose gel. Colonies with the appropriate size insert were obtained and named pTP31. The p31 sequence contained in the insert is shown in FIG. 12. Underlined sequences were chemically synthesized. Others were derived from DNA.</p>
    <p>3.4.3. Screening of transformants</p>
    <p>Bacterial transformants containing either the vector alone, or the vector with the p31 DNA (pTP31.2) were grown in L-Broth with 0.02% ampicillin to an OD<sub>650</sub> of 0.5. Cultures were induced by the addition of IPTG to a final concentration of 2 mM and grown for 3 more hours. Bacteria from 1 ml cultures were pelleted and resuspended in 200 l of gel sample buffer. The cells were disrupted by three cycles of freezing and thawing, boiled, and the extracts loaded onto 12.5% polyacrylamide-SDS minigels. Proteins were electrophoresed and transferred to nitrocellulose by electro-blotting. The nitrocellulose filters were reacted with serum EW5111 (diluted 1:100; positive reference serum from the CDC that reacts strongly with viral p31), horse radish peroxidase-conjugated goat anti-human IgG and HRP substrate. A prominent band at -30,000 d and several lower molecular weight species were seen in gels of extracts from transformants with the p31 DNA, but not in extracts from bacteria transformed with the vector alone.</p>
    <p>3.4.4. Characterization of recombinant protein</p>
    <p>Lysozyme-NP40 extracts were prepared from bacteria transformed with pTP31.2 or vector alone. Five ml cultures were grown, the cells pelleted and resuspended in 1 ml of 50 mM Tris-HCl pH 8, 0.5 mM EDTA, 1 mg/ml lysozyme and incubated at 0 C. for 15 min NaCl, MgCl<sub>2</sub>, and NP40 were added to final concentrations of 0.4, 5 mM and 0.5% respectively, mixed and incubated with DNAse I (100 g/ml) at 0 C. for 30 min. When EW5111 serum (diluted 1:100) was preincubated with a 1:10 dilution of the cell extracts from bacteria transformed with pTP31.2, prior to reaction with a virus blot, the viral p31 band was completely eliminated, while reactivity with other viral proteins remained unaffected. In contrast, extracts from bacteria transformed with the vector alone did not absorb out the p31 reactive antibodies. The viral p31 protein is thus the product of the C-terminal or endonuclease region of the pol gene of HIV-1.</p>
    <p>3.5. Expression of SOD-p31 fusion protein</p>
    <p>In order to generate a fused protein SOD-p31 that can be expressed in E. coli, 2 l of the fragment ARV248NL were ligated to 100 ng of a NcoI-SalI digest of pSODCF2  Steimer et al. (1986) J. Virol. 58: 9-16!, a plasmid that contains the human SOD coding region under the regulation of the tacl promoter, in 20 l final volume for 18 h at 14 C. Competent D1210 cells were transformed with 5 l of the ligation mix and colonies were selected on L-broth ampicillin plates. Colonies with the correct insert were called E. coli D1210 (pTSp31). The expression of the fusion protein was analyzed in the same manner as the direct expression of p31. The analysis showed production of large amounts of a protein of about 47 kd that is immunoreactive with EW5111 but not with normal human sera. Further characterization of the recombinant protein is described in Section 4.3.3.3.</p>
    <p>3.6. Expression of SOD-env5b fusion protein</p>
    <p>3.6.1. Host-Vector System</p>
    <p>A hybrid protein consisting of human superoxide dismutase (SOD) fused to a viral envelope (env-5b) protein is synthesized by E. coli strain D1210 transformed with plasmid pSOD/env5b. Plasmid pSOD/env5b is a bacterial expression vector which contains the sequence coding for human SOD  Hallewell et al. (1985) Nucl. Acid. Res. 13: 2017! fused to the env-5b region of the viral env gene  Sanchez-Pescador et al. (1985) Science 227: 484! as well as pBR322 sequences including the ampicillin resistant (amp<sup>R</sup>) gene.</p>
    <p>Expression of SOD/env-5b fusion protein is non-constitutive and it is under transcriptional control of a hybrid tac promoter  De Boer et al. (1983) Proc. Natl. Acad. Sci. USA 80: 21-25! derived from sequences of the trp and the lac UV5 promoters. Expression of SOD/env-5b is induced in bacterial transformants with IPTG.</p>
    <p>E. coli D1210, a lacl-repressor overproducing strain, carries the lacI and lacy<sup>+</sup> alleles on the chromosome but otherwise is identical to E. coli HB101 (F<sup>-</sup> lacI<sup>+</sup>, lacO<sup>+</sup>, lacY<sup>-</sup>, gal<sup>-</sup>, pro<sup>-</sup>, leu<sup>-</sup>, thi<sup>-</sup>, end<sup>-</sup>, hsm<sup>-</sup>, hsr<sup>-</sup>, recA<sup>-</sup>, rpsL<sup>-</sup>) from which it was derived  Sadler et al. (1980) Gene 8: 279-300!.</p>
    <p>3.6.2. Construction of pSOD/env5b</p>
    <p>Plasmid pSOD/env5b is a pBR322 derivative which contains the sequences coding for env-5b  Sanchez-Pescador et al. (1985) supra! under transcriptional control of a hybrid tac promoter. De Boer et al. (1983) supra.</p>
    <p>Plasmid pSOD/env5b (FIG. 25) was constructed by cloning a 322 base pairs synthetic DNA fragment coding for env-5b into plasmid pSODCF2  Steimer et al. (1986) J. Virol. 58: 9-16!, a vector containing the human SOD gene under the control of the tacI promoter. Plasmid pSODCF2 is derived from vector ptac5  Hallewell et al. (1985) Nucl. Acid. Res. 13: 2017! a pBR322 derivative which contains the tac promoter, Shine Delgarno sequences and a polylinker as a substitution of the original pBR322 sequences comprised between the EcoRI and PvuII restriction sites.</p>
    <p>The 322 base pairs synthetic DNA fragment codes for the env-5b fragment (amino acid residues 557 to 677 as numbered in FIG. 5) with an additional methionine to allow for translational initiation site. The synthetic gene was prepared by synthesizing oligonucleotides varying in length between 34 and 48 bases, and purifying and kinasing them individually according to standard procedures. Warner et al. (1984) DNA 3: 401. The solution was phenol extracted and ethanol precipitated. The pellet was redissolved in 30 l of a buffer containing 20 mM Tris-HCl (pH 8.0), 10 mM MgCl<sub>2</sub>, and 10 mM dithiothreitol, and heated to 90 C. After allowing to cool over 3 hours, the solution was made 3 mM in ATP, brought up to a volume of 45 l, and 5 l of T4 DNA ligase (410<sup>5</sup> U/ml Biolabs) added. The ligation was stopped after 10 minutes at 25 C. by phenol extraction and ethanol precipitation. The pellet was redissolved in 80 l H<sub>2</sub> O in 10 l of 10 high salt restriction buffer and digested for 2 hours at 37 C. with 5 l each of NcoI and SalI (Biolabs). The full-length gene was then purified on a 7% native polyacrylamide gel, electroeluted, and cloned into NcoI/SalI-digested pBS-100.</p>
    <p>The cloned env-5b gene was isolated as an NcoI/SalI fragment from the resulting plasmid and cloned into NcoI/SalI-digested pSODCF2 to give plasmid pSOD/env5b. The resulting bacterial expression vector was used to express the SOD/env-5b fusion protein in E. coli under the control of the tac-1 promoter. Hallewell et al. (1985) Nucleic Acids Res. 13: 2017.</p>
    <p>FIG. 15 shows the nucleotide sequence of the SOD/env-5b insert cloned in pSOD/env5b and the amino acid sequences derived for it. Sequences coding for human SOD were derived from a human cDNA isolated as described in Hallewell et al. (1985), supra. Sequences coding for env-5b were chemically synthesized by the phosphoramidite method as originally described by Beaucage &amp; Caruthers, (1981) Tetrahedron. Lett. 32: 1859.</p>
    <p>3.6.3. Expression of SOD/env-5b fusion protein</p>
    <p>E. coli D1210 cells were made competent for transformation following a standard procedure and transformed with pSOD/env5b. The transformation mix was plated on agar plates made with L-broth containing 100 g/ml ampicillin. Plates were incubated for 12 hours at 37 C.</p>
    <p>Single ampicillin resistant colonies were transferred into L-broth containing 100 g/ml ampicillin and grown at 37 C. Expression of env-5b was induced by adding 100 mM IPTG to a final concentration of 1 mM followed by incubation at 37 C. for 2 hours.</p>
    <p>Cells from induced cultures were pelleted and resuspended in Laemmli sample buffer. Laemmli (1970) Nature 227: 680. After 3 cycles of boiling and freezing, portions of resultant lysates were analyzed on standard denaturing acylamide gels. Laemmli (1970), supra. Proteins were visualized by staining with Coomassie blue.</p>
    <p>The extent of expression was initially determined by appearance of new protein bands for induced candidate samples compared with controls. Proteins of molecular weights expected for the cloned DNA were determined by visual inspection of the gel lanes with reference to a standard protein band of known amount.</p>
    <p>Authenticity of the expressed proteins was determined by standard western transfer of proteins to nitrocellulose and analysis with appropriate human or rabbit immune sera or mouse monoclonal antibodies or by ELISA assays of soluble E. coli proteins using human immune sera from AIDS patients.</p>
    <p>3.6.4. Protein purification</p>
    <p>Single colonies of D1210 (pSOD/env5b) were inoculated into 2 ml aliquots of L-broth containing 100 g/ml ampicillin and the cultures were grown overnight at 37 C. The culture was made 15% glycerol by adding 1/4 volume of 75% sterile glycerol. The glycerol cell stocks aliquoted and quickly frozen in liquid nitrogen. The aliquots were stored at -70 C.</p>
    <p>A frozen aliguot of the seed stock (200 l) was thawed and used to inoculate 110 ml of L-broth with 0.01% ampicillin medium. The culture was grown to saturation at 37 C. with agitation. A 15 ml portion of the culture was added to each of six (6) flasks containing 1.5 L of L-broth with 0.01% ampicillin medium. The cultures were grown at 37 C. with agitation. During late log phase (after approximately 6 h.), env-5b expression was induced by addition of IPTG to a final concentration of 1 mM. Incubation was continued for an additional 2-4 h. at 37 C.</p>
    <p>After the cells had completed the growth period they were harvested by centrifugation and pooled. The packed cells were stored at either 4 C. or -20 C. The frozen (or refrigerated) cells were thawed and suspended in 2 volumes of cell lysis solution (1.0 mM PMSF, 0.18 mM EDTA, 230 mM tris-HCl, 0.425 mg/ml Lysozyme) on ice for 1 h. Approximately 1.5 volumes of DNA Digestion Solution (1.0 mM PMSF, 30 units/ml DNAse I, 1.0 mM MgCl<sub>2</sub>) was added and then the cell lysate was sonicated at room temperature for 30 minutes. The lysate was centrifuged at 113,000 g for 15 minutes.</p>
    <p>The supernatant was discarded and the pellet was suspended in solubilization buffer (0.1% -mercaptoethanol in 7M guanidine-HCl) and rocked for 3 h. at 2-8 C. The solution was subjected to ultracentrifugation at 25,500 g for 4 h. at 2-8 C. The pellet was discarded, and the supernatant was diluted with 6 volumes of water, and then incubated at room temperature for 1-2 h. The diluted supernatant was centrifuged at 11,300 g for 30 minutes. Following centrifugation, the supernatant was discarded and the pellet was resuspended in 10-15 volumes of elution and resuspension buffer (0.1% -mercaptoethanol, 50 mM Tris-HCl in 3.5M guanidine-HCl, pH 7.4). The suspension was centrifuged at 2,000 g for 10 minutes at 2-8 C. to remove any insoluble material.</p>
    <p>The resolubilized fraction was chromatographed on a Sephacryl S-300 column and eluted with Elution and Resuspension Buffer. Fractions were collected and the absorbance was monitored at 280 nm. Fractions containing env-5b as determined by SDS-PAGE were pooled. The pooled fractions were concentrated to 1/5 the original volume by ultrafiltration in an Amicon unit under N<sub>2</sub> pressure.</p>
    <p>The concentrate was dialyzed (MW cutoff 2000). against 50 to 100 volumes of urea-containing dialysis buffer (0.1% B-Mercaptoethanol, 50 mM Tris-HCl in 7.5M urea, pH 7.4) for 16-24 h. at -8 C. The protein concentration of the dialyzed env-5b was adjusted to 1.02.0 mg/ml (based on modified Lowry assay).</p>
    <p>3.7. Expression of -gal-env fusion proteins.</p>
    <p>3.7.1. Host-vector system</p>
    <p>A partial env protein is synthesized by E. coli D1210 transformed with plasmid pII-3. Plasmid pII-3 (ATCC No. 67549) is a bacterial expression vector which contains the sequence for 2/3 (carboxyl end) of the env protein fused to E. coli b-galactosidase DNA in the vector pNL291. Expression of the -gal-env fusion protein is induced with IPTG.</p>
    <p>3.7.2. Construction of pII-3</p>
    <p>Plasmid pII-3 was constructed by cloning a 1855 bp BglII-XhoI fragment coding for 2/3 of the env protein. The fragment extends from nt 6604 to nt 8460 (FIG. 5) and codes for env amino acid residues from number 276 to the end of the env protein.</p>
    <p>To prepare the plasmid, the 1856 bp fragment was isolated by gel electrophoresis. The BglII-XhoI fragment was cloned into pNL291 which had been previously digested with BamHI and XhoI. Plasmid pNL291 is a derivative pUR291  Ruther et al. (1983) EMBO J. 2: 1791-1794! in which a polylinker containing the additional restriction sites NcoI, PvuII and XhoI was substituted between the BamHI and SalI sites. Plasmid pUR291 is an inducible E. coli expression vector which produces -galactosidase C-terminal fusion proteins.</p>
    <p>3.7.3. Expression and characterization of fusion protein</p>
    <p>E. coli D1210 cells were transformed with 25-50 ng of pII-3. The transformation mix was plated onto L-Broth agar plates containing 100 g/ml ampicillin. Single amp<sup>R</sup> colonies were grown in L-Broth/amp and expression of the fusion protein was induced by IPTG addition (1 mM) followed by incubation. Cell extracts were prepared and analyzed by SDS gel electrophoresis and immunoassays using human immune sera or rabbit immune sera.</p>
    <p>Results of this analysis showed that a large molecular weight protein is induced with IPTG. Expression levels are about 20 mg/l. This fusion protein reacted with human immune sera from AIDS patients in ELISA assays.</p>
    <p>4. Expression of HIV Polypeptides in Yeast</p>
    <p>4.1. Expression of envH peptide</p>
    <p>4.1.1. Host-vector system</p>
    <p>A partial env protein is synthesized by S. cerevisiae 2150-2-3 transformed with plasmid pDPC303. Plasmid pDPC303 is a yeast expression vector which contains the sequence coding for 2/3 of the env protein, envH, as well as pBR322 sequences including the amp<sup>R</sup> gene and 2-micron sequences including the yeast leu 2-04 gene. Expression of envH is under regulation of the yeast pyruvate kinase promoter and terminator sequences. Yeast strain S. cerevisiae 2150-2-3 has the following genotype: Mat a, ade 1, leu 2-112, cir. This strain was obtained from Dr. Leland Hartwell, University of Washington.</p>
    <p>4.1.2. Construction of pDPC303</p>
    <p>Plasmid pDPC303 contains an "expression cassette" (described below) for enVH cloned into the BamHI site of vector pC1/1. Vector pC1/1 contains pBR322 and 2 micron sequences including the amp<sup>R</sup> and yeast leu 2-04 markers. It was derived from pJDB219d  Beggs (1978) Nature 275: 104! by replacing the pMB9 region with pBR322 sequences.</p>
    <p>The "expression cassette" for envH consists of the following sequences fused together in this order (5' to 3'): yeast pyruvate kinase (PYK) promoter, envH coding region and PYK terminator. The PYK promoter and terminator regions were derived from the PYK gene isolated as described in Burke et al. (1983) J. Biol. Chem. 258: 2193-2201.</p>
    <p>The envH fragment cloned into the expression cassette was derived from ARV-2 DNA and comprises a 1395 bp fragment which codes for env amino acid residues 27 through 491 coded by nt. 5857 to nt 7242 (FIG. 5). In addition, there are 5 extra codons fused in reading frame in the 5' end, the first codon corresponding to a methionine, and 4 extra codons fused in reading frame at the 3' end followed by a stop codon. The extra codons were incorporated to facilitate cloning procedures exclusively.</p>
    <p>FIG. 11 shows the coding strand of the nucleotide sequence cloned in pDPC303 and the amino acid sequence derived from it. DNA sequences that are not underlined in the figure were derived directly from the ARV-2 9B DNA described above. All other sequences were either chemically synthesized, or derived from the PYK vector.</p>
    <p>4.1.3. Transformation and expression</p>
    <p>Yeast cells S. cerevisiae 2150-2-3 (Mat a, ade 1, leu 2-04, cir) were transformed as described by Hinnen et al. (1978) Proc. Natl. Acad. Sci. USA 75: 1929-1933, and plated onto leu<sup>-</sup> selective plates. Single colonies were inoculated into leu<sup>-</sup> selective media and grown to saturation. Cells were harvested and the env protein was purified and characterized as described below.</p>
    <p>4.1.4. Purification of envH protein</p>
    <p>4.1.4.1. Cell breakage</p>
    <p>Frozen S. cerevisiae 2150-2-3 (pDPC303) are thawed and suspended in 1 volume of lysis buffer (1 g/ml pepstatin, 0.001M PMSF, 0.001M EDTA, 0.15M NaCl, 0.05M Tris-HCl pH 8.0), and 1 volume of acid-washed glass beads are added. Cells are broken in a noncontinuous system using a 300 ml glass unit of Dyno-mill at 3000 rpm for 10 min. The jacket is kept cool by a -20 C. ethylene glycol solution. Glass beads are decanted by letting the mixture set for 3 minutes on ice. The cell extract is recovered and centrifuged at 18,000 rpm (39,200 g) for 35 min. The supernatant is discarded and the precipitate (pellet 1) is further treated as indicated below.</p>
    <heading>4.1.4.2. SDS extraction of insoluble material</heading> <p>Pellet 1 is resuspended in 4 volumes of Tris-HCl buffer (0.01M Tris-HCl, pH 8.0, 0.01M NaCl, 0.001M PMSF, 1 g/ml pepstatin, 0.001M EDTA, 0.1% SDS) and extracted for 2 h at 4 C. with agitation. The solution is centrifuged at 6,300 x g for 15 min. The insoluble fraction (pellet 2) is resuspended in 4 volumes (360 ml) of PBS (per liter: 0.2 g KCl, 0.2 g KH<sub>2</sub> PO<sub>4</sub>, 8.0 g NaCl, 2.9 g Na<sub>2</sub> HPO<sub>4</sub>.12H<sub>2</sub> O), 0.1% SDS, 0.001M EDTA, 0.001M PMSF, 1 g/ml pepstatin, and centrifuged at 6,300 x g for 15 min. The pellet (pellet 3) is suspended in 4 volumes of PBS, 0.2% SDS, 0.001M EDTA, 0.001M PMSF, 1 g/ml pepstatin and is extracted for 12 h at 4 C. with agitation on a tube rocker. The solution is centrifuged at 6,300 x g for 15 min. The soluble fraction is recovered for further purification as indicated below. (The pellet can be reextracted by resuspending it in 4 volumes of 2.3% SDS, 5% -mercaptoethanol, and boiling for 5 min. After boiling, the solution is centrifuged at 6,300 x g for 15 min. The soluble fraction is recovered for further purification.)</p>
    <p>4.1.4.3. Selective precipitation and gel filtration</p>
    <p>The soluble fraction is concentrated by precipitation with 30% ammonium sulfate at 4 C. The pellet (pellet 4) is resuspended in 2.3% SDS, 5% -mercaptoethanol, and chromatographed on an ACA 34 (LKB Products) gel filtration column. The column is equilibrated with PBS, 0.1% SDS, at room temperature. Chromatography is developed in the same solution with a flow rate of 0.3 ml/min. Five ml fractions are collected, pooled and characterized by protein gel electrophoresis, Western analysis, and ELISA. If needed, pooled fractions are concentrated by vacuum dialysis on Spectrapor #2 (MW cutoff. 12-14K).</p>
    <p>4.1.5. Characterization of recombinant envH</p>
    <p>SDS polyacrylamide gel-analysis (12% acrylamide gels) showed that a new 55,000 dalton protein was being synthesized in yeast cells transformed with the env-containing vector. The 55,000 dalton protein is absent from cells transformed with control plasmid (vector without env insert). The identity of env was confirmed by both ELISA (Section 3.1.6.3.) and Western analysis using AIDS patient serum. In both assays the 55,000 dalton protein showed immunoreactivity. No reactivity was obtained with serum from a normal individual.</p>
    <p>4.2. Expression of env subregion polypeptides</p>
    <p>The following example describes the expression of DNA coding regions defined as env-1, env-2, env-3, env-4 and env-5b (see FIG. 13). Env-1 and env-4 approximate the N- and C-terminal halves of gp120env, respectively. Env-2 and env-3 approximate the entire env polypeptide moieties of gp120 and gp41, respectively. Env-5b corresponds to the region of gp41env thought to be external to the cell membrane.</p>
    <p>4.2.1. Env-1</p>
    <p>4.2.1.1. GAP promoter</p>
    <p>Proviral ARV-2 sequences were isolated from phage ARV-2 (7D) and subcloned into plasmid pUC19. Yanisch et al. (1985) Gene 33: 103-119. The plasmid containing the proviral sequences was named pUC19ARV7D/7. To isolate the env-1 region, the plasmid was cut with FokI at cys27 of the env region (nt5857, FIG. 5) and with BqlII at arg276 (nt6604, FIG. 5). This provided a nucleotide segment with a coding capacity for 28 kD from the N-terminus of gp120 without the 29 amino acid signal sequence of the env gone product. A synthetic, NcoI/FokI adaptor, with the following sequence, was ligated to the env-1 segment: ##STR10## A second adaptor, for BqlII/SalI, was also ligated to the env-1 segment. This second adaptor had the following sequence: ##STR11## The first synthetic adaptor contains an in-frame initiation codon, and the second synthetic adaptor contains an in-frame stop codon.</p>
    <p>The env-1 segment modified with the synthetic adaptors was then ligated into pPGAP1 previously linearized with NcoI and SalI. Plasmid pPGAP1 has been previously described. EPO Pub. No. 164,556. It contains a yeast glyceraldehyde-3-phosphate dehydrogenase (GAPDH) promoter and terminator sequences flanking NcoI and SalI cloning sites. The ligation of the modified env-1 segment into pPGAP1 produces plasmid pPGAP/FGenv, which contains the env-1 sequence fused directly to the GAPDH promoter and terminator sequences. The expression cassette containing GAPDH promoter-env-1-GAPDH terminator was excised by digestion of pPGAP/FGenv with BamHI and gel purification of the fragment. The expression cassette was cloned into BamHI-digested pCl/1 (see Section 4.1.2) to yield pCl/1FGenv.</p>
    <p>Plasmid pCl/1FGenv was used to transform yeast strain AB110 (Mat, ura 3-52, leu 2-04 or both leu 2-3 and leu 2-112, pep 4-3, his 4-580, cir; see EPO Pub. No. 620,662 &amp; Section 4.5.2) as described previously. Hinnen et al. (1978) Proc. Natl. Acad. Sci. USA 75: 1929. Yeast colonies transformed with the expression plasmid were grown in synthetic complete media lacking leucine at a concentration of 2% glucose. Large-scale cultures were grown in YEP medium with 2% glucose as described in Section 4.1.3. Lysates were prepared from yeast cultures as described previously, and were then separated into soluble and insoluble fractions by centrifugation, and analyzed by polyacrylamide gel electrophoresis, see Section 4.1.4. A heavily expressed protein corresponding to env-1 was readily discernible in the insoluble fraction by Coomassie blue staining. This protein also migrated at a molecular weight of approximately 28 kD, as predicted from the DNA sequence.</p>
    <p>4.2.2. Env-2</p>
    <p>The env-2 polypeptide is similar to the previously described envH (Section 4.1.1) and corresponds to the vital gp120 glycoprotein. Env-2 differs from envH in the 5 amino terminal residues and is under the regulatory control of the GAPDH promoter as described for the expression of env-1. Env-2 is a polypeptide having the amino acid sequence of gp120env residues 26510.</p>
    <p>4.2.2.1. Construction of pAB24-GAP-env2</p>
    <p>Plasmid pAB24-GAP-env2 (FIG. 28) contains an "expression cassette" for the env gene cloned into BamHI site of the yeast shuttle vector pAB24 (below). Expression of the env gene is under regulatory control of the GAPDH promoter and the PYK terminator. Construction of pAB24-GAP-env2 was accomplished by ligating (i) an approximately 952 bp BamPII-StuI fragment from plasmid pCl/1-FGenv (Section 4.2.1) which contains the GAPDH promoter and the envelope sequences coding for amino acids 26-267, and (ii) an approximately 1474 bp StuI-BamHI fragment from plasmid pDPC302 (which is similar to pDPC303 in Section 4.1.1. except that it extends 57 nucleotides in the 3' direction of the envelope coding region), which codes for env amino acids 267-510 and the PYK terminator, into plasmid pAB24 which had been previously digested with BamHI and treated with alkaline phosphatase. The direction of transcription in the "expression cassette" is in the opposite direction of the tetracycline gene of the pBR322 sequences.</p>
    <p>Plasmid pAB24 (FIG. 27) is a yeast shuttle vector which contains the complete 2 sequence  Broach, in: Molecular Biology of the Yeast Saccharomyces, Vol. 1, p. 445, Cold Spring Harbor Press (1981)! and pBR322 sequences. It also contains the yeast URA3 gene derived from plasmid YEp24  Botstein et al. (1979) Gene 8: 17!and the yeast LEU<sup>2d</sup> gene derived from plasmid p(I1/1. EPO Pub. No. 116,201. Plasmid pAB24 was constructed by digesting YEp24 with EcoRI and regigating the vector to remove the partial 2 sequences. The resulting plasmid, YEp24RI, was linearized by digestion with ClaI and ligated with the complete 2 plasmid which had been linearized with ClaI. The resulting plasmid, pCBou, was then digested with XbaI and the 8605 bp vector fragment was gel isolated. This isolated XbaI fragment was ligated with a 4460 bp XbaI fragment containing the LEU<sup>2</sup> d gene isolated from pCl/1 the orientation of the LEU<sup>2</sup> d gene is in the same direction as the URA3 gene. Insertion of the expression cassette was in the unique BamHI site of the pBR322 sequences, this interrupting the gene for bacterial resistance to tetracycline.</p>
    <heading>4.2.2.2. Transformation and expression</heading> <p>Yeast cells S. cerevisiae 2150-2-3 (Mat a, ade 1, leu 2-04, cir) were transformed as described by Hinnen et al.  (1978) Proc. Natl. Acad. Sci. USA 75: 1929-1933! and plated onto leu-selective plates. Single colonies were inoculated into leu-selective media and grown to saturation. The culture was further inoculated into YEP 2% glucose media. Cells were harvested and the env-2 protein was purified and characterized as described below.</p>
    <p>4.2.2.3. Purification of env-2 protein</p>
    <p>Frozen S. cerevisiae 2150-2-3 (pKB24-GAP-env2) were thawed and suspended in 1 volume of lysis buffer (1 g/ml pepstatin, 0.001M PMSF, 0.001M EDTA, 0.15M NaCl, 0.05M Tris-HCl pH 8.0), and 1 volume of acid-washed glass beads are added. Cells are broken in a non-continuous system using a 300 ml glass unit of Dyno-mill at 3000 rpm for 10-25 min. The jacket is kept cool by a -20 C. ethylene glycol solution. Glass beads are-decanted by letting the mixture set for approximately 3 minutes on ice. The cell extract is recovered and centrifuged at 18,000 rpm (39,200 x g) for 35 min. The supernatant is discarded and the precipitate (pellet 1) is further treated at indicated below.</p>
    <p>Pellet 1 is resuspended in 4 volumes of Tris-HCl buffer (0.01M Tris-HCl, pH 8.0, 0.01M NaCl, M PMSF, 1 g/ml pepstatin, 0.001M EDTA, 0.1% SDS) and extracted for 2 h at 4 C. with agitation. The solution is centrifuged at 6,300 x g for 15 min. The insoluble fraction (pellet 2) is resuspended in 4 volumes of PBS (per liter: 0.2 g KCl, 0.2 g KH<sub>2</sub> PO<sub>4</sub>, 8.0 g NaCl, 2.9 g Na<sub>2</sub> HPO<sub>4</sub>.12H<sub>2</sub> O), 0.1% SDS, 0.001M EDTA, 0.001M PMSF, 1 g/ml pepstatin, end centrifuged at 6,300 x g for 15 min. The pellet (pellet 3) is suspended in 4 volumes of PBS, 0.2% SDS, 0.001M EDTA, 0.001M PMSF, 1 g/ml pepstatin and is extracted for 123 h at 4 C. with agitation on a tube rocker. The solution is centrifuged at 6,300 x g for 15 min. The soluble fraction is recovered for further purification as indicated below. (The pellet can be reextracted by resuspending it in 4 volumes of 2.3% SDS, 5% -mercaptoethanol, and boiling for 5 min. After boiling, the solution is centrifuged at 6,300 x g for 15 min. The soluble fraction is recovered for further purification.)</p>
    <p>The soluble fraction is concentrated by precipitation with 30% ammonium sulfate at 4 C. The pellet (pellet 4) is resuspended in 2.3% SDS, 5% -mercaptoethanol, and chromatographed on an ACA 34 (LKB Products) gel filtration column. The column is equilibrated with PBS, 0.1% SDS, at room temperature. Chromatography is developed in the same solution with a flow rate of 0.30.1 ml/min. 52 ml fractions are collected, pooled and characterized by protein gel electrophoresis, Western analysis, and ELISA. If needed, pooled fractions are concentrated by vacuum dialysis on Spectrapor #2 (MW cutoff 12-14K), or by membrane filtration using an Amicon concentrator using a PM10 membrane.</p>
    <p>SDS polyacrylamide gel analysis (12% acrylamide gels) showed that a new 55,000 dalton protein was being synthesized in yeast cells transformed with the env-containing vector. The 55,000 dalton protein is absent from cells transformed with control plasmid (vector without env insert). The identity of env-2 was confirmed by both ELISA and Western analysis using AIDS patient serum. In both assays the 55,000 dalton protein showed immunoreactivity. No reactivity was obtained with serum from a pool of normal individuals.</p>
    <heading>4.2.2.4 Immunogenicity</heading> <p>To determine the immunogenicity of polypeptides expressed in yeast, purified env-2 (Section 4.2.2.3) was used to immunize rabbits. Rabbits received perilymphnodal injections of 250 g of purified env-2 polypeptide in complete Freund's adjuvant. Twenty-one days later, the rabbits were boosted with intramuscular injections of 250 g of env-2 polypeptide in incomplete Freund's adjuvant. Ten days later the animals were bled and then set up on a schedule of boosting, bleeding 10 days later, and reboosting after 21 days. Antibody titers to env-2 polypeptide for the rabbits were measured using ELISA. Antibodies from the rabbits reacted specifically with env glycoproteins in virus (gp120) and in infected cells (gp160). Thus, the polypeptide moiety representing the N-terminal domain of the env region is immunogenic when expressed in a heterologous host species.</p>
    <p>4.2.3. Env- 3</p>
    <p>42.3.1. GAP promoter</p>
    <p>Env-3, the gp41env equivalent (AA 529-855 of env), was prepared by cutting the env insert of plasmid pUC19ARV7D/7 (Section 4.2.1.1) at the HgaI site (nt7401, FIG. 5) and at the XhoI site 3' to the env termination codon (nt8460, FIG. 5). The resulting env-3 segment was modified by the addition of synthetic adapters. The 5' end was modified by the addition of an NcoI/HgaI adapter which reintroduced the coding sequence to met529 (nt7817, FIG. 5). The linker had the following sequence: ##STR12## A second synthetic XhoI/SalI adapter molecule was prepared and ligated to the XhoI end of env-3, the adapter having the following sequence: ##STR13## The HqaI/XhoI fragment was cloned together with the above linkers into NcoI/SalI-digested pPGAP/FGenv (Section 4.2.1.1), and the resulting plasmid, pPGAP/HXenv, was digested with BamHI. The BamHI expression cassette was cloned into pCl/1. The resulting expression vector pCl/1GAP/P/HXenv was expressed as described above after transformation of yeast strain AB110 (Section 4.2.1). Extreme toxicity, as evidenced by slow growth of cells, was observed when this gene was expressed constitutively in yeast under control of the GAPDH promoter.</p>
    <p>4.2.3.2. ADH-2/GAPDH promoter</p>
    <p>In order to express env-3 under the control of the glucose-regulable ADH-2/GAPDH (or ADH-2/GAP) promoter, the Nco-I/BamHI fragment containing the env-3 coding region and the GAPDH terminator Was excised from pPGAP/HXenv. This was cloned together with the ADH-2/GAPDH promoter as a BamHI/Nco-I fragment (from pJS103) into BamHI-digested and phosphatized pCl/1. The resulting expression vector, pCl/1ADH-2/GAP/HXenv, was expressed as described below after transformation of yeast strain AB110.</p>
    <p>Plasmid pJS103, which contains the hybrid ADH-2/GAPDH promoter employed above, was constructed as follows. The ADH-2 portion of the promoter was constructed by cutting a plasmid containing the wild-type ADH2 gene from plasmid pADR2  Beier et al. (1982) Nature 300: 724-728! with restriction enzyme EcoR5, which cuts at position +66 relative to the ATG start codon, as Well as in two other sites in pADR2, outside of the ADH2 region. The resulting mixture of a vector fragment and two smaller fragments was resected with Ba131 exonuclease to remove about 300 bp. Synthetic XhoI linkers were ligated onto the Ba131-treated DNA. The resulting DNA linker vector fragment (about 5 kb) was separated from the linkers by column chromatography, cut with restriction enzyme XhoI, religated, and used to transform E. coli to ampicillin resistance. The positions of the XhoI linker were determined by DNA sequencing. One plasmid which contained an XhoI linker within the 5'-nontranscribed region of the ADH2 gene (position -232 from ATG) was cut with the restriction enzyme XhoI, treated with nuclease S1, and subsequently treated with the restriction enzyme EcoRI to create a linear vector molecule having 1 blunt end at the site of the XhoI linker and an EcoRI end. The GAP portion of the promoter was constructed by cutting plasmid pPGAP with the enzymes BamHI and EcoRI, followed by the isolation of the 0.4 Kbp DNA fragment. This purified fragment was then completely digested with the AluI and an approximately 200 bp fragment was isolated. This GAP promoter fragment was ligated to the ADH-2 fragment present on the linear vector described above to give plasmid pJS103.</p>
    <p>S. cerevisiae AB110 was transformed with the ADH-2/GAPDH constructions, and the cultures grown initially in synthetic complete media lacking leucine with 8% glucose. Env-3 was induced by diluting the culture 1:25 into YEP with 1% glucose and allowing growth at 30 C. for 24 hours. Normal cell growth was observed; however, complex expression products were observed. In an immunoblot assay with AIDS sera, the protein appeared as a multiplet of immunoreactive bands, including a major band at about 37 kD and five additional bands of increasing molecular weight. As demonstrated by treatment with endoglycosidase-H prior to gel and immunoblot analysis, these additional bands were due to glycosylation. Inspection of the gp41DNA sequence shows five potential N-linked carbohydrate addition sites. Since env-3 encodes a polypeptide with a calculated molecular weight of 40.5 kD, the gel mobility of env-3 at around 37 kD may indicate either aberrant electrophoresis properties, or C-terminal processing analogous to that proposed for infected T-cell-derived gp41.</p>
    <p>4.2.4. Env-4</p>
    <p>The env-1 region approximates the carboxy-terminal half of the gp120 glycoprotein from the envelope gene and corresponds to amino acids Glu-272 to Arg-509 (FIG. 5). Env-4 also contains a methionine at the N-terminus which serves as an initiation codon for the least expression system.</p>
    <p>Expression of the env-1 protein was initially attempted as a direct expression product in least which failed to provide any detectable product. Successful expression of the env-4 protein was achieved as an SOD fusion product in yeast.</p>
    <p>4.2.4.1. pBS24/SF2env4/GAP</p>
    <p>The 3' end of the env-4 coding sequence was modified by M13 mutagenesis to generate 2 stop codons in frame after Arg-509, and by adding restriction sites for HindIII and SalI. Plasmid pSV-7c/env (Section 2.1.2) was digested with HindlII and XhoI and an approximate 2830 bp fragment was gel isolated. The fragment was cloned into M13-mp19 and single stranded template was generated. M13 mutagenesis was performed using the following primer:</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="equation">5'-GAACATAGCTGTCGACAAGCTTCATCATCTTTTTTCT-3'.</pre>
    
    <p>A single plaque designated M13Fenv3-447 was isolated and confirmed by M13 sequencing to contain the inserted stop codons and new restriction sites for HindlII and SalI. M13 replicative-form DNA was prepared for M13Fenv3-447 by standard methods, and an approximately 713 bp BglII (position 6604, FIG. 5) to SalI fragment was excised and gel purified. This fragment was ligated to the following NcoI-BglIl linker, which codes for a methionine initiation codon and the first four amino acids of the env-4 protein: ##STR14## and then cloned into pPGAP1  EPO Pub. No. 164,556! which was previously digested with NcoI and SalI and gel isolated. The approximate 1130 bp BamHI-SalI fragment containing the GAPDH promoter and the env-4 gene was excised and cloned into pBS24 (below), which was previously digested with BamHI and SalI and gel isolated, to give plasmid pBS24/SF2env4/GAP.</p>
    <p>Plasmid pBS24/SF2env4/GAP was transformed into S. cerevisiae strains 2150-2-3 and AB110 as described previously. Cultures from single colonies were grown and analyzed for expression by SDS polyacrylamide gel electrophoresis. No env-4 protein was detected by either Coomassie stained gels or western blot analysis.</p>
    <p>4.2.4.2. pBS24</p>
    <p>Plasmid pBS24 is a derivative of pAB24 as described in Section 4.2.2.1. Plasmid pAB24 was digested with BamHI and SalI (which cut within the tetracycline gene of the pBR322 sequences) and gel purified. The vector was then ligated with a synthetic adapter of the following sequence which created new unique BqlII and BamHI sites: ##STR15## The resulting vector, pAB24BL was then digested with BamHI and BglII and gel purified. The linearized vector was ligated with the BamHI cassette excised and purified from pSOD/env-5b (Section 4.2.5) to give pBS24. The cassette contains the hybrid ADH-2/GAPDH promoter and -factor terminator with an NcoI-SalI insert of the SOD/env-5b fusion gene. The cassette is oriented in pBS24 such that the direction of transcription from the ADH-2/GAPDH promoter is in the opposite direction to that of the inactivated tetracycline gene of the pBR322 sequences.</p>
    <heading>4.2.4.3. pBS24/SOD-SF2 env4</heading> <p>Since there was no detectable expression of env-4 from the direct expression system, an SOD/env-4 fusion gene was constructed. Plasmid pBS24/SF2env4/GAP was digested with NcoI and SalI and the approximate 713 bp env-4 gene was gel isolated. The env-1 gene was ligated into pSODCF-2 (Section 3.5) which was previously digested with NcoI and SalI and gel purified, to give pCF2-SODenv4. Plasmid pSODCF2 is a bacterial expression vector which allows for the C-terminal fusions of heterologous genes with the human SOD gene and is under the control of the tacl promoter. When plasmid pCF2-SODenv4 was transformed into E. coli strains D1210 and RR1M15 and analyzed for expression as described previously (Sections 3.4.4 and 3.6), no expression of SOD/env-4 fusion protein was detected. Plasmid pCF2-SOD-env4 was digested with StuI and SalI to isolate the 3' half of the SOD gene fused with the env-4' gene. This fragment was ligated to pSI8 which had been previously digested with StuI and SalI and gel isolated to give plasmid pS18-SODenv4. Plasmid pSI8 (Section 4.2.5) contains the hybrid ADH-2/GAPDH promoter and the -factor terminator sequences flanking the SOD-insulin fusion gene. The resulting plasmid pSI8-SOD/env-4 was digested with BamHI and SalI to excise a fragment containing the ADH-2/GAPDH promoter and SOD/env-4 fusion gene. This fragment was gel isolated and cloned into pBS24 which had been previously digested with BamHI and SalI and gel isolated to give pBS24/SOD-SF2env4 (FIG. 26).</p>
    <heading>4.2.4.4. Transformation and Expression</heading> <p>The plasmid pBS24/SOD-SF2env4 was transformed into yeast cells S. cerevisiae 2150-2-3 and S. cerevisiae AB116 (mat a, leu 2, trp 1, ura 3-58, pro 1-1122 (prot. B), pep 4-3 (prot. A), pre 1-407 (prot. C), cir) as described previously  Hinnen et al. (1978) Proc. Natl. Acad. Sci. USA 75: 1929-1933! and plated onto Leu<sup>-</sup> sorbitol plates. Strain AB116 was isolated by curing S. cerevisiae strain BJ2168 of its 2 micron plasmid by standard methods. BJ2168 is available from the Yeast Genetic Stock Center, Department of Biophysics and Medical Physics, University of California, Berkeley 94720.</p>
    <p>Cultures were grown as follows: a loopfull of cells from the individual colonies were inoculated into 3 ml of Leu<sup>-</sup>, 8% glucose media and incubated in an air shaker at 30 C. for 16-18 h. The 3 ml culture was transferred to 40 ml of fresh Leu<sup>-</sup>, 8% glucose media and incubated in an air shaker at 30 for 24 h. The 40 ml culture was transferred to 1 liter of YEP, 1% glucose media (or YEP, 2.5% Ethanol media for 2150-2-3 cells) and incubated in an air shaker at 30 for an additional 48 h. The cells were harvested by centrifugation and stored at -20 C.</p>
    <p>Expression of the SOD/env-4 fusion protein was analyzed by both polyacrylamide gel electrophoresis and western blot. Cells are disrupted by glass bead lysis method in 0.1M NaPo<sub>4</sub> (pH 7.4), 0.1% Triton lysis buffer. After lysis the soluble and insoluble fractions are separated by centrifugation. The insoluble pellet is solubilized in Laemmli gel loading buffer and run on a 12.5% acrylamide SDS gel. Laemmli (1970) Nature 27: 680. A band migrating at approximately 44 Kd molecular weight (expected size of SOD/env-4 fusion protein) was observed. This same band also reacted on a western blot with AIDS positive human sera but not with control sera. Expression levels of SOD/env-4 fusion protein were approximately equal for the two different strains. Protein purification was carried out essentially as described in Section 4.2.2.3. for env-2.</p>
    <heading>4.2.5. ySOD/env-5b fusion protein</heading> <p>The region of the envelope gene corresponding to amino acids alanine-557 through tryptophan-677 is termed env-5b as described in Section 3.6 and is also expressed as a stable fusion protein with hSOD in yeast. A StuI-SalI fragment containing most of the SOD gene fused to the env-5b gene was removed from pSOD/env5b (Section 3.6) and cloned into pSI8 which had been previously digested with StuI and SalI and gel isolated to give pSIa/SOD-env5b. Plasmid pSI8 (described below) contains the ADH-2/GAPDH hybrid promoter and -factor terminator sequences flanking the SOD-insulin fusion gene. The resulting plasmid, designated pSIa/SOD-env5b, was digested with BamHI and the fragment containing the promoter-ySODenv-5b fusion-terminator was isolated and cloned into the BamHI cut and phosphatased pAB24 to give pYSOD/env-5b, which was used to transform the yeast strain 2150, as described above. Expression of the ySOD/env-5b fusion protein was induced by diluting a starter culture into YEP containing 1% ethanol.</p>
    <p>Lysates were isolated and prepared as describe above. A heavily expressed protein corresponding to the SOD/env-5 fusion was readily discerned in the insoluble fraction by Coomassie blue staining. This protein migrated at a molecular weight of approximately 30.6 kD, as predicted from its DNA sequence. This fusion protein was also subsequently shown to have a high proportion of reactivity to AIDS patients' sera.</p>
    <p>Plasmid pSI8 is a derivative pYASI1, the latter being described in commonly owned U.S. patent application Ser. No. 845,737, filed on 28 Mar. 1986 by Cousens et al. and EPO Pub. No. 196,056, the disclosures of which are expressly incorporated herein by reference. Essentially, pSI8 contains: The hybrid ADH-2/GAPDH promoter (as a 1.3 kb Bam-Nco fragment) derived from plasmid pJS104 (described below); an SOD-insulin fusion gene (as a 736 bp Nco-Sal fragment) derived from a derivative of pYSI1 (U.S. Ser. No. 845,737); and the -factor terminator isolated as a 277 bp SalI-EcoRI fragment in which the EcoRI site has been filled in with Klenow fragment and BamHI linkers ligated to give a SalI-BamHI fragment  Brake et al. (1984) Proc. Natl. Acad. Sci. USA 81: 4642-4646!; all cloned into a pBR322 derivative in which the segment between EcoRI and SalI is deleted and Bam linkers attached. Plasmid pJS104 is the same as plasmid pJS103 (Section 4.2.3.2), except that the GAPDH fragment of the ADH-2/GAPDH promoter is about 400 bp, as opposed to 200 bp. The construction of pJS104 was the same as pJS103, except that during the preparation of the GAPDH portion of the promoter, the 0.4 Kbp BamdI-EcoRI fragment was partially digested with AluI to create a blunt-end near the BamHI site.</p>
    <heading>4.2.6. Env 4-5</heading> <p>The env 4-5 polypeptide corresponds to the region of the envelope gene which approximates the C-terminus of gp120 and the N-terminus of gp41, amino acids 272 to 673 (FIG. 5). Plasmid pBS24.1/SOD-SF2env4-5 was used to express the env 4-5 polypeptide as an SOD fusion protein under the regulatory control of the ADH-2/GAPDH promoter in yeast.</p>
    <p>4.2.6.1. Construction of pBS24.1/SOD-SF2env4-5</p>
    <p>To construct plasmid pBS24.1/SOD-SF2env4-5 a 1.2 kbp BglII-SalI fragment which corresponds to nucleotides 6603-7795 (see FIG. 3) Was isolated from pSV7dARV160T-tpa and ligated, into pBS24/SOD-SF2env4 (see Section 4.2.4.3.) which had previously been digested with BglII and SalI. Plasmid pSV7dARV160T-tpa is a plasmid which contains the HIV envelope gene to which two stop codons, a. HindIII, and SalI site had been introduced at position 7798 by M13 mutagenesis with the following mutagenic primer:</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="equation">5'-CTTTATATACGTCGACAAGCTTCATCAGCTAAACCAA-3'</pre>
    
    <heading>4.2.6.2. Transformation and expression</heading> <p>The plasmid pBS24.1/SOD-SF2env 4-5 was transformed into yeast cells S. cerevisiae AB116  mat a, leu 2, trp 1, ura 3-58, pro 1-1122 (prot. B), pep 4-3 (prot. A), pre 1-407 (prot. c), cir! as described previously  Hinnen, et al. (1978) Proc. Natl. Acad. Sci. USA 75: 1929-1933! and plated on Leu<sup>-</sup> sorbitol plates. The cultures were grown and expression analyzed as previously described in section 4.2.4.4. Expression levels of the SOD/env4-5 fusion protein were approximately 15-20% of the insoluble protein fraction as estimated by Coomassie-blue staining Of PAGE-treated samples.</p>
    <p>4.2.6.3. Protein purification</p>
    <p>Cultures of AB116 (pBS24.1/SOD-SF2env4-5) were grown in 3 ml Leu.sup.(-), 8% glucose medium overnight at 30 C. The 3 ml was inoculated into 50 ml Leu<sup>-</sup>, 8% glucose medium for 24 h at 30 C. The 50 ml was then inoculated into 1 liter of YEP, 1% glucose and grown 30-48 h for induction of protein expression. The cells expressing SOD/env 4-5 were harvested and further processed as described for the purification of env-2 (Section 4.2.2.3.).</p>
    <p>4.3. p31pol</p>
    <p>4.3.1. GAP/ADH2 promoter</p>
    <p>The ARV248NL fragment (Section 3.4.2.3) was cloned into pBS100 previously cut with NcoI and SalI. pBS100 (below) is a bacterial vector derived from pAB12 with a BamHI cassette consisting of the GAP-ADH2 promoter (i.e., the ADH-2/GAPDH promoter), an ARV-env gene as an NcoI-SalI fragment, and the GAP terminator. The BamHI cassette from a positive clone of pBS100/p31/GAP-ADH2 was cloned into pAB24 (Section 4.2.2.1), a yeast vector with both ura and leu selection capabilities. Both orientations of the cassette in this vector were screened for and used to transform the yeast strain AB110 (Mat a, Ura 3-52, leu 2-04, or both leu 2-3 and leu 2-112, pep 4-3, his 4-580, cir). These cells were plated in both ura<sup>-</sup>  and leu<sup>-</sup> plates. Also, ura<sup>-</sup>  cells were plated onto leu<sup>-</sup> plates.</p>
    <p>Three different induction procedures were done: (1). Ura<sup>-</sup> colonies patched on ura<sup>-</sup> plates were induced for 24 h in YEP/1% glucose. Both a Western and a polyacrylamide gel were run on these samples. Both results were negative. (2). Colonies from ura<sup>-</sup>  plates patched on leu<sup>-</sup> plates were induced in either leu<sup>-</sup> /3% ethanol or YEP/1% glucose for 24 h. A Western and a polyacrylamide gel were run on these samples and the results were also negative. (3). Colonies from leu<sup>-</sup> plates patched on leu<sup>-</sup> plates were induced in either leu<sup>-</sup> /3% ethanol or YEP/1% glucose for 24 h. The polyacrylamide gel showed a negative result. No Western was run on these samples.</p>
    <p>Plasmid pBS100 is a yeast expression cassette vector cloned into a pBR322 derivative, pAB12. The expression cassette contains the hybrid ADH-2/GAPDH promoter and the GAPDH terminator flanking a gene segment from the envelope gene. The ADH-2/GAPDH promoter is a 1200 bp BamHI-NcoI fragment isolated from pJS103 (Section 4.2.3.2) and the GAPDH terminator is a 900 bp SalI-BamHI fragment isolated from plasmid pPGAP1 (Section 4.2.1.1). Plasmid pBS100 also contains a non-essential fragment between the NcoI and SalI sites which is replaced by gene fragments of interest. The expression cassette can be removed from pBS100 by digestion with BamHI and cloned into yeast shuttle vectors for introduction into yeast cells.</p>
    <p>Plasmid pAB12 is a pBR322 derivative lacking the region between the single HindIII and SalI sites and containing a BamHI linker inserted between the unique EcoRI site. This vector was constructed by digesting pBR322 to completion with HindIII and SalI, followed by limited digestion with Bal31 nuclease, repair of the ends so created with the Klenow fragment of E. coli DNA polymerase I, end blunt-end ligation with T4 DNA ligase to reform closed covalent circles. The plasmid was then opened up with EcoRI, treated with the Klenow fragment of E. coli DNA polymerase I (to fill-in the 5' overhangs), blunt-end ligated with BamHI linkers, digested with BamHI to remove excess linkers, end then ligated to form closed circles.</p>
    <heading>4.3.2. GAP promoter</heading> <p>The pBS100/p31/GAP-ADH2 plasmid was cut with BamHI and NcoI and the fragment containing the p31 gene (NcoI-SalI) and the GAP terminator (SalI-BamHI) was gel purified. pCl/1-alpha 1 antitrypsin/GAP was also cut with NcoI and SalI and the fragment including the GAP promoter (NcoI-BamHI) and a portion of pCl/1 (BamHI-BalI) was gel isolated as well. Both fragments were ligated with the yeast vector pCl/1 previously cut with BamHI and SalI. The BamHI cassette can only be cloned in a single orientation in this case. The resulting DNA was used to transform yeast strains AB110 and PO17 (Mat a, leu 2-04, cir) and the cells were plated on leu<sup>-</sup> plates. The transformation using strain PO17 gave no transformants.</p>
    <p>Colonies from leu<sup>-</sup> plates were grown in 3 ml of leu<sup>-</sup> /2% glucose for 24 h. Yeast was analyzed on polyacrylamide gels stained by Coomassie Blue with negative results. No Western was run on these samples.</p>
    <p>4.3.3. SOD-p31 fusion protein</p>
    <p>The construction of the p31pol expression vector is shown schematically in FIG. 23. The DNA and amino acid sequences of the SOD/p31 insert are shown in FIG. 24.</p>
    <p>4.3.3.1. DC1/1-DSP31-GAP-ADH2</p>
    <p>For the construction of a gene for a fused protein SOD-p31 to be expressed in yeast, plasmid pSI4/392 was used. This plasmid contains the SOD gene fused to the proinsulin gene under the regulation of the ADH-2/GAP promoter. The proinsulin gene is located between EcoRI and SalI restriction sites. To substitute the proinsulin gene with the ARV248NL (Section 3.4.2.3) fragment, two oligomers designated ARV-300 and ARV-301, respectively, were synthesized using phosphoramidite chemistry. The sequences generate cohesive ends for EcoRI and NcoI on each side of the molecule when the two oligomers are annealed. ARV-300 and ARV-301 have the sequences: ##STR16##</p>
    <p>Two micrograms of pSI4/39-2 linearized with EcoRI were ligated to 100 picomoles each of phosphorylated ARV-300 and dephosphorylated ARV-301 in the presence of ATP and T4 DNA ligase in a final volume of 35 l. The reaction was carried out at 14 C. for 18 h. The DNA was further digested with SalI and the fragments were resolved on a 1% low melting point agarose gel and a fragment containing the vector plus the SOD gene (-6.5 kb) was purified as described above and resuspended in 50 l of TE. Five l of this preparation were ligated to 5 l of ARV248NL in 20 l final volume for 18 h at 14 C. and 5 l used to transform competent HB101 cells. The resultant plasmid was called pSP31. Twenty g of this plasmid were digested with BamHI and a fragment of about 2900 bp was isolated by gel electrophoresis, resuspended in TE and ligated to pCl/1 previously cut with BamHI. This DNA (the pCl/1-pSP31-GAP-ADH2 derivative) was used to transform HB101 and transformants with the BamHI cassette were obtained.</p>
    <p>4.3.3.2. Transformation and expression</p>
    <p>Yeast strains 2150, P017, and AB110 were transformed with the pCl/1-pSP31-GAP-ADH2 derivative, both short and long orientations. The strain 2150 gave no transformants. All other transformants were patched on leu<sup>-</sup> plates.</p>
    <p>Yeast strain PO17 (Mat a, leu 2-04, cir) was obtained by isolating a spontaneous revertant strain 21502-3 (Mat a, ade 1, leu 2-04, cir). To isolate the revertant P017, 2150-2-3 yeast cells were grown in YEPD, washed in medium without adenine and about 610<sup>8</sup> cells were plated onto six adenine minus (ade<sup>-</sup>) plates. Four candidate revertants were tested for other genetic markers by steaking on plates without uracyl (ura<sup>-</sup>), plates with no leucine (leu<sup>-</sup>), end minimal plates plus leucine. Growth was observed on ura<sup>-</sup>  and minimal plus leu plates; no growth was observed on leu<sup>-</sup> plates. Revertants were crossed with strain AB103.1 (Mat a, pep 4-3, leu 2-3, leu 2-112, ura 3.52, his 4-580) to determine if the reversion was due to extragenic suppression. Based on tetrad analysis, none of the four independent ade<sup>+</sup> revertants were due to extragenic suppression. Based on good growth and high spore viability, one of the revertants was selected and named PO17.</p>
    <p>Three different kinds of inductions were tried: (1). PO17 colonies were induced in either a 10 ml culture of YEP/1% glucose or a leu<sup>-</sup> /3% ethanol culture for 24 h. The yeast pellets were analyzed by both polyacrylamide gels and Westerns and even though the Coomassie-stained gel showed a negative result, the Western did light up a band of the correct molecular weight with both induction methods. (2). PO17 colonies were induced in a 30 ml culture of YEP/1% ethanol for 48 h. Aliguots were analyzed by PAGE at various time points during the induction. The Coomassie-stained gel shows a band in the correct molecular weight range (47-50 kd) that appears after 14 h in YEP/1% ethanol and reaches a maximum intensity at 24 h of induction. The Western result correlates well with the Coomassie-stained gel, showing strong bands at 24 and 18 h. (3). AB110 colonies were induced in either leu<sup>-</sup> /3% ethanol or YEP/1% glucose for 24 h. PAGE and Westerns were run and the results were negative for the PAGE and positive for the. Western, in both induction methods.</p>
    <p>Expression and immunoreactivity were characterized as described below.</p>
    <p>Cells from one patch PO17 (pCl/1-pSP31-GAP-ADH2) were inoculated in 50 ml of leu<sup>-</sup> /7.1% glucose and grown overnight at 30 C. The saturated culture was inoculated into 500 ml of the same leu<sup>-</sup> /7.1% glucose medium and incubated overnight at 30 C. The saturated 500 ml culture was used to inoculate a 10 L fermenter with YEPD. Cells were then harvested abut five days later.</p>
    <p>A sample of cells from the fermenter were analyzed by PAGE electrophoresis to determine expression levels of p31-SOD. In addition, Western analysis using a Trimar serum was performed on the samples to determine its immunoreactivity.</p>
    <p>The following procedure was used to prepare samples for PAGE and Westerns:</p>
    <p>a. Cells (4 g) were resuspended in 7 ml of lysis buffer (50 mM Tris-HCl, pH 8.0, 1 mM EDTA, 150 mM NaCl, 1 mM PMSF) in a centrifuge tube and 4 ml of yeast-size glass beads were added.</p>
    <p>b. Cells were vortexed at top speed in a VWR vortex for 10 minutes (1 min. on ice, 1 min. vortexing).</p>
    <p>c. The cell lysate was centrifuged at 18,000 rpm (39,000 x g) for 10 minutes in a JA 20 rotor. Both insoluble (pellet) and soluble (supernatant) fractions were further analyzed.</p>
    <p>d. The supernatant was diluted in sample buffer (1/4 dilution), boiled 10 minutes and cooled to room temperature.</p>
    <p>e. The pellet obtained in "c" was boiled during 10 minutes in sample buffer (5 ml) and cooled to room temperature. The mixture was centrifuged at 18,000 rmp (39,000 x g) for 15 minutes in a JA 20 rotor. The supernatant was recovered and was diluted 1/4 in sample buffer.</p>
    <p>f. Samples from the supernatant (obtained in "d") and solubilized pellet (obtained in "e") were loaded on a 12% PAGE. A band corresponding to SOD-p31 (between MW 68 and 43 Kd) was present in the insoluble fraction (samples of solubilized pellet).</p>
    <p>Immunoreactivity of SOD-p31 band was tested by Western Analysis using serum Trimark0036 from an AIDS patient. For this purpose, proteins were fractionated on a 12% PAGE as previously described and transferred to nitrocellulose filter paper. The filter paper was then treated with serum 0036, followed by a second goat anti-human antibody conjugated with horseradish peroxidase (HRP). Color was developed using an HRP substrate. Bands Corresponding to p31-SOD were present in samples corresponding to the insoluble (Pellet) fractions. Bands of smaller size, present in both soluble and insoluble fractions, also react with the sera and most probably correspond to degradation products of SOD-p31.</p>
    <p>4.3.3.3. Purification and characterization</p>
    <p>Frozen bacteria (Section 3.5) or yeast cells expressing the p31-SOD fusion protein are thawed at room temperature and suspended in 1.5 volumes of lysis buffer (20 mM Tris-Cl, pH 8.0, 2 mM EDTA, 1 mM PMSF, for bacteria; 50 mM Tris-Cl, pH 8.0, 2 mM EDTA, 1 mM PMSP for yeast), and mixed with 1 volume of acid-washed glass beads.</p>
    <p>Cells are broken for 15 min in a non-continuous mode using the glass chamber of a Dynomill unit at 3,000 rpm, connected to a -20 C. cooling unit. Glass beads are decanted for 2-3 min on ice, the cell lysate is removed. The decanted glass beads are washed twice with 30 ml of lysis buffer at 4 C. The cell lysate is centrifuged at 39,000 x g for 30 min.</p>
    <p>The pellet obtained from the above centrifugation is washed once with lysis buffer, after vortexing and suspending it at 4 C. (same centrifugation as above). The washed pellet is treated with 0.2% SDS (for bacteria) or 0.1% SDS (for yeast) in lysis buffer and agitated by rocking at 4 C. for 10 min. The lysate is centrifuged at 39,000 x g for 30 min. The pellet is boiled in sample buffer (67.5 mM Tris-Cl, pH 7.0, 5% B-mercaptoethanol, 2.3% SDS) for 10 min and centrifuged for 10 min at 39,000 x g. The supernatant is recovered and further centrifuged at 100,000 x g for 60 min (60 Ti rotor). This step is replaced by a 0.45 m filtration when yeast is used. The supernatant from the above centrifugation is loaded (maximum 50 mg of protein) on a gel filtration column (2.590 cm, ACA 34 LKB) with a flow rate of 0.3-0.4 ml/min, equilibrated with phosphate-buffered saline (PBS), 0.1% SDS. The fractions containing SOD-p31 are pooled and concentrated either by vacuum dialysis or using a YM5 Amicon membrane at 40 psi. The protein is stored at -20 C. as concentrated solution.</p>
    <p>Gel electrophoresis analysis shows that the SOD-p31 protein migrates having a molecular weight of about 46 kd and is over 90% pure.</p>
    <p>4.4. Reverse transcriptase (RT)</p>
    <p>The AIDS retrovital reverse transcriptase (RT) is an RNA-dependent DNA polymerase found in virions in low quantities. RT is encoded within a domain of the vital pol gene. The mature enzyme is derived by proteolytic processing from a large polypeptide precursor whose cleavage is thought to be mediated by a vital protease.</p>
    <p>The amino terminal sequence of HIV RT has been reported  Veronese et al. (1986) Science 231: 1289-1291! and corresponds to proline-156 of the polymerase gene (nucleotide 2403, FIG. 5). The carboxy terminal end of the RT gene is estimated to be at valine-691 (nucleotide 3708).</p>
    <heading>4.4.1. pAB24/RT4 expression vector</heading> <p>A 6.1 kb EcoRI fragment of cloned proviral DNA was cloned into pUCl9 at the EcoRI site and designated pUC-ARVSA. Plasmid pUC-ARV8A was digested with BalI and KpnI which liberates a 1535 bp fragment containing coding sequence for amino acids proline-180 through tryptophane-690. This fragment was extended, using synthetic oligonucleotides, to include proline-164 at the N-terminus and alanine-693 at the C-terminus. The synthetic DNA also provides a methionine initiation codon.</p>
    <p>The RT-encoding fragment was modified by the addition of synthetic oligonucleotide adapters. The 5' synthetic. DNA has the following sequence: ##STR17## The 3' adaptor was a KpnI/SalI adaptor with an in-frame stop codon having the following sequence: ##STR18## The 5' adaptor contains a HindIII site within the initiation codon so that digestion with HindIII and SalI can facilitate subsequent cloning of the RT sequence into additional expression vectors.</p>
    <p>The synthetic linkers and the cloned DNA were ligated by standard techniques into the vector pPGAP1 (Section 4.2.1.1) which had been previously digested with NcoI-SalI and gel isolated. The resulting expression cassette containing the GAPDH promoter, the RT gene, and GAPDH terminator was excised with BamHI and cloned into pAB24 which had been previously digested with pamHI and treated with alkaline phosphatase. The resulting expression plasmid was designated pAB24/RT4 and is shown in FIG. 20.</p>
    <p>4.4.2. Transformation and expression</p>
    <p>Plasmid pAB24-RT4 was used to transform yeast strain AB110, and leucine prototrophs grown in leucin-deficient media to 3 ml, followed by growth in YEPD to the 1 l level. Travis et al. (1985) g. Biol. Chem. 260: 4384-4389.</p>
    <heading>4.4.3. Purification</heading> <p>Cells from a 11 culture were pelleted by centrifugation at 2,500 rpm for 10 min. The cell pellet was resuspended in 300 ml of 50 mM Tris-HCl, pH 7.5, 14 mM -mercaptoethanol, 1.2M sorbitol, and 200 g/ml Zymolyase. Spheroplast formation, monitored by light microscopy, was allowed to proceed for 90 min at 30 C. After a low speed centrifugation, the pelleted spheroplasts were lysed in 40 ml of a buffer containing 50 mM Tris-HCl, pH 7.5, 0.1% Triton X-100, and 1 mM DTT at room temperature. The yeast lysate was clarified by centrifugation at 20,000 rpm for 2 hours and the supernatant was fractionated by stepwise NH<sub>4</sub> SO<sub>4</sub> precipitation. Greater than 90% of RT activity was in the 0-30% NH<sub>4</sub> SO<sub>4</sub> insoluble fraction. This NH<sub>4</sub> SO<sub>4</sub> pellet was resuspended is 20 ml of reverse transcriptase buffer (RTB; 50 mM Tris-HCl, pH 7.5, 2 mM B-mercaptoethanol, 8.2 mM EDTA, 0.1% Triton X-100, 20% vol/vol glycerol) containing 50 mM KCl. An Amicon pressure filtration device was used for desalting. The extract was then applied to a cellulose phosphate column (Sigma C-2383) (2.5 cm30 cm) preeguilibrated in RTB containing 50 mM KCl. The column was washed with 100 ml of the same buffer. A linear gradient of 50 to 800 mM KCl in RTB was used for elution. Individual fractions were monitored for RT activity. The peak fractions of RT activity were between 150 and 225 mM KCl. This material (about 15 ml total) was pooled, desalted by Amicon filtration using RTB containing 50 mM KCl, and then applied to a single-stranded DNA cellulose column (Sigma D-8273) (1.0 cm10 cm) preequilibrated in RTB at 50 mM KCl. The column was washed with 30 ml of this buffer and eluted with a linear gradient of RTB from 50 to 800 mM KCl. Fractions were monitored for RT activity and peak fractions were pooled.</p>
    <p>4.4.4. Electrophoresis and immunoblotting</p>
    <p>Recombinant RT was analyzed by polyacrylamide gel electrophoresis and immunoblotting techniques using AIDS patients' sera. The recombinant protein gave an apparent gel mobility of approximately 66 kD, indicating an extremely close approximation to the native p66 species, and in good agreement with the calculated molecular weight of 62.5 kD. It was also noted that during purification, processing of this 66 KD protein occurred giving a second major species with an estimated molecular weight of 511.5 KD. This processing is presumably due to a yeast protease since the region thought to encode ARV protease was not included in this expression construction. Previously, HIV gag-pol fusions expressed in yeast were shown to be processed when this protease region was included. Kramer et al. (1986) Science 231: 1580-1584. That the p66 and p51 species produced in yeast had identical N-termini (Pro.Ile,Ser.Pro.Ile, etc.) was confirmed by gas phase sequence analysis of the purified proteins (15 cycles; as a mixture, Panel A, lane 3). Thus, yeast may mimic the natural maturation processes for HIV RTs, giving rise to both p66 and p51. The sequence analysis also showed that the N-termini methionine derived from the synthetic initiation codon was removed in vivo. Interestingly, the processing observed in yeast indicate that HIV protease is not necessarily required for this processing event in vivo. Preliminary immunoblot analysis of recombinant RT also indicates a high degree of reactivity with AIDS sera. Of 20 sera tested, 19 scored positive.</p>
    <heading>4.4.5. RT activity assay</heading> <p>Analysis of recombinant RT activity by enzymatic assay was performed on crude yeast lysates and the purified enzyme. Cruse lysate was prepared as follows. Cells from 25 ml culture were pelleted by centrifugation at 2,500 rpm for 10 min. The cell pellet was resuspended in 7.5 ml of 50 mM Tris-HCl, pH 7.5, 14 mM B-mercaptoethanol, 1.2M sorbitol, and 200 g/ml Zymolyase. Spheroplast formation, monitored by light microscopy, was allowed to proceed for 90 min at 30 C. After a low speed centrifugation, the pelleted spheroplasts were lysed in 1 ml of a buffer containing 50 mM Tris-HCl, pH 7.5, 0.1% Triton X-100, and 1 mM DTT at room temperature. The yeast lysate was clarified by centrifugation.</p>
    <p>Using the assay conditions described for RT isolated from virions  Veronese (1986) Science 31: 1289-1292!, the relative activity of yeast-derived RT was assayed using various primer template combinations. Thus, relative to (dT)-<sub>5</sub> (dA)<sub>n</sub> (100%), the enzyme activity with (dT)-<sub>5</sub> (dA)<sub>n</sub> was 4.3%, with (dG)-<sub>15</sub> (rC)<sub>n</sub> was 71.1%, and with (dG)-<sub>15</sub> (rC<sup>m</sup>)<sub>n</sub> was 2.4%. Yeast extracts form cells containing control plasmids gave background levels of incorporation, excluding the possibility of host-encoded RT activity. In all cases, the enzyme reactions were linear for greater than 90 minutes. The results are shown in the following table:</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">______________________________________  Time cpm Incorporated______________________________________  10'   860  20'  2110  30'  2720  40'  4046  50'  3834  60'  5360  90   8284  120' 10626  160' 11814  180' 11750______________________________________</pre>
    
    <p>4.5 p25gag</p>
    <p>4.5.1. Host-vector system</p>
    <p>Protein p25gag is synthesized by Saccharomyces cerevisiae AB110 transformed with plasmid pCl/1-p25-ADH-GAP. Plasmid pCl/1-p25-ADH-GAP is a yeast expression vector which contains the sequence coding for p25gag  Sanchez-Pescador et al. (1985) Science 227: 484! as well as pBR322 sequences including the ampicillin-resistant (amp<sup>R</sup>) gene and 2 micron (2) sequences  Broach, in Molecular Biology of the Yeast Saccharomyces, Vol. 1, p. 445 (Cold Spring Harbor Press, 1981)!, including the yeast leucine (leu) 2-04 gene.</p>
    <p>Expression of p25 is non-constitutive and it is under regulation of a hybrid ADH-2/GAPDH promoter derived from promoter sequences of the yeast alcohol dehydrogenase gene (ADH2)  Beier et al. (1982) Nature 300: 724-728! and glyceraldehyde-3-phosphate dehydrogenase (GAPDH)  EPO Pub. No. 120,551! and of the GAPDH terminator. Induction of p25 expression is achieved by low concentration of glucose in the growth medium. Yeast strain S. cerevisiae AB110 has the following genotype: Mat , ura3-52, leu2-04, or both leu2-3 and leu2-112, pep4-3, his4-580, cir. This strain was obtained as described below.</p>
    <p>4.5.2. Saccharomyces cerevisiae AB110</p>
    <p>Yeast strain S. cerevisiae 2150-2-3 (available from Lee Hartwell, University of Washington) was crossed with yeast S. cerevisiae strain AB103.1 transformant containing pCl/1 derivative. The diploids were sporulated and the tetrads dissected. Strains were maintained on leucine selective plates in order to ensure maintenance of the plasmid, since the patents are auxotrophs. A series of colonies were screened for their genotype with respect to a number of markers (Mat , ura3, leu2, pep4-3).</p>
    <p>The strain AB110 has the following genotype: Mat., ura3-52, leu2-04 or both leu2-3 and leu2-112, pep4-3, his4-580, cir, and is obtained by curing the above strain AB110 (pCl/1 derivative) of its resident plasmid by growth in the presence of leucine (absence of selective pressure) and then selection for leu<sup>-</sup> colonies by replica plating.</p>
    <heading>4.5.3. pCl/1-p25-ADH-GAP</heading> <p>Plasmid pCl/1-p25-ADH-GAP is a yeast expression vector which contains an "expression cassette" (see below) for p25gag cloned into the BamHI site of vector pCl/1. Vector pCl/1 was previously described (Section 4.1.2).</p>
    <p>An "expression cassette" for p25gag consists of the following sequences fused together in this order (5' to 3'): yeast hybrid ADH-2/GAPDH promoter, a gene for p25gag, and GAPDH terminator. FIG. 21 shows a schematic of the construction of pC1/1-p25TADH-GAP. The fragment shown at the top of the figure, containing p25gag, was constructed from ARV-2 DNA, in which the sequences coding for leu (position 7) to an SphI site at gly (position 90) (see FIG. 22 for amino acid numbers) were repeated by synthetic DNA. A PstI site (encircled in the figure) was inserted at leu (position 7) by replacing the natural CTA with a CTG codon. A natural PstI site at Ala79 was removed by making a silent third position change (dA to dC) in this codon. Two stop codons (indicated by an arrow) and a SalI site (circled) were placed adjacent to the C-terminal leu (position 232) codon by substituting Synthetic DNA from the ScrFI site at pro (position 225) to the SalI using synthetic. oligonucleotides: An additional methionine codon (indicated with an asterisk) was used as the initiation codon. The PstI-SalI fragment had been previously cloned in the construction of pGAG25-10 (see Section 3.1.2. and FIGS. 7 &amp; 8).</p>
    <p>To construct the yeast expression vector, a 682 bp PstI-SalI fragment containing the p25gag gene was isolated from plasmid pGAG25-10 (Section 3.1.2) and ligated with a synthetic NcoI-PstI linker, which has the following sequence: ##STR19## This was cloned into pBS100 (Section 4.3.2) which had been previously digested with NcoI and SalI and gel isolated. The BamHI expression cassette was excised from the resulting plasmid ,and ligated into BamHI digested and phosphatased treated pCl/1 to yield pCl/1-p25-ADH-GAP. The p25gag produced in yeast differs that produced in E. coli (Section 3.1.) by the presence Of the naturally occurring proline after the methionine at the N-terminus.</p>
    <p>FIG. 22 shows the nucleotide sequence of the p25gag insert cloned in pCl/1-p25-ADH-GAP and the amino acid sequence derived from it. DNA sequences that are not underlined in the p25 region-are derived directly from the ARV-2 provital DNA. Underlined sequences were chemically synthesized by the phosphoramidite method as originally described by Beaucage &amp; Caruthers, (1981) Tetrahedron Lett. 22: 1859.</p>
    <p>4.5.4. Transformation and expression</p>
    <p>Yeast cells were transformed following the procedure of Hinnen et al. (1978) Proc. Natl. Acad. Sci. USA 75: 1929. The transformation mix was plated onto selective leu<sup>-</sup>  agar plates. Plates were incubated at 30 C. for 2 to 4 days.</p>
    <p>Single transformant colonies were transferred into leu<sup>-</sup>, 8% glucose medium and grown at 30 C. until saturation. For induction of expression, a 1/25 dilution of the saturated culture into YEP/1% glucose was made and the cells were grown to saturation. Cells were harvested, lysed with glass beads and the insoluble material was collected by centrifugation. The pellet was resuspended in gel sample buffer and boiled. Extracts were fractionated on standard denaturing acrylamide gels. Laemmli (1970) Nature 227: 680. Proteins were visualized by staining with Coomassie blue.</p>
    <p>The extent of expression was initially determined by appearance of a new protein of the expected molecular weight in extract of transformants harboring pCl/1-p25-ADH-GAP as compared with control extracts (cells transformed with vector without p25gag insert). Immunoreactivity of the p25 protein was determined by standard western analysis  Towbin et al. (1979) Proc. Natl. Acad. Sci. USA 76: 4350! using human serum from an AIDS patient. The western analysis showed immune reaction of a protein of abut 25,000 daltons. No reaction was observed with non-immune serum.</p>
    <p>For preparation of a master seed stock, transformant cells were streaked onto a leu<sup>-</sup> plate and incubated at 30 C. for 2 days. Five single colonies were picked and individually inoculated into 5 ml of leu<sup>-</sup>, 8% glucose liquid medium and grown overnight at 30 C. A 1 ml aliquot was used to test expression of p25 and 15% glycerol was added to the rest of each culture. Glycerol cell stocks were aliquoted into 1 ml vials, labeled and quickly frozen in liquid nitrogen. Aliguots of the culture corresponding to the highest expressor were selected as the master seed stock and stored at -70 C. The seed stock was tested for absence of bacterial contamination.</p>
    <p>4.5.5. Protein Purification</p>
    <p>Cells from the patch which gives highest expression of p25 were used to inoculate 40-60 ml of leu<sup>-</sup>, 7.1% glucose medium. Cultures were grown to saturation at 25-35 C. with agitation. An aliquot (10-50 ml) of the first culture were added to 400-600 ml of leu<sup>-</sup> 7.1% glucose medium. Cultures were grown to saturation at 25-35 C. with agitation. An aliguot (100-500 ml) of the second inoculum was added to 8-12 L of YEP/1% glucose containing 1 ml of antifoam in a 16 L fermentor. Cultures were grown for 24 to 48 hours at 25-35 C. with agitation. Following fermentation, the approximately 8-12 L of yeast culture were centrifuged through a continuous flow centrifuge, and the cells were harvested.</p>
    <p>Frozen (or refrigerated) packed cells (50-150 g portions) were thawed and suspended in Lysis Buffer (1 mM PMSF, 2 mM EDTA, 150 mM HaCl, 50 mM Tris-HCl, pH 8.0; total volume of cells and Lysis Buffer m 280 ml), and 160-178 g of acid-washed glass beads were added. Cells were broken in a non-continuous system using a 300 ml glass unit of a Dyno-mill at 3,000500 rpm for 10-25 minutes. The exterior jacket temperature of the Dynamill was maintained at -15 to -20 C. by an ethylene glycol solution. Glass beads were allowed to settle by letting the mixture sit on ice. The cell lysate was decanted. Glass beads were washed with Lysis Buffer and the wash was added to the cell lysate. The lysate was centrifuged at 39,000 x g (30-70 minutes, depending on rotor) for 30 minutes. The pellet was discarded, and the soluble fraction was further centrifuged at 100,000 x g for 60 minutes at 2-8 C. The fatty layer (about 1/4 of total volume) was aspirated off. The supernatant was decanted, and the pellet was discarded.</p>
    <p>The soluble fraction obtained in the previous step was diluted ten fold by adding 9 volumes of 0.03M 5 Tris-HCl, 1 mM EDTA, pH 9.0. The pH of the diluted soluble fraction was adjusted to 9.00.5, then the material was chromatographed at 4-8 C. on a DEAE Sephacel column eguilibrated with 0.03M Tris-HCl, 1 mM EDTA, pH 9.0. Material was eluted using the same buffer; absorbance of the eluate was monitored at 280 nm, and 20-25 ml fractions were collected. Fractions were assayed by SDS-PAGE, and those containing p25 protein were pooled.</p>
    <p>The pooled fractions were concentrated to a protein concentration of 20-25 mg/ml by ultrafiltration in an Amicon unit under N<sub>2</sub> pressure. The concentrate was chromatographed on an ACA54 (LKB) column equilibrated with at least 1 column volume of 0.03M TrisHCl, 1 mM EDTA, pH 9.0. Material was eluted using the same buffer; absorbance of the eluate was monitored at 280 nm, and 10-30 ml fractions were collected. Fractions containing p25 protein as determined by SD-PAGE were pooled. The fraction pool was concentrated to approximately 1-2 mg/ml total protein by ultrafiltration in an Amicon unit under N<sub>2</sub> pressure. Protein concentration of the Bulk p25 was adjusted to 1.2-1.8 mg/ml (based on Lowry assay).</p>
    <heading>4.6. p53gag</heading> <p>The gag proteins from the HIV retrovirus are derived from a gag precursor polypeptide, designated p53gag. By combining HIV DNA derived from pUC-8A, (a subclone containing the insert from ARV-2 (8A) cloned into pUCl9) with synthetic oligonucleotides and cloning into a yeast expression vector the-complete p53gag precursor protein was synthesized by yeast.</p>
    <p>4.6.1. Construction of pC1/1-GAP-p53</p>
    <p>A 1424 bp SacI-BglII fragment was gel isolated from pUC-8A. This fragment corresponds to nucleotides 225 to 1650 as shown in FIG. 5. The SacI-BglII fragment was further digested with HgaI and then ligated with the following oligonucleotide linker: ##STR20## The ligated linker fragment was digested with NcoI and BglIl and then cloned into pPGAP-IGF1 which had previously been digested with Ncol-and BglII. The NcoI site of the linker functions to regenerate the N-terminal methionine of p53. Plasmid pPGAP-IGF1 is plasmid pPGAP1 as described in Section 4.2.1.1. with a 220 bp NcoI-SalI fragment encoding IGF-I. The IGF-I sequences only serve as a matter of convenience due to the presence of a BglII site 60 bp from the 3' end of the insert. The resulting plasmid, pPGAP-p53/IGF1, was digested with BqlII and SalI and ligated with a 201 bp synthetic oligonucleotide fragment which encodes the C-terminus of the p53 protein to give plasmid pPGAP-p53. The 1505 bp NcoI-SalI fragment contains the entire coding region of p53 from amino acid methionine-1 to glutamine-502. Plasmid pPGAP-p53 was digested with BamHI to isolate the expression cassette containing the GAPDH promoter -p53 gene -GAPDH terminator as a 2843 bp BamHI fragment which was then cloned into pCl/1 which had previously been digested with BamHI. The resulting plasmid was called pCl/1-GAP-p53.</p>
    <p>4.6.2. Transformation and expression</p>
    <p>S. cerevisiae strain JSC302 was constructed by transforming strain AB116 to G418 resistance with plasmid pDM15 (FIG. 14). Plasmid pDM15 consists of Kluyveromyces lactis ADH1 promoter and terminator sequences flanking the G418 gene, pBR322 sequences, and a GAP promoter-ADR1 expression cassette. This integrating plasmid was targeted to the ADR1 locus.</p>
    <p>Plasmid pC1/1-GAP-p53 was transformed into yeast strain S. cerevisiae JSC302 following the procedure of Hinnen et al. (1978) Proc. Natl. Acad. Sci. USA 75: 1929. The transformation mix was plated onto selective leu<sup>-</sup> agar plates. The plates were incubated at 30 C. for 2 to 4 days. Single transformant colonies were transferred into leu<sup>-</sup>, 8% glucose medium and grown at 80 C. until saturation. For induction of expression, a 1/25 dilution of the saturated culture into YEP., 1% glucose was made and the cells were grown to saturation. Cells were harvested, lysed with glass beads and the insoluble material was collected by centrifugation. The pellet was resuspended in gel sample buffer and boiled. Extracts were fractionated on standard denaturing polyacrylamide gels. Proteins were visualized by staining with Coomassie blue dye. A band of the appropriate molecular weight for p53 was present in the cultures transformed with pCl/1-GAP-p53 but not in control extracts of JSC302.</p>
    <p>4.6.3. Protein purification</p>
    <p>Cultures of JSC302 (pCl/1-GAP-p53) were grown under the conditions described above except that the culture in YEP, 1% glucose was allowed to grow for only 24 hours. Cells were harvested and processed using the glass bead lysis procedure with Triton lysis buffer. Triton Lysis Buffer (0.1%, Triton X-100 (10 mM Tris HCl 8.0), 62.5 mM EDTA pH 8.0, 50 mM, Tris HCl pM 8.0). The proteins were precipitated with 40% NH<sub>2</sub> SO<sub>4</sub>. The pellet was resuspended in H<sub>x</sub> O and dialyzed with 50 mM phosphate pH 7.0, 1 mM EDTA, 1 g/ml Leupeptin, 1 g/ml Leupeptin, 1 g/ml Aprotinin, 1 mM PMSF (Phenylmethyl-sulfonylfluoride).</p>
    <p>The solution is applied to a Mono-Q resin (Pharmacia) on a FPLC column (Pharmacia). The column eluted with a 0-1M NaCl gradient. Fractions are collected and analyzed by Coomassie staining western blot analysis for the presence of p53. The peak fraction is made up to 20% glycerol. The column may be repeated to obtain higher purity of the protein.</p>
    <p>5. Immunoassay for Anti-HIV Abs Using Recombinant HIV Polypeptides</p>
    <p>Diagnostic assays based on the ELISA technique and employing recombinantly produced vital antigens have been developed for the detection Of antibodies to HIV. Micro titer plate based ELlSAs and an immunoblot strip ELlSAs have been configured which use either three or four recombinant vital antigens: the major core protein p25gag, the endonuclease region of the vital polymerase gene p31pol, and one or two polypeptides from the envelope gene, either from gp120 and/or gp41 coding regions.</p>
    <p>5.1. ELISA-A</p>
    <p>5.1.1. Assay Protocol</p>
    <p>Stock solutions of purified p25gag protein (Section 3.1.5) (1.25 mg/ml in 20 mM sodium phosphate, 0.1% SDS, pH 7.2), purified env-2 protein (Section 4.2.2) (2 mg/ml in 20 mM sodium phosphate, 0.1% SDS, pH 7.2), and purified SOD-p31 fusion protein (Section 4.3.3) (2 mg/ml in 20 mM sodium phosphate, 0.1% SDS, pH 7.2) were prepared.</p>
    <p>For coating microtiter-plates (Dynatech Immulon I), 1 part each of the stock solutions of p25gag, envy and SOD-p31 were added to 997 parts of borate coating buffer (0.05M borate, pH 9.0). One hundred microliters of the coating solution was added to each well, and the plates were covered and incubated 2 h at 37 C. or 12 h at 4 C. The coating solution was then aspirated from the wells and the plates washed 6 x with wash solution (0.137M 0.8% NaCl, 0.05% Triton X-100).</p>
    <p>Serum samples were diluted 1:100 in dilution solution (0.1% casein, 1 mM EDTA, 1% Triton X-100, 0.5M NaCl, 0.01% thimerosal, pH 7.5) with yeast protein (strain AB103.1) extract (1:40 dilution, approximately 2 mg protein per ml in PBS containing 1% Triton X-100, 2 mM PMSF, 0.01% thimerosal) and E. coli protein. extract (1:40 dilution, approximately 1 mg protein per ml in PBS containing 1% Triton X-100, 2 mM PMSF, 0.01% thimerosal) added to the dilution solution. Extraction procedures were similar to those described in 16 and 18 above but using nonrecombinant strains. One hundred microliters of diluted serum was added to each well and incubated 30 min at 37 C. The plates were then washed 6 x with wash solution.</p>
    <p>Goat anti-human Ig labeled with horseradish peroxidase (Cappel) diluted 1:8000 in dilution solution without added yeast and E. coli extracts were added at 100 l/well to the plates and incubated 30 min at 37 C. The plates were then washed 6 x with wash solution. Substrate-solution (10 ml citrate buffer, 10.5 g citric acid/liter dH<sub>2</sub> O, pH to 4.0 with 6M NaOH), 0.1 ml ABTS  15 mg/ml 2,2'-azino-di-(3-ethylbenzthiazolene sulfonic acid) in dH<sub>2</sub> O! and 3.33 l H<sub>2</sub> O<sub>2</sub> at 100 l/well was then added to the plates and the plates wrapped in foil and incubated at 37 C. for 30 min. The reaction was then stopped by adding 50 l/well of 10% SDS. Readings were made with a Dynatech ELISA reader set for dual wavelength reading: absorbance wavelength of 1 (410 nm) and reference wavelength of 4.</p>
    <heading>5.1.2. Results</heading> <p>The following sera were tested:</p>
    <p>A. 89 consecutive blood donors from the Kansas City Blood Bank ("normal blood donors"): log nos. 1001-1081, 1085-1092.</p>
    <p>B. 52 sera from patients with lymphadenopathy syndrome (LAD) or AIDS or sexual partners of persons with LAD or AIDS (referred to as "contacts")--all obtained from UCSF AIDS Serum Bank panel: log nos. 4601-4652.</p>
    <p>The positive/negative cut-off used was 5 x (average background signal signal with diluent alone) and was determined to be 0.195. Thus, sera with signals below 0.195 were rated (-); those above were rated (+). Each sample was also evaluated by the commercially available ABBOTT HTLV III EIA kit (Abbott Labs) and by Western analysis.</p>
    <p>Tests on the normal blood donor samples indicated all except one were negative in the invention ELISA. This normal serum scored negative in the ABBOTT HTLV III EIA test, but was actually positive, as confirmed by Western analysis.</p>
    <p>The results of the tests on the 52 sera from LAD and AIDS patients and contacts are tabulated below:</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">______________________________________Serum             ABBOTT     InventionNo.      Diagnosis             EIA        ELISA     Western______________________________________4601     Contacts +          1.89  +   +02       Contacts -          0.04  -   -03       Contacts +          1.44  +   +04       Contacts +          1.92  +   +05       Contacts -          0.04  -   -06       Contacts +          &gt;2    +   +07       Contacts +          1.37  +   +08       Contacts +          1.60  +   +09       Contacts +          &gt;2    +   +10       Contacts +          &gt;2    +   +11       Contacts +          1.94  +   +12       Contacts +          &gt;2    +   +13       Contacts +          &gt;2    +   +14       Contacts +          &gt;2    +   +15       Contacts +          1.97  +   +16       AIDS     +          0.61  +   +17       AIDS     +          &gt;2    +   +18       AIDS     +          &gt;2    +   +19       AIDS     +          1.58  +   +20       AIDS     +          1.58  +   +21       AIDS     +          0.76  +   +22       AIDS     +          1.74  +   +23       LAD      +          1.26  +   +24       LAD      +          &gt;2    +   +25       AIDS     +          1.04  +   +26       AIDS     +          1.24  +   +27       AIDS     +          1.40  +   +28       AIDS     -          0.07  -   -29       LAD      +          1.93  +   +30       Contacts +          1.96  +   +31       AIDS     +          1.76  +   +32       AIDS     +          0.90  +   +33       AIDS     +          1.69  +   +34       LAD      +          1.09  +   +35       AIDS     +          1.54  +   +36       AIDS     +          1.22  +   +37       AIDS     +          1.96  +   +38       AIDS     -          &gt;2    +   +39       LAD      +          1.85  +   +40       LAD      +          &gt;2    +   +41       LAD      +          0.84  +   +42       LAD      +          1.59  +   +43       LAD      +          1.71  +   +44       AIDS     +          1.40  +   +45       LAD      +          &gt;2    +   +46       AIDS     +          1.38  +   +47       AIDS     +          1.29  +   +48       LAD      +          1.93  +   +49       LAD      +/-        0.48  +   +50       LAD      -          0.04  -   -51       LAD      -          0.07  -   -52       LAD      +          1.92  +   +______________________________________</pre>
    
    <p>The above results show that the ELISA-A, using recombinant HIV proteins, is at least as good as the ABBOTT HTLV III EIA test or Western analysis.</p>
    <p>In the ELISA-A reported in this example, the yeast and bacterial extracts were added to the serum to bind serum antibodies to yeast and bacteria to prevent such antibodies from binding to minor contaminants in the recombinant HIV-1 protein preparations. Both yeast and bacterial extracts were required since the recombinant polypeptides included polypeptides expressed in yeast and polypeptides expressed in bacteria. If all the polypeptides were expressed in the same type of organism, only one extract would be needed. For instance, if a p25gag polypeptide expressed in yeast was substituted for the bacterially produced p25gag polypeptide of the example, only yeast extract would be added to the serum samples.</p>
    <heading>5.2. ELISA-B</heading> <p>5.2.1. Assay Protocol</p>
    <p>Frozen stocks of the following purified proteins were thawed and used to make a solution containing: p25gag (Section 4.5) 1.25 g/ml, SOD-p31 (Section 4.3.3) 1.00 g/ml, SOD-env5b (Section 3.6) 1.2.5g/ml, and env-2 (Section 4.2.2) 0.50 g/ml in 0.05M sodium borate, pH 9.0. For coating microtiter plates, 100 l of the above solution was pipetted into each well of Immulon I round bottom microtiter plates (Dynatech Laboratories), and incubated for 2 h at 37 C. The coating solution was then aspirated from the wells. The plates were washed three times with 200 l/well Wash Buffer  100 mM Sodium Phosphate pH 7.4, 140 mM Sodium Chloride, 0.1% Casein (Sigma), 0.05% Triton X-100 (Sigma), 0.01% (w/v) Thimerosal (Sigma)! and then washed two times with 200 /well PBS (10 mM Sodium Phosphate pH 6.7, 150 mM Sodium Chloride). The plates were then post coated by incubating with 200 l/well of Postcoat solution  PBS, 0.1% (w/v) Casein, 2 mM PMSF (phenylmethyl-sulfonylfluoride)! for 30 minutes at 15-30 C. The Postcoat solution was aspirated off. The plates were dried in a lyophilizer (such as: Virtis Unitop 600 SL lyophilizer, Virtis Company) overnight or by incubating for 2 h at 25 C.</p>
    <p>To assay samples in the ELISA-B assy, 100 of Samples Diluent  100 mM Sodium Phosphate, pH 7.4, 0.5 M Sodium Chloride, 1 mM EDTA, 0.1% casein, 1% Triton X-100, 100 g/ml yeast extract (see below), 100 g/ml E. coli extract (see below), and 0.01% Thimerosal! was added to each of the coated wells on the plates. Ten l of the samples to be assayed or controls were pipetted into the wells. The plates were then sealed and incubated for 1 h at 37 C. The sample was then aspirated and the plates were washed three times with 200 l/well of distilled water. The plates were then incubated with a goat-anti-human IgG-HRP (horseradish peroxidase) conjugated antibody (available from commercial sources, i.e., Tago or Cappal, or the conjugate may be synthesized by the Nakane procedure). The anti-human IgG-HRP conjugate war diluted 1/3200 in Conjugate Solution I  PBS, 5% noral goat serum, 0.01% ANS (8-Anilino-1-naphthalene sulfonic acid, ammonium salt), 0.01% Thimerosal!. A final dilution to 1/32000 was made in Conjugate Diluent MT  116 mM Sodium Phosphate, pH 7.4, 0.622 M Sodium Chloride, 0.56% BSA, 2.22% normal goat serum, 1.11% Triton X-100, 0.11% Casein, 0.0044% ANS, 0.01% Thimerosal! immediately before use. One hundred l of the diluted conjugate was added per well (except for blank wells). The plates were sealed and incubated for 1 h at 37 C. The conjugate solution was aspirated off and the plates were washed three times with 200 l/well of distilled water. Then 100 l of Developer  made fresh: 50 mM Sodium Citrate adjusted to pH 5.1 with 1 M Phosphoric Acid, 0.6 /ml 30% H<sub>2</sub> O<sub>2</sub>, OPD tablet (Sigma) (1 tablet/5 ml of buffer)! was added to each well and incubated thirty minutes at room temperature (15-30 C.). The reaction was stopped by the addition of 50 l of (N H<sub>2</sub> SO<sub>4</sub> to each well.</p>
    <p>The microtiter plates were analyzed on a standard ELISA reader (such as Biotek EIA Autoreader Model EL310, Biotek Instruments) by reading the absorbance of developed color at 492 nm. The results were analyzed by comparing the values generated for the samples against an assay cutoff valve (cutoff=0.5average absorbance value for the positive controls.)</p>
    <p>Yeast and E. coli extracts used in the Sample Diluent were prepared in an analogous manner to the purification process for recombinant polypeptides in Sections 3.1.4.2-3.2.5 and 4.1.4.1-4.1.4.3, except non-recombinant strains S. cerevisiae AB103.1 and E. coli D1210 were used.</p>
    <p>5.2.2. Results</p>
    <p>The ability of ELISA-B to detect the presence of antibodies directed against HIV was compared to a licensed and commercially available ELISA produced by DuPont.</p>
    <p>493 serum samples were run in ELISA-B and the DuPont screening ELISA. The panel of sera was composed of specimens from various sources: 205 samples from CDC (clinically categorized as AIDS, ARC, contact, false positive from licensed screening ELISA, and 4 negative); 101 samples from UCSF (clinically categorized as AIDS, ARC, and contact); 187 samples obtained from Interstate Blood Bank, Pa., (initially scored positive on licensed screening ELISA). In this group of samples there were eight (8) discrepancies. Four (4) samples were found positive by ELISA-B and negative in the DuPont ELISA. Correlation between the DuPont assay and ELISA-B was 98.4%. Concensus data indicated that of the four (4) samples found positive by ELISA-B and negative by the DuPont ELISA, three (3) (LW 47, 4202, and 4225) were true positives (DuPont ELISA false negatives) and one (4279) was an ELISA-B false positive. Concensus data also indicated that the four samples found negative in ELISA-B and positive in the DuPont ELISA (LW 12, 20061, 20145, 20162) were negative (DuPont ELISA false positives). For the eight (8) discrepant samples ELISA-B differed from the concensus data in one (1) case and the DuPont ELISA differed in seven (7) cases. Also note that one sample was false positive in both assays.</p>
    <p>The positivity or negativity of a specimen in these panels was determined from the concensus results of commercial viral ELISAs, the microtiter plate assay using recombinant antigens, the strip ELISA, Western Blot data (when available) and clinical data, if available.</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">                                  TABLE__________________________________________________________________________Correlation of ELISA-B with the DuPont Screening ELISA DiscrepantSamples                        Strip ELISA  ELISA-B    DuPont ELISA                        P-                          P-                            GP-                               GPPanels OD<sub>490nm</sub>       S/CO  OD<sub>490nm</sub>                  S/CO  24                          31                            41 120__________________________________________________________________________UCSF(N = 100)CDC(N = 205)4110   2.25 4.4   1.66 4.22  - 4+                            4+ 4133   1.21 2.4   1.25 3.18                            - 2+ -4204   .56  1.1   .33  .90   - - -  -4218   .60  1.2   2.37 6.4   - - -  -4225   .54  1.1   .07  .2    - - -  -4279   .98  3.6   .18  .5    - - -  -Interstate (LW)(N = 58)12     .28  .5    1.17 2.0   - - -  -47     .46  .90          (1.7)             .46  .80                     (.54)                        1+                          - -  -Interstate I(N = 67)20061  .22  .48   1.71 2.5                     (2.1)                        - - -  -Interstate II(N = 62)20145  .13  .22   &gt;3.00                  4.4                     (3.2)                        - - -  -20162  .18  .32   .94  1.4                     (1.0)                        - + +  +__________________________________________________________________________ S/CO = signal: cut off ratio if = or &gt; 1.0 samples is positive</pre>
    
    <heading>5.3. Dot Blot Assay</heading> <p>Nitrocellular strips (0.35 cm) are spotted with 50 ng polypeptide in PBS (sporting volume 2 ). After spotting the strips are dried at room temperature for 1 h or more. The strips are then post-coated in a 5% solution of Carnation non-fat dry milk in PBS, 0.01% Thimerosal, for 15-60 min at room temperature. Each test solution sample is diluted 1:50 in 0.5 ml of the post-coating solution in a test tube. A post-coated strip is then placed in the tube and incubated in the sample with rocking at 37 C. for 1 h. The strip is then removed from the tube and washed with post-coating solution. The strip is then incubated for 15 min at room temperature in goat anti-human Ig reagent labeled with horse radish peroxidase diluted 1:500 in post-coating solution. After incubation in the labeled antibody, the strip is washed serially with PBS, 1% Triton, and distilled water. The strips are developed by incubating them in substrate solution (see 23 above) for 15 min at room temperature.</p>
    <p>Positive samples will cause a visually perceptible color change at the spotting site. Normal (negative) sera sample yield no color change or give a faint signal that is discernible from a positive signal. Competition assays may be run on sera giving faint signals to verify that they are negative. In the competition assay, polypeptide (10-25 g/ml) is added to the test sample and incubated from 1 h at 37 C. before the strip is incubated in the sample. With authentic positive sera the signal is completely blocked by the added polypeptide, whereas with normal (negative) sera there is no change in signal.</p>
    <p>5.4. Immunoblot Strip ELISA Assay</p>
    <p>In the immunoblot strip ELISA, recombinant derived vital antigens are individually coated in bands on a nitrocellulose strip and reacted with samples to bind anti-HIV specific antibodies.</p>
    <p>Frozen aliguots of the individual recombinant polypeptides were thawed and diluted to the appropriate concentration in Coating Solution  PBS pH adjusted to 7.4, 375 mg/l Napthiol Blue Black (used only as an inert marker)! as follows: p25gag (Section 4.5) 3-4 g/ml, SOD-p31 (Section 4.3.3) 1-1.6 g/ml, SOD/env-5b (Section 3.6) 1.0 g/ml, env-2 (Section 4.2.2) 0.5 g/ml. The individual protein solutions were coated as individual bands on a sheet of nitrocellulose paper; BA85 NC; Schleicher &amp; Schuell, Inc., Keene, N.H., (pre-wetted with PBS buffer). The coating apparatus can be any apparatus which allows for solutions to be applied to filter paper under vacuum pressure as discrete bands, such as Minifold II Slot-Blot System; Schleicher &amp; Schuell.</p>
    <p>The coated sheets are dried overnight. The dried, coated sheets are immersed for five minutes in Blocking Buffer (PBS, 1% (w/v) Casein, 0.01% Thimerosal, pH adjusted to 7.4) and then dried overnight. The coated and blocked nitrocellulose sheets were attached to a backing sheet of paper with double faced tape and then cut into individual strips which contain the individual bands corresponding to each of the recombinant vital antigens.</p>
    <p>The sample to be assayed was diluted 1/100 with Sample Diluent by adding 10 l of the sample to 1 ml of Sample Diluent  PBS, 0.1% Casein, 1 mM. EDTA, 2.0% Triton X-100, 1 mg/ml Yeast extract, 500 g/ml E. coli extract, 0.2 g/ml YP45 extract, 0.01% Thimerosal, pH adjusted to 7.4!. The strips were then individually soaked in the diluted sample solution with agitation for 2 h at room temperature (15 to 30 C.). The strips were removed from the diluted sample solution and washed four times with water. The strips were then incubated with 1 ml/strip of Goat anti-human IgG-HRP conjugate diluted in Conjugate Buffer  PBS pH 7.2, 0.3% Casein, 5% normal goat serum, 0.01% ANS, 0.01% Thimerosal! with agitation for 30 minutes at room temperature (15-30 C.).</p>
    <p>The strips were removed from the conjugate solution and then washed three times with water. The strips were then incubated with 1 ml/strip of Developer Buffer  10 mM Sodium Phosphate, 20 mM Sodium Chloride, 0.8 l/ml 30% hydrogen peroxide, 0.05% (w/v) 4-chloro-1-naphthol, 16.6% methanol! with agitation for 15 minutes at room temperature (15-30 C.). The reaction was stopped by removing the strips from the Developer Buffer and washing two times in water. The color developed on the band is compared with positive control bands (IgG applied at 0.25 g/ml and 1.25 g/ml).</p>
    <p>Yeast and bacterial proteins were used to preabsorb antisera for cross-reactive antibodies. Yeast extract was prepared by processing bulk yeast S. cerevisiae (Red Star Yeast, grade 1; Red Star, Oakland, Calif.) through the standard lysis procedure as described previously for the purification of yeast recombinant proteins. The insoluble cellular debris was separated from the soluble protein fraction. The E. coli extract was prepared by growing E. coli strain D1210 (pSODCF2), lysing the cells, and solubilizing the pellet as described in Section 3.6.4. YP45 extract is a yeast protein and was prepared by growing. S. cerevisiae AB110 and following the purification procedure described in Section 4.3.3.3. A 45 kd yeast protein was purified which helps in preabsorbing nonspecific cross-reactive antibodies found in some area.</p>
    <p>6. Serology Studies with Recombinant HIV Polypeptides</p>
    <p>6.1 Immunoblot with Env Polypeptides</p>
    <p>Recombinant polypeptides originating from different regions of the env gene of the ARV-2 isolate of HIV (Section 4.2) were used to characterize the anti-envelope antibody response of virus seropositive individuals. The sera characterized included specimens from AIDS patients, ARC patients and clinically healthy homosexual men with documented exposure to the virus through sexual contact with AIDS or ARC patients (contacts.)</p>
    <p>Regions of the envelope gene of the ARV-2 isolate of HIV were cloned into yeast expression vectors (env-1, env-2, env-3) or bacterial expression vectors (env-5b), and used to produce recombinant proteins as described in Sections 3 and 4.</p>
    <p>Extracts of yeast for SDS polyacrylamide gel electrophoresis were prepared as follows: 1 ml of yeast grown to 20 O.D. (A450) were pelleted and suspended in 200 l of electrophoresis sample buffer. The mixture was boiled for 10 min and then centrifuged at 12,000 g for 2 min to remove cell debris prior to electrophoresis. To prepare bacterial extracts for gel electrophoresis, bacterial cells from 1 ml of culture were pelleted, suspended in 200 l of electrophoresis sample buffer, and disrupted by 3 cycles of freezing and thawing. The mixture was boiled for 10 minutes prior to electrophoresis.</p>
    <p>Yeast or bacterial extracts were electrophoresed on standard Laemmli discontinuous SDS-polyacrylamide (12% acrylamide) gels using a Biorad Laboratories minigel apparatus. Laemmli (1970) Nature 227: 680-685. The extracts (100 l) were loaded into an 8 cm well in the stacking gel prior to electrophoresis. Following electrophoresis, proteins were transferred to nitrocellulose filters. Towbin et al. (1979) Proc. Natl. Acad. Sci. USA 76: 4350-4355. Strips of the filter with the electroblotted lysates were cut and each strip reacted with an individual serum sample diluted 1/100.</p>
    <p>Sera from 85 seropositive individuals, including AIDS patients, ARC patients, contacts and clinically healthy individuals, as well as a pool of sera from normal human blood donors, were assayed on env-1, env-2 and env-3. The blots were processed with Carnation nonfat dry milk as described previously. Johnson et al. (1984) Gene Anal. Techn. 1: 3-8. The enzyme conjugate (diluted 1/200) was goat antiserum to human immunoglobulin G (Cappel Laboratories) and the chromogen was HRP color reagent (4-chloro-1-Napthol; Biorad Laboratories).</p>
    <p>Positive immunoblot assays obtained with env-1, -2, and -3 are shown in FIG. 30. A prominent band at 28,000 was seen in the env-1 blot with a patient's serum with anti-env-1 antibodies. In the immunoblot of env-2, the antibody-positive serum reacted with a protein of approximately 55 kD. With env-3, the positive serum reacted with a series of bands migrating between 30,000 and 35,000. The multiple immunoreactive species in the env-3 immunoblot are due to differences in glycosylation (see Section 4.2.3) of this polypeptide during its synthesis in yeast. Five potential glycosylation sites are contained within the coding region for env-3.</p>
    <p>The results of assays of the reactivity of sera obtained from the various patient groups in the 85 serum panel with the env-1, env-2 and env-3 are presented in table below:</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">______________________________________Clinical Number    Number Reacting With:Group    Tested    env-1    env-2   env-3______________________________________Contacts 21        4 (19%)  21 (100%)                               21 (100%)ARC      26        3 (12%)  26 (100%)                               26 (100%)AIDS     38        6 (16%)  36 (95%)                               36 (95%)______________________________________</pre>
    
    <p>All of the specimens selected for this analysis were positive in an enzymes linked immunosorbent assay (ELISA) for antibodies to the AIDS retrovirus. Weiss et al. (1985) J.A.M.A. 253: 221-225. Sera from all of the contacts and ARC patients had antibodies that reacted with env-2 and env-3 indicating that they had antibodies to both gp120 and gp41 that could be detected with these non-glycosylated polypeptides in immunoblot assays. Thirty-six of the 38 AIDS patients (95%) had antibodies that reacted with env-2. The same thirty-six sera also reacted with env-3. The two AIDS sera that did not react in immunoblot assays with env-2 or env-3 either lacked antibodies detected using these recombinant antigens, or their antibody titers were below the limit of detection in these assays, at least in generating an immune response to sequential epitopes.</p>
    <p>When these sera were tested in immunoblot assays with env-1, only a minority in each diagnostic group scored positive. The percentage of contacts, ARC patients and AIDS patients with antibodies to env-1 was 19%, 12% and 16%, respectively. Since env-1 corresponds to the amino-terminal half of env-2 (amino acids 26-491, the majority of gp120), this observation suggests that the carboxyl terminal portion of gp120 is much more immunogenic in infected individuals than the amino-terminal half of the polypeptide, at least in generating an immune response to sequential epitopes.</p>
    <p>Because the env-1 polypeptide was also represented in env-2, it could not be determined directly from the immunoblot results if env-1 positive sera reacted exclusively with env-1 or were also reactive with the carboxyl-terminal half of env-2. To evaluate the reactivity of env-1 positive sera with the carboxyl-terminal half of env-2, competition experiments were carried out. Sera were reacted with env-1 and env-2 in immunoblot assays in the presence or absence of an excess of env-1 in the serum diluent. An env-1 positive serum, diluted 1/100, was incubated with: env-1 immunoblot with no addition, env-1 immunoblot with 50 g/ml env-1 in diluent, env-2 immunoblot with no addition, env-2 immunoblot with 50 g/ml of env-1 diluent and env-2 immunoblot with 50 g/ml of env-2 in diluent. The reactivity of the env-1 serum sample with the env-1 blot was completely eliminated by preincubation with env-1. However, reactivity with the env-2 blot was still evident. Similar results were observed with all of the env-1 positive sera, indicating that they all reacted with the carboxyl terminal as well as the amino terminal half of the gp120 polypeptide.</p>
    <p>Immunoblots of extracts from bacteria expressing a subregion of gp41 (aa 557-667) as an hSOD fusion, referred to as env-5b, with an AIDS patient's serum, showed a prominent immunoreactive species at 32,000 daltons that was absent from immunoblots of extracts from bacteria transformed with the vector lacking the env insert (Section 3.6). This was approximately the molecular weight expected for a polypeptide coded for by the sum of the hSOD and any gp41 sequences. Twenty serum samples shown previously to contain antibodies that reacted With env-3 in immunoblot assays were reacted with env-5b immunoblots. All 20 specimens reacted with the fusion protein.</p>
    <p>Immunoblot assays detect antibodies which recognize principally sequential determinants. The results presented here indicate that virus seropositive individuals frequently have mounted immune responses to sequential epitopes within the carboxyl terminal half of the gp120 polypeptide. In addition; such individuals have also mounted immune responses to sequential epitopes within gp41 (env-5b). The prevalence of antibodies to these two any regions detected by using recombinant polypeptides from a single virus isolate suggests that there are epitopes within both regions that are highly conserved. The failure of most virus seropositive specimens to react with the amino terminal half of gp120 represented by any-1 may indicate that there are not such highly conserved immunodominant sequential epitopes within this region. These data suggest that either the amino terminal region of gp120 is a poor immunogen in humans, or humans respond in a strain-specific fashion So that only those individuals infected with a highly homologous virus are detected with env-1 from ARV-2. Another explanation centers around the possibility that conformational epitopes may be a feature of the N-terminal portion of gp120; immunoblotting would miss antibodies directed at conformational determinants. Another possible explanation is that the immunogenicity of the amino terminal region of gp120 is masked by glycosylation or by structural constraints due to association of gp120 with gp41 and/or the vital membrane.</p>
    <p>It was also found that all sera reacting in immunoblots with env-3 also reacted with the fusion protein env-5b. In fact, several sera that were only weakly positive on env-3 were clearly positive on env-5b. The sensitivity of an immunoblot assay is limited by the amount of antigen that is present in the preparation. Since env-5b expression levels were high compared to the levels of expression of env-3 in yeast (env-5b represented approximately 2-5% of the total bacterial protein, whereas env-3 was less than 0.1% of the total yeast protein), it was not unexpected the env-5b immunoblot assays were more sensitive than the env-3 immunoblot assays.</p>
    <p>6.2 ELISA with Env Polypeptides</p>
    <p>To provide a quantitative assessment of the antibody response detected using these recombinant polypeptides representing regions of two subunits of the HIV envelope glycoprotein complex, env recombinant polypeptides (Section 4.2) were purified and enzyme-linked immunosorbent assays (ELISA) configured.</p>
    <p>Test sera were the same as described in Section 6.1. The ELISA procedures were modifications of those described previously. See Brun-Vezinet et al. (1984) Lancet I:1253-1256; Saxinger et al. (1983) Lab. Invest. 49: 371-377; Weiss et al. (1985) J.A.M.A, 253: 221-225; Steimer et all (1986) J. Virol. 58: 9-160.</p>
    <p>For virus ELISAs, microtiter plates (Dynatech, Immulon I) were coated with 5 g/ml of SDS-disrupted sucrose gradient purified ARV-2 virus. Human serum samples, diluted 1/100, were added to the wells and the plates incubated at 37 C. After 1 h, the plates were washed and goat antiserum to human immunoglobulin conjugated with horseradish peroxidase (Cappel Laboratories) and diluted 1/4000 was added to the wells. The plates were incubated for 30 min. at 37 C., washed and the substrate solution (150 g/ml, 2,2'-azino-di-3'-ethylbenzylthiazoline sulfate in 0.1M citrate, 0.001% H<sub>2</sub> O<sub>2</sub>, pH 4.0) was added for 30 minutes. The plates were read on a ELISA reader at 415 nm with a reference wavelength of 600 nm. Samples were scored as positive when their assay result was greater than five times the absorbance obtained with the negative control pooled normal human serum (NHS).</p>
    <p>For recombinant env. polypeptide ELISAs, microtiter plates were coated with 2 g/ml of purified env-2 or env-5b (Section 7.1). Serum samples were assayed for antibodies to these antigens by the procedure described above. Included in the serum diluent for the env-2 antibody ELISA was included extract (final concentration 100 g/ml) from yeast transformed with the vector alone lacking the env-2 insert. Similarly, 100 g/ml of an extract from untransformed E. coli was added to the diluent for the env-5b antibody ELISA.</p>
    <p>FIG. 16 (panel A) shows the results of a titration of antibodies in an AIDS patient's serum that reacted with env-2. Pooled normal human serum (NHS) (0) and serum 0036 from an AIDS patient () were diluted 1/25 and then by serial 2-fold dilutions. The titer of env-2 antibodies in this particular serum specimen was approximately 1400. The signal with the control sample (NHS) was low at all dilutions.</p>
    <p>Env-5b was purified and used as antigen in an ELISA for antibodies to the gp41 polypeptide. FIG. 16 (panel B) shows the results of a titration of env-5b antibodies in the same serum sample that was titered previously in the env-2 ELISA. No appreciable signal was seen with NHS.</p>
    <p>Serum samples from 88 blood donors, all seronegative in an ELISA for AIDS retrovirus antibodies were tested at a dilution of 1/160 in the env-2 and env-5b antibody ELISAs (FIG. 17). The average signal in the env-2 antibody ELISA was 0.0450.033 with a range of 0 to 0.149. In the env-5b ELISA these same sera yielded an average signal of 0.0340.030 with a range of 0 to 0.144. The cut-off for both assays was set at five times the average signal of these 88 seronegative specimens. For the env-2 antibody ELISA the cut-off was 0.225, and for the env-5b assay it was 0.170.</p>
    <p>The results of the env-2 and env-5b ELISAs with the same panel of virus seropositive specimens described previously (see the table in Section 7.1) are presented in FIG. 18. The dotted line in the top panel designates the cut-off (0.174) in the env-2 ELISA. The dotted line in the bottom panel (0.154) is the cut-off in env-5b antibody ELISA. Each data point is the average of duplicate assays. All of the sera, regardless of diagnosis, yielded the maximum signal in the env-5b antibody ELISA.</p>
    <p>Serum antibodies were clearly detected in the env-2 ELISA with specimens from 20 of the 21 contacts, 24 of the 25 ARC patients, and 35 of the 38 AIDS patients. The remaining five sera, one from a contact, one from an ARC patient and three from AIDS patients, are considered to be borderline. The contact and ARC sera scored just below the assay cut-off while the three AIDS sera scored just above the cut-off: These same sera were all clearly positive in the env-5b antibody ELISA.</p>
    <p>FIG. 19 compares antibody titers of the above sera for both env-2 and env-5b. These serum samples were assayed in the env-2 and env-5b ELISAs at dilutions ranging from 1/100 to 1/51,200. The titers reported are the dilution at which half-maximum absorbance was attained. The env-2 titers are shown as dark circles, and the env-5 titers are open circles. The horizontal lines show the mean titers of sera in each group. Borderline sera (less than 0.15 above assay cut-off) are shown as "". Sera with titers below 1/100, but not borderline, are plotted as having titers of 1/100.</p>
    <p>Antibody titers in sera obtained from all three groups were higher to env-5b than to env-2). The average titer of env-2 antibodies in sera obtained from contacts was 2224 (range &lt;100-7975) compared to an average titer of env-5b antibodies of 8988 (range 650-25,000). Among the ARC patients the average env-2 and env-5b antibody titers were 2480 (range &lt;100-12,800) and 12,560 (range 42062,000), respectively. Finally, for AIDS patients the average env-2 antibody titer was 1394 (range &lt;100-12,800) compared to an average env-5b antibody titer of 7059 (range &lt;100-42,000).</p>
    <p>The ratio of env-5b to env-2 antibody titers was 5.06, 4.04 and 5.06 for contacts, ARC patients and AIDS patients respectively. Since these ratios remain constant, regardless of diagnosis, it is unlikely that there is a selective decline in the antibody titer to one or the other of the env polypeptides with progressive disease. Instead, it indicates that individuals with low titer antibodies to one env polypeptide tend to have low titers of antibodies to the other polypeptide. Examining the env-5b antibody titers of those individuals with low (&lt;200) titers of env-2 antibodies (table below) supported this conclusion; these sera all had correspondingly low env-5b antibody titers.</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">______________________________________Serum   Clinical   Env-2 Antibody                          Env-5b AntibodyNumber  Group      Titer       Titer______________________________________4687    C*         &lt;100        13004623    ARC        115         31004648    ARC        125         10004672    ARC        &lt;100        5904677    ARC        &lt;100        4504685    ARC        170         4204616    AIDS       &lt;100        4304619    AIDS       100         9004620    AIDS       130         6204625    AIDS       &lt;100        15004626    AIDS       100         43004627    AIDS       100         2004631    AIDS       170         5004637    AIDS       190         4804653    AIDS       &lt;100        25004654    AIDS       100         4004660    AIDS       100         7604664    AIDS       155         3004667    AIDS       100         680______________________________________ *Contacts.</pre>
    
    <p>6.3 ELISA with Gag Polypeptides</p>
    <p>Large-scale purification of p25gag from bacterial extracts (Section 3.1.5) provided sufficient antigen for a survey of p25gag seropositivity among various risk and patient groups. These sera were tested first in a virus ELISA to identify those with antibodies to the virus and then in the p25gag ELISA to determine the proportion of virus seropositive individuals with antibodies to p25gag.</p>
    <p>Ninety-six sera from random blood donors, all of whom were negative for AIDS virus antibodies in the virus ELISA, scored negative in the ELISA for p25gag antibodies. The average signal in the p25gag ELISA of these sera was 0.0340.016 optical density units (OD) with a range of 0.011 to 0.089 (not shown).</p>
    <p>A panel of 100 sera was then examined, consisting of 28 specimens from high-risk individuals with no symptoms of the disease, but with potential exposure to the virus through sexual contact with AIDS patients (contacts), 33 sera from patients with AIDS-related complex (ARC), and samples from 39 patients with AIDS. Eighty-six of the 100 specimens were positive in the virus ELISA, an indication that these individuals had mounted an immune response to one or more viral antigens. The number of virus ELISA positive contacts was 21 (75%), ARC patients was 27 (81.8%), and AIDS patients was 38 (97.4%). The results of the p25gag antibody ELISA of these specimens are summarized in the table below. Of the 86 sera from the total panel that were positive in the virus ELISA, only 34 (39.5%) shored positive in the p25gag ELISA. When the results were grouped according to diagnosis, however, the contacts had the highest number of virus seropositive individuals with antibodies to p25gag (71%), the ARC patients were intermediate (48%), and the AIDS patients were the lowest (16%). Also tested were the 14 virus ELISA negative specimens from this panel of 100 sera in the p25gag ELISA, and all scored negative (not shown).</p>
    <p>The data presented in the table suggest that with an increase in the severity of the disease, there is a decline in the proportion of virus seropositive individuals with antibodies to p25gag. To determine the significance of the observed differences in p25gag seropositivity between groups, we used an X<sup>2</sup> test with pairwise comparison of groups. This analysis shows that the probability that the observed differences in frequency of p25gag seropositivity between the contacts and AIDS patients could have been due to chance was less than 1 in 2000. Similarly, the probability that the difference between ARC and AIDS patients was due to chance was very Small (1 in 200). A comparison of contacts and ARC patients, however, gave a higher probability (1 in 10) that the observed difference was due to chance. A larger sample size would be required to establish the significance of this difference with greater certainty.</p>
    <p>These results suggest that monitoring p25gag seropositivity of individuals infected with the AIDS retrovirus might be of diagnostics value. Although patients with more severe forms of the disease clearly have antibodies to other vital antigens, such as the envelope glycoproteins  Barin et al. (1985) Science 228: 1097; Montagnier et al. (1985) Virology 144: 283!, they are less likely than those in earlier stages to have antibodies to p25gag.</p>
    <p>The above results were obtained with p25gag antigen produced in E. coli. Similar results have been seen with the antigen produced in yeast.</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">______________________________________p25gag Antibody ELISA of VirusSeropositive Specimens<sup>a</sup>             NumberCategory or diagnostic             scoring Percentage<sup>b</sup>(number in group) positive                     positive______________________________________Contacts (21)     15      71ARC patients (27) 13      48AIDS patients (38)              6      16total (86)        34      39______________________________________ <sup>a</sup> Samples were categorized as virus seropositive if they yielded greater than five times the signal obtained with normal human serum (NHS) in a virus ELISA assay using disrupted ARV2 virus as antigen. The p25gag antibody ELISA procedure was as follows: ELISA plates were coated with 5 g/ml of p25gag purified from bacterial extracts. In the diluent was included a lysate from untransformed E. coli. All were diluted 1/100 for assay. The  conjugate was horseradish peroxidase conjugated goat antiseru to human immunoglobulin (Cappel, No. 32010081). <sup>b</sup> A sample was scored as positive when the average OD reading was greater than five times the signal with pooled normal human serum (NHS). In this particular assay this was OD<sub>414</sub> = 0.200.</pre>
    
    <heading>6.4. Western and ELISA with Pol Polypeptides</heading> <p>A panel of 10 sera that scored positive in the ELISA for vital antibodies with disrupted virus as the antigen was selected to compare the immunoreactivity of vital p31 with those of the two recombinant proteins. This panel included eight sera that were positive and two sera that were negative for antibodies to viral p31 in virus immunoblots. Lysates of pTP31.2 (Section 3.4.2) and pTSp31 (Section 3.5) transformants were electrophoresed and electroblotted, and strips of the blots were reacted with individual serum samples (see table below). The eight sera that were positive on vital p31 also reacted with the 30- and 48-kDa bands in lysates of bacterial cells containing pTP31.2 (expressing p31pol) and pTSp31 (expressing SOD-p31pol fusion), respectively. The two sera that were negative on vital p31 were also negative on both of the recombinant proteins. Pooled normal human serum was negative in all three Western blot assays.</p>
    <p>The SOD-p31 fusion protein was purified and used as a source of antigen for an ELISA for testing sera for antibodies to p31. Prior to deciding to pursue the fusion protein as a source of antigen for the assay, sera from 300 random blood donors and a panel of 100 sera obtained from high-risk, AIDS, and ARC patients was screened for antibodies to purified human SOD in an ELISA. None of these sera scored positive (data not shown).</p>
    <p>The ELISA protocol was as follows: Microtiter plates were coated with 2 g/ml of SOD-p31 in borate buffer (pH 9). In the diluent was included extract from E. coli (pTAC7). This extract was necessary to absorb out antibodies in human sera that reacted with minor contaminants in the purified SOD-p31. The ELISA protocol from this point was identical to the procedure described for other recombinant HIV antigens described above.</p>
    <p>A panel of sera from 100 consecutive blood donors that were all seronegative in the virus ELISA was tested in the p31 ELISA. These sera 811 scored very low in the assay (FIG. 31a). The average ELISA result for these sera when they were assayed at a 1/100 dilution was 0.026 with a range of 0.001 to 0.117 and a standard deviation of 0.012. The results obtained with 85 virus-seropositive samples are presented in FIG. 31b. The ELISA results with sera that did not react with the p31 band in the virus Western blot assay are indicated by shading. In general, sera that were positive in virus Western blots for p31 antibodies were clearly positive in the p31 antibody ELISA. There were, however, three sera that did not react with p31 in virus immunoblots that were clearly positive in the p31 ELISA.</p>
    <p>Three possible explanations for vital p31 immunoblot-negative sera scoring positive in the p31 ELISA were considered. First, the ELISA may have been more sensitive than the virus immunoblot assay for detecting antibodies to p31. Second, these sera may have scored positive in the ELISA owing to immunological reactivity with E. coli proteins contaminating the SOD-p31 preparation that were not absorbed out by the bacterial lysate in the antibody diluent. Third, these sera may have scored positive in the ELISA because they contained antibodies to the SOD portion of the fusion protein.</p>
    <p>To clarify this, these three sera were tested on blots of directly expressed p31 by using the pTP31.2 lysate, which should contain a much higher concentration of p31 antigen than virus. All three of these sera reacted with the p31 species in this assay (data not shown). Thus, these sera scored positive in the ELISA because of its greater sensitivity than virus immunoblots for detecting p31 antibodies, not because of reaction with E. coli contaminants or SOD.</p>
    <p>We also tested the remaining eight vital p31 Western blot-negative sera in immunoblot assays of pTP31.2 lysates. The five sera scoring lowest in the ELISA (FIG. 31b) failed to react with p31 in this assay. However, the three remaining sera, scoring between 0.15 and 0.3 in the ELISA, reacted in immunoblots with directly expressed p31 (data not shown). Thus, we concluded that of this panel of 85 seropositive Samples, 80 (95%) had detectable antibodies to the p31 antigen.</p>
    <p>These data clearly demonstrate the utility of the HIV pol endonuclease polypeptide in serodiagnosing HIV infection. It should be noted that serological results were identical when SOD-p31 produced in yeast was used instead of the bacterially expressed protein.</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">______________________________________Comparison of Western blot results of variousvirus-seropositive samples in viral p31,recombinant p31 expressed directly in E. coliand as a fusion protein with human SODReactivity with:                           RecombinantSerum   Viral p31   Recombinant p31                           SOD-p31______________________________________4607    +           +           +4608    +           +           +4620    +           +           +4625    +           +           +4626    +           +           +4642    -           -           -4643    +           +           +4646    +           +           +4659    -           -           -0036    +           +           +NHS<sup>b</sup>   -           -           -______________________________________ <sup>a</sup> Strips from immunoblots of electrophoresed virus, pTP31.2 extracts and pTS31 extracts were reacted with a 1/100 dilution of each serum. <sup>b</sup> NHS, pooled normal human sera obtained from Medical Specialities Laboratories.</pre>
    
    <p>7. HIV Immunization</p>
    <p>Recombinantly produced vital envelope protein was used to generate anti-HIV antibodies in experimental animals which were capable of neutralizing the infectivity of the virus. The purified env-2 (Section 4.2.2) was injected into both mice and guinea pigs and the sera had neutralizing activity in an in vitro neutralization assay.</p>
    <p>The HIV neutralization assay that was developed measures the ability of serum specimens to neutralize directly the infectivity of HIV in a tissue culture system (described in commonly owned, co-pending U.S. patent application No. 946,539, the disclosure of which is hereby incorporated by reference). Diluted sera are mixed with an equal volume of virus inoculum, the mixture is incubated for 30 minutes at room temperature and then 0.1 ml of the mixture is added to a 1 ml culture of permissive HUT-78 cells (110<sup>4</sup> cells/ml) in 24-well microtiter plates. Seven days later the cells are harvested, lysed with 1% Triton X-100 in PBS.</p>
    <p>Infection is monitored by measuring the levels of intracellular p25gag antigen with a capture enzyme-linked immunosorbent assay (ELISA) Steimer et al. (1986) Virology 150: 283-290. This assay uses a murine monoclonal antibody immobilized on the assay plate to capture p25gag and a rabbit polyclonal antiserum as the detecting reagent. The virus inoculum that was used for all strains of HIV was adjusted to yield 40-80 ng/ml of intracellular p25gag and approximately 10% of the HUT-78 cells in the culture were infected under these conditions.</p>
    <p>7.1. Env-2 immunization of mice</p>
    <p>In order to test if env-2 polypeptide derived from yeast was capable of eliciting neutralizing antibodies, mice were immunized with env-2 (Section 4.2.2).</p>
    <p>Three Balb/c mice were injected three times with 10 g of env-2 in alum at two week intervals. The mice were bled one week following the theta injection and their sera tested for env-2 antibodies in ELISA. Normal mouse serum from a pool of sera obtained from unimmunized Balb/C mice, was included as a control. The sera was diluted 1:10, and by serial 2-fold dilutions, and reacted in the env-2 ELISA. All three mice had mounted an antibody response to the injected env-2 (see below). The sera were then tested in the neutralization assay at a 1:20 dilution. Sera from two of the immunized mice showed significant neutralization of HIV at a 1:20 dilution (see below).</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">______________________________________                  Neutralization                  activitySerum        Env-2     (percentsample       antibody titer                  neutralization)______________________________________normal       &lt;10       18mouse 1      420       97mouse 2      230        4mouse 3      200       76______________________________________</pre>
    
    <heading>7.2. Env-2 immunization of guinea pigs</heading> <p>Twelve Hartley guinea pigs were used to study the effectiveness of the env-2 polypeptide (Section 4.2.2) in eliciting neutralizing antibodies. The sera were also tested for their ability to neutralize different strains of the HIV virus.</p>
    <p>Six of the guinea pigs were immunized with 50 Mg of antigen with adjuvant (see below) and six were immunized with the adjuvant alone according to the following protocol:</p>
    <p>Adjuvant: contains 0.5 mg Monophosphoryl Lipid A (MPL), 0.5 mg Trehalose Dimycolate (TDM), 0.5 mg Cell Wall Skeleton (CWS) (isolated from attenuated tubercle bacillus: Bacillus-Calmette-Guerin) lyophilized in 40 l of oil (Squalene) and 0.2% Tveen 80 in rarer (adjuvant obtained from Ribi Immunochem Research, Inc., Hamilton, Mo.).</p>
    <p>Vaccine: Reconstitute the adjuvant by adding 600 l of PBS and vortexing vigorously for 2-3 minutes. Mix 300 l of the adjuvant with 300 g of the env-2 polypeptide and PBS to bring total volume to 600 l. Vortex vigorously for 2-3 minutes; Inject each guinea pig with 100 l of vaccine in foot pad.</p>
    <p>Control: Dilute remaining 300 l of adjuvant with 300 l of PBS and vortex vigorously for 2-3 minutes. The six control guinea pigs are injected with 100 l each in foot pad.</p>
    <p>Immunization: Two to four days prior to immunization Schedule: obtain "prebleed" blood sample for guinea pigs: Day 0: Primary immunization; Day 21: second immunization; Day 28: obtain first blood sample; Day 42: third immunization; Day 49: obtain second blood sample.</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">______________________________________Results:Immunized   ELISA     against   NeutralizationGuinea      Titers<sup>2</sup>                 env-2     Titer<sup>2</sup>Pigs<sup>1</sup>  Bleed 1   Bleed 2   of Bleed 2______________________________________A            81       n.d.      n.d.B           171       605       &lt;10C           111       n.d.      n.d.D           510       1,274     &lt;10E           501       10,756    220F           n.d.      4,050     40______________________________________ControlGuineaPigs<sup>1</sup>______________________________________A'          &lt;25       &lt;25       &lt;10B'          &lt;25       &lt;25       &lt;10C'          &lt;25       &lt;25       &lt;10D'          &lt;25       &lt;25       &lt;25E'          &lt;25       &lt;25       n.d.F'          &lt;25       &lt;25       n.d.______________________________________ n.d. = not determined <sup>1</sup> sera was heat inactivated <sup>2</sup> dilution that gives 50% of inhibition of viral inoculum (ARV2)</pre>
    
    <p>The sera from guinea pigs E and F were tested for their ability to neutralize various isolates of HIV which have been associated with AIDS.</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">______________________________________        Neutralizing                   NeutralizingHIV          Titer of Guinea                   Titer of GuineaIsolate      Pig E, Bleed 2                   Pig F, Bleed 2______________________________________LAV<sup>1</sup>     70        30ARV-2<sup>2</sup>  220        40ARV-3<sup>3</sup>  &lt;10        n.d.ARV-33<sup>3</sup> &lt;10        n.d.HIV-Zr6<sup>4</sup>        &lt;10        n.d.______________________________________ <sup>1</sup> BarreSinoussi et al. (1983) Science 220:868-871. <sup>2</sup> Levy et al. (1984) Science 225:740-842. <sup>3</sup> Staben et al. (1986) in: Vaccines 86, Cold Spring Harbor Laboratory, New York, pp. 345-350. <sup>4</sup> Srinivassen et al. (1987) Gene 52:71-82.</pre>
    
    <heading>8. Deposits of Biological Materials</heading> <p>Samples of organisms that contain HIV clones or express the above-described hTLR polypeptides were deposited at the American Type Culture Collection (ATCC), 12301 Parklawn Drive, Rockville, Md. under the provisions of the Budapest Treaty. The accession numbers and dates of these deposits are listed below.</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">______________________________________              ATCC              AccessionDeposited Material No.      Deposit Date______________________________________-ARV-2(7D) 40143    26 October 1984-ARV-2(8A) 40144    26 October 1984-ARV-2(9B) 40158    25 January 1985E. coli HB101 (pSV7c/env)              67593    23 December 1987E. coli HB101 (pCMV6ARV120tpa)              68249    8 March 1990E. coli D1210 (pGAG25-10)              53246    27 August 1985E. coli D1210 (pSOD/env5b)              74159    12 June 1992E. coli D1210 (pTP31.2)              69014    16 June 1992E. coli D1210 (pII-3)              67549    28 October 1987Saccharomyces cerevisiae P017              20768    27 August 1985(pC1/1-pSP31-GAP-ADH2)Saccharomyces cerevisiae 2150              20769    27 August 1985(pDPC303)Saccharomyces cerevisiae 2150-2-3              20827    23 December 1986(pAB24-GAP-env2)Saccharomyces cerevisiae  2168!              74162    16 June 1992(pBS24/SOD-SFenv4)Saccharomyces cerevisiae AB110              74160    12 June 1992(pC1/1-p25-ADH-GAP)Saccharomyces cerevisiae AB116              74158    11 June 1992(pBS24.1/SOD-SF2env4-5)Saccharomyces cerevisiae              74163    17 June 1992(pAB24/RT4)Saccharomyces cerevisiae JSC302              74161    16 June 1992______________________________________</pre>
    
    <p>These deposits are provided for the convenience of those skilled in the art. These deposits are neither an admission that such deposits are required to practice the present invention, nor that equivalent embodiments are not within the skill of the art in view of the present disclosure. The public availability of these deposits is not a grant of a license to make, use or sell the deposited materials under this or and other patent. The nucleic acid sequences of the deposited materials are incorporated in the present disclosure by reference and are controlling if in conflict with any sequence described herein.</p>
    <p>Although the foregoing invention has been described in some detail by way of illustration and example for purposes of clarity of understanding, it will be obvious that changes and modifications may be practiced within the scope of the appended claims.</p>
    </div></div></div><div class="patent-section patent-tabular-section"><a id="backward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Cited Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4708818">US4708818</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Oct 8, 1985</td><td class="patent-data-table-td patent-date-value">Nov 24, 1987</td><td class="patent-data-table-td ">Institut Pasteur</td><td class="patent-data-table-td ">Human immunodeficiency viruses associated with Acquired Immune Deficiency Syndrome (AIDS), a diagnostic method for AIDS and pre-AIDS, and a kit therefor</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4740461">US4740461</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Dec 27, 1983</td><td class="patent-data-table-td patent-date-value">Apr 26, 1988</td><td class="patent-data-table-td ">Genetics Institute, Inc.</td><td class="patent-data-table-td ">Vectors and methods for transformation of eucaryotic cells</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4981790">US4981790</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Dec 6, 1985</td><td class="patent-data-table-td patent-date-value">Jan 1, 1991</td><td class="patent-data-table-td ">Dana Farber Cancer Institute</td><td class="patent-data-table-td ">Genetic engineering</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4988624">US4988624</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jul 3, 1986</td><td class="patent-data-table-td patent-date-value">Jan 29, 1991</td><td class="patent-data-table-td ">Kanegafuchi Kagaku Kogyo Kabushiki Kaisha</td><td class="patent-data-table-td ">Antitumor agents</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5024939">US5024939</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Sep 25, 1987</td><td class="patent-data-table-td patent-date-value">Jun 18, 1991</td><td class="patent-data-table-td ">Genentech, Inc.</td><td class="patent-data-table-td ">Transient expression system for producing recombinant protein</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5156949">US5156949</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Dec 24, 1987</td><td class="patent-data-table-td patent-date-value">Oct 20, 1992</td><td class="patent-data-table-td ">Chiron Corporation</td><td class="patent-data-table-td ">Immunoassays for antibody to human immunodeficiency virus using recombinant antigens</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5168062">US5168062</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Sep 10, 1990</td><td class="patent-data-table-td patent-date-value">Dec 1, 1992</td><td class="patent-data-table-td ">University Of Iowa Research Foundation</td><td class="patent-data-table-td ">Transfer vectors and microorganisms containing human cytomegalovirus immediate-early promoter-regulatory DNA sequence</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP0173177B1?cl=en">EP0173177B1</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Aug 16, 1985</td><td class="patent-data-table-td patent-date-value">Apr 22, 1992</td><td class="patent-data-table-td ">BEHRINGWERKE Aktiengesellschaft</td><td class="patent-data-table-td ">Enhancer for eukaryotic expression systems</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP0260148A2?cl=en">EP0260148A2</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Sep 11, 1987</td><td class="patent-data-table-td patent-date-value">Mar 16, 1988</td><td class="patent-data-table-td ">Genentech, Inc.</td><td class="patent-data-table-td ">Improved recombinant expression method, vector and transformed cells</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP0323997A1?cl=en">EP0323997A1</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jul 22, 1988</td><td class="patent-data-table-td patent-date-value">Jul 19, 1989</td><td class="patent-data-table-td ">Celltech Limited</td><td class="patent-data-table-td ">Recombinant dna expression vectors</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="npl-citations"></a><div class="patent-section-header"><span class="patent-section-title">Non-Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th colspan="3"class="patent-data-table-th">Reference</th></tr></thead><tr><td class="patent-data-table-td ">1</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Boshart M; Weber F; Jahn G; Dorsch Hasler K; Fleckenstein B; Schaffner W, A very strong enhancer is located upstream of an immediate early gene of human cytomegalovirus, Cell, (Jun. 1985) 41 (2) 521 30.</td></tr><tr><td class="patent-data-table-td ">2</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Boshart M; Weber F; Jahn G; Dorsch-Hasler K; Fleckenstein B; Schaffner W, "<a href='http://scholar.google.com/scholar?q="A+very+strong+enhancer+is+located+upstream+of+an+immediate+early+gene+of+human+cytomegalovirus%2C"'>A very strong enhancer is located upstream of an immediate early gene of human cytomegalovirus,</a>" Cell, (Jun. 1985) 41 (2) 521-30.</td></tr><tr><td class="patent-data-table-td ">3</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Burke et al., J. of Biol. Chem. (1986) "<a href='http://scholar.google.com/scholar?q="The+Functional+Domains+of+Coagulation+Factor+VIII%3AC"'>The Functional Domains of Coagulation Factor VIII:C</a>" 261:12574-78.</td></tr><tr><td class="patent-data-table-td ">4</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Burke et al., J. of Biol. Chem. (1986) The Functional Domains of Coagulation Factor VIII:C 261:12574 78.</td></tr><tr><td class="patent-data-table-td ">5</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Chapman et al., Nucleic Acids Research (1991) "<a href='http://scholar.google.com/scholar?q="Effect+of+intron+A+from+human+cytomegalovirus+%28Towne%29+immediate-early+gene+on+heterlogous+expression+in+mammalian+cells"'>Effect of intron A from human cytomegalovirus (Towne) immediate-early gene on heterlogous expression in mammalian cells</a>" 19(14):3979-3986.</td></tr><tr><td class="patent-data-table-td ">6</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Chapman et al., Nucleic Acids Research (1991) Effect of intron A from human cytomegalovirus (Towne) immediate early gene on heterlogous expression in mammalian cells 19(14):3979 3986.</td></tr><tr><td class="patent-data-table-td ">7</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Copeland et al., "<a href='http://scholar.google.com/scholar?q="Complete+Amino+Acid+Seq+.+.+.+%2C"'>Complete Amino Acid Seq . . . ,</a>" FEBS Lett 162:390-5 (1983).</td></tr><tr><td class="patent-data-table-td ">8</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Copeland et al., Complete Amino Acid Seq . . . , FEBS Lett 162:390 5 (1983).</td></tr><tr><td class="patent-data-table-td ">9</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Foecking M K; Hofstetter H, "<a href='http://scholar.google.com/scholar?q="Powerful+and+versatile+enhancer-promoter+unit+for+mammalian+expression+vectors%2C"'>Powerful and versatile enhancer-promoter unit for mammalian expression vectors,</a>" Gene, (1986) 45 (1) 101-5.</td></tr><tr><td class="patent-data-table-td ">10</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Foecking M K; Hofstetter H, Powerful and versatile enhancer promoter unit for mammalian expression vectors, Gene, (1986) 45 (1) 101 5.</td></tr><tr><td class="patent-data-table-td ">11</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Gorman C M; Moffat L F; Howard B H, "<a href='http://scholar.google.com/scholar?q="Recombinant+genomes+which+express+chloramphenicol+acetyltransferase+in+mammalian+cells%2C"'>Recombinant genomes which express chloramphenicol acetyltransferase in mammalian cells,</a>" Mol Cell Biol, (Sep. 1982) 2 (9) 1044-51.</td></tr><tr><td class="patent-data-table-td ">12</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Gorman C M; Moffat L F; Howard B H, Recombinant genomes which express chloramphenicol acetyltransferase in mammalian cells, Mol Cell Biol, (Sep. 1982) 2 (9) 1044 51.</td></tr><tr><td class="patent-data-table-td ">13</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">LeFemina et al. (1980) in Fields, et al. (eds.) Animal Virus Genetics , Structural Organization of the DNA Molecules from Human Cytomegalovirus Acad. Press, N.Y., 18:39 55.</td></tr><tr><td class="patent-data-table-td ">14</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">LeFemina et al. (1980) in Fields, et al. (eds.) Animal Virus Genetics, "<a href='http://scholar.google.com/scholar?q="Structural+Organization+of+the+DNA+Molecules+from+Human+Cytomegalovirus"'>Structural Organization of the DNA Molecules from Human Cytomegalovirus</a>" Acad. Press, N.Y., 18:39-55.</td></tr><tr><td class="patent-data-table-td ">15</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Lusky et al., Cell (1984) "<a href='http://scholar.google.com/scholar?q="Characterization+of+the+Bovine+Papilloma+Virus+Plasmid+Maintenance+Sequences"'>Characterization of the Bovine Papilloma Virus Plasmid Maintenance Sequences</a>" 36:391-401.</td></tr><tr><td class="patent-data-table-td ">16</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Lusky et al., Cell (1984) Characterization of the Bovine Papilloma Virus Plasmid Maintenance Sequences 36:391 401.</td></tr><tr><td class="patent-data-table-td ">17</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Myers et al., Cell (1981) "<a href='http://scholar.google.com/scholar?q="SV40+Gene+Expression+is+Modulated+by+the+Cooperative+Binding+of+T+Antigen+to+DNA"'>SV40 Gene Expression is Modulated by the Cooperative Binding of T Antigen to DNA</a>" 25:373-384.</td></tr><tr><td class="patent-data-table-td ">18</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Myers et al., Cell (1981) SV40 Gene Expression is Modulated by the Cooperative Binding of T Antigen to DNA 25:373 384.</td></tr><tr><td class="patent-data-table-td ">19</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Reitz et al., "<a href='http://scholar.google.com/scholar?q="Characterization+and+Distribution+of+Nucleic+.+.+.+%2C"'>Characterization and Distribution of Nucleic . . . ,</a>" Proc Natl. Acad Sci (USA) 78:1887-1891 (1981).</td></tr><tr><td class="patent-data-table-td ">20</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Reitz et al., Characterization and Distribution of Nucleic . . . , Proc Natl. Acad Sci (USA) 78:1887 1891 (1981).</td></tr><tr><td class="patent-data-table-td ">21</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Rio et al., Cell (1983) "<a href='http://scholar.google.com/scholar?q="SV40+T+Antigen+Binding+Site+Mutations+That+Affect+Autoregulation"'>SV40 T Antigen Binding Site Mutations That Affect Autoregulation</a>" 32:1227-1240.</td></tr><tr><td class="patent-data-table-td ">22</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Rio et al., Cell (1983) SV40 T Antigen Binding Site Mutations That Affect Autoregulation 32:1227 1240.</td></tr><tr><td class="patent-data-table-td ">23</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Seiki et al., "<a href='http://scholar.google.com/scholar?q="Human+Adult+T-Cell+Leukemia+.+.+.+"'>Human Adult T-Cell Leukemia . . . </a>", Proc Natl Acad Sci 80:3618-3622 (1983).</td></tr><tr><td class="patent-data-table-td ">24</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Seiki et al., Human Adult T Cell Leukemia . . . , Proc Natl Acad Sci 80:3618 3622 (1983).</td></tr><tr><td class="patent-data-table-td ">25</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Stenberg R M; Thomsen D R; Stinski M F, "<a href='http://scholar.google.com/scholar?q="Structural+analysis+of+the+major+immediate+early+gene+of+human+cytomegalovirus%2C"'>Structural analysis of the major immediate early gene of human cytomegalovirus,</a>" J Virol, (Jan. 1984) 49 (1) 190-9.</td></tr><tr><td class="patent-data-table-td ">26</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Stenberg R M; Thomsen D R; Stinski M F, Structural analysis of the major immediate early gene of human cytomegalovirus, J Virol, (Jan. 1984) 49 (1) 190 9.</td></tr><tr><td class="patent-data-table-td ">27</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Stern, "<a href='http://scholar.google.com/scholar?q="Predicting+Antigenic+Sites+on+Proteins%2C"'>Predicting Antigenic Sites on Proteins,</a>" Tibtech 9:163-168 (1991).</td></tr><tr><td class="patent-data-table-td ">28</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Stern, Predicting Antigenic Sites on Proteins, Tibtech 9:163 168 (1991).</td></tr><tr><td class="patent-data-table-td ">29</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Stinski M F; Thomsen D R; Stenberg R M; Goldstein L C, "<a href='http://scholar.google.com/scholar?q="Organization+and+expression+of+the+immediate+early+genes+of+human+cytomegalovirus%2C"'>Organization and expression of the immediate early genes of human cytomegalovirus,</a>" J Virol, (Apr. 1983) 46 (1) 1-14.</td></tr><tr><td class="patent-data-table-td ">30</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Stinski M F; Thomsen D R; Stenberg R M; Goldstein L C, Organization and expression of the immediate early genes of human cytomegalovirus, J Virol, (Apr. 1983) 46 (1) 1 14.</td></tr><tr><td class="patent-data-table-td ">31</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Subramani et al., Molecular and Cellular Biology (1981) "<a href='http://scholar.google.com/scholar?q="Expression+of+the+Mouse+Dihydrofolate+Reductase+Complementary+Deoxyribonucleic+Acid+in+Simian+Virus+40+Vectors"'>Expression of the Mouse Dihydrofolate Reductase Complementary Deoxyribonucleic Acid in Simian Virus 40 Vectors</a>" 1:854-864.</td></tr><tr><td class="patent-data-table-td ">32</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Subramani et al., Molecular and Cellular Biology (1981) Expression of the Mouse Dihydrofolate Reductase Complementary Deoxyribonucleic Acid in Simian Virus 40 Vectors 1:854 864.</td></tr><tr><td class="patent-data-table-td ">33</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Thomsen D R; Stenberg R M; Goins W F; Stinski M F, "<a href='http://scholar.google.com/scholar?q="Promoter-regulatory+region+of+the+major+immediate+early+gene+of+human+cytomegalovirus%2C"'>Promoter-regulatory region of the major immediate early gene of human cytomegalovirus,</a>" Proc Natl Acad Sci U S A, (Feb. 1984) 81 (3) 659-63.</td></tr><tr><td class="patent-data-table-td ">34</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Thomsen D R; Stenberg R M; Goins W F; Stinski M F, Promoter regulatory region of the major immediate early gene of human cytomegalovirus, Proc Natl Acad Sci U S A, (Feb. 1984) 81 (3) 659 63.</td></tr><tr><td class="patent-data-table-td ">35</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Ulmer et al., Science (1993) "<a href='http://scholar.google.com/scholar?q="Heterologous+Protection+Against+Influenza+by+Injection+of+DNA+Encoding+a+Viral+Protein"'>Heterologous Protection Against Influenza by Injection of DNA Encoding a Viral Protein</a>" 259:1745-49.</td></tr><tr><td class="patent-data-table-td ">36</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Ulmer et al., Science (1993) Heterologous Protection Against Influenza by Injection of DNA Encoding a Viral Protein 259:1745 49.</td></tr><tr><td class="patent-data-table-td ">37</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">van Zonneveld A J; Veerman H; Pannekoek H, "<a href='http://scholar.google.com/scholar?q="Autonomous+functions+of+structural+domains+on+human+tissue-type+plasminogen+activator%2C"'>Autonomous functions of structural domains on human tissue-type plasminogen activator,</a>" Proc Natl Acad Sci U S A, (Jul., 1986) 83 (13) 4670-4.</td></tr><tr><td class="patent-data-table-td ">38</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">van Zonneveld A J; Veerman H; Pannekoek H, Autonomous functions of structural domains on human tissue type plasminogen activator, Proc Natl Acad Sci U S A, (Jul., 1986) 83 (13) 4670 4.</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="forward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Referenced by</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Citing Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6451595">US6451595</a></td><td class="patent-data-table-td patent-date-value">Jul 24, 1999</td><td class="patent-data-table-td patent-date-value">Sep 17, 2002</td><td class="patent-data-table-td ">Viromed Limited</td><td class="patent-data-table-td ">Murine leukemia virus with pol, env, and gag coding regions deleted; gene therapy; gene expression and engineering</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6534285">US6534285</a></td><td class="patent-data-table-td patent-date-value">Apr 12, 2000</td><td class="patent-data-table-td patent-date-value">Mar 18, 2003</td><td class="patent-data-table-td ">Genentech, Inc.</td><td class="patent-data-table-td ">Composition comprising predetermined polypeptide sequence of AIDS-associated retrovirus comprising at least one antigenic determinant that specifically binds complementary antibody</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6602705">US6602705</a></td><td class="patent-data-table-td patent-date-value">Dec 30, 1999</td><td class="patent-data-table-td patent-date-value">Aug 5, 2003</td><td class="patent-data-table-td ">Chiron Corporation</td><td class="patent-data-table-td ">Expression of HIV polypeptides and production of virus-like particles</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6893840">US6893840</a></td><td class="patent-data-table-td patent-date-value">Oct 12, 2001</td><td class="patent-data-table-td patent-date-value">May 17, 2005</td><td class="patent-data-table-td ">Chiron Corporation</td><td class="patent-data-table-td ">Cytomegalovirus Intron A fragments</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7074590">US7074590</a></td><td class="patent-data-table-td patent-date-value">Jun 21, 2001</td><td class="patent-data-table-td patent-date-value">Jul 11, 2006</td><td class="patent-data-table-td ">Maxygen, Inc.</td><td class="patent-data-table-td ">Chimeric promoters</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7348177">US7348177</a></td><td class="patent-data-table-td patent-date-value">Mar 11, 2003</td><td class="patent-data-table-td patent-date-value">Mar 25, 2008</td><td class="patent-data-table-td ">Novartis Vaccines And Diagnostics, Inc.</td><td class="patent-data-table-td ">Expression of HIV polypeptides and production of virus-like particles</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7662916">US7662916</a></td><td class="patent-data-table-td patent-date-value">Jul 25, 2003</td><td class="patent-data-table-td patent-date-value">Feb 16, 2010</td><td class="patent-data-table-td ">Novartis Vaccines &amp; Diagnostics, Inc</td><td class="patent-data-table-td ">Modified HIV Env polypeptides</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7718401">US7718401</a></td><td class="patent-data-table-td patent-date-value">Feb 6, 2008</td><td class="patent-data-table-td patent-date-value">May 18, 2010</td><td class="patent-data-table-td ">Novartis Vaccines And Diagnostics, Inc.</td><td class="patent-data-table-td ">Expression of HIV polypeptides and production of virus-like particles</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7846687">US7846687</a></td><td class="patent-data-table-td patent-date-value">Apr 11, 2005</td><td class="patent-data-table-td patent-date-value">Dec 7, 2010</td><td class="patent-data-table-td ">Novartis Vaccines &amp; Diagnostics, Inc.</td><td class="patent-data-table-td ">Modified viral sequences which enhance genic expression in microorganismal host</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7935805">US7935805</a></td><td class="patent-data-table-td patent-date-value">Jul 5, 2000</td><td class="patent-data-table-td patent-date-value">May 3, 2011</td><td class="patent-data-table-td ">Novartis Vaccines &amp; Diagnostics, Inc</td><td class="patent-data-table-td ">Polynucleotides encoding antigenic HIV Type C polypeptides, polypeptides and uses thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7943375">US7943375</a></td><td class="patent-data-table-td patent-date-value">Jun 25, 2008</td><td class="patent-data-table-td patent-date-value">May 17, 2011</td><td class="patent-data-table-td ">Novartis Vaccines &amp; Diagnostics, Inc</td><td class="patent-data-table-td ">Polynucleotides encoding antigenic HIV type C polypeptides, polypeptides and uses thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8168418">US8168418</a></td><td class="patent-data-table-td patent-date-value">Apr 30, 2010</td><td class="patent-data-table-td patent-date-value">May 1, 2012</td><td class="patent-data-table-td ">Susan W Barnett</td><td class="patent-data-table-td ">Expression of HIV polypeptides and production of virus-like particles</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8263394">US8263394</a></td><td class="patent-data-table-td patent-date-value">Jan 23, 2008</td><td class="patent-data-table-td patent-date-value">Sep 11, 2012</td><td class="patent-data-table-td ">Novartis Vaccines &amp; Diagnostics Inc.</td><td class="patent-data-table-td ">Polynucleotides encoding antigenic HIV type B polypeptides, polypeptides, and uses thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8541230">US8541230</a></td><td class="patent-data-table-td patent-date-value">May 16, 2011</td><td class="patent-data-table-td patent-date-value">Sep 24, 2013</td><td class="patent-data-table-td ">Novartis Vaccines And Diagnostics, Inc</td><td class="patent-data-table-td ">Polynucleotides encoding antigenic HIV type C polypeptides, polypeptides and uses thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/USRE41158">USRE41158</a></td><td class="patent-data-table-td patent-date-value">Jan 12, 2009</td><td class="patent-data-table-td patent-date-value">Mar 2, 2010</td><td class="patent-data-table-td ">Novartis Vaccines And Diagnostics, Inc.</td><td class="patent-data-table-td ">Diagnostic probes</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2196472A2?cl=en">EP2196472A2</a></td><td class="patent-data-table-td patent-date-value">Apr 14, 2000</td><td class="patent-data-table-td patent-date-value">Jun 16, 2010</td><td class="patent-data-table-td ">Novartis Vaccines and Diagnostics, Inc.</td><td class="patent-data-table-td ">Compositions and methods for generating an immune response utilizing alphavirus-based vector systems</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2206785A1?cl=en">EP2206785A1</a></td><td class="patent-data-table-td patent-date-value">Dec 30, 1999</td><td class="patent-data-table-td patent-date-value">Jul 14, 2010</td><td class="patent-data-table-td ">Novartis Vaccines and Diagnostics, Inc.</td><td class="patent-data-table-td ">Improved expression of HIV polypeptides and production of virus-like particles</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2385065A1?cl=en">EP2385065A1</a></td><td class="patent-data-table-td patent-date-value">Oct 15, 2008</td><td class="patent-data-table-td patent-date-value">Nov 9, 2011</td><td class="patent-data-table-td ">Perseid Therapeutics LLC</td><td class="patent-data-table-td ">Immunosuppressive polypeptides and nucleic acids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2612867A1?cl=en">EP2612867A1</a></td><td class="patent-data-table-td patent-date-value">Oct 15, 2008</td><td class="patent-data-table-td patent-date-value">Jul 10, 2013</td><td class="patent-data-table-td ">Perseid Therapeutics LLC</td><td class="patent-data-table-td ">Immunosuppressive polypeptides and nucleic acids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2612868A1?cl=en">EP2612868A1</a></td><td class="patent-data-table-td patent-date-value">Oct 15, 2008</td><td class="patent-data-table-td patent-date-value">Jul 10, 2013</td><td class="patent-data-table-td ">Perseid Therapeutics LLC</td><td class="patent-data-table-td ">Immunosuppressive polypeptides and nucleic acids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2000000629A1?cl=en">WO2000000629A1</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jun 24, 1999</td><td class="patent-data-table-td patent-date-value">Jan 6, 2000</td><td class="patent-data-table-td ">Kim Jong Mook</td><td class="patent-data-table-td ">High efficiency retroviral vectors that contain none of viral coding sequences</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2002079470A1?cl=en">WO2002079470A1</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Mar 29, 2002</td><td class="patent-data-table-td patent-date-value">Oct 10, 2002</td><td class="patent-data-table-td ">A Christie King</td><td class="patent-data-table-td ">Disease-specific promoter chimeras for gene expression</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2009058564A2?cl=en">WO2009058564A2</a></td><td class="patent-data-table-td patent-date-value">Oct 15, 2008</td><td class="patent-data-table-td patent-date-value">May 7, 2009</td><td class="patent-data-table-td ">Maxygen Inc</td><td class="patent-data-table-td ">Immunosuppressive polypeptides and nucleic acids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2013164754A2?cl=en">WO2013164754A2</a></td><td class="patent-data-table-td patent-date-value">Apr 29, 2013</td><td class="patent-data-table-td patent-date-value">Nov 7, 2013</td><td class="patent-data-table-td ">Pfizer Inc.</td><td class="patent-data-table-td ">Prostate-associated antigens and vaccine-based immunotherapy regimens</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="classifications"></a><div class="patent-section-header"><span class="patent-section-title">Classifications</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th"> </th><th class="patent-data-table-th"> </th></tr></thead><tr><td class="patent-data-table-td ">U.S. Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=QIBEBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S456000">435/456</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=QIBEBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S023720">536/23.72</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=QIBEBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S455000">435/455</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=QIBEBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S974000">435/974</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=QIBEBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S320100">435/320.1</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=QIBEBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S069300">435/69.3</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=QIBEBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S091410">435/91.41</a></span></td></tr><tr><td class="patent-data-table-td ">International Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=QIBEBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12Q0001680000">C12Q1/68</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=QIBEBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=A61P0031140000">A61P31/14</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=QIBEBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C07K0014195000">C07K14/195</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=QIBEBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=A61P0037020000">A61P37/02</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=QIBEBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C07K0016000000">C07K16/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=QIBEBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=A61K0039395000">A61K39/395</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=QIBEBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0001210000">C12N1/21</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=QIBEBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0001190000">C12N1/19</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=QIBEBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0007020000">C12N7/02</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=QIBEBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=A61K0031000000">A61K31/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=QIBEBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C07H0021040000">C07H21/04</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=QIBEBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=A61P0031120000">A61P31/12</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=QIBEBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=G01N0033569000">G01N33/569</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=QIBEBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0015090000">C12N15/09</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=QIBEBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=A61P0037000000">A61P37/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=QIBEBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0007000000">C12N7/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=QIBEBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12R0001190000">C12R1/19</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=QIBEBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0015480000">C12N15/48</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=QIBEBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12P0021020000">C12P21/02</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=QIBEBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=A61P0031180000">A61P31/18</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=QIBEBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0015000000">C12N15/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=QIBEBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C07K0014005000">C07K14/005</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=QIBEBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C07K0014155000">C07K14/155</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=QIBEBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C07K0019000000">C07K19/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=QIBEBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=A61K0039210000">A61K39/21</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=QIBEBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=G01N0033577000">G01N33/577</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=QIBEBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0015490000">C12N15/49</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=QIBEBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=A61P0031000000">A61P31/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=QIBEBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12R0001865000">C12R1/865</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=QIBEBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=G01N0033530000">G01N33/53</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=QIBEBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=A61K0038000000">A61K38/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=QIBEBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0015810000">C12N15/81</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=QIBEBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0015850000">C12N15/85</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=QIBEBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C07K0014160000">C07K14/16</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=QIBEBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12Q0001700000">C12Q1/70</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=QIBEBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0009020000">C12N9/02</a></span></td></tr><tr><td class="patent-data-table-td ">Cooperative Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=QIBEBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=Y10S435/82">Y10S435/82</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=QIBEBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=Y10S530/826">Y10S530/826</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=QIBEBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=Y10S435/974">Y10S435/974</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=QIBEBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=Y10S435/81">Y10S435/81</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=QIBEBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=Y10S435/975">Y10S435/975</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=QIBEBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N2830/36">C12N2830/36</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=QIBEBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C07K2319/00">C07K2319/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=QIBEBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N2830/002">C12N2830/002</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=QIBEBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N2830/704">C12N2830/704</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=QIBEBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N9/0089">C12N9/0089</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=QIBEBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N2740/16222">C12N2740/16222</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=QIBEBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N2830/55">C12N2830/55</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=QIBEBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C07K2319/003">C07K2319/003</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=QIBEBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C07K14/005">C07K14/005</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=QIBEBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C07K2319/40">C07K2319/40</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=QIBEBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N15/81">C12N15/81</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=QIBEBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C07K2319/735">C07K2319/735</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=QIBEBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N2740/16122">C12N2740/16122</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=QIBEBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N15/85">C12N15/85</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=QIBEBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N2830/15">C12N2830/15</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=QIBEBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N2800/108">C12N2800/108</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=QIBEBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N2830/702">C12N2830/702</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=QIBEBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=A61K38/00">A61K38/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=QIBEBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N2830/005">C12N2830/005</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=QIBEBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12Q1/703">C12Q1/703</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=QIBEBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C07K2319/75">C07K2319/75</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=QIBEBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N2830/00">C12N2830/00</a></span></td></tr><tr><td class="patent-data-table-td ">European Classification</td><td class="patent-data-table-td "><span class="nested-value">C07K14/005</span>, <span class="nested-value">C12N9/00P32</span>, <span class="nested-value">C12N15/81</span>, <span class="nested-value">C12N15/85</span>, <span class="nested-value">C12Q1/70B2B</span></td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="legal-events"></a><div class="patent-section-header"><span class="patent-section-title">Legal Events</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Date</th><th class="patent-data-table-th">Code</th><th class="patent-data-table-th">Event</th><th class="patent-data-table-th">Description</th></tr></thead><tr><td class="patent-data-table-td patent-date-value">Jan 27, 2011</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20060419</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">MERGER;ASSIGNOR:CHIRON CORPORATION;REEL/FRAME:025706/0431</span></div><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">NOVARTIS VACCINES AND DIAGNOSTICS, INC., CALIFORNI</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Dec 22, 2009</td><td class="patent-data-table-td ">B1</td><td class="patent-data-table-td ">Reexamination certificate first reexamination</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">CLAIMS 1-13, 15 AND 17 ARE DETERMINED TO BE PATENTABLE AS AMENDED. CLAIMS 14, 16 AND 18-24, DEPENDENT ON AN AMENDED CLAIM, ARE DETERMINED TO BE PATENTABLE.</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Apr 15, 2009</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">12</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Nov 7, 2006</td><td class="patent-data-table-td ">RR</td><td class="patent-data-table-td ">Request for reexamination filed</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20060425</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Apr 27, 2005</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">8</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">May 17, 2001</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">4</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Apr 11, 1997</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">CHIRON CORPORATION, CALIFORNIA</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNORS:ROSENBERG, STEVEN;CHAPMAN, BARBARA S.;THAYER, RICHARD M.;AND OTHERS;REEL/FRAME:008444/0009;SIGNING DATES FROM 19970402 TO 19970407</span></div></td></tr></table><div class="patent-section-footer"></div></div><div class="modal-dialog" id="patent-images-lightbox"><div class="patent-lightbox-controls"><div class="patent-lightbox-rotate-controls"><div class="patent-lightbox-rotation-text">Rotate</div><div class="rotate-icon rotate-ccw-icon"></div><div class="rotate-icon rotate-cw-icon"></div></div><div class="patent-lightbox-index-counter"></div><a class="patent-lightbox-fullsize-link" target="_blank">Original Image</a><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_right.png" alt="Next page"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_left.png" alt="Previous page"width="21" height="21" /></div></div><div class="modal-dialog-content"><div class="patent-lightbox-image-holder"><div class="patent-lightbox-placeholder"></div></div></div></div><script>_OC_initPatentsAtb({image_not_available_html: " Image not available"});</script></div></div></div></td></tr></table><script>(function() {var href = window.location.href;if (href.indexOf('?') !== -1) {var parameters = href.split('?')[1].split('&');for (var i = 0; i < parameters.length; i++) {var param = parameters[i].split('=');if (param[0] == 'focus') {var elem = document.getElementById(param[1]);if (elem) {elem.focus();}}}}})();</script><script>_OC_addFlags({LockSrc:"/books/javascript/lock_8a2b04e7bf975d5171d8e4c0b6365c7a.js", Host:"http://www.google.com/", IsBooksRentalEnabled:1, IsWebstoreDisplayCaseEnabled:1, IsObfuscationEnabled:1, IsBrowsingHistoryEnabled:1, IsWebReaderSvgEnabled:0, IsGeoLayerEnabled:1, IsImageModeNotesEnabled:1, IsCopyMenuItemEnabled:1, IsGiftingEnabled:0, IsWebReaderUniversalPaginatorEnabled:0, IsOfflineBubbleEnabled:1, IsReaderEnabledForPlayRequests:1, IsFutureOnSaleVolumesEnabled:1, IsOfflineRestrictedCopyEnabled:1, IsBooksUnifiedLeftNavEnabled:1, IsRestrictedCopyEnabled:1, IsZipitFolderCollectionEnabled:1, IsEndOfSampleRecommendationsEnabled:1, IsRatingsOnBookcardsEnabled:1, IsAdsDisabled:0, IsIframePageDisplayEnabled:0, IsEmbeddedMediaEnabled:1, IsImageModeAnnotationsEnabled:1, IsMyLibraryGooglePlusEnabled:1, IsImagePageProviderEnabled:0, IsBookcardListPriceSmall:0, IsInternalUser:0, IsBooksShareButtonEnabled:0, IsPreOrdersEnabled:0, IsDisabledRandomBookshelves:0, WebstoreDisplayCasePosition:3});_OC_Run({"enable_p13n":false,"add_vol_to_collection_base_url":"http://www.google.com/patents?op=add\u0026sig=ACfU3U2-FRblFlKSb4iUketzy6i19Gt7QA\u0026id=QIBEBAABERAJ","remove_vol_from_collection_base_url":"http://www.google.com/patents?op=remove\u0026sig=ACfU3U18YTgljmFmNmrdKNJJMPbMZniA-w\u0026id=QIBEBAABERAJ","logged_in":false,"p13n_save_user_settings_url":"http://www.google.com/patents?op=edit_user_settings\u0026sig=ACfU3U20_-rl9d3bQW7AYaeepo7QYAmE6Q","is_cobrand":false,"sign_in_url":"https://www.google.com/accounts/Login?service=\u0026continue=http://www.google.com/patents%3Fhl%3Den\u0026hl=en","is_play_enabled":true}, {"volume_id":"","is_ebook":true,"volumeresult":{"has_flowing_text":false,"has_scanned_text":true,"can_download_pdf":false,"can_download_epub":false,"is_pdf_drm_enabled":false,"is_epub_drm_enabled":false,"download_pdf_url":"http://www.google.com/patents/download/Vector_for_expression_of_a_polypeptide_i.pdf?id=QIBEBAABERAJ\u0026output=pdf\u0026sig=ACfU3U1pfrr2JH526rbDOXyh88iVPrZNnA"},"sample_url":"http://www.google.com/patents/reader?id=QIBEBAABERAJ\u0026printsec=frontcover\u0026output=reader\u0026source=gbs_atb_hover","is_browsable":true,"is_public_domain":true}, {});</script><div id="footer_table" style="font-size:83%;text-align:center;position:relative;top:20px;height:4.5em;margin-top:2em"><div style="margin-bottom:8px"><a href=http://www.google.com/><nobr>Google&nbsp;Home</nobr></a> - <a href=//www.google.com/patents/sitemap/><nobr>Sitemap</nobr></a> - <a href=http://www.google.com/googlebooks/uspto.html><nobr>USPTO Bulk Downloads</nobr></a> - <a href=/intl/en/privacy/><nobr>Privacy Policy</nobr></a> - <a href=/intl/en/policies/terms/><nobr>Terms of Service</nobr></a> - <a href=https://support.google.com/faqs/answer/2539193?hl=en><nobr>About Google Patents</nobr></a> - <a href="http://www.google.com/tools/feedback/intl/en/error.html" onclick="try{_OC_startFeedback({productId: '72792',locale: 'en'});return false;}catch(e){}"><nobr>Send Feedback</nobr></a></div><span>Data provided by IFI CLAIMS Patent Services</span><br><span >&copy;2012 Google</span></div> <script type="text/javascript">var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));</script><script type="text/javascript">var pageTracker = _gat._getTracker("UA-27188110-1");pageTracker._setCookiePath("/patents/");pageTracker._trackPageview();</script> </body></html>