<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN""http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html><head><title>Patent US5700676 - Modified subtilisins having amino acid alterations - Google Patents</title><script>(function(){(function(){function e(a){this.t={};this.tick=function(a,c,b){var d=void 0!=b?b:(new Date).getTime();this.t[a]=[d,c];if(void 0==b)try{window.console.timeStamp("CSI/"+a)}catch(e){}};this.tick("start",null,a)}var a;window.performance&&(a=window.performance.timing);var f=a?new e(a.responseStart):new e;window.jstiming={Timer:e,load:f};if(a){var c=a.navigationStart,d=a.responseStart;0<c&&d>=c&&(window.jstiming.srt=d-c)}if(a){var b=window.jstiming.load;0<c&&d>=c&&(b.tick("_wtsrt",void 0,c),b.tick("wtsrt_",
"_wtsrt",d),b.tick("tbsd_","wtsrt_"))}try{a=null,window.chrome&&window.chrome.csi&&(a=Math.floor(window.chrome.csi().pageT),b&&0<c&&(b.tick("_tbnd",void 0,window.chrome.csi().startE),b.tick("tbnd_","_tbnd",c))),null==a&&window.gtbExternal&&(a=window.gtbExternal.pageT()),null==a&&window.external&&(a=window.external.pageT,b&&0<c&&(b.tick("_tbnd",void 0,window.external.startE),b.tick("tbnd_","_tbnd",c))),a&&(window.jstiming.pt=a)}catch(g){}})();})();
</script><link rel="stylesheet" href="/patents/css/_8a2b04e7bf975d5171d8e4c0b6365c7a/kl_intl_patents_bundle.css" type="text/css" /><script src="/books/javascript/atb_8a2b04e7bf975d5171d8e4c0b6365c7a__en.js"></script><script>function googleTranslateElementInit() {new google.translate.TranslateElement({pageLanguage: "en",gaTrack: true,gaId: "UA-27188110-1",multilanguagePage: true});}</script><script src="//translate.google.com/translate_a/element.js?cb=googleTranslateElementInit"></script><meta name="DC.type" content="Patent"><meta name="DC.title" content="Modified subtilisins having amino acid alterations"><meta name="DC.contributor" content="Richard Ray Bott" scheme="inventor"><meta name="DC.contributor" content="Robert Mark Caldwell" scheme="inventor"><meta name="DC.contributor" content="Brian C. Cunningham" scheme="inventor"><meta name="DC.contributor" content="David Aaron Estell" scheme="inventor"><meta name="DC.contributor" content="Scott Douglas Power" scheme="inventor"><meta name="DC.contributor" content="James Allen Wells" scheme="inventor"><meta name="DC.contributor" content="Genencor International Inc." scheme="assignee"><meta name="DC.date" content="1995-6-7" scheme="dateSubmitted"><meta name="DC.description" content="Novel carbonyl hydrolase mutants derived from the amino acid sequence of naturally-occurring or recombinant non-human carbonyl hydrolases and DNA sequences encoding the same. The mutant carbonyl hydrolases, in general, are obtained by in vitro modification of a precursor DNA sequence encoding the naturally-occurring or recombinant carbonyl hydrolase to encode the substitution, insertion or deletion of one or more amino acids in the amino acid sequence of a precursor carbonyl hydrolase. Such mutants have one or more properties which are different than the same property of the precursor hydrolase."><meta name="DC.date" content="1997-12-23" scheme="issued"><meta name="DC.relation" content="US:4002572" scheme="references"><meta name="DC.relation" content="US:4458066" scheme="references"><meta name="DC.relation" content="US:4543329" scheme="references"><meta name="DC.relation" content="US:4760025" scheme="references"><meta name="DC.relation" content="US:RE34606" scheme="references"><meta name="DC.relation" content="WO:1985003949:A1" scheme="references"><meta name="citation_reference" content="Abrahmsen, L., et al., &quot;Engineering Subtilisin and Its Substrates for Efficient Ligation of Peptide Bonds in Aqueous Solution.&quot; Biochemistry, 30:4151-4159 (1991)."><meta name="citation_reference" content="Abrahmsen, L., et al., Engineering Subtilisin and Its Substrates for Efficient Ligation of Peptide Bonds in Aqueous Solution. Biochemistry, 30:4151 4159 (1991)."><meta name="citation_reference" content="Allinger, et al., in Organic Chemistry, Worth Publishers, New York, NY p. 757 (1971)."><meta name="citation_reference" content="Baughman, G., et al., &quot;Translational regulation of the L11 ribosomal protein operon of Escherichia coli: Analysis of the mRNA target site using olignucleotide-directed mutagenesis.&quot; Proc. Natl. Acad. Sci. USA, 81:5389-5393 (1984)."><meta name="citation_reference" content="Baughman, G., et al., Translational regulation of the L11 ribosomal protein operon of Escherichia coli: Analysis of the mRNA target site using olignucleotide directed mutagenesis. Proc. Natl. Acad. Sci. USA, 81:5389 5393 (1984)."><meta name="citation_reference" content="Cantor, C.R., et al., &quot;Biophysical Chemistry: Part I: The conformation of biological macromolecules.&quot; W.H. Freeman and Company, San Francisco, pp. 3-154, 253-309 (1980)."><meta name="citation_reference" content="Cantor, C.R., et al., Biophysical Chemistry: Part I: The conformation of biological macromolecules. W.H. Freeman and Company, San Francisco, pp. 3 154, 253 309 (1980)."><meta name="citation_reference" content="Carter, et al., Science, 237:394 399 (1987)."><meta name="citation_reference" content="Carter, et al., Science, 237:394-399 (1987)."><meta name="citation_reference" content="Carter, P., et al., &quot;Improved Oligonucleotide Site-Directed Mutagenesis Using M13 Vectors.&quot; Nucleic Acids Res., 13:4431-4443 (1985)."><meta name="citation_reference" content="Carter, P., et al., Improved Oligonucleotide Site Directed Mutagenesis Using M13 Vectors. Nucleic Acids Res., 13:4431 4443 (1985)."><meta name="citation_reference" content="Carter, P.J., et al., &quot;The Use of Double Mutants to Detect Structural Changes in the Active Site of the Tyrosyl-tRNA Synthetase (Bacillus stearothermophilus).&quot; Cell, 33:835-840 (1984)."><meta name="citation_reference" content="Carter, P.J., et al., Cell, 38:835 840 (1984)."><meta name="citation_reference" content="Carter, P.J., et al., Cell, 38:835-840 (1984)."><meta name="citation_reference" content="Carter, P.J., et al., The Use of Double Mutants to Detect Structural Changes in the Active Site of the Tyrosyl tRNA Synthetase ( Bacillus stearothermophilus ). Cell, 33:835 840 (1984)."><meta name="citation_reference" content="Craik, C.S., et al., Science, 228:291 297 (1985)."><meta name="citation_reference" content="Craik, C.S., et al., Science, 228:291-297 (1985)."><meta name="citation_reference" content="Drenth, J., et al., Eur. J. Biochem., 26, 177 181 (1972)."><meta name="citation_reference" content="Drenth, J., et al., Eur. J. Biochem., 26, 177-181 (1972)."><meta name="citation_reference" content="Estell, D.A., et al., &quot;Probing Steric and Hydrophobic Effects on Enzyme-Substrate Interactions by Protein Engineering.&quot; Science, 233:659-663 (1986)."><meta name="citation_reference" content="Estell, D.A., et al., Probing Steric and Hydrophobic Effects on Enzyme Substrate Interactions by Protein Engineering. Science, 233:659 663 (1986)."><meta name="citation_reference" content="Fersht, A., Enzyme Structure and Mechanism, Freeman, San Francisco. pp. 226 287 (1977)."><meta name="citation_reference" content="Fersht, A., Enzyme Structure and Mechanism, Freeman, San Francisco. pp. 226-287 (1977)."><meta name="citation_reference" content="Gardell, et al., Nature, 317:551 555 (1985)."><meta name="citation_reference" content="Gardell, et al., Nature, 317:551-555 (1985)."><meta name="citation_reference" content="Grantham, R., &quot;Amino Acid Difference Formula to Help Explain Protein Evolution.&quot; Science, 185:962-964 (1974)."><meta name="citation_reference" content="Grantham, R., Amino Acid Difference Formula to Help Explain Protein Evolution. Science, 185:962 964 (1974)."><meta name="citation_reference" content="Inouye, S., et al., &quot;Effects of Mutations at Glycine Residues in the Hydrophobic Region of the Escherichia coli Prolipoprotein Signal Peptide on the Secretion across the Membrane.&quot; J. Biol. Chem., 259:3729-3733 (1984)."><meta name="citation_reference" content="Inouye, S., et al., &quot;Role of positive charge on the amino-terminal region of the signal peptide in protein secretion across the membrane.&quot; Proc. Natl. Acad. Sci. USA, 79:3438-3441 (1982)."><meta name="citation_reference" content="Inouye, S., et al., Effects of Mutations at Glycine Residues in the Hydrophobic Region of the Escherichia coli Prolipoprotein Signal Peptide on the Secretion across the Membrane. J. Biol. Chem., 259:3729 3733 (1984)."><meta name="citation_reference" content="Inouye, S., et al., Role of positive charge on the amino terminal region of the signal peptide in protein secretion across the membrane. Proc. Natl. Acad. Sci. USA, 79:3438 3441 (1982)."><meta name="citation_reference" content="Iverson, et al., Science, 243:1184 1188 (1989)."><meta name="citation_reference" content="Iverson, et al., Science, 243:1184-1188 (1989)."><meta name="citation_reference" content="Jacobs, M., et al., &quot;Cloning, sequencing and expression of subtilisin Carlsberg from Bacillus licheniformis.&quot; Nucleic Acids Res., 13: 8913-8926 (1985)."><meta name="citation_reference" content="Jacobs, M., et al., Cloning, sequencing and expression of subtilisin Carlsberg from Bacillus licheniformis. Nucleic Acids Res., 13: 8913 8926 (1985)."><meta name="citation_reference" content="Jaye, M., et al., &quot;Isolation of a human anti-haemophilic factor IX cDNA clone using a unique 52-base synthetic oligonucleotide probe deduced from the amino acid sequence of bovine factor IX.&quot; Nucleic Acids Res., 11:2325-2335 (1983)."><meta name="citation_reference" content="Jaye, M., et al., Isolation of a human anti haemophilic factor IX cDNA clone using a unique 52 base synthetic oligonucleotide probe deduced from the amino acid sequence of bovine factor IX. Nucleic Acids Res., 11:2325 2335 (1983)."><meta name="citation_reference" content="Jencks, W.P., Catalysis in Chemistry and Enzymology, McGraw Hill, pp. 321 436 (1969)."><meta name="citation_reference" content="Jencks, W.P., Catalysis in Chemistry and Enzymology, McGraw-Hill, pp. 321-436 (1969)."><meta name="citation_reference" content="Kaiser, E.T., et al., Ann. Rev. Biochem., 54:565 595 (1985)."><meta name="citation_reference" content="Kaiser, E.T., et al., Ann. Rev. Biochem., 54:565-595 (1985)."><meta name="citation_reference" content="Kerjan, P., et al., Eur. J. Biochem., 98:353 362 (1979)."><meta name="citation_reference" content="Kerjan, P., et al., Eur. J. Biochem., 98:353-362 (1979)."><meta name="citation_reference" content="Kraut, J., Ann. Rev. Biochem., 46:331 358 (1977)."><meta name="citation_reference" content="Kraut, J., Ann. Rev. Biochem., 46:331-358 (1977)."><meta name="citation_reference" content="Kurihara, M., et al., &quot;Isolation and Sequence of the Chymotryptic Peptides and the Complete Amino Acid Sequence.&quot; J. Biol. Chem., 247:5619-5631 (1972)."><meta name="citation_reference" content="Kurihara, M., et al., Isolation and Sequence of the Chymotryptic Peptides and the Complete Amino Acid Sequence. J. Biol. Chem., 247:5619 5631 (1972)."><meta name="citation_reference" content="Markland, F.S., Carlsberg Res. Comm., 41:237 291 (1976)."><meta name="citation_reference" content="Markland, F.S., Carlsberg Res. Comm., 41:237-291 (1976)."><meta name="citation_reference" content="Markland, F.S., et al., The Enzymes, ed. Boyer, P.D., Acad Press, New York, vol. III, pp. 561 608 (1971)."><meta name="citation_reference" content="Markland, F.S., et al., The Enzymes, ed. Boyer, P.D., Acad Press, New York, vol. III, pp. 561-608 (1971)."><meta name="citation_reference" content="Matthews, D.A., et al., J. Biol. Chem., 250:7120 7126 (1975)."><meta name="citation_reference" content="Matthews, D.A., et al., J. Biol. Chem., 250:7120-7126 (1975)."><meta name="citation_reference" content="Mattoccia, E., et al., &quot;Mutation in the A Block of the Yeast tRNALeu3 Gene that Allows Transcription but Abolishes Splicing and 5&#39;-End Maturation.&quot; Cell, 32:67-76 (1983)."><meta name="citation_reference" content="Mattoccia, E., et al., Mutation in the A Block of the Yeast tRNA Leu 3 Gene that Allows Transcription but Abolishes Splicing and 5 End Maturation. Cell, 32:67 76 (1983)."><meta name="citation_reference" content="Meloun, B., et al., FEBS Letters, vol. 183, &quot;Complete primary structure of thermitase from Thermoactinomyces vulgaris and its structural features related to the subtilisin-type proteinases&quot;, pp. 195-199, 1985."><meta name="citation_reference" content="Meloun, B., et al., FEBS Letters, vol. 183, Complete primary structure of thermitase from Thermoactinomyces vulgaris and its structural features related to the subtilisin type proteinases , pp. 195 199, 1985."><meta name="citation_reference" content="Nedkov, P., et al., Hoppe Seyler s Z. Physiol. Chem., 364:1537 1540 (1983)."><meta name="citation_reference" content="Nedkov, P., et al., Hoppe-Seyler&#39;s Z. Physiol. Chem., 364:1537-1540 (1983)."><meta name="citation_reference" content="Ottesen, et al., Meth. Enzymol., XIX:199 215 (1970)."><meta name="citation_reference" content="Ottesen, et al., Meth. Enzymol., XIX:199-215 (1970)."><meta name="citation_reference" content="Pahler, A., et al., The EMBO Journal, vol. 3, &quot;Three-dimensional structure of fungal proteinase K reveals similarity to bacterial subtilisin&quot;, pp. 1311-1314, 1984."><meta name="citation_reference" content="Pahler, A., et al., The EMBO Journal, vol. 3, Three dimensional structure of fungal proteinase K reveals similarity to bacterial subtilisin , pp. 1311 1314, 1984."><meta name="citation_reference" content="Paterson, A. and Clarke, P.H., J. Gen. Micro. 114:65 85 (1979)."><meta name="citation_reference" content="Paterson, A. and Clarke, P.H., J. Gen. Micro. 114:65-85 (1979)."><meta name="citation_reference" content="Perry, L.J., et al., Science, 226:555 557 (1984)."><meta name="citation_reference" content="Perry, L.J., et al., Science, 226:555-557 (1984)."><meta name="citation_reference" content="Philipp, M., et al., Mol. Cell. Biochem., 51:5 32 (1983)."><meta name="citation_reference" content="Philipp, M., et al., Mol. Cell. Biochem., 51:5-32 (1983)."><meta name="citation_reference" content="Polgar, et al., Adv. Enzymol., 33:381 400 (1970)."><meta name="citation_reference" content="Polgar, et al., Adv. Enzymol., 33:381-400 (1970)."><meta name="citation_reference" content="Polgar, L. and Sajgo, M., Biochem. Biophy. Acta., 667:351 354 (1981)."><meta name="citation_reference" content="Polgar, L. and Sajgo, M., Biochem. Biophy. Acta., 667:351-354 (1981)."><meta name="citation_reference" content="Poulos, T.L., et al., J. Biol. Chem., 251:1097 1103 (1976)."><meta name="citation_reference" content="Poulos, T.L., et al., J. Biol. Chem., 251:1097-1103 (1976)."><meta name="citation_reference" content="Rastetter, W.H., &quot;Enzyme engineering: applications and promise.&quot; Trends Biotechnol., 1:80-84 (1983)."><meta name="citation_reference" content="Rastetter, W.H., Enzyme engineering: applications and promise. Trends Biotechnol., 1:80 84 (1983)."><meta name="citation_reference" content="Ricchelli, F., et al., &quot;Effects of pH and Urea on the Conformational Properties of Subtilisin DY.&quot; Biochemical J., 207:201-205 (1982)."><meta name="citation_reference" content="Ricchelli, F., et al., Effects of pH and Urea on the Conformational Properties of Subtilisin DY. Biochemical J., 207:201 205 (1982)."><meta name="citation_reference" content="Robertus, J.D., et al., &quot;An X-Ray Crystallographic Study of the Binding of Peptide Chloromethyl Ketone INhibitors to Subtilisin BPN&#39;,&quot; Biochemistry, 11:2439-2449 (1972)."><meta name="citation_reference" content="Robertus, J.D., et al., An X Ray Crystallographic Study of the Binding of Peptide Chloromethyl Ketone INhibitors to Subtilisin BPN , Biochemistry, 11:2439 2449 (1972)."><meta name="citation_reference" content="Robertus, J.D., et al., Biochemistry, 11:4293 4303 (1972)."><meta name="citation_reference" content="Robertus, J.D., et al., Biochemistry, 11:4293-4303 (1972)."><meta name="citation_reference" content="Russell, A. J., et al., Journal of Molecular Biology, vol. 193, &quot;Electrostatic effects on modification of charged groups in the active site cleft of subtilisin by protein engineering&quot;, pp. 803-813, 1987."><meta name="citation_reference" content="Russell, A. J., et al., Journal of Molecular Biology, vol. 193, Electrostatic effects on modification of charged groups in the active site cleft of subtilisin by protein engineering , pp. 803 813, 1987."><meta name="citation_reference" content="Russell, A. J., et al., Nature, vol. 328, &quot;Rational modification of enzyme catalysis by engineering surface charge&quot;, pp. 496-500, 1987."><meta name="citation_reference" content="Russell, A. J., et al., Nature, vol. 328, Rational modification of enzyme catalysis by engineering surface charge , pp. 496 500, 1987."><meta name="citation_reference" content="Shortle, D., et al., &quot;Gap misrepair mutagenesis: Efficient site-directed induction of transition, transversion, and frameshift mutations in vitro.&quot; Proc. Natl. Acad. Sci. USA., 79:1588-1592 (1982)."><meta name="citation_reference" content="Shortle, D., et al., Gap misrepair mutagenesis: Efficient site directed induction of transition, transversion, and frameshift mutations in vitro. Proc. Natl. Acad. Sci. USA., 79:1588 1592 (1982)."><meta name="citation_reference" content="Singleton, et al., Dictionary of Microbiology and Molecular Biology, Second Edition. John Wiley and Sons. Chichester. p. 741 (1987)."><meta name="citation_reference" content="Smith, E.L., &quot;The Complete Sequence; Comparison with Subtilisin BPN; Evolutionary Relationships.&quot; J. Biol. Chem., 243:2184-2191 (1968)."><meta name="citation_reference" content="Smith, E.L., The Complete Sequence; Comparison with Subtilisin BPN; Evolutionary Relationships. J. Biol. Chem., 243:2184 2191 (1968)."><meta name="citation_reference" content="Stahl, M.L., et al., &quot;Replacement of the Bacillus subtilis Subtilisin Structural Gene with and In Vitro-Derived Deletion Mutation.&quot; J. Bacteriol., 158:411-418 (1984)."><meta name="citation_reference" content="Stahl, M.L., et al., Replacement of the Bacillus subtilis Subtilisin Structural Gene with and In Vitro Derived Deletion Mutation. J. Bacteriol., 158:411 418 (1984)."><meta name="citation_reference" content="Stauffer, C.E., et al., &quot;The Effect on Subtilisin Activity of Oxidizing a Methionine Residue.&quot; J. Biol. Chem. 244:5333-5338 (1969)."><meta name="citation_reference" content="Stauffer, C.E., et al., The Effect on Subtilisin Activity of Oxidizing a Methionine Residue. J. Biol. Chem. 244:5333 5338 (1969)."><meta name="citation_reference" content="Stroud, R.M., Sci. Amer., 131:74 88 (1974)."><meta name="citation_reference" content="Stroud, R.M., Sci. Amer., 131:74-88 (1974)."><meta name="citation_reference" content="Svendsen, I., &quot;Chemical Modifications of the Subtilisins with Special Reference to the Binding of Large Substrates. A Review.&quot; Carlsberg Res. Commun., 41:237-291 (1976)."><meta name="citation_reference" content="Svendsen, I., Chemical Modifications of the Subtilisins with Special Reference to the Binding of Large Substrates. A Review. Carlsberg Res. Commun., 41:237 291 (1976)."><meta name="citation_reference" content="Svendsen, I., et al., &quot;Complete Amino Acid Sequence of Alkaline Mesentericopeptidase: A subtilisin isolate from a strain of Bacillus mesentericus.&quot; FEBS, 196:228-232 (1986)."><meta name="citation_reference" content="Svendsen, I., et al., Complete Amino Acid Sequence of Alkaline Mesentericopeptidase: A subtilisin isolate from a strain of Bacillus mesentericus. FEBS, 196:228 232 (1986)."><meta name="citation_reference" content="Thomas, et al., Nature, 318:37 38 (1985)."><meta name="citation_reference" content="Thomas, et al., Nature, 318:37-38 (1985)."><meta name="citation_reference" content="Toney et al., Science, 243:1485 1488 (1989)."><meta name="citation_reference" content="Toney et al., Science, 243:1485-1488 (1989)."><meta name="citation_reference" content="Uehara, H., et al., J. Bacteriology, 139:583 590 (1979)."><meta name="citation_reference" content="Uehara, H., et al., J. Bacteriology, 139:583-590 (1979)."><meta name="citation_reference" content="Ulmer, K.M. Science, 219:666 671 (1983)."><meta name="citation_reference" content="Ulmer, K.M. Science, 219:666-671 (1983)."><meta name="citation_reference" content="Vasantha, et al., J. Bacteriol., 165:837 842 (1986)."><meta name="citation_reference" content="Vasantha, et al., J. Bacteriol., 165:837-842 (1986)."><meta name="citation_reference" content="Vasantha, N., et al., &quot;Genes for Alkaline Protease and Neutral Protease from Bacillus amyloliquefaciens Contain a Large Open Reading Frame Between the Regions Coding for Signal Sequence and Mature Protein.&quot; J. Bacteriol., 159:811-819 (1984)."><meta name="citation_reference" content="Vasantha, N., et al., Genes for Alkaline Protease and Neutral Protease from Bacillus amyloliquefaciens Contain a Large Open Reading Frame Between the Regions Coding for Signal Sequence and Mature Protein. J. Bacteriol., 159:811 819 (1984)."><meta name="citation_reference" content="Villafranca, D.E., et al., Science, 222:782 788 (1983)."><meta name="citation_reference" content="Villafranca, D.E., et al., Science, 222:782-788 (1983)."><meta name="citation_reference" content="Wallace, B.R., et al., &quot;Oligonucleotide directed mutagenesis of the human β-globin gene: a general method for producing specific point mutations in cloned DNA.&quot; Nucleic Acids Res., 9:3647-3656 (1981)."><meta name="citation_reference" content="Wallace, B.R., et al., Oligonucleotide directed mutagenesis of the human globin gene: a general method for producing specific point mutations in cloned DNA. Nucleic Acids Res., 9:3647 3656 (1981)."><meta name="citation_reference" content="Watson, J.D., in Molecular Biology of the Gene, 3rd Edition. Benjamin/Cummings Publ. Co., Inc. Menlo Park, CA pp. 313 (1987)."><meta name="citation_reference" content="Wells, et al., &quot;Subtilisin: An Enzyme Designed to be Engineered,&quot; in Proteins: Form and Function (Bradshaw et al., eds.). Elsevier Trends Journals, Cambridge, UK, pp. 45-57 (1990)."><meta name="citation_reference" content="Wells, et al., Cold Spring Harbor Symposia on Quantitative Biology, LII:647 652 (1987)."><meta name="citation_reference" content="Wells, et al., Cold Spring Harbor Symposia on Quantitative Biology, LII:647-652 (1987)."><meta name="citation_reference" content="Wells, et al., Nucleic Acids Res., 11:7911 7923 (1983)."><meta name="citation_reference" content="Wells, et al., Nucleic Acids Res., 11:7911-7923 (1983)."><meta name="citation_reference" content="Wells, et al., Subtilisin: An Enzyme Designed to be Engineered, in Proteins: Form and Function (Bradshaw et al., eds.). Elsevier Trends Journals, Cambridge, UK, pp. 45 57 (1990)."><meta name="citation_reference" content="Wells, J.A., &quot;Cloning, sequencing, and secretion of Bacillus amyloliquefaciens subtilisin in Bacillus subtilis.&quot; Nucleic Acids Res., 11:8913-8926 (1985)."><meta name="citation_reference" content="Wells, J.A., Cloning, sequencing, and secretion of Bacillus amyloliquefaciens subtilisin in Bacillus subtilis. Nucleic Acids Res., 11:8913 8926 (1985)."><meta name="citation_reference" content="Wells, J.A., et al., &quot;Designing Substrate Specificity by Protein Engineering of Electrostatic Interactions.&quot; Proc. Nat. Acad. Sci. USA, 84:1219-1223 (1987)."><meta name="citation_reference" content="Wells, J.A., et al., &quot;Protein Engineering of Subtilisin.&quot; Protein Engineering, pp. 279-287 (1987)."><meta name="citation_reference" content="Wells, J.A., et al., Designing Substrate Specificity by Protein Engineering of Electrostatic Interactions. Proc. Nat. Acad. Sci. USA, 84:1219 1223 (1987)."><meta name="citation_reference" content="Wells, J.A., et al., Protein Engineering of Subtilisin. Protein Engineering, pp. 279 287 (1987)."><meta name="citation_reference" content="Wilkinson, A.H., et al., Nature, 307:187 188 (1984)."><meta name="citation_reference" content="Wilkinson, A.H., et al., Nature, 307:187-188 (1984)."><meta name="citation_reference" content="Wilkinson, A.J., et al., Biochemistry, 22:3581 3586 (1983)."><meta name="citation_reference" content="Wilkinson, A.J., et al., Biochemistry, 22:3581-3586 (1983)."><meta name="citation_reference" content="Winter, G., et al., &quot;Redesigning enzyme structure by site-directed mutagenesis: tyrosyl tRNA synthetase and ATP binding.&quot; Nature, 299:756-758 (1982)."><meta name="citation_reference" content="Winter, G., et al., Redesigning enzyme structure by site directed mutagenesis: tyrosyl tRNA synthetase and ATP binding. Nature, 299:756 758 (1982)."><meta name="citation_reference" content="Wright, C.S., &quot;Structure of Subtilisin BPN&#39; at 2·5 A Resolution.&quot; Nature, 221:235-242 (1969)."><meta name="citation_reference" content="Wright, C.S., Structure of Subtilisin BPN at 2 5 A Resolution. Nature, 221:235 242 (1969)."><meta name="citation_reference" content="Zaghloul, et al., J. Bacteriol., 164:550 555 (1985)."><meta name="citation_reference" content="Zaghloul, et al., J. Bacteriol., 164:550-555 (1985)."><meta name="citation_patent_number" content="US:5700676"><meta name="citation_patent_application_number" content="US:08/486,746"><link rel="canonical" href="http://www.google.com/patents/US5700676"/><meta property="og:url" content="http://www.google.com/patents/US5700676"/><meta name="title" content="Patent US5700676 - Modified subtilisins having amino acid alterations"/><meta name="description" content="Novel carbonyl hydrolase mutants derived from the amino acid sequence of naturally-occurring or recombinant non-human carbonyl hydrolases and DNA sequences encoding the same. The mutant carbonyl hydrolases, in general, are obtained by in vitro modification of a precursor DNA sequence encoding the naturally-occurring or recombinant carbonyl hydrolase to encode the substitution, insertion or deletion of one or more amino acids in the amino acid sequence of a precursor carbonyl hydrolase. Such mutants have one or more properties which are different than the same property of the precursor hydrolase."/><meta property="og:title" content="Patent US5700676 - Modified subtilisins having amino acid alterations"/><meta property="og:type" content="book"/><meta property="og:site_name" content="Google Books"/><meta property="og:image" content="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><link rel="image_src" href="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><script>(function(){try{var aa=function(a,b,c,d){d=d||{};d._sn=["cfg",b,c].join(".");window.gbar.logger.ml(a,d)};var g=window.gbar=window.gbar||{},l=window.gbar.i=window.gbar.i||{},m={},n;function _tvn(a,b){var c=parseInt(a,10);return isNaN(c)?b:c}function _tvf(a,b){var c=parseFloat(a);return isNaN(c)?b:c}function _tvv(a){return!!a}function p(a,b,c){(c||g)[a]=b}g.bv={n:_tvn("2",0),r:"",f:".67.",e:"0",m:_tvn("0",1)};
function q(a,b,c){var d="on"+b;if(a.addEventListener)a.addEventListener(b,c,!1);else if(a.attachEvent)a.attachEvent(d,c);else{var f=a[d];a[d]=function(){var a=f.apply(this,arguments),b=c.apply(this,arguments);return void 0==a?b:void 0==b?a:b&&a}}}var s=function(a){return function(){return g.bv.m==a}},ba=s(1),ca=s(2);p("sb",ba);p("kn",ca);l.a=_tvv;l.b=_tvf;l.c=_tvn;l.i=aa;var da=window.gbar.i.i;var t,u,v,w;function ea(a){v=a}function fa(a){var b;if(b=v&&window.encodeURIComponent)b=a.href,b=!b.match(/^http[s]?:\/\/accounts\.google\.[^/]*\/ClearSID/i)&&!b.match(/^http[s]?:\/\/[^/]*\/accounts\/ClearSID/i);if(b=b&&encodeURIComponent(v()))a.href=a.href.replace(/([?&]continue=)[^&]*/,"$1"+b)}function ga(a){window.gApplication&&(a.href=window.gApplication.getTabUrl(a.href))}
function ha(a){var b=document.forms[0].q,c=window.encodeURIComponent&&b&&b.value,b=b&&b.placeholder;c&&c!=b&&(a.href=a.href.replace(/([?&])q=[^&]*|$/,function(a,b){return(b||"&")+"q="+encodeURIComponent(c)}))}n=l.a("")?ga:ha;
function x(a,b,c,d,f,e){var h=document.getElementById(a);if(h){var k=h.style;k.left=d?"auto":b+"px";k.right=d?b+"px":"auto";k.top=c+"px";k.visibility=u?"hidden":"visible";f&&e?(k.width=f+"px",k.height=e+"px"):(x(t,b,c,d,h.offsetWidth,h.offsetHeight),u=u?"":a)}}
var y=[],ia=function(a,b){y.push(b)},ja=function(a){a=a||window.event;var b=a.target||a.srcElement;a.cancelBubble=!0;null==t&&(a=document.createElement(Array.every||window.createPopup?"iframe":"div"),a.frameBorder="0",t=a.id="gbs",a.src="javascript:''",b.parentNode.appendChild(a),q(document,"click",z));var c=b,b=0;"gb3"!=c.className&&(c=c.parentNode);a=c.getAttribute("aria-owns")||"gbi";var d=c.offsetWidth,f=20<c.offsetTop?46:24;document.getElementById("tphdr")&&(f-=3);var e=!1;do b+=c.offsetLeft||
0;while(c=c.offsetParent);var c=(document.documentElement.clientWidth||document.body.clientWidth)-b-d,h,d=document.body,k=document.defaultView;k&&k.getComputedStyle?(d=k.getComputedStyle(d,""))&&(h=d.direction):h=d.currentStyle?d.currentStyle.direction:d.style.direction;h="rtl"==h;if("gbi"==a){for(d=0;k=y[d++];)k();A(null,window.navExtra);h&&(b=c,e=!0)}else h||(b=c,e=!0);u!=a&&z();x(a,b,f,e)},z=function(){u&&x(u,0,0)},A=function(a,b){var c,d=document.getElementById("gbi"),f=a;f||(f=d.firstChild);
for(;b&&(c=b.pop());){var e=d,h=c,k=f;w||(w="gb2");e.insertBefore(h,k).className=w}},ka=function(a,b,c){if((b=document.getElementById(b))&&a){a.className="gb4";var d=document.createElement("span");d.appendChild(a);d.appendChild(document.createTextNode(" | "));d.id=c;b.appendChild(d)}},la=function(){return document.getElementById("gb_70")},ma=function(){return!!u};p("qs",n);p("setContinueCb",ea);p("pc",fa);p("tg",ja);p("close",z);p("addLink",ka);p("almm",A);p("si",la);p("adh",ia);p("op",ma);var B=function(){},C=function(){},F=function(a){var b=new Image,c=D;b.onerror=b.onload=b.onabort=function(){try{delete E[c]}catch(a){}};E[c]=b;b.src=a;D=c+1},E=[],D=0;p("logger",{il:C,ml:B,log:F});var G=window.gbar.logger;var H={},na={},I=[],oa=l.b("0.1",.1),pa=l.a("1",!0),qa=function(a,b){I.push([a,b])},ra=function(a,b){H[a]=b},sa=function(a){return a in H},J={},K=function(a,b){J[a]||(J[a]=[]);J[a].push(b)},ta=function(a){K("m",a)},L=function(a,b){var c=document.createElement("script");c.src=a;c.async=pa;Math.random()<oa&&(c.onerror=function(){c.onerror=null;B(Error("Bundle load failed: name="+(b||"UNK")+" url="+a))});(document.getElementById("xjsc")||document.getElementsByTagName("body")[0]||
document.getElementsByTagName("head")[0]).appendChild(c)},N=function(a){for(var b=0,c;(c=I[b])&&c[0]!=a;++b);!c||c[1].l||c[1].s||(c[1].s=!0,M(2,a),c[1].url&&L(c[1].url,a),c[1].libs&&m.d&&m.d(c[1].libs))},O=function(a){K("gc",a)},P=null,ua=function(a){P=a},M=function(a,b,c){if(P){a={t:a,b:b};if(c)for(var d in c)a[d]=c[d];try{P(a)}catch(f){}}};p("mdc",H);p("mdi",na);p("bnc",I);p("qGC",O);p("qm",ta);p("qd",J);p("lb",N);p("mcf",ra);p("bcf",qa);p("aq",K);p("mdd","");p("has",sa);
p("trh",ua);p("tev",M);var Q=l.b("0.1",.001),R=0;
function _mlToken(a,b){try{if(1>R){R++;var c,d=a,f=b||{},e=encodeURIComponent,h=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&jexpid=",e("17483"),"&srcpg=",e("prop=22"),"&jsr=",Math.round(1/Q),"&ogev=",e("AqXtU4CQFtKOogT0zoHIAQ"),"&ogf=",g.bv.f,"&ogrp=",e("1"),"&ogv=",e("1407723702.0"),"&oggv="+e("es_plusone_gc_20140723.0_p0"),"&ogd=",e("com"),"&ogc=",e("NZL"),"&ogl=",e("en")];f._sn&&(f._sn="og."+
f._sn);for(var k in f)h.push("&"),h.push(e(k)),h.push("="),h.push(e(f[k]));h.push("&emsg=");h.push(e(d.name+":"+d.message));var r=h.join("");S(r)&&(r=r.substr(0,2E3));c=r;var Aa=window.gbar.logger._aem(a,c);F(Aa)}}catch(Na){}}var S=function(a){return 2E3<=a.length},va=function(a,b){return b};function T(a){B=a;p("_itl",S,G);p("_aem",va,G);p("ml",B,G);a={};H.er=a}l.a("")?T(function(a){throw a;}):l.a("1")&&Math.random()<Q&&T(_mlToken);I.push(["m",{url:"//ssl.gstatic.com/gb/js/scm_7385cc5883250b43a39405734c1bea59.js"}]);g.mcf("c",{});g.sg={c:""};if(l.a("1")){var wa=l.a("");I.push(["gc",{auto:wa,url:"//ssl.gstatic.com/gb/js/abc/gci_91f30755d6a6b787dcc2a4062e6e9824.js",libs:"googleapis.client:plusone:gapi.iframes"}]);var xa={version:"gci_91f30755d6a6b787dcc2a4062e6e9824.js",index:"",lang:"en"};H.gc=xa;var U=function(a){window.googleapis&&window.iframes?a&&a():(a&&O(a),N("gc"))};p("lGC",U);l.a("1")&&p("lPWF",U)};window.__PVT="";if(l.a("1")&&l.a("1")){var V=function(a){U(function(){K("pw",a);N("pw")})};p("lPW",V);I.push(["pw",{url:"//ssl.gstatic.com/gb/js/abc/pwm_45f73e4df07a0e388b0fa1f3d30e7280.js"}]);var W=[],ya=function(a){W[0]=a},za=function(a,b){var c=b||{};c._sn="pw";B(a,c)},Ba={signed:W,elog:za,base:"https://plusone.google.com/u/0",loadTime:(new Date).getTime()};H.pw=Ba;var X=function(a,b){for(var c=b.split("."),d=function(){var b=arguments;a(function(){for(var a=g,d=0,e=c.length-1;d<e;++d)a=a[c[d]];a[c[d]].apply(a,b)})},f=g,e=0,h=c.length-1;e<h;++e)f=
f[c[e]]=f[c[e]]||{};return f[c[e]]=d};X(V,"pw.clk");X(V,"pw.hvr");p("su",ya,g.pw)};function Ca(){function a(){for(var b;(b=e[h++])&&"m"!=b[0]&&!b[1].auto;);b&&(M(2,b[0]),b[1].url&&L(b[1].url,b[0]),b[1].libs&&m.d&&m.d(b[1].libs));h<e.length&&setTimeout(a,0)}function b(){0<f--?setTimeout(b,0):a()}var c=l.a("1"),d=l.a(""),f=3,e=I,h=0,k=window.gbarOnReady;if(k)try{k()}catch(r){da(r,"ml","or")}d?p("ldb",a):c?q(window,"load",b):b()}p("rdl",Ca);var Da={D:1,H:2,da:3,p:4,W:5,M:6,F:7,g:8,ha:9,U:10,L:11,T:12,S:13,N:14,Q:15,P:16,fa:17,w:18,O:19,ga:20,ea:21,u:22,G:23,ja:24,ka:25,ia:26,A:27,j:28,o:29,k:30,ca:31,Z:32,$:33,J:34,K:35,ba:36,aa:37,Y:38,B:39,R:40,v:41,X:42,V:43,h:48,C:49,I:500},Y=[1,2,3,4,5,6,9,10,11,13,14,28,29,30,34,35,37,38,39,40,41,42,43,48,49,500];var Z=l.b("0.001",1E-4),Ea=l.b("1",1),Fa=!1,Ga=!1;if(l.a("1")){var Ha=Math.random();Ha<=Z&&(Fa=!0);Ha<=Ea&&(Ga=!0)}var Ia=Da,$=null;function Ja(){var a=0,b=function(b,d){l.a(d)&&(a|=b)};b(1,"");b(2,"");b(4,"");b(8,"");return a}
function Ka(a,b){var c=Z,d=Fa,f;f=a;if(!$){$={};for(var e=0;e<Y.length;e++){var h=Y[e];$[h]=!0}}if(f=!!$[f])c=Ea,d=Ga;if(d){d=encodeURIComponent;g.rp?(f=g.rp(),f="-1"!=f?f:"1"):f="1";c=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&oge=",a,"&ogex=",d("17483"),"&ogev=",d("AqXtU4CQFtKOogT0zoHIAQ"),"&ogf=",g.bv.f,"&ogp=",d("22"),"&ogrp=",d(f),"&ogsr=",Math.round(1/c),"&ogv=",d("1407723702.0"),"&oggv="+
d("es_plusone_gc_20140723.0_p0"),"&ogd=",d("com"),"&ogl=",d("en"),"&ogc=",d("NZL"),"&ogus=",Ja()];if(b){"ogw"in b&&(c.push("&ogw="+b.ogw),delete b.ogw);var k;f=b;e=[];for(k in f)0!=e.length&&e.push(","),e.push(La(k)),e.push("."),e.push(La(f[k]));k=e.join("");""!=k&&(c.push("&ogad="),c.push(d(k)))}F(c.join(""))}}function La(a){"number"==typeof a&&(a+="");return"string"==typeof a?a.replace(".","%2E").replace(",","%2C"):a}C=Ka;p("il",C,G);var Ma={};H.il=Ma;setTimeout(function(){C(Ia.g)},0);}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var b=window.gbar.i.i;var c=window.gbar;var f=function(d){try{var a=document.getElementById("gbom");a&&d.appendChild(a.cloneNode(!0))}catch(e){b(e,"omas","aomc")}};c.aomc=f;}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var a=window.gbar;a.mcf("pm",{p:""});}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{window.gbar.rdl();}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
if (window['_OC_timingAction']) {window['_OC_timingAction']('patents_refpage');}</script><style>#gbar,#guser{font-size:13px;padding-top:1px !important;}#gbar{float:left;height:22px}#guser{padding-bottom:7px !important;text-align:right}.gbh,.gbd{border-top:1px solid #c9d7f1;font-size:1px}.gbh{height:0;position:absolute;top:24px;width:100%}#gbs,.gbm{background:#fff;left:0;position:absolute;text-align:left;visibility:hidden;z-index:1000}.gbm{border:1px solid;border-color:#c9d7f1 #36c #36c #a2bae7;z-index:1001}.gb1{margin-right:.5em}.gb1,.gb3{zoom:1}.gb2{display:block;padding:.2em .5em}.gb2,.gb3{text-decoration:none !important;border-bottom:none}a.gb1,a.gb4{text-decoration:underline !important}a.gb1,a.gb2,a.gb3,a.gb4{color:#00c !important}.gbi .gb3,.gbi .gb2,.gbi .gb4{color:#dd8e27 !important}.gbf .gb3,.gbf .gb2,.gbf .gb4{color:#900 !important}a.gb2:hover{background:#36c;color:#fff !important}#gbar .gbz0l{color:#000 !important;cursor:default;font-weight:bold;text-decoration:none !important}
#gbar { padding:.3em .6em !important;}</style></head><body  topmargin="3" marginheight="3"><div id=gbar><nobr><a onclick=gbar.qs(this);gbar.logger.il(1,{t:1}); class=gb1 id=gb_1 href="https://www.google.com/search?sa=N&tab=tw">Search</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:2}); class=gb1 id=gb_2 href="http://www.google.com/search?hl=en&tbm=isch&source=og&sa=N&tab=ti">Images</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:8}); class=gb1 id=gb_8 href="http://maps.google.com/maps?hl=en&sa=N&tab=tl">Maps</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:78}); class=gb1 id=gb_78 href="https://play.google.com/?hl=en&sa=N&tab=t8">Play</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:36}); class=gb1 id=gb_36 href="http://www.youtube.com/results?sa=N&tab=t1">YouTube</a> <a onclick=gbar.logger.il(1,{t:5}); class=gb1 id=gb_5 href="http://news.google.com/nwshp?hl=en&tab=tn">News</a> <a onclick=gbar.logger.il(1,{t:23}); class=gb1 id=gb_23 href="https://mail.google.com/mail/?tab=tm">Gmail</a> <a onclick=gbar.logger.il(1,{t:25}); class=gb1 id=gb_25 href="https://drive.google.com/?tab=to">Drive</a> <a class=gb3 href="http://www.google.com/intl/en/options/" onclick="this.blur();gbar.tg(event);return !1" aria-haspopup=true><u>More</u> <small>&#9660;</small></a><div class=gbm id=gbi><a onclick=gbar.logger.il(1,{t:24}); class=gb2 id=gb_24 href="https://www.google.com/calendar?tab=tc">Calendar</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:51}); class=gb2 id=gb_51 href="http://translate.google.com/?hl=en&sa=N&tab=tT">Translate</a><a onclick=gbar.logger.il(1,{t:17}); class=gb2 id=gb_17 href="http://www.google.com/mobile/?hl=en&tab=tD">Mobile</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:10}); class=gb2 id=gb_10 href="http://www.google.com/search?hl=en&tbo=u&tbm=bks&source=og&sa=N&tab=tp">Books</a><a onclick=gbar.logger.il(1,{t:212}); class=gb2 id=gb_212 href="https://wallet.google.com/manage/?tab=ta">Wallet</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:6}); class=gb2 id=gb_6 href="http://www.google.com/search?hl=en&tbo=u&tbm=shop&source=og&sa=N&tab=tf">Shopping</a><a onclick=gbar.logger.il(1,{t:30}); class=gb2 id=gb_30 href="http://www.blogger.com/?tab=tj">Blogger</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:27}); class=gb2 id=gb_27 href="http://www.google.com/finance?sa=N&tab=te">Finance</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:31}); class=gb2 id=gb_31 href="https://plus.google.com/photos?sa=N&tab=tq">Photos</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:12}); class=gb2 id=gb_12 href="http://www.google.com/search?hl=en&tbo=u&tbm=vid&source=og&sa=N&tab=tv">Videos</a><div class=gb2><div class=gbd></div></div><a onclick=gbar.logger.il(1,{t:66}); href="http://www.google.com/intl/en/options/" class=gb2>Even more &raquo;</a></div></nobr></div><div id=guser width=100%><nobr><span id=gbn class=gbi></span><span id=gbf class=gbf></span><span id=gbe></span><a target=_top id=gb_70 href="https://www.google.com/accounts/Login?service=&continue=http://www.google.com/patents%3Fhl%3Den&hl=en" class=gb4>Sign in</a><div style="display: none"><div class=gbm id=gbd5 aria-owner=gbg5><div class=gbmc><ol id=gbom class=gbmcc></ol></div></div></div></nobr></div><div class=gbh style=left:0></div><div class=gbh style=right:0></div><div role="alert" style="position: absolute; left: 0; right: 0;"><a href="http://www.google.com/patents/us5700676?hl=en&amp;output=html_text" title="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."><img border="0" src="http://www.google.com/images/cleardot.gif"alt="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."></a></div><div id="guser"><nobr></nobr></div><div style="clear:both;"></div><div id="gb-top-search-box" class="gb-top-search-box-small gb-reset"><table><tr><td class="logo"><a href="http://www.google.com/patents" class="logo-link"><img class="logo-img" src="/intl/en/images/logos/google_logo_41.png" alt="Go to Google Books Home" height="41"/></a></td><td><form action="http://www.google.com/search" name="f" id="vheadf" method="get"><span id="hf"></span><input type="hidden" name="tbm" value="pts"/><input type="hidden" name="tbo" value="1"/><input type="hidden" name="hl" value="en"/><table><tr><td><div class="inputs"><table><tr><td><div class="text-input"><input type="text" name="q" id="vheadq" class="text" maxlength="2048" size="31" value="" title="Search Patents" accesskey="s" autocomplete="off"/><script>window._OC_autoDir &&window._OC_autoDir('vheadq', 'tia-vheadq');</script></div></td><td><div class="submit-input"><input name="btnG" class="submit" type="submit" value=""/></div></td></tr></table></div></td><td class="col-ext-links"><div class="ext-links"><a href="http://www.google.com/advanced_patent_search">&lt;nobr&gt;Advanced Patent Search&lt;/nobr&gt;</a></div></td></tr></table></form></td></tr></table></div><div class="kd-appbar"><h2 class="kd-appname"><a href="/patents">Patents</a></h2><div class="kd-buttonbar left" id="left-toolbar-buttons"><a id="appbar-write-review-link" href=""></a><a id="appbar-view-print-sample-link" href=""></a><a id="appbar-view-ebook-sample-link" href=""></a><a id="appbar-patents-prior-art-finder-link" href="https://www.google.com/patents/related/US5700676"></a><a id="appbar-patents-discuss-this-link" href="http://www.google.com/url?id=tNNEBAABERAJ&amp;q=http://patents.stackexchange.com/redirect/google-patents%3Fpatent%3DUS5700676&amp;usg=AFQjCNEf_X6s4tJ0JmU27DnOBrjgTUaTJg" data-is-grant="true"></a><a id="appbar-read-patent-link" href="//docs.google.com/viewer?url=patentimages.storage.googleapis.com/pdfs/US5700676.pdf"></a><a id="appbar-download-pdf-link" href="//patentimages.storage.googleapis.com/pdfs/US5700676.pdf"></a></div><div class="kd-buttonbar right" id="right-toolbar-buttons"></div></div><div id="books-microdata" itemscope=""itemtype="http://schema.org/Book"itemid="http://www.google.com/patents/US5700676" style="display:none"><span itemprop="description">Novel carbonyl hydrolase mutants derived from the amino acid sequence of naturally-occurring or recombinant non-human carbonyl hydrolases and DNA sequences encoding the same. The mutant carbonyl hydrolases, in general, are obtained by in vitro modification of a precursor DNA sequence encoding the naturally-occurring...</span><span itemprop="url">http://www.google.com/patents/US5700676?utm_source=gb-gplus-share</span><span class="main-title" itemprop="name">Patent US5700676 - Modified subtilisins having amino acid alterations</span><img itemprop="image" src="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"alt="Patent US5700676 - Modified subtilisins having amino acid alterations" title="Patent US5700676 - Modified subtilisins having amino acid alterations"></div><div style="display: none"><ol id="ofe-gear-menu-contents" class="gbmcc"><li class="gbe gbmtc"><a class="gbmt goog-menuitem-content" id="" href="http://www.google.com/advanced_patent_search">Advanced Patent Search</a></li></ol></div><table id="viewport_table" cellpadding="0" style="clear:both" cellspacing="0"><tr><td id="viewport_td"><div class=vertical_module_list_row><div id=intl_patents class=about_content><div id=intl_patents_v><table class="patent-bibdata"><tr><td class="patent-bibdata-heading">Publication number</td><td class="single-patent-bibdata">US5700676 A</td></tr><tr><td class="patent-bibdata-heading">Publication type</td><td class="single-patent-bibdata">Grant</td></tr><tr><td class="patent-bibdata-heading">Application number</td><td class="single-patent-bibdata">US 08/486,746</td></tr><tr><td class="patent-bibdata-heading">Publication date</td><td class="single-patent-bibdata">Dec 23, 1997</td></tr><tr><td class="patent-bibdata-heading">Filing date</td><td class="single-patent-bibdata">Jun 7, 1995</td></tr><tr><td class="patent-bibdata-heading">Priority date<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The priority date is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the date listed."></span></td><td class="single-patent-bibdata">May 29, 1984</td></tr><tr><td class="patent-bibdata-heading">Fee status<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The fee status is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the status or dates listed."></span></td><td class="single-patent-bibdata">Paid</td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Also published as</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/US5763257">US5763257</a>, </span><span class="patent-bibdata-value"><a href="/patents/US5801038">US5801038</a></span></span></td></tr><tr class="patent-bibdata-list-row alternate-patent-number"><td class="patent-bibdata-heading">Publication number</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value">08486746, </span><span class="patent-bibdata-value">486746, </span><span class="patent-bibdata-value">US 5700676 A, </span><span class="patent-bibdata-value">US 5700676A, </span><span class="patent-bibdata-value">US-A-5700676, </span><span class="patent-bibdata-value">US5700676 A, </span><span class="patent-bibdata-value">US5700676A</span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Inventors</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Richard+Ray+Bott%22">Richard Ray Bott</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Robert+Mark+Caldwell%22">Robert Mark Caldwell</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Brian+C.+Cunningham%22">Brian C. Cunningham</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22David+Aaron+Estell%22">David Aaron Estell</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Scott+Douglas+Power%22">Scott Douglas Power</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22James+Allen+Wells%22">James Allen Wells</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Original Assignee</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=inassignee:%22Genencor+International+Inc.%22">Genencor International Inc.</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Export Citation</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/US5700676.bibtex">BiBTeX</a>, </span><span class="patent-bibdata-value"><a href="/patents/US5700676.enw">EndNote</a>, </span><span class="patent-bibdata-value"><a href="/patents/US5700676.ris">RefMan</a></span></span></td></tr><tr class="patent-internal-links"><td colspan=2><span class="patent-bibdata-value"><a href="#backward-citations">Patent Citations</a> (6),</span> <span class="patent-bibdata-value"><a href="#npl-citations">Non-Patent Citations</a> (141),</span> <span class="patent-bibdata-value"><a href="#forward-citations">Referenced by</a> (100),</span> <span class="patent-bibdata-value"><a href="#classifications">Classifications</a> (26),</span> <span class="patent-bibdata-value"><a href="#legal-events">Legal Events</a> (6)</span> </td></tr><tr><td colspan=2 class="patent-bibdata-external-link-spacer-top"></td></tr><tr class="patent-bibdata-external-link-spacer-bottom"></tr><tr><td colspan=2><span class="patent-bibdata-heading">External Links:&nbsp;</span><span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=tNNEBAABERAJ&q=http://patft.uspto.gov/netacgi/nph-Parser%3FSect2%3DPTO1%26Sect2%3DHITOFF%26p%3D1%26u%3D/netahtml/PTO/search-bool.html%26r%3D1%26f%3DG%26l%3D50%26d%3DPALL%26RefSrch%3Dyes%26Query%3DPN/5700676&usg=AFQjCNEuV2JygoMX6enmwHj-UcQGk7TgGw">USPTO</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=tNNEBAABERAJ&q=http://assignments.uspto.gov/assignments/q%3Fdb%3Dpat%26pat%3D5700676&usg=AFQjCNFwrFDQMapFA6PmExvAiqHLnE2OLA">USPTO Assignment</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=tNNEBAABERAJ&q=http://worldwide.espacenet.com/publicationDetails/biblio%3FCC%3DUS%26NR%3D5700676A%26KC%3DA%26FT%3DD&usg=AFQjCNHyXymm3o5yuwkdXH-vVQH7UX6uBQ">Espacenet</a></span></span></td></tr><tr class="patent-bibdata-group-spacer"></tr></table><div class="number-and-title"><span class="patent-title"><invention-title mxw-id="PT54244278" lang="EN" load-source="patent-office">Modified subtilisins having amino acid alterations</invention-title></span><br><span class="patent-number">US 5700676 A</span></div><div class="patent-section patent-abstract-section"><div class="patent-section-header"><span class="patent-section-title">Abstract</span></div><div class="patent-text"><abstract mxw-id="PA37710355" lang="EN" load-source="patent-office"> <div class="abstract">Novel carbonyl hydrolase mutants derived from the amino acid sequence of naturally-occurring or recombinant non-human carbonyl hydrolases and DNA sequences encoding the same. The mutant carbonyl hydrolases, in general, are obtained by in vitro modification of a precursor DNA sequence encoding the naturally-occurring or recombinant carbonyl hydrolase to encode the substitution, insertion or deletion of one or more amino acids in the amino acid sequence of a precursor carbonyl hydrolase. Such mutants have one or more properties which are different than the same property of the precursor hydrolase.</div>
  </abstract></div></div><div class="patent-section patent-drawings-section"><div class="patent-section-header"><span class="patent-section-title">Images<span class="patent-section-count">(42)</span></span></div><div class="patent-drawings-body"><div class="patent-drawings-carousel"><div class="drawings"><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5700676-3.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5700676-3.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5700676-4.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5700676-4.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5700676-5.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5700676-5.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5700676-6.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5700676-6.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5700676-7.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5700676-7.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5700676-8.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5700676-8.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5700676-9.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5700676-9.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5700676-10.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5700676-10.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5700676-11.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5700676-11.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5700676-12.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5700676-12.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5700676-13.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5700676-13.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5700676-14.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5700676-14.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5700676-15.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5700676-15.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5700676-16.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5700676-16.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5700676-17.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5700676-17.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5700676-18.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5700676-18.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5700676-19.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5700676-19.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5700676-20.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5700676-20.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5700676-21.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5700676-21.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5700676-22.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5700676-22.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5700676-23.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5700676-23.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5700676-24.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5700676-24.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5700676-25.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5700676-25.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5700676-26.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5700676-26.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5700676-27.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5700676-27.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5700676-28.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5700676-28.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5700676-29.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5700676-29.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5700676-30.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5700676-30.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5700676-31.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5700676-31.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5700676-32.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5700676-32.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5700676-33.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5700676-33.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5700676-34.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5700676-34.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5700676-35.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5700676-35.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5700676-36.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5700676-36.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5700676-37.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5700676-37.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5700676-38.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5700676-38.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5700676-39.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5700676-39.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5700676-40.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5700676-40.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5700676-41.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5700676-41.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5700676-42.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5700676-42.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5700676-43.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5700676-43.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5700676-44.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5700676-44.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div></div></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img" alt="Previous page"src="/googlebooks/images/kennedy/page_left.png"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img" alt="Next page"src="/googlebooks/images/kennedy/page_right.png"width="21" height="21" /></div></div></div><div class="patent-post-drawings"></div><div class="patent-section patent-claims-section"><div class="patent-section-header"><span class="patent-section-title">Claims<span class="patent-section-count">(57)</span></span></div><div class="patent-text"><div mxw-id="PCLM5168345" lang="EN" load-source="patent-office" class="claims">
    <claim-statement>What is claimed is:</claim-statement> <div class="claim"> <div num="1" class="claim">
      <div class="claim-text">1. A substantially pure subtilisin modified by a substitution of at least one amino acid at a residue position with a different naturally occurring amino acid, said residue position being selected from the group of equivalent amino acid residues of subtilisin naturally produced by Bacillus amyloliquefaciens consisting of Ile107, Lys170, Tyr171, Pro172, Lys213, His67, Leu135, Gly97, Leu126, Gly127, Gly128, Pro129, Tyr214 and Gly215 wherein the subtilisin which is modified is selected from the group consisting of subtilisins derived from procaryotes, yeast and fungi.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="2" class="claim">
      <div class="claim-text">2. A modified substilisn according to claim 1 wherein said position is selected from the group consisting of Ile107, Lys170, Tyr171, Pro172, and Lys213.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="3" class="claim">
      <div class="claim-text">3. A modified subtilisin according to claim 1 wherein said residue position is Ile107.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="4" class="claim">
      <div class="claim-text">4. A modified subtilisin according to claim 1 wherein said residue position is Ile107 and said different naturally occurring amino acid is valine.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="5" class="claim">
      <div class="claim-text">5. A modified subtilisin according to claim 1 wherein said residue position is Lys170.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="6" class="claim">
      <div class="claim-text">6. A modified subtilisin according to claim 1 wherein said residue position is Lys170 and said different naturally occurring amino acid is selected from the group consisting of glutamic acid and arginine.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="7" class="claim">
      <div class="claim-text">7. A modified subtilisin according to claim 1 wherein said residue position is Tyr171.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="8" class="claim">
      <div class="claim-text">8. A modified subtilisin according to claim 1 wherein said residue position is Tyr171 and said different naturally occurring amino acid is selected from the group consisting of phenylalanine, lysine, arginine, glutamic acid and glutamine.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="9" class="claim">
      <div class="claim-text">9. A modified subtilisin according to claim 1 wherein said residue position is Pro172.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="10" class="claim">
      <div class="claim-text">10. A modified subtilisin according to claim 1 wherein said residue position is Pro172 and said different naturally occurring amino acid is selected from the group consisting of aspartic acid and asparagine.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="11" class="claim">
      <div class="claim-text">11. A modified subtilisin according to claim 1 wherein said residue position is Lys213.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="12" class="claim">
      <div class="claim-text">12. A modified subtilisin according to claim 1 wherein said residue position is Lys213 and said different naturally occurring amino acid is selected from the group consisting of arginine and threonine.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="13" class="claim">
      <div class="claim-text">13. A modified subtilisin according to claim 1 wherein said residue position is His67.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="14" class="claim">
      <div class="claim-text">14. A modified subtilisin according to claim 1 wherein said residue position is Leu135.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="15" class="claim">
      <div class="claim-text">15. A modified subtilisin according to claim 1 wherein said residue position is Gly97.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="16" class="claim">
      <div class="claim-text">16. A modified subtilisin according to claim 1 wherein said residue position is Leu126.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="17" class="claim">
      <div class="claim-text">17. A modified subtilisin according to claim 1 wherein said residue position is Gly127.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="18" class="claim">
      <div class="claim-text">18. A modified subtilisin according to claim 1 wherein said residue position is Gly128.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="19" class="claim">
      <div class="claim-text">19. A modified subtilisin according to claim 1 wherein said residue position is Pro129.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="20" class="claim">
      <div class="claim-text">20. A modified subtilisin according to claim 1 wherein said residue position is Tyr214.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="21" class="claim">
      <div class="claim-text">21. A modified subtilisin according to claim 1 wherein said residue position is Gly215.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="22" class="claim">
      <div class="claim-text">22. A modified subtilisin according to claim 1 further comprising a substitution of an amino acid at a second residue position with a different naturally occurring amino acid, said second residue position being selected from the group of equivalent amino acid residues of subtilisin naturally produced by Bacillus amyloliquefaciens consisting of Asp32, Ser33, His64, Tyr104, Ala152, Asn155, Glu156, Gly166, Gly169, Phe189, Tyr217 and Met222.</div>
    </div>
    </div> <div class="claim"> <div num="23" class="claim">
      <div class="claim-text">23. A substantially pure subtilisin modified by a substitution of at least one amino acid at a residue position with a different naturally occurring amino acid, said residue position being selected from the group of equivalent amino acid residues of subtilisin naturally produced by Bacillus amyloliquefaciens consisting of His67, Ile107, Leu135, Gly97, Ala98, Asp99, Gly100, Ser101, Gly102, Gln103, Leu126, Gly127, Gly128, Pro129, Lys213, Tyr214, Gly215, Gly153, Asn154, Gly157, Thr158, Ser159, Gly160, Ser161, Ser162, Ser163, Thr164, Val165, Tyr167, Pro168, Lys170, Tyr171 and Pro172 wherein said modified subtilisin has altered substrate specificity as compared to the same subtilisin having the amino acid naturally occurring at the residue position, wherein the subtilisin which is modified is selected from the group consisting of subtilisins derived from procaryotes, yeast and fungi.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="24" class="claim">
      <div class="claim-text">24. A modified subtilisin according to claim 23 wherein said residue position is Ala98.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="25" class="claim">
      <div class="claim-text">25. A modified subtilisin according to claim 23 wherein said residue position is Gly100.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="26" class="claim">
      <div class="claim-text">26. A modified subtilisin according to claim 23 wherein said residue position is Ser101.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="27" class="claim">
      <div class="claim-text">27. A modified subtilisin according to claim 23 wherein said residue position is Gly102.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="28" class="claim">
      <div class="claim-text">28. A modified subtilisin according to claim 23 wherein said residue position is Gln103.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="29" class="claim">
      <div class="claim-text">29. A modified subtilisin according to claim 23 wherein said residue position is Gly153.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="30" class="claim">
      <div class="claim-text">30. A modified subtilisin according to claim 23 wherein said residue position is Asn154.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="31" class="claim">
      <div class="claim-text">31. A modified subtilisin according to claim 23 wherein said residue position is Gly157.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="32" class="claim">
      <div class="claim-text">32. A modified subtilisin according to claim 23 wherein said residue position is Thr158.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="33" class="claim">
      <div class="claim-text">33. A modified subtilisin according to claim 23 wherein said residue position is Ser159.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="34" class="claim">
      <div class="claim-text">34. A modified subtilisin according to claim 23 wherein said residue position is Gly160.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="35" class="claim">
      <div class="claim-text">35. A modified subtilisin according to claim 23 wherein said residue position is Ser161.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="36" class="claim">
      <div class="claim-text">36. A modified subtilisin according to claim 23 wherein said residue position is Ser162.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="37" class="claim">
      <div class="claim-text">37. A modified subtilisin according to claim 23 wherein said residue position is Ser163.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="38" class="claim">
      <div class="claim-text">38. A modified subtilisin according to claim 23 wherein said residue position is Thr164.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="39" class="claim">
      <div class="claim-text">39. A modified subtilisin according to claim 23 wherein said residue position is Val165.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="40" class="claim">
      <div class="claim-text">40. A modified subtilisin according to claim 23 wherein said residue position is Tyr167.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="41" class="claim">
      <div class="claim-text">41. A modified subtilisin according to claim 23 wherein said residue position is Pro168.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="42" class="claim">
      <div class="claim-text">42. A modified subtilisin according to claim 23 further comprising a substitution of an amino acid at a second residue position with a different naturally occurring amino acid, said second residue position being selected from the group of equivalent amino acid residues of subtilisin naturally produced by Bacillus amyloliquefaciens consisting of Tyr104, Ala152, Glu156, Gly166, Gly169, Phe189 and Tyr217.</div>
    </div>
    </div> <div class="claim"> <div num="43" class="claim">
      <div class="claim-text">43. A substantially pure subtilisin modified by at least a combination of substitutions of amino acids at residue positions with different naturally occurring amino acids, said combination of residue positions being selected from the group of equivalent amino acid residues of subtilisin naturally produced by Bacillus amyloliquefaciens consisting of Glu156/Gly169/Tyr217, Gly156/Gly166/Tyr217 and Glu156/Tyr217, wherein said modified subtilisin has altered substrate specificity and kinetics as compared to the same subtilisin having the amino acids naturally occurring at the residue positions, wherein the subtilisin which is modified is selected from the group consisting of subtilisins derived from procaryotes, yeast and fungi.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="44" class="claim">
      <div class="claim-text">44. A subtilisin according to claim 43 further comprising the substitution of Met50 with Phe.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="45" class="claim">
      <div class="claim-text">45. A subtilisin according to claim 43 wherein said Glu156 is substituted with Ser or Gln, said Gly169 is substituted with Ala and said Tyr217 is substituted with Leu.</div>
    </div>
    </div> <div class="claim"> <div num="46" class="claim">
      <div class="claim-text">46. A substantially pure subtilisin modified by at least a combination of substitutions of amino acids at residue positions with different naturally occurring amino acids, said combination of residue positions being selected from the group of equivalent amino acid residues of subtilisin naturally produced by Bacillus amyloliquefaciens consisting of Thr22/Ser87, Ser24/Ser87, Ala45/Ala48, Ser49/Lys94, Ser49/Val95, Met50/Val95, Met50/Gly110, Glu156/Gly166, Glu156/Gly169, Gly166/Met222, Gly169/Met222, Tyr21/Thr22, Tyr21/Thr22/Ser87, Met50/Glu156/Gly166/Tyr217, Met50/Glu156/Tyr217, Met50/Glu156/Gly169/Tyr217, Met50/Ile107/Lys213, Ser204/Lys213, and Ile107/Lys213, wherein the subtilisin which is modified is selected from the group consisting of subtilisins derived from procaryotes, yeast and fungi.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="47" class="claim">
      <div class="claim-text">47. A modified subtilisin according to claims 1, 23, 43 or 46 wherein said subtilisin is derived from a Bacillus subtilisin.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="48" class="claim">
      <div class="claim-text">48. A modified subtilisin according to claim 47 wherein Bacillus subtilisin is Bacillus amyloliquefaciens subtilisin.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="49" class="claim">
      <div class="claim-text">49. A composition comprising a modified subtilisin according to claims 1, 23, 43 or 46 in combination with a detergent.</div>
    </div>
    </div> <div class="claim"> <div num="50" class="claim">
      <div class="claim-text">50. A substantially pure subtilisin modified by a substitution of an amino acid at the residue position equivalent to Ser101 of the Bacillus amyloliquefaciens subtilisin with a different naturally occurring amino acid, wherein the subtilisin which is modified is selected from the group consisting of subtilisins derived from procaryotes, yeast and fungi.</div>
    </div>
    </div> <div class="claim"> <div num="51" class="claim">
      <div class="claim-text">51. A substantially pure subtilisin modified by a substitution of an amino acid at the residue position equivalent to Ile107 of the Bacillus amyloliquefaciens subtilisin with a different naturally occurring amino acid, wherein the subtilisin which is modified is selected from the group consisting of subtilisins derived from procaryotes, yeast and fungi.</div>
    </div>
    </div> <div class="claim"> <div num="52" class="claim">
      <div class="claim-text">52. A substantially pure subtilisin modified by a substitution of an amino acid at the residue position equivalent to Gly128 of the Bacillus amyloliquefaciens subtilisin with a different naturally occurring amino acid, wherein the subtilisin which is modified is selected from the group consisting of subtilisins derived from procaryotes, yeast and fungi.</div>
    </div>
    </div> <div class="claim"> <div num="53" class="claim">
      <div class="claim-text">53. A substantially pure subtilisin modified by a substitution of an amino acid at the residue position equivalent to Pro129 of the Bacillus amyloliquefaciens subtilisin with a different naturally occurring amino acid, wherein the subtilisin which is modified is selected from the group consisting of subtilisins derived from procaryotes, yeast and fungi.</div>
    </div>
    </div> <div class="claim"> <div num="54" class="claim">
      <div class="claim-text">54. A substantially pure subtilisin modified by a substitution of an amino acid at the residue position equivalent to Leu135 of the Bacillus amyloliquefaciens subtilisin with a different naturally occurring amino acid, wherein the subtilisin which is modified is selected from the group consisting of subtilisins derived from procaryotes, yeast and fungi.</div>
    </div>
    </div> <div class="claim"> <div num="55" class="claim">
      <div class="claim-text">55. A substantially pure subtilisin modified by a substitution of an amino acid at the residue position equivalent to Thr164 of the Bacillus amyloliquefaciens subtilisin with a different naturally occurring amino acid, wherein the subtilisin which is modified is selected from the group consisting of subtilisins derived from procaryotes, yeast and fungi.</div>
    </div>
    </div> <div class="claim"> <div num="56" class="claim">
      <div class="claim-text">56. A substantially pure subtilisin modified by a substitution of an amino acid at the residue position equivalent to Val165 of the Bacillus amyloliquefaciens subtilisin with a different naturally occurring amino acid, wherein the subtilisin which is modified is selected from the group consisting of subtilisins derived from procaryotes, yeast and fungi.</div>
    </div>
    </div> <div class="claim"> <div num="57" class="claim">
      <div class="claim-text">57. A substantially pure subtilisin modified by a substitution of an amino acid at the residue position equivalent to Asp99 of the Bacillus amyloliquefaciens subtilisin with a different naturally occurring amino acid, wherein said modified subtilisin has altered substrate specificity as compared to the same subtilisin having the amino acid naturally occurring at the residue position equivalent to Asp99, wherein the subtilisin which is modified is selected from the group consisting of subtilisins derived from procaryotes, yeast and fungi.</div>
    </div>
  </div> </div></div></div><div class="patent-section patent-description-section"><div class="patent-section-header"><span class="patent-section-title">Description</span></div><div class="patent-text"><div mxw-id="PDES67056760" lang="EN" load-source="patent-office" class="description">
    <p>This application is a Divisional of U.S. Ser. No. 08/212,291, filed Mar. 14, 1994, which is a Continuation of Ser. No. 07/898,382, filed Jun. 9, 1992, now abandoned which is a Continuation application of Ser. No. 07/747,459, filed Aug. 12, 1991, now abandoned, which is a Continuation application of Ser. No. 07/540,868, filed Jun. 14, 1990, now abandoned, which is a Continuation application of Ser. No. 07/035,652, filed Apr. 5, 1987, now abandoned, which is a Continuation-in-Part application of Ser. No. 06/858,594, filed Apr. 30, 1986, now abandoned, which is a Continuation-in-Part application of the following: Ser. No. 06/614,612, filed May 29, 1984, which issued as U.S. Pat. No. 4,760,025, Ser. No. 06/614,615, filed May 29, 1984, now abandoned, Ser. No. 06/614,617, filed May 29, 1984, now abandoned, and Ser. No. 06/614,491, filed May 29, 1984, and now abandoned.</p>
    <heading>FIELD OF THE INVENTION</heading> <p>The present invention relates to novel carbonyl hydrolase mutants derived from the amino acid sequence of naturally-occurring or recombinant non-human carbonyl hydrolases and to DNA sequences encoding the same. Such mutant carbonyl hydrolases, in general, are obtained by in vitro modification of a precursor DNA sequence encoding the naturally-occurring or recombinant carbonyl hydrolase to encode the substitution, insertion or deletion of one or more amino acids in a precursor amino acid sequence.</p>
    <heading>BACKGROUND OF THE INVENTION</heading> <p>Serine proteases are a subgroup of carbonyl hydrolase. They comprise a diverse class of enzymes having a wide range of specificities and biological functions. Stroud, R. M. (1974) Sci Amer. 131, 74-88. Despite their functional diversity, the catalytic machinery of serine proteases has been approached by at least two genetically distinct familites of enzymes: the Bacillus subtilisins and the mammalian and homologous bacterial serine proteases (e.g., trypsin and S. gresius trypsin). These two families of serine pro, eases show remarkably similar mechanisms of catalysis. Kraut, J. (1977) Ann. Rev. Biochem. 46, 331-358. Furthermore, although the primary structure is unrelated, the tertiary structure of these two enzyme families bring together a conserved catalytic triad of amino acids consisting of serine, histidine and aspartate.</p>
    <p>Subtilisin is a serine endoprotease (MW 27,500) which is secreted in large amounts from a wide variety of Bacillus species. The protein sequence of subtilisin has been determined from at least four different species of Bacillus. Markland, F. S., et al. (1971) in The Enzymes, ed. Boyer P. D., Acad Press, New York, Vol. III, pp. 561-608; Nedkov, P. et al. (1983) Hoppe-Seyler's Z. Physiol. Chem. 364, 1537-1540. The three-dimensional crystallographic structure of subtilisin BPN' (from B. amyloligoefaciens) to 2.5A resolution has also been reported. Wright, C. S., et al. (1969) Nature 221, 235-242; Drenth, J. et al. (1972) Eur. J. Biochem. 26, 177-181. These studies indicate that although subtilisin is genetically unrelated to the mammalian serine proteases, it has a similar active site structure. The x-ray crystal structures of subtilisin containing covalently bound peptide inhibitors (Robertus, J. D., et al. (1972) Biochemistry 11, 2439-2449), product complexes (Robertus, J. D., et al. (1972) Biochemistry 11, 4293-4303), and transition state analogs (Matthews, D. A., et al (1975) J. Biol. Chem. 250, 7120-7126; Poulos, T. L., et al. (1976) J. Biol. Chem. 251, 1097-1103), which have been reported have also provided information regarding the active site and putative substrate binding cleft of subtilisin. In addition, a large number of kinetic and chemical modification studies have been reported for subtilisin (Philipp, M., et al. (1983) Mol. Cell. Biochem. 51, 5-32; Svendsen, I. B. (1976) Carlsberg Res. Comm. 41, 237-291; Markland, F. S. Id.) as well as at least one report wherein the side chain of methione at residue 222 of subtilisin was converted by hydrogen peroxide to methionine-sulfoxide (Stauffer, D. C., et al. (1965) J. Biol. Chem. 244, 5333-5338).</p>
    <p>Substrate specificity is a ubiquitous feature of biological macromolecules that is determined by chemical forces including hydrogen bonding, electrostatic, hydrophobic and steric interactions. Jencks, W. P., in Catalysis in Chemistry and Enzymology (McGraw-Hill, 1969) pp. 321-436; Fersht, A., in Enzyme Structure and Mechanism (Freeman, San Francisco, 1977) pp. 226-287. Substrate specificity studies of enzymes, however, have been limited to the traditional means of probing the relative importance of these binding forces. Although substrate analogs can be synthesized chemically, the production of modified enzyme analogs has been limited to chemically modified enzyme derivatives (Kaiser, E. T., et al. (1985) Ann. Rev. Biochem. 54, 565-595 or naturally occurring mutants. Kraut, J. (1977) Ann. Rev. Biochem. 46, 331-358.</p>
    <p>The recent development of various in vitro techniques to manipulate the DNA sequences encoding naturally-occuring polypeptides as well as recent developments in the chemical synthesis of relatively short sequences of single and double stranded DNA has resulted in the speculation that such techniques can be used to modify enzymes to improve some functional property in a predictable way. Ulmer, K. M. (1983) Science 219, 666-671. The only working example disclosed therein, however, is the substitution of a single amino acid within the active site of tyrosyl-tRNA synthetase (Cys35→Ser) which lead to a reduction in enzymatic activity. See Winter, G., et al. (1982) Nature 299, 756-758; and Wilkinson, A. J., et al. (1983) Biochemistry 22, 3581-3586 (Cys35→Gly mutation also resulted in decreased activity).</p>
    <p>When the same t-RNA synthetase was modified by substituting a different amino acid residue within the active site with two different amino acids, one of the mutants (Thr51→Ala) reportedly demonstrated a predicted moderate increase in kcat/Km whereas a second mutant (Thr51→Pro) demonstrated a massive increase in kcat/Km which could not be explained with certainty. Wilkinson, A. H., et al. (1984) Nature 307, 187-188.</p>
    <p>Another reported example of a single substitution of an amino acid residue is the substitution of cysteine for isoleucine at the third residue of T4 lysozyme. Perry, L. J., et al. (1984) Science 226, 555-557. The resultant mutant lysozyme was mildly oxidized to form a disulfide bond between the new cysteine residue at position 3 and the native cysteine at position 97. This crosslinked mutant was initially described by the author as being enzymatically identical to, but more thermally stable than, the wild type enzyme. However, in a "Note Added in Proof", the author indicated that the enhanced stability observed was probably due to a chemical modification of cysteine at residue 54 since the mutant lysozyme with a free thiol at Cys54 has a thermal stability identical to the wild type lysozyme.</p>
    <p>Similarly, a modified dehydrofolate reductase from E. coli has been reported to be modified by similar methods to introduce a cysteine which could be crosslinked with a naturally-occurring cysteine in the reductase. Villafranca, D. E., et al. (1983) Science 222, 782-788. The author indicates that this mutant is fully reactive in the reduced state but has significantly diminished activity in the oxidized state. In addition, two other substitutions of specific amino acid residues are reported which resulted in mutants which had diminished or no activity.</p>
    <p>As set forth below, several laboratories have also reported the use of site directed mutagensis to produce the mutation of more than one amino acid residue within a polypeptide.</p>
    <p>The amino-terminal region of the signal peptide of the prolipoprotein of the E. coli outer membrane was stated to be altered by the substitution or deletion of residues 2 and 3 to produce a charge change in that region of the polypeptide. Inoyye, S., et al. (1982) Proc. Nat. Acad. Sci. USA 79, 3438-3441. The same laboratory also reported the substitution and deletion of amino acid redisues 9 and 14 to determine the effects of such substitution on the hydrophobic region of the same signal sequence. Inouye, S., et al. (1984) J. Biol. Chem. 259, 3729-3733. In the case of mutants at residues 2 and 3 the authors state that the results obtained were consistant with the proposed loop model for explaining the functions of the signal sequence. However, as reported the mutations at residues 9 and 14 produced results indicating that the signal peptide has unexpeded flexibility in terms of the relationship between its primary structure and function in protein secretion.</p>
    <p>Double mutants in the active site of tyrosyl-t-RNA synthetase have also been reported. Carter, P. J., et al. (1984) Cell 38, 835-840. In this report, the improved affinity of the previously described Thr51→Pro mutant for ATP was probed by producing a second mutation in the active site of the enzyme. One of the double mutants, Gly35/Pro51, reportedly demonstrated an unexpected result in that it bound ATP in the transition state better than was expected from the two single mutants. Moreover, the author warns, at least for one double mutant, that it is not readily predictable how one substitution alters the effect caused by the other substitution and that care must be taken in interpreting such substitutions.</p>
    <p>A mutant is disclosed in U.S. Pat. No. 4,532,207, wherein a polyarginine tail was attached to the C-terminal residue of β-urogastrone by modifying the DNA sequence encoding the polypeptide. As disclosed, the polyarginine tail changed the electrophoretic mobility of the urogastrone-polyaginine hybrid permiting selective purification. The polyarginine was subsequently removed, according to the patentee, by a polyarginine specific exopeptidase to produce the purified urogastrone. Properly construed, this reference discloses hybrid polypeptides which do not constitute mutant polypeptides containing the substitution, insertion or deletion of one or more amino acids of a naturally occurring polypeptide.</p>
    <p>Single and double mutants of rat pancreatic trypsin have also been reported. Craik, C. S., et al. (1985) Science 228, 291-297. As reported, glycine residues at positions 216 and 226 were replaced with alanine residues to produce three trypsin mutants (two single mutants and one double mutant). In the case of the single mutants, the authors stated expectation was to observe a differential effect on Km. They instead reported a change in specificity (kcat/Km) which was primarily the result of a decrease in kcat. In contrast, the double mutant reportedly demonstrated a differential increase in Km for lysyl and arginyl substrates as compared to wild type trypsin but had virtually no catalytic activity.</p>
    <p>The references discussed above are provided solely for their disclosure prior to the filing date of the instant case, and nothing herein is to be construed as an admission that the inventors are not entitled to antedate such disclosure by virtue of prior invention or priority based on earlier filed applications.</p>
    <p>Based on the above references, however, it is apparent that the modification of the amino acid sequence of wild type enzymes often results in the decrease or destruction of biological activity. Moreover, these references do not address the mutation of the particular carbonyl hydrolases disclosed herein.</p>
    <p>Accordingly, it is an object herein to provide carbonyl hydrolase mutants which have at least one property which is different from the same property of the carbonyl hydrolase precursor from which the amino acid of said mutant is derived.</p>
    <p>It is a further object to provide mutant DNA sequences encoding such carbonyl hydrolase mutants as well as expression vectors containing such mutant DNA sequences.</p>
    <p>Still further, another object of the present invention is to provide host cells transformed with such vectors as well as host cells which are capable of expressing such mutants either intracellularly or extracellularly.</p>
    <heading>SUMMARY OF THE INVENTION</heading> <p>The invention includes carbonyl hydrolase mutants, preferably having at least one property which is substantially different from the same property of the precursor non-human carbonyl hydrolase from which the amino acid sequence of the mutant is derived. These properties include oxidative stability, substrate, specificity catalytic activity, thermal stability, alkaline stability, pH activity profile and resistance to proteolytic degradation. The precursor carbonyl hydrolase may be naturally occurring carbonyl hydrolases or recombinant carbonyl hydrolases. The amino acid sequence of the carbonyl hydrolase mutant is derived by the substitution, deletion or insertion of one or more amino acids of the precursor carbonyl hydrolase amino acid sequence.</p>
    <p>The invention also includes mutant DNA sequences encoding such carbonyl hydrolase mutants. These mutant DNA sequences are derived from a precursor DNA sequence which encodes a naturally occurring or recombinant precursor carbonyl hydrolase. The mutant DNA sequence is derived by modifying the precursor DNA sequence to encode the substitution, deletion or insertion of one or more amino acids encoded by the precursor DNA sequence. These recombinant DNA sequences encode mutants having an amino acid sequence which does not exist in nature and at least one property which is substantially different from the same property of the precursor carbonyl hydrolase encoded by the precursor DNA sequence.</p>
    <p>Further the invention includes expression vectors containing such mutant DNA sequences as well as host cells transformed with such vectors which are capable of expressing said carbonyl hydrolase mutants.</p>
    <heading>BRIEF DESCRIPTION OF THE DRAWINGS</heading> <p>FIGS. 1A and 1B shows the nucleotide sequence of the coding strand, correlated with the amino acid sequence of B. amyloliquefaciens subtilisin gene. Promoter (p) ribosome binding site (rbs) and termination (term) regions of the DNA sequence as well as sequences encoding the presequence (PRE) putative prosequence (PRO) and mature form (MAT) of the hydrolase are also shown.</p>
    <p>FIG. 2 is a schematic diagram showing the substrate binding cleft of subtilisin together with substrate.</p>
    <p>FIG. 3 is a stereo view of the S-1 binding subsite of B. amyloliquefaciens subtilisin showing a lysine P-1 substrate bound in the site in two different ways.</p>
    <p>FIG. 3A shows Lysine P-1 substrate bound to form a salt bridge with a Glu at position 156.</p>
    <p>FIG. 3B shows Lysine P-1 substrate bound to form a salt bridge with Glu at position 166.</p>
    <p>FIG. 4 is a schematic diagram of the active site of subtilisin Asp32, His64 and Ser221.</p>
    <p>FIGS. 5A-1, 5A-2, 5B-1, 5B-2 and 5C depict the amino acid sequence of subtilisin obtained from various sources. The residues directly beneath each residue of B. amyloliquefaciens subtilisin are equivalent residues which (1) can be mutated in a similar manner to that described for B. amyloliquefaciens subtilisin, or (2) can be used as a replacement amino acid residue in B. amyloliquefaciens subtilisin. FIG. 5C depicts conserved residues of B. amyloliquefaciens subtilisin when compared to other subtilisin sequences.</p>
    <p>FIGS. 6A and 6B depict the inactivation of the mutants Met222L and Met222Q when exposed to various organic oxidants.</p>
    <p>FIGS. 7A and 7B depicts the ultraviolet spectrum of Met222F subtilisin and the difference spectrum generated after inactivation by diperdodecanoic acid (DPDA).</p>
    <p>FIG. 8 shows the pattern of cyanogen bromide digests of untreated and DPDA oxidized subtilisin Met222F on high resolution SDS-pyridine peptide gels.</p>
    <p>FIG. 9 depicts a map of the cyanogen bromide fragments of FIG. 8 and their alignment with the sequence of subtilisin Met222F.</p>
    <p>FIG. 10 depicts the construction of mutations between codons 45 and 50 of B. amyloliquefaciens subtilisin.</p>
    <p>FIG. 11 depicts the construction of mutations between codons 122 and 127 of B. amyloliquefaciens subtilisin.</p>
    <p>FIG. 12 depicts the effect of DPDA on the activity of subtilisin mutants at positions 50 and 124 in subtilisin Met222F.</p>
    <p>FIG. 13 depicts the construction of mutations at codon 166 of B. amyloliquefaciens subtilisin.</p>
    <p>FIG. 14 depicts the effect of hydrophobicity of the P-1 substrate side-chain on the kinetic parameters of wild-type B. amyloliquefaciens subtilisin.</p>
    <p>FIGS. 15A and 15B depicts the effect of position 166 side-chain substitutions on P-1 substrate specificity. FIG. 15A shows position 166 mutant subtilisins containing non-branched alkyl and aromatic side-chain substitutions arranged in order of increasing molecular volume. FIG. 15B shows a series of mutant enzymes progressing through β- and γ-branched aliphatic side chain substitutions of increasing molecular volume.</p>
    <p>FIGS. 16A, 16B, 16C and 16D depicts the effect of position 166 side-chain volumn on log kcat/Km for various P-1 substrates.</p>
    <p>FIG. 17 shows the substrate specificity differences between Ile166 and wild-type (Gly166) B. amyloliquefaciens subtilisin against a series of alphatic and aromatic substrates. Each bar represents the difference in log kcat/Km for Ile166 minus wild-type (Gly166) subtilisin.</p>
    <p>FIG. 18 depicts the construction of mutations at codon 169 of B. amyloliquefaciens subtilisin.</p>
    <p>FIG. 19 depicts the construction of mutations at codon 104 of B. amyloliquefaciens subtilisin.</p>
    <p>FIG. 20 depicts the construction of mutations at codon 152 B. amyloliquefaciens subtilisin.</p>
    <p>FIG. 21 depicts the construction of single mutations at codon 156 and double mutations at codons 156 and 166 of B. amyloliquefaciens subtilisin.</p>
    <p>FIG. 22 depicts the construction of mutations at codon 217 for B. amyloliquefaciens subtilisin.</p>
    <p>FIG. 23 depicts the kcat/Km versus pH profile for mutations at codon 156 and 166 in B. amyloliquefaciens subtilisin.</p>
    <p>FIG. 23A depicts the kcat/Km versus pH profile for mutations at codon 156 and 166 in B. amyloliquefaciens subtilisin.</p>
    <p>FIG. 24 depicts the kcat/Km versus pH profile for mutations at codon 222 in B. amyloliquefaciens subtilisin.</p>
    <p>FIG. 25 depicts the constructing mutants at codons 94, 95 and 96.</p>
    <p>FIGS. 26 and 27 depict substrate specificity of proteins for 4 substrates.</p>
    <p>FIGS. 28A, B, C and D depict the effect of charge in the P-1 binding sites due to substitutions at codon 156 and 166.</p>
    <p>FIGS. 29A and B are a stereoview of the P-1 binding site of subtilisin BPN' showing a lysine P-1 substrate bound in the site in two ways. In 29A, Lysine P-1 substrate is built to form a salt bridge with a Glu at codon 156. In 29B, Lysine P-1 substrate is built to form a salt bridge with Glu at codon 166.</p>
    <p>FIGS. 30A, 30B and 30C demonstrates residual enzyme activity versus temperature curves for purified wild-type (Panel A), C22/C87 (Panel B) and C24/C87 (Panel C).</p>
    <p>FIG. 31 depicts the strategy for producing point mutations in the subtilisin coding sequence by misincorporation of α-thioldeoxynucleotide triphosphates.</p>
    <p>FIG. 32 depicts the autolytic stability of purified wild type and mutant subtilisins 170E, 107V, 213R and 107V/213R at alkaline pH.</p>
    <p>FIG. 33 depicts the autolytic stability of purified wild type and mutant subtilisins V50, F50 and F50/V107/R213 at alkaline pH.</p>
    <p>FIG. 34 depicts the strategy for constructing plasmids containing random cassette mutagenesis over residues 197 through 228.</p>
    <p>FIGS. 35A and 35B depicts the oligodeoxynucleotides used for random cassette mutagenesis over residues 197 through 228.</p>
    <p>FIG. 36 depicts the construction of mutants at codon 204.</p>
    <p>FIG. 37 depicts the oligodeoxynucleotides used for synthesizing mutants at codon 204.</p>
    <heading>DETAILED DESCRIPTION</heading> <p>The inventors have discovered that various single and multiple in vitro mutations involving the substitution, deletion or insertion of one or more amino acids within a non-human carbonyl hydrolase amino acid sequence can confer advantageous properties to such mutants when compared to the non-mutated carbonyl hydrolase.</p>
    <p>Specifically, B. amyloliquefaciens subtilisin, an alkaline bacterial protease, has been mutated by modifying the DNA encoding the subtilisin to encode the substitution of one or more amino acids at various amino acid residues within the mature form of the subtilisin molecule. These in vitro mutant subtilisins have at least one property which is different when compared to the same property of the precursor subtilisin. These modified properties fall into several categories including: oxidative stability, substrate specificity, thermal stability, alkaline stability, catalytic activity, pH activity profile, resistance to proteolytic degradation, Km, kcat and Km/kcat ratio.</p>
    <p>Carbonyl hydrolases are enzymes which hydrolyze compounds containing ##STR1## bonds in which X is oxygen or nitrogen. They include naturally-occurring carbonyl hydrolases and recombinant carbonyl hydrolases. Naturally occurring carbonyl hydrolases principally include hydrolases, e.g. lipases and peptide hydrolases, e.g. subtilisins or metalloproteases. Peptide hydrolases include α-aminoacylpeptide hydrolase, peptidylamino-acid hydrolase, acylamino hydrolase, serine carboxypeptidase, metallocarboxypeptidase, thiol proteinase, carboxylproteinase and metalloproteinase. Serine, metallo, thiol and acid proteases are included, as well as endo and exoproteases.</p>
    <p>"Recombinant carbonyl hydrolase" refers to a carbonyl hydrolase in which the DNA sequence encoding the naturally occurring carbonyl hydrolase is modified to produce a mutant DNA sequence which encodes the substitution, insertion or deletion of one or more amino acids in the carbonyl hydrolase amino acid sequence. Suitable modification methods are disclosed herein and in EPO Publication No. 0130756 published Jan. 9, 1985.</p>
    <p>Subtilisins are bacterial carbonyl hydrolases which generally act to cleave peptide bonds of proteins or peptides. As used herein, "subtilisin" means a naturally occurring subtilisin or a recombinant subtilisin. A series of naturally occurring subtilisins is known to be produced and often secreted by various bacterial species. Amino acid sequences of the members of this series are not entirely homologous. However, the subtilisins in this series exhibit the same or similar type of proteolytic activity. This class of serine proteases shares a common amino acid sequence defining a catalytic triad which distinguishes them from the chymotrypsin related class of serine proteases. The subtilisins and chymotrypsin related serine proteases both have catalytic triad comprising aspartate, histidine and serine. In the subtilisin related proteases the relative order of these amino acids, reading from the amino to carboxy terminus is aspartate-histidine-serine. In the chymotrypsin related proteases the relative order, however is histidine-aspartate-serine. Thus, subtilisin herein refers to a serine protease having the catalytic triad of subtilisin related proteases.</p>
    <p>"Recombinant subtilisin" refers to a subtilisin in which the DNA sequence encoding the subtilisin is modified to produce a mutant DNA sequence which encodes the substitution, deletion or insertion of one or more amino acids in the naturally occurring subtilisin amino acid sequence. Suitable methods to produce such modification include those disclosed herein and in EPO Publication No. 0130756. For example, the subtilisin multiple mutant herein containing the substitution of methionine at amino acid residues 50, 124 and 222 with phenylalanine, isoleucine and glutamine, respectively, can be considered to be derived from the recombinant subtilisin containing the substitution of glutamine at residue 222 (Gln222) disclosed in EPO Publication No. 0130756. The multiple mutant thus is produced by the substitution of phenylalanine for methionine at residue 50 and isoleucine for methionine at residue 124, in the Gln222 recombinant subtilisin.</p>
    <p>"Non-human carbonyl hydrolases" and their genes may be obtained from many procaryotic and eucaryotic organisms. Suitable examples of procaryotic organisms include gram negative organisms such as E. coli or pseudomonas and gram positive bacteria such as Micrococcus or bacillus. Examples of eucaryotic organisms from which carbonyl hydrolase and their genes may be obtained include yeast such as S. cerevisiae, fungi such as Aspergillus sp., and non-human mammalian sources such as, for example, Bovine sp. from which the gene encoding the carbonyl hydrolase chymosin can be obtained. As with subtilisins, a series of carbonyl hydrolases can be obtained from various related species which have amino acid sequences which are not entirely homologous between the members of that series but which nevertheless exhibit the same or similar type of biological activity. Thus, non-human carbonyl hydrolase as used herein has a functional definition which refers to carbonyl hydrolases which are associated, directly or indirectly, with procaryotic and non-human eucaryotic sources.</p>
    <p>A "carbonyl hydrolase mutant" has an amino acid sequence which is derived from the amino acid sequence of a non-human "precursor carbonyl hydrolase". The precursor carbonyl hydrolases include naturally-occurring carbonyl hydrolases and recombinant carbonyl hydrolases. The amino acid sequence of the carbonyl hydrolase mutant is "derived" from the precursor hydrolase amino acid sequence by the substitution, deletion or insertion of one or more amino acids of the precursor amino acid sequence. Such modification is of the "precursor DNA sequence" which encodes the amino acid sequence of the precursor carbonyl hydrolase rathern than manipulation of the precursor carbonyl hydrolase per se. Suitable methods for such manipulation of the precursor DNA sequence include methods disclosed herein and in EPO Publication No. 0130756.</p>
    <p>Specific residues of B. amyloliquefaciens subtilisin are identified for substitution, insertion or deletion. These amino acid position numbers refer to those assigned to the B. amyloliquefaciens subtilisin sequence presented in FIG. 1. The invention, however, is not limited to the mutation of this particular subtilisin but extends to precursor carbonyl hydrolases containing amino acid residues which are "equivalent" to the particular identified residues in B. amyloliquefaciens subtilisin.</p>
    <p>A residue (amino acid) of a precursor carbonyl hydrolase is equivalent to a residue of B. amyloliquefaciens subtilisin if it is either homologous (i.e., corresponding in position in either primary or tertiary structure) or analagous to a specific residue or portion of that residue in B. amyloliquefaciens subtilisin (i.e., having the same or similar functional capacity to combine, react, or interact chemically).</p>
    <p>In order to establish homology to primary structure, the amino acid sequence of a precursor carbonyl hydrolase is directly comparted to the B. amyloliquefaciens subtilisin primary sequence and particularly to a set of residues known to be invariant in all subtilisins for which sequence is known (FIG. 5C). After aligning the conserved residues, allowing for necessary insertions and deletions in order to maintain alignment (i.e., avoiding the elimination of conserved residues through arbitrary deletion and insertion), the residues equivalent to particular amino acids in the primary sequence of B. amyloliquefaciens subtilisin are defined. Alignment of conserved residues preferably should conserve 100% of such residues. However, alignment of greater than 75% or as little as 50% of conserved residues is also adequate to define equivalent residues. Conservation of the catalytic triad, Asp32/His64/Ser221 should be maintained.</p>
    <p>For example, in FIGS. 5A-1 and 5A-2 the amino acid sequence of subtilisin from B. amyloliquefaciens B. subtilisin var. I168 and B. lichenformis (carlsbergensis) are aligned to provide the maximum amount of homology between amino acid sequences. A comparison of these sequences shows that there are a number of conserved residues contained in each sequence. These residues are identified in FIG. 5C.</p>
    <p>These conserved residues thus may be used to define the corresponding equivalent amino acid residues of B. amyloliquefaciens subtilisin in other carbonyl hydrolases such as thermitase derived from Thermoactinomyces. These two particular sequences are aligned in FIG. 5B to produce the maximum homology of conserved residues. As can be seen there are a number of insertions and deletions in the thermitase sequence as compared to B. amyloliquefaciens subtilisin. Thus, the equivalent amino acid of Tyr217 in B. amyloliquefaciens subtilisin in thermitase is the particular lysine shown beneath Tyr217.</p>
    <p>In FIG. 5A, the equivalent amino acid at position 217 in B. amyloliquefaciens subtilisin is Tyr. Likewise, in B. subtilis subtilisin position 217 is also occupied by Tyr but in B. licheniformis position 217 is occupied by Leu.</p>
    <p>Thus, these particular residues in thermitase, and subtilisin from B. subtilisin and B. licheniformis may be substituted by a different amino acid to produce a mutant carbonyl hydrolase since they are equivalent in primary structure to Tyr217 in B. amyloliquefaciens subtilisin. Equivalent amino acids of course are not limited to those for Tyr217 but extend to any residue which is equivalent to a residue in B. amyloliquefaciens whether such residues are conserved or not.</p>
    <p>Equivalent residues homologous at the level of tertiary structure for a precursor carbonyl hydrolase whose tertiary structure has been determined by x-ray crystallography, are defined as those for which the atomic coordinates of 2 or more of the main chain atoms of a particular amino acid residue of the precursor carbonyl hydrolase and B. amyloliquefaciens subtilisin (N on N, CA on CA, C on C, and O on O) are within 0.13 nm and preferably 0.1 nm after alignment. Alignment is achieved after the best model has been oriented and positioned to give the maximum overlap of atomic coordinates of non-hydrogen protein atoms of the carbonyl hydrolase in question to the B. amyloliquefaciens subtilisin. The best model is the crystallographic model giving the lowest R factor for experimental diffraction data at the highest resolution available. ##EQU1##</p>
    <p>Equivalent residues which are functionally analogous to a specific residue of B. amyloliquefaciens subtilisin are defined as those amino acids of the precursor carbonyl hydrolases which may adopt a conformation such that they either alter, modify or contribute to protein structure, substrate binding or catalysis in a manner defined and attributed to a specific residue of the B. amyloliquefaciens subtilisin as described herein. Further, they are those residues of the precursor carbonyl hydrolase (for which a tertiary structure has been obtained by x-ray crystallography), which occupy an analogous position to the extent that although the main chain atoms of the given residue may not satisfy the criteria of equivalence on the basis of occupying a homologous position, the atomic coordinates of at least two of the side chain atoms of the residue lie with 0.13 nm of the corresponding side chain atoms of B. amyloliquefaciens subtilisin. The three dimensional structures would be aligned as outlined above.</p>
    <p>Some of the residues identified for substitution, insertion or deletion are conserved residues whereas others are not. In the case of residues which are not conserved, the replacement of one or more amino acids is limited to substitutions which produce a mutant which has an amino acid sequence that does not correspond to one found in nature. In the case of conserved residues, such replacements should not result in a naturally occurring sequence. The carbonyl hydrolase mutants of the present invention include the nature forms of carbonyl hydrolase mutants as well as the pro- and prepro-forms of such hydrolase mutants. The prepro-forms are the preferred construction since this facilitates the expression, secretion and maturation of the carbonyl hydrolase mutants.</p>
    <p>"Prosequence" refers to a sequence of amino acids bound to the N-terminal portion of the mature form of a carbonyl hydrolase which when removed results in the appearance of the "mature" form of the carbonyl hydrolase. Many proteolytic enzymes are found in nature as translational proenzyme products and, in the absence of post-translational processing, are expressed in this fashion. The preferred prosequence for producing carbonyl hydrolase mutants, specifically subtilisin mutants, is the putative prosequence of B. amyloliquifaciens subtilisin although other subtilisin prosequences may be used.</p>
    <p>A "signal sequence" or "presequence" refers to any sequence of amino acids bound to the N-terminal portion of a carbonyl hydrolase or to the N-terminal portion of a prohydrolase which may participate in the secretion of the mature or pro forms of the hydrolase. This definition of signal sequence is a functional one, meant to include all those amino acid sequences, encoded by the N-terminal portion of the subtilisin gene or other secretable carbonyl hydrolases, which participate in the effectuation of the secretion of subtilisin or other carbonyl hydrolases under native conditions. The present invention utilizes such sequences to effect the secretion of the carbonyl hydrolase mutants as defined herein.</p>
    <p>A "prepro" form of a carbonyl hydrolase mutant consists of the mature form of the hydrolase having a prosequence operably linked to the amino-terminus of the hydrolase and a "pre" or "signal" sequence operably linked to the amino terminus of the prosequence.</p>
    <p>"Expression vector" refers to a DNA construct containing a DNA sequence which is operably linked to a suitable control sequence capable of effecting the expression of said DNA in a suitable host. Such control sequences include a promoter to effect transcription, an optional operator sequence to control such transcription, a sequence encoding suitable mRNA ribosome binding sites, and sequences which control termination of transcription and translation. The vector may be a plasmid, a phage particle, or simply a potential genomic insert. Once transformed into a suitable host, the vector may replicate and function independently of the host genome, or may, in some instances, integrate into the genome itself. In the present specification, "plasmid" and "vector" are sometimes used interchangeably as the plasmid is the most commonly used form of vector at present. However, the invention is intended to include such other forms of expression vectors which serve equivalent functions and which are, or become, known in the art.</p>
    <p>The "host cells" used in the present invention generally are procaryotic or eucaryotic hosts which preferably have been manipulated by the methods disclosed in EPO Publication No. 0130756 to render them incapable of secreting enzymatically active endoprotease. A preferred host cell for expressing subtilisin is the Bacillus strain BG2036 which is deficient in enzymatically active neutral protease and alkaline protease (subtilisin). The construction of strain BG2036 is described in detail in EPO Publicatin No. 0130756 and further described by Yang, M. Y., et al. (1984) J. Bacteriol. 160, 15-21. Such host cells are distinguishible from those disclosed in PCT Publication No. 03949 wherein enzymatically inactive mutants of intracellular proteases in E. coli are disclosed. Other host cells for expressing subtilisin include Bacillus subtilis I168 (EPO Publication No. 0130756).</p>
    <p>Host cells are transformed or transfected with vectors constructed using recombinant DNA techniques. Such transformed host cells are capable of either replicating vectors encoding the carbonyl hydrolase mutants or expressing the desired carbonyl hydrolase mutant. In the case of vectors which encode the pre or prepro form of the carbonyl hydrolase mutant, such mutants, when expressed, are typically secreted from the host cell into the host cell medium.</p>
    <p>"Operably linked" when describing the relationship between two DNA regions simply means that they are functionally related to each other. For example, a presequence is operably linked to a peptide if it functions as a signal sequence, participating in the secretion of the mature form of the protein most probably involving cleavage of the signal sequence. A promoter is operably linked to a coding sequence if it controls the transcription of the sequence; a ribosome binding site is operably linked to a coding sequence if it is positioned so as to permit translation.</p>
    <p>The genes encoding the naturally-occurring precursor carbonyl hydrolase may be obtained in accord with the general methods described in EPO Publication No. 0130756. As can be seen from the examples disclosed therein, the methods generally comprise synthesizing labelled probes having putative sequences encoding regions of the hydrolase of interest, preparing genomic libraries from organisims expressing the hydrolase, and screening the libraries for the gene of interest by hybridization to the probes. Positively hybridizing clones are then mapped and sequenced.</p>
    <p>The cloned carbonyl hydrolase is then used to transform a host cell in order to express the hydrolase. The hydrolase gene is then ligated into a high copy number plasmid. This plasmid replicates in hosts in the sense that it contains the well-known elements necessary for plasmid replication: a promoter operably linked to the gene in question (which may be supplied as the gene's own homologous promotor if it is recognized, i.e., transcribed, by the host), a transcription termination and polyadenylation region (necessary for stability of the mRNA transcribed by the host from the hydrolase gene in certain eucaryotic host cells) which is exogenous or is supplied by the endogenous terminator region of the hydrolase gene and, desirably, a selection gene such as an antibiotic resistance gene that enables continuous cultural maintenance of plasmid-infected host cells by growth in antibiotic-containing media. High copy number plasmids also contain an origin of replication for the host, thereby enabling large numbers of plasmids to be generated in the cytoplasm without chromosonal limitations. However, it is within the scope herein to integrate multiple copies of the hydrolase gene into host genome. This is facilitated by procaryotic and eucaryotic organisms which are particularly susceptible to homologous recombination.</p>
    <p>Once the carbonyl hydrolase gene has been cloned, a number of modifications are undertaken to enhance the use of the gene beyond synthesis of the naturally-ocurring precursor carbonyl hydrolase. Such modifications include the production of recombinant carbonyl hydrolases as disclosed in EPO Publication No. 0130756 and the production of carbonyl hydrolase mutants described herein.</p>
    <p>The following cassette mutagenesis method may be used to facilitate the construction and identification of the carbonyl hydrolase mutants of the present invention although other methods including site-directed mutagenesis may be used. First, the gene encoding the hydrolase is obtained and sequenced in whole or in part. Then the sequence is scanned for a point at which it is desired to make a mutation (deletion, insertion or substitution) of one or more amino acids in the expressed enzyme. The sequences flanking this point are evaluated for the presence of restriction sites for replacing a short segment of the gene with an oligonucleotide pool which when expressed will encode various mutants. Such restriction sites are preferably unique sites within the hydrolase gene so as to facilitate the replacement of the gene segment. However, any convenient restriction site which is not overly redundant in the hydrolase gene may be used, provided the gene fragments generated by restriction digestion can be reassembled in proper sequence. If restriction sites are not present at locations within a convenient distance from the selected point (from 10 to 15 nucleotides), such sites are generated by substituting nucleotides in the gene in such a fashion that neither the reading frame nor the amino acids encoded are changed in the final construction. The task of locating suitable flanking regions and evaluating the needed changes to arrive at two convenient restriction site sequences is made routine by the redundancy of the genetic code, a restriction enzyme map of the gene and the large number of different restriction enzymes. Note that if a convenient flanking restriction site is available, the above method need be used only in connection with the flanking region which does not contain a site.</p>
    <p>Mutation of the gene in order to change its sequence to conform to the desired sequence is accomplished by M13 primer extension in accord with generally known methods. Once the gene is cloned, the restriction sites flanking the sequence to be mutated are digested with the cognate restriction enzymes and a plurality of end termini-complementary oligonucleotide cassettes are ligated into the gene. The mutagenesis is enormously simplified by this method because all of the oligonucleotides can be synthesized so as to have the same restriction sites, and no synthetic linkers are necessary to create the restriction sites.</p>
    <p>The number of commercially available restriction enzymes having sites not present in the gene of interest is generally large. A suitable DNA sequence computer search program simplifies the task of finding potential 5' and 3' convenient flanking sites. A primary constraint is that any mutation introduced in creation of the restriction site must be silent to the final construction amino acid coding sequence. For a candidate restriction site 5' to the target codon a sequence must exist in the gene which contains at least all the nucleotides but for one in the recognition sequence 5' to the cut of the candidate enzyme. For example, the blunt cutting enzyme SmaI (CCC/GGG) would be a 5' candidate if a nearby 5' sequence contained NCC, CNC, or CCN. Furthermore, if N needed to be altered to C this alteraiton must leave the amino acid coding sequence intact. In cases where a permanent silent mutation is necessary to introduce a restriction site one may want to avoid the introduction of a rarely used codon. A similar situation of SmaI would apply for 3' flanking sites except the sequence NGG, GNG, or GGN must exist. The criteria for locating candidate enzymes is most relaxed for blunt cutting enzymes and most stringent for 4 base overhang enzymes. In general many candidate sites are available. For the codon-221 target described herein a BalI site (TGG/CCA) would have been engineered in one base pair 5' from the KpnI site. A 3' EcoRV site (GAT/ATC) could have been employed 11 base pairs 5' to the PstI site. A cassette having termini ranging from a blunt end up to a four base-overhang will function without difficulty. In retrospect, this hypothetical EcoRV site would have significantly shortened the oligonucleotide cassette employed (9 and 13 base pairs) thus allowing greater purity and lower pool bias problems. Flanking sites should obviously be chosen which cannot themselves ligate so that ligation of the oligonucleotide cassette can be assured in a single orientation.</p>
    <p>The mutant carbonyl hydrolases expressed upon transformation of suitable hosts are screened for enzymes exhibiting one or more properties which are substantially different from the properties of the precursor carbonyl hydrolases, e.g., changes in substrate specificity, oxidative stability, thermal stability, alkaline stability, resistance to proteolytic degradation, pH-activity profiles and the like.</p>
    <p>The carbonyl hydrolase mutants of the present invention may also be generated by random mutagenesis, See for example the methods disclosed by Shortle, D., et al. (1985) Genetics, 110, 539; Shortle, D., et al. (1986) Proteins: Structure, Function and Genetics, 1, 81; Shortle, D. (1986) J. Cell. Biochem, 30, 281; Alber, T., et al. (1985) Proc. Natl. Acad. of Sci., 82, 747; Matsumura, M., et al. (1985) J. Biochem., 260, 15298; Liao, H., et al. (1986) Proc. Natl. Acad. of Sci., 83 576; and the random mutagenesis method disclosed herein.</p>
    <p>When combined with the alkaline stability screening procedure disclosed herein, mutants obtained by random mutagenesis were identified which demonstrated either increased or decreased alkaline or thermal stability.</p>
    <p>A change in substrate specificity is defined as a difference between the kcat/Km ratio of the precursor carbonyl hydrolase and that of the hydrolase mutant. The kcat/Km ratio is a measure of catalytic efficienty. Carbonyl hydrolase mutants with increased or diminished kcat/Km ratios are described in the examples. Generally, the objective will be to secure a mutant having a greater (numerically large) kcat/Km ratio for a given substrate, thereby enabling the use of the enzyme to more efficiently act on a target substrate. A substantial change in kcat/Km ratio is preferably at least 2-fold increase or decrease. However, smaller increases or decreases in the ratio (e.g., at least 1.5-fold) are also considered substantial. An increase in kcat/Km ratio for one substrate may be accompanied by a reduction in kcat/Km ratio for another substrate. This is a shift in substrate specificity, and mutants exhibiting such shifts have utility where the precursor hydrolase is undesirable, e.g. to prevent undesired hydrolysis of a particular substrate in an admixture of substrates. Km and kcat are measured in accord with known procedures, as described in EPO Publication No. 0130756 or as described herein.</p>
    <p>Oxidative stability is measured either by known procedures or by the methods described hereinafter. A substantial change in oxidative stability is evidenced by at least about 50% increase or decrease (preferably decrease) in the rate of loss of enzyme activity when exposed to various oxidizing conditions. Such oxidizing conditions are exposure to the organic oxidant diperdodecanoic acid (DPDA) under the conditions described in the examples.</p>
    <p>Alkaline stability is measured either by known procedures or by the methods described herein. A substantial change in alkaline stability is evidenced by at least about a 5% or greater increase or decrease (preferably increase) in the half life of the enzymatic activity of a mutant when compared to the precursor carbonyl hydrolase. In the case of subtilisins, alkaline stability was measured as a function of autoproteolytic degradation of subtilisin at alkaline pH, e.g. for example, 0.1M sodium phosphate, pH 12 at 25° or 30° C.</p>
    <p>Thermal stability is measured either by known procedures or by the methods described herein. A substantial change in thermal stability is evidenced by at least about a 5% or greater increase or decrease (preferably increase) in the half-life of the catalytic activity of a mutant when exposed to a relatively high temperature and neutral pH as compared to the precursor carbonyl hydrolase. In the case of subtilisins, thermal stability is measured by the autoproteolytic degradation of subtilisin at elevated temperatures and neutral pH, e.g., for example 2 mM calcium chloride, 50 mM MOPS pH 7.0 at 59° C.</p>
    <p>The inventors have produced mutant subtilisins containing the substitution of the amino acid residues of B. amyloliquefaciens subtilisin shown in Table I. The wild type amino acid sequence and DNA sequence of B. amyloliquefaciens subtilisin is shown in FIG. 1.</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">              TABLE I______________________________________Residue    Replacement Amino Acid______________________________________Tyr21      FThr22      CSer24      CAsp32      N Q SSer33      A TAsp36      A GGly46      VAla48      E V RSer49      C LMet50      C F VAsn77      DSer87      CLys94      CVal95      CTyr104     A C D E F G H I K L M N P Q R S T V WIle107     VGly110     C RMet124     I LAla152     G SAsn155     A D H Q TGlu156     Q SGly166     A C D E F H I K L M N P Q R S T V W YGly169     A C D E F H I K L M N P Q R S T V W YLys170     E RTyr171     FPro172     E QPhe189     A C D E G H I K L M N P Q R S T V W YAsp197     R AMet199     ISer204     C R L PLys213     R TTyr217     A C D E F G H I K L M N P Q R S T V WSer221     A CMet222     A C D E F G H I K L N P Q R S T V W Y______________________________________</pre>
    
    <p>The different amino acids substituted are represented in Table I by the following single letter designations:</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">______________________________________Amino acidor residue       3-letter                    1-letterthereof          symbol  symbol______________________________________Alanine          Ala     AGlutamate        Glu     EGlutamine        Gln     QAspartate        Asp     DAsparagine       Asn     NLeucine          Leu     LGlycine          Gly     GLysine           Lys     KSerine           Ser     SValine           Val     VArginine         Arg     RThreonine        Thr     TProline          Pro     POIsoleucine       Ile     IMethionine       Met     MPhenylalanine    Phe     FTyrosine         Tyr     YCysteine         Cys     CTryptophan       Trp     WHistidine        His     H______________________________________</pre>
    
    <p>Except where otherwise indicated by context, wild-type amino acids are represented by the above three-letter symbols and replaced amino acids by the above single-letter symbols. Thus, if the methionine at residue 50 in B. amyloliquefaciens subtilisin is replaced by Phenylalanine, this mutation (mutant) may be designated Met50F or F50. Similar designations will be used for multiple mutants.</p>
    <p>In addition to the amino acids used to replace the residues disclosed in Table I, other replacements of amino acids at the residues are expected to produce mutant subtilisins having useful properties. These residues and replacement amino acids are shown in Table II.</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">              TABLE II______________________________________Residue        Replacement Amino Acid(s)______________________________________Tyr-21         LThr22          KSer24          AAsp32Ser33          GGly46Ala48Ser49Met50          L K I VAsn77          DSer87          NLys94          R QVal95          L ITyr104Met124         K AAla152         C L I T MAsn155Glu156         A T M L YGly166Gly169Tyr171         K R E QPro172         D NPhe189Tyr217Ser221Met222______________________________________</pre>
    
    <p>Each of the mutant subtilisins in Table I contain the replacement of a single residue of the B. amyloliquefaciens amino acid sequence. These particular residues were chosen to probe the influence of such substitutions on various properties of B. amyloliquefacien subtilisin.</p>
    <p>Thus, the inventors have identified Met124 and Met222 as important residues which if substituted with another amino acid produce a mutant subtilisin with enhanced oxidative stability. For Met124, Leu and Ile are preferred replacement amino acids. Preferred amino acids for replacement of Met222 are disclosed in EPO Publication No. 0130756.</p>
    <p>Various other specific residues have also been identified as being important with regard to substrate specificity. These residues include Tyr104, Ala152, Glu156, Gly166, Gly169, Phe189 and Tyr217 for which mutants containing the various replacement amino acids presented in Table I have already been made, as well as other residues presented below for which mutants have yet to be made.</p>
    <p>The identification of these residues, including those yet to be mutated, is based on the inventors' high resolution crystal structure of B. amyloliquefaciens subtilisin to 1.8 A (see Table III), their experience with in vitro mutagenesis of subtilisin and the literature on subtilisin. This work and the above referenced x-ray crystal structures of subtilisin containing covalently bound peptide inhibitors, product complexes and transition state analogs has helped in identifying an extended peptide binding cleft in subtilisin. This substrate binding cleft together with substrate is schematically diagramemed in FIG. 2, according to the nomenclature of Schechter, I., et al. (1967) Biochem Bio. Res. Commun. 27, 157. The scissile bond in the substrate is identified by an arrow. The P and P' designations refer to the amino acids which are positioned respectively toward the amino or carboxy terminus relative to the scissle bond. The S and S' designations refer to subsites in the substrate binding cleft of subtilisin which interact with the corresponding substrate amino acid residues.</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">______________________________________1      ALA      N      19.434   53.195                                 -21.7561      ALA      C      18.731   50.995                                 -21.3241      ALA      CB     21.099   51.518                                 -21.1832      GLN      CA     17.219   49.008                                 -21.4342      GLN      O      18.765   47.165                                 -21.6912      GLN      CG     15.028   47.905                                 -21.9272      GLN      DE1    13.023   48.612                                 -22.8673      SER      N      17.477   47.205                                 -19.8523      SER      C      16.735   44.918                                 -19.4903      SER      CB     18.588   45.838                                 -18.0694      VAL      N      16.991   43.646                                 -19.7254      VAL      C      16.129   41.934                                 -18.2904      VAL      CB     16.008   41.622                                 -20.8224      VAL      CG2    16.037   42.266                                 -22.1865      PRO      CA     15.384   41.415                                 -16.0275      PRO      O      14.885   39.763                                 -17.1465      PRO      CG     13.841   43.215                                 -15.9216      TYR      N      16.363   39.240                                 -15.4876      TYR      C      15.359   36.975                                 -15.5286      TYR      CB     17.824   37.323                                 -14.8346      TYR      CD1    18.437   35.452                                 -16.3466      TYR      CE1    18.535   34.070                                 -16.6536      TYR      CZ     18.222   33.154                                 -15.6287      GLY      N      14.464   37.362                                 -14.6307      GLY      C      12.400   36.535                                 -15.6708      VAL      N      12.441   37.529                                 -16.5418      VAL      C      12.363   36.433                                 -18.7358      VAL      CB     11.765   38.900                                 -18.5678      VAL      CG2    10.991   39.919                                 -17.7339      SER      CA     14.419   35.342                                 -19.5629      SER      O      14.112   33.014                                 -19.8019      SER      OG     16.162   36.747                                 -20.35810     GLN      CA     13.964   32.636                                 -16.87610     GLN      O      12.785   30.642                                 -17.41310     GLN      CG     14.295   31.617                                 -14.58810     GLN      DE1    14.554   33.068                                 -12.74411     ILE      N      11.625   32.575                                 -17.67011     ILE      C      10.209   31.792                                 -19.60511     ILE      CB     9.132    32.669                                 -17.47511     ILE      CG2    9.162    32.655                                 -15.94111     IYS      N      11.272   32.185                                 -20.27712     LYS      C      10.456   33.006                                 -22.52212     LYS      CB     11.257   30.646                                 -22.21612     LYS      CD     12.543   28.517                                 -22.15912     LYS      NZ     14.476   27.680                                 -20.93513     ALA      CA     9.325    35.198                                 -22.63113     ALA      O      9.338    35.804                                 -24.90114     PRO      N      11.332   35.950                                 -23.89314     PRO      C      11.786   35.557                                 -26.31714     PRO      CB     13.461   36.580                                 -24.69214     PRO      CD     12.281   35.936                                 -22.75815     ALA      CA     11.379   33.450                                 -27.36715     ALA      O      10.008   33.710                                 -29.27816     LEU      N      9.085    34.138                                 -27.24016     LEU      C      7.912    35.925                                 -28.52116     LEU      CB     6.746    34.623                                 -26.69816     LEU      CD1    5.001    33.234                                 -27.80917     HIS      N      8.665    36.828                                 -27.92217     HIS      C      9.510    37.981                                 -29.89017     HIS      CB     9.708    39.100                                 -27.65217     HIS      ND1    9.930    39.887                                 -25.27217     HIS      CE1    9.226    39.914                                 -24.14418     SER      H      10.443   37.033                                 -30.0221      ALA      CA     19.811   51.774                                 -21.9651      ALA      O      18.376   51.197                                 -20.1752      GLN      N      18.268   49.886                                 -22.0412      GLN      C      17.875   47.706                                 -20.9922      GLN      CB     16.125   48.760                                 -22.4492      GLN      CD     13.912   47.762                                 -22.9302      GLN      NE2    14.115   46.917                                 -23.9263      SER      CA     17.950   45.868                                 -19.4373      SER      O      15.590   45.352                                 -19.2293      SER      OG     17.682   46.210                                 -17.0494      VAL      CA     15.946   42.619                                 -19.6394      VAL      O      17.123   41.178                                 -18.0864      VAL      CG1    14.874   40.572                                 -20.7415      PRO      N      15.239   42.106                                 -17.3315      PRO      C      15.501   39.905                                 -16.2495      PRO      CB     14.150   41.880                                 -15.2635      PRO      CD     14.044   42.986                                 -17.4176      TYR      CA     16.628   37.803                                 -15.7156      TYR      O      15.224   35.943                                 -16.2356      TYR      CG     18.021   35.847                                 -15.0556      TYR      CD2    17.696   34.908                                 -14.0716      TYR      CE2    17.815   33.539                                 -14.3796      TYR      OH     18.312   31.838                                 -15.9967      GLY      CA     13.211   36.640                                 -14.3767      GLY      O      11.747   35.478                                 -15.8838      VAL      CA     11.777   37.523                                 -17.8368      VAL      O      11.639   35.716                                 -19.6708      VAL      CG1    11.106   38.893                                 -19.9439      SER      N      13.661   36.318                                 -18.7759      SER      C      14.188   33.920                                 -18.9659      SER      CB     15.926   35.632                                 -19.50510     GLN      N      14.115   33.887                                 -17.66210     GLN      C      12.687   31.887                                 -17.27710     GLN      CB     14.125   32.885                                 -15.41010     GLN      CD     14.486   31.911                                 -13.14710     GLN      NE2    14.552   30.960                                 -12.25111     ILE      CA     10.373   31.904                                 -18.10211     ILE      O      9.173    31.333                                 -20.18011     ILE      CG1    9.066    34.117                                 -18.04911     ILE      CD1    7.588    34.648                                 -17.92312     LYS      CA     11.388   32.119                                 -21.72212     LYS      O      10.173   32.703                                 -23.68612     LYS      CG     12.283   29.830                                 -21.42312     LYS      CE     13.023   27.467                                 -21.16613     ALA      N      10.109   34.138                                 -21.99113     ALA      C      10.026   35.716                                 -23.86313     ALA      CB     8.885    36.195                                 -21.56514     PRO      CA     11.985   36.430                                 -25.22014     PRO      O      11.778   36.047                                 -27.44514     PRO      CG     13.328   36.978                                 -23.22115     ALA      N      11.560   34.236                                 -26.12915     ALA      C      10.082   33.795                                 -28.03215     ALA      CB     11.552   31.969                                 -27.06216     LEU      CA     7.791    34.558                                 -27.82816     LEU      O      7.342    36.126                                 -29.58816     LEU      CG     5.790    33.465                                 -26.52216     LEU      CD2    6.694    32.287                                 -26.28317     HIS      CA     8.890    38.151                                 -28.53017     HIS      O      9.107    38.622                                 -30.85617     HIS      CG     9.185    39.288                                 -26.26217     HIS      CD2    8.008    38.924                                 -25.69417     HIS      NE2    8.079    39.328                                 -24.38118     SER      CA     11.109   36.739                                 -31.32218     SER      C      10.159   36.123                                 -32.35318     SER      CB     12.311   35.799                                 -31.17219     GLN      N      9.080    35.485                                 -31.94319     GLN      C      7.142    36.111                                 -33.30319     GLN      CB     7.221    33.849                                 -32.28019     GLN      CD     6.923    31.707                                 -31.18119     GLN      NE2    7.362    30.852                                 -30.25620     GLY      CA     6.369    38.387                                 -32.85920     GLY      O      4.263    39.276                                 -32.21521     TYR      CA     4.118    37.831                                 -29.76321     TYR      O      5.422    38.074                                 -27.75621     TYR      CG     2.973    35.784                                 -30.70821     TYR      CD2    3.650    34.794                                 -31.39721     TYR      CE2    3.193    34.261                                 -32.58821     TYR      OH     1.501    34.241                                 -34.25022     THR      CA     4.262    40.527                                 -27.12922     THR      O      3.287    41.725                                 -25.32522     THR      OG1    4.319    42.457                                 -28.59723     GLY      N      1.939    40.285                                 -26.45323     GLY      C      -0.157   41.631                                 -26.11824     SER      N      -0.023   41.967                                 -27.37124     SER      C      -2.383   42.626                                 -27.86424     SER      CB     -0.734   43.120                                 -29.52025     ASN      N      -3.059   43.692                                 -27.51525     ASN      C      -5.015   42.875                                 -26.20525     ASN      CB     -5.165   43.227                                 -28.70025     ASN      OD1    -4.965   43.767                                 -31.08326     VAL      N      -4.177   42.449                                 -25.29226     VAL      C      -4.792   42.652                                 -22.98726     VAL      CB     -3.714   40.503                                 -23.82126     VAL      CG2    -3.598   39.576                                 -25.01827     LYS      CA     -6.133   43.524                                 -21.17527     LYS      O      -6.405   41.873                                 -19.41327     LYS      CG     -8.046   44.575                                 -22.49027     LYS      CE     -10.304  45.497                                 -23.13728     VAL      N      -4.818   43.462                                 -19.20028     VAL      C      -4.758   43.959                                 -16.82828     VAL      CB     -2.926   42.666                                 -17.93228     VAL      CG2    -2.667   41.805                                 -19.17329     ALA      CA     -5.747   44.330                                 -14.63929     ALA      O      -4.666   42.845                                 -13.10430     VAL      N      -4.057   45.033                                 -13.07230     VAL      C      -3.958   45.409                                 -10.68130     VAL      CB     -1.886   45.810                                 -12.14930     VAL      CG2    -1.053   45.236                                 -13.30731     ILF      CA     -5.328   44.846                                 -8.67931     ILE      O      -3.825   43.915                                 -6.99731     ILE      CG1    -7.298   43.707                                 -9.79831     ILE      CD1    -8.617   42.856                                 -9.71732     ASP      CA     -2.944   46.467                                 -6.25532     ASP      O      -4.197   48.418                                 -5.50232     ASP      CG     -0.483   45.702                                 -6.27332     ASP      OD2    -0.081   46.429                                 -5.33033     SER      CA     -1.895   49.857                                 -4.80133     SER      O      -1.706   52.136                                 -5.36333     SER      OG     0.535    50.025                                 -4.77434     GLY      CA     -2.255   51.728                                 -8.16534     GLY      O      -0.144   50.831                                 -8.76135     ILE      CA     0.208    52.438                                 -10.99535     ILE      O      -0.327   54.638                                 -11.74435     ILE      CG1    -0.530   50.210                                 -12.09735     ILE      CD1    -0.962   49.485                                 -13.42436     ASP      CA     2.359    55.618                                 -11.23218     SER      O      10.547   36.112                                 -33.53418     SER      OG     13.321   36.450                                 -30.39919     GLN      CA     8.082    34.962                                 -32.87819     GLN      O      6.297    35.972                                 -34.21919     GLN      CG     7.975    32.602                                 -31.82319     GLN      DE1    5.719    31.833                                 -31.44420     GLY      N      7.205    37.223                                 -32.58720     GLY      C      5.181    38.492                                 -31.88021     TYR      N      5.202    37.801                                 -30.76121     TYR      C      4.579    38.552                                 -28.52521     TYR      CB     3.498    36.431                                 -29.44321     TYR      CD1    1.795    36.332                                 -31.25821     TYR      CE1    1.306    35.797                                 -32.44621     TYR      C2     2.003    34.755                                 -33.06722     THR      N      3.902    39.680                                 -28.28822     THR      C      3.091    40.922                                 -26.24422     THR      CB     5.133    41.759                                 -27.61122     THR      CG2    6.476    41.323                                 -28.22923     GLY      CA     0.809    40.600                                 -25.54223     GLY      O      -1.013   42.095                                 -25.33024     SER      CA     -0.897   42.957                                 -28.01224     SER      O      -2.813   41.508                                 -28.16024     SER      OG     0.563    43.652                                 -29.72825     ASN      CA     -4.519   43.687                                 -27.39325     ASN      O      -6.233   42.668                                 -26.19025     ASN      CG     -4.960   44.170                                 -29.88525     ASN      ND2    -4.747   45.461                                 -29.59426     VAL      CA     -4.674   41.679                                 -24.14326     VAL      O      -3.858   43.419                                 -22.68926     VAL      CG1    -4.160   39.802                                 -22.54827     LYS      N      -5.910   42.613                                 -22.30127     LYS      C      -5.815   42.782                                 -19.84127     LYS      CB     -7.590   43.981                                 -21.14927     LYS      CD     -9.321   45.302                                 -22.02027     LYS      NZ     -9.686   46.253                                 -24.26428     VAL      CA     -4.457   42.950                                 -17.89728     VAL      O      -4.209   45.095                                 -16.81728     VAL      CG1    -2.466   42.105                                 -16.58929     ALA      N      -5.484   43.527                                 -15.81329     ALA      C      -4.750   44.010                                 -13.55329     ALA      C9     -7.172   44.107                                 -14.10130     VAL      CA     -3.146   44.962                                 -11.91030     VAL      O      -4.155   46.648                                 -10.57830     VAL      CG1    -0.990   45.901                                 -10.90031     ILE      N      -4.514   44.515                                 -9.87731     ILE      C      -4.346   44.933                                 -7.54631     ILE      CB     -6.457   43.776                                 -8.50131     ILE      CG2    -7.278   44.038                                 -7.22532     ASP      N      -4.044   46.193                                 -7.22732     ASP      C      -3.071   47.889                                 -5.70532     ASP      CB     -1.695   46.129                                 -7.09232     ASP      DD1    0.034    44.592                                 -6.57633     SER      N      -1.931   48.512                                 -5.39433     SER      C      -1.952   50.976                                 -5.80833     SER      CB     -0.651   49.922                                 -3.93934     GLY      N      -2.173   50.740                                 -7.08434     GLY      C      -1.035   51.648                                 -9.05735     ILE      N      -0.965   52.431                                 -10.10235     ILE      C      0.568    53.919                                 -11.26335     ILE      CB     -0.042   51.694                                 -12.36735     ILE      CG2    1.149    51.741                                 -13.36236     ASP      N      1.816    54.253                                 -10.97136     ASP      C      2.281    55.956                                 -12.70236     ASP      O      3.004    55.471                                 -13.57936     ASP      CG     4.339    57.099                                 -10.80436     ASP      DD2    5.448    57.277                                 -10.26337     SER      CA     1.183    57.221                                 -14.51237     SER      O      2.545    58.303                                 -16.15137     SER      OG     -0.090   59.133                                 -13.87938     SER      CA     4.261    59.505                                 -14.48738     SER      O      6.543    59.251                                 -15.28538     SER      OG     5.376    59.865                                 -12.23439     HIS      CA     6.637    56.574                                 -15.29139     HIS      O      5.738    55.878                                 -17.41939     HIS      CG     8.014    54.609                                 -14.45639     HIS      CD2    8.769    54.345                                 -13.38939     HIS      NE2    9.986    53.910                                 -13.80840     PRO      CA     7.988    56.697                                 -18.83140     PRO      O      8.032    55.097                                 -20.57840     PRO      CG     10.053   57.405                                 -17.90241     ASP      N      8.481    54.328                                 -18.48541     ASP      DD1    10.325   51.395                                 -20.42941     ASP      CB     9.799    52.239                                 -18.22441     ASP      C      7.311    52.163                                 -18.83942     LEU      N      6.185    52.803                                 -18.55842     LEU      C      3.924    52.907                                 -19.37642     LEU      CB     4.421    52.158                                 -17.00842     LEU      CD1    4.535    51.546                                 -14.58143     LYS      N      3.018    52.135                                 -19.94643     LYS      C      0.637    52.156                                 -20.01843     LYS      CB     2.021    52.389                                 -22.16943     LYS      CD     0.998    52.862                                 -24.33943     LYS      NZ     0.337    51.757                                 -26.41844     VAL      CA     -1.407   52.639                                 -18.76544     VAL      O      -2.623   53.906                                 -20.43444     VAL      CG1    -2.724   52.941                                 -16.58245     ALA      N      -3.494   51.951                                 -19.87145     ALA      C      -5.841   52.507                                 -20.05345     ALA      CB     -4.831   50.580                                 -21.38946     GLY      CA     -7.082   52.837                                 -18.00146     GLY      O      -5.938   52.006                                 -16.03547     GLY      CA     -8.014   52.246                                 -14.38847     GLY      O      -9.988   53.481                                 -14.18548     ALA      CA     -10.255  52.870                                 -11.38248     ALA      O      -9.066   51.720                                 -9.72549     SER      N      -10.149  53.547                                 -9.03749     SER      C      -10.947  52.986                                 -6.78349     SER      CB     -9.092   54.588                                 -7.02950     MET      N      -10.835  52.007                                 -5.93250     MET      C      -11.463  51.962                                 -3.56150     MET      CB     -12.012  50.018                                 -4.99650     MET      SD     -13.460  49.889                                 -7.25651     VAL      N      -10.427  52.760                                 -3.42251     VAL      C      -10.630  54.562                                 -1.90751     VAL      CB     -8.443   53.155                                 -2.00051     VAL      CG2    -7.764   51.815                                 -2.30252     PRO      CA     -12.372  55.933                                 -0.82152     PRO      O      -11.771  58.220                                 -0.92552     PRO      CG     -13.583  54.103                                 0.08553     SER      N      -10.442  56.906                                 0.29953     SER      C      -8.420   58.245                                 -0.32653     SER      CB     -9.004   57.707                                 2.06954     GLU      N      -8.254   57.523                                 -1.39354     GLU      C      -7.767   57.303                                 -3.78554     GLU      CB     -6.134   56.599                                 -2.15454     GLU      CD     -4.066   56.062                                 -0.97036     ASP      CB     3.712    55.720                                 -10.51436     ASP      DD1    3.755    57.974                                 -11.42937     SER      N      1.304    56.822                                 -13.11137     SER      C      2.377    58.095                                 -14.94937     SER      CB     -0.093   58.049                                 -14.78838     SER      N      3.163    58.614                                 -14.00138     SER      C      5.466    58.705                                 -14.99238     SER      CB     4.742    60.435                                 -13.39839     HIS      N      5.454    57.390                                 -14.89239     HIS      C      6.681    56.401                                 -16.77839     HIS      CB     6.637    55.203                                 -14.51539     HIS      ND1    8.795    54.356                                 -15.56139     HIS      CE1    9.970    53.930                                 -15.13040     PRO      N      7.807    56.836                                 -17.38740     PRO      C      8.156    55.280                                 -19.35740     PRO      CB     9.247    57.533                                 -19.16140     PRO      CD     8.988    57.452                                 -16.77641     ASP      DD2    11.148   50.399                                 -18.66841     ASP      CG     10.473   51.307                                 -19.21141     ASP      CA     8.645    52.959                                 -18.96641     ASP      O      7.396    50.947                                 -18.97742     LEU      CA     4.892    52.147                                 -18.46642     LEU      O      3.993    54.163                                 -19.49042     LEU      CG     5.182    51.363                                 -15.94642     LEU      CD2    5.273    49.877                                 -16.35043     LYS      CA     1.893    52.685                                 -20.72143     LYS      O      0.504    50.920                                 -19.82043     LYS      CG     0.685    52.436                                 -22.91043     LYS      CE     -0.180   52.584                                 -25.26044     VAL      N      -0.191   53.035                                 -19.49044     VAL      C      -2.571   52.887                                 -19.73144     VAL      CB     -1.480   53.351                                 -17.38344     VAL      CG2    -0.197   53.194                                 -16.55345     ALA      CA     -4.619   51.977                                 -20.81045     ALA      O      -6.703   53.085                                 -20.70346     GLY      N      -5.910   52.356                                 -18.76846     GLY      C      -6.987   52.443                                 -16.53847     GLY      N      -8.092   52.658                                 -15.79347     GLY      C      -9.179   52.757                                 -13.57248     ALA      N      -9.221   52.446                                 -12.33048     ALA      C      -9.790   52.675                                 -9.96848     ALA      CB     -11.558  52.100                                 -11.61749     SER      CA     -9.752   53.355                                 -7.65249     SER      O      -11.972  53.677                                 -6.90849     SER      OG     -8.879   54.255                                 -5.65050     MET      CA     -11.852  51.549                                 -4.97450     MET      O      -11.997  51.398                                 -2.57550     MET      CG     -11.912  49.463                                 -6.38950     MET      CE     -12.808  50.111                                 -8.90351     VAL      CA     -9.968   53.170                                 -2.06751     VAL      O      -10.237  55.437                                 -2.68251     VAL      CG1    -7.892   53.579                                 -0.63152     PRO      N      -11.621  54.693                                 -1.05652     PRO      C      -11.490  57.123                                 -0.44052     PRO      CB     -13.400  55.594                                 0.24452     PRO      CD     -12.164  53.620                                 -0.17553     SER      CA     -9.538   57.982                                 0.68253     SER      O      -7.679   59.224                                 -0.03853     SER      OG     -8.256   56.521                                 2.12754     GLU      CA     -7.204   57.648                                 -2.42154     GLU      O      -7.533   56.243                                 -4.37954     GLU      CG     -5.289   56.959                                 -0.92754     GLU      OE1    -3.545   55.694                                 -1.96854     GLU      OE2    -3.900   55.777                                 0.27155     THR      CA     -9.433   58.121                                 -5.44155     THR      O      -9.433   57.919                                 -7.81055     THR      OG1    -9.885   60.510                                 -5.41856     ASN      N      -7.482   58.403                                 -6.87756     ASN      DD1    -5.075   58.967                                 -10.33756     ASN      CB     -5.898   59.694                                 -8.20856     ASN      C      -6.012   57.094                                 -8.30557     PRO      N      -6.362   56.261                                 -9.25857     PRO      CD     -7.384   56.433                                 -10.27257     PRO      CA     -5.679   54.961                                 -9.33257     PRO      O      -3.509   54.128                                 -9.94558     PHE      CA     -2.747   56.577                                 -11.22258     PHE      O      -0.635   57.497                                 -10.68058     PHE      CG     -3.983   56.968                                 -13.35758     PHE      CD2    -5.211   57.630                                 -13.45958     PHE      CE2    -6.194   57.095                                 -14.27659     GLN      N      -2.044   57.119                                 -8.99059     GLN      C      -0.807   56.403                                 -7.00059     GLN      CB     -1.862   58.668                                 -7.08959     GLN      CD     -1.790   60.157                                 -5.15059     GLN      NE2    -2.959   59.685                                 -4.74260     ASP      CA     0.851    54.792                                 -6.30460     ASP      O      2.827    55.550                                 -5.23160     ASP      CG     2.077    52.538                                 -6.38060     ASP      DD2    2.915    51.841                                 -7.03061     ASN      ND2    -1.364   57.747                                 -2.34761     ASN      CG     -0.040   57.670                                 -2.39961     ASN      CA     1.557    55.734                                 -2.70061     ASN      O      2.933    54.862                                 -0.90262     ASN      CA     2.877    52.348                                 -1.70962     ASN      O      4.951    51.313                                 -1.77062     ASN      CG     2.371    50.103                                 -0.69762     ASN      ND2    2.622    50.208                                 0.60163     SER      CA     5.189    51.696                                 -4.70963     SER      O      5.593    49.790                                 -6.26963     SER      OG     6.871    50.698                                 -3.41864     MIS      CA     3.994    48.059                                 -4.93564     HIS      O      3.861    46.974                                 -7.10864     HIS      CG     3.144    46.021                                 -3.72664     HIS      CD2    4.054    45.194                                 -3.13564     HIS      NE2    3.556    43.920                                 -3.36865     GLY      CA     1.552    48.264                                 -7.83065     GLY      O      2.230    48.078                                 -10.13466     THR      CA     4.064    50.117                                 -9.95466     THR      O      5.333    48.789                                 -11.46166     THR      OG1    3.637    52.425                                 -9.40667     HIS      N      5.685    48.443                                 -9.27467     HIS      C      6.091    46.141                                 -10.14367     HIS      CB     7.300    47.071                                 -8.06467     HIS      ND1    8.590    44.907                                 -8.27667     HIS      CE1    9.857    44.491                                 -8.29968     VAL      N      4.892    45.749                                 -9.73168     VAL      C      3.856    44.860                                 -11.74068     VAL      CB     2.939    44.252                                 -9.38668     VAL      CG2    3.319    43.705                                 -8.00069     ALA      CA     3.037    46.468                                 -13.42969     ALA      O      4.028    45.913                                 -15.56570     GLY      N      5.340    46.782                                 -13.91470     GLY      C      7.046    45.370                                 -15.02171     THR      N      6.820    44.431                                 -14.13871     THR      C      6.224    42.506                                 -15.54371     THR      CB     7.119    42.070                                 -13.19155     THR      N      -8.571   58.251                                 -4.24955     THR      C      -8.764   58.139                                 -6.77955     THR      CB     -10.586  59.200                                 -5.30355     THR      CG2    -11.432  59.143                                 -4.01756     ASN      ND2    -4.930   61.179                                 -9.88156     ASN      CG     -5.273   59.925                                 -9.55556     ASN      CA     -6.762   58.425                                 -8.20056     ASN      O      -5.104   56.866                                 -7.47057     PRO      CG     -7.123   55.257                                 -11.17757     PRO      CB     -6.644   54.178                                 -10.23557     PRO      C      -4.301   55.082                                 -9.96658     PHE      N      -3.998   56.262                                 -10.49158     PHE      C      -1.712   57.129                                 -10.25358     PHE      CB     -2.943   57.502                                 -12.42358     PHE      CD1    -3.756   55.780                                 -14.05958     PHE      CB1    -4.722   55.255                                 -14.92858     PHE      CZ     -5.949   55.939                                 -15.05159     GLN      CA     -1.172   57.583                                 -7.93459     GLN      O      -1.639   56.083                                 -6.11559     GLN      CG     -0.942   59.261                                 -6.03459     GLN      DE1    -1.404   61.288                                 -4.83660     ASP      N      0.410    55.895                                 -7.21160     ASP      C      1.631    55.267                                 -5.09060     ASP      CB     1.596    53.744                                 -7.18860     ASP      DD1    1.746    52.337                                 -5.19061     ASN      N      0.959    55.265                                 -3.95061     ASN      DD1    0.666    58.566                                 -2.87561     ASN      CB     0.531    56.401                                 -1.78461     ASN      C      2.291    54.632                                 -1.94062     ASN      N      2.210    53.434                                 -2.46862     ASN      C      4.124    51.893                                 -2.47962     ASN      CB     1.783    51.319                                 -1.42162     ASN      DD1    2.633    49.077                                 -1.34363     SER      N      4.152    52.104                                 -3.76163     SER      C      5.071    50.256                                 -5.20963     SER      CB     6.523    51.958                                 -4.01264     HIS      N      4.202    49.475                                 -4.63964     HIS      C      3.366    47.759                                 -6.26164     HIS      CB     3.184    47.501                                 -3.74764     HIS      ND1    2.107    45.247                                 -4.24164     HIS      CE1    2.416    43.966                                 -4.05465     GLY      N      2.287    48.428                                 -6.58765     GLY      C      2.392    48.636                                 -9.03766     THR      N      3.233    49.659                                 -8.83266     THR      C      5.089    49.009                                 -10.29166     THR      CB     4.744    51.511                                 -9.66766     THR      CG2    5.536    52.078                                 -10.84967     HIS      CA     6.703    47.361                                 -9.45867     HIS      O      6.649    45.638                                 -11.15067     HIS      CG     8.595    46.275                                 -8.14867     HIS      CD2    9.904    46.678                                 -8.07667     HIS      NS2    10.678   45.514                                 -8.18668     VAL      CA     4.142    44.607                                 -10.26668     VAL      O      4.114    43.942                                 -12.53568     VAL      CG1    1.960    43.260                                 -10.02069     ALA      N      3.373    46.049                                 -12.11369     ALA      C      4.193    46.390                                 -14.41169     ALA      CB     2.332    47.851                                 -13.38670     GLY      CA     6.595    46.805                                 -14.67070     GLY      O      7.604    45.154                                 -16.11971     THR      CA     7.177    43.019                                 -14.44671     THR      O      6.602    41.828                                 -16.49571     THR      OG1    8.191    42.592                                 -12.39071     THR      CG2    7.274    40.583                                 -13.59672     VAL      CA     3.976    42.491                                 -16.48472     VAL      O      4.341    42.380                                 -18.86072     VAL      CG1    1.512    42.480                                 -17.17073     ALA      N      4.504    44.417                                 -17.88073     ALA      C      5.433    46.333                                 -19.35573     ALA      CB     3.107    45.441                                 -19.43374     ALA      CA     7.470    47.591                                 -18.85974     ALA      O      7.959    46.640                                 -21.05475     LEU      N      7.650    48.784                                 -21.03975     LEU      C      9.192    48.568                                 -22.96675     LEU      CB     7.548    50.471                                 -22.80975     LEU      CD1    6.079    52.436                                 -22.30076     ASN      N      9.147    48.103                                 -24.16976     ASN      DD1    10.950   45.840                                 -27.92876     ASN      CB     10.010   46.651                                 -25.90876     ASN      C      10.783   49.048                                 -25.64377     ASN      N      11.804   49.664                                 -25.07177     ASN      C      13.707   51.029                                 -25.34877     ASN      CB     11.335   52.076                                 -25.11777     ASN      DD1    12.032   51.346                                 -22.91778     SER      N      14.125   52.267                                 -25.16478     SER      C      15.810   52.742                                 -23.43678     SER      CB     15.905   53.941                                 -25.58779     ILE      N      14.858   52.565                                 -22.52979     ILE      C      14.617   51.683                                 -20.23079     ILE      CB     14.471   54.174                                 -20.69779     ILE      CG2    14.997   55.320                                 -21.61280     GLY      N      14.995   51.768                                 -18.98180     GLY      C      14.612   49.448                                 -18.21981     VAL      N      13.513   48.766                                 -17.98081     VAL      C      12.511   46.919                                 -19.21781     VAL      CB     13.001   46.755                                 -16.67781     VAL      CG2    11.638   47.261                                 -16.23182     LEU      CA     11.312   45.020                                 -20.25682     LEU      O      10.858   43.356                                 -18.60082     LEU      CG     11.430   43.568                                 -22.36682     LEU      CD2    12.359   42.675                                 -23.19283     GLY      CA     8.133    43.321                                 -19.11483     GLY      O      8.546    41.822                                 -21.02684     VAL      CA     6.973    39.807                                 -19.88884     VAL      O      6.424    39.472                                 -22.19484     VAL      CG1    5.680    37.677                                 -19.55785     ALA      N      5.156    40.926                                 -21.02485     ALA      C      4.213    42.683                                 -22.39685     ALA      CB     2.846    40.663                                 -21.74886     PRO      CA     5.413    44.635                                 -23.20586     PRO      O      4.291    46.605                                 -23.84986     PRO      CG     7.030    43.468                                 -24.54687     SER      N      3.548    44.676                                 -24.76987     SER      C      1.103    45.132                                 -24.89787     SER      CB     2.401    44.777                                 -26.92788     ALA      N      1.017    44.564                                 -23.74288     ALA      CA     -0.273   44.353                                 -23.08488     ALA      O      -0.174   46.717                                 -22.43589     SER      OG     -4.146   47.102                                 -24.28089     SER      CA     -3.001   46.867                                 -22.22789     SER      O      -3.793   45.864                                 -20.20990     LEU      CA     -2.378   47.667                                 -18.59390     LEU      O      -3.582   49.604                                 -18.21590     LEU      CG     -0.233   47.851                                 -17.17490     LEU      CD2    1.160    48.524                                 -17.04791     TYR      CA     -5.258   48.678                                 -16.13772     VAL      N      4.930    42.887                                 -15.42772     VAL      C      4.312    43.084                                 -17.83172     VAL      CB     2.516    42.867                                 -16.08572     VAL      CG2    2.142    42.327                                 -14.72373     ALA      CA     4.587    45.091                                 -19.16773     ALA      O      5.062    47.188                                 -20.21674     ALA      N      6.544    46.429                                 -18.63574     ALA      C      7.740    47.648                                 -20.34274     ALA      CB     8.653    47.446                                 -17.92575     LEU      CA     7.812    48.968                                 -22.45675     LEU      O      10.162   48.750                                 -22.25375     LEU      CG     6.123    50.913                                 -22.37975     LEU      CD2    5.096    50.442                                 -23.40576     ASN      ND2    12.285   46.432                                 -26.30476     ASN      CG     11.195   46.274                                 -26.80276     ASN      CA     10.359   47.738                                 -24.93876     ASN      O      10.157   49.479                                 -26.61977     ASN      CA     12.220   50.957                                 -25.68177     ASN      O      14.364   49.979                                 -25.31377     ASN      CG     11.250   52.027                                 -23.61677     ASN      ND2    10.294   52.741                                 -23.02578     SER      CA     15.513   52.614                                 -24.90678     SER      O      16.982   53.071                                 -23.16478     SER      OG     15.926   53.870                                 -26.99979     ILE      CA     15.155   52.784                                 -21.12079     ILE      O      13.843   50.841                                 -20.67979     ILE      CG1    12.945   54.032                                 -20.81479     ILE      CD1    12.135   55.176                                 -20.15580     GLY      CA     14.476   50.940                                 -17.91380     GLY      O      15.719   48.994                                 -18.54481     VAL      CA     13.411   47.286                                 -18.06181     VAL      D      12.260   47.739                                 -20.11781     VAL      CG1    14.030   47.084                                 -15.57382     VEU      N      12.126   45.645                                 -19.21682     LEU      C      10.390   44.028                                 -19.51082     LEU      CB     12.206   44.219                                 -21.22982     LEU      CD1    10.796   44.657                                 -23.22383     GLY      N      9.131    44.180                                 -19.81683     GLY      C      8.027    42.011                                 -19.92584     VAL      N      7.272    41.112                                 -19.28384     VAL      C      6.164    40.030                                 -21.14084     VAL      CB     6.256    38.920                                 -18.84184     VAL      CG2    7.190    38.507                                 -17.70585     ALA      CA     4.217    41.194                                 -22.15885     ALA      O      3.260    43.401                                 -22.03086     PRO      N      5.240    43.186                                 -23.05986     PRO      C      4.321    45.371                                 -23.94786     PRO      CB     6.822    44.784                                 -23.81386     PRO      CD     6.377    42.440                                 -23.63687     SER      CA     2.489    45.324                                 -25.52987     SER      O      0.162    45.513                                 -25.61987     SER      OG     3.591    45.143                                 -27.58388     ALA      CB     -0.163   43.510                                 -21.82888     ALA      C      -0.898   45.717                                 -22.69089     SER      N      -2.219   45.691                                 -22.67889     SER      CB     -4.343   46.903                                 -22.89889     SER      C      -3.136   46.780                                 -20.72790     LEU      N      -2.446   47.656                                 -20.03790     LEU      C      -3.483   48.430                                 -17.86490     LEU      CB     -0.951   48.273                                 -18.42690     LEU      CD1    -0.028   46.341                                 -17.21991     TYR      N      -4.264   47.944                                 -16.93891     TYR      C      -4.873   48.750                                 -14.68591     TYR      D      -4.496   47.749                                 -14.0291     TYR      CG     -7.094   48.237                                 -17.74191     TYR      CD2    -7.971   49.275                                 -18.14991     TYR      CE2    -8.315   49.421                                 -19.49291     TYR      OH     -8.182   48.752                                 -21.76492     ALA      CA     -4.549   50.199                                 -12.70792     ALA      O      -6.723   50.898                                 -12.05093     VAL      N      -5.959   48.993                                 -11.12993     VAL      C      -6.708   49.014                                 -8.89993     VAL      CB     -7.957   47.555                                 -10.61193     VAL      CG2    -8.195   47.370                                 -12.07294     LYS      CA     -6.378   50.464                                 -6.99994     LYS      O      -8.458   50.480                                 -5.78394     LYS      CG     -5.394   52.320                                 -5.46794     LYS      CE     -4.399   54.208                                 -4.19995     VAL      N      -6.909   49.071                                 -5.02695     VAL      C      -6.919   48.499                                 -2.56895     VAL      CB     -8.104   47.030                                 -4.31995     VAL      CG2    -6.900   46.100                                 -4.33296     LEU      CA     -4.782   49.103                                 -1.48696     LEU      O      -3.942   51.121                                 -2.33696     LEU      CG     -3.593   46.799                                 -2.07296     LEU      CD2    -4.489   46.082                                 -1.04597     GLY      CA     -3.890   52.307                                 0.28797     GLY      O      -1.619   51.463                                 0.16598     ALA      CB     -0.428   55.478                                 1.51098     ALA      C      0.188    53.118                                 1.91799     ASP      N      -0.504   52.573                                 2.91299     ASP      DD1    -2.730   50.902                                 4.00399     ASP      CB     -0.648   51.603                                 5.17599     ASP      C      0.146    50.165                                 3.320100    GLY      N      -0.424   49.883                                 2.168100    GLY      C      -1.520   47.651                                 2.002101    SER      N      -2.342   48.128                                 2.908101    SER      C      -4.759   47.894                                 2.532101    SER      CB     -3.716   47.447                                 4.817102    GLY      N      -5.821   47.092                                 2.577102    GLY      C      -8.166   46.536                                 2.528103    GLN      N      -9.377   47.058                                 2.498103    GLN      C      -10.963  45.232                                 2.022103    GLN      CB     -11.671  47.307                                 3.274103    GLN      CD     -12.360  49.104                                 4.915103    GLN      NE2    -13.419  49.197                                 4.112104    TYR      CA     -12.068  43.126                                 1.508104    TYR      D      -12.939  43.276                                 -0.687104    TYR      CG     -11.629  40.829                                 2.472104    TYR      CD2    -10.379  40.959                                 1.860104    TYR      CE2    -9.352   40.057                                 2.171104    TYR      OH     -8.481   38.191                                 3.324105    SER      CA     -14.877  45.166                                 -0.034105    SER      D      -14.759  45.935                                 -2.258105    SER      OG     -15.209  47.039                                 1.450106    TRP      CA     -12.421  47.391                                 -1.948106    TRP      O      -12.021  46.648                                 -4.245106    TRP      CG     -11.645  49.111                                 -0.206106    TRP      CD2    -10.658  49.812                                 0.581106    TRP      CE2    -11.359  50.573                                 1.561106    TRP      CZ2    -10.671  51.318                                 2.500106    TRP      CH2    -9.293   51.291                                 2.455107    ILE      CA     -10.765  44.250                                 -3.325107    ILE      D      -11.695  43.474                                 -5.398107    ILE      CG1    -8.634   43.784                                 -1.936107    ILE      CD1    -8.233   42.998                                 -0.62791     TYR      CB     -6.686   48.093                                 -16.31491     TYR      CD1    -6.595   47.415                                 -18.75591     TYR      CE1    -6.905   47.572                                 -20.09091     TYR      CZ     -7.794   48.582                                 -20.46392     ALA      N      -4.895   49.958                                 -14.01492     ALA      C      -5.823   50.033                                 -11.90392     ALA      CB     -3.997   51.621                                 -12.48893     VAL      CA     -7.183   48.854                                 -10.32593     VAL      D      -6.181   47.993                                 -8.37293     VAL      CG1    -9.213   47.488                                 -9.72594     LYS      N      -6.907   50.217                                 -8.32794     LYS      C      -7.331   49.985                                 -5.89494     LYS      CB     -6.051   51.976                                 -6.81894     LYS      CD     -4.868   53.785                                 -5.58294     LYS      NZ     -3.735   55.544                                 -4.38795     VAL      CA     -7.646   48.457                                 -3.92095     VAL      O      -7.425   48.156                                 -1.50195     VAL      CG1    -8.868   46.852                                 -5.61996     LEU      N      -5.676   48.974                                 -2.60496     LEU      C      -4.331   50.559                                 -1.32196     LEU      CB     -3.509   48.241                                 -1.57396     LEU      CD1    -2.207   46.184                                 -2.16397     GLY      N      -4.326   50.975                                 -0.08697     GLY      C      -2.363   52.437                                 0.38598     ALA      N      -1.954   53.648                                 0.75898     ALA      CA     -0.563   54.068                                 0.96598     ALA      O      1.393    52.921                                 1.66399     ASP      DD2    -2.631   51.042                                 6.15199     ASP      CG     -2.083   51.131                                 5.04099     ASP      CA     0.101    51.610                                 3.85599     ASP      O      0.735    49.313                                 4.029100    GLY      CA     -0.343   48.521                                 1.615100    GLY      O      -1.649   46.512                                 1.479101    SER      CA     -3.542   47.388                                 3.315101    SER      O      -4.758   48.972                                 1.907101    SER      OG     -4.411   48.634                                 5.209102    GLY      CA     -7.077   47.422                                 1.896102    GLY      O      -7.888   45.431                                 3.030103    GLN      CA     -10.535  46.297                                 3.020103    GLN      O      -10.779  45.482                                 0.817103    GLN      CG     -11.368  48.005                                 4.586103    GLN      OE1    -12.159  49.816                                 5.902104    TYR      N      -11.611  44.141                                 2.451104    TYR      C      -13.031  43.690                                 0.473104    TYR      CB     -12.697  41.866                                 2.143104    TYR      CD1    -11.819  39.789                                 3.377104    TYR      CE1    -10.805  38.885                                 3.707104    TYR      CZ     -9.564   39.022                                 3.081105    SER      N      -13.909  44.572                                 0.903105    SER      C      -14.172  45.920                                 -1.159105    SER      CB     -15.880  46.121                                 0.601106    TRP      N      -13.079  46.625                                 -0.834106    TRP      C      -11.895  46.436                                 -3.012106    TRP      CB     -11.321  48.254                                 -1.355106    TRP      CD1    -12.862  49.524                                 0.264106    TRP      NE1    -12.691  50.358                                 1.360106    TRP      CE3    -9.275   49.852                                 0.576106    TRP      CZ3    -8.568   50.563                                 1.525107    ILE      N      -11.339  45.330                                 -2.481107    ILE      C      -11.955  43.594                                 -4.190107    ILE      CB     -9.944   43.183                                 -2.523107    ILE      CG2    -9.632   41.930                                 -3.381108    ILE      N      -12.994  43.292                                 -3.577108    ILE      CA     -14.116  42.722                                 -4.32108    ILE      O      -14.894  43.329                                 -6.552108    ILE      CG1    -14.726  41.077                                 -2.482108    ILE      CD1    -15.452  40.845                                 -1.131109    ASN      CA     -15.204  46.018                                 -5.916109    ASN      O      -14.660  46.272                                 -8.235109    ASN      CG     -16.528  47.486                                 -4.353109    ASN      ND2    -16.633  48.447                                 -3.442110    GLY      CA     -11.952  45.917                                 -7.865110    GLY      O      -11.929  44.929                                 -10.034111    ILE      CA     -12.603  42.334                                 -9.099111    ILE      O      -13.921  42.384                                 -11.148111    ILE      CG1    -11.421  40.501                                 -7.655111    ILE      CD1    -11.588  39.706                                 -6.336112    GLU      CA     -16.118  43.376                                 -10.046112    GLU      O      -16.467  44.130                                 -12.246112    GLU      CG     -17.847  42.917                                 -8.135112    GLU      DE1    -19.041  40.866                                 -8.016113    TRP      N      -15.094  45.403                                 -10.971113    TRP      C      -14.076  45.663                                 -13.140113    TRP      CB     -13.882  47.553                                 -11.434113    TRP      CD1    -14.148  49.736                                 -12.681113    TRP      NE1    -13.597  50.443                                 -13.723113    TRP      CE3    -11.451  47.645                                 -13.809113    TRP      CZ3    -10.610  47.899                                 -14.879114    ALA      N      -13.089  44.801                                 -12.832114    ALA      C      -13.199  43.179                                 -14.752114    ALA      CB     -11.299  43.192                                 -13.140115    ILE      CA     -15.070  41.640                                 -14.897115    ILE      O      -16.077  42.225                                 -17.070115    ILE      CG1    -15.218  39.836                                 -13.043115    ILE      CD1    -16.004  39.411                                 -11.743116    ALA      CA     -17.390  44.440                                 -16.050116    ALA      O      -17.323  45.255                                 -18.343117    ASN      N      -15.423  45.390                                 -17.122117    ASN      C      -13.827  44.974                                 -19.034117    ASN      CB     -13.615  46.958                                 -17.426117    ASN      DD1    -14.565  49.082                                 -17.773118    ASN      N      -14.223  43.725                                 -18.967118    ASN      C      -12.240  42.444                                 -19.843118    ASN      CB     -14.247  42.863                                 -21.279118    ASN      DD1    -16.510  42.321                                 -20.759119    MET      N      -11.686  42.500                                 -18.675119    MET      C      -18.025  40.734                                 -18.928119    MET      CB     -9.810   42.461                                 -17.055119    MET      SD     -8.788   44.943                                 -17.526120    ASP      N      -8.904   40.437                                 -23.084120    ASP      C      -7.822   38.390                                 -18.856120    ASP      CB     -7.555   39.156                                 -21.236120    ASP      DD1    -7.801   40.706                                 -19.584121    VAL      N      -7.021   39.117                                 -18.115121    VAL      C      -6.296   39.534                                 -15.786121    VAL      CB     -4.755   38.587                                 -17.496121    VAL      CG2    -4.707   37.916                                 -18.846122    ILE      CA     -6.248   39.799                                 -13.397122    ILE      O      -4.829   38.012                                 -12.469122    ILE      CG1    -8.686   40.392                                 -13.063122    ILE      CD1    -9.976   -39.788                                 -12.383123    ASN      CA     -3.145   39.854                                 -11.232123    ASN      O      -3.708   41.631                                 -9.833123    ASN      CG     -0.692   40.048                                 -10.777123    ASN      ND2    -0.346   40.747                                 -9.720124    MET      CA     -3.650   39.973                                 -7.438108    ILE      C      -14.639  43.694                                 -5.386108    ILE      CB     -15.246  42.265                                 -3.320108    ILE      CG2    -16.568  42.024                                 -4.095109    ASN      N      -14.751  44.958                                 -4.981109    ASN      C      -14.232  46.067                                 -7.084109    ASN      CB     -15.280  47.359                                 -5.207109    ASN      DD1    -17.455  46.695                                 -4.646110    GLY      N      -12.951  45.908                                 -6.774110    GLY      C      -12.108  44.712                                 -8.812111    ILE      N      -12.379  43.539                                 -8.246111    ILE      C      -13.859  42.560                                 -9.942111    ILE      CB     -12.734  40.948                                 -8.364111    ILE      CG2    -13.122  39.791                                 -9.347112    GLU      N      -14.893  43.075                                 -9.280112    GLU      C      -15.872  44.347                                 -11.171112    GLU      C5     -17.229  43.899                                 -9.141112    GLU      CD     -18.724  41.824                                 -8.685112    GLU      DE2    -19.123  41.928                                 -9.866113    TRP      CA     -14.756  46.400                                 -12.000113    TRP      O      -14.319  45.932                                 -14.332113    TRP      CG     -13.486  48.556                                 -12.481113    TRP      CD2    -12.441  48.552                                 -13.463113    TRP      CE2    -12.545  49.761                                 -14.215113    TRP      CZ2    -11.696  50.045                                 -15.274113    TRP      CH2    -10.752  49.074                                 -15.603114    ALA      CA     -12.333  44.065                                 -13.874114    ALA      D      -12.963  43.074                                 -15.978115    ILE      N      -14.174  42.540                                 -14.119115    ILE      C      -15.928  42.485                                 -15.856115    ILE      CB     -16.000  40.840                                 -13.922115    ILE      CG2    -17.151  40.168                                 -14.755116    ALA      N      -16.534  43.527                                 -15.267116    ALA      C      -16.706  45.069                                 -17.278116    ALA      CB     -18.011  45.510                                 -15.151117    ASN      CA     -14.553  45.967                                 -18.139117    ASN      D      -12.997  45.436                                 -19.820117    ASN      CG     -14.400  48.177                                 -16.939117    ASN      ND2    -14.931  48.249                                 -15.736118    ASN      CA     -13.760  42.642                                 -19.832118    ASN      O      -11.617  42.309                                 -20.932118    ASN      CG     -15.737  43.060                                 -21.395118    ASN      ND2    -16.136  44.096                                 -22.133119    MET      CA     -10.232  42.222                                 -18.478119    MET      O      -10.888  39.838                                 -18.759119    MET      CG     -9.880   43.883                                 -16.502119    MET      C5     -9.982   46.061                                 -18.263120    ASP      CA     -8.480   39.118                                 -20.030120    ASP      O      -8.038   37.189                                 -18.690120    ASP      CG     -8.237   39.730                                 -22.454120    ASP      OD2    -9.327   39.135                                 -22.739121    VAL      CA     -6.226   38.601                                 -16.974121    VAL      D      -6.284   40.788                                 -15.909121    VAL      CG1    -3.758   38.176                                 -16.427122    ILE      N      -6.318   38.978                                 -14.590122    ILE      C      -5.020   39.262                                 -12.627122    ILE      CB     -7.476   39.604                                 -12.466122    ILE      CG2    -7.221   39.883                                 -10.954123    ASN      N      -4.263   40.222                                 -12.110123    ASN      C      -3.502   40.404                                 -9.861123    ASN      CB     -1.828   40.478                                 -11.697123    ASN      OD1    -0.063   38.990                                 -11.018124    MET      N      -3.458   39.604                                 -8.832124    MET      C      -2.423   39.603                                 -6.614124    MET      O      -2.306   38.508                                 -6.098124    MET      CG     -6.158   40.082                                 -7.478124    MET      CB     -7.940   38.095                                 -7.542125    SER      CA     -0.193   40.287                                 -5.769125    SER      O      0.235    41.617                                 -3.805125    SER      OG     1.444    40.496                                 -7.575126    LEU      CA     -1.842   40.347                                 -2.386126    LEU      O      -2.844   38.136                                 -2.529126    LEU      CG     -3.988   41.447                                 -3.333126    LEU      CD2    -4.179   42.760                                 -4.073127    GLY      CA     -3.035   37.871                                 0.193127    GLY      O      -2.446   39.030                                 2.220128    GLY      CA     -4.475   37.496                                 3.642128    GLY      O      -4.903   35.158                                 3.276129    PRO      CA     -4.671   34.525                                 5.998129    PRO      O      -6.338   32.887                                 6.305129    PRO      CG     -4.419   36.116                                 7.727130    SER      N      -7.051   35.015                                 5.912130    SER      C      -9.218   34.884                                 4.726130    SER      CB     -9.069   35.351                                 7.216131    GLY      N      -10.083  33.967                                 4.349131    GLY      C      -12.205  34.713                                 3.542132    SER      N      -13.040  35.058                                 2.594132    SER      C      -15.289  34.805                                 1.936132    SER      CB     -14.590  36.927                                 3.145133    ALA      N      -16.547  34.588                                 2.294133    ALA      C      -17.650  34.965                                 0.097133    ALA      CB     -18.866  33.828                                 1.996134    ALA      CA     -17.872  37.259                                 -0.792134    ALA      O      -16.781  37.585                                 -2.869135    LEU      N      -15.478  27.229                                 -1.046135    LEU      C      -14.158  36.005                                 -2.705135    LEU      CB     -13.038  37.328                                 -0.798135    LEU      CD1    -11.460  38.415                                 -2.292136    LYS      N      -14.509  34.825                                 -2.173136    LYS      C      -15.544  33.739                                 -4.150136    LYS      CB     -14.903  32.341                                 -2.186136    LSY      CD     -15.083  29.892                                 -2.134136    LYS      NZ     -15.308  28.411                                 -4.160137    ALA      CA     -17.795  34.416                                 -4.883137    ALA      O      -17.705  35.049                                 -7.208138    ALA      N      -16.529  36.301                                 -5.729138    ALA      C      -14.903  36.696                                 -7.557138    ALA      CB     -15.522  38.567                                 -5.934139    VAL      CA     -12.946  35.291                                 -7.837139    VAL      O      -13.208  34.070                                 -9.877139    VAL      CG1    -10.919  33.856                                 -7.866140    ASP      N      -14.593  33.536                                 -8.122140    ASP      C      -16.023  33.131                                 -10.084140    ASP      CB     -16.149  31.549                                 -8.138140    ASP      DD1    -14.178  30.403                                 -7.202141    LYS      N      -16.658  34.263                                 -9.820141    LYS      C      -16.373  35.415                                 -11.946141    LYS      CB     -18.039  36.275                                 -10.325141    LYS      CD     -19.586  38.187                                 -10.536141    LYS      NZ     -21.138  40.037                                 -10.275142    ALA      CA     -14.173  36.192                                 -12.614142    ALA      O      -13.770  35.169                                 -14.755143    VAL      N      -13.582  33.886                                 -12.832143    VAL      C      -14.346  32.233                                 -14.496143    VAL      CB     -12.551  31.673                                 -12.714143    VAL      CG2    -11.305  32.195                                 -12.014144    ALA      CA     -16.744  31.834                                 -14.641124    MET      CB     -4.943   39.387                                 -6.890124    MET      SD     -7.585   39.472                                 -6.450125    SER      N      -1.454   40.496                                 -6.502125    SER      C      -0.422   40.712                                 -4.326125    SER      CB     1.021    41.027                                 -6.328126    LEU      N      -1.433   40.075                                 -3.775126    LEU      C      -2.438   39.056                                 -1.807126    LEU      CB     -2.791   41.568                                 -2.410126    LEU      CD1    -5.278   41.131                                 -2.578127    GLY      N      -2.522   39.082                                 -0.481127    GLY      C      -3.176   38.180                                 1.682128    GLY      N      -4.121   37.443                                 2.222128    GLY      C      -4.644   36.038                                 4.104129    PRO      N      -4.519   35.857                                 5.402129    PRO      C      -6.116   34.086                                 6.082129    PRO      CB     -4.060   34.684                                 7.384129    PRO      CD     -4.239   36.870                                 6.418130    SER      CA     -8.470   34.611                                 6.023130    SER      O      -8.949   35.881                                 4.029130    SER      OG     -8.723   34.626                                 8.403131    GLY      CA     -10.824  34.229                                 3.074131    GLY      O      -12.495  34.722                                 4.751132    SER      CA     -14.407  35.433                                 3.011132    SER      O      -14.799  34.586                                 0.824132    SER      OG     -14.693  37.539                                 1.875133    ALA      CA     -17.507  34.057                                 1.324133    ALA      O      -17.743  34.437                                 -1.014134    ALA      N      -17.683  36.288                                 0.294134    ALA      C      -16.635  37.369                                 -1.674134    ALA      CB     -18.263  38.600                                 -0.187135    LEU      CA     -14.197  37.244                                 -1.804135    LEU      O      -13.794  36.020                                 -3.890135    LEU      CG     -11.693  37.130                                 -1.508135    LEU      CD2    -10.582  36.807                                 -0.519136    LYS      CA     -14.543  33.597                                 -3.013136    LYS      O      -15.279  33.431                                 -5.305136    LYS      CG     -14.743  31.067                                 -3.043136    LYS      CE     -15.473  28.707                                 -2.778137    ALA      N      -16.744  34.260                                 -3.847137    ALA      C      -17.338  35.303                                 -6.045137    ALA      CB     -19.094  34.941                                 -4.263138    ALA      CA     -16.001  37.311                                 -6.685138    ALA      O      -14.985  36.843                                 -8.762139    VAL      N      -13.950  35.959                                 -7.027139    VAL      C      -13.623  34.228                                 -8.720139    VAL      CB     -11.830  34.671                                 -6.968139    VAL      CG2    -11.078  35.780                                 -6.253140    ASP      CA     -15.274  32.496                                 -8.929140    ASP      O      -16.080  32.579                                 -11.190140    ASP      CG     -15.388  30.640                                 -7.186140    ASP      DD2    -16.139  30.132                                 -6.329141    LYS      CA     -17.373  35.006                                 -10.868141    LYS      O      -16.700  35.248                                 -13.111141    LYS      CG     -18.884  37.056                                 -11.306141    LYS      CE     -20.572  39.051                                 -11.250142    ALA      N      -15.167  35.848                                 -11.566142    ALA      C      -13.818  35.010                                 -13.521142    ALA      CB     -12.870  36.697                                 -11.948143    VAL      CA     -13.168  32.705                                 -13.650143    VAL      O      -14.140  31.886                                 -15.639143    VAL      CG1    -12.300  30.370                                 -13.461144    ALA      N      -15.531  32.238                                 -13.875144    ALA      C      -16.928  32.681                                 -15.861144    ALA      O      -17.380  32.263                                 -16.959145    SER      N      -16.507  33.948                                 -15.706145    SER      C      -15.609  34.773                                 -17.829145    SER      CB     -17.016  36.376                                 -16.414146    GLY      N      -14.577  33.986                                 -17.565146    GLY      C      -12.273  34.491                                 -18.385147    VAL      N      -12.150  35.162                                 -17.254147    VAL      C      -9.850   34.836                                 -16.323147    VAL      CB     -11.152  36.977                                 -15.889147    VAL      CG2    -12.340  37.915                                 -16.230148    VAL      CA     -7.482   34.230                                 -16.008148    VAL      D      -6.840   36.133                                 -14.750148    VAL      CG1    -5.079   33.403                                 -16.281149    VAL      N      p31 7.258                           34.355                                 -13.531149    VAL      C      -5.700   34.385                                 -11.613149    VAL      CB     -8.224   34.890                                 -11.315149    VAL      CG2    -9.456   35.386                                 -12.096150    VAL      CA     -3.393   34.987                                 -10.901150    VAL      O      -3.592   36.778                                 -9.400150    VAL      CG1    -0.973   34.633                                 -11.461151    ALA      N      -2.568   34.946                                 -8.595151    ALA      C      -1.080   35.036                                 -6.657151    ALA      CB     -3.557   35.390                                 -6.307152    ALA      CA     0.714    35.438                                 -5.112152    ALA      O      -0.728   34.466                                 -3.467153    ALA      N      1.125    33.302                                 -3.912153    ALA      C      0.931    32.725                                 -1.511153    ALA      CB     1.750    31.038                                 -3.195154    GLY      CA     2.043    34.211                                 0.125154    GLY      O      4.189    33.267                                 -0.118155    ASN      CA     5.344    34.787                                 2.037155    ASN      O      6.101    34.829                                 4.295155    ASN      CG     5.890    36.702                                 0.500155    ASN      ND2    5.454    37.965                                 0.352156    GLU      CA     4.633    32.537                                 4.970156    GLU      O      5.374    30.637                                 6.222156    GLU      CG     2.491    32.442                                 6.368156    GLU      DE1    1.744    34.322                                 5.312157    GLY      N      6.389    31.057                                 4.227157    GLY      C      6.503    28.622                                 4.553158    THR      N      7.147    27.793                                 5.382158    THR      OG1    8.707    25.487                                 6.217158    THR      CA     6.552    26.487                                 5.702158    THR      O      6.479    27.335                                 7.977159    SER      OG     3.141    25.904                                 10.525159    SER      CA     4.835    25.210                                 8.855159    SER      O      3.339    23.281                                 9.030160    GLY      CA     5.434    21.504                                 8.895160    GLY      O      4.806    21.326                                 6.555161    SER      CA     2.654    19.777                                 7.054161    SER      O      0.696    20.347                                 5.869161    SER      OG     1.854    18.028                                 8.585162    SER      CA     0.167    22.725                                 7.113162    SER      O      1.533    23.840                                 5.394162    SER      OG     0.184    23.091                                 9.480163    SER      CA     -0.611   24.750                                 3.990163    SER      O      -1.078   26.548                                 5.504163    SER      OG     -1.992   25.718                                 2.331164    THR      CA     0.609    28.340                                 4.312164    THR      O      0.485    30.502                                 3.278164    THR      OG1    2.984    28.282                                 3.692165    VAL      N      -0.513   28.742                                 2.190165    VAL      C      -2.028   30.545                                 1.497144    ALA      CB     -17.942  31.968                                 -13.700145    SER      CA     -16.682  34.917                                 -16.786145    SER      O      -15.910  35.321                                 -18.893145    SER      OG     -15.882  36.955                                 -15.849146    GLY      CA     -13.619  33.799                                 -18.675146    GLY      O      -11.420  34.386                                 -19.266147    VAL      CA     -10.874  35.856                                 -16.912147    VAL      O      -10.171  33.991                                 -15.486147    VAL      CG1    -9.896   37.803                                 -15.570148    VAL      N      -8.583   35.018                                 -16.603148    VAL      C      -7.157   34.907                                 -14.701148    VAL      CB     -6.273   34.126                                 -16.950148    VAL      CG2    -6.590   33.432                                 -18.262149    VAL      CA     -6.987   34.965                                 -12.249149    VAL      O      -5.624   33.173                                 -11.439149    VAL      CG1    -7.893   35.619                                 -10.009150    VAL      N      -4.732   35.301                                 -11.404150    VAL      C      -3.157   35.625                                 -9.559150    VAL      CB     -2.274   35.305                                 -11.951150    VAL      CG2    -2.675   34.843                                 -13.301151    ALA      CA     -2.361   35.582                                 -7.287151    ALA      O      -0.618   33.889                                 -6.904152    ALA      N      -0.490   35.907                                 -5.822152    ALA      C      0.304    34.320                                 -4.158152    ALA      CB     1.266    36.607                                 -4.294153    ALA      CA     0.840    32.250                                 -2.943153    ALA      O      0.317    32.192                                 -0.599154    GLY      N      1.827    33.693                                 -1.244154    GLY      C      3.519    34.069                                 0.550155    ASN      N      3.958    34.788                                 1.568155    ASN      C      5.399    34.258                                 3.462155    ASN      OD1    6.123    36.065                                 -0.534156    GLU      N      4.711    33.168                                 3.675156    GLU      C      5.522    31.328                                 5.183156    GLU      CB     3.205    31.980                                 5.100156    GLU      CD     2.394    33.951                                 6.270156    GLU      OE2    3.106    34.456                                 7.146157    GLY      CA     7.306    29.917                                 4.387157    GLY      O      5.416    28.346                                 4.009158    THR      CG2    8.079    25.396                                 3.850158    THR      CB     7.564    25.346                                 5.296158    THR      C      6.100    26.480                                 7.157159    SER      N      5.338    25.441                                 7.497159    SER      CB     3.673    26.105                                 9.212159    SER      C      4.494    23.720                                 8.944160    GLY      N      5.574    22.967                                 8.835160    GLY      C      4.576    21.045                                 7.738161    SER      N      3.525    20.310                                 8.116161    SER      C      1.477    20.708                                 6.786161    SER      CB     2.344    18.293                                 7.271162    SER      N      1.303    21.841                                 7.459162    SER      C      0.430    23.552                                 5.848162    SER      CB     -0.213   23.666                                 8.242163    SER      N      -0.679   23.921                                 5.197163    SER      C      -0.441   26.177                                 4.513163    SER      CB     -1.890   24.642                                 3.211164    THR      N      0.387    26.952                                 3.852164    THR      C      0.185    29.286                                 3.194164    THR      CB     2.095    28.518                                 4.818164    THR      CG2    2.397    27.610                                 6.001165    VAL      CA     -0.959   29.542                                 1.010165    VAL      O      -2.929   30.132                                 2.280165    VAL      CB     -1.339   28.624                                 -0.161165    VAL      CG2    -0.210   27.716                                 -0.699166    GLY      CA     -2.945   32.778                                 1.626166    GLY      O      -4.124   32.206                                 -0.396167    TYR      CA     -6.223   34.046                                 0.113167    TYR      O      -5.474   36.283                                 0.084167    TYR      CG     -7.791   32.984                                 1.709167    TYR      CD2    -8.710   32.116                                 1.133167    TYR      CE2    -9.068   30.955                                 1.809167    TYR      OM     -8.880   29.481                                 3.658168    PRO      CG     -6.943   36.376                                 -3.938168    PRO      CB     -7.964   35.344                                 -3.505168    PRO      C      -6.398   33.336                                 -3.270169    GLY      N      -5.086   33.193                                 -3.189169    GLY      C      -4.937   30.702                                 -3.470170    LYS      N      -5.402   30.579                                 -2.255170    LYS      C      -7.055   28.773                                 -2.516170    LYS      CB     -6.246   29.294                                 -0.286170    LYS      CD     -6.250   28.289                                 2.031170    LYS      NZ     -4.259   27.463                                 3.215171    TYR      CA     -9.012   29.043                                 -3.859171    TYR      O      -7.760   28.714                                 -5.928171    TYR      CG     -10.497  30.984                                 -3.047171    TYR      CD2    -10.456  32.374                                 -3.026171    TYR      CE2    -10.941  33.088                                 -1.936171    TYR      OM     -12.008  33.119                                 0.170172    PRO      CA     -9.093   26.417                                 -6.596172    PRO      O      -8.525   26.784                                 -8.881172    PRO      CG     -10.600  25.271                                 -5.096173    SER      N      -10.097  28.167                                 -8.019173    SER      C      -9.025   29.773                                 -9.595173    SER      CB     -11.528  29.623                                 -9.481174    VAL      N      -8.162   29.944                                 -8.614174    VAL      C      -5.754   30.131                                 -9.068174    VAL      CB     -6.899   31.775                                 -7.596174    VAL      CG2    -8.220   32.503                                 -7.323175    ILE      CA     -3.569   30.156                                 -10.024175    ILE      O      -2.450   31.958                                 -8.955175    ILE      CG1    -3.857   29.978                                 -12.524175    ILE      CD1    -3.692   30.529                                 -13.946176    ALA      CA     -1.335   30.517                                 -6.870176    ALA      O      0.453    29.215                                 -7.838177    VAL      N      0.864    31.410                                 -7.180177    VAL      C      3.225    31.693                                 -6.473177    VAL      CB     2.439    32.607                                 -8.755177    VAL      CG2    1.374    32.552                                 -9.845178    GLY      CA     5.168    30.703                                 -5.339178    GLY      O      6.499    31.435                                 -7.286179    ALA      CA     8.715    32.037                                 -5.859179    ALA      C      10.198   30.481                                 -4.719180    VAL      N      10.659   31.162                                 -6.885180    VAL      C      13.048   31.585                                 -7.171180    VAL      CB     12.075   29.514                                 -8.166180    VAL      CG2    11.675   30.129                                 -9.500181    ASP      CA     15.451   32.108                                 -7.039181    ASP      O      15.339   31.090                                 -9.292181    ASP      CG     17.120   30.534                                 -5.971181    ASP      DD2    17.680   30.256                                 -4.887182    SER      CA     17.622   32.214                                 -10.191182    SER      O      18.365   70.452                                 -11.670182    SER      OG     18.016   34.561                                 -10.475183    SER      CA     18.716   28.645                                 -9.444183    SER      O      17.859   26.415                                 -9.397165    VAL      CG1    -1.947   29.357                                 -1.374166    GLY      N      -1.910   31.821                                 1.129166    GLY      C      -4.098   32.859                                 0.617167    TYR      N      -5.054   33.730                                 0.970167    TYR      C      -5.993   35.389                                 -0.606167    TYR      CB     -7.464   34.252                                 0.964167    TYR      CD1    -7.208   32.703                                 2.947167    TYR      CE1    -7.567   31.528                                 3.615167    TYR      CZ     -8.486   30.671                                 3.046168    PRO      N      -6.380   35.499                                 -1.850168    PRO      CD     -6.273   36.752                                 -2.624168    PRO      CA     -7.134   34.457                                 -2.560168    PRO      O      -7.097   32.520                                 -3.912169    GLY      CA     -4.446   32.077                                 -3.927169    GLY      O      -4.880   29.733                                 -4.249170    LYS      CA     -5.856   29.265                                 -1.745170    LYS      O      -7.308   27.554                                 -2.524170    LYS      CG     -5.795   28.106                                 0.585170    LYS      CE     -5.731   27.271                                 3.029171    TYR      N      -7.838   29.616                                 -3.148171    TYR      C      -8.603   28.309                                 -5.113171    TYR      CB     -9.962   30.224                                 -4.242171    TYR      CD1    -11.060  30.303                                 -1.982171    TYR      CE1    -11.520  31.003                                 -0.867171    TYR      CZ     -11.528  32.398                                 -0.886172    PRO      N      -9.297   27.204                                 -5.374172    PRO      C      -9.233   27.156                                 -7.909172    PRO      CB     -10.167  25.329                                 -6.513172    PRO      CD     -10.364  26.669                                 -4.514173    SER      CA     -10.220  28.818                                 -9.330173    SER      O      -8.966   30.233                                 -10.742173    SER      OG     -11.595  30.546                                 -8.406174    VAL      CA     -7.053   30.891                                 -8.855174    VAL      D      -5.612   29.152                                 -8.344174    VAL      CG1    -5.796   32.837                                 -7.617175    ILE      N      -4.911   30.729                                 -9.885175    ILE      C      -2.714   30.736                                 -8.894175    ILE      CB     -2.953   30.524                                 -11.419175    ILE      CG2    -1.451   30.089                                 -11.512176    ALA      N      -2.220   30.028                                 -7.925176    ALA      C      0.120    30.301                                 -7.310176    ALA      CB     -1.639   29.838                                 -5.541177    ALA      CA     2.261    31.534                                 -7.656177    VAL      O      3.178    32.657                                 -5.721177    VAL      CG1    3.842    32.667                                 -9.392178    GLY      N      4.077    30.654                                 -6.358178    GLY      C      6.446    31.233                                 -6.074179    ALA      N      7.512    31.447                                 -5.287179    ALA      C      9.939    31.099                                 -5.779179    ALA      CB     9.025    33.251                                 -4.973180    VAL      CA     11.970   30.482                                 -6.981180    VAL      O      12.712   32.691                                 -7.627180    VAL      CG1    11.271   28.251                                 -7.855181    ASP      N      14.267   31.203                                 -6.800181    ASP      C      15.942   31.804                                 -8.462181    ASP      CB     16.446   31.921                                 -5.914181    ASP      DD1    17.105   29.785                                 -6.972182    SER      N      17.087   32.386                                 -8.847182    SER      C      18.153   30.817                                 -10.494182    SER      CB     18.678   33.313                                 -10.464183    SER      N      18.258   30.042                                 -9.423183    SER      C      17.581   27.614                                 -9.547183    SER      CB     19.256   28.323                                 -8.007183    SER      OG     20.589   28.615                                 -8.251184    ASN      CA     15.144   27.317                                 -9.580184    ASN      O      14.138   25.759                                 -8.097184    ASN      CG     14.990   26.998                                 -12.076184    ASN      ND2    15.352   26.210                                 -13.076185    GLN      CA     15.276   26.646                                 -5.835185    GLN      O      14.159   28.726                                 -5.386185    GLN      CG     16.539   26.242                                 -3.614185    GLN      OE1    18.864   25.799                                 -4.061186    ARG      N      13.278   26.958                                 -4.448186    ARG      C      12.780   28.782                                 -2.866186    ARG      CB     11.215   26.843                                 -3.116186    ARG      CD     9.467    26.337                                 -1.468186    ARG      CZ     9.961    26.879                                 1.059186    ARG      NH2    10.966   26.321                                 1.783187    ALA      CA     12.728   31.064                                 -1.895187    ALA      D      11.156   30.043                                 -0.387188    SER      N      13.051   30.770                                 0.549188    SER      C      11.356   30.847                                 2.412188    SER      CB     13.767   30.456                                 2.938189    PHE      N      10.943   32.010                                 1.974189    PHE      C      8.499    32.198                                 1.609189    PHE      CB     9.787    34.217                                 2.243189    PHE      CD1    9.147    34.830                                 -0.121189    PHE      CE1    9.483    35.187                                 -1.411189    PHE      CZ     10.786   35.586                                 -1.725190    SER      CA     7.626    31.096                                 -0.391190    SER      O      7.034    29.083                                 0.866190    SER      OG     7.136    30.337                                 -2.618191    SER      CA     4.341    29.686                                 0.987191    SER      O      4.543    28.268                                 -0.995191    SER      OG     2.729    31.285                                 1.954192    VAL      CA     3.629    25.932                                 0.391192    VAL      O      1.559    25.698                                 1.598192    VAL      CG1    6.144    25.727                                 0.722193    GLY      N      1.938    24.172                                 0.047193    GLY      C      0.081    23.029                                 -0.901194    PRO      N      -1.023   22.289                                 -0.722194    PRO      C      -2.237   22.605                                 -2.914194    PRO      CB     -2.769   20.783                                 -1.210194    PRO      CD     -1.633   21.954                                 0.578195    GLU      CA     -3.145   24.850                                 -3.252195    GLU      O      -2.516   26.398                                 -4.936195    GLU      CG     -4.942   25.134                                 -1.435195    GLU      OE1    -3.110   24.960                                 0.165196    LEU      N      -0.829   25.264                                 -3.870196    LEU      C      0.228    25.376                                 -6.059196    LEU      CB     1.540    25.739                                 -3.854196    LEU      CD1    2.739    27.716                                 -4.639197    ASP      N      0.140    26.208                                 -7.093197    ASP      C      1.307    25.738                                 -9.293197    ASP      CB     -1.067   26.598                                 -9.191197    ASP      DD1    -2.804   25.155                                 -8.354198    VAL      N      2.013    26.889                                 -9.344198    VAL      C      4.157    27.950                                 -9.514198    VAL      CB     2.894    27.476                                 -11.637198    VAL      CG2    2.337    28.919                                 -11.484199    MET      CA     6.439    28.802                                 -9.498199    MET      O      6.696    29.518                                 -11.793199    MET      CG     7.365    26.849                                 -8.139199    MET      CE     8.227    27.755                                 -5.587200    ALA      CA     7.991    31.929                                 -11.055200    ALA      O      9.127    32.524                                 -9.060184    ASN      N      16.373   28.094                                 -9.602184    ASN      C      14.931   26.720                                 -8.197184    ASN      CB     15.014   26.341                                 -10.722184    ASN      DD1    14.700   28.184                                 -12.277185    GLN      N      15.542   27.247                                 -7.159185    GLN      C      14.200   27.494                                 -5.203185    GLN      CB     16.599   26.568                                 -5.101185    GLN      CD     18.011   26.102                                 -3.206185    GLN      NE2    18.266   26.386                                 -1.934186    ARG      CA     12.185   27.774                                 -3.841186    ARG      O      13.698   28.384                                 -2.093186    ARG      CG     10.214   27.471                                 -2.161186    ARG      NE     9.866    26.333                                 -0.117186    ARG      NH1    9.367    27.880                                 1.658187    ALA      N      12.294   30.009                                 -2.853187    ALA      C      12.262   30.604                                 -0.517187    ALA      CB     12.144   32.402                                 -2.344188    SER      CA     12.671   30.286                                 1.868188    SER      O      10.740   30.111                                 3.212188    SER      OG     14.137   31.826                                 2.841189    PHE      CA     9.697    32.688                                 2.418189    PHE      O      7.389    32.556                                 2.011189    PHE      CG     10.117   34.696                                 0.867189    PHE      CD2    11.415   35.116                                 0.567189    PHE      CE2    11.769   35.545                                 -0.701190    SER      N      8.703    31.526                                 0.499190    SER      C      6.663    30.162                                 0.328190    SER      CB     8.181    30.590                                 -1.708191    SER      N      5.388    30.551                                 0.326191    SER      C      4.261    28.330                                 0.223191    SER      CB     3.015    30.411                                 0.911192    VAL      N      3.756    27.310                                 0.928192    VAL      C      2.254    25.291                                 0.686192    VAL      CB     4.781    25.127                                 1.088192    VAL      CG2    4.617    25.104                                 2.592193    GLY      CA     0.629    23.564                                 0.410193    GLY      O      0.530    23.244                                 -2.015194    PRO      CA     -1.662   21.651                                 -1.873194    PRO      O      -2.403   22.244                                 -4.085194    PRO      CG     -2.311   20.622                                 0.213195    GLU      N      -2.522   23.793                                 -2.439195    GLU      C      -2.095   25.631                                 -4.058195    GLU      CB     -4.043   25.786                                 -2.470195    GLU      CO     -4.315   24.860                                 -0.100195    GLU      DE2    -5.138   24.520                                 0.785196    LEU      CA     0.241    25.929                                 -4.664196    LEU      O      0.305    24.121                                 -6.153196    LEU      CG     2.770    26.178                                 -4.642196    LEU      CD2    4.027    25.721                                 -3.911197    ASP      CA     0.032    25.774                                 -8.480197    ASP      O      1.655    24.734                                 -9.914197    ASP      CG     -2.406   26.351                                 -8.549197    ASP      DD2    -3.035   27.327                                 -8.088198    VAL      CA     3.206    26.970                                 -10.209198    VAL      O      3.752    28.699                                 -8.587198    VAL      CG1    1.930    26.726                                 -12.537199    MET      N      5.374    27.916                                 -10.016199    MET      C      6.845    29.810                                 -10.578199    MET      CB     7.660    27.970                                 -9.077199    MET      SO     6.755    27.449                                 -6.568200    ALA      N      7.426    30.942                                 -10.103200    ALA      C      9.088    32.666                                 -10.272200    ALA      CB     6.932    32.870                                 -11.638201    PRO      N      9.927    33.455                                 -10.95201    PRO      C      10.450   35.127                                 -9.23201    PRO      CB     11.817   34.723                                 -11.400201    PRO      CO     9.941    33.616                                 -12.405202    GLY      CA     10.473   36.204                                 -7.044202    GLY      O      11.352   37.124                                 -4.979203    VAL      CA     13.948   36.929                                 -5.716203    VAL      C      15.133   37.731                                 -7.593203    VAL      CG1    16.096   36.106                                 -4.612204    SER      N      14.865   39.182                                 -5.859204    SER      C      15.067   40.619                                 -7.872204    SER      CB     17.087   39.976                                 -6.326205    ILE      N      13.771   40.965                                 -8.008205    ILE      C      13.207   42.749                                 -9.478205    ILE      CB     11.532   40.833                                 -9.144205    ILE      CG2    10.899   41.281                                 -10.467206    GLN      N      13.956   43.095                                 -10.489206    GLN      C      13.002   44.978                                 -11.630206    GLN      CB     15.455   44.708                                 -11.740206    GLN      CO     17.285   45.145                                 -10.007206    GLN      NE2    16.556   46.260                                 -9.857207    SER      CA     11.217   46.571                                 -11.987207    SER      O      11.919   48.657                                 -11.004207    SER      OG     8.993    46.056                                 -12.613208    THR      CG2    9.171    50.339                                 -14.754208    THR      CB     8.620    50.415                                 -13.357208    THR      C      9.197    50.488                                 -10.803209    LEU      N      9.656    51.613                                 -10.228209    LEU      C      8.673    53.610                                 -9.262209    LEU      CB     10.335   52.192                                 -7.958209    LEU      CD1    11.968   51.114                                 -6.472210    PRO      N      7.790    54.139                                 -8.444210    PRO      C      8.383    56.573                                 -8.639210    PRO      CB     6.302    55.733                                 -7.517210    PRO      CO     7.193    53.491                                 -7.271211    GLY      CA     9.069    58.765                                 -9.410211    GLY      O      11.176   59.005                                 -10.259212    ASN      CA     10.903   57.422                                 -12.643212    ASN      O      13.188   57.181                                 -12.420212    ASN      CG     11.803   58.185                                 -14.814212    ASN      ND2    12.273   59.159                                 -15.576213    LYS      CA     12.810   54.946                                 -10.537213    LYS      O      11.775   53.039                                 -11.613213    LYS      CG     13.206   56.694                                 -8.767213    LYS      CE     14.105   58.218                                 -6.870214    TYR      N      13.681   52.703                                 -10.444214    TYR      C      14.383   50.600                                 -9.489214    TYR      CB     14.641   50.981                                 -11.984214    TYR      CD1    14.689   52.847                                 -13.678214    TYR      CE1    14.230   53.475                                 -14.814214    TYR      CZ     13.204   52.895                                 -15.550215    GLY      N      14.058   49.347                                 -9.158215    GLY      C      14.130   47.325                                 -7.749216    ALA      N      14.810   46.638                                 -6.831216    ALA      C      13.682   44.922                                 -5.512216    ALA      CB     15.715   44.354                                 -6.887217    TYR      CA     11.964   43.488                                 -4.440217    TYR      O      12.202   41.442                                 -5.656217    TYR      CG     10.117   45.291                                 -4.214217    TYR      CD2    9.016    45.933                                 -4.785217    TYR      CE2    8.654    47.219                                 -4.381217    TYR      DM     8.953    49.160                                 -2.988218    ASN      CA     11.640   39.942                                 -3.227201    PRO      CA     11.013   34.130                                 -10.238201    PRO      O      9.579    35.907                                 -9.682201    PRO      CG     11.392   34.040                                 -12.678202    GLY      N      10.925   35.204                                 -8.021202    GLY      C      11.580   36.658                                 -6.115203    VAL      N      12.815   36.503                                 -6.613203    VAL      C      14.706   38.017                                 -6.469203    VAL      CB     14.814   35.688                                 -5.351203    VAL      CG2    14.079   34.741                                 -4.378204    SER      CA     15.572   40.281                                 -6.487204    SER      C      15.786   40.685                                 -8.889204    SER      DG     17.752   41.186                                 -6.672205    ILE      CA     13.069   41.234                                 -9.225205    ILE      O      12.675   43.498                                 -8.648205    ILE      DG1    11.436   39.336                                 -8.810205    ILE      CD1    12.257   38.412                                 -9.771206    GLN      CA     14.204   44.517                                 -10.834206    GLN      O      12.669   44.318                                 -12.621206    GLN      CG     16.684   44.163                                 -10.980206    GLN      OE1    18.328   44.936                                 -9.353207    SER      N      12.359   46.064                                 -11.214207    SER      C      11.089   48.093                                 -11.749207    SER      CB     9.918    45.853                                 -11.569208    THR      N      10.054   48.664                                 -12.326208    THR      OG1    7.570    49.414                                 -13.144208    THR      CA     9.675    50.092                                 -12.173208    THR      O      8.423    49.807                                 -10.049209    LEU      CA     9.192    52.158                                 -8.959209    LEU      O      9.140    54.227                                 -10.222209    LEU      CG     10.804   50.816                                 -7.416209    LEU      CD2    9.607    50.282                                 -6.649210    PRO      CA     7.273    55.517                                 -8.649210    PRO      O      9.491    56.445                                 -8.104210    PRO      CG     6.004    54.379                                 -6.944211    GLY      N      8.077    57.665                                 -9.355211    GLY      C      10.094   58.454                                 -10.490212    ASN      N      9.851    57.770                                 -11.587212    ASN      C      12.059   56.753                                 -12.056212    ASN      CB     11.224   58.595                                 -13.499212    ASN      DD1    11.853   57.054                                 -15.323213    LYS      N      11.803   55.749                                 -11.247213    LYS      C      12.668   53.459                                 -10.866213    LYS      CB     12.769   55.241                                 -9.059213    LYS      CO     13.246   57.030                                 -7.312213    LYS      NZ     15.048   58.705                                 -7.921214    TYR      CA     13.800   51.246                                 -10.722214    TYR      O      15.211   51.253                                 -8.817214    TYR      CG     14.130   51.621                                 -13.246214    TYR      CD2    13.129   51.065                                 -14.014214    TYR      CE2    12.654   51.669                                 -15.178214    TYR      OH     12.756   53.458                                 -16.696215    GLY      CA     14.622   48.772                                 -7.905215    GLY      O      13.249   46.917                                 -8.521216    ALA      CA     14.454   45.203                                 -6.781216    ALA      O      13.948   45.527                                 -4.475217    TYR      N      12.758   43.982                                 -5.575217    TYR      C      12.033   41.928                                 -4.547217    TYR      CB     10.473   43.862                                 -4.570217    TYR      CD1    10.846   45.991                                 -3.236217    TYR      CE1    10.459   47.267                                 -2.790217    TYR      CZ     9.358    47.882                                 -3.391218    ASN      N      11.750   41.386                                 -3.391218    ASN      C      10.204   39.636                                 -2.749218    ASN      O      9.763    40.347                                 -1.81218    ASN      CG     14.031   39.566                                 -2.34218    ASN      ND2    14.660   39.644                                 -1.165219    GLY      CA     8.382    38.130                                 -2.649219    GLY      O      7.873    37.500                                 -4.876220    THR      CA     5.697    35.936                                 -4.179220    THR      O      4.417    36.742                                 -5.958220    THR      OG1    4.136    35.543                                 -2.451221    SER      N      4.738    38.238                                 -4.303221    SER      C      4.760    39.641                                 -6.383221    SER      CB     3.323    40.383                                 -4.546222    MET      N      6.060    39.389                                 -6.485222    MET      SO     7.768    41.533                                 -4.993222    MET      CB     8.351    40.015                                 -7.218222    MET      C      6.877    38.435                                 -8.567223    ALA      N      6.554    37.246                                 -8.041223    ALA      C      5.200    36.068                                 -9.707223    ALA      CB     6.509    34.807                                 -7.923224    SER      CA     2.758    36.488                                 -9.700224    SER      O      2.145    36.593                                 -12.057224    SER      OG     0.492    36.899                                 -9.157225    PRO      CA     3.095    39.130                                 -12.439225    PRO      O      3.406    38.650                                 -14.804225    PRO      CG     4.411    40.402                                 -10.764226    HIS      N      4.769    37.626                                 -13.299226    HIS      C      4.418    35.947                                 -15.061226    HIS      CB     6.608    36.046                                 -13.765226    HIS      ND1    8.048    37.488                                 -12.170226    HIS      CB1    9.270    38.052                                 -12.236227    VAL      N      3.593    35.366                                 -14.199227    VAL      C      1.479    35.197                                 -15.421227    VAL      CB     2.103    33.444                                 -13.619227    VAL      CG2    3.204    32.665                                 -12.891228    ALA      CA     0.011    37.109                                 -15.517228    ALA      O      -0.253   37.435                                 -17.828229    GLY      N      1.791    38.028                                 -16.941229    GLY      C      2.420    37.197                                 -19.187230    ALA      N      2.711    35.988                                 -18.646230    ALA      C      1.424    34.500                                 -20.153230    ALA      CB     3.298    33.624                                 -18.709231    ALA      CA     -1.010   34.416                                 -19.744231    ALA      O      -1.909   35.056                                 -21.852232    ALA      N      -0.778   36.657                                 -20.721232    ALA      C      -0.281   37.284                                 -23.078232    ALA      CB     -0.742   39.121                                 -21.377233    LEU      CA     1.617    36.293                                 -24.209233    LEU      O      0.696    35.231                                 -26.111233    LEU      CG     3.996    36.994                                 -23.453233    LEU      CD2    4.241    37.853                                 -24.680234    ILE      CD1    0.306    30.664                                 -21.657234    ILE      CB     -0.811   32.014                                 -23.570234    ILE      CA     -0.406   33.076                                 -24.644234    ILE      O      -1.883   33.144                                 -26.544235    LEU      CA     -3.596   35.028                                 -25.423235    LEU      O      -4.109   35.914                                 -27.589235    LEU      CG     -5.140   34.899                                 -23.342235    LEU      CD2    -6.252   34.138                                 -24.120236    SER      CA     -1.764   37.237                                 -27.986236    SER      O      -1.746   36.634                                 -30.290236    SER      OG     0.599    37.571                                 -27.582237    LYS      CA     -0.846   34.085                                 -29.95237    LYS      O      -2.378   32.951                                 -31.44237    LYS      CG     0.677    32.240                                 -30.71218    ASN      CB     12.553   39.340                                 -2.154218    ASN      OD1    14.612   39.709                                 -3.422219    GLY      N      9.670    38.554                                 -3.289219    GLY      C      7.570    37.384                                 -3.681220    THR      N      6.561    36.638                                 -3.205220    THR      C      4.879    37.044                                 -4.864220    THR      CB     4.825    34.819                                 -3.526220    THR      CG2    5.704    33.696                                 -2.900221    SER      CA     3.984    39.201                                 -5.169221    SER      O      4.117    40.208                                 -7.277221    SER      OG     3.435    40.282                                 -3.149222    MET      CE     6.471    42.771                                 -5.173222    MET      CG     8.506    41.399                                 -6.602222    MET      CA     6.916    39.670                                 -7.638222    MET      O      7.084    38.567                                 -9.775223    ALA      CA     6.469    36.020                                 -8.885223    ALA      O      5.133    35.948                                 -10.929224    SER      N      4.076    36.360                                 -9.038224    SER      C      2.661    37.161                                 -11.039224    SER      CB     1.801    36.995                                 -8.603225    PRO      N      3.156    38.411                                 -11.159225    PRO      C      3.764    38.469                                 -13.626225    PRO      CB     3.653    40.511                                 -12.054225    PRO      CO     3.735    39.224                                 -10.054226    HIS      CA     5.446    36.879                                 -14.362226    HIS      O      4.425    35.809                                 -16.293226    HIS      CG     7.814    36.859                                 -13.358226    HIS      CD2    8.883    37.118                                 -14.167226    HIS      NE2    9.771    37.866                                 -13.443227    VAL      CA     2.583    34.388                                 -14.727227    VAL      O      1.018    34.773                                 -16.490227    VAL      CG1    1.076    32.476                                 -14.246228    ALA      N      1.003    36.242                                 -14.814228    ALA      C      0.543    37.538                                 -16.868228    ALA      CB     -0.307   38.353                                 -14.668229    GLY      CA     2.352    38.408                                 -18.239229    GLY      O      2.189    37.375                                 -20.384230    ALA      CA     2.794    34.801                                 -19.546230    ALA      O      1.380    34.205                                 -21.343231    ALA      N      0.385    34.623                                 -19.328231    ALA      C      -1.256   35.423                                 -20.864231    ALA      CB     -1.932   34.664                                 -18.549232    ALA      CA     -1.013   37.663                                 -21.792232    ALA      O      -0.841   37.501                                 -24.187233    LEU      N      0.935    36.726                                 -22.967233    LEU      C      0.821    35.169                                 -24.880233    LEU      CB     3.063    35.877                                 -23.907233    LEU      CD1    5.259    36.342                                 -22.921234    ILE      N      0.357    34.199                                 -24.047234    ILE      CG1    0.454    31.223                                 -23.105234    ILE      CG2    -1.803   30.900                                 -24.091234    ILE      C      -1.621   33.597                                 -25.434235    LEU      N      -2.390   34.465                                 -24.779235    LEU      C      -3.258   35.843                                 -26.672235    LEU      CB     -4.432   35.765                                 -24.378235    LEU      CD1    -5.652   35.683                                 -22.145236    SER      N      -2.094   36.438                                 -26.798236    SER      C      -1.491   36.292                                 -29.144236    SER      CB     -0.633   38.234                                 -27.733237    LYS      N      -1.046   35.067                                 -28.882237    LYS      C      -2.113   33.277                                 -30.268237    LYS      CB     0.272    33.112                                 -29.551237    LYS      CO     2.020    31.535                                 -30.442237    LYS      CE     2.345    30.762                                 -31.7238    HIS      N      -2.951   32.989                                 -29.238    HIS      C      -5.334   32.899                                 -28.691238    HIS      CB     -3.948   30.862                                 -28.511238    HIS      ND1    -1.707   29.679                                 -28.835238    HIS      CE1    -1.086   28.851                                 -29.642239    PRO      N      -5.848   33.917                                 -29.365239    PRO      C      -8.204   34.052                                 -28.532239    PRO      CB     -7.018   35.977                                 -29.713239    PRO      CO     -5.436   34.439                                 -30.668240    ASN      CA     -9.529   32.041                                 -29.216240    ASN      O      -10.540  30.610                                 -27.576240    ASN      CG     -7.971   30.827                                 -30.889240    ASN      ND2    -7.670   29.509                                 -30.986241    TRP      CA     -8.304   30.124                                 -26.120241    TRP      O      -9.043   31.833                                 -24.686241    TRP      CG     -6.094   28.903                                 -26.557241    TRP      CD2    -4.839   28.324                                 -26.155241    TRP      CE2    -4.414   27.476                                 -27.216241    TRP      C22    -3.195   26.786                                 -27.174241    TRP      CH2    -2.470   26.873                                 -26.005242    THR      CA     -10.458  30.119                                 -22.911242    THR      O      -8.335   29.674                                 -21.937242    THR      OG1    -10.837  27.786                                 -22.476243    ASN      N      -9.946   30.659                                 -20.611243    ASN      OD1    -11.465  31.518                                 -16.788243    ASN      CB     -9.708   31.530                                 -18.332243    ASN      C      -8.657   29.303                                 -19.010244    THR      N      -9.564   28.362                                 -19.283244    THR      C      -8.133   26.393                                 -19.802244    THR      CB     -10.665  26.098                                 -19.494244    THR      CG2    -10.503  24.595                                 -19.158245    GLN      CA     -6.964   26.362                                 -21.962245    GLN      O      -4.573   26.393                                 -21.447245    GLN      CG     -8.265   25.526                                 -23.989245    GLN      OE1    -9.306   26.769                                 -25.727246    VAL      N      -5.697   28.304                                 -21.218246    VAL      C      -3.936   28.462                                 -19.467246    VAL      CB     -4.779   30.555                                 -20.621246    VAL      CG2    -5.169   31.138                                 -21.959247    ARG      CA     -4.380   27.714                                 -17.168247    ARG      O      -2.705   25.985                                 -16.764247    ARG      CG     -4.987   27.095                                 -14.852247    ARG      NE     -5.440   26.757                                 -12.546247    ARG      NH1    -7.064   27.484                                 -11.210248    SER      N      -4.480   25.505                                 -18.131248    SER      C      -2.657   24.086                                 -19.073248    SER      CB     -5.034   23.408                                 -19.372249    SER      N      -2.500   24.853                                 -20.136249    SER      C      -0.071   25.302                                 -19.940249    SER      CB     -1.369   25.758                                 -22.068250    LEU      N      -0.289   26.333                                 -19.160250    LEU      CD1    -0.373   30.453                                 -17.268250    LEU      CB     0.178    28.063                                 -17.505250    LEU      C      1.092    25.694                                 -17.265251    GLN      N      0.068    25.007                                 -16.714251    GLN      DE1    -2.819   23.424                                 -12.935251    GLN      CG     -1.218   24.814                                 -13.994251    GLN      CA     0.381    23.941                                 -15.745251    GLN      O      1.743    22.014                                 -15.616252    ASN      CA     1.082    21.206                                 -18.282252    ASN      O      2.809    20.442                                 -19.768252    ASN      CG     -1.036   19.926                                 -18.571237    LYS      NZ     3.525    29.848                                 -31.596238    HIS      CA     -4.168   32.163                                 -29.379238    HIS      O      -5.713   32.584                                 -27.562238    HIS      CG     -3.009   29.921                                 -29.237238    HIS      CD2    -3.137   29.258                                 -30.394238    HIS      NE2    -1.948   28.600                                 -30.599239    PRO      CA     -6.908   34.779                                 -28.773239    PRO      O      -8.949   34.519                                 -27.662239    PRO      CG     -6.666   35.294                                 -31.027240    ASN      N      -8.386   32.969                                 -29.227240    ASN      C      -9.508   31.180                                 -27.980240    ASN      CB     -9.403   31.249                                 -30.535240    ASN      DD1    -7.008   31.590                                 -31.147241    TRP      N      -8.354   31.006                                 -27.304241    TRP      C      -9.106   30.638                                 -24.936241    TRP      CB     -6.879   29.830                                 -25.679241    TRP      CD1    -6.338   28.433                                 -27.818241    TRP      NE1    -5.362   27.547                                 -28.211241    TRP      CE3    -4.097   28.406                                 -24.981241    TRP      C23    -2.912   27.667                                 -24.943242    THR      N      -9.727   29.781                                 -24.142242    THR      C      -9.469   30.176                                 -21.747242    THR      CB     -11.579  29.032                                 -22.675242    THR      CG2    -12.494  28.907                                 -23.895243    ASN      ND2    -11.787  30.404                                 -18.747243    ASN      CG     -11.093  31.131                                 -17.905243    ASN      CA     -9.053   30.731                                 -19.444243    ASN      O      -7.593   29.136                                 -18.440244    THR      CA     -9.381   26.934                                 -19.059244    THR      O      -7.324   25.757                                 -19.111244    THR      DG1    -11.735  26.675                                 -18.684245    GLN      N      -8.082   26.716                                 -21.073245    GLN      C      -5.647   27.020                                 -21.520245    GLN      CB     -7.330   26.599                                 -23.397245    GLN      CO     -8.493   25.873                                 -25.428245    GLN      NE2    -7.745   25.312                                 -26.370246    VAL      CA     -4.477   29.040                                 -20.770246    VAL      O      -2.705   28.227                                 -19.361246    VAL      CG1    -3.544   31.272                                 -20.027247    ARG      N      -4.767   28.240                                 -18.462247    ARG      C      -3.770   26.292                                 -17.340247    ARG      CB     -5.533   27.667                                 -16.149247    ARG      CO     -6.056   27.179                                 -13.793247    ARG      CZ     -5.893   26.866                                 -11.315247    ARG      NH2    -5.177   26.428                                 -10.270248    SER      CA     -4.039   24.131                                 -18.426248    SER      O      -1.848   23.253                                 -18.583248    SER      OG     -6.146   23.090                                 -18.532249    SER      CA     -1.223   24.874                                 -20.851249    SER      O      1.026    24.705                                 -20.049249    SER      OG     -0.300   25.419                                 -22.956250    LEU      CD2    1.824    29.814                                 -18.222250    LEU      CG     0.352    29.438                                 -18.151250    LEU      CA     0.718    26.837                                 -18.216250    LEU      O      2.283    25.421                                 -17.032251    GLN      NE2    -2.750   25.512                                 -12.237251    GLN      CO     -2.345   24.550                                 -13.034251    GLN      CB     -0.857   23.621                                 -14.877251    GLN      C      0.959    22.664                                 -16.361252    ASN      N      0.633    22.394                                 -17.590252    ASN      C      2.394    21.359                                 -18.991252    ASN      CB     0.004    20.780                                 -19.292252    ASN      OD1    -0.836   19.355                                 -17.502252    ASN      ND2    -2.234   19.834                                 -19.161253    THR      CA     4.256    22.717                                 -19.713253    THR      O      6.348    23.733                                 -19.427253    THR      DG1    3.595    24.957                                 -20.428254    THR      N      5.218    23.177                                 -17.551254    THR      C      7.466    22.700                                 -16.612254    THR      CB     5.664    23.558                                 -15.132254    THR      CG2    4.530    24.549                                 -14.802255    THR      CA     9.771    22.594                                 -15.817255    THR      O      9.439    22.786                                 -13.474255    THR      DG1    11.082   23.709                                 -17.321256    LYS      N      9.606    20.702                                 -14.314256    LYS      C      10.522   20.333                                 -12.063256    LYS      CB     9.024    18.590                                 -13.249256    LYS      CO     10.286   16.948                                 -11.777256    LYS      NZ     9.243    14.869                                 -11.054257    LEU      CA     11.272   21.038                                 -9.893257    LEU      O      12.096   20.565                                 -7.732257    LEU      CG     11.357   23.620                                 -10.568257    LEU      CD2    12.678   23.468                                 -11.325258    GLY      CA     10.602   15.793                                 -6.879258    GLY      O      8.283    18.956                                 -7.202259    ASP      CA     7.757    17.896                                 -4.516259    ASP      O      6.859    20.039                                 -4.214259    ASP      CG     6.781    17.128                                 -2.241259    ASP      DD2    7.098    16.299                                 -1.321260    SER      CA     4.481    19.587                                 -5.529260    SER      O      3.500    21.503                                 -4.446260    SER      DG     2.745    17.937                                 -5.448261    PHE      CA     3.831    20.468                                 -1.885261    PHE      O      3.944    22.848                                 -1.432261    PHE      CG     3.549    20.337                                 0.715261    PHE      CD2    4.401    21.060                                 1.558261    PHE      CE2    3.945    21.602                                 2.748262    TYR      N      5.778    21.758                                 -2.305262    TYR      C      6.820    23.689                                 -3.545262    TYR      CB     8.123    22.455                                 -1.851262    TYR      CD1    8.084    20.484                                 -0.364262    TYR      CE1    8.062    19.873                                 0.882262    TYR      CZ     8.069    20.672                                 2.018262    TYR      N      6.626    23.104                                 -4.693263    TYR      C      5.626    23.680                                 -6.956263    TYR      CB     7.928    22.768                                 -6.681263    TYR      CD1    10.064   24.046                                 -6.657263    TYR      CE1    11.335   24.326                                 -6.168263    TYR      CZ     11.838   23.618                                 -5.106264    GLY      N      4.471    23.161                                 -6.516264    GLY      C      3.847    22.196                                 -8.556265    LYS      N      3.436    22.477                                 -9.754265    LYS      C      5.188    22.232                                 -11.464265    LYS      CB     2.755    22.071                                 -12.044265    LYS      CO     0.710    20.548                                 -12.079265    LYS      NZ     -1.678   20.757                                 -12.489266    GLY      CA     7.120    23.612                                 -11.325266    GLY      O      6.177    25.793                                 -11.648267    LEU      CA     8.490    26.660                                 -13.097267    LEU      O      7.953    25.909                                 -15.298267    LEU      CG     10.432   28.050                                 -14.058267    LEU      CD2    11.924   27.921                                 -14.327268    ILE      CA     6.406    28.035                                 -15.944268    ILE      O      8.539    28.793                                 -16.912268    ILE      CG1    6.099    30.541                                 -15.592268    ILE      CD1    5.399    31.769                                 -16.262253    THR      N      3.018    22.505                                 -18.923253    THR      C      5.381    23.247                                 -18.818253    THR      CB     4.086    23.672                                 -20.952253    THR      CG2    3.147    23.130                                 -22.032254    THR      CA     6.216    23.612                                 -16.588254    THR      O      7.402    21.580                                 -17.095254    THR      DG1    5.129    22.178                                 -15.040255    THR      N      8.499    23.296                                 -16.076255    THR      C      9.621    22.031                                 -14.414255    THR      CB     11.080   23.455                                 -15.897255    THR      CG2    12.286   22.628                                 -15.406256    LYS      CA     9.364    20.063                                 -13.010256    LYS      O      11.662   20.274                                 -12.592256    LYS      CG     9.018    17.805                                 -11.921256    LYS      CE     10.212   15.940                                 -10.623257    LEU      N      10.212   20.674                                 -10.824257    LEU      C      11.250   20.232                                 -8.614257    LEU      CB     11.187   22.547                                 -9.522257    LEU      CD1    11.245   25.003                                 -9.921258    GLY      N      10.431   19.282                                 -8.298258    GLY      C      9.168    18.703                                 -6.373259    ASP      N      9.024    18.282                                 -5.150259    ASP      C      6.659    18.941                                 -4.709259    ASP      CB     7.996    17.540                                 -3.053259    ASP      OD1    5.611    17.527                                 -2.354260    SER      N      5.560    18.610                                 -5.312260    SER      C      4.046    20.362                                 -4.289260    SER      CB     3.345    18.919                                 -6.289261    PHE      N      4.241    19.778                                 -3.112261    PHE      C      4.544    21.846                                 -1.863261    PHE      CB     4.053    19.749                                 -0.563261    PHE      CD1    2.206    20.163                                 1.125261    PHE      CE1    1.737    20.717                                 2.315261    PHE      CZ     2.605    21.465                                 3.114262    TYR      CA     6.688    22.914                                 -2.251262    TYR      O      7.201    14.853                                 -3.393262    TYR      CG     8.146    21.892                                 -0.454262    TYR      CD2    8.149    22.669                                 0.698262    TYR      CE2    8.114    22.069                                 1.962262    TYR      OH     7.965    20.029                                 3.205263    TYR      CA     6.812    23.655                                 -6.022263    TYR      O      5.781    24.117                                 -8.111263    TYR      CG     9.279    23.035                                 -6.068263    TYR      CD2    9.800    22.342                                 -4.995263    TYR      CE2    11.062   22.640                                 -4.491263    TYR      DH     13.065   23.949                                 -4.597264    GLY      CA     3.301    23.064                                 -7.412264    GLY      O      4.647    21.274                                 -8.365265    LYS      CA     3.834    21.798                                 -10.971265    LYS      O      5.684    21.563                                 -12.386265    LYS      CG     1.490    21.563                                 -11.305265    LYS      CE     -0.692   20.496                                 -11.391266    GLY      N      5.787    23.226                                 -10.817266    GLY      C      7.155    25.052                                 -11.818267    LEU      N      8.262    25.336                                 -12.480267    LEU      C      7.804    26.771                                 -14.437267    LEU      CB     10.010   26.855                                 -13.214267    LEU      CD1    10.096   29.331                                 -13.250268    ILE      N      7.064    27.863                                 -14.632268    ILE      C      7.426    28.246                                 -17.065268    ILE      CB     5.369    29.210                                 -15.899268    ILE      CG2    4.243    28.925                                 -14.867269    ASN      N      7.007    27.843                                 -18.237269    ASN      CA     7.802    27.975                                 -19.269    ASN      O      5.965    27.760                                 -20.269    ASN      CG     9.161    26.806                                 -21.210269    ASN      ND2    10.011   25.796                                 -21.472270    VAL      CA     5.863    30.418                                 -21.614270    VAL      O      5.097    29.969                                 -23.872270    VAL      CG1    6.849    32.757                                 -21.879271    GLN      N      7.325    29.701                                 -23.352271    GLN      C      6.869    27.934                                 -25.031271    GLN      CB     9.104    29.220                                 -24.964271    GLN      CO     10.901   28.585                                 -26.582271    GLN      NE2    11.702   28.553                                 -25.510272    ALA      CA     6.224    25.712                                 -24.240272    ALA      O      3.898    25.505                                 -25.001273    ALA      N      4.247    26.661                                 -23.135273    ALA      C      2.081    27.528                                 -24.020273    ALA      CB     2.736    27.773                                 -21.585274    ALA      CB     2.952    30.391                                 -26.210274    ALA      C      1.730    28.367                                 -27.090275    GLN      N      2.350    27.194                                 -27.314275    GLN      C      2.147    27.261                                 -29.777275    GLN      OT     1.193    27.361                                 -30.590275    GLN      CG     0.501    24.664                                 -27.447275    GLN      OE1    -1.376   23.895                                 -28.729269    ASN      C      6.839    28.554                                 -20.485269    ASN      CB     8.432    26.653                                 -19.895269    ASN      OD1    8.993    27.626                                 -22.122270    VAL      N      6.908    29.868                                 -20.724270    VAL      C      6.059    30.007                                 -23.054270    VAL      CB     5.656    31.950                                 -21.422270    VAL      CG2    4.420    32.362                                 -22.232271    GLN      CA     7.603    29.270                                 -24.744271    GLN      O      6.213    27.806                                 -26.091271    GLN      CG     9.406    28.618                                 -26.338271    GLN      DE1    11.369   28.579                                 -27.718272    ALA      N      6.977    26.999                                 -24.092272    ALA      C      4.701    25.958                                 -24.164272    ALA      CB     6.743    24.742                                 -23.172273    ALA      CA     2.840    26.921                                 -22.859273    ALA      O      0.899    27.219                                 -24.255274    ALA      N      2.755    28.404                                 -24.782274    ALA      CA     2.109    29.144                                 -25.847274    ALA      O      0.980    28.949                                 -27.921275    GLN      CA     2.048    26.389                                 -28.527275    GLN      O      3.260    27.807                                 -29.916275    GLN      CB     0.666    25.734                                 -28.520275    GLN      CO     -0.823   23.936                                 -27.631275    GLN      NE2    -1.373   23.411                                 -26.538______________________________________</pre>
    
    <p>The above structural studies together with the above referenced kinetic data and kinetic data presented herein indicate that the subsites in the binding cleft of subtilisin are capable of interacting with substrate amino acid residues from P-4 to P-2'.</p>
    <p>The most extensively studied of the above residues are Gly166, Gly169 and Ala152. There amino acids were identified as residues within the S-1 subsite. As seen in FIG. 3, which is a stereoview of the S-1 subsite, Gly166 and Gly169 occupy positions at the bottom of the S-1 subsite, whereas Ala152 occupies a position near the top of S-1, close to the catalytic Ser221.</p>
    <p>All 19 amino acid substitutions of Gly166 and Gly169 have been made. As will be indicated in the examples which follow, the preferred replacement amino acids for Gly166 and/or Gly169 will depend on the specific amino acid occupying the P-1 position of a given substrate.</p>
    <p>The only substitutions of Ala152 presently made and analyzed comprise the replacement of Ala152 with Gly and Ser. The results of these substitutions on P-1 specificity will be presented in the examples.</p>
    <p>In addition to those residues specifically associated with specificity for the P-1 substrate amino acid, Tyr104 has been identified as being involved with P-4 specificity. Substitutions at Phe189 and Tyr217, however, are expected to respectively effect P-2' and P-1' specificity.</p>
    <p>The catalytic activity of subtilisin has also been modified by single amino acid substitutions at Asn155. The catalytic triad of subtilisin is shown in FIG. 4. As can be seen, Ser221, His64 and Asp32 are positioned to facilitate nucleophilic attach by the serine hydoxylate on the carbonyl of the scissile peptide bond. Several hydrogen bonds may also help to stabilize the transition state complex for the tetrahedral substrate intermediate. One hydrogen bond is between aspartate and the positively charged histidine, ND1. Kossiakoff, A. A., et al. (1981) Biochem. 20, 6462-6474. A second hydrogen bond forms between the scissile amide nitrogen of the substrate and the (NE2) proton on the histidine. A third set of hydrogen bonds forms between the enzyme and the oxyanion that is produced from the carbonyl oxygen of the substrate. This latter set of hydrogen bonds is formed differently by the mammalian serine proteases and substilisin. A fourth hydrogen bond appears to exist between the amide nitrogen of the peptide bond between P-1 and P-2 and the carbonyl oxygen of Ser125. Specifically, x-ray crystallographic studies of chymotrypsin (Henderson, R. (1970) J. Mol. Biol. 54, 341) indicate that two hydrogen bonds form between the substrate oxyanion and two main-chain amide protons from the enzyme (Gly193 and the catalytic Ser195). Crystallographic studies of subtilisin (Robertus, et al. (1972) Biochem. 11, 4293-4303; Matthews, et al. (1975) J. Biol. Chem. 250, 7120-7126; Poulos, et al. (1976) J. Biol. Chem. 250, 1097-1103) show that two hydrogen bonds are also formed with the oxyanion; one hydrogen bond donor is from the catalytic serine-221 main-chain amide while the other is from one of the NE2 protons of the asparagine-155 side chain. See FIG. 4.</p>
    <p>Asn155 was substituted with Ala, Asp, His, Glu and Thr. These substitutions were made to investigate the the stabilization of the charged tetrahedral intermediate of the transition state complex by the potential hydrogen bond between the side chain of Asn155 and the oxyanion of the intermediate. These particular substitutions caused large decreases in substrate turnover, kcat (200 to 4,000 fold), marginal decreases in substrate binding Km (up to 7 fold), and a loss in transition state stabilization energy of 2.2 to 4.7 kcal/mol. The retention of Km and the drop in kcat will make these mutant enzymes useful as binding proteins for specific peptide sequences, the nature of which will be determined by the specificity of the precursor protease.</p>
    <p>Various other amino acid residues have been identified which affect alkaline stability. In some cases, mutants having altered alkaline stability also have altered thermal stability.</p>
    <p>In B. amyloliquefaciens subtilisin residues Asp36, Ile107, Lys170, Ser204 and Lys213 have been identified as residues which upon substitution with a different amino acid alter the alkaline stability of the mutated enzyme as compared to the precursor enzyme. The substitution of Asp36 with Ala and the substitution of Lys170 with Glu each resulted in a mutant enzyme having a lower alkaline stability as compared to the wild type subtilisin. When Ile107 was substituted with Val, Ser204 substituted with Cys, Arg or Leu or Lys213 substituted with Arg, the mutant subtilisin had a greater alkaline stability as compared to the wild type subtilisin. However, the mutant Ser204P demonstrated a decrease in alkaline stability.</p>
    <p>In addition, other residues, previously identified as being associated with the modification of other properties of subtilisin, also affect alkaline stability. These residues include Ser24, Met50, Glu156, Gly166, Gly169 and Tyr217. Specifically the following particular substitutions, result in an increased alkaline stability: Ser24C, Met50F, Gly156Q or S, Gly166A, H, K, N or Q, Gly169S or A, and Tyr217F, K, R or L. The mutant Met50V, on the other hand, results in a decrease in the alkaline stability of the mutant subtilisin as compared to wild type subtilisin.</p>
    <p>Other residues involved in alkaline stability based on the alkaline stability screen include the mutants of Table I for residues Asp197 and Met222.</p>
    <p>Various other residues have been identified as being involved in thermal stability as determined by the thermal stability screen herein. These residues include the above identified residues which effect alkaline stability and Met199 and Tyr21. These latter two residues are also believed to be important for alkaline stability. Mutants at these residues include I199 and F21.</p>
    <p>The amino acid sequence of B. amyloliquefaciens substilisin has also been modified by substituting two or more amino acids of the wild-type sequence. Six categories of multiply substituted mutant subtilisin have been identified. The first two categories comprise thermally and oxidatively stable mutants. The next three other categories comprise mutants which combine the useful properties of any of several single mutations of B. amyloliquefaciens subtilisin. The last category comprises mutants which have modified alkaline and/or thermal stability.</p>
    <p>The first category is double mutants in which two cysteine residues have been substituted at various amino acid residue positions within the subtilisin BPN' molecule. Formation of disulfide bridges between the two substituted cysteine residues results in mutant subtilisins with altered thermal stability and catalytic activity. These mutants include A21/C22/C87 and C24/C87 which will be described in more detail in Example 11.</p>
    <p>The second category of multiple subtilisin mutants comprises mutants which are stable in the presence of various oxidizing agents such as hydrogen peroxide or peracids. Examples 1 and 2 describe these mutants which include F50/I124/Q222, F50/I124, F50/Q222, F50/L124/Q222, I124/Q222 and L124/Q222.</p>
    <p>The third category of multiple subtilisin mutants comprises mutants with substitutions at position 222 combined with various substitutions at positions. 166 or 169. These mutants, for example, combine the property of oxidative stability of the A222 mutation with the altered substrate specificity of the various 166 or 169 substitutions. Such multiple mutants include A166/A222, A166/C222, F166/C222, K166/A222, K166/C222, V166/A222 and V166/C222. The K166/A222 mutant subtilisin, for example, has a kcat/Km ratio which is approximately two times greater than that of the single A222 mutant subtilisin when compared using a substrate with phenylalanine as the P-1 amino acid. This category of multiple mutant is described in more detail in Example 12.</p>
    <p>The fourth category of multiple mutants combines substitutions at position 156 (Glu to Q or S) with the substitution of Lys at position 166. Either of these single mutations improve enzyme performance upon substrates with glutamate as the P-1 amino acid. When these single mutations are combined, the resulting multiple enzyme mutants perform better than either precursor. See Example 9.</p>
    <p>The fifth category of multiple mutants contain the substitution of up to four amino acids of the B. amyloliquefaciens subtilisin sequence. These mutants have specific properties which are virtually identicle to the properties of the subtilisin from B. licheniformis. The subtilisin from B. licheniformis differs from B. amyloliquefaciens subtilisin at 87 out of 275 amino acids. The multiple mutant F50/S156/A169/L217 was found to have similar substrate specificity and kinetics to the licheniformis enzyme. (See Example 13.) However, this is probably due to only three of the mutations (S156, A169 and L217) which are present in the substrate binding region of the enzyme. It is quite surprising that, by making only three changes out of the 87 different amino acids between the sequence of the two enzymes, the B. amyloliquifaciens enzyme was converted into an enzyme with properties similar to B. licheniformis enzyme. Other enzymes in this series include F50/Q156/N166/L217 and F50/S156/L217.</p>
    <p>The sixth category of multiple mutants includes the combination of substitutions at position 107 (Ile to V) with the substitution of Lys at position 213 with Arg, and the combination of substitutions of position 204 (preferably Set to C or L but also to all other amino acids) with the substituion of Lys at position 213 with R. Other multiple mutants which have altered alkaline stability include Q156/K166, Q156/N166, S156/K166, S156/N166 (previously identified as having altered substrate specificity), and F50/S156/A169/L217 (previously identified as a mutant of B. amyloliquifaciens subtilisin having properties similar to subtilisin from B. licheniformis). The mutant F50/V107/R213 was constructed based on the observed increase in alkaline stability for the single mutants F50, V107 and R213. It was determined that the V107/R213 mutant had an increased alkaline stability as compared to the wild type subtilisin. In this particular mutant, the increased alkaline stability was the result of the cumulative stability of each of the individual mutations. Similarly, the mutant F50/V107/R213 had an even greater alkaline stability as compared to the V107/R213 mutant indicating that the increase in the alkaline stability due to the F50 mutation was also cumulative.</p>
    <p>Table IV summarizes the multiple mutants which have been made including those not mentioned above.</p>
    <p>In addition, based in part on the above results, substitution at the following residues in subtilisin is expected to produce a multiple mutant having increased thermal and alkaline stability: Ser24, Met50, Ile107, Glu156, Gly166, Gly169, Ser204, Lys213, Gly215, and Tyr217.</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">              TABLE IV______________________________________          Triple, QuadrupleDouble Mutants or Other Multiple______________________________________C22/C87        F50/I124/Q222C24/C87        F50/L124/Q222V45/V48        F50/L124/A222C49/C94        A21/C22/C87C49/C95        F50/S156/N166/L217C50/C95        F50/Q156/N166/L217C50/C110       F50/S156/A169/L217F50/I124       F50/S156/L217F50/Q222       F50/Q156/K166/L217I124/Q222      F50/S156/K166/L217Q156/K166      F50/Q156/K166/K217Q156/K166      F50/S156/K166/K217Q156/N166      F50/V107/R213S156/D166       S153/S156/A158/G159/S160/Δ161-S156/K166      164/I165/S166/A169/R170!S156/N166      L204/R213S156/A169      R213/204A, E, Q, D, N, G, K,A166/A222      V, R, T, P, I, M, F, Y, WA166/C222      or HF166/A222      V107/R213F166/C222K166/A222K166/C222V166/A222V166/C222A169/A222A169/A222A169/C222A21/C22______________________________________</pre>
    
    <p>In addition to the above identified amino acid residues, other amino acid residues of subtilisin are also considered to be important with regard to substrate specificity. These are the aforementioned residues which have yet to be mutated. Mutation of each of these residues is expected to produce changes in the substrate specificity of subtilisin. Moreover, multiple mutations among these residues and among the previously identified residues are also expected to produce subtilisin mutants having novel substrate specificity.</p>
    <p>Particularly important residues are His67, Ile107, Leu126 and Leu135. Mutation of His67 should alter the S-1' subsite, thereby altering the specificity of the mutant for the P-1' substrate residue. Changes at this position could also affect the pH activity profile of the mutant. This residue was identified based on the inventor's substrate modeling from product inhibitor complexes.</p>
    <p>Ile107 is involved in P-4 binding. Mutation at this position thus should alter specificity for the P-4 substrate residue. Ile107 was also identified by molecular modeling from product inhibitor complexes.</p>
    <p>The S-2 binding site includes the Leu126 residue. Modification at this position should therefore affect P-2 specificity. Moreover, this residue is believed to be important to convert subtilisin to an amino peptidase. The pH activity profile should also be modified by appropriate substitution. These residues were identified from inspection of the refined model, the three dimensional structure from modeling studies. A longer side chain is expected to preclude binding of any side chain at the S-2 subsite. Therefore, binding would be restricted to subsites S-1, S-1', S-2', S-3' and cleavage would be forced to occur after the amino terminal peptide.</p>
    <p>Leu135 is in the S-4 subsite and if mutated should alter substrate specificity for P-4 if mutated. This residue was identified by inspection of the three-dimensional structure and modeling based on the product inhibitor complex of F222.</p>
    <p>In addition to these sites, specific amino acid residues within the segments 97-103, 126-129 and 213-215 are also believed to be important to substrate binding.</p>
    <p>Segments 97-103 and 126-129 form an antiparallel beta sheet with the main chain of substrate residues P-4 through P-2. Mutating residues in those regions should affect the substrate orientation through main chain (enzyme)--main chain (substrate) interactions, since the main chain of these substrate residues do not interact with these particular residues within the S-4 through S-2 subsites.</p>
    <p>Within the segment 97-103, Gly97 and Asp99 may be mutated to alter the position of residues 101-103 within the segment. Changes at these sites must be compatible, however. In B. amyloliquifaciens subtilisin Asp99 stabilizes a turn in the main chain tertiary folding that affects the direction of residues 101-103. B. licheniformis subtilisin Asp97, functions in an analogous manner.</p>
    <p>In addition to Gly97 and Asp99, Ser101 interacts with Asp99 in B. amyliquefaciens subtilisin to stabilize the same main chain turn. Alterations at this residue should alter the 101-103 main chain direction. Mutations at Glu103 are also expected to affect the 101-103 main chain direction.</p>
    <p>The side chain of Gly102 interacts with the substrate P-3 amino acid. Side chains of substituted amino acids thus are expected to significantly affect specificity for the P-3 substrate amino acids.</p>
    <p>All the amino acids within the 127-129 segment are considered important to substrate specificity. Gly 127 is positioned such that its side chain interacts with the S-1 and S-3 subsites. Altering this residue thus should alter the specificity for P-1 and P-3 residues of the substrate.</p>
    <p>The side chain of Gly128 comprises a part of both the S-2 and S-4 subsites. Altered specificity for P-2 and P-4 therefore would be expected upon mutation. Moreover, such mutation may convert subtilisin into an amino peptidase for the same reasons substitutions of Leu126 would be expected to produce that result.</p>
    <p>The Pro129 residue is likely to restrict the conformational freedom of the sequence 126-133, residues which may play a major role in determining P-1 specificity. Replacing Pro may introduce more flexibility thereby broadening the range of binding capabilities of such mutants.</p>
    <p>The side chain of Lys213. is located within the S-3 subsite. All of the amino acids within the 213-215 segment are also considered to be important to substrate specificity. Accordingly, altered P-3 substrate specificity is expected upon mutation of this residue.</p>
    <p>The Tyr214 residue does not interact with substrate but is positioned such that it could affect the conformation of the hair pin loop 204-217.</p>
    <p>Finally, mutation of the Gly215 residue should affect the S-3' subsite, and thereby alter P-3' specificity.</p>
    <p>In addition to the above substitutions of amino acids, the insertion or deletion of one or more amino acids within the external loop comprising residues 152-172 may also affect specificity. This is because these residues may play a role in the "secondary contact region" described in the model of streptomyces subtilisin inhibitor complexed with subtilisin. Hirono, et al. (1984) J. Mol. Biol. 178, 389-413. Thermitase K has a deletion in this region, which eliminates several of these "secondary contact" residues. In particular, deletion of residues 161 through 164 is expected to produce a mutant subtilisin having modified substrate specificity. In addition, a rearrangement in this area induced by the deletion should alter the position of many residues involved in substrate binding, predominantly at P-1. This, in turn, should affect overall activity against proteinaceous substrates.</p>
    <p>The effect of deletion of residues 161 through 164 has been shown by comparing the activity of the wild type (WT) enzyme with a mutant enzyme containing this deletion as well as multiple substitutions (i.e., S153/S156/A158/G159/S160/Δ161-164/I165/S166/A169/R170). This produced the following results:</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">              TABLE V______________________________________  kcat       Km      kcat/Km______________________________________WT       50           1.4e-4  3.6e5Deletion  8           5.0e-6  1.6e6mutant______________________________________</pre>
    
    <p>The WT has a kcat 6 times greater than the deletion mutant but substrate binding is 28 fold tighter by the deletion mutant. The overall efficiency of the deletion mutant is thus 4.4 times higher than the WT enzyme.</p>
    <p>All of these above identified residues which have yet to be substituted, deleted or inserted into are presented in Table VI.</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">              TABLE VI______________________________________Substitution/Insertion/DeletionResidues______________________________________   His67        Ala152   Leu126       Ala153   Leu135       Gly154   Gly97        Asn155   Asp99        Gly156   Ser101       Gly157   Gly102       Gly160   Glu103       Thr158   Leu126       Ser159   Gly127       Ser161   Gly128       Ser162   Pro129       Ser163   Tyr214       Thr164   Gly215       Val165                Gly166                Tyr167                Pro168                Gly169                Lys170                Tyr171                Pro172______________________________________</pre>
    
    <p>The mutants herein may be obtained as salts. It is clear that the ionization state of a protein will be dependent on the pH of the surrounding medium, if it is in solution, or of the solution from which it is prepared, if it is in solid form. Acidic proteins are commonly prepared as, for example, the ammonium, sodium, or potassium salts; basic proteins as the chlorides, sulfates, or phosphates. Accordingly, the present application includes both electrically neutral and salt forms of the designated carbonyl hydrolases, and the term carbonyl hydrolase referes to the organic structural backbone regardless of ionization state.</p>
    <p>The carbonyl hydrolase mutants are particularly useful in the food processing and cleaning arts. The carbonyl hydrolases, including mutants, are produced by fermentation as described herein and recovered by suitable techniques. See for example K. Anstrup, 1974, Industrial Aspects of Biochemistry, ed. B. Spencer pp. 23-46. They are formulated with detergents or other surfactants in accord with methods known per se for use in industrial processes, especially laundry. In the latter case the enzymes are combined with detergents, builders, bleach and/or flourescent whitening agents as is known in the art for proteolytic enzymes. Suitable detergents include linear alkyl benzene sulfonates, alkyl ethoxylated sulfate, sulfated linear alcohol or ethoxylated linear alcohol. The compositions may be formulated in granular or liquid form. See for example U.S. Pat. Nos. 3,623,957; 4,404,128; 4,381,247; 4,404,115; 4,318,818; 4,261,868; 4,242,219; 4,142,999; 4,111,855; 4,011,169; 4,090,973; 3,985,686; 3,790,482; 3,749,671; 3,560,392; 3,558,498; and 3,557,002.</p>
    <p>The following disclosure is intended to serve as a representation of embodiments herein, and should not be construed as limiting the scope of this application. These specific examples disclose the construction of certain of the above identified mutants. The construction of the other mutants, however, is apparent from the disclosure herein and that presented in EPO Publication No. 0130756.</p>
    <heading>Glossary of Experimental Manipulations</heading> <p>In order to simplify the Examples certain frequently occurring methods will be referenced by shorthand phrases.</p>
    <p>Plasmids are designated by a small p proceeded and/or followed by capital letters and/or numbers. The starting plasmids herein are commercially available, are available on an unrestricted basis, or can be constructed from such available plasmids in accord with published procedures.</p>
    <p>"Klenow treatment" refers to the process of filling a recessed 3' end of double stranded DNA with deoxyribonucleotides complementary to the nucleotides making up the protruding 5' end of the DNA strand or exonucleolytic removal of a protruding 3' end of a double stranded DNA fragment. This process is usually used to fill in a recessed end resulting from a restriction enzyme cleavage of DNA. This creates a blunt or flush end, as may-be required for further ligations. Treatment with Klenow is accomplished by reacting (generally for 15 minutes at 15° C.) the appropriate complementary deoxyribonucleotides with the DNA to be filled in under the catalytic activity (usually 10 units) of the Klenow fragment of E. coli DNA polymerase I ("Klenow"). Klenow and the other reagents needed are commercially available. The procedure has been published extensively. See for example T. Maniatis, et al., 1982, Molecular Cloning, pp. 107-108.</p>
    <p>"Digestion" of DNA refers to catalytic cleavage of the DNA with an enzyme that acts only at certain locations in the DNA. Such enzymes are called restriction enzymes, and the sites for which each is specific is called a restriction site. "Partial" digestion refers to incomplete digestion by restriction enzyme, i.e., conditions are chosen that result in cleavage of some but not all of the sites for a given restriction endonuclease in a DNA substrate. The various restriction enzymes used herein are commercially available and their reaction conditions, cofactors and other requirements as established by the enzyme suppliers were used. Restriction enzymes commonly are designated by abbreviations composed of a capital letter followed by other letters and then, generally, a number representing the microorganism from which each restriction enzyme originally was obtained. In general, about 1 μg of plasmid or DNA gragment is used with about 1 unit of enzyme in about 20 μl of buffer solution. Appropriate buffers and substrate amounts for particular restriction enzymes are specified by the manufacturer. Incubation times of about 1 hour at 37° C. are ordinarily used, but may vary in accordance with the supplier's instructions. After incubation, protein is removed by extraction with phenol and chloroform, and the digested nucleic acid is recovered from the aqueous fraction by precipitation with ethanol. Digestion with a restriction enzyme infrequently is followed with bacterial alkaline phosphatase hydrolysis of the terminal 5' phosphates to prevent the two restriction cleaved ends of a DNA fragment from "circularizing" or forming a closed loop that would impede insertion of another DNA fragment at the restriction site. Unless otherwise stated, digestion of plasmids is not followed by 5' terminal dephosphorylation. Procedures and reagents for dephosphorylation are conventional (T. Maniatis, et al., Id., pp. 133-134).</p>
    <p>"Recovery" or "isolation" of a given fragment of DNA from a restriction digest means separation of the digest on 6 percent polyacrylamide gel electrophoresis, identification of the fragment of interest by molecular weight (using DNA fragments of known molecular weight as markers), removal of the gel section containing the desired fragment, and separation of the gel from DNA. This procedure is known generally. For example, see R. Lawn, et al., 1981, "Nucleic Acids Res." 9:6103-6114, and D. Goeddel, et al., 1980, "Nucleic Acids Res." 8:4057.</p>
    <p>"Southern Analysis" is a method by which the presence of DNA sequences in a digest or DNA-containing composition is confirmed by hybridization to a known, labelled oligonucleotide or DNA fragment. For the purposes herein, Southern analysis shall mean separation of digests on 1 percent agarose and depurination as described by G. Wahl, et al., 1979, "Proc. Nat. Acad. Sci. U.S.A." 76:3683-3687, transfer to nitrocellulose by the method of E. Southern, 1975, "J. Mol. Biol." 98:503-517, and hybridization as described by T. Maniatis, et al., 1978, "cell" 15:687-701.</p>
    <p>"Transformation" means introducing DNA into an organism so that the DNA is replicable, either as an extrachromosomal element or chromosomal integrant. Unless otherwise stated, the method used herein for transformation of E. coli is the CaCl<sub>2</sub> method of Mandel, et al., 1970, "J. Mol. Biol." 53:154, and for Bacillus, the method of Anagnostopolous, et al., 1961, "J. Bact." 81:741-746.</p>
    <p>"Ligation" refers to the process of forming phosphodiester bonds between two double stranded nucleic acid fragments (T. Maniatis, et al., Id., p. 146). Unless otherwise stated, ligation was accomplished using known buffers and conditions with 10 units of T4 DNA ligase ("ligase") per 0.5 μg of approximately equimolar amounts of the DNA fragments to be ligated. Plasmids from the transformants were prepared, analyzed by restriction mapping and/or sequenced by the method of Messing, et al., 1981, "Nucleic Acids Res.", 2:309.</p>
    <p>"Preparation" of DNA from transformants means isolating plasmid DNA from microbial culture. Unless otherwise stated, the alkaline/SDS method of Maniatis, et al., Id., p. 90, was used.</p>
    <p>"Oligonucleotides" are short length single or double stranded polydeoxynucleotides which were chemically synthesized by the method of Crea, et al., 1980, "Nucleic Acids Res." 8:2331-2348 (except that mesitylene nitrotriazole was used as a condensing agent) and then purified on polyacrylamide gels.</p>
    <p>All mutant plasmids were transformed (Anagnostopoulos, C., et al. (1961) J. Bacteriol. 81, 741-746) into BG2036 (Yang, M. (1984) J. Bacteriol. 160, 15-21) to express mutant subtilisins as described (Estell, D. A., et al. (1985) J. Biol. Chem. 260, 6518-6521).</p>
    <p>All literature citations are expressly incorporated by reference.</p>
    <p>The following is presented by way of example and is not to be construed as a limitation to the scope of the invention.</p>
    <heading>EXAMPLE 1</heading> <heading>Identification of Peracid Oxidizable Residues of Subtilisin Q222 and L222</heading> <p>The activity of naturally-occurring subtilisin is rapidly reduced up to 85% by a series of oxidants. One of the most characterized modification is the conversion of Met222 to met-sulfoxide in the presence of hydrogen peroxide. Stauffer, C. E., et al. (1969) J. Biol. Chem. 244, 5333-5338. This defect has been eliminated by substituting a variety of non-oxidizable amino acids into this position by site-directed mutagenesis of the B. amyloliquifaciens enzyme, thereby confering enhanced stability to hydrogen peroxide. See EPO Publication No. 0130756 and Estell, D. A., et al. (1985) J. Biol. Chem. 260, 6518. However, as shown in FIGS. 6A and 6B, organic peracid oxidants can still inactivate the mutant enzymes Met222L and Met222Q (L222 and Q222). This example describes the identification of peracid oxidizable sites in 222 substituted mutant subtilisins.</p>
    <p>The first step was to determine the type of amino acid involved in peracid oxidation. Except under drastic conditions (Means, G. E., et al. (1971) Chemical Modifications of Proteins, Holden-Day, S. F., CA, pp. 160-162), organic peracids modify only methionine and tryptophan in subtilisin. In order to rule out tryptophan as a candidate, difference spectra of the enzyme over the 250 nm to 350 nm range were determined during an inactivation titration employing the reagent, diperdodecanoic acid (DPDA) as the oxidant. Despite quantitative inactivation of the enzyme, no change in absorbance over this wavelength range was noted as shown in FIGS. 7A and 7B. Oxidation of tryptophan would be expected to result in marked changes over this region. Fontana, A., et al. (1980) Methods in Peptide and Protein Sequence Analysis (C. Birr ed.) Elsevier, New York, p. 309. The absence of tryptophan modification implied oxidation of one or more of the remaining methionines of B. amyloliquefaciens subtilisin. See FIGS. 1A and 1B.</p>
    <p>To confirm this result the recombinant subtilisin Met222F was cleaved with cyanogen bromide (CNBr) both before and after oxidation by DPDA. The peptides produced by CNBr cleavage were analyzed on high resolution SDS-pyridine peptide gels (SPG).</p>
    <p>Subtilisin Met222F (F222) was oxidized in the following manner. Purified F222 was resuspended in 0.1M sodium borate pH 9.5 at 10 mg/ml and was added to a final concentration of 26 diperdodecanoic acid (DPDA) at 26 mg/ml was added to produce an effective active oxygen concentration of 30 ppm. The sample was incubated for at least 30 minutes at room temperature and then quenched with 0.1 volume of 1M Tris pH 8.6 buffer to produce a final concentration of 0.1M Tris pH 8.6). 3 mM phenylmethylsulfonyl fluoride (PMSF) was added and 2.5 ml of the sample was applied to a Pharmacia PD10 column equilibrated in 10 mM sodium phosphate pH 6.2, 1 mM PMSF. 3.5 ml of 10 mM sodium phosphate pH 6.2, 1 mM PMSF was applied and the eluant collected. The sample was assumed to be at 7 mg/ml based on the observation that a 2.5 ml sample of untreated F222 at 10 mg/ml in phosphate buffer when treated with PMSF and desalted in the same manner on a Pharmacia PD10 column produced a concentration of about 7 mg/ml.</p>
    <p>F222 and DPDA oxidized F222 were precipitated with 9 volumes of acetone at -20° C. For 100 ug of protein, the acetone mixture was vortexed and centrifuged in a Fischer tabletop centrifuge for 10 minutes. The pellets were washed once with 0.5 ml acetone and then dried. The sample was carefully resuspended at 10 mg/ml in 8M urea in 88% formic acid and allowed to sit for 5 minutes. An equal volume of 200 mg/ml CNBr in 88% formic acid was added (5 mg/ml protein) and the samples incubated for 2 hours at room temperature in the dark. Prior to gel electrophoresis, the samples were lyophilized for 3-4 hours in a Spin Vac (Savant Instruments) and the pellets were resuspended at 2-5 mg/ml in sample buffer (1% pyridine, 5% NaDodSO<sub>4</sub>, 5% glycerol and bromophenol blue) and disassociated at 95° C. for 3 minutes.</p>
    <p>The samples were electrophoresed on discontinuous polyacrylamide gels as described by Kyte and Rodriguez (Kyte, J., et al. (1983) Anal. Bioch. 133, 515-522).</p>
    <p>The gels were stained using the Pharmacia silver staining technique (Sammons, D. W., et al. (1981) Electrophoresis 2 135-141).</p>
    <p>The results of this experiment are shown in FIG. 8. As can be seen, F222 treated with CNBr only gives nine resolved bands on SPG. However, when F222 is also treated with DPDA prior to cleavage, bands X, 7 and 9 disappear whereas bands 5 and 6 are greatly increased in intensity.</p>
    <p>In order to determine which of the methionines were effected, each of the CNBr peptides was isolated by reversed phase HPLC and further characterized. The buffer system in both Solvent A (aqueous) and Solvent B (organic) for all HPLC separations was 0.05% TEA-TFA. Solutions were prepared by adding equal volumes of neat triethylamine and neat trifloroacetic acid to the solvent. Programs for the gradients were generated on a Waters Systems Controller. In all Cases unless noted, solvent A consisted of 0.05% TEA-TFA in H2O, solvent B was 0.05% TEA-TFA in 1-propanol, and the flow rate was 0.5 ml/minute.</p>
    <p>For HPLC analysis, two injections of 1 mg enzyme digest were used. Three samples were acetone precipitated, washed and dried as above. The dried 1 mg samples were resuspended at 10 mg/ml in 8M urea, 88% formic acid; an equal volume of 200 mg/ml CNBr in 88% formic acid was added (5 mg/ml protein). After incubation for 2 hours in the dark at room temperature, the samples were desalted on a 0.8 cm×7 cm column of Tris Acryl GF05 coarse resin (IBF, Paris, France) equilibrated with 40% solvent B, 60% solvent A. 200 ul samples were applied at a flow rate of 1 ml a minute and 1.0-1.2 ml collected by monitoring the absorbance at 280 nm. Prior to injection on the HPLC, each desalted sample was diluted with 3 volumes of solvent A. The samples were injected at 1.0 ml/min (2 minutes) and the flow then adjusted to 0.5 ml/min (100% A). After 2 minutes, a linear gradient to 60% B at 1.0% B/min was initiated. From each 1 mg run, the pooled peaks were sampled (50 ul) and analyzed by gel electrophoresis as described above.</p>
    <p>Each polypeptide isolated by reversed phase HPLC was further analyzed for homogeneity by SPG. The position of each peptide on the known gene sequence (Wells, J. A., et al. (1983) Nucleic Acids Res. 11 7911-7924) was obtained through a combination of amino acid compositional analysis and, where needed, amino terminal sequencing.</p>
    <p>Prior to such analysis the following peptides were to rechromatographed.</p>
    <p>1. CNBr peptides from F222 not treated with DPDA:</p>
    <p>Peptide 5 was subjected to two additional reversed phase separations. The 10 cm C4 column was equilibrated to 80% A/20% B and the pooled sample applied and washed for 2 minutes. Next an 0.5% ml B/min gradient was initiated. Fractions from this separation were again rerun, this time on the 25 cm C4 column, and employing 0.05% TEA-TFA in acetonitrile/1-propanol (1:1) for solvent B. The gradient was identical to the one just described.</p>
    <p>Peptide "X" was subjected to one additional separation after the initial chromatography. The sample was applied and washed for 2 minutes at 0.5 ml/min (100% A), and a 0.5% ml B/min gradient was initiated.</p>
    <p>Peptides 7 and 9 were rechromatographed in a similar manner to the first rerun of peptide 5.</p>
    <p>Peptide 8 was purified to homogeneity after the initial separation.</p>
    <p>2. CNBr Peptides from DPDA Oxidized F222:</p>
    <p>Peptides 5 and 6 from a CNBr digest of the oxidized F222 were purified in the same manner as peptide 5 from the untreated enzyme.</p>
    <p>Amino acid compositional analysis was obtained as follows. Samples (˜nM each amino acid) were dried, hydrolyzed in vacuo with 100 ul 6N HCl at 106° C. for 24 hours and then dried in a Speed Vac. The samples were analyzed on a Beckmann 6300 AA analyzer employing ninhydrin detection.</p>
    <p>Amino terminal sequence data was obtained as previously described (Rodriguez, H., et al. (1984) Anal. Biochem. 134, 538-547).</p>
    <p>The results are shown in Table VII and FIG. 9.</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">              TABLE VII______________________________________Amino and COOH terminii of CNBr fragmentsTerminus and MethodFragment    amino, method                   COOH, method______________________________________X            1, sequence                    50, composition9            51, sequence                   119, composition7           125, sequence                   199, composition8           200, sequence                   275, composition 5ox         1, sequence                   119, composition 6ox        120, composition                   199, composition______________________________________</pre>
    
    <p>Peptides 5ox and 6ox refer to peptides 5 and 6 isolated from CNBr digests of the oxidized protein where their respective levels are enhanced.</p>
    <p>From the data in Table VII and the comparison of SPG tracks for the oxidized and native protein digests in FIG. 8, it is apparent that (1) Met50 is oxidized leading to the loss of peptides X and 9 and the appearance of 5; and (2) Met124 is also oxidized leading to the loss of peptide 7 and the accumulation of peptide 6. Thus oxidation of B. amyloliquifaciens subtilisin with the peracid, diperdocecanoic acid leads to the specific oxidation of methionines 50 and 124.</p>
    <heading>EXAMPLE 2</heading> <heading>Substitution at Met50 and Met124 in Subtilisin Met222Q</heading> <p>The choice of amino acid for substitution at Met50 was based on the available sequence data for subtilisins from B. licheniformis (Smith, E. C., et al. (1968) J. Biol. Chem. 243, 2184-2191), B.DY (Nedkov, P., et al. (1983) Hoppe Sayler's Z. Physiol. Chem. 364 1537-1540), B. amylosacchariticus (Markland, F. S., et al. (1967) J. Biol. Chem. 242 5198-5211) and B. subtilis (Stahl, M. L., et al. (1984) J. Bacteriol. 158, 411-418). In all cases, position 50 is a phenylalanine. See FIGS. 5A-1, 5A-2, 5B-1, 5B-2 and 5C. Therefore; Phe50 was chosen for construction.</p>
    <p>At position 124, all known subtilisins possess a methionine. See FIGS. 5A-1, 5A-2, 5B-1, 5B-2 and 5C. Molecular modelling of the x-ray derived protein structure was therefore required to determine the most probable candidates for substitution. From all 19 candidates, isoleucine and leucine were chosen as the best residues to employ. In order to test whether or not modification at one site but not both was sufficient to increase oxidative stability, all possible combinations were built on the Q222 backbone (F50/Q222, I124/Q222, F50/I124/Q222).</p>
    <p>A. Construction of Mutations Between Codons 45 and 50</p>
    <p>All manipulations for cassette mutagenesis were carried out on pS4.5 using methods disclosed in EPO Publication No. 0130756 and Wells, J. A., et al, (1985) Gene 34, 315-323. The pΔ50 in FIG. 10, line 4, mutations was produced using the mutagenesis primer shown in FIG. 10, line 6, and employed an approach designated as restriction-purification which is described below. Wells, J. A., et al. (1986) Phil. Trans. R. Soc. Lond. A (in press). Briefly, a M13 template containing the subtilisin gene, M13mpll-SUBT was used for heteroduplex synthesis (Adelman, et al (1983), DNA 2, 183-193). Following transfection of JM101 (ATCC 33876), the 1.5 kb EcoRI-BamHI fragment containing the subtilisin gene was subcloned from M13mpll SUBT rf into a recipient vector fragment of pBS42 the construction of which is described in EPO Publication No. 0130756. To enrich for the mutant sequence (pΔ50, line 4), the resulting plasmid pool was digested with KpnI, and linear molecules were purified by polyacrylamide gel electrophoresis. Linear molecules were ligated back to a circular form, and transformed into E. coli MM294 cells (ATCC 31446). Isolated plasmids were screened by restriction analysis for the KpnI site. KpnI<sup>+</sup>  plasmids were sequenced and confirmed the pΔ50 sequence. Asterisks in FIG. 11 indicate the bases that are mutated from the wid type sequence (line 4). pΔ50 (line 4) was cut with StuI and EcoRI and the 0.5 Kb fragment containing the 5' half of the subtilisin gene was purified (fragment 1). pΔ50 (line 4) was digested with KpnI and EcoRI and the 4.0 Kb fragment containing the 3' half of the subtilisin gone and vector sequences was purified (fragment 2). Fragments 1 and 2 (line 5), and duplex DNA cassettes coding for mutations desired (shaded sequence, line 6) were mixed in a molar ratio of 1:1:10, respectively. For the particular construction of this example the DNA cassette contained the triplet TTT for codon 50 which encodes Phe. This plasmid was designated pF-50. The mutant subtilisin was designated F-50.</p>
    <p>B. Construction of Mutation Between Codons 122 and 127</p>
    <p>The procedure of Example 2A was followed in substantial detail except that the mutagenesis primer of FIG. 11, line 7 was used and restriction-purification for the EcoRV site in pΔ124 was used. In addition, the DNA cassette (shaded sequence, FIG. 11, line 6) contained the triplet ATT for codon 124 which encodes Ile and CTT for Leu. Those plasmids which contained the substitution of Ile for Met124 were designeated pI124. The Mutant subtilisin was designated I124.</p>
    <p>C. Construction of Various F50/I124/Q222 Multiple Mutants</p>
    <p>The triple mutant, F50/I124/Q222, was constructed from a three-way ligation in which each fragment contained one of the three mutations. The single mutant Q222 (pQ222) was prepared by cassette mutagenesis as described in EPO Publication No. 0130756. The F50 mutation was contained on a 2.2 kb AvaII to PvuII fragment from pF50; the I124 mutation was contained on a 260 bp PvuII to AvaII fragment from pI124; and the Q222 mutation was contained on 2.7 kb AvaII to AvaII fragment from pQ222. The three fragments were ligated together and transformed into E. coli MM294 cells. Restriction analysis of plasmids from isolated transformants confirmed the construction. To analyze the final construction it was convenient that the AvaII site at position 798 in the wild-type subtilisin gene was eliminated by the I124 construction.</p>
    <p>The F50/Q222 and I124/Q222 mutants were constructed in a similar manner except that the appropriate fragment from pS4.5 was used for the final construction.</p>
    <p>D. Oxidative Stability of Q222 Mutants</p>
    <p>The above mutants were analyzed for stability to peracid oxidation. As shown in FIG. 12, upon incubation with diperdodecanoic acid (protein 2 mg/mL, oxidant 75 ppm 0!), both the I124/Q222 and the F50/I124/Q222 are completely stable whereas the F50/Q222 and the Q222 are inactivated. This indicates that conversion of M124 to I124 in subtilisin Q222 is sufficient to confer resistance to organic peracid oxidants.</p>
    <heading>EXAMPLE 3</heading> <heading>Subtilisin Mutants Having Altered Substrate Specificity-Hydrophobic Substitutions at Residues 166</heading> <p>Subtilisin contains an extended binding cleft which is hydrophobic in character. A conserved glycine at residue 166 was replaced with twelve non-ionic amino acids which can project their side-chains into the S-1 subsite. These mutants were constructed to determine the effect of changes in size and hydrophobicity on the binding of various substrates.</p>
    <p>A. Kinetics for Hydrolysis of Substrates Having Altered P-1 Amino Acids by Subtilisin BPN' from B. Amyloliquefaciens</p>
    <p>Wild-type subtilisin was purified from B. subtilis culture supernatants expressing the B. amyloliquefaciens subtilisin gene (Wells, J. A., et al. (1983) Nucleic Acids Res. 11, 7911-7925) as previously described (Estell, D. A., et al. (1985) J. Biol. Chem. 260, 6518-6521). Details of the synthesis of tetrapeptide substrates having the form succinyl-L-AlaL-AlaL-ProL- X!-p-nitroanilide (where X is the P1 amino acid) are described by DelMar, E. G., et al. (1979) Anal. Biochem. 99, 316-320. Kinetic parameters, Km(M) and kcat(s<sup>-1</sup>) were measured using a modified progress curve analysis (Estell, D. A., et al. (1985) J. Biol. Chem. 260, 6518-6521). Briefly, plots of rate versus product concentration were fit to the differential form of the rate equation using a non-linear regression algorithm. Errors in kcat and Km for all values reported are less than five percent. The various substrates in Table VIII are ranged in order of decreasing hydrophobicity. Nozaki, Y. (1971), J. Biol. Chem. 246, 2211-2217; Tanford C. (1978) Science 200, 1012).</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">              TABLE VIII______________________________________P1 substrate                    kcat/KmAmino Acid  kcat(S<sup>-1</sup>)                1/Km(M<sup>-1</sup>)                           (s-<sup>1</sup> M-1)______________________________________Phe         50       7,100      360,000Tyr         28       40,000     1,100,000Leu         24       3,100      75,000Met         13       9,400      120,000His         7.9      1,600      13,000Ala         1.9      5,500      11,000Gly         0.003    8,300      21Gln         3.2      2,200      7,100Ser         2.8      1,500      4,200Glu         0.54     32         16______________________________________</pre>
    
    <p>The ratio of kcat/Km (also referred to as catalytic efficienty) is the apparent second order rate constant for the conversion of free enzyme plus substrate (E+S) to enzyme plus products (E+P) (Jencks, W. P., Catalysis in Chemistry and Enzymology (McGraw-Hill, 1969) pp. 321-436; Fersht, A., Enzyme Structure and Mechanism (Freeman, San Francisco, 1977) pp. 226-287). The log (kcat/Km) is proportional to transition state binding energy, ΔG<sub>T</sub>.sup.≠. A plot of the log kcat/Km versus the hydrophobicity of the P1 side-chain (FIG. 14) shows a strong correlation (r=0.98), with the exception of the glycine substrate which shows evidence for non-productive binding. These data show that relative differences between transition-state binding energies can be accounted for by differences in P-1 side-chain hydrophobicity. When the transition-state binding energies are calculated for these substrates and plotted versus their respective side-chain bydrophobicities, the line slope is 1.2 (not shown). A slope greater than unity, as is also the case for chymotrypsin (Fersht, A., Enzyme Structure and Mechanism (Freeman, San Francisco, 1977) pp. 226-287; Harper, J. W., et al. (1984) Biochemistry, 23, 2995-3002), suggests that the P1 binding cleft is more hydrophobic than ethanol or dioxane solvents that were used to empirically determine the hydrophobicity of amino acids (Nozaki, Y., et al. J. Biol. Chem. (1971) 246, 2211-2217; Tanford, C. (1978) Science 200, 1012).</p>
    <p>For amide hydrolysis by subtilisin BPN' kcat can be interpreted as the acylation rate constant and Km as the dissociation constant, for the Michaelis complex (E.S), Ks. Gutfreund, H., et al (1956) Biochem. J. 63, 656. The fact that the log kcat, as well as log 1/Km, correlates with substrate hydrophobicity is consistent with proposals (Robertus, J. D., et al. (1972) Biochemistry 11, 2439-2449; Robertus, J. D., et al. (1972) Biochemistry 11, 4293-4303) that during the acylation step the P-1 side-chain moves deeper into the hydrophobic cleft as the substrate advances from the Michaelis complex (E-S) to the tetrahedral transition-state complex (E.S.sup.≠). However, these data can also be interpreted as the hydrophobicity of the P1 side-chain effecting the orientation, and thus the susceptibility of the scissile peptide bond to nucleophilic attack by the hydroxyl group of the catalytic Ser221.</p>
    <p>The dependence of kcat/Km on P-1 side chain hydrophobicity suggested that the kcat/Km for hydrophobic substrates may be increased by increasing the hydrophobicity of the S-1 binding subsite. To test this hypothesis, hydrophobic amino acid substitutions of Gly166 were produced.</p>
    <p>Since hydrophobicity of aliphatic side-chains is directly proportional to side-chain surface area (Rose, G. D., et al. (1985) Science 229, 834-838; Reynolds, J. A., et al. (1974) Proc. Natl. Acad. Sci. USA 71, 2825-2927), increasing the hydrophobicity in the S-1 subsite may also sterically hinder binding of larger substrates. Because of difficulties in predicting the relative importance of these two opposing effects, we elected to generate twelve non-charged mutations at position 166 to determine the resulting specificities against non-charged substrates of varied size and hydrophobicity.</p>
    <p>B. Cassette Mutagenesis of the P1 Binding Cleft</p>
    <p>The preparation of mutant subtilisims containing the substitution of the hydrophobic amino acids Ala, Val and Phe into residue 166. has been described in EPO Publication No. 0130756. The same method was used to produce the remaining hydrophobic mutants at residue 166. In applying this method, two unique and silent restriction sites were introduced in the subtilisin genes to closely flank the target codon 166. As can be seen in FIG. 13, the wild type sequence (line 1) was altered by site-directed mutagenesis in M13 using the indicated 37mer mutagenesis primer, to introduce a 13 bp delection (dashedline) and unique SacI and XmaI sites (underlined sequences).that closely flank codon 166. The subtilisin gene fragment was subcloned back into the E. coli-B. subtilis shuttle plasmid, pBS42, giving the plasmid pΔ166 (FIG. 13, line 2). pΔ166 was cut open with SacI and XmaI, and gapped linear molecules were purified (FIG. 13, line 3). Pools of synthetic oligonucleotides containing the mutation of interest were annealed to give duplex DNA cassettes that were ligated into gapped pΔ166 (underlined and overlined sequences in FIG. 13, line 4). This construction restored the coding sequence except over position 166 (NNN; line 4). Mutant sequences were confirmed by dideoxy sequencing. Asterisks denote sequence changes from the wild type sequence. Plasmids containing each mutant B. amyloliquefaciens subtilisin gene were expressed at roughly equivalent levels in a protease deficient strain of B. subtilis, BG2036 as previously described. EPO Publication No. 0130756; Yang, M., et al. (1984) J. Bacteriol. 160, 15-21; Estell, D. A., et al (1985) J. Biol. Chem. 260, 6518-6521.</p>
    <p>C. Narrowing Substrate Specificity by Steric Hindrance</p>
    <p>To probe the change in substrate specificity caused by steric alterations in the. S-1 subsite, position 166 mutants were kinetically analyzed versus P1 substrates of increasing size (i.e., Ala, Met, Phe and Tyr). Ratios of kcat/Km are presented in log form in FIGS. 15A and 15B to allow direct comparisons of transition-state binding energies between various enzyme-substrate pairs.</p>
    <p>According to transition state theory, the free enery difference between the free enzyme plus substrate (E+S) and the transition state complex (E.S.sup.≠) can be calculated from equation (1),</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="equation">.sup.Δ G<sub>T</sub>.sup.≠ =-RT ln kcat/Km+RT ln kT/h(1)</pre>
    
    <p>in which kcat is the turnover number, Km is the Michaelis constant, R is the gas constant, T is the temperature, k is Boltzmann's constant, and h is Planck's constant. Specificity differences are ezpressed quantitatively as differences between transition state binding energies (i.e ΔΔG<sub>t</sub>.sup.≠) and can be calculated from equation (2).</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="equation">.sup.ΔΔ G<sub>T</sub>.sup.≠ =-RT ln (kcat/Km)<sub>A</sub> /(kcat/Km)<sub>B</sub>                                          (2)</pre>
    
    <p>A and B represent either two different substrates assayed againt the same enzyme, or two mutant enzymes assayed against the same substrate.</p>
    <p>As can be seen from FIG. 15A, as the size of the side-chain at position 166 increases the substrate preference shifts from large to small P-1 side-chains. Enlarging the side-chain at position 166 causes kcat/Km to decrease in proportion to the size of the P-1 substrate side-chain (e.g., from Gly166 (wild-type) through W166, the kcat/Km for the Tyr substrate is decreased most followed in order by the Phe, Met and Ala P-1 substrates).</p>
    <p>Specific steric changes in the position 166 side-chain, such as he presence of a β-hydroxyl group, β- or γ-aliphatic branching, cause large decreases in kcat/Km for larger P1 substrates. Introducing a β-hydroxyl group in going from A166 (FIGS. 15A and 15B) to S166 (FIG. 15B), causes an 8 fold and 4 fold reduction in kcat/Km for Phe and Tyr substrates, respectively, while the values for Ala and Met substrates are unchanged. -Producing a E-branched structure, in going from S166 to T166, results in a drop of 14 and 4 fold in kcat/Km for Phe and Tyr, respectively. These differences are slightly magnified for V166 which is slightly larger and isosteric with T166. Enlarging the β-branched substituents from V166 to I166 causes a lowering of kcat/Km between two and six fold toward Met, Phe and Tyr substrates. Inserting a 7-branched structure, by replacing M166 (FIG. 15A) with L166 (FIG. 15B), produces a 5 fold and 18 fold decrease in kcat/Km for Phe and Tyr substrates, respectively. Aliphatic γ-branched appears to induce less steric hindrance toward the Phe P-1 substrate than β-branching, as evidenced by the 100 fold decrease in kcat/Km for the Phe substrate in going from L166 to I166.</p>
    <p>Reductions in kcat/Km resulting from increases in side chain size in the S-1 subsite, or Specific structural features such as β- and γ-branching, are quantitatively illustrated in FIGS. 16A, 16B, 16C and 16D. The kcat/Km values for the position 166 mutants determined for the Ala, Met, Phe, and Tyr P-1 substrates (top panel through bottom panel, respectively), are plotted versus the position 166 side-chain volumes (Chothla, C. (1984) Ann. Rev. Biochem. 53, 537-572). Catalytic efficiency for the Ala substrate reaches a maximum for I166, and for the Met substrate it reaches a maximum between V166 and L166. The Phe substrate shows a broad kcat/Km peak but is optimal with A166. Here, the β-branched position 166 substitutions form a line that is parallel to, but roughly 50 fold lower in kcat/Km than side-chains of similar size  i.e., C166 versus T166, L166 versus I166!. The Tyr substrate is most efficiently utilized by wild type enzyme (Gly166), and there is a steady decrease as one proceeds to large position 166 side-chains. The β-branched and γ-branched substitutions form a parallel line below the other non-charged substitutions of similar molecular volume.</p>
    <p>The optimal substitution at position 166 decreases in volume with increasing volume of the P1 substrate  i.e., I166/Ala substrate, L166/Met substrate, A166/Phe substrate, Gly166/Tyr substrate!. The combined volumes for these optimal pairs may approximate the volume for productive binding in the S-1 subsite, For the optimal pairs, Gly166/Tyr substrate, A166/Phe substrate, L166/Met substrate, V166/Met substrate, and I166/Ala substrate, the combined volumes are 266,295,313,339 and 261 A<sup>3</sup>, respectively. Subtracting the volume of the peptide backbone from each pair (i.e., two times the volume of glycine), an average side-chain volume of 160±32A<sup>3</sup> for productive binding can be calculated.</p>
    <p>The effect of volume, in excess to the productive binding volume, on the drop in transition-state binding energy can be estimated from the Tyr substrate curve (bottom panel, FIGS. 16A, 16B, 16C and 16D), because these data, and modeling studies (FIG. 2), suggest that any substitution beyond glycine causes steric repulsion. A best-fit line drawn to all the data (r=0.87) gives a slope indicating a loss of roughly 3 kcal/mol in transition state binding energy per 100 A<sup>3</sup> of excess volume. (100 A<sup>3</sup> is approximately the size of a leucyl side-chain.)</p>
    <p>D. Enhanced Catalytic Efficiency Correlates with Increasing Hydrophobicity of the Position 166 Substitution</p>
    <p>Substantial increases in kcat/Km occur with enlargement of the position 166 side-chain, except for the Tyr P-1 substrate (FIGS. 16A, 16B, 16C and 16D). For example, kcat/Km increases in progressing from Gly166 to I166 for the Ala substrate (net of ten-fold), from Gly166 to L166 for the Met substrate (net of ten-fold) and from Gly166 to A166 for the Phe substrate (net of two-fold). The increases in kcat/Km cannot be entirely explained by the attractive terms in the van der Waals potential energy function because of their strong distance dependence (1/r<sup>6</sup>) and because of the weak nature of these attractive forces (Jencks, W. P., Catalysis in Chemistry and Enzymology (McGraw-Hill, 1969) pp. 321-436; Fersht, A., Enzyme Structure and Mechanism (Freeman, San Francisco, 1977) pp. 226-287; Levitt, M. (1976) J. Mol. Biol. 104, 59-107). For example, Levitt (Levitt, M. (1976) J. Mol. Biol. 104, 59-107) has calculated that the van der Waals attraction between two methionyl residues would produce a maximal interaction energy of roughly -0.2 kcal/mol. This energy would translate to only 1.4 fold increase in kcat/Km.</p>
    <p>The increases of catalytic efficiency caused by side-chain substitutions at position 166 are better accounted for by increases in the hydrophobicity of the S-1 subsite. The increase kcat/Km observed for the Ala and Met substrates with increasing position 166 side-chain size would be expected, because hydrophobicity is roughly proportional to side-chain surface area (Rose, G. D., et al. (1985) Science 229, 834-838; Reynolds, J. A., et al. (1974) Proc. Natl. Acad. Sci. USA 51, 2825-2927).</p>
    <p>Another example that can be interpreted as a hydrophobic effect is seen when comparing kcat/Km for isosteric substitutions that differ in hydrophobicity such as S166 and C166 (FIGS. 16A, 16B, 16C and 16D). Cysteine is considerably more hydrophobic than serine (-1.0 versus +0.3 kcal/mol) (Nozaki, Y., et al. (1971) J. Biol. Chem. 246, 2211-2217; Tanford, C. (1978) Science 200, 1012). The difference in hydrophobicity correlates with the observation that C166 becomes more efficient relative to Ser166 as the hydrophobicity of the substrates increases (i.e., Ala&lt;Met&lt;Tye&lt;Phe). Steric hindrance cannot explain these differences because serine is considerably smaller than cysteine (99 versus 118A<sup>3</sup>). Paul, I. C., Chemistry of the --SH Group (ed. S. Patai, Wiley Interscience, New York, 1974) pp. 111-149.</p>
    <p>E. Production of an Elastase-Like Specificity in Subtilisin</p>
    <p>The I166 mutation illustrates particularly well that large changes in specificity can be produced by altering the structure and hydrophobicity of the S-1 subsite by a Single mutation (FIG. 17). Progressing through the small hydrophobic substrates, a maximal specificity improvement over wild type occurs for the Val substrate (16 fold in kcat/Km). As the substrate side chain size increases, these enhancements shrink to near unity (i.e., Leu and His substrates). The I166 enzyme becomes poorer against larger aromatic substrates of increasing size (e.g., I166 is over 1,000 fold worse against the Tyr substrate than is Gly166). We interpret the increase in catalytic efficiency toward the small hydrophobic substrates for I166 compared to Gly166 to the greater hydrophobicity of isoluecine (i.e., -1.8 kcal/mol versus 0). Nozaki, Y., et al. (1971) J. Biol. Chem. 246, 2211-2217; Tanford, C. (1978) Science 200, 1012. The decrease in catalytic efficiency toward the very large substrates for I166 versus Gly166 is attributed to steric repulsion.</p>
    <p>The specificity differences between Gly166 and I166 are similar to the specificity differences between chymotrypsin and the evolutionary relative, elastase (Harper, J. W., et al (1984) Biochemistry 23, 2995-3002). In elastase, the bulky amino acids, Thr and Val, block access to the P-1 binding site for large hydrophobic substrates that are preferred by chymotrypsin. In addition, the catalytic efficiencies toward small hydrophobic substrates are greater for elastase than for chymotrypsin as we obeseve for I166 versus Gly166 in subtilisin.</p>
    <heading>EXAMPLE 4</heading> <heading>Substitution of Ionic Amino Acids for Gly166</heading> <p>The construction of subtilisin mutants containing the substitution of the ionic amino acids Asp, Ash, Gln, Lys and Ang are disclosed in EPO Publication No. 0130756. In addition, a limited analysis of substrate specificity was presented therein. The present example describes the construction of the mutant subtilisin containing Glu at position 166 (E166) and presents some of the specificity data on these mutants. Further data on position 166 and 156 single and double mutants will be presented infra.</p>
    <p>pΔ166, described in Example 3, was digested with SacI and XmaI. The double strand DNA cassette (underlined and overlined) of line 4 in FIG. 13 contained the triplet GAA for the codon 166 to encode the replacement of Glu for Gly166. This mutant plasmid designated pQ166 was propagated in BG2036 as described. This mutant subtilisin, together with the other mutants containing ionic substituent amino acids at residue 166, were isolated as described and further analyzed for variations in substrate specificity.</p>
    <p>Each of these mutants was analyzed with the tetrapeptide substrates, succinyl-L-AlaL-AlaProL-X-p-nitroanilide, where X was Phe, Ala and Glu.</p>
    <p>The results of this analysis are shown in Table IX.</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">              TABLE IX______________________________________     P-1 Substrate     (kcat/Km × 10<sup>=4</sup>)Position 166       Phe          Ala    Glu______________________________________Gly (wild type)       36.0         1.4    0.002Asp (D)     0.5          0.4    &lt;0.001Glu (E)     3.5          0.4    &lt;0.001Asn (N)     18.0         1.2    0.004Gln (Q)     57.0         2.6    0.002Lys (K)     52.0         2.8    1.2Arg (R)     42.0         5.0    0.08______________________________________</pre>
    
    <p>These results indicate that charged amino acid substitutions at Gly166 have improved catalytic efficiencies (kcat/Km) for oppositely charged P-1 substrates (as much as 500 fold) and poorer catalytic efficiency for like charged P-1 substrates.</p>
    <heading>EXAMPLE 5</heading> <heading>Substitution of Glycine at Position 169</heading> <p>The substitution of Gly169 in B. amyloliquefaciens subtilisin with Ala and Ser is described in EPO Publication No. 0130756. The same method was used to make the remaining 17 mutants containing all other substituent amino acids for position 169.</p>
    <p>The construction protocol is summarized in FIG. 18. The overscored and underscored double stranded DNA cassettes used contained the following triplet encoding the substitution of the indicated amino acid at residue 169.</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">______________________________________    GCT          A    TGT          C    GAT          D    GAA          E    TTC          F    GGC          G    CAC          H    ATC          I    AAA          K    CTT          L    ATG          M    AAC          N    CCT          P    CAA          Q    AGA          R    AGC          S    ACA          T    GTT          V    TGG          W    TAC          Y______________________________________</pre>
    
    <p>Each of the plasmids containing a substituted Gly169 was designated pX169, where X represents the substituent amino acid. The mutant subtilisins were simialrly designated.</p>
    <p>Two of the above mutant subtilisins, A169 and S169, were analyzed for substrate specificity against synthetic substrates containing Phe, Leu, Ala and Arg in the P-1 position. The following results are shown in Table X.</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">              TABLE X______________________________________Effect of Serine and Alanine Mutationsat Position 169 on P-1. Substrate Specificity      P-1 Substrate (kcat/Km × 10<sup>-4</sup>)Position 169 Phe     Leu      Ala   Arg______________________________________Gly (wild type)        40      10       1     0.4A169         120     20       1     0.9S169         50      10       1     0.6______________________________________</pre>
    
    <p>These results indicate that substitutions of Ala and Ser at Gly169 have remarkably similar catalytic efficiencies against a range of P-1 substrates compared to their position 166 counterparts. This is probably because position 169 is at the bottom of the P-1 specificity subsite.</p>
    <heading>EXAMPLE 6</heading> <heading>Substitution at Position 104</heading> <p>Tyr104 has been substituted with Ala, His, Leu, Met and Ser. The method used was a modification of the site directed mutagenesis method. According to the protocol of FIG. 19, a primer (shaded in line 4) introduced a unique HindIII site and a frame shift mutation at codon 104. Restriction-purification for the unique HindIII site facilitated the isolation of the mutant sequence (line 4). Restriction-selection against this HindIII site using pimers in line 5 was used to obtain position 104 mutants.</p>
    <p>The following triplets were used in the primers of FIG. 19, line 5 for the 104 codon which substituted the following amino acids.</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">______________________________________GCT          Ala       TTC        PheATG          Met       CCT        ProCTT          Leu       ACA        ThrAGC          Ser       TGG        TrpCAC          His       TAC        TyrCAA          Gln       GTT        ValGAA          Glu       AGA        ArgGGC          Gly       AAC        AsnATC          Ile       GAT        AspAAA          Lys       TGT        Cys______________________________________</pre>
    
    <p>The following substrates were used to analyze the substrate specificity of these mutants to give the indicated results in Table XI.</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">              TABLE XI______________________________________  kcat      Km          Kcat/KmSubstrate    WT      H104    WT    H104  WT    H104______________________________________sAAPFpNA 50.0    22.0    1.4e-4                          7.le-4                                3.6e5 3.1e4sAAPApNA 3.2     2.0     2.3e-4                          1.9e-3                                1.4e4   1e3sFAPFpNA 26.0    38.0    1.8e-4                          4.le-4                                1.5e5 9.1e4sFAPApNA 0.32    2.4     7.3e-5                          1.5e-4                                4.4e3 1.6e4______________________________________</pre>
    
    <p>From these data it is clear that the substitution of His for Tyr at position 104 produces an enzyme which is more efficient (higher kcat/Km) when Phe is at the P-4 substrate position than when Ala is at the P-4 substrate position.</p>
    <heading>EXAMPLE 7</heading> <heading>Substitution of Ala152</heading> <p>Ala152 has been substituted by Gly and Ser to determine the effect of such substitutions on substrate specificity.</p>
    <p>The wild type DNA sequence was mutated by the V152/P153 primer (FIG. 20, line 4) using the above restriction-purification approach for the new KpnI site. Other mutant primers (shaded sequences FIG. 20; S152, line 5 and G152, line 6) mutated the new KpnI site away and such mutants were isolated using the restriction-selection procedure as described above for loss of the KpnI site.</p>
    <p>The results of these substitutions for the above synthetic substrates containing the P-1 amino acids Phe, Leu and Ala are shown in Table XII.</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">              TABLE XII______________________________________     P-1 Substrate     (kcat/Km × 10<sup>-4</sup>)Position 152       Phe          Leu    Ala______________________________________Gly (G)     0.2          0.4    &lt;0.04Ala (wild type)       40.0         10.0   1.0Ser (S)     1.0          0.5    0.2______________________________________</pre>
    
    <p>These results indicate that, in contrast to positions 166 and 169, replacement of Ala152 with Ser or Gly causes a dramatic reduction in catalytic efficiencies across all substrates tested. This suggests Ala152, at the top of the S-1 subsite, may be the optimal amino acid because Ser and Gly are homologous Ala substitutes.</p>
    <heading>EXAMPLE 8</heading> <heading>Substitution at Position 156</heading> <p>Mutants containing the substitution of Ser and Gln for Glu156 have been constructed according to the overall method depicted in FIG. 21. This method was designed to facilitate the construciton of multiple mutants at position 156 and 166 as will be described hereinafter. However, by regenerating the wild type Gly166, single mutations at Glu156 were obtained.</p>
    <p>The plasmid pΔ166 is already depicted in line 2 of FIG. 13. The synthetic oligonucleotides at the top right of FIG. 21 represent the same DNA cassettes depicted in line 4 of FIG. 13. The plasmid p166 in FIG. 21 thus represents the mutant plasmids of Examples 3 and 4. In this particular example, p166 contains the wild type Gly166.</p>
    <p>Construction of position 156 single mutants were prepared by ligation of the three fragments (1-3) indicated at the bottom of FIG. 21. Fragment 3, containing the carboxy-terminal portion of the subtilisin gene including the wild type position 166 codon, was isolated as a 610 bp SacI-BamHI fragment. Fragment 1 contained the vector sequences, as well as the amino-terminal sequences Of the subtilisin gene through codon 151. To produce fragment 1, a unique KpnI site at codon 152 was introduced into the wild type subtilisin sequence from pS4.1. Site-directed in M13 employed a primer having the sequence 5'-TA-GTC-GTT-GCG-GTA-CCC-GGT-AAC-GAA-3' to produce the mutation. Enrichment for the mutant sequence was accomplished by restriction with KpnI, purification and self ligation. The mutant sequence containing the KpnI site was confirmed by direct plasmid sequencing to give pV-152. pV-152 (˜1 μg) was digested with KpnI and treated with 2 units of DNA polymerase I large fragment (Klenow fragment from Boeringer-Mannheim) plus 50 μM deoxynucleotide triphosphates at 37° C. for 30 min. This created a blunt end that terminated with codon 151. The DNA was extracted with 1:1 volumes phenol and CHCl<sub>3</sub> and DNA in the aqueous phase was precipitated by addition of 0.1 volumes 5M ammonium acetate and two volumes ethanol. After centrifugation and washing the DNA pellet with 70% ethanol, the DNA was lyophilized. DNA was digested with BamHI and the 4.6 kb piece (fragment 1) was purified by acrylamide gel electrophoresis followed by electroelution. Fragment 2 was a duplex synthetic DNA cassette which when ligated with fragments 1 and 3 properly restored the coding sequence except at codon 156. The top strand was synthesized to contain a glutamine codon, and the complementary bottom strand coded for serine at 156. Ligation of heterophosphorylated cassettes leads to a large and favorable bias for the phosphorylated over the non-phosphorylated oligonucleotide sequence in the final segrated plasmid product. Therefore, to obtain Q156 the top strand was phosphorylated, and annealed to the non-phosphorylated bottom strand prior to ligation. Similarly, to obtain S156 the bottom strand was phosphorylated and annealed to the non-phosphorylated top strand. Mutant sequences were isolated after ligation and transformation, and were confirmed by restriction analysis and DNA sequencing as before. To express variant subtilisins, plasmids were transformed into a subtilisin-neutral protease deletion mutant of B. subtilis, BG2036, as previously described. Cultures were fermented in shake flasks for 24 h at 37° C. in LB media containing 12.5 mg/mL chloraphenicol and subtilisin was purified from culture supernatants as described. Purity of subtilisin was greater than 95% as judged by SDS PAGE.</p>
    <p>These mutant plasmids designated pS156 and pQ156 and mutant subtilisins designated S156 and Q156 were analyzed with the above synthetic substrates where P-1 comprised the amino acids Glu, Gln, Met and Lys. The results of this analyses are presented in Example 9.</p>
    <heading>EXAMPLE 9</heading> <heading>Multiple Mutants with Altered Substrate Specificity--Substitution at Positions 156 and 166</heading> <p>Single substitutions of position 166 are described in Examples 3 and 4. Example 8 describes single substitutions at position 156 as well as the protocol of FIG. 21 whereby various double mutants comprising the substitution of various amino acids at positions 156 and 166 can be made. This example describes the construction and substrate specificity of subtilisin containing substitutions at position 156 and 166 and summarized some of the data for single and double mutants at positions 156 and 166 with various substrates.</p>
    <p>K166 is a common replacement amino acid in the 156/166 mutants described herein. The replacement of Lys for Gly166 was achieved by using the synthetic DNA cassette at the top right of FIG. 21 which contained the triplet AAA for NNN. This produced fragment 2 with Lys substituting for Gly166.</p>
    <p>The 156 substituents were Gln and Ser. The Gln and Ser substitutions at Gly156 are contained within fragment 3 (bottom right FIG. 21).</p>
    <p>The multiple mutants were produced by combining fragments 1, 2 and 3 as described in Example 8. The mutants Q156/K166 and S156/K166 were selectively generated by differential phosphorylation as described. Alternatively, the double 156/166 mutants, c.f. Q156/K166 and S156/K166, were prepared by ligation of the 4.6 kb SacI-BamHI fragment from the relevant p156 plasmid containing the 0.6 kb SacI-BamHI fragment from the relevant p166 plasmid.</p>
    <p>These mutants, the single mutant K166, and the S156 and Q156 mutants of Example 8 were analyzed for substitute specificity against synthetic polypeptides containing Phe or Glu as the P-1 substrate residue. The results are presented in Table XIII.</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">              TABLE XIII______________________________________     Sub-     strate                       kcat/KmEnzymes   P-1                          (mutant)Compared.sup.(b)     Residue kcat   Km     kcat/Km                                  kcat/Km (wt)______________________________________Glu-156/Gly-166     Phe     50.00  1.4 × 10<sup>-4</sup>                           3.6 × 10<sup>5</sup>                                  (1)(WT)      Glu     0.54   3.4 × 10<sup>-2</sup>                           1.6 × 10<sup>1</sup>                                  (1)Lys-166   Phe     20.00  4.0 × 10<sup>-5</sup>                           5.2 × 10<sup>5</sup>                                  1.4     Glu     0.70   5.6 × 10<sup>-5</sup>                           1.2 × 10<sup>4</sup>                                  750Gln-156/Lys-166     Phe     30.00  1.9 × 10<sup>-5</sup>                           1.6 × 10<sup>6</sup>                                  4.4     G1u     1.60   3.1 × 10<sup>-5</sup>                           5.0 × 10<sup>4</sup>                                  3100Ser-156/Lys-166     Phe     30.00  1.8 × 10<sup>-5</sup>                           1.6 × 10<sup>6</sup>                                  4.4     Glu     0.60   3.9 × 10<sup>-5</sup>                           1.6 × 10<sup>4</sup>                                  1000Ser-156   Phe     34.00  4.7 × 10<sup>-5</sup>                           7.3 × 10<sup>5</sup>                                  2.0     Glu     0.40   1.8 × 10<sup>-3</sup>                           1.1 × 10<sup>2</sup>                                  6.9Glu-156   Phe     48.00  4.5 × 10<sup>-5</sup>                           1.1 × 10<sup>6</sup>                                  3.1     Glu     0.90   3.3 × 10<sup>-3</sup>                           2.7 × 10<sup>2</sup>                                  17______________________________________</pre>
    
    <p>As can be seen in Table XIV, either of these single mutations improve enzyme performance upon substrates with glutamate at the P-1 enzyme binding site. When these single mutations were combined, the resulting multiple enzyme mutants are better than either parent. These single or multiple mutations also alter the relative pH activity profiles of the enzymes as shown in FIGS. 23A and 23B.</p>
    <p>To isolate the contribution of electrostatics to substrate specificity from other chemical binding forces, these various single and double mutants were analyzed for their ability to bind and cleave synthetic substrates containing Glu, Gln, Met and Lys as the P-1 substrate amino acid. This permitted comparisons between side-chains that were more sterically similar but differed in charge (e.g., Glu versus Gln, Lys versus Met). Similarly, mutant enzymes were assayed against homologous P-1 substrates that were most sterically similar but differed in charge (Table XIV).</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">                                  TABLE XIV__________________________________________________________________________Kinetics of Position 156/166 SubtilisinsDetermined for Different P1 SubstratesEnzymePosition.sup.(a)   Net  P-1 Substrate 1og kcat/Km (log 1/Km).sup.(c)156   166  Charge.sup.(b)        Glu   Gln   Met   Lys__________________________________________________________________________Glu   Asp  -2   n.d.  3.02 (2.56)                    3.93 (2.74)                          4.23 (3.00)Glu   Glu  -2   n.d.  3.06 (2.91)                    3.86 (3.28)                          4.48 (3.69)Glu   Asn  -1   1.62 (2.22)              3.85 (3.14)                    4.99 (3.85)                          4.15 (2.88)Glu   Gln  -1   1.20 (2.12)              4.36 (3.64)                    5.43 (4.36)                          4.10 (3.15)Gln   Asp  -1   1.30 (1.79)              3.40 (3.08)                    4.94 (3.87)                          4.41 (3.22)Ser   Asp  -1   1.23 (2.13)              3.41 (3.09)                    4.67 (3.68)                          4.24 (3.07)Glu   Met  -1   1.20 (2.30)              3.89 (3.19)                    5.64 (4.83)                          4.70 (3.89)Glu   Ala  -1   n.d.  4.34 (3.55)                    5.65 (4.46)                          4.90 (3.24)Glu   Gly (wt)   -1   1.20 (1.47)              3.85 (3.35)                    5.07 (3.97)                          4.60 (3.13)Gln   Gly  0    2.42 (2.48)              4.53 (3.81)                    5.77 (4.61)                          3.76 (2.82)Ser   Gly  0    2.31 (2.73)              4.09 (3.68)                    5.61 (4.55)                          3.46 (2.74)Gln   Asn  0    2.04 (2.72)              4.51 (3.76)                    5.79 (4.66)                          3.75 (2.74)Ser   Asn  0    1.91 (2.78)              4.57 (3.82)                    5.72 (4.64)                          3.68 (2.80)Glu   Arg  0    2.91 (3.30)              4.26 (3.50)                    5.32 (4.22)                          3.19 (2.80)Glu   Lys  0    4.09 (4.25)              4.70 (3.88)                    6.15 (4.45)                          4.23 (2.93)Gln   Lys  +1   4.70 (4.50)              4.64 (3.68)                    5.97 (4.68)                          3.23 (2.75)Ser   Lys  +1   4.21 (4.40)              4.84 (3.94)                    6.16 (4.90)                          3.73 (2.84)Maximum difference:log kcat/Km (log 1/Km).sup.(d)        3.5 (3.0)              1.8 (1.4)                    2.3 (2.2)                          -1.3 (-1.0)__________________________________________________________________________</pre>
    
    <p>Footnotes to Table XIV:</p>
    
    
    
    
    
    <p>The kcat/Km ratios shown are the second order rate constants for the conversion of substrate to product, and represent the catalytic efficiency of the enzyme. These ratios are presented in logarithmic form to scale the data, and because log kcat/Km is proportional to the lowering of transition-state activation energy (ΔG<sub>T</sub>). Mutations at position 156 and 166 produce changes in catalytic efficiency toward Glu, Gln, Met and Lys P-1 substrates of 3100, 60, 200 and 20 fold, respectively. Making the P-1 binding-site more positively charged  e.g., compare Gln156/Lys166 (Q156/K166) versus Glu156/Met166 (Glu156/M166)! dramatically increased kcat/Km toward the Glu P-1 substrate (up to 3100 fold), and decreased the catalytic efficiency toward the Lys P-1 substrate (up to 10 fold). In addition, the results show that the catalytic efficiency of wild type enzyme can be greatly improved toward any of the four P-1 substrates by mutagenesis of the P-1 binding site.</p>
    <p>The changes in kcat/Km are caused predominantly by changes in 1/Km. Because 1/Km is approximately equal to 1/Ks, the enzyme-substrate association constant, the mutations primarily cause a change in substrate binding. These mutations produce smaller effects on kcat that run parallel to the effects on 1/Km. The changes in kcat suggest either an alteration in binding in the P-1 binding site in going from the Michaelis-complex E-S) to the transition-state complex (E-S≠) as previously proposed (Robertus, J. D., et al. (1972) Biochemistry 11, 2439-2449; Robertus, J. D., et al. (1972) Biochemistry 11, 4293-4303), or change in the position of the scissile peptide bond over the catalytic serine in the E*S complex.</p>
    <p>Changes in substrate preference that arise from changes in the net charge in the P-1 binding site show trands that are best accounted for by electrostatic effects (FIG. 28). As the P-1 binding cleft becomes more positively charged, the average catalytic efficiency increases much more for the Glu P-1 substrate than for its neutral and isosteric P-1 homolog, Gln (FIG. 28A). Furthermore, at the positive extreme both substrates have nearly identical catalytic efficiencies.</p>
    <p>In contrast, as the P-1 site becomes more positively charged the catalytic efficiency toward the Lys P-1 substrate decreases, and diverges sharply from its neutral and isosteric homolog, Met (FIG. 28B). The similar and parallel upward trend seen with increasing positive charge for the Met and Glu P-1 substrates probably results from the fact that all the substrates are succinylated on their amino-terminal end, and thus carry a formal negative charge.</p>
    <p>The trends observed in log kcat/Km are dominated by changes in the Km term (FIGS. 28C and 28D). As the pocket becomes more positively charged, the log 1/Km values converge for Glu and Gln P-1 substrates (FIG. 28C), and diverge for Lys and Met P-1 substrates (FIG. 28D). Although less pronounced effects are seen in log kcat, the effects of P-1 charge on log kcat parallel those seen in log 1/Km and become larger as the P-1 pocket becomes more positively charged. This may result from the fact that the transition-state is a tetrahedral anion, and a net positive charge in the enzyme may serve to provide some added stabilization to the transition-state.</p>
    <p>The effect of the change in P-1 binding-site charge on substrate preference can be estimated from the differences in slopes between the charged and neutral isosteric P-1 substrates (FIG. 28B). The average change in substrate preference (Δlog kcat/Km) between charged and neutral isosteric substrates increases roughly 10-fold as the complementary charge or the enzyme increases (Table XV). When comparing Glu versus Lys, this difference is 100-fold and the change in substrate preference appears predominantly in the Km term.</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">              TABLE XV______________________________________Differential Effect on Binding SiteCharge on log kcat/Km or (log 1/Km)for P-1 Substrates that Differ in Charge .sup.(a)Change in P-1 Binding        Δlog kcat/Km                        (Δlog 1/Km)Site Charge.sup.(b)        GluGln     MetLys   GluLys______________________________________-2 to -1     n.d.       1.2 (1.2)                            n.d.-1 to 0      0.7 (0.6)  1.3 (0.8)                            2.1 (1.4) 0 to +1     1.5 (1.3)  0.5 (0.3)                            2.0 (1.5)Avg. change in        1.1 (1.0)  1.0 (0.8)                            2.1 (1.5)log kcat/Km or(log 1/Km) perunit charge change______________________________________ .sup.(a) The difference in the slopes of curves were taken between the P1 substrates over the charge interval given for log (k(cat/Km) (FIG. 3A, B) and (log 1/Km) (FIG. 3D, D). Values represent the differential effect a charge change has in distinguishing the substrates that are compared. .sup.(b) Charge in P1 binding site is defined as the sum of charges from positions 156 and 166.</pre>
    
    <p>The free energy of electrostatic interactions in the structure and energetics of salt-bridge formation depends on the distance between the charges and the microscopic dielectric of the media. To dissect these structural and microenvironmental effects, the energies involved in specific salt-bridges were evaluated. In addition to the possible salt-bridges shown (FIGS. 29A and 29B), reasonable salt-bridges can be built between a Lys P-1 substrate and Asp at position 166, and between a Glu P-1 substrate and a Lys at position 166 (not shown). Although only one of these structures is confirmed by X-ray crystalography (Poulos, T. L., et al. (1976) J. Mol. Biol. 257 1097-1103), all models have favorable torsion angles (Sielecki, A. R., et al. (1979) J. Mol. Biol. 134, 781-804), and do not introduce unfavorable van der Waals contacts.</p>
    <p>The change in charged P-1 substrate preference brought about by formation of the model salt-bridges above are shown in Table XVI.</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">                                  TABLE XVI__________________________________________________________________________Effect of Salt Bridge Formation Between Enzymeand Substrate on P1 Substrate Preference.sup.(a)                           Change                    Substrate.sup.(d)                           in Substrate          Enzyme               P-1  Preference                           PreferenceEnzymes Compared.sup.(b)          Position               Substrates                    Δlog (kcat/Km)                           ΔΔlog (kcat/Km)1       2      Changed               Compared                    1   2  (1-2)__________________________________________________________________________Glu156/Asp166   Gln156/Asp166          156  LysMet                    +0.30                        -0.53                           0.83Glu156/Asn166   Gln156/Asn166          156  LysMet                    -0.84                        -2.04                           1.20Glu156/Gly166   Gln156/Gly166          156  LysMet                    -0.47                        -2.10                           1.63Glu156/Lsy-166   Gln156/Lys166          156  LysMet                    -1.92                        -2.74                           0.82                 Ave ΔΔlog (kcat/Km)                           1.10 ± 0.3Glu156/Asp166   Glu156/Asn166          166  LysMet                    +0.30                        -0.84                           1.14Glu156/Glu166   Glu156/Glu166          166  LysMet                    +0.62                        -1.33                           1.95Gln156/Asp166   Gln156/Asn166          166  LysMet                    -0.53                        -2.04                           1.51Ser156/Asp166   Ser156/Asn166          166  LysMet                    -0.43                        -2.04                           1.61Glu156/Lys166   Glu156/Met166          166  GluGln                    -0.63                        -2.69                           2/06                 Ave ΔΔlog (kcat/Km)                           1.70 +0.3__________________________________________________________________________ Footnotes to Table XVI: .sup.(a) Molecular modeling shows it is possible to form a salt bridge between the indicated charged P1 substrate and a complementary charge in the P1 binding site of the enzyme at the indicated position changed. .sup.(b) Enzymes compared have sterically similar amino acid substitution that differ in charge at the indicated position. .sup.(c) The P1 substrates compared are structurally similar but differ i charge. The charged P1 substrate is complementary to the charge change at the position indicated between enzymes 1 and 2. .sup.(d) Date from Table XIV was used to compute the difference in log (kcat/Km) between the charged and the noncharged P1 substrate (i.e., the substrate preference). The substrate preference is shown separately for enzyme 1 and 2. .sup.(e) The difference in substrate preference between enzyme 1 (more highly charged) and enzyme 2 (more neutral) represents the rate change accompanying the electrostatic interaction.</pre>
    
    <p>The difference between catalytic efficiencies (i.e., Δlog kcat/Km) for the charged and neutral P-1 substrates (e.g., Lys minus Met or Glu minus Gln) give the substrate preference for each enzyme. The change in substrate preference (ΔΔlog kcat/Km) between the charged and more neutral enzyme homologs (e.g., Glu156/Gly166 minus Gln156(Q156)/Gly166) reflects the change in catalytic efficiency that may be attributed solely to electrostatic effects.</p>
    <p>These results show that the average change in substrate preference is considerably greater when electrostatic substitutions are produced at position 166 (50-fold in kcat/Km) versus position 156 (12-fold in kcat/Km). From these ΔΔlog kcat/Km values, an average change in transition-state stabilization energy can be calculated of -1.5 and -2.4 kcal/mol for substitutions at positions 156 and 166, respectively. This should represent the stabilization energy contributed from a favorable electrostatic interaction for the binding of free enzyme and substrate to form the transition-state complex.</p>
    <p>At least three factors can contribute to the higher transition-state binding energies for electrostatic interactions at position 166. These include: (i) smaller charge separation for salt-bridges at position 166; (ii) more stable side-chain geometries for salt-bridges at position 166; and (iii) microscopic dielectric constants at positions 166.</p>
    <p>It is unreasonable to expect all of the energy difference to be due to shorter salt bridges at position 166, because these would have to be 1.6 times shorter than at position 156 for which crystalographic data (Mathews, D. A., et al. (1975) J. Biol. Chem. 250, 7120-7126) indicate are optimally formed. Furthermore, molecular models of salt-bridges appear as structurally reasonable at 156 as at 166.</p>
    <p>The binding energies may be more easily explained as differences in the microscopic dielectric constants at position 156 and 166. Assuming a salt-bridge distance of 3A, ˜2.7A), the calculated dielectric constant at position 156 would be 72 (ΔGe=Z<sub>1</sub> Z<sub>2</sub> /rD where Z is the charge on particle 1 and 2, r is the charge separation, and D is the dielectric constant). This corresponds closely with the dielectric constant of 78 for water at this temperature, and qualitatively fits with the fact that position 156 is located on the surface of the enzyme, and is freely exposed to solvent. A calculated dielectric constant for a salt-bridge at position 166 is 45, and fits with the fact that position 166 is more buried and less accessible to solvent. Furthermore, our estimate, based on the hydrophobicity, of the P-1 binding site, indicates that P-1 binding site has an overall dielectric constant close to that of ethanol (D=25).</p>
    <p>A large number of mutant comparisons is necessary to show a statistically significant difference between salt-bridges at positive 156 and 166 because there is considerable variation in ΔΔlog kcat/Km for different mutant comparisons at the same position. The change in susbtrate preference from putative salt-bridges at position 156 varies from six to 40-fold in kcat/Km, and those at position 166 vary 14 to 120 fold.</p>
    <p>In addition to variation produced by factors mentioned above, it is possible that the P-1 side chains are not binding in the same ways between the enzymes compared, even though the comparisons are nearly isosteric in each case. For example, the Lys P-1 substrate side chain may contact Glu156 in Glu156/Asp166 (Glu156/D166) and Asp166 in Gln156/Asp166 (Q156/D166). Thus, one salt-bridge may be substitued for another. It is also possible that complementary charges within the P-1 binding site, e.g., Glu156/Lys166 (Glu156/K166), can form an intramolecular salt-bridge so that the charged side-chains are not free to interact independently with the substrate. Given these caveats it is remarkable that greater variation in substrate preference is not seen by electrostatic substitutions at each position.</p>
    <heading>EXAMPLE 10</heading> <heading>Substitutions at Position 217</heading> <p>Tyr217 has been substituted by all other 19 amino acids. Cassette mutagenesis as described in EPO publication No. 0130756 was used according to the protocol of FIG. 22. The EcoRV restriction site was used for restriction-purification of p.sup.Δ 217.</p>
    <p>Since this position is involved in substrate binding, mutations here affect kinetic parameters of the enzyme. An example is the substitution of Leu for Tyr at position 217. For the substrate sAAPFpNa, this mutant has a kcat of 277 5' and a Km of 4.7×10<sup>-4</sup> with a kcat/Km ratio of 6×10<sup>5</sup>. This represents a 5.5-fold increase in kcat with a 3-fold increase in Km over the wild type enzyme.</p>
    <p>In addition, replacement of Tyr217 by Lys, Arg, Phe or Leu results in mutant enzymes which are more stable at pH of about 9-11 than the WT enzyme. Conversely, replacement of Tyr217 by Asp, Glu, Gly or Pro results in enzymes which are less stable at pH of about 9-11 than the WT enzyme.</p>
    <heading>EXAMPLE 11</heading> <heading>Multiple Mutants having Altered Thermal Stability</heading> <p>B. amyloliquefacien subtilisin does not contain any cysteine residues. Thus, any attempt to produce thermal stability by Cys cross-linkage required the substitution of more than one amino acid in subtilisin with Cys. The following subtilisin residues were multiply substituted with cysteine:</p>
    <p>Thr22/Ser87</p>
    <p>Ser24/Ser87</p>
    <p>Mutagenesis of Ser24 to Cys was carried out with a 5' phosphorylated oligonucleotide primer having the sequence</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="equation">5'-pC-TAC-ACT-GGA-T*GC*-AAT-GTT-AAA-G-3'.</pre>
    
    <p>(Asterisks show the location of mismatches and the underlined sequence shows the position of the altered Sau3A site.) The B. amyloliquefaciens subtilisin gene on a 1.5 kb EcoRI-BAMHI fragment from pS4.5 was cloned into M13mpll and single stranded DNA was isolated. This template (M13mpllSUBT) was double primed with the 5' phosphorylated M13 universal sequencing primer and the mutagenesis primer. Adelman, et al. (1983) DNA 2, 183-193. The heteroduplex was transfected into competent JM101 cells and plaques were probed for the mutant sequence (Zoller, M. J., et al. (1982) Nucleic Acid Res. 10, 6487-6500; Wallace, et al. (1981) Nucleic Acid Res. 9, 3647-3656) using a tetramethylammonium chloride hybridization protocol (Wood, et al. (1985) Proc. Natl. Acad. Sci. USA 82, 1585-1588). The Ser87 to Cys mutation was prepared in a similar fashion using a 5' phosphorylated primer having the sequence</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="equation">5'-pGGC-GTT-GCG-CCA-T*GC-GCA-TCA-CT-3'.</pre>
    
    <p>(The asterisk indicates the position of the mismatch and the underlined sequence shows the position of a new MstI site.) The C24 and C87 mutations were obtained at a frequency of one and two percent, respectively. Mutant sequences were confirmed by dideoxy sequencing in M13.</p>
    <p>Mutagenesis of Tyr21/Thr22 to A21/C22 was carried out with a 5' phosphorylated oligonucleotide primer having the sequence</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="equation">5'-pAC-TCT-CAA-GGC-G*C*T*-T*G*T*-GGC*-TCA-AAT-GTT-3'.</pre>
    
    <p>(The asterisks show mismatches to the wild type sequence and the underlined sequence shows the position of an altered Sau3A site.) Manipulations for heteroduplex synthesis were identical to those described for C24. Because direct cloning of the heteroduplex DNA fragment can yield increased frequencies of mutagenesis, the EcoRI-BamHI subtilisin fragment was purified and ligated into pBS42. E. coli MM 294 cells were transformed with the ligation mixture and plasmid DNA was purified from isolated transformants. Plasmid DNA was screened for the loss of the Sau3A site at codon 23 that was eliminated by the mutagenesis primer. Two out of 16 plasmid preparations had lost the wild type Sau3A site. The mutant sequence was confirmed by dideoxy sequencing in M13.</p>
    <p>Double mutants, C22/C87 and C24/C87, were constructed by ligating fragments sharing a common ClaI site that separated the single parent cystine codons. Specifically, the 500 bp EcoRI-ClaI fragment containing the 5' portion of the subtilisin gene (including codons 22 and 24) was ligated with the 4.7 kb ClaI-EcoRI fragment that contained the 3' portion of the subtilisin gene (including codon 87) plus pBS42 vector sequence. E. coli MM 294 was transformed with ligation mixtures and plasmid DNA was purified from individual transformants. Double-cysteine plasmid constructions were identified by restriction site markers originating from the parent cysteine mutants (i.e., C22 and C24, Sau3A minus; Cys87, MstI plus). Plasmids from E. coli were transformed into B. subtilis BG2036. The thermal stability of these mutants as compared to wild type subtilisin are presented in FIGS. 30A, 30B and 30C and Tables XVII and XVIII.</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">              TABLE XVII______________________________________Effect of DTT on the Half-Time ofAutolytic Inactivation of Wild-Typeand Disulfide Mutants of Subtilisin*   t<sub>1/2</sub>     -DDT        +DTTEnzyme    min             -DTT/+DTT______________________________________Wild-type 95          85      1.1C22/C87   44          25      1.8C24/C87   92          62      1.5______________________________________ *Purified enzymes were either treated or not treated with 25 mM DTT and dialyzed with or without 10 mM DTT in 2 mM CaCl<sub>2</sub>, 50 mM Tris (pH 7.5 for 14 hr. at 4° C. Enzyme concentrations were adjusted to 80 μ aliquots were quenched on ice and assayed for residual activity. Ha1ftime for autolytic inactivation were determined from Semilog plots of 1og<sub>10</sub> (residual activity) versus time. These plots were linear for over 90% of the inactivation.</pre>
    
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">              TABLE XVIII______________________________________Effect of Mutations in Subtilisinon the Half-Time of AutolyticInactivation at 58° C.*          t<sub>1/2</sub>   Enzyme min______________________________________   Wild-type          120   C22    22   C24    120   C87    104   C22/C87          43   C24/C87          115______________________________________ *Half-times for autolytic inactivation were determined for wildtype and mutant subtjlisins as described in the legend to Table III. Unpurified an nonreduced enzymes were used directly from B. subtilis culture supernatants.</pre>
    
    <p>It has been demonstrated that double-cysteine mutants of subtilisin are efficiently secreted and that disulfide bonds are formed in vivo in B. subtilis (date not shown). The introduction of disulfide bonds in subtilisin extends upon previous work in dihydrofolate reductase and T4 lysozyme (Perry, L. J., et al. (1984) Science 226, 555-557), where single cysteines were introduced near pre-existing cysteines and disulfides were oxidized in vitro. Analyses of physical properties of the subtilisin disulfides, unlike the T4 lysozyme disulfide (Perry, L. J., et al. (1986) Biochemistry, in press), were not complicated by the presence of free cysteines other than those involved in disulfide formation. Because most naturally occuring disulfides occur in secreted proteins, subtilisin is an excellent model system to identify the structural requirements for in vitro formation of stable disulfide bonds in secreted proteins.</p>
    <p>Thermal Stability and Autolytic Stability</p>
    <p>The data presented here do not address reverisble thermostability of subtilisin directly because of complications arising from autolysis and aggregation. For example, studies monitoring the change in the circular dichroic eliptcity at 220 nm versus temperature of phenylmethanesulfonyl fluoride-inhibited subtilisin show typical melt profiles that are coincident with the autolysis curves. However, at the end of thermal melt, SDS-PAGE shows that &gt;90% of the subtilisin is autolyzed. Moreover, Brown and Schleich (Brown, M. F., et al. (1975) Biochemistry 14, 3069-3074) have shown that diisopropylfluorophosphate-inhibited subtilisin irreversibly aggregates in denaturants, which precludes reversible denaturation studies. Thus, until these problems are overcome, subtilisin is not an ideal system for studying the thermodynamics of protein folding.</p>
    <p>Although there appears to be a relationship between autolytic stability and conformational stability, the disulfides introduced into subtilisin did not improve the autolytic stability of the mutant enzymes when compared to the wild-type enzyme. However, the disulfide bonds did provide a margin of autolytic stability when compared to their corresponding reduced double-cysteine enzyme. Inspection of a highly refined x-ray structure of wild-type B. amyloliquefaciens subtilisin reveals a hydrogen bond between Thr22 and Ser87. Because cysteine is a poor hydrogen donor or acceptor (Paul, I. C. (1974) in Chemistry of the --SH Group (Patai, S., ed.) pp. 111-149, Wiley Interscience, New York) weakening of 22/87 hydrogen bond may explain why the C22 and C87 single-cysteine mutant proteins are less autolytically stable than either C24 or wild-type (Table XVIII). The fact that C22 is less autolytically stable than C87 may be the result of the Tyr21A mutation (Table XVIII). indeed, recent construction and analysis of Tyr21/C22 shows the mutant protein has an autolytic stability closer to that of C87. In summary, the C22 and C87 of single-cysteine mutations destabilize the protein toward autolysis, and disulfide bond formation increases the stability to a level less than or equal to that of wild-type enzyme.</p>
    <p>These data suggest that the stabilizing effect of an engineered disulfide can be lowered when the parent cysteine mutations disrupt pre-existing stabilizing interactions. Similar conclusions have been drawn from reversible thermal unfolding studies of disulfide cross-linked T4 lysozyme mutants that contain destabilizing mutations. Therefore, a strategy to stbilize a protein by introduction of a disulfide bond should consider avoiding the disruption of stabilizing interactions as well as producing a disulfide with good bond geometry.</p>
    <heading>EXAMPLE 12</heading> <heading>Multiple Mutants Containing Substitutions at Position 222 and Position 166 or 169</heading> <p>Double mutants 166/222 and 169/222 were prepared by ligating together (1) the 2.3 kb AcaII fragment from pS4.5 which contains the 5' portion of the subtilisin gene and vector sequences, (2) the 200 bp AvaII fragment which contains the relevant 166 or 169 mutations from the respective 166 or 169 plasmids, and (3) the 2.2 kb AvaII fragment which contains the relevant 222 mutation 3' and of the subtilisin genes and vector sequence from the respective p222 plasmid.</p>
    <p>Although mutations at position 222 improve oxidation stability they also tend to increase the Km. An example is shown in Table XIX. In this case the A222 mutation was combined with the K166 mutation to give an enzyme with kcat and Km intermediate between the two parent enzymes.</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">              TABLE XIX______________________________________          kcat Km______________________________________WT               50     1.4 × 10<sup>-4</sup>A222             42     9.9 × 10<sup>-4</sup>K166             21     3.7 × 10<sup>-5</sup>K166/A222        29     2.0 × 10<sup>-4</sup>______________________________________ substrate sAAPFpNa</pre>
    
    <heading>EXAMPLE 13</heading> <heading>Multiple Mutants Containing Substitutions at Positions 50, 156, 166, 217 and Combinations Thereof</heading> <p>The double mutant S156/A169 was prepared by ligation of two fragments, each containing one of the relevant mutations. The plasmid pS156 was cut with XmaI and treated with S1 nuclease to create a blunt end at codon 167. After removal of the nuclease by phenol/chloroform extraction and ethanol precipitation, the DNA was digested with BamHI and the approximately 4 kb fragment containing the vector plus the 5' portion of the subtilisin gene through codon 167 was purified.</p>
    <p>The pA169 plasmid was digested with KpnI and treated with DNA polymerase Klenow fragment plus 50 μM dNTPs to create a blunt end codon at codon 168. The Klenow was removed by phenol/chloroform extraction and ethanol precipitation. The DNA was digested with BamHI and the 590 bp fragment including codon 168 through the carboxy terminus of the subtilisin gene was isolated. The two fragments were then ligated to give S156/A169.</p>
    <p>Triple and quadruple mutants were prepared by ligating together (1) the 220 bp PvuII/HaeII fragment containing the relevant 156, 166 and/or 169 mutations from the respective p156, p166 and/or p169 double of single mutant plasmid, (2) the 550 bp HaeII/BamHI fragment containing the relevant 217 mutant from the respective p217 plasmid, and (3) the 3.9 kb PvuII/BamHI fragment containing the F50 mutation and vector sequences. The multiple mutant F50/S156/A169/L217, as well as B. amyloliquefaciens subtilisin, B. lichenformis subtilisin and the single mutant L217 were analyzed with the above synthetic polypeptides where the P-1 amino acid in the substrate was Lys, His, Ala, Gln, Tyr, Phe, Met and Leu. These results are shown in FIGS. 26 and 27.</p>
    <p>These results show that the F50/S156/A169/L217 mutant has substrate specificity similar to that of the B. licheniformis enzyme and differs dramatically from the wild type enzyme. Although only data for the L217 mutant are shown, none of the single mutants (e.g., F50, S156 or A169) showed this effect. Although B. licheniformis differs in 88 residue positions from B. amyloliquefaciens, the combination of only these four mutations accounts for most of the differences in substrate specificity between the two enzymes.</p>
    <heading>EXAMPLE 14</heading> <heading>Subtilisin Mutants Having Altered Alkaline Stability</heading> <p>A random mutagenesis technique was used to generate single and multiple mutations within the B. amyloliquefaciens subtilisin gene. Such mutants were screened for altered alkaline stability. Clones having increased (positive) alkaline stability and decreased (negative) alkaline stability were isolated and sequenced to identify the mutations within the subtilisin gene. Among the positive clones, the mutants V107 and R213 were identified. These single mutants were subsequently combined to produce the mutant V107/R213.</p>
    <p>One of the negative clones (V50) from the random mutagenesis experiments resulted in a marked decrease in alkaline stability. Another mutant (P50) was analyzed for alkaline stability to determine the effect of a different substitution at position 50. The F50 mutant was found to have a greater alkaline stability than wild type subtilisin and when combined with the double mutant V107/R213 resulted in a mutant having an alkaline stability which reflected the aggregate of the alkaline stabilities for each of the individual mutants.</p>
    <p>The single mutant R204 and double mutant C204/R213 were identified by alkaline screening after random cassette mutagenesis over the region from position 197 to 228. The C204/R213 mutant was thereafter modified to produce mutants containing the individual mutations C204 and R213 to determine the contribution of each of the individual mutations. Cassette mutagenesis using pooled oligonucleotides to substitute all amino acids at position 204, was utilized to determine which substitution at position 204 would maximize the increase in alkaline stability. The mutation from Lys213 to Arg was maintained constant for each of these substitutions at position 204.</p>
    <p>A. Construction of pB0180, an E. coli-B. subtilis Shuttle Plasmid</p>
    <p>The 2.9 kb EcoRI-BamHI fragment from pBR327 (Covarrubias, L., et al. (1981) Gene 13, 25-35) was ligated to the 3.7 kb EcoRI-BamHI fragment of pBD64 (Gryczan, T., et al. (1980) J. Bacteriol., <sub>--</sub>, 246-253) to give the recombinant plasmid pB0153. The unique EcoRI recognition sequence in pBD64 was eliminated by digestion with EcoRI followed by treatment with Klenow and deoxynucleotide triphosphates (Maniatis, T., et al. (eds.) (1982) in Molecular Cloning, A Laboratory Manual, Cold Spring Harbor Laboratory, Cold Spring Harbor, N.Y.). Blunt end ligation and transformation yielded pB0154. The unique AvaI recognition sequence in pBO154 was eliminated in a similar manner to yield pBO171. pB0171 was digested with BamHI and PvuII and treated with Klenow and deoxynucleotide triphosphates to create blunt ends. The 6.4 kb fragment was purified, ligated and transformed into LE392 cells (Enquest, L. W., et al. (1977) J. Mol. Biol. 111, 97-120), to yield pB0172 which retains the unique BamHI site. To facilitate subcloning of subtilisin mutants, a unique and silent KpnI site starting at codon 166 was introduced into the subtilisin gene from pS4.5 (Wells, J. A., et al. (1983) Nucleic Acids Res., 11, 7911-7925) by site-directed mutagenesis. The KpnI+ plasmid was digested with EcoRI and treated with Klenow and deoxynucleotide triphosphates to create a blunt end. The Klenow was inactivated by heating for 20 min at 68° C., and the DNA was digested with BamHI. The 1.5 kb blunt EcoRI-BamHI fragment containing the entire subtilisin was ligated with the 5.8 kb NruI-BamHI from pB0172 to yield pBO180. The ligation of the blunt NruI end to the blunt EcoRI end recreated an EcoRI site. Proceeding clockwise around pB0180 from the EcoRI site at the 5' end of the subtilisin gene is the unique BamHI site at the 3' end of the subtilisin gene, the chloramphenicol and neomycin resistance genes and. UB110 gram positive replication origin derived from pBD64, the ampicillin resistance gene and gram negative replication origin derived from pBR327.</p>
    <p>B. Construction of Random Mutagenesis Library</p>
    <p>The 1.5 kb EcoRI-BamHI fragment containing the B. amyloliquefaciens subtilisin gene (Wells et al., 1983) from pB0180 was cloned into M13mpll to give M13mpll SUBT essentially as previously described (Wells, J. A., et al. (1986) J. Biol. Chem., 261,6564-6570). Deoxyuridine containing template DNA was prepared according to Kunkel (Kunkel, T. A. (1985) Proc. Natl. Acad. Sci. USA, 82 488-492). Uridine containing template DNA (Kunkel, 1985) was purified by CsCl density gradients (Maniatis, T. et al. (eds.) (1982) in Molecular Cloning, A Laboratory Manual, Cold Spring Harbor Laboratory, Cold Spring Harbor,. N.Y.). A primer (AvaI)<sup>-</sup> having the sequence</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="equation">5'GAAAAAAGACCCT*AGCGTCGCTTA</pre>
    
    <p>ending at codon -11, was used to alter the unique AvaI recognition sequence within the subtilisin gene. (The asterisk denotes the mismatches from the wild-type sequence and underlined is the altered AvaI site.) The 5' phosphorylated AvaI primer ˜320 pmol) and ˜40 pmol (˜120 μg) of uridine containing M13mpll SUBT template in 1.88 ml of 53 mM NaCl, 7.4 mM MgCl2 and 7.4 mM Tris.HCl (pH 7.5) were annealed by heating to 90° C. for 2 min. and cooling 15 min at 24° C. (FIG. 31). Primer extension at 24° C. was initiated by addition of 100 μL containing 1 mM in all four deoxynucleotide triphosphates, and 20 μl Klenow fragment (5 units/l). The extension reaction was stopped every 15 seconds over ten min by addition of 10 μl 0.25M EDTA (pH 8) to 50 μl aliquots of the reaction mixture. Samples were pooled, phenol chlorophorm extract and DNA was precipitated twice by addition of 2.5 vol 100% ethanol, and washed twice with 70% ethanol. The pellet was dried, and redissolved in 0.4 ml 1 mM EDTA, 10 mM Tris (pH 8). Misincorporation of α-thiodeoxynucleotides onto the 3' ends of the pool of randomly terminated template was carried out by incubating four 0.2 ml solutions each containing one-fourth of the randomly terminated template mixture (˜20 μg), 0.25 mM of a given α-thiodeoxynucleotide triphosphate, 100 units AMV polymerase, 50 mM KCL, 10 mM MgCl2, 0.4 mM dithiothreitol, and 50 mM Tris (pH 8.3) (Champoux, J. J. (1984) Genetics, 2, 454-464). After incubation at 37° C. for 90 minutes, misincorporation reactions were sealed by incubation for five minutes at 37° C. with 50 mM all four deoxynucleotide triphosphates (pH 8), and 50 units AMV polymerase. Reactions were stopped by addition of 25 mM EDTA (final), and heated at 68° C. for ten min to inactivate AMV polymerase. After ethanol precipitation and resuspension, synthesis of closed circular heteroduplexes was carried out for two days at 14° C. under the same conditions used for the timed extension reactions above, except the reactions also contained 1000 units T4 DNA ligase, 0.5 mM ATP and 1 mM β-mercaptoethanol. Simultaneous restriction of each heteroduplex pool with KpnI, BamHI, and EcoRI confirmed that the extension reactions were nearly quantitative. Heteroduplex DNA in each reaction mixture was methylated by incubation with 80μM S-adenosylmethionine and 150 units dam methylase for 1 hour at 37° C. Methylation reactions were stopped by heating at 68° C. for 15 min. One-half of each of the four methylated heteroduplex reactions were transformed into 2.5 ml competent E. coli JM101 (Messing, J. (1979) Recombinant DNA Tech. Bull., 2, 43-48). The number of independent transformants from each of the four transformations ranged from 0.4-2.0×10<sup>5</sup>. After growing out phage pools, RF DNA from each of the four transformations was isolated and purified by centrifugation through CsCl density gradients. Approximately 2 μg of RF DNA from each of the four pools was digested with EcoRI, BamHI and AvaI. The 1.5 kb EcoRI-BamHI fragment (i.e., AvaI resistant) was purified on low gel temperature agarose and ligated into the 5.5 kb EcoRI-BamHI vector fragment of pB0180. The total number of independent transformants from each α-thiodeoxynucleotide misincorporation plasmid library ranged from 1.2-2.4×10<sup>4</sup>. The pool of plasmids from each of the four transformations was grown out in 200 ml LB media containing 12.5 μg/ml cmp and plasmid DNA was purified by centrifugation through CsCl density gradients.</p>
    <p>C. Expression and Screening of Subtilisin Point Mutants</p>
    <p>Plasmid DNA from each of the four misincorporation pools was transformed (Anagnostopoulos, C., et al. (1967, J. Bacteriol., 81, 741-746) into BG2036, a strain of B. subtilis deficient in extracellular protease genes (Yang, M. Y. et al. (1984) J. Bacteriol., 160, 15-21). For each transformation, 5 μg of DNA produced approximately 2.5×10<sup>5</sup> independent BG2036 transformants, and liquid culture aliquots from the four libraries were stored in 10% glycerol at 70° C. Thawed aliquots of frozen cultures were plated on LB/5 μg/ml cmp/1.6% skim milk plates (Wells, J. A., et al. (1983) Nucleic Acids Res., 11, 7911-7925), and fresh colonies were arrayed onto 96-well microtiter plates containing 150 l per well LB media plus 12.5 μg/ml cmp. After 1 h at room temperature, a replica was stamped (using a matched 96 prong stamp) onto a 132 mm BA 85 nitrocellulose filter (Schleicher and Scheull) which was layered on a 140 mm diameter LB/cmp/skim milk plate. Cells were grown about 16 h at 30° C. until halos of proteolysis were roughly 5-7 mm in diameter and filters were transferred directly to a freshly prepared agar plate at 37° C. containing only 1.6% skim milk and 50 mM sodium phosphate pH 11.5. Filters were incubated on plates for 3-6 h at 37° C. to produce halos of about 5 mm for wild-type subtilisin and were discarded. The plates were stained for 10 min at 24° C. with Coomassie blue solution (0.25% Coomassie blue (R-250) 25% ethanol) and destained with 25% ethanol, 10% acetic acid for 20 min. Zones of proteolysis appeared as blue halos on a white background on the underside of the plate and were compared to the original growth plate that was similarly stained and destained as a control. Clones were considered positive that produced proportionately larger zones of proteolysis on the high pH plates relative to the original growth plate. Negative clones gave smaller halos under alkaline conditions. Positive and negative clones were restreaked to colony purify and screened again in triplicate to confirm alkaline pH results.</p>
    <p>D. Identification and Analysis of Mutant Subtilisins</p>
    <p>Plasmid DNA from 5 ml overnight cultures of more alkaline active B. subtilis clones was prepared according to Birnboim and Doly (1979) except that incubation with 2 mg/ml lysozyme proceeded for 5 min at 37° C. to ensure cell lysis and an additional phenol/CHCl<sub>3</sub> extraction was employed to remove contaminants. The 1.5 kb EcoRI-BamHI fragment containing the subtilisin gene was ligated into M13mpll and template DNA was prepared for DNA sequencing (Messing, J., et al. (1982) Gene, 19 269-276). Three DNA sequencing primers ending at codon 26, +95, and +155 were synthesized to match the subtilisin coding sequence. For preliminary sequence identification a single track of DNA sequence, corresponding to the dNTPaS misincorporation library from which the mutant came, was applied over the entire mature protein coding sequence (i.e., a single dideoxyguanosine sequence track was applied to identify a mutant from the dGTPas library). A complete four track of DNA sequence was performed 200 bp over the site of mutagenesis to confirm and identify the mutant sequence (Sanger, F., et al., (1980) J. Mol. Biol., 143, 161-178). Confirmed positive and negative bacilli clones were cultured in LB media containing 12.5 μg/mL cmp and purified from culture supernatants as previously described (Estell, D. A., et al. (1985) J. Biol. Chem., 260, 6518-6521). Enzymes were greater than 98% pure as analyzed by SDS-polyacrylamide gel electrophoresis (Laemmli, U. K. (1970), Nature, 227, 680-685), and protein concentrations were calculated from the absorbance at 280 nm (ε<sub>280</sub> <sup>0</sup>.1% =1.17, Maturbara, H., et al. (1965), J. Biol. Chem, , 1125-1130).</p>
    <p>Enzyme activity was measured with 200 μg/mL succinyl-L-AlaL-AlaL-ProL-Phep-nitroanilide (Sigma) in 0.1M Tris pH 8.6 or 0.1M CAPS pH 10.8 at 25° C. Specific activity (μ moles product/min-mg) was calculated from the change in absorbance at 410 nm from production of p-nitroaniline with time per mg of enzyme (E410=8,480 M-1 cm-1; Del Mar, E. G., et al. (1979), Anal. Biochem., 99, 316-320). Alkaline autolytic stability studies were performed on purified enzymes (200 μg/mL) in 0.1M potassium phosphate (pH 12.0) at 37C. At various times aliquots were assayed for residual enzyme activity (Wells, J. A., et al. (1986) J. Biol. Chem., 261, 6564-6570).</p>
    <p>E. Results</p>
    <p>1. Optimization and analysis of mutagenesis frequency</p>
    <p>A set of primer-template molecules that were randomly 3'-terminated over the subtilisin gene (FIG. 31) was produced by variable extension from a fixed 5'-primer (The primer mutated a unique AvaI site at codon 11 in the subtilisin gene). This was achieved by stopping polymerase reactions with EDTA after various times of extension. The extent and distribution of duplex formation over the 1 kb subtilisin gene fragment was assessed by multiple restriction digestion (not shown). For example, production of new HinfI fragments identified when polymerase extension had proceeded past Ile110, Leu233, and Asp259 in the subtilisin gene.</p>
    <p>Misincorporation of each dNTPαs at randomly terminated 3' ends by AMV reverse transcriptase (Zakour, R. A., et al. (1982), Nature, 295, 708-710; Zakour, R. A., et al. (1984), Nucleic Acids Res., 12, 6615-6628) used conditions previously described (Champoux, J. J., (1984), Genetics, 2, 454-464). The efficiency of each misincorporation reaction was estimated to be greater than 80% by the addition of each dNTPαs to the AvaI restriction primer, and analysis by polyacrylamide gel electrophoresis (Champoux, J. J., (1984). Misincorporations were sealed by polymerization with all four dNTP's and closed circular DNA was produced by reaction with DNA ligase.</p>
    <p>Several manipulations were employed to maximize the yield of the mutant sequences in the heteroduplex. These included the use of a deoxyuridine containing template (Kunkel, T. A. (1985), Proc. Natl. Acad. Sci. USA, 82 488-492; Pukkila, P. J. et al. (1983), Genetics, 104, 571-582), invitro methylation of the mutagenic strand (Kramer, W. et al. (1982) Nucleic Acids Res., 10 6475-6485), and the use of AvaI restriction-selection against the wild-type template strand which contained a unique AvaI site. The separate contribution of each of these enrichment procedures to the final mutagenesis frequency was not determined, except that prior to AvaI restriction-selection roughly one-third of the segregated clones in each of the four pools still retained a wild-type AvaI site within the subtilisin gene. After AvaI restriction-selection greater than 98% of the plasmids lacked the wild-type AvaI site.</p>
    <p>The 1.5 kb EcoRI-BamHI subtilisin gene fragment that was resistant to AvaI restriction digestion, from each of the four CsCl purified M13 RF pools was isolated on low melting agarose. The fragment was ligated in situ from the agarose with a similarly cut E. coli-B. subtilis shuttle vector, pB0180, and transformed directly into E. coli LE392. Such direct ligation and transformation of DNA isolated from agarose avoided loses and allowed large numbers of recombinants to be obtained (&gt;100,000 per μg equivalent of input M13 pool).</p>
    <p>The frequency of mutagenesis for each of the four dNTPαs misincorporation reactions was estimated from the frequency that unique restriction sites were eliminated (Table XX). The unique restriction sites chosen for this analysis, ClaI, PvuII, and KpnI, were distributed over the subtilisin gene starting at codons 35, 104, and 166, respectively. As a control, the mutagenesis frequency was determined at the PstI site located in the β lactamase gene which was outside the window of mutagenesis. Because the absolute mutagenesis frequency was close to the percentage of undigested plasmid DNA, two rounds of restriction-selection were necessary to reduce the background of surviving uncut wild-type plasmid DNA below the mutant plasmid (Table XX). The background of surviving plasmid from wild-type DNA probably represents the sum total of spontaneous mutations, uncut wild-type plasmid, plus the efficiency with which linear DNA can transform E. coli. Subtracting the frequency for unmutagenized DNA (background) from the frequency for mutant DNA, and normalizing for the window of mutagenesis sampled by a given restriction analysis (4-6 bp) provides an estimate of the mutagenesis efficiency over the entire coding sequence (<sub>--</sub> 1000 bp).</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">              TABLE XX______________________________________α-thiol  Re-                            %dNTP   striction          % resistant clones.sup.(c)                        % resistant                                 mutantsmisincor-  Site Se-          1st    2nd        clones over                                     perporated.sup.(b)  lection round  round                      Total Background.sup.(d)                                     10000bp.sup.(e)______________________________________None   PstI    0.32   0.7  0.002 0        --G      PstI    0.33   1.0  0.003 0.001    0.2T      PstI    0.32   &lt;0.5 &lt;0.002                            0        0C      PstI    0.43   3.0  0.013 0.011    3None   ClaI    0.28   5    0.014 0        --G      ClaI    2.26   85   1.92  1.91     380T      ClaI    0.48   31   0.15  0.14     35C      ClaI    0.55   15   0.08  0.066    17None   PvuII   0.08   29   0.023 0        --G      PvuII   0.41   90   0.37  0.35     88T      PvuII   0.10   67   0.067 0.044    9C      PvuII   0.76   53   0.40  0.38     95None   KpnI    0.41   3    0.012 0        --G      KpnI    0.98   35   0.34  0.33     83T      KpnI    0.36   15   0.054 0.042    8C      KpnI    1.47   26   0.38  0.37     93______________________________________ .sup.(a) Mutagenesis frequency is estimated from the frequency for obtaining mutations that alter unique restriction sites within the mutagenized subtilisin gene (i.e., ClaI, PvuII, or KpnI) compared to mutation frequencies of the PstI site, that is outside the window of mutagenesis. .sup.(b) Plasmid DNA was from wildtype (none) or mutagenized by dNTPαS misincorporation as described. .sup.(c) Percentage of resistant clones was calculated from the fraction of clones obtained after three fold or greater overdigestion of the plasmid with the indicated restriction enzyme compared to a nondigested control. Restrictionresistant plasmid DNA from the first round was subjected to a second round of restrictionselection. The tota1 represents the product of the fractions of resistant clones obtained from both round of selection and gives percentage of restrictionsite mutant clones in the original starting pool. Frequencies were derived from counting at least 2 colonies and usually greater than 100. .sup.(d) Percent resistant clones was calculated by subtracting the percentage of restrictionresistant clones obtained for wildtype DNA (i.e. none) from that obtained for mutant DNA. .sup.(e) This extrapolates from the frequency of mutation over each restriction site to the entire subtilisin gene (˜1 kb). This has been normalized to the number of possible bases (4-6 bp) within each restriction site that can be mutagenized by a given misincorporation event.</pre>
    
    <p>From this analysis, the average percentage of subtilisin genes containing mutations that result from dGTPαs, dCTPαs, or dTTPαs misincorporation was estimated to be 90, 70, and 20 percent, respectively. These high mutagenesis frequencies were generally quite variable depending upon the dNTPαs and misincorporation efficiencies at this site. Misincorporation efficiency has been reported to be both dependent on the kind of mismatch, and the context of primer (Champoux, J. J., (1984); Skinner, J. A., et al. (1986) Nucleic Acids Res., 14, 6945-6964). Biased misincorporation efficiency of dGTPαs and dCTPαs over dTTPαs has been previously observed (Shortle, D., et al. (1985), Genetics, 110, 539-555). Unlike the dGTPαs, dCTPαs, and dTTPαs libraries the efficiency of mutagenesis for the dATPαs misincorporation library could not be accurately assessed because 90% of the restriction-resistant plasmids analyzed simply lacked the subtilisin gene insert. This problem probably arose from self-ligation of the vector when the dATPαs mutagenized subtilisin gene was subcloned from M13 into pB0180. Correcting for the vector background, we estimate the mutagenesis frequency around 20 percent in the dATPαs misincorporation library. In a separate experiment (not shown), the mutagenesis efficiencies for dGTPαs and dTTPαs misincorporation were estimated to be around 50 and 30 percent, respectively, based on the frequency of reversion of an inactivating mutation at codon 169.</p>
    <p>The location and identity of each mutation was determined by a single track of DNA sequencing corresponding to the misincorporated αthiodeoxynucleotide over the entire gene followed by a complete four track of DNA sequencing focused over the site of mutation. Of 14 mutants identified, the distribution was similar to that reported by Shortle and Lin (1985), except we did not observe nucleotide insertion or deletion mutations. The proportion of AG mutations was highest in the G misincorporation library, and some unexpected point mutations appeared in the dTTPas and dCTPas libraries.</p>
    <p>2. Screening and Identification of Alkaline Stability Mutants of Subtilisin</p>
    <p>It is possible to screen colonies producing subtilisin by halos of casein digestion (Wells, J. A. et al. (1983) Nucleic Acids Res., 11, 7911-7925). However, two problems were posed by screening colonies under high alkaline conditions (&gt;pH 11). First, B. subtilis will not grow at high pH, and we have been unable to transform an alkylophilic strain of bacillus. This problem was overcome by adopting a replica plating strategy in which colonies were grown on filters at neutral pH to produce subtilisin and filters subsequently transferred to casein plates at pH 11.5 to assay subtilisin activity. However, at pH 11.5 the casein micells no longer formed a turbid background and thus prevented a clear observation of proteolysis halos. The problem was overcome by briefly staining the plate with Coomassie blue to amplify proteolysis zones and acidifying the plates to develop casein micell turbidity. By comparison of the halo size produced on the reference growth plate (pH 7) to the high pH plate (pH 11.5), it was possible to identify mutant subtilisins that had increased (positives) or decreased (negatives) stability under alkaline conditions.</p>
    <p>Roughly 1000 colonies were screened from each of the four misincorporation libraries. The percentage of colonies showing a differential loss of activity at pH 11.5 versus pH 7 represented 1.4, 1.8, 1.4, and 0.6% of the total colonies screened from the thiol dGTPαs, dATPαs, dTTPαs, and dCTPαs libraries, respectively. Several of these negative clones were sequenced and all were found to contain a single base change as expected from the misincorporation library from which they came. Negative mutants included A36, E170 and V50. Two positive mutants were identified as V107 and R213. The ratio of negatives to positives Was roughly 50:1.</p>
    <p>3. Stability and Activity of Subtilisin Mutants at Alkaline pH</p>
    <p>Subtilisin mutants were purified and their autolytic stabilities were measured by the time course of inactivation at pH 12.0 (FIGS. 32 and 33). Positive mutants identified from the screen (i.e., V107 and R213) were more resistant to alkaline induced autolytic inactivation compared to wild-type; negative mutants (i.e., E170 and V50) were less resistant. We had advantageously produced another mutant at position 50 (F50) by site-directed mutagenesis. This mutant was more stable than wild-type enzyme to alkaline autolytic inactivation (FIG. 33) At the termination of the autolysis study, SDS-PAGE analysis confirmed that each subtilisin variant had autolyzed to an extent consistent with the remaining enzyme activity.</p>
    <p>The stabilizing effects of V107, R213, and F50 are cumulative. See Table XXI. The double mutant, V107/R213 (made by subcloning the 920 bp EcoRI-KpnI fragment of pB0180V107 into the 6.6 kb EcoRI-KpnI fragment of pB0180R213), is more stable than either single mutant. The triple mutant, F50/V107/R213 (made by subcloning the 735 bp EcoRI-PvuII fragment of pF50 (Example 2) into the 6.8 kb EcoRI-PvuII fragment of pB0180/V107, is more stable than the double mutant V107/R213 or F50. The inactivation curves show a biphasic character that becomes more pronounced the more stable the mutant analyzed. This may result from some destablizing chemical modification(s) (eg., deamidation) during the autolysis study and/or reduced stabilization caused by complete digestion of larger autolysis peptides. These alkaline autolysis studies have been repeated on separately purified enzyme batches with essentially the same results. Rates of autolysis should depend both on the conformational stability as well as the specific activity of the subtilisin variant (Wells, J. A., et al. (1986), J. Biol. Chem., 261, 6564-6570). It was therefore possible that the decreases in autolytic inactivation rates may result from decreases in specific activity of the more stable mutant under alkaline conditions. In general the opposite appears to be the case. The more stable mutants, if anything, have a relatively higher specific activity than wild-type under alkaline conditions and the less stable mutants have a relatively lower specific activity. These subtle effects on specific activity for V107/R213 and F50/V107/R213 are cumulative at both pH 8.6 and 10.8. The changes in specific activity may reflect slight differences in substrate specificity, however, it is noteworthy that only positions 170 and 107 are within 6 Å of a bound model substrate (Robertus, J. D., et al. (1972), Biochemistry 11, 2438-2449).</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">              TABLE XXI______________________________________Relationship between relative specific acitivityat pH 8.6 or 10.8 and alkaline autolytic stability                  Alkaline                  autolysis     Relative specific activity                  half-timeEnzyme    pH 8.6       pH 10.8 (min)<sup>b</sup>______________________________________Wild-type 100 ± 1   100 ± 3                           86Q170       46 ± 1    28 ± 2                           13V107      126 ± 3    99 ± 5                          102R213       97 ± 1   102 ± 1                          115V107/R213 116 ± 2   106 ± 3                          130V50        66 ± 4    61 ± 1                           58F50       123 ± 3   157 ± 7                          131F50/V107/ 126 ± 2   152 ± 3                          168R213______________________________________ .sup.(a) Relative specific activity was the average from triplicate activity determinations divided by the wildtype value at the same pH. The average specific activity of wildtype enzyme at pH 8.6 and 10.8 was 70 μmo1es/min-moles/minmg and 37 μmoles/minmg, respectively. .sup.(b) Time to reach 50% activity was taken from FIGS. 32 and 33.</pre>
    
    <p>F. Random Cassette Mutagenesis of Residues 197 through 228</p>
    <p>Plasmid pΔ222 (Wells, et al. (1985) Gene 34, 315-323) was digested with PstI and BamHI and the 0.4 kb PstI/BamHI fragment (fragment 1, see FIG. 34) purified from a polyacrylamide gel by electroelution (Maniatis, et al. (1982) Molecular Cloning, A Laboratory Manual. Cold Spring Harbor Laboratory, Cold Spring Harbor, N.Y.).</p>
    <p>The 1.5 kb EcoRI/BamHI fragment from pS4.5 was cloned into M13mp9. Site directed mutagenesis was performed to create the A197 mutant and simultaneously insert a silent SstI site over codons 195-196. The mutant EcoRI/BamHI fragment was cloned back into pBS42. The A197 plasmid was digested with BamHI and SstI and the 5.3 kb BamHI/SstI fragment (fragment 2) was purified from low melting agarose.</p>
    <p>Complimentary oligonucleotides were synthesized to span the region from SstI (codons 195-196) to PstI (codons 228-230). These oligodeoxynucleotides were designed to (1) restore codon 197 to the wild type, (2) re-create a silent KpnI site present in pΔ222 at codons 219-220, (3) create a silent SmaI site over codons 210-211, and (4) eliminate the PstI site over codons 228-230 (see FIG. 35). Oligodeoxynucleotides were synthesized with 2% contaminating nucleotides at each cycle of synthesis, e.g., dATP reagent was spiked with 2% dCTP, 2% dGTP, and 2% dTTP. For 97-mers, this 2% poisoning should give the following percentages of non-mutant, single mutants and double or higher mutants per strand with two or more misincorporations per complimentary strand: 14% non-mutant, 28% single mutant, and 57% with ≧2 mutations, according to the general formula ##EQU2## where μ is the average number of mutations and n is a number class of mutations and f is the fraction of the total having that number of mutations. Complimentary oligodeoxynucleotide pools were phosphorylated and annealed (fragment 3) and then ligated at 2-fold molar excess over fragments 1 and 2 in a three-way ligation.</p>
    <p>E. coli MM294 was transformed with the ligation reaction, the transformation pool grown up over night and the pooled plasmid DNA was isolated. This pool represented 3.4×10<sup>4</sup> independent transformants. This plasmid pool was digested with PstI and then used to retransform E. coli. A second plasmid pool was prepared and used to transform B. subtilis (BG2036). Approximately 40% of the BG2036 transformants actively expressed subtilisin as judged by halo-clearing on casein plates. Several of the non-expressing transformants were sequenced and found to have insertions or deletions in the synthetic cassettes. Expressing BG2036 mutants were arrayed in microtiter dishes with 150 μl of LB/12.5 μg/mL chloramphenicol (cmp) per well, incubated at 37° C. for 3-4 hours and then stamped in duplicate onto nitrocellulose filters laid on LB 1.5% skim milk/5 μg/mL cmp plates and incubated overnight at 33° C. (until halos were approximately 4-8 mm in diameter). Filters were then lifted to stacks of filter paper saturated with 1 x Tide commercial grade detergent, 50 mM Na<sub>2</sub> CO<sub>3</sub>, pH 11.5 and incubated at 65° C. for 90 min. Overnight growth plates were Commassis stained and destained to establish basal levels of expression. After this treatment, filters were returned to pH 7/skim milk/20 μg/mL tetracycline plates and incubated at 37° C. for 4 hours to overnight.</p>
    <p>Mutants identified by the high pH stability screen to be more alkaline stable Here purified and analyzed for autolytic stability at high pH or high temperature. The double mutant C204/R213 was more stable than wild type at either high pH or high temperature (Table XXII).</p>
    <p>This mutant was dissected into single mutant parents (C204 and R213) by cutting at the unique SmaI restriction site (FIG. 35) and either ligating wild type sequence 3' to the SmaI site to create the single C204 mutant or ligating wild type sequence 5' to the SmaI site to create the single R213 mutant. Of the two single parents, C204 was nearly as alkaline stable as the parent double mutant (C204/R213) and slightly more thermally stable. See Table XXII. The R213 mutant was only slightly more stable than wild type under both conditions (not shown).</p>
    <p>Another mutant identified from the screen of the 197 to 228 random cassette mutagenesis was R204. This mutant was more stable than wild type at both high pH and high temperature but less stable than C204.</p>
    <heading>TABLE XXII</heading> <heading>Stability of subtilisin Variants</heading> <p>Purified enzymes (200 μg/mL) were incubated in 0.1M phosphate, pH 12 at 30° C. for alkaline autolysis, or in 2 mM CaCl<sub>2</sub>, 50 mM MOPS, pH 7.0 at 62° C. for thermal autolysis. At various times samples were assayed for residual enzyme activity. Inactivations were roughly pseudo-first order, and t 1/2 gives the time it took to reach 50% of the starting activity in two separate experiments.</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">______________________________________       t 1/2          t 1/2       (alkaline      (therinal       autolysis)     autolysis)         Exp.   Exp.      Exp. Exp.Subtilisin variant         #1     #2        #1   #2______________________________________wild type     30     25        20   23F50/V107/R213 49     41        18   23R204          35     32        24   27C204          43     46        38   40C204/R213     50     52        32   36L204/R213     32     30        20   21______________________________________</pre>
    
    <p>G. Random Mutagenesis at Codon 204</p>
    <p>Based on the above results, codon 204 was targeted for random mutagenesis. Mutagenic DNA cassettes (for codon at 204) all contained a fixed R213 mutation which was found to slightly augment the stability of the C204 mutant.</p>
    <p>Plasmid DNA encoding the subtilisin mutant C204/R213 was digested with SstI and EcoRI and a 1.0 kb EcoRI/SstI fragment was isolated by electro-elution from polyacrylamide gel (fragment 1, see FIG. 35).</p>
    <p>C204/R213 was also digested with SmaI and EcoRI and the large 4.7 kb fragment, including vector sequences and the 3' portion of coding region, was isolated from low melting agarose (fragment 2, see FIG. 36).</p>
    <p>Fragments 1 and 2 were combined in four separate three-way ligations with heterophosphorylated fragments 3 (see FIGS. 36 and 37). This hetero-phosphorylation of synthetic duplexes should preferentially drive the phosphorylated strand into the plasmid ligation product. Four plasmid pools, corresponding to the four ligations, were restricted with SmaI in order to linearize any single cut C204/R213 present from fragment 2 isolation, thus reducing the background of C204/R213. E. coli was then re-transformed with SmaI-restricted plasmid pools to yield a second set of plasmid pools which are essentially free of C204/R213 and any non-segregated heterduplex material.</p>
    <p>These second enriched plasmid pools were then used to transform B. subtilis (BG2036) and the resulting four mutant pools were screened for clones expressing subtilisin resistant to high pH/temperature inactivation. Mutants found positive by such a screen were further characterized and identified by sequencing.</p>
    <p>The mutant L204/R213 was found to be slightly more stable than the wild type subtilisin. See Table XXII.</p>
    <p>Having described the preferred embodiments of the present invention, it will appear to those ordinarily skilled in the art that various modifications may be made to the disclosed embodiments, and that such modifications are intended to be within the scope of the present invention.</p>
    </div></div></div><div class="patent-section patent-tabular-section"><a id="backward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Cited Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4002572">US4002572</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Nov 18, 1975</td><td class="patent-data-table-td patent-date-value">Jan 11, 1977</td><td class="patent-data-table-td ">Gist-Brocades N.V.</td><td class="patent-data-table-td ">Alkaline protease produced by a bacillus</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4458066">US4458066</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Mar 24, 1981</td><td class="patent-data-table-td patent-date-value">Jul 3, 1984</td><td class="patent-data-table-td ">University Patents, Inc.</td><td class="patent-data-table-td ">Process for preparing polynucleotides</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4543329">US4543329</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jul 6, 1982</td><td class="patent-data-table-td patent-date-value">Sep 24, 1985</td><td class="patent-data-table-td ">Schering Aktiengesellschaft</td><td class="patent-data-table-td ">Process for the specific cleavage of protein sequences from proteins</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4760025">US4760025</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">May 29, 1984</td><td class="patent-data-table-td patent-date-value">Jul 26, 1988</td><td class="patent-data-table-td ">Genencor, Inc.</td><td class="patent-data-table-td ">Subtilisin enzyme having genetic enginereed amino acids at specific positions</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/USRE34606">USRE34606</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jul 20, 1990</td><td class="patent-data-table-td patent-date-value">May 10, 1994</td><td class="patent-data-table-td ">Genencor, Inc.</td><td class="patent-data-table-td ">Modified enzymes and methods for making same</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1985003949A1?cl=en">WO1985003949A1</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Mar 6, 1985</td><td class="patent-data-table-td patent-date-value">Sep 12, 1985</td><td class="patent-data-table-td ">Harvard College</td><td class="patent-data-table-td ">Hosts and methods for producing recombinant products in high yields</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="npl-citations"></a><div class="patent-section-header"><span class="patent-section-title">Non-Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th colspan="3"class="patent-data-table-th">Reference</th></tr></thead><tr><td class="patent-data-table-td ">1</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Abrahmsen, L., et al., "<a href='http://scholar.google.com/scholar?q="Engineering+Subtilisin+and+Its+Substrates+for+Efficient+Ligation+of+Peptide+Bonds+in+Aqueous+Solution."'>Engineering Subtilisin and Its Substrates for Efficient Ligation of Peptide Bonds in Aqueous Solution.</a>" Biochemistry, 30:4151-4159 (1991).</td></tr><tr><td class="patent-data-table-td ">2</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Abrahmsen, L., et al., Engineering Subtilisin and Its Substrates for Efficient Ligation of Peptide Bonds in Aqueous Solution. Biochemistry, 30:4151 4159 (1991).</td></tr><tr><td class="patent-data-table-td ">3</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Allinger, et al., in Organic Chemistry, Worth Publishers, New York, NY p. 757 (1971).</td></tr><tr><td class="patent-data-table-td ">4</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Baughman, G., et al., "<a href='http://scholar.google.com/scholar?q="Translational+regulation+of+the+L11+ribosomal+protein+operon+of+Escherichia+coli%3A+Analysis+of+the+mRNA+target+site+using+olignucleotide-directed+mutagenesis."'>Translational regulation of the L11 ribosomal protein operon of Escherichia coli: Analysis of the mRNA target site using olignucleotide-directed mutagenesis.</a>" Proc. Natl. Acad. Sci. USA, 81:5389-5393 (1984).</td></tr><tr><td class="patent-data-table-td ">5</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Baughman, G., et al., Translational regulation of the L11 ribosomal protein operon of Escherichia coli: Analysis of the mRNA target site using olignucleotide directed mutagenesis. Proc. Natl. Acad. Sci. USA, 81:5389 5393 (1984).</td></tr><tr><td class="patent-data-table-td ">6</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Cantor, C.R., et al., "<a href='http://scholar.google.com/scholar?q="Biophysical+Chemistry%3A+Part+I%3A+The+conformation+of+biological+macromolecules."'>Biophysical Chemistry: Part I: The conformation of biological macromolecules.</a>" W.H. Freeman and Company, San Francisco, pp. 3-154, 253-309 (1980).</td></tr><tr><td class="patent-data-table-td ">7</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Cantor, C.R., et al., Biophysical Chemistry: Part I: The conformation of biological macromolecules. W.H. Freeman and Company, San Francisco, pp. 3 154, 253 309 (1980).</td></tr><tr><td class="patent-data-table-td ">8</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Carter, et al., Science, 237:394 399 (1987).</td></tr><tr><td class="patent-data-table-td ">9</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Carter, et al., Science, 237:394-399 (1987).</td></tr><tr><td class="patent-data-table-td ">10</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Carter, P., et al., "<a href='http://scholar.google.com/scholar?q="Improved+Oligonucleotide+Site-Directed+Mutagenesis+Using+M13+Vectors."'>Improved Oligonucleotide Site-Directed Mutagenesis Using M13 Vectors.</a>" Nucleic Acids Res., 13:4431-4443 (1985).</td></tr><tr><td class="patent-data-table-td ">11</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Carter, P., et al., Improved Oligonucleotide Site Directed Mutagenesis Using M13 Vectors. Nucleic Acids Res., 13:4431 4443 (1985).</td></tr><tr><td class="patent-data-table-td ">12</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Carter, P.J., et al., "<a href='http://scholar.google.com/scholar?q="The+Use+of+Double+Mutants+to+Detect+Structural+Changes+in+the+Active+Site+of+the+Tyrosyl-tRNA+Synthetase+%28Bacillus+stearothermophilus%29."'>The Use of Double Mutants to Detect Structural Changes in the Active Site of the Tyrosyl-tRNA Synthetase (Bacillus stearothermophilus).</a>" Cell, 33:835-840 (1984).</td></tr><tr><td class="patent-data-table-td ">13</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Carter, P.J., et al., Cell, 38:835 840 (1984).</td></tr><tr><td class="patent-data-table-td ">14</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Carter, P.J., et al., Cell, 38:835-840 (1984).</td></tr><tr><td class="patent-data-table-td ">15</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Carter, P.J., et al., The Use of Double Mutants to Detect Structural Changes in the Active Site of the Tyrosyl tRNA Synthetase ( Bacillus stearothermophilus ). Cell, 33:835 840 (1984).</td></tr><tr><td class="patent-data-table-td ">16</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Craik, C.S., et al., Science, 228:291 297 (1985).</td></tr><tr><td class="patent-data-table-td ">17</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Craik, C.S., et al., Science, 228:291-297 (1985).</td></tr><tr><td class="patent-data-table-td ">18</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Drenth, J., et al., Eur. J. Biochem., 26, 177 181 (1972).</td></tr><tr><td class="patent-data-table-td ">19</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Drenth, J., et al., Eur. J. Biochem., 26, 177-181 (1972).</td></tr><tr><td class="patent-data-table-td ">20</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Estell, D.A., et al., "<a href='http://scholar.google.com/scholar?q="Probing+Steric+and+Hydrophobic+Effects+on+Enzyme-Substrate+Interactions+by+Protein+Engineering."'>Probing Steric and Hydrophobic Effects on Enzyme-Substrate Interactions by Protein Engineering.</a>" Science, 233:659-663 (1986).</td></tr><tr><td class="patent-data-table-td ">21</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Estell, D.A., et al., Probing Steric and Hydrophobic Effects on Enzyme Substrate Interactions by Protein Engineering. Science, 233:659 663 (1986).</td></tr><tr><td class="patent-data-table-td ">22</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Fersht, A., Enzyme Structure and Mechanism, Freeman, San Francisco. pp. 226 287 (1977).</td></tr><tr><td class="patent-data-table-td ">23</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Fersht, A., Enzyme Structure and Mechanism, Freeman, San Francisco. pp. 226-287 (1977).</td></tr><tr><td class="patent-data-table-td ">24</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Gardell, et al., Nature, 317:551 555 (1985).</td></tr><tr><td class="patent-data-table-td ">25</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Gardell, et al., Nature, 317:551-555 (1985).</td></tr><tr><td class="patent-data-table-td ">26</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Grantham, R., "<a href='http://scholar.google.com/scholar?q="Amino+Acid+Difference+Formula+to+Help+Explain+Protein+Evolution."'>Amino Acid Difference Formula to Help Explain Protein Evolution.</a>" Science, 185:962-964 (1974).</td></tr><tr><td class="patent-data-table-td ">27</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Grantham, R., Amino Acid Difference Formula to Help Explain Protein Evolution. Science, 185:962 964 (1974).</td></tr><tr><td class="patent-data-table-td ">28</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Inouye, S., et al., "<a href='http://scholar.google.com/scholar?q="Effects+of+Mutations+at+Glycine+Residues+in+the+Hydrophobic+Region+of+the+Escherichia+coli+Prolipoprotein+Signal+Peptide+on+the+Secretion+across+the+Membrane."'>Effects of Mutations at Glycine Residues in the Hydrophobic Region of the Escherichia coli Prolipoprotein Signal Peptide on the Secretion across the Membrane.</a>" J. Biol. Chem., 259:3729-3733 (1984).</td></tr><tr><td class="patent-data-table-td ">29</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Inouye, S., et al., "<a href='http://scholar.google.com/scholar?q="Role+of+positive+charge+on+the+amino-terminal+region+of+the+signal+peptide+in+protein+secretion+across+the+membrane."'>Role of positive charge on the amino-terminal region of the signal peptide in protein secretion across the membrane.</a>" Proc. Natl. Acad. Sci. USA, 79:3438-3441 (1982).</td></tr><tr><td class="patent-data-table-td ">30</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Inouye, S., et al., Effects of Mutations at Glycine Residues in the Hydrophobic Region of the Escherichia coli Prolipoprotein Signal Peptide on the Secretion across the Membrane. J. Biol. Chem., 259:3729 3733 (1984).</td></tr><tr><td class="patent-data-table-td ">31</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Inouye, S., et al., Role of positive charge on the amino terminal region of the signal peptide in protein secretion across the membrane. Proc. Natl. Acad. Sci. USA, 79:3438 3441 (1982).</td></tr><tr><td class="patent-data-table-td ">32</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Iverson, et al., Science, 243:1184 1188 (1989).</td></tr><tr><td class="patent-data-table-td ">33</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Iverson, et al., Science, 243:1184-1188 (1989).</td></tr><tr><td class="patent-data-table-td ">34</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Jacobs, M., et al., "<a href='http://scholar.google.com/scholar?q="Cloning%2C+sequencing+and+expression+of+subtilisin+Carlsberg+from+Bacillus+licheniformis."'>Cloning, sequencing and expression of subtilisin Carlsberg from Bacillus licheniformis.</a>" Nucleic Acids Res., 13: 8913-8926 (1985).</td></tr><tr><td class="patent-data-table-td ">35</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Jacobs, M., et al., Cloning, sequencing and expression of subtilisin Carlsberg from Bacillus licheniformis. Nucleic Acids Res., 13: 8913 8926 (1985).</td></tr><tr><td class="patent-data-table-td ">36</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Jaye, M., et al., "<a href='http://scholar.google.com/scholar?q="Isolation+of+a+human+anti-haemophilic+factor+IX+cDNA+clone+using+a+unique+52-base+synthetic+oligonucleotide+probe+deduced+from+the+amino+acid+sequence+of+bovine+factor+IX."'>Isolation of a human anti-haemophilic factor IX cDNA clone using a unique 52-base synthetic oligonucleotide probe deduced from the amino acid sequence of bovine factor IX.</a>" Nucleic Acids Res., 11:2325-2335 (1983).</td></tr><tr><td class="patent-data-table-td ">37</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Jaye, M., et al., Isolation of a human anti haemophilic factor IX cDNA clone using a unique 52 base synthetic oligonucleotide probe deduced from the amino acid sequence of bovine factor IX. Nucleic Acids Res., 11:2325 2335 (1983).</td></tr><tr><td class="patent-data-table-td ">38</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Jencks, W.P., Catalysis in Chemistry and Enzymology, McGraw Hill, pp. 321 436 (1969).</td></tr><tr><td class="patent-data-table-td ">39</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Jencks, W.P., Catalysis in Chemistry and Enzymology, McGraw-Hill, pp. 321-436 (1969).</td></tr><tr><td class="patent-data-table-td ">40</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Kaiser, E.T., et al., Ann. Rev. Biochem., 54:565 595 (1985).</td></tr><tr><td class="patent-data-table-td ">41</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Kaiser, E.T., et al., Ann. Rev. Biochem., 54:565-595 (1985).</td></tr><tr><td class="patent-data-table-td ">42</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Kerjan, P., et al., Eur. J. Biochem., 98:353 362 (1979).</td></tr><tr><td class="patent-data-table-td ">43</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Kerjan, P., et al., Eur. J. Biochem., 98:353-362 (1979).</td></tr><tr><td class="patent-data-table-td ">44</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Kraut, J., Ann. Rev. Biochem., 46:331 358 (1977).</td></tr><tr><td class="patent-data-table-td ">45</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Kraut, J., Ann. Rev. Biochem., 46:331-358 (1977).</td></tr><tr><td class="patent-data-table-td ">46</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Kurihara, M., et al., "<a href='http://scholar.google.com/scholar?q="Isolation+and+Sequence+of+the+Chymotryptic+Peptides+and+the+Complete+Amino+Acid+Sequence."'>Isolation and Sequence of the Chymotryptic Peptides and the Complete Amino Acid Sequence.</a>" J. Biol. Chem., 247:5619-5631 (1972).</td></tr><tr><td class="patent-data-table-td ">47</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Kurihara, M., et al., Isolation and Sequence of the Chymotryptic Peptides and the Complete Amino Acid Sequence. J. Biol. Chem., 247:5619 5631 (1972).</td></tr><tr><td class="patent-data-table-td ">48</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Markland, F.S., Carlsberg Res. Comm., 41:237 291 (1976).</td></tr><tr><td class="patent-data-table-td ">49</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Markland, F.S., Carlsberg Res. Comm., 41:237-291 (1976).</td></tr><tr><td class="patent-data-table-td ">50</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Markland, F.S., et al., The Enzymes, ed. Boyer, P.D., Acad Press, New York, vol. III, pp. 561 608 (1971).</td></tr><tr><td class="patent-data-table-td ">51</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Markland, F.S., et al., The Enzymes, ed. Boyer, P.D., Acad Press, New York, vol. III, pp. 561-608 (1971).</td></tr><tr><td class="patent-data-table-td ">52</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Matthews, D.A., et al., J. Biol. Chem., 250:7120 7126 (1975).</td></tr><tr><td class="patent-data-table-td ">53</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Matthews, D.A., et al., J. Biol. Chem., 250:7120-7126 (1975).</td></tr><tr><td class="patent-data-table-td ">54</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Mattoccia, E., et al., "<a href='http://scholar.google.com/scholar?q="Mutation+in+the+A+Block+of+the+Yeast+tRNALeu3+Gene+that+Allows+Transcription+but+Abolishes+Splicing+and+5%27-End+Maturation."'>Mutation in the A Block of the Yeast tRNALeu3 Gene that Allows Transcription but Abolishes Splicing and 5'-End Maturation.</a>" Cell, 32:67-76 (1983).</td></tr><tr><td class="patent-data-table-td ">55</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Mattoccia, E., et al., Mutation in the A Block of the Yeast tRNA Leu 3 Gene that Allows Transcription but Abolishes Splicing and 5 End Maturation. Cell, 32:67 76 (1983).</td></tr><tr><td class="patent-data-table-td ">56</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Meloun, B., et al., FEBS Letters, vol. 183, "<a href='http://scholar.google.com/scholar?q="Complete+primary+structure+of+thermitase+from+Thermoactinomyces+vulgaris+and+its+structural+features+related+to+the+subtilisin-type+proteinases"'>Complete primary structure of thermitase from Thermoactinomyces vulgaris and its structural features related to the subtilisin-type proteinases</a>", pp. 195-199, 1985.</td></tr><tr><td class="patent-data-table-td ">57</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Meloun, B., et al., FEBS Letters, vol. 183, Complete primary structure of thermitase from Thermoactinomyces vulgaris and its structural features related to the subtilisin type proteinases , pp. 195 199, 1985.</td></tr><tr><td class="patent-data-table-td ">58</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Nedkov, P., et al., Hoppe Seyler s Z. Physiol. Chem., 364:1537 1540 (1983).</td></tr><tr><td class="patent-data-table-td ">59</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Nedkov, P., et al., Hoppe-Seyler's Z. Physiol. Chem., 364:1537-1540 (1983).</td></tr><tr><td class="patent-data-table-td ">60</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Ottesen, et al., Meth. Enzymol., XIX:199 215 (1970).</td></tr><tr><td class="patent-data-table-td ">61</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Ottesen, et al., Meth. Enzymol., XIX:199-215 (1970).</td></tr><tr><td class="patent-data-table-td ">62</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Pahler, A., et al., The EMBO Journal, vol. 3, "<a href='http://scholar.google.com/scholar?q="Three-dimensional+structure+of+fungal+proteinase+K+reveals+similarity+to+bacterial+subtilisin"'>Three-dimensional structure of fungal proteinase K reveals similarity to bacterial subtilisin</a>", pp. 1311-1314, 1984.</td></tr><tr><td class="patent-data-table-td ">63</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Pahler, A., et al., The EMBO Journal, vol. 3, Three dimensional structure of fungal proteinase K reveals similarity to bacterial subtilisin , pp. 1311 1314, 1984.</td></tr><tr><td class="patent-data-table-td ">64</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Paterson, A. and Clarke, P.H., J. Gen. Micro. 114:65 85 (1979).</td></tr><tr><td class="patent-data-table-td ">65</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Paterson, A. and Clarke, P.H., J. Gen. Micro. 114:65-85 (1979).</td></tr><tr><td class="patent-data-table-td ">66</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Perry, L.J., et al., Science, 226:555 557 (1984).</td></tr><tr><td class="patent-data-table-td ">67</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Perry, L.J., et al., Science, 226:555-557 (1984).</td></tr><tr><td class="patent-data-table-td ">68</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Philipp, M., et al., Mol. Cell. Biochem., 51:5 32 (1983).</td></tr><tr><td class="patent-data-table-td ">69</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Philipp, M., et al., Mol. Cell. Biochem., 51:5-32 (1983).</td></tr><tr><td class="patent-data-table-td ">70</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Polgar, et al., Adv. Enzymol., 33:381 400 (1970).</td></tr><tr><td class="patent-data-table-td ">71</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Polgar, et al., Adv. Enzymol., 33:381-400 (1970).</td></tr><tr><td class="patent-data-table-td ">72</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Polgar, L. and Sajgo, M., Biochem. Biophy. Acta., 667:351 354 (1981).</td></tr><tr><td class="patent-data-table-td ">73</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Polgar, L. and Sajgo, M., Biochem. Biophy. Acta., 667:351-354 (1981).</td></tr><tr><td class="patent-data-table-td ">74</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Poulos, T.L., et al., J. Biol. Chem., 251:1097 1103 (1976).</td></tr><tr><td class="patent-data-table-td ">75</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Poulos, T.L., et al., J. Biol. Chem., 251:1097-1103 (1976).</td></tr><tr><td class="patent-data-table-td ">76</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Rastetter, W.H., "<a href='http://scholar.google.com/scholar?q="Enzyme+engineering%3A+applications+and+promise."'>Enzyme engineering: applications and promise.</a>" Trends Biotechnol., 1:80-84 (1983).</td></tr><tr><td class="patent-data-table-td ">77</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Rastetter, W.H., Enzyme engineering: applications and promise. Trends Biotechnol., 1:80 84 (1983).</td></tr><tr><td class="patent-data-table-td ">78</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Ricchelli, F., et al., "<a href='http://scholar.google.com/scholar?q="Effects+of+pH+and+Urea+on+the+Conformational+Properties+of+Subtilisin+DY."'>Effects of pH and Urea on the Conformational Properties of Subtilisin DY.</a>" Biochemical J., 207:201-205 (1982).</td></tr><tr><td class="patent-data-table-td ">79</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Ricchelli, F., et al., Effects of pH and Urea on the Conformational Properties of Subtilisin DY. Biochemical J., 207:201 205 (1982).</td></tr><tr><td class="patent-data-table-td ">80</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Robertus, J.D., et al., "<a href='http://scholar.google.com/scholar?q="An+X-Ray+Crystallographic+Study+of+the+Binding+of+Peptide+Chloromethyl+Ketone+INhibitors+to+Subtilisin+BPN%27%2C"'>An X-Ray Crystallographic Study of the Binding of Peptide Chloromethyl Ketone INhibitors to Subtilisin BPN',</a>" Biochemistry, 11:2439-2449 (1972).</td></tr><tr><td class="patent-data-table-td ">81</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Robertus, J.D., et al., An X Ray Crystallographic Study of the Binding of Peptide Chloromethyl Ketone INhibitors to Subtilisin BPN , Biochemistry, 11:2439 2449 (1972).</td></tr><tr><td class="patent-data-table-td ">82</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Robertus, J.D., et al., Biochemistry, 11:4293 4303 (1972).</td></tr><tr><td class="patent-data-table-td ">83</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Robertus, J.D., et al., Biochemistry, 11:4293-4303 (1972).</td></tr><tr><td class="patent-data-table-td ">84</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Russell, A. J., et al., Journal of Molecular Biology, vol. 193, "<a href='http://scholar.google.com/scholar?q="Electrostatic+effects+on+modification+of+charged+groups+in+the+active+site+cleft+of+subtilisin+by+protein+engineering"'>Electrostatic effects on modification of charged groups in the active site cleft of subtilisin by protein engineering</a>", pp. 803-813, 1987.</td></tr><tr><td class="patent-data-table-td ">85</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Russell, A. J., et al., Journal of Molecular Biology, vol. 193, Electrostatic effects on modification of charged groups in the active site cleft of subtilisin by protein engineering , pp. 803 813, 1987.</td></tr><tr><td class="patent-data-table-td ">86</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Russell, A. J., et al., Nature, vol. 328, "<a href='http://scholar.google.com/scholar?q="Rational+modification+of+enzyme+catalysis+by+engineering+surface+charge"'>Rational modification of enzyme catalysis by engineering surface charge</a>", pp. 496-500, 1987.</td></tr><tr><td class="patent-data-table-td ">87</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Russell, A. J., et al., Nature, vol. 328, Rational modification of enzyme catalysis by engineering surface charge , pp. 496 500, 1987.</td></tr><tr><td class="patent-data-table-td ">88</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Shortle, D., et al., "<a href='http://scholar.google.com/scholar?q="Gap+misrepair+mutagenesis%3A+Efficient+site-directed+induction+of+transition%2C+transversion%2C+and+frameshift+mutations+in+vitro."'>Gap misrepair mutagenesis: Efficient site-directed induction of transition, transversion, and frameshift mutations in vitro.</a>" Proc. Natl. Acad. Sci. USA., 79:1588-1592 (1982).</td></tr><tr><td class="patent-data-table-td ">89</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Shortle, D., et al., Gap misrepair mutagenesis: Efficient site directed induction of transition, transversion, and frameshift mutations in vitro. Proc. Natl. Acad. Sci. USA., 79:1588 1592 (1982).</td></tr><tr><td class="patent-data-table-td ">90</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Singleton, et al., Dictionary of Microbiology and Molecular Biology, Second Edition. John Wiley and Sons. Chichester. p. 741 (1987).</td></tr><tr><td class="patent-data-table-td ">91</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Smith, E.L., "<a href='http://scholar.google.com/scholar?q="The+Complete+Sequence%3B+Comparison+with+Subtilisin+BPN%3B+Evolutionary+Relationships."'>The Complete Sequence; Comparison with Subtilisin BPN; Evolutionary Relationships.</a>" J. Biol. Chem., 243:2184-2191 (1968).</td></tr><tr><td class="patent-data-table-td ">92</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Smith, E.L., The Complete Sequence; Comparison with Subtilisin BPN; Evolutionary Relationships. J. Biol. Chem., 243:2184 2191 (1968).</td></tr><tr><td class="patent-data-table-td ">93</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Stahl, M.L., et al., "<a href='http://scholar.google.com/scholar?q="Replacement+of+the+Bacillus+subtilis+Subtilisin+Structural+Gene+with+and+In+Vitro-Derived+Deletion+Mutation."'>Replacement of the Bacillus subtilis Subtilisin Structural Gene with and In Vitro-Derived Deletion Mutation.</a>" J. Bacteriol., 158:411-418 (1984).</td></tr><tr><td class="patent-data-table-td ">94</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Stahl, M.L., et al., Replacement of the Bacillus subtilis Subtilisin Structural Gene with and In Vitro Derived Deletion Mutation. J. Bacteriol., 158:411 418 (1984).</td></tr><tr><td class="patent-data-table-td ">95</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Stauffer, C.E., et al., "<a href='http://scholar.google.com/scholar?q="The+Effect+on+Subtilisin+Activity+of+Oxidizing+a+Methionine+Residue."'>The Effect on Subtilisin Activity of Oxidizing a Methionine Residue.</a>" J. Biol. Chem. 244:5333-5338 (1969).</td></tr><tr><td class="patent-data-table-td ">96</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Stauffer, C.E., et al., The Effect on Subtilisin Activity of Oxidizing a Methionine Residue. J. Biol. Chem. 244:5333 5338 (1969).</td></tr><tr><td class="patent-data-table-td ">97</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Stroud, R.M., Sci. Amer., 131:74 88 (1974).</td></tr><tr><td class="patent-data-table-td ">98</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Stroud, R.M., Sci. Amer., 131:74-88 (1974).</td></tr><tr><td class="patent-data-table-td ">99</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Svendsen, I., "<a href='http://scholar.google.com/scholar?q="Chemical+Modifications+of+the+Subtilisins+with+Special+Reference+to+the+Binding+of+Large+Substrates.+A+Review."'>Chemical Modifications of the Subtilisins with Special Reference to the Binding of Large Substrates. A Review.</a>" Carlsberg Res. Commun., 41:237-291 (1976).</td></tr><tr><td class="patent-data-table-td ">100</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Svendsen, I., Chemical Modifications of the Subtilisins with Special Reference to the Binding of Large Substrates. A Review. Carlsberg Res. Commun., 41:237 291 (1976).</td></tr><tr><td class="patent-data-table-td ">101</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Svendsen, I., et al., "<a href='http://scholar.google.com/scholar?q="Complete+Amino+Acid+Sequence+of+Alkaline+Mesentericopeptidase%3A+A+subtilisin+isolate+from+a+strain+of+Bacillus+mesentericus."'>Complete Amino Acid Sequence of Alkaline Mesentericopeptidase: A subtilisin isolate from a strain of Bacillus mesentericus.</a>" FEBS, 196:228-232 (1986).</td></tr><tr><td class="patent-data-table-td ">102</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Svendsen, I., et al., Complete Amino Acid Sequence of Alkaline Mesentericopeptidase: A subtilisin isolate from a strain of Bacillus mesentericus. FEBS, 196:228 232 (1986).</td></tr><tr><td class="patent-data-table-td ">103</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Thomas, et al., Nature, 318:37 38 (1985).</td></tr><tr><td class="patent-data-table-td ">104</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Thomas, et al., Nature, 318:37-38 (1985).</td></tr><tr><td class="patent-data-table-td ">105</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Toney et al., Science, 243:1485 1488 (1989).</td></tr><tr><td class="patent-data-table-td ">106</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Toney et al., Science, 243:1485-1488 (1989).</td></tr><tr><td class="patent-data-table-td ">107</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Uehara, H., et al., J. Bacteriology, 139:583 590 (1979).</td></tr><tr><td class="patent-data-table-td ">108</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Uehara, H., et al., J. Bacteriology, 139:583-590 (1979).</td></tr><tr><td class="patent-data-table-td ">109</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Ulmer, K.M. Science, 219:666 671 (1983).</td></tr><tr><td class="patent-data-table-td ">110</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Ulmer, K.M. Science, 219:666-671 (1983).</td></tr><tr><td class="patent-data-table-td ">111</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Vasantha, et al., J. Bacteriol., 165:837 842 (1986).</td></tr><tr><td class="patent-data-table-td ">112</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Vasantha, et al., J. Bacteriol., 165:837-842 (1986).</td></tr><tr><td class="patent-data-table-td ">113</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Vasantha, N., et al., "<a href='http://scholar.google.com/scholar?q="Genes+for+Alkaline+Protease+and+Neutral+Protease+from+Bacillus+amyloliquefaciens+Contain+a+Large+Open+Reading+Frame+Between+the+Regions+Coding+for+Signal+Sequence+and+Mature+Protein."'>Genes for Alkaline Protease and Neutral Protease from Bacillus amyloliquefaciens Contain a Large Open Reading Frame Between the Regions Coding for Signal Sequence and Mature Protein.</a>" J. Bacteriol., 159:811-819 (1984).</td></tr><tr><td class="patent-data-table-td ">114</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Vasantha, N., et al., Genes for Alkaline Protease and Neutral Protease from Bacillus amyloliquefaciens Contain a Large Open Reading Frame Between the Regions Coding for Signal Sequence and Mature Protein. J. Bacteriol., 159:811 819 (1984).</td></tr><tr><td class="patent-data-table-td ">115</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Villafranca, D.E., et al., Science, 222:782 788 (1983).</td></tr><tr><td class="patent-data-table-td ">116</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Villafranca, D.E., et al., Science, 222:782-788 (1983).</td></tr><tr><td class="patent-data-table-td ">117</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Wallace, B.R., et al., "<a href='http://scholar.google.com/scholar?q="Oligonucleotide+directed+mutagenesis+of+the+human+%CE%B2-globin+gene%3A+a+general+method+for+producing+specific+point+mutations+in+cloned+DNA."'>Oligonucleotide directed mutagenesis of the human β-globin gene: a general method for producing specific point mutations in cloned DNA.</a>" Nucleic Acids Res., 9:3647-3656 (1981).</td></tr><tr><td class="patent-data-table-td ">118</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Wallace, B.R., et al., Oligonucleotide directed mutagenesis of the human globin gene: a general method for producing specific point mutations in cloned DNA. Nucleic Acids Res., 9:3647 3656 (1981).</td></tr><tr><td class="patent-data-table-td ">119</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Watson, J.D., in Molecular Biology of the Gene, 3rd Edition. Benjamin/Cummings Publ. Co., Inc. Menlo Park, CA pp. 313 (1987).</td></tr><tr><td class="patent-data-table-td ">120</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Wells, et al., "<a href='http://scholar.google.com/scholar?q="Subtilisin%3A+An+Enzyme+Designed+to+be+Engineered%2C"'>Subtilisin: An Enzyme Designed to be Engineered,</a>" in Proteins: Form and Function (Bradshaw et al., eds.). Elsevier Trends Journals, Cambridge, UK, pp. 45-57 (1990).</td></tr><tr><td class="patent-data-table-td ">121</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Wells, et al., Cold Spring Harbor Symposia on Quantitative Biology, LII:647 652 (1987).</td></tr><tr><td class="patent-data-table-td ">122</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Wells, et al., Cold Spring Harbor Symposia on Quantitative Biology, LII:647-652 (1987).</td></tr><tr><td class="patent-data-table-td ">123</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Wells, et al., Nucleic Acids Res., 11:7911 7923 (1983).</td></tr><tr><td class="patent-data-table-td ">124</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Wells, et al., Nucleic Acids Res., 11:7911-7923 (1983).</td></tr><tr><td class="patent-data-table-td ">125</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Wells, et al., Subtilisin: An Enzyme Designed to be Engineered, in Proteins: Form and Function (Bradshaw et al., eds.). Elsevier Trends Journals, Cambridge, UK, pp. 45 57 (1990).</td></tr><tr><td class="patent-data-table-td ">126</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Wells, J.A., "<a href='http://scholar.google.com/scholar?q="Cloning%2C+sequencing%2C+and+secretion+of+Bacillus+amyloliquefaciens+subtilisin+in+Bacillus+subtilis."'>Cloning, sequencing, and secretion of Bacillus amyloliquefaciens subtilisin in Bacillus subtilis.</a>" Nucleic Acids Res., 11:8913-8926 (1985).</td></tr><tr><td class="patent-data-table-td ">127</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Wells, J.A., Cloning, sequencing, and secretion of Bacillus amyloliquefaciens subtilisin in Bacillus subtilis. Nucleic Acids Res., 11:8913 8926 (1985).</td></tr><tr><td class="patent-data-table-td ">128</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Wells, J.A., et al., "<a href='http://scholar.google.com/scholar?q="Designing+Substrate+Specificity+by+Protein+Engineering+of+Electrostatic+Interactions."'>Designing Substrate Specificity by Protein Engineering of Electrostatic Interactions.</a>" Proc. Nat. Acad. Sci. USA, 84:1219-1223 (1987).</td></tr><tr><td class="patent-data-table-td ">129</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Wells, J.A., et al., "<a href='http://scholar.google.com/scholar?q="Protein+Engineering+of+Subtilisin."'>Protein Engineering of Subtilisin.</a>" Protein Engineering, pp. 279-287 (1987).</td></tr><tr><td class="patent-data-table-td ">130</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Wells, J.A., et al., Designing Substrate Specificity by Protein Engineering of Electrostatic Interactions. Proc. Nat. Acad. Sci. USA, 84:1219 1223 (1987).</td></tr><tr><td class="patent-data-table-td ">131</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Wells, J.A., et al., Protein Engineering of Subtilisin. Protein Engineering, pp. 279 287 (1987).</td></tr><tr><td class="patent-data-table-td ">132</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Wilkinson, A.H., et al., Nature, 307:187 188 (1984).</td></tr><tr><td class="patent-data-table-td ">133</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Wilkinson, A.H., et al., Nature, 307:187-188 (1984).</td></tr><tr><td class="patent-data-table-td ">134</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Wilkinson, A.J., et al., Biochemistry, 22:3581 3586 (1983).</td></tr><tr><td class="patent-data-table-td ">135</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Wilkinson, A.J., et al., Biochemistry, 22:3581-3586 (1983).</td></tr><tr><td class="patent-data-table-td ">136</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Winter, G., et al., "<a href='http://scholar.google.com/scholar?q="Redesigning+enzyme+structure+by+site-directed+mutagenesis%3A+tyrosyl+tRNA+synthetase+and+ATP+binding."'>Redesigning enzyme structure by site-directed mutagenesis: tyrosyl tRNA synthetase and ATP binding.</a>" Nature, 299:756-758 (1982).</td></tr><tr><td class="patent-data-table-td ">137</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Winter, G., et al., Redesigning enzyme structure by site directed mutagenesis: tyrosyl tRNA synthetase and ATP binding. Nature, 299:756 758 (1982).</td></tr><tr><td class="patent-data-table-td ">138</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Wright, C.S., "<a href='http://scholar.google.com/scholar?q="Structure+of+Subtilisin+BPN%27+at+2%C2%B75+A+Resolution."'>Structure of Subtilisin BPN' at 2·5 A Resolution.</a>" Nature, 221:235-242 (1969).</td></tr><tr><td class="patent-data-table-td ">139</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Wright, C.S., Structure of Subtilisin BPN at 2 5 A Resolution. Nature, 221:235 242 (1969).</td></tr><tr><td class="patent-data-table-td ">140</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Zaghloul, et al., J. Bacteriol., 164:550 555 (1985).</td></tr><tr><td class="patent-data-table-td ">141</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Zaghloul, et al., J. Bacteriol., 164:550-555 (1985).</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="forward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Referenced by</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Citing Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5837517">US5837517</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">May 6, 1996</td><td class="patent-data-table-td patent-date-value">Nov 17, 1998</td><td class="patent-data-table-td ">Novo Nordisk A/S</td><td class="patent-data-table-td ">Detergents with enzymes produced by mutant genes</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5972683">US5972683</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Aug 18, 1998</td><td class="patent-data-table-td patent-date-value">Oct 26, 1999</td><td class="patent-data-table-td ">National Science Council</td><td class="patent-data-table-td ">Mutant type subtilisin YaB and its application</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5985639">US5985639</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Mar 19, 1996</td><td class="patent-data-table-td patent-date-value">Nov 16, 1999</td><td class="patent-data-table-td ">Henkel Kommanditgesellschaft Auf Aktien</td><td class="patent-data-table-td ">Mutant proteolytic enzymes and method of production</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6180776">US6180776</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jul 22, 1998</td><td class="patent-data-table-td patent-date-value">Jan 30, 2001</td><td class="patent-data-table-td ">Myriad Genetics, Inc.</td><td class="patent-data-table-td ">MTS2 gene</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6190900">US6190900</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jul 22, 1998</td><td class="patent-data-table-td patent-date-value">Feb 20, 2001</td><td class="patent-data-table-td ">Novo Nordisk A/S</td><td class="patent-data-table-td ">Adjusted protease; for amplified stability and wash performance in detergent</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6197567">US6197567</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Mar 12, 1997</td><td class="patent-data-table-td patent-date-value">Mar 6, 2001</td><td class="patent-data-table-td ">Novo Nordisk A/S</td><td class="patent-data-table-td ">Modified subtilisins and detergent compositions containing same</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6197589">US6197589</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">May 7, 1998</td><td class="patent-data-table-td patent-date-value">Mar 6, 2001</td><td class="patent-data-table-td ">Henkel Kommanditgesellschaft Auf Aktien</td><td class="patent-data-table-td ">Enzymes for detergents</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6218146">US6218146</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jul 22, 1998</td><td class="patent-data-table-td patent-date-value">Apr 17, 2001</td><td class="patent-data-table-td ">Myriad Genetics, Inc.</td><td class="patent-data-table-td ">MTS2 gene</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6261828">US6261828</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Apr 29, 1998</td><td class="patent-data-table-td patent-date-value">Jul 17, 2001</td><td class="patent-data-table-td ">Novo Nordisk A/S</td><td class="patent-data-table-td ">One step process by treating with amylolytic enzyme, abrading monocomponent endoglucanse, and streak-reducing monocomponent endoglucase; reduces problem of streaks and creases on finished garments</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6271012">US6271012</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jun 29, 1994</td><td class="patent-data-table-td patent-date-value">Aug 7, 2001</td><td class="patent-data-table-td ">Genencor International, Inc.</td><td class="patent-data-table-td ">An engineered proteolytic enzyme with improved washing performance and improved stability, laundry detergents</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6287841">US6287841</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jul 19, 1993</td><td class="patent-data-table-td patent-date-value">Sep 11, 2001</td><td class="patent-data-table-td ">Genencor International, Inc.</td><td class="patent-data-table-td ">High alkaline serine protease</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6300116">US6300116</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Nov 4, 1997</td><td class="patent-data-table-td patent-date-value">Oct 9, 2001</td><td class="patent-data-table-td ">Novozymes A/S</td><td class="patent-data-table-td ">Modified protease having improved autoproteolytic stability</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6331427">US6331427</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Mar 26, 1999</td><td class="patent-data-table-td patent-date-value">Dec 18, 2001</td><td class="patent-data-table-td ">Millennium Pharmaceuticals, Inc.</td><td class="patent-data-table-td ">Protease homologs</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6436690">US6436690</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Mar 3, 1995</td><td class="patent-data-table-td patent-date-value">Aug 20, 2002</td><td class="patent-data-table-td ">The Procter &amp; Gamble Company</td><td class="patent-data-table-td ">Isolated bpn&#39; variant having modified amino acid sequence of subtilisin bpn&#39; wild-type amino acid sequence comprising substitution at one or more positions in one of loop regions; cleaning compositions</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6440717">US6440717</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Mar 2, 1995</td><td class="patent-data-table-td patent-date-value">Aug 27, 2002</td><td class="patent-data-table-td ">The Procter &amp; Gamble Company</td><td class="patent-data-table-td ">BPN′ variants having decreased adsorption and increased hydrolysis</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6451574">US6451574</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Sep 23, 1997</td><td class="patent-data-table-td patent-date-value">Sep 17, 2002</td><td class="patent-data-table-td ">The Procter &amp; Gamble Company</td><td class="patent-data-table-td ">Proteinase K variants having decreased adsorption and increased hydrolysis</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6455295">US6455295</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Mar 8, 1995</td><td class="patent-data-table-td patent-date-value">Sep 24, 2002</td><td class="patent-data-table-td ">The Procter &amp; Gamble Company</td><td class="patent-data-table-td ">Subtilisin Carlsberg variants having decreased adsorption and increased hydrolysis</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6465235">US6465235</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Dec 18, 1997</td><td class="patent-data-table-td patent-date-value">Oct 15, 2002</td><td class="patent-data-table-td ">Genenco International, Inc.</td><td class="patent-data-table-td ">Non-human carbonyl hydrolase mutants, DNA sequences and vectors encoding same and hosts transformed with said vectors</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6475765">US6475765</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Mar 9, 1995</td><td class="patent-data-table-td patent-date-value">Nov 5, 2002</td><td class="patent-data-table-td ">Procter &amp; Gamble Company</td><td class="patent-data-table-td ">Subtilisin DY variants having decreased adsorption and increased hydrolysis</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6506589">US6506589</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jun 7, 1995</td><td class="patent-data-table-td patent-date-value">Jan 14, 2003</td><td class="patent-data-table-td ">Novozymes, A/S</td><td class="patent-data-table-td ">Useful mutations of bacterial alkaline protease</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6509021">US6509021</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Aug 14, 1996</td><td class="patent-data-table-td patent-date-value">Jan 21, 2003</td><td class="patent-data-table-td ">Henkel Kommanditgesellschaft Auf Aktien</td><td class="patent-data-table-td ">Use of mutated subtilisin protease in cosmetic products</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6586221">US6586221</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Oct 13, 1994</td><td class="patent-data-table-td patent-date-value">Jul 1, 2003</td><td class="patent-data-table-td ">Genencor International, Inc.</td><td class="patent-data-table-td ">Carbonyl hydrolase variants derived from the DNA sequences of naturally-occurring or recombinant non-human carbonyl hydrolases are disclosed. The variant carbonyl hydrolases, in general, are obtained by in vitro modification of a precursor</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6599730">US6599730</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Mar 7, 1995</td><td class="patent-data-table-td patent-date-value">Jul 29, 2003</td><td class="patent-data-table-td ">Procter &amp; Gamble Company</td><td class="patent-data-table-td ">Subtilisin for use in cleaning hard surfaces</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6632646">US6632646</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Oct 2, 2000</td><td class="patent-data-table-td patent-date-value">Oct 14, 2003</td><td class="patent-data-table-td ">Novozymes A/S</td><td class="patent-data-table-td ">Modified subtilisins and detergent compositions containing same</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6682924">US6682924</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jun 5, 2000</td><td class="patent-data-table-td patent-date-value">Jan 27, 2004</td><td class="patent-data-table-td ">Novozymes A/S</td><td class="patent-data-table-td ">Protease variants and compositions</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6727085">US6727085</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Dec 13, 2000</td><td class="patent-data-table-td patent-date-value">Apr 27, 2004</td><td class="patent-data-table-td ">Fanoe Tina Sejersgaard</td><td class="patent-data-table-td ">Subtilase variants having an improved wash performance on egg stains</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6808913">US6808913</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Nov 27, 2002</td><td class="patent-data-table-td patent-date-value">Oct 26, 2004</td><td class="patent-data-table-td ">Novozymes A/S</td><td class="patent-data-table-td ">Comprises genetically engineered subtilisin for improved washing abilty of deteregents</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7087415">US7087415</a></td><td class="patent-data-table-td patent-date-value">Jul 31, 2001</td><td class="patent-data-table-td patent-date-value">Aug 8, 2006</td><td class="patent-data-table-td ">Athena Biotechnologies, Inc.</td><td class="patent-data-table-td ">Methods and compositions for directed gene assembly</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7098017">US7098017</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Sep 6, 2001</td><td class="patent-data-table-td patent-date-value">Aug 29, 2006</td><td class="patent-data-table-td ">Novozymes A/S</td><td class="patent-data-table-td ">Enzymatic polypeptide for use in the detergents</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7129076">US7129076</a></td><td class="patent-data-table-td patent-date-value">Apr 25, 2003</td><td class="patent-data-table-td patent-date-value">Oct 31, 2006</td><td class="patent-data-table-td ">Genencor International, Inc.</td><td class="patent-data-table-td ">Multiply-substituted protease variants with altered net charge for use in detergents</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7157416">US7157416</a></td><td class="patent-data-table-td patent-date-value">Jul 20, 2001</td><td class="patent-data-table-td patent-date-value">Jan 2, 2007</td><td class="patent-data-table-td ">Genencor International, Inc.</td><td class="patent-data-table-td ">Protease enzyme produced from any Bacillus bacteria, alkali metal halide salt and a polyol such as glycerol, propylene glycol or sucrose</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7189553">US7189553</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Dec 6, 2002</td><td class="patent-data-table-td patent-date-value">Mar 13, 2007</td><td class="patent-data-table-td ">Novozymes A/S</td><td class="patent-data-table-td ">Modified subtilisins and detergent, compositions containing same</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7192757">US7192757</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jul 2, 2004</td><td class="patent-data-table-td patent-date-value">Mar 20, 2007</td><td class="patent-data-table-td ">Novozymes A/S</td><td class="patent-data-table-td ">Subtilase variants</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7223386">US7223386</a></td><td class="patent-data-table-td patent-date-value">Sep 10, 2003</td><td class="patent-data-table-td patent-date-value">May 29, 2007</td><td class="patent-data-table-td ">Dow Corning Corporation</td><td class="patent-data-table-td ">comprising silicone matrix, a hydrophilic carrier, and at least one active agent comprising proteins, particularly enzymes such as hydrolases and glucose oxidase for enzymatic debridement, clotting formation and clot removal and wound healing</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7297354">US7297354</a></td><td class="patent-data-table-td patent-date-value">Apr 26, 2001</td><td class="patent-data-table-td patent-date-value">Nov 20, 2007</td><td class="patent-data-table-td ">Land O&#39;lakes, Inc.</td><td class="patent-data-table-td ">Protein material</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7303907">US7303907</a></td><td class="patent-data-table-td patent-date-value">Jul 8, 2003</td><td class="patent-data-table-td patent-date-value">Dec 4, 2007</td><td class="patent-data-table-td ">Health Protection Agency</td><td class="patent-data-table-td ">Transmissible spongiform encephalopathy agent inactivated by exposing the TSE agent to a thermostable proteolytic enzyme at elevated temperatures; infectivity determined by detection of dimers of prion protein</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7320887">US7320887</a></td><td class="patent-data-table-td patent-date-value">Apr 30, 2004</td><td class="patent-data-table-td patent-date-value">Jan 22, 2008</td><td class="patent-data-table-td ">Henkel Kommanditgesellschaft Auf Aktien</td><td class="patent-data-table-td ">Alkaline protease variants</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7332320">US7332320</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Dec 20, 2002</td><td class="patent-data-table-td patent-date-value">Feb 19, 2008</td><td class="patent-data-table-td ">Genencor International, Inc.</td><td class="patent-data-table-td ">Protease producing an altered immunogenic response and methods of making and using the same</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7566448">US7566448</a></td><td class="patent-data-table-td patent-date-value">Jul 23, 2003</td><td class="patent-data-table-td patent-date-value">Jul 28, 2009</td><td class="patent-data-table-td ">Genencor International, Inc.</td><td class="patent-data-table-td ">Reduced aerosol generating formulations</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7618801">US7618801</a></td><td class="patent-data-table-td patent-date-value">Oct 30, 2007</td><td class="patent-data-table-td patent-date-value">Nov 17, 2009</td><td class="patent-data-table-td ">Danison US Inc.</td><td class="patent-data-table-td ">Streptomyces protease</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7879788">US7879788</a></td><td class="patent-data-table-td patent-date-value">Sep 29, 2009</td><td class="patent-data-table-td patent-date-value">Feb 1, 2011</td><td class="patent-data-table-td ">Danisco Us Inc.</td><td class="patent-data-table-td ">Methods of cleaning using a streptomyces 1AG3 serine protease</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7888093">US7888093</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Oct 31, 2003</td><td class="patent-data-table-td patent-date-value">Feb 15, 2011</td><td class="patent-data-table-td ">Novozymes A/S</td><td class="patent-data-table-td ">Subtilase variants</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7985569">US7985569</a></td><td class="patent-data-table-td patent-date-value">May 31, 2007</td><td class="patent-data-table-td patent-date-value">Jul 26, 2011</td><td class="patent-data-table-td ">Danisco Us Inc.</td><td class="patent-data-table-td ">Multiple substitutions, DNA encoding these proteases, vectors, host cells transformed with the vector DNA, and enzymes produced by the host cells; detergent compositions, animal feeds, textile and leather processing</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8110391">US8110391</a></td><td class="patent-data-table-td patent-date-value">Oct 11, 2007</td><td class="patent-data-table-td patent-date-value">Feb 7, 2012</td><td class="patent-data-table-td ">Health Protection Agency</td><td class="patent-data-table-td ">Degradation and detection of TSE infectivity</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8124399">US8124399</a></td><td class="patent-data-table-td patent-date-value">Mar 28, 2003</td><td class="patent-data-table-td patent-date-value">Feb 28, 2012</td><td class="patent-data-table-td ">Danisco Us Inc.</td><td class="patent-data-table-td ">Enhanced protein expression in Bacillus</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8183024">US8183024</a></td><td class="patent-data-table-td patent-date-value">Nov 10, 2009</td><td class="patent-data-table-td patent-date-value">May 22, 2012</td><td class="patent-data-table-td ">Danisco Us Inc.</td><td class="patent-data-table-td ">Compositions and methods comprising a subtilisin variant</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8241885">US8241885</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Sep 22, 2010</td><td class="patent-data-table-td patent-date-value">Aug 14, 2012</td><td class="patent-data-table-td ">University Of Maryland, College Park</td><td class="patent-data-table-td ">Engineered proteases for affinity purification and processing of fusion proteins</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8343735">US8343735</a></td><td class="patent-data-table-td patent-date-value">May 12, 2008</td><td class="patent-data-table-td patent-date-value">Jan 1, 2013</td><td class="patent-data-table-td ">Danisco Us Inc.</td><td class="patent-data-table-td ">Modified secretion system to increase expression of polypeptides in bacteria</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8383366">US8383366</a></td><td class="patent-data-table-td patent-date-value">Jan 23, 2012</td><td class="patent-data-table-td patent-date-value">Feb 26, 2013</td><td class="patent-data-table-td ">Danisco Us Inc.</td><td class="patent-data-table-td ">Enhanced protein expression in Bacillus</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8383387">US8383387</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Nov 28, 2011</td><td class="patent-data-table-td patent-date-value">Feb 26, 2013</td><td class="patent-data-table-td ">Novozymes A/S</td><td class="patent-data-table-td ">Subtilases</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8455234">US8455234</a></td><td class="patent-data-table-td patent-date-value">Jun 16, 2011</td><td class="patent-data-table-td patent-date-value">Jun 4, 2013</td><td class="patent-data-table-td ">Danisco Us Inc.</td><td class="patent-data-table-td ">Multiple mutation variants of serine protease</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8530219">US8530219</a></td><td class="patent-data-table-td patent-date-value">Nov 10, 2009</td><td class="patent-data-table-td patent-date-value">Sep 10, 2013</td><td class="patent-data-table-td ">Danisco Us Inc.</td><td class="patent-data-table-td ">Compositions and methods comprising a subtilisin variant</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8535927">US8535927</a></td><td class="patent-data-table-td patent-date-value">Nov 19, 2004</td><td class="patent-data-table-td patent-date-value">Sep 17, 2013</td><td class="patent-data-table-td ">Danisco Us Inc.</td><td class="patent-data-table-td ">Micrococcineae serine protease polypeptides and compositions thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8546122">US8546122</a></td><td class="patent-data-table-td patent-date-value">Jan 17, 2013</td><td class="patent-data-table-td patent-date-value">Oct 1, 2013</td><td class="patent-data-table-td ">Novozymes A/S</td><td class="patent-data-table-td ">Subtilases</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8569035">US8569035</a></td><td class="patent-data-table-td patent-date-value">Apr 29, 2013</td><td class="patent-data-table-td patent-date-value">Oct 29, 2013</td><td class="patent-data-table-td ">Novozymes A/S</td><td class="patent-data-table-td ">Subtilase variants</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8623630">US8623630</a></td><td class="patent-data-table-td patent-date-value">Oct 15, 2012</td><td class="patent-data-table-td patent-date-value">Jan 7, 2014</td><td class="patent-data-table-td ">Danisco Us Inc.</td><td class="patent-data-table-td ">Modified secretion system to increase expression of polypeptides in bacteria</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8648015">US8648015</a></td><td class="patent-data-table-td patent-date-value">May 12, 2010</td><td class="patent-data-table-td patent-date-value">Feb 11, 2014</td><td class="patent-data-table-td ">Danisco Us Inc.</td><td class="patent-data-table-td ">Systematic evaluation of sequence and activity relationships using site evaluation libraries for engineering multiple properties</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8728790">US8728790</a></td><td class="patent-data-table-td patent-date-value">Dec 9, 2010</td><td class="patent-data-table-td patent-date-value">May 20, 2014</td><td class="patent-data-table-td ">Danisco Us Inc.</td><td class="patent-data-table-td ">Compositions and methods comprising protease variants</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8741609">US8741609</a></td><td class="patent-data-table-td patent-date-value">Dec 14, 2010</td><td class="patent-data-table-td patent-date-value">Jun 3, 2014</td><td class="patent-data-table-td ">Danisco Us Inc.</td><td class="patent-data-table-td ">Detergent compositions containing Geobacillus stearothermophilus lipase and methods of use thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8753861">US8753861</a></td><td class="patent-data-table-td patent-date-value">Nov 10, 2009</td><td class="patent-data-table-td patent-date-value">Jun 17, 2014</td><td class="patent-data-table-td ">Danisco Us Inc.</td><td class="patent-data-table-td ">Protease comprising one or more combinable mutations</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8779112">US8779112</a></td><td class="patent-data-table-td patent-date-value">Mar 12, 2008</td><td class="patent-data-table-td patent-date-value">Jul 15, 2014</td><td class="patent-data-table-td ">Danisco Us Inc.</td><td class="patent-data-table-td ">Modified protease propeptides</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20040146938">US20040146938</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Oct 2, 2003</td><td class="patent-data-table-td patent-date-value">Jul 29, 2004</td><td class="patent-data-table-td ">Jack Nguyen</td><td class="patent-data-table-td ">Methods of generating and screening for proteases with altered specificity</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20070077617">US20070077617</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Sep 29, 2004</td><td class="patent-data-table-td patent-date-value">Apr 5, 2007</td><td class="patent-data-table-td ">Kazuaki Okuno</td><td class="patent-data-table-td ">Polypeptide cleavage method using ompt protease variant</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20110008869">US20110008869</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Sep 22, 2010</td><td class="patent-data-table-td patent-date-value">Jan 13, 2011</td><td class="patent-data-table-td ">University Of Maryland College Park</td><td class="patent-data-table-td ">Engineered proteases for affinity purification and processing of fusion proteins</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20120088288">US20120088288</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Nov 28, 2011</td><td class="patent-data-table-td patent-date-value">Apr 12, 2012</td><td class="patent-data-table-td ">Novozymes A/S</td><td class="patent-data-table-td ">Subtilases</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP1998266A2?cl=en">EP1998266A2</a></td><td class="patent-data-table-td patent-date-value">Feb 18, 2002</td><td class="patent-data-table-td patent-date-value">Dec 3, 2008</td><td class="patent-data-table-td ">Merck Patent GmbH</td><td class="patent-data-table-td ">Method for identification of T-cell epitopes and use for preparing molecules with reduced immunogenicity</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2287320A1?cl=en">EP2287320A1</a></td><td class="patent-data-table-td patent-date-value">Jan 16, 2003</td><td class="patent-data-table-td patent-date-value">Feb 23, 2011</td><td class="patent-data-table-td ">Genencor International, Inc.</td><td class="patent-data-table-td ">Multiply-substituted protease variants</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2287321A1?cl=en">EP2287321A1</a></td><td class="patent-data-table-td patent-date-value">Jan 16, 2003</td><td class="patent-data-table-td patent-date-value">Feb 23, 2011</td><td class="patent-data-table-td ">Genencor International, Inc.</td><td class="patent-data-table-td ">Multiply-substituted protease variants</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2339016A2?cl=en">EP2339016A2</a></td><td class="patent-data-table-td patent-date-value">Mar 28, 2003</td><td class="patent-data-table-td patent-date-value">Jun 29, 2011</td><td class="patent-data-table-td ">Genencor International, Inc.</td><td class="patent-data-table-td ">Enhanced protein expression in bacillus</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2339017A2?cl=en">EP2339017A2</a></td><td class="patent-data-table-td patent-date-value">Mar 28, 2003</td><td class="patent-data-table-td patent-date-value">Jun 29, 2011</td><td class="patent-data-table-td ">Genencor International, Inc.</td><td class="patent-data-table-td ">Enhanced protein expression in bacillus</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2341070A2?cl=en">EP2341070A2</a></td><td class="patent-data-table-td patent-date-value">Mar 28, 2003</td><td class="patent-data-table-td patent-date-value">Jul 6, 2011</td><td class="patent-data-table-td ">Genencor International, Inc.</td><td class="patent-data-table-td ">Ehanced protein expression in bacillus</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2460823A1?cl=en">EP2460823A1</a></td><td class="patent-data-table-td patent-date-value">May 12, 2008</td><td class="patent-data-table-td patent-date-value">Jun 6, 2012</td><td class="patent-data-table-td ">Danisco US Inc.</td><td class="patent-data-table-td ">A modified secretion system to increase expression of polypeptides in bacteria</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2578679A1?cl=en">EP2578679A1</a></td><td class="patent-data-table-td patent-date-value">Jun 3, 2009</td><td class="patent-data-table-td patent-date-value">Apr 10, 2013</td><td class="patent-data-table-td ">Danisco US Inc.</td><td class="patent-data-table-td ">Compositions and methods comprising variant microbial proteases</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2578680A1?cl=en">EP2578680A1</a></td><td class="patent-data-table-td patent-date-value">Jun 3, 2009</td><td class="patent-data-table-td patent-date-value">Apr 10, 2013</td><td class="patent-data-table-td ">Danisco US Inc.</td><td class="patent-data-table-td ">Compositions and methods comprising variant microbial proteases</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2589651A2?cl=en">EP2589651A2</a></td><td class="patent-data-table-td patent-date-value">Nov 10, 2009</td><td class="patent-data-table-td patent-date-value">May 8, 2013</td><td class="patent-data-table-td ">Danisco US Inc.</td><td class="patent-data-table-td ">Compositions and methods comprising serine protease variants</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2647692A2?cl=en">EP2647692A2</a></td><td class="patent-data-table-td patent-date-value">Nov 10, 2009</td><td class="patent-data-table-td patent-date-value">Oct 9, 2013</td><td class="patent-data-table-td ">Danisco US Inc.</td><td class="patent-data-table-td ">Compositions and methods comprising serine protease variants</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2003101404A2?cl=en">WO2003101404A2</a></td><td class="patent-data-table-td patent-date-value">Mar 11, 2003</td><td class="patent-data-table-td patent-date-value">Dec 11, 2003</td><td class="patent-data-table-td ">Dow Corning</td><td class="patent-data-table-td ">Preparations for topical skin use and treatment</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2009058679A1?cl=en">WO2009058679A1</a></td><td class="patent-data-table-td patent-date-value">Oct 24, 2008</td><td class="patent-data-table-td patent-date-value">May 7, 2009</td><td class="patent-data-table-td ">Danisco Us Inc Genencor Div</td><td class="patent-data-table-td ">Streptomyces protease</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2009094084A1?cl=en">WO2009094084A1</a></td><td class="patent-data-table-td patent-date-value">Dec 19, 2008</td><td class="patent-data-table-td patent-date-value">Jul 30, 2009</td><td class="patent-data-table-td ">Danisco Us Inc Genencor Div</td><td class="patent-data-table-td ">Enhanced protein production in bacillus</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2009149144A2?cl=en">WO2009149144A2</a></td><td class="patent-data-table-td patent-date-value">Jun 3, 2009</td><td class="patent-data-table-td patent-date-value">Dec 10, 2009</td><td class="patent-data-table-td ">Danisco Us Inc.</td><td class="patent-data-table-td ">Compositions and methods comprising variant microbial proteases</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2010115021A2?cl=en">WO2010115021A2</a></td><td class="patent-data-table-td patent-date-value">Apr 1, 2010</td><td class="patent-data-table-td patent-date-value">Oct 7, 2010</td><td class="patent-data-table-td ">Danisco Us Inc.</td><td class="patent-data-table-td ">Compositions and methods comprising alpha-amylase variants with altered properties</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2010115028A2?cl=en">WO2010115028A2</a></td><td class="patent-data-table-td patent-date-value">Apr 1, 2010</td><td class="patent-data-table-td patent-date-value">Oct 7, 2010</td><td class="patent-data-table-td ">Danisco Us Inc.</td><td class="patent-data-table-td ">Cleaning system comprising an alpha-amylase and a protease</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2010144283A1?cl=en">WO2010144283A1</a></td><td class="patent-data-table-td patent-date-value">Jun 2, 2010</td><td class="patent-data-table-td patent-date-value">Dec 16, 2010</td><td class="patent-data-table-td ">Danisco Us Inc.</td><td class="patent-data-table-td ">Bacillus strain for increased protein production</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2011084412A1?cl=en">WO2011084412A1</a></td><td class="patent-data-table-td patent-date-value">Dec 14, 2010</td><td class="patent-data-table-td patent-date-value">Jul 14, 2011</td><td class="patent-data-table-td ">Danisco Us Inc.</td><td class="patent-data-table-td ">Detergent compositions containing thermobifida fusca lipase and methods of use thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2011084417A1?cl=en">WO2011084417A1</a></td><td class="patent-data-table-td patent-date-value">Dec 14, 2010</td><td class="patent-data-table-td patent-date-value">Jul 14, 2011</td><td class="patent-data-table-td ">Danisco Us Inc.</td><td class="patent-data-table-td ">Detergent compositions containing geobacillus stearothermophilus lipase and methods of use thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2011084599A1?cl=en">WO2011084599A1</a></td><td class="patent-data-table-td patent-date-value">Dec 16, 2010</td><td class="patent-data-table-td patent-date-value">Jul 14, 2011</td><td class="patent-data-table-td ">Danisco Us Inc.</td><td class="patent-data-table-td ">Detergent compositions containing bacillus subtilis lipase and methods of use thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2011130222A2?cl=en">WO2011130222A2</a></td><td class="patent-data-table-td patent-date-value">Apr 12, 2011</td><td class="patent-data-table-td patent-date-value">Oct 20, 2011</td><td class="patent-data-table-td ">Danisco Us Inc.</td><td class="patent-data-table-td ">Compositions and methods comprising variant proteases</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2011140364A1?cl=en">WO2011140364A1</a></td><td class="patent-data-table-td patent-date-value">May 5, 2011</td><td class="patent-data-table-td patent-date-value">Nov 10, 2011</td><td class="patent-data-table-td ">Danisco Us Inc.</td><td class="patent-data-table-td ">Compositions and methods comprising subtilisin variants</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2011150157A2?cl=en">WO2011150157A2</a></td><td class="patent-data-table-td patent-date-value">May 26, 2011</td><td class="patent-data-table-td patent-date-value">Dec 1, 2011</td><td class="patent-data-table-td ">Danisco Us Inc.</td><td class="patent-data-table-td ">Detergent compositions containing streptomyces griseus lipase and methods of use thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2012149317A1?cl=en">WO2012149317A1</a></td><td class="patent-data-table-td patent-date-value">Apr 27, 2012</td><td class="patent-data-table-td patent-date-value">Nov 1, 2012</td><td class="patent-data-table-td ">Danisco Us Inc.</td><td class="patent-data-table-td ">Detergent compositions containing bacillus agaradhaerens mannanase and methods of use thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2012149325A1?cl=en">WO2012149325A1</a></td><td class="patent-data-table-td patent-date-value">Apr 27, 2012</td><td class="patent-data-table-td patent-date-value">Nov 1, 2012</td><td class="patent-data-table-td ">Danisco Us Inc.</td><td class="patent-data-table-td ">Detergent compositions containing geobacillus tepidamans mannanase and methods of use thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2012149333A1?cl=en">WO2012149333A1</a></td><td class="patent-data-table-td patent-date-value">Apr 27, 2012</td><td class="patent-data-table-td patent-date-value">Nov 1, 2012</td><td class="patent-data-table-td ">Danisco Us Inc.</td><td class="patent-data-table-td ">Detergent compositions containing bacillus sp. mannanase and methods of use thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2012151480A2?cl=en">WO2012151480A2</a></td><td class="patent-data-table-td patent-date-value">May 4, 2012</td><td class="patent-data-table-td patent-date-value">Nov 8, 2012</td><td class="patent-data-table-td ">The Procter &amp; Gamble Company</td><td class="patent-data-table-td ">Compositions and methods comprising serine protease variants</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2012151534A1?cl=en">WO2012151534A1</a></td><td class="patent-data-table-td patent-date-value">May 4, 2012</td><td class="patent-data-table-td patent-date-value">Nov 8, 2012</td><td class="patent-data-table-td ">Danisco Us Inc.</td><td class="patent-data-table-td ">Compositions and methods comprising serine protease variants</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2013033318A1?cl=en">WO2013033318A1</a></td><td class="patent-data-table-td patent-date-value">Aug 30, 2012</td><td class="patent-data-table-td patent-date-value">Mar 7, 2013</td><td class="patent-data-table-td ">Danisco Us Inc.</td><td class="patent-data-table-td ">Compositions and methods comprising a lipolytic enzyme variant</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2013096653A1?cl=en">WO2013096653A1</a></td><td class="patent-data-table-td patent-date-value">Dec 20, 2012</td><td class="patent-data-table-td patent-date-value">Jun 27, 2013</td><td class="patent-data-table-td ">Danisco Us Inc.</td><td class="patent-data-table-td ">Compositions and methods comprising a lipolytic enzyme variant</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2013138288A1?cl=en">WO2013138288A1</a></td><td class="patent-data-table-td patent-date-value">Mar 12, 2013</td><td class="patent-data-table-td patent-date-value">Sep 19, 2013</td><td class="patent-data-table-td ">Monosol, Llc.</td><td class="patent-data-table-td ">Water soluble compositions incorporating enzymes, and method of making same</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2013158364A1?cl=en">WO2013158364A1</a></td><td class="patent-data-table-td patent-date-value">Apr 2, 2013</td><td class="patent-data-table-td patent-date-value">Oct 24, 2013</td><td class="patent-data-table-td ">Monosol, Llc</td><td class="patent-data-table-td ">Powdered pouch and method of making same</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2014059360A1?cl=en">WO2014059360A1</a></td><td class="patent-data-table-td patent-date-value">Oct 11, 2013</td><td class="patent-data-table-td patent-date-value">Apr 17, 2014</td><td class="patent-data-table-td ">Danisco Us Inc.</td><td class="patent-data-table-td ">Compositions and methods comprising a lipolytic enzyme variant</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2014071410A1?cl=en">WO2014071410A1</a></td><td class="patent-data-table-td patent-date-value">Nov 5, 2013</td><td class="patent-data-table-td patent-date-value">May 8, 2014</td><td class="patent-data-table-td ">Danisco Us Inc.</td><td class="patent-data-table-td ">Compositions and methods comprising thermolysin protease variants</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="classifications"></a><div class="patent-section-header"><span class="patent-section-title">Classifications</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th"> </th><th class="patent-data-table-th"> </th></tr></thead><tr><td class="patent-data-table-td ">U.S. Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=tNNEBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S221000">435/221</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=tNNEBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S222000">435/222</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=tNNEBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S023200">536/23.2</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=tNNEBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S220000">435/220</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=tNNEBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S320100">435/320.1</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=tNNEBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S069100">435/69.1</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=tNNEBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc510/defs510.htm&usg=AFQjCNGksTcLVF3GE9680P9u4elFLUcm_g#C510S300000">510/300</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=tNNEBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S252300">435/252.3</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=tNNEBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S252310">435/252.31</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=tNNEBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc510/defs510.htm&usg=AFQjCNGksTcLVF3GE9680P9u4elFLUcm_g#C510S301000">510/301</a></span></td></tr><tr><td class="patent-data-table-td ">International Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=tNNEBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0015100000">C12N15/10</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=tNNEBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0009560000">C12N9/56</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=tNNEBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C11D0003386000">C11D3/386</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=tNNEBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0009540000">C12N9/54</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=tNNEBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0015750000">C12N15/75</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=tNNEBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0015570000">C12N15/57</a></span></td></tr><tr><td class="patent-data-table-td ">Cooperative Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=tNNEBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C11D3/386">C11D3/386</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=tNNEBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N9/54">C12N9/54</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=tNNEBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12R1/07">C12R1/07</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=tNNEBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N15/75">C12N15/75</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=tNNEBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N15/102">C12N15/102</a></span></td></tr><tr><td class="patent-data-table-td ">European Classification</td><td class="patent-data-table-td "><span class="nested-value">C12N9/54</span>, <span class="nested-value">C11D3/386</span>, <span class="nested-value">C12R1/07</span>, <span class="nested-value">C12N15/10B</span>, <span class="nested-value">C12N15/75</span></td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="legal-events"></a><div class="patent-section-header"><span class="patent-section-title">Legal Events</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Date</th><th class="patent-data-table-th">Code</th><th class="patent-data-table-th">Event</th><th class="patent-data-table-th">Description</th></tr></thead><tr><td class="patent-data-table-td patent-date-value">Jun 23, 2009</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">12</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">May 12, 2009</td><td class="patent-data-table-td ">B1</td><td class="patent-data-table-td ">Reexamination certificate first reexamination</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">THE PATENTABILITY OF CLAIMS 1-22, 43-46 AND 50-56 IS CONFIRMED. CLAIM 57 IS CANCELLED. CLAIM 23 IS DETERMINED TO BE PATENTABLE AS AMENDED. CLAIMS 24-42 AND 47-49, DEPENDENT ON AN AMENDED CLAIM, ARE DETERMINED TO BE PATENTABLE.</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Dec 18, 2007</td><td class="patent-data-table-td ">RR</td><td class="patent-data-table-td ">Request for reexamination filed</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20071017</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">May 26, 2005</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">8</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">May 31, 2001</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">4</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Aug 18, 1997</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">GENENCOR INTERNATIONAL, INC., CALIFORNIA</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNOR:GENENTECH, INC.;REEL/FRAME:008668/0354</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">19970811</span></div></td></tr></table><div class="patent-section-footer"></div></div><div class="modal-dialog" id="patent-images-lightbox"><div class="patent-lightbox-controls"><div class="patent-lightbox-rotate-controls"><div class="patent-lightbox-rotation-text">Rotate</div><div class="rotate-icon rotate-ccw-icon"></div><div class="rotate-icon rotate-cw-icon"></div></div><div class="patent-lightbox-index-counter"></div><a class="patent-lightbox-fullsize-link" target="_blank">Original Image</a><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_right.png" alt="Next page"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_left.png" alt="Previous page"width="21" height="21" /></div></div><div class="modal-dialog-content"><div class="patent-lightbox-image-holder"><div class="patent-lightbox-placeholder"></div></div></div></div><script>_OC_initPatentsAtb({image_not_available_html: " Image not available"});</script></div></div></div></td></tr></table><script>(function() {var href = window.location.href;if (href.indexOf('?') !== -1) {var parameters = href.split('?')[1].split('&');for (var i = 0; i < parameters.length; i++) {var param = parameters[i].split('=');if (param[0] == 'focus') {var elem = document.getElementById(param[1]);if (elem) {elem.focus();}}}}})();</script><script>_OC_addFlags({LockSrc:"/books/javascript/lock_8a2b04e7bf975d5171d8e4c0b6365c7a.js", Host:"http://www.google.com/", IsBooksRentalEnabled:1, IsWebstoreDisplayCaseEnabled:1, IsObfuscationEnabled:1, IsBrowsingHistoryEnabled:1, IsWebReaderSvgEnabled:0, IsGeoLayerEnabled:1, IsImageModeNotesEnabled:1, IsCopyMenuItemEnabled:1, IsGiftingEnabled:0, IsWebReaderUniversalPaginatorEnabled:0, IsOfflineBubbleEnabled:1, IsReaderEnabledForPlayRequests:1, IsFutureOnSaleVolumesEnabled:1, IsOfflineRestrictedCopyEnabled:1, IsBooksUnifiedLeftNavEnabled:1, IsRestrictedCopyEnabled:1, IsZipitFolderCollectionEnabled:1, IsEndOfSampleRecommendationsEnabled:1, IsRatingsOnBookcardsEnabled:1, IsAdsDisabled:0, IsIframePageDisplayEnabled:0, IsEmbeddedMediaEnabled:1, IsImageModeAnnotationsEnabled:1, IsMyLibraryGooglePlusEnabled:1, IsImagePageProviderEnabled:0, IsBookcardListPriceSmall:0, IsInternalUser:0, IsBooksShareButtonEnabled:0, IsPreOrdersEnabled:0, IsDisabledRandomBookshelves:0, WebstoreDisplayCasePosition:3});_OC_Run({"enable_p13n":false,"add_vol_to_collection_base_url":"http://www.google.com/patents?op=add\u0026sig=ACfU3U305Pz08UmztuYcfOocqt7Agr8eOA\u0026id=tNNEBAABERAJ","remove_vol_from_collection_base_url":"http://www.google.com/patents?op=remove\u0026sig=ACfU3U1ECLyDVuTvZOgT-Ij0TlT5RroV2A\u0026id=tNNEBAABERAJ","logged_in":false,"p13n_save_user_settings_url":"http://www.google.com/patents?op=edit_user_settings\u0026sig=ACfU3U3NjkUN1xHFwtW7Aoq_S1XWN59zRw","is_cobrand":false,"sign_in_url":"https://www.google.com/accounts/Login?service=\u0026continue=http://www.google.com/patents%3Fhl%3Den\u0026hl=en","is_play_enabled":true}, {"volume_id":"","is_ebook":true,"volumeresult":{"has_flowing_text":false,"has_scanned_text":true,"can_download_pdf":false,"can_download_epub":false,"is_pdf_drm_enabled":false,"is_epub_drm_enabled":false,"download_pdf_url":"http://www.google.com/patents/download/Modified_subtilisins_having_amino_acid_a.pdf?id=tNNEBAABERAJ\u0026output=pdf\u0026sig=ACfU3U2kGfFgKTugfH0keiGjEyas-uQSlg"},"sample_url":"http://www.google.com/patents/reader?id=tNNEBAABERAJ\u0026printsec=frontcover\u0026output=reader\u0026source=gbs_atb_hover","is_browsable":true,"is_public_domain":true}, {});</script><div id="footer_table" style="font-size:83%;text-align:center;position:relative;top:20px;height:4.5em;margin-top:2em"><div style="margin-bottom:8px"><a href=http://www.google.com/><nobr>Google&nbsp;Home</nobr></a> - <a href=//www.google.com/patents/sitemap/><nobr>Sitemap</nobr></a> - <a href=http://www.google.com/googlebooks/uspto.html><nobr>USPTO Bulk Downloads</nobr></a> - <a href=/intl/en/privacy/><nobr>Privacy Policy</nobr></a> - <a href=/intl/en/policies/terms/><nobr>Terms of Service</nobr></a> - <a href=https://support.google.com/faqs/answer/2539193?hl=en><nobr>About Google Patents</nobr></a> - <a href="http://www.google.com/tools/feedback/intl/en/error.html" onclick="try{_OC_startFeedback({productId: '72792',locale: 'en'});return false;}catch(e){}"><nobr>Send Feedback</nobr></a></div><span>Data provided by IFI CLAIMS Patent Services</span><br><span >&copy;2012 Google</span></div> <script type="text/javascript">var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));</script><script type="text/javascript">var pageTracker = _gat._getTracker("UA-27188110-1");pageTracker._setCookiePath("/patents/");pageTracker._trackPageview();</script> </body></html>