<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN""http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html><head><title>Patent US5786464 - Overexpression of mammalian and viral proteins - Google Patents</title><script>(function(){(function(){function e(a){this.t={};this.tick=function(a,c,b){var d=void 0!=b?b:(new Date).getTime();this.t[a]=[d,c];if(void 0==b)try{window.console.timeStamp("CSI/"+a)}catch(e){}};this.tick("start",null,a)}var a;window.performance&&(a=window.performance.timing);var f=a?new e(a.responseStart):new e;window.jstiming={Timer:e,load:f};if(a){var c=a.navigationStart,d=a.responseStart;0<c&&d>=c&&(window.jstiming.srt=d-c)}if(a){var b=window.jstiming.load;0<c&&d>=c&&(b.tick("_wtsrt",void 0,c),b.tick("wtsrt_",
"_wtsrt",d),b.tick("tbsd_","wtsrt_"))}try{a=null,window.chrome&&window.chrome.csi&&(a=Math.floor(window.chrome.csi().pageT),b&&0<c&&(b.tick("_tbnd",void 0,window.chrome.csi().startE),b.tick("tbnd_","_tbnd",c))),null==a&&window.gtbExternal&&(a=window.gtbExternal.pageT()),null==a&&window.external&&(a=window.external.pageT,b&&0<c&&(b.tick("_tbnd",void 0,window.external.startE),b.tick("tbnd_","_tbnd",c))),a&&(window.jstiming.pt=a)}catch(g){}})();})();
</script><link rel="stylesheet" href="/patents/css/_8a2b04e7bf975d5171d8e4c0b6365c7a/kl_intl_patents_bundle.css" type="text/css" /><script src="/books/javascript/atb_8a2b04e7bf975d5171d8e4c0b6365c7a__en.js"></script><script>function googleTranslateElementInit() {new google.translate.TranslateElement({pageLanguage: "en",gaTrack: true,gaId: "UA-27188110-1",multilanguagePage: true});}</script><script src="//translate.google.com/translate_a/element.js?cb=googleTranslateElementInit"></script><meta name="DC.type" content="Patent"><meta name="DC.title" content="Overexpression of mammalian and viral proteins"><meta name="DC.contributor" content="Brian Seed" scheme="inventor"><meta name="DC.contributor" content="The General Hospital Corporation" scheme="assignee"><meta name="DC.date" content="1994-9-19" scheme="dateSubmitted"><meta name="DC.description" content="The invention features a synthetic gene encoding a protein normally expressed in mammalian cells wherein at least one non-preferred or less preferred codon in the natural gene encoding the mammalian protein has been replaced by a preferred codon encoding the same amino acid."><meta name="DC.date" content="1998-7-28" scheme="issued"><meta name="DC.relation" content="US:4652639" scheme="references"><meta name="DC.relation" content="US:5082767" scheme="references"><meta name="DC.relation" content="US:5270171" scheme="references"><meta name="DC.relation" content="US:5276268" scheme="references"><meta name="DC.relation" content="US:5405776" scheme="references"><meta name="DC.relation" content="US:5464774" scheme="references"><meta name="citation_reference" content="Bosch, M. L., et al., Journal of Virology, vol. 68, No. 11, &quot;Insertion of N-linked glycosylation sites in the variable regions of the human immunodeficiency virus type-1 surface protein through AAT triplet reiteration&quot;, pp. 7566-7569, 1994."><meta name="citation_reference" content="Bosch, M. L., et al., Journal of Virology, vol. 68, No. 11, Insertion of N linked glycosylation sites in the variable regions of the human immunodeficiency virus type 1 surface protein through AAT triplet reiteration , pp. 7566 7569, 1994."><meta name="citation_reference" content="Chou, K. C., et al., AIDS Research and Human Retroviruses, vol. 8, No. 12, Diagrammatization of codon usage in 339 human immunodeficiency virus proteins and its biological implication , pp. 1967 1976, 1992."><meta name="citation_reference" content="Chou, K.-C., et al., AIDS Research and Human Retroviruses, vol. 8, No. 12, &quot;Diagrammatization of codon usage in 339 human immunodeficiency virus proteins and its biological implication&quot;, pp. 1967-1976, 1992."><meta name="citation_reference" content="Cochrane et al., &quot;Identification and Charaterization of Intragenic Sequences Which Repress Human Immunodeficiency Virus Structural Gene Expression&quot;, Journal of Virology, 65(10):5305-5314, (1991)."><meta name="citation_reference" content="Cochrane et al., Identification and Charaterization of Intragenic Sequences Which Repress Human Immunodeficiency Virus Structural Gene Expression , Journal of Virology, 65(10):5305 5314, (1991)."><meta name="citation_reference" content="Cohen, J., et al., &quot;Expression of the HIV-1 env (gp160) in the yeast Saccharomyces cerevisiae via expression/secretion vectors and partial characterization of the gene product&quot;, in Modern Approaches, New Vaccines, pages 1988."><meta name="citation_reference" content="Cohen, J., et al., (abstract), in Modern Approaches New Vaccines, &quot;Expression of the HIV-1 env (gp160) in the yeast Saccaromyces cerevisiae via expression/secretion vectors and partial characterization of the gene product&quot;, 1988."><meta name="citation_reference" content="Cohen, J., et al., (abstract), in Modern Approaches New Vaccines, Expression of the HIV 1 env (gp160) in the yeast Saccaromyces cerevisiae via expression/secretion vectors and partial characterization of the gene product , 1988."><meta name="citation_reference" content="Cohen, J., et al., Expression of the HIV 1 env (gp160) in the yeast Saccharomyces cerevisiae via expression/secretion vectors and partial characterization of the gene product , in Modern Approaches, New Vaccines, pages 1988."><meta name="citation_reference" content="Coulombe and Skup, Gene 46:89 95 (1986)."><meta name="citation_reference" content="Coulombe and Skup, Gene 46:89-95 (1986)."><meta name="citation_reference" content="D Agostino et al., The Rev Protein of Human Immunodeficiency Virus Type 1 Promotes Polysomal Association and Translation of gag/pol and vpu/env mRNAs , Molucular and Cellular Biology, 12(3):1375 1386, (1992)."><meta name="citation_reference" content="D&#39;Agostino et al., &quot;The Rev Protein of Human Immunodeficiency Virus Type 1 Promotes Polysomal Association and Translation of gag/pol and vpu/env mRNAs&quot;, Molucular and Cellular Biology, 12(3):1375-1386, (1992)."><meta name="citation_reference" content="Feinberg et al., &quot;HTLV-III Expression and Production Involve Complex Regulation at the Levels of Splicing and Translation of Viral RNA&quot;, Cell, 46:807-817, (1986)."><meta name="citation_reference" content="Feinberg et al., HTLV III Expression and Production Involve Complex Regulation at the Levels of Splicing and Translation of Viral RNA , Cell, 46:807 817, (1986)."><meta name="citation_reference" content="Fortkamp, E., et al., DNA, vol. 5, No. 6, &quot;Cloning and expression in Escherichia coli of a synthetic DNA for hirudin, the blood coagulation inhibitor in the leech&quot;, pp. 511-517, 1986."><meta name="citation_reference" content="Fortkamp, E., et al., DNA, vol. 5, No. 6, Cloning and expression in Escherichia coli of a synthetic DNA for hirudin, the blood coagulation inhibitor in the leech , pp. 511 517, 1986."><meta name="citation_reference" content="Hammarskjold et al., &quot;Regulation of Human Immunodeficiency Virus env Expression by the rev Gene Product&quot;Journal of Virology, 63(5):1959-1966, (1989)."><meta name="citation_reference" content="Hammarskjold et al., Regulation of Human Immunodeficiency Virus env Expression by the rev Gene Product Journal of Virology, 63(5):1959 1966, (1989)."><meta name="citation_reference" content="Hernan et al., Biochemistry 31:861908628 (1992)."><meta name="citation_reference" content="Holler, T. P., et al., Gene, vol. 136, Nos. 1 &amp; 2, &quot;HIV-1 integrase expressed in Escherichia coli from SyntheticC Gene&quot;, PP. 323-328 , 1993."><meta name="citation_reference" content="Holler, T. P., et al., Gene, vol. 136, Nos. 1 &amp; 2, HIV 1 integrase expressed in Escherichia coli from SyntheticC Gene , PP. 323 328 , 1993."><meta name="citation_reference" content="Inouye et al., FEBS Letters 341:277 280 (1994)."><meta name="citation_reference" content="Inouye et al., FEBS Letters 341:277-280 (1994)."><meta name="citation_reference" content="Kamiya et al., Jpn J. Cancer Res. 80:200 203 (1989)."><meta name="citation_reference" content="Kamiya et al., Jpn J. Cancer Res. 80:200-203 (1989)."><meta name="citation_reference" content="McCarrey, Nucleic Acids Res. 18:949 955 (1990)."><meta name="citation_reference" content="McCarrey, Nucleic Acids Res. 18:949-955 (1990)."><meta name="citation_reference" content="Nakamura et al., &quot;Two types of linkage between codon usage and gene-expression levels&quot;, FEBS LETTERS, 289(1):123-125, (1991)."><meta name="citation_reference" content="Nakamura et al., Two types of linkage between codon usage and gene expression levels , FEBS LETTERS , 289(1):123 125, (1991)."><meta name="citation_reference" content="Newgard et al., Proc. Natl. Acad. Sci. USA 83:8132 8136 (1986)."><meta name="citation_reference" content="Newgard et al., Proc. Natl. Acad. Sci. USA 83:8132-8136 (1986)."><meta name="citation_reference" content="Rangwala, S. H., et al., Gene, vol. 122, No. 2, &quot;High-level production of active HIV-1 protease in Escherichia coli&quot;, pp. 263-269."><meta name="citation_reference" content="Rangwala, S. H., et al., Gene, vol. 122, No. 2, High level production of active HIV 1 protease in Escherichia coli , pp. 263 269."><meta name="citation_reference" content="Robinson et al., &quot;Codon usage can affect efficiency of translation of genes in Escherichia coli&quot;, Nucleic Acids Research, 12(17):6663-6671, (1984)."><meta name="citation_reference" content="Robinson et al., Codon usage can affect efficiency of translation of genes in Escherichia coli , Nucleic Acids Research, 12(17):6663 6671, (1984)."><meta name="citation_reference" content="Scorer, C. A., et al., Gene, vol. 136, Nos. 1 &amp; 2, &quot;The intracellular production and secretion of HIV-1 envelope protein in the methylotrophic yeast Picha pastoris&quot;, pp. 111-119, 1993."><meta name="citation_reference" content="Scorer, C. A., et al., Gene, vol. 136, Nos. 1 &amp; 2, The intracellular production and secretion of HIV 1 envelope protein in the methylotrophic yeast Picha pastoris , pp. 111 119, 1993."><meta name="citation_reference" content="Sharp et al., Nucleic Acids Res. 16:8207 8211 (1988)."><meta name="citation_reference" content="Sharp et al., Nucleic Acids Res. 16:8207-8211 (1988)."><meta name="citation_reference" content="van Hemert, F. J., et al., Journal of Molecular Evolution, vol. 41, No. 2, &quot;The tendency of lentiviral open reading frames to become A-rich: Constraints imposed by viral genome organization and cellular tRNA availability&quot;, pp. 132-140, 1995."><meta name="citation_reference" content="van Hemert, F. J., et al., Journal of Molecular Evolution, vol. 41, No. 2, The tendency of lentiviral open reading frames to become A rich: Constraints imposed by viral genome organization and cellular tRNA availability , pp. 132 140, 1995."><meta name="citation_reference" content="Williams et al., Nucleic Acids Research 16:10453 10467 (1988)."><meta name="citation_reference" content="Williams et al., Nucleic Acids Research 16:10453-10467 (1988)."><meta name="citation_reference" content="Zhang et al., &quot;Graphic Analysis of Codon Usage Strategy in 1490 Human Proteins&quot;, Journal of Protein Chemistry, 12(3):329-335, (1993)."><meta name="citation_reference" content="Zhang et al., &quot;Low-usage codons in Escherichia coli, yeast, fruit fly and primates&quot;, Gene, 105(1):61-72 (1991)."><meta name="citation_reference" content="Zhang et al., Graphic Analysis of Codon Usage Strategy in 1490 Human Proteins , Journal of Protein Chemistry, 12(3):329 335, (1993)."><meta name="citation_reference" content="Zhang et al., Low usage codons in Escherichia coli , yeast, fruit fly and primates , Gene, 105(1):61 72 (1991)."><meta name="citation_patent_number" content="US:5786464"><meta name="citation_patent_application_number" content="US:08/324,243"><link rel="canonical" href="http://www.google.com/patents/US5786464"/><meta property="og:url" content="http://www.google.com/patents/US5786464"/><meta name="title" content="Patent US5786464 - Overexpression of mammalian and viral proteins"/><meta name="description" content="The invention features a synthetic gene encoding a protein normally expressed in mammalian cells wherein at least one non-preferred or less preferred codon in the natural gene encoding the mammalian protein has been replaced by a preferred codon encoding the same amino acid."/><meta property="og:title" content="Patent US5786464 - Overexpression of mammalian and viral proteins"/><meta property="og:type" content="book"/><meta property="og:site_name" content="Google Books"/><meta property="og:image" content="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><link rel="image_src" href="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><script>(function(){try{var aa=function(a,b,c,d){d=d||{};d._sn=["cfg",b,c].join(".");window.gbar.logger.ml(a,d)};var g=window.gbar=window.gbar||{},l=window.gbar.i=window.gbar.i||{},m={},n;function _tvn(a,b){var c=parseInt(a,10);return isNaN(c)?b:c}function _tvf(a,b){var c=parseFloat(a);return isNaN(c)?b:c}function _tvv(a){return!!a}function p(a,b,c){(c||g)[a]=b}g.bv={n:_tvn("2",0),r:"",f:".67.",e:"0",m:_tvn("0",1)};
function q(a,b,c){var d="on"+b;if(a.addEventListener)a.addEventListener(b,c,!1);else if(a.attachEvent)a.attachEvent(d,c);else{var f=a[d];a[d]=function(){var a=f.apply(this,arguments),b=c.apply(this,arguments);return void 0==a?b:void 0==b?a:b&&a}}}var s=function(a){return function(){return g.bv.m==a}},ba=s(1),ca=s(2);p("sb",ba);p("kn",ca);l.a=_tvv;l.b=_tvf;l.c=_tvn;l.i=aa;var da=window.gbar.i.i;var t,u,v,w;function ea(a){v=a}function fa(a){var b;if(b=v&&window.encodeURIComponent)b=a.href,b=!b.match(/^http[s]?:\/\/accounts\.google\.[^/]*\/ClearSID/i)&&!b.match(/^http[s]?:\/\/[^/]*\/accounts\/ClearSID/i);if(b=b&&encodeURIComponent(v()))a.href=a.href.replace(/([?&]continue=)[^&]*/,"$1"+b)}function ga(a){window.gApplication&&(a.href=window.gApplication.getTabUrl(a.href))}
function ha(a){var b=document.forms[0].q,c=window.encodeURIComponent&&b&&b.value,b=b&&b.placeholder;c&&c!=b&&(a.href=a.href.replace(/([?&])q=[^&]*|$/,function(a,b){return(b||"&")+"q="+encodeURIComponent(c)}))}n=l.a("")?ga:ha;
function x(a,b,c,d,f,e){var h=document.getElementById(a);if(h){var k=h.style;k.left=d?"auto":b+"px";k.right=d?b+"px":"auto";k.top=c+"px";k.visibility=u?"hidden":"visible";f&&e?(k.width=f+"px",k.height=e+"px"):(x(t,b,c,d,h.offsetWidth,h.offsetHeight),u=u?"":a)}}
var y=[],ia=function(a,b){y.push(b)},ja=function(a){a=a||window.event;var b=a.target||a.srcElement;a.cancelBubble=!0;null==t&&(a=document.createElement(Array.every||window.createPopup?"iframe":"div"),a.frameBorder="0",t=a.id="gbs",a.src="javascript:''",b.parentNode.appendChild(a),q(document,"click",z));var c=b,b=0;"gb3"!=c.className&&(c=c.parentNode);a=c.getAttribute("aria-owns")||"gbi";var d=c.offsetWidth,f=20<c.offsetTop?46:24;document.getElementById("tphdr")&&(f-=3);var e=!1;do b+=c.offsetLeft||
0;while(c=c.offsetParent);var c=(document.documentElement.clientWidth||document.body.clientWidth)-b-d,h,d=document.body,k=document.defaultView;k&&k.getComputedStyle?(d=k.getComputedStyle(d,""))&&(h=d.direction):h=d.currentStyle?d.currentStyle.direction:d.style.direction;h="rtl"==h;if("gbi"==a){for(d=0;k=y[d++];)k();A(null,window.navExtra);h&&(b=c,e=!0)}else h||(b=c,e=!0);u!=a&&z();x(a,b,f,e)},z=function(){u&&x(u,0,0)},A=function(a,b){var c,d=document.getElementById("gbi"),f=a;f||(f=d.firstChild);
for(;b&&(c=b.pop());){var e=d,h=c,k=f;w||(w="gb2");e.insertBefore(h,k).className=w}},ka=function(a,b,c){if((b=document.getElementById(b))&&a){a.className="gb4";var d=document.createElement("span");d.appendChild(a);d.appendChild(document.createTextNode(" | "));d.id=c;b.appendChild(d)}},la=function(){return document.getElementById("gb_70")},ma=function(){return!!u};p("qs",n);p("setContinueCb",ea);p("pc",fa);p("tg",ja);p("close",z);p("addLink",ka);p("almm",A);p("si",la);p("adh",ia);p("op",ma);var B=function(){},C=function(){},F=function(a){var b=new Image,c=D;b.onerror=b.onload=b.onabort=function(){try{delete E[c]}catch(a){}};E[c]=b;b.src=a;D=c+1},E=[],D=0;p("logger",{il:C,ml:B,log:F});var G=window.gbar.logger;var H={},na={},I=[],oa=l.b("0.1",.1),pa=l.a("1",!0),qa=function(a,b){I.push([a,b])},ra=function(a,b){H[a]=b},sa=function(a){return a in H},J={},K=function(a,b){J[a]||(J[a]=[]);J[a].push(b)},ta=function(a){K("m",a)},L=function(a,b){var c=document.createElement("script");c.src=a;c.async=pa;Math.random()<oa&&(c.onerror=function(){c.onerror=null;B(Error("Bundle load failed: name="+(b||"UNK")+" url="+a))});(document.getElementById("xjsc")||document.getElementsByTagName("body")[0]||
document.getElementsByTagName("head")[0]).appendChild(c)},N=function(a){for(var b=0,c;(c=I[b])&&c[0]!=a;++b);!c||c[1].l||c[1].s||(c[1].s=!0,M(2,a),c[1].url&&L(c[1].url,a),c[1].libs&&m.d&&m.d(c[1].libs))},O=function(a){K("gc",a)},P=null,ua=function(a){P=a},M=function(a,b,c){if(P){a={t:a,b:b};if(c)for(var d in c)a[d]=c[d];try{P(a)}catch(f){}}};p("mdc",H);p("mdi",na);p("bnc",I);p("qGC",O);p("qm",ta);p("qd",J);p("lb",N);p("mcf",ra);p("bcf",qa);p("aq",K);p("mdd","");p("has",sa);
p("trh",ua);p("tev",M);var Q=l.b("0.1",.001),R=0;
function _mlToken(a,b){try{if(1>R){R++;var c,d=a,f=b||{},e=encodeURIComponent,h=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&jexpid=",e("17483"),"&srcpg=",e("prop=22"),"&jsr=",Math.round(1/Q),"&ogev=",e("blPsU_XDC-G7sASVhIDoBQ"),"&ogf=",g.bv.f,"&ogrp=",e("1"),"&ogv=",e("1407723702.0"),"&oggv="+e("es_plusone_gc_20140723.0_p0"),"&ogd=",e("com"),"&ogc=",e("CZE"),"&ogl=",e("en")];f._sn&&(f._sn="og."+
f._sn);for(var k in f)h.push("&"),h.push(e(k)),h.push("="),h.push(e(f[k]));h.push("&emsg=");h.push(e(d.name+":"+d.message));var r=h.join("");S(r)&&(r=r.substr(0,2E3));c=r;var Aa=window.gbar.logger._aem(a,c);F(Aa)}}catch(Na){}}var S=function(a){return 2E3<=a.length},va=function(a,b){return b};function T(a){B=a;p("_itl",S,G);p("_aem",va,G);p("ml",B,G);a={};H.er=a}l.a("")?T(function(a){throw a;}):l.a("1")&&Math.random()<Q&&T(_mlToken);I.push(["m",{url:"//ssl.gstatic.com/gb/js/scm_7385cc5883250b43a39405734c1bea59.js"}]);g.mcf("c",{});g.sg={c:""};if(l.a("1")){var wa=l.a("");I.push(["gc",{auto:wa,url:"//ssl.gstatic.com/gb/js/abc/gci_91f30755d6a6b787dcc2a4062e6e9824.js",libs:"googleapis.client:plusone:gapi.iframes"}]);var xa={version:"gci_91f30755d6a6b787dcc2a4062e6e9824.js",index:"",lang:"en"};H.gc=xa;var U=function(a){window.googleapis&&window.iframes?a&&a():(a&&O(a),N("gc"))};p("lGC",U);l.a("1")&&p("lPWF",U)};window.__PVT="";if(l.a("1")&&l.a("1")){var V=function(a){U(function(){K("pw",a);N("pw")})};p("lPW",V);I.push(["pw",{url:"//ssl.gstatic.com/gb/js/abc/pwm_45f73e4df07a0e388b0fa1f3d30e7280.js"}]);var W=[],ya=function(a){W[0]=a},za=function(a,b){var c=b||{};c._sn="pw";B(a,c)},Ba={signed:W,elog:za,base:"https://plusone.google.com/u/0",loadTime:(new Date).getTime()};H.pw=Ba;var X=function(a,b){for(var c=b.split("."),d=function(){var b=arguments;a(function(){for(var a=g,d=0,e=c.length-1;d<e;++d)a=a[c[d]];a[c[d]].apply(a,b)})},f=g,e=0,h=c.length-1;e<h;++e)f=
f[c[e]]=f[c[e]]||{};return f[c[e]]=d};X(V,"pw.clk");X(V,"pw.hvr");p("su",ya,g.pw)};function Ca(){function a(){for(var b;(b=e[h++])&&"m"!=b[0]&&!b[1].auto;);b&&(M(2,b[0]),b[1].url&&L(b[1].url,b[0]),b[1].libs&&m.d&&m.d(b[1].libs));h<e.length&&setTimeout(a,0)}function b(){0<f--?setTimeout(b,0):a()}var c=l.a("1"),d=l.a(""),f=3,e=I,h=0,k=window.gbarOnReady;if(k)try{k()}catch(r){da(r,"ml","or")}d?p("ldb",a):c?q(window,"load",b):b()}p("rdl",Ca);var Da={D:1,H:2,da:3,p:4,W:5,M:6,F:7,g:8,ha:9,U:10,L:11,T:12,S:13,N:14,Q:15,P:16,fa:17,w:18,O:19,ga:20,ea:21,u:22,G:23,ja:24,ka:25,ia:26,A:27,j:28,o:29,k:30,ca:31,Z:32,$:33,J:34,K:35,ba:36,aa:37,Y:38,B:39,R:40,v:41,X:42,V:43,h:48,C:49,I:500},Y=[1,2,3,4,5,6,9,10,11,13,14,28,29,30,34,35,37,38,39,40,41,42,43,48,49,500];var Z=l.b("0.001",1E-4),Ea=l.b("1",1),Fa=!1,Ga=!1;if(l.a("1")){var Ha=Math.random();Ha<=Z&&(Fa=!0);Ha<=Ea&&(Ga=!0)}var Ia=Da,$=null;function Ja(){var a=0,b=function(b,d){l.a(d)&&(a|=b)};b(1,"");b(2,"");b(4,"");b(8,"");return a}
function Ka(a,b){var c=Z,d=Fa,f;f=a;if(!$){$={};for(var e=0;e<Y.length;e++){var h=Y[e];$[h]=!0}}if(f=!!$[f])c=Ea,d=Ga;if(d){d=encodeURIComponent;g.rp?(f=g.rp(),f="-1"!=f?f:"1"):f="1";c=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&oge=",a,"&ogex=",d("17483"),"&ogev=",d("blPsU_XDC-G7sASVhIDoBQ"),"&ogf=",g.bv.f,"&ogp=",d("22"),"&ogrp=",d(f),"&ogsr=",Math.round(1/c),"&ogv=",d("1407723702.0"),"&oggv="+
d("es_plusone_gc_20140723.0_p0"),"&ogd=",d("com"),"&ogl=",d("en"),"&ogc=",d("CZE"),"&ogus=",Ja()];if(b){"ogw"in b&&(c.push("&ogw="+b.ogw),delete b.ogw);var k;f=b;e=[];for(k in f)0!=e.length&&e.push(","),e.push(La(k)),e.push("."),e.push(La(f[k]));k=e.join("");""!=k&&(c.push("&ogad="),c.push(d(k)))}F(c.join(""))}}function La(a){"number"==typeof a&&(a+="");return"string"==typeof a?a.replace(".","%2E").replace(",","%2C"):a}C=Ka;p("il",C,G);var Ma={};H.il=Ma;setTimeout(function(){C(Ia.g)},0);}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var b=window.gbar.i.i;var c=window.gbar;var f=function(d){try{var a=document.getElementById("gbom");a&&d.appendChild(a.cloneNode(!0))}catch(e){b(e,"omas","aomc")}};c.aomc=f;}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var a=window.gbar;a.mcf("pm",{p:""});}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{window.gbar.rdl();}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
if (window['_OC_timingAction']) {window['_OC_timingAction']('patents_refpage');}</script><style>#gbar,#guser{font-size:13px;padding-top:1px !important;}#gbar{float:left;height:22px}#guser{padding-bottom:7px !important;text-align:right}.gbh,.gbd{border-top:1px solid #c9d7f1;font-size:1px}.gbh{height:0;position:absolute;top:24px;width:100%}#gbs,.gbm{background:#fff;left:0;position:absolute;text-align:left;visibility:hidden;z-index:1000}.gbm{border:1px solid;border-color:#c9d7f1 #36c #36c #a2bae7;z-index:1001}.gb1{margin-right:.5em}.gb1,.gb3{zoom:1}.gb2{display:block;padding:.2em .5em}.gb2,.gb3{text-decoration:none !important;border-bottom:none}a.gb1,a.gb4{text-decoration:underline !important}a.gb1,a.gb2,a.gb3,a.gb4{color:#00c !important}.gbi .gb3,.gbi .gb2,.gbi .gb4{color:#dd8e27 !important}.gbf .gb3,.gbf .gb2,.gbf .gb4{color:#900 !important}a.gb2:hover{background:#36c;color:#fff !important}#gbar .gbz0l{color:#000 !important;cursor:default;font-weight:bold;text-decoration:none !important}
#gbar { padding:.3em .6em !important;}</style></head><body  topmargin="3" marginheight="3"><div id=gbar><nobr><a onclick=gbar.qs(this);gbar.logger.il(1,{t:1}); class=gb1 id=gb_1 href="https://www.google.com/search?sa=N&tab=tw">Search</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:2}); class=gb1 id=gb_2 href="http://www.google.com/search?hl=en&tbm=isch&source=og&sa=N&tab=ti">Images</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:8}); class=gb1 id=gb_8 href="http://maps.google.com/maps?hl=en&sa=N&tab=tl">Maps</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:78}); class=gb1 id=gb_78 href="https://play.google.com/?hl=en&sa=N&tab=t8">Play</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:36}); class=gb1 id=gb_36 href="http://www.youtube.com/results?sa=N&tab=t1">YouTube</a> <a onclick=gbar.logger.il(1,{t:5}); class=gb1 id=gb_5 href="http://news.google.com/nwshp?hl=en&tab=tn">News</a> <a onclick=gbar.logger.il(1,{t:23}); class=gb1 id=gb_23 href="https://mail.google.com/mail/?tab=tm">Gmail</a> <a onclick=gbar.logger.il(1,{t:25}); class=gb1 id=gb_25 href="https://drive.google.com/?tab=to">Drive</a> <a class=gb3 href="http://www.google.com/intl/en/options/" onclick="this.blur();gbar.tg(event);return !1" aria-haspopup=true><u>More</u> <small>&#9660;</small></a><div class=gbm id=gbi><a onclick=gbar.logger.il(1,{t:24}); class=gb2 id=gb_24 href="https://www.google.com/calendar?tab=tc">Calendar</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:51}); class=gb2 id=gb_51 href="http://translate.google.com/?hl=en&sa=N&tab=tT">Translate</a><a onclick=gbar.logger.il(1,{t:17}); class=gb2 id=gb_17 href="http://www.google.com/mobile/?hl=en&tab=tD">Mobile</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:10}); class=gb2 id=gb_10 href="http://www.google.com/search?hl=en&tbo=u&tbm=bks&source=og&sa=N&tab=tp">Books</a><a onclick=gbar.logger.il(1,{t:212}); class=gb2 id=gb_212 href="https://wallet.google.com/manage/?tab=ta">Wallet</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:6}); class=gb2 id=gb_6 href="http://www.google.com/search?hl=en&tbo=u&tbm=shop&source=og&sa=N&tab=tf">Shopping</a><a onclick=gbar.logger.il(1,{t:30}); class=gb2 id=gb_30 href="http://www.blogger.com/?tab=tj">Blogger</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:27}); class=gb2 id=gb_27 href="http://www.google.com/finance?sa=N&tab=te">Finance</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:31}); class=gb2 id=gb_31 href="https://plus.google.com/photos?sa=N&tab=tq">Photos</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:12}); class=gb2 id=gb_12 href="http://www.google.com/search?hl=en&tbo=u&tbm=vid&source=og&sa=N&tab=tv">Videos</a><div class=gb2><div class=gbd></div></div><a onclick=gbar.logger.il(1,{t:66}); href="http://www.google.com/intl/en/options/" class=gb2>Even more &raquo;</a></div></nobr></div><div id=guser width=100%><nobr><span id=gbn class=gbi></span><span id=gbf class=gbf></span><span id=gbe></span><a target=_top id=gb_70 href="https://www.google.com/accounts/Login?service=&continue=http://www.google.com/patents%3Fhl%3Den&hl=en" class=gb4>Sign in</a><div style="display: none"><div class=gbm id=gbd5 aria-owner=gbg5><div class=gbmc><ol id=gbom class=gbmcc></ol></div></div></div></nobr></div><div class=gbh style=left:0></div><div class=gbh style=right:0></div><div role="alert" style="position: absolute; left: 0; right: 0;"><a href="http://www.google.com/patents/us5786464?hl=en&amp;output=html_text" title="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."><img border="0" src="http://www.google.com/images/cleardot.gif"alt="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."></a></div><div id="guser"><nobr></nobr></div><div style="clear:both;"></div><div id="gb-top-search-box" class="gb-top-search-box-small gb-reset"><table><tr><td class="logo"><a href="http://www.google.com/patents" class="logo-link"><img class="logo-img" src="/intl/en/images/logos/google_logo_41.png" alt="Go to Google Books Home" height="41"/></a></td><td><form action="http://www.google.com/search" name="f" id="vheadf" method="get"><span id="hf"></span><input type="hidden" name="tbm" value="pts"/><input type="hidden" name="tbo" value="1"/><input type="hidden" name="hl" value="en"/><table><tr><td><div class="inputs"><table><tr><td><div class="text-input"><input type="text" name="q" id="vheadq" class="text" maxlength="2048" size="31" value="" title="Search Patents" accesskey="s" autocomplete="off"/><script>window._OC_autoDir &&window._OC_autoDir('vheadq', 'tia-vheadq');</script></div></td><td><div class="submit-input"><input name="btnG" class="submit" type="submit" value=""/></div></td></tr></table></div></td><td class="col-ext-links"><div class="ext-links"><a href="http://www.google.com/advanced_patent_search">&lt;nobr&gt;Advanced Patent Search&lt;/nobr&gt;</a></div></td></tr></table></form></td></tr></table></div><div class="kd-appbar"><h2 class="kd-appname"><a href="/patents">Patents</a></h2><div class="kd-buttonbar left" id="left-toolbar-buttons"><a id="appbar-write-review-link" href=""></a><a id="appbar-view-print-sample-link" href=""></a><a id="appbar-view-ebook-sample-link" href=""></a><a id="appbar-patents-prior-art-finder-link" href="https://www.google.com/patents/related/US5786464"></a><a id="appbar-patents-discuss-this-link" href="http://www.google.com/url?id=TylHBAABERAJ&amp;q=http://patents.stackexchange.com/redirect/google-patents%3Fpatent%3DUS5786464&amp;usg=AFQjCNHBT-IfhsgL28bMS7bgdWTd9-mQ3Q" data-is-grant="true"></a><a id="appbar-read-patent-link" href="//docs.google.com/viewer?url=patentimages.storage.googleapis.com/pdfs/US5786464.pdf"></a><a id="appbar-download-pdf-link" href="//patentimages.storage.googleapis.com/pdfs/US5786464.pdf"></a></div><div class="kd-buttonbar right" id="right-toolbar-buttons"></div></div><div id="books-microdata" itemscope=""itemtype="http://schema.org/Book"itemid="http://www.google.com/patents/US5786464" style="display:none"><span itemprop="description">The invention features a synthetic gene encoding a protein normally expressed in mammalian cells wherein at least one non-preferred or less preferred codon in the natural gene encoding the mammalian protein has been replaced by a preferred codon encoding the same amino acid....</span><span itemprop="url">http://www.google.com/patents/US5786464?utm_source=gb-gplus-share</span><span class="main-title" itemprop="name">Patent US5786464 - Overexpression of mammalian and viral proteins</span><img itemprop="image" src="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"alt="Patent US5786464 - Overexpression of mammalian and viral proteins" title="Patent US5786464 - Overexpression of mammalian and viral proteins"></div><div style="display: none"><ol id="ofe-gear-menu-contents" class="gbmcc"><li class="gbe gbmtc"><a class="gbmt goog-menuitem-content" id="" href="http://www.google.com/advanced_patent_search">Advanced Patent Search</a></li></ol></div><table id="viewport_table" cellpadding="0" style="clear:both" cellspacing="0"><tr><td id="viewport_td"><div class=vertical_module_list_row><div id=intl_patents class=about_content><div id=intl_patents_v><table class="patent-bibdata"><tr><td class="patent-bibdata-heading">Publication number</td><td class="single-patent-bibdata">US5786464 A</td></tr><tr><td class="patent-bibdata-heading">Publication type</td><td class="single-patent-bibdata">Grant</td></tr><tr><td class="patent-bibdata-heading">Application number</td><td class="single-patent-bibdata">US 08/324,243</td></tr><tr><td class="patent-bibdata-heading">Publication date</td><td class="single-patent-bibdata">Jul 28, 1998</td></tr><tr><td class="patent-bibdata-heading">Filing date</td><td class="single-patent-bibdata">Sep 19, 1994</td></tr><tr><td class="patent-bibdata-heading">Priority date<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The priority date is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the date listed."></span></td><td class="single-patent-bibdata">Sep 19, 1994</td></tr><tr><td class="patent-bibdata-heading">Fee status<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The fee status is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the status or dates listed."></span></td><td class="single-patent-bibdata">Paid</td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Also published as</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/CA2200342A1">CA2200342A1</a>, </span><span class="patent-bibdata-value"><a href="/patents/CA2200342C">CA2200342C</a>, </span><span class="patent-bibdata-value"><a href="/patents/DE69535147D1">DE69535147D1</a>, </span><span class="patent-bibdata-value"><a href="/patents/DE69535147T2">DE69535147T2</a>, </span><span class="patent-bibdata-value"><a href="/patents/DE69535147T3">DE69535147T3</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP0781329A1">EP0781329A1</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP0781329A4">EP0781329A4</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP0781329B1">EP0781329B1</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP0781329B2">EP0781329B2</a>, </span><span class="patent-bibdata-value"><a href="/patents/WO1996009378A1">WO1996009378A1</a></span></span></td></tr><tr class="patent-bibdata-list-row alternate-patent-number"><td class="patent-bibdata-heading">Publication number</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value">08324243, </span><span class="patent-bibdata-value">324243, </span><span class="patent-bibdata-value">US 5786464 A, </span><span class="patent-bibdata-value">US 5786464A, </span><span class="patent-bibdata-value">US-A-5786464, </span><span class="patent-bibdata-value">US5786464 A, </span><span class="patent-bibdata-value">US5786464A</span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Inventors</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Brian+Seed%22">Brian Seed</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Original Assignee</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=inassignee:%22The+General+Hospital+Corporation%22">The General Hospital Corporation</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Export Citation</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/US5786464.bibtex">BiBTeX</a>, </span><span class="patent-bibdata-value"><a href="/patents/US5786464.enw">EndNote</a>, </span><span class="patent-bibdata-value"><a href="/patents/US5786464.ris">RefMan</a></span></span></td></tr><tr class="patent-internal-links"><td colspan=2><span class="patent-bibdata-value"><a href="#backward-citations">Patent Citations</a> (6),</span> <span class="patent-bibdata-value"><a href="#npl-citations">Non-Patent Citations</a> (49),</span> <span class="patent-bibdata-value"><a href="#forward-citations">Referenced by</a> (46),</span> <span class="patent-bibdata-value"><a href="#classifications">Classifications</a> (22),</span> <span class="patent-bibdata-value"><a href="#legal-events">Legal Events</a> (11)</span> </td></tr><tr><td colspan=2 class="patent-bibdata-external-link-spacer-top"></td></tr><tr class="patent-bibdata-external-link-spacer-bottom"></tr><tr><td colspan=2><span class="patent-bibdata-heading">External Links:&nbsp;</span><span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=TylHBAABERAJ&q=http://patft.uspto.gov/netacgi/nph-Parser%3FSect2%3DPTO1%26Sect2%3DHITOFF%26p%3D1%26u%3D/netahtml/PTO/search-bool.html%26r%3D1%26f%3DG%26l%3D50%26d%3DPALL%26RefSrch%3Dyes%26Query%3DPN/5786464&usg=AFQjCNE25-eRM1HlnzsAQ-n2Oqa2oAeRZw">USPTO</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=TylHBAABERAJ&q=http://assignments.uspto.gov/assignments/q%3Fdb%3Dpat%26pat%3D5786464&usg=AFQjCNGLWGJM_wYNtogPGHk5Z59c-GeIfQ">USPTO Assignment</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=TylHBAABERAJ&q=http://worldwide.espacenet.com/publicationDetails/biblio%3FCC%3DUS%26NR%3D5786464A%26KC%3DA%26FT%3DD&usg=AFQjCNFStot1VbFr4YdfB5ZM6Gz4GPvS3A">Espacenet</a></span></span></td></tr><tr class="patent-bibdata-group-spacer"></tr></table><div class="number-and-title"><span class="patent-title"><invention-title mxw-id="PT54319502" lang="EN" load-source="patent-office">Overexpression of mammalian and viral proteins</invention-title></span><br><span class="patent-number">US 5786464 A</span></div><div class="patent-section patent-abstract-section"><div class="patent-section-header"><span class="patent-section-title">Abstract</span></div><div class="patent-text"><abstract mxw-id="PA37794407" lang="EN" load-source="patent-office"> <div class="abstract">The invention features a synthetic gene encoding a protein normally expressed in mammalian cells wherein at least one non-preferred or less preferred codon in the natural gene encoding the mammalian protein has been replaced by a preferred codon encoding the same amino acid.</div>
  </abstract></div></div><div class="patent-section patent-drawings-section"><div class="patent-section-header"><span class="patent-section-title">Images<span class="patent-section-count">(10)</span></span></div><div class="patent-drawings-body"><div class="patent-drawings-carousel"><div class="drawings"><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5786464-2.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5786464-2.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5786464-3.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5786464-3.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5786464-4.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5786464-4.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5786464-5.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5786464-5.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5786464-6.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5786464-6.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5786464-7.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5786464-7.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5786464-8.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5786464-8.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5786464-9.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5786464-9.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5786464-10.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5786464-10.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5786464-11.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5786464-11.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div></div></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img" alt="Previous page"src="/googlebooks/images/kennedy/page_left.png"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img" alt="Next page"src="/googlebooks/images/kennedy/page_right.png"width="21" height="21" /></div></div></div><div class="patent-post-drawings"></div><div class="patent-section patent-claims-section"><div class="patent-section-header"><span class="patent-section-title">Claims<span class="patent-section-count">(18)</span></span></div><div class="patent-text"><div mxw-id="PCLM5260762" lang="EN" load-source="patent-office" class="claims">
    <claim-statement>What is claimed is:</claim-statement> <div class="claim"> <div num="1" class="claim">
      <div class="claim-text">1. A synthetic gene encoding a protein normally expressed in mammalian cells wherein at least one non-preferred or less preferred codon in the natural gene encoding said protein has been replaced by a preferred codon encoding the same amino acid, said preferred codons being selected from the group consisting of gcc, cgc, aac, gac, tgc, cag, ggc, cac, atc, ctg, aag, ccc, ttc, agc, acc, tac, and gtg, said less preferred codons being selected from the group consisting of ggg, att, ctc, tcc, and gtc, said non-preferred codons being all codons other than said preferred codons and said less preferred codons, wherein said synthetic gene permits the expression of said protein in a mammalian host cell at a level which is at least 110% of that expressed by said natural gene in an in vitro mammalian cell culture system under identical conditions.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="2" class="claim">
      <div class="claim-text">2. The synthetic gene of claim 1 wherein said synthetic gene permits the expression of said protein in a mammalian host cell at a level which is at least 150% of that expressed by said natural gene in an in vitro mammalian cell culture system under identical conditions.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="3" class="claim">
      <div class="claim-text">3. The synthetic gene of claim 1 wherein said synthetic gene permits the expression of said protein in a mammalian host cell at a level which is at least 200% of that expressed by said natural gene in an in vitro mammalian cell culture system under identical conditions.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="4" class="claim">
      <div class="claim-text">4. The synthetic gene of claim 1 wherein said synthetic gene permits the expression of said protein in a mammalian host cell at a level which is at least 500% of that expressed by said natural gene in an in vitro mammalian cell culture system under identical conditions.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="5" class="claim">
      <div class="claim-text">5. The synthetic gene of claim 1 wherein said synthetic gene permits the expression of said protein in a mammalian host cell at a level which is at least ten times that expressed by said natural gene in an in vitro mammalian cell culture system under identical conditions.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="6" class="claim">
      <div class="claim-text">6. The synthetic gene of claim 1 wherein at least 10% of the codons in said natural gene are non-preferred codons.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="7" class="claim">
      <div class="claim-text">7. The synthetic gene of claim 1 wherein at least 50% of the codons in said natural gene are non-preferred codons.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="8" class="claim">
      <div class="claim-text">8. The synthetic gene of claim 1 wherein at least 50% of the non-preferred codons and less preferred codons present in said natural gene have been replaced by preferred codons.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="9" class="claim">
      <div class="claim-text">9. The synthetic gene of claim 1 wherein at least 90% of the non-preferred codons and less preferred codons present in said natural gene have been replaced by preferred codons.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="10" class="claim">
      <div class="claim-text">10. The synthetic gene of claim 1 wherein said protein is a retroviral protein.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="11" class="claim">
      <div class="claim-text">11. The synthetic gene of claim 1 wherein said protein is a human protein.</div>
    </div>
    </div> <div class="claim"> <div num="12" class="claim">
      <div class="claim-text">12. A synthetic gene encoding a lentiviral protein wherein at least one non-preferred or less preferred codon in the natural gene encoding said lentiviral protein has been replaced by a preferred codon encoding the same amino acid, said preferred codons being selected from the group consisting of gcc, cgc, aac, gac, tgc, cag, ggc, cac, atc, ctg, aag, ccc, ttc, agc, acc, tac, and gtg, said less preferred codons being selected from the group consisting of ggg, att, ctc, tcc, and gtc, said non-preferred codons being all codons other than said preferred codons and said less preferred codons, wherein said synthetic gene permits the expression of said lentiviral protein in a mammalian host cell at a level which is at least 110% of that expressed by said natural gene in an in vitro mammalian cell culture system under identical conditions.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="13" class="claim">
      <div class="claim-text">13. The synthetic gene of claim 12 wherein said lentiviral protein is an HIV protein.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="14" class="claim">
      <div class="claim-text">14. The synthetic gene of claim 13 wherein said lentiviral protein is selected from the group consisting of gag, pol, and env.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="15" class="claim">
      <div class="claim-text">15. The synthetic gene of claim 13 wherein said protein is gp120.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="16" class="claim">
      <div class="claim-text">16. The synthetic gene of claim 13 wherein said protein is gp160.</div>
    </div>
    </div> <div class="claim"> <div num="17" class="claim">
      <div class="claim-text">17. A method for preparing a synthetic gene encoding a protein normally expressed by mammalian cells, comprising identifying non-preferred and less preferred codons in the natural gene encoding said protein and replacing one or more of said non-preferred and less preferred codons with a preferred codon encoding the same amino acid as the replaced codon, said preferred codons being selected from the group consisting of gcc, cgc, aac, gac, tgc, cag, ggc, cac, atc, ctg, aag, ccc, ttc, agc, acc, tac, and gtg, said less preferred codons being selected from the group consisting of ggg, att, ctc, tcc, and gtc, said non-preferred codons being all codons other than said preferred codons and said less preferred codons, wherein said synthetic gene permits the expression of said protein in a mammalian host cell at a level which is at least 110% of that expressed by said natural gene in an in vitro mammalian cell culture system under identical conditions.</div>
    </div>
    </div> <div class="claim"> <div num="18" class="claim">
      <div class="claim-text">18. A method for preparing a synthetic gene encoding a lentiviral protein, comprising identifying non-preferred and less preferred codons in the natural gene encoding said lentiviral protein and replacing one or more of said non-preferred and less preferred codons with a preferred codon encoding the same amino acid as the replaced codon, said preferred codons being selected from the group consisting of gcc, cgc, aac, gac, tgc, cag, ggc, cac, atc, ctg, aag, ccc, ttc, agc, acc, tac, and gtg, said less preferred codons being selected from the group consisting of ggg, att, ctc, tcc, and gtc, said non-preferred codons being all codons other than said preferred codons and said less preferred codons, wherein said synthetic gene permits the expression of said lentiviral protein in a mammalian host cell at a level which is at least 110% of that expressed by said natural gene in an in vitro mammalian cell culture system under identical conditions.</div>
    </div>
  </div> </div></div></div><div class="patent-section patent-description-section"><div class="patent-section-header"><span class="patent-section-title">Description</span></div><div class="patent-text"><div mxw-id="PDES67152695" lang="EN" load-source="patent-office" class="description">
    <p>This invention was made with Government support under Contract #AI27849 and DK43031 awarded by the National Institutes of Health. The Government has certain rights in this invention.</p>
    <heading>FIELD OF THE INVENTION</heading> <p>The invention concerns genes and methods for expressing eukaryotic and viral proteins at high levels in eukaryotic cells.</p>
    <heading>BACKGROUND OF THE INVENTION</heading> <p>Expression of eukaryotic gene products in prokaryotes is sometimes limited by the presence of codons that are infrequently used in E. coli. Expression of such genes can be enhanced by systematic substitution of the endogenous codons with codons overrepresented in highly expressed prokaryotic genes (Robinson et al. 1984). It is commonly supposed that rare codons cause pausing of the ribosome, which leads to a failure to complete the nascent polypeptide chain and an uncoupling of transcription and translation. The mRNA 3' end of the stalled ribosome is exposed to cellular ribonucleases, which decreases the stability of the transcript.</p>
    <heading>SUMMARY OF THE INVENTION</heading> <p>The invention features a synthetic gene encoding a protein normally expressed in mammalian cells wherein at least one non-preferred or less preferred codon in the natural gene encoding the mammalian protein has been replaced by a preferred codon encoding the same amino acid.</p>
    <p>Preferred codons are: Ala (gcc); Arg (cgc); Asn (aac); Asp (gac) Cys (tgc); Gln (cag); Gly (ggc); His (cac); Ile (atc); Leu (ctg); Lys (aag); Pro (ccc); Phe (ttc); Ser (agc); Thr (acc); Tyr (tac); and Val (gtg). Less preferred codons are: Gly (ggg); Ile (att); Leu (ctc); Ser (tcc); Val (gtc). All codons which do not fit the description of preferred codons or less preferred codons are non-preferred codons.</p>
    <p>By "protein normally expressed in mammalian cells" is meant a protein which is expressed in mammalian cells under natural conditions. The term includes genes in the mammalian genome encoding polypeptides such as Factor VIII, Factor IX, interleukins, and other proteins. The term also includes genes which are expressed in a mammalian cell under disease conditions such as oncogenes as well as genes which are encoded by a virus (including a retrovirus) which are expressed in mammalian cells post-infection</p>
    <p>In preferred embodiments, the synthetic gene is capable of expressing said mammalian protein at a level which is at least 110%, 150%, 200%, 500%, 1,000%, or 10,000% of that expressed by said natural gene in an in vitro mammalian cell culture system under identical conditions (i.e., same cell type, same culture conditions, same expression vector).</p>
    <p>Suitable cell culture systems for measuring expression of the synthetic gene and corresponding natural gene are described below. Other suitable expression systems employing mammalian cells are well known to those skilled in the art and are described in, for example, the standard molecular biology reference works noted below. Vectors suitable for expressing the synthetic and natural genes are described below and in the standard reference works described below. By "expression" is meant protein expression. Expression can be measured using an antibody specific for the protein of interest. Such antibodies and measurement techniques are well known to those skilled in the art. By "natural gene" is meant the gene sequence which naturally encodes the protein.</p>
    <p>In other preferred embodiments at least 10%, 20%, 30%, 40%, 50%, 60%, 70%, 80%, or 90% of the codons in the natural gene are non-preferred codons.</p>
    <p>In a preferred embodiment the protein is a retroviral protein. In a more preferred embodiment the protein is a lentiviral protein. In an even more preferred embodiment the protein is an HIV protein. In other preferred embodiments the protein is gag, pol, env, gp120, or gp160. In other preferred embodiments the protein is a human protein.</p>
    <p>The invention also features a method for preparing a synthetic gene encoding a protein normally expressed by mammalian cells. The method includes identifying non-preferred and less-preferred codons in the natural gene encoding the protein and replacing one or more of the non-preferred and less-preferred codons with a preferred codon encoding the same amino acid as the replaced codon.</p>
    <p>Under some circumstances (e.g., to permit introduction of a restriction site) it may be desirable to replace a non-preferred codon with a less preferred codon rather than a preferred codon.</p>
    <p>It is not necessary to replace all less preferred or non-preferred codons with preferred codons. Increased expression can be accomplished even with partial replacement.</p>
    <p>In other preferred embodiments the invention features vectors (including expression vectors) comprising the synthetic gene.</p>
    <p>By "vector" is meant a DNA molecule, derived, e.g., from a plasmid, bacteriophage, or mammalian or insect virus, into which fragments of DNA may be inserted or cloned. A vector will contain one or more unique restriction sites and may be capable of autonomous replication in a defined host or vehicle organism such that the cloned sequence is reproducible. Thus, by "expression vector" is meant any autonomous element capable of directing the synthesis of a protein. Such DNA expression vectors include mammalian plasmids and viruses.</p>
    <p>The invention also features synthetic gene fragments which encode a desired portion of the protein. Such synthetic gene fragments are similar to the synthetic genes of the invention except that they encode only a portion of the protein. Such gene fragments preferably encode at least 50, 100, 150, or 500 contiguous amino acids of the protein.</p>
    <p>In constructing the synthetic genes of the invention it may be desirable to avoid CpG sequences as these sequences may cause gene silencing.</p>
    <p>The codon bias present in the HIV gp120 envelope gene is also present in the gag and pol proteins. Thus, replacement of a portion of the non-preferred and less preferred codons found in these genes with preferred codons should produce a gene capable of higher level expression. A large fraction of the codons in the human genes encoding Factor VIII and Factor IX are non-preferred codons or less preferred codons. Replacement of a portion of these codons with preferred codons should yield genes capable of higher level expression in mammalian cell culture. Conversely, it may be desirable to replace preferred codons in a naturally occurring gene with less-preferred codons as a means of lowering expression.</p>
    <p>Standard reference works describing the general principles of recombinant DNA technology include Watson, J. D. et al., Molecular Biology of the Gene, Volumes I and II, the Benjamin/Cummings Publishing Company, Inc., publisher, Menlo Park, Calif. (1987); Darnell, J. E. et al., Molecular Cell Biology, Scientific American Books, Inc., Publisher, New York, N.Y. (1986); Old, R. W., et al., Principles of Gene Manipulation: An Introduction to Genetic Engineering, 2d edition, University of California Press, publisher, Berkeley, Calif. (1981); Maniatis, T., et al., Molecular Cloning: A Laboratory Manual, 2nd Ed. Cold Spring Harbor Laboratory, publisher, Cold Spring Harbor, N.Y. (1989); and Current Protocols in Molecular Biology, Ausubel et al., Wiley Press, New York, N.Y. (1989).</p>
    <heading>DETAILED DESCRIPTION</heading> <heading>Description of the Drawings</heading> <p>FIG. 1 depicts the sequence of the synthetic gp120 and a synthetic gp160 gene in which codons have been replaced by those found in highly expressed human genes.</p>
    <p>FIG. 2 is a schematic drawing of the synthetic gp120 (HIV-1 MN) gene. The shaded portions marked v1 to v5 indicate hypervariable regions. The filled box indicates the CD4 binding site. A limited number of the unique restriction sites ares shown: H (Hind3), Nh (Nhe1), P (Pst1), Na (Nae1), M (Mlu1), R (EcoR1), A (Age1) and No (Not1). The chemically synthesized DNA fragments which served as PCR templates are shown below the gp120 sequence, along with the locations of the primers used for their amplification.</p>
    <p>FIG. 3 is a photograph of the results of transient transfection assays used to measure gp120 expression. Gel electrophoresis of immunoprecipitated supernatants of 293T cells transfected with plasmids expressing gp120 encoded by the IIIB isolate of HIV-1 (gp120IIIb), by the MN isolate (gp120mn), by the MN isolate modified by substitution of the endogenous leader peptide with that of the CD5 antigen (gp120mnCD5L), or by the chemically synthesized gene encoding the MN variant with the human CD5Leader (syngp120mn). Supernatants were harvested following a 12 hour labeling period 60 hours post-transfection and immunoprecipitated with CD4:IgG1 fusion protein and protein A sepharose.</p>
    <p>FIG. 4 is a graph depicting the results of ELISA assays used to measure protein levels in supernatants of transiently transfected 293T cells. Supernatants of 293T cells transfected with plasmids expressing gp120 encoded by the IIIB isolate of HIV-1 (gp120 IIIb), by the MN isolate (gp120mn), by the MN isolate modified by substitution of the endogenous leader peptide with that of CD5 antigen (gp120mn CD5L), or by the chemically synthesized gene encoding the MN variant with human CDS leader (syngp120mn) were harvested after 4 days and tested in a gp120/CD4 ELISA. The level of gp120 is expressed in ng/ml.</p>
    <p>FIG. 5, panel A is a photograph of a gel illustrating the results of an immunoprecipitation assay used to measure expression of the native and synthetic gp120 in the presence of rev in trans and the RRE in cis. In this experiment 293T cells were transiently transfected by calcium phosphate coprecipitation of 10 g of plasmid expressing: (A) the synthetic gp120MN sequence and RRE in cis, (B) the gp120 portion of HIV-1 IIIB, (C) the gp120 portion of HIV-1 IIIB and RRE in cis, all in the presence or absence of rev expression. The RRE constructs gp120IIIbRRE and syngp120mnRRE were generated using an Eag1/Hpa1 RRE fragment cloned by PCR from a HIV-1 HXB2 proviral clone. Each gp120 expression plasmid was cotransfected with 10 g of either pCMVrev or CDM7 plasmid DNA. Supernatants were harvested 60 hours post transfection, immunoprecipitated with CD4:IgG fusion protein and protein A agarose, and run on a 7% reducing SDS-PAGE. The gel exposure time was extended to allow the induction of gp120IIIbrre by rev to be demonstrated. FIG. 5, panel B is a shorter exposure of a similar experiment in which syngp120mnrre was cotransfected with or without pCMVrev. FIG. 5, panel C is a schematic diagram of the constructs used in panel A.</p>
    <p>FIG. 6 is a comparison of the sequence of the wildtype rat THY-1 gene (wt) and a synthetic rat THY-1 gene (env) constructed by chemical synthesis and having the most prevalent codons found in the HIV-1 env gene.</p>
    <p>FIG. 7 is a schematic diagram of the synthetic ratTHY-1 gene. The solid black box denotes the signal peptide. The shaded box denotes the sequences in the precursor which direct the attachment of a phosphatidylinositol glycan anchor. Unique restriction sites used for assembly of the THY-1 constructs are marked H (Hind3), M (Mlu1), S (Sac1) and No (Not1). The position of the synthetic oligonucleotides employed in the construction are shown at the bottom of the figure.</p>
    <p>FIG. 8 is a graph depicting the results of flow cytometry analysis. In this experiment 293T cells were transiently transfected with either wildtype rat THY-1 (dark line), ratTHY-1 with envelope codons (light line) or vector only (dotted line). 293T cells were transfected with the different expression plasmids by calcium phosphate coprecipitation and stained with anti-ratTHY-1 monoclonal antibody OX7 followed by a polyclonal FITC- conjugated anti-mouse IgG antibody 3 days after transfection.</p>
    <p>FIG. 9, panel A is a photograph of a gel illustrating the results of immunoprecipitation analysis of supernatants of human 293T cells transfected with either syngp120mn (lane A) or a construct syngp120mn.rTHY-1env which has the rTHY-1env gene in the 3' untranslated region of the syngp120mn gene (lane B). The syngp120mn.rTHY-1env construct was generated by inserting a Not1 adapter into the blunted Hind3 site of the rTHY-1env plasmid. Subsequently, a 0.5 kb Not1 fragment containing the rTHY-1env gene was cloned into the Not1 site of the syngp120mn plasmid and tested for correct orientation. Supernatants of 35 S labelled cells were harvested 72 hours post transfection, precipitated with CD4:IgG fusion protein and protein A agarose, and run on a 7% reducing SDS-PAGE. FIG. 9, panel B is a schematic diagram of the constructs used in the experiment depicted in panel A of this figure.</p>
    <heading>DESCRIPTION OF THE PREFERRED EMBODIMENTS</heading> <heading>Construction of a Synthetic gp120 Gene Having Codons Found in Highly Expressed Human Genes</heading> <p>A codon frequency table for the envelope precursor of the LAV subtype of HIV-1 was generated using software developed by the University of Wisconsin Genetics Computer Group. The results of that tabulation are contrasted in Table 1 with the pattern of codon usage by a collection of highly expressed human genes. For any amino acid encoded by degenerate codons, the most favored codon of the highly expressed genes is different from the most favored codon of the HIV envelope precursor. Moreover a simple rule describes the pattern of favored envelope codons wherever it applies: preferred codons maximize the number of adenine residues in the viral RNA. In all cases but one this means that the codon in which the third position is A is the most frequently used. In the special case of serine, three codons equally contribute one A residue to the mRNA; together these three comprise 85% of the codons actually used in envelope transcripts. A particularly striking example of the A bias is found in the codon choice for arginine, in which the AGA triplet comprises 88% of all codons. In addition to the preponderance of A residues, a marked preference is seen for uridine among degenerate codons whose third residue must be a pyrimidine. Finally, the inconsistencies among the less frequently used variants can be accounted for by the observation that the dinucleotide CpG is underrepresented; thus the third position is less likely to be G whenever the second position is C, as in the codons for alanine, proline, serine and threonine; and the CGX triplets for arginine are hardly used at all.</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">              TABLE 1______________________________________Codon Frequency in the HIV-1 IIIb env gene andin highly expressed human genes.            High Env______________________________________AlaGC       C             53     27    T             17     18    A             13     50    G             17      5ArgCG       C             37      0    T              7      4    A              6      0    G             21      0AG       A             10     88    G             18      8AsnAA       C             78     30    T             22     70AspGA       C             75     33    T             25     67LeuCT       C             26     10    T              5      7    A              3     17    G             58     17TT       A              2     30    G              6     20LysAA       A             18     68    G             82     32ProCC       C             48     27    T             19     14    A             16     55    G             17      5PheTT       C             80     26    T             20     74CysTG       C             68     16    T             32     84GlnCA       A             12     55    G             88     45GluGA       A             25     67    G             75     33GlyGG       C             50      6    T             12     13    A             14     53    G             24     28HisCA       C             79     25    T             21     75IleAT       C             77     25    T             18     31    A              5     44SerTC       C             28      8    T             13      8    A              5     22    G              9      0AG       C             34     22    T             10     41ThrAC       C             57     20    T             14     22    A             14     51    G             15      7TyrTA       C             74      8    T             26     92ValGT       C             25     12    T              7      9    A              5     62    G             64     18______________________________________ Codon frequency was calculated using the GCG program established the the University of Wisconsin Genetics Computer Group. Numbers represent the percentage of cases in which the particular codon is used. Codon usage frequencies of envelope genes of other HIV1 virus isolates are comparable and show a similar bias.</pre>
    
    <p>In order to produce a gp120 gene capable of high level expression in mammalian cells, a synthetic gene encoding the gp120 segment of HIV-1 was constructed (syngp120mn), based on the sequence of the most common North American subtype, HIV-1 MN (Shaw et al. 1984; Gallo et al. 1986). In this synthetic gp120 gene nearly all of the native codons have been systematically replaced with codons most frequently used in highly expressed human genes (FIG. 1). This synthetic gene was assembled from chemically synthesized oligonucleotides of 150 to 200 bases in length. If oligonucleotides exceeding 120 to 150 bases are chemically synthesized, the percentage of full-length product can be low, and the vast excess of material consists of shorter oligonucleotides. Since these shorter fragments inhibit cloning and PCR procedures, it can be very difficult to use oligonucleotides exceeding a certain length. In order to use crude synthesis material without prior purification, single-stranded oligonucleotide pools were PCR amplified before cloning. PCR products were purified in agarose gels and used as templates in the next PCR step. Two adjacent fragments could be co-amplified because of overlapping sequences at the end of either fragment. These fragments, which were between 350 and 400 bp in size, were subcloned into a pCDM7-derived plasmid containing the leader sequence of the CD5 surface molecule followed by a Nhe1/Pst1/Mlu1/EcoR1/BamH1 polylinker. Each of the restriction enzymes in this polylinker represents a site that is present at either the 5' or 3' end of the PCR-generated fragments. Thus, by sequential subcloning of each of the 4 long fragments, the whole gp120 gene was assembled. For each fragment 3 to 6 different clones were subcloned and sequenced prior to assembly. A schematic drawing of the method used to construct the synthetic gp120 is shown in FIG. 2. The sequence of the synthetic gp120 gene (and a synthetic gp160 gene created using the same approach) is presented in FIG. 1.</p>
    <p>The mutation rate was considerable. The most commonly found mutations were short (1 nucleotide) and long (up to 30 nucleotides) deletions. In some cases it was necessary to exchange parts with either synthetic adapters or pieces from other subclones without mutation in that particular region. Some deviations from strict adherence to optimized codon usage were made to accommodate the introduction of restriction sites into the resulting gene to facilitate the replacement of various segments (FIG. 2). These unique restriction sites were introduced into the gene at approximately 100 bp intervals. The native HIV leader sequence was exchanged with the highly efficient leader peptide of the human CD5 antigen to facilitate secretion. The plasmid used for construction is a derivative of the mammalian expression vector pCDM7 transcribing the inserted gene under the control of a strong human CMV immediate early promoter.</p>
    <p>To compare the wild-type and synthetic gp120 coding sequences, the synthetic gp120 coding sequence was inserted into a mammalian expression vector and tested in transient transfection assays. Several different native gp120 genes were used as controls to exclude variations in expression levels between different virus isolates and artifacts induced by distinct leader sequences. The gp120 HIV IIIb construct used as control was generated by PCR using a Sal1/Xho1 HIV-1 HXB2 envelope fragment as template. To exclude PCR induced mutations a Kpn1/Ear1 fragment containing approximately 1.2 kb of the gene was exchanged with the respective sequence from the proviral clone. The wildtype gp120mn constructs used as controls were cloned by PCR from HIV-1 MN infected C8166 cells (AIDS Repository, Rockville, Md.) and expressed gp120 either with a native envelope or a CD5 leader sequence. Since proviral clones were not available in this case, two clones of each construct were tested to avoid PCR artifacts. To determine the amount of secreted gp120 semi-quantitatively supernatants of 293T cells transiently transfected by calcium phosphate coprecipitation were immunoprecipitated with soluble CD4:immunoglobulin fusion protein and protein A sepharose.</p>
    <p>The results of this analysis (FIG. 3) show that the synthetic gene product is expressed at a very high level compared to that of the native gp120 controls. The molecular weight of the synthetic gp120 gene was comparable to control proteins (FIG. 3) and appeared to be in the range of 100 to 110 kd. The slightly faster migration can be explained by the fact that in some tumor cell lines like 293T glycosylation is either not complete or altered to some extent.</p>
    <p>To compare expression more accurately gp120 protein levels were quantitated using a gp120 ELISA with CD4 in the demobilized phase. This analysis shows (FIG. 4) that ELISA data were comparable to the immunoprecipitation data, with a gp120 concentration of approximately 125 ng/ml for the synthetic gp120 gene, and less than the background cutoff (5 ng/ml) for all the native gp120 genes. Thus, expression of the synthetic gp120 gene appears to be at least one order of magnitude higher than wildtype gp120 genes. In the experiment shown the increase was at least 25 fold.</p>
    <p>The Role of rev in gp120 Expression</p>
    <p>Since rev appears to exert its effect at several steps in the expression of a viral transcript, the possible role of non-translational effects in the improved expression of the synthetic gp120 gene was tested. First, to rule out the possibility that negative signals elements conferring either increased mRNA degradation or nucleic retention were eliminated by changing the nucleotide sequence, cytoplasmic mRNA levels were tested. Cytoplasmic RNA was prepared by NP40 lysis of transiently transfected 293T cells and subsequent elimination of the nuclei by centrifugation. Cytoplasmic RNA was subsequently prepared from lysates by multiple phenol extractions and precipitation, spotted on nitrocellulose using a slot blot apparatus, and finally hybridized with an envelope-specific probe.</p>
    <p>Briefly, cytoplasmic mRNA 293 cells transfected with CDM7, gp120IIIB, or syngp120 was isolated 36 hours post transfection. Cytoplasmic RNA of Hela cells infected with wildtype vaccinia virus or recombinant virus expressing gp120IIIb or the synthetic gp120 gene was under the control of the 7.5 promoter was isolated 16 hours post infection. Equal amounts were spotted on nitrocellulose using a slot blot device and hybridized with randomly labelled 1.5 kb gp120IIIb and syngp120 fragments or human beta-actin. RNA expression levels were quantitated by scanning the hybridized membranes with a phosphoimager. The procedures used are described in greater detail below.</p>
    <p>This experiment demonstrated that there was no significant difference in the mRNA levels of cells transfected with either the native or synthetic gp120 gene. In fact, in some experiments cytoplasmic mRNA level of the synthetic gp120 gene was even lower than that of the native gp120 gene.</p>
    <p>These data were confirmed by measuring expression from recombinant vaccinia viruses. Human 293 cells or Hela cells were infected with vaccinia virus expressing wildtype gp120IIIb or syngp120mn at a multiplicity of infection of at least 10. Supernatants were harvested 24 hours post infection and immunoprecipitated with CD4:immunoglobin fusion protein and protein A sepharose. The procedures used in this experiment are described in greater detail below.</p>
    <p>This experiment showed that the increased expression of the synthetic gene was still observed when the endogenous gene product and the synthetic gene product were expressed from vaccinia virus recombinants under the control of the strong mixed early and late 7.5 k promoter. Because vaccinia virus mRNAs are transcribed and translated in the cytoplasm, increased expression of the synthetic envelope gene in this experiment cannot be attributed to improved export from the nucleus. This experiment was repeated in two additional human cell types, the kidney cancer cell line 293 and HeLa cells. As with transfected 293T cells, mRNA levels were similar in 293 cells infected with either recombinant vaccinia virus.</p>
    <p>Codon Usage in Lentivirus</p>
    <p>Because it appears that codon usage has a significant impact on expression in mammalian cells, the codon frequency in the envelope genes of other retroviruses was examined. This study found no clear pattern of codon preference between retroviruses in general. However, if viruses from the lentivirus genus, to which HIV-1 belongs were analyzed separately, codon usage bias almost identical to that of HIV-1 was found. A codon frequency table from the envelope glycoproteins of a variety of (predominantly type C) retroviruses excluding the lentiviruses was prepared, and a codon frequency table created from the envelope sequences of four lentiviruses not closely related to HIV-1 (caprine arthritis encephalitis virus, equine infectious anemia virus, feline immunodeficiency virus, and visna virus) (Table 2). The codon usage pattern for lentiviruses is strikingly similar to that of HIV-1. In all cases but one, the preferred codon for HIV-1 is the same as the preferred codon for the other lentiviruses. The exception is proline, which is encoded by CCT in 41% of non-HIV lentiviral envelope residues, and by CCA in 40% of residues, a situation which clearly also reflects a significant preference for the triplet ending in A. The pattern of codon usage by the non-lentiviral envelope proteins does not show a similar predominance of A residues, and is also not as skewed toward third position C and G residues as is the codon usage for the highly expressed human genes. In general non-lentiviral retroviruses appear to exploit the different codons more equally, a pattern they share with less highly expressed human genes.</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">              TABLE 2______________________________________Codon frequency in the envelope gene oflentiviruses (lenti) and non-lentiviral retroviruses (other).            Other                 Lenti______________________________________AlaGC       C             45     13    T             26     37    A             20     46    G              9      3ArgCG       C             14      2    T              6      3    A             16      5    G             17      3AG       A             31     51    G             15     26AsnAA       C             49     31    T             51     69AspGA       C             55     33    T             51     69LeuCT       C             22      8    T             14      9    A             21     16    G             19     11TT       A             15     41    G             10     16LysAA       A             60     63    G             40     37ProCC       C             42     14    T             30     41    A             20     40    G              7      5PheTT       C             52     25    T             48     75CysTG       C             53     21    T             47     79GlnCA       A             52     69    C             48     31GluGA       A             57     68    G             43     32GlyCC       C             21      8    T             13      9    A             37     56    G             29     26HisCA       C             51     38    T             49     62IleAT       C             38     16    T             31     22    A             31     61SerTC       C             38     10    T             17     16    A             18     24    G              6      5AG       C             13     20    T              7     25ThrAC       C             44     18    T             27     20    A             19     55    G             10      8TyrTA       C             48     28    T             52     72ValGT       C             36      9    T             17     10    A             22     54    G             25     27______________________________________ Codon frequency was calculated using the GCG program established by the University of Wisconsin Genetics Computer Group. Numbers represent the percentage in which a particular codon is used. Codon usage of nonlentiviral retroviruses was compiled from the envelope precursor sequences of bovine leukemia virus feline leukemia virus, human Tcell leukemia virus type I, human Tcell lymphotropic virus type II, the mink cell focusforming isolate of murine leukemia virus (MuLV), the Rauscher spleen focusforming isolate, the 10A1 isolate, the 4070A amphotropic isolate and the myeloproliferative leukemia virus isolate, and from rat leukemia virus, simian sarcoma virus, simian Tcell leukemia virus, leukemogenic retrovirus T1223/B and gibbon ape leukemia virus. The codon frequency tables for the nonHIV, nonSIV lentiviruses were compiled from the envelope precursor sequences for caprine arthritis encephalitis virus equine infectious anemia virus, feline immunodeficiency virus, and visna virus.</pre>
    
    <p>In addition to the prevalence of A containing codons, lentiviral codons adhere to the HIV pattern of strong CpG underrepresentation, so that the third position for alanine, proline, serine and threonine triplets is rarely G. The retroviral envelope triplets show a similar, but less pronounced, underrepresentation of CpG. The most obvious difference between lentiviruses and other retroviruses with respect to CpG prevalence lies in the usage of the CGX variant of arginine triplets, which is reasonably frequently represented among the retroviral envelope coding sequences, but is almost never present among the comparable lentivirus sequences.</p>
    <p>Differences in rev Dependence Between Native and Synthetic gp120</p>
    <p>To examine whether regulation by rev is connected to HIV-1 codon usage, the influence of rev on the expression of both native and synthetic gene was investigated. Since regulation by rev requires the rev-binding site RRE in cis, constructs were made in which this binding site was cloned into the 3' untranslated region of both the native and the synthetic gene. These plasmids were co-transfected with rev or a control plasmid in trans into 293T cells, and gp120 expression levels in supernatants were measured semiquantitatively by immunoprecipitation. The procedures used in this experiment are described in greater detail below.</p>
    <p>As shown in FIG. 5, panels A and B, rev upregulates the native gp120 gene, but has no effect on the expression of the synthetic gp120 gene. Thus, the action of rev is not apparent on a substrate which lacks the coding sequence of endogenous viral envelope sequences.</p>
    <p>Expression of a synthetic rat THY-1 gene with HIV envelope codons</p>
    <p>The above-described experiment suggest that in fact "envelope sequences" have to be present for rev regulation. In order to test this hypothesis, a synthetic version of the gene encoding the small, typically highly expressed cell surface protein, rat THY-1 antigen, was prepared. The synthetic version of the rat THY-1 gene was designed to have a codon usage like that of HIV gp120. In designing this synthetic gene AUUUA sequences, which are associated with mRNA instability, were avoided. In addition, two restriction sites were introduced to simplify manipulation of the resulting gene (FIG. 6). This synthetic gene with the HIV envelope codon usage (rTHY-1env) was generated using three 150 to 170 mer oligonucleotides (FIG. 7). In contrast to the syngp120mn gene, PCR products were directly cloned and assembled in pUC12, and subsequently cloned into pCDM7.</p>
    <p>Expression levels of native rTHY-1 and rTHY-1 with the HIV envelope codons were quantitated by immunofluorescence of transiently transfected 293T cells. FIG. 8 shows that the expression of the native THY-1 gene is almost two orders of magnitude above the background level of the control transfected cells (pCDM7). In contrast, expression of the synthetic rat THY-1 is substantially lower than that of the native gene (shown by the shift of the peak towards a lower channel number).</p>
    <p>To prove that no negative sequence elements promoting mRNA degradation were inadvertently introduced, a construct was generated in which the rTHY-1env gene was cloned at the 3' end of the synthetic gp120 gene (FIG. 9, panel B). In this experiment 293T cells were transfected with either the syngp120mn gene or the syngp120/rat THY-1 env fusion gene (syngp120mn.rTHY-1env). Expression was measured by immunoprecipitation with CD4:IgG fusion protein and protein A agarose. The procedures used in this experiment are described in greater detail below.</p>
    <p>Since the synthetic gp120 gene has a UAG stop codon, rTHY-1env is not translated from this transcript. If negative elements conferring enhanced degradation were present in the sequence, gp120 protein levels expressed from this construct should be decreased in comparison to the syngp120mn construct without rTHY-1env. FIG. 9, panel A, shows that the expression of both constructs is similar, indicating that the low expression must be linked to translation.</p>
    <p>Rev-dependent expression of synthetic rat THY-1 gene with envelope codons</p>
    <p>To explore whether rev is able to regulate expression of a rat THY-1 gene having env codons, a construct was made with a rev-binding site in the 3' end of the rTHY1env open reading frame. To measure rev-responsiveness of the a rat THY-1env construct having a 3' RRE, human 293T cells were cotransfected with ratTHY-1envrre and either CDM7 or pCMVrev. At 60 hours post transfection cells were detached with 1 mM EDTA in PBS and stained with the OX-7 anti rTHY-1 mouse monoclonal antibody and a secondary FITC-conjugated antibody. Fluorescence intensity was measured using an EPICS XL cytofluorometer. These procedures are described in greater detail below.</p>
    <p>In repeated experiments, a slight increase of rTHY-1env expression was detected if rev was cotransfected with the rTHY-1env gene. To further increase the sensitivity of the assay system a construct expressing a secreted version of rTHY-1env was generated. This construct should produce more reliable data because the accumulated amount of secreted protein in the supernatant reflects the result of protein production over an extended period, in contrast to surface expressed protein, which appears to more closely reflect the current production rate. A gene capable of expressing a secreted form was prepared by PCR using forward and reverse primers annealing 3' of the endogenous leader sequence and 5' of the sequence motif required for phosphatidylinositol glycan anchorage respectively. The PCR product was cloned into a plasmid which already contained a CD5 leader sequence, thus generating a construct in which the membrane anchor has been deleted and the leader sequence exchanged by a heterologous (and probably more efficient) leader peptide.</p>
    <p>The rev-responsiveness of the secreted form ratTHY-1env was measured by immunoprecipitation of supernatants of human 293T cells cotransfected with a plasmid expressing a secreted form of ratTHY-1env and the RRE sequence in cis (rTHY-1envPI-rre) and either CDM7 or pCMVrev. The rTHY-1envPI-RRE construct was made by PCR using the oligonucleotides cgcggggctagcgcaaagagtaataagtttaac as forward and cgcggatcccttgtattttgtactaata a as reverse primers and the synthetic rTHY-1env construct as template. After digestion with Nhe1 and Not1 the PCR fragment was cloned into a plasmid containing CD5 leader and RRE sequences. Supernatants of <sup>35</sup> S labelled cells were harvested 72 hours post transfection, precipitated with a mouse monoclonal antibody OX7 against rTHY-1 and anti-mouse IgG sepharose, and run on a 12% reducing SDS-PAGE.</p>
    <p>In this experiment the induction of rTHY-1env by rev was much more prominent and clearcut than in the above-described experiment and strongly suggests that rev is able to translationally regulate transcripts that are suppressed by low-usage codons.</p>
    <p>Rev-independent expression of a rTHY-1env:immunoglobulin fusion protein</p>
    <p>To test whether low-usage codons must be present throughout the whole coding sequence or whether a short region is sufficient to confer rev-responsiveness, a rTHY-1env:immunoglobulin fusion protein was generated. In this construct the rTHY-1env gene (without the sequence motif responsible for phosphatidylinositol glycan anchorage) is linked to the human IgG1 hinge, CH2 and CH3 domains. This construct was generated by anchor PCR using primers with Nhe1 and BamHI restriction sites and rTHY-1env as template. The PCR fragment was cloned into a plasmid containing the leader sequence of the CD5 surface molecule and the hinge, CH2 and CH3 parts of human IgG1 immunoglobulin. A Hind3/Eag1 fragment containing the rTHY-1enveg1 insert was subsequently cloned into a pCDM7-derived plasmid with the RRE sequence.</p>
    <p>To measure the response of the rTHY-1env/immunoglobin fusion gene (rTHY-1enveg1rre) to rev human 293T cells cotransfected with rTHY-1enveg1rre and either pCDM7 or pCMVrev. The rTHY-1enveg1rre construct was made by anchor PCR using forward and reverse primers with Nhe1 and BamH1 restriction sites respectively. The PCR fragment was cloned into a plasmid containing a CD5 leader and human IgG1 hinge, CH2 and CH3 domains. Supernatants of <sup>35</sup> S labelled cells were harvested 72 hours post transfection, precipitated with a mouse monoclonal antibody OX7 against rTHY-1 and anti mouse IgG sepharose, and run on a 12% reducing SDS-PAGE. The procedures used are described in greater detail below.</p>
    <p>As with the product of the rTHY-1envPI-gene, this rTHY-1env/immunoglobulin fusion protein is secreted into the supernatant. Thus, this gene should be responsive to rev-induction. However, in contrast to rTHY-1envPI-, cotransfection of rev in trans induced no or only a negligible increase of rTHY-1enveg1 expression.</p>
    <p>The expression of rTHY-1:immunoglobulin fusion protein with native rTHY-1 or HIV envelope codons was measured by immunoprecipitation. Briefly, human 293T cells transfected with either rTHY-1enveg1 (env codons) or rTHY-1wteg1 (native codons). The rTHY-1wteg1 construct was generated in manner similar to that used for the rTHY-1enveg1 construct, with the exception that a plasmid containing the native rTHY-1 gene was used as template. Supernatants of <sup>35</sup> S labelled cells were harvested 72 hours post transfection, precipitated with a mouse monoclonal antibody OX7 against rTHY-1 and anti mouse IgG sepharose, and run on a 12% reducing SDS-PAGE. The procedures used in this experiment are described in greater detail below.</p>
    <p>Expression levels of rTHY-1enveg1 were decreased in comparison to a similar construct with wildtype rTHY-1 as the fusion partner, but were still considerably higher than rTHY-1env. Accordingly, both parts of the fusion protein influenced expression levels. The addition of rTHY-1env did not restrict expression to an equal level as seen for rTHY-1env alone. Thus, regulation by rev appears to be ineffective if protein expression is not almost completely suppressed.</p>
    <p>Codon preference in HIV-1 envelope genes</p>
    <p>Direct comparison between codon usage frequency of HIV envelope and highly expressed human genes reveals a striking difference for all twenty amino acids. One simple measure of the statistical significance of this codon preference is the finding that among the nine amino acids with two fold codon degeneracy, the favored third residue is A or U in all nine. The probability that all nine of two equiprobable choices will be the same is approximately 0.004, and hence by any conventional measure the third residue choice cannot be considered random. Further evidence of a skewed codon preference is found among the more degenerate codons, where a strong selection for triplets bearing adenine can be seen. This contrasts with the pattern for highly expressed genes, which favor codons bearing C, or less commonly G, in the third position of codons with three or more fold degeneracy.</p>
    <p>The systematic exchange of native codons with codons of highly expressed human genes dramatically increased expression of gp120. A quantitative analysis by ELISA showed that expression of the synthetic gene was at least 25 fold higher in comparison to native gp120 after transient transfection into human 293 cells. The concentration levels in the ELISA experiment shown were rather low. Since an ELISA was used for quantification which is based on gp120 binding to CD4, only native, non-denatured material was detected. This may explain the apparent low expression. Measurement of cytoplasmic mRNA levels demonstrated that the difference in protein expression is due to translational differences and not mRNA stability.</p>
    <p>Retroviruses in general do not show a similar preference towards A and T as found for HIV. But if this family was divided into two subgroups, lentiviruses and non-lentiviral retroviruses, a similar preference to A and, less frequently, T, was detected at the third codon position for lentiviruses. Thus, the availing evidence suggests that lentiviruses retain a characteristic pattern of envelope codons not because of an inherent advantage to the reverse transcription or replication of such residues, but rather for some reason peculiar to the physiology of that class of viruses. The major difference between lentiviruses and non-complex retroviruses are additional regulatory and non-essentially accessory genes in lentiviruses, as already mentioned. Thus, one simple explanation for the restriction of envelope expression might be that an important regulatory mechanism of one of these additional molecules is based on it. In fact, it is known that one of these proteins, rev, which most likely has homologues in all lentiviruses. Thus codon usage in viral mRNA is used to create a class of transcripts which is susceptible to the stimulatory action of rev. This hypothesis was proved using a similar strategy as above, but this time codon usage was changed into the inverse direction. Codon usage of a highly expressed cellular gene was substituted with the most frequently used codons in the HIV envelope. As assumed, expression levels were considerably lower in comparison to the native molecule, almost two orders of magnitude when analyzed by immunofluorescence of the surface expressed molecule (see 4.7). If rev was coexpressed in trans and an RRE element was present in cis only a slight induction was found for the surface molecule. However, if THY-1 was expressed as a secreted molecule, the induction by rev was much more prominent, supporting the above hypothesis. This can probably be explained by accumulation of secreted protein in the supernatant, which considerably amplifies the rev effect. If rev only induces a minor increase for surface molecules in general, induction of HIV envelope by rev cannot have the purpose of an increased surface abundance, but rather of an increased intracellular gp160 level. It is completely unclear at the moment why this should be the case.</p>
    <p>To test whether small subtotal elements of a gene are sufficient to restrict expression and render it rev-dependent rTHY1env:immunoglobulin fusion proteins were generated, in which only about one third of the total gene had the envelope codon usage. Expression levels of this construct were on an intermediate level, indicating that the rTHY-1env negative sequence element is not dominant over the immunoglobulin part. This fusion protein was not or only slightly rev-responsive, indicating that only genes almost completely suppressed can be rev-responsive.</p>
    <p>Another characteristic feature that was found in the codon frequency tables is a striking underrepresentation of CpG triplets. In a comparative study of codon usage in E. coli, yeast, drosophila and primates it was shown that in a high number of analyzed primate genes the 8 least used codons contain all codons with the CpG dinucleotide sequence. Avoidance of codons containing this dinucleotide motif was also found in the sequence of other retroviruses. It seems plausible that the reason for underrepresentation of CpG-bearing triplets has something to do with avoidance of gene silencing by methylation of CpG cytosines. The expected number of CpG dinucleotides for HIV as a whole is about one fifth that expected on the basis of the base composition. This might indicate that the possibility of high expression is restored, and that the gene in fact has to be highly expressed at some point during viral pathogenesis.</p>
    <p>The results presented herein clearly indicate that codon preference has a severe effect on protein levels, and suggest that translational elongation is controlling mammalian gene expression. However, other factors may play a role. First, abundance of not maximally loaded mRNA's in eukaryotic cells indicates that initiation is rate limiting for translation in at least some cases, since otherwise all transcripts would be completely covered by ribosomes. Furthermore, if ribosome stalling and subsequent mRNA degradation were the mechanism, suppression by rare codons could most likely not be reversed by any regulatory mechanism like the one presented herein. One possible explanation for the influence of both initiation and elongation on translational activity is that the rate of initiation, or access to ribosomes, is controlled in part by cues distributed throughout the RNA, such that the lentiviral codons predispose the RNA to accumulate in a pool of poorly initiated RNAs. However, this limitation need not be kinetic; for example, the choice of codons could influence the probability that a given translation product, once initiated, is properly completed. Under this mechanism, abundance of less favored codons would incur a significant cumulative probability of failure to complete the nascent polypeptide chain. The sequestered RNA would then be lent an improved rate of initiation by the action of rev. Since adenine residues are abundant in rev-responsive transcripts, it could be that RNA adenine methylation mediates this translational suppression.</p>
    <p>Detailed Procedures</p>
    <p>The following procedures were used in the above-described experiments.</p>
    <p>Sequence Analysis</p>
    <p>Sequence analyses employed the software developed by the University of Wisconsin Computer Group.</p>
    <p>Plasmid constructions</p>
    <p>Plasmid constructions employed the following methods. Vectors and insert DNA was digested at a concentration of 0.5 g/10 l in the appropriate restriction buffer for 1-4 hours (total reaction volume approximately 30 l). Digested vector was treated with 10% (v/v) of 1 g/ml calf intestine alkaline phosphatase for 30 min prior to gel electrophoresis. Both vector and insert digests (5 to 10 l each) were run on a 1.5% low melting agarose gel with TAE buffer. Gel slices containing bands of interest were transferred into a 1.5 ml reaction tube, melted at 65 C. and directly added to the ligation without removal of the agarose. Ligations were typically done in a total volume of 25 l in 1 Low Buffer 1 Ligation Additions with 200-400 U of ligase, 1 l of vector, and 4 l of insert. When necessary, 5' overhanging ends were filled by adding 1/10 volume of 250 M dNTPs and 2-5 U of Klenow polymerase to heat inactivated or phenol extracted digests and incubating for approximately 20 min at room temperature. When necessary, 3' overhanging ends were filled by adding 1/10 volume of 2.5 mM dNTPs and 5-10 U of T4 DNA polymerase to heat inactivated or phenol extracted digests, followed by incubation at 37 C. for 30 min. The following buffers were used in these reactions: 10 Low buffer (60 mM Tris HCl, pH 7.5, 60 mM MgCl<sub>2</sub>, 50 mM NaCl, 4 mg/ml BSA, 70 mM -mercaptoethanol, 0.02% NaN<sub>3</sub>); 10 Medium buffer (60 mM Tris HCl, pH 7.5, 60 mM MgCl<sub>2</sub>, 50 mM NaCl, 4 mg/ml BSA, 70 mM -mercaptoethanol, 0.02% NaN<sub>3</sub>); 10 High buffer (60 mM Tris HCl, pH 7.5, 60 mM MgCl<sub>2</sub>, 50 mM NaCl, 4 mg/ml BSA, 70 mM -mercaptoethanol, 0.02% NaN<sub>3</sub>); 10 Ligation additions (1 mM ATP, 20 mM DTT, 1 mg/ml BSA, 10 mM spermidine); 50 TAE (2M Tris acetate, 50 mM EDTA).</p>
    <p>Oligonucleotide synthesis and purification</p>
    <p>Oligonucleotides were produced on a Milligen 8750 synthesizer (Millipore). The columns were eluted with 1 ml of 30% ammonium hydroxide, and the eluted oligonucleotides were deblocked at 55 C. for 6 to 12 hours. After deblockiong, 150 l of oligonucleotide were precipitated with 10 volume of unsaturated n-butanol in 1.5 ml reaction tubes, followed by centrifugation at 15,000 rpm in a microfuge. The pellet was washed with 70% ethanol and resuspended in 50 l of H<sub>2</sub> O. The concentration was determined by measuring the optical density at 260 nm in a dilution of 1:333 (1 OD<sub>260</sub> =30 g/ml).</p>
    <p>The following oligonucleotides were used for construction of the synthetic gp120 gene (all sequences shown in this text are in 5' to 3' direction).</p>
    <p>oligo 1 forward (Nhe1): cgc ggg cta gcc acc gag aag ctg (SEQ ID NO:1).</p>
    <p>oligo 1: acc gag aag ctg tgg gtg acc gtg tac tac ggc gtg ccc gtg tgg aag ag ag gcc acc acc acc ctg ttc tgc gcc agc gac gcc aag gcg tac gac acc gag gtg cac aac gtg tgg gcc acc cag gcg tgc gtg ccc acc gac ccc aac ccc cag gag gtg gag ctc gtg aacgtg acc gag aac ttc aac atg (SEQ ID NO:2).</p>
    <p>oligo 1 reverse: cca cca tgt tgt tct tcc aca tgt tga agt tct c (SEQ ID NO:3).</p>
    <p>oligo 2 forward: gac cga gaa ctt caa cat gtg gaa gaa caa cat (SEQ ID NO:4)</p>
    <p>oligo 2: tgg aag aac aac atg gtg gag cag atg cat gag gac atc atc agc ctg tgg gac cag agc ctg aag ccc tgc gtg aag ctg acc cc ctg tgc gtg acc tg aac tgc acc gac ctg agg aac acc acc aac acc aac ac agc acc gcc aac aac aac agc aac agc gag ggc acc atc aag ggc ggc gag atg (SEQ ID NO:5).</p>
    <p>oligo 2 reverse (Pst1): gtt gaa gct gca gtt ctt cat ctc gcc gcc ctt (SEQ ID NO:6).</p>
    <p>oligo 3 forward (Pst1): gaa gaa ctg cag ctt caa cat cac cac cag c (SEQ ID NO:7).</p>
    <p>oligo 3: aac atc acc acc agc atc cgc gac aag atg cag aag gag tac gcc ctg ctg tac aag ctg gat atc gtg agc atc gac aac gac agc acc agc tac cgc ctg atc tcc tgc aac acc agc gtg atc acc cag gcc tgc ccc aag atc agc ttc gag ccc atc ccc atc cac tac tgc gcc ccc gcc ggc ttc gcc (SEQ ID NO:8).</p>
    <p>oligo 3 reverse: gaa ctt ctt gtc ggc ggc gaa gcc ggc ggg (SEQ ID NO:9).</p>
    <p>oligo 4 forward: gcg ccc ccg ccg gct tcg cca tcc tga agt gca acg aca aga agt tc (SEQ ID NO:10)</p>
    <p>oligo 4: gcc gac aag aag ttc agc ggc aag ggc agc tgc aag aac gtg agc acc gtg cag tgc acc cac ggc atc cgg ccg gtg gtg agc acc cag ctc ctg ctg aac ggc agc ctg gcc gag gag gag gtg gtg atc cgc agc gag aac ttc acc gac aac gcc aag acc atc atc gtg cac ctg aat gag agc gtg cag atc (SEQ ID NO:11)</p>
    <p>oligo 4 reverse (Mlu1): agt tgg gac gcg tgc agt tga tct gca cgc tct c (SEQ ID NO:12).</p>
    <p>oligo 5 forward (Mlu1): gag agc gtg cag atc aac tgc acg cgt ccc (SEQ ID NO:13).</p>
    <p>oligo 5: aac tgc acg cgt ccc aac tac aac aag cgc aag cgc atc cac atc ggc ccc ggg cgc gcc ttc tac acc acc aag aac atc atc ggc acc atc ctc cag gcc cac tgc aac atc tct aga (SEQ ID NO:14).</p>
    <p>oligo 5 reverse: gtc gtt cca ctt ggc tct aga gat gtt gca (SEQ ID NO:15).</p>
    <p>oligo 6 forward: gca aca tct cta gag cca agt gga acg ac (SEQ ID NO:16).</p>
    <p>oligo 6: gcc aag tgg aac gac acc ctg cgc cag atc gtg agc aag ctg aag gag cag ttc aag aac aag acc atc gtg ttc ac cag agc agc ggc ggc gac ccc gag atc gtg atg cac agc ttc aac tgc ggc ggc (SEQ ID NO:17).</p>
    <p>oligo 6 reverse (EcoR1): gca gta gaa gaa ttc gcc gcc gca gtt ga (SEQ ID NO:18).</p>
    <p>oligo 7 forward (EcoR1): tca act gcg gcg gcg aat tct tct act gc (SEQ ID NO:19).</p>
    <p>oligo 7: ggc gaa ttc ttc tac tgc aac acc agc ccc ctg ttc aac agc acc tgg aac ggc aac aac acc tgg aac aac acc acc ggc agc aac aac aat att acc ctc cag tgc aag atc aag cag atc atc aac atg tgg cag gag gtg ggc aag gcc atg tac gcc ccc ccc atc gag ggc cag atc cgg tgc agc agc (SEQ ID NO:20)</p>
    <p>oligo 7 reverse: gca gac cgg tga tgt tgc tgc tgc acc gga tct ggc cct c (SEQ ID NO:21).</p>
    <p>oligo 8 forward: cga ggg cca gat ccg gtg cag cag caa cat cac cgg tct g (SEQ ID NO:22).</p>
    <p>oligo 8: aac atc acc ggt ctg ctg ctg acc cgc gac ggc ggc aag gac acc gac acc aac gac acc gaa atc ttc cgc ccc ggc ggc ggc gac atg cgc gac aac tgg aga tct gag ctg tac aag tac aag gtg gtg acg atc gag ccc ctg ggc gtg gcc ccc acc aag gcc aag cgc cgc gtg gtg cag cgc gag aag cgc (SEQ ID NO:23).</p>
    <p>oligo 8 reverse (Not1): cgc ggg cgg ccg ctt tag cgc ttc tcg cgc tgc acc ac (SEQ ID NO:24).</p>
    <p>The following oligonucleotides were used for the construction of the ratTHY-1env gene.</p>
    <p>oligo 1 forward (BamH1/Hind3): cgc ggg gga tcc aag ctt acc atg att cca gta ata agt (SEQ ID NO:25).</p>
    <p>oligo 1: atg aat cca gta ata agt ata aca tta tta tta agt gta tta caa atg agt aga gga caa aga gta ata agt tta aca gca tct tta gta aat caa aat ttg aga tta gat tgt aga cat gaa aat aat aca aat ttg cca ata caa cat gaa ttt tca tta acg (SEQ ID NO:26).</p>
    <p>oligo 1 reverse (EcoR1/Mlu1): cgc ggg gaa ttc acg cgt taa tga aaa ttc atg ttg (SEQ ID NO:27).</p>
    <p>oligo 2 forward (BamH1/Mlu1): cgc gga tcc acg cgt gaa aaa aaa aaa cat (SEQ ID NO:28).</p>
    <p>oligo 2: cgt gaa aaa aaa aaa cat gta tta agt gga aca tta gga gta cca gaa cat aca tat aga agt aga gta aat ttg ttt agt gat aga ttc ata aaa gta tta aca tta gca aat ttt aca aca aaa gat gaa gga gat tat atg tgt gag (SEQ ID NO:29).</p>
    <p>oligo 2 reverse (EcoR1/Sac1): cgc gaa ttc gag ctc aca cat ata atc tcc (SEQ ID NO:30).</p>
    <p>oligo 3 forward (BamH1/Sac1): cgc gga tcc gag ctc aga gta agt gga caa (SEQ ID NO:31).</p>
    <p>oligo 3: ctc aga gta agt gga caa aat cca aca agt agt aat aaa aca ata aat gta ata aga gat aaa tta gta aaa tgt ga gga ata agt tta tta gta caa aat aca agt tgg tta tta tta tta tta tta agt tta agt ttt tta caa gca aca gat ttt ata agt tta tga (SEQ ID NO:32).</p>
    <p>oligo 3 reverse (EcoR1/Not1): cgc gaa ttc gcg gcc gct tca taa act tat aaa atc (SEQ ID NO:33).</p>
    <p>Polymerase Chain Reaction</p>
    <p>Short, overlapping 15 to 25 mer oligonucleotides annealing at both ends were used to amplify the long oligonuclotides by polymerase chain reaction (PCR). Typical PCR conditions were: 35 cycles, 55 C. annealing temperature, 0.2 sec extension time. PCR products were gel purified, phenol extracted, and used in a subsequent PCR to generate longer fragments consisting of two adjacent small fragments. These longer fragments were cloned into a CDM7-derived plasmid containing a leader sequence of the CD5 surface molecule followed by a Nhe1/Pst1/Mlu1/EcoR1/BamH1 polylinker.</p>
    <p>The following solutions were used in these reactions: 10 PCR buffer (500 mM KCl, 100 mM Tris HCl, pH 7.5, 8 mM MgCl<sub>2</sub>, 2 mM each dNTP). The final buffer was complemented with 10% DMSO to increase fidelity of the Taq polymerase.</p>
    <p>Small scale DNA preparation</p>
    <p>Transformed bacteria were grown in 3 ml LB cultures for more than 6 hours or overnight. Approximately 1.5 ml of each culture was poured into 1.5 ml microfuge tubes, spun for 20 seconds to pellet cells and resuspended in 200 l of solution I. Subsequently 400 l of solution II and 300 l of solution III were added. The microfuge tubes were capped, mixed and spun for &gt;30 sec. Supernatants were transferred into fresh tubes and phenol extracted once. DNA was precipitated by filling the tubes with isopropanol, mixing, and spinning in a microfuge for &gt;2 min. The pellets were rinsed in 70 % ethanol and resuspended in 50 l dH<sub>2</sub> O containing 10 l of RNAse A. The following media and solutions were used in these procedures: LB medium (1.0% NaCl, 0.5% yeast extract, 1.0% trypton); solution I (10 mM EDTA pH 8.0); solution II (0.2M NaOH, 1.0% SDS); solution III (2.5M KOAc, 2.5M glacial aceatic acid); phenol (pH adjusted to 6.0, overlaid with TE); TE (10 mM Tris HCl, pH 7.5, 1 mM EDTA pH 8.0).</p>
    <p>Large scale DNA preparation</p>
    <p>One liter cultures of transformed bacteria were grown 24 to 36 hours (MC1061p3 transformed with pCDM derivatives) or 12 to 16 hours (MC1061 transformed with pUC derivatives) at 37 C. in either M9 bacterial medium (pCDM derivatives) or LB (pUC derivatives). Bacteria were spun down in 1 liter bottles using a Beckman J6 centrifuge at 4,200 rpm for 20 min. The pellet was resuspended in 40 ml of solution I. Subsequently, 80 ml of solution II and 40 ml of solution III were added and the bottles were shaken semivigorously until lumps of 2 to 3 mm size developed. The bottle was spun at 4,200 rpm for 5 min and the supernatant was poured through cheesecloth into a 250 ml bottle. Isopropanol was added to the top and the bottle was spun at 4,200 rpm for 10 min. The pellet was resuspended in 4.1 ml of solution I and added to 4.5 g of cesium chloride, 0.3 ml of 10 mg/ml ethidium bromide, and 0.1 ml of 1% Triton X100 solution. The tubes were spun in a Beckman J2 high speed centrifuge at 10,000 rpm for 5 min. The supernatant was transferred into Beckman Quick Seal ultracentrifuge tubes, which were then sealed and spun in a Beckman ultracentrifuge using a NVT90 fixed angle rotor at 80,000 rpm for &gt;2.5 hours. The band was extracted by visible light using a 1 ml syringe and 20 gauge needle. An equal volume of dH<sub>2</sub> O was added to the extracted material. DNA was extracted once with n-butanol saturated with 1M sodium chloride, followed by addition of an equal volume of 10M ammonium acetate/1 mM EDTA. The material was poured into a 13 ml snap tube which was than filled to the top with absolute ethanol, mixed, and spun in a Beckman J2 centrifuge at 10,000 rpm for 10 min. The pellet was rinsed with 70% ethanol and resuspended in 0.5 to 1 ml of H<sub>2</sub> O. The DNA concentration was determined by measuring the optical density at 260 nm in a dilution of 1:200 (1 OD<sub>260</sub> =50 g/ml).</p>
    <p>The following media and buffers were used in these procedures: M9 bacterial medium (10 g M9 salts, 10 g casamino acids (hydrolysed), 10 ml M9 additions, 7.5 g/ml tetracycline (500 l of a 15 mg/ml stock solution), 12.5 g/ml ampicillin (125 l of a 10 mg/ml stock solution); M9 additions (10 mM CaCl<sub>2</sub>, 100 mM MgSO<sub>4</sub>, 200 g/ml thiamine, 70% glycerol); LB medium (1.0% NaCl, 0.5% yeast extract, 1.0% trypton); Solution I (10 mM EDTA pH 8.0); Solution II (0.2M NaOH 1.0% SDS); Solution III (2.5M KOAC 2.5M HOAc)</p>
    <p>Sequencing</p>
    <p>Synthetic genes were sequenced by the Sanger dideoxynucleotide method. In brief, 20 to 50 g double-stranded plasmid DNA were denatured in 0.5M NaOH for 5 min. Subsequently the DNA was precipitated with 1/10 volume of sodium acetate (pH 5.2) and 2 volumes of ethanol and centrifuged for 5 min. The pellet was washed with 70% ethanol and resuspended at a concentration of 1 g/l. The annealing reaction was carried out with 4 g of template DNA and 40 ng of primer in 1 annealing buffer in a final volume of 10 l. The reaction was heated to 65 C. and slowly cooled to 37 C. In a separate tube 1 l of 0.1M DTT, 2 l of labeling mix, 0.75 l of dH<sub>2</sub> O, 1 l of  <sup>35</sup> S! dATP (10 Ci), and 0.25 l of Sequenase (12 U/l) were added for each reaction. Five l of this mix were added to each annealed primer-template tube and incubated for 5 min at room temperature. For each labeling reaction 2.5 l of each of the 4 termination mixes were added on a Terasaki plate and prewarmed at 37 C. At the end of the incubation period 3.5 l of labeling reaction were added to each of the 4 termination mixes. After 5 min, 4 l of stop solution were added to each reaction and the Terasaki plate was incubated at 80 C. for 10 min in an oven. The sequencing reactions were run on 5% denaturing polyacrylamide gel. An acrylamide solution was prepared by adding 200 ml of 10 TBE buffer and 957 ml of dH<sub>2</sub> O to 100 g of acrylamide:bisacrylamide (29:1). 5% polyacrylamide 46% urea and 1 TBE gel was prepared by combining 38 ml of acrylamide solution and 28 g urea. Polymerization was initiated by the addition of 400 l of 10% ammonium peroxodisulfate and 60 l of TEMED. Gels were poured using silanized glass plates and sharktooth combs and run in 1 TBE buffer at 60 to 100 W for 2 to 4 hours (depending on the region to be read). Gels were transferred to Whatman blotting paper, dried at 80 C. for about 1 hour, and exposed to x-ray film at room temperature. Typically exposure time was 12 hours. The following solutions were used in these procedures: 5 Annealing buffer (200 mM Tris HCl, pH 7.5, 100 mM MgCl<sub>2</sub>, 250 mM NaCl); Labelling Mix (7.5 M each dCTP, dGTP, and dTTP); Termination Mixes (80 M each dNTP, 50 mM NaCl, 8 M ddNTP (one each)); Stop solution (95% formamide, 20 mM EDTA, 0.05% bromphenol blue, 0.05% xylencyanol); 5 TBE (0.9M Tris borate, 20 mM EDTA); Polyacrylamide solution (96.7 g polyacrylamide, 3.3 g bisacrylamide, 200 ml 1 TBE, 957 ml dH<sub>2</sub> O).</p>
    <p>RNA isolation</p>
    <p>Cytoplasmic RNA was isolated from calcium phosphate transfected 293T cells 36 hours post transfection and from vaccinia infected Hela cells 16 hours post infection essentially as described by Gilman. (Gilman Preparation of cytoplasmic RNA from tissue culture cells. In Current Protocols in Molecular Biology, Ausubel et al, eds., Wiley &amp; Sons, New York, 1992). Briefly, cells were lysed in 400 l lysis buffer, nuclei were spun out, and SDS and proteinase K were added to 0.2% and 0.2 mg/ml respectively. The cytoplasmic extracts were incubated at 37 C. for 20 min, phenol/chloroform extracted twice, and precipitated. The RNA was dissolved in 100 l buffer I and incubated at 37 C. for 20 min. The reaction was stopped by adding 25 l stop buffer and precipitated again.</p>
    <p>The following solutions were used in this procedure: Lysis Buffer (TE containing with 50 mM Tris pH 8.0, 100 mM NaCl, 5 mM MgCl<sub>2</sub>, 0.5% NP40); Buffer I (TE buffer with 10 mM MgCl<sub>2</sub>, 1 mM DTT, 0.5 U/l placental RNAse inhibitor, 0.1 U/l RNAse free DNAse I); Stop buffer (50 mM EDTA 1.5M NaOAc 1.0% SDS).</p>
    <p>Slot blot analysis</p>
    <p>For slot blot analysis 10 g of cytoplasmic RNA was dissolved in 50 l dH<sub>2</sub> O to which 150 l of 10 SSC/18% formaldehyde were added. The solubilized RNA was then incubated at 65 C. for 15 min and spotted onto with a slot blot apparatus. Radioactively labelled probes of 1.5 kb gp120IIIb and syngp120mn fragments were used for hybridization. Each of the two fragments was random labelled in a 50 l reaction with 10 l of 5 oligo-labelling buffer, 8 l of 2.5 mg/ml BSA, 4 l of  <sup>32</sup> P!-dCTP (20 uCi/l; 6000 Ci/mmol), and 5 U of Klenow fragment. After 1 to 3 hours incubation at 37 C. 100 l of TE were added and unincorporated  <sup>32</sup> P!-dCTP was eliminated using a G50 spin column. Activity was measured in a Beckman beta-counter, and equal specific activities were used for hybridization. Membranes were pre-hybridized for 2 hours and hybridized for 12 to 24 hours at 42 C. with 0.510<sup>6</sup> cpm probe per ml hybridization fluid. The membrane was washed twice (5 min) with washing buffer I at room temperature, for one hour in washing buffer II at 65 C., and then exposed to x-ray film. Similar results were obtained using a 1.1 kb Not1/Sfi1 fragment of pCDM7 containing the 3' untranslated region. Control hybridizations were done in parallel with a random-labelled human beta-actin probe. RNA expression was quantitated by scanning the hybridized nitrocellulose membranes with a Magnetic Dynamics phosphorimager.</p>
    <p>The following solutions were used in this procedure: 5 Oligo-labelling buffer (250 mM Tris HCl, pH 8.0, 25 mM MgCl<sub>2</sub>, 5 mM -mercaptoethanol, 2 mM dATP, 2 mM dGTP, 2 mM dTTP, 1M Hepes pH 6.6, 1 mg/ml hexanucleotides  dNTP!6); Hybridization Solution (<sub>--</sub> M sodium phosphate, 250 mM NaCl, 7% SDS, 1 mM EDTA, 5% dextrane sulfate, 50% formamide, 100 g/ml denatured salmon sperm DNA); Washing buffer I (2 SSC, 0.1% SDS); Washing buffer II (0.5 SSC, 0.1% SDS); 20 SSC (3M NaCl, 0.3M Na<sub>3</sub> citrate, pH adjusted to 7.0).</p>
    <p>Vaccinia recombination</p>
    <p>Vaccinia recombination used a modification of the method described by Romeo and Seed (Romeo and Seed, Cell, 64: 1037, 1991). Briefly, CV1 cells at 70 to 90% confluency were infected with 1 to 3 l of a wildtype vaccinia stock WR (210<sup>8</sup> pfu/ml) for 1 hour in culture medium without calf serum. After 24 hours, the cells were transfected by calcium phosphate with 25 g TKG plasmid DNA per dish. After an additional 24 to 48 hours the cells were scraped off the plate, spun down, and resuspended in a volume of 1 ml. After 3 freeze/thaw cycles trypsin was added to 0.05 mg/ml and lysates were incubated for 20 min. A dilution series of 10, 1 and 0.1 l of this lysate was used to infect small dishes (6 cm) of CV1 cells, that had been pretreated with 12.5 g/ml mycophenolic acid, 0.25 mg/ml xanthin and 1.36 mg/ml hypoxanthine for 6 hours. Infected cells were cultured for 2 to 3 days, and subsequently stained with the monoclonal antibody NEA9301 against gp120 and an alkaline phosphatase conjugated secondary antibody. Cells were incubated with 0.33 mg/ml NBT and 0.16 mg/ml BCIP in AP-buffer and finally overlaid with 1% agarose in PBS. Positive plaques were picked and resuspended in 100 l Tris pH 9.0. The plaque purification was repeated once. To produce high titer stocks the infection was slowly scaled up. Finally, one large plate of Hela cells was infected with half of the virus of the previous round. Infected cells were detached in 3 ml of PBS, lysed with a Dounce homogenizer and cleared from larger debris by centrifugation. VPE-8 recombinant vaccinia stocks were kindly provided by the AIDS repository, Rockville, Md., and express HIV-1 IIIB gp120 under the 7.5 mixed early/late promoter (Earl et al., J. Virol., 65:31, 1991). In all experiments with recombinant vaccina cells were infected at a multiplicity of infection of at least 10.</p>
    <p>The following solution was used in this procedure: AP buffer (100 mM Tris HCl, pH 9.5, 100 mM NaCl, 5 mM MgCl<sub>2</sub>)</p>
    <p>Cell culture</p>
    <p>The monkey kidney carcinoma cell lines CV1 and Cos7, the human kidney carcinoma cell line 293T, and the human cervix carcinoma cell line Hela were obtained from the American Tissue Typing Collection and were maintained in supplemented IMDM. They were kept on 10 cm tissue culture plates and typically split 1:5 to 1:20 every 3 to 4 days.</p>
    <p>The following medium was used in this procedure: Supplemented IMDM (90% Iscove's modified Dulbecco Medium, 10% calf serum, iron-complemented, heat inactivated 30 min 56 C., 0.3 mg/ml L-glutamine, 25 g/ml gentamycin 0.5 mM -mercaptoethanol (pH adjusted with 5M NaOH, 0.5 ml)).</p>
    <p>Transfection</p>
    <p>Calcium phosphate transfection of 293T cells was performed by slowly adding and under vortexing 10 g plasmid DNA in 250 l 0.25M CaCl<sub>2</sub> to the same volume of 2 HEBS buffer while vortexing. After incubation for 10 to 30 min at room temperature the DNA precipitate was added to a small dish of 50 to 70% confluent cells. In cotransfection experiments with rev, cells were transfected with 10 g gp120IIIb, gp120IIIbrre, syngp120mnrre or rTHY-1enveg1rre and 10 g of pCMVrev or CDM7 plasmid DNA.</p>
    <p>The following solutions were used in this procedure: 2 HEBS buffer (280 mM NaCl, 10 mM KCl, 1.5 mM sterile filtered); 0.25 mM CaCl<sub>2</sub> (autoclaved).</p>
    <p>Immunoprecipitation</p>
    <p>After 48 to 60 hours medium was exchanged and cells were incubated for additional 12 hours in Cys/Met-free medium containing 200 Ci of <sup>35</sup> S-translabel. Supernatants were harvested and spun for 15 min at 3000 rpm to remove debris. After addition of protease inhibitors leupeptin, aprotinin and PMSF to 2.5 g/ml, 50 g/ml, 100 g/ml respectively, 1 ml of supernatant was incubated with either 10 l of packed protein A sepharose alone (rTHY-1enveg1rre) or with protein A sepharose and 3 g of a purified CD4/immunoglobulin fusion protein (kindly provided by Behring) (all gp120 constructs) at 4 C. for 12 hours on a rotator. Subsequently the protein A beads were washed 5 times for 5 to 15 min each time. After the final wash 10 l of loading buffer containing was added, samples were boiled for 3 min and applied on 7% (all gp120 constructs) or 10% (rTHY-1enveg1rre) SDS polyacrylamide gels (Tris pH 8.8 buffer in the resolving, Tris pH 6.8 buffer in the stacking gel, Tris-glycine running buffer, Maniatis et al. 1989). Gels were fixed in 10% acetic acid and 10% methanol, incubated with Amplify for 20 min, dried and exposed for 12 hours.</p>
    <p>The following buffers and solutions were used in this procedure: Wash buffer (100 mM Tris, pH 7.5, 150 mM NaCl, 5 mM CaCl<sub>2</sub>, 1% NP-40); 5 Running Buffer (125 mM Tris, 1.25M Glycine, 0.5% SDS); Loading buffer (10% glycerol, 4% SDS, 4% -mercaptoethanol, 0.02% bromphenol blue).</p>
    <p>Immunofluorescence</p>
    <p>293T cells were transfected by calcium phosphate coprecipitation and analyzed for surface THY-1 expression after 3 days. After detachment with 1 mM EDTA/PBS, cells were stained with the monoclonal antibody OX-7 in a dilution of 1:250 at 4 C. for 20 min, washed with PBS and subsequently incubated with a 1:500 dilution of a FITC-conjugated goat anti-mouse immunoglobulin antiserum. Cells were washed again, resuspended in 0.5 ml of a fixing solution, and analyzed on an EPICS XL cytofluorometer (Coulter).</p>
    <p>The following solutions were used in this procedure: PBS (137 mM NaCl, 2.7 mM KCl, 4.3 mM Na<sub>2</sub> HPO<sub>4</sub>, 1.4 mM KH<sub>2</sub> PO<sub>4</sub>, pH adjusted to 7.4); Fixing solution (2% formaldehyde in PBS).</p>
    <p>ELISA</p>
    <p>The concentration of gp120 in culture supernatants was determined using CD4-coated ELISA plates and goat anti-gp120 antisera in the soluble phase. Supernatants of 293T cells transfected by calcium phosphate were harvested after 4 days, spun at 3000 rpm for 10 min to remove debris and incubated for 12 hours at 4 C. on the plates. After 6 washes with PBS 100 l of goat anti-gp120 antisera diluted 1:200 were added for 2 hours. The plates were washed again and incubated for 2 hours with a peroxidase-conjugated rabbit anti-goat IgG antiserum 1:1000. Subsequently the plates were washed and incubated for 30 min with 100 l of substrate solution containing 2 mg/ml o-phenylenediamine in sodium citrate buffer. The reaction was finally stopped with 100 l of 4M sulfuric acid. Plates were read at 490 nm with a Coulter microplate reader. Purified recombinant gp120IIIb was used as a control. The following buffers and solutions were used in this procedure: Wash buffer (0.1% NP40 in PBS); Substrate solution (2 mg/ml o-phenylenediamine in sodium citrate buffer).</p>
    <p>Use</p>
    <p>The synthetic genes of the invention are useful for expressing a protein normally expressed in mammalian cells in cell culture (e.g. for commercial production of human proteins such as hGH, TPA, Factor VII, and Factor IX). The synthetic genes of the invention are also useful for gene therapy.</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">__________________________________________________________________________SEQUENCE LISTING(1) GENERAL INFORMATION:(iii) NUMBER OF SEQUENCES: 37(2) INFORMATION FOR SEQ ID NO:1:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 24 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:1:CGCGGGCTAGCCACCGAGAAGCTG24(2) INFORMATION FOR SEQ ID NO:2:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 196 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:2:ACCGAGAAGCTGTGGGTGACCGTGTACTACGGCGTGCCCGTGTGGAAGAGAGGCCACCAC60CACCCTGTTCTGCGCCAGCGACGCCAAGGCGTACGACACCGAGGTGCACAACGTGTGGGC120CACCCAGGCGTGCGTGCCCACCGACCCCAACCCCCAGGAGGTGGAGCTCGTGAACGTGAC180CGAGAACTTCAACATG196(2) INFORMATION FOR SEQ ID NO:3:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 34 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:3:CCACCATGTTGTTCTTCCACATGTTGAAGTTCTC34(2) INFORMATION FOR SEQ ID NO:4:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 33 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:4:GACCGAGAACTTCAACATGTGGAAGAACAACAT33(2) INFORMATION FOR SEQ ID NO:5:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 192 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:5:TGGAAGAACAACATGGTGGAGCAGATGCATGAGGACATCATCAGCCTGTGGGACCAGAGC60CTGAAGCCCTGCGTGAAGCTGACCCCCTGTGCGTGACCTGAACTGCACCGACCTGAGGAA120CACCACCAACACCAACACAGCACCGCCAACAACAACAGCAACAGCGAGGGCACCATCAAG180GGCGGCGAGATG192(2) INFORMATION FOR SEQ ID NO:6:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 33 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:6:GTTGAAGCTGCAGTTCTTCATCTCGCCGCCCTT33(2) INFORMATION FOR SEQ ID NO:7:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 31 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:7:GAAGAACTGCAGCTTCAACATCACCACCAGC31(2) INFORMATION FOR SEQ ID NO:8:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 195 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:8:AACATCACCACCAGCATCCGCGACAAGATGCAGAAGGAGTACGCCCTGCTGTACAAGCTG60GATATCGTGAGCATCGACAACGACAGCACCAGCTACCGCCTGATCTCCTGCAACACCAGC120GTGATCACCCAGGCCTGCCCCAAGATCAGCTTCGAGCCCATCCCCATCCACTACTGCGCC180CCCGCCGGCTTCGCC195(2) INFORMATION FOR SEQ ID NO:9:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 30 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:9:GAACTTCTTGTCGGCGGCGAAGCCGGCGGG30(2) INFORMATION FOR SEQ ID NO:10:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 47 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:10:GCGCCCCCGCCGGCTTCGCCATCCTGAAGTGCAACGACAAGAAGTTC47(2) INFORMATION FOR SEQ ID NO:11:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 198 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:11:GCCGACAAGAAGTTCAGCGGCAAGGGCAGCTGCAAGAACGTGAGCACCGTGCAGTGCACC60CACGGCATCCGGCCGGTGGTGAGCACCCAGCTCCTGCTGAACGGCAGCCTGGCCGAGGAG120GAGGTGGTGATCCGCAGCGAGAACTTCACCGACAACGCCAAGACCATCATCGTGCACCTG180AATGAGAGCGTGCAGATC198(2) INFORMATION FOR SEQ ID NO:12:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 34 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:12:AGTTGGGACGCGTGCAGTTGATCTGCACGCTCTC34(2) INFORMATION FOR SEQ ID NO:13:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 30 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:13:GAGAGCGTGCAGATCAACTGCACGCGTCCC30(2) INFORMATION FOR SEQ ID NO:14:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 120 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:14:AACTGCACGCGTCCCAACTACAACAAGCGCAAGCGCATCCACATCGGCCCCGGGCGCGCC60TTCTACACCACCAAGAACATCATCGGCACCATCCTCCAGGCCCACTGCAACATCTCTAGA120(2) INFORMATION FOR SEQ ID NO:15:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 30 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:15:GTCGTTCCACTTGGCTCTAGAGATGTTGCA30(2) INFORMATION FOR SEQ ID NO:16:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 29 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:16:GCAACATCTCTAGAGCCAAGTGGAACGAC29(2) INFORMATION FOR SEQ ID NO:17:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 131 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:17:GCCAAGTGGAACGACACCCTGCGCCAGATCGTGAGCAAGCTGAAGGAGCAGTTCAAGAAC60AAGACCATCGTGTTCACCAGAGCAGCGGCGGCGACCCCGAGATCGTGATGCACAGCTTCA120ACTGCGGCGGC131(2) INFORMATION FOR SEQ ID NO:18:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 29 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:18:GCAGTAGAAGAATTCGCCGCCGCAGTTGA29(2) INFORMATION FOR SEQ ID NO:19:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 29 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:19:TCAACTGCGGCGGCGAATTCTTCTACTGC29(2) INFORMATION FOR SEQ ID NO:20:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 195 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:20:GGCGAATTCTTCTACTGCAACACCAGCCCCCTGTTCAACAGCACCTGGAACGGCAACAAC60ACCTGGAACAACACCACCGGCAGCAACAACAATATTACCCTCCAGTGCAAGATCAAGCAG120ATCATCAACATGTGGCAGGAGGTGGGCAAGGCCATGTACGCCCCCCCCATCGAGGGCCAG180ATCCGGTGCAGCAGC195(2) INFORMATION FOR SEQ ID NO:21:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 40 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:21:GCAGACCGGTGATGTTGCTGCTGCACCGGATCTGGCCCTC40(2) INFORMATION FOR SEQ ID NO:22:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 40 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:22:CGAGGGCCAGATCCGGTGCAGCAGCAACATCACCGGTCTG40(2) INFORMATION FOR SEQ ID NO:23:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 242 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:23:AACATCACCGGTCTGCTGCTGCTGCTGACCCGGACGGCGGCAAGGACACCGACACCAACG60ACACCGAAATCTTCCGCGACGGCGGCAAGGACACCAACGACACCGAAATCTTCCGCCCCG120GCGGCGGCGACATGCGCGACAACTGGAGATCTGAGCTGTACAAGTACAAGGTGGTGACGA180TCGAGCCCCTGGGCGTGGCCCCCACCAAGGCCAAGCGCGCGGTGGTGCAGCGCGAGAAGC240GC242(2) INFORMATION FOR SEQ ID NO:24:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 38 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:24:CGCGGGCGGCCGCTTTAGCGCTTCTCGCGCTGCACCAC38(2) INFORMATION FOR SEQ ID NO:25:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 39 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:25:CGCGGGGGATCCAAGCTTACCATGATTCCAGTAATAAGT39(2) INFORMATION FOR SEQ ID NO:26:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 165 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:26:ATGAATCCAGTAATAAGTATAACATTATTATTAAGTGTATTACAAATGAGTAGAGGACAA60AGAGTAATAAGTTTAACAGCATCTTTAGTAAATCAAAATTTGAGATTAGATTGTAGACAT120GAAAATAATACAAATTTGCCAATACAACATGAATTTTCATTAACG165(2) INFORMATION FOR SEQ ID NO:27:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 36 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:27:CGCGGGGAATTCACGCGTTAATGAAAATTCATGTTG36(2) INFORMATION FOR SEQ ID NO:28:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 30 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:28:CGCGGATCCACGCGTGAAAAAAAAAAACAT30(2) INFORMATION FOR SEQ ID NO:29:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 149 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:29:CGTGAAAAAAAAAAACATGTATTAAGTGGAACATTAGGAGTACCAGAACATACATATAGA60AGTAGAGTAATTTGTTTAGTGATAGATTCATAAAAGTATTAACATTAGCAAATTTTACAA120CAAAAGATGAAGGAGATTATATGTGTGAG149(2) INFORMATION FOR SEQ ID NO:30:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 30 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:30:CGCGAATTCGAGCTCACACATATAATCTCC30(2) INFORMATION FOR SEQ ID NO:31:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 30 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:31:CGCGGATCCGAGCTCAGAGTAAGTGGACAA30(2) INFORMATION FOR SEQ ID NO:32:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 170 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:32:CTCAGAGTAAGTGGACAAAATCCAACAAGTAGTAATAAAACAATAAATGTAATAAGAGAT60AAATTAGTAAAATGTGAGGAATAAGTTTATTAGTACAAAATACAAGTTGGTTATTATTAT120TATTATTAAGTTTAAGTTTTTTACAAGCAACAGATTTTATAAGTTTATGA170(2) INFORMATION FOR SEQ ID NO:33:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 36 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:33:CGCGAATTCGCGGCCGCTTCATAAACTTATAAAATC36(2) INFORMATION FOR SEQ ID NO:34:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 1632 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:34:CTCGAGATCCATTGTGCTCTAAAGGAGATACCCGGCCAGACACCCTCACCTGCGGTGCCC60AGCTGCCCAGGCTGAGGCAAGAGAAGGCCAGAAACCATGCCCATGGGGTCTCTGCAACCG120CTGGCCACCTTGTACCTGCTGGGGATGCTGGTCGCTTCCGTGCTAGCCACCGAGAAGCTG180TGGGTGACCGTGTACTACGGCGTGCCCGTGTGGAAGGAGGCCACCACCACCCTGTTCTGC240GCCAGCGACGCCAAGGCGTACGACACCGAGGTGCACAACGTGTGGGCCACCCAGGCGTGC300GTGCCCACCGACCCCAACCCCCAGGAGGTGGAGCTCGTGAACGTGACCGAGAACTTCAAC360ATGTGGAAGAACAACATGGTGGAGCAGATGCATGAGGACATCATCAGCCTGTGGGACCAG420AGCCTGAAGCCCTGCGTGAAGCTGACCCCCCTGTGCGTGACCCTGAACTGCACCGACCTG480AGGAACACCACCAACACCAACAACAGCACCGCCAACAACAACAGCAACAGCGAGGGCACC540ATCAAGGGCGGCGAGATGAACAACTGCAGCTTCAACATCACCACCAGCATCCGCGACAAG600ATGCAGAAGGAGTACGCCCTGCTGTACAAGCTGGATATCGTGAGCATCGACAACGACAGC660ACCAGCTACCGCCTGATCTCCTGCAACACCAGCGTGATCACCCAGGCCTGGCCCAAGATC720AGCTTCGAGCCCATCCCCATCCACTACTGCGCCCCCGCCGGCTTCGCCATCCTGAAGTGC780AACGACAAGAAGTTCAGCGGCAAGGGCAGCTGCAAGAACGTGAGCACCGTGCAGTGCACC840CACGGCATCCGGCCGGTGGTGAGCACCCAGCTCCTGCTGAACGGCAGCCTGGCCGAGGAG900GAGGTGGTGATCCGCAGCGAGAACTTCACCGACAACGCCAAGACCATCATCGTGCACCTG960AATGAGAGCGTGCAGATCAACTGCACGCGTCCCAACTACAACAAGCGCAAGCGCATCCAC1020ATCGGCCCCGGGCGCGCCTTCTACACCACCAAGAACATCATCGGCACCATCCGCCAGGCC1080CACTGCAACATCTCTAGAGCCAAGTGGAACGACACCCTGCGCCAGATCGTGAGCAAGCTG1140AAGGAGCAGTTCAAGAACAAGACCATCGTGTTCAACCAGAGCAGCGGCGGCGACCCCGAG1200ATCGTGATGCACAGCTTCAACTGCGGCGGCGAATTCTTCTACTGCAACACCAGCCCCCTG1260TTCAACAGCACCTGGAACGGCAACAACACCTGGAACAACACCACCGGCAGCAACAACAAT1320ATTACCCTCCAGTGCAAGATCAAGCAGATCATCAACATGTGGCAGGAGGTGGGCAAGGCC1380ATGTACGCCCCCCCCATCGAGGGCCAGATCCGGTGCAGCAGCAACATCACCGGTCTGCTG1440CTGACCCGCGACGGCGGCAAGGACACCGACACCAACGACACCGAAATCTTCCGCCCCGGC1500GGCGGCGACATGCGCGACAACTGGAGATCTGAGCTGTACAAGTACAAGGTGGTGACGATC1560GAGCCCCTGGGCGTGGCCCCCACCAAGGCCAAGCGCCGCGTGGTGCAGCGCGAGAAGCGC1620TAAAGCGGCCGC1632(2) INFORMATION FOR SEQ ID NO:35:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 2481 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:35:ACCGAGAAGCTGTGGGTGACCGTGTACTACGGCGTGCCCGTGTGGAAGGAGGCCACCACC60ACCCTGTTCTGCGCCAGCGACGCCAAGGCGTACGACACCGAGGTGCACAACGTGTGGGCC120ACCCAGGCGTGCGTGCCCACCGACCCCAACCCCCAGGAGGTGGAGCTCGTGAACGTGACC180GAGAACTTCAACATGTGGAAGAACAACATGCTGGAGCAGATGCATGAGGACATCATCAGC240CTGTGGGACCAGAGCCTGAAGCCCTGCGTGAAGCTGACCCCCCTGTGCGTGACCCTGAAC300TGCACCGACCTGAGGAACACCACCAACACCAACAACAGCACCGCCAACAACAACAGCAAC360AGCGAGGGCACCATCAAGGGCGGCGAGATGAAGAACTGCAGCTTCAACATCACCACCAGC420ATCCGCGACAAGATGCAGAAGGAGTACGCCCTGCTGTACAAGCTGGATATCGTGAGCATC480CACAACGACAGCACCAGCTACCGCCTGATCTCCTGCAACACCAGCGTGATCACCCAGGCC540TGCCCCAAGATCAGCTTCGAGCCCATCCCCATCCACTACTGCGCCCCCGCCGGCTTCGCC600ATCCTGAAGTGCAACGACAAGAAGTTCAGCGGCAAGGGCAGCTGCAAGAACGTGACCACC660GTGCAGTGCACCCACGGCATCCGGCCGGTGGTGAGCACCCAGCTCCTGCTGAACGGCAGC720CTGGCCGAGGAGGAGGTGGTGATCCGCAGCGAGAACTTCACCGACAACGCCAAGACCATC780ATCGTGCACCTGAATGAGAGCGTGCAGATCAACTGCACGCGTCCCAACTACAACAAGCGC840AAGCGCATCCACATCGGCCCCGGGCGCGCCTTCTACACCACCAAGAACATCATCGGCACC900ATCCGCCAGGCCCACTGCAACATCTCTAGAGCCAAGTGGAACGACACCCTGCGCCAGATC960GTGAGCAAGCTGAAGGAGCAGTTCAAGAACAAGACCATCGTGTTCAACCAGAGCAGCGGC1020GGCGACCCCGAGATCGTGATGCACAGCTTCAACTGCGGCGGCGAATTCTTCTACTGCAAC1080ACCAGCCCCCTGTTCAACAGCACCTGGAACGGCAACAACACCTGGAACAACACCACCGGC1140AGCAACAACAATATTACCCTCCAGTGCAAGATCAAGCAGATCATCAACATGTGGCAGGAG1200GTGGGCAAGGCCATGTACGCCCCCCCCATCGAGGGCCAGATCCGGTGCAGCAGCAACATC1260ACCGGTCTGCTGCTGACCCGCGACGGCGGCAAGGACACCGACACCAACGACACCGAAATC1320TTCCGCCCCGGCGGCGGCGACATGCGCGACAACTGGAGATCTGAGCTGTACAAGTACAAG1380GTGGTGACGATCGAGCCCCTGGGCGTGGCCCCCACCAAGGCCAAGCGCCGCGTGGTGCAG1440CGCGAGAAGCGGGCCGCCATCGGCGCCCTGTTCCTGGGCTTCCTGGGGGCGGCGGGCAGC1500ACCATGGGGGCCGCCAGCGTGACCCTGACCGTGCAGGCCCGCCTGCTCCTGAGCGGCATC1560GTGCAGCAGCAGAACAACCTCCTCCGCGCCATCGAGGCCCAGCAGCATATGCTCCAGCTC1620ACCGTGTGGGGCATCAAGCAGCTCCAGGCCCGCGTGCTGGCCGTGGAGCGCTACCTGAAG1680GACCAGCAGCTCCTGGGCTTCTGGGGCTGCTCCGGCAAGCTGATCTGCACCACCACGGTA1740CCCTGGAACGCCTCCTGGAGCAACAAGAGCCTGGACGACATCTGGAACAACATGACCTGG1800ATGCAGTGGGAGCGCGAGATCGATAACTACACCAGCCTGATCTACAGCCTGCTGGAGAAG1860AGCCAGACCCAGCAGGAGAAGAACGAGCAGGAGCTGCTGGAGCTGGACAACTGGGCGAGC1920CTGTGGAACTGGTTCGACATCACCAACTGGCTGTGGTACATCAAAATCTTCATCATGATT1980GTGGGCGGCCTGGTGGGCCTCCGCATCGTGTTCGCCGTGCTGAGCATCGTGAACCGCGTG2040CGCCAGGGCTACAGCCCCCTGAGCCTCCAGACCCGGCCCCCCGTGCCGCGCGGGCCCGAC2100CGCCCCGAGGGCATCGAGGAGGAGGGCGGCGAGCGCGACCGCGACACCAGCGGCAGGCTC2160GTGCACGGCTTCCTGGCGATCATCTGGGTCGACCTCCGCAGCCTGTTCCTGTTCAGCTAC2220CACCACCGCGACCTGCTGCTGATCGCCGCCCGCATCGTGGAACTCCTAGGCCGCCGCGGC2280TGGGAGGTGCTGAAGTACTGGTGGAACCTCCTCCAGTATTGGAGCCAGGAGCTGAAGTCC2340AGCGCCGTGAGCCTGCTGAACGCCACCGCCATCGCCGTGGCCGAGGGCACCGACCGCGTG2400ATCGAGGTGCTCCAGAGGGCCGGGAGGGCGATCCTGCACATCCCCACCCGCATCCGCCAG2460GGGCTCGAGAGGGCGCTGCTG2481(2) INFORMATION FOR SEQ ID NO:36:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 486 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:36:ATGAATCCAGTAATAAGTATAACATTATTATTAAGTGTATTACAAATGAGTAGAGGACAA60AGAGTAATAAGTTTAACAGCATGTTTAGTAAATCAAAATTTGAGATTAGATTGTAGACAT120GAAAATAATACACCTTTGCCAATACAACATGAATTTTCATTAACGCGTGAAAAAAAAAAA180CATGTATTAAGTGGAACATTAGGAGTACCAGAACATACATATAGAAGTAGAGTAAATTTG240TTTAGTGATAGATTCATAAAAGTATTAACATTAGCAAATTTTACAACAAAAGATGAAGGA300GATTATATGTGTGAGCTCAGAGTAAGTGGACAAAATCCAACAAGTAGTAATAAAACAATA360AATGTAATAAGAGATAAATTAGTAAAATGTGGAGGAATAAGTTTATTAGTACAAAATACA420AGTTGGTTATTATTATTATTATTAAGTTTAAGTTTTTTACAAGCAACAGATTTTATAAGT480TTATGA486(2) INFORMATION FOR SEQ ID NO:37:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 485 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:37:ATGAACCCAGTCATCAGCATCACTCTCCTGCTTTCAGTCTTGCAGATGTCCCGAGGACAG60AGGGTGATCAGCCTGACAGCCTGCCTGGTGAACAGAACCTTCGACTGGACTGCCGTCATG120AGAATAACACCAACTTGCCCATCCAGCATGAGTTCAGCCTGACCCGAGAGAAGAAGAAGC180ACGTGCTGTCAGGCACCCTGGGGGTTCCCGAGCACACTTACCGCTCCCGCGTCAACCTTT240TCAGTGACCGCTTTATCAAGGTCCTTACTCTAGCCAACTTGACCACCAAGGATGAGGGCG300ACTACATGTGTGAACTTCGAGTCTCGGGCCAGAATCCCACAAGCTCCAATAAAACTATCA360ATGTGATCAGAGACAAGCTGGTCAAGTGTGGTGGCATAAGCCTGCTGGTTCAAAACACTT420CCTGGCTGCTGCTGCTCCTGCTTTCCCTCTCCTTCCTCCAAGCCACGGACTTCATTTCTC480TGTGA485__________________________________________________________________________</pre>
    
    </div></div></div><div class="patent-section patent-tabular-section"><a id="backward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Cited Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4652639">US4652639</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">May 6, 1982</td><td class="patent-data-table-td patent-date-value">Mar 24, 1987</td><td class="patent-data-table-td ">Amgen</td><td class="patent-data-table-td ">Manufacture and expression of structural genes</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5082767">US5082767</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Feb 27, 1989</td><td class="patent-data-table-td patent-date-value">Jan 21, 1992</td><td class="patent-data-table-td ">Hatfield G Wesley</td><td class="patent-data-table-td ">Codon pair utilization</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5270171">US5270171</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jun 18, 1990</td><td class="patent-data-table-td patent-date-value">Dec 14, 1993</td><td class="patent-data-table-td ">Boris Cercek</td><td class="patent-data-table-td ">Cancer-associated SCM-recognition factor, preparation and method of use</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5276268">US5276268</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jul 29, 1991</td><td class="patent-data-table-td patent-date-value">Jan 4, 1994</td><td class="patent-data-table-td ">Hoechst Aktiengesellschaft</td><td class="patent-data-table-td ">Phosphinothricin-resistance gene, and its use</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5405776">US5405776</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jan 24, 1992</td><td class="patent-data-table-td patent-date-value">Apr 11, 1995</td><td class="patent-data-table-td ">Life Technologies, Inc.</td><td class="patent-data-table-td ">Genetic engineering</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5464774">US5464774</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Dec 31, 1987</td><td class="patent-data-table-td patent-date-value">Nov 7, 1995</td><td class="patent-data-table-td ">The Salk Institute For Biological Studies</td><td class="patent-data-table-td ">DNA sequence</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="npl-citations"></a><div class="patent-section-header"><span class="patent-section-title">Non-Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th colspan="3"class="patent-data-table-th">Reference</th></tr></thead><tr><td class="patent-data-table-td ">1</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Bosch, M. L., et al., Journal of Virology, vol. 68, No. 11, "<a href='http://scholar.google.com/scholar?q="Insertion+of+N-linked+glycosylation+sites+in+the+variable+regions+of+the+human+immunodeficiency+virus+type-1+surface+protein+through+AAT+triplet+reiteration"'>Insertion of N-linked glycosylation sites in the variable regions of the human immunodeficiency virus type-1 surface protein through AAT triplet reiteration</a>", pp. 7566-7569, 1994.</td></tr><tr><td class="patent-data-table-td ">2</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Bosch, M. L., et al., Journal of Virology, vol. 68, No. 11, Insertion of N linked glycosylation sites in the variable regions of the human immunodeficiency virus type 1 surface protein through AAT triplet reiteration , pp. 7566 7569, 1994.</td></tr><tr><td class="patent-data-table-td ">3</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Chou, K. C., et al., AIDS Research and Human Retroviruses, vol. 8, No. 12, Diagrammatization of codon usage in 339 human immunodeficiency virus proteins and its biological implication , pp. 1967 1976, 1992.</td></tr><tr><td class="patent-data-table-td ">4</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Chou, K.-C., et al., AIDS Research and Human Retroviruses, vol. 8, No. 12, "<a href='http://scholar.google.com/scholar?q="Diagrammatization+of+codon+usage+in+339+human+immunodeficiency+virus+proteins+and+its+biological+implication"'>Diagrammatization of codon usage in 339 human immunodeficiency virus proteins and its biological implication</a>", pp. 1967-1976, 1992.</td></tr><tr><td class="patent-data-table-td ">5</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Cochrane et al., "<a href='http://scholar.google.com/scholar?q="Identification+and+Charaterization+of+Intragenic+Sequences+Which+Repress+Human+Immunodeficiency+Virus+Structural+Gene+Expression"'>Identification and Charaterization of Intragenic Sequences Which Repress Human Immunodeficiency Virus Structural Gene Expression</a>", Journal of Virology, 65(10):5305-5314, (1991).</td></tr><tr><td class="patent-data-table-td ">6</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Cochrane et al., Identification and Charaterization of Intragenic Sequences Which Repress Human Immunodeficiency Virus Structural Gene Expression , Journal of Virology, 65(10):5305 5314, (1991).</td></tr><tr><td class="patent-data-table-td ">7</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Cohen, J., et al., "<a href='http://scholar.google.com/scholar?q="Expression+of+the+HIV-1+env+%28gp160%29+in+the+yeast+Saccharomyces+cerevisiae+via+expression%2Fsecretion+vectors+and+partial+characterization+of+the+gene+product"'>Expression of the HIV-1 env (gp160) in the yeast Saccharomyces cerevisiae via expression/secretion vectors and partial characterization of the gene product</a>", in Modern Approaches, New Vaccines, pages 1988.</td></tr><tr><td class="patent-data-table-td ">8</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Cohen, J., et al., (abstract), in Modern Approaches New Vaccines, "<a href='http://scholar.google.com/scholar?q="Expression+of+the+HIV-1+env+%28gp160%29+in+the+yeast+Saccaromyces+cerevisiae+via+expression%2Fsecretion+vectors+and+partial+characterization+of+the+gene+product"'>Expression of the HIV-1 env (gp160) in the yeast Saccaromyces cerevisiae via expression/secretion vectors and partial characterization of the gene product</a>", 1988.</td></tr><tr><td class="patent-data-table-td ">9</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Cohen, J., et al., (abstract), in Modern Approaches New Vaccines, Expression of the HIV 1 env (gp160) in the yeast Saccaromyces cerevisiae via expression/secretion vectors and partial characterization of the gene product , 1988.</td></tr><tr><td class="patent-data-table-td ">10</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Cohen, J., et al., Expression of the HIV 1 env (gp160) in the yeast Saccharomyces cerevisiae via expression/secretion vectors and partial characterization of the gene product , in Modern Approaches, New Vaccines, pages 1988.</td></tr><tr><td class="patent-data-table-td ">11</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Coulombe and Skup, Gene 46:89 95 (1986).</td></tr><tr><td class="patent-data-table-td ">12</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Coulombe and Skup, Gene 46:89-95 (1986).</td></tr><tr><td class="patent-data-table-td ">13</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">D Agostino et al., The Rev Protein of Human Immunodeficiency Virus Type 1 Promotes Polysomal Association and Translation of gag/pol and vpu/env mRNAs , Molucular and Cellular Biology, 12(3):1375 1386, (1992).</td></tr><tr><td class="patent-data-table-td ">14</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">D'Agostino et al., "<a href='http://scholar.google.com/scholar?q="The+Rev+Protein+of+Human+Immunodeficiency+Virus+Type+1+Promotes+Polysomal+Association+and+Translation+of+gag%2Fpol+and+vpu%2Fenv+mRNAs"'>The Rev Protein of Human Immunodeficiency Virus Type 1 Promotes Polysomal Association and Translation of gag/pol and vpu/env mRNAs</a>", Molucular and Cellular Biology, 12(3):1375-1386, (1992).</td></tr><tr><td class="patent-data-table-td ">15</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Feinberg et al., "<a href='http://scholar.google.com/scholar?q="HTLV-III+Expression+and+Production+Involve+Complex+Regulation+at+the+Levels+of+Splicing+and+Translation+of+Viral+RNA"'>HTLV-III Expression and Production Involve Complex Regulation at the Levels of Splicing and Translation of Viral RNA</a>", Cell, 46:807-817, (1986).</td></tr><tr><td class="patent-data-table-td ">16</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Feinberg et al., HTLV III Expression and Production Involve Complex Regulation at the Levels of Splicing and Translation of Viral RNA , Cell, 46:807 817, (1986).</td></tr><tr><td class="patent-data-table-td ">17</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Fortkamp, E., et al., DNA, vol. 5, No. 6, "<a href='http://scholar.google.com/scholar?q="Cloning+and+expression+in+Escherichia+coli+of+a+synthetic+DNA+for+hirudin%2C+the+blood+coagulation+inhibitor+in+the+leech"'>Cloning and expression in Escherichia coli of a synthetic DNA for hirudin, the blood coagulation inhibitor in the leech</a>", pp. 511-517, 1986.</td></tr><tr><td class="patent-data-table-td ">18</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Fortkamp, E., et al., DNA, vol. 5, No. 6, Cloning and expression in Escherichia coli of a synthetic DNA for hirudin, the blood coagulation inhibitor in the leech , pp. 511 517, 1986.</td></tr><tr><td class="patent-data-table-td ">19</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Hammarskjold et al., "<a href='http://scholar.google.com/scholar?q="Regulation+of+Human+Immunodeficiency+Virus+env+Expression+by+the+rev+Gene+Product"'>Regulation of Human Immunodeficiency Virus env Expression by the rev Gene Product</a>"Journal of Virology, 63(5):1959-1966, (1989).</td></tr><tr><td class="patent-data-table-td ">20</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Hammarskjold et al., Regulation of Human Immunodeficiency Virus env Expression by the rev Gene Product Journal of Virology, 63(5):1959 1966, (1989).</td></tr><tr><td class="patent-data-table-td ">21</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Hernan et al., Biochemistry 31:861908628 (1992).</td></tr><tr><td class="patent-data-table-td ">22</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Holler, T. P., et al., Gene, vol. 136, Nos. 1 &amp; 2, "<a href='http://scholar.google.com/scholar?q="HIV-1+integrase+expressed+in+Escherichia+coli+from+SyntheticC+Gene"'>HIV-1 integrase expressed in Escherichia coli from SyntheticC Gene</a>", PP. 323-328 , 1993.</td></tr><tr><td class="patent-data-table-td ">23</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Holler, T. P., et al., Gene, vol. 136, Nos. 1 &amp; 2, HIV 1 integrase expressed in Escherichia coli from SyntheticC Gene , PP. 323 328 , 1993.</td></tr><tr><td class="patent-data-table-td ">24</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Inouye et al., FEBS Letters 341:277 280 (1994).</td></tr><tr><td class="patent-data-table-td ">25</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Inouye et al., FEBS Letters 341:277-280 (1994).</td></tr><tr><td class="patent-data-table-td ">26</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Kamiya et al., Jpn J. Cancer Res. 80:200 203 (1989).</td></tr><tr><td class="patent-data-table-td ">27</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Kamiya et al., Jpn J. Cancer Res. 80:200-203 (1989).</td></tr><tr><td class="patent-data-table-td ">28</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">McCarrey, Nucleic Acids Res. 18:949 955 (1990).</td></tr><tr><td class="patent-data-table-td ">29</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">McCarrey, Nucleic Acids Res. 18:949-955 (1990).</td></tr><tr><td class="patent-data-table-td ">30</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Nakamura et al., "<a href='http://scholar.google.com/scholar?q="Two+types+of+linkage+between+codon+usage+and+gene-expression+levels"'>Two types of linkage between codon usage and gene-expression levels</a>", FEBS LETTERS, 289(1):123-125, (1991).</td></tr><tr><td class="patent-data-table-td ">31</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Nakamura et al., Two types of linkage between codon usage and gene expression levels , FEBS LETTERS , 289(1):123 125, (1991).</td></tr><tr><td class="patent-data-table-td ">32</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Newgard et al., Proc. Natl. Acad. Sci. USA 83:8132 8136 (1986).</td></tr><tr><td class="patent-data-table-td ">33</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Newgard et al., Proc. Natl. Acad. Sci. USA 83:8132-8136 (1986).</td></tr><tr><td class="patent-data-table-td ">34</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Rangwala, S. H., et al., Gene, vol. 122, No. 2, "<a href='http://scholar.google.com/scholar?q="High-level+production+of+active+HIV-1+protease+in+Escherichia+coli"'>High-level production of active HIV-1 protease in Escherichia coli</a>", pp. 263-269.</td></tr><tr><td class="patent-data-table-td ">35</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Rangwala, S. H., et al., Gene, vol. 122, No. 2, High level production of active HIV 1 protease in Escherichia coli , pp. 263 269.</td></tr><tr><td class="patent-data-table-td ">36</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Robinson et al., "<a href='http://scholar.google.com/scholar?q="Codon+usage+can+affect+efficiency+of+translation+of+genes+in+Escherichia+coli"'>Codon usage can affect efficiency of translation of genes in Escherichia coli</a>", Nucleic Acids Research, 12(17):6663-6671, (1984).</td></tr><tr><td class="patent-data-table-td ">37</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Robinson et al., Codon usage can affect efficiency of translation of genes in Escherichia coli , Nucleic Acids Research, 12(17):6663 6671, (1984).</td></tr><tr><td class="patent-data-table-td ">38</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Scorer, C. A., et al., Gene, vol. 136, Nos. 1 &amp; 2, "<a href='http://scholar.google.com/scholar?q="The+intracellular+production+and+secretion+of+HIV-1+envelope+protein+in+the+methylotrophic+yeast+Picha+pastoris"'>The intracellular production and secretion of HIV-1 envelope protein in the methylotrophic yeast Picha pastoris</a>", pp. 111-119, 1993.</td></tr><tr><td class="patent-data-table-td ">39</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Scorer, C. A., et al., Gene, vol. 136, Nos. 1 &amp; 2, The intracellular production and secretion of HIV 1 envelope protein in the methylotrophic yeast Picha pastoris , pp. 111 119, 1993.</td></tr><tr><td class="patent-data-table-td ">40</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Sharp et al., Nucleic Acids Res. 16:8207 8211 (1988).</td></tr><tr><td class="patent-data-table-td ">41</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Sharp et al., Nucleic Acids Res. 16:8207-8211 (1988).</td></tr><tr><td class="patent-data-table-td ">42</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">van Hemert, F. J., et al., Journal of Molecular Evolution, vol. 41, No. 2, "<a href='http://scholar.google.com/scholar?q="The+tendency+of+lentiviral+open+reading+frames+to+become+A-rich%3A+Constraints+imposed+by+viral+genome+organization+and+cellular+tRNA+availability"'>The tendency of lentiviral open reading frames to become A-rich: Constraints imposed by viral genome organization and cellular tRNA availability</a>", pp. 132-140, 1995.</td></tr><tr><td class="patent-data-table-td ">43</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">van Hemert, F. J., et al., Journal of Molecular Evolution, vol. 41, No. 2, The tendency of lentiviral open reading frames to become A rich: Constraints imposed by viral genome organization and cellular tRNA availability , pp. 132 140, 1995.</td></tr><tr><td class="patent-data-table-td ">44</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Williams et al., Nucleic Acids Research 16:10453 10467 (1988).</td></tr><tr><td class="patent-data-table-td ">45</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Williams et al., Nucleic Acids Research 16:10453-10467 (1988).</td></tr><tr><td class="patent-data-table-td ">46</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Zhang et al., "<a href='http://scholar.google.com/scholar?q="Graphic+Analysis+of+Codon+Usage+Strategy+in+1490+Human+Proteins"'>Graphic Analysis of Codon Usage Strategy in 1490 Human Proteins</a>", Journal of Protein Chemistry, 12(3):329-335, (1993).</td></tr><tr><td class="patent-data-table-td ">47</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Zhang et al., "<a href='http://scholar.google.com/scholar?q="Low-usage+codons+in+Escherichia+coli%2C+yeast%2C+fruit+fly+and+primates"'>Low-usage codons in Escherichia coli, yeast, fruit fly and primates</a>", Gene, 105(1):61-72 (1991).</td></tr><tr><td class="patent-data-table-td ">48</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Zhang et al., Graphic Analysis of Codon Usage Strategy in 1490 Human Proteins , Journal of Protein Chemistry, 12(3):329 335, (1993).</td></tr><tr><td class="patent-data-table-td ">49</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Zhang et al., Low usage codons in Escherichia coli , yeast, fruit fly and primates , Gene, 105(1):61 72 (1991).</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="forward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Referenced by</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Citing Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6114148">US6114148</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Sep 20, 1996</td><td class="patent-data-table-td patent-date-value">Sep 5, 2000</td><td class="patent-data-table-td ">The General Hospital Corporation</td><td class="patent-data-table-td ">High level expression of proteins</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6534312">US6534312</a></td><td class="patent-data-table-td patent-date-value">Jun 28, 1999</td><td class="patent-data-table-td patent-date-value">Mar 18, 2003</td><td class="patent-data-table-td ">Merck &amp; Co., Inc.</td><td class="patent-data-table-td ">Vaccines comprising synthetic genes</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6602705">US6602705</a></td><td class="patent-data-table-td patent-date-value">Dec 30, 1999</td><td class="patent-data-table-td patent-date-value">Aug 5, 2003</td><td class="patent-data-table-td ">Chiron Corporation</td><td class="patent-data-table-td ">Expression of HIV polypeptides and production of virus-like particles</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6649409">US6649409</a></td><td class="patent-data-table-td patent-date-value">Mar 29, 2000</td><td class="patent-data-table-td patent-date-value">Nov 18, 2003</td><td class="patent-data-table-td ">Statens Serum Institut</td><td class="patent-data-table-td ">Method for producing a nucleotide sequence construct with optimized codons for an HIV genetic vaccine based on a primary, early HIV isolate and synthetic envelope BX08 constructs</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6653125">US6653125</a></td><td class="patent-data-table-td patent-date-value">Feb 17, 2000</td><td class="patent-data-table-td patent-date-value">Nov 25, 2003</td><td class="patent-data-table-td ">Merck &amp; Co., Inc.</td><td class="patent-data-table-td ">Synthetic hepatitis C genes</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6656706">US6656706</a></td><td class="patent-data-table-td patent-date-value">Jun 1, 2001</td><td class="patent-data-table-td patent-date-value">Dec 2, 2003</td><td class="patent-data-table-td ">The United States Of America As Represented By The Department Of Health And Human Services</td><td class="patent-data-table-td ">Molecular clones with mutated HIV gag/pol, SIV gag and SIV env genes</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6780974">US6780974</a></td><td class="patent-data-table-td patent-date-value">Oct 15, 2001</td><td class="patent-data-table-td patent-date-value">Aug 24, 2004</td><td class="patent-data-table-td ">Cellomics, Inc.</td><td class="patent-data-table-td ">Synthetic DNA encoding an orange seapen-derived green fluorescent protein with codon preference of mammalian expression systems and biosensors</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6924365">US6924365</a></td><td class="patent-data-table-td patent-date-value">Sep 28, 1999</td><td class="patent-data-table-td patent-date-value">Aug 2, 2005</td><td class="patent-data-table-td ">Transkaryotic Therapies, Inc.</td><td class="patent-data-table-td ">Optimized messenger RNA</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7045599">US7045599</a></td><td class="patent-data-table-td patent-date-value">Jul 14, 2004</td><td class="patent-data-table-td patent-date-value">May 16, 2006</td><td class="patent-data-table-td ">Prolume Ltd.</td><td class="patent-data-table-td ">Synthetic DNA encoding an orange seapen-derived green fluorescent protein with codon preference of mammalian expression systems and biosensors</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7132262">US7132262</a></td><td class="patent-data-table-td patent-date-value">Jul 30, 2003</td><td class="patent-data-table-td patent-date-value">Nov 7, 2006</td><td class="patent-data-table-td ">Smithkline Beecham Corp.</td><td class="patent-data-table-td ">administering codon optimized polynucleotides that code human papillomavirus polypeptides comprising early proteins or fragments to cells of mammals; gene therapy; DNA vaccines</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7238497">US7238497</a></td><td class="patent-data-table-td patent-date-value">Jul 14, 2004</td><td class="patent-data-table-td patent-date-value">Jul 3, 2007</td><td class="patent-data-table-td ">Prolume, Ltd.</td><td class="patent-data-table-td ">Synthetic DNA encoding an orange seapen-derived green fluorescent protein with codon preference of mammalian expression systems and biosensors</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7291711">US7291711</a></td><td class="patent-data-table-td patent-date-value">Dec 23, 2004</td><td class="patent-data-table-td patent-date-value">Nov 6, 2007</td><td class="patent-data-table-td ">University Of Miami</td><td class="patent-data-table-td ">Mutant fluorescent protein derived from green and red flourescent protein for use as imaging and reporter tools in biological systems; imaging diagnostics</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7348177">US7348177</a></td><td class="patent-data-table-td patent-date-value">Mar 11, 2003</td><td class="patent-data-table-td patent-date-value">Mar 25, 2008</td><td class="patent-data-table-td ">Novartis Vaccines And Diagnostics, Inc.</td><td class="patent-data-table-td ">Expression of HIV polypeptides and production of virus-like particles</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7413874">US7413874</a></td><td class="patent-data-table-td patent-date-value">Dec 9, 2002</td><td class="patent-data-table-td patent-date-value">Aug 19, 2008</td><td class="patent-data-table-td ">University Of Miami</td><td class="patent-data-table-td ">Nucleic acid encoding fluorescent proteins from aquatic species</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7561972">US7561972</a></td><td class="patent-data-table-td patent-date-value">Jul 31, 2008</td><td class="patent-data-table-td patent-date-value">Jul 14, 2009</td><td class="patent-data-table-td ">Dna Twopointo, Inc.</td><td class="patent-data-table-td ">designing a polynucleotide sequence that encodes a polypeptide sequence; When the respective sequence element encodes a portion of polypeptide sequence, the sequence element is incorporated into the polynucleotide at a frequency of occurrence that is within the frequency range specified; computer medium</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7561973">US7561973</a></td><td class="patent-data-table-td patent-date-value">Jul 31, 2008</td><td class="patent-data-table-td patent-date-value">Jul 14, 2009</td><td class="patent-data-table-td ">Dna Twopointo, Inc.</td><td class="patent-data-table-td ">designing nucleic acids with desired properties, particularly for the expression of encoded proteins; drug design; reliable algorithm with which to assess the likely level of protein expressed from a synthetic gene</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7608422">US7608422</a></td><td class="patent-data-table-td patent-date-value">Aug 19, 2003</td><td class="patent-data-table-td patent-date-value">Oct 27, 2009</td><td class="patent-data-table-td ">The United States Of America As Represented By The Secretary Of The Department Of Health And Human Services</td><td class="patent-data-table-td ">Gene therapy; vaccines; inducing antibodies and/or cytotoxic or helper T lymphocytes</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7662916">US7662916</a></td><td class="patent-data-table-td patent-date-value">Jul 25, 2003</td><td class="patent-data-table-td patent-date-value">Feb 16, 2010</td><td class="patent-data-table-td ">Novartis Vaccines &amp; Diagnostics, Inc</td><td class="patent-data-table-td ">Modified HIV Env polypeptides</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7718401">US7718401</a></td><td class="patent-data-table-td patent-date-value">Feb 6, 2008</td><td class="patent-data-table-td patent-date-value">May 18, 2010</td><td class="patent-data-table-td ">Novartis Vaccines And Diagnostics, Inc.</td><td class="patent-data-table-td ">Expression of HIV polypeptides and production of virus-like particles</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7728118">US7728118</a></td><td class="patent-data-table-td patent-date-value">Sep 17, 2004</td><td class="patent-data-table-td patent-date-value">Jun 1, 2010</td><td class="patent-data-table-td ">Promega Corporation</td><td class="patent-data-table-td ">Expression vector comprising modified luciferase for use as gene transfer and report agent in heterologous gene expression</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7790419">US7790419</a></td><td class="patent-data-table-td patent-date-value">Jan 27, 2003</td><td class="patent-data-table-td patent-date-value">Sep 7, 2010</td><td class="patent-data-table-td ">Oxford Biomedica (Uk) Ltd.</td><td class="patent-data-table-td ">Antiviral vectors</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7879540">US7879540</a></td><td class="patent-data-table-td patent-date-value">Aug 24, 2000</td><td class="patent-data-table-td patent-date-value">Feb 1, 2011</td><td class="patent-data-table-td ">Promega Corporation</td><td class="patent-data-table-td ">Nucleotide sequences modified to present reduced transcriptional characteristics subsequent to gene expression</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7906282">US7906282</a></td><td class="patent-data-table-td patent-date-value">Apr 12, 2007</td><td class="patent-data-table-td patent-date-value">Mar 15, 2011</td><td class="patent-data-table-td ">Promega Corporation</td><td class="patent-data-table-td ">Synthetic nucleic acid molecule compositions and methods of preparation</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7935805">US7935805</a></td><td class="patent-data-table-td patent-date-value">Jul 5, 2000</td><td class="patent-data-table-td patent-date-value">May 3, 2011</td><td class="patent-data-table-td ">Novartis Vaccines &amp; Diagnostics, Inc</td><td class="patent-data-table-td ">Polynucleotides encoding antigenic HIV Type C polypeptides, polypeptides and uses thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7943375">US7943375</a></td><td class="patent-data-table-td patent-date-value">Jun 25, 2008</td><td class="patent-data-table-td patent-date-value">May 17, 2011</td><td class="patent-data-table-td ">Novartis Vaccines &amp; Diagnostics, Inc</td><td class="patent-data-table-td ">Polynucleotides encoding antigenic HIV type C polypeptides, polypeptides and uses thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8008006">US8008006</a></td><td class="patent-data-table-td patent-date-value">Jul 5, 2007</td><td class="patent-data-table-td patent-date-value">Aug 30, 2011</td><td class="patent-data-table-td ">Promega Corporation</td><td class="patent-data-table-td ">Synthetic nucleic acid molecule compositions and methods of preparation</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8030017">US8030017</a></td><td class="patent-data-table-td patent-date-value">Jul 31, 2009</td><td class="patent-data-table-td patent-date-value">Oct 4, 2011</td><td class="patent-data-table-td ">Promega Corporation</td><td class="patent-data-table-td ">Thermostable luciferases and methods of production</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8076100">US8076100</a></td><td class="patent-data-table-td patent-date-value">Oct 2, 2002</td><td class="patent-data-table-td patent-date-value">Dec 13, 2011</td><td class="patent-data-table-td ">The United States Of America As Represented By The Department Of Health And Human Services</td><td class="patent-data-table-td ">Genetic engineering; vaccine; induction antibodies</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8106176">US8106176</a></td><td class="patent-data-table-td patent-date-value">Oct 3, 2006</td><td class="patent-data-table-td patent-date-value">Jan 31, 2012</td><td class="patent-data-table-td ">Instituto di Richerche di Biologia Molecolare P. Angeletti SpA</td><td class="patent-data-table-td ">Matrix metalloproteinase 11 vaccine</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8126653">US8126653</a></td><td class="patent-data-table-td patent-date-value">Jul 31, 2008</td><td class="patent-data-table-td patent-date-value">Feb 28, 2012</td><td class="patent-data-table-td ">Dna Twopointo, Inc.</td><td class="patent-data-table-td ">Synthetic nucleic acids for expression of encoded proteins</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8168418">US8168418</a></td><td class="patent-data-table-td patent-date-value">Apr 30, 2010</td><td class="patent-data-table-td patent-date-value">May 1, 2012</td><td class="patent-data-table-td ">Susan W Barnett</td><td class="patent-data-table-td ">Expression of HIV polypeptides and production of virus-like particles</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8263394">US8263394</a></td><td class="patent-data-table-td patent-date-value">Jan 23, 2008</td><td class="patent-data-table-td patent-date-value">Sep 11, 2012</td><td class="patent-data-table-td ">Novartis Vaccines &amp; Diagnostics Inc.</td><td class="patent-data-table-td ">Polynucleotides encoding antigenic HIV type B polypeptides, polypeptides, and uses thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8401798">US8401798</a></td><td class="patent-data-table-td patent-date-value">Jul 31, 2008</td><td class="patent-data-table-td patent-date-value">Mar 19, 2013</td><td class="patent-data-table-td ">Dna Twopointo, Inc.</td><td class="patent-data-table-td ">Systems and methods for constructing frequency lookup tables for expression systems</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8492522">US8492522</a></td><td class="patent-data-table-td patent-date-value">Dec 21, 2011</td><td class="patent-data-table-td patent-date-value">Jul 23, 2013</td><td class="patent-data-table-td ">Istituto Di Ricerche Di Biologia Molecolare P. Angeletti Spa</td><td class="patent-data-table-td ">Matrix metalloproteinase 11 vaccine</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8541230">US8541230</a></td><td class="patent-data-table-td patent-date-value">May 16, 2011</td><td class="patent-data-table-td patent-date-value">Sep 24, 2013</td><td class="patent-data-table-td ">Novartis Vaccines And Diagnostics, Inc</td><td class="patent-data-table-td ">Polynucleotides encoding antigenic HIV type C polypeptides, polypeptides and uses thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP1433851A2?cl=en">EP1433851A2</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Dec 30, 1999</td><td class="patent-data-table-td patent-date-value">Jun 30, 2004</td><td class="patent-data-table-td ">Chiron Corporation</td><td class="patent-data-table-td ">Improved expression of HIV polypeptides and production of virus-like particles</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2366787A2?cl=en">EP2366787A2</a></td><td class="patent-data-table-td patent-date-value">Jan 12, 2007</td><td class="patent-data-table-td patent-date-value">Sep 21, 2011</td><td class="patent-data-table-td ">The Government Of The U.S.A, As Represented By The Secretary, Dept. Of Health And Human Services</td><td class="patent-data-table-td ">Codon optimized IL-15 and IL-15R-Alpha genes for expression in mammalian cells</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2385065A1?cl=en">EP2385065A1</a></td><td class="patent-data-table-td patent-date-value">Oct 15, 2008</td><td class="patent-data-table-td patent-date-value">Nov 9, 2011</td><td class="patent-data-table-td ">Perseid Therapeutics LLC</td><td class="patent-data-table-td ">Immunosuppressive polypeptides and nucleic acids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2612867A1?cl=en">EP2612867A1</a></td><td class="patent-data-table-td patent-date-value">Oct 15, 2008</td><td class="patent-data-table-td patent-date-value">Jul 10, 2013</td><td class="patent-data-table-td ">Perseid Therapeutics LLC</td><td class="patent-data-table-td ">Immunosuppressive polypeptides and nucleic acids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2612868A1?cl=en">EP2612868A1</a></td><td class="patent-data-table-td patent-date-value">Oct 15, 2008</td><td class="patent-data-table-td patent-date-value">Jul 10, 2013</td><td class="patent-data-table-td ">Perseid Therapeutics LLC</td><td class="patent-data-table-td ">Immunosuppressive polypeptides and nucleic acids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2000045823A1?cl=en">WO2000045823A1</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Feb 7, 2000</td><td class="patent-data-table-td patent-date-value">Aug 10, 2000</td><td class="patent-data-table-td ">Chiron Corp</td><td class="patent-data-table-td ">Electrically-mediated enhancement of dna vaccine immunity and efficacy in vivo</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2007084342A2?cl=en">WO2007084342A2</a></td><td class="patent-data-table-td patent-date-value">Jan 12, 2007</td><td class="patent-data-table-td patent-date-value">Jul 26, 2007</td><td class="patent-data-table-td ">Us Gov Health &amp; Human Serv</td><td class="patent-data-table-td ">Codon optimi zed il- 15 and il- 15r-alpha genes for expression in mammalian cells</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2009058564A2?cl=en">WO2009058564A2</a></td><td class="patent-data-table-td patent-date-value">Oct 15, 2008</td><td class="patent-data-table-td patent-date-value">May 7, 2009</td><td class="patent-data-table-td ">Maxygen Inc</td><td class="patent-data-table-td ">Immunosuppressive polypeptides and nucleic acids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2012098111A1?cl=en">WO2012098111A1</a></td><td class="patent-data-table-td patent-date-value">Jan 17, 2012</td><td class="patent-data-table-td patent-date-value">Jul 26, 2012</td><td class="patent-data-table-td ">Philip Morris Products S.A.</td><td class="patent-data-table-td ">Vectors for nucleic acid expression in plants</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2012098119A2?cl=en">WO2012098119A2</a></td><td class="patent-data-table-td patent-date-value">Jan 17, 2012</td><td class="patent-data-table-td patent-date-value">Jul 26, 2012</td><td class="patent-data-table-td ">Philip Morris Products S.A.</td><td class="patent-data-table-td ">Protein expression in plants</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2013049227A2?cl=en">WO2013049227A2</a></td><td class="patent-data-table-td patent-date-value">Sep 26, 2012</td><td class="patent-data-table-td patent-date-value">Apr 4, 2013</td><td class="patent-data-table-td ">Geneart Ag</td><td class="patent-data-table-td ">High efficiency, small volume nucleic acid synthesis</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="classifications"></a><div class="patent-section-header"><span class="patent-section-title">Classifications</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th"> </th><th class="patent-data-table-th"> </th></tr></thead><tr><td class="patent-data-table-td ">U.S. Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=TylHBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S023500">536/23.5</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=TylHBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S069100">435/69.1</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=TylHBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S023720">536/23.72</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=TylHBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S252300">435/252.3</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=TylHBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S091500">435/91.5</a></span></td></tr><tr><td class="patent-data-table-td ">International Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=TylHBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C07K0014435000">C07K14/435</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=TylHBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C07K0014705000">C07K14/705</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=TylHBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0015670000">C12N15/67</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=TylHBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0015850000">C12N15/85</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=TylHBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0015090000">C12N15/09</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=TylHBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C07K0014160000">C07K14/16</a></span></td></tr><tr><td class="patent-data-table-td ">Cooperative Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=TylHBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C07K14/43595">C07K14/43595</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=TylHBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C07K14/70503">C07K14/70503</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=TylHBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N15/67">C12N15/67</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=TylHBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C07K14/005">C07K14/005</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=TylHBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N2740/16122">C12N2740/16122</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=TylHBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N15/85">C12N15/85</a></span></td></tr><tr><td class="patent-data-table-td ">European Classification</td><td class="patent-data-table-td "><span class="nested-value">C07K14/005</span>, <span class="nested-value">C07K14/705B</span>, <span class="nested-value">C07K14/435A5</span>, <span class="nested-value">C12N15/85</span>, <span class="nested-value">C12N15/67</span></td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="legal-events"></a><div class="patent-section-header"><span class="patent-section-title">Legal Events</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Date</th><th class="patent-data-table-th">Code</th><th class="patent-data-table-th">Event</th><th class="patent-data-table-th">Description</th></tr></thead><tr><td class="patent-data-table-td patent-date-value">Sep 3, 2013</td><td class="patent-data-table-td ">B2</td><td class="patent-data-table-td ">Reexamination certificate second reexamination</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">THE PATENTABILITY OF CLAIMS 12, 13 AND 18 IS CONFIRMED.CLAIM 8 WAS PREVIOUSLY CANCELLED.CLAIMS 1-7,9-11, 17 AND 19-30 ARE CANCELLED.CLAIMS 14-16 WERE NOT REEXAMINED.</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Sep 25, 2012</td><td class="patent-data-table-td ">CC</td><td class="patent-data-table-td ">Certificate of correction</td><td class="patent-data-table-td "></td></tr><tr><td class="patent-data-table-td patent-date-value">Aug 7, 2012</td><td class="patent-data-table-td ">RR</td><td class="patent-data-table-td ">Request for reexamination filed</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20120622</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Apr 24, 2012</td><td class="patent-data-table-td ">B1</td><td class="patent-data-table-td ">Reexamination certificate first reexamination</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">CLAIM 8 IS CANCELLED. CLAIMS 1, 12, 17 AND 18 ARE DETERMINED TO BE PATENTABLE AS AMENDED. CLAIMS 2-7, 9-11 AND 13-16, DEPENDENT ON AN AMENDED CLAIM, ARE DETERMINED TO BE PATENTABLE. NEW CLAIMS 19-30 ARE ADDED AND DETERMINED TO BE PATENTABLE.</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Mar 15, 2011</td><td class="patent-data-table-td ">RR</td><td class="patent-data-table-td ">Request for reexamination filed</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20101209</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Jan 28, 2010</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">12</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">May 4, 2009</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">NATIONAL INSTITUTES OF HEALTH (NIH), U.S. DEPT. OF</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">CONFIRMATORY LICENSE;ASSIGNOR:GENERAL HOSPITAL CORPORATION DBA MASS;REEL/FRAME:022630/0437</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20090424</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Aug 29, 2005</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">8</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Jan 3, 2002</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">4</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Nov 9, 1999</td><td class="patent-data-table-td ">CC</td><td class="patent-data-table-td ">Certificate of correction</td><td class="patent-data-table-td "></td></tr><tr><td class="patent-data-table-td patent-date-value">Jul 17, 1995</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">GENERAL HOSPITAL CORPORATION, MASSACHUSETTS</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">CORRECTIVE ASSIGNMENT TO CORRECT THE SERIAL NUMBER ON A DOCUMENT PREVIOUSLY RECORDED ON REEL 7384 FRAME 0113;ASSIGNOR:SEED, BRIAN;REEL/FRAME:007561/0429</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">19950201</span></div></td></tr></table><div class="patent-section-footer"></div></div><div class="modal-dialog" id="patent-images-lightbox"><div class="patent-lightbox-controls"><div class="patent-lightbox-rotate-controls"><div class="patent-lightbox-rotation-text">Rotate</div><div class="rotate-icon rotate-ccw-icon"></div><div class="rotate-icon rotate-cw-icon"></div></div><div class="patent-lightbox-index-counter"></div><a class="patent-lightbox-fullsize-link" target="_blank">Original Image</a><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_right.png" alt="Next page"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_left.png" alt="Previous page"width="21" height="21" /></div></div><div class="modal-dialog-content"><div class="patent-lightbox-image-holder"><div class="patent-lightbox-placeholder"></div></div></div></div><script>_OC_initPatentsAtb({image_not_available_html: " Image not available"});</script></div></div></div></td></tr></table><script>(function() {var href = window.location.href;if (href.indexOf('?') !== -1) {var parameters = href.split('?')[1].split('&');for (var i = 0; i < parameters.length; i++) {var param = parameters[i].split('=');if (param[0] == 'focus') {var elem = document.getElementById(param[1]);if (elem) {elem.focus();}}}}})();</script><script>_OC_addFlags({LockSrc:"/books/javascript/lock_8a2b04e7bf975d5171d8e4c0b6365c7a.js", Host:"http://www.google.com/", IsBooksRentalEnabled:1, IsWebstoreDisplayCaseEnabled:1, IsObfuscationEnabled:1, IsBrowsingHistoryEnabled:1, IsWebReaderSvgEnabled:0, IsGeoLayerEnabled:1, IsImageModeNotesEnabled:1, IsCopyMenuItemEnabled:1, IsGiftingEnabled:0, IsWebReaderUniversalPaginatorEnabled:0, IsOfflineBubbleEnabled:1, IsReaderEnabledForPlayRequests:1, IsFutureOnSaleVolumesEnabled:1, IsOfflineRestrictedCopyEnabled:1, IsBooksUnifiedLeftNavEnabled:1, IsRestrictedCopyEnabled:1, IsZipitFolderCollectionEnabled:1, IsEndOfSampleRecommendationsEnabled:1, IsRatingsOnBookcardsEnabled:1, IsAdsDisabled:0, IsIframePageDisplayEnabled:0, IsEmbeddedMediaEnabled:1, IsImageModeAnnotationsEnabled:1, IsMyLibraryGooglePlusEnabled:1, IsImagePageProviderEnabled:0, IsBookcardListPriceSmall:0, IsInternalUser:0, IsBooksShareButtonEnabled:0, IsPreOrdersEnabled:0, IsDisabledRandomBookshelves:0, WebstoreDisplayCasePosition:3});_OC_Run({"enable_p13n":false,"add_vol_to_collection_base_url":"http://www.google.com/patents?op=add\u0026sig=ACfU3U19B-Hq6wKJvTo62p5JrzUTZDe9aQ\u0026id=TylHBAABERAJ","remove_vol_from_collection_base_url":"http://www.google.com/patents?op=remove\u0026sig=ACfU3U3d70iYwTCfrbJFZYwDXcVpGOySYg\u0026id=TylHBAABERAJ","logged_in":false,"p13n_save_user_settings_url":"http://www.google.com/patents?op=edit_user_settings\u0026sig=ACfU3U2TqzmwHXedBmnhhrAXRc5f6QNAqQ","is_cobrand":false,"sign_in_url":"https://www.google.com/accounts/Login?service=\u0026continue=http://www.google.com/patents%3Fhl%3Den\u0026hl=en","is_play_enabled":true}, {"volume_id":"","is_ebook":true,"volumeresult":{"has_flowing_text":false,"has_scanned_text":true,"can_download_pdf":false,"can_download_epub":false,"is_pdf_drm_enabled":false,"is_epub_drm_enabled":false,"download_pdf_url":"http://www.google.com/patents/download/Overexpression_of_mammalian_and_viral_pr.pdf?id=TylHBAABERAJ\u0026output=pdf\u0026sig=ACfU3U3r9opvHhsgcWgUY1aLQdjrevT-Fg"},"sample_url":"http://www.google.com/patents/reader?id=TylHBAABERAJ\u0026printsec=frontcover\u0026output=reader\u0026source=gbs_atb_hover","is_browsable":true,"is_public_domain":true}, {});</script><div id="footer_table" style="font-size:83%;text-align:center;position:relative;top:20px;height:4.5em;margin-top:2em"><div style="margin-bottom:8px"><a href=http://www.google.com/><nobr>Google&nbsp;Home</nobr></a> - <a href=//www.google.com/patents/sitemap/><nobr>Sitemap</nobr></a> - <a href=http://www.google.com/googlebooks/uspto.html><nobr>USPTO Bulk Downloads</nobr></a> - <a href=/intl/en/privacy/><nobr>Privacy Policy</nobr></a> - <a href=/intl/en/policies/terms/><nobr>Terms of Service</nobr></a> - <a href=https://support.google.com/faqs/answer/2539193?hl=en><nobr>About Google Patents</nobr></a> - <a href="http://www.google.com/tools/feedback/intl/en/error.html" onclick="try{_OC_startFeedback({productId: '72792',locale: 'en'});return false;}catch(e){}"><nobr>Send Feedback</nobr></a></div><span>Data provided by IFI CLAIMS Patent Services</span><br><span >&copy;2012 Google</span></div> <script type="text/javascript">var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));</script><script type="text/javascript">var pageTracker = _gat._getTracker("UA-27188110-1");pageTracker._setCookiePath("/patents/");pageTracker._trackPageview();</script> </body></html>