<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN""http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html><head><title>Patent US5792640 - General method to clone hybrid restriction endonucleases using lig gene - Google Patents</title><script>(function(){(function(){function e(a){this.t={};this.tick=function(a,c,b){var d=void 0!=b?b:(new Date).getTime();this.t[a]=[d,c];if(void 0==b)try{window.console.timeStamp("CSI/"+a)}catch(e){}};this.tick("start",null,a)}var a;window.performance&&(a=window.performance.timing);var f=a?new e(a.responseStart):new e;window.jstiming={Timer:e,load:f};if(a){var c=a.navigationStart,d=a.responseStart;0<c&&d>=c&&(window.jstiming.srt=d-c)}if(a){var b=window.jstiming.load;0<c&&d>=c&&(b.tick("_wtsrt",void 0,c),b.tick("wtsrt_",
"_wtsrt",d),b.tick("tbsd_","wtsrt_"))}try{a=null,window.chrome&&window.chrome.csi&&(a=Math.floor(window.chrome.csi().pageT),b&&0<c&&(b.tick("_tbnd",void 0,window.chrome.csi().startE),b.tick("tbnd_","_tbnd",c))),null==a&&window.gtbExternal&&(a=window.gtbExternal.pageT()),null==a&&window.external&&(a=window.external.pageT,b&&0<c&&(b.tick("_tbnd",void 0,window.external.startE),b.tick("tbnd_","_tbnd",c))),a&&(window.jstiming.pt=a)}catch(g){}})();})();
</script><link rel="stylesheet" href="/patents/css/_8a2b04e7bf975d5171d8e4c0b6365c7a/kl_intl_patents_bundle.css" type="text/css" /><script src="/books/javascript/atb_8a2b04e7bf975d5171d8e4c0b6365c7a__en.js"></script><script>function googleTranslateElementInit() {new google.translate.TranslateElement({pageLanguage: "en",gaTrack: true,gaId: "UA-27188110-1",multilanguagePage: true});}</script><script src="//translate.google.com/translate_a/element.js?cb=googleTranslateElementInit"></script><meta name="DC.type" content="Patent"><meta name="DC.title" content="General method to clone hybrid restriction endonucleases using lig gene"><meta name="DC.contributor" content="Srinivasan Chandrasegaran" scheme="inventor"><meta name="DC.contributor" content="The Johns Hopkins University" scheme="assignee"><meta name="DC.date" content="1995-12-20" scheme="dateSubmitted"><meta name="DC.description" content="The present invention reveals methods for cloning hybrid restriction endonucleases and for enzymatically inactivating a target DNA. The method for cloning hybrid restriction endonucleases involves co-expression of a ligase. A first plasmid contains a gene encoding a DNA ligase. A second plasmid contains a gene encoding a hybrid restriction endonuclease and is compatible with the first plasmid. The method involves transfecting host cells with the first plasmid, so that DNA ligase is produced, followed by transfecting the cells with the second plasmid. The method for enzymatically inactivating a target DNA involves preparing a plasmid, phage, virus or any other delivery vehicle such as a liposome containing a gene encoding a nuclease, delivering the gene into cells, inducing the cells to produce the nuclease and enzymatically inactivating the target DNA."><meta name="DC.date" content="1998-8-11" scheme="issued"><meta name="DC.relation" content="US:5200333" scheme="references"><meta name="citation_reference" content="Adrian Whitehouse et al., A Carboxy Terminal Domain of the hMSH 2 Gene Product is Sufficient for Binding Specific Mismatched Oligonucleotides, Biochemical and Biophysical Research Communications, 1996, vol. 225, 289 295, Article No. 1168."><meta name="citation_reference" content="Adrian Whitehouse et al., A Carboxy Terminal Domain of the hMSH-2 Gene Product is Sufficient for Binding Specific Mismatched Oligonucleotides, Biochemical and Biophysical Research Communications, 1996, vol. 225, 289-295, Article No. 1168."><meta name="citation_reference" content="Alla Lishanski et al., Mutation Detection by Mismatch Binding Protein, MutS, in Amplified DNA: Application to the Cystic Fibrosis Gene, Proc. Natl. Acad. Sci. USA, Mar. 1994, pp. 2674 2678, vol. 91."><meta name="citation_reference" content="Alla Lishanski et al., Mutation Detection by Mismatch Binding Protein, MutS, in Amplified DNA: Application to the Cystic Fibrosis Gene, Proc. Natl. Acad. Sci. USA, Mar. 1994, pp. 2674-2678, vol. 91."><meta name="citation_reference" content="Baohua Huang et al., Splase: A New Class IIS Zinc Finger Restriction Endonuclease with Specificity for Sp1 Binding Sites, Journal of Protein Chemistry, 1996, pp. 481 489, vol. 15."><meta name="citation_reference" content="Baohua Huang et al., Splase: A New Class IIS Zinc-Finger Restriction Endonuclease with Specificity for Sp1 Binding Sites, Journal of Protein Chemistry, 1996, pp. 481-489, vol. 15."><meta name="citation_reference" content="Lin Li et al., Alteration of the Cleavage Distance of FOK Restriction Endonuclease by Insertion Mutagenesis, Pro. Natl. Acad. Sci, USA, Apr. 1993, pp. 2764 2768, vol. 90."><meta name="citation_reference" content="Lin Li et al., Alteration of the Cleavage Distance of FOK Restriction Endonuclease by Insertion Mutagenesis, Pro. Natl. Acad. Sci, USA, Apr. 1993, pp. 2764-2768, vol. 90."><meta name="citation_reference" content="Lin Li et al., C terminal Deletion Mutants of the FOK I Restriction Endonuclease, Gene, 133 1993, pp. 79 84."><meta name="citation_reference" content="Lin Li et al., C-terminal Deletion Mutants of the FOK I Restriction Endonuclease, Gene, 133 1993, pp. 79-84."><meta name="citation_reference" content="Lin Li et al., Functional Domains in FOK I Restriction Endonuclease, Proc. Natl. Acad. Sci. USA, May 1992, pp. 4275 4279, vol. 89."><meta name="citation_reference" content="Lin Li et al., Functional Domains in FOK I Restriction Endonuclease, Proc. Natl. Acad. Sci. USA, May 1992, pp. 4275-4279, vol. 89."><meta name="citation_reference" content="Rima Youil et al., Screening for Mutations by Enzyme Mismatch Cleavage with T4 Endonuclease VII, Proc. Natl. Acad. Sci. USA, Jan. 1995, pp. 87 91, vol. 92."><meta name="citation_reference" content="Rima Youil et al., Screening for Mutations by Enzyme Mismatch Cleavage with T4 Endonuclease VII, Proc. Natl. Acad. Sci. USA, Jan. 1995, pp. 87-91, vol. 92."><meta name="citation_reference" content="Yang Gyun Kim et al., Chimeric Restriction Endonuclease, Proc. Natl. Acad. Sci. USA, Feb. 1994, pp. 883 887, vol. 91."><meta name="citation_reference" content="Yang Gyun Kim et al., Hybrid Restriction Enzymes: Zinc Finger Fusions to FOK I Cleavage Domain, Proc. Natl. Acad. Sci. USA, Feb. 1996, 1156 1160, vol. 93."><meta name="citation_reference" content="Yang Gyun Kim et al., Insertion and Deletion Mutants of FOK I Restriction Endonuclease, The Journal of Biological Chemistry, Dec. 16, 1994, pp. 31978 31982, vol. 269, No. 50."><meta name="citation_reference" content="Yang-Gyun Kim et al., Chimeric Restriction Endonuclease, Proc. Natl. Acad. Sci. USA, Feb. 1994, pp. 883-887, vol. 91."><meta name="citation_reference" content="Yang-Gyun Kim et al., Hybrid Restriction Enzymes: Zinc Finger Fusions to FOK I Cleavage Domain, Proc. Natl. Acad. Sci. USA, Feb. 1996, 1156-1160, vol. 93."><meta name="citation_reference" content="Yang-Gyun Kim et al., Insertion and Deletion Mutants of FOK I Restriction Endonuclease, The Journal of Biological Chemistry, Dec. 16, 1994, pp. 31978-31982, vol. 269, No. 50."><meta name="citation_reference" content="Yen Choo et al., In Vivo Repression by a Site Specific DNA Binding Protein Designed Against an Oncogenic Sequence, Nature, Dec. 15, 1994, pp. 642 645, vol. 372."><meta name="citation_reference" content="Yen Choo et al., In Vivo Repression by a Site-Specific DNA-Binding Protein Designed Against an Oncogenic Sequence, Nature, Dec. 15, 1994, pp. 642-645, vol. 372."><meta name="citation_patent_number" content="US:5792640"><meta name="citation_patent_application_number" content="US:08/575,361"><link rel="canonical" href="http://www.google.com/patents/US5792640"/><meta property="og:url" content="http://www.google.com/patents/US5792640"/><meta name="title" content="Patent US5792640 - General method to clone hybrid restriction endonucleases using lig gene"/><meta name="description" content="The present invention reveals methods for cloning hybrid restriction endonucleases and for enzymatically inactivating a target DNA. The method for cloning hybrid restriction endonucleases involves co-expression of a ligase. A first plasmid contains a gene encoding a DNA ligase. A second plasmid contains a gene encoding a hybrid restriction endonuclease and is compatible with the first plasmid. The method involves transfecting host cells with the first plasmid, so that DNA ligase is produced, followed by transfecting the cells with the second plasmid. The method for enzymatically inactivating a target DNA involves preparing a plasmid, phage, virus or any other delivery vehicle such as a liposome containing a gene encoding a nuclease, delivering the gene into cells, inducing the cells to produce the nuclease and enzymatically inactivating the target DNA."/><meta property="og:title" content="Patent US5792640 - General method to clone hybrid restriction endonucleases using lig gene"/><meta property="og:type" content="book"/><meta property="og:site_name" content="Google Books"/><meta property="og:image" content="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><link rel="image_src" href="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><script>(function(){try{var aa=function(a,b,c,d){d=d||{};d._sn=["cfg",b,c].join(".");window.gbar.logger.ml(a,d)};var g=window.gbar=window.gbar||{},l=window.gbar.i=window.gbar.i||{},m={},n;function _tvn(a,b){var c=parseInt(a,10);return isNaN(c)?b:c}function _tvf(a,b){var c=parseFloat(a);return isNaN(c)?b:c}function _tvv(a){return!!a}function p(a,b,c){(c||g)[a]=b}g.bv={n:_tvn("2",0),r:"",f:".67.",e:"0",m:_tvn("0",1)};
function q(a,b,c){var d="on"+b;if(a.addEventListener)a.addEventListener(b,c,!1);else if(a.attachEvent)a.attachEvent(d,c);else{var f=a[d];a[d]=function(){var a=f.apply(this,arguments),b=c.apply(this,arguments);return void 0==a?b:void 0==b?a:b&&a}}}var s=function(a){return function(){return g.bv.m==a}},ba=s(1),ca=s(2);p("sb",ba);p("kn",ca);l.a=_tvv;l.b=_tvf;l.c=_tvn;l.i=aa;var da=window.gbar.i.i;var t,u,v,w;function ea(a){v=a}function fa(a){var b;if(b=v&&window.encodeURIComponent)b=a.href,b=!b.match(/^http[s]?:\/\/accounts\.google\.[^/]*\/ClearSID/i)&&!b.match(/^http[s]?:\/\/[^/]*\/accounts\/ClearSID/i);if(b=b&&encodeURIComponent(v()))a.href=a.href.replace(/([?&]continue=)[^&]*/,"$1"+b)}function ga(a){window.gApplication&&(a.href=window.gApplication.getTabUrl(a.href))}
function ha(a){var b=document.forms[0].q,c=window.encodeURIComponent&&b&&b.value,b=b&&b.placeholder;c&&c!=b&&(a.href=a.href.replace(/([?&])q=[^&]*|$/,function(a,b){return(b||"&")+"q="+encodeURIComponent(c)}))}n=l.a("")?ga:ha;
function x(a,b,c,d,f,e){var h=document.getElementById(a);if(h){var k=h.style;k.left=d?"auto":b+"px";k.right=d?b+"px":"auto";k.top=c+"px";k.visibility=u?"hidden":"visible";f&&e?(k.width=f+"px",k.height=e+"px"):(x(t,b,c,d,h.offsetWidth,h.offsetHeight),u=u?"":a)}}
var y=[],ia=function(a,b){y.push(b)},ja=function(a){a=a||window.event;var b=a.target||a.srcElement;a.cancelBubble=!0;null==t&&(a=document.createElement(Array.every||window.createPopup?"iframe":"div"),a.frameBorder="0",t=a.id="gbs",a.src="javascript:''",b.parentNode.appendChild(a),q(document,"click",z));var c=b,b=0;"gb3"!=c.className&&(c=c.parentNode);a=c.getAttribute("aria-owns")||"gbi";var d=c.offsetWidth,f=20<c.offsetTop?46:24;document.getElementById("tphdr")&&(f-=3);var e=!1;do b+=c.offsetLeft||
0;while(c=c.offsetParent);var c=(document.documentElement.clientWidth||document.body.clientWidth)-b-d,h,d=document.body,k=document.defaultView;k&&k.getComputedStyle?(d=k.getComputedStyle(d,""))&&(h=d.direction):h=d.currentStyle?d.currentStyle.direction:d.style.direction;h="rtl"==h;if("gbi"==a){for(d=0;k=y[d++];)k();A(null,window.navExtra);h&&(b=c,e=!0)}else h||(b=c,e=!0);u!=a&&z();x(a,b,f,e)},z=function(){u&&x(u,0,0)},A=function(a,b){var c,d=document.getElementById("gbi"),f=a;f||(f=d.firstChild);
for(;b&&(c=b.pop());){var e=d,h=c,k=f;w||(w="gb2");e.insertBefore(h,k).className=w}},ka=function(a,b,c){if((b=document.getElementById(b))&&a){a.className="gb4";var d=document.createElement("span");d.appendChild(a);d.appendChild(document.createTextNode(" | "));d.id=c;b.appendChild(d)}},la=function(){return document.getElementById("gb_70")},ma=function(){return!!u};p("qs",n);p("setContinueCb",ea);p("pc",fa);p("tg",ja);p("close",z);p("addLink",ka);p("almm",A);p("si",la);p("adh",ia);p("op",ma);var B=function(){},C=function(){},F=function(a){var b=new Image,c=D;b.onerror=b.onload=b.onabort=function(){try{delete E[c]}catch(a){}};E[c]=b;b.src=a;D=c+1},E=[],D=0;p("logger",{il:C,ml:B,log:F});var G=window.gbar.logger;var H={},na={},I=[],oa=l.b("0.1",.1),pa=l.a("1",!0),qa=function(a,b){I.push([a,b])},ra=function(a,b){H[a]=b},sa=function(a){return a in H},J={},K=function(a,b){J[a]||(J[a]=[]);J[a].push(b)},ta=function(a){K("m",a)},L=function(a,b){var c=document.createElement("script");c.src=a;c.async=pa;Math.random()<oa&&(c.onerror=function(){c.onerror=null;B(Error("Bundle load failed: name="+(b||"UNK")+" url="+a))});(document.getElementById("xjsc")||document.getElementsByTagName("body")[0]||
document.getElementsByTagName("head")[0]).appendChild(c)},N=function(a){for(var b=0,c;(c=I[b])&&c[0]!=a;++b);!c||c[1].l||c[1].s||(c[1].s=!0,M(2,a),c[1].url&&L(c[1].url,a),c[1].libs&&m.d&&m.d(c[1].libs))},O=function(a){K("gc",a)},P=null,ua=function(a){P=a},M=function(a,b,c){if(P){a={t:a,b:b};if(c)for(var d in c)a[d]=c[d];try{P(a)}catch(f){}}};p("mdc",H);p("mdi",na);p("bnc",I);p("qGC",O);p("qm",ta);p("qd",J);p("lb",N);p("mcf",ra);p("bcf",qa);p("aq",K);p("mdd","");p("has",sa);
p("trh",ua);p("tev",M);var Q=l.b("0.1",.001),R=0;
function _mlToken(a,b){try{if(1>R){R++;var c,d=a,f=b||{},e=encodeURIComponent,h=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&jexpid=",e("17483"),"&srcpg=",e("prop=22"),"&jsr=",Math.round(1/Q),"&ogev=",e("EKjtU8f_IIynsAT7zoD4Cg"),"&ogf=",g.bv.f,"&ogrp=",e("1"),"&ogv=",e("1407723702.0"),"&oggv="+e("es_plusone_gc_20140723.0_p0"),"&ogd=",e("com"),"&ogc=",e("NOR"),"&ogl=",e("en")];f._sn&&(f._sn="og."+
f._sn);for(var k in f)h.push("&"),h.push(e(k)),h.push("="),h.push(e(f[k]));h.push("&emsg=");h.push(e(d.name+":"+d.message));var r=h.join("");S(r)&&(r=r.substr(0,2E3));c=r;var Aa=window.gbar.logger._aem(a,c);F(Aa)}}catch(Na){}}var S=function(a){return 2E3<=a.length},va=function(a,b){return b};function T(a){B=a;p("_itl",S,G);p("_aem",va,G);p("ml",B,G);a={};H.er=a}l.a("")?T(function(a){throw a;}):l.a("1")&&Math.random()<Q&&T(_mlToken);I.push(["m",{url:"//ssl.gstatic.com/gb/js/scm_7385cc5883250b43a39405734c1bea59.js"}]);g.mcf("c",{});g.sg={c:""};if(l.a("1")){var wa=l.a("");I.push(["gc",{auto:wa,url:"//ssl.gstatic.com/gb/js/abc/gci_91f30755d6a6b787dcc2a4062e6e9824.js",libs:"googleapis.client:plusone:gapi.iframes"}]);var xa={version:"gci_91f30755d6a6b787dcc2a4062e6e9824.js",index:"",lang:"en"};H.gc=xa;var U=function(a){window.googleapis&&window.iframes?a&&a():(a&&O(a),N("gc"))};p("lGC",U);l.a("1")&&p("lPWF",U)};window.__PVT="";if(l.a("1")&&l.a("1")){var V=function(a){U(function(){K("pw",a);N("pw")})};p("lPW",V);I.push(["pw",{url:"//ssl.gstatic.com/gb/js/abc/pwm_45f73e4df07a0e388b0fa1f3d30e7280.js"}]);var W=[],ya=function(a){W[0]=a},za=function(a,b){var c=b||{};c._sn="pw";B(a,c)},Ba={signed:W,elog:za,base:"https://plusone.google.com/u/0",loadTime:(new Date).getTime()};H.pw=Ba;var X=function(a,b){for(var c=b.split("."),d=function(){var b=arguments;a(function(){for(var a=g,d=0,e=c.length-1;d<e;++d)a=a[c[d]];a[c[d]].apply(a,b)})},f=g,e=0,h=c.length-1;e<h;++e)f=
f[c[e]]=f[c[e]]||{};return f[c[e]]=d};X(V,"pw.clk");X(V,"pw.hvr");p("su",ya,g.pw)};function Ca(){function a(){for(var b;(b=e[h++])&&"m"!=b[0]&&!b[1].auto;);b&&(M(2,b[0]),b[1].url&&L(b[1].url,b[0]),b[1].libs&&m.d&&m.d(b[1].libs));h<e.length&&setTimeout(a,0)}function b(){0<f--?setTimeout(b,0):a()}var c=l.a("1"),d=l.a(""),f=3,e=I,h=0,k=window.gbarOnReady;if(k)try{k()}catch(r){da(r,"ml","or")}d?p("ldb",a):c?q(window,"load",b):b()}p("rdl",Ca);var Da={D:1,H:2,da:3,p:4,W:5,M:6,F:7,g:8,ha:9,U:10,L:11,T:12,S:13,N:14,Q:15,P:16,fa:17,w:18,O:19,ga:20,ea:21,u:22,G:23,ja:24,ka:25,ia:26,A:27,j:28,o:29,k:30,ca:31,Z:32,$:33,J:34,K:35,ba:36,aa:37,Y:38,B:39,R:40,v:41,X:42,V:43,h:48,C:49,I:500},Y=[1,2,3,4,5,6,9,10,11,13,14,28,29,30,34,35,37,38,39,40,41,42,43,48,49,500];var Z=l.b("0.001",1E-4),Ea=l.b("1",1),Fa=!1,Ga=!1;if(l.a("1")){var Ha=Math.random();Ha<=Z&&(Fa=!0);Ha<=Ea&&(Ga=!0)}var Ia=Da,$=null;function Ja(){var a=0,b=function(b,d){l.a(d)&&(a|=b)};b(1,"");b(2,"");b(4,"");b(8,"");return a}
function Ka(a,b){var c=Z,d=Fa,f;f=a;if(!$){$={};for(var e=0;e<Y.length;e++){var h=Y[e];$[h]=!0}}if(f=!!$[f])c=Ea,d=Ga;if(d){d=encodeURIComponent;g.rp?(f=g.rp(),f="-1"!=f?f:"1"):f="1";c=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&oge=",a,"&ogex=",d("17483"),"&ogev=",d("EKjtU8f_IIynsAT7zoD4Cg"),"&ogf=",g.bv.f,"&ogp=",d("22"),"&ogrp=",d(f),"&ogsr=",Math.round(1/c),"&ogv=",d("1407723702.0"),"&oggv="+
d("es_plusone_gc_20140723.0_p0"),"&ogd=",d("com"),"&ogl=",d("en"),"&ogc=",d("NOR"),"&ogus=",Ja()];if(b){"ogw"in b&&(c.push("&ogw="+b.ogw),delete b.ogw);var k;f=b;e=[];for(k in f)0!=e.length&&e.push(","),e.push(La(k)),e.push("."),e.push(La(f[k]));k=e.join("");""!=k&&(c.push("&ogad="),c.push(d(k)))}F(c.join(""))}}function La(a){"number"==typeof a&&(a+="");return"string"==typeof a?a.replace(".","%2E").replace(",","%2C"):a}C=Ka;p("il",C,G);var Ma={};H.il=Ma;setTimeout(function(){C(Ia.g)},0);}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var b=window.gbar.i.i;var c=window.gbar;var f=function(d){try{var a=document.getElementById("gbom");a&&d.appendChild(a.cloneNode(!0))}catch(e){b(e,"omas","aomc")}};c.aomc=f;}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var a=window.gbar;a.mcf("pm",{p:""});}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{window.gbar.rdl();}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
if (window['_OC_timingAction']) {window['_OC_timingAction']('patents_refpage');}</script><style>#gbar,#guser{font-size:13px;padding-top:1px !important;}#gbar{float:left;height:22px}#guser{padding-bottom:7px !important;text-align:right}.gbh,.gbd{border-top:1px solid #c9d7f1;font-size:1px}.gbh{height:0;position:absolute;top:24px;width:100%}#gbs,.gbm{background:#fff;left:0;position:absolute;text-align:left;visibility:hidden;z-index:1000}.gbm{border:1px solid;border-color:#c9d7f1 #36c #36c #a2bae7;z-index:1001}.gb1{margin-right:.5em}.gb1,.gb3{zoom:1}.gb2{display:block;padding:.2em .5em}.gb2,.gb3{text-decoration:none !important;border-bottom:none}a.gb1,a.gb4{text-decoration:underline !important}a.gb1,a.gb2,a.gb3,a.gb4{color:#00c !important}.gbi .gb3,.gbi .gb2,.gbi .gb4{color:#dd8e27 !important}.gbf .gb3,.gbf .gb2,.gbf .gb4{color:#900 !important}a.gb2:hover{background:#36c;color:#fff !important}#gbar .gbz0l{color:#000 !important;cursor:default;font-weight:bold;text-decoration:none !important}
#gbar { padding:.3em .6em !important;}</style></head><body  topmargin="3" marginheight="3"><div id=gbar><nobr><a onclick=gbar.qs(this);gbar.logger.il(1,{t:1}); class=gb1 id=gb_1 href="https://www.google.com/search?sa=N&tab=tw">Search</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:2}); class=gb1 id=gb_2 href="http://www.google.com/search?hl=en&tbm=isch&source=og&sa=N&tab=ti">Images</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:8}); class=gb1 id=gb_8 href="http://maps.google.com/maps?hl=en&sa=N&tab=tl">Maps</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:78}); class=gb1 id=gb_78 href="https://play.google.com/?hl=en&sa=N&tab=t8">Play</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:36}); class=gb1 id=gb_36 href="http://www.youtube.com/results?sa=N&tab=t1">YouTube</a> <a onclick=gbar.logger.il(1,{t:5}); class=gb1 id=gb_5 href="http://news.google.com/nwshp?hl=en&tab=tn">News</a> <a onclick=gbar.logger.il(1,{t:23}); class=gb1 id=gb_23 href="https://mail.google.com/mail/?tab=tm">Gmail</a> <a onclick=gbar.logger.il(1,{t:25}); class=gb1 id=gb_25 href="https://drive.google.com/?tab=to">Drive</a> <a class=gb3 href="http://www.google.com/intl/en/options/" onclick="this.blur();gbar.tg(event);return !1" aria-haspopup=true><u>More</u> <small>&#9660;</small></a><div class=gbm id=gbi><a onclick=gbar.logger.il(1,{t:24}); class=gb2 id=gb_24 href="https://www.google.com/calendar?tab=tc">Calendar</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:51}); class=gb2 id=gb_51 href="http://translate.google.com/?hl=en&sa=N&tab=tT">Translate</a><a onclick=gbar.logger.il(1,{t:17}); class=gb2 id=gb_17 href="http://www.google.com/mobile/?hl=en&tab=tD">Mobile</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:10}); class=gb2 id=gb_10 href="http://www.google.com/search?hl=en&tbo=u&tbm=bks&source=og&sa=N&tab=tp">Books</a><a onclick=gbar.logger.il(1,{t:212}); class=gb2 id=gb_212 href="https://wallet.google.com/manage/?tab=ta">Wallet</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:6}); class=gb2 id=gb_6 href="http://www.google.com/search?hl=en&tbo=u&tbm=shop&source=og&sa=N&tab=tf">Shopping</a><a onclick=gbar.logger.il(1,{t:30}); class=gb2 id=gb_30 href="http://www.blogger.com/?tab=tj">Blogger</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:27}); class=gb2 id=gb_27 href="http://www.google.com/finance?sa=N&tab=te">Finance</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:31}); class=gb2 id=gb_31 href="https://plus.google.com/photos?sa=N&tab=tq">Photos</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:12}); class=gb2 id=gb_12 href="http://www.google.com/search?hl=en&tbo=u&tbm=vid&source=og&sa=N&tab=tv">Videos</a><div class=gb2><div class=gbd></div></div><a onclick=gbar.logger.il(1,{t:66}); href="http://www.google.com/intl/en/options/" class=gb2>Even more &raquo;</a></div></nobr></div><div id=guser width=100%><nobr><span id=gbn class=gbi></span><span id=gbf class=gbf></span><span id=gbe></span><a target=_top id=gb_70 href="https://www.google.com/accounts/Login?service=&continue=http://www.google.com/patents%3Fhl%3Den&hl=en" class=gb4>Sign in</a><div style="display: none"><div class=gbm id=gbd5 aria-owner=gbg5><div class=gbmc><ol id=gbom class=gbmcc></ol></div></div></div></nobr></div><div class=gbh style=left:0></div><div class=gbh style=right:0></div><div role="alert" style="position: absolute; left: 0; right: 0;"><a href="http://www.google.com/patents/us5792640?hl=en&amp;output=html_text" title="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."><img border="0" src="http://www.google.com/images/cleardot.gif"alt="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."></a></div><div id="guser"><nobr></nobr></div><div style="clear:both;"></div><div id="gb-top-search-box" class="gb-top-search-box-small gb-reset"><table><tr><td class="logo"><a href="http://www.google.com/patents" class="logo-link"><img class="logo-img" src="/intl/en/images/logos/google_logo_41.png" alt="Go to Google Books Home" height="41"/></a></td><td><form action="http://www.google.com/search" name="f" id="vheadf" method="get"><span id="hf"></span><input type="hidden" name="tbm" value="pts"/><input type="hidden" name="tbo" value="1"/><input type="hidden" name="hl" value="en"/><table><tr><td><div class="inputs"><table><tr><td><div class="text-input"><input type="text" name="q" id="vheadq" class="text" maxlength="2048" size="31" value="" title="Search Patents" accesskey="s" autocomplete="off"/><script>window._OC_autoDir &&window._OC_autoDir('vheadq', 'tia-vheadq');</script></div></td><td><div class="submit-input"><input name="btnG" class="submit" type="submit" value=""/></div></td></tr></table></div></td><td class="col-ext-links"><div class="ext-links"><a href="http://www.google.com/advanced_patent_search">&lt;nobr&gt;Advanced Patent Search&lt;/nobr&gt;</a></div></td></tr></table></form></td></tr></table></div><div class="kd-appbar"><h2 class="kd-appname"><a href="/patents">Patents</a></h2><div class="kd-buttonbar left" id="left-toolbar-buttons"><a id="appbar-write-review-link" href=""></a><a id="appbar-view-print-sample-link" href=""></a><a id="appbar-view-ebook-sample-link" href=""></a><a id="appbar-patents-prior-art-finder-link" href="https://www.google.com/patents/related/US5792640"></a><a id="appbar-patents-discuss-this-link" href="http://www.google.com/url?id=LxFHBAABERAJ&amp;q=http://patents.stackexchange.com/redirect/google-patents%3Fpatent%3DUS5792640&amp;usg=AFQjCNEqdT_DUItdFoWW0cbZL21Kio1XBQ" data-is-grant="true"></a><a id="appbar-read-patent-link" href="//docs.google.com/viewer?url=patentimages.storage.googleapis.com/pdfs/US5792640.pdf"></a><a id="appbar-download-pdf-link" href="//patentimages.storage.googleapis.com/pdfs/US5792640.pdf"></a></div><div class="kd-buttonbar right" id="right-toolbar-buttons"></div></div><div id="books-microdata" itemscope=""itemtype="http://schema.org/Book"itemid="http://www.google.com/patents/US5792640" style="display:none"><span itemprop="description">The present invention reveals methods for cloning hybrid restriction endonucleases and for enzymatically inactivating a target DNA. The method for cloning hybrid restriction endonucleases involves co-expression of a ligase. A first plasmid contains a gene encoding a DNA ligase. A second plasmid contains...</span><span itemprop="url">http://www.google.com/patents/US5792640?utm_source=gb-gplus-share</span><span class="main-title" itemprop="name">Patent US5792640 - General method to clone hybrid restriction endonucleases using lig gene</span><img itemprop="image" src="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"alt="Patent US5792640 - General method to clone hybrid restriction endonucleases using lig gene" title="Patent US5792640 - General method to clone hybrid restriction endonucleases using lig gene"></div><div style="display: none"><ol id="ofe-gear-menu-contents" class="gbmcc"><li class="gbe gbmtc"><a class="gbmt goog-menuitem-content" id="" href="http://www.google.com/advanced_patent_search">Advanced Patent Search</a></li></ol></div><table id="viewport_table" cellpadding="0" style="clear:both" cellspacing="0"><tr><td id="viewport_td"><div class=vertical_module_list_row><div id=intl_patents class=about_content><div id=intl_patents_v><table class="patent-bibdata"><tr><td class="patent-bibdata-heading">Publication number</td><td class="single-patent-bibdata">US5792640 A</td></tr><tr><td class="patent-bibdata-heading">Publication type</td><td class="single-patent-bibdata">Grant</td></tr><tr><td class="patent-bibdata-heading">Application number</td><td class="single-patent-bibdata">US 08/575,361</td></tr><tr><td class="patent-bibdata-heading">Publication date</td><td class="single-patent-bibdata">Aug 11, 1998</td></tr><tr><td class="patent-bibdata-heading">Filing date</td><td class="single-patent-bibdata">Dec 20, 1995</td></tr><tr><td class="patent-bibdata-heading">Priority date<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The priority date is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the date listed."></span></td><td class="single-patent-bibdata">Apr 3, 1992</td></tr><tr><td class="patent-bibdata-heading">Fee status<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The fee status is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the status or dates listed."></span></td><td class="single-patent-bibdata">Paid</td></tr><tr class="patent-bibdata-list-row alternate-patent-number"><td class="patent-bibdata-heading">Publication number</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value">08575361, </span><span class="patent-bibdata-value">575361, </span><span class="patent-bibdata-value">US 5792640 A, </span><span class="patent-bibdata-value">US 5792640A, </span><span class="patent-bibdata-value">US-A-5792640, </span><span class="patent-bibdata-value">US5792640 A, </span><span class="patent-bibdata-value">US5792640A</span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Inventors</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Srinivasan+Chandrasegaran%22">Srinivasan Chandrasegaran</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Original Assignee</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=inassignee:%22The+Johns+Hopkins+University%22">The Johns Hopkins University</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Export Citation</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/US5792640.bibtex">BiBTeX</a>, </span><span class="patent-bibdata-value"><a href="/patents/US5792640.enw">EndNote</a>, </span><span class="patent-bibdata-value"><a href="/patents/US5792640.ris">RefMan</a></span></span></td></tr><tr class="patent-internal-links"><td colspan=2><span class="patent-bibdata-value"><a href="#backward-citations">Patent Citations</a> (1),</span> <span class="patent-bibdata-value"><a href="#npl-citations">Non-Patent Citations</a> (22),</span> <span class="patent-bibdata-value"><a href="#forward-citations">Referenced by</a> (88),</span> <span class="patent-bibdata-value"><a href="#classifications">Classifications</a> (11),</span> <span class="patent-bibdata-value"><a href="#legal-events">Legal Events</a> (15)</span> </td></tr><tr><td colspan=2 class="patent-bibdata-external-link-spacer-top"></td></tr><tr class="patent-bibdata-external-link-spacer-bottom"></tr><tr><td colspan=2><span class="patent-bibdata-heading">External Links:&nbsp;</span><span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=LxFHBAABERAJ&q=http://patft.uspto.gov/netacgi/nph-Parser%3FSect2%3DPTO1%26Sect2%3DHITOFF%26p%3D1%26u%3D/netahtml/PTO/search-bool.html%26r%3D1%26f%3DG%26l%3D50%26d%3DPALL%26RefSrch%3Dyes%26Query%3DPN/5792640&usg=AFQjCNE-BBcKH7uUN3MMIVlJVQ20IbgtWA">USPTO</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=LxFHBAABERAJ&q=http://assignments.uspto.gov/assignments/q%3Fdb%3Dpat%26pat%3D5792640&usg=AFQjCNEbAnrHjgQgJelP0KALKYS-bWPvZA">USPTO Assignment</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=LxFHBAABERAJ&q=http://worldwide.espacenet.com/publicationDetails/biblio%3FCC%3DUS%26NR%3D5792640A%26KC%3DA%26FT%3DD&usg=AFQjCNH05T-ZZvojTolmlIHbuDchWhziRg">Espacenet</a></span></span></td></tr><tr class="patent-bibdata-group-spacer"></tr></table><div class="number-and-title"><span class="patent-title"><invention-title mxw-id="PT54313461" lang="EN" load-source="patent-office">General method to clone hybrid restriction endonucleases using lig gene</invention-title></span><br><span class="patent-number">US 5792640 A</span></div><div class="patent-section patent-abstract-section"><div class="patent-section-header"><span class="patent-section-title">Abstract</span></div><div class="patent-text"><abstract mxw-id="PA37799582" lang="EN" load-source="patent-office"> <div class="abstract">The present invention reveals methods for cloning hybrid restriction endonucleases and for enzymatically inactivating a target DNA. The method for cloning hybrid restriction endonucleases involves co-expression of a ligase. A first plasmid contains a gene encoding a DNA ligase. A second plasmid contains a gene encoding a hybrid restriction endonuclease and is compatible with the first plasmid. The method involves transfecting host cells with the first plasmid, so that DNA ligase is produced, followed by transfecting the cells with the second plasmid. The method for enzymatically inactivating a target DNA involves preparing a plasmid, phage, virus or any other delivery vehicle such as a liposome containing a gene encoding a nuclease, delivering the gene into cells, inducing the cells to produce the nuclease and enzymatically inactivating the target DNA.</div>
  </abstract></div></div><div class="patent-section patent-drawings-section"><div class="patent-section-header"><span class="patent-section-title">Images<span class="patent-section-count">(18)</span></span></div><div class="patent-drawings-body"><div class="patent-drawings-carousel"><div class="drawings"><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5792640-1.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5792640-1.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5792640-2.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5792640-2.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5792640-3.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5792640-3.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5792640-4.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5792640-4.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5792640-5.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5792640-5.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5792640-6.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5792640-6.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5792640-7.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5792640-7.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5792640-8.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5792640-8.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5792640-9.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5792640-9.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5792640-10.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5792640-10.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5792640-11.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5792640-11.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5792640-12.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5792640-12.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5792640-13.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5792640-13.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5792640-14.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5792640-14.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5792640-15.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5792640-15.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5792640-16.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5792640-16.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5792640-17.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5792640-17.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5792640-18.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5792640-18.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div></div></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img" alt="Previous page"src="/googlebooks/images/kennedy/page_left.png"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img" alt="Next page"src="/googlebooks/images/kennedy/page_right.png"width="21" height="21" /></div></div></div><div class="patent-post-drawings"></div><div class="patent-section patent-claims-section"><div class="patent-section-header"><span class="patent-section-title">Claims<span class="patent-section-count">(32)</span></span></div><div class="patent-text"><div mxw-id="PCLM59486807" lang="EN" load-source="patent-office" class="claims">
    <claim-statement>What is claimed is:</claim-statement> <div class="claim"> <div num="1" class="claim">
      <div class="claim-text">1. A method for producing a nuclease in a cell wherein<div class="claim-text">said nuclease, when produced in a cell comprising a target nucleic acid which comprises a target nucleotide sequence specifically binds to said target nucleotide sequence and cleaves said target nucleic acid specifically bound to said nuclease,</div> <div class="claim-text">said method comprising the steps of:</div> <div class="claim-text">a) providing a preparation of a first polynucleotide encoding said nuclease, wherein said preparation does not comprise a second polynucleotide encoding an enzyme which protects said target nucleic acid from said nuclease by specifically binding to said target nucleotide sequence and enzymatically modifying said target nucleic acid such that said target nucleic acid is not cleaved by said nuclease; and</div> <div class="claim-text">b) delivering said preparation of said first polynucleotide encoding said nuclease into said cell under conditions such that said first polynucleotide expresses said nuclease, thereby producing said nuclease.</div> </div>
    </div>
    </div> <div class="claim-dependent"> <div num="2" class="claim">
      <div class="claim-text">2. The method of claim 1, wherein said polynucleotide encoding said nuclease is delivered into said cell by liposomes.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="3" class="claim">
      <div class="claim-text">3. The method of claim 1, wherein said delivering step further comprises integrating said first polynucleotide encoding said nuclease into a chromosome of said cell.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="4" class="claim">
      <div class="claim-text">4. The method of claim 1, wherein said polynucleotide encoding said nuclease further comprises control elements.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="5" class="claim">
      <div class="claim-text">5. The method of claim 1, wherein said cell is a prokaryotic cell.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="6" class="claim">
      <div class="claim-text">6. The method of claim 5, wherein said cell is an E. coli cell.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="7" class="claim">
      <div class="claim-text">7. The method of claim 1, wherein said cell is a eukaryotic cell.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="8" class="claim">
      <div class="claim-text">8. The method of claim 7, wherein said cell is a plant cell.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="9" class="claim">
      <div class="claim-text">9. The method of claim 7, wherein said cell is a mammalian cell.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="10" class="claim">
      <div class="claim-text">10. The method of claim 1, wherein said nuclease is a naturally occurring restriction endonuclease.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="11" class="claim">
      <div class="claim-text">11. The method of claim 1, wherein said nuclease is a non-naturally occurring enzyme.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="12" class="claim">
      <div class="claim-text">12. The method of claim 11, wherein said nuclease comprises a recognition domain which specifically binds to said target sequence and a separate catalytic domain which cleaves nucleotide sequences non-specifically and is obtained from a nuclease not comprising said recognition domain.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="13" class="claim">
      <div class="claim-text">13. A method according to claim 12 wherein said catalytic domain is obtained from the FokI restriction endonuclease.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="14" class="claim">
      <div class="claim-text">14. A method according to claim 12 wherein said recognition domain comprises a zinc finger domain.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="15" class="claim">
      <div class="claim-text">15. The method of claim 14, wherein said nuclease is selected from the group consisting of ZF-QDR-F<sub>N</sub>, ZF-Sp1C-F<sub>N</sub>, ZF-QNR-F<sub>N</sub>, ZF-QQR-F<sub>N</sub> and ZFHD1-F<sub>N</sub>.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="16" class="claim">
      <div class="claim-text">16. A method of claim 1 wherein said cell in which said nuclease is to be produced comprises said target nucleic acid and said target nucleic acid is to be specifically inactivated,<div class="claim-text">wherein further said nuclease produced in said cell specifically inactivates said target nucleic acid by specifically binding to said target nucleotide sequence and cleaving said target nucleic acid bound to said nuclease.</div> </div>
    </div>
    </div> <div class="claim-dependent"> <div num="17" class="claim">
      <div class="claim-text">17. The method of claim 16, wherein said target nucleic acid is exogenous to said cell.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="18" class="claim">
      <div class="claim-text">18. The method of claim 17, wherein said target nucleic acid is a self-replicating DNA, linear or circular.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="19" class="claim">
      <div class="claim-text">19. The method of claim 17, wherein said target nucleic acid is a replication intermediate of an RNA tumor virus.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="20" class="claim">
      <div class="claim-text">20. The method of claim 16, wherein said target nucleic acid is a DNA endogenous to said cell.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="21" class="claim">
      <div class="claim-text">21. The method of claim 20, wherein said target nucleic acid is chromosomal DNA of said cell.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="22" class="claim">
      <div class="claim-text">22. A method according to claim 1 wherein said target nucleic acid is a DNA:RNA hybrid.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="23" class="claim">
      <div class="claim-text">23. A method according to claim 1 wherein said target nucleotide sequence comprises a sequence of more than 6 nucleotides.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="24" class="claim">
      <div class="claim-text">24. A method according to claim 23 wherein said target nucleotide sequence comprises a sequence of more than 8 nucleotides.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="25" class="claim">
      <div class="claim-text">25. A method of claim 1 wherein<div class="claim-text">said cell in which said nuclease is to be produced is to be cloned, and</div> <div class="claim-text">said cell comprises said target nucleic acid and said target nucleic acid is required for cloning of said cell,</div> <div class="claim-text">said method further comprising the steps of:</div> <div class="claim-text">c) providing a third polynucleotide encoding a DNA ligase; and</div> <div class="claim-text">d) delivering said third polynucleotide to said cell before or concurrently with said step of delivering said preparation of said first polynucleotide into said cell, under conditions such that said DNA ligase is produced whereby said target nucleic acid is protected by said DNA ligase from inactivation by said nuclease produced in said cell; and</div> <div class="claim-text">e) cloning said cell producing said nuclease and said ligase.</div> </div>
    </div>
    </div> <div class="claim-dependent"> <div num="26" class="claim">
      <div class="claim-text">26. The method of claim 25, wherein said cell is a prokaryotic cell.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="27" class="claim">
      <div class="claim-text">27. The method of claim 26, wherein said cell is an E. coli cell.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="28" class="claim">
      <div class="claim-text">28. The method of claim 25, wherein said is a eukaryotic cell.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="29" class="claim">
      <div class="claim-text">29. The method of claim 28, wherein said cell is a plant cell.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="30" class="claim">
      <div class="claim-text">30. The method of claim 28, wherein said cell is a mammalian cell.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="31" class="claim">
      <div class="claim-text">31. The method of claim 25, wherein said cell is a mutant or engineered strain that produces an increased level of DNA ligase.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="32" class="claim">
      <div class="claim-text">32. The method of claim 25, wherein said nuclease is selected from the group consisting of ZF-QDR-F<sub>N</sub>, ZF-SP1C-F<sub>N</sub>, ZF-QNR-F<sub>N</sub>, ZF-QQR-F<sub>N</sub> and ZFHD1-F<sub>N</sub>.</div>
    </div>
  </div> </div></div></div><div class="patent-section patent-description-section"><div class="patent-section-header"><span class="patent-section-title">Description</span></div><div class="patent-text"><div mxw-id="PDES67158268" lang="EN" load-source="patent-office" class="description">
    <p>This patent application was supported in part by grant GM 42140 from the National Institutes of Health and by grant MCB-9415861 from the National Science Foundation. The Government has certain rights in the invention.</p>
    <p>This application is a continuation-in-part of U.S. application Ser. No. 08/346,293, filed Nov. 23, 1994, issued as U.S. Pat. No. 5,487,994, which is a continuation-in-part of Ser. No. 08/126,564, filed Sep. 27, 1993, issued as U.S. Pat. No. 5,436,150, Jul. 25, 1995, which is a continuation-in-part of U.S. application Ser. No. 08/017,493, filed Feb. 12, 1993, abandoned, which is a continuation-in-part of U.S. application Ser. No. 07/862,831, Apr. 3, 1992, issued as U.S. Pat. No. 5,356,802.</p>
    <heading>BACKGROUND OF THE INVENTION</heading> <p>1. Field of the Invention</p>
    <p>The present invention relates to hybrid genes which encode hybrid restriction endonucleases. The hybrid restriction endonucleases are designed to recognize DNA at given base sites and to enzymatically cleave the DNA at distant sites.</p>
    <p>More specifically, the present invention relates to a method for cloning hybrid restriction endonucleases and to a method for enzymatically inactivating a target DNA.</p>
    <p>2. Description of the Related Art</p>
    <p>Since their discovery nearly 25 years ago (1), Type II restriction enzymes have played a crucial role in the development of the recombinant DNA technology and the field of molecular biology. The Type II restriction (R) endonucleases and modification (M) methylases are relatively simple bacterial enzymes that recognize specific sequences in duplex DNA. While the former cleave DNA, the latter methylate adenine or cytosine residues within the recognition site so as to protect the host-genome against cleavage by the former. So far, over 2500 restriction and modification enzymes have been identified and these are found in widely diverse organisms (2). These enzymes fall into numerous "isoschizomer" (identically cleaving) groups with about 200 sequence-specificities.</p>
    <p>Discovery of new enzymes involves tedious and time-consuming effort that requires extensive screening of bacteria and other microorganisms (3). Even when one finds a new enzyme, more often than not, it falls into the already-discovered isoschizomer groups. Furthermore, most naturally occurring restriction enzymes recognize sequences that are 4-6 bp long. Although these enzymes are very useful in manipulating recombinant DNA, they are not suitable for producing large DNA segments. For example, restriction enzymes that recognize DNA sequences 6 bp long, result in cuts as often as every 4096 bases. In many instances, it is preferable to have fewer but longer DNA strands, especially during genome mapping. Rare cutters like NotI, that recognizes 8 bp-long sequences, cut human DNA (which contains about 3 billion bp) every 65536 bases on average. So far, only a few endonucleases with recognition sequences longer than 6 bp (rare cutters) have been identified (New England Biolabs catalog).</p>
    <p>R-M (restriction-modification) systems appear to have a single biological function--namely, to protect cells from infection by foreign DNA that would otherwise destroy them. The phage genomes are usually small. It stands to reason, then, that bacteria select for R-M systems with small recognition sites (4-6 bp) because these sites occur more frequently in the phages. Therefore, a long term goal in the field of restriction-modification enzymes has been to generate restriction endonucleases with longer recognition sites by mutating or engineering existing enzymes (3).</p>
    <p>The FokI restriction endonuclease from Flavobacterium okeanokoites belongs to the Type IIS class of endonucleases. FokI recognizes the asymmetric sequence 5'-GGATG-3' and cleaves double-stranded DNA at staggered sites 9 and 13 nucleotides away from the recognition site. The cloning and sequencing of the FokI restriction-modification system have been reported. Several research groups have purified FokI endonuclease and characterized its properties. Previous reports by the present inventor on proteolytic fragments of FokI endonuclease using trypsin have revealed an N-terminal DNA-binding domain and a C-terminal catalytic domain with non-specific DNA cleavage activity (4-7). These reports have suggested that the two domains are connected by a linker region which is susceptible to cleavage by trypsin. The present inventor has also shown that insertion of four (or seven) codons between the recognition and cleavage domains of FokI can alter the cleavage distance of FokI within its substrate.</p>
    <p>Recently, Waugh and Sauer have shown that single amino acid substitutions uncouple the DNA-binding and strand scission activities of FokI endonuclease (28). Furthermore, they have obtained a novel class of FokI restriction mutants that cleave hemi-methylated DNA substrates (29). The modular structure of FokI suggested that it may be feasible to construct hybrid endonucleases with novel sequence-specificity by linking other DNA-binding proteins to the cleavage domain of FokI endonuclease. Recently, the present inventor reported the construction of the first "chimeric" restriction endonuclease by linking the Ubx homeo domain to the cleavage domain of FokI (8).</p>
    <p>To further probe the linker region, the present inventor constructed several insertion and deletion mutants of FokI endonuclease. A detailed description of the process for making and using and the properties of these mutants are disclosed in U.S. patent application Ser. No. 08/346,293, allowed, the entire contents of which are hereby incorporated by reference and relied upon.</p>
    <p>Unlike the Ubx homeo domain, zinc finger proteins, because of their modular structure, offer a better framework for designing chimeric restriction enzymes with tailor-made sequence-specificities. The Cys<sub>2</sub> His<sub>2</sub> zinc finger proteins are a class of DNA-binding proteins that contain sequences of the form (Tyr,Phe)-Xaa-Cys-Xaa<sub>2-4</sub> -Cys-Xaa<sub>3</sub> -Phe-Xaa<sub>5</sub> -Leu-Xaa<sub>2</sub> -His-Xaa<sub>3</sub> -<sub>5</sub> -His (SEQ ID NO:1-18) usually in tandem arrays (9). Each of these sequences binds a zinc(ii) ion to form the structural domain termed a zinc finger. These proteins, like many sequence-specific DNA-binding proteins, bind to the DNA by inserting an -helix into the major groove of the double helix (10).</p>
    <p>The crystallographic structure of the three zinc finger domain of zif268 bound to a cognate oligonucleotide reveals that each finger interacts with a triplet within the DNA substrate. Each finger, because of variations of certain key amino acids from one zinc finger to the next, makes its own unique contribution to DNA-binding affinity and specificity.</p>
    <p>The zinc fingers, because they appear to bind as independent modules, can be linked together in a peptide designed to bind a predetermined DNA site. Although, more recent studies suggest that the zinc finger--DNA recognition is more complex than originally perceived (11,12), it still appears that zinc finger motifs will provide an excellent framework for designing DNA-binding proteins with a variety of new sequence-specificities.</p>
    <p>In theory, one can design a zinc finger for each of the 64 possible triplet codons and using a combination of these fingers, one could design a protein for sequence-specific recognition of any segment of DNA. Studies to understand the rules relating to zinc finger sequences/DNA-binding preferences and redesigning of DNA-binding specificities of zinc finger proteins are well underway (13-15).</p>
    <p>An alternative approach to the design of zinc finger proteins with new specificities involves the selection of desirable mutants from a library of randomized fingers displayed on phage (16-20). The ability to design or select zinc fingers with desired specificity implies that DNA-binding proteins containing zinc fingers will be made to order. Therefore, we reasoned that one could design "artificial" nucleases that will cut DNA at any preferred site by making fusions of zinc finger proteins to the cleavage domain of FokI endonuclease. We thus undertook the deliberate creation of zinc finger hybrid restriction enzymes, the cloning of the hybrid enzymes and the characterization of their DNA cleavage properties.</p>
    <p>One of the main difficulties in cloning or overproducing restriction enzymes is their potential lethality. The restriction enzymes can enzymatically attack and destroy the host DNA. This is circumvented by first cloning a methylase gene (M). The methylase gene modifies the restriction enzyme sites and provides protection against chromosomal cleavage. A restriction endonuclease gene (R) is then introduced into the host on a separate compatible plasmid.</p>
    <p>Our work on hybrid restriction endonuclease genes has indicated that they are likewise lethal, since there are no corresponding methylase genes available to protect the host genome from cleavage by the hybrid endonuclease. We now report on a method for cloning the genes for hybrid restriction endonucleases and on a method for using nucleases to enzymatically destroy a target DNA. Furthermore, the method for cloning can be used to clone either mutant or wild type restriction endonucleases.</p>
    <heading>SUMMARY OF THE INVENTION</heading> <p>The present invention reveals methods for cloning restriction endonucleases by the co-expression of ligase and for inducing nucleases to enzymatically inactivate a target DNA, thereby destroying the target DNA.</p>
    <p>The method for cloning hybrid restriction endonucleases requires:</p>
    <p>a) preparing a first plasmid containing a gene encoding a DNA ligase;</p>
    <p>b) transfecting cells with the first plasmid so that DNA ligase is produced;</p>
    <p>c) preparing a second compatible plasmid containing a gene encoding a hybrid restriction endonuclease;</p>
    <p>d) transfecting the cells with the second plasmid; and</p>
    <p>e) cloning the cells.</p>
    <p>Both prokaryotic cells, e.g., E. coli cells, and eukaryotic cells, e.g., plant cells and mammalian cells, can be used in this method. Furthermore, mutant strains of cells that produce increased levels of DNA ligase can be used in this method.</p>
    <p>Examples of genes encoding a hybrid restriction endonuclease that can be used in the method include ZF-QDR-F<sub>N</sub>, ZF-Sp1C-F<sub>N</sub>, ZF-QNR-F<sub>N</sub>, ZF-QQR-F<sub>N</sub> and ZFHD1-F<sub>N</sub>.</p>
    <p>The method for enzymatically inactivating a target DNA requires:</p>
    <p>a) preparing a plasmid, virus, phage or any other delivery vehicle such as a liposome containing a gene encoding a nuclease, wherein the nuclease specifically recognizes and enzymatically inactivates the target DNA;</p>
    <p>b) delivering the plasmid, virus, phage or any other delivery vehicle such as a liposome containing the gene encoding a nuclease into cells;</p>
    <p>c) inducing the cells to produce the nuclease; and</p>
    <p>d) enzymatically inactivating the target DNA.</p>
    <p>Both prokaryotic cells, e.g., E. coli cells, and eukaryotic cells, e.g., plant cells and mammalian cells, can be used in this method. Furthermore, either naturally occurring endonucleases or engineered hybrid nucleases can be used in this method. The use of a hybrid restriction endonuclease is preferred. Examples of genes encoding a hybrid restriction endonuclease that can be used in the method include ZF-QDR-F<sub>N</sub>, ZF-Sp1C-F<sub>N</sub>, ZF-QNR-F<sub>N</sub>, ZF-QQR-F<sub>N</sub> and ZFHD1-F<sub>N</sub>.</p>
    <p>Furthermore, genes encoding the hybrid restriction nucleases with appropriate control elements (e.g., viral promoters) can be integrated into the chromosome of cells using appropriate plasmid, virus, phage or any other delivery vehicle such as a liposome containing a gene encoding a nuclease. Thus, a delivery vehicle is defined as any plasmid, virus, phage or any other physical structure such as a liposome which is able to contain a gene. Exposure to the appropriate virus will induce the production of the hybrid restriction endonuclease and stop the propagation of the virus.</p>
    <p>In addition, the target DNA may be DNA exogenous to DNA of the cells or a DNA endogenous to DNA of the cells. The exogenous DNA target may be a self-replicating DNA, linear or circular, or a DNA intermediate of an RNA tumor virus. The endogenous DNA target may be the chromosomal DNA of the cells.</p>
    <heading>BRIEF DESCRIPTION OF THE DRAWINGS</heading> <p>FIGS. 1A and 1B show the construction of expression vectors of ZF-F<sub>N</sub>. FIG. 1A shows the structure of the plasmid pET-15b:ZF-F<sub>N</sub>. FIG. 1B shows the map of the ZF-F<sub>N</sub> gene. The methods to construct the overproducer clones of ZF-F<sub>N</sub> and the protein purification procedures are described in detail herein and elsewhere (26,35). ZF-F<sub>N</sub> is a generic abbreviation for a fusion product of zinc fingers (ZF) and the FokI endonuclease domain (F<sub>N</sub>).</p>
    <p>FIG. 2 shows the purification of ZF-F<sub>N</sub> hybrid enzymes. FIG. 2A shows the SDS/PAGE profiles at each step in the purification of ZF-QNR-F<sub>N</sub> hybrid enzyme. ZF-QNR-F<sub>N</sub> is a specific abbreviation for a fusion product between the eukaryotic transcription factor Sp1 (a specific zinc finger) and the FokI endonuclease domain (F<sub>N</sub>). QNR indicates the sequence specific contact residues, that is, glutamine-asparagine-arginine, for the zinc finger recognition domain of the fusion product. Lane 1 shows protein standards; lane 2 shows crude extract from induced cells; lane 3 shows the results of purification after His-bind resin column chromatography; lane 4 shows the results after SP-sepharose column chromatography; and lane 5 shows the results after gel filtration column chromatography.</p>
    <p>FIG. 2B shows the SDS/PAGE profile of ZF-QDR-F<sub>N</sub> hybrid enzyme. ZF-QDR-F<sub>N</sub> is a specific abbreviation for a fusion product between the designed consensus protein CP (a specific zinc finger) and the FokI endonuclease domain (F<sub>N</sub>). QDR indicates the sequence specific contact residues, that is, glutamine-aspartic acid-arginine, for the zinc finger recognition domain of the fusion product. Lane 1 shows protein standards and lane 2 the purified ZF-QDR-F<sub>N</sub> fusion protein.</p>
    <p>FIG. 2C shows the Western blot profile at each step of purification of ZF-QNR-F<sub>N</sub> hybrid enzyme using antisera raised against FokI endonuclease. Lane 1 shows the crude extract from induced cells; lane 2 shows the results of purification after His-bind resin column chromatography; lane 3 shows the results after SP-sepharose column chromatography; and lane 4 shows the results after gel filtration column chromatography. The arrow indicates the intact fusion protein.</p>
    <p>FIG. 3 shows the cleavage of  DNA (48.5 Kb) substrate by the hybrid enzymes, ZF-F<sub>N</sub>. Lane 1 shows a kb ladder; lane 2 shows  DNA; and lane 3 shows  DNA digested with the hybrid ZF-QDR-F<sub>N</sub>.</p>
    <p>The substrate cleaves into 5.5 kb and 43 kb fragments (open arrows). Lane 4 shows  DNA digested with ZF-QNR-F<sub>N</sub>. The substrate cleaves into 9.5 kb and 39 kb fragments (closed arrows). Lane 5 shows high molecular weight markers from BRL (top to bottom: 48.5, 38.4, 33.5, 29.9, 24.8, 22.6, 19.4, 17.0, 15.0,, 12.2, 10.1, 8.6 and 8.3 kb respectively). Weaker bands result from cleavage at the minor DNA-binding sites. tRNA of the reaction mixture runs outside the region shown in the figure.</p>
    <p>FIG. 4 shows the kinetics of cleavage of the lambda DNA substrate. FIG. 4A shows the cleavage of  DNA substrate by ZF-QDR-F<sub>N</sub>. FIG. 4B shows the cleavage of  DNA substrate by ZF-QNR-F<sub>N</sub>. FIG. 4C shows the cleavage of  DNA substrate by wild-type FokI endonuclease. The reaction conditions were as described for FIG. 3. Aliquots (12 l each) were removed from a 90 l reaction mixture at 0, 15, 30, 60, 120, 240 and 360 minutes respectively. The products were analyzed by agarose gel electrophoresis. The arrows indicate the major cleavage products of the lambda DNA substrate. The lambda DNA was digested with 18 units of wild-type FokI in a volume of 90 l using NEB (New England Biolabs) buffer.</p>
    <p>FIG. 5 shows the effect of reaction conditions on the cleavage activity of the hybrid enzyme, ZF-QNR-F<sub>N</sub>. FIG. 5A shows the effect of temperature; FIG. 5B shows the effect of KCl concentration; and FIG. 5C shows the effect of MgCl<sub>2</sub> concentration on the cleavage activity of the hybrid enzyme, ZF-QNR-F<sub>N</sub>. The reaction conditions were as described for FIG. 3 except for the variables which are shown on the top of the figures. Arrows show the major cleavage products from the lambda DNA substrate.</p>
    <p>FIG. 6 shows an analysis of the distance of cleavage from the recognition site by ZF-F<sub>N</sub> hybrid enzymes. FIGS. 6A and 6B show the cleavage products from the substrates by ZF-QDR-F<sub>N</sub> and ZF-QNR-F<sub>N</sub>, respectively. Lane 1 and 4 show the (G+A) sequencing reaction; lane 2 shows substrates containing <sup>32</sup> P-label on the top strand, 5'-GAG GAG GCT-3' and 5'-GAG GGA TGT-3', respectively; lane 3 shows ZF-F<sub>N</sub> digestion products; lane 5 shows substrates containing <sup>32</sup> P-labeled on the bottom strand, 5'-AGC CTC CTC-3' and 5'-ACA TCC CTC-3', respectively; and lane 6 shows ZF-F<sub>N</sub> digestion products. The location of the DNA-binding sites for the hybrid enzymes are indicated by vertical lines.</p>
    <p>FIGS. 6C and 6D show the map of the major recognition and cleavage site(s) of ZF-QDR-F<sub>N</sub> and ZF-QNR-F<sub>N</sub> hybrid enzymes on DNA (SEQ ID NO:19 and SEQ ID NO:20), respectively. The recognition site is shown by bold type and the site(s) of cleavage are indicated by arrows. The percent cleavage at each location are shown in brackets. ZF-QDR-F<sub>N</sub> is indicated as Zif-QDR-F<sub>N</sub> in FIG. 6C and ZF-QNR-F<sub>N</sub> is indicated as Zif-QNR-F<sub>N</sub> in FIG. 6D. Thus, the designations ZF and Zif are interchangeable.</p>
    <p>FIG. 7 shows the structure of plasmid pACYC lig. The E. coli ligase gene, lig, was inserted into the NcoI site of plasmid pACYC184.</p>
    <p>FIG. 8 shows a comparison of transformation efficiency of two different hybrid endonuclease genes, pET-15b:ZFHD1-F<sub>N</sub> and pET-15b:ZF-QQR-F<sub>N</sub>, in BL21 (DE3) cells, with and without pACYC lig. The top panel shows BL21 (DE3) (pACYC lig) cells and the bottom panel shows BL21 (DE3) cells. Column A shows pET-15ZFHD1-F<sub>N</sub> ; Column B shows pET-15b:ZF-QQR-F<sub>N</sub> ; and Column C shows the control pTZ19R. The plasmid pTZ19R (pTZ) that does not carry a hybrid endonuclease gene was used as standard control to compare the efficiency of transformation of the competent cells. BL21 (DE3) (pACYC lig) transforms at about 5-10 fold lower efficiency as compared to BL21 (DE3) cells.</p>
    <p>ZFHD1-F<sub>N</sub> is a specific abbreviation for a fusion product of the eukaryotic transcription factor Sp1 (a specific zinc finger), the Ubx homeo domain (HD) and the FokI endonuclease domain (F<sub>N</sub>). ZF-QQR-F<sub>N</sub> is a specific abbreviation for a fusion product between the eukaryotic transcription factor Sp1 (a specific zinc finger) and the FokI endonuclease domain (F<sub>N</sub>). QQR indicates the sequence specific contact residues, that is, glutamine-glutamine-arginine, for the zinc finger recognition domain of the fusion product.</p>
    <p>FIG. 9 shows the characterization of pET-15b plasmid containing the ZF-QDR-F<sub>N</sub> hybrid gene from RR1 and BL21 (DE3) cells. The plasmids were digested with NdeI and XhoI to excise the ZF-QDR-F<sub>N</sub> hybrid gene. Only mutants which are 0.9 kb larger than the original construct were obtained from the BL21 (DE3) cells. This is consistent with the jumping in of IS1 element to disrupt the hybrid gene that is lethal to the cells. See, for example, Birkenbihl et al. (37). Lane 1 shows a kb ladder; lane 2 shows a plasmid from RR1 cells; and lane 3 shows a plasmid from BL21 (DE3) cells.</p>
    <p>ZF-QDR-F<sub>N</sub> is a specific abbreviation for a fusion product between the eukaryotic transcription factor Sp1 (a specific zinc finger) and the FokI endonuclease domain (F<sub>N</sub>). QDR indicates the sequence specific contact residues, that is, glutamine-aspartic acid-arginine, for the zinc finger recognition domain of the fusion product.</p>
    <p>FIGS. 10A and 10B show BL21 (DE3) (pACYC lig) cells with plasmids that carry hybrid endonuclease genes grown on LB-Amp-Tet plates (Luria-Bertocni media containing 50 g/ml ampicillin and 20 g/ml tetracycline) with and without isopropyl -D-thiogalactoside (IPTG). FIG. 10A shows BL21 (DE3) (pACYC lig) pET-15b:ZF-QNR-F<sub>N</sub>) cells: Left, -IPTG and Right, +IPTG. FIG. 10B shows BL21 (DE3) (pACYC lig) (pET-15b:ZF-QDR-F<sub>N</sub>) cells: Left, -IPTG and Right, +IPTG.</p>
    <p>ZF-QNR-F<sub>N</sub> is indicated as Zif-QNR-F<sub>N</sub> in FIG. 10A and ZF-QDR-F<sub>N</sub> is indicated as Zif-QDR-F<sub>N</sub> in FIG. 10B.</p>
    <p>FIGS. 11A-11C show appropriate control cells grown on LB-Amp-Tet plates with and without IPTG. Left, -IPTG and Right, +IPTG. FIG. 11A shows BL21 (DE3) (pACYC lig) (pET-15b). In this clone, pET-15b does not have any insert.</p>
    <p>FIG. 11B shows BL21 (DE3) (pACYC lig) (pET-15b:29 kDa). The pET-15b carries a gene encoding a non-lethal 29 kDa fragment cloned downstream of the tac promoter. The protein is inducible by IPTG.</p>
    <p>FIG. 11C shows control RRI(pACYC fokIM) (pRRS fokIR). The growth of the control cells are not inhibited by IPTG. All cells grow well in the absence or presence of IPTG.</p>
    <p>FIGS. 12A-12D show the analyses of the cleavage products of the <sup>32</sup> P-labelled DNA duplexes containing a single binding-site by ZF-F<sub>N</sub> fusions. FIGS. 12A and 12B correspond to ZF-QQR-F<sub>N</sub> and FIGS. 12C and 12D correspond to ZF-Sp1C-F<sub>N</sub>. The <sup>32</sup> P-labelled strand of the DNA duplex is indicated on each figure. Lane 1 shows (G+A) sequencing reactions. Lane 2 shows <sup>32</sup> P-labelled substrates. Lane 3 shows ZF-QQR-F<sub>N</sub> digestion products and lane 4 shows ZF-Sp1C-F<sub>N</sub> digestion products.</p>
    <p>ZF-Sp1C-F<sub>N</sub> is a specific abbreviation for a fusion product between the three zinc finger motif of the eukaryotic transcription factor Sp1 (a specific zinc finger) and the FokI endonuclease domain (F<sub>N</sub>).</p>
    <p>FIG. 13 shows an analysis of the cleavage of a DNA-RNA hybrid by the ZF-QQR-F<sub>N</sub> hybrid enzyme. The <sup>32</sup> P-labelled DNA strand also contains the binding site 5'-GGGGAAGAA-3'. The cleavage products were analyzed by PAGE as described for FIG. 12. Lane 1 shows (G+A) sequencing reactions. Lane 2 shows <sup>32</sup> P-labelled DNA-RNA hybrid. Lane 3 shows ZF-QQR-F<sub>N</sub> digestion products and lane 4 shows ZF-Sp1C-F<sub>N</sub> digestion products.</p>
    <p>FIG. 14 shows the map of the recognition and cleavage site(s) of ZF-QQR-F<sub>N</sub> hybrid enzyme on DNA (SEQ ID NO:21) and on DNA-RNA hybrid (SEQ ID NO:22 and SEQ ID NO:23) and ZF-Sp1C-F<sub>N</sub> hybrid enzyme on DNA (SEQ ID NO:24). The recognition site is shown in bold-faced type and the site(s) of cleavage are indicated by arrows.</p>
    <p>FIG. 15 shows the binding of ZF-QQR-F<sub>N</sub> to DNA duplex and DNA-RNA hybrid. The left and right panels show the results of gel shift assays for DNA-DNA and DNA-RNA duplexes, respectively. The conditions for gel mobility-shift experiments were as described by Shi and Berg (34).</p>
    <heading>DETAILED DESCRIPTION OF THE INVENTION</heading> <p>The present invention discloses a method for cloning hybrid restriction endonucleases and to a method for enzymatically inactivating a target DNA. The latter method involves the use of genes encoding nucleases, including site-specific hybrid restriction endonucleases. The hybrid restriction endonucleases are capable of specifically recognizing, binding to, inactivating and destroying the target DNA. The potential lethality of the hybrid restriction enzymes is initially circumvented by using E. coli DNA lig gene in the former method, i.e., for cloning hybrid restriction endonucleases.</p>
    <p>More specifically, the hybrid endonuclease genes are cloned into a tightly controlled expression system to lessen any deleterious effect to the cell and also to increase the level of DNA ligase by placing the E.coli DNA lig gene on a compatible plasmid. This vector expresses the DNA ligase constitutively. Later, in the method for enzymatically inactivating a target DNA, the cells are induced to produce the hybrid restriction endonucleases and to enzymatically inactivate the target DNA.</p>
    <p>The methods of the present invention is exemplified by the following non-limiting examples.</p>
    <p>EXPERIMENTAL PROCEDURES:</p>
    <p>The complete nucleotide sequence of the FokI R-M (restriction-modification) system has been published (21,22). Experimental protocols for PCR have been described (4). The procedures for cell growth and purification of proteins using His-bind resin (23) was as outlined in the Novagen pET system manual. The protocol for SDS/PAGE was as described by Laemmli (24).</p>
    <p>1. Cell Transformation Assay</p>
    <p>E. coli strain RR1 and E.coli strain BL21 (DE3) were the host in all experiments. E. coli strain RR1, as reported by Studier et al. (26), and E.coli strain BL21 (DE3), as reported by Maniatis et al. (38), were transformed as described in Maniatis et al. (38). Briefly, The cells were grown to 0.2 OD and incubated with 100 mM CaCl<sub>2</sub> for 16 hours at 4 C. to make them competent. These cells were then transfected with DNA as described in Maniatis et al. (38).</p>
    <p>2. Construction of the Clones Producing the Hybrid Enzymes ZF-F<sub>N</sub> Using PCR</p>
    <p>The PCR-generated DNAs using oligos 5'-CCCCTGAAGGAGATATACATATG-3', (SEQ ID NO:25), start primer, and 5'-GGACTAGTCCCTTCTTATTCTGGTG-3', (SEQ ID NO:26), stop primer, were digested with NdeI/SpeI and then ligated into NdeI/SpeI-cleaved pET-15b Ubx-F<sub>N</sub> vector which contains the FokI nuclease (F<sub>N</sub>) domain. This construct replaces the Ubx homeodomain with the genes coding for zinc finger proteins. The ligation mixture was used to transfect competent RR1 (pACYC184:lig) cells. The glycine linker (Gly<sub>4</sub> Ser)<sub>3</sub> (SEQ ID NO:27) was inserted between the zinc finger motifs and the FokI nuclease domain using previously described procedures (16). The zinc finger fusion constructs were confirmed by Sanger's dideoxy sequencing method (25). The pET-15b:ZF-F<sub>N</sub> plasmids were then transferred to BL21 (DE3) that carries the compatible plasmid pACYC184:lig.</p>
    <p>3. Purification of ZF-F<sub>N</sub> Endonucleases</p>
    <p>The procedure for the purification of the zinc finger fusion proteins were as follows: 4 L of cells BL21 (DE3) (pACYC184:lig, pET-15:ZF-F<sub>N</sub>) were grown in LB containing 100 g/ml of ampicillin and 20 g/ml of tetracycline at 37 C. When OD<sub>600</sub> reached 0.4, the growth temperature was shifted to 22 C. The cells were induced at OD<sub>600</sub> =0.5 with 0.7 mM of IPTG. After 4 hrs. of induction at 22 C., the cells were harvested by centrifugation. Induction at 22 C. maximizes the yield of soluble hybrid endonucleases in the crude extracts when compared to induction at 30 C. or 37 C.</p>
    <p>The cells were resuspended in Novagen's 1bind buffer and then disrupted by sonication on ice. After centrifugation at 4 C. for 2 hrs, the crude extract was passed through a 0.45 micron filter and applied to the His-bind affinity column. The column was washed with 1bind buffer (10 vol.) and 1wash buffer (6 vol.) as described in Novagen's manual. In addition, the column was washed with 1wash buffer (4 vol.) containing 100 mM imidazole. The column was eluted with 1elute buffer containing 400 mM imidazole.</p>
    <p>Fractions containing the fusion proteins were identified by probing the immunoblots with rabbit polyclonal antibody against FokI endonuclease. The eluted fractions containing the hybrid proteins were diluted with 3 volumes of buffer A (10 mM Tris base, 15 mM NaH<sub>2</sub> PO<sub>4</sub>.H<sub>2</sub> O, 10% glycerol, 100 M ZnCl<sub>2</sub>, 3 mM DTT, pH 8.0) to reduce salt concentration to 125 mM NaCl and then applied to a SP-sepharose column and eluted with a 0.2M-1M linear salt gradient.</p>
    <p>Fractions containing the fusion proteins were concentrated using a SP-sepharose column and then loaded onto a S-100 HR gel-filtration column equilibrated with buffer A containing 0.5M NaCl. Following gel-filtration step, pure fractions were combined and the fusion proteins were stored in 50% glycerol at -20 C. or at -70 C. for long-time storage. After the final step of purification, the yield of each purified zinc finger fusion protein was greater than 100 g per 10 gm of cell paste. The low yield can be attributed to the following: (1) the gene product is toxic to the cells and (2) a large portion of the fusion protein is lost as inclusion bodies.</p>
    <p>4. Construction of ZF-ONR Fusions with Different Linkers</p>
    <p>The three ZF-QNR-F<sub>N</sub> constructs with different linkers were prepared using synthetic oligomers as described below. The inserts for the linkers were made by annealing the appropriate oligomers. These include: ##STR1##</p>
    <p>The annealed oligonucleotide duplex made from each pair of primers are flanked by SpeI compatible 5'-overhangs at both ends. The oligonucleotide duplexes were phosphorylated by using T<sub>4</sub> polynucleotide kinase and ATP. The plasmid pET-15b:ZF-QNR-F<sub>N</sub> was digested with SpeI, dephosphorylated using calf intestinal phosphatase and then gel-purified. The phosphorylated inserts were then ligated into the linearized plasmid.</p>
    <p>Several clones were screened for the appropriate inserts by restriction enzyme digestion. Plasmid with the right orientation of the inserts were further confirmed by DNA sequencing. The hybrid enzymes with different linkers were partially purified using His-bind affinity column and SP-sepharose column as described above. The DNA cleavage properties of the partially purified proteins were analyzed using the lambda DNA substrate as described above.</p>
    <heading>EXAMPLES</heading> <p>1. Construction of Overproducer Clones of ZF-F<sub>N</sub> Using PCR</p>
    <p>Two plasmids containing three zinc fingers each (ZF-QDR and ZF-QNR) were shown to preferentially bind to 5'-G(G/A)G G(C/T/A)G GC(T/A)-3' and 5'-G(G/A)G GA(T/A) GG(G/T)-3' sequences in double-stranded DNA, respectively (13-15). We used the PCR technique to link the zinc finger proteins to the cleavage domain -(F<sub>N</sub>) of FokI endonuclease (FIG. 1A). The hybrid gene, ZF-F<sub>N</sub> was cloned as a XhoI/NdeI fragment into pET-15b vector (26), which contains a T<sub>7</sub> promoter for expression of the hybrid protein. We also inserted a glycine linker (Gly<sub>4</sub> Ser)<sub>3</sub> (SEQ ID NO:21), between the domains of the fusion protein to confer added flexibility to the linker region (FIG. 1B).</p>
    <p>This construct links the zinc finger proteins through the glycine linker to the C-terminal 196-amino acids of FokI that constitute the FokI cleavage domain (8). This construct also tags the hybrid protein with six consecutive histidine residues at the N-terminus. These residues serve as the affinity tag for the purification of the hybrid proteins by metal chelation chromatography (23) with Novagen's His-bind resin. This histidine tag, if necessary, can be subsequently removed by thrombin. The hybrid endonucleases with His tag were used in all experiments described below.</p>
    <p>The clones carrying the hybrid genes may not be viable since there is no methylase available to protect the host genome from cleavage by the hybrid endonuclease. We have circumvented this problem as follows: (i) The hybrid genes were cloned into a tightly controlled expression system (26) to avoid any deleterious effect to the cell. (ii) In addition, we increased the level of DNA ligase within the cell by placing the E. coli lig gene on a compatible plasmid pACYC184, downstream of the chloramphenicol promoter. This vector expresses DNA ligase constitutively. BL21 (DE3) served as the host for these experiments. It contains a chromosomal copy of T<sub>7</sub> RNA polymerase gene under lacUV5 control, the expression of which is induced by the addition of isopropyl, -D-thio-galactoside (IPTG).</p>
    <p>After induction of the recombinant cells with 0.7 mM IPTG, the hybrid proteins were purified to homogeneity using His-bind resin, SP-sepharose column and gel filtration chromatography. The SDS/PAGE (25) profiles of the purified hybrid enzymes are shown in FIGS. 2A and 2B. Their size is 38 kDa and agrees well with that predicted for the fusion proteins. Identities of the hybrid proteins were further confirmed by probing the immunoblot with rabbit antiserum raised against FokI endonuclease (FIG. 2C).</p>
    <p>2. Analysis of the Cleavage Activity of the ZF-F<sub>N</sub> Hybrid Enzymes</p>
    <p>To determine whether the zinc finger fusion proteins cleave DNA, we used 48.5-kb  DNA as the substrate. The DNA (30 g/ml; 10 nM) was incubated with the enzymes (10 nM) in 35 mM Tris.HCl (pH 8.5), 75 mM KCl, 100 M ZnCl<sub>2</sub>, 3 mM DTT containing 5% (v/v) glycerol, 25 g/ml yeast tRNA and 50 g/ml BSA for 20 min at room temperature in a total volume of 25 l. MgCl<sub>2</sub> was then added to a final concentration of 2 mM and the mixture incubated at room temperature for 4 more hrs. The reaction products were analyzed by 0.5% agarose gel electrophoresis.</p>
    <p>The ZF-QNR-F<sub>N</sub> fusion protein cleaves  DNA into 9.5 kb and 39 kb fragments (FIG. 3, lane 4). The cleavage is highly specific and the reaction proceeds almost to completion. The ZF-QDR-F<sub>N</sub> fusion protein cleaves  DNA primarily into a 5.5 kb and a 43 kb fragments (FIG. 3, lane 3). This appears to be the major site of cleavage. There are two other minor sites within the  genome for this fusion protein. Addition of yeast RNA to the reaction mixture reduces cleavage at the minor site(s). Under these reaction conditions, there was no detectable random nonspecific cleavage as seen from the non-smearing of the agarose gels.</p>
    <p>The cleavage is sensitive to buffer conditions, pH and the purity of the DNA substrate. The kinetics of the cleavage of the lambda DNA substrate using ZF-QDR-F<sub>N</sub> and ZF-QNR-F<sub>N</sub> fusions are shown in the FIG. 4. The cleavage occurs mainly at the major DNA binding site within the lambda genome at short incubation time. The cleavage at the secondary sites become more pronounced with longer incubation times in the case of ZF-QDR-F<sub>N</sub> fusion (FIG. 4A).</p>
    <p>The cleavage occurs predominantly at the major DNA binding site in the case of the ZF-QNR-F<sub>N</sub> fusion. Only a few weaker bands appear even after long incubation times suggesting that there is only one major DNA binding site for ZF-QNR-F<sub>N</sub> in the lambda DNA substrate (FIG. 4B).</p>
    <p>The reactions appear to proceed almost to completion (&gt;95% cleavage) within 4 hrs. The kinetics of the cleavage of the lambda DNA substrate by wild-type FokI is shown in FIG. 4C. The cleavage reaction by FokI endonuclease proceeds to completion within 15 minutes. The rate and efficiency of cleavage by the hybrid endonucleases are much lower compared to wild-type FokI.</p>
    <p>We have also studied the effect of temperature and salt concentrations (KCl and MgCl<sub>2</sub>) on ZF-QNR-F<sub>N</sub> fusion protein cleavage activity using the lambda DNA as a substrate. The results of these experiments are shown in FIG. 5. The cleavage efficiency by ZF-QNR-F<sub>N</sub> appears to decrease with increasing temperatures (FIG. 5A). Room temperature (22 C.) appears to be the optimal temperature for the cleavage reaction. This may indicate the decreased binding of the ZF-QNR-F<sub>N</sub> fusion protein to the lambda DNA substrate at higher temperatures. The optimal salt concentration for cleavage appears to be 75 mM KCl. Under these conditions, the reaction proceeds to completion (FIG. 5B).</p>
    <p>The cleavage efficiency appears to drop off with increasing KCl concentration. This can be attributed to the instability of the protein-DNA complex at higher salt concentrations. The effect of increasing MgCl<sub>2</sub> (co-factor) concentration on the cleavage reaction is shown in FIG. 5C. The efficiency of cleavage increases with MgCl<sub>2</sub> concentration and the reactions proceed to completion. However, with increasing MgCl<sub>2</sub> the nonspecific cleavage by the FokI nuclease domain becomes more pronounced. The optimal MgCl<sub>2</sub> concentration for the cleavage reaction appears to be between 2-3 mM.</p>
    <p>These experiments demonstrate that cleavage activity of the ZF-QDR-F<sub>N</sub> and ZF-QNR-F<sub>N</sub> fusions are quite reproducible. Furthermore, they also show that the reaction conditions can be optimized for site-specific cleavage as well as for the complete cleavage of the substrate.</p>
    <p>These results are consistent with what is known about zinc finger-DNA interactions. The zinc finger-DNA recognition appears to be by virtue of only two base contacts of the triplet per zinc finger (10). Therefore, zinc fingers may recognize more than one DNA sequence differing by one base in the central triplets. This may explain why the ZF-QDR-F<sub>N</sub> hybrid enzyme recognizes several DNA sites with different affinities, and then cuts these sites with different efficiencies. Thus, the subsite bindings of relatively moderate affinity may contribute to the degeneracy of cleavage. On the other hand, the ZF-QNR-F<sub>N</sub> fusion suggests that a hybrid restriction enzyme with a high sequence-specificity can be engineered by using the appropriate zinc finger motifs in the fusion constructs.</p>
    <p>3. Analysis of the DNA-Sequence Preference of the ZF-F<sub>N</sub> Hybrid Enzymes</p>
    <p>Determination of the major DNA-binding sites of ZF-QDR-F<sub>N</sub> and ZF-QNR-F<sub>N</sub> fusion proteins were done in two steps: First, by using a series of known restriction enzyme digests of the  DNA followed by cleavage with the fusion protein, the site was localized within a 1-2 Kb region of the genome. Second, a 300 bp  DNA fragment containing the major cleavage site was isolated. This substrate was end-labeled with <sup>32</sup> P on the top DNA strand or the bottom DNA strand. The products of cleavage of each labeled substrate were analyzed by denaturing polyacrylamide gel electrophoresis (25) followed by autoradiography (FIGS. 6A-6B). More specifically, cleavage products of the <sup>32</sup> P-labeled DNA substrate containing a single binding-site by ZF-F<sub>N</sub> along with (G+A) sequencing reactions were separated by electrophoresis on a 8% polyacrylamide gel containing 6M urea. The gel was dried and exposed to an x-ray film for 6 hrs.</p>
    <p>The map of the primary recognition and cleavage site(s) of the ZF-QDR-F<sub>N</sub> and ZF-QNR-F<sub>N</sub> fusion proteins found in the  genome are shown in FIG. 6C and 6D, respectively. The ZF-QDR-F<sub>N</sub> fusion protein preferentially binds to 5'-GAG GAG GCT-3', which is one of the four predicted consensus sites that occur in the  genome. The ZF-QNR-F<sub>N</sub> fusion does not bind to any of the four predicted consensus sites that are present in the  genome. It preferentially binds to the 5'-GAG GGA TGT-3' site that occurs only once in the genome. The two bases that are different from the reported consensus recognition site of ZF-QNR are underlined. The reported consensus DNA binding sites of the zinc finger proteins were determined by affinity-based screening (13-15). This method utilizes a library of DNA binding sites. Under representation of any of the possible sites within this library may lead to the identification of a subsite as the optimal DNA binding site.</p>
    <p>Alternatively, the fusion of the zinc finger proteins to the FokI cleavage domain may alter the DNA sequence-specificity. This is unlikely because the binding sites for the previously reported Ubx-F<sub>N</sub> and one of the two ZF-F<sub>N</sub> fusions described here agree with the reported consensus DNA sites. As many more zinc finger fusions are engineered and characterized, this apparent discrepancy may be resolved. If the sequence-specificity of the hybrids is indeed altered, then we need to develop a fast and efficient screening method to identify or select the DNA binding sites of the hybrid restriction enzymes.</p>
    <p>The specificity of the two hybrid restriction enzymes described here are different. More than likely, the specificity of these enzymes are determined solely by the DNA-binding properties of the zinc finger motifs. It appears that the hybrid endonucleases do turnover, that is, the fusion proteins come off the substrate after cleavage. Both enzymes cleave the top strand near the binding site; they cut the bottom strand at two distinct locations. Both fusions show multiple cuts on both strands of the DNA substrate (FIGS. 6A-6D). One possibility is that the cleavage domain is not optimally positioned for cutting. Naturally occurring Type IIS enzymes with multiple cut sites have been reported in the literature (27). The variations in the cleavage pattern of the two hybrid enzymes can be attributed to the differences in the mode of binding of the zinc finger motifs to their respective DNA-binding sites and to the orientation of the nuclease domain within the enzyme-DNA complex.</p>
    <p>4. ZF-ONR Fusions with Different Linkers Between the Recognition and Cleavage Domains</p>
    <p>Five different ZF-QNR-F<sub>N</sub> hybrids containing different linkers (Table I) were constructed using synthetic oligomers. The fusion protein from each construct was partially purified using His-bind affinity column and SP-sepharose column. The presence of the fusion proteins were confirmed by Western blots using polyclonal antisera raised against restriction FokI endonuclease. Only small amounts of intact fusion proteins were obtained in the case of the hybrids with (Gly<sub>4</sub> Ser) (SEQ ID NO:32) and (Gly<sub>4</sub> Ser)<sub>2</sub> (SEQ ID NO:35) linkers and therefore, they were not tested for sequence-specific cleavage activity.</p>
    <p>The fusions with no linker, (GlyGly) and (Gly<sub>4</sub> Ser)<sub>3</sub> (SEQ ID NO:27) were partially purified as described above; the hybrid enzymes from constructs with no linker or (GlyGly) linker showed only minimal sequence-specific cleavage (Table I). (Gly<sub>4</sub> Ser)<sub>3</sub> (SEQ ID NO:27) appears to be the optimal spacer between the zinc finger and the FokI cleavage domain. This spacer appears to provide the added flexibility to the two functional domains of the zinc finger hybrids that is necessary for optimal DNA cleavage.</p>
    <p>5. Cloning ZF-FN in the Presence of lig Gene</p>
    <p>The E.coli lig gene was inserted into the NcoI site of plasmid pACYC184. The plasmid was prepared as described by Chang and Cohen (39) and carries the tetracycline drug marker. The recombinant plasmid carrying the lig gene in the same orientation as the chloramphenicol promoter was identified as pACYC lig (FIG. 7). This recombinant was transfected into competent E.coli strain BL21 (DE3). The hybrid restriction endonuclease genes on a separate compatible plasmid, pET-15b were transfected into competent BL21 (DE3) (pACYC lig) as well as competent BL21(DE3) cells.</p>
    <p>0.1 ml of the transformation mix was plated on LB-Amp-Tet plates (FIG. 8). The pTZ19R that does not carry a hybrid endonuclease gene was used as a standard control to compare the efficiency of transformation of the competent cells. BL21 (DE3) (pACYC lig) transformed at about 5-10 fold lower efficiency as compared to the BL21(DE3) cells (FIG. 8).</p>
    <p>6. Transformation Efficiency of Hybrid Genes</p>
    <p>The transformation efficiency of two different endonuclease genes, pET-15b:ZFHD1-F<sub>N</sub> and pET-15b:ZF-QQR-F<sub>N</sub>, into BL21 (DE3), with and without pACYC lig is summarized in Table II. BL21 (DE3) with pACYC lig transform about 2-fold better compared to BL21 (DE3) without the pACYC lig. Taking into consideration 5-10 fold lower efficiency of BL21 (DE3) (pACYC lig) as compared to BL21 (DE3), this translates into about 10-20 fold difference between E.coli strains with and without pACYC lig.</p>
    <p>7. Induction of Hybrid Nuclease Activity</p>
    <p>In a different example, pET-15b:ZF-QDR-F<sub>N</sub> (where the hybrid endonuclease gene is under the control of a T<sub>7</sub> promoter) was transfected into two different E.coli strains, namely RR1 and BL21 (DE3) both without the plasmid, pACYC lig carrying the lig gene. While pET-15b:ZF-QDR-F<sub>N</sub> was stably maintained in RR1, it is unstable when it is transfected into BL21 (DE3), which has a copy of the T<sub>7</sub> RNA polymerase gene in its chromosome. Only mutants of the hybrid endonuclease gene were obtained upon transfection of competent BL21 (DE3) cells. Plasmids from six different clones were isolated and analyzed by digestion with NdeI/XhoI. While three clones showed that the hybrid restriction endonuclease gene was deleted, the others contained inserts that were 0.9 kb larger than the original gene (FIG. 9). The hybrid gene appears to be disrupted by the insertion of an IS1 element.</p>
    <p>We circumvented this problem by transfecting the pET-15:ZF-QDR-F<sub>N</sub> into E.coli BL21 (DE3) cells that carry the compatible plasmid (pACYC lig) which has the E.coli lig gene inserted downstream of the chloramphenicol promoter. This plasmid expresses the DNA ligase constitutively. The pET-15b:ZF-QDR-F<sub>N</sub> is stable within these cells. Induction of these clones with IPTG result in the production of the hybrid enzyme. This is an important finding and development since this implies that any hybrid endonuclease will be tolerated by the cells provided they can express the DNA ligase constitutively and thereby repair the damage. No methylase is needed to protect the host genome from cleavage by the hybrid endonuclease.</p>
    <p>We have cloned several hybrid endonucleases using this approach. This is summarized in Table III. ZF-Sp1C-F<sub>N</sub> is a specific abbreviation for a fusion product between the three zinc finger motif of the eukaryotic transcription factor Sp1 (a specific zinc finger) and the FokI endonuclease domain (F<sub>N</sub>).</p>
    <p>This patent application is for a method of cloning any hybrid endonuclease gene in any type of cell wherein any DNA ligase is produced at an increased level compared to the normal level of DNA ligase in the specific cell type. More specifically, the method of this patent application includes the cloning of any hybrid endonuclease gene in any prokaryotic (e.g., E. coli, mutants of E. coli, etc.) or eukaryotic (e.g., yeast, plant or mammalian, etc.) cell that has been altered to produce increased levels of any type of DNA ligase (e.g., T<sub>4</sub> ligase gene, etc.) within the cell.</p>
    <p>8. Induction of Anti-Bacterial Activity with Hybrid Nuclease</p>
    <p>A specific application for these engineered sequence-specific endonucleases is in the cleavage, and thereby inactivation of genes in vivo. Several methods are currently available to express foreign genes in a number of bacterial, fungal, plant and animal species. These include transient expression via episomal or viral vectors or by microinjection. Such methods could be used for the delivery and expression of hybrid endonucleases within cells. Essentially any DNA intermediate is a potential target or substrate for cleavage by a hybrid endonuclease. These include RNA tumor viruses which replicate through a DNA intermediate. It should be possible to target one or more hybrid endonucleases against these specific DNA intermediates provided the gene sequences are known. Expression in vivo of such hybrid restriction enzymes would in effect destroy the corresponding gene. This targeted gene inactivation by the hybrid endonucleases could provide a basis for various anti-viral and anti-bacterial therapies and for a way to inactivate human, animal or plant genes.</p>
    <p>In another example, plasmids containing one of two different hybrid endonucleases, namely pET-15b:ZF-QDR-F<sub>N</sub> and pET-15b:ZF-QNR-F<sub>N</sub>, were separately transfected into E.coli BL21 (DE3) (pACYC lig) by standard CaCl<sub>2</sub> procedure. The clones were then plated on LB-Amp-Tet plates with and without IPTG. Induction with IPTG turns on the production of T<sub>7</sub> RNA polymerase, which lead to the production of the hybrid restriction enzymes. The constitutively produced ligase cannot cope and repair the damage resulting from the hybrid restriction enzymes. Therefore, the clones should not be viable upon induction with IPTG.</p>
    <p>FIGS. 10A and 10B show the results obtained from such an experiment. BL21 (DE3) (pACYC lig) containing the hybrid endonuclease genes on a compatible plasmid grow well on LB-Amp-Tet plates without IPTG. No growth is observed when they are grown on LB-Amp-Tet plates containing 1 mM IPTG. Control BL21 (DE3) (pACYC lig) (pET-15b) strain that does not carry the hybrid restriction endonuclease gene grow well on LB-Amp-Tet plates with and without IPTG (FIG. 11).</p>
    <p>This example shows that bacteria carrying the hybrid restriction enzymes gene can be forced to self-destruct by inducing the hybrid restriction enzymes. This example also provides proof of concept for potential use of hybrid restriction enzymes as therapeutic agents. Obviously, the hybrid restriction endonuclease genes could also be delivered into cells via a plasmid, virus, phage or any other delivery vehicle that infects a particular type of bacterial or mammalian cells, including plant and animal cells.</p>
    <p>Bacteriophages have been shown to be effective in the treatment of experimental E.coli infection (10,11). More recently, bacteriophage was shown to prevent destruction of skin grafts by Pseudomonas aeruginosa (12). These bacteriophages can be engineered to carry the lethal hybrid endonuclease genes targeted against their hosts. These bacteriophages will be more effective in the destruction of the bacteria they infect. The present invention specifically includes this concept as well. The present invention also contemplates the delivery of other normal as well as mutant site-specific restriction enzymes using a similar approach.</p>
    <p>9. Site-Specific Cleavage of DNA-RNA Hybrids</p>
    <p>As mentioned above, zinc finger proteins of the type Cys<sub>2</sub> His<sub>2</sub> are quite prevalent in human and other eukaryotic genomes (33). Recently, two such proteins, ZF-SP1C and ZF-QQR, were shown to bind DNA-RNA hybrids with affinities comparable to those of DNA duplexes (34). We have converted these specific zinc finger proteins into site-specific endonucleases by linking them to the FokI cleavage domain (F<sub>N</sub>), as described above. Here, we show that the ZF-QQR-F<sub>N</sub> fusion enzyme binds to and cleaves DNA-RNA hybrids in a sequence-dependent manner.</p>
    <p>Crystal structures of zinc finger protein-DNA complexes (10-12) have shown that the proteins contact one strand of the DNA much more than the other strand. The three-zinc finger binding unit from the transcription factor Sp1 (ZF-Sp1C) was shown to bind a 19-base pair DNA fragment containing the site 5'-GGGGCGGGG-3' with a similar affinity to that of DNA-RNA hybrid with the DNA strand containing this guanine-rich site (34). Furthermore, a designed consensus sequence-based protein (14), ZF-QQR was shown to bind DNA-RNA hybrid containing the site 5'-GGGGAAGAA-3' on the DNA strand about five times as tightly as the DNA duplex. We reasoned that by linking these zinc finger proteins to the FokI cleavage domain, it might be possible to engineer site-specific endonucleases that cleave DNA-RNA hybrids. PCR technology was used to link the zinc finger proteins to the cleavage domain of FokI to generate the hybrid genes, ZF-QQR-FP and ZF-SP1C-F<sub>N</sub>, respectively, as described above. This construction links the zinc finger proteins through a glycine linker to the C-terminal 196-amino acids of FokI endonuclease that constitutes the FokI cleavage domain (4,8).</p>
    <p>The cleavage properties of the purified ZF-F<sub>N</sub> fusions were analyzed using <sup>32</sup> P-labelled DNA substrates each containing a single binding site. The plasmid pBluescript KS(+) containing the ZF-QQR site was digested with KpnI (or SacI), then dephosphorylated using calf intestinal phosphatase (CIP) and rephosphorylated using T<sub>4</sub> polynucleotide kinase and -<sup>32</sup> P-ATP. The DNA was digested with SacI (or KpnI) and the small fragment with the <sup>32</sup> P label on the top strand (or the bottom strand) was purified by gel electrophoresis.</p>
    <p>The restriction enzymes BamHI (or EcoRI) were used to cleave pBluescript KS(+) containing the ZF-Sp1C site to prepare <sup>32</sup> P labelled substrates. The <sup>32</sup> P labelled substrates with 300 ng of cold DNA (tRNA or lambda DNA) in 35 mM Tris.HCl (pH 7.5), 75 mM KCl, 100 M ZnCl<sub>2</sub>, 5 mM DTT, 0.5 g BSA and 5% glycerol were incubated with ZF-F<sub>N</sub> fusions for 30 minutes at room temperature. MgCl<sub>2</sub> was then added to a final concentration of 2 mM and digested for 5 hours. The products from the reaction mixture were then analyzed by PAGE. The cleavage products along with (G+A) sequencing reaction (36) were separated by electrophoresis on an 8% polyacrylamide gel containing 6M urea. The gel was dried and then exposed to an x-ray film for 6 hours.</p>
    <p>ZF-Sp1C-F<sub>N</sub> binds to 5'-GGGGCGGGG-3' and cleaves 5' to this site on both strands of the DNA duplex (FIGS. 12C and 12D). ZF-QQR-F<sub>N</sub> binds to the 5'-GGGGAAGAA-3' site and cleaves upstream of this site on both strands of the DNA substrate (FIGS. 12A and 12B). Cleavage is also observed 3' to the site. This is likely associated with binding to a secondary site. This part of the example indicates that both fusion proteins bind to their appropriate binding sites within the substrate and, as expected, cleave upstream of their respective binding sites.</p>
    <p>To examine the specificity and cleavage of the DNA-RNA hybrids, analogous <sup>32</sup> P-labelled substrate with the DNA strand containing 5'-GGGGAAGAA-3'site was digested with ZF-QQR-F<sub>N</sub>. The results from this part of the example are shown in FIG. 13.</p>
    <p>The <sup>32</sup> P-labelled DNA-RNA hybrid was prepared as follows: The DNA was chemically synthesized and purified. The oligonucleotide was the phosphorylated using T<sub>4</sub> polynucleotide kinase and -<sup>32</sup> P-ATP. The RNA strand was then transcribed using T<sub>3</sub> RNA polymerase. The RNA-DNA hybrid was digested with ZF-QQR-F<sub>N</sub> fusion enzyme as described for FIG. 12 and then analyzed by PAGE.</p>
    <p>ZF-QQR-F<sub>N</sub> binds to the 5'-GGGGAAGAA-3'site and cleaves 5' to the binding site on the DNA strand of the DNA-RNA hybrid (FIG. 13, lane 3). However, the cut site(s) is shifted one base closer to the binding site compared to the DNA duplex cleavage. This may be attributed to the fact that DNA-RNA hybrids probably have structures distorted away from the B-form due to the presence of 2' hydroxyl groups in the RNA strand. ZF-Sp1C-F<sub>N</sub> does not bind to and cleave the DNA-RNA hybrid which contains the 5'-GGGGAAGAA-3' site that is specific for ZF-QQR-F<sub>N</sub> (FIG. 13, lane 4). In addition, only minor cleavage was observed 3' to the 5'-GGGGAAGAA-3' binding site unlike that of the DNA duplex. ZF-QQR-F<sub>N</sub> appears to preferentially contact the DNA strand of the DNA-RNA hybrid. The complementary RNA strand greatly reduces the secondary site binding of ZF-QQR-F<sub>N</sub>.</p>
    <p>In another part of this example, 32P-labelled substrate with the DNA strand containing the 5'-GGGGCGGGG-3' site was digested with ZF-Sp1C-F<sub>N</sub> fusion enzyme. The cleavage of the DNA strand of the DNA-RNA hybrid was not detectable. This result is consistent with the dissociation constants (kd) reported for ZF-Sp1C (40 nM) compared to ZF-QQR (2.8 nM) binding to DNA-RNA hybrids (34). As expected, the cleavage of the RNA-DNA hybrid by ZF-Sp1C-F<sub>N</sub> is about 15-20 fold lower than that of ZF-QQR-F<sub>N</sub>. The cleavage properties of ZF-F<sub>N</sub> fusion enzymes are summarized in FIG. 14.</p>
    <p>To compare the efficiency of cleavage of the DNA-RNA hybrid and the DNA-DNA duplex, both <sup>32</sup> P-labelled substrates were digested with ZF-QQR-F<sub>N</sub> in the same reaction tube. While the cleavage of the DNA duplex proceeded to completion within 4 hours, only about 1% of the DNA-RNA hybrid was cleaved during this time. Thus, the site-specific cleavage of the DNA-RNA hybrid occurs at a much lower rate (100 fold) compared to that of the DNA duplex.</p>
    <p>One possible explanation for the lower rate of the cleavage of the DNA-RNA hybrid may be that the fusion of the ZF-QQR protein to the FokI cleavage domain alters its binding affinity for the substrate. Alternatively, the FokI cleavage domain probably has evolved to cleave only the DNA strand. To delineate between these two possibilities, we determined the apparent dissociation constant of ZF-QQR-F<sub>N</sub> binding to DNA-RNA hybrid by using gel mobility-shift assays (FIG. 15). We obtained an apparent dissociation constant of 3 nM similar to the one reported for ZF-QQR (2.8 nM). Thus, ZF-QQR fusion to the FokI cleavage domain does not alter its sequence-specificity or its substrate affinity. Therefore, it appears that the FokI cleavage domain has evolved to restrict only DNA strands and not RNA strands.</p>
    <p>To further examine the cleavage of the RNA strand, a DNA-RNA hybrid with the DNA strand containing the binding site 5'-GGGGAAGAA-3' and the <sup>32</sup> P-label on the RNA strand was digested with ZF-QQR-F<sub>N</sub> fusion enzyme. No detectable cleavage of the RNA strand by ZF-QQR-F<sub>N</sub> fusion enzyme was observed. Zinc finger fusions that cleave RNA strands may be developed by the fusion of zinc finger proteins to various RNA cleaving moieties. Alternatively, in vitro molecular evolution methods could be adopted to identify enzymes that will cleave the RNA strand.</p>
    <p>In summary, this example shows that an engineered zinc finger fusion enzyme, ZF-QQR-F<sub>N</sub>, can cleave RNA-DNA hybrids in a sequence-dependent manner. Thus, these fusion enzymes may have further important biological applications. Essentially any DNA-RNA intermediate is a potential substrate for cleavage by these zinc finger fusion enzymes. These include RNA tumor viruses which replicate through a DNA intermediate. It should be possible to target one or more of these hybrid endonucleases against these specific DNA-RNA intermediates provided that the gene sequences are known and that the zinc finger proteins that bind to these sequences are available. Expression in vivo of such hybrid restriction enzymes would in effect destroy the corresponding gene. This targeted gene inactivation by the hybrid endonucleases could provide a basis for various anti-viral therapies.</p>
    <p>The following scientific article have been cited throughout the present application and are hereby incorporated by reference in their entirety and relied upon:</p>
    <p>1. Smith H. O. &amp; Wilcox, K W., J. Mol. Biol. 51: 379-391, 1970.</p>
    <p>2. Wilson, G. G., The NEB Transcript 5: 1, 1993.</p>
    <p>3. Roberts, R., The NEB Transcript 5: 13, 1993.</p>
    <p>4. Li, L., Wu, L. P. &amp; Chandrasegaran, S., Proc. Natl. Acad. Sci U.S.A. 89: 4279-4275, 1992.</p>
    <p>5. Li, L., Wu, L. P., Clark R. &amp; Chandrasegaran, S., Gene 133: 79-84, 1993.</p>
    <p>6. Li, L. &amp; Chandrasegaran, S., Proc. Natl. Acad. Sci U.S.A. 90: 2764-2768, 1993.</p>
    <p>7. Kim, Y.-G., Li, L. &amp; Chandrasegaran, S., J. Biol. Chem. 269: 31978-31982, 1994.</p>
    <p>8. Kim, Y.-G. &amp; Chandrasegaran, S., Proc. Natl. Acad. Sci. U.S.A. 91: 883-887, 1994.</p>
    <p>9. Berg, J. M., Proc. Natl. Acad. Sci. U.S.A. 85: 99-102, 1988.</p>
    <p>10. Pavietich, N. P. &amp; Pabo, C. O., Science 252: 809-817, 1992.</p>
    <p>11. Pavietich, N. P. &amp; Pabo, C. O., Science 261: 1701-1707, 1993.</p>
    <p>12. Fairall, L., Schwabe, J. W. R., Chapman, L., Finch, J. T. &amp; Rhodes, Nature (London) 366: 483-487, 1993.</p>
    <p>13. Desjarlais, J. R. &amp; Berg, J. M., Proc. Natl. Acad. Sci. U.S.A. 89: 7345 7349, 1992.</p>
    <p>14. Desjarlais, J. R. &amp; Berg, J. M., Proc. Natl. Acad. Sci U.S.A. 90: 2256-2260, 1993.</p>
    <p>15. Desjarlais, J. R. &amp; Berg, J. M., Proc. Natl. Acad. Sci U.S.A. 91: 11099-11103, 1994.</p>
    <p>16. Rebar, E. J. &amp; Pabo, C. O., Science 263: 671-673, 1994.</p>
    <p>17. Choo, Y. &amp; Klug, A., Proc. Natl. Acad. Sci. U.S.A. 91: 11163-11167, 1994.</p>
    <p>18. Choo, Y. &amp; Klug, A., Proc. Natl. Acad. Sci. U.S.A. 91: 11168-11172, 1994.</p>
    <p>19. Jamieson, A. C., Kim, S.-H. &amp; Wells, J. A., Biochemistry 33: 5689-5695, 1994.</p>
    <p>20. Wu, H., Yang, W.-P. &amp; Barbas III, C. F., Proc.</p>
    <p>Natl. Acad. Sci. U.S.A. 92: 344-348, 1995.</p>
    <p>21. Kita, K., Kotani, H., Sugisaki H. &amp; Takanami, M., J. Biol. Chem. 264: 5751-5756, 1989.</p>
    <p>22. Looney, M. C., Moran, L. S., Jack, W. E., Feehery, G. R., Benner, J. S., Slatko, B. E. &amp; Wilson, G. G., Gene 80: 193-208, 1989.</p>
    <p>23. Hochuli, E., Dobeli, H. &amp; Schacher, A., J. Chromatogr. 411: 177-184, 1987.</p>
    <p>24. Laemili, U. K., Nature (London) 222: 680-685, 1970.</p>
    <p>25. Sanger, F., Nicklen S. &amp; Coulson, A. R., Proc. Natl. Acad. Sci. U.S.A. 74: 5463-5467, 1977.</p>
    <p>26. Studier, F. W., Rosenberg, A. H., Dunn J. J. &amp; Dubendorff, J. W., Methods in Enzymol. 185: 60-89, 1990.</p>
    <p>27. Szybalski, W., Kim, S. C. Hasan, N. &amp; Podhajska, A. J., Gene 100: 13-26, 1991.</p>
    <p>28. Waugh, D. S. and Sauer, R. T., Proc. Natl. Acad. Sci. U.S.A. 90: 9596-9600, 1993.</p>
    <p>29. Waugh, D. S. and Sauer, R. T., J. Biol. Chem. 269: 12298-12303, 1994.</p>
    <p>30. Smith, W. H. &amp; Huggins, M. B., J. Gen. Microbiol. 128: 307-318, 1982.</p>
    <p>31. Smith, W. H., Huggins, M. B. &amp; Shaw, K. M., J. Gen Microbiol. 133: 1111-1126, 1987.</p>
    <p>32. Soothill, J. S., Burns 20: 209-211, 1994.</p>
    <p>33. Rhodes, D. &amp; Klug, A., Sci. Am. 268: 56-59, 1993.</p>
    <p>34. Shi, Y. &amp; Berg, J. M., Science 268: 282-284, 1995.</p>
    <p>35. Hochuli, E., Dobeli, H. &amp; Schacher, A., J. Chromatogr. 411: 177-184, 1987.</p>
    <p>36. Maxam, A. M. &amp; Gilbert, W., Proc. Natl. Acad. Aci. U.S.A. 74: 560-564, 1977.</p>
    <p>37. Birkenbihl, R. P. &amp; Vielmetter, W., Mol. Gen. Genet. 220: 147-153, 1989.</p>
    <p>38. Maniatis, T., Fritsch, E. F. and Sambrook, J., Molecular Cloning, A Laboratory Manual, Cold Spring Harbor Laboratory, New York, 1982.</p>
    <p>39. Chang A. C. Y. &amp; Cohen, S. N., J. Bacteriol. 134: 1141-1156, 1978.</p>
    <p>While the invention has been described in connection with what is presently considered to be the most practical and preferred embodiments, it is to be understood that the invention is not limited to the disclosed embodiments, but on the contrary is intended to cover various modifications and equivalent arrangements included within the spirit and scope of the appended claims.</p>
    <p>Thus, it is to be understood that variations in the present invention can be made without departing from the novel aspects of this invention as defined in the claims.</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">              TABLE I______________________________________Zif-ONR fusions withLinker          Expression<sup>1</sup>                     Activity (specific)<sup>2</sup>______________________________________No Linker       +         +GG              +         +G<sub>4</sub> S (SEQ ID NO: 32)           -         n.t.(G<sub>4</sub> S)<sub>2</sub> (SEQ ID NO: 35)           -         n.t.(G<sub>4</sub> S)<sub>3</sub> (SEQ ID NO: 27)           ++        +++______________________________________ <sup>1</sup> The fusion protein from each construct was partially purified usin Hisbind affinity column and SPsepharose column. The expression of the fusions were confirmed by Western blot using polyclonal antisera raised against FokI. <sup>2</sup> Sequencespecific cleavage was assayed using lambda DNA as substrate. n.t., Not tested.</pre>
    
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">              TABLE III______________________________________Comparison of the Efficiency of transformation of thehybrid endonuclease genes in E. coli withand without pACYC lig     Number of Colonies       pACYC    Plate  Plate  Aver-BL21 (DE3)  lig      #1     #2     age  Ratio<sup>1</sup>______________________________________pET-15b:ZFHD1-FN       -        135    111    123                                   2.4pET-15b:ZFHDl-F<sub>N</sub>       +        309    292    300pET-15b:ZF-QQR-F<sub>N</sub>       -        375    500    437                                   2.3pET-15b:ZF-QQR-F<sub>N</sub>       +        850    1153   1001______________________________________ <sup>1</sup> Taking into consideration 5-10 fold lower transformation efficienc of BL2l (DE3) (pACYC lig) competent cells as compared to BL21 (DE3), this translates into about 0-20 fold difference between E. coli strains with and without pACYC lig.</pre>
    
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">                                  TABLE III__________________________________________________________________________Summary of hybrid endonucleases cloned using E. coli DNA lig gene.          Sequence-               Predicted          Specific-               CleavageFusion Protein  Linker  Activity               Sites     Indentified Cleavage Site__________________________________________________________________________ZF-QNR-F<sub>N</sub>  (SEQ ID NO: 27) (G<sub>4</sub> S)<sub>3</sub>          +                ##STR2## 5'-GAG GGATGT-3'ZF-QDR-F<sub>N</sub>  (G<sub>4</sub> S)<sub>3</sub>          +                ##STR3## 5'-GAG GAG GCT-3'ZF-QQR-F<sub>N</sub>  (G<sub>4</sub> S)<sub>3</sub>          +                ##STR4## 5'-GGG GAAGAA-3'ZF-SP1C-F<sub>N</sub>  (G<sub>4</sub> S)<sub>3</sub>          +                ##STR5## 5'-GGG GCG GGG-3'__________________________________________________________________________</pre>
    
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">__________________________________________________________________________SEQUENCE LISTING(1) GENERAL INFORMATION:(iii) NUMBER OF SEQUENCES: 35(2) INFORMATION FOR SEQ ID NO:1:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 23 amino acids(B) TYPE: amino acid(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:1:TyrXaaCysXaaXaaCysXaaXaaXaaPheXaaXaaXaaXaaXaaLeu151015XaaXaaHisXaaXaaXaaHis20(2) INFORMATION FOR SEQ ID NO:2:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 24 amino acids(B) TYPE: amino acid(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:2:TyrXaaCysXaaXaaCysXaaXaaXaaPheXaaXaaXaaXaaXaaLeu151015XaaXaaHisXaaXaaXaaXaaHis20(2) INFORMATION FOR SEQ ID NO:3:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 25 amino acids(B) TYPE: amino acid(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:3:TyrXaaCysXaaXaaCysXaaXaaXaaPheXaaXaaXaaXaaXaaLeu151015XaaXaaHisXaaXaaXaaXaaXaaHis2025(2) INFORMATION FOR SEQ ID NO:4:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 24 amino acids(B) TYPE: amino acid(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:4:TyrXaaCysXaaXaaXaaCysXaaXaaXaaPheXaaXaaXaaXaaXaa151015LeuXaaXaaHisXaaXaaXaaHis20(2) INFORMATION FOR SEQ ID NO:5:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 25 amino acids(B) TYPE: amino acid(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:5:TyrXaaCysXaaXaaXaaCysXaaXaaXaaPheXaaXaaXaaXaaXaa151015LeuXaaXaaHisXaaXaaXaaXaaHis2025(2) INFORMATION FOR SEQ ID NO:6:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 26 amino acids(B) TYPE: amino acid(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:6:TyrXaaCysXaaXaaXaaCysXaaXaaXaaPheXaaXaaXaaXaaXaa151015LeuXaaXaaHisXaaXaaXaaXaaXaaHis2025(2) INFORMATION FOR SEQ ID NO:7:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 25 amino acids(B) TYPE: amino acid(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:7:TyrXaaCysXaaXaaXaaXaaCysXaaXaaXaaPheXaaXaaXaaXaa151015XaaLeuXaaXaaHisXaaXaaXaaHis2025(2) INFORMATION FOR SEQ ID NO:8:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 26 amino acids(B) TYPE: amino acid(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:8:TyrXaaCysXaaXaaXaaXaaCysXaaXaaXaaPheXaaXaaXaaXaa151015XaaLeuXaaXaaHisXaaXaaXaaXaaHis2025(2) INFORMATION FOR SEQ ID NO:9:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 27 amino acids(B) TYPE: amino acid(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:9:TyrXaaCysXaaXaaXaaXaaCysXaaXaaXaaPheXaaXaaXaaXaa151015XaaLeuXaaXaaHisXaaXaaXaaXaaXaaHis2025(2) INFORMATION FOR SEQ ID NO:10:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 23 amino acids(B) TYPE: amino acid(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:10:PheXaaCysXaaXaaCysXaaXaaXaaPheXaaXaaXaaXaaXaaLeu151015XaaXaaHisXaaXaaXaaHis20(2) INFORMATION FOR SEQ ID NO:11:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 24 amino acids(B) TYPE: amino acid(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:11:PheXaaCysXaaXaaCysXaaXaaXaaPheXaaXaaXaaXaaXaaLeu151015XaaXaaHisXaaXaaXaaXaaHis20(2) INFORMATION FOR SEQ ID NO:12:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 25 amino acids(B) TYPE: amino acid(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:12:PheXaaCysXaaXaaCysXaaXaaXaaPheXaaXaaXaaXaaXaaLeu151015XaaXaaHisXaaXaaXaaXaaXaaHis2025(2) INFORMATION FOR SEQ ID NO:13:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 24 amino acids(B) TYPE: amino acid(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:13:PheXaaCysXaaXaaXaaCysXaaXaaXaaPheXaaXaaXaaXaaXaa151015LeuXaaXaaHisXaaXaaXaaHis20(2) INFORMATION FOR SEQ ID NO:14:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 25 amino acids(B) TYPE: amino acid(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:14:PheXaaCysXaaXaaXaaCysXaaXaaXaaPheXaaXaaXaaXaaXaa151015LeuXaaXaaHisXaaXaaXaaXaaHis2025(2) INFORMATION FOR SEQ ID NO:15:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 26 amino acids(B) TYPE: amino acid(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:15:PheXaaCysXaaXaaXaaCysXaaXaaXaaPheXaaXaaXaaXaaXaa151015LeuXaaXaaHisXaaXaaXaaXaaXaaHis2025(2) INFORMATION FOR SEQ ID NO:16:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 25 amino acids(B) TYPE: amino acid(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:16:PheXaaCysXaaXaaXaaXaaCysXaaXaaXaaPheXaaXaaXaaXaa151015XaaLeuXaaXaaHisXaaXaaXaaHis2025(2) INFORMATION FOR SEQ ID NO:17:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 26 amino acids(B) TYPE: amino acid(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:17:PheXaaCysXaaXaaXaaXaaCysXaaXaaXaaPheXaaXaaXaaXaa151015XaaLeuXaaXaaHisXaaXaaXaaXaaHis2025(2) INFORMATION FOR SEQ ID NO:18:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 27 amino acids(B) TYPE: amino acid(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:18:PheXaaCysXaaXaaXaaXaaCysXaaXaaXaaPheXaaXaaXaaXaa151015XaaLeuXaaXaaHisXaaXaaXaaXaaXaaHis2025(2) INFORMATION FOR SEQ ID NO:19:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 39 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: double(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:19:NNNNNNNNNNNNNNNNNNNNNNNNNGAGGAGGCTNNNNN39(2) INFORMATION FOR SEQ ID NO:20:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 39 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: double(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:20:NNNNNNNNNNNNNNNNNNNNNNNNNGAGGGATGTNNNNN39(2) INFORMATION FOR SEQ ID NO:21:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 74 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:21:GAGCTCCTTTGGATCCAAGCTTCCCGGGGAAGAATTCGAGGAGGCTCTCGAGGTCGACTT60CCTCTAGAGGTACC74(2) INFORMATION FOR SEQ ID NO:22:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 74 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: double(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:22:GAGCTCCTTTGGATCCAAGCTTCCCGGGGAAGAATTCGAGGAGGCTCTCGAGGTCGACTT60CCTCTAGAGGTACC74(2) INFORMATION FOR SEQ ID NO:23:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 74 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:23:CUCGACCAAACCUAGGUUCGAAGGGCCCCUUCUUAAGCUCCUCCGAGAGCUCCAGCUGAA60GGAGAUCUCCAUGG74(2) INFORMATION FOR SEQ ID NO:24:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 87 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: double(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:24:GGATCCAAGCTTAGCGATCTGCCTGCAGGTCGACTCTAGCCAGGGGCGGGGTGGTCTAGT60ATCGTTCAATGATACTTCATGGAATTC87(2) INFORMATION FOR SEQ ID NO:25:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 23 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:25:CCCCTGAAGGAGATATACATATG23(2) INFORMATION FOR SEQ ID NO:26:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 25 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:26:GGACTAGTCCCTTCTTATTCTGGTG25(2) INFORMATION FOR SEQ ID NO:27:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 15 amino acids(B) TYPE: amino acid(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:27:GlyGlyGlyGlySerGlyGlyGlyGlySerGlyGlyGlyGlySer151015(2) INFORMATION FOR SEQ ID NO:28:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 14 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:28:CTGACGGGGGCCAA14(2) INFORMATION FOR SEQ ID NO:29:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 15 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:29:CTAGTTGGCCCCCGT15(2) INFORMATION FOR SEQ ID NO:30:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 24 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:30:CTAGACGGGGGAGGCGGCAGTCAA24(2) INFORMATION FOR SEQ ID NO:31:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 24 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:31:CTAGTTGACTGCCGCCTCCCCCGT24(2) INFORMATION FOR SEQ ID NO:32:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 5 amino acids(B) TYPE: amino acid(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:32:GlyGlyGlyGlySer15(2) INFORMATION FOR SEQ ID NO:33:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 39 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:33:CTAGACGGGGGAGGCGGCAGTGGAGGTGGCGGATCACAA39(2) INFORMATION FOR SEQ ID NO:34:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 39 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:34:CTAGTTGTGATCCGCCACCTCCACCGTCGCCTCCCCCGT39(2) INFORMATION FOR SEQ ID NO:35:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 10 amino acids(B) TYPE: amino acid(D) TOPOLOGY: linear(xi) SEQUENCE DESCRIPTION: SEQ ID NO:35:GlyGlyGlyGlySerGlyGlyGlyGlySer1510__________________________________________________________________________</pre>
    
    </div></div></div><div class="patent-section patent-tabular-section"><a id="backward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Cited Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5200333">US5200333</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Mar 19, 1991</td><td class="patent-data-table-td patent-date-value">Apr 6, 1993</td><td class="patent-data-table-td ">New England Biolabs, Inc.</td><td class="patent-data-table-td ">Digesting DNA library coding for restriction edonuclease with a restriction enzyme to obtain DNA coding for methlase, transforming a host cell with genetically engineered DNA, analyzing extract for endonuclease activity and isolating</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="npl-citations"></a><div class="patent-section-header"><span class="patent-section-title">Non-Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th colspan="3"class="patent-data-table-th">Reference</th></tr></thead><tr><td class="patent-data-table-td ">1</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Adrian Whitehouse et al., A Carboxy Terminal Domain of the hMSH 2 Gene Product is Sufficient for Binding Specific Mismatched Oligonucleotides, Biochemical and Biophysical Research Communications, 1996, vol. 225, 289 295, Article No. 1168.</td></tr><tr><td class="patent-data-table-td ">2</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Adrian Whitehouse et al., A Carboxy Terminal Domain of the hMSH-2 Gene Product is Sufficient for Binding Specific Mismatched Oligonucleotides, Biochemical and Biophysical Research Communications, 1996, vol. 225, 289-295, Article No. 1168.</td></tr><tr><td class="patent-data-table-td ">3</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Alla Lishanski et al., Mutation Detection by Mismatch Binding Protein, MutS, in Amplified DNA: Application to the Cystic Fibrosis Gene, Proc. Natl. Acad. Sci. USA, Mar. 1994, pp. 2674 2678, vol. 91.</td></tr><tr><td class="patent-data-table-td ">4</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Alla Lishanski et al., Mutation Detection by Mismatch Binding Protein, MutS, in Amplified DNA: Application to the Cystic Fibrosis Gene, Proc. Natl. Acad. Sci. USA, Mar. 1994, pp. 2674-2678, vol. 91.</td></tr><tr><td class="patent-data-table-td ">5</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Baohua Huang et al., Splase: A New Class IIS Zinc Finger Restriction Endonuclease with Specificity for Sp1 Binding Sites, Journal of Protein Chemistry, 1996, pp. 481 489, vol. 15.</td></tr><tr><td class="patent-data-table-td ">6</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Baohua Huang et al., Splase: A New Class IIS Zinc-Finger Restriction Endonuclease with Specificity for Sp1 Binding Sites, Journal of Protein Chemistry, 1996, pp. 481-489, vol. 15.</td></tr><tr><td class="patent-data-table-td ">7</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Lin Li et al., Alteration of the Cleavage Distance of FOK Restriction Endonuclease by Insertion Mutagenesis, Pro. Natl. Acad. Sci, USA, Apr. 1993, pp. 2764 2768, vol. 90.</td></tr><tr><td class="patent-data-table-td ">8</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Lin Li et al., Alteration of the Cleavage Distance of FOK Restriction Endonuclease by Insertion Mutagenesis, Pro. Natl. Acad. Sci, USA, Apr. 1993, pp. 2764-2768, vol. 90.</td></tr><tr><td class="patent-data-table-td ">9</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Lin Li et al., C terminal Deletion Mutants of the FOK I Restriction Endonuclease, Gene, 133 1993, pp. 79 84.</td></tr><tr><td class="patent-data-table-td ">10</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Lin Li et al., C-terminal Deletion Mutants of the FOK I Restriction Endonuclease, Gene, 133 1993, pp. 79-84.</td></tr><tr><td class="patent-data-table-td ">11</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Lin Li et al., Functional Domains in FOK I Restriction Endonuclease, Proc. Natl. Acad. Sci. USA, May 1992, pp. 4275 4279, vol. 89.</td></tr><tr><td class="patent-data-table-td ">12</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Lin Li et al., Functional Domains in FOK I Restriction Endonuclease, Proc. Natl. Acad. Sci. USA, May 1992, pp. 4275-4279, vol. 89.</td></tr><tr><td class="patent-data-table-td ">13</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Rima Youil et al., Screening for Mutations by Enzyme Mismatch Cleavage with T4 Endonuclease VII, Proc. Natl. Acad. Sci. USA, Jan. 1995, pp. 87 91, vol. 92.</td></tr><tr><td class="patent-data-table-td ">14</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Rima Youil et al., Screening for Mutations by Enzyme Mismatch Cleavage with T4 Endonuclease VII, Proc. Natl. Acad. Sci. USA, Jan. 1995, pp. 87-91, vol. 92.</td></tr><tr><td class="patent-data-table-td ">15</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Yang Gyun Kim et al., Chimeric Restriction Endonuclease, Proc. Natl. Acad. Sci. USA, Feb. 1994, pp. 883 887, vol. 91.</td></tr><tr><td class="patent-data-table-td ">16</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Yang Gyun Kim et al., Hybrid Restriction Enzymes: Zinc Finger Fusions to FOK I Cleavage Domain, Proc. Natl. Acad. Sci. USA, Feb. 1996, 1156 1160, vol. 93.</td></tr><tr><td class="patent-data-table-td ">17</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Yang Gyun Kim et al., Insertion and Deletion Mutants of FOK I Restriction Endonuclease, The Journal of Biological Chemistry, Dec. 16, 1994, pp. 31978 31982, vol. 269, No. 50.</td></tr><tr><td class="patent-data-table-td ">18</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Yang-Gyun Kim et al., Chimeric Restriction Endonuclease, Proc. Natl. Acad. Sci. USA, Feb. 1994, pp. 883-887, vol. 91.</td></tr><tr><td class="patent-data-table-td ">19</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Yang-Gyun Kim et al., Hybrid Restriction Enzymes: Zinc Finger Fusions to FOK I Cleavage Domain, Proc. Natl. Acad. Sci. USA, Feb. 1996, 1156-1160, vol. 93.</td></tr><tr><td class="patent-data-table-td ">20</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Yang-Gyun Kim et al., Insertion and Deletion Mutants of FOK I Restriction Endonuclease, The Journal of Biological Chemistry, Dec. 16, 1994, pp. 31978-31982, vol. 269, No. 50.</td></tr><tr><td class="patent-data-table-td ">21</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Yen Choo et al., In Vivo Repression by a Site Specific DNA Binding Protein Designed Against an Oncogenic Sequence, Nature, Dec. 15, 1994, pp. 642 645, vol. 372.</td></tr><tr><td class="patent-data-table-td ">22</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Yen Choo et al., In Vivo Repression by a Site-Specific DNA-Binding Protein Designed Against an Oncogenic Sequence, Nature, Dec. 15, 1994, pp. 642-645, vol. 372.</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="forward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Referenced by</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Citing Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6265196">US6265196</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Mar 31, 1999</td><td class="patent-data-table-td patent-date-value">Jul 24, 2001</td><td class="patent-data-table-td ">Johns Hopkins University</td><td class="patent-data-table-td ">Preparing delivery vehicle containing nuclease having a dna binding domain or gene encoding the nuclease, delivering into cells, then deactivating target</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6410248">US6410248</a></td><td class="patent-data-table-td patent-date-value">Jan 29, 1999</td><td class="patent-data-table-td patent-date-value">Jun 25, 2002</td><td class="patent-data-table-td ">Massachusetts Institute Of Technology</td><td class="patent-data-table-td ">General strategy for selecting high-affinity zinc finger proteins for diverse DNA target sites</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6444421">US6444421</a></td><td class="patent-data-table-td patent-date-value">Apr 2, 1998</td><td class="patent-data-table-td patent-date-value">Sep 3, 2002</td><td class="patent-data-table-td ">The United States Of America As Represented By The Department Of Health And Human Services</td><td class="patent-data-table-td ">Methods for detecting intermolecular interactions in vivo and in vitro</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6453242">US6453242</a></td><td class="patent-data-table-td patent-date-value">Jan 12, 1999</td><td class="patent-data-table-td patent-date-value">Sep 17, 2002</td><td class="patent-data-table-td ">Sangamo Biosciences, Inc.</td><td class="patent-data-table-td ">Selection of sites for targeting by zinc finger proteins and methods of designing zinc finger proteins to bind to preselected sites</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6479626">US6479626</a></td><td class="patent-data-table-td patent-date-value">Mar 1, 1999</td><td class="patent-data-table-td patent-date-value">Nov 12, 2002</td><td class="patent-data-table-td ">Massachusetts Institute Of Technology</td><td class="patent-data-table-td ">Poly zinc finger proteins with improved linkers</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6503717">US6503717</a></td><td class="patent-data-table-td patent-date-value">Dec 6, 2000</td><td class="patent-data-table-td patent-date-value">Jan 7, 2003</td><td class="patent-data-table-td ">Sangamo Biosciences, Inc.</td><td class="patent-data-table-td ">Detecting gene asociated with preferential phenotype; obtain preferential nucleotide sequences, transduce cells, culture cells, recover cell of preferential phenotype, monitor gene expression in recovered cells</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6534261">US6534261</a></td><td class="patent-data-table-td patent-date-value">Jan 12, 1999</td><td class="patent-data-table-td patent-date-value">Mar 18, 2003</td><td class="patent-data-table-td ">Sangamo Biosciences, Inc.</td><td class="patent-data-table-td ">Regulation of endogenous gene expression in cells using zinc finger proteins</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6599692">US6599692</a></td><td class="patent-data-table-td patent-date-value">Sep 14, 1999</td><td class="patent-data-table-td patent-date-value">Jul 29, 2003</td><td class="patent-data-table-td ">Sangamo Bioscience, Inc.</td><td class="patent-data-table-td ">Functional genomics using zinc finger proteins</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6607882">US6607882</a></td><td class="patent-data-table-td patent-date-value">Jan 6, 2000</td><td class="patent-data-table-td patent-date-value">Aug 19, 2003</td><td class="patent-data-table-td ">Sangamo Biosciences, Inc.</td><td class="patent-data-table-td ">Regulation of endogenous gene expression in cells using zinc finger proteins</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6733970">US6733970</a></td><td class="patent-data-table-td patent-date-value">May 8, 2001</td><td class="patent-data-table-td patent-date-value">May 11, 2004</td><td class="patent-data-table-td ">Gendaq Limited</td><td class="patent-data-table-td ">Screening system for zinc finger polypeptides for a desired binding ability</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6746838">US6746838</a></td><td class="patent-data-table-td patent-date-value">May 26, 1998</td><td class="patent-data-table-td patent-date-value">Jun 8, 2004</td><td class="patent-data-table-td ">Gendaq Limited</td><td class="patent-data-table-td ">Protein which is capable of binding to any predefined nucleic acid sequence</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6777185">US6777185</a></td><td class="patent-data-table-td patent-date-value">Aug 9, 2001</td><td class="patent-data-table-td patent-date-value">Aug 17, 2004</td><td class="patent-data-table-td ">Sangamo Biosciences, Inc.</td><td class="patent-data-table-td ">Functional genomics using zinc finger proteins</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6785613">US6785613</a></td><td class="patent-data-table-td patent-date-value">Mar 28, 2002</td><td class="patent-data-table-td patent-date-value">Aug 31, 2004</td><td class="patent-data-table-td ">Sangamo Biosciences, Inc.</td><td class="patent-data-table-td ">Selection of target segments within target genes using a correspondence regime between different triplets of three bases and the three possible positions of a triplet within a nine-base site; a computer program product</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6790941">US6790941</a></td><td class="patent-data-table-td patent-date-value">Feb 9, 2000</td><td class="patent-data-table-td patent-date-value">Sep 14, 2004</td><td class="patent-data-table-td ">The Scripps Research Institute</td><td class="patent-data-table-td ">Zinc finger protein derivatives and methods therefor</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6794136">US6794136</a></td><td class="patent-data-table-td patent-date-value">Nov 20, 2000</td><td class="patent-data-table-td patent-date-value">Sep 21, 2004</td><td class="patent-data-table-td ">Sangamo Biosciences, Inc.</td><td class="patent-data-table-td ">Comprises iterative cycles of rational design, site selection, redesign and site selection of the redesigned molecule</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6824978">US6824978</a></td><td class="patent-data-table-td patent-date-value">Nov 3, 2000</td><td class="patent-data-table-td patent-date-value">Nov 30, 2004</td><td class="patent-data-table-td ">Sangamo Biosciences, Inc.</td><td class="patent-data-table-td ">Regulation of endogenous gene expression in cells using zinc finger proteins</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6866997">US6866997</a></td><td class="patent-data-table-td patent-date-value">May 26, 1998</td><td class="patent-data-table-td patent-date-value">Mar 15, 2005</td><td class="patent-data-table-td ">Gendaq Limited</td><td class="patent-data-table-td ">Designing a protein which is capable of binding to any predefined nucleic acid sequence</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6903185">US6903185</a></td><td class="patent-data-table-td patent-date-value">May 13, 2002</td><td class="patent-data-table-td patent-date-value">Jun 7, 2005</td><td class="patent-data-table-td ">Massachusetts Institute Of Technology</td><td class="patent-data-table-td ">Generating chimeric zinc finger protein; obtain nucleotide sequences coding chimeric protein, transform cells, express nucleotide sequences, recover protein</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6933113">US6933113</a></td><td class="patent-data-table-td patent-date-value">Aug 28, 2001</td><td class="patent-data-table-td patent-date-value">Aug 23, 2005</td><td class="patent-data-table-td ">Sangamo Biosciences, Inc.</td><td class="patent-data-table-td ">Using recombinant zinc finger proteins; use in human therapeutic applications such as cancer therapy and altering plant phenotypes, including disease resistance, fruit ripening</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6977154">US6977154</a></td><td class="patent-data-table-td patent-date-value">Mar 17, 1999</td><td class="patent-data-table-td patent-date-value">Dec 20, 2005</td><td class="patent-data-table-td ">Gendaq Limited</td><td class="patent-data-table-td ">Nucleic acid binding proteins</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6979539">US6979539</a></td><td class="patent-data-table-td patent-date-value">Jul 2, 2001</td><td class="patent-data-table-td patent-date-value">Dec 27, 2005</td><td class="patent-data-table-td ">Sangamo Biosciences, Inc.</td><td class="patent-data-table-td ">Genetic engineering; antiinflammatory agents, rheumatic diseases</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7013219">US7013219</a></td><td class="patent-data-table-td patent-date-value">Sep 16, 2002</td><td class="patent-data-table-td patent-date-value">Mar 14, 2006</td><td class="patent-data-table-td ">Sangamo Biosciences, Inc.</td><td class="patent-data-table-td ">Regulation of endogenous gene expression in cells using zinc finger proteins</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7026462">US7026462</a></td><td class="patent-data-table-td patent-date-value">Dec 6, 2001</td><td class="patent-data-table-td patent-date-value">Apr 11, 2006</td><td class="patent-data-table-td ">Sangamo Biosciences, Inc.</td><td class="patent-data-table-td ">Regulation of angiogenesis with zinc finger proteins</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7030215">US7030215</a></td><td class="patent-data-table-td patent-date-value">Nov 20, 2001</td><td class="patent-data-table-td patent-date-value">Apr 18, 2006</td><td class="patent-data-table-td ">Sangamo Biosciences, Inc.</td><td class="patent-data-table-td ">Position dependent recognition of GNN nucleotide triplets by zinc fingers</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7067317">US7067317</a></td><td class="patent-data-table-td patent-date-value">Apr 30, 2001</td><td class="patent-data-table-td patent-date-value">Jun 27, 2006</td><td class="patent-data-table-td ">Sangamo Biosciences, Inc.</td><td class="patent-data-table-td ">Wound healing agent</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7070934">US7070934</a></td><td class="patent-data-table-td patent-date-value">Jun 5, 2003</td><td class="patent-data-table-td patent-date-value">Jul 4, 2006</td><td class="patent-data-table-td ">Sangamo Biosciences, Inc.</td><td class="patent-data-table-td ">Genetic engineered zinc finger domain; hormone receptors transcription control; polynucleotides; modulate gene expression; generation transgenic plants, animals</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7153949">US7153949</a></td><td class="patent-data-table-td patent-date-value">Apr 20, 2005</td><td class="patent-data-table-td patent-date-value">Dec 26, 2006</td><td class="patent-data-table-td ">Massachusetts Institute Of Technology</td><td class="patent-data-table-td ">Nucleic acid encoding poly-zinc finger proteins with improved linkers</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7163824">US7163824</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Aug 15, 2002</td><td class="patent-data-table-td patent-date-value">Jan 16, 2007</td><td class="patent-data-table-td ">Sangamo Biosciences, Inc.</td><td class="patent-data-table-td ">Cell containing nucleic acid molecule comprising polynucleotide sequence encoding engineered zinc finger protein operably linked to promoter, contacting target site in endogenous cellular gene with protein to inhibit transcription</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7177766">US7177766</a></td><td class="patent-data-table-td patent-date-value">Apr 2, 2001</td><td class="patent-data-table-td patent-date-value">Feb 13, 2007</td><td class="patent-data-table-td ">Sangamo Biosciences, Inc.</td><td class="patent-data-table-td ">Selection of sites for targeting by zinc finger proteins and methods of designing zinc finger proteins to bind to preselected sites</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7220719">US7220719</a></td><td class="patent-data-table-td patent-date-value">May 13, 2004</td><td class="patent-data-table-td patent-date-value">May 22, 2007</td><td class="patent-data-table-td ">Sangamo Biosciences, Inc.</td><td class="patent-data-table-td ">Using recombinant zinc finger proteins; use in human therapeutic applications such as cancer therapy and altering plant phenotypes, including disease resistance, fruit ripening</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7235354">US7235354</a></td><td class="patent-data-table-td patent-date-value">May 12, 2004</td><td class="patent-data-table-td patent-date-value">Jun 26, 2007</td><td class="patent-data-table-td ">Sangamo Biosciences, Inc.</td><td class="patent-data-table-td ">Using expressed sequence tags to correlate gene expression profiles with stress/disease phenotypes</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7241574">US7241574</a></td><td class="patent-data-table-td patent-date-value">May 24, 2004</td><td class="patent-data-table-td patent-date-value">Jul 10, 2007</td><td class="patent-data-table-td ">Gendaq Ltd.</td><td class="patent-data-table-td ">Preparing DNA binding protein comprising multiple zinc finger domains for use in identifying modulators of protein-nucleic acid recognition; gene expression inhibition</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7262054">US7262054</a></td><td class="patent-data-table-td patent-date-value">Jan 22, 2002</td><td class="patent-data-table-td patent-date-value">Aug 28, 2007</td><td class="patent-data-table-td ">Sangamo Biosciences, Inc.</td><td class="patent-data-table-td ">Zinc finger proteins for DNA binding and gene regulation in plants</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7273923">US7273923</a></td><td class="patent-data-table-td patent-date-value">Jan 22, 2002</td><td class="patent-data-table-td patent-date-value">Sep 25, 2007</td><td class="patent-data-table-td ">Sangamo Biosciences, Inc.</td><td class="patent-data-table-td ">Plant protein for use as tool in cintrolling gene expression plants</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7361635">US7361635</a></td><td class="patent-data-table-td patent-date-value">Aug 29, 2003</td><td class="patent-data-table-td patent-date-value">Apr 22, 2008</td><td class="patent-data-table-td ">Sangamo Biosciences, Inc.</td><td class="patent-data-table-td ">Simultaneous modulation of multiple genes</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7407776">US7407776</a></td><td class="patent-data-table-td patent-date-value">Sep 17, 2004</td><td class="patent-data-table-td patent-date-value">Aug 5, 2008</td><td class="patent-data-table-td ">Sangamo Biosciences, Inc.</td><td class="patent-data-table-td ">Engineered zinc finger proteins for regulation of gene expression</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7491531">US7491531</a></td><td class="patent-data-table-td patent-date-value">Jan 6, 2003</td><td class="patent-data-table-td patent-date-value">Feb 17, 2009</td><td class="patent-data-table-td ">Sangamo Biosciences, Inc.</td><td class="patent-data-table-td ">Randomized libraries of zinc finger proteins</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7560440">US7560440</a></td><td class="patent-data-table-td patent-date-value">Jul 14, 2006</td><td class="patent-data-table-td patent-date-value">Jul 14, 2009</td><td class="patent-data-table-td ">Sangamo Bioschiences, Inc.</td><td class="patent-data-table-td ">Endothelial growth factors; antiischemic agents; neovascularization</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7585849">US7585849</a></td><td class="patent-data-table-td patent-date-value">Aug 11, 2005</td><td class="patent-data-table-td patent-date-value">Sep 8, 2009</td><td class="patent-data-table-td ">Sangamo Biosciences, Inc.</td><td class="patent-data-table-td ">Position dependent recognition of GNN nucleotide triplets by zinc fingers</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7595376">US7595376</a></td><td class="patent-data-table-td patent-date-value">Dec 14, 2006</td><td class="patent-data-table-td patent-date-value">Sep 29, 2009</td><td class="patent-data-table-td ">Massachusetts Institute Of Technology</td><td class="patent-data-table-td ">Poly zinc finger proteins with improved linkers</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7605140">US7605140</a></td><td class="patent-data-table-td patent-date-value">Apr 26, 2005</td><td class="patent-data-table-td patent-date-value">Oct 20, 2009</td><td class="patent-data-table-td ">Sangamo Biosciences, Inc.</td><td class="patent-data-table-td ">Endothelial growth factors; antiischemic agents; neovascularization</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7700523">US7700523</a></td><td class="patent-data-table-td patent-date-value">Sep 1, 2006</td><td class="patent-data-table-td patent-date-value">Apr 20, 2010</td><td class="patent-data-table-td ">Gendaq Limited</td><td class="patent-data-table-td ">zinc finger motifs are randomised in overlapping regions and to smart libraries incorporating limited directed randomisation at selected position</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7705139">US7705139</a></td><td class="patent-data-table-td patent-date-value">Oct 19, 2006</td><td class="patent-data-table-td patent-date-value">Apr 27, 2010</td><td class="patent-data-table-td ">Sangamo Biosciences, Inc.</td><td class="patent-data-table-td ">Zinc finger proteins for DNA binding and gene regulation in plants</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7759059">US7759059</a></td><td class="patent-data-table-td patent-date-value">Aug 31, 2006</td><td class="patent-data-table-td patent-date-value">Jul 20, 2010</td><td class="patent-data-table-td ">Gendaq Limited, C/O Brobeck Hale And Dorr</td><td class="patent-data-table-td ">Preparing DNA binding protein comprising multiple zinc finger domains for use in identifying modulators of protein-nucleic acid recognition; gene expression inhibition</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7788044">US7788044</a></td><td class="patent-data-table-td patent-date-value">Feb 12, 2007</td><td class="patent-data-table-td patent-date-value">Aug 31, 2010</td><td class="patent-data-table-td ">Sangamo Biosciences, Inc.</td><td class="patent-data-table-td ">Selection of sites for targeting by zinc finger proteins and methods of designing zinc finger proteins to bind to preselected sites</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7851216">US7851216</a></td><td class="patent-data-table-td patent-date-value">Jul 17, 2002</td><td class="patent-data-table-td patent-date-value">Dec 14, 2010</td><td class="patent-data-table-td ">Gendaq, Ltd.</td><td class="patent-data-table-td ">Binding nucleotide sequences; connecting gaps; drug mixture with carrier</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7888121">US7888121</a></td><td class="patent-data-table-td patent-date-value">Aug 6, 2004</td><td class="patent-data-table-td patent-date-value">Feb 15, 2011</td><td class="patent-data-table-td ">Sangamo Biosciences, Inc.</td><td class="patent-data-table-td ">Cleaving cellular chromatin and inserting expression vector comprising nucleotide sequences coding zinc finger-cleavage half domain genetic exchange; genome engineering and homologous recombination</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7928195">US7928195</a></td><td class="patent-data-table-td patent-date-value">Aug 13, 2009</td><td class="patent-data-table-td patent-date-value">Apr 19, 2011</td><td class="patent-data-table-td ">Massachusetts Institute Of Technology</td><td class="patent-data-table-td ">Poly zinc finger proteins with improved linkers</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7939327">US7939327</a></td><td class="patent-data-table-td patent-date-value">Feb 28, 2008</td><td class="patent-data-table-td patent-date-value">May 10, 2011</td><td class="patent-data-table-td ">Sangamo Biosciences, Inc.</td><td class="patent-data-table-td ">Simultaneous modulation of multiple genes</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7943553">US7943553</a></td><td class="patent-data-table-td patent-date-value">Jul 12, 2006</td><td class="patent-data-table-td patent-date-value">May 17, 2011</td><td class="patent-data-table-td ">Sangamo Biosciences, Inc.</td><td class="patent-data-table-td ">Randomized libraries of zinc finger proteins</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7972854">US7972854</a></td><td class="patent-data-table-td patent-date-value">Feb 3, 2005</td><td class="patent-data-table-td patent-date-value">Jul 5, 2011</td><td class="patent-data-table-td ">Sangamo Biosciences, Inc.</td><td class="patent-data-table-td ">Methods and compositions for targeted cleavage and recombination</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7985887">US7985887</a></td><td class="patent-data-table-td patent-date-value">Aug 17, 2006</td><td class="patent-data-table-td patent-date-value">Jul 26, 2011</td><td class="patent-data-table-td ">Sangamo Biosciences, Inc.</td><td class="patent-data-table-td ">Regulation of endogenous gene expression in cells using zinc finger proteins</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8021867">US8021867</a></td><td class="patent-data-table-td patent-date-value">Oct 18, 2006</td><td class="patent-data-table-td patent-date-value">Sep 20, 2011</td><td class="patent-data-table-td ">Duke University</td><td class="patent-data-table-td ">Rationally-designed meganucleases with altered sequence specificity and DNA-binding affinity</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8071564">US8071564</a></td><td class="patent-data-table-td patent-date-value">Sep 3, 2009</td><td class="patent-data-table-td patent-date-value">Dec 6, 2011</td><td class="patent-data-table-td ">Sangamo Biosciences, Inc.</td><td class="patent-data-table-td ">Regulation of angiogenesis with zinc finger proteins</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8106255">US8106255</a></td><td class="patent-data-table-td patent-date-value">Jan 22, 2003</td><td class="patent-data-table-td patent-date-value">Jan 31, 2012</td><td class="patent-data-table-td ">Dana Carroll</td><td class="patent-data-table-td ">Targeted chromosomal mutagenasis using zinc finger nucleases</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8119361">US8119361</a></td><td class="patent-data-table-td patent-date-value">Sep 26, 2011</td><td class="patent-data-table-td patent-date-value">Feb 21, 2012</td><td class="patent-data-table-td ">Duke University</td><td class="patent-data-table-td ">Methods of cleaving DNA with rationally-designed meganucleases</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8119381">US8119381</a></td><td class="patent-data-table-td patent-date-value">Sep 26, 2011</td><td class="patent-data-table-td patent-date-value">Feb 21, 2012</td><td class="patent-data-table-td ">Duke University</td><td class="patent-data-table-td ">Rationally-designed meganucleases with altered sequence specificity and DNA-binding affinity</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8124369">US8124369</a></td><td class="patent-data-table-td patent-date-value">Sep 27, 2011</td><td class="patent-data-table-td patent-date-value">Feb 28, 2012</td><td class="patent-data-table-td ">Duke University</td><td class="patent-data-table-td ">Method of cleaving DNA with rationally-designed meganucleases</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8129134">US8129134</a></td><td class="patent-data-table-td patent-date-value">Sep 27, 2011</td><td class="patent-data-table-td patent-date-value">Mar 6, 2012</td><td class="patent-data-table-td ">Duke University</td><td class="patent-data-table-td ">Methods of cleaving DNA with rationally-designed meganucleases</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8133697">US8133697</a></td><td class="patent-data-table-td patent-date-value">Sep 27, 2011</td><td class="patent-data-table-td patent-date-value">Mar 13, 2012</td><td class="patent-data-table-td ">Duke University</td><td class="patent-data-table-td ">Methods of cleaving DNA with rationally-designed meganucleases</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8143015">US8143015</a></td><td class="patent-data-table-td patent-date-value">Sep 26, 2011</td><td class="patent-data-table-td patent-date-value">Mar 27, 2012</td><td class="patent-data-table-td ">Duke University</td><td class="patent-data-table-td ">Methods of cleaving DNA with rationally-designed meganucleases</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8143016">US8143016</a></td><td class="patent-data-table-td patent-date-value">Sep 27, 2011</td><td class="patent-data-table-td patent-date-value">Mar 27, 2012</td><td class="patent-data-table-td ">Duke University</td><td class="patent-data-table-td ">Methods of cleaving DNA with rationally-designed meganucleases</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8148098">US8148098</a></td><td class="patent-data-table-td patent-date-value">Sep 27, 2011</td><td class="patent-data-table-td patent-date-value">Apr 3, 2012</td><td class="patent-data-table-td ">Duke University</td><td class="patent-data-table-td ">Methods of cleaving DNA with rationally-designed meganucleases</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8163514">US8163514</a></td><td class="patent-data-table-td patent-date-value">Sep 27, 2011</td><td class="patent-data-table-td patent-date-value">Apr 24, 2012</td><td class="patent-data-table-td ">Duke University</td><td class="patent-data-table-td ">Methods of cleaving DNA with rationally-designed meganucleases</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8268618">US8268618</a></td><td class="patent-data-table-td patent-date-value">Aug 16, 2006</td><td class="patent-data-table-td patent-date-value">Sep 18, 2012</td><td class="patent-data-table-td ">Sangamo Biosciences, Inc.</td><td class="patent-data-table-td ">Repressing endogenous CCR5 gene expression in cells using engineered zinc finger proteins</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8304222">US8304222</a></td><td class="patent-data-table-td patent-date-value">Apr 26, 2012</td><td class="patent-data-table-td patent-date-value">Nov 6, 2012</td><td class="patent-data-table-td ">Duke University</td><td class="patent-data-table-td ">Rationally-designed meganucleases with altered sequence specificity and heterodimer formation</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8338157">US8338157</a></td><td class="patent-data-table-td patent-date-value">Sep 10, 2010</td><td class="patent-data-table-td patent-date-value">Dec 25, 2012</td><td class="patent-data-table-td ">Precision Biosciences, Inc.</td><td class="patent-data-table-td ">Rationally-designed meganuclease variants of lig-34 and I-crei for maize genome engineering</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8349810">US8349810</a></td><td class="patent-data-table-td patent-date-value">Jun 22, 2011</td><td class="patent-data-table-td patent-date-value">Jan 8, 2013</td><td class="patent-data-table-td ">Sangamo Biosciences, Inc.</td><td class="patent-data-table-td ">Methods for targeted cleavage and recombination of CCR5</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8377674">US8377674</a></td><td class="patent-data-table-td patent-date-value">Jun 25, 2012</td><td class="patent-data-table-td patent-date-value">Feb 19, 2013</td><td class="patent-data-table-td ">Duke University</td><td class="patent-data-table-td ">Method for producing genetically-modified cells with rationally-designed meganucleases with altered sequence specificity</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8383766">US8383766</a></td><td class="patent-data-table-td patent-date-value">Aug 15, 2007</td><td class="patent-data-table-td patent-date-value">Feb 26, 2013</td><td class="patent-data-table-td ">Sangamo Biosciences, Inc.</td><td class="patent-data-table-td ">Zinc finger protein for use as transcriptional modulator</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8409861">US8409861</a></td><td class="patent-data-table-td patent-date-value">Dec 15, 2005</td><td class="patent-data-table-td patent-date-value">Apr 2, 2013</td><td class="patent-data-table-td ">Sangamo Biosciences, Inc.</td><td class="patent-data-table-td ">Targeted deletion of cellular DNA sequences</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8420782">US8420782</a></td><td class="patent-data-table-td patent-date-value">Jan 31, 2012</td><td class="patent-data-table-td patent-date-value">Apr 16, 2013</td><td class="patent-data-table-td ">Ulla Bonas</td><td class="patent-data-table-td ">Modular DNA-binding domains and methods of use</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8440431">US8440431</a></td><td class="patent-data-table-td patent-date-value">Mar 22, 2012</td><td class="patent-data-table-td patent-date-value">May 14, 2013</td><td class="patent-data-table-td ">Regents Of The University Of Minnesota</td><td class="patent-data-table-td ">TAL effector-mediated DNA modification</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8440432">US8440432</a></td><td class="patent-data-table-td patent-date-value">Mar 22, 2012</td><td class="patent-data-table-td patent-date-value">May 14, 2013</td><td class="patent-data-table-td ">Regents Of The University Of Minnesota</td><td class="patent-data-table-td ">Tal effector-mediated DNA modification</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8450471">US8450471</a></td><td class="patent-data-table-td patent-date-value">Mar 22, 2012</td><td class="patent-data-table-td patent-date-value">May 28, 2013</td><td class="patent-data-table-td ">Regents Of The University Of Minnesota</td><td class="patent-data-table-td ">TAL effector-mediated DNA modification</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8470973">US8470973</a></td><td class="patent-data-table-td patent-date-value">Aug 25, 2011</td><td class="patent-data-table-td patent-date-value">Jun 25, 2013</td><td class="patent-data-table-td ">Ulla Bonas</td><td class="patent-data-table-td ">Modular DNA-binding domains and methods of use</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8524500">US8524500</a></td><td class="patent-data-table-td patent-date-value">Jul 16, 2010</td><td class="patent-data-table-td patent-date-value">Sep 3, 2013</td><td class="patent-data-table-td ">Sangamo Biosciences, Inc.</td><td class="patent-data-table-td ">Methods and compositions for targeted cleavage and recombination</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8524874">US8524874</a></td><td class="patent-data-table-td patent-date-value">Jan 16, 2013</td><td class="patent-data-table-td patent-date-value">Sep 3, 2013</td><td class="patent-data-table-td ">Sangamo Biosciences</td><td class="patent-data-table-td ">Position dependent recognition of GNN nucleotide triplets by zinc fingers</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8586363">US8586363</a></td><td class="patent-data-table-td patent-date-value">Dec 10, 2010</td><td class="patent-data-table-td patent-date-value">Nov 19, 2013</td><td class="patent-data-table-td ">Regents Of The University Of Minnesota</td><td class="patent-data-table-td ">TAL effector-mediated DNA modification</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8586526">US8586526</a></td><td class="patent-data-table-td patent-date-value">May 17, 2011</td><td class="patent-data-table-td patent-date-value">Nov 19, 2013</td><td class="patent-data-table-td ">Sangamo Biosciences, Inc.</td><td class="patent-data-table-td ">DNA-binding proteins and uses thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8617807">US8617807</a></td><td class="patent-data-table-td patent-date-value">Jul 14, 2006</td><td class="patent-data-table-td patent-date-value">Dec 31, 2013</td><td class="patent-data-table-td ">Gendaq Limited</td><td class="patent-data-table-td ">Nucleic acid binding proteins</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8680021">US8680021</a></td><td class="patent-data-table-td patent-date-value">Aug 28, 2006</td><td class="patent-data-table-td patent-date-value">Mar 25, 2014</td><td class="patent-data-table-td ">Sangamo Biosciences, Inc.</td><td class="patent-data-table-td ">Zinc finger proteins for DNA binding and gene regulation in plants</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8697853">US8697853</a></td><td class="patent-data-table-td patent-date-value">Jan 10, 2013</td><td class="patent-data-table-td patent-date-value">Apr 15, 2014</td><td class="patent-data-table-td ">Regents Of The University Of Minnesota</td><td class="patent-data-table-td ">TAL effector-mediated DNA modification</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/USRE39229">USRE39229</a></td><td class="patent-data-table-td patent-date-value">Aug 17, 1995</td><td class="patent-data-table-td patent-date-value">Aug 8, 2006</td><td class="patent-data-table-td ">Gendaq Limited</td><td class="patent-data-table-td ">libraries of DNA sequences encoding zinc finger binding motifs for display on a particle; designing zinc finger binding polypeptides for binding to a target sequence</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/USRE42150">USRE42150</a></td><td class="patent-data-table-td patent-date-value">Aug 7, 2006</td><td class="patent-data-table-td patent-date-value">Feb 15, 2011</td><td class="patent-data-table-td ">Gendaq, Limited</td><td class="patent-data-table-td ">Binding proteins for recognition of DNA</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/USRE42211">USRE42211</a></td><td class="patent-data-table-td patent-date-value">Mar 25, 2003</td><td class="patent-data-table-td patent-date-value">Mar 8, 2011</td><td class="patent-data-table-td ">Gendaq, Limited</td><td class="patent-data-table-td ">Relating to binding proteins for recognition of DNA</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2292274A1?cl=en">EP2292274A1</a></td><td class="patent-data-table-td patent-date-value">Sep 8, 2005</td><td class="patent-data-table-td patent-date-value">Mar 9, 2011</td><td class="patent-data-table-td ">Sangamo BioSciences, Inc.</td><td class="patent-data-table-td ">Compositions and methods for protein production</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2005084190A2?cl=en">WO2005084190A2</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Feb 3, 2005</td><td class="patent-data-table-td patent-date-value">Sep 15, 2005</td><td class="patent-data-table-td ">Sangamo Biosciences Inc</td><td class="patent-data-table-td ">Methods and compostions for targeted cleavage and recombination</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="classifications"></a><div class="patent-section-header"><span class="patent-section-title">Classifications</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th"> </th><th class="patent-data-table-th"> </th></tr></thead><tr><td class="patent-data-table-td ">U.S. Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=LxFHBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S199000">435/199</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=LxFHBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S091100">435/91.1</a></span></td></tr><tr><td class="patent-data-table-td ">International Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=LxFHBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0009220000">C12N9/22</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=LxFHBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12Q0001680000">C12Q1/68</a></span></td></tr><tr><td class="patent-data-table-td ">Cooperative Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=LxFHBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C07K2319/09">C07K2319/09</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=LxFHBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12Q1/683">C12Q1/683</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=LxFHBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C07K2319/81">C07K2319/81</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=LxFHBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C07K2319/60">C07K2319/60</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=LxFHBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N9/22">C12N9/22</a></span></td></tr><tr><td class="patent-data-table-td ">European Classification</td><td class="patent-data-table-td "><span class="nested-value">C12Q1/68B6A</span>, <span class="nested-value">C12N9/22</span></td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="legal-events"></a><div class="patent-section-header"><span class="patent-section-title">Legal Events</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Date</th><th class="patent-data-table-th">Code</th><th class="patent-data-table-th">Event</th><th class="patent-data-table-th">Description</th></tr></thead><tr><td class="patent-data-table-td patent-date-value">Jan 25, 2011</td><td class="patent-data-table-td ">B2</td><td class="patent-data-table-td ">Reexamination certificate second reexamination</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">CLAIMS 5-7 AND 10-11 WERE PREVIOUSLY CANCELLED. CLAIMS 1-4, 8-9, 12-14, 16-21, 23-31, 33 AND 36-42 ARE CANCELLED. NEW CLAIMS 44-49 ARE ADDED AND DETERMINED TO BE PATENTABLE. CLAIMS 15, 22, 32, 34-35 AND 43 WERE NOT REEXAMINED.</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Jan 22, 2010</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">12</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Jan 12, 2009</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">NATIONAL INSTITUTES OF HEALTH (NIH), U.S. DEPT. OF</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">CONFIRMATORY LICENSE;ASSIGNOR:JOHNS HOPKINS UNIVERSITY;REEL/FRAME:022092/0230</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">19990716</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Sep 30, 2008</td><td class="patent-data-table-td ">RR</td><td class="patent-data-table-td ">Request for reexamination filed</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20080808</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Sep 29, 2008</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">NATIONAL INSTITUTES OF HEALTH (NIH), U.S. DEPT. OF</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">EXECUTIVE ORDER 9424, CONFIRMATORY LICENSE;ASSIGNOR:JOHNS HOPKINS UNIVERSITY;REEL/FRAME:021590/0481</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">19990716</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Sep 9, 2008</td><td class="patent-data-table-td ">B1</td><td class="patent-data-table-td ">Reexamination certificate first reexamination</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">CLAIMS 5-7 AND 10-11 ARE CANCELLED. CLAIMS 1, 8, 9 AND 12 ARE DETERMINED TO BE PATENTABLE AS AMENDED. CLAIMS 3-4, 8-9, 13-14, 16-21 AND 23-24, DEPENDENT ON AN AMENDED CLAIM, ARE DETERMINED TO BE PATENTABLE. NEW CLAIMS 33-43 ARE ADDED AND DETERMINED TO BE PATENTABLE. CLAIMS 2, 15, 22 AND 25-32 WERE NOT REEXAMINED.</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">May 22, 2007</td><td class="patent-data-table-td ">RR</td><td class="patent-data-table-td ">Request for reexamination filed</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20070312</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Mar 14, 2006</td><td class="patent-data-table-td ">SULP</td><td class="patent-data-table-td ">Surcharge for late payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">7</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Mar 14, 2006</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">8</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Mar 1, 2006</td><td class="patent-data-table-td ">REMI</td><td class="patent-data-table-td ">Maintenance fee reminder mailed</td><td class="patent-data-table-td "></td></tr><tr><td class="patent-data-table-td patent-date-value">Mar 5, 2002</td><td class="patent-data-table-td ">REMI</td><td class="patent-data-table-td ">Maintenance fee reminder mailed</td><td class="patent-data-table-td "></td></tr><tr><td class="patent-data-table-td patent-date-value">Feb 8, 2002</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">4</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Sep 9, 1999</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">NATIONAL INSTITUTES OF HEALTH, THE, MARYLAND</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">CONFIRMATORY LICENSE;ASSIGNOR:JOHNS HOPKINS UNIVERSITY;REEL/FRAME:010216/0459</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">19990716</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Feb 9, 1999</td><td class="patent-data-table-td ">CC</td><td class="patent-data-table-td ">Certificate of correction</td><td class="patent-data-table-td "></td></tr><tr><td class="patent-data-table-td patent-date-value">Apr 5, 1996</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">JOHNS HOPKINS UNIVERSITY, THE, MARYLAND</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNOR:CHANDRASEGARAN, SRINIVASAN;REEL/FRAME:007994/0798</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">19960117</span></div></td></tr></table><div class="patent-section-footer"></div></div><div class="modal-dialog" id="patent-images-lightbox"><div class="patent-lightbox-controls"><div class="patent-lightbox-rotate-controls"><div class="patent-lightbox-rotation-text">Rotate</div><div class="rotate-icon rotate-ccw-icon"></div><div class="rotate-icon rotate-cw-icon"></div></div><div class="patent-lightbox-index-counter"></div><a class="patent-lightbox-fullsize-link" target="_blank">Original Image</a><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_right.png" alt="Next page"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_left.png" alt="Previous page"width="21" height="21" /></div></div><div class="modal-dialog-content"><div class="patent-lightbox-image-holder"><div class="patent-lightbox-placeholder"></div></div></div></div><script>_OC_initPatentsAtb({image_not_available_html: " Image not available"});</script></div></div></div></td></tr></table><script>(function() {var href = window.location.href;if (href.indexOf('?') !== -1) {var parameters = href.split('?')[1].split('&');for (var i = 0; i < parameters.length; i++) {var param = parameters[i].split('=');if (param[0] == 'focus') {var elem = document.getElementById(param[1]);if (elem) {elem.focus();}}}}})();</script><script>_OC_addFlags({LockSrc:"/books/javascript/lock_8a2b04e7bf975d5171d8e4c0b6365c7a.js", Host:"http://www.google.com/", IsBooksRentalEnabled:1, IsWebstoreDisplayCaseEnabled:1, IsObfuscationEnabled:1, IsBrowsingHistoryEnabled:1, IsWebReaderSvgEnabled:0, IsGeoLayerEnabled:1, IsImageModeNotesEnabled:1, IsCopyMenuItemEnabled:1, IsGiftingEnabled:0, IsWebReaderUniversalPaginatorEnabled:0, IsOfflineBubbleEnabled:1, IsReaderEnabledForPlayRequests:1, IsFutureOnSaleVolumesEnabled:1, IsOfflineRestrictedCopyEnabled:1, IsBooksUnifiedLeftNavEnabled:1, IsRestrictedCopyEnabled:1, IsZipitFolderCollectionEnabled:1, IsEndOfSampleRecommendationsEnabled:1, IsRatingsOnBookcardsEnabled:1, IsAdsDisabled:0, IsIframePageDisplayEnabled:0, IsEmbeddedMediaEnabled:1, IsImageModeAnnotationsEnabled:1, IsMyLibraryGooglePlusEnabled:1, IsImagePageProviderEnabled:0, IsBookcardListPriceSmall:0, IsInternalUser:0, IsBooksShareButtonEnabled:0, IsPreOrdersEnabled:0, IsDisabledRandomBookshelves:0, WebstoreDisplayCasePosition:3});_OC_Run({"enable_p13n":false,"add_vol_to_collection_base_url":"http://www.google.com/patents?op=add\u0026sig=ACfU3U17LdXG1NvkJEM2Qc5eZaMRZUesDw\u0026id=LxFHBAABERAJ","remove_vol_from_collection_base_url":"http://www.google.com/patents?op=remove\u0026sig=ACfU3U3uaQpcCKQ64TdfqE_wOwwErZshuQ\u0026id=LxFHBAABERAJ","logged_in":false,"p13n_save_user_settings_url":"http://www.google.com/patents?op=edit_user_settings\u0026sig=ACfU3U3m9fUDpYOGJcx--rj8SY_5BNILOg","is_cobrand":false,"sign_in_url":"https://www.google.com/accounts/Login?service=\u0026continue=http://www.google.com/patents%3Fhl%3Den\u0026hl=en","is_play_enabled":true}, {"volume_id":"","is_ebook":true,"volumeresult":{"has_flowing_text":false,"has_scanned_text":true,"can_download_pdf":false,"can_download_epub":false,"is_pdf_drm_enabled":false,"is_epub_drm_enabled":false,"download_pdf_url":"http://www.google.com/patents/download/General_method_to_clone_hybrid_restricti.pdf?id=LxFHBAABERAJ\u0026output=pdf\u0026sig=ACfU3U00hdITuSOY38XIwZ2-SV1nQYfOTw"},"sample_url":"http://www.google.com/patents/reader?id=LxFHBAABERAJ\u0026printsec=frontcover\u0026output=reader\u0026source=gbs_atb_hover","is_browsable":true,"is_public_domain":true}, {});</script><div id="footer_table" style="font-size:83%;text-align:center;position:relative;top:20px;height:4.5em;margin-top:2em"><div style="margin-bottom:8px"><a href=http://www.google.com/><nobr>Google&nbsp;Home</nobr></a> - <a href=//www.google.com/patents/sitemap/><nobr>Sitemap</nobr></a> - <a href=http://www.google.com/googlebooks/uspto.html><nobr>USPTO Bulk Downloads</nobr></a> - <a href=/intl/en/privacy/><nobr>Privacy Policy</nobr></a> - <a href=/intl/en/policies/terms/><nobr>Terms of Service</nobr></a> - <a href=https://support.google.com/faqs/answer/2539193?hl=en><nobr>About Google Patents</nobr></a> - <a href="http://www.google.com/tools/feedback/intl/en/error.html" onclick="try{_OC_startFeedback({productId: '72792',locale: 'en'});return false;}catch(e){}"><nobr>Send Feedback</nobr></a></div><span>Data provided by IFI CLAIMS Patent Services</span><br><span >&copy;2012 Google</span></div> <script type="text/javascript">var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));</script><script type="text/javascript">var pageTracker = _gat._getTracker("UA-27188110-1");pageTracker._setCookiePath("/patents/");pageTracker._trackPageview();</script> </body></html>