<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN""http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html><head><title>Patent US5795716 - Analyzing nucleic acid sequences - Google Patents</title><script>(function(){(function(){function e(a){this.t={};this.tick=function(a,c,b){var d=void 0!=b?b:(new Date).getTime();this.t[a]=[d,c];if(void 0==b)try{window.console.timeStamp("CSI/"+a)}catch(e){}};this.tick("start",null,a)}var a;window.performance&&(a=window.performance.timing);var f=a?new e(a.responseStart):new e;window.jstiming={Timer:e,load:f};if(a){var c=a.navigationStart,d=a.responseStart;0<c&&d>=c&&(window.jstiming.srt=d-c)}if(a){var b=window.jstiming.load;0<c&&d>=c&&(b.tick("_wtsrt",void 0,c),b.tick("wtsrt_",
"_wtsrt",d),b.tick("tbsd_","wtsrt_"))}try{a=null,window.chrome&&window.chrome.csi&&(a=Math.floor(window.chrome.csi().pageT),b&&0<c&&(b.tick("_tbnd",void 0,window.chrome.csi().startE),b.tick("tbnd_","_tbnd",c))),null==a&&window.gtbExternal&&(a=window.gtbExternal.pageT()),null==a&&window.external&&(a=window.external.pageT,b&&0<c&&(b.tick("_tbnd",void 0,window.external.startE),b.tick("tbnd_","_tbnd",c))),a&&(window.jstiming.pt=a)}catch(g){}})();})();
</script><link rel="stylesheet" href="/patents/css/_8a2b04e7bf975d5171d8e4c0b6365c7a/kl_intl_patents_bundle.css" type="text/css" /><script src="/books/javascript/atb_8a2b04e7bf975d5171d8e4c0b6365c7a__en.js"></script><script>function googleTranslateElementInit() {new google.translate.TranslateElement({pageLanguage: "en",gaTrack: true,gaId: "UA-27188110-1",multilanguagePage: true});}</script><script src="//translate.google.com/translate_a/element.js?cb=googleTranslateElementInit"></script><meta name="DC.type" content="Patent"><meta name="DC.title" content="Analyzing nucleic acid sequences"><meta name="DC.contributor" content="Mark S. Chee" scheme="inventor"><meta name="DC.contributor" content="Chee; Mark S." scheme="assignee"><meta name="DC.date" content="1994-10-21" scheme="dateSubmitted"><meta name="DC.description" content="A computer system for analyzing nucleic acid sequences is provided. The computer system is used to perform multiple methods for determining unknown bases by analyzing the fluorescence intensities of hybridized nucleic acid probes. The results of individual experiments are improved by processing nucleic acid sequences together. Comparative analysis of multiple experiments is also provided by displaying reference sequences in one area and sample sequences in another area on a display device."><meta name="DC.date" content="1998-8-18" scheme="issued"><meta name="DC.relation" content="US:4741043" scheme="references"><meta name="DC.relation" content="US:4965725" scheme="references"><meta name="DC.relation" content="US:5002867" scheme="references"><meta name="DC.relation" content="US:5202231" scheme="references"><meta name="DC.relation" content="US:5273632" scheme="references"><meta name="DC.relation" content="US:5384261" scheme="references"><meta name="DC.relation" content="US:5445934" scheme="references"><meta name="DC.relation" content="US:5470710" scheme="references"><meta name="DC.relation" content="US:5492806" scheme="references"><meta name="DC.relation" content="US:5525464" scheme="references"><meta name="DC.relation" content="US:5527681" scheme="references"><meta name="DC.relation" content="WO:1989010977:A1" scheme="references"><meta name="DC.relation" content="WO:1992010588:A1" scheme="references"><meta name="DC.relation" content="WO:1995011995:A1" scheme="references"><meta name="DC.relation" content="WO:1995035505:A1" scheme="references"><meta name="citation_reference" content="Drmanac et al. An algorithm for the DNA Sequence Generation form k Tuple Word Contents of the Minimal Number of Random Fragments. J. of Biomolecular Structure &amp; Dynamics 8: 1085 1102, 1991."><meta name="citation_reference" content="Drmanac et al. An algorithm for the DNA Sequence Generation form k-Tuple Word Contents of the Minimal Number of Random Fragments. J. of Biomolecular Structure &amp; Dynamics 8: 1085-1102, 1991."><meta name="citation_reference" content="Drmanac et al. DNA Sequence Determination by hybridization:A strategy for Effeicient Large Scale Sequencing. Science: 260: 1649 1652, 1993."><meta name="citation_reference" content="Drmanac et al. DNA Sequence Determination by hybridization:A strategy for Effeicient Large Scale Sequencing. Science: 260: 1649-1652, 1993."><meta name="citation_reference" content="Southern et al. Analyzing and Comparing Nucleic Acid Sequences by Hybridization to Arrays of Oligonucleotides: Evaluation using Experimentals Models. Genomics 13: 1008 1017, 1992."><meta name="citation_reference" content="Southern et al. Analyzing and Comparing Nucleic Acid Sequences by Hybridization to Arrays of Oligonucleotides: Evaluation using Experimentals Models. Genomics 13: 1008-1017, 1992."><meta name="citation_reference" content="Strezoska et al. DNA sequencing by Hybridization : 100 bases read by a non gel based method. PNAS 88: 10089 10093, 1991."><meta name="citation_reference" content="Strezoska et al. DNA sequencing by Hybridization : 100 bases read by a non-gel based method. PNAS 88: 10089-10093, 1991."><meta name="citation_patent_number" content="US:5795716"><meta name="citation_patent_application_number" content="US:08/327,525"><link rel="canonical" href="http://www.google.com/patents/US5795716"/><meta property="og:url" content="http://www.google.com/patents/US5795716"/><meta name="title" content="Patent US5795716 - Analyzing nucleic acid sequences"/><meta name="description" content="A computer system for analyzing nucleic acid sequences is provided. The computer system is used to perform multiple methods for determining unknown bases by analyzing the fluorescence intensities of hybridized nucleic acid probes. The results of individual experiments are improved by processing nucleic acid sequences together. Comparative analysis of multiple experiments is also provided by displaying reference sequences in one area and sample sequences in another area on a display device."/><meta property="og:title" content="Patent US5795716 - Analyzing nucleic acid sequences"/><meta property="og:type" content="book"/><meta property="og:site_name" content="Google Books"/><meta property="og:image" content="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><link rel="image_src" href="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><script>(function(){try{var aa=function(a,b,c,d){d=d||{};d._sn=["cfg",b,c].join(".");window.gbar.logger.ml(a,d)};var g=window.gbar=window.gbar||{},l=window.gbar.i=window.gbar.i||{},m={},n;function _tvn(a,b){var c=parseInt(a,10);return isNaN(c)?b:c}function _tvf(a,b){var c=parseFloat(a);return isNaN(c)?b:c}function _tvv(a){return!!a}function p(a,b,c){(c||g)[a]=b}g.bv={n:_tvn("2",0),r:"",f:".67.",e:"0",m:_tvn("0",1)};
function q(a,b,c){var d="on"+b;if(a.addEventListener)a.addEventListener(b,c,!1);else if(a.attachEvent)a.attachEvent(d,c);else{var f=a[d];a[d]=function(){var a=f.apply(this,arguments),b=c.apply(this,arguments);return void 0==a?b:void 0==b?a:b&&a}}}var s=function(a){return function(){return g.bv.m==a}},ba=s(1),ca=s(2);p("sb",ba);p("kn",ca);l.a=_tvv;l.b=_tvf;l.c=_tvn;l.i=aa;var da=window.gbar.i.i;var t,u,v,w;function ea(a){v=a}function fa(a){var b;if(b=v&&window.encodeURIComponent)b=a.href,b=!b.match(/^http[s]?:\/\/accounts\.google\.[^/]*\/ClearSID/i)&&!b.match(/^http[s]?:\/\/[^/]*\/accounts\/ClearSID/i);if(b=b&&encodeURIComponent(v()))a.href=a.href.replace(/([?&]continue=)[^&]*/,"$1"+b)}function ga(a){window.gApplication&&(a.href=window.gApplication.getTabUrl(a.href))}
function ha(a){var b=document.forms[0].q,c=window.encodeURIComponent&&b&&b.value,b=b&&b.placeholder;c&&c!=b&&(a.href=a.href.replace(/([?&])q=[^&]*|$/,function(a,b){return(b||"&")+"q="+encodeURIComponent(c)}))}n=l.a("")?ga:ha;
function x(a,b,c,d,f,e){var h=document.getElementById(a);if(h){var k=h.style;k.left=d?"auto":b+"px";k.right=d?b+"px":"auto";k.top=c+"px";k.visibility=u?"hidden":"visible";f&&e?(k.width=f+"px",k.height=e+"px"):(x(t,b,c,d,h.offsetWidth,h.offsetHeight),u=u?"":a)}}
var y=[],ia=function(a,b){y.push(b)},ja=function(a){a=a||window.event;var b=a.target||a.srcElement;a.cancelBubble=!0;null==t&&(a=document.createElement(Array.every||window.createPopup?"iframe":"div"),a.frameBorder="0",t=a.id="gbs",a.src="javascript:''",b.parentNode.appendChild(a),q(document,"click",z));var c=b,b=0;"gb3"!=c.className&&(c=c.parentNode);a=c.getAttribute("aria-owns")||"gbi";var d=c.offsetWidth,f=20<c.offsetTop?46:24;document.getElementById("tphdr")&&(f-=3);var e=!1;do b+=c.offsetLeft||
0;while(c=c.offsetParent);var c=(document.documentElement.clientWidth||document.body.clientWidth)-b-d,h,d=document.body,k=document.defaultView;k&&k.getComputedStyle?(d=k.getComputedStyle(d,""))&&(h=d.direction):h=d.currentStyle?d.currentStyle.direction:d.style.direction;h="rtl"==h;if("gbi"==a){for(d=0;k=y[d++];)k();A(null,window.navExtra);h&&(b=c,e=!0)}else h||(b=c,e=!0);u!=a&&z();x(a,b,f,e)},z=function(){u&&x(u,0,0)},A=function(a,b){var c,d=document.getElementById("gbi"),f=a;f||(f=d.firstChild);
for(;b&&(c=b.pop());){var e=d,h=c,k=f;w||(w="gb2");e.insertBefore(h,k).className=w}},ka=function(a,b,c){if((b=document.getElementById(b))&&a){a.className="gb4";var d=document.createElement("span");d.appendChild(a);d.appendChild(document.createTextNode(" | "));d.id=c;b.appendChild(d)}},la=function(){return document.getElementById("gb_70")},ma=function(){return!!u};p("qs",n);p("setContinueCb",ea);p("pc",fa);p("tg",ja);p("close",z);p("addLink",ka);p("almm",A);p("si",la);p("adh",ia);p("op",ma);var B=function(){},C=function(){},F=function(a){var b=new Image,c=D;b.onerror=b.onload=b.onabort=function(){try{delete E[c]}catch(a){}};E[c]=b;b.src=a;D=c+1},E=[],D=0;p("logger",{il:C,ml:B,log:F});var G=window.gbar.logger;var H={},na={},I=[],oa=l.b("0.1",.1),pa=l.a("1",!0),qa=function(a,b){I.push([a,b])},ra=function(a,b){H[a]=b},sa=function(a){return a in H},J={},K=function(a,b){J[a]||(J[a]=[]);J[a].push(b)},ta=function(a){K("m",a)},L=function(a,b){var c=document.createElement("script");c.src=a;c.async=pa;Math.random()<oa&&(c.onerror=function(){c.onerror=null;B(Error("Bundle load failed: name="+(b||"UNK")+" url="+a))});(document.getElementById("xjsc")||document.getElementsByTagName("body")[0]||
document.getElementsByTagName("head")[0]).appendChild(c)},N=function(a){for(var b=0,c;(c=I[b])&&c[0]!=a;++b);!c||c[1].l||c[1].s||(c[1].s=!0,M(2,a),c[1].url&&L(c[1].url,a),c[1].libs&&m.d&&m.d(c[1].libs))},O=function(a){K("gc",a)},P=null,ua=function(a){P=a},M=function(a,b,c){if(P){a={t:a,b:b};if(c)for(var d in c)a[d]=c[d];try{P(a)}catch(f){}}};p("mdc",H);p("mdi",na);p("bnc",I);p("qGC",O);p("qm",ta);p("qd",J);p("lb",N);p("mcf",ra);p("bcf",qa);p("aq",K);p("mdd","");p("has",sa);
p("trh",ua);p("tev",M);var Q=l.b("0.1",.001),R=0;
function _mlToken(a,b){try{if(1>R){R++;var c,d=a,f=b||{},e=encodeURIComponent,h=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&jexpid=",e("17483"),"&srcpg=",e("prop=22"),"&jsr=",Math.round(1/Q),"&ogev=",e("7qXtU-3iEMOCogTCz4GQCg"),"&ogf=",g.bv.f,"&ogrp=",e("1"),"&ogv=",e("1407723702.0"),"&oggv="+e("es_plusone_gc_20140723.0_p0"),"&ogd=",e("com"),"&ogc=",e("NZL"),"&ogl=",e("en")];f._sn&&(f._sn="og."+
f._sn);for(var k in f)h.push("&"),h.push(e(k)),h.push("="),h.push(e(f[k]));h.push("&emsg=");h.push(e(d.name+":"+d.message));var r=h.join("");S(r)&&(r=r.substr(0,2E3));c=r;var Aa=window.gbar.logger._aem(a,c);F(Aa)}}catch(Na){}}var S=function(a){return 2E3<=a.length},va=function(a,b){return b};function T(a){B=a;p("_itl",S,G);p("_aem",va,G);p("ml",B,G);a={};H.er=a}l.a("")?T(function(a){throw a;}):l.a("1")&&Math.random()<Q&&T(_mlToken);I.push(["m",{url:"//ssl.gstatic.com/gb/js/scm_7385cc5883250b43a39405734c1bea59.js"}]);g.mcf("c",{});g.sg={c:""};if(l.a("1")){var wa=l.a("");I.push(["gc",{auto:wa,url:"//ssl.gstatic.com/gb/js/abc/gci_91f30755d6a6b787dcc2a4062e6e9824.js",libs:"googleapis.client:plusone:gapi.iframes"}]);var xa={version:"gci_91f30755d6a6b787dcc2a4062e6e9824.js",index:"",lang:"en"};H.gc=xa;var U=function(a){window.googleapis&&window.iframes?a&&a():(a&&O(a),N("gc"))};p("lGC",U);l.a("1")&&p("lPWF",U)};window.__PVT="";if(l.a("1")&&l.a("1")){var V=function(a){U(function(){K("pw",a);N("pw")})};p("lPW",V);I.push(["pw",{url:"//ssl.gstatic.com/gb/js/abc/pwm_45f73e4df07a0e388b0fa1f3d30e7280.js"}]);var W=[],ya=function(a){W[0]=a},za=function(a,b){var c=b||{};c._sn="pw";B(a,c)},Ba={signed:W,elog:za,base:"https://plusone.google.com/u/0",loadTime:(new Date).getTime()};H.pw=Ba;var X=function(a,b){for(var c=b.split("."),d=function(){var b=arguments;a(function(){for(var a=g,d=0,e=c.length-1;d<e;++d)a=a[c[d]];a[c[d]].apply(a,b)})},f=g,e=0,h=c.length-1;e<h;++e)f=
f[c[e]]=f[c[e]]||{};return f[c[e]]=d};X(V,"pw.clk");X(V,"pw.hvr");p("su",ya,g.pw)};function Ca(){function a(){for(var b;(b=e[h++])&&"m"!=b[0]&&!b[1].auto;);b&&(M(2,b[0]),b[1].url&&L(b[1].url,b[0]),b[1].libs&&m.d&&m.d(b[1].libs));h<e.length&&setTimeout(a,0)}function b(){0<f--?setTimeout(b,0):a()}var c=l.a("1"),d=l.a(""),f=3,e=I,h=0,k=window.gbarOnReady;if(k)try{k()}catch(r){da(r,"ml","or")}d?p("ldb",a):c?q(window,"load",b):b()}p("rdl",Ca);var Da={D:1,H:2,da:3,p:4,W:5,M:6,F:7,g:8,ha:9,U:10,L:11,T:12,S:13,N:14,Q:15,P:16,fa:17,w:18,O:19,ga:20,ea:21,u:22,G:23,ja:24,ka:25,ia:26,A:27,j:28,o:29,k:30,ca:31,Z:32,$:33,J:34,K:35,ba:36,aa:37,Y:38,B:39,R:40,v:41,X:42,V:43,h:48,C:49,I:500},Y=[1,2,3,4,5,6,9,10,11,13,14,28,29,30,34,35,37,38,39,40,41,42,43,48,49,500];var Z=l.b("0.001",1E-4),Ea=l.b("1",1),Fa=!1,Ga=!1;if(l.a("1")){var Ha=Math.random();Ha<=Z&&(Fa=!0);Ha<=Ea&&(Ga=!0)}var Ia=Da,$=null;function Ja(){var a=0,b=function(b,d){l.a(d)&&(a|=b)};b(1,"");b(2,"");b(4,"");b(8,"");return a}
function Ka(a,b){var c=Z,d=Fa,f;f=a;if(!$){$={};for(var e=0;e<Y.length;e++){var h=Y[e];$[h]=!0}}if(f=!!$[f])c=Ea,d=Ga;if(d){d=encodeURIComponent;g.rp?(f=g.rp(),f="-1"!=f?f:"1"):f="1";c=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&oge=",a,"&ogex=",d("17483"),"&ogev=",d("7qXtU-3iEMOCogTCz4GQCg"),"&ogf=",g.bv.f,"&ogp=",d("22"),"&ogrp=",d(f),"&ogsr=",Math.round(1/c),"&ogv=",d("1407723702.0"),"&oggv="+
d("es_plusone_gc_20140723.0_p0"),"&ogd=",d("com"),"&ogl=",d("en"),"&ogc=",d("NZL"),"&ogus=",Ja()];if(b){"ogw"in b&&(c.push("&ogw="+b.ogw),delete b.ogw);var k;f=b;e=[];for(k in f)0!=e.length&&e.push(","),e.push(La(k)),e.push("."),e.push(La(f[k]));k=e.join("");""!=k&&(c.push("&ogad="),c.push(d(k)))}F(c.join(""))}}function La(a){"number"==typeof a&&(a+="");return"string"==typeof a?a.replace(".","%2E").replace(",","%2C"):a}C=Ka;p("il",C,G);var Ma={};H.il=Ma;setTimeout(function(){C(Ia.g)},0);}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var b=window.gbar.i.i;var c=window.gbar;var f=function(d){try{var a=document.getElementById("gbom");a&&d.appendChild(a.cloneNode(!0))}catch(e){b(e,"omas","aomc")}};c.aomc=f;}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var a=window.gbar;a.mcf("pm",{p:""});}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{window.gbar.rdl();}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
if (window['_OC_timingAction']) {window['_OC_timingAction']('patents_refpage');}</script><style>#gbar,#guser{font-size:13px;padding-top:1px !important;}#gbar{float:left;height:22px}#guser{padding-bottom:7px !important;text-align:right}.gbh,.gbd{border-top:1px solid #c9d7f1;font-size:1px}.gbh{height:0;position:absolute;top:24px;width:100%}#gbs,.gbm{background:#fff;left:0;position:absolute;text-align:left;visibility:hidden;z-index:1000}.gbm{border:1px solid;border-color:#c9d7f1 #36c #36c #a2bae7;z-index:1001}.gb1{margin-right:.5em}.gb1,.gb3{zoom:1}.gb2{display:block;padding:.2em .5em}.gb2,.gb3{text-decoration:none !important;border-bottom:none}a.gb1,a.gb4{text-decoration:underline !important}a.gb1,a.gb2,a.gb3,a.gb4{color:#00c !important}.gbi .gb3,.gbi .gb2,.gbi .gb4{color:#dd8e27 !important}.gbf .gb3,.gbf .gb2,.gbf .gb4{color:#900 !important}a.gb2:hover{background:#36c;color:#fff !important}#gbar .gbz0l{color:#000 !important;cursor:default;font-weight:bold;text-decoration:none !important}
#gbar { padding:.3em .6em !important;}</style></head><body  topmargin="3" marginheight="3"><div id=gbar><nobr><a onclick=gbar.qs(this);gbar.logger.il(1,{t:1}); class=gb1 id=gb_1 href="https://www.google.com/search?sa=N&tab=tw">Search</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:2}); class=gb1 id=gb_2 href="http://www.google.com/search?hl=en&tbm=isch&source=og&sa=N&tab=ti">Images</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:8}); class=gb1 id=gb_8 href="http://maps.google.com/maps?hl=en&sa=N&tab=tl">Maps</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:78}); class=gb1 id=gb_78 href="https://play.google.com/?hl=en&sa=N&tab=t8">Play</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:36}); class=gb1 id=gb_36 href="http://www.youtube.com/results?sa=N&tab=t1">YouTube</a> <a onclick=gbar.logger.il(1,{t:5}); class=gb1 id=gb_5 href="http://news.google.com/nwshp?hl=en&tab=tn">News</a> <a onclick=gbar.logger.il(1,{t:23}); class=gb1 id=gb_23 href="https://mail.google.com/mail/?tab=tm">Gmail</a> <a onclick=gbar.logger.il(1,{t:25}); class=gb1 id=gb_25 href="https://drive.google.com/?tab=to">Drive</a> <a class=gb3 href="http://www.google.com/intl/en/options/" onclick="this.blur();gbar.tg(event);return !1" aria-haspopup=true><u>More</u> <small>&#9660;</small></a><div class=gbm id=gbi><a onclick=gbar.logger.il(1,{t:24}); class=gb2 id=gb_24 href="https://www.google.com/calendar?tab=tc">Calendar</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:51}); class=gb2 id=gb_51 href="http://translate.google.com/?hl=en&sa=N&tab=tT">Translate</a><a onclick=gbar.logger.il(1,{t:17}); class=gb2 id=gb_17 href="http://www.google.com/mobile/?hl=en&tab=tD">Mobile</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:10}); class=gb2 id=gb_10 href="http://www.google.com/search?hl=en&tbo=u&tbm=bks&source=og&sa=N&tab=tp">Books</a><a onclick=gbar.logger.il(1,{t:212}); class=gb2 id=gb_212 href="https://wallet.google.com/manage/?tab=ta">Wallet</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:6}); class=gb2 id=gb_6 href="http://www.google.com/search?hl=en&tbo=u&tbm=shop&source=og&sa=N&tab=tf">Shopping</a><a onclick=gbar.logger.il(1,{t:30}); class=gb2 id=gb_30 href="http://www.blogger.com/?tab=tj">Blogger</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:27}); class=gb2 id=gb_27 href="http://www.google.com/finance?sa=N&tab=te">Finance</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:31}); class=gb2 id=gb_31 href="https://plus.google.com/photos?sa=N&tab=tq">Photos</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:12}); class=gb2 id=gb_12 href="http://www.google.com/search?hl=en&tbo=u&tbm=vid&source=og&sa=N&tab=tv">Videos</a><div class=gb2><div class=gbd></div></div><a onclick=gbar.logger.il(1,{t:66}); href="http://www.google.com/intl/en/options/" class=gb2>Even more &raquo;</a></div></nobr></div><div id=guser width=100%><nobr><span id=gbn class=gbi></span><span id=gbf class=gbf></span><span id=gbe></span><a target=_top id=gb_70 href="https://www.google.com/accounts/Login?service=&continue=http://www.google.com/patents%3Fhl%3Den&hl=en" class=gb4>Sign in</a><div style="display: none"><div class=gbm id=gbd5 aria-owner=gbg5><div class=gbmc><ol id=gbom class=gbmcc></ol></div></div></div></nobr></div><div class=gbh style=left:0></div><div class=gbh style=right:0></div><div role="alert" style="position: absolute; left: 0; right: 0;"><a href="http://www.google.com/patents/us5795716?hl=en&amp;output=html_text" title="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."><img border="0" src="http://www.google.com/images/cleardot.gif"alt="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."></a></div><div id="guser"><nobr></nobr></div><div style="clear:both;"></div><div id="gb-top-search-box" class="gb-top-search-box-small gb-reset"><table><tr><td class="logo"><a href="http://www.google.com/patents" class="logo-link"><img class="logo-img" src="/intl/en/images/logos/google_logo_41.png" alt="Go to Google Books Home" height="41"/></a></td><td><form action="http://www.google.com/search" name="f" id="vheadf" method="get"><span id="hf"></span><input type="hidden" name="tbm" value="pts"/><input type="hidden" name="tbo" value="1"/><input type="hidden" name="hl" value="en"/><table><tr><td><div class="inputs"><table><tr><td><div class="text-input"><input type="text" name="q" id="vheadq" class="text" maxlength="2048" size="31" value="" title="Search Patents" accesskey="s" autocomplete="off"/><script>window._OC_autoDir &&window._OC_autoDir('vheadq', 'tia-vheadq');</script></div></td><td><div class="submit-input"><input name="btnG" class="submit" type="submit" value=""/></div></td></tr></table></div></td><td class="col-ext-links"><div class="ext-links"><a href="http://www.google.com/advanced_patent_search">&lt;nobr&gt;Advanced Patent Search&lt;/nobr&gt;</a></div></td></tr></table></form></td></tr></table></div><div class="kd-appbar"><h2 class="kd-appname"><a href="/patents">Patents</a></h2><div class="kd-buttonbar left" id="left-toolbar-buttons"><a id="appbar-write-review-link" href=""></a><a id="appbar-view-print-sample-link" href=""></a><a id="appbar-view-ebook-sample-link" href=""></a><a id="appbar-patents-prior-art-finder-link" href="https://www.google.com/patents/related/US5795716"></a><a id="appbar-patents-discuss-this-link" href="http://www.google.com/url?id=JI1HBAABERAJ&amp;q=http://patents.stackexchange.com/redirect/google-patents%3Fpatent%3DUS5795716&amp;usg=AFQjCNEUfgvNbqqyDhOVdphdGKqqvAkF7Q" data-is-grant="true"></a><a id="appbar-read-patent-link" href="//docs.google.com/viewer?url=patentimages.storage.googleapis.com/pdfs/US5795716.pdf"></a><a id="appbar-download-pdf-link" href="//patentimages.storage.googleapis.com/pdfs/US5795716.pdf"></a></div><div class="kd-buttonbar right" id="right-toolbar-buttons"></div></div><div id="books-microdata" itemscope=""itemtype="http://schema.org/Book"itemid="http://www.google.com/patents/US5795716" style="display:none"><span itemprop="description">A computer system for analyzing nucleic acid sequences is provided. The computer system is used to perform multiple methods for determining unknown bases by analyzing the fluorescence intensities of hybridized nucleic acid probes. The results of individual experiments are improved by processing nucleic...</span><span itemprop="url">http://www.google.com/patents/US5795716?utm_source=gb-gplus-share</span><span class="main-title" itemprop="name">Patent US5795716 - Analyzing nucleic acid sequences</span><img itemprop="image" src="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"alt="Patent US5795716 - Analyzing nucleic acid sequences" title="Patent US5795716 - Analyzing nucleic acid sequences"></div><div style="display: none"><ol id="ofe-gear-menu-contents" class="gbmcc"><li class="gbe gbmtc"><a class="gbmt goog-menuitem-content" id="" href="http://www.google.com/advanced_patent_search">Advanced Patent Search</a></li></ol></div><table id="viewport_table" cellpadding="0" style="clear:both" cellspacing="0"><tr><td id="viewport_td"><div class=vertical_module_list_row><div id=intl_patents class=about_content><div id=intl_patents_v><table class="patent-bibdata"><tr><td class="patent-bibdata-heading">Publication number</td><td class="single-patent-bibdata">US5795716 A</td></tr><tr><td class="patent-bibdata-heading">Publication type</td><td class="single-patent-bibdata">Grant</td></tr><tr><td class="patent-bibdata-heading">Application number</td><td class="single-patent-bibdata">US 08/327,525</td></tr><tr><td class="patent-bibdata-heading">Publication date</td><td class="single-patent-bibdata">Aug 18, 1998</td></tr><tr><td class="patent-bibdata-heading">Filing date</td><td class="single-patent-bibdata">Oct 21, 1994</td></tr><tr><td class="patent-bibdata-heading">Priority date<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The priority date is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the date listed."></span></td><td class="single-patent-bibdata">Oct 21, 1994</td></tr><tr><td class="patent-bibdata-heading">Fee status<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The fee status is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the status or dates listed."></span></td><td class="single-patent-bibdata">Paid</td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Also published as</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/EP0717113A2">EP0717113A2</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP0717113A3">EP0717113A3</a>, </span><span class="patent-bibdata-value"><a href="/patents/US5974164">US5974164</a>, </span><span class="patent-bibdata-value"><a href="/patents/US6242180">US6242180</a>, </span><span class="patent-bibdata-value"><a href="/patents/US6607887">US6607887</a>, </span><span class="patent-bibdata-value"><a href="/patents/US6733964">US6733964</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20020012925">US20020012925</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20050186581">US20050186581</a></span></span></td></tr><tr class="patent-bibdata-list-row alternate-patent-number"><td class="patent-bibdata-heading">Publication number</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value">08327525, </span><span class="patent-bibdata-value">327525, </span><span class="patent-bibdata-value">US 5795716 A, </span><span class="patent-bibdata-value">US 5795716A, </span><span class="patent-bibdata-value">US-A-5795716, </span><span class="patent-bibdata-value">US5795716 A, </span><span class="patent-bibdata-value">US5795716A</span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Inventors</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Mark+S.+Chee%22">Mark S. Chee</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Original Assignee</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=inassignee:%22Chee%3B+Mark+S.%22">Chee; Mark S.</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Export Citation</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/US5795716.bibtex">BiBTeX</a>, </span><span class="patent-bibdata-value"><a href="/patents/US5795716.enw">EndNote</a>, </span><span class="patent-bibdata-value"><a href="/patents/US5795716.ris">RefMan</a></span></span></td></tr><tr class="patent-internal-links"><td colspan=2><span class="patent-bibdata-value"><a href="#backward-citations">Patent Citations</a> (15),</span> <span class="patent-bibdata-value"><a href="#npl-citations">Non-Patent Citations</a> (8),</span> <span class="patent-bibdata-value"><a href="#forward-citations">Referenced by</a> (146),</span> <span class="patent-bibdata-value"><a href="#classifications">Classifications</a> (46),</span> <span class="patent-bibdata-value"><a href="#legal-events">Legal Events</a> (11)</span> </td></tr><tr><td colspan=2 class="patent-bibdata-external-link-spacer-top"></td></tr><tr class="patent-bibdata-external-link-spacer-bottom"></tr><tr><td colspan=2><span class="patent-bibdata-heading">External Links:&nbsp;</span><span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=JI1HBAABERAJ&q=http://patft.uspto.gov/netacgi/nph-Parser%3FSect2%3DPTO1%26Sect2%3DHITOFF%26p%3D1%26u%3D/netahtml/PTO/search-bool.html%26r%3D1%26f%3DG%26l%3D50%26d%3DPALL%26RefSrch%3Dyes%26Query%3DPN/5795716&usg=AFQjCNFOfNhj-0xulTCJvpIUfJbkumVBjg">USPTO</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=JI1HBAABERAJ&q=http://assignments.uspto.gov/assignments/q%3Fdb%3Dpat%26pat%3D5795716&usg=AFQjCNHOU1a3f9p3Ftgvwb0nY4zHjWa0VQ">USPTO Assignment</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=JI1HBAABERAJ&q=http://worldwide.espacenet.com/publicationDetails/biblio%3FCC%3DUS%26NR%3D5795716A%26KC%3DA%26FT%3DD&usg=AFQjCNFTT_Y9UlgzMWOV3aAfjuBfNOKc-g">Espacenet</a></span></span></td></tr><tr class="patent-bibdata-group-spacer"></tr></table><div class="number-and-title"><span class="patent-title"><invention-title mxw-id="PT129066614" lang="EN" load-source="ifi">Analyzing nucleic acid sequences</invention-title></span><br><span class="patent-number">US 5795716 A</span></div><div class="patent-section patent-abstract-section"><div class="patent-section-header"><span class="patent-section-title">Abstract</span></div><div class="patent-text"><abstract mxw-id="PA37804281" lang="EN" load-source="patent-office"> <div class="abstract">A computer system for analyzing nucleic acid sequences is provided. The computer system is used to perform multiple methods for determining unknown bases by analyzing the fluorescence intensities of hybridized nucleic acid probes. The results of individual experiments are improved by processing nucleic acid sequences together. Comparative analysis of multiple experiments is also provided by displaying reference sequences in one area and sample sequences in another area on a display device.</div>
  </abstract></div></div><div class="patent-section patent-drawings-section"><div class="patent-section-header"><span class="patent-section-title">Images<span class="patent-section-count">(26)</span></span></div><div class="patent-drawings-body"><div class="patent-drawings-carousel"><div class="drawings"><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5795716-1.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5795716-1.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5795716-2.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5795716-2.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5795716-3.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5795716-3.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5795716-4.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5795716-4.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5795716-5.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5795716-5.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5795716-6.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5795716-6.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5795716-7.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5795716-7.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5795716-8.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5795716-8.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5795716-9.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5795716-9.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5795716-10.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5795716-10.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5795716-11.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5795716-11.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5795716-12.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5795716-12.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5795716-13.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5795716-13.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5795716-14.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5795716-14.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5795716-15.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5795716-15.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5795716-16.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5795716-16.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5795716-17.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5795716-17.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5795716-18.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5795716-18.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5795716-19.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5795716-19.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5795716-20.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5795716-20.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5795716-21.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5795716-21.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5795716-22.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5795716-22.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5795716-23.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5795716-23.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5795716-24.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5795716-24.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5795716-25.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5795716-25.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5795716-26.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5795716-26.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div></div></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img" alt="Previous page"src="/googlebooks/images/kennedy/page_left.png"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img" alt="Next page"src="/googlebooks/images/kennedy/page_right.png"width="21" height="21" /></div></div></div><div class="patent-post-drawings"></div><div class="patent-section patent-claims-section"><div class="patent-section-header"><span class="patent-section-title">Claims<span class="patent-section-count">(10)</span></span></div><div class="patent-text"><div mxw-id="PCLM5271230" lang="EN" load-source="patent-office" class="claims">
    <claim-statement>What is claimed is:</claim-statement> <div class="claim"> <div num="1" class="claim">
      <div class="claim-text">1. A computer program product that identifies an unknown base in a sample nucleic acid sequence, comprising:<div class="claim-text">computer code that receives a plurality of signals corresponding to probe intensities for a plurality of nucleic acid probes, each probe intensity indicating an extent of hybridization of a nucleic acid probe with at least one nucleic acid sequence including said sample sequence, and each nucleic acid probe differing from each other by at least a single base;</div> <div class="claim-text">computer code that performs a comparison of said plurality of probe intensities to each other;</div> <div class="claim-text">computer code that generates a base call identifying said unknown base according to results of said comparison and said sequences of said nucleic acid probes; and</div> <div class="claim-text">a computer readable medium that stores said computer codes.</div> </div>
    </div>
    </div> <div class="claim"> <div num="2" class="claim">
      <div class="claim-text">2. A computer program product that identifies an unknown base in a sample nucleic acid sequence, comprising:<div class="claim-text">computer code that receives a plurality of signals corresponding to probe intensities for a plurality of nucleic acid probes, each probe intensity indicating an extent of hybridization of a nucleic acid probe with said sample sequence, and each nucleic acid probe differing from each other by at least a single base;</div> <div class="claim-text">computer code that calculates a ratio of a higher probe intensity to a lower probe intensity;</div> <div class="claim-text">computer code that generates a base call identifying said unknown base according to a base of a nucleic acid probe having said higher probe intensity if said ratio is greater than a predetermined ratio value; and</div> <div class="claim-text">a computer readable medium that stores said computer codes.</div> </div>
    </div>
    </div> <div class="claim"> <div num="3" class="claim">
      <div class="claim-text">3. A computer program product that identifies an unknown base in a sample nucleic acid sequence, comprising:<div class="claim-text">computer code that receives a first set of signals corresponding to a first set of probe intensities, each probe intensity in said first set indicating an extent of hybridization of a nucleic acid probe with a reference nucleic acid sequence, and each nucleic acid probe differing from each other by at least a single base;</div> <div class="claim-text">computer code that receives a second set of signals corresponding to a second set of probe intensities, each probe intensity in said second set indicating an extent of hybridization of a nucleic acid probe with said sample sequence, and each nucleic acid probe differing from each other by at least a single base;</div> <div class="claim-text">computer code that performs a comparison of at least one of said probe intensities in said first set and at least one of said probe intensities in said second set;</div> <div class="claim-text">computer code that generates a base call identifying said unknown base according to results of said comparisons said sequence of said nucleic acid probe; and</div> <div class="claim-text">a computer readable medium that stores said computer codes.</div> </div>
    </div>
    </div> <div class="claim"> <div num="4" class="claim">
      <div class="claim-text">4. A computer program product that identifies an unknown base in a sample nucleic acid sequence, comprising:<div class="claim-text">computer code that receives signals corresponding to statistics about a plurality of experiments, each of said experiments producing probe intensities, each probe intensity indicating an extent of hybridization of a nucleic acid probe with a reference nucleic acid sequence, and each nucleic acid probe differing from each other by at least a single base;</div> <div class="claim-text">computer code that receives a plurality of signals corresponding to probe intensities, each probe intensity indicating an extent of hybridization of a nucleic acid probe with said sample sequence, and each nucleic acid probe differing from each other by at least a single base;</div> <div class="claim-text">computer code that performs a comparison of at least one of said plurality of probe intensities with said statistics;</div> <div class="claim-text">computer code that generates a base call identifying said unknown base according to results of said comparison and said sequence of said nucleic acid probe; and</div> <div class="claim-text">a computer readable medium that stores said computer codes.</div> </div>
    </div>
    </div> <div class="claim"> <div num="5" class="claim">
      <div class="claim-text">5. A system that identifies an unknown base in a sample nucleic acid sequence, comprising:<div class="claim-text">a processor; and</div> <div class="claim-text">a computer readable medium coupled to said processor for storing a computer program comprising:<div class="claim-text">computer code that receives a plurality of signals corresponding to probe intensities for a plurality of nucleic acid probes, each probe intensity indicating an extent of hybridization of a nucleic acid probe with at least one nucleic acid sequence including said sample sequence, and each nucleic acid probe differing from each other by at least a single base;</div> <div class="claim-text">computer code that performs a comparison of said plurality of probe intensities to each other; and</div> <div class="claim-text">computer code that generates a base call identifying said unknown base according to results of said comparison and said sequences of said nucleic acid probes.</div> </div> </div>
    </div>
    </div> <div class="claim"> <div num="6" class="claim">
      <div class="claim-text">6. A system that identifies an unknown base in a sample nucleic acid sequence, comprising:<div class="claim-text">a processor; and</div> <div class="claim-text">a computer readable medium coupled to said processor for storing a computer program comprising:<div class="claim-text">computer code that receives a plurality of signals corresponding to probe intensities for a plurality of nucleic acid probes, each probe intensity indicating an extent of hybridization of a nucleic acid probe with said sample sequence, and each nucleic acid probe differing from each other by at least a single base;</div> <div class="claim-text">computer code that calculates a ratio of a higher probe intensity to a lower probe intensity; and</div> <div class="claim-text">computer code that generates a base call identifying said unknown base according to a base of a nucleic acid probe having said higher probe intensity if said ratio is greater than a predetermined ratio value.</div> </div> </div>
    </div>
    </div> <div class="claim"> <div num="7" class="claim">
      <div class="claim-text">7. A system that identifies an unknown base in a sample nucleic acid sequence, comprising:<div class="claim-text">a processor; and</div> <div class="claim-text">a computer readable medium coupled to said processor for storing a computer program comprising:<div class="claim-text">computer code that receives a first set of signals corresponding to probe intensities, each probe intensity in said first set indicating an extent of hybridization of a nucleic acid probe with a reference nucleic acid sequence, and each nucleic acid probe differing from each other by at least a single base;</div> <div class="claim-text">computer code that receives a second set of signals corresponding to probe intensities, each probe intensity in said second set indicating an extent of hybridization of a nucleic acid probe with said sample sequence, and each nucleic acid probe differing from each other by at least a single base;</div> <div class="claim-text">computer code that performs a comparison of at least one of said probe intensities in said first set and at least one of said probe intensities in said second set; and</div> <div class="claim-text">computer code that generates a base call identifying said unknown base according to results of said comparison and said sequence of nucleic acid probe.</div> </div> </div>
    </div>
    </div> <div class="claim"> <div num="8" class="claim">
      <div class="claim-text">8. A system that identifies an unknown base in a sample nucleic acid sequence, comprising:<div class="claim-text">a processor; and</div> <div class="claim-text">a computer readable medium coupled to said processor for storing a computer program comprising:<div class="claim-text">computer code that receives signals corresponding to statistics about a plurality of experiments, each of said experiments producing probe intensities, each probe intensity indicating an extent of hybridization of a nucleic acid probe with a reference nucleic acid sequence, and each nucleic acid probe differing from each other by at least a single base;</div> <div class="claim-text">computer code that receives a plurality of signals corresponding to probe intensities, each probe intensity indicating an extent of hybridization of a nucleic acid probe with said sample sequence, and each nucleic acid probe differing from each other by at least a single base;</div> <div class="claim-text">computer code that performs a comparison of at least one of said plurality of probe intensities with said statistics; and</div> <div class="claim-text">computer code that generates a base call identifying said unknown base according to results of said comparison and said sequence of said nucleic acid probe.</div> </div> </div>
    </div>
    </div> <div class="claim-dependent"> <div num="9" class="claim">
      <div class="claim-text">9. A system according to claims 5, 6, 7, or 8, wherein the plurality of nucleic acid probes are in an array of probes.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="10" class="claim">
      <div class="claim-text">10. A system according to claims 5, 6, 7, or 8, wherein the plurality of probe intensities are fluorescent intensities.</div>
    </div>
  </div> </div></div></div><div class="patent-section patent-description-section"><div class="patent-section-header"><span class="patent-section-title">Description</span></div><div class="patent-text"><div mxw-id="PDES67163166" lang="EN" load-source="patent-office" class="description">
    <heading>GOVERNMENT RIGHTS NOTICE</heading> <p>Portions of the material in this specification arose in the course of or under contract nos. 92ER81275 (SBIR) between Affymetrix, Inc. and the Department of Energy and/or H600813-1, -2 between Affymetrix, Inc. and the National Institutes of Health.</p>
    <heading>COPYRIGHT NOTICE</heading> <p>A portion of the disclosure of this patent document contains material which is subject to copyright protection. The copyright owner has no objection to the xeroxographic reproduction by anyone of the patent document or the patent disclosure in exactly the form it appears in the Patent and Trademark Office patent file or records, but otherwise reserves all copyright rights whatsoever.</p>
    <heading>MICROFICHE APPENDIX</heading> <p>Microfiche Appendices A to E comprising five (5) sheets, totaling 272 frames are included herewith.</p>
    <heading>BACKGROUND OF THE INVENTION</heading> <p>The present invention relates to the field of computer systems. More specifically, the present invention relates to computer systems for visualizing biological sequences, as well as for evaluating and comparing biological sequences.</p>
    <p>Devices and computer systems for forming and using arrays of materials on a substrate are known. For example, PCT application WO92/10588, incorporated herein by reference for all purposes, describes techniques for sequencing or sequence checking nucleic acids and other materials. Arrays for performing these operations may be formed in arrays according to the methods of, for example, the pioneering techniques disclosed in U.S. Pat. No. 5,143,854 and U.S. patent application Ser. No. 08/249,188, both incorporated herein by reference for all purposes.</p>
    <p>According to one aspect of the techniques described therein, an array of nucleic acid probes is fabricated at known locations on a chip or substrate. A fluorescently labeled nucleic acid is then brought into contact with the chip and a scanner generates an image file indicating the locations where the labeled nucleic acids bound to the chip. Based upon the identities of the probes at these locations, it becomes possible to extract information such as the monomer sequence of DNA or RNA. Such systems have been used to form, for example, arrays of DNA that may be used to study and detect mutations relevant to cystic fibrosis, the P53 gene (relevant to certain cancers), HIV, and other genetic characteristics.</p>
    <p>Improved computer systems and methods are needed to evaluate, analyze, and process the vast amount of information now used and made available by these pioneering technologies.</p>
    <heading>SUMMARY OF THE INVENTION</heading> <p>An improved computer-aided system for visualizing and determining the sequence of nucleic acids is disclosed. The computer system provides, among other things, improved methods of analyzing fluorescent image files of a chip containing hybridized nucleic acid probes in order to call bases in sample nucleic acid sequences.</p>
    <p>According to one aspect of the invention, a computer system is used to identify an unknown base in a sample nucleic acid sequence by the steps of:</p>
    <p>inputting multiple probe intensities, each of the probe intensities being associated with a probe;</p>
    <p>the computer system comparing the multiple probe intensities where each of the probe intensities is substantially proportional to a probe hybridizing with at least one sequence; and</p>
    <p>calling the unknown base according to the comparison of the multiple probe intensities.</p>
    <p>According to one specific aspect of the invention, a higher probe intensity is compared to a lower probe intensity to call the unknown base. According to another specific aspect of the invention, probe intensities of a sample sequence are compared to probe intensities of a reference sequence. According to yet another specific aspect of the invention, probe intensities of a sample sequence are compared to statistics about probe intensities of a reference sequence from multiple experiments.</p>
    <p>According to another aspect of the invention, a method is disclosed of processing reference and sample nucleic acid sequences to reduce the variations between the experiments by the steps of:</p>
    <p>providing a plurality of nucleic acid probes;</p>
    <p>labeling the reference nucleic acid sequence with a first marker;</p>
    <p>labeling the sample nucleic acid sequence with a second marker; and</p>
    <p>hybridizing the labeled reference and sample nucleic acid sequences at the same time.</p>
    <p>According to yet another aspect of the invention, a computer system is used for comparative analysis and visualization of multiple sequences by the steps of:</p>
    <p>displaying at least one reference sequence in a first area on a display device; and</p>
    <p>displaying at least one sample sequence in a second area on said display device;</p>
    <p>whereby a user is capable of visually comparing the multiple sequences.</p>
    <p>A further understanding of the nature and advantages of the inventions herein may be realized by reference to the remaining portions of the specification and the attached drawings.</p>
    <heading>BRIEF DESCRIPTION OF THE DRAWINGS</heading> <p>FIG. 1 illustrates an overall system for forming and analyzing arrays of biological materials such as DNA or RNA;</p>
    <p>FIG. 2A is an illustration of the software for the overall system; FIG. 2B illustrates the global layout of a chip formed in the overall system; and FIG. 2C illustrates conceptually the binding of probes on chips;</p>
    <p>FIG. 3 illustrates the high level flow of the intensity ratio method;</p>
    <p>FIG. 4A illustrates the high level flow of one implementation of the reference method and FIG. 4B shows an analysis table for use with the reference method;</p>
    <p>FIG. 5A illustrates the high level flow of another implementation of the reference method; FIG. 5B shows a data table for use with the reference method; FIG. 5C shows a graph of the normalized sample base intensities minus the normalized reference base intensities; and FIG. 5D shows other graphs of data in the data table;</p>
    <p>FIG. 6 illustrates the high level flow of the statistical method;</p>
    <p>FIG. 7 illustrates the pooling processing of a reference and sample nucleic acid sequence;</p>
    <p>FIG. 8 illustrates the main screen and the associated pull down menus for comparative analysis and visualization of multiple experiments;</p>
    <p>FIG. 9 illustrates an intensity graph window for a selected base;</p>
    <p>FIG. 10 illustrates multiple intensity graph windows for selected bases;</p>
    <p>FIG. 11 illustrates the intensity ratio method correctly calling a mutation in solutions with varying concentrations;</p>
    <p>FIG. 12 illustrates the reference method correctly calling a mutant base where the intensity ratio method incorrectly called the mutant base; and</p>
    <p>FIG. 13 illustrates the output of the VIEWSEQ™ program with four pretreatment samples and four posttreatment samples.</p>
    <heading>DESCRIPTION OF THE PREFERRED EMBODIMENT</heading> <heading>CONTENTS</heading> <p>I. General</p>
    <p>II. Intensity Ratio Method</p>
    <p>III. Reference Method</p>
    <p>IV. Statistical Method</p>
    <p>V. Pooling Processing</p>
    <p>VI. Comparative Analysis</p>
    <p>VII. Examples</p>
    <p>VIII. Appendices</p>
    <p>I. General</p>
    <p>The present invention provides methods of analyzing hybridization intensity files for a chip containing hybridized nucleic acid probes. In a representative embodiment, the files represent fluorescence data from a biological array, but the files may also represent other data such as radioactive intensity data. For purposes of illustration, the present invention is described as being part of a computer system that designs a chip mask, synthesizes the probes on the chip, labels the nucleic acids, and scans the hybridized nucleic acid probes. Such a system is fully described in U.S. patent application Ser. No. 08/249,188 which has been incorporated by reference for all purposes. However, the present invention may be used separately from the overall system for analyzing data generated by such systems.</p>
    <p>FIG. 1 illustrates a computerized system for forming and analyzing arrays of biological materials such as RNA or DNA. A computer 100 is used to design arrays of biological polymers such as RNA or DNA. The computer 100 may be, for example, an appropriately programmed Sun Workstation or personal computer or workstation, such as an IBM PC equivalent, including appropriate memory and a CPU. The computer system 100 obtains inputs from a user regarding characteristics of a gene of interest, and other inputs regarding the desired features of the array. Optionally, the computer system may obtain information regarding a specific genetic sequence of interest from an external or internal database 102 such as GenBank. The output of the computer system 100 is a set of chip design computer files 104 in the form of, for example, a switch matrix, as described in PCT application WO 92/10092, and other associated computer files.</p>
    <p>The chip design files are provided to a system 106 that designs the lithographic masks used in the fabrication of arrays of molecules such as DNA. The system or process 106 may include the hardware necessary to manufacture masks 110 and also the necessary computer hardware and software 108 necessary to lay the mask patterns out on the mask in an efficient manner. As with the other features in FIG. 1, such equipment may or may not be located at the same physical site, but is shown together for ease of illustration in FIG. 1. The system 106 generates masks 110 or other synthesis patterns such as chrome-on-glass masks for use in the fabrication of polymer arrays.</p>
    <p>The masks 110, as well as selected information relating to the design of the chips from system 100, are used in a synthesis system 112. Synthesis system 112 includes the necessary hardware and software used to fabricate arrays of polymers on a substrate or chip 114. For example, synthesizer 112 includes a light source 116 and a chemical flow cell 118 on which the substrate or chip 114 is placed. Mask 110 is placed between the light source and the substrate/chip, and the two are translated relative to each other at appropriate times for deprotection of selected regions of the chip. Selected chemical reagents are directed through flow cell 118 for coupling to deprotected regions, as well as for washing and other operations. All operations are preferably directed by an appropriately programmed computer 119, which may or may not be the same computer as the computer(s) used in mask design and mask making.</p>
    <p>The substrates fabricated by synthesis system 112 are optionally diced into smaller chips and exposed to marked receptors. The receptors may or may not be complementary to one or more of the molecules on the substrate. The receptors are marked with a label such as a fluorescein label (indicated by an asterisk in FIG. 1) and placed in scanning system 120. Scanning system 120 again operates under the direction of an appropriately programmed digital computer 122, which also may or may not be the same computer as the computers used in synthesis, mask making, and mask design. The scanner 120 includes a detection device 124 such as a confocal microscope or CCD (charge-coupled device) that is used to detect the locations where labeled receptor (*) has bound to the substrate. The output of scanner 120 is an image file(s) 124 indicating, in the case of fluorescein labeled receptor, the fluorescence intensity (photon counts or other related measurements, such as voltage) as a function of position on the substrate. Since higher photon counts will be observed where the labeled receptor has bound more strongly to the array of polymers, and since the monomer sequence of the polymers on the substrate is known as a function of position, it becomes possible to determine the sequence(s) of polymer(s) on the substrate that are complementary to the receptor.</p>
    <p>The image file 124 is provided as input to an analysis system 126 that incorporates the visualization and analysis methods of the present invention. Again, the analysis system may be any one of a wide variety of computer system(s), but in a preferred embodiment the analysis system is based on a Sun Workstation or equivalent. The present invention provides various methods of analyzing the chip design files and the image files, providing appropriate output 128. The present invention may further be used to identify specific mutations in a receptor such as DNA or RNA.</p>
    <p>FIG. 2A provides a simplified illustration of the overall software system used in the operation of one embodiment of the invention. As shown in FIG. 2A, the system first identifies the genetic sequence(s) or targets that would be of interest in a particular analysis at step 202. The sequences of interest may, for example, be normal or mutant portions of a gene, genes that identify heredity, or provide forensic information. sequence selection may be provided via manual input of text files or may be from external sources such as GenBank. At step 204 the system evaluates the gene to determine or assist the user in determining which probes would be desirable on the chip, and provides an appropriate "layout" on the chip for the probes. A wild-type probe is a probe that will ideally hybridize with the gene of interest and thus a wild-type gene (also called the chip wild-type) would ideally hybridize with all the wild-type probes on the chip. The layout implements desired characteristics such as arrangement on the chip that permits "reading" of genetic sequence and/or minimization of edge effects, ease of synthesis, and the like.</p>
    <p>FIG. 2B illustrates the global layout of a chip. Chip 114 is composed of multiple units where each unit may contain different tilings for the chip wild-type sequence. Unit 1 is shown in greater detail and shows that each unit is composed of multiple cells which are areas on the chip that may contain probes. Conceptually, each unit is composed of multiple sets of related cells. As used herein, the term cell refers to a region on a substrate that contains many copies of a molecule or molecules of interest. Each unit is composed of multiple cells that may be placed in rows and columns. In one embodiment, a set of five related cells includes the following: a wild-type cell 220, "mutation" cells 222, and a "blank" cell 224. Cell 220 contains a wild-type probe that is the complement of a portion of the wild-type sequence. Cells 222 contain "mutation" probes for the wild-type sequence. For example, if the wild-type probe is 3'-ACGT, the probes 3'-ACAT, 3'-ACCT, 3'-ACGT, and 3'-ACTT may be the "mutation" probes. Cell 224 is the "blank" cell because it contains no probes (also called the "blank" probe). As the blank cell contains no probes, labeled receptors should not bind to the chip in this area. Thus, the blank cell provides an area that can be used to measure the background intensity.</p>
    <p>In one embodiment, numerous tiling processes are available including sequence tiling, block tiling, and opt-tiling, as described below. Of course, a wide range of layout strategies may be used according to the invention herein, without departing from the scope of the invention. For example, the probes may be tiled on a substrate in an apparently random fashion where a computer system is utilized to keep track of the probe locations and correlate the data obtained from the substrate.</p>
    <p>Opt-tiling is the process of tiling additional probes for suspected mutations. As a simple example of opt-tiling, suppose the wild-type target sequence is 5'-ACGTATGCA-3' and it is suspected that a mutant sequence has a possible T base mutation at the underlined base position. Suppose further that the chip will be synthesized with a "4×3" tiling strategy, meaning that probes of four monomers are used and that the monomers in position 3, counting left to right, of the probe are varied.</p>
    <p>In opt-tiling, extra probes are tiled for each suspected mutation. The extra probes are tiled as if the mutation base is a wild-type base. The following shows the probes that may be generated for this example:</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">              TABLE 1______________________________________Probe Sequences (From 3'-end)4 × 3 Opt-Tiling______________________________________Wild    TGCA     GCAT     CATA   ATAC   TACGA sub.  TGAA     GCAT     CAAA   ATAC   TAAGC sub.  TGCA     GCCT     CACA   ATCC   TACGG sub.  TGGA     GCGT     CAGA   ATGC   TAGGT sub.  TGTA     GCTT     CATA   ATTC   TATGWild    TGCA     GCAA     CAAA   AAAC   AACGA sub.  TGAA     GCAA     CAAA   AAAC   AAAGC sub.  TGCA     GCCA     CACA   AACC   AACGG sub.  TGGA     GCGA     CAGA   AAGC   AAGGT sub.  TGTA     GCTA     CATA   AATC   AATG______________________________________</pre>
    
    <p>In the first "chip" above, the top row of the probes (along with one probe below each of the four wild-type probes) should bind to the target DNA sequence. However, if the target sequence has a T base mutation as suspected, the labeled mutant sequence will not bind that strongly to the probes in the columns around column 3. For example, the mutant receptor that could bind with the probes in column 2 is 5'-CGTT which may not bind that strongly to any of the probes in column 2 because there are T bases at the ends of the receptor and probes (i.e., not complementary). This often results in a relatively dark scanned area around a mutation.</p>
    <p>Opt-tiling generates the second "chip" above to handle the suspected mutation as a wild-type base. Thus, the mutant receptor 5'-CGTT should bind strongly to the wild-type probe of column 2 (along with one probe below) and the mutation can be further detected.</p>
    <p>Again referring to FIG. 2A, at step 206 the masks for the synthesis are designed. At step 208 the software utilizes the mask design and layout information to make the DNA or other polymer chips. This software 208 will control relative translation of a substrate and the mask, the flow of desired reagents through a flow cell, the synthesis temperature of the flow cell, and other parameters. At step 210, another piece of software is used in scanning a chip thus synthesized and exposed to a labeled receptor. The software controls the scanning of the chip, and stores the data thus obtained in a file that may later be utilized to extract sequence information.</p>
    <p>At step 212 a computer system according to the present invention utilizes the layout information and the fluorescence information to evaluate the hybridized nucleic acid probes on the chip. Among the important pieces of information obtained from DNA chips are the identification of mutant receptors and determination of genetic sequence of a particular receptor.</p>
    <p>FIG. 2C illustrates the binding of a particular target DNA to an array of DNA probes 114. As shown in this simple example, the following probes are formed in the array (only one probe is shown for the wild-type probe): ##STR1##</p>
    <p>As shown, the set of probes differ by only one base so the probes are designed to determine the identity of the base at that position in the nucleic acid sequence.</p>
    <p>When a fluorescein-labeled (or otherwise marked) target with the sequence 5'-TCTTGCA is exposed to the array, it is complementary only to the probe 3'-AGAACGT, and fluorescein will be primarily found on the surface of the chip where 3'-AGAACGT is located. Thus, for each set of probes that differ by only one base, the image file will contain four fluorescence intensities, one for each probe. Each fluorescence intensity can therefore be associated with the base of each probe that is different from the other probes. Additionally, the image file will contain a "blank" cell which can be used as the fluorescence intensity of the background. By analyzing the five fluorescence intensities associated with a specific base location, it becomes possible to extract sequence information from such arrays using the methods of the invention disclosed herein.</p>
    <p>The present invention calls bases by assigning the bases the following codes:</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">______________________________________Code      Group          Meaning______________________________________A         A              AdenineC         C              CytosineG         G              GuanineT         T(U)           Thymine (Uracil)M         A or C         aMinoR         A or G         puRineW         A or T(U)      Weak interaction                    (2 H bonds)Y         C or T(U)      pYrimidineS         C or G         Strong interaction                    (3 H bonds)K         G or T(U)      KetoV         A, C or G      not T(U)H         A, C or T(U)   not GD         A, G or T(U)   not CB         C, G or T(U)   not AN         A, C, G, or T(U)                    Insufficient intensity                    to callX         A, C, G, or T(U)                    Insufficient                    discrimination to                    call______________________________________</pre>
    
    <p>Most of the codes conform to the IUPAC standard. However, code N has been redefined and code X has been added.</p>
    <p>II. Intensity Ratio Method</p>
    <p>The intensity ratio method is a method of calling bases in a sample nucleic acid sequence. The intensity ratio method is most accurate when there is good discrimination between the fluorescence intensities of hybrid matches and hybrid mismatches. If there is insufficient discrimination, the intensity ratio method assigns a corresponding ambiguity code to the unknown base.</p>
    <p>For simplicity, the intensity ratio method will be described as being used to identify one unknown base in a sample nucleic acid sequence. In practice, the method is used to identify many or all the bases in a nucleic acid sequence.</p>
    <p>The unknown base will be identified by evaluation of up to four mutation probes and a "blank" cell, which is a location where a labeled receptor should not bind to the chip since no probe is present. For example, suppose a DNA sequence of interest or target sequence contains the sequence 5'-AGAACCTGC-3' with a possible mutation at the underlined base position. Suppose that 5-mer probes are to be synthesized for the target sequence. A representative wild-type probe of 5'-TTGGA is complementary to the region of the sequence around the possible mutation. The "mutation" probes will be the same as the wild-type probe except for a different base at the third position as follows: 3'-TTAGA, 3'-TTCGA, 3'-TTGGA, and 3'-TTTGA.</p>
    <p>If the fluorescently marked sample sequence is exposed to the above four mutation probes, the intensity should be highest for the probe that binds most strongly to the sample sequence. Therefore, if the probe 3'-TTTGA shows the highest intensity, the unknown base in the sample will generally be called an A mutation because the probes are complementary to the sample sequence.</p>
    <p>The mutation probes are identical to the wild-type probes except that they each contain one of the four A, C, G, or T "mutations" for the unknown base. Although one of the "mutation" probes will optimally be identical to the wild-type probe, such redundant probes are intentionally synthesized for quality control and design consistency.</p>
    <p>The identity of the unknown base is preferably determined by evaluating the relative fluorescence intensities of up to four of the mutation probes, and the "blank" cell. Because each mutation probe is identifiable by the mutation base, a mutation probe's intensity will be referred to as the "base intensity" of the mutation base.</p>
    <p>As a simple example of the intensity ratio method, suppose a gene of interest (target) is an HIV protease gene with the sequence 5'-ATGTGGACAGTTGTA-3' (SEQ ID NO:1). Suppose further that a sample sequence is suspected to have the same sequence as the target sequence except for a mutation of base C to base T at the underlined base position. Although hundreds of probes may be synthesized on the chip, the complementary mutation probes synthesized to detect a mutation in the sample sequence at the suspected mutation position may be as follows:</p>
    <p>3'-TATC</p>
    <p>3'-TCTC</p>
    <p>3'-TGTC (wild-type)</p>
    <p>3'-TTTC</p>
    <p>The mutation probe 3'-TGTC is also the wild-type probe as it should bind most strongly with the target sequence.</p>
    <p>After the sample sequence is labeled, hybridized on the chip, and scanned, suppose the following fluorescence intensities were obtained:</p>
    <p>3'-TATC→45</p>
    <p>3'-TCTC-→8</p>
    <p>3'-TGTC→32</p>
    <p>3'-TTTC→12</p>
    <p>where the intensity is measured by the photon count detected by the scanner. The "blank" cell had a fluorescence intensity of 2. The photon counts in the examples herein are representative (not actual data) and provided for illustration purposes. In practice, the actual photon counts will vary greatly depending on the experiment parameters and the scanner utilized.</p>
    <p>Although each fluorescence intensity is from a probe, the probes may be characterized by their unique mutation base so the bases may be said to have the following intensities:</p>
    <p>A→45</p>
    <p>C→8</p>
    <p>G→32</p>
    <p>T→12</p>
    <p>Thus, base A will be described as having an intensity of 45, which corresponds to the intensity of the mutation probe with the mutation base A.</p>
    <p>Initially, each mutation base intensity is reduced by the background or "blank" cell intensity. This is done as follows:</p>
    <p>A→45-2=43</p>
    <p>C→8-2=6</p>
    <p>G→32-2=30</p>
    <p>T→12-2=10</p>
    <p>Then, the base intensities are sorted in descending order of intensity. The above bases would be sorted as follows:</p>
    <p>A→43</p>
    <p>G→30</p>
    <p>T→10</p>
    <p>C→6</p>
    <p>Next, the highest intensity base is compared to the second highest intensity base. Thus, the ratio of the intensity of base A to the intensity of base G is calculated as follows: A:G=43/30=1.4. The ratio A:G is then compared to a predetermined ratio cutoff, which is a number that specifies the ratio required to identify the unknown base. For example, if the ratio cutoff is 1.2, the ratio A:G is greater than the ratio cutoff (1.4&gt;1.2) and the unknown base is called by the mutation probe containing the mutation A. As probes are complementary to the sample sequence, the sample sequence is called as having a mutation T, resulting in a called sample sequence of 5'-ATGTGGATAGTTGTA-3' (SEQ ID NO:2).</p>
    <p>As another example, suppose everything else is the same as in the previous example except that the sorted background adjusted intensities were as follows:</p>
    <p>C→42</p>
    <p>A→40</p>
    <p>G→10</p>
    <p>T→8</p>
    <p>The ratio of the highest intensity base to the second highest intensity base (C:A) is 1.05. Because this ratio is not greater than the ratio cutoff of 1.2, the unknown base will be called as being ambiguously one of two or more bases as follows.</p>
    <p>The second highest intensity base is then compared to the third highest base. The ratio of A:G is 4. The ratio of A:G is then compared to the ratio cutoff of 1.2. As the ratio A:G is greater than the ratio cutoff (4&gt;1.2), the unknown base is called by the mutation probes containing the mutations C or A. As probes are complementary to the sample sequence, the sample sequence is called as having either a mutation G or T, resulting in a sample sequence of 5'-ATGTGGAKAGTTGTA-3' (SEQ ID NO:3) where K is the IUPAC code for G or T(U).</p>
    <p>The ratio cutoff in the previous examples was equal to 1.2. However, the ratio cutoff will generally need to be adjusted to produce optimal results for the specific chip design and wild-type target. Also, although the ratio cutoff used has been the same for each ratio comparison, the ratio cutoff may vary depending on whether the ratio comparisons involve the highest, second highest, third highest, etc. intensity base.</p>
    <p>FIG. 3 illustrates the high level flow of the intensity ratio method. At step 302 the four base intensities are adjusted by subtracting the background or "blank" cell intensity from each base intensity. Preferably, if a base intensity is then less than or equal to zero, the base intensity is set equal to a small positive number to prevent division by zero or negative numbers in future calculations.</p>
    <p>At step 304 the base intensities are sorted by intensity. Each base is then associated with a number from 1 to 4. The base with the highest intensity is 1, second highest 2, third highest 3, and fourth highest 4. Thus, the intensity of base 1≧base 2≧base 3≧base 4≧.</p>
    <p>At step 306 the highest intensity base (base 1) is checked to see if it has sufficient intensity to call the unknown base. The intensity is checked by determining if the intensity of base 1 is greater than a predetermined background difference cutoff. The background difference cutoff is a number that specifies the intensity a base intensity must be over the background intensity in order to correctly call the unknown base. Thus, the background adjusted base intensity must be greater than the background difference cutoff or the unknown is not callable.</p>
    <p>If the intensity of base 1 is not greater than the background difference cutoff, the unknown base is assigned the code N (insufficient intensity) as shown at step 308. Otherwise, the ratio of the intensity of base 1 to base 2 is calculated as shown at step 310.</p>
    <p>At step 312 the ratio of intensities of bases 1:2 is compared to the ratio cutoff. If the ratio 1:2 is greater than the ratio cutoff, the unknown base is called as the complement of the highest intensity base (base 1) as shown at step 314. Otherwise, the ratio of the intensity of base 2 to base 3 is calculated as shown at step 316.</p>
    <p>At step 318 the ratio of intensities of bases 2:3 is compared to the ratio cutoff. If the ratio 2:3 is greater than the ratio cutoff, the unknown base is called as being an ambiguity code specifying the complements of the highest or second highest intensity bases (base 1 or 2) as shown at step 320. Otherwise, the ratio of the intensities of bases 3 to base 4 is calculated as shown at step 322.</p>
    <p>At step 324 the ratio of intensities of bases 3:4 is compared to the ratio cutoff. If the ratio 3:4 is greater than the ratio cutoff, the unknown base is called as being an ambiguity code specifying the complements of the highest, second highest, or third highest bases (base 1, 2 or 3) as shown at step 326. Otherwise, the unknown base is assigned the code X (insufficient discrimination) as shown at step 328.</p>
    <p>The advantage of the intensity ratio method is that it is very accurate when there is good discrimination between the fluorescence intensities of hybrid matches and hybrid mismatches. However, if the base corresponding to a correct hybrid gives a lower intensity than a mismatch (e.g., as a result of cross-hybridization), incorrect identification of the base will result. For this reason, however, the method is useful for comparative assessment of hybridization quality and as an indicator of sequence-specific problem spots. For example, the intensity ratio method has been used to determine that ambiguities and miscalls tend to be very different from sequence to sequence, and reflect predominantly the composition and repetitiveness of the sequence. It has also been used to assess improvements obtained by varying hybridization conditions, sample preparation, and post-hybridization treatments (e.g., RNase treatment).</p>
    <p>III. Reference Method</p>
    <p>The reference method is a method of calling bases in a sample nucleic acid sequence. The reference method depends very little on discrimination between the fluorescence intensities of hybrid matches and hybrid mismatches, and therefore is much less sensitive to cross-hybridization. The method compares the probe intensities of a reference sequence to the probe intensities of a sample sequence. Any significant changes are flagged as possible mutations. There are two implementations of the reference method disclosed herein.</p>
    <p>For simplicity, the reference method will be described as being used to identify one unknown base in a sample nucleic acid sequence. In practice, the method is used to identify many or all the bases in a nucleic acid sequence.</p>
    <p>The unknown base will be called by comparing the probe intensities of a reference sequence to the probe intensities of a sample sequence. Preferably, the probe intensities of the reference sequence and the sample sequence are from chips having the same chip wild-type. However, the reference sequence may or may not be exactly the same as the chip wild-type, as it may have mutations.</p>
    <p>The bases at the same position in the reference and sample sequences will each be associated with up to four mutation probes and a "blank" cell. The unknown base in the sample sequence is called by comparing probe intensities of the sample sequence to probe intensities of the reference sequence. For example, suppose the chip wild-type contains the sequence 5'-AGACCTTGC-3' and it is suspected that the sample has a possible mutation at the underlined base position, which is the unknown base that will be called by the reference method. The "mutation" probes for the sample sequence may be as follows: 3'-GAAA, 3'-GCAA, 3'-GGAA, and 3'-GTAA, where 3'-GGAA is the wild-type probe.</p>
    <p>Suppose further that a reference sequence, which differs from the chip wild-type by one base mutation, has the sequence 5'-AGACATTGC-3' where the mutation base is underlined. The "mutation" probes for the reference sequence may be as follows: 3'-TGAAA, 3'-TGCAA, 3'-TGGAA, and 3'-TGTAA, where 3'-TGTAA is the reference wild-type probe since the reference sequence is known. Although generally the sample and reference sequences were tiled with the same chip wild-type, this is not required, and the tiling methods do not have to be identical as shown by the use of two probe lengths in the example. Thus, the unknown base will be called by comparing the "mutation" probes of the sample sequence to the "mutation" probes of the reference sequence. As before, because each mutation probe is identifiable by the mutation base, the mutation probes' intensities will be referred to as the "base intensities" of their respective mutation bases.</p>
    <p>As a simple example of one implementation of the reference method, suppose a gene of interest (target) has the sequence 5'-AAAACTGAAAA-3' (SEQ. ID NO:4). Suppose a reference sequence has the sequence 5'-AAAACCGAAAA-3' (SEQ ID NO:5), which differs from the target sequence by the underlined base. The reference sequence is marked and exposed to probes on a chip with the target sequence being the chip wild-type. Suppose further that a sample sequence is suspected to have the same sequence as the target sequence except for a mutation at the underlined base position in 5'-AAAACTGAAAA-3' (SEQ ID NO:4). The sample sequence is also marked and exposed to probes on a chip with the target sequence being the chip wild-type. After hybridization and scanning, the following probe intensities (not actual data) were found for the respective complementary probes:</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">______________________________________Reference           Sample______________________________________3'-TGAC -&gt; 12       3'-GACT -&gt; 113'-TGCC -&gt; 9        3'-GCCT -&gt; 303'-TGGC -&gt; 80       3'-GGCT -&gt; 603'-TGTC -&gt; 15       3'-GTCT -&gt; 6______________________________________</pre>
    
    <p>Although each fluorescence intensity is from a probe, the probes may be identified by their unique mutation base so the bases may be said to have the following intensities:</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">______________________________________  Reference     Sample______________________________________  A -&gt; 12       A -&gt; 11  C -&gt; 9        C -&gt; 30  G -&gt; 80       G -&gt; 60  T -&gt; 15       T -&gt; 6______________________________________</pre>
    
    <p>Thus, base A of the reference sequence will be described as having an intensity of 12, which corresponds to the intensity of the mutation probe with the mutation base A. The reference method will now be described as calling the unknown base in the sample sequence by using these intensities.</p>
    <p>FIG. 4A illustrates the high level flow of one implementation of the reference method. For illustration purposes, the reference method is described as filling in the columns (identified by the numbers along the bottom) of the analysis table shown in FIG. 4B. However, the generation of an analysis table is not necessary to practice the method. The analysis table is shown to aid the reader in understanding the method.</p>
    <p>At step 402 the four base intensities of the reference and sample sequences are adjusted by subtracting the background or "blank" cell intensity from each base intensity. Each set of "mutation" probes has an associated "blank" cell. Suppose that the reference "blank" cell intensity is 1 and the sample "blank" cell intensity is 2. The base intensities are then background subtracted as follows:</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">______________________________________Reference           Sample______________________________________A -&gt; 12 - 1 = 11    A -&gt; 11 - 2 = 9C -&gt; 9 - 1 = 8      C -&gt; 30 - 2 = 28G -&gt; 80 - 1 = 79    G -&gt; 60 - 2 = 58T -&gt; 15 - 1 = 14    T -&gt; 6 - 2 = 4______________________________________</pre>
    
    <p>Preferably, if a base intensity is then less than or equal to zero, the base intensity is set equal to a small positive number to prevent division by zero or negative numbers in future calculations.</p>
    <p>For identification, the position of each base of interest in the reference and sample sequences is placed in column 1 of the analysis table. Also, since the reference sequence is a known sequence, the base at this position is known and is referred to as the reference wild-type. The reference wild-type is placed in column 2 of the analysis table, which is C for this example.</p>
    <p>At step 404 the base intensity associated with the reference wild-type (column 2 of the analysis table) is checked to see if it has sufficient intensity to call the unknown base. In this example, the reference wild-type is C. However, the base intensity associated with the wild-type is the G base intensity, which is 79 in this example. This is because the base intensities actually represent the complementary "mutation" probes. The G base intensity is checked by determining if its intensity is greater than a predetermined background difference cutoff. The background difference cutoff is a number that specifies the intensity the base intensities must be above the background intensity in order to correctly call the unknown base. Thus, the base intensity associated with the reference wild-type must be greater than the background difference cutoff or the unknown base is not callable.</p>
    <p>If the background difference cutoff is 5, the base intensity associated with the reference wild-type has sufficient intensity (79&gt;5) so a P (pass) is placed in column 3 of the analysis table as shown at step 406. Otherwise, at step 407 an F (fail) is placed in column 3 of the analysis table.</p>
    <p>At step 408 the ratio of the base intensity associated with the reference wild-type to each of the possible bases are calculated. The ratio of the base intensity associated with the reference wild-type to itself will be 1 and the other ratios will usually be greater than 1. The base intensity associated with the reference wild-type is G so the following ratios are calculated:</p>
    <p>G:A→79/11=7.2</p>
    <p>G:C→79/8=9.9</p>
    <p>G:G→79/79=1.0</p>
    <p>G:T→79/14=5.6</p>
    <p>These ratios are placed in columns 4 through 7 of the analysis table, respectively.</p>
    <p>At step 410 the highest base intensity associated with the sample sequence is checked to see if it has sufficient intensity to call the unknown base. The highest base intensity is checked by determining if the intensity is greater than the background difference cutoff. Thus, the highest base intensity must be greater than the background difference cutoff or the unknown base is not callable.</p>
    <p>Again, if the background difference cutoff is 5, the highest base intensity, which is G in this example, has sufficient intensity (58&gt;5) so a P (pass) is placed in column 8 of the analysis table as shown at step 412. Otherwise, at step 413 an F (fail) is placed in column 8 of the analysis table.</p>
    <p>At step 414 the ratios of the highest base intensity of the sample to each of the possible bases are calculated. The ratio of the highest base intensity to itself will be 1 and the other ratios will usually be greater than 1. Thus, the highest base intensity is G so the following ratios are calculated:</p>
    <p>G:A→58/9=6.4</p>
    <p>G:C→58/28=2.3</p>
    <p>G:G→58/58=1.0</p>
    <p>G:T→58/4=14.5</p>
    <p>These ratios are placed in columns 9 through 12 of the analysis table, respectively.</p>
    <p>At step 416 if both the reference and sample sequence probes failed to have sufficient intensity to call the unknown base, meaning there is an `F` in columns 3 and 8 of the analysis table, the unknown base is assigned the code N (insufficient intensity) as shown at step 418. An `N` is placed in column 17 of the analysis table. Additionally, a confidence code of 9 is placed in column 18 of the analysis table where the confidence codes have the following meanings:</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">______________________________________Code        Meaning______________________________________0           Probable reference wild-type1           Probable mutation2           Reference sufficient intensity,       insufficient intensity in sample       suggests possible mutation3           Borderline differences, unknown base       ambiguous4           Sample sufficient intensity, insufficient       intensity in reference to allow       comparison5-8         Currently unassigned9           Insufficient intensity in reference and       sample, no interpretation possible______________________________________</pre>
    
    <p>The confidence codes are useful for indicating to the user the resulting analysis of the reference method.</p>
    <p>At step 420 if only the reference sequence probes failed to have sufficient intensity to call the unknown base, meaning there is an `F` in column 3 and a `P` in column 8 of the analysis table, the unknown base is assigned the code N (insufficient intensity) as shown at step 422. An `N` is placed in column 17 and a confidence code of 4 is placed in column 18 of the analysis table.</p>
    <p>At step 424 if only the sample sequence probes failed to have sufficient intensity to call the unknown base, meaning there is a `P` in column 3 and a `F` in column 8 of the analysis table, the unknown base is assigned the code N (insufficient intensity) as shown at step 426. An `N` is placed in column 17 and a confidence code of 2 is placed in column 18 of the analysis table.</p>
    <p>In this example, both the reference and sample sequence probes have sufficient intensity to call the unknown base. At step 428 the ratios of the reference ratios to the sample ratios for each base type are calculated. Thus, the ratio A:A (column 4 to column 9) is placed in column 13 of the analysis table. The ratio C:C (column 5 to column 10) is placed in column 14 of the analysis table. The ratio G:G (column 6 to column 11) is placed in column 15 of the analysis table. Lastly, the ratio T:T (column 7 to column 12) is placed in column 16 of the analysis table. These ratios are calculated as follows:</p>
    <p>A:A→7.2/6.4=1.1</p>
    <p>C:C→9.9/2.3=4.3</p>
    <p>G:G→1.0/1.0=1.0</p>
    <p>T:T→5.6/14.5=0.4</p>
    <p>The unknown base is called by comparing these ratios of ratios to two predetermined values as follows.</p>
    <p>At step 430 if all the ratios of ratios (columns 13 to 16 of the analysis table) are less than a predetermined lower ratio cutoff, the unknown base is assigned the code of the reference wild-type as shown at step 432. Thus, the code for the reference wild-type (as shown in column 2) would be placed in column 17 and a confidence code of 0 would be placed in column 18 of the analysis table.</p>
    <p>At step 434 if all the ratios of ratios are less than a predetermined upper ratio cutoff, the unknown base is assigned an ambiguity code that indicates the unknown base may be any one of the bases that has a complementary ratio of ratios greater than the lower ratio cutoff and less than the upper ratio cutoff as shown at step 436. Thus, if the ratio of ratios for A:A, C:C and G:G are all greater than the lower ratio cutoff and less than the upper ratio cutoff, the unknown base would be assigned the code B (meaning "not A"). This is because the ratios of ratios are complementary to their respective base as follows:</p>
    <p>A:A→T</p>
    <p>C:C→G</p>
    <p>G:G→C</p>
    <p>so the unknown base would be called as being either C, G, or T, which is identified by the IUPAC code B. This ambiguity code would be placed in column 17 and a confidence code of 3 would be placed in column 18 of the analysis table.</p>
    <p>At step 438 at least one of the ratios of ratios is greater than the upper ratio cutoff and the unknown base is called as the base complementary to the highest ratio of ratios. The code for the base complementary to the highest ratio of ratios would be placed in column 17 and a confidence code of 1 would be placed in column 18 of the analysis table.</p>
    <p>Assume for the purposes of this example that the lower ratio cutoff is 1.5 and the upper ratio cutoff is 3. Again, the ratios of ratios are as follows:</p>
    <p>A:A→1.1</p>
    <p>C:C→4.3</p>
    <p>G:G→1.0</p>
    <p>T:T→0.4</p>
    <p>As all the ratios of ratios are not less than the upper ratio cutoff, the unknown base is called the base complementary to the highest ratio of ratios. The highest ratio of ratios is C:C, which has a complementary base G. Thus, the unknown base is called G which is placed in column 17 and a confidence code of 1 is placed in column 18 of the analysis table.</p>
    <p>The example shows how the unknown base in the sample nucleic acid sequence was correctly called as base G. Although the complementary "mutation" probe associated with the base G (3'-GCCT) did not have the highest fluorescence intensity, the unknown base was called as base G because the associated "mutation" probe had the highest ratio increase over the other "mutation" probes.</p>
    <p>FIG. 5A illustrates the high level flow of another implementation of the reference method. As in the previous implementation, this implementation also compares the probe intensities of a reference sequence to the probe intensities of a sample sequence. However, this implementation differs conceptually from the previous implementation in that neighboring probe intensities are also analyzed, resulting in more accurate base calling.</p>
    <p>As a simple example of this implementation of the reference method, suppose a reference sequence has a sequence of 5'-AAACCCAATCCACATCA-3' (SEQ ID NO:6) and a sample sequence has a sequence of 5'-AAACCCAGTCCACATCA-3' (SEQ ID NO:7), where the mutant base is underlined. Thus, there is a mutation of A to G. Suppose further that the reference and sample sequences are tiled on chips with the reference sequence being the chip wild-type. This implementation of the reference method will be described as identifying this mutation base.</p>
    <p>For illustration purposes, this implementation of the reference method is described as filling in a data table shown in FIG. 5B (SEQ ID NO:28, and SEQ ID NO:29). Although the data table contains more data than is required for this implementation, the portions of the data table that are produced by steps in FIG. 5A are shown with the same reference numerals. The generation of a data table is not necessary, however, and is shown to aid the reader in understanding the method. The mutant base position is at position 241 in the reference and sample sequences, which is shown in bold in the data table.</p>
    <p>At step 502 the base intensities of the reference and sample sequences are adjusted by subtracting the background or "blank" cell intensity from each base intensity. Preferably, if a base intensity is then less than or equal to zero, the base intensity is set equal to a small positive number to prevent division by zero or negative numbers. In the data table, data 502A is the background subtracted base intensities for the reference sequence and data 502B is the background subtracted base intensities for the sample sequence (also called the "mutant" sequence in the data table).</p>
    <p>At step 504 the base intensity associated with the reference wild-type is checked to see if it has sufficient intensity to call the unknown base. In this example, the reference wild-type is base A at position 241. The base intensity associated with the reference wild-type is identified by a lower case "a" in the left hand column. Thus, the base intensities in the data table are not identified by their complements and the reference wild-type at the mutation position has an intensity of 385. The reference wild-type intensity of 385 is checked by determining if its intensity is greater than a predetermined background difference cutoff. The background difference cutoff is a number that specifies the intensity the base intensities must be over the background intensity in order to correctly call the unknown base. Thus, the base intensity associated with the reference wild-type must be greater than the background difference cutoff or the unknown base is not callable.</p>
    <p>If the base intensity associated with the reference wild-type is not greater than the background difference cutoff, the wild-type sequence would fail to have sufficient intensity as shown at step 506. Otherwise, at step 508 the wild-type sequence would pass by having sufficient intensity.</p>
    <p>At step 510 calculations are performed on the background subtracted base intensities of the reference sequence in order to "normalize" the intensities. Each position in the reference sequence has four background subtracted base intensities associated with it. The ratio of the intensity of each base to the sum of the intensities of the possible bases (all four) is calculated, resulting in four ratios, one for each base as shown in the data table. Thus, the following ratios would be calculated at each position in the reference sequence:</p>
    <p>A ratio=A/(A+C+G+T)</p>
    <p>C ratio=C/(A+C+G+T)</p>
    <p>G ratio=G/(A+C+G+T)</p>
    <p>T ratio=T/(A+C+G+T)</p>
    <p>At position 241, A ratio would be the wild-type ratio. These ratios are generally calculated in order to "normalize" the intensity data as the photon counts may vary widely from experiment to experiment. Thus, the ratios provide a way of reconciling the intensity variations across experiments. Preferably, if the photon counts do not vary widely from experiment to experiment, the probe intensities do not need to be "normalized."</p>
    <p>At step 512 the highest base intensity associated with the sample sequence is checked to see if it has sufficient intensity to call the unknown base. The intensity is checked by determining if the highest intensity sample base is greater than the background difference cutoff. If the intensity is not greater than the background difference cutoff, the sample sequence fails to have sufficient intensity as shown at step 514. Otherwise, at step 516 the sample sequence passes by having sufficient intensity.</p>
    <p>At step 518 calculations are performed on the background subtracted base intensities of the sample sequence in order to "normalize" the intensities. Each position in the sample sequence has four background subtracted base intensities associated with it. The ratios of the intensity of each base to the sum of the intensities of the possible bases (all four) are calculated, resulting in four ratios, one for each base as shown in the data table.</p>
    <p>At step 520 if either the reference or sample sequences failed to have sufficient intensity, the unknown base is assigned the code N (insufficient intensity) as shown at step 522.</p>
    <p>At step 524 the normalized base intensity ratios of the reference sequence are subtracted from the normalized base intensity ratios of the sample sequence. Thus, at each position the following calculations are performed:</p>
    <p>A Difference=Sample A Ratio-Reference A Ratio</p>
    <p>C Difference=Sample C Ratio-Reference C Ratio</p>
    <p>G Difference=Sample G Ratio-Reference G Ratio</p>
    <p>T Difference=Sample T Ratio-Reference T Ratio</p>
    <p>where the reference and sample ratios are calculated at steps 510 and 518, respectively. The base differences resulting from these calculations are shown in the data table.</p>
    <p>At step 526 each position is checked to see if there is a base difference greater than an upper difference cutoff and a base difference lower than a lower difference cutoff. For example, FIG. 5C shows a graph the normalized sample base intensities minus the normalized reference base intensities. Suppose that the upper difference cutoff is 0.15 and the lower difference cutoff is -0.15 as shown by the horizontal lines in FIG. 5C. At the mutation position (labeled with a reference 0), the G difference is 0.28 which is greater than 0.15, the upper difference cutoff. Similarly, the A difference is -0.32 which is less than -0.15, the lower difference cutoff. As there is a base difference above the upper difference cutoff and a base difference below the lower difference cutoff, there may be mutation at this position.</p>
    <p>If there is neither a base difference above the upper difference cutoff nor a base difference below the lower difference cutoff, the base at that position is assigned the code of the reference wild-type base as shown at step 528.</p>
    <p>At step 530 the ratio of the highest background subtracted base intensity in the sample to the background subtracted reference wild-type base intensity is calculated. For example, at the mutation position 241 in the data table, the highest background subtracted base intensity in the sample is 571 (base G). The background subtracted reference wild-type base intensity is 385 (base A). Thus, the ratio of 571:385 is calculated and results in 1.48 as shown in the data table.</p>
    <p>At step 532 these ratios are compared to a ratio at a neighboring position. The ratio for the n<sup>th</sup> position is subtracted from the ratio for the r<sup>th</sup> position, where r=n+1. For example, at the mutation position 241 in the data table, the ratio at position 242 (which equals 1.02) is subtracted from the ratio at position 241 (which equals 1.48). It has been found that a mutant can be confidently detected by analyzing the difference of these neighboring ratios.</p>
    <p>FIG. 5D shows other graphs of data in the data table. Of particular importance is the graph identified as 532 because this is a graph of the calculations at step 532. The pattern shown in a box in graph 532 has been found to be characteristic of a mutation. Thus, if this pattern is detected, the base is called as the base (or bases) with a normalized difference greater than the upper difference cutoff as shown at step 536. For example, the pattern was detected and at step 526 it was shown that base G had a normalized difference of 0.28, which is greater than the upper difference cutoff of 0.15. Therefore, the base at position 241 in the sample sequence is called a base G, which is a mutation from the reference sequence (A to G).</p>
    <p>If the pattern is not detected at step 534, the base at that position is assigned the code of the reference wild-type base as shown at step 538.</p>
    <p>This second implementation of the reference method is preferable in some instances as it takes into account probe intensities of neighboring probes. The first implementation may not have detected the A to G mutation in this example.</p>
    <p>The advantage of the reference method is that the correct base can be called even in the presence of significant levels of cross-hybridization, as long as ratios of intensities are fairly consistent from experiment to experiment. In practice, the number of miscalls and ambiguities is significantly reduced, while the number of correct calls is actually increased, making the reference method very useful for identifying candidate mutations. The reference method has also been used to compare the reproducibility of experiments in terms of base calling.</p>
    <p>IV. Statistical Method</p>
    <p>The statistical method is a method of calling bases in a sample nucleic acid sequence. The statistical method utilizes the statistical variation across experiments to call the bases. Therefore, the statistical method is preferable when data from multiple experiments is available and the data is fairly consistent across the experiments. The method compares the probe intensities of a sample sequence to statistics of probe intensities of a reference sequence in multiple experiments.</p>
    <p>For simplicity, the statistical method will be described as being used to identify one unknown base in a sample nucleic acid sequence. In practice, the method is used to identify many or all the bases in a nucleic acid sequence.</p>
    <p>The unknown base will be called by comparing the probe intensities of a sample sequence to statistics on probe intensities of a reference sequence in multiple experiments. Generally, the probe intensities of the sample sequence and the reference sequence experiments are from chips having the same chip wild-type. However, the reference sequence may or may not be equal to the chip wild-type, as it may have mutations.</p>
    <p>A base at the same position in the reference and sample sequences will be associated with up to four mutation probes and a "blank" cell. As before, because each mutation probe is identifiable by the mutation base, the mutation probes' intensities will be referred to as the "base intensities" of their respective mutation bases.</p>
    <p>As a simple example of the statistical method, suppose a gene of interest (target) has the sequence 5'-AAAACTGAAAA-3' (SEQ ID NO:4). Suppose a reference sequence has the sequence 5'-AAAACCGAAAA-3' (SEQ ID NO:5), which differs from the target sequence by the underlined base. Suppose further that a sample sequence is suspected to have the same sequence as the target sequence except for a T base mutation at the underlined base position in 5'-AAAACTGAAAA-3' (SEQ ID NO:4). Suppose that in multiple experiments the reference sequence is marked and exposed to probes on a chip. Suppose further the sample sequence is also marked and exposed to probes on a chip.</p>
    <p>The following are complementary "mutation" probes that could be used for a reference experiment and the sample sequence:</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">______________________________________  Reference     Sample______________________________________  3'-TGAC       3'-GACT  3'-TGCC       3'-GCCT  3'-TGGC       3'-GGCT  3'-TGTC       3'-GTCT______________________________________</pre>
    
    <p>The "mutation" probes shown for the reference sequence may be from only one experiment, the other experiments may have different "mutation" probes, chip wild-types, tiling methods, and the like. Although each fluorescence intensity is from a probe, since the probes may be identified by their unique mutation bases, the probe intensities may be identified by their respective bases as follows:</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">______________________________________Reference           Sample______________________________________3'-TGAC -&gt; A        3'-GACT -&gt; A3'-TGCC -&gt; C        3'-GCCT -&gt; C3'-TGGC -&gt; G        3'-GGCT -&gt; G3'-TGTC -&gt; T        3'-GTCT -&gt; T______________________________________</pre>
    
    <p>Thus, base A of the reference sequence will be described as having an intensity which corresponds to the intensity of the mutation probe with the mutation base A. The statistical method will now be described as calling the unknown base in the sample sequence by using this example.</p>
    <p>FIG. 6 illustrates the high level flow of the statistical method. At step 602 the four base intensities associated with the sample sequence and each of the multiple reference experiments are adjusted by subtracting the background or "blank" cell intensity from each base intensity. Preferably, if a base intensity is then less than or equal to zero, the base intensity is set equal to a small positive number to prevent division by zero or negative numbers.</p>
    <p>At step 604 the intensities of the reference wild-type bases in the multiple experiments are checked to see if they all have sufficient intensity to call the unknown base. The intensities are checked by determining if the intensity of the reference wild-type base of an experiment is greater than a predetermined background difference cutoff. The wild-type probe shown earlier for the reference sequence is 3'-TGGC, and thus the G base intensity is the wild-type base intensity. These steps are analogous to steps in the other two methods described herein.</p>
    <p>If the intensity of any one of the reference wild-type bases is not greater than the background difference cutoff, the wild-type experiments fail to have sufficient intensity as shown at step 606. Otherwise, at step 608 the wild-type experiments pass by having sufficient intensity.</p>
    <p>At step 610 calculations are performed on the background subtracted base intensities of each of the reference experiments in order to "normalize" the intensities. Each reference experiment has four background subtracted base intensities associated with it: one wild-type and three for the other possible bases. In this example, the G base intensity is the wild-type, the A, C, and T base intensities being the "other" intensities. The ratios of the intensity of each base to the sum of the intensities of the possible bases (all four) are calculated, giving one wild-type ratio and three "other" ratios. Thus, the following ratios would be calculated:</p>
    <p>A ratio=A/(A+C+G+T)</p>
    <p>C ratio=C/(A+C+G+T)</p>
    <p>G ratio=G/(A+C+G+T)</p>
    <p>T ratio=T/(A+C+G+T)</p>
    <p>where G ratio is the wild-type ratio and A, C, and T ratios are the "other" ratios. These four ratios are calculated for each reference experiment. Thus if the number of reference experiments is n, there would be 4n ratios calculated. These ratios are generally calculated in order to "normalize" the intensity data, as the photon counts may vary widely from experiment to experiment. However, if the probe intensities do not vary widely from experiment to experiment, the probe intensities do not need to be "normalized."</p>
    <p>At step 612 statistics are prepared for the ratios calculated for each of the reference experiments. As stated before, each reference experiment will be associated with one wild-type ratio and three "other" ratios. The mean and standard deviation are calculated for all the wild-type ratios. The mean and standard deviation are also calculated for each of the other ratios, resulting in three other means and standard deviations for each of the bases that is not the wild-type base. Therefore, the following would be calculated:</p>
    <p>Mean and standard deviation of A ratios</p>
    <p>Mean and standard deviation of C ratios</p>
    <p>Mean and standard deviation of G ratios</p>
    <p>Mean and standard deviation of T ratios</p>
    <p>where the mean and standard deviation of the G ratios are also known as the wild-type mean and the wild-type standard deviation, respectively. The mean and standard deviation of the A, C, and T means and standard deviations are also known collectively as the "other" means and standard deviations.</p>
    <p>Suppose that the preceding calculations produced the following data:</p>
    <p>A ratios→mean=0.16 std. dev.=0.003</p>
    <p>C ratios→mean=0.03 std. dev.=0.002</p>
    <p>G ratios→mean=0.71 std. dev.=0.050</p>
    <p>T ratios→mean=0.11 std. dev.=0.004</p>
    <p>In one embodiment, the steps up to and including step 612 are performed in a preprocessing stage for the multiple wild-type experiments. The results of the preprocessing stage are stored in a file so that the reference calculations do not have to be repeatedly calculated, improving performance. Microfiche Appendices C and D contain the programming code to perform the preprocessing stage.</p>
    <p>At step 614 the highest base intensity associated with the sample sequence is checked to see if it has sufficient intensity to call the unknown base. The intensity is checked by determining if the highest intensity unknown base is greater than the background difference cutoff. If the intensity is not greater than the background difference cutoff, the sample sequence fails to have sufficient intensity as shown at step 616. Otherwise, at step 618 the sample sequence passes by having sufficient intensity.</p>
    <p>At step 620 calculations are performed on the four background subtracted intensities of the sample sequence. The ratios of the background subtracted intensity of each base to the sum of the background subtracted intensities of the possible bases (all four) are calculated, giving four ratios, one for each base. For consistency, the ratio associated with the reference wild-type base is called the wild-type ratio, with there being three "other" ratios. Thus, the following ratios are calculated:</p>
    <p>A ratio=A/(A+C+G+T)</p>
    <p>C ratio=C/(A+C+G+T)</p>
    <p>G ratio=G/(A+C+G+T)</p>
    <p>T ratio=T/(A+C+G+T)</p>
    <p>where ratio G is the wild-type ratio and ratios A, C, and T are the "other" ratios.</p>
    <p>Suppose the background subtracted intensities associated with the sample are as follows:</p>
    <p>A→310</p>
    <p>C→50</p>
    <p>G→26</p>
    <p>T→100</p>
    <p>Then, the corresponding ratios would be as follows:</p>
    <p>A ratio=310/(310+50+26+100)=0.64</p>
    <p>C ratio=50/(310+50+26+100)=0.10</p>
    <p>G ratio=26/(310+50+26+100)=0.05</p>
    <p>T ratio=100/(310+50+26+100)=0.21</p>
    <p>At step 622 if either the reference experiments or the sample sequence failed to have sufficient intensity, the unknown base is assigned the code N (insufficient intensity) as shown at step 624.</p>
    <p>At step 626 the wild-type and "other" ratios associated with the sample sequence are compared to statistical expressions. The statistical expressions include four predetermined standard deviation cutoffs, one associated with each base. Thus, there is a standard deviation cutoff for each of the bases A, C, G, and T. The localized standard deviation cutoffs allow the unknown base to be called with higher precision because each standard deviation cutoff can be set to a different value. Suppose the standard deviation cutoffs are set as follows:</p>
    <p>A standard deviation cutoff→4</p>
    <p>C standard deviation cutoff→2</p>
    <p>G standard deviation cutoff→8</p>
    <p>T standard deviation cutoff→4</p>
    <p>The wild-type base ratio associated with the sample is compared to a corresponding statistical expression:</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="equation">WT ratio≧WT mean-(WT std. dev.*WT base std. dev. cutoff)</pre>
    
    <p>where the WT base std. dev. cutoff is the standard deviation cutoff for the wild-type base. As the wild-type base is G, the above comparison solves to the following:</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="equation">0.05≧0.71-(0.050*8)</pre>
    
    <p>
      </p> <pre xml:space="preserve" listing-type="equation">0.05≧0.31</pre>
    
    <p>which is not a true expression (0.05 is not greater than 0.31).</p>
    <p>Each of the "other" ratios associated with the sample is compared to a corresponding statistical expression:</p>
    <p>Other ratio&gt;Other mean+(Other std. dev.*Other base std. dev. cutoff)</p>
    <p>where the Other base std. dev. cutoff is the standard deviation cutoff for the particular "other" base. Thus, the above comparison solves to the following three expressions:</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="equation">A→0.64&gt;0.16+(0.003*4) 0.64&gt;0.17</pre>
    
    <p>
      </p> <pre xml:space="preserve" listing-type="equation">C→0.10&gt;0.03+(0.002*2) 0.10&gt;0.03</pre>
    
    <p>
      </p> <pre xml:space="preserve" listing-type="equation">T→0.21&gt;0.11+(0.004*4) 0.21&gt;0.13</pre>
    
    <p>which are all true expressions.</p>
    <p>At step 628 if only the wild-type ratio of the sample sequence was greater than the statistical expression, the unknown base is assigned the code of the reference wild-type base as shown at step 630.</p>
    <p>At step 632 if one or more of the "other" ratios of the sample sequence were greater than their respective statistical expressions, the unknown base is assigned an ambiguity code that indicates the unknown base may be any one of the complements of these bases, including the reference wild-type. In this example, the "other" ratios for A, C, and T were all greater than their corresponding statistical expression. Thus, the unknown base would be called the complements of these bases, represented by the subset T, G, and A. Thus, the unknown base would be assigned the code D (meaning "not C").</p>
    <p>If none of the ratios are greater than their respective statistical expressions, the unknown base is assigned the code X (insufficient discrimination) as shown at step 636.</p>
    <p>The statistical method provides accurate base calling because it utilizes statistical data from multiple reference experiments to call the unknown base. The statistical method has also been used to implement confidence estimates and calling of mixed sequences.</p>
    <p>V. Pooling Processing</p>
    <p>The present invention provides pooling processing which is a method of processing reference and sample nucleic acid sequences together to reduce variations across individual experiments. In the representative embodiment discussed herein, the reference and sample nucleic acid sequences are labeled with different fluorescent markers emitting light at different wavelengths. However, the nucleic acids may be labeled with other types of markers including distinguishable radioactive markers.</p>
    <p>After the reference and sample nucleic acid sequences are labeled with different color fluorescent markers, the labeled reference and sample nucleic acid sequences are then combined and processed together. An apparatus for detecting targets labeled with different markers is provided in U.S. application Ser. No. 08/195,889 and is hereby incorporated by reference for all purposes.</p>
    <p>FIG. 7 illustrates the pooling processing of a reference and sample nucleic acid sequence. At step 702 a reference nucleic acid sequence is marked with a fluorescent dye, such as a fluorescein. At step 704 a sample nucleic acid sequence is marked with a dye that, upon excitation, emits light of a different wavelength than that of the fluorescent dye of the reference sequence. For example, the sample nucleic acid sequence may be marked with rhodamine.</p>
    <p>At step 706 the labelel reference sequence and the labeled sample sequence are combined. After this step, processing continues in the same manner as for only one labeled sequence. At step 708 the sequences are fragmented. The fragmented nucleic acid sequences are then hybridized on a chip containing probes as shown at step 710.</p>
    <p>At step 712 a scanner generates image files that indicate the locations where the labeled nucleic acids bound to the chip. In general, the scanner generates an image file by focusing excitation light on the hybridized chip and detecting the fluorescent light that is emitted. The marker emitting the fluorescent light can be identified by the wavelength of the light. For example, the fluorescence peak of fluorescein is about 530 nm while that of a typical rhodamine dye is about 580 nm.</p>
    <p>The scanner creates an image file for the data associated with each fluorescent marker, indicating the locations where the correspondingly labeled nucleic acid bound to the chip. Based upon an analysis of the fluorescence intensities and locations, it becomes possible to extract information such as the monomer sequence of DNA or RNA.</p>
    <p>Pooling processing reduces variations across individual experiments because much of the test environment is common. Although pooling processing has been described as being used to improve the combined processing of reference and sample nucleic acid sequences, the process may also be used for two reference sequences, two sample sequences, or multiple sequences by utilizing multiple distinguishable markers.</p>
    <p>VI. Comparative Analysis (VIEWSEQ™)</p>
    <p>The present invention provides a method of comparative analysis and visualization of multiple experiments. The method allows the intensity ratio, reference, and statistical methods to be run on multiple datafiles simultaneously. This permits different experimental conditions, sample preparations, and analysis parameters to be compared in terms of their effects on sequence calling. The method also provides verification and editing functions, which are essential to reading sequences, as well as navigation and analysis tools.</p>
    <p>FIG. 8 illustrates the main screen and the associated pull down menus for comparative analysis and visualization of multiple experiments (SEQ ID NO:9). The windows shown are from an appropriately programmed Sun Workstation. However, the comparative analysis software may also be implemented on or ported to a personal computer, including IBM PCs and compatibles, or other workstation environments. A window 802 is shown having pull down menus for the following functions: File 804, Edit 806, View 808, Highlight 810, and Help 812.</p>
    <p>The main section of the window is divided into a reference sequence area 814 and a sample sequence area 816. The reference sequence area is where known sequences are displayed and is divided into a reference name subarea 818 and reference base subarea 820. The reference name subarea is shown with the filenames that contain the reference sequences. The chip wild-type is identified by the filename with the extension ".wt#" where the # indicates a unit on the chip. The reference base subarea contains the bases of the reference sequences. A capital C 822 is displayed to the right of the reference sequence that is the chip wild-type for the current analysis. Although the chip wild-type sequence has associated fluorescence intensities, the other reference sequences shown below the chip wild-type may be known sequences that have not been tiled on the chip. These may or may not have associated fluorescence intensities. The reference sequences other than the chip wild-type are used for sequence comparisons and may be in the form of simple ASCII text files.</p>
    <p>Sample sequence area 816 is where sample or unknown experimental sequences are displayed for comparison with the reference sequences. The sample sequence area is divided into a sample name subarea 824 and sample base subarea 826. The sample name subarea is shown with filenames that contain the sample sequences. The filename extensions indicate the method used to call the sample sequence where ".cq#" denotes the intensity ratio method, ".rq#" denotes the reference method, and ".sq#" denotes the statistical method (# indicates the unit on the chip). The sample base subarea contains the bases of the sample sequences. The bases of the sample sequences are identified by the codes previously set forth which, for the most part, conform to the IUPAC standard.</p>
    <p>Window 802 also contains a message panel 828. When the user selects a base with an input device in the reference or sample base subarea, the base becomes highlighted and the pathname of the file containing the base is displayed in the message panel. The base's position in the nucleic acid sequence is also displayed in the message panel.</p>
    <p>In pull down menu File 804, the user is able to load files of experimental sequences that have been tiled and scanned on a chip. There is a chip wild-type associated with each experimental sequence. The chip wild-type associated with the first experimental sequence loaded is read and shown as the chip wild-type in reference sequence area 814. The user is also able to load files of known nucleic acid sequences as reference sequences for comparison purposes. As before, these known reference sequences may or may not have associated probe intensity data. Additionally, in this menu the user is able to save sequences that are selected on the screen into a project file that can be loaded in at a later time. The project file also contains any linkage of the sequences, where sequences are linked for comparison purposes. Sequences to be saved, both reference and sample, are chosen by selecting the sequence filename with an input device in the reference or sample name subareas.</p>
    <p>In pull down menu Edit 806, the user is able to link together sequences in the reference and sample sequence areas. After the user has selected one reference and one or more sample sequences, the sample sequences can be linked to the reference sequence by selecting an entry in the pull down menu. Once the sequences are linked, a link number 830 is displayed next to each of the linked sequences. Each group of linked sequences is associated with a unique link number, so the user can easily identify which sequences are linked together. Linking sequences permits the user to more easily compare sequences of related interest. The user is also able to remove and display links in this menu.</p>
    <p>In pull down menu View 808, the user is able to display intensity graphs for selected bases. Once a base is selected in the reference or-sample base subareas, the user may request an intensity graph showing the hybridized probe intensities of the selected base and a delineated neighborhood of bases near the selected base. Intensity graphs may be displayed for one or multiple selected bases. The user is also able to prepare comment files and reports from this menu.</p>
    <p>FIG. 9 illustrates an intensity graph window for a selected base at position 120 (SEQ ID NO:31). The filename containing the sequence data is displayed at 904. The graph shows the intensities for each of the hybridized probes associated with a base. Each grouping of four vertical bars on the graph, which are labeled as "a", "ic", "g", and "t" on line 906, shows the background subtracted intensities of probes having the indicated substitution base. In one embodiment, the called bases are shown in red. The wild-type base is shown at line 908, the called base is shown at line 910, and the base position is shown at line 912. In FIG. 9, the base selected is at position 120, as shown by arrow 914. The wild-type base at this position is T; however, the called base is M which means the base is either A or C (amino). The user is able to use intensity graphs to visually compare the intensities of each of the possible calls.</p>
    <p>FIG. 10 illustrates multiple intensity graph windows for selected bases (SEQ ID NO:33, SEQ ID NO:34 and SEQ ID NO:35). There are three intensity graph windows 1002, 1004, and 1006 as shown. Each window may be associated with a different experiment, where the sequence analyzed in the experiment may be either a reference (if it has associated probe intensity data as in the chip wild-type) or a sample sequence. The windows are aligned and a rectangular box 1008 shows the selected bases' position in each of the sequences (position 162 in FIG. 10). The rectangular box aids the user in identifying the selected bases.</p>
    <p>Referring again to FIG. 8, in pull down menu Highlight 810, the user is able to compare the sequences of references and samples. At least four comparisons are available to the user, including the following: sample sequences to the chip wild-type sequence, sample sequences to any reference sequences, sample sequences to any linked reference sequences, and reference sequences to the chip wild-type sequence. For example, after the user has linked a reference and sample sequence, the user can compare the bases in the linked sequences. Bases in the sample sequence that are different from the reference sequence will then be indicated on the display device to the user (e.g., base is shown in a different color). In another example, the user is able to perform a comparison that will help identify sample sequences. After a sample is linked to multiple reference sequences, each base in the sample sequence that does not match the wild-type sequence is checked to see if it matches one of the linked reference sequences. The bases that match a linked reference sequence will then be indicated on the display device to the user. The user may then more easily identify the sample sequence as being one of the reference sequences.</p>
    <p>In pull down menu Help 812, the user is able to get information and instructions regarding the comparative analysis program, the calling methods, and the IUPAC definitions used in the program.</p>
    <p>FIG. 11 illustrates the intensity ratio method correctly calling a mutation in solutions with varying concentrations (SEQ ID NO:10, SEQ ID NO:11, SEQ ID NO:12, SEQ ID NO:13, SEQ ID NO:14, SEQ ID NO:15, SEQ ID NO:16, SEQ ID NO:17, and SEQ ID NO:18). A window 1102 is shown with a chip wild-type 1104 and a mutant sequence 1106. The mutant sequence differs from the chip wild-type at the position indicated by the rectangular box 1108. The chip wild-type and mutant sequences are a region of HIV Pol Gene spanning mutations occurring in AZT drug therapy.</p>
    <p>There are seven sample sequences that are called using the intensity ratio method. The sample sequences are actually solutions of different proportions of the chip wild-type sequence and the mutant sequence. Thus, there are sample solutions 1110, 1112, 1114, 1116, 1118, 1120, and 1122. The solutions are 15-mer tilings across the chip wild-type with increased percentages of the mutant sequence from 0 to 100% by weight. The following shows the proportions of the sample solutions:</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">______________________________________Sample Solution         Chip Wild-Type:Mutant______________________________________1110          100:01112          90:101114          75:251116          50:501118          25:751120          10:901122           0:100______________________________________</pre>
    
    <p>For example, sample solution 1114 contains 75% chip wild-type sequence and 25% mutant sequence.</p>
    <p>Now referring to the bases called in rectangular box 1108 for the sample solutions, the intensity ratio method correctly calls sample solution 1110 as having a base A as in the chip-wild type sequence. This is correct because sample solution 1110 is 100% chip wild-type sequence. The intensity ratio method also calls sample solution 1112 as having a base A because the sample solution is 90% chip wild-type sequence.</p>
    <p>The intensity ratio method calls the identified base in sample solutions 1114 and 1116 as being an R, which is an ambiguity IUPAC code denoting A or G (purine). This also a correct base call because the sample solutions have from 75% to 50% chip-wild type sequence and from 25% to 50% mutation sequence. Thus, the intensity ratio method correctly calls the base in this transition state.</p>
    <p>Sample solutions 1118, 1120, and 1122 are called by the intensity ratio method as having a mutation base G at the specified location. This is a correct base call because the sample solutions primarily consist of the mutation sequence (75%, 90%, and 100% respectively). Again, the intensity ratio method correctly called the bases.</p>
    <p>These experiments also show that the base calling methods of the present invention may also be used for solutions of more than one nucleic acid sequence.</p>
    <p>FIG. 12 illustrates the reference method correctly calling a mutant base where the intensity ratio method incorrectly called the mutant base (SEQ ID NO:36, SEQ ID NO:37, SEQ ID NO:38, and SEQ ID NO:39). There are three intensity graph windows 1202, 1204, and 1206 as shown. The windows are aligned and a rectangular box 1208 outlines the bases of interest. Window 1202 shows a sample sequence called using the intensity ratio method. However, the base in the rectangular box 1208 was incorrectly called base c, as there is actually a base A at that position. The intensity ratio method incorrectly called the base as C because the probe intensity associated with base C is much higher than the other probe intensities.</p>
    <p>Window 1204 shows a reference sequence called using the intensity ratio method. As the reference sequence is known, it is not necessary to know the method used to call the reference sequence. However, it is important to have probe intensities for a reference sequence to use the reference method. The reference sequence is called base C at the position indicated by the rectangular box.</p>
    <p>Window 1206 shows the sample sequence called using the reference method. The reference method correctly calls the specified base as being base A. Thus, for some cases the reference method is preferable to the intensity ratio method because it compares probe intensities of a sample sequence to probe intensities of a reference sequence.</p>
    <heading>VII. EXAMPLES</heading> <heading>Example 1</heading> <p>The intensity ratio method was used in sequence analysis of various polymorphic HIV-1 clones using a protease chip. Single stranded DNA of a 382 nt region was used with 4 different clones (HXB2, SF2, NY5, pPol4mut18). Results were compared to results from an ABI sequencer. The results are illustrated below:</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">______________________________________     ABI        Protease Chip       Sense  Antisense  Sense                              Antisense______________________________________No call     0      4          9    4Ambiguous   6      14         17   8Wrong call  2      3          3    1TOTAL       8      21         29   13______________________________________SUMMARY______________________________________     ABI (sense) - 99.5%     Chip (sense) - 98.1%     ABI (antisense) - 98.6%     Chip (antisense) - 99.1%______________________________________</pre>
    
    <heading>Example 2</heading> <p>HIV protease genotyping was performed using the described chips and CALLSEQ™ intensity ratio calculations. Samples were evaluated from AIDS patients before and after ddI treatment. Results were confirmed with ABI sequencing.</p>
    <p>FIG. 13 illustrates the output of the VIEWSEQ™ program with four pretreatment samples and four posttreatment samples (SEQ ID NO:19, SEQ ID NO:20, SEQ ID NO:21, SEQ ID NO:22, SEQ ID NO 23, SEQ ID NO:24, SEQ ID NO:25, SEQ ID NO:26, and SEQ ID NO:27). Note the base change at position 207 where a mutation has arisen. Even adjacent two additional mutations (gt), the "a" mutation has been properly detected.</p>
    <heading>VIII. APPENDICES</heading> <p>The Microfiche appendices (copyright Affymetrix, Inc.) provide C++ source code and header files for implementing the present invention. Appendix A contains the source code files (.cc files) for CALLSEQ™, which is a base calling program that implements the intensity ratio, reference, and statistical methods of the present invention. Appendix B contains the header files (.h files) for CALLSEQ™. Appendices C and D contain the source code and header files, respectively, for a program that performs a preprocessing stage for the statistical method of CALLSEQ™.</p>
    <p>Appendix E contains the source code and header files for VIEWSEQ™, which is a comparative analysis and visualization program according to the present invention. Appendices A-E are written for a Sun Workstation.</p>
    <p>The above description is illustrative and not restrictive. Many variations of the invention will become apparent to those of skill in the art upon review of this disclosure. Merely by way of example, while the invention is illustrated with particular reference to the evaluation of DNA (natural or unnatural), the methods can be used in the analysis from chips with other materials synthesized thereon, such as RNA. The scope of the invention should, therefore, be determined not with reference to the above description, but instead should be determined with reference to the appended claims along with their full scope of equivalents.</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">__________________________________________________________________________SEQUENCE LISTING(1) GENERAL INFORMATION:(iii) NUMBER OF SEQUENCES: 39(2) INFORMATION FOR SEQ ID NO:1:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 15 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(ii) MOLECULE TYPE: DNA (oligonucleotide)(xi) SEQUENCE DESCRIPTION: SEQ ID NO:1:ATGTGGACAGTTGTA15(2) INFORMATION FOR SEQ ID NO:2:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 15 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(ii) MOLECULE TYPE: DNA (oligonucleotide)(xi) SEQUENCE DESCRIPTION: SEQ ID NO:2:ATGTGGATAGTTGTA15(2) INFORMATION FOR SEQ ID NO:3:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 15 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(ii) MOLECULE TYPE: DNA (oligonucleotide)(xi) SEQUENCE DESCRIPTION: SEQ ID NO:3:ATGTGGAKAGTTGTA15(2) INFORMATION FOR SEQ ID NO:4:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 11 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(ii) MOLECULE TYPE: DNA (oligonucleotide)(xi) SEQUENCE DESCRIPTION: SEQ ID NO:4:AAAACTGAAAA11(2) INFORMATION FOR SEQ ID NO:5:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 11 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(ii) MOLECULE TYPE: DNA (oligonucleotide)(xi) SEQUENCE DESCRIPTION: SEQ ID NO:5:AAAACCGAAAA11(2) INFORMATION FOR SEQ ID NO:6:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 17 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(ii) MOLECULE TYPE: DNA (oligonucleotide)(xi) SEQUENCE DESCRIPTION: SEQ ID NO:6:AAACCCAATCCACATCA17(2) INFORMATION FOR SEQ ID NO:7:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 17 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(ii) MOLECULE TYPE: DNA (oligonucleotide)(xi) SEQUENCE DESCRIPTION: SEQ ID NO:7:AAACCCAGTCCACATCA17(2) INFORMATION FOR SEQ ID NO:8:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 31 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(ii) MOLECULE TYPE: DNA (oligonucleotide)(xi) SEQUENCE DESCRIPTION: SEQ ID NO:8:GGGGAAGCAGATTTGGGTACCACCCAAGTAT31(2) INFORMATION FOR SEQ ID NO:9:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 31 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(ii) MOLECULE TYPE: DNA (oligonucleotide)(xi) SEQUENCE DESCRIPTION: SEQ ID NO:9:GGGGAAGCAGATTTGAAMACCACCCAAGTAT31(2) INFORMATION FOR SEQ ID NO:10:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 59 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(ii) MOLECULE TYPE: DNA (oligonucleotide)(xi) SEQUENCE DESCRIPTION: SEQ ID NO:10:GCATTAGTAGAGATATGTACAGAAATGGAAAAGGAAGGGAAAATTTCAAAAATTGGGCC59(2) INFORMATION FOR SEQ ID NO:11:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 59 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(ii) MOLECULE TYPE: DNA (oligonucleotide)(xi) SEQUENCE DESCRIPTION: SEQ ID NO:11:GCATTAGTAGAAATTTGTACAGAGATGGAAAAGGAAGGGAAAATTTCAAAAATTGGGCC59(2) INFORMATION FOR SEQ ID NO:12:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 59 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(ii) MOLECULE TYPE: DNA (oligonucleotide)(xi) SEQUENCE DESCRIPTION: SEQ ID NO:12:GCATTAGTAGAGATATGGAGAGRARDGGRAANNNAAGGGAAAATTNNNAAAATTGGGCC59(2) INFORMATION FOR SEQ ID NO:13:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 59 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(ii) MOLECULE TYPE: DNA (oligonucleotide)(xi) SEQUENCE DESCRIPTION: SEQ ID NO:13:GCATTAGTAGAGATATGKASAGRARDGGRAANNNAAGGGAAAAKTNNNAAAATTGGGCC59(2) INFORMATION FOR SEQ ID NO:14:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 59 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(ii) MOLECULE TYPE: DNA (oligonucleotide)(xi) SEQUENCE DESCRIPTION: SEQ ID NO:14:GCATTAGTAGAGATATGKASAGRRRDGGRAANNNAAGGGAAAADTYNNAAAATTGGGCC59(2) INFORMATION FOR SEQ ID NO:15:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 59 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(ii) MOLECULE TYPE: DNA (oligonucleotide)(xi) SEQUENCE DESCRIPTION: SEQ ID NO:15:GCATTAGTAGAGATATGTASAGRRADGGAAANGGAAGGGAAAATTNNNNAAATTGGGCC59(2) INFORMATION FOR SEQ ID NO:16:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 59 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(ii) MOLECULE TYPE: DNA (oligonucleotide)(xi) SEQUENCE DESCRIPTION: SEQ ID NO:16:GCATTAGTAGAGATATGTACAGRGAGGGAAANGGAAGGGAAAATTNNNNAAATTGGGCC59(2) INFORMATION FOR SEQ ID NO:17:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 59 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(ii) MOLECULE TYPE: DNA (oligonucleotide)(xi) SEQUENCE DESCRIPTION: SEQ ID NO:17:GCATTAGTAGAGATATGTASAGRGAGGGAAANGGAAGGGAAAATTNNNNAAATTGGGCC59(2) INFORMATION FOR SEQ ID NO:18:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 59 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(ii) MOLECULE TYPE: DNA (oligonucleotide)(xi) SEQUENCE DESCRIPTION: SEQ ID NO:18:GCATTAGTAGGAGGNNNGACAGGGRKGGAAANNMAAGGGAAAAKTNNNAAAATTGGGCC59(2) INFORMATION FOR SEQ ID NO:19:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 160 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(ii) MOLECULE TYPE: DNA (oligonucleotide)(xi) SEQUENCE DESCRIPTION: SEQ ID NO:19:TCGAGATAATCTATGTCCTCGTCTACTATGTCATAATCTTCTTTACTTAAACGGTCCTTT60TACCTTTGGTTTTTACTATCCCCCTTAACCTCCAAAATAGTTTCATTCTGTCATGCTAGT120CTATGGACATCTTTAGACACCTGTATTTCGATATCCATGT160(2) INFORMATION FOR SEQ ID NO:20:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 160 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(ii) MOLECULE TYPE: DNA (oligonucleotide)(xi) SEQUENCE DESCRIPTION: SEQ ID NO:20:NNGAGATANNNTATGTCCTCGTCYACTATGTNANNNNNNNNNNNNNNNAAACGGTCCTNN60NNNNNNNNNNNNNNNNNNNNCNNCNTAACCTCCAAAATANNNNNNNTCTNNNNNANNNNT120CTANNNGNAGNNNNAGANARNCCNNNNNNNNNATNCATGT160(2) INFORMATION FOR SEQ ID NO:21:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 160 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(ii) MOLECULE TYPE: DNA (oligonucleotide)(xi) SEQUENCE DESCRIPTION: SEQ ID NO:21:TCGAGATAATCTATGTCCTCGTCTACTATGTCATAATNNNNNNNACTTAAACGGTCCTTT60TACCTTTGGTTTTTACTATCCCCCTTAACCTCCAAAATAGTTTCATTCTGNCATANNAGT120CTATGNGNNGNNNTAGACAGNCCNNNNTCGATATCCATGT160(2) INFORMATION FOR SEQ ID NO:22:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 160 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(ii) MOLECULE TYPE: DNA (oligonucleotide)(xi) SEQUENCE DESCRIPTION: SEQ ID NO:22:TCGAGATAATCTATGTCCTCGTCTACTATGTCATAATCTTCTTTACTTAAACGGTCCTTT60TACCTTTGGTTTTTACTATCCNNCTTAACCTCCAAAATAGTTTCATTCTGTCATACTAGT120CTATGGGTAGCTTTAGACCNCCGTATTTCGATATCCATGT160(2) INFORMATION FOR SEQ ID NO:23:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 160 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(ii) MOLECULE TYPE: DNA (oligonucleotide)(xi) SEQUENCE DESCRIPTION: SEQ ID NO:23:TCGAGATAATCTATGTCCTCGTCTACTATGTCATAATCTTCTTTACTTAAACGGTCCTTT60TACCTTTGGTTTTTACTATCCCNCTTAACCTCCAAAATAGTTTCATTCTGTCATACTAGT120CTATGGGTAGCTTTAGACCCCCGTATTTCGATATCCATGT160(2) INFORMATION FOR SEQ ID NO:24:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 160 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(ii) MOLECULE TYPE: DNA (oligonucleotide)(xi) SEQUENCE DESCRIPTION: SEQ ID NO:24:NCGGGATANTNTATGTCCTCGTCYACTATGTCANNNNNCNNNCNNNNCAAACGGTCCNCC60NNNNNCNNNNNNCNNCYANGAANCYCAACCTCCAAAATANNNNNNNTCTNNNNNANNNCN120CTNNNNNNAGNGNNAGACACCTGTATNNNNNTATNCAYGT160(2) INFORMATION FOR SEQ ID NO:25:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 160 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(ii) MOLECULE TYPE: DNA (oligonucleotide)(xi) SEQUENCE DESCRIPTION: SEQ ID NO:25:TCGRGATAATCTATGTCCTCGTCTACTATGTCATAATCCNNNCNNCTCAAACGGTCCTYC60CNNNNYTGGTTNYTACTATCCCCCTTAACCTCCAAAATAGTTTCATTCTGNCATACNNST120CTANNNNNAGNGTTAGACACCTGTATTTCGATATCCATGT160(2) INFORMATION FOR SEQ ID NO:26:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 160 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(ii) MOLECULE TYPE: DNA (oligonucleotide)(xi) SEQUENCE DESCRIPTION: SEQ ID NO:26:TCGAGATAATCTATGTCCTCGTCTACTATGTCATAATCCNNCCTACTCAAACGGTCCTTC60TACCTTTGGTTTTTACTATCCMCCTTAACCTCCAAAATAGTTTCATTCTGTCATACTAGT120CTATGAGTAGCTTTAGACACCTGTATTTCGATATCCATGT160(2) INFORMATION FOR SEQ ID NO:27:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 160 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(ii) MOLECULE TYPE: DNA (oligonucleotide)(xi) SEQUENCE DESCRIPTION: SEQ ID NO:27:TCGAGATAATCTATGTCCTCGTCTACTATGTCATAATCTTCTTTACYCAAACGGTCCTNC60TACCTTTGGTTTTTACTATCCCMCTTAACCTCCAAAATAGTTTCATTCTGTCATACTAGT120CTATGAGTAGCTTTAGACACCTGTATTTCGATATCCATGT160(2) INFORMATION FOR SEQ ID NO:28:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 17 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(ii) MOLECULE TYPE: DNA (oligonucleotide)(xi) SEQUENCE DESCRIPTION: SEQ ID NO:28:AAACCCAATCCACATCM17(2) INFORMATION FOR SEQ ID NO:29:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 17 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(ii) MOLECULE TYPE: DNA (oligonucleotide)(xi) SEQUENCE DESCRIPTION: SEQ ID NO:29:MMACNCANNCCACANNM17(2) INFORMATION FOR SEQ ID NO:30:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 11 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(ii) MOLECULE TYPE: DNA (oligonucleotide)(xi) SEQUENCE DESCRIPTION: SEQ ID NO:30:TTGGGTACCAC11(2) INFORMATION FOR SEQ ID NO:31:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 11 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(ii) MOLECULE TYPE: DNA (oligonucleotide)(xi) SEQUENCE DESCRIPTION: SEQ ID NO:31:TTGAAMACCAC11(2) INFORMATION FOR SEQ ID NO:32:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 11 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(ii) MOLECULE TYPE: DNA (oligonucleotide)(xi) SEQUENCE DESCRIPTION: SEQ ID NO:32:ACAGAAATGGA11(2) INFORMATION FOR SEQ ID NO:33:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 11 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(ii) MOLECULE TYPE: DNA (oligonucleotide)(xi) SEQUENCE DESCRIPTION: SEQ ID NO:33:AGAGRATDGGR11(2) INFORMATION FOR SEQ ID NO:34:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 11 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(ii) MOLECULE TYPE: DNA (oligonucleotide)(xi) SEQUENCE DESCRIPTION: SEQ ID NO:34:ASAGRRADGGA11(2) INFORMATION FOR SEQ ID NO:35:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 11 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(ii) MOLECULE TYPE: DNA (oligonucleotide)(xi) SEQUENCE DESCRIPTION: SEQ ID NO:35:ACAGGGRRGGA11(2) INFORMATION FOR SEQ ID NO:36:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 11 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(ii) MOLECULE TYPE: DNA (oligonucleotide)(xi) SEQUENCE DESCRIPTION: SEQ ID NO:36:CTGGGGGGTAT11(2) INFORMATION FOR SEQ ID NO:37:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 11 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(ii) MOLECULE TYPE: DNA (oligonucleotide)(xi) SEQUENCE DESCRIPTION: SEQ ID NO:37:CTGGCCSGTGT11(2) INFORMATION FOR SEQ ID NO:38:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 11 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(ii) MOLECULE TYPE: DNA (oligonucleotide)(xi) SEQUENCE DESCRIPTION: SEQ ID NO:38:CTGGGCGGTAT11(2) INFORMATION FOR SEQ ID NO:39:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 11 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(ii) MOLECULE TYPE: DNA (oligonucleotide)(xi) SEQUENCE DESCRIPTION: SEQ ID NO:39:CTGGCACGTGT11__________________________________________________________________________</pre>
    
    </div></div></div><div class="patent-section patent-tabular-section"><a id="backward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Cited Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4741043">US4741043</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Nov 4, 1985</td><td class="patent-data-table-td patent-date-value">Apr 26, 1988</td><td class="patent-data-table-td ">Cell Analysis Systems, Inc.</td><td class="patent-data-table-td ">Method of and an apparatus for image analyses of biological specimens</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4965725">US4965725</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Apr 8, 1988</td><td class="patent-data-table-td patent-date-value">Oct 23, 1990</td><td class="patent-data-table-td ">Nueromedical Systems, Inc.</td><td class="patent-data-table-td ">Neural network based automated cytological specimen classification system and method</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5002867">US5002867</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Oct 24, 1988</td><td class="patent-data-table-td patent-date-value">Mar 26, 1991</td><td class="patent-data-table-td ">Macevicz Stephen C</td><td class="patent-data-table-td ">Nucleic acid sequence determination by multiple mixed oligonucleotide probes</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5202231">US5202231</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jun 18, 1991</td><td class="patent-data-table-td patent-date-value">Apr 13, 1993</td><td class="patent-data-table-td ">Drmanac Radoje T</td><td class="patent-data-table-td ">Detecting probes which are exactly complementary to part of nucleic acid fragment being analyzed, then computerized comparison for sequence determination</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5273632">US5273632</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Nov 19, 1992</td><td class="patent-data-table-td patent-date-value">Dec 28, 1993</td><td class="patent-data-table-td ">University Of Utah Research Foundation</td><td class="patent-data-table-td ">Methods and apparatus for analysis of chromatographic migration patterns</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5384261">US5384261</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Nov 22, 1991</td><td class="patent-data-table-td patent-date-value">Jan 24, 1995</td><td class="patent-data-table-td ">Affymax Technologies N.V.</td><td class="patent-data-table-td ">Channel blocking on rotating stage, treatment with light</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5445934">US5445934</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Sep 30, 1992</td><td class="patent-data-table-td patent-date-value">Aug 29, 1995</td><td class="patent-data-table-td ">Affymax Technologies N.V.</td><td class="patent-data-table-td ">Selective photoremovable groups on surface</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5470710">US5470710</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Oct 22, 1993</td><td class="patent-data-table-td patent-date-value">Nov 28, 1995</td><td class="patent-data-table-td ">University Of Utah</td><td class="patent-data-table-td ">Automated hybridization/imaging device for fluorescent multiplex DNA sequencing</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5492806">US5492806</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Apr 12, 1993</td><td class="patent-data-table-td patent-date-value">Feb 20, 1996</td><td class="patent-data-table-td ">Hyseq, Inc.</td><td class="patent-data-table-td ">Method of determining an ordered sequence of subfragments of a nucleic acid fragment by hybridization of oligonucleotide probes</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5525464">US5525464</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Feb 28, 1994</td><td class="patent-data-table-td patent-date-value">Jun 11, 1996</td><td class="patent-data-table-td ">Hyseq, Inc.</td><td class="patent-data-table-td ">Method of sequencing by hybridization of oligonucleotide probes</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5527681">US5527681</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Nov 5, 1992</td><td class="patent-data-table-td patent-date-value">Jun 18, 1996</td><td class="patent-data-table-td ">Affymax Technologies N.V.</td><td class="patent-data-table-td ">Immobilized molecular synthesis of systematically substituted compounds</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1989010977A1?cl=en">WO1989010977A1</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">May 2, 1989</td><td class="patent-data-table-td patent-date-value">Nov 16, 1989</td><td class="patent-data-table-td ">Isis Innovation</td><td class="patent-data-table-td ">Analysing polynucleotide sequences</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1992010588A1?cl=en">WO1992010588A1</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Dec 6, 1991</td><td class="patent-data-table-td patent-date-value">Jun 25, 1992</td><td class="patent-data-table-td ">Affymax Tech Nv</td><td class="patent-data-table-td ">Sequencing by hybridization of a target nucleic acid to a matrix of defined oligonucleotides</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1995011995A1?cl=en">WO1995011995A1</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Oct 26, 1994</td><td class="patent-data-table-td patent-date-value">May 4, 1995</td><td class="patent-data-table-td ">Affymax Tech Nv</td><td class="patent-data-table-td ">Arrays of nucleic acid probes on biological chips</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1995035505A1?cl=en">WO1995035505A1</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jun 16, 1995</td><td class="patent-data-table-td patent-date-value">Dec 28, 1995</td><td class="patent-data-table-td ">Univ Leland Stanford Junior</td><td class="patent-data-table-td ">Method and apparatus for fabricating microarrays of biological samples</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="npl-citations"></a><div class="patent-section-header"><span class="patent-section-title">Non-Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th colspan="3"class="patent-data-table-th">Reference</th></tr></thead><tr><td class="patent-data-table-td ">1</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Drmanac et al. An algorithm for the DNA Sequence Generation form k Tuple Word Contents of the Minimal Number of Random Fragments. J. of Biomolecular Structure &amp; Dynamics 8: 1085 1102, 1991.</td></tr><tr><td class="patent-data-table-td ">2</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Drmanac et al. An algorithm for the DNA Sequence Generation form k-Tuple Word Contents of the Minimal Number of Random Fragments. J. of Biomolecular Structure &amp; Dynamics 8: 1085-1102, 1991.</td></tr><tr><td class="patent-data-table-td ">3</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Drmanac et al. DNA Sequence Determination by hybridization:A strategy for Effeicient Large Scale Sequencing. Science: 260: 1649 1652, 1993.</td></tr><tr><td class="patent-data-table-td ">4</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Drmanac et al. DNA Sequence Determination by hybridization:A strategy for Effeicient Large Scale Sequencing. Science: 260: 1649-1652, 1993.</td></tr><tr><td class="patent-data-table-td ">5</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Southern et al. Analyzing and Comparing Nucleic Acid Sequences by Hybridization to Arrays of Oligonucleotides: Evaluation using Experimentals Models. Genomics 13: 1008 1017, 1992.</td></tr><tr><td class="patent-data-table-td ">6</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Southern et al. Analyzing and Comparing Nucleic Acid Sequences by Hybridization to Arrays of Oligonucleotides: Evaluation using Experimentals Models. Genomics 13: 1008-1017, 1992.</td></tr><tr><td class="patent-data-table-td ">7</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Strezoska et al. DNA sequencing by Hybridization : 100 bases read by a non gel based method. PNAS 88: 10089 10093, 1991.</td></tr><tr><td class="patent-data-table-td ">8</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Strezoska et al. DNA sequencing by Hybridization : 100 bases read by a non-gel based method. PNAS 88: 10089-10093, 1991.</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="forward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Referenced by</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Citing Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6110426">US6110426</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Dec 30, 1997</td><td class="patent-data-table-td patent-date-value">Aug 29, 2000</td><td class="patent-data-table-td ">The Board Of Trustees Of The Leland Stanford Junior University</td><td class="patent-data-table-td ">Methods for fabricating microarrays of biological samples</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6171793">US6171793</a></td><td class="patent-data-table-td patent-date-value">Apr 19, 1999</td><td class="patent-data-table-td patent-date-value">Jan 9, 2001</td><td class="patent-data-table-td ">Affymetrix, Inc.</td><td class="patent-data-table-td ">Method for scanning gene probe array to produce data having dynamic range that exceeds that of scanner</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6187540">US6187540</a></td><td class="patent-data-table-td patent-date-value">Nov 9, 1998</td><td class="patent-data-table-td patent-date-value">Feb 13, 2001</td><td class="patent-data-table-td ">Identigene, Inc.</td><td class="patent-data-table-td ">Accuracy in classification of infants; obtain infant cells, store cells in protective device, convey protective device to genotyping building, evaluate protective device for rupture, genotype sample, genotype second sample and compare</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6207373">US6207373</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Feb 25, 1998</td><td class="patent-data-table-td patent-date-value">Mar 27, 2001</td><td class="patent-data-table-td ">Nanogen, Inc.</td><td class="patent-data-table-td ">Methods for determining nature of repeat units in DNA</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6228575">US6228575</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Feb 7, 1997</td><td class="patent-data-table-td patent-date-value">May 8, 2001</td><td class="patent-data-table-td ">Affymetrix, Inc.</td><td class="patent-data-table-td ">Classification of exogenous nucleotide seqences; mixing oligonucleotide probes and sample, analyze hybridization pattern and classify nucleotide sequences in sample</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6300063">US6300063</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">May 8, 1997</td><td class="patent-data-table-td patent-date-value">Oct 9, 2001</td><td class="patent-data-table-td ">Affymetrix, Inc.</td><td class="patent-data-table-td ">Polymorphism detection</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6395493">US6395493</a></td><td class="patent-data-table-td patent-date-value">Aug 24, 2000</td><td class="patent-data-table-td patent-date-value">May 28, 2002</td><td class="patent-data-table-td ">Nanogen, Inc.</td><td class="patent-data-table-td ">Screening sample and classification duplicated nuclelotide sequences; obtain nucleotide sequence array, incubate with sample, monitor positioning and hybridization patterns</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6451996">US6451996</a></td><td class="patent-data-table-td patent-date-value">Jan 21, 2000</td><td class="patent-data-table-td patent-date-value">Sep 17, 2002</td><td class="patent-data-table-td ">Callida Genomics, Inc.</td><td class="patent-data-table-td ">Easily robotized using computers</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6489096">US6489096</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Oct 15, 1998</td><td class="patent-data-table-td patent-date-value">Dec 3, 2002</td><td class="patent-data-table-td ">Princeton University</td><td class="patent-data-table-td ">Systems and methods for enhanced quantitative analysis of hybridization intensity measurements obtained from oligonucleotide probes and other probes exposed to target samples are provided by virtue of the present invention. one</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6516276">US6516276</a></td><td class="patent-data-table-td patent-date-value">Jun 18, 1999</td><td class="patent-data-table-td patent-date-value">Feb 4, 2003</td><td class="patent-data-table-td ">Eos Biotechnology, Inc.</td><td class="patent-data-table-td ">Generating probability values; data processing; computers</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6535624">US6535624</a></td><td class="patent-data-table-td patent-date-value">Sep 1, 1999</td><td class="patent-data-table-td patent-date-value">Mar 18, 2003</td><td class="patent-data-table-td ">Large Scale Proteomics Corporation</td><td class="patent-data-table-td ">Gel electrophoresis image combining for improved dynamic range</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6562565">US6562565</a></td><td class="patent-data-table-td patent-date-value">Jun 26, 1996</td><td class="patent-data-table-td patent-date-value">May 13, 2003</td><td class="patent-data-table-td ">The Regents Of The University Of California</td><td class="patent-data-table-td ">Determining copy number of preferential nucleotide sequences; generate solid support, conjugate with target sequences, incubate with nucleotide sequence samples, detect binding, compare to control</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6569598">US6569598</a></td><td class="patent-data-table-td patent-date-value">Apr 17, 2001</td><td class="patent-data-table-td patent-date-value">May 27, 2003</td><td class="patent-data-table-td ">Syntrix Biochip, Inc.</td><td class="patent-data-table-td ">Photoresist films comprising aromatic polyamides that are stable and impervious to solvents used as barrier layers for preferential solid-phase synthesis with micron-scale resolution</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6586186">US6586186</a></td><td class="patent-data-table-td patent-date-value">Aug 24, 2001</td><td class="patent-data-table-td patent-date-value">Jul 1, 2003</td><td class="patent-data-table-td ">Affymetrix, Inc.</td><td class="patent-data-table-td ">Nucleotide sequences for use in the detection of gene rearrangements</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6593133">US6593133</a></td><td class="patent-data-table-td patent-date-value">Jul 2, 1999</td><td class="patent-data-table-td patent-date-value">Jul 15, 2003</td><td class="patent-data-table-td ">Nsgene A/S</td><td class="patent-data-table-td ">Neurotrophic factors</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6600996">US6600996</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Mar 28, 1997</td><td class="patent-data-table-td patent-date-value">Jul 29, 2003</td><td class="patent-data-table-td ">Affymetrix, Inc.</td><td class="patent-data-table-td ">Computer-aided techniques for analyzing biological sequences</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6689563">US6689563</a></td><td class="patent-data-table-td patent-date-value">Dec 13, 2000</td><td class="patent-data-table-td patent-date-value">Feb 10, 2004</td><td class="patent-data-table-td ">Brown University Research Foundation</td><td class="patent-data-table-td ">System and methods for sequencing by hybridization</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6699659">US6699659</a></td><td class="patent-data-table-td patent-date-value">Dec 21, 1998</td><td class="patent-data-table-td patent-date-value">Mar 2, 2004</td><td class="patent-data-table-td ">Affymetrix, Inc.</td><td class="patent-data-table-td ">Genetic engineering; using computers</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6733964">US6733964</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Mar 27, 1998</td><td class="patent-data-table-td patent-date-value">May 11, 2004</td><td class="patent-data-table-td ">Affymetrix Inc.</td><td class="patent-data-table-td ">Computer-aided visualization and analysis system for sequence evaluation</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6734284">US6734284</a></td><td class="patent-data-table-td patent-date-value">Sep 15, 2000</td><td class="patent-data-table-td patent-date-value">May 11, 2004</td><td class="patent-data-table-td ">Nsgene A/S</td><td class="patent-data-table-td ">Exhibits high affinity for the gfr alpha 3- ret receptor complex and unique subregions in its amino acid sequence</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6753148">US6753148</a></td><td class="patent-data-table-td patent-date-value">Mar 25, 2002</td><td class="patent-data-table-td patent-date-value">Jun 22, 2004</td><td class="patent-data-table-td ">Nanogen, Inc.</td><td class="patent-data-table-td ">Methods and apparatus for detecting variants utilizing base stacking</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6824866">US6824866</a></td><td class="patent-data-table-td patent-date-value">Apr 7, 2000</td><td class="patent-data-table-td patent-date-value">Nov 30, 2004</td><td class="patent-data-table-td ">Affymetrix, Inc.</td><td class="patent-data-table-td ">Porous silica substrates for polymer synthesis and assays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6902921">US6902921</a></td><td class="patent-data-table-td patent-date-value">May 23, 2002</td><td class="patent-data-table-td patent-date-value">Jun 7, 2005</td><td class="patent-data-table-td ">454 Corporation</td><td class="patent-data-table-td ">Genetically engineered adenosine triphosphate generating polypeptide; sequences determination; reporter molecules</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6924094">US6924094</a></td><td class="patent-data-table-td patent-date-value">Nov 28, 2000</td><td class="patent-data-table-td patent-date-value">Aug 2, 2005</td><td class="patent-data-table-td ">Affymetrix, Inc.</td><td class="patent-data-table-td ">Chip-based species identification and phenotypic characterization of microorganisms</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6936416">US6936416</a></td><td class="patent-data-table-td patent-date-value">Sep 12, 2001</td><td class="patent-data-table-td patent-date-value">Aug 30, 2005</td><td class="patent-data-table-td ">Affymetrix, Inc.</td><td class="patent-data-table-td ">Expression monitoring for human cytomegalovirus (HCMV) infection</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6941317">US6941317</a></td><td class="patent-data-table-td patent-date-value">Sep 14, 1999</td><td class="patent-data-table-td patent-date-value">Sep 6, 2005</td><td class="patent-data-table-td ">Eragen Biosciences, Inc.</td><td class="patent-data-table-td ">Graphical user interface for display and analysis of biological sequence data</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6942968">US6942968</a></td><td class="patent-data-table-td patent-date-value">Aug 30, 2000</td><td class="patent-data-table-td patent-date-value">Sep 13, 2005</td><td class="patent-data-table-td ">Illumina, Inc.</td><td class="patent-data-table-td ">A layout comprising substrate with an array of geometrically shaped well, containing bioactive agent and signal transducer element, surfaces coated with reflective spectrum absorptive coatings, for detecting a target analyte in a sample</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6953663">US6953663</a></td><td class="patent-data-table-td patent-date-value">Jun 27, 2003</td><td class="patent-data-table-td patent-date-value">Oct 11, 2005</td><td class="patent-data-table-td ">Affymetrix, Inc.</td><td class="patent-data-table-td ">Polymorphism detection</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6956114">US6956114</a></td><td class="patent-data-table-td patent-date-value">Apr 11, 2002</td><td class="patent-data-table-td patent-date-value">Oct 18, 2005</td><td class="patent-data-table-td ">&#39;454 Corporation</td><td class="patent-data-table-td ">Sulfurylase-luciferase fusion proteins and thermostable sulfurylase</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6993173">US6993173</a></td><td class="patent-data-table-td patent-date-value">Sep 30, 2002</td><td class="patent-data-table-td patent-date-value">Jan 31, 2006</td><td class="patent-data-table-td ">Duke University</td><td class="patent-data-table-td ">Methods for estimating probe cell locations in high-density synthetic DNA microarrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7034143">US7034143</a></td><td class="patent-data-table-td patent-date-value">Oct 13, 1999</td><td class="patent-data-table-td patent-date-value">Apr 25, 2006</td><td class="patent-data-table-td ">Brown University Research Foundation</td><td class="patent-data-table-td ">Systems and methods for sequencing by hybridization</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7094609">US7094609</a></td><td class="patent-data-table-td patent-date-value">Sep 19, 1997</td><td class="patent-data-table-td patent-date-value">Aug 22, 2006</td><td class="patent-data-table-td ">Burstein Technologies, Inc.</td><td class="patent-data-table-td ">Spatially addressable combinatorial chemical arrays in encoded optical disk format</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7108968">US7108968</a></td><td class="patent-data-table-td patent-date-value">Apr 2, 1999</td><td class="patent-data-table-td patent-date-value">Sep 19, 2006</td><td class="patent-data-table-td ">Affymetrix, Inc.</td><td class="patent-data-table-td ">Mycobacterial rpoB sequences</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7209836">US7209836</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jul 16, 1999</td><td class="patent-data-table-td patent-date-value">Apr 24, 2007</td><td class="patent-data-table-td ">Perkinelmer Las, Inc.</td><td class="patent-data-table-td ">Method and system for automatically creating crosstalk-corrected data of a microarray</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7211390">US7211390</a></td><td class="patent-data-table-td patent-date-value">Feb 26, 2004</td><td class="patent-data-table-td patent-date-value">May 1, 2007</td><td class="patent-data-table-td ">454 Life Sciences Corporation</td><td class="patent-data-table-td ">Method of sequencing a nucleic acid</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7244559">US7244559</a></td><td class="patent-data-table-td patent-date-value">Mar 21, 2001</td><td class="patent-data-table-td patent-date-value">Jul 17, 2007</td><td class="patent-data-table-td ">454 Life Sciences Corporation</td><td class="patent-data-table-td ">Sequences determination of preferential nucleotide sequences; provide primers and nucleotide sequence patterns, mix, anneal primer to nucleotide sequences, mix nucleotides, determine sequence of sample</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7252948">US7252948</a></td><td class="patent-data-table-td patent-date-value">Feb 4, 2005</td><td class="patent-data-table-td patent-date-value">Aug 7, 2007</td><td class="patent-data-table-td ">Affymetrix, Inc.</td><td class="patent-data-table-td ">Using oligonucleotide sequences based on the Mycobacterium tubercluosis rpoB gene by comparing hybridization patterns of target nucleic organism to hybridization patterns in a database; diagnosis of drug resistance in HIV patients</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7264929">US7264929</a></td><td class="patent-data-table-td patent-date-value">Nov 19, 2002</td><td class="patent-data-table-td patent-date-value">Sep 4, 2007</td><td class="patent-data-table-td ">454 Life Sciences Corporation</td><td class="patent-data-table-td ">Method of sequencing a nucleic acid</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7276580">US7276580</a></td><td class="patent-data-table-td patent-date-value">Sep 12, 2003</td><td class="patent-data-table-td patent-date-value">Oct 2, 2007</td><td class="patent-data-table-td ">Biogen Idec Ma Inc.</td><td class="patent-data-table-td ">Neurotrophic factors</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7323298">US7323298</a></td><td class="patent-data-table-td patent-date-value">Jul 30, 1996</td><td class="patent-data-table-td patent-date-value">Jan 29, 2008</td><td class="patent-data-table-td ">The Board Of Trustees Of The Leland Stanford Junior University</td><td class="patent-data-table-td ">Microarray for determining the relative abundances of polynuceotide sequences</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7323305">US7323305</a></td><td class="patent-data-table-td patent-date-value">Sep 22, 2004</td><td class="patent-data-table-td patent-date-value">Jan 29, 2008</td><td class="patent-data-table-td ">454 Life Sciences Corporation</td><td class="patent-data-table-td ">Determining sequence of preferential nucleotide sequences comprising, fragmenting large template nucleic, deliver to microreation chamber filled with water-in-oil emulsion, amplify and bind copies to beads, incubate with array and sequence</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7323307">US7323307</a></td><td class="patent-data-table-td patent-date-value">Aug 4, 2004</td><td class="patent-data-table-td patent-date-value">Jan 29, 2008</td><td class="patent-data-table-td ">Affymetrix, Inc.</td><td class="patent-data-table-td ">Classifying messenger RNA sequences and identifying exon splice junctions for use in prediciting peptide binding domain and protein active site; protein engineering</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7335762">US7335762</a></td><td class="patent-data-table-td patent-date-value">Aug 15, 2002</td><td class="patent-data-table-td patent-date-value">Feb 26, 2008</td><td class="patent-data-table-td ">454 Life Sciences Corporation</td><td class="patent-data-table-td ">Cuvette for use in the concurrent monitoring, recording and analysis of lignt emission in microarrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7358228">US7358228</a></td><td class="patent-data-table-td patent-date-value">Dec 21, 2006</td><td class="patent-data-table-td patent-date-value">Apr 15, 2008</td><td class="patent-data-table-td ">Biogen Idec Ma Inc.</td><td class="patent-data-table-td ">Methods of treating pain using neurotrophic factors</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7378236">US7378236</a></td><td class="patent-data-table-td patent-date-value">Aug 14, 1995</td><td class="patent-data-table-td patent-date-value">May 27, 2008</td><td class="patent-data-table-td ">The Board Of Trustees Of The Leland Stanford Junior University</td><td class="patent-data-table-td ">Method for analyzing gene expression patterns</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7442370">US7442370</a></td><td class="patent-data-table-td patent-date-value">Jan 31, 2003</td><td class="patent-data-table-td patent-date-value">Oct 28, 2008</td><td class="patent-data-table-td ">Biogen Idec Ma Inc.</td><td class="patent-data-table-td ">Polymer conjugates of mutated neublastin</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7499806">US7499806</a></td><td class="patent-data-table-td patent-date-value">Feb 14, 2003</td><td class="patent-data-table-td patent-date-value">Mar 3, 2009</td><td class="patent-data-table-td ">Illumina, Inc.</td><td class="patent-data-table-td ">Image processing in microsphere arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7575865">US7575865</a></td><td class="patent-data-table-td patent-date-value">Aug 1, 2005</td><td class="patent-data-table-td patent-date-value">Aug 18, 2009</td><td class="patent-data-table-td ">454 Life Sciences Corporation</td><td class="patent-data-table-td ">Methods of amplifying and sequencing nucleic acids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7582420">US7582420</a></td><td class="patent-data-table-td patent-date-value">Jul 12, 2002</td><td class="patent-data-table-td patent-date-value">Sep 1, 2009</td><td class="patent-data-table-td ">Illumina, Inc.</td><td class="patent-data-table-td ">Immobilization; amplification, genotype; detecting infections</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7595883">US7595883</a></td><td class="patent-data-table-td patent-date-value">Sep 16, 2003</td><td class="patent-data-table-td patent-date-value">Sep 29, 2009</td><td class="patent-data-table-td ">The Board Of Trustees Of The Leland Stanford Junior University</td><td class="patent-data-table-td ">Biological analysis arrangement and approach therefor</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7608395">US7608395</a></td><td class="patent-data-table-td patent-date-value">Sep 15, 2005</td><td class="patent-data-table-td patent-date-value">Oct 27, 2009</td><td class="patent-data-table-td ">Baylor Research Institute</td><td class="patent-data-table-td ">Predicts the likelihood of renal involvement, diagnose SLE, distinguish SLE gene expression profiles from profiles of patients infected with influenza, and to monitor disease progression and treatment</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7622281">US7622281</a></td><td class="patent-data-table-td patent-date-value">May 19, 2005</td><td class="patent-data-table-td patent-date-value">Nov 24, 2009</td><td class="patent-data-table-td ">The Board Of Trustees Of The Leland Stanford Junior University</td><td class="patent-data-table-td ">Methods and compositions for clonal amplification of nucleic acid</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7625697">US7625697</a></td><td class="patent-data-table-td patent-date-value">Jul 17, 2001</td><td class="patent-data-table-td patent-date-value">Dec 1, 2009</td><td class="patent-data-table-td ">The Board Of Trustees Of The Leland Stanford Junior University</td><td class="patent-data-table-td ">Constructing gene layouts; prepare reporter molecules, mix reporter molecules with test and control cells, compare pattern of reporter molecule expression, monitor expression pattern of reporter molecules from test cells</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7634363">US7634363</a></td><td class="patent-data-table-td patent-date-value">Dec 7, 2006</td><td class="patent-data-table-td patent-date-value">Dec 15, 2009</td><td class="patent-data-table-td ">Affymetrix, Inc.</td><td class="patent-data-table-td ">Polymorphisms; microarrays; hybridization intensity; point mutations</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7647186">US7647186</a></td><td class="patent-data-table-td patent-date-value">Jul 15, 2005</td><td class="patent-data-table-td patent-date-value">Jan 12, 2010</td><td class="patent-data-table-td ">Illumina, Inc.</td><td class="patent-data-table-td ">Oligonucleotide ordering system</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7655463">US7655463</a></td><td class="patent-data-table-td patent-date-value">Feb 12, 2008</td><td class="patent-data-table-td patent-date-value">Feb 2, 2010</td><td class="patent-data-table-td ">Biogen Idec Ma Inc.</td><td class="patent-data-table-td ">Treating neuropathic pain, particularly diabetic peripheral neuropathy, by administering a truncated neublastin polypeptide; activating a RET receptor tyrosine kinase</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7674587">US7674587</a></td><td class="patent-data-table-td patent-date-value">Feb 16, 2007</td><td class="patent-data-table-td patent-date-value">Mar 9, 2010</td><td class="patent-data-table-td ">Affymetrix, Inc.</td><td class="patent-data-table-td ">Analyzing the hybridization intensities, by using oligonucleotide probes, which is complementary to target nucleic acid sequence, using principal component analysis; genetic engineering ; rapid efficient method for analyzing polymorphic or biallelic markers</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7881875">US7881875</a></td><td class="patent-data-table-td patent-date-value">Jul 13, 2006</td><td class="patent-data-table-td patent-date-value">Feb 1, 2011</td><td class="patent-data-table-td ">Affymetrix, Inc.</td><td class="patent-data-table-td ">Methods for selecting a collection of single nucleotide polymorphisms</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7914988">US7914988</a></td><td class="patent-data-table-td patent-date-value">Apr 2, 2007</td><td class="patent-data-table-td patent-date-value">Mar 29, 2011</td><td class="patent-data-table-td ">Illumina, Inc.</td><td class="patent-data-table-td ">Gene expression profiles to predict relapse of prostate cancer</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7955794">US7955794</a></td><td class="patent-data-table-td patent-date-value">Jun 20, 2002</td><td class="patent-data-table-td patent-date-value">Jun 7, 2011</td><td class="patent-data-table-td ">Illumina, Inc.</td><td class="patent-data-table-td ">Using gene probe assay to amplify and/or genotype a variety of nucleotide sequences samples simultaneously</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7962291">US7962291</a></td><td class="patent-data-table-td patent-date-value">Oct 2, 2006</td><td class="patent-data-table-td patent-date-value">Jun 14, 2011</td><td class="patent-data-table-td ">Affymetrix, Inc.</td><td class="patent-data-table-td ">Methods and computer software for detecting splice variants</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7991564">US7991564</a></td><td class="patent-data-table-td patent-date-value">Dec 15, 2009</td><td class="patent-data-table-td patent-date-value">Aug 2, 2011</td><td class="patent-data-table-td ">Affymetrix, Inc.</td><td class="patent-data-table-td ">Methods for high throughput genotyping</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8003354">US8003354</a></td><td class="patent-data-table-td patent-date-value">Feb 15, 2008</td><td class="patent-data-table-td patent-date-value">Aug 23, 2011</td><td class="patent-data-table-td ">Illumina, Inc.</td><td class="patent-data-table-td ">Multiplex nucleic acid reactions</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8023113">US8023113</a></td><td class="patent-data-table-td patent-date-value">Apr 6, 2009</td><td class="patent-data-table-td patent-date-value">Sep 20, 2011</td><td class="patent-data-table-td ">The Board Of Trustees Of The Leland Stanford Junior University</td><td class="patent-data-table-td ">Biological analysis arrangement and approach therefor</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8039817">US8039817</a></td><td class="patent-data-table-td patent-date-value">May 1, 2009</td><td class="patent-data-table-td patent-date-value">Oct 18, 2011</td><td class="patent-data-table-td ">Illumina, Inc.</td><td class="patent-data-table-td ">Compensator for multiple surface imaging</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8071962">US8071962</a></td><td class="patent-data-table-td patent-date-value">Aug 12, 2011</td><td class="patent-data-table-td patent-date-value">Dec 6, 2011</td><td class="patent-data-table-td ">Illumina, Inc.</td><td class="patent-data-table-td ">Compensator for multiple surface imaging</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8073666">US8073666</a></td><td class="patent-data-table-td patent-date-value">Dec 22, 2009</td><td class="patent-data-table-td patent-date-value">Dec 6, 2011</td><td class="patent-data-table-td ">Illumina, Inc.</td><td class="patent-data-table-td ">Systems and methods for ordering oligonucleotides</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8076063">US8076063</a></td><td class="patent-data-table-td patent-date-value">Dec 3, 2003</td><td class="patent-data-table-td patent-date-value">Dec 13, 2011</td><td class="patent-data-table-td ">Illumina, Inc.</td><td class="patent-data-table-td ">Multiplexed methylation detection methods</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8095323">US8095323</a></td><td class="patent-data-table-td patent-date-value">Jan 25, 2010</td><td class="patent-data-table-td patent-date-value">Jan 10, 2012</td><td class="patent-data-table-td ">Affymetrix, Inc.</td><td class="patent-data-table-td ">Polymorphism detection</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8110363">US8110363</a></td><td class="patent-data-table-td patent-date-value">Feb 25, 2011</td><td class="patent-data-table-td patent-date-value">Feb 7, 2012</td><td class="patent-data-table-td ">Illumina, Inc.</td><td class="patent-data-table-td ">Expression profiles to predict relapse of prostate cancer</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8119114">US8119114</a></td><td class="patent-data-table-td patent-date-value">Jun 27, 2008</td><td class="patent-data-table-td patent-date-value">Feb 21, 2012</td><td class="patent-data-table-td ">Biogen Idec Ma Inc.</td><td class="patent-data-table-td ">Polymer conjugates of mutated neublastin</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8143599">US8143599</a></td><td class="patent-data-table-td patent-date-value">Oct 25, 2011</td><td class="patent-data-table-td patent-date-value">Mar 27, 2012</td><td class="patent-data-table-td ">Illumina, Inc.</td><td class="patent-data-table-td ">Compensator for multiple surface imaging</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8158359">US8158359</a></td><td class="patent-data-table-td patent-date-value">Jun 24, 2010</td><td class="patent-data-table-td patent-date-value">Apr 17, 2012</td><td class="patent-data-table-td ">454 Lice Sciences Corporation</td><td class="patent-data-table-td ">Methods of amplifying and sequencing nucleic acids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8163875">US8163875</a></td><td class="patent-data-table-td patent-date-value">Apr 16, 2004</td><td class="patent-data-table-td patent-date-value">Apr 24, 2012</td><td class="patent-data-table-td ">Biogen Idec Ma Inc.</td><td class="patent-data-table-td ">Polymer conjugated glycosylated neublastin</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8170808">US8170808</a></td><td class="patent-data-table-td patent-date-value">May 4, 2011</td><td class="patent-data-table-td patent-date-value">May 1, 2012</td><td class="patent-data-table-td ">Affymetrix, Inc.</td><td class="patent-data-table-td ">Methods and computer software for detecting splice variants</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8200440">US8200440</a></td><td class="patent-data-table-td patent-date-value">May 19, 2008</td><td class="patent-data-table-td patent-date-value">Jun 12, 2012</td><td class="patent-data-table-td ">Affymetrix, Inc.</td><td class="patent-data-table-td ">System, method, and computer software product for genotype determination using probe array data</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8217146">US8217146</a></td><td class="patent-data-table-td patent-date-value">Dec 7, 2009</td><td class="patent-data-table-td patent-date-value">Jul 10, 2012</td><td class="patent-data-table-td ">Biogen Idec Ma Inc.</td><td class="patent-data-table-td ">Neurotrophic factors and methods of use thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8242463">US8242463</a></td><td class="patent-data-table-td patent-date-value">Feb 17, 2012</td><td class="patent-data-table-td patent-date-value">Aug 14, 2012</td><td class="patent-data-table-td ">Illumina, Inc.</td><td class="patent-data-table-td ">Compensator for multiple surface imaging</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8263553">US8263553</a></td><td class="patent-data-table-td patent-date-value">Aug 18, 2005</td><td class="patent-data-table-td patent-date-value">Sep 11, 2012</td><td class="patent-data-table-td ">Biogen Idec Ma Inc.</td><td class="patent-data-table-td ">Neublastin variants</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8278630">US8278630</a></td><td class="patent-data-table-td patent-date-value">Jul 9, 2012</td><td class="patent-data-table-td patent-date-value">Oct 2, 2012</td><td class="patent-data-table-td ">Illumina, Inc.</td><td class="patent-data-table-td ">Compensator for multiple surface imaging</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8293684">US8293684</a></td><td class="patent-data-table-td patent-date-value">Nov 29, 2007</td><td class="patent-data-table-td patent-date-value">Oct 23, 2012</td><td class="patent-data-table-td ">Exiqon</td><td class="patent-data-table-td ">Locked nucleic acid reagents for labelling nucleic acids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8313904">US8313904</a></td><td class="patent-data-table-td patent-date-value">Nov 26, 2008</td><td class="patent-data-table-td patent-date-value">Nov 20, 2012</td><td class="patent-data-table-td ">The Board Of Trustees Of The Leland Stanford Junior University</td><td class="patent-data-table-td ">Biological analysis arrangement and approach therefor</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8329655">US8329655</a></td><td class="patent-data-table-td patent-date-value">May 1, 2008</td><td class="patent-data-table-td patent-date-value">Dec 11, 2012</td><td class="patent-data-table-td ">Biogen Idec Ma Inc.</td><td class="patent-data-table-td ">Methods for increasing vascularization</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8440407">US8440407</a></td><td class="patent-data-table-td patent-date-value">Dec 15, 2011</td><td class="patent-data-table-td patent-date-value">May 14, 2013</td><td class="patent-data-table-td ">Illumina, Inc.</td><td class="patent-data-table-td ">Gene expression profiles to predict relapse of prostate cancer</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8481259">US8481259</a></td><td class="patent-data-table-td patent-date-value">Mar 8, 2010</td><td class="patent-data-table-td patent-date-value">Jul 9, 2013</td><td class="patent-data-table-td ">Intelligent Bio-Systems, Inc.</td><td class="patent-data-table-td ">Methods and devices for sequencing nucleic acids in smaller batches</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8498825">US8498825</a></td><td class="patent-data-table-td patent-date-value">Jun 23, 2011</td><td class="patent-data-table-td patent-date-value">Jul 30, 2013</td><td class="patent-data-table-td ">Affymetrix, Inc.</td><td class="patent-data-table-td ">Methods for high throughput genotyping</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8546772">US8546772</a></td><td class="patent-data-table-td patent-date-value">Sep 28, 2012</td><td class="patent-data-table-td patent-date-value">Oct 1, 2013</td><td class="patent-data-table-td ">Illumina, Inc.</td><td class="patent-data-table-td ">Compensator for multiple surface imaging</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8586947">US8586947</a></td><td class="patent-data-table-td patent-date-value">Aug 23, 2013</td><td class="patent-data-table-td patent-date-value">Nov 19, 2013</td><td class="patent-data-table-td ">Illumina, Inc.</td><td class="patent-data-table-td ">Compensator for multiple surface imaging</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8612161">US8612161</a></td><td class="patent-data-table-td patent-date-value">Mar 17, 2009</td><td class="patent-data-table-td patent-date-value">Dec 17, 2013</td><td class="patent-data-table-td ">Intelligent Biosystems Inc.</td><td class="patent-data-table-td ">Methods and compositions for base calling nucleic acids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8623598">US8623598</a></td><td class="patent-data-table-td patent-date-value">Mar 17, 2009</td><td class="patent-data-table-td patent-date-value">Jan 7, 2014</td><td class="patent-data-table-td ">Intelligent Bio Systems, Inc.</td><td class="patent-data-table-td ">Methods and compositions for inhibiting undesired cleaving of labels</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8642732">US8642732</a></td><td class="patent-data-table-td patent-date-value">Mar 21, 2012</td><td class="patent-data-table-td patent-date-value">Feb 4, 2014</td><td class="patent-data-table-td ">Biogen Idec Ma Inc.</td><td class="patent-data-table-td ">Polymer-conjugated glycosylated neublastin</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8698102">US8698102</a></td><td class="patent-data-table-td patent-date-value">Oct 17, 2013</td><td class="patent-data-table-td patent-date-value">Apr 15, 2014</td><td class="patent-data-table-td ">Illumina, Inc.</td><td class="patent-data-table-td ">Compensator for multiple surface imaging</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8709788">US8709788</a></td><td class="patent-data-table-td patent-date-value">Nov 19, 2012</td><td class="patent-data-table-td patent-date-value">Apr 29, 2014</td><td class="patent-data-table-td ">The Board Of Trustees Of The Leland Stanford Junior University</td><td class="patent-data-table-td ">Biological analysis arrangement and approach therefor</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8722862">US8722862</a></td><td class="patent-data-table-td patent-date-value">Aug 18, 2005</td><td class="patent-data-table-td patent-date-value">May 13, 2014</td><td class="patent-data-table-td ">Biogen Idec Ma Inc.</td><td class="patent-data-table-td ">Refolding transforming growth factor beta family proteins</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8741630">US8741630</a></td><td class="patent-data-table-td patent-date-value">Dec 17, 2010</td><td class="patent-data-table-td patent-date-value">Jun 3, 2014</td><td class="patent-data-table-td ">Illumina, Inc.</td><td class="patent-data-table-td ">Methods of detecting targets on an array</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8759311">US8759311</a></td><td class="patent-data-table-td patent-date-value">Oct 1, 2010</td><td class="patent-data-table-td patent-date-value">Jun 24, 2014</td><td class="patent-data-table-td ">Sanofi</td><td class="patent-data-table-td ">Use of siRNA targetting SIPA1L1 for the reduction of adipogenesis</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8778848">US8778848</a></td><td class="patent-data-table-td patent-date-value">Jun 8, 2012</td><td class="patent-data-table-td patent-date-value">Jul 15, 2014</td><td class="patent-data-table-td ">Illumina, Inc.</td><td class="patent-data-table-td ">Patterned flow-cells useful for nucleic acid analysis</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8778849">US8778849</a></td><td class="patent-data-table-td patent-date-value">Oct 26, 2012</td><td class="patent-data-table-td patent-date-value">Jul 15, 2014</td><td class="patent-data-table-td ">Illumina, Inc.</td><td class="patent-data-table-td ">Microarray fabrication system and method</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP1564306A2?cl=en">EP1564306A2</a></td><td class="patent-data-table-td patent-date-value">Feb 17, 2005</td><td class="patent-data-table-td patent-date-value">Aug 17, 2005</td><td class="patent-data-table-td ">Affymetrix, Inc.</td><td class="patent-data-table-td ">Methods for fragmenting and labeling DNA</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP1645640A2?cl=en">EP1645640A2</a></td><td class="patent-data-table-td patent-date-value">Oct 5, 2005</td><td class="patent-data-table-td patent-date-value">Apr 12, 2006</td><td class="patent-data-table-td ">Affymetrix, Inc. (a Delaware Corporation)</td><td class="patent-data-table-td ">Methods for amplifying and analyzing nucleic acids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP1647600A2?cl=en">EP1647600A2</a></td><td class="patent-data-table-td patent-date-value">Sep 19, 2005</td><td class="patent-data-table-td patent-date-value">Apr 19, 2006</td><td class="patent-data-table-td ">Affymetrix, Inc. (A US Entity)</td><td class="patent-data-table-td ">Methods for identifying biological samples by addition of nucleic acid bar-code tags</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP1652580A1?cl=en">EP1652580A1</a></td><td class="patent-data-table-td patent-date-value">Oct 28, 2005</td><td class="patent-data-table-td patent-date-value">May 3, 2006</td><td class="patent-data-table-td ">Affymetrix, Inc.</td><td class="patent-data-table-td ">High throughput microarray, package assembly and methods of manufacturing arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP1655381A1?cl=en">EP1655381A1</a></td><td class="patent-data-table-td patent-date-value">Nov 2, 2005</td><td class="patent-data-table-td patent-date-value">May 10, 2006</td><td class="patent-data-table-td ">Affymetrix, Inc. (A US Entity)</td><td class="patent-data-table-td ">Nucleic acid labeling methods</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP1655598A2?cl=en">EP1655598A2</a></td><td class="patent-data-table-td patent-date-value">Oct 29, 2005</td><td class="patent-data-table-td patent-date-value">May 10, 2006</td><td class="patent-data-table-td ">Affymetrix, Inc.</td><td class="patent-data-table-td ">System, method, and product for multiple wavelength detection using single source excitation</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2157524A2?cl=en">EP2157524A2</a></td><td class="patent-data-table-td patent-date-value">Sep 3, 2004</td><td class="patent-data-table-td patent-date-value">Feb 24, 2010</td><td class="patent-data-table-td ">THE GOVERNMENT OF THE UNITED STATES OF AMERICA, as represented by THE SECRETARY, DEPARTMENT OF HEALTH AND HUMAN SERVICES</td><td class="patent-data-table-td ">Methods for identifying, diagnosing, and predicting survival of lymphomas</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2264188A1?cl=en">EP2264188A1</a></td><td class="patent-data-table-td patent-date-value">Jun 17, 2004</td><td class="patent-data-table-td patent-date-value">Dec 22, 2010</td><td class="patent-data-table-td ">Illumina, Inc.</td><td class="patent-data-table-td ">Methods and compositions for whole genome amplification and genotyping</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2287608A2?cl=en">EP2287608A2</a></td><td class="patent-data-table-td patent-date-value">Mar 10, 2006</td><td class="patent-data-table-td patent-date-value">Feb 23, 2011</td><td class="patent-data-table-td ">Firalis SAS</td><td class="patent-data-table-td ">Biomarkers for cardiovascular side-effects induced by cox-2 inhibitory compounds</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2290079A2?cl=en">EP2290079A2</a></td><td class="patent-data-table-td patent-date-value">Jul 5, 1999</td><td class="patent-data-table-td patent-date-value">Mar 2, 2011</td><td class="patent-data-table-td ">NsGene A/S</td><td class="patent-data-table-td ">Neurotrophic factors</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2290363A1?cl=en">EP2290363A1</a></td><td class="patent-data-table-td patent-date-value">Mar 10, 2006</td><td class="patent-data-table-td patent-date-value">Mar 2, 2011</td><td class="patent-data-table-td ">Firalis SAS</td><td class="patent-data-table-td ">Biomarkers for cardiovascular side-effects induced by cox-2 inhibitory compounds</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2299267A2?cl=en">EP2299267A2</a></td><td class="patent-data-table-td patent-date-value">Nov 14, 2006</td><td class="patent-data-table-td patent-date-value">Mar 23, 2011</td><td class="patent-data-table-td ">Novartis AG</td><td class="patent-data-table-td ">Biomarkers for anti-NogoA antibody treatment in spinal cord injury</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2299377A1?cl=en">EP2299377A1</a></td><td class="patent-data-table-td patent-date-value">Jul 25, 2007</td><td class="patent-data-table-td patent-date-value">Mar 23, 2011</td><td class="patent-data-table-td ">The Government of the U.S.A. as represented by The Secretary of the dept. of Health &amp; Human Services</td><td class="patent-data-table-td ">Methods for identifying, diagnosing and predicting survival of lymphomas</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2316973A1?cl=en">EP2316973A1</a></td><td class="patent-data-table-td patent-date-value">Jun 9, 2004</td><td class="patent-data-table-td patent-date-value">May 4, 2011</td><td class="patent-data-table-td ">The Trustees Of Boston University</td><td class="patent-data-table-td ">Detection methods for disorders of the lung</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2327795A1?cl=en">EP2327795A1</a></td><td class="patent-data-table-td patent-date-value">Jun 9, 2004</td><td class="patent-data-table-td patent-date-value">Jun 1, 2011</td><td class="patent-data-table-td ">The Trustees Of Boston University</td><td class="patent-data-table-td ">Detection methods for disorders of the lung</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2327796A1?cl=en">EP2327796A1</a></td><td class="patent-data-table-td patent-date-value">Jun 9, 2004</td><td class="patent-data-table-td patent-date-value">Jun 1, 2011</td><td class="patent-data-table-td ">The Trustees Of Boston University</td><td class="patent-data-table-td ">Detection methods for disorders of the lung</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2360278A1?cl=en">EP2360278A1</a></td><td class="patent-data-table-td patent-date-value">Apr 14, 2006</td><td class="patent-data-table-td patent-date-value">Aug 24, 2011</td><td class="patent-data-table-td ">Trustees Of Boston University</td><td class="patent-data-table-td ">Diagnostic for lung disorders using class prediction</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2360279A1?cl=en">EP2360279A1</a></td><td class="patent-data-table-td patent-date-value">Apr 14, 2006</td><td class="patent-data-table-td patent-date-value">Aug 24, 2011</td><td class="patent-data-table-td ">Trustees Of Boston University</td><td class="patent-data-table-td ">Diagnostic for lung disorders using class prediction</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2390347A1?cl=en">EP2390347A1</a></td><td class="patent-data-table-td patent-date-value">Apr 14, 2006</td><td class="patent-data-table-td patent-date-value">Nov 30, 2011</td><td class="patent-data-table-td ">Trustees Of Boston University</td><td class="patent-data-table-td ">Diagnostic for lung disorders using class prediction</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2450456A2?cl=en">EP2450456A2</a></td><td class="patent-data-table-td patent-date-value">Nov 2, 2007</td><td class="patent-data-table-td patent-date-value">May 9, 2012</td><td class="patent-data-table-td ">Yale University</td><td class="patent-data-table-td ">Assessment of oocyte competence</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2594981A2?cl=en">EP2594981A2</a></td><td class="patent-data-table-td patent-date-value">Nov 21, 2006</td><td class="patent-data-table-td patent-date-value">May 22, 2013</td><td class="patent-data-table-td ">Illumina Inc.</td><td class="patent-data-table-td ">Confocal imaging methods and apparatus</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2605018A1?cl=en">EP2605018A1</a></td><td class="patent-data-table-td patent-date-value">Mar 8, 2007</td><td class="patent-data-table-td patent-date-value">Jun 19, 2013</td><td class="patent-data-table-td ">The Trustees of the Boston University</td><td class="patent-data-table-td ">Diagnostic and prognostic methods for lung disorders using gene expression profiles from nose epithelial cells</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2000011218A1?cl=en">WO2000011218A1</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Aug 20, 1999</td><td class="patent-data-table-td patent-date-value">Mar 2, 2000</td><td class="patent-data-table-td ">Affymetrix Inc</td><td class="patent-data-table-td ">Expression monitoring for human cytomegalovirus (hcmv) infection</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2000022173A1?cl=en">WO2000022173A1</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Oct 14, 1999</td><td class="patent-data-table-td patent-date-value">Apr 20, 2000</td><td class="patent-data-table-td ">Univ Princeton</td><td class="patent-data-table-td ">Quantitative analysis of hybridization patterns and intensities in oligonucleotide arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2001012860A1?cl=en">WO2001012860A1</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Aug 16, 2000</td><td class="patent-data-table-td patent-date-value">Feb 22, 2001</td><td class="patent-data-table-td ">Affymetrix Inc</td><td class="patent-data-table-td ">Methods, systems and computer software for designing and synthesizing sequence arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2001081632A1?cl=en">WO2001081632A1</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Apr 24, 2001</td><td class="patent-data-table-td patent-date-value">Nov 1, 2001</td><td class="patent-data-table-td ">Affymetrix Inc</td><td class="patent-data-table-td ">Methods for monitoring the expression of alternatively spliced genes</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2005054516A2?cl=en">WO2005054516A2</a></td><td class="patent-data-table-td patent-date-value">Nov 24, 2004</td><td class="patent-data-table-td patent-date-value">Jun 16, 2005</td><td class="patent-data-table-td ">Advandx Inc</td><td class="patent-data-table-td ">Peptide nucleic acid probes for analysis of certain staphylococcus species</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2008021290A2?cl=en">WO2008021290A2</a></td><td class="patent-data-table-td patent-date-value">Aug 9, 2007</td><td class="patent-data-table-td patent-date-value">Feb 21, 2008</td><td class="patent-data-table-td ">Homestead Clinical Corp</td><td class="patent-data-table-td ">Organ-specific proteins and methods of their use</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2009021673A1?cl=en">WO2009021673A1</a></td><td class="patent-data-table-td patent-date-value">Aug 7, 2008</td><td class="patent-data-table-td patent-date-value">Feb 19, 2009</td><td class="patent-data-table-td ">Hoffmann La Roche</td><td class="patent-data-table-td ">Predictive markers for egfr inhibitors treatment</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2009021674A1?cl=en">WO2009021674A1</a></td><td class="patent-data-table-td patent-date-value">Aug 7, 2008</td><td class="patent-data-table-td patent-date-value">Feb 19, 2009</td><td class="patent-data-table-td ">Hoffmann La Roche</td><td class="patent-data-table-td ">Predictive markers for egfr inhibitor treatment</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2009021681A2?cl=en">WO2009021681A2</a></td><td class="patent-data-table-td patent-date-value">Aug 7, 2008</td><td class="patent-data-table-td patent-date-value">Feb 19, 2009</td><td class="patent-data-table-td ">Hoffmann La Roche</td><td class="patent-data-table-td ">Egfr inhibitor treatment marker</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2009021684A2?cl=en">WO2009021684A2</a></td><td class="patent-data-table-td patent-date-value">Aug 7, 2008</td><td class="patent-data-table-td patent-date-value">Feb 19, 2009</td><td class="patent-data-table-td ">Hoffmann La Roche</td><td class="patent-data-table-td ">Predictive marker for egfr inhibitor treatment</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2009113779A2?cl=en">WO2009113779A2</a></td><td class="patent-data-table-td patent-date-value">Mar 5, 2009</td><td class="patent-data-table-td patent-date-value">Sep 17, 2009</td><td class="patent-data-table-td ">National Cancer Center</td><td class="patent-data-table-td ">Method for measuring chromosome, gene or specific nucleotide sequence copy numbers using snp array</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2009149359A2?cl=en">WO2009149359A2</a></td><td class="patent-data-table-td patent-date-value">Jun 5, 2009</td><td class="patent-data-table-td patent-date-value">Dec 10, 2009</td><td class="patent-data-table-td ">The United States Of America, As Represented By The Secretary, Dept. Of Health And Human Services</td><td class="patent-data-table-td ">Survival predictor for diffuse large b cell lymphoma</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2010048337A2?cl=en">WO2010048337A2</a></td><td class="patent-data-table-td patent-date-value">Oct 21, 2009</td><td class="patent-data-table-td patent-date-value">Apr 29, 2010</td><td class="patent-data-table-td ">Illumina, Inc.</td><td class="patent-data-table-td ">Preservation of information related to genomic dna methylation</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2011006169A1?cl=en">WO2011006169A1</a></td><td class="patent-data-table-td patent-date-value">Jul 12, 2010</td><td class="patent-data-table-td patent-date-value">Jan 13, 2011</td><td class="patent-data-table-td ">Portola Pharmaceuticals, Inc.</td><td class="patent-data-table-td ">Methods for diagnosis and treatment of thrombotic disorders mediated by cyp2c19*2</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2011056688A2?cl=en">WO2011056688A2</a></td><td class="patent-data-table-td patent-date-value">Oct 27, 2010</td><td class="patent-data-table-td patent-date-value">May 12, 2011</td><td class="patent-data-table-td ">Caris Life Sciences, Inc.</td><td class="patent-data-table-td ">Molecular profiling for personalized medicine</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2011093939A1?cl=en">WO2011093939A1</a></td><td class="patent-data-table-td patent-date-value">Nov 16, 2010</td><td class="patent-data-table-td patent-date-value">Aug 4, 2011</td><td class="patent-data-table-td ">Illumina, Inc.</td><td class="patent-data-table-td ">Focusing methods and optical systems and assemblies using the same</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2011112465A1?cl=en">WO2011112465A1</a></td><td class="patent-data-table-td patent-date-value">Mar 4, 2011</td><td class="patent-data-table-td patent-date-value">Sep 15, 2011</td><td class="patent-data-table-td ">Illumina, Inc.</td><td class="patent-data-table-td ">Systems, methods, and apparatuses for detecting optical signals from a sample</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2011159942A1?cl=en">WO2011159942A1</a></td><td class="patent-data-table-td patent-date-value">Jun 16, 2011</td><td class="patent-data-table-td patent-date-value">Dec 22, 2011</td><td class="patent-data-table-td ">Illumina, Inc.</td><td class="patent-data-table-td ">Conformational probes and methods for sequencing nucleic acids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2012049192A1?cl=en">WO2012049192A1</a></td><td class="patent-data-table-td patent-date-value">Oct 12, 2011</td><td class="patent-data-table-td patent-date-value">Apr 19, 2012</td><td class="patent-data-table-td ">F. Hoffmann-La Roche Ag</td><td class="patent-data-table-td ">Ipp complex as marker for erlotinib treatment</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2012050920A1?cl=en">WO2012050920A1</a></td><td class="patent-data-table-td patent-date-value">Sep 28, 2011</td><td class="patent-data-table-td patent-date-value">Apr 19, 2012</td><td class="patent-data-table-td ">Illumina, Inc.</td><td class="patent-data-table-td ">Compositions and methods for sequencing nucleic acids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2012055929A1?cl=en">WO2012055929A1</a></td><td class="patent-data-table-td patent-date-value">Oct 26, 2011</td><td class="patent-data-table-td patent-date-value">May 3, 2012</td><td class="patent-data-table-td ">Illumina, Inc.</td><td class="patent-data-table-td ">Sequencing methods</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2012096703A1?cl=en">WO2012096703A1</a></td><td class="patent-data-table-td patent-date-value">Oct 21, 2011</td><td class="patent-data-table-td patent-date-value">Jul 19, 2012</td><td class="patent-data-table-td ">Illumina, Inc.</td><td class="patent-data-table-td ">Systems, methods, and apparatuses to image a sample for biological or chemical analysis</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2012148477A1?cl=en">WO2012148477A1</a></td><td class="patent-data-table-td patent-date-value">Dec 15, 2011</td><td class="patent-data-table-td patent-date-value">Nov 1, 2012</td><td class="patent-data-table-td ">Cellular Research, Inc.</td><td class="patent-data-table-td ">Digital counting of individual molecules by stochastic attachment of diverse label-tags</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2013123125A1?cl=en">WO2013123125A1</a></td><td class="patent-data-table-td patent-date-value">Feb 14, 2013</td><td class="patent-data-table-td patent-date-value">Aug 22, 2013</td><td class="patent-data-table-td ">President And Fellows Of Harvard College</td><td class="patent-data-table-td ">Assembly of nucleic acid sequences in emulsions</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2014078700A1?cl=en">WO2014078700A1</a></td><td class="patent-data-table-td patent-date-value">Nov 15, 2013</td><td class="patent-data-table-td patent-date-value">May 22, 2014</td><td class="patent-data-table-td ">Myriad Genetics, Inc.</td><td class="patent-data-table-td ">Gene signatures for cancer prognosis</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2014085434A1?cl=en">WO2014085434A1</a></td><td class="patent-data-table-td patent-date-value">Nov 26, 2013</td><td class="patent-data-table-td patent-date-value">Jun 5, 2014</td><td class="patent-data-table-td ">Pontificia Universidad Catolica De Chile</td><td class="patent-data-table-td ">Compositions and methods for diagnosing thyroid tumors</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="classifications"></a><div class="patent-section-header"><span class="patent-section-title">Classifications</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th"> </th><th class="patent-data-table-th"> </th></tr></thead><tr><td class="patent-data-table-td ">U.S. Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=JI1HBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S006110">435/6.11</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JI1HBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S024320">536/24.32</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JI1HBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc382/defs382.htm&usg=AFQjCNEqDOPTZQm3eGeSecvjb7gyEoTppw#C382S179000">382/179</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JI1HBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S024330">536/24.33</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JI1HBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S091200">435/91.2</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JI1HBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S091100">435/91.1</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JI1HBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S024300">536/24.3</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JI1HBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S023100">536/23.1</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JI1HBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc382/defs382.htm&usg=AFQjCNEqDOPTZQm3eGeSecvjb7gyEoTppw#C382S178000">382/178</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JI1HBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S287200">435/287.2</a></span></td></tr><tr><td class="patent-data-table-td ">International Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=JI1HBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=G06F0019200000">G06F19/20</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JI1HBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=G06F0019220000">G06F19/22</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JI1HBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12Q0001680000">C12Q1/68</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JI1HBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C40B0060140000">C40B60/14</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JI1HBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C40B0040060000">C40B40/06</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JI1HBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=B01J0019000000">B01J19/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JI1HBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=G01N0015140000">G01N15/14</a></span></td></tr><tr><td class="patent-data-table-td ">Cooperative Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=JI1HBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=B01J2219/00432">B01J2219/00432</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JI1HBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=B01J2219/00722">B01J2219/00722</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JI1HBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=B01J2219/00529">B01J2219/00529</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JI1HBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=B82Y30/00">B82Y30/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JI1HBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=B01J2219/00689">B01J2219/00689</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JI1HBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12Q1/68">C12Q1/68</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JI1HBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=B01J19/0046">B01J19/0046</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JI1HBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C40B60/14">C40B60/14</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JI1HBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=B01J2219/0061">B01J2219/0061</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JI1HBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=B01J2219/00711">B01J2219/00711</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JI1HBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12Q1/6827">C12Q1/6827</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JI1HBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=G06F19/22">G06F19/22</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JI1HBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=G06F19/20">G06F19/20</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JI1HBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=B01J2219/00612">B01J2219/00612</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JI1HBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=B01J2219/00617">B01J2219/00617</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JI1HBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=B01J2219/00659">B01J2219/00659</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JI1HBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C40B40/06">C40B40/06</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JI1HBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=G01N15/1475">G01N15/1475</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JI1HBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=B01J2219/00626">B01J2219/00626</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JI1HBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12Q1/6874">C12Q1/6874</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JI1HBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=B01J2219/00608">B01J2219/00608</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JI1HBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=B01J2219/00695">B01J2219/00695</a></span></td></tr><tr><td class="patent-data-table-td ">European Classification</td><td class="patent-data-table-td "><span class="nested-value">B82Y30/00</span>, <span class="nested-value">G06F19/22</span>, <span class="nested-value">B01J19/00C</span>, <span class="nested-value">C12Q1/68E4</span>, <span class="nested-value">C12Q1/68B6</span>, <span class="nested-value">C12Q1/68</span>, <span class="nested-value">G06F19/20</span></td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="legal-events"></a><div class="patent-section-header"><span class="patent-section-title">Legal Events</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Date</th><th class="patent-data-table-th">Code</th><th class="patent-data-table-th">Event</th><th class="patent-data-table-th">Description</th></tr></thead><tr><td class="patent-data-table-td patent-date-value">Jun 27, 2012</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20120625</span></div><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">GENERAL ELECTRIC CAPITAL CORPORATION, AS AGENT, MA</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">SECURITY AGREEMENT;ASSIGNOR:AFFYMETRIX, INC.;REEL/FRAME:028465/0541</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Feb 18, 2010</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">12</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Jun 16, 2009</td><td class="patent-data-table-td ">B1</td><td class="patent-data-table-td ">Reexamination certificate first reexamination</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">CLAIMS 1-8 ARE DETERMINED TO BE PATENTABLE AS AMENDED. CLAIMS 9-10, DEPENDENT ON AN AMENDED CLAIM, ARE DETERMINED TO BE PATENTABLE.</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Jan 15, 2008</td><td class="patent-data-table-td ">RR</td><td class="patent-data-table-td ">Request for reexamination filed</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20071127</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Oct 17, 2006</td><td class="patent-data-table-td ">CC</td><td class="patent-data-table-td ">Certificate of correction</td><td class="patent-data-table-td "></td></tr><tr><td class="patent-data-table-td patent-date-value">Feb 21, 2006</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">8</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Mar 5, 2002</td><td class="patent-data-table-td ">REMI</td><td class="patent-data-table-td ">Maintenance fee reminder mailed</td><td class="patent-data-table-td "></td></tr><tr><td class="patent-data-table-td patent-date-value">Feb 15, 2002</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">4</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Feb 8, 1999</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">AFFYMETRIX, INC., A DELAWARE CORPORATION, CALIFORN</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">MERGER;ASSIGNOR:AFFYMETRIX, INC., A CALIFORNIA CORPORATION;REEL/FRAME:009748/0134</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">19980929</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Aug 26, 1996</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">AFFYMETRIX, INC., CALIFORNIA</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNOR:AFFYMAX TECHNOLOGIES N.V.;REEL/FRAME:008105/0731</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">19960418</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Aug 4, 1995</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">AFFYMAX TECHNOLOGIES N.V., NETHERLANDS ANTILLES</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNORS:CHEE, MARK S.;WANG, CHUNWEI;JEVONS, LUIS C.;AND OTHERS;REEL/FRAME:007580/0173;SIGNING DATES FROM 19950131 TO 19950224</span></div></td></tr></table><div class="patent-section-footer"></div></div><div class="modal-dialog" id="patent-images-lightbox"><div class="patent-lightbox-controls"><div class="patent-lightbox-rotate-controls"><div class="patent-lightbox-rotation-text">Rotate</div><div class="rotate-icon rotate-ccw-icon"></div><div class="rotate-icon rotate-cw-icon"></div></div><div class="patent-lightbox-index-counter"></div><a class="patent-lightbox-fullsize-link" target="_blank">Original Image</a><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_right.png" alt="Next page"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_left.png" alt="Previous page"width="21" height="21" /></div></div><div class="modal-dialog-content"><div class="patent-lightbox-image-holder"><div class="patent-lightbox-placeholder"></div></div></div></div><script>_OC_initPatentsAtb({image_not_available_html: " Image not available"});</script></div></div></div></td></tr></table><script>(function() {var href = window.location.href;if (href.indexOf('?') !== -1) {var parameters = href.split('?')[1].split('&');for (var i = 0; i < parameters.length; i++) {var param = parameters[i].split('=');if (param[0] == 'focus') {var elem = document.getElementById(param[1]);if (elem) {elem.focus();}}}}})();</script><script>_OC_addFlags({LockSrc:"/books/javascript/lock_8a2b04e7bf975d5171d8e4c0b6365c7a.js", Host:"http://www.google.com/", IsBooksRentalEnabled:1, IsWebstoreDisplayCaseEnabled:1, IsObfuscationEnabled:1, IsBrowsingHistoryEnabled:1, IsWebReaderSvgEnabled:0, IsGeoLayerEnabled:1, IsImageModeNotesEnabled:1, IsCopyMenuItemEnabled:1, IsGiftingEnabled:0, IsWebReaderUniversalPaginatorEnabled:0, IsOfflineBubbleEnabled:1, IsReaderEnabledForPlayRequests:1, IsFutureOnSaleVolumesEnabled:1, IsOfflineRestrictedCopyEnabled:1, IsBooksUnifiedLeftNavEnabled:1, IsRestrictedCopyEnabled:1, IsZipitFolderCollectionEnabled:1, IsEndOfSampleRecommendationsEnabled:1, IsRatingsOnBookcardsEnabled:1, IsAdsDisabled:0, IsIframePageDisplayEnabled:0, IsEmbeddedMediaEnabled:1, IsImageModeAnnotationsEnabled:1, IsMyLibraryGooglePlusEnabled:1, IsImagePageProviderEnabled:0, IsBookcardListPriceSmall:0, IsInternalUser:0, IsBooksShareButtonEnabled:0, IsPreOrdersEnabled:0, IsDisabledRandomBookshelves:0, WebstoreDisplayCasePosition:3});_OC_Run({"enable_p13n":false,"add_vol_to_collection_base_url":"http://www.google.com/patents?op=add\u0026sig=ACfU3U0sgnmzrPzQhgJg11tesJjVKlln8A\u0026id=JI1HBAABERAJ","remove_vol_from_collection_base_url":"http://www.google.com/patents?op=remove\u0026sig=ACfU3U0cxHD0UsD-bqMWakdSv5AEiwrTkA\u0026id=JI1HBAABERAJ","logged_in":false,"p13n_save_user_settings_url":"http://www.google.com/patents?op=edit_user_settings\u0026sig=ACfU3U26j5Nk5pXjNBss3pDMpHwDvUYW5w","is_cobrand":false,"sign_in_url":"https://www.google.com/accounts/Login?service=\u0026continue=http://www.google.com/patents%3Fhl%3Den\u0026hl=en","is_play_enabled":true}, {"volume_id":"","is_ebook":true,"volumeresult":{"has_flowing_text":false,"has_scanned_text":true,"can_download_pdf":false,"can_download_epub":false,"is_pdf_drm_enabled":false,"is_epub_drm_enabled":false,"download_pdf_url":"http://www.google.com/patents/download/Analyzing_nucleic_acid_sequences.pdf?id=JI1HBAABERAJ\u0026output=pdf\u0026sig=ACfU3U2C9ItEuowe9eFtb0Tc2yvaUXCqAQ"},"sample_url":"http://www.google.com/patents/reader?id=JI1HBAABERAJ\u0026printsec=frontcover\u0026output=reader\u0026source=gbs_atb_hover","is_browsable":true,"is_public_domain":true}, {});</script><div id="footer_table" style="font-size:83%;text-align:center;position:relative;top:20px;height:4.5em;margin-top:2em"><div style="margin-bottom:8px"><a href=http://www.google.com/><nobr>Google&nbsp;Home</nobr></a> - <a href=//www.google.com/patents/sitemap/><nobr>Sitemap</nobr></a> - <a href=http://www.google.com/googlebooks/uspto.html><nobr>USPTO Bulk Downloads</nobr></a> - <a href=/intl/en/privacy/><nobr>Privacy Policy</nobr></a> - <a href=/intl/en/policies/terms/><nobr>Terms of Service</nobr></a> - <a href=https://support.google.com/faqs/answer/2539193?hl=en><nobr>About Google Patents</nobr></a> - <a href="http://www.google.com/tools/feedback/intl/en/error.html" onclick="try{_OC_startFeedback({productId: '72792',locale: 'en'});return false;}catch(e){}"><nobr>Send Feedback</nobr></a></div><span>Data provided by IFI CLAIMS Patent Services</span><br><span >&copy;2012 Google</span></div> <script type="text/javascript">var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));</script><script type="text/javascript">var pageTracker = _gat._getTracker("UA-27188110-1");pageTracker._setCookiePath("/patents/");pageTracker._trackPageview();</script> </body></html>