<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN""http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html><head><title>Patent US5858731 - Nucleotide sequences including nondegenerate and degenerate nucleotide bases ... - Google Patents</title><script>(function(){(function(){function e(a){this.t={};this.tick=function(a,c,b){var d=void 0!=b?b:(new Date).getTime();this.t[a]=[d,c];if(void 0==b)try{window.console.timeStamp("CSI/"+a)}catch(e){}};this.tick("start",null,a)}var a;window.performance&&(a=window.performance.timing);var f=a?new e(a.responseStart):new e;window.jstiming={Timer:e,load:f};if(a){var c=a.navigationStart,d=a.responseStart;0<c&&d>=c&&(window.jstiming.srt=d-c)}if(a){var b=window.jstiming.load;0<c&&d>=c&&(b.tick("_wtsrt",void 0,c),b.tick("wtsrt_",
"_wtsrt",d),b.tick("tbsd_","wtsrt_"))}try{a=null,window.chrome&&window.chrome.csi&&(a=Math.floor(window.chrome.csi().pageT),b&&0<c&&(b.tick("_tbnd",void 0,window.chrome.csi().startE),b.tick("tbnd_","_tbnd",c))),null==a&&window.gtbExternal&&(a=window.gtbExternal.pageT()),null==a&&window.external&&(a=window.external.pageT,b&&0<c&&(b.tick("_tbnd",void 0,window.external.startE),b.tick("tbnd_","_tbnd",c))),a&&(window.jstiming.pt=a)}catch(g){}})();})();
</script><link rel="stylesheet" href="/patents/css/_8a2b04e7bf975d5171d8e4c0b6365c7a/kl_intl_patents_bundle.css" type="text/css" /><script src="/books/javascript/atb_8a2b04e7bf975d5171d8e4c0b6365c7a__en.js"></script><script>function googleTranslateElementInit() {new google.translate.TranslateElement({pageLanguage: "en",gaTrack: true,gaId: "UA-27188110-1",multilanguagePage: true});}</script><script src="//translate.google.com/translate_a/element.js?cb=googleTranslateElementInit"></script><meta name="DC.type" content="Patent"><meta name="DC.title" content="Nucleotide sequences including nondegenerate and degenerate nucleotide bases which can be ligated together for longer molecules; for rapid and efficient synthesis of preselected sequences for dna sequencing, polymerase chain reaction"><meta name="DC.contributor" content="Joseph A. Sorge" scheme="inventor"><meta name="DC.contributor" content="Daniel Davis Shoemaker" scheme="inventor"><meta name="DC.contributor" content="Stratagene" scheme="assignee"><meta name="DC.date" content="1996-2-20" scheme="dateSubmitted"><meta name="DC.description" content="An oligonucleotide library is described that is1 useful for producing an oligonucleotide of preselected sequence comprising a plurality of oligonucleotide members comprising one or more oligonucleotide species and having the compositional formula (X)a (N)b ; wherein X represents a non-degenerate nucleotide base and N represents a degenerate nucleotide base; &quot;a&quot; represents the number of non-degenerate nucleotide positions and is from 3 to 8; &quot;b&quot; represents the number of degenerate nucleotide positions and is from 0 to 4 but not greater that &quot;a&quot;; and wherein each of the oligonucleotide species is capable of forming a hybridization complex with at least one other of the oligonucleotide species in the library such that a single ligation event of the hybridization complex with another hybridization complex derived from the library produces a ligation reaction product comprising greater than 12 contiguous nucleotide base pairs."><meta name="DC.date" content="1999-1-12" scheme="issued"><meta name="DC.relation" content="US:5114839" scheme="references"><meta name="DC.relation" content="US:5366882" scheme="references"><meta name="DC.relation" content="US:5371006" scheme="references"><meta name="DC.relation" content="US:5599921" scheme="references"><meta name="DC.relation" content="US:5663062" scheme="references"><meta name="DC.relation" content="WO:1989011211:A2" scheme="references"><meta name="citation_reference" content="Studier. &quot;A Strategy for High-Volume Sequencing of Cosmid DNAs: Random and directed Priming with a Library of Oligonucleotides&quot;. Proceedings of the National Academy of Sciences USA 86: 6917-6922 (1989)."><meta name="citation_reference" content="Studier. A Strategy for High Volume Sequencing of Cosmid DNAs: Random and directed Priming with a Library of Oligonucleotides . Proceedings of the National Academy of Sciences USA 86: 6917 6922 (1989)."><meta name="citation_reference" content="Szbalski. &quot;Proposal for Sequencing DNA Using Litigation of Hexamers to Generate Sequential Elongation Primers&quot;. Gene 90: 177-178 (1990)."><meta name="citation_reference" content="Szbalski. Proposal for Sequencing DNA Using Litigation of Hexamers to Generate Sequential Elongation Primers . Gene 90: 177 178 (1990)."><meta name="citation_patent_number" content="US:5858731"><meta name="citation_patent_application_number" content="US:08/313,232"><link rel="canonical" href="http://www.google.com/patents/US5858731"/><meta property="og:url" content="http://www.google.com/patents/US5858731"/><meta name="title" content="Patent US5858731 - Nucleotide sequences including nondegenerate and degenerate nucleotide bases which can be ligated together for longer molecules; for rapid and efficient synthesis of preselected sequences for dna sequencing, polymerase chain reaction"/><meta name="description" content="An oligonucleotide library is described that is1 useful for producing an oligonucleotide of preselected sequence comprising a plurality of oligonucleotide members comprising one or more oligonucleotide species and having the compositional formula (X)a (N)b ; wherein X represents a non-degenerate nucleotide base and N represents a degenerate nucleotide base; &quot;a&quot; represents the number of non-degenerate nucleotide positions and is from 3 to 8; &quot;b&quot; represents the number of degenerate nucleotide positions and is from 0 to 4 but not greater that &quot;a&quot;; and wherein each of the oligonucleotide species is capable of forming a hybridization complex with at least one other of the oligonucleotide species in the library such that a single ligation event of the hybridization complex with another hybridization complex derived from the library produces a ligation reaction product comprising greater than 12 contiguous nucleotide base pairs."/><meta property="og:title" content="Patent US5858731 - Nucleotide sequences including nondegenerate and degenerate nucleotide bases which can be ligated together for longer molecules; for rapid and efficient synthesis of preselected sequences for dna sequencing, polymerase chain reaction"/><meta property="og:type" content="book"/><meta property="og:site_name" content="Google Books"/><meta property="og:image" content="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><link rel="image_src" href="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><script>(function(){try{var aa=function(a,b,c,d){d=d||{};d._sn=["cfg",b,c].join(".");window.gbar.logger.ml(a,d)};var g=window.gbar=window.gbar||{},l=window.gbar.i=window.gbar.i||{},m={},n;function _tvn(a,b){var c=parseInt(a,10);return isNaN(c)?b:c}function _tvf(a,b){var c=parseFloat(a);return isNaN(c)?b:c}function _tvv(a){return!!a}function p(a,b,c){(c||g)[a]=b}g.bv={n:_tvn("2",0),r:"",f:".67.",e:"0",m:_tvn("0",1)};
function q(a,b,c){var d="on"+b;if(a.addEventListener)a.addEventListener(b,c,!1);else if(a.attachEvent)a.attachEvent(d,c);else{var f=a[d];a[d]=function(){var a=f.apply(this,arguments),b=c.apply(this,arguments);return void 0==a?b:void 0==b?a:b&&a}}}var s=function(a){return function(){return g.bv.m==a}},ba=s(1),ca=s(2);p("sb",ba);p("kn",ca);l.a=_tvv;l.b=_tvf;l.c=_tvn;l.i=aa;var da=window.gbar.i.i;var t,u,v,w;function ea(a){v=a}function fa(a){var b;if(b=v&&window.encodeURIComponent)b=a.href,b=!b.match(/^http[s]?:\/\/accounts\.google\.[^/]*\/ClearSID/i)&&!b.match(/^http[s]?:\/\/[^/]*\/accounts\/ClearSID/i);if(b=b&&encodeURIComponent(v()))a.href=a.href.replace(/([?&]continue=)[^&]*/,"$1"+b)}function ga(a){window.gApplication&&(a.href=window.gApplication.getTabUrl(a.href))}
function ha(a){var b=document.forms[0].q,c=window.encodeURIComponent&&b&&b.value,b=b&&b.placeholder;c&&c!=b&&(a.href=a.href.replace(/([?&])q=[^&]*|$/,function(a,b){return(b||"&")+"q="+encodeURIComponent(c)}))}n=l.a("")?ga:ha;
function x(a,b,c,d,f,e){var h=document.getElementById(a);if(h){var k=h.style;k.left=d?"auto":b+"px";k.right=d?b+"px":"auto";k.top=c+"px";k.visibility=u?"hidden":"visible";f&&e?(k.width=f+"px",k.height=e+"px"):(x(t,b,c,d,h.offsetWidth,h.offsetHeight),u=u?"":a)}}
var y=[],ia=function(a,b){y.push(b)},ja=function(a){a=a||window.event;var b=a.target||a.srcElement;a.cancelBubble=!0;null==t&&(a=document.createElement(Array.every||window.createPopup?"iframe":"div"),a.frameBorder="0",t=a.id="gbs",a.src="javascript:''",b.parentNode.appendChild(a),q(document,"click",z));var c=b,b=0;"gb3"!=c.className&&(c=c.parentNode);a=c.getAttribute("aria-owns")||"gbi";var d=c.offsetWidth,f=20<c.offsetTop?46:24;document.getElementById("tphdr")&&(f-=3);var e=!1;do b+=c.offsetLeft||
0;while(c=c.offsetParent);var c=(document.documentElement.clientWidth||document.body.clientWidth)-b-d,h,d=document.body,k=document.defaultView;k&&k.getComputedStyle?(d=k.getComputedStyle(d,""))&&(h=d.direction):h=d.currentStyle?d.currentStyle.direction:d.style.direction;h="rtl"==h;if("gbi"==a){for(d=0;k=y[d++];)k();A(null,window.navExtra);h&&(b=c,e=!0)}else h||(b=c,e=!0);u!=a&&z();x(a,b,f,e)},z=function(){u&&x(u,0,0)},A=function(a,b){var c,d=document.getElementById("gbi"),f=a;f||(f=d.firstChild);
for(;b&&(c=b.pop());){var e=d,h=c,k=f;w||(w="gb2");e.insertBefore(h,k).className=w}},ka=function(a,b,c){if((b=document.getElementById(b))&&a){a.className="gb4";var d=document.createElement("span");d.appendChild(a);d.appendChild(document.createTextNode(" | "));d.id=c;b.appendChild(d)}},la=function(){return document.getElementById("gb_70")},ma=function(){return!!u};p("qs",n);p("setContinueCb",ea);p("pc",fa);p("tg",ja);p("close",z);p("addLink",ka);p("almm",A);p("si",la);p("adh",ia);p("op",ma);var B=function(){},C=function(){},F=function(a){var b=new Image,c=D;b.onerror=b.onload=b.onabort=function(){try{delete E[c]}catch(a){}};E[c]=b;b.src=a;D=c+1},E=[],D=0;p("logger",{il:C,ml:B,log:F});var G=window.gbar.logger;var H={},na={},I=[],oa=l.b("0.1",.1),pa=l.a("1",!0),qa=function(a,b){I.push([a,b])},ra=function(a,b){H[a]=b},sa=function(a){return a in H},J={},K=function(a,b){J[a]||(J[a]=[]);J[a].push(b)},ta=function(a){K("m",a)},L=function(a,b){var c=document.createElement("script");c.src=a;c.async=pa;Math.random()<oa&&(c.onerror=function(){c.onerror=null;B(Error("Bundle load failed: name="+(b||"UNK")+" url="+a))});(document.getElementById("xjsc")||document.getElementsByTagName("body")[0]||
document.getElementsByTagName("head")[0]).appendChild(c)},N=function(a){for(var b=0,c;(c=I[b])&&c[0]!=a;++b);!c||c[1].l||c[1].s||(c[1].s=!0,M(2,a),c[1].url&&L(c[1].url,a),c[1].libs&&m.d&&m.d(c[1].libs))},O=function(a){K("gc",a)},P=null,ua=function(a){P=a},M=function(a,b,c){if(P){a={t:a,b:b};if(c)for(var d in c)a[d]=c[d];try{P(a)}catch(f){}}};p("mdc",H);p("mdi",na);p("bnc",I);p("qGC",O);p("qm",ta);p("qd",J);p("lb",N);p("mcf",ra);p("bcf",qa);p("aq",K);p("mdd","");p("has",sa);
p("trh",ua);p("tev",M);var Q=l.b("0.1",.001),R=0;
function _mlToken(a,b){try{if(1>R){R++;var c,d=a,f=b||{},e=encodeURIComponent,h=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&jexpid=",e("17483"),"&srcpg=",e("prop=22"),"&jsr=",Math.round(1/Q),"&ogev=",e("nkjsU_ecIc6ONubHguAH"),"&ogf=",g.bv.f,"&ogrp=",e("1"),"&ogv=",e("1407723702.0"),"&oggv="+e("es_plusone_gc_20140723.0_p0"),"&ogd=",e("com"),"&ogc=",e("BRA"),"&ogl=",e("en")];f._sn&&(f._sn="og."+
f._sn);for(var k in f)h.push("&"),h.push(e(k)),h.push("="),h.push(e(f[k]));h.push("&emsg=");h.push(e(d.name+":"+d.message));var r=h.join("");S(r)&&(r=r.substr(0,2E3));c=r;var Aa=window.gbar.logger._aem(a,c);F(Aa)}}catch(Na){}}var S=function(a){return 2E3<=a.length},va=function(a,b){return b};function T(a){B=a;p("_itl",S,G);p("_aem",va,G);p("ml",B,G);a={};H.er=a}l.a("")?T(function(a){throw a;}):l.a("1")&&Math.random()<Q&&T(_mlToken);I.push(["m",{url:"//ssl.gstatic.com/gb/js/scm_7385cc5883250b43a39405734c1bea59.js"}]);g.mcf("c",{});g.sg={c:""};if(l.a("1")){var wa=l.a("");I.push(["gc",{auto:wa,url:"//ssl.gstatic.com/gb/js/abc/gci_91f30755d6a6b787dcc2a4062e6e9824.js",libs:"googleapis.client:plusone:gapi.iframes"}]);var xa={version:"gci_91f30755d6a6b787dcc2a4062e6e9824.js",index:"",lang:"en"};H.gc=xa;var U=function(a){window.googleapis&&window.iframes?a&&a():(a&&O(a),N("gc"))};p("lGC",U);l.a("1")&&p("lPWF",U)};window.__PVT="";if(l.a("1")&&l.a("1")){var V=function(a){U(function(){K("pw",a);N("pw")})};p("lPW",V);I.push(["pw",{url:"//ssl.gstatic.com/gb/js/abc/pwm_45f73e4df07a0e388b0fa1f3d30e7280.js"}]);var W=[],ya=function(a){W[0]=a},za=function(a,b){var c=b||{};c._sn="pw";B(a,c)},Ba={signed:W,elog:za,base:"https://plusone.google.com/u/0",loadTime:(new Date).getTime()};H.pw=Ba;var X=function(a,b){for(var c=b.split("."),d=function(){var b=arguments;a(function(){for(var a=g,d=0,e=c.length-1;d<e;++d)a=a[c[d]];a[c[d]].apply(a,b)})},f=g,e=0,h=c.length-1;e<h;++e)f=
f[c[e]]=f[c[e]]||{};return f[c[e]]=d};X(V,"pw.clk");X(V,"pw.hvr");p("su",ya,g.pw)};function Ca(){function a(){for(var b;(b=e[h++])&&"m"!=b[0]&&!b[1].auto;);b&&(M(2,b[0]),b[1].url&&L(b[1].url,b[0]),b[1].libs&&m.d&&m.d(b[1].libs));h<e.length&&setTimeout(a,0)}function b(){0<f--?setTimeout(b,0):a()}var c=l.a("1"),d=l.a(""),f=3,e=I,h=0,k=window.gbarOnReady;if(k)try{k()}catch(r){da(r,"ml","or")}d?p("ldb",a):c?q(window,"load",b):b()}p("rdl",Ca);var Da={D:1,H:2,da:3,p:4,W:5,M:6,F:7,g:8,ha:9,U:10,L:11,T:12,S:13,N:14,Q:15,P:16,fa:17,w:18,O:19,ga:20,ea:21,u:22,G:23,ja:24,ka:25,ia:26,A:27,j:28,o:29,k:30,ca:31,Z:32,$:33,J:34,K:35,ba:36,aa:37,Y:38,B:39,R:40,v:41,X:42,V:43,h:48,C:49,I:500},Y=[1,2,3,4,5,6,9,10,11,13,14,28,29,30,34,35,37,38,39,40,41,42,43,48,49,500];var Z=l.b("0.001",1E-4),Ea=l.b("1",1),Fa=!1,Ga=!1;if(l.a("1")){var Ha=Math.random();Ha<=Z&&(Fa=!0);Ha<=Ea&&(Ga=!0)}var Ia=Da,$=null;function Ja(){var a=0,b=function(b,d){l.a(d)&&(a|=b)};b(1,"");b(2,"");b(4,"");b(8,"");return a}
function Ka(a,b){var c=Z,d=Fa,f;f=a;if(!$){$={};for(var e=0;e<Y.length;e++){var h=Y[e];$[h]=!0}}if(f=!!$[f])c=Ea,d=Ga;if(d){d=encodeURIComponent;g.rp?(f=g.rp(),f="-1"!=f?f:"1"):f="1";c=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&oge=",a,"&ogex=",d("17483"),"&ogev=",d("nkjsU_ecIc6ONubHguAH"),"&ogf=",g.bv.f,"&ogp=",d("22"),"&ogrp=",d(f),"&ogsr=",Math.round(1/c),"&ogv=",d("1407723702.0"),"&oggv="+
d("es_plusone_gc_20140723.0_p0"),"&ogd=",d("com"),"&ogl=",d("en"),"&ogc=",d("BRA"),"&ogus=",Ja()];if(b){"ogw"in b&&(c.push("&ogw="+b.ogw),delete b.ogw);var k;f=b;e=[];for(k in f)0!=e.length&&e.push(","),e.push(La(k)),e.push("."),e.push(La(f[k]));k=e.join("");""!=k&&(c.push("&ogad="),c.push(d(k)))}F(c.join(""))}}function La(a){"number"==typeof a&&(a+="");return"string"==typeof a?a.replace(".","%2E").replace(",","%2C"):a}C=Ka;p("il",C,G);var Ma={};H.il=Ma;setTimeout(function(){C(Ia.g)},0);}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var b=window.gbar.i.i;var c=window.gbar;var f=function(d){try{var a=document.getElementById("gbom");a&&d.appendChild(a.cloneNode(!0))}catch(e){b(e,"omas","aomc")}};c.aomc=f;}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var a=window.gbar;a.mcf("pm",{p:""});}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{window.gbar.rdl();}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
if (window['_OC_timingAction']) {window['_OC_timingAction']('patents_refpage');}</script><style>#gbar,#guser{font-size:13px;padding-top:1px !important;}#gbar{float:left;height:22px}#guser{padding-bottom:7px !important;text-align:right}.gbh,.gbd{border-top:1px solid #c9d7f1;font-size:1px}.gbh{height:0;position:absolute;top:24px;width:100%}#gbs,.gbm{background:#fff;left:0;position:absolute;text-align:left;visibility:hidden;z-index:1000}.gbm{border:1px solid;border-color:#c9d7f1 #36c #36c #a2bae7;z-index:1001}.gb1{margin-right:.5em}.gb1,.gb3{zoom:1}.gb2{display:block;padding:.2em .5em}.gb2,.gb3{text-decoration:none !important;border-bottom:none}a.gb1,a.gb4{text-decoration:underline !important}a.gb1,a.gb2,a.gb3,a.gb4{color:#00c !important}.gbi .gb3,.gbi .gb2,.gbi .gb4{color:#dd8e27 !important}.gbf .gb3,.gbf .gb2,.gbf .gb4{color:#900 !important}a.gb2:hover{background:#36c;color:#fff !important}#gbar .gbz0l{color:#000 !important;cursor:default;font-weight:bold;text-decoration:none !important}
#gbar { padding:.3em .6em !important;}</style></head><body  topmargin="3" marginheight="3"><div id=gbar><nobr><a onclick=gbar.qs(this);gbar.logger.il(1,{t:1}); class=gb1 id=gb_1 href="https://www.google.com/search?sa=N&tab=tw">Search</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:2}); class=gb1 id=gb_2 href="http://www.google.com/search?hl=en&tbm=isch&source=og&sa=N&tab=ti">Images</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:8}); class=gb1 id=gb_8 href="http://maps.google.com/maps?hl=en&sa=N&tab=tl">Maps</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:78}); class=gb1 id=gb_78 href="https://play.google.com/?hl=en&sa=N&tab=t8">Play</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:36}); class=gb1 id=gb_36 href="http://www.youtube.com/results?sa=N&tab=t1">YouTube</a> <a onclick=gbar.logger.il(1,{t:5}); class=gb1 id=gb_5 href="http://news.google.com/nwshp?hl=en&tab=tn">News</a> <a onclick=gbar.logger.il(1,{t:23}); class=gb1 id=gb_23 href="https://mail.google.com/mail/?tab=tm">Gmail</a> <a onclick=gbar.logger.il(1,{t:25}); class=gb1 id=gb_25 href="https://drive.google.com/?tab=to">Drive</a> <a class=gb3 href="http://www.google.com/intl/en/options/" onclick="this.blur();gbar.tg(event);return !1" aria-haspopup=true><u>More</u> <small>&#9660;</small></a><div class=gbm id=gbi><a onclick=gbar.logger.il(1,{t:24}); class=gb2 id=gb_24 href="https://www.google.com/calendar?tab=tc">Calendar</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:51}); class=gb2 id=gb_51 href="http://translate.google.com/?hl=en&sa=N&tab=tT">Translate</a><a onclick=gbar.logger.il(1,{t:17}); class=gb2 id=gb_17 href="http://www.google.com/mobile/?hl=en&tab=tD">Mobile</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:10}); class=gb2 id=gb_10 href="http://www.google.com/search?hl=en&tbo=u&tbm=bks&source=og&sa=N&tab=tp">Books</a><a onclick=gbar.logger.il(1,{t:212}); class=gb2 id=gb_212 href="https://wallet.google.com/manage/?tab=ta">Wallet</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:6}); class=gb2 id=gb_6 href="http://www.google.com/search?hl=en&tbo=u&tbm=shop&source=og&sa=N&tab=tf">Shopping</a><a onclick=gbar.logger.il(1,{t:30}); class=gb2 id=gb_30 href="http://www.blogger.com/?tab=tj">Blogger</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:27}); class=gb2 id=gb_27 href="http://www.google.com/finance?sa=N&tab=te">Finance</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:31}); class=gb2 id=gb_31 href="https://plus.google.com/photos?sa=N&tab=tq">Photos</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:12}); class=gb2 id=gb_12 href="http://www.google.com/search?hl=en&tbo=u&tbm=vid&source=og&sa=N&tab=tv">Videos</a><div class=gb2><div class=gbd></div></div><a onclick=gbar.logger.il(1,{t:66}); href="http://www.google.com/intl/en/options/" class=gb2>Even more &raquo;</a></div></nobr></div><div id=guser width=100%><nobr><span id=gbn class=gbi></span><span id=gbf class=gbf></span><span id=gbe></span><a target=_top id=gb_70 href="https://www.google.com/accounts/Login?service=&continue=http://www.google.com/patents%3Fhl%3Den&hl=en" class=gb4>Sign in</a><div style="display: none"><div class=gbm id=gbd5 aria-owner=gbg5><div class=gbmc><ol id=gbom class=gbmcc></ol></div></div></div></nobr></div><div class=gbh style=left:0></div><div class=gbh style=right:0></div><div role="alert" style="position: absolute; left: 0; right: 0;"><a href="http://www.google.com/patents/us5858731?hl=en&amp;output=html_text" title="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."><img border="0" src="http://www.google.com/images/cleardot.gif"alt="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."></a></div><div id="guser"><nobr></nobr></div><div style="clear:both;"></div><div id="gb-top-search-box" class="gb-top-search-box-small gb-reset"><table><tr><td class="logo"><a href="http://www.google.com/patents" class="logo-link"><img class="logo-img" src="/intl/en/images/logos/google_logo_41.png" alt="Go to Google Books Home" height="41"/></a></td><td><form action="http://www.google.com/search" name="f" id="vheadf" method="get"><span id="hf"></span><input type="hidden" name="tbm" value="pts"/><input type="hidden" name="tbo" value="1"/><input type="hidden" name="hl" value="en"/><table><tr><td><div class="inputs"><table><tr><td><div class="text-input"><input type="text" name="q" id="vheadq" class="text" maxlength="2048" size="31" value="" title="Search Patents" accesskey="s" autocomplete="off"/><script>window._OC_autoDir &&window._OC_autoDir('vheadq', 'tia-vheadq');</script></div></td><td><div class="submit-input"><input name="btnG" class="submit" type="submit" value=""/></div></td></tr></table></div></td><td class="col-ext-links"><div class="ext-links"><a href="http://www.google.com/advanced_patent_search">&lt;nobr&gt;Advanced Patent Search&lt;/nobr&gt;</a></div></td></tr></table></form></td></tr></table></div><div class="kd-appbar"><h2 class="kd-appname"><a href="/patents">Patents</a></h2><div class="kd-buttonbar left" id="left-toolbar-buttons"><a id="appbar-write-review-link" href=""></a><a id="appbar-view-print-sample-link" href=""></a><a id="appbar-view-ebook-sample-link" href=""></a><a id="appbar-patents-prior-art-finder-link" href="https://www.google.com/patents/related/US5858731"></a><a id="appbar-patents-discuss-this-link" href="http://www.google.com/url?id=gx1JBAABERAJ&amp;q=http://patents.stackexchange.com/redirect/google-patents%3Fpatent%3DUS5858731&amp;usg=AFQjCNGYHJox-lvQl4WuYBgMLB-tJKo9-A" data-is-grant="true"></a><a id="appbar-read-patent-link" href="//docs.google.com/viewer?url=patentimages.storage.googleapis.com/pdfs/US5858731.pdf"></a><a id="appbar-download-pdf-link" href="//patentimages.storage.googleapis.com/pdfs/US5858731.pdf"></a></div><div class="kd-buttonbar right" id="right-toolbar-buttons"></div></div><div id="books-microdata" itemscope=""itemtype="http://schema.org/Book"itemid="http://www.google.com/patents/US5858731" style="display:none"><span itemprop="description">An oligonucleotide library is described that is1 useful for producing an oligonucleotide of preselected sequence comprising a plurality of oligonucleotide members comprising one or more oligonucleotide species and having the compositional formula (X)a (N)b ; wherein X represents a non-degenerate nucleotide...</span><span itemprop="url">http://www.google.com/patents/US5858731?utm_source=gb-gplus-share</span><span class="main-title" itemprop="name">Patent US5858731 - Nucleotide sequences including nondegenerate and degenerate nucleotide bases which can be ligated together for longer molecules; for rapid and efficient synthesis of preselected sequences for dna sequencing, polymerase chain reaction</span><img itemprop="image" src="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"alt="Patent US5858731 - Nucleotide sequences including nondegenerate and degenerate nucleotide bases which can be ligated together for longer molecules; for rapid and efficient synthesis of preselected sequences for dna sequencing, polymerase chain reaction" title="Patent US5858731 - Nucleotide sequences including nondegenerate and degenerate nucleotide bases which can be ligated together for longer molecules; for rapid and efficient synthesis of preselected sequences for dna sequencing, polymerase chain reaction"></div><div style="display: none"><ol id="ofe-gear-menu-contents" class="gbmcc"><li class="gbe gbmtc"><a class="gbmt goog-menuitem-content" id="" href="http://www.google.com/advanced_patent_search">Advanced Patent Search</a></li></ol></div><table id="viewport_table" cellpadding="0" style="clear:both" cellspacing="0"><tr><td id="viewport_td"><div class=vertical_module_list_row><div id=intl_patents class=about_content><div id=intl_patents_v><table class="patent-bibdata"><tr><td class="patent-bibdata-heading">Publication number</td><td class="single-patent-bibdata">US5858731 A</td></tr><tr><td class="patent-bibdata-heading">Publication type</td><td class="single-patent-bibdata">Grant</td></tr><tr><td class="patent-bibdata-heading">Application number</td><td class="single-patent-bibdata">US 08/313,232</td></tr><tr><td class="patent-bibdata-heading">Publication date</td><td class="single-patent-bibdata">Jan 12, 1999</td></tr><tr><td class="patent-bibdata-heading">Filing date</td><td class="single-patent-bibdata">Feb 20, 1996</td></tr><tr><td class="patent-bibdata-heading">Priority date<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The priority date is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the date listed."></span></td><td class="single-patent-bibdata">May 8, 1991</td></tr><tr><td class="patent-bibdata-heading">Fee status<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The fee status is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the status or dates listed."></span></td><td class="single-patent-bibdata">Paid</td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Also published as</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/WO1993020096A1">WO1993020096A1</a></span></span></td></tr><tr class="patent-bibdata-list-row alternate-patent-number"><td class="patent-bibdata-heading">Publication number</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value">08313232, </span><span class="patent-bibdata-value">313232, </span><span class="patent-bibdata-value">US 5858731 A, </span><span class="patent-bibdata-value">US 5858731A, </span><span class="patent-bibdata-value">US-A-5858731, </span><span class="patent-bibdata-value">US5858731 A, </span><span class="patent-bibdata-value">US5858731A</span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Inventors</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Joseph+A.+Sorge%22">Joseph A. Sorge</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Daniel+Davis+Shoemaker%22">Daniel Davis Shoemaker</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Original Assignee</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=inassignee:%22Stratagene%22">Stratagene</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Export Citation</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/US5858731.bibtex">BiBTeX</a>, </span><span class="patent-bibdata-value"><a href="/patents/US5858731.enw">EndNote</a>, </span><span class="patent-bibdata-value"><a href="/patents/US5858731.ris">RefMan</a></span></span></td></tr><tr class="patent-internal-links"><td colspan=2><span class="patent-bibdata-value"><a href="#backward-citations">Patent Citations</a> (6),</span> <span class="patent-bibdata-value"><a href="#npl-citations">Non-Patent Citations</a> (4),</span> <span class="patent-bibdata-value"><a href="#forward-citations">Referenced by</a> (8),</span> <span class="patent-bibdata-value"><a href="#classifications">Classifications</a> (27),</span> <span class="patent-bibdata-value"><a href="#legal-events">Legal Events</a> (11)</span> </td></tr><tr><td colspan=2 class="patent-bibdata-external-link-spacer-top"></td></tr><tr class="patent-bibdata-external-link-spacer-bottom"></tr><tr><td colspan=2><span class="patent-bibdata-heading">External Links:&nbsp;</span><span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=gx1JBAABERAJ&q=http://patft.uspto.gov/netacgi/nph-Parser%3FSect2%3DPTO1%26Sect2%3DHITOFF%26p%3D1%26u%3D/netahtml/PTO/search-bool.html%26r%3D1%26f%3DG%26l%3D50%26d%3DPALL%26RefSrch%3Dyes%26Query%3DPN/5858731&usg=AFQjCNEo9-X8JoNEELf1HUDhEnw0ap5zqw">USPTO</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=gx1JBAABERAJ&q=http://assignments.uspto.gov/assignments/q%3Fdb%3Dpat%26pat%3D5858731&usg=AFQjCNEcb6wGhmOd15the5nnRtEgoG4FSQ">USPTO Assignment</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=gx1JBAABERAJ&q=http://worldwide.espacenet.com/publicationDetails/biblio%3FCC%3DUS%26NR%3D5858731A%26KC%3DA%26FT%3DD&usg=AFQjCNF1rZYafXBIr4dyXVBoiC6ROiw3bA">Espacenet</a></span></span></td></tr><tr class="patent-bibdata-group-spacer"></tr></table><div class="number-and-title"><span class="patent-title"><invention-title mxw-id="PT129693292" lang="EN" load-source="ifi">Nucleotide sequences including nondegenerate and degenerate nucleotide bases which can be ligated together for longer molecules; for rapid and efficient synthesis of preselected sequences for dna sequencing, polymerase chain reaction</invention-title></span><br><span class="patent-number">US 5858731 A</span></div><div class="patent-section patent-abstract-section"><div class="patent-section-header"><span class="patent-section-title">Abstract</span></div><div class="patent-text"><abstract mxw-id="PA131529235" lang="EN" load-source="patent-office"> <div class="abstract">An oligonucleotide library is described that is1 useful for producing an oligonucleotide of preselected sequence comprising a plurality of oligonucleotide members comprising one or more oligonucleotide species and having the compositional formula (X)<sub>a</sub> (N)<sub>b</sub> ; wherein X represents a non-degenerate nucleotide base and N represents a degenerate nucleotide base; "a" represents the number of non-degenerate nucleotide positions and is from 3 to 8; "b" represents the number of degenerate nucleotide positions and is from 0 to 4 but not greater that "a"; and wherein each of the oligonucleotide species is capable of forming a hybridization complex with at least one other of the oligonucleotide species in the library such that a single ligation event of the hybridization complex with another hybridization complex derived from the library produces a ligation reaction product comprising greater than 12 contiguous nucleotide base pairs.</div>
  </abstract></div></div><div class="patent-section patent-drawings-section"><div class="patent-section-header"><span class="patent-section-title">Images<span class="patent-section-count">(6)</span></span></div><div class="patent-drawings-body"><div class="patent-drawings-carousel"><div class="drawings"><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5858731-1.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5858731-1.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5858731-2.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5858731-2.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5858731-3.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5858731-3.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5858731-4.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5858731-4.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5858731-5.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5858731-5.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5858731-6.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5858731-6.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div></div></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img" alt="Previous page"src="/googlebooks/images/kennedy/page_left.png"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img" alt="Next page"src="/googlebooks/images/kennedy/page_right.png"width="21" height="21" /></div></div></div><div class="patent-post-drawings"></div><div class="patent-section patent-claims-section"><div class="patent-section-header"><span class="patent-section-title">Claims<span class="patent-section-count">(20)</span></span></div><div class="patent-text"><div mxw-id="PCLM59502376" lang="EN" load-source="patent-office" class="claims">
    <claim-statement>What is claimed is:</claim-statement> <div class="claim"> <div num="1" class="claim">
      <div class="claim-text">1. An oligonucleotide library useful for producing an oligonucleotide of preselected sequence comprising a plurality of oligonucleotide members comprising one or more oligonucleotide species and having the compositional formula (X)<sub>a</sub> (N)<sub>b</sub> ; wherein X represents a non-degenerate nucleotide base and N represents a degenerate nucleotide base; "a" represents the number of non-degenerate nucleotide positions and is from 3 to 8; "b" represents the number of degenerate nucleotide positions and is from 0 to 4 but not greater than "a"; and wherein each of said oligonucleotide species is capable of forming a hybridization complex with at least one other of said oligonucleotide species in said library, said hybridization complex having at least one unpaired nucleotide base overhang such that a single ligation event of said hybridization complex with another hybridization complex derived from said library having a complementary unpaired nucleotide base overhang produces a ligation reaction product comprising greater than 12 contiguous nucleotide base pairs.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="2" class="claim">
      <div class="claim-text">2. The oligonucleotide library of claim 1 wherein said hybridization complexes comprise at least one nucleotide.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="3" class="claim">
      <div class="claim-text">3. The oligonucleotide library of claim 1 wherein said library contains no greater than 60,000 different oligonucleotide species.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="4" class="claim">
      <div class="claim-text">4. The oligonucleotide library of claim 1 wherein said compositional formula is selected from the group consisting of (X)<sub>5</sub> (N)<sub>1</sub>, (X)<sub>4</sub> (N)<sub>2</sub>, (X)<sub>6</sub> (N)<sub>1</sub>, (X)<sub>5</sub> (N)<sub>2</sub> and (X)<sub>6</sub> (N)<sub>2</sub>.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="5" class="claim">
      <div class="claim-text">5. The oligonucleotide library of claim 4 wherein said compositional formula is (X)<sub>6</sub> (N)<sub>2</sub>.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="6" class="claim">
      <div class="claim-text">6. The oligonucleotide library of claim 5 wherein said compositional formula is 5'-XXNXXNXX-3'.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="7" class="claim">
      <div class="claim-text">7. The oligonucleotide library of claim 1 wherein X and N are the nucleotide bases A, T, G, C, or analogs thereof.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="8" class="claim">
      <div class="claim-text">8. The oligonucleotide library of claim 5 wherein said library has 4,096 different oligonucleotide members.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="9" class="claim">
      <div class="claim-text">9. The oligonucleotide library of claim 1 wherein each of said oligonucleotide members have a 5' terminal phosphate.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="10" class="claim">
      <div class="claim-text">10. A kit useful for producing an oligonucleotide of preselected sequence comprising, in separate enclosures, one or more libraries according to claims 1, 2, 3, 4, 5, 6 or 7, and a suitable ligase or polymerase.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="11" class="claim">
      <div class="claim-text">11. A method for producing an oligonucleotide of preselected nucleotide sequence comprising the steps of:<div class="claim-text">a) selecting at least one oligonucleotide member from a library according to claims 1, 2, 3, 4, 5, 6 or 7;</div> <div class="claim-text">b) hybridizing in an aqueous ligation buffer the oligonucleotide member(s) selected in step (a) having a preselected nucleotide sequence and capable of hybridizing to form a ligation reaction substrate; and</div> <div class="claim-text">c) ligating said ligation reaction substrate to form a ligation reaction product containing said oligonucleotide of preselected nucleotide sequence.</div> </div>
    </div>
    </div> <div class="claim-dependent"> <div num="12" class="claim">
      <div class="claim-text">12. The method of claim 11 wherein four different oligonucleotide members are selected in step (a).</div>
    </div>
    </div> <div class="claim-dependent"> <div num="13" class="claim">
      <div class="claim-text">13. The method of claim 11 wherein six different oligonucleotide members are selected in step (a).</div>
    </div>
    </div> <div class="claim"> <div num="14" class="claim">
      <div class="claim-text">14. An oligonucleotide composition useful for producing an oligonucleotide of preselected sequence comprising a plurality of different oligonucleotide species, each having the compositional formula (X)<sub>a</sub> (N)<sub>b</sub> and having a length of from 5 to 12 nucleotides; wherein X represents a non-degenerate nucleotide base and N represents a degenerate nucleotide base; "a" represents the number of non-degenerate nucleotide positions and is from 3 to 8; "b" represents the number of degenerate nucleotide positions and is from 1 to 4 but not greater than "a".</div>
    </div>
    </div> <div class="claim-dependent"> <div num="15" class="claim">
      <div class="claim-text">15. The oligonucleotide composition of claim 14 wherein said compositional formula is selected from the group consisting of (X)<sub>5</sub> (N)<sub>1</sub>, (X)<sub>4</sub> (N)<sub>2</sub>, (X)<sub>6</sub> (N)<sub>1</sub>, (X)<sub>5</sub> (N)<sub>2</sub> and (X)<sub>6</sub> (N)<sub>2</sub>.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="16" class="claim">
      <div class="claim-text">16. The oligonucleotide composition of claim 15 wherein said compositional formula is (X)<sub>6</sub> (N)<sub>2</sub>.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="17" class="claim">
      <div class="claim-text">17. The oligonucleotide composition of claim 16 wherein said compositional formula is 5'-XXNXXNXX-3'.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="18" class="claim">
      <div class="claim-text">18. The oligonucleotide composition of claim 14 wherein said composition contains all possible combinations of nucleotide sequences at positions designated by an N.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="19" class="claim">
      <div class="claim-text">19. The oligonucleotide composition of claim 14 wherein X and N are the nucleotide bases A, T, G, C, or analogs thereof.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="20" class="claim">
      <div class="claim-text">20. The oligonucleotide composition of claim 14 wherein each of said oligonucleotides have a 5' terminal phosphate.</div>
    </div>
  </div> </div></div></div><div class="patent-section patent-description-section"><div class="patent-section-header"><span class="patent-section-title">Description</span></div><div class="patent-text"><div mxw-id="PDES67231514" lang="EN" load-source="patent-office" class="description">
    <p>This application is a national stage filing of International Application PCT/US93/03230, filed Apr. 2, 1993, which is a continuation-in-part of U.S. application Ser. No. 07/863,412, filed Apr. 3, 1992, abandoned. This application is also a continuation-in-part of U.S. application Ser. No. 07/765,694, filed Sep. 25, 1991, abandoned sand a continuation-in-part of U.S. application Ser. No. 07/697,936, filed May 8, 1991, now U.S. Pat. No. 5,599,921.</p>
    <heading>TECHNICAL FIELD</heading> <p>The present invention relates to libraries of oligonucleotides. In addition, the invention relates to methods using libraries of oligonucleotides to produce longer oligonucleotides of preselected nucleotide sequence.</p>
    <heading>BACKGROUND</heading> <p>Oligonucleotides are widely utilized in molecular biological manipulations including DNA sequencing, cycle sequencing, polymerase chain reactions, in vitro mutagenesis, cloning methodologies involving polylinkers and adapters, synthesis of genes by hybridization and ligation of multiple oligonucleotides, and the like methods. Traditionally, oligonucleotides are prepared by chemical synthesis methods de novo each time they are required. Chemical synthesis of oligonucleotides is time consuming and costly.</p>
    <p>One approach to DNA sequencing is called "primer walking" which utilizes known sequence information of a target nucleic acid to be sequenced to design a distal primer which is then used to obtain additional, downstream sequence information. Although primer walking is conceptually appealing, because of its simplicity and the ordered nature of the sequence information obtained, this method can be is expensive and time-consuming because after each sequence is determined, a new, customized primer must be chemically synthesized. Because a single oligonucleotide synthesis requires the preparation of more oligonucleotide than is required for the single sequencing step to be performed, material is wasted resulting in excess cost, and synthesis time slows the sequential sequencing steps.</p>
    <p>Recently, Studier proposed a strategy to simplify the preparation of unique oligonucleotides in the form of a library of pre-synthesized oligonucleotides representing every possible nucleotide sequence in the size range of oligonucleotides from 8 to 10 nucleotides in length. Studier, Proc.Natl.Acad.Sci., 86:6917-6921 (1989). However, the library poses technical difficulties insofar as the library must contain from 4<sup>8</sup> (65,536) to 4<sup>10</sup> (1,048,576) members, respectively, which is generally considered to be so large as to be unmanageable. In addition, oligonucleotides of 8mer to 10mer length are less preferred sequencing primers than longer oligonucleotides of 12mer to 18mer length.</p>
    <p>Szybalski proposed the use of a library of hexameric oligonucleotides comprising every possible combination of nucleotide bases, representing a library having 46 (4,096) members, as a means to reduce the size of the library. Szybalski, Gene, 90:177-178 (1990). Theoretically, pairs of hexamers from the library were proposed to be capable of being individually ligated while hybridized to a template to form 12 nucleotide (nt), 18-nt, or 24-nt oligonucleotides in length, thereby forming every possible nucleotide sequence from a library having 4,096 members. This same approach has been described in U.S. Pat. No. 5,114,639 to Blocker. This approach requires ligation of the hexamer pairs in the presence of template DNA (i.e., DNA molecule to be sequenced).</p>
    <p>Accordingly, there continues to exist a need for preparing oligonucleotides suitable for priming PCR, cycle-sequencing and the like reactions without de novo oligonucleotide synthesis or the above-described problems. The present invention meets that need.</p>
    <heading>BRIEF SUMMARY OF THE INVENTION</heading> <p>It has now been discovered that the problems inherent in the prior art can be overcome by the preparation and use of economically feasible, small libraries of oligonucleotides which allow cohesive-end ligation and construction of longer primers suitable for a variety of uses. Additionally, the methods of the present invention permit the production of an excess of primers in a non-template directed manner.</p>
    <p>In accordance with the present invention oligonucleotide compositions and libraries of oligonucleotide compositions are utilized for producing longer oligonucleotides of preselected nucleotide sequence. Difficulties in template-dependent variability of ligation reactions are overcome by avoiding template-dependent ligation, and relying instead upon the use of complementarity in the overhangs of short duplexes of oligonucleotides. It is shown herein that reproducible ligation of oligonucleotides can be obtained between short DNA duplexes having complementary overhangs.</p>
    <p>As an example, Scheme I below shows at the first step ligation of two duplexes having complementary dinucleotide overhangs formed by hybridizing two octanucleotides (8mers). ##STR1##</p>
    <p>According to Scheme I, 8mer/8mer DNA duplexes are first formed from hybridized 8mer oligonucleotides A, B, C, and D to form a 16mer oligonucleotide primer of preselected sequence designated as an A:C ligation reaction product, representing a ligation of oligonucleotides A and C. The colons (::::) between nucleotide bases indicates the formation of a phosphodiester bond between ligated adjacent bases. The 16mer can be used directly as a primer in a sequencing reaction without purification of the ligation reaction admixture.</p>
    <p>As illustrated in Scheme I, the ligation reaction is typically dependent upon the presence of a 5'-terminal phosphate group, where ligation is mediated enzymatically by a ligase, although other ligation chemistries are possible and contemplated. In that regard, the library used in the present method of Scheme I may be provided with or without the 5'-terminal phosphate. If absent, the user of the library may add the phosphate prior to conducting the ligation reaction.</p>
    <p>In accordance with a related embodiment of the invention, library size limitations are overcome by the use of degenerate oligonucleotides in which a single oligonucleotide composition contains multiple oligonucleotide species. For example, a representative degenerate octanucleotide composition can be described by the formula 5'-XXNXXNXX-3', where nucleotides (or analogs thereof) designated "X" are the same at any one position for all octanucleotides in the composition (non-degenerate) and nucleotides designated "N" can be any one of A, T, G, C, and preferably a mixture of all four, or analogs thereof (degenerate).</p>
    <p>The combination of the specificity of hybridization between complementary duplex overhangs and the non-opposing positions of the degenerate nucleotides in the oligonucleotide compositions allows for a unique solution to problems associated with ligation variability and library size. As an example, Scheme II below illustrates the combination of overhangs and degeneracy in a ligation reaction to favor the formation of a single species of 16mer oligonucleotide reaction product. ##STR2##</p>
    <p>According to Scheme II, 8mer/8mer DNA duplexes are first formed from 8mer oligonucleotides An, Bn, Cn, and Dn, and the resulting overhangs on the duplexes are ligated to form a 16mer oligonucleotide primer of preselected sequence designated as an A:C ligation reaction product. The "n" in the oligonucleotide designation indicates that the oligonucleotide is a degenerate oligonucleotide composition. Furthermore, it is important to note that upon hybridization and ligation of the non-annealing oligonucleotides, the oligonucleotides where N positions destabilize the complex are selected away, and the ligation reaction product is enriched for certain oligonucleotides. The 16mer ligation reaction product can be used as a primer in a sequencing reaction without purification from the ligation reaction mixture.</p>
    <p>Following the above approaches, numerous embodiments are contemplated by the present invention, particularly with respect to degenerate oligonucleotides. In one embodiment, the invention contemplates an oligonucleotide library useful for producing an oligonucleotide of preselected sequence comprising a plurality of oligonucleotide members comprising one or more oligonucleotide species and having the compositional formula (X)<sub>a</sub> (N)<sub>b</sub>. "X" represents a non-degenerate nucleotide base and N represents a degenerate nucleotide base. "a" represents the number of non-degenerate nucleotide positions and is from 3 to 8, and "b" represents the number of degenerate nucleotide positions and is from 0 to 4 but not greater that "a". Each of the oligonucleotide species in the library is capable of forming a hybridization complex with at least one other of the oligonucleotide species in the library such that a single ligation event of the hybridization complex with another hybridization complex derived from the library produces a ligation reaction product comprising greater than 12 contiguous nucleotide base pairs.</p>
    <p>By "single ligation event" is meant that the formation of a single bond between adjacent nucleotides of a ligation reaction substrate comprised of at least two hybridization complexes forms a double-stranded ligation reaction product having a length greater than 12 contiguous nucleotide pairs.</p>
    <p>The combination of two hybridization complexes is typically represented as shown in Scheme II, above. Therefore, in preferred embodiments each hybridization complex comprises at least one unpaired nucleotide. Preferably, the complex has one to three unpaired (overhanging) nucleotides, and more preferably has two unpaired nucleotides.</p>
    <p>A particularly preferred oligonucleotide library has the compositional formula is selected from the group consisting of (X)<sub>5</sub> (N)<sub>1</sub>, (X)<sub>4</sub> (N)<sub>2</sub>, (X)<sub>6</sub> (N)<sub>1</sub>, (X)<sub>5</sub> (N)<sub>2</sub> and (X)<sub>6</sub> (N)<sub>2</sub>, particularly (X)<sub>6</sub> (N)<sub>2</sub>, and more particularly 5'-XXNXXNXX-3'.</p>
    <p>Also contemplated by the invention is a kit useful for producing an oligonucleotide primer of preselected sequence comprising, in separate enclosures, one or more libraries of the invention. A kit may also include a suitable ligase and/or a polymerase for use in practicing the methods of the invention.</p>
    <p>Further in accordance with the present invention are methods for producing an oligonucleotide of preselected nucleotide sequence using one or more oligonucleotide members from a library of the invention. A preferred method comprises the steps of:</p>
    <p>a) selecting at least one oligonucleotide member from a library according to the invention;</p>
    <p>b) hybridizing in an aqueous ligation buffer the oligonucleotide member(s) selected in step (a) having a preselected nucleotide sequence and capable of hybridizing to form a ligation reaction substrate; and</p>
    <p>c) ligating the ligation reaction substrate to form a ligation reaction product containing the oligonucleotide of preselected nucleotide sequence.</p>
    <p>Also in accordance with the present invention, oligonucleotide compositions useful for producing oligonucleotide primers or hybridization probes are provided. In one embodiment, the oligonucleotide compositions comprise a plurality of different oligonucleotide species each having the compositional formula (X)<sub>a</sub> (N)<sub>b</sub> and a length of from 5 to 12 nucleotides; wherein X represents a non-degenerate nucleotide base and N represents a degenerate nucleotide base; "a" represents the number of non-degenerate nucleotide positions and is from 3 to 8; "b" represents the number of degenerate nucleotide positions and is from 0 to 4 but not greater that "a". In preferred embodiments the nucleotides X and N are the nucleotide bases A, T, G, C, or analogs thereof.</p>
    <p>Particularly preferred is an oligonucleotide composition wherein the oligonucleotide species in the composition each have a nucleotide sequence according to the compositional formula selected from the group consisting of (X)<sub>5</sub> (N)<sub>1</sub>, (X)<sub>4</sub> (N)<sub>2</sub>, (X)<sub>6</sub> (N)<sub>1</sub>, (X)<sub>5</sub> (N)<sub>2</sub> and (X)<sub>6</sub> (N)<sub>2</sub> particularly (X)<sub>6</sub> (N)<sub>2</sub>, and more particularly 5'-XXNXXNXX-3'.</p>
    <p>Other embodiments will be apparent to one skilled in the art.</p>
    <heading>BRIEF DESCRIPTION OF THE DRAWINGS</heading> <p>FIG. 1 illustrates a typical ligation reaction scheme according to one embodiment using 6mer/8mer DNA duplexes as examples to form a 14mer oligonucleotide primer of preselected sequence.</p>
    <p>FIG. 2 is a flow chart diagramming the steps in a ligation method for producing an oligonucleotide having a preselected nucleotide sequence using two libraries (A and B) according to the methods described herein. The flow chart shows a preferred embodiment using 6mer/8mer duplex DNA molecules having cohesive GC termini, with the 8mer oligonucleotide of library B having a phosphorylated 5' terminus to form a 14mer.</p>
    <p>FIG. 3 illustrates an agarose gel electrophoretic analysis of the polymerase chain reaction (PCR) product formed using an oligonucleotide primer (14mer) produced according to the ligation reaction shown in FIG. 1 as described in Example 3. The schematic in the upper portion of FIG. 3 illustrates the amplification by PCR of a 980 base pair (bp) fragment from an M13mp18 using a primer pair (primers A and B). The lower portion of FIG. 3 illustrates the agarose gel analysis showing that the 980 bp PCR product was amplified from ssM13mp18 using the following primer pairs: Lane 1, 19mer primer A' and 22mer primer B (both chemically synthesized); Lane 2, 14mer primer A (created by ligation) and 22mer primer B (chemically synthesized); Lane 3, 6mer/8mer (no ligase control) and 22mer primer B. Also shown is a lane of PhiX174/Hae III markers containing 1353, 1078, 872 and 603 bp fragments. The primer sequences are shown in Table 1 at page 62.</p>
    <p>FIG. 4 illustrates the sequencing results of cycle-sequencing ssm13mp18 using an oligonucleotide 14mer primer produced as described in Example 4. The left panel shows a sequencing gel profile produced using the chemically synthesized 19mer positive control (Primer A'), the center panel shows a sequencing gel profile produced using the 14mer ligation product (primer A), and the right panel shows a sequencing gel profile produced using the ligation 25 reaction product when no ligase is added to the ligation reaction.</p>
    <p>FIG. 5 illustrates the sequencing gel results of cycle-sequencing pBluescript® plasmid using a 16mer oligonucleotide primer produced as described in Example 6 by ligation of degenerate oligonucleotide compositions. Sequencing results are shown using the primers in primer set 6 described in Table 2, in 12 consecutive lanes, where the four lanes labeled 6A illustrate sequencing with a chemically synthesized positive control 16mer primer (oligonucleotide 6), the four lanes labeled 6B illustrate sequencing with the ligation reaction product produced from ligation of degenerate oligonucleotides 6An, 6Bn, 6Cn and 6Dn, and the four lanes labeled 6C illustrate sequencing with the same degenerate oligonucleotides without ligase added to the ligation reaction. Similar 12 lanes A, B and C are shown for sequencing results using primer set 7n and for primer set 8n. A control sequencing ladder is shown in four lanes on the extreme left of the Figure using a universal primer (1) for sequencing pBluescript® plasmid.</p>
    <p>FIG. 6 illustrates the sequencing gel results of sequencing pBluescript® plasmid with reverse transcriptase using a 16mer oligonucleotide primer produced as described in Example 7 by ligation of degenerate oligonucleotide compositions. Sequencing results are shown using the primers in primer set 7 in 16 consecutive lanes (A-D), where the four lanes labeled A illustrate a negative control by sequencing with a reaction product produced from a mock-ligation reaction of degenerate oligonucleotide compositions 7An, 7Bn, 7Cn and 7Dn without ligase added to the ligation reaction, the four lanes labeled B illustrate sequencing with a positive control chemically synthesized 16mer primer (oligonucleotide 7), the four lanes labeled C illustrate sequencing with the ligation reaction product from degenerate oligonucleotide compositions 7An, 7Bn, 7Cn and 7Dn with ligase added to the ligation reaction, and the four lanes labeled D illustrate sequencing as in the C lanes, except the primer used was formed by ligation of the non-degenerate oligonucleotides 7A, 7B, 7C and 7D.</p>
    <p>FIG. 7 illustrates the sequencing gel results of sequencing pBluescript® plasmid with Sequenase using a 16mer oligonucleotide primer produced as described in Example 8 by ligation of degenerate oligonucleotide compositions. Sequencing results are shown using the same format and with the same primers as described in the legend to FIG. 5, except that the nucleotide sequences were produced using Sequenase sequencing.</p>
    <heading>DETAILED DESCRIPTION OF THE INVENTION</heading> <p>Definitions</p>
    <p>Nucleotide: as used herein refers to a monomeric unit of DNA or RNA consisting of a sugar moiety (pentose), a phosphate, and a nitrogenous heterocyclic base, or a functional analog thereof. The base is linked to the sugar moiety via the glycosidic carbon (1' carbon of the pentose) and that combination of base and sugar is a nucleoside. When the nucleoside contains a phosphate group bonded to the 3' or 5' position of the pentose it is referred to as a nucleotide. The nucleotides are adenine, thymine, cytosine, guanine, uracil, and analogs thereof.</p>
    <p>Base Pair (bp): A partnership of the nucleotides adenine (A) with thymine (T), or of cytosine (C) with guanine (G) in a double stranded DNA molecule. In RNA, uracil (U) is substituted for thymine. Nucleotide analogs may also pair up to form a base pair so long as the partnership is specific and complementary in a manner analogous to the above nucleotides.</p>
    <p>Nucleic Acid: A polymer of nucleotides, either single or double stranded.</p>
    <p>Oligonucleotide: The term "oligonucleotide" or "oligo" as used herein in reference to primers, probes and nucleic acid fragments or segments is defined as a polymeric molecule comprised of a plurality of deoxyribonucleotides, ribonucleotides, or analogs thereof, preferably at least 5. Its exact size will depend on many factors, which in turn depends on the ultimate conditions of use.</p>
    <p>Gene: A nucleic acid whose nucleotide sequence codes for an RNA or polypeptide. A gene can be either RNA or DNA.</p>
    <p>Complementary Bases: Nucleotides or analogs thereof that normally pair up when oligonucleotides adopt a double stranded configuration. Chemical modifications to the ribose or phosphate backbones, or to the bases, can be incorporated to form a nucleotide analog so long as the analog does not inhibit hybridization with another oligonucleotide, or analog thereof, and the analog has its own specific complementary base.</p>
    <p>Complementary Nucleotide Sequence: A sequence of nucleotides in an oligonucleotide capable of hybridizing to another oligonucleotide for a length of time sufficient to permit a desired event, e.g., a ligation reaction, a primer extension reaction, or ligand detection.</p>
    <p>Conserved: A nucleotide sequence is conserved with respect to a preselected (reference) sequence if it non-randomly hybridizes to an exact complement of the preselected sequence.</p>
    <p>Library: A collection of separate oligonucleotide compositions, each composition representing a different member of the library, and each present in a separate enclosure.</p>
    <p>Hybridization: The pairing of substantially complementary nucleotide sequences to form a duplex or heteroduplex by the establishment of hydrogen bonds between complementary bases. It is a specific, i.e. non-random, interaction between two complementary polynucleotides that can be competitively inhibited.</p>
    <p>Nucleotide Analog: A purine-like or pyrimidine-like nucleotide that differs structurally from A, T, G, C, or U, but is sufficiently similar in function to substitute for the normal nucleotide in an oligonucleotide nucleic acid molecule and participate in complementary hybridization with a complementary nucleotide.</p>
    <p>Overhang: A region of single strandedness at a termini of a double-stranded (duplex) oligonucleotide molecule that is typically available to hybridize to a complementary single-stranded overhang.</p>
    <p>Upstream: In the direction opposite to the direction of DNA transcription, and therefore going from 5' to 3' on the non-coding strand, or 3' to 5' on the coding strand or mRNA.</p>
    <p>Downstream: Further along a DNA sequence in the direction of sequence transcription, that is traveling in a 3'- to 5'-direction along the non-coding strand of the DNA or a 5'- to 3'-direction along the coding strand.</p>
    <p>B. Oligonucleotide Libraries</p>
    <p>The present invention provides a means for the rapid and inexpensive synthesis of oligonucleotides of preselected sequence for use in DNA sequencing, cycle sequencing, polymerase chain reaction (PCR) procedures, and other applications requiring oligonucleotides. The approach described herein utilizes oligonucleotide compositions that are used to form longer oligonucleotides of preselected sequence suitable for use as primers in molecular biological manipulations, particularly DNA sequencing and PCR, or as hybridization reagents.</p>
    <p>Typically, an oligonucleotide composition of this invention is organized with other oligonucleotide compositions into a collection, or library, comprised of a plurality of different library members, each member representing a different oligonucleotide composition containing oligonucleotide species of known sequence. Users of the library select those members which, upon ligation according to the methods described herein, form a longer oligonucleotide of preselected sequence suitable for use as a primer or hybridization reagent.</p>
    <p>A library of this invention is comprised of different members (oligonucleotides or oligonucleotide compositions) each present in a separate enclosure and each having different nucleotide sequence(s). The oligonucleotides in an individual enclosure can be homogeneous, i.e., all oligonucleotide molecules in an enclosure are identical in sequence. Alternatively, the oligonucleotides in an individual enclosure can be heterogeneous, i.e., the enclosure contains an oligonucleotide composition of this invention having oligonucleotides of different but related nucleotide sequences (i.e., degenerate).</p>
    <p>Thus a library of this invention comprises a collection (plurality) of oligonucleotide members, each having a different preselected nucleotide sequence, that can be used to construct larger oligonucleotides of preselected sequence by combining the members. In order to construct any possible nucleotide sequence by combining members, the library preferably contains every possible combination of nucleotide sequence.</p>
    <p>The size of the library that has no degeneracies is determined by the expression 4<sup>n</sup> where the superscript "n" represents the number of bases in each of the oligonucleotide subunits making up the library. Thus, a small increase in oligonucleotide length dramatically increases the size of the library. For example, a library of oligos having 6 nucleotide bases, also referred to as a 6mer, contains 4096 different oligos, whereas a 9mer library has over 260,000 primers.</p>
    <p>The library, in one embodiment typically has at least 100 members, preferably about 100 to 100,000 different members, and more preferably 1,000 to 60,000 members. Consistent with the objectives herein for reducing a library size, a library of less that 30,000 members is particularly preferred, and even more preferably contains about 1,000 to 5,000 members.</p>
    <p>Each member of a library of this invention comprises one or more oligonucleotide species and typically has a length of at least 5 nucleotides, preferably 5 to 12 nucleotides, more preferably about 5 to 10 nucleotides, and particularly about 6 to 8 nucleotides, although longer oligonucleotides can be present in the library.</p>
    <p>In a preferred embodiment a library contains a plurality of different oligonucleotide members which are degenerate oligonucleotide compositions according to the present invention. Although other oligonucleotide libraries are described herein, those containing degenerate oligonucleotide compositions are particularly preferred because the degeneracy reduces the size of the library without limiting capacity or usefulness of the library, thereby reducing manufacturing costs and library management.</p>
    <p>The use of degenerate oligonucleotide compositions in the present invention is particularly advantageous in that it significantly reduces the library size needed to provide every possible nucleotide sequence in the oligonucleotides of the library. Of course, the degree of library size reduction is a function of the number of degenerate nucleotide positions included in a oligonucleotide.</p>
    <p>Thus, in one embodiment, the invention provides an oligonucleotide library useful for producing an oligonucleotide of preselected sequence comprising a plurality of oligonucleotide members comprising one or more oligonucleotide species and having the compositional formula (X)<sub>a</sub> (N)<sub>b</sub>, wherein X represents a non-degenerate nucleotide base and N represents a degenerate nucleotide base, "a" represents the number of non-degenerate nucleotide positions and is from 3 to 8, and "b" represents the number of degenerate nucleotide positions and is from 0 to 4 but not greater that "a". Each oligonucleotide species of the library in this embodiment is capable of forming a hybridization complex with at least one other oligonucleotide species in the library such that a single ligation event of the hybridization complex with another hybridization complex derived from the library produces a ligation reaction product comprising greater than 12 contiguous nucleotide base pairs.</p>
    <p>As described for the degenerate oligonucleotide compositions of this invention, the library can comprise any combination of lengths, number of degenerate nucleotides, and choices of nucleotide bases within the degenerate nucleotide position. Preferred degenerate libraries comprise oligonucleotide members according to a compositional formula selected from the group consisting of (X)<sub>5</sub> (N)<sub>1</sub>, (X)<sub>4</sub> (N)<sub>2</sub>, (X)<sub>6</sub> (N)<sub>1</sub>, (X)<sub>5</sub> (N)<sub>2</sub> and (X)<sub>6</sub> (N)<sub>2</sub>, such that the position of the degenerate nucleotide(s) can be at any of a variety of positions. A particularly preferred degenerate library comprises octanucleotide members according to the compositional formula (X)<sub>6</sub> (N)<sub>2</sub>. Exemplary of this preferred embodiment is a library with a degenerate octanucleotide composition having two degenerate positions, thereby requiring 4,096 different compositions in order to account for all possible sequences. A particularly preferred library contains 4,096 different octanucleotide compositions, wherein each composition comprises octanucleotides according to the formula: 5'-XXNXXNXX-3' as described herein.</p>
    <p>In an alternate embodiment, each oligonucleotide member in a library comprises one or more species of oligonucleotide having a sequence that is complementary with at least one other species of the library along a linear (contiguous) stretch of nucleotides. Thus, each species of this library has a nucleotide sequence such that it can form, upon complementary hybridization with another species of the library, a double-stranded (ds) duplex DNA molecule having an overhang on at least one terminus. The individual species of this library can vary in length and in sequence from one another, so long as the library is designed as defined herein to allow the selection of hybridizable pairs to form duplex DNA having the overhangs as required to practice the methods of this invention.</p>
    <p>In another embodiment, a library is comprised of a plurality of enclosures, each enclosure containing a different oligonucleotide species having a "common" (i.e., shared by all the oligonucleotide species in the library) nucleotide sequence of at least 1 nucleotide, and preferably from 1 to 3 nucleotides, in length located at one terminus of the oligonucleotide, which terminus is the same in each oligonucleotide species of the library.</p>
    <p>The oligonucleotide species in a library in this embodiment can therefore be represented by the formula YZ, where Y represents the nucleotide sequence that is the same (common) in all species of the library, and Z represents the nucleotide sequence that is different for each species of the library. Preferably, Z defines the region of complementarity when present in a duplex of the present invention such that the overhang in the duplex is defined by the sequence represented by Y. Y is at least one nucleotide in length, preferably from 1 to 3 nucleotides in length, more preferably 2 nucleotides long, and Z can be from 5 to 10 nucleotides in length, preferably 6 to 8.</p>
    <p>A library preferably contains all possible nucleotide sequences definable with a four nucleotide base (A, T, G, and C) vocabulary in a given length. Thus, a library where X or Z is six nucleotides in length preferably has 4,096 different members.</p>
    <p>However, there are instances where less than all four bases are desired at a certain nucleotide position in members of the library. For example, a certain position in the oligonucleotide may be limited to only GC (such at a two base terminal portion).</p>
    <p>In one embodiment, Y defines the 5' terminus of the oligonucleotide. In another embodiment, all the members of the library have a 5' phosphate, preferably at the 5' terminus of the oligonucleotide. In another embodiment, the invention contemplates a library where all the oligonucleotide members are free from phosphate.</p>
    <p>Particularly preferred oligonucleotide libraries are comprised of oligonucleotides as described above but having a dinucleotide termini that has a sequence selected from the group consisting of GG, CC, TT, AA, TC, CT, GA, AG, TG, GT, AC, CA, NI and IN, where N is selected from the group consisting of A, T, G and C, and where I is inosine. Alternatively, the dinucleotide termini can be CI, IC, TA or AT for the reasons described herein.</p>
    <p>Degenerate Oligonucleotide Compositions</p>
    <p>In a related embodiment, the present invention provides oligonucleotide compositions each comprising a population of sequence-related oligonucleotide species which differ in nucleotide sequence only at preselected nucleotide positions. The nucleotide base difference in these compositions can be represented as the letter "N", designating A, T, G, or C (or analogs thereof) at that position as in, for example, an 8mer oligonucleotide composition having the formula 5'-XXNXXNXX-3', wherein X designates the nucleotide bases in the sequence of the oligonucleotide that are common to all oligonucleotide species within the composition. The N position may, but need not necessarily, comprise all possible nucleotide bases, or analogs thereof.</p>
    <p>Oligonucleotide compositions in which at least one nucleotide base position is represented by an "N" are referred to as degenerate oligonucleotide compositions because the oligonucleotide species in the composition, although different in part, contain redundancies with regard to their respective sequences at positions of the sequence defined by an X that is the same in each oligonucleotide species in the composition.</p>
    <p>In one embodiment, the invention contemplates degenerate oligonucleotide compositions in which nucleotides at the position "N" is within the region of the oligonucleotide species of the composition selected to hybridize to a complementary oligonucleotide species (partner) in the duplex. An example of this embodiment is represented by the formula for a 6mer/8mer duplex:</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="equation">5'-XXXXXX-3'</pre>
    
    <p>
      </p> <pre xml:space="preserve" listing-type="equation">3'-XXNXXNXX-5'</pre>
    
    <p>In the above example the degenerate nucleotides (N) are at positions 3 and 6 of the 8mer when positions are counted in the direction of 5' to 3', and can be referred to as N<sub>3</sub> and N<sub>6</sub>. Such an 8mer is referred to as an N<sub>3</sub>,N<sub>6</sub> -8mer oligonucleotide composition. It is to be appreciated that other formulas can also be suitably utilized to designate oligonucleotide compositions in this embodiment, by placing a degenerate nucleotide at any position in the oligonucleotide such as is shown by any of the following representative 8mer compositions: N<sub>1</sub>,N<sub>2</sub> -; N<sub>1</sub>,N<sub>3</sub> -; N<sub>1</sub>,N<sub>4</sub> -; N<sub>1</sub>,N<sub>5</sub> -; N<sub>1</sub> N<sub>6</sub> -; N<sub>1</sub>,N<sub>7</sub> -; N<sub>1</sub>,N<sub>8</sub> -; N<sub>2</sub>,N<sub>3</sub> -; N<sub>2</sub> N<sub>4</sub> -; N<sub>2</sub> N<sub>5</sub> -; N<sub>2</sub>,N<sub>6</sub> -; N<sub>2</sub>,N<sub>7</sub> -; N<sub>2</sub> N<sub>8</sub> -; N<sub>3</sub>,N<sub>4</sub> -; N<sub>3</sub>,N<sub>5</sub> -; N<sub>3</sub>,N<sub>6</sub> -; N<sub>3</sub>, N<sub>7</sub> -; N<sub>3</sub>,N<sub>8</sub> -; N<sub>4</sub>,N<sub>5</sub> -; N<sub>4</sub>,N<sub>6</sub> -; N<sub>4</sub>,N<sub>7</sub> -; N<sub>4</sub>,N<sub>8</sub> -; N<sub>5</sub>,N<sub>6</sub> -; N<sub>5</sub>,N<sub>6</sub> -; N<sub>5</sub>,N<sub>8</sub> -; N<sub>6</sub>,N<sub>7</sub> -; N<sub>6</sub>,N<sub>5</sub> - and N<sub>7</sub>,N<sub>8</sub> -. Thus, octanucleotide embodiments where there are two degenerate (N) nucleotides and six conserved nucleotides (X) can be represented generally by the compositional formula (X)<sub>6</sub> (N)<sub>2</sub>. Extending this formula, the present invention also provides an oligonucleotide composition according to the compositional formula (X)<sub>a</sub> (N)<sub>b</sub> wherein "a" can be from 3-8 and "b" can be from 0-4 but is not greater than "a".</p>
    <p>As used herein, the term "compositional" indicates the molar ratio of the elements (nucleotides) in the formula irrespective of the order or placement of the "N" residues relative to the "X" residues.</p>
    <p>Degenerate 8mer compositions can be admixed with other complementary nucleotides to form an admixture comprising a duplex having at least one overhang, such as an N<sub>3</sub>,N<sub>6</sub> -8mer/N<sub>3</sub>,N<sub>6</sub> -8mer duplex or a 6mer/N<sub>3</sub>,N<sub>6</sub> -8mer duplex.</p>
    <p>Although there are 16 different N<sub>3</sub>,N<sub>6</sub> -8mer species in a composition, only one of the 16 species is the most energetically favorable complement to the admixed complementary oligonucleotides and thus favored to form a duplex. Upon use in a ligation reaction of this invention, the duplexed N<sub>3</sub>,N<sub>6</sub> -8mer will preferentially participate in the ligation reaction to form the preselected ligation reaction product as can be represented by the equation: ##STR3##</p>
    <p>The above ligation reaction product is represented as 5'-XXXXXX:XXXXXXXX-3' without any N's because the selectivity of complementary hybridization can will produce a predominant species of ligation reaction product complementary to the template. Stated differently, the hybridization conditions can be selected to be sufficiently stringent to greatly favor the desired complementary oligonucleotides to be selected and ligated from the degenerate pool of oligonucleotides. Therefore, reaction schemes are shown with a loss of the "N" upon formation of the ligation reaction product to indicate the selectivity of the hybridization reaction upon ligation with a complementary sequence.</p>
    <p>The above hybridization and ligation reactions can be utilized with the degenerate nucleotides at any of the positions hybridizing in the duplex, and need not be limited to positions N<sub>3</sub> or N<sub>6</sub>. In addition, the degenerate nucleotide positions can be different in the two duplexes (A &amp; B) to be ligated. Furthermore, as will be readily apparent to one skilled in the art, this embodiment can be practiced using multiple degenerate nucleotides and need not be limited to the use of two degenerate positions in an 8mer. Additionally, the degenerate nucleotides (N) can be in any of the oligonucleotides to be ligated, either or both upper (or lower) oligonucleotides, either or both upstream (or downstream) oligonucleotides, in all four oligonucleotides, or any combination thereof.</p>
    <p>Desirable and preferred conditions for obtaining reproducible results in the practicing the present embodiment using degenerate oligonucleotide compositions are (1) to use molar excesses of the degenerate oligonucleotides relative to non-degenerate oligonucleotides to favor formation of the desired duplex, such as a 16 fold molar excess of N<sub>3</sub>,N<sub>6</sub> -8mer relative to 6mer, and (2) to use hybridization conditions that favor formation of complementarity in the duplexes, so as to reduce the production of unwanted ligation reaction products. Hybridization conditions can be controlled by presently known variables to favor complementary hybridization. Where both oligonucleotides are degenerate, the molar ratio is preferably adjusted so that the target species are equimolar.</p>
    <p>In a particularly preferred embodiment, the invention relies on several of the above elements. A preferred composition of degenerate 8mer oligonucleotides (octanucleotides) is defined by the compositional formula 5'-XXNXXNXX-3'. All of the nucleotides in the composition have the same sequence at positions designated by "X", thereby defining a common sequence between all species of oligonucleotide within the composition. Furthermore, the composition contains different oligonucleotides whose sequences differ by the presence of the degenerate position nucleotide "N". Preferably, all four possible nucleotide bases, or their respective analogs, are represented in all possible combinations such that a composition contains oligonucleotide species representing every possible sequence defined by the formula. Thus, where there are two degenerate positions "N", there can be 16 different is oligonucleotide species.</p>
    <p>In a preferred embodiment, the composition of octanucleotides each have a 5' terminal phosphate to facilitate ligation.</p>
    <p>A particularly preferred ligation scheme utilizing degenerate 8mer/8mer duplexes where one of the 8mers has a 5' terminal phosphate is shown in reaction Scheme II. Following reactions as shown in Scheme II, a requirement for production of a single preselected ligation reaction product when using a degenerate oligonucleotide composition is that the non-ligated overhangs (5' termini of An or Dn) should not hybridize efficiently (1) to each other, (2) to the ligated center overhangs (5' termini of Bn or Cn), or (3) to themselves.</p>
    <p>Using the ligation reaction of Scheme II, one can produce any 16mer of a preselected nucleotide sequence using only a library of 4,096 different (X)<sub>6</sub> (N)<sub>2</sub> octanucleotide compositions.</p>
    <p>The ligation reaction of Scheme II was used in Example 5 with degenerate 8mer oligonucleotide compositions, and the resulting ligation reaction product was used in cycle sequencing (Example 6), in reverse transcriptase sequencing (Example 7), and in sequencing reactions using a modified T<sub>7</sub> DNA polymerase (Example 8).</p>
    <p>In one embodiment, the invention contemplates compositions and methods that utilize a degenerate 8mer oligonucleotide composition according to the formula 5'-NNXXXXXX-3'. Each composition has 16 different oligonucleotide species in which the sequence NN can be AA, AT, AG, AC, TT, TA, TG, TC, GG, GA, GT, GC, CC, CA, CT, CG, or analogs thereof which allow specific and complementary hybridization.</p>
    <p>When a degenerate 8mer composition is used according to the present invention in a 6mer/8mer duplex, for example, the sequence NN can be positioned in the 2-base overhang in which case the resulting ligation product is represented by the formula:</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="equation">5'-XXXXXX:NNXXXXXX-3'</pre>
    
    <p>In this embodiment, 16 different 14mer ligation reaction products are formed rather than a single 14mer oligonucleotide. When used as a primer, however, the species with favorable energy complementarity will hybridize and prime the target, and the non-favorable species do not participate in hybridization to template. A colon (:) between oligonucleotides is used to indicate that the adjacent nucleotides have become ligated. Thus, the ligation of degenerate oligonucleotides in this embodiment can be represented as follows: ##STR4##</p>
    <p>In a related embodiment, the invention describes the use of degenerate oligonucleotides of various lengths to form ligation reaction products from multiple oligonucleotides. By selection of "nested" overlapping oligonucleotides that serve as templates for one another, one can build 18mers, 22mers, 24mers, 30mers, and the like. In this embodiment, one or more degenerate oligonucleotide compositions provide regions of complementarity for hybridization of two unique oligonucleotides, having at least four nucleotides of complementarity to each of the two oligonucleotides to be ligated. The template overlaps and thereby joins both unique oligonucleotides.</p>
    <p>In accordance with this embodiment, two different degenerate 8mer compositions can be used to direct the ligation of three oligonucleotides, A, B, and C as follows: ##STR5##</p>
    <p>One skilled in the art having the benefit of the present disclosure will appreciate that additional permutations for using the degenerate oligonucleotide compositions are provided by the present invention. For example a 22mer can be produced by combining a 6mer/8n-mer (i.e., a degenerate 8mer) duplex, an 8n-mer/8n-mer duplex, and an 8n-mer/6mer duplex as follows: ##STR6##</p>
    <p>Similarly, a 24mer oligonucleotide can be produced using a 6mer/8n-mer duplex, a 6mer-6mer/8n- mer triplex, and a 6mer/8n-mer duplex, as follows: ##STR7## or using three 8n-mer/8n-mer duplexes as follows: ##STR8##</p>
    <p>Other possible combinations are readily apparent, and therefore the invention need not be limited by the above examples.</p>
    <p>Double-Stranded (Duplex) DNA Libraries</p>
    <p>One contemplated class of libraries of the invention contains double-stranded (duplex) DNA molecules, referred to as duplex DNA or a duplex. For example, using a 6mer/8mer duplex as exemplary, a library would have up to 4096 different members representing all possible nucleotides sequences in the hexanucleotide (6mer) component of the 6mer/8mer duplex. All members of the library have the same nucleotide sequence in the overhang. Typically, each different duplex is present in a separate enclosure.</p>
    <p>In accordance with the present invention, two or more double-stranded (ds) DNA duplexes which have complementary overhanging (cohesive) termini are ligated to form a ligation reaction product comprising a larger oligonucleotide of preselected sequence. The cohesive termini on the duplexes increase the specificity and reaction rate of the ligation reaction. The use of a DNA duplex comprised of two complementary 8mers (an 8mer/8mer duplex) having a two nucleotide overhang is used as exemplary, and is shown in Scheme I. The downstream duplex preferably contains a phosphate group on the 5' terminus of 8mer oligonucleotide "C" to provide a ligation substrate for DNA ligase. Note that the ligation reaction produces a 16mer ligation reaction product on the upper (sense) strand and two non-ligated oligonucleotides on the lower (non-coding) strand because of the absence of a 5'phosphate on the 5' end of the 8mer designated "B".</p>
    <p>All duplexes of the invention have two domains: at least one overhang, and a region of complementarity between the oligonucleotides of the duplex. Two overhangs arise, for example, where two 8mer oligonucleotides are offset in complementarity to form a duplex having a region of 6 hybridized nucleotides and having two nucleotides in an overhang at each termini.</p>
    <p>In one preferred embodiment, the oligonucleotide of the duplex providing the overhang is adapted for ligation to the terminal 3' nucleotide of another oligonucleotide to direct ligation between two duplexes, designated upstream and downstream duplexes to connote that upon ligation, a single, ligated oligonucleotide of preselected sequence is formed. Such adaptation is preferably in the form of a 5'-terminal phosphate on one overhanging oligonucleotide and a 5'-terminal hydroxyl group on the other overhanging oligonucleotide to permit enzymatic ligation of only one strand and not its complementary oligonucleotide. Other reactive moieties that function as the adaptation means may also be applied to the present technology as to provide directed ligation, and are therefore contemplated.</p>
    <p>Thus the library can be provided with or without a 5' phosphate on the oligonucleotide of the duplex that contributes to the overhang. For example, in one embodiment of a 6mer/8mer duplex library, the 5' terminus of all the 8mers in the library contain a phosphate. Alternatively, a single library can be provided that lacks a 5' terminal phosphate, and the user can add the 5' terminal phosphate as needed.</p>
    <p>Phosphorylation of the 5' termini of oligonucleotides is well known in the art. A useful phosphorylating reaction admixture comprises 30 microliters (ul) of a reaction buffer containing 50 Mm Tris-Hcl, Ph 7.5, 10 Mm MgCl<sub>2</sub>, 5 Mm DTT, 0.1 to 10 micrograms (ug) of oligonucleotide, 1 Mm ATP, 50 micrograms of bovine serum albumin and 20 units of bacteriophage T4 polynucleotide kinase. The phosphorylating admixture is then maintained (incubated) at 37° C. for 30 minutes, whereupon the reaction is stopped, typically by adding 1 μl of 0.5M EDTA to the maintained admixture. If desired,  gamma-<sup>32</sup> P! ATP can be added to the reaction to produce labeled oligonucleotide.</p>
    <p>In one embodiment, the invention contemplates matched first and second libraries, where the first library comprises 6mer/8mer duplexes lacking a 5'phosphate on the 8mer, and the second library comprises 6mer/8mer duplexes each having a 5'phosphate. The libraries are "matched" because the overhangs of the oligonucleotides in the first library are complementary to the overhangs in the second library.</p>
    <p>An example of two matched libraries is shown in FIG. 2, wherein the first and second libraries (represented as libraries A and B) each comprise up to 4096 different duplex DNA members present in separate wells of a microtiter plate, each member comprised of a 6mer/8mer duplex and having complementary GC overhangs formed by the 8mer component.</p>
    <p>Thus one embodiment contemplates a library comprised of a plurality of members, each member comprising a different duplex. Each duplex member of the library is present in a separate enclosure (package). Each duplex member has a unique nucleotide sequence and each duplex in the library has the same nucleotide sequence in the overhang region of the duplex. The region of complementarity in the duplexes of the library is at least 5, and preferably from 5 to 7, nucleotides in length, thereby defining the size of the library. Preferably the region of complementarity, and therefore the length of the shorter oligonucleotide of the duplex is 6 nucleotides, and therefore the library has a size of 4096 oligonucleotide members. The overhang is at least one nucleotide, and preferably is 1 to 3 nucleotides in length.</p>
    <p>A preferred library has a dinucleotide overhang where the overhangs have a sequence selected from the group consisting of GG, CC, TT, AA, TC, CT, GA, AG, TG, GT, AC, CA, NI and IN, where N is selected from the group consisting of A, T, G and C, and where I is inosine.</p>
    <p>Because the primers produced by the present invention are to be utilized in primer extension reactions, the presence of a common region of sequence in the center of the primer derived from the overhang puts a limitation on the use of the primer. To find a complementary match in a template, one must scan a region of the template for the occurrence of the common sequence in order to design a primer to that region. For example, if the library used has a two base overhang, e.g., TC, then template must be scanned for the presence of the complementary AG in order to design a primer to hybridize to that region. The statistical likelihood of locating a AG in a random sequence is one in 16.</p>
    <p>It is preferred to design the overhang in a manner to reduce the amount of template sequence that must be scanned in order to locate a match to the overhang region of the primer being produced by the present methods.</p>
    <p>To that end, libraries having the overhanging nucleotide sequence NI or IN are preferred, particularly CI or IC. By this design, both the first and second libraries can have the same termini, thereby reducing the number of different libraries required to represent every possible sequence. The presence of inosine (I) in the resulting oligonucleotide primer increases the probability of identifying a template sequence having a sequence complementary to the region of the primer derived from the overhang region, as discussed herein.</p>
    <p>In another embodiment, the invention contemplates the use of the dinucleotide TA or AT in the overhang region. Because thymidine is a small pyrimidine, a mismatch in the overhang region when the oligonucleotide is used as a primer can be tolerated. The internally mismatched nucleotide "T" in the primer will still allow the primer to effectively prime PCR and sequencing reactions. Thus, this approach also increases the probability of finding a sequence in a template to be primed that has complementarity to the primer oligonucleotide.</p>
    <p>Oligonucleotide Synthesis</p>
    <p>The oligonucleotide compositions of the present invention can be prepared using any suitable method, such as, for example, the phosphotriester or phosphodiester methods known in the art; see Narang et al., Meth. Enzymol., 68:90, (1979); U.S. Pat. No. 4,356,270; and Brown et al., Meth. Enzymol., 68:109, (1979). Exemplary synthesis is described in Example 1.</p>
    <p>The oligonucleotides can be labeled, i.e., operatively linked to an indicating means or group, and used to detect the presence of a specific nucleotide sequence in a target template. Typically such indicating means is a label such as radioactive atoms, chemically modified nucleotide bases, and the like.</p>
    <p>Radioactive elements operatively linked to or present as part of a oligonucleotide provide a useful means to facilitate the detection of a DNA duplex. A typical radioactive element is one that produces beta ray emissions. Elements that emit beta rays, such as <sup>3</sup> H, <sup>12</sup> C, <sup>32</sup> P, <sup>33</sup> P and <sup>35</sup> S represent a class of beta ray emission-producing radioactive element labels. A radioactive oligonucleotide is typically prepared by enzymatic incorporation of radioactively labeled nucleotides into a nucleic acid using DNA kinase.</p>
    <p>Alternatives to radioactively labeled oligonucleotides are oligonucleotides that are chemically modified to contain metal complexing agents, biotin-containing groups, fluorescent compounds, and the like.</p>
    <p>One useful metal complexing agent is a lanthanide chelate formed by a lanthanide and an aromatic beta-diketone, the lanthanide being bound to the nucleic acid or oligonucleotide via a chelate forming compound such as an EDTA-analogue so that a fluorescent lanthanide complex is formed. See U.S. Pat. No. 4,374,120, Pat. No. 4,569,790 and published Patent Application Nos. EP0139675 and WO87/02708.</p>
    <p>Biotin or acridine ester-labeled oligonucleotides and their use to label oligonucleotides have been described. See U.S. Pat. No. 4,707,404, published Patent Application EP0212951 and European Patent No. 0087636. Useful fluorescent marker compounds include fluorescein, rhodamine, Texas Red, NBD and the like.</p>
    <p>A labeled oligonucleotide present in a DNA duplex renders the duplex itself labeled and therefore distinguishable over other nucleic acids present in a sample to be assayed. Detecting the presence of the label in the duplex and thereby the presence of the duplex, typically involves separating the DNA duplex from any labeled oligonucleotide probe that is not hybridized to a DNA duplex.</p>
    <p>Techniques for the separation of single stranded oligo, such as non-hybridized labeled oligo, from DNA duplex are well known, and typically involve the separation of single stranded from double stranded nucleic acids on the basis of their chemical properties. More often separation techniques involve the use of a heterogeneous hybridization format in which the non-hybridized probe is separated, typically by washing, from the DNA duplex that is bound to an insoluble matrix. Exemplary is the Southern blot technique, in which the matrix is a nitrocellulose sheet and the label is <sup>33</sup> P or <sup>32</sup> P. Southern, J. Mol. Biol., 98:503 (1975).</p>
    <p>Nucleotide bases other than the common four nucleotides (A,T,G or C), or the RNA equivalent nucleotide uracil (U), can be used in the present invention and are designated nucleotide analogs. Thus, for convenience, the term "nucleotide base" is meant, in the context of the present invention to include all possible compounds that can function in the present invention, including common deoxy- or ribonucleotides, nucleotide analogs, and other chemical compounds which might be developed that are suitable for use in a "oligonucleotide" having the property of being incorporated in a duplex DNA molecule without destabilizing the structure of the duplex, and facilitating complementary hybridization. Thus a nucleotide base can be an analog that is a nucleoside with a non-destabilizing moiety, such as a non-protruding side group, phosphate backbone analogs, ribose or deoxyribose analogs, and the like analogs.</p>
    <p>By non-destabilizing hybridization is meant that the nucleotide can participate in DNA-DNA or DNA-RNA (or analog) duplex formation (base pairing) without significantly preventing the ordinary complementary hybridization of adjacent nucleotides in the oligonucleotide that would otherwise hybridize to their complement. Inosine is an example of a non-destabilizing hybridizable nucleotide, with specificity for A, T or C, but not G. Other nucleotides having this property can also be used in the present invention.</p>
    <p>In addition, other modifications are contemplated that are designed to increase an oligonucleotide's resistance to degradation by exonucleolytic enzymes. A preferred modification is to have a thio-phosphodiester linkage between the terminal and penultimate nucleotide at the 3' terminus of an oligonucleotide used in a library or method of this invention.</p>
    <p>Synthesis of degenerate oligonucleotide compositions is accomplished nearly the same as homogeneous oligonucleotide compositions, and can be accomplished by a variety of synthetic chemical means that are not to be construed as limiting to the present invention. Typically, where the synthesis step is to be carried out adding the nucleotides A, T, G, or C at the position corresponding to the degenerate (N) position that is degenerate in the composition, a mixture of all of the nucleotide precursors are added to the synthesis reaction such that all four nucleotides are randomly incorporated into the oligonucleotide at that position, simultaneously producing four different sequences, and preferably in equimolar amounts. However, less than all four common nucleotides can be used at an "N" position. Exemplary syntheses of degenerate oligonucleotides are described herein at Example 1.</p>
    <p>C. Methods for Producing Oligonucleotides of Preselected Sequence</p>
    <p>The invention contemplates methods for producing an oligonucleotide of preselected nucleotide sequence by combining the subunit oligonucleotides of a library described herein. The combining step generally involves ligation of two duplex DNA molecules having complementary cohesive (overhanging) termini, and having known nucleotide sequences. By selecting the duplex DNA molecules, or oligonucleotide subunits for assembling the duplexes, from a pre-made library, the method simplifies the process by eliminating the need for de novo chemical synthesis.</p>
    <p>Thus, the invention describes a method in which oligonucleotides are selected from a library of this invention having sequences preselected to hybridize and form the desired ds DNA molecules. Thus, in this embodiment, the method comprises:</p>
    <p>a) selecting at least one oligonucleotide member from a library of this invention;</p>
    <p>b) hybridizing in an aqueous ligation buffer the oligonucleotide member(s) selected in step (a) having a preselected nucleotide sequence and capable of hybridizing to form a ligation reaction substrate; and</p>
    <p>c) ligating the ligation reaction substrate to form a ligation reaction product containing the oligonucleotide of preselected nucleotide sequence.</p>
    <p>A ligation reaction substrate is a structure where the complementary overhangs of two (or more) ds DNA molecules (hybridized complexes) are hybridized so as to provide a 3'-hydroxylated terminus immediately adjacent to a 5'-phosphorylated terminus. A ligation reaction product is a ligation reaction substrate having had a ligation reaction performed between the adjacent 3'-hydroxylated terminus and the 5'-phosphorylated terminus resulting in a bond between the 3' and 5' termini. Where a ligase is used, the bond is a phosphodiester bond. An exemplary structure after ligation is shown in FIG. 1.</p>
    <p>In a related embodiment, the method generally comprises the steps of:</p>
    <p>a) providing in an aqueous ligation buffer an admixture of first and second double-stranded (ds) DNA molecules of preselected nucleotide sequence, each of said ds DNA molecules comprised of two hybridized oligonucleotides that form an overhang, said oligonucleotides having a length of at least 5 nucleotide bases, and the overhangs comprised of at least one nucleotide base, wherein the sequence of said first ds DNA overhang is complementary to the sequence of said second ds DNA overhang, to form a ligation reaction admixture;</p>
    <p>b) hybridizing the overhangs of said first and second ds DNA molecules to form a ligation reaction substrate; and</p>
    <p>c) ligating said ligation reaction substrate to form a ligation reaction product containing said oligonucleotide of preselected nucleotide sequence.</p>
    <p>Preferably the oligonucleotide provided in step (a) is 5 to 10 bases in length and the overhangs are from 1 to 3 bases in length, and more preferably the overhangs are 2 bases long.</p>
    <p>In one embodiment, the invention contemplates the ligation of two duplexes having structures as described above and shown in FIG. 1.</p>
    <p>Although the example shown in FIG. 1 illustrates a 5' overhang on both duplexes A and B, the invention can also be practiced with complementary 3' overhangs.</p>
    <p>Extending the reaction scheme of FIG. 1, the present invention contemplates duplexes having at least one, and preferably 1 to 3, nucleotides in the complementary overhangs, although a two base overhang is preferred. Furthermore, the length of the oligonucleotide in the region of complementarity in a duplex is at least 5 nucleotides, and is preferably 5 to 10 nucleotides, although it is particularly preferred when the complementarity is formed by 5 to 6 nucleotides because of the resulting library size, as discussed further herein. Thus, the invention can utilize duplexes having one blunt end comprised of one the following pairs of oligonucleotide structures: duplexes having a one nucleotide overhang: 5mer/6mer, 6mer/7mer or 7mer/8mer; having a two nucleotide overhang: 5mer/7mer, 6mer/8mer or 7mer/9mer; or having a three nucleotide overhang: 5mer/8mer, 6mer/9mer or 7mer/10mer. Particularly preferred duplexes of this type are 5mer/7mer or 6mer/8mer duplexes.</p>
    <p>In another embodiment, the duplexes can each have two overhangs as shown by 8mer/8mer duplexes illustrated in Schemes I and II. Thus, there is no requirement that one terminus of a duplex be blunt-ended as illustrated for the outside termini in the example shown in FIG. 1, so long as complementary overhangs are provided for ligation of two duplexes. A preferred duplex is the 8mer/8mer duplex having a 2 nucleotide overhang at each termini.</p>
    <p>Insofar as hybridization occurs rapidly, the providing step (a) and the hybridizing step (b) can be performed simultaneously.</p>
    <p>Ligation can be effected by any means that results in the formation of a bond between adjacent 3' hydroxyl and 5' hydroxy groups (or analogs) of adjacent oligonucleotides. These ligation means can include chemical or enzymatic methods. Particularly preferred enzymatic means are conducted by the use of bacteriophage T4 DNA ligase, as exemplified herein, which results in the formation of a phosphodiester bond.</p>
    <p>In preferred embodiments, the overhang is a 5' overhang. In a particularly preferred embodiment, the 5' overhang of one duplex is phosphorylated.</p>
    <p>Typically, the provided ds DNA molecules are selected from a library of ds DNA molecules as described herein.</p>
    <p>In addition, rather than admixing duplex DNA molecules, the present method for producing an oligonucleotide of preselected nucleotide sequence can be practiced by providing oligonucleotides according to this invention having preselected nucleotide sequences as to form the above-defined duplexes having complementary termini.</p>
    <p>Thus, the oligonucleotides forming the oligonucleotide components of the upstream and downstream duplexes can be added together to form a ligation reaction admixture in the form of single-stranded oligonucleotides or in the form of prehybridized duplexes.</p>
    <p>In this single-stranded oligonucleotide embodiment, and using the 8mer/8mer duplex as exemplary, four 8mer oligonucleotides are added together to form a ligation reaction admixture. This embodiment is described in Example 5 herein. The admixed oligonucleotides first anneal to their respective complementary oligonucleotides to form the two (upstream and downstream) duplexes, such as the duplexes shown in the first step of Scheme I, and subsequently are ligated by DNA ligase to form a ligation product as described herein.</p>
    <p>Similarly, two 6mer oligonucleotides and two 8mers oligonucleotides can be admixed to form two 6mer/8mer duplexes with complementary dinucleotide overhangs as seen for the duplexes in FIG. 1 .</p>
    <p>Using the above outlined approaches for producing an oligonucleotide of predetermined nucleotide sequence, the present invention contemplates providing one or more libraries of oligonucleotides designed to allow the user to select specific oligonucleotides from the library to build a longer oligonucleotide of preselected sequence.</p>
    <p>Thus in one embodiment, first and second oligonucleotides are provided that can hybridize to form a first duplex DNA, and third and fourth oligonucleotides are provided that can hybridize to form a second duplex DNA. The sequence of the four provided oligonucleotides are preselected as to form the first and second duplex DNA molecules such that they have complementary overhangs.</p>
    <p>In one embodiment, a means for directing the hybridization of complementary overhangs is contemplated to selectively direct the hybridization of the desired overhangs. To that end, it is preferred to use combinations of nucleotide sequences in the two complementary overhangs that prevents self hybridization. This is accomplished by using sequences in the overhang which are not self-complementary. An example of self complementarity are the sequences AT, TA, GC and CG when present in an overhang. Where two duplex DNA molecules are to be ligated as shown in reaction Scheme I in which a duplex contains two overhangs, it is preferred required that the sequence of the overhangs be selected such that only one of the overhangs is complementary with the target overhang to which it is to be ligated, so as to minimize unwanted ligations.</p>
    <p>In addition, hybridization direction can be affected by the kinetics of the reaction between complementary termini. For example, the use of molar excesses of one species, will favor hybridization between species rather than self hybridization of the minority species.</p>
    <p>A preferred method comprises (1) selecting a first oligonucleotide from a first library defined by the formula YZ as defined previously, (2) phosphorylating the selected oligonucleotide, (3) admixing the phosphorylated oligonucleotide with a second and third oligonucleotide selected from a second library defined by the formula Z and a fourth oligonucleotide selected from the first library. The four oligonucleotides have preselected nucleotide sequences as to hybridize and form a ligation reaction substrate.</p>
    <p>Alternatively, first, second and third libraries are provided: the first and second libraries having an oligonucleotide defined by the formula YZ as before, where oligonucleotides from the second library contain 5' phosphorylated termini, and the third library having an oligonucleotide defined by the formula Z. In this alternative, no phosphorylation step is required.</p>
    <p>Although exemplary and preferred, the invention is not to be limited to the ligation of two duplexes (A and B; derived from Libraries A and B, respectively) having complementary overhangs as shown in FIG. 1. Also contemplated is the ligation of three or more duplexes, for example, duplexes A, B and C, each having overhangs adapted to direct the orderly assembly by hybridization and ligation of the three duplexes to form a ligated oligonucleotide. In this case, duplex B has a first and second overhang, duplex A comprises an overhang complementary to the first overhang of duplex B, and duplex C comprises an overhang complementary to the second overhang of duplex B. Furthermore, the overhang of C is not complementary to either first overhang of duplex B or to the overhang of duplex A, thereby minimizing unwanted hybridizations and ligations. A similar rationale is applied to the ligation of 4 or more duplexes according to the present invention.</p>
    <p>In each case, the methods can be practiced by providing oligonucleotides into a hybridization admixture, or by providing pre-assembled duplexes and admixing the duplexes. Preferably, the methods are practiced by selecting oligonucleotides or duplexes from a library of this invention.</p>
    <p>Thus the present invention contemplates the use of two distinct types of libraries: duplex DNA libraries and oligonucleotide libraries.</p>
    <p>In another embodiment, the provided oligonucleotides having a 5' phosphate contains a thiol-phosphodiester linkage between the penultimate nucleotide and the 3' terminal nucleotide. The thiol linkage reduces the resulting ligated oligonucleotide's susceptibility to exonucleolytic degradation, increasing the lifetime of the oligonucleotide in primer extension reactions, or other applications where an exonuclease may be present.</p>
    <p>In another embodiment the provided oligonucleotides are obtained by using degenerate oligonucleotide compositions in which the selected oligonucleotides for ligation are present as species in the degenerate oligonucleotide compositions. The compositions to be admixed for hybridization are selected to favor a single specific complement together with the other non-complementary oligonucleotides such that unwanted duplexes are selected against, thereby enriching for the production of a specific ligation reaction product. A preferred method uses the degenerate octanucleotide composition defined herein by the formula 5'-XXNXXNXX-3'.</p>
    <p>In one embodiment, the method for producing an oligonucleotide of preselected nucleotide sequence comprises the steps of:</p>
    <p>a) admixing first, second, third and fourth octanucleotide compositions to form a hybridization admixture, wherein each octanucleotide composition comprises octanucleotides according to the formula: 5'-XXNXXNXX-3' as defined earlier;</p>
    <p>the first and second octanucleotide compositions are selected to contain sequences such that complementary hybridization of oligonucleotides of the first and second compositions forms a first double-stranded (ds) DNA molecule of the formula:</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="equation">5'-P-XXNXXNXX-3'</pre>
    
    <p>
      </p> <pre xml:space="preserve" listing-type="equation">3'-XXNXXNXX-5',</pre>
    
    <p>the third and fourth octanucleotide compositions are selected to contain sequences such that complementary hybridization of oligonucleotides of the third and fourth compositions forms a second double-stranded (ds) DNA molecule of the formula:</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="equation">5'-XXNXXNXX-3'</pre>
    
    <p>
      </p> <pre xml:space="preserve" listing-type="equation">3'-XXNXXNXX-5',</pre>
    
    <p>and wherein the first, second, third and fourth octanucleotide compositions are selected such that the first ds DNA molecule has a 5' overhang that is complementary to only one of the 5' overhangs present on the second ds DNA molecule;</p>
    <p>b) hybridizing the octanucleotide compositions admixed in step (a) in an aqueous ligation buffer such that the complementary overhangs hybridize to form a ligation reaction substrate; and</p>
    <p>c) ligating the ligation reaction substrate to form a ligation reaction product containing the oligonucleotide of preselected nucleotide sequence.</p>
    <p>In preferred embodiments using degenerate oligonucleotide compositions, the 5' terminus to be ligated on the overhang of the second ds DNA molecule is phosphorylated. The 5' terminal phosphate facilitates enzymatic ligation when using DNA ligase, and assures that the oligonucleotides of the complementary strand are not ligated (e.g., oligonucleotides A and C are ligated in Scheme II, whereas oligonucleotides B and D are not ligated).</p>
    <p>The above method using degenerate oligonucleotide compositions need not be limited to using compositions containing the above-specified octanucleotide composition, as it is apparent to one skilled in the art that octanucleotide compositions based on a formula having the degenerate nucleotides "N" in positions other than N<sub>3</sub>,N<sub>6</sub> - may be utilized as described earlier. However, N<sub>3</sub>,N<sub>6</sub> - is particularly advantageous when using the 8mer/8mer format because all the "N's" are positioned across from "X's" such that complementary hybridization selects the appropriate hybridization partner in the duplex out of 16 possible different sequences.</p>
    <p>A library of 4,096 9mer degenerate oligonucleotide compositions (members) could be formed using oligonucleotide compositions according to the compositional formula: 5'-XXNXXNNXX-3'. Such 9mers can be used to produce 18mers as follows: ##STR9##</p>
    <p>Alternatively, a library of 1024 7mers can be formed with the sequence 5'-XXNNXXX-3. Such 7mers can be used to produce 14mers as follows: ##STR10##</p>
    <p>The particular advantage to using degenerate octanucleotides having two degenerate "N" nucleotides is that such a composition provides a collection of 16 unique species of octanucleotide, and because a library of this type of octanucleotide can contain every possible octanucleotide sequence by having 4,096 different compositions (families), each containing 16 different octanucleotides.</p>
    <p>A library of 4,096 different compositions provides a manageable library size while at the same time provides materials sufficient to construct de novo, without chemical synthesis, a large diversity of oligonucleotides having a predetermined sequence OF LENGTH 16, 24, etc.</p>
    <p>As a means for assisting the library user in the task of accessing the correct families of compositions for combination to construct by ligation a larger oligonucleotide, it is noted that the sequence of each oligonucleotide in a composition is known, and those sequence data can be stored, managed and sorted by automated means to provide rapid selection of appropriate compositions for building a preselected sequence.</p>
    <p>Thus, a computerized computation system can be used to rapidly and easily identify appropriate compositions for admixture and ligation according to the present methods upon identifying the preselected oligonucleotide to be constructed by ligation.</p>
    <p>Computers are particularly well suited for such automation, and can be used to support automated robotic systems for (1) maintaining a library, (2) selecting desired compositions upon instruction of a oligonucleotide primer to construct, and (3) admixing the biochemical reagents together with selected oligonucleotide compositions and carrying out the recited hybridization and ligation steps.</p>
    <p>Hybridization of Oligonucleotides to Form a Ligation Reaction Substrate</p>
    <p>A hybridization reaction mixture is typically prepared by admixing effective amounts of one or more oligonucleotide compositions of the present invention, and other components compatible with a hybridization reaction. These oligonucleotide compositions can contain two as DNA molecules having complementary overhangs, can contain three ds DNA molecules having complementary overhangs, or can contain degenerate or non-degenerate oligonucleotide compositions having sequences that, upon hybridization, form the requisite ds DNA molecules for ligation as described herein.</p>
    <p>The hybridization reaction mixture is maintained in the contemplated method under hybridizing conditions for a time period sufficient for the oligonucleotides having complementarity to the predetermined sequence on corresponding complementary oligonucleotides to hybridize to those complementary nucleic acid sequences and form a hybridization product, i.e., first a ds DNA duplex, and then a ds DNA product having hybridized overhangs as shown in FIG. 1, FIG. 2, Scheme I or Scheme II.</p>
    <p>The phrase "hybridizing conditions" when used with a maintenance time period, indicates subjecting the hybridization reaction admixture, in the context of the concentrations of reactants and accompanying reagents in the admixture, to time, temperature and pH conditions sufficient to allow one or more oligonucleotides to anneal with the complementary sequences present in the admixture, to form a nucleic acid duplex. Such time, temperature and pH conditions required to accomplish hybridization depend, as is well known in the art, on the length of the oligonucleotide to be hybridized, the degree of complementarity between the oligonucleotides (i.e., the length of complementary nucleotides in the sequence), the guanidine and cytosine content of the oligonucleotide, the stringency of hybridization desired, and the presence of salts or additional reagents in the hybridization reaction admixture as may affect the kinetics of hybridization. Methods for optimizing hybridization conditions for a given hybridization reaction admixture are well known in the art.</p>
    <p>Typical hybridizing conditions include the use of solutions buffered to pH values between 4 and 9, and are typically carried out at temperatures from 0 degrees C. (0° C.) to 37° C., preferably about 2° C. to about 30° C., more preferably about 4° C. and for time periods from 0.5 seconds to 24 hours, preferably 2 minutes (min) to 1 hour. Exemplary are the conditions described in Examples 2 and 5.</p>
    <p>Effective amounts of the oligonucleotides to be ligated that are present in the hybridization reaction admixture are generally well known and are typically expressed in terms of molar ratios between the oligonucleotides or duplexes to be hybridized. Preferred ratios are hybridization reaction mixtures containing from equimolar to ten-fold excess amounts of non-degenerate species to be hybridized. As is well known, deviations from equal molarity will produce hybridization reaction products. Thus although ratios where one of the two components (oligonucleotide A and oligonucleotide B or duplex A and duplex B as shown in FIG. 1) can be in as much as 100 fold molar excess relative to the other component, excesses of less than 50 fold, preferably less than 10 fold, and more preferably less the 2 fold are desirable in practicing the invention, except when degeneracies in one of the species to be hybridized necessitate the use of unequal molar ratios.</p>
    <p>Effective concentrations of oligonucleotides in a hybridization reaction admixture are typically in the range of about 0.5 to 50 micromolar (uM), preferably about 1 to 10 uM, and more preferably about 5 uM.</p>
    <p>In typical priming applications when using an oligonucleotide of this invention, the template is double-stranded, and therefore both the sense and anti-sense strands can potentially serve as template for a primer. Duplicate priming in a single sequencing reaction is undesirable because it produces unreadable sequencing ladders, and could potentially occur if certain oligonucleotides of the duplexes to be ligated (that are complementary to the desired oligonucleotide primer) are available for priming reactions on the template strand opposite to the strand to be sequenced.</p>
    <p>For example, according to the scheme of FIG. 2, three products are formed in the ligation reaction: a 14mer, an 8mer and a 6mer. The 14mer is the desired primer. The 8mer could potentially act as a primer to the template strand complementary to the strand that the 14mer will prime in low temperature primer extension reactions, thus providing the unwanted duplicate priming. The 6mer could also theoretically prime, but is generally too short to initiate significant primer extension reactions. Thus it is desirable to block the capacity of the 8mer in this example to prime.</p>
    <p>Thus, in one embodiment, it is preferred that the sequence of the oligonucleotide that provides the overhang and that is complementary to the final ligated oligonucleotide primer (e.g., the 8mer from duplex A derived from Library A in FIG. 2) has a 3' terminal nucleotide which is non-complementary to the sequence of its complementary oligonucleotide (e.g., the 6mer from duplex A derived from Library A in FIG. 2). In this situation, the 8mer from duplex A, which could otherwise provide a 3' terminus to initiate primer extension has a mismatch for the template to be extended at its 3' terminus, and thus cannot prime the template efficiently. Thus, the mismatch inhibits the 8mer from participating in a priming event.</p>
    <p>With the design of a 3' mismatch as above, the ligation reaction product can be used directly in primer extension reactions where the 8mer from duplex A might prime without the need to purify the ligated oligonucleotide from the ligation reaction admixture.</p>
    <p>Where the complementarity in the overhangs is selected to also be self-complementary, the use of unequal molar ratios can be utilized in one embodiment to favor hybridization and subsequent ligation of one pair over another. For example, referring to FIG. 2 that shows the hybridization of ds DNA molecules from library A and B, note that the sequence of the overhangs shown is self-complementary. To deter self-ligation of the ds DNA molecules from library B, a ten fold excess of library A molecules will favor A-B combinations over B-B combinations. A-A combinations are prevented due to the absence of a 5'phosphate on the terminus of the ds DNA molecule. Thus, the combination of molar ratio imbalances and phosphorylation of only one of the two species allows an effective and reproducible method for producing oligonucleotides according to the present methods. This embodiment is also described in the Examples.</p>
    <p>Ligation of a Ligation Reaction Substrate to Form Oligonucleotide Primers of Preselected Sequence</p>
    <p>In producing a ligated oligonucleotide primer by the methods of this invention, the ligation reaction substrate is treated to ligation reaction conditions for a time period sufficient to form a phosphodiester bond between adjacent ligatable ends, namely between the 3' hydroxyl group at the 3'-terminus of oligonucleotide A and the 5' phosphate group at the 5'-terminus of oligonucleotide B. The product of an exemplary reaction is shown in FIG. 1, and in Schemes I and II.</p>
    <p>The ligation step can be performed by any means available for forming a phosphodiester between the adjacent 5' and 3' termini, including enzymatic and chemical synthesis means. Preferred is the use of the enzyme ligase for catalytically inducing the ligation reaction.</p>
    <p>Ligation reaction conditions are generally well known in the art and depend, in part, on the ligase to be used for forming the phosphodiester bond, and on the stability of the ligation reaction substrate.</p>
    <p>A preferred ligase is bacteriophage T4 DNA ligase, such as is obtained from recombinant Escherichia coli, which can be obtained from a variety of commercial vendors.</p>
    <p>Stability of the ligation reaction substrate is maintained by preserving hybridization reaction conditions during the manipulations after hybridization and during the ligation reaction. The substrate can vary in stability depending on the length of the hybridized oligonucleotide. For the shorter oligonucleotides contemplated by this invention, for example the hexameric oligonucleotides, it is preferred that hybridization and ligation reaction conditions be conducted below 30 degrees Centigrade (30° C.) and preferably between 4° C. and 22° C.</p>
    <p>Ligation and hybridization can be accomplished in a single reaction step, and is preferred for convenience.</p>
    <p>Ligation reaction conditions for ligation of the duplex DNA molecules typically require 0.5 to 100 uM of each primer, preferably about 10 to 50 uM, and more preferably about 20 uM primer which corresponds to about 500 nanograms (ng) of each oligonucleotide in a ligation reaction volume of 10 ul.</p>
    <p>In one embodiment where the objective is to favor a preselected orientation of the ligated fragments rather than self-ligation, it is preferred to use about 5 to 10 fold molar excesses of the upstream (non-phosphorylated) duplex DNA molecule relative to the amount of the downstream duplex DNA molecule. Such a molar ratio reduces self-ligation of the downstream duplex DNA molecules. Other orientation-preference modifications can be utilized to further control the ligation reaction.</p>
    <p>The ligation reaction conditions further require about 5 to 500 uM, preferably about 100 uM, rATP and 1x ligase buffer described herein. The ligation reaction admixture may also contains about 0 to 25 percent, preferably 7 to 12, more preferably 10 percent by weight, polyethylene glycol (PEG) 8000 because the PEG increases the rate of the ligation reaction. Other ligase buffers can be utilized, as is well known. The important factors in the present ligation method are the concentrations of the rATP and the input oligonucleotides, as described herein.</p>
    <p>Ligase used in the present method is preferably T4 DNA ligase at a concentration of about 0.5 to 10, preferably about 1 to 5, Weiss units per 10 ul ligation reaction volume.</p>
    <p>Hybridization and ligation reaction temperatures depend, as is known on the GC content of the oligonucleotide to be hybridized. For example, a hexanucleotide duplex comprised of all AT pairs require lower temperatures, typically below 22 degrees Centigrade (C.) and while a hexanucleotide of all GC pairs will hybridize and ligate up to about 37 degrees C. Thus the preferred range, depending on nucleotide content is about 4 to 30 degrees, preferably about 10 to 25 degrees, and more preferably 14 to 22 degrees C. Ligation reaction times can vary from about 0.5 min to 2 hours, although typically 5 min to 1 hour, and more preferably 5 to 15 minutes, are utilized.</p>
    <p>Thereafter, the ligation reaction product formed is recovered, thereby producing the oligonucleotide of preselected nucleotide sequence. Typically, recovery comprises simply collecting the ligation reaction product and using it directly without further manipulations. Alternatively, the ligated oligonucleotide can be separated from the non-ligated precursor oligonucleotides, e.g., the 6mer and 8mer in the ligation product shown in FIG. 2. Separation techniques can include size separation or affinity isolation based on the presence of a terminal group such a biotin and the like and described further herein.</p>
    <p>The resulting ligated oligonucleotide primer (ligation reaction product) formed by the action of the ligation reaction conditions on the ligation reaction substrate can be isolated from the other oligonucleotides in the ligation admixture or can be used directly as described further herein.</p>
    <p>In preferred embodiments, the use of terminus modifications are contemplated that will control the ligation reaction and prevent unwanted ligations. An exemplary and preferred terminus modification is to phosphorylate the 5' terminus of the oligonucleotide to be ligated, but not phosphorylate the 5' terminus of the oligonucleotide in the complementary strand that participates in the overhang hybridization. See FIG. 1 for this example. The strategy provides at least two advantages.</p>
    <p>First, the ligation reaction in this preferred strategy includes a 5'phosphate, and therefore, only the strand of choice is ligated, and not the oligonucleotides in the complementary strand. The result is that the non-ligated oligonucleotides are short relative to the ligated oligonucleotide. The short non-ligated oligonucleotides are generally too short to prime a PCR reaction or prime a DNA sequencing reaction. Therefore, it is not required that the non-ligated oligonucleotides be removed from the desired ligated oligonucleotide product prior to its use in PCR or sequencing reactions.</p>
    <p>Second, insofar as 5' phosphate is required for enzymatic ligation by T4 DNA ligase, non-phosphorylated 5' termini cannot participate in self-ligations to form improper concatamers. Using FIG. 1 as an example, the ds DNA molecule from library A cannot form A-A ligations because the A molecule is not phosphorylated.</p>
    <p>The optional separation of the oligonucleotide primer product from the shorter non-ligated oligonucleotides in a ligation reaction admixture can be accomplished by a variety of means following denaturation which destabilizes the hybridized duplexes. Exemplary separation methods include size fractionation of the oligonucleotides after denaturation of the ligation reaction product on gel sieve chromatography, on polyacrylamide gels and the like sizing methods.</p>
    <p>Recovery of one strand from the duplex containing a ligation reaction product can be accomplished by a variety of means if desired, although selective recovery of the ligated oligonucleotide is not required for either PCR, sequencing or cycle-sequencing as shown in the Examples. Modification of either of the oligonucleotides before ligation to introduce a terminal group which affords selective retrieval of the desired oligonucleotide primer product is a preferred means for retrieval.</p>
    <p>The biotin-avidin affinity system used for detection of non-isotopically labelled nucleotides is readily adapted to affinity based retrieval systems. In that system, biotin is introduced onto the 3' terminus of the downstream oligonucleotide or onto the 5' terminus of the upstream oligonucleotide which become incorporated into the ligated oligonucleotide product (e.g., the 3' terminus of the 8mer in library B or the 5' terminus of the 6mer in library A of FIG. 2). Thereafter, the presence of the biotin "tag" provides the means to selectively retrieve by elution the desired ligation reaction product.</p>
    <p>The recovering step comprises the steps of admixing ligation reaction product having a biotin modified terminus with a suspension containing a solid phase comprising a solid support having avidin or streptavidin affixed thereto, to form an avidin binding admixture containing a liquid phase and a solid phase. Solid supports are generally well known, as are methods for fixing protein, such as avidin or streptavidin, to the solid support. Avidin or Streptavidin is available from a variety of commercial vendors.</p>
    <p>The solid support can be in a variety of formats designed for easy recovery of the support away from the liquid phase to facilitate washing and eluting steps. These can take the form of beads, that can be physically separated based on size, by filtration or sedimentation. Alternatively, the solid support can have a functional property that makes separation simple, such as magnetism, or a biological binding affinity. Particularly preferred are magnetic beads that can be removed by magnetic fields.</p>
    <p>The avidin binding admixture is then maintained under conditions compatible with a binding reaction between avidin and biotin for a time period sufficient for said avidin to bind to said biotin and form a biotin-avidin complex in the solid phase. The binding conditions are very flexible as the binding affinity between biotin and avidin is very high, and typically are aqueous solutions which do not denature DNA hybrids (duplex DNA). Time periods for binding are extremely fast, typically under one hour, and as fast as 1 to 10 minutes at room temperature.</p>
    <p>The ligation reaction product is then eluted away from the solid phase to form the isolated oligonucleotide of preselected nucleotide sequence. Elution can be accomplished by any condition which disrupts (denatures) DNA duplexes to form single stranded oligonucleotides. Exemplary is the use of high temperature, e.g., greater than 65 degrees in 1x ligation buffer, or the like denaturing conditions, such as 0.1 to 0.5M NaOH.</p>
    <p>Use in Directed Sequencing</p>
    <p>Directed sequencing (primer walking) is a multi-step process in which a large sequence of nucleotides is determined by the steps of: (1) determining a first region of nucleotide sequence, (2) 5 preparing a sequencing primer based on the downstream 3' region of the determined sequence to design the sequencing primer as to be complementary to the template at that downstream region, (3) determining a second region of nucleotide sequence using the sequencing primer designed from the previously determined sequence in a primer extension-based sequencing procedure (e.g., dideoxy sequencing), and (4) repeating steps (2) and (3) for as many cycles as needed to walk down the entire sequence to be determined. This approach is termed directed sequencing because the choice of primer directs the subsequent sequencing steps and thereby sequentially orders the sequence information obtained. Directed sequencing is typically compared to and preferred over random sequencing methods where the sequence information obtained is not directed in any particular order.</p>
    <p>In preferred embodiments for practicing the present methods as applied to directed sequencing, the use of the oligonucleotide compositions provide a particular advantage over previous directed sequencing methods.</p>
    <p>For example, using previous techniques, after a region of nucleic acid sequence was determined, a new sequencing primer would be required to complete the next "directed" sequencing step. That required primer must be chemically synthesized, which consumes time and the expense of custom oligonucleotide synthesis.</p>
    <p>By the present invention, the required sequencing primer can be constructed from a pre-existing "library" of oligonucleotide or duplex DNA compositions according to this invention by:</p>
    <p>(1) selecting a nucleotide sequence in the region of the template for designing a directed sequencing primer,</p>
    <p>(2) selecting four oligonucleotide compositions (degenerate or non-degenerate) or two duplex DNA compositions from one or more libraries of the present invention, as needed, to produce an oligonucleotide having complementarity to the region of the template selected for directed sequencing, and</p>
    <p>(3) following the methods herein for admixing and ligating the selected compositions to form the oligonucleotide primers of preselected sequence. In the process of forming the ligated oligonucleotide primer, the resulting ligation reaction product is ready for sequencing without further manipulation. Exemplary is the ligation of non-degenerate oligonucleotides to form a 14mer oligonucleotide primer and sequencing of ssm13mp18 described in Example 2 and shown in FIG. 4. A further example is the ligation of degenerate oligonucleotides to form a 16mer primer and sequencing of template as described in Examples 6, 7 and 8.</p>
    <p>The methods for producing ligated primers can be applied to a variety of methods for manipulating and analyzing nucleic acid molecules, as will be apparent to one skilled in the art.</p>
    <p>For example, a ligation reaction product can be used in primer extension reactions to produce primer extension reaction products. After producing the ligation reaction product, the resulting oligonucleotide is used in a primer extension reaction to form a primer extension reaction product containing the ligated primer. By providing excess amounts, relative to template, of the oligonucleotide to a hybridization reaction one can cycle through successive rounds comprising (1) hybridization of the oligonucleotide primer, (2) primer extension, and (3) denaturation to remove the primer extension product. By doing so the excess oligonucleotides will repeatedly hybridize to the template and extend to cyclically produce primer extension product. This process is referred to as cycle-extending because repeated primer extension product is formed by cycling through the above steps.</p>
    <p>In cycle-extending, the denaturation step is typically a heat treatment manipulation to melt the duplex DNA. Such heat treatment necessitates that the polymerase used in the primer extension step be heat stable, or that additional polymerase be added to each primer extension reaction admixture at each cycle. The primer extension step in cycle-extending is preferably conducted with a heat stable polymerase as described herein for the polymerase chain reaction (PCR) methods.</p>
    <p>In a related embodiment, chain terminators such as are used in dideoxy sequencing reaction can be used in the primer extension step of the above cycle-extending method. This allows the repeated production of sequencing reaction products in a cycle-sequencing method. By including the reagents normally used in a dideoxy sequencing reaction at the primer extension step, one can produce amounts of sequencing reaction product in excess of the amount normally provided after one dideoxy sequencing reaction, thereby increasing the sensitivity of the normal sequencing reaction. An exemplary cycle-sequencing procedure is described in Example 4.</p>
    <p>Use in PCR Reactions</p>
    <p>Polymerase chain reactions (PCR) utilize primer extension primers in a pairwise array as is well known. The PCR reaction, however, consumes mass quantities of the primers as each primer becomes incorporated in the primer extension product at each PCR cycle. Therefore, the present oligonucleotide libraries and methods are particularly well suited to solving the problem of PCR primer preparation insofar as the PCR primers can be synthesized by ligation as described herein from the pre-existing libraries of this invention rather than chemically synthesized de novo.</p>
    <p>For example, to conduct a PCR reaction on a DNA sequence, one selects the desired PCR primer pair, and determines for each primer, the 3' primer and the 5' primer, which oligonucleotides of preselected sequence to produce, using the present methods. Thereafter, one admixes the prepared oligonucleotide compositions with a target for PCR amplification to form a PCR reaction admixture, ready for the PCR reaction.</p>
    <p>Other permutations on PCR reaction methodologies will readily be apparent to one skilled in the art.</p>
    <p>PCR amplification methods are described in detail in U.S. Pat. Nos. 4,683,192, 4,683,202, 4,800,159, and 4,965,188, and at least in several texts including "PCR Technology: Principles and Applications for DNA Amplification", H. Erlich, ed., Stockton Press, New York (1989); and "PCR Protocols: A Guide to Methods and Applications", Innis et al., eds., Academic Press, San Diego, Calif. (1990).</p>
    <p>The PCR reaction is performed by mixing the PCR primer pair, preferably a predetermined amount thereof, with the template nucleic acid having the sequence to be amplified, preferably a predetermined amount thereof, in a PCR buffer to form a PCR reaction admixture. The admixture is maintained under polynucleotide synthesizing conditions for a time period, which is typically predetermined, sufficient for the formation of a PCR reaction product, thereby producing an amplified PCR reaction product.</p>
    <p>The PCR reaction is performed using any suitable method. Generally it occurs in a buffered aqueous solution, i.e., a PCR buffer, preferably at a pH of 7-9, most preferably about 8. Preferably, a molar excess (for genomic nucleic acid, usually about 10<sup>6</sup> :1 primer:template) of the primer is admixed to the buffer containing the template strand. A large molar excess is preferred to improve the efficiency of the process.</p>
    <p>The PCR buffer also contains the deoxyribonucleotide triphosphates DATP, dCTP, dGTP, and dTTP and a polymerase, typically thermostable, all in adequate amounts for primer extension (polynucleotide synthesis) reaction. The resulting solution (PCR admixture) is heated to about 90° C.-100° C. for about 1 to 10 minutes, preferably from 1 to 5 minutes. After this heating period the solution is allowed to cool to 35° to 60° C., and preferably 40° to 50° C. depending upon the actual base composition as is known, which is preferable for primer hybridization. The synthesis reaction may occur at from room temperature up to a temperature above which the polymerase (inducing agent) no longer functions efficiently. Thus, for example, if DNA polymerase is used as inducing agent, the temperature is generally no greater than about 40° C. An exemplary PCR buffer comprises the following: 50 mM KCl; 10 mM Tris-HCl; pH 8.3; 1.5 mM MgCl<sub>2</sub> ; 0.001% (wt/vol) gelatin, 200 μM dATP; 200 μM dTTP; 200 μM dCTP; 200 μM dGTP; and 2.5 units Thermus aquaticus DNA polymerase I (U.S. Pat. No. 4,889,818) per 100 microliters of buffer. Other exemplary PCR reactions are described in Example 3.</p>
    <p>The inducing agent may be any compound or system which will function to accomplish the synthesis of primer extension products, including enzymes. Suitable enzymes for this purpose include, for example, E. coli DNA polymerase I, Klenow fragment of E. coli DNA polymerase I, T4 DNA polymerase, other available DNA polymerases, reverse transcriptase, and other enzymes, including heat-stable enzymes, which will facilitate combination of the nucleotides in the proper manner to form the primer extension products which are complementary to each nucleic acid strand. Generally, the synthesis will be initiated at the 3' end of each primer and proceed in the direction of 5' to 3' along the template strand, until synthesis terminates, producing molecules of different lengths. There may be inducing agents, however, which initiate synthesis at the 5' end and proceed in the above direction, using the same process as described above.</p>
    <p>The inducing agent also may be a compound or system which will function to accomplish the synthesis of RNA primer extension products, including enzymes. In preferred embodiments, the inducing agent may be a DNA-dependent RNA polymerase such as T7 RNA polymerase, T3 RNA polymerase or SP6 RNA polymerase. These polymerases produce a complementary RNA polynucleotide. The high turn over rate of the RNA polymerase amplifies the starting polynucleotide as has been described by Chamberlin et al., The Enzymes, ed. P. Boyer, PP. 87-108, Academic Press, New York (1982). Another advantage of T7 RNA polymerase is that mutations can be introduced into the polynucleotide synthesis by replacing a portion of cDNA with one or more mutagenic oligodeoxynucleotides (polynucleotides) and transcribing the partially-mismatched template directly as has been previously described by Joyce et al., Nucleic Acid Research, 17:711-722 (1989). Amplification systems based on transcription have been described by Gingeras et al., in PCR Protocols, A Guide to Methods and Applications, pp 245-252, Academic Press, Inc., San Diego, Calif. (1990).</p>
    <p>If the inducing agent is a DNA-dependent RNA polymerase and therefore incorporates ribonucleotide triphosphates, sufficient amounts of ATP, CTP, GTP and UTP are admixed to the primer extension reaction admixture and the resulting solution is treated as described above.</p>
    <p>PCR is typically carried out by thermocycling i.e., repeatedly increasing and decreasing the temperature of a PCR reaction admixture within a temperature range whose lower limit is about 10° C. to about 40° C. and whose upper limit is about 90° C. to about 100° C. The increasing and decreasing can be continuous, but is preferably phasic with time periods of relative temperature stability at each of temperatures favoring polynucleotide synthesis, denaturation and hybridization.</p>
    <p>D. Oligonucleotide Library Kits</p>
    <p>Many of the reagents described herein (e.g., nucleic acids such as the oligonucleotides and duplex DNA molecules in a library of this invention) have a number of forms, particularly variably protonated forms. As the skilled practitioner will understand, representation herein of one form of a compound or reagent is intended to include all forms thereof.</p>
    <p>The reagents described herein can be packaged in kit form. As used herein, the term "package" refers to a solid matrix or material customarily utilized in a system and capable of holding within fixed limits one or more of the reagent components for use in a method of the present invention. Such materials include glass and plastic (e.g., polyethylene, polypropylene and polycarbonate) bottles, vials, paper, plastic and plastic-foil laminated envelopes and the like. Thus, for example, a package can be a glass vial used to contain the appropriate quantities of oligonucleotide compositions, restriction enzyme(s), DNA polymerase, polynucleotide ligase, or a combination thereof. An aliquot of each component sufficient to perform at least one ligation reaction will be provided in each container.</p>
    <p>A particularly preferred kit contains a library of the present invention together with a polynucleotide ligase, such as DNA ligase.</p>
    <p>Kits useful for producing a preselected primer for sequencing of a specific nucleic acid sequence or for conducting a PCR amplification reaction using a primer extension reaction methodology also typically include, in separate containers within the kit, dNTPs where N is adenine, thymine, guanine and cytosine, and other like agents for performing sequencing reactions.</p>
    <p>The reagent species of any system described herein can be provided in solution, as a liquid dispersion or as a substantially dry powder, e.g., the oligonucleotides may be provided in lyophilized form.</p>
    <p>In one embodiment, the present invention contemplates a kit for producing an oligonucleotide of preselected priming specificity, which kit comprises a plurality of separate packages (containers) within an enclosure, each package containing a different oligonucleotide or duplex DNA molecules composition according to this invention.</p>
    <p>Kits having a plurality of such compositions are also referred to herein as libraries.</p>
    <p>In one embodiment a preferred library contains up to 4096 separate containers, each containing a different 8mer as defined herein, where each 8mer in the library has the same dinucleotide sequence at its 5' terminus. In a related embodiment a library contains up to 4096 separate containers, each containing a different duplex DNA molecule comprised a 6mer/8mer oligos, where each 8mer has the same dinucleotide sequence at its 5' terminus. Exemplary is the library shown in FIG. 4.</p>
    <p>Alternatively, a preferred library contains 4,096 containers each containing a different degenerate octanucleotide composition as defined herein. Particularly preferred is a library where each degenerate octanucleotide composition has octanucleotides according to the formula 5'-XXNXXNXX-3'as described herein.</p>
    <p>In another embodiment, a kit comprises two libraries, a first having a shorter oligonucleotide and a second having a longer oligonucleotide according to the formula YZ as described before, where the shorter oligonucleotide is complementary to the sequence Z. Exemplary is a kit having a first library of 6mers, and a second library of 8mers, as described herein. optimally, the 8mer library is phosphorylated at the 5' termini.</p>
    <p>The oligonucleotides of the first and second libraries have sequences such that the complementary hybridization of a member of the first library with a member of the second library forms a double-stranded DNA molecule having at least a one nucleotide base overhang of the second library oligonucleotide sequence, and such that the terminal nucleotide(s) of all oligonucleotides in the second library that form the overhang have the same nucleotide sequence. Preferably, the overhang formed by complementary hybridization is a 5' terminal overhang, and more preferably the overhang is a dinucleotide sequence selected from the group consisting of GG, CC, TT, AA, TC, CT, GA, AG, TG, GT, AC, CA, NI and IN, wherein N is selected from the group consisting of A, T, G and C, and wherein I is inosine.</p>
    <p>Alternatively, a kit comprises three libraries, namely a 6mer, an 8mer, and an 8mer that is phosphorylated at their 5' termini. This kit provides all the materials to assemble a pair of duplexes A and B without the need for the manipulative step of phosphorylation.</p>
    <p>Preferred kits contain organized enclosures such that the different oligonucleotides are distributed in a preselected array. For example, a 96-well microtiter tray is an enclosure that provides an array of 96 containers (wells). A larger library can be comprised of a series of microtiter trays, such as is shown in FIG. 2. Such organized arrays provides a convenient and manageable way to identify and access the different members of the library, and is amenable to automated processes for oligonucleotide synthesis according to the present methods.</p>
    <heading>EXAMPLES</heading> <p>The following examples are intended to illustrate, but not limit, the present invention.</p>
    <heading>1. Oligonucleotide Synthesis</heading> <p>Most of the oligonucleotides in the Examples were synthesized from the 3' end on an Applied Biosystems Inc. (ABI) DNA synthesizer, model 391 or 392, with ABI reagents at a 0.2 um scale using standard protected nucleotide cyanoethyl phosphoramidite chemistry and deprotection protocols according to the manufacturer's instructions.</p>
    <p>Where there is a oligonucleotide composition having a common position (N) that is degenerate, a mixture of all four phosphoramidites was used to extend the growing nucleotide polymer by the addition of any one of the four bases (A, C, G, or T). All oligonucleotides are written herein in the standard 5' to 3' orientation.</p>
    <p>The oligonucleotides were chemically phosphorylated where appropriate at their 5'-end by is the addition, and subsequent deprotection of, 1-Dimethoxytrityl-2,2'-sulfonyldiethanol-1-CED™ phosphoramidite (ABI). In an alternate procedure, oligonucleotides were phosphorylated by the use of T4 polynucleotide kinase.</p>
    <p>The crude oligonucleotides were purified by PAGE and then desalted and lyophilized by standard procedures.</p>
    <p>Some of the oligonucleotides used in the Examples are described in Table 1:</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">              TABLE 1______________________________________Oligo No.  N-mer    Sequence______________________________________1     6         5'-ACGACG-3'2     8         5'-GCCGTCGT-3'3     8         5'-P-GCCAGTGC-3'<sup>a</sup>4     6         5'-GCACTG-3'5     14        5'-ACGACGGCCAGTGC-3'<sup>b</sup> (SEQ ID no: 1)A'    19        5'-TAAAACGACGACGGCCAGT-3'           (SEQ ID no: 3)B     22        5'-TGATTCCAACGAGGAAAGCACG-3'           (SEQ ID no: 4)______________________________________ <sup>a</sup> Oligonucleotide No. 3 is shown with a phosphate group at its 5' terminus. <sup>b</sup> The sequence of oligonucleotide No. 5 shows the product of ligatio of oligonucleotide Nos. 1 and 3, and is aligned over oligonucleotide A' t illustrate the common sequences.</pre>
    
    <heading>2. Ligation of Oligonucleotides to Form a Ligation Reaction Product</heading> <p>The oligonucleotides Nos. 1, 2, 3 and 4 shown in Table 1 were synthesized, and oligonucleotide No. 3 was chemically phosphorylated on its 5' terminus, as described in Example 1. The oligonucleotides Nos. 1, 2, 3 and 4 so produced were admixed into a ligation reaction admixture containing the following in 1X ligation buffer (50 mM Tris-HCl, pH 7.5, 7 mM MgCl<sub>2</sub>, 1 mM DTT) : 10% PEG 8000, 100 uM rATP, 5 uM of each of oligonucleotides Nos. 1, 2, 3 and 4, and 4 units of T4 DNA ligase. The ligation reaction admixture was then maintained (incubated) at 14° C. for 60 minutes to allow the oligonucleotides to anneal to their complements to form duplexes, for the cohesive overhangs to hybridize, and for the ligation reaction to occur and form a ligation reaction product comprising a 14mer hybridized to oligonucleotide Nos. 2 and 4. The reaction scheme is shown in FIG. 1.</p>
    <p>When analyzed on a 20% denaturing acrylamide gel, greater than 90% ligation efficiency was observed in the presence of ligase, whereas only nonligated oligonucleotides were observed in the absence of ligase. Similar ligation reactions conducted at 14° C. were greater than 90% complete after 5 minutes, and greater that 99% complete after 15 minutes.</p>
    <p>In an alternate procedure, oligonucleotide No. 3 was phosphorylated by the use of T4 DNA kinase.</p>
    <heading>3. Ligation Reaction Product as a Polymerase Reaction Primer</heading> <p>The 14mer oligonucleotide ligation reaction product produced in Example 2 was used directly, without purification from the ligation reaction admixture, as a polymerase chain reaction (PCR) primer. To that end, a PCR admixture was prepared by admixing the following in 1X PCR buffer (10 mM Tris-HCl, pH 8.3, 50 mM KCl, 4 mM MgCl<sub>2</sub>, 0.1% gelatin): 250 uM dATP, 250 uM dGTP, 250 uM dTTP, 250 uM dCTP, 5 ng ssM13mp18 template DNA, 200 ng of oligonucleotide No. B (22mer), either 200 ng of oligonucleotide No. A' (19mer) or 300 ng of ligation reaction product (14mer) produced in Example 2, and 1 unit Taq DNA polymerase.</p>
    <p>The PCR admixture was then cycled in a 9600 Gene Amplification System PCR thermocycler (Perkin Elmer Cetus) under the following conditions: first, the admixture was maintained at 95° C. for 5 minutes, next the admixture was cycled through the following three temperatures at the indicated times for 25 cycles: 95° C. for 1 min, 40° C. for 1 min, and 72 ° C. for 1.5 min; thereafter, the admixture was maintained at 72° C. for 5 minutes to form a PCR product.</p>
    <p>The resulting PCR product was analyzed by gel electrophoresis on a 1% agarose gel, the electrophoresed gel was stained with ethidium bromide, and the electrophoresed PCR products were visualized using ultraviolet light. The PCR reaction scheme and the gel analysis results are shown in FIG. 3.</p>
    <p>Based on the template (ssM13mp18) sequence and the choice of PCR primers, a 980 base pair (bp) fragment is expected to be amplified by the use of either of the primer pairs B/A or B/A', where A is the 14mer ligation reaction product formed in Example 2, and A' is the control 19mer. An amplified 980 bp PCR product is observed when either the control 19mer/22mer (lane 1) or ligation reaction product 14mer/22mer (lane 2) primer pairs were used, indicating that a ligation reaction product formed by the present methods is able to prime template accurately in a PCR reaction when added to the PCR admixture as unpurified ligation reaction product. No PCR product is formed when ligase is not added to the ligation reaction admixture (lane 3) indicating that 6mers and 8mers are not able to prime in a PCR reaction under the conditions tested.</p>
    <heading>4. Cycle-Sequencing Using a Ligation Reaction Product as Primer</heading> <p>The 14mer oligonucleotide in the ligation reaction product formed in Example 2 was used to prime a DNA sequencing reaction, in a cycle-sequencing format. To that end, a sequencing reaction admixture was prepared by admixing the following in 1X sequence buffer (10 mM Tris-HCl, pH 8.3, 50 mM KCl, 4 mM MgCl<sub>2</sub>, 0.1% gelatin): 0.5 uM DATP, 180 uM ddTTP, 5 uM dGTP, 30 uM ddGTP, 5 uM dTTP, 300 uM ddTTP, 5 uM dCTP, 180 uM ddCTP, 10 uCi α<sup>33</sup> P-dATP (1332 Ci/mmole; 10 uCi/ml; New England Nuclear), 100 ng ssM13mp18 template DNA, either 10 ng of oligonucleotide No. A' (19mer) or 10 ng of ligation reaction product (14mer) produced in Example 2, and 1 unit Taq DNA polymerase.</p>
    <p>The sequencing reaction admixture was then cycled in a 9600 Gene Amplification System PCR thermocycler (Perkin Elmer Cetus) under the following conditions: first, the admixture was maintained at 95° C. for 5 minutes, next the admixture was cycled through the following three temperatures at the indicated times for 30 cycles: 95° C. for 10 seconds, 50° C. for 20 seconds, and 72° C. for 30 seconds; thereafter, the admixture was maintained at 72° C. for 5 minutes to form a cycle-sequencing reaction product. The sequencing reaction was stopped by the addition of 5 uL of stop dye mix (90% formamide, 0.05% bromophenol blue, 0.05% xylene cyanol), and 2 ul was electrophoresed on a 6% acrylamide/7M urea sequencing gel.</p>
    <p>The results of the sequencing gel analysis is shown in FIG. 4. The 14mer ligation reaction product in unpurified form specifically primed a unique sequence ladder using the ssM13mp18 template, as shown in the center panel of FIG. 4. The sequence ladder produced by a chemically synthesized 19mer (oligonucleotide A') primer was indistinguishable (left panel of FIG. 4) from the ladder formed by the ligation reaction product, indicating that primer produced by the present methods efficiently primes sequencing reactions. In the absence of ligase, no sequence ladder is seen (right panel) indicating that 6mers and 8mers do not significantly prime a sequencing reaction under the conditions utilized.</p>
    <p>5. Ligation of Degenerate Oligonucleotides to Form a Ligation Reaction Product</p>
    <p>Ligation of degenerate oligonucleotide compositions was compared to ligation of a homogeneous (non-degenerate) oligonucleotide composition to determine if the resulting ligation reaction products can be reproducibly prepared. Oligonucleotide primer sets 6, 6n, 7, 7n, 8, and 8n described herein are shown in Table 2. Using primer sets 7 and 7n as exemplary of the primer sets, the oligonucleotide Nos. 7A, 7B, 7C, 7D (primer set 7) and the corresponding degenerate oligonucleotides 7An, 7Bn, 7Cn, and 7Dn (primer set 7n) having sequences shown in Table 2 were synthesized, and oligonucleotide Nos. 7C and 7Cn were chemically phosphorylated on their 5' termini, essentially as described in Example 1 except that they were produced commercially by Genosys, Inc. (The Woodlands, Tex.).</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">              TABLE 2______________________________________Oligo No.  N-mer   Sequence______________________________________6      16      5'-AAACCCGACAGGACTA-3' (SEQ ID no: 5)6A     8       5'-AAACCCGA-3'6An<sup>a</sup>  8       5'-AANCCNGA-3'6B     8       5'-TGTCGGGT-3'6Bn    8       5'-TGNCGNGT-3'6C     8       5'-P-CAGGACTA-3'6Cn    8       5'-P-CANGANTA-3'6D     8       5'-CCTAGTCC-3'6Dn    8       5'-CCNAGNCC-3'7      16      5'-GGTAACTATCGTCTTG-3' (SEQ ID no: 6)7A     8       5'-GGTAACTA-3'7An    8       5'-GGNAANTA-3'7B     8       5'-GATAGTTA-3'7Bn    8       5'-GANAGNTA-3'7C     8       5'-P-TCGTCTTG-3'7Cn    8       5'-P-TCNTCNTG-3'7D     8       5'-CCCAAGAC-3'7Dn    8       5'-CCNAANAC-3'8      16      5'-TTTTCTACGGGGTCTG-3' (SEQ ID no: 7)8A     8       5'-TTTTCTAC-3'8An    8       5'-TTNTCNAC-3'8B     8       5'-CCGTAGAA-3'8Bn    8       5'-CCNTANAA-3'8C     8       5'-P-GGGGTCTG-3'8Cn    8       5'-P-GGNGTNTG-3'8D     8       5'-AGCAGACC-3'8Dn    8       5'-AGNAGNCC-3'______________________________________ <sup>a</sup> The "n" designates the oligonucleotide as being a degenerate oligonucleotide composition, having all four nucleotides (A, T, G and C) at the position indicated by "N" as described herein.</pre>
    
    <p>The oligonucleotides so produced were admixed into a 10 microliter (ul) ligation reaction admixture containing 1X ligation buffer (50 mM Tris-HCl, pH 7.5, 7 MM MgCl<sub>2</sub>, 1 mM DTT), 100 uM rATP, 125 nanograms (ng) each of non-degenerate oligonucleotides A, B, C and D or 625 ng of degenerate oligonucleotides An, Bn, Cn and Dn, and 4 units of T4 DNA ligase. The ligation reaction admixture was first incubated on ice (0° C.) for 30 min, then maintained (incubated) at 65° C. for 5 minutes. The oligonucleotides thereby anneal to complementary sequences to form duplexes, the cohesive overhangs in the duplexes hybridize, and the ligation reaction proceeds to form a ligation reaction product comprising a 16 mer hybridized to oligonucleotide Nos. A and C. Scheme I shows the use of non-degenerate oligonucleotides, and the corresponding Scheme II shows the use of degenerate oligonucleotides. Note that where degenerate oligonucleotides are used, only specific complementary oligonucleotides participate in the hybridization, and corresponding degenerate non-complementary oligonucleotide species of the oligonucleotide composition do not participate in hybridization.</p>
    <p>The ligation reaction products produced using primer sets 7 or 7n were analyzed by labeling an aliquot of the ligation reaction admixture with <sup>33</sup> P-ATP and T4-polymucleotide kinase. Thereafter, the labeled products were loaded and electrophoresed on a 20% acrylamide/7M urea (denaturing) gel prepared in 1X TBE, and run at 60 watts (W) constant power, greater than 90% ligation efficiency was observed in the presence of ligase when non-degenerate oligonucleotides were used, whereas only nonligated oligonucleotides were observed in the absence of ligase. Similarly, a detectable amount of ligation occurred where degenerate oligonucleotides were used.</p>
    <heading>6. Degenerate Oligonucleotide Ligation Reaction Product as a Cycle Sequencing Reaction Primer</heading> <p>The degenerate oligonucleotide ligation reaction product produced in Example 5 from primer set 7n was used directly, without purification from the ligation reaction admixture, as a primer for cycle sequencing of the pBluescript® plasmid IISK<sup>+</sup>. To that end, cycle sequencing was performed using the Cyclist™ DNA Sequencing Kit (Stratagene). Two ul of the ligation reaction admixture, or 5 ng of synthesized 16mer (oligonucleotide 7) were used to sequence 300 ng of pBluescript® plasmid IISK<sup>+</sup>.</p>
    <p>The cycle sequencing reaction admixture contained 10 mM Tris-HCl, pH 8.8, 50 mM KCl, 4 mM MgCl<sub>2</sub>, 0.001% gelatin, 2 uM dATP, 5 uM dGTP, 5 uM dTTP, 5 uM dCTP, 10 microcuries (uCi) of α<sup>33</sup> P-dATP (1332 Ci/mmole; 10 uCi/ml; New England Nuclear), 2 units of Tag DNA polymerase, and one of the following; 180 uM ddATP, 180 uM ddCTP, 30 uM ddGTP or 300 uM ddTTP. The reaction admixture was initially denatured at 95° C. for 5 min, and then was cycled 30 times in a 9600 Gene Amplification System PCR thermocycler (Perkin Elmer Cetus) through a profile of 95° C. for 20 seconds, 50° C. for 30 seconds, and 72° C. for 40 seconds. Thereafter, 5 ul of stop dye (95% formamide, 20 mM EDTA, 0.05% bromophenol blue, 0.05% xylene cyanol) were added the admixture, and 2 ul of the resulting admixture were loaded and electrophoresed on 6% acrylamide/7M urea sequencing gel at a constant power of 60 watts. oligonucleotide primer sets 6, 6n, 8 and 8n were similarly prepared from the oligonucleotides shown in Table 2, ligated as in Example 5 and used in cycle sequencing as described above for primer sets 7 and 7n, and the sequencing results analyzed on sequencing gels. The sequencing gel results are shown in FIG. 5.</p>
    <p>The results show that the sequencing ladder (four lanes corresponding to A, T, G and C) produced by three different degenerate oligonucleotide primers (shown in the B lanes for primer sets 6n, 7n or 8n) upon ligation according to the present methods is essentially indistinguishable from the sequencing ladder produced by using a synthetic 16mer primer (shown in the A lanes for oligonucleotides 6, 7 or 8), indicating that the primer produced by the present methods efficiently primes a cycle sequencing reaction without undesirable interfering reaction. In the absence of added ligase (shown in the C lanes), no sequencing ladder is seen indicating that the non-ligated 8mers of the degenerate oligonucleotide composition do not significantly prime the cycle sequencing reaction.</p>
    <heading>7. Reverse Transcriptase Sequencing Using a Degenerate Oligonucleotide Ligation Reaction Product as Primer</heading> <p>The oligonucleotide in the ligation reaction product formed by using degenerate primer set 7n was used to prime a DNA sequencing reaction, in a reverse transcriptase (RT) sequencing format. To that end, a 10 ul RT sequencing reaction admixture was prepared by first admixing the following: 1 ul 10X reverse transcriptase buffer (50 mM Tris-HCl, pH 8.3, 20 mM KCl, 10 mM MgCl<sub>2</sub>), 800 ng single-stranded pbluescript® plasmid II SK<sup>+</sup>  DNA (+ strand), 2 ul containing ligation reaction product from primer set 7 or 7n, or 5 ng of 16mer control oligonucleotide 7 (SEQ ID NO 6 ) and sufficient distilled water to bring the volume to 10 ul. This first admixture was incubated for 2-3 min at 50° C. Thereafter, 2.5 ul of label mix was added containing 5 uCi α<sup>33</sup> P-dATP (1332 Ci/mmole; 10 uCi/ml; New England Nuclear), 2 uM dCTP, 2 uM dGTP, 2 uM dTTP and 5 units avian myeloblastosis virus (AMV) reverse transcriptase (Stratagene), to form an RT labeling reaction admixture. The admixture was incubated 3-5 min at 50° C. to form labeled reaction product, and then 2.5 ul of the labeled reaction product was transferred to a fresh prewarmed (37° C.) tube containing 2.5 ul of one of the four ddNTP mixes (Sequenase Kit) and 0.5 units of Sequenase diluted in enzyme dilution buffer (10 mM Tris-HCl, pH 7.5, 5 mM DTT, 0.5 mg/ml BSA) to form an RT sequencing reaction admixture. The ddNTP mix contains 80 uM dATP, 80 uM dCTP, 80 uM dGTP, 80 uM dTPP, 50 uM NaCl, and 8 uM of one of the ddNTPs. The RT sequencing reaction admixture was incubated 3-5 min at 37° C., and then 4 ul of stop solution (95% formamide, 0.025% bromophenol blue, 0.025% xylene cyanol) was added to the incubated admixture. Two ul of the resulting RT sequencing reaction admixture were loaded onto and electrophoresed on a sequencing gel as described in Example 3 for cycle sequencing.</p>
    <p>The resulting RT sequencing reactions analysed on a sequencing gel are shown in FIG. 6. The 16mer degenerate oligonucleotide (7n) ligation reaction product in unpurified form (i.e., used directly from the ligation reaction admixture) specifically primed a unique sequence ladder using the pBluescript® plasmid template forming a clean and readable sequencing 5 ladder, as shown in the B lanes of FIG. 6. The sequence ladder produced by a chemically synthesized 16mer (oligonucleotide 7) primer (shown in the C lanes) or the ladder produced by the non-degenerate ligation reaction product (primer set 7A-D; shown in the D lanes) was indistinguishable from the ladder formed by the degenerate oligonucleotide ligation reaction product, indicating that primer produced by the present methods with degenerate oligonucleotides efficiently primes sequencing reactions. In the absence of ligase, no sequence ladder is seen (shown in the A lanes) indicating that non-ligated 8mers do not significantly prime a sequencing reaction under the conditions utilized.</p>
    <heading>8. Modified T<sub>7</sub> DNA Polymerase Sequencing Using a Degenerate Oligonucleotide Ligation Reaction Product as Primer</heading> <p>The oligonucleotide in the ligation reaction product formed by using degenerate oligonucleotide set 7n was used to prime a DNA sequencing reaction, in a sequencing format that uses a modified T<sub>7</sub> DNA polymerase that exhibits reduces 3'to 5' exonuclease activity. The polymerase used in this case is referred to as Sequenase. To that end, a 10 ul Sequenase sequencing reaction admixture was prepared by first admixing the following: 2 ul 5X Sequenase buffer (40 mM Tris-HCl, pH 7.5, 20 mM KCl, 50 mM NaCl), 3 ug denatured double-stranded pBluescript® plasmid DNA, 2 ul containing ligation reaction product from degenerate oligonucleotide primer set 7n or 5 ng of 16mer control oligonucleotide 7 (SEQ ID NO 6) and sufficient distilled water to bring the volume to 10 ul. This first admixture was incubated for 2-3 min at 50° C. Thereafter, 2.5 ul of label mix was added containing 5 uCi α<sup>33</sup> P-dATP (1332 Ci/mmole; 10 uCi/ml; New England Nuclear), 1.5 uM dCTP, 1.5 uM dGTP, 1.5 uM dTTP and 5 units Sequenase (U.S. Biochemicals), to form a Sequenase labeling reaction admixture. The admixture was incubated 3-5 min at 50° C. to form labeled reaction product, and then 2.5 ul of the labeled reaction product was transferred to a fresh prewarmed (37° C.) tube containing 2.5 ul of one of the four ddNTP mix (Example 4) and 0.5 units of Sequenase diluted in enzyme dilution buffer (Example 4) to form a Sequenase sequencing reaction admixture. The Sequenase sequencing reaction admixture was incubated 3-5 min at 37° C., and then 4 ul of stop solution (Example 4) was added to the incubated admixture. Two ul of the resulting Sequenase sequencing reaction admixture were loaded onto and electrophoresed on a sequencing gel as described in Example 3 for cycle sequencing.</p>
    <p>Oligonucleotide primer sets 6n and 8n were similarly prepared from the oligonucleotides shown in Table 2, ligated as in Example 5 and used in Sequenase sequencing as described above for primer set 7n, and the sequencing results analyzed on sequencing gels.</p>
    <p>The resulting Sequenase sequencing reactions analyzed on a sequencing gel are shown in FIG. 7.</p>
    <p>The results show that the sequencing ladder (four lanes corresponding to A, T, G and C) produced by three different degenerate oligonucleotide primers (shown in the B lanes for primer sets 6n, 7n or 8n) upon ligation according to the present methods is essentially indistinguishable from the sequencing ladder produced by using a synthetic 16mer primer (shown in the A lanes for oligonucleotides 6, 7 or 8), indicating that the primer produced by the present methods efficiently primes a Sequenase sequencing reaction without undesirable interfering reaction. In the absence of added ligase (shown in the C lanes), no sequencing ladder is seen indicating that the non-ligated 8mers of the degenerate oligonucleotide composition do not significantly prime the Sequenase sequencing reaction. The random specs present on the illustrated gel is an artifact of static electricity during the film development process, can generally be prevented, and is not to be considered a limitation in resolution for the present sequencing method.</p>
    <p>The foregoing specification, including the specific embodiments and examples, is illustrative of the present invention and is not intended to limit the invention in any way. It will be apparent to those skilled in the art that numerous variations and modifications to the above-described embodiments of the invention will be possible without departing from the spirit and scope of the invention. It is intended that the following claims be interpreted to embrace all such variations and modifications.</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">__________________________________________________________________________SEQUENCE LISTING(1) GENERAL INFORMATION:(iii) NUMBER OF SEQUENCES: 7(2) INFORMATION FOR SEQ ID NO:1:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 14 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: unknown(D) TOPOLOGY: unknown(xi) SEQUENCE DESCRIPTION: SEQ ID NO:1:ACGACGGCCAGTGC14(2) INFORMATION FOR SEQ ID NO:2:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 16 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: unknown(D) TOPOLOGY: unknown(xi) SEQUENCE DESCRIPTION: SEQ ID NO:2:TGACGACGTCCAGTGC16(2) INFORMATION FOR SEQ ID NO:3:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 19 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: unknown(D) TOPOLOGY: unknown(xi) SEQUENCE DESCRIPTION: SEQ ID NO:3:TAAAACGACGACGGCCAGT19(2) INFORMATION FOR SEQ ID NO:4:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 22 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: unknown(D) TOPOLOGY: unknown(xi) SEQUENCE DESCRIPTION: SEQ ID NO:4:TGATTCCAACGAGGAAAGCACG22(2) INFORMATION FOR SEQ ID NO:5:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 16 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: unknown(D) TOPOLOGY: unknown(xi) SEQUENCE DESCRIPTION: SEQ ID NO:5:AAACCCGACAGGACTA16(2) INFORMATION FOR SEQ ID NO:6:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 16 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: unknown(D) TOPOLOGY: unknown(xi) SEQUENCE DESCRIPTION: SEQ ID NO:6:GGTAACTATCGTCTTG16(2) INFORMATION FOR SEQ ID NO:7:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 16 base pairs(B) TYPE: nucleic acid(C) STRANDEDNESS: unknown(D) TOPOLOGY: unknown(xi) SEQUENCE DESCRIPTION: SEQ ID NO:7:TTTTCTACGGGGTCTG16__________________________________________________________________________</pre>
    
    </div></div></div><div class="patent-section patent-tabular-section"><a id="backward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Cited Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5114839">US5114839</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">May 24, 1989</td><td class="patent-data-table-td patent-date-value">May 19, 1992</td><td class="patent-data-table-td ">Gesellschaft Fur Biotechnologische Forsching Mbh</td><td class="patent-data-table-td ">Process for dna sequencing using oligonucleotide bank</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5366882">US5366882</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Dec 17, 1993</td><td class="patent-data-table-td patent-date-value">Nov 22, 1994</td><td class="patent-data-table-td ">New England Biolabs</td><td class="patent-data-table-td ">Isolated dna</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5371006">US5371006</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jan 11, 1993</td><td class="patent-data-table-td patent-date-value">Dec 6, 1994</td><td class="patent-data-table-td ">New England Biolabs, Inc.</td><td class="patent-data-table-td ">Isolated DNA encoding the NotI restriction endonuclease and related methods for producing the same</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5599921">US5599921</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">May 8, 1991</td><td class="patent-data-table-td patent-date-value">Feb 4, 1997</td><td class="patent-data-table-td ">Stratagene</td><td class="patent-data-table-td ">Oligonucleotide families useful for producing primers</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5663062">US5663062</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Nov 30, 1993</td><td class="patent-data-table-td patent-date-value">Sep 2, 1997</td><td class="patent-data-table-td ">Stratagene</td><td class="patent-data-table-td ">Oligonucleotide libraries useful for producing primers</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1989011211A2?cl=en">WO1989011211A2</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">May 24, 1989</td><td class="patent-data-table-td patent-date-value">Nov 30, 1989</td><td class="patent-data-table-td ">Biotechnolog Forschung Gmbh</td><td class="patent-data-table-td ">Oligonucleotide bank and process for dna sequencing</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="npl-citations"></a><div class="patent-section-header"><span class="patent-section-title">Non-Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th colspan="3"class="patent-data-table-th">Reference</th></tr></thead><tr><td class="patent-data-table-td ">1</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Studier. "<a href='http://scholar.google.com/scholar?q="A+Strategy+for+High-Volume+Sequencing+of+Cosmid+DNAs%3A+Random+and+directed+Priming+with+a+Library+of+Oligonucleotides"'>A Strategy for High-Volume Sequencing of Cosmid DNAs: Random and directed Priming with a Library of Oligonucleotides</a>". Proceedings of the National Academy of Sciences USA 86: 6917-6922 (1989).</td></tr><tr><td class="patent-data-table-td ">2</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Studier. A Strategy for High Volume Sequencing of Cosmid DNAs: Random and directed Priming with a Library of Oligonucleotides . Proceedings of the National Academy of Sciences USA 86: 6917 6922 (1989).</td></tr><tr><td class="patent-data-table-td ">3</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Szbalski. "<a href='http://scholar.google.com/scholar?q="Proposal+for+Sequencing+DNA+Using+Litigation+of+Hexamers+to+Generate+Sequential+Elongation+Primers"'>Proposal for Sequencing DNA Using Litigation of Hexamers to Generate Sequential Elongation Primers</a>". Gene 90: 177-178 (1990).</td></tr><tr><td class="patent-data-table-td ">4</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Szbalski. Proposal for Sequencing DNA Using Litigation of Hexamers to Generate Sequential Elongation Primers . Gene 90: 177 178 (1990).</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="forward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Referenced by</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Citing Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7087430">US7087430</a></td><td class="patent-data-table-td patent-date-value">Jun 12, 2003</td><td class="patent-data-table-td patent-date-value">Aug 8, 2006</td><td class="patent-data-table-td ">Pherin Pharmaceuticals, Inc.</td><td class="patent-data-table-td ">Human VNO cDNA libraries</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7264931">US7264931</a></td><td class="patent-data-table-td patent-date-value">Sep 5, 2001</td><td class="patent-data-table-td patent-date-value">Sep 4, 2007</td><td class="patent-data-table-td ">Rutgers, The State University Of New Jersey</td><td class="patent-data-table-td ">Method for sequencing nucleic acid sequences without chain termination</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7537892">US7537892</a></td><td class="patent-data-table-td patent-date-value">Aug 16, 2005</td><td class="patent-data-table-td patent-date-value">May 26, 2009</td><td class="patent-data-table-td ">Searete Llc</td><td class="patent-data-table-td ">Determining the identity of an unknown nucleotide at a position of interest on a target nucleic acid analyte; Reducing the required number of unique hybridizing sequences that may be used to hybridize to a nucleic acid sequence of interest under hybridizing conditions</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8716033">US8716033</a></td><td class="patent-data-table-td patent-date-value">Feb 12, 2007</td><td class="patent-data-table-td patent-date-value">May 6, 2014</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Oligosaccharide modification and labeling of proteins</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8785212">US8785212</a></td><td class="patent-data-table-td patent-date-value">Jan 14, 2011</td><td class="patent-data-table-td patent-date-value">Jul 22, 2014</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Oligosaccharide modification and labeling of proteins</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP1399588A2?cl=en">EP1399588A2</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Mar 27, 2002</td><td class="patent-data-table-td patent-date-value">Mar 24, 2004</td><td class="patent-data-table-td ">Applied Minds, Inc.</td><td class="patent-data-table-td ">Method and sequences for determinate nucleic acid hybridization</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP1551986A2?cl=en">EP1551986A2</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Sep 30, 2003</td><td class="patent-data-table-td patent-date-value">Jul 13, 2005</td><td class="patent-data-table-td ">Parallele Bioscience, Inc.</td><td class="patent-data-table-td ">Polynucleotide synthesis and labeling by kinetic sampling ligation</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2002020843A1?cl=en">WO2002020843A1</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Sep 5, 2001</td><td class="patent-data-table-td patent-date-value">Mar 14, 2002</td><td class="patent-data-table-td ">Abram Gabriel</td><td class="patent-data-table-td ">A method for sequencing nucleic acid sequences without chain termination</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="classifications"></a><div class="patent-section-header"><span class="patent-section-title">Classifications</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th"> </th><th class="patent-data-table-th"> </th></tr></thead><tr><td class="patent-data-table-td ">U.S. Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=gx1JBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S091100">435/91.1</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=gx1JBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S024330">536/24.33</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=gx1JBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S024300">536/24.3</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=gx1JBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S025300">536/25.3</a></span></td></tr><tr><td class="patent-data-table-td ">International Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=gx1JBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C40B0060140000">C40B60/14</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=gx1JBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C07H0021000000">C07H21/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=gx1JBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=B01J0019000000">B01J19/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=gx1JBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0015100000">C12N15/10</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=gx1JBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C40B0040060000">C40B40/06</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=gx1JBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12Q0001680000">C12Q1/68</a></span></td></tr><tr><td class="patent-data-table-td ">Cooperative Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=gx1JBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12Q1/6811">C12Q1/6811</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=gx1JBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C40B60/14">C40B60/14</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=gx1JBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=B01J2219/00315">B01J2219/00315</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=gx1JBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=B01J2219/005">B01J2219/005</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=gx1JBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=B01J19/0046">B01J19/0046</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=gx1JBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=B01J2219/00722">B01J2219/00722</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=gx1JBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12Q1/6837">C12Q1/6837</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=gx1JBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C07H21/00">C07H21/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=gx1JBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=B01J2219/00322">B01J2219/00322</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=gx1JBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N15/10">C12N15/10</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=gx1JBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=B01J2219/00664">B01J2219/00664</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=gx1JBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C40B40/06">C40B40/06</a></span></td></tr><tr><td class="patent-data-table-td ">European Classification</td><td class="patent-data-table-td "><span class="nested-value">C07H21/00</span>, <span class="nested-value">C12Q1/68A8</span>, <span class="nested-value">C12Q1/68B10A</span>, <span class="nested-value">B01J19/00C</span>, <span class="nested-value">C12N15/10</span></td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="legal-events"></a><div class="patent-section-header"><span class="patent-section-title">Legal Events</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Date</th><th class="patent-data-table-th">Code</th><th class="patent-data-table-th">Event</th><th class="patent-data-table-th">Description</th></tr></thead><tr><td class="patent-data-table-td patent-date-value">Jul 10, 2012</td><td class="patent-data-table-td ">RR</td><td class="patent-data-table-td ">Request for reexamination filed</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20120529</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Dec 2, 2011</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">MERGER;ASSIGNOR:STRATAGENE CALIFORNIA;REEL/FRAME:027320/0674</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20100521</span></div><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">AGILENT TECHNOLOGIES RESEARCH CORPORATION, CALIFOR</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20111129</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNOR:AGILENT TECHNOLOGIES RESEARCH CORPORATION;REEL/FRAME:027318/0233</span></div><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">CATALYST ASSETS LLC, WYOMING</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Nov 29, 2011</td><td class="patent-data-table-td ">RR</td><td class="patent-data-table-td ">Request for reexamination filed</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20110929</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Jul 12, 2010</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">12</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Jul 12, 2006</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">8</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Jan 29, 2004</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">STRATAGENE, CALIFORNIA</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNOR:UNION BANK OF CALIFORNIA, N.A.;REEL/FRAME:014885/0317</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20040115</span></div><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">STRATAGENE 11011 N. TORREY PINES ROADLA JOLLA, CAL</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNOR:UNION BANK OF CALIFORNIA, N.A. /AR;REEL/FRAME:014885/0317</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Jan 26, 2004</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">STRATAGENE CALIFORNIA, CALIFORNIA</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">CHANGE OF NAME;ASSIGNOR:STRATAGENE;REEL/FRAME:016182/0616</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20031209</span></div><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">STRATAGENE CALIFORNIA 11011 N. TORREY PINES ROADLA</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">CHANGE OF NAME;ASSIGNOR:STRATAGENE /AR;REEL/FRAME:016182/0616</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Jun 20, 2002</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">4</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Feb 4, 1998</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">STRATAGENE, CALIFORNIA</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNORS:SORGE, JOSEPH A.;SHOEMAKER, DANIEL;REEL/FRAME:009007/0913;SIGNING DATES FROM 19980128 TO 19980130</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Jun 10, 1996</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">UNION BANK OF CALIFORNIA, N.A., CALIFORNIA</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNOR:UNION BANK;REEL/FRAME:007991/0247</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">19960401</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Jan 22, 1996</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">UNION BANK, CALIFORNIA</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">SECURITY INTEREST;ASSIGNOR:STRATAGENE;REEL/FRAME:007837/0408</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">19951201</span></div></td></tr></table><div class="patent-section-footer"></div></div><div class="modal-dialog" id="patent-images-lightbox"><div class="patent-lightbox-controls"><div class="patent-lightbox-rotate-controls"><div class="patent-lightbox-rotation-text">Rotate</div><div class="rotate-icon rotate-ccw-icon"></div><div class="rotate-icon rotate-cw-icon"></div></div><div class="patent-lightbox-index-counter"></div><a class="patent-lightbox-fullsize-link" target="_blank">Original Image</a><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_right.png" alt="Next page"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_left.png" alt="Previous page"width="21" height="21" /></div></div><div class="modal-dialog-content"><div class="patent-lightbox-image-holder"><div class="patent-lightbox-placeholder"></div></div></div></div><script>_OC_initPatentsAtb({image_not_available_html: " Image not available"});</script></div></div></div></td></tr></table><script>(function() {var href = window.location.href;if (href.indexOf('?') !== -1) {var parameters = href.split('?')[1].split('&');for (var i = 0; i < parameters.length; i++) {var param = parameters[i].split('=');if (param[0] == 'focus') {var elem = document.getElementById(param[1]);if (elem) {elem.focus();}}}}})();</script><script>_OC_addFlags({LockSrc:"/books/javascript/lock_8a2b04e7bf975d5171d8e4c0b6365c7a.js", Host:"http://www.google.com/", IsBooksRentalEnabled:1, IsWebstoreDisplayCaseEnabled:1, IsObfuscationEnabled:1, IsBrowsingHistoryEnabled:1, IsWebReaderSvgEnabled:0, IsGeoLayerEnabled:1, IsImageModeNotesEnabled:1, IsCopyMenuItemEnabled:1, IsGiftingEnabled:0, IsWebReaderUniversalPaginatorEnabled:0, IsOfflineBubbleEnabled:1, IsReaderEnabledForPlayRequests:1, IsFutureOnSaleVolumesEnabled:1, IsOfflineRestrictedCopyEnabled:1, IsBooksUnifiedLeftNavEnabled:1, IsRestrictedCopyEnabled:1, IsZipitFolderCollectionEnabled:1, IsEndOfSampleRecommendationsEnabled:1, IsRatingsOnBookcardsEnabled:1, IsAdsDisabled:0, IsIframePageDisplayEnabled:0, IsEmbeddedMediaEnabled:1, IsImageModeAnnotationsEnabled:1, IsMyLibraryGooglePlusEnabled:1, IsImagePageProviderEnabled:0, IsBookcardListPriceSmall:0, IsInternalUser:0, IsBooksShareButtonEnabled:0, IsPreOrdersEnabled:0, IsDisabledRandomBookshelves:0, WebstoreDisplayCasePosition:3});_OC_Run({"enable_p13n":false,"add_vol_to_collection_base_url":"http://www.google.com/patents?op=add\u0026sig=ACfU3U3jmjSSZajUw8ZklS6Djl6U9AoF4A\u0026id=gx1JBAABERAJ","remove_vol_from_collection_base_url":"http://www.google.com/patents?op=remove\u0026sig=ACfU3U3pDZQEwira51F4aUEcki5yGLROsg\u0026id=gx1JBAABERAJ","logged_in":false,"p13n_save_user_settings_url":"http://www.google.com/patents?op=edit_user_settings\u0026sig=ACfU3U3zJkGLww_lC_I7Hr_GEgcxF9pAeA","is_cobrand":false,"sign_in_url":"https://www.google.com/accounts/Login?service=\u0026continue=http://www.google.com/patents%3Fhl%3Den\u0026hl=en","is_play_enabled":true}, {"volume_id":"","is_ebook":true,"volumeresult":{"has_flowing_text":false,"has_scanned_text":true,"can_download_pdf":false,"can_download_epub":false,"is_pdf_drm_enabled":false,"is_epub_drm_enabled":false,"download_pdf_url":"http://www.google.com/patents/download/Nucleotide_sequences_including_nondegene.pdf?id=gx1JBAABERAJ\u0026output=pdf\u0026sig=ACfU3U1VxqZvg1CDYunpMsg83_d52xin4w"},"sample_url":"http://www.google.com/patents/reader?id=gx1JBAABERAJ\u0026printsec=frontcover\u0026output=reader\u0026source=gbs_atb_hover","is_browsable":true,"is_public_domain":true}, {});</script><div id="footer_table" style="font-size:83%;text-align:center;position:relative;top:20px;height:4.5em;margin-top:2em"><div style="margin-bottom:8px"><a href=http://www.google.com/><nobr>Google&nbsp;Home</nobr></a> - <a href=//www.google.com/patents/sitemap/><nobr>Sitemap</nobr></a> - <a href=http://www.google.com/googlebooks/uspto.html><nobr>USPTO Bulk Downloads</nobr></a> - <a href=/intl/en/privacy/><nobr>Privacy Policy</nobr></a> - <a href=/intl/en/policies/terms/><nobr>Terms of Service</nobr></a> - <a href=https://support.google.com/faqs/answer/2539193?hl=en><nobr>About Google Patents</nobr></a> - <a href="http://www.google.com/tools/feedback/intl/en/error.html" onclick="try{_OC_startFeedback({productId: '72792',locale: 'en'});return false;}catch(e){}"><nobr>Send Feedback</nobr></a></div><span>Data provided by IFI CLAIMS Patent Services</span><br><span >&copy;2012 Google</span></div> <script type="text/javascript">var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));</script><script type="text/javascript">var pageTracker = _gat._getTracker("UA-27188110-1");pageTracker._setCookiePath("/patents/");pageTracker._trackPageview();</script> </body></html>