<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN""http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html><head><title>Patent US5871908 - Process for the determination of in vitro amplified nucleic acids - Google Patents</title><script>(function(){(function(){function e(a){this.t={};this.tick=function(a,c,b){var d=void 0!=b?b:(new Date).getTime();this.t[a]=[d,c];if(void 0==b)try{window.console.timeStamp("CSI/"+a)}catch(e){}};this.tick("start",null,a)}var a;window.performance&&(a=window.performance.timing);var f=a?new e(a.responseStart):new e;window.jstiming={Timer:e,load:f};if(a){var c=a.navigationStart,d=a.responseStart;0<c&&d>=c&&(window.jstiming.srt=d-c)}if(a){var b=window.jstiming.load;0<c&&d>=c&&(b.tick("_wtsrt",void 0,c),b.tick("wtsrt_",
"_wtsrt",d),b.tick("tbsd_","wtsrt_"))}try{a=null,window.chrome&&window.chrome.csi&&(a=Math.floor(window.chrome.csi().pageT),b&&0<c&&(b.tick("_tbnd",void 0,window.chrome.csi().startE),b.tick("tbnd_","_tbnd",c))),null==a&&window.gtbExternal&&(a=window.gtbExternal.pageT()),null==a&&window.external&&(a=window.external.pageT,b&&0<c&&(b.tick("_tbnd",void 0,window.external.startE),b.tick("tbnd_","_tbnd",c))),a&&(window.jstiming.pt=a)}catch(g){}})();})();
</script><link rel="stylesheet" href="/patents/css/_4ff636b3d23669b7103f3b3a3a18b4cd/kl_intl_patents_bundle.css" type="text/css" /><script src="/books/javascript/atb_4ff636b3d23669b7103f3b3a3a18b4cd__en.js"></script><script>function googleTranslateElementInit() {new google.translate.TranslateElement({pageLanguage: "en",gaTrack: true,gaId: "UA-27188110-1",multilanguagePage: true});}</script><script src="//translate.google.com/translate_a/element.js?cb=googleTranslateElementInit"></script><meta name="DC.type" content="Patent"><meta name="DC.title" content="Process for the determination of in vitro amplified nucleic acids"><meta name="DC.contributor" content="Karsten Henco" scheme="inventor"><meta name="DC.contributor" content="Manfred Eigen" scheme="inventor"><meta name="DC.contributor" content="Detlev Riesner" scheme="inventor"><meta name="DC.contributor" content="Evotec Biosystems Gmbh" scheme="assignee"><meta name="DC.date" content="1993-2-4" scheme="dateSubmitted"><meta name="DC.description" content="A process for the qualitative and quantitative determination of at least one in vitro amplified nucleic acid in a sealed reaction chamber, wherein during or subsequent to the amplification of the nucleic acid at least one substance (probe) is present which interacts with the nucleic acid to be detected; wherein spectroscopically measurable parameters of said substance (probe) are subject to variation, creating a measurable signal; wherein the sample to be measured is exposed to the action of a gradient capable of at least partially denaturing nucleic acids; with detection of the measurable parameter undergoing variation through the action of the gradient; and the entire amplification reaction, including qualitative and quantitative determination, may be carried out in a sealed reaction chamber (measuring compartment) without intermittent opening, permitting to automatically operate the diagnostic method of DNA and RNA amplification in qualitative and quantitative fashion on large series of samples."><meta name="DC.date" content="1999-2-16" scheme="issued"><meta name="DC.relation" content="EP:0381501" scheme="references"><meta name="DC.relation" content="JP:H09201533" scheme="references"><meta name="DC.relation" content="US:5184020" scheme="references"><meta name="DC.relation" content="WO:1990001563:A1" scheme="references"><meta name="DC.relation" content="WO:1990005023:A1" scheme="references"><meta name="DC.relation" content="WO:1990015881:A1" scheme="references"><meta name="DC.relation" content="WO:1991002815:A1" scheme="references"><meta name="DC.relation" content="WO:1991002817:A1" scheme="references"><meta name="citation_reference" content="J.E. Hearst, &quot;A photochemical investigation of the dynamics of the oligonucleotide hybridization&quot;, Ann. Rev. Phys. Chem., Bd. 39, pp. 291-315, 1988."><meta name="citation_reference" content="J.E. Hearst, A photochemical investigation of the dynamics of the oligonucleotide hybridization , Ann. Rev. Phys. Chem., Bd. 39, pp. 291 315, 1988."><meta name="citation_reference" content="K. Henco et al., &quot;Quantitative PCR: the determination of template copy numbers by temperature gradient gel electrohoresis (TGGE)&quot;, Nucleic Acids Research, Bd. 18, Nr. 22, pp. 6733-6734, 1990."><meta name="citation_reference" content="K. Henco et al., Quantitative PCR: the determination of template copy numbers by temperature gradient gel electrohoresis (TGGE) , Nucleic Acids Research, Bd. 18, Nr. 22, pp. 6733 6734, 1990."><meta name="citation_reference" content="Kingsbury et al., &quot;Rapid Detection and Identification of Infection Agents&quot;, Academic Press, Inc., pp. 245-256, 1985."><meta name="citation_reference" content="Kingsbury et al., Rapid Detection and Identification of Infection Agents , Academic Press, Inc., pp. 245 256, 1985."><meta name="citation_reference" content="Y. Jinno et al., &quot;Use of psoralen as extinguisher of contaminated DNA in PCT&quot;, Nucleic Acids Research, Bd. 18, Nr. 22, p. 6739."><meta name="citation_reference" content="Y. Jinno et al., Use of psoralen as extinguisher of contaminated DNA in PCT , Nucleic Acids Research, Bd. 18, Nr. 22, p. 6739."><meta name="citation_patent_number" content="US:5871908"><meta name="citation_patent_application_number" content="US:08/157,195"><link rel="canonical" href="http://www.google.com/patents/US5871908"/><meta property="og:url" content="http://www.google.com/patents/US5871908"/><meta name="title" content="Patent US5871908 - Process for the determination of in vitro amplified nucleic acids"/><meta name="description" content="A process for the qualitative and quantitative determination of at least one in vitro amplified nucleic acid in a sealed reaction chamber, wherein during or subsequent to the amplification of the nucleic acid at least one substance (probe) is present which interacts with the nucleic acid to be detected; wherein spectroscopically measurable parameters of said substance (probe) are subject to variation, creating a measurable signal; wherein the sample to be measured is exposed to the action of a gradient capable of at least partially denaturing nucleic acids; with detection of the measurable parameter undergoing variation through the action of the gradient; and the entire amplification reaction, including qualitative and quantitative determination, may be carried out in a sealed reaction chamber (measuring compartment) without intermittent opening, permitting to automatically operate the diagnostic method of DNA and RNA amplification in qualitative and quantitative fashion on large series of samples."/><meta property="og:title" content="Patent US5871908 - Process for the determination of in vitro amplified nucleic acids"/><meta property="og:type" content="book"/><meta property="og:site_name" content="Google Books"/><meta property="og:image" content="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><link rel="image_src" href="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><script>(function(){try{var aa=function(a,b,c,d){d=d||{};d._sn=["cfg",b,c].join(".");window.gbar.logger.ml(a,d)};var g=window.gbar=window.gbar||{},l=window.gbar.i=window.gbar.i||{},m={},n;function _tvn(a,b){var c=parseInt(a,10);return isNaN(c)?b:c}function _tvf(a,b){var c=parseFloat(a);return isNaN(c)?b:c}function _tvv(a){return!!a}function p(a,b,c){(c||g)[a]=b}g.bv={n:_tvn("2",0),r:"",f:".67.",e:"0",m:_tvn("0",1)};
function q(a,b,c){var d="on"+b;if(a.addEventListener)a.addEventListener(b,c,!1);else if(a.attachEvent)a.attachEvent(d,c);else{var f=a[d];a[d]=function(){var a=f.apply(this,arguments),b=c.apply(this,arguments);return void 0==a?b:void 0==b?a:b&&a}}}var s=function(a){return function(){return g.bv.m==a}},ba=s(1),ca=s(2);p("sb",ba);p("kn",ca);l.a=_tvv;l.b=_tvf;l.c=_tvn;l.i=aa;var da=window.gbar.i.i;var t,u,v,w;function ea(a){v=a}function fa(a){var b;if(b=v&&window.encodeURIComponent)b=a.href,b=!b.match(/^http[s]?:\/\/accounts\.google\.[^/]*\/ClearSID/i)&&!b.match(/^http[s]?:\/\/[^/]*\/accounts\/ClearSID/i);if(b=b&&encodeURIComponent(v()))a.href=a.href.replace(/([?&]continue=)[^&]*/,"$1"+b)}function ga(a){window.gApplication&&(a.href=window.gApplication.getTabUrl(a.href))}
function ha(a){var b=document.forms[0].q,c=window.encodeURIComponent&&b&&b.value,b=b&&b.placeholder;c&&c!=b&&(a.href=a.href.replace(/([?&])q=[^&]*|$/,function(a,b){return(b||"&")+"q="+encodeURIComponent(c)}))}n=l.a("")?ga:ha;
function x(a,b,c,d,f,e){var h=document.getElementById(a);if(h){var k=h.style;k.left=d?"auto":b+"px";k.right=d?b+"px":"auto";k.top=c+"px";k.visibility=u?"hidden":"visible";f&&e?(k.width=f+"px",k.height=e+"px"):(x(t,b,c,d,h.offsetWidth,h.offsetHeight),u=u?"":a)}}
var y=[],ia=function(a,b){y.push(b)},ja=function(a){a=a||window.event;var b=a.target||a.srcElement;a.cancelBubble=!0;null==t&&(a=document.createElement(Array.every||window.createPopup?"iframe":"div"),a.frameBorder="0",t=a.id="gbs",a.src="javascript:''",b.parentNode.appendChild(a),q(document,"click",z));var c=b,b=0;"gb3"!=c.className&&(c=c.parentNode);a=c.getAttribute("aria-owns")||"gbi";var d=c.offsetWidth,f=20<c.offsetTop?46:24;document.getElementById("tphdr")&&(f-=3);var e=!1;do b+=c.offsetLeft||
0;while(c=c.offsetParent);var c=(document.documentElement.clientWidth||document.body.clientWidth)-b-d,h,d=document.body,k=document.defaultView;k&&k.getComputedStyle?(d=k.getComputedStyle(d,""))&&(h=d.direction):h=d.currentStyle?d.currentStyle.direction:d.style.direction;h="rtl"==h;if("gbi"==a){for(d=0;k=y[d++];)k();A(null,window.navExtra);h&&(b=c,e=!0)}else h||(b=c,e=!0);u!=a&&z();x(a,b,f,e)},z=function(){u&&x(u,0,0)},A=function(a,b){var c,d=document.getElementById("gbi"),f=a;f||(f=d.firstChild);
for(;b&&(c=b.pop());){var e=d,h=c,k=f;w||(w="gb2");e.insertBefore(h,k).className=w}},ka=function(a,b,c){if((b=document.getElementById(b))&&a){a.className="gb4";var d=document.createElement("span");d.appendChild(a);d.appendChild(document.createTextNode(" | "));d.id=c;b.appendChild(d)}},la=function(){return document.getElementById("gb_70")},ma=function(){return!!u};p("qs",n);p("setContinueCb",ea);p("pc",fa);p("tg",ja);p("close",z);p("addLink",ka);p("almm",A);p("si",la);p("adh",ia);p("op",ma);var B=function(){},C=function(){},F=function(a){var b=new Image,c=D;b.onerror=b.onload=b.onabort=function(){try{delete E[c]}catch(a){}};E[c]=b;b.src=a;D=c+1},E=[],D=0;p("logger",{il:C,ml:B,log:F});var G=window.gbar.logger;var H={},na={},I=[],oa=l.b("0.1",.1),pa=l.a("1",!0),qa=function(a,b){I.push([a,b])},ra=function(a,b){H[a]=b},sa=function(a){return a in H},J={},K=function(a,b){J[a]||(J[a]=[]);J[a].push(b)},ta=function(a){K("m",a)},L=function(a,b){var c=document.createElement("script");c.src=a;c.async=pa;Math.random()<oa&&(c.onerror=function(){c.onerror=null;B(Error("Bundle load failed: name="+(b||"UNK")+" url="+a))});(document.getElementById("xjsc")||document.getElementsByTagName("body")[0]||
document.getElementsByTagName("head")[0]).appendChild(c)},N=function(a){for(var b=0,c;(c=I[b])&&c[0]!=a;++b);!c||c[1].l||c[1].s||(c[1].s=!0,M(2,a),c[1].url&&L(c[1].url,a),c[1].libs&&m.d&&m.d(c[1].libs))},O=function(a){K("gc",a)},P=null,ua=function(a){P=a},M=function(a,b,c){if(P){a={t:a,b:b};if(c)for(var d in c)a[d]=c[d];try{P(a)}catch(f){}}};p("mdc",H);p("mdi",na);p("bnc",I);p("qGC",O);p("qm",ta);p("qd",J);p("lb",N);p("mcf",ra);p("bcf",qa);p("aq",K);p("mdd","");p("has",sa);
p("trh",ua);p("tev",M);var Q=l.b("0.1",.001),R=0;
function _mlToken(a,b){try{if(1>R){R++;var c,d=a,f=b||{},e=encodeURIComponent,h=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&jexpid=",e("17483"),"&srcpg=",e("prop=22"),"&jsr=",Math.round(1/Q),"&ogev=",e("PljpU8P8LrCisQSd6oHgCA"),"&ogf=",g.bv.f,"&ogrp=",e("1"),"&ogv=",e("1407464522.0"),"&oggv="+e("es_plusone_gc_20140807.0_p0"),"&ogd=",e("com"),"&ogc=",e("EGY"),"&ogl=",e("en")];f._sn&&(f._sn="og."+
f._sn);for(var k in f)h.push("&"),h.push(e(k)),h.push("="),h.push(e(f[k]));h.push("&emsg=");h.push(e(d.name+":"+d.message));var r=h.join("");S(r)&&(r=r.substr(0,2E3));c=r;var Aa=window.gbar.logger._aem(a,c);F(Aa)}}catch(Na){}}var S=function(a){return 2E3<=a.length},va=function(a,b){return b};function T(a){B=a;p("_itl",S,G);p("_aem",va,G);p("ml",B,G);a={};H.er=a}l.a("")?T(function(a){throw a;}):l.a("1")&&Math.random()<Q&&T(_mlToken);I.push(["m",{url:"//ssl.gstatic.com/gb/js/scm_7385cc5883250b43a39405734c1bea59.js"}]);g.mcf("c",{});g.sg={c:""};if(l.a("1")){var wa=l.a("");I.push(["gc",{auto:wa,url:"//ssl.gstatic.com/gb/js/abc/gci_91f30755d6a6b787dcc2a4062e6e9824.js",libs:"googleapis.client:plusone:gapi.iframes"}]);var xa={version:"gci_91f30755d6a6b787dcc2a4062e6e9824.js",index:"",lang:"en"};H.gc=xa;var U=function(a){window.googleapis&&window.iframes?a&&a():(a&&O(a),N("gc"))};p("lGC",U);l.a("1")&&p("lPWF",U)};window.__PVT="";if(l.a("1")&&l.a("1")){var V=function(a){U(function(){K("pw",a);N("pw")})};p("lPW",V);I.push(["pw",{url:"//ssl.gstatic.com/gb/js/abc/pwm_45f73e4df07a0e388b0fa1f3d30e7280.js"}]);var W=[],ya=function(a){W[0]=a},za=function(a,b){var c=b||{};c._sn="pw";B(a,c)},Ba={signed:W,elog:za,base:"https://plusone.google.com/u/0",loadTime:(new Date).getTime()};H.pw=Ba;var X=function(a,b){for(var c=b.split("."),d=function(){var b=arguments;a(function(){for(var a=g,d=0,e=c.length-1;d<e;++d)a=a[c[d]];a[c[d]].apply(a,b)})},f=g,e=0,h=c.length-1;e<h;++e)f=
f[c[e]]=f[c[e]]||{};return f[c[e]]=d};X(V,"pw.clk");X(V,"pw.hvr");p("su",ya,g.pw)};function Ca(){function a(){for(var b;(b=e[h++])&&"m"!=b[0]&&!b[1].auto;);b&&(M(2,b[0]),b[1].url&&L(b[1].url,b[0]),b[1].libs&&m.d&&m.d(b[1].libs));h<e.length&&setTimeout(a,0)}function b(){0<f--?setTimeout(b,0):a()}var c=l.a("1"),d=l.a(""),f=3,e=I,h=0,k=window.gbarOnReady;if(k)try{k()}catch(r){da(r,"ml","or")}d?p("ldb",a):c?q(window,"load",b):b()}p("rdl",Ca);var Da={D:1,H:2,da:3,p:4,W:5,M:6,F:7,g:8,ha:9,U:10,L:11,T:12,S:13,N:14,Q:15,P:16,fa:17,w:18,O:19,ga:20,ea:21,u:22,G:23,ja:24,ka:25,ia:26,A:27,j:28,o:29,k:30,ca:31,Z:32,$:33,J:34,K:35,ba:36,aa:37,Y:38,B:39,R:40,v:41,X:42,V:43,h:48,C:49,I:500},Y=[1,2,3,4,5,6,9,10,11,13,14,28,29,30,34,35,37,38,39,40,41,42,43,48,49,500];var Z=l.b("0.001",1E-4),Ea=l.b("1",1),Fa=!1,Ga=!1;if(l.a("1")){var Ha=Math.random();Ha<=Z&&(Fa=!0);Ha<=Ea&&(Ga=!0)}var Ia=Da,$=null;function Ja(){var a=0,b=function(b,d){l.a(d)&&(a|=b)};b(1,"");b(2,"");b(4,"");b(8,"");return a}
function Ka(a,b){var c=Z,d=Fa,f;f=a;if(!$){$={};for(var e=0;e<Y.length;e++){var h=Y[e];$[h]=!0}}if(f=!!$[f])c=Ea,d=Ga;if(d){d=encodeURIComponent;g.rp?(f=g.rp(),f="-1"!=f?f:"1"):f="1";c=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&oge=",a,"&ogex=",d("17483"),"&ogev=",d("PljpU8P8LrCisQSd6oHgCA"),"&ogf=",g.bv.f,"&ogp=",d("22"),"&ogrp=",d(f),"&ogsr=",Math.round(1/c),"&ogv=",d("1407464522.0"),"&oggv="+
d("es_plusone_gc_20140807.0_p0"),"&ogd=",d("com"),"&ogl=",d("en"),"&ogc=",d("EGY"),"&ogus=",Ja()];if(b){"ogw"in b&&(c.push("&ogw="+b.ogw),delete b.ogw);var k;f=b;e=[];for(k in f)0!=e.length&&e.push(","),e.push(La(k)),e.push("."),e.push(La(f[k]));k=e.join("");""!=k&&(c.push("&ogad="),c.push(d(k)))}F(c.join(""))}}function La(a){"number"==typeof a&&(a+="");return"string"==typeof a?a.replace(".","%2E").replace(",","%2C"):a}C=Ka;p("il",C,G);var Ma={};H.il=Ma;setTimeout(function(){C(Ia.g)},0);}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var b=window.gbar.i.i;var c=window.gbar;var f=function(d){try{var a=document.getElementById("gbom");a&&d.appendChild(a.cloneNode(!0))}catch(e){b(e,"omas","aomc")}};c.aomc=f;}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var a=window.gbar;a.mcf("pm",{p:""});}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{window.gbar.rdl();}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
if (window['_OC_timingAction']) {window['_OC_timingAction']('patents_refpage');}</script><style>#gbar,#guser{font-size:13px;padding-top:1px !important;}#gbar{float:left;height:22px}#guser{padding-bottom:7px !important;text-align:right}.gbh,.gbd{border-top:1px solid #c9d7f1;font-size:1px}.gbh{height:0;position:absolute;top:24px;width:100%}#gbs,.gbm{background:#fff;left:0;position:absolute;text-align:left;visibility:hidden;z-index:1000}.gbm{border:1px solid;border-color:#c9d7f1 #36c #36c #a2bae7;z-index:1001}.gb1{margin-right:.5em}.gb1,.gb3{zoom:1}.gb2{display:block;padding:.2em .5em}.gb2,.gb3{text-decoration:none !important;border-bottom:none}a.gb1,a.gb4{text-decoration:underline !important}a.gb1,a.gb2,a.gb3,a.gb4{color:#00c !important}.gbi .gb3,.gbi .gb2,.gbi .gb4{color:#dd8e27 !important}.gbf .gb3,.gbf .gb2,.gbf .gb4{color:#900 !important}a.gb2:hover{background:#36c;color:#fff !important}#gbar .gbz0l{color:#000 !important;cursor:default;font-weight:bold;text-decoration:none !important}
#gbar { padding:.3em .6em !important;}</style></head><body  topmargin="3" marginheight="3"><div id=gbar><nobr><a onclick=gbar.qs(this);gbar.logger.il(1,{t:1}); class=gb1 id=gb_1 href="https://www.google.com/search?sa=N&tab=tw">Search</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:2}); class=gb1 id=gb_2 href="http://www.google.com/search?hl=en&tbm=isch&source=og&sa=N&tab=ti">Images</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:8}); class=gb1 id=gb_8 href="http://maps.google.com/maps?hl=en&sa=N&tab=tl">Maps</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:78}); class=gb1 id=gb_78 href="https://play.google.com/?hl=en&sa=N&tab=t8">Play</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:36}); class=gb1 id=gb_36 href="http://www.youtube.com/results?sa=N&tab=t1">YouTube</a> <a onclick=gbar.logger.il(1,{t:5}); class=gb1 id=gb_5 href="http://news.google.com/nwshp?hl=en&tab=tn">News</a> <a onclick=gbar.logger.il(1,{t:23}); class=gb1 id=gb_23 href="https://mail.google.com/mail/?tab=tm">Gmail</a> <a onclick=gbar.logger.il(1,{t:25}); class=gb1 id=gb_25 href="https://drive.google.com/?tab=to">Drive</a> <a class=gb3 href="http://www.google.com/intl/en/options/" onclick="this.blur();gbar.tg(event);return !1" aria-haspopup=true><u>More</u> <small>&#9660;</small></a><div class=gbm id=gbi><a onclick=gbar.logger.il(1,{t:24}); class=gb2 id=gb_24 href="https://www.google.com/calendar?tab=tc">Calendar</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:51}); class=gb2 id=gb_51 href="http://translate.google.com/?hl=en&sa=N&tab=tT">Translate</a><a onclick=gbar.logger.il(1,{t:17}); class=gb2 id=gb_17 href="http://www.google.com/mobile/?hl=en&tab=tD">Mobile</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:10}); class=gb2 id=gb_10 href="http://www.google.com/search?hl=en&tbo=u&tbm=bks&source=og&sa=N&tab=tp">Books</a><a onclick=gbar.logger.il(1,{t:212}); class=gb2 id=gb_212 href="https://wallet.google.com/manage/?tab=ta">Wallet</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:6}); class=gb2 id=gb_6 href="http://www.google.com/search?hl=en&tbo=u&tbm=shop&source=og&sa=N&tab=tf">Shopping</a><a onclick=gbar.logger.il(1,{t:30}); class=gb2 id=gb_30 href="http://www.blogger.com/?tab=tj">Blogger</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:27}); class=gb2 id=gb_27 href="http://www.google.com/finance?sa=N&tab=te">Finance</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:31}); class=gb2 id=gb_31 href="https://plus.google.com/photos?sa=N&tab=tq">Photos</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:12}); class=gb2 id=gb_12 href="http://www.google.com/search?hl=en&tbo=u&tbm=vid&source=og&sa=N&tab=tv">Videos</a><div class=gb2><div class=gbd></div></div><a onclick=gbar.logger.il(1,{t:66}); href="http://www.google.com/intl/en/options/" class=gb2>Even more &raquo;</a></div></nobr></div><div id=guser width=100%><nobr><span id=gbn class=gbi></span><span id=gbf class=gbf></span><span id=gbe></span><a target=_top id=gb_70 href="https://www.google.com/accounts/Login?service=&continue=http://www.google.com/patents%3Fhl%3Den&hl=en" class=gb4>Sign in</a><div style="display: none"><div class=gbm id=gbd5 aria-owner=gbg5><div class=gbmc><ol id=gbom class=gbmcc></ol></div></div></div></nobr></div><div class=gbh style=left:0></div><div class=gbh style=right:0></div><div role="alert" style="position: absolute; left: 0; right: 0;"><a href="http://www.google.com/patents/us5871908?hl=en&amp;output=html_text" title="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."><img border="0" src="http://www.google.com/images/cleardot.gif"alt="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."></a></div><div id="guser"><nobr></nobr></div><div style="clear:both;"></div><div id="gb-top-search-box" class="gb-top-search-box-small gb-reset"><table><tr><td class="logo"><a href="http://www.google.com/patents" class="logo-link"><img class="logo-img" src="/intl/en/images/logos/google_logo_41.png" alt="Go to Google Books Home" height="41"/></a></td><td><form action="http://www.google.com/search" name="f" id="vheadf" method="get"><span id="hf"></span><input type="hidden" name="tbm" value="pts"/><input type="hidden" name="tbo" value="1"/><input type="hidden" name="hl" value="en"/><table><tr><td><div class="inputs"><table><tr><td><div class="text-input"><input type="text" name="q" id="vheadq" class="text" maxlength="2048" size="31" value="" title="Search Patents" accesskey="s" autocomplete="off"/><script>window._OC_autoDir &&window._OC_autoDir('vheadq', 'tia-vheadq');</script></div></td><td><div class="submit-input"><input name="btnG" class="submit" type="submit" value=""/></div></td></tr></table></div></td><td class="col-ext-links"><div class="ext-links"><a href="http://www.google.com/advanced_patent_search">&lt;nobr&gt;Advanced Patent Search&lt;/nobr&gt;</a></div></td></tr></table></form></td></tr></table></div><div class="kd-appbar"><h2 class="kd-appname"><a href="/patents">Patents</a></h2><div class="kd-buttonbar left" id="left-toolbar-buttons"><a id="appbar-write-review-link" href=""></a><a id="appbar-view-print-sample-link" href=""></a><a id="appbar-view-ebook-sample-link" href=""></a><a id="appbar-patents-prior-art-finder-link" href="https://www.google.com/patents/related/US5871908"></a><a id="appbar-patents-discuss-this-link" href="http://www.google.com/url?id=aGlJBAABERAJ&amp;q=http://patents.stackexchange.com/redirect/google-patents%3Fpatent%3DUS5871908&amp;usg=AFQjCNFAX85ZIT9Hy25QSRjPJeb25J9DPQ" data-is-grant="true"></a><a id="appbar-read-patent-link" href="//docs.google.com/viewer?url=patentimages.storage.googleapis.com/pdfs/US5871908.pdf"></a><a id="appbar-download-pdf-link" href="//patentimages.storage.googleapis.com/pdfs/US5871908.pdf"></a></div><div class="kd-buttonbar right" id="right-toolbar-buttons"></div></div><div id="books-microdata" itemscope=""itemtype="http://schema.org/Book"itemid="http://www.google.com/patents/US5871908" style="display:none"><span itemprop="description">A process for the qualitative and quantitative determination of at least one in vitro amplified nucleic acid in a sealed reaction chamber, wherein during or subsequent to the amplification of the nucleic acid at least one substance (probe) is present which interacts with the nucleic acid to be detected;...</span><span itemprop="url">http://www.google.com/patents/US5871908?utm_source=gb-gplus-share</span><span class="main-title" itemprop="name">Patent US5871908 - Process for the determination of in vitro amplified nucleic acids</span><img itemprop="image" src="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"alt="Patent US5871908 - Process for the determination of in vitro amplified nucleic acids" title="Patent US5871908 - Process for the determination of in vitro amplified nucleic acids"></div><div style="display: none"><ol id="ofe-gear-menu-contents" class="gbmcc"><li class="gbe gbmtc"><a class="gbmt goog-menuitem-content" id="" href="http://www.google.com/advanced_patent_search">Advanced Patent Search</a></li></ol></div><table id="viewport_table" cellpadding="0" style="clear:both" cellspacing="0"><tr><td id="viewport_td"><div class=vertical_module_list_row><div id=intl_patents class=about_content><div id=intl_patents_v><table class="patent-bibdata"><tr><td class="patent-bibdata-heading">Publication number</td><td class="single-patent-bibdata">US5871908 A</td></tr><tr><td class="patent-bibdata-heading">Publication type</td><td class="single-patent-bibdata">Grant</td></tr><tr><td class="patent-bibdata-heading">Application number</td><td class="single-patent-bibdata">US 08/157,195</td></tr><tr><td class="patent-bibdata-heading">PCT number</td><td class="single-patent-bibdata">PCT/EP1993/000254</td></tr><tr><td class="patent-bibdata-heading">Publication date</td><td class="single-patent-bibdata">Feb 16, 1999</td></tr><tr><td class="patent-bibdata-heading">Filing date</td><td class="single-patent-bibdata">Feb 4, 1993</td></tr><tr><td class="patent-bibdata-heading">Priority date<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The priority date is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the date listed."></span></td><td class="single-patent-bibdata">Feb 5, 1992</td></tr><tr><td class="patent-bibdata-heading">Fee status<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The fee status is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the status or dates listed."></span></td><td class="single-patent-bibdata">Paid</td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Also published as</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/DE4234086A1">DE4234086A1</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP0581953A1">EP0581953A1</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP0581953B1">EP0581953B1</a>, </span><span class="patent-bibdata-value"><a href="/patents/WO1993016194A1">WO1993016194A1</a></span></span></td></tr><tr class="patent-bibdata-list-row alternate-patent-number"><td class="patent-bibdata-heading">Publication number</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value">08157195, </span><span class="patent-bibdata-value">157195, </span><span class="patent-bibdata-value">PCT/1993/254, </span><span class="patent-bibdata-value">PCT/EP/1993/000254, </span><span class="patent-bibdata-value">PCT/EP/1993/00254, </span><span class="patent-bibdata-value">PCT/EP/93/000254, </span><span class="patent-bibdata-value">PCT/EP/93/00254, </span><span class="patent-bibdata-value">PCT/EP1993/000254, </span><span class="patent-bibdata-value">PCT/EP1993/00254, </span><span class="patent-bibdata-value">PCT/EP1993000254, </span><span class="patent-bibdata-value">PCT/EP199300254, </span><span class="patent-bibdata-value">PCT/EP93/000254, </span><span class="patent-bibdata-value">PCT/EP93/00254, </span><span class="patent-bibdata-value">PCT/EP93000254, </span><span class="patent-bibdata-value">PCT/EP9300254, </span><span class="patent-bibdata-value">US 5871908 A, </span><span class="patent-bibdata-value">US 5871908A, </span><span class="patent-bibdata-value">US-A-5871908, </span><span class="patent-bibdata-value">US5871908 A, </span><span class="patent-bibdata-value">US5871908A</span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Inventors</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Karsten+Henco%22">Karsten Henco</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Manfred+Eigen%22">Manfred Eigen</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Detlev+Riesner%22">Detlev Riesner</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Original Assignee</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=inassignee:%22Evotec+Biosystems+Gmbh%22">Evotec Biosystems Gmbh</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Export Citation</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/US5871908.bibtex">BiBTeX</a>, </span><span class="patent-bibdata-value"><a href="/patents/US5871908.enw">EndNote</a>, </span><span class="patent-bibdata-value"><a href="/patents/US5871908.ris">RefMan</a></span></span></td></tr><tr class="patent-internal-links"><td colspan=2><span class="patent-bibdata-value"><a href="#backward-citations">Patent Citations</a> (8),</span> <span class="patent-bibdata-value"><a href="#npl-citations">Non-Patent Citations</a> (8),</span> <span class="patent-bibdata-value"><a href="#forward-citations">Referenced by</a> (67),</span> <span class="patent-bibdata-value"><a href="#classifications">Classifications</a> (10),</span> <span class="patent-bibdata-value"><a href="#legal-events">Legal Events</a> (10)</span> </td></tr><tr><td colspan=2 class="patent-bibdata-external-link-spacer-top"></td></tr><tr class="patent-bibdata-external-link-spacer-bottom"></tr><tr><td colspan=2><span class="patent-bibdata-heading">External Links:&nbsp;</span><span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=aGlJBAABERAJ&q=http://patft.uspto.gov/netacgi/nph-Parser%3FSect2%3DPTO1%26Sect2%3DHITOFF%26p%3D1%26u%3D/netahtml/PTO/search-bool.html%26r%3D1%26f%3DG%26l%3D50%26d%3DPALL%26RefSrch%3Dyes%26Query%3DPN/5871908&usg=AFQjCNHLUi89LDPHq-Jpksgzq1dAnmcIcw">USPTO</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=aGlJBAABERAJ&q=http://assignments.uspto.gov/assignments/q%3Fdb%3Dpat%26pat%3D5871908&usg=AFQjCNG1QXmqOGjykLgK6LBB8r7Y4i09Lg">USPTO Assignment</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=aGlJBAABERAJ&q=http://worldwide.espacenet.com/publicationDetails/biblio%3FCC%3DUS%26NR%3D5871908A%26KC%3DA%26FT%3DD&usg=AFQjCNGI3pRhw9W2vqRdoAgTG05uIj2FDg">Espacenet</a></span></span></td></tr><tr class="patent-bibdata-group-spacer"></tr></table><div class="number-and-title"><span class="patent-title"><invention-title mxw-id="PT54392323" lang="EN" load-source="patent-office">Process for the determination of in vitro amplified nucleic acids</invention-title></span><br><span class="patent-number">US 5871908 A</span></div><div class="patent-section patent-abstract-section"><div class="patent-section-header"><span class="patent-section-title">Abstract</span></div><div class="patent-text"><abstract mxw-id="PA37878215" lang="EN" load-source="patent-office"> <div class="abstract">A process for the qualitative and quantitative determination of at least one in vitro amplified nucleic acid in a sealed reaction chamber,</div>
    <div class="abstract">wherein during or subsequent to the amplification of the nucleic acid at least one substance (probe) is present which interacts with the nucleic acid to be detected;</div>
    <div class="abstract">wherein spectroscopically measurable parameters of said substance (probe) are subject to variation, creating a measurable signal;</div>
    <div class="abstract">wherein the sample to be measured is exposed to the action of a gradient capable of at least partially denaturing nucleic acids;</div>
    <div class="abstract">with detection of the measurable parameter undergoing variation through the action of the gradient; and</div>
    <div class="abstract">the entire amplification reaction, including qualitative and quantitative determination, may be carried out in a sealed reaction chamber (measuring compartment) without intermittent opening,</div>
    <div class="abstract">permitting to automatically operate the diagnostic method of DNA and RNA amplification in qualitative and quantitative fashion on large series of samples.</div>
  </abstract></div></div><div class="patent-section patent-drawings-section"><div class="patent-section-header"><span class="patent-section-title">Images<span class="patent-section-count">(13)</span></span></div><div class="patent-drawings-body"><div class="patent-drawings-carousel"><div class="drawings"><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5871908-1.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5871908-1.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5871908-2.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5871908-2.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5871908-3.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5871908-3.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5871908-4.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5871908-4.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5871908-5.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5871908-5.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5871908-6.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5871908-6.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5871908-7.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5871908-7.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5871908-8.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5871908-8.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5871908-9.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5871908-9.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5871908-10.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5871908-10.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5871908-11.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5871908-11.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5871908-12.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5871908-12.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5871908-13.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5871908-13.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div></div></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img" alt="Previous page"src="/googlebooks/images/kennedy/page_left.png"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img" alt="Next page"src="/googlebooks/images/kennedy/page_right.png"width="21" height="21" /></div></div></div><div class="patent-post-drawings"></div><div class="patent-section patent-claims-section"><div class="patent-section-header"><span class="patent-section-title">Claims<span class="patent-section-count">(43)</span></span></div><div class="patent-text"><div mxw-id="PCLM5353755" lang="EN" load-source="patent-office" class="claims">
    <claim-statement>We claim:</claim-statement> <div class="claim"> <div num="1" class="claim">
      <div class="claim-text">1. A process for the qualitative and quantitative analysis, in a reaction means comprising a sealed reaction chamber, of at least one in vitro amplified nucleic acid in a sample comprising the steps of:<div class="claim-text">including in the sample, during or subsequent to amplification of the nucleic acid, at least one probe being an oligo- or polynucleotide that hybridizes with the nucleic acid, and dye that intercalates with the nucleic acid, or a combination thereof, and which interacts with the nucleic acid to be detected, said probe having a spectroscopically measurable parameter;</div> <div class="claim-text">exposing the sample to the action of a gradient that, at least partially, denatures the amplified nucleic acid in the sample and that effects variation in the spectroscopically measurable parameter of the probe, creating a measurable signal;</div> <div class="claim-text">detecting the measurable signal; and</div> <div class="claim-text">optionally carrying out the amplification reaction and the qualitative and quantitative analysis without opening the sealed reaction chamber.</div> </div>
    </div>
    </div> <div class="claim-dependent"> <div num="2" class="claim">
      <div class="claim-text">2. The process according to claim 1, wherein the spectroscopically measurable parameter of the probe is at least one luminescent or fluorescent dye, and the probe includes a nucleic acid portion, which interacts with the in vitro amplified nucleic acid during the denaturation accompanied by a change in the measurable signal.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="3" class="claim">
      <div class="claim-text">3. The process according to claim 1, wherein the measurable signal is detected (a) using wave length variation, shift in luminescence or fluorescence intensity, variation in fluorescence polarization, variation in excited state lifetime, or a combination thereof, or (b) using the principle of energy transfer, or (c) through a concentration effect.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="4" class="claim">
      <div class="claim-text">4. The process according to claim 1, wherein the spectroscopically measurable parameter includes a plurality of dyes distinguishable from each other spectroscopically.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="5" class="claim">
      <div class="claim-text">5. The process according to claim 4, wherein a laser excites luminescence of the dyes.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="6" class="claim">
      <div class="claim-text">6. The process according to claim 1, wherein the reaction mixture includes at least one co-amplified nucleic acid standard, the sequence of which is homologous to a sequence to be analyzed, with the exception of at least one point mutation.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="7" class="claim">
      <div class="claim-text">7. The process according to claim 1, that includes at least one co-amplified nucleic acid standard having a primer region, the sequence of which is homologous to the primer region of the amplified nucleic acid.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="8" class="claim">
      <div class="claim-text">8. The process according to claim 7, wherein the nucleic acid standard is a natural component of the amplified nucleic acid.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="9" class="claim">
      <div class="claim-text">9. The process according to claim 1, wherein amplification is carried out (a) in free solution or (b) using a primer attached to a solid phase, the amplified nucleic acid hybridizes with the probe, and the analysis is determined either attached to the solid phase or within the free solution.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="10" class="claim">
      <div class="claim-text">10. The process according to claim 7, wherein the probe is is at least one molecule of fluorescent dye linked to a nucleic acid molecule, the sequence of which is identical or homologous to the amplified nucleic acid to be detected or to the co-amplified nucleic acid standard.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="11" class="claim">
      <div class="claim-text">11. The process according to claim 10, wherein the fluorescent dye linked to the nucleic acid molecule is added to the reaction mixture after completing amplification, and is hybridized with the amplified nucleic acid by thermal denaturation with subsequent renaturation.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="12" class="claim">
      <div class="claim-text">12. The process according to claim 10, wherein the fluorescent dye linked to the nucleic acid molecule is added to the reaction mixture prior to completing amplification, and the probe is a non-amplifiable double-stranded RNA or a non-amplifiable chemically modified nucleic acid.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="13" class="claim">
      <div class="claim-text">13. The process according to claim 1, wherein a primer of a primer pair is used for the amplification, which primer encodes a G:C-rich region at the 5' terminus.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="14" class="claim">
      <div class="claim-text">14. The process according to claim 1, wherein the probe is an oligo- or polynucleotide having at least two chemical structural elements, wherein (a) each chemical structural element can be detected, upon interacting with electromagnetic waves, by absorption or emission of radiation and (b) one of the structural elements, upon interacting with electromagnetic waves, can link to another position on the oligo- or polynucleotide.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="15" class="claim">
      <div class="claim-text">15. The process according to claim 14, wherein the chemical structural elements have a chromophoric system.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="16" class="claim">
      <div class="claim-text">16. The process according to claim 15, wherein the chromophoric system luminesces via a dye substituent thereon.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="17" class="claim">
      <div class="claim-text">17. The process according to claim 14, wherein the chemical structural element that can link to another position on the oligo- or polynucleotide is a photochemical crosslinker.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="18" class="claim">
      <div class="claim-text">18. The process according to claim 17, wherein the photochemical crosslinkers are psoralene or a psoralene derivative.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="19" class="claim">
      <div class="claim-text">19. The process according to claim 14, wherein spacing between the two chemical structural elements is between 8 to 12 nucleotide positions.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="20" class="claim">
      <div class="claim-text">20. The process according to claim 1, wherein the reaction means comprises a plurality of recesses in a sheet system, each recess thermally weldable, accommodates ready-for-use reagent mixtures in lyophilized or matrix-bound form, and permits direct optical measurement.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="21" class="claim">
      <div class="claim-text">21. The process according to claim 20, wherein the reagent mixtures are stored in spatially separated matrices, and, subsequent to sealing the reaction chamber, are introduced into the reaction process.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="22" class="claim">
      <div class="claim-text">22. The process according to claim 1, wherein the analysis is effected by microtitration.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="23" class="claim">
      <div class="claim-text">23. The process according to claim 1, wherein the gradient is a time-controlled temperature gradient, and the variation of the spectroscopically measurable parameter is monitored as a function of time, temperature, or time and temperature.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="24" class="claim">
      <div class="claim-text">24. The process according to claim 23, wherein the analysis is by temperature gel electrophoresis, chromatography, or directly in homogenous solution, or a combination thereof.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="25" class="claim">
      <div class="claim-text">25. The process according to claim 24, wherein the presence, number, homology, or combination thereof of the amplified nucleic acid depends on the monitored spectroscopically measurable parameter.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="26" class="claim">
      <div class="claim-text">26. The process according to claim 1, wherein the analysis is effected using a data processing system.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="27" class="claim">
      <div class="claim-text">27. The process of claim 1 wherein the probe is an oligo- or polynucleotide having at least one chemical structural element (a) having a stable bond that, upon interacting with electromagnetic waves, is capable of cleavage and subsequent linkage with the amplified nucleic acid and (b) that can be detected, upon interacting with electromagnetic waves, by absorption or emission of radiation, wherein said structural element is not a purine or pyrimidine substituent of naturally occurring nucleotide components.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="28" class="claim">
      <div class="claim-text">28. The process of claim 27 wherein the chemical structural element having a stable bond is psoralene or a psoralene derivative.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="29" class="claim">
      <div class="claim-text">29. The process of claim 27 wherein the chemical structural element that can be detected luminesces.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="30" class="claim">
      <div class="claim-text">30. The process of claim 27 wherein one of the chemical structural elements is located 8 to 12 nucleotides away from another of the chemical structural elements.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="31" class="claim">
      <div class="claim-text">31. The process of claim 1 wherein the reaction means includes (A) at least one multiple-well-containing sheet, each well being a reaction chamber that includes the probe for and lyophilized amplification reagents and (B) a sealing sheet cooperating with the multiple-well-containing sheet in a manner independently sealing each reaction chamber with a seal that becomes an interior surface of the reaction chamber.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="32" class="claim">
      <div class="claim-text">32. The process of claim 31 wherein the reagents are present in at least one water-soluble matrix.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="33" class="claim">
      <div class="claim-text">33. The process of claim 32 wherein the matrix includes a stabilizer.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="34" class="claim">
      <div class="claim-text">34. The process of claim 32 wherein the matrix includes a sugar.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="35" class="claim">
      <div class="claim-text">35. The process of claim 32 wherein the matrix includes trehalose or saccharose.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="36" class="claim">
      <div class="claim-text">36. The process of claim 32 wherein the reagents include amplification primers, buffer components, at least one polymerase, and co-factors.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="37" class="claim">
      <div class="claim-text">37. The process of claim 31 wherein the reagents include amplification primers, buffer components, at least one polymerase, and co-factors.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="38" class="claim">
      <div class="claim-text">38. The process of claim 31 wherein at least one reaction chamber of the well-containing sheet includes a reagent/probe-containing matrix and the chamber interior surface of the corresponding seal includes hybridization reagents.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="39" class="claim">
      <div class="claim-text">39. The process of claim 31 wherein the reaction means is composed of kit systems.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="40" class="claim">
      <div class="claim-text">40. The process of claim 1 including computer-controlled, time-dependent thermostating of the reaction chamber.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="41" class="claim">
      <div class="claim-text">41. The process of claim 1 including optical-excitation-effecting emitting of a fluorescence signal and optical detection of the fluorescence signal.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="42" class="claim">
      <div class="claim-text">42. The process of claim 41 wherein the excitation is by a laser.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="43" class="claim">
      <div class="claim-text">43. The process according to claim 13, wherein said primer has from 15 to 20 G:C residues at the 5' terminus.</div>
    </div>
  </div> </div></div></div><div class="patent-section patent-description-section"><div class="patent-section-header"><span class="patent-section-title">Description</span></div><div class="patent-text"><div mxw-id="PDES67245698" lang="EN" load-source="patent-office" class="description">
    <p>This invention is directed to a process for determining at least one in vitro amplified nucleic acid in a reaction chamber, a device suitable for operating the process of the invention, and means for operating the process of the invention.</p>
    <p>Increasingly, modern analytics returns to basic principles of molecular biology. Very recently, the polymerase chain reaction (PCR) was found to be a very useful tool for analytics utilizing basic principles of molecular biology, and is applicable to all the fields of analytics where nucleic acids play a direct or indirect role.</p>
    <p>In principle, PCR analytics  R. K. Saiki et al. Science 239, 487-491 (1988)! and other enzymatic amplification techniques  J. C. Guatelli et al., Proc. Natl. Acad. Sci. 87, 1874-1878 (1990)! are able to detect low titers of DNA or RNA copies in aqueous solution. Ultimately, a single copy is sufficient for enzymatic amplification. However, a number of problems have shown in routine qualitative application as well as in the efforts to combine PCR with a quantitative copy number determination. The difficulties of amplification analytics are associated with:</p>
    <p>undesirable amplification by mispriming;</p>
    <p>non-uniform primer consumption;</p>
    <p>heteroduplex formation;</p>
    <p>varying cycle efficiencies;</p>
    <p>various amplification artifacts;</p>
    <p>problems in quantification;</p>
    <p>risk of contamination;</p>
    <p>"erroneously positive" results;</p>
    <p>"erroneously negative" results;</p>
    <p>cost of testing;</p>
    <p>readiness for application in series;</p>
    <p>expenses in handling as compared to the immunological ELISA process for protein analytics.</p>
    <p>WO 91/02815 describes the reliable qualitative and quantitative detection of specific DNA and RNA from biological sample material using a DNA/RNA amplification method in combination with, e.g., temperature gradient gel electrophoresis  cf., K. Henco &amp; M. Heibey Nucleic Acids Res. 19, 6733-6734 (1990); J. Kang et al. Biotech. Forum Europe 8, 590-593, (1991); G. Gilliland et al. Proc. Natl. Acad. Sci. 87, 2725-2729 (1990)!.</p>
    <p>By way of the amplification strategy described in WO 91/02815 it has been accomplished to combine the sensitivity of the amplification techniques with reliable and extremely precise quantification (variation 15%). The method allows for controlling or overcoming the above-mentioned problems as are occurring with amplification processes. Using a nearly ideal internal standard of a defined number of copies, which standard differs from the template to be determined in only one single base position, amplification may be carried out with the initial ratio of template and standard being maintained  K. Henco and M. Heibey, Nucleic Acids Res. 19, 6733-6734 (1990)!. Template and standard are made subsequently in a temperature gradient process using a time- or space/temperature gradient in the gel electrophoresis system. The ratio of template and standard may be determined in a simple fashion by hybridizing the reaction mixture with a radioactive-labeled or fluorescent-labeled standard. Heteroduplex formation, in other cases occurring interferingly, cannot distort the results in such system. Rather, heteroduplex formation is utilized for the actual measurement.</p>
    <p>Usual problems caused by non-uniform consumption of primers, mispriming reactions, saturation effects, and variations in amplification efficiency do not have any negative influence on the results of quantification.</p>
    <p>This technology has been applied to major analytical issues in medicine. Thus, for example, it has become possible to quantifyingly detect cytomegalic infections or viremia in transplanted patients or newborn children. Here, the process has proven particularly worthwhile since, in view of infection rates in the European countries of more than 90% here and there, the sole detection of cytomegalic viruses is of minor importance if not a titer determination is carried out simultaneously indicating the acute condition of viremia. With viral diseases, the quantifying aspect is gaining significance, particularly in context with the increasingly applied models of antiviral therapy. Frequently, antiviral therapies, as for instance in the case of HIV using AZT, are systemically extremely straining for the patient and, as an antiviral therapeutic agent, can only be employed for a limited period of time. In future, therefore, it will be very important to have an economic and simple technique allowing for both titer determination or determination of virus gene activity and indicating therapy-induced drift to resistant virus strains.</p>
    <p>Having from small to low sample numbers (10-20), hitherto applied gel electrophoretic methods for separating the marked homoduplexes and heteroduplexes have proven extraordinarily efficient and correct with respect to the results supplied. However, a severe drawback is the relatively time- and personnel-intensive analytical technique making it difficult to employ this very profound analytical technique as a general diagnostic method for DNA and RNA analytics. Using the temperature gel electrophoretic technique, it is hardly possible to perform sample numbers&gt;50 per operating person and day. Likewise, automatization is but difficult to achieve.</p>
    <p>Thus, it is an object of the invention to provide a process which avoids the above-mentioned disadvantages of prior art processes and, more specifically, to design a detection process for nucleic acids in such fashion that gel electrophoretic separation is not required, in order to ensure a more simple and automatable procedure. At the same time it is intended to create a device allowing for reliable and simple observation of the detected amplified nucleic acids using the process of the invention, where the device is to allow automatable evaluation to the largest extent.</p>
    <p>According to the invention, this problem is solved by a process for the qualitative and quantitative determination of at least one in vitro amplified nucleic acid in a sealed reaction chamber,</p>
    <p>wherein during or subsequent to the amplification of the nucleic acid at least one substance (probe) is present which interacts with the nucleic acid to be detected;</p>
    <p>wherein spectroscopically measurable parameters of said substance (probe) are subject to variation, creating a measurable signal;</p>
    <p>wherein the sample to be measured is exposed to the action of a gradient capable of at least partially denaturing nucleic acids;</p>
    <p>with detection of the measurable parameter under-going variation through the action of the gradient; and</p>
    <p>the entire amplification reaction, including qualitative and quantitative determination, may be carried out in a sealed reaction chamber (measuring compartment) without intermittent opening.</p>
    <p>The substance (probe), the parameter of which is to be detectable spectroscopically, preferably contains at least one fluorescent residue, preferably having intercalating properties, and a nucleic acid proportion. The interaction with the in vitro amplified nucleic acid as a function of the denaturation condition is accompanied by a change in the spectroscopically measured signal. This, for example, may take place by intercalation of the dye into the nucleic acid double helix or by dilution or concentration effects within the measuring compartment.</p>
    <p>In another preferred embodiment of the process according to the invention, the nucleic acid denaturation process initiated by the gradient is detected using wave length variation and/or shift in fluorescent intensity and/or variation in excited state lifetime, or using the principle of the so-called energy transfer (Forster Transfer), or via concentration effects, or using various, preferably hydrophobic interactive properties of the labeled probe.</p>
    <p>Likewise, the process of the invention permits simultaneous or sequential detection of multiple different amplified nucleic acids. This is effected by using a multiplicity of dyes which may be distinguished from each other spectroscopically, which permit to analyze the various amplified nucleic acids and/or through which at least one independent calibrating substance is introduced. This, in particular, is possible where the various nucleic acids to be analyzed interact with differently labeled participants in hybridization.</p>
    <p>Registration of the measuring signal is conducted, for instance, by measuring the fluorescence generated by the dyes which, in particular, may be excited continuously or in pulses by a laser.</p>
    <p>A further preferred embodiment of the process according to the invention is based on that the amplified nucleic acids contain at least one co-amplified nucleic acid standard, the sequence of which is homologous to a sequence to be determined and preferably identical, however, with the exception of at least one point mutation which, in particular, lies in a sequence region of lowest stability. However, care must be taken that the point mutation lies outside the primer binding sites if enzymatic amplifications are performed. The nucleic acid standard may also be a natural component of the nucleic acid to be analyzed.</p>
    <p>According to the invention, it is also possible to observe successful amplification of a specific nucleic acid without adding a labeled standard fragment to the reaction batch after amplification has taken place. In this preferred procedure of the invention, specifically those primers required for amplification are employed (EP-A-0,469,755) which then hybridize at the corresponding sites in the sequence of the nucleic acids in question. However, the corresponding sequences between the primer sites may be so different that when passing the temperature gradient, both sequences--the amplified test and standard sequence--denature separately, and preferably, denature in co-operative fashion.</p>
    <p>This allows use of sequences having a sequence deviation to such extent that heteroduplex formation is no longer possible. Thus, it is no longer necessary to add a labeled standard fragment after amplification has taken place. Different melting temperatures of both sequences may be influenced, for example, by greatly varying the length of the sequence or by selecting a poly-A/T-sequence. The question whether an amplification reaction has taken place may then be decided by employing the process of the invention, for example, in a temperature gradient with simultaneous presence of ethidium bromide, where quantitative detection is likewise possible.</p>
    <p>The process according to the invention permits to perform the amplification in homogenous phase or on a solid phase, preferably using a primer attached to a solid phase, to the extended sequence of which the labeled probe can hybridize. Thus, the concentration of the probe can be determined either specifically at the solid phase support or within the free solution. Preferably, at least one molecule of fluorescent dye is linked to a nucleic acid molecule, the sequence of which is identical or homologous to the nucleic acid to be detected or to the co-amplified nucleic acid standard.</p>
    <p>Once the nucleic acid molecule with the fluorescent dye linked thereto has been added to the reaction mixture after amplification has taken place, hybridization with the amplified nucleic acids is effected, preferably by a thermal denaturation step with a subsequent renaturation step. According to the invention, however, it is also possible to add the nucleic acid molecule having the linked fluorescent dye to the reaction mixture before amplification has taken place. Here, the probe is to be added as a non-amplifiable double-stranded RNA or as a non-amplifiable chemically modified nucleic acid.</p>
    <p>For nucleic acid amplification, a possible embodiment uses a primer of the primer pair employed for amplification, which primer contains a G:C-rich region at the 5' terminus, for example from 15 to 20 G:C residues.</p>
    <p>The standardization and quantification of the process according to the invention by the above preferred embodiments of the, for instance, fluorescent probes as described so far, involves a tolerable limitation. Thus, the probes used for standardization may be added only subsequent to effected amplification. This means that initially, during the amplification reaction they must be stored spatially separated from the amplification process. Likewise, it may be advantageous for various applications to provide probes used as standard nucleic acids and differing in more than one position from the nucleic acid to be determined. In order to improve the signal/noise ratio in the following determination using the employed probe, it is desirable not to be forced to add deficient probe to the mixture to be amplified.</p>
    <p>Thus, in a particularly preferred embodiment of the process according to the invention, as the probe there is used an oligo- or polynucleotide as a single-stranded nucleic acid which, however, cannot participate in the amplification reaction because of chemical modification. Only by suitable manipulation following the amplification reaction, the single-stranded probe is exposed and may then hybridize with the corresponding nucleic acids to be determined. Thus, for example, the probe, if present in the form of a single-stranded nucleic acid, may be inactivated in the form of a "hairpin structure" and may thus be prevented from participating in the amplification reaction.</p>
    <p>The oligo- or polynucleotides to be used as probes in a particularly preferred fashion have one or more structural elements with at least two chemical substituents, each being capable of interacting with electromagnetic waves, with cleavage or linkage of stable bonds, or by absorption or emission of radiation. As the substituent particularly suitable for interacting with electromagnetic radiation, with cleavage and linkage of stable bonds, specifically covalent bonds, psoralene or its derivatives have proven successful. As the structural elements serving as the actual markers of the probe, particularly luminescent dyes such as fluorescent dyes having high quantum yield such as dyes from the thiazole orange class have proven beneficial. Preferable are dyes having large Stokes shift which, dependent on hybridization condition, alter the luminescent properties.</p>
    <p>It may be advantageous that the spectra of the structural elements at the respective sensitive sites which, on the one hand, are to be excited for cleavage and linkage of, for instance, covalent bonds or, on the other hand, are to be regarded as absorption or emission maxima, are apart so far that each excitation will not interfere with the function of the other structural element.</p>
    <p>Likewise, it is possible to combine both chemical structures in a single chemical structure if linkage and cleavage of bonds each occur at different wave lengths such as possible maximum fluorescence of this structure. Thus, the respective functions, probe fixation within a non-amplified structure on the one hand, and spectroscopic identification of said structure on the other hand, cannot interfere with each other.</p>
    <p>Where two separated structural elements have said separated functions it was found beneficial that they should not fall below a distance of at least 10 nucleotides on the oligo- or polynucleotide strand.</p>
    <p>The preferred embodiment of the process according to the invention using the oligo- or polynucleotides of the invention preferably functions in such fashion that the probes are added to the above-described mixture of substances, the amplification reaction is carried out as described, and subsequently, induced by radiation for example, the masked probe is released to hybridize with the amplification product to be determined and, as described, to be detected by a time/temperature gradient in homogenous solution, by temperature gel electrophoresis, or by a chromatographic process, for example.</p>
    <p>Using this variant of the process according to the invention, it is possible to perform the analysis in such way that the hybridization labeled probe no longer has to be located separately within the compartment of the reaction chamber in order to add it to the reaction mixture immediately prior to the actual analysis. Thus, it is possible to provide the labeled probe with the other reagents in a form in which it cannot participate in the subsequent amplification process. Thereby, it is no longer necessary to separate amplification mixture and probe within the measuring chamber.</p>
    <p>In order to operate the process, sheet-like systems having hollows or recesses serving as reaction chambers (compartments) are preferably used. Preferably, the sheet systems are thermally weldable and suited to accommodate ready-for-use reagent mixtures in freeze-dried or matrix-bound form. Furthermore, direct optical measurement of the reaction chamber contents is possible. Hence, the sheet material is transmittant or transparent at least for specific wave length regions of electromagnetic radiation. Preferably, the reagents needed to perform the process according to the invention are stored in spatially separated matrices, and subsequent to sealing the reaction chamber, are introduced into the reaction process. Preferably, the reaction chambers are separated from each other at a distance of the holes in a commercially available microtitration plate. This has the advantage that equipment suitable for processing microtitration plates may be used in the technology described according to the invention.</p>
    <p>In order to analyze a mixture of amplified nucleic acids, preferably a time-controlled temperature gradient is applied after addition of substances needed in the reaction, and the denaturation behavior of the nucleic acids is measured. This is done through the variation of spectroscopic parameters of the substance interacting with the nucleic acid. Variation of the spectroscopic parameter is monitored over time or in equivalent fashion as a function of temperature change.</p>
    <p>Evaluation of the function of variation in spectroscopic behavior of the substance interacting with the nucleic acid permits to conclude the presence or number or homology of an examined nucleic acid with the corresponding standard. Preferably, evaluation of these data is done on-line using a data processing system.</p>
    <p>In addition to eliminating the above-mentioned drawbacks in prior art, the process according to the invention is advantageous in that amplification of nucleic acids and subsequent analytics may be carried out in a single hermetically sealable reaction compartment. Thereby, disposal of these biological materials without opening the compartments is possible, and a potential source of contamination is eliminated. Furthermore, such procedure also permits storage of test sheets of the above-mentioned type in sealed condition over prolonged periods of time so that archiving of the often valuable substances is made possible. However, storage preferably is done in frozen condition. Likewise, the process according to the invention advantageously permits the experiments to be repeatable, optionally at a later time even after prolonged interim storage, or the amplified mixture to be preparatively processable and analyzable.</p>
    <p>The device for performing the process of the invention has a means for time-dependent thermostatting of the reaction chambers to be used in the process. Preferably, the thermostatting is controlled by a programmable unit. The read-out unit of the device according to the invention preferably consists of an optical unit capable of registering photons. Particularly preferred are such units which are suitable for registering emitted fluorescent light. Likewise contemplated are equipments capable of detecting other spectroscopic properties such as nuclear spin or electron spin etc. which can be correlated to conformational changes of the nucleic acid double-helix or other structural variables, or the use of chromatographic procedures. Using the method of hydrophobic interaction chromatography, molecules having hydrophobic ligands, as represented by partially denaturing structures of the substances to be analyzed, may be separated from the duplexes.</p>
    <p>The device for operating the process of the invention is capable of accommodating a means for operating the process which is assembled of a system of reaction compartments, preferably a sheet system with ready-to-use reagents in freeze-dried form. Preferably, the reaction compartments are arranged in microtitration form. Preferably, the reagents of the means for operating the process are fixated and/or stored in at least one water-soluble matrix. Preferably, the matrix contains stabilizers such as sugars, particularly trehalose or saccharose. Preferably, the means for operating the process of the invention comprises reaction compartments and/or other reagent reservoirs, amplification primers, buffer components, and at least one polymerase and usual co-factors for performing the amplification reaction. In another preferred embodiment of the means for operating the process of the invention, the reaction chamber or reaction compartment is provided with an additional separate reagent reservoir in a matrix located within the sheet sealing the compartment. Here, preferably, the labeled probe with the buffer substances required for hybridization are stored.</p>
    <p>Preferably, the means for operating the process of the invention is arranged in kit systems comprising reaction vessels such as sheet systems with storable and directly usable reagent mixtures, where is merely necessary to charge the reaction vessels with the sample to be analyzed which then in hermetically sealed condition is subjected to an amplification procedure and subsequent analysis.</p>
    <p>The process according to the invention is particularly suitable for analyzing mixtures of substances, preferably nucleic acids where at least one component in the temperature region of the time/temperature gradient is subject to thermal conversion. By adding and co-amplifying a standard having precisely known number of copies the process of the invention can be standardized and permits quantitative statements about the amplified nucleic acids of the sample to be examined. Using the process of the invention it is possible, for example, to detect mutations, point mutations, deletions, insertions, and rearrangements within the DNA/RNA nucleic acid. Using the quantitative analysis it is also possible to determine the concentrations of such changes in the nucleic acid. The samples may be derived from most various materials such as live, dead, fossil tissue as well as tissue no longer active in metabolism in vivo, or from body fluids, from in vitro cell cultures, or from environmental samples. The process according to the invention allows for qualitative and quantitative detection of cellular genes and genes of infectious pathogens directly or via their RNA gene products as a wild type sequence or as variants.</p>
    <p>Moreover, the process of the invention may also be employed for the examination and determination of potentially toxic substances or potential pharmaceutical agents or chemical or biological pesticides by examining their effect on nucleic acids or their amplifications in cellular or non-cellular systems.</p>
    <heading>BRIEF DESCRIPTION OF THE DRAWINGS</heading> <p>FIG. 1</p>
    <p>FIG. 1 is a schematic description of a preferred procedure of the process according to the invention. In a receiver of reaction compartments in the form of a plate for microtitration, the reagents required for specific amplification of a nucleic acid are located, for example, in lyophilized form, including the probe. Merely the sample to be analyzed, for example, in the form of an aqueous solution, is added prior to the amplification reaction. A homogenous solution is prepared.</p>
    <p>Subsequently, the reaction compartments are sealed with a second sheet, with the second sheet containing at least one further matrix with reagents not participating in the actual amplification reaction. The sheet is positioned in a thermostatting block in order to carry out the enzymatic amplification reaction. Here, amplifications may be performed both at homogenous temperature and in periodically varying temperature cycles (PCR). Subsequent to amplification, the reaction mixture is contacted with the second reagent reservoir, and a homogenous solution is prepared. Following performance of a denaturation/renaturation process, an optical detection system records the laser-induced luminescence (fluorescence, phosphorescence) as a function of a linear temperature gradient which is time-controlled via the thermoblock. The initial temperature may be, e.g., 5 C. minimum, the final temperature 100 C. maximum.</p>
    <p>FIG. 2</p>
    <p>The illustration schematically shows the course of the process according to the invention using the example of intercalating dyes on a molecular level, which are not fixated to a nucleic acid probe (e.g., ethidium bromide or thiazole orange dyes). These dyes intercalate under native conditions between adjacent base pairs in double-stranded DNA or RNA. In intercalated condition, the fluorescence yield increases up to 20 fold and the lifetime of the excited state about 10 fold. If such system is subjected to a thermal gradient, the thermodynamically most unstable regions of the nucleic acid helix begin to denature initially. Mispairing as generated in heteroduplex formation destabilizes the corresponding sequence region and results in premature opening of the latter. Dye molecules initially bound in this region will be released, resulting in a decrease of the overall fluorescence signal. Here, the dye concentration is to be selected such that free dye and bound dye are in thermodynamic equilibrium and free dye is present in significant excess. Only at higher temperatures the corresponding sequence region of the homoduplex will open, giving rise to further stepwise decrease of the fluorescence signal. From the intensity ratio of both steps, the relative ratio of amounts of homoduplex and heteroduplex can be determined.</p>
    <p>FIGS. 3a and 3b</p>
    <p>FIG. 3a illustrates the design of a standard and a fluorescent-labeled probe, respectively, in a preferred embodiment. The standard should differ in at least one base position from the nucleic acid to be detected. Preferably, this mutation is located in the sequence region of lowest thermodynamic stability and may readily be introduced using primer P3. However, the mutation is to be located outside the primer binding sites of P1 and P2. A primer (P2) may contain a GC-rich region at the 5' terminus, while the other primer (P1) preferably is chemically linked through a spacer to a fluorescent dye which may intercalate in the region of the double helix. The application principle of the fluorescent-labeled probe is illustrated schematically in FIG. 3b.</p>
    <p>FIG. 4</p>
    <p>In a preferred embodiment of the process according to the invention, primers are used as described in Thuong, N. T. &amp; Chassignol, M. Tetrahedron Letters 28, 4157-4160 (1987); Thuong, N. T. et al., Proc. Natl. Acad. Sci. U.S.A. 84, 5129-5133 (1987); Helene, C., in DNA-Ligand Interactions, Plenum Publishing Corporation, 127-140 (1987); W. Gushelbauer &amp; W. Saenger, Ed. Some of them are already commercially available (Appligene, Strabourg, France). They contain a spacer-linked fluorescent dye capable of intercalating if the primer is located in a double-helical region. Upon thermal denaturation of the double-helix the fluorescent properties of the dye are modified.</p>
    <p>FIG. 5</p>
    <p>This figure schematically shows a preferred preparation of the probes used in the process of the invention, which are modified and do not participate in the amplification reaction.</p>
    <p>The European patent applications EP 0,469,755 and EP 0,379,369 describe the generation of nucleic acid structures. Therein, a structure is generated in representative fashion, the ends of which are complementary to each other. For this purpose, primers p1 and (p1) are employed initially. To prepare the probes, an amplified product is prepared from the corresponding amplified product of the standard using a primer p1 containing the structural element capable of forming stable bonds ("chemical clamp").</p>
    <p>Preferably, p1 is selected such that a restriction site having a 5' overhanging end is produced at the 3' side. Thus, following restriction a structure is formed which may be filled up with nucleotides which in turn bear, for instance, a chromophor which is the actual marker of the probe. According to this preparation process, since both strands contain the dye marker and the "chemical clamp", separation of strands and recovery are necessary. However, this does not cause any problems and can be accomplished, for example, by using preparative high pressure liquid chromatography.</p>
    <p>Chemical linkage of the complementarily hybridizing ends of the single-stranded hairpin structure is then induced photochemically or chemically, thus preventing template activity of this probe during the phase of enzymatic amplification likewise illustrated in FIG. 5. The probe present in the form of a hairpin loop cannot be involved in the amplification due to its masked structure. Following amplification, for example, when using psoralene derivatives as the "chemical clamp", clamping may be re-opened in the short-wave UV region. Thus, the probe single-strand is released in the denaturation/renaturation process and may be re-linked by repeated irradiation at 360 nm if psoralene derivatives are used. This situation is illustrated schematically in FIG. 6.</p>
    <p>If it is possible to induce chemical linkage to the complementary strand by radiation again, this chemical linkage attains a further advantageous function. It can assume the function of the previously used G/C clamp and thus ensure that the hybridized strands during analysis are no longer able to separate from each other, thus enabling irreversible denaturation. As mentioned before, compounds having the psoralene basic structure are preferably used as the "chemical clamp", which possess the property of reacting covalently with pyrimidine bases on activation with light having a wave length of 360 nm. However, there is the proviso that the pyrimidine bases be positioned on the opposite strand, shifted by one base position  J. E. Hearst, Ann. Rev. Phys. Chem., 39, 291-315 (1988)!.</p>
    <p>Another alternative in the synthesis of the probe preferred according to the invention is shown in FIG. 7. Here, the "chemical clamp" is introduced via primer (p1). Thus, only the desired strand will later bear the chemical crosslinker. The crosslinker which, in particular, can be activated photochemically, advantageously acts to inactivate the adjacent restriction site of the restriction enzyme R1. In this fashion, it is ensured that subsequently, the dye providing the actual labeling of the probe will be incorporated on the desired strand only. Compared to the above-described probe construction, the somewhat longer primer (p1) with modification is unfavorable.</p>
    <p>A third possible preferred embodiment of the nucleic acid probe according to the invention is obtained by using a single-stranded probe likewise having a crosslinked ligand, the position of which, however, being located at the 3' end, while the labeling dye molecule is incorporated at the 5' end. This situation is illustrated in FIG. 8. In this case, the probe is constructed such that the primer binding site is missing at the 3' end, the 5' end is identical with the p1 primer sequence, and the 3' end cannot be extended enzymatically, for example, by incorporating a dideoxy-nucleotide at the terminus. The crosslinking agent spatially separates the homologous double-strand region having the marker structural element from the hybrid single-strand region not bearing same. If the single-strand region is not uniformly synthesized as a consequence of an inaccurate amplification process, this will not adversely affect the result of probe construction.</p>
    <p>Following this approach of probe construction, the probe will not participate in the amplification process because it is not extendable at the 3' end and thus, even in case of intermediate hybridization with amplified counter-strand, no elongated primer binding site will be synthesized enzymatically.</p>
    <p>Thus, following amplification the crosslinking ligand does not have to be liberated initially from a crosslinking structure by irradiating to be subsequently re-crosslinked with amplified product in the re-hybridized structure; a single irradiation procedure is sufficient to cause crosslinking with the amplified product.</p>
    <p>FIGS. 9a-e</p>
    <p>FIGS. 9a-9e schematically illustrate possible experimental results which may be obtained using the technique of the invention. At the ordinate, the fluorescence signal I is plotted in relation to the respective intensity (I<sub>0</sub>) at the lowest experimental temperature (native). The x-axis describes the course of temperature in the heat-up process which preferably is linear over time. The two dotted lines mark the position of the standard homoduplex (A2) as well as the position of heteroduplex (A1) consisting of labeled probe and the wild type sequence (1) to be expected. In case the sequence to be analyzed bears at least one further mutation, stepwise decline of the fluorescence signal occurs already at lower temperature which is characteristic for the mutation (B3). The relative height of the steps to each other (1/2, 3/4, 6/7/8) directly represents the relative concentration ratio of the involved molecules. A result as in FIG. 9c is obtained when only the standard nucleic acid is amplified and the biological sample does not contain any corresponding nucleic acid or merely small amounts of same. A result as in FIG. 9d is obtained when the amplification reaction has proceeded improperly and not even the standard has been amplified. In case the biological sample contains more than one homologous nucleic acid species, this will give rise to several steps of the temperature-dependent course of fluorescence (6 and 7) which are different from each other (FIG. 9e).</p>
    <p>FIGS. 10a and 10b</p>
    <p>FIGS. 10a and 10b describe the schematic course of temperature and the related single steps in the process according to the invention. In FIG. 10a, the course of analysis using the PCR technique is illustrated, in FIG. 10b, the course of an amplification at homogenous Temperature (37 C.), e.g., in the 3SR technique (see below), is given.</p>
    <p>1) Addition of biological sample to the lyophilized amplification batch and sealing the reaction compartment.</p>
    <p>2) Amplification at homogenous temperature or with use of temperature programs (PCR).</p>
    <p>3) Admixing the labeled probe(s) in the hybridization buffer.</p>
    <p>4) Denaturation step at 98 C.</p>
    <p>5) Probe re-association with the amplified DNA.</p>
    <p>6) Time/temperature gradient with optical on-line control.</p>
    <p>FIG. 11</p>
    <p>FIG. 11 schematically illustrates a possible embodiment of the process of the invention for automatized large-scale series. The assay kit includes the sheet components with the storable and ready-to-use reagents. The bar coding permits definition of assay type and allocation of positions. The write-on magnetic strip then is available for sample-specific data. Following analysis, the test sheets may be disposed off without opening or may be archived for possible further analyses.</p>
    <p>FIG. 12</p>
    <p>The schematic representation illustrates a special embodiment of the process according to the invention. A primer used for amplification is fixated to the surface of, e.g., magneto beads. For the purpose of amplification and hybridization, magneto beads can be maintained in the form of a suspension by a magnetic field; for the purpose of laser fluorescence observation, however, during the dissociation process induced by the temperature gradient they may be withdrawn from the solution and fixated at a defined spot. Thus, the laser beam may be directed directly to the particle surface, and the fluorescence during dissociation of the probe may be monitored specifically. This process succeeds, e.g., with use of a single melting domain and permits use of oligomeric, non-intercalating fluorescent dyes as optical markers.</p>
    <p>Reaction and analysis may be carried out in a single reaction compartment. Preferably, a microtitration form is used which permits to simultaneously subject 96 samples or portions of 96 samples (strips of 8 or 16) to analytics. Preferably, the reaction vessel is constructed such that it has volumes of from 20 to 100 l. Instead of single reaction vessels, sheets are preferably used which have hollows or recesses accommodating the samples. Per se known sheets (PCT/EP 89/01320, PCT/EP 89/01387, PCT/DE 91/0082, PCT/DE 91/00081, PCT/DE 91/00083) are specifically suitable for the process of the invention since they are thermostattable with particular efficiency, cheap in production and without problems in environmentally compatible disposal. The use of optically lucid sheets for the visible light region permits online registration of fluorescence signals of commercially available fluorescent dyes.</p>
    <p>The sheets may be charged with generally required reagents (enzymes, primers, buffers, stabilizers, etc.) and preserved for long periods of time in lyophilized condition. Preferably, trehalose or saccharose are used as the stabilizers. Following addition of the samples to be analyzed, the reaction vessels arranged in series may be sealed hermetically. This may be effected by fusing a cover sheet with the sheet bearing the reaction compartments. Similarly, sheets can be used which are coated with a thermoplastic polymer and can be fused at below the melting temperature of the actual support sheets. In the cover region of the support sheet, above the reaction compartments, reagents may be fixated or placed in compartments, which are not to participate in the reaction process from the beginning. In the case of the invention this is, e.g., the specifically labeled probe which is lyophilized and stabilized in a buffer mixture required subsequent to amplification reaction for hybridizing the amplification product and the labeled probe. Subsequent to sealing the reaction compartments, the amplification reaction takes place at homogenous temperature (3 SR, Self-sustained Sequence Replication; TAS, Transcription based amplification system)  J. C. Guatelli et al., Proc. Natl. Acad. Sci. 87, 1874-1878 (1990); Kwoh, D. Y., Davis, G. R., Whitfield, K. M., Chapelle, H. L., Dimichele, L. J. &amp; Gingeras, T. R. Proc. Natl. Acad. Sci. U.S.A. 86, 1173-1177, (1989)! or in a thermocycler as a polymerase chain reaction (PCR). In this step, standard and template are amplified at a constant ratio so that the reaction final product contains ca. from 100 ng to 1 g of amplified nucleic acid.</p>
    <p>In a possible, technically very simple embodiment (FIG. 2), the reaction compartment includes a luminescent dye, preferably a fluorescent dye, preferably having intercalating properties, binding at multiple positions in double-helical structures and having modified spectroscopical properties in the bound state. When in the analysis to follow, using the time/temperature gradient, the corresponding double-stranded structures are denatured, the dye is liberated again. This process is recorded spectroscopically. However, a necessary condition for this procedure is to select the concentration of the free dye such that it is greater than the number of free binding sites. On the other hand, liberation of the dye from a double-strand region results in binding in another structure without change in fluorescence signal.</p>
    <p>In another preferred embodiment, the aqueous solution of the reaction compartment is contacted with a fluorescent dye-labeled probe preferably stored in compartments at the sealing sheet to dissolve the probe (FIGS. 3 and 4). Following denaturation and renaturation, the probe is monitored by fluorescence spectroscopy. At low temperatures the probe is in the double-stranded hybrid with amplified standard (homoduplex) and amplified template (heteroduplex). Now, the reaction compartment is heated over time, preferably in linear fashion, where initially, the heteroduplex is denatured partially or completely and subsequently, the homoduplex is denatured partially or completely. From the relative ratios of the denaturation signals measured over the steps of decrease in fluorescence (FIGS. 9a to 9e), the template titer may be calculated precisely.</p>
    <p>Preferably, a fluorescent marker is used as the denaturation signal. Particularly suitable for this purpose are fluorescent dyes possessing the property of strongly fluorescing only when incorporated between base pairs (intercalation). If such dyes are released from a double helix due to a denaturation process, this can be registered by a change in fluorescence intensity (decrease in fluorescence). Where double helices have different stabilities as is the case with homoduplex and heteroduplex, the signal changes will occur at different temperatures and may be analyzed and evaluated separately. Furthermore, the precise denaturation temperature reflects possible differences in sequence as an indicator for so-called virus drifts which may occur due to mutations.</p>
    <p>Occasionally, intercalating dyes have another favorable property relating to the excited state half-life. In the case of ethidium bromide, the half-life of the excited state is greater by more than 10 fold if the fluorescent molecule is in the intercalated state. Thereby, a pulsed laser excitation may be used where in the subsequent phase of fluorescence emission, the fluorescence intensity can be measured without the influence of scattered light from light used for excitation. Using the process according to the invention, it is possible to employ dyes such as ethidium bromide, preferably at low concentrations (preferably from 10<sup>-10</sup> to 10<sup>-7</sup> M).</p>
    <p>Apart from direct fluorescence intensity measurement, a so-called Forster Transfer (energy transfer) between closely adjacent fluorophores or the parameter of fluorescence polarization may also be used according to the invention.</p>
    <p>High specificity of the process according to the invention is achieved when using a probe covalently linked to one or more dye molecules (FIG. 4). Preferably, this is achieved by using a primer in the preparation of the probe, which terminally or internally bears at least one dye molecule. Only if the probe is in the state of base pairing of homoduplex or heteroduplex, maximum fluorescence intensity of the fluorescent dye intercalated in the double-strand is obtained. The fluorescence intensity of the reaction batches conducted in parallel may be observed simultaneously by using a camera. Single evaluation of the channels results in a fluorescence development as is representatively illustrated in FIG. 9. The relative heights of the fluorescence variations may be converted directly to copy numbers by a computer connected on-line.</p>
    <p>In a further preferred embodiment of the process according to the invention, the reaction is carried out within the compartment, with partial coupling to a solid phase support (FIG. 12). The advantage of this procedure is founded thereon that by using laser optics it is possible to selectively measure the surface of a solid phase support or alternatively, the solution without the solid phase support. For practical operation, this means that a fluorescent dye may be used which does not necessarily change its measurable parameters via intercalation. Thus, probes having optional, oligomeric dyes may be used which permit to obtain an increase in fluorescence intensity by more than tenfold. The thermally induced denaturation process (see above) may then be monitored by measuring the dissociation of the probe from surface-bound, amplified nucleic acids (sample and standard) into the free solution, utilizing the decrease in fluorescence (dilution effect). This approach is confined to analyses where the double-strand region used for standardization lies within the thermodynamically most stable melting region of the nucleic acid, or where the probe has only one melting region.</p>
    <p>Addition of a labeled probe subsequent to amplification may be abandoned if the probe is not capable of participating in the amplification process due to specific properties and performance of the amplification process. According to the invention, this may be accomplished if the probe is present in a thermodynamically stable double-stranded structure remaining stable during the amplification process and not participating in the reaction process. For example, labeled RNA double helices of high thermodynamic stability or chemically modified probes may be employed. In this fashion, the amplification process may be controlled such that the probe steadily remains double-stranded in the amplification reaction and does not participate in the amplification process, be it the case that the denaturation temperature is insufficient or enzymes involved will not amplify the double-stranded RNA or the modified probe.</p>
    <p>Amplification techniques bear the great danger that subsequent to amplification, amplified DNA or RNA molecules might escape into the environment. Particular danger is constantly present due to aerosol formation which is difficult to control industrially. However, the process of the invention in its specific embodiment permits to carry out amplification reactions and analytics in a hermetically sealed reaction vessel and subsequently to dispose same in sealed condition. This provides a definite contribution to molecular laboratory hygiene and to reliability of results in routine diagnostics.</p>
    <p>Routine diagnostics are closely associated with the necessity of archiving results and--if possible--the samples analyzed. Again, the process of the invention offers an almost ideal opportunity:</p>
    <p>The sheets may be provided with a test-specific bar labeling and thus, may be characterized and compiled for data with respect to target assay, manufacturing date, expiry date, etc.</p>
    <p>The sheets may be stored in a hermetically sealed and archived condition for prolonged periods of time.</p>
    <p>Analyses may be repeated and checked at a later time.</p>
    <p>In case the analytical method furnishes evidence for the presence of new interesting DNA/RNA variants, the sample mixture may be removed and preparatively separated on the flat temperature gel electrophoresis separation system and, e.g., may be subjected to sequence analysis.</p>
    <p>For smaller series, temperature gradient gel electrophoresis had shown its reliability for quantitative amplification analytics. In this technique such PCR amplification products are preferably used which bear G:C-rich, primer-encoded sequences at the more stable fragment end. These so-called G:C clamps ensure the reversible course of melting required for temperature gel electrophoresis, and suppress untimely dissociation of the fragments into single strands. For a variety of analyses using temperature-dependent gel electrophoresis, G:C clamps which are expensive in use and may cause difficulties in amplification will no longer be required. Sequential liberation of fluorescent dyes in an assay may quite well be irreversible without distorting the result.</p>
    <p>The process according to the invention has considerable economic significance for more economic manufacturing of the test kits as well as with respect to costs for operating the analytics. The operation may be automatized almost completely, and the result of the analysis may be printed in the form of a hard copy report. Thus, the temperature gel electrophoresis technique is not limited to expensive, quantifying analyses of great valuation but may also be employed for low-cost analyses. For example, this relates to the field of microbiology, human genetics, phytoanalytics, forensic analytics, and industrial research such as, e.g., research on active agents, i.e., screening of target substances for active agents which may be evaluated through DNA or RNA amplification or modification, down to simple toxicity assays.</p>
    <p>The process according to the invention having its useful characteristics such as</p>
    <p>PCR/3SR/TAS amplification efficiency</p>
    <p>suitability for serial or single examinations</p>
    <p>on-line registration</p>
    <p>simultaneous nucleic acid qualification/quantification</p>
    <p>single-batch multi-component analysis</p>
    <p>elimination of danger of contamination by amplification products</p>
    <p>low costs of assay reagents and assay operation</p>
    <p>generally valid--no assay-specific--procedure guidelines</p>
    <p>permits new utilization prospects for amplification techniques such as PCR, 3SR or TAS.</p>
    <p>In genetic analytics, efforts envisaging a long-term prospect are made which permit, for instance, to detect severe genetic diseases from traces of biopsy material or amniotic fluid. Here, the burden of costs for such large-scale programs likewise is a decisive limitation as is the fact that only in very few cases analysis of a single gene locus is sufficient to permit statements regarding the transmission potential. For the most frequent lethal hereditary disease among the Caucasian population, the cystic fibrosis, more than 18 further mutations in addition to the initially discovered mutation (Delta 508) have been described, which may induce mucoviscidosis. The process of the invention permits parallel analysis of a great number of loci, with minor additional expense, and if necessary--without great additional expense--to cover further assay items. For example, the process of the invention also permits to examine point mutations responsible for certain genetic diseases. Hitherto, such diseases can only be diagnosed using the allele-specific oligonucleotide hybridization technique (ASO) which is very difficult to handle.</p>
    <p>Of similar significance is the technique for thalassemia screenings.</p>
    <p>Thus, it becomes clear that the process according to the invention can simplify the previous methods for detecting defined mutations and design them to be substantially lower in costs. Instead of an artificially introduced mutation, in order to make norms or standards, the desired, naturally occurring mutant may be employed or used as a probe. Thus, for operating the process according to the invention, all those assays are possible which are carried out using, e.g., differential oligonucleotide hybridizations as filter assays.</p>
    <p>Epidemiological studies on infectious and hereditary diseases emerge as an increasingly important working field for international medicine. One may remind the alarming rate of propagation of modern viral diseases such as AIDS (HIV) caused by the changed sociobiological structures. One decisive precondition for vaccination and therapy approaches is an epidemiological registration of the actual situation as comprehensive and careful as possible with epidemiological-prognostic significance. This relates not only to the actual appearance of the virus but also to the geographic distribution of its spectrum of variants, with data-level registration of the symptoms and signs of the disease. Only by profound automatized analytics such studies will become economically justifiable.</p>
    <p>For quite a time, pharmaceutical research touches the limits of what it feasible in its very own fields of activity. Many attempts have been made to combat the obvious limitations of conventional pharmaceutical research by virtue of new conceptions. Reference is made to the example of strategic-intelligent drug design which is to permit well-aimed synthesis of effector molecules having precalculated structure on the basis of well-founded knowledge about a target structure such as that of a receptor and thus, to replace conventional screening methods. A second strategy with a promising future is evolutionary biology with its potential, utilizing evolutionary systems, to generate substances having the desired activity potential. Future pharmaceutical research has to avoid personnel-intensive, randomized synthesis programs for both cost- and project-related reasons.</p>
    <p>Here, the process according to the invention may provide an important contribution. The suitability for screenings using quantifying DNA/RNA assays at low cost permits to develop new assays for automatized detection of active substances in oncology, with viral and bacterial infections and assays for toxicological studies. One could think of new assay systems in combination with cellular in vitro systems which will increasingly replace animal models. Neither must one forget that time periods for cultivation and assays will be cut down substantially since, e.g., variations on the mRNA level are detected immediately without the necessity to time-consumingly measure cellular subsequent reactions. For a number of assays it might become possible to replace transformed cell cultures by primary cultures, e.g., from blood. Simultaneous high sensitivity would also allow for registering differentiation parameters to measure cell type-specific activities. For example, one could think of certain leucocyte subpopulations (macrophages, T4 cells, etc.) with their characteristic receptor functions and infectability by viruses such as HIV.</p>
    <p>Biological experiments with recombinant systems, particularly open-air experiments require from both science and environmental protection careful analytics of genes, registration of gene persistence in populations, gene variations, and gene activities. As an example, there may be mentioned antiviral therapies on plants where the attempt is made to create virus resistance against certain viruses by designing transgenic plants which produce viral envelope proteins and thus, protect the plant from infection by an exogenic virus. open-air experiments or interbreedings into naturally occurring populations require precise knowledge, for example:</p>
    <p>How is the segregation behavior of the recombinant plants with respect to the recombinant locus?</p>
    <p>How active are envelope protein genes in relation to population and number of generations?</p>
    <p>How does the viral target organism react to the new selection stress?</p>
    <p>Here, the process according to the invention permits screenings and, in particular, registration of sample collectives in one single assay to obtain quantitative statements about populations. At the same time, it becomes possible to monitor drift phenomenons in development of resistance.</p>
    <p>Traditionally, microbiological assays in bacteriology have been conducted by cultivation or direct dot hybridization. Hitherto, such assays could not be displaced by more profound assays just for the aspect of costs alone. The process according to the invention can completely change this situation. In this process, it is not a decisive issue of costs if a sample has to be assayed for multiple pathogens simultaneously, or if a pathogen has to be analyzed in an antibiogram.</p>
    <p>The last-mentioned type of analysis may play a major role particularly in virological analyses if virus resistances have to be tested or registered quantitatively. To virology, the process of the invention is advantageous since it permits greatly simplified analytics by combining culturing and testing process of a virus propagation, which is favorable in cost and lower in expense.</p>
    <p>The food industry primarily has conducted extensive controls to examine some specific microbiological classes of pathogens, setting out, however, from a variety of sample substances of starting materials, of process steps, of equipment containments, and of final products. In similar fashion as in the pharmaceutical industry, processing involves large-scale batches of considerable economic value. Here, minimizing the durations of analyses, combined with reliable test results are critical factors. For example, salmonellal assays frequently require time-consuming pathogen culturing processes. The process according to the invention, with its likewise quantitative results in screenings makes it possible to abandon culturing processes and saves costly process operation steps and storage.</p>
    <p>Frequently, phytopathological analyses are prohibitively costly. Even if, as a rule, only representative sample collectives of seed or ornamental and useful plants are delivered to analysis, for example, for virus infection, the number of samples so far permitted almost exclusively immunological ELISA processes for cost reasons, which often do not work at all or unsatisfactorily. Hence, phytoanalytics require modified concepts of analyzing to meet the demands. These concepts are provided by the process of the invention.</p>
    <p>As now, for the first time, RNA parameters have been found which permit precise analysis of forest damage (Riesner et al., in preparation), the process according to the invention allows for the set-up of an extensive program for registration of forest damages as well as their temporal and geographic spreading.</p>
    <p>Research in molecular biology increasingly strives for analytic processes suitable for application in series in order to, for example, successfully carry out the so-called HUGO project on complete elucidation of the human genome sequence and of genomes of other important organisms. In future, this project will not only have the object of completely analyzing a single genome but also to analyze certain loci having disease-related potential, as is necessary--for comparison--in identity examinations (HLA analytics, haplotype association, genetic variability, etc.).</p>
    <p>The process according to the invention makes it possible on a DNA level (amplified loci, gene dosage) and on a mRNA level to monitor expression optimizing, promoter controlling, etc., and to utilize same in the screening or mutagenesis process. Here, the process has proven to be an important complementary technique to "cell sorting".</p>
    </div></div></div><div class="patent-section patent-tabular-section"><a id="backward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Cited Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5184020">US5184020</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Oct 26, 1989</td><td class="patent-data-table-td patent-date-value">Feb 2, 1993</td><td class="patent-data-table-td ">Hearst David P</td><td class="patent-data-table-td ">For treating photoreactive compounds</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP0381501B1?cl=en">EP0381501B1</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Feb 1, 1990</td><td class="patent-data-table-td patent-date-value">Jun 8, 1994</td><td class="patent-data-table-td ">Eastman Kodak Company</td><td class="patent-data-table-td ">Containment cuvette for PCR and method of use</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="http://www.google.com/url?id=aGlJBAABERAJ&amp;q=http://worldwide.espacenet.com/publicationDetails/biblio%3FCC%3DJP%26NR%3DH09201533A%26KC%3DA%26FT%3DD&amp;usg=AFQjCNE3AufXYtauS3J3zo8vt93iQiE7iA">JPH09201533A</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value"></td><td class="patent-data-table-td patent-date-value"></td><td class="patent-data-table-td "> </td><td class="patent-data-table-td citation-no-title">Title not available</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1990001563A1?cl=en">WO1990001563A1</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jul 24, 1989</td><td class="patent-data-table-td patent-date-value">Feb 22, 1990</td><td class="patent-data-table-td ">George D Cimino</td><td class="patent-data-table-td ">Identification of allele specific nucleic acid sequences by hybridization with crosslinkable oligonucleotide probes</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1990005023A1?cl=en">WO1990005023A1</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Nov 3, 1989</td><td class="patent-data-table-td patent-date-value">May 17, 1990</td><td class="patent-data-table-td ">Max Planck Gesellschaft</td><td class="patent-data-table-td ">Apparatus for selectively adjusting the temperature of a test specimen to various values</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1990015881A1?cl=en">WO1990015881A1</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jun 12, 1990</td><td class="patent-data-table-td patent-date-value">Dec 27, 1990</td><td class="patent-data-table-td ">Cis Bio Int</td><td class="patent-data-table-td ">Method for detecting specific nucleic acid sequences and applications of same</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1991002815A1?cl=en">WO1991002815A1</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Aug 18, 1990</td><td class="patent-data-table-td patent-date-value">Mar 7, 1991</td><td class="patent-data-table-td ">Diagen Inst Molekularbio</td><td class="patent-data-table-td ">Process and device for separating and detecting constituents of a mixture of substances by temperature gradient gel electrophoresis</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1991002817A1?cl=en">WO1991002817A1</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Aug 20, 1990</td><td class="patent-data-table-td patent-date-value">Mar 7, 1991</td><td class="patent-data-table-td ">Cetus Corp</td><td class="patent-data-table-td ">Quantitation of nucleic acids using the polymerase chain reaction</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="npl-citations"></a><div class="patent-section-header"><span class="patent-section-title">Non-Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th colspan="3"class="patent-data-table-th">Reference</th></tr></thead><tr><td class="patent-data-table-td ">1</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">J.E. Hearst, "<a href='http://scholar.google.com/scholar?q="A+photochemical+investigation+of+the+dynamics+of+the+oligonucleotide+hybridization"'>A photochemical investigation of the dynamics of the oligonucleotide hybridization</a>", Ann. Rev. Phys. Chem., Bd. 39, pp. 291-315, 1988.</td></tr><tr><td class="patent-data-table-td ">2</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">J.E. Hearst, A photochemical investigation of the dynamics of the oligonucleotide hybridization , Ann. Rev. Phys. Chem., Bd. 39, pp. 291 315, 1988.</td></tr><tr><td class="patent-data-table-td ">3</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">K. Henco et al., "<a href='http://scholar.google.com/scholar?q="Quantitative+PCR%3A+the+determination+of+template+copy+numbers+by+temperature+gradient+gel+electrohoresis+%28TGGE%29"'>Quantitative PCR: the determination of template copy numbers by temperature gradient gel electrohoresis (TGGE)</a>", Nucleic Acids Research, Bd. 18, Nr. 22, pp. 6733-6734, 1990.</td></tr><tr><td class="patent-data-table-td ">4</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">K. Henco et al., Quantitative PCR: the determination of template copy numbers by temperature gradient gel electrohoresis (TGGE) , Nucleic Acids Research, Bd. 18, Nr. 22, pp. 6733 6734, 1990.</td></tr><tr><td class="patent-data-table-td ">5</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Kingsbury et al., "<a href='http://scholar.google.com/scholar?q="Rapid+Detection+and+Identification+of+Infection+Agents"'>Rapid Detection and Identification of Infection Agents</a>", Academic Press, Inc., pp. 245-256, 1985.</td></tr><tr><td class="patent-data-table-td ">6</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Kingsbury et al., Rapid Detection and Identification of Infection Agents , Academic Press, Inc., pp. 245 256, 1985.</td></tr><tr><td class="patent-data-table-td ">7</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Y. Jinno et al., "<a href='http://scholar.google.com/scholar?q="Use+of+psoralen+as+extinguisher+of+contaminated+DNA+in+PCT"'>Use of psoralen as extinguisher of contaminated DNA in PCT</a>", Nucleic Acids Research, Bd. 18, Nr. 22, p. 6739.</td></tr><tr><td class="patent-data-table-td ">8</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Y. Jinno et al., Use of psoralen as extinguisher of contaminated DNA in PCT , Nucleic Acids Research, Bd. 18, Nr. 22, p. 6739.</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="forward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Referenced by</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Citing Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6174670">US6174670</a></td><td class="patent-data-table-td patent-date-value">Jun 4, 1997</td><td class="patent-data-table-td patent-date-value">Jan 16, 2001</td><td class="patent-data-table-td ">University Of Utah Research Foundation</td><td class="patent-data-table-td ">Achieved with rapid thermal cycling and use of double stranded dna dyes or specific hybridization probes</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6472153">US6472153</a></td><td class="patent-data-table-td patent-date-value">Oct 26, 1999</td><td class="patent-data-table-td patent-date-value">Oct 29, 2002</td><td class="patent-data-table-td ">Epoch Biosciences, Inc.</td><td class="patent-data-table-td ">Hybridization-triggered fluorescent detection of nucleic acids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6814934">US6814934</a></td><td class="patent-data-table-td patent-date-value">Nov 12, 1997</td><td class="patent-data-table-td patent-date-value">Nov 9, 2004</td><td class="patent-data-table-td ">Russell Gene Higuchi</td><td class="patent-data-table-td ">Instrument for monitoring nucleic acid amplification</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6872530">US6872530</a></td><td class="patent-data-table-td patent-date-value">Apr 24, 2002</td><td class="patent-data-table-td patent-date-value">Mar 29, 2005</td><td class="patent-data-table-td ">Spectrumedix, Llc</td><td class="patent-data-table-td ">By using a temperature gradient electrophoresis in the presence of a probe; for generating data indicative of the presence of a nucleotide polymorphism, mutation, or methylated cytosine in nucleotide containing compounds</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6951930">US6951930</a></td><td class="patent-data-table-td patent-date-value">Aug 21, 2002</td><td class="patent-data-table-td patent-date-value">Oct 4, 2005</td><td class="patent-data-table-td ">Epoch Biosciences, Inc.</td><td class="patent-data-table-td ">Hybridization-triggered fluorescent detection of nucleic acids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7175750">US7175750</a></td><td class="patent-data-table-td patent-date-value">Nov 5, 2002</td><td class="patent-data-table-td patent-date-value">Feb 13, 2007</td><td class="patent-data-table-td ">Spectrumedix Llc</td><td class="patent-data-table-td ">System and method for temperature gradient capillary electrophoresis</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7282126">US7282126</a></td><td class="patent-data-table-td patent-date-value">Jul 14, 2003</td><td class="patent-data-table-td patent-date-value">Oct 16, 2007</td><td class="patent-data-table-td ">Spectrumedix Llc</td><td class="patent-data-table-td ">Amplifying the sample polynucleotide to obtain first amplicons including the first variant site. combining with polynucleotide controls with differing bases corresponding to the first variant site; preparing duplexes</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7297495">US7297495</a></td><td class="patent-data-table-td patent-date-value">Oct 3, 2005</td><td class="patent-data-table-td patent-date-value">Nov 20, 2007</td><td class="patent-data-table-td ">Epoch Biosciences, Inc.</td><td class="patent-data-table-td ">Conjugating minor groove binders to olignucleotides to distinguish perfectly matched targets and those with single mismatches</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7303879">US7303879</a></td><td class="patent-data-table-td patent-date-value">Aug 2, 2004</td><td class="patent-data-table-td patent-date-value">Dec 4, 2007</td><td class="patent-data-table-td ">Applera Corporation</td><td class="patent-data-table-td ">Determination of SNP allelic frequencies using temperature gradient electrophoresis</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7387887">US7387887</a></td><td class="patent-data-table-td patent-date-value">Apr 20, 2004</td><td class="patent-data-table-td patent-date-value">Jun 17, 2008</td><td class="patent-data-table-td ">University Of Utah Research Foundation</td><td class="patent-data-table-td ">standard PCR solution with additional saturating double-stranded (ds) DNA binding dye; fluorescence; cyanine dyes</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7435541">US7435541</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Apr 4, 2002</td><td class="patent-data-table-td patent-date-value">Oct 14, 2008</td><td class="patent-data-table-td ">Sequenom, Inc.</td><td class="patent-data-table-td ">Restriction enzyme genotyping</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7456281">US7456281</a></td><td class="patent-data-table-td patent-date-value">Jul 13, 2006</td><td class="patent-data-table-td patent-date-value">Nov 25, 2008</td><td class="patent-data-table-td ">Idaho Technology, Inc.</td><td class="patent-data-table-td ">Nucleic acid melting analysis with saturation dyes</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7582429">US7582429</a></td><td class="patent-data-table-td patent-date-value">Oct 22, 2003</td><td class="patent-data-table-td patent-date-value">Sep 1, 2009</td><td class="patent-data-table-td ">University Of Utah Research Foundation</td><td class="patent-data-table-td ">Amplicon melting analysis with saturation dyes</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7611674">US7611674</a></td><td class="patent-data-table-td patent-date-value">Jan 11, 2007</td><td class="patent-data-table-td patent-date-value">Nov 3, 2009</td><td class="patent-data-table-td ">Applied Biosystems, Llc</td><td class="patent-data-table-td ">Heating/ cooling device for control of a reaction vessel receiver having several recesses arranged in a regular pattern to receive a microtiter plate with severalreaction vessels; thermocycler receiver divided into thermally decoupled segments actuated independentally; automatic; robotics; DNA</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7659056">US7659056</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Oct 23, 2000</td><td class="patent-data-table-td patent-date-value">Feb 9, 2010</td><td class="patent-data-table-td ">Ingeny Holding B.V.</td><td class="patent-data-table-td ">amplification step is performed in or on the gel which takes a shorter time</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7670832">US7670832</a></td><td class="patent-data-table-td patent-date-value">Aug 15, 2005</td><td class="patent-data-table-td patent-date-value">Mar 2, 2010</td><td class="patent-data-table-td ">University Of Utah Research Foundation</td><td class="patent-data-table-td ">System for fluorescence monitoring</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7727479">US7727479</a></td><td class="patent-data-table-td patent-date-value">Jun 12, 2006</td><td class="patent-data-table-td patent-date-value">Jun 1, 2010</td><td class="patent-data-table-td ">Applied Biosystems, Llc</td><td class="patent-data-table-td ">Device for the carrying out of chemical or biological reactions</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7732136">US7732136</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Aug 25, 2004</td><td class="patent-data-table-td patent-date-value">Jun 8, 2010</td><td class="patent-data-table-td ">Pall Genesystems</td><td class="patent-data-table-td ">Using multicompartment apparatus for cyclic propagation of preferential nucleotide sequences; DNA amplification</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7759469">US7759469</a></td><td class="patent-data-table-td patent-date-value">Mar 8, 2006</td><td class="patent-data-table-td patent-date-value">Jul 20, 2010</td><td class="patent-data-table-td ">Roche Diagnostics Operations, Inc.</td><td class="patent-data-table-td ">substituted indole nucleosides as both terminal as well as internal building blocks of labeled oligonucleotide probes for the detection, analysis and quantitation of nucleic acids; nucleoside with a 3-substituted methanesulfonylamino indole as nucleobase</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7785786">US7785786</a></td><td class="patent-data-table-td patent-date-value">Jan 23, 2006</td><td class="patent-data-table-td patent-date-value">Aug 31, 2010</td><td class="patent-data-table-td ">Quest Diagnostics Investments Incorporated</td><td class="patent-data-table-td ">Methods for detecting nucleic acids using multiple signals</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7803551">US7803551</a></td><td class="patent-data-table-td patent-date-value">Aug 11, 2009</td><td class="patent-data-table-td patent-date-value">Sep 28, 2010</td><td class="patent-data-table-td ">University Of Utah Research Foundation</td><td class="patent-data-table-td ">Amplifying the target nucleic acid with primers in the presence of a double-stranded DNA binding dye; monitoring fluorescence of the dye; melting curve is generated for the target nucleic acid by measuring fluorescence from the dsDNA binding dye as the mixture is heated.</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7955802">US7955802</a></td><td class="patent-data-table-td patent-date-value">Dec 13, 2007</td><td class="patent-data-table-td patent-date-value">Jun 7, 2011</td><td class="patent-data-table-td ">Luminex Corporation</td><td class="patent-data-table-td ">Systems and methods for multiplex analysis of PCR in real time</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7966746">US7966746</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Apr 24, 2007</td><td class="patent-data-table-td patent-date-value">Jun 28, 2011</td><td class="patent-data-table-td ">Medical Instill Technologies, LLC</td><td class="patent-data-table-td ">Needle penetrable and laser resealable lyophilization method</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7972778">US7972778</a></td><td class="patent-data-table-td patent-date-value">Mar 11, 2004</td><td class="patent-data-table-td patent-date-value">Jul 5, 2011</td><td class="patent-data-table-td ">Applied Biosystems, Llc</td><td class="patent-data-table-td ">Using multicompartment apparatus for replicating nucleotide sequences on a miniaturized scale; fertility, immunology, cytology and pharmaceutical screening</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8003370">US8003370</a></td><td class="patent-data-table-td patent-date-value">Dec 15, 2008</td><td class="patent-data-table-td patent-date-value">Aug 23, 2011</td><td class="patent-data-table-td ">California Institute Of Technology</td><td class="patent-data-table-td ">Thermal cycling apparatus</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8008046">US8008046</a></td><td class="patent-data-table-td patent-date-value">Dec 15, 2008</td><td class="patent-data-table-td patent-date-value">Aug 30, 2011</td><td class="patent-data-table-td ">California Institute Of Technology</td><td class="patent-data-table-td ">Thermal cycling method</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8039216">US8039216</a></td><td class="patent-data-table-td patent-date-value">Jul 28, 2010</td><td class="patent-data-table-td patent-date-value">Oct 18, 2011</td><td class="patent-data-table-td ">Quest Diagnostics Investments Incorporated</td><td class="patent-data-table-td ">Methods for detecting nucleic acids using multiple signals</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8048628">US8048628</a></td><td class="patent-data-table-td patent-date-value">May 24, 2007</td><td class="patent-data-table-td patent-date-value">Nov 1, 2011</td><td class="patent-data-table-td ">Duke University</td><td class="patent-data-table-td ">Methods for nucleic acid amplification on a printed circuit board</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8067159">US8067159</a></td><td class="patent-data-table-td patent-date-value">Aug 13, 2007</td><td class="patent-data-table-td patent-date-value">Nov 29, 2011</td><td class="patent-data-table-td ">Applied Biosystems, Llc</td><td class="patent-data-table-td ">Microfluidics; electromagnetics; electrophoresis; DNA sequence analysis</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8093002">US8093002</a></td><td class="patent-data-table-td patent-date-value">Jul 9, 2010</td><td class="patent-data-table-td patent-date-value">Jan 10, 2012</td><td class="patent-data-table-td ">University Of Utah Research Foundation</td><td class="patent-data-table-td ">Amplicon melting analysis with saturation dyes</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8105782">US8105782</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Oct 23, 2001</td><td class="patent-data-table-td patent-date-value">Jan 31, 2012</td><td class="patent-data-table-td ">Cancer Research Technology Limited</td><td class="patent-data-table-td ">Materials and methods relating to nucleic acid amplification and profiling</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8147668">US8147668</a></td><td class="patent-data-table-td patent-date-value">Oct 23, 2006</td><td class="patent-data-table-td patent-date-value">Apr 3, 2012</td><td class="patent-data-table-td ">Duke University</td><td class="patent-data-table-td ">Apparatus for manipulating droplets</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8171652">US8171652</a></td><td class="patent-data-table-td patent-date-value">Jun 27, 2011</td><td class="patent-data-table-td patent-date-value">May 8, 2012</td><td class="patent-data-table-td ">Medical Instill Technologies, Inc.</td><td class="patent-data-table-td ">Penetrable and resealable lyophilization method</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8221605">US8221605</a></td><td class="patent-data-table-td patent-date-value">Dec 27, 2007</td><td class="patent-data-table-td patent-date-value">Jul 17, 2012</td><td class="patent-data-table-td ">Duke University</td><td class="patent-data-table-td ">Apparatus for manipulating droplets</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8232091">US8232091</a></td><td class="patent-data-table-td patent-date-value">May 17, 2007</td><td class="patent-data-table-td patent-date-value">Jul 31, 2012</td><td class="patent-data-table-td ">California Institute Of Technology</td><td class="patent-data-table-td ">Thermal cycling system</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8257925">US8257925</a></td><td class="patent-data-table-td patent-date-value">May 16, 2011</td><td class="patent-data-table-td patent-date-value">Sep 4, 2012</td><td class="patent-data-table-td ">Applied Biosystems, Llc</td><td class="patent-data-table-td ">Method for detecting the presence of a single target nucleic acid in a sample</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8268246">US8268246</a></td><td class="patent-data-table-td patent-date-value">Aug 11, 2008</td><td class="patent-data-table-td patent-date-value">Sep 18, 2012</td><td class="patent-data-table-td ">Advanced Liquid Logic Inc</td><td class="patent-data-table-td ">PCB droplet actuator fabrication</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8278071">US8278071</a></td><td class="patent-data-table-td patent-date-value">Aug 13, 2007</td><td class="patent-data-table-td patent-date-value">Oct 2, 2012</td><td class="patent-data-table-td ">Applied Biosystems, Llc</td><td class="patent-data-table-td ">Method for detecting the presence of a single target nucleic acid in a sample</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8288105">US8288105</a></td><td class="patent-data-table-td patent-date-value">May 25, 2011</td><td class="patent-data-table-td patent-date-value">Oct 16, 2012</td><td class="patent-data-table-td ">Luminex Corporation</td><td class="patent-data-table-td ">Systems and methods for multiplex analysis of PCR in real time</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8349276">US8349276</a></td><td class="patent-data-table-td patent-date-value">Jan 30, 2006</td><td class="patent-data-table-td patent-date-value">Jan 8, 2013</td><td class="patent-data-table-td ">Duke University</td><td class="patent-data-table-td ">Apparatuses and methods for manipulating droplets on a printed circuit board</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8388909">US8388909</a></td><td class="patent-data-table-td patent-date-value">Oct 9, 2009</td><td class="patent-data-table-td patent-date-value">Mar 5, 2013</td><td class="patent-data-table-td ">Duke University</td><td class="patent-data-table-td ">Apparatuses and methods for manipulating droplets</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8389288">US8389288</a></td><td class="patent-data-table-td patent-date-value">Jan 18, 2010</td><td class="patent-data-table-td patent-date-value">Mar 5, 2013</td><td class="patent-data-table-td ">Applied Biosystems, Llc</td><td class="patent-data-table-td ">Device for the carrying out of chemical or biological reactions</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8394249">US8394249</a></td><td class="patent-data-table-td patent-date-value">Jun 30, 2009</td><td class="patent-data-table-td patent-date-value">Mar 12, 2013</td><td class="patent-data-table-td ">Duke University</td><td class="patent-data-table-td ">Methods for manipulating droplets by electrowetting-based techniques</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8524506">US8524506</a></td><td class="patent-data-table-td patent-date-value">Nov 7, 2007</td><td class="patent-data-table-td patent-date-value">Sep 3, 2013</td><td class="patent-data-table-td ">Duke University</td><td class="patent-data-table-td ">Methods for sampling a liquid flow</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8551698">US8551698</a></td><td class="patent-data-table-td patent-date-value">Aug 13, 2007</td><td class="patent-data-table-td patent-date-value">Oct 8, 2013</td><td class="patent-data-table-td ">Applied Biosystems, Llc</td><td class="patent-data-table-td ">Method of loading sample into a microfluidic device</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8563275">US8563275</a></td><td class="patent-data-table-td patent-date-value">Aug 11, 2012</td><td class="patent-data-table-td patent-date-value">Oct 22, 2013</td><td class="patent-data-table-td ">Applied Biosystems, Llc</td><td class="patent-data-table-td ">Method and device for detecting the presence of a single target nucleic acid in a sample</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8676383">US8676383</a></td><td class="patent-data-table-td patent-date-value">Sep 5, 2007</td><td class="patent-data-table-td patent-date-value">Mar 18, 2014</td><td class="patent-data-table-td ">Applied Biosystems, Llc</td><td class="patent-data-table-td ">Device for carrying out chemical or biological reactions</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8721972">US8721972</a></td><td class="patent-data-table-td patent-date-value">May 14, 2012</td><td class="patent-data-table-td patent-date-value">May 13, 2014</td><td class="patent-data-table-td ">Applied Biosystems, Llc</td><td class="patent-data-table-td ">Device for the carrying out of chemical or biological reactions</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8734629">US8734629</a></td><td class="patent-data-table-td patent-date-value">May 5, 2011</td><td class="patent-data-table-td patent-date-value">May 27, 2014</td><td class="patent-data-table-td ">Advanced Liquid Logic, Inc.</td><td class="patent-data-table-td ">Droplet actuator and methods</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20060078929">US20060078929</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Sep 30, 2005</td><td class="patent-data-table-td patent-date-value">Apr 13, 2006</td><td class="patent-data-table-td ">Clondiag Chip Technologies Gmbh</td><td class="patent-data-table-td ">Device for the amplification and detection of nucleic acids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP1055734A2?cl=en">EP1055734A2</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">May 23, 2000</td><td class="patent-data-table-td patent-date-value">Nov 29, 2000</td><td class="patent-data-table-td ">Tosoh Corporation</td><td class="patent-data-table-td ">Method for assaying nucleic acid</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP1132484A2?cl=en">EP1132484A2</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Mar 8, 2001</td><td class="patent-data-table-td patent-date-value">Sep 12, 2001</td><td class="patent-data-table-td ">Fuji Photo Film Co., Ltd.</td><td class="patent-data-table-td ">Method for testing complementation of nucleic acid</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP1509622A1?cl=en">EP1509622A1</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">May 21, 2003</td><td class="patent-data-table-td patent-date-value">Mar 2, 2005</td><td class="patent-data-table-td ">Perkin Elmer Life Science Inc.</td><td class="patent-data-table-td ">Fluorescence polarization detection of nucleic acids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2228454A1?cl=en">EP2228454A1</a></td><td class="patent-data-table-td patent-date-value">Mar 8, 2010</td><td class="patent-data-table-td patent-date-value">Sep 15, 2010</td><td class="patent-data-table-td ">Roche Diagnostics GmbH</td><td class="patent-data-table-td ">Multiplex quantitative nucleic acid amplification and melting assay</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2000066777A2?cl=en">WO2000066777A2</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Apr 26, 2000</td><td class="patent-data-table-td patent-date-value">Nov 9, 2000</td><td class="patent-data-table-td ">Qualicon Inc</td><td class="patent-data-table-td ">Assay based on the analysis of dna melting curves</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2001040511A2?cl=en">WO2001040511A2</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Nov 29, 2000</td><td class="patent-data-table-td patent-date-value">Jun 7, 2001</td><td class="patent-data-table-td ">Martin Alan Lee</td><td class="patent-data-table-td ">Detection system</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2002034942A2?cl=en">WO2002034942A2</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Oct 23, 2001</td><td class="patent-data-table-td patent-date-value">May 2, 2002</td><td class="patent-data-table-td ">Cancer Res Ventures Ltd</td><td class="patent-data-table-td ">Nucleic acid amplification-based methods for the determination of a methylation profile and reagents therefor</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2003033743A1?cl=en">WO2003033743A1</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Oct 18, 2002</td><td class="patent-data-table-td patent-date-value">Apr 24, 2003</td><td class="patent-data-table-td ">Z Y Guo</td><td class="patent-data-table-td ">System and method for temperature gradient capillary electrophoresis</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2010079127A1?cl=en">WO2010079127A1</a></td><td class="patent-data-table-td patent-date-value">Jan 7, 2010</td><td class="patent-data-table-td patent-date-value">Jul 15, 2010</td><td class="patent-data-table-td ">Roche Diagnostics Gmbh</td><td class="patent-data-table-td ">Nucleic acid amplification with allele-specific suppression of sequence variants</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2011069676A2?cl=en">WO2011069676A2</a></td><td class="patent-data-table-td patent-date-value">Dec 10, 2010</td><td class="patent-data-table-td patent-date-value">Jun 16, 2011</td><td class="patent-data-table-td ">Roche Diagnostics Gmbh</td><td class="patent-data-table-td ">Preferential amplification of mrna over dna using chemically modified primers</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2011069677A1?cl=en">WO2011069677A1</a></td><td class="patent-data-table-td patent-date-value">Dec 10, 2010</td><td class="patent-data-table-td patent-date-value">Jun 16, 2011</td><td class="patent-data-table-td ">Roche Diagnostics Gmbh</td><td class="patent-data-table-td ">Allele-specific amplification of nucleic acids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2012038049A2?cl=en">WO2012038049A2</a></td><td class="patent-data-table-td patent-date-value">Sep 15, 2011</td><td class="patent-data-table-td patent-date-value">Mar 29, 2012</td><td class="patent-data-table-td ">F. Hoffmann-La Roche Ag</td><td class="patent-data-table-td ">Amplification of distant nucleic acid targets using engineered primers</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2012065705A1?cl=en">WO2012065705A1</a></td><td class="patent-data-table-td patent-date-value">Nov 12, 2011</td><td class="patent-data-table-td patent-date-value">May 24, 2012</td><td class="patent-data-table-td ">F. Hoffmann-La Roche Ag</td><td class="patent-data-table-td ">Novel complex mutation in the epidermal growth factor receptor kinase domain</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2013041194A1?cl=en">WO2013041194A1</a></td><td class="patent-data-table-td patent-date-value">Sep 12, 2012</td><td class="patent-data-table-td patent-date-value">Mar 28, 2013</td><td class="patent-data-table-td ">Roche Diagnostics Gmbh</td><td class="patent-data-table-td ">Use of g-clamp for improved allele-specific pcr</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2013068103A1?cl=en">WO2013068103A1</a></td><td class="patent-data-table-td patent-date-value">Nov 7, 2012</td><td class="patent-data-table-td patent-date-value">May 16, 2013</td><td class="patent-data-table-td ">Roche Diagnostics Gmbh</td><td class="patent-data-table-td ">Novel complex mutations in the epidermal growth factor receptor kinase domain</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2013079212A1?cl=en">WO2013079212A1</a></td><td class="patent-data-table-td patent-date-value">Nov 30, 2012</td><td class="patent-data-table-td patent-date-value">Jun 6, 2013</td><td class="patent-data-table-td ">Roche Diagnostics Gmbh</td><td class="patent-data-table-td ">Suppression of non-specific amplification with high-homology oligonucleotides</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2014086707A1?cl=en">WO2014086707A1</a></td><td class="patent-data-table-td patent-date-value">Dec 2, 2013</td><td class="patent-data-table-td patent-date-value">Jun 12, 2014</td><td class="patent-data-table-td ">Roche Diagnostics Gmbh</td><td class="patent-data-table-td ">Novel mutations in the epidermal growth factor receptor kinase domain</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="classifications"></a><div class="patent-section-header"><span class="patent-section-title">Classifications</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th"> </th><th class="patent-data-table-th"> </th></tr></thead><tr><td class="patent-data-table-td ">U.S. Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=aGlJBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S006110">435/6.11</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=aGlJBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S091200">435/91.2</a></span></td></tr><tr><td class="patent-data-table-td ">International Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=aGlJBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12Q0001680000">C12Q1/68</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=aGlJBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=B01L0007000000">B01L7/00</a></span></td></tr><tr><td class="patent-data-table-td ">Cooperative Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=aGlJBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12Q1/686">C12Q1/686</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=aGlJBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12Q1/6825">C12Q1/6825</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=aGlJBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=B01L7/52">B01L7/52</a></span></td></tr><tr><td class="patent-data-table-td ">European Classification</td><td class="patent-data-table-td "><span class="nested-value">B01L7/52</span>, <span class="nested-value">C12Q1/68D4</span>, <span class="nested-value">C12Q1/68B2H</span></td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="legal-events"></a><div class="patent-section-header"><span class="patent-section-title">Legal Events</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Date</th><th class="patent-data-table-th">Code</th><th class="patent-data-table-th">Event</th><th class="patent-data-table-th">Description</th></tr></thead><tr><td class="patent-data-table-td patent-date-value">Nov 26, 2013</td><td class="patent-data-table-td ">B1</td><td class="patent-data-table-td ">Reexamination certificate first reexamination</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">CLAIMS 1 AND 24 ARE DETERMINED TO BE PATENTABLE AS AMENDED.CLAIMS 6-9, 13, 23, 25-26, 40, 41 AND 43, DEPENDENT ON AN AMENDED CLAIM, ARE DETERMINED TO BE PATENTABLE.CLAIMS 2-5, 10-12, 14-22, 27-39 AND 42 WERE NOT REEXAMINED.</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Aug 14, 2012</td><td class="patent-data-table-td ">RR</td><td class="patent-data-table-td ">Request for reexamination filed</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20120621</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Aug 9, 2010</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">12</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Dec 28, 2009</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">EVOTEC AG, GERMANY</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">CHANGE OF NAME;ASSIGNOR:EVOTEC BIOSYSTEMS AG;REEL/FRAME:023708/0489</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20090109</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Aug 7, 2006</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">8</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Nov 5, 2002</td><td class="patent-data-table-td ">CC</td><td class="patent-data-table-td ">Certificate of correction</td><td class="patent-data-table-td "></td></tr><tr><td class="patent-data-table-td patent-date-value">Jul 29, 2002</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">4</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Feb 5, 1999</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">EVOTEC BIOSYSTEMS AG, GERMANY</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">CHANGE OF NAME;ASSIGNOR:EVOTEC BIOSYSTEMS GMBH;REEL/FRAME:009737/0846</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">19980807</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Sep 23, 1994</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">EVOTEC BIOSYSTEMS GMBH, GERMANY</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNOR:QIAGEN GMBH;REEL/FRAME:007153/0047</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">19940531</span></div><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">QIAGEN GMBH</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">CHANGE OF NAME;ASSIGNOR:DIAGEN INSTITUT FUR MOLEKULARBIOLOGISCHE DIAGNOSTIK GMBH;REEL/FRAME:007154/0637</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">19940325</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Dec 8, 1993</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">DIAGEN INSTITUT FUR MOLEKULARBIOLOGISCHE, GERMANY</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNORS:HENCO, KARSTEN;EIGEN, MANFRED;RIESNER, DETLEV;REEL/FRAME:007019/0786</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">19931203</span></div></td></tr></table><div class="patent-section-footer"></div></div><div class="modal-dialog" id="patent-images-lightbox"><div class="patent-lightbox-controls"><div class="patent-lightbox-rotate-controls"><div class="patent-lightbox-rotation-text">Rotate</div><div class="rotate-icon rotate-ccw-icon"></div><div class="rotate-icon rotate-cw-icon"></div></div><div class="patent-lightbox-index-counter"></div><a class="patent-lightbox-fullsize-link" target="_blank">Original Image</a><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_right.png" alt="Next page"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_left.png" alt="Previous page"width="21" height="21" /></div></div><div class="modal-dialog-content"><div class="patent-lightbox-image-holder"><div class="patent-lightbox-placeholder"></div></div></div></div><script>_OC_initPatentsAtb({image_not_available_html: " Image not available"});</script></div></div></div></td></tr></table><script>(function() {var href = window.location.href;if (href.indexOf('?') !== -1) {var parameters = href.split('?')[1].split('&');for (var i = 0; i < parameters.length; i++) {var param = parameters[i].split('=');if (param[0] == 'focus') {var elem = document.getElementById(param[1]);if (elem) {elem.focus();}}}}})();</script><script>_OC_addFlags({LockSrc:"/books/javascript/lock_4ff636b3d23669b7103f3b3a3a18b4cd.js", Host:"http://www.google.com/", IsBooksRentalEnabled:1, IsWebstoreDisplayCaseEnabled:1, IsObfuscationEnabled:1, IsBrowsingHistoryEnabled:1, IsWebReaderSvgEnabled:0, IsGeoLayerEnabled:1, IsImageModeNotesEnabled:1, IsCopyMenuItemEnabled:1, IsGiftingEnabled:0, IsWebReaderUniversalPaginatorEnabled:0, IsOfflineBubbleEnabled:1, IsReaderEnabledForPlayRequests:1, IsFutureOnSaleVolumesEnabled:1, IsOfflineRestrictedCopyEnabled:1, IsBooksUnifiedLeftNavEnabled:1, IsRestrictedCopyEnabled:1, IsZipitFolderCollectionEnabled:1, IsEndOfSampleRecommendationsEnabled:1, IsRatingsOnBookcardsEnabled:1, IsAdsDisabled:0, IsIframePageDisplayEnabled:0, IsEmbeddedMediaEnabled:1, IsImageModeAnnotationsEnabled:1, IsMyLibraryGooglePlusEnabled:1, IsImagePageProviderEnabled:0, IsBookcardListPriceSmall:0, IsInternalUser:0, IsBooksShareButtonEnabled:0, IsPreOrdersEnabled:0, IsDisabledRandomBookshelves:0, WebstoreDisplayCasePosition:3});_OC_Run({"enable_p13n":false,"add_vol_to_collection_base_url":"http://www.google.com/patents?op=add\u0026sig=ACfU3U2yYX76Ejvms-RYDtnoWPGFoQRemw\u0026id=aGlJBAABERAJ","remove_vol_from_collection_base_url":"http://www.google.com/patents?op=remove\u0026sig=ACfU3U0RxkGmlBDUoDTkLDclyL50Frv-hQ\u0026id=aGlJBAABERAJ","logged_in":false,"p13n_save_user_settings_url":"http://www.google.com/patents?op=edit_user_settings\u0026sig=ACfU3U2MFFHiSD7e9gWpYlfs33f_ahiSFg","is_cobrand":false,"sign_in_url":"https://www.google.com/accounts/Login?service=\u0026continue=http://www.google.com/patents%3Fhl%3Den\u0026hl=en","is_play_enabled":true}, {"volume_id":"","is_ebook":true,"volumeresult":{"has_flowing_text":false,"has_scanned_text":true,"can_download_pdf":false,"can_download_epub":false,"is_pdf_drm_enabled":false,"is_epub_drm_enabled":false,"download_pdf_url":"http://www.google.com/patents/download/Process_for_the_determination_of_in_vitr.pdf?id=aGlJBAABERAJ\u0026output=pdf\u0026sig=ACfU3U26Xsx19thgNxe740XPUH1pleJI3Q"},"sample_url":"http://www.google.com/patents/reader?id=aGlJBAABERAJ\u0026printsec=frontcover\u0026output=reader\u0026source=gbs_atb_hover","is_browsable":true,"is_public_domain":true}, {});</script><div id="footer_table" style="font-size:83%;text-align:center;position:relative;top:20px;height:4.5em;margin-top:2em"><div style="margin-bottom:8px"><a href=http://www.google.com/><nobr>Google&nbsp;Home</nobr></a> - <a href=//www.google.com/patents/sitemap/><nobr>Sitemap</nobr></a> - <a href=http://www.google.com/googlebooks/uspto.html><nobr>USPTO Bulk Downloads</nobr></a> - <a href=/intl/en/privacy/><nobr>Privacy Policy</nobr></a> - <a href=/intl/en/policies/terms/><nobr>Terms of Service</nobr></a> - <a href=https://support.google.com/faqs/answer/2539193?hl=en><nobr>About Google Patents</nobr></a> - <a href="http://www.google.com/tools/feedback/intl/en/error.html" onclick="try{_OC_startFeedback({productId: '72792',locale: 'en'});return false;}catch(e){}"><nobr>Send Feedback</nobr></a></div><span>Data provided by IFI CLAIMS Patent Services</span><br><span >&copy;2012 Google</span></div> <script type="text/javascript">var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));</script><script type="text/javascript">var pageTracker = _gat._getTracker("UA-27188110-1");pageTracker._setCookiePath("/patents/");pageTracker._trackPageview();</script> </body></html>