<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN""http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html><head><title>Patent US5872244 - 3&#39; protected nucleotides for enzyme catalyzed template-independent creation ... - Google Patents</title><script>(function(){(function(){function e(a){this.t={};this.tick=function(a,c,b){var d=void 0!=b?b:(new Date).getTime();this.t[a]=[d,c];if(void 0==b)try{window.console.timeStamp("CSI/"+a)}catch(e){}};this.tick("start",null,a)}var a;window.performance&&(a=window.performance.timing);var f=a?new e(a.responseStart):new e;window.jstiming={Timer:e,load:f};if(a){var c=a.navigationStart,d=a.responseStart;0<c&&d>=c&&(window.jstiming.srt=d-c)}if(a){var b=window.jstiming.load;0<c&&d>=c&&(b.tick("_wtsrt",void 0,c),b.tick("wtsrt_",
"_wtsrt",d),b.tick("tbsd_","wtsrt_"))}try{a=null,window.chrome&&window.chrome.csi&&(a=Math.floor(window.chrome.csi().pageT),b&&0<c&&(b.tick("_tbnd",void 0,window.chrome.csi().startE),b.tick("tbnd_","_tbnd",c))),null==a&&window.gtbExternal&&(a=window.gtbExternal.pageT()),null==a&&window.external&&(a=window.external.pageT,b&&0<c&&(b.tick("_tbnd",void 0,window.external.startE),b.tick("tbnd_","_tbnd",c))),a&&(window.jstiming.pt=a)}catch(g){}})();})();
</script><link rel="stylesheet" href="/patents/css/_8a2b04e7bf975d5171d8e4c0b6365c7a/kl_intl_patents_bundle.css" type="text/css" /><script src="/books/javascript/atb_8a2b04e7bf975d5171d8e4c0b6365c7a__en.js"></script><script>function googleTranslateElementInit() {new google.translate.TranslateElement({pageLanguage: "en",gaTrack: true,gaId: "UA-27188110-1",multilanguagePage: true});}</script><script src="//translate.google.com/translate_a/element.js?cb=googleTranslateElementInit"></script><meta name="DC.type" content="Patent"><meta name="DC.title" content="3&#39; protected nucleotides for enzyme catalyzed template-independent creation of phosphodiester bonds"><meta name="DC.contributor" content="Andrew C. Hiatt" scheme="inventor"><meta name="DC.contributor" content="Floyd Rose" scheme="inventor"><meta name="DC.contributor" content="Andrew C. Hiatt" scheme="assignee"><meta name="DC.contributor" content="Floyd D. Rose" scheme="assignee"><meta name="DC.date" content="1995-6-7" scheme="dateSubmitted"><meta name="DC.description" content="A method for the stepwise creation of phosphodiester bonds between desired nucleosides resulting in the synthesis of polynucleotides having a predetermined nucleotide sequence by preparing an initiation substrate containing a free and unmodified 3&#39;-hydroxyl group; attaching a mononucleotide selected according to the order of the predetermined nucleotide sequence to the 3&#39;-hydroxyl of the initiating substrate in a solution containing a catalytic amount of an enzyme capable of catalyzing the 5&#39; to 3&#39; phosphodiester linkage of the 5&#39;-phosphate of the mononucleotide to the 3&#39;-hydroxyl of the initiating substrate, wherein the mononucleotide contains a protected 3&#39;-hydroxyl group, whereby the protected mononucleotide is covalently linked to the initiating substrate and further additions are hindered by the 3&#39;-hydroxyl protecting group. Methods in which a mononucleotide immobilized on a solid support is added to a free polynucleotide chain are also disclosed."><meta name="DC.date" content="1999-2-16" scheme="issued"><meta name="DC.relation" content="JP:S5538324" scheme="references"><meta name="DC.relation" content="US:4096324" scheme="references"><meta name="DC.relation" content="US:4423212" scheme="references"><meta name="DC.relation" content="US:4605735" scheme="references"><meta name="DC.relation" content="US:4689405" scheme="references"><meta name="DC.relation" content="US:4719176" scheme="references"><meta name="DC.relation" content="US:4816571" scheme="references"><meta name="DC.relation" content="US:4820812" scheme="references"><meta name="DC.relation" content="US:4863849" scheme="references"><meta name="DC.relation" content="US:4876335" scheme="references"><meta name="DC.relation" content="US:4948882" scheme="references"><meta name="DC.relation" content="US:4950745" scheme="references"><meta name="DC.relation" content="US:4980460" scheme="references"><meta name="DC.relation" content="US:5003059" scheme="references"><meta name="DC.relation" content="US:5032680" scheme="references"><meta name="DC.relation" content="US:5039796" scheme="references"><meta name="DC.relation" content="US:5047524" scheme="references"><meta name="DC.relation" content="US:5091519" scheme="references"><meta name="DC.relation" content="US:5093232" scheme="references"><meta name="DC.relation" content="US:5112963" scheme="references"><meta name="DC.relation" content="US:5256549" scheme="references"><meta name="DC.relation" content="US:5258506" scheme="references"><meta name="DC.relation" content="US:5262530" scheme="references"><meta name="DC.relation" content="US:5262536" scheme="references"><meta name="DC.relation" content="US:5264563" scheme="references"><meta name="DC.relation" content="US:5264566" scheme="references"><meta name="DC.relation" content="US:5268266" scheme="references"><meta name="DC.relation" content="US:5268464" scheme="references"><meta name="DC.relation" content="US:5281701" scheme="references"><meta name="DC.relation" content="US:5348868" scheme="references"><meta name="DC.relation" content="US:5362866" scheme="references"><meta name="DC.relation" content="US:5367066" scheme="references"><meta name="DC.relation" content="US:5380833" scheme="references"><meta name="DC.relation" content="US:5436143" scheme="references"><meta name="DC.relation" content="US:5652099" scheme="references"><meta name="citation_reference" content="Bennett et al., Biochemistry , vol. 12, 20:3956 3960 (1973)."><meta name="citation_reference" content="Bennett et al., Biochemistry, vol. 12, 20:3956-3960 (1973)."><meta name="citation_reference" content="Bollum, Fed. Proc. Soc. Exp. Biol. Med. , 17, 193 (1958)."><meta name="citation_reference" content="Bollum, Fed. Proc. Soc. Exp. Biol. Med., 17, 193 (1958)."><meta name="citation_reference" content="Canard et al., Proc. Nat l. Acad. Sci. USA , 92, 10859 10863 (Nov. 1995)."><meta name="citation_reference" content="Canard et al., Proc. Nat&#39;l. Acad. Sci. USA, 92, 10859-10863 (Nov. 1995)."><meta name="citation_reference" content="Chang and Bollum, Biochemistry , vol. 10, 3:536 542 (1971)."><meta name="citation_reference" content="Chang and Bollum, Biochemistry, vol. 10, 3:536-542 (1971)."><meta name="citation_reference" content="Deng and Wu, Meth. Enzymol. , 100: 96 116 (1983)."><meta name="citation_reference" content="Deng and Wu, Meth. Enzymol., 100: 96-116 (1983)."><meta name="citation_reference" content="England and Uhlenbeck, Biochemistry , vol. 17, 11:2069 2076 (1978)."><meta name="citation_reference" content="England and Uhlenbeck, Biochemistry, vol. 17, 11:2069-2076 (1978)."><meta name="citation_reference" content="Flugel et al., Biochem. Biophys. Acta. , 308:35 40 (1973)."><meta name="citation_reference" content="Flugel et al., Biochem. Biophys. Acta., 308:35-40 (1973)."><meta name="citation_reference" content="Hinton and Gumport, Nucleic Acid Res. , 7:453 464 (1979)."><meta name="citation_reference" content="Hinton and Gumport, Nucleic Acid Res., 7:453-464 (1979)."><meta name="citation_reference" content="Hyman, Edward David, The Hyman Method: Oligonucleotide Synthesis and Plasmid Preparation, 1995."><meta name="citation_reference" content="Kaufmann et al., Eur. J. Biochem , 24:4 11 (1971)."><meta name="citation_reference" content="Kaufmann et al., Eur. J. Biochem, 24:4-11 (1971)."><meta name="citation_reference" content="Kossel and Roychoudrury, Eur. J. Biochem, 22:271 276 (1971)."><meta name="citation_reference" content="Kossel and Roychoudrury, Eur. J. Biochem, 22:271-276 (1971)."><meta name="citation_reference" content="Metzker et al., Nucleic Acids Res. , 22(20), 4259 4267 (1994)."><meta name="citation_reference" content="Metzker et al., Nucleic Acids Res., 22(20), 4259-4267 (1994)."><meta name="citation_reference" content="Modak, Biochemistry , 17, 3116 3120 (1978)."><meta name="citation_reference" content="Modak, Biochemistry, 17, 3116-3120 (1978)."><meta name="citation_reference" content="Sarfati et al., J. Biol. Chem. , 265(31), 18902 18906 (1990)."><meta name="citation_reference" content="Sarfati et al., J. Biol. Chem., 265(31), 18902-18906 (1990)."><meta name="citation_patent_number" content="US:5872244"><meta name="citation_patent_application_number" content="US:08/486,535"><link rel="canonical" href="http://www.google.com/patents/US5872244"/><meta property="og:url" content="http://www.google.com/patents/US5872244"/><meta name="title" content="Patent US5872244 - 3&#39; protected nucleotides for enzyme catalyzed template-independent creation of phosphodiester bonds"/><meta name="description" content="A method for the stepwise creation of phosphodiester bonds between desired nucleosides resulting in the synthesis of polynucleotides having a predetermined nucleotide sequence by preparing an initiation substrate containing a free and unmodified 3&#39;-hydroxyl group; attaching a mononucleotide selected according to the order of the predetermined nucleotide sequence to the 3&#39;-hydroxyl of the initiating substrate in a solution containing a catalytic amount of an enzyme capable of catalyzing the 5&#39; to 3&#39; phosphodiester linkage of the 5&#39;-phosphate of the mononucleotide to the 3&#39;-hydroxyl of the initiating substrate, wherein the mononucleotide contains a protected 3&#39;-hydroxyl group, whereby the protected mononucleotide is covalently linked to the initiating substrate and further additions are hindered by the 3&#39;-hydroxyl protecting group. Methods in which a mononucleotide immobilized on a solid support is added to a free polynucleotide chain are also disclosed."/><meta property="og:title" content="Patent US5872244 - 3&#39; protected nucleotides for enzyme catalyzed template-independent creation of phosphodiester bonds"/><meta property="og:type" content="book"/><meta property="og:site_name" content="Google Books"/><meta property="og:image" content="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><link rel="image_src" href="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><script>(function(){try{var aa=function(a,b,c,d){d=d||{};d._sn=["cfg",b,c].join(".");window.gbar.logger.ml(a,d)};var g=window.gbar=window.gbar||{},l=window.gbar.i=window.gbar.i||{},m={},n;function _tvn(a,b){var c=parseInt(a,10);return isNaN(c)?b:c}function _tvf(a,b){var c=parseFloat(a);return isNaN(c)?b:c}function _tvv(a){return!!a}function p(a,b,c){(c||g)[a]=b}g.bv={n:_tvn("2",0),r:"",f:".67.",e:"0",m:_tvn("0",1)};
function q(a,b,c){var d="on"+b;if(a.addEventListener)a.addEventListener(b,c,!1);else if(a.attachEvent)a.attachEvent(d,c);else{var f=a[d];a[d]=function(){var a=f.apply(this,arguments),b=c.apply(this,arguments);return void 0==a?b:void 0==b?a:b&&a}}}var s=function(a){return function(){return g.bv.m==a}},ba=s(1),ca=s(2);p("sb",ba);p("kn",ca);l.a=_tvv;l.b=_tvf;l.c=_tvn;l.i=aa;var da=window.gbar.i.i;var t,u,v,w;function ea(a){v=a}function fa(a){var b;if(b=v&&window.encodeURIComponent)b=a.href,b=!b.match(/^http[s]?:\/\/accounts\.google\.[^/]*\/ClearSID/i)&&!b.match(/^http[s]?:\/\/[^/]*\/accounts\/ClearSID/i);if(b=b&&encodeURIComponent(v()))a.href=a.href.replace(/([?&]continue=)[^&]*/,"$1"+b)}function ga(a){window.gApplication&&(a.href=window.gApplication.getTabUrl(a.href))}
function ha(a){var b=document.forms[0].q,c=window.encodeURIComponent&&b&&b.value,b=b&&b.placeholder;c&&c!=b&&(a.href=a.href.replace(/([?&])q=[^&]*|$/,function(a,b){return(b||"&")+"q="+encodeURIComponent(c)}))}n=l.a("")?ga:ha;
function x(a,b,c,d,f,e){var h=document.getElementById(a);if(h){var k=h.style;k.left=d?"auto":b+"px";k.right=d?b+"px":"auto";k.top=c+"px";k.visibility=u?"hidden":"visible";f&&e?(k.width=f+"px",k.height=e+"px"):(x(t,b,c,d,h.offsetWidth,h.offsetHeight),u=u?"":a)}}
var y=[],ia=function(a,b){y.push(b)},ja=function(a){a=a||window.event;var b=a.target||a.srcElement;a.cancelBubble=!0;null==t&&(a=document.createElement(Array.every||window.createPopup?"iframe":"div"),a.frameBorder="0",t=a.id="gbs",a.src="javascript:''",b.parentNode.appendChild(a),q(document,"click",z));var c=b,b=0;"gb3"!=c.className&&(c=c.parentNode);a=c.getAttribute("aria-owns")||"gbi";var d=c.offsetWidth,f=20<c.offsetTop?46:24;document.getElementById("tphdr")&&(f-=3);var e=!1;do b+=c.offsetLeft||
0;while(c=c.offsetParent);var c=(document.documentElement.clientWidth||document.body.clientWidth)-b-d,h,d=document.body,k=document.defaultView;k&&k.getComputedStyle?(d=k.getComputedStyle(d,""))&&(h=d.direction):h=d.currentStyle?d.currentStyle.direction:d.style.direction;h="rtl"==h;if("gbi"==a){for(d=0;k=y[d++];)k();A(null,window.navExtra);h&&(b=c,e=!0)}else h||(b=c,e=!0);u!=a&&z();x(a,b,f,e)},z=function(){u&&x(u,0,0)},A=function(a,b){var c,d=document.getElementById("gbi"),f=a;f||(f=d.firstChild);
for(;b&&(c=b.pop());){var e=d,h=c,k=f;w||(w="gb2");e.insertBefore(h,k).className=w}},ka=function(a,b,c){if((b=document.getElementById(b))&&a){a.className="gb4";var d=document.createElement("span");d.appendChild(a);d.appendChild(document.createTextNode(" | "));d.id=c;b.appendChild(d)}},la=function(){return document.getElementById("gb_70")},ma=function(){return!!u};p("qs",n);p("setContinueCb",ea);p("pc",fa);p("tg",ja);p("close",z);p("addLink",ka);p("almm",A);p("si",la);p("adh",ia);p("op",ma);var B=function(){},C=function(){},F=function(a){var b=new Image,c=D;b.onerror=b.onload=b.onabort=function(){try{delete E[c]}catch(a){}};E[c]=b;b.src=a;D=c+1},E=[],D=0;p("logger",{il:C,ml:B,log:F});var G=window.gbar.logger;var H={},na={},I=[],oa=l.b("0.1",.1),pa=l.a("1",!0),qa=function(a,b){I.push([a,b])},ra=function(a,b){H[a]=b},sa=function(a){return a in H},J={},K=function(a,b){J[a]||(J[a]=[]);J[a].push(b)},ta=function(a){K("m",a)},L=function(a,b){var c=document.createElement("script");c.src=a;c.async=pa;Math.random()<oa&&(c.onerror=function(){c.onerror=null;B(Error("Bundle load failed: name="+(b||"UNK")+" url="+a))});(document.getElementById("xjsc")||document.getElementsByTagName("body")[0]||
document.getElementsByTagName("head")[0]).appendChild(c)},N=function(a){for(var b=0,c;(c=I[b])&&c[0]!=a;++b);!c||c[1].l||c[1].s||(c[1].s=!0,M(2,a),c[1].url&&L(c[1].url,a),c[1].libs&&m.d&&m.d(c[1].libs))},O=function(a){K("gc",a)},P=null,ua=function(a){P=a},M=function(a,b,c){if(P){a={t:a,b:b};if(c)for(var d in c)a[d]=c[d];try{P(a)}catch(f){}}};p("mdc",H);p("mdi",na);p("bnc",I);p("qGC",O);p("qm",ta);p("qd",J);p("lb",N);p("mcf",ra);p("bcf",qa);p("aq",K);p("mdd","");p("has",sa);
p("trh",ua);p("tev",M);var Q=l.b("0.1",.001),R=0;
function _mlToken(a,b){try{if(1>R){R++;var c,d=a,f=b||{},e=encodeURIComponent,h=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&jexpid=",e("17483"),"&srcpg=",e("prop=22"),"&jsr=",Math.round(1/Q),"&ogev=",e("6IntU-7MDYvWsQS7sYBw"),"&ogf=",g.bv.f,"&ogrp=",e("1"),"&ogv=",e("1407723702.0"),"&oggv="+e("es_plusone_gc_20140723.0_p0"),"&ogd=",e("com"),"&ogc=",e("FRA"),"&ogl=",e("en")];f._sn&&(f._sn="og."+
f._sn);for(var k in f)h.push("&"),h.push(e(k)),h.push("="),h.push(e(f[k]));h.push("&emsg=");h.push(e(d.name+":"+d.message));var r=h.join("");S(r)&&(r=r.substr(0,2E3));c=r;var Aa=window.gbar.logger._aem(a,c);F(Aa)}}catch(Na){}}var S=function(a){return 2E3<=a.length},va=function(a,b){return b};function T(a){B=a;p("_itl",S,G);p("_aem",va,G);p("ml",B,G);a={};H.er=a}l.a("")?T(function(a){throw a;}):l.a("1")&&Math.random()<Q&&T(_mlToken);I.push(["m",{url:"//ssl.gstatic.com/gb/js/scm_7385cc5883250b43a39405734c1bea59.js"}]);g.mcf("c",{});g.sg={c:""};if(l.a("1")){var wa=l.a("");I.push(["gc",{auto:wa,url:"//ssl.gstatic.com/gb/js/abc/gci_91f30755d6a6b787dcc2a4062e6e9824.js",libs:"googleapis.client:plusone:gapi.iframes"}]);var xa={version:"gci_91f30755d6a6b787dcc2a4062e6e9824.js",index:"",lang:"en"};H.gc=xa;var U=function(a){window.googleapis&&window.iframes?a&&a():(a&&O(a),N("gc"))};p("lGC",U);l.a("1")&&p("lPWF",U)};window.__PVT="";if(l.a("1")&&l.a("1")){var V=function(a){U(function(){K("pw",a);N("pw")})};p("lPW",V);I.push(["pw",{url:"//ssl.gstatic.com/gb/js/abc/pwm_45f73e4df07a0e388b0fa1f3d30e7280.js"}]);var W=[],ya=function(a){W[0]=a},za=function(a,b){var c=b||{};c._sn="pw";B(a,c)},Ba={signed:W,elog:za,base:"https://plusone.google.com/u/0",loadTime:(new Date).getTime()};H.pw=Ba;var X=function(a,b){for(var c=b.split("."),d=function(){var b=arguments;a(function(){for(var a=g,d=0,e=c.length-1;d<e;++d)a=a[c[d]];a[c[d]].apply(a,b)})},f=g,e=0,h=c.length-1;e<h;++e)f=
f[c[e]]=f[c[e]]||{};return f[c[e]]=d};X(V,"pw.clk");X(V,"pw.hvr");p("su",ya,g.pw)};function Ca(){function a(){for(var b;(b=e[h++])&&"m"!=b[0]&&!b[1].auto;);b&&(M(2,b[0]),b[1].url&&L(b[1].url,b[0]),b[1].libs&&m.d&&m.d(b[1].libs));h<e.length&&setTimeout(a,0)}function b(){0<f--?setTimeout(b,0):a()}var c=l.a("1"),d=l.a(""),f=3,e=I,h=0,k=window.gbarOnReady;if(k)try{k()}catch(r){da(r,"ml","or")}d?p("ldb",a):c?q(window,"load",b):b()}p("rdl",Ca);var Da={D:1,H:2,da:3,p:4,W:5,M:6,F:7,g:8,ha:9,U:10,L:11,T:12,S:13,N:14,Q:15,P:16,fa:17,w:18,O:19,ga:20,ea:21,u:22,G:23,ja:24,ka:25,ia:26,A:27,j:28,o:29,k:30,ca:31,Z:32,$:33,J:34,K:35,ba:36,aa:37,Y:38,B:39,R:40,v:41,X:42,V:43,h:48,C:49,I:500},Y=[1,2,3,4,5,6,9,10,11,13,14,28,29,30,34,35,37,38,39,40,41,42,43,48,49,500];var Z=l.b("0.001",1E-4),Ea=l.b("1",1),Fa=!1,Ga=!1;if(l.a("1")){var Ha=Math.random();Ha<=Z&&(Fa=!0);Ha<=Ea&&(Ga=!0)}var Ia=Da,$=null;function Ja(){var a=0,b=function(b,d){l.a(d)&&(a|=b)};b(1,"");b(2,"");b(4,"");b(8,"");return a}
function Ka(a,b){var c=Z,d=Fa,f;f=a;if(!$){$={};for(var e=0;e<Y.length;e++){var h=Y[e];$[h]=!0}}if(f=!!$[f])c=Ea,d=Ga;if(d){d=encodeURIComponent;g.rp?(f=g.rp(),f="-1"!=f?f:"1"):f="1";c=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&oge=",a,"&ogex=",d("17483"),"&ogev=",d("6IntU-7MDYvWsQS7sYBw"),"&ogf=",g.bv.f,"&ogp=",d("22"),"&ogrp=",d(f),"&ogsr=",Math.round(1/c),"&ogv=",d("1407723702.0"),"&oggv="+
d("es_plusone_gc_20140723.0_p0"),"&ogd=",d("com"),"&ogl=",d("en"),"&ogc=",d("FRA"),"&ogus=",Ja()];if(b){"ogw"in b&&(c.push("&ogw="+b.ogw),delete b.ogw);var k;f=b;e=[];for(k in f)0!=e.length&&e.push(","),e.push(La(k)),e.push("."),e.push(La(f[k]));k=e.join("");""!=k&&(c.push("&ogad="),c.push(d(k)))}F(c.join(""))}}function La(a){"number"==typeof a&&(a+="");return"string"==typeof a?a.replace(".","%2E").replace(",","%2C"):a}C=Ka;p("il",C,G);var Ma={};H.il=Ma;setTimeout(function(){C(Ia.g)},0);}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var b=window.gbar.i.i;var c=window.gbar;var f=function(d){try{var a=document.getElementById("gbom");a&&d.appendChild(a.cloneNode(!0))}catch(e){b(e,"omas","aomc")}};c.aomc=f;}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var a=window.gbar;a.mcf("pm",{p:""});}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{window.gbar.rdl();}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
if (window['_OC_timingAction']) {window['_OC_timingAction']('patents_refpage');}</script><style>#gbar,#guser{font-size:13px;padding-top:1px !important;}#gbar{float:left;height:22px}#guser{padding-bottom:7px !important;text-align:right}.gbh,.gbd{border-top:1px solid #c9d7f1;font-size:1px}.gbh{height:0;position:absolute;top:24px;width:100%}#gbs,.gbm{background:#fff;left:0;position:absolute;text-align:left;visibility:hidden;z-index:1000}.gbm{border:1px solid;border-color:#c9d7f1 #36c #36c #a2bae7;z-index:1001}.gb1{margin-right:.5em}.gb1,.gb3{zoom:1}.gb2{display:block;padding:.2em .5em}.gb2,.gb3{text-decoration:none !important;border-bottom:none}a.gb1,a.gb4{text-decoration:underline !important}a.gb1,a.gb2,a.gb3,a.gb4{color:#00c !important}.gbi .gb3,.gbi .gb2,.gbi .gb4{color:#dd8e27 !important}.gbf .gb3,.gbf .gb2,.gbf .gb4{color:#900 !important}a.gb2:hover{background:#36c;color:#fff !important}#gbar .gbz0l{color:#000 !important;cursor:default;font-weight:bold;text-decoration:none !important}
#gbar { padding:.3em .6em !important;}</style></head><body  topmargin="3" marginheight="3"><div id=gbar><nobr><a onclick=gbar.qs(this);gbar.logger.il(1,{t:1}); class=gb1 id=gb_1 href="https://www.google.com/search?sa=N&tab=tw">Search</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:2}); class=gb1 id=gb_2 href="http://www.google.com/search?hl=en&tbm=isch&source=og&sa=N&tab=ti">Images</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:8}); class=gb1 id=gb_8 href="http://maps.google.com/maps?hl=en&sa=N&tab=tl">Maps</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:78}); class=gb1 id=gb_78 href="https://play.google.com/?hl=en&sa=N&tab=t8">Play</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:36}); class=gb1 id=gb_36 href="http://www.youtube.com/results?sa=N&tab=t1">YouTube</a> <a onclick=gbar.logger.il(1,{t:5}); class=gb1 id=gb_5 href="http://news.google.com/nwshp?hl=en&tab=tn">News</a> <a onclick=gbar.logger.il(1,{t:23}); class=gb1 id=gb_23 href="https://mail.google.com/mail/?tab=tm">Gmail</a> <a onclick=gbar.logger.il(1,{t:25}); class=gb1 id=gb_25 href="https://drive.google.com/?tab=to">Drive</a> <a class=gb3 href="http://www.google.com/intl/en/options/" onclick="this.blur();gbar.tg(event);return !1" aria-haspopup=true><u>More</u> <small>&#9660;</small></a><div class=gbm id=gbi><a onclick=gbar.logger.il(1,{t:24}); class=gb2 id=gb_24 href="https://www.google.com/calendar?tab=tc">Calendar</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:51}); class=gb2 id=gb_51 href="http://translate.google.com/?hl=en&sa=N&tab=tT">Translate</a><a onclick=gbar.logger.il(1,{t:17}); class=gb2 id=gb_17 href="http://www.google.com/mobile/?hl=en&tab=tD">Mobile</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:10}); class=gb2 id=gb_10 href="http://www.google.com/search?hl=en&tbo=u&tbm=bks&source=og&sa=N&tab=tp">Books</a><a onclick=gbar.logger.il(1,{t:212}); class=gb2 id=gb_212 href="https://wallet.google.com/manage/?tab=ta">Wallet</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:6}); class=gb2 id=gb_6 href="http://www.google.com/search?hl=en&tbo=u&tbm=shop&source=og&sa=N&tab=tf">Shopping</a><a onclick=gbar.logger.il(1,{t:30}); class=gb2 id=gb_30 href="http://www.blogger.com/?tab=tj">Blogger</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:27}); class=gb2 id=gb_27 href="http://www.google.com/finance?sa=N&tab=te">Finance</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:31}); class=gb2 id=gb_31 href="https://plus.google.com/photos?sa=N&tab=tq">Photos</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:12}); class=gb2 id=gb_12 href="http://www.google.com/search?hl=en&tbo=u&tbm=vid&source=og&sa=N&tab=tv">Videos</a><div class=gb2><div class=gbd></div></div><a onclick=gbar.logger.il(1,{t:66}); href="http://www.google.com/intl/en/options/" class=gb2>Even more &raquo;</a></div></nobr></div><div id=guser width=100%><nobr><span id=gbn class=gbi></span><span id=gbf class=gbf></span><span id=gbe></span><a target=_top id=gb_70 href="https://www.google.com/accounts/Login?service=&continue=http://www.google.com/patents%3Fhl%3Den&hl=en" class=gb4>Sign in</a><div style="display: none"><div class=gbm id=gbd5 aria-owner=gbg5><div class=gbmc><ol id=gbom class=gbmcc></ol></div></div></div></nobr></div><div class=gbh style=left:0></div><div class=gbh style=right:0></div><div role="alert" style="position: absolute; left: 0; right: 0;"><a href="http://www.google.com/patents/us5872244?hl=en&amp;output=html_text" title="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."><img border="0" src="http://www.google.com/images/cleardot.gif"alt="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."></a></div><div id="guser"><nobr></nobr></div><div style="clear:both;"></div><div id="gb-top-search-box" class="gb-top-search-box-small gb-reset"><table><tr><td class="logo"><a href="http://www.google.com/patents" class="logo-link"><img class="logo-img" src="/intl/en/images/logos/google_logo_41.png" alt="Go to Google Books Home" height="41"/></a></td><td><form action="http://www.google.com/search" name="f" id="vheadf" method="get"><span id="hf"></span><input type="hidden" name="tbm" value="pts"/><input type="hidden" name="tbo" value="1"/><input type="hidden" name="hl" value="en"/><table><tr><td><div class="inputs"><table><tr><td><div class="text-input"><input type="text" name="q" id="vheadq" class="text" maxlength="2048" size="31" value="" title="Search Patents" accesskey="s" autocomplete="off"/><script>window._OC_autoDir &&window._OC_autoDir('vheadq', 'tia-vheadq');</script></div></td><td><div class="submit-input"><input name="btnG" class="submit" type="submit" value=""/></div></td></tr></table></div></td><td class="col-ext-links"><div class="ext-links"><a href="http://www.google.com/advanced_patent_search">&lt;nobr&gt;Advanced Patent Search&lt;/nobr&gt;</a></div></td></tr></table></form></td></tr></table></div><div class="kd-appbar"><h2 class="kd-appname"><a href="/patents">Patents</a></h2><div class="kd-buttonbar left" id="left-toolbar-buttons"><a id="appbar-write-review-link" href=""></a><a id="appbar-view-print-sample-link" href=""></a><a id="appbar-view-ebook-sample-link" href=""></a><a id="appbar-patents-prior-art-finder-link" href="https://www.google.com/patents/related/US5872244"></a><a id="appbar-patents-discuss-this-link" href="http://www.google.com/url?id=pGhJBAABERAJ&amp;q=http://patents.stackexchange.com/redirect/google-patents%3Fpatent%3DUS5872244&amp;usg=AFQjCNGW84k-9elAPu7QtL5uf6B_1qKPFg" data-is-grant="true"></a><a id="appbar-read-patent-link" href="//docs.google.com/viewer?url=patentimages.storage.googleapis.com/pdfs/US5872244.pdf"></a><a id="appbar-download-pdf-link" href="//patentimages.storage.googleapis.com/pdfs/US5872244.pdf"></a></div><div class="kd-buttonbar right" id="right-toolbar-buttons"></div></div><div id="books-microdata" itemscope=""itemtype="http://schema.org/Book"itemid="http://www.google.com/patents/US5872244" style="display:none"><span itemprop="description">A method for the stepwise creation of phosphodiester bonds between desired nucleosides resulting in the synthesis of polynucleotides having a predetermined nucleotide sequence by preparing an initiation substrate containing a free and unmodified 3&#39;-hydroxyl group; attaching a mononucleotide selected...</span><span itemprop="url">http://www.google.com/patents/US5872244?utm_source=gb-gplus-share</span><span class="main-title" itemprop="name">Patent US5872244 - 3&#39; protected nucleotides for enzyme catalyzed template-independent creation of phosphodiester bonds</span><img itemprop="image" src="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"alt="Patent US5872244 - 3&#39; protected nucleotides for enzyme catalyzed template-independent creation of phosphodiester bonds" title="Patent US5872244 - 3&#39; protected nucleotides for enzyme catalyzed template-independent creation of phosphodiester bonds"></div><div style="display: none"><ol id="ofe-gear-menu-contents" class="gbmcc"><li class="gbe gbmtc"><a class="gbmt goog-menuitem-content" id="" href="http://www.google.com/advanced_patent_search">Advanced Patent Search</a></li></ol></div><table id="viewport_table" cellpadding="0" style="clear:both" cellspacing="0"><tr><td id="viewport_td"><div class=vertical_module_list_row><div id=intl_patents class=about_content><div id=intl_patents_v><table class="patent-bibdata"><tr><td class="patent-bibdata-heading">Publication number</td><td class="single-patent-bibdata">US5872244 A</td></tr><tr><td class="patent-bibdata-heading">Publication type</td><td class="single-patent-bibdata">Grant</td></tr><tr><td class="patent-bibdata-heading">Application number</td><td class="single-patent-bibdata">US 08/486,535</td></tr><tr><td class="patent-bibdata-heading">Publication date</td><td class="single-patent-bibdata">Feb 16, 1999</td></tr><tr><td class="patent-bibdata-heading">Filing date</td><td class="single-patent-bibdata">Jun 7, 1995</td></tr><tr><td class="patent-bibdata-heading">Priority date<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The priority date is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the date listed."></span></td><td class="single-patent-bibdata">Sep 2, 1994</td></tr><tr><td class="patent-bibdata-heading">Fee status<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The fee status is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the status or dates listed."></span></td><td class="single-patent-bibdata">Paid</td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Also published as</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/CA2197479A1">CA2197479A1</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP0781293A1">EP0781293A1</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP0781293A4">EP0781293A4</a>, </span><span class="patent-bibdata-value"><a href="/patents/WO1996007669A1">WO1996007669A1</a></span></span></td></tr><tr class="patent-bibdata-list-row alternate-patent-number"><td class="patent-bibdata-heading">Publication number</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value">08486535, </span><span class="patent-bibdata-value">486535, </span><span class="patent-bibdata-value">US 5872244 A, </span><span class="patent-bibdata-value">US 5872244A, </span><span class="patent-bibdata-value">US-A-5872244, </span><span class="patent-bibdata-value">US5872244 A, </span><span class="patent-bibdata-value">US5872244A</span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Inventors</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Andrew+C.+Hiatt%22">Andrew C. Hiatt</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Floyd+Rose%22">Floyd Rose</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Original Assignee</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=inassignee:%22Andrew+C.+Hiatt%22">Andrew C. Hiatt</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=inassignee:%22Floyd+D.+Rose%22">Floyd D. Rose</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Export Citation</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/US5872244.bibtex">BiBTeX</a>, </span><span class="patent-bibdata-value"><a href="/patents/US5872244.enw">EndNote</a>, </span><span class="patent-bibdata-value"><a href="/patents/US5872244.ris">RefMan</a></span></span></td></tr><tr class="patent-internal-links"><td colspan=2><span class="patent-bibdata-value"><a href="#backward-citations">Patent Citations</a> (35),</span> <span class="patent-bibdata-value"><a href="#npl-citations">Non-Patent Citations</a> (27),</span> <span class="patent-bibdata-value"><a href="#forward-citations">Referenced by</a> (36),</span> <span class="patent-bibdata-value"><a href="#classifications">Classifications</a> (17),</span> <span class="patent-bibdata-value"><a href="#legal-events">Legal Events</a> (10)</span> </td></tr><tr><td colspan=2 class="patent-bibdata-external-link-spacer-top"></td></tr><tr class="patent-bibdata-external-link-spacer-bottom"></tr><tr><td colspan=2><span class="patent-bibdata-heading">External Links:&nbsp;</span><span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=pGhJBAABERAJ&q=http://patft.uspto.gov/netacgi/nph-Parser%3FSect2%3DPTO1%26Sect2%3DHITOFF%26p%3D1%26u%3D/netahtml/PTO/search-bool.html%26r%3D1%26f%3DG%26l%3D50%26d%3DPALL%26RefSrch%3Dyes%26Query%3DPN/5872244&usg=AFQjCNGk1M6nZwaxBq9cEqo9zCxLVKOY4w">USPTO</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=pGhJBAABERAJ&q=http://assignments.uspto.gov/assignments/q%3Fdb%3Dpat%26pat%3D5872244&usg=AFQjCNFn29wNUIdE2m4lW5_2hgunWIBBjw">USPTO Assignment</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=pGhJBAABERAJ&q=http://worldwide.espacenet.com/publicationDetails/biblio%3FCC%3DUS%26NR%3D5872244A%26KC%3DA%26FT%3DD&usg=AFQjCNH4YpER92Er7A-0u6OMBy65VY5VJA">Espacenet</a></span></span></td></tr><tr class="patent-bibdata-group-spacer"></tr></table><div class="number-and-title"><span class="patent-title"><invention-title mxw-id="PT54392133" lang="EN" load-source="patent-office">3' protected nucleotides for enzyme catalyzed template-independent creation of phosphodiester bonds</invention-title></span><br><span class="patent-number">US 5872244 A</span></div><div class="patent-section patent-abstract-section"><div class="patent-section-header"><span class="patent-section-title">Abstract</span></div><div class="patent-text"><abstract mxw-id="PA37878025" lang="EN" load-source="patent-office"> <div class="abstract">A method for the stepwise creation of phosphodiester bonds between desired nucleosides resulting in the synthesis of polynucleotides having a predetermined nucleotide sequence by preparing an initiation substrate containing a free and unmodified 3'-hydroxyl group; attaching a mononucleotide selected according to the order of the predetermined nucleotide sequence to the 3'-hydroxyl of the initiating substrate in a solution containing a catalytic amount of an enzyme capable of catalyzing the 5' to 3' phosphodiester linkage of the 5'-phosphate of the mononucleotide to the 3'-hydroxyl of the initiating substrate, wherein the mononucleotide contains a protected 3'-hydroxyl group, whereby the protected mononucleotide is covalently linked to the initiating substrate and further additions are hindered by the 3'-hydroxyl protecting group. Methods in which a mononucleotide immobilized on a solid support is added to a free polynucleotide chain are also disclosed.</div>
  </abstract></div></div><div class="patent-section patent-drawings-section"><div class="patent-section-header"><span class="patent-section-title">Images<span class="patent-section-count">(4)</span></span></div><div class="patent-drawings-body"><div class="patent-drawings-carousel"><div class="drawings"><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5872244-1.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5872244-1.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5872244-2.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5872244-2.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5872244-3.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5872244-3.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5872244-4.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5872244-4.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div></div></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img" alt="Previous page"src="/googlebooks/images/kennedy/page_left.png"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img" alt="Next page"src="/googlebooks/images/kennedy/page_right.png"width="21" height="21" /></div></div></div><div class="patent-post-drawings"></div><div class="patent-section patent-claims-section"><div class="patent-section-header"><span class="patent-section-title">Claims<span class="patent-section-count">(6)</span></span></div><div class="patent-text"><div mxw-id="PCLM59505464" lang="EN" load-source="patent-office" class="claims">
    <claim-statement>We claim:</claim-statement> <div class="claim"> <div num="1" class="claim">
      <div class="claim-text">1. A mononucleoside 5'-triphosphate having a removable blocking moiety protecting the 3' position which is an ether and which has the following formula: ##STR17## wherein R<sub>2</sub> is a triphosphate group and R<sub>1</sub> is selected from the group consisting of methyl, CH<sub>3</sub> (CH<sub>2</sub>)<sub>N</sub>, wherein N is an integer from 1-10, methoxymethyl, methylthiomethyl, benzyloxymethyl, t-butyoxymethyl, 2-methoxyethoxymethyl, 2,2,2-trichloroethoxymethyl, bis(2-chloroethoxy)methyl, 2-(trimethylsilyl)ethoxymethyl, tetrahydropyranyl, 3'-bromotetrahydropyranyl, tetrahydrothiopyranyl, 4'-methoxytetrahydropyranyl, 4'-methoxytetrahydrothiopyranyl, 4'-methoxytetrahydrothiopyranyl S,S-dioxido, tetrahydrofuranyl, tetrahydrothiofuranyl, 1-ethoxyethyl, 1-methyl-1-methoxyethyl, 1-(isopropoxy)ethyl, tri(p-methoxyphenyl)methyl, di(p-methoxy)phenylmethyl, 2,2,2-trichloroethyl, 2-(phenylselenyl)ethyl, butyl, allyl, cinnamyl, p-chlorophenyl, benzyl, p-methoxybenzyl, o-nitrobenzyl, p-nitrobenzyl, p-halobenzyl, p-cyanobenzyl, 3-methyl-2-picolyl N-oxido, diphenylmethyl, 5-dibenzosuberyl, triphenylmethyl, α-naphthyldiphenylmethyl, p-methoxyphenyldiphenylmethyl, p-(p'-bromophenacyloxy)phenyldiphenylmethyl, 9-anthryl, 9-(9-phenyl-10-oxo)anthryl, benzisothiazolyl S,S-dioxido, trimethylsilyl, triethylsilyl, isopropyldimethylsilyl, t-butyl dimethylsilyl, (triphenylmethyl)dimethylsilyl, methyldiiosopropylsilyl, methyl di-t-butylsilyl, tribenzylsilyl, tri-p-xylylsilyl, triisopropylsilyl and triphenylsilyl; B is a nucleotide base.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="2" class="claim">
      <div class="claim-text">2. The mononucleoside 5'-triphosphate of claim 1, wherein R<sub>1</sub> is selected from the group consisting of bis(2-chloroethoxy)methyl, 4-methoxytetrahydropyranyl, tetrahydrofuranyl, 1-ethoxyethyl, tri(p-methoxyphenyl)methyl, di(p-methoxy)phenylmethyl and t-butyldimethylsilyl.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="3" class="claim">
      <div class="claim-text">3. The mononucleoside 5'-triphosphate of claim 1, wherein R<sub>1</sub> is selected from the group consisting of methyl, CH<sub>3</sub> (CH<sub>2</sub>)<sub>N</sub>, wherein N is an integer from 1-10, methoxymethyl, 2-methoxyethoxymethyl, trimethylsilyl and triethylsilyl.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="4" class="claim">
      <div class="claim-text">4. The mononucleoside 5'-triphosphate of claim 1, wherein R<sub>1</sub> is a 1-ethoxyethyl group and said mononucleoside 5'-triphosphate is a deoxynucleoside.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="5" class="claim">
      <div class="claim-text">5. The mononucleoside 5'-triphosphate of claim 1, wherein R<sub>1</sub> is a 1-ethoxyethyl group.</div>
    </div>
    </div> <div class="claim"> <div num="6" class="claim">
      <div class="claim-text">6. A mononucleoside 5'-triphosphate having a removable blocking moiety protecting the 3' position which is an ether and which has the following formula: ##STR18## wherein R<sub>2</sub> is a triphosphate group; R<sub>1</sub> is a silyl group; and B is a nucleotide base.</div>
    </div>
  </div> </div></div></div><div class="patent-section patent-description-section"><div class="patent-section-header"><span class="patent-section-title">Description</span></div><div class="patent-text"><div mxw-id="PDES67245508" lang="EN" load-source="patent-office" class="description">
    <heading>RELATED APPLICATIONS</heading> <p>This Application is a continuation-in-part of U.S. patent application Ser. No. 08/300,484 filed Sep. 2, 1994.</p>
    <heading>TECHNICAL FIELD</heading> <p>This invention relates to the synthesis of oligonucleotides and other nucleic acid polymers using template independent enzymes.</p>
    <heading>BACKGROUND OF THE INVENTION</heading> <p>Oligonucleotides are presently synthesized in vitro using organic synthesis methods. These methods include the phophoramidite method described in Adams et al., J. Amer. Chem. Soc., 105:661 (1983) and Froehler et al., Tetrahedron Lett., 24:3171 (1983) and the phosphotriester method described in German Offenlegungsshrift 264432. Other organic synthesis methods include those described by Froehler et al., U.S. Pat. No. 5,264,566 in which H-phosphonates are used to produce oligonucleotides.</p>
    <p>The phosphoramidite method of phosphodiester bond formation and oligonucleotide synthesis represents the current state of the art employed by most laboratories for the coupling of desired nucleotides without the use of a template. In this method, the coupling reaction is initiated by a nucleoside attached to a solid support.</p>
    <p>The 5'-hydroxyl group of the immobilized nucleoside is free for coupling with the second nucleoside of the chain to be assembled. Since the growing oligonucleotide chain projects a 5'-hydroxyl available for reaction with a mononucleotide, the direction of synthesis if referred to as 3' to 5'.</p>
    <p>Each successive mononucleotide to be added to the growing oligonucleotide chain contains a 3'-phosphoramidate moiety which reacts with the 5'-hydroxyl group of the support-bound nucleotide to form a 5' to 3' internucleotide phosphodiester bond. The 5'-hydroxyl group of the incoming mononucleotide is protected, usually by a trityl group, in order to prevent the uncontrolled polymerization of the nucleosides. After each incoming nucleoside is added, the protected 5'-hydroxyl group is deprotected, so that it is available for reaction with the next incoming nucleoside having a 3'-phosphoramidite group and a protected 5'-hydroxyl. This is followed by deprotection and addition of the next incoming nucleotide, and so forth.</p>
    <p>Between each nucleoside addition step, unreacted chains which fail to participate in phosphodiester bond formation with the desired nucleoside are chemically "capped" to prevent their further elongation. This usually involves chemical acetylation.</p>
    <p>This method and the other currently used organic methods while widely accepted require large amounts of costly monomers that require complex organic synthesis schemes to produce. In addition, these methods are complex in that the phosphoramidite method requires an oxidation step after each condensation reaction. The phosphotriester method requires that the subpopulation of oligonucleotides that have not had a monomer added in a particular cycle be capped in a separate reaction to prevent further chain elongation of these oligonucleotides.</p>
    <p>Other drawbacks of virtually all chemical methods of phosphodiester bond formation, is that the reaction must be performed in organic solvents and in the absence of water. Many of these organic solvents are toxic or otherwise hazardous. Another drawback to chemical synthesis is that it is at best 98 percent efficient at each cycle. In other words, following each nucleotide addition, at least 2 percent of the growing oligonucleotide chains are capped, resulting in a yield loss. The total yield loss for the nucleotide chain being synthesized thus increases with each nucleotide added to the sequence.</p>
    <p>For example, assuming a yield of 98 percent per nucleotide addition, the synthesis of a polynucleotide consisting of 70 mononucleotides would experience a yield loss of nearly 75 percent. Furthermore, the object nucleotide chain would require isolation from a reaction mixture of polynucleotides, nearly 75 percent of which consist of capped oligonucleotides ranging between 1 and 69 nucleotides in length.</p>
    <p>This inherent inefficiency in chemical synthesis of oligonucleotides ultimately limits the length of oligonucleotide that can be efficiently produced to oligonucleotides having 50 nucleic acid residues or less.</p>
    <p>A need exists for a method which improves the efficiency of phosphodiester bond formation and which could ultimately be capable of producing shorter chain oligonucleotides in higher yields and longer chain polynucleotides in acceptable yields. In addition, a need exists for a polynucleotide synthesis system which is compatible with pre-existing polynucleotides, such as vector DNAs, so that desired polynucleotide sequences can readily be added on to the pre-existing sequences. Chemical coupling by the phosphoramidite method is not compatible with "add-on" synthesis to pre-existing polynucleotides. Enzyme catalyzed phosphodiester bond formation, however, can be performed in an aqueous environment utilizing either single or double stranded oligo- or polynucleotides to initiate the reaction. These reaction conditions also greatly minimize the use of toxic and hazardous materials.</p>
    <p>The 3' to 5' direction of synthesis inherent to the phosphoramidite method of phosphodiester bond formation cannot be enzyme catalyzed. All known enzymes capable of catalyzing the formation of phosphodiester bonds do so in the 5' to 3' direction since the growing polynucleotide strand always projects a 3'-hydroxyl available for attachment of the next nucleoside.</p>
    <p>There are many enzymes capable of catalyzing the formation of phosphodiester bonds. One class of such enzymes, the polymerases, are largely template dependent in that they add a complementary nucleotide to the 3' hydroxyl of the growing strand of a double stranded polynucleotide. However, some polymerases are template independent and primarily catalyze the formation of single stranded nucleotide polymers. Another class of enzyme, the ligases, are template independent and form a phosphodiester bond between two polynucleotides or between a polynucleotide and a mononucleotide.</p>
    <p>Addition of single nucleotides to DNA fragments, catalyzed by deoxynucleotidyl terminal transferase (TdTase), has previously been described by Deng and Wu, Meth. Enzymol., 100:96-116, 1983. These reaction conditions did not involve transient protection of the 3'-hydroxyl nor were they intended to be used for the sequential creation of phosphodiester bonds to synthesize a predetermined nucleotide sequence. The presence of unprotected 3'-hydroxyls resulted in a highly heterogeneous population of reaction products.</p>
    <p>Similarly, prior attempts to catalyze synthesis of very short pieces of RNA or DNA using protected nucleotide monophosphates or diphosphates resulted in unacceptably low levels of the desired phosphodiester bond formation or required excessive amounts of enzyme to achieve acceptable efficiencies. These problems were largely due to unavoidable heterogeneity of the mononucleotide building blocks or to the very high turnover number of the enzyme, necessitating extremely long incubation times (see, for example, Hinton and Gumport, Nucleic Acids Res. 7:453-464, 1979; Kaufman et al., Eur. J. Biochem., 24: 4-11, 1971). These experiments were limited to 5'-monophosphates and diphosphates. No attempts have been made to catalyze controlled DNA synthesis using 5'-triphosphates protected at the 3' position.</p>
    <p>Enzyme catalyzed creation of a single phosphodiester bond between the free 3<sup>1</sup> -hydroxyl group of an oligonucleotide chain and the 5'-phosphate of a mononucleotide requires protection of the 3'-hydroxyl of the mononucleotide in order to prevent multiple phosphodiester bond formations and hence repeated mononucleotide additions. Protection of the 3'-hydroxyl of the mononucleotide ideally involves a transient blocking group which can readily be removed in order to allow subsequent reactions. Flugel et al., Biochem. Biophys. Acta. 308:35-40, 1973, report that nucleoside triphosphates with blocked 3'-hydroxyl groups cannot be prepared directly. This lack of 3' blocked triphosphates necessitated previous processes to utilize lower energy and thus more inefficient 3' blocked monophosphates and diposphates. Synthetic techniques to create 3' block triphosphates would be highly desirable, because this could enable stepwise enzyme catalyzed phosphodiester bond formation leading to polynucleotide synthesis.</p>
    <p>These prior attempts at synthesizing oligonucleotides using a template independent polymerase were extremely inefficient resulting in the production of very short oligonucleotides. The inefficiency of these methods made these methods useless for practical synthesis of oligonucleotides.</p>
    <p>The present invention allows the creation of phosphodiester bonds between nucleotides using a template independent polymerase to create oligonucleotides having a predetermined sequence. This enzyme catalysis can vastly improve the efficiency of phosphodiester bond formation between desired nucleotides compared to current techniques of chemical coupling and can be carried out in the presence of other biological molecules such as pre-existing sequences of single or double stranded DNA as well as other types of enzymes. In addition, the very high specificity inherent to enzyme catalysis allows only coupling of a 5'-phosphate to a 3'-hydroxyl. The coupling of two mononucleosides, as well as various other side reactions inherent to chemical coupling techniques, simply do not occur.</p>
    <p>A further advantage of the present invention is realized by using 3' blocked triphosphates having high energy phosphate bonds which an enzyme can utilize to drive the reaction to greater completion level than when other monophosphates and diphosphates are used. In addition, triphosphates are less strongly hydrated than the diphosphate, which also tends to drive catalytic hydrolysis of the triphosphate to completion.</p>
    <p>Clearly, the availability of a homogeneous population of protected mononucleotide triphosphates and enzymes capable of efficiently joining protected nucleotides to initiating substrates will enable the creation of a highly uniform population of synthetic polynucleotides resulting from stepwise phosphodiester bond formation.</p>
    <heading>SUMMARY OF THE INVENTION</heading> <p>A number of methods have been discovered by which the 3'-hydroxyl group of a deoxynucleotide triphosphate can be effectively protected and deprotected and wherein the protected nucleotide is utilized by a template independent polymerase to create a phosphodiester bond permitting the synthesis of oligonucleotides or polynucleotides having a desired predetermined sequence.</p>
    <p>Therefore, in accordance with the present invention, a method is provided for the synthesis of a polynucleotide of a predetermined sequence of which method includes the steps of:</p>
    <p>A. providing an initiating substrate comprising a nucleoside having an unprotected 3'-hydroxyl group; and</p>
    <p>B. reacting under enzymatic conditions in the presence of a catalytic amount of an enzyme the 3'-hydroxyl group of the initiating substrate with a nucleoside 5'-triphosphate having a removable blocking moiety protecting the 3' position of the nucleoside 5'-triphosphate and selected according to the order of the predetermined sequence, so that enzyme catalyzes the formation of a 5' to 3' phosphodiester linkage between the unprotected 3'-hydroxyl group of the initiating substrate and the 5'-phosphate of the nucleoside 5'-triphosphate to produce the polynucleotide.</p>
    <p>In other embodiments of the present invention, the method further comprises the step:</p>
    <p>C. removing the blocking moiety protecting the 3' position of said nucleotide 5'-triphosphate to produce an initiating substrate having an unprotected 3'-hydroxyl group.</p>
    <p>In other embodiments, steps (b) and (c) are repeated at least once to add additional nucleotides to the initiating substrate by alternatively adding a nucleoside 5'-triphosphate with a removable blocking moiety at its 3' position, deblocking the 3' position of the terminal nucleoside and then adding another nucleoside 5'-triphosphate with a removable blocking group at its 3' position. Repetition of steps (b) and (c) can also be carried out to produce an oligonucleotide or polynucleotide having a predetermined sequence.</p>
    <p>The present invention contemplates initiating substrates that are deoxynucleosides, nucleotides, single or double stranded oligonucleotides, single or double stranded polynucleotides, oligonucleotides attached to nonnucleoside molecules and the like.</p>
    <p>The present invention contemplates embodiments in which the substrate is immobilized on a solid support. Preferred solid supports include cellulose, Sepharose, controlled-pore glass, silica, Fractosil, polystyrene, styrene divinyl benzene, agarose, and crosslinked agarose and the like.</p>
    <p>The present invention contemplates the use of template independent polynucleotide polymerases such as terminal deoxynucleotidyl transferase from any number of sources including eukaryotes and protharyotes.</p>
    <p>The methods of the present invention utilize removable blocking moieties that block the 3' position of nucleoside 5'-triphosphates used in the methods. Preferred removable blocking moieties can be removed in under 10 minutes to produce a hydroxyl group at the 3' position of the 3' nucleoside. Removable blocking groups contemplated include carbonitriles, phosphates, carbonates, carbamates, esters, ethers, borates, nitrates, sugars, phosphoramidates, phenylsulfenates, sulfates and sulfones.</p>
    <p>The methods of the present invention contemplate removing the removable blocking moiety using a deblocking solution that preferably contains divalent cations such as Co++ and a biological buffer such as comprises a buffer selected from the group consisting of dimethylarsinic acid, tris hydroxymethyl! amino methane, and 3- m-morpholine! propanosulphonic acid. Other embodiments of the present invention utilize an enzyme present in the deblocking solution to remove the removable blocking moiety.</p>
    <p>The present invention also contemplates methods in which the nucleoside 5'-triphosphate having the removable blocking moiety at its 3' position is immobilized in a solid support and reacted with free initiating substrates. The solid support is linked to the nucleoside 5'-triphosphate at the 3'-hydroxyl group, thereby acting as a removable blocking moiety at the 3' position. Attachment of the nucleoside to the support is transient, thereby enabling the release of the newly synthesized product from the support and regeneration of the free and unmodified 3'-hydroxyl to allow the next nucleotide addition to occur.</p>
    <p>Thus, in some embodiments of the present invention the deblocking solution would remove the removable blocking moiety at the position of the nucleoside and thus release the growing polynucleotide from the solid support.</p>
    <p>The present invention also includes polynucleotides having a predetermined sequence provided according to the methods of this invention. Applications for using polynucleotides and oligonucleotides of the present invention in molecular cloning and/or expression of genes, peptides or proteins.</p>
    <p>Also contemplated by the present invention are compositions of matter comprising a catalytic amount of a template independent enzyme and a nucleoside 5'-triphosphate having a removable blocking moiety protecting the 3' position of said nucleoside 5'-triphosphate. Additional compositions of matter further comprising an initiating substrate are also contemplated.</p>
    <heading>BRIEF DESCRIPTION OF THE DRAWINGS</heading> <p>FIG. 1. A diagram showing enzymatic synthesis of an oligonucleotide using a template independent polymerase and a nucleoside 5' triphosphate having a removable blocking moiety at its 3' position is shown.</p>
    <p>FIG. 2. A diagram of a nucleotide having a removable blocking moiety at its 3' position is shown.</p>
    <p>FIG. 3. A diagram showing an apparatus for automating the enzymatic synthesis of polynucleotides is shown.</p>
    <p>FIG. 4. A diagram showing an apparatus for automating the enzymatic synthesis of polynucleotides is shown.</p>
    <heading>BRIEF DESCRIPTION OF THE INVENTION</heading> <heading>A. Definitions</heading> <p>DNA: Deoxyribonucleic acid.</p>
    <p>RNA: Ribonucleic acid.</p>
    <p>Nucleotide: A subunit of a nucleic acid comprising a phosphate group, a 5-carbon sugar and nitrogen containing base. In RNA, the 5-carbon sugar is ribose. In DNA, it is a 2-deoxyribose. The term also includes analogs of such subunits.</p>
    <p>Nucleoside: Includes a nucleosidyl unit and is used interchangeably therewith, and refers to a subunit of a nucleic acid which comprises a 5-carbon sugar and a nitrogen containing base. The term includes not only those nucleosidyl units having A, G, C, T and U as their bases, but also analogs and modified forms of the naturally-occurring bases, such as pseudoisocytosine and pseudouracil and other modified bases (such as 8-substituted purines). In RNA, the 5-carbon sugar is ribose; in DNA, it is 2'-deoxyribose. The term nucleoside also includes other analogs of such subunits, including those which have modified sugars such as 2'-O-alkyl ribose.</p>
    <p>Polynucleotide: A nucleotide multimer generally about 50 nucleotides or more in length. These are usually of biological origin or are obtained by enzymatic means. ##STR1## wherein phosphodiester groups may be used as internucleosidyl phosphorus group linkages (or links) to connect nucleosidyl units.</p>
    <p>Hydrocarbyl: An organic radical composed of carbon and hydrogen which may be aliphatic (including alkyl, alkenyl, and alkynyl groups and groups which have a mixture of saturated and unsaturated bonds), alicyclic (carbocyclic), aryl (aromatic) or combination thereof; and may refer to straight-chained, branched-chain, or cyclic structures or to radicals having a combination thereof, as well as to radicals substituted with halogen atom(s) or heteroatoms, such as nitrogen, oxygen, and sulfur and their functional groups (such as amino, alkoxy, aryloxy, lactone groups and the like), which are commonly found in organic compounds and radicals.</p>
    <p>Non-nucleoside monomeric unit: A monomeric unit wherein the base, the sugar and/or the phosphorus backbone or other internuclosidyl linkage of a nucleoside has been replaced by other chemical moieties.</p>
    <p>Polypeptide and Peptide: A linear series of amino acid residues connected on to the other by peptide bonds between the alpha-amino and carboxyl groups of adjacent residues.</p>
    <p>Protein: A linear series of greater than about 50 amino acid residues connected one to the other as in a polypeptide.</p>
    <p>Gene: A segment of DNA coding for an RNA transcript that is itself a structural RNA, such as ribosomal RNA or codes for a polypeptide. The segment of DNA is also equipped with a suitable promoter, termination sequence and optionally other regulatory DNA sequences.</p>
    <p>Structural Gene: A gene coding for a structural RNA and being equipped with a suitable promoter, termination sequence and optionally other regulatory DNA sequences.</p>
    <p>Promoter: A recognition site on a DNA sequence or group of DNA sequences that provide an expression control element for a gene and to which RNA polymerase specifically binds and initiates RNA synthesis (transcription) of that gene.</p>
    <p>Oligonucleotide: A chain of nucleosides which are linked by internucleoside linkages which is generally from about 2 to about 50 nucleosides in length. They may be chemically synthesized from nucleoside monomers or produced by enzymatic means. The term oligonucleotide refers to a chain of nucleosides which have internucleosidyl linkages linking the nucleoside monomer and, thus, includes oligonucleotide containing nucleoside analogs, oligonucleotide having internucleosidyl linkages such that one or more of the phosphorous group linkages between monomeric units has been replaced by a non-phosphorous linkage such as a formacetal linkage, a thioformacetal linkage, a sulfamate linkage, or a carbamate linkage. It also includes nucleoside/non-nucleoside polymers wherein both the sugar and the phosphorous moiety have been replaced or modified such as mopholino base analogs, or polyamide base analogs. It also includes nucleoside/non-nucleoside polymers wherein the vase, the sugar, and the phosphate backbone of a nucleoside are either replaced by a non-nucleoside moiety or wherein a non-nucleoside moiety is inserted into the nucleoside/non-nucleoside polymer. Thus an oligonucleotide may be partially or entirely phophonothioates, phosphorothioate phosphorodithioate phosphoramidate or neutral phosphate ester such as phosphotriesters oligonucleotide analogs.</p>
    <p>Removable Blocking Moiety: A removable blocking moiety is a moiety which is attached to the oxygen at the 3' position of a nucleoside or the equivalent position in a nucleoside analog. The removable blocking moiety prevents reaction of the 3' oxygen when present and is removable under deblocking conditions so that the 3' oxygen can then participate in a chemical reaction.</p>
    <heading>A. Methods</heading> <p>Generally, the present invention provides methods for synthesizing oligonucleotides and polynucleotides having a predetermined sequence using a template independent polymerase and nucleoside having the 3' position blocked with a removable blocking moiety so that single nucleosides are added to the growing oligonucleotide. Single nucleosides are added to the growing chain by removing the blocking moiety at the 3' position of the terminal nucleoside of the growing oligonucleotide so that the next blocked nucleoside can be added to the oligonucleotide. This process is then repeated until the oligonucleotide having the predetermined sequence is produced.</p>
    <p>Thus, in accordance with this embodiment of the present invention, a method comprises the steps of:</p>
    <p>(a) providing an initiating substrate comprising a nucleoside having an unprotected 3'-hydroxyl group; and</p>
    <p>(b) reacting under enzymatic conditions in the presence of a catalytic amount of an enzyme said 3'-hydroxyl group of said initiating substrate with a nucleoside 5'-triphosphate having a removable blocking moiety protecting the 3' position of said nucleoside 5'- triphosphate and selected according to the order of said predetermined sequence, whereby said enzyme catalyzes the formation of a 5' to 3' phophodiester linkage between said unprotected 3'-hydroxyl group of said initiating substrate and the 5'-phosphate of said nucleoside 5'-triphosphate to produce said polynucleotide.</p>
    <p>In preferred embodiments, the methods of the present invention further comprises the step of:</p>
    <p>(c) removing the blocking moiety protecting the 3' position of said nucleoside 5'-triphosphate to produce an initiating substrate having an unprotected 3'-hydroxyl group.</p>
    <p>This additional step regenerates a reactive atom at the 3' position of the terminal nucleoside so that this atom can be used to form a bond with the next nucleoside and thus extend the length of the oligonucleotide by one nucleoside.</p>
    <p>The methods of the present invention also include methods in which the above steps (b) and (c) are repeated at least once to produce an oligonucleotide. This process can be repeated many times to produce oligonucleotides of selected length. This process can also be repeated many times such that each particular nucleoside added to the oligonucleotide having a preselected sequence.</p>
    <p>1. Initiating Substrates</p>
    <p>An initiating substrate of the present invention is prepared containing a nucleoside with a free and unmodified 3'-hydroxyl group. As is well understood by those of ordinary skill in the art, nucleotide derivatives of the nucleosides adenosine, cytidine, guanosine, uridine and thymidine can be assembled to form oligonucleotides and polynucleotides. According to the method of the present invention, the initiating substrate may contain a single nucleoside having a free and unmodified 3'-hydroxyl group, or a preassembled oligo- or polynucleotide may be provided as an initiating substrate, so long as the oligo- or polynucleotide has a free and unmodified 3' -hydroxyl group.</p>
    <p>One skilled in the art will understand that an initiating substrate could be provided in a form in which a nucleoside has a removable blocking moiety at its 3' position which is subsequently removed using a deblocking process so that the initiating substrate now has the free unprotected 3' hydroxyl group useful in the present invention.</p>
    <p>The initiating substrates of the present invention include the termini of polynucleotides frequently generated and used in various cloning and molecular biology techniques. Examples of these initiating substrates include the termini of DNA or RNA vectors, single-stranded or double-stranded fragments, single-stranded or double-stranded RNA fragments and RNA or DNA oligonucleotides.</p>
    <p>In the preferred embodiments, initiating substrates will consist wholly or in part of an oligo- or polynucleotide. The initiating substrate can be any arrangement of nucleosides which enables the enzyme to create a phosphodiester bond between the 3'-hydroxyl of a nucleoside and the 5'-phosphate of a mononucleotide. Initiating substrates may be based wholly or in part on ribonucleic acids (RNA) or deoxyribonucleic acids (DNA) and may be single stranded or multi-stranded. In addition, initiating substrates can include other types of naturally occurring or synthetic molecules (non-nucleosides) which may enable or enhance the ability of the enzyme to create a phosphodiester bond or which may facilitate the manipulation of reaction components and by-products. An example of this would be a linker molecule (commonly used linkers consist of C, O, N, and H e.g. AFFI-GEL™ 10: R--OCH<sub>2</sub> CONH(CH<sub>2</sub>)<sub>2</sub> NHCO(CH<sub>2</sub>)<sub>2</sub> COON(CH<sub>2</sub>)<sub>2</sub> which would serve to provide a convenient method for attaching an initiating substrate to a solid support.</p>
    <p>The sequential creation of phosphodiester bonds and hence the addition of nucleotides to the initiating substrate may be performed entirely in solution, or the initiating substrate may be attached to an insoluble matrix. Attachment to an insoluble matrix will permit the rapid separation of the substrate from unreacted reagents in order to prepare the substrate for the addition of the next nucleotide. For this reason, the substrate is preferably affixed to a solid support matrix during each reaction creating a phosphodiester bond.</p>
    <p>Insoluble matrices suitable for use as solid supports include cellulose, SEPHAROSE™, controlled-pore glass (CPG), polystyrene, silica, agarose, and the like.</p>
    <p>Reagents, buffers and solvents suitable for use with the present invention are capable of flowing through the solid support matrix, by which means the initiating substrate is brought into contact with these materials. The growing nucleotide chain remains attached to the solid support as the various reagents, buffers and solvents sequentially flow therethrough. The solid support matrix is preferably contained within a synthesis column, to which reagents, buffers and solvents are provided.</p>
    <p>Attachment of the initiating substrate to the solid support can be by covalent bonding. Numerous methods for the covalent attachment of molecules to insoluble matrices have been described and are well understood by those of ordinary skill in the art. In the preferred embodiment an oligonucleotide chain may be linked to alkylamine derivatized polystyrene or CPG by way of a covalent phosphoramidate bond although numerous strategies for linking oligonucleotides to solid supports have been described. The choice of an appropriate linking strategy will depend on the specific requirements of stability, charge interactions, solubility and the like.</p>
    <p>Alternatively, attachment of the initiating substrate to the solid support can be by non-covalent interactions. Numerous methods for the transient attachment of molecules to insoluble matrices have been described and are well understood by those of ordinary skill in the art. For example, an oligonucleotide derivative containing single or multiple biotin molecules may be attached to avidin-agarose or streptavidin-agarose to form a non-covalent linkage between the oligonucleotide and the insoluble agarose matrix.</p>
    <p>In general, it is envisioned that single and double stranded oligo- and polynucleotides based on DNA or RNA may be covalently or non-covalently bound to solid supports to form a variety of initiating substrates. Regardless of the strategy employed to attach an initiating substrate to an insoluble matrix, a nucleoside with a free and unmodified 3'-hydroxyl group will always be available for enzyme catalyzed creation of a phosphodiester bond.</p>
    <p>2. Template Independent Enzymes</p>
    <p>Mononucleotides are added to the free and unmodified 3'-hydroxyl group of the initiating substrate by reacting the substrate with the 5'-phosphate of the selected mononucleotide in the presence of a catalytic amount of an enzyme capable of creating the phosphodiester bond covalently linking the 5'-phosphate of the mononucleotide with the 3'-hydroxyl of the substrate. The enzyme is preferable a template independent enzyme such as a template independent polynucleotide polymerase. Template independent enzymes such as template independent polynucleotide polymerases are capable of catalyzing the formation of a phosphodiester bond between the nucleotides without requiring the presence of a complementary nucleotide strand for activity. Thus, the template independent enzymes such as template independent polynucleotide polymerases are able to catalyze the formation of single-stranded nucleic acid polymers without requiring a complementary nucleic acid strand to act as a template. Examples of template independent polynucleotide polymerases include terminal deoxynucleotidyl transferases. Template independent polynucleotide polymerases can be isolated from a number of sources including calf thymus and other sources of lymphocytes. A particularly preferred polymerase is terminal deoxynucleotidyl transferase (TdTase, EC 2.7.7.31).</p>
    <p>Enzymes capable of being utilized with the present invention can be readily identified by those of ordinary skill in the art, and are employed under appropriate and well understood conditions. Example enzymatic conditions for deoxynucleotidyl transferase include a pH of 6.8 maintained by a potassium cacodylate buffer, 8 mmol/l of MgCl<sub>2</sub>, 1 mmol of β mercaptoethambol, 0.33 mmol/l of ZnSO<sub>4</sub>. One skilled in the art will understand that these enzymatic conditions may vary while still allowing the enzyme to catalyze the desired reaction.</p>
    <p>3. Nucleosides Having Removable Blocking Moieties</p>
    <p>In accordance with the present invention, the mononucleotide has its 3' position protected by a removable blocking moiety so that a single phosphodiester linkage is formed between the free 3'-hydroxyl of the initiating substrate and the 5'-phosphate group of the mononucleotide. The removable blocking moiety protecting the 3' position of the mononucleotide prevents the catalytic creation of multiple phosphodiester bonds and hence multiple nucleotide additions.</p>
    <p>The present invention contemplates a nucleoside 5'-phosphate of the present invention has a removable blocking moiety protecting the 3' position having the following formula: ##STR2## wherein R2 is triphosphate, diphosphate or monophosphate; and wherein R1 is a hydrocarbyl. In preferred embodiments the nucleoside 5' phosphate of the above formula has an R2 group which is triphosphate and an R1 group which is a hydrocarbyl.</p>
    <p>Nucleotides having a removable blocking moiety protecting the 3' position suitable for use with the present invention have a structure corresponding to Formula 1, that has a structure which is compatible with the utilization of the entire nucleotide for the creation of a phosphodiester bond by the enzyme. ##STR3##</p>
    <p>B is the nucleotide base and R<sub>2</sub> represents the appropriate mono-, di- or triphosphate. R<sub>1</sub> can be an ester linkage, COR<sub>1</sub> ', which forms the structure nucleotide-3'-O--CO--R<sub>1</sub> '. R<sub>1</sub> ' can be any alkyl or aryl group compatible with the utilization of the molecule by the enzyme for the creation of an internucleotide phosphodiester bond. The chemistry of esters as protecting groups for hydroxyls is well established. Removable blocking moieties including formate, benzoyl formate, acetate, substituted acetate, propionate, isobutyrate, levulinate, crotonate, benzoate, napthoate and many other esters have been described in detail (See, Greene, T. W., Protective Groups in Organic Chemistry, John Wiley &amp; Sons, New York, 1981). Esters in general are readily removed, usually in the presence of base, to regenerate the hydroxyl group and thus are useful as removable blocking moieties.</p>
    <p>Ester removable blocking moieties are formed by reacting the nucleotide with the appropriate acid anhydride. Alternatively, a carboxylic acid can be esterified with the 3'-hydroxyl of the nucleotide in the presence of water after activation by reaction with carbonyl diimidazole (See, Schafer et al., Meth Enzymol., 126, 682-712.)</p>
    <p>The present invention also contemplates a nucleoside 5'-phosphate having a removable blocking moiety protecting the 3' position which is an ester and which has the following formula: ##STR4## wherein R2 is triphosphate, diphosphate or monophosphate; and wherein R1 is any aliphatic or aromatic organic ester.</p>
    <p>The present invention also contemplates a nucleoside 5'-phosphate having a removable blocking moiety protecting the 3' position which is an ester and which has the following formula: ##STR5## wherein R is selected from the group consisting of: formate, benzoylformate, chloroacetate, dicholoroacetate, trichloroacetate, trifluoroacetate, methoxyacetate, triphenylmethoxyacetate, phenoxyacetate, p-chlorophenoxyacetate, 2,6-dichloro-4-methylphenoxyacetate, 2,6-dichloro-4-(1,1,3,3-tetramethylbutyl)phenoxyacetate, 2,4-bis(1,1-dimethylpropyl)phenoxyacetate, chlorodiphenylacetate, phenylacetate, 3-phenylpropionate, 3-benzoylpropionate, isobutyrate, monosuccinoate, 4-oxopentanoate, pivaloate, adamanioate, crotonate, 4-methoxycrotonate, (E)-2-methyl-2-butenoate, o-(dibromomethyl)benzoate, o-(methoxycarbonyl)benzoate, p-phenylbenzoate, 2,4,6-trimethylbenzoate and α-naphthoate.</p>
    <p>The present invention also contemplates a nucleoside 5'-phosphate having a removable blocking moiety protecting the 3' position which is an ester and which has the following formula: ##STR6## wherein R2 is triphosphate, diphosphate or monophosphate; and wherein R is selected from the group consisting of: H, CH<sub>3</sub>, CH<sub>3</sub> (CH<sub>2</sub>)<sub>N</sub> where N is an integer from 1 to 12, (CH<sub>3</sub>)<sub>x+1</sub> (CH)<sub>x</sub> where x is an integer from 1 to 12, (CH<sub>3</sub>)<sub>x+1</sub> (CH)<sub>x</sub> (CH<sub>2</sub>)<sub>n</sub> where x and n are independent integers from 1 to 12, C<sub>x</sub> (CH<sub>3</sub>)<sub>3x-</sub>(x-1) (CH2)<sub>n</sub> where x and n are independent integers from 1 to 12, ##STR7## where R1, R3, R4, R5 and R6 is CH<sub>3</sub>, H or NO<sub>2</sub>.</p>
    <p>In preferred embodiments, the nucleoside 5' phosphates of the present invention are deoxynucleoside 5' triphosphate having a removable blocking moiety protecting the 3' position which can be any of the blocking groups disclosed in this specification or equivalents of those groups.</p>
    <p>An alternative type of removable blocking moiety utilizes an ether linkage which forms the structure nucleotide --3'--O--R'<sub>1</sub>. In this instance R'<sub>1</sub> can be methyl, substituted meythyl, ethyl, substituted ethyl, butyl, allyl, cinnamyl, benzyl, substituted benzyl, anthryl or silyl. The chemistry involved in using ethers as removable blocking moieties for hydroxyls is well known in the art. Numerous ethers have been described and are useful for transiently protecting hydroxyls and similar chemical groups.</p>
    <p>In other embodiments, a nucleoside 5' phosphate of the present invention has a removable blocking moiety protecting the 3' position which is an ether and which has the following formula: ##STR8## wherein R2 is triphosphate, diphosphate or monophosphate; and wherein R1 is an ether selected from the group consisting of a substituted or unsubstituted: aliphatic group, aromatic group or silyl group.</p>
    <p>In other embodiments, a nucleoside 5' phosphate of the present invention has a removable blocking moiety protecting the 3' position which is an ether and which has the following formula: ##STR9## wherein R2 is triphosphate, diphosphate or monophosphate; and wherein R1 is an ether selected from the group consisting of a methyl, methoxymethyl, methylthiomethyl, benzyloxymethyl, t-butoxymethyl, 2-methoxyethoxymethyl, 2,2,2-trichloroethoxymethyl, bis(2-chloroethoxy)methyl, 2-(trimethylsilyl)ethoxymethyl, tetrahydropyarnyl, 3-bromotetrahydropyranyl, tetrahydrothiopyranyl, 4-methoxytetrahydropranyl, 4-methoxytetrahydrothiopyranyl, 4-methoxytetrahydrothiopyranyl S,S-dioxido, tetrahydrofuranyl, tetrahydrothiofuranyl, 1-ethoxyethyl, 1-methyl-1-methoxyethyl, 1-(isopropoxy)ethyl, 2,2,2-trichloroethyl, 2-(phenylselenyl)ethyl, butyl, allyl, cinnamyl, p-chlorophenyl, benzyl, p-methoxybenzyl, o-nitrobenzyl, p-nitrobenzyl, p-halobenzyl, p-cyanobenzyl, 3-methyl-2-picolyl N-oxido, diphenylmethyl, 5-dibenzosuberyl, triphenylmethyl, α-naphthyldiphenylmethyl, p-methoxyphenyldiphenylmethyl, p-(p'-bromophenacyloxy)phenyldiphenylmethyl, 9-anthryl, 9-(9-phenyl-10-oxo)anthryl, benzisothiazolyl S,S-dioxido, trimethylsilyl, triethylsilyl, isopropyldimethylsilyl, t-butyldimethylsilyl, (triphenylmethyl)dimethylsilyl, methyldiisopropylsilyl, methyldi-t-butylsilyl, tribenzylsilyl, tri-p-xylylsilyl, triisopropylsilyl and triphenylsilyl.</p>
    <p>In preferred embodiments, a nucleoside 5' phosphate of the present invention has a removable blocking moiety protecting the 3' position which is an ether and which has the following formula: ##STR10## wherein R2 is triphosphate, diphosphate or monophosphate; and wherein R1 is selected from the group consisting of bis(2-chloroethoxy)methyl ether, 4-methoxytetrahydropyranyl ether, tetrahydrofuranyl ether, 1-ethoxyethyl ether, tri(p-methoxyphenyl)methyl ether, di(p-methoxy)phenylmethyl ether, t-butyldimethylsily ether.</p>
    <p>In more preferred embodiments, a nucleoside 5'phosphate of the present invention has a removable blocking moiety protecting the 3' position which is an ether and which has the following formula: ##STR11## wherein R2 is triphosphate, diphosphate or monophosphate; and wherein R1 is CH<sub>3</sub>, CH<sub>3</sub> (CH<sub>2</sub>)<sub>N</sub> where N is an integer from 1-10, methyl, methoxymethyl, methoxyethoxymethyl, trimethlsilyl, and triethylsilyl. In a more preferred embodiment, the nucleoside 5'-phosphate of the present invention has an R1 group which is CH(OC<sub>2</sub> H<sub>5</sub>)CH<sub>3</sub> and R2 is triphosphate and said nucleoside 5'-phosphate is a deoxynucleoside.</p>
    <p>Additional well known removable blocking moieties useful for protecting for hydroxyls include carbonitriles, phosphates, carbonates, carbamates, borates, nitrates, phosphoramidates, and phenylsulfenates. Most of these chemical modifications to the nucleotide can be removed by chemical reactions. Some modifications may also be removed by enzymatic digestion resulting in the regeneration of the 3' hydroxyl. These would include phosphates, glycosides, and certain esters.</p>
    <p>In other embodiments, a nucleoside 5'-phosphate of the present invention has a removable blocking moiety protecting the 3' position having the following formula: ##STR12## wherein R2 is triphosphate, diphosphate or monophosphate; and wherein R1 is selected from the group consisting of phosphate, phosphoramidate and phosphoramide. In preferred embodiments the nucleoside 5' phosphate of the above formula has an R2 group which is triphosphate and an R1 group which is phosphate.</p>
    <p>More preferred embodiments of the present invention contemplate a nucleoside 5'-phosphate having a removable blocking moiety protecting the 3' position which has the following formula: ##STR13## wherein R2 is triphosphate, diphosphate or monophosphate; and wherein R1 is selected from the group consisting of phosphate, toluic acid ester or ethoxyethyl ether.</p>
    <p>The present invention also contemplates a nucleoside 5'-triphosphate having a removable blocking moiety protecting the 3' position which is an ester produced by directly esterifying the nucleoside 5'-triphosphate. In preferred embodiments the nucleoside 5'-triphosphate has an ester is benzoate or acetate as the removable blocking moiety protecting the 3' position. In the more preferred embodiments, the nucleoside 5'-triphosphate produced by direct esterification is a deoxynucleoside.</p>
    <p>The present invention also contemplates a nucleoside 5'-triphosphate having a removable blocking moiety protecting the 3' position which is an ether produced by directly attaching an ether to the nucleoside 5'-triphosphate.</p>
    <p>Particularly preferred are deoxynucleoside 5'triphosphates produced by directly attaching an ether to the 3' position.</p>
    <p>Attachment of the nucleotide having a removable blocking moiety protecting the 3'- position to the free and unmodified 3'-hydroxyl of the initiating substrate is then accomplished by reacting  incubating! the aforementioned nucleotide and the substrate with an enzyme capable of forming a phosphodiester bond between the two. Specifically, this bond would link the 5'-phosphate of the mononucleotide with the 3'-hydroxyl of the initiating substrate. This reaction can be performed either free in solution or, in one embodiment of the invention, the initiating substrate is immobilized on a solid support.</p>
    <p>Particularly preferred are removable blocking moieties and deblocking reaction conditions that allow the blocking moiety to be removed in under 10 minutes to produce a hydroxyl group at the 3' position of the 3'-terminal nucleoside. Other preferred removable blocking moieties and deblocking conditions allow the blocking moiety to be removed in less than 8, 7, 6, 5, 4, 3, 2, or 1 minutes,</p>
    <p>4. Reactions</p>
    <p>In preferred embodiments, the preferred enzyme is TdTase, and specific examples of uses of this enzyme are set forth below. However, the present invention should not be considered limited to the TdTase catalyzed synthesis of DNA and use of other enzymes capable of catalyzing the formation of a 5' to 3' phosphodiester linkage between the 3' hydroxyl group of the substrate and the 5' phosphate of the nucleoside having the removable blocking moiety is contemplated by the present invention. One skilled in the art will understand that enzyme reaction conditions are selected to allow the desired catalysis to occur and may be performed under appropriate conditions, and these conditions are well known in the art.</p>
    <p>The reacting is performed typically between 25° C. and 42° C. for an appropriate period of time, typically between about one minute and about 30 minutes. Very short reaction times may be particularly useful if the removable blocking moiety is unstable.</p>
    <p>For TdTase catalyzed reactions, the enzymatic conditions, which may serve as the solution in which the substrate is reacted, contains from about 0.20 and about 200 μM of the nucleotide having the removable blocking moiety protecting the 3'-hydroxyl, and from about 0.20 to 200 μM of free and unmodified 3'-hydroxyls derived from the initiating substrate. One particularly preferred buffer contains from about 10 to about 500 mM potassium cacodylate buffer (pH between 6.5 and 7.5), and from about 0.01 to about 10 mM of a divalent cation (e.g. CoCl<sub>2</sub> or MnCl<sub>2</sub>). Other buffer compositions and components may be suitable for particular desired embodiment of the present invention.</p>
    <p>For example, enzymatic conditions for deoxynucleotidyl transferase include a pH of 6.8 maintained by a potassium cacodylate buffer, 8 mmol/l of MgCl<sub>2</sub>, 1 mmol.l of β mercaptoethanol, 0.33 mmol/l of ZnSO<sub>4</sub>. One skilled in the art will understand that these enzymatic conditions may vary while still allowing the enzyme to catalyze the desired reaction.</p>
    <p>The enzyme capable of catalyzing the formation of 5' to 3' phosphodiester linkages between the 3' hydroxyl group of the initiating substrate and the 5' phosphate of the nucleoside being added is present in a catalytic amount. A catalytic amount of enzyme is typically sufficient to catalyze the formation of phosphodiester bond between greater than 99% of the free 3' hydroxyls of the initiating substrate and the 5' phosphate of the nucleoside within 1 hour. Preferably, the catalytic amount of enzyme and the enzymatic conditions are such that greater than 99% of the free 3' hydroxyls of the initiating substrate are reacted within 10 minutes. In other preferred embodiments, the catalytic amount of enzyme and the enzymatic conditions are such that greater than 90% of the free 3' hydroxyls of the initiating substrate are reacted in less than 5 minutes, for example 4, 3, 2 or 1 minutes. In other preferred embodiments, the catalytic amount of enzyme and enzymatic conditions are such that greater than 99% of the free 3' hydroxyls of the initiating substrate are reacted within 2 minutes.</p>
    <p>The TdTase enzyme is present in the buffer at a level between about 1 and 200 units per μL. One unit of TdTase catalyzes the transfer of 1 nmol of dATP to p(dT)<sub>6-12</sub> in 60 minutes at 37° C. Commercially available forms of TdTase include calf thymus TdTase, available from a variety of suppliers (e.g. Sigma Chemical Co., St. Louis Mo., Promega Corp, Madison, Wis., Gibco-BRL, Gaithersburg, Md.). Calf thymus TdTase may also be prepared by the procedures described by Modak, Biochemistry, 17, 3116-20 (1978), and by Bollum, Fed. Proc. Soc. Exp. Biol. Med. 17, 193 (1958).</p>
    <p>While the substrate containing a free and unmodified 3'-hydroxyl group and the mononucleotide having the removable blocking moiety protecting the 3 '-hydroxyl group can be reacted in the presence of the TdTase in the buffer solution, the substrate is preferably immobilized on a solid support, and more preferably in a synthesis column to which the buffer solution containing the reaction components is delivered.</p>
    <p>After the appropriate incubation time, the enzyme, unreacted mononucleotide, buffer and divalent cation are separated from the initiating substrate. If the reaction was performed using a free and soluble substrate, it can be separated by conventional size exclusion chromatography or similar types of separation techniques including but not limited to ion exchange chromatography and affinity chromatography. For initiating substrates immobilized on solid supports, separation is achieved by washing the support with water or a suitable buffer.</p>
    <p>One advantage to the present invention is that the level of unreacted hydroxyl groups on the initiating substrate after the aforementioned enzyme reaction can be exceptionally low, less than 0.1%. This minimizes the necessity for capping unreacted hydroxyl groups. In some embodiments of the present invention it may be desirable to cap the unreacted substrates before the next step in the synthesis cycle. The appropriate chemistry for accomplishing this can be derived from any of the protection strategies described previously but must be permanently affixed during all the subsequent cycles. An example of capping is acetylation by reaction of free 3'-hydroxyls with acetic anhydride and pyridine which would be applicable when acetylation (or other esterifications) are not used as the protecting group on the mononucleotide. Alternatively capping can be accomplished by reaction with t-butyldimethylchlorosilane in acetonitrile and pyridine to form a silyl ether which would be applicable when similar ethers are not used to protect the mononucleotide. In the preferred embodiment, these reactions are intended primarily for modifying immobilized initiating substrates in order to rapidly and efficiently provide appropriate capping conditions.</p>
    <p>After the appropriate incubation time, capping reagents are separated from the initiating substrate. If the reaction was performed using a soluble substrate, it can be separated by conventional size exclusion chromatography or similar types of separation techniques including but not limited to ion exchange and affinity chromatography. For initiating substrates immobilized on solid supports, separation is achieved by washing the support with water or a suitable buffer.</p>
    <p>The removable blocking moieties protecting the 3' position on the initiating substrate after the reaction may be removed or deblocked (deprotected) to regenerate a free and unmodified 3'-hydroxyl available for addition of the next nucleotide. One skilled in the art will understand that this may be accomplished by either chemical or enzymatic methods. For example, ester protecting groups may be removed using an esterase when R<sub>1</sub> of the ester protecting group discussed above is a suitable substrate for the esterase. Otherwise, the ester linkage may be cleaved by base hydrolysis, which is accomplished by contacting the protected 3'-hydroxyl group with a suitable concentration of base for a sufficient period of time. Cleavage of ester protecting groups has been well studied and appropriate reaction conditions can be readily identified that will cleave the ester but will not cleave the linkage used for capping (e.g. an ether).</p>
    <p>The present invention incorporates the unexpected discovery that certain removable blocking moieties, the aromatic 3'-O esters of deoxynucleotide triphosphates, are unstable in commonly used buffers containing divalent cations. The instability is attributable to the presence of both the buffer and the divalent cation, and does not result from the presence of the buffer alone or the cation alone. Buffers destabilizing the ester protecting groups may contain dimethylarsinic acid (cacodylic acid), tris(hydroxymethyl) aminomethane, sodium acetate and sodium phosphate. Divalent cations destabilizing to ester blocking groups include cobalt, manganese and magnesium ions. The toluic acid ester of a deoxynucleotide triphosphate is unstable in a mixture of 1 mM CoCl<sub>2</sub>, 100 μM potassium cacodylate, pH 6.8.</p>
    <p>Conditions for the removal of removable blocking moieties such as ethers, carbonates, nitrates, and other protecting groups are well studied and many are compatible with the integrity of a polynucleotide chain. Removal of blocking moieties such as phosphate protecting groups, the hydroxyl is regenerated by enzymatic digestion with a phosphatase. For removal of blocking moieties when the protecting group is a sugar moiety, regeneration of the hydroxyl can be accomplished by enzymatic hydrolysis using a glycosidase.</p>
    <p>If the removal or deblocking reaction is performed in solution, the deprotection reagents are simply added to the solution. If the reaction is performed with the initiating substrate immobilized on a solid support, then the hydroxyl group regeneration step is performed by washing the solid support with the deprotection reagents. When synthesis columns are utilized to contain the solid support, the hydroxyl group regeneration step is performed by washing the column with the appropriate agents.</p>
    <p>After the appropriate period for removal, the initiating substrate (including both those that received an additional nucleotide and those that are capped) is again separated from the other reaction components. If the reaction was performed using a soluble substrate, it can be separated by conventional size exclusion chromatography or similar types of separation techniques including but not limited to ion exchange and affinity chromatography. For initiating substrates immobilized on solid supports, separation is achieved by washing the support with water or a suitable buffer.</p>
    <p>As will be appreciated, the above described steps of enzyme catalyzed phosphodiester bond formation between a nucleotide having a removable blocking moiety at its 3' position and an initiating substrate, separation of the initiating substrate from reaction components, capping of unreacted initiating substrate, again separating the initiating substrate from reaction components, removing the removable blocking moiety to regenerate the 3'-hydroxyl group, and again separating the initiating substrate from reaction components are repeated as necessary until the desired object polynucleotide chain is completely synthesized.</p>
    <p>Cleavage of a newly synthesized polynucleotide strand from the solid support and/or from the initiating substrate can be accomplished by either chemical or enzymatic reactions. In the case of a chemical reaction, if the initiating substrate terminal nucleoside (containing the free and unmodified 3'-hydroxyl group) is a deoxyguanosine methylated at the 7 position of the base:</p>
    <p>Support-dCCCCCCCCCCC-Me<sup>7</sup> -G-object polynucleotide (SEQ. ID No. 1) reaction with 1M piperidine in water at 90° C. will cleave the chain at this position yielding only the desired polynucleotide in solution. This method can yield a polynucleotide chain containing only the predetermined sequence and can be performed either on immobilized chains (to effect cleavage) or on solution synthesized chains to remove the initiating substrate. Alternatively, the dG<sup>7me</sup> can be positioned at any location within the initiating substrate or the object polynucleotide where cleavage is desired. Other examples of modified base-specific cleavage of polynucleotide chains have been extensively described in the literature (See, Ambrose and Pless, Meth. Enzymol.,I Vol 152: 522-538.)</p>
    <p>Enzymatic removal of the polynucleotide chain may be accomplished by reaction with a specific restriction endonuclease. For example, if the initiating substrate oligonucleotide has the following structure:</p>
    <p>Support-dCCCCCCCCCCCCCCCTGCA-3'-OH (SEQ ID No. 2) and the object polynucleotide begins with a G, the resulting newly synthesized chain can be cleaved from the support by reaction with Pst 1 restriction enzyme. This method assumes there are no additional Pst 1 restriction sites in the newly synthesized chain and that one has annealed an appropriate oligonucleotide to the Pst 1 site to render it in a double stranded form for recognition by the enzyme (e.g. an annealing oligonucleotide with the following structure: 3'-dGGGGGGGGGGGGGGGACGTC-5' (SEQ ID No. 3) for the example above). Depending on the desired first nucleotide of the object polynucleotide, as well as the ultimate sequence of the polynucleotide, one can choose from a wide variety of restriction enzymes to accomplish the cleavage of only the desired sequence. This method can yield a polynucleotide chain containing only the predetermined sequence and can be performed either on immobilized chains (to effect cleavage) or on solution synthesized chains to remove the initiating substrate. Alternatively, appropriate restriction endonuclease recognition sequences can be positioned at any location within the initiating substrate or the object polynucleotide where cleavage is desired.</p>
    <p>The combined initiating substrate and object polynucleotide can be cleaved from the solid support by chemical methods. How the cleavage is performed will depend upon the nature of the initiating substrate and how it was attached to the solid support. Covalent labile bonds, such as for example a trityl group, can be cleaved by washing the support with an appropriate protic acid. Numerous other cleavage strategies have been described. In the case of a non-covalent attachment, as for example avidin-biotin binding, release of the combined substrate and object polynucleotide will occur upon incubation with 8M guanidine-HCl, pH 1.5.</p>
    <p>If the entire synthesis was performed using a soluble initiating substrate, the initiating substrate containing the object polynucleotide can be separated from the various capped oligo- and polynucleotides by conventional chromatographic techniques, such as polyacrylamide gel electrophoresis. Similarly, if the initiating substrate is cleaved from the object polynucleotide by chemical or enzymatic means (e.g. by reaction with piperidine or by restriction endonucleases digestion as described above) conventional chromatography can be used to purify the object polynucleotide.</p>
    <p>If the synthesis was performed using an initiating substrate immobilized to a solid support, cleavage from the solid support can be accomplished by either chemical or enzymatic means to retrieve either the combined initiating substrate and object polynucleotide or the object polynucleotide alone. In each instance, the object polynucleotide will be contaminated with capped oligo- and polynucleotides which can be separated from the object polynucleotide by polyacrylamide gel electrophoresis.</p>
    <p>An alternative strategy for the synthesis and recovery of the object polynucleotide involves immobilization of the nucleotide. In this instance, the nucleotide is protected at the 3'-hydroxyl by a linker which is attached to a solid support. The linker attachment to the nucleotide can be by an ester or by any of the aforementioned protecting group strategies. Solid supports containing various functional groups (e.g. amines, amides, biotin, avidin, and the like) are generally available and can be adapted to the particular requirements of the nucleotide linker. For example, a nucleotide linker containing a biotin molecule can be bound to agarose using an avidin functional group attached to the agarose.</p>
    <p>Using an immobilized nucleotide, the TdTase reaction would join a free initiating substrate, in solution, to the immobilized nucleotide, thereby immobilizing only those initiating substrates which have participated in the enzyme reaction. Initiating substrates which had not participated in the TdTase reaction would be easily removed by rinsing the solid support with an appropriate buffer. Regeneration of the 3' hydroxyl on the initiating substrate is accomplished by the same techniques as described previously.</p>
    <p>Subsequent to the regeneration and cleavage step, the initiating substrate is rinsed away from the solid support and separated from the regeneration/cleavage solution containing free nucleotides by conventional techniques such as size exclusion chromatography, ion exchange or affinity chromatography. The next immobilized nucleotide, contained on a new population of solid support particles, is then mixed with the initiating substrate and the appropriate buffers in order to repeat the TdTase coupling reaction.</p>
    <p>By immobilizing the nucleotide rather than the initiating substrate, a capping reaction is obviated since the object polynucleotide is separated from unreacted initiating substrate at every cycle. Similarly, if the cleavage reaction fails to release all of the object polynucleotide chains, those polynucleotides which continue to be attached to the solid support are removed prior to the subsequent TdTase reaction.</p>
    <p>It is envisioned that various newly synthesized polynucleotide chains will subsequently be joined together by a polymerase/ligase type of reaction in order to form longer polynucleotide sequences that are in a double stranded form. For example, newly synthesized polynucleotides A and B may have the structures depicted below:</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="equation">A: 3'-p(dN)-dCCCCCCCCC-5' (SEQ ID No. 4)</pre>
    
    <p>
      </p> <pre xml:space="preserve" listing-type="equation">B: 3'-p(dN)-dGGGGGGGGG-5' (SEQ ID No. 5)</pre>
    
    <p>where p(dN) is the predetermined object polynucleotide sequence unique to either the A or B polynucleotide. In the presence of the Klenow fragment of DNA polymerase I, and T4 DNA ligase, as well as the appropriate buffers and nucleotides, a double stranded polynucleotide will be formed in which the two object polynucleotides have been "stitched" together to form the longer double stranded polynucleotide C:</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="equation">C: 5'-p(dN)-dCCCCCCCCCp(dN)-3' (SEQ ID No. 4)</pre>
    
    <p>
      </p> <pre xml:space="preserve" listing-type="equation">3'-p(dN)-dGGGGGGGGGp(dN)-5' (SEQ ID No. 5)</pre>
    
    <p>This reaction can be performed when one of the polynucleotides is still attached to a solid support or when both polynucleotides have been released into solution by the techniques described previously.</p>
    <heading>B. Polynucleotides</heading> <p>The present invention contemplates oligonucleotides and polynucleotides produced using the methods of this invention. These polynucleotides preferably have a predetermined nucleotide sequence that was produced by selecting the order in which the individual nucleotides were added to the initiating substrate so when synthesis is completed a polynucleotide having a preselected sequence is produced. Alternatively, random combination of polynucleotides could also be produced by introducing all four blocked nucleotides during an individual coupling reaction or during numerous individual coupling reactions.</p>
    <p>In preferred embodiments, a polynucleotide produced according to the methods of this invention is greater than five nucleotides in length. Polynucleotides produced according to the methods of the present invention may contain large numbers of nucleotides, for example 10, 20, 30, 40, 50, 60, 70, 80, 90, 100, 110, 120, 130, 140, 150, 160, 170, 180, 190, 200 and greater than 200 nucleotides. The length of a polynucleotide produced using the methods of the present invention may be of a length that is intermediate between the aforementioned nucleotide lengths, such as 5, 15, 16, 25, 26 or any other numeral intermediate between the specific lengths. The length of the polynucleotide produced according to the present invention is limited only by the efficiency of the processes of the present invention.</p>
    <p>Polynucleotides produced by the methods of the present invention can contain nucleotide sequences that have a variety of biologic and molecular biologic uses. One skilled in the art will understand the uses for long polynucleotides having a predetermined sequence. For example, many manipulations commonly performed in modern molecular biology could be greatly simplified through the availability of inexpensive, long polynucleotides having a predetermined sequence.</p>
    <p>Examples of molecular biology procedures and manipulations that would be simplified using the polynucleotides produced according to the methods of the present invention include cloning and expression of various nucleic acids both in vitro and in vivo. For examples of techniques and manipulations that are simplified using polynucleotides produced using the methods of the present invention see, Methods in Enzymology, Vol. 152 edited by Berger and Kimmel; Maniatis et al., Molecular Cloning a Laboratory Manual, Cold Spring Harbor Press, 1990; Current Protocols in Molecular Biology, edited by Ausubel et al., John Haley and Sons, New York, 1987.</p>
    <p>For example, polynucleotides of the present invention could be used to introduce restriction sites into a nucleic acid, to introduce various nucleotide sequences having biological activity such as promoters, and to adjust reading frames. The number of possible applications using oligonucleotide and polynucleotide produced according to the present invention is large as one skilled in the art will understand. Oligonucleotides and polynucleotides produced according to the present invention are especially useful when the application requires long oligonucleotides and polynucleotides.</p>
    <heading>C. Compositions of Matter</heading> <p>The present invention also contemplates compositions of matter comprising a catalytic amount of a template independent enzyme and a nucleoside 5'-triphosphate having a removable blocking moiety protecting the 3' position of the nucleoside 5' triphosphate and other preferred compositions further comprise an initiating substrate of the present invention.</p>
    <p>The present invention contemplates compositions having an amount of template independent enzyme capable of catalyzing the formation of a 5' to 3' phosphodiester linkage between 99 percent of the unprotected 3' hydroxyl groups present on an initiating substrate of the present invention and a nucleoside 5'-triphosphate having a removable blocking group protecting its 3' position within 10 minutes. Other compositions are contemplated that contain an amount of enzyme capable of performing the same reaction to the same extent within 2 minutes.</p>
    <p>The compositions contemplated by the present invention includes compositions in which the template independent polynucleotide enzyme present is a template independent polynucleotide polymerase. Examples of preferred template independent polynucleotide polymerases include TdTase and enzymes with similar activities.</p>
    <p>The composition of the present invention includes a nucleoside having a removable blocking moiety protecting the 3' position of the nucleoside. Particularly preferred are nucleoside 5'-triphosphates having a removable blocking moiety protecting the 3' position of the nucleoside. The various useful removable blocking moieties are described herein.</p>
    <p>In preferred compositions, the nucleoside having the removable blocking moiety protecting the 3' position is present at a concentration of 1 nanomolar to 100 mmolar. In other preferred compositions, the nucleoside is present at a concentration of 1 micromolar to 1 millimolar. In other preferred embodiments, the nucleoside 5'-triphosphate having the removable blocking moiety protecting its 3' position is present at a concentration of 10 times the Km of the enzyme present in the composition.</p>
    <heading>D. Automated Processes and Apparatus</heading> <p>The present invention contemplates the incorporation of the method described herein in an automated process in an apparatus and in devices. For example, the various buffer and reagent solutions of the inventive process can be provided to synthesis columns containing initiating substrates affixed to solid support matrices by the use of flexible tubing attached to peristaltic pumps or similar devices controlled by a microprocessor programmed to meter the exact quantities of the materials in the correct sequence.</p>
    <p>Regardless of the equipment employed, it can be appreciated that the method of the present invention can create single phosphodiester bonds between desired nucleosides with very high efficiency and can potentially be used to produce long chain polynucleotides in high yields.</p>
    <p>One example of such an automated process is depicted by a porous frit 13 in a glass or plastic vessel 15 shown in FIG. 3. The insoluble matrix 11 consists of a solid support such as cellulose, SEPHAROSE™ or CPG to which a nucleotide, nucleoside or polynucleotide is covalently linked at the 5'-position of the terminal nucleotide or to which an oligo- or poly- nucleotide or nucleoside having a terminal nucleoside with a free 3'-hydroxyl group is covalently attached via the 5'-hydroxyl group. The matrix II may itself be a covalent component of the porous frit 13 or it may be a separate entity.</p>
    <p>The various solutions involved in the synthesis cycle are stored in stock containers 21, 23, 25, 27, 28 and 29. Solutions are introduced into the vessel 15 through tubes 41, 43, 47, 48, 49 and 50 attached to pumps 31, 33, 35, 37, 38, 39 and 40. The composition of the stock solutions would depend on the stability of the various components of the mixture. A simplified automated process would combine many of the various reagents as follows:</p>
    <p>The stock containers 21, 23, 25 and 27 contain buffer solutions 51, 53, 55 and 57, respectively, having a concentration between about 10 and about 500 mM of sodium cacodylate (pH 7.0 at 25° C.), between about 0.1 and about 1.0 mM of dithiothreitol. Each buffer solution also contains between about 0.10 and about 200 units per μL of an enzyme (e.g. TdTase) suitable for phosphodiester bond formation. Buffer solution 51 in stock container 21 also contains between about 0.20 and about 200 μM of deoxyadenosine 5'-triphosphate having a blocked 3'-hydroxyl group. Buffer solutions 53, 55 and 57 in stock containers 23, 25 and 27, respectively, contain equivalent concentrations of deoxycytosine 5'-triphosphate, deoxyguanosine 5'-triphosphate and thymidine 5'-triphosphate respectively, each of which also has blocked 3'-hydroxyl groups. Buffer solution 58 in stock container 28 contains an appropriate reagent for deblocking the blocked 3'-hydroxyl groups of the four nucleosides as described previously. Stock solution 59 in stock container 29 contains a suitable neutralization buffer at pH 7.0, such as 0.1M sodium cacodylate. Stock solution 30 in container 60 contains a suitable enzymatic solution or chemical reagent for releasing the final product from the solid support as described previously.</p>
    <p>The various stock solutions are drawn into the tubing, which each feed onto the matrix. Recycling of buffer solutions 51, 53, 55 and 57 from the vessel 15 to their respective stock containers 21, 23, 25 and 27 can be accomplished by way of the tubing 61, 63, 65 or 67. Allocation of fluid to the appropriate tubing can be accomplished by a distributor, 71, which directs the fluid from the vessel 15. Distributor devices, such as multiport stopcocks and fraction collectors are familiar to one of ordinary skill in the art. Movement of the liquid through tubing which is downstream from the distributor (e.g. 61, 63, 65, 67, 69, 73) can be accomplished by additional pumping as needed (e.g. pump 83, 85, 87, 89, 91, 93). At least one microprocessor controls the peristaltic pumps and distributor so as to provide for the sequential addition and recycling of the nucleotides to form a nucleotide chain having a predetermined nucleotide sequence. In the preferred processes, the initiating substrate linked to the matrix 11 is first exposed to one of the solutions 51, 53, 55, or 57 for a sufficient time to enable attachment of the nucleotide to the initiating substrate. This solution is then recycled into the appropriate container (21, 23, 25, or 27).</p>
    <p>The amount of TdTase and 5'-nucleoside triphosphate contained in the buffer solutions 51, 53, 55 and 57 is sufficient for the synthesis of a predetermined quantity of an object nucleotide chain. For example, for the automated synthesis of 1 nmol of a nucleotide chain consisting of 1,000 bases (about 330,000 MW and about 330 μg), each buffer solution will contain an excess of each 5'-nucleoside triphosphate (about 500 nmol) and an excess of TdTase (about 100 to about 1,000 units). only a small fraction of the buffer solution containing the TdTase and the 5'-nucleoside triphosphate is used for each cycle of nucleotide addition. Matrix 11 is next exposed to solution 58 for a sufficient period of time to remove blocking groups from the growing oligo- or polynucleotide chain. This solution is not recycled but is distributed to tube 69 by the distributor 71, utilizing pump 91.</p>
    <p>Matrix 11 is then briefly exposed to solution 59 in order to wash out the deblocking reagent. The next enzyme/nucleotide solution, either 51, 53, 55 or 57, is then added to matrix 11 to continue the cycle.</p>
    <p>Finally, after the desire oligo- or polynucleotide is synthesized, cleavage of the object polynucleotide from the solid support occurs by the controlled addition of solution 60 which can be a restriction endonuclease solution or a solution to effect the chemical cleavage from the solid support (e.g., 1M piperidine) as described above. The microprocessor directs the distributor 71 and pump 93 to move the final product through tube 73 to be recovered for final workup.</p>
    <p>As an example of the control of the various reactions by the microprocessor, the synthesis of the oligonucleotide ACGT onto an initiating substrate would involve the sequence of commands shown below. The duration of and between each command is sufficient to allow any particular reaction or fluid movement to proceed adequately.</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">______________________________________Microprocessor Command            Intended Result______________________________________1.  Pump 31 on.      Solution 51 added to vessel 152.  Pump 31 off.     Nucleotide addition reaction proceeds3.  Distributor 71 on,                Recycle reaction fluid    pump 83 on.      fluid via tube 614.  Distributor 71 off,                Solution 58 added to vessel 15;    pump 83 off.     initiate deblocking reaction5.  Pump 38 off.     Deblocking reaction proceeds6.  Distributor 71 on,                Discard deblocking fluid via tube 69    pump 91 on.7.  Pump 39 on.      Neutralize/wash reaction chamber8.  Distributor 71 off,                Solution 53 added to vessel 15    pump 69 off, pump    33 on.______________________________________</pre>
    
    <p>This cycle is repeated for the other nucleotides until the desired sequence is synthesized. When collection of the final product is desired, the microprocessor gives the following commands after step 7 above.</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">______________________________________1.  Pump 40 on.  Solution 60 added to vessel 152.  Pump 40 off. Cleavage reaction of the initiating            substrate proceeds3.  Distributor 71 on,            Collection of synthesized DNA via tube 73    pump 93 on______________________________________</pre>
    
    <p>The alternative strategy envisions the use of immobilized nucleotide triphosphates in order to separate the object nucleotide from non-reacting substrate polynucleotides at every cycle. The automated process using immobilized nucleotide is considerably different from the process involving an immobilized substrate polynucleotide. After the coupling reaction of the triphosphate and the polynucleotide, the eluate contains unreacted polynucleotides, reaction buffer, and TdTase enzyme. The object polynucleotide is attached to the solid support. In order to recycle the enzyme back to its reservoir, the contaminating polynucleotide is first removed by passing the solution through a column containing hydroxyl apatite, for example, or a similar polynucleotide adsorption medium through which the enzyme will pass. This column will have sufficient capacity to adsorb all of the anticipated contaminating polynucleotides produced by every cycle.</p>
    <p>After the deblocking step, the object polynucleotide is now contained in a solution of nucleotide triphosphates (with 3'-hydroxyls), and deblocking buffer (e.g., NaoH or phosphatase). These two contaminants can be removed by size exclusion chromatography (e.g., SEPHAROSE™ CL-6B) or by any of a number of commonly used techniques for separating small molecules from oligo- and polynucleotides. An example of this is adsorption of the object polynucleotide by annealing to oligo dA-cellulose column (3'-5') which would simply require the initiating substrate to contain oligo dT. A-T annealing is the preferred embodiment in the automated process since elution of the object polynucleotide can be accomplished by incubation with H<sub>2</sub> O. The annealing of the object nucleotide simply requires the neutralization of the cleavage reaction by addition of a sufficient quantity of HCl or by the inclusion of an appropriate amount of NaCl (.sup.˜ 0.1-0.5M) in the deblocking buffers.</p>
    <p>An automated process incorporating the immobilized nucleotide triphosphate alternative method of the present invention is depicted in FIG. 4. The process utilizes a nucleotide triphosphate immobilized to a solid support by, but not limited to, techniques describe previously, and compromising stock solutions 151, 153, 155, 157 in stock containers 121, 123, 125, 127. The stock solutions contain a tethered nucleotide, appropriate buffers and sufficient enzyme to effect the synthesis of the desired amount of predetermined sequence. The immobilization material has fluid dynamic properties allowing it to be moved through the various tubes as required. Substances which have these characteristics (e.g. gels and viscous suspensions) are familiar to one of ordinary skill in the art. The reaction vessel, 115, contains a reaction chamber, 111, and a stopcock, 113. Stopcock 113 has three positions A, B, C. Position A aligns a hole of sufficient diameter with the tubing so as to allow the various components of the synthesis to pass unimpeded. Position B aligns a porous frit to which is covalently attached oligonucleotides of deoxyadenosine (dA) approximately 20 bases in length. The quantity of oligo dA is sufficient to anneal the entire quantity of oligo dT, attached to the initiating substrate as described above. In position B, only solutes can pass through and no immobilization material (e.g. those contained in solutions 151, 153, 155, 157). Position C closes all flow. Reaction chamber 111 contains the initiating substrate in water, solution 161. As mentioned above the initiating substrate contains oligo dT which is ≧20 nucleotides in length. Stock containers 121, 123, 125, 127, 128 and 129 are connected to the reaction vessel 115 by way of peristaltic tubing or some similar material to effect transport of the reagents contained in the stock containers. Additionally, vessel 115 is connected to tubing, 181, which contains a distributor, 171 which serves to divert the flow of solutes either to tubing 183 or tubing 185 or recycled back to stock containers after passing through adsorption media (e.g. hydroxylapatite) contained in 130, 132, 134 or 136 via tubes 187, 189, 191 or 193. Tubing 183 feeds back to vessel 115; tubing 185 feeds into a discard container. Solute movement through the tubing is facilitated by pumps, 131, 133, 135, 137, 138, 139, 163 (e.g. peristaltic pumps) or similar devices which will force fluids, gels or viscous suspensions through tubing to desired destinations.</p>
    <p>The automated process for synthesis involves the following flow of solutes and stopcock positions controlled by at least one microprocessor. The microprocessor controls pumps, the distributors and the stopcock positions:</p>
    <p>1) Stopcock 113, position C (blocked); stopcock 171 in discard position (tube 185); tethered nucleotide, buffers (solution 151, 153, 155, or 157) are combined with substrate oligonucleotide or polynucleotide (solution 161) to yield a tethered oligonucleotide or polynucleotide.</p>
    <p>2) Stopcock 113, position B (oligo-dA frit); distributor 171 in recycle position (tube 187, 189, 191 or 193); unreacted polynucleotide is adsorbed in containers 130, 132, 134 or 136; enzyme, buffers are recycled.</p>
    <p>3) Stopcock 113, position B (oligo-dA frit); stopcock 171 in discard position (tube 185); cleavage buffer (solution 158) added to immobilized polynucleotide yielding a free polynucleotide annealed to the oligo-dA frit; released mononucleotides discarded.</p>
    <p>4) Stopcock 113, position B (oligo-dA frit); stopcock 171 in recycle position (tube 183); water (solution 159) is passed through the chamber and frit to release the annealed polynucleotide from the frit and return the polynucleotide to the reaction chamber. The free polynucleotide resides in tubing 183 during Step 5.</p>
    <p>5) Stopcock 113, position A (completely open); stopcock 171 in discard position (tube 185); immobilized substrate discarded prior to entry of polynucleotide back into reaction chamber.</p>
    <p>6) The final product is recovered via tube 185 with stopcock 113 in position A.</p>
    <p>It will be appreciated that for these separation techniques to be effective, the starting oligonucleotide or polynucleotide substrate should consist of at least approximately 20 nucleotides. The composition of the starting oligonucleotide or polynucleotide can be anything that will enable the subsequent purification steps as well as the ultimate cleavage of the object oligonucleotide or polynucleotide from the starting oligo- or polynucleotide. An example of a nucleotide modification that would enable final separation of starting oligonucleotide or polynucleotide from the object polynucleotide is biotinylation of the primary amines of dA, dC, or dG. Additionally, a starting oligonucleotide substrate containing 7-methyl guanosine at the 3' end will provide a cleavage site, as described previously, for ultimate recovery of the object polynucleotide.</p>
    <p>Thus, it can be appreciated that, regardless of the equipment employed, the method of the present invention efficiently produces oligonucleotide or polynucleotides in high yield, with a significant reduction in the number of unreacted sequences per cycle. This greatly simplifies the ultimate isolation of the object nucleotide chain for further experimentation. Once isolated, the nucleotide chain may be "stitched" together with other polynucleotides and formed into double stranded DNA as described above or may be amplified by conventional means such as by polymerase chain reactions for use in recombinant DNA end use applications.</p>
    <heading>E. Kits</heading> <p>The present invention also contemplates a kit for carrying out the present inventive procedure. Typically, a kit would contain all the solutions and substances needed for performing the instant synthesis procedure together with instructions for carrying out the procedure. A typical kit for carrying out the claimed process would include an initiating substrate of the present invention, various nucleoside 5' triphosphates of the present invention having a removable blocking moiety protecting the 3' position, an enzyme of the present invention capable of catalyzing the formation of a 5' to 3' phosphodiester linkage between the unprotected 3' hydroxyl group of the initiating substrate and the 5' phosphate of the blocked 5'-triphosphate. Additional components and solutions optionally included in the kit are various required reaction solutions and reaction buffers, reaction vessels in which to perform the assay, deblocking chemicals, solutions or enzymes of the present invention.</p>
    <p>A kit for carrying out the instant synthesis may also contain initiating substrates that are attached to a solid support. The kit may contain a variety of initiating substrates attached to solid supports, so that the first nucleoside of a desired oligonucleotide can be selected by selecting the appropriate initiating substrate.</p>
    <p>In other kits for carrying out the present process, initiating substrates having oligonucleotides of a preselected nucleotide sequence are provided to allow oligonucleotides and polynucleotides having this preselected nucleotide sequence incorporated into its 5' to be produced. Kits with this type of initiating substrate can provide easy synthesis of oligonucleotides having, for example, a restriction endonuclease cleavage site present in its nucleotide sequence.</p>
    <p>Other kits contemplated by the present invention include initiating substrates having various derivatized nucleotides, nucleoside analogs, or non-nucleoside molecules that allow oligonucleotides produced using those initiating substrates to have useful properties such as easily coupling to other molecules, unique biologic activity or other unique features. Other kits would have an initiating substrate of the present invention such as double-stranded oligonucleotides.</p>
    <p>The present invention also contemplates kits for producing nucleoside 5'-phosphate and nucleoside analogs having a removable blocking moiety protecting its 3' position. These kits would allow a user to produce nucleoside 5' triphosphates and equivalents that are useful in the practicing of the present invention.</p>
    <p>The present invention also contemplates kits that contain additional components for carrying out other molecular biologic procedures in conjunction with the methods of the present invention. For example, components of the present invention may be present in a kit that contains vectors and concomitant cell lines for expression of a protein or enzymes for desired modifications of amplification of the nascent or fully synthesized object polynucleotide.</p>
    <p>The following examples further illustrate the present invention, and are not to be construed as limiting the scope thereof. Unless otherwise indicated, materials were obtained from Promega, Fisher, Aldrich, Sigma, Pharmacia, Gibco-BRL, Bio-Rad and New England Biolabs. All parts and percentages are by weight unless expressly indicated to be otherwise, and all temperatures are in degrees Celsius.</p>
    <heading>EXAMPLES</heading> <heading>Example 1. Synthesis of Protected Nucleotides</heading> <p>A. Synthesis of Protected Nucleotides by Reaction of the 3' Hydroxyl with Carboxylic Acids</p>
    <p>i. Toluic acid. One hundred μL of 1M toluic acid (either the para or ortho isomer) in anhydrous N,N-dimethylformamide (DMF) was mixed, in a nitrogen atmosphere, with 100 μL of 1M carbonyldiimidazole, also in anhydrous DMF. Formation of the imidazolide was allowed to proceed at room temperature for 30 seconds. To this mixture was added 100 μL of a 50 mM solution of deoxynucleoside 5'-triphosphate in water. Formation of the toluoyl-dNTP ester proceeded at room temperature for 12 hours.</p>
    <p>The triphosphates (including both 3'-hydroxy unreacted triphosphates and the 3'-toluoyl triphosphates) were separated from the other reaction components by precipitation in the presence of 9 volumes of acetone. The insoluble nucleoside triphosphates were recovered by centrifugation and removal of the soluble components. The nucleosides were then redissolved in 100 μL of water, toluoyl ester was separated from the starting nucleotide by chromatography on Whatman 3 MM cellulose paper which had been prewashed first in isopropanol, butanol, and water in the proportion of 2:2:3 by volume, and then in water alone, prior to drying. The solvent to achieve separation by ascending chromatography contained isopropanol, butanol, and water also in the proportion of 2:2:3 by volume. Detection of the various separated components was by ultraviolet light absorption at 254 nm. The dNTP-3'-O-toluate was cut from the paper and eluted into water. After concentration to dryness in vacuo, the nucleotide ester was redissolved in water to a final concentration of .sup.˜ 0.1-1 mM. This material was then subjected to mass spectroscopic analysis to confirm the structure. The predicted mass numbers for the toluoyl esters of dATP, dCTP, dGTP, and TTP are 608, 584, 624, and 599 respectively. In each case these mass numbers were observed. These mass numbers were not observed in the spectra obtained from the unprotected deoxynucleoside triphosphates.</p>
    <p>In related experiments, a variety of esters have been formed from carboxylic acids to yield aromatic or aliphatic protecting groups at the 3' position.</p>
    <p>ii. Benzoic acid and dimethylbenzoic acid. Benzoic acid as well as the 2,6- and 3,5-dimethylbenzoic acid isomers were esterified to nucleotide triphosphates by the same methods described above in order to evaluate position effects of methyl groups on the overall kinetics of subsequent enzyme reactions.</p>
    <p>iii. 4-Nitrobenzoic acid. Esterified to the 3'-hydroxyl using the same methods.</p>
    <p>iv. 2 -Napthoic acid. Esterified to the 3'-hydroxyl using the same methods.</p>
    <p>v. Isovaleric acid. Esterified to the 3'-hydroxyl using the same methods.</p>
    <p>Depending on the particular stability requirements, the procedures are readily adaptable to the utilization of virtually any carboxylic acid for esterification and protection of the 3'-hydroxyl of a nucleotide triphosphate.</p>
    <p>B. Synthesis of Protected Nucleotides by Reaction of the 3'-hydroxyl with an Ether</p>
    <p>2.5 mg of deoxynucleoside triphosphate was dissolved in 100 μL anhydrous DMSO containing 5.2 mg para-toluene sulfonic acid. The solution was cooled to 0° C.; 200 μL of ethyl vinyl ether was then added and allowed to react for 3 minutes. 200 μL of 1M Tris-Cl, pH 9.0 was then added with vigorous shaking resulting in the formation of two liquid phases. The ether phase was discarded and to the aqueous phase was added 10 volumes of absolute ethanol to precipitate the nucleotides. After incubation for 10 minutes at -20° C. the nucleotide pellet was obtained by centrifugation, and was redissolved in 0.25M NaCl followed by the addition of 10 volumes of ethanol. The final precipitated nucleotide pellet was dissolved in 100 μL of water and applied to Whatman 3 MM paper for chromatography to separate the nucleotide ether from unreacted nucleotides. Ascending chromatography was performed as described above with a solvent of isopropanol, butanol, and water in the proportion of 2:2:3 by volume. The nucleotide ether, where the protecting group is an ethoxyethyl moiety, migrates with an R<sub>f</sub> (relative to the unreacted nucleotide) of 1.25. This species was cut out of the paper and eluted with water to yield the purified derivative.</p>
    <p>C. Synthesis of Protected Nucleotides by Phosphorylation of the 3'-hydroxyl</p>
    <p>i. Chemical synthesis. 2.0 mg of deoxynucleoside triphosphate was dissolved in 60 μL anhydrous DMSO, 2 μL orthophosphoric acid, and 6 μL triethylamine. To start the reaction, 6 μL of trichloroacetonitrile was added and the mixture was incubated at 37° C. for 30 minutes. The reaction was cooled to room temperature and 5 μL of 5M NaCl was added followed by 1.4 mL of acetone. The precipitation of nucleotide was allowed to proceed at -20° C. for 10 minutes; nucleotide was recovered by centrifugation, redissolved in 100 μL 0.25M NaCl and reprecipitated by the addition of 1.4 mL of absolute ethanol. The final nucleotide was recovered by centrifugation, dissolved in 100 μL of water and applied to Whatman 3 MM paper. Separation of nucleotide tetraphosphate (5'-triphosphate, 3'-monophosphate) was by ascending chromatography in isopropanol, butanol, and water in the proportion of 2:2:3 by volume. The nucleotide tetraphosphate migrates with an R<sub>f</sub> (relative to the unreacted nucleotide) of 0.90.</p>
    <p>Alternatively, the same phosphorylation reaction components (phosphoric acid, teithylamine, and deoxynucleotide) can be dissolved in formamide and the reaction allowed to proceed at 70° C.</p>
    <p>Alternative chromatography solvents include 1-propanol, concentrated ammonia, water (55:20:25), in which case the Rf of the tetraphosphate or 3 mm paper is approximately 0.8 relative to unreacted triphosphate.</p>
    <p>ii. Enzymatic synthesis. Deoxynucleoside 3'-monophosphates (Sigma) were phosphorylated at the 5' position using polynucleotide kinase. The reaction was performed at pH 9.0 to minimize the inherent 3' phosphatase activity of the enzyme, in a solution consisting of 50 mM Tris-Cl (pH 9.0), 10 MM MgCl<sub>2</sub>, 1.5 mM spermine, 5 mM dithiothreitol, 3 mM 3'-dNMP, 30 mM ATP, and 20 units of polynucleotide kinase (Sigma or Pharmacia) in a final volume of 200 μL for 16 hours at room temperature. The phosphorylation was monitored by chromatography (Whatman 3 MM paper) after removal of the ATP by chromatography through Affi-Gel 601 (Bio-Rad).</p>
    <p>The nucleoside 5'-monophosphate 3'-monophosphate was further phosphorylated at the 5' position using nucleoside monophosphate kinase and pyruvate kinase in a solution containing 50 mM Tris-Cl (pH 7.4), 10 mM MgCl<sub>2</sub>, 1.5 mM spermine, 5 mM dithiothreitol, 30 mM ATP, 4 mM phosphoenolpyruvate, 10 mM KCl 150 μg/mL pyruvate kinase (Sigma), and 100 μg/mL nucleoside monophosphate kinase (Boehringer Mannheim) . The reaction proceeded at room temperature for 30 minutes (for dA) to 4 hours (for dT, dC, dG). The deoxynucleotides were again separated from ATP by chromatography on Affi-Gel 601 followed by concentration to dryness in vacuo and dissolution in 200 μL of water. Purification of the tetraphosphate from other nucleotides was by paper chromatography as described above.</p>
    <p>D. Synthesis of a Benzoylated Nucleotide Tethered to Agarose Beads</p>
    <p>One hundred μL of 1M p-aminobenzoic acid in 10 mM sodium hydroxide, 90% DMF, pH 10, was mixed with 100 μL of 1M N-succinimidyl 3-(2-pyridylthio)propionate also in basic DMF. Coupling of the succinimidyl to the amine was allowed to proceed at room temperature for four hours. The reaction was monitored and the coupled product purified by thin layer chromatography on silica gel using a mixture of chloroform and methanol as the solvent. Silica gel containing the coupled product was extracted with DMF, filtered, and added to an equal volume (.sup.˜ 100 μL) of 1M carbonyl diimidazole in anhydrous DMF followed immediately by the addition of 100 μL of 50 mM deoxynucleoside triphosphate. The product, a dNTP coupled by an ester linkage to a tether containing an amide and a disulfide bond, was treated with 2-mercaptoethanol in a nitrogen atmosphere to expose the sulfhydryl, and subsequently purified by purification from 10 volumes of absolute ethanol. The purified product, under nitrogen atmosphere, was then incubated with 0.2 mL Affi-Gel™ 501, an organomercurial crosslinked agarose, in 50 mM sodium phosphate, pH 6, at room temperature for one hour to allow covalent mercaptide bonds to form.</p>
    <p>E. Production of Nucleoside Having a Removable 3' Blocking Moiety Using Alternate Strategies</p>
    <p>An alternative strategy for making ester and ether triphosphates involves chemical phosphorylation of a nucleoside which already contains an ester or ether protecting group at the 3' position.</p>
    <p>Isovaleroyl Ester. One hundred μL of 1M isovaleric acid dissolved in anhydrous N,N-dimethylformamide (DMF) under a nitrogen atmosphere is mixed with 100 μL of 1M carbonyldiimidazole, also in anhydrous DMF. Formation of the imidazolide is allowed to proceed at room temperature for 30 seconds. To this mixture is added 100 μL of 5'-O-dimethoxytrityl thymidine (DMT thymidine; Sigma Chemical Co.) also dissolved in anhydrous DMF. Formation of the isovaleroyl ester at the 3' hydroxyl proceeds at room temperature for 12 hours.</p>
    <p>The 5'-DMT 3'-isovalerate thymidine is purified by thin layer chromatography or HPLC and recovered from solvents in vacuo. To remove the 5' protecting group, the compound is reacted in 1 mL of methylene chloride with 5 equivalents of finely powdered, anhydrous zinc bromide with stirring at room temperature. The reaction is monitored by TLC to determine the optimum time for specific removal of the DMT group. The 5'-OH 3'-isovaleroyl ester of thymidine is then recovered after TLC or HPLC purification. These procedures are generally applicable to all four nucleosides.</p>
    <p>To phosphorylate the 5' hydroxyl, the ester is dissolved or resuspended on 0.5 ml triethyl phosphate. 0.4 mmol phosphoryl chloride (POCl<sub>3</sub>) is added and the reaction is allowed to proceed at room temperature for 1-14 hours. Purification of the 5'-monophosphate-3'O-isovaleroyl ester is by chromatography on DEAE Sephadex A-25 eluted with a linear gradient of triethylammonium carbonate (pH 7-8). The monophosphates are eluted at a buffer concentration of approximately 0.2-0.3M.</p>
    <p>The purified monophosphate ester (0.1 mmol) is converted into its pyridinium salt with the pyridinium form of Dowex-50W X-8 cation exchange resin. The tributylammonium salt is prepared by addition of tributylamine (0.2 mmol), and the product is dried by addition and evaporation of anhydrous pyridine and N,N-dimethylformamide. To a solution of the anhydrous tributylammonium salt in N,N-dimethylformamide (1 ml) is added 1,1-carbonylbis(imidazole) (0.5 mmol). Formation of the imidazolide may be monitored by TLC chromatography or HPLC. After the reaction is complete, 35 μL of methanol is added to react with remaining carbonylbis(imidazole) and the solution is kept at room temperature for 5 minutes. Tributylammonium pyrophosphate (0.5 mmol) in N,N-dimethylformamide (5 ml) is then added dropwise with stirring. The mixture is kept for several hours at room temperature then evaporated to dryness. The triphosphate is isolated by chromatography as above and is eluted at approximately 0.4-0.5M triethylammonium carbonate buffer. These phosphorylation procedures are generally applicable to all four nucleosides.</p>
    <p>Depending on the particular stability requirements, the procedures described above are readily adaptable to the utilization of virtually any carboxylic acid for esterification and protection of the 3'-hydroxyl followed by the chemical phosphorylation of the 5' hydroxyl to produce the 3' protected nucleotide triphosphate. These carboxylic acids include but are not limited to the following classes: formic, benzoylformic, chloroacetic, fluoroacetic, methoxyacetic, phenoxyacetic, chlorophenoxyacetic, phenylacetic, propionic, butyric, isovaleric, succinic, pentanoic, pivalic, adamantane carboxylic, crotonic, butenoic, substituted benzoic (e.g. nitrobenzoic, methylbenzoic, chlorobenzoic, phenylbenzoic), napthoic acids.</p>
    <p>As is understood by those of ordinary skill in the art, there are numerous methods for creating esters for the purpose of transiently protecting hydroxyl groups. In addition, there are numerous strategies for transiently protecting one hydroxyl group (e.g. the 5' hydroxyl of a nucleoside) in order to introduce an ester at another hydroxyl group (e.g. the 3' hydroxyl of a nucleoside).</p>
    <p>Silyl Ether. 1 mmol of thymidine is dissolved in 1 ml of N,N-dimethylformamide at room temperature. To this solution is added 20 mmol of imidazole and 10 mmol of tert-butyldimethylsilylchloride (t-BDMS). The reaction is allowed to proceed at room temperature until both hydroxyl groups are silylated.</p>
    <p>The disilylated nucleoside is recover by TLC or HPLC and the solvents removed in vacuo. The 5' silyl protecting group is then selectively removed by incubation in 80% acetic acid. The reaction is monitored by TLC to determine the optimum time for removal of the 5' protecting group. The resulting 3' silylated nucleoside is purified by TLC or HPLC, phosphorylation at the 5' position is by the procedures described above yielding a nucleotide triphosphate 3'-t-BDMS ether. These procedures are generally applicable to all four nucleosides.</p>
    <p>Methoxymethyl Ether. 1 mmol of DMT-thymidine is dissolved in diisopropylethylamine and reacted with 4 mmol chloromethyl methyl ether (Aldrich Chemical Co.) at 0° C. for 1 hour followed by warming to room temperature and further reaction for 8 hours. The thymidine 5'-DMT 3'-methoxymethyl ether is purified by TLC or HPLC and recovered from solvents in vacuo.</p>
    <p>To remove the 5' protecting group, the compound is reacted in 1 mL of methylene chloride with 5 equivalents of finely powdered, anhydrous zinc bromide with stirring at room temperature. The reaction is monitored by TLC to determine the optimum time for specific removal of the DMT group. The 5'-OH 3'-methoxymethyl ether of thymidine is then recovered after TLC or HPLC purification. Phosphorylation at the 5' position is accomplished by the procedures described above yielding a nucleotide triphosphate 3'-methoxymethyl ether. These procedures are generally applicable to all four nucleosides.</p>
    <p>As is understood by those of ordinary skill in the art, there are numerous methods for creating ethers for the purpose of transiently protecting hydroxyl groups. In addition, there are numerous strategies for transiently protecting one hydroxyl group (e.g. the 5' hydroxyl of a nucleoside) in order to introduce an ether at another hydroxyl group (e.g. the 3' hydroxyl of a nucleoside).</p>
    <p>Ethers useful for protection of nucleoside 3' hydroxyl groups include but are not limited to the following classes: methyl, substituted methyl (e.g. benzyloxymethyl, methoxyethoxymethyl, trimethylsilyl ethoxymethyl), pyranyl, furanyl, sustituted ethyl (e.g. ethoxyethyl, methoxyethyl, methylmethoxy ethyl), butyl, allyl, cinnamyl, benzyl and substituted benzyl (e.g. methoxybenzyl, halobenzyl, nitrobenzyl), anthryl and silyl ethers.</p>
    <heading>Example 2. Regeneration of the 3' Hydroxyl from Protected Nucleotides</heading> <p>A. Deprotection of nucleotide 3'-O esters. The ester protecting groups were removed from the 3'-hydroxyl of dNTPs by incubation in 1 mM CoCl<sub>2</sub> and 100 mM potassium cacodylate, pH 6.8. The conversion of the ester to the hydroxyl was evaluated by cellulose and paper chromatography and was found to be nearly quantitative after 15 minutes of incubation at 37° C. These deprotection conditions were equally effective for each of the four nucleotides. Further evaluation revealed that the instability was due to both the buffer and the divalent cation.</p>
    <p>The instability of the toluoyl esters in other commonly used TdTase coupling reaction buffers was explored. The relative degree of instability due to the various buffers (in the presence of 1 mM COCl<sub>2</sub>) was found to be cacodylate&gt;tris(hydroxymethyl) aminomethane&gt;sodium acetate or phosphate. Instability due to the cations (in cacodylate buffer) was found to be Co<sup>++</sup> &gt;Mn<sup>++</sup> &gt;Mg<sup>++</sup>. Degradation of the esters was first observable after about three minutes of incubation. Incubation in buffer alone or cation alone produced no observable degradation. In common with many other types of esters, these deoxynucleotide esters were also sensitive to basic conditions e.g. incubation in 10-100 mM NaOH. The other esters of dNTPs (isovaleroyl, dimethylbenzoyl, napthoyl, and nitrobenzoyl) were also relatively unstable in the buffered divalent cations.</p>
    <p>In general, these results identify unexpected properties of the esters of dNTPs and provide a convenient and gentle method for the rapid removal of these esters from the growing polynucleotide chain after a coupling reaction. These deprotection conditions are sufficiently gentle to enable the synthesis of an object polynucleotide onto a pre-existing double stranded DNA without denaturation of the DNA at every cycle. This capacity for "add-on" synthesis using a double-stranded polynucleotide as the initiating substrate is demonstrated in Example 5 below.</p>
    <p>B. Deprotection of nucleotide 3'-O ethers. The 3' ethoxy ethyl ether of the nucleoside triphosphates were stable in cobalt containing buffers but could be readily removed by incubation in 5% acetic acid at room temperature or by incubation in 0.5N HCl/THF at 0° C.</p>
    <p>C. Deprotection of nucleotide 3'-phosphates. The 3'-phosphate was specifically removed by incubation of the nucleotide in a solution containing 50 mM sodium acetate, pH 5.5, 10 mM MgCl<sub>2</sub>, and 20 units of nuclease P1, an enzyme which specifically removes phosphates from the 3' position of mononucleotides. The reaction was allowed to proceed for 90 minutes at 37° C. This enzyme would not be appropriate in the case of a protected nucleotide attached to an initiating substrate since it is also a phosphodiesterase. In this case an alternative phosphatase can be used, which is described below.</p>
    <heading>Example 3. Efficiency of Enzyme Catalyzed Phosphodiester Bond Formation Using Protected Deoxynucleotidyl Triphosphates</heading> <p>The ability of a polymerizing enzyme, TdTase, to catalyze the creation of a phosphodiester bond between an initiating polynucleotide substrate and a 3'-O-protected deoxynucleotidyl triphosphate was measured using a transferase/ligase assay. In this assay, transfer of a nucleotide to an initiating substrate DNA, such as a linearized vector, will inhibit the ability of the vector to be relegated into a circular form. The relative quantity of circular vector DNA in each reaction can then be measured by bacterial transformation.</p>
    <p>100 μM deoxynucleotide 3' ethoxy ethyl ether, or 3' phosphate were incubated with Pst 1-digested Puc 8 vector DNA (1 μg) in the presence of 1 mM CoCl<sub>2</sub>, 0.1 mM DTT, potassium cacodylate, pH 6.8, and 40 units TdTase (Promega) in a total volume of 25 μL. In the case of the 3' ester, the same reaction was performed with the exception that the CoCls was replaced with MnCl2 and the cacodylate buffer was replaced with Tris-Cl. The reactions were allowed to proceed at 37° C. for 15 minutes at which time they were terminated by the addition of 1 μL of 100 mM Na<sub>2</sub> EDTA, 0.1% sodium dodecyl sulfate. The Puc 8 DNA was separated from the other components of the reaction by chromatography through aqueous packed Sepharose™ CL-6B and was then used in a ligase reaction. The ligation reaction consisted of the Puc 8 DNA, 1 mM Na<sub>2</sub> ATP, 50 mM Tris-Cl, pH 8.0, 1 MM MgCl<sub>2</sub>, 100 μg/mL bovine serum albumin and 100 units of T4 DNA ligase (New England Biolabs). The ligation reaction was allowed to proceed at 16° C. for 18 hours. The Puc 8 DNA was again recovered by Sepharose™ CL6B chromatography.</p>
    <p>The inhibition of the ligation reaction due to the addition of a nucleotide to the Puc 8 DNA by TdTase was quantified by a bacterial transformation assay. Competent E. coli JM109 bacteria (Promega) were incubated with 100 ng of the Puc 8 DNA according to the instructions provided with the transformation competent cells. Briefly, this involved a heat shock of the admixture for one minute at 42° C., incubation of the bacteria in LB broth at 37° C. for one hour, and overnight growth of the bacteria on LB agar Petri plates containing 50 μg/mL ampicillin. Colonies from each transformation were then counted.</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">______________________________________dNTP in TdTase        Vector      Religation                             Trans-Reaction     Substrate   formed   Colonies______________________________________none         Pst 1-Puc 8 yes      1,381(positive control)none         "           no       325(background)dideoxy-ATP  "           yes      342dATP 3'-O toluate        "           yes      316dATP 3'-O ether        "           yes      330dATP 3'-phosphate        "           yes      340dATP-3'OH    "           yes      636______________________________________</pre>
    
    <p>The results demonstrate that the protected nucleotides are utilized by TdTase for the creation of phosphodiester bonds. The covalent attachment of the protected nucleotide to the vector DNA blocks the vector from religation. In the case of the unprotected nucleotide (dATP 3'-OH) the enzyme may be predominantly adding homopolymer tails to a population of vector molecules leaving some vectors unmodified.</p>
    <p>The efficiency of the TdTase catalyzed transfer, as measured by the numbers of colonies in excess of the background value, were comparable when comparing the protected mononucleotide with dideoxynucleotide. The absence of transformed colonies above the background value compared to a control which produced greater than 1000 colonies, indicates a TdTase catalyzed transfer of protected mononucleotide to &gt;99.9% of the initiating substrate 3' hydroxyls.</p>
    <heading>Example 4. Inhibition of Phosphodiester Bond Formation by Protected Nucleotides</heading> <p>Attachment of a protected mononucleotide to vector DNA will prevent the subsequent attachment of a biotin labelled nucleotide so long as the protecting group is affixed to the 3'-hydroxyl. This inhibition of vector biotinylation can be readily quantified by blotting assays after agarose gel electrophoresis.</p>
    <p>Vector DNA (either Puc 8 or pBluescript) digested with the appropriate restriction enzyme, was reacted with approximately 100 μM protected nucleotide for varying times in the presence of 25 units TdTase (Promega or BRL) in appropriate buffers in a final volume of 25 μL. To the reaction mix was then added 1 μL of 300 μM biotinylated dUTP (Sigma or Boehringer) and the reaction was allowed to proceed for 1-3 minutes at which point the reaction was stopped by the addition of 1 μL of 1% sodium dodecylsulfate, 50 mM Na<sub>2</sub> EDTA. The mixture was heated to 75° C. for 1 minute then electrophoresed in an agarose gel to visualize the DNA. In related assays, the vector DNA was purified from the other components of the reaction prior to the addition of biotinylated nucleotide. Purification was by centrifugal chromatography on Sepharose CL-6B. This step was included to avoid the possibility that low molecular weight inhibitors were slowing the activity of the TdTase.</p>
    <p>The incorporation of biotin into the DNA was measured using a standard dye reaction procedure. The DNA was first blotted onto a piece of nitrocellulose paper. The nitrocellulose paper with the DNA adhering to it was then heated to 80° C. in a drying oven for 30 minutes and re-wetted in 25 mL of 50 mM Tris-Cl pH 8.1, 150 mM NaCl, 0.1% Triton-X-100 (TBST) and 10% (w/v) Carnation non-fat dry milk, a solution which is intended to enhance the contrast of the final dye reaction. After 1 hour of incubation in the milk solution, a fresh solution of TBST containing approximately 1 μg/mL of streptavidin alkaline phosphatase (Fisher Scientific #OB5000-ALPH) was added to the paper. Binding of streptavidin to biotin proceeded for 1 hour at room temperature. The paper was then transferred to 25 mL of fresh TBST for 10 minutes to wash off excess streptavidin-alkalin phosphatase. This washing step was repeated four times. The paper was then transferred to 10 mL of 100 mM Tris-Cl, pH 9.5, 150 mM NaCl, 5 mM MgCl2, 300 μg/mL nitrotetrazolium blue and 150 μg/mL bromochloroindolyl phosphate to visualize the quantity of bound streptavidin phosphatase by enzymatic release of the chromophoric bromochloro indole.</p>
    <p>The results of the inhibition assays using a variety of blocking groups is summarized below.</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">______________________________________3' protecting       Biotinylation                   Reactiongroup (%)   time (min)  time (min)                            Inhibition______________________________________para-toluoyl       0.5-5       0.5-5    &gt;50%benzoyl     "           "        "isovaleroyl "           "        "dimethylbenzoyl       "           "        "ethoxyethyl "           "        "phosphate   "           "        "______________________________________</pre>
    
    <heading>Example 5. DNA Synthesis Using Protected dNTPs: Synthesis of a New Restriction Site in the Puc 8 Vector</heading> <p>To demonstrate the synthesis of a desired DNA sequence directly onto a vector DNA by the TdTase catalyzed addition of the protected dNTPs, we performed sequential reactions on Pst 1-digested Puc 8 DNA in order to introduce a new restriction site into the vector. The sequence at the termini of the Pst1 Puc 8 DNA is: ##STR14## where the dotted lines indicate the annealed complementary strands of the vector. Sequential coupling and cleavage reaction were performed using the toluoyl esters of dNTPs as follows:</p>
    <p>First coupling reaction--100 mM potassium cacodylate, pH 6.8, 1 mM CoCl<sub>2</sub>, 0.1 mM DTT, 0.1 mg/mL BSA, 100 μM dTTP-3'O-toluate, 40 units TdTase (Promega), 37° C., 2 minutes.</p>
    <p>Stop reaction--1 μL 100 mM Na<sub>2</sub> EDTA, 1 μL 10% sodium dodecyl sulfate, 65° C., 2 minutes.</p>
    <p>DNA recovery--Centrifugation through 0.5 mL packed Sepharose™ CL-6B in water.</p>
    <p>Ester cleavage reaction--100 mM potassium cacodylate, pH 6.8, 1 mM CoCl<sub>2</sub>, 0.1 mM DTT, 0.1 mg/mL BSA, 37° C., 30 minutes.</p>
    <p>Second coupling reaction--100 μM dGTP-3'O-toluate, 40 units TdTase (Promega), 37° C., 2 minutes.</p>
    <p>Repeat stop, recovery and cleavage.</p>
    <p>Third coupling reaction--100 μM dCTP-3'O-toluate, 40 units TdTase (Promega), 37°C., 2 minutes.</p>
    <p>Repeat stop, recovery and cleavage.</p>
    <p>Fourth coupling reaction--100 μM dATP-3'O-toluate, 40 units TdTase (Promega), 37° C., 2 minutes.</p>
    <p>Repeat stop, recovery and cleavage.</p>
    <p>Final recovery of DNA--Centrifugation through 0.5 mL packed SEPHAROSE™ CL-6B in water.</p>
    <p>A similar series of reaction were performed using the 3'-phosphates of the dNTPs with some modifications.</p>
    <p>First coupling reaction--100 mM potassium cacodylate, pH 6.8, 1 mM CoCl<sub>2</sub>, 0.1 mM DTT, 0.1 mg/mL BSA, 100 μM dTTP-3'-phosphate, 40 units TdTase (Promega), 37° C., 2 minutes.</p>
    <p>Stop reaction--1 μL 100 mM Na<sub>2</sub> EDTA, 1 μL 10% sodium dodecyl sulfate, 65° C., 2 minutes.</p>
    <p>DNA recovery--Centrifugation through 0.5 mL packed SEPHAROSE™ CL-6B in water.</p>
    <p>Phosphate cleavage reaction--0.1m Tris.Hcl, pH 9.0, 0.1m Nacl, 10 mM MgCl<sub>2</sub>, and 20 units of alkaline phosphatase, 37° C., 30 minutes.</p>
    <p>Second coupling reaction--100 μM dGTP-3'-phosphate, 40 units TdTase (Promega), 37° C., 2 minutes.</p>
    <p>Repeat stop, recovery and cleavage.</p>
    <p>Third coupling reaction--100 μM dCTP-3'-phosphate, 40 units TdTase (Promega), 37° C., 2 minutes.</p>
    <p>Repeat stop, recovery and cleavage.</p>
    <p>Fourth coupling reaction--100 μM dATP-3'phosphate, 40 units TdTase (Promega), 37° C., 2 minutes.</p>
    <p>Repeat stop, recovery and cleavage.</p>
    <p>Final recovery of DNA--Centrifugation through 0.5 mL packed SEPHAROSE™ CL-6B in water.</p>
    <p>The modified vector DNA was intended to have the following new DNA sequence: ##STR15##</p>
    <p>To demonstrate the presence of this new sequence in the vector, the modified Pst 1-Puc 8 was religated as previously described for 18 hours at 16° C. The resulting recircularized or concatemerized plasmid would have the following new structure in the Puc 8 polylinker: ##STR16## where the underlined portion is the recognition sequence for the Sph 1 restriction enzyme, which did not previously exist in the vector.</p>
    <p>The relegated vector was passed through a CL-6B spun column and incubated in the Sph 1 restriction enzyme buffer and 10 units of Sph 1 (New England Biolabs). Agarose gel electrophoresis revealed that the original Puc 8 DNA contained no Sph 1 recognition sequences and that the recovered DNA after the TdTase reactions contained the desired sequence.</p>
    <p>To demonstrate the significance of the blocking groups, an identical protocol was followed using unblocked nucleotide triphosphates in the synthesis reactions. The final religation product contained no detectible Sph 1 sequences.</p>
    <p>The foregoing examples and description of the preferred embodiment should be taken as illustrating, rather than as limiting, the present invention as defined by the claims. As will be readily appreciated, numerous variations and combinations of the features set forth above can be utilized without departing from the present invention as set forth in the claims. All such modifications are intended to be included within the scope of the following claims.</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">__________________________________________________________________________SEQUENCE LISTING(1) GENERAL INFORMATION:(iii) NUMBER OF SEQUENCES: 8(2) INFORMATION FOR SEQ ID NO: 1:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 12(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(ix) FEATURE:(D) OTHER INFORMATION: base number 12 is m7g(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 1:CCCCCCCCCCCG12(2) INFORMATION FOR SEQ ID NO: 2:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 19(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(ii) SEQUENCE DESCRIPTION: SEQ ID NO: 2:CCCCCCCCCCCCCCCTGCA19(2) INFORMATION FOR SEQ ID NO: 3:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 20(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(ii) SEQUENCE DESCRIPTION: SEQ ID NO: 3:CTGCAGGGGGGGGGGGGGGG20(2) INFORMATION FOR SEQ ID NO: 4:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 9(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(ii) SEQUENCE DESCRIPTION: SEQ ID NO: 4:CCCCCCCCC9(2) INFORMATION FOR SEQ ID NO: 5:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 9(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(ii) SEQUENCE DESCRIPTION: SEQ ID NO: 5:GGGGGGGGG9(2) INFORMATION FOR SEQ ID NO: 6:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 5(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(ii) SEQUENCE DESCRIPTION: SEQ ID NO: 6:CTGCA5(2) INFORMATION FOR SEQ ID NO: 7:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 9(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(ii) SEQUENCE DESCRIPTION: SEQ ID NO: 7:CTGCATGCA9(2) INFORMATION FOR SEQ ID NO: 8:(i) SEQUENCE CHARACTERISTICS:(A) LENGTH: 10(B) TYPE: nucleic acid(C) STRANDEDNESS: single(D) TOPOLOGY: linear(ii) SEQUENCE DESCRIPTION: SEQ ID NO: 8:CTGCATGCAG10__________________________________________________________________________</pre>
    
    </div></div></div><div class="patent-section patent-tabular-section"><a id="backward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Cited Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4096324">US4096324</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jan 30, 1976</td><td class="patent-data-table-td patent-date-value">Jun 20, 1978</td><td class="patent-data-table-td ">The Upjohn Company</td><td class="patent-data-table-td ">Anti-viral, antileukemia</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4423212">US4423212</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Dec 18, 1980</td><td class="patent-data-table-td patent-date-value">Dec 27, 1983</td><td class="patent-data-table-td ">The Upjohn Company</td><td class="patent-data-table-td ">1-(2-deoxy-b-d-pentofuranosyl)-5,6-dihydro-s-triazines</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4605735">US4605735</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Feb 9, 1984</td><td class="patent-data-table-td patent-date-value">Aug 12, 1986</td><td class="patent-data-table-td ">Wakunaga Seiyaku Kabushiki Kaisha</td><td class="patent-data-table-td ">Non radioactive affinity probe for nucleic acids; immunoassay; genetic engineering</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4689405">US4689405</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jan 4, 1984</td><td class="patent-data-table-td patent-date-value">Aug 25, 1987</td><td class="patent-data-table-td ">Gesellschaft Fur Biotechnologische Forschung Mbh (Gbf)</td><td class="patent-data-table-td ">Process for the simultaneous synthesis of several oligonucleotides on a solid phase</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4719176">US4719176</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Oct 31, 1983</td><td class="patent-data-table-td patent-date-value">Jan 12, 1988</td><td class="patent-data-table-td ">Klotz Irving M</td><td class="patent-data-table-td ">Enzyme-free diagnostic binding reagents</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4816571">US4816571</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jun 4, 1987</td><td class="patent-data-table-td patent-date-value">Mar 28, 1989</td><td class="patent-data-table-td ">Applied Biosystems, Inc.</td><td class="patent-data-table-td ">Genetic engineering method</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4820812">US4820812</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Feb 20, 1987</td><td class="patent-data-table-td patent-date-value">Apr 11, 1989</td><td class="patent-data-table-td ">Wakunaga Seiyaku Kabushiki Kaisha</td><td class="patent-data-table-td ">Oligonucleotide derivatives and production thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4863849">US4863849</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jul 18, 1985</td><td class="patent-data-table-td patent-date-value">Sep 5, 1989</td><td class="patent-data-table-td ">New York Medical College</td><td class="patent-data-table-td ">Automatable process for sequencing nucleotide</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4876335">US4876335</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jun 30, 1987</td><td class="patent-data-table-td patent-date-value">Oct 24, 1989</td><td class="patent-data-table-td ">Wakunaga Seiyaku Kabushiki Kaisha</td><td class="patent-data-table-td ">Hybridization probe</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4948882">US4948882</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">May 4, 1987</td><td class="patent-data-table-td patent-date-value">Aug 14, 1990</td><td class="patent-data-table-td ">Syngene, Inc.</td><td class="patent-data-table-td ">Single-stranded labelled oligonucleotides, reactive monomers and methods of synthesis</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4950745">US4950745</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Oct 30, 1987</td><td class="patent-data-table-td patent-date-value">Aug 21, 1990</td><td class="patent-data-table-td ">Daicel Chemical Industries, Ltd.</td><td class="patent-data-table-td ">Used in genetic engineering</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4980460">US4980460</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Mar 30, 1987</td><td class="patent-data-table-td patent-date-value">Dec 25, 1990</td><td class="patent-data-table-td ">Commissariat A L&#39;energie Atomique</td><td class="patent-data-table-td ">Protected nucleosides which permit more efficient oligonucleotide syntheses</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5003059">US5003059</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jun 20, 1988</td><td class="patent-data-table-td patent-date-value">Mar 26, 1991</td><td class="patent-data-table-td ">Genomyx, Inc.</td><td class="patent-data-table-td ">Determining DNA sequences by mass spectrometry</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5032680">US5032680</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Feb 24, 1989</td><td class="patent-data-table-td patent-date-value">Jul 16, 1991</td><td class="patent-data-table-td ">Kuraray Co., Ltd.</td><td class="patent-data-table-td ">Antitumor</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5039796">US5039796</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Aug 21, 1989</td><td class="patent-data-table-td patent-date-value">Aug 13, 1991</td><td class="patent-data-table-td ">Hoechst Aktiengesellschaft</td><td class="patent-data-table-td ">Using phosphoramine compound</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5047524">US5047524</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Dec 21, 1988</td><td class="patent-data-table-td patent-date-value">Sep 10, 1991</td><td class="patent-data-table-td ">Applied Biosystems, Inc.</td><td class="patent-data-table-td ">Solid phase syntheisis o nonsuelladle porous polystyrene support by phosphoramide or hydrogen phosphonate chemistry</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5091519">US5091519</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">May 1, 1986</td><td class="patent-data-table-td patent-date-value">Feb 25, 1992</td><td class="patent-data-table-td ">Amoco Corporation</td><td class="patent-data-table-td ">Nucleotide compositions with linking groups</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5093232">US5093232</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Dec 23, 1986</td><td class="patent-data-table-td patent-date-value">Mar 3, 1992</td><td class="patent-data-table-td ">Chiron Corporation</td><td class="patent-data-table-td ">Nucleic acid probes</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5112963">US5112963</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Nov 14, 1988</td><td class="patent-data-table-td patent-date-value">May 12, 1992</td><td class="patent-data-table-td ">Max-Planck-Gesellschaft Zur Foerderung Der Wissenschaften E.V.</td><td class="patent-data-table-td ">Modified oligonucleotides</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5256549">US5256549</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Apr 27, 1990</td><td class="patent-data-table-td patent-date-value">Oct 26, 1993</td><td class="patent-data-table-td ">Chiron Corporation</td><td class="patent-data-table-td ">Purification of synthetic oligomers</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5258506">US5258506</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Aug 25, 1989</td><td class="patent-data-table-td patent-date-value">Nov 2, 1993</td><td class="patent-data-table-td ">Chiron Corporation</td><td class="patent-data-table-td ">Photolabile reagents for incorporation into oligonucleotide chains</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5262530">US5262530</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jul 27, 1990</td><td class="patent-data-table-td patent-date-value">Nov 16, 1993</td><td class="patent-data-table-td ">Applied Biosystems, Inc.</td><td class="patent-data-table-td ">Solid phase synthesis on polystyrene support, phosphoramidite chemistry</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5262536">US5262536</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Dec 5, 1990</td><td class="patent-data-table-td patent-date-value">Nov 16, 1993</td><td class="patent-data-table-td ">E. I. Du Pont De Nemours And Company</td><td class="patent-data-table-td ">Reagents for the preparation of 5&#39;-tagged oligonucleotides</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5264563">US5264563</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Dec 14, 1992</td><td class="patent-data-table-td patent-date-value">Nov 23, 1993</td><td class="patent-data-table-td ">Ixsys Inc.</td><td class="patent-data-table-td ">Sequentially coupling monomers on supports to form atleast two tuplet sin separate reaction vessels, mixing and into two or more reaction vessels, repeating the process</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5264566">US5264566</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Apr 26, 1990</td><td class="patent-data-table-td patent-date-value">Nov 23, 1993</td><td class="patent-data-table-td ">Genentech, Inc.</td><td class="patent-data-table-td ">Method for in vitro oligonucleotide synthesis using H-phosphonates</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5268266">US5268266</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Apr 11, 1986</td><td class="patent-data-table-td patent-date-value">Dec 7, 1993</td><td class="patent-data-table-td ">Genetics Institute, Inc.</td><td class="patent-data-table-td ">Process and nucleic acid construct for producing reagent complexes useful in determining target nucleotide sequences</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5268464">US5268464</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Mar 15, 1993</td><td class="patent-data-table-td patent-date-value">Dec 7, 1993</td><td class="patent-data-table-td ">Ciba-Geigy Corporation</td><td class="patent-data-table-td ">Process for the preparation of nucleosides</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5281701">US5281701</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jul 12, 1991</td><td class="patent-data-table-td patent-date-value">Jan 25, 1994</td><td class="patent-data-table-td ">Applied Biosystems, Inc.</td><td class="patent-data-table-td ">Process and compounds for RNA synthesis</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5348868">US5348868</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Apr 24, 1992</td><td class="patent-data-table-td patent-date-value">Sep 20, 1994</td><td class="patent-data-table-td ">Beckman Instruments, Inc.</td><td class="patent-data-table-td ">Incubating with methylamine and tert-butylamine</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5362866">US5362866</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Mar 17, 1992</td><td class="patent-data-table-td patent-date-value">Nov 8, 1994</td><td class="patent-data-table-td ">Molecular Biosystems, Inc.</td><td class="patent-data-table-td ">Oligonucleotide polymeric support system with an oxidation cleavable link</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5367066">US5367066</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jul 24, 1991</td><td class="patent-data-table-td patent-date-value">Nov 22, 1994</td><td class="patent-data-table-td ">Chiron Corporation</td><td class="patent-data-table-td ">Oligonucleotides with selectably cleavable and/or abasic sites</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5380833">US5380833</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Dec 13, 1991</td><td class="patent-data-table-td patent-date-value">Jan 10, 1995</td><td class="patent-data-table-td ">Chiron Corporation</td><td class="patent-data-table-td ">Polynucleotide reagents containing selectable cleavage sites</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5436143">US5436143</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Dec 23, 1992</td><td class="patent-data-table-td patent-date-value">Jul 25, 1995</td><td class="patent-data-table-td ">Hyman; Edward D.</td><td class="patent-data-table-td ">Genetic engineering; one vessel; enzyme deactivation</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5652099">US5652099</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Aug 18, 1994</td><td class="patent-data-table-td patent-date-value">Jul 29, 1997</td><td class="patent-data-table-td ">Conrad; Michael J.</td><td class="patent-data-table-td ">For use in synthesis of fluorescent oligonucleotides</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="http://www.google.com/url?id=pGhJBAABERAJ&amp;q=http://worldwide.espacenet.com/publicationDetails/biblio%3FCC%3DJP%26NR%3DS5538324A%26KC%3DA%26FT%3DD&amp;usg=AFQjCNEpigTqFPg0OMANYfQSG3JThkXMNg">JPS5538324A</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value"></td><td class="patent-data-table-td patent-date-value"></td><td class="patent-data-table-td "> </td><td class="patent-data-table-td citation-no-title">Title not available</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="npl-citations"></a><div class="patent-section-header"><span class="patent-section-title">Non-Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th colspan="3"class="patent-data-table-th">Reference</th></tr></thead><tr><td class="patent-data-table-td ">1</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Bennett et al., Biochemistry , vol. 12, 20:3956 3960 (1973).</td></tr><tr><td class="patent-data-table-td ">2</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Bennett et al., Biochemistry, vol. 12, 20:3956-3960 (1973).</td></tr><tr><td class="patent-data-table-td ">3</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Bollum, Fed. Proc. Soc. Exp. Biol. Med. , 17, 193 (1958).</td></tr><tr><td class="patent-data-table-td ">4</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Bollum, Fed. Proc. Soc. Exp. Biol. Med., 17, 193 (1958).</td></tr><tr><td class="patent-data-table-td ">5</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Canard et al., Proc. Nat l. Acad. Sci. USA , 92, 10859 10863 (Nov. 1995).</td></tr><tr><td class="patent-data-table-td ">6</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Canard et al., Proc. Nat'l. Acad. Sci. USA, 92, 10859-10863 (Nov. 1995).</td></tr><tr><td class="patent-data-table-td ">7</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Chang and Bollum, Biochemistry , vol. 10, 3:536 542 (1971).</td></tr><tr><td class="patent-data-table-td ">8</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Chang and Bollum, Biochemistry, vol. 10, 3:536-542 (1971).</td></tr><tr><td class="patent-data-table-td ">9</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Deng and Wu, Meth. Enzymol. , 100: 96 116 (1983).</td></tr><tr><td class="patent-data-table-td ">10</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Deng and Wu, Meth. Enzymol., 100: 96-116 (1983).</td></tr><tr><td class="patent-data-table-td ">11</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">England and Uhlenbeck, Biochemistry , vol. 17, 11:2069 2076 (1978).</td></tr><tr><td class="patent-data-table-td ">12</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">England and Uhlenbeck, Biochemistry, vol. 17, 11:2069-2076 (1978).</td></tr><tr><td class="patent-data-table-td ">13</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Flugel et al., Biochem. Biophys. Acta. , 308:35 40 (1973).</td></tr><tr><td class="patent-data-table-td ">14</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Flugel et al., Biochem. Biophys. Acta., 308:35-40 (1973).</td></tr><tr><td class="patent-data-table-td ">15</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Hinton and Gumport, Nucleic Acid Res. , 7:453 464 (1979).</td></tr><tr><td class="patent-data-table-td ">16</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Hinton and Gumport, Nucleic Acid Res., 7:453-464 (1979).</td></tr><tr><td class="patent-data-table-td ">17</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Hyman, Edward David, The Hyman Method: Oligonucleotide Synthesis and Plasmid Preparation, 1995.</td></tr><tr><td class="patent-data-table-td ">18</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Kaufmann et al., Eur. J. Biochem , 24:4 11 (1971).</td></tr><tr><td class="patent-data-table-td ">19</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Kaufmann et al., Eur. J. Biochem, 24:4-11 (1971).</td></tr><tr><td class="patent-data-table-td ">20</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Kossel and Roychoudrury, Eur. J. Biochem, 22:271 276 (1971).</td></tr><tr><td class="patent-data-table-td ">21</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Kossel and Roychoudrury, Eur. J. Biochem, 22:271-276 (1971).</td></tr><tr><td class="patent-data-table-td ">22</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Metzker et al., Nucleic Acids Res. , 22(20), 4259 4267 (1994).</td></tr><tr><td class="patent-data-table-td ">23</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Metzker et al., Nucleic Acids Res., 22(20), 4259-4267 (1994).</td></tr><tr><td class="patent-data-table-td ">24</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Modak, Biochemistry , 17, 3116 3120 (1978).</td></tr><tr><td class="patent-data-table-td ">25</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Modak, Biochemistry, 17, 3116-3120 (1978).</td></tr><tr><td class="patent-data-table-td ">26</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Sarfati et al., J. Biol. Chem. , 265(31), 18902 18906 (1990).</td></tr><tr><td class="patent-data-table-td ">27</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Sarfati et al., J. Biol. Chem., 265(31), 18902-18906 (1990).</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="forward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Referenced by</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Citing Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7057026">US7057026</a></td><td class="patent-data-table-td patent-date-value">Aug 23, 2002</td><td class="patent-data-table-td patent-date-value">Jun 6, 2006</td><td class="patent-data-table-td ">Solexa Limited</td><td class="patent-data-table-td ">Labelled nucleotides</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7345159">US7345159</a></td><td class="patent-data-table-td patent-date-value">Nov 6, 2003</td><td class="patent-data-table-td patent-date-value">Mar 18, 2008</td><td class="patent-data-table-td ">The Trustees Of Columbia University In The City Of New York</td><td class="patent-data-table-td ">Massive parallel method for decoding DNA and RNA</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7414116">US7414116</a></td><td class="patent-data-table-td patent-date-value">Aug 22, 2003</td><td class="patent-data-table-td patent-date-value">Aug 19, 2008</td><td class="patent-data-table-td ">Illumina Cambridge Limited</td><td class="patent-data-table-td ">Labelled nucleotides</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7427673">US7427673</a></td><td class="patent-data-table-td patent-date-value">Dec 4, 2002</td><td class="patent-data-table-td patent-date-value">Sep 23, 2008</td><td class="patent-data-table-td ">Illumina Cambridge Limited</td><td class="patent-data-table-td ">Labelled nucleotides</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7541444">US7541444</a></td><td class="patent-data-table-td patent-date-value">Aug 22, 2003</td><td class="patent-data-table-td patent-date-value">Jun 2, 2009</td><td class="patent-data-table-td ">Illumina Cambridge Limited</td><td class="patent-data-table-td ">Invention relates to nucleotide or nucleoside molecules modified by reversible covalent attachment of 3&#39;-OH blocking groups thereto, and which molecules may be used in reactions where blocked nucleotide or nucleoside molecules are required, such as in sequencing reactions, polynucleotide synthesis</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7566537">US7566537</a></td><td class="patent-data-table-td patent-date-value">Dec 13, 2005</td><td class="patent-data-table-td patent-date-value">Jul 28, 2009</td><td class="patent-data-table-td ">Illumina Cambridge Limited</td><td class="patent-data-table-td ">Labelled nucleotides</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7592435">US7592435</a></td><td class="patent-data-table-td patent-date-value">Jul 27, 2006</td><td class="patent-data-table-td patent-date-value">Sep 22, 2009</td><td class="patent-data-table-td ">Illumina Cambridge Limited</td><td class="patent-data-table-td ">increase the intensity of the fluorescent signal after nucleic acid sequencing reactions; a modified fluorescent labelled guanine-containing nucleotides which exhibit enhanced fluorophore intensity</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7622279">US7622279</a></td><td class="patent-data-table-td patent-date-value">Mar 3, 2005</td><td class="patent-data-table-td patent-date-value">Nov 24, 2009</td><td class="patent-data-table-td ">The Trustees Of Columbia University In The City Of New York</td><td class="patent-data-table-td ">DNA or RNA comprising self-priming moiety are bound to a solid substrate via 1,3-dipolar azide-alkyne cycloaddition chemistry</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7635578">US7635578</a></td><td class="patent-data-table-td patent-date-value">Aug 20, 2007</td><td class="patent-data-table-td patent-date-value">Dec 22, 2009</td><td class="patent-data-table-td ">The Trustees Of Columbia University In The City Of New York</td><td class="patent-data-table-td ">Massive parallel method for decoding DNA and RNA</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7713698">US7713698</a></td><td class="patent-data-table-td patent-date-value">Aug 20, 2007</td><td class="patent-data-table-td patent-date-value">May 11, 2010</td><td class="patent-data-table-td ">The Trustees Of Columbia University In The City Of New York</td><td class="patent-data-table-td ">Massive parallel method for decoding DNA and RNA</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7771973">US7771973</a></td><td class="patent-data-table-td patent-date-value">Jun 1, 2009</td><td class="patent-data-table-td patent-date-value">Aug 10, 2010</td><td class="patent-data-table-td ">Illumina Cambridge Limited</td><td class="patent-data-table-td ">Modified nucleotides</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7772384">US7772384</a></td><td class="patent-data-table-td patent-date-value">Sep 9, 2008</td><td class="patent-data-table-td patent-date-value">Aug 10, 2010</td><td class="patent-data-table-td ">Illumina Cambridge Limited</td><td class="patent-data-table-td ">nucleotides having a removable labels, used in polynucleotide sequencing</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7785796">US7785796</a></td><td class="patent-data-table-td patent-date-value">Jul 23, 2009</td><td class="patent-data-table-td patent-date-value">Aug 31, 2010</td><td class="patent-data-table-td ">Illumina Cambridge Limited</td><td class="patent-data-table-td ">sequence determination of an immobilized target polynucleotide comprising monitoring the sequential incorporation of nucleotides complementary to the immobilized target polynucleotide; the nucleotides have a base that is linked to a detectable label via a cleavable linker; detection of the labels</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7790869">US7790869</a></td><td class="patent-data-table-td patent-date-value">Jun 5, 2007</td><td class="patent-data-table-td patent-date-value">Sep 7, 2010</td><td class="patent-data-table-td ">The Trustees Of Columbia University In The City Of New York</td><td class="patent-data-table-td ">A nucleotide having a base attached to a detectable label through a cleavable linker; has a deoxyribose comprising a cleavable chemical group capping the 3&#39; OH group of the deoxyribose and wherein both the cleavable linker and the capping cleavable chemical group are cleavable by light</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7795424">US7795424</a></td><td class="patent-data-table-td patent-date-value">Jul 24, 2008</td><td class="patent-data-table-td patent-date-value">Sep 14, 2010</td><td class="patent-data-table-td ">Illumina Cambridge Limited</td><td class="patent-data-table-td ">Labelled nucleotides</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7816503">US7816503</a></td><td class="patent-data-table-td patent-date-value">Sep 21, 2009</td><td class="patent-data-table-td patent-date-value">Oct 19, 2010</td><td class="patent-data-table-td ">Illumina Cambridge Limited</td><td class="patent-data-table-td ">increase the intensity of the fluorescent signal after nucleic acid sequencing reactions; a modified fluorescent labelled guanine-containing nucleotides which exhibit enhanced fluorophore intensity</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7893227">US7893227</a></td><td class="patent-data-table-td patent-date-value">Dec 5, 2006</td><td class="patent-data-table-td patent-date-value">Feb 22, 2011</td><td class="patent-data-table-td ">Lasergen, Inc.</td><td class="patent-data-table-td ">Polymorphic identification; fluorescent dyes</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7897737">US7897737</a></td><td class="patent-data-table-td patent-date-value">Dec 5, 2006</td><td class="patent-data-table-td patent-date-value">Mar 1, 2011</td><td class="patent-data-table-td ">Lasergen, Inc.</td><td class="patent-data-table-td ">Polymorphic identification; fluorescent dyes</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7964352">US7964352</a></td><td class="patent-data-table-td patent-date-value">Nov 11, 2008</td><td class="patent-data-table-td patent-date-value">Jun 21, 2011</td><td class="patent-data-table-td ">Lasergen, Inc.</td><td class="patent-data-table-td ">3′-OH unblocked nucleotides and nucleosides, base modified with labels and photocleavable, terminating groups and methods for their use in DNA sequencing</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8071739">US8071739</a></td><td class="patent-data-table-td patent-date-value">Jul 20, 2010</td><td class="patent-data-table-td patent-date-value">Dec 6, 2011</td><td class="patent-data-table-td ">Illumina Cambridge Limited</td><td class="patent-data-table-td ">Modified nucleotides</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8084590">US8084590</a></td><td class="patent-data-table-td patent-date-value">Jun 21, 2010</td><td class="patent-data-table-td patent-date-value">Dec 27, 2011</td><td class="patent-data-table-td ">Illumina Cambridge Limited</td><td class="patent-data-table-td ">Labelled nucleotides</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8088575">US8088575</a></td><td class="patent-data-table-td patent-date-value">Jul 19, 2010</td><td class="patent-data-table-td patent-date-value">Jan 3, 2012</td><td class="patent-data-table-td ">The Trustees Of Columbia University In The City Of New York</td><td class="patent-data-table-td ">Massive parallel method for decoding DNA and RNA</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8133669">US8133669</a></td><td class="patent-data-table-td patent-date-value">May 21, 2009</td><td class="patent-data-table-td patent-date-value">Mar 13, 2012</td><td class="patent-data-table-td ">Trilink Biotechnologies</td><td class="patent-data-table-td ">Chemically modified nucleoside 5′-triphosphates for thermally initiated amplification of nucleic acid</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8148064">US8148064</a></td><td class="patent-data-table-td patent-date-value">Jul 19, 2010</td><td class="patent-data-table-td patent-date-value">Apr 3, 2012</td><td class="patent-data-table-td ">Illumina Cambridge Limited</td><td class="patent-data-table-td ">Labelled nucleotides</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8148503">US8148503</a></td><td class="patent-data-table-td patent-date-value">Jun 11, 2009</td><td class="patent-data-table-td patent-date-value">Apr 3, 2012</td><td class="patent-data-table-td ">Lasergen, Inc.</td><td class="patent-data-table-td ">Nucleotides and nucleosides and methods for their use in DNA sequencing</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8158346">US8158346</a></td><td class="patent-data-table-td patent-date-value">Jul 13, 2010</td><td class="patent-data-table-td patent-date-value">Apr 17, 2012</td><td class="patent-data-table-td ">Illumina Cambridge Limited</td><td class="patent-data-table-td ">Labelled nucleotides</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8198029">US8198029</a></td><td class="patent-data-table-td patent-date-value">Jan 7, 2011</td><td class="patent-data-table-td patent-date-value">Jun 12, 2012</td><td class="patent-data-table-td ">Lasergen, Inc</td><td class="patent-data-table-td ">3&#39;-OH unblocked nucleotides and nucleosides base modified with non-cleavable, terminating groups and methods for their use in DNA sequencing</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8212015">US8212015</a></td><td class="patent-data-table-td patent-date-value">Oct 6, 2010</td><td class="patent-data-table-td patent-date-value">Jul 3, 2012</td><td class="patent-data-table-td ">Illumina Cambridge Limited</td><td class="patent-data-table-td ">Modified nucleosides and nucleotides and uses thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8298792">US8298792</a></td><td class="patent-data-table-td patent-date-value">Feb 8, 2011</td><td class="patent-data-table-td patent-date-value">Oct 30, 2012</td><td class="patent-data-table-td ">The Trustees Of Columbia University In The City Of New York</td><td class="patent-data-table-td ">Four-color DNA sequencing by synthesis using cleavable fluorescent nucleotide reversible terminators</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8361727">US8361727</a></td><td class="patent-data-table-td patent-date-value">May 24, 2011</td><td class="patent-data-table-td patent-date-value">Jan 29, 2013</td><td class="patent-data-table-td ">Lasergen, Inc.</td><td class="patent-data-table-td ">3′OH-unblocked, nucleotides and nucleosides base modified with labels and photocleavable, terminating groups and methods for their use in DNA sequencing</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8394586">US8394586</a></td><td class="patent-data-table-td patent-date-value">Mar 28, 2012</td><td class="patent-data-table-td patent-date-value">Mar 12, 2013</td><td class="patent-data-table-td ">Illumina Cambridge Limited</td><td class="patent-data-table-td ">Labelled nucleotides</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8399188">US8399188</a></td><td class="patent-data-table-td patent-date-value">Sep 4, 2007</td><td class="patent-data-table-td patent-date-value">Mar 19, 2013</td><td class="patent-data-table-td ">Illumina, Inc.</td><td class="patent-data-table-td ">Compositions and methods for nucleotide sequencing</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8497360">US8497360</a></td><td class="patent-data-table-td patent-date-value">Feb 28, 2012</td><td class="patent-data-table-td patent-date-value">Jul 30, 2013</td><td class="patent-data-table-td ">Lasergen, Inc.</td><td class="patent-data-table-td ">Nucleotides and nucleosides and methods for their use in DNA sequencing</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8597881">US8597881</a></td><td class="patent-data-table-td patent-date-value">Oct 25, 2011</td><td class="patent-data-table-td patent-date-value">Dec 3, 2013</td><td class="patent-data-table-td ">Illumina Cambridge Limited</td><td class="patent-data-table-td ">Modified nucleotides</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2657869A2?cl=en">EP2657869A2</a></td><td class="patent-data-table-td patent-date-value">Aug 28, 2008</td><td class="patent-data-table-td patent-date-value">Oct 30, 2013</td><td class="patent-data-table-td ">Applied Biosystems, LLC</td><td class="patent-data-table-td ">Alternative nucleic acid sequencing methods</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2002016023A2?cl=en">WO2002016023A2</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Aug 17, 2001</td><td class="patent-data-table-td patent-date-value">Feb 28, 2002</td><td class="patent-data-table-td ">Thomas M Brennan</td><td class="patent-data-table-td ">Method and apparatus for in situ synthesis on a solid support</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="classifications"></a><div class="patent-section-header"><span class="patent-section-title">Classifications</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th"> </th><th class="patent-data-table-th"> </th></tr></thead><tr><td class="patent-data-table-td ">U.S. Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=pGhJBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S026260">536/26.26</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=pGhJBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S026700">536/26.7</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=pGhJBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S026600">536/26.6</a></span></td></tr><tr><td class="patent-data-table-td ">International Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=pGhJBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12P0019340000">C12P19/34</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=pGhJBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0015090000">C12N15/09</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=pGhJBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C07H0019100000">C07H19/10</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=pGhJBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C07H0021000000">C07H21/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=pGhJBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C07H0019200000">C07H19/20</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=pGhJBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12P0019300000">C12P19/30</a></span></td></tr><tr><td class="patent-data-table-td ">Cooperative Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=pGhJBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C07H19/10">C07H19/10</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=pGhJBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12P19/30">C12P19/30</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=pGhJBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C07H21/00">C07H21/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=pGhJBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C07H19/20">C07H19/20</a></span></td></tr><tr><td class="patent-data-table-td ">European Classification</td><td class="patent-data-table-td "><span class="nested-value">C07H19/10</span>, <span class="nested-value">C07H21/00</span>, <span class="nested-value">C07H19/20</span>, <span class="nested-value">C12P19/30</span></td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="legal-events"></a><div class="patent-section-header"><span class="patent-section-title">Legal Events</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Date</th><th class="patent-data-table-th">Code</th><th class="patent-data-table-th">Event</th><th class="patent-data-table-th">Description</th></tr></thead><tr><td class="patent-data-table-td patent-date-value">Oct 4, 2011</td><td class="patent-data-table-td ">B1</td><td class="patent-data-table-td ">Reexamination certificate first reexamination</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">THE PATENTABILITY OF CLAIM 6 IS CONFIRMED. CLAIMS 1 AND 3 ARE DETERMINED TO BE PATENTABLE AS AMENDED. CLAIMS 2, 4 AND 5, DEPENDENT ON AN AMENDED CLAIM, ARE DETERMINED TO BE PATENTABLE. NEW CLAIMS 7-9 ARE ADDED AND DETERMINED TO BE PATENTABLE.</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Oct 19, 2010</td><td class="patent-data-table-td ">RR</td><td class="patent-data-table-td ">Request for reexamination filed</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20100723</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Aug 24, 2010</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">MERGER;ASSIGNOR:SOLEXA, INC.;REEL/FRAME:024864/0907</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20100208</span></div><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">ILLUMINA, INC., CALIFORNIA</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Jul 14, 2010</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">12</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Aug 4, 2006</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">8</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Apr 29, 2005</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">SOLEXA, INC., CALIFORNIA</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNORS:HIATT, ANDREW C.;ROSE, FLOYD D.;REEL/FRAME:016182/0247</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20050428</span></div><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">SOLEXA, INC. 25861 INDUSTRIAL BLVD.HAYWARD, CALIFO</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNORS:HIATT, ANDREW C. /AR;REEL/FRAME:016182/0247</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Jul 1, 2002</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">4</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Oct 5, 1999</td><td class="patent-data-table-td ">CC</td><td class="patent-data-table-td ">Certificate of correction</td><td class="patent-data-table-td "></td></tr><tr><td class="patent-data-table-td patent-date-value">May 28, 1997</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">HIATT, ANDREW C., CALIFORNIA</span></div><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">ROSE, FLOYD D., CALIFORNIA</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNOR:ROSE-HIATT BIOTECHNOLOGY PARTNERSHIP;REEL/FRAME:008551/0714</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">19970520</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Jun 7, 1995</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">ROSE HIATT BIOTECHNOLOGY PARTNERSHIP, CALIFORNIA</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNORS:HIATT, ANDREW C.;ROSE, FLOYD;REEL/FRAME:007535/0613</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">19950606</span></div></td></tr></table><div class="patent-section-footer"></div></div><div class="modal-dialog" id="patent-images-lightbox"><div class="patent-lightbox-controls"><div class="patent-lightbox-rotate-controls"><div class="patent-lightbox-rotation-text">Rotate</div><div class="rotate-icon rotate-ccw-icon"></div><div class="rotate-icon rotate-cw-icon"></div></div><div class="patent-lightbox-index-counter"></div><a class="patent-lightbox-fullsize-link" target="_blank">Original Image</a><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_right.png" alt="Next page"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_left.png" alt="Previous page"width="21" height="21" /></div></div><div class="modal-dialog-content"><div class="patent-lightbox-image-holder"><div class="patent-lightbox-placeholder"></div></div></div></div><script>_OC_initPatentsAtb({image_not_available_html: " Image not available"});</script></div></div></div></td></tr></table><script>(function() {var href = window.location.href;if (href.indexOf('?') !== -1) {var parameters = href.split('?')[1].split('&');for (var i = 0; i < parameters.length; i++) {var param = parameters[i].split('=');if (param[0] == 'focus') {var elem = document.getElementById(param[1]);if (elem) {elem.focus();}}}}})();</script><script>_OC_addFlags({LockSrc:"/books/javascript/lock_8a2b04e7bf975d5171d8e4c0b6365c7a.js", Host:"http://www.google.com/", IsBooksRentalEnabled:1, IsWebstoreDisplayCaseEnabled:1, IsObfuscationEnabled:1, IsBrowsingHistoryEnabled:1, IsWebReaderSvgEnabled:0, IsGeoLayerEnabled:1, IsImageModeNotesEnabled:1, IsCopyMenuItemEnabled:1, IsGiftingEnabled:0, IsWebReaderUniversalPaginatorEnabled:0, IsOfflineBubbleEnabled:1, IsReaderEnabledForPlayRequests:1, IsFutureOnSaleVolumesEnabled:1, IsOfflineRestrictedCopyEnabled:1, IsBooksUnifiedLeftNavEnabled:1, IsRestrictedCopyEnabled:1, IsZipitFolderCollectionEnabled:1, IsEndOfSampleRecommendationsEnabled:1, IsRatingsOnBookcardsEnabled:1, IsAdsDisabled:0, IsIframePageDisplayEnabled:0, IsEmbeddedMediaEnabled:1, IsImageModeAnnotationsEnabled:1, IsMyLibraryGooglePlusEnabled:1, IsImagePageProviderEnabled:0, IsBookcardListPriceSmall:0, IsInternalUser:0, IsBooksShareButtonEnabled:0, IsPreOrdersEnabled:0, IsDisabledRandomBookshelves:0, WebstoreDisplayCasePosition:3});_OC_Run({"enable_p13n":false,"add_vol_to_collection_base_url":"http://www.google.com/patents?op=add\u0026sig=ACfU3U3PdNXoi863_dUFUPLDuNfJhTLO2g\u0026id=pGhJBAABERAJ","remove_vol_from_collection_base_url":"http://www.google.com/patents?op=remove\u0026sig=ACfU3U28V9mVsk0CNLVUEMTl15BFuqwybQ\u0026id=pGhJBAABERAJ","logged_in":false,"p13n_save_user_settings_url":"http://www.google.com/patents?op=edit_user_settings\u0026sig=ACfU3U1nUzHEkkcY45JwZfavn8ic8J9fUA","is_cobrand":false,"sign_in_url":"https://www.google.com/accounts/Login?service=\u0026continue=http://www.google.com/patents%3Fhl%3Den\u0026hl=en","is_play_enabled":true}, {"volume_id":"","is_ebook":true,"volumeresult":{"has_flowing_text":false,"has_scanned_text":true,"can_download_pdf":false,"can_download_epub":false,"is_pdf_drm_enabled":false,"is_epub_drm_enabled":false,"download_pdf_url":"http://www.google.com/patents/download/3_protected_nucleotides_for_enzyme_catal.pdf?id=pGhJBAABERAJ\u0026output=pdf\u0026sig=ACfU3U1iXgwmOoz91ePV5mjSgM6GeMeN_A"},"sample_url":"http://www.google.com/patents/reader?id=pGhJBAABERAJ\u0026printsec=frontcover\u0026output=reader\u0026source=gbs_atb_hover","is_browsable":true,"is_public_domain":true}, {});</script><div id="footer_table" style="font-size:83%;text-align:center;position:relative;top:20px;height:4.5em;margin-top:2em"><div style="margin-bottom:8px"><a href=http://www.google.com/><nobr>Google&nbsp;Home</nobr></a> - <a href=//www.google.com/patents/sitemap/><nobr>Sitemap</nobr></a> - <a href=http://www.google.com/googlebooks/uspto.html><nobr>USPTO Bulk Downloads</nobr></a> - <a href=/intl/en/privacy/><nobr>Privacy Policy</nobr></a> - <a href=/intl/en/policies/terms/><nobr>Terms of Service</nobr></a> - <a href=https://support.google.com/faqs/answer/2539193?hl=en><nobr>About Google Patents</nobr></a> - <a href="http://www.google.com/tools/feedback/intl/en/error.html" onclick="try{_OC_startFeedback({productId: '72792',locale: 'en'});return false;}catch(e){}"><nobr>Send Feedback</nobr></a></div><span>Data provided by IFI CLAIMS Patent Services</span><br><span >&copy;2012 Google</span></div> <script type="text/javascript">var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));</script><script type="text/javascript">var pageTracker = _gat._getTracker("UA-27188110-1");pageTracker._setCookiePath("/patents/");pageTracker._trackPageview();</script> </body></html>