<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN""http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html><head><title>Patent US5969119 - Sequencing based on repeated cycles of duplex extension along a single ... - Google Patents</title><script>(function(){(function(){function e(a){this.t={};this.tick=function(a,c,b){var d=void 0!=b?b:(new Date).getTime();this.t[a]=[d,c];if(void 0==b)try{window.console.timeStamp("CSI/"+a)}catch(e){}};this.tick("start",null,a)}var a;window.performance&&(a=window.performance.timing);var f=a?new e(a.responseStart):new e;window.jstiming={Timer:e,load:f};if(a){var c=a.navigationStart,d=a.responseStart;0<c&&d>=c&&(window.jstiming.srt=d-c)}if(a){var b=window.jstiming.load;0<c&&d>=c&&(b.tick("_wtsrt",void 0,c),b.tick("wtsrt_",
"_wtsrt",d),b.tick("tbsd_","wtsrt_"))}try{a=null,window.chrome&&window.chrome.csi&&(a=Math.floor(window.chrome.csi().pageT),b&&0<c&&(b.tick("_tbnd",void 0,window.chrome.csi().startE),b.tick("tbnd_","_tbnd",c))),null==a&&window.gtbExternal&&(a=window.gtbExternal.pageT()),null==a&&window.external&&(a=window.external.pageT,b&&0<c&&(b.tick("_tbnd",void 0,window.external.startE),b.tick("tbnd_","_tbnd",c))),a&&(window.jstiming.pt=a)}catch(g){}})();})();
</script><link rel="stylesheet" href="/patents/css/_8a2b04e7bf975d5171d8e4c0b6365c7a/kl_intl_patents_bundle.css" type="text/css" /><script src="/books/javascript/atb_8a2b04e7bf975d5171d8e4c0b6365c7a__en.js"></script><script>function googleTranslateElementInit() {new google.translate.TranslateElement({pageLanguage: "en",gaTrack: true,gaId: "UA-27188110-1",multilanguagePage: true});}</script><script src="//translate.google.com/translate_a/element.js?cb=googleTranslateElementInit"></script><meta name="DC.type" content="Patent"><meta name="DC.title" content="Sequencing based on repeated cycles of duplex extension along a single stranded template; oligonucleotide probes having a blocking moiety in a terminal position so that only a single extension of the extended duplex occurs in a single cycle"><meta name="DC.contributor" content="Stephen C. Macevicz" scheme="inventor"><meta name="DC.contributor" content="Lynx Therapeutics, Inc." scheme="assignee"><meta name="DC.date" content="1997-6-10" scheme="dateSubmitted"><meta name="DC.description" content="Disclosed are oligonucleotides probes which are useful for determining a sequence of nucleotides in a target ploynucleotide. In one embodiment, the probe is of the formula: HO-(3&#39;)(B)j(5&#39;)-OP(═O-)NH-(B)k-Bt*, wherein B is a nucleotide or an analog thereof; j is in range of from 1 to 12; is in the range of from 0 to 12, such that the sum of j and k is less than or equal to 12; and Bt* is a labeled chain-terminating moiety. In s second embodiment, the probe has a formula selected from the group consisting of (1) OP(═O)(O--)O--(5═)(B)sRRRR(B)wBt*, and (2) OP(═O)(O--)(5&#39;)(B)sRRRR(B)w(3&#39;)OP(═O)(O--)O, wherein: is a deoxyribonucleotide or an analog thereof; R is a ribonucleotide; s is in the range of from 1 to 8; w is in the range of from 0 to 8, such that the sum of j and k is less than or equal to 8; and Bt* is a labeled chain-terminating moiety."><meta name="DC.date" content="1999-10-19" scheme="issued"><meta name="DC.relation" content="US:3846402" scheme="references"><meta name="DC.relation" content="US:4362876" scheme="references"><meta name="DC.relation" content="US:4661450" scheme="references"><meta name="DC.relation" content="US:4876187" scheme="references"><meta name="DC.relation" content="US:5011769" scheme="references"><meta name="DC.relation" content="US:5149796" scheme="references"><meta name="DC.relation" content="US:5218103" scheme="references"><meta name="DC.relation" content="US:5264562" scheme="references"><meta name="DC.relation" content="US:5272250" scheme="references"><meta name="DC.relation" content="US:5476925" scheme="references"><meta name="DC.relation" content="US:5625050" scheme="references"><meta name="DC.relation" content="US:5652355" scheme="references"><meta name="DC.relation" content="US:5750341" scheme="references"><meta name="DC.relation" content="WO:1991008313:A1" scheme="references"><meta name="DC.relation" content="WO:1995004748:A1" scheme="references"><meta name="citation_reference" content="Atabekov et al., FEBS Letters 232(1) : 96 98 (1988)."><meta name="citation_reference" content="Atabekov et al., FEBS Letters 232(1) : 96-98 (1988)."><meta name="citation_reference" content="Bannwarth W., Helvetica Chimica Acta 71 : 1517 1527 (1988)."><meta name="citation_reference" content="Bannwarth W., Helvetica Chimica Acta 71 : 1517-1527 (1988)."><meta name="citation_reference" content="Grayaznov et al., J. of the American Chemical Society 116 (7) : 3143 3144 (1994)."><meta name="citation_reference" content="Grayaznov et al., J. of the American Chemical Society 116 (7) : 3143-3144 (1994)."><meta name="citation_reference" content="Gryaznov et al., Nucleic Acids Research 20(13) : 3403 3409 (1992)."><meta name="citation_reference" content="Gryaznov et al., Nucleic Acids Research 20(13) : 3403-3409 (1992)."><meta name="citation_reference" content="Inoue et al., FEBS Letters 215(2) :327 330 (1987)."><meta name="citation_reference" content="Inoue et al., FEBS Letters 215(2) :327-330 (1987)."><meta name="citation_reference" content="Kempe et al., Nucleic Acids Research 10(21) :6695 6714 (1982)."><meta name="citation_reference" content="Kempe et al., Nucleic Acids Research 10(21) :6695-6714 (1982)."><meta name="citation_reference" content="Lavina et al., Biochemie 75 : 25 27 (1993)."><meta name="citation_reference" content="Lavina et al., Biochemie 75 : 25-27 (1993)."><meta name="citation_reference" content="Perreault et al., Nature 344 :565 567 (1990)."><meta name="citation_reference" content="Perreault et al., Nature 344 :565-567 (1990)."><meta name="citation_patent_number" content="US:5969119"><meta name="citation_patent_application_number" content="US:08/872,446"><link rel="canonical" href="http://www.google.com/patents/US5969119"/><meta property="og:url" content="http://www.google.com/patents/US5969119"/><meta name="title" content="Patent US5969119 - Sequencing based on repeated cycles of duplex extension along a single stranded template; oligonucleotide probes having a blocking moiety in a terminal position so that only a single extension of the extended duplex occurs in a single cycle"/><meta name="description" content="Disclosed are oligonucleotides probes which are useful for determining a sequence of nucleotides in a target ploynucleotide. In one embodiment, the probe is of the formula: HO-(3&#39;)(B)j(5&#39;)-OP(═O-)NH-(B)k-Bt*, wherein B is a nucleotide or an analog thereof; j is in range of from 1 to 12; is in the range of from 0 to 12, such that the sum of j and k is less than or equal to 12; and Bt* is a labeled chain-terminating moiety. In s second embodiment, the probe has a formula selected from the group consisting of (1) OP(═O)(O--)O--(5═)(B)sRRRR(B)wBt*, and (2) OP(═O)(O--)(5&#39;)(B)sRRRR(B)w(3&#39;)OP(═O)(O--)O, wherein: is a deoxyribonucleotide or an analog thereof; R is a ribonucleotide; s is in the range of from 1 to 8; w is in the range of from 0 to 8, such that the sum of j and k is less than or equal to 8; and Bt* is a labeled chain-terminating moiety."/><meta property="og:title" content="Patent US5969119 - Sequencing based on repeated cycles of duplex extension along a single stranded template; oligonucleotide probes having a blocking moiety in a terminal position so that only a single extension of the extended duplex occurs in a single cycle"/><meta property="og:type" content="book"/><meta property="og:site_name" content="Google Books"/><meta property="og:image" content="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><link rel="image_src" href="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><script>(function(){try{var aa=function(a,b,c,d){d=d||{};d._sn=["cfg",b,c].join(".");window.gbar.logger.ml(a,d)};var g=window.gbar=window.gbar||{},l=window.gbar.i=window.gbar.i||{},m={},n;function _tvn(a,b){var c=parseInt(a,10);return isNaN(c)?b:c}function _tvf(a,b){var c=parseFloat(a);return isNaN(c)?b:c}function _tvv(a){return!!a}function p(a,b,c){(c||g)[a]=b}g.bv={n:_tvn("2",0),r:"",f:".67.",e:"0",m:_tvn("0",1)};
function q(a,b,c){var d="on"+b;if(a.addEventListener)a.addEventListener(b,c,!1);else if(a.attachEvent)a.attachEvent(d,c);else{var f=a[d];a[d]=function(){var a=f.apply(this,arguments),b=c.apply(this,arguments);return void 0==a?b:void 0==b?a:b&&a}}}var s=function(a){return function(){return g.bv.m==a}},ba=s(1),ca=s(2);p("sb",ba);p("kn",ca);l.a=_tvv;l.b=_tvf;l.c=_tvn;l.i=aa;var da=window.gbar.i.i;var t,u,v,w;function ea(a){v=a}function fa(a){var b;if(b=v&&window.encodeURIComponent)b=a.href,b=!b.match(/^http[s]?:\/\/accounts\.google\.[^/]*\/ClearSID/i)&&!b.match(/^http[s]?:\/\/[^/]*\/accounts\/ClearSID/i);if(b=b&&encodeURIComponent(v()))a.href=a.href.replace(/([?&]continue=)[^&]*/,"$1"+b)}function ga(a){window.gApplication&&(a.href=window.gApplication.getTabUrl(a.href))}
function ha(a){var b=document.forms[0].q,c=window.encodeURIComponent&&b&&b.value,b=b&&b.placeholder;c&&c!=b&&(a.href=a.href.replace(/([?&])q=[^&]*|$/,function(a,b){return(b||"&")+"q="+encodeURIComponent(c)}))}n=l.a("")?ga:ha;
function x(a,b,c,d,f,e){var h=document.getElementById(a);if(h){var k=h.style;k.left=d?"auto":b+"px";k.right=d?b+"px":"auto";k.top=c+"px";k.visibility=u?"hidden":"visible";f&&e?(k.width=f+"px",k.height=e+"px"):(x(t,b,c,d,h.offsetWidth,h.offsetHeight),u=u?"":a)}}
var y=[],ia=function(a,b){y.push(b)},ja=function(a){a=a||window.event;var b=a.target||a.srcElement;a.cancelBubble=!0;null==t&&(a=document.createElement(Array.every||window.createPopup?"iframe":"div"),a.frameBorder="0",t=a.id="gbs",a.src="javascript:''",b.parentNode.appendChild(a),q(document,"click",z));var c=b,b=0;"gb3"!=c.className&&(c=c.parentNode);a=c.getAttribute("aria-owns")||"gbi";var d=c.offsetWidth,f=20<c.offsetTop?46:24;document.getElementById("tphdr")&&(f-=3);var e=!1;do b+=c.offsetLeft||
0;while(c=c.offsetParent);var c=(document.documentElement.clientWidth||document.body.clientWidth)-b-d,h,d=document.body,k=document.defaultView;k&&k.getComputedStyle?(d=k.getComputedStyle(d,""))&&(h=d.direction):h=d.currentStyle?d.currentStyle.direction:d.style.direction;h="rtl"==h;if("gbi"==a){for(d=0;k=y[d++];)k();A(null,window.navExtra);h&&(b=c,e=!0)}else h||(b=c,e=!0);u!=a&&z();x(a,b,f,e)},z=function(){u&&x(u,0,0)},A=function(a,b){var c,d=document.getElementById("gbi"),f=a;f||(f=d.firstChild);
for(;b&&(c=b.pop());){var e=d,h=c,k=f;w||(w="gb2");e.insertBefore(h,k).className=w}},ka=function(a,b,c){if((b=document.getElementById(b))&&a){a.className="gb4";var d=document.createElement("span");d.appendChild(a);d.appendChild(document.createTextNode(" | "));d.id=c;b.appendChild(d)}},la=function(){return document.getElementById("gb_70")},ma=function(){return!!u};p("qs",n);p("setContinueCb",ea);p("pc",fa);p("tg",ja);p("close",z);p("addLink",ka);p("almm",A);p("si",la);p("adh",ia);p("op",ma);var B=function(){},C=function(){},F=function(a){var b=new Image,c=D;b.onerror=b.onload=b.onabort=function(){try{delete E[c]}catch(a){}};E[c]=b;b.src=a;D=c+1},E=[],D=0;p("logger",{il:C,ml:B,log:F});var G=window.gbar.logger;var H={},na={},I=[],oa=l.b("0.1",.1),pa=l.a("1",!0),qa=function(a,b){I.push([a,b])},ra=function(a,b){H[a]=b},sa=function(a){return a in H},J={},K=function(a,b){J[a]||(J[a]=[]);J[a].push(b)},ta=function(a){K("m",a)},L=function(a,b){var c=document.createElement("script");c.src=a;c.async=pa;Math.random()<oa&&(c.onerror=function(){c.onerror=null;B(Error("Bundle load failed: name="+(b||"UNK")+" url="+a))});(document.getElementById("xjsc")||document.getElementsByTagName("body")[0]||
document.getElementsByTagName("head")[0]).appendChild(c)},N=function(a){for(var b=0,c;(c=I[b])&&c[0]!=a;++b);!c||c[1].l||c[1].s||(c[1].s=!0,M(2,a),c[1].url&&L(c[1].url,a),c[1].libs&&m.d&&m.d(c[1].libs))},O=function(a){K("gc",a)},P=null,ua=function(a){P=a},M=function(a,b,c){if(P){a={t:a,b:b};if(c)for(var d in c)a[d]=c[d];try{P(a)}catch(f){}}};p("mdc",H);p("mdi",na);p("bnc",I);p("qGC",O);p("qm",ta);p("qd",J);p("lb",N);p("mcf",ra);p("bcf",qa);p("aq",K);p("mdd","");p("has",sa);
p("trh",ua);p("tev",M);var Q=l.b("0.1",.001),R=0;
function _mlToken(a,b){try{if(1>R){R++;var c,d=a,f=b||{},e=encodeURIComponent,h=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&jexpid=",e("17483"),"&srcpg=",e("prop=22"),"&jsr=",Math.round(1/Q),"&ogev=",e("C5rtU4G2C5KGogTDxoGgDw"),"&ogf=",g.bv.f,"&ogrp=",e("1"),"&ogv=",e("1407723702.0"),"&oggv="+e("es_plusone_gc_20140723.0_p0"),"&ogd=",e("com"),"&ogc=",e("JPN"),"&ogl=",e("en")];f._sn&&(f._sn="og."+
f._sn);for(var k in f)h.push("&"),h.push(e(k)),h.push("="),h.push(e(f[k]));h.push("&emsg=");h.push(e(d.name+":"+d.message));var r=h.join("");S(r)&&(r=r.substr(0,2E3));c=r;var Aa=window.gbar.logger._aem(a,c);F(Aa)}}catch(Na){}}var S=function(a){return 2E3<=a.length},va=function(a,b){return b};function T(a){B=a;p("_itl",S,G);p("_aem",va,G);p("ml",B,G);a={};H.er=a}l.a("")?T(function(a){throw a;}):l.a("1")&&Math.random()<Q&&T(_mlToken);I.push(["m",{url:"//ssl.gstatic.com/gb/js/scm_7385cc5883250b43a39405734c1bea59.js"}]);g.mcf("c",{});g.sg={c:""};if(l.a("1")){var wa=l.a("");I.push(["gc",{auto:wa,url:"//ssl.gstatic.com/gb/js/abc/gci_91f30755d6a6b787dcc2a4062e6e9824.js",libs:"googleapis.client:plusone:gapi.iframes"}]);var xa={version:"gci_91f30755d6a6b787dcc2a4062e6e9824.js",index:"",lang:"en"};H.gc=xa;var U=function(a){window.googleapis&&window.iframes?a&&a():(a&&O(a),N("gc"))};p("lGC",U);l.a("1")&&p("lPWF",U)};window.__PVT="";if(l.a("1")&&l.a("1")){var V=function(a){U(function(){K("pw",a);N("pw")})};p("lPW",V);I.push(["pw",{url:"//ssl.gstatic.com/gb/js/abc/pwm_45f73e4df07a0e388b0fa1f3d30e7280.js"}]);var W=[],ya=function(a){W[0]=a},za=function(a,b){var c=b||{};c._sn="pw";B(a,c)},Ba={signed:W,elog:za,base:"https://plusone.google.com/u/0",loadTime:(new Date).getTime()};H.pw=Ba;var X=function(a,b){for(var c=b.split("."),d=function(){var b=arguments;a(function(){for(var a=g,d=0,e=c.length-1;d<e;++d)a=a[c[d]];a[c[d]].apply(a,b)})},f=g,e=0,h=c.length-1;e<h;++e)f=
f[c[e]]=f[c[e]]||{};return f[c[e]]=d};X(V,"pw.clk");X(V,"pw.hvr");p("su",ya,g.pw)};function Ca(){function a(){for(var b;(b=e[h++])&&"m"!=b[0]&&!b[1].auto;);b&&(M(2,b[0]),b[1].url&&L(b[1].url,b[0]),b[1].libs&&m.d&&m.d(b[1].libs));h<e.length&&setTimeout(a,0)}function b(){0<f--?setTimeout(b,0):a()}var c=l.a("1"),d=l.a(""),f=3,e=I,h=0,k=window.gbarOnReady;if(k)try{k()}catch(r){da(r,"ml","or")}d?p("ldb",a):c?q(window,"load",b):b()}p("rdl",Ca);var Da={D:1,H:2,da:3,p:4,W:5,M:6,F:7,g:8,ha:9,U:10,L:11,T:12,S:13,N:14,Q:15,P:16,fa:17,w:18,O:19,ga:20,ea:21,u:22,G:23,ja:24,ka:25,ia:26,A:27,j:28,o:29,k:30,ca:31,Z:32,$:33,J:34,K:35,ba:36,aa:37,Y:38,B:39,R:40,v:41,X:42,V:43,h:48,C:49,I:500},Y=[1,2,3,4,5,6,9,10,11,13,14,28,29,30,34,35,37,38,39,40,41,42,43,48,49,500];var Z=l.b("0.001",1E-4),Ea=l.b("1",1),Fa=!1,Ga=!1;if(l.a("1")){var Ha=Math.random();Ha<=Z&&(Fa=!0);Ha<=Ea&&(Ga=!0)}var Ia=Da,$=null;function Ja(){var a=0,b=function(b,d){l.a(d)&&(a|=b)};b(1,"");b(2,"");b(4,"");b(8,"");return a}
function Ka(a,b){var c=Z,d=Fa,f;f=a;if(!$){$={};for(var e=0;e<Y.length;e++){var h=Y[e];$[h]=!0}}if(f=!!$[f])c=Ea,d=Ga;if(d){d=encodeURIComponent;g.rp?(f=g.rp(),f="-1"!=f?f:"1"):f="1";c=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&oge=",a,"&ogex=",d("17483"),"&ogev=",d("C5rtU4G2C5KGogTDxoGgDw"),"&ogf=",g.bv.f,"&ogp=",d("22"),"&ogrp=",d(f),"&ogsr=",Math.round(1/c),"&ogv=",d("1407723702.0"),"&oggv="+
d("es_plusone_gc_20140723.0_p0"),"&ogd=",d("com"),"&ogl=",d("en"),"&ogc=",d("JPN"),"&ogus=",Ja()];if(b){"ogw"in b&&(c.push("&ogw="+b.ogw),delete b.ogw);var k;f=b;e=[];for(k in f)0!=e.length&&e.push(","),e.push(La(k)),e.push("."),e.push(La(f[k]));k=e.join("");""!=k&&(c.push("&ogad="),c.push(d(k)))}F(c.join(""))}}function La(a){"number"==typeof a&&(a+="");return"string"==typeof a?a.replace(".","%2E").replace(",","%2C"):a}C=Ka;p("il",C,G);var Ma={};H.il=Ma;setTimeout(function(){C(Ia.g)},0);}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var b=window.gbar.i.i;var c=window.gbar;var f=function(d){try{var a=document.getElementById("gbom");a&&d.appendChild(a.cloneNode(!0))}catch(e){b(e,"omas","aomc")}};c.aomc=f;}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var a=window.gbar;a.mcf("pm",{p:""});}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{window.gbar.rdl();}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
if (window['_OC_timingAction']) {window['_OC_timingAction']('patents_refpage');}</script><style>#gbar,#guser{font-size:13px;padding-top:1px !important;}#gbar{float:left;height:22px}#guser{padding-bottom:7px !important;text-align:right}.gbh,.gbd{border-top:1px solid #c9d7f1;font-size:1px}.gbh{height:0;position:absolute;top:24px;width:100%}#gbs,.gbm{background:#fff;left:0;position:absolute;text-align:left;visibility:hidden;z-index:1000}.gbm{border:1px solid;border-color:#c9d7f1 #36c #36c #a2bae7;z-index:1001}.gb1{margin-right:.5em}.gb1,.gb3{zoom:1}.gb2{display:block;padding:.2em .5em}.gb2,.gb3{text-decoration:none !important;border-bottom:none}a.gb1,a.gb4{text-decoration:underline !important}a.gb1,a.gb2,a.gb3,a.gb4{color:#00c !important}.gbi .gb3,.gbi .gb2,.gbi .gb4{color:#dd8e27 !important}.gbf .gb3,.gbf .gb2,.gbf .gb4{color:#900 !important}a.gb2:hover{background:#36c;color:#fff !important}#gbar .gbz0l{color:#000 !important;cursor:default;font-weight:bold;text-decoration:none !important}
#gbar { padding:.3em .6em !important;}</style></head><body  topmargin="3" marginheight="3"><div id=gbar><nobr><a onclick=gbar.qs(this);gbar.logger.il(1,{t:1}); class=gb1 id=gb_1 href="https://www.google.com/search?sa=N&tab=tw">Search</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:2}); class=gb1 id=gb_2 href="http://www.google.com/search?hl=en&tbm=isch&source=og&sa=N&tab=ti">Images</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:8}); class=gb1 id=gb_8 href="http://maps.google.com/maps?hl=en&sa=N&tab=tl">Maps</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:78}); class=gb1 id=gb_78 href="https://play.google.com/?hl=en&sa=N&tab=t8">Play</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:36}); class=gb1 id=gb_36 href="http://www.youtube.com/results?sa=N&tab=t1">YouTube</a> <a onclick=gbar.logger.il(1,{t:5}); class=gb1 id=gb_5 href="http://news.google.com/nwshp?hl=en&tab=tn">News</a> <a onclick=gbar.logger.il(1,{t:23}); class=gb1 id=gb_23 href="https://mail.google.com/mail/?tab=tm">Gmail</a> <a onclick=gbar.logger.il(1,{t:25}); class=gb1 id=gb_25 href="https://drive.google.com/?tab=to">Drive</a> <a class=gb3 href="http://www.google.com/intl/en/options/" onclick="this.blur();gbar.tg(event);return !1" aria-haspopup=true><u>More</u> <small>&#9660;</small></a><div class=gbm id=gbi><a onclick=gbar.logger.il(1,{t:24}); class=gb2 id=gb_24 href="https://www.google.com/calendar?tab=tc">Calendar</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:51}); class=gb2 id=gb_51 href="http://translate.google.com/?hl=en&sa=N&tab=tT">Translate</a><a onclick=gbar.logger.il(1,{t:17}); class=gb2 id=gb_17 href="http://www.google.com/mobile/?hl=en&tab=tD">Mobile</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:10}); class=gb2 id=gb_10 href="http://www.google.com/search?hl=en&tbo=u&tbm=bks&source=og&sa=N&tab=tp">Books</a><a onclick=gbar.logger.il(1,{t:212}); class=gb2 id=gb_212 href="https://wallet.google.com/manage/?tab=ta">Wallet</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:6}); class=gb2 id=gb_6 href="http://www.google.com/search?hl=en&tbo=u&tbm=shop&source=og&sa=N&tab=tf">Shopping</a><a onclick=gbar.logger.il(1,{t:30}); class=gb2 id=gb_30 href="http://www.blogger.com/?tab=tj">Blogger</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:27}); class=gb2 id=gb_27 href="http://www.google.com/finance?sa=N&tab=te">Finance</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:31}); class=gb2 id=gb_31 href="https://plus.google.com/photos?sa=N&tab=tq">Photos</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:12}); class=gb2 id=gb_12 href="http://www.google.com/search?hl=en&tbo=u&tbm=vid&source=og&sa=N&tab=tv">Videos</a><div class=gb2><div class=gbd></div></div><a onclick=gbar.logger.il(1,{t:66}); href="http://www.google.com/intl/en/options/" class=gb2>Even more &raquo;</a></div></nobr></div><div id=guser width=100%><nobr><span id=gbn class=gbi></span><span id=gbf class=gbf></span><span id=gbe></span><a target=_top id=gb_70 href="https://www.google.com/accounts/Login?service=&continue=http://www.google.com/patents%3Fhl%3Den&hl=en" class=gb4>Sign in</a><div style="display: none"><div class=gbm id=gbd5 aria-owner=gbg5><div class=gbmc><ol id=gbom class=gbmcc></ol></div></div></div></nobr></div><div class=gbh style=left:0></div><div class=gbh style=right:0></div><div role="alert" style="position: absolute; left: 0; right: 0;"><a href="http://www.google.com/patents/us5969119?hl=en&amp;output=html_text" title="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."><img border="0" src="http://www.google.com/images/cleardot.gif"alt="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."></a></div><div id="guser"><nobr></nobr></div><div style="clear:both;"></div><div id="gb-top-search-box" class="gb-top-search-box-small gb-reset"><table><tr><td class="logo"><a href="http://www.google.com/patents" class="logo-link"><img class="logo-img" src="/intl/en/images/logos/google_logo_41.png" alt="Go to Google Books Home" height="41"/></a></td><td><form action="http://www.google.com/search" name="f" id="vheadf" method="get"><span id="hf"></span><input type="hidden" name="tbm" value="pts"/><input type="hidden" name="tbo" value="1"/><input type="hidden" name="hl" value="en"/><table><tr><td><div class="inputs"><table><tr><td><div class="text-input"><input type="text" name="q" id="vheadq" class="text" maxlength="2048" size="31" value="" title="Search Patents" accesskey="s" autocomplete="off"/><script>window._OC_autoDir &&window._OC_autoDir('vheadq', 'tia-vheadq');</script></div></td><td><div class="submit-input"><input name="btnG" class="submit" type="submit" value=""/></div></td></tr></table></div></td><td class="col-ext-links"><div class="ext-links"><a href="http://www.google.com/advanced_patent_search">&lt;nobr&gt;Advanced Patent Search&lt;/nobr&gt;</a></div></td></tr></table></form></td></tr></table></div><div class="kd-appbar"><h2 class="kd-appname"><a href="/patents">Patents</a></h2><div class="kd-buttonbar left" id="left-toolbar-buttons"><a id="appbar-write-review-link" href=""></a><a id="appbar-view-print-sample-link" href=""></a><a id="appbar-view-ebook-sample-link" href=""></a><a id="appbar-patents-prior-art-finder-link" href="https://www.google.com/patents/related/US5969119"></a><a id="appbar-patents-discuss-this-link" href="http://www.google.com/url?id=Bp9MBAABERAJ&amp;q=http://patents.stackexchange.com/redirect/google-patents%3Fpatent%3DUS5969119&amp;usg=AFQjCNFfLIZaKaOwHmExqVxtpXVGz1jDKQ" data-is-grant="true"></a><a id="appbar-read-patent-link" href="//docs.google.com/viewer?url=patentimages.storage.googleapis.com/pdfs/US5969119.pdf"></a><a id="appbar-download-pdf-link" href="//patentimages.storage.googleapis.com/pdfs/US5969119.pdf"></a></div><div class="kd-buttonbar right" id="right-toolbar-buttons"></div></div><div id="books-microdata" itemscope=""itemtype="http://schema.org/Book"itemid="http://www.google.com/patents/US5969119" style="display:none"><span itemprop="description">Disclosed are oligonucleotides probes which are useful for determining a sequence of nucleotides in a target ploynucleotide. In one embodiment, the probe is of the formula: HO-(3&#39;)(B)j(5&#39;)-OP(═O-)NH-(B)k-Bt*, wherein B is a nucleotide or an analog thereof; j is in range of from 1 to 12; is in the range...</span><span itemprop="url">http://www.google.com/patents/US5969119?utm_source=gb-gplus-share</span><span class="main-title" itemprop="name">Patent US5969119 - Sequencing based on repeated cycles of duplex extension along a single stranded template; oligonucleotide probes having a blocking moiety in a terminal position so that only a single extension of the extended duplex occurs in a single cycle</span><img itemprop="image" src="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"alt="Patent US5969119 - Sequencing based on repeated cycles of duplex extension along a single stranded template; oligonucleotide probes having a blocking moiety in a terminal position so that only a single extension of the extended duplex occurs in a single cycle" title="Patent US5969119 - Sequencing based on repeated cycles of duplex extension along a single stranded template; oligonucleotide probes having a blocking moiety in a terminal position so that only a single extension of the extended duplex occurs in a single cycle"></div><div style="display: none"><ol id="ofe-gear-menu-contents" class="gbmcc"><li class="gbe gbmtc"><a class="gbmt goog-menuitem-content" id="" href="http://www.google.com/advanced_patent_search">Advanced Patent Search</a></li></ol></div><table id="viewport_table" cellpadding="0" style="clear:both" cellspacing="0"><tr><td id="viewport_td"><div class=vertical_module_list_row><div id=intl_patents class=about_content><div id=intl_patents_v><table class="patent-bibdata"><tr><td class="patent-bibdata-heading">Publication number</td><td class="single-patent-bibdata">US5969119 A</td></tr><tr><td class="patent-bibdata-heading">Publication type</td><td class="single-patent-bibdata">Grant</td></tr><tr><td class="patent-bibdata-heading">Application number</td><td class="single-patent-bibdata">US 08/872,446</td></tr><tr><td class="patent-bibdata-heading">Publication date</td><td class="single-patent-bibdata">Oct 19, 1999</td></tr><tr><td class="patent-bibdata-heading">Filing date</td><td class="single-patent-bibdata">Jun 10, 1997</td></tr><tr><td class="patent-bibdata-heading">Priority date<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The priority date is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the date listed."></span></td><td class="single-patent-bibdata">Apr 17, 1995</td></tr><tr><td class="patent-bibdata-heading">Fee status<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The fee status is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the status or dates listed."></span></td><td class="single-patent-bibdata">Paid</td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Also published as</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/CA2218017A1">CA2218017A1</a>, </span><span class="patent-bibdata-value"><a href="/patents/CA2218017C">CA2218017C</a>, </span><span class="patent-bibdata-value"><a href="/patents/DE10184524T1">DE10184524T1</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP0871646A1">EP0871646A1</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP0871646A4">EP0871646A4</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP0871646B1">EP0871646B1</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP2298786A1">EP2298786A1</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP2298787A1">EP2298787A1</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP2433949A1">EP2433949A1</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP2433950A1">EP2433950A1</a>, </span><span class="patent-bibdata-value"><a href="/patents/US5750341">US5750341</a>, </span><span class="patent-bibdata-value"><a href="/patents/US6306597">US6306597</a>, </span><span class="patent-bibdata-value"><a href="/patents/WO1996033205A1">WO1996033205A1</a></span></span></td></tr><tr class="patent-bibdata-list-row alternate-patent-number"><td class="patent-bibdata-heading">Publication number</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value">08872446, </span><span class="patent-bibdata-value">872446, </span><span class="patent-bibdata-value">US 5969119 A, </span><span class="patent-bibdata-value">US 5969119A, </span><span class="patent-bibdata-value">US-A-5969119, </span><span class="patent-bibdata-value">US5969119 A, </span><span class="patent-bibdata-value">US5969119A</span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Inventors</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Stephen+C.+Macevicz%22">Stephen C. Macevicz</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Original Assignee</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=inassignee:%22Lynx+Therapeutics,+Inc.%22">Lynx Therapeutics, Inc.</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Export Citation</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/US5969119.bibtex">BiBTeX</a>, </span><span class="patent-bibdata-value"><a href="/patents/US5969119.enw">EndNote</a>, </span><span class="patent-bibdata-value"><a href="/patents/US5969119.ris">RefMan</a></span></span></td></tr><tr class="patent-internal-links"><td colspan=2><span class="patent-bibdata-value"><a href="#backward-citations">Patent Citations</a> (15),</span> <span class="patent-bibdata-value"><a href="#npl-citations">Non-Patent Citations</a> (16),</span> <span class="patent-bibdata-value"><a href="#forward-citations">Referenced by</a> (15),</span> <span class="patent-bibdata-value"><a href="#classifications">Classifications</a> (14),</span> <span class="patent-bibdata-value"><a href="#legal-events">Legal Events</a> (10)</span> </td></tr><tr><td colspan=2 class="patent-bibdata-external-link-spacer-top"></td></tr><tr class="patent-bibdata-external-link-spacer-bottom"></tr><tr><td colspan=2><span class="patent-bibdata-heading">External Links:&nbsp;</span><span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=Bp9MBAABERAJ&q=http://patft.uspto.gov/netacgi/nph-Parser%3FSect2%3DPTO1%26Sect2%3DHITOFF%26p%3D1%26u%3D/netahtml/PTO/search-bool.html%26r%3D1%26f%3DG%26l%3D50%26d%3DPALL%26RefSrch%3Dyes%26Query%3DPN/5969119&usg=AFQjCNEYr-Ux0_fNDUnhIO-HHJLbcBSMAw">USPTO</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=Bp9MBAABERAJ&q=http://assignments.uspto.gov/assignments/q%3Fdb%3Dpat%26pat%3D5969119&usg=AFQjCNGMlu_gOk6mMabbVGLKT-rydCLVIw">USPTO Assignment</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=Bp9MBAABERAJ&q=http://worldwide.espacenet.com/publicationDetails/biblio%3FCC%3DUS%26NR%3D5969119A%26KC%3DA%26FT%3DD&usg=AFQjCNErAARj_5CJWeycYMB134hy3EQ1SQ">Espacenet</a></span></span></td></tr><tr class="patent-bibdata-group-spacer"></tr></table><div class="number-and-title"><span class="patent-title"><invention-title mxw-id="PT129002307" lang="EN" load-source="ifi">Sequencing based on repeated cycles of duplex extension along a single stranded template; oligonucleotide probes having a blocking moiety in a terminal position so that only a single extension of the extended duplex occurs in a single cycle</invention-title></span><br><span class="patent-number">US 5969119 A</span></div><div class="patent-section patent-abstract-section"><div class="patent-section-header"><span class="patent-section-title">Abstract</span></div><div class="patent-text"><abstract mxw-id="PA37975631" lang="EN" load-source="patent-office"> <div class="abstract">Disclosed are oligonucleotides probes which are useful for determining a sequence of nucleotides in a target ploynucleotide. In one embodiment, the probe is of the formula: HO-(3')(B)j(5')-OP(═O-)NH-(B)k-Bt*, wherein B is a nucleotide or an analog thereof; j is in range of from 1 to 12; is in the range of from 0 to 12, such that the sum of j and k is less than or equal to 12; and Bt* is a labeled chain-terminating moiety. In s second embodiment, the probe has a formula selected from the group consisting of (1) OP(═O)(O--)O--(5═)(B)sRRRR(B)wBt*, and (2) OP(═O)(O--)(5')(B)sRRRR(B)w(3')OP(═O)(O--)O, wherein: is a deoxyribonucleotide or an analog thereof; R is a ribonucleotide; s is in the range of from 1 to 8; w is in the range of from 0 to 8, such that the sum of j and k is less than or equal to 8; and Bt* is a labeled chain-terminating moiety.</div>
  </abstract></div></div><div class="patent-section patent-drawings-section"><div class="patent-section-header"><span class="patent-section-title">Images<span class="patent-section-count">(5)</span></span></div><div class="patent-drawings-body"><div class="patent-drawings-carousel"><div class="drawings"><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5969119-1.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5969119-1.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5969119-2.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5969119-2.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5969119-3.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5969119-3.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5969119-4.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5969119-4.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US5969119-5.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US5969119-5.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div></div></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img" alt="Previous page"src="/googlebooks/images/kennedy/page_left.png"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img" alt="Next page"src="/googlebooks/images/kennedy/page_right.png"width="21" height="21" /></div></div></div><div class="patent-post-drawings"></div><div class="patent-section patent-claims-section"><div class="patent-section-header"><span class="patent-section-title">Claims<span class="patent-section-count">(2)</span></span></div><div class="patent-text"><div mxw-id="PCLM5460882" lang="EN" load-source="patent-office" class="claims">
    <claim-statement>I claim:</claim-statement> <div class="claim"> <div num="1" class="claim">
      <div class="claim-text">1. An oligonucleotide probe of the formula:<div class="claim-text"> <pre xml:space="preserve" listing-type="equation">HO--(3')(B)j(5')--OP(═O)(O--)NH--(B)k-Bt*</pre> </div> <div class="claim-text">wherein:</div> <div class="claim-text">B is a nucleotide or an analog thereof;</div> <div class="claim-text">j is in the range of from 1 to 12;</div> <div class="claim-text">k is in the range of from 1 to 12, such that the sum of j and k is less than or equal to 12;</div> <div class="claim-text">Bt* is a labeled, non-extendable chain-terminating moiety.</div> </div>
    </div>
    </div> <div class="claim"> <div num="2" class="claim">
      <div class="claim-text">2. An oligonucleotide probe selected from the group consisting of:<div class="claim-text"> <pre xml:space="preserve" listing-type="equation">OP(═O)(O)O--(5')(B)sRRRR(B)wBt*</pre> </div> <div class="claim-text">and</div> <div class="claim-text"> <pre xml:space="preserve" listing-type="equation">OP(═O)(O--)O--(5')(B)sRRRR(B)w(3')OP(═O)(O--)O</pre> </div> <div class="claim-text">wherein:</div> <div class="claim-text">B is a deoxyribonucleotide or an analog thereof;</div> <div class="claim-text">R is a ribonucleotide;</div> <div class="claim-text">s is in the range of from 1 to 8;</div> <div class="claim-text">w is in the range of from 0 to 8, such that the sum of s and w is less than or equal to 8; and</div> <div class="claim-text">Bt* is a labeled, non-extendable chain-terminating moiety.</div> </div>
    </div>
  </div> </div></div></div><div class="patent-section patent-description-section"><div class="patent-section-header"><span class="patent-section-title">Description</span></div><div class="patent-text"><div mxw-id="PDES67352841" lang="EN" load-source="patent-office" class="description">
    <p>This is a division of application Ser. No. 08/424,663, filed Apr. 17, 1995.</p>
    <heading>FIELD OF THE INVENTION</heading> <p>The invention relates generally to methods for determining the nucleotide sequence of a polynucleotide, and more particularly, to a method of identifying nucleotides in a template by stepwise extension of one or more primers by successive ligations of oligonucleotide blocks.</p>
    <heading>BACKGROUND</heading> <p>Analysis of polynucleotides with currently available techniques provides a spectrum of information ranging from the confirmation that a test polynucleotide is the same or different than a standard or an isolated fragment to the express identification and ordering of each nucleoside of the test polynucleotide. Not only are such techniques crucial for understanding the function and control of genes and for applying many of the basic techniques of molecular biology, but they have also become increasingly important as tools in genomic analysis and a great many non-research applications, such as genetic identification, forensic analysis, genetic counselling, medical diagnostics, and the like. In these latter applications both techniques providing partial sequence information, such as fingerprinting and sequence comparisons, and techniques providing full sequence determination have been employed, e.g. Gibbs et al, Proc. Natl. Acad. Sci., 86: 1919-1923 (1989); Gyllensten et al, Proc. Natl. Acad. Sci, 85: 7652-7656 (1988); Carrano et al, Genomics, 4: 129-136 (1989);Caetano-Anolles et al, Mol. Gen. Genet., 235: 157-165 (1992); Brenner and Livak, Proc. Natl. Acad. Sci., 86: 8902-8906 (1989); Green et al, PCR Methods and Applications, 1: 77-90 (1991); and Versalovic et al, Nucleic Acids Research, 19: 6823-6831 (1991).</p>
    <p>Native DNA consists of two linear polymers, or strands of nucleotides. Each strand is a chain of nucleosides linked by phosphodiester bonds. The two strands are held together in an antiparallel orientation by hydrogen bonds between complementary bases of the nucleotides of the two strands: deoxyadenosine (A) pairs with thymidine (T) and deoxyguanosine (G) pairs with deoxycytidine (C).</p>
    <p>Presently there are two basic approaches to DNA sequence determination: the dideoxy chain termination method, e.g. Sanger et al, Proc. Natl. Acad. Sci., 74: 5463-5467 (1977); and the chemical degradation method, e.g. Maxam et al, Proc. Natl. Acad. Sci., 74: 560-564 (1977). The chain termination method has been improved in several ways, and serves as the basis for all currently available automated DNA sequencing machines, e.g. Sanger et al, J. Mol. Biol., 143: 161-178 (1980); Schreier et al, J. Mol. Biol., 129: 169-172 (1979); Smith et al, Nucleic Acids Research, 13: 2399-2412 (1985); Smith et al, Nature, 321: 674-679 (1987); Prober et al, Science, 238: 336-341 (1987); Section II, Meth. Enzymol., 155: 51-334 (1987); Church et al, Science, 240: 185-188 (1988); Hunkapiller et al, Science, 254: 59-67 (1991); Bevan et al, PCR Methods and Applications, 1: 222-228 (1992).</p>
    <p>Both the chain termination and chemical degradation methods require the generation of one or more sets of labeled DNA fragments, each having a common origin and each terminating with a known base. The set or sets of fragments must then be separated by size to obtain sequence information. In both methods, the DNA fragments are separated by high resolution gel electrophoresis, which must have the capacity of distinguishing very large fragments differing in size by no more than a single nucleotide. Unfortunately, this step severely limits the size of the DNA chain that can be sequenced at one time. Sequencing using these techniques can reliably accommodate a DNA chain of up to about 400-450 nucleotides, Bankier et al, Meth. Enzymol., 155: 51-93 (1987); and Hawkins et al, Electrophoresis, 13: 552-559 (1992).</p>
    <p>Several significant technical problems have seriously impeded the application of such techniques to the sequencing of long target polynucleotides, e.g. in excess of 500-600 nucleotides, or to the sequencing of high volumes of many target polynucleotides. Such problems include i) the gel electrophoretic separation step which is labor intensive, is difficult to automate, and introduces an extra degree of variability in the analysis of data, e.g. band broadening due to temperature effects, compressions due to secondary structure in the DNA sequencing fragments, inhomogeneities in the separation gel, and the like; ii) nucleic acid polymerases whose properties, such as processivity, fidelity, rate of polymerization, rate of incorporation of chain terminators, and the like, are often sequence dependent; iii) detection and analysis of DNA sequencing fragments which are typically present in fmol quantities in spacially overlapping bands in a gel; iv) lower signals because the labelling moiety is distributed over the many hundred spacially separated bands rather than being concentrated in a single homogeneous phase, and v) in the case of single-lane fluorescence detection, the availability of dyes with suitable emission and absorption properties, quantum yield, and spectral resolvability, e.g. Trainor, Anal. Biochem., 62: 418-426 (1990); Connell et al, Biotechniques, 5: 342-348 (1987); Karger et al, Nucleic Acids Research, 19: 4955-4962 (1991); Fung et al, U.S. Pat. No. 4,855,225; and Nishikawa et al, Electrophoresis, 12: 623-631 (1991).</p>
    <p>Another problem exists with current technology in the area of diagnostic sequencing. An ever widening array of disorders, susceptibilities to disorders, prognoses of disease conditions, and the like, have been correlated with the presence of particular DNA sequences, or the degree of variation (or mutation) in DNA sequences, at one or more genetic loci. Examples of such phenomena include human leukocyte antigen (HLA) typing, cystic fibrosis, tumor progression and heterogeneity, p53 proto-oncogene mutations, ras proto-oncogene mutations, and the like, e.g. Gyllensten et al, PCR Methods and Applications, 1: 91-98 (1991); Santamaria et al, International application PCT/US92/01675; Tsui et al, International application PCT/CA90/00267; and the like. A difficulty in determining DNA sequences associated with such conditions to obtain diagnostic or prognostic information is the frequent presence of multiple subpopulations of DNA, e.g. allelic variants, multiple mutant forms, and the like. Distinguishing the presence and identity of multiple sequences with current sequencing technology is virtually impossible, without additional work to isolate and perhaps clone the separate species of DNA.</p>
    <p>A major advance in sequencing technology could be made if an alternative approach was available for sequencing DNA that did not require high resolution electrophoretic separations of DNA fragments, that generated signals more amenable to analysis, and that provided a means for readily analyzing DNA from heterozygous genetic loci.</p>
    <p>An objective of the invention is to provide such an alternative approach to presently available DNA sequencing technologies.</p>
    <heading>SUMMARY OF THE INVENTION</heading> <p>The invention provides a method of nucleic acid sequence analysis based on repeated cycles of duplex extension along a single stranded template. Preferably, such extension starts from a duplex formed between an initializing oligonucleotide and the template. The initializing oligonucleotide is extended in an initial extension cycle by ligating an oligonucleotide probe to its end to form an extended duplex. The extended duplex is then repeatedly extended by subsequent cycles of ligation. During each cycle, the identity of one or more nucleotides in the template is determined by a label on, or associated with, a successfully ligated oligonucleotide probe. Preferably, the oligonucleotide probe has a blocking moiety, e.g. a chain-terminating nucleotide, in a terminal position so that only a single extension of the extended duplex takes place in a single cycle. The duplex is further extended in subsequent cycles by removing the blocking moiety and regenerating an extendable terminus.</p>
    <p>In one aspect of the invention, a plurality of different initializing oligonucleotides is provided for separate samples of the template. Each initializing oligonucleotide forms a duplex with the template such that the end undergoing extension is one or more nucleotides out of register, or phase, with that of every other initializing oligonucleotide of the plurality. In other words, the starting nucleotide for extension is different by one or more nucleotides for each of the different initializing oligonucleotides. In this manner, after each cycle of extension with oligonucleotide probes of the same length, the same relative phase exists between the ends of the initializing oligonucleotides on the different templates. Thus, in a preferred embodiment, where, for example, i) the initializing oligonucleotides are out of phase by one nucleotide, ii) 9-mer oligonucleotide probes are used in the extension step, and iii) nine different initializing oligonucleotides are employed, nine template nucleotides will be identified simultaneously in each extension cycle.</p>
    <p>The invention also includes an oligonucleotide probe of the formula: HO--(3')(B)j(5')--OP(═O)(O--)NH--(B)k-Bt* wherein B is a nucleotide or an analog thereof; j is in the range of from 1 to 12; k is in the range of from 0 to 12, such that the sum of j and k is less than or equal to 12; and Bt* is a labeled, non-extendable chain-terminating moiety.</p>
    <p>In another aspect, the invention includes an oligonucleotide selected from OP(═O)(O--)O--(5')(B)sRRRR(B)wBt* and OP(═O)(O--)O--(5')(B)sRRRR(B)w(3')OP(═O)(O--)O, wherein B is a deoxyribonucleotide or an analog thereof; R is a ribonucleotide; s is in the range of from 1 to 8; w is in the range of from 0 to 8, such that the sum of s and w is less than or equal to 8; and Bt* is a labeled, non-extendable chain-terminating moiety.</p>
    <heading>BRIEF DESCRIPTION OF THE DRAWINGS</heading> <p>FIG. 1 diagrammatically illustrates parallel extensions of multiple templates in accordance with the invention.</p>
    <p>FIG. 2 diagrammatically illustrates an embodiment of the invention employing acid-labile linkages.</p>
    <p>FIG. 3A diagrammatically illustrates an embodiment of the invention employing RNase H labile oligonucleotides with 3'→5' extensions.</p>
    <p>FIG. 3B diagrammatically illustrates an embodiment of the invention employing RNase H labile oligonucleotides with 5'→3' extensions.</p>
    <p>FIG. 4 diagrammatically illustrates an embodiment of the invention employing ligation followed by polymerase extension and cleavage.</p>
    <heading>DEFINITIONS</heading> <p>As used herein "sequence determination," "determining a nucleotide sequence," "sequencing," and like terms, in reference to polynucleotides includes determination of partial as well as full sequence information of the polynucleotide. That is, the term includes sequence comparisons, fingerprinting, and like levels of information about a target polynucleotide, as well as the express identification and ordering of each nucleoside of the test polynucleotide.</p>
    <p>"Perfectly matched duplex" in reference to the protruding stands of probes and target polynucleotides means that the protruding strand from one forms a double stranded structure with the other such that each nucleotide in the double stranded structure undergoes Watson-Crick basepairing with a nucleotide on the opposite strand. The term also comprehends the pairing of nucleoside analogs, such as deoxyinosine, nucleosides with 2-aminopurine bases, and the like, that may be employed to reduce the degeneracy of the probes.</p>
    <p>The term "oligonucleotide" as used herein includes linear oligomers of nucleosides or analogs thereof, including deoxyribonucleosides, ribonucleosides, and the like. Usually oligonucleotides range in size from a few monomeric units, e.g. 3-4, to several hundreds of monomeric units. Whenever an oligonucleotide is represented by a sequence of letters, such as "ATGCCTG," it will be understood that the nucleotides are in 5'→3' order from left to right and that "A" denotes deoxyadenosine, "C" denotes deoxycytidine, "G" denotes deoxyguanosine, and "T" denotes thymidine, unless otherwise noted.</p>
    <p>As used herein, "nucleoside" includes the natural nucleosides, including 2'-deoxy and 2'-hydroxyl forms, e.g. as described in Kornberg and Baker, DNA Replication, 2nd Ed. (Freeman, San Francisco, 1992). "Analogs" in reference to nucleosides includes synthetic nucleosides having modified base moieties and/or modified sugar moieties, e.g. described generally by Scheit, Nucleotide Analogs (John Wiley, New York, 1980). Such analogs include synthetic nucleosides designed to enhance binding properties, reduce degeneracy, increase specificity, and the like.</p>
    <p>As used herein, "ligation" means to form a covalent bond or linkage between the termini of two or more nucleic acids, e.g. oligonucleotides and/or polynucleotides, in template-driven reaction. The nature of the bond or linkage may vary widely and the ligation may be carried out enzymatically or chemically.</p>
    <heading>Detailed Description of the Invention</heading> <p>The invention provides a method of sequencing nucleic acids which obviates electrophoretic separation of similarly sized DNA fragments, and which eliminates the difficulties associated with the detection and analysis of spacially overlapping bands of DNA fragments in gel or like medium. The invention also obviates the need to generate DNA fragments from long single stranded templates with a DNA polymerase.</p>
    <p>The general scheme of one aspect of the invention is shown diagrammatically in FIG. 1. As described more fully below, the invention is not meant to be limited by the particular features of this embodiment. Template (20) comprising polynucleotide (50) of unknown sequence and binding region (40) is attached to solid phase support (10). Preferably, for embodiments employing N-mer probes, the template is divided into N aliquots, and for each aliquot a different initializing oligonucleotide i<sub>k</sub> is provided that forms a perfectly matched duplex at a location in binding region (40) different from that of the other initializing oligonucleotides. That is, the initializing oligonucleotides i<sub>1</sub> -i<sub>N</sub> form a set of duplexes with the template in the binding region (40), such that the ends of the duplexes proximal to the unknown sequence are from 0 to N-1 nucleotides from the start of the unknown sequence. Thus, in the first cycle of ligations with N-mer probes, a terminal nucleotide (16) of probe (30) ligated to i<sub>1</sub> in FIG. 1 will be complementary to the N-1 nucleotide of binding region (40). Likewise, a terminal nucleotide (17) of probe (30) ligated to i<sub>2</sub> in FIG. 1 will be complementary to the N-2 nucleotide of binding region (40); a terminal nucleotide (18) of probe (30) ligated to i<sub>3</sub> in FIG. 1 will be complementary to the N-3 nucleotide of binding region (40), and so on. Finally, a terminal nucleotide (15) of probe (30) ligated to in will be complementary to the first nucleotide of unknown sequence (50). In the second cycle of ligations, a terminal nucleotide (19) of probe (31) will be complementary to the second nucleotide (19) of unknown sequence (50) in duplexes starting with initializing oligonucleotide i<sub>1</sub>. Likewise, terminal nucleotides of probes ligated to duplexes starting with initializing oligonucleotides i<sub>2</sub>, i<sub>3</sub>, i<sub>4</sub>, and so on, will be complementary to the third, fourth, and fifth nucleotides of unknown sequence (50).</p>
    <p>In the above embodiment, the oligonucleotide probes are labeled so that the identity of the nucleotide abutting the extended duplex can be determined from the label.</p>
    <p>Binding region (40) has a known sequence, but can vary greatly in length and composition. It must be sufficiently long to accommodate the hybridization of an initializing oligonucleotide. Different binding regions can be employed with either identical or different initializing oligonucleotides, but for convenience of preparation, it is preferable to provide identical binding regions and different initializing oligonucleotides. Thus, all the templates are prepared identically and then separated into aliquots for use with different initializing oligonucleotides. Preferably, the binding region should be long enough to accommodate a set of different initializing oligonucleotides, each hydridizing to the template to produce a different sting point for subsequent ligations. Most preferably, the binding region is between about 20 to 50 nucleotides in length.</p>
    <p>Initializing oligonucleotides are selected to form highly stable duplexes with the binding region that remain intact during any washing steps of the extension cycles. This is conveniently achieved by selecting the length(s) of the initializing oligonucleotides to be considerably longer than that, or those, of the oligonucleotide probes and/or by selecting them to be GC-rich. Initializing oligonucleotides may also be cross-linked to the template strand by a variety of techniques, e.g. Summerton et al, U.S. Pat. No. 4,123,610; or they may be comprised of nucleotide analogs that form duplexes of greater stability than their natural counterparts, e.g. peptide nucleic acids, Science, 254: 1497-1500 (1991); Hanvey et al, Science, 258: 1481-1485 (1992); and PCT applications PCT/EP92/01219 and PCT/EP92/01220.</p>
    <p>Preferably, the length of the initializing oligonucleotide is from about 20 to 30 nucleotides and its composition comprises a sufficient percentage of G's and C's to provide a duplex melting temperature that exceeds those of the oligonucleotide probes being employed by about 10-50° C. More preferably, the duplex melting temperature of the initializing oligonucleotide exceeds those of the oligonucleotide probes by about 20-50° C. The number, N, of distinct initializing oligonucleotides employed in a sequencing operation can vary from one, in the case where a single nucleotide is identified at each cycle, to a plurality whose size is limited only by the size of oligonucleotide probe that can be practically employed. Factors limiting the size of the oligonucleotide probe include the difficulty in preparing mixtures having sufficiently high concentrations of individual probes to drive hybridization reactions at a reasonable rate, the susceptibility of longer probes to forming secondary structures, reduction in sensitivity to single base mismatches, and the like. Preferably, N is in the range of from 1 to 16; more preferably, N is in the range of from 1 to 12; and most preferably, N is in the range of from 1 to 8.</p>
    <p>A wide variety of oligonucleotide probes can be used with the invention. Generally, the oligonucleotide probes should be capable of being ligated to an initializing oligonucleotide or extended duplex to generate the extended duplex of the next extension cycle; the ligation should be template-driven in that the probe should form a duplex with the template prior to ligation; the probe should possess a blocking moiety to prevent multiple probe ligations on the same template in a single extension cycle, the probe should be capable of being treated or modified to regenerate an extendable end after ligation, and the probe should possess a signaling moiety that permits the acquisition of sequence information relating to the template after a successful ligation. As described more fully below, depending on the embodiment, the extended duplex or initializing oligonucleotide may be extended in either the 5'→3' direction or the 3'→5' direction by oligonucleotide probes. Generally, the oligonucleotide probe need not form a perfectly matched duplex with the template, although such binding is usually preferred. In preferred embodiments in which a single nucleotide in the template is identified in each extension cycle, perfect base pairing is only required for identifying that particular nucleotide. For example, in embodiments where the oligonucleotide probe is enzymatically ligated to an extended duplex, perfect base pairing--i.e. proper Watson-Crick base pairing--is required between the terminal nucleotide of the probe which is ligated and its complement in the template. Generally, in such embodiments, the rest of the nucleotides of the probe serve as "spacers" that ensure the next ligation will take place at a predetermined site, or number of bases, along the template. That is, their pairing, or lack thereof, does not provide further sequence information. Likewise, in embodiments that rely on polymerase extension for base identification, the probe primarily serves as a spacer, so specific hybridization to the template is not critical, although it is desirable.</p>
    <p>Preferably, the oligonucleotide probes are applied to templates as mixtures comprising oligonucleotides of all possible sequences of a predetermined length. The complexity of such mixtures can be reduced by a number of methods, including using so-called degeneracy-reducing analogs, such as deoxyinosine and the like, e.g. as taught by Kong Thoo Lin et al, Nucleic Acids Research, 20: 5149-5152; U.S. Pat. No. 5,002,867; Nichols et al, Nature, 369: 492-493 (1994); or by separately applying multiple mixtures of oligonucleotide probes, e.g. four mixtures comprising four disjoint subsets of oligonucleotide sequences that taken together would comprise all possible sequences of the predetermined length.</p>
    <p>Initializing oligonucleotides and oligonucleotide probes of the invention are conveniently synthesized on an automated DNA synthesizer, e.g. an Applied Biosystems, Inc. (Foster City, Calif.) model 392 or 394 DNA/RNA Synthesizer, using standard chemistries, such as phosphoramidite chemistry, e.g. disclosed in the following references: Beaucage and Iyer, Tetrahedron, 48: 2223-2311 (1992); Molko et al, U.S. Pat. No. 4,980,460; Koster et al, U.S. Pat. No. 4,725,677; Caruthers et al, U.S. Pat. Nos. 4,415,732; 4,458,066; and 4,973,679; and the like. Alternative chemistries, e.g. resulting in non-natural backbone groups, such as phosphorothioate, phosphoramidate, and the like, may also be employed provided that the resulting oligonucleotides are compatible with the ligation and other reagents of a particular embodiment. Mixtures of oligonucleotide probes are readily synthesized using well known techniques, e.g. as disclosed in Telenius et al, Genomics, 13: 718-725 (1992); Welsh et al, Nucleic Acids Research, 19: 5275-5279 (1991); Grothues et al, Nucleic Acids Research, 21: 1321-1322 (1993); Hartley, European patent application 90304496.4; and the like. Generally, these techniques simply call for the application of mixtures of the activated monomers to the growing oligonucleotide during the coupling steps where one desires to introduce the degeneracy.</p>
    <p>When conventional ligases are employed in the invention, as described more fully below, the 5' end of the probe may be phosphorylated in some embodiments. A 5' monophosphate can be attached to an oligonucleotide either chemically or enzymatically with a kinase, e.g. Sambrook et al, Molecular Cloning: A Laboratory Manual, 2nd Edition (Cold Spring Harbor Laboratory, New York, 1989). Chemical phosphorylation is described by Horn and Urdea, Tetrahedron Lett., 27: 4705 (1986), and reagents for carrying out the disclosed protocols are commercially available, e.g. 5' Phosphate-ON™ from Clontech Laboratories (Palo Alto, Calif.). Preferably, when required, oligonucleotide probes are chemically phosphorylated.</p>
    <p>The probes of the invention can be labeled in a variety of ways, including the direct or indirect attachment of fluorescent moieties, colorimetric moieties, and the like. Many comprehensive reviews of methodologies for labeling DNA and constructing DNA probes provide guidance applicable to constructing probes of the present invention. Such reviews include Matthews et al, Anal. Biochem., Vol 169, pgs. 1-25 (1988); Haugland, Handbook of Fluorescent Probes and Research Chemicals (Molecular Probes, Inc., Eugene, 1992); Keller and Manak, DNA Probes, 2nd Edition (Stockton Press, New York, 1993); and Eckstein, editor, Oligonucleotides and Analogues: A Practical Approach (IRL Press, Oxford, 1991); and the like. Many more particular methodologies applicable to the invention are disclosed in the following sample of references: Fung et al, U.S. Pat. No. 4,757,141; Hobbs, Jr., et al U.S. Pat. No. 5,151,507; Cruickshank, U.S. Pat. No. 5,091,519; (synthesis of functionalized oligonucleotides for attachment of reporter groups); Jablonski et al, Nucleic Acids Research, 14: 6115-6128 (1986)(enzyme-oligonucleotide conjugates); and Urdea et al, U.S. Pat. No. 5,124,246 (branched DNA).</p>
    <p>Preferably, the probes are labeled with one or more fluorescent dyes, e.g. as disclosed by Menchen et al, U.S. Pat. No. 5,188,934; Begot et al PCT application PCT/US90/05565.</p>
    <p>Guidance in selecting hybridization conditions for the application of oligonucleotide probes to templates can be found in numerous references, e.g. Wetmur, Critical Reviews in Biochemistry and Molecular Biology, 26: 227-259 (1991); Dove and Davidson, J. Mol. Biol. 5: 467-478 (1962); Hutton, Nucleic Acids Research, 10: 3537-3555 (1977); Breslauer et al, Proc. Natl. Acad. Sci. 83: 3746-3750 (1986); Innis et al, editors, PCR Protocols (Academic Press, New York, 1990); and the like.</p>
    <p>Generally, when an oligonucleotide probe anneals to a template in juxtaposition to an end of the extended duplex, the duplex and probe are ligated, i.e. are caused to be covalently linked to one another. Ligation can be accomplished either enzymatically or chemically. Chemical ligation methods are well known in the art, e.g. Ferris et al, Nucleosides &amp; Nucleotides, 8: 407-414 (1989); Shabarova et al, Nucleic Acids Research, 19: 4247-4251 (1991); and the like. Preferably, enzymatic ligation is carried out using a ligase in a standard protocol. Many ligases are known, and are suitable for use in the invention, e.g. Lehman, Science, 186: 790-797 (1974); Engler et al, DNA Ligases, pages 3-30 in Boyer, editor, The Enzymes, Vol. 15B (Academic Press, New York, 1982); and the like. Preferred ligases include T4 DNA ligase, T7 DNA ligase, E. coli DNA ligase, Taq ligase, Pfu ligase, and Tth ligase. Protocols for their use are well known, e.g. Sambrook et al (cited above); Barany, PCR Methods and Applications, 1: 5-16 (1991); Marsh et al, Strategies, 5: 73-76 (1992); and the like. Generally, ligases require that a 5' phosphate group be present for ligation to the 3' hydroxyl of an abutting strand.</p>
    <heading>Preparing Target Polynucleotides</heading> <p>Preferably, a target polynucleotide is conjugated to a binding region to form a template, and the template is attached to a solid phase support, such as a magnetic particle, polymeric microsphere, filter material, or the like, which permits the sequential application of reagents without complicated and time-consuming purification steps. The length of the target polynucleotide can vary widely; however, for convenience of preparation, lengths employed in conventional sequencing are preferred. For example, lengths in the range of a few hundred basepairs, 200-300, to 1 to 2 kilobase pairs are preferred.</p>
    <p>The target polynucleotides can be prepared by various conventional methods. For example, target polynucleotides can be prepared as inserts of any of the conventional cloning vectors, including those used in conventional DNA sequencing. Extensive guidance for selecting and using appropriate cloning vectors is found in Sambrook et al, Molecular Cloning: A Laboratory Manual, Second Edition (Cold Spring Harbor Laboratory, New York, 1989), and like references. Sambrook et al and Innis et al, editors, PCR Protocols (Academic Press, New York, 1990) also provide guidance for using polymerase chain reactions to prepare target polynucleotides. Preferably, cloned or PCR-amplified target polynucleotides are prepared which permit attachment to magnetic beads, or other solid supports, for ease of separating the target polynucleotide from other reagents used in the method. Protocols for such preparative techniques are described fully in Wahlberg et al, Electrophoresis, 13: 547-551 (1992); Tong et al, Anal. Chem., 64: 2672-2677 (1992); Hultman et al, Nucleic Acids Research, 17: 4937-4946 (1989); Hultman et al, Biotechniques, 10: 84-93 (1991); Syvanen et al, Nucleic Acids Research, 16: 11327-11338 (1988); Dattagupta et al, U.S. Pat. No. 4,734,363; Uhlen, PCT application PCT/GB89/00304; and like references. Kits are also commercially available for practicing such methods, e.g. Dynabeads™ template preparation kit from Dynal AS. (Oslo, Norway).</p>
    <p>Generally, the size and shape of a microparticle or beads employed in the method of the invention is not critical; however, microparticles in the size range of a few, e.g. 1-2, to several hundred, e.g. 200-1000 μm diameter are preferable, as they minimize reagent and sample usage while permitting the generation of readily detectable signals, e.g. from fluorescently labeled probes.</p>
    <heading>Schemes for Ligating, Capping, and Regenerating Extendable Termini</heading> <p>In one aspect, the invention calls for repeated steps of ligating and identifying of oligonucleotide probes. However, since the ligation of multiple probes to the same extended duplex in the same step would usually introduce identification problems, it is useful to prevent multiple extensions and to regenerate extendable termini. Moreover, if the ligation step is not 100% efficient, it would be desirable to cap extended duplexes that fail to undergo ligation so that they do not participate in any further ligation steps. That is, a capping step preferably occurs after a ligation step, by analogy with other synthetic chemical processes, such as polynucleotide synthesis, e.g. Andrus et al, U.S. Pat. No. 4,816,571. This would remove a potentially significant source of noise from signals generated in subsequent identification steps.</p>
    <p>Below, several exemplary schemes for carrying out ligation, capping, regeneration, and identification steps in accordance with the invention are described. They are presented for purposes of guidance and are not meant to be limiting.</p>
    <p>A scheme for extending an initializing oligonucleotide or an extended duplex in the 3'→5' direction is illustrated in FIG. 2. Template (20) is attached to solid phase support (10) by its 5' end. This can be conveniently accomplished via a biotin, or like linking moiety, using conventional techniques. Initializing oligonucleotide (200) having a 5' phosphate group is annealed to template (20) as described above prior to the initial cycle of ligation and identification. An oligonucleotide probe (202) of the following form is employed:</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="equation">HO-(3')BBB . . . BBB(5')-OP(═O)(O<sup>-</sup>)NH-B<sub>t</sub> *</pre>
    
    <p>where BBB . . . BBB represents the sequence of nucleotides of oligonucleotide probe (202) and B<sub>t</sub> * is a labeled chain-terminating moiety linked to the 5' carbon of the oligonucleotide via a phosphoramidate group, or other labile linkage, such as a photocleavable linkage. The nature of B<sub>t</sub> * may vary widely. It can be a labeled nucleoside (e.g. coupled via a 5'P→3'N phosphoramidate) or other moiety, so long as it prevents successive ligations. It may simply be a label connected by a linker, such as described in Agrawal and Tang, International application number PCT/US91/08347. An important feature of the oligonucleotide probe is that after annealing and ligation (204), the label may be removed and the extendable end regenerated by treating the phosphoramidate linkage with acid, e.g. as taught by Letsinger et al, J. Am. Chem. Soc., 94: 292-293 (1971); Letsinger et al, Biochem., 15: 2810-2816 (1976); Gryaznov et al, Nucleic Acid Research, 20: 3403-3409 (1992); and like references. By way of example, hydrolysis of the phosphoramidate may be accomplished by treatment with 0.8% trifluoroacetic acid in dichloromethane for 40 minutes at room temperature. Thus, after annealing, ligating, and identifying the ligated probe via the label on B<sub>t</sub> *, the chain-terminating moiety is cleaved by acid hydrolysis (206) thereby breaking the phosphorus linkage and leaving a 5' monophosphate on the ligated oligonucleotide. The steps can be repeated (208) in successive cycles. In one aspect of this embodiment, a single initializing oligonucleotide may be employed such that only one nucleotide is identified in each sequencing cycle. For such an embodiment, the above probe preferably has the following form:</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="equation">HO--(3')B(5')--OP(═O)(O<sup>-</sup>)NHBB . . . BBB-B<sub>t</sub> *</pre>
    
    <p>Thus, after each ligation step and acid cleavage step the duplex will be extended by one nucleotide.</p>
    <p>A capping step may be introduced prior to hydrolysis. For example, probe (202) may have the form:</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="equation">HO--(3')BB . . . Bp B. . . BB(5')-OP(═O)(O--)NH--B<sub>t</sub> *</pre>
    
    <p>where "p " is a exonuclease resistant linkage, such as phosphorothioate, methylphosphonate, or the like. In such an embodiment, capping can be achieved by treating the extended duplexes with an exonuclease, such as λ exonuclease, which will cleave the unligated extended duplexes back to the exonuclease resistant linkage. The presence of this linkage at the 5' end of the extended duplex will then prevent it from participating in subsequent ligations. Clearly, many other capping methodologies may be employed, e.g. acylation, ligation of an inert oligonucleotide, or the like. When free 3' hydroxyls are involved, capping may be accomplished by extending the duplex with a DNA polymerase in the presence of chain-terminating nucleoside triphosphates, e.g. dideoxynucleoside triphosphates, or the like.</p>
    <p>The phosphoramidate linkage described above is an example of a general class of internucleosidic linkages referred to herein as "chemically scissile internucleosidic linkages." These are internucleosidic linkages that may be cleaved by treating them with characteristic chemical or physical conditions, such as an oxidizing environment, a reducing environment, light of a characteristic wavelength (for photolabile linkages), or the like. Other examples of chemically scissile internucleosidic linkages which may be used in accordance with the invention are described in Urdea U.S. Pat. No. 5,380,833; Gryaznov et al, Nucleic Acids Research, 21: 1403-1408 (1993)(disulfide); Gryaznov et al, Nucleic Acids Research, 22: 2366-2369 (1994)(bromoacetyl); Urdea et al, International application PCT/US91/05287 (photolabile); and like references.</p>
    <p>Further chemically scissile linkages that may be employed with the invention include chain-terminating nucleotides that may be chemically converted into an extendable nucleoside. Examples of such compounds are described in the following references: Canard et al, International application PCT/FR94/00345; Ansorge, German patent application No. DE 4141178 A1; Metzker et al, Nucleic Acids Research, 22: 4259-4267 (1994); Cheeseman, U.S. Pat. No. 5,302,509; Ross et al, International application PCT/US90/06178; and the like.</p>
    <p>A scheme for extending an initializing oligonucleotide or an extended duplex in the 5'→3' direction is illustrated in FIG. 3A. Template (20) is attached to solid phase support (10) by its 3' end. As above, this can be conveniently accomplished via a biotin, or like linking moiety, using conventional techniques. Initializing oligonucleotide (300) having a 3' hydroxyl group is annealed to template (20) as described above prior to the initial cycle of ligation and identification. An oligonucleotide probe (302) of the following form is employed:</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="equation">OP(═O)(O<sup>-</sup>)O--(5')BBB . . . BBBRRRRB<sub>t</sub> *</pre>
    
    <p>where BBB . . . BBBRRRR represents the sequence of 2'-deoxynucleotides of oligonucleotide probe (302), "RRRR" represent a sequence of four ribonucleosides of probe (302), and B<sub>t</sub> * is a labeled chain-terminating moiety, as described above. Such mixed RNA-DNA oligonucleotides are readily synthesized using conventional automated DNA synthesizers, e.g. Duck et al, U.S. Pat. No. 5,011,769. RNase H will cleave the probe specifically in the center of the four ribonucleotide segment, Hogrefe et al, J. Biol. Chem., 265: 5561-5566 (1990), leaving a 3' hydroxyl (312) on the extended duplex, which may participate in subsequent ligation steps. Thus, a cycle in the present embodiment proceeds by annealing probe (302) to template (20) and ligating (304) to form extended duplex (306). After identification via B<sub>t</sub> *, the extended duplex is treated with RNase H to cleave the label and regenerate an extendable end. The cycle is then repeated (314). Capping (310) can be carried out prior to RNase H treatment by extending the unligated ends with a DNA polymerase in the presence of the four dideoxynucleoside triphosphates, ddATP, ddCTP, ddGTP, and ddTTP.</p>
    <p>As illustrated in FIG. 3B, a similar scheme can be employed for 3'→5' extensions. In such an embodiment, initiating oligonucleotide or extended duplex (330) has a 5' monophosphate and the oligonucleotide probe (332) has the form:</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="equation">HO--(3')BBB . . . BBBRRRRB . . . BB<sub>t</sub> *</pre>
    
    <p>As above, after annealing, ligating (334), and identifying (338), extended duplex (336) is cleaved by RNase H which in this case leaves a 5' monophosphate (342) at the terminus of the extended duplex. With the regenerated extendable end, the cycle can be repeated (344). A capping step can be included prior to RNase H hydrolysis by either ligating an unlabeled non-RNA-containing probe, or by removing any remaining 5' monophosphates by treatment with a phosphatase.</p>
    <p>Identification of nucleotides can be accomplished by polymerase extension following ligation. As exemplified in FIG. 4, for this embodiment, template (20) is attached to solid phase support (10) as described above and initializing oligonucleotide (400) having a 3' hydroxyl is annealed to the template prior to the initial cycle. Oligonucleotide probes (402) of the form:</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="equation">OP(═O)(O<sup>-</sup>)O--(5')BBB . . . BBBRRRRB . . . B(3')OP(═O)(O<sup>-</sup>)O</pre>
    
    <p>are annealed to template (20) and ligated (404) to form extended duplex (406). The 3' monophosphate, which prevents successive ligations of probes in the same cycle, is removed with phosphatase (408) to expose a free 3' hydroxyl (410). Clearly, alternative blocking approaches may also be used. Extended duplex (406) is further extended by a nucleic acid polymerase in the presence of labeled dideoxynucleoside triphosphates (412), thereby permitting the identification of a nucleotide of template (20) by the label of the incorporated dideoxynucleotide. The labeled dideoxynucleotide and a portion of probe (402) are then cleaved (414), for example, by RNase H treatment, to regenerate an extendable end on extended duplex (406). The cycle is then repeated (416).</p>
    <p>In order to reduce the number of separate annealing reactions that must be carried out, the oligonucleotide probes may be grouped into mixtures, or subsets, of probes whose perfectly matched duplexes with complementary sequences have similar stability or free energy of binding. Such subsets of oligonucleotide probes having similar duplex stability are referred to herein as "stringency classes" of oligonucleotide probes. The mixtures, or stringency classes, of oligonucleotide probes are then separately combined with the target polynucleotide under conditions such that substantially only oligonucleotide probes complementary to the target polynucleotide form duplexes. That is, the stringency of the hybridization reaction is selected so that substantially only perfectly complementary oligonucleotide probes form duplexes. These perfectly matched duplexes are then ligated to form extended duplexes. For a given oligonucleotide probe length, the number of oligonucleotide probes within each stringency class can vary widely. Selection of oligonucleotide probe length and stringency class size depends on several factors, such as length of target sequence and how it is prepared, the extent to which the hybridization reactions can be automated, the degree to which the stringency of the hybridization reaction can be controlled, the presence or absence of oligonucleotide probes with complementary sequences, and the like. Guidance in selecting an appropriate size of stringency class for a particular embodiment can be found in the general literature on nucleic acid hybridization and polymerase chain reaction methodology, e.g. Gotoh, Adv. Biophys. 16: 1-52 (1983); Wetmer, Critical Reviews in Biochemistry and Molecular Biology 26: 227-259 (1991); Breslauer et al, Proc. Natl. Acad. Sci. 83: 3746-3750 (1986); Wolf et al, Nucleic Acids Research, 15: 2911-2926 (1987); Innis et al, editors, PCR Protocols (Academic Press, New York, 1990); McGraw et al, Biotechniques, 8: 674-678 (1990), and the like. Stringency can be controlled by varying several parameters, including temperature, salt concentration, concentration of certain organic solvents, such as formamide, and the like. Preferably, temperature is used to define the stringency classes because the activity of the various polymerases or ligases employed limits the degree to which salt concentration or organic solvent concentration can be varied for ensuring specific annealing of the oligonucleotide probes.</p>
    <p>Generally, the larger the stringency class the greater the complexity of the hybridizing mixture and the lower the concentration of any particular oligonucleotide probe in the mixture. A lower concentration of a oligonucleotide probe having a complementary site on a target polynucleotide reduces the relative likelihood of the oligonucleotide probe hybridizing and being ligated. This, in turn, leads to reduced sensitivity. Larger stringency classes also have a greater variance in the stabilities of the duplexes that form between a oligonucleotide probe and a complementary sequence. On the other hand, smaller stringency classes require a larger number of hybridization reactions to ensure that all oligonucleotide probes of a set are hybridized to a target polynucleotide.</p>
    <p>For example, when 8-mer oligonucleotide probes are employed stringency classes may include between about 50 to about 500 oligonucleotide probes each. Thus, several hundred to several thousand hybridization/ligation reactions are required. For larger sized oligonucleotide probes, much larger stringency classes are required to make the number of hybridization/extension reactions practical, e.g. 10<sup>4</sup> -10<sup>5</sup>, or more.</p>
    <p>Oligonucleotide probes of the same stringency class can be synthesized simultaneously, in a manner similar to which fully random oligonucleotide probes are synthesized, e.g. as disclosed in Telenius et al, Genomics, 13: 718-725 (1992); Welsh et al, Nucleic Acids Research, 19: 5275-5279 (1991); Grothues et al, Nucleic Acids Research, 21: 1321-1322 (1993); Hartley, European patent application 90304496.4; and the like. The difference is that at each cycle different mixtures of monomers are applied to the growing oligonucleotide probe chain, wherein the proportion of each monomer in the mixture is dictated by the proportion of each nucleoside at the position of the oligonucleotide probe in the stringency class. Stringency classes are readily formed by computing the free energy of duplex formation by available algorithms, e.g. Breslauer et al, Proc. Natl. Acad. Sci., 83: 3746-3750 (1986); Lowe et al, Nucleic Acids Research, 18: 1757-1761 (1990); or the like. The oligonucleotide probes can be ordered according to the free energy of binding to their complement under standard reaction conditions, e.g. with a standard bubble sort, Baase, Computer Algorithms (Addison-Wesley, Menlo Park, 1978). For example the following is the list of ten 6-mers with the greatest stability (from top to bottom) in terms of free energy of duplex formation under standard hybridization conditions and the least stability in terms of free energy of duplex formation (the free energies being computed via Breslauer (cited above)):</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">______________________________________      Oligonucleotide probe       Ranking Sequence (5'→3')______________________________________  1        GCGCGC    2 CGCGCG    3 CCCGCG    4 CGCCCG    5 CGCGCC    6 CGCGGC    7 CGGCGC    8 GCCGCG    9 GCGCCG   10 GCGCGG    . .    . .    . .  4087 TCATAT  4088 TGATAT  4089 CATATA  4090 TATATG  4091 ATCATG  4092 ATGATG  4093 CATCAT  4094 CATGAT  4095 CATATG  4096 ATATAT______________________________________</pre>
    
    <p>Thus, if a stringency class consisted of the first ten 6-mers the mixture monomers for the first (3'-most) position would be 0:4:6:0 (A:C:G:T), for the second position it would be 0:6:4:0, and so on. If a stringency class consisted of the last ten 6-mers the mixture of monomers for the first position would be 1:0:4:5, for the second position it would be 5:0:0:5, and so on. The resulting mixtures may then be further enriched for sequences of the desired stingency class by thermal elution, e.g. Miyazawa et al, J. Mol. Biol., 11: 223-237 (1965).</p>
    <p>More conveniently, stringency classes containing several hundred to several thousands of oligonucleotides may be synthesized directly by a variety of parallel synthesis approaches, e.g. Frank et al, U.S. Pat. No. 4,689,405; Matson et al, Anal. Biochem., 224: 110-116 (1995); Fodor et al, International application PCT/US93/04145; Pease et al, Proc. Natl. Acad. Sci., 91: 5022-5026 (1994); Southern et al, J. Biotechnology, 35: 217-227 (1994), Brennan, International application PCT/US94/05896; or the like.</p>
    <p>In some cases it may be desirable to form additional stringency classes of oligonucleotide probes by placing in a separate subset oligonucleotide probes having complementary sequences to other oligonucleotide probes in a subset or oligonucleotide probe that are susceptible of forming oligonucleotide probe-dimers.</p>
    <p>Clearly, one of ordinary skill in the art could combine features of the embodiments set forth above to design still further embodiments in accordance with the invention, but not expressly set forth above.</p>
    <p>The invention also includes systems and apparatus for carrying out method of the invention automatically. Such systems and apparatus can take a variety of forms depending on several design constraints, including i) the nature of the solid phase support used to anchor the target polynucleotide, ii) the degree of parallel operation desired, iii) the detection scheme employed; iv) whether reagents are reused or discarded, and the like. Generally, the apparatus comprises a series of reagent reservoirs, one or more reaction vessels containing target polynucleotide, preferably attached to a solid phase support, e.g. magnetic beads, one or more detection stations, and a computer controlled means for transferring in a predetermined manner reagents from the reagent reservoirs to and from the reaction vessels and the detection stations. The computer controlled means for transferring reagents and controlling temperature can be implemented by a variety of general purpose laboratory robots, such as that disclosed by Harrison et al, Biotechniques, 14: 88-97 (1993); Fujita et al, Biotechniques, 9: 584-591 (1990); Wada et al, Rev. Sci. Instrum., 54: 1569-1572 (1983); or the like. Such laboratory robots are also available commercially, e.g. Applied Biosystems model 800 Catalyst (Foster City, Calif.).</p>
    <p>A variety of kits may be provided for carrying out different embodiments of the invention. Generally, kits of the invention include oligonucleotide probes, initializing oligonucleotides, and a detection system. Kits further include ligation reagents and instructions for practicing the particular embodiment of the invention. In embodiments employing protein ligases, RNase H, nucleic acid polymerases, or other enzymes, their respective buffers may be included. In some cases, these buffers may be identical. Preferably, kits also include a solid phase support, e.g. magnetic beads, for anchoring templates. In one preferred kit, fluorescently labeled oligonucleotide probes are provided such that probes corresponding to different terminal nucleotides of the target polynucleotide carry distinct spectrally resolvable fluorescent dyes. As used herein, "spectrally resolvable" means that the dyes may be distinguished on the basis of their spectral characteristics, particularly fluorescence emission wavelength, under conditions of operation. Thus, the identity of the one or more terminal nucleotides would be correlated to a distinct color, or perhaps ratio of intensities at different wavelengths. More preferably, four such probes are provided that allow a one-to-one correspondence between each of four spectrally resolvable fluorescent dyes and the four possible terminal nucleotides on a target polynucleotide. Sets of spectrally resolvable dyes are disclosed in U.S. Pat. Nos. 4,855,225 and 5,188,934; International application PCT/US90/05565; and Lee et al, Nucleic Acids Researchs, 20: 2471-2483 (1992).</p>
    <heading>EXAMPLE 1</heading> <heading>Sequencing a Target Polynucleotide Amplified from pUC19 with Four Initializing Oligonucleotides</heading> <p>In this example, a template comprising a binding region and a portion of the pUC19 plasmid is amplified by PCR and attached to magnetic beads. Four initializing oligonucleotides are employed in separate reactions as indicated below. 8-mer oligonucleotide probes are employed having 4 central ribonucleotides and both 5' and 3' monophosphates, as shown in the following formula:</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="equation">OP(═O)(O<sup>-</sup>)O--(5')BBRRRRBB(3')--OP(═O)(O<sup>-</sup>)O</pre>
    
    <p>After annealing, probes are enzymatically ligated to the initializing oligonucleotides and the magnetic bead supports are washed. The 3' phosphates of the ligated probes are removed with phosphatase, after which the probes are extended with DNA polymerase in the presence of the four labeled dideoxynucleoside triphosphate chain terminators. After washing and identification of the extended nucleotide, the ligated probes are cleaved at the ribonucleotide moiety with RNAse H to remove the label and to regenerate an extendable end.</p>
    <p>The following double stranded fragment comprising a 36-mer binding region is ligated into a Sac I/Sma I-digested pUC19 (upper strand: SEQ ID NO. 1; lower strand: SEQ ID NO. 9):</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="equation">CCTCTCCCTTCCCTCTCCTCCCTCTCCCCTCTCCCTC TCGAGGAGAGGGAAGGGAGAGGAGGGAGAGGGGAGAGGGAGGGCC</pre>
    
    <p>After isolation and amplification, a 402 basepair fragment of the modified pUC19 is amplified by PCR for use as a template. The fragment spans a region of pUC19 from position 41 to the binding region inserted adjacent to the Sac I site in the polylinker region (position 413 of the unmodified pUC19), Yanisch-Perron et al, Gene, 33: 103-119 (1985). Two 18-mer oligonucleotide probes are employed having sequences 5'-CCCTCTCCCCTCTCCCTCx-3' (SEQ ID NO. 10) and 5'-GCAGCTCCCGGAGACGGT-3', (SEQ ID NO. 11) where "x" is a 3' biotin moiety is attached during synthesis using a commercially available reagent with manufacturer's protocol, e.g. 3' Biotin-ON CPG (Clontech Laboratories, Palo Alto, Calif.). The amplified template is isolated and attached to streptavidin-coated magnetic beads (Dynabeads) using manufacturer's protocol, Dynabeads Template Preparation Kit, with M280-streptavidin (Dynal, Inc., Great Neck, N.Y.). A sufficient quantity of the biotinylated 313 basepair fragment is provided to load about 300 μg of Dynabeads M280-Streptavidin.</p>
    <p>The binding region sequence is chosen so that the duplexes formed with the initiating oligonucleotides have compositions of about 66% GC to enhance duplex stability. The sequence is also chosen to prevent secondary structure formation and fortuitous hybridization of an initializing oligonucleotide to more than one location within the binding region. Any shifting of position of a given initializing oligonucleotide within the binding region results in a significant number of mismatched bases.</p>
    <p>After loading, the non-biotinylated strand of template is removed by heat denaturation, after which the magnetic beads are washed and separated into four aliquots. The template attached to the magnetic beads has the following sequence, where the defined sequence separated by the ellipses ( . . . ) are listed in the attached sequence listing as two separate sequences, SEQ ID NO. 12 and SEQ ID NO. 13:</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="equation">(Magnetic bead)-(linker)-(3')-CTCCCTCTCCCCTCTCCCTCCTCTCCCTTCCTCTCCTCGAGCTTAAGT . . . CTCGACG-(5')</pre>
    
    <p>The following four oligonucleotides are employed as initializing oligonucleotides in each of the separate aliquots of template:</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="equation">(SEQ ID NO. 2) 5'-GAGGAGAGGGAAGGAGAGGAG</pre>
    
    <p>
      </p> <pre xml:space="preserve" listing-type="equation">(SEQ ID NO. 3) 5'-GGAGGAGAGGGAAGGAGAGGA</pre>
    
    <p>
      </p> <pre xml:space="preserve" listing-type="equation">(SEQ ID NO. 4) 5'-GGGAGGAGAGGGAAGGAGAGG</pre>
    
    <p>
      </p> <pre xml:space="preserve" listing-type="equation">(SEQ ID NO. 5) 5'-AGGGAGGAGAGGGAAGGAGAG</pre>
    
    <p>Reactions and washes below are generally carried out in 50 μL volumes of manufacturer's (New England Biolabs') recommended buffers for the enzymes employed, unless otherwise indicated. Standard buffers are also described in Sambrook et al, Molecular Cloning, 2nd Edition (Cold Spring Harbor Laboratory Press, 1989).</p>
    <p>96 stingency classes of 684 or 682 oligonucleotide probes each (2 subsets for each of 48 different annealing temperatures) are formed which together contain all 8-mer probes for each of the four aliquots. The probes of each of the 96 classes are separately annealed to the target polynucleotide in reaction mixtures having the same components, with the exception that extensions and ligations carried out with Sequenase and T4 DNA ligase at temperatures less than 37° C. and extensions and ligations carried out with Taq Stoffel fragment and a thermostable ligase otherwise.</p>
    <p>The 48 stringency conditions are defined by annealing temperatures which range from 22° C. to 70° C., such that each grouping of subsets at the same temperature differ in annealing temperature by 1° C. from that of the subset groupings containing the next highest and next lowest stringency classes. The range of annealing temperatures (22-70° C.) is roughly bounded by the temperatures 5-10 degrees below the temperatures at which the least stable and most stable 8-mers, respectively, are expected to have about fifty percent maximum annealing in a standard PCR buffer solution.</p>
    <p>After 5-10 minutes incubation at 80° C., the reaction mixtures are brought down to their respective annealing temperatures over a period of 20-30 minutes. After ligation, washing and treatment with phosphatase, 2 units of polymerase and labeled dideoxynucleotide triphosphates (0.08 mM final reaction concentration and labeled with TAMRA (tetramethylrhodamine), FAM (fluorescein), ROX (rhodamine X), and JOE (2',7'-dimethoxy-4',5'-dichlorofluorescein)) are added. After 15 minutes, the beads are washed with H<sub>2</sub> O and the identity of the extended nucleotide is determined by illuminating each reaction mixture with standard wavelengths, e.g Users Manual, model 373 DNA Sequencer (Applied Biosystems, Foster City, Calif.).</p>
    <p>After identification, the reaction mixtures are treated with RNase H using the manufacturer's suggested protocol and washed. The RNase H treated extended duplexes have regenerated 3' hydroxyls and are ready for the next cycle of ligation/extension/cleavage. The cycles are carried out until all the nucleotides of the test sequence are identified.</p>
    <heading>EXAMPLE 2</heading> <heading>Sequencing a Target Polynucleotide Amplified from pUC19 with One Initializing Oligonucleotide</heading> <p>In this example, a template is prepared in accordance with Example 1, except that since extension is in the 5'→3' direction in this example, the biotin moiety is attached to the 5' end of the primer hybridizing to the CT-rich strand of the binding region. Thus, in this example, the binding region of the single stranded template will be a GA-rich segment (essentially the complement of the binding region of Example 1). Two 18-mer oligonucleotide probes are employed having sequences 5'-xGAGGGAGAGGGGAGAGGG-3' (SEQ ID NO. 7) and 5'-ACCGTCTCCGGGAGCTGC-3', where "x" is a 5' biotin moiety is attached during synthesis using commercially available reagents with manufacturers' protocols, e.g. the Amninolink aminoalkylphosphoramidite linking agent (Applied Biosystems, Foster City, Calif.) and Biotin-X-NHS Ester available form Clontech Laboratories (Palo Alto, Calif.).</p>
    <p>A single 21-mer initializing oligonucleotide is employed with the following sequence:</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="equation">(SEQ ID NO. 8) 5'--OP(═O)(O<sup>-</sup>)O--CCTCTCCCTTCCCTCTCCTCC-3'</pre>
    
    <p>6-mer oligonucleotide probes are employed that have an acid labile phosphoramidate linkage between the 3'-most nucleoside and 3'-penultimate nucleoside of the probe, as shown in the following formula:</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="equation">HO--(3')B(5')--OP(═O)(O<sup>-</sup>)NH--(3')BBBBB<sub>t</sub> *</pre>
    
    <p>where B<sub>t</sub> * is a JOE-, FAM-, TAMRA-, or ROX-labeled dideoxynucleoside, such that the label corresponds to the identity of the 3'-most nucleotide (so 16 different labeled dideoxynucleosides are used in the synthesis of the probes).</p>
    <p>As above, the 6-mer probes are prepared in 96 stringency classes of 42 or 43 probes each (2 subsets for each of 48 different annealing temperatures). Hybridizations and ligations are carried out as described above. After ligation and washing, a nucleoside in the target polynucleotide is identified by the fluorescent signal of the oligonucleotide probe. Acid cleavage is then carried out by treating the extended duplex with 0.8% trifluoroacetic acid in dichloromethane for 40 minutes at room temperature to regenerate an extendable end on the extended duplex. The process continues until the sequence of the target polynucleotide is determined.</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">__________________________________________________________________________#             SEQUENCE LISTING   - -  - - (1) GENERAL INFORMATION:   - -    (iii) NUMBER OF SEQUENCES: 13   - -  - - (2) INFORMATION FOR SEQ ID NO:1:   - -      (i) SEQUENCE CHARACTERISTICS:       (A) LENGTH: 37 base - #pairs       (B) TYPE: nucleic acid       (C) STRANDEDNESS: single       (D) TOPOLOGY: linear   - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:1:   - - CCTCTCCCTT CCCTCTCCTC CCTCTCCCCT CTCCCTC      - #- #      37  - -  - - (2) INFORMATION FOR SEQ ID NO:2:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 21 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:2:  - - GAGGAGAGGG AAGGAGAGGA G           - #                  - #   - #21  - -  - - (2) INFORMATION FOR SEQ ID NO:3:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 21 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:3:  - - GGAGGAGAGG GAAGGAGAGG A           - #                  - #   - #21  - -  - - (2) INFORMATION FOR SEQ ID NO:4:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 21 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:4:  - - GGGAGGAGAG GGAAGGAGAG G           - #                  - #   - #21  - -  - - (2) INFORMATION FOR SEQ ID NO:5:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 21 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:5:  - - AGGGAGGAGA GGGAAGGAGA G           - #                  - #   - #21  - -  - - (2) INFORMATION FOR SEQ ID NO:6:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 18 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:6:  - - GAGGGAGAGG GGAGAGGG             - #                  - # - #  18  - -  - - (2) INFORMATION FOR SEQ ID NO:7:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 18 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:7:  - - ACCGTCTCCG GGAGCTGC             - #                  - # - #  18  - -  - - (2) INFORMATION FOR SEQ ID NO:8:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 21 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:8:  - - CCTCTCCCTT CCCTCTCCTC C           - #                  - #   - #21  - -  - - (2) INFORMATION FOR SEQ ID NO:9:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 45 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:9:  - - CCGGGAGGGA GAGGGGAGAG GGAGGAGAGG GAAGGGAGAG GAGCT   - #- #45  - -  - - (2) INFORMATION FOR SEQ ID NO:10:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 18 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:10:  - - CCCTCTCCCC TCTCCCTC             - #                  - # - #  18  - -  - - (2) INFORMATION FOR SEQ ID NO:11:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 18 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:11:  - - GCAGCTCCCG GAGACGGT             - #                  - # - #  18  - -  - - (2) INFORMATION FOR SEQ ID NO:12:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 48 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:12:  - - TGAATTCGAG CTCCTCTCCT TCCCTCTCCT CCCTCTCCCC TCTCCCTC  - #48  - -  - - (2) INFORMATION FOR SEQ ID NO:13:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 7 base p - #airs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:13:  - - GCAGCTC                 - #                  - #                  -#           7__________________________________________________________________________</pre>
    
    </div></div></div><div class="patent-section patent-tabular-section"><a id="backward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Cited Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US3846402">US3846402</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">May 2, 1972</td><td class="patent-data-table-td patent-date-value">Nov 5, 1974</td><td class="patent-data-table-td ">Max Planck Gesellschaft</td><td class="patent-data-table-td ">Thiophosphate analogues of the nucleoside diphosphates and triphosphates and a method for the preparation thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4362876">US4362876</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Aug 11, 1978</td><td class="patent-data-table-td patent-date-value">Dec 7, 1982</td><td class="patent-data-table-td ">The Sherwin-Williams Company</td><td class="patent-data-table-td ">Preparation of dihydroxyquinoline and certain derivatives</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4661450">US4661450</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">May 3, 1983</td><td class="patent-data-table-td patent-date-value">Apr 28, 1987</td><td class="patent-data-table-td ">Molecular Genetics Research And Development Limited Partnership</td><td class="patent-data-table-td ">Genetic engineering</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4876187">US4876187</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Dec 5, 1985</td><td class="patent-data-table-td patent-date-value">Oct 24, 1989</td><td class="patent-data-table-td ">Meiogenics, Inc.</td><td class="patent-data-table-td ">Nucleic acid compositions with scissile linkage useful for detecting nucleic acid sequences</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5011769">US5011769</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Apr 29, 1988</td><td class="patent-data-table-td patent-date-value">Apr 30, 1991</td><td class="patent-data-table-td ">Meiogenics U.S. Limited Partnership</td><td class="patent-data-table-td ">Methods for detecting nucleic acid sequences</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5149796">US5149796</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Apr 30, 1991</td><td class="patent-data-table-td patent-date-value">Sep 22, 1992</td><td class="patent-data-table-td ">City Of Hope</td><td class="patent-data-table-td ">Chimeric DNA-RNA catalytic sequences</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5218103">US5218103</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jan 22, 1991</td><td class="patent-data-table-td patent-date-value">Jun 8, 1993</td><td class="patent-data-table-td ">University Patents, Inc.</td><td class="patent-data-table-td ">Nucleoside thiophosphoramidites</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5264562">US5264562</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Apr 24, 1991</td><td class="patent-data-table-td patent-date-value">Nov 23, 1993</td><td class="patent-data-table-td ">Gilead Sciences, Inc.</td><td class="patent-data-table-td ">Oligonucleotide analogs with novel linkages</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5272250">US5272250</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jul 10, 1992</td><td class="patent-data-table-td patent-date-value">Dec 21, 1993</td><td class="patent-data-table-td ">Spielvogel Bernard F</td><td class="patent-data-table-td ">Boronated phosphoramidate compounds</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5476925">US5476925</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jan 23, 1995</td><td class="patent-data-table-td patent-date-value">Dec 19, 1995</td><td class="patent-data-table-td ">Northwestern University</td><td class="patent-data-table-td ">Oligodeoxyribonucleotides including 3&#39;-aminonucleoside-phosphoramidate linkages and terminal 3&#39;-amino groups</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5625050">US5625050</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Mar 31, 1994</td><td class="patent-data-table-td patent-date-value">Apr 29, 1997</td><td class="patent-data-table-td ">Amgen Inc.</td><td class="patent-data-table-td ">Modified oligonucleotides and intermediates useful in nucleic acid therapeutics</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5652355">US5652355</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jul 23, 1992</td><td class="patent-data-table-td patent-date-value">Jul 29, 1997</td><td class="patent-data-table-td ">Worcester Foundation For Experimental Biology</td><td class="patent-data-table-td ">Hybrid oligonucleotide phosphorothioates</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5750341">US5750341</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Apr 17, 1995</td><td class="patent-data-table-td patent-date-value">May 12, 1998</td><td class="patent-data-table-td ">Lynx Therapeutics, Inc.</td><td class="patent-data-table-td ">Cycles of duplex extension along a single stranded dna template</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1991008313A1?cl=en">WO1991008313A1</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Dec 3, 1990</td><td class="patent-data-table-td patent-date-value">Jun 5, 1991</td><td class="patent-data-table-td ">Isis Pharmaceuticals Inc</td><td class="patent-data-table-td ">Antisense oligonucleotide inhibition of papillomavirus</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1995004748A1?cl=en">WO1995004748A1</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Aug 9, 1994</td><td class="patent-data-table-td patent-date-value">Feb 16, 1995</td><td class="patent-data-table-td ">Kevin P Anderson</td><td class="patent-data-table-td ">Oligomers for modulating viral processes</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="npl-citations"></a><div class="patent-section-header"><span class="patent-section-title">Non-Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th colspan="3"class="patent-data-table-th">Reference</th></tr></thead><tr><td class="patent-data-table-td ">1</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Atabekov et al., FEBS Letters 232(1) : 96 98 (1988).</td></tr><tr><td class="patent-data-table-td ">2</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Atabekov et al., FEBS Letters 232(1) : 96-98 (1988).</td></tr><tr><td class="patent-data-table-td ">3</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Bannwarth W., Helvetica Chimica Acta 71 : 1517 1527 (1988).</td></tr><tr><td class="patent-data-table-td ">4</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Bannwarth W., Helvetica Chimica Acta 71 : 1517-1527 (1988).</td></tr><tr><td class="patent-data-table-td ">5</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Grayaznov et al., J. of the American Chemical Society 116 (7) : 3143 3144 (1994).</td></tr><tr><td class="patent-data-table-td ">6</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Grayaznov et al., J. of the American Chemical Society 116 (7) : 3143-3144 (1994).</td></tr><tr><td class="patent-data-table-td ">7</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Gryaznov et al., Nucleic Acids Research 20(13) : 3403 3409 (1992).</td></tr><tr><td class="patent-data-table-td ">8</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Gryaznov et al., Nucleic Acids Research 20(13) : 3403-3409 (1992).</td></tr><tr><td class="patent-data-table-td ">9</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Inoue et al., FEBS Letters 215(2) :327 330 (1987).</td></tr><tr><td class="patent-data-table-td ">10</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Inoue et al., FEBS Letters 215(2) :327-330 (1987).</td></tr><tr><td class="patent-data-table-td ">11</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Kempe et al., Nucleic Acids Research 10(21) :6695 6714 (1982).</td></tr><tr><td class="patent-data-table-td ">12</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Kempe et al., Nucleic Acids Research 10(21) :6695-6714 (1982).</td></tr><tr><td class="patent-data-table-td ">13</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Lavina et al., Biochemie 75 : 25 27 (1993).</td></tr><tr><td class="patent-data-table-td ">14</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Lavina et al., Biochemie 75 : 25-27 (1993).</td></tr><tr><td class="patent-data-table-td ">15</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Perreault et al., Nature 344 :565 567 (1990).</td></tr><tr><td class="patent-data-table-td ">16</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Perreault et al., Nature 344 :565-567 (1990).</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="forward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Referenced by</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Citing Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7138506">US7138506</a></td><td class="patent-data-table-td patent-date-value">May 9, 2002</td><td class="patent-data-table-td patent-date-value">Nov 21, 2006</td><td class="patent-data-table-td ">Genetic Id, Na, Inc.</td><td class="patent-data-table-td ">Universal microarray system</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7563576">US7563576</a></td><td class="patent-data-table-td patent-date-value">Jan 22, 2007</td><td class="patent-data-table-td patent-date-value">Jul 21, 2009</td><td class="patent-data-table-td ">Illumina, Inc.</td><td class="patent-data-table-td ">Using probes having priming sequence complementary to primer sequence, variable sequence and nucleotide; decoding microsphere array sensors</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7960119">US7960119</a></td><td class="patent-data-table-td patent-date-value">Jul 20, 2009</td><td class="patent-data-table-td patent-date-value">Jun 14, 2011</td><td class="patent-data-table-td ">Illumina, Inc.</td><td class="patent-data-table-td ">Combinatorial decoding of random nucleic acid arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8173198">US8173198</a></td><td class="patent-data-table-td patent-date-value">Jul 23, 2009</td><td class="patent-data-table-td patent-date-value">May 8, 2012</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Deposition of metal oxides onto surfaces as an immobilization vehicle for carboxylated or phophated particles or polymers</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8206917">US8206917</a></td><td class="patent-data-table-td patent-date-value">Jun 13, 2011</td><td class="patent-data-table-td patent-date-value">Jun 26, 2012</td><td class="patent-data-table-td ">Illumina, Inc.</td><td class="patent-data-table-td ">Combinatorial decoding of random nucleic acid arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8222040">US8222040</a></td><td class="patent-data-table-td patent-date-value">Aug 28, 2007</td><td class="patent-data-table-td patent-date-value">Jul 17, 2012</td><td class="patent-data-table-td ">Lightspeed Genomics, Inc.</td><td class="patent-data-table-td ">Nucleic acid sequencing by selective excitation of microparticles</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8323929">US8323929</a></td><td class="patent-data-table-td patent-date-value">Apr 7, 2009</td><td class="patent-data-table-td patent-date-value">Dec 4, 2012</td><td class="patent-data-table-td ">Becton, Dickinson And Company</td><td class="patent-data-table-td ">Methods for detecting nucleic acid sequence variations</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8329404">US8329404</a></td><td class="patent-data-table-td patent-date-value">Mar 2, 2012</td><td class="patent-data-table-td patent-date-value">Dec 11, 2012</td><td class="patent-data-table-td ">Applied Biosystems Llc</td><td class="patent-data-table-td ">Reagents, methods, and libraries for bead-based sequencing</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8361713">US8361713</a></td><td class="patent-data-table-td patent-date-value">Oct 12, 2007</td><td class="patent-data-table-td patent-date-value">Jan 29, 2013</td><td class="patent-data-table-td ">Illumina, Inc.</td><td class="patent-data-table-td ">System and apparatus for sequential processing of analytes</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8431691">US8431691</a></td><td class="patent-data-table-td patent-date-value">Jul 21, 2008</td><td class="patent-data-table-td patent-date-value">Apr 30, 2013</td><td class="patent-data-table-td ">Applied Biosystems Llc</td><td class="patent-data-table-td ">Reagents, methods, and libraries for bead-based sequencing</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8481264">US8481264</a></td><td class="patent-data-table-td patent-date-value">Sep 18, 2009</td><td class="patent-data-table-td patent-date-value">Jul 9, 2013</td><td class="patent-data-table-td ">Pacific Biosciences Of California, Inc.</td><td class="patent-data-table-td ">Immobilized nucleic acid complexes for sequence analysis</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8502867">US8502867</a></td><td class="patent-data-table-td patent-date-value">Mar 19, 2010</td><td class="patent-data-table-td patent-date-value">Aug 6, 2013</td><td class="patent-data-table-td ">Lightspeed Genomics, Inc.</td><td class="patent-data-table-td ">Synthetic aperture optics imaging method using minimum selective excitation patterns</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8563246">US8563246</a></td><td class="patent-data-table-td patent-date-value">Jun 25, 2012</td><td class="patent-data-table-td patent-date-value">Oct 22, 2013</td><td class="patent-data-table-td ">Illumina, Inc.</td><td class="patent-data-table-td ">Combinatorial decoding of random nucleic acid arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8728729">US8728729</a></td><td class="patent-data-table-td patent-date-value">Dec 4, 2012</td><td class="patent-data-table-td patent-date-value">May 20, 2014</td><td class="patent-data-table-td ">Illumina, Inc.</td><td class="patent-data-table-td ">Method for sequential sequencing nucleic acids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8759077">US8759077</a></td><td class="patent-data-table-td patent-date-value">Aug 28, 2007</td><td class="patent-data-table-td patent-date-value">Jun 24, 2014</td><td class="patent-data-table-td ">Lightspeed Genomics, Inc.</td><td class="patent-data-table-td ">Apparatus for selective excitation of microparticles</td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="classifications"></a><div class="patent-section-header"><span class="patent-section-title">Classifications</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th"> </th><th class="patent-data-table-th"> </th></tr></thead><tr><td class="patent-data-table-td ">U.S. Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=Bp9MBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S022100">536/22.1</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=Bp9MBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S023100">536/23.1</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=Bp9MBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S024300">536/24.3</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=Bp9MBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S006120">435/6.12</a></span></td></tr><tr><td class="patent-data-table-td ">International Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=Bp9MBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C07H0021020000">C07H21/02</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=Bp9MBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12Q0001680000">C12Q1/68</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=Bp9MBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C07H0021000000">C07H21/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=Bp9MBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C07H0021040000">C07H21/04</a></span></td></tr><tr><td class="patent-data-table-td ">Cooperative Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=Bp9MBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C07H21/02">C07H21/02</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=Bp9MBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12Q1/6869">C12Q1/6869</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=Bp9MBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C07H21/04">C07H21/04</a></span></td></tr><tr><td class="patent-data-table-td ">European Classification</td><td class="patent-data-table-td "><span class="nested-value">C12Q1/68E</span>, <span class="nested-value">C07H21/04</span>, <span class="nested-value">C07H21/02</span></td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="legal-events"></a><div class="patent-section-header"><span class="patent-section-title">Legal Events</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Date</th><th class="patent-data-table-th">Code</th><th class="patent-data-table-th">Event</th><th class="patent-data-table-th">Description</th></tr></thead><tr><td class="patent-data-table-td patent-date-value">Mar 24, 2011</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">12</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Sep 7, 2010</td><td class="patent-data-table-td ">B1</td><td class="patent-data-table-td ">Reexamination certificate first reexamination</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">THE PATENTABILITY OF CLAIM 1 IS CONFIRMED. NEW CLAIMS 3-10 ARE ADDED AND DETERMINED TO BE PATENTABLE. CLAIM 2 WAS NOT REEXAMINED. NEWLY PRESENTED CLAIMS 11-14 ARE CANCELLED.</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Aug 26, 2008</td><td class="patent-data-table-td ">RR</td><td class="patent-data-table-td ">Request for reexamination filed</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20080630</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Nov 14, 2007</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">ILLUMINA, INC., CALIFORNIA</span></div><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">MACEVICZ, STEPHEN C., CALIFORNIA</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">CASE NO. CO702845MEJ-COMPLAINT FILED IN THE UNITED STATES DISTRICT COURT/SAN JOSE DIVISION ON MAY 31, 2007,;ASSIGNOR:APPLERA CORPORATION-APPLIED BIOSYSTEMS GORUP;REEL/FRAME:020125/0275</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20070531</span></div><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">SOLEXA, INC., CALIFORNIA</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Apr 19, 2007</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">8</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">May 13, 2005</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">SOLEXA, INC., CALIFORNIA</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNOR:LYNX THERAPEUTICS, INC.;REEL/FRAME:016547/0784</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20050304</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">May 12, 2003</td><td class="patent-data-table-td ">SULP</td><td class="patent-data-table-td ">Surcharge for late payment</td><td class="patent-data-table-td "></td></tr><tr><td class="patent-data-table-td patent-date-value">May 12, 2003</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">4</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">May 7, 2003</td><td class="patent-data-table-td ">REMI</td><td class="patent-data-table-td ">Maintenance fee reminder mailed</td><td class="patent-data-table-td "></td></tr><tr><td class="patent-data-table-td patent-date-value">Jun 10, 1997</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">LYNX THERAPEUTICS, INC., CALIFORNIA</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">MERGER;ASSIGNOR:SPECTRAGEN, INC.;REEL/FRAME:008605/0332</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">19961023</span></div></td></tr></table><div class="patent-section-footer"></div></div><div class="modal-dialog" id="patent-images-lightbox"><div class="patent-lightbox-controls"><div class="patent-lightbox-rotate-controls"><div class="patent-lightbox-rotation-text">Rotate</div><div class="rotate-icon rotate-ccw-icon"></div><div class="rotate-icon rotate-cw-icon"></div></div><div class="patent-lightbox-index-counter"></div><a class="patent-lightbox-fullsize-link" target="_blank">Original Image</a><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_right.png" alt="Next page"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_left.png" alt="Previous page"width="21" height="21" /></div></div><div class="modal-dialog-content"><div class="patent-lightbox-image-holder"><div class="patent-lightbox-placeholder"></div></div></div></div><script>_OC_initPatentsAtb({image_not_available_html: " Image not available"});</script></div></div></div></td></tr></table><script>(function() {var href = window.location.href;if (href.indexOf('?') !== -1) {var parameters = href.split('?')[1].split('&');for (var i = 0; i < parameters.length; i++) {var param = parameters[i].split('=');if (param[0] == 'focus') {var elem = document.getElementById(param[1]);if (elem) {elem.focus();}}}}})();</script><script>_OC_addFlags({LockSrc:"/books/javascript/lock_8a2b04e7bf975d5171d8e4c0b6365c7a.js", Host:"http://www.google.com/", IsBooksRentalEnabled:1, IsWebstoreDisplayCaseEnabled:1, IsObfuscationEnabled:1, IsBrowsingHistoryEnabled:1, IsWebReaderSvgEnabled:0, IsGeoLayerEnabled:1, IsImageModeNotesEnabled:1, IsCopyMenuItemEnabled:1, IsGiftingEnabled:0, IsWebReaderUniversalPaginatorEnabled:0, IsOfflineBubbleEnabled:1, IsReaderEnabledForPlayRequests:1, IsFutureOnSaleVolumesEnabled:1, IsOfflineRestrictedCopyEnabled:1, IsBooksUnifiedLeftNavEnabled:1, IsRestrictedCopyEnabled:1, IsZipitFolderCollectionEnabled:1, IsEndOfSampleRecommendationsEnabled:1, IsRatingsOnBookcardsEnabled:1, IsAdsDisabled:0, IsIframePageDisplayEnabled:0, IsEmbeddedMediaEnabled:1, IsImageModeAnnotationsEnabled:1, IsMyLibraryGooglePlusEnabled:1, IsImagePageProviderEnabled:0, IsBookcardListPriceSmall:0, IsInternalUser:0, IsBooksShareButtonEnabled:0, IsPreOrdersEnabled:0, IsDisabledRandomBookshelves:0, WebstoreDisplayCasePosition:3});_OC_Run({"enable_p13n":false,"add_vol_to_collection_base_url":"http://www.google.com/patents?op=add\u0026sig=ACfU3U3Xkrh2qtnlXB5Li79QrvwClcaTAQ\u0026id=Bp9MBAABERAJ","remove_vol_from_collection_base_url":"http://www.google.com/patents?op=remove\u0026sig=ACfU3U2tv_iBW_nK4Ml5X08lbxYh0f2qxw\u0026id=Bp9MBAABERAJ","logged_in":false,"p13n_save_user_settings_url":"http://www.google.com/patents?op=edit_user_settings\u0026sig=ACfU3U38afaCVRjT30A6nsFBST9FpJNcng","is_cobrand":false,"sign_in_url":"https://www.google.com/accounts/Login?service=\u0026continue=http://www.google.com/patents%3Fhl%3Den\u0026hl=en","is_play_enabled":true}, {"volume_id":"","is_ebook":true,"volumeresult":{"has_flowing_text":false,"has_scanned_text":true,"can_download_pdf":false,"can_download_epub":false,"is_pdf_drm_enabled":false,"is_epub_drm_enabled":false,"download_pdf_url":"http://www.google.com/patents/download/Sequencing_based_on_repeated_cycles_of_d.pdf?id=Bp9MBAABERAJ\u0026output=pdf\u0026sig=ACfU3U3qVXiQ9kGX9NNoMjiHQIs0zTPoDA"},"sample_url":"http://www.google.com/patents/reader?id=Bp9MBAABERAJ\u0026printsec=frontcover\u0026output=reader\u0026source=gbs_atb_hover","is_browsable":true,"is_public_domain":true}, {});</script><div id="footer_table" style="font-size:83%;text-align:center;position:relative;top:20px;height:4.5em;margin-top:2em"><div style="margin-bottom:8px"><a href=http://www.google.com/><nobr>Google&nbsp;Home</nobr></a> - <a href=//www.google.com/patents/sitemap/><nobr>Sitemap</nobr></a> - <a href=http://www.google.com/googlebooks/uspto.html><nobr>USPTO Bulk Downloads</nobr></a> - <a href=/intl/en/privacy/><nobr>Privacy Policy</nobr></a> - <a href=/intl/en/policies/terms/><nobr>Terms of Service</nobr></a> - <a href=https://support.google.com/faqs/answer/2539193?hl=en><nobr>About Google Patents</nobr></a> - <a href="http://www.google.com/tools/feedback/intl/en/error.html" onclick="try{_OC_startFeedback({productId: '72792',locale: 'en'});return false;}catch(e){}"><nobr>Send Feedback</nobr></a></div><span>Data provided by IFI CLAIMS Patent Services</span><br><span >&copy;2012 Google</span></div> <script type="text/javascript">var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));</script><script type="text/javascript">var pageTracker = _gat._getTracker("UA-27188110-1");pageTracker._setCookiePath("/patents/");pageTracker._trackPageview();</script> </body></html>