<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN""http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html><head><title>Patent US6018026 - For rapid, inexpensive and adaptable high volume screening of ligands to ... - Google Patents</title><script>(function(){(function(){function e(a){this.t={};this.tick=function(a,c,b){var d=void 0!=b?b:(new Date).getTime();this.t[a]=[d,c];if(void 0==b)try{window.console.timeStamp("CSI/"+a)}catch(e){}};this.tick("start",null,a)}var a;window.performance&&(a=window.performance.timing);var f=a?new e(a.responseStart):new e;window.jstiming={Timer:e,load:f};if(a){var c=a.navigationStart,d=a.responseStart;0<c&&d>=c&&(window.jstiming.srt=d-c)}if(a){var b=window.jstiming.load;0<c&&d>=c&&(b.tick("_wtsrt",void 0,c),b.tick("wtsrt_",
"_wtsrt",d),b.tick("tbsd_","wtsrt_"))}try{a=null,window.chrome&&window.chrome.csi&&(a=Math.floor(window.chrome.csi().pageT),b&&0<c&&(b.tick("_tbnd",void 0,window.chrome.csi().startE),b.tick("tbnd_","_tbnd",c))),null==a&&window.gtbExternal&&(a=window.gtbExternal.pageT()),null==a&&window.external&&(a=window.external.pageT,b&&0<c&&(b.tick("_tbnd",void 0,window.external.startE),b.tick("tbnd_","_tbnd",c))),a&&(window.jstiming.pt=a)}catch(g){}})();})();
</script><link rel="stylesheet" href="/patents/css/_8a2b04e7bf975d5171d8e4c0b6365c7a/kl_intl_patents_bundle.css" type="text/css" /><script src="/books/javascript/atb_8a2b04e7bf975d5171d8e4c0b6365c7a__en.js"></script><script>function googleTranslateElementInit() {new google.translate.TranslateElement({pageLanguage: "en",gaTrack: true,gaId: "UA-27188110-1",multilanguagePage: true});}</script><script src="//translate.google.com/translate_a/element.js?cb=googleTranslateElementInit"></script><meta name="DC.type" content="Patent"><meta name="DC.title" content="For rapid, inexpensive and adaptable high volume screening of ligands to receptors; for determining the presence of cell growth factors in samples; for purifying cell growth factors and isoforms"><meta name="DC.contributor" content="Andrzej Z. Sledziewski" scheme="inventor"><meta name="DC.contributor" content="Lillian Anne Bell" scheme="inventor"><meta name="DC.contributor" content="Wayne R. Kindsvogel" scheme="inventor"><meta name="DC.contributor" content="Zymogenetics, Inc." scheme="assignee"><meta name="DC.date" content="1997-11-26" scheme="dateSubmitted"><meta name="DC.description" content="Methods for producing secreted receptor analogs and biologically active peptide dimers are disclosed. The methods for producing secreted receptor analogs and biologically active peptide dimers utilize a DNA sequence encoding a receptor analog or a peptide requiring dimerization for biological activity joined to a dimerizing protein. The receptor analog includes a ligand-binding domain. Polypeptides comprising essentially the extracellular domain of a human PDGF receptor fused to dimerizing proteins, the portion being capable of binding human PDGF or an isoform thereof, are also disclosed. The polypeptides may be used within methods for determining the presence of and for purifying human PDGF or isoforms thereof."><meta name="DC.date" content="2000-1-25" scheme="issued"><meta name="DC.relation" content="EP:0088994" scheme="references"><meta name="DC.relation" content="EP:0116201:A1" scheme="references"><meta name="DC.relation" content="EP:0120694:A2" scheme="references"><meta name="DC.relation" content="EP:0162319:A2" scheme="references"><meta name="DC.relation" content="EP:0171496" scheme="references"><meta name="DC.relation" content="EP:0173494:A2" scheme="references"><meta name="DC.relation" content="EP:0239400:A2" scheme="references"><meta name="DC.relation" content="EP:0244221:A1" scheme="references"><meta name="DC.relation" content="EP:0256654:A2" scheme="references"><meta name="DC.relation" content="EP:0315062:A2" scheme="references"><meta name="DC.relation" content="EP:0327369:A2" scheme="references"><meta name="DC.relation" content="US:4474893" scheme="references"><meta name="DC.relation" content="US:4599311" scheme="references"><meta name="DC.relation" content="US:4761371" scheme="references"><meta name="DC.relation" content="US:4766073" scheme="references"><meta name="DC.relation" content="US:4816397" scheme="references"><meta name="DC.relation" content="US:4816567" scheme="references"><meta name="DC.relation" content="US:5015571" scheme="references"><meta name="DC.relation" content="US:5098833" scheme="references"><meta name="DC.relation" content="US:5116964" scheme="references"><meta name="DC.relation" content="US:5155027" scheme="references"><meta name="DC.relation" content="US:5750375" scheme="references"><meta name="DC.relation" content="WO:1986001533:A1" scheme="references"><meta name="DC.relation" content="WO:1990006953:A2" scheme="references"><meta name="citation_reference" content="Alber and Kawasaki, J. Mol. Appl. Genet. 1 : 419 434, 1982."><meta name="citation_reference" content="Alber and Kawasaki, J. Mol. Appl. Genet. 1: 419-434, 1982."><meta name="citation_reference" content="Bailon et al., Bio/Technology 5 :1195 1198, 1987."><meta name="citation_reference" content="Bailon et al., Bio/Technology 5:1195-1198, 1987."><meta name="citation_reference" content="Ballou et al., J. Biol. Chem. 255 :5986 5991, 1980."><meta name="citation_reference" content="Ballou et al., J. Biol. Chem. 255:5986-5991, 1980."><meta name="citation_reference" content="Beggs, Nature : 104 108, 1978."><meta name="citation_reference" content="Beggs, Nature: 104-108, 1978."><meta name="citation_reference" content="Boni Schnetzler and Pilch, Proc. Natl. Acad. Sci. USA 84 :7832 7836, 1987."><meta name="citation_reference" content="Boni-Schnetzler and Pilch, Proc. Natl. Acad. Sci. USA 84:7832-7836, 1987."><meta name="citation_reference" content="Broach et al., Gene 8 :121 133, 1979."><meta name="citation_reference" content="Broach et al., Gene 8:121-133, 1979."><meta name="citation_reference" content="Capon et al., Nature 337 :525 531, 1989."><meta name="citation_reference" content="Capon et al., Nature 337:525-531, 1989."><meta name="citation_reference" content="Carlson et al., Mol. Cell. Biol. 3 :439 447, 1983."><meta name="citation_reference" content="Carlson et al., Mol. Cell. Biol. 3:439-447, 1983."><meta name="citation_reference" content="Claesson Welsh et al., Mol. Cell. Biol. 8:3476 3486, 1988."><meta name="citation_reference" content="Claesson-Welsh et al., Mol. Cell. Biol. 8:3476-3486, 1988."><meta name="citation_reference" content="Clarke et al., Meth. Enzym. 68 :436 442, 1979."><meta name="citation_reference" content="Clarke et al., Meth. Enzym. 68:436-442, 1979."><meta name="citation_reference" content="Derynck et al., Nature 316 :701 705, 1985."><meta name="citation_reference" content="Derynck et al., Nature 316:701-705, 1985."><meta name="citation_reference" content="Ebina et al., Cell 40 :747 758, 1985."><meta name="citation_reference" content="Ebina et al., Cell 40:747-758, 1985."><meta name="citation_reference" content="Ellis et al., J. Cell Biol. 150 :14a, 1987."><meta name="citation_reference" content="Ellis et al., J. Cell Biol. 150:14a, 1987."><meta name="citation_reference" content="Escobedo et al., Science 240 :1532 1534, 1988."><meta name="citation_reference" content="Escobedo et al., Science 240:1532-1534, 1988."><meta name="citation_reference" content="Gascoigne et al., Proc. Natl. Acad. Sci. USA 84 :2936 2940, 1987."><meta name="citation_reference" content="Gascoigne et al., Proc. Natl. Acad. Sci. USA 84:2936-2940, 1987."><meta name="citation_reference" content="Gronwald et al., Proc. Natl. Acad. Sci. USA 85 :3435 3439, 1988."><meta name="citation_reference" content="Gronwald et al., Proc. Natl. Acad. Sci. USA 85:3435-3439, 1988."><meta name="citation_reference" content="Grosschedl and Baltimore, Cell 41 : 885 897, 1985."><meta name="citation_reference" content="Grosschedl and Baltimore, Cell 41: 885-897, 1985."><meta name="citation_reference" content="Hart et al., Science 240 :1529 1531, 1988."><meta name="citation_reference" content="Hart et al., Science 240:1529-1531, 1988."><meta name="citation_reference" content="Heldin et al., Embo J. 7 :1387 1393, 1988."><meta name="citation_reference" content="Heldin et al., Embo J. 7:1387-1393, 1988."><meta name="citation_reference" content="Hieter et al., Cell 22 : 197 207, 1980."><meta name="citation_reference" content="Hieter et al., Cell 22: 197-207, 1980."><meta name="citation_reference" content="Honjo et al., Cell 18 :559 568, 1979."><meta name="citation_reference" content="Honjo et al., Cell 18:559-568, 1979."><meta name="citation_reference" content="Kuwana et al., Biochemical and Biophysical Research Communications 149 (3) : 960 968, 1987."><meta name="citation_reference" content="Kuwana et al., Biochemical and Biophysical Research Communications 149 (3) : 960-968, 1987."><meta name="citation_reference" content="Livneh et al., J. Biol. Chem. 261 : 12490 12497, 1986."><meta name="citation_reference" content="Livneh et al., J. Biol. Chem. 261: 12490-12497, 1986."><meta name="citation_reference" content="Loh et al., Cell 33 :85 93, 1983."><meta name="citation_reference" content="Loh et al., Cell 33:85-93, 1983."><meta name="citation_reference" content="Maddon et al., Cell 42 : 93 104, 1985."><meta name="citation_reference" content="Maddon et al., Cell 42: 93-104, 1985."><meta name="citation_reference" content="Maddon et al., Cell 47 : 333 348, 1986."><meta name="citation_reference" content="Maddon et al., Cell 47: 333-348, 1986."><meta name="citation_reference" content="Mariuzza and Winter, J. Biol. Chem. 264 :7310 7316, 1989."><meta name="citation_reference" content="Mariuzza and Winter, J. Biol. Chem. 264:7310-7316, 1989."><meta name="citation_reference" content="Marx, Science 229 :455 456, 1985."><meta name="citation_reference" content="Marx, Science 229:455-456, 1985."><meta name="citation_reference" content="Munro and Pelham, EMBO J. 3 :3087 3093, 1984."><meta name="citation_reference" content="Munro and Pelham, EMBO J. 3:3087-3093, 1984."><meta name="citation_reference" content="Neuberger et al., Nature 312 : 604 609, 1984."><meta name="citation_reference" content="Neuberger et al., Nature 312: 604-609, 1984."><meta name="citation_reference" content="Oi and Morrison Biotechniques 4 :214 221, 1986."><meta name="citation_reference" content="Oi and Morrison Biotechniques 4:214-221, 1986."><meta name="citation_reference" content="Radeke et al., Nature 325 593 597, 1987."><meta name="citation_reference" content="Radeke et al., Nature 325593-597, 1987."><meta name="citation_reference" content="Riedel et al., Nature 324 :68 70, 1986."><meta name="citation_reference" content="Riedel et al., Nature 324:68-70, 1986."><meta name="citation_reference" content="Riedel et al., Protein Engineering 1 (3):237, 1987."><meta name="citation_reference" content="Riedel et al., Protein Engineering 1(3):237, 1987."><meta name="citation_reference" content="Riedel et al., Science 236 :197 200, 1987."><meta name="citation_reference" content="Riedel et al., Science 236:197-200, 1987."><meta name="citation_reference" content="Roth et al., J. Cell. Biol. 102 :1271 1283, 1986."><meta name="citation_reference" content="Roth et al., J. Cell. Biol. 102:1271-1283, 1986."><meta name="citation_reference" content="Rubin et al., J. Immun. 135 :3172 3177, 1985."><meta name="citation_reference" content="Rubin et al., J. Immun. 135:3172-3177, 1985."><meta name="citation_reference" content="Russell et al., Nature 304 : 652 654, 1983."><meta name="citation_reference" content="Russell et al., Nature 304: 652-654, 1983."><meta name="citation_reference" content="Smith et al., Science 238 :1704 1707, 1987."><meta name="citation_reference" content="Smith et al., Science 238:1704-1707, 1987."><meta name="citation_reference" content="Struhl et al., Proc. Natl. Acad. Sci. USA 76 :1035 1039, 1979."><meta name="citation_reference" content="Struhl et al., Proc. Natl. Acad. Sci. USA 76:1035-1039, 1979."><meta name="citation_reference" content="Traunecker et al., Highly efficient neutralization of HIV with recombinant CD4 immunoglobulin molecules, Nature, 339: 68 70, May 4, 1989."><meta name="citation_reference" content="Traunecker et al., Highly efficient neutralization of HIV with recombinant CD4-immunoglobulin molecules, Nature, 339: 68-70, May 4, 1989."><meta name="citation_reference" content="Treiger et al., J. Immunol. 136 :4099 4105, 1986."><meta name="citation_reference" content="Treiger et al., J. Immunol. 136:4099-4105, 1986."><meta name="citation_reference" content="Ullrich et al., Nature 309 :418 425, 1984."><meta name="citation_reference" content="Ullrich et al., Nature 309:418-425, 1984."><meta name="citation_reference" content="Ullrich et al., Nature 313 : 756 761, 1985."><meta name="citation_reference" content="Ullrich et al., Nature 313: 756-761, 1985."><meta name="citation_reference" content="von Heijne, Eur. J. Biochem. 133 :17 21, 1983."><meta name="citation_reference" content="von Heijne, Eur. J. Biochem. 133:17-21, 1983."><meta name="citation_reference" content="von Heijne, Nucl Acids Res. 14 : 4683 4690, 1986."><meta name="citation_reference" content="von Heijne, Nucl Acids Res. 14: 4683-4690, 1986."><meta name="citation_reference" content="Wood et al., Nature 314 :446 449, 1985."><meta name="citation_reference" content="Wood et al., Nature 314:446-449, 1985."><meta name="citation_reference" content="Yarden et al., Nature 323 :226 232, 1986."><meta name="citation_reference" content="Yarden et al., Nature 323:226-232, 1986."><meta name="citation_patent_number" content="US:6018026"><meta name="citation_patent_application_number" content="US:08/980,400"><link rel="canonical" href="http://www.google.com/patents/US6018026"/><meta property="og:url" content="http://www.google.com/patents/US6018026"/><meta name="title" content="Patent US6018026 - For rapid, inexpensive and adaptable high volume screening of ligands to receptors; for determining the presence of cell growth factors in samples; for purifying cell growth factors and isoforms"/><meta name="description" content="Methods for producing secreted receptor analogs and biologically active peptide dimers are disclosed. The methods for producing secreted receptor analogs and biologically active peptide dimers utilize a DNA sequence encoding a receptor analog or a peptide requiring dimerization for biological activity joined to a dimerizing protein. The receptor analog includes a ligand-binding domain. Polypeptides comprising essentially the extracellular domain of a human PDGF receptor fused to dimerizing proteins, the portion being capable of binding human PDGF or an isoform thereof, are also disclosed. The polypeptides may be used within methods for determining the presence of and for purifying human PDGF or isoforms thereof."/><meta property="og:title" content="Patent US6018026 - For rapid, inexpensive and adaptable high volume screening of ligands to receptors; for determining the presence of cell growth factors in samples; for purifying cell growth factors and isoforms"/><meta property="og:type" content="book"/><meta property="og:site_name" content="Google Books"/><meta property="og:image" content="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><link rel="image_src" href="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><script>(function(){try{var aa=function(a,b,c,d){d=d||{};d._sn=["cfg",b,c].join(".");window.gbar.logger.ml(a,d)};var g=window.gbar=window.gbar||{},l=window.gbar.i=window.gbar.i||{},m={},n;function _tvn(a,b){var c=parseInt(a,10);return isNaN(c)?b:c}function _tvf(a,b){var c=parseFloat(a);return isNaN(c)?b:c}function _tvv(a){return!!a}function p(a,b,c){(c||g)[a]=b}g.bv={n:_tvn("2",0),r:"",f:".67.",e:"0",m:_tvn("0",1)};
function q(a,b,c){var d="on"+b;if(a.addEventListener)a.addEventListener(b,c,!1);else if(a.attachEvent)a.attachEvent(d,c);else{var f=a[d];a[d]=function(){var a=f.apply(this,arguments),b=c.apply(this,arguments);return void 0==a?b:void 0==b?a:b&&a}}}var s=function(a){return function(){return g.bv.m==a}},ba=s(1),ca=s(2);p("sb",ba);p("kn",ca);l.a=_tvv;l.b=_tvf;l.c=_tvn;l.i=aa;var da=window.gbar.i.i;var t,u,v,w;function ea(a){v=a}function fa(a){var b;if(b=v&&window.encodeURIComponent)b=a.href,b=!b.match(/^http[s]?:\/\/accounts\.google\.[^/]*\/ClearSID/i)&&!b.match(/^http[s]?:\/\/[^/]*\/accounts\/ClearSID/i);if(b=b&&encodeURIComponent(v()))a.href=a.href.replace(/([?&]continue=)[^&]*/,"$1"+b)}function ga(a){window.gApplication&&(a.href=window.gApplication.getTabUrl(a.href))}
function ha(a){var b=document.forms[0].q,c=window.encodeURIComponent&&b&&b.value,b=b&&b.placeholder;c&&c!=b&&(a.href=a.href.replace(/([?&])q=[^&]*|$/,function(a,b){return(b||"&")+"q="+encodeURIComponent(c)}))}n=l.a("")?ga:ha;
function x(a,b,c,d,f,e){var h=document.getElementById(a);if(h){var k=h.style;k.left=d?"auto":b+"px";k.right=d?b+"px":"auto";k.top=c+"px";k.visibility=u?"hidden":"visible";f&&e?(k.width=f+"px",k.height=e+"px"):(x(t,b,c,d,h.offsetWidth,h.offsetHeight),u=u?"":a)}}
var y=[],ia=function(a,b){y.push(b)},ja=function(a){a=a||window.event;var b=a.target||a.srcElement;a.cancelBubble=!0;null==t&&(a=document.createElement(Array.every||window.createPopup?"iframe":"div"),a.frameBorder="0",t=a.id="gbs",a.src="javascript:''",b.parentNode.appendChild(a),q(document,"click",z));var c=b,b=0;"gb3"!=c.className&&(c=c.parentNode);a=c.getAttribute("aria-owns")||"gbi";var d=c.offsetWidth,f=20<c.offsetTop?46:24;document.getElementById("tphdr")&&(f-=3);var e=!1;do b+=c.offsetLeft||
0;while(c=c.offsetParent);var c=(document.documentElement.clientWidth||document.body.clientWidth)-b-d,h,d=document.body,k=document.defaultView;k&&k.getComputedStyle?(d=k.getComputedStyle(d,""))&&(h=d.direction):h=d.currentStyle?d.currentStyle.direction:d.style.direction;h="rtl"==h;if("gbi"==a){for(d=0;k=y[d++];)k();A(null,window.navExtra);h&&(b=c,e=!0)}else h||(b=c,e=!0);u!=a&&z();x(a,b,f,e)},z=function(){u&&x(u,0,0)},A=function(a,b){var c,d=document.getElementById("gbi"),f=a;f||(f=d.firstChild);
for(;b&&(c=b.pop());){var e=d,h=c,k=f;w||(w="gb2");e.insertBefore(h,k).className=w}},ka=function(a,b,c){if((b=document.getElementById(b))&&a){a.className="gb4";var d=document.createElement("span");d.appendChild(a);d.appendChild(document.createTextNode(" | "));d.id=c;b.appendChild(d)}},la=function(){return document.getElementById("gb_70")},ma=function(){return!!u};p("qs",n);p("setContinueCb",ea);p("pc",fa);p("tg",ja);p("close",z);p("addLink",ka);p("almm",A);p("si",la);p("adh",ia);p("op",ma);var B=function(){},C=function(){},F=function(a){var b=new Image,c=D;b.onerror=b.onload=b.onabort=function(){try{delete E[c]}catch(a){}};E[c]=b;b.src=a;D=c+1},E=[],D=0;p("logger",{il:C,ml:B,log:F});var G=window.gbar.logger;var H={},na={},I=[],oa=l.b("0.1",.1),pa=l.a("1",!0),qa=function(a,b){I.push([a,b])},ra=function(a,b){H[a]=b},sa=function(a){return a in H},J={},K=function(a,b){J[a]||(J[a]=[]);J[a].push(b)},ta=function(a){K("m",a)},L=function(a,b){var c=document.createElement("script");c.src=a;c.async=pa;Math.random()<oa&&(c.onerror=function(){c.onerror=null;B(Error("Bundle load failed: name="+(b||"UNK")+" url="+a))});(document.getElementById("xjsc")||document.getElementsByTagName("body")[0]||
document.getElementsByTagName("head")[0]).appendChild(c)},N=function(a){for(var b=0,c;(c=I[b])&&c[0]!=a;++b);!c||c[1].l||c[1].s||(c[1].s=!0,M(2,a),c[1].url&&L(c[1].url,a),c[1].libs&&m.d&&m.d(c[1].libs))},O=function(a){K("gc",a)},P=null,ua=function(a){P=a},M=function(a,b,c){if(P){a={t:a,b:b};if(c)for(var d in c)a[d]=c[d];try{P(a)}catch(f){}}};p("mdc",H);p("mdi",na);p("bnc",I);p("qGC",O);p("qm",ta);p("qd",J);p("lb",N);p("mcf",ra);p("bcf",qa);p("aq",K);p("mdd","");p("has",sa);
p("trh",ua);p("tev",M);var Q=l.b("0.1",.001),R=0;
function _mlToken(a,b){try{if(1>R){R++;var c,d=a,f=b||{},e=encodeURIComponent,h=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&jexpid=",e("17483"),"&srcpg=",e("prop=22"),"&jsr=",Math.round(1/Q),"&ogev=",e("7y_uU7CbH7HjsATCpIEI"),"&ogf=",g.bv.f,"&ogrp=",e("1"),"&ogv=",e("1407723702.0"),"&oggv="+e("es_plusone_gc_20140723.0_p0"),"&ogd=",e("com"),"&ogc=",e("DEU"),"&ogl=",e("en")];f._sn&&(f._sn="og."+
f._sn);for(var k in f)h.push("&"),h.push(e(k)),h.push("="),h.push(e(f[k]));h.push("&emsg=");h.push(e(d.name+":"+d.message));var r=h.join("");S(r)&&(r=r.substr(0,2E3));c=r;var Aa=window.gbar.logger._aem(a,c);F(Aa)}}catch(Na){}}var S=function(a){return 2E3<=a.length},va=function(a,b){return b};function T(a){B=a;p("_itl",S,G);p("_aem",va,G);p("ml",B,G);a={};H.er=a}l.a("")?T(function(a){throw a;}):l.a("1")&&Math.random()<Q&&T(_mlToken);I.push(["m",{url:"//ssl.gstatic.com/gb/js/scm_7385cc5883250b43a39405734c1bea59.js"}]);g.mcf("c",{});g.sg={c:""};if(l.a("1")){var wa=l.a("");I.push(["gc",{auto:wa,url:"//ssl.gstatic.com/gb/js/abc/gci_91f30755d6a6b787dcc2a4062e6e9824.js",libs:"googleapis.client:plusone:gapi.iframes"}]);var xa={version:"gci_91f30755d6a6b787dcc2a4062e6e9824.js",index:"",lang:"en"};H.gc=xa;var U=function(a){window.googleapis&&window.iframes?a&&a():(a&&O(a),N("gc"))};p("lGC",U);l.a("1")&&p("lPWF",U)};window.__PVT="";if(l.a("1")&&l.a("1")){var V=function(a){U(function(){K("pw",a);N("pw")})};p("lPW",V);I.push(["pw",{url:"//ssl.gstatic.com/gb/js/abc/pwm_45f73e4df07a0e388b0fa1f3d30e7280.js"}]);var W=[],ya=function(a){W[0]=a},za=function(a,b){var c=b||{};c._sn="pw";B(a,c)},Ba={signed:W,elog:za,base:"https://plusone.google.com/u/0",loadTime:(new Date).getTime()};H.pw=Ba;var X=function(a,b){for(var c=b.split("."),d=function(){var b=arguments;a(function(){for(var a=g,d=0,e=c.length-1;d<e;++d)a=a[c[d]];a[c[d]].apply(a,b)})},f=g,e=0,h=c.length-1;e<h;++e)f=
f[c[e]]=f[c[e]]||{};return f[c[e]]=d};X(V,"pw.clk");X(V,"pw.hvr");p("su",ya,g.pw)};function Ca(){function a(){for(var b;(b=e[h++])&&"m"!=b[0]&&!b[1].auto;);b&&(M(2,b[0]),b[1].url&&L(b[1].url,b[0]),b[1].libs&&m.d&&m.d(b[1].libs));h<e.length&&setTimeout(a,0)}function b(){0<f--?setTimeout(b,0):a()}var c=l.a("1"),d=l.a(""),f=3,e=I,h=0,k=window.gbarOnReady;if(k)try{k()}catch(r){da(r,"ml","or")}d?p("ldb",a):c?q(window,"load",b):b()}p("rdl",Ca);var Da={D:1,H:2,da:3,p:4,W:5,M:6,F:7,g:8,ha:9,U:10,L:11,T:12,S:13,N:14,Q:15,P:16,fa:17,w:18,O:19,ga:20,ea:21,u:22,G:23,ja:24,ka:25,ia:26,A:27,j:28,o:29,k:30,ca:31,Z:32,$:33,J:34,K:35,ba:36,aa:37,Y:38,B:39,R:40,v:41,X:42,V:43,h:48,C:49,I:500},Y=[1,2,3,4,5,6,9,10,11,13,14,28,29,30,34,35,37,38,39,40,41,42,43,48,49,500];var Z=l.b("0.001",1E-4),Ea=l.b("1",1),Fa=!1,Ga=!1;if(l.a("1")){var Ha=Math.random();Ha<=Z&&(Fa=!0);Ha<=Ea&&(Ga=!0)}var Ia=Da,$=null;function Ja(){var a=0,b=function(b,d){l.a(d)&&(a|=b)};b(1,"");b(2,"");b(4,"");b(8,"");return a}
function Ka(a,b){var c=Z,d=Fa,f;f=a;if(!$){$={};for(var e=0;e<Y.length;e++){var h=Y[e];$[h]=!0}}if(f=!!$[f])c=Ea,d=Ga;if(d){d=encodeURIComponent;g.rp?(f=g.rp(),f="-1"!=f?f:"1"):f="1";c=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&oge=",a,"&ogex=",d("17483"),"&ogev=",d("7y_uU7CbH7HjsATCpIEI"),"&ogf=",g.bv.f,"&ogp=",d("22"),"&ogrp=",d(f),"&ogsr=",Math.round(1/c),"&ogv=",d("1407723702.0"),"&oggv="+
d("es_plusone_gc_20140723.0_p0"),"&ogd=",d("com"),"&ogl=",d("en"),"&ogc=",d("DEU"),"&ogus=",Ja()];if(b){"ogw"in b&&(c.push("&ogw="+b.ogw),delete b.ogw);var k;f=b;e=[];for(k in f)0!=e.length&&e.push(","),e.push(La(k)),e.push("."),e.push(La(f[k]));k=e.join("");""!=k&&(c.push("&ogad="),c.push(d(k)))}F(c.join(""))}}function La(a){"number"==typeof a&&(a+="");return"string"==typeof a?a.replace(".","%2E").replace(",","%2C"):a}C=Ka;p("il",C,G);var Ma={};H.il=Ma;setTimeout(function(){C(Ia.g)},0);}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var b=window.gbar.i.i;var c=window.gbar;var f=function(d){try{var a=document.getElementById("gbom");a&&d.appendChild(a.cloneNode(!0))}catch(e){b(e,"omas","aomc")}};c.aomc=f;}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var a=window.gbar;a.mcf("pm",{p:""});}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{window.gbar.rdl();}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
if (window['_OC_timingAction']) {window['_OC_timingAction']('patents_refpage');}</script><style>#gbar,#guser{font-size:13px;padding-top:1px !important;}#gbar{float:left;height:22px}#guser{padding-bottom:7px !important;text-align:right}.gbh,.gbd{border-top:1px solid #c9d7f1;font-size:1px}.gbh{height:0;position:absolute;top:24px;width:100%}#gbs,.gbm{background:#fff;left:0;position:absolute;text-align:left;visibility:hidden;z-index:1000}.gbm{border:1px solid;border-color:#c9d7f1 #36c #36c #a2bae7;z-index:1001}.gb1{margin-right:.5em}.gb1,.gb3{zoom:1}.gb2{display:block;padding:.2em .5em}.gb2,.gb3{text-decoration:none !important;border-bottom:none}a.gb1,a.gb4{text-decoration:underline !important}a.gb1,a.gb2,a.gb3,a.gb4{color:#00c !important}.gbi .gb3,.gbi .gb2,.gbi .gb4{color:#dd8e27 !important}.gbf .gb3,.gbf .gb2,.gbf .gb4{color:#900 !important}a.gb2:hover{background:#36c;color:#fff !important}#gbar .gbz0l{color:#000 !important;cursor:default;font-weight:bold;text-decoration:none !important}
#gbar { padding:.3em .6em !important;}</style></head><body  topmargin="3" marginheight="3"><div id=gbar><nobr><a onclick=gbar.qs(this);gbar.logger.il(1,{t:1}); class=gb1 id=gb_1 href="https://www.google.com/search?sa=N&tab=tw">Search</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:2}); class=gb1 id=gb_2 href="http://www.google.com/search?hl=en&tbm=isch&source=og&sa=N&tab=ti">Images</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:8}); class=gb1 id=gb_8 href="http://maps.google.com/maps?hl=en&sa=N&tab=tl">Maps</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:78}); class=gb1 id=gb_78 href="https://play.google.com/?hl=en&sa=N&tab=t8">Play</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:36}); class=gb1 id=gb_36 href="http://www.youtube.com/results?sa=N&tab=t1">YouTube</a> <a onclick=gbar.logger.il(1,{t:5}); class=gb1 id=gb_5 href="http://news.google.com/nwshp?hl=en&tab=tn">News</a> <a onclick=gbar.logger.il(1,{t:23}); class=gb1 id=gb_23 href="https://mail.google.com/mail/?tab=tm">Gmail</a> <a onclick=gbar.logger.il(1,{t:25}); class=gb1 id=gb_25 href="https://drive.google.com/?tab=to">Drive</a> <a class=gb3 href="http://www.google.com/intl/en/options/" onclick="this.blur();gbar.tg(event);return !1" aria-haspopup=true><u>More</u> <small>&#9660;</small></a><div class=gbm id=gbi><a onclick=gbar.logger.il(1,{t:24}); class=gb2 id=gb_24 href="https://www.google.com/calendar?tab=tc">Calendar</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:51}); class=gb2 id=gb_51 href="http://translate.google.com/?hl=en&sa=N&tab=tT">Translate</a><a onclick=gbar.logger.il(1,{t:17}); class=gb2 id=gb_17 href="http://www.google.com/mobile/?hl=en&tab=tD">Mobile</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:10}); class=gb2 id=gb_10 href="http://www.google.com/search?hl=en&tbo=u&tbm=bks&source=og&sa=N&tab=tp">Books</a><a onclick=gbar.logger.il(1,{t:212}); class=gb2 id=gb_212 href="https://wallet.google.com/manage/?tab=ta">Wallet</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:6}); class=gb2 id=gb_6 href="http://www.google.com/search?hl=en&tbo=u&tbm=shop&source=og&sa=N&tab=tf">Shopping</a><a onclick=gbar.logger.il(1,{t:30}); class=gb2 id=gb_30 href="http://www.blogger.com/?tab=tj">Blogger</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:27}); class=gb2 id=gb_27 href="http://www.google.com/finance?sa=N&tab=te">Finance</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:31}); class=gb2 id=gb_31 href="https://plus.google.com/photos?sa=N&tab=tq">Photos</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:12}); class=gb2 id=gb_12 href="http://www.google.com/search?hl=en&tbo=u&tbm=vid&source=og&sa=N&tab=tv">Videos</a><div class=gb2><div class=gbd></div></div><a onclick=gbar.logger.il(1,{t:66}); href="http://www.google.com/intl/en/options/" class=gb2>Even more &raquo;</a></div></nobr></div><div id=guser width=100%><nobr><span id=gbn class=gbi></span><span id=gbf class=gbf></span><span id=gbe></span><a target=_top id=gb_70 href="https://www.google.com/accounts/Login?service=&continue=http://www.google.com/patents%3Fhl%3Den&hl=en" class=gb4>Sign in</a><div style="display: none"><div class=gbm id=gbd5 aria-owner=gbg5><div class=gbmc><ol id=gbom class=gbmcc></ol></div></div></div></nobr></div><div class=gbh style=left:0></div><div class=gbh style=right:0></div><div role="alert" style="position: absolute; left: 0; right: 0;"><a href="http://www.google.com/patents/us6018026?hl=en&amp;output=html_text" title="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."><img border="0" src="http://www.google.com/images/cleardot.gif"alt="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."></a></div><div id="guser"><nobr></nobr></div><div style="clear:both;"></div><div id="gb-top-search-box" class="gb-top-search-box-small gb-reset"><table><tr><td class="logo"><a href="http://www.google.com/patents" class="logo-link"><img class="logo-img" src="/intl/en/images/logos/google_logo_41.png" alt="Go to Google Books Home" height="41"/></a></td><td><form action="http://www.google.com/search" name="f" id="vheadf" method="get"><span id="hf"></span><input type="hidden" name="tbm" value="pts"/><input type="hidden" name="tbo" value="1"/><input type="hidden" name="hl" value="en"/><table><tr><td><div class="inputs"><table><tr><td><div class="text-input"><input type="text" name="q" id="vheadq" class="text" maxlength="2048" size="31" value="" title="Search Patents" accesskey="s" autocomplete="off"/><script>window._OC_autoDir &&window._OC_autoDir('vheadq', 'tia-vheadq');</script></div></td><td><div class="submit-input"><input name="btnG" class="submit" type="submit" value=""/></div></td></tr></table></div></td><td class="col-ext-links"><div class="ext-links"><a href="http://www.google.com/advanced_patent_search">&lt;nobr&gt;Advanced Patent Search&lt;/nobr&gt;</a></div></td></tr></table></form></td></tr></table></div><div class="kd-appbar"><h2 class="kd-appname"><a href="/patents">Patents</a></h2><div class="kd-buttonbar left" id="left-toolbar-buttons"><a id="appbar-write-review-link" href=""></a><a id="appbar-view-print-sample-link" href=""></a><a id="appbar-view-ebook-sample-link" href=""></a><a id="appbar-patents-prior-art-finder-link" href="https://www.google.com/patents/related/US6018026"></a><a id="appbar-patents-discuss-this-link" href="http://www.google.com/url?id=Z-1NBAABERAJ&amp;q=http://patents.stackexchange.com/redirect/google-patents%3Fpatent%3DUS6018026&amp;usg=AFQjCNFPmUi4pKDQudxw8Ql42W6Ey5c5Zw" data-is-grant="true"></a><a id="appbar-read-patent-link" href="//docs.google.com/viewer?url=patentimages.storage.googleapis.com/pdfs/US6018026.pdf"></a><a id="appbar-download-pdf-link" href="//patentimages.storage.googleapis.com/pdfs/US6018026.pdf"></a></div><div class="kd-buttonbar right" id="right-toolbar-buttons"></div></div><div id="books-microdata" itemscope=""itemtype="http://schema.org/Book"itemid="http://www.google.com/patents/US6018026" style="display:none"><span itemprop="description">Methods for producing secreted receptor analogs and biologically active peptide dimers are disclosed. The methods for producing secreted receptor analogs and biologically active peptide dimers utilize a DNA sequence encoding a receptor analog or a peptide requiring dimerization for biological activity...</span><span itemprop="url">http://www.google.com/patents/US6018026?utm_source=gb-gplus-share</span><span class="main-title" itemprop="name">Patent US6018026 - For rapid, inexpensive and adaptable high volume screening of ligands to receptors; for determining the presence of cell growth factors in samples; for purifying cell growth factors and isoforms</span><img itemprop="image" src="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"alt="Patent US6018026 - For rapid, inexpensive and adaptable high volume screening of ligands to receptors; for determining the presence of cell growth factors in samples; for purifying cell growth factors and isoforms" title="Patent US6018026 - For rapid, inexpensive and adaptable high volume screening of ligands to receptors; for determining the presence of cell growth factors in samples; for purifying cell growth factors and isoforms"></div><div style="display: none"><ol id="ofe-gear-menu-contents" class="gbmcc"><li class="gbe gbmtc"><a class="gbmt goog-menuitem-content" id="" href="http://www.google.com/advanced_patent_search">Advanced Patent Search</a></li></ol></div><table id="viewport_table" cellpadding="0" style="clear:both" cellspacing="0"><tr><td id="viewport_td"><div class=vertical_module_list_row><div id=intl_patents class=about_content><div id=intl_patents_v><table class="patent-bibdata"><tr><td class="patent-bibdata-heading">Publication number</td><td class="single-patent-bibdata">US6018026 A</td></tr><tr><td class="patent-bibdata-heading">Publication type</td><td class="single-patent-bibdata">Grant</td></tr><tr><td class="patent-bibdata-heading">Application number</td><td class="single-patent-bibdata">US 08/980,400</td></tr><tr><td class="patent-bibdata-heading">Publication date</td><td class="single-patent-bibdata">Jan 25, 2000</td></tr><tr><td class="patent-bibdata-heading">Filing date</td><td class="single-patent-bibdata">Nov 26, 1997</td></tr><tr><td class="patent-bibdata-heading">Priority date<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The priority date is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the date listed."></span></td><td class="single-patent-bibdata">Jan 22, 1988</td></tr><tr><td class="patent-bibdata-heading">Fee status<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The fee status is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the status or dates listed."></span></td><td class="single-patent-bibdata">Lapsed</td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Also published as</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/US6291212">US6291212</a>, </span><span class="patent-bibdata-value"><a href="/patents/US6291646">US6291646</a>, </span><span class="patent-bibdata-value"><a href="/patents/US6300099">US6300099</a>, </span><span class="patent-bibdata-value"><a href="/patents/US6323323">US6323323</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20020173621">US20020173621</a></span></span></td></tr><tr class="patent-bibdata-list-row alternate-patent-number"><td class="patent-bibdata-heading">Publication number</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value">08980400, </span><span class="patent-bibdata-value">980400, </span><span class="patent-bibdata-value">US 6018026 A, </span><span class="patent-bibdata-value">US 6018026A, </span><span class="patent-bibdata-value">US-A-6018026, </span><span class="patent-bibdata-value">US6018026 A, </span><span class="patent-bibdata-value">US6018026A</span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Inventors</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Andrzej+Z.+Sledziewski%22">Andrzej Z. Sledziewski</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Lillian+Anne+Bell%22">Lillian Anne Bell</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Wayne+R.+Kindsvogel%22">Wayne R. Kindsvogel</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Original Assignee</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=inassignee:%22Zymogenetics,+Inc.%22">Zymogenetics, Inc.</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Export Citation</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/US6018026.bibtex">BiBTeX</a>, </span><span class="patent-bibdata-value"><a href="/patents/US6018026.enw">EndNote</a>, </span><span class="patent-bibdata-value"><a href="/patents/US6018026.ris">RefMan</a></span></span></td></tr><tr class="patent-internal-links"><td colspan=2><span class="patent-bibdata-value"><a href="#backward-citations">Patent Citations</a> (24),</span> <span class="patent-bibdata-value"><a href="#npl-citations">Non-Patent Citations</a> (96),</span> <span class="patent-bibdata-value"><a href="#forward-citations">Referenced by</a> (66),</span> <span class="patent-bibdata-value"><a href="#classifications">Classifications</a> (23),</span> <span class="patent-bibdata-value"><a href="#legal-events">Legal Events</a> (8)</span> </td></tr><tr><td colspan=2 class="patent-bibdata-external-link-spacer-top"></td></tr><tr class="patent-bibdata-external-link-spacer-bottom"></tr><tr><td colspan=2><span class="patent-bibdata-heading">External Links:&nbsp;</span><span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=Z-1NBAABERAJ&q=http://patft.uspto.gov/netacgi/nph-Parser%3FSect2%3DPTO1%26Sect2%3DHITOFF%26p%3D1%26u%3D/netahtml/PTO/search-bool.html%26r%3D1%26f%3DG%26l%3D50%26d%3DPALL%26RefSrch%3Dyes%26Query%3DPN/6018026&usg=AFQjCNG39QTEDGhkYdDjF3JVBaCKZ5I3Lg">USPTO</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=Z-1NBAABERAJ&q=http://assignments.uspto.gov/assignments/q%3Fdb%3Dpat%26pat%3D6018026&usg=AFQjCNHv3afOUE0V0Zj2UUtyBFYEwXXp_A">USPTO Assignment</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=Z-1NBAABERAJ&q=http://worldwide.espacenet.com/publicationDetails/biblio%3FCC%3DUS%26NR%3D6018026A%26KC%3DA%26FT%3DD&usg=AFQjCNG53NkZ3aj3ItJLLzBdj6Lcsj6QYg">Espacenet</a></span></span></td></tr><tr class="patent-bibdata-group-spacer"></tr></table><div class="number-and-title"><span class="patent-title"><invention-title mxw-id="PT128969038" lang="EN" load-source="ifi">For rapid, inexpensive and adaptable high volume screening of ligands to receptors; for determining the presence of cell growth factors in samples; for purifying cell growth factors and isoforms</invention-title></span><br><span class="patent-number">US 6018026 A</span></div><div class="patent-section patent-abstract-section"><div class="patent-section-header"><span class="patent-section-title">Abstract</span></div><div class="patent-text"><abstract mxw-id="PA38024597" lang="EN" load-source="patent-office"> <div class="abstract">Methods for producing secreted receptor analogs and biologically active peptide dimers are disclosed. The methods for producing secreted receptor analogs and biologically active peptide dimers utilize a DNA sequence encoding a receptor analog or a peptide requiring dimerization for biological activity joined to a dimerizing protein. The receptor analog includes a ligand-binding domain. Polypeptides comprising essentially the extracellular domain of a human PDGF receptor fused to dimerizing proteins, the portion being capable of binding human PDGF or an isoform thereof, are also disclosed. The polypeptides may be used within methods for determining the presence of and for purifying human PDGF or isoforms thereof.</div>
  </abstract></div></div><div class="patent-section patent-drawings-section"><div class="patent-section-header"><span class="patent-section-title">Images<span class="patent-section-count">(21)</span></span></div><div class="patent-drawings-body"><div class="patent-drawings-carousel"><div class="drawings"><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6018026-2.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6018026-2.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6018026-3.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6018026-3.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6018026-4.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6018026-4.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6018026-5.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6018026-5.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6018026-6.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6018026-6.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6018026-7.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6018026-7.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6018026-8.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6018026-8.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6018026-9.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6018026-9.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6018026-10.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6018026-10.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6018026-11.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6018026-11.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6018026-12.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6018026-12.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6018026-13.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6018026-13.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6018026-14.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6018026-14.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6018026-15.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6018026-15.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6018026-16.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6018026-16.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6018026-17.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6018026-17.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6018026-18.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6018026-18.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6018026-19.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6018026-19.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6018026-20.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6018026-20.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6018026-21.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6018026-21.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6018026-22.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6018026-22.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div></div></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img" alt="Previous page"src="/googlebooks/images/kennedy/page_left.png"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img" alt="Next page"src="/googlebooks/images/kennedy/page_right.png"width="21" height="21" /></div></div></div><div class="patent-post-drawings"></div><div class="patent-section patent-claims-section"><div class="patent-section-header"><span class="patent-section-title">Claims<span class="patent-section-count">(29)</span></span></div><div class="patent-text"><div mxw-id="PCLM59555280" lang="EN" load-source="patent-office" class="claims">
    <claim-statement>We claim:</claim-statement> <div class="claim"> <div num="1" class="claim">
      <div class="claim-text">1. A biologically active, dimerized polypeptide fusion, comprising:<div class="claim-text">first and second polypeptide chains, wherein each of said polypeptide chains comprises a non-immunoglobulin polypeptide requiring dimerization for biological activity joined to a dimerizing protein heterologous to said non-immunoglobulin polypeptide.</div> </div>
    </div>
    </div> <div class="claim-dependent"> <div num="2" class="claim">
      <div class="claim-text">2. The biologically active, dimerized polypeptide fusion of claim 1 which is a homodimer.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="3" class="claim">
      <div class="claim-text">3. The biologically active, dimerized polypeptide fusion of claim 1 wherein the dimerizing protein of one of said polypeptide chains comprises an immunoglobulin heavy chain constant region.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="4" class="claim">
      <div class="claim-text">4. The biologically active, dimerized polypeptide fusion of claim 3 wherein the immunoglobulin heavy chain constant region is joined to an immunoglobulin hinge region.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="5" class="claim">
      <div class="claim-text">5. The biologically active, dimerized polypeptide fusion of claim 3 wherein the immunoglobulin heavy chain constant region is joined to an immunoglobulin variable region.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="6" class="claim">
      <div class="claim-text">6. The biologically active, dimerized polypeptide fusion of claim 5 wherein the immunoglobulin variable region is selected from the group consisting of V<sub>H</sub>, Vκ, and Vλ.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="7" class="claim">
      <div class="claim-text">7. The biologically active, dimerized polypeptide fusion of claim 1 wherein the dimerizing protein one of said polypeptide chains comprises an immunoglobulin heavy chain constant region domain selected from the group consisting of C<sub>H</sub> 1, C<sub>H</sub> 2, C<sub>H</sub> 3, and C<sub>H</sub> 4 of a γ, α, ε, μ, or δ class immunoglobulin heavy chain.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="8" class="claim">
      <div class="claim-text">8. The biologically active, dimerized polypeptide fusion of claim 1 wherein the dimerizing protein of one of said polypeptide chains comprises an immunoglobulin light chain constant region.</div>
    </div>
    </div> <div class="claim"> <div num="9" class="claim">
      <div class="claim-text">9. A biologically active, multimerized polypeptide fusion, comprising:<div class="claim-text">a non-immunoglobulin polypeptide requiring multimerization for biological activity joined to an immunoglobulin light chain constant region; and</div> <div class="claim-text">an immunoglobulin heavy chain constant region domain selected from the group consisting of C<sub>H</sub> 1, C<sub>H</sub> 2, C<sub>H</sub> 3, and C<sub>H</sub> 4.</div> </div>
    </div>
    </div> <div class="claim-dependent"> <div num="10" class="claim">
      <div class="claim-text">10. The biologically active, multimerized polypeptide fusion of claim 9 which is a tetramer comprising four polypeptide fusions each having a non-immunoglobulin polypeptide joined to a multimerizing protein.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="11" class="claim">
      <div class="claim-text">11. The biologically active, multimerized polypeptide fusion of claim 9 including a multimerizing protein which comprises an immunoglobulin heavy chain constant region domain selected from the group consisting of C<sub>H</sub> 1, C<sub>H</sub> 2, C<sub>H</sub> 3, and C<sub>H</sub> 4 of a γ, α, ε, μ, or δ class immunoglobulin heavy chain.</div>
    </div>
    </div> <div class="claim"> <div num="12" class="claim">
      <div class="claim-text">12. A biologically active, multimerized polypeptide fusion comprising: a non-immunoglobulin polypeptide requiring multimerization for biological activity joined to an immunoglobulin heavy chain constant region; and<div class="claim-text">an immunoglobulin heavy chain.</div> </div>
    </div>
    </div> <div class="claim-dependent"> <div num="13" class="claim">
      <div class="claim-text">13. The biologically active, multimerized polypeptide fusion of claim 12 wherein the immunoglobulin heavy chain constant region is joined to an immunoglobulin hinge region.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="14" class="claim">
      <div class="claim-text">14. The biologically active, multimerized polypeptide fusion of claim 12 wherein the immunoglobulin heavy chain constant region is joined to an immunoglobulin variable region.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="15" class="claim">
      <div class="claim-text">15. The biologically active, multimerized polypeptide fusion of claim 14 wherein the immunoglobulin variable region is selected from the group consisting of V<sub>H</sub>, Vκ, and Vλ.</div>
    </div>
    </div> <div class="claim"> <div num="16" class="claim">
      <div class="claim-text">16. A biologically active, heteromultimeric polypeptide fusion, comprising:<div class="claim-text">a first polypeptide fusion comprising a first non-immunoglobulin polypeptide joined to a first multimerizing protein heterologous to said first non-immunoglobulin polypeptide and a second polypeptide fusion comprising a second non-immunoglobulin polypeptide joined to a second multimerizing protein heterologous to said second non-immunoglobulin polypeptide, wherein said first multimerizing protein and said second multimerizing protein are capable of associating to form a multimer.</div> </div>
    </div>
    </div> <div class="claim-dependent"> <div num="17" class="claim">
      <div class="claim-text">17. The biologically active, heteromultimeric polypeptide fusion of claim 16 wherein the first and second non-immunoglobulin polypeptides each comprise an amino acid sequence selected from the group consisting of (A) the amino acid sequence Sequence ID NO:2, and (B) the amino acid sequence of Sequence ID NO:36.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="18" class="claim">
      <div class="claim-text">18. The biologically active, heteromultimeric polypeptide fusion of claim 16 wherein the first multimerizing protein is different from the second multimerizing protein.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="19" class="claim">
      <div class="claim-text">19. The biologically active, heteromultimeric polypeptide fusion of claim 18 wherein the first and second non-immunoglobulin polypeptides are the same.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="20" class="claim">
      <div class="claim-text">20. The biologically active, heteromultimeric polypeptide fusion of claim 16 wherein the first and second multimerizing proteins each comprise an immunoglobulin heavy chain constant region or an immunoglobulin light chain constant region.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="21" class="claim">
      <div class="claim-text">21. The biologically active, heteromultimeric polypeptide fusion of claim 16 which comprises a first polypeptide fusion having a first non-immunoglobulin polypeptide joined to a first immunoglobulin constant region and a second polypeptide fusion having a second non-immunoglobulin polypeptide fused to a second immunoglobulin constant region different from the first immunoglobulin constant region.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="22" class="claim">
      <div class="claim-text">22. The biologically active, heteromultimeric polypeptide fusion of claim 21 wherein the first multimerizing protein comprises an immunoglobulin heavy chain constant region and the second multimerizing protein comprises an immunoglobulin light chain constant region.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="23" class="claim">
      <div class="claim-text">23. The biologically active, heteromultimeric polypeptide fusion of claim 20 wherein one of said multimerizing proteins comprises an immunoglobulin heavy chain constant region joined to an immunoglobulin hinge region.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="24" class="claim">
      <div class="claim-text">24. The biologically active, heteromultimeric polypeptide fusion of claim 20 wherein one of said multimerizing proteins comprises an immunoglobulin heavy chain constant region joined to an immunoglobulin variable region.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="25" class="claim">
      <div class="claim-text">25. The biologically active, heteromultimeric polypeptide fusion of claim 24 wherein the immunoglobulin variable region is selected from the group consisting of V<sub>H</sub>, Vκ, and Vλ.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="26" class="claim">
      <div class="claim-text">26. The biologically active, heteromultimeric polypeptide fusion of claim 16 wherein one of said multimerizing proteins comprises an immunoglobulin heavy chain constant region domain selected from the group consisting of C<sub>H</sub> 1, C<sub>H</sub> 2, C<sub>H</sub> 3, and C<sub>H</sub> 4 of a γ, α, ε, μ, or δ class immunoglobulin heavy chain.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="27" class="claim">
      <div class="claim-text">27. The biologically active, heteromultimeric polypeptide fusion of claim 16 wherein said multimerized polypeptide fusion comprises a first polypeptide fusion comprising a first receptor or receptor domain requiring multimerization for activity joined to a first immunoglobulin constant region and a second polypeptide fusion comprising a second receptor or receptor domain requiring multimerization for activity joined to a second immunoglobulin constant region.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="28" class="claim">
      <div class="claim-text">28. The biologically active, heteromultimeric polypeptide fusion of claim 27 wherein one of said receptor domains comprises a ligand binding domain.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="29" class="claim">
      <div class="claim-text">29. The biologically active, heteromultimeric polypeptide fusion of claim 16 which is a heterotetramer comprising four polypeptide fusions each having a non-immunoglobulin polypeptide joined to a multimerizing protein.</div>
    </div>
  </div> </div></div></div><div class="patent-section patent-description-section"><div class="patent-section-header"><span class="patent-section-title">Description</span></div><div class="patent-text"><div mxw-id="PDES67420765" lang="EN" load-source="patent-office" class="description">
    <heading>CROSS REFERENCE TO RELATED APPLICATION</heading> <p>This application is a divisional of U.S. application Ser. No. 8/477,329, filed Jun. 7, 1995, now U.S. Pat. No. 5,750,375 which is a continuation of U.S. application Ser. No. 180,195, filed Jan. 11, 1994,now U.S. Pat. No. 5,567,584, which is a continuation of U.S. application Ser. No. 07/634,510 filed Dec. 27, 1990, now abandoned, which is a continuation-in-part of U.S. application Ser. No. 7/347,291, filed May 2, 1989, now U.S. Pat. No. 5,155,027 which is a continuation-in-part of U.S. application Ser. No/7/146,877, filed Jan. 22, 1988, now abandoned.</p>
    <heading>TECHNICAL FIELD TECHNICAL FIELD</heading> <p>The present invention is generally directed toward the expression of proteins, and more specifically, toward the expression of growth factor receptor analogs and biologically active dimerized polypeptide fusions.</p>
    <heading>BACKGROUND OF THE INVENTION</heading> <p>In higher eucaryotic cells, the interaction between receptors and ligands (e.g., hormones) is of central importance in the transmission of and response to a variety of extracellular signals. It is generally accepted that hormones and growth factors elicit their biological functions by binding to specific recognition sites (receptors) in the plasma membranes of their target cells. Upon ligand binding, a receptor undergoes a conformational change, triggering secondary cellular responses that result in the activation or inhibition of intracellular processes. The stimulation or blockade of such an interaction by pharmacological means has important therapeutic implications for a wide variety of illnesses.</p>
    <p>Ligands fall into two classes: those that have stimulatory activity, termed agonists; and those that block the effects elicited by the original ligands, termed antagonists. The discovery of agonists that differ in structure and composition from the original ligand may be medically useful. In particular, agonists that are smaller than the original ligand may be especially useful. The bioavailability of these smaller agonists may be greater than that of the original ligand. This may be of particular importance for topical applications and for instances when diffusion of the agonist to its target sites is inhibited by poor circulation. Agonists may also have slightly different spectra of biological activity and/or different potencies, allowing them to be used in very specific situations. Agonists that are smaller and chemically simpler than the native ligand may be produced in greater quantity and at lower cost. The identification of antagonists which specifically block, for example, growth factor receptors has important pharmaceutical applications. Antagonists that block receptors against the action of endogenous, native ligand may be used as therapeutic agents for conditions including atherosclerosis, autocrine tumors, fibroplasia and keloid formation.</p>
    <p>The discovery of new ligands that may be used in pharmaceutical applications has centered around designing compounds by chemical modification, complete synthesis, and screening potential ligands by complex and costly screening procedures. The process of designing a new ligand usually begins with the alteration of the structure of the original effector molecule. If the original effector molecule is known to be chemically simple, for example, a catecholamine or prostaglandin, the task is relatively straightforward. However, if the ligand is structurally complex, for example, a peptide hormone or a growth factor, finding a molecule which is functionally equivalent to the original ligand becomes extremely difficult.</p>
    <p>Currently, potential ligands are screened using radioligand binding methods (Lefkowitz et al., Biochem, Biophys. Res. Comm. 60: 703-709, 1974; Aurbach et al., Science 186: 1223-1225, 1974; Atlas et al., Proc. Natl. Acad. Sci. USA 71: 4246-4248, 1974). Potential ligands can be directly assayed by binding the radiolabeled compounds to responsive cells, to the membrane fractions of disrupted cells, or to solubilized receptors. Alternatively, potential ligands may be screened by their ability to compete with a known labeled ligand for cell surface receptors.</p>
    <p>The success of these procedures depends on the availability of reproducibly high quality preparations of membrane fractions or receptor molecules, as well as the isolation of responsive cell lines. The preparation of membrane fractions and soluble receptor molecules involves extensive manipulations and complex purification steps. The isolation of membrane fractions requires gentle manipulation of the preparation, a procedure which does not lend itself to commercial production. It is very difficult to maintain high biological activity and biochemical purity of receptors when they are purified by classical protein chemistry methods. Receptors, being integral membrane proteins, require cumbersome purification procedures, which include the use of detergents and other solvents that interfere with their biological activity. The use of these membrane preparations in ligand binding assays typically results in low reproducibility due to the variability of the membrane preparations.</p>
    <p>As noted above, ligand binding assays require the isolation of responsive cell lines. Often, only a limited subset of cells is responsive to a particular agent, and such cells may be responsive only under certain conditions. In addition, these cells may be difficult to grow in culture or may possess a low number of receptors. Currently available cell types responsive to platelet-derived growth factor (PDGF), for example, contain only a low number (up to 4×10<sup>5</sup> ; see Bowen-Pope and Ross, J. Biol. Chem. 257: 5161-5171, 1982) of receptors per cell, thus requiring large numbers of cells to assay PDGF analogs or antagonists.</p>
    <p>Presently, only a few naturally-occurring secreted receptors, for example, the interleukin-2 receptor (IL-2-R) have been identified. Rubin et al. (J. Immun. 135: 3172-3177, 1985) have reported the release of large quantities of IL-2-R into the culture medium of activated T-cell lines. Bailon et al. (Bio/Technology 5: 1195-1198, 1987) have reported the use of a matrix-bound interleukin-2 receptor to purify recombinant interleukin-2.</p>
    <p>Three other receptors have been secreted from mammalian cells. The insulin receptor (Ellis et al., J. Cell Biol. 150: 14a, 1987), the HIV-1 envelope glyco-protein cellular receptor CD4 (Smith et al., Science 238: 1704-1707, 1987), the murine IL-7 receptor (Cell 60: 941-951, 1990) and the epidermal growth factor (EGF) receptor (Livneh et al., J. Biol. Chem. 261: 12490-12497, 1986) have been secreted from mammalian cells using truncated cDNAs that encode portions of the extracellular domains.</p>
    <p>Naturally-occurring, secreted receptors have not been widely identified, and there have been only a limited number of reports of secreted recombinant receptors. Secreted receptors may be used in a variety of assays, which include assays to determine the presence of ligand in biological fluids and assays to screen for potential agonists and antagonists. Current methods for ligand screening and ligand/receptor binding assays have been limited to those using preparations of whole cells or cell membranes for as a source for receptor molecules. The low reproducibility and high cost of producing such preparations does not lend itself to commercial production. There is therefore a need in the art for a method of producing secreted receptors. There is a further need in the art for an assay system that permits high volume screening of compounds that may act on higher eucaryotic cells via specific surface receptors. This assay system should be rapid, inexpensive and adaptable to high volume screening. The present invention discloses such a method and assay system, and further provides other related advantages.</p>
    <heading>DISCLOSURE OF INVENTION</heading> <p>Briefly stated, the present invention discloses methods for producing secreted receptor analogs, including receptor analogs and secreted platelet-derived growth factor receptor (PDGF-R) analogs. In addition, the present invention discloses methods for producing secreted biologically active dimerized polypeptide fusions.</p>
    <p>Within one aspect of the invention a method for producing a secreted PDGF-R analog is disclosed, comprising (a) introducing into a eukaryotic host cell a DNA construct comprising a transcriptional promoter operatively linked to a secretory signal sequence followed downstream of and in proper reading frame with a DNA sequence encoding at least a portion of the ligand-binding domain of a PDGF-R, the portion including a ligand-binding domain; (b) growing the host cell in an appropriate growth medium under physiological conditions to allow the secretion of a PDGF-R analog encoded by said DNA sequence; and (c) isolating the PDGF-R analog from the host cell.</p>
    <p>Within one embodiment of the present invention, a PDGF-R analog comprising the amino acid sequence of FIGS. 1A-1B (Sequence ID Numbers 1 and 2) from isoleucine, number 29, to methionine, number 441, is secreted. Within another embodiment, a PDGF-R analog comprising the amino acid sequence of FIGS. 1A-1B (Sequence ID Numbers 1 and 2) from isoleucine, number 29 to lysine, number 531 is secreted. Within yet another embodiment of the invention, a PDGF-R analog comprising the amino acid sequence of FIGS. 11A-11B (Sequence ID Numbers 35 and 36) from glutamine, number 24 to glutamic acid, number 524 is secreted.</p>
    <p>Yet another aspect of the present invention discloses a method for producing a secreted, biologically active dimerized polypeptide fusion. The method generally comprises a) introducing into a eukaryotic host cell a DNA construct comprising a transcriptional promoter operatively linked to a secretory signal sequence followed downstream by and in proper reading frame with a DNA sequence encoding a non-immunoglobulin polypeptide requiring dimerization for biological activity joined to a dimerizing protein; (b) growing the host cell in an appropriate growth medium under physiological conditions to allow the secretion of a dimerized polypeptide fusion encoded by said DNA sequence; and (c) isolating the biologically active dimerized polypeptide fusion from the host cell.</p>
    <p>Within one embodiment, the dimerizing protein is yeast invertase. Within another embodiment, the dimerizing protein is at least a portion of an immunoglobulin light chain. Within another embodiment, the dimerizing protein is at least a portion of an immunoglobulin heavy chain.</p>
    <p>In another aspect of the invention, a method is disclosed for producing a secreted, biologically active dimerized polypeptide fusion, comprising (a) introducing into a eukaryotic host cell a first DNA construct comprising a transcriptional promoter operatively linked to a first secretory signal sequence followed downstream by and in proper reading frame with a first DNA sequence encoding a non-immunoglobulin polypeptide requiring dimerization for biological activity joined to an immunoglobulin light chain constant region; (b) introducing into the host cell a second DNA construct comprising a transcriptional promoter operatively linked to a second secretory signal sequence followed downstream by and in proper reading frame with a second DNA sequence encoding an immunoglobulin heavy chain constant region domain selected from the group consisting of C<sub>H</sub> 1, C<sub>H</sub> 2, C<sub>H</sub> 3, and C<sub>H</sub> 4; (c) growing the host cell in an appropriate growth medium under physiological conditions to allow the secretion of a dimerized polypeptide fusion encoded by said first and second DNA sequences; and (d) isolating the dimerized polypeptide fusion from the host cell. In one embodiment, the second DNA sequence further encodes an immuunoglobulin heavy chain hinge region wherein the hinge region is joined to the heavy chain constant region domain. In a preferred embodiment, the second DNA sequence further encodes an immunoglobulin variable region joined upstream of and in proper reading frame with the immunoglobulin heavy chain constant region.</p>
    <p>In another aspect of the invention, a method is disclosed for producing a secreted, biologically active dimerized polypeptide fusion, comprising (a) introducing into a eukaryotic host cell a first DNA construct comprising a transcriptional promoter operatively linked to a first secretory signal sequence followed downstream by and in proper reading frame with a first DNA sequence encoding a non-immunoglobulin polypeptide requiring dimerization for biological activity joined to an immunoglobulin heavy chain constant region domain selected from the group consisting of C<sub>H</sub> 1, C<sub>H</sub> 2, C<sub>H</sub> 3, and C<sub>H</sub> 4; (b) introducing into the host cell a second DNA construct comprising a transcriptional promoter operatively linked to a second secretory signal sequence followed downstream by and in proper reading frame with a second DNA sequence encoding an immunoglobulin light chain constant region; (c) growing the host cell in an appropriate growth medium under physiological conditions to allow the secretion of a dimerized polypeptide fusion encoded by said first and second DNA sequences; and (d) isolating the dimerized polypeptide fusion from the host cell. In one embodiment, the first DNA sequence further encodes an immunoglobulin heavy chain hinge region wherein the hinge region is joined to the heavy chain constant region domain. In a preferred embodiment, the second DNA sequence further encodes an immunoglobulin variable region joined upstream of and in proper reading frame with the immunoglobulin light chain constant region.</p>
    <p>In another aspect of the invention, a method is disclosed for producing a secreted, biologically active dimerized polypeptide fusion, comprising (a) introducing into a eukaryotic host cell a DNA construct comprising a transcriptional promoter operatively linked to a secretory signal sequence followed downstream by and in proper reading frame with a DNA sequence encoding a non-immunoglobulin polypeptide requiring dimerization for biological activity joined to an immunoglobulin heavy chain constant region domain selected from the group consisting of C<sub>H</sub> 1, C<sub>H</sub> 2, C<sub>H</sub> 3, and C<sub>H</sub> 4; (b) growing the host cell in an appropriate growth medium under physiological conditions to allow the secretion of a dimerized polypeptide fusion encoded by said first and second DNA sequences; and (c) isolating the biologically active dimerized polypeptide fusion from the host cell. In one embodiment, the DNA sequence further encodes an immunoglobulin heavy chain hinge region wherein the hinge region is joined to the heavy chain constant region domain.</p>
    <p>In another aspect of the invention, a method is disclosed for producing a secreted, biologically active dimerized polypeptide fusion, comprising (a) introducing into a eukaryotic host cell a first DNA construct comprising a transcriptional promoter operatively linked to a first secretory signal sequence followed downstream by and in proper reading frame with a first DNA sequence encoding a first polypeptide chain of a non-immunoglobulin polypeptide dimer requiring dimerization for biological activity joined to an immunoglobulin heavy chain constant region domain, selected from the group consisting of C<sub>H</sub> 1, C<sub>H</sub> 2, C<sub>H</sub> 3, and C<sub>H</sub> 4; (b) introducing into the host cell a second DNA construct comprising a transcriptional promoter operatively linked to a second secretory signal sequence followed downstream by and in proper reading frame with a second DNA sequence encoding a second polypeptide chain of the non-immunoglobulin polypeptide dimer joined to an immunoglobulin light chain constant region domain; (c) growing the host cell in an appropriate growth medium under physiological conditions to allow the secretion of a dimerized polypeptide fusion encoded by said first and second DNA sequences wherein said dimerized polypeptide fusion exhibits biological activity characteristic of said non-immunoglobulin polypeptide dimer; and (d) isolating the dimerized polypeptide fusion from the host cell. In one embodiment the first DNA sequence further encodes an immunoglobulin heavy chain hinge region domain wherein the hinge region is joined to the immunoglobulin heavy chain constant region domain.</p>
    <p>Within one embodiment of the present invention, a biologically active dimerized polypeptide fusion comprising the amino acid sequence of FIGS. 1A-1B (Sequence ID Numbers 1 and 2) from isoleucine, number 29, to methionine, number 441, is secreted. Within another embodiment, a biologically active dimerized polypeptide fusion comprising the amino acid sequence of FIGS. 1A-1B (Sequence ID Numbers 1 and 2) from isoleucine, number 29 to lysine, number 531 is secreted. Within another embodiment of the invention, a biologically active dimerized polypeptide fusion comprising the amino acid sequence of FIGS. 11A-11B (Sequence ID Numbers 35 and 36) from glutamine, number 24 to glutamic acid, number 524 is secreted. Within yet another embodiment of the invention, a biologically active dimerized polypeptide fusion comprising the amino acid sequence of FIGS. 1A-1B (Sequence ID Numbers 1 and 2) from isoleucine, number 29 to lysine, number 531 dimerized to the amino acid sequence of FIGS. 11A-11B (sequence ID Numbers 35 and 36) from glutamine, number 24 to glutamic acid, number 524 is secreted.</p>
    <p>In yet another aspect of the invention, a method is disclosed for producing a secreted receptor analog, comprising (a) introducing into a eukaryotic host cell a DNA construct comprising a transcriptional promoter operatively linked to at least one secretory signal sequence followed downstream by and in proper reading frame with a DNA sequence encoding a ligand-binding domain of a receptor requiring dimerization for biological activity joined to a dimerizing protein; (b) growing the host cell in an appropriate growth medium under physiological conditions to allow the secretion of a receptor analog encoded by said DNA sequence; and (c) isolating the receptor analog from the host cell.</p>
    <p>In yet another aspect of the invention, a method is disclosed for producing a secreted receptor analog, comprising (a) introducing into a eukaryotic host cell a first DNA construct comprising a transcriptional promoter operatively linked to a first secretory signal sequence followed downstream by and in proper reading frame with a first DNA sequence encoding a ligand-binding domain of a receptor requiring dimerization for biological activity joined to an immunoglobulin light chain constant region; (b) introducing into the host cell a second DNA construct comprising a transcriptional promoter operatively linked to a second secretory signal sequence followed downstream by and in proper reading frame with a second DNA sequence encoding an immunoglobulin heavy chain constant region domain, selected from the group consisting of C<sub>H</sub> 1, C<sub>H</sub> 2, C<sub>H</sub> 3, and C<sub>H</sub> 4; (c) growing the host cell in an appropriate growth medium under physiological conditions to allow the secretion of a receptor analog encoded by said first and second DNA sequences; and (d) isolating the receptor analog from the host cell. In one embodiment, the second DNA sequence further encodes an immunoglobulin heavy chain hinge region wherein the hinge region is joined to the heavy chain constant region domain. In a preferred embodiment, the second DNA sequence further encodes an immunoglobulin variable region joined upstream of and in proper reading frame with the immunoglobulin heavy chain constant region.</p>
    <p>In another aspect of the invention, a method is disclosed for producing a secreted receptor analog, comprising (a) introducing into a eukaryotic host cell a DNA construct comprising a transcriptional promoter operatively linked to a secretory signal sequence followed downstream by and in proper reading frame with a DNA sequence encoding a ligand-binding domain of a receptor requiring dimerization for biological activity joined to an immunoglobulin heavy chain constant region domain, selected from the group C<sub>H</sub> 1, C<sub>H</sub> 2, C<sub>H</sub> 3, and C<sub>H</sub> 4; (b) growing the host cell in an appropriate growth medium under physiological conditions to allow the secretion of the receptor analog; and (c) isolating the receptor analog from the host cell. In one embodiment, the DNA sequence further encodes an immunoglobulin heavy chain hinge region wherein the hinge region is joined to the heavy chain constant region domain.</p>
    <p>In another aspect of the invention, a method is disclosed for producing a secreted receptor analog, comprising (a) introducing into a eukaryotic host cell a first DNA construct comprising a transcriptional promoter operatively linked to a first secretory signal sequence followed downstream of and in proper reading frame with a first DNA sequence encoding a ligand-binding domain of a receptor requiring dimerization for biological activity joined to an immunoglobulin heavy chain constant region domain, selected from the group C<sub>H</sub> 1, C<sub>H</sub> 2, C<sub>H</sub> 3, and C<sub>H</sub> 4; (b) introducing into the host cell a second DNA construct comprising a transcriptional promoter operatively linked to a second secretory signal sequence followed downstream by and in proper reading frame with a second DNA sequence encoding an immunoglobulin light chain constant region; (c) growing the host cell in an appropriate growth medium under physiological conditions to allow the secretion of a receptor analog encoded by said first and second DNA sequences; and (d) isolating the receptor analog from the host cell. In one embodiment, the first DNA sequence further encodes an immunoglobulin heavy chain hinge region wherein the hinge region is joined to the heavy chain constant region domain. In a preferred embodiment, the second DNA sequence further encodes an immunoglobulin variable region joined upstream of and in proper reading frame with the immunoglobulin light chain constant region.</p>
    <p>In another aspect of the invention, a method is disclosed for producing a secreted receptor analog, comprising (a) introducing into a eukaryotic host cell a first DNA construct comprising a transcriptional promoter operatively linked to a first secretory signal sequence followed downstream in proper reading frame by a first DNA sequence encoding a first polypeptide chain of a ligand-binding domain of a receptor requiring dimerization for biological activity joined to an immunoglobulin heavy chain constant region domain, selected from the group C<sub>H</sub> 1, C<sub>H</sub> 2, C<sub>H</sub> 3, and C<sub>H</sub> 4; (b) introducing into the host cell a second DNA construct comprising a transcriptional promoter operatively linked to a second secretory signal sequence followed downstream by and in proper reading frame with a second DNA sequence encoding a second polypeptide chain of the ligand-binding domain of said receptor joined to an immunoglobulin light chain constant region domain; (c) growing the host cell in an appropriate growth medium under physiological conditions to allow the secretion of a receptor analog encoded by said first and second DNA sequences; and (d) isolating the receptor analog from the host cell. In one embodiment the first DNA sequence further encodes an immunoglobulin heavy chain hinge region domain wherein the hinge region is joined to the immunoglobulin heavy chain constant region domain.</p>
    <p>Host cells for use in the present invention include cultured mammalian cells and fungal cells. In a preferred embodiment strains of the yeast Saccharomyces cerevisiae are used as host cells. Within another preferred embodiment cultured rodent myeloma cells are used as host cells.</p>
    <p>Within one embodiment of the present invention, a receptor analog is a PDGF-R analog comprising the amino acid sequence of FIG. 1 (Sequence ID Numbers 1 and 2) from isoleucine, number 29, to methionine, number 441. Within another embodiment a PDGF-R analog comprises the amino acid sequence of FIG. 1 (Sequence ID Numbers 1 and 2) from isoleucine, number 29, to lysine, number 531. Within another embodiment of the invention, a PDGF-R analog comprises the amino acid sequence of FIG. 11 (Sequence ID Numbers 35 and 36) from glutamine, number 24 to glutamic acid, number 524 is secreted. Within yet another embodiment of the invention, a PDGF-R analog comprises the amino acid sequence of FIG. 1 (Sequence ID Numbers 1 and 2) from isoleucine, number 29 to lysine, number 531 and the amino acid sequence of FIG. 11 (Sequence ID Numbers 35 and 36) from glutamine, number 24 to glutamic acid, number 524 is secreted.</p>
    <p>PDGF-R analogs produced by the above-disclosed methods may be used, for instance, within a method for determining the presence of human PDGF or an isoform thereof in a biological sample.</p>
    <p>A method for determining the presence of human PDGF or an isoform thereof in a biological sample is disclosed and comprises (a) incubating a polypeptide comprising a PDGF receptor analog fused to a dimerizing protein with a biological sample suspected of containing PDGF or an isoform thereof under conditions that allow the formation of receptor/ligand complexes; and (b) detecting the presence of receptor/ligand complexes, and therefrom determining the presence of PDGF or an isoform thereof. Suitable biological samples in this regard include blood, urine, plasma, serum, platelet and other cell lysates, platelet releasates, cell suspensions, cell-conditioned culture media, and chemically or physically separated portions thereof.</p>
    <p>These and other aspects of the present invention will become evident upon reference to the following detailed description and attached drawings.</p>
    <heading>BRIEF DESCRIPTION OF THE DRAWINGS</heading> <p>FIGS. 1A-1D (Sequence ID Numbers 1 and 2) illustrate the nucleotide sequence of a representative PDGF β-receptor cDNA and the derived amino acid sequence of the primary translation product and corresponds to Sequence ID Number 1). Numbers above the lines refer to the nucleotide sequence; numbers below the lines refer to the amino acid sequence.</p>
    <p>FIG. 2 illustrates the construction of pBTL10, pBTL11 and pBTL12.</p>
    <p>FIG. 3 illustrates the construction of pCBS22.</p>
    <p>FIG. 4 illustrates the construction of pBTL13 and pBTL14.</p>
    <p>FIG. 5 illustrates the construction of pBTL15.</p>
    <p>FIG. 6 illustrates the construction of pBTL22 and pBTL26.</p>
    <p>FIG. 7 illustrates the construction of pSDL114. Symbols used are S.S., signal sequence, C<sub>k</sub>, immunoglobulin light chain constant region sequence; μ prom, μ promoter; μ enh, μ enhancer.</p>
    <p>FIG. 8 illustrates the construction of pSDLB113. Symbols used are S.S., signal sequence; C<sub>H</sub> 1 , C<sub>H</sub> 2, C<sub>H</sub> 3, immunoglobulin heavy chain constant region domain sequences; H, immunoglobulin heavy chain hinge region sequence; M, immunoglobulin membrane anchor sequences; c.sub.γ 1M, immunoglobulin heavy chain constant region and membrane anchor sequences.</p>
    <p>FIG. 9 illustrates the constructions pBTL115, pBTL114, pφ5V<sub>H</sub> HuC.sub.γ 1M-neo, plCφ5V.sub.κ HuC.sub.κ -neo. Symbols used are set forth in FIGS. 7 and 8, and also include L<sub>H</sub>, mouse immunoglobulin heavy chain signal sequence; V<sub>H</sub>, mouse immunoglobulin heavy chain variable region sequence; E, mouse immunoglobulin heavy chain enhancer sequence; L.sub.κ, mouse immunoglobulin light chain signal sequence; φ5V.sub.κ, mouse immunoglobulin light chain variable region sequence; Neo<sup>R</sup>, E. coli neomycin resistance gene.</p>
    <p>FIG. 10 illustrates the constructions Zem229R, pφ5V<sub>H</sub> Fab-neo and pWKI. Symbols used are set forth in FIG. 9.</p>
    <p>FIGS. 11A-11D illustrate the sequence of a representative PDGF α-receptor CDNA and the deduced amino acid sequence (using standard one-letter codes) encoded by the CDNA and corresponds to Sequence ID Numbers 35 and 36. Numbers at the ends of the lines refer to nucleotide positions. Numbers below the sequence refer to amino acid positions.</p>
    <p>FIG. 12 illustrates the assembly of a cDNA molecule encoding a PDGF α-receptor. Complementary DNA sequences are shown as lines. Only those portions of the vectors adjacent to the cDNA inserts are shown.</p>
    <heading>DETAILED DESCRIPTION OF THE INVENTION</heading> <p>Prior to setting forth the invention, it may be helpful to an understanding thereof to set forth definitions of certain terms to be used hereinafter.</p>
    <heading>DNA Construct</heading> <p>A DNA molecule, or a clone of such a molecule, either single- or double-stranded that has been modified through human intervention to contain segments of DNA combined and juxtaposed in a manner that as a whole would not otherwise exist in nature.</p>
    <p>DNA constructs contain the information necessary to direct the expression and/or secretion of DNA sequences encoding polypeptides of interest. DNA constructs will generally include promoters, enhancers and transcription terminators. DNA constructs containing the information necessary to direct the secretion of a polypeptide will also contain at least one secretory signal sequence.</p>
    <heading>Secretory Signal Sequence</heading> <p>A DNA sequence encoding a secretory peptide. A secretory peptide is an amino acid sequence that acts to direct the secretion of a mature polypeptide or protein from a cell. Secretory peptides are characterized by a core of hydrophobic amino acids and are typically (but not exclusively) found at the amino termini of newly sythesized proteins. Very often the secretory peptide is cleaved from the mature protein during secretion. Such secretory peptides contain processing sites that allow cleavage of the signal peptides from the mature proteins as it passes through the secretory pathway. Processing sites may be encoded within the signal peptide or may be added to the signal peptide by, for example, in vitro mutagenesis. Certain secretory peptides may be used in concert to direct the secretion of polypeptides and proteins. One such secretory peptide that may be used in combination with other secretory peptides is the third domain of the yeast Barrier protein.</p>
    <heading>Receptor Analog</heading> <p>A non-immunoglobulin polypeptide comprising a portion of a receptor which is capable of binding ligand and/or is recognized by anti-receptor antibodies. The amino acid sequence of the receptor analog may contain additions, substitutions or deletions as compared to the native receptor sequence. A receptor analog may be, for example, the ligand-binding domain of a receptor joined to another protein. Platelet-derived growth factor receptor (PDGF-R) analogs may, for example, comprise a portion of a PDGF receptor capable of binding anti-PDGF receptor antibodies, PDGF, PDGF isoforms, PDGF analogs, or PDGF antagonists.</p>
    <heading>Dimerizing Protein</heading> <p>A polypeptide chain having affinity for a second polypeptide chain, such that the two chains associate under physiological conditions to form a dimer. The second polypeptide chain may be the same or a different chain.</p>
    <heading>Biological Activity</heading> <p>A function or set of activities performed by a molecule in a biological context (i.e., in an organism or an in vitro facsimile thereof). biological activities may include the induction of extracellular matrix secretion from responsive cell lines, the induction of hormone secretion, the induction of chemotaxis, the induction of mitogenesis, the induction of differentiation, or the inhibition of cell division of responsive cells. A recombinant protein or peptide is considered to be biologically active if it exhibits one or more biological activities of its native counterpart.</p>
    <heading>Ligand</heading> <p>A molecule capable of being bound by the ligand-binding domain of a receptor or by a receptor analog. The molecule may be chemically synthesized or may occur in nature. Ligands may be grouped into agonists and antagonists. Agonists are those molecules whose binding to a receptor induces the response pathway within a cell. Antagonists are those molecules whose binding to a receptor blocks the response pathway within a cell.</p>
    <heading>Joined</heading> <p>Two or more DNA coding sequences are said to be joined when, as a result of in-frame fusions between the DNA coding sequences or as a result of the removal of intervening sequences by normal cellular processing, the DNA coding sequences are translated into a polypeptide fusion.</p>
    <p>As noted above, the present invention provides methods for producing biologically active dimerized polypeptide fusions and secreted receptor analogs, which include, for example, PDGF receptor analogs. Secreted receptor analogs may be used to screen for new compounds that act as agonists or antagonists when interacting with cells containing membrane-bound receptors. In addition, the methods of the present invention provide dimerized non-immunoglobulin polypeptide fusions of therapeutic value that are biologically active only as diners. Moreover, the present invention provides methods of producing polypeptide dimers that are biologically active only as non-covalently associated diners. Secreted, biologically active dimers that may be produced using the resent invention include nerve growth factor, colony stimulating factor-1, factor XIII, and transforming growth factor β.</p>
    <p>As used herein, the ligand-binding domain of a receptor is that portion of the receptor that is involved with binding the natural ligand. While not wishing to be bound by theory, the binding of a natural ligand to a receptor is believed to induce a conformational change which elicits a response to the change within the response pathway of the cell. For membrane-bound receptors, the ligand-binding domain is generally believed to comprise the extracellular domain for the receptor. The structure of receptors may be predicted from the primary translation products using the hydrophobicity plot function of, for example, P/C Gene or Intelligenetics Suite (Intelligenetics, Mt. View, Calif.) or may be predicted according to the methods described, for example, by Kyte and Doolittle, J. Mol. Biol. 157:105-132, 1982). The ligand-binding domain of the PDGF β-receptor, for example, has been predicted to include amino acids 29-531 of the published sequence (Gronwald et al., ibid.). The ligand-binding domain of the PDGF α-receptor has been predicted to include amino acids 25-500 of the published α-receptor sequence (Matsui et al., ibid.). As used herein, the ligand-binding domain of the PDGF β-receptor includes amino acids 29-441 of the sequence of FIGS. 1A-1D (Sequence ID Number 1) and C-terminal extensions up to and including amino acid 531 (FIG. 1B). The ligand-binding domain of the PDGF α-receptor is understood to include amino acids 24-524 of FIGS. 11A &amp; 11B (Sequence ID Numbers 35 and 36).</p>
    <p>Receptor analogs that may be used in the present invention include the ligand-binding domains of the epidermal growth factor receptor (EGF-R) and the insulin receptor. As used herein, a ligand-binding domain is that portion of the receptor that is involved in binding ligand and is generally a portion or essentially all of the extracellular domain that extends from the plasma membrane into the extracellular space. The ligand-binding domain of the EGF-R, for example, resides in the extracellular domain. EGF-R diners have been found to exhibit higher ligand-binding affinity than EGF-R monomers (Boni-Schnetzler and Pilch, Proc. Natl. Acad. Sci. USA §A:7832-7836, 1987). The insulin receptor (Ullrich et al., Nature 313:756-761, 1985) requires dimerization for biological activity.</p>
    <p>Another example of a receptor that may be secreted from a host cell is a platelet-derived growth factor receptor (PDGF-R). Two classes of PDGF-Rs, which recognized different isoforms of PDGF, have been identified. (PDGF is a disulfide-bonded, two-chain molecule, which is made up of an A chain and a B chain. These chains may be combined as AB heterodimers, AA homodimers or BB homodimers. These dimeric molecules are referred to herein as "isoforms".) The β-receptor (PDGFβ-R), which recognizes only the BB isoform of PDGF (PDGF-BB), has been described (Claesson-Welsh et al., Mol. Cell, Biol. 8:3476-3486, 1988; Gronwald et al., Proc. Natl. Acad. Sci. USA 85:3435-3439, 1988). The α-receptor (PDGFα-R), which recognizes all three PDGF isoforms (PDGF-AA, PDGF-AB and PDGF-BB), has been described by Matsui et al. (Science 243:800-804, 1989) and Kelly and Murray (pending commonly assigned U.S. patent application Ser. No. 07/355,018, which is incorporated herein by reference). The primary translation products of these receptors indicate that each includes an extracellular domain implicated in the ligand-binding process, a transmembrane domain, and a cytoplasmic domain containing a tyrosine kinase activity.</p>
    <p>The present invention provides a standardized assay system, not previously available in the art, for determining the presence of PDGF, PDGF isoforms, PDGF agonists or PDGF antagonists using a secreted PDGF receptor analogs. Such an assay system will typically involve combining the secreted PDGF receptor analog with a biological sample under physiological conditions which permit the formation of receptor-ligand complexes, followed by detecting the presence of the receptor-ligand complexes. The term physiological conditions is meant to include those conditions found within the host organism and include, for example, the conditions of osmolarity, salinity and pH. Detection may be achieved through the use of a label attached to the PDGF receptor analog or through the use of a labeled antibody which is reactive with the receptor analog or the ligand. A wide variety of labels may be utilized, such as radionuclides, fluorophores, enzymes and luminescers. Receptor-ligand complexes may also be detected visually, i.e., in immunoprecipitation assays which do not require the use of a label. This assay system provides secreted PDGF receptor analogs that may be utilized in a variety of screening assays for, for example, screening for analogs of PDGF. The present invention also provides a methods for measuring the level of PDGF and PDGF isoforms in biological fluids.</p>
    <p>As noted above, the present invention provides methods for producing dimerized polypeptide fusions that require dimerization for biological activity or enhancement of biological activity. Polypeptides requiring dimerization for biological activity include, in addition to certain receptors, nerve growth factor, colony-stimulating factor-1 (CSF-1), transforming growth factor β (TGF-β) , PDGF, and factor XIII. Nerve growth factor is a non-covalently linked diner (Harper et al., J. Biol. Chem. 257: 8541-8548, 1982). CSF-1, which specifically stimulates the proliferation and differentiation of cells of mononuclear phagocytic lineage, is a disulfide-bonded homodimer (Retternmier et al., Mol. Cell. Biol. 7: 2378-2387, 1987). TGF-β is biologically active as a disulfide-bonded dimer (Assoian et al., J. Biol. Chem. 258: 7155-7160, 1983). Factor XIII is a plasma protein that exists as a two chain homodimer n its activated form (Ichinose et al., Biochem. 25: 6900-6906, 1986). PDGF, as noted above, is a disulfide-bonded, two chain molecule (Murray et al., U.S. Pat. No. 4,766,073).</p>
    <p>The present invention provides methods by which receptor analogs, including receptor analogs and PDGF-R analogs, requiring dimerization for activity may be secreted from host cells. The methods described herein are particularly advantageous in that they allow the production of large quantities of purified receptors. The receptors may be used in assays for the screening of potential ligands, in assays for binding studies, as imaging agents, and as agonists and antagonists within therapeutic agents.</p>
    <p>A DNA sequence encoding a human PDGF receptor may be isolated as a cDNA using techniques known in the art (see, for example, Okayama and Berg, Mol. Cell. Biol. 2:161-170, 1982; Mol. Cell. Biol. 3: 280-289, 1983) from a library of human genomic or cDNA sequences. Such libraries may be prepared by standard procedures, such as those disclosed by Gubler and Hoffman (Gene 25: 263-269, 1983). It is preferred that the molecule is a CDNA molecule because CDNA lack introns and are therefore more suited to manipulation and expression in transfected or transformed cells. Sources of mRNA for use in the preparation of a CDNA library include the MG-63 human osteosarcoma cell line (available from ATCC under accession number CRL 1427), diploid human dermal fibroblasts and human embryo fibroblast and brain cells (Matsui et al., ibid.). A CDNA encoding a PDGFβ-R has been cloned from a diploid human dermal fibroblast cDNA library using oligonucleotide probes complementary to sequences of the mouse PDGF receptor (Gronwald et al., ibid.). A PDGFα-R cDNA has been isolated by Matsui et al. (ibid.) from human embryo fibroblast and brain cells. Alternatively, a cDNA encoding a PDGFα-R may be isolated from a library prepared from MG-63 human osteosarcoma cells using a cDNA probe containing sequences encoding the transmembrane and cytoplasmic domains of the PDGFβ-R (described by Kelly and Murray, ibid.). Partial CDNA clones (fragments) can be extended by re-screening of the library with the cloned CDNA fragment until the full sequence is obtained. In one embodiment, a ligand-binding domain of a PDGF receptor is encoded by the sequence of FIGS. 1A &amp; 1B (Sequence ID Number 1) from amino acid 29 through amino acid 441. In another embodiment, a ligand-binding domain of a PDGF receptor is encoded by the sequence of FIGS. 1A &amp; 1B (Sequence ID Number 1) from amino acid 29 through amino acid 531. In yet another embodiment, a ligand-binding domain of a PDGF receptor is encoded by the sequence of FIGS. 11A &amp; 11B (Sequence ID Numbers 35 and 36) from amino acid 24 through amino acid 524. One skilled in the art may envision the use of a smaller DNA sequence encoding the ligand-binding domain of a PDGF receptor containing at least 400 amino acids of the extracellular domain.</p>
    <p>DNA sequences encoding EGF-R (Ullrich et al., Nature 3: 418-425, 1984), the insulin receptor (Ullrich et al., Nature 313: 756-761, 1985), nerve growth factor (Ullrich et al. Nature 303: 821-825, 1983), colony stimulating factor-1 (Rettenmier et al., ibid.), transforming growth factor β (Derynck et al., Nature 3: 701-705, 1985), PDGF (Murray et al., ibid.), and factor XIII (Ichinose et al., ibid.) may also be used within the present invention.</p>
    <p>To direct polypeptides requiring dimerization for biological activity or receptor analogs into the secretory pathway of the host cell, at least one secretory signal sequence is used in conjunction with the DNA sequence of interest. Preferred secretory signals include the alpha factor signal sequence (pre-pro sequence) (Kurjan and Herkowitz, Cell 30: 933-943, 1982; Kurjan et al., U.S. Pat. No. 4,546,082; Brake, EP 116,201, 1983), the PHO5 signal sequence (Beck et al., WO 86/00637), the BAR1 secretory signal sequence (MacKay et al., U.S. Pat. No. 4,613,572; MacKay, WO 87/002670), immunoglobulin V<sub>H</sub> signal sequences (Loh et al., Cell 33: 85-93, 1983; Watson Nuc, Acids, Res. 12: 5145-5164, 1984) and immunoglobulin V.sub.κ  signal sequences (Watson, ibid.). Particularly preferred signal sequences are the SUC2 signal sequence (Carlson et al., Mol. Cell. Biol. 3: 439-447, 1983) and PDGF receptor signal sequences. Alternatively, secretory signal sequences may be synthesized according to the rules established, for example, by von Heinje (Eur. J. Biochem. 133: 17-21, 1983; J. Mol. Biol, 184: 99-105, 1985; Nuc. Acids. Res. 14: 4683-3690, 1986).</p>
    <p>Secretory signal sequences may be used singly or may be combined. For example, a first secretory signal sequence may be used singly or combined with a sequence encoding the third domain of Barrier (described in co-pending commonly assigned U.S. patent application Ser. No. 07/104,316, which is incorporated by reference herein in its entirety). The third domain of Barrier may be positioned in proper reading frame 3' of the DNA sequence of interest or 5' to the DNA sequence and in proper reading frame with both the secretory signal sequence and the DNA sequence of interest.</p>
    <p>In one embodiment of the present invention, a sequence encoding a dimerizing protein is joined to a sequence encoding a polypeptide chain of a polypeptide dimer or a receptor analog, and this fused sequence is joined in proper reading frame to a secretory signal sequence. As shown herein, the present invention utilizes such an arrangement to drive the association of the polypeptide or receptor analog to form a biologically active molecule upon secretion. Suitable dimerizing proteins include the S. cerevisiae repressible acid phosphatase (Mizunaga et al., J. Biochem. (Tokyo) 103: 321-326, 1988), the S. cerevisiae type 1 killer preprotoxin (Sturley et al., EMBO J. 5: 3381-3390, 1986), the S. calsbergensis alpha galactosidase melibiase (Sumner-Smith et al., Gene 36: 333-340, 1985), the S. cerevisiae invertase (Carlson et al., Mol. Cell. Biol. 3: 39-447, 1983), the Neurospora crassa ornithine decarboxylase (Digangi et al., J. Biol. Chem. 6: 7889-7893, 1987), immunoglobulin heavy chain hinge regions (Takahashi et al., Cell 29: 671-679, 1982), and other dimerizing immunoglobulin sequences. In a preferred embodiment, S. cerevisiae invertase is used to drive the association of polypeptides into dimers. Portions of dimerizing proteins, such as those mentioned above, may be used as dimerizing proteins where those portions are capable of associating as a dimer in a covalent or noncovlent manner. Such portions may be determined by, for example, altering a sequence encoding a dimerizing protein through In vitro mutagenesis to delete portions of the coding sequence. These deletion mutants may be expressed in the appropriate host to determine which portions retain the capability of associating as dimers. Portions of immunoglobulin gene sequences may be used to drive the association of non-immunoglobulin polypeptides. These portions correspond to discrete domains of immunoglobulins. Immunoglobulins comprise variable and constant regions, which in turn comprise discrete domains that show similarity in their three-dimensional conformations. These discrete domains correspond to immunoglobulin heavy chain constant region domain exons, immunoglobulin heavy chain variable region domain exons, immunoglobulin light chain varable region domain exons and immunoglobulin light chain constant region domain exons in immunoglobulin genes (Hood et al., in Immunology, The Benjamin/Cummings Publishing Company, Inc., Menlo Park, Calif.; Honjo et al., Cell 18: 559-568, 1979; Takahashi et al., Cell 29: 671-679, 1982; and Honjo, Ann. Rev. Immun. 1:499-528, 1983)). Particularly preferred portions of immunoglobulin heavy chains include Fab and Fab' fragments. (An Fab fragment is a portion of an immunoglobulin heavy chain that includes a heavy chain variable region domain and a heavy chain constant region omain. An Fab' fragment is a portion of an mmunoglobulin heavy chain that includes a heavy chain variable region domain, a heavy chain constant region domain and a heavy chain hinge region domain.)</p>
    <p>It is preferred to use an immunoglobulin light chain constant region in association with at least one immunoglobulin heavy chain constant region domain. In a another embodiment, an immunoglobulin light chain constant region is associated with at least one immunoglobulin heavy chain constant region domain joined to an immunoglobulin hinge region. In one set of embodiments, an immunoglobulin light chain constant region joined in frame with a polypeptide chain of a non-immunoglobulin polypeptide dimer or receptor analog and is associated with at least one heavy chain constant region. In a preferred set of embodiments a variable region is joined upstream of and in proper reading frame with at least one immunoglobulin heavy chain constant region. In another set of embodiments, an immunoglobulin heavy chain is joined in frame with a polypeptide chain of a non-immunoglobulin polypeptide dimer or receptor analog and is associated with an immunoglobulin light chain constant region. In yet another set of embodiments, a polypeptide chain of a non-immunoglobulin polypeptide dimer or receptor analog is joined to at least one immunoglobulin heavy chain constant region which is joined to an immunoglobulin hinge region and is associated with an immunoglobulin light chain constant region. In a preferred set of embodiments an immunoglobulin variable region is joined upstream of and in proper reading frame with the immunoglobulin light chain constant region.</p>
    <p>Immunoglobulin heavy chain constant region domains include C<sub>H</sub> 1, C<sub>H</sub> 2, C<sub>H</sub> 3, and C<sub>H</sub> 4 of any class of immunoglobulin heavy chain including γ, α, ε, μ, and δ classes (Honjo, ibid., 1983) A particularly preferred immunoglobulin heavy chain constant region domain is human C<sub>H</sub> 1. Immunoglobulin variable regions include V<sub>H</sub>, V.sub.κ, or V.sub.λ.</p>
    <p>DNA sequences encoding immunoglobulins may be cloned from a variety of genomic or CDNA libraries known in the art. The techniques for isolating such DNA sequences using probe-based methods are conventional techniques and are well known to those skilled in the art. Probes for isolating such DNA sequences may be based on published DNA sequences (see, for example, Hieter et al., Cell 2: 197-207, 1980). Alternatively, the polymerase chain reaction (PCR) method disclosed by Mullis et al. (U.S. Pat. No. 4,683,195) and Mullis (U.S. Pat. No. 4,683,202), incorporated herein by reference may be used. The choice of library and selection of probes for the isolation of such DNA sequences is within the level of ordinary skill in the art.</p>
    <p>Host cells for use in practicing the present invention include eukaryotic cells capable of being transformed or transfected with exogenous DNA and grown in culture, such as cultured mammalian and fungal cells. Fungal cells, including species of yeast (e.g., Saccharomyces spp., Schizosaccharomyces spp.), or filamentous fungi (e.g., Aspergillus spp., Neurospora spp.) may be used as host cells within the present invention. Strains of the yeast Saccharomyces cerevisiae are particularly preferred.</p>
    <p>Expression units for use in the present invention will generally comprise the following elements, operably linked in a 5' to 3' orientation: a transcriptional promoter, a secretory signal sequence a DNA sequence encoding nonimmunoglobulin polypeptide requiring dimerization for biological activity joined to a dimerizing protein and a transcriptional terminator. The selection of suitable promoters, signal sequences and terminators will be determined by the selected host cell and will be evident to one skilled in the art and are discussed more specifically below.</p>
    <p>Suitable yeast vectors for use in the present invention include YRp7 (Struhl et al., Proc. Natl. Acad. Sci. USA 7: 1035-1039, 1978), YEp13 (Broach et al., Gene 8: 121-133, 1979), pJDB249 and pJDB219 (Beggs, Nature 275:104-108, 1978) and derivatives thereof. Such vectors will generally include a selectable marker, which may be one of any number of genes that exhibit a dominant phenotype for which a phenotypic assay exists to enable transformants to be selected. Preferred selectable markers are those that complement host cell auxotrophy, provide antibiotic resistance or enable a cell to utilize specific carbon sources, and include LEU2 (Broach et al. ibid.), URA3 (Botstein et al., Gene p: 17, 1979), HIS3 (Struhl et al., ibid.) or POTI (Kawasaki and Bell, EP 171,142). Other suitable selectable markers include the CAT gene, which confers chloramphenicol resistance on yeast cells.</p>
    <p>Preferred promoters for use in yeast include promoters from yeast glycolytic genes (Hitzeman et al., J. Biol. Chem. 255: 12073-12080, 1980; Alber and Kawasaki, J. Mol. A Genet. 1: 419-434, 1982; Kawasaki, U.S. Pat. No. 4,599,311) or alcohol dehydrogenase genes (Young et al., in Genetic Engineering of Microorganisms for Chemicals, Hollaender et al., (eds.), p. 355, Plenum, New York, 1982; Ammerer, Meth. Enzymol. 101: 192-201, 1983). In this regard, particularly preferred promoters are the TPI1 promoter (Kawasaki, U.S. Pat. No. 4,599,311, 1986) and the ADH2-4<sup>c</sup>  promoter (Russell et al., Nature 304: 652-654, 1983 and Irani and Kilgore, described in pending, commonly assigned U.S. patent application Ser. No. 07/183,130, which is incorporated herein by reference). The expression units may also include a transcriptional terminator. A preferred transcriptional terminator is the TPI1 terminator (Alber and Kawasaki, ibid.).</p>
    <p>In addition to yeast, proteins of the present invention can be expressed in filamentous fungi, for example, strains of the fungi Aspercillus (McKnight and Upshall, described in commonly assigned U.S. Pat. No. 4,935,349, which is incorporated herein by reference). Examples of useful promoters include those derived from Aspergillus nidulans glycolytic genes, such as the ADH3 promoter (McKnight et al., EMBO J. 4: 2093-2099, 1985) and the tpiA promoter. An example of a suitable terminator is the ADH3 terminator (McKnight et al., ibid.). The expression units utilizing such components are cloned into vectors that are capable of insertion into the chromosomal DNA of Aspergillus.</p>
    <p>Techniques for transforming fungi are well known in the literature, and have been described, for instance, by Beggs (ibid.), Hinnen et al. (Proc. Natl. Acad. Sci. USA 75: 1929-1933, 1978), Yelton et al., (Proc. Natl. Acad. Sci. USA 81: 1740-1747, 1984), and Russell (Nature 301: 167-169, 1983). The genotype of the host cell will generally contain a genetic defect that is complemented by the selectable marker present on the expression vector. Choice of a particular host and selectable marker is well within the level of ordinary skill in the art.</p>
    <p>In a preferred embodiment, a Saccharomyces cerevisiae host cell that contains a genetic deficiency in a gene required for asparagine-linked glycosylation of glycoproteins is used. Preferably, the S. cerevisiae host cell contains a genetic deficiency in the MNN9 gene (described in pending, commonly assigned U.S. patent application Ser. Nos. 116,095 and 189,547 which are incorporated by reference herein in their entirety). Most preferably, the S. cerevisiae host cell contains a disruption of the MNN9 gene. S. cerevisiae host cells having such defects may be prepared using standard techniques of mutation and selection. Ballou et al. (J. Biol. Chem. 255: 5986-5991, 1980) have described the isolation of mannoprotein biosynthesis mutants that are defective in genes which affect asparagine-linked glycosylation. Briefly, mutagenized S. cerevisiae cells were screened using fluoresceinated antibodies directed against the outer mannose chains present on wild-type yeast. Mutant cells that did not bind antibody were further characterized and were found to be defective in the addition of asparagine-linked oligosaccharide moieties. To optimize production of the heterologous proteins, it is preferred that the host strain carries a mutation, such as the S. cerevisiae pep4 mutation (Jones, Genetics 85: 23-33, 1977), which results in reduced proteolytic activity.</p>
    <p>In addition to fungal cells, cultured mammalian cells may be used as host cells within the present invention. Preferred cell lines are rodent myeloma cell lines, which include p3X63Ag8 (ATCC TIB 9), OF (ATCC CRL 1646), NS-1 (ATCC TIB 18) and 210-RCY-Agl (Galfre et al., Nature 277: 131, 1979). A particularly preferred rodent rmyeloma cell line is SP2/0-Ag14 (ATCC CRL 1581). In addition, a number of other cell lines may be used within the present invention, including COS-1 (ATCC CRL 1650), BHK, p363.Ag.8.653 (ATCC CRL 1580) Rat Hep I (ATCC CRL 1600), Rat Hep II (ATCC CRL 1548), TCMK (ATCC CCL 139), Human lung (ATCC CCL 75.1), Human hepatoma (ATCC HTB-52), Hep G2 (ATCC HB 8065), Mouse liver (ATCC CC 29.1), 293 (ATCC CRL 1573; Graham et al., J. Gen. Virol. 36: 59-72, 1977) and DUKX cells (Urlaub and Chasin, Proc. Patl. Acad. Sci USA 77: 4216-4220, 1980) A preferred BHK cell line is the tk<sup>--</sup> ts13 BHK cell line (Waechter and Baserga, Proc. Natl. Acad. Sci USA 79: 1106-1110, 1982). A preferred BHK cell line is the tk-ts13 BHK cell line (Waechter and Baserga, Proc. Natl. Acad. Sci. USA 79: 1106-1110, 1982). A tk<sup>-</sup>  BHK cell line is available from the American Type Culture Collection, Rockville, Md., under accession number CRL 1632. A particularly preferred tk<sup>-</sup>  BHK cell line is BHK 570 which is available from the American Type Culture Collection under accession number 10314.</p>
    <p>Mammalian expression vectors for use in carrying out the present invention will include a promoter capable of directing the transcription of a cloned gene or cDNA. referred promoters include viral promoters and cellular promoters. Preferred viral promoters include the major late promoter from adenovirus 2 (Kaufman and Sharp, Mol. Cell. Biol. 2: 1304-13199, 1982) and the SV40 promoter (Subramani et al., Mol. Cell. Biol, 1: 854-864, 1981). Preferred cellular promoters include the mouse metallothionein 1 promoter (Palmiter et al., Science 222: 809-814, 1983) and a mouse V.sub.κ promoter (Grant et al., Nuc. Acids Res. 11: 5496, 1987). A particularly preferred promoter is a mouse V<sub>H</sub> promoter (Loh et al., ibid.). Such expression vectors may also contain a set of RNA splice sites located downstream from the promoter and upstream from the DNA sequence encoding the peptide or protein of interest. Preferred RNA splice sites may be obtained from adenovirus and/or immunoglobulin genes. Also contained in the expression vectors is a polyadenylation signal located downstream of the coding sequence of interest. Polyadenylation signals include the early or late polyadenylation signals from SV40 (Kaufman and Sharp, ibid.), the polyadenylation signal from the adenovirus 5 E1B region and the human growth hormone gene terminator (DeNoto et al., Nuc. Acids Res. 2: 3719-3730, 1981). A particularly preferred polyadenylation signal is the V<sub>H</sub> gene terminator (Loh et al., ibid.). The expression vectors may include a noncoding viral leader sequence, such as the adenovirus 2 tripartite leader, located between the promoter and the RNA splice sites. Preferred vectors may also include enhancer sequences, such as the SV40 enhancer and the mouse μ enhancer (Gillies, Cell 33: 717-728, 1983). Expression vectors may also include sequences encoding the adenovirus VA RNAs.</p>
    <p>Cloned DNA sequences may be introduced into cultured mammalian cells by, for example, calcium phosphate-mediated transfection (Wigler et al., Cell 14: 725, 1978; Corsaro and Pearson, Somatic Cell Genetics 7: 603, 1981; Graham and Van der Eb, Virology 52: 456, 1973.) Other techniques for introducing cloned DNA sequences into mammalian cells, such as electroporation (Neumann et al., EMBO J. 1: 841-845, 1982), may also be used. In order to identify cells that have integrated the cloned DNA, a selectable marker is generally introduced into the cells along with the gene or cDNA of interest. Preferred selectable markers for use in cultured mammalian cells include genes that confer resistance to drugs, such as neomycin, hygromycin, and methotrexate. The selectable marker may be an amplifiable selectable marker. A preferred amplifiable selectable marker is the DHFR gene. A particularly preferred amplifiable marker is the DHFR<sup>r</sup> CDNA (Simonsen and Levinson, Proc. Natl. Adac. Sci. USA 80: 2495-2499, 1983). Selectable markers are reviewed by Thilly (Mammalian Cell Technology, Butterworth Publishers, Stoneham, Mass.) and the choice of selectable markers is well within the level of ordinary skill in the art.</p>
    <p>Selectable markers may be introduced into the cell on a separate plasmid at the same time as the gene of interest, or they may be introduced on the same plasmid. If on the same plasmid, the selectable marker and the gene of interest may be under the control of different promoters or the same promoter, the latter arrangement producing a dicistronic message. Constructs of this type are known in the art (for example, Levinson and Simonsen, U.S. Pat. No. 4,713,339). It may also be advantageous to add additional DNA, known as "carrier DNA" to the mixture which is introduced into the cells.</p>
    <p>Transfected mammalian cells are allowed to grow for a period of time, typically 1-2 days, to begin expressing the DNA sequence(s) of interest. Drug selection is then applied to select for growth of cells that are expressing the selectable marker in a stable fashion. For cells that have been transfected with an amplifiable selectable marker the drug concentration may be increased in a stepwise manner to select for increased copy number of the cloned sequences, thereby increasing expression levels.</p>
    <p>Host cells containing DNA constructs of the resent invention are grown in an appropriate growth medium. As used herein, the term "appropriate growth medium" means a medium containing nutrients required for the growth of cells. Nutrients required for cell growth may include a carbon source, a nitrogen source, essential amino acids, vitamins, minerals and growth factors. The growth medium will generally select for cells containing the DNA construct by, for example, drug selection or deficiency in an essential nutrient which are complemented by the selectable marker on the DNA construct or co-transfected with the DNA construct. Yeast cells, for example, are preferably grown in a chemically defined medium, comprising a non-amino acid nitrogen source, inorganic salts, vitamins and essential amino acid supplements. The pH of the medium is preferably maintained at a pH greater than 2 and less than 8, preferably at pH 6.5. Methods for maintaining a stable pH include buffering and constant pH control, preferably through the addition of sodium hydroxide. Preferred buffering agents include succinic acid and Bis-Tris (Sigma Chemical Co., St. Louis, Mo.). Yeast cells having a defect in a gene required for asparagine-linked glycosylation are preferably grown in a medium containing an osmotic stabilizer. A preferred osmotic stabilizer is sorbitol supplemented into the medium at a concentration between 0.1 M and 1.5 M., preferably at 0.5 M or 1.0 M. Cultured mammalian cells are generally grown in commercially available serum-containing or serum-free media. Selection of a medium appropriate for the particular cell line used is within the level of ordinary skill in the art.</p>
    <p>The culture medium from appropriately grown transformed or transfected host cells is separated from the cell material, and the presence of dimerized polypeptide fusions or secreted receptor analogs is demonstrated. A preferred method of detecting receptor analogs, for example, is by the binding of the receptors or portions of receptors to a receptor-specific antibody. An anti-receptor antibody may be a monoclonal or polyclonal antibody raised against the receptor in question, for example, an anti-PDGF receptor monoclonal antibody may be used to assay for the presence of PDGF receptor analogs. Another antibody, which may be used for detecting substance P-tagged peptides and proteins, is a commercially available rat anti-substance P monoclonal antibody which may be utilized to visualize peptides or proteins that are fused to the C-terminal amino acids of substance P. Ligand binding assays may also be used to detect the presence of receptor analogs. In the case of PDGF receptor analogs, it is preferable to use fetal foreskin fibroblasts, which express PDGF receptors, to compete against the PDGF receptor analogs of the present invention for labeled PDGF and PDGF isoforms.</p>
    <p>Assays for detection of secreted, biologically active peptide dimers and receptor analogs may include Western transfer, protein blot or colony filter. A Western transfer filter may be prepared using the method described by Towbin et al. (Proc. Hatl. Acad. Sci. USA 76: 4350-4354, 1979). Briefly, samples are electrophoresed in a sodium dodecylsulfate polyacrylamide gel. The proteins in the gel are electrophoretically transferred to nitrocellulose paper. Protein blot filters may be prepared by filtering supernatant samples or concentrates through nitrocellulose filters using, for example, a Minifold (Schleicher &amp; Schuell, Keene, N.H.). Colony filters may be prepared by growing colonies on a nitrocellulose filter that has been laid across an appropriate growth medium. In this method, a solid medium is preferred. The cells are allowed to grow on the filters for at least 12 hours. The cells are removed from the filters by washing with an appropriate buffer that does not remove the proteins bound to the filters. A preferred buffer comprises 25 mM Tris-base, 19 mM glycine, pH 8.3, 20% methanol.</p>
    <p>The dimerized polypeptide fusions and receptor analogs present on the Western transfer, protein blot or colony filters may be visualized by specific antibody binding using methods known in the art. For example, Towbin et al. (ibid.) describe the visualization of proteins immobilized on nitrocellulose filters with a specific antibody followed by a labeled second antibody, directed against the first antibody. Kits and reagents required for visualization are commercially available, for example, from Vector Laboratories, (Burlingame, Calif.), and Sigma Chemical Company (St. Louis, Mo.).</p>
    <p>Secreted, biologically active dimerized polypeptide fusions and receptor analogs may be isolated from the medium of host cells grown under conditions that allow the secretion of the biologically active dimerized polypeptide fusions and receptor analogs. The cell material is removed from the culture medium, and the biologically active dimerized polypeptide fusions and receptor analogs are isolated using isolation techniques known in the art. Suitable isolation techniques include precipitation and fractionation by a variety of chromatographic methods, including gel filtration, ion exchange chromatography and immunoaffinity chromatography. A particularly preferred purification method is immunoaffinity chromatography using an antibody directed against the receptor analog or dimerized polypeptide fusion. The antibody is preferably immobilized or attached to a solid support or substrate. A particularly preferred substrate is CNBr-activated Sepharose (Pharmacia LKB Technologies, Inc., Piscataway, N.J.). By this method, the medium is combined with the antibody/substrate under conditions that will allow binding to occur. The complex may be washed to remove unbound material, and the receptor analog or peptide dimer is released or eluted through the use of conditions unfavorable to complex formation. Particularly useful methods of elution include changes in pH, wherein the immobilized antibody has a high affinity for the ligand at a first pH and a reduced affinity at a second (higher or lower) pH; changes in concentration of certain chaotropic agents; or through the use of detergents.</p>
    <p>The secreted PDGF receptor analogs of the present invention can be used within a variety of assays for detecting the presence of and/or screening for native PDGF, PDGF isoforms or PDGF-like molecules. These assays will typically involve combining PDGF receptor analogs, which may be bound to a solid substrate such as polymeric microtiter plate wells, with a biological sample under conditions that permit the formation of receptor/ligand complexes. Screening assays for the detection of PDGF, PDGF isoforms or PDGF-like molecules will typically involve combining soluble PDGF receptor analogs with a biological sample and incubating the mixture with a PDGF isoform or mixture of PDGF isoforms bound to a solid substrate such as polymeric microtiter plates, under conditions that permit the formation of receptor/ligand complexes. Detection may be achieved through the use of a label attached to the receptor or through the use of a labeled antibody which is reactive with the receptor. Alternatively, the labeled antibody may be reactive with the ligand. A wide variety of labels may be utilized, such as radionuclides, fluorophores, enzymes and luminescers. Complexes may also be detected visually, i.e., in immunoprecipitation assays, which do not require the use of a label.</p>
    <p>Secreted PDGF receptor analogs of the present invention may also be labeled with a radioisotope or other imaging agent and used for in vivo diagnostic purposes. Preferred radioisotope imaging agents include iodine-125 and technetium-99, with technetium-99 being particularly preferred. Methods for producing protein-isotope conjugates are well known in the art, and are described by, for example, Eckelman et al. (U.S. Pat. No. 4,652,440), Parker et al. (WO 87/05030) and Wilber et al. (EP 203,764). Alternatively, the secreted receptor analogs may be bound to spin label enhancers and used for magnetic resonance (MR) imaging. Suitable spin label enhancers include stable, sterically hindered, free radical compounds such as nitroxides. Methods for labeling ligands for MR imaging are disclosed by, for example, Coffman et al. (U.S. Pat. No. 4,656,026). For administration, the labeled receptor analogs are combined with a pharmaceutically acceptable carrier or diluent, suchEs sterile saline or sterile water. Administration is preferably by bolus injection, preferably intravenously. These imaging agents are particularly useful in identifying the locations of atherosclerotic plaques, PDGF-producing tumors, and receptor-bound PDGF.</p>
    <p>The secreted PDGF receptor analogs of the present invention may also be utilized within diagnostic kits. Briefly, the subject receptor analogs are preferably provided in a lyophilized form or immobilized onto the walls of a suitable container, either alone or in conjunction with antibodies capable of binding to native PDGF or selected PDGF isoform(s) or specific ligands. The antibodies, which may be conjugated to a label or unconjugated, are generally included in the kits with suitable buffers, such as phosphate, stabilizers, inert proteins or the like. Generally, these materials are present in less than about 5% weight based upon the amount of active receptor analog, and are usually present in an amount of at least about 0.001% weight. It may also be desirable to include an inert excipient to dilute the active ingredients. Such an excipient may be present from about 1% to 99% weight of the total composition. In addition, the kits will typically include other standard reagents, instructions and, depending upon the nature of the label involved, reactants that are required to produce a detectable product. Where an antibody capable of binding to the receptor or receptor/ligand complex is employed, this antibody will usually be provided in a separate vial. The antibody is typically conjugated to a label and formulated in an analogous manner with the formulations briefly described above. The diagnostic kits, including the containers, may be produced and packaged using conventional kit manufacturing procedures.</p>
    <p>As noted above, the secreted PDGF receptor analogs of the present invention may be utilized within methods for purifying PDGF from a variety of samples. Within a preferred method, the secreted PDGF receptor analogs are immobilized or attached to a substrate or support material, such as polymeric tubes, beads, polysaccharide particulates, polysaccharide-containing materials, polyacrylamide or other water insoluble polymeric materials. Methods for immobilization are well known in the art (Mosbach et al., U.S. Pat. No. 4,415,665; Clarke et al., Meth. Enzymology : 436-442, 1979). A common method of immobilization is CNBr activation. Activated substrates are commercially available from a number of suppliers, including Pharmacia (Piscataway, N.J.), Pierce chemical Co. (Rockford, Ill.) and Bio-Rad Laboratories (Richmond, Calif.). A preferred substrate is CNBr-activated Sepharose (Pharmacia, Piscataway, N.J.). Generally, the substrate/receptor analog complex will be in the form of a column. The sample is then combined with the immobilized receptor analog under conditions that allow binding to occur. The substrate with immobilized receptor analog is first equilibrated with a buffer solution of a composition in which the receptor analog has been previously found to bind its ligand. The sample, in the solution used for equilibration, is then applied to the substrate/receptor analog complex. Where the complex is in the form of a column, it is preferred that the sample be passed over the column two or more times to permit full binding of ligand to receptor analog. The complex is then washed with the same solution to elute unbound material. In addition, a second wash solution may be used to minimize nonspecific binding. The bound material may then be released or eluted through the use of conditions unfavorable to complex formation. Particularly useful methods include changes in pH, wherein the immobilized receptor has a high affinity for PDGF at a first pH and reduced affinity at a second (higher or lower) pH; changes in concentration of certain chaotropic agents; or through the use of detergents.</p>
    <p>The secreted PDGF receptor analogs fused to dimerizing proteins of the present invention may be used in pharmaceutical compositions for topical or intravenous application. The secreted PDGF receptor analogs of the present invention may be useful in the treatment of atherosclerosis by, for example, binding endogenous PDGF to prevent smooth muscle cell proliferation. The PDGF receptor analogs fused to dimerizing proteins are used in combination with a physiologically acceptable carrier or diluent. Preferred carriers and diluents include saline and sterile water. Pharmaceutical compositions may also contain stabilizers and adjuvants. The resulting aqueous solutions may be packaged for use or filtered under aseptic conditions and lyophilized, the lyophilized preparation being combined with a sterile aqueous solution prior to administration.</p>
    <p>The following examples are offered by way of illustration and not by way of limitation.</p>
    <heading>EXAMPLES</heading> <p>Enzymes, including restriction enzymes, DNA polymerase I (Klenow fragment), T4 DNA polymerase, T4 DNA ligase and T4 polynucleotide kinase, were obtained from New England Biolabs (Beverly, Mass.), GIBCO-BRL (Gaithersburg, Md.) and Boerhinger-Mannheim Biochemicals (Indianapolis, Ind.) and were used as directed by the manufacturer or as described in Maniatis et al. (Molecular Cloning: A Laboratory Manual, Cold Spring Harbor Laboratory, NY, 1982) and Sambrook et al. (Molecular Cloning: A Laboratory Manual/Second Edition, Cold Spring Harbor Laboratory, NY, 1989).</p>
    <heading>Example 1</heading> <heading>Cloning PDGF Receptor cDNAs</heading> <p>A. Cloning the PDGF β-Receptor</p>
    <p>A cDNA encoding the PDGF β-receptor was cloned as follows. Complementary DNA (cDNA) libraries were prepared from poly(A) RNA from diploid human dermal fibroblast cells, prepared by explant from a normal adult, essentially as described by Hagen et al. (Proc, Natl. Acad. Sci. USA 83: 2412-2416, 1986). Briefly, the poly(A) RNA was primed with oligo d(T) and cloned into λgtll using Eco RI linkers. The random primed library was screened for the presence of human PDGF receptor cDNA's using three oligonucleotide probes complementary to sequences of the mouse PDGF receptor (Yarden et al., Nature 323: 226-232, 1986). Approximately one million phage from the random primed human fibroblast cell library were screened using oligonucleotides ZC904, ZC905 and ZC906 (Table 1; Sequence ID Numbers 5, 6 and 7, respectively). Eight positive clones were identified and plaque purified. Two clones, designated RP41 and RP51, were selected for further analysis by restriction enzyme mapping and DNA sequence analysis. RP51 was found to contain 356 bp of 5'-noncoding sequence, the ATG translation initiation codon and 738 bp of the amino terminal coding sequence. RP41 was found to overlap clone RP51 and contained 2649 bp encoding amino acids 43-925 of the β-receptor protein.</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">              TABLE 1______________________________________Oligonucleotide Sequences______________________________________ZC871 (Sequence ID Number 3)  5' CTC TCT TCC TCA GGT AAA TGA GTG CCA GGG CCG GCA   AGC CCC CGC TCC 3'   -  ZC872 (Sequence ID Number 4)     5' CCG GGG AGC GGG GGC TTG CCG GCC CTG GCA CTC ATT  TAC CTG AGG AAG AGA CAG CT 3'   -        ZC904 (Sequence ID Number 5)         5' CAT GGG CAC GTA ATC TAT AGA TTC ATC CTT GCT CAT  ATC CAT GTA 3'   -            ZC905 (Sequence ID Number 6)             5' TCT TGC CAG GGC ACC TGG GAC ATC TGT TCC CAC ATC              ACC GG 3'   -                ZC906 (Sequence ID Number 7)                 5' AAG CTG TCC TCT GCT TCA GCC AGA GGT CCT GGG CAG                  CC 3'   -                    ZC1380 (Sequence ID Number 8)                     5' CAT GGT GGA ATT CCT GCT GAT 3'   -ZC1447 (Sequence ID Number 9)  5' TG GTT GTG CAG AGC TGA GGA AGA GAT GGA 3'   -  ZC1453 (Sequence ID Number 10)     5' AAT TCA TTA TGT TGT TGC AAG CCT TCT TGT TCC TGC  TAG CTG GTT TCG CTG TTA A 3'   -        ZC1454 (Sequence ID Number 11)         5' GAT CTT AAC AGC GAA ACC AGC TAG CAG GAA CAA GAA  GGC TTG CAA CAA CAT AAT G 3'   -            ZC1478 (Sequence ID Number 12)             5' ATC GCG AGC ATG CAG ATC TGA 3'   -                ZC1479 (Sequence ID Number 13)                 5' AGC TTC AGA TCT GCA TGC TGC CGA T 3'   -                    ZC1776 (Sequence ID Number 14)                     5' AGC TGA GCG CAA ATG TTG TGT CGA GTG CCC ACC                     GTG  CCC AGC TTA GAA TTC T 3'   -ZC1777 (Sequence ID Number 15)  5' CTA GAG AAT TCT AAG CTG GGC ACG GTG GGC ACT CGA  CAC AAC ATT TGC GCT C 3'   -  ZC1846 (Sequence ID Number 16)     5' GAT CGG CCA CTG TCG GTG CGC TGC ACG CTG CGC AAC  GCT GTG GGC CAG GAC ACG CAG GAG GTC ATC GTG GTG  CCA CAC TCC TTG CCC TTT AAG CA 3'   -        ZC1847 (Sequence ID Number 17)         5' AGC TTG CTT AAA GGG CAA GGA GTG TGG CAC CAC GAT  GAC CTC CTG CGT GTC CTG GCC CAC AGC GTT GCG CAG  CGT GCA GCG CAC CGA CAG TGG CC 3'   -            ZC1886 (Sequence ID Number 18)             5' CCA GTG CCA AGC TTG TCT AGA CTT ACC TTT AAA GGG              CAA GGA G 3'   -                ZC1892 (Sequence ID Number 19)                 5' AGC TTG AGC GT 3'   -                    ZC1893 (Sequence ID Number 20)                     5' CTA GAC GCT CA 3'   -ZC1894 (Sequence ID Number 21)  5' AGC TTC CAG TTC TTC GGC CTC ATG TCA GTT CTT CGG  CCT CAT GTG AT 3'   -  ZC1895 (Sequence ID Number 22)     5' CTA GAT CAC ATG AGG CCG AAG AAC TGA CAT GAG GCC  GAA GAA CTG GA 3'   -        ZC2181 (Sequence ID Number 23)         5' AAT TCG GAT CCA CCA TGG GCA CCA GCC ACC CGG CGT  TCC TGG TGT TAG GCT GCC TGC TGA CCG GCC 3'   -            ZC2182 (Sequence ID Number 24)             5' TGA GCC TGA TCC TGT GCC AAC TGA GCC TGC CAT CGA              TCC TGC CAA ACG AGA ACG AGA AGG TTG TGC AGC TA 3'               -               ZC2183 (Sequence ID Number 25)                 5' AAT TTA GCT GCA CAA CCT TCT CGT TCT CGT TTG GCA                  GGA TCG ATG GCA GGC TCA GTT GGC ACA GGA TCA 3'                   -                   ZC2184 (Sequence ID Number 26)                     5' GGC TCA GGC CGG TCA GCA GGC AGC CTA ACA CCA                     GGA  ACG CCG GGT GGC TGG TGC CCA TGG TGG ATC CG 3'   -ZC2311 (Sequence ID Number 27)  5' TGA TCA CCA TGG CTC AAC TG 3'   -  ZC2351 (Sequence ID Number 28)     5' CGA ATT CCA C 3'   -        ZC2352 (Sequence ID Number 29)         5' CAT GGT GGA ATT CGA GCT 3'   -            ZC2392 (Sequence ID Number 30)             5' ACG TAA GCT TGT CTA GAC TTA CCT TCA GAA CGC AGG              GTG GG 3'______________________________________</pre>
    
    <p>The 3'-end of the CDNA was not isolated in the first cloning and was subsequently isolated by screening 6×10<sup>5</sup> phage of the oligo d(T)-primed CDNA library with a 630 bp Sst I-Eco RI fragment derived from the 3'-end of clone RP41. One isolate, designated TO91, was further analyzed by restriction enzyme mapping and DNA sequencing. This clone was found to comprise the 3'-end of the receptor coding region and 1986 bp of 3' untranslated sequence.</p>
    <p>Clones RP51, RP41 and TO91 were ligated together to construct a full-length CDNA encoding the entire PDGF β-receptor. RP41 was digested with Acc I and Bam HI to isolate the 2.12 kb fragment. RP51 was digested with Eco RI and Acc I to isolate the 982 bp fragment. The 2.12 kb RP41 fragment and the 982 bp RP51 fragment were joined in a three-part ligation with pUC13, which had been linearized by digestion with Eco RI and Bam HI. The resultant plasmid was designated 51/41. Plasmid 51/41 was digested with Eco RI and Bam HI to isolate the 3 kb fragment comprising the partial PDGF receptor cDNA. OT91 was digested with Bam HI and Xba I to isolate the 1.4 kb fragment containing the 3' portion of the PDGF receptor cDNA. The Eco RI-Bam HI 51/41 fragment, the Bam HI-Xba I TO91 fragment and the Eco RI-Xba I digested pUC13 were joined in a three-part ligation. The resultant plasmid was designated pR-RX1 (FIG. 2).</p>
    <p>B. Cloning the PDGF-α Receptor</p>
    <p>A cDNA encoding to PDGF α-receptor was cloned as follows. RNA was prepared by the method of Chirgwin et al. (Biochemistry 18: 5294, 1979) and twice purified on oligo d(T) cellulose to yield poly(A)+RNA. Complementary DNA was prepared in Igtlo phage using a kit purchased from Invitrogen (San Diego, CA). The resulting λ phage DNA was packaged with a coat particle mixture from Stratagene Cloning Systems (La Jolla, Calif.), infected into E. coli strain C600 Hfl<sup>-</sup>  and titered.</p>
    <p>Approximately 1.4×10<sup>6</sup> phage recombinants were plated to produce plaques for screening. Nitrocellulose filter lifts were prepared according to standard methods and were hybridized to a radiolabeled PDGF β-receptor DNA fragment (Gronwald et al., ibid.) comprising the 1.9 kb Fsp I-Hind III fragment that encodes the transmembrane and cytoplasmic domains of the PDGF β-receptor cDNA. Hybridization was performed for 36 hours at 42° C. in a mixture containing 40% formamide, 5×SSCP (SSC containing 25 mM phosphate buffer, pH 6.5), 200 μg/ml denatured salmon sperm DNA, 3×Denhardt's, and 10% dextran sulfate. Following hybridization, the filters were washed extensively at room temperature in 2×SSC, then for 15 minutes at 47-48° C. Following an exposure to X-ray film, the filters were treated to increasingly stringent wash conditions followed by film recording until a final wash with 0.1×SSC at 65° C. was reached. Film analysis showed that a "family" of plaques that hybridized at lower wash stringency but not at the highest stringency. This "family" was selected for further analysis.</p>
    <p>Two λ phage clones from the "family" obtained from the initial screening were subcloned into the Not I site of the pUCtype plasmid vector pBluescript SK<sup>+</sup>  (obtained from Stratagene Cloning Systems, La Jolla, Calif.) and were analyzed by restriction mapping and sequence analysis. Restriction enzyme analysis of a phage clone, designated α1-1, revealed a restriction fragment pattern dissimilar from that of the PDGF β-receptor CDNA with the exception of a common Bgl II-Bgl II band of approximately 160 bp. The PDGF β-receptor cDNA contains two similarly spaced Bgl II sites within the region coding for the second tyrosine kinase domain.</p>
    <p>Restriction analysis of a second plasmid subclone (designated α1-7) revealed an overlap of the 5' approximately 1.2 kb of clone α1-1, and an additional approximately 2.2 kb of sequence extending in the 5' direction. Sequence analysis revealed that the 3' end of this clone encodes the second tyrosine kinase domain, which contains regions of near sequence identity to the corresponding regions in the PDGF β-receptor. The 5' end of clone α1-7 contained non-receptor sequences. Two additional β-receptor clones were obtained by probing with α1-1 sequences. Clone α1-1 was digested with Not I and Spe I, and a 230 bp fragment was recovered. Clone α1-1 was also digested with Bam HI and Not I, and a 550 bp fragment was recovered. A clone that hybridized to the 230 bp probe was designated α5-1. This clone contained the 5'-most coding sequence for the PDGF α-receptor. Another clone, designated α6-3, hybridized to the 550 bp probe and was found to contain 3' coding and non-coding sequences, including the poly(A) tail.</p>
    <p>Clone α1-1 was radiolabeled (<sup>32</sup> p) and used to probe a northern blot (Thomas, Methods Enzymol, 100: 225-265, 1983) of the MG-63 poly(A)+RNA used to prepare the cDNA library. A single band of approximately 6.6 kb was observed. RNA prepared from receptor-positive cell lines including the human fibroblast SK4, WI-38 and 7573 cell lines; the mouse fibroblast line DI 3T3; the U2-OS human osteosarcoma cell line and baboon aortic smooth muscle cells, and RNA prepared from receptor-negative lines including A431 (an epithelial cell line) and VA 13 (SV40-transformed WI-38 cells) were probed by northern format with the α1-1 cDNA. In all cases, the amount of the 6.6 kb band detected in these RNA correlated well with the relative levels of α-receptor detected on the respective cell surfaces. The 6.6 kb RNA was not detected in RNA prepared from any tested cell line of hematopoietic origin, in agreement with a lack of PDGF α-receptor protein detected on these cell types.</p>
    <p>Clones α1-1 and and α1-7 were joined at a unique Pst I site in the region encoding the transmembrane portion of the receptor. Clone α1-1 was digested with Xba I and Pst I and the receptor sequence fragment was recovered. Clone α1-7 was digested with Pst I and Bam HI and the receptor fragment was recovered. The two fragments were ligated with Xba I+Bam HI-digested pIC19R (Marsh et al. Gene 32: 481-486, 1984) to construct plasmid pα17R (FIG. 12).</p>
    <p>The remainder of the 5═-most α-receptor sequence was obtained from clone α5-1 as an Sst I-Cla I fragment. This fragment was joined to the Eco RI-Sst I receptor fragment of pα17R and cloned into Eco RI+Cla I-digested pBluescript SK+plasmid to construct plasmid pα17B (FIG. 12). FIGS. 11A-11D (Sequence ID Numbers 35 and 36) shows the nucleotide sequence and deduced amino acid sequence of the cDNA present in pα17B.</p>
    <heading>Example 2</heading> <heading>Construction of a SUC2 Signal Sequence-PDGF β-Receptor Fusion</heading> <p>To direct the PDGF β-receptor into the yeast secretory pathway, the PDGF β-receptor cDNA was joined to a sequence encoding the Saccharomyces cerevisiae SUC2 signal sequence. Oligonucleotides ZC1453 and ZC1454 (Sequence ID Numbers 10 and 11; Table 1) were designed to form an adapter encoding the SUC2 secretory signal flanked by a 5' Eco RI adhesive end and a 3' Bgl II adhesive end. ZC1453 and ZC1454 were annealed under conditions described by Maniatis et al. (ibid.). Plasmid pR-RX1 was digested with Bgl II and Sst II to isolate the 1.7 kb fragment comprising the PDGF β-receptor coding sequence from amino acids 28 to 596. Plasmid pR-RX1 was also cut with Sst II and Hind III to isolate the 1.7 kb fragment comprising the coding sequence from amino acids 597 through the translation termination codon and 124 bp of 3' untranslated DNA. The two 1.7 kb pR-RX1 fragments and the ZC1453/ZC1454 adapter were joined with pUC19, which had been linearized by digestion with Eco RI and Hind III. The resultant plasmid, comprising the SUC2 signal sequence fused in-frame with the PDGF β-receptor cDNA, was designated pBTL10 (FIG. 2).</p>
    <heading>Example 3</heading> <heading>Construction of pCBS22</heading> <p>The BAR1 gene product, Barrier, is an exported protein that has been shown to have three domains. When used in conjunction with a first signal sequence, the third domain of Barrier protein has been shown to aid in the secretion of proteins into the medium (MacKay et al., U.S. patent application Ser. No. 104,316).</p>
    <p>The portion of the BAR1 gene encoding the third domain of Barrier was joined to a sequence encoding the C-terminal portion of substance P (subP; Munro and Pelham, EMBO J. 3: 3087-3093, 1984). The presence of the substance P amino acids on the terminus of the fusion protetn allowed the protein to be detected using commercially available anti-substance P antibodies. Plasmid pZV9 (deposited as a transformant in E. coli strain RR1, ATCC accession no. 53283), comprising the entire BAR2 coding region and its associated flanking regions, was cut with Sal I and Bam HI to isolate the 1.3 kb BAR1 fragment. This fragment was subcloned into pUC13, which had been cut with Sal I and Bam HI, to generate the plasmid designated pZV17. Plasmid pZV17 was digested with Eco RI to remove the 3'-most 0.5 kb of the BAR1 coding region. The vector-BAR1 fragment was religated to create the plasmid designated pJH66 (FIG. 3). Plasmid pJH66 was linearized with Eco RI and blunt-ended with DNA polymerase I (Klenow fragment). Kinased Bam HI linkers (5'CCG GAT CCG G 3') were added and excess linkers were removed by digestion with Bam HI before religation. The resultant plasmid was designated pSW8 (FIG. 3).</p>
    <p>Plasmid pSW81, comprising the TPI1 promoter, the BAR1 coding region fused to the coding region of the C-terminal portion of substance P (Munro and Pelhamr, EMBO J. 3: 3087-3093, 1984) and the TPI1 terminator, was derived from pSW8. Plasmid pSW8 was cut with Sal I and Bam HI to isolate the 824 bp fragment encoding amino acids 252 through 526 of BAR1. Plasmid pPM2, containing the synthetic oligonucleotide sequence encoding the dimer form of the C-terminal portion of substance P (subP) in M13mp8, was obtained from Hugh Pelham (MRC Laboratory of Molecular Biology, Cambridge, England). Plasmid pPM2 was linearized by digestion with Bam HI and Sal I and ligated with the 824 bp BAR1 fragment from pSW8. The resultant plasmid, pSW14, was digested with Sal I and Sma I to isolate the 871 bp BAR1-substance P fragment. Plasmid pSW16, comprising a fragment of BAR1 encoding amino acids 1 through 250, was cut with Xba I and Sal I to isolate the 767 bp BAR1 fragment. This fragment was ligated with the 871 bp BAR1-substance P fragment in a three-part ligation with pUC18 cut with Xba I and Sma I. The resultant plasmid, designated pSW15, was digested with Xba I and Sma I to isolate the 1.64 kb BAR1-substance P fragment. The ADH1 promoter was obtained from pRL029. Plasmid pRL029, comprising the ADH1 promoter and the BAR1 5' region encoding amino acids 1 to 33 in pUC18, -was digested with Sph I and Xba I to isolate the 0.42 kb ADH1 promoter fragment. The TPI1 terminator (Alber and Kawasaki, ibid.) was provided as a linearized fragment containing the TPI1 terminator and pUC18 with a Klenow-filled Xba I end and an Sph I end. This fragment was ligated with the 0.42 kb ADH1 promoter fragment and the 1.64 kb BAR1-substance P fragment in a three-part ligation to produce plasmid pSW22.</p>
    <p>The ADH1 promoter and the coding region of BAR1, from the translation initiation ATG through the Eco RV site present in pSW22, were removed by digestion with Hind III and Eco RV. The 3.9 kb vector fragment, comprising the 401 bp between the Eco RV and the Eco RI sites of the BAR1 gene fused to subp and the TPI1 terminator, was isolated by gel electrophoresis. Oligonucleotide ZC1478 (Sequence ID Number 12; Table 1) was kinased and annealed with oligonucleotide ZC1479 (Sequence ID Number 13; Table 1) using conditions described by Maniatis et al. (ibid.). The annealed oligonucleotides formed an adapter comprising a Hind III adhesive end and a polylinker encoding Bgl II, Sph I, Nru I and Eco RV restriction sites. The ZC1479/ZC1478 adapter was ligated with the gel-purified pSW22 fragment. The resultant plasmid was designated pCBS22 (FIG. 3).</p>
    <heading>Example 4</heading> <heading>Construction of pBTL13</heading> <p>In order to enhance the secretion of the β-receptor and to facilitate the identification of the secreted protein, a sequence encoding the third domain of BAR1 fused to the C-terminal amino acids of substance P was fused in frame with the 5' 1240 bp of the PDGF β-receptor. Plasmid pBTL10 (Example 2) was digested with Sph I and Sst I to isolate the 4 kb fragment comprising the SUC2 signal sequence, a portion of the PDGF β-receptor CDNA and the pUC19 vector sequences. Plasmid pCBS22 was digested with Sph I and Sst I to isolate the 1.2 kb fragment comprising the BAR1-subp fusion and the TPI1 terminator. These two fragments were ligated, and the resultant plasmid was designated pBTL13 (FIG. 4).</p>
    <heading>Example 5</heading> <heading>Construction of an Expression Vector Encoding the Entire PDGF β-Receptor</heading> <p>The entire PDGF β-receptor was directed into the secretory pathway by fusing a SUC2 signal sequence to the 5' end of the PDGF β-receptor coding sequence. This fusion was placed behind the TPIL promoter and inserted into the vector YEp13 for expression in yeast.</p>
    <p>The TPI1 promoter was obtained from plasmid pTPIC10 (Alber and Kawasaki, J. Mol. Apnl. Genet. 1: 410-434, 1982), and plasmid PFATPOT (Kawasaki and Bell, EP 171,142; ATCC 20699). Plasmid pTPIC10 was cut at the unique Kpn I site, the TPI1 coding region was removed with Bal-31 exonuclease, and an Eco RI linker (sequence: GGA ATT CC) was added to the 3' end of the promoter. Digestion with Bgl II and Eco RI yielded a TPI1 promoter fragment having Bgl II and Eco RI sticky ends. This fragment was then joined to plasmid YRp7' (Stinchcomb et al., Nature 282: 39-43, 1979) that had been cut with Bgl II and Eco RI (partial). The resulting plasmid, TE32, was cleaved with Eco RI (partial) and Bam HI to remove a portion of the tetracycline resistance gene. The linearized plasmid was then recircularized by the addition of an Eco RI-Bam HI linker to produce plasmid TEA32. Plasmid TEA32 was digested with Bgl II and Eco RI, and the 900 bp partial TPI1 promoter fragment was gel-purified. Plasmid pIC19H (Marsh et al., Gene 32:481-486, 1984) was cut with Bgl II and Eco RI and the vector fragment was gel purified. The TPI1 promoter fragment was then ligated to the linearized pIC19H and the mixture was used to transform E. coli RR1. Plasmid DNA was prepared and screened for the presence of a ˜900 bp Bgl II-Eco RI fragment. A correct plasmid was selected and designated PICTPIP.</p>
    <p>The TPI1 promoter was then subcloned to place convenient restriction sites at its ends. Plasmid pIC7 (Marsh et al., ibid.) was digested with Eco RI, the fragment ends were blunted with DNA polymerase I (Klenow fragment), and the linear DNA was recircularized using T4 DNA ligase. The resulting plasmid was used to transform E. coli RR1. Plasmid DNA was prepared from the transformants and was screened for the loss of the Eco RI site. A plasmid having the correct restriction pattern was designated pIC7RI*. Plasmid pIC7RI* was digested with Hind III and Nar I, and the 2500 bp fragment was gel-purified. The partial TPI1 promoter fragment (ca. 900 bp) was removed from pICTPIP using Nar I and Sph I and was gel-purified. The remainder of the TPI1 promoter was obtained from plasmid pFATPOT by digesting the plasmid with Sph I and Hind III, and a 1750 bp fragment, which included a portion of the TPI1 promoter fragment from PICTPIP, and the fragment from pFATPOT were then combined in a triple ligation to produce pMVR1 (FIG. 2).</p>
    <p>The TPI1 promoter was then joined to the SUC2-PDGF β-receptor fusion. Plasmid PBTL10 (Example 2) was digested with Eco RI and Hind III to isolate the 3.4 kb fragment comprising the SUC2 signal sequence and the entire PDGF β-receptor coding region. Plasmid pMVR1 was digested with Bgl II and Eco RI to isolate the 0.9 kb TPI1 promoter fragment. The TPI1 promoter fragment and the fragment derived from pBTL10 were joined with YEp13, which had been linearized by digestion with Bam HI and Hind III, in a three-part ligation. The resultant plasmid was designated pBTL12 (FIG. 2).</p>
    <heading>Example 6</heading> <heading>Construction of an Expression Vector Encoding the 5' Extracellular Portion of the PDGF β-Receptor</heading> <p>The extracellular portion of the PDGF β-receptor was directed into the secretory pathway by fusing the coding sequence to the SUC2 signal sequence. This fusion was placed in an expression vector behind the TPI1 promoter. Plasmid pBTL10 (Example 2) was digested with Eco RI and Sph I to isolate the approximately 1.3 kb fragment comprising the SUC2 signal sequence and the PDGF β-receptor extracellular domain coding sequence. Plasmid pMVR1 (Example 5) was digested with Bgl II and Eco RI to isolate the 0.9 kb TPI1 promoter fragment. The TPI1 promoter fragment was joined with the fragment derived from pBTL10 and YEp13, which had been linearized by digestion with Bam HI and Sph I, in a three-part ligation. The resultant plasmid was designated pBTL11 (FIG. 2).</p>
    <heading>Example 7</heading> <p>Construction of Yeast Expression Vectors pBTL14 and pBTL15, and The Expression of PDGF β-Receptor-BAR1-subP</p>
    <heading>Fusions</heading> <p>A. Construction of pBTL14</p>
    <p>The SUC2-PDGFβ-R fusion was joined with the third domain of BAR1 to enhance the secretion of the receptor, and the expression unit was cloned into a derivative of YEp13 termed pJH50. YEp13 was modified to destroy the Sal I site near the LEU2 gene. This was achieved by partially digesting YEp13 with Sal I followed by a complete digestion with Xho I. The 2.0 kb Xho I-Sal I fragment comprising the LEU2 gene and the 8.0 kb linear YEp13 vector fragment were isolated and ligated together. The ligation mixture was transformed into E. coli strain RR1. DNA was prepared from the transformants and was analyzed by digestion with Sal I and Xho I. A clone was isolated which showed a single Sal I site and an inactive Xho I site indicating that the LEU2 fragment had inserted in the opposite orientation relative to the parent plasmid YEp13. The plasmid was designated pJH50.</p>
    <p>Referring to FIG. 4, plasmid pBTL12 (Example 5) was digested with Sal I and Pst I to isolate the 2.15 kb fragment comprising 270 bp of YEp13 vector sequence, the TPI1 promoter, the SUC2 signal sequence, and 927 bp of PDGF β-receptor cDNA. Plasmid pBTL13 (Example 4) was digested with Pst I and Hind III to isolate the 1.48 kb fragment comprising 313 bp of PDGF β-receptor CDNA, the BAR1-subP fusion and the TPI1 terminator. The fragments derived from pBTL12 and pBTL13 were joined with pJH50, which had been linearized by digestion with Hind III and Sal I, in a three-part ligation. The resultant plasmid was designated pBTL14.</p>
    <p>B. Construction of pBTL15</p>
    <p>Referring to FIG. 5, a yeast expression vector was constructed comprising the TPI1 promoter, the SUC2 signal sequence, 1.45 kb of PDGF β-receptor CDNA sequence fused to the BAR1-subP fusion and the TPI1 terminator. Plasmid pBTL12 (Example 5) was digested with Sal I and Fsp I to isolate the 2.7 kb fragment comprising the TPI1 promoter, the SUC2 signal sequence, the PDGFβ-R coding sequence, and 270 bp of YEp13 vector sequence. Plasmid pBTL13 (Example 4) was digested with Nru I and Hind III to isolate the 1.4 kb fragment comprising the BAR1-subP fusion, the TPI1 terminator and 209 bp of 3' PDGF β-receptor cDNA sequence. The fragments derived from pBTL12 and pBTL13 were joined in a three-part ligation with pJH50, which had been linearized by digestion with Hind III and Sal I. The resultant plasmid was designated pBTL15.</p>
    <p>C. Expression of PDGFβ-R-subP Fusions from pBTL14 and pBTL15</p>
    <p>Yeast expression vectors pBTL14 and pBTL15 were transformed into Saccharomyces cerevisiae strains ZY100 (MATa leu2-3,112 ade2-101 suc2-A9 ca12 pep4::TPI1prom-CAT) and ZY400 (MATa leu2-3.112 ade2-101 suc2-Δ9 gal2 pep4::TPI1irom-CAT Δmnn9::URA3). Transformations were carried out using the method essentially described by Beggs (ibid.). Transformants were selected for their ability to grow on -LEUDS (Table 2).</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">              TABLE 2______________________________________Media RecipesLeuThrTrpAmino Acid Mixture______________________________________   4 g adenine  3 g L-arginine  5 g L-aspartic acid  2 g L-histidine free base  6 g L-isoleucine  4 g L-lysine-mono hydrochloride  2 g L-methionine  6 g L-phenylalanine  5 g L-serine  5 g L-tyrosine  4 g uracil  6 g L-valine______________________________________</pre>
    
    <p>Mix all the ingredients and grind with a mortar and pestle until the mixture is finely ground.</p>
    <p>-LEUDS</p>
    <p>20 g glucose</p>
    <p>6.7 g Yeast Nitrogen Base without amino acids (DIFCO Laboratories Detroit, Mich.)</p>
    <p>0.6 g -LeuThrTrp Amino Acid Mixture</p>
    <p>182.2 g sorbitol</p>
    <p>18 g Agar</p>
    <p>Mix all the ingredients in distilled water. Add distilled water to a final volume of 1 liter. Autoclave 15 minutes. After autoclaving add 150 mg L-threonine and 40 mg L-tryptophan. Pour plates and allow to solidify.</p>
    <p>-LEUDS+sodium succinate. pH 6.5</p>
    <p>20 g Yeast Nitrogen Base without amino acids</p>
    <p>0.6 g -LeuTrpThr Amino Acid Mixture</p>
    <p>182.2 g sorbitol</p>
    <p>11.8 g succinic acid</p>
    <p>Mix all ingredients in distilled water to a final volume of 1 liter. Adjust the pH of the solution to pH 6.5. Autoclave 15 minutes. After autoclaving add 150 mg L-threonine and 40 mg L-tryptophan.</p>
    <p>Fermentation Medium</p>
    <p>7 g/l yeast nitrogen base without amino acids or ammonium sulfate (DIFCO Laboratories)</p>
    <p>0.6 g/l ammonium sulfate</p>
    <p>0.5 M sorbitol</p>
    <p>0.39 g/l adenine sulfate</p>
    <p>0.01% polypropylene glycol</p>
    <p>Mix all ingredients in distilled water. Autoclave 15 minutes. Add 80 ml 50% glucose for each liter of medium.</p>
    <p>Super Synthetic -LEUD. pH 6.5 [liquid or solid medium]</p>
    <p>6.7 g Yeast Nitrogen Base without amino acids or ammonium sulfate (DIFCO)</p>
    <p>6 g ammonium sulfate</p>
    <p>160 g adenine</p>
    <p>0.6 g -LeuThrTrp Amino Acid Mixture</p>
    <p>20 g glucose</p>
    <p>11.8g succinic acid</p>
    <p>Mix all ingredients and add distilled water to a final volume of 800 ml. Adjust the pH of the solution to pH 6.4. Autoclave 15 minutes. For solid medium, add 18 g agar before autoclaving, autoclave and pour plates.</p>
    <p>Super Synthetic -LEUDS, pH 6,4 (liquid or solid medium)</p>
    <p>Use the same recipe as Super Synthetic -LEUD, pH 6.4, but add 182.2 g sorbitol before autoclaving.</p>
    <p>YEPD</p>
    <p>20 g glucose</p>
    <p>20 g Bacto Peptone (DIFCO Laboratories)</p>
    <p>10 g Bacto Yeast Extract (DIFCO Labloratories)</p>
    <p>18 g agar</p>
    <p>4 ml adenine 1%</p>
    <p>8 ml 1% L-leucine</p>
    <p>Mix all ingredients in distilled water, and bring to a final volume of 1 liter. Autoclave 25 minutes and pour plates.</p>
    <p>The transformants were assayed for binding to an anti-PDGF β-receptor monoclonal antibody (PR7212) or an anti-substance P antibody by protein blot assay. ZY100[pBTL14] and ZY100[pBTL15] transformants were grown overnight at 30° C. in 5 ml Super Synthetic -LEUD, pH 6.4 (Table 2). ZY400[pBTL14] and ZY400[pBTL15] transformants were grown overnight at 30° C. in 5 ml Super Synthetic-LEUDS, pH 6.4 (Table 2). The cultures were pelleted by centrifugation and the supernatants were assayed for the presence of secreted PDGF β-receptor anlogs by protein blot assay using methods described in Example 18. Results of assays using PR7212 are shown in Table 3.</p>
    <heading>Example 8</heading> <heading>Construction of a SUC2-PDGFβ-R Fusion Comprising the Complete PDGFβ-R Extracellular Domain</heading> <p>A. Construction of pBTL22</p>
    <p>The PDGFβ-R coding sequence present in pBTL10 was modified to delete the coding region 3' to the extracellular PDGFβ-R domain. As shown in FIG. 6, plasmid pBTL10 was digested with Sph I and Bam HI and with Sph I and Sst II to isolate the 4.77 kb fragment and the 466 bp fragment, respectively. The 466 bp fragment was then digested with Sau 3A to isolate the 0.17 kb fragment. The 0.17 kb fragment and the 4.77 kb were joined by ligation. The resultant plasmid was designated pBTL21.</p>
    <p>Plasmid pBTL21, containing a Bam HI site that was regenerated by the ligation of the Bam HI and Sau 3A sites, was digested with Hind III and Bam HI to isolate the 4.2 kb fragment. Synthetic oligonucleotides ZC1846 (Sequence ID Number 16; Table 1) and ZC1847 (Sequence ID Number 17; Table 1) were designed to form an adapter encoding the PDGFβ-R from the Sau 3A site after bp 1856 (FIG. 1B; (Sequence ID Number 1)) to the end of the extracellular domain at 1958 bp (FIG. 1B; Sequence ID Number 1), having a 5' Bam HI adhesive end that destroys the Bam HI site and a 3' Hind III adhesive end. oligonucleotides ZC1846 and ZC1847 were annealed under conditions described by Maniatis et. al. (ibid.). The 4.2 pBTL21 fragment and the ZC1846/ZC1847 adapter were joined by lrgation. The resultant plasmid, designated pBTL22, comprises the SUC2 signal sequence fused in proper reading frame to the extracellular domain of PDGFβ-R in the vector pUC19 (FIG. 6).</p>
    <p>B. Construction of pBTL28</p>
    <p>An in-frame translation stop codon was inserted immediately after the coding region of the PDGFβ-R in pBTL22 using oligonucleotides ZC1892 (Sequence ID Number 19; Table 1) and ZC1893 (Sequence ID Number 20; Table 1). These oligonucleotides were designed to form an adapter encoding a stop codon in-frame with the PDGFβ-R coding sequence from pBTL22 flanked by a 5' Hind III adhesive end and a 3' Xba I adhesive end. Plasmid pBTL22 was digested with Eco RI and Hind III to isolate the 1.6 kb SUC2-PDGFβ-R R fragment. Plasmid pMVR1 was digested with Eco RI and Xba I to isolate the 3.68 kb fragment comprising the TPI1 promoter, pIC7RI* vector sequences and the TPI1 terminator. Oligonucleotides ZC1892 and ZClB93 were annealed to form a Hind III-Xba I adapter. The 1.6 kb SUC2-PDGFβ-R fragment, the 3.86 kb pMVR1 fragment and the ZC1892/ZC1893 adapter were joined in a three-part ligation. The resultant plasmid was designated pBTL27.</p>
    <p>The expression unit present in pBTL27 was inserted into the yeast expression vector pJH50 by first digesting pJH50 with Bam HI and Sal I to isolate the 10.3 kb vector fragment. Plasmid pBTL27 was digested with Bgl II and Eco RI and with Xho I and Eco RI to isolate the 0.9 kb TPI1 promoter fragment and the 1.65 kb fragment, respectively. The 10.3 kb pJH50 vector fragment, the 0.9 kb TPI1 promoter fragment and 1.65 kb fragment were joined in a three-part ligation. The resultant plasmid was designated pBTL28.</p>
    <p>C. Construction of Plasmid pBTL30</p>
    <p>The PDGFβ-R coding sequence present in plasmid pBTL22 was modified to encode the twelve C-terminal amino acids of substance P and an in-frame stop codon. Plasmid pBTL22 was digested with Eco RI and Hind III to isolate the 1.6 kb SUC2-PDGFβ-R fragment. Plasmid pMVR1 was digested with Eco RI and Xba I to isolate the 3.68 kb fragment comprising the TPI1 promoter, pIC7RI* and the TPI1 terminator. Synthetic oligonucleotides ZC1894 (Sequence ID Number 21; Table 1) and ZC1895 (Sequence ID Number 22; Table 1) were annealed to form an adapter containing the codons for the twelve C-terminal amino acids of substance P followed by an in-frame stop codon and flanked on the 5' end with a Hind III adhesive end and on the 3' end with an Xba I adhesive end. The ZC1894/ZC1895 adapter, the 1.6 kb SUC2-PDGFβ-R fragment and the pMVR1 fragment were joined in a three-part ligation. The resultant plasmid, designated pBTL29, was digested with Eco RI and Xho I to isolate the 1.69 kb SUC2-PDGFβ-R-subP-TPI1 terminator fragment. Plasmid pBTL27 was digested with Bgl II and Eco RI to isolate the 0.9 kb TPI1 promoter fragment. Plasmid pJH50 was digested with Bam HI and Sal I to isolate the 10.3 kb vector fragment. The 1.69 kb pBTL29 fragment, the 0.9 kb TPI1 promoter fragment and the 10.3 kb vector fragment were joined in a three-part ligation. The resulting plasmid was designated pBTL30.</p>
    <heading>Example 9</heading> <heading>Construction and Expression of a SUC2-PDGFβ-R-IgG Hinge</heading> <heading>Expression Vector</heading> <p>An expression unit comprising the TPI1 promoter, the SUC2 signal sequence, the PDGFβ-R extracellular domain, an immunoglobulin hinge region and the TPI1 terminator was constructed. Plasmid pBTL22 was digested with Eco RI and Hind III to isolate the 1.56 kb fragment. Plasmid pMVR1 was digested with Eco RI and Xba I to isolate the 3.7 kb fragment, comprising the TPI1 promoter, pIC7RI* vector sequences and the TPI1 terminator. Oligonucleotides ZC1776 (Sequence ID Number 14; Table 1) and ZC1777 (Sequence ID Number 15; Table 1) were designed to form, when annealed, an adapter encoding an immunoglobulin hinge region with a 5' Hind III adhesive end and a 3'Xba I adhesive end. Oligonucleotides ZC1776 and ZC1777 were annealed under conditions described by Maniatis et al. (ibid.). The 1.56 kb pBTL22 fragment, the 3.7 kb fragment and the ZC1776/ZC1777 adapter were joined in a three-part ligation, resulting in plasmid pBTL24.</p>
    <p>The expression unit of pBTL24, comprising the TPI1 promoter, SUC2 signal sequence, PDGFβ-R extracellular domain sequence, hinge region sequence, and TPI1 terminator, was inserted into pJH50. Plasmid pBTL24 was digested with Xho I and Hind III to isolate the 2.4 kb expression unit. Plasmid pJH50 was digested with Hind III and Sal I to isolate the 9.95 kb fragment. The 2.4 kb pBTL24 fragment and 9.95 kb pJH50 vector fragment were joined by ligation. The resultant plasmid was designated pBTL25.</p>
    <p>Plasmid pBTL25 was transformed into Saccharomyces cerevisiae strain ZY400 using the method essentially described by Beggs (ibid.). Transformants were selected for their ability to grow on -LEUDS (Table 2). The transformants were tested for their ability to bind the anti-PDGFβ-R monoclonal antibody PR7212 using the colony assay method described in Example 18. Plasmid pBTL25 transformants were patched onto nitrocellulose filters that had been wetted and supported by YEPD solid medium. Antibody PR7212 was found to bind to the PDGFβ-R-IgG hinge fusion secreted by ZY400[pBTL25] transformants.</p>
    <heading>Example 10</heading> <heading>Construction and Expression of a SUC2 Signal Sequence-PDGFβ-R Extracellular Domain-SUC2 Fusion</heading> <p>As shown in FIG. 6, an expression unit comprising the TPI1 promoter, SUC2 signal sequence, PDGFβ-R extracellular domain sequence, and SUC2 coding sequence was constructed as follows. Plasmid pBTL22 was digested with Fco RI and Hind III to isolate the 1.6 kb SUC2-PDGFβ-R fragment. Plasmid pMVR1 was digested with Bgl II and Eco RI to isolate the 0.9 kb TPI1 promoter fragment. The SUC2 coding region was obtained from pJH40. Plasmid pJH40 was constructed by inserting the 2.0 kb Hind III-Hind III SUC2 fragment from pRB58 (Carlson et al., Cell 28:145-154, 1982) into the Hind III site of pUC19 followed by the destruction of the Hind III site 3' to the coding region. Plasmid pJH40 was digested with Hind III and Sal I to isolate the 2.0 kb SUC2 coding sequence. Plasmid pJH50 was digested with Sal I and Bam HI to isolate the 10.3 kb vector fragment. The 0.9 kb Bgl II-Eco RI TPI1 promoter fragment, the 1.6 kb Eco RI-Hind III SUC2-PDGFβ-R, the 2.0 kb Hind III-Sal I SUC2 fragment and the 10.3 kb Bam HI-Sal I vector fragment were joined in a four-part ligation. The resultant plasmid was designated pBTL26 (FIG. 6).</p>
    <p>Plasmid pBTL26 was transformed into Saccharomyces cerevisae strain ZY400 using the method essentially described by Beggs (ibid.). Transformants were selected for their ability to grow on -LEUDS (Table 2). ZY400 transformants (ZY400[pBTL26]) were assayed by protein blot (Example 18), colony blot (Example 18) and competition assay.</p>
    <p>Protein blot assays were carried out on ZY400[pBTL26] and ZY400[pJH5O] (control) transformants that had been grown in flasks. Two hundred-fifty microliters of a 5 ml overnight cultures of ZY400[pBTL26] and ZY400 [pJH50] in -LEUDS+sodium succinate, pH 6.5 (Table 2) were inoculated into 50 ml of -LEUDS+sodium succinate, pH 6.5. The cultures were incubated for 35 hours in an airbath shaker at 30° C. The culture supernatants were harvested by centrifugation. The culture supernatants were assayed as described in Example 18 and were found to bind PR7212 antibody.</p>
    <p>Colony assays were carried out on ZY400[pBTL26] transformants. ZY400[pBTL26] transformants were patched onto wetted nitrocellulose filters that were supported on a YEPD plate. The colony assay carried out as described in Example 8.A. showed that ZY400[pBTL26] antibodies bound PR7212 antibodies.</p>
    <p>Competition binding assays were carried out on ZY400[pBTL26] and ZY400[pJH50] transformants. The transformants were grown in two liters of fermentation medium (Table 2) in a New Brunswick Bioflo2 fermentor (New Brunswick, Philadelphia, Pa.) with continuous pH control at pH 6.4. The cultures were adjusted to pH 7.5 immediately prior to harvesting. Culture supernatants were concentrated in an Amicon concentrator (Amicon, San Francisco, Calif.) using an Amicon 10<sup>4</sup> mw spiral filter cartridge. The concentrated supernatants were further concentrated using Amicon Centriprep 10's. Fifteen milliliters of the concentrated supernatant samples were added to the Centripreps, and the Centripreps were spun in a Beckman GRP centrifuge (Beckman Instruments Inc., Carlsbad, Calif.) at setting 5 for a total of 60 minutes. The concentrates were removed from the Centripreps and were assayed in the competition assay.</p>
    <p>The competition binding assay measured the amount of <sup>125</sup> I-PDGF left to bind to fetal foreskin fibroblast cells after preincubation with the concentrate containing the PDGFβ-R-SUC2 fusion protein. PDGF-AA and PDGF-AB were iodinated using the Iodopead method (Pierce Chemical). PDGF-BB<sub>Tyr</sub> was iodinated and purified as described in Example 18.F. The concentrate was serially diluted in binding medium (Table 4). The dilutions were mixed with 0.5 ng of iodinated PDGF-AA, PDGF-BB<sub>Tyr</sub> or PDGF-AB, and the mixtures were incubated for two hours at room temperature. Three hundred micrograms of unlabeled PDGF-BB was added to each sample mixture. The sample mixtures were added to 24-well plates containing confluent fetal foreskin fibroblast cells (AG1523, available from the Human Genetic Mutant Cell Repository, Camden, N.J.). The cells were incubated with the mixture for four hours at 4° C. The supernatants were aspirated from the wells, and the wells were rinsed three times with phosphate buffered saline that was held at 4° C. (PBS; Sigma, St. Louis, Mo.). Five hundred microliters of PBS+1% NP-40 was added to each well, and the plates were shaken on a platform shaker for five minutes. The cells were harvested and the amount of iodinated PDGF was determined. The results of the competition binding assay showed that the PDGFβ-R-SUC2 fusion protein was able to competitively bind all three isoforms of PDGF.</p>
    <p>The PDGFβ-R produced from ZY400 [pBTL26] transformants was tested for cross reactivity to fibroblast growth factor (FGF) and transforming growth factorβ (TGF-β) using the competition assay essentially described above. Supernatant concentrates from ZY400[pBTL26] and ZY400[JH50] (control) transformants were serially diluted in binding medium (Table 4). The dilutions were mixed with 7.9 ng of iodinated FGF or 14 ng of iodinated TGF-β, and the mixtures were incubated for two hours at room temperature. Fourteen micrograms of unlabeled FGF was added to each mixture containing labeled FGF, and 7 μg of unlabeled TGF-β was added to each mixture containing labeled TGF-β. The sample mixtures were added to 24-well plates containing confuent human dermal fibroblast cells. (Human dermal fibroblast cells express both FGF receptors and TGFβ receptors.) The cells were incubated with the mixtures for four hours at 4° C. Five hundred microliters of PBS+1% NP-40 was added to each well, and the plates were shaken on a platform shaker for five minutes. The cells were harvested and the amount of iodinated FGF or TGF-β bound to the cells was determined. The results of these assays showed that the PDGFβ-R-SUC2 fusion protein did not cross react with FGF or TGF-β.</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">              TABLE 4______________________________________Reagent Recipes______________________________________Binding Medium500 ml    Ham's F-12 medium    12 ml 1M HEPES, pH 7.4    5 ml 100× PSN (Penicillin/Streptomycin/Neomycin,   Gibco)    1 g rabbit serum albuminWestern Transfer Buffer 25 mM    Tris, pH 8.3    19 mM glycine, pH 8.3    20% methanolWestern Buffer A 50 ml    1 M Tris, pH 7.4    20 ml 0.25 mM EDTA, pH 7.0    5 ml 10% NP-40  37.5 ml 4 M NaCl   2.5 g gelatinThe Tris, EDTA, NP-40 and NaCl were diluted to a  final volume of one liter with distilled water. The  gelatin was added to 300 ml of this solution and the  solution was heated in a microwave until the gelatin was  in solution. The gelatin solution was added back to the  remainder of the first solution and stirred at 4° C. until cool. The buffer was stored at 4° C.  Western Buffer B     50 ml    1 M Tris, pH 7.4    20 ml 0.25 M EDTA, pH 7.0    5 ml 10% NP-40  58.4 g NaCl   2.5 g gelatin    4 g N-lauroyl sarcosine______________________________________</pre>
    
    <p>The Tris, EDTA, NP-40, and the NaCl were mixed and diluted to a final volume of one liter. The gelatin was added to 300 ml of this solution and heated in a microwave until the gelatin was in solution. The gelatin solution was added back to the original solution and the N-lauroyl sarcosine was added. The final mixture was stirred at 4° C. until the solids were completely dissolved. This buffer was stored at 4° C.</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">______________________________________2× Loading Buffer______________________________________36 ml    0.5 M Tris-HCl, pH 6.8  16 ml glycerol  16 ml 20% SDS   4 ml 0.5% Bromphenol Blue in 0.5 M Tris-HCl, pH 6.8______________________________________</pre>
    
    <p>Mix all ingredients. Immediately before use, add 100 μl β-mercaptoethanol to each 900 μl dye mix</p>
    <heading>Example 11</heading> <heading>Construction and Expression of PDGF Receptor Analogs From BHK cells</heading> <p>A. Construction of pBTL114 and pBTL115</p>
    <p>The portions of the PDGF β-receptor extracellular domain present in pBTL14 and pBTL15 were placed in a mammalian expression vector. Plasmids pBTL14 and pBTL15 were digested with Eco RI to isolate the 1695 bp and 1905 bp SUC2 signal-PDGFβ-R-BAR1 fragments. The 1695 bp fragment and the 1905 bp fragment were each ligated to Zem229R that had been linearized by digestion with Eco RI.</p>
    <p>The vector Zem229R was constructed as shown in FIG. 10 from Zem229. Plasmid Zem229 is a pUC18-based expression vector containing a unique Bam HI site for insertion of cloned DNA between the mouse metallothionein-1 promoter and SV40 transcription terminator and an expression unit containing the SV40 early promoter, mouse dihydrofolate reductase gene, and SV40 transcription terminator. Zem229 was modified to delete the Eco RI sites flanking the Bam HI cloning site and to replace the Bamr HI site with a single Eco RI cloning site. The plasmid was partially digested with Eco RI, treated with DNA polymerase I (Klenow fragment) and dNTPs, and religated. Digestion of the plasmid with Bam HI followed by ligaion of the linearized plasmid with a Bam HI-Eco I adapter resulted in a unique Eco RI cloning site. The resultant plasmid was designated Zem229R.</p>
    <p>The ligation mixtures were transformed into E. coli strain RR1. Plasmid DNA was prepared and the plasmids were subjected to restriction enzyme analysis. A plasmid having the 1695 bp pBTL14 fragment inserted into Zem229R in the correct orientation was designated pBTL114 (FIG. 9). A plasmid having the 1905 bp pBTL15 fragment inserted into Zem229R in the correct orientation was designated pBTL115 (FIG. 9).</p>
    <p>B. Expression of Secreted PDGF β-Receptor Analogs in tk<sup>--</sup> ts13 BHK Cells</p>
    <p>Plasmids pBTL114 and pBTL115 were each transfected into tk<sup>--</sup> ts13 cells using calcium phosphate precipitation (essentially as described by Graham and van der Eb, J. Gen. Virol. 36: 59-72, 1977). The transfected cells were grown in Dulbecco's modified Eagle's medium (DMEM) containing 10% fetal calf serum, 1×PSN antibiotic mix (Gibco 600-5640), 2.0 mM L-glutamine. The cells were selected in 250 nM methotrexate (MTX) for 14 days, and the resulting colonies were screened by the immunofilter assay (McCracken and Brown, Biotechniques, 82-87, March/April 1984). Plates were rinsed with PBS or No Serum medium (DMEM plus 1×PSN antibiotic mix). Teflon® mesh (Spectrum Medical Industries, Los Angeles, Calif.) was then placed over the cells. Nitrocellulose filters were wetted with PBS or No Serum medium, as appropriate, and placed over the mesh. After six hours incubation at 37° C., filters were removed and placed in Wester buffer A (Table 4) overnight at room temperature. The filters were developed using the antibody PR7212 and the procedure described in Example 8. The filters showed that conditioned media from pBTL114-transfected and pBTL115-transfected BHK cells bound the PR7212 antibody indicating the presence of biologically active secreted PDGFβ-R.</p>
    <heading>Example 12</heading> <heading>Expression of PDGF β-Receptor Analogs in Cultured Mouse Myeloma Cells</heading> <p>A. Construction of pICpPRE8</p>
    <p>The immunoglobulin μ heavy chain promoter and enhancer were sublconed into pIC19H to provide a unique Hind III site 3' to the promoter. Plasmid pμ (Grosschedl and Baltimore, Cell 41: 885-897, 1985) was digested with Sal I and Eco RI to isolate the 3.1 kb fragment comprising the μ promoter. Plasmid pIC19H was linearized by digestion with Eco RI and Xho I. The p promoter fragment and the linearized pIC19H vector fragment were joined by ligation. The resultant plasmid, designated pICμ3, was digested with Ava II to isolate the 700 bp μ promoter fragment. The 700 bp fragment was blunt-ended by treatment with DNA polymerase I (Klenow fragment) and deoxynucleotide triphosphates. Plasmid pIC19H was linearized by digestion with Xho I, and the adhesive ends were filled in by treatment with DNA polymerase I (Klenow fragement) and deoxynucleotide triphosphates. The blunt-ended Ava II fragment was ligated with the blunt-ended, linearized pIC19H, and the ligation mixture was transformed into E. coli JM83. Plasmid DNA was prepared from the transformants and was analyzed by restriction digest. A plasmid with a Bgl II site 5' to the promoter was designated pICμIPR1(-). Plasmid pICμPR1(-) was digested with Hind III and Bgl II to isolate the 700 bp μ promoter fragment. Plasmid pIC19R was linearized by digestion with Hind III and Bam HI. The 700 bp promoter fragment was joined with the linearized pIC19R by ligation. The resultant plasmid, designated pICμPR7, comprised the μ promoter with an unique Sma I site 5' to the promoter and a unique Hind III site 3' to the promoter.</p>
    <p>The immunoglobulin heavy chain μ enhancer (Gillies et al., Cell 33: 717-728, 1983) was inserted into the unique Sma I site to generate plasmid pICμPRE8. Plasmid pJ4 (obtained from F. Blattner, Univ. Wisconsin, Madison, Wis.), comprising the 1.5 kb Hind III-Eco RI A enhancer fragment in the vector pAT153 (Amersham, Arlington Heights, Ill.), was digested with Hind III and Eco RI to isolate the 1.5 kb enhancer fragment. The adhesive ends of the enhancer fragment were filled in by treatment with T4 DNA polymerase and deoxynucleotide triphosphates. The blunt-ended fragment and pICμPR7, which had been linearized by digestion with Sma I, were joined by ligation. The ligation mixture was transformed into E. coli RR1. Plasmid DNA was prepared from the transformants, and the plasmids were analyzed by restriction digests. A plasmid comprising the μ enhancer and the A promoter was designated pICμPRE8 (FIG. 7).</p>
    <p>B. Construction of pSDL114</p>
    <p>The DNA sequence encoding the extracellular domain of the PDGF β-receptor was joined with the DNA sequence encoding the human immunoglobulin light chain constant region. The PDGF β-receptor extracellular domain was obtained from mpBTL22, which comprised the Eco RI-Hind III fragment from pBTL22 (Example 8.A.) cloned into Eco RI-Hind III cut M13mp18. Single stranded DNA was prepared from a mpBTL22 phage clone, and the DNA was subjected to in vitro mutagenesis using the oligonucleotide ZC1886 (Table 1) and the method described by Kunkel (Proc. Natl. Acad. Sci. USA 82: 488-492, 1985). A phage clone comprising the mutagenized PDGFβ-R with a donor splice site (5° splice site) at the 3' end of the PDGFβ-R extracellular domain was designated pBTLR-HX (FIG. 7).</p>
    <p>The native PDGFβ-R signal sequence was obtained from pPR5. Plasmid PPR5, comprising 738 bp of 5' coding sequence with an Eco RI site immediately 5' to the translation initiation codon, was constructed by in vitro mutagenesis of the PDGFβ-R CDNA fragment from RP51 (Example 1). Replicative form DNA of RP51 was digested with Eco RI to is6late the 1.09 kb PDGFβ-R fragment. The PDGFβ-R fragment was cloned into the Eco RI site of M13mp18. Single stranded template DNA was prepared from a phage clone containing the PDGFβ-R fragment in the proper orientation. The template DNA was subjected to in vitro mutagenesis using oligonucleotide ZC1380 (Sequence ID Number 8; Table 1) and the method described by Zoller and Smith (Meth. Enzymol. 100: 468-500, 1983). The mutagenesis resulted in the placement of an Eco RI site immediately 5' to the translation initiation codon. Mutagenized phage clones were analyzed by dideoxy sequence analysis. A phage clone containing the ZC1380 mutation was selected, and replicative form (Rf) DNA was prepared from the phage clone. The Rf DNA was digested with Eco RI and Acc I to isolate the 0.63 kb fragment. Plasmid pR-RXI (Example 1) was digested with Acc I and Eco RI to isolate the 3.7 kb fragment. The 0.63 kb fragment and the 3.7 kb fragment were joined by ligation resulting in plasmid pPR5 (FIG. 7).</p>
    <p>As shown in FIG. 7, the PDGFβ-R signal peptide and part of the extracellular domain were obtained from plasmid pPR5 as a 1.4 kb Eco RI-Sph I fragment. Replicative form DNA from phage clone pBTLR-HX was digested with Sph I and Hind III to isolate the approximately 0.25 kb PDGFβ-R fragment. Plasmid pUC19 was linearized by digestion with Eco RI and Hind III. The 1.4 kb Eco RI-Sph I PDGFβ-R fragment, the 0.25 kb Sph I-Hind III fragment from pBTLR-HX and the Eco RI-Hind III cut pUC19 were joined in a three-part ligation. The resultant plasmid, pSDL110, was digested with Eco RI and Hind III to isolate the 1.65 kb PDGFβ-R fragment.</p>
    <p>Plasmid pICHuCr3.9.11 was used as the source of the human immunoglobulin light chain gene (FIG. 7). The human immunoglobulin light chain gene was isolated from a human genomic library using an oligonucleotide probe (5' TGT GAC ACT CTC CTG GGA GTT A 3'; Sequence ID Number 32), which was based on a published human kappa C gene sequence (Hieter et al., Cell 22: 197-207, 1980). The human light chain (kappa) constant region was subdloned as a 1.1 kb Sph I-Hinf I genomic fragment of the human kappa gene, which has been treated with DNA polymerase DNA I (Klenow Fragment) to fill in the Hinf I adhesive end, into Sph I-Hinc II cut pUC19. The 1.1 kb human kappa constant region was susbsequently isolated as a 1.1 kb Sph I-Bam HI fragment that was subdloned into Sph I-Bgl II cut pIC19R (Marsh et al., ibid.). The resultant plasmid was designated pICHuCX3.9.11. Plasmid pICHuC<sub>78</sub> 3.9.11 was digested with Hind III and Eco RI to isolate the 1.1 kb kappa constant region gene. Plasmid pIC19H was linearized by digestion with Eco RI. The 1.65 kb PDGFβ-R fragment, the 1.1 kb human kappa constant region fragment and the linearized pIC19H were joined in a three part ligation. The resultant plasmid, pSDL112, was digested with Bam HI and Cla I to isolate the 2.75 kb fragment. Plasmid ppPRE8 was linearized with Bgl II and Cla I. The 2.75 kb fragment and the linearized ppPRE8 were joined by ligation. The resultant plasmid was designated pSDL114 (FIG. 7).</p>
    <p>Plasmid pSDL114 was linearized by digestion with Cla I and was cotransfected with Pvu I-digested p416 into SP2/0-Ag14 (ATCC CRL 1581) by electroporation using the method essentially described by Neumann et al. (EMBO J. 1: 841-845, 1982). (Plasmid p416 comprises the Adenovirus 5 ori, SV40 enhancer, Adenovirus 2 major late promoter, Adenovirus 2 tripartite leader, 5' and 3' splice sites, the DHFR<sup>r</sup> CDNA, the SV40 polyadenylation signal and 1 ML-1 (Lusky and Botchan, Nature 2: 79-81, 1981) vector sequences.) Transfectants were selected in growth medium containing methotrexate.</p>
    <p>Media from drug resistant clones were tested for the presence of secreted PDGF β-receptor analogs by enzyme-linked immunosorbant assay (ELISA). Ninety-six well assay plates were prepared by incubating 100 μl of 1 μg/ml polyclonal goat anti-human kappa chain (Cappel Laboratories, Melvern, Pa.) diluted in phosphate buffered saline (PBS; Sigma) overnight at 4° C. Excess antibody was removed by three washes with 0.5% Tween 20 in PBS. One hundred microliters of spent media was added to each well, and the well were incubated for one hour at 4° C. Unbound proteins were removed by eight washes with 0.5% Tween 20 -in PBS. One hundred microliters of peroxidase-conjugated goat anti-human kappa antibody (diluted 1:1000 in a solution containing 5% chicken serum (GIBCO-BRL)+0.5% Tween 20 in PBS) was added to each well and the wells were incubated for one hour at 4° C. One hundred microliters of chromophore (100 u1 ABTS (2,2'-Azinobis(3-ethylbenz-thiazoline sulfonic acid) diammonium salt; Sigma)+1 μl 30% H<sub>2</sub> O<sub>2</sub> +12.5 ml citrate/phosphate buffer (9.04 g/l citric acid, 10.16 g/l Na<sub>2</sub> HPO<sub>4</sub>)) was added to each well, and the wells were incubated to thirty minutes at room temperature. The samples were measured at 405 nm. The results of the assay showed that the PDGFβ-R analog secreted by the transfectants contained an immunoglobulin light chain.</p>
    <p>Spent media from drug resistant clones was also tested for the presence of secreted PDGF β-receptor analogs by immunoprecipitation. Approximately one million drug resistant transfectants were metabolically labeled by growth in DMEM medium lacking cysteine+2% calf serum for 18 hours at 37° C. in the presence of 50 μCI <sup>35</sup> 5-cysteine. Media was harvested from the labeled cells and 250 μl of the spent media was assayed by immunoprecipitation with the anti-PDGF β-receptor antibody PR7212 to detect the prsence of metabolically labeled PDGF β-receptor analogs. PR7212, diluted in PBS, was added to the media to a final concentration of 2.5 μg per 250 μl spent media. Five microliters of rabbit anti-mouse Ig diluted in PBS was added to the PR7212/media mixtures. The immunocomplexes were precipitated by the addition of 50 μl 10% fixed Staph A (weight/volume in PBS). The immunocomplexes were analyzed on 8% SDS-polyacrylamide gels followed by autoradiography overnight at -70° C. The results of the immunoprecipitation showed that the PDGF β-receptor analog secreted by the transfectants was bound by the anti-PDGF β-receptor antibody. The combined results of the ELISA and immunoprecipitation assays showed that the PDGF β-receptor analog secreted by the transfectants contained both the PDGF β-receptor ligand-binding domain and the human light chain constant region.</p>
    <p>C. Cotransfection of pSDL114 with an Immunoglobulin Heavy Chain</p>
    <p>Plasmid pSDL114 was cotransfected with pφ5V<sub>H</sub> huC.sub.γ 1M-neo, which encodes a neomycin resistance gene expression unit and a complete mouse/human chimeric immunoglobulin heavy chain gene expression unit.</p>
    <p>Plasmid pφ5V<sub>H</sub> huC.sub.γ 1M-neo was constructed as follows. The mouse immunoglobulin heavy chain gene was isolated from a lambda genomic DNA library constructed from the murine hybridoma cell line NR-ML-05 (Serafini et al., Eur. J. Nucl. Med. 14: 232, 1988) using an oligonucleotide probe designed to span the V<sub>H</sub> /D/J<sub>H</sub> junction (5'GCA TAG TAG TTA CCA TAT CCT CTT GCA CAG 3'; Sequence ID Number 33). The human immunoglobulin gamma-1 C gene was isolated from a human genomic library using a cloned human gamma-4 constant region gene (Ellison et al., DNA 1: 11-18, 1981). The mouse immunoglobulin variable region was isolated as a 5.3 kb Sst I-Hind III fragment from the original phage clone and the human gamma-1 C gene was obtained from the original phage clone as a 6.0 kb Hind III-Xho I fragment. The chimeric gamma-1 C gene was created by joining the V<sub>H</sub> and C<sub>H</sub> fragments via the common Hind III site and incorporating them with the E. coli neomycin resistance gene expression unit into pIC19H to yield pφ5V<sub>H</sub> huC.sub.γ 1M-neo.</p>
    <p>Plasmid pSDL114 was linearized by digestion with Cla I and was co-transfected into SP2/0-Ag14 cells with Asp 718 linearized pφ5V<sub>H</sub> huC.sub.γ 1M-neo. The transfectants were selected in growth medium containing methotrexate and neomycin. Media from drug-resistant clones were tested for their ability to bind PDGF in a competition binding assay.</p>
    <p>The competition binding assay measured the amount of <sup>125</sup> -PDGF left to bind to human dermal fibroblast cells after preincubation with the spent media from pSDL114-pφ5V<sub>H</sub> huC.sub.γ 1M-neo transfected cells. The media were serially diluted in binding medium (Table 4). The dilutions were mixed with 0.5 ng of iodinated PDGF-BB or iodinated PDGF-AA, and the mixtures were incubated for two hours at room temperature. Three hundred micrograms of unlabeled PDGF-BB or unlabeled PDGF-AA was added to one tube from each series. The sample mixtures were added to 24 well plates containing confluent human dermal fibroblast cells. The cells were incubated with the mixture for four hours at 4° C. The supernatants were aspirated from the wells, and the wells were rinsed three times with phosphate buffered saline that was held a 4° C. (PBS; Sigma, St. Louis, Mo.). Five hundred microliters of PBS+1% NP-40 was added to each well, and the plates were shaken on a platform shaker for five minutes. The cells were harvested and the amount of iodinated PDGF was determined. The results of the competition binding assay showed that the protein produced from pSDL114-pφ5V<sub>H</sub> huC.sub.γ 1M- neo transfected cells was able to competetively bind PDGF-BB but did not bind PDGF-AA.</p>
    <p>The PDGF β-receptor analog produced from a pSDL114-pφ5V<sub>H</sub> huC.sub.γ 1M-neo transfectant was assayed to determine if the receptor analog was able to bind PDGF-BB with high affinity. Eight and one half milliliters of spent media containing the PDGFβ-R analogs from a pSDL114-pφ5V<sub>H</sub> huC.sub.γ 1M-neo transfectant was added to 425 μl of Sepharose Cl-4B-Protein A beads (Sigma, St. Louis, Mo.), and the mixture was incubated for 10 minutes at 4° C. The beads were pelleted by centrifugation and washed with binding medium (Table 4). Following the wash the beads were resuspended in 8.5 ml of binding media, and 0.25 ml aliquots were dispensed to 1.5 ml tubes. Binding reactions were prepared by adding iodinated PDGF-BB<sub>Tyr</sub> (Example 18.F.) diluted in DMEM+10% fetal calf serum to the identical aliquots of receptor-bound beads to final PDGF-BB<sub>Tyr</sub> concentrations of between 4.12 pM and 264 pM. Nonspecific binding was determined by adding a 100 fold excess of unlabeled BB to an identical set of binding reactions. Mixtures were incubated overnight at 4° C.</p>
    <p>The beads were pelleted by centrifugation, and unbound PDGF-BB was removed with three washes in PBS. The beads were resuspended in 100 μl of PBS and were counted. Results of the assay showed that the PDGFβ-R analog was able to bind PDGF-BB with high affinity.</p>
    <p>D. Construction of pSDL113.</p>
    <p>As shown in FIG. 8, the DNA sequence encoding the extracellular domain of the PDGF β-receptor was joined with the DNA sequence encoding a human immunoglobulin heavy chain constant region joined to a hinge sequence. Plasmid pSDL110 was digested with Eco RI and Hind III to isolate the 1.65 kb PDGFβ-R fragment. Plasmid pICHu.sub.γ -1M was used as the source of the heavy chain constant region and hinge region. Plasmid pICHu.sub.γ -1M comprises the approximately 6 kb Hind III-Xho I fragment of a human gamma-1 C gene subcloned into pUC19 that had been linearized by digestion with Hind III and SaI I. Plasmid pICHu.sub.γ -1M was digested with Hind III and Eco RI to isolate the 6 kb fragment encoding the human heavy chain constant region. Plasmid pIC19H was linearized by digestion with Eco RI. The 1.65 kb PDGFβ-R fragment, the 6 kb heavy chain constant region fragment and the linearized pIC19H were joined in a three part ligation. The resultant plasmid, pSDL111, was digested with Bam HI to isolate the 7.7 kb fragment. Plasmid pμPRE8 was linearized with Bgl II and was treated with calf intestinal phosphatase to prevent self-ligation. The 7.7 kb fragment and the linearized pμPRE8 were joined by ligation. A plasmid containing the insert in the proper orientation was designated pSDL113 (FIG. 8).</p>
    <p>Plasmid pSDL113 is linearized by digestion with Cla I and is cotransfected with Pvu I-digested p416 into SP2/0-Ag14 by electroporation. Transfectants are selected in growth medium containing methotrexate.</p>
    <p>Media from drug resistant clones are tested for the presence of secreted PDGFβ-R analogs by immunoprecipitation using the method described in Example 12.B.</p>
    <p>E. Cotransfection of pSDL113 with an Immunoglobulin Light Chain Gene</p>
    <p>Plasmid pSDL113 is linearized by digestion with Cla I and was cotransfected with pICφ5V.sub.κ HuC.sub.κ -Neo, which encodes a neomycin resistance gene and a mouse/human chimeric immunoglobulin light chain gene. The mouse immunoglobulin light chain gene was isolated from a lambda genomic DNA library constructed from the murine hybridoma cell line NR-ML-05 (Woodhouse et al., ibid.) using an oligonucleotide probe designed to span the V.sub.κ /J.sub.κ  junction (5' ACC GAA CGT GAG AGG AGT GCT ATA A 3'; Sequence ID Number 34). The human immunoglobulin light chain constant region gene was isolated as described in Example 12.B. The mouse NR-ML-05 immunoglobulin light chain variable region gene was subcloned from the original mouse genomic phage clone into pIC19R as a 3 kb Xba I-Hinc II fragment. The human kappa C gene was subcloned from the original human genomic phage clone into pUC19 as a 2.0 kb Hind III-Eco RI fragment. The chimeric kappa gene was created by joining the NR-ML-05 light chain variable region gene and human light chain constant region gene via the common Sph I site and incoporating them with the E. coli neomycin resistance gene into pIC19H to yield pICφ5V.sub.κ HuC.sub.κ -Neo (FIG. 9).</p>
    <p>The linearized pSDL113 and pICφ5V.sub.κ HuC.sub.κ -Neo are transfected into SP2/0-Ag14 cells by electroporation. The transfectants are selected in growth medium containing methotrexate and neomycin.</p>
    <p>F. Cotransfection of pSDL113 and pSDL114</p>
    <p>A clone of SP2/0-Ag14 stably transfected with pSDL114 and p416 was co-transfected with Cla I-digested pSDL113 and Bam HI-digested pICneo by electroporation. (Plasmid pICneo comprises the SV40 promoter operatively linked to the E. coli neomycin resistance gene and pIC19H vector sequences.) Transfected cells were selected in growth medium containing methotrexate and G418. Media from drug-resistanct clones were tested for their ability to bind PDGF-BB or PDGF-AA in a competition binding assay as described in Example 12.C. The results of the assay showed that the transfectants secreted a PDGF β-receptor analog which was capable of competitively binding PDGF-BB but did not detectably bind to PDGF-AA.</p>
    <p>G. Cotransfection of pSDL114 with Fab</p>
    <p>A clone of SP2/0-AG14 stably transfected with pSDL114 and p416 was transfected with the Fab region of the human gamma-4 gene (γ<sub>4</sub>) in plasmid pφ5V<sub>H</sub> Fab-neo.</p>
    <p>Plasmid pφ5V<sub>H</sub> Fab-neo was constructed by first digesting plasmid p24BRH (Ellison et al., DNA 1: 11, 1988) was digested with Xma I and Eco RI to isolate the 0.2 kb fragment comprising the immunoglobulin 3' untranslated region. Synthetic oligonucleotides ZC871 (Sequence ID Number 3; Table 1) and ZC872 (Sequence ID Number 4; Table 1) were kinased and annealed using essentially the methods described by Maniatis et al. (ibid.). The annealed oligonucleotides ZC871/ZC872 formed an Sst I-Xma I adapter. The ZC871/ZC872 adapter, the 0.2 kb p24BRH fragment and Sst I-Eco RI linearized pUC19 were joined in a three-part ligation to form plasmid pγ<sub>4</sub> 3'. Plasmid pγ<sub>4</sub> 3' was linearized by digestion with Bam HI and Hind III. Plasmid p24BRH was cut with Hind III and Bgl II to isolate the 0.85 kb fragment comprising the C<sub>H</sub> 1 region. The pγ<sub>4</sub> 3' fragment and the Hind III-Bgl II p24BRH fragment were joined by ligation to form plasmid pγ<sub>4</sub> Fab. Plasmid pγ<sub>4</sub> Fab was digested with Hind III and Eco RI to isolate the 1.2 kb fragment comprising γ<sub>4</sub> Fab. Plasmid pICneo, comprising the SV40 promoter operatively linked to the E. coli neomycin resistance gene and pIC19H vector sequences, was linearized by digestion with Sst I and Eco RI. Plasmid pφ5V<sub>H</sub>, comprising the mouse immunoglobulin heavy chain gene variable region and pUC18 vector sequences, was digested with Sst I and Hind III to isolate the 5.3 kb V<sub>H</sub> fragment. The linearized pICneo was joined with the 5.3 kb Sst I-Hind III fragment and the 1.2 kb Hind III-Eco RI fragment in a three-part ligation. The resultant plasmid was designated pφ5V<sub>H</sub> Fab-neo (FIG. 10).</p>
    <p>A pSDL114/p416-transfected SP2/0-AG14 clone was transfected with Sca I-linearized pφ5V<sub>H</sub> Fab-neo. Transfected cells were selected in growth medium containing methotrexate and G418. Media from drug-resistant clones were tested for their ability to bind PDGF in a competition binding assay as described in Example 12.C. The results of the assay showed that the PDGF β-receptor analog secreted from the transfectants was capable of competitively binding PDGF-BB.</p>
    <p>H. Cotransfection of pSDL114 with Fab'</p>
    <p>A stably transfected SP2/0-AG14 isolate containing pSDL114 and p416 was transfected with plasmid pWKI, which contained the Fab' portion of an immunoglobulin heavy chain gene. Plasmid pWKI was constructed as follows.</p>
    <p>The immunoglobulin gamma-1 Fab' sequence, comprising the C<sub>H</sub> 1 and hinge regions sequences, was derived from the gamma-1 gene clone described in Example 12.C. The gamma-1 gene clone was digested with Hind III and Eco RI to isolate the 3.0 kb fragment, which was subcloned into Hind III-Eco RI linearized M13mp19. Single-stranded template DNA from the resultant phage was subjected to site-directed mutagenesis using oligonucleotide ZC1447 (Sequence ID Number 9; Table 1) and essentially the method of Zoller and Smith (ibid.). A phage clone was identified having a ZC1447 induced deletion resulting in the fusion of the hinge region to a DNA sequence encoding the amino acids Ala-Leu-His-Asn-His-Tyr-Thr-Glu-Lys-Ser-Leu-Ser-Leu-Ser-Pro-Gly-Lys (Sequence ID Number 31) followed in-frame by a stop codon. Replicative form DNA from a positive phage clone was digested with Hind III and Eco RI to isolate the 1.9 kb fragment comprising the C<sub>H</sub> 1 and hinge regions. Plasmid pφ5V<sub>H</sub> was digested with Sst I and Hind III to isolate the 5.3 kb fragment comprising the mouse immunoglobulin heavy chain gene variable region. Plasmid pICneo was linearized by digestion with Sst I and Eco RI. The linearized pICneo was joined with the 5.3 kb Hind III-Sst I fragment and the 1.9 kb Hind III-Eco RI fragment in a three-part ligation. The resultant plasmid was designated pWKI (FIG. 10).</p>
    <p>An SP2/0-AG14 clone stably transfected with pSDL114 and p416 was transfected with Asp 718-linearized pWKI. Transfected cells were selected by growth in medium containing methotrexate and G418. Media samples from transfected cells were assayed using the competition assay described in Example 12.C. Results from the assays showed that the transfected cells produced a PDGF β-receptor analog capable of competetively binding PDGF-BB.</p>
    <heading>Example 13</heading> <heading>Purification and Characterization of PDGF β-Receptor Analogs from Mammalian cells Co-transfected With pSDL113 and pSDL114</heading> <p>A. Purification of PDGF β-Receptor Analogs</p>
    <p>The PDGF β-receptor analog was purified from conditioned culture media from a clone of transfected cells grown in a hollow fiber system. The media was passed over a protein-A sepharose column, and the column was washed sequentially with phosphate buffered saline, pH 7.2 (PBS; Sigma, St. Louis, Mo.) and 0.1 M citrate, pH 5.0. The PDGF β-receptor analog was eluted from the protein-A column with 0.1 M citrate pH 2.5 and immediately neutralized by the addition of Tris-base, pH 7.4. The eluate fractions containing PDGF β-receptor analog, as determined by silver stain, were pooled and chromatographed over an S-200 column (Pharmacia LKB Technologies, Inc., Piscataway, N.J.) equilibrated with PBS. The peak fractions from the S-200 column were pooled and concentrated on a centriprep-10 concentrator (Amicon). Glycerol (10% final volume) was added to the preparation and the sample frozen at -80° C. PDGF β-receptor analogs purified from pSDL114+pSDL113 co-transfected cells were termed "tetrameric PDGF α-receptors".</p>
    <p>B. Measurement of The Relative Binding Affinity of Tetrameric PDGF β-Receptor Analog by Soluble Receptor Assay</p>
    <p>Purified tetrameric PDGF β-receptor analog was compared to detergent solubilized extracts of human dermal fibroblasts for <sup>125</sup> I-labeled PDGF-BB binding activity in a soluble receptor assay essentially as described by Hart et al. (J. Biol. Chem. 262: 10780-10785, 1987). Human dermal fibroblast cells were extracted at 20×10<sup>6</sup> cell equivalents per ml in TNEN extraction buffer (20 mM Tris-HCl, pH 8.0, 100 mM NaCl, 1 mM EDTA, 0.5% Nonidet P-40, 1 mM PMSF, 10% glycerol). Two hundred and fifty thousand PDGF β-receptor-subunits per cell was used to calculate the tetrameric PDGF β-receptor analog number per volume of extract. This value has been previously published by Seifert et al. (J. Biol Chem. 264: 8771-8778, 1989). The PDGF β-receptor analog number was determined from the protein concentration of the PDGF β-receptor analog assuming an average molecular weight of 140 kDa for each immunoglobulin-PDGF β-receptor monomer, and four monomers per tetramer. Thus, each tetrameric molecule contains four receptor molecules.</p>
    <p>Increasing amounts of either detergent solubilized extracts of human dermal fibroblast cells or purified PDGF β-receptor analog were incubated with 1 ng of <sup>125</sup> I-labeled PDGF-BB for one hour at 37° C. The sample was then diluted with 1 ml binding media and was added to monolayers of human dermal fibroblast cells grown in 24-well culture dishes. The samples were incubated for two hours at 4° C. The wells were washed to remove unbound, <sup>125</sup> I-labeled PDGF-BB. On half of a milliliter of extraction buffer (PBS+1% Nonidet P-40) was added to each well followed by a 5 minute incubation. The extraction mixtures were harvested and counted in a gamma counter.</p>
    <p>The results showed that the PDGF β-receptor analog had the same relative binding affinity as solubilized PDGF β-receptor-subunit from mammalian cells in a solution phase binding assay.</p>
    <p>C. Determination of the Binding Affinity of the PDGF β-Receptor Analog in a Solid Phase Format</p>
    <p>The apparent dissociation constant K<sub>D</sub> (app) of the PDGF β-receptor analog was determined essentially as described by Bowen-Pope and Ross (Methods in Enzymology 109: 69-100, 1985), using the concentration of <sup>125</sup> I-labeled PDGF-BB giving half-maximal specific <sup>125</sup> I-labeled PDGF-BB binding. Saturation binding assays to determine the concentration of <sup>125</sup> I-labeled PDGF-BB that gave half-maximal binding to immobilized PDGF β-receptor analog were conducted as follows.</p>
    <p>Affinity purified goat anti-human IgG, H- and L-chain (Commercially available from Cappel Labs) was diluted into 0.1 M Na<sub>2</sub> HCO<sub>3</sub>, pH 9.6 to a concentration of 2 μg/ml. One hundred microliters of the antibody solution was coated onto each well of 96-well microtiter plates for 18 hours at 4° C. The wells were washed once with ELISA C buffer (PBS+0.05% Tween-20) followed by an incubation with 175 μl/well of ELISA B buffer (PBS+1% BSA+0.05% Tween-20) to block the wells. The wells were washed once with ELISA B buffer. One hundred microliters of 12.1 ng/ml or 24.3 ng/ml of tetrameric PDGF β-receptor analog protein diluted in ELISA B was added to each well and the plates were incubated for 2 hours at 37° C. Unbound protein was removed from the wells by two washes with ELISA C. <sup>125</sup> I-labeled PDGF-BB<sub>Tyr</sub> (Example 18.F.) was serially diluted into binding media (25 mM HEPES, pH 7.2, 0.25% rabbit serum albumin diluted in HAMs P-12 medium (GIBCO-BRL)), and 100 μl of the dilutions were added to the wells. The plates were incubated for two hours at room temperature. The unbound <sup>125</sup> I-labeled PDGF-BB was removed, and the wells were washed three times with binding media. Following the last wash, 100 μl of 0.1 M citrate, pH 2.5 was added to each well. After five minutes, the citrate buffer was removed, transferred to a tube and counted in a gamma counter. The counts reflect counts of <sup>125</sup> I-labeled PDGF-BB<sub>Tyr</sub> bound by the receptor analog. Nonspecific binding for each concentration of <sup>125</sup> I-labeled PDGF-BB<sub>Tyr</sub> was determined by a parallel assay wherein separate wells coated only with goat anti-human IgG were incubated with the <sup>125</sup> I-labeled PDGF-BB concentrations. Nonspecific binding was determined to be 2.8% of the total input counts per well and averaged 6% of the total counts bound.</p>
    <p>Saturation binding assay on 12.1 and 24.3 ng/ml of tetrameric PDGF β-receptor analog gave half-maximal binding at 0.8 and 0.82 ng/ml <sup>125</sup> I-labeled PDGF-BB<sub>Tyr</sub>, respectively. By Scatchard analysis (Scatchard, Ann. NY Acad. Sci 51: 660-667, 1949) these values were shown to correspond to a K<sub>D</sub> (app) of 2.7×10<sup>-11</sup> which agree with the published values for PDGF receptors on mammalian cells.</p>
    <heading>Example 14</heading> <heading>Solid Phase Ligand Binding Assay Using the PDGF β-Receptor Analog</heading> <p>A. Solid Phase Radioreceptor Competition Binding Assay</p>
    <p>In a solid phase radioreceptor competition binding assay (RRA), the wells of 96-well microtiter plates were coated with 100 μl of 2 μg/ml affinity purified goat anti-human IgG (Cappel Labs) diluted in 0.1 M Na<sub>2</sub> HCO<sub>3</sub>, pH 9.6. After an eighteen hour incubation at 4° C., the wells were washed once with ELISA C. The wells were blocked by incubation for 2 hours at 37° C. with 175 μl/well ELISA B. The wells were washed once with ELISA B then incubated for 2 hours at 37° C. with 50 ng/ml tetrameric PDGF β-receptor analog diluted in ELISA B. The unbound receptor was removed, and the test wells were incubated with increasing concentrations of serially diluted, unlabeled PDGF-BB (diluted in binding media. Following a two hour incubation at room temperature, the wells were washed three times with binding media. One hundred microliters of 5 ng/ml <sup>125</sup> I-labeled PDGF-BB<sub>Tyr</sub> (Example 18.F.) was added to each well, and the plates were incubated for an additional two hours at room temperature. The wells were washed three times with binding media followed by a 5 minute incubation with 100 μl/well of 0.1 M citrate, pH 2.5. The samples were harvested and counted in a gamma counter.</p>
    <p>Radioreceptor assay (RRA) competition binding curves were generated for PDGF β-receptor analog protein plated at 48.6 ng/ml. The sensitivity of the assays is 1 ng/ml of PDGF-BB, with 8 ng/ml giving 50% inhibition in <sup>125</sup> I-PDGF-BB binding, and a working range between 1 and 32 ng/ml of PDGF-BB. The values were similar to those obtained using monolayers of SK-5 cells in an RRA.</p>
    <p>B. Use of Tetrameric PDGF β-Receptor Analogs As Antagonists for PDGF-Stimulated Mitogenesis.</p>
    <p>A tetrameric PDGF β-receptor analog, purified as described in Example 13, was analyzed for the ability to neutralize PDGF-stimulated mitogenesis in mouse 3T3 cells. Increasing amounts of the purified tetrameric PDGF β-receptor analog were mixed with 5 ng of PDGF. The mixtures were then added to cultures of mouse 3T3 cells. The ability of the PDGF to stimulate a mitogenic response, as measured by the incorporation of <sup>3</sup> H-thymidine, was determined essentially as described (Raines and Ross, Methods in Enzymology 109: 749-773, 1985, which is incorporated by reference herein). The tetrameric PDGF β-receptor analog demonstrated a dose response inhibition of PDGF-BB-stimulated <sup>3</sup> H-thymidine incorporation, while having essentially no effect on PDGF-AA- and PDGF-AB-stimulated <sup>3</sup> H-thymidine incorporation.</p>
    <p>C. Binding of Tetrameric PDGF β-receptor Analog to Immobilized PDGF.</p>
    <p>A tetrameric PDGF β-receptor analog, purified as described in Example 13, was analyzed for its ability to bind to immobilized PDGF. PDGF-BB (100 ng/ml) was coated onto wells a 96-well microtiter plate, and the plates were incubated 18 hours at 4° C. followed by one wash with ELISA C buffer. The wells were incubated for 2 hours 37° C. with ELISA B buffer to block the wells. Increasing concentrations of <sup>125</sup> I-labeled tetrameric PDGF β-receptor analog, diluted in binding media, was added to the wells for two hours at room temperature. The wells were washed four times with ELISA C buffer to remove unbound receptor analog. One hundred microliters of 1 M H<sub>2</sub> SO<sub>4</sub> was added to each well and the plates were incubated for five minutes at room temperature. The solution was then harvested and transferred to tubes to be counted in a gamma counter. Nonspecific binding was determined to be less than 10% of the total counts bound.</p>
    <p>A receptor competition binding assay was developed using this assay format. The assay was carried out as described above, and simultaneous to the addition of the <sup>125</sup> I-labeled tetrameric PDGF β-receptor analog, increasing amounts of PDGF-AA, AB or BB were added to the PDGF-BB coated wells. Under these condtions, only PDGF-BB was found to significantly block the binding of the labeled PDGF β-receptor analog to the immobilized PDGF-BB.</p>
    <heading>Example 15</heading> <heading>Construction and Expression of PDGFα-R Analogs in Cultured Mouse Myeloma Cells</heading> <p>A. Construction of an optimized PDGFα-R cDNA</p>
    <p>The PDGF α-receptor coding region was optimized for expression in mammalian cells as follows. The 5' end of the cDNA was modified to include an optimized Kozak consensus translation initiation sequence (Kozak, Nuc. Acids Res. 12: 857-872, 1984) and Eco RI and Bam HI sites just 5' of the initiation methionine codon. Oligonucleotides ZC2181, ZC2182, ZC2183 and ZC2184 (Sequence ID Numbers 23, 24, 25 and 26, respectively; Table 1) were designed to form, when annealed, an adapter having an Eco RI adhesive end, a Bam HI restriction site, a sequence encoding a Kozak consensus sequence 5' to the initation methionine codon, a mammalian codon optimized sequence encoding amino acids 1-42 of FIGS. 11A-11D, and an Eco RI adhesive end that destroys the Eco RI site within the PDGFα-R coding sequence. The adapter also introduced a diagnostic Cla I site 3' to the initiation methionine codon. Oligonucleotides ZC2181, ZC2182, ZC2183 and ZC2184 were kinased, annealed and ligated. Plasmid pα17B was linearized by partial digestion with Eco RI. The linearized pα17B was ligated with the ZC2181/2C2182/ZC2183/ZC2184 oligonucleotide adapter, and the ligation mixture was transformed into E. coli. Plasmid DNA prepared from the transformants was analyzed by restriction analysis and a positive clone having the oligonucleotide adapter in the correct orientation was digested with Eco RI and Pst I to isolate the 1.6 kb fragment. This fragment was subcloned into Eco RI+Pst I-linearized M13mp19. The resultant phage clone was designated 792-8. Single-stranded 792-8 DNA was sequenced to confirm the orientation of the adapter.</p>
    <p>A fragment encoding the ligand-binding domain of the PDGF α-receptor (PDGFα-R) was then generated as follows. Restriction sites and a splice donor sequence were introduced at the 3' end of the PDGFα-R extracellular domain by PCR amplification of the 792-8 DNA and oligonucleotides ZC2311 and ZC2392 (Sequence ID Numbers 27 and 30, Table 1). Oligonucleotide ZC2311 is a sense primer encoding nucleotides 1470 to 1489 of FIG. 11B. Oligonucleotide ZC2392 is an antisense primer that encodes nucleotides 1759 to 1776 of FIG. 11B followed by a splice donor and Xba I and Hind III restriction sites. The 792-8 DNA was amplified using manufacturer recommended (Perkin Elmer Cetus, Norwalk, Conn.) conditions and the GeneAmp™ DNA amplification reagent kit (Perkin Elmer Cetus), and blunt-ended 329 bp fragment was isolated. The blunt-end fragment was digested with Nco I and Hind III and ligated with Sma I-digested pUC18. A plasmid having an insert with the Nco I site distal to the Hind III site present in the pUC18 polylinker was designated pUC18 Sma-PCR Nco HIII #13. The Hind III site present in the insert was not regenerated upon ligation with the linearized pUC18. Plasmid pUC18 Sma-PCR Nco HIII #13 was digested with Nco I and Hind III to isolate the 355 bp PDGFα-R containing fragment encoding PDGFαR. oligonucleotides ZC2351 and ZC2352 (Table 1; Sequence ID Numbers 28 and 29) were kinased and annealed to form an Sst I-Nco I adapter encoding an internal Eco RI site and a Kozak consensus translation initiation site. The 355 bp Nco I-Hind III fragment, the ZC2351/ZC2352 adapter and a 1273 bp Nco I fragment comprising the extracellular domain of of PDGF α-R derived from 792-8 were ligated with Hind III+SstI-digested pUC18. and tranformed into E. coli. Plasmid DNA was isolated from the transformants and analyzed by restriction analysis. None of the isolates contained the 1273 bp Nco I fragment. A plasmid containing the Nco I-Hind III fragment and the ZC2351/ZC2352 adapter was desginated pUC18 Hin Sst Δ Nco #46. Plasmid pUC18 Hin Sst Δ Nco #46 was linearized by digestion and joined by ligation with the 1273 bp Nco I fragment comprisising the extracellular domain of the PDGFα-R from clone α18 R-19. The ligations were transformed into E. coli, and plasmid DNA was isolated from the transformants. Analysis of the plasmid DNA showed that only clones with the Nco I fragment in the wrong orientation were isolated. A clone having the Nco I fragment in the wrong orientation was digested with Nco I, religated and transformed into E. coli. Plasmid DNA was isolated from the transformants and was analyzed by restriction analysis. A plasmid having the Nco I insert in the correct orientation was digested to completion with Hind III and partially digested with Sst I to isolate the 1.6 kb fragment comprising the extracellular domain of the PDGFα-R preceded by a consensus initiation sequence (Kozak, ibid.) and followed by a splice donor site.</p>
    <p>B. Construction of pPAB7</p>
    <p>The DNA sequence encoding the extracellular domain of the PDGFα-R was joined to the immunoglobulin μ enhancer-promoter and to a DNA sequence encoding an immunoglobulin light chain constant region. The immunoglobulin μ enhancer-promoter was obtained from plasmid pJH1 which was derived from plasmid PICμPRE8 (Example 12.A.) by digestion with Eco RI and Sst I to isolate the 2.2 kb fragment comprising the immunoglobulin enhancer and heavy chain variable region promoter. The 2.2 kb Sst I-Eco RI fragment was ligated with Sst I+Eco RI-linearized pUC19. The resulting plasmid, designated pJH1, contained the immunoglobulin enhancer and heavy chain variable region promoter immediately 5' to the pUC19 linker sequences. Plasmid pJH1 was linearized by digestion with Sst I and Hind III and joined with the 1.6 kb partial Sst I-Hind III fragment containing the PDGFα-R extracellular domain sequences. The resulting plasmid having the immunoglobulin μ enhancer-promoter joined to the PDGFα-R extracellular domain was designated pPAB6. Plasmid pSDL112 was digested with Hind III to isolate the 1.2 kb fragment encoding the immunoglobulin light chain constant region (Cκ). The 1.2 kb Hind III fragment was ligated with Hind III-linearized pPAB6. A plasmid having the C.sub.κ  sequence in the correct orientation was desginated pPAB7.</p>
    <p>C. Construction of pPAB9</p>
    <p>The partial Sst I-Hind III fragment encoding the extracellular domain of the PDGFα-R was joined to the immunoglobulin heavy chain constant region. For convenience, the internal Xba I site in plasmid pJH1 was removed by digestion with Xba I, blunt-ending with T4 DNA polymerase, and religation. A plasmid which did not contain the internal Xba I site, but retained the Xba I site in the polylinker was desginated 11.28.3.6. Plasmid 11.28.3.6 was linearized by digestion with Sst I and Xba I. Plasmid pPAB6 was digested to completion with Hind III and partially digested with Sst I to isolate the 1.6 kb Sst I-Hind III fragment containing the PDGFα-R extracellular domain. Plasmid pφ5V<sub>H</sub> huC.sub.γ 1M-neo (Example 12.C.) was digested wtih Hind III and Xba I to isolate the 6.0 kb fragment encoding the immunoglobulin heavy chain constant region (huC.sub.γ 1M). The Sst I-Hind III-linearized 11.28.3.6, the 1.6 kb Sst I-Hind III PDGFα-R fragment and the 6.0 kb Hind III-Xba I huC.sub.γ 1M fragment were ligated to form plasmid pPAB9.</p>
    <p>D. Expression of pPAB9 in Mammalian Cells</p>
    <p>Bgl II-linearized pPAB7 and Pvu I-linearized pPAB9 were cotransfected with Pvu I-linearized p416 into SP2/0-Ag14 cells by electroporation. Transfected cells were initially selected in growth medium containing 50 nM methotrexate and were subsequently amplified in a growth medium containing 100 μM methotrexate. Media from drug resistant clones were tested for the presence of secreted PDGF α-receptor analogs by enzyme-linked immunosorbant assay (ELISA). Ninety-six well assay plates were prepared by incubating 100 μl of 1 μg/ml monoclonal antibody 292.1.8 which is specific for the PDGF α-receptor diluted in phosphate buffered saline (PBS; Sigma] overnight at 4° C. Excess antibody was removed by three washes with 0.5% Tween 20 in PBS. One hundred microliters of spent media was added to each well, and the plates were incubated for one hour at 4° C. Unbound proteins were removed by eight washes with 0.5% Tween 20 in PBS. One hundred microliters of peroxidase-conjugated goat anti-human IgG heavy chain antibody (diluted 1:1000 in a solution containing 5% chicken serum (GIBCO-BRL)+0.5% Tween 20 in PBS) was added to each well, and the plates were incubated for one hour at 4° C. One hundred microliters of chromophore (100 μl ABTS [2,2'-Azinobis(3-ethylbenz-thiazoline sulfonic acid] diammonium salt; Sigma]+1 μl 30% H<sub>2</sub> O<sub>2</sub> +12.5 ml citrate/phosphate buffer [9.04 g/l citric acid, 10.16 g/l Na<sub>2</sub> HPO<sub>4</sub> ]) was added to each well, and the wells were incubated for 30 minutes at room temperature. The samples were measured at 405 nm. The results of the assay showed that the PDGF α-receptor analogs secreted by the transfectants contained an immunoglobulin heavy chain.</p>
    <p>Analysis of spent media from transfected cells by Northern analysis, Western analysis and by radioimmunoprecipitation showed that the transfectants did not express a PDGF α-receptor analog from the pPAB7 construction. Transfectants were subsequently treated as containing only pPAB9.</p>
    <p>Drug resistant clones were also tested for the presence of secreted PDGF α-receptor analogs by immunoprecipitation. For each clone, approximately one million drug resistant transfectants were grown in DMEM lacking cysteine+2% calf serum for 18 hours at 37° C. in the presence of 50 μCi <sup>35</sup> S-cysteine. The spent media was harvested from the labeled cells and 250 μl of medium from each clone was assayed for binding to the anti-PDGF α-receptor antibody 292.18. Monoclonal antibody 292.18 diluted in PBS was added to each sample to a final concentration of 2.5 μg per 250 μl spent media. Five microliters of rabbit anti-mouse Ig diluted in PBS was added to each sample, and the immunocomplexes were precipitated by the addition of 50 μl 10% fixed Staph A (weight/volume in PBS). The immunocomplexes were analyzed on 8% SDS-polyacrylamide gels followed by autoradiography overnight at -70° C. The results of the immunoprecipitation showed that the PDGF α-receptor analog secreted by the transfectants was bound by the anti-PDGF α-receptor antibody. The combined results of the ELISA and immunoprecipitation assays showed that the PDGF α-receptor analog secreted by the transfectants contained both the PDGF α-receptor ligand-binding domain and the human heavy chain.</p>
    <p>Spent medium from drug-resistant clones were tested for their ability to bind PDGF in a competition binding assay esssentially as described in Example 12.C. The results of the assay showed that the transfectants secreted a PDGF α-receptor analog capable of binding PDGF-AA. A clone containing the pPAB9 was desginated 3.17.1.57.</p>
    <p>E. Co-expression of pPAB7 and pPAB9 in Mammalian Cells</p>
    <p>Bgl II-linearized pPAB7 and Bam HI-linearized pICneo were cotransfected into clone 3.17.1.57, and transfected cells were selected in the presence of neomycin. Media from drug resistant cells were assayed for the presence of immunoglobulin heavy chain, immunoglobulin light chain and the PDGF α-receptor ligand-binding domain by ELISA essentially as described above. Briefly, ninety-six well assay plates were prepared by incubating 100 μl of 1 μg/ml goat anti-human IgG Fc antibody (Sigma) or 100 μl of 1 μg/ml 292.18 overnight at 4° C. Excess antibody was removed by three washes with 0.5% Tween 20 in PBS. One hundred microliters of spent media was added to each well of each plate, and the plates were incubated for one hour at 4° C. Unbound proteins were removed by eight washes with 0.5% Tween 20 in PBS. One hundred microliters of peroxidase-conjugated goat anti-human IgG antibody (diluted 1:1000 in a solution containing 5% chicken serum (GIBCO-BRL)+0.5% Tween 20 in PBS) was added to each well of the plate coated with the anti-Fc antibody, and 100 μl of peroxidase-conjugated goat anti human kappa antibody (diluted 1:1000 in a solution containing 5% chicken serum (GIBCO-BRL)+0.5% Tween 20 in PBS) was added to each well of the plate coated with 292.18. The plates were incubated for one hour at 4° C. One hundred microliters of chromophore (100 μl ABTS (2,2'-Azinobis(3-ethylbenz-thiazoline sulfonic acid) diammonium salt; Sigma]+1 μl 30% H<sub>2</sub> O<sub>2</sub> +12.5 ml citrate/phosphate buffer [9.04 g/l citric acid, 10.16 g/l Na<sub>2</sub> HPO<sub>4</sub> ]) was added to each well of each plate, and the plates were incubated to 30 minutes at room temperature. The samples were measured at 405 nm, the wavelength giving maximal absorbance of the chromogenic substrate, to identify clones having absorbances higher than background indicating the presence of immunoglobulin heavy chain. Clones that gave positive results in both ELISA assays (showing that the clones produced proteins containing heavy chain regions, light chain constant regions and the PDGF α-receptor ligand-binding region) were selected for further characterization.</p>
    <p>Drug resistant clones that were positive for both ELISA assays were subsequently tested for the presence of secreted PDGF α-receptor analogs by immunoprecipitation. For each positive clone, approximately one million drug resistant transfectants were grown in DMEM lacking cysteine+2% calf serum for 18 hours at 37° C. in the presence of 50 μCI <sup>35</sup> S-cysteine. The spent media was harvested from the labeled cells and 250 μl of medium from each clone was assayed for binding to monoclonal antibody 292.18. Monoclonal antibody 292.18 diluted in PBS was added to each sample to a final concentration of 2.5 μg. Five microliters of rabbit anti-mouse Ig diluted in PBS was added to each sample and the immunocomplexes were precipitated by the addition of 50 μl 10% fixed Staph A (weight/volume in PBS). The immunocomplexes were analyzed on 8% SDS-polyacrylamide gels followed by autoradiography overnight at -70° C. The results of the immunoprecipitation showed that the PDGF α-receptor analog secreted by the transfectants was bound by the anti-PDGF α-receptor antibody. The combined results of the ELISA and immunoprecipitation assays showed that the PDGF α-receptor analog secreted by the transfectants contained the PDGF α-receptor ligand-binding domain, the human heavy chain and the human light chain constant region. A clone that secreted a PDGF α-receptor analog that was positive for both the above-described ELISA assays and the immunoprecipitation assay was designated 5.6.2.1.</p>
    <heading>Example 16</heading> <heading>Purification and characterization of PDGF α-Receptor Analogs</heading> <p>A. Purification of PDGF α-Receptor Analogs From Clone 3.17.1.57</p>
    <p>The PDGF α-Receptor analog was purified from the conditioned culture media of clone 3.17.1.57 by cycling cell-conditioned medium over an immunoaffinity column composed of monoclonal antibody 292.18 bound to a CNBr-activated Sepharose 4B resin, which is specific for the PDGF α-receptor. The column was washed with PBA, then eluted with 0.1 M citrate, pH 3.0. The peak column fractions containing the α-receptor were pooled, neutralized to pH 7.2 by the addition of 2 M Tris, pH 7.4, then passed over a protein-A Sepharose column. This column was washed sequentially with PBS, then with 0.1 M citrate, pH 5.0. The PDGF α-receptor analog was then eluted with 0.1 M citrate, pH 3.0. The peak eluate fractions were pooled, and glycerol was added to a final concentration of lot. The sample was concentrated on a centriprep 10 concentrator (Amicon). The PDGF α-receptor analog purified from clone 3.17.1.57 was termed a "dimeric PDGF α-receptor analog".</p>
    <p>B. Purification of PDGF α-Receptor Analogs From Clone 5.6.2.1</p>
    <p>The PDGF α-receptor analog was purified from the conditioned culture media of clone 5.6.2.1 by cycling cell-conditioned medium over the immunoaffinity column described above. The column was washed with PBS then eluted with 0.1 M citrate, pH 3.0. The peak column fractions containing the α-receptor were pooled, neutralized to pH 7.2 by the addition of 2 M Tris (what pH 7.4), then passed over a protein-A sepharose column. This column was washed sequentially with PBS then with 0.1 M citrate, pH 5.0. The PDGF α-receptor analog was then eluted with 0.1 M citrate, pH 3.0. The peak eluate fractions were pooled and glycerol was added to a final concentration of 10%. The sample was concentrated on a centriprep 10 concentrator. The PDGF α-receptor analogs purified from clone 5.6.2.1 was termed a "tetrameric PDGF α-receptor analog".</p>
    <heading>Example 17</heading> <p>A. Use of the PDGF α-receptor Analogs in Ligand Binding Studies</p>
    <p>Purified tetrameric PDGF α-receptor analog and purified dimeric PDGF α-receptor analog were compared to monolayers of a control cell line of canine kidney epithelial cells, which do not naturally express the PDGF α-receptor, transfected with the human PDGF α-receptor cDNA for ligand binding activity. The dissociation constant (Kd) of the receptor preparations was determined by saturation binding and subsequent Scatchard analysis.</p>
    <p>Ligand binding of the purified PDGF α-receptor analogs was determined using a solid phase binding assay. Affinity-purified goat anti-human IgG was diluted to a concentration of 2 μg/ml in 0.1 M Na<sub>2</sub> HCO<sub>3</sub>, pH 9.6 and 100 μl/well of the solution was used to coat 96-well microtiter plates for 18 hours at 4° C. Excess antibody was removed from the wells with one wash with ELISA C buffer (PBS, 0.05% Tween-20). The plates were incubated with 175 μl/well of ELISA B buffer (PBS, 1% BSA, 0.05% Tween-20) to block the wells, followed by two washes with ELISA C buffer. One hundred microliters of 50 ng/ml PDGF α-receptor analog (dimeric or tetrameric) diluted in ELISA buffer B was added to each well and the plates were incubated over night at 4° C. Unbound protein was removed from the wells with two washes with ELISA buffer B. <sup>125</sup> I-labeled PDGF-AA was serially diluted in binding media (Hams F-12, 25 mM HEPES pH 7.2, 0.25% rabbit serum albumin), and 100 μl of each dilution was added to the wells. The samples were incubated for two hours at room temperature. Unbound <sup>125</sup> I-labeled PDGF-AA was removed with three washes with binding media. One hundred microliters of 0.1 M citrate, pH 2.5 was added to each well, and the plates were incubated for five minutes. After the incubation, the citrate buffer was removed and transferred to a tube for counting in a gamma counter. Nonspecific binding for each concentration of <sup>125</sup> I-labeled PDGF-AA was determined by a parallel assay wherein separate wells coated only with goat anti-human IgG were incubated with the <sup>125</sup> I-labeled PDGF-AA samples.</p>
    <p>A saturation binding assay was performed on alpha T-7 cells transfected with the PDGF α-receptor. The cells were grown to confluency in 24-well culture plates. The cells were washed one time with binding media. Iodinated PDGF-AA was serially diluted in binding media. One milliliter of each dilution was added to the wells, and the plates were incubated for 3 hours at 4° C. Unbound <sup>125</sup> I-labeled PDGF-AA was removed and the cells were washed three times with binding media. PBS containing 1% Triton X-100 was added to the cells for 5 minutes. The extracts were harvested and counted in a gamma counter. Nonspecific binding was determined at a single concentration of <sup>125</sup> -labeled PDGF-AA using a 500-fold excess PDGF-BB.</p>
    <p>The dissociation constants determined by Scatchard analysis (ibid.) of the saturation binding assays for the tetrameric PDGF α-receptor analog, dimeric PDGF α-receptor analog and the control cells (Table 5).</p>
    <heading>Table 5</heading> <heading>Dissociation Constants for the Tetrameric PDGF α-Receptor, the Dimeric PDGF α-receptor and control cells Transfected with the PDGF α-receptor</heading> <p>
      </p> <pre xml:space="preserve" listing-type="tabular">              TABLE 5______________________________________Dissociation Constants for the Tetrameric PDGF α-Receptor,  the Dimeric PDGF α-receptor and control cells Transfected with the PDGF α-receptorReceptor               kD______________________________________Tetrameric PDGF α-receptor analog                    1.6 × 10<sup>-11</sup>  Dimeric PDGF α-receptor analog 8.51 × 10<sup>-11</sup>  Control cells [PDGF α-receptor]  3.7 × 10<sup>-11</sup>______________________________________</pre>
    
    <p>A solid-phase competition binding assay was established using the tetrameric PDGF α-receptor analog. Ninety six-well microtiter plates were coated with goat anti-human IgG (2 μg/ml), the wells blocked with ELISA B buffer, 50 ng/ml of purified tetrameric PDGF α-receptor analog diluted in binding media was added, and the plates were incubated two hours at room temperature. Unbound receptor was removed and the wells were washed with binding media. The plates were incubated for two hours at room temperature with increasing concentrations of either PDGF-AA or PDGF-BB diluted in binding media. The wells were washed, then incubated for two hours at room temperature with 3 ng/ml <sup>125</sup> I-labeled PDGF-AA diluted in binding media. Unbound labeled PDGF-AA was removed, the wells were subsequently washed with binding media, and the bound, labeled PDGF-AA was harvested by the addition of 0.1 M citrate, pH 2.5, as described for the saturation binding studies. PDGF-AB, PDGF-AA and PDGF-BB were found to compete for receptor binding with <sup>125</sup> I-PDGF-AA.</p>
    <p>B. Use of Tetrameric PDGF α-Receptor Analogs As Antagonists for PDGF-Stimulated Mitogenesis.</p>
    <p>A dimeric PDGF α-receptor analog, purified as described in Example 16.B., was analyzed for the ability to neutralize PDGF-stimulated mitogenesis in mouse 3T3 cells. Increasing amounts of the purified tetrameric PDGF α-receptor analog were mixed with PDGF-AA, -AB or -BB ranging 0.6 to 5 ng. The mixtures were then added to cultures of confluent mouse 3T3 cells. The ability of the PDGF to stimulate a mitogenic response, as measured by the incorporation of <sup>3</sup> H-thymidine, was determined essentially as described (Raines and Ross, Methods in Enzymology 109: 749-773, 1985, which is incorporated by reference herein). The dimeric PDGF α-receptor analog demonstrated a dose response inhibition of PDGF-stimulated <sup>3</sup> H-thymidine incorporation for all three isoforms of PDGF.</p>
    <p>C. Inverse Ligand-Receptor Radioreceptor Assay</p>
    <p>An inverse ligand-receptor radioreceptor assay was designed to screen for the presence of PDGF-BB, PDGF-BB binding proteins, PDGF-BB related molecules, and PDGF-β receptor antagonists in test samples. PDGF-BB (100 ng/ml) was coated onto the walls of 96-well microtiter plates, and the plates were incubated at 4° C. for 16 hours. The wells were washed once with ELISA C buffer and then incubated with ELISA B buffer to block the nonspecific binding sites. To the wells were added 50 μl of either PDGF standard or a test sample and 50 μl of <sup>125</sup> I-labeled tetrameric PDGF β-receptor analog. The samples were incubated for one hour at room temperature. The wells were washed once with ELISA C buffer, and 0.1 M citrate, pH 2.5 containing 1% NP-40 was added to each well to disrupt the ligand-receptor analog bond and elute the bound receptor analog. The acid wash was collected and counted in a gamma counter. The presence of PDGF or a molecule which mimics PDGF or otherwise interferes with the binding of the well-bound PDGF-BB with its receptor will cause a decrease in the binding of the radiolabeled tetrameric PDGF β-receptor. Using this assay, PDGF-BB was found to inhibit receptor binding while PDGF-AA and PDGF-AB caused no significant decrease in receptor binding.</p>
    <heading>Example 18</heading> <heading>Assay Methods</heading> <p>A. Preparation of Nitrocellulose Filters for Colony Assay</p>
    <p>Colonies of transformants were tested for secretion of PDGF β-receptor analogs by first growing the cells on nitrocellulose filters that had been laid on top of solid growth medium. Nitrocellulose filters (Schleicher &amp; Schuell, Keene, N.H.) were placed on top of solid growth medium and were allowed to be completely wetted. Test colonies were patched onto the wetted filters and were grown at 30° C. for approximately 40 hours. The filters were then removed from the solid medium, and the cells were removed by four successive rinses with Western Transfer Buffer (Table 4). The nitrocellulose filters were soaked in Western Buffer A (Table 4) for one hour at room temperature on a shaking platform with two changes of buffer. Secreted PDGFβ-R analogs were visualized on the filters described below.</p>
    <p>B. Preparation of Protein Blot Filters</p>
    <p>A nitrocellulose filter was soaked in Western Buffer A (Table 4) without the gelatin and placed in a Minifold (Schleicher &amp; Schuell, Keene, N.H.). Five milliliters of culture supernatant was added without dilution to the Minifold wells, and the liquid was allowed to pass through the nitrocellulose filter by gravity. The nitrocellulose filter was removed from the minifold and was soaked in Western Buffer A (Table 3) for one hour on a shaking platform at room temperature. The buffer was changed three times during the hour incubation.</p>
    <p>C. Preparation of Western Blot Filters</p>
    <p>The transformants were analyzed by Western blot, essentially as described by Towbin et al. (Proc. Natl. Acad. Sci. USA 76: 4350-4354, 1979) and Gordon et al. (U.S. Pat. No. 4,452,901). Culture supernatants from appropriately grown transformants were diluted with three volumes of 95% ethanol. The ethanol mixtures were incubated overnight at -70° C. The precipitates were spun out of solution by centrifugation in an SS-24 rotor at 18,000 rpm for 20 minutes. The supernatants were discarded and the precipitate pellets were resuspended in 200 μl of dH<sub>2</sub> O. Two hundred microliters of 2× loading buffer (Table 4) was added to each sample, and the samples were incubated in a boiling water bath for 5 minutes.</p>
    <p>The samples were electrophoresed in a 15% sodium dodecylsulfate polyacrylamide gel under non-reducing conditions. The proteins were electrophoretically transferred to nitrocellulose paper using conditions described by Towbin et al. (ibid.). The nitrocellulose filters were then incubated in Western Buffer A (Table 4) for 75 minutes at room temperature on a platform rocker.</p>
    <p>D. Processing the Filters for Visualization with Antibody</p>
    <p>Filters prepared as described above were screened for proteins recognized by the binding of a PDGF β-receptor specific monoclonal antibody, designated PR7212. The filters were removed from the Western Buffer A (Table 4) and placed in sealed plastic bags containing a 10 ml solution comprising 10 μg/ml PR7212 monoclonal antibody diluted in Western Buffer A. The filters were incubated on a rocking platform overnight at 4° C. or for one hour at room temperature. Excess antibody was removed with three 15-minute washes with Western Buffer A on a shaking platform at room temperature.</p>
    <p>Ten microliters biotin-conjugated horse anti-mouse antibody (Vector Laboratories, Burlingame, Calif.) in 20 ml Western Buffer A was added to the filters. The filters were re-incubated for one hour at room temperature on a platform shaker, and unbound conjugated antibody was removed with three fifteen-minute washes with Western Buffer A.</p>
    <p>The filters were pre-incubated for one hour at room temperature with a solution comprising 50 μl Vectastain Reagent A (Vector Laboratories) in 10 ml of Western Buffer A that had been allowed to incubate at room temperature for 30 minutes before use. The filters were washed with one quick wash with distilled water followed by three 15-minute washes with Western Buffer B (Table 4) at room temperature. The Western Buffer B washes were followed by one wash with distilled water.</p>
    <p>During the preceding wash step, the substrate reagent was prepared. Sixty mg of horseradish peroxidase reagent (Bio-Rad, Richmond, Calif.) was dissolved in 20 ml HPLC grade methanol. Ninety milliliters of distilled water was added to the dissolved peroxidase followed by 2.5 ml 2 M Tris, pH 7.4 and 3.8 ml 4 M NaCl. One hundred microliters of 30% H<sub>2</sub> O<sub>2</sub> was added just before use. The washed filters were incubated with 75 ml of substrate and incubated at room temperature for 10 minutes with vigorous shaking. After the 10 minute incubation, the buffer was changed, and the filters were incubated for an additional 10 minutes. The filters were then washed in distilled water for one hour at room temperature. Positives were scored as those samples which exhibited coloration.</p>
    <p>E. Processing the Filters For Visualization with an Anti-Substance P Antibody</p>
    <p>Filters prepared as described above were probed with an anti-substance P antibody. The filters were removed from the Western Buffer A and rinsed with Western transfer buffer, followed by a 5-minute wash in phosphate buffered saline (PBS, Sigma, St. Louis, Mo.). The filters were incubated with a 10 ml solution containing 0.5 M 1-ethyl-3-3-dimethylamino propyl carbodiimide (Sigma) in 1.0 M NH<sub>4</sub> Cl for 40 minutes at room temperature. After incubation, the filters were washed three times, for 5 minutes per wash, in PBS. The filters were blocked with 2% powdered milk diluted in PBS.</p>
    <p>The filters were then incubated with a rat anti-substance P monoclonal antibody (Accurate Chemical &amp; Scientific Corp., Westbury, N.Y.). Ten microliters of the antibody was diluted in 10 ml of antibody solution (PBS containing 20% fetal calf serum and 0.5% Tween-20). The filters were incubated at room temperature for 1 hour. Unbound antibody was removed with four 5-minute washes with PBS.</p>
    <p>The filters were then incubated with a biotin-conjugated rabbit anti-rat peroxidase antibody (Cappel Laboratories, Melvern, Pa.). The conjugated antibody was diluted 1:1000 in 10 ml of antibody solution for 2 hours at room temperature. Excess conjugated antibody was removed with four 5-minute washes with PBS.</p>
    <p>The filters were pre-incubated for 30 minutes at room temperature with a solution containing 50 μl Vectastain Reagent A (Vector Laboratories) and 50 μl Vectastain Reagent B (Vector Laboratories) in 10 ml of antibody solution that had been allowed to incubate for 30 minutes before use. Excess Vectastain reagents were removed by four 5-minute washes with PBS.</p>
    <p>During the preceding wash step, the substrate reagent was prepared. Sixty milligrams of horseradish peroxidase reagent (Bio-Rad Laboratories, Richmond, Calif.) was dissolved in 25 ml of HPLC grade methanol. Approximately 100 ml of PBS and 200 μl H<sub>2</sub> O<sub>2</sub> were added just before use. The filters were incubated with the substrate reagent for 10 to 20 minutes. The substrate was removed by a vigorous washing distilled water.</p>
    <p>F. Iodination of PDGF-BB.</p>
    <p>A PDGF-BB mutant molecule having a tyrosine replaing the phenylalanine at position 23 (PDGF-BB<sub>Tyr</sub>) was iodinated and subsequently purified, using a purification method which produces 125I-labeled PDGF-BB with a higher specific activity than primary-labeled material and which was found to substantially derease the nonspecific binding component. The PDGF-BB<sub>Tyr</sub> was labeled using the Iodobead method (Pierce Chemical). The labeled protein was gel filtered over a C-25 desalting column (Pharmacia LKB Technologies) equilibrated with 10 mM acetic acid, 0.25% gelatin and 100 mM NaCl. The peak fractions were pooled and pH adjusted to 7.2 by the addition of Tris-base. The labeled mixture was chromatographed over an affinity column composed of PDGF β-receptor analog protein coupled to CnBr-activated Sepharose (Pharmacia LKB Technologies, Inc.). The column was washed with phosphate buffered saline and eluted with 0.1 M citrate, pH 2.5 containing 0.25% gelatin. The peak eluate fractions were pooled and assayed by ELISA to determine the PDGF-BB concentration.</p>
    <p>Although the foregoing invention has been described in some detail by way of illustration and example for purposes of clarity of understanding, it will be evident that certain changes and modifications may be practiced within the scope of the appended claims.</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">__________________________________________________________________________#             SEQUENCE LISTING   - -  - - (1) GENERAL INFORMATION:   - -    (iii) NUMBER OF SEQUENCES: 36   - -  - - (2) INFORMATION FOR SEQ ID NO:1:   - -      (i) SEQUENCE CHARACTERISTICS:       (A) LENGTH: 4465 base - #pairs       (B) TYPE: nucleic acid       (C) STRANDEDNESS: double       (D) TOPOLOGY: linear   - -     (ii) MOLECULE TYPE: cDNA   - -    (iii) HYPOTHETICAL: N   - -     (iv) ANTI-SENSE: N   - -     (vi) ORIGINAL SOURCE:       (A) ORGANISM: Homo sapi - #ens       (D) DEVELOPMENTAL STAGE: - #Adult       (F) TISSUE TYPE: Skin       (G) CELL TYPE: fibrobla - #sts   - -    (vii) IMMEDIATE SOURCE:       (B) CLONE: pR-rX1   - -     (ix) FEATURE:       (A) NAME/KEY: CDS       (B) LOCATION: 354..3671       (D) OTHER INFORMATION:   - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:1:   - - CCCTCAGCCC TGCTGCCCAG CACGAGCCTG TGCTCGCCCT GCCCAACGCA GA -#CAGCCAGA     60   - - CCCAGGGCGG CCCCTCTGGC GGCTCTGCTC CTCCCGAAGG ATGCTTGGGG AG -#TGAGGCGA    120   - - AGCTGGGCGC TCCTCTCCCC TACAGCAGCC CCCTTCCTCC ATCCCTCTGT TC -#TCCTGAGC    180   - - CTTCAGGAGC CTGCACCAGT CCTGCCTGTC CTTCTACTCA GCTGTTACCC AC -#TCTGGGAC    240   - - CAGCAGTCTT TCTGATAACT GGGAGAGGGC AGTAAGGAGG ACTTCCTGGA GG -#GGGTGACT    300   - - GTCCAGAGCC TGGAACTGTG CCCACACCAG AAGCCATCAG CAGCAAGGAC AC - #C ATG 356              - #                  - #                  - #     Met              - #                  - #                  - #       1  - - CGG CTT CCG GGT GCG ATG CCA GCT CTG GCC CT - #C AAA GGC GAG CTG CTG404 Arg Leu Pro Gly Ala Met Pro Ala Leu Ala Le - #u Lys Gly Glu Leu Leu          5    - #              10    - #              15  - - TTG CTG TCT CTC CTG TTA CTT CTG GAA CCA CA - #G ATC TCT CAG GGC CTG452 Leu Leu Ser Leu Leu Leu Leu Leu Glu Pro Gl - #n Ile Ser Gln Gly Leu     20         - #         25         - #         30  - - GTC GTC ACA CCC CCG GGG CCA GAG CTT GTC CT - #C AAT GTC TCC AGC ACC500 Val Val Thr Pro Pro Gly Pro Glu Leu Val Le - #u Asn Val Ser Ser Thr 35             - #     40             - #     45  - - TTC GTT CTG ACC TGC TCG GGT TCA GCT CCG GT - #G GTG TGG GAA CGG ATG548 Phe Val Leu Thr Cys Ser Gly Ser Ala Pro Va - #l Val Trp Glu Arg Met  50                 - # 55                 - # 60                 - # 65  - - TCC CAG GAG CCC CCA CAG GAA ATG GCC AAG GC - #C CAG GAT GGC ACC TTC596 Ser Gln Glu Pro Pro Gln Glu Met Ala Lys Al - #a Gln Asp Gly Thr Phe             70 - #                 75 - #                 80  - - TCC AGC GTG CTC ACA CTG ACC AAC CTC ACT GG - #G CTA GAC ACG GGA GAA644 Ser Ser Val Leu Thr Leu Thr Asn Leu Thr Gl - #y Leu Asp Thr Gly Glu         85     - #             90     - #             95  - - TAC TTT TGC ACC CAC AAT GAC TCC CGT GGA CT - #G GAG ACC GAT GAG CGG692 Tyr Phe Cys Thr His Asn Asp Ser Arg Gly Le - #u Glu Thr Asp Glu Arg    100          - #       105          - #       110  - - AAA CGG CTC TAC ATC TTT GTG CCA GAT CCC AC - #C GTG GGC TTC CTC CCT740 Lys Arg Leu Tyr Ile Phe Val Pro Asp Pro Th - #r Val Gly Phe Leu Pro115              - #   120              - #   125  - - AAT GAT GCC GAG GAA CTA TTC ATC TTT CTC AC - #G GAA ATA ACT GAG ATC788 Asn Asp Ala Glu Glu Leu Phe Ile Phe Leu Th - #r Glu Ile Thr Glu Ile 130                 1 - #35                 1 - #40                 1 -#45   - - ACC ATT CCA TGC CGA GTA ACA GAC CCA CAG CT - #G GTG GTG ACA CTGCAC      836  Thr Ile Pro Cys Arg Val Thr Asp Pro Gln Le - #u Val Val Thr Leu His            150  - #               155  - #               160  - - GAG AAG AAA GGG GAC GTT GCA CTG CCT GTC CC - #C TAT GAT CAC CAA CGT884 Glu Lys Lys Gly Asp Val Ala Leu Pro Val Pr - #o Tyr Asp His Gln Arg        165      - #           170      - #           175  - - GGC TTT TCT GGT ATC TTT GAG GAC AGA AGC TA - #C ATC TGC AAA ACC ACC932 Gly Phe Ser Gly Ile Phe Glu Asp Arg Ser Ty - #r Ile Cys Lys Thr Thr    180          - #       185          - #       190  - - ATT GGG GAC AGG GAG GTG GAT TCT GAT GCC TA - #C TAT GTC TAC AGA CTC980 Ile Gly Asp Arg Glu Val Asp Ser Asp Ala Ty - #r Tyr Val Tyr Arg Leu195              - #   200              - #   205  - - CAG GTG TCA TCC ATC AAC GTC TCT GTG AAC GC - #A GTG CAG ACT GTG GTC    1028 Gln Val Ser Ser Ile Asn Val Ser Val Asn Al - #a Val Gln Thr Val Val 210                 2 - #15                 2 - #20                 2 -#25   - - CGC CAG GGT GAG AAC ATC ACC CTC ATG TGC AT - #T GTG ATC GGG AATGAG     1076  Arg Gln Gly Glu Asn Ile Thr Leu Met Cys Il - #e Val Ile Gly Asn Glu            230  - #               235  - #               240  - - GTG GTC AAC TTC GAG TGG ACA TAC CCC CGC AA - #A GAA AGT GGG CGG CTG    1124 Val Val Asn Phe Glu Trp Thr Tyr Pro Arg Ly - #s Glu Ser Gly Arg Leu        245      - #           250      - #           255  - - GTG GAG CCG GTG ACT GAC TTC CTC TTG GAT AT - #G CCT TAC CAC ATC CGC    1172 Val Glu Pro Val Thr Asp Phe Leu Leu Asp Me - #t Pro Tyr His Ile Arg    260          - #       265          - #       270  - - TCC ATC CTG CAC ATC CCC AGT GCC GAG TTA GA - #A GAC TCG GGG ACC TAC    1220 Ser Ile Leu His Ile Pro Ser Ala Glu Leu Gl - #u Asp Ser Gly Thr Tyr275              - #   280              - #   285  - - ACC TGC AAT GTG ACG GAG AGT GTG AAT GAC CA - #T CAG GAT GAA AAG GCC    1268 Thr Cys Asn Val Thr Glu Ser Val Asn Asp Hi - #s Gln Asp Glu Lys Ala 290                 2 - #95                 3 - #00                 3 -#05   - - ATC AAC ATC ACC GTG GTT GAG AGC GGC TAC GT - #G CGG CTC CTG GGAGAG     1316  Ile Asn Ile Thr Val Val Glu Ser Gly Tyr Va - #l Arg Leu Leu Gly Glu            310  - #               315  - #               320  - - GTG GGC ACA CTA CAA TTT GCT GAG CTG CAT CG - #G AGC CGG ACA CTG CAG    1364 Val Gly Thr Leu Gln Phe Ala Glu Leu His Ar - #g Ser Arg Thr Leu Gln        325      - #           330      - #           335  - - GTA GTG TTC GAG GCC TAC CCA CCG CCC ACT GT - #C CTG TGG TTC AAA GAC    1412 Val Val Phe Glu Ala Tyr Pro Pro Pro Thr Va - #l Leu Trp Phe Lys Asp    340          - #       345          - #       350  - - AAC CGC ACC CTG GGC GAC TCC AGC GCT GGC GA - #A ATC GCC CTG TCC ACG    1460 Asn Arg Thr Leu Gly Asp Ser Ser Ala Gly Gl - #u Ile Ala Leu Ser Thr355              - #   360              - #   365  - - CGC AAC GTG TCG GAG ACC CGG TAT GTG TCA GA - #G CTG ACA CTG GTT CGC    1508 Arg Asn Val Ser Glu Thr Arg Tyr Val Ser Gl - #u Leu Thr Leu Val Arg 370                 3 - #75                 3 - #80                 3 -#85   - - GTG AAG GTG GCA GAG GCT GGC CAC TAC ACC AT - #G CGG GCC TTC CATGAG     1556  Val Lys Val Ala Glu Ala Gly His Tyr Thr Me - #t Arg Ala Phe His Glu            390  - #               395  - #               400  - - GAT GCT GAG GTC CAG CTC TCC TTC CAG CTA CA - #G ATC AAT GTC CCT GTC    1604 Asp Ala Glu Val Gln Leu Ser Phe Gln Leu Gl - #n Ile Asn Val Pro Val        405      - #           410      - #           415  - - CGA GTG CTG GAG CTA AGT GAG AGC CAC CCT GA - #C AGT GGG GAA CAG ACA    1652 Arg Val Leu Glu Leu Ser Glu Ser His Pro As - #p Ser Gly Glu Gln Thr    420          - #       425          - #       430  - - GTC CGC TGT CGT GGC CGG GGC ATG CCC CAG CC - #G AAC ATC ATC TGG TCT    1700 Val Arg Cys Arg Gly Arg Gly Met Pro Gln Pr - #o Asn Ile Ile Trp Ser435              - #   440              - #   445  - - GCC TGC AGA GAC CTC AAA AGG TGT CCA CGT GA - #G CTG CCG CCC ACG CTG    1748 Ala Cys Arg Asp Leu Lys Arg Cys Pro Arg Gl - #u Leu Pro Pro Thr Leu 450                 4 - #55                 4 - #60                 4 -#65   - - CTG GGG AAC AGT TCC GAA GAG GAG AGC CAG CT - #G GAG ACT AAC GTGACG     1796  Leu Gly Asn Ser Ser Glu Glu Glu Ser Gln Le - #u Glu Thr Asn Val Thr            470  - #               475  - #               480  - - TAC TGG GAG GAG GAG CAG GAG TTT GAG GTG GT - #G AGC ACA CTG CGT CTG    1844 Tyr Trp Glu Glu Glu Gln Glu Phe Glu Val Va - #l Ser Thr Leu Arg Leu        485      - #           490      - #           495  - - CAG CAC GTG GAT CGG CCA CTG TCG GTG CGC TG - #C ACG CTG CGC AAC GCT    1892 Gln His Val Asp Arg Pro Leu Ser Val Arg Cy - #s Thr Leu Arg Asn Ala    500          - #       505          - #       510  - - GTG GGC CAG GAC ACG CAG GAG GTC ATC GTG GT - #G CCA CAC TCC TTG CCC    1940 Val Gly Gln Asp Thr Gln Glu Val Ile Val Va - #l Pro His Ser Leu Pro515              - #   520              - #   525  - - TTT AAG GTG GTG GTG ATC TCA GCC ATC CTG GC - #C CTG GTG GTG CTC ACC    1988 Phe Lys Val Val Val Ile Ser Ala Ile Leu Al - #a Leu Val Val Leu Thr 530                 5 - #35                 5 - #40                 5 -#45   - - ATC ATC TCC CTT ATC ATC CTC ATC ATG CTT TG - #G CAG AAG AAG CCACGT     2036  Ile Ile Ser Leu Ile Ile Leu Ile Met Leu Tr - #p Gln Lys Lys Pro Arg            550  - #               555  - #               560  - - TAC GAG ATC CGA TGG AAG GTG ATT GAG TCT GT - #G AGC TCT GAC GGC CAT    2084 Tyr Glu Ile Arg Trp Lys Val Ile Glu Ser Va - #l Ser Ser Asp Gly His        565      - #           570      - #           575  - - GAG TAC ATC TAC GTG GAC CCC ATG CAG CTG CC - #C TAT GAC TCC ACG TGG    2132 Glu Tyr Ile Tyr Val Asp Pro Met Gln Leu Pr - #o Tyr Asp Ser Thr Trp    580          - #       585          - #       590  - - GAG CTG CCG CGG GAC CAG CTT GTG CTG GGA CG - #C ACC CTC GGC TCT GGG    2180 Glu Leu Pro Arg Asp Gln Leu Val Leu Gly Ar - #g Thr Leu Gly Ser Gly595              - #   600              - #   605  - - GCC TTT GGG CAG GTG GTG GAG GCC ACG GCT CA - #T GGC CTG AGC CAT TCT    2228 Ala Phe Gly Gln Val Val Glu Ala Thr Ala Hi - #s Gly Leu Ser His Ser 610                 6 - #15                 6 - #20                 6 -#25   - - CAG GCC ACG ATG AAA GTG GCC GTC AAG ATG CT - #T AAA TCC ACA GCCCGC     2276  Gln Ala Thr Met Lys Val Ala Val Lys Met Le - #u Lys Ser Thr Ala Arg            630  - #               635  - #               640  - - AGC AGT GAG AAG CAA GCC CTT ATG TCG GAG CT - #G AAG ATC ATG AGT CAC    2324 Ser Ser Glu Lys Gln Ala Leu Met Ser Glu Le - #u Lys Ile Met Ser His        645      - #           650      - #           655  - - CTT GGG CCC CAC CTG AAC GTG GTC AAC CTG TT - #G GGG GCC TGC ACC AAA    2372 Leu Gly Pro His Leu Asn Val Val Asn Leu Le - #u Gly Ala Cys Thr Lys    660          - #       665          - #       670  - - GGA GGA CCC ATC TAT ATC ATC ACT GAG TAC TG - #C CGC TAC GGA GAC CTG    2420 Gly Gly Pro Ile Tyr Ile Ile Thr Glu Tyr Cy - #s Arg Tyr Gly Asp Leu675              - #   680              - #   685  - - GTG GAC TAC CTG CAC CGC AAC AAA CAC ACC TT - #C CTG CAG CAC CAC TCC    2468 Val Asp Tyr Leu His Arg Asn Lys His Thr Ph - #e Leu Gln His His Ser 690                 6 - #95                 7 - #00                 7 -#05   - - GAC AAG CGC CGC CCG CCC AGC GCG GAG CTC TA - #C AGC AAT GCT CTGCCC     2516  Asp Lys Arg Arg Pro Pro Ser Ala Glu Leu Ty - #r Ser Asn Ala Leu Pro            710  - #               715  - #               720  - - GTT GGG CTC CCC CTG CCC AGC CAT GTG TCC TT - #G ACC GGG GAG AGC GAC    2564 Val Gly Leu Pro Leu Pro Ser His Val Ser Le - #u Thr Gly Glu Ser Asp        725      - #           730      - #           735  - - GGT GGC TAC ATG GAC ATG AGC AAG GAC GAG TC - #G GTG GAC TAT GTG CCC    2612 Gly Gly Tyr Met Asp Met Ser Lys Asp Glu Se - #r Val Asp Tyr Val Pro    740          - #       745          - #       750  - - ATG CTG GAC ATG AAA GGA GAC GTC AAA TAT GC - #A GAC ATC GAG TCC TCC    2660 Met Leu Asp Met Lys Gly Asp Val Lys Tyr Al - #a Asp Ile Glu Ser Ser755              - #   760              - #   765  - - AAC TAC ATG GCC CCT TAC GAT AAC TAC GTT CC - #C TCT GCC CCT GAG AGG    2708 Asn Tyr Met Ala Pro Tyr Asp Asn Tyr Val Pr - #o Ser Ala Pro Glu Arg 770                 7 - #75                 7 - #80                 7 -#85   - - ACC TGC CGA GCA ACT TTG ATC AAC GAG TCT CC - #A GTG CTA AGC TACATG     2756  Thr Cys Arg Ala Thr Leu Ile Asn Glu Ser Pr - #o Val Leu Ser Tyr Met            790  - #               795  - #               800  - - GAC CTC GTG GGC TTC AGC TAC CAG GTG GCC AA - #T GGC ATG GAG TTT CTG    2804 Asp Leu Val Gly Phe Ser Tyr Gln Val Ala As - #n Gly Met Glu Phe Leu        805      - #           810      - #           815  - - GCC TCC AAG AAC TGC GTC CAC AGA GAC CTG GC - #G GCT AGG AAC GTG CTC    2852 Ala Ser Lys Asn Cys Val His Arg Asp Leu Al - #a Ala Arg Asn Val Leu    820          - #       825          - #       830  - - ATC TGT GAA GGC AAG CTG GTC AAG ATC TGT GA - #C TTT GGC CTG GCT CGA    2900 Ile Cys Glu Gly Lys Leu Val Lys Ile Cys As - #p Phe Gly Leu Ala Arg835              - #   840              - #   845  - - GAC ATC ATG CGG GAC TCG AAT TAC ATC TCC AA - #A GGC AGC ACC TTT TTG    2948 Asp Ile Met Arg Asp Ser Asn Tyr Ile Ser Ly - #s Gly Ser Thr Phe Leu 850                 8 - #55                 8 - #60                 8 -#65   - - CCT TTA AAG TGG ATG GCT CCG GAG AGC ATC TT - #C AAC AGC CTC TACACC     2996  Pro Leu Lys Trp Met Ala Pro Glu Ser Ile Ph - #e Asn Ser Leu Tyr Thr            870  - #               875  - #               880  - - ACC CTG AGC GAC GTG TGG TCC TTC GGG ATC CT - #G CTC TGG GAG ATC TTC    3044 Thr Leu Ser Asp Val Trp Ser Phe Gly Ile Le - #u Leu Trp Glu Ile Phe        885      - #           890      - #           895  - - ACC TTG GGT GGC ACC CCT TAC CCA GAG CTG CC - #C ATG AAC GAG CAG TTC    3092 Thr Leu Gly Gly Thr Pro Tyr Pro Glu Leu Pr - #o Met Asn Glu Gln Phe    900          - #       905          - #       910  - - TAC AAT GCC ATC AAA CGG GGT TAC CGC ATG GC - #C CAG CCT GCC CAT GCC    3140 Tyr Asn Ala Ile Lys Arg Gly Tyr Arg Met Al - #a Gln Pro Ala His Ala915              - #   920              - #   925  - - TCC GAC GAG ATC TAT GAG ATC ATG CAG AAG TG - #C TGG GAA GAG AAG TTT    3188 Ser Asp Glu Ile Tyr Glu Ile Met Gln Lys Cy - #s Trp Glu Glu Lys Phe 930                 9 - #35                 9 - #40                 9 -#45   - - GAG ATT CGG CCC CCC TTC TCC CAG CTG GTG CT - #G CTT CTC GAG AGACTG     3236  Glu Ile Arg Pro Pro Phe Ser Gln Leu Val Le - #u Leu Leu Glu Arg Leu            950  - #               955  - #               960  - - TTG GGC GAA GGT TAC AAA AAG AAG TAC CAG CA - #G GTG GAT GAG GAG TTT    3284 Leu Gly Glu Gly Tyr Lys Lys Lys Tyr Gln Gl - #n Val Asp Glu Glu Phe        965      - #           970      - #           975  - - CTG AGG AGT GAC CAC CCA GCC ATC CTT CGG TC - #C CAG GCC CGC TTG CCT    3332 Leu Arg Ser Asp His Pro Ala Ile Leu Arg Se - #r Gln Ala Arg Leu Pro    980          - #       985          - #       990  - - GGG TTC CAT GGC CTC CGA TCT CCC CTG GAC AC - #C AGC TCC GTC CTC TAT    3380 Gly Phe His Gly Leu Arg Ser Pro Leu Asp Th - #r Ser Ser Val Leu Tyr995              - #   1000              - #  1005  - - ACT GCC GTG CAG CCC AAT GAG GGT GAC AAC GA - #C TAT ATC ATC CCC CTG    3428 Thr Ala Val Gln Pro Asn Glu Gly Asp Asn As - #p Tyr Ile Ile Pro Leu 1010                1015 - #                1020 - #               1025  - - CCT GAC CCC AAA CCC GAG GTT GCT GAC GAG GG - #C CCA CTG GAG GGT TCC    3476 Pro Asp Pro Lys Pro Glu Val Ala Asp Glu Gl - #y Pro Leu Glu Gly Ser            1030 - #               1035  - #              1040  - - CCC AGC CTA GCC AGC TCC ACC CTG AAT GAA GT - #C AAC ACC TCC TCA ACC    3524 Pro Ser Leu Ala Ser Ser Thr Leu Asn Glu Va - #l Asn Thr Ser Ser Thr        1045     - #           1050      - #          1055  - - ATC TCC TGT GAC AGC CCC CTG GAG CCC CAG GA - #C GAA CCA GAG CCA GAG    3572 Ile Ser Cys Asp Ser Pro Leu Glu Pro Gln As - #p Glu Pro Glu Pro Glu    1060         - #       1065          - #      1070  - - CCC CAG CTT GAG CTC CAG GTG GAG CCG GAG CC - #A GAG CTG GAA CAG TTG    3620 Pro Gln Leu Glu Leu Gln Val Glu Pro Glu Pr - #o Glu Leu Glu Gln Leu1075             - #   1080              - #  1085  - - CCG GAT TCG GGG TGC CCT GCG CCT CGG GCG GA - #A GCA GAG GAT AGC TTC    3668 Pro Asp Ser Gly Cys Pro Ala Pro Arg Ala Gl - #u Ala Glu Asp Ser Phe 1090                1095 - #                1100 - #               1105  - - CTG TAGGGGGCTG GCCCCTACCC TGCCCTGCCT GAAGCTCCCC CCCTGCCAG - #C    3721 Leu  - - ACCCAGCATC TCCTGGCCTG GCCTGACCGG GCTTCCTGTC AGCCAGGCTG CC -#CTTATCAG   3781   - - CTGTCCCCTT CTGGAAGCTT TCTGCTCCTG ACGTGTTGTG CCCCAAACCC TG -#GGGCTGGC   3841   - - TTAGGAGGCA AGAAAACTGC AGGGGCCGTG ACCAGCCCTC TGCCTCCAGG GA -#GGCCAACT   3901   - - GACTCTGAGC CAGGGTTCCC CCAGGGAACT CAGTTTTCCC ATATGTAAGA TG -#GGAAAGTT   3961   - - AGGCTTGATG ACCCAGAATC TAGGATTCTC TCCCTGGCTG ACACGGTGGG GA -#GACCGAAT   4021   - - CCCTCCCTGG GAAGATTCTT GGAGTTACTG AGGTGGTAAA TTAACATTTT TT -#CTGTTCAG   4081   - - CCAGCTACCC CTCAAGGAAT CATAGCTCTC TCCTCGCACT TTTTATCCAC CC -#AGGAGCTA   4141   - - GGGAAGAGAC CCTAGCCTCC CTGGCTGCTG GCTGAGCTAG GGCCTAGCTT GA -#GCAGTGTT   4201   - - GCCTCATCCA GAAGAAAGCC AGTCTCCTCC CTATGATGCC AGTCCCTGCG TT -#CCCTGGCC   4261   - - CGAGCTGGTC TGGGGCCATT AGGCAGCCTA ATTAATGCTG GAGGCTGAGC CA -#AGTACAGG   4321   - - ACACCCCCAG CCTGCAGCCC TTGCCCAGGG CACTTGGAGC ACACGCAGCC AT -#AGCAAGTG   4381   - - CCTGTGTCCC TGTCCTTCAG GCCCATCAGT CCTGGGGCTT TTTCTTTATC AC -#CCTCAGTC   4441   - - TTAATCCATC CACCAGAGTC TAGA          - #                  - #  4465  - -  - - (2) INFORMATION FOR SEQ ID NO:2:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 1106 amino - #acids      (B) TYPE: amino acid      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: protein  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:2:  - - Met Arg Leu Pro Gly Ala Met Pro Ala Leu Al - #a Leu Lys Gly Glu Leu   1               5 - #                 10 - #                 15  - - Leu Leu Leu Ser Leu Leu Leu Leu Leu Glu Pr - #o Gln Ile Ser Gln Gly         20     - #             25     - #             30  - - Leu Val Val Thr Pro Pro Gly Pro Glu Leu Va - #l Leu Asn Val Ser Ser     35         - #         40         - #         45  - - Thr Phe Val Leu Thr Cys Ser Gly Ser Ala Pr - #o Val Val Trp Glu Arg 50             - #     55             - #     60  - - Met Ser Gln Glu Pro Pro Gln Glu Met Ala Ly - #s Ala Gln Asp Gly Thr  65                 - # 70                 - # 75                 - # 80  - - Phe Ser Ser Val Leu Thr Leu Thr Asn Leu Th - #r Gly Leu Asp Thr Gly             85 - #                 90 - #                 95  - - Glu Tyr Phe Cys Thr His Asn Asp Ser Arg Gl - #y Leu Glu Thr Asp Glu        100      - #           105      - #           110  - - Arg Lys Arg Leu Tyr Ile Phe Val Pro Asp Pr - #o Thr Val Gly Phe Leu    115          - #       120          - #       125  - - Pro Asn Asp Ala Glu Glu Leu Phe Ile Phe Le - #u Thr Glu Ile Thr Glu130              - #   135              - #   140  - - Ile Thr Ile Pro Cys Arg Val Thr Asp Pro Gl - #n Leu Val Val Thr Leu 145                 1 - #50                 1 - #55                 1 -#60   - - His Glu Lys Lys Gly Asp Val Ala Leu Pro Va - #l Pro Tyr Asp HisGln             165  - #               170  - #               175  - - Arg Gly Phe Ser Gly Ile Phe Glu Asp Arg Se - #r Tyr Ile Cys Lys Thr        180      - #           185      - #           190  - - Thr Ile Gly Asp Arg Glu Val Asp Ser Asp Al - #a Tyr Tyr Val Tyr Arg    195          - #       200          - #       205  - - Leu Gln Val Ser Ser Ile Asn Val Ser Val As - #n Ala Val Gln Thr Val210              - #   215              - #   220  - - Val Arg Gln Gly Glu Asn Ile Thr Leu Met Cy - #s Ile Val Ile Gly Asn 225                 2 - #30                 2 - #35                 2 -#40   - - Glu Val Val Asn Phe Glu Trp Thr Tyr Pro Ar - #g Lys Glu Ser GlyArg             245  - #               250  - #               255  - - Leu Val Glu Pro Val Thr Asp Phe Leu Leu As - #p Met Pro Tyr His Ile        260      - #           265      - #           270  - - Arg Ser Ile Leu His Ile Pro Ser Ala Glu Le - #u Glu Asp Ser Gly Thr    275          - #       280          - #       285  - - Tyr Thr Cys Asn Val Thr Glu Ser Val Asn As - #p His Gln Asp Glu Lys290              - #   295              - #   300  - - Ala Ile Asn Ile Thr Val Val Glu Ser Gly Ty - #r Val Arg Leu Leu Gly 305                 3 - #10                 3 - #15                 3 -#20   - - Glu Val Gly Thr Leu Gln Phe Ala Glu Leu Hi - #s Arg Ser Arg ThrLeu             325  - #               330  - #               335  - - Gln Val Val Phe Glu Ala Tyr Pro Pro Pro Th - #r Val Leu Trp Phe Lys        340      - #           345      - #           350  - - Asp Asn Arg Thr Leu Gly Asp Ser Ser Ala Gl - #y Glu Ile Ala Leu Ser    355          - #       360          - #       365  - - Thr Arg Asn Val Ser Glu Thr Arg Tyr Val Se - #r Glu Leu Thr Leu Val370              - #   375              - #   380  - - Arg Val Lys Val Ala Glu Ala Gly His Tyr Th - #r Met Arg Ala Phe His 385                 3 - #90                 3 - #95                 4 -#00   - - Glu Asp Ala Glu Val Gln Leu Ser Phe Gln Le - #u Gln Ile Asn ValPro             405  - #               410  - #               415  - - Val Arg Val Leu Glu Leu Ser Glu Ser His Pr - #o Asp Ser Gly Glu Gln        420      - #           425      - #           430  - - Thr Val Arg Cys Arg Gly Arg Gly Met Pro Gl - #n Pro Asn Ile Ile Trp    435          - #       440          - #       445  - - Ser Ala Cys Arg Asp Leu Lys Arg Cys Pro Ar - #g Glu Leu Pro Pro Thr450              - #   455              - #   460  - - Leu Leu Gly Asn Ser Ser Glu Glu Glu Ser Gl - #n Leu Glu Thr Asn Val 465                 4 - #70                 4 - #75                 4 -#80   - - Thr Tyr Trp Glu Glu Glu Gln Glu Phe Glu Va - #l Val Ser Thr LeuArg             485  - #               490  - #               495  - - Leu Gln His Val Asp Arg Pro Leu Ser Val Ar - #g Cys Thr Leu Arg Asn        500      - #           505      - #           510  - - Ala Val Gly Gln Asp Thr Gln Glu Val Ile Va - #l Val Pro His Ser Leu    515          - #       520          - #       525  - - Pro Phe Lys Val Val Val Ile Ser Ala Ile Le - #u Ala Leu Val Val Leu530              - #   535              - #   540  - - Thr Ile Ile Ser Leu Ile Ile Leu Ile Met Le - #u Trp Gln Lys Lys Pro 545                 5 - #50                 5 - #55                 5 -#60   - - Arg Tyr Glu Ile Arg Trp Lys Val Ile Glu Se - #r Val Ser Ser AspGly             565  - #               570  - #               575  - - His Glu Tyr Ile Tyr Val Asp Pro Met Gln Le - #u Pro Tyr Asp Ser Thr        580      - #           585      - #           590  - - Trp Glu Leu Pro Arg Asp Gln Leu Val Leu Gl - #y Arg Thr Leu Gly Ser    595          - #       600          - #       605  - - Gly Ala Phe Gly Gln Val Val Glu Ala Thr Al - #a His Gly Leu Ser His610              - #   615              - #   620  - - Ser Gln Ala Thr Met Lys Val Ala Val Lys Me - #t Leu Lys Ser Thr Ala 625                 6 - #30                 6 - #35                 6 -#40   - - Arg Ser Ser Glu Lys Gln Ala Leu Met Ser Gl - #u Leu Lys Ile MetSer             645  - #               650  - #               655  - - His Leu Gly Pro His Leu Asn Val Val Asn Le - #u Leu Gly Ala Cys Thr        660      - #           665      - #           670  - - Lys Gly Gly Pro Ile Tyr Ile Ile Thr Glu Ty - #r Cys Arg Tyr Gly Asp    675          - #       680          - #       685  - - Leu Val Asp Tyr Leu His Arg Asn Lys His Th - #r Phe Leu Gln His His690              - #   695              - #   700  - - Ser Asp Lys Arg Arg Pro Pro Ser Ala Glu Le - #u Tyr Ser Asn Ala Leu 705                 7 - #10                 7 - #15                 7 -#20   - - Pro Val Gly Leu Pro Leu Pro Ser His Val Se - #r Leu Thr Gly GluSer             725  - #               730  - #               735  - - Asp Gly Gly Tyr Met Asp Met Ser Lys Asp Gl - #u Ser Val Asp Tyr Val        740      - #           745      - #           750  - - Pro Met Leu Asp Met Lys Gly Asp Val Lys Ty - #r Ala Asp Ile Glu Ser    755          - #       760          - #       765  - - Ser Asn Tyr Met Ala Pro Tyr Asp Asn Tyr Va - #l Pro Ser Ala Pro Glu770              - #   775              - #   780  - - Arg Thr Cys Arg Ala Thr Leu Ile Asn Glu Se - #r Pro Val Leu Ser Tyr 785                 7 - #90                 7 - #95                 8 -#00   - - Met Asp Leu Val Gly Phe Ser Tyr Gln Val Al - #a Asn Gly Met GluPhe             805  - #               810  - #               815  - - Leu Ala Ser Lys Asn Cys Val His Arg Asp Le - #u Ala Ala Arg Asn Val        820      - #           825      - #           830  - - Leu Ile Cys Glu Gly Lys Leu Val Lys Ile Cy - #s Asp Phe Gly Leu Ala    835          - #       840          - #       845  - - Arg Asp Ile Met Arg Asp Ser Asn Tyr Ile Se - #r Lys Gly Ser Thr Phe850              - #   855              - #   860  - - Leu Pro Leu Lys Trp Met Ala Pro Glu Ser Il - #e Phe Asn Ser Leu Tyr 865                 8 - #70                 8 - #75                 8 -#80   - - Thr Thr Leu Ser Asp Val Trp Ser Phe Gly Il - #e Leu Leu Trp GluIle             885  - #               890  - #               895  - - Phe Thr Leu Gly Gly Thr Pro Tyr Pro Glu Le - #u Pro Met Asn Glu Gln        900      - #           905      - #           910  - - Phe Tyr Asn Ala Ile Lys Arg Gly Tyr Arg Me - #t Ala Gln Pro Ala His    915          - #       920          - #       925  - - Ala Ser Asp Glu Ile Tyr Glu Ile Met Gln Ly - #s Cys Trp Glu Glu Lys930              - #   935              - #   940  - - Phe Glu Ile Arg Pro Pro Phe Ser Gln Leu Va - #l Leu Leu Leu Glu Arg 945                 9 - #50                 9 - #55                 9 -#60   - - Leu Leu Gly Glu Gly Tyr Lys Lys Lys Tyr Gl - #n Gln Val Asp GluGlu             965  - #               970  - #               975  - - Phe Leu Arg Ser Asp His Pro Ala Ile Leu Ar - #g Ser Gln Ala Arg Leu        980      - #           985      - #           990  - - Pro Gly Phe His Gly Leu Arg Ser Pro Leu As - #p Thr Ser Ser Val Leu    995          - #       1000          - #      1005  - - Tyr Thr Ala Val Gln Pro Asn Glu Gly Asp As - #n Asp Tyr Ile Ile Pro1010             - #   1015              - #  1020  - - Leu Pro Asp Pro Lys Pro Glu Val Ala Asp Gl - #u Gly Pro Leu Glu Gly 1025                1030 - #                1035 - #               1040  - - Ser Pro Ser Leu Ala Ser Ser Thr Leu Asn Gl - #u Val Asn Thr Ser Ser            1045 - #               1050  - #              1055  - - Thr Ile Ser Cys Asp Ser Pro Leu Glu Pro Gl - #n Asp Glu Pro Glu Pro        1060     - #           1065      - #          1070  - - Glu Pro Gln Leu Glu Leu Gln Val Glu Pro Gl - #u Pro Glu Leu Glu Gln    1075         - #       1080          - #      1085  - - Leu Pro Asp Ser Gly Cys Pro Ala Pro Arg Al - #a Glu Ala Glu Asp Ser1090             - #   1095              - #  1100  - - Phe Leu 1105  - -  - - (2) INFORMATION FOR SEQ ID NO:3:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 57 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other nucleic acid  - -    (iii) HYPOTHETICAL: N  - -     (iv) ANTI-SENSE: N  - -    (vii) IMMEDIATE SOURCE:      (B) CLONE: ZC871  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:3:  - - ATTATACGCT CTCTTCCTCA GGTAAATGAG TGCCAGGGCC GGCAAGCCCC CG - #CTCCA 57  - -  - - (2) INFORMATION FOR SEQ ID NO:4:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 56 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other nucleic acid  - -    (iii) HYPOTHETICAL: N  - -     (iv) ANTI-SENSE: N  - -    (vii) IMMEDIATE SOURCE:      (B) CLONE: ZC872  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:4:  - - CCGGGGAGCG GGGGCTTGCC GGCCCTGGCA CTCATTTACC TGAGGAAGAG AG - #AGCT 56  - -  - - (2) INFORMATION FOR SEQ ID NO:5:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 45 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other nucleic acid  - -    (iii) HYPOTHETICAL: N  - -     (iv) ANTI-SENSE: N  - -    (vii) IMMEDIATE SOURCE:      (B) CLONE: ZC904  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:5:  - - CATGGGCACG TAATCTATAG ATTCATCCTT GCTCATATCC ATGTA   - #- #45  - -  - - (2) INFORMATION FOR SEQ ID NO:6:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 38 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other nucleic acid  - -    (iii) HYPOTHETICAL: N  - -     (iv) ANTI-SENSE: N  - -    (vii) IMMEDIATE SOURCE:      (B) CLONE: ZC906  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:6:  - - AAGCTGTCCT CTGCTTCAGC CAGAGGTCCT GGGCAGCC      - #- #     38  - -  - - (2) INFORMATION FOR SEQ ID NO:7:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 38 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other nucleic acid  - -    (iii) HYPOTHETICAL: N  - -     (iv) ANTI-SENSE: N  - -    (vii) IMMEDIATE SOURCE:      (B) CLONE: ZC906  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:7:  - - AAGCTGTCCT CTGCTTCAGC CAGAGGTCCT GGGCAGCC      - #- #     38  - -  - - (2) INFORMATION FOR SEQ ID NO:8:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 21 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other nucleic acid  - -    (iii) HYPOTHETICAL: N  - -     (iv) ANTI-SENSE: N  - -    (vii) IMMEDIATE SOURCE:      (B) CLONE: ZC1380  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:8:  - - CATGGTGGAA TTCCTGCTGA T           - #                  - #   - #21  - -  - - (2) INFORMATION FOR SEQ ID NO:9:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 29 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other nucleic acid  - -    (iii) HYPOTHETICAL: N  - -     (iv) ANTI-SENSE: N  - -    (vii) IMMEDIATE SOURCE:      (B) CLONE: ZC1447  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:9:  - - TGGTTGTGCA GAGCTGAGGA AGAGATGGA         - #                  - #   29  - -  - - (2) INFORMATION FOR SEQ ID NO:10:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 55 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other nucleic acid  - -    (iii) HYPOTHETICAL: N  - -     (iv) ANTI-SENSE: N  - -    (vii) IMMEDIATE SOURCE:      (B) CLONE: ZC1453  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:10:  - - AATTCATTAT GTTGTTGCAA GCCTTCTTGT TCCTGCTAGC TGGTTTCGCT GT - #TAA 55  - -  - - (2) INFORMATION FOR SEQ ID NO:11:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 55 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other nucleic acid  - -    (iii) HYPOTHETICAL: N  - -     (iv) ANTI-SENSE: N  - -    (vii) IMMEDIATE SOURCE:      (B) CLONE: ZC1454  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:11:  - - GATCTTAACA GCGAAACCAG CTAGCAGGAA CAAGAAGGCT TGCAACAACA TA - #ATG 55  - -  - - (2) INFORMATION FOR SEQ ID NO:12:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 21 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other nucleic acid  - -    (iii) HYPOTHETICAL: N  - -     (iv) ANTI-SENSE: N  - -    (vii) IMMEDIATE SOURCE:      (B) CLONE: ZC1478  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:12:  - - ATCGCGAGCA TGCAGATCTG A           - #                  - #   - #21  - -  - - (2) INFORMATION FOR SEQ ID NO:13:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 25 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other nucleic acid  - -    (iii) HYPOTHETICAL: N  - -     (iv) ANTI-SENSE: N  - -    (vii) IMMEDIATE SOURCE:      (B) CLONE: ZC1479  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:13:  - - AGCTTCAGAT CTGCATGCTG CCGAT          - #                  - #   25  - -  - - (2) INFORMATION FOR SEQ ID NO:14:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 52 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other nucleic acid  - -    (iii) HYPOTHETICAL: N  - -     (iv) ANTI-SENSE: N  - -    (vii) IMMEDIATE SOURCE:      (B) CLONE: ZC1776  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:14:  - - AGCTGAGCGC AAATGTTGTG TCGAGTGCCC ACCGTGCCCA GCTTAGAATT CT - # 52  - -  - - (2) INFORMATION FOR SEQ ID NO:15:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 52 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other nucleic acid  - -    (iii) HYPOTHETICAL: N  - -     (iv) ANTI-SENSE: N  - -    (vii) IMMEDIATE SOURCE:      (B) CLONE: ZC1777  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:15:  - - CTAGAGAATT CTAAGCTGGG CACGGTGGGC ACTCGACACA ACATTTGCGC TC - # 52  - -  - - (2) INFORMATION FOR SEQ ID NO:16:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 95 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other nucleic acid  - -    (iii) HYPOTHETICAL: N  - -     (iv) ANTI-SENSE: N  - -    (vii) IMMEDIATE SOURCE:      (B) CLONE: ZC1846  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:16:  - - GATCGGCCAC TGTCGGTGCG CTGCACGCTG CGCAACGCTG TGGGCCAGGA CA -#CGCAGGAG     60   - - GTCATCGTGG TGCCACACTC CTTGCCCTTT AAGCA       - #- #       95   - -  - - (2) INFORMATION FOR SEQ ID NO:17:   - -      (i) SEQUENCE CHARACTERISTICS:       (A) LENGTH: 95 base - #pairs       (B) TYPE: nucleic acid       (C) STRANDEDNESS: single       (D) TOPOLOGY: linear   - -     (ii) MOLECULE TYPE: Other nucleic acid   - -    (iii) HYPOTHETICAL: N   - -     (iv) ANTI-SENSE: N   - -    (vii) IMMEDIATE SOURCE:       (B) CLONE: ZC1847   - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:17:   - - AGCTTGCTTA AAGGGCAAGG AGTGTGGCAC CACGATGACC TCCTGCGTGT CC -#TGGCCCAC     60   - - AGCGTTGCGC AGCGTGCAGC GCACCGACAG TGGCC       - #- #       95   - -  - - (2) INFORMATION FOR SEQ ID NO:18:   - -      (i) SEQUENCE CHARACTERISTICS:       (A) LENGTH: 43 base - #pairs       (B) TYPE: nucleic acid       (C) STRANDEDNESS: single       (D) TOPOLOGY: linear   - -     (ii) MOLECULE TYPE: Other nucleic acid   - -    (iii) HYPOTHETICAL: N   - -     (iv) ANTI-SENSE: N   - -    (vii) IMMEDIATE SOURCE:       (B) CLONE: ZC1886   - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:18:   - - CCAGTGCCAA GCTTGTCTAG ACTTACCTTT AAAGGGCAAG GAG    - #- # 43  - -  - - (2) INFORMATION FOR SEQ ID NO:19:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 11 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other nucleic acid  - -    (iii) HYPOTHETICAL: N  - -     (iv) ANTI-SENSE: N  - -    (vii) IMMEDIATE SOURCE:      (B) CLONE: ZC1892  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:19:  - - AGCTTGAGCG T               - #                  - # - #       11  - -  - - (2) INFORMATION FOR SEQ ID NO:20:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 11 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other nucleic acid  - -    (iii) HYPOTHETICAL: N  - -     (iv) ANTI-SENSE: N  - -    (vii) IMMEDIATE SOURCE:      (B) CLONE: ZC1893  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:20:  - - CTAGACGCTC A               - #                  - # - #       11  - -  - - (2) INFORMATION FOR SEQ ID NO:21:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 47 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other nucleic acid  - -    (iii) HYPOTHETICAL: N  - -     (iv) ANTI-SENSE: N  - -    (vii) IMMEDIATE SOURCE:      (B) CLONE: ZC1894  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:21:  - - AGCTTCCAGT TCTTCGGCCT CATGTCAGTT CTTCGGCCTC ATGTGAT   - #47  - -  - - (2) INFORMATION FOR SEQ ID NO:22:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 47 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other nucleic acid  - -    (iii) HYPOTHETICAL: N  - -     (iv) ANTI-SENSE: N  - -    (vii) IMMEDIATE SOURCE:      (B) CLONE: ZC1895  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:22:  - - CTAGATCACA TGAGGCCGAA GAACTGACAT GAGGCCGAAG AACTGGA   - #47  - -  - - (2) INFORMATION FOR SEQ ID NO:23:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 66 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other nucleic acid  - -    (iii) HYPOTHETICAL: N  - -     (iv) ANTI-SENSE: N  - -    (vii) IMMEDIATE SOURCE:      (B) CLONE: ZC2181  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:23:  - - AATTCGGATC CACCATGGGC ACCAGCCACC CGGCGTTCCT GGTGTTAGGC TG -#CCTGCTGA     60   - - CCGGCC                 - #                  - #                  -#           66  - -  - - (2) INFORMATION FOR SEQ ID NO:24:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 71 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other nucleic acid  - -    (iii) HYPOTHETICAL: N  - -     (iv) ANTI-SENSE: N  - -    (vii) IMMEDIATE SOURCE:      (B) CLONE: ZC2182  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:24:  - - TGAGCCTGAT CCTGTGCCAA CTGAGCCTGC CATCGATCCT GCCAAACGAG AA -#CGAGAAGG     60   - - TTGTGCAGCT A               - #                  - #  - #       71  - -  - - (2) INFORMATION FOR SEQ ID NO:25:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 69 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other nucleic acid  - -    (iii) HYPOTHETICAL: N  - -     (iv) ANTI-SENSE: N  - -    (vii) IMMEDIATE SOURCE:      (B) CLONE: ZC2183  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:25:  - - AATTTAGCTG CACAACCTTC TCGTTCTCGT TTGGCAGGAT CGATGGCAGG CT -#CAGTTGGC     60   - - ACAGGATCA                - #                  - #- #         69  - -  - - (2) INFORMATION FOR SEQ ID NO:26:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 68 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other nucleic acid  - -    (iii) HYPOTHETICAL: N  - -     (iv) ANTI-SENSE: N  - -    (vii) IMMEDIATE SOURCE:      (B) CLONE: ZC2184  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:26:  - - GGCTCAGGCC GGTCAGCAGG CAGCCTAACA CCAGGAACGC CGGGTGGCTG GT -#GCCCATGG     60   - - TGGATCCG                - #                  - #- #          68   - -  - - (2) INFORMATION FOR SEQ ID NO:27:   - -      (i) SEQUENCE CHARACTERISTICS:       (A) LENGTH: 20 base - #pairs       (B) TYPE: nucleic acid       (C) STRANDEDNESS: single       (D) TOPOLOGY: linear   - -     (ii) MOLECULE TYPE: Other nucleic acid   - -    (iii) HYPOTHETICAL: N   - -     (iv) ANTI-SENSE: N   - -    (vii) IMMEDIATE SOURCE:       (B) CLONE: ZC2311   - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:27:   - - TGATCACCAT GGCTCAACTG            - #                  - #   - # 20  - -  - - (2) INFORMATION FOR SEQ ID NO:28:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 10 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other nucleic acid  - -    (iii) HYPOTHETICAL: N  - -     (iv) ANTI-SENSE: N  - -    (vii) IMMEDIATE SOURCE:      (B) CLONE: ZC2351  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:28:  - - CGAATTCCAC                - #                  - #- #        10  - -  - - (2) INFORMATION FOR SEQ ID NO:29:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 26 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other nucleic acid  - -    (iii) HYPOTHETICAL: N  - -     (iv) ANTI-SENSE: N  - -    (vii) IMMEDIATE SOURCE:      (B) CLONE: ZC2352  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:29:  - - ATTATACGCA TGGTGGAATT CGAGCT          - #                  - #   26  - -  - - (2) INFORMATION FOR SEQ ID NO:30:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 41 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other nucleic acid  - -    (iii) HYPOTHETICAL: N  - -     (iv) ANTI-SENSE: N  - -    (vii) IMMEDIATE SOURCE:      (B) CLONE: ZC2392  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:30:  - - ACGTAAGCTT GTCTAGACTT ACCTTCAGAA CGCAGGGTGG G    - #  - #   41  - -  - - (2) INFORMATION FOR SEQ ID NO:31:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 17 amino - #acids      (B) TYPE: amino acid      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: peptide  - -      (v) FRAGMENT TYPE: C-terminal  - -    (vii) IMMEDIATE SOURCE:      (B) CLONE: pWK1  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:31:  - - Ala Leu His Asn His Tyr Thr Glu Lys Ser Le - #u Ser Leu Ser Pro Gly 1               5   - #                10  - #                15  - - Lys  - -  - - (2) INFORMATION FOR SEQ ID NO:32:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 22 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other nucleic acid  - -    (iii) HYPOTHETICAL: N  - -     (iv) ANTI-SENSE: N  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:32:  - - TGTGACACTC TCCTGGGAGT TA           - #                  - #   22  - -  - - (2) INFORMATION FOR SEQ ID NO:33:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 30 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other nucleic acid  - -    (iii) HYPOTHETICAL: N  - -     (iv) ANTI-SENSE: N  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:33:  - - GCATAGTAGT TACCATATCC TCTTGCACAG         - #                  - #   30  - -  - - (2) INFORMATION FOR SEQ ID NO:34:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 25 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other nucleic acid  - -    (iii) HYPOTHETICAL: N  - -     (iv) ANTI-SENSE: N  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:34:  - - ACCGAACGTG AGAGGAGTGC TATAA          - #                  - #   25  - -  - - (2) INFORMATION FOR SEQ ID NO:35:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 4054 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: double      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: cDNA  - -    (iii) HYPOTHETICAL: N  - -     (iv) ANTI-SENSE: N  - -     (vi) ORIGINAL SOURCE:      (A) ORGANISM: Homo sapi - #ens  - -    (vii) IMMEDIATE SOURCE:      (B) CLONE: p-alpha-17B  - -     (ix) FEATURE:      (A) NAME/KEY: CDS      (B) LOCATION: 205..3471      (D) OTHER INFORMATION:  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:35:  - - GCCCTGGGGA CGGACCGTGG GCGGCGCGCA GCGGCGGGAC GCGTTTTGGG GA -#CGTGGTGG     60   - - CCAGCGCCTT CCTGCAGACC CACAGGGAAG TACTCCCTTT GACCTCCGGG GA -#GCTGCGAC    120   - - CAGGTTATAC GTTGCTGGTG GAAAAGTGAC AATTCTAGGA AAAGAGCTAA AA -#GCCGGATC    180   - - GGTGACCGAA AGTTTCCCAG AGCT ATG GGG ACT TCC CAT C - #CG GCG TTC CTG 231              - #         Met Gly Thr Ser His - #Pro Ala Phe Leu              - #           1       - #        5  - - GTC TTA GGC TGT CTT CTC ACA GGG CTG AGC CT - #A ATC CTC TGC CAG CTT279 Val Leu Gly Cys Leu Leu Thr Gly Leu Ser Le - #u Ile Leu Cys Gln Leu  10                 - # 15                 - # 20                 - # 25  - - TCA TTA CCC TCT ATC CTT CCA AAT GAA AAT GA - #A AAG GTT GTG CAG CTG327 Ser Leu Pro Ser Ile Leu Pro Asn Glu Asn Gl - #u Lys Val Val Gln Leu             30 - #                 35 - #                 40  - - AAT TCA TCC TTT TCT CTG AGA TGC TTT GGG GA - #G AGT GAA GTG AGC TGG375 Asn Ser Ser Phe Ser Leu Arg Cys Phe Gly Gl - #u Ser Glu Val Ser Trp         45     - #             50     - #             55  - - CAG TAC CCC ATG TCT GAA GAA GAG AGC TCC GA - #T GTG GAA ATC AGA AAT423 Gln Tyr Pro Met Ser Glu Glu Glu Ser Ser As - #p Val Glu Ile Arg Asn     60         - #         65         - #         70  - - GAA GAA AAC AAC AGC GGC CTT TTT GTG ACG GT - #C TTG GAA GTG AGC AGT471 Glu Glu Asn Asn Ser Gly Leu Phe Val Thr Va - #l Leu Glu Val Ser Ser 75             - #     80             - #     85  - - GCC TCG GCG GCC CAC ACA GGG TTG TAC ACT TG - #C TAT TAC AAC CAC ACT519 Ala Ser Ala Ala His Thr Gly Leu Tyr Thr Cy - #s Tyr Tyr Asn His Thr  90                 - # 95                 - #100                 - #105  - - CAG ACA GAA GAG AAT GAG CTT GAA GGC AGG CA - #C ATT TAC ATC TAT GTG567 Gln Thr Glu Glu Asn Glu Leu Glu Gly Arg Hi - #s Ile Tyr Ile Tyr Val            110  - #               115  - #               120  - - CCA GAC CCA GAT GTA GCC TTT GTA CCT CTA GG - #A ATG ACG GAT TAT TTA615 Pro Asp Pro Asp Val Ala Phe Val Pro Leu Gl - #y Met Thr Asp Tyr Leu        125      - #           130      - #           135  - - GTC ATC GTG GAG GAT GAT GAT TCT GCC ATT AT - #A CCT TGT CGC ACA ACT663 Val Ile Val Glu Asp Asp Asp Ser Ala Ile Il - #e Pro Cys Arg Thr Thr    140          - #       145          - #       150  - - GAT CCC GAG ACT CCT GTA ACC TTA CAC AAC AG - #T GAG GGG GTG GTA CCT711 Asp Pro Glu Thr Pro Val Thr Leu His Asn Se - #r Glu Gly Val Val Pro155              - #   160              - #   165  - - GCC TCC TAC GAC AGC AGA CAG GGC TTT AAT GG - #G ACC TTC ACT GTA GGG759 Ala Ser Tyr Asp Ser Arg Gln Gly Phe Asn Gl - #y Thr Phe Thr Val Gly 170                 1 - #75                 1 - #80                 1 -#85   - - CCC TAT ATC TGT GAG GCC ACC GTC AAA GGA AA - #G AAG TTC CAG ACCATC      807  Pro Tyr Ile Cys Glu Ala Thr Val Lys Gly Ly - #s Lys Phe Gln Thr Ile            190  - #               195  - #               200  - - CCA TTT AAT GTT TAT GCT TTA AAA GCA ACA TC - #A GAG CTG GAT CTA GAA855 Pro Phe Asn Val Tyr Ala Leu Lys Ala Thr Se - #r Glu Leu Asp Leu Glu        205      - #           210      - #           215  - - ATG GAA GCT CTT AAA ACC GTG TAT AAG TCA GG - #G GAA ACG ATT GTG GTC903 Met Glu Ala Leu Lys Thr Val Tyr Lys Ser Gl - #y Glu Thr Ile Val Val    220          - #       225          - #       230  - - ACC TGT GCT GTT TTT AAC AAT GAG GTG GTT GA - #C CTT CAA TGG ACT TAC951 Thr Cys Ala Val Phe Asn Asn Glu Val Val As - #p Leu Gln Trp Thr Tyr235              - #   240              - #   245  - - CCT GGA GAA GTG AAA GGC AAA GGC ATC ACA AT - #A CTG GAA GAA ATC AAA999 Pro Gly Glu Val Lys Gly Lys Gly Ile Thr Il - #e Leu Glu Glu Ile Lys 250                 2 - #55                 2 - #60                 2 -#65   - - GTC CCA TCC ATC AAA TTG GTG TAC ACT TTG AC - #G GTC CCC GAG GCCACG     1047  Val Pro Ser Ile Lys Leu Val Tyr Thr Leu Th - #r Val Pro Glu Ala Thr            270  - #               275  - #               280  - - GTG AAA GAC AGT GGA GAT TAC GAA TGT GCT GC - #C CGC CAG GCT ACC AGG    1095 Val Lys Asp Ser Gly Asp Tyr Glu Cys Ala Al - #a Arg Gln Ala Thr Arg        285      - #           290      - #           295  - - GAG GTC AAA GAA ATG AAG AAA GTC ACT ATT TC - #T GTC CAT GAG AAA GGT    1143 Glu Val Lys Glu Met Lys Lys Val Thr Ile Se - #r Val His Glu Lys Gly    300          - #       305          - #       310  - - TTC ATT GAA ATC AAA CCC ACC TTC AGC CAG TT - #G GAA GCT GTC AAC CTG    1191 Phe Ile Glu Ile Lys Pro Thr Phe Ser Gln Le - #u Glu Ala Val Asn Leu315              - #   320              - #   325  - - CAT GAA GTC AAA CAT TTT GTT GTA GAG GTG CG - #G GCC TAC CCA CCT CCC    1239 His Glu Val Lys His Phe Val Val Glu Val Ar - #g Ala Tyr Pro Pro Pro 330                 3 - #35                 3 - #40                 3 -#45   - - AGG ATA TCC TGG CTG AAA AAC AAT CTG ACT CT - #G ATT GAA AAT CTCACT     1287  Arg Ile Ser Trp Leu Lys Asn Asn Leu Thr Le - #u Ile Glu Asn Leu Thr            350  - #               355  - #               360  - - GAG ATC ACC ACT GAT GTG GAA AAG ATT CAG GA - #A ATA AGG TAT CGA AGC    1335 Glu Ile Thr Thr Asp Val Glu Lys Ile Gln Gl - #u Ile Arg Tyr Arg Ser        365      - #           370      - #           375  - - AAA TTA AAG CTG ATC CGT GCT AAG GAA GAA GA - #C AGT GGC CAT TAT ACT    1383 Lys Leu Lys Leu Ile Arg Ala Lys Glu Glu As - #p Ser Gly His Tyr Thr    380          - #       385          - #       390  - - ATT GTA GCT CAA AAT GAA GAT GCT GTG AAG AG - #C TAT ACT TTT GAA CTG    1431 Ile Val Ala Gln Asn Glu Asp Ala Val Lys Se - #r Tyr Thr Phe Glu Leu395              - #   400              - #   405  - - TTA ACT CAA GTT CCT TCA TCC ATT CTG GAC TT - #G GTC GAT GAT CAC CAT    1479 Leu Thr Gln Val Pro Ser Ser Ile Leu Asp Le - #u Val Asp Asp His His 410                 4 - #15                 4 - #20                 4 -#25   - - GGC TCA ACT GGG GGA CAG ACG GTG AGG TGC AC - #A GCT GAA GGC ACGCCG     1527  Gly Ser Thr Gly Gly Gln Thr Val Arg Cys Th - #r Ala Glu Gly Thr Pro            430  - #               435  - #               440  - - CTT CCT GAT ATT GAG TGG ATG ATA TGC AAA GA - #T ATT AAG AAA TGT AAT    1575 Leu Pro Asp Ile Glu Trp Met Ile Cys Lys As - #p Ile Lys Lys Cys Asn        445      - #           450      - #           455  - - AAT GAA ACT TCC TGG ACT ATT TTG GCC AAC AA - #T GTC TCA AAC ATC ATC    1623 Asn Glu Thr Ser Trp Thr Ile Leu Ala Asn As - #n Val Ser Asn Ile Ile    460          - #       465          - #       470  - - ACG GAG ATC CAC TCC CGA GAC AGG AGT ACC GT - #G GAG GGC CGT GTG ACT    1671 Thr Glu Ile His Ser Arg Asp Arg Ser Thr Va - #l Glu Gly Arg Val Thr475              - #   480              - #   485  - - TTC GCC AAA GTG GAG GAG ACC ATC GCC GTG CG - #A TGC CTG GCT AAG AAT    1719 Phe Ala Lys Val Glu Glu Thr Ile Ala Val Ar - #g Cys Leu Ala Lys Asn 490                 4 - #95                 5 - #00                 5 -#05   - - CTC CTT GGA GCT GAG AAC CGA GAG CTG AAG CT - #G GTG GCT CCC ACCCTG     1767  Leu Leu Gly Ala Glu Asn Arg Glu Leu Lys Le - #u Val Ala Pro Thr Leu            510  - #               515  - #               520  - - CGT TCT GAA CTC ACG GTG GCT GCT GCA GTC CT - #G GTG CTG TTG GTG ATT    1815 Arg Ser Glu Leu Thr Val Ala Ala Ala Val Le - #u Val Leu Leu Val Ile        525      - #           530      - #           535  - - GTG ATC ATC TCA CTT ATT GTC CTG GTT GTC AT - #T TGG AAA CAG AAA CCG    1863 Val Ile Ile Ser Leu Ile Val Leu Val Val Il - #e Trp Lys Gln Lys Pro    540          - #       545          - #       550  - - AGG TAT GAA ATT CGC TGG AGG GTC ATT GAA TC - #A ATC AGC CCG GAT GGA    1911 Arg Tyr Glu Ile Arg Trp Arg Val Ile Glu Se - #r Ile Ser Pro Asp Gly555              - #   560              - #   565  - - CAT GAA TAT ATT TAT GTG GAC CCG ATG CAG CT - #G CCT TAT GAC TCA AGA    1959 His Glu Tyr Ile Tyr Val Asp Pro Met Gln Le - #u Pro Tyr Asp Ser Arg 570                 5 - #75                 5 - #80                 5 -#85   - - TGG GAG TTT CCA AGA GAT GGA CTA GTG CTT GG - #T CGG GTC TTG GGGTCT     2007  Trp Glu Phe Pro Arg Asp Gly Leu Val Leu Gl - #y Arg Val Leu Gly Ser            590  - #               595  - #               600  - - GGA GCG TTT GGG AAG GTG GTT GAA GGA ACA GC - #C TAT GGA TTA AGC CGG    2055 Gly Ala Phe Gly Lys Val Val Glu Gly Thr Al - #a Tyr Gly Leu Ser Arg        605      - #           610      - #           615  - - TCC CAA CCT GTC ATG AAA GTT GCA GTG AAG AT - #G CTA AAA CCC ACG GCC    2103 Ser Gln Pro Val Met Lys Val Ala Val Lys Me - #t Leu Lys Pro Thr Ala    620          - #       625          - #       630  - - AGA TCC AGT GAA AAA CAA GCT CTC ATG TCT GA - #A CTG AAG ATA ATG ACT    2151 Arg Ser Ser Glu Lys Gln Ala Leu Met Ser Gl - #u Leu Lys Ile Met Thr635              - #   640              - #   645  - - CAC CTG GGG CCA CAT TTG AAC ATT GTA AAC TT - #G CTG GGA GCC TGC ACC    2199 His Leu Gly Pro His Leu Asn Ile Val Asn Le - #u Leu Gly Ala Cys Thr 650                 6 - #55                 6 - #60                 6 -#65   - - AAG TCA GGC CCC ATT TAC ATC ATC ACA GAG TA - #T TGC TTC TAT GGAGAT     2247  Lys Ser Gly Pro Ile Tyr Ile Ile Thr Glu Ty - #r Cys Phe Tyr Gly Asp            670  - #               675  - #               680  - - TTG GTC AAC TAT TTG CAT AAG AAT AGG GAT AG - #C TTC CTG AGC CAC CAC    2295 Leu Val Asn Tyr Leu His Lys Asn Arg Asp Se - #r Phe Leu Ser His His        685      - #           690      - #           695  - - CCA GAG AAG CCA AAG AAA GAG CTG GAT ATC TT - #T GGA TTG AAC CCT GCT    2343 Pro Glu Lys Pro Lys Lys Glu Leu Asp Ile Ph - #e Gly Leu Asn Pro Ala    700          - #       705          - #       710  - - GAT GAA AGC ACA CGG AGC TAT GTT ATT TTA TC - #T TTT GAA AAC AAT GGT    2391 Asp Glu Ser Thr Arg Ser Tyr Val Ile Leu Se - #r Phe Glu Asn Asn Gly715              - #   720              - #   725  - - GAC TAC ATG GAC ATG AAG CAG GCT GAT ACT AC - #A CAG TAT GTC CCC ATG    2439 Asp Tyr Met Asp Met Lys Gln Ala Asp Thr Th - #r Gln Tyr Val Pro Met 730                 7 - #35                 7 - #40                 7 -#45   - - CTA GAA AGG AAA GAG GTT TCT AAA TAT TCC GA - #C ATC CAG AGA TCACTC     2487  Leu Glu Arg Lys Glu Val Ser Lys Tyr Ser As - #p Ile Gln Arg Ser Leu            750  - #               755  - #               760  - - TAT GAT CGT CCA GCC TCA TAT AAG AAG AAA TC - #T ATG TTA GAC TCA GAA    2535 Tyr Asp Arg Pro Ala Ser Tyr Lys Lys Lys Se - #r Met Leu Asp Ser Glu        765      - #           770      - #           775  - - GTC AAA AAC CTC CTT TCA GAT GAT AAC TCA GA - #A GGC CTT ACT TTA TTG    2583 Val Lys Asn Leu Leu Ser Asp Asp Asn Ser Gl - #u Gly Leu Thr Leu Leu    780          - #       785          - #       790  - - GAT TTG TTG AGC TTC ACC TAT CAA GTT GCC CG - #A GGA ATG GAG TTT TTG    2631 Asp Leu Leu Ser Phe Thr Tyr Gln Val Ala Ar - #g Gly Met Glu Phe Leu795              - #   800              - #   805  - - GCT TCA AAA AAT TGT GTC CAC CGT GAT CTG GC - #T GCT CGC AAC GTC CTC    2679 Ala Ser Lys Asn Cys Val His Arg Asp Leu Al - #a Ala Arg Asn Val Leu 810                 8 - #15                 8 - #20                 8 -#25   - - CTG GCA CAA GGA AAA ATT GTG AAG ATC TGT GA - #C TTT GGC CTG GCCAGA     2727  Leu Ala Gln Gly Lys Ile Val Lys Ile Cys As - #p Phe Gly Leu Ala Arg            830  - #               835  - #               840  - - GAC ATC ATG CAT GAT TCG AAC TAT GTG TCG AA - #A GGC AGT ACC TTT CTG    2775 Asp Ile Met His Asp Ser Asn Tyr Val Ser Ly - #s Gly Ser Thr Phe Leu        845      - #           850      - #           855  - - CCC GTG AAG TGG ATG GCT CCT GAG AGC ATC TT - #T GAC AAC CTC TAC ACC    2823 Pro Val Lys Trp Met Ala Pro Glu Ser Ile Ph - #e Asp Asn Leu Tyr Thr    860          - #       865          - #       870  - - ACA CTG AGT GAT GTC TGG TCT TAT GGC ATT CT - #G CTC TGG GAG ATC TTT    2871 Thr Leu Ser Asp Val Trp Ser Tyr Gly Ile Le - #u Leu Trp Glu Ile Phe875              - #   880              - #   885  - - TCC CTT GGT GGC ACC CCT TAC CCC GGC ATG AT - #G GTG GAT TCT ACT TTC    2919 Ser Leu Gly Gly Thr Pro Tyr Pro Gly Met Me - #t Val Asp Ser Thr Phe 890                 8 - #95                 9 - #00                 9 -#05   - - TAC AAT AAG ATC AAG AGT GGG TAC CGG ATG GC - #C AAG CCT GAC CACGCT     2967  Tyr Asn Lys Ile Lys Ser Gly Tyr Arg Met Al - #a Lys Pro Asp His Ala            910  - #               915  - #               920  - - ACC AGT GAA GTC TAC GAG ATC ATG GTG AAA TG - #C TGG AAC AGT GAG CCG    3015 Thr Ser Glu Val Tyr Glu Ile Met Val Lys Cy - #s Trp Asn Ser Glu Pro        925      - #           930      - #           935  - - GAG AAG AGA CCC TCC TTT TAC CAC CTG AGT GA - #G ATT GTG GAG AAT CTG    3063 Glu Lys Arg Pro Ser Phe Tyr His Leu Ser Gl - #u Ile Val Glu Asn Leu    940          - #       945          - #       950  - - CTG CCT GGA CAA TAT AAA AAG AGT TAT GAA AA - #A ATT CAC CTG GAC TTC    3111 Leu Pro Gly Gln Tyr Lys Lys Ser Tyr Glu Ly - #s Ile His Leu Asp Phe955              - #   960              - #   965  - - CTG AAG AGT GAC CAT CCT GCT GTG GCA CGC AT - #G CGT GTG GAC TCA GAC    3159 Leu Lys Ser Asp His Pro Ala Val Ala Arg Me - #t Arg Val Asp Ser Asp 970                 9 - #75                 9 - #80                 9 -#85   - - AAT GCA TAC ATT GGT GTC ACC TAC AAA AAC GA - #G GAA GAC AAG CTGAAG     3207  Asn Ala Tyr Ile Gly Val Thr Tyr Lys Asn Gl - #u Glu Asp Lys Leu Lys            990  - #               995  - #               1000  - - GAC TGG GAG GGT GGT CTG GAT GAG CAG AGA CT - #G AGC GCT GAC AGT GGC    3255 Asp Trp Glu Gly Gly Leu Asp Glu Gln Arg Le - #u Ser Ala Asp Ser Gly        1005     - #           1010      - #          1015  - - TAC ATC ATT CCT CTG CCT GAC ATT GAC CCT GT - #C CCT GAG GAG GAG GAC    3303 Tyr Ile Ile Pro Leu Pro Asp Ile Asp Pro Va - #l Pro Glu Glu Glu Asp    1020         - #       1025          - #      1030  - - CTG GGC AAG AGG AAC AGA CAC AGC TCG CAG AC - #C TCT GAA GAG AGT GCC    3351 Leu Gly Lys Arg Asn Arg His Ser Ser Gln Th - #r Ser Glu Glu Ser Ala1035             - #   1040              - #  1045  - - ATT GAG ACG GGT TCC AGC AGT TCC ACC TTC AT - #C AAG AGA GAG GAC GAG    3399 Ile Glu Thr Gly Ser Ser Ser Ser Thr Phe Il - #e Lys Arg Glu Asp Glu 1050                1055 - #                1060 - #               1065  - - ACC ATT GAA GAC ATC GAC ATG ATG GAC GAC AT - #C GGC ATA GAC TCT TCA    3447 Thr Ile Glu Asp Ile Asp Met Met Asp Asp Il - #e Gly Ile Asp Ser Ser            1070 - #               1075  - #              1080  - - GAC CTG GTG GAA GAC AGC TTC CTG TAACTGGCGG AT - #TCGAGGGG TTCCTTCCAC    3501 Asp Leu Val Glu Asp Ser Phe Leu        1085  - - TTCTGGGGCC ACCTCTGGAT CCCGTTCAGA AAACCACTTT ATTGCAATGC GG -#AGGTTGAG   3561   - - AGGAGGACTT GGTTGATGTT TAAAGAGAAG TTCCCAGCCA AGGGCCTCGG GG -#AGCGTTCT   3621   - - AAATATGAAT GAATGGGATA TTTTGAAATG AACTTTGTCA GTGTTGCCTC TT -#GCAATGCC   3681   - - TCAGTAGCAT CTCAGTGGTG TGTGAAGTTT GGAGATAGAT GGATAAGGGA AT -#AATAGGCC   3741   - - ACAGAAGGTG AACTTTGTGC TTCAAGGACA TTGGTGAGAG TCCAACAGAC AC -#AATTTATA   3801   - - CTGCGACAGA ACTTCAGCAT TGTAATTATG TAAATAACTC TAACCAAGGC TG -#TGTTTAGA   3861   - - TTGTATTAAC TATCTTCTTT GGACTTCTGA AGAGACCACT CAATCCATCC TG -#TACTTCCC   3921   - - TCTTGAAACC TGATGTAGCT GCTGTTGAAC TTTTTAAAGA AGTGCATGAA AA -#ACCATTTT   3981   - - TGAACCTTAA AAGGTACTGG TACTATAGCA TTTTGCTATC TTTTTTAGTG TT -#AAAGAGAT   4041   - - AAAGAATAAT AAG              - #                  - #   - #    4054  - -  - - (2) INFORMATION FOR SEQ ID NO:36:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 1089 amino - #acids      (B) TYPE: amino acid      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: protein  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:36:  - - Met Gly Thr Ser His Pro Ala Phe Leu Val Le - #u Gly Cys Leu Leu Thr   1               5 - #                 10 - #                 15  - - Gly Leu Ser Leu Ile Leu Cys Gln Leu Ser Le - #u Pro Ser Ile Leu Pro         20     - #             25     - #             30  - - Asn Glu Asn Glu Lys Val Val Gln Leu Asn Se - #r Ser Phe Ser Leu Arg     35         - #         40         - #         45  - - Cys Phe Gly Glu Ser Glu Val Ser Trp Gln Ty - #r Pro Met Ser Glu Glu 50             - #     55             - #     60  - - Glu Ser Ser Asp Val Glu Ile Arg Asn Glu Gl - #u Asn Asn Ser Gly Leu  65                 - # 70                 - # 75                 - # 80  - - Phe Val Thr Val Leu Glu Val Ser Ser Ala Se - #r Ala Ala His Thr Gly             85 - #                 90 - #                 95  - - Leu Tyr Thr Cys Tyr Tyr Asn His Thr Gln Th - #r Glu Glu Asn Glu Leu        100      - #           105      - #           110  - - Glu Gly Arg His Ile Tyr Ile Tyr Val Pro As - #p Pro Asp Val Ala Phe    115          - #       120          - #       125  - - Val Pro Leu Gly Met Thr Asp Tyr Leu Val Il - #e Val Glu Asp Asp Asp130              - #   135              - #   140  - - Ser Ala Ile Ile Pro Cys Arg Thr Thr Asp Pr - #o Glu Thr Pro Val Thr 145                 1 - #50                 1 - #55                 1 -#60   - - Leu His Asn Ser Glu Gly Val Val Pro Ala Se - #r Tyr Asp Ser ArgGln             165  - #               170  - #               175  - - Gly Phe Asn Gly Thr Phe Thr Val Gly Pro Ty - #r Ile Cys Glu Ala Thr        180      - #           185      - #           190  - - Val Lys Gly Lys Lys Phe Gln Thr Ile Pro Ph - #e Asn Val Tyr Ala Leu    195          - #       200          - #       205  - - Lys Ala Thr Ser Glu Leu Asp Leu Glu Met Gl - #u Ala Leu Lys Thr Val210              - #   215              - #   220  - - Tyr Lys Ser Gly Glu Thr Ile Val Val Thr Cy - #s Ala Val Phe Asn Asn 225                 2 - #30                 2 - #35                 2 -#40   - - Glu Val Val Asp Leu Gln Trp Thr Tyr Pro Gl - #y Glu Val Lys GlyLys             245  - #               250  - #               255  - - Gly Ile Thr Ile Leu Glu Glu Ile Lys Val Pr - #o Ser Ile Lys Leu Val        260      - #           265      - #           270  - - Tyr Thr Leu Thr Val Pro Glu Ala Thr Val Ly - #s Asp Ser Gly Asp Tyr    275          - #       280          - #       285  - - Glu Cys Ala Ala Arg Gln Ala Thr Arg Glu Va - #l Lys Glu Met Lys Lys290              - #   295              - #   300  - - Val Thr Ile Ser Val His Glu Lys Gly Phe Il - #e Glu Ile Lys Pro Thr 305                 3 - #10                 3 - #15                 3 -#20   - - Phe Ser Gln Leu Glu Ala Val Asn Leu His Gl - #u Val Lys His PheVal             325  - #               330  - #               335  - - Val Glu Val Arg Ala Tyr Pro Pro Pro Arg Il - #e Ser Trp Leu Lys Asn        340      - #           345      - #           350  - - Asn Leu Thr Leu Ile Glu Asn Leu Thr Glu Il - #e Thr Thr Asp Val Glu    355          - #       360          - #       365  - - Lys Ile Gln Glu Ile Arg Tyr Arg Ser Lys Le - #u Lys Leu Ile Arg Ala370              - #   375              - #   380  - - Lys Glu Glu Asp Ser Gly His Tyr Thr Ile Va - #l Ala Gln Asn Glu Asp 385                 3 - #90                 3 - #95                 4 -#00   - - Ala Val Lys Ser Tyr Thr Phe Glu Leu Leu Th - #r Gln Val Pro SerSer             405  - #               410  - #               415  - - Ile Leu Asp Leu Val Asp Asp His His Gly Se - #r Thr Gly Gly Gln Thr        420      - #           425      - #           430  - - Val Arg Cys Thr Ala Glu Gly Thr Pro Leu Pr - #o Asp Ile Glu Trp Met    435          - #       440          - #       445  - - Ile Cys Lys Asp Ile Lys Lys Cys Asn Asn Gl - #u Thr Ser Trp Thr Ile450              - #   455              - #   460  - - Leu Ala Asn Asn Val Ser Asn Ile Ile Thr Gl - #u Ile His Ser Arg Asp 465                 4 - #70                 4 - #75                 4 -#80   - - Arg Ser Thr Val Glu Gly Arg Val Thr Phe Al - #a Lys Val Glu GluThr             485  - #               490  - #               495  - - Ile Ala Val Arg Cys Leu Ala Lys Asn Leu Le - #u Gly Ala Glu Asn Arg        500      - #           505      - #           510  - - Glu Leu Lys Leu Val Ala Pro Thr Leu Arg Se - #r Glu Leu Thr Val Ala    515          - #       520          - #       525  - - Ala Ala Val Leu Val Leu Leu Val Ile Val Il - #e Ile Ser Leu Ile Val530              - #   535              - #   540  - - Leu Val Val Ile Trp Lys Gln Lys Pro Arg Ty - #r Glu Ile Arg Trp Arg 545                 5 - #50                 5 - #55                 5 -#60   - - Val Ile Glu Ser Ile Ser Pro Asp Gly His Gl - #u Tyr Ile Tyr ValAsp             565  - #               570  - #               575  - - Pro Met Gln Leu Pro Tyr Asp Ser Arg Trp Gl - #u Phe Pro Arg Asp Gly        580      - #           585      - #           590  - - Leu Val Leu Gly Arg Val Leu Gly Ser Gly Al - #a Phe Gly Lys Val Val    595          - #       600          - #       605  - - Glu Gly Thr Ala Tyr Gly Leu Ser Arg Ser Gl - #n Pro Val Met Lys Val610              - #   615              - #   620  - - Ala Val Lys Met Leu Lys Pro Thr Ala Arg Se - #r Ser Glu Lys Gln Ala 625                 6 - #30                 6 - #35                 6 -#40   - - Leu Met Ser Glu Leu Lys Ile Met Thr His Le - #u Gly Pro His LeuAsn             645  - #               650  - #               655  - - Ile Val Asn Leu Leu Gly Ala Cys Thr Lys Se - #r Gly Pro Ile Tyr Ile        660      - #           665      - #           670  - - Ile Thr Glu Tyr Cys Phe Tyr Gly Asp Leu Va - #l Asn Tyr Leu His Lys    675          - #       680          - #       685  - - Asn Arg Asp Ser Phe Leu Ser His His Pro Gl - #u Lys Pro Lys Lys Glu690              - #   695              - #   700  - - Leu Asp Ile Phe Gly Leu Asn Pro Ala Asp Gl - #u Ser Thr Arg Ser Tyr 705                 7 - #10                 7 - #15                 7 -#20   - - Val Ile Leu Ser Phe Glu Asn Asn Gly Asp Ty - #r Met Asp Met LysGln             725  - #               730  - #               735  - - Ala Asp Thr Thr Gln Tyr Val Pro Met Leu Gl - #u Arg Lys Glu Val Ser        740      - #           745      - #           750  - - Lys Tyr Ser Asp Ile Gln Arg Ser Leu Tyr As - #p Arg Pro Ala Ser Tyr    755          - #       760          - #       765  - - Lys Lys Lys Ser Met Leu Asp Ser Glu Val Ly - #s Asn Leu Leu Ser Asp770              - #   775              - #   780  - - Asp Asn Ser Glu Gly Leu Thr Leu Leu Asp Le - #u Leu Ser Phe Thr Tyr 785                 7 - #90                 7 - #95                 8 -#00   - - Gln Val Ala Arg Gly Met Glu Phe Leu Ala Se - #r Lys Asn Cys ValHis             805  - #               810  - #               815  - - Arg Asp Leu Ala Ala Arg Asn Val Leu Leu Al - #a Gln Gly Lys Ile Val        820      - #           825      - #           830  - - Lys Ile Cys Asp Phe Gly Leu Ala Arg Asp Il - #e Met His Asp Ser Asn    835          - #       840          - #       845  - - Tyr Val Ser Lys Gly Ser Thr Phe Leu Pro Va - #l Lys Trp Met Ala Pro850              - #   855              - #   860  - - Glu Ser Ile Phe Asp Asn Leu Tyr Thr Thr Le - #u Ser Asp Val Trp Ser 865                 8 - #70                 8 - #75                 8 -#80   - - Tyr Gly Ile Leu Leu Trp Glu Ile Phe Ser Le - #u Gly Gly Thr ProTyr             885  - #               890  - #               895  - - Pro Gly Met Met Val Asp Ser Thr Phe Tyr As - #n Lys Ile Lys Ser Gly        900      - #           905      - #           910  - - Tyr Arg Met Ala Lys Pro Asp His Ala Thr Se - #r Glu Val Tyr Glu Ile    915          - #       920          - #       925  - - Met Val Lys Cys Trp Asn Ser Glu Pro Glu Ly - #s Arg Pro Ser Phe Tyr930              - #   935              - #   940  - - His Leu Ser Glu Ile Val Glu Asn Leu Leu Pr - #o Gly Gln Tyr Lys Lys 945                 9 - #50                 9 - #55                 9 -#60   - - Ser Tyr Glu Lys Ile His Leu Asp Phe Leu Ly - #s Ser Asp His ProAla             965  - #               970  - #               975  - - Val Ala Arg Met Arg Val Asp Ser Asp Asn Al - #a Tyr Ile Gly Val Thr        980      - #           985      - #           990  - - Tyr Lys Asn Glu Glu Asp Lys Leu Lys Asp Tr - #p Glu Gly Gly Leu Asp    995          - #       1000          - #      1005  - - Glu Gln Arg Leu Ser Ala Asp Ser Gly Tyr Il - #e Ile Pro Leu Pro Asp1010             - #   1015              - #  1020  - - Ile Asp Pro Val Pro Glu Glu Glu Asp Leu Gl - #y Lys Arg Asn Arg His 1025                1030 - #                1035 - #               1040  - - Ser Ser Gln Thr Ser Glu Glu Ser Ala Ile Gl - #u Thr Gly Ser Ser Ser            1045 - #               1050  - #              1055  - - Ser Thr Phe Ile Lys Arg Glu Asp Glu Thr Il - #e Glu Asp Ile Asp Met        1060     - #           1065      - #          1070  - - Met Asp Asp Ile Gly Ile Asp Ser Ser Asp Le - #u Val Glu Asp Ser Phe    1075         - #       1080          - #      1085  - - Leu__________________________________________________________________________</pre>
    
    </div></div></div><div class="patent-section patent-tabular-section"><a id="backward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Cited Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4474893">US4474893</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jul 1, 1981</td><td class="patent-data-table-td patent-date-value">Oct 2, 1984</td><td class="patent-data-table-td ">The University of Texas System Cancer Center</td><td class="patent-data-table-td ">Antibody with binding affinity for two antigens</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4599311">US4599311</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Aug 13, 1982</td><td class="patent-data-table-td patent-date-value">Jul 8, 1986</td><td class="patent-data-table-td ">Kawasaki Glenn H</td><td class="patent-data-table-td ">Glycolytic promotersfor regulated protein expression: protease inhibitor</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4761371">US4761371</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Feb 12, 1985</td><td class="patent-data-table-td patent-date-value">Aug 2, 1988</td><td class="patent-data-table-td ">Genentech, Inc.</td><td class="patent-data-table-td ">Genetic engineering</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4766073">US4766073</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Aug 13, 1986</td><td class="patent-data-table-td patent-date-value">Aug 23, 1988</td><td class="patent-data-table-td ">Zymogenetics Inc.</td><td class="patent-data-table-td ">Expression of biologically active PDGF analogs in eucaryotic cells</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4816397">US4816397</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Mar 23, 1984</td><td class="patent-data-table-td patent-date-value">Mar 28, 1989</td><td class="patent-data-table-td ">Celltech, Limited</td><td class="patent-data-table-td ">Genetic engineering</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4816567">US4816567</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Apr 8, 1983</td><td class="patent-data-table-td patent-date-value">Mar 28, 1989</td><td class="patent-data-table-td ">Genentech, Inc.</td><td class="patent-data-table-td ">Immunology, vectors, dna</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5015571">US5015571</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Apr 16, 1987</td><td class="patent-data-table-td patent-date-value">May 14, 1991</td><td class="patent-data-table-td ">Scripps Clinic And Research Foundation</td><td class="patent-data-table-td ">Polypeptide-induced monoclonal receptors to protein ligands</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5098833">US5098833</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Feb 23, 1989</td><td class="patent-data-table-td patent-date-value">Mar 24, 1992</td><td class="patent-data-table-td ">Genentech, Inc.</td><td class="patent-data-table-td ">DNA sequence encoding a functional domain of a lymphocyte homing receptor</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5116964">US5116964</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Nov 22, 1989</td><td class="patent-data-table-td patent-date-value">May 26, 1992</td><td class="patent-data-table-td ">Genentech, Inc.</td><td class="patent-data-table-td ">Hybrid immunoglobulins</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5155027">US5155027</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">May 2, 1989</td><td class="patent-data-table-td patent-date-value">Oct 13, 1992</td><td class="patent-data-table-td ">Zymogenetics, Inc.</td><td class="patent-data-table-td ">Culturing cells having DNA construct encoding a polypeptide having a nonimmuno- and an immunologlobulin chain which dimerize to become biologically active</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5750375">US5750375</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jun 7, 1995</td><td class="patent-data-table-td patent-date-value">May 12, 1998</td><td class="patent-data-table-td ">Zymogenetics, Inc.</td><td class="patent-data-table-td ">Methods of producing secreted receptor analogs and biologically active dimerized polypeptide fusions</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP0088994B1?cl=en">EP0088994B1</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Mar 9, 1983</td><td class="patent-data-table-td patent-date-value">Jun 19, 1991</td><td class="patent-data-table-td ">Schering Corporation</td><td class="patent-data-table-td ">Hybrid dna, binding composition prepared thereby and processes therefor</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP0116201A1?cl=en">EP0116201A1</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Oct 26, 1983</td><td class="patent-data-table-td patent-date-value">Aug 22, 1984</td><td class="patent-data-table-td ">Chiron Corporation</td><td class="patent-data-table-td ">Secretory expression in eukaryotes</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP0120694A2?cl=en">EP0120694A2</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Mar 23, 1984</td><td class="patent-data-table-td patent-date-value">Oct 3, 1984</td><td class="patent-data-table-td ">Celltech Limited</td><td class="patent-data-table-td ">Processes for the production of multichain polypeptides or proteins</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP0162319A2?cl=en">EP0162319A2</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Apr 24, 1985</td><td class="patent-data-table-td patent-date-value">Nov 27, 1985</td><td class="patent-data-table-td ">Tasuku Honjo</td><td class="patent-data-table-td ">Method of joining heterologous genes</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP0171496B1?cl=en">EP0171496B1</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Mar 8, 1985</td><td class="patent-data-table-td patent-date-value">May 26, 1993</td><td class="patent-data-table-td ">Research Development Corporation of Japan</td><td class="patent-data-table-td ">Process for the production of a chimera monoclonal antibody</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP0173494A2?cl=en">EP0173494A2</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Aug 7, 1985</td><td class="patent-data-table-td patent-date-value">Mar 5, 1986</td><td class="patent-data-table-td ">The Board Of Trustees Of The Leland Stanford Junior University</td><td class="patent-data-table-td ">Chimeric receptors by DNA splicing and expression</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP0239400A2?cl=en">EP0239400A2</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Mar 26, 1987</td><td class="patent-data-table-td patent-date-value">Sep 30, 1987</td><td class="patent-data-table-td ">Medical Research Council</td><td class="patent-data-table-td ">Recombinant antibodies and methods for their production</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP0244221A1?cl=en">EP0244221A1</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Apr 29, 1987</td><td class="patent-data-table-td patent-date-value">Nov 4, 1987</td><td class="patent-data-table-td ">Genentech, Inc.</td><td class="patent-data-table-td ">Hybrid receptors, nucleic acid encoding them, their preparation, and their use in determination of ligands and their antagonists and agonists</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP0256654A2?cl=en">EP0256654A2</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jul 7, 1987</td><td class="patent-data-table-td patent-date-value">Feb 24, 1988</td><td class="patent-data-table-td ">Centocor, Inc.</td><td class="patent-data-table-td ">Chimeric murine/human immunoglobulin specific for tumour-associated 17-1A Antigen</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP0315062A2?cl=en">EP0315062A2</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Oct 27, 1988</td><td class="patent-data-table-td patent-date-value">May 10, 1989</td><td class="patent-data-table-td ">Bristol-Myers Squibb Company</td><td class="patent-data-table-td ">Production of chimeric antibodies by homologous recombination</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP0327369A2?cl=en">EP0327369A2</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Feb 2, 1989</td><td class="patent-data-table-td patent-date-value">Aug 9, 1989</td><td class="patent-data-table-td ">The Regents Of The University Of California</td><td class="patent-data-table-td ">Human plalelet-derived growth factor receptor</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1986001533A1?cl=en">WO1986001533A1</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Sep 3, 1985</td><td class="patent-data-table-td patent-date-value">Mar 13, 1986</td><td class="patent-data-table-td ">Celltech Ltd</td><td class="patent-data-table-td ">Production of chimeric antibodies</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1990006953A2?cl=en">WO1990006953A2</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Dec 20, 1989</td><td class="patent-data-table-td patent-date-value">Jun 28, 1990</td><td class="patent-data-table-td ">Genentech Inc</td><td class="patent-data-table-td ">Method for preparing water soluble polypeptides</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="npl-citations"></a><div class="patent-section-header"><span class="patent-section-title">Non-Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th colspan="3"class="patent-data-table-th">Reference</th></tr></thead><tr><td class="patent-data-table-td ">1</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Alber and Kawasaki, J. Mol. Appl. Genet. 1 : 419 434, 1982.</td></tr><tr><td class="patent-data-table-td ">2</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Alber and Kawasaki, J. Mol. Appl. Genet. 1: 419-434, 1982.</td></tr><tr><td class="patent-data-table-td ">3</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Bailon et al., Bio/Technology 5 :1195 1198, 1987.</td></tr><tr><td class="patent-data-table-td ">4</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Bailon et al., Bio/Technology 5:1195-1198, 1987.</td></tr><tr><td class="patent-data-table-td ">5</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Ballou et al., J. Biol. Chem. 255 :5986 5991, 1980.</td></tr><tr><td class="patent-data-table-td ">6</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Ballou et al., J. Biol. Chem. 255:5986-5991, 1980.</td></tr><tr><td class="patent-data-table-td ">7</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Beggs, Nature : 104 108, 1978.</td></tr><tr><td class="patent-data-table-td ">8</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Beggs, Nature: 104-108, 1978.</td></tr><tr><td class="patent-data-table-td ">9</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Boni Schnetzler and Pilch, Proc. Natl. Acad. Sci. USA 84 :7832 7836, 1987.</td></tr><tr><td class="patent-data-table-td ">10</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Boni-Schnetzler and Pilch, Proc. Natl. Acad. Sci. USA 84:7832-7836, 1987.</td></tr><tr><td class="patent-data-table-td ">11</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Broach et al., Gene 8 :121 133, 1979.</td></tr><tr><td class="patent-data-table-td ">12</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Broach et al., Gene 8:121-133, 1979.</td></tr><tr><td class="patent-data-table-td ">13</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Capon et al., Nature 337 :525 531, 1989.</td></tr><tr><td class="patent-data-table-td ">14</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Capon et al., Nature 337:525-531, 1989.</td></tr><tr><td class="patent-data-table-td ">15</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Carlson et al., Mol. Cell. Biol. 3 :439 447, 1983.</td></tr><tr><td class="patent-data-table-td ">16</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Carlson et al., Mol. Cell. Biol. 3:439-447, 1983.</td></tr><tr><td class="patent-data-table-td ">17</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Claesson Welsh et al., Mol. Cell. Biol. 8:3476 3486, 1988.</td></tr><tr><td class="patent-data-table-td ">18</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Claesson-Welsh et al., Mol. Cell. Biol. 8:3476-3486, 1988.</td></tr><tr><td class="patent-data-table-td ">19</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Clarke et al., Meth. Enzym. 68 :436 442, 1979.</td></tr><tr><td class="patent-data-table-td ">20</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Clarke et al., Meth. Enzym. 68:436-442, 1979.</td></tr><tr><td class="patent-data-table-td ">21</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Derynck et al., Nature 316 :701 705, 1985.</td></tr><tr><td class="patent-data-table-td ">22</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Derynck et al., Nature 316:701-705, 1985.</td></tr><tr><td class="patent-data-table-td ">23</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Ebina et al., Cell 40 :747 758, 1985.</td></tr><tr><td class="patent-data-table-td ">24</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Ebina et al., Cell 40:747-758, 1985.</td></tr><tr><td class="patent-data-table-td ">25</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Ellis et al., J. Cell Biol. 150 :14a, 1987.</td></tr><tr><td class="patent-data-table-td ">26</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Ellis et al., J. Cell Biol. 150:14a, 1987.</td></tr><tr><td class="patent-data-table-td ">27</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Escobedo et al., Science 240 :1532 1534, 1988.</td></tr><tr><td class="patent-data-table-td ">28</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Escobedo et al., Science 240:1532-1534, 1988.</td></tr><tr><td class="patent-data-table-td ">29</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Gascoigne et al., Proc. Natl. Acad. Sci. USA 84 :2936 2940, 1987.</td></tr><tr><td class="patent-data-table-td ">30</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Gascoigne et al., Proc. Natl. Acad. Sci. USA 84:2936-2940, 1987.</td></tr><tr><td class="patent-data-table-td ">31</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Gronwald et al., Proc. Natl. Acad. Sci. USA 85 :3435 3439, 1988.</td></tr><tr><td class="patent-data-table-td ">32</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Gronwald et al., Proc. Natl. Acad. Sci. USA 85:3435-3439, 1988.</td></tr><tr><td class="patent-data-table-td ">33</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Grosschedl and Baltimore, Cell 41 : 885 897, 1985.</td></tr><tr><td class="patent-data-table-td ">34</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Grosschedl and Baltimore, Cell 41: 885-897, 1985.</td></tr><tr><td class="patent-data-table-td ">35</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Hart et al., Science 240 :1529 1531, 1988.</td></tr><tr><td class="patent-data-table-td ">36</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Hart et al., Science 240:1529-1531, 1988.</td></tr><tr><td class="patent-data-table-td ">37</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Heldin et al., Embo J. 7 :1387 1393, 1988.</td></tr><tr><td class="patent-data-table-td ">38</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Heldin et al., Embo J. 7:1387-1393, 1988.</td></tr><tr><td class="patent-data-table-td ">39</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Hieter et al., Cell 22 : 197 207, 1980.</td></tr><tr><td class="patent-data-table-td ">40</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Hieter et al., Cell 22: 197-207, 1980.</td></tr><tr><td class="patent-data-table-td ">41</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Honjo et al., Cell 18 :559 568, 1979.</td></tr><tr><td class="patent-data-table-td ">42</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Honjo et al., Cell 18:559-568, 1979.</td></tr><tr><td class="patent-data-table-td ">43</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Kuwana et al., Biochemical and Biophysical Research Communications 149 (3) : 960 968, 1987.</td></tr><tr><td class="patent-data-table-td ">44</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Kuwana et al., Biochemical and Biophysical Research Communications 149 (3) : 960-968, 1987.</td></tr><tr><td class="patent-data-table-td ">45</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Livneh et al., J. Biol. Chem. 261 : 12490 12497, 1986.</td></tr><tr><td class="patent-data-table-td ">46</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Livneh et al., J. Biol. Chem. 261: 12490-12497, 1986.</td></tr><tr><td class="patent-data-table-td ">47</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Loh et al., Cell 33 :85 93, 1983.</td></tr><tr><td class="patent-data-table-td ">48</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Loh et al., Cell 33:85-93, 1983.</td></tr><tr><td class="patent-data-table-td ">49</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Maddon et al., Cell 42 : 93 104, 1985.</td></tr><tr><td class="patent-data-table-td ">50</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Maddon et al., Cell 42: 93-104, 1985.</td></tr><tr><td class="patent-data-table-td ">51</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Maddon et al., Cell 47 : 333 348, 1986.</td></tr><tr><td class="patent-data-table-td ">52</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Maddon et al., Cell 47: 333-348, 1986.</td></tr><tr><td class="patent-data-table-td ">53</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Mariuzza and Winter, J. Biol. Chem. 264 :7310 7316, 1989.</td></tr><tr><td class="patent-data-table-td ">54</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Mariuzza and Winter, J. Biol. Chem. 264:7310-7316, 1989.</td></tr><tr><td class="patent-data-table-td ">55</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Marx, Science 229 :455 456, 1985.</td></tr><tr><td class="patent-data-table-td ">56</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Marx, Science 229:455-456, 1985.</td></tr><tr><td class="patent-data-table-td ">57</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Munro and Pelham, EMBO J. 3 :3087 3093, 1984.</td></tr><tr><td class="patent-data-table-td ">58</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Munro and Pelham, EMBO J. 3:3087-3093, 1984.</td></tr><tr><td class="patent-data-table-td ">59</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Neuberger et al., Nature 312 : 604 609, 1984.</td></tr><tr><td class="patent-data-table-td ">60</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Neuberger et al., Nature 312: 604-609, 1984.</td></tr><tr><td class="patent-data-table-td ">61</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Oi and Morrison Biotechniques 4 :214 221, 1986.</td></tr><tr><td class="patent-data-table-td ">62</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Oi and Morrison Biotechniques 4:214-221, 1986.</td></tr><tr><td class="patent-data-table-td ">63</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Radeke et al., Nature 325 593 597, 1987.</td></tr><tr><td class="patent-data-table-td ">64</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Radeke et al., Nature 325593-597, 1987.</td></tr><tr><td class="patent-data-table-td ">65</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Riedel et al., Nature 324 :68 70, 1986.</td></tr><tr><td class="patent-data-table-td ">66</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Riedel et al., Nature 324:68-70, 1986.</td></tr><tr><td class="patent-data-table-td ">67</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Riedel et al., Protein Engineering 1 (3):237, 1987.</td></tr><tr><td class="patent-data-table-td ">68</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Riedel et al., Protein Engineering 1(3):237, 1987.</td></tr><tr><td class="patent-data-table-td ">69</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Riedel et al., Science 236 :197 200, 1987.</td></tr><tr><td class="patent-data-table-td ">70</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Riedel et al., Science 236:197-200, 1987.</td></tr><tr><td class="patent-data-table-td ">71</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Roth et al., J. Cell. Biol. 102 :1271 1283, 1986.</td></tr><tr><td class="patent-data-table-td ">72</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Roth et al., J. Cell. Biol. 102:1271-1283, 1986.</td></tr><tr><td class="patent-data-table-td ">73</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Rubin et al., J. Immun. 135 :3172 3177, 1985.</td></tr><tr><td class="patent-data-table-td ">74</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Rubin et al., J. Immun. 135:3172-3177, 1985.</td></tr><tr><td class="patent-data-table-td ">75</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Russell et al., Nature 304 : 652 654, 1983.</td></tr><tr><td class="patent-data-table-td ">76</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Russell et al., Nature 304: 652-654, 1983.</td></tr><tr><td class="patent-data-table-td ">77</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Smith et al., Science 238 :1704 1707, 1987.</td></tr><tr><td class="patent-data-table-td ">78</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Smith et al., Science 238:1704-1707, 1987.</td></tr><tr><td class="patent-data-table-td ">79</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Struhl et al., Proc. Natl. Acad. Sci. USA 76 :1035 1039, 1979.</td></tr><tr><td class="patent-data-table-td ">80</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Struhl et al., Proc. Natl. Acad. Sci. USA 76:1035-1039, 1979.</td></tr><tr><td class="patent-data-table-td ">81</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Traunecker et al., Highly efficient neutralization of HIV with recombinant CD4 immunoglobulin molecules, Nature, 339: 68 70, May 4, 1989.</td></tr><tr><td class="patent-data-table-td ">82</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Traunecker et al., Highly efficient neutralization of HIV with recombinant CD4-immunoglobulin molecules, Nature, 339: 68-70, May 4, 1989.</td></tr><tr><td class="patent-data-table-td ">83</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Treiger et al., J. Immunol. 136 :4099 4105, 1986.</td></tr><tr><td class="patent-data-table-td ">84</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Treiger et al., J. Immunol. 136:4099-4105, 1986.</td></tr><tr><td class="patent-data-table-td ">85</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Ullrich et al., Nature 309 :418 425, 1984.</td></tr><tr><td class="patent-data-table-td ">86</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Ullrich et al., Nature 309:418-425, 1984.</td></tr><tr><td class="patent-data-table-td ">87</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Ullrich et al., Nature 313 : 756 761, 1985.</td></tr><tr><td class="patent-data-table-td ">88</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Ullrich et al., Nature 313: 756-761, 1985.</td></tr><tr><td class="patent-data-table-td ">89</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">von Heijne, Eur. J. Biochem. 133 :17 21, 1983.</td></tr><tr><td class="patent-data-table-td ">90</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">von Heijne, Eur. J. Biochem. 133:17-21, 1983.</td></tr><tr><td class="patent-data-table-td ">91</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">von Heijne, Nucl Acids Res. 14 : 4683 4690, 1986.</td></tr><tr><td class="patent-data-table-td ">92</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">von Heijne, Nucl Acids Res. 14: 4683-4690, 1986.</td></tr><tr><td class="patent-data-table-td ">93</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Wood et al., Nature 314 :446 449, 1985.</td></tr><tr><td class="patent-data-table-td ">94</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Wood et al., Nature 314:446-449, 1985.</td></tr><tr><td class="patent-data-table-td ">95</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Yarden et al., Nature 323 :226 232, 1986.</td></tr><tr><td class="patent-data-table-td ">96</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Yarden et al., Nature 323:226-232, 1986.</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="forward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Referenced by</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Citing Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6291646">US6291646</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">May 30, 2000</td><td class="patent-data-table-td patent-date-value">Sep 18, 2001</td><td class="patent-data-table-td ">Zymogenetics, Inc.</td><td class="patent-data-table-td ">A receptor polypeptide having a ligand-binding domain joined to an immunoglobulin constant region; therapy for atherosclerosis, autocrine tumors, fibroplasia and keloid formation</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6323323">US6323323</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Oct 25, 1999</td><td class="patent-data-table-td patent-date-value">Nov 27, 2001</td><td class="patent-data-table-td ">Zymogenetics, Inc.</td><td class="patent-data-table-td ">Ligand-binding, dimerized polypeptide fusions</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6537540">US6537540</a></td><td class="patent-data-table-td patent-date-value">May 26, 2000</td><td class="patent-data-table-td patent-date-value">Mar 25, 2003</td><td class="patent-data-table-td ">Targeted Genetics Corporation</td><td class="patent-data-table-td ">Administering recombinant adeno-associated virus vector; polynucleotide encoding fusion polypeptide with extracellular domain of the TNF receptor and a constant domain of an IgG1 molecule</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7189400">US7189400</a></td><td class="patent-data-table-td patent-date-value">Oct 4, 2001</td><td class="patent-data-table-td patent-date-value">Mar 13, 2007</td><td class="patent-data-table-td ">Genetics Institute, Llc</td><td class="patent-data-table-td ">Fusion protein for use in the treatment and prevention of autoimmune and cancer diseases</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7198789">US7198789</a></td><td class="patent-data-table-td patent-date-value">Oct 4, 2002</td><td class="patent-data-table-td patent-date-value">Apr 3, 2007</td><td class="patent-data-table-td ">Genetics Institute, Llc</td><td class="patent-data-table-td ">Methods and compositions for modulating interleukin-21 receptor activity</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7241593">US7241593</a></td><td class="patent-data-table-td patent-date-value">Feb 11, 2003</td><td class="patent-data-table-td patent-date-value">Jul 10, 2007</td><td class="patent-data-table-td ">Zymogenetics, Inc.</td><td class="patent-data-table-td ">Comprises fusion protein comprising platelet derived growth factor for treating connective tissue, tumor and bone disorders</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7300774">US7300774</a></td><td class="patent-data-table-td patent-date-value">Dec 9, 1999</td><td class="patent-data-table-td patent-date-value">Nov 27, 2007</td><td class="patent-data-table-td ">The Regents Of The University Of California</td><td class="patent-data-table-td ">Injected locally as a vaccine adjuvant or tumor immunotherapy agent to prevent them from diffusing away</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7314623">US7314623</a></td><td class="patent-data-table-td patent-date-value">Jul 15, 2003</td><td class="patent-data-table-td patent-date-value">Jan 1, 2008</td><td class="patent-data-table-td ">Wyeth</td><td class="patent-data-table-td ">Methods and compositions for modulating T helper (Th) cell development and function</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7332298">US7332298</a></td><td class="patent-data-table-td patent-date-value">Mar 22, 2005</td><td class="patent-data-table-td patent-date-value">Feb 19, 2008</td><td class="patent-data-table-td ">The Regents Of The University Of California</td><td class="patent-data-table-td ">Nucleic acids encoding multimeric proteins of TNF superfamily ligands</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7381794">US7381794</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Mar 8, 2005</td><td class="patent-data-table-td patent-date-value">Jun 3, 2008</td><td class="patent-data-table-td ">Zymogenetics, Inc.</td><td class="patent-data-table-td ">Dimeric fusion proteins and materials and methods for producing them</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7393662">US7393662</a></td><td class="patent-data-table-td patent-date-value">Sep 3, 2004</td><td class="patent-data-table-td patent-date-value">Jul 1, 2008</td><td class="patent-data-table-td ">Centocor, Inc.</td><td class="patent-data-table-td ">Human EPO mimetic hinge core mimetibodies, compositions, methods and uses</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7556941">US7556941</a></td><td class="patent-data-table-td patent-date-value">Oct 17, 2006</td><td class="patent-data-table-td patent-date-value">Jul 7, 2009</td><td class="patent-data-table-td ">Zymogenetics, Inc.</td><td class="patent-data-table-td ">Comprises fusion protein comprising platelet derived growth factor for treating connective tissue, tumor and bone disorders; used to stimulate the production of bone and/or connective tissue in both humans and non-human animals</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7705123">US7705123</a></td><td class="patent-data-table-td patent-date-value">Dec 17, 2007</td><td class="patent-data-table-td patent-date-value">Apr 27, 2010</td><td class="patent-data-table-td ">Genetics Institute, Llc</td><td class="patent-data-table-td ">Fusion protein contains antibody fragment, or Fc fragment; biodrug for use in the treatment and prevention of autoimmune and cancer diseases</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7718176">US7718176</a></td><td class="patent-data-table-td patent-date-value">Mar 27, 2009</td><td class="patent-data-table-td patent-date-value">May 18, 2010</td><td class="patent-data-table-td ">Cna Development Llc</td><td class="patent-data-table-td ">Human EPO mimetic hinge core mimetibodies, compositions, methods and uses</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7731946">US7731946</a></td><td class="patent-data-table-td patent-date-value">Nov 12, 2007</td><td class="patent-data-table-td patent-date-value">Jun 8, 2010</td><td class="patent-data-table-td ">Wyeth Llc</td><td class="patent-data-table-td ">Methods and compositions for modulating T helper (TH) cell development and function</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7736653">US7736653</a></td><td class="patent-data-table-td patent-date-value">Nov 13, 2004</td><td class="patent-data-table-td patent-date-value">Jun 15, 2010</td><td class="patent-data-table-td ">Hanmi Pharm. Co., Ltd</td><td class="patent-data-table-td ">Pharmaceutical composition comprising an immunoglobulin Fc region as a carrier</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7737260">US7737260</a></td><td class="patent-data-table-td patent-date-value">Nov 13, 2004</td><td class="patent-data-table-td patent-date-value">Jun 15, 2010</td><td class="patent-data-table-td ">Hanmi Pharm. Co., Ltd</td><td class="patent-data-table-td ">a protein conjugate comprising a physiologically active polypeptide, a non-peptide polymer and an immunoglobulin Fc fragment, with improved in vivo duration and stability; nonimmunogenic</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7763445">US7763445</a></td><td class="patent-data-table-td patent-date-value">Apr 21, 2008</td><td class="patent-data-table-td patent-date-value">Jul 27, 2010</td><td class="patent-data-table-td ">Zymogenetics, Inc.</td><td class="patent-data-table-td ">Dimeric fusion proteins and materials and methods for producing them</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7794718">US7794718</a></td><td class="patent-data-table-td patent-date-value">Oct 15, 2008</td><td class="patent-data-table-td patent-date-value">Sep 14, 2010</td><td class="patent-data-table-td ">Perseid Therapeutics, LLC</td><td class="patent-data-table-td ">fusion protein comprising polypeptide with a human immunoglobulin (Ig) polypeptide; has an ability to bind CD80 or CD86 or an extracellular domain of either CD80 or CD86, and/or an ability to inhibit a T cell immune response</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7901688">US7901688</a></td><td class="patent-data-table-td patent-date-value">Aug 3, 2005</td><td class="patent-data-table-td patent-date-value">Mar 8, 2011</td><td class="patent-data-table-td ">Transtech Pharma, Inc.</td><td class="patent-data-table-td ">Rage fusion proteins</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7932055">US7932055</a></td><td class="patent-data-table-td patent-date-value">Jun 22, 2007</td><td class="patent-data-table-td patent-date-value">Apr 26, 2011</td><td class="patent-data-table-td ">Novo Nordisk A/S</td><td class="patent-data-table-td ">Soluble heterodimeric CD94/NKG2 receptors fusion proteins</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7981423">US7981423</a></td><td class="patent-data-table-td patent-date-value">Aug 3, 2005</td><td class="patent-data-table-td patent-date-value">Jul 19, 2011</td><td class="patent-data-table-td ">Transtech Pharma, Inc.</td><td class="patent-data-table-td ">Rage fusion proteins</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7981424">US7981424</a></td><td class="patent-data-table-td patent-date-value">Apr 25, 2007</td><td class="patent-data-table-td patent-date-value">Jul 19, 2011</td><td class="patent-data-table-td ">Transtech Pharma, Inc.</td><td class="patent-data-table-td ">RAGE fusion proteins, formulations, and methods of use thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7994292">US7994292</a></td><td class="patent-data-table-td patent-date-value">Jan 28, 2010</td><td class="patent-data-table-td patent-date-value">Aug 9, 2011</td><td class="patent-data-table-td ">Genetics Institute, Llc</td><td class="patent-data-table-td ">MU-1, member of the cytokine receptor family</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8007805">US8007805</a></td><td class="patent-data-table-td patent-date-value">Aug 5, 2004</td><td class="patent-data-table-td patent-date-value">Aug 30, 2011</td><td class="patent-data-table-td ">Paladin Labs, Inc.</td><td class="patent-data-table-td ">Chimeric antigens for breaking host tolerance to foreign antigens</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8025873">US8025873</a></td><td class="patent-data-table-td patent-date-value">Aug 5, 2004</td><td class="patent-data-table-td patent-date-value">Sep 27, 2011</td><td class="patent-data-table-td ">Paladin Labs, Inc.</td><td class="patent-data-table-td ">Chimeric antigens for eliciting an immune response</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8029803">US8029803</a></td><td class="patent-data-table-td patent-date-value">Feb 13, 2003</td><td class="patent-data-table-td patent-date-value">Oct 4, 2011</td><td class="patent-data-table-td ">Paladin Labs, Inc.</td><td class="patent-data-table-td ">Chimeric antigens for eliciting an immune response</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8039589">US8039589</a></td><td class="patent-data-table-td patent-date-value">Oct 31, 2007</td><td class="patent-data-table-td patent-date-value">Oct 18, 2011</td><td class="patent-data-table-td ">Mayo Foundation For Medical Education And Research</td><td class="patent-data-table-td ">B7-DC variants</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8053414">US8053414</a></td><td class="patent-data-table-td patent-date-value">Oct 31, 2007</td><td class="patent-data-table-td patent-date-value">Nov 8, 2011</td><td class="patent-data-table-td ">The Johns Hopkins University</td><td class="patent-data-table-td ">Methods of using B7-DC molecules to induce or enhance an immune response</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8053558">US8053558</a></td><td class="patent-data-table-td patent-date-value">Oct 31, 2007</td><td class="patent-data-table-td patent-date-value">Nov 8, 2011</td><td class="patent-data-table-td ">The Johns Hopkins University</td><td class="patent-data-table-td ">Dendritic cell co-stimulatory molecules</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8071095">US8071095</a></td><td class="patent-data-table-td patent-date-value">Apr 20, 2010</td><td class="patent-data-table-td patent-date-value">Dec 6, 2011</td><td class="patent-data-table-td ">Perseid Therapeutics, LLC</td><td class="patent-data-table-td ">Immunosuppressive polypeptides and nucleic acids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8110665">US8110665</a></td><td class="patent-data-table-td patent-date-value">Apr 14, 2010</td><td class="patent-data-table-td patent-date-value">Feb 7, 2012</td><td class="patent-data-table-td ">Hanmi Holdings Co., Ltd.</td><td class="patent-data-table-td ">Pharmaceutical composition comprising an immunoglobulin FC region as a carrier</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8153595">US8153595</a></td><td class="patent-data-table-td patent-date-value">Jul 11, 2008</td><td class="patent-data-table-td patent-date-value">Apr 10, 2012</td><td class="patent-data-table-td ">The Johns Hopkins University</td><td class="patent-data-table-td ">B7-DC variants immunogenic compositions and methods of use thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8268587">US8268587</a></td><td class="patent-data-table-td patent-date-value">Jan 21, 2011</td><td class="patent-data-table-td patent-date-value">Sep 18, 2012</td><td class="patent-data-table-td ">Perseid Therapeutics Llc</td><td class="patent-data-table-td ">Immunosuppressive polypeptides and nucleic acids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8273864">US8273864</a></td><td class="patent-data-table-td patent-date-value">Sep 9, 2011</td><td class="patent-data-table-td patent-date-value">Sep 25, 2012</td><td class="patent-data-table-td ">Mayo Foundation For Medical Education And Research</td><td class="patent-data-table-td ">Nucleic acid molecules encoding B7-DC variants</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8283447">US8283447</a></td><td class="patent-data-table-td patent-date-value">Jan 21, 2011</td><td class="patent-data-table-td patent-date-value">Oct 9, 2012</td><td class="patent-data-table-td ">Perseid Therapeutics Llc</td><td class="patent-data-table-td ">Immunosuppressive polypeptides and nucleic acids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8318176">US8318176</a></td><td class="patent-data-table-td patent-date-value">Jan 21, 2011</td><td class="patent-data-table-td patent-date-value">Nov 27, 2012</td><td class="patent-data-table-td ">Perseid Therapeutics Llc</td><td class="patent-data-table-td ">Immunosuppressive polypeptides and nucleic acids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8344120">US8344120</a></td><td class="patent-data-table-td patent-date-value">Jun 13, 2011</td><td class="patent-data-table-td patent-date-value">Jan 1, 2013</td><td class="patent-data-table-td ">Transtech Pharma, Inc.</td><td class="patent-data-table-td ">Nucleic acid molecules encoding rage fusion proteins</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8377448">US8377448</a></td><td class="patent-data-table-td patent-date-value">Jan 14, 2010</td><td class="patent-data-table-td patent-date-value">Feb 19, 2013</td><td class="patent-data-table-td ">The Board Of Trustees Of The Leland Standford Junior University</td><td class="patent-data-table-td ">CD47 related compositions and methods for treating immunological diseases and disorders</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8445230">US8445230</a></td><td class="patent-data-table-td patent-date-value">Jan 21, 2011</td><td class="patent-data-table-td patent-date-value">May 21, 2013</td><td class="patent-data-table-td ">Perseid Therapeutics Llc</td><td class="patent-data-table-td ">Immunosuppressive polypeptides and nucleic acids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8445447">US8445447</a></td><td class="patent-data-table-td patent-date-value">Mar 7, 2012</td><td class="patent-data-table-td patent-date-value">May 21, 2013</td><td class="patent-data-table-td ">The Johns Hopkins University</td><td class="patent-data-table-td ">B7-DC variants immunogenic compositions and methods of use thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8460927">US8460927</a></td><td class="patent-data-table-td patent-date-value">Oct 25, 2011</td><td class="patent-data-table-td patent-date-value">Jun 11, 2013</td><td class="patent-data-table-td ">Mayo Foundation For Medical Education And Research</td><td class="patent-data-table-td ">B7-H1 antibodies and method of use</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8465745">US8465745</a></td><td class="patent-data-table-td patent-date-value">Aug 15, 2011</td><td class="patent-data-table-td patent-date-value">Jun 18, 2013</td><td class="patent-data-table-td ">Akshaya Bio Inc.</td><td class="patent-data-table-td ">Chimeric antigens for eliciting an immune response</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8491899">US8491899</a></td><td class="patent-data-table-td patent-date-value">Mar 10, 2011</td><td class="patent-data-table-td patent-date-value">Jul 23, 2013</td><td class="patent-data-table-td ">Perseid Therapeutics Llc</td><td class="patent-data-table-td ">Immunosuppressive polypeptides and nucleic acids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8496935">US8496935</a></td><td class="patent-data-table-td patent-date-value">Mar 16, 2011</td><td class="patent-data-table-td patent-date-value">Jul 30, 2013</td><td class="patent-data-table-td ">Perseid Therapeutics Llc</td><td class="patent-data-table-td ">Immunosuppressive polypeptides and nucleic acids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8709416">US8709416</a></td><td class="patent-data-table-td patent-date-value">Dec 20, 2011</td><td class="patent-data-table-td patent-date-value">Apr 29, 2014</td><td class="patent-data-table-td ">Amplimmune, Inc.</td><td class="patent-data-table-td ">Compositions of PD-1 antagonists and methods of use</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8747833">US8747833</a></td><td class="patent-data-table-td patent-date-value">Jan 24, 2011</td><td class="patent-data-table-td patent-date-value">Jun 10, 2014</td><td class="patent-data-table-td ">Mayo Foundation For Medical Education And Research</td><td class="patent-data-table-td ">B7-H1 and methods of diagnosis, prognosis, and treatment of cancer</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2042517A1?cl=en">EP2042517A1</a></td><td class="patent-data-table-td patent-date-value">Sep 26, 2003</td><td class="patent-data-table-td patent-date-value">Apr 1, 2009</td><td class="patent-data-table-td ">Xencor, Inc.</td><td class="patent-data-table-td ">Optimized FC variants and methods for their generation</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2192132A2?cl=en">EP2192132A2</a></td><td class="patent-data-table-td patent-date-value">Feb 8, 2006</td><td class="patent-data-table-td patent-date-value">Jun 2, 2010</td><td class="patent-data-table-td ">ZymoGenetics, Inc.</td><td class="patent-data-table-td ">Anti-IL-20, anti-IL22 and anti-IL-22RA antibodies and binding partners and methods of using in inflammation</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2275816A2?cl=en">EP2275816A2</a></td><td class="patent-data-table-td patent-date-value">Mar 22, 2007</td><td class="patent-data-table-td patent-date-value">Jan 19, 2011</td><td class="patent-data-table-td ">Viral Logic Systems Technology Corp.</td><td class="patent-data-table-td ">Methods for identifying polypeptide targets and uses thereof for treating immunological diseases</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2298805A2?cl=en">EP2298805A2</a></td><td class="patent-data-table-td patent-date-value">Sep 26, 2003</td><td class="patent-data-table-td patent-date-value">Mar 23, 2011</td><td class="patent-data-table-td ">Xencor, Inc.</td><td class="patent-data-table-td ">Optimized Fc variants and methods for their generation</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2322931A2?cl=en">EP2322931A2</a></td><td class="patent-data-table-td patent-date-value">Mar 22, 2007</td><td class="patent-data-table-td patent-date-value">May 18, 2011</td><td class="patent-data-table-td ">Viral Logic Systems Technology Corp.</td><td class="patent-data-table-td ">Methods for identifying polypeptide targets and uses thereof for treating immunological diseases</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2327423A2?cl=en">EP2327423A2</a></td><td class="patent-data-table-td patent-date-value">Feb 21, 2007</td><td class="patent-data-table-td patent-date-value">Jun 1, 2011</td><td class="patent-data-table-td ">Wyeth LLC</td><td class="patent-data-table-td ">Human antibodies against human interleukin-22 (IL-22)</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2345671A1?cl=en">EP2345671A1</a></td><td class="patent-data-table-td patent-date-value">Sep 26, 2003</td><td class="patent-data-table-td patent-date-value">Jul 20, 2011</td><td class="patent-data-table-td ">Xencor Inc.</td><td class="patent-data-table-td ">Optimized fc variants and methods for their generation</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2364996A1?cl=en">EP2364996A1</a></td><td class="patent-data-table-td patent-date-value">Sep 26, 2003</td><td class="patent-data-table-td patent-date-value">Sep 14, 2011</td><td class="patent-data-table-td ">Xencor Inc.</td><td class="patent-data-table-td ">Optimized FC variants and methods for their generation</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2368911A1?cl=en">EP2368911A1</a></td><td class="patent-data-table-td patent-date-value">Mar 26, 2004</td><td class="patent-data-table-td patent-date-value">Sep 28, 2011</td><td class="patent-data-table-td ">Xencor Inc.</td><td class="patent-data-table-td ">Optimized Fc variants and methods for their generation</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2397493A1?cl=en">EP2397493A1</a></td><td class="patent-data-table-td patent-date-value">May 12, 2006</td><td class="patent-data-table-td patent-date-value">Dec 21, 2011</td><td class="patent-data-table-td ">ZymoGenetics, Inc.</td><td class="patent-data-table-td ">Compositions and methods for modulating immune responses</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2399932A1?cl=en">EP2399932A1</a></td><td class="patent-data-table-td patent-date-value">May 12, 2006</td><td class="patent-data-table-td patent-date-value">Dec 28, 2011</td><td class="patent-data-table-td ">ZymoGenetics, Inc.</td><td class="patent-data-table-td ">Compositions and methods for modulating immune responses</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2431392A1?cl=en">EP2431392A1</a></td><td class="patent-data-table-td patent-date-value">Feb 21, 2007</td><td class="patent-data-table-td patent-date-value">Mar 21, 2012</td><td class="patent-data-table-td ">Wyeth LLC</td><td class="patent-data-table-td ">Antibodies against human IL-22 and uses therefor</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2623112A1?cl=en">EP2623112A1</a></td><td class="patent-data-table-td patent-date-value">Jun 29, 2009</td><td class="patent-data-table-td patent-date-value">Aug 7, 2013</td><td class="patent-data-table-td ">Zymogenetics, Inc.</td><td class="patent-data-table-td ">Soluble hybrid Fc gamma receptors and related methods</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2001040467A1?cl=en">WO2001040467A1</a></td><td class="patent-data-table-td patent-date-value">Dec 1, 2000</td><td class="patent-data-table-td patent-date-value">Jun 7, 2001</td><td class="patent-data-table-td ">Zymogenetics Inc</td><td class="patent-data-table-td ">Human cytokine receptor</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2003068802A2?cl=en">WO2003068802A2</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Feb 11, 2003</td><td class="patent-data-table-td patent-date-value">Aug 21, 2003</td><td class="patent-data-table-td ">Zymogenetics Inc</td><td class="patent-data-table-td ">Materials and methods for preparing dimeric growth factors</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2007147898A1?cl=en">WO2007147898A1</a></td><td class="patent-data-table-td patent-date-value">Jun 22, 2007</td><td class="patent-data-table-td patent-date-value">Dec 27, 2007</td><td class="patent-data-table-td ">Novo Nordisk As</td><td class="patent-data-table-td ">Soluble heterodimeric receptors and uses thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2009046407A2?cl=en">WO2009046407A2</a></td><td class="patent-data-table-td patent-date-value">Oct 6, 2008</td><td class="patent-data-table-td patent-date-value">Apr 9, 2009</td><td class="patent-data-table-td ">Zymogenetics Inc</td><td class="patent-data-table-td ">B7 FAMILY MEMBER zB7H6 AND RELATED COMPOSITIONS AND METHODS</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2010039533A2?cl=en">WO2010039533A2</a></td><td class="patent-data-table-td patent-date-value">Sep 23, 2009</td><td class="patent-data-table-td patent-date-value">Apr 8, 2010</td><td class="patent-data-table-td ">Wyeth</td><td class="patent-data-table-td ">Methods for predicting production of activating signals by cross-linked binding proteins</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2011130533A1?cl=en">WO2011130533A1</a></td><td class="patent-data-table-td patent-date-value">Apr 14, 2011</td><td class="patent-data-table-td patent-date-value">Oct 20, 2011</td><td class="patent-data-table-td ">Centocor Ortho Biotech Inc.</td><td class="patent-data-table-td ">Engineered plant cysteine proteases and their uses</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="classifications"></a><div class="patent-section-header"><span class="patent-section-title">Classifications</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th"> </th><th class="patent-data-table-th"> </th></tr></thead><tr><td class="patent-data-table-td ">U.S. Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=Z-1NBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc530/defs530.htm&usg=AFQjCNEOd3srpZKCSH0W93D5fiJ4_0S25w#C530S350000">530/350</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=Z-1NBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc530/defs530.htm&usg=AFQjCNEOd3srpZKCSH0W93D5fiJ4_0S25w#C530S388240">530/388.24</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=Z-1NBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc530/defs530.htm&usg=AFQjCNEOd3srpZKCSH0W93D5fiJ4_0S25w#C530S402000">530/402</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=Z-1NBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc530/defs530.htm&usg=AFQjCNEOd3srpZKCSH0W93D5fiJ4_0S25w#C530S387300">530/387.3</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=Z-1NBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc530/defs530.htm&usg=AFQjCNEOd3srpZKCSH0W93D5fiJ4_0S25w#C530S399000">530/399</a></span></td></tr><tr><td class="patent-data-table-td ">International Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=Z-1NBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0015810000">C12N15/81</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=Z-1NBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0015090000">C12N15/09</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=Z-1NBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C07K0014490000">C07K14/49</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=Z-1NBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C07K0014720000">C07K14/72</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=Z-1NBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C07K0014705000">C07K14/705</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=Z-1NBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C07K0001000000">C07K1/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=Z-1NBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12P0021040000">C12P21/04</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=Z-1NBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0015620000">C12N15/62</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=Z-1NBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C07K0019000000">C07K19/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=Z-1NBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C07K0016460000">C07K16/46</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=Z-1NBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C07K0014710000">C07K14/71</a></span></td></tr><tr><td class="patent-data-table-td ">Cooperative Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=Z-1NBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C07K2317/52">C07K2317/52</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=Z-1NBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C07K14/72">C07K14/72</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=Z-1NBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C07K2319/30">C07K2319/30</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=Z-1NBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C07K2317/56">C07K2317/56</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=Z-1NBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C07K14/71">C07K14/71</a></span></td></tr><tr><td class="patent-data-table-td ">European Classification</td><td class="patent-data-table-td "><span class="nested-value">C07K14/71</span>, <span class="nested-value">C07K14/72</span></td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="legal-events"></a><div class="patent-section-header"><span class="patent-section-title">Legal Events</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Date</th><th class="patent-data-table-th">Code</th><th class="patent-data-table-th">Event</th><th class="patent-data-table-th">Description</th></tr></thead><tr><td class="patent-data-table-td patent-date-value">Mar 14, 2012</td><td class="patent-data-table-td ">FP</td><td class="patent-data-table-td ">Expired due to failure to pay maintenance fee</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20120125</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Jan 25, 2012</td><td class="patent-data-table-td ">LAPS</td><td class="patent-data-table-td ">Lapse for failure to pay maintenance fees</td><td class="patent-data-table-td "></td></tr><tr><td class="patent-data-table-td patent-date-value">Aug 29, 2011</td><td class="patent-data-table-td ">REMI</td><td class="patent-data-table-td ">Maintenance fee reminder mailed</td><td class="patent-data-table-td "></td></tr><tr><td class="patent-data-table-td patent-date-value">Jul 25, 2007</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">8</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Jul 4, 2006</td><td class="patent-data-table-td ">B1</td><td class="patent-data-table-td ">Reexamination certificate first reexamination</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">THE PATENTABILITY OF CLAIMS 9-11 AND 13-29 IS CONFIRMED. CLAIMS 1, 7 AND 12 ARE DETERMINED TO BE PATENTABLE AS AMENDED. CLAIMS 2-6 AND 8, DEPENDENT ON AN AMENDED CLAIM, ARE DETERMINED TO BE PATENTABLE. NEW CLAIM 30 IS ADDED AND DETERMINED TO BE PATENTABLE.</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Apr 27, 2004</td><td class="patent-data-table-td ">RR</td><td class="patent-data-table-td ">Request for reexamination filed</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20040316</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Jul 25, 2003</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">4</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Jul 30, 2002</td><td class="patent-data-table-td ">CC</td><td class="patent-data-table-td ">Certificate of correction</td><td class="patent-data-table-td "></td></tr></table><div class="patent-section-footer"></div></div><div class="modal-dialog" id="patent-images-lightbox"><div class="patent-lightbox-controls"><div class="patent-lightbox-rotate-controls"><div class="patent-lightbox-rotation-text">Rotate</div><div class="rotate-icon rotate-ccw-icon"></div><div class="rotate-icon rotate-cw-icon"></div></div><div class="patent-lightbox-index-counter"></div><a class="patent-lightbox-fullsize-link" target="_blank">Original Image</a><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_right.png" alt="Next page"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_left.png" alt="Previous page"width="21" height="21" /></div></div><div class="modal-dialog-content"><div class="patent-lightbox-image-holder"><div class="patent-lightbox-placeholder"></div></div></div></div><script>_OC_initPatentsAtb({image_not_available_html: " Image not available"});</script></div></div></div></td></tr></table><script>(function() {var href = window.location.href;if (href.indexOf('?') !== -1) {var parameters = href.split('?')[1].split('&');for (var i = 0; i < parameters.length; i++) {var param = parameters[i].split('=');if (param[0] == 'focus') {var elem = document.getElementById(param[1]);if (elem) {elem.focus();}}}}})();</script><script>_OC_addFlags({LockSrc:"/books/javascript/lock_8a2b04e7bf975d5171d8e4c0b6365c7a.js", Host:"http://www.google.com/", IsBooksRentalEnabled:1, IsWebstoreDisplayCaseEnabled:1, IsObfuscationEnabled:1, IsBrowsingHistoryEnabled:1, IsWebReaderSvgEnabled:0, IsGeoLayerEnabled:1, IsImageModeNotesEnabled:1, IsCopyMenuItemEnabled:1, IsGiftingEnabled:0, IsWebReaderUniversalPaginatorEnabled:0, IsOfflineBubbleEnabled:1, IsReaderEnabledForPlayRequests:1, IsFutureOnSaleVolumesEnabled:1, IsOfflineRestrictedCopyEnabled:1, IsBooksUnifiedLeftNavEnabled:1, IsRestrictedCopyEnabled:1, IsZipitFolderCollectionEnabled:1, IsEndOfSampleRecommendationsEnabled:1, IsRatingsOnBookcardsEnabled:1, IsAdsDisabled:0, IsIframePageDisplayEnabled:0, IsEmbeddedMediaEnabled:1, IsImageModeAnnotationsEnabled:1, IsMyLibraryGooglePlusEnabled:1, IsImagePageProviderEnabled:0, IsBookcardListPriceSmall:0, IsInternalUser:0, IsBooksShareButtonEnabled:0, IsPreOrdersEnabled:0, IsDisabledRandomBookshelves:0, WebstoreDisplayCasePosition:3});_OC_Run({"enable_p13n":false,"add_vol_to_collection_base_url":"http://www.google.com/patents?op=add\u0026sig=ACfU3U0OZKaNc1Ib875pugyeVbk_9YBYqg\u0026id=Z-1NBAABERAJ","remove_vol_from_collection_base_url":"http://www.google.com/patents?op=remove\u0026sig=ACfU3U0f1oJAxaaK_BdS3nq06RYe9dSDUQ\u0026id=Z-1NBAABERAJ","logged_in":false,"p13n_save_user_settings_url":"http://www.google.com/patents?op=edit_user_settings\u0026sig=ACfU3U18cZBthyPcUIV2Oeo0u_WnkCOkiA","is_cobrand":false,"sign_in_url":"https://www.google.com/accounts/Login?service=\u0026continue=http://www.google.com/patents%3Fhl%3Den\u0026hl=en","is_play_enabled":true}, {"volume_id":"","is_ebook":true,"volumeresult":{"has_flowing_text":false,"has_scanned_text":true,"can_download_pdf":false,"can_download_epub":false,"is_pdf_drm_enabled":false,"is_epub_drm_enabled":false,"download_pdf_url":"http://www.google.com/patents/download/For_rapid_inexpensive_and_adaptable_high.pdf?id=Z-1NBAABERAJ\u0026output=pdf\u0026sig=ACfU3U3G1TQW0mSpo3OR-E9MBdghRopuhA"},"sample_url":"http://www.google.com/patents/reader?id=Z-1NBAABERAJ\u0026printsec=frontcover\u0026output=reader\u0026source=gbs_atb_hover","is_browsable":true,"is_public_domain":true}, {});</script><div id="footer_table" style="font-size:83%;text-align:center;position:relative;top:20px;height:4.5em;margin-top:2em"><div style="margin-bottom:8px"><a href=http://www.google.com/><nobr>Google&nbsp;Home</nobr></a> - <a href=//www.google.com/patents/sitemap/><nobr>Sitemap</nobr></a> - <a href=http://www.google.com/googlebooks/uspto.html><nobr>USPTO Bulk Downloads</nobr></a> - <a href=/intl/en/privacy/><nobr>Privacy Policy</nobr></a> - <a href=/intl/en/policies/terms/><nobr>Terms of Service</nobr></a> - <a href=https://support.google.com/faqs/answer/2539193?hl=en><nobr>About Google Patents</nobr></a> - <a href="http://www.google.com/tools/feedback/intl/en/error.html" onclick="try{_OC_startFeedback({productId: '72792',locale: 'en'});return false;}catch(e){}"><nobr>Send Feedback</nobr></a></div><span>Data provided by IFI CLAIMS Patent Services</span><br><span >&copy;2012 Google</span></div> <script type="text/javascript">var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));</script><script type="text/javascript">var pageTracker = _gat._getTracker("UA-27188110-1");pageTracker._setCookiePath("/patents/");pageTracker._trackPageview();</script> </body></html>