<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN""http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html><head><title>Patent US6103490 - Heterologous polypeptides expressed in filamentous fungi, processes for ... - Google Patents</title><script>(function(){(function(){function e(a){this.t={};this.tick=function(a,c,b){var d=void 0!=b?b:(new Date).getTime();this.t[a]=[d,c];if(void 0==b)try{window.console.timeStamp("CSI/"+a)}catch(e){}};this.tick("start",null,a)}var a;window.performance&&(a=window.performance.timing);var f=a?new e(a.responseStart):new e;window.jstiming={Timer:e,load:f};if(a){var c=a.navigationStart,d=a.responseStart;0<c&&d>=c&&(window.jstiming.srt=d-c)}if(a){var b=window.jstiming.load;0<c&&d>=c&&(b.tick("_wtsrt",void 0,c),b.tick("wtsrt_",
"_wtsrt",d),b.tick("tbsd_","wtsrt_"))}try{a=null,window.chrome&&window.chrome.csi&&(a=Math.floor(window.chrome.csi().pageT),b&&0<c&&(b.tick("_tbnd",void 0,window.chrome.csi().startE),b.tick("tbnd_","_tbnd",c))),null==a&&window.gtbExternal&&(a=window.gtbExternal.pageT()),null==a&&window.external&&(a=window.external.pageT,b&&0<c&&(b.tick("_tbnd",void 0,window.external.startE),b.tick("tbnd_","_tbnd",c))),a&&(window.jstiming.pt=a)}catch(g){}})();})();
</script><link rel="stylesheet" href="/patents/css/_8a2b04e7bf975d5171d8e4c0b6365c7a/kl_intl_patents_bundle.css" type="text/css" /><script src="/books/javascript/atb_8a2b04e7bf975d5171d8e4c0b6365c7a__en.js"></script><script>function googleTranslateElementInit() {new google.translate.TranslateElement({pageLanguage: "en",gaTrack: true,gaId: "UA-27188110-1",multilanguagePage: true});}</script><script src="//translate.google.com/translate_a/element.js?cb=googleTranslateElementInit"></script><meta name="DC.type" content="Patent"><meta name="DC.title" content="Heterologous polypeptides expressed in filamentous fungi, processes for making same, and vectors for making same"><meta name="DC.contributor" content="Randy Michael Berka" scheme="inventor"><meta name="DC.contributor" content="Daniel Cullen" scheme="inventor"><meta name="DC.contributor" content="Gregory Lawrence Gray" scheme="inventor"><meta name="DC.contributor" content="Kirk James Hayenga" scheme="inventor"><meta name="DC.contributor" content="Virgil Bryan Lawlis" scheme="inventor"><meta name="DC.contributor" content="Genencor International, Inc." scheme="assignee"><meta name="DC.date" content="1994-6-17" scheme="dateSubmitted"><meta name="DC.description" content="Novel vectors are disclosed for expressing and secreting heterologous polypeptides from filamentous fungi. Such vectors are used in novel processes to express and secrete such heterologous polypeptides. The vectors used for transforming a filamentous fungus to express and secrete a heterologous polypeptide include a DNA sequence encoding a heterologous polypeptide and a DNA sequence encoding a signal sequence which is functional in a secretory system in a given filamentous fungus and which is operably linked to the sequence encoding the heterologous polypeptide. Such signal sequences may be the signal sequence normally associated with the heterologous polypeptides or may be derived from other sources. The vector may also contain DNA sequences encoding a promoter sequence which is functionally recognized by the filamentous fungus and which is operably linked to the DNA sequence encoding the signal sequence. Preferably functional polyadenylation sequences are operably linked to the 3&#39; terminus of the DNA sequence encoding the heterologous polypeptides. Each of the above described vectors are used in novel processes to transform a filamentous fungus wherein the DNA sequences encoding the signal sequence and heterologous polypeptide are expressed. The thus synthesized polypeptide is thereafter secreted from the filamentous fungus."><meta name="DC.date" content="2000-8-15" scheme="issued"><meta name="DC.relation" content="EP:0077109:A2" scheme="references"><meta name="DC.relation" content="EP:0126206:A2" scheme="references"><meta name="DC.relation" content="EP:0184438:A2" scheme="references"><meta name="DC.relation" content="US:4486533" scheme="references"><meta name="DC.relation" content="US:4666847" scheme="references"><meta name="DC.relation" content="US:4794175" scheme="references"><meta name="DC.relation" content="US:4816405" scheme="references"><meta name="DC.relation" content="US:4863864" scheme="references"><meta name="DC.relation" content="US:5198345" scheme="references"><meta name="DC.relation" content="WO:1986003774:A1" scheme="references"><meta name="DC.relation" content="WO:1986006097:A1" scheme="references"><meta name="citation_reference" content="Akins, R. A., et al., &quot;General Method for Cloning Neurospora crassa Nuclear GEnes by Complementation of Mutants&quot;, Molecular and Cellular Biology, 5(9):2272-2278 (1985)."><meta name="citation_reference" content="Akins, R. A., et al., General Method for Cloning Neurospora crassa Nuclear GEnes by Complementation of Mutants , Molecular and Cellular Biology, 5(9):2272 2278 (1985)."><meta name="citation_reference" content="Alton, N. K., et al., &quot;5&#39;-Untranslated sequences of two structural genes in the qa gene cluster of Neurospora crassa&quot;, Proc. Natl. Acad. Sci. USA, 79:1655-1959 (1982)."><meta name="citation_reference" content="Alton, N. K., et al., 5 Untranslated sequences of two structural genes in the qa gene cluster of Neurospora crassa , Proc. Natl. Acad. Sci. USA, 79:1655 1959 (1982)."><meta name="citation_reference" content="Arends, H., et al., &quot;Nucleotide sequence of the cloned mRNA and gene of the ADP/ATP carrier from Neurospora crassa&quot;, The EMBO Journal, 3(2):377-382 (1984)."><meta name="citation_reference" content="Arends, H., et al., Nucleotide sequence of the cloned mRNA and gene of the ADP/ATP carrier from Neurospora crassa , The EMBO Journal, 3(2):377 382 (1984)."><meta name="citation_reference" content="Ballance, D. J., et al., &quot;Development of a high-frequency transforming vector for Aspergillus nidulans&quot;, Gene, 36:321-331 (1985)."><meta name="citation_reference" content="Ballance, D. J., et al., &quot;Gene cloning in Aspergillus nidulans:isolation of the isocitrate lyase gene (acuD)&quot;, Mol. Gen. Genet., 202:271-275 (1986)."><meta name="citation_reference" content="Ballance, D. J., et al., &quot;Transformation of Aspergillus nidulans By The Orotidine-5&#39;-Phosphate Decarboxylase Gene&quot;, Biochemical and Biophysical Research Communications, 112(1):284-289 (1983)."><meta name="citation_reference" content="Ballance, D. J., et al., Development of a high frequency transforming vector for Aspergillus nidulans , Gene, 36:321 331 (1985)."><meta name="citation_reference" content="Ballance, D. J., et al., Gene cloning in Aspergillus nidulans: isolation of the isocitrate lyase gene (acuD) , Mol. Gen. Genet., 202:271 275 (1986)."><meta name="citation_reference" content="Ballance, D. J., et al., Transformation of Aspergillus nidulans By The Orotidine 5 Phosphate Decarboxylase Gene , Biochemical and Biophysical Research Communications, 112(1):284 289 (1983)."><meta name="citation_reference" content="Ballance, D.J., et al., &quot;Development of a high-frequency transforming vector for Aspergillus nidulans&quot;, Gene 36:321-331 (1985)."><meta name="citation_reference" content="Ballance, D.J., et al., Development of a high frequency transforming vector for Aspergillus nidulans , Gene 36:321 331 (1985)."><meta name="citation_reference" content="Barnes, D. E., et al., &quot;Behaviour of recombinant plasmids in Aspergillus nidulans:structure and stability&quot;, Current Genetics, 10:767-775 (1986)."><meta name="citation_reference" content="Barnes, D. E., et al., Behaviour of recombinant plasmids in Aspergillus nidulans: structure and stability , Current Genetics, 10:767 775 (1986)."><meta name="citation_reference" content="Bech, A. M., et al., Partial primary structure of Mucor miehei protease , Neth. Milk Dairy J. 35:275 280 (1981)."><meta name="citation_reference" content="Bech, A.-M., et al., &quot;Partial primary structure of Mucor miehei protease&quot;, Neth. Milk Dairy J. 35:275-280 (1981)."><meta name="citation_reference" content="Beppu, T., &quot;The cloning and expression of chymosin (rennin) genes in microorganisms&quot;, Trends in Biotech., 1:85-89 (1983)."><meta name="citation_reference" content="Beppu, T., The cloning and expression of chymosin (rennin) genes in microorganisms , Trends in Biotech., 1:85 89 (1983)."><meta name="citation_reference" content="Berse, B., et al., &quot;Cloning and characterization of the ornithine carbamoyltransferase gene from Aspergillus nidulans&quot;, Gene, 25:109-117 (1983)."><meta name="citation_reference" content="Berse, B., et al., Cloning and characterization of the ornithine carbamoyltransferase gene from Aspergillus nidulans , Gene, 25:109 117 (1983)."><meta name="citation_reference" content="Bissett Can. J. Bot. (1984) 62:924 931."><meta name="citation_reference" content="Bissett Can. J. Bot. (1984) 62:924-931."><meta name="citation_reference" content="Boel, E., et al., &quot;Two different types of intervening sequences in the glucoamylase gene from Aspergillus niger&quot;, EMBO Journal, 3:1581-1585 (1984)."><meta name="citation_reference" content="Boel, E., et al., Two different types of intervening sequences in the glucoamylase gene from Aspergillus niger , EMBO Journal, 3:1581 1585 (1984)."><meta name="citation_reference" content="Briggs, M. S., et al., &quot;Molecular Mechanisms of Protein Secretion: The Role of the Signal Sequence&quot;, Advances in Protein Chemistry, 38:109-180 (1986)."><meta name="citation_reference" content="Briggs, M. S., et al., Molecular Mechanisms of Protein Secretion: The Role of the Signal Sequence , Advances in Protein Chemistry, 38:109 180 (1986)."><meta name="citation_reference" content="Bull, J. H., et al., Heavily methylated amplified DNA in transformants of Neurospora crassa, Nature, 310:701 704 (1984)."><meta name="citation_reference" content="Bull, J. H., et al., Heavily methylated amplified DNA in transformants of Neurospora crassa, Nature, 310:701-704 (1984)."><meta name="citation_reference" content="Buxton, F. P., et al., &quot;Cloning of the Structural Gene for Orotidine 5&#39;-Phosphate Carboxylase of Neurospora crassa by Expression in Escherichia coli&quot;, Mol. Gen. Genet, 190:403-405 (1983)."><meta name="citation_reference" content="Buxton, F. P., et al., &quot;The transformation of mycelial spheroplasts of Neurospora crassa and the attempted isolation of an autonomous replicator&quot;, Mol. Gen. Genet, 196:339-344 (1984)."><meta name="citation_reference" content="Buxton, F. P., et al., &quot;Transformation of Aspergillus niger using the argB gene of Aspergillus nidulans&quot;, Gene, 37:207-214 (1985)."><meta name="citation_reference" content="Buxton, F. P., et al., Cloning of the Structural Gene for Orotidfine 5 Phosphate Carboxylase of Neurospora crassa by Expression in Escherichia coli ."><meta name="citation_reference" content="Buxton, F. P., et al., Cloning of the Structural Gene for Orotidfine 5&#39;-Phosphate Carboxylase of Neurospora crassa by Expression in Escherichia coli."><meta name="citation_reference" content="Buxton, F. P., et al., Cloning of the Structural Gene for Orotidine 5 Phosphate Carboxylase of Neurospora crassa by Expression in Escherichia coli , Mol. Gen. Genet, 190:403 405 (1983)."><meta name="citation_reference" content="Buxton, F. P., et al., The transformation of mycelial spheroplasts of Neurospora crassa and the attempted isolation of an autonomous replicator , Mol. Gen. Genet, 196:339 344 (1984)."><meta name="citation_reference" content="Buxton, F. P., et al., Transformation of Aspergillus niger using the argB gene of Aspergillus nidulans , Gene, 37:207 214 (1985)."><meta name="citation_reference" content="Caddick, M. X., et al., &quot;Cloning of the regulatory gene areA mediating nitrogen metabolite repression in Aspergillus nidulans&quot;, The EMBO Journal, 5(5):1087-1090 (1986)."><meta name="citation_reference" content="Caddick, M. X., et al., Cloning of the regulatory gene areA mediating nitrogen metabolite repression in Aspergillus nidulans , The EMBO Journal, 5(5):1087 1090 (1986)."><meta name="citation_reference" content="Case, M.E., et al., &quot;Efficient transformation of Neurospora crassa by utilizing hybrid plasmid DNA&quot;, Proc. Natl. Acad. Sci. USA 76: 5259-5263 (1979)."><meta name="citation_reference" content="Case, M.E., et al., Efficient transformation of Neurospora crassa by utilizing hybrid plasmid DNA , Proc. Natl. Acad. Sci. USA 76: 5259 5263 (1979)."><meta name="citation_reference" content="Clements, J. M., et al., &quot;Transcription and processing signals in the 3-phosphoglycerate kinase (PGK) gene from Aspergillus nidulans&quot;, Gene, 44:97-105 (1986)."><meta name="citation_reference" content="Clements, J. M., et al., Transcription and processing signals in the 3 phosphoglycerate kinase (PGK) gene from Aspergillus nidulans , Gene, 44:97 105 (1986)."><meta name="citation_reference" content="Cullen, D., et al., &quot;Expression of bovine chymosin by Aspergillus nidulans&quot;Heredity, vol. 57, p. 131, Abstract No. 20 (1986)."><meta name="citation_reference" content="Cullen, D., et al., Expression of bovine chymosin by Aspergillus nidulans Heredity, vol. 57, p. 131, Abstract No. 20 (1986)."><meta name="citation_reference" content="Dhawale, S. S., et al., &quot;Transformation of Neurospora crassa with circular and linear DNA and analysis of the fate of the transforming DNA&quot;, Current Genetics, 10:205-212 (1985)."><meta name="citation_reference" content="Dhawale, S. S., et al., Transformation of Neurospora crassa with circular and linear DNA and analysis of the fate of the transforming DNA , Current Genetics, 10:205 212 (1985)."><meta name="citation_reference" content="Doy, C. H., et al., &quot;Genomic Clones of Aspergillus nidulans Containing alcA, the Structural Gene for Alcohol Dehydrogenase and alcR, a Regulatory Gene for Ethanol Metabolism&quot;, DNA, 4(2):105-114 (1985)."><meta name="citation_reference" content="Doy, C. H., et al., Genomic Clones of Aspergillus nidulans Containing alcA, the Structural Gene for Alcohol Dehydrogenase and alcR, a Regulatory Gene for Ethanol Metabolism , DNA, 4(2):105 114 (1985)."><meta name="citation_reference" content="Durrens, P., et al., &quot;Heterologous insertion of transforming DNA and generation of new deletions associated with transformation in Aspergillus nidulans&quot;, Mol. Gen. Genet., 203:544-549 (1986)."><meta name="citation_reference" content="Durrens, P., et al., Heterologous insertion of transforming DNA and generation of new deletions associated with transformation in Aspergillus nidulans , Mol. Gen. Genet., 203:544 549 (1986)."><meta name="citation_reference" content="Emtage, J. S., et al., &quot;Synthesis of calf prochymosin (prorennin) in Escherichia coli&quot;, Proc. Natl. Acad. Sci. USA, 80:3671-3675 (1983)."><meta name="citation_reference" content="Emtage, J. S., et al., Synthesis of calf prochymosin (prorennin) in Escherichia coli , Proc. Natl. Acad. Sci. USA, 80:3671 3675 (1983)."><meta name="citation_reference" content="Freer, S.N., &quot;Fungal Nucleic Acids&quot;, In: Agricultural Research Service, USDA, Peoria, Illinois, pp. 175-193, (1983)."><meta name="citation_reference" content="Freer, S.N., Fungal Nucleic Acids , In: Agricultural Research Service, USDA, Peoria, Illinois, pp. 175 193, (1983)."><meta name="citation_reference" content="Golff, C. g., et al., Expression of calf prochymosin in Saccharomyces cerevisiae, Gene, 27:35 46 (1984)."><meta name="citation_reference" content="Golff, C. g., et al., Expression of calf prochymosin in Saccharomyces cerevisiae, Gene, 27:35-46 (1984)."><meta name="citation_reference" content="Grant, D. M., et al., &quot;Transformation of Neurospora crassa with Recombinant Plasmids Containing the Cloned Glutamate Dehydrogenase (am) Gene:Evidence for Autonomous Replication of the Transforming Plasmid&quot;, Molecular and Cellular Biology, 4(10):2041-2051 (1984)."><meta name="citation_reference" content="Grant, D. M., et al., Transformation of Neurospora crassa with Recombinant Plasmids Containing the Cloned Glutamate Dehydrogenase (am) Gene:Evidence for Autonomous Replication of the Transforming Plasmid , Molecular and Cellular Biology, 4(10):2041 2051 (1984)."><meta name="citation_reference" content="Hayenga, K. et al., &quot;Expression and Secretion of Bovine Calf Chymosin by Aspergillus nidulans&quot;, Journal of Cellular Biochemistry Supp. vol. 0, No. 10, Part A, Abstract E109 (1986)."><meta name="citation_reference" content="Hayenga, K. et al., Expression and Secretion of Bovine Calf Chymosin by Aspergillus nidulans , Journal of Cellular Biochemistry Supp. vol. 0, No. 10, Part A, Abstract E109 (1986)."><meta name="citation_reference" content="Herrera Estrella et al. Mol. Microbiol. (1990) 4:839 43."><meta name="citation_reference" content="Herrera-Estrella et al. Mol. Microbiol. (1990) 4:839-43."><meta name="citation_reference" content="Heyneker, H. L., et al., &quot;Cloning strategies in Aspergillus for enzyme production: prochymosin as a model system&quot;, Proc. of Bio. Expo., 86:145-149 (1986)."><meta name="citation_reference" content="Heyneker, H. L., et al., Cloning strategies in Aspergillus for enzyme production: prochymosin as a model system , Proc. of Bio. Expo., 86:145 149 (1986)."><meta name="citation_reference" content="Hidaka, M., et al., &quot;Cloning and structural analysis of the calf prochymosin gene&quot;, Gene, 43:197-203 (1986)."><meta name="citation_reference" content="Hidaka, M., et al., Cloning and structural analysis of the calf prochymosin gene , Gene, 43:197 203 (1986)."><meta name="citation_reference" content="Huiet, L., et al., &quot;The qa repressor gene of Neurospora crassa:Wild-type and mutant nucleotide sequences&quot;, Proc. Natl. Acad. Sci. USA, 83:3381-3385 (1986)."><meta name="citation_reference" content="Huiet, L., et al., The qa repressor gene of Neurospora crassa: Wild type and mutant nucleotide sequences , Proc. Natl. Acad. Sci. USA, 83:3381 3385 (1986)."><meta name="citation_reference" content="Hynes, M. J., &quot;Topical Review--Transformation of Filamentous Fungi&quot;, Experimental Mycology, 10:1-8 (1986)."><meta name="citation_reference" content="Hynes, M. J., et al., &quot;Isolation of Genomic Clones Containing the amdS gene of Aspergillus nidulans and Their Use in the Analysis of Structural and Regulatory Mutations&quot;, Molecular and Cellular Biology, 3(8):1430-1439 (1983)."><meta name="citation_reference" content="Hynes, M. J., et al., Isolation of Genomic Clones Containing the amdS gene of Aspergillus nidulans and Their Use in the Analysis of Structural and Regulatory Mutations , Molecular and Cellular Biology, 3(8):1430 1439 (1983)."><meta name="citation_reference" content="Hynes, M. J., Topical Review Transformation of Filamentous Fungi , Experimental Mycology, 10:1 8 (1986)."><meta name="citation_reference" content="Innis, M. A., et al. Expression, Glycosylation, and Secretion of an Aspergillus Glucoamylase by Saccharomyces cerevisiae, Science, 228:21 26 (1985)."><meta name="citation_reference" content="Innis, M. A., et al. Expression, Glycosylation, and Secretion of an Aspergillus Glucoamylase by Saccharomyces cerevisiae, Science, 228:21-26 (1985)."><meta name="citation_reference" content="Johnstone, I. L., &quot;Cloning an Aspergillus nidulans developmental gene by transformation&quot;, The EMBO Journal, 4(5):1307-1311 (1985)."><meta name="citation_reference" content="Johnstone, I. L., &quot;Transformation of Aspergillus nidulans&quot;, Microbiological Sciences, 2(10):307-311 (1985)."><meta name="citation_reference" content="Johnstone, I. L., Cloning an Aspergillus nidulans developmental gene by transformation , The EMBO Journal, 4(5):1307 1311 (1985)."><meta name="citation_reference" content="Johnstone, I. L., Transformation of Aspergillus nidulans , Microbiological Sciences, 2(10):307 311 (1985)."><meta name="citation_reference" content="K u ck, U., et al., II. Recombination: a) DNA Transformation in Cyanobacteria, Algae and Fungi:Trends and Perspectives , Progress in Botany, 46:201 204 (1984)."><meta name="citation_reference" content="Kawaguchi, F., et al., Renaturation and activation of calf prochymosin produced in an insoluble form in Escherichia coli, Journal of Biotechnology, 1:307 315 (1984)."><meta name="citation_reference" content="Kawaguchi, F., et al., Renaturation and activation of calf prochymosin produced in an insoluble form in Escherichia coli, Journal of Biotechnology, 1:307-315 (1984)."><meta name="citation_reference" content="Keesey, J. K., Jr., et al., &quot;Cloning of the trp-1 Gene from Neurospora crassa by Complementation of a trpC Mutation in Escherichia coli&quot;, Journal of Bacteriology, 152(2):954-958 (1982)."><meta name="citation_reference" content="Keesey, J. K., Jr., et al., Cloning of the trp 1 Gene from Neurospora crassa by Complementation of a trpC Mutation in Escherichia coli , Journal of Bacteriology, 152(2):954 958 (1982)."><meta name="citation_reference" content="Kelly, J. M., et al., &quot;Transformation of Aspergillus niger by the amdS gene of Aspergillus nidulans&quot;, The EMBO Journal, 4(2):475-479 (1985)."><meta name="citation_reference" content="Kelly, J. M., et al., Transformation of Aspergillus niger by the amdS gene of Aspergillus nidulans , The EMBO Journal, 4(2):475 479 (1985)."><meta name="citation_reference" content="Kinnaird, J. H., et al., &quot;The complete nucleotide sequence of the Neurospora crassa am (NADP-specific glutamate dehydrogenase) gene&quot;, Gene, 26:253-260 (1983)."><meta name="citation_reference" content="Kinnaird, J. H., et al., The complete nucleotide sequence of the Neurospora crassa am (NADP specific glutamate dehydrogenase) gene , Gene, 26:253 260 (1983)."><meta name="citation_reference" content="Kinsey, J. A., et al., &quot;Transformation of Neurospora crassa with the Cloned am (Glutamate Dehydrogenase) Gene&quot;, Molecular and Cellular Biology, 4(1):117-122 (1984)."><meta name="citation_reference" content="Kinsey, J. A., et al., Transformation of Neurospora crassa with the Cloned am (Glutamate Dehydrogenase) Gene , Molecular and Cellular Biology, 4(1):117 122 (1984)."><meta name="citation_reference" content="Kos, A., et al., &quot;Isolation and characterization of the Aspergillus niger trpC gene&quot;, Gene, 39:231-238 (1985)."><meta name="citation_reference" content="Kos, A., et al., Isolation and characterization of the Aspergillus niger trpC gene , Gene, 39:231 238 (1985)."><meta name="citation_reference" content="Kuck, U., et al., &quot;II. Recombination: a) DNA Transformation in Cyanobacteria, Algae and Fungi:Trends and Perspectives&quot;, Progress in Botany, 46:201-204 (1984)."><meta name="citation_reference" content="Kuiper, M. T. R., et al., &quot;A recombinant plasmid carrying the mitochondrial plasmid sequence of Neurospora intermedia LaBelle yields new plasmid derivatives in Neurospora crassa transformants&quot;, Current Genetics, 9:471-477 (1985)."><meta name="citation_reference" content="Kuiper, M. T. R., et al., A recombinant plasmid carrying the mitochondrial plasmid sequence of Neurospora intermedia LaBelle yields new plasmid derivatives in Neurospora crassa transformants , Current Genetics, 9:471 477 (1985)."><meta name="citation_reference" content="Legerton, T. LK., et al., Cloning and characterization of the multifunctional his 3 gene of Neurospora crassa, Gene, 39:129 140 (1985)."><meta name="citation_reference" content="Legerton, T. LK., et al., Cloning and characterization of the multifunctional his-3 gene of Neurospora crassa, Gene, 39:129-140 (1985)."><meta name="citation_reference" content="M u nger, K., et al., isolation and structural organization of the Neurospoa crassa copper metallothionein gene , The EMBO Journal, 4(10):2665 2668 (1985)."><meta name="citation_reference" content="Martson, F. A. O., et al., &quot;Purification of Calf Prochymosin (Prorennin) Synthesized in Escherichia coli&quot;, Biotechnology 2:800-804 (1984)."><meta name="citation_reference" content="Martson, F. A. O., et al., Purification of Calf Prochymosin (Prorennin) Synthesized in Escherichia coli , Biotechnology 2:800 804 (1984)."><meta name="citation_reference" content="McCaman, M. T., et al., Enzymatic properties and processing of bovine prochymosin synthesized in Escherichia coli, Journal of Biotechnology, 2:177 190 (1985)."><meta name="citation_reference" content="McCaman, M. T., et al., Enzymatic properties and processing of bovine prochymosin synthesized in Escherichia coli, Journal of Biotechnology, 2:177-190 (1985)."><meta name="citation_reference" content="McGuire, J., &quot;Cloning, Expression, Purification and Testing of Chymosin&quot;, from Proceedings of Bio Expo 86. The American Commercial and Industrial Conference and Exposition in Biotechnology, pp. 121-141 (1986)."><meta name="citation_reference" content="McGuire, J., Cloning, Expression, Purification and Testing of Chymosin , from Proceedings of Bio Expo 86. The American Commercial and Industrial Conference and Exposition in Biotechnology, pp. 121 141 (1986)."><meta name="citation_reference" content="McKnight G. L., et al., &quot;Nucleotide Sequece of the Triosephosphate Isomerawse Gene from Aspergillus nidulans:Implications for a Differential Loss of Introns&quot;, Cell, 46:143-147 (1986)."><meta name="citation_reference" content="McKnight G. L., et al., Nucleotide Sequece of the Triosephosphate Isomerawse Gene from Aspergillus nidulans: Implications for a Differential Loss of Introns , Cell, 46:143 147 (1986)."><meta name="citation_reference" content="McKnight, G. L., &quot;Lack of an Aspergillus Promoter Function and Intron Splicing in Saccaromyces&quot;, J. Cell. Biochem. Suppl., 9c:137 (1985)."><meta name="citation_reference" content="McKnight, G. L., Lack of an Aspergillus Promoter Function and Intron Splicing in Saccaromyces , J. Cell. Biochem. Suppl., 9c:137 (1985)."><meta name="citation_reference" content="Mellor, J., et al., Efficient synthesis of enzymatically active calf chymosin in Saccaromyces cerevisiae, Gene, 24:1 14 (1983)."><meta name="citation_reference" content="Mellor, J., et al., Efficient synthesis of enzymatically active calf chymosin in Saccaromyces cerevisiae, Gene, 24:1-14 (1983)."><meta name="citation_reference" content="Miller, B. L., et al., &quot;Direct and Indirect Gene Replacements in Aspergillus nidulans&quot;, Molecular and Cellular Biology, 5(7):1714-1721 (1985)."><meta name="citation_reference" content="Miller, B. L., et al., Direct and Indirect Gene Replacements in Aspergillus nidulans , Molecular and Cellular Biology, 5(7):1714 1721 (1985)."><meta name="citation_reference" content="Moir, D. T., et al., &quot;Molecular cloning and characterization of double-stranded cDNA coding for bovine chymosin&quot;, Gene, 19:127-138 (1982)."><meta name="citation_reference" content="Moir, D. T., et al., Molecular cloning and characterization of double stranded cDNA coding for bovine chymosin , Gene, 19:127 138 (1982)."><meta name="citation_reference" content="Moir, D. T., et al., Production of Calf Chymosin by the Yeast S. cerevisiae, Developments in Industrial Microbiology, 26:75 85 (1985)."><meta name="citation_reference" content="Moir, D. T., et al., Production of Calf Chymosin by the Yeast S. cerevisiae, Developments in Industrial Microbiology, 26:75-85 (1985)."><meta name="citation_reference" content="Moir, D., et al., &quot;Molecular cloning and characterization of double-dtranded cDNA coding for bovine chymosin&quot;, Chemical Abstracts, vol. 98, No. 15, p. 173, Abstract No. 120575m (1983)."><meta name="citation_reference" content="Moir, D., et al., Molecular cloning and characterization of double dtranded cDNA coding for bovine chymosin , Chemical Abstracts, vol. 98, No. 15, p. 173, Abstract No. 120575m (1983)."><meta name="citation_reference" content="Mullaney, E. J., et al., &quot;Primary structure of the trpC gene from Aspergillus nidulans&quot;, Mol. Gen. Genet., 199:37-45 (1985)."><meta name="citation_reference" content="Mullaney, E. J., et al., Primary structure of the trpC gene from Aspergillus nidulans , Mol. Gen. Genet., 199:37 45 (1985)."><meta name="citation_reference" content="Munger, K., et al., &quot;isolation and structural organization of the Neurospoa crassa copper metallothionein gene&quot;, The EMBO Journal, 4(10):2665-2668 (1985)."><meta name="citation_reference" content="Newbury, S. F., et al., &quot;Sequence analysis of the pyr-4 (orotidine 5&#39;-P decarboxylase) gene of Neurospora crassa&quot;, Gene, 43:51-58 (1986)."><meta name="citation_reference" content="Newbury, S. F., et al., Sequence analysis of the pyr 4 (orotidine 5 P decarboxylase) gene of Neurospora crassa , Gene, 43:51 58 (1986)."><meta name="citation_reference" content="Nishimori, K., et al., &quot;Cloning in Escherichia coli of the Structural Gene of Prorennin, the Precursor of Calf Milk-Clotting Enzyme Rennin&quot;, J. Biochem. 90:901-904 (1981)."><meta name="citation_reference" content="Nishimori, K., et al., &quot;Expression of cloned calf prochymosin cDNA under control of the tryptophan promoter&quot;, Gene, 29:41-419 (1984)."><meta name="citation_reference" content="Nishimori, K., et al., Cloning in Escherichia coli of the Structural Gene of Prorennin, the Precursor of Calf Milk Clotting Enzyme Rennin , J. Biochem. 90:901 904 (1981)."><meta name="citation_reference" content="Nishimori, K., et al., Expression of cloned calf prochymosin cDNA under control of the tryptophan promoter , Gene, 29:41 419 (1984)."><meta name="citation_reference" content="Nishimori, K., et al., Expression of cloned calf prochymosin gene sequence in Escherichia coli, Gene, 19:337 344 (1982)."><meta name="citation_reference" content="Nishimori, K., et al., Expression of cloned calf prochymosin gene sequence in Escherichia coli, Gene, 19:337-344 (1982)."><meta name="citation_reference" content="Nishimori, K., et al., Nucleotide Sequence of Calf Prorennin cDNA Cloned in Escherichia coli, J. Biochem., 91:1085 1088 (1982)."><meta name="citation_reference" content="Nishimori, K., et al., Nucleotide Sequence of Calf Prorennin cDNA Cloned in Escherichia coli, J. Biochem., 91:1085-1088 (1982)."><meta name="citation_reference" content="Nunberg, J. H., et al., Molecular Cloning and Characterization of the Glucoamylase Gene of Aspergillus awamori, Molecular and Cellular Biology, 4(11):2306 2315 (1984)."><meta name="citation_reference" content="Nunberg, J. H., et al., Molecular Cloning and Characterization of the Glucoamylase Gene of Aspergillus awamori, Molecular and Cellular Biology, 4(11):2306-2315 (1984)."><meta name="citation_reference" content="Orbach, M. J., et al., &quot;Cloning and Characterization of the Gene for β-Tubulin from a Benomyl-Resistant Mutant of Neurospora crassa and Its Use as a Dominant Selectable Marker&quot;, Molecular and Cellular Biology, 6:2452-2461 (1986)."><meta name="citation_reference" content="Orbach, M. J., et al., Cloning and Characterization of the Gene for Tubulin from a Benomyl Resistant Mutant of Neurospora crassa and Its Use as a Dominant Selectable Marker , Molecular and Cellular Biology, 6:2452 2461 (1986)."><meta name="citation_reference" content="Paietta, J., et al., &quot;Plasmid recovery from transformants and the isolation of chromosomal DNA segments improving plasmid replication in Neurospora crassa&quot;, Current Genetics, 9:383-388 (1985)."><meta name="citation_reference" content="Paietta, J., et al., Plasmid recovery from transformants and the isolation of chromosomal DNA segments improving plasmid replication in Neurospora crassa , Current Genetics, 9:383 388 (1985)."><meta name="citation_reference" content="Pentill a , M. E., et al., Cloning of Aspergillus niger genes in yeast. Expression of the gene coding Aspergillus glucosidase , Mol. Gen. Genet., 194:494 499 (1984)."><meta name="citation_reference" content="Pentilla et al. Gene 61 (1987) 155 164."><meta name="citation_reference" content="Pentilla et al. Gene 61 (1987) 155-164."><meta name="citation_reference" content="Pentilla, M. E., et al., &quot;Cloning of Aspergillus niger genes in yeast. Expression of the gene coding Aspergillus β-glucosidase&quot;, Mol. Gen. Genet., 194:494-499 (1984)."><meta name="citation_reference" content="Rutledge, B. J., &quot;Molecular characterization of the qa-4 gene of Neurospora crassa&quot;, Gene, 32:275-287 (1984)."><meta name="citation_reference" content="Rutledge, B. J., Molecular characterization of the qa 4 gene of Neurospora crassa , Gene, 32:275 287 (1984)."><meta name="citation_reference" content="Saunders, G., et al., &quot;Fungal cloning vectors&quot;, Trends in Biotechnology, 4:93-98 (1986)."><meta name="citation_reference" content="Saunders, G., et al., Fungal cloning vectors , Trends in Biotechnology, 4:93 98 (1986)."><meta name="citation_reference" content="Sebald, W., et al., &quot;H+ -ATPase (ATP Synthase):Structure, Function, Boogenesis The F0 F1 Complex of Coupling Membranes&quot;, Abteilung fur Biotechnologische Forschung mbH., W. Germany, pp. 67-75 (1984)."><meta name="citation_reference" content="Sebald, W., et al., H ATPase (ATP Synthase):Structure, Function, Boogenesis The F 0 F 1 Complex of Coupling Membranes , Abteilung f u r Biotechnologische Forschung mbH., W. Germany, pp. 67 75 (1984)."><meta name="citation_reference" content="Shoemaker, S., et al., Molecular Cloning of Exo Cellobiohydrolase I Derived From Trichoderma reesei Strain L27, Biotechnology, 1:691 697 (1983)."><meta name="citation_reference" content="Shoemaker, S., et al., Molecular Cloning of Exo-Cellobiohydrolase I Derived From Trichoderma reesei Strain L27, Biotechnology, 1:691-697 (1983)."><meta name="citation_reference" content="Smith, R. A., et al., &quot;Heterologous Protein Secretion from Yeast&quot;, Science, 229:1219-1224 (1985)."><meta name="citation_reference" content="Smith, R. A., et al., Heterologous Protein Secretion from Yeast , Science, 229:1219 1224 (1985)."><meta name="citation_reference" content="Smith, R.A., et al., &quot;Heterologous Protein Secretion from Yeast&quot;, Science 2293:1219-1224 (1985)."><meta name="citation_reference" content="Smith, R.A., et al., Heterologous Protein Secretion from Yeast , Science 2293:1219 1224 (1985)."><meta name="citation_reference" content="Stahl, U., et al., &quot;Replication and expressin of a bacterial-mitochondrial hybrid plasmid in the fungus Podospora anserina&quot;, Proc. Natl. Acad. Sci. USA 79:3641-3645 (1982)."><meta name="citation_reference" content="Stahl, U., et al., Replication and expressin of a bacterial mitochondrial hybrid plasmid in the fungus Podospora anserina , Proc. Natl. Acad. Sci. USA 79:3641 3645 (1982)."><meta name="citation_reference" content="Stohl, L. L., et al., &quot;Characterization of deletion derivatives of an autonomously replicating Neurospora plasmid&quot;, Nucleic Acids Research, 12:6169-6178 (1984)."><meta name="citation_reference" content="Stohl, L. L., et al., Characterization of deletion derivatives of an autonomously replicating Neurospora plasmid , Nucleic Acids Research, 12:6169 6178 (1984)."><meta name="citation_reference" content="Stohl, L. L., et al., Construction of a shuttle vector for the filamentous fungus Neurospora crassa, Proc. Natl. Acad. Sci. USA, 80:1058 1062 (1983)."><meta name="citation_reference" content="Stohl, L. L., et al., Construction of a shuttle vector for the filamentous fungus Neurospora crassa, Proc. Natl. Acad. Sci. USA, 80:1058-1062 (1983)."><meta name="citation_reference" content="Tilburn, J., et al., &quot;Transformation by integration in Aspergillus nidulans&quot;, Gene, 26:205-221 (1983)."><meta name="citation_reference" content="Tilburn, J., et al., Transformation by integration in Aspergillus nidulans , Gene, 26:205 221 (1983)."><meta name="citation_reference" content="Tudzynski, U., et al., &quot;Transformation to Senescence with Plasmid Like DNA in the Ascomyete Podospora anserina&quot;, Current Genetics 2:181-184 (1980)."><meta name="citation_reference" content="Tudzynski, U., et al., Transformation to Senescence with Plasmid Like DNA in the Ascomyete Podospora anserina , Current Genetics 2:181 184 (1980)."><meta name="citation_reference" content="Turner, G., et al., &quot;Cloning and Transformation in Aspergillus&quot;, Gene Manipulations in Fungi, (1985)."><meta name="citation_reference" content="Turner, G., et al., Cloning and Transformation in Aspergillus , Gene Manipulations in Fungi, (1985)."><meta name="citation_reference" content="Upshall, A., &quot;Filamentous Fungi in Biotechnology&quot;, Biotechniques, 4:158-166 (1986)."><meta name="citation_reference" content="Upshall, A., &quot;Genetic and molecular characterization of argB+ transformants of Aspergillus nidulans&quot;, Current Genetics, 10:593-599 (1986)."><meta name="citation_reference" content="Upshall, A., Filamentous Fungi in Biotechnology , Biotechniques, 4:158 166 (1986)."><meta name="citation_reference" content="Upshall, A., Genetic and molecular characterization of argB transformants of Aspergillus nidulans , Current Genetics, 10:593 599 (1986)."><meta name="citation_reference" content="Viebrock, A., et al., &quot;The imported preprotein of the proteolipid subunit of the mitochondrial ATP synthase from Neurospora crassa. Molecular cloning and sequencing of the mRNA&quot;, The EMBO Journal, 1(5):565-571 (1982)."><meta name="citation_reference" content="Viebrock, A., et al., The imported preprotein of the proteolipid subunit of the mitochondrial ATP synthase from Neurospora crassa. Molecular cloning and sequencing of the mRNA , The EMBO Journal, 1(5):565 571 (1982)."><meta name="citation_reference" content="Vollmer, S. J., et al., &quot;Efficient cloning of genes of Neurospora crassa&quot;, Proc. Natl. Acad. Sci. USA, 83:4869-4873 (1986)."><meta name="citation_reference" content="Vollmer, S. J., et al., Efficient cloning of genes of Neurospora crassa , Proc. Natl. Acad. Sci. USA, 83:4869 4873 (1986)."><meta name="citation_reference" content="Weiss, R. L., et al., Expression of Aspergillus Genes in Neurospora, Gene Manipulation in Fungi, (1985)."><meta name="citation_reference" content="Wernars, K., et al., &quot;Gene amplification in Aspergillus nidulans by transformation with vectors containing the amdS gene&quot;, Current Genetics, 9:361-368 (1985)."><meta name="citation_reference" content="Wernars, K., et al., Gene amplification in Aspergillus nidulans by transformation with vectors containing the amdS gene , Current Genetics, 9:361 368 (1985)."><meta name="citation_reference" content="Woudt, L. P., et al., &quot;   &quot;, Nucleic Acids Research, 11:5353-6346 (1983)."><meta name="citation_reference" content="Woudt, L. P., et al., , Nucleic Acids Research, 11:5353 6346 (1983)."><meta name="citation_reference" content="Yelton, M. M., et al., &quot;A cosmid for selecting genes by complementation in Aspergillus nidulans:Section of the developmentally regulared yA locus&quot;, Proc. Natl. Acad. Sci. USA, 82:834-838 (1985)."><meta name="citation_reference" content="Yelton, M. M., et al., &quot;Transformation of Aspergillus nidulans by using a trpC plasmid&quot;, Proc. Natl. Acad. Sci. USA, 81:1470-1474 (1984)."><meta name="citation_reference" content="Yelton, M. M., et al., A cosmid for selecting genes by complementation in Aspergillus nidulans: Section of the developmentally regulared yA locus , Proc. Natl. Acad. Sci. USA, 82:834 838 (1985)."><meta name="citation_reference" content="Yelton, M. M., et al., Transformation of Aspergillus nidulans by using a trpC plasmid , Proc. Natl. Acad. Sci. USA, 81:1470 1474 (1984)."><meta name="citation_reference" content="Yelton, M.M., et al., &quot;Transformation of Aspergillus nidulansby using a trpC plasmid&quot;, Proc. Natl. Acad. Sci. USA 81:1470-1474 (1984)."><meta name="citation_reference" content="Yelton, M.M., et al., Transformation of Aspergillus nidulans by using a trpC plasmid , Proc. Natl. Acad. Sci. USA 81:1470 1474 (1984)."><meta name="citation_patent_number" content="US:6103490"><meta name="citation_patent_application_number" content="US:08/261,989"><link rel="canonical" href="http://www.google.com/patents/US6103490"/><meta property="og:url" content="http://www.google.com/patents/US6103490"/><meta name="title" content="Patent US6103490 - Heterologous polypeptides expressed in filamentous fungi, processes for making same, and vectors for making same"/><meta name="description" content="Novel vectors are disclosed for expressing and secreting heterologous polypeptides from filamentous fungi. Such vectors are used in novel processes to express and secrete such heterologous polypeptides. The vectors used for transforming a filamentous fungus to express and secrete a heterologous polypeptide include a DNA sequence encoding a heterologous polypeptide and a DNA sequence encoding a signal sequence which is functional in a secretory system in a given filamentous fungus and which is operably linked to the sequence encoding the heterologous polypeptide. Such signal sequences may be the signal sequence normally associated with the heterologous polypeptides or may be derived from other sources. The vector may also contain DNA sequences encoding a promoter sequence which is functionally recognized by the filamentous fungus and which is operably linked to the DNA sequence encoding the signal sequence. Preferably functional polyadenylation sequences are operably linked to the 3&#39; terminus of the DNA sequence encoding the heterologous polypeptides. Each of the above described vectors are used in novel processes to transform a filamentous fungus wherein the DNA sequences encoding the signal sequence and heterologous polypeptide are expressed. The thus synthesized polypeptide is thereafter secreted from the filamentous fungus."/><meta property="og:title" content="Patent US6103490 - Heterologous polypeptides expressed in filamentous fungi, processes for making same, and vectors for making same"/><meta property="og:type" content="book"/><meta property="og:site_name" content="Google Books"/><meta property="og:image" content="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><link rel="image_src" href="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><script>(function(){try{var aa=function(a,b,c,d){d=d||{};d._sn=["cfg",b,c].join(".");window.gbar.logger.ml(a,d)};var g=window.gbar=window.gbar||{},l=window.gbar.i=window.gbar.i||{},m={},n;function _tvn(a,b){var c=parseInt(a,10);return isNaN(c)?b:c}function _tvf(a,b){var c=parseFloat(a);return isNaN(c)?b:c}function _tvv(a){return!!a}function p(a,b,c){(c||g)[a]=b}g.bv={n:_tvn("2",0),r:"",f:".67.",e:"0",m:_tvn("0",1)};
function q(a,b,c){var d="on"+b;if(a.addEventListener)a.addEventListener(b,c,!1);else if(a.attachEvent)a.attachEvent(d,c);else{var f=a[d];a[d]=function(){var a=f.apply(this,arguments),b=c.apply(this,arguments);return void 0==a?b:void 0==b?a:b&&a}}}var s=function(a){return function(){return g.bv.m==a}},ba=s(1),ca=s(2);p("sb",ba);p("kn",ca);l.a=_tvv;l.b=_tvf;l.c=_tvn;l.i=aa;var da=window.gbar.i.i;var t,u,v,w;function ea(a){v=a}function fa(a){var b;if(b=v&&window.encodeURIComponent)b=a.href,b=!b.match(/^http[s]?:\/\/accounts\.google\.[^/]*\/ClearSID/i)&&!b.match(/^http[s]?:\/\/[^/]*\/accounts\/ClearSID/i);if(b=b&&encodeURIComponent(v()))a.href=a.href.replace(/([?&]continue=)[^&]*/,"$1"+b)}function ga(a){window.gApplication&&(a.href=window.gApplication.getTabUrl(a.href))}
function ha(a){var b=document.forms[0].q,c=window.encodeURIComponent&&b&&b.value,b=b&&b.placeholder;c&&c!=b&&(a.href=a.href.replace(/([?&])q=[^&]*|$/,function(a,b){return(b||"&")+"q="+encodeURIComponent(c)}))}n=l.a("")?ga:ha;
function x(a,b,c,d,f,e){var h=document.getElementById(a);if(h){var k=h.style;k.left=d?"auto":b+"px";k.right=d?b+"px":"auto";k.top=c+"px";k.visibility=u?"hidden":"visible";f&&e?(k.width=f+"px",k.height=e+"px"):(x(t,b,c,d,h.offsetWidth,h.offsetHeight),u=u?"":a)}}
var y=[],ia=function(a,b){y.push(b)},ja=function(a){a=a||window.event;var b=a.target||a.srcElement;a.cancelBubble=!0;null==t&&(a=document.createElement(Array.every||window.createPopup?"iframe":"div"),a.frameBorder="0",t=a.id="gbs",a.src="javascript:''",b.parentNode.appendChild(a),q(document,"click",z));var c=b,b=0;"gb3"!=c.className&&(c=c.parentNode);a=c.getAttribute("aria-owns")||"gbi";var d=c.offsetWidth,f=20<c.offsetTop?46:24;document.getElementById("tphdr")&&(f-=3);var e=!1;do b+=c.offsetLeft||
0;while(c=c.offsetParent);var c=(document.documentElement.clientWidth||document.body.clientWidth)-b-d,h,d=document.body,k=document.defaultView;k&&k.getComputedStyle?(d=k.getComputedStyle(d,""))&&(h=d.direction):h=d.currentStyle?d.currentStyle.direction:d.style.direction;h="rtl"==h;if("gbi"==a){for(d=0;k=y[d++];)k();A(null,window.navExtra);h&&(b=c,e=!0)}else h||(b=c,e=!0);u!=a&&z();x(a,b,f,e)},z=function(){u&&x(u,0,0)},A=function(a,b){var c,d=document.getElementById("gbi"),f=a;f||(f=d.firstChild);
for(;b&&(c=b.pop());){var e=d,h=c,k=f;w||(w="gb2");e.insertBefore(h,k).className=w}},ka=function(a,b,c){if((b=document.getElementById(b))&&a){a.className="gb4";var d=document.createElement("span");d.appendChild(a);d.appendChild(document.createTextNode(" | "));d.id=c;b.appendChild(d)}},la=function(){return document.getElementById("gb_70")},ma=function(){return!!u};p("qs",n);p("setContinueCb",ea);p("pc",fa);p("tg",ja);p("close",z);p("addLink",ka);p("almm",A);p("si",la);p("adh",ia);p("op",ma);var B=function(){},C=function(){},F=function(a){var b=new Image,c=D;b.onerror=b.onload=b.onabort=function(){try{delete E[c]}catch(a){}};E[c]=b;b.src=a;D=c+1},E=[],D=0;p("logger",{il:C,ml:B,log:F});var G=window.gbar.logger;var H={},na={},I=[],oa=l.b("0.1",.1),pa=l.a("1",!0),qa=function(a,b){I.push([a,b])},ra=function(a,b){H[a]=b},sa=function(a){return a in H},J={},K=function(a,b){J[a]||(J[a]=[]);J[a].push(b)},ta=function(a){K("m",a)},L=function(a,b){var c=document.createElement("script");c.src=a;c.async=pa;Math.random()<oa&&(c.onerror=function(){c.onerror=null;B(Error("Bundle load failed: name="+(b||"UNK")+" url="+a))});(document.getElementById("xjsc")||document.getElementsByTagName("body")[0]||
document.getElementsByTagName("head")[0]).appendChild(c)},N=function(a){for(var b=0,c;(c=I[b])&&c[0]!=a;++b);!c||c[1].l||c[1].s||(c[1].s=!0,M(2,a),c[1].url&&L(c[1].url,a),c[1].libs&&m.d&&m.d(c[1].libs))},O=function(a){K("gc",a)},P=null,ua=function(a){P=a},M=function(a,b,c){if(P){a={t:a,b:b};if(c)for(var d in c)a[d]=c[d];try{P(a)}catch(f){}}};p("mdc",H);p("mdi",na);p("bnc",I);p("qGC",O);p("qm",ta);p("qd",J);p("lb",N);p("mcf",ra);p("bcf",qa);p("aq",K);p("mdd","");p("has",sa);
p("trh",ua);p("tev",M);var Q=l.b("0.1",.001),R=0;
function _mlToken(a,b){try{if(1>R){R++;var c,d=a,f=b||{},e=encodeURIComponent,h=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&jexpid=",e("17483"),"&srcpg=",e("prop=22"),"&jsr=",Math.round(1/Q),"&ogev=",e("Z4XtU8LfCtHNsQSEq4GADg"),"&ogf=",g.bv.f,"&ogrp=",e("1"),"&ogv=",e("1407723702.0"),"&oggv="+e("es_plusone_gc_20140723.0_p0"),"&ogd=",e("com"),"&ogc=",e("GBR"),"&ogl=",e("en")];f._sn&&(f._sn="og."+
f._sn);for(var k in f)h.push("&"),h.push(e(k)),h.push("="),h.push(e(f[k]));h.push("&emsg=");h.push(e(d.name+":"+d.message));var r=h.join("");S(r)&&(r=r.substr(0,2E3));c=r;var Aa=window.gbar.logger._aem(a,c);F(Aa)}}catch(Na){}}var S=function(a){return 2E3<=a.length},va=function(a,b){return b};function T(a){B=a;p("_itl",S,G);p("_aem",va,G);p("ml",B,G);a={};H.er=a}l.a("")?T(function(a){throw a;}):l.a("1")&&Math.random()<Q&&T(_mlToken);I.push(["m",{url:"//ssl.gstatic.com/gb/js/scm_7385cc5883250b43a39405734c1bea59.js"}]);g.mcf("c",{});g.sg={c:""};if(l.a("1")){var wa=l.a("");I.push(["gc",{auto:wa,url:"//ssl.gstatic.com/gb/js/abc/gci_91f30755d6a6b787dcc2a4062e6e9824.js",libs:"googleapis.client:plusone:gapi.iframes"}]);var xa={version:"gci_91f30755d6a6b787dcc2a4062e6e9824.js",index:"",lang:"en"};H.gc=xa;var U=function(a){window.googleapis&&window.iframes?a&&a():(a&&O(a),N("gc"))};p("lGC",U);l.a("1")&&p("lPWF",U)};window.__PVT="";if(l.a("1")&&l.a("1")){var V=function(a){U(function(){K("pw",a);N("pw")})};p("lPW",V);I.push(["pw",{url:"//ssl.gstatic.com/gb/js/abc/pwm_45f73e4df07a0e388b0fa1f3d30e7280.js"}]);var W=[],ya=function(a){W[0]=a},za=function(a,b){var c=b||{};c._sn="pw";B(a,c)},Ba={signed:W,elog:za,base:"https://plusone.google.com/u/0",loadTime:(new Date).getTime()};H.pw=Ba;var X=function(a,b){for(var c=b.split("."),d=function(){var b=arguments;a(function(){for(var a=g,d=0,e=c.length-1;d<e;++d)a=a[c[d]];a[c[d]].apply(a,b)})},f=g,e=0,h=c.length-1;e<h;++e)f=
f[c[e]]=f[c[e]]||{};return f[c[e]]=d};X(V,"pw.clk");X(V,"pw.hvr");p("su",ya,g.pw)};function Ca(){function a(){for(var b;(b=e[h++])&&"m"!=b[0]&&!b[1].auto;);b&&(M(2,b[0]),b[1].url&&L(b[1].url,b[0]),b[1].libs&&m.d&&m.d(b[1].libs));h<e.length&&setTimeout(a,0)}function b(){0<f--?setTimeout(b,0):a()}var c=l.a("1"),d=l.a(""),f=3,e=I,h=0,k=window.gbarOnReady;if(k)try{k()}catch(r){da(r,"ml","or")}d?p("ldb",a):c?q(window,"load",b):b()}p("rdl",Ca);var Da={D:1,H:2,da:3,p:4,W:5,M:6,F:7,g:8,ha:9,U:10,L:11,T:12,S:13,N:14,Q:15,P:16,fa:17,w:18,O:19,ga:20,ea:21,u:22,G:23,ja:24,ka:25,ia:26,A:27,j:28,o:29,k:30,ca:31,Z:32,$:33,J:34,K:35,ba:36,aa:37,Y:38,B:39,R:40,v:41,X:42,V:43,h:48,C:49,I:500},Y=[1,2,3,4,5,6,9,10,11,13,14,28,29,30,34,35,37,38,39,40,41,42,43,48,49,500];var Z=l.b("0.001",1E-4),Ea=l.b("1",1),Fa=!1,Ga=!1;if(l.a("1")){var Ha=Math.random();Ha<=Z&&(Fa=!0);Ha<=Ea&&(Ga=!0)}var Ia=Da,$=null;function Ja(){var a=0,b=function(b,d){l.a(d)&&(a|=b)};b(1,"");b(2,"");b(4,"");b(8,"");return a}
function Ka(a,b){var c=Z,d=Fa,f;f=a;if(!$){$={};for(var e=0;e<Y.length;e++){var h=Y[e];$[h]=!0}}if(f=!!$[f])c=Ea,d=Ga;if(d){d=encodeURIComponent;g.rp?(f=g.rp(),f="-1"!=f?f:"1"):f="1";c=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&oge=",a,"&ogex=",d("17483"),"&ogev=",d("Z4XtU8LfCtHNsQSEq4GADg"),"&ogf=",g.bv.f,"&ogp=",d("22"),"&ogrp=",d(f),"&ogsr=",Math.round(1/c),"&ogv=",d("1407723702.0"),"&oggv="+
d("es_plusone_gc_20140723.0_p0"),"&ogd=",d("com"),"&ogl=",d("en"),"&ogc=",d("GBR"),"&ogus=",Ja()];if(b){"ogw"in b&&(c.push("&ogw="+b.ogw),delete b.ogw);var k;f=b;e=[];for(k in f)0!=e.length&&e.push(","),e.push(La(k)),e.push("."),e.push(La(f[k]));k=e.join("");""!=k&&(c.push("&ogad="),c.push(d(k)))}F(c.join(""))}}function La(a){"number"==typeof a&&(a+="");return"string"==typeof a?a.replace(".","%2E").replace(",","%2C"):a}C=Ka;p("il",C,G);var Ma={};H.il=Ma;setTimeout(function(){C(Ia.g)},0);}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var b=window.gbar.i.i;var c=window.gbar;var f=function(d){try{var a=document.getElementById("gbom");a&&d.appendChild(a.cloneNode(!0))}catch(e){b(e,"omas","aomc")}};c.aomc=f;}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var a=window.gbar;a.mcf("pm",{p:""});}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{window.gbar.rdl();}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
if (window['_OC_timingAction']) {window['_OC_timingAction']('patents_refpage');}</script><style>#gbar,#guser{font-size:13px;padding-top:1px !important;}#gbar{float:left;height:22px}#guser{padding-bottom:7px !important;text-align:right}.gbh,.gbd{border-top:1px solid #c9d7f1;font-size:1px}.gbh{height:0;position:absolute;top:24px;width:100%}#gbs,.gbm{background:#fff;left:0;position:absolute;text-align:left;visibility:hidden;z-index:1000}.gbm{border:1px solid;border-color:#c9d7f1 #36c #36c #a2bae7;z-index:1001}.gb1{margin-right:.5em}.gb1,.gb3{zoom:1}.gb2{display:block;padding:.2em .5em}.gb2,.gb3{text-decoration:none !important;border-bottom:none}a.gb1,a.gb4{text-decoration:underline !important}a.gb1,a.gb2,a.gb3,a.gb4{color:#00c !important}.gbi .gb3,.gbi .gb2,.gbi .gb4{color:#dd8e27 !important}.gbf .gb3,.gbf .gb2,.gbf .gb4{color:#900 !important}a.gb2:hover{background:#36c;color:#fff !important}#gbar .gbz0l{color:#000 !important;cursor:default;font-weight:bold;text-decoration:none !important}
#gbar { padding:.3em .6em !important;}</style></head><body  topmargin="3" marginheight="3"><div id=gbar><nobr><a onclick=gbar.qs(this);gbar.logger.il(1,{t:1}); class=gb1 id=gb_1 href="https://www.google.com/search?sa=N&tab=tw">Search</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:2}); class=gb1 id=gb_2 href="http://www.google.com/search?hl=en&tbm=isch&source=og&sa=N&tab=ti">Images</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:8}); class=gb1 id=gb_8 href="http://maps.google.com/maps?hl=en&sa=N&tab=tl">Maps</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:78}); class=gb1 id=gb_78 href="https://play.google.com/?hl=en&sa=N&tab=t8">Play</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:36}); class=gb1 id=gb_36 href="http://www.youtube.com/results?sa=N&tab=t1">YouTube</a> <a onclick=gbar.logger.il(1,{t:5}); class=gb1 id=gb_5 href="http://news.google.com/nwshp?hl=en&tab=tn">News</a> <a onclick=gbar.logger.il(1,{t:23}); class=gb1 id=gb_23 href="https://mail.google.com/mail/?tab=tm">Gmail</a> <a onclick=gbar.logger.il(1,{t:25}); class=gb1 id=gb_25 href="https://drive.google.com/?tab=to">Drive</a> <a class=gb3 href="http://www.google.com/intl/en/options/" onclick="this.blur();gbar.tg(event);return !1" aria-haspopup=true><u>More</u> <small>&#9660;</small></a><div class=gbm id=gbi><a onclick=gbar.logger.il(1,{t:24}); class=gb2 id=gb_24 href="https://www.google.com/calendar?tab=tc">Calendar</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:51}); class=gb2 id=gb_51 href="http://translate.google.com/?hl=en&sa=N&tab=tT">Translate</a><a onclick=gbar.logger.il(1,{t:17}); class=gb2 id=gb_17 href="http://www.google.com/mobile/?hl=en&tab=tD">Mobile</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:10}); class=gb2 id=gb_10 href="http://www.google.com/search?hl=en&tbo=u&tbm=bks&source=og&sa=N&tab=tp">Books</a><a onclick=gbar.logger.il(1,{t:212}); class=gb2 id=gb_212 href="https://wallet.google.com/manage/?tab=ta">Wallet</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:6}); class=gb2 id=gb_6 href="http://www.google.com/search?hl=en&tbo=u&tbm=shop&source=og&sa=N&tab=tf">Shopping</a><a onclick=gbar.logger.il(1,{t:30}); class=gb2 id=gb_30 href="http://www.blogger.com/?tab=tj">Blogger</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:27}); class=gb2 id=gb_27 href="http://www.google.com/finance?sa=N&tab=te">Finance</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:31}); class=gb2 id=gb_31 href="https://plus.google.com/photos?sa=N&tab=tq">Photos</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:12}); class=gb2 id=gb_12 href="http://www.google.com/search?hl=en&tbo=u&tbm=vid&source=og&sa=N&tab=tv">Videos</a><div class=gb2><div class=gbd></div></div><a onclick=gbar.logger.il(1,{t:66}); href="http://www.google.com/intl/en/options/" class=gb2>Even more &raquo;</a></div></nobr></div><div id=guser width=100%><nobr><span id=gbn class=gbi></span><span id=gbf class=gbf></span><span id=gbe></span><a target=_top id=gb_70 href="https://www.google.com/accounts/Login?service=&continue=http://www.google.com/patents%3Fhl%3Den&hl=en" class=gb4>Sign in</a><div style="display: none"><div class=gbm id=gbd5 aria-owner=gbg5><div class=gbmc><ol id=gbom class=gbmcc></ol></div></div></div></nobr></div><div class=gbh style=left:0></div><div class=gbh style=right:0></div><div role="alert" style="position: absolute; left: 0; right: 0;"><a href="http://www.google.com/patents/us6103490?hl=en&amp;output=html_text" title="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."><img border="0" src="http://www.google.com/images/cleardot.gif"alt="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."></a></div><div id="guser"><nobr></nobr></div><div style="clear:both;"></div><div id="gb-top-search-box" class="gb-top-search-box-small gb-reset"><table><tr><td class="logo"><a href="http://www.google.com/patents" class="logo-link"><img class="logo-img" src="/intl/en/images/logos/google_logo_41.png" alt="Go to Google Books Home" height="41"/></a></td><td><form action="http://www.google.com/search" name="f" id="vheadf" method="get"><span id="hf"></span><input type="hidden" name="tbm" value="pts"/><input type="hidden" name="tbo" value="1"/><input type="hidden" name="hl" value="en"/><table><tr><td><div class="inputs"><table><tr><td><div class="text-input"><input type="text" name="q" id="vheadq" class="text" maxlength="2048" size="31" value="" title="Search Patents" accesskey="s" autocomplete="off"/><script>window._OC_autoDir &&window._OC_autoDir('vheadq', 'tia-vheadq');</script></div></td><td><div class="submit-input"><input name="btnG" class="submit" type="submit" value=""/></div></td></tr></table></div></td><td class="col-ext-links"><div class="ext-links"><a href="http://www.google.com/advanced_patent_search">&lt;nobr&gt;Advanced Patent Search&lt;/nobr&gt;</a></div></td></tr></table></form></td></tr></table></div><div class="kd-appbar"><h2 class="kd-appname"><a href="/patents">Patents</a></h2><div class="kd-buttonbar left" id="left-toolbar-buttons"><a id="appbar-write-review-link" href=""></a><a id="appbar-view-print-sample-link" href=""></a><a id="appbar-view-ebook-sample-link" href=""></a><a id="appbar-patents-prior-art-finder-link" href="https://www.google.com/patents/related/US6103490"></a><a id="appbar-patents-discuss-this-link" href="http://www.google.com/url?id=J8tQBAABERAJ&amp;q=http://patents.stackexchange.com/redirect/google-patents%3Fpatent%3DUS6103490&amp;usg=AFQjCNFrbWmVwD2_UK_qJrp_otElSuTFgA" data-is-grant="true"></a><a id="appbar-read-patent-link" href="//docs.google.com/viewer?url=patentimages.storage.googleapis.com/pdfs/US6103490.pdf"></a><a id="appbar-download-pdf-link" href="//patentimages.storage.googleapis.com/pdfs/US6103490.pdf"></a></div><div class="kd-buttonbar right" id="right-toolbar-buttons"></div></div><div id="books-microdata" itemscope=""itemtype="http://schema.org/Book"itemid="http://www.google.com/patents/US6103490" style="display:none"><span itemprop="description">Novel vectors are disclosed for expressing and secreting heterologous polypeptides from filamentous fungi. Such vectors are used in novel processes to express and secrete such heterologous polypeptides. The vectors used for transforming a filamentous fungus to express and secrete a heterologous polypeptide...</span><span itemprop="url">http://www.google.com/patents/US6103490?utm_source=gb-gplus-share</span><span class="main-title" itemprop="name">Patent US6103490 - Heterologous polypeptides expressed in filamentous fungi, processes for making same, and vectors for making same</span><img itemprop="image" src="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"alt="Patent US6103490 - Heterologous polypeptides expressed in filamentous fungi, processes for making same, and vectors for making same" title="Patent US6103490 - Heterologous polypeptides expressed in filamentous fungi, processes for making same, and vectors for making same"></div><div style="display: none"><ol id="ofe-gear-menu-contents" class="gbmcc"><li class="gbe gbmtc"><a class="gbmt goog-menuitem-content" id="" href="http://www.google.com/advanced_patent_search">Advanced Patent Search</a></li></ol></div><table id="viewport_table" cellpadding="0" style="clear:both" cellspacing="0"><tr><td id="viewport_td"><div class=vertical_module_list_row><div id=intl_patents class=about_content><div id=intl_patents_v><table class="patent-bibdata"><tr><td class="patent-bibdata-heading">Publication number</td><td class="single-patent-bibdata">US6103490 A</td></tr><tr><td class="patent-bibdata-heading">Publication type</td><td class="single-patent-bibdata">Grant</td></tr><tr><td class="patent-bibdata-heading">Application number</td><td class="single-patent-bibdata">US 08/261,989</td></tr><tr><td class="patent-bibdata-heading">Publication date</td><td class="single-patent-bibdata">Aug 15, 2000</td></tr><tr><td class="patent-bibdata-heading">Filing date</td><td class="single-patent-bibdata">Jun 17, 1994</td></tr><tr><td class="patent-bibdata-heading">Priority date<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The priority date is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the date listed."></span></td><td class="single-patent-bibdata">Aug 29, 1985</td></tr><tr><td class="patent-bibdata-heading">Fee status<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The fee status is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the status or dates listed."></span></td><td class="single-patent-bibdata">Paid</td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Also published as</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/US5364770">US5364770</a>, </span><span class="patent-bibdata-value"><a href="/patents/US5578463">US5578463</a>, </span><span class="patent-bibdata-value"><a href="/patents/US6171817">US6171817</a></span></span></td></tr><tr class="patent-bibdata-list-row alternate-patent-number"><td class="patent-bibdata-heading">Publication number</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value">08261989, </span><span class="patent-bibdata-value">261989, </span><span class="patent-bibdata-value">US 6103490 A, </span><span class="patent-bibdata-value">US 6103490A, </span><span class="patent-bibdata-value">US-A-6103490, </span><span class="patent-bibdata-value">US6103490 A, </span><span class="patent-bibdata-value">US6103490A</span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Inventors</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Randy+Michael+Berka%22">Randy Michael Berka</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Daniel+Cullen%22">Daniel Cullen</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Gregory+Lawrence+Gray%22">Gregory Lawrence Gray</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Kirk+James+Hayenga%22">Kirk James Hayenga</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Virgil+Bryan+Lawlis%22">Virgil Bryan Lawlis</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Original Assignee</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=inassignee:%22Genencor+International,+Inc.%22">Genencor International, Inc.</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Export Citation</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/US6103490.bibtex">BiBTeX</a>, </span><span class="patent-bibdata-value"><a href="/patents/US6103490.enw">EndNote</a>, </span><span class="patent-bibdata-value"><a href="/patents/US6103490.ris">RefMan</a></span></span></td></tr><tr class="patent-internal-links"><td colspan=2><span class="patent-bibdata-value"><a href="#backward-citations">Patent Citations</a> (11),</span> <span class="patent-bibdata-value"><a href="#npl-citations">Non-Patent Citations</a> (185),</span> <span class="patent-bibdata-value"><a href="#forward-citations">Referenced by</a> (9),</span> <span class="patent-bibdata-value"><a href="#classifications">Classifications</a> (30),</span> <span class="patent-bibdata-value"><a href="#legal-events">Legal Events</a> (7)</span> </td></tr><tr><td colspan=2 class="patent-bibdata-external-link-spacer-top"></td></tr><tr class="patent-bibdata-external-link-spacer-bottom"></tr><tr><td colspan=2><span class="patent-bibdata-heading">External Links:&nbsp;</span><span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=J8tQBAABERAJ&q=http://patft.uspto.gov/netacgi/nph-Parser%3FSect2%3DPTO1%26Sect2%3DHITOFF%26p%3D1%26u%3D/netahtml/PTO/search-bool.html%26r%3D1%26f%3DG%26l%3D50%26d%3DPALL%26RefSrch%3Dyes%26Query%3DPN/6103490&usg=AFQjCNF2Z38CX5b8pzk6SWKLKwv_UX0WoA">USPTO</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=J8tQBAABERAJ&q=http://assignments.uspto.gov/assignments/q%3Fdb%3Dpat%26pat%3D6103490&usg=AFQjCNHAHtYsYW9n1Xalqd8HCBPb7I49fg">USPTO Assignment</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=J8tQBAABERAJ&q=http://worldwide.espacenet.com/publicationDetails/biblio%3FCC%3DUS%26NR%3D6103490A%26KC%3DA%26FT%3DD&usg=AFQjCNFUsYkSosyHCiBpAxgeHHzGVauidg">Espacenet</a></span></span></td></tr><tr class="patent-bibdata-group-spacer"></tr></table><div class="number-and-title"><span class="patent-title"><invention-title mxw-id="PT54640501" lang="EN" load-source="patent-office">Heterologous polypeptides expressed in filamentous fungi, processes for making same, and vectors for making same</invention-title></span><br><span class="patent-number">US 6103490 A</span></div><div class="patent-section patent-abstract-section"><div class="patent-section-header"><span class="patent-section-title">Abstract</span></div><div class="patent-text"><abstract mxw-id="PA38111229" lang="EN" load-source="patent-office"> <div class="abstract">Novel vectors are disclosed for expressing and secreting heterologous polypeptides from filamentous fungi. Such vectors are used in novel processes to express and secrete such heterologous polypeptides. The vectors used for transforming a filamentous fungus to express and secrete a heterologous polypeptide include a DNA sequence encoding a heterologous polypeptide and a DNA sequence encoding a signal sequence which is functional in a secretory system in a given filamentous fungus and which is operably linked to the sequence encoding the heterologous polypeptide. Such signal sequences may be the signal sequence normally associated with the heterologous polypeptides or may be derived from other sources. The vector may also contain DNA sequences encoding a promoter sequence which is functionally recognized by the filamentous fungus and which is operably linked to the DNA sequence encoding the signal sequence. Preferably functional polyadenylation sequences are operably linked to the 3' terminus of the DNA sequence encoding the heterologous polypeptides. Each of the above described vectors are used in novel processes to transform a filamentous fungus wherein the DNA sequences encoding the signal sequence and heterologous polypeptide are expressed. The thus synthesized polypeptide is thereafter secreted from the filamentous fungus.</div>
  </abstract></div></div><div class="patent-section patent-drawings-section"><div class="patent-section-header"><span class="patent-section-title">Images<span class="patent-section-count">(23)</span></span></div><div class="patent-drawings-body"><div class="patent-drawings-carousel"><div class="drawings"><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6103490-3.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6103490-3.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6103490-4.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6103490-4.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6103490-5.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6103490-5.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6103490-6.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6103490-6.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6103490-7.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6103490-7.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6103490-8.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6103490-8.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6103490-9.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6103490-9.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6103490-10.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6103490-10.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6103490-11.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6103490-11.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6103490-12.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6103490-12.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6103490-13.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6103490-13.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6103490-14.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6103490-14.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6103490-15.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6103490-15.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6103490-16.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6103490-16.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6103490-17.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6103490-17.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6103490-18.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6103490-18.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6103490-19.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6103490-19.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6103490-20.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6103490-20.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6103490-21.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6103490-21.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6103490-22.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6103490-22.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6103490-23.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6103490-23.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6103490-24.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6103490-24.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6103490-25.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6103490-25.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div></div></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img" alt="Previous page"src="/googlebooks/images/kennedy/page_left.png"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img" alt="Next page"src="/googlebooks/images/kennedy/page_right.png"width="21" height="21" /></div></div></div><div class="patent-post-drawings"></div><div class="patent-section patent-claims-section"><div class="patent-section-header"><span class="patent-section-title">Claims<span class="patent-section-count">(48)</span></span></div><div class="patent-text"><div mxw-id="PCLM5610531" lang="EN" load-source="patent-office" class="claims">
    <claim-statement>What is claimed is:</claim-statement> <div class="claim"> <div num="1" class="claim">
      <div class="claim-text">1. A transformed Trichoderma expression host capable of secreting a heterologous polypeptide, said host being transformed with a vector comprising promoter DNA from a fungal gene operably linked to coding DNA, said coding DNA comprising DNA coding for a signal peptide and said heterologous polypeptide.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="2" class="claim">
      <div class="claim-text">2. The Trichoderma host of claim 1 wherein said fungal gene is from a filamentous fungus.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="3" class="claim">
      <div class="claim-text">3. The Trichoderma host of claim 2 wherein said filamentous fungus is selected from the group consisting of Aspergillus, Mucor and Humicola.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="4" class="claim">
      <div class="claim-text">4. The Trichoderma host of claim 3 wherein said gene from said Aspergillus or Humicola comprises a glucoamylase gene.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="5" class="claim">
      <div class="claim-text">5. The Trichodema host of claim 4 wherein said glucoamylase gene comprises the glucoamylase gene from Aspergillus niger or Aspergillus nidulans.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="6" class="claim">
      <div class="claim-text">6. The Trichoderma host of claim 3 wherein said gene from Mucor comprises a carboxyl protease gene.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="7" class="claim">
      <div class="claim-text">7. The Trichoderma host of claim 1 or 2 wherein said heterologous polypeptide is biochemically active.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="8" class="claim">
      <div class="claim-text">8. The Trichoderma host of claim 1 or 2 wherein said heterologous polypeptide comprises a mammalian polypeptide.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="9" class="claim">
      <div class="claim-text">9. The Trichoderma host of claim 8 wherein said mammalian polypeptide comprises chymosin or prochymosin.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="10" class="claim">
      <div class="claim-text">10. The Trichoderma host of claim 1 or 2 wherein said heterologous polypeptide comprises a polypeptide from a filamentous fungus other than said Trichoderma host.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="11" class="claim">
      <div class="claim-text">11. The Trichoderma host of claim 1 or 2 wherein said heterologous polypeptide comprises an enzyme.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="12" class="claim">
      <div class="claim-text">12. The Trichoderma host of claim 1 or 2 wherein said enzyme is selected from the group consisting of chymosin, prochymosin, Aspergillus niger glucoamylase, Humicola grisea glucoamylase and Mucor miehei carboxyl protease.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="13" class="claim">
      <div class="claim-text">13. The Trichoderma host of claim 1 or 2 wherein said signal peptide is from a polypeptide secreted from a filamentous fungus.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="14" class="claim">
      <div class="claim-text">14. The Trichoderma host of claim 13 wherein said secreted polypeptide is selected from the group consisting of Aspergillus glucoamylase, Humicola glucoamylase, and Mucor carboxyl protease.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="15" class="claim">
      <div class="claim-text">15. The Trichoderma host of claim 1 or 2 wherein said signal peptide is from a source other than a filamentous fungus.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="16" class="claim">
      <div class="claim-text">16. The Trichoderma host of claim 15 wherein said signal peptide comprises the signal peptide from a secreted mammalian polypeptide.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="17" class="claim">
      <div class="claim-text">17. The Trichoderma host of claim 16 wherein said mammalian polypeptide comprises preprochymosin.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="18" class="claim">
      <div class="claim-text">18. The Trichoderma host of claim 1 or 2 wherein said vector further compries DNA encoding a selection characteristic expressible in said Trichoderma host.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="19" class="claim">
      <div class="claim-text">19. The Trichoderma host of claim 18 wherein said selection characteristic is selected from the group consisting of acetamidase, pyr4, argB and trpC.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="20" class="claim">
      <div class="claim-text">20. The Trichoderma host of claim 1 or 2 wherein said host comprises Trichoderma reesei.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="21" class="claim">
      <div class="claim-text">21. The process of claim 20 wherein said fungal gene is from a filamentous fungus.</div>
    </div>
    </div> <div class="claim"> <div num="22" class="claim">
      <div class="claim-text">22. A process for producing polypeptide comprising:<div class="claim-text">culturing a member of the genus Trichoderma transformed with a vector comprising promote DNA from a fungal gene operably linked to coding DNA, said coding DNA comprising DNA coding for a signal peptide and a heterologous polypeptide wherein said culturing is under conditions which permit the expression of said coding DNA and secretion of said heterologous-polypeptide.</div> </div>
    </div>
    </div> <div class="claim-dependent"> <div num="23" class="claim">
      <div class="claim-text">23. The process of claim 22 wherein said fungal gene is from a filamentous fungus.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="24" class="claim">
      <div class="claim-text">24. The process of claim 22 or 23 wherein said culturing is carried out in a culture medium comprising utilizable carbon, nitrogen and phosphate sources, surfactant and trace elements.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="25" class="claim">
      <div class="claim-text">25. The process of claim 22 or 23 further comprising the step of isolating said secreted heterologous polypeptide.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="26" class="claim">
      <div class="claim-text">26. The process of claim 23 wherein said filamentous fungus is selected from the group consisting of Aspegillus, Mucor or Humicola.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="27" class="claim">
      <div class="claim-text">27. The process of claim 26 wherein said gene from said Aspergillus or Humicola comprises a glucoamylase gene.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="28" class="claim">
      <div class="claim-text">28. The process of claim 26 wherein said gene from said Mucor comprises a carboxyl protease gene.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="29" class="claim">
      <div class="claim-text">29. The process of claims 22 or 23 wherein said heterologous polypeptide is biochemically active.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="30" class="claim">
      <div class="claim-text">30. The process of claims 22 or 23 wherein said heterologous polypeptide comprises a mammalian polypeptide.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="31" class="claim">
      <div class="claim-text">31. The process of claim 30 wherein said mammalian polypeptide comprises chymosin or prochymosin.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="32" class="claim">
      <div class="claim-text">32. The process of claims 22 or 23 wherein said heterologous polypeptide comprises a polypeptide from a filamentous fungus other than said Trichoderma host.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="33" class="claim">
      <div class="claim-text">33. The process of claims 22 or 23 wherein said heterologous polypeptide comprises an enzyme.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="34" class="claim">
      <div class="claim-text">34. The process of claims 22 or 23 wherein said enzyme is selected from the group consisting of chymosin, prochymosin, Aspergillus niger glucoamylase, Humicola grisea glucoamylase and Mucor miehei carboxyl protease.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="35" class="claim">
      <div class="claim-text">35. The process of claims 22 or 23 wherein said signal peptide is from a polypeptide secreted from a filamentous fungus.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="36" class="claim">
      <div class="claim-text">36. The process of claims 22 or 23 wherein said signal peptide is from a secreted polypeptide selected from the group consisting of Aspergillus glucoamylase, Humicola glucoamylase, and Mucor carboxyl protease.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="37" class="claim">
      <div class="claim-text">37. The process of claims 22 or 23 wherein said signal peptide is from a source other than a filamentous fungus.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="38" class="claim">
      <div class="claim-text">38. The process of claims 22 or 23 wherein said signal peptide comprises the signal peptide from a secreted mammalian polypeptide.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="39" class="claim">
      <div class="claim-text">39. The process of claims 22 or 23 where said mammalian polypeptide comprises preprochymosin.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="40" class="claim">
      <div class="claim-text">40. The process of claims 22 or 23 wherein said vector further comprises DNA encoding a selecion characteristic expressible in said Trichoderma host.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="41" class="claim">
      <div class="claim-text">41. The process of claims 22 or 23 wherein said election characteristic is selected from the grozp consisting of the acetamidase gene, pyr4, argB and trpC.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="42" class="claim">
      <div class="claim-text">42. The process of claims 22 or 23 wherein said Trichoderma host is selected from the group consisting of Trichoderma reesei and Trichoderma viride.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="43" class="claim">
      <div class="claim-text">43. The process of claim 26 wherein said filamentous fungus is Mucor or Humicola.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="44" class="claim">
      <div class="claim-text">44. The process of claim 26 wherein said gene from said Humicola comprises a glucoamylase gene.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="45" class="claim">
      <div class="claim-text">45. The process of claim 26 wherein said gene from said Mucor comprises a carboxyl protease gene.</div>
    </div>
    </div> <div class="claim"> <div num="46" class="claim">
      <div class="claim-text">46. A process for transforming Trichoderma comprising<div class="claim-text">a) transforming a selected strain of Trichodenna with a vector comprising promoter DNA from a fungal gene operably linked to DNA encoding a polypeptide and DNA encoding a selection characteristic expressible in said selected Trichoderma, and</div> <div class="claim-text">b) selecting at least one Trichoderma transformant based on the expression of said selection characteristic.</div> </div>
    </div>
    </div> <div class="claim-dependent"> <div num="47" class="claim">
      <div class="claim-text">47. The process of claim 46 wherein said expressible selection characteristic comprises a pyr4 gene.</div>
    </div>
    </div> <div class="claim"> <div num="48" class="claim">
      <div class="claim-text">48. A process for producing a polypeptide comprising:<div class="claim-text">Tansforming a Trichoderma host, with a vector comprising a DNA construct comprising promoter DNA from a fungal gene operably linked to coding DNA, said coding DNA comprising DNA coding for a signal peptide and a heterologous polypeptide, and</div> <div class="claim-text">culturing said transformed filamentous fungus under conditions which permit the expression of said coding DNA and secretion of said beterologous polypeptide.</div> </div>
    </div>
  </div> </div></div></div><div class="patent-section patent-description-section"><div class="patent-section-header"><span class="patent-section-title">Description</span></div><div class="patent-text"><div mxw-id="PDES67517516" lang="EN" load-source="patent-office" class="description">
    <p>This is a continuation of application Ser. No. 07/413,010 filed Sep. 25, 1989, now U.S. Pat. No. 5,364,770, which is a continuation of application Ser. No. 07/163,219 filed Feb. 26, 1988, now abandoned, which is a continuation of application Ser. No. 06/882,224 filed Jul. 7, 1986, now abandoned which is a continuation-in-part of application Ser. No. 06/771,374 filed Aug. 29, 1985, now abandoned.</p>
    <heading>FIELD OF THE INVENTION</heading> <p>The present invention is directed to heterologous polypeptides expressed and secreted by filamentous fungi and to vectors and processes for expressing and secreting such polypeptides. More particularly, the invention discloses transformation vectors and processes using the same for expressing and secreting biologically active bovine chymosin and heterologous glucoamylase by a filamentous fungus.</p>
    <heading>BACKGROUND OF THE INVENTION</heading> <p>The expression of DNA sequences encoding heterologous polypeptides (i.e., polypeptides not normally expressed and secreted by a host organism) has advanced to a state of considerable sophistication. For example, it has been reported that various DNA sequences encoding pharmacologically desirable polypeptides [e.g., human growth hormone (1), human tissue plasminogen activator (2), various human interferons (6), urokinase (5), Factor VIII (4), and human serum albumin (3)] and industrially important enzymes [e.g., chymosin (7), alpha amylases (8), and alkaline proteases (9)] have been cloned and expressed in a number of different expression hosts. Such expression has been achieved by transforming prokaryotic organisms [e.g., E. coli (10) or B. subtilis (11)] or eukaryotic organisms [e.g., Saccharomyces cerevisiae (7), Kluyveromyces lactis (12) or Chinese Hamster Ovary cells (2)] with DNA sequences encoding the heterologous polypeptide.</p>
    <p>Some polypeptides, when expressed in heterologous hosts, do not have the same level of biological activity as their naturally produced counterparts when expressed in various host organisms. For example, bovine chymosin has very low biological activity when expressed by E. coli (13) or S. cerevisiae (7). This reduced biological activity in E. coli is not due to the natural inability of E. coli to glycosylate the polypeptide since chymosin is not normally glycosylated (14). Such relative inactivity, both in E. coli and S. cerevisiae, however, appears to be primarily due to improper folding of the polypeptide chain as evidenced by the partial post expression activation of such expressed polypeptides by various procedures. In such procedures, expressed chymosin may be sequentially denatured and renatured in a number of ways to increase biological activity: e.g., treatment with urea (13), exposure to denaturing/renaturing pH (13) and denaturation and cleavage of disulfide bonds followed by renaturation and regeneration of covalent sulfur linkages (15). Such denaturation/renaturation procedures, however, are not highly efficient [e.g., 30% or less recovery of biological activity for rennin (13)], and add considerable time and expense in producing a biologically active polypeptide.</p>
    <p>Other heterologous polypeptides are preferably expressed in higher eukaryotic hosts (e.g., mammalian cells). Such polypeptides are usually glycopolypeptides which require an expression host which can recognize and glycosylate certain amino acid sequences in the heterologous polypeptide. Such mammalian tissue culture systems, however, often do not secrete large amounts of heterologous polypeptides when compared with microbial systems. Moreover, such systems are technically difficult to maintain and consequently are expensive to operate.</p>
    <p>Transformation and expression in a filamentous fungus involving complementation of aroD mutants of N. crassa lacking biosynthetic dehydroquinase has been reported (16). Since then, transformation based on complementation of glutamate dehydrogenase deficient N. crassa mutants has also been developed (17). In each case the dehydroquinase (ga2) and glutamate dehydrogenase (am) genes used for complementation were derived from N. crassa and therefore involved homologous expression. Other examples of homologous expression in filamentous fungi include the complementation of the auxotrophic markers trpC, (18) and argB (19) in A. nidulans and the transformation of A. nidulans to acetamide or acrylamide utilization by expression of the A. nidulans gene encoding acetamidase (20).</p>
    <p>Expression of heterologous polypeptides in filamentous fungi has been limited to the transformation and expression of fungal and bacterial polypeptides. For example, A. nidulans, deficient in orotidine-5'-phosphate decarboxylase, has been transformed with a plasmid containing DNA sequences encoding the pyr4 gene derived from N. crassa (21,32). A. nicer has also been transformed to utilize acetamide and acrylamide by expression of the gene encoding acetamidase derived from A. nidulans (22).</p>
    <p>Examples of heterologous expression of bacterial polypeptides in filamentous fungi include the expression of a bacterial phosphotransferase in N. crassa (23) Dictyostellium discoideum (24) and Cephalosporium acremonium (25).</p>
    <p>In each of these examples of homologous and heterologous fungal expression, the expressed polypeptides were maintained intracellularly in the filamentous fungi.</p>
    <p>Accordingly, an object of the invention herein is to provide for the expression and secretion of heterologous polypeptides by and from filamentous fungi including vectors for transforming such fungi and processes for expressing and secreting such heterologous polypeptides.</p>
    <heading>SUMMARY OF THE INVENTION</heading> <p>The inventor includes novel vectors for expressing and secreting heterologous polypeptides from filamentous fungi. Such vectors are used in novel processes to express and secrete such heterologous polypeptides. The vectors used for transforming a filamentous fungus to express and secrete a heterologous polypeptide include a DNA sequence encoding a heterologous polypeptide and a DNA sequence encoding a signal sequence which is functional in a secretory system in a given filamentous fungus and which is operably linked to the sequence encoding the heterologous polypeptide. Such signal sequences may be the signal sequence normally associated with the heterologous polypeptides or may be derived from other sources.</p>
    <p>The vector may also contain DNA sequences encoding a promoter sequence which is functionally recognized by the filamentous fungus and which is operably linked to the DNA sequence encoding the signal sequence. Preferably functional polyadenylation sequences are operably linked to the 3' terminus of the DNA sequence encoding the heterologous polypeptides.</p>
    <p>Each of the above described vectors are used in novel processes to transform a filamentous fungus wherein the DNA sequences encoding the signal sequence and heterologous polypeptide are expressed. The thus synthesized polypeptide is thereafter secreted from the filamentous fungus.</p>
    <heading>BRIEF DESCRIPTION OF THE DRAWINGS</heading> <p>FIG. 1 is a restriction map of the Aspergillus niger glucoamylase inserts in pGa1 and pGa5.</p>
    <p>FIG. 2 depicts the construction of pDJB-gam-1.</p>
    <p>FIG. 3 depicts the construction of mp19GAPR.</p>
    <p>FIGS. 4, 5, 6, and 7 depict the construction of pGRG1, pGRG2, pGRG3, and pGRG4.</p>
    <p>FIG. 8 shows the strategy used to generate mp19 GAPR.sup.Δ C1-<sup>66</sup> C4 from mp19 GAPR.</p>
    <p>FIG. 9 depicts the construction of pCR160.</p>
    <p>FIG. 10 is a partial restriction map of the Mucor miehei carboxyl protease gene including 5' and 3' flanking sequences.</p>
    <p>FIGS. 11 A,B. and C is the DNA sequence of Mucor miehei, carboxyl protease including the entire coding sequence and 5' and 3' flanking sequences.</p>
    <p>FIG. 12 depicts the construction of pMeJB1-7.</p>
    <p>FIGS. 13 A and B are a partial nucleotide and restriction map of ANS-1.</p>
    <p>FIG. 14 depicts the construction of pDJB-3.</p>
    <p>FIG. 15 depicts the construction of plasmid pCJ:GRG1 through pCJ:GRG4.</p>
    <p>FIG. 16 depicts a restriction endonuclease cleavage map of the 3.7 Kb BamHI fragment from pRSH1.</p>
    <p>FIG. 17 depicts the construction of pCJ:RSH1 and pCJ:RSH2.</p>
    <p>FIG. 18 depicts the expression of H. grisea glucoamylase from A. nidulans.</p>
    <heading>DETAILED DESCRIPTION</heading> <p>The inventors have demonstrated that heterologous polypeptides from widely divergent sources can be expressed and secreted by filamentous fungi. Specifically, bovine chymosin, glucoamylase from Aspergillus niger and Humicola grises and the carboxyl protease from Mucor miehei have been expressed in and secreted from A. nidulans. In addition, bovine chymosin has been expressed and secreted from A. awamori and Trichoderma reesei. Biologically active chymosin was detected in the culture medium without further treatment. This result was surprising in that the vectors used to transform A. nidulans were constructed to secrete prochymosin which requires exposure to an acidic environment (approximately pH 2) to produce biologically active chymosin.</p>
    <p>In general, a vector containing DNA sequences encoding functional promoter and terminator sequences (including polyadenylation sequences) are operably linked to DNA sequences encoding various signal sequences and heterologous polypeptides. The thus constructed vectors are used to transform a filamentous fungus. Viable transformants may thereafter be identified by screening for the expression and secretion of the heterologous polypeptide.</p>
    <p>Alternatively, an expressible selection characteristic may be used to isolate transformants by incorporating DNA sequences encoding the selection characteristic into the transformation vector. Examples of such selection characteristics include resistance to various antibiotics, (e.g., aminoglycosides, benomyl etc.) and sequences encoding genes which complement an auxotrophic defect (e.g. pyr4 complementation of pyr4 deficient A. nidulans, A. awamori or Trichoderma reesei or ArgB complementation of ArgB deficient A. nidulans or A. awamori) or sequences encoding genes which confer a nutritional (e.g., acetamidase) or morphological marker in the expression host.</p>
    <p>In the preferred embodiments disclosed a DNA sequence encoding the ANS-1 sequence derived from A. nidulans is included in the construction of the transformation vectors of the present invention. This sequence increases the transformation efficiency of the vector. Such sequences, however, are not considered to be absolutely necessary to practice the invention.</p>
    <p>In addition, certain DNA sequences derived from the bacterial plasmid pBR325 form part of the disclosed transformation vectors. These sequences also are not believed to be necessary for transforming filamentous fungi. These sequences instead provide for bacterial replication of the vectors during vector construction. Other plasmid sequences which may also be used during vector construction include pBR322 (ATCC 37019), RK-2 (ATCC 37125), pMB9 (ATCC 37019) and pSC101 (ATCC 37032).</p>
    <p>The disclosed preferred embodiments are presented by way of example and are not intended to limit the scope of the invention.</p>
    <p>Definitions</p>
    <p>By "expressing polypeptides" is meant the expression of DNA sequences encoding the polypeptide.</p>
    <p>"Polypeptides" are polymers of α-amino acids which are covalently linked through peptide bonds. Polypeptides include low molecular weight polymers as well as high molecular weight polymers more commonly referred to as proteins. In addition, a polypeptide can be a phosphopolypeptide, glycopolypeptide or metallopolypeptide. Further, one or more polymer chains may be combined to form a polypeptide.</p>
    <p>As used herein a "heterologous polypeptide" is a polypeptide which is not normally expressed and secreted by the filamentous fungus used to express that particular polypeptide. Heterologous polypeptides include polypeptides derived from prokaryotic sources (e.g., α-amylase from Bacillus species, alkaline protease from Bacillus species, and various hydrolytic enzymes from Pseudomonas, etc.), polypeptides derived from eukaryotic sources (e.g., bovine chymosin, human tissue plasminogen activator, human growth hormone, human interferon, urokinase, human serum albumin, factor VIII etc.), and polypeptides derived from fungal sources other than the expression host (e.g., glucoamylase from A. niger and Humicola grisea expressed in A. nidulans, the carboxyl protease from Mucor miehei expressed in A. nidulans, etc.).</p>
    <p>Heterologous polypeptides also include hybrid polypeptides which comprise a combination of partial or complete polypeptide sequences derived from at least two different polypeptides each of which may be homologous or heterologous with regard to the fungal expression host. Examples of such hybrid polypeptides include: 1) DNA sequences encoding prochymosin fused to DNA sequences encoding the A. niger glucoamylase signal and pro sequence alone or in conjunction with various amounts of amino-terminal mature glucoamylase codons, and 2) DNA sequences encoding fungal glucoamylase or any fungal carboxy protease, human tissue plasminogen activator or human growth hormone fused to DNA sequences encoding a functional signal sequence alone or in conjunction with various amounts of amino-terminal propeptide condons or mature codons associated with the functional signal.</p>
    <p>Further, the heterologous polypeptides of the present invention also include: 1) naturally occuring allellic variations that may exist or occur in the sequence of polypeptides derived from the above prokaryotic, eukaryotic and fungal sources as well as those used to form the above hybrid polypeptides, and 2) engineered variations in the above heterologous polypeptides brought about, for example, by way of site specific mutagenesis wherein various deletions, insertions or substitutions of one or more of the amino acids in the heterologous polypeptides are produced.</p>
    <p>A "biochemically active heterologous polypeptide" is a heterologous polypeptide which is secreted in active form as evidenced by its ability to mediate: 1) the biochemical activity mediated by its naturally occurring counterpart, or 2) in the case of hybrid polypeptides, the biochemical activity mediated by at least one of the naturally occurring counterparts comprising the hybrid polypeptides.</p>
    <p>Each of the above defined heterologous polypeptides is encoded by a heterologous DNA sequence which contains a stop signal which is recognized by the filamentous fungus in which expression and secretion occurs. When recognized by the host, the stop signal terminates translation of the mRNA encoding the heterologous polypeptide.</p>
    <p>The "filamentous fungi" of the present invention are eukaryotic microorganisms and include all filamentous forms of the subdivision Eumycotina (26). These fungi are characterized by a vegatative mycelium composed of chitin, cellulose, and other complex polysaccharides. The filamentous fungi of the present invention are morphologically, physiologically, and genetically distinct from yeasts. Vegetative growth by filamentous fungi is by hyphal elongation and carbon catabolism is obligately aerobic. In contrast, vegetative growth by yeasts such as S. cerevisiae is by budding of a unicellular thallus, and carbon catabolism may be fermentative. S cerevisiae has a prominent, very stable diploid phase whereas, diploids exist only briefly prior to meiosis in filamentous fungi like Aspergilli and Neurospora. S. cervisiae has 17 chromosomes as opposed to 8 and 7 for A. nidulans and N. crassa respectively. Recent illustrations of differences between S. cerevisiae and filamentous fungi include the inability of S. cerevisiae to process Aspergillus and Trichoderma introns and the inability to recognize many transcriptional regulators of filamentous fungi (27).</p>
    <p>Various species of filamentous fungi may be used as expression hosts including the following genera: Aspergillus, Trichoderma, Neurospora, Podospora, Endothia Mucor, Cochiobolus and Pyricularia. Specific expression hosts include A. nidulans (18, 19, 20, 21, 61), A. niger (22), A. awomari, e.g., NRRL 3112, ATCC 22342 (NRRL 3112), ATCC 44733, ATCC 14331 and strain UVK 143f, A. oryzae, e.g., ATCC 11490, N. crassa (16, 17, 23), Trichoderma reesei, e.g. NRRL 15709, ATCC 13631, 56764, 56765, 56466, 56767, and Trichoderma viride, e.g., ATCC 32098 and 32086.</p>
    <p>As used herein, a "promotor sequence" is a DNA sequence which is recognized by the particular filamentous fungus for expression purposes. It is operably linked to a DNA sequence encoding the above defined polypeptides. Such linkage comprises positioning of the promoter with respect to the initiation codon of the DNA sequence encoding the signal sequence of the disclosed transformation vectors. The promoter sequence contains transcription and translation control sequences which mediate the expression of the signal sequence and heterologous polypeptide. Examples include the promoter from A. niger glucoamylase (39,48), the Mucor miehei carboxyl protease herein, and A. niger α-glucosidase (28), Trichoderma reesei cellobiohydrolase I (29), A. nidulans trpC (18) and higher eukaryotic promoters such as the SV40 early promoter (24).</p>
    <p>Likewise a "terminator sequence" is a DNA sequence which is recognized by the expression host to terminate transcription. It is operably linked to the 3' end of the DNA encoding the heterologous polypeptide to be expressed. Examples include the terminator from A. nidulans trpC (18), A. niger glucoamylase (39,48), A. niger α-glucosidase (28), and the Mucor miehei carboxyl protease herein, although any fungal terminator is likely to be functional in the present invention.</p>
    <p>A "polyadenylation sequence" is a DNA sequence which when transcribed is recognized by the expression host to add polyadenosine residues to transcribed mRNA. It is operably linked to the 3' end of the DNA encoding the heterologous polypeptide to be expressed. Examples include polyadenylation sequences from A. nidulans trpc (18), A. niger glucoamylase (39,48), A. niger α-glucosidase (28), and the Mucor miehei carboxyl protease herein. Any fungal polyadenylation sequence, however, is likely to be functional in the present invention.</p>
    <p>A "signal sequence" is an amino acid sequence which when operably linked to the amino-terminus of a heterologous polypeptide permits the secretion of such heterologus polypeptide from the host filamentous fungus. Such signal sequences may be the signal sequence normally associated with the heterologous polypeptide (i.e., a native signal sequence) or may be derived from other sources (i.e., a foreign signal sequence). Signal sequences are operably linked to a heterologous polypeptide either by utilizing a native signal sequence or by joining a DNA sequence encoding a foreign signal sequence to a DNA sequence encoding the heterologous polypeptide in the proper reading frame to permit translation of the signal sequence and heterologous polypeptide. Signal sequences useful in practicing the present invention include signals derived from bovine preprochymosin (15), A. niger glucoamylase (39), the Mucor miehei carboxyl protease herein and Trichoderma reesei cellulases (29). However, any signal sequence capable of permitting secretion of a heterologous polypeptide is contemplated by the present invention.</p>
    <p>A "propeptide" or "pro sequence" is an amino acid sequence positioned at the amino terminus of a mature biologically active polypeptide. When so positioned the resultant polypeptide is called a zymogen. Zymogens, generally, are biologically inactive and can be converted to mature active polypeptides by catalytic or autocatalytic cleavage of the propeptide from the zymogen.</p>
    <p>In one embodiment of the invention a "transformation vector" is a DNA sequence encoding a heterologous polypeptide and a DNA sequence encoding a heterologous or homologous signal sequence operably linked thereto. In addition, a transformation vector may include DNA sequences encoding functional promoter and polyadenylation sequences. Each of the above transformation vectors may also include sequences encoding an expressible selection characteristic as well as sequences which increase the efficiency of fungal transformation.</p>
    <p>"Transformation" is a process wherein a transformation vector is introduced into a filamentous fungus. The methods of transformation of the present invention have resulted in the stable integration of all or part of the transformation vector into the genome of the filamentous fungus. However, transformation resulting in the maintenance of a self-replicating extra-chromosomal transformation vector is also contemplated.</p>
    <p>General Methods</p>
    <p>"Digestion" of DNA refers to catalytic cleavage of the DNA with an enzyme that acts only at certain locations in the DNA. Such enzymes are called restriction enzymes, and the sites for which each is specific is called a restriction site. "Partial" digestion refers to incomplete digestion by a restriction enzyme, i.e., conditions are chosen that result in cleavage of some but not all of the sites for a given restriction endonuclease in a DNA substrate. The various restriction enzymes used herein are commercially available and their reaction conditions, cofactors and other requirements as established by the enzyme suppliers were used. In general, about 1 microgram of plasmid or DNA fragment is used with about 1 unit of enzyme and about 20 microliters of buffer solution. Appropriate buffers and substrate amounts with particular restriction enzymes are specified by the manufacturer. Incubation times of about one hour at 37° C. are ordinarily used, but may vary in accordance with the supplier's instructions. After incubation, protein is removed by extraction with phenol and chloroform, and the digested nucleic acid is recovered from the aqueous fraction by precipitation with ethanol. Digestion with a restriction enzyme may be followed by bacterial alkaline phosphatase hydrolysis of the terminal 5' phosphates to prevent the two ends of a DNA fragment from forming a closed loop that would impede insertion of another DNA fragment at the restriction site upon ligation.</p>
    <p>"Recovery" or "isolation" of a given fragment of DNA from a restriction digest means separation of the digest by a polyacrylamide gel electrophoresis, identification of the fragment of interest, removal of the gel section containing the desired fragment, and separation of the DNA from the gel generally by electroelution (30).</p>
    <p>"Ligation" refers to the process of forming phosphodiester bonds between two double-stranded nucleic acid fragments (30). Unless otherwise stated, ligation was accomplished using known buffers in conditions with one unit of T4 DNA ligase ("ligase") per 0.5 microgram of approximately equal molar amounts of the DNA fragments to be ligated.</p>
    <p>"Oligonucleotides" are short length single or double stranded polydeoxynucleotides which were chemically synthesized by the method of Crea et al., (31) and then purified on polyacrylamide gels.</p>
    <p>"Transformation" means introducing DNA in to an organism so that the DNA is maintained, either as an extrachromosomal element or chromosomal integrant. Unless otherwise stated, the method used herein for transformation of E. coli was the CaCl<sub>2</sub> method (30).</p>
    <p>A. nidulans strain G191 (University of Glasgow culture collection) was transformed by incubating A. nidulans sphaeroplasts with the transformation vector. The genotype of strain G191 is pabaA1 (requires p-aminobenzoic acid), fwA1 (a color marker), mauA2 (monoamine non-utilizing), and pyrG89 (deficient for orotidine phosphate decarboxlase). Sphaeroplasts were prepared by the cellophane method of Ballance et al. (21) with the following modifications. To digest A. nidulans cell walls, Novozyme 234 (Novo Industries, Denmark) was first partially purified. A 100 to 500 mg sample of Novozyme 234 was dissolved in 2.5 ml of 0.6M KCl. The 2.5 ml aliquot was loaded into a PD10 column (Pharmacia-Upsulla, Sweden) equilibrated with 0.6M KCl. The enzymes were eluted with 3.5 ml of the same buffer.</p>
    <p>Cellophane discs were incubated in Novozyme 234 (5 mg/ml) for 2 hours, then washed with 0.6M KCl. The digest and washings were combined, filtered through miracloth (Calbiochem-Behring Corp., La Jolla, Calif.), and washed as described (21). Centrifugations were in 50 or 15 ml conical tubes at ca. 1000×g for 10 min. Following incubation on ice for 20 min, 2 ml of the polyethylene glycol 4000 solution (250 mg/ml) was added, incubated at room temperature for 5 min. followed by the addition of 4 ml of 0.6M KCl, 50 mM CaCl<sub>2</sub>. Transformed protoplasts were centrifuged, resuspended in 0.6M KCl, 50 mM CaCl<sub>2</sub>, and plated as described (21). Zero controls comprised protoplasts incubated with 20 μl of 20 mM Tris-HCl, 1 mM EDTA, pH7.4 without plasmid DNA. Positive controls comprised transformation with 5 μg of pDJB3 constructed as described herein. Regeneration frequencies were determined by plating dilutions on minimal media supplemented with 5-10 ppm paba and 500 ppm uridine. Regeneration ranged from 0.5 to 5%.</p>
    <p>Because of the low transformation frequencies associated with pDJB1, the derivative containing the Mucor acid protease gene (pMeJB1-7) was expected to give extremely low transformation frequencies. Consequently, to obtain pmeJBl1-7 transformants of A. nidulans, cotransformation was used. This was accomplished by first constructing a non-selectable vector containing ANS-1, and then transforming sphaeroplasts with a mixture of pmeJB1-7 and the non-selectable vector containing the ANS-1 fragment. The rationale for this approach was that the ANS-1 bearing vector would integrate in multiple copies and provide regions of homology for pMeJB1--7 integration. The ANS-1 vector was prepared by subcloning the PstI-PvuII fragment of ANS-1 (FIGS. 12A and 13B) from pDJB-3 into pUC18 (33).</p>
    <p>The two plasmids (pMeJB1-7 and the ANS-1 containing vector) were mixed (2.5 μg each) and the above mentioned transformation protocol followed.</p>
    <p>Transformants obtained with vectors PGRG1-pGRG4 and pDJB-gam were transferred after 3 or 4 days incubation at 37° C. Minimal media agar plates supplemented with 5 ppm p-aminobenzoic acid were centrally inoculated with mycelial transfers from transformants. Three to five days following inoculation of minimal medium plates, spore suspensions were prepared by vortexing a mycelial fragment in 1 ml distilled H<sub>2</sub> O, 0.02% tween-80. Approximately 5×10<sup>4</sup> spores were inoculated into 250 ml baffled flasks containing 50 ml of the following medium: (g/1) Maltodextrin M-040 (Grain Processing Corp., Muscatine, Iowa) 50 g, NaNO<sub>3</sub> 6 g, MgSO<sub>4</sub> 0.7H20 0.5 g, KCl 0.52 g, KH<sub>2</sub> PO<sub>4</sub> g, 1 ml trace element solution (34), 1 ml MAZU DF-60P antifoam (Mazer Chemicals, Inc., Gurnee, Ill.), 10 ppm p-aminobenzoic acid, and 50 ppm streptomycin sulfate. Alternatives to MAZU, such as bovine serum albumin or other appropriate surfactant may be used. Mucor acid protease secretion was tested in Aspergillus complete medium (20 g dextrose, 1 g peptone, 20 g malt extract per liter). Carbon source regulation of chymosin secretion by Aspercillus nidulans transformants was assessed by measuring secretion in the above-mentioned starch medium relative to the same medium supplemented with 1% fructose, sucrose, or dextrose instead of 5% starch. In all cases, the media were incubated at 37° C. on a rotary shaker (150 rpm). A pDJB3-derived transformant was included as a control.</p>
    <p>Western blots of the various secreted chymosins and Mucor miehei carboxyl protease were performed according to Towbin, et. al (35). Due to the high concentration of salt in chymosin culture broths and the effect this salt has on gel electrophoresis a desalting step was necessary. Pre-poured G-25 columns (Pharmacia, PD10) were equilibriated with 50 mM Na<sub>2</sub> HPO<sub>4</sub>, pH 6.0. A 2.5 ml aliquot of culture broth was applied to the column. The protein was eluted with 3.5 ml of the same buffer. The heterologous polypeptides present on the blots were detected by contacting the nitrocellulose blots first with rabbit anti-chymosin (36) or rabbit anti-Mucor miehei carboxy protease serum (36). The blots were next contacted with goat-anti-rabbit serum conjugated with horseradish peroxidase (Bio-Rad, Richmond, Calif.) and developed. Prior to loading on the gels, 50 μl of medium (desalted in the case of chymosin) was mixed with 25 μl of SDS sample buffer. β-mercaptoethanol was added to a final concetration of 1%. The sample was heated in a 95° C. bath for 5 minutes after which 40-50 μl of sample was loaded on the gel. Each gel was also loaded with 2 μl each of 650, 65 and 6.5 μg/ml chymosin standards and molecular weight markers. Western blots of pmeDJ1-7 transformants were similarly analyzed except that gel permeation was not performed.</p>
    <p>Protease activity was detected as described by Sokol, et. al. (37). Luria broth was supplemented with 1-1.5% skim milk (Difco) and 30-35 ml was poured into a 150 mm petri dish. An aliquot of 2 to 5 μl of culture medium was spotted on the plate. The plate was incubated over night at 37° C. in a humidity box. The activity was determined based on the amount of milk clotting occurring on the plate measured in mm. The plates were co-spotted with dilutions of 100 CHU/ml or 16.6 CHU/ml rennin (CHU-Chr Hansen Unit, Chr Hansen's Laboratorium, A./S., Copenhagen). The relationship between the diameter of the coagulation zone (mm) and the centration of enzyme is logarithmic.</p>
    <p>In order to distinguish between types of proteases, pepstatin, an inhibitor of the chymosin type of carboxyl protease, was used to inhibit protease activity attributable to chymosin. Samples of chymosin mutants and control broths were preincubated with a 1:100 dilution of 10 mM pepstatin in DMSO for 5 minutes before analyzing for protease activity.</p>
    <p>Glucoamylase secretion by pDJB-gam-1 transformants in 5% starch media was assessed using an assay based on the ability of glucoamylase to catalyze the conversion of p-nitrophenol-a-glucopyranoside (PNPAG) (38) to free glucose and p-nitrophenoxide. The substrate, PNPAG, was dissolved in DMSO at 150 mg/ml and 3 to 15 μl aliquots were diluted to 200 ul with 0.2 M sodium acetate, 1 mM calcium chloride at pH 4.3. A 25 μl sample was placed into a microtitre plate well. An equal volume of standards ranging from 0 to 10 Sigma A. niger units/ml (Sigma Chemical Co., St. Louis, Mo.) were placed in separate wells. To each well, 200 μl of PNPAG solution at 2.25 to 11.25 mg/ml was added. The reaction was allowed to proceed at 60° C. for 0.5 to 1 hour. The time depended upon the concentration of enzyme. The reaction was terminated by the addition of 50 μl of 2 M trizma base. The plate was read at 405 nm. The concentration of enzyme was calculated from a standard curve.</p>
    <p>Unless otherwise stated, chromosomal DNA was extracted from filamentous fungi by the following procedure. The filamentous fungus was grown in an appropriate medium broth for 3 to 4 days. Mycelia were harvested by filtering the culture through fine cheesecloth. The mycelia were rinsed thoroughly in a buffer of 50 mM tris-HCl, pH7.5, 5 mM EDTA. Excess liquid was removed by squeezing the mycelia in the cheesecloth. About 3 to 5 grams of wet mycelia were combined with an equivalent amount of sterile, acid-washed sand in a mortar and pestle. The mixture was ground for five minutes to form a fine paste. The mixture was ground for another five minutes after adding 10 ml of 50 mM tris-HCl, pH 7.5, 5 mM EDTA. The slurry was poured into a 50 ml capped centrifuge tube and extracted with 25 ml of phenol-chloroform (equilibrated with an equal volume of 50 mM tris-HCl, pH 7.5, 5 mM EDTA). The phases were separated by low speed centrifugation. The aqueous phase was saved and reextracted three times. The aqueous phases were combined (about 20 ml total volume) and mixed with 2 ml of 3 M sodium acetate, pH 5.4 in sterile centrifuge tubes. Ice cold isopropanol (25 ml) was added and the tubes were placed at -20° C. for one hour. The tubes were then centrifuged at high speed to pellet the nucleic acids, and the supernatant fluid was discarded. Pellets were allowed to air dry for 30 minutes before resuspending in 400 μl of 10 mM tris-HCl, pH 7.5, 1 mM EDTA (TE buffer). Pancreatic ribonuclease (Sigma Chemical Co., St. Louis, Mo.) was added to a final concentration of 10 μg per ml, and the tubes were incubated for 30 minutes at room temperature (30). Ribonuclease was then removed by extraction with phenol-chloroform. The aqueous layer was carefully removed and placed in a tube which contained 40 μl of 3M sodium acetate, pH 5.4. Ice cold ethanol was layered into the solution. The DNA precipitated at the interface and was spooled onto a glass rod. This DNA was dried and resuspended in a small volume (100 to 200 μl) of TE buffer. The concentration of DNA was determined spectrophotometrically at 260 nm (30).</p>
    <p>To confirm the chromosomal integration of chymosin DNA sequences in selected transformants Southern hybridizations were performed (30). Spore suspensions of transformants were inoculated into Aspergillus complete medium and incubated at 37° C. on a rotary shaker for 24-48 hrs. The medium was non-selective in that it was supplemented with 5 ppm p-aminobenzoic acid and contained sufficient uracil for growth of the auxotrophic parent. In effect, these Southerns also tested for the stability of the transforaants. The mycelium was filtered, ground in sand, and the DNA purified as previously described. Transformant DNA was then digested with various restriction enzymes and fragments separated by agarose gel electrophoresis. Control lanes included digested pDJB3 transformant DNA and undigested DNA. Gels were stained with ethidium bromide, photographed, blotted to nitrocellulose or nytran (Schleicher and Schuell, Keene, N.H.), and probed with radiolabeled plasmids or specific fragments.</p>
    <heading>EXAMPLE 1</heading> <heading>Expression and Secretion of Aspergillus niger Glucoamylase by Aspergillus nidulans</heading> <p>A. Construction of pGA1</p>
    <p>Aspergillus niger (Culture #7, Culture Collection Genencor, Inc., South San Francisco, Calif.) was grown in potato dextrose broth (Difco, Detroit, Mich.) at 30° C. for 3 days with vigorous aeration. Chromosomal DNA was extracted as previously described.</p>
    <p>A synthetic oligonucleotide was used as a hybridization probe to detect the glucoamylase gene from Aspergillus niger. The oligonucleotide was 28 bases in length (28 mer) and corresponded to the first 91/3 codons of the published glucoamylase coding sequence (39):</p>
    <p>MetSerPheArgSerLeuLeuAlaLeuSer</p>
    <p>5'ATGTCGTTCCGATCTCTACTCGCCCTGA 3'</p>
    <p>The oligonucleotide was synthesized on a Biosearch automated DNA synthesizer (Biosearch, San Rafael, Calif.) using the reagents and protocols specified by the manufacturer.</p>
    <p>Genomic DNA from Aspergillus niger was analyzed for the presence of glucoanylase sequences by the method of Southern (30). Briefly, 10 μg of Aspergillus niger DNA was digested with EcoRl restriction endonuclease. The digested DNA was subjected to electrophoresis on a 1% agarose gel according to standard methods (30). DNA was transferred from the gel to a nitrocellulose membrane (Schleicher &amp; Schuell, Inc., Keene, N.H.) by blotting in 10×SSC (1.5 M NaCl, 0.15 M trisodium citrate) (30). DNA was fixed to the nitrocellulose by baking in an 80° C. vacuum oven, followed by hybridization at low stringency (2,40) with radiolabeled oligonucleotide probe. Radiolabeling of the synthetic oligonucleotide was done at 37° C. in a 50 μl reaction that contained 70 mM tris-HCl, pH 7.5, 10 mM Mgcl<sub>2</sub>, 5 mM dithiothreitol, 30 pmoles of synthetic oligonucleotide, 20 pmoles of gamma-[32P]ATP (Amersham, Chicago, Ill.; specific activity 5000 Ci/mmol), and 5 units of T4 polynucleotide kinase (New England Biolabs). After hybridization, the filters were washed 15 minutes in 2×SSC, 0.1% sodium dodecylsulfate (SDS) and twice in 2×SSC at 37° C. Filters were air dried, wrapped in Saran-Wrap (Dow Chemical) and applied to Kodak XOmat-AR X-ray film at -70° C. to obtain an autoradiographic image. After developing the autoradiogram, a band of hybridization was clearly visible corresponding to a 3.5 kilobase-pair EcoRl fragment.</p>
    <p>Genomic DNA from Aspergillus niger was digested with EcoRl and size-fractionated by polyacrylamide gel electrophoresis according to standard methods (30). DNA fragments 3 to 4 kb in size were excised and eluted from the gel (30). This DNA fraction was used to generate a library of clones in the Escherichia coli cloning vector pBR322 (ATCC 37019). The cloning vector was cleaved with EcoRl and dephosphorylated with bacterial alkaline phosphatase (Bethesda Research Labs). A typical dephosphorylation reaction consisted of 1 μg of digested vector DNA and 1 unit of alkaline phosphatase in 50 μl of 50 mM tris-HCl, pH 8.0, 50 mM NaCl. The reaction was incubated at 65° C. for one hour. The phosphatase was removed by extraction with phenol-chloroform. The EcoRl, size-selected Aspergillus niger DNA was then ligated with EcoRl cleaved and dephosphorylated pBR322. A typical ligation reaction contained the following: 100 ng each of vector and insert DNAs, 1 unit of T4 DNA ligase (Bethesda Research Labs), 25 mM tris-HCl, pH 7.5, 10 nM MgCl<sub>2</sub>, 10 mM dithiothreitol, and 1 mM ATP in a 10 μl volume. Ligation reactions were incubated at 16° C. for 18 to 24 hours. The ligated DNA was used to transform competent E. coli. 294 cells (ATCC 31446) prepared by the method of Morrison (41). Transformants were selected on LB agar plates (30) which contained carbenecillin at a final concentration of 50 μg per ml. Transformants which harbored glucoamylase gene sequences were identified by colony hybridization methods (30) using the glucoamylase-specific 28 mer as a probe. Hybridizing colonies were purified, and plasmid DNAs were isolated from each by the alkaline-SDS miniscreen procedure (30). The plasmids selected in this manner all contained a 3.5 kb EcoRl fragment which hybridized to the synthetic glucoamylase probe. One such plasmid, designated pGal, was selected for further analysis. A 1.1 kb EcoRl-BglII fragment from the insert in pGal was subcloned into M13 mp9 (42) and partially sequenced by the dideoxy chain termination method (43) to confirm that the cloned DNA encoded the glucoamylase gene. A restriction endonuclease cleavage map of the 3.5 kb EcoRl fragment contained in pGal is depicted in FIG. 1. It was generated by single and double restriction digests followed by orientation of the DNA fragments with respect to known restriction sites in pBR322 (44).</p>
    <p>B. Construction of pGa5</p>
    <p>The nucleotide sequence and restriction map of pGal indicated that pGal contained the entire glucoamylase coding region and 221 nucleotides of 5' flanking DNA. The sequences in this 5' region were strikingly similar to typical eukaryotic promoter sequences with TATAAAT and CAAT boxes located upstream of the ATG start codon (48).</p>
    <p>However, to insure that possible upstream activation sites of the Aspergillus niger glucoamylase gene were included in the final transformation vector a larger genomic fragment which contained at least 1000 bp of 5' flanking DNA was cloned. Southern blotting experiments similar to those already described identified a 6.5 kb ClaI fragment which hybridized to a radiolabeled EcoRI glucoamylase fragment from pgal. The EcoRI fragment was radiolabeled by nick translation (30) with alpha-[32P]dCTP (Amersham; specific activity 3000 Ci/mmol). A nick translation kit (Bethesda Research Labs) was used for the labeling reaction by following the instructions supplied by the manufacturer. Filters were hybridized and washed under stringent conditions (30).</p>
    <p>The 6.5 kb ClaI fragment identified by hybridization was cloned in a manner similar to that described previously. Aspergillus niger DNA was digested with Clal and size-fractionated by polyacrylamide gel electrophoresis. DNA fragments migrating between 5.5 and 8 kb were excised and eluted from the gel. This fraction was ligated to ClaI cleaved and dephosporylated pBR325 (45). The ligation mixture was used to transform competent E. coli 294 cells. Transformants were selected on LB agar plates containing carbenecillin (50 μg/ml). Colonies which contained glucoamylase gene sequences were identified by colony hybridization (30). Plasmid DNA extracted from hybridizing colonies contained a 6.5 kb ClaI fragment which included the 3.5 kb EcoRl fragment cloned previously in pGal. These recombinant plasmids encoded the Aspergillus niger glucoamylase gene as confirmed by supercoil-DNA sequencing (46) with the synthetic oligonucleotide (28 mer) as a sequencing primer. A restriction endonuclease cleavage map of the 6.5 kb ClaI fragment was constructed using single and double digests of the DNA cloned in pBR325. Restriction sites in the vector DNA were used as reference points to orient the fragment. This restriction map is shown in FIG. 1. Location of the glucoamylase gene was deduced by comparing restriction sites of pGa5 to those of the previously published glucoamylase genes (39, 47, 48). From the mapping data it was estimated that approximately 3.3 kb of the 5'-flanking DNA and about 1 kb of 3'-flanking DNA were contained within the cloned fragment.</p>
    <p>Plasmid pGa5 was deposited with the ATCC on Aug. 28, 1985 in E. coli 294 and has been assigned number 53249.</p>
    <p>C. Vector for Expression and Secretion of Aspergillus niger Glucoamylase</p>
    <p>The 6.5 kb ClaI fragment from pGa5 containing the glucoamylase gene was cloned into the E. coli.-Aspergillus nidulans shuttle vector pDJB3 as depicted in FIG. 2. The pDJB3 shuttle vector possesses a selectable beta-lactamase gene and origin or replication from E. coli plasmid pBR325, the pyr4 gene from Neuospora crassa which relieves the auxotrophic requirement for uridine in Aspergillus nidulans strain G191, a sequence known as ANS1 from Aspergillus nidulans which promotes a high frequency of stable integrative transformants in Aspergillus nidulans, unique EcoRl and ClaI restriction sites for cloning. pDJB is constructed as depicted in FIG. 14. Plasmid pFB6 (32) is digested to completion with BglII and partially digested with HindIII. Fragment B containing the pyr4 gene (ca. 2 Kb) is purified by gel electrophoresis and ligated into HindIII/Bam HI digested pBR325 (fragment A) yielding plasmid pDJB1. The ANS-1 sequence is cloned by ligating EcoRI digested A. nidulans genomic DNA (strain G191 or other FGSC#4- derived strains) into EcoRl cleaved pFB6. The resulting pool of EcoRI fragments in pFB6 is used to transform a ura3- S. cerevisiae (E.G. ATCC 44769, 44770 etc.). An autonomously replicating plasmid, pIntA, is purified from the S. cerevisiae transformant. pIntA is digested with ScoRI, the ANS-1 fragment is purified by gel electrophoresis and ligated into EcoRI digested pDJB1, yielding plasmid pDJB2. pDJB2 is partially digested with EcoRI, treated with DNA polymerase I (Klenow), and re-ligated to yield plasmid pDJB3. The partial nucleotide sequence and restriction map of the ANS-1 fragment is showin in FIG. 13A and 13B.</p>
    <p>Plasmid pGa5 was digested with ClaI and the large fragment (fragment A) was separated from the vector by agarose gel electrophoresis. This fragment was ligated with pDJB3 which had been cleaved with ClaI and dephosphorylated (fragment B). The ligation mixture was used to transform competent E. coli 294 cells. Transformants were selected on LB agar supplemented with carbenecillin (50 μg/ml). Analysis of plasmid DNAs from these transformants indicated that the glucoamylase fragment had been inserted as expected. Both orientations of the glucoamylase fragment were obtained by screening various transformants. One plasmid, designated pDJB-gam1 was arbitrarily chosen for transformation of Aspergillus nidulans protoplasts.</p>
    <p>D. Expression and Secretion of Glucoamylase</p>
    <p>Aspergillus nidulans Strain G191 was transformed with pDJB-gam-1 as previously described. Five transformants designated pDJB-gam-1-4, 9, 10, 11 &amp; 13 were analyzed for glucoamylase activity as previously described. The results are shown in Table I.</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">              TABLE I______________________________________          Glucoamylase Activity           Sample (Sigma Units/ml)______________________________________pDJB3          0.129  pDJB-gam-1-4 0.684  pDJB-gam-1-9 0.662  pDJB-gam-1-10 0.131  pDJB-gam-1-11 0.509  pDJB-gam-1-13 0.565  A. niger  2.698______________________________________</pre>
    
    <p>As can be seen, each pDJB-gam-1 transformant produced more glucoamylase activity than the control indicating that biologically active glucoamylase was expressed and secreted from the transformed fungi.</p>
    <heading>EXAMPLE 2</heading> <heading>Expression and Secretion of Bovine Chymosin from Aspergillus nidulans</heading> <p>Expression vectors were constructed encoding either a natural precursor of bovine chymosin (preprochymosin) or a fusion precursor in which DNA sequences for Aspergillus niger glucoamylase and prochymosin were precisely fused. The strategy for the construction of these vectors involved the following steps. First, a DNA sequence containing a portion of the glucoamylase promoter and a portion of the glucoamylase 5'-coding region was cloned upstream from a DNA sequence corresponding to the amino-terminal portion of preprochymosin. Next, nucleotides between the DNA fragments were deleted by M13 site-specific mutagenesis (40) using specific primer sequences. Finally, a segment of DNA containing the fused sequences was incorporated with the remaining portion of the prochymosin sequence into an expression vector which employed the 5'- and 3'-regulatory sequences of the Aspergillus niger glucoamylase gene. These steps are outlined in FIGS. 3 through 7.</p>
    <p>A. Construction of mpl9 GAPR</p>
    <p>Plasmid pGa5 is used to derive a 337 bp EcoRl-RsaI DNA fragment (fragment A) bearing a portion of the glucoamylase promoter and an amino-terminal segment of the coding region. Fragment A was ligated with EcoRl and SmaI digested M13mp19 RF-DNA (fragment B). The ligation mixture was used to transform E. coli. JM101 (ATCC 33876). Clear plaques were analyzed for the presence of fragment A by restriction analysis of the corresponding RF-DNA. One isolate containing fragment A, designated mp19R-Rsa was digested with PstI and XbaI and the large fragment (fragment C) was isolated. A small XbaI-PstI sequence (fragment D) derived from pR3 (49) containing 5' preprochymosin sequences; was purified by electrophoresis and ligated to fragment C to produce the phage template mp19GAPR as shown in FIG. 3.</p>
    <p>B. Site Specific Deletion Mutagenesis</p>
    <p>As shown in FIG. 8 mp19GAPRΔC1 was derived from mp19 GAPR by deleting the nucleotides between the glucoamylase signal peptide codons and the codons for prochymosin by site-specific mutagenesis. Thus, in mp19GAPRΔC1 the glucoamylase signal peptide codons are precisely fused to the first codon of prochymosin. Site-specific miutagenesis was done as previously described (40) except that only one oligonucleotide was used to prime second strand synthesis on the single-stranded M13 template (FIG. 4) (40). The synthetic oligonucleotide used to derive mp19GAPRΔC1 (primer 1) was 5' GCTCGGGGTTGGCAGCTGAGATCACCAG 3'. Plaques containing the desired deletion were identified by hybridization with the primer radio-labeled as previously described.</p>
    <p>In mp19GAPRΔC3 the nucleotides between those immediately preceding the initiation codon of glucoamylase and the ATG start codon of preprochymosin were joined by site-specific mutagenesis using the synthetic oligonucleotide (primer 3)</p>
    <p>5' ACTCCCCCACCGCAATGAGGTGTCTCGT 3'.</p>
    <p>The resulting mutation linked the glucoamylase promoter region precisely to the initiation codon of preprochymosin as depicted in FIG. 8.</p>
    <p>C. Construction of Vectors for the Expression and Secretion of Bovine Chymosin</p>
    <p>As further depicted in FIG. 4 each of the fusions between the glucoamylase sequences and 5' prochymosin sequences (m19GAPRΔC1 and mp19GAPRΔC3) were combined with the 3' prochymosin sequences and the Saccharomyces cerevisiae phosphoglyceratekinase (PGK) terminator in the Aspergillus nidulans transformation vector pDJB3. The replicative form of mp19GAPRΔC1 and mp19GAPRΔC3 was digested with EcoRl and PstI. The smaller fragment (fragment 1) was isolated. Plasmid pBR322 was also digested with EcoRl and PstI and the larger vector fragment (fragment 2) was isolated. Fragments 1 and 2 were joined by ligation and used to transform E. coli. 294. A tetracycline resistant colony containing either plasmid pBR322GAPRΔC1, or pBR322GAPRΔC3 was isolated. Fragment 2 was also treated with E. coli. polymerase I (Klenow fragment). The resulting blunt ended fragment was circularized by ligation ahd used to transform E. coli 294. One tetracycline resistant colony containing plasmid pBR322ΔRP was isolated and then digested with HindIII and SalI. The larger vector fragment (fragment 3) was isolated. The plasmid pCR160 was digested with HindIII and PstI and fragment 5 (containing the yeast PGK terminator fused to 3' prochymosin codons) was isolated. Fragments 3, 4, and 5 were joined by ligation and used to transform E. coli 294. A tetracycline resistant transformant containing plasmid pBR322GAPR.sup.Δ C1 or pBR322GAPR.sup.Δ C3 was isolated.</p>
    <p>Plasmid pCR160 contains the yeast 2 μm origin of replication to allow its maintenance as a plasmid in yeast, the yeast TRP-1 gene as a yeast selection marker, an E. coli origin or replication and ampicillin resistance gene from the plasmid pBR322, and a prorennin expression unit. The prorennin expression unit contained the promoter from the yeast PGK gene, the prorennin coding region, and the terminator from the PGK gene. Construction of this plasmid was accomplished as depicted in FIG. 9 in the following manner: Plasmid YEpIPT (50) was partially digested with HindIII followed by a complete EcoRl digestion, and the vector fragment A was isolated. A second plasmid pPGK-1600 (51) was partially digested with both EoRl and HindIII, and the PGK promoter fragment B was isolated. Fragments A and B were ligated to give the intermediate pintl which was again partially digested with EcoRl and the HindIII, and the vector fragment C was isolated. The PGK terminator fragment D was isolated following HindIII and Sau3A digestion of the plasmid pB1 (52). The prorennin fragment E was isolated by cleaving pR1 (49) DNA with Ecorl and BclI. Fragments C, D, and E were then ligated to produce the yeast expression plasmid pCR160. The nucleotide sequence of the PGK promoter, structural gene and terminator have been reported (53).</p>
    <p>Plasmids pBR322GAPRΔC1 and pBR322GAPRΔC3 contain a complete transcriptional unit for each of the forms of prochymosin. This transcriptional unit contains a precursor prochymosin coding sequence, the glucoamylase promoter, and the yeast PGK terminator. However, derivatives of these plasmids and plasmids pBR322GAPRΔC2 and pBR322GAPRΔ4, described hereinafter, [designated pInt1 (1-4) FIG. 5)] produced no detectable chymosin when used to transform A. nidulans G191. It is not understood why these derivative plasmids failed to express and secrete chymosin. However, in light of subsequent results it appears that the yeast PKG terminator and/or the short glucoamylase promotor sequence in these plasmids is not recognized by A. nidulans G191. Based on these results, the pBR322GAPRΔC plasmids were further modified.</p>
    <p>In the following steps the transcriptional unit was moved onto the Aspercillus nidulans transformation vector pDJB3. Additional glucoamylase 5' flanking sequences were incorporated just 5' of the promoter to insure the presence of possible upstream activation sites which could be involved in regulating expression. Further, the PKG terminator was replaced with the A. niger glucoamylase terminator from pGa5. Specifically, in FIG. 5 each plasmid (pBR322GAPRΔC1 or pBR322GAPRΔC2) was digested with ClaI and fragment 6 was isolated. Plasmid pDJB3 was also digested with ClaI and treated with bacterial alkaline phosphatase in order to minimize self-ligation. This digested plasmid (fragment 7) was joined to fragment 6 and one ampicillin resistant colony containing plasmid pINTI-1 or pIntI-3 was isolated. These plasmids were digested with XhoI and NsiI and the larger vector fragment (fragment 8) was isolated. Plasmid pGa5 which contains the entire glucoamylase gene as well as extensive 5' and 3' flanking sequences was digested with XhoI and NsiI and the smaller fragment (fragment 9, containing approximately 1700 bp of these 5' sequences) was isolated. Fragments 8 and 9 were joined by ligation and used to transform E. coli 294. One ampicillin resistant colony containing plasmid plnt2-1 or plnt2-3 was isolated. These plasmids differ most significantly from the final vectors (see FIG. 7) in that they contain the yeast PGK terminator rather than the glucoamylase terminator.</p>
    <p>Additional steps in the construction of chymosin expression vectors are outlined in FIG. 6. Plasmid pR1 (49) was used to isolate a small BclI-Asp718 DNA fragment (fragment A) which comprised the 3'--end of prochymosin cDNA. Fragment A was subsequently cloned into pUC18 (33) that was digested with Asp718 and BamHI (fragment B). Similarly, a 1.2 kb ClaI-Asp718 DNA fragment (fragment D) was isolated from plasmid pGa5, and cloned into AccI and Asp718 cleaved pUC18 (fragment C). The resulting intermediate plasmids, pUC-int1 and pUC-int2, were digested with SalI and HindIII, and fragments E and F were isolated. These fragments were then ligated to produce pUC-int3 which contained the 3' end of prochymosin followed by the glucoamylase terminator sequences on a HindIII-Asp718 fragment (fragment H).</p>
    <p>A new cloning vector, designated pBR-link, was created by inserting a synthetic oligonucleotide linker (containing XhoI and ClaI sites) into the unique BamHI site of pBR322. This linker connoted the following seguence:</p>
    <p>5' GATCCATCGATCTCGAGATCGATC 3'</p>
    <p>3' GTAGCTAGAGCTCTAGCTACCTAG 5'</p>
    <p>The larger HindIII-XhoI fragment of this vector (fragment G) was purified by electrophoresis. Similarly, the XhoI-Asp718 restriction fragments (fragments I) of plasmids pInt2-1 and pInt2-3 were isolated electrophoretically. Fragments G and H were ligated with each of the different I-fragments in a series of three-way ligations to produce the intermediates pInt3-1 and plnt3-3. These key intermediates contained the glucoamylase promoter regions, various signal and propeptide fusions to the prochymosin (or preprochymosin) sequences followed by the glucoamylase terminator region all within convenient ClaI restriction sites. Because certain ClaI sites, such as those in the linker of pBR-link, are inhibited by E. coli. DNA methylation, the plasmids pInt3-1 through pInt3-4 were transformed into a dam-strain of E. coli, designated GM48, (ATCC 39099) from which the plasmids were re-isolated. The unmethylated DNA was digested with ClaI and fragment J was purified by electrophoresis. Fragment J from each of the glucoamylase-pro-chymosin fusions was subsequently cloned into the unique ClaI site of pDJB3 (fragment K) to produce the final expression vectors pGRG1 and pGRG3.</p>
    <p>D. Expression and Secretion of Bovine Chymosin</p>
    <p>Aspergillus nidulans G191 was transformed with pGRG1 and pGRG3 as previously described.</p>
    <p>Five pGRG1 and five pGRG3 transformants were analyzed. Western analysis (not shown) indicated that each transformant secreted a protein which reacted with anti-chymosin and which migrated at the same or slightly higher molecular weight of bovine chymosin. The higher molecular weight species may be due to incorrect processing, media effects, or glycosylation. Integration was confirmed for one transformant of pGRG3 by Southern hybridization (results not shown). Each transformant was also assayed for chymosin activity. The results of this assay are shown in Table II.</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">              TABLE II______________________________________                        Range of   No. of Transformants Chymosin  Transformant Tested Activity μg/ml______________________________________pDJB3      1                 0-0.13  pGRG1 5 0-1.5  pGRG3 5 0.05-7.0______________________________________</pre>
    
    <p>These results indicate that pGRG2 and pGRG3 both secrete a protease, at various levels, above the pDJB3 control. Occasionally, background proteolytic activity was detected in pDTB3 control broths. As will be shown hereinafter this protease activity of transformants is associated with the aspartic acid family of carboxyl proteases of which chymosin is a member.</p>
    <heading>EXAMPLE 3</heading> <heading>Expression and Secretion of Fusion Polypeptides from Aspergillus Nidulans</heading> <p>Two fusion polypeptides were constructed for expression and secretion from A. nidulans. One fusion polypeptide contained an amino-terminal portion consisting of the pro sequence and first ten amino acids of Aspergillus niger glucoamylase and a carboxyl-terminal portion consisting of bovine prochymosin. The second fusion polypeptide contained an amino-terminal portion consisting of the pro sequence only of Aspergillus niger glucoamylase and a carboxyl-terminal portion consisting of bovine prochymosin.</p>
    <p>A. Vectors for Expressing and Secreting Fusion Polypeptides</p>
    <p>Vectors encoding the above fusion polypeptides were constructed by deleting specific sequences from mp19GAPR followed by the same manipulations as described above for constructing pGRG1 and pGRG3. As shown in FIG. 8, in mp19GAPRΔC2 the nucleotides between the glucoamylase propeptide codons and the codons of prochymosin were deleted using the site-specific mutagenesis method described above. The sequence of the oligonucleotide synthesized for this mutagenesis (primer 2) was</p>
    <p>5' TGATTTCCAAGCGCGCTGAGATCACCAG 3'.</p>
    <p>This mutation was intended to fuse the glucoamylase promoter, signal peptide, and propeptide codons to the first codon of prochymosin. In mp19GAPRΔC4 the nucleotide seguences between the tenth codon of mature glucoamylase and the codons of prochymosin were deleted by M13 site-specific mutagenesis with the synthetic oligonucleotide (primer 4)</p>
    <p>5' TGAGCAACGAAGCGGCTGAGATCACCAG 3'.</p>
    <p>This deletion fused the glucoamylase promoter region, signal peptide sequence, propeptide sequence, plus ten codons of the mature glucoamylase to the codons of prochymosin as shown in FIG. 8. These expression and secretion vectors designated as pGRG2 and pGRG4 were used to transform A. nidulans.</p>
    <p>B. Expression and Secretion of Chymosin from Aspergillus Nidulans Transformed With PGRG2 and PGRG4</p>
    <p>pGRG2 and pGRG4 transformants were cultured as previously described. The culture medium was assayed for chymosin activity by Western blot and gave results similar to those obtained for pGRG1 and pGRG3. Integration of one pGRG2 transformant was confirmed by Southern Analysis (results not shown). The results of the chymosin assay are presented in Table III.</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">              TABLE III______________________________________                        Range of   No. of Transformants Chymosin  Transformant Tested Activity μg/ml______________________________________pDJB3      1                 0-0.13  pGRG2 1 0.001-0.42  pGRG4 6 0.004-0.75______________________________________</pre>
    
    <p>Again each of the transformants demonstrated protease activity above the pDJB3 control indicating that a protease was expressed and secreted by the transformants. As with pGRG1 and pGRG3, these proteases belong to the aspartic acid family of carboxyl proteases as evidenced by the pepstatin inhibition. Significantly, these results indicate that hybrid polypeptides have been expressed in a filamentous fungus.</p>
    <heading>EXAMPLE 4</heading> <heading>Pepstatin Inhibition Study</heading> <p>Three of the above vectors containing the various constructions involving chymosin were analyzed in the pepstatin inhibition assay as described supra. The results are shown in Table IV.</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">              TABLE IV______________________________________           Chymosin  Sample Activity (μ/ml)______________________________________pDJB3           0  pDJB3 pepstatin 0  PGRG1 0.2  pGRG1 pepstatin 0.05  pGRG2 0.1  pGRG2 pepstatin 0  PGRG3 3  pGRG3 pepstatin 0.6______________________________________</pre>
    
    <p>The samples preincubated with pepstatin show a marked decrease in activity indicating that the protease produced by the transformants is of the aspartic acid family of acid proteases to whic chymosin is a member. This data together with the results from the Western analysis indicates that biologically active chymosin is expressed and secreted by A. nidulans G191 transformed with pGRG1, pGRG2, pGRG3 and pGRG4.</p>
    <p>The variation in the amount of chymosin activity detected for different vector constructions in Example II and Example III may reflect differences in the recognition of the various signals incorporated in each transformation vector. Within a particular construction, the variation in chymosin activity may be related to the copy number of the vector incorporated into the fungal genome and/or to the location of such integration.</p>
    <heading>EXAMPLE 5</heading> <heading>Carbon Source Studies</heading> <p>One vector, pGRG4, was used to transform A. nidulans G191 which was thereafter grown on the various carbon sources previously described. The results of this assay are shown in Table V.</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">              TABLE V______________________________________Amount of chymosin activity produced on various carbon  sources (.sup.μ g/ml)         starch glucose   fructose                                sucrose______________________________________pDJB3     0      0           0     0  pGRG4 3.5 3.5 0.9 1.75______________________________________</pre>
    
    <p>These results clearly show that chymosin is secreted regardless of the carbon source. This suggests that transcriptional regulation of the glucoamylase promotor is unlike that in A. niger, i.e. not strongly inducible by starch.</p>
    <heading>EXAMPLE 6</heading> <heading>Expression and Secretion of Mucor meihei Carboxyl Protease</heading> <p>A. Carboxyl Protease Genomic Probe</p>
    <p>The partial primary structure of Mucor miehei acid protease (54) was inspected for the region of lowest genetic redundancy. Residues 187-191 (using the pig pepsin numbering system), try-tyr-phe-trp-asp, were selected. Oligonucleotides complementary to the coding sequence corresponding to this amino acid sequence,</p>
    <p>5'-GC(G/A)TCCCA(G/A)AA(G/A)TA(G/A)TA-3',</p>
    <p>were synthesized (31) and labelled using gamma 32P-ATP and T4 polynucleotide kinase (30).</p>
    <p>B. Cloning of Mucor meihei Carboxyl Protease</p>
    <p>Genomic DNA from Mucor iniehei (Centraal Bureau Voor Schimmelcultures, Holland 370.75) was prepared as follows. Cells grown in YMB medium (3 g/l yeast extract, 3 g/l malt extract, 5 g/l peptone, 10 g/l glucose) were collected by centrifugation, washed twice with 0.5M NaCl, and lyophilized. Cell walls were then disrupted by adding sand to the cells and grinding the mixture with a mortar and pestle. The resulting powder was suspended (15 ml. per gram dry weight) in a solution containing 25% sucrose, 50 mM Tris-HCl (pH 8.0), and 10 mM EDTA. SDS was added to a final concentration of 0.1% and the suspension was extracted once with a half-volume of phenol and three times with half volumes of chloroform. The final aqueous phase was dialysed extensively against 10 mM Tris-HCl, pH 8.0 and 1 mM EDTA. The DNA was then precipitated by the addition of sodium acetate, pH 5.5, to a concentration of 0.3 M. followed by the addition of 2.5 volumes of cold ethanol. Aliquots of this DNA were digested with a variety of restriction endonucleases according to the manufacturers' directions and then analyzed for sequences complementary to sequences of the probes described above, using the method of Southern. A positively hybridizing band of approximately 2.5 kb (kilobases) was identified in the HindIII digested DNA. HindIII digested genomic DNA was separated by polyacrylamide gel electrophoresis and a gel fragment containing DNA of 2.0-3.0 kb was electroeluted as previously described. The electroeluted DNA, presumed to be enriched for sequences corresponding to the Mucor miehei acid protease gene, was ethanol precipitated. The cloning vector pBR322 (ATCC 37019) was digested with HindIII and dephosphorylated using bacterial alkaline phosphatase. In a typical 10 ul reaction 100 ng of vector and 100 mg of the size enriched DNA were joined in the presence of ATP and T4 DNA ligase. The reaction was used to transform E. coli 294 (ATCC 31446) by the calcium shock procedure (30). About 2.0×10<sup>4</sup> ampicillin resistant clones were obtained. Approximately 98% of these contained cloned inserts as indicated by their failure to grown on tetracycline containing medium. These colonies were tested by a standard colony hybridization procedure for the presence of sequences complementary to those of the DNA probes. One positively hybridizing colony, containing plasmid pMe5'muc, was found to contain a HindIII insert of the expected 2.5 kb size. The termini of this fragment were subcloned into M13 sequencing vectors (33) and their sequences determined by the dideoxy chain termination method. One terminus contained sequences corresponding to the known amino terminal amino acid sequence of the acid protease gene. The adjacent 3' region was seguenced in order to obtain more C terminal coding sequences. The sequencing strategy is shown in FIG. 10. In this way the entire coding sequence for the mature form of the protein was obtained. The 5' end of the fragment was found to occur 112 bp (base pairs) upstream of the codon corresponding the mature amino terminus. Since this upstream region contained no in frame initiation codons it was presumed to be part of a propeptide.</p>
    <p>In order to obtain DNA containing the initiation codon as well as 5' untranslated sequences a more 5' clone was isolated as follows. A HindIII-ClaI 813 bP 5' subfragment of the pMe5'Cla Hind III insert was isolated and labelled by the nick translation method (30). This labelled fragment was used to probe ClaI digested Mucor miehei genomic DNA by the method of Southern. This experiment revealed a single band of hybridization corresponding to a molecular weight of approximately 1300 bp. Size enriched DNA of this size was isolated and cloned into ClaI digested and dephosphorylated pBR322 as described above.</p>
    <p>Approximately 9000 ampicillin resistant colonies were obtained. About 90% of these contained cloned inserts as indicated by their failure to grow on tetracycline containing medium. These colonies were tested by a standard colony hybridization procedure for the presence of sequences complementary to those of the nick translated probe. One positively hybridizing colony, containing plasmid pMe2, was found to contain a ClaI insert of the expected 1.3 kb size. Sequencing of the ends of this fragment showed that one terminus corresponded to sequences near the ClaI site of the Hind III fragment in pMe5'Cla and thus permitted orientation of the fragment which is shown in FIG. 10. Further sequencing of the ClaI fragment disclosed the initiation codon and 51 untranslated sequences. The entire coding sequence and the 5' and 3' flanking sequences are shown in FIG. 10. Comparison of the deduced primary structure with that determined by direct amino acid sequencing indicates that the Mucor protein is made as a precursor with an amino-terminal extension of 69 residues. Based on the structural features generally present in leader peptides it is likely that residues -21 to -1 comprise a leader peptide and that residues 21-69 comprise a propeptide analogous to that found in the zymogen forms of other acid proteases including chymosin and pepsin (55).</p>
    <p>C. Mucor meihei carboxyl protease Expression and Secretion Vector</p>
    <p>A vector for expressing and secreting Mucor miehei carboxy protease includes the entire native Mucor miehei acid protease transcriptional unit including the coding sequence, 5' flanking sequences (promoter), and 3' flanking sequences (terminator and polyadenylation site).</p>
    <p>The overall strategy for making this vector is depicted in FIG. 12. The Aspergillus nidulans transformation vector pDJB1 was digested with ClaI and EcoRl and the larger vector fragment (fragment 1) was isolated. The plasmid pMe5'Cla was digested with EcoRl and ClaI and fragment 2 was isolated. This fragment contains the 5' codons of the acid protease together with about 500 bp of 5' flanking sequences. Fragments 1 and 2 were joined by ligation and used to transform E. coli 294. One ampicillin resistant colony containing plasmid pMeJBint was isolated. This plasmid was digested with ClaI and treated with bacterial alkaline phosphatase in order to reduce self ligation and is designated fragment 3. Plasmid pMe2 was digested with ClaI and the smaller fragment (fragment 4) was isolated. This fragment contains the Mucor miehei acid protease 3' codons and about 1800 bp of 3' flanking sequences. Fragments 3 and 4 were joined by ligation and used to transform E. coli 294. One ampicillin resistant colony containing plasmid pMeJB1-7 was isolated. This vector was used to transform Aspergillus nidulans.</p>
    <p>D. Expression and Secretion of Mucor miehei Carboxyl Protease by Aspergillus nidulans</p>
    <p>Southern blot analysis of six transformants indicated the presence of the entire Mucor miehei acid protease gene in the Aspergillus nidulans genome (results not shown). In addition, each of the transformants were analyzed by Western blots (results not shown) and for acid protease activity. The results of the protease assay are shown in Table VI.</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">              TABLE VI______________________________________          Protease  Transformant Activity (mg/ml)______________________________________1              0.003  2 0.007  3 0.003  4 0.005  5 0.005  6 0.012______________________________________</pre>
    
    <p>These experiments demonstrate expression and secretion of a protein that reacts with specific antibody to Mucor miehei carboxyl protease and which has milk clotting activity. The protein has an apparent molecular weight by electrophoretic analysis that is slightly greater than that of the authentic (Mucor miehei derived) protein. This may indicate that Aspergillus nidulans glycosylates this glycoprotein to a different than Mucor miehei. Because the Aspergillus nidulans derived Mucor meihei acid protease appears to have the same specific activity as the authentic material it appears that it has been processed (by the cell or autocatalytically) to the mature form. The unprocessed forms of other acid proteases such as chymosin and pepsin are zymogens which require processing (autocatalytic) before activity is obtained.</p>
    <p>The varying levels of expression in the various transformants may reflect the position or copy number of the protease gene in the Aspercillus nidulans genome. However, the expression and secretion of biologically active carboxyl protease indicates the A. nidulans recognizes the promoter, signal and terminator signals of Mucor miehei carboxyl protease.</p>
    <heading>EXAMPLE 7</heading> <heading>Expression and Secretion of Chymosin Encoded by pGRG1-4 from A. Awamori and Trichoderma reesei pyrG Auxotrophic Mutants</heading> <p>The plasmids pGRG1 through pGRG4 (pGRG1-4) were also used to transform orotidine-5'-phosphate decarboxylase (OMPCase) deficient mutants of A. awamori and Trichoderma reesei. The pvr4 gene from N. crassa encoded by the pGRG1-4 plasmid complements these OMPCase mutants in the absence of uridine to permit the isolation of successful transformants. The thus transformed mutants of A. awamori and T. reesei secreted detectable amounts of bichemically active chymosin into the culture medium.</p>
    <p>A. Production of pvrG Auxotrophs</p>
    <p>The method used to obtain pyrG auxotrophic mutants of A. awamori and T. reesei involved selection on the pyrimidine analog 5-fluoro-orotic acid (FOA) (56). The mechanism by which FOA kills wild-type cells is unknown. However, in view of the resistance of OMPCase-deficient mutants to FOA, it is likely that the toxicity occurs through conversion of FOA to 5-fluoro-UMP. Whether cell death is caused by a flouoridated ribonucleotide or deoxyribonucleotide is uncertain.</p>
    <p>The following methods describe the isolation of OMPCase-deficient (FOA-resistant) mutants of T. reesei and A. awamori:</p>
    <heading>1. Trichoderma reesei</heading> <p>A fresh spore suspension of T. reesei strain P37 (NRRL 15709) was washed three times in sterile distilled water containing 0.1% Tween-80. Fifteen milliliters of this spore suspension (1×10<sup>7</sup> spores per ml) were placed in a sterile petri dish (100×20 mm) with a sterile magnetic stirring bar. The lid was removed and the spores were exposed to ultraviolet (UV) light at 254 nm (7000 uW per cm2), in the dark at a distance of 25 cm from the UV light source. The spores were stirred constantly. UV exposure continued for three minutes (sufficient to give 70% killed spores). The irradiated spore suspension was collected in a 50 ml centrifuge tube and stored in the dark for one hour to prevent photoreactivation. Spores were pelleted by centrifugation and the pellet was resuspended in 200 ul of sterile water containing 0.01% Tween-80.</p>
    <p>The suspension was diluted and plated onto YNB agar medium (0.7% yeast nitrogen base without amino acids, 2% glucose, 10 mM uridine, 2% agar) (56) containing 0.15% FOA (SCM Specialty Chemicals, Gainsville, Fla.). After 4 days incubation at 30° C., 75 colonies were picked to fresh YNB agar that contained FOA. Sixty-two of the 75 colonies grew and were toothpicked to minimal agar (6 g/l NANO<sub>3</sub>, 0.52 g/l KCl, 1.52 g/l KH<sub>2</sub> PO<sub>4</sub>, 1 ml/l trace elements solution, 1% glucose, 0.1% MgSo<sub>4</sub>, 20 g/l agar) and minimal agar plus 1 mg/ml uridine to determine uridine requirements. All of the 62 isolates grew on minimal agar with uridine, but 9 isolates failed to grow on minimal agar alone. These 9 strains were repicked to minimal agar and minimal agar with uridine. Two of the strains grew only on minimal agar supplemented with uridine. One of these, designated T. reesei pyrG29, grew well on minimal medium with uridine with no background growth on minimal medium alone.</p>
    <heading>2. Aspergillus awamori</heading> <p>(i) Production of A. awamori strain UVK 143f- a Hyperproducer of Glucoamylase</p>
    <p>Spores of A. awamori strain NRRL 3112 were obtained after 5-7 days growth on Potato Dextrose Agar (PDA, Difco Co.) at 30° C. Spores were harvested by washing the surface of the plate with sterile 0.1% Tween-80 in distilled H<sub>2</sub> O and gently scraping the spores free. Spores were washed by centrifugation and resuspended in the same buffer to give a final concentration of between 1×10<sup>7</sup> to 2×10<sup>8</sup> spores/ml. Preparations were stored at 4° C.</p>
    <p>Two ml of spores was added to a sterile petri plate. The top of the dish was removed and spores were exposed to an ultraviolet (UV) lamp (15 watt, germicidal). Conditions of time exposure and distance from the lamp were adjusted such that 90 to 99.9% of the spores were killed. Surviving spores were plated onto PDA medium and grown to form discrete independent colonies.</p>
    <p>Spores from individual mutagenized colonies were inoculated into 50 ml of screening media consisting of 5% corn meal, 0.5% yeast extract, 2% corn steep liquor, adjusted to pH 4.5 prior to sterilization in 250 ml flasks. However, any number of media containing corn or corn starch as the carbon source would be expected to give similar results. Cultures were grown for 4-5 days at 30-35° C. with constant shaking. Samples were removed either daily or at the end of the run for assays.</p>
    <p>Estimates of glucoamylase activity were made by measuring the release of a color producing group (para-nitro-phenol) from a colorless substrate (para-nitro-phenyl-alpha-glucoside, PNPAG).</p>
    <p>The following protocol was utilized:</p>
    <p>Substrate--180 mg PNPAG was dissolved in 250 ml of 0.1M NaAcetate buffer, pH 4.7. Store at 4° C.</p>
    <p>Assay--1 ml of substrate was equilibrated at 40° C. in a water bath. 0.2 ml of sample (or diluted sample) was added and incubated for 30 minutes at 40° C. 9 ml of 0.1M Na<sub>2</sub> Co<sub>3</sub> was added with the mixture being kept at room temperature for 15 minutes for color development. The mixture was filtered through Wattman 42 filter paper and the absorbance at 420 nm was read in a suitable spectrophotometer. All mutant PNPAG levels were compared to the standard amount produced by the parent strain and were reported as percent of PNPAG hyrolysis of the parent.</p>
    <p>One glucoamylase-hyperproducing strain designated UVK 143f was selected for auxotrophic mutagenesis.</p>
    <p>(ii) Auxotrophic Mutagenesis</p>
    <p>Preparation of spores from A. awamori strain UVK143f, UV mutagenesis, and mutant analysis were the same as for T. reesei with the following modifications:</p>
    <p>a. 2.5 minutes was required to give 70% killing with UW light.</p>
    <p>b. Minimal medium was used instead of YNB agar.</p>
    <p>c. The FOA concentration was 0.1%.</p>
    <p>Fifteen pyrG mutants were found. Three of these isolates, designated pyr4-5, pyr4-7, and pyr4-8 were selected for transformation experiments.</p>
    <p>B. Transformation of A. awamori and T. reesei pyr Auxotrophs</p>
    <p>A. awamori and T. reesei auxotrophs were transformed by a modification of the procedure previously described for A. nidulans. Approximately 1×10<sup>8</sup> spores were inoculated into yeast extract glucose (YEG) medium which is 2% glucose, 0.5% yeast extract supplemented with 1 mg/ml uridine. The cultures were incubated on a 37° C. shaker (200 rpm) for 12 to 15 hours [T. reesei was incubated at 30° C.]. Germlings were harvested by centrifugation, washed once with sterile YEG medium, then incubated at 30° C. in 50% YEG medium containing 0.6 M KCl, 0.5% Novozyme 234 (Novo Industries, Denmark), 0.5% MgSO<sub>4</sub> ·7<sub>2</sub> O, 0.05% bovine serum albumin in a sterile 200 ml plastic bottle (Corning Corp., Corning, N.Y.). After 30 minutes of shaking at 150 rpm, the protoplast suspension was vigorously pipetted up and down five times with a 10 ml pipette to disperse the clumps. The protoplast suspension was further incubated as above for one hour then filtered through sterile miracloth (Calbiochem-Behring Corp., LaJolla, Calif.) that was wet with 0.6 M KCl. The filtered protoplasts were centrifuged, washed, and transformed with each of the plasmids pGRG1-4 as described previously.</p>
    <p>The following modifications were made for A. awamori transformation:</p>
    <p>1. 0.7 M KCl was used instead of 0.6 M KCl.</p>
    <p>2. 1.2 M sorbitol was used instead of 0.6 M KCl to osmotically stabilize the transformation and regeneration medium.</p>
    <p>C. Analysis of A. awamori and T. reesei Transformants</p>
    <p>Both A. awamori and T. reesei transformants secreted chymosin polypeptides into the culture medium. This was determined by analyzing culture filtrates (results not shown) for both enzymatically active chymosin (milk clotting assay) and chymosin polypeptides that reacted with specific chymosin antibodies (enzyme immunoassays and Western immunoblotting techniques).</p>
    <heading>EXAMPLE 8</heading> <heading>Expression and Secretion of Heterologous Polypeptides from argB Auxotrophic Mutants of Aspergillus Species</heading> <p>The expression and secretion of heterologous polypeptides from argB auxotrophs of Aspergillus species has also been achieved.</p>
    <p>This example describes the complementary transformation of A. nidulans and A. awamori argB auxotrophs with vectors containing the argB gene from A. nidulans and DNA sequences encoding the heterologous polypeptides of plasmids pGRG1-4. The argB gene encodes ornithine transcarbamylase (OTC).</p>
    <p>The A. nidulans argB auxotroph containing the genetic markers biA1, argB2, metG1 used herein was obtained from Dr. P. Weglenski, Department of Genetics, Warsaw University, Al. Ujazdowskie 4,00-478 Warsaw, Poland. The A. awamori argB mutant was derived as follows.</p>
    <p>A. Isolation of Aspergillus awamori argB Auxotrophic Mutants</p>
    <p>A fresh suspension of A. awamori strain UVK 143k spores was prepared and UV mutagenesis was performed as described above except that the exposure time was sufficient to kill 95% of the spores. The spores were then centrifuged, washed with sterile water, and resuspended in 25 ml of sterile minimal medium. These suspensions were incubated in a 37° C. shaker with vigorous aeration. Under these conditions, wild-type spores will germinate and grow into vegetative mycelia, but auxotrophic mutants will not. The culture was aseptically filtered through sterile miracloth every six to eight hours for three days. This step removes most of the wild-type mycelia while the ungerminated auxotrophs pass through the miracloth filter (i.e., filtration enrichment). At each filtration step the filtered spores were centrifuged and resuspended in fresh minimal medium. After three days of enrichment the spores were diluted and plated on minimal agar supplemented with 50 mM citrulline. The plates were incubated at 37° C. for two to three days. Individual colonies were toothpicked from these plates to two screening plates--one plate that contained minimal agar plus 10 mM orhithine and one plate that contained minimal agar plus 50 mM citrulline. The rationale for picking colonies to these screening plates is as follows. OTC (the argB gene product) catalyzes the conversion of ornithine to citrulline in the arginine biosynthetic pathway. Thus argB mutants (deficient in OTC) will grow on minimal medium plus citrulline but not on minimal medium with ornithine. Screening of approximately 4000 colonies by this method yielded 15 possible araB mutants. One of these strains, designated A. awamori argB3, gave no background growth on minimal medium and grew very well on minimal medium supplemented with either arginine or citrulline. Assays to determine the level of OTC activity (57) indicated that the argB3 mutant produced at least 30-fold less OTC activity than wild-type. On the basis of these data the A. awamori argB3 strain was selected for transformation experiments.</p>
    <p>B. Construction of argb-based Prochymosin Expression Vectors for Transformation of Aspergillus Species</p>
    <p>In this construction (see FIG. 15) the first step was to combine the transformation enhancing sequence ANS-1 and the selectable argB gene on the same plasmid. In order to achieve this, plasmid pBB116 (59), which contains the argB gene from A. nidulans, was digested with PstI and BamHI and the indicated fragment A, which contains the argB structural gene, was isolated. Plasmid pDJB2 (59) was digested with EcoRl and EstI, and the indicated fragment B, which contains the ANS-1 sequence, was isolated. In a three part ligation fragments A and B were joined together with fragment C, which contains the large EcoRl-BamHI fragment of plasmid vector pUC18 (33) to give plasmid pARG-DJB.</p>
    <p>In the second step of this construction a synthetic DNA polylinker containing ClaI sites was inserted into pARG-DJB in order to allow the insertion of ClaI fragments which contain various prochymosin expression units. Plasmid pARG-DJB was digested with BamHI and then dephoshporylated with bacterial alkaline phosphatase. The indicated synthetic DNA polylinker was phosphorylated with T4 polynucleotide kinase, and then ligated to the cleaved pARG-DJB to give pCJ16L. Because this plasmid was found to be resistant to digestion with ClaI, it was first used to transform the E. coli dam- mutant strain GM48 in order to prevent methylation of the ClaI sites. Upon isolation of the plasmid from GM48 transformants, it was successfully cleaved with ClaI and dephosphorylated with bacterial alkaline phosphatase.</p>
    <p>In the final step of this construciton of ClaI-cleaved pCJ16L vector was joined to each of the ClaI prochymosin. expression fragments from plasmids pGRG1 through pGRG4. The resulting four plasmids, pCJ::GRG1 through pCJ::GRG4, were used to transform the argB mutants of A. nidulans and A. awamori to prototrophy. Resulting transformants were analyzed for expression of prochymcsin polypeptides.</p>
    <p>C. Analysis of A. nidulans and A. awamori transformants</p>
    <p>Secreted chymosin polypeptides from A. awamori and A. nidulans transformed with pCJ::GRG1 through pCJ::GRG4 were detected by the milk clotting assay and by enzyme immunoassays and Western immunoblotting techniques. In each case (results not shown) the transformed fungi secreted biochemically active chymosin into the culture medium.</p>
    <heading>EXAMPLE 9</heading> <heading>Expression and Secretion of Humicola grisea Gluccamylase from A. nidulans</heading> <p>The glucoamylase gene from the fungus Humicola grisea was isolated and cloned. This gene was thereafter ligated into the ArgB expression plasmid pCJ16L. The resulting vectors, pCJ:RSH1 and pCJ:RSH2 were used to transform argB deficient A. nidulans (Example 8) which resulted in the expression and secretion of Humicola grisea glucoamylase.</p>
    <p>A. Isolation and Cloning of Humicola grisea Glucoamylase Gene</p>
    <heading>1. Purification of Humicola grisea Glucoamylase</heading> <p>Authentic H. grisea (var. thermoidea NRRL 15219) glucoamylase was obtained from A. E. Staley Company (lot no. 1500-149-8A). The enzyme was purified to homogeneity through chromatography on a 4.6 mm×250 mm Synchrompak C4 reversed phase column (SynChrom, Inc., Linden, Ind.). The column was initially equilibrated with 0.05% triethylamine and 0.05% trifluoroacetic acid (solvent A) at 0.5 ml/min. After injection of the glycomaylase sample (40 μg) the column was washed for 2 minutes with solvent A, and then eluted with a gradient of 5% solvent B per minute (solvent B is 0.05% triethylamine, 0.05% trifluoroacetic acid in acetonitrile) to 40% solvent B. The slope of the gradient was then changed to 0.5% solvent B per minute, and the glucoamylase was eluted at approximately 55% solvent B. At this point the glucoamylase was judged to be homogenous by sodium dodecylsulftte polyacrylamide gel electrophoresis.</p>
    <heading>2. Amino Acid Sequence of H. grisea Glucoamylase</heading> <p>The amino terminal sequence of purified H. grisea glucoamylase was obtained as described previously (60). The sequence read as follows:</p>
    <p>AAVDTFINTEKPSAXNSL</p>
    <p>These and other lettered peptide sequences presented herein refer to amino acid seguences wherein each letter corresponds to the following amino acids:</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">______________________________________Amino acid  or residue 3-letter 1-letter  thereof symbol symbol______________________________________Alanine          Ala     A  Glutamate Glu E  Glutamine Gln Q  Aspartate Asp D  Asparagine Asn N  Leucine Leu L  Glycine Gly G  Lysine Lys K  Serine Ser S  Valine Val V  Arginine Arg R  Threonine Thr T  Proline Pro P  Isoleucine Ile I  Methionine Met M  Phenylalanine Phe F  Tyrosine Tyr Y  Cysteine Cys C  Tryptophan Trp W  Histidine His H______________________________________</pre>
    
    <p>In order to obtain peptide fragments for additional amino acid sequencing, purified glucoamylase (1 mg/ml) was digested in 2% acetic acid for 2 hours at 108° C. The material was injected directly onto a Synchrompak C4 column (4.8 mm×100 mm) equilibrated as described above. After washing for 2 minutes with 100% solvent A (see above), the peptides were eluted with a linear gradient of solvent C (1% per minute). Solvent C was composed of 0.05% triethylamine, 0.05% trifluoroacetic acid in propanol. At this point three peptides were selected for futher analysis. One peptide (GA3) was sequenced directly. A mixture of two other peptides (GA1 and GA2) was subjected to further purification on a 4.8 mm×250 mm Synchrompak C4 column as follows. The mixture of GA1 and GA2 was diluted with three volumes of solvent A and injected onto the column. After washing for 2 minutes, the peptides were eluted with a linear gradient of solvent D (0.5% per minute). Solvent D was 0.05% triethylamine, 0.05% trifluoroacetic acid in 35% propanol:65% acetonitrile. Separated GA1 and GA2 were then purified again using the same protocol and the amino acid sequences were determined as described above. The sequences of peptides GA1, GA2 and GA3 are as follows:  GA1PLWSITVPIKATGXAVQYKYIK    VXQL   - GA2AAVRPLINPEKPIAWNXLKANIGPN   - GA3INTEKPIAWNKLLANIGPNGKAAPGAAAGVVIASPSRTD</p>
    <heading>3. Synthetic Oligonucleotide Probes</heading> <p>The genomic DNA encoding the H. grisea glucoamylase gene was cloned as follows. A synthetic mixture of 48 oligonucleotides was used as a hybridization probe to detect the glucoamylase gene. The oligonucleotides were 17 bases in length (17 mer) and corresponded to a sequence of six amino acids (underlined in the GA1 peptide, supra) from H. grisea glucoamylase:Gln Tyr Lys Tyr Ile Lys  5' CAA TAT AAA TAT ATT AA 3'  G   C   G   C   C                  A</p>
    <p>The mixture of 48 oligonucleotides was synthesized in six pools, each containing eight different synthetic 17 mers.  pool 1: 5' CAATATAAATATATTAA 3'   G  C  G  pool 2: 5' CAATATAAATACATTAA 3'   G  C  G  pool 3: 5' CAATATAAATATATCAA 3'   G  C  G  pool 4: 5' CAATATAAATACATCAA 3'   G  C  G  pool 5: 5' CAATATAAATATATAAA 3'   G  C  G  pool 6: 5' CAATATAAATACATAAA 3'   G  C  G</p>
    <p>The oligonucleotides were synthesized on a Biosearch automated DNA synthesizer (Biosearch, San Rafael, Calif.) using reagents and protocols specified by the manufacturer.</p>
    <heading>4. Selection of Correct Oligonucleotide Probe</heading> <p>Genomic DNA from H. grisea was analyzed for the presence of glucoanylase sequences by the method of Southern (30). Briefly, H. grisea DNA was digested with BamHI restriction endonuclease. Six aliquots of this digested DNA (one for each probe pool) were fractionated according to size by electrophoresis on a 1% agarose gel. After blotting the DNA to nitrocellulose, as previously described, the DNA was fixed to the nitrocellulose filter at 80° C. in a vacuum oven. The filter was cut into six strips, corresponding to the six aliquots of BamHI digested DNA, and each strip was hybridized for 18 hours at low stringency (2, 40) with one of the pools of synthetic oliconucleotide probes. (The probes were radiolabeled with gamma-[32P]ATP using T4 polynucleotide kinase as previously described.) After hybridization, the filters were washed 15 minutes in 2×SSC, 0.1% SDS at 37° C., and twice in 2×SSC at the same temperature. Filters were air dried, wrapped in Saran-Wrap, and applied to Kodak XOmat-AR X-ray film to obtain an autoradiographic image. After developing the autoradiogram, a faint band of hybridization corresponding to a 3.7 Kb BamHI fragment was visible from the strip that was hybridized with pool 3.</p>
    <p>In order tc improve the hybridization signal, pool 3 was re-synthesized as eight individual oligonucleotides. The Southern hybridization experiments were repeated using each of the eight oligonucleotides as probes. Only one of these 17 mer probes was found to hybridize to the 3.7 Kb BamHI fragment of H. grisea genomic DNA. The sequence of the oligonucleotide was 5+ CAGTACAAGTATATCAA 3'. This 17 mer was used as the hybridization probe for the cloning of the H. grisea glucoamylase gene.</p>
    <heading>5. Cloning of Glucoamylase Gene Sequences</heading> <p>Genomic DNA from H. grisea was digested with BamHI and size-fractionated by polyacrylamide gel electrophoresis according to standard methods (30). DNA fragments 2 to 4 Kb in size were excised and eluted from the gel. This DNA fraction was used to generate a library of clones in the E. coli cloning vector pBR322 (ATCC 37019). The cloning vector was digested with BamHI and dephosphorylated with bacterial alkaline phosphatase. The phosphatase was removed by extraction with phenol-chloroform (1:1 v/v). The BamHI cleaved size-selected H. grisea DNA was ligated to the BamHI cleaved and dephosphorylated pBR322. The thus ligated DNA was used to transform competent E. coli 294 cells (ATCC 31446) prepared by the method of Morrison (41). Transformants were selected on LB agar plates (30) which contained carbenecillin at a concentration of 50 μg/ml. Transformants which harbored glucoamylase gene sequences were identified by colony hybridization methods (30) using the specific 17 mer (described above) as a probe. Hybridizing colonies were purified, find plasmids were isolated from each by the alkaline-SDS miniscreen procedure (30). The plasmids selected in this manner all contained a 3.7 Kb BamHI fragment which hybridized to the glucoamylaste-specific 17 mer probe. One such plasmid, designated pRSH1, was selected for further analysis.</p>
    <p>A 600 bp Sau3A fragment from pRSH1 was subcloned into bacteriophage M13mp18 (33) and partially sequenced by the dideoxy chain termination method (43) to confirm that the cloned DNA encoded the glucoamylase gene. A restriction endonuclease cleavage map of the 3.7 Kb BamHI fragment contained in pRSH1 is shown in FIG. 16. It was generated following single and double restriction digests followed by orientation of the DNA fragments with respect to known restriction sites in pBR322 (44). On the basis of the DNA sequencing data we obtained and the restriction map, we determined that there was a high probability that the entire coding sequence of the glucoamylase gene was contained within the 3.7 Kb BamHI fragment in pRSH1.</p>
    <p>B. Construction of argB Vector Containing Humicola grisea Glucoamylase Gene</p>
    <p>The 3.7 Kb BamhI fragment from pRSH1 was cloned (in both orientations) into pCJ16L which contains a selectable argB gene from A. nidulans (FIG. 17). The resulting vectors, PCJ:RSH1 and pCJ:RSH2, were used to transform argB-deficient A. nidulans.</p>
    <p>C. Expression and Secretion of H. grisea glucoamylase</p>
    <p>Prototrophic; transformants were purified and innoculated into minimal medium with starch as the sole carbon source (this medium is the same as that described for the production of chymosin except that the pH was adjusted to 5.0). Culture filtrates were assayed for H. grisea glucoamylase activity. FIG. 18 shows the extracellular production of H. grisea glucoamylase by A. nidulans transformed with pCJ:RSH1. The negative control was non-transformed argB deficient A. nidulans.</p>
    <p>Although thea foregoing refers to particular preferred embodiments, it will be understood that the present invention is not so limited. It will occur to those ordinarily skilled in the art that various modifications may be made to the disclosed embodiments and that such modifications are intended to be within the scope of the present invention.</p>
    <p>The references grouped in the following bibliography and respectively cited parenthetically by number in the foregoing text, are hereby incorporated by reference.</p>
    <heading>BIBLIOGRAPHY</heading> <p>1. Hitzeman, R. A., et al., 1984. In Recombinant DNA Products: Insulin-Interferon-Growth Hormone, A. P. Bollon, ed., CRC Press, Boca Raton, Fla.; Goeddel, D. V. et al., 1979. Nature, 281: 544-548.</p>
    <p>2. Haynes, J. et al., 1983, Nucleic Acids Res., 11:687-706; Pennica, D. et al., 1983, Nature, 301:214-221.</p>
    <p>3. Lawn, R., et al., 1981, Nucl. Acids Res., 9:6103-6114.</p>
    <p>4. Wood, W. I., et al., 1984, Nature, 312:330-337.</p>
    <p>5. Heyneker, H. L. et al., European Patent Office Publication No. 0092182 published Oct. 26, 1983.</p>
    <p>6. Tuite, M. F., et al., 1982, EMBO J., 1:603-608.</p>
    <p>7. Mellor, J., et al., 1982, Gene, 24:1-14.</p>
    <p>8. Sibakov, M., et al., 1983, FEMS Microbiol Lett., 17:81-85.</p>
    <p>9. Wells, J. A., et al., 1982, Nucl. Acids Res., 11:7911-7925.</p>
    <p>10. Baty, D., et al., 1981, Gene, 16:79-87.</p>
    <p>11. Sibakov, M., et al., 1984, Genetics and Biotechnology of Bacilli, A. T. Ganesan and J. A. Hoch, eds., Academic Press, N.Y.</p>
    <p>12. Edens, L., et al., 1984, European Patent Office Publication No. EP0129268A2.</p>
    <p>13. Marston, F. A. O., et al., 1984, Biotechnol., 2:800-804.</p>
    <p>14. Foltmann, B., et al., 1979, J. Biol. Chem., 254:8447-8456.</p>
    <p>15. Hayenga, K. J., et al., 1984, European Patent Application EP0114507.</p>
    <p>16. Case, M. E., et al., 1979, Proc. Natl. Acad. Sci. USA, 76:5259-5263; and Lambowitz U.S. Pat. No. 4,486,533.</p>
    <p>17. Kinsey, J. A. and J. A. Rambosek, 1984, Molecular and Cellular Biology 4:117-122.</p>
    <p>18. Yelton, M., et al., 1984, Proc. Natl. Acad. Sci. USA, 81:1470-1474; Mullaney, E. J. et al., 1985, Mol. Gen. Genet., 199:37-45.</p>
    <p>19. John, M. A. and J. F. Peberdy, 1984, Enzyme Microb. Technol., 6:386-389.</p>
    <p>20. Tilburn, et al., 1982, Gene, 26:205-221.</p>
    <p>21. Ballance, D. J., et al., 1983, Biochem. Biophys. Res. Comm., 112:284-289.</p>
    <p>22. Kelly, J. M. and M. Hynes, 1985, EMBO, 4:475-479.</p>
    <p>23. Bull, J. H. and J. C. Wooton, 1984, Nature, 310:701-704.</p>
    <p>24. Barclay, S. L. and E. Meller, 1983, Molecular and Cellular Biology, 3:2117-2130.</p>
    <p>25. Ingolia, P. L., et al., 1984, p. 28, ASM Conf., Genet. and Mol. Bio. Ind. Microorganisms, (Abstr.)</p>
    <p>26. Alexopoulos, C. J., 1962, Introductory Mycology, John Wiley &amp; Sons, Inc., New York.</p>
    <p>27. Innis, M. A., et al., 1985, Science, 228:21-26</p>
    <p>28. Penttila, M. E., et al., 1984, Mol. Gen. Genet., 194:491-499.</p>
    <p>29. Shoemaker, S. P., et al., 1984, European Patent Application EPO137280A1.</p>
    <p>30. Maniatis, T., et al., 1982, Molecular Cloning, Cold Spring Harbor Press, Cold Spring Harbor, N.Y.</p>
    <p>31. Crea, R., et al., 1978, Proc. Natl. Acad. Sci. USA, 75:5765-5769.</p>
    <p>32. Buxton, F. and Radford A., 1983, Mol. Gen. Gent., 190:403-405.</p>
    <p>33. Yanisch-Perron, C., Vieira, J., and Messing, J., 1985, Gene, 33:103-119.</p>
    <p>34. Clutterbuck, J. H., 1974, Aspergillus nidulans, In: Handbook of Genetics. King, R. C. ed., p. 447-510, Plenum Press, New York.</p>
    <p>35. Towbin, H., et al., 1979, Proc. Natl. Acad. Sci. USA, 76:4350-4354.</p>
    <p>36. Uchiyama, H., et al., 1981, J. Biochem, 90:483-487.</p>
    <p>37. Sokol, P. A., et al., 1979, J. Clin. Microbiol., 9:538-540.</p>
    <p>38. Pollock, M. R., 1961, J. Gen. Microbiol., 26:239-253.</p>
    <p>39. Nunberg, J. H., et al., 1984, Mol. Cell. Biol., 4:2306-2315.</p>
    <p>40. Adelman, J. P., et al., 1983, DNA, 2:183-193.</p>
    <p>41. Morrison, D. A. 1979, Methods Enzymol., 68:326-331.</p>
    <p>42. Messing, J., 1983, Methods Enzymol., 101:20-78.</p>
    <p>43. Sanger, F., et al., 1977, Proc. Natl. Acad. Sci. USA, 74:5463-5467.</p>
    <p>44. Bolivar, F., et al., 1977, Gene, 2:95-113.</p>
    <p>45. Bolivar, S., 1978, Gene, 4:121-136.</p>
    <p>46. Wallace, R. B., et al., 1981, Nucl. Acids Res., 9:3647-3656.</p>
    <p>47. Nunberg, J. H., et al., 1984, International Patent Publication No. WO84/020921.</p>
    <p>48. Boel, E., et al., 1984, EMBO J., 2:1581-1585.</p>
    <p>49. European Patent Office Publication No. 0116778 published Aug. 29, 1984.</p>
    <p>50. Hitzeman, R. A., et al., 1983, Science, 219:620-625.</p>
    <p>51. Hitzeman, R. A., et al., 1983, Nucl. Acids Res., 11:2745-2763.</p>
    <p>52. Hitzeman, R. A., et al., 1980, J. Biol. Chem., 255:12073.</p>
    <p>53. Hitzeman, R. A., et al., 1982, Nucl. Acids Res., 10:7791-7808.</p>
    <p>54. Bech, A. M. and Foltmann, B., 1981, Neth. Milk Dairy J., 35:275-280.</p>
    <p>55. Foltmann, B. and Pedersen, V. B., 1977, pp. 3-22, In: Acid Proteases, J. Tang, ed., Plenum Press, N.Y.</p>
    <p>56. Boeke, J. D., et al., 1984, Mol. Gen. Genet., 197:345-346.</p>
    <p>57. Basabe, J. R., et al., 1979, Anal. Biochem., 94:356-360.</p>
    <p>58. Berse, B., et al., 1983, Gene, 25:109-117.</p>
    <p>59. Balance, D. J., et al., 1985, Gene, 36:321-331.</p>
    <p>60. Rodriguez, H., et al., 1984, Anal. Biochem., 134:538-547.</p>
    <p>61. Johnston, I. L., et al., 1985, EMBO J., 4:1307-1311.</p>
    </div></div></div><div class="patent-section patent-tabular-section"><a id="backward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Cited Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4486533">US4486533</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jan 24, 1983</td><td class="patent-data-table-td patent-date-value">Dec 4, 1984</td><td class="patent-data-table-td ">St. Louis University</td><td class="patent-data-table-td ">Filamentous fungi functional replicating extrachromosomal element</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4666847">US4666847</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Dec 1, 1981</td><td class="patent-data-table-td patent-date-value">May 19, 1987</td><td class="patent-data-table-td ">Collaborative Research, Inc.</td><td class="patent-data-table-td ">Recombinant DNA means and method</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4794175">US4794175</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Apr 27, 1987</td><td class="patent-data-table-td patent-date-value">Dec 27, 1988</td><td class="patent-data-table-td ">Cetus Corporation</td><td class="patent-data-table-td ">Degradation starch to glucose</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4816405">US4816405</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Oct 16, 1987</td><td class="patent-data-table-td patent-date-value">Mar 28, 1989</td><td class="patent-data-table-td ">The Regents Of The University Of California</td><td class="patent-data-table-td ">Vectors for transformation by ascomycetes</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4863864">US4863864</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Dec 13, 1985</td><td class="patent-data-table-td patent-date-value">Sep 5, 1989</td><td class="patent-data-table-td ">Suntory Limited</td><td class="patent-data-table-td ">Genetic engineering, dna, cultures</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5198345">US5198345</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Dec 20, 1985</td><td class="patent-data-table-td patent-date-value">Mar 30, 1993</td><td class="patent-data-table-td ">Gist-Brocades N.V.</td><td class="patent-data-table-td ">Aspergillus, DNA construct for transforming host</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP0077109A2?cl=en">EP0077109A2</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Oct 13, 1982</td><td class="patent-data-table-td patent-date-value">Apr 20, 1983</td><td class="patent-data-table-td ">Unilever N.V.</td><td class="patent-data-table-td ">DNA molecules comprising the genes for preprochymosin and its maturation forms, and microorganisms transformed thereby</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP0126206A2?cl=en">EP0126206A2</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jan 26, 1984</td><td class="patent-data-table-td patent-date-value">Nov 28, 1984</td><td class="patent-data-table-td ">Cetus Corporation</td><td class="patent-data-table-td ">Glucoamylase cDNA</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP0184438A2?cl=en">EP0184438A2</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Dec 4, 1985</td><td class="patent-data-table-td patent-date-value">Jun 11, 1986</td><td class="patent-data-table-td ">Gist-Brocades N.V.</td><td class="patent-data-table-td ">Transforming aspergillus niger, and plasmids for use therein</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1986003774A1?cl=en">WO1986003774A1</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Dec 13, 1985</td><td class="patent-data-table-td patent-date-value">Jul 3, 1986</td><td class="patent-data-table-td ">Hansens Lab</td><td class="patent-data-table-td ">A method of transforming fungi with a vector</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1986006097A1?cl=en">WO1986006097A1</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Apr 14, 1986</td><td class="patent-data-table-td patent-date-value">Sep 23, 1986</td><td class="patent-data-table-td ">Allelix Inc</td><td class="patent-data-table-td ">Promoters of filamentous fungi and use thereof</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="npl-citations"></a><div class="patent-section-header"><span class="patent-section-title">Non-Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th colspan="3"class="patent-data-table-th">Reference</th></tr></thead><tr><td class="patent-data-table-td ">1</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Akins, R. A., et al., "<a href='http://scholar.google.com/scholar?q="General+Method+for+Cloning+Neurospora+crassa+Nuclear+GEnes+by+Complementation+of+Mutants"'>General Method for Cloning Neurospora crassa Nuclear GEnes by Complementation of Mutants</a>", Molecular and Cellular Biology, 5(9):2272-2278 (1985).</td></tr><tr><td class="patent-data-table-td ">2</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Akins, R. A., et al., General Method for Cloning Neurospora crassa Nuclear GEnes by Complementation of Mutants , Molecular and Cellular Biology, 5(9):2272 2278 (1985).</td></tr><tr><td class="patent-data-table-td ">3</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Alton, N. K., et al., "<a href='http://scholar.google.com/scholar?q="5%27-Untranslated+sequences+of+two+structural+genes+in+the+qa+gene+cluster+of+Neurospora+crassa"'>5'-Untranslated sequences of two structural genes in the qa gene cluster of Neurospora crassa</a>", Proc. Natl. Acad. Sci. USA, 79:1655-1959 (1982).</td></tr><tr><td class="patent-data-table-td ">4</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Alton, N. K., et al., 5 Untranslated sequences of two structural genes in the qa gene cluster of Neurospora crassa , Proc. Natl. Acad. Sci. USA, 79:1655 1959 (1982).</td></tr><tr><td class="patent-data-table-td ">5</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Arends, H., et al., "<a href='http://scholar.google.com/scholar?q="Nucleotide+sequence+of+the+cloned+mRNA+and+gene+of+the+ADP%2FATP+carrier+from+Neurospora+crassa"'>Nucleotide sequence of the cloned mRNA and gene of the ADP/ATP carrier from Neurospora crassa</a>", The EMBO Journal, 3(2):377-382 (1984).</td></tr><tr><td class="patent-data-table-td ">6</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Arends, H., et al., Nucleotide sequence of the cloned mRNA and gene of the ADP/ATP carrier from Neurospora crassa , The EMBO Journal, 3(2):377 382 (1984).</td></tr><tr><td class="patent-data-table-td ">7</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Ballance, D. J., et al., "<a href='http://scholar.google.com/scholar?q="Development+of+a+high-frequency+transforming+vector+for+Aspergillus+nidulans"'>Development of a high-frequency transforming vector for Aspergillus nidulans</a>", Gene, 36:321-331 (1985).</td></tr><tr><td class="patent-data-table-td ">8</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Ballance, D. J., et al., "<a href='http://scholar.google.com/scholar?q="Gene+cloning+in+Aspergillus+nidulans%3Aisolation+of+the+isocitrate+lyase+gene+%28acuD%29"'>Gene cloning in Aspergillus nidulans:isolation of the isocitrate lyase gene (acuD)</a>", Mol. Gen. Genet., 202:271-275 (1986).</td></tr><tr><td class="patent-data-table-td ">9</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Ballance, D. J., et al., "<a href='http://scholar.google.com/scholar?q="Transformation+of+Aspergillus+nidulans+By+The+Orotidine-5%27-Phosphate+Decarboxylase+Gene"'>Transformation of Aspergillus nidulans By The Orotidine-5'-Phosphate Decarboxylase Gene</a>", Biochemical and Biophysical Research Communications, 112(1):284-289 (1983).</td></tr><tr><td class="patent-data-table-td ">10</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Ballance, D. J., et al., Development of a high frequency transforming vector for Aspergillus nidulans , Gene, 36:321 331 (1985).</td></tr><tr><td class="patent-data-table-td ">11</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Ballance, D. J., et al., Gene cloning in Aspergillus nidulans: isolation of the isocitrate lyase gene (acuD) , Mol. Gen. Genet., 202:271 275 (1986).</td></tr><tr><td class="patent-data-table-td ">12</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Ballance, D. J., et al., Transformation of Aspergillus nidulans By The Orotidine 5 Phosphate Decarboxylase Gene , Biochemical and Biophysical Research Communications, 112(1):284 289 (1983).</td></tr><tr><td class="patent-data-table-td ">13</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Ballance, D.J., et al., "<a href='http://scholar.google.com/scholar?q="Development+of+a+high-frequency+transforming+vector+for+Aspergillus+nidulans"'>Development of a high-frequency transforming vector for Aspergillus nidulans</a>", Gene 36:321-331 (1985).</td></tr><tr><td class="patent-data-table-td ">14</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Ballance, D.J., et al., Development of a high frequency transforming vector for Aspergillus nidulans , Gene 36:321 331 (1985).</td></tr><tr><td class="patent-data-table-td ">15</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Barnes, D. E., et al., "<a href='http://scholar.google.com/scholar?q="Behaviour+of+recombinant+plasmids+in+Aspergillus+nidulans%3Astructure+and+stability"'>Behaviour of recombinant plasmids in Aspergillus nidulans:structure and stability</a>", Current Genetics, 10:767-775 (1986).</td></tr><tr><td class="patent-data-table-td ">16</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Barnes, D. E., et al., Behaviour of recombinant plasmids in Aspergillus nidulans: structure and stability , Current Genetics, 10:767 775 (1986).</td></tr><tr><td class="patent-data-table-td ">17</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Bech, A. M., et al., Partial primary structure of Mucor miehei protease , Neth. Milk Dairy J. 35:275 280 (1981).</td></tr><tr><td class="patent-data-table-td ">18</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Bech, A.-M., et al., "<a href='http://scholar.google.com/scholar?q="Partial+primary+structure+of+Mucor+miehei+protease"'>Partial primary structure of Mucor miehei protease</a>", Neth. Milk Dairy J. 35:275-280 (1981).</td></tr><tr><td class="patent-data-table-td ">19</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Beppu, T., "<a href='http://scholar.google.com/scholar?q="The+cloning+and+expression+of+chymosin+%28rennin%29+genes+in+microorganisms"'>The cloning and expression of chymosin (rennin) genes in microorganisms</a>", Trends in Biotech., 1:85-89 (1983).</td></tr><tr><td class="patent-data-table-td ">20</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Beppu, T., The cloning and expression of chymosin (rennin) genes in microorganisms , Trends in Biotech., 1:85 89 (1983).</td></tr><tr><td class="patent-data-table-td ">21</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Berse, B., et al., "<a href='http://scholar.google.com/scholar?q="Cloning+and+characterization+of+the+ornithine+carbamoyltransferase+gene+from+Aspergillus+nidulans"'>Cloning and characterization of the ornithine carbamoyltransferase gene from Aspergillus nidulans</a>", Gene, 25:109-117 (1983).</td></tr><tr><td class="patent-data-table-td ">22</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Berse, B., et al., Cloning and characterization of the ornithine carbamoyltransferase gene from Aspergillus nidulans , Gene, 25:109 117 (1983).</td></tr><tr><td class="patent-data-table-td ">23</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Bissett Can. J. Bot. (1984) 62:924 931.</td></tr><tr><td class="patent-data-table-td ">24</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Bissett Can. J. Bot. (1984) 62:924-931.</td></tr><tr><td class="patent-data-table-td ">25</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Boel, E., et al., "<a href='http://scholar.google.com/scholar?q="Two+different+types+of+intervening+sequences+in+the+glucoamylase+gene+from+Aspergillus+niger"'>Two different types of intervening sequences in the glucoamylase gene from Aspergillus niger</a>", EMBO Journal, 3:1581-1585 (1984).</td></tr><tr><td class="patent-data-table-td ">26</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Boel, E., et al., Two different types of intervening sequences in the glucoamylase gene from Aspergillus niger , EMBO Journal, 3:1581 1585 (1984).</td></tr><tr><td class="patent-data-table-td ">27</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Briggs, M. S., et al., "<a href='http://scholar.google.com/scholar?q="Molecular+Mechanisms+of+Protein+Secretion%3A+The+Role+of+the+Signal+Sequence"'>Molecular Mechanisms of Protein Secretion: The Role of the Signal Sequence</a>", Advances in Protein Chemistry, 38:109-180 (1986).</td></tr><tr><td class="patent-data-table-td ">28</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Briggs, M. S., et al., Molecular Mechanisms of Protein Secretion: The Role of the Signal Sequence , Advances in Protein Chemistry, 38:109 180 (1986).</td></tr><tr><td class="patent-data-table-td ">29</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Bull, J. H., et al., Heavily methylated amplified DNA in transformants of Neurospora crassa, Nature, 310:701 704 (1984).</td></tr><tr><td class="patent-data-table-td ">30</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Bull, J. H., et al., Heavily methylated amplified DNA in transformants of Neurospora crassa, Nature, 310:701-704 (1984).</td></tr><tr><td class="patent-data-table-td ">31</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Buxton, F. P., et al., "<a href='http://scholar.google.com/scholar?q="Cloning+of+the+Structural+Gene+for+Orotidine+5%27-Phosphate+Carboxylase+of+Neurospora+crassa+by+Expression+in+Escherichia+coli"'>Cloning of the Structural Gene for Orotidine 5'-Phosphate Carboxylase of Neurospora crassa by Expression in Escherichia coli</a>", Mol. Gen. Genet, 190:403-405 (1983).</td></tr><tr><td class="patent-data-table-td ">32</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Buxton, F. P., et al., "<a href='http://scholar.google.com/scholar?q="The+transformation+of+mycelial+spheroplasts+of+Neurospora+crassa+and+the+attempted+isolation+of+an+autonomous+replicator"'>The transformation of mycelial spheroplasts of Neurospora crassa and the attempted isolation of an autonomous replicator</a>", Mol. Gen. Genet, 196:339-344 (1984).</td></tr><tr><td class="patent-data-table-td ">33</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Buxton, F. P., et al., "<a href='http://scholar.google.com/scholar?q="Transformation+of+Aspergillus+niger+using+the+argB+gene+of+Aspergillus+nidulans"'>Transformation of Aspergillus niger using the argB gene of Aspergillus nidulans</a>", Gene, 37:207-214 (1985).</td></tr><tr><td class="patent-data-table-td ">34</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Buxton, F. P., et al., Cloning of the Structural Gene for Orotidfine 5 Phosphate Carboxylase of Neurospora crassa by Expression in Escherichia coli .</td></tr><tr><td class="patent-data-table-td ">35</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Buxton, F. P., et al., Cloning of the Structural Gene for Orotidfine 5'-Phosphate Carboxylase of Neurospora crassa by Expression in Escherichia coli.</td></tr><tr><td class="patent-data-table-td ">36</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Buxton, F. P., et al., Cloning of the Structural Gene for Orotidine 5 Phosphate Carboxylase of Neurospora crassa by Expression in Escherichia coli , Mol. Gen. Genet, 190:403 405 (1983).</td></tr><tr><td class="patent-data-table-td ">37</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Buxton, F. P., et al., The transformation of mycelial spheroplasts of Neurospora crassa and the attempted isolation of an autonomous replicator , Mol. Gen. Genet, 196:339 344 (1984).</td></tr><tr><td class="patent-data-table-td ">38</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Buxton, F. P., et al., Transformation of Aspergillus niger using the argB gene of Aspergillus nidulans , Gene, 37:207 214 (1985).</td></tr><tr><td class="patent-data-table-td ">39</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Caddick, M. X., et al., "<a href='http://scholar.google.com/scholar?q="Cloning+of+the+regulatory+gene+areA+mediating+nitrogen+metabolite+repression+in+Aspergillus+nidulans"'>Cloning of the regulatory gene areA mediating nitrogen metabolite repression in Aspergillus nidulans</a>", The EMBO Journal, 5(5):1087-1090 (1986).</td></tr><tr><td class="patent-data-table-td ">40</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Caddick, M. X., et al., Cloning of the regulatory gene areA mediating nitrogen metabolite repression in Aspergillus nidulans , The EMBO Journal, 5(5):1087 1090 (1986).</td></tr><tr><td class="patent-data-table-td ">41</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Case, M.E., et al., "<a href='http://scholar.google.com/scholar?q="Efficient+transformation+of+Neurospora+crassa+by+utilizing+hybrid+plasmid+DNA"'>Efficient transformation of Neurospora crassa by utilizing hybrid plasmid DNA</a>", Proc. Natl. Acad. Sci. USA 76: 5259-5263 (1979).</td></tr><tr><td class="patent-data-table-td ">42</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Case, M.E., et al., Efficient transformation of Neurospora crassa by utilizing hybrid plasmid DNA , Proc. Natl. Acad. Sci. USA 76: 5259 5263 (1979).</td></tr><tr><td class="patent-data-table-td ">43</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Clements, J. M., et al., "<a href='http://scholar.google.com/scholar?q="Transcription+and+processing+signals+in+the+3-phosphoglycerate+kinase+%28PGK%29+gene+from+Aspergillus+nidulans"'>Transcription and processing signals in the 3-phosphoglycerate kinase (PGK) gene from Aspergillus nidulans</a>", Gene, 44:97-105 (1986).</td></tr><tr><td class="patent-data-table-td ">44</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Clements, J. M., et al., Transcription and processing signals in the 3 phosphoglycerate kinase (PGK) gene from Aspergillus nidulans , Gene, 44:97 105 (1986).</td></tr><tr><td class="patent-data-table-td ">45</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Cullen, D., et al., "<a href='http://scholar.google.com/scholar?q="Expression+of+bovine+chymosin+by+Aspergillus+nidulans"'>Expression of bovine chymosin by Aspergillus nidulans</a>"Heredity, vol. 57, p. 131, Abstract No. 20 (1986).</td></tr><tr><td class="patent-data-table-td ">46</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Cullen, D., et al., Expression of bovine chymosin by Aspergillus nidulans Heredity, vol. 57, p. 131, Abstract No. 20 (1986).</td></tr><tr><td class="patent-data-table-td ">47</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Dhawale, S. S., et al., "<a href='http://scholar.google.com/scholar?q="Transformation+of+Neurospora+crassa+with+circular+and+linear+DNA+and+analysis+of+the+fate+of+the+transforming+DNA"'>Transformation of Neurospora crassa with circular and linear DNA and analysis of the fate of the transforming DNA</a>", Current Genetics, 10:205-212 (1985).</td></tr><tr><td class="patent-data-table-td ">48</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Dhawale, S. S., et al., Transformation of Neurospora crassa with circular and linear DNA and analysis of the fate of the transforming DNA , Current Genetics, 10:205 212 (1985).</td></tr><tr><td class="patent-data-table-td ">49</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Doy, C. H., et al., "<a href='http://scholar.google.com/scholar?q="Genomic+Clones+of+Aspergillus+nidulans+Containing+alcA%2C+the+Structural+Gene+for+Alcohol+Dehydrogenase+and+alcR%2C+a+Regulatory+Gene+for+Ethanol+Metabolism"'>Genomic Clones of Aspergillus nidulans Containing alcA, the Structural Gene for Alcohol Dehydrogenase and alcR, a Regulatory Gene for Ethanol Metabolism</a>", DNA, 4(2):105-114 (1985).</td></tr><tr><td class="patent-data-table-td ">50</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Doy, C. H., et al., Genomic Clones of Aspergillus nidulans Containing alcA, the Structural Gene for Alcohol Dehydrogenase and alcR, a Regulatory Gene for Ethanol Metabolism , DNA, 4(2):105 114 (1985).</td></tr><tr><td class="patent-data-table-td ">51</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Durrens, P., et al., "<a href='http://scholar.google.com/scholar?q="Heterologous+insertion+of+transforming+DNA+and+generation+of+new+deletions+associated+with+transformation+in+Aspergillus+nidulans"'>Heterologous insertion of transforming DNA and generation of new deletions associated with transformation in Aspergillus nidulans</a>", Mol. Gen. Genet., 203:544-549 (1986).</td></tr><tr><td class="patent-data-table-td ">52</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Durrens, P., et al., Heterologous insertion of transforming DNA and generation of new deletions associated with transformation in Aspergillus nidulans , Mol. Gen. Genet., 203:544 549 (1986).</td></tr><tr><td class="patent-data-table-td ">53</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Emtage, J. S., et al., "<a href='http://scholar.google.com/scholar?q="Synthesis+of+calf+prochymosin+%28prorennin%29+in+Escherichia+coli"'>Synthesis of calf prochymosin (prorennin) in Escherichia coli</a>", Proc. Natl. Acad. Sci. USA, 80:3671-3675 (1983).</td></tr><tr><td class="patent-data-table-td ">54</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Emtage, J. S., et al., Synthesis of calf prochymosin (prorennin) in Escherichia coli , Proc. Natl. Acad. Sci. USA, 80:3671 3675 (1983).</td></tr><tr><td class="patent-data-table-td ">55</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Freer, S.N., "<a href='http://scholar.google.com/scholar?q="Fungal+Nucleic+Acids"'>Fungal Nucleic Acids</a>", In: Agricultural Research Service, USDA, Peoria, Illinois, pp. 175-193, (1983).</td></tr><tr><td class="patent-data-table-td ">56</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Freer, S.N., Fungal Nucleic Acids , In: Agricultural Research Service, USDA, Peoria, Illinois, pp. 175 193, (1983).</td></tr><tr><td class="patent-data-table-td ">57</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Golff, C. g., et al., Expression of calf prochymosin in Saccharomyces cerevisiae, Gene, 27:35 46 (1984).</td></tr><tr><td class="patent-data-table-td ">58</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Golff, C. g., et al., Expression of calf prochymosin in Saccharomyces cerevisiae, Gene, 27:35-46 (1984).</td></tr><tr><td class="patent-data-table-td ">59</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Grant, D. M., et al., "<a href='http://scholar.google.com/scholar?q="Transformation+of+Neurospora+crassa+with+Recombinant+Plasmids+Containing+the+Cloned+Glutamate+Dehydrogenase+%28am%29+Gene%3AEvidence+for+Autonomous+Replication+of+the+Transforming+Plasmid"'>Transformation of Neurospora crassa with Recombinant Plasmids Containing the Cloned Glutamate Dehydrogenase (am) Gene:Evidence for Autonomous Replication of the Transforming Plasmid</a>", Molecular and Cellular Biology, 4(10):2041-2051 (1984).</td></tr><tr><td class="patent-data-table-td ">60</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Grant, D. M., et al., Transformation of Neurospora crassa with Recombinant Plasmids Containing the Cloned Glutamate Dehydrogenase (am) Gene:Evidence for Autonomous Replication of the Transforming Plasmid , Molecular and Cellular Biology, 4(10):2041 2051 (1984).</td></tr><tr><td class="patent-data-table-td ">61</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Hayenga, K. et al., "<a href='http://scholar.google.com/scholar?q="Expression+and+Secretion+of+Bovine+Calf+Chymosin+by+Aspergillus+nidulans"'>Expression and Secretion of Bovine Calf Chymosin by Aspergillus nidulans</a>", Journal of Cellular Biochemistry Supp. vol. 0, No. 10, Part A, Abstract E109 (1986).</td></tr><tr><td class="patent-data-table-td ">62</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Hayenga, K. et al., Expression and Secretion of Bovine Calf Chymosin by Aspergillus nidulans , Journal of Cellular Biochemistry Supp. vol. 0, No. 10, Part A, Abstract E109 (1986).</td></tr><tr><td class="patent-data-table-td ">63</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Herrera Estrella et al. Mol. Microbiol. (1990) 4:839 43.</td></tr><tr><td class="patent-data-table-td ">64</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Herrera-Estrella et al. Mol. Microbiol. (1990) 4:839-43.</td></tr><tr><td class="patent-data-table-td ">65</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Heyneker, H. L., et al., "<a href='http://scholar.google.com/scholar?q="Cloning+strategies+in+Aspergillus+for+enzyme+production%3A+prochymosin+as+a+model+system"'>Cloning strategies in Aspergillus for enzyme production: prochymosin as a model system</a>", Proc. of Bio. Expo., 86:145-149 (1986).</td></tr><tr><td class="patent-data-table-td ">66</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Heyneker, H. L., et al., Cloning strategies in Aspergillus for enzyme production: prochymosin as a model system , Proc. of Bio. Expo., 86:145 149 (1986).</td></tr><tr><td class="patent-data-table-td ">67</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Hidaka, M., et al., "<a href='http://scholar.google.com/scholar?q="Cloning+and+structural+analysis+of+the+calf+prochymosin+gene"'>Cloning and structural analysis of the calf prochymosin gene</a>", Gene, 43:197-203 (1986).</td></tr><tr><td class="patent-data-table-td ">68</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Hidaka, M., et al., Cloning and structural analysis of the calf prochymosin gene , Gene, 43:197 203 (1986).</td></tr><tr><td class="patent-data-table-td ">69</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Huiet, L., et al., "<a href='http://scholar.google.com/scholar?q="The+qa+repressor+gene+of+Neurospora+crassa%3AWild-type+and+mutant+nucleotide+sequences"'>The qa repressor gene of Neurospora crassa:Wild-type and mutant nucleotide sequences</a>", Proc. Natl. Acad. Sci. USA, 83:3381-3385 (1986).</td></tr><tr><td class="patent-data-table-td ">70</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Huiet, L., et al., The qa repressor gene of Neurospora crassa: Wild type and mutant nucleotide sequences , Proc. Natl. Acad. Sci. USA, 83:3381 3385 (1986).</td></tr><tr><td class="patent-data-table-td ">71</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Hynes, M. J., "<a href='http://scholar.google.com/scholar?q="Topical+Review--Transformation+of+Filamentous+Fungi"'>Topical Review--Transformation of Filamentous Fungi</a>", Experimental Mycology, 10:1-8 (1986).</td></tr><tr><td class="patent-data-table-td ">72</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Hynes, M. J., et al., "<a href='http://scholar.google.com/scholar?q="Isolation+of+Genomic+Clones+Containing+the+amdS+gene+of+Aspergillus+nidulans+and+Their+Use+in+the+Analysis+of+Structural+and+Regulatory+Mutations"'>Isolation of Genomic Clones Containing the amdS gene of Aspergillus nidulans and Their Use in the Analysis of Structural and Regulatory Mutations</a>", Molecular and Cellular Biology, 3(8):1430-1439 (1983).</td></tr><tr><td class="patent-data-table-td ">73</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Hynes, M. J., et al., Isolation of Genomic Clones Containing the amdS gene of Aspergillus nidulans and Their Use in the Analysis of Structural and Regulatory Mutations , Molecular and Cellular Biology, 3(8):1430 1439 (1983).</td></tr><tr><td class="patent-data-table-td ">74</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Hynes, M. J., Topical Review Transformation of Filamentous Fungi , Experimental Mycology, 10:1 8 (1986).</td></tr><tr><td class="patent-data-table-td ">75</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Innis, M. A., et al. Expression, Glycosylation, and Secretion of an Aspergillus Glucoamylase by Saccharomyces cerevisiae, Science, 228:21 26 (1985).</td></tr><tr><td class="patent-data-table-td ">76</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Innis, M. A., et al. Expression, Glycosylation, and Secretion of an Aspergillus Glucoamylase by Saccharomyces cerevisiae, Science, 228:21-26 (1985).</td></tr><tr><td class="patent-data-table-td ">77</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Johnstone, I. L., "<a href='http://scholar.google.com/scholar?q="Cloning+an+Aspergillus+nidulans+developmental+gene+by+transformation"'>Cloning an Aspergillus nidulans developmental gene by transformation</a>", The EMBO Journal, 4(5):1307-1311 (1985).</td></tr><tr><td class="patent-data-table-td ">78</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Johnstone, I. L., "<a href='http://scholar.google.com/scholar?q="Transformation+of+Aspergillus+nidulans"'>Transformation of Aspergillus nidulans</a>", Microbiological Sciences, 2(10):307-311 (1985).</td></tr><tr><td class="patent-data-table-td ">79</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Johnstone, I. L., Cloning an Aspergillus nidulans developmental gene by transformation , The EMBO Journal, 4(5):1307 1311 (1985).</td></tr><tr><td class="patent-data-table-td ">80</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Johnstone, I. L., Transformation of Aspergillus nidulans , Microbiological Sciences, 2(10):307 311 (1985).</td></tr><tr><td class="patent-data-table-td ">81</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">K u ck, U., et al., II. Recombination: a) DNA Transformation in Cyanobacteria, Algae and Fungi:Trends and Perspectives , Progress in Botany, 46:201 204 (1984).</td></tr><tr><td class="patent-data-table-td ">82</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Kawaguchi, F., et al., Renaturation and activation of calf prochymosin produced in an insoluble form in Escherichia coli, Journal of Biotechnology, 1:307 315 (1984).</td></tr><tr><td class="patent-data-table-td ">83</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Kawaguchi, F., et al., Renaturation and activation of calf prochymosin produced in an insoluble form in Escherichia coli, Journal of Biotechnology, 1:307-315 (1984).</td></tr><tr><td class="patent-data-table-td ">84</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Keesey, J. K., Jr., et al., "<a href='http://scholar.google.com/scholar?q="Cloning+of+the+trp-1+Gene+from+Neurospora+crassa+by+Complementation+of+a+trpC+Mutation+in+Escherichia+coli"'>Cloning of the trp-1 Gene from Neurospora crassa by Complementation of a trpC Mutation in Escherichia coli</a>", Journal of Bacteriology, 152(2):954-958 (1982).</td></tr><tr><td class="patent-data-table-td ">85</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Keesey, J. K., Jr., et al., Cloning of the trp 1 Gene from Neurospora crassa by Complementation of a trpC Mutation in Escherichia coli , Journal of Bacteriology, 152(2):954 958 (1982).</td></tr><tr><td class="patent-data-table-td ">86</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Kelly, J. M., et al., "<a href='http://scholar.google.com/scholar?q="Transformation+of+Aspergillus+niger+by+the+amdS+gene+of+Aspergillus+nidulans"'>Transformation of Aspergillus niger by the amdS gene of Aspergillus nidulans</a>", The EMBO Journal, 4(2):475-479 (1985).</td></tr><tr><td class="patent-data-table-td ">87</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Kelly, J. M., et al., Transformation of Aspergillus niger by the amdS gene of Aspergillus nidulans , The EMBO Journal, 4(2):475 479 (1985).</td></tr><tr><td class="patent-data-table-td ">88</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Kinnaird, J. H., et al., "<a href='http://scholar.google.com/scholar?q="The+complete+nucleotide+sequence+of+the+Neurospora+crassa+am+%28NADP-specific+glutamate+dehydrogenase%29+gene"'>The complete nucleotide sequence of the Neurospora crassa am (NADP-specific glutamate dehydrogenase) gene</a>", Gene, 26:253-260 (1983).</td></tr><tr><td class="patent-data-table-td ">89</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Kinnaird, J. H., et al., The complete nucleotide sequence of the Neurospora crassa am (NADP specific glutamate dehydrogenase) gene , Gene, 26:253 260 (1983).</td></tr><tr><td class="patent-data-table-td ">90</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Kinsey, J. A., et al., "<a href='http://scholar.google.com/scholar?q="Transformation+of+Neurospora+crassa+with+the+Cloned+am+%28Glutamate+Dehydrogenase%29+Gene"'>Transformation of Neurospora crassa with the Cloned am (Glutamate Dehydrogenase) Gene</a>", Molecular and Cellular Biology, 4(1):117-122 (1984).</td></tr><tr><td class="patent-data-table-td ">91</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Kinsey, J. A., et al., Transformation of Neurospora crassa with the Cloned am (Glutamate Dehydrogenase) Gene , Molecular and Cellular Biology, 4(1):117 122 (1984).</td></tr><tr><td class="patent-data-table-td ">92</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Kos, A., et al., "<a href='http://scholar.google.com/scholar?q="Isolation+and+characterization+of+the+Aspergillus+niger+trpC+gene"'>Isolation and characterization of the Aspergillus niger trpC gene</a>", Gene, 39:231-238 (1985).</td></tr><tr><td class="patent-data-table-td ">93</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Kos, A., et al., Isolation and characterization of the Aspergillus niger trpC gene , Gene, 39:231 238 (1985).</td></tr><tr><td class="patent-data-table-td ">94</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Kuck, U., et al., "<a href='http://scholar.google.com/scholar?q="II.+Recombination%3A+a%29+DNA+Transformation+in+Cyanobacteria%2C+Algae+and+Fungi%3ATrends+and+Perspectives"'>II. Recombination: a) DNA Transformation in Cyanobacteria, Algae and Fungi:Trends and Perspectives</a>", Progress in Botany, 46:201-204 (1984).</td></tr><tr><td class="patent-data-table-td ">95</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Kuiper, M. T. R., et al., "<a href='http://scholar.google.com/scholar?q="A+recombinant+plasmid+carrying+the+mitochondrial+plasmid+sequence+of+Neurospora+intermedia+LaBelle+yields+new+plasmid+derivatives+in+Neurospora+crassa+transformants"'>A recombinant plasmid carrying the mitochondrial plasmid sequence of Neurospora intermedia LaBelle yields new plasmid derivatives in Neurospora crassa transformants</a>", Current Genetics, 9:471-477 (1985).</td></tr><tr><td class="patent-data-table-td ">96</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Kuiper, M. T. R., et al., A recombinant plasmid carrying the mitochondrial plasmid sequence of Neurospora intermedia LaBelle yields new plasmid derivatives in Neurospora crassa transformants , Current Genetics, 9:471 477 (1985).</td></tr><tr><td class="patent-data-table-td ">97</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Legerton, T. LK., et al., Cloning and characterization of the multifunctional his 3 gene of Neurospora crassa, Gene, 39:129 140 (1985).</td></tr><tr><td class="patent-data-table-td ">98</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Legerton, T. LK., et al., Cloning and characterization of the multifunctional his-3 gene of Neurospora crassa, Gene, 39:129-140 (1985).</td></tr><tr><td class="patent-data-table-td ">99</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">M u nger, K., et al., isolation and structural organization of the Neurospoa crassa copper metallothionein gene , The EMBO Journal, 4(10):2665 2668 (1985).</td></tr><tr><td class="patent-data-table-td ">100</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Martson, F. A. O., et al., "<a href='http://scholar.google.com/scholar?q="Purification+of+Calf+Prochymosin+%28Prorennin%29+Synthesized+in+Escherichia+coli"'>Purification of Calf Prochymosin (Prorennin) Synthesized in Escherichia coli</a>", Biotechnology 2:800-804 (1984).</td></tr><tr><td class="patent-data-table-td ">101</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Martson, F. A. O., et al., Purification of Calf Prochymosin (Prorennin) Synthesized in Escherichia coli , Biotechnology 2:800 804 (1984).</td></tr><tr><td class="patent-data-table-td ">102</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">McCaman, M. T., et al., Enzymatic properties and processing of bovine prochymosin synthesized in Escherichia coli, Journal of Biotechnology, 2:177 190 (1985).</td></tr><tr><td class="patent-data-table-td ">103</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">McCaman, M. T., et al., Enzymatic properties and processing of bovine prochymosin synthesized in Escherichia coli, Journal of Biotechnology, 2:177-190 (1985).</td></tr><tr><td class="patent-data-table-td ">104</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">McGuire, J., "<a href='http://scholar.google.com/scholar?q="Cloning%2C+Expression%2C+Purification+and+Testing+of+Chymosin"'>Cloning, Expression, Purification and Testing of Chymosin</a>", from Proceedings of Bio Expo 86. The American Commercial and Industrial Conference and Exposition in Biotechnology, pp. 121-141 (1986).</td></tr><tr><td class="patent-data-table-td ">105</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">McGuire, J., Cloning, Expression, Purification and Testing of Chymosin , from Proceedings of Bio Expo 86. The American Commercial and Industrial Conference and Exposition in Biotechnology, pp. 121 141 (1986).</td></tr><tr><td class="patent-data-table-td ">106</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">McKnight G. L., et al., "<a href='http://scholar.google.com/scholar?q="Nucleotide+Sequece+of+the+Triosephosphate+Isomerawse+Gene+from+Aspergillus+nidulans%3AImplications+for+a+Differential+Loss+of+Introns"'>Nucleotide Sequece of the Triosephosphate Isomerawse Gene from Aspergillus nidulans:Implications for a Differential Loss of Introns</a>", Cell, 46:143-147 (1986).</td></tr><tr><td class="patent-data-table-td ">107</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">McKnight G. L., et al., Nucleotide Sequece of the Triosephosphate Isomerawse Gene from Aspergillus nidulans: Implications for a Differential Loss of Introns , Cell, 46:143 147 (1986).</td></tr><tr><td class="patent-data-table-td ">108</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">McKnight, G. L., "<a href='http://scholar.google.com/scholar?q="Lack+of+an+Aspergillus+Promoter+Function+and+Intron+Splicing+in+Saccaromyces"'>Lack of an Aspergillus Promoter Function and Intron Splicing in Saccaromyces</a>", J. Cell. Biochem. Suppl., 9c:137 (1985).</td></tr><tr><td class="patent-data-table-td ">109</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">McKnight, G. L., Lack of an Aspergillus Promoter Function and Intron Splicing in Saccaromyces , J. Cell. Biochem. Suppl., 9c:137 (1985).</td></tr><tr><td class="patent-data-table-td ">110</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Mellor, J., et al., Efficient synthesis of enzymatically active calf chymosin in Saccaromyces cerevisiae, Gene, 24:1 14 (1983).</td></tr><tr><td class="patent-data-table-td ">111</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Mellor, J., et al., Efficient synthesis of enzymatically active calf chymosin in Saccaromyces cerevisiae, Gene, 24:1-14 (1983).</td></tr><tr><td class="patent-data-table-td ">112</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Miller, B. L., et al., "<a href='http://scholar.google.com/scholar?q="Direct+and+Indirect+Gene+Replacements+in+Aspergillus+nidulans"'>Direct and Indirect Gene Replacements in Aspergillus nidulans</a>", Molecular and Cellular Biology, 5(7):1714-1721 (1985).</td></tr><tr><td class="patent-data-table-td ">113</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Miller, B. L., et al., Direct and Indirect Gene Replacements in Aspergillus nidulans , Molecular and Cellular Biology, 5(7):1714 1721 (1985).</td></tr><tr><td class="patent-data-table-td ">114</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Moir, D. T., et al., "<a href='http://scholar.google.com/scholar?q="Molecular+cloning+and+characterization+of+double-stranded+cDNA+coding+for+bovine+chymosin"'>Molecular cloning and characterization of double-stranded cDNA coding for bovine chymosin</a>", Gene, 19:127-138 (1982).</td></tr><tr><td class="patent-data-table-td ">115</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Moir, D. T., et al., Molecular cloning and characterization of double stranded cDNA coding for bovine chymosin , Gene, 19:127 138 (1982).</td></tr><tr><td class="patent-data-table-td ">116</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Moir, D. T., et al., Production of Calf Chymosin by the Yeast S. cerevisiae, Developments in Industrial Microbiology, 26:75 85 (1985).</td></tr><tr><td class="patent-data-table-td ">117</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Moir, D. T., et al., Production of Calf Chymosin by the Yeast S. cerevisiae, Developments in Industrial Microbiology, 26:75-85 (1985).</td></tr><tr><td class="patent-data-table-td ">118</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Moir, D., et al., "<a href='http://scholar.google.com/scholar?q="Molecular+cloning+and+characterization+of+double-dtranded+cDNA+coding+for+bovine+chymosin"'>Molecular cloning and characterization of double-dtranded cDNA coding for bovine chymosin</a>", Chemical Abstracts, vol. 98, No. 15, p. 173, Abstract No. 120575m (1983).</td></tr><tr><td class="patent-data-table-td ">119</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Moir, D., et al., Molecular cloning and characterization of double dtranded cDNA coding for bovine chymosin , Chemical Abstracts, vol. 98, No. 15, p. 173, Abstract No. 120575m (1983).</td></tr><tr><td class="patent-data-table-td ">120</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Mullaney, E. J., et al., "<a href='http://scholar.google.com/scholar?q="Primary+structure+of+the+trpC+gene+from+Aspergillus+nidulans"'>Primary structure of the trpC gene from Aspergillus nidulans</a>", Mol. Gen. Genet., 199:37-45 (1985).</td></tr><tr><td class="patent-data-table-td ">121</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Mullaney, E. J., et al., Primary structure of the trpC gene from Aspergillus nidulans , Mol. Gen. Genet., 199:37 45 (1985).</td></tr><tr><td class="patent-data-table-td ">122</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Munger, K., et al., "<a href='http://scholar.google.com/scholar?q="isolation+and+structural+organization+of+the+Neurospoa+crassa+copper+metallothionein+gene"'>isolation and structural organization of the Neurospoa crassa copper metallothionein gene</a>", The EMBO Journal, 4(10):2665-2668 (1985).</td></tr><tr><td class="patent-data-table-td ">123</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Newbury, S. F., et al., "<a href='http://scholar.google.com/scholar?q="Sequence+analysis+of+the+pyr-4+%28orotidine+5%27-P+decarboxylase%29+gene+of+Neurospora+crassa"'>Sequence analysis of the pyr-4 (orotidine 5'-P decarboxylase) gene of Neurospora crassa</a>", Gene, 43:51-58 (1986).</td></tr><tr><td class="patent-data-table-td ">124</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Newbury, S. F., et al., Sequence analysis of the pyr 4 (orotidine 5 P decarboxylase) gene of Neurospora crassa , Gene, 43:51 58 (1986).</td></tr><tr><td class="patent-data-table-td ">125</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Nishimori, K., et al., "<a href='http://scholar.google.com/scholar?q="Cloning+in+Escherichia+coli+of+the+Structural+Gene+of+Prorennin%2C+the+Precursor+of+Calf+Milk-Clotting+Enzyme+Rennin"'>Cloning in Escherichia coli of the Structural Gene of Prorennin, the Precursor of Calf Milk-Clotting Enzyme Rennin</a>", J. Biochem. 90:901-904 (1981).</td></tr><tr><td class="patent-data-table-td ">126</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Nishimori, K., et al., "<a href='http://scholar.google.com/scholar?q="Expression+of+cloned+calf+prochymosin+cDNA+under+control+of+the+tryptophan+promoter"'>Expression of cloned calf prochymosin cDNA under control of the tryptophan promoter</a>", Gene, 29:41-419 (1984).</td></tr><tr><td class="patent-data-table-td ">127</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Nishimori, K., et al., Cloning in Escherichia coli of the Structural Gene of Prorennin, the Precursor of Calf Milk Clotting Enzyme Rennin , J. Biochem. 90:901 904 (1981).</td></tr><tr><td class="patent-data-table-td ">128</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Nishimori, K., et al., Expression of cloned calf prochymosin cDNA under control of the tryptophan promoter , Gene, 29:41 419 (1984).</td></tr><tr><td class="patent-data-table-td ">129</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Nishimori, K., et al., Expression of cloned calf prochymosin gene sequence in Escherichia coli, Gene, 19:337 344 (1982).</td></tr><tr><td class="patent-data-table-td ">130</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Nishimori, K., et al., Expression of cloned calf prochymosin gene sequence in Escherichia coli, Gene, 19:337-344 (1982).</td></tr><tr><td class="patent-data-table-td ">131</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Nishimori, K., et al., Nucleotide Sequence of Calf Prorennin cDNA Cloned in Escherichia coli, J. Biochem., 91:1085 1088 (1982).</td></tr><tr><td class="patent-data-table-td ">132</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Nishimori, K., et al., Nucleotide Sequence of Calf Prorennin cDNA Cloned in Escherichia coli, J. Biochem., 91:1085-1088 (1982).</td></tr><tr><td class="patent-data-table-td ">133</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Nunberg, J. H., et al., Molecular Cloning and Characterization of the Glucoamylase Gene of Aspergillus awamori, Molecular and Cellular Biology, 4(11):2306 2315 (1984).</td></tr><tr><td class="patent-data-table-td ">134</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Nunberg, J. H., et al., Molecular Cloning and Characterization of the Glucoamylase Gene of Aspergillus awamori, Molecular and Cellular Biology, 4(11):2306-2315 (1984).</td></tr><tr><td class="patent-data-table-td ">135</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Orbach, M. J., et al., "<a href='http://scholar.google.com/scholar?q="Cloning+and+Characterization+of+the+Gene+for+%CE%B2-Tubulin+from+a+Benomyl-Resistant+Mutant+of+Neurospora+crassa+and+Its+Use+as+a+Dominant+Selectable+Marker"'>Cloning and Characterization of the Gene for β-Tubulin from a Benomyl-Resistant Mutant of Neurospora crassa and Its Use as a Dominant Selectable Marker</a>", Molecular and Cellular Biology, 6:2452-2461 (1986).</td></tr><tr><td class="patent-data-table-td ">136</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Orbach, M. J., et al., Cloning and Characterization of the Gene for Tubulin from a Benomyl Resistant Mutant of Neurospora crassa and Its Use as a Dominant Selectable Marker , Molecular and Cellular Biology, 6:2452 2461 (1986).</td></tr><tr><td class="patent-data-table-td ">137</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Paietta, J., et al., "<a href='http://scholar.google.com/scholar?q="Plasmid+recovery+from+transformants+and+the+isolation+of+chromosomal+DNA+segments+improving+plasmid+replication+in+Neurospora+crassa"'>Plasmid recovery from transformants and the isolation of chromosomal DNA segments improving plasmid replication in Neurospora crassa</a>", Current Genetics, 9:383-388 (1985).</td></tr><tr><td class="patent-data-table-td ">138</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Paietta, J., et al., Plasmid recovery from transformants and the isolation of chromosomal DNA segments improving plasmid replication in Neurospora crassa , Current Genetics, 9:383 388 (1985).</td></tr><tr><td class="patent-data-table-td ">139</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Pentill a , M. E., et al., Cloning of Aspergillus niger genes in yeast. Expression of the gene coding Aspergillus glucosidase , Mol. Gen. Genet., 194:494 499 (1984).</td></tr><tr><td class="patent-data-table-td ">140</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Pentilla et al. Gene 61 (1987) 155 164.</td></tr><tr><td class="patent-data-table-td ">141</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Pentilla et al. Gene 61 (1987) 155-164.</td></tr><tr><td class="patent-data-table-td ">142</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Pentilla, M. E., et al., "<a href='http://scholar.google.com/scholar?q="Cloning+of+Aspergillus+niger+genes+in+yeast.+Expression+of+the+gene+coding+Aspergillus+%CE%B2-glucosidase"'>Cloning of Aspergillus niger genes in yeast. Expression of the gene coding Aspergillus β-glucosidase</a>", Mol. Gen. Genet., 194:494-499 (1984).</td></tr><tr><td class="patent-data-table-td ">143</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Rutledge, B. J., "<a href='http://scholar.google.com/scholar?q="Molecular+characterization+of+the+qa-4+gene+of+Neurospora+crassa"'>Molecular characterization of the qa-4 gene of Neurospora crassa</a>", Gene, 32:275-287 (1984).</td></tr><tr><td class="patent-data-table-td ">144</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Rutledge, B. J., Molecular characterization of the qa 4 gene of Neurospora crassa , Gene, 32:275 287 (1984).</td></tr><tr><td class="patent-data-table-td ">145</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Saunders, G., et al., "<a href='http://scholar.google.com/scholar?q="Fungal+cloning+vectors"'>Fungal cloning vectors</a>", Trends in Biotechnology, 4:93-98 (1986).</td></tr><tr><td class="patent-data-table-td ">146</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Saunders, G., et al., Fungal cloning vectors , Trends in Biotechnology, 4:93 98 (1986).</td></tr><tr><td class="patent-data-table-td ">147</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Sebald, W., et al., "<a href='http://scholar.google.com/scholar?q="H%2B+-ATPase+%28ATP+Synthase%29%3AStructure%2C+Function%2C+Boogenesis+The+F0+F1+Complex+of+Coupling+Membranes"'>H+ -ATPase (ATP Synthase):Structure, Function, Boogenesis The F0 F1 Complex of Coupling Membranes</a>", Abteilung fur Biotechnologische Forschung mbH., W. Germany, pp. 67-75 (1984).</td></tr><tr><td class="patent-data-table-td ">148</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Sebald, W., et al., H ATPase (ATP Synthase):Structure, Function, Boogenesis The F 0 F 1 Complex of Coupling Membranes , Abteilung f u r Biotechnologische Forschung mbH., W. Germany, pp. 67 75 (1984).</td></tr><tr><td class="patent-data-table-td ">149</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Shoemaker, S., et al., Molecular Cloning of Exo Cellobiohydrolase I Derived From Trichoderma reesei Strain L27, Biotechnology, 1:691 697 (1983).</td></tr><tr><td class="patent-data-table-td ">150</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Shoemaker, S., et al., Molecular Cloning of Exo-Cellobiohydrolase I Derived From Trichoderma reesei Strain L27, Biotechnology, 1:691-697 (1983).</td></tr><tr><td class="patent-data-table-td ">151</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Smith, R. A., et al., "<a href='http://scholar.google.com/scholar?q="Heterologous+Protein+Secretion+from+Yeast"'>Heterologous Protein Secretion from Yeast</a>", Science, 229:1219-1224 (1985).</td></tr><tr><td class="patent-data-table-td ">152</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Smith, R. A., et al., Heterologous Protein Secretion from Yeast , Science, 229:1219 1224 (1985).</td></tr><tr><td class="patent-data-table-td ">153</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Smith, R.A., et al., "<a href='http://scholar.google.com/scholar?q="Heterologous+Protein+Secretion+from+Yeast"'>Heterologous Protein Secretion from Yeast</a>", Science 2293:1219-1224 (1985).</td></tr><tr><td class="patent-data-table-td ">154</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Smith, R.A., et al., Heterologous Protein Secretion from Yeast , Science 2293:1219 1224 (1985).</td></tr><tr><td class="patent-data-table-td ">155</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Stahl, U., et al., "<a href='http://scholar.google.com/scholar?q="Replication+and+expressin+of+a+bacterial-mitochondrial+hybrid+plasmid+in+the+fungus+Podospora+anserina"'>Replication and expressin of a bacterial-mitochondrial hybrid plasmid in the fungus Podospora anserina</a>", Proc. Natl. Acad. Sci. USA 79:3641-3645 (1982).</td></tr><tr><td class="patent-data-table-td ">156</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Stahl, U., et al., Replication and expressin of a bacterial mitochondrial hybrid plasmid in the fungus Podospora anserina , Proc. Natl. Acad. Sci. USA 79:3641 3645 (1982).</td></tr><tr><td class="patent-data-table-td ">157</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Stohl, L. L., et al., "<a href='http://scholar.google.com/scholar?q="Characterization+of+deletion+derivatives+of+an+autonomously+replicating+Neurospora+plasmid"'>Characterization of deletion derivatives of an autonomously replicating Neurospora plasmid</a>", Nucleic Acids Research, 12:6169-6178 (1984).</td></tr><tr><td class="patent-data-table-td ">158</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Stohl, L. L., et al., Characterization of deletion derivatives of an autonomously replicating Neurospora plasmid , Nucleic Acids Research, 12:6169 6178 (1984).</td></tr><tr><td class="patent-data-table-td ">159</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Stohl, L. L., et al., Construction of a shuttle vector for the filamentous fungus Neurospora crassa, Proc. Natl. Acad. Sci. USA, 80:1058 1062 (1983).</td></tr><tr><td class="patent-data-table-td ">160</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Stohl, L. L., et al., Construction of a shuttle vector for the filamentous fungus Neurospora crassa, Proc. Natl. Acad. Sci. USA, 80:1058-1062 (1983).</td></tr><tr><td class="patent-data-table-td ">161</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Tilburn, J., et al., "<a href='http://scholar.google.com/scholar?q="Transformation+by+integration+in+Aspergillus+nidulans"'>Transformation by integration in Aspergillus nidulans</a>", Gene, 26:205-221 (1983).</td></tr><tr><td class="patent-data-table-td ">162</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Tilburn, J., et al., Transformation by integration in Aspergillus nidulans , Gene, 26:205 221 (1983).</td></tr><tr><td class="patent-data-table-td ">163</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Tudzynski, U., et al., "<a href='http://scholar.google.com/scholar?q="Transformation+to+Senescence+with+Plasmid+Like+DNA+in+the+Ascomyete+Podospora+anserina"'>Transformation to Senescence with Plasmid Like DNA in the Ascomyete Podospora anserina</a>", Current Genetics 2:181-184 (1980).</td></tr><tr><td class="patent-data-table-td ">164</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Tudzynski, U., et al., Transformation to Senescence with Plasmid Like DNA in the Ascomyete Podospora anserina , Current Genetics 2:181 184 (1980).</td></tr><tr><td class="patent-data-table-td ">165</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Turner, G., et al., "<a href='http://scholar.google.com/scholar?q="Cloning+and+Transformation+in+Aspergillus"'>Cloning and Transformation in Aspergillus</a>", Gene Manipulations in Fungi, (1985).</td></tr><tr><td class="patent-data-table-td ">166</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Turner, G., et al., Cloning and Transformation in Aspergillus , Gene Manipulations in Fungi, (1985).</td></tr><tr><td class="patent-data-table-td ">167</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Upshall, A., "<a href='http://scholar.google.com/scholar?q="Filamentous+Fungi+in+Biotechnology"'>Filamentous Fungi in Biotechnology</a>", Biotechniques, 4:158-166 (1986).</td></tr><tr><td class="patent-data-table-td ">168</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Upshall, A., "<a href='http://scholar.google.com/scholar?q="Genetic+and+molecular+characterization+of+argB%2B+transformants+of+Aspergillus+nidulans"'>Genetic and molecular characterization of argB+ transformants of Aspergillus nidulans</a>", Current Genetics, 10:593-599 (1986).</td></tr><tr><td class="patent-data-table-td ">169</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Upshall, A., Filamentous Fungi in Biotechnology , Biotechniques, 4:158 166 (1986).</td></tr><tr><td class="patent-data-table-td ">170</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Upshall, A., Genetic and molecular characterization of argB transformants of Aspergillus nidulans , Current Genetics, 10:593 599 (1986).</td></tr><tr><td class="patent-data-table-td ">171</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Viebrock, A., et al., "<a href='http://scholar.google.com/scholar?q="The+imported+preprotein+of+the+proteolipid+subunit+of+the+mitochondrial+ATP+synthase+from+Neurospora+crassa.+Molecular+cloning+and+sequencing+of+the+mRNA"'>The imported preprotein of the proteolipid subunit of the mitochondrial ATP synthase from Neurospora crassa. Molecular cloning and sequencing of the mRNA</a>", The EMBO Journal, 1(5):565-571 (1982).</td></tr><tr><td class="patent-data-table-td ">172</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Viebrock, A., et al., The imported preprotein of the proteolipid subunit of the mitochondrial ATP synthase from Neurospora crassa. Molecular cloning and sequencing of the mRNA , The EMBO Journal, 1(5):565 571 (1982).</td></tr><tr><td class="patent-data-table-td ">173</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Vollmer, S. J., et al., "<a href='http://scholar.google.com/scholar?q="Efficient+cloning+of+genes+of+Neurospora+crassa"'>Efficient cloning of genes of Neurospora crassa</a>", Proc. Natl. Acad. Sci. USA, 83:4869-4873 (1986).</td></tr><tr><td class="patent-data-table-td ">174</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Vollmer, S. J., et al., Efficient cloning of genes of Neurospora crassa , Proc. Natl. Acad. Sci. USA, 83:4869 4873 (1986).</td></tr><tr><td class="patent-data-table-td ">175</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Weiss, R. L., et al., Expression of Aspergillus Genes in Neurospora, Gene Manipulation in Fungi, (1985).</td></tr><tr><td class="patent-data-table-td ">176</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Wernars, K., et al., "<a href='http://scholar.google.com/scholar?q="Gene+amplification+in+Aspergillus+nidulans+by+transformation+with+vectors+containing+the+amdS+gene"'>Gene amplification in Aspergillus nidulans by transformation with vectors containing the amdS gene</a>", Current Genetics, 9:361-368 (1985).</td></tr><tr><td class="patent-data-table-td ">177</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Wernars, K., et al., Gene amplification in Aspergillus nidulans by transformation with vectors containing the amdS gene , Current Genetics, 9:361 368 (1985).</td></tr><tr><td class="patent-data-table-td ">178</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Woudt, L. P., et al., "<a href='http://scholar.google.com/scholar?q="+++"'>   </a>", Nucleic Acids Research, 11:5353-6346 (1983).</td></tr><tr><td class="patent-data-table-td ">179</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Woudt, L. P., et al., , Nucleic Acids Research, 11:5353 6346 (1983).</td></tr><tr><td class="patent-data-table-td ">180</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Yelton, M. M., et al., "<a href='http://scholar.google.com/scholar?q="A+cosmid+for+selecting+genes+by+complementation+in+Aspergillus+nidulans%3ASection+of+the+developmentally+regulared+yA+locus"'>A cosmid for selecting genes by complementation in Aspergillus nidulans:Section of the developmentally regulared yA locus</a>", Proc. Natl. Acad. Sci. USA, 82:834-838 (1985).</td></tr><tr><td class="patent-data-table-td ">181</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Yelton, M. M., et al., "<a href='http://scholar.google.com/scholar?q="Transformation+of+Aspergillus+nidulans+by+using+a+trpC+plasmid"'>Transformation of Aspergillus nidulans by using a trpC plasmid</a>", Proc. Natl. Acad. Sci. USA, 81:1470-1474 (1984).</td></tr><tr><td class="patent-data-table-td ">182</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Yelton, M. M., et al., A cosmid for selecting genes by complementation in Aspergillus nidulans: Section of the developmentally regulared yA locus , Proc. Natl. Acad. Sci. USA, 82:834 838 (1985).</td></tr><tr><td class="patent-data-table-td ">183</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Yelton, M. M., et al., Transformation of Aspergillus nidulans by using a trpC plasmid , Proc. Natl. Acad. Sci. USA, 81:1470 1474 (1984).</td></tr><tr><td class="patent-data-table-td ">184</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Yelton, M.M., et al., "<a href='http://scholar.google.com/scholar?q="Transformation+of+Aspergillus+nidulansby+using+a+trpC+plasmid"'>Transformation of Aspergillus nidulansby using a trpC plasmid</a>", Proc. Natl. Acad. Sci. USA 81:1470-1474 (1984).</td></tr><tr><td class="patent-data-table-td ">185</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Yelton, M.M., et al., Transformation of Aspergillus nidulans by using a trpC plasmid , Proc. Natl. Acad. Sci. USA 81:1470 1474 (1984).</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="forward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Referenced by</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Citing Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6379928">US6379928</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Dec 10, 1999</td><td class="patent-data-table-td patent-date-value">Apr 30, 2002</td><td class="patent-data-table-td ">Genencor International, Inc.</td><td class="patent-data-table-td ">Heterologous polypeptides expressed in filamentous fungi, processes for making same, and vectors for making same</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7659065">US7659065</a></td><td class="patent-data-table-td patent-date-value">Aug 11, 2005</td><td class="patent-data-table-td patent-date-value">Feb 9, 2010</td><td class="patent-data-table-td ">Ramot At Tel Aviv University Ltd.</td><td class="patent-data-table-td ">Transgenic fungi expressing Bcl-2 and methods of using Bcl-2 or portions thereof for improving biomass production, survival, longevity and stress resistance of fungi</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8044192">US8044192</a></td><td class="patent-data-table-td patent-date-value">Aug 18, 2008</td><td class="patent-data-table-td patent-date-value">Oct 25, 2011</td><td class="patent-data-table-td ">Danisco Us Inc.</td><td class="patent-data-table-td ">Trichoderma promoter</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8138321">US8138321</a></td><td class="patent-data-table-td patent-date-value">Sep 21, 2007</td><td class="patent-data-table-td patent-date-value">Mar 20, 2012</td><td class="patent-data-table-td ">Danisco Us Inc.</td><td class="patent-data-table-td ">Acetolactate synthase (ALS) selectable marker from Trichoderma reesei</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8198046">US8198046</a></td><td class="patent-data-table-td patent-date-value">Jun 21, 2007</td><td class="patent-data-table-td patent-date-value">Jun 12, 2012</td><td class="patent-data-table-td ">Danisco Us Inc.</td><td class="patent-data-table-td ">KEX2 cleavage regions of recombinant fusion proteins</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8455631">US8455631</a></td><td class="patent-data-table-td patent-date-value">Aug 25, 2011</td><td class="patent-data-table-td patent-date-value">Jun 4, 2013</td><td class="patent-data-table-td ">Danisco Us Inc.</td><td class="patent-data-table-td ">Trichoderma promoter</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8518667">US8518667</a></td><td class="patent-data-table-td patent-date-value">May 15, 2012</td><td class="patent-data-table-td patent-date-value">Aug 27, 2013</td><td class="patent-data-table-td ">Danisco Us Inc.</td><td class="patent-data-table-td ">KEX2 cleavage regions of recombinant fusion proteins</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2607485A1?cl=en">EP2607485A1</a></td><td class="patent-data-table-td patent-date-value">Jun 21, 2007</td><td class="patent-data-table-td patent-date-value">Jun 26, 2013</td><td class="patent-data-table-td ">Danisco US Inc.</td><td class="patent-data-table-td ">KEX2 cleavage regions of recombinant fusion proteins</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2607486A1?cl=en">EP2607486A1</a></td><td class="patent-data-table-td patent-date-value">Jun 21, 2007</td><td class="patent-data-table-td patent-date-value">Jun 26, 2013</td><td class="patent-data-table-td ">Danisco US Inc.</td><td class="patent-data-table-td ">KEX2 cleavage regions of recombinant fusion proteins</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="classifications"></a><div class="patent-section-header"><span class="patent-section-title">Classifications</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th"> </th><th class="patent-data-table-th"> </th></tr></thead><tr><td class="patent-data-table-td ">U.S. Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=J8tQBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S069100">435/69.1</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=J8tQBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S183000">435/183</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=J8tQBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S484000">435/484</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=J8tQBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S023200">536/23.2</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=J8tQBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S254400">435/254.4</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=J8tQBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S254600">435/254.6</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=J8tQBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S320100">435/320.1</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=J8tQBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S023100">536/23.1</a></span></td></tr><tr><td class="patent-data-table-td ">International Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=J8tQBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0001190000">C12N1/19</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=J8tQBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0015620000">C12N15/62</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=J8tQBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0015800000">C12N15/80</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=J8tQBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0009340000">C12N9/34</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=J8tQBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0001150000">C12N1/15</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=J8tQBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0009580000">C12N9/58</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=J8tQBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0009640000">C12N9/64</a></span></td></tr><tr><td class="patent-data-table-td ">Cooperative Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=J8tQBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C07K2319/003">C07K2319/003</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=J8tQBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C07K2319/61">C07K2319/61</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=J8tQBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N15/80">C12N15/80</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=J8tQBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N15/62">C12N15/62</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=J8tQBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N9/2428">C12N9/2428</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=J8tQBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N9/58">C12N9/58</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=J8tQBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C07K2319/02">C07K2319/02</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=J8tQBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N9/6481">C12N9/6481</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=J8tQBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C07K2319/00">C07K2319/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=J8tQBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C07K2319/50">C07K2319/50</a></span></td></tr><tr><td class="patent-data-table-td ">European Classification</td><td class="patent-data-table-td "><span class="nested-value">C12N9/64F23A</span>, <span class="nested-value">C12N9/24A2B1C</span>, <span class="nested-value">C12N15/62</span>, <span class="nested-value">C12N15/80</span>, <span class="nested-value">C12N9/58</span></td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="legal-events"></a><div class="patent-section-header"><span class="patent-section-title">Legal Events</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Date</th><th class="patent-data-table-th">Code</th><th class="patent-data-table-th">Event</th><th class="patent-data-table-th">Description</th></tr></thead><tr><td class="patent-data-table-td patent-date-value">Feb 15, 2012</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">12</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Oct 25, 2011</td><td class="patent-data-table-td ">B1</td><td class="patent-data-table-td ">Reexamination certificate first reexamination</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">THE PATENTABILITY OF CLAIMS 1-20, 46 AND 47 IS CONFIRMED. CLAIMS 21, 22 AND 48 ARE DETERMINED TO BEPATENTABLE AS AMENDED. CLAIMS 23-45, DEPENDENT ON AN AMENDED CLAIM, ARE DETERMINED TO BE PATENTABLE.</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Aug 25, 2009</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">DANISCO US INC., CALIFORNIA</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">CHANGE OF NAME;ASSIGNOR:GENENCOR INTERNATIONAL, INC.;REEL/FRAME:023134/0632</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20070207</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Aug 18, 2009</td><td class="patent-data-table-td ">RR</td><td class="patent-data-table-td ">Request for reexamination filed</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20090604</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Jan 17, 2008</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">8</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Jan 14, 2004</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">4</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Jun 18, 2002</td><td class="patent-data-table-td ">CC</td><td class="patent-data-table-td ">Certificate of correction</td><td class="patent-data-table-td "></td></tr></table><div class="patent-section-footer"></div></div><div class="modal-dialog" id="patent-images-lightbox"><div class="patent-lightbox-controls"><div class="patent-lightbox-rotate-controls"><div class="patent-lightbox-rotation-text">Rotate</div><div class="rotate-icon rotate-ccw-icon"></div><div class="rotate-icon rotate-cw-icon"></div></div><div class="patent-lightbox-index-counter"></div><a class="patent-lightbox-fullsize-link" target="_blank">Original Image</a><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_right.png" alt="Next page"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_left.png" alt="Previous page"width="21" height="21" /></div></div><div class="modal-dialog-content"><div class="patent-lightbox-image-holder"><div class="patent-lightbox-placeholder"></div></div></div></div><script>_OC_initPatentsAtb({image_not_available_html: " Image not available"});</script></div></div></div></td></tr></table><script>(function() {var href = window.location.href;if (href.indexOf('?') !== -1) {var parameters = href.split('?')[1].split('&');for (var i = 0; i < parameters.length; i++) {var param = parameters[i].split('=');if (param[0] == 'focus') {var elem = document.getElementById(param[1]);if (elem) {elem.focus();}}}}})();</script><script>_OC_addFlags({LockSrc:"/books/javascript/lock_8a2b04e7bf975d5171d8e4c0b6365c7a.js", Host:"http://www.google.com/", IsBooksRentalEnabled:1, IsWebstoreDisplayCaseEnabled:1, IsObfuscationEnabled:1, IsBrowsingHistoryEnabled:1, IsWebReaderSvgEnabled:0, IsGeoLayerEnabled:1, IsImageModeNotesEnabled:1, IsCopyMenuItemEnabled:1, IsGiftingEnabled:0, IsWebReaderUniversalPaginatorEnabled:0, IsOfflineBubbleEnabled:1, IsReaderEnabledForPlayRequests:1, IsFutureOnSaleVolumesEnabled:1, IsOfflineRestrictedCopyEnabled:1, IsBooksUnifiedLeftNavEnabled:1, IsRestrictedCopyEnabled:1, IsZipitFolderCollectionEnabled:1, IsEndOfSampleRecommendationsEnabled:1, IsRatingsOnBookcardsEnabled:1, IsAdsDisabled:0, IsIframePageDisplayEnabled:0, IsEmbeddedMediaEnabled:1, IsImageModeAnnotationsEnabled:1, IsMyLibraryGooglePlusEnabled:1, IsImagePageProviderEnabled:0, IsBookcardListPriceSmall:0, IsInternalUser:0, IsBooksShareButtonEnabled:0, IsPreOrdersEnabled:0, IsDisabledRandomBookshelves:0, WebstoreDisplayCasePosition:3});_OC_Run({"enable_p13n":false,"add_vol_to_collection_base_url":"http://www.google.com/patents?op=add\u0026sig=ACfU3U1m5knxPHeWWS-PXwTLR5Ap9mBqkg\u0026id=J8tQBAABERAJ","remove_vol_from_collection_base_url":"http://www.google.com/patents?op=remove\u0026sig=ACfU3U256CpUarUytDASGjacIAS9DGafVg\u0026id=J8tQBAABERAJ","logged_in":false,"p13n_save_user_settings_url":"http://www.google.com/patents?op=edit_user_settings\u0026sig=ACfU3U1E-1zlaIZHdT0wILHiRg8rLPBE6g","is_cobrand":false,"sign_in_url":"https://www.google.com/accounts/Login?service=\u0026continue=http://www.google.com/patents%3Fhl%3Den\u0026hl=en","is_play_enabled":true}, {"volume_id":"","is_ebook":true,"volumeresult":{"has_flowing_text":false,"has_scanned_text":true,"can_download_pdf":false,"can_download_epub":false,"is_pdf_drm_enabled":false,"is_epub_drm_enabled":false,"download_pdf_url":"http://www.google.com/patents/download/Heterologous_polypeptides_expressed_in_f.pdf?id=J8tQBAABERAJ\u0026output=pdf\u0026sig=ACfU3U0UwBJtdzNXb7A6Ga5JQREaEZWGbg"},"sample_url":"http://www.google.com/patents/reader?id=J8tQBAABERAJ\u0026printsec=frontcover\u0026output=reader\u0026source=gbs_atb_hover","is_browsable":true,"is_public_domain":true}, {});</script><div id="footer_table" style="font-size:83%;text-align:center;position:relative;top:20px;height:4.5em;margin-top:2em"><div style="margin-bottom:8px"><a href=http://www.google.com/><nobr>Google&nbsp;Home</nobr></a> - <a href=//www.google.com/patents/sitemap/><nobr>Sitemap</nobr></a> - <a href=http://www.google.com/googlebooks/uspto.html><nobr>USPTO Bulk Downloads</nobr></a> - <a href=/intl/en/privacy/><nobr>Privacy Policy</nobr></a> - <a href=/intl/en/policies/terms/><nobr>Terms of Service</nobr></a> - <a href=https://support.google.com/faqs/answer/2539193?hl=en><nobr>About Google Patents</nobr></a> - <a href="http://www.google.com/tools/feedback/intl/en/error.html" onclick="try{_OC_startFeedback({productId: '72792',locale: 'en'});return false;}catch(e){}"><nobr>Send Feedback</nobr></a></div><span>Data provided by IFI CLAIMS Patent Services</span><br><span >&copy;2012 Google</span></div> <script type="text/javascript">var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));</script><script type="text/javascript">var pageTracker = _gat._getTracker("UA-27188110-1");pageTracker._setCookiePath("/patents/");pageTracker._trackPageview();</script> </body></html>