<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN""http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html><head><title>Patent US6107023 - Isolating nucleotide sequences in mix; joining nucleotide linker to ... - Google Patents</title><script>(function(){(function(){function e(a){this.t={};this.tick=function(a,c,b){var d=void 0!=b?b:(new Date).getTime();this.t[a]=[d,c];if(void 0==b)try{window.console.timeStamp("CSI/"+a)}catch(e){}};this.tick("start",null,a)}var a;window.performance&&(a=window.performance.timing);var f=a?new e(a.responseStart):new e;window.jstiming={Timer:e,load:f};if(a){var c=a.navigationStart,d=a.responseStart;0<c&&d>=c&&(window.jstiming.srt=d-c)}if(a){var b=window.jstiming.load;0<c&&d>=c&&(b.tick("_wtsrt",void 0,c),b.tick("wtsrt_",
"_wtsrt",d),b.tick("tbsd_","wtsrt_"))}try{a=null,window.chrome&&window.chrome.csi&&(a=Math.floor(window.chrome.csi().pageT),b&&0<c&&(b.tick("_tbnd",void 0,window.chrome.csi().startE),b.tick("tbnd_","_tbnd",c))),null==a&&window.gtbExternal&&(a=window.gtbExternal.pageT()),null==a&&window.external&&(a=window.external.pageT,b&&0<c&&(b.tick("_tbnd",void 0,window.external.startE),b.tick("tbnd_","_tbnd",c))),a&&(window.jstiming.pt=a)}catch(g){}})();})();
</script><link rel="stylesheet" href="/patents/css/_8a2b04e7bf975d5171d8e4c0b6365c7a/kl_intl_patents_bundle.css" type="text/css" /><script src="/books/javascript/atb_8a2b04e7bf975d5171d8e4c0b6365c7a__en.js"></script><script>function googleTranslateElementInit() {new google.translate.TranslateElement({pageLanguage: "en",gaTrack: true,gaId: "UA-27188110-1",multilanguagePage: true});}</script><script src="//translate.google.com/translate_a/element.js?cb=googleTranslateElementInit"></script><meta name="DC.type" content="Patent"><meta name="DC.title" content="Isolating nucleotide sequences in mix; joining nucleotide linker to nucleotide sequences by ligation with subsequent amplification, denaturation and hybridization of nucleotide sequences, isolating preferential nucleotide sequences"><meta name="DC.contributor" content="Gregory R. Reyes" scheme="inventor"><meta name="DC.contributor" content="Jungsuh Kim" scheme="inventor"><meta name="DC.contributor" content="Genelabs Technologies, Inc." scheme="assignee"><meta name="DC.date" content="1988-6-17" scheme="dateSubmitted"><meta name="DC.description" content="A method of isolating genomic or RNA-derived duplex fragments which are unique to one of two fragment mixtures. The fragments in positive-source and negative-source mixtures are separately equipped with end linkers, and each mixture is amplified by successive primed-strand replications, using a single primer which is homologous to the associated linker. The second-source linker is biotinylated, and the fragments in this mixture are hybridized in molar excess with the fragments in the positive-source mixture. DNA species which are not hybridized with the biotinylated species, i.e., species that are unique to the positive-source mixture, are isolated after removal of hybridized species by affinity chromatography. Also disclosed is a method of amplifying a mixture of DNA fragments by repeated linker/primer replication."><meta name="DC.date" content="2000-8-22" scheme="issued"><meta name="DC.relation" content="EP:0224126:A2" scheme="references"><meta name="DC.relation" content="US:4683195" scheme="references"><meta name="citation_reference" content="Maniatis et al., Molecular Cloning: A Laboratory Manual, Cold Spring Harbor, NY, 1982."><meta name="citation_patent_number" content="US:6107023"><meta name="citation_patent_application_number" content="US:07/208,512"><link rel="canonical" href="http://www.google.com/patents/US6107023"/><meta property="og:url" content="http://www.google.com/patents/US6107023"/><meta name="title" content="Patent US6107023 - Isolating nucleotide sequences in mix; joining nucleotide linker to nucleotide sequences by ligation with subsequent amplification, denaturation and hybridization of nucleotide sequences, isolating preferential nucleotide sequences"/><meta name="description" content="A method of isolating genomic or RNA-derived duplex fragments which are unique to one of two fragment mixtures. The fragments in positive-source and negative-source mixtures are separately equipped with end linkers, and each mixture is amplified by successive primed-strand replications, using a single primer which is homologous to the associated linker. The second-source linker is biotinylated, and the fragments in this mixture are hybridized in molar excess with the fragments in the positive-source mixture. DNA species which are not hybridized with the biotinylated species, i.e., species that are unique to the positive-source mixture, are isolated after removal of hybridized species by affinity chromatography. Also disclosed is a method of amplifying a mixture of DNA fragments by repeated linker/primer replication."/><meta property="og:title" content="Patent US6107023 - Isolating nucleotide sequences in mix; joining nucleotide linker to nucleotide sequences by ligation with subsequent amplification, denaturation and hybridization of nucleotide sequences, isolating preferential nucleotide sequences"/><meta property="og:type" content="book"/><meta property="og:site_name" content="Google Books"/><meta property="og:image" content="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><link rel="image_src" href="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><script>(function(){try{var aa=function(a,b,c,d){d=d||{};d._sn=["cfg",b,c].join(".");window.gbar.logger.ml(a,d)};var g=window.gbar=window.gbar||{},l=window.gbar.i=window.gbar.i||{},m={},n;function _tvn(a,b){var c=parseInt(a,10);return isNaN(c)?b:c}function _tvf(a,b){var c=parseFloat(a);return isNaN(c)?b:c}function _tvv(a){return!!a}function p(a,b,c){(c||g)[a]=b}g.bv={n:_tvn("2",0),r:"",f:".67.",e:"0",m:_tvn("0",1)};
function q(a,b,c){var d="on"+b;if(a.addEventListener)a.addEventListener(b,c,!1);else if(a.attachEvent)a.attachEvent(d,c);else{var f=a[d];a[d]=function(){var a=f.apply(this,arguments),b=c.apply(this,arguments);return void 0==a?b:void 0==b?a:b&&a}}}var s=function(a){return function(){return g.bv.m==a}},ba=s(1),ca=s(2);p("sb",ba);p("kn",ca);l.a=_tvv;l.b=_tvf;l.c=_tvn;l.i=aa;var da=window.gbar.i.i;var t,u,v,w;function ea(a){v=a}function fa(a){var b;if(b=v&&window.encodeURIComponent)b=a.href,b=!b.match(/^http[s]?:\/\/accounts\.google\.[^/]*\/ClearSID/i)&&!b.match(/^http[s]?:\/\/[^/]*\/accounts\/ClearSID/i);if(b=b&&encodeURIComponent(v()))a.href=a.href.replace(/([?&]continue=)[^&]*/,"$1"+b)}function ga(a){window.gApplication&&(a.href=window.gApplication.getTabUrl(a.href))}
function ha(a){var b=document.forms[0].q,c=window.encodeURIComponent&&b&&b.value,b=b&&b.placeholder;c&&c!=b&&(a.href=a.href.replace(/([?&])q=[^&]*|$/,function(a,b){return(b||"&")+"q="+encodeURIComponent(c)}))}n=l.a("")?ga:ha;
function x(a,b,c,d,f,e){var h=document.getElementById(a);if(h){var k=h.style;k.left=d?"auto":b+"px";k.right=d?b+"px":"auto";k.top=c+"px";k.visibility=u?"hidden":"visible";f&&e?(k.width=f+"px",k.height=e+"px"):(x(t,b,c,d,h.offsetWidth,h.offsetHeight),u=u?"":a)}}
var y=[],ia=function(a,b){y.push(b)},ja=function(a){a=a||window.event;var b=a.target||a.srcElement;a.cancelBubble=!0;null==t&&(a=document.createElement(Array.every||window.createPopup?"iframe":"div"),a.frameBorder="0",t=a.id="gbs",a.src="javascript:''",b.parentNode.appendChild(a),q(document,"click",z));var c=b,b=0;"gb3"!=c.className&&(c=c.parentNode);a=c.getAttribute("aria-owns")||"gbi";var d=c.offsetWidth,f=20<c.offsetTop?46:24;document.getElementById("tphdr")&&(f-=3);var e=!1;do b+=c.offsetLeft||
0;while(c=c.offsetParent);var c=(document.documentElement.clientWidth||document.body.clientWidth)-b-d,h,d=document.body,k=document.defaultView;k&&k.getComputedStyle?(d=k.getComputedStyle(d,""))&&(h=d.direction):h=d.currentStyle?d.currentStyle.direction:d.style.direction;h="rtl"==h;if("gbi"==a){for(d=0;k=y[d++];)k();A(null,window.navExtra);h&&(b=c,e=!0)}else h||(b=c,e=!0);u!=a&&z();x(a,b,f,e)},z=function(){u&&x(u,0,0)},A=function(a,b){var c,d=document.getElementById("gbi"),f=a;f||(f=d.firstChild);
for(;b&&(c=b.pop());){var e=d,h=c,k=f;w||(w="gb2");e.insertBefore(h,k).className=w}},ka=function(a,b,c){if((b=document.getElementById(b))&&a){a.className="gb4";var d=document.createElement("span");d.appendChild(a);d.appendChild(document.createTextNode(" | "));d.id=c;b.appendChild(d)}},la=function(){return document.getElementById("gb_70")},ma=function(){return!!u};p("qs",n);p("setContinueCb",ea);p("pc",fa);p("tg",ja);p("close",z);p("addLink",ka);p("almm",A);p("si",la);p("adh",ia);p("op",ma);var B=function(){},C=function(){},F=function(a){var b=new Image,c=D;b.onerror=b.onload=b.onabort=function(){try{delete E[c]}catch(a){}};E[c]=b;b.src=a;D=c+1},E=[],D=0;p("logger",{il:C,ml:B,log:F});var G=window.gbar.logger;var H={},na={},I=[],oa=l.b("0.1",.1),pa=l.a("1",!0),qa=function(a,b){I.push([a,b])},ra=function(a,b){H[a]=b},sa=function(a){return a in H},J={},K=function(a,b){J[a]||(J[a]=[]);J[a].push(b)},ta=function(a){K("m",a)},L=function(a,b){var c=document.createElement("script");c.src=a;c.async=pa;Math.random()<oa&&(c.onerror=function(){c.onerror=null;B(Error("Bundle load failed: name="+(b||"UNK")+" url="+a))});(document.getElementById("xjsc")||document.getElementsByTagName("body")[0]||
document.getElementsByTagName("head")[0]).appendChild(c)},N=function(a){for(var b=0,c;(c=I[b])&&c[0]!=a;++b);!c||c[1].l||c[1].s||(c[1].s=!0,M(2,a),c[1].url&&L(c[1].url,a),c[1].libs&&m.d&&m.d(c[1].libs))},O=function(a){K("gc",a)},P=null,ua=function(a){P=a},M=function(a,b,c){if(P){a={t:a,b:b};if(c)for(var d in c)a[d]=c[d];try{P(a)}catch(f){}}};p("mdc",H);p("mdi",na);p("bnc",I);p("qGC",O);p("qm",ta);p("qd",J);p("lb",N);p("mcf",ra);p("bcf",qa);p("aq",K);p("mdd","");p("has",sa);
p("trh",ua);p("tev",M);var Q=l.b("0.1",.001),R=0;
function _mlToken(a,b){try{if(1>R){R++;var c,d=a,f=b||{},e=encodeURIComponent,h=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&jexpid=",e("17483"),"&srcpg=",e("prop=22"),"&jsr=",Math.round(1/Q),"&ogev=",e("CKLtU8bcHrbYsQTW6ICYDA"),"&ogf=",g.bv.f,"&ogrp=",e("1"),"&ogv=",e("1407723702.0"),"&oggv="+e("es_plusone_gc_20140723.0_p0"),"&ogd=",e("com"),"&ogc=",e("USA"),"&ogl=",e("en")];f._sn&&(f._sn="og."+
f._sn);for(var k in f)h.push("&"),h.push(e(k)),h.push("="),h.push(e(f[k]));h.push("&emsg=");h.push(e(d.name+":"+d.message));var r=h.join("");S(r)&&(r=r.substr(0,2E3));c=r;var Aa=window.gbar.logger._aem(a,c);F(Aa)}}catch(Na){}}var S=function(a){return 2E3<=a.length},va=function(a,b){return b};function T(a){B=a;p("_itl",S,G);p("_aem",va,G);p("ml",B,G);a={};H.er=a}l.a("")?T(function(a){throw a;}):l.a("1")&&Math.random()<Q&&T(_mlToken);I.push(["m",{url:"//ssl.gstatic.com/gb/js/scm_7385cc5883250b43a39405734c1bea59.js"}]);g.mcf("c",{});g.sg={c:""};if(l.a("1")){var wa=l.a("");I.push(["gc",{auto:wa,url:"//ssl.gstatic.com/gb/js/abc/gci_91f30755d6a6b787dcc2a4062e6e9824.js",libs:"googleapis.client:plusone:gapi.iframes"}]);var xa={version:"gci_91f30755d6a6b787dcc2a4062e6e9824.js",index:"",lang:"en"};H.gc=xa;var U=function(a){window.googleapis&&window.iframes?a&&a():(a&&O(a),N("gc"))};p("lGC",U);l.a("1")&&p("lPWF",U)};window.__PVT="";if(l.a("1")&&l.a("1")){var V=function(a){U(function(){K("pw",a);N("pw")})};p("lPW",V);I.push(["pw",{url:"//ssl.gstatic.com/gb/js/abc/pwm_45f73e4df07a0e388b0fa1f3d30e7280.js"}]);var W=[],ya=function(a){W[0]=a},za=function(a,b){var c=b||{};c._sn="pw";B(a,c)},Ba={signed:W,elog:za,base:"https://plusone.google.com/u/0",loadTime:(new Date).getTime()};H.pw=Ba;var X=function(a,b){for(var c=b.split("."),d=function(){var b=arguments;a(function(){for(var a=g,d=0,e=c.length-1;d<e;++d)a=a[c[d]];a[c[d]].apply(a,b)})},f=g,e=0,h=c.length-1;e<h;++e)f=
f[c[e]]=f[c[e]]||{};return f[c[e]]=d};X(V,"pw.clk");X(V,"pw.hvr");p("su",ya,g.pw)};function Ca(){function a(){for(var b;(b=e[h++])&&"m"!=b[0]&&!b[1].auto;);b&&(M(2,b[0]),b[1].url&&L(b[1].url,b[0]),b[1].libs&&m.d&&m.d(b[1].libs));h<e.length&&setTimeout(a,0)}function b(){0<f--?setTimeout(b,0):a()}var c=l.a("1"),d=l.a(""),f=3,e=I,h=0,k=window.gbarOnReady;if(k)try{k()}catch(r){da(r,"ml","or")}d?p("ldb",a):c?q(window,"load",b):b()}p("rdl",Ca);var Da={D:1,H:2,da:3,p:4,W:5,M:6,F:7,g:8,ha:9,U:10,L:11,T:12,S:13,N:14,Q:15,P:16,fa:17,w:18,O:19,ga:20,ea:21,u:22,G:23,ja:24,ka:25,ia:26,A:27,j:28,o:29,k:30,ca:31,Z:32,$:33,J:34,K:35,ba:36,aa:37,Y:38,B:39,R:40,v:41,X:42,V:43,h:48,C:49,I:500},Y=[1,2,3,4,5,6,9,10,11,13,14,28,29,30,34,35,37,38,39,40,41,42,43,48,49,500];var Z=l.b("0.001",1E-4),Ea=l.b("1",1),Fa=!1,Ga=!1;if(l.a("1")){var Ha=Math.random();Ha<=Z&&(Fa=!0);Ha<=Ea&&(Ga=!0)}var Ia=Da,$=null;function Ja(){var a=0,b=function(b,d){l.a(d)&&(a|=b)};b(1,"");b(2,"");b(4,"");b(8,"");return a}
function Ka(a,b){var c=Z,d=Fa,f;f=a;if(!$){$={};for(var e=0;e<Y.length;e++){var h=Y[e];$[h]=!0}}if(f=!!$[f])c=Ea,d=Ga;if(d){d=encodeURIComponent;g.rp?(f=g.rp(),f="-1"!=f?f:"1"):f="1";c=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&oge=",a,"&ogex=",d("17483"),"&ogev=",d("CKLtU8bcHrbYsQTW6ICYDA"),"&ogf=",g.bv.f,"&ogp=",d("22"),"&ogrp=",d(f),"&ogsr=",Math.round(1/c),"&ogv=",d("1407723702.0"),"&oggv="+
d("es_plusone_gc_20140723.0_p0"),"&ogd=",d("com"),"&ogl=",d("en"),"&ogc=",d("USA"),"&ogus=",Ja()];if(b){"ogw"in b&&(c.push("&ogw="+b.ogw),delete b.ogw);var k;f=b;e=[];for(k in f)0!=e.length&&e.push(","),e.push(La(k)),e.push("."),e.push(La(f[k]));k=e.join("");""!=k&&(c.push("&ogad="),c.push(d(k)))}F(c.join(""))}}function La(a){"number"==typeof a&&(a+="");return"string"==typeof a?a.replace(".","%2E").replace(",","%2C"):a}C=Ka;p("il",C,G);var Ma={};H.il=Ma;setTimeout(function(){C(Ia.g)},0);}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var b=window.gbar.i.i;var c=window.gbar;var f=function(d){try{var a=document.getElementById("gbom");a&&d.appendChild(a.cloneNode(!0))}catch(e){b(e,"omas","aomc")}};c.aomc=f;}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var a=window.gbar;a.mcf("pm",{p:""});}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{window.gbar.rdl();}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
if (window['_OC_timingAction']) {window['_OC_timingAction']('patents_refpage');}</script><style>#gbar,#guser{font-size:13px;padding-top:1px !important;}#gbar{float:left;height:22px}#guser{padding-bottom:7px !important;text-align:right}.gbh,.gbd{border-top:1px solid #c9d7f1;font-size:1px}.gbh{height:0;position:absolute;top:24px;width:100%}#gbs,.gbm{background:#fff;left:0;position:absolute;text-align:left;visibility:hidden;z-index:1000}.gbm{border:1px solid;border-color:#c9d7f1 #36c #36c #a2bae7;z-index:1001}.gb1{margin-right:.5em}.gb1,.gb3{zoom:1}.gb2{display:block;padding:.2em .5em}.gb2,.gb3{text-decoration:none !important;border-bottom:none}a.gb1,a.gb4{text-decoration:underline !important}a.gb1,a.gb2,a.gb3,a.gb4{color:#00c !important}.gbi .gb3,.gbi .gb2,.gbi .gb4{color:#dd8e27 !important}.gbf .gb3,.gbf .gb2,.gbf .gb4{color:#900 !important}a.gb2:hover{background:#36c;color:#fff !important}#gbar .gbz0l{color:#000 !important;cursor:default;font-weight:bold;text-decoration:none !important}
#gbar { padding:.3em .6em !important;}</style></head><body  topmargin="3" marginheight="3"><div id=gbar><nobr><a onclick=gbar.qs(this);gbar.logger.il(1,{t:1}); class=gb1 id=gb_1 href="https://www.google.com/search?sa=N&tab=tw">Search</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:2}); class=gb1 id=gb_2 href="http://www.google.com/search?hl=en&tbm=isch&source=og&sa=N&tab=ti">Images</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:8}); class=gb1 id=gb_8 href="http://maps.google.com/maps?hl=en&sa=N&tab=tl">Maps</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:78}); class=gb1 id=gb_78 href="https://play.google.com/?hl=en&sa=N&tab=t8">Play</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:36}); class=gb1 id=gb_36 href="http://www.youtube.com/results?sa=N&tab=t1">YouTube</a> <a onclick=gbar.logger.il(1,{t:5}); class=gb1 id=gb_5 href="http://news.google.com/nwshp?hl=en&tab=tn">News</a> <a onclick=gbar.logger.il(1,{t:23}); class=gb1 id=gb_23 href="https://mail.google.com/mail/?tab=tm">Gmail</a> <a onclick=gbar.logger.il(1,{t:25}); class=gb1 id=gb_25 href="https://drive.google.com/?tab=to">Drive</a> <a class=gb3 href="http://www.google.com/intl/en/options/" onclick="this.blur();gbar.tg(event);return !1" aria-haspopup=true><u>More</u> <small>&#9660;</small></a><div class=gbm id=gbi><a onclick=gbar.logger.il(1,{t:24}); class=gb2 id=gb_24 href="https://www.google.com/calendar?tab=tc">Calendar</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:51}); class=gb2 id=gb_51 href="http://translate.google.com/?hl=en&sa=N&tab=tT">Translate</a><a onclick=gbar.logger.il(1,{t:17}); class=gb2 id=gb_17 href="http://www.google.com/mobile/?hl=en&tab=tD">Mobile</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:10}); class=gb2 id=gb_10 href="http://www.google.com/search?hl=en&tbo=u&tbm=bks&source=og&sa=N&tab=tp">Books</a><a onclick=gbar.logger.il(1,{t:212}); class=gb2 id=gb_212 href="https://wallet.google.com/manage/?tab=ta">Wallet</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:6}); class=gb2 id=gb_6 href="http://www.google.com/search?hl=en&tbo=u&tbm=shop&source=og&sa=N&tab=tf">Shopping</a><a onclick=gbar.logger.il(1,{t:30}); class=gb2 id=gb_30 href="http://www.blogger.com/?tab=tj">Blogger</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:27}); class=gb2 id=gb_27 href="http://www.google.com/finance?sa=N&tab=te">Finance</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:31}); class=gb2 id=gb_31 href="https://plus.google.com/photos?sa=N&tab=tq">Photos</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:12}); class=gb2 id=gb_12 href="http://www.google.com/search?hl=en&tbo=u&tbm=vid&source=og&sa=N&tab=tv">Videos</a><div class=gb2><div class=gbd></div></div><a onclick=gbar.logger.il(1,{t:66}); href="http://www.google.com/intl/en/options/" class=gb2>Even more &raquo;</a></div></nobr></div><div id=guser width=100%><nobr><span id=gbn class=gbi></span><span id=gbf class=gbf></span><span id=gbe></span><a target=_top id=gb_70 href="https://www.google.com/accounts/Login?service=&continue=http://www.google.com/patents%3Fhl%3Den&hl=en" class=gb4>Sign in</a><div style="display: none"><div class=gbm id=gbd5 aria-owner=gbg5><div class=gbmc><ol id=gbom class=gbmcc></ol></div></div></div></nobr></div><div class=gbh style=left:0></div><div class=gbh style=right:0></div><div role="alert" style="position: absolute; left: 0; right: 0;"><a href="http://www.google.com/patents/us6107023?hl=en&amp;output=html_text" title="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."><img border="0" src="http://www.google.com/images/cleardot.gif"alt="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."></a></div><div id="guser"><nobr></nobr></div><div style="clear:both;"></div><div id="gb-top-search-box" class="gb-top-search-box-small gb-reset"><table><tr><td class="logo"><a href="http://www.google.com/patents" class="logo-link"><img class="logo-img" src="/intl/en/images/logos/google_logo_41.png" alt="Go to Google Books Home" height="41"/></a></td><td><form action="http://www.google.com/search" name="f" id="vheadf" method="get"><span id="hf"></span><input type="hidden" name="tbm" value="pts"/><input type="hidden" name="tbo" value="1"/><input type="hidden" name="hl" value="en"/><table><tr><td><div class="inputs"><table><tr><td><div class="text-input"><input type="text" name="q" id="vheadq" class="text" maxlength="2048" size="31" value="" title="Search Patents" accesskey="s" autocomplete="off"/><script>window._OC_autoDir &&window._OC_autoDir('vheadq', 'tia-vheadq');</script></div></td><td><div class="submit-input"><input name="btnG" class="submit" type="submit" value=""/></div></td></tr></table></div></td><td class="col-ext-links"><div class="ext-links"><a href="http://www.google.com/advanced_patent_search">&lt;nobr&gt;Advanced Patent Search&lt;/nobr&gt;</a></div></td></tr></table></form></td></tr></table></div><div class="kd-appbar"><h2 class="kd-appname"><a href="/patents">Patents</a></h2><div class="kd-buttonbar left" id="left-toolbar-buttons"><a id="appbar-write-review-link" href=""></a><a id="appbar-view-print-sample-link" href=""></a><a id="appbar-view-ebook-sample-link" href=""></a><a id="appbar-patents-prior-art-finder-link" href="https://www.google.com/patents/related/US6107023"></a><a id="appbar-patents-discuss-this-link" href="http://www.google.com/url?id=aL1QBAABERAJ&amp;q=http://patents.stackexchange.com/redirect/google-patents%3Fpatent%3DUS6107023&amp;usg=AFQjCNGHclQVcW45-NfTJG_RBBXTP68yLQ" data-is-grant="true"></a><a id="appbar-read-patent-link" href="//docs.google.com/viewer?url=patentimages.storage.googleapis.com/pdfs/US6107023.pdf"></a><a id="appbar-download-pdf-link" href="//patentimages.storage.googleapis.com/pdfs/US6107023.pdf"></a></div><div class="kd-buttonbar right" id="right-toolbar-buttons"></div></div><div id="books-microdata" itemscope=""itemtype="http://schema.org/Book"itemid="http://www.google.com/patents/US6107023" style="display:none"><span itemprop="description">A method of isolating genomic or RNA-derived duplex fragments which are unique to one of two fragment mixtures. The fragments in positive-source and negative-source mixtures are separately equipped with end linkers, and each mixture is amplified by successive primed-strand replications, using a single...</span><span itemprop="url">http://www.google.com/patents/US6107023?utm_source=gb-gplus-share</span><span class="main-title" itemprop="name">Patent US6107023 - Isolating nucleotide sequences in mix; joining nucleotide linker to nucleotide sequences by ligation with subsequent amplification, denaturation and hybridization of nucleotide sequences, isolating preferential nucleotide sequences</span><img itemprop="image" src="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"alt="Patent US6107023 - Isolating nucleotide sequences in mix; joining nucleotide linker to nucleotide sequences by ligation with subsequent amplification, denaturation and hybridization of nucleotide sequences, isolating preferential nucleotide sequences" title="Patent US6107023 - Isolating nucleotide sequences in mix; joining nucleotide linker to nucleotide sequences by ligation with subsequent amplification, denaturation and hybridization of nucleotide sequences, isolating preferential nucleotide sequences"></div><div style="display: none"><ol id="ofe-gear-menu-contents" class="gbmcc"><li class="gbe gbmtc"><a class="gbmt goog-menuitem-content" id="" href="http://www.google.com/advanced_patent_search">Advanced Patent Search</a></li></ol></div><table id="viewport_table" cellpadding="0" style="clear:both" cellspacing="0"><tr><td id="viewport_td"><div class=vertical_module_list_row><div id=intl_patents class=about_content><div id=intl_patents_v><table class="patent-bibdata"><tr><td class="patent-bibdata-heading">Publication number</td><td class="single-patent-bibdata">US6107023 A</td></tr><tr><td class="patent-bibdata-heading">Publication type</td><td class="single-patent-bibdata">Grant</td></tr><tr><td class="patent-bibdata-heading">Application number</td><td class="single-patent-bibdata">US 07/208,512</td></tr><tr><td class="patent-bibdata-heading">Publication date</td><td class="single-patent-bibdata">Aug 22, 2000</td></tr><tr><td class="patent-bibdata-heading">Filing date</td><td class="single-patent-bibdata">Jun 17, 1988</td></tr><tr><td class="patent-bibdata-heading">Priority date<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The priority date is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the date listed."></span></td><td class="single-patent-bibdata">Jun 17, 1988</td></tr><tr><td class="patent-bibdata-heading">Fee status<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The fee status is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the status or dates listed."></span></td><td class="single-patent-bibdata">Paid</td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Also published as</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/CA1335070C">CA1335070C</a>, </span><span class="patent-bibdata-value"><a href="/patents/DE68923704D1">DE68923704D1</a>, </span><span class="patent-bibdata-value"><a href="/patents/DE68923704T2">DE68923704T2</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP0419571A1">EP0419571A1</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP0419571A4">EP0419571A4</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP0419571B1">EP0419571B1</a>, </span><span class="patent-bibdata-value"><a href="/patents/WO1989012695A1">WO1989012695A1</a></span></span></td></tr><tr class="patent-bibdata-list-row alternate-patent-number"><td class="patent-bibdata-heading">Publication number</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value">07208512, </span><span class="patent-bibdata-value">208512, </span><span class="patent-bibdata-value">US 6107023 A, </span><span class="patent-bibdata-value">US 6107023A, </span><span class="patent-bibdata-value">US-A-6107023, </span><span class="patent-bibdata-value">US6107023 A, </span><span class="patent-bibdata-value">US6107023A</span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Inventors</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Gregory+R.+Reyes%22">Gregory R. Reyes</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Jungsuh+Kim%22">Jungsuh Kim</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Original Assignee</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=inassignee:%22Genelabs+Technologies,+Inc.%22">Genelabs Technologies, Inc.</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Export Citation</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/US6107023.bibtex">BiBTeX</a>, </span><span class="patent-bibdata-value"><a href="/patents/US6107023.enw">EndNote</a>, </span><span class="patent-bibdata-value"><a href="/patents/US6107023.ris">RefMan</a></span></span></td></tr><tr class="patent-internal-links"><td colspan=2><span class="patent-bibdata-value"><a href="#backward-citations">Patent Citations</a> (2),</span> <span class="patent-bibdata-value"><a href="#npl-citations">Non-Patent Citations</a> (1),</span> <span class="patent-bibdata-value"><a href="#forward-citations">Referenced by</a> (44),</span> <span class="patent-bibdata-value"><a href="#classifications">Classifications</a> (14),</span> <span class="patent-bibdata-value"><a href="#legal-events">Legal Events</a> (17)</span> </td></tr><tr><td colspan=2 class="patent-bibdata-external-link-spacer-top"></td></tr><tr class="patent-bibdata-external-link-spacer-bottom"></tr><tr><td colspan=2><span class="patent-bibdata-heading">External Links:&nbsp;</span><span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=aL1QBAABERAJ&q=http://patft.uspto.gov/netacgi/nph-Parser%3FSect2%3DPTO1%26Sect2%3DHITOFF%26p%3D1%26u%3D/netahtml/PTO/search-bool.html%26r%3D1%26f%3DG%26l%3D50%26d%3DPALL%26RefSrch%3Dyes%26Query%3DPN/6107023&usg=AFQjCNGKNMFwY5p-7lTzmp_lwywQLe9aHA">USPTO</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=aL1QBAABERAJ&q=http://assignments.uspto.gov/assignments/q%3Fdb%3Dpat%26pat%3D6107023&usg=AFQjCNExK6n3_-8s7LqV8hpQEcb37bp9xw">USPTO Assignment</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=aL1QBAABERAJ&q=http://worldwide.espacenet.com/publicationDetails/biblio%3FCC%3DUS%26NR%3D6107023A%26KC%3DA%26FT%3DD&usg=AFQjCNHc4_F85xV6vY0vI4qBErTqwhYRJg">Espacenet</a></span></span></td></tr><tr class="patent-bibdata-group-spacer"></tr></table><div class="number-and-title"><span class="patent-title"><invention-title mxw-id="PT129055765" lang="EN" load-source="ifi">Isolating nucleotide sequences in mix; joining nucleotide linker to nucleotide sequences by ligation with subsequent amplification, denaturation and hybridization of nucleotide sequences, isolating preferential nucleotide sequences</invention-title></span><br><span class="patent-number">US 6107023 A</span></div><div class="patent-section patent-abstract-section"><div class="patent-section-header"><span class="patent-section-title">Abstract</span></div><div class="patent-text"><abstract mxw-id="PA38114887" lang="EN" load-source="patent-office"> <div class="abstract">A method of isolating genomic or RNA-derived duplex fragments which are unique to one of two fragment mixtures. The fragments in positive-source and negative-source mixtures are separately equipped with end linkers, and each mixture is amplified by successive primed-strand replications, using a single primer which is homologous to the associated linker. The second-source linker is biotinylated, and the fragments in this mixture are hybridized in molar excess with the fragments in the positive-source mixture. DNA species which are not hybridized with the biotinylated species, i.e., species that are unique to the positive-source mixture, are isolated after removal of hybridized species by affinity chromatography. Also disclosed is a method of amplifying a mixture of DNA fragments by repeated linker/primer replication.</div>
  </abstract></div></div><div class="patent-section patent-drawings-section"><div class="patent-section-header"><span class="patent-section-title">Images<span class="patent-section-count">(4)</span></span></div><div class="patent-drawings-body"><div class="patent-drawings-carousel"><div class="drawings"><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6107023-1.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6107023-1.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6107023-2.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6107023-2.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6107023-3.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6107023-3.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6107023-4.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6107023-4.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div></div></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img" alt="Previous page"src="/googlebooks/images/kennedy/page_left.png"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img" alt="Next page"src="/googlebooks/images/kennedy/page_right.png"width="21" height="21" /></div></div></div><div class="patent-post-drawings"></div><div class="patent-section patent-claims-section"><div class="patent-section-header"><span class="patent-section-title">Claims<span class="patent-section-count">(14)</span></span></div><div class="patent-text"><div mxw-id="PCLM5614333" lang="EN" load-source="patent-office" class="claims">
    <claim-statement>It is claimed:</claim-statement> <div class="claim"> <div num="1" class="claim">
      <div class="claim-text">1. A method of isolating duplex DNA fragments which are present in a mixture of different-sequence duplex DNA fragments derived from a positive source, but absent from a mixture of different-sequence duplex DNA fragments derived from a negative source, said method comprising<div class="claim-text">attaching a double-strand linker to the positive-source fragments, and separately, to the negative-source fragments, by ligating the linker to both strands of said positive-source and negative-source fragments, at both ends of said positive-source and negative-source fragments,</div> <div class="claim-text">amplifying the number of each linker-carrying fragment in each fragment mixture by successively repeating the steps of (i) denaturing the fragments to produce single fragment strands with linker regions at each strand end, (ii) hybridizing the single strands with a single-strand primer whose sequence is complementary to the linker region at one end of each strand, to form a strand/primer complex, and (iii) converting the strand/primer complexes to double-strand fragments in the presence of polymerase and deoxynucleotides,</div> <div class="claim-text">denaturing the amplified fragments in the two amplified fragment mixtures and hybridizing the denatured fragments in the two mixtures under conditions in which the linker regions associated with the positive-source strands do not hybridize with the linker regions associated with the negative-source strands, and</div> <div class="claim-text">selectively isolating DNA species which are not hybridized with DNA fragment strands from the negative source.</div> </div>
    </div>
    </div> <div class="claim-dependent"> <div num="2" class="claim">
      <div class="claim-text">2. The method of claim 1, wherein the fragment mixtures are derived from messenger RNA isolated from positive and negative sources, respectively, and the duplex fragments in each mixture are produced by converting the messenger RNA species in each mixture to double-strand cDNA fragments.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="3" class="claim">
      <div class="claim-text">3. The method of claim 1, wherein the positive source is a body-fluid sample from an individual infected with a selected microorganism, and the negative-source is the same body fluid sample obtained from a noninfected individual.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="4" class="claim">
      <div class="claim-text">4. The method of claim 1, wherein the fragments in each mixture have blunt ends, and the linkers attached to fragments have blunt ends which form, in a blunt end-to-blunt end linker dimer, a restriction site which can be cleaved with a selected restriction endonuclease.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="5" class="claim">
      <div class="claim-text">5. The method of claim 4, wherein the linker attached to the positive-source fragments and the linker attached to the negative-sequence fragments have non-homologous sequences.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="6" class="claim">
      <div class="claim-text">6. The method of claim 4, wherein the linker attached to the positive-source fragment has an internal restriction site at which the fragment can be cut by a selected restriction endonuclease, and the method further includes cloning isolated duplex DNA fragments, after digestion with said endonuclease, at a suitable vector cloning site.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="7" class="claim">
      <div class="claim-text">7. The method of claim 1, wherein said hybridizing is carried out in a molar excess of the negative-source fragments.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="8" class="claim">
      <div class="claim-text">8. The method of claim 1, wherein the negative-source fragments are biotinylated, prior to said hybridizing, and said removing includes removing biotinylated duplex fragments by affinity chromatography.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="9" class="claim">
      <div class="claim-text">9. The method of claim 8, wherein the linkers attached to the negative-source fragments have an internal restriction site which can be cleaved with a selected restriction endonuclease to produce a sticky end, and the negative-source fragments are biotinylated by digesting the amplified fragments with the selected endonuclease, and filling in the sticky end with polymerase in the presence of biotinylated deoxynucleotides.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="10" class="claim">
      <div class="claim-text">10. The method of claim 1, which further includes amplifying the number of each of the isolated DNA species by successively repeating the steps of (i) denaturing the fragments to produce single fragment strands, (ii) hybridizing the single strands with a single-strand primer whose sequence is complementary to the primer region at one end of each strand, to form a strand/primer complex, and (iii) converting the strand/primer complexes to double-strand fragments in the presence of polymerase and deoxynucleotides.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="11" class="claim">
      <div class="claim-text">11. The method of claim 10, for further isolating duplex fragments which are unique to the positive-source mixture, which further includes denaturing the negative-source mixture and the amplified isolated DNA species, and hybridizing the denatured fragments in the two mixtures under conditions in which the linker regions associated with the positive-source strands do not hybridize with the linker regions associated with the negative-source strands, and<div class="claim-text">selectively isolating DNA species which are not hybridized with DNA fragment strands from the second mixture.</div> </div>
    </div>
    </div> <div class="claim"> <div num="12" class="claim">
      <div class="claim-text">12. A method of amplifying a mixture of different sequence duplex DNA fragments, comprising<div class="claim-text">attaching a double-strand linker to the fragments, by ligating the linkers to both strands of the fragments, at both fragment ends,</div> <div class="claim-text">denaturing the fragments to produce single fragment strands with linker regions at both strand ends,</div> <div class="claim-text">hybridizing the single strands with a primer whose sequence is complementary to a linker region on each fragment strand, to form strand/primer complexes,</div> <div class="claim-text">converting the strand/primer complexes to double-strand fragments in the presence of polymerase and deoxynucleotides, and</div> <div class="claim-text">repeating said denaturing, hybridizing, and converting steps until a desired degree of amplification is achieved.</div> </div>
    </div>
    </div> <div class="claim-dependent"> <div num="13" class="claim">
      <div class="claim-text">13. The method of claim 12, wherein the fragments are genomic fragments.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="14" class="claim">
      <div class="claim-text">14. The method of claim 12, wherein the fragments are produced by converting messenger RNA species to double-strand cDNA fragments.</div>
    </div>
  </div> </div></div></div><div class="patent-section patent-description-section"><div class="patent-section-header"><span class="patent-section-title">Description</span></div><div class="patent-text"><div mxw-id="PDES67521318" lang="EN" load-source="patent-office" class="description">
    <heading>1. FIELD OF THE INVENTION</heading> <p>The present invention relates to methods of isolating DNA fragments which are unique to one of two fragment mixtures, and in particular, to a method in which the fragments have been amplified by a novel linker/primer procedure.</p>
    <heading>2. REFERENCES</heading> <p>Anderson, M. L. M., and Young, B. D., in Nucleic Acid Hybridization: A Practical Approach, B. D. Hames and S. J. Higgins, eds., IRL Press, Oxford (1985), 73.</p>
    <p>Britten, R. J. and Davidson, E. H., in Nucleic Acid Hybridization: A Practical Approach, B. D. Hames and S. J. Higgins, eds., IRL Press, Oxford (1985), 3.</p>
    <p>Britten, R. J. and Kohne, D. E., Science, (1968), 161:529.</p>
    <p>Davidson, E., in Gene Activity in Early Development, Academic Press, N.Y. (1976).</p>
    <p>Hames, B. D. and Higgins, S. J., eds., Nucleic Acid Hybridization: A Practical Approach, IRL Press, Oxford (1985).</p>
    <p>Kauffman, R. S. and Fields, B. N., in Fundamental Virology, B. N. Fields and D. M. Knipe, eds., Raven Press, New York (1986) 161.</p>
    <p>Kohne, D. E., et al, Biochemistry (1977) 16(2):5329.</p>
    <p>Maniatis, T., et al, Molecular Cloning: A Laboratory Manual, Cold Spring Harbor Laboratory (1982), 280.</p>
    <p>Stanbury, J. B., et al, eds., The Metabolic Basis of Inherited Disease, McGraw-Hill Book Company, New York (1978).</p>
    <p>Thompson, J., et al, Anal Biochem (1987) 303:334.</p>
    <heading>3. BACKGROUND OF THE INVENTION</heading> <p>The ability to identify and isolate nucleic acid sequences which are unique to one of two cellular or body-fluid sources has important applications in medicine. One application is in understanding and predicting certain disease states, based on the presence or absence of a given messenger RNA (mRNA) species. For example, an absent or altered mRNA coding for a specific protein in a particular cell type is often the direct cause of a hereditary disease, while the presence of an added mRNA species may signal the beginning of malignant transformation or the latent presence of an otherwise undetectable infectious agent. Although some hereditary diseases--such as sickle cell anemia, other hemoglobinopathies and the thalassemias--are due to changes in the nature or presence of high-abundance mRNAs, a large percentage of hereditary diseases have been shown to be or are likely to be caused by the absence of or alterations in specific proteins coded for by low-abundance mRNAs. These include Lesch-Nyhan Syndrome, Hunter's Syndrome, Hurler's Syndrome, Tay-Sachs Disease and adenosine deaminase deficiency, among others (Stanbury).</p>
    <p>It is also known that several oncogenes exist whose aberrant activation leads to malignant transformation (Van Beverow), and the detection of changes in mRNA species will have important applications in the early detection of such transformation.</p>
    <p>Another application of unique-sequence isolation methods is in the diagnosis and study of viral or other cell or fluid borne agents. For example, the transcription of virus-specific mRNA(s) may be the first indication of reactivation of a latent viral infection (Kauffman). The method may also be used to isolate and identify viral agents.</p>
    <p>Methods of isolating unique genomic sequences have applications to the study of gene expression during cell activation, embryonic development or cell cycle progression, and for analyzing genetic diseases which are related to the presence or absence of disease-related genomic regions.</p>
    <p>One major problem in the detection and isolation of unique mRNAs, complementary DNAs (cDNAs), or genomic fragments, is interference caused by numerous high-abundance species present in the source material. For example, in any given cell type, there may be 10,000-30,000 distinct mRNA species (Davidson), and these can range in concentration from several hundred thousand molecules per cell, for high-abundance species, to only a few molecules per cell, for low-abundance species. Thus, if the unique species of interest has a relatively low-abundance, the ratio of unique species may be as low as 1 in 10<sup>6</sup>.</p>
    <p>A number of nucleic acid hybridization techniques aimed at isolating and analyzing unique nucleic acid species have been developed heretofore. These techniques rely on the ability of denatured nucleic acids to reassociate in a sequence specific manner by Watson-Crick base pairing interactions (Britten 1968, 1985). The rate at which renaturation occurs is determined by the sequence complexity of the sample, the absolute and relative concentrations of various species, the length of DNA fragments, and the conditions under which the reaction takes place (Hames).</p>
    <p>In one hybridization method, a selected gene probe cDNA is labeled and added in single-strand form to a saturating mixture of mRNA transcripts. The presence of the probe-related transcript species can be assessed by the amount of labeled probe cDNA incorporated into double-strand material. This method is generally suitable for high- and moderate-abundance transcripts, but lacks the sensitivity required for the identification and analysis of low-abundance mRNAs due to high background levels.</p>
    <p>Filter hybridization to a conventional cDNA library may be used to identify differences in low-abundance mRNAs from two different sources (Anderson). However, the one million or more library clones needed to insure the presence of virtually all low-abundance mRNAs effectively eliminates the utility of this technique for screening programs.</p>
    <p>Nucleic acid subtraction techniques have also been proposed for use in studying differences in low-abundance mRNAs between related cell types (Maniatis). Here mRNA or cDNA preparations from two different sources are hybridized to completion, and the unhybridized remainder is examined for the presence of species of interest. This method, as it has been practiced heretofore, requires relatively large amounts of mRNA and/or cDNA starting material, especially where the species of interest is a relatively low-abundance mRNA. This limitation precludes the use of the method for many body-fluid or cellular samples of interest where the total concentration of unique species is low.</p>
    <heading>4. SUMMARY OF THE INVENTION</heading> <p>It is therefore one object of the invention to provide, for isolating RNA or genomic sequences which are unique to one of two mixtures, a method which overcomes many of the above-mentioned problems associated with the prior art.</p>
    <p>A more specific object of the invention is to provide such a method which allows identification of unique species which are present at low concentration and/or low molar ratio in a mixture of genomic or RNA-derived duplex DNA fragments.</p>
    <p>Another specific object of the invention is to provide such a method in which the degree of resolution and sensitivity can be increased as needed by repeated fragment amplification and/or selection steps.</p>
    <p>Still another object of the invention is to provide a novel linker/primer DNA amplification procedure which is used in the method for unique-sequence isolation.</p>
    <p>The invention includes, in one aspect, a method of isolating duplex DNA fragments which are unique to one of two fragment mixtures, i.e., fragments which are present in a mixture of duplex DNA fragments derived from a positive source, but absent from a fragment mixture derived from a negative source. In practicing the method, double-strand linkers are attached to each of the fragment mixtures, and the number of fragments in each mixture is amplified by successively repeating the steps of (i) denaturing the fragments to produce single fragment strands, (ii) hybridizing the single strands with a primer whose sequence is complementary to the linker region at one end of each strand, to form strand/primer complexes and (iii) converting the strand/primer complexes to double-strand fragments in the presence of polymerase and deoxynucleotides. After the desired fragment amplification is achieved, the two fragment mixtures are denatured, then hybridized under conditions in which the linker regions associated with the two mixtures do not hybridize. DNA species which are unique to the positive-source mixture, i.e., which are not hybridized with DNA fragment strands from the negative-source mixture, are then selectively isolated.</p>
    <p>The fragment mixtures may be obtained from genomic DNA or derived from messenger RNA preparations. In one preferred embodiment, the fragments in the two mixtures are each blunt ended, then ligated with separate blunt-end primers. The linker attached to the positive-source fragment has an internal restriction site at which the fragment can be cut by a selected restriction endonuclease, allowing unique fragments which are isolated by the method to be cloned by insertion at a suitable vector cloning site.</p>
    <p>Also in a preferred embodiment, the negative-source fragments are biotinylated, prior to hybridizing, and are present in large molar excess in the hybridization reaction. The negative-source fragments preferably contain an internal sticky-end restriction site at which the fragment can be cleaved, and biotinylated by filling with a polymerase in the presence of biotinylated nucleotides.</p>
    <p>The species isolated in the above method may be further amplified and selected, to enhance enrichment of the unique sequence(s).</p>
    <p>Also forming part of the invention is a method for amplifying duplex DNA fragments. The method includes attaching a double-strand linker to the fragments, and replicating the two strands in each fragment by denaturing the fragments to produce single fragment strands with linker-strand ends, hybridizing the single strands with a primer whose sequence is complementary to a linker-strand end on each fragment strand, to form strand/primer complexes, and converting the strand/primer complexes to double-strand fragments in the presence of polymerase and deoxynucleotides. The fragment-replication steps are repeated until a desired degree of amplification is achieved.</p>
    <p>These and other objects and features of the invention will become more fully appreciated when the following detailed description of the invention is read in conjunction with the accompanying drawings.</p>
    <heading>BRIEF DESCRIPTION OF THE DRAWINGS</heading> <p>FIG. 1 is a flow diagram of the linker/primer duplex amplification method of the invention;</p>
    <p>FIGS. 2A-2C shows the sequences of three exemplary linkers used in practicing the invention, after attachment to opposite ends of blunt-ended duplex fragments (solid lines);</p>
    <p>FIG. 3 illustrates one method for biotinylating amplified cDNA fragments, in practicing the invention;</p>
    <p>FIG. 4 is a flow diagram of the method of the invention for isolating DNA fragments which are unique to one fragment source;</p>
    <p>FIG. 5 illustrates the isolation method shown in FIG. 4 applied to isolation of an RNA-derived sequence which is unique to individuals infected with a given viral agent; and</p>
    <p>FIG. 6 illustrates the amplification method of FIG. 1, as it is used for confirming the presence of ET-NANB viral agent in bile from an infected animal, and for assaying the presence of the viral agent in infected individuals.</p>
    <heading>DETAILED DESCRIPTION OF THE INVENTION</heading> <p>I. Amplifying DNA Fragments</p>
    <p>A. Sources of DNA Fragments</p>
    <p>FIG. 1 illustrates the method of amplifying duplex DNA fragments according to the invention. The duplex fragments are typically present in a fragment mixture, and typically a mixture of cDNA fragments produced from messenger RNA (mRNA) transcript species, although genomic DNA fragments or linearized vectors or vector fragments are also suitable.</p>
    <p>Methods for isolating mRNA species from cellular or body-fluid samples, such as serum or bile, are well known. One method involves formation of a vanadyl-RNA complex, extraction of protein with chloroform/phenol, and precipitation with cold ethanol. In a second method, the RNA is extracted from a guanidium isothiocyanate mixture with phenol, followed by a chloroform:isoamyl alcohol extraction, and precipitation of RNA from the aqueous phase with cold ethanol. The reader is referred to Maniatis, pp. 188-198, and references cited therein for details.</p>
    <p>As a rule, eukaryotic mRNAs are characterized by a poly A termination sequence which allows isolation by affinity chromatography, using oligo dT bound to a solid support. In addition, or alternatively, total isolated RNA can be further fractionated by density gradient centrifugation, or agarose gel electrophoresis, to obtain a desired size fraction of RNA species.</p>
    <p>Production of duplex cDNAs from the isolated mRNA transcripts is by conventional oligo dT or random priming of first strand synthesis. The former method is advantageous where duplex cDNAs derived only from poly A RNAs are desired. In this method, the RNA transcripts are primed with oligo dT, to promote first-strand cDNA synthesis by reverse transcriptase in the presence of all four deoxynucleotides. The 3' hairpin formed from the first-strand synthesis product is used to prime the synthesis of the second strand by E. coli DNA polymerase I, according to well-known methods (Maniatis, pp. 213-216).</p>
    <p>The random priming method of cDNA duplex formation is preferred where (a) duplex formation is not to be limited to poly A species, or (b) full-length duplex cDNAs are not required. The method for first-strand cDNA synthesis utilizes an arbitrary sequence primer which is commercially available.</p>
    <p>Following second-strand synthesis, the cDNA fragments are preferably blunt-ended, for example, by filling in sticky ends with the large fragment of E. coli DNA polymerase I (Klenow fragment) as described below in the Materials section.</p>
    <p>Genomic DNA fragments, either in a fragment mixture or a purified preparation, are also suitable for amplification, according to the method of the invention. Genomic DNA from a selected cell source can be isolated by standard procedures, which typically include successive phenol and phenol/chloroform extractions with ethanol precipitation. The isolated DNA may be further fractionated to yield chromosomes or chromosomal regions of interest. The duplex DNA is fragmented preferably by partial or complete digestion with one or more selected restriction endonucleases, although mechanical shearing may be employed. The fragmented genomic pieces may be size fractionated, or further treated to remove repetitive DNA. Other sources of double-stranded DNA fragments can include extrachromosomal material, e.g., mitochondrial DNA, double-stranded DNA viruses, or viruses which have as part of their life cycle a double-stranded intermediate, e.g., a retrovirus.</p>
    <p>Cellular sources of genomic DNA fragments or RNA transcripts used for producing cDNA fragments include cultured cell lines, or isolated cells or cell types obtained from tissue (or whole organs or entire organisms). Cell sources are of interest in a variety of subtraction techniques where it is desired to identify or isolate particular RNA transcripts or genomic material which are unique to one of two related cell sources. Body-fluid sources of DNA and/or mRNA transcripts are of interest primarily where the fluid is known or suspected to contain a viral agent or other microbe of interest. Example 3, for instance, describes cDNA fragment mixtures produced from RNA isolated from bile, taken before and after infection of cynomolgus monkeys with enterically transmitted non-A/non-B (ET-NANB) hepatitis virus.</p>
    <p>Linearized or fragmented plasmid DNA, or fragmented phage DNA is another source of DNA fragments which one might wish to amplify. The vector DNA is obtained from purified plasmid or phage DNA according to conventional techniques, and linearized and/or fragmented by digestion with selected restriction endonuclease(s). Example 1 describes DNA fragments obtained by HaeIII digestion of phiX174 phage and linearized piAN13 plasmid.</p>
    <p>B. Fragment Linkers</p>
    <p>According to an important feature of the invention, the duplex DNA fragments to be amplified are ligated at their opposite end to a linker, to provide a priming sequence for strand duplication. The linker is preferably a short duplex DNA fragment, typically about 20-30 basepairs in length, having a defined basepair sequence. One end of the linker is designed for enzymatic ligation to opposite fragment ends. Where the fragments are blunt-ended cDNAs or genomic fragments, this linker end is also blunt, for ligation to the fragment ends in the presence of T4 DNA ligase, as described in Example 1. Where the DNA fragment mixture is formed by endonuclease digestion of genomic DNA, and produces fragments with sticky ends, the linkers can be prepared with complementary sticky ends at the fragment-ligation end.</p>
    <p>Since the ligation reaction involving the linkers also produces linker-to-linker ligation, the linkers are preferably designed so that dimers can be selectively cleaved by restriction endonuclease digestion. This can be done by constructing the blunt end of the linker with a sequence representing one-half of a selected restriction-site sequence. The restriction site formed by blunt-end dimerization is preferably a rare cutter site, so that digestion of the fragments with the associated endonuclease, after linker addition to the DNA fragments, does not cleave a significant number of fragments internally, and thereby produce fragments with a linker at one end only.</p>
    <p>FIG. 2 shows DNA fragments (solid lines) with exemplary linkers A, B, or C attached to opposite fragment ends. As seen, each of the three linkers has one-half of an NruI site at its blunt end, whereby linker dimers formed by blunt end ligation can be cleaved by NruI digestion. It should be noted that linker ligation to the DNA fragments is directional, and linker self-ligation is limited to dimer formation, due to the one staggered end of the linker. The linkers are joined to blunt-ended fragments as indicated above, and as illustrated at the top in FIG. 1. It will be appreciated that analogous linkers with one sticky end for ligation to corresponding fragment sticky ends could be similarly employed.</p>
    <p>Also as seen in FIG. 2, the linkers preferably include one or more internal restriction sites at which the fragments can be cleaved after amplification. The internal linker restriction site(s) can serve three purposes. First, where the amplified fragments are used in the DNA subtraction method described in Section II below, the restriction site can be used to cleave away a major portion of the linker in one of the two fragment mixtures, to prevent hybridization between linker regions in strands from the two different mixtures, where the same linker is used in the amplification of both fragment mixtures. Secondly, the linker cutting site(s) allows the amplified fragments (or the hybridized fragments in the Section II subtraction method), to be equipped with desired sticky end sites for cloning into a cloning vector. Thirdly, as will be seen in Section II, the linker restriction site may be used to create a single-strand end, for purposes of biotinylating the fragments. In particular, an internal EcoRI site produces an AATT overhang which can be filled in with commercially available biotinylated dUTP and dATP nucleotides.</p>
    <p>Synthetic duplex oligonucleotide linkers having selected sequences, such as those shown for the three exemplary linkers in FIG. 2, can be prepared using commercially available automated oligonucleotide synthesizers. Alternatively, custom designed synthetic oligonucleotides may be purchased, for example, from Synthetic Genetics (San Diego, Calif.).</p>
    <p>C. Fragment Amplification</p>
    <p>The method steps described above are shown at the top in FIG. 1. These steps include obtaining duplex fragments, blunt-ending and attaching a linker to opposite fragments ends to produce linker-carrying fragments, such as illustrated in FIG. 2, and treating with a selected endonuclease to cut linker multimers at their blunt-end junctions.</p>
    <p>The linker-fragments from above are amplified by the repeated fragment duplication according to the following steps. First, the fragments are mixed with a large molar excess of a single-strand oligonucleotide primer, typically a 10<sup>6</sup> -10<sup>9</sup> molar excess. The primer sequence is homologous to the fragment end linker; that is, the primer sequence and length is such as to promote hybridization to the complementary-linker regions of the two strands under moderately stringent reannealing conditions. Another requirement is that the primer, when hybridized to the linker region of the denatured fragment strands, be capable of priming polymerase-catalyzed strand replication; that is, the internal end of the primer provides a free 3'-OH. In the case of the linkers shown in FIG. 2, preferred primer sequences are:</p>
    <p>d(5'-GGAATTCGCGGCCGCTCG-3') for linker A;</p>
    <p>d(5'-GGTTGCGGCCGCTCG-3') for linker B; and</p>
    <p>d(5'-GACGCGTGAATTCTCG-3") for linker C.</p>
    <p>Single-strand oligonucleotide primers having the desired sequence are prepared by conventional methods or can be obtained from commercial laboratories, as above.</p>
    <p>The denatured fragments and primer are cooled, typically between about 37-60° C., to allow primer hybridization with the fragment linker ends. The cooling period is kept quite short, typically less than about 5 minutes, to minimize strand-strand hybridization which would be expected on long-term reannealing. At this point, or at the primer-addition stage, the four deoxynucleotides and, optionally a DNA polymerase capable of catalyzing second-strand, primed replication are added, and the reaction mixture is brought to a temperature suitable for enzymatic strand replication.</p>
    <p>In the method described in Example 1 below, the deoxynucleotides and DNA polymerase were added prior to fragment denaturation. After heat denaturing, the mixture was annealed at 50° C. for two minutes, then brought to 72° C. for 5-12 minutes for primed, second-strand replication. The DNA polymerase used is Thermus aquatics DNA polymerase (Taq DNA polymerase), which is relatively heat-stable at up to 95° C. for brief periods.</p>
    <p>It will be appreciated from the above, and from FIG. 1, that the single added primer hybridizes with the 3'-end linker region in each DNA strand, and thus a doubling of fragment number occurs. The above replication procedure, which involves fragment denaturation by heating, cooling to form a fragment strand-primer complex, and second strand replication of the complex in the presence of DNA polymerase, is repeated until a desired concentration of fragments is achieved. In the above example, which employs a heat-stable Taq polymerase, the three replication steps are carried out simply by heating the fragment mixture to a denaturing temperature (above the T<sub>m</sub> of the fragments), cooling briefly to allow fragment/primer complex formation, and incubating for a period sufficient for second-strand synthesis.</p>
    <p>Since the concentration of fragments doubles at each round of replication, a 10<sup>3</sup> fold amplification can be achieved with about 10 rounds of replication, and a 10<sup>6</sup> fold amplification with about 20 rounds of replication. Thus, starting picogram amounts of DNA fragments will yield microgram amounts of fragment material after about 20 rounds of replication.</p>
    <p>Example 1 illustrates the application of the fragment amplification method to a linearized DNA plasmid (piAN13) and to a fragment mixture produced by HaeIII digestion of phage phiX174 fragments. Example 3 illustrates the application of the method to cDNA fragment mixtures derived from a mixture of mRNAs isolated from a bile sample.</p>
    <p>D. Utility</p>
    <p>The amplification method is useful for DNA duplex fragment amplification where limited amounts of genomic or cDNA fragment material is available and/or as a simple method of amplifying duplex DNA material where DNA sequences are unknown. For example, in the probe binding study reported in Example 3, the method was used to increase by several orders of magnitude, the number of cDNA fragments derived from bile infected with enterically-transmitted non-A/non-B hepatitis (ET-NANB) viral agent. The amplification allowed relatively small amount of viral agent present in the bile to be easily identified as an amplified cDNA fraction.</p>
    <p>The method is useful for amplifying the amount of a fragment or fragments available for cloning, to enhance cloning efficiency. One advantageous feature of the method for cloning is the ability to provide the amplified fragments with end cloning sites suitable for the cloning vector which has been selected.</p>
    <p>Another application, described in Section II below, involves amplifying two fragment mixtures, for use in DNA hybridization and subtraction, for isolating fragments which are unique to one of the two fragment mixtures.</p>
    <p>The method can be applied to purified, enriched, or subfractionated DNA fragments, for increasing the amount of fragment material available for further processing, e.g., further enrichment. For example, in the fragment-isolation method described in Section II, the isolated unique sequence fragments can be further enriched by additional amplification and selection. As another example, a DNA subfraction obtained by gel electrophoresis can be selectively amplified. Here the fragment mixture, prior to fractionation, is equipped with an end linker, and the selected fragment band is further isolated, either in situ within the gel, or after selective elution, by addition of primer, polymerase, and nucleotides, with repeated heating and cooling replication steps.</p>
    <p>II. Isolating Unique Sequences</p>
    <p>A. Hybridizing Fragment Mixtures</p>
    <p>FIG. 4 is a flow diagram of the method for isolating unique sequences from one of two fragment mixtures, according to the invention. The two fragment mixtures are referred to herein as positive- and negative-source mixtures, and are derived from positive and negative sources of nucleic acid species, respectively. The positive source may be any cell, cell group, tissue, organ, subcellular fractions, or other defined source of RNA transcripts or genomic fragments, including vector fragments, which contain a number of different-sequence species. The negative source is similar to the positive source, but lacks one or more positive source species.</p>
    <p>By way of example, the positive and negative sources may be cell or tissue samples which are infected (positive source) and non-infected (negative source) with a microbial agent. Here the nucleic acid species associated with the infecting agent, either in the form of genomic material or RNA transcript species, are unique to the infected (positive) source. Similarly, the positive and negative sources may be a body fluid, such as serum or bile, from infected and non-infected subjects, respectively.</p>
    <p>Where the present method is applied to the study of hereditary diseases, the positive source is typically a normal cell line or tissue capable of producing the full range of normal proteins associated with that cell, and the negative source is a genetically altered cell which is suspected of an altered or lost mRNA species. That is, the positive source will yield a normal mRNA species which is not present in the genetically altered, negative source.</p>
    <p>As another example, it is known that malignant transformation in cells may be triggered by the activation of oncogenes, as evidenced by expression of new RNA transcripts associated with the malignant state. In applying the method of the invention to isolating and identifying such RNA species, the malignant cells or tissue serves as the positive source of unique (tumor-related) RNA species, and normal, non-transformed cells or tissue, as the negative source. Other examples of negative and positive sources of transcripts or DNA duplex fragments are contemplated.</p>
    <p>The positive and negative sources shown in FIG. 4 yield mixtures of mRNA species, and these are converted to corresponding duplex DNA fragments as discussed in Section IA above. The duplex fragments in the two mixtures are blunt-ended and separately ligated at opposite fragment ends to a linker, as described in Section IB. The linker attached to the positive-source fragment is preferably different than that attached to the negative-source fragments. For example, in the method outlined in Example 1, the positive source fragments are ligated to linker A in FIG. 2, and the negative-source material, with linker C. Attaching linkers with different sequences to the two fragment mixtures avoids the problem of linker hybridization when the two mixtures are hybridized. Alternatively, the same linker may be attached to both fragment mixtures, if the linker contains a restriction site which allows substantially all of the linker to be removed by restriction endonuclease digestion from the negative-source fragment mixture before hybridization.</p>
    <p>Each fragment mixture is then amplified by repeated replications, as described in Section IC above. In a preferred embodiment, the amplified, negative-source fragment strands are biotinylated or otherwise equipped with a ligand which can be used to selectively bind and remove strands or duplex species containing the ligand, by affinity chromatography. A variety of methods for biotinylating duplex fragments are available. For example, biotin-labeled deoxynucleotides, such as Bio-11-UTP and Bio-7-dATP, can be included in the amplification reaction mixture during the final rounds of replication. Duplex fragments are readily photobiotinylated by a standard procedure, as outlined in the protocol supplied by the manufacturer, Clontech (Palo Alto, Calif.). In one preferred method, the amplified fragments are treated with an endonuclease, such as EcoRI, which produces a 5'-AATT-3' sticky end. The sticky end is filled in with Klenow fragment in the presence of Bio-11-UTP and/or Bio-7-dATP, both of which are commercially available.</p>
    <p>The two fragment mixtures are now combined and heat-denatured to single strand form. Preferably the negative-source fragments are present in substantial molar excess, typically about a tenfold excess, to insure that non-unique fragments in the positive-source mixture have a high probability of hybridizing with complementary strands from the negative-source fragments.</p>
    <p>The positive- and negative-source fragments are hybridized under conditions in which hybridization between the linkers in the two different mixtures does not occur. This may be accomplished, as noted above, by using different-sequence linkers in the two mixtures, by cleaving the linkers from the negative-source fragment mixture prior to hybridization, or by hybridizing the fragment strands under conditions which preclude linker hybridization. Alternatively, the hybridization mixture may contain a large molar excess of primer oligonucleotides effective to hybridize with the positive-source linker end regions, thus preventing interstrand linker hybridization.</p>
    <p>The hybridization reaction may be carried out according to classical hybridization techniques, such as detailed in Hames, or more preferably, by rapid hybridization techniques, such as the phenol emulsion reassociation technique (PERT), as described by Kohne, or the guanidinium thiocyanate method, as described by Thompson.</p>
    <p>In the PERT hybridization method, the fragments are hybridized at a final DNA concentration of preferably between about 5-50 ug/ml in a phenol reaction mixture, and the reaction is carried out at 60-70° C. for up to 5-6 hours. The rate of annealing can be followed by hyperchromic shift at 280 nm or by diluting an aliquot of the reaction mixture with buffer, and quickly passing the material through a hydroxyapatite (HAP) column (Kohne).</p>
    <p>In the guanidine thiocyanate method, the fragments are hybridized at a final DNA concentration of preferably between about 5-50 ug/ml in a 4M guanidine thiocyanate solution containing 8 mM DTT, and 20 mM Na citrate, pH 5.8. The mixture is heated to 65° C. for 5 minutes, to insure complete denaturation, then incubated at 26° C. for periods up to 72 hours, until complete hybridization has occurred (Thompson). The formation of DNA/DNA hybrids can be confirmed as above.</p>
    <p>B. Isolating Unique-Sequence Fragments</p>
    <p>Following hybridization, the reaction components are contacted with a solid support effective to preferentially bind non-unique fragments. In a preferred embodiment, where the negative-source strands are biotinylated, the reaction components are separated on an avidin or streptavidin column which is effective to bind all biotinylated strands and non-biotinylated (positive-source) strands which are hybridized therewith. Since the biotinylated strands are present in large molar excess, virtually all common-sequence strands from the positive-source will be bound to a complementary, biotinylated strand, and thus removed by the affinity chromatography.</p>
    <p>Methods for separating the hybridization mixture on a streptavidin column are detailed in Example 2. Briefly, the hybridization mixture is passed through a column packed with streptavidin-agarose and washed extensively with elution buffer. The eluates from several washes are pooled and the nucleic acid is concentrated by ethanol precipitation, yielding a fragment mixture which is enriched for sequences which are unique to the positive-source material.</p>
    <p>According to an important feature of the invention, the fragments isolated will contain the end linkers originally attached to the positive-source fragments, and thus can be further amplified, by the amplification method described in Section I. Where the positive-source linker is different than that on the negative-source fragments, additional amplification (employing the "positive-source" primer) will selectively amplify positive-source fragments, providing further enrichment of sequences which are unique to the positive source. As indicated in FIG. 4, the isolated amplified material may be rehybridized with the negative source fragments (which may be additionally amplified if necessary), and unique fragments again isolated by affinity chromatography, to enhance enrichment.</p>
    <p>The final isolated fragments can be treated with DNA polymerase to ensure complete strand replication in the duplex species, then treated with a selected linker-site endonuclease, to provide fragment ends suitable for cloning or the like. Alternatively, or in addition, the isolated fragments can be radiolabeled according to known procedures, for use as a DNA probe.</p>
    <p>C. Utility</p>
    <p>FIG. 5 illustrates one application of the subtraction method, for isolation and cloning of a viral agent present in RNA transcript in an infected source, such as selected tissue, or a body fluid, such as bile. Here RNA mixtures isolated from infected (positive) and non-infected (negative) sources are used to produce cDNA duplex fragment mixtures, which are blunt-ended, ligated to a selected linker, and amplified (Section I). After biotinylating the negative-source fragments, the two fragment mixtures are combined, denatured, reannealed, and separated by affinity chromatography (Section II). The isolated fragments, which contain a selected restriction site in their end-linker regions, are cut at this site and cloned in a suitable expression vector, such as the lambda gt11 vector. The resulting lambda gt11 phage constitute a fragment library enriched for sequences unique to the positive source material.</p>
    <p>To select for desired isolates which produce a viral antigen, host cells are infected with library phage, and reacted with antiserum from an infected individual, to bind virus-related antibodies to host cells producing viral antigens. The cells are then washed to remove unbound antibody and contacted with a reporter-labeled anti-human IgG antibody, to label cells which have bound virus-specific antibodies. These cells are identified by the presence of the reporter. The vector(s) containing a viral insert can be used for continued antigen production, or as a source viral sequence probe.</p>
    <p>Alternatively, the cloned isolated fragments in the fragment library can be identified and selected using DNA probes. By way of illustration, the fragment library generated as above can be replica plated and hybridized with labeled cDNA probes from infected and non-infected sources, prepared as in Section I. Probe hybridization can be carried out by conventional Southern blotting. Those library vectors which contain a viral-agent insert will hybridize with viral-agent cDNA probes present in the positive-source cDNA probes, but not with negative-source probes.</p>
    <p>Methods similar to the DNA subtraction and probe selection method just described can be applied to identifying and isolating nucleic acid fragments which are unique to cells expressing oncogenes, or cells deficient in transcripts related to genetic disease.</p>
    <p>FIG. 6 illustrates how the amplification technique described in Section I can be used to confirm that the unique-sequence fragment clone(s) identified as above are in fact unique to the positive source. Here the two cDNA fragment mixtures prepared from positive and negative sources are amplified, as in Section I, and fractionated by gel electrophoresis. The size fractionated gel fragments are then transferred by conventional Southern blotting to filters and hybridized with the isolated cloned insert of interest, after probe labeling. If the probe sequence is unique to the positive source fragments, hybridization will be seen with amplified positive-source fragments only, as indicated in FIG. 6. As indicated in FIG. 6, the region of probe binding is fairly broad, as would be expected since the amplified fragments will have a range of sizes. This method of screening amplified cDNA fragments is detailed in Example 3, where the method has been used to confirm that an isolated ET-NANB fragment insert was unique to cDNA from an infected bile source.</p>
    <p>The following examples illustrate the method of fragment amplification and fragment isolation described above, but are in no way intended to limit the scope of the method or its applications.</p>
    <heading>Materials</heading> <p>Plasmid piAN13 was obtained from Dr. Brian Seed (MIT). E. coli strain KM392, a suppressor positive strain, was obtained from Dr. Kevin Moore, DNAX, Palo Alto, Calif. E. coli strain LG75, a suppressor minus strain, was obtained from Dr. Brian Seed (MIT). Lambda phage Ch21 was obtained from Dr. Frederick Blattner (University of Wisconsin), and a NotI site was inserted between the EcoRI and HindIII sites (Ch21LJ), by replacing the 2 kb EcoRI/HindIII fragment with an oligonucleotide containing a NotI site. E. coli strain MC1061 harboring the P3 plasmid was obtained from Dr. Brian Seed (MIT).</p>
    <p>Terminal transferase (calf thymus), alkaline phosphatase (calf intestine), polynucleotide kinase, E. coli DNA polymerase I (Klenow fragment), and S1 nuclease were obtained from Boehringer Mannheim Biochemicals (Indianapolis, Ind.); phiX174 fragments, produced by digesting the phiX174 phage to completion with HaeIII, were obtained from BRL Laboratories (Bethesda, Md.).</p>
    <p>SmaI, EcoRI, NotI, T4 DNA ligase and T4 DNA polymerase were obtained from New England Biolabs (Beverly, Mass.); and streptavidin agarose, from Bethesda Research Labs (Bethesda, Md.). Low-gelling temperature agarose (Sea Plaque) was obtained from FMC (Rockland, Me.). Nitrocellulose filters were obtained from Schleicher and Schuell.</p>
    <p>Synthetic oligonucleotide linkers and primers were prepared using commercially available automated oligonucleotide synthesizers. Alternatively, custom designed synthetic oligonucleotides may be purchased, for example, from Synthetic Genetics (San Diego, Calif.). cDNA synthesis kit and random priming labeling kits were obtained from Boehringer-Mannheim Biochemical (BMB, Indianapolis, Ind.).</p>
    <p>Kinasing of single strands prior to annealing or for labeling is achieved using an excess, e.g., approximately 10 units of polynucleotide kinase to 1 nmole substrate in the presence of 50 mM Tris, pH 7.6, 10 mM MgCl<sub>2</sub>, 5 mM dithiothreitol, 1-2 mM ATP, 1.7 pmoles gamma-<sup>32</sup> p-ATP (2.9 mCi/mmole), 0.1 mM spermidine, 0.1 mM EDTA.</p>
    <p>Site specific DNA cleavage is performed by treating with the suitable restriction enzyme (or enzymes) under conditions which are generally understood in the art, and the particulars of which are specified by the manufacturer of these commercially available restriction enzymes. See, e.g., New England Biolabs, Product Catalog. In general, about 1 ug of plasmid or DNA sequence is cleaved by one unit of enzyme in about 20 ul of buffer solution after 1 hr digestion at 37° C.; in the examples herein, typically, an excess of restriction enzyme is used to insure complete digestion of the DNA substrate.</p>
    <p>Incubation times of about one hour to two hours at about 37° C. are workable, although variations can be easily tolerated. After each incubation, protein is removed by extraction with phenol/chloroform, and may be followed by ether extraction, and the nucleic acid recovered from aqueous fractions by precipitation with ethanol (70%). If desired, size separation of the cleaved fragments may be performed by polyacrylamide gel or agarose gel electrophoresis using standard techniques. A general description of size separations is found in Methods in Enzymology (1980) 65:499-560.</p>
    <p>Restriction cleaved fragments may be blunt ended by treating with the large fragment of E. coli DNA polymerase I (Klenow reagent) in the presence of the four deoxynucleotide triphosphates (dNTPs) using incubation times of about 15 to 25 min at 20° to 25° C. in 50 mM Tris pH 7.6, 50 mM NaCl, 6 mM MgCl<sub>2</sub>, 6 mM DTT and 0.1-1.0 mM dNTPs. The Klenow fragment fills in at 5' single-stranded overhangs in the presence of the four nucleotides. If desired, selective repair can be performed by supplying only one of the, or selected, dNTPs within the limitations dictated by the nature of the overhang. After treatment with Klenow reagent, the mixture is extracted with phenol/chloroform and ethanol precipitated. Treatment under appropriate conditions with S1 nuclease results in hydrolysis of any single-stranded portions of DNA. In particular, the nicking of 5' hairpins formed on synthesis of cDNA is achieved.</p>
    <p>Ligations are performed in 15-50 ul volumes under the following standard conditions and temperatures: for example, 20 mM Tris-Cl pH 7.5, 10 mM MgCl<sub>2</sub>, 10 mM DTT, 33 mg/ml BSA, 10 mM-50 mM NaCl, and either 40 mM ATP, 0.01-0.02 (Weiss) units T4 DNA ligase at 14° C. (for "sticky end" ligation) or 1 mM ATP, 0.3-0.6 (Weiss) units T4 DNA ligase at 14° C. (for "blunt end" ligation). Intermolecular "sticky end" ligations are usually performed at 33-100 mg/ml total DNA concentrations (5-100 nM total end concentration). Intermolecular blunt end ligations are performed at 1 mM total ends concentration.</p>
    <p>In vector constructions employing "vector fragments", the vector fragment is commonly treated with bacterial alkaline phosphatase (BAP) or calf intestinal alkaline phosphatase (CIP) in order to remove the 5' phosphate and prevent self-ligation of the vector. Digestions are conducted at pH 8 in approximately 10 mM Tris-HCl, 1 mM EDTA using about 1 unit per mg of BAP at 60° C. for one hour or 1 unit or CIP per mg of vector at 37° C. for about one hour. In order to recover the nucleic acid fragments, the preparation is extracted with phenol/chloroform and ethanol precipitated. Alternatively, religation can be prevented in vectors which have been double digested by additional restriction enzyme digestion and separation of the unwanted fragments.</p>
    <heading>EXAMPLE 1</heading> <heading>Amplifying Duplex DNA Fragments</heading> <p>A. Modification and Amplification of piAN13 plasmid</p>
    <p>Plasmid piAN13 carries a supF suppressor gene capable of suppressing an amber mutation. The plasmid was linearized by digestion with SmaI and ligated to the linker whose sequence is shown at A in FIG. 2, at a 1:100 molar ratio in the presence of 0.3-0.6 Weiss units of T4 DNA ligase. The linker sticky ends were filled in with Klenow fragment, conventionally, and the mixture was treated with NruI to cleave linker dimers. SmaI linkers were attached to the blunt-ends of the plasmid fragments under ligation conditions like those above. After treatment with SmaI to remove redundant SmaI linkers, the fragments were recircularized. This was done at a fragment concentration of about 1 ug/ml, in the presence of 4,000 units/ml DNA ligase, at about 14° C. for 18 hours. The resulting plasmids were thus modified to contain an A-sequence linker on either side of the original SmaI site.</p>
    <p>E. coli strain MC1061 harboring the P3 plasmid was transformed with the recircularized plasmid, and selection for successful plasmids was made on the basis of bacterial growth in the presence of kanamycin, ampicillin, and tetracycline. Plasmids were isolated from the resistant bacterial colonies, and cut with SmaI to produce blunt-end, linearized fragments having the desired A-sequence end linkers.</p>
    <p>To 100 ul of 10 mM Tris-Cl buffer, pH 8.3, containing 1.5 mM MgCl<sub>2</sub> (Buffer A) was added 1×10<sup>-3</sup> ug of the linearized plasmid, 2 uM of a primer having the sequence d(5'-GGAATTCGCGGCCGCTCG-3'), 200 uM each of dATP, dCTP, dGTP, and dTTP, and 5 units of Thermus aquaficus DNA polymerase (Taq polymerase). The reaction mixture was heated to 94° C. for 1 minute for denaturation, allowed to cool to 50° C. for 2 min for primer annealing, and then heated to 72° C. for 5-12 min to allow for primer extension by Taq polymerase. The replication reaction, involving successive heating, cooling, and polymerase reaction, was repeated an additional 25 times with the aid of a Perkin Elmer Cetus DNA thermal cycler.</p>
    <p>B. Amplification of phiX174 Fragments</p>
    <p>To a 50 ul solution containing 100 ng of the HaeIII blunt-end fragments was added 300 ng of the linker whose sequence is shown at C in FIG. 2, with ligation of the linker in the presence of DNA ligase being carried out as above. The mixture was treated with NruI to cleave linker dimers.</p>
    <p>To 100 ul of Buffer A was added 1 ng of the phiX174/HaeIII fragments, 2 uM of a primer having the sequence d(5'-CACGCGTGAATTCTCG-3'), 200 uM each of dATP, dCTP, dGTP, and dTTP and 5 units of Taq DNA polymerase. The reaction mixture was heated to 94° C. for 1 minute for denaturation, allowed to cool to 50° C. for 2 min for primer annealing, and then heated to 72° C. for 5-12 min to allow for primer extension by Taq polymerase. The replication reaction steps were repeated 25 times, as above. The amplified fragments are referred to below as phiX/linker-C fragments.</p>
    <p>A second preparation of phiX174 fragments was amplified by similar methods, but using the sequence-A linker employed in the piAN13 amplification. This preparation is referred to below as phiX/linker-A fragments.</p>
    <heading>EXAMPLE 2</heading> <heading>Selection for piAN13 Sequences</heading> <p>A. Biotinylation of Amplified phiX174 Fragments</p>
    <p>The amplified phiX174/linker-C fragment produced in Example 1B are treated with EcoRI to yield 5' protruding ends having an AATT sequence overhang. The fragments are biotinylated by Klenow fill-in reaction according to known procedures (Maniatis, pp. 113). Briefly, the reaction mixture consists of 100 ug/ml fragments in 100 mM potassium phosphate (pH 7.2), 2 mM CoCl<sub>2</sub>, 0.2 mM DTT, 40 uM Bio-7-dATP, 100 uM dTTP, and 100 units/ml Klenow fragment. After incubation at 25° C. for 45 minutes, the reaction is terminated by heating at 65° C. for 10 minutes.</p>
    <p>B. Hybridization Reaction</p>
    <p>Amplified piAN13 vector is mixed at various mole ratios with the phiX/Linker-A fragments produced in Example 1B. Mixtures contained weight ratios of 1:100, 1:30 and 1:3 piAN13 to phiX/Linker-A fragments are prepared. Each mixture is then mixed with a 10-fold molar excess of biotinylated phiX/Linker-C fragments prepared as above, in an annealing buffer containing 2M guanidinium isothiocyanate (pH 5.8). The mixtures are heated to 65° C. for 5 minutes, then allowed to cool to 25° C. and hybridized for a 12-hour period.</p>
    <p>C. Separation by Affinity Chromatography</p>
    <p>A 1 ml silanized syringe plugged with silanized glass wool is packed with 0.5 ml streptavidin-agarose and washed with 10 mM Tris-Cl, 0.5 M NaCl, pH 7.0, containing 1 mM EDTA (elution buffer). A portion of each of the three hybridization mixtures from above is EtOH precipitated (70%), resuspended in 10 mM Tris HCl, 1 mM EDTA pH 7.0, and then loaded onto the column which is washed with several volumes of column elution buffer.</p>
    <p>For each mixture, the eluate fractions from the streptavidin column are combined and the DNA concentrated by ethanol precipitation. The DNA is dissolved in Buffer A to a concentration of 0.01 ug/ml, and mixed with 2 uM of the above primer having the sequence d(5'-GGAATTCCGGCCGCTCG-3'), and 200 uM each of dATP, dCTP, dGTP, and dTTP, and 5 units Taq polymerase. The reaction mixture is denatured at 94° C. for 1 minute, cooled to 50° C. for primer annealing, and primer extension performed at 72° C. for 5-12 minutes.</p>
    <p>The fragments from the each of the three eluate mixtures above are cut with NotI to produce NotI sticky ends in the linker regions of the fragments, as can be appreciated from FIG. 2A. The fragments (13 ug/ml) are mixed with 60 ug/ml NotI-digested phage Ch21aLJ, and inserted at the NotI site in the phage in the presence of T4 DNA ligase. After encapsidation, the phage are used to infect E. coli strain KM392, a suppressor-positive strain, and E. coli strain LG75, a suppressor-minus strain. The number of plaques produced in the KM392 strain provides a measure of total number of recombinant phage, since this strain is able to suppress the phage amber mutation, and thus allow phage expression of its A and B genes. In the LG75 strain, by contrast, only phage which have incorporated the piAN13 fragment produce plaques, since the sup F gene is required for growth of phage in suppressor minus bacterial strains. The ratio of plaques on the two host strains thus indicates the ratio of piAN13 fragments in the fragment mixtures. As a control, the ratio of fragments in the three mixtures before hybridization and streptavidin separation is also determined.</p>
    <heading>EXAMPLE 3</heading> <heading>Screening Amplified cDNA Fragment Mixtures</heading> <p>A. Bile-Derived cDNAs</p>
    <p>Two cynomolgus monkeys were intravenously injected with a 10% suspension of stool obtained from a human volunteer stool positive for ET-NANB, as evidenced by binding of virus-like particles (VLPs) in the stool to immune serum from a known ET-NANB patient. The animals developed elevated levels of alanine aminotransferase (ALT) between 24-36 days after inoculation, and excreted 27-30 nm VLPs in their stools in the pre-acute phase of infection. The animals became seropositive for VLPs in the inoculum.</p>
    <p>The bile duct of each infected animal was cannulated and about 1 cc of bile was collected. RNA was extracted from the bile by hot phenol extraction, using a standard RNA isolation procedure. Double-strand cDNA was formed from the isolated RNA by a random primer for first-strand generation, using a cDNA synthesis kit obtained from Boehringer-Mannheim (Indianapolis, Ind.).</p>
    <p>A cDNA fragment mixture from non-infected cynomolgus monkey bile was prepared similarly.</p>
    <p>B. Stool-Derived cDNAs</p>
    <p>cDNA fragments obtained from human stool samples were prepared as follows. A 10% suspension of stool samples from healthy or ET-NANB-infected individuals was layered over a 30% sucrose density gradient cushion, and centrifuged at 25 kg for 6 hr in an SW27 rotor, at 15° C. The pelleted material contained 27-32 nm VLP particles characteristic of ET-NANB infection in the infected-stool sample. RNA was isolated from the sucrose-gradient pellets in both the infected and non-infected samples, and the isolated RNA was used to produce cDNA fragments as described in Example 1.</p>
    <p>C. Amplification of the cDNA Fragment Mixtures</p>
    <p>The cDNA fragment mixtures from infected and non-infected bile source, and from infected and non-infected human-stool source were each amplified by linker/primer replication, according to the method of the invention. Briefly, the fragments in each sample were blunt-ended with DNA Pol I then extracted with phenol/chloroform and precipitated with ethanol. The blunt-ended material was ligated with linker A shown in FIG. 2, under conditions like those described in Example 1, and the mixtures were digested with NruI to remove linker dimers. Each mixture was brought to a final concentration of 0.01 ug/ml in Buffer A, and to this was added 2 uM of a primer having the sequence d(5'-GGAATTCGCGGCCGCTCG-3'), and 200 uM each of dATP, dCTP, dGTP, and dTTP. The reaction mixture was heated to 94° C. for 1 minute for denaturation and allowed to cool to 50° C. for 2 min for primer annealing, and then heated to 72° C. for 5-12 min to allow for primer extension by Taq polymerase. The replication reaction, involving successive heating, cooling, and polymerase reaction, was repeated an additional 25 times with the aid of a Perkin Elmer Cetus DNA thermal cycler.</p>
    <p>D. Screening Amplified cDNA Mixtures</p>
    <p>The four amplified cDNA fragment mixtures from above were fractionated by agarose gel electrophoresis, using a 2% agarose matrix. After transfer of the DNA fragments from the agarose gels to nitrocellulose paper, the filters were hybridized to a random-labeled <sup>32</sup> P 1.33 kb fragment associated with ET-NANB viral agent. The derivation and sequence of this fragment, which is carried in a pTZ-KF1(ET1.1) plasmid identified by ATCC No. 67717, is described in co-owned patent application for "Enterically Transmitted Non-A/Non-B Hepatitis Viral Agent", filed on even date herewith. The cloned fragment was obtained by digesting the pTZ-KF1(ET1.1) plasmid with EcoRI, (ii) isolating the released 1.33 kb ET-NANB-fragment, and (iii) randomly labeling the isolated fragment.</p>
    <p>The probe hybridization with the fractionated cDNA mixtures was performed by conventional Southern blotting methods (Maniatis, pp. 382-389). FIG. 6 shows the hybridization pattern obtained with cDNAs from infected (A) and non-infected (B) bile sources, and from infected (C) and non-infected (D) human stool sources. As seen, the ET-NANB probe hybridized with amplified fragments obtained from both of the infected sources, but was non-homologous to sequences obtained from either of the non-infected sources. This procedure demonstrates the utility of the described invention since hybridization analysis of an equivalent amount of cDNA prior to amplification would not have yielded detectable hybridization signal.</p>
    <p>While the invention has been described with reference to particular embodiments, methods, construction and use, it will be apparent to those skilled in the art that various changes and modifications can be made without departing from the invention.</p>
    </div></div></div><div class="patent-section patent-tabular-section"><a id="backward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Cited Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4683195">US4683195</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Feb 7, 1986</td><td class="patent-data-table-td patent-date-value">Jul 28, 1987</td><td class="patent-data-table-td ">Cetus Corporation</td><td class="patent-data-table-td ">One of the first Polymerase Chain Reaction (PCR) patents</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP0224126A2?cl=en">EP0224126A2</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Nov 12, 1986</td><td class="patent-data-table-td patent-date-value">Jun 3, 1987</td><td class="patent-data-table-td ">The University of Calgary</td><td class="patent-data-table-td ">Covalently linked complementary oligodeoxynucleotides as universal nucleic acid sequencing primer linkers</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="npl-citations"></a><div class="patent-section-header"><span class="patent-section-title">Non-Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th colspan="3"class="patent-data-table-th">Reference</th></tr></thead><tr><td class="patent-data-table-td ">1</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Maniatis et al., Molecular Cloning: A Laboratory Manual, Cold Spring Harbor, NY, 1982.</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="forward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Referenced by</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Citing Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7202039">US7202039</a></td><td class="patent-data-table-td patent-date-value">Nov 3, 2004</td><td class="patent-data-table-td patent-date-value">Apr 10, 2007</td><td class="patent-data-table-td ">Affymetrix, Inc.</td><td class="patent-data-table-td ">Complexity management of genomic DNA</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7267966">US7267966</a></td><td class="patent-data-table-td patent-date-value">Jul 12, 2001</td><td class="patent-data-table-td patent-date-value">Sep 11, 2007</td><td class="patent-data-table-td ">Affymetrix, Inc.</td><td class="patent-data-table-td ">Complexity management and analysis of genomic DNA</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7297778">US7297778</a></td><td class="patent-data-table-td patent-date-value">Oct 4, 2002</td><td class="patent-data-table-td patent-date-value">Nov 20, 2007</td><td class="patent-data-table-td ">Affymetrix, Inc.</td><td class="patent-data-table-td ">Complexity management of genomic DNA</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7622281">US7622281</a></td><td class="patent-data-table-td patent-date-value">May 19, 2005</td><td class="patent-data-table-td patent-date-value">Nov 24, 2009</td><td class="patent-data-table-td ">The Board Of Trustees Of The Leland Stanford Junior University</td><td class="patent-data-table-td ">Methods and compositions for clonal amplification of nucleic acid</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7655791">US7655791</a></td><td class="patent-data-table-td patent-date-value">Nov 13, 2002</td><td class="patent-data-table-td patent-date-value">Feb 2, 2010</td><td class="patent-data-table-td ">Rubicon Genomics, Inc.</td><td class="patent-data-table-td ">DNA amplification and sequencing using DNA molecules generated by random fragmentation</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7670810">US7670810</a></td><td class="patent-data-table-td patent-date-value">Jun 17, 2004</td><td class="patent-data-table-td patent-date-value">Mar 2, 2010</td><td class="patent-data-table-td ">Illumina, Inc.</td><td class="patent-data-table-td ">Methods and compositions for whole genome amplification and genotyping</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7718403">US7718403</a></td><td class="patent-data-table-td patent-date-value">Mar 8, 2004</td><td class="patent-data-table-td patent-date-value">May 18, 2010</td><td class="patent-data-table-td ">Rubicon Genomics, Inc.</td><td class="patent-data-table-td ">Amplification and analysis of whole genome and whole transcriptome libraries generated by a DNA polymerization process</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7741463">US7741463</a></td><td class="patent-data-table-td patent-date-value">Jul 14, 2006</td><td class="patent-data-table-td patent-date-value">Jun 22, 2010</td><td class="patent-data-table-td ">Illumina Cambridge Limited</td><td class="patent-data-table-td ">Used in solid-phase nucleic acid amplification; producing template polynucleotides that have common sequences at their 5&#39; ends and at their 3&#39; ends</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7754429">US7754429</a></td><td class="patent-data-table-td patent-date-value">Oct 5, 2007</td><td class="patent-data-table-td patent-date-value">Jul 13, 2010</td><td class="patent-data-table-td ">Illumina Cambridge Limited</td><td class="patent-data-table-td ">Method for pair-wise sequencing a plurity of target polynucleotides</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7790418">US7790418</a></td><td class="patent-data-table-td patent-date-value">Dec 7, 2001</td><td class="patent-data-table-td patent-date-value">Sep 7, 2010</td><td class="patent-data-table-td ">Illumina Cambridge Limited</td><td class="patent-data-table-td ">Isothermal amplification of nucleic acids on a solid support</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7803550">US7803550</a></td><td class="patent-data-table-td patent-date-value">Nov 13, 2008</td><td class="patent-data-table-td patent-date-value">Sep 28, 2010</td><td class="patent-data-table-td ">Rubicon Genomics, Inc.</td><td class="patent-data-table-td ">Attaching one strand of a stem-loop oligonucleotide having an inverted repeat and a loop to a double stranded nucleic acid molecule, and displacing one strand of the oligonucleotide from the oligonucleotide-attached nucleic acid molecule by strand displacement or by nick translation polymerization</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7960120">US7960120</a></td><td class="patent-data-table-td patent-date-value">Apr 1, 2010</td><td class="patent-data-table-td patent-date-value">Jun 14, 2011</td><td class="patent-data-table-td ">Illumina Cambridge Ltd.</td><td class="patent-data-table-td ">Method for pair-wise sequencing a plurality of double stranded target polynucleotides</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7972820">US7972820</a></td><td class="patent-data-table-td patent-date-value">May 5, 2010</td><td class="patent-data-table-td patent-date-value">Jul 5, 2011</td><td class="patent-data-table-td ">Illumina Cambridge Limited</td><td class="patent-data-table-td ">Isothermal amplification of nucleic acids on a solid support</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7985547">US7985547</a></td><td class="patent-data-table-td patent-date-value">Nov 17, 2008</td><td class="patent-data-table-td patent-date-value">Jul 26, 2011</td><td class="patent-data-table-td ">Affymetrix, Inc.</td><td class="patent-data-table-td ">Capturing sequences adjacent to type-IIs restriction sites for genomic library mapping</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7985565">US7985565</a></td><td class="patent-data-table-td patent-date-value">Jun 2, 2003</td><td class="patent-data-table-td patent-date-value">Jul 26, 2011</td><td class="patent-data-table-td ">Illumina, Inc.</td><td class="patent-data-table-td ">Method of nucleic acid amplification</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8017335">US8017335</a></td><td class="patent-data-table-td patent-date-value">Jul 20, 2006</td><td class="patent-data-table-td patent-date-value">Sep 13, 2011</td><td class="patent-data-table-td ">Illumina Cambridge Limited</td><td class="patent-data-table-td ">Method for sequencing a polynucleotide template</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8029999">US8029999</a></td><td class="patent-data-table-td patent-date-value">Oct 18, 2007</td><td class="patent-data-table-td patent-date-value">Oct 4, 2011</td><td class="patent-data-table-td ">Affymetrix, Inc.</td><td class="patent-data-table-td ">Complexity management of genomic DNA</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8053192">US8053192</a></td><td class="patent-data-table-td patent-date-value">Feb 4, 2008</td><td class="patent-data-table-td patent-date-value">Nov 8, 2011</td><td class="patent-data-table-td ">Illumina Cambridge Ltd.</td><td class="patent-data-table-td ">Methods for indexing samples and sequencing multiple polynucleotide templates</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8071312">US8071312</a></td><td class="patent-data-table-td patent-date-value">Sep 28, 2010</td><td class="patent-data-table-td patent-date-value">Dec 6, 2011</td><td class="patent-data-table-td ">Rubicon Genomics, Inc.</td><td class="patent-data-table-td ">Methods for producing and using stem-loop oligonucleotides</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8105784">US8105784</a></td><td class="patent-data-table-td patent-date-value">May 3, 2011</td><td class="patent-data-table-td patent-date-value">Jan 31, 2012</td><td class="patent-data-table-td ">Illumina Cambridge Limited</td><td class="patent-data-table-td ">Method for pairwise sequencing of target polynucleotides</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8143008">US8143008</a></td><td class="patent-data-table-td patent-date-value">Apr 30, 2010</td><td class="patent-data-table-td patent-date-value">Mar 27, 2012</td><td class="patent-data-table-td ">Illumina, Inc.</td><td class="patent-data-table-td ">Method of nucleic acid amplification</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8168388">US8168388</a></td><td class="patent-data-table-td patent-date-value">Nov 24, 2006</td><td class="patent-data-table-td patent-date-value">May 1, 2012</td><td class="patent-data-table-td ">Illumina Cambridge Ltd</td><td class="patent-data-table-td ">Preparation of nucleic acid templates for solid phase amplification</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8182989">US8182989</a></td><td class="patent-data-table-td patent-date-value">Aug 18, 2011</td><td class="patent-data-table-td patent-date-value">May 22, 2012</td><td class="patent-data-table-td ">Illumina Cambridge Ltd.</td><td class="patent-data-table-td ">Methods for indexing samples and sequencing multiple polynucleotide templates</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8182994">US8182994</a></td><td class="patent-data-table-td patent-date-value">Sep 15, 2009</td><td class="patent-data-table-td patent-date-value">May 22, 2012</td><td class="patent-data-table-td ">Illumina Cambridge Limited</td><td class="patent-data-table-td ">Centroid markers for image analysis of high denisty clusters in complex polynucleotide sequencing</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8192930">US8192930</a></td><td class="patent-data-table-td patent-date-value">Feb 8, 2007</td><td class="patent-data-table-td patent-date-value">Jun 5, 2012</td><td class="patent-data-table-td ">Illumina Cambridge Limited</td><td class="patent-data-table-td ">Method for sequencing a polynucleotide template</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8206913">US8206913</a></td><td class="patent-data-table-td patent-date-value">Mar 3, 2010</td><td class="patent-data-table-td patent-date-value">Jun 26, 2012</td><td class="patent-data-table-td ">Rubicon Genomics, Inc.</td><td class="patent-data-table-td ">Amplification and analysis of whole genome and whole transcriptome libraries generated by a DNA polymerization process</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8236505">US8236505</a></td><td class="patent-data-table-td patent-date-value">Dec 19, 2011</td><td class="patent-data-table-td patent-date-value">Aug 7, 2012</td><td class="patent-data-table-td ">Illumina Cambridge Limited</td><td class="patent-data-table-td ">Method for pairwise sequencing of target polynucleotides</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8247177">US8247177</a></td><td class="patent-data-table-td patent-date-value">Aug 4, 2011</td><td class="patent-data-table-td patent-date-value">Aug 21, 2012</td><td class="patent-data-table-td ">Illumina Cambridge Limited</td><td class="patent-data-table-td ">Method for sequencing a polynucleotide template</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8399199">US8399199</a></td><td class="patent-data-table-td patent-date-value">Nov 1, 2011</td><td class="patent-data-table-td patent-date-value">Mar 19, 2013</td><td class="patent-data-table-td ">Rubicon Genomics</td><td class="patent-data-table-td ">Use of stem-loop oligonucleotides in the preparation of nucleic acid molecules</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8409804">US8409804</a></td><td class="patent-data-table-td patent-date-value">Mar 2, 2006</td><td class="patent-data-table-td patent-date-value">Apr 2, 2013</td><td class="patent-data-table-td ">Rubicon Genomics, Inc.</td><td class="patent-data-table-td ">Isolation of CpG islands by thermal segregation and enzymatic selection-amplification method</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8431348">US8431348</a></td><td class="patent-data-table-td patent-date-value">Jun 29, 2012</td><td class="patent-data-table-td patent-date-value">Apr 30, 2013</td><td class="patent-data-table-td ">Illumina Cambridge Limited</td><td class="patent-data-table-td ">Method for pairwise sequencing of target polynucleotides</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8440404">US8440404</a></td><td class="patent-data-table-td patent-date-value">Mar 3, 2005</td><td class="patent-data-table-td patent-date-value">May 14, 2013</td><td class="patent-data-table-td ">Rubicon Genomics</td><td class="patent-data-table-td ">Methods and compositions for generating and amplifying DNA libraries for sensitive detection and analysis of DNA methylation</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8476044">US8476044</a></td><td class="patent-data-table-td patent-date-value">Apr 15, 2008</td><td class="patent-data-table-td patent-date-value">Jul 2, 2013</td><td class="patent-data-table-td ">Illumina, Inc.</td><td class="patent-data-table-td ">Method of nucleic acid amplification</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8541172">US8541172</a></td><td class="patent-data-table-td patent-date-value">Apr 12, 2012</td><td class="patent-data-table-td patent-date-value">Sep 24, 2013</td><td class="patent-data-table-td ">Illumina Cambridge Limited</td><td class="patent-data-table-td ">Method for sequencing a polynucelotide template</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8563478">US8563478</a></td><td class="patent-data-table-td patent-date-value">Apr 20, 2010</td><td class="patent-data-table-td patent-date-value">Oct 22, 2013</td><td class="patent-data-table-td ">Illumina Cambridge Limited</td><td class="patent-data-table-td ">Method of preparing libraries of template polynucleotides</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8652810">US8652810</a></td><td class="patent-data-table-td patent-date-value">Aug 17, 2006</td><td class="patent-data-table-td patent-date-value">Feb 18, 2014</td><td class="patent-data-table-td ">Illumina, Inc.</td><td class="patent-data-table-td ">Methods of nucleic acid amplification and sequencing</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8679758">US8679758</a></td><td class="patent-data-table-td patent-date-value">Aug 6, 2012</td><td class="patent-data-table-td patent-date-value">Mar 25, 2014</td><td class="patent-data-table-td ">Affymetrix, Inc.</td><td class="patent-data-table-td ">Complexity management of genomic DNA</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8728737">US8728737</a></td><td class="patent-data-table-td patent-date-value">Feb 13, 2013</td><td class="patent-data-table-td patent-date-value">May 20, 2014</td><td class="patent-data-table-td ">Rubicon Genomics, Inc.</td><td class="patent-data-table-td ">Attaching a stem-loop oligonucleotide to a double stranded DNA molecule</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8728764">US8728764</a></td><td class="patent-data-table-td patent-date-value">Oct 2, 2009</td><td class="patent-data-table-td patent-date-value">May 20, 2014</td><td class="patent-data-table-td ">Illumina Cambridge Limited</td><td class="patent-data-table-td ">Nucleic acid sample enrichment for sequencing applications</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8765381">US8765381</a></td><td class="patent-data-table-td patent-date-value">Apr 4, 2013</td><td class="patent-data-table-td patent-date-value">Jul 1, 2014</td><td class="patent-data-table-td ">Illumina Cambridge Limited</td><td class="patent-data-table-td ">Method for pairwise sequencing of target polynucleotides</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8778610">US8778610</a></td><td class="patent-data-table-td patent-date-value">Feb 27, 2013</td><td class="patent-data-table-td patent-date-value">Jul 15, 2014</td><td class="patent-data-table-td ">Rubicon Genomics, Inc.</td><td class="patent-data-table-td ">Methods for preparing amplifiable DNA molecules</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP1709059A2?cl=en">EP1709059A2</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Dec 9, 2004</td><td class="patent-data-table-td patent-date-value">Oct 11, 2006</td><td class="patent-data-table-td ">Sacred Heart Medical Center</td><td class="patent-data-table-td ">Methods of designing, synthesizing, and propagating reference nucleic acids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2374900A1?cl=en">EP2374900A1</a></td><td class="patent-data-table-td patent-date-value">Mar 8, 2004</td><td class="patent-data-table-td patent-date-value">Oct 12, 2011</td><td class="patent-data-table-td ">Rubicon Genomics, Inc.</td><td class="patent-data-table-td ">Polynucleotides for the amplification and analysis of whole genome and whole transcriptome libraries generated by a dna polymerization process</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2002031205A2?cl=en">WO2002031205A2</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Oct 11, 2001</td><td class="patent-data-table-td patent-date-value">Apr 18, 2002</td><td class="patent-data-table-td ">Paul B Fisher</td><td class="patent-data-table-td ">Method for rapid subtraction hybridization to isolate a gene which is differentially expressed between two populations of cells</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="classifications"></a><div class="patent-section-header"><span class="patent-section-title">Classifications</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th"> </th><th class="patent-data-table-th"> </th></tr></thead><tr><td class="patent-data-table-td ">U.S. Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=aL1QBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S006160">435/6.16</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=aL1QBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S024200">536/24.2</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=aL1QBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S091100">435/91.1</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=aL1QBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S024300">536/24.3</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=aL1QBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S025400">536/25.4</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=aL1QBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S091200">435/91.2</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=aL1QBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S024330">536/24.33</a></span></td></tr><tr><td class="patent-data-table-td ">International Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=aL1QBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0015090000">C12N15/09</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=aL1QBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0015100000">C12N15/10</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=aL1QBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12Q0001680000">C12Q1/68</a></span></td></tr><tr><td class="patent-data-table-td ">Cooperative Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=aL1QBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N15/1096">C12N15/1096</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=aL1QBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12Q1/6809">C12Q1/6809</a></span></td></tr><tr><td class="patent-data-table-td ">European Classification</td><td class="patent-data-table-td "><span class="nested-value">C12N15/10D</span>, <span class="nested-value">C12Q1/68A6</span></td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="legal-events"></a><div class="patent-section-header"><span class="patent-section-title">Legal Events</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Date</th><th class="patent-data-table-th">Code</th><th class="patent-data-table-th">Event</th><th class="patent-data-table-th">Description</th></tr></thead><tr><td class="patent-data-table-td patent-date-value">Aug 9, 2012</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">RELEASE BY SECURED PARTY;ASSIGNOR:LADATECH LLC;REEL/FRAME:028753/0106</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20100310</span></div><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">GENETIC VENTURES LLC, ILLINOIS</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Jul 31, 2012</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNOR:LADATECH, LLC;REEL/FRAME:028690/0700</span></div><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">ILLUMINA, INC., CALIFORNIA</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20120726</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Apr 16, 2012</td><td class="patent-data-table-td ">SULP</td><td class="patent-data-table-td ">Surcharge for late payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">11</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Apr 16, 2012</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">12</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Apr 2, 2012</td><td class="patent-data-table-td ">REMI</td><td class="patent-data-table-td ">Maintenance fee reminder mailed</td><td class="patent-data-table-td "></td></tr><tr><td class="patent-data-table-td patent-date-value">Apr 16, 2011</td><td class="patent-data-table-td ">SULP</td><td class="patent-data-table-td ">Surcharge for late payment</td><td class="patent-data-table-td "></td></tr><tr><td class="patent-data-table-td patent-date-value">May 9, 2010</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20100505</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">SECURITY AGREEMENT;ASSIGNOR:LADATECH, LLC;REEL/FRAME:024351/0967</span></div><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">GENETIC VENTURES LLC,ILLINOIS</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Apr 22, 2010</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">SECURITY AGREEMENT;ASSIGNOR:LADATECH, LLC;REEL/FRAME:024263/0840</span></div><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">GENETIC VENTURES LLC,ILLINOIS</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20100310</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Nov 17, 2009</td><td class="patent-data-table-td ">B1</td><td class="patent-data-table-td ">Reexamination certificate first reexamination</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">THE PATENTABILITY OF CLAIMS 12-14 IS CONFIRMED. NEW CLAIM 15 IS ADDED AND DETERMINED TO BE PATENTABLE. CLAIMS 1-11 WERE NOT REEXAMINED.</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Mar 30, 2009</td><td class="patent-data-table-td ">PRDP</td><td class="patent-data-table-td ">Patent reinstated due to the acceptance of a late maintenance fee</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20090330</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Aug 29, 2008</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">8</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Aug 29, 2008</td><td class="patent-data-table-td ">SULP</td><td class="patent-data-table-td ">Surcharge for late payment</td><td class="patent-data-table-td "></td></tr><tr><td class="patent-data-table-td patent-date-value">Apr 8, 2008</td><td class="patent-data-table-td ">RR</td><td class="patent-data-table-td ">Request for reexamination filed</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20080118</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Mar 3, 2008</td><td class="patent-data-table-td ">REMI</td><td class="patent-data-table-td ">Maintenance fee reminder mailed</td><td class="patent-data-table-td "></td></tr><tr><td class="patent-data-table-td patent-date-value">Jun 28, 2007</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">LADATECH, LLC, NEW YORK</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNOR:GENELABS TECHNOLOGIES, INC.;REEL/FRAME:019494/0028</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20070627</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Feb 23, 2004</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">4</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Aug 5, 1988</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">GENELABS INCORPORATED, 505 PENOBSCOT DRIVE, REDWOO</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST.;ASSIGNORS:REYES, GREGORY R.;KIM, JUNGSUH;REEL/FRAME:004927/0437</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">19880622</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNORS:REYES, GREGORY R.;KIM, JUNGSUH;REEL/FRAME:4927/437</span></div><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">GENELABS INCORPORATED, A CORP. OF CA,CALIFORNIA</span></div><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">GENELABS INCORPORATED, A CORP. OF CA, CALIFORNIA</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNORS:REYES, GREGORY R.;KIM, JUNGSUH;REEL/FRAME:004927/0437</span></div></td></tr></table><div class="patent-section-footer"></div></div><div class="modal-dialog" id="patent-images-lightbox"><div class="patent-lightbox-controls"><div class="patent-lightbox-rotate-controls"><div class="patent-lightbox-rotation-text">Rotate</div><div class="rotate-icon rotate-ccw-icon"></div><div class="rotate-icon rotate-cw-icon"></div></div><div class="patent-lightbox-index-counter"></div><a class="patent-lightbox-fullsize-link" target="_blank">Original Image</a><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_right.png" alt="Next page"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_left.png" alt="Previous page"width="21" height="21" /></div></div><div class="modal-dialog-content"><div class="patent-lightbox-image-holder"><div class="patent-lightbox-placeholder"></div></div></div></div><script>_OC_initPatentsAtb({image_not_available_html: " Image not available"});</script></div></div></div></td></tr></table><script>(function() {var href = window.location.href;if (href.indexOf('?') !== -1) {var parameters = href.split('?')[1].split('&');for (var i = 0; i < parameters.length; i++) {var param = parameters[i].split('=');if (param[0] == 'focus') {var elem = document.getElementById(param[1]);if (elem) {elem.focus();}}}}})();</script><script>_OC_addFlags({LockSrc:"/books/javascript/lock_8a2b04e7bf975d5171d8e4c0b6365c7a.js", Host:"http://www.google.com/", IsBooksRentalEnabled:1, IsWebstoreDisplayCaseEnabled:1, IsObfuscationEnabled:1, IsBrowsingHistoryEnabled:1, IsWebReaderSvgEnabled:0, IsGeoLayerEnabled:1, IsImageModeNotesEnabled:1, IsCopyMenuItemEnabled:1, IsGiftingEnabled:0, IsWebReaderUniversalPaginatorEnabled:0, IsOfflineBubbleEnabled:1, IsReaderEnabledForPlayRequests:1, IsFutureOnSaleVolumesEnabled:1, IsOfflineRestrictedCopyEnabled:1, IsBooksUnifiedLeftNavEnabled:1, IsRestrictedCopyEnabled:1, IsZipitFolderCollectionEnabled:1, IsEndOfSampleRecommendationsEnabled:1, IsRatingsOnBookcardsEnabled:1, IsAdsDisabled:0, IsIframePageDisplayEnabled:0, IsEmbeddedMediaEnabled:1, IsImageModeAnnotationsEnabled:1, IsMyLibraryGooglePlusEnabled:1, IsImagePageProviderEnabled:0, IsBookcardListPriceSmall:0, IsInternalUser:0, IsBooksShareButtonEnabled:0, IsPreOrdersEnabled:0, IsDisabledRandomBookshelves:0, WebstoreDisplayCasePosition:3});_OC_Run({"enable_p13n":false,"add_vol_to_collection_base_url":"http://www.google.com/patents?op=add\u0026sig=ACfU3U26IHipNJRNUGlUyO4_UDEt1F4bug\u0026id=aL1QBAABERAJ","remove_vol_from_collection_base_url":"http://www.google.com/patents?op=remove\u0026sig=ACfU3U0_T5dtMG_3Zbm-AIIBvp05LXNm1Q\u0026id=aL1QBAABERAJ","logged_in":false,"p13n_save_user_settings_url":"http://www.google.com/patents?op=edit_user_settings\u0026sig=ACfU3U39f_5tiXnF_-OHgpf-HtT8BOrIXg","is_cobrand":false,"sign_in_url":"https://www.google.com/accounts/Login?service=\u0026continue=http://www.google.com/patents%3Fhl%3Den\u0026hl=en","is_play_enabled":true}, {"volume_id":"","is_ebook":true,"volumeresult":{"has_flowing_text":false,"has_scanned_text":true,"can_download_pdf":false,"can_download_epub":false,"is_pdf_drm_enabled":false,"is_epub_drm_enabled":false,"download_pdf_url":"http://www.google.com/patents/download/Isolating_nucleotide_sequences_in_mix_jo.pdf?id=aL1QBAABERAJ\u0026output=pdf\u0026sig=ACfU3U0-jYl2HMkHJwjlNtXZonTR37w5oQ"},"sample_url":"http://www.google.com/patents/reader?id=aL1QBAABERAJ\u0026printsec=frontcover\u0026output=reader\u0026source=gbs_atb_hover","is_browsable":true,"is_public_domain":true}, {});</script><div id="footer_table" style="font-size:83%;text-align:center;position:relative;top:20px;height:4.5em;margin-top:2em"><div style="margin-bottom:8px"><a href=http://www.google.com/><nobr>Google&nbsp;Home</nobr></a> - <a href=//www.google.com/patents/sitemap/><nobr>Sitemap</nobr></a> - <a href=http://www.google.com/googlebooks/uspto.html><nobr>USPTO Bulk Downloads</nobr></a> - <a href=/intl/en/privacy/><nobr>Privacy Policy</nobr></a> - <a href=/intl/en/policies/terms/><nobr>Terms of Service</nobr></a> - <a href=https://support.google.com/faqs/answer/2539193?hl=en><nobr>About Google Patents</nobr></a> - <a href="http://www.google.com/tools/feedback/intl/en/error.html" onclick="try{_OC_startFeedback({productId: '72792',locale: 'en'});return false;}catch(e){}"><nobr>Send Feedback</nobr></a></div><span>Data provided by IFI CLAIMS Patent Services</span><br><span >&copy;2012 Google</span></div> <script type="text/javascript">var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));</script><script type="text/javascript">var pageTracker = _gat._getTracker("UA-27188110-1");pageTracker._setCookiePath("/patents/");pageTracker._trackPageview();</script> </body></html>