<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN""http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html><head><title>Patent US6114148 - High level expression of proteins - Google Patents</title><script>(function(){(function(){function e(a){this.t={};this.tick=function(a,c,b){var d=void 0!=b?b:(new Date).getTime();this.t[a]=[d,c];if(void 0==b)try{window.console.timeStamp("CSI/"+a)}catch(e){}};this.tick("start",null,a)}var a;window.performance&&(a=window.performance.timing);var f=a?new e(a.responseStart):new e;window.jstiming={Timer:e,load:f};if(a){var c=a.navigationStart,d=a.responseStart;0<c&&d>=c&&(window.jstiming.srt=d-c)}if(a){var b=window.jstiming.load;0<c&&d>=c&&(b.tick("_wtsrt",void 0,c),b.tick("wtsrt_",
"_wtsrt",d),b.tick("tbsd_","wtsrt_"))}try{a=null,window.chrome&&window.chrome.csi&&(a=Math.floor(window.chrome.csi().pageT),b&&0<c&&(b.tick("_tbnd",void 0,window.chrome.csi().startE),b.tick("tbnd_","_tbnd",c))),null==a&&window.gtbExternal&&(a=window.gtbExternal.pageT()),null==a&&window.external&&(a=window.external.pageT,b&&0<c&&(b.tick("_tbnd",void 0,window.external.startE),b.tick("tbnd_","_tbnd",c))),a&&(window.jstiming.pt=a)}catch(g){}})();})();
</script><link rel="stylesheet" href="/patents/css/_8a2b04e7bf975d5171d8e4c0b6365c7a/kl_intl_patents_bundle.css" type="text/css" /><script src="/books/javascript/atb_8a2b04e7bf975d5171d8e4c0b6365c7a__en.js"></script><script>function googleTranslateElementInit() {new google.translate.TranslateElement({pageLanguage: "en",gaTrack: true,gaId: "UA-27188110-1",multilanguagePage: true});}</script><script src="//translate.google.com/translate_a/element.js?cb=googleTranslateElementInit"></script><meta name="DC.type" content="Patent"><meta name="DC.title" content="High level expression of proteins"><meta name="DC.contributor" content="Brian Seed" scheme="inventor"><meta name="DC.contributor" content="Jurgen Haas" scheme="inventor"><meta name="DC.contributor" content="The General Hospital Corporation" scheme="assignee"><meta name="DC.date" content="1996-9-20" scheme="dateSubmitted"><meta name="DC.description" content="The invention features a synthetic gene encoding a protein normally expressed in a mammalian cell wherein at least one non-preferred or less preferred codon in the natural gene encoding the protein has been replaced by a preferred codon encoding the same amino acid."><meta name="DC.date" content="2000-9-5" scheme="issued"><meta name="DC.relation" content="EP:0335635:A1" scheme="references"><meta name="DC.relation" content="EP:0345242:A2" scheme="references"><meta name="DC.relation" content="US:4652639" scheme="references"><meta name="DC.relation" content="US:5082767" scheme="references"><meta name="DC.relation" content="US:5270171" scheme="references"><meta name="DC.relation" content="US:5276268" scheme="references"><meta name="DC.relation" content="US:5405776" scheme="references"><meta name="DC.relation" content="US:5464774" scheme="references"><meta name="DC.relation" content="US:5786464" scheme="references"><meta name="DC.relation" content="WO:1996009378:A1" scheme="references"><meta name="DC.relation" content="WO:1996027675:A1" scheme="references"><meta name="DC.relation" content="WO:1997026333:A1" scheme="references"><meta name="citation_reference" content="Bosch et al., Journal of Virology 68:7566 7569 (1994)."><meta name="citation_reference" content="Bosch et al., Journal of Virology 68:7566-7569 (1994)."><meta name="citation_reference" content="Chiu et al., &quot;Engineered GFP as a Vital Reporter in Plants&quot; Current Biology 6:325-330 (1996)."><meta name="citation_reference" content="Chiu et al., Engineered GFP as a Vital Reporter in Plants Current Biology 6:325 330 (1996)."><meta name="citation_reference" content="Chou et al., AIDS Research and Human Retroviruses 8:1967 1976 (1992)."><meta name="citation_reference" content="Chou et al., AIDS Research and Human Retroviruses 8:1967-1976 (1992)."><meta name="citation_reference" content="Cochrane et al., J. of Virology 65 (10):5305 5314 (1991)."><meta name="citation_reference" content="Cochrane et al., J. of Virology 65 (10):5305-5314 (1991)."><meta name="citation_reference" content="Cohen et al. (abstract), in Modern Approaches New Vaccines, 64 (1988)."><meta name="citation_reference" content="Coulombe and Skup, Gene 46:89 95 (1986)."><meta name="citation_reference" content="Coulombe and Skup, Gene 46:89-95 (1986)."><meta name="citation_reference" content="D Agostino et al., Mol. And Cell. Biol. 12(3):1375 1386 (1992)."><meta name="citation_reference" content="D&#39;Agostino et al., Mol. And Cell. Biol. 12(3):1375-1386 (1992)."><meta name="citation_reference" content="Feinberg et al., Cell 46:807 817 (1986)."><meta name="citation_reference" content="Feinberg et al., Cell 46:807-817 (1986)."><meta name="citation_reference" content="Fortkamp et al., DNA 5:511 517 (1986)."><meta name="citation_reference" content="Fortkamp et al., DNA 5:511-517 (1986)."><meta name="citation_reference" content="Grantham et al., Nature 319:727 728 (1986)."><meta name="citation_reference" content="Grantham et al., Nature 319:727-728 (1986)."><meta name="citation_reference" content="Haas et al., &quot;Codon Usage Limitation in the Expression of HIV-1 Envelope Glycoprotein&quot; Current Biology 6:315-324 (1996)."><meta name="citation_reference" content="Haas et al., Codon Usage Limitation in the Expression of HIV 1 Envelope Glycoprotein Current Biology 6:315 324 (1996)."><meta name="citation_reference" content="Hammarskjold et al. J. of Virol. 63(5):1959 1966 (1989)."><meta name="citation_reference" content="Hammarskjold et al. J. of Virol. 63(5):1959-1966 (1989)."><meta name="citation_reference" content="Haseloff et al., &quot;GFP in Plants&quot; Trends in Genetics 11:328-329 (1995)."><meta name="citation_reference" content="Haseloff et al., GFP in Plants Trends in Genetics 11:328 329 (1995)."><meta name="citation_reference" content="Hernan et al., Biochemistry 31:8619 8628 (1992)."><meta name="citation_reference" content="Hernan et al., Biochemistry 31:8619-8628 (1992)."><meta name="citation_reference" content="Holler et al., Gene 136:323 328 (1993)."><meta name="citation_reference" content="Holler et al., Gene 136:323-328 (1993)."><meta name="citation_reference" content="Inouye et al., FEBS Letters 341:277 280 (1994)."><meta name="citation_reference" content="Inouye et al., FEBS Letters 341:277-280 (1994)."><meta name="citation_reference" content="Kamiya et al., Jpn. J. Cancer Res. 80:200 203 (1989)."><meta name="citation_reference" content="Kamiya et al., Jpn. J. Cancer Res. 80:200-203 (1989)."><meta name="citation_reference" content="Kypr et al., Nature 327:20 (1987)."><meta name="citation_reference" content="McCarrey, Nucleic Acids Res. 18:949 955 (1990)."><meta name="citation_reference" content="McCarrey, Nucleic Acids Res. 18:949-955 (1990)."><meta name="citation_reference" content="Nakamura et al., FEBS Letters 289(1):123 125 (1991)."><meta name="citation_reference" content="Nakamura et al., FEBS Letters 289(1):123-125 (1991)."><meta name="citation_reference" content="Newgard et al., Proc. Natl. Acad. Sci. USA 83:8132 8136 (1986)."><meta name="citation_reference" content="Newgard et al., Proc. Natl. Acad. Sci. USA 83:8132-8136 (1986)."><meta name="citation_reference" content="Rangwala et al., Gene 122:263 269 (1992)."><meta name="citation_reference" content="Rangwala et al., Gene 122:263-269 (1992)."><meta name="citation_reference" content="Robinson et al., Nucl. Acids. Res. 12(17):6663 6671 (1984)."><meta name="citation_reference" content="Robinson et al., Nucl. Acids. Res. 12(17):6663-6671 (1984)."><meta name="citation_reference" content="Scorer et al., Gene 136:111 119 (1993)."><meta name="citation_reference" content="Scorer et al., Gene 136:111-119 (1993)."><meta name="citation_reference" content="Seetharam et al., &quot;Mistranslation in IGF-1 During Over-Expression of the Protein in Escherichia Coli Using a Synthetic Gene Containing Low Frequency Codons,&quot; Biochem. Biophys. Res. Comm. 155:518-523 (1988)."><meta name="citation_reference" content="Seetharam et al., Mistranslation in IGF 1 During Over Expression of the Protein in Escherichia Coli Using a Synthetic Gene Containing Low Frequency Codons, Biochem. Biophys. Res. Comm. 155:518 523 (1988)."><meta name="citation_reference" content="Sharp et al., Nucleic Acids Res. 16:8207 8211 (1988)."><meta name="citation_reference" content="Sharp et al., Nucleic Acids Res. 16:8207-8211 (1988)."><meta name="citation_reference" content="Sharp, Nature 324:114 (1986)."><meta name="citation_reference" content="van Hemert et al., Journal of Molecular Evolution 41:132 140 (1995)."><meta name="citation_reference" content="van Hemert et al., Journal of Molecular Evolution 41:132-140 (1995)."><meta name="citation_reference" content="Williams et al., Nucleic Acids Research 16/22:10453 10467 (1988)."><meta name="citation_reference" content="Williams et al., Nucleic Acids Research 16/22:10453-10467 (1988)."><meta name="citation_reference" content="Zhang et al., Gene 105(1):61 72 (1991)."><meta name="citation_reference" content="Zhang et al., Gene 105(1):61-72 (1991)."><meta name="citation_reference" content="Zhang et al., J. of Protein Chem. 12(3):329 335 (1993)."><meta name="citation_reference" content="Zhang et al., J. of Protein Chem. 12(3):329-335 (1993)."><meta name="citation_reference" content="Zolotukhin et al., &quot;A &quot;Humanized&quot; Green Fluorescent Protein cDNA Adapted for High-Level Expression in Mammalian Cells&quot; Journal of Virology 70:4646-4654 (1996)."><meta name="citation_reference" content="Zolotukhin et al., A Humanized Green Fluorescent Protein cDNA Adapted for High Level Expression in Mammalian Cells Journal of Virology 70:4646 4654 (1996)."><meta name="citation_patent_number" content="US:6114148"><meta name="citation_patent_application_number" content="US:08/717,294"><link rel="canonical" href="http://www.google.com/patents/US6114148"/><meta property="og:url" content="http://www.google.com/patents/US6114148"/><meta name="title" content="Patent US6114148 - High level expression of proteins"/><meta name="description" content="The invention features a synthetic gene encoding a protein normally expressed in a mammalian cell wherein at least one non-preferred or less preferred codon in the natural gene encoding the protein has been replaced by a preferred codon encoding the same amino acid."/><meta property="og:title" content="Patent US6114148 - High level expression of proteins"/><meta property="og:type" content="book"/><meta property="og:site_name" content="Google Books"/><meta property="og:image" content="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><link rel="image_src" href="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><script>(function(){try{var aa=function(a,b,c,d){d=d||{};d._sn=["cfg",b,c].join(".");window.gbar.logger.ml(a,d)};var g=window.gbar=window.gbar||{},l=window.gbar.i=window.gbar.i||{},m={},n;function _tvn(a,b){var c=parseInt(a,10);return isNaN(c)?b:c}function _tvf(a,b){var c=parseFloat(a);return isNaN(c)?b:c}function _tvv(a){return!!a}function p(a,b,c){(c||g)[a]=b}g.bv={n:_tvn("2",0),r:"",f:".67.",e:"0",m:_tvn("0",1)};
function q(a,b,c){var d="on"+b;if(a.addEventListener)a.addEventListener(b,c,!1);else if(a.attachEvent)a.attachEvent(d,c);else{var f=a[d];a[d]=function(){var a=f.apply(this,arguments),b=c.apply(this,arguments);return void 0==a?b:void 0==b?a:b&&a}}}var s=function(a){return function(){return g.bv.m==a}},ba=s(1),ca=s(2);p("sb",ba);p("kn",ca);l.a=_tvv;l.b=_tvf;l.c=_tvn;l.i=aa;var da=window.gbar.i.i;var t,u,v,w;function ea(a){v=a}function fa(a){var b;if(b=v&&window.encodeURIComponent)b=a.href,b=!b.match(/^http[s]?:\/\/accounts\.google\.[^/]*\/ClearSID/i)&&!b.match(/^http[s]?:\/\/[^/]*\/accounts\/ClearSID/i);if(b=b&&encodeURIComponent(v()))a.href=a.href.replace(/([?&]continue=)[^&]*/,"$1"+b)}function ga(a){window.gApplication&&(a.href=window.gApplication.getTabUrl(a.href))}
function ha(a){var b=document.forms[0].q,c=window.encodeURIComponent&&b&&b.value,b=b&&b.placeholder;c&&c!=b&&(a.href=a.href.replace(/([?&])q=[^&]*|$/,function(a,b){return(b||"&")+"q="+encodeURIComponent(c)}))}n=l.a("")?ga:ha;
function x(a,b,c,d,f,e){var h=document.getElementById(a);if(h){var k=h.style;k.left=d?"auto":b+"px";k.right=d?b+"px":"auto";k.top=c+"px";k.visibility=u?"hidden":"visible";f&&e?(k.width=f+"px",k.height=e+"px"):(x(t,b,c,d,h.offsetWidth,h.offsetHeight),u=u?"":a)}}
var y=[],ia=function(a,b){y.push(b)},ja=function(a){a=a||window.event;var b=a.target||a.srcElement;a.cancelBubble=!0;null==t&&(a=document.createElement(Array.every||window.createPopup?"iframe":"div"),a.frameBorder="0",t=a.id="gbs",a.src="javascript:''",b.parentNode.appendChild(a),q(document,"click",z));var c=b,b=0;"gb3"!=c.className&&(c=c.parentNode);a=c.getAttribute("aria-owns")||"gbi";var d=c.offsetWidth,f=20<c.offsetTop?46:24;document.getElementById("tphdr")&&(f-=3);var e=!1;do b+=c.offsetLeft||
0;while(c=c.offsetParent);var c=(document.documentElement.clientWidth||document.body.clientWidth)-b-d,h,d=document.body,k=document.defaultView;k&&k.getComputedStyle?(d=k.getComputedStyle(d,""))&&(h=d.direction):h=d.currentStyle?d.currentStyle.direction:d.style.direction;h="rtl"==h;if("gbi"==a){for(d=0;k=y[d++];)k();A(null,window.navExtra);h&&(b=c,e=!0)}else h||(b=c,e=!0);u!=a&&z();x(a,b,f,e)},z=function(){u&&x(u,0,0)},A=function(a,b){var c,d=document.getElementById("gbi"),f=a;f||(f=d.firstChild);
for(;b&&(c=b.pop());){var e=d,h=c,k=f;w||(w="gb2");e.insertBefore(h,k).className=w}},ka=function(a,b,c){if((b=document.getElementById(b))&&a){a.className="gb4";var d=document.createElement("span");d.appendChild(a);d.appendChild(document.createTextNode(" | "));d.id=c;b.appendChild(d)}},la=function(){return document.getElementById("gb_70")},ma=function(){return!!u};p("qs",n);p("setContinueCb",ea);p("pc",fa);p("tg",ja);p("close",z);p("addLink",ka);p("almm",A);p("si",la);p("adh",ia);p("op",ma);var B=function(){},C=function(){},F=function(a){var b=new Image,c=D;b.onerror=b.onload=b.onabort=function(){try{delete E[c]}catch(a){}};E[c]=b;b.src=a;D=c+1},E=[],D=0;p("logger",{il:C,ml:B,log:F});var G=window.gbar.logger;var H={},na={},I=[],oa=l.b("0.1",.1),pa=l.a("1",!0),qa=function(a,b){I.push([a,b])},ra=function(a,b){H[a]=b},sa=function(a){return a in H},J={},K=function(a,b){J[a]||(J[a]=[]);J[a].push(b)},ta=function(a){K("m",a)},L=function(a,b){var c=document.createElement("script");c.src=a;c.async=pa;Math.random()<oa&&(c.onerror=function(){c.onerror=null;B(Error("Bundle load failed: name="+(b||"UNK")+" url="+a))});(document.getElementById("xjsc")||document.getElementsByTagName("body")[0]||
document.getElementsByTagName("head")[0]).appendChild(c)},N=function(a){for(var b=0,c;(c=I[b])&&c[0]!=a;++b);!c||c[1].l||c[1].s||(c[1].s=!0,M(2,a),c[1].url&&L(c[1].url,a),c[1].libs&&m.d&&m.d(c[1].libs))},O=function(a){K("gc",a)},P=null,ua=function(a){P=a},M=function(a,b,c){if(P){a={t:a,b:b};if(c)for(var d in c)a[d]=c[d];try{P(a)}catch(f){}}};p("mdc",H);p("mdi",na);p("bnc",I);p("qGC",O);p("qm",ta);p("qd",J);p("lb",N);p("mcf",ra);p("bcf",qa);p("aq",K);p("mdd","");p("has",sa);
p("trh",ua);p("tev",M);var Q=l.b("0.1",.001),R=0;
function _mlToken(a,b){try{if(1>R){R++;var c,d=a,f=b||{},e=encodeURIComponent,h=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&jexpid=",e("17483"),"&srcpg=",e("prop=22"),"&jsr=",Math.round(1/Q),"&ogev=",e("llPsU6wPjuKwBM7ggcgC"),"&ogf=",g.bv.f,"&ogrp=",e("1"),"&ogv=",e("1407723702.0"),"&oggv="+e("es_plusone_gc_20140723.0_p0"),"&ogd=",e("com"),"&ogc=",e("CZE"),"&ogl=",e("en")];f._sn&&(f._sn="og."+
f._sn);for(var k in f)h.push("&"),h.push(e(k)),h.push("="),h.push(e(f[k]));h.push("&emsg=");h.push(e(d.name+":"+d.message));var r=h.join("");S(r)&&(r=r.substr(0,2E3));c=r;var Aa=window.gbar.logger._aem(a,c);F(Aa)}}catch(Na){}}var S=function(a){return 2E3<=a.length},va=function(a,b){return b};function T(a){B=a;p("_itl",S,G);p("_aem",va,G);p("ml",B,G);a={};H.er=a}l.a("")?T(function(a){throw a;}):l.a("1")&&Math.random()<Q&&T(_mlToken);I.push(["m",{url:"//ssl.gstatic.com/gb/js/scm_7385cc5883250b43a39405734c1bea59.js"}]);g.mcf("c",{});g.sg={c:""};if(l.a("1")){var wa=l.a("");I.push(["gc",{auto:wa,url:"//ssl.gstatic.com/gb/js/abc/gci_91f30755d6a6b787dcc2a4062e6e9824.js",libs:"googleapis.client:plusone:gapi.iframes"}]);var xa={version:"gci_91f30755d6a6b787dcc2a4062e6e9824.js",index:"",lang:"en"};H.gc=xa;var U=function(a){window.googleapis&&window.iframes?a&&a():(a&&O(a),N("gc"))};p("lGC",U);l.a("1")&&p("lPWF",U)};window.__PVT="";if(l.a("1")&&l.a("1")){var V=function(a){U(function(){K("pw",a);N("pw")})};p("lPW",V);I.push(["pw",{url:"//ssl.gstatic.com/gb/js/abc/pwm_45f73e4df07a0e388b0fa1f3d30e7280.js"}]);var W=[],ya=function(a){W[0]=a},za=function(a,b){var c=b||{};c._sn="pw";B(a,c)},Ba={signed:W,elog:za,base:"https://plusone.google.com/u/0",loadTime:(new Date).getTime()};H.pw=Ba;var X=function(a,b){for(var c=b.split("."),d=function(){var b=arguments;a(function(){for(var a=g,d=0,e=c.length-1;d<e;++d)a=a[c[d]];a[c[d]].apply(a,b)})},f=g,e=0,h=c.length-1;e<h;++e)f=
f[c[e]]=f[c[e]]||{};return f[c[e]]=d};X(V,"pw.clk");X(V,"pw.hvr");p("su",ya,g.pw)};function Ca(){function a(){for(var b;(b=e[h++])&&"m"!=b[0]&&!b[1].auto;);b&&(M(2,b[0]),b[1].url&&L(b[1].url,b[0]),b[1].libs&&m.d&&m.d(b[1].libs));h<e.length&&setTimeout(a,0)}function b(){0<f--?setTimeout(b,0):a()}var c=l.a("1"),d=l.a(""),f=3,e=I,h=0,k=window.gbarOnReady;if(k)try{k()}catch(r){da(r,"ml","or")}d?p("ldb",a):c?q(window,"load",b):b()}p("rdl",Ca);var Da={D:1,H:2,da:3,p:4,W:5,M:6,F:7,g:8,ha:9,U:10,L:11,T:12,S:13,N:14,Q:15,P:16,fa:17,w:18,O:19,ga:20,ea:21,u:22,G:23,ja:24,ka:25,ia:26,A:27,j:28,o:29,k:30,ca:31,Z:32,$:33,J:34,K:35,ba:36,aa:37,Y:38,B:39,R:40,v:41,X:42,V:43,h:48,C:49,I:500},Y=[1,2,3,4,5,6,9,10,11,13,14,28,29,30,34,35,37,38,39,40,41,42,43,48,49,500];var Z=l.b("0.001",1E-4),Ea=l.b("1",1),Fa=!1,Ga=!1;if(l.a("1")){var Ha=Math.random();Ha<=Z&&(Fa=!0);Ha<=Ea&&(Ga=!0)}var Ia=Da,$=null;function Ja(){var a=0,b=function(b,d){l.a(d)&&(a|=b)};b(1,"");b(2,"");b(4,"");b(8,"");return a}
function Ka(a,b){var c=Z,d=Fa,f;f=a;if(!$){$={};for(var e=0;e<Y.length;e++){var h=Y[e];$[h]=!0}}if(f=!!$[f])c=Ea,d=Ga;if(d){d=encodeURIComponent;g.rp?(f=g.rp(),f="-1"!=f?f:"1"):f="1";c=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&oge=",a,"&ogex=",d("17483"),"&ogev=",d("llPsU6wPjuKwBM7ggcgC"),"&ogf=",g.bv.f,"&ogp=",d("22"),"&ogrp=",d(f),"&ogsr=",Math.round(1/c),"&ogv=",d("1407723702.0"),"&oggv="+
d("es_plusone_gc_20140723.0_p0"),"&ogd=",d("com"),"&ogl=",d("en"),"&ogc=",d("CZE"),"&ogus=",Ja()];if(b){"ogw"in b&&(c.push("&ogw="+b.ogw),delete b.ogw);var k;f=b;e=[];for(k in f)0!=e.length&&e.push(","),e.push(La(k)),e.push("."),e.push(La(f[k]));k=e.join("");""!=k&&(c.push("&ogad="),c.push(d(k)))}F(c.join(""))}}function La(a){"number"==typeof a&&(a+="");return"string"==typeof a?a.replace(".","%2E").replace(",","%2C"):a}C=Ka;p("il",C,G);var Ma={};H.il=Ma;setTimeout(function(){C(Ia.g)},0);}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var b=window.gbar.i.i;var c=window.gbar;var f=function(d){try{var a=document.getElementById("gbom");a&&d.appendChild(a.cloneNode(!0))}catch(e){b(e,"omas","aomc")}};c.aomc=f;}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var a=window.gbar;a.mcf("pm",{p:""});}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{window.gbar.rdl();}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
if (window['_OC_timingAction']) {window['_OC_timingAction']('patents_refpage');}</script><style>#gbar,#guser{font-size:13px;padding-top:1px !important;}#gbar{float:left;height:22px}#guser{padding-bottom:7px !important;text-align:right}.gbh,.gbd{border-top:1px solid #c9d7f1;font-size:1px}.gbh{height:0;position:absolute;top:24px;width:100%}#gbs,.gbm{background:#fff;left:0;position:absolute;text-align:left;visibility:hidden;z-index:1000}.gbm{border:1px solid;border-color:#c9d7f1 #36c #36c #a2bae7;z-index:1001}.gb1{margin-right:.5em}.gb1,.gb3{zoom:1}.gb2{display:block;padding:.2em .5em}.gb2,.gb3{text-decoration:none !important;border-bottom:none}a.gb1,a.gb4{text-decoration:underline !important}a.gb1,a.gb2,a.gb3,a.gb4{color:#00c !important}.gbi .gb3,.gbi .gb2,.gbi .gb4{color:#dd8e27 !important}.gbf .gb3,.gbf .gb2,.gbf .gb4{color:#900 !important}a.gb2:hover{background:#36c;color:#fff !important}#gbar .gbz0l{color:#000 !important;cursor:default;font-weight:bold;text-decoration:none !important}
#gbar { padding:.3em .6em !important;}</style></head><body  topmargin="3" marginheight="3"><div id=gbar><nobr><a onclick=gbar.qs(this);gbar.logger.il(1,{t:1}); class=gb1 id=gb_1 href="https://www.google.com/search?sa=N&tab=tw">Search</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:2}); class=gb1 id=gb_2 href="http://www.google.com/search?hl=en&tbm=isch&source=og&sa=N&tab=ti">Images</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:8}); class=gb1 id=gb_8 href="http://maps.google.com/maps?hl=en&sa=N&tab=tl">Maps</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:78}); class=gb1 id=gb_78 href="https://play.google.com/?hl=en&sa=N&tab=t8">Play</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:36}); class=gb1 id=gb_36 href="http://www.youtube.com/results?sa=N&tab=t1">YouTube</a> <a onclick=gbar.logger.il(1,{t:5}); class=gb1 id=gb_5 href="http://news.google.com/nwshp?hl=en&tab=tn">News</a> <a onclick=gbar.logger.il(1,{t:23}); class=gb1 id=gb_23 href="https://mail.google.com/mail/?tab=tm">Gmail</a> <a onclick=gbar.logger.il(1,{t:25}); class=gb1 id=gb_25 href="https://drive.google.com/?tab=to">Drive</a> <a class=gb3 href="http://www.google.com/intl/en/options/" onclick="this.blur();gbar.tg(event);return !1" aria-haspopup=true><u>More</u> <small>&#9660;</small></a><div class=gbm id=gbi><a onclick=gbar.logger.il(1,{t:24}); class=gb2 id=gb_24 href="https://www.google.com/calendar?tab=tc">Calendar</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:51}); class=gb2 id=gb_51 href="http://translate.google.com/?hl=en&sa=N&tab=tT">Translate</a><a onclick=gbar.logger.il(1,{t:17}); class=gb2 id=gb_17 href="http://www.google.com/mobile/?hl=en&tab=tD">Mobile</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:10}); class=gb2 id=gb_10 href="http://www.google.com/search?hl=en&tbo=u&tbm=bks&source=og&sa=N&tab=tp">Books</a><a onclick=gbar.logger.il(1,{t:212}); class=gb2 id=gb_212 href="https://wallet.google.com/manage/?tab=ta">Wallet</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:6}); class=gb2 id=gb_6 href="http://www.google.com/search?hl=en&tbo=u&tbm=shop&source=og&sa=N&tab=tf">Shopping</a><a onclick=gbar.logger.il(1,{t:30}); class=gb2 id=gb_30 href="http://www.blogger.com/?tab=tj">Blogger</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:27}); class=gb2 id=gb_27 href="http://www.google.com/finance?sa=N&tab=te">Finance</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:31}); class=gb2 id=gb_31 href="https://plus.google.com/photos?sa=N&tab=tq">Photos</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:12}); class=gb2 id=gb_12 href="http://www.google.com/search?hl=en&tbo=u&tbm=vid&source=og&sa=N&tab=tv">Videos</a><div class=gb2><div class=gbd></div></div><a onclick=gbar.logger.il(1,{t:66}); href="http://www.google.com/intl/en/options/" class=gb2>Even more &raquo;</a></div></nobr></div><div id=guser width=100%><nobr><span id=gbn class=gbi></span><span id=gbf class=gbf></span><span id=gbe></span><a target=_top id=gb_70 href="https://www.google.com/accounts/Login?service=&continue=http://www.google.com/patents%3Fhl%3Den&hl=en" class=gb4>Sign in</a><div style="display: none"><div class=gbm id=gbd5 aria-owner=gbg5><div class=gbmc><ol id=gbom class=gbmcc></ol></div></div></div></nobr></div><div class=gbh style=left:0></div><div class=gbh style=right:0></div><div role="alert" style="position: absolute; left: 0; right: 0;"><a href="http://www.google.com/patents/us6114148?hl=en&amp;output=html_text" title="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."><img border="0" src="http://www.google.com/images/cleardot.gif"alt="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."></a></div><div id="guser"><nobr></nobr></div><div style="clear:both;"></div><div id="gb-top-search-box" class="gb-top-search-box-small gb-reset"><table><tr><td class="logo"><a href="http://www.google.com/patents" class="logo-link"><img class="logo-img" src="/intl/en/images/logos/google_logo_41.png" alt="Go to Google Books Home" height="41"/></a></td><td><form action="http://www.google.com/search" name="f" id="vheadf" method="get"><span id="hf"></span><input type="hidden" name="tbm" value="pts"/><input type="hidden" name="tbo" value="1"/><input type="hidden" name="hl" value="en"/><table><tr><td><div class="inputs"><table><tr><td><div class="text-input"><input type="text" name="q" id="vheadq" class="text" maxlength="2048" size="31" value="" title="Search Patents" accesskey="s" autocomplete="off"/><script>window._OC_autoDir &&window._OC_autoDir('vheadq', 'tia-vheadq');</script></div></td><td><div class="submit-input"><input name="btnG" class="submit" type="submit" value=""/></div></td></tr></table></div></td><td class="col-ext-links"><div class="ext-links"><a href="http://www.google.com/advanced_patent_search">&lt;nobr&gt;Advanced Patent Search&lt;/nobr&gt;</a></div></td></tr></table></form></td></tr></table></div><div class="kd-appbar"><h2 class="kd-appname"><a href="/patents">Patents</a></h2><div class="kd-buttonbar left" id="left-toolbar-buttons"><a id="appbar-write-review-link" href=""></a><a id="appbar-view-print-sample-link" href=""></a><a id="appbar-view-ebook-sample-link" href=""></a><a id="appbar-patents-prior-art-finder-link" href="https://www.google.com/patents/related/US6114148"></a><a id="appbar-patents-discuss-this-link" href="http://www.google.com/url?id=6yZRBAABERAJ&amp;q=http://patents.stackexchange.com/redirect/google-patents%3Fpatent%3DUS6114148&amp;usg=AFQjCNEA9ue647UTjCpIOmlL3WUxPdstOg" data-is-grant="true"></a><a id="appbar-read-patent-link" href="//docs.google.com/viewer?url=patentimages.storage.googleapis.com/pdfs/US6114148.pdf"></a><a id="appbar-download-pdf-link" href="//patentimages.storage.googleapis.com/pdfs/US6114148.pdf"></a></div><div class="kd-buttonbar right" id="right-toolbar-buttons"></div></div><div id="books-microdata" itemscope=""itemtype="http://schema.org/Book"itemid="http://www.google.com/patents/US6114148" style="display:none"><span itemprop="description">The invention features a synthetic gene encoding a protein normally expressed in a mammalian cell wherein at least one non-preferred or less preferred codon in the natural gene encoding the protein has been replaced by a preferred codon encoding the same amino acid....</span><span itemprop="url">http://www.google.com/patents/US6114148?utm_source=gb-gplus-share</span><span class="main-title" itemprop="name">Patent US6114148 - High level expression of proteins</span><img itemprop="image" src="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"alt="Patent US6114148 - High level expression of proteins" title="Patent US6114148 - High level expression of proteins"></div><div style="display: none"><ol id="ofe-gear-menu-contents" class="gbmcc"><li class="gbe gbmtc"><a class="gbmt goog-menuitem-content" id="" href="http://www.google.com/advanced_patent_search">Advanced Patent Search</a></li></ol></div><table id="viewport_table" cellpadding="0" style="clear:both" cellspacing="0"><tr><td id="viewport_td"><div class=vertical_module_list_row><div id=intl_patents class=about_content><div id=intl_patents_v><table class="patent-bibdata"><tr><td class="patent-bibdata-heading">Publication number</td><td class="single-patent-bibdata">US6114148 A</td></tr><tr><td class="patent-bibdata-heading">Publication type</td><td class="single-patent-bibdata">Grant</td></tr><tr><td class="patent-bibdata-heading">Application number</td><td class="single-patent-bibdata">US 08/717,294</td></tr><tr><td class="patent-bibdata-heading">Publication date</td><td class="single-patent-bibdata">Sep 5, 2000</td></tr><tr><td class="patent-bibdata-heading">Filing date</td><td class="single-patent-bibdata">Sep 20, 1996</td></tr><tr><td class="patent-bibdata-heading">Priority date<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The priority date is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the date listed."></span></td><td class="single-patent-bibdata">Sep 20, 1996</td></tr><tr><td class="patent-bibdata-heading">Fee status<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The fee status is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the status or dates listed."></span></td><td class="single-patent-bibdata">Paid</td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Also published as</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/CA2265976A1">CA2265976A1</a>, </span><span class="patent-bibdata-value"><a href="/patents/CA2265976C">CA2265976C</a>, </span><span class="patent-bibdata-value"><a href="/patents/CN1237977A">CN1237977A</a>, </span><span class="patent-bibdata-value"><a href="/patents/CN1307195C">CN1307195C</a>, </span><span class="patent-bibdata-value"><a href="/patents/DE69738586D1">DE69738586D1</a>, </span><span class="patent-bibdata-value"><a href="/patents/DE69738586T2">DE69738586T2</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP0929564A1">EP0929564A1</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP0929564A4">EP0929564A4</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP0929564B1">EP0929564B1</a>, </span><span class="patent-bibdata-value"><a href="/patents/WO1998012207A1">WO1998012207A1</a></span></span></td></tr><tr class="patent-bibdata-list-row alternate-patent-number"><td class="patent-bibdata-heading">Publication number</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value">08717294, </span><span class="patent-bibdata-value">717294, </span><span class="patent-bibdata-value">US 6114148 A, </span><span class="patent-bibdata-value">US 6114148A, </span><span class="patent-bibdata-value">US-A-6114148, </span><span class="patent-bibdata-value">US6114148 A, </span><span class="patent-bibdata-value">US6114148A</span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Inventors</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Brian+Seed%22">Brian Seed</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Jurgen+Haas%22">Jurgen Haas</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Original Assignee</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=inassignee:%22The+General+Hospital+Corporation%22">The General Hospital Corporation</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Export Citation</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/US6114148.bibtex">BiBTeX</a>, </span><span class="patent-bibdata-value"><a href="/patents/US6114148.enw">EndNote</a>, </span><span class="patent-bibdata-value"><a href="/patents/US6114148.ris">RefMan</a></span></span></td></tr><tr class="patent-internal-links"><td colspan=2><span class="patent-bibdata-value"><a href="#backward-citations">Patent Citations</a> (12),</span> <span class="patent-bibdata-value"><a href="#npl-citations">Non-Patent Citations</a> (61),</span> <span class="patent-bibdata-value"><a href="#forward-citations">Referenced by</a> (53),</span> <span class="patent-bibdata-value"><a href="#classifications">Classifications</a> (32),</span> <span class="patent-bibdata-value"><a href="#legal-events">Legal Events</a> (12)</span> </td></tr><tr><td colspan=2 class="patent-bibdata-external-link-spacer-top"></td></tr><tr class="patent-bibdata-external-link-spacer-bottom"></tr><tr><td colspan=2><span class="patent-bibdata-heading">External Links:&nbsp;</span><span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=6yZRBAABERAJ&q=http://patft.uspto.gov/netacgi/nph-Parser%3FSect2%3DPTO1%26Sect2%3DHITOFF%26p%3D1%26u%3D/netahtml/PTO/search-bool.html%26r%3D1%26f%3DG%26l%3D50%26d%3DPALL%26RefSrch%3Dyes%26Query%3DPN/6114148&usg=AFQjCNFi063Ofx293lYboJvzV0NnZwk2YQ">USPTO</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=6yZRBAABERAJ&q=http://assignments.uspto.gov/assignments/q%3Fdb%3Dpat%26pat%3D6114148&usg=AFQjCNHj4AoPdbeNeAa-ZhY6Svsq-IEhDg">USPTO Assignment</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=6yZRBAABERAJ&q=http://worldwide.espacenet.com/publicationDetails/biblio%3FCC%3DUS%26NR%3D6114148A%26KC%3DA%26FT%3DD&usg=AFQjCNFusGRqDv2Ewe_M-1FwreHg5GH4mw">Espacenet</a></span></span></td></tr><tr class="patent-bibdata-group-spacer"></tr></table><div class="number-and-title"><span class="patent-title"><invention-title mxw-id="PT54652398" lang="EN" load-source="patent-office">High level expression of proteins</invention-title></span><br><span class="patent-number">US 6114148 A</span></div><div class="patent-section patent-abstract-section"><div class="patent-section-header"><span class="patent-section-title">Abstract</span></div><div class="patent-text"><abstract mxw-id="PA38122435" lang="EN" load-source="patent-office"> <div class="abstract">The invention features a synthetic gene encoding a protein normally expressed in a mammalian cell wherein at least one non-preferred or less preferred codon in the natural gene encoding the protein has been replaced by a preferred codon encoding the same amino acid.</div>
  </abstract></div></div><div class="patent-section patent-drawings-section"><div class="patent-section-header"><span class="patent-section-title">Images<span class="patent-section-count">(18)</span></span></div><div class="patent-drawings-body"><div class="patent-drawings-carousel"><div class="drawings"><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6114148-1.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6114148-1.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6114148-2.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6114148-2.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6114148-3.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6114148-3.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6114148-4.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6114148-4.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6114148-5.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6114148-5.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6114148-6.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6114148-6.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6114148-7.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6114148-7.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6114148-8.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6114148-8.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6114148-9.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6114148-9.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6114148-10.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6114148-10.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6114148-11.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6114148-11.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6114148-12.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6114148-12.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6114148-13.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6114148-13.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6114148-14.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6114148-14.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6114148-15.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6114148-15.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6114148-16.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6114148-16.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6114148-17.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6114148-17.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6114148-18.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6114148-18.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div></div></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img" alt="Previous page"src="/googlebooks/images/kennedy/page_left.png"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img" alt="Next page"src="/googlebooks/images/kennedy/page_right.png"width="21" height="21" /></div></div></div><div class="patent-post-drawings"></div><div class="patent-section patent-claims-section"><div class="patent-section-header"><span class="patent-section-title">Claims<span class="patent-section-count">(18)</span></span></div><div class="patent-text"><div mxw-id="PCLM5622714" lang="EN" load-source="patent-office" class="claims">
    <claim-statement>What is claimed is:</claim-statement> <div class="claim"> <div num="1" class="claim">
      <div class="claim-text">1. A synthetic gene encoding a protein normally expressed in an eukaryotic cell wherein at least one non-preferred or less preferred codon in a natural gene encoding said protein has been replaced by a preferred codon encoding the same amino acid, said synthetic gene expressing said protein at a level which is at least 110% of that expressed by said natural gene in an in vitro mammalian cell culture system under identical conditions.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="2" class="claim">
      <div class="claim-text">2. The synthetic gene of claim 1 wherein said synthetic gene expresses said protein at a level which is at least 150% of that expressed by said natural gene in an in vitro cell culture system under identical conditions.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="3" class="claim">
      <div class="claim-text">3. The synthetic gene of claim 1 wherein said synthetic gene expresses said protein at a level which is at least 200% of that expressed by said natural gene in an in vitro cell culture system under identical conditions.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="4" class="claim">
      <div class="claim-text">4. The synthetic gene of claim 1 wherein said synthetic gene expresses said protein at a level which is at least 500% of that expressed by said natural gene in an in vitro cell culture system under identical conditions.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="5" class="claim">
      <div class="claim-text">5. The synthetic gene of claim 1 wherein said synthetic gene comprises fewer than 5 occurrences of the sequence CG.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="6" class="claim">
      <div class="claim-text">6. The synthetic gene of claim 1 wherein at least 10% of the codons in said natural gene are non-preferred codons.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="7" class="claim">
      <div class="claim-text">7. The synthetic gene of claim 1 wherein at least 50% of the codons in said natural gene are non-preferred codons.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="8" class="claim">
      <div class="claim-text">8. The synthetic gene of claim 1 wherein at least 50% of the non-preferred codons and less preferred codons present in said natural gene have been replaced by preferred codons.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="9" class="claim">
      <div class="claim-text">9. The synthetic gene of claim 1 wherein at least 90% of the non-preferred codons and less preferred codons present in said natural gene have been replaced by preferred codons.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="10" class="claim">
      <div class="claim-text">10. The synthetic gene of claim 1 wherein said protein is normally expressed by a mammalian cell.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="11" class="claim">
      <div class="claim-text">11. The synthetic gene of claim 1 wherein said protein is a human protein.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="12" class="claim">
      <div class="claim-text">12. The synthetic gene of claim 1 wherein 20% of the codons are preferred codons.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="13" class="claim">
      <div class="claim-text">13. An expression vector comprising the synthetic gene of claim 1.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="14" class="claim">
      <div class="claim-text">14. A mammalian cell which harbors the synthetic gene of claim 1.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="15" class="claim">
      <div class="claim-text">15. The synthetic gene of claim 11 wherein said human protein is Factor VIII.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="16" class="claim">
      <div class="claim-text">16. The expression vector of claim 13, said expression vector being a mammalian expression vector.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="17" class="claim">
      <div class="claim-text">17. The synthetic gene of claim 15 wherein said gene has the coding sequence present in SEQ ID NO:42.</div>
    </div>
    </div> <div class="claim"> <div num="18" class="claim">
      <div class="claim-text">18. A method for preparing a synthetic gene encoding a protein normally expressed by mammalian cells, comprising identifying non-preferred and less-preferred codons in the natural gene encoding said protein and replacing one or more of said non-preferred and less-preferred codons with a preferred codon encoding the same amino acid as the replaced codon, so that a synthetic gene is prepared.</div>
    </div>
  </div> </div></div></div><div class="patent-section patent-description-section"><div class="patent-section-header"><span class="patent-section-title">Description</span></div><div class="patent-text"><div mxw-id="PDES67529700" lang="EN" load-source="patent-office" class="description">
    <heading>FILED OF THE INVENTION</heading> <p>The invention concerns genes and methods for expressing eukaryotic and viral proteins at high levels in eukaryotic cells.</p>
    <heading>BACKGROUND OF THE INVENTION</heading> <p>Expression of eukaryotic gene products in prokaryotes is sometimes limited by the presence of codons that are infrequently used in E. coli. Expression of such genes can be enhanced by systematic substitution of the endogenous codons with codons over represented in highly expressed prokaryotic genes (Robinson et al., Nucleic Acids Res. 12:6663, 1984). It is commonly supposed that rare codons cause pausing of the ribosome, which leads to a failure to complete the nascent polypeptide chain and a uncoupling of transcription and translation. Pausing of the ribosome is thought to lead to exposure of the 3' end of the mRNA to cellular ribonucleases.</p>
    <heading>SUMMARY OF THE INVENTION</heading> <p>The invention features a synthetic gene encoding a protein normally expressed in a mammalian cell or other eukaryotic cell wherein at least one non-preferred or less preferred codon in the natural gene encoding the protein has been replaced by a preferred codon encoding the same amino acid.</p>
    <p>Preferred codons are: Ala (gcc); Arg (cgc); Asn (aac); Asp (gac) Cys (tgc); Gln (cag); Gly (ggc); His (cac); Ile (atc); Leu (ctg); Lys (aag); Pro (ccc); Phe (ttc); Ser (agc); Thr (acc); Tyr (tac); and Val (gtg). Less preferred codons are: Gly (ggg); Ile (att); Leu (ctc); Ser (tcc); Val (gtc); and Arg (agg). All codons which do not fit the description of preferred codons or less preferred codons are non-preferred codons. In general, the degree of preference of a particular codon is indicated by the prevalence of the codon in highly expressed human genes as indicated in Table 1 under the heading "High." For example, "atc" represents 77% of the Ile codons in highly expressed mammalian genes and is the preferred Ile codon; "att" represents 18% of the Ile codons in highly expressed mammalian genes and is the less preferred Ile codon. The sequence "ata" represents only 5% of the Ile codons in highly expressed human genes as is a non-preferred Ile codon. Replacing a codon with another codon that is more prevalent in highly expressed human genes will generally increase expression of the gene in mammalian cells. Accordingly, the invention includes replacing a less preferred codon with a preferred codon as well as replacing a non-preferred codon with a preferred or less preferred codon.</p>
    <p>By "protein normally expressed in a mammalian cell" is meant a protein which is expressed in mammalian under natural conditions. The term includes genes in the mammalian genome such as those encoding Factor VIII, Factor IX, interleukins, and other proteins. The term also includes genes which are expressed in a mammalian cell under disease conditions such as oncogenes as well as genes which are encoded by a virus (including a retrovirus) which are expressed in mammalian cells post-infection. By "protein normally expressed in a eukaryotic cell" is meant a protein which is expressed in a eukaryote under natural conditions. The term also includes genes which are expressed in a mammalian cell under disease conditions.</p>
    <p>In preferred embodiments, the synthetic gene is capable of expressing the mammalian or eukaryotic protein at a level which is at least 110%, 150%, 200%, 500%, 1,000%, 5,000% or even 10,000% of that expressed by the "natural" (or "native") gene in an in vitro mammalian cell culture system under identical conditions (i.e., same cell type, same culture conditions, same expression vector).</p>
    <p>Suitable cell culture systems for measuring expression of the synthetic gene and corresponding natural gene are described below. Other suitable expression systems employing mammalian cells are well known to those skilled in the art and are described in, for example, the standard molecular biology reference works noted below. Vectors suitable for expressing the synthetic and natural genes are described below and in the standard reference works described below. By "expression" is meant protein expression. Expression can be measured using an antibody specific for the protein of interest. Such antibodies and measurement techniques are well known to those skilled in the art. By "natural gene" and "native gene" is meant the gene sequence (including naturally occurring allelic variants) which naturally encodes the protein, i.e., the native or natural coding sequence.</p>
    <p>In other preferred embodiments at least 10%, 20%, 30%, 40%, 50%, 60%, 70%, 80%, or 90% of the codons in the natural gene are non-preferred codons.</p>
    <p>In other preferred embodiments at least 10%, 20%, 30%, 40%, 50%, 60%, 70%, 80%, or 90% of the non-preferred codons in the natural gene are replaced with preferred codons or less preferred codons.</p>
    <p>In other preferred embodiments at least 10%, 20%, 30%, 40%, 50%, 60%, 70%, 80%, or 90% of the non-preferred codons in the natural gene are replaced with preferred codons.</p>
    <p>In a preferred embodiment the protein is a retroviral protein. In a more preferred embodiment the protein is a lentiviral protein. In an even more preferred embodiment the protein is an HIV protein. In other preferred embodiments the protein is gag, pol, env, gp120, or gp160. In other preferred embodiments the protein is a human protein. In more preferred embodiments, the protein is human Factor VIII and the protein in B region deleted human Factor VIII. In another preferred embodiment the protein is green flourescent protein.</p>
    <p>In various preferred embodiments at least 30%, 40%, 50%, 60%, 70%, 80%, 90%, and 95% of the codons in the synthetic gene are preferred or less preferred codons.</p>
    <p>The invention also features an expression vector comprising the synthetic gene.</p>
    <p>In another aspect the invention features a cell harboring the synthetic gene. In various preferred embodiments the cell is a prokaryotic cell and the cell is a mammalian cell.</p>
    <p>In preferred embodiments the synthetic gene includes fewer than 50, fewer than 40, fewer than 30, fewer than 20, fewer than 10, fewer than 5, or no "cg" sequences.</p>
    <p>The invention also features a method for preparing a synthetic gene encoding a protein normally expressed by a mammalian cell or other eukaryotic cell. The method includes identifying non-preferred and less-preferred codons in the natural gene encoding the protein and replacing one or more of the non-preferred and less-preferred codons with a preferred codon encoding the same amino acid as the replaced codon.</p>
    <p>Under some circumstances (e.g., to permit introduction of a restriction site) it may be desirable to replace a non-preferred codon with a less preferred codon rather than a preferred codon.</p>
    <p>It is not necessary to replace all less preferred or non-preferred codons with preferred codons. Increased expression can be accomplished even with partial replacement of less preferred or non-preferred codons with preferred codons. Under some circumstances it may be desirable to only partially replace non-preferred codons with preferred or less preferred codons in order to obtain an intermediate level of expression.</p>
    <p>In other preferred embodiments the invention features vectors (including expression vectors) comprising one or more the synthetic genes.</p>
    <p>By "vector" is meant a DNA molecule, derived, e.g., from a plasmid, bacteriophage, or mammalian or insect virus, into which fragments of DNA may be inserted or cloned. A vector will contain one or more unique restriction sites and may be capable of autonomous replication in a defined host or vehicle organism such that the cloned sequence is reproducible. Thus, by "expression vector" is meant any autonomous element capable of directing the synthesis of a protein. Such DNA expression vectors include mammalian plasmids and viruses.</p>
    <p>The invention also features synthetic gene fragments which encode a desired portion of the protein. Such synthetic gene fragments are similar to the synthetic genes of the invention except that they encode only a portion of the protein. Such gene fragments preferably encode at least 50, 100, 150, or 500 contiguous amino acids of the protein.</p>
    <p>In constructing the synthetic genes of the invention it may be desirable to avoid CpG sequences as these sequences may cause gene silencing. Thus, in a preferred embodiment the coding region of the synthetic gene does not include the sequence "cg."</p>
    <p>The codon bias present in the HIV gp120 env gene is also present in the gag and pol genes. Thus, replacement of a portion of the non-preferred and less preferred codons found in these genes with preferred codons should produce a gene capable of higher level expression. A large fraction of the codons in the human genes encoding Factor VIII and Factor IX are non-preferred codons or less preferred codons. Replacement of a portion of these codons with preferred codons should yield genes capable of higher level expression in mammalian cell culture.</p>
    <p>The synthetic genes of the invention can be introduced into the cells of a living organism. For example, vectors (viral or non-viral) can be used to introduce a synthetic gene into cells of a living organism for gene therapy.</p>
    <p>Conversely, it may be desirable to replace preferred codons in a naturally occurring gene with less-preferred codons as a means of lowering expression.</p>
    <p>Standard reference works describing the general principles of recombinant DNA technology include Watson et al., Molecular Biology of the Gene, Volumes I and II, the Benjamin/Cummings Publishing Company, Inc., publisher, Menlo Park, Calif. (1987); Darnell et al., Molecular Cell Biology, Scientific American Books, Inc., Publisher, New York, N.Y. (1986); Old et al., Principles of Gene Manipulation: An Introduction to Genetic Engineering, 2d edition, University of California Press, publisher, Berkeley, Calif. (1981); Maniatis et al., Molecular Cloning: A Laboratory Manual, 2nd Ed. Cold Spring Harbor Laboratory, publisher, Cold Spring Harbor, N.Y. (1989); and Current Protocols in Molecular Biology, Ausubel et al., Wiley Press, New York, N.Y. (1992).</p>
    <p>By "transformed cell" is meant a cell into which (or into an ancestor of which) has been introduced, by means of recombinant DNA techniques, a selected DNA molecule, e.g., a synthetic gene.</p>
    <p>By "positioned for expression" is meant that a DNA molecule, e.g., a synthetic gene, is positioned adjacent to a DNA sequence which directs transcription and translation of the sequence (i.e., facilitates the production of the protein encoded by the synthetic gene.</p>
    <heading>DESCRIPTION OF THE DRAWINGS</heading> <p>FIG. 1 depicts the sequence of the synthetic gp120 (SEQ ID No.:34) and a synthetic gp160 gene (SEQ ID No.:35) in which codons have been replaced by those found in highly expressed human genes.</p>
    <p>FIG. 2 is a schematic drawing of the synthetic gp120 (HIV-1 MN) gene. The shaded portions marked v1 to v5 indicate hypervariable regions. The filled box indicates the CD4 binding site. A limited number of the unique restriction sites ares shown: H (Hind3), Nh (Nhe1), P (Pst1), Na (Nae1), M (Mlu1), R (EcoR1), A (Age1) and No (Not1). The chemically synthesized DNA fragments which served as PCR templates are shown below the gp120 sequence, along with the locations of the primers used for their amplification.</p>
    <p>FIG. 3 is a photograph of the results of transient transfection assays used to measure gp120 expression. Gel electrophoresis of immunoprecipitated supernatants of 293T cells transfected with plasmids expressing gp120 encoded by the IIIB isolate of HIV-1 (gp120IIIb), by the MN isolate of HIV-1 (gp120mn), by the MN isolate of HIV-1 modified by substitution of the endogenous leader peptide with that of the CD5 antigen (gp120mnCD5L), or by the chemically synthesized gene encoding the MN variant of HIV-1 with the human CD5Leader (syngp120mn). Supernatants were harvested following a 12 hour labeling period 60 hours posttransfection and immunoprecipitated with CD4:IgG1 fusion protein and protein A sepharose.</p>
    <p>FIG. 4 is a graph depicting the results of ELISA assays used to measure protein levels in supernatants of transiently transfected 293T cells. Supernatants of 293T cells transfected with plasmids expressing gp120 encoded by the IIIB isolate of HIV-1 (gp120 IIIb), by the MN isolate of HIV-1 (gp120mn), by the MN isolate of HIV-1 modified by substitution of the endogenous leader peptide with that of CD5 antigen (gp120mn CD5L), or by the chemically synthesized gene encoding the MN variant of HIV-1 with human CDS leader (syngp120mn) were harvested after 4 days and tested in a gp120/CD4 ELISA. The level of gp120 is expressed in ng/ml.</p>
    <p>FIG. 5A is a photograph of a gel illustrating the results of a immunoprecipitation assay used to measure expression of the native and synthetic gp120 in the presence of rev in trans and the RRE in cis. In this experiment 293T cells were transiently transfected by calcium phosphate co-precipitation of 10 g of plasmid expressing: (A) the synthetic gp120MN sequence and RRE in cis, (B) the gp120 portion of HIV-1 IIIB, (C) the gp120 portion of HIV-1 IIIB and RRE in cis, all in the presence or absence of rev expression. The RRE constructs gp120IIIbRRE and syngp120mnRRE were generated using an Eag1/Hpa1RRE fragment cloned by PCR from a HIV-1 HXB2 proviral clone. Each gp120 expression plasmid was cotransfected with 10 g of either pCMVrev or CDM7 plasmid DNA. Supernatants were harvested 60 hours post transfection, immunoprecipitated with CD4:IgG fusion protein and protein A agarose, and run on a 7% reducing SDS-PAGE. The gel exposure time was extended to allow the induction of gp120IIIbrre by rev to be demonstrated.</p>
    <p>FIG. 5B is a shorter exposure of a similar experiment in which syngp120mnrre was cotransfected with or without pCMVrev.</p>
    <p>FIG. 5C is a schematic diagram of the constructs used in FIG. 5A.</p>
    <p>FIG. 6 is a comparison of the sequence of the wild-type ratTHY-1 gene (wt) (SEQ ID No.:37) and a synthetic ratTHY-1 gene (env) (SEQ ID No.:36) constructed by chemical synthesis and having the most prevalent codons found in the HIV-1 env gene.</p>
    <p>FIG. 7 is a schematic diagram of the synthetic ratTHY-1 gene. The solid black box denotes the signal peptide. The shaded box denotes the sequences in the precursor which direct the attachment of a phophatidyl-inositol glycan anchor. Unique restriction sites used for assembly of the THY-1 constructs are marked H (Hind3), M (Mlu1), S (Sac1) and No (Not1). The position of the synthetic oligonucleotides employed in the construction are shown at the bottom of the figure.</p>
    <p>FIG. 8 is a graph depicting the results of flow cytometry analysis. In this experiment 293T cells transiently transfected with either a wild-type ratTHY-1 expression plasmid (thick line), ratTHY-1 with envelope codons expression plasmid (thin line), or vector only (dotted line) by calcium phosphate co-precipitation. Cells were stained with anti-ratTHY-1 monoclonal antibody OX7 followed by a polyclonal FITC-conjugated anti-mouse IgG antibody 3 days after transfection.</p>
    <p>FIG. 9A is a photograph of a gel illustrating the results of immunoprecipitation analysis of supernatants of human 293T cells transfected with either syngp120mn (A) or a construct syngp120mn.rTHY-1env which has the rTHY-1env gene in the 3' untranslated region of the syngp120mn gene (B). The syngp120mn.rTHY-1env construct was generated by inserting a Not1 adapter into the blunted Hind3 site of the rTHY-1env plasmid. Subsequently, a 0.5 kb Not1 fragment containing the rTHY-1env gene was cloned into the Not1 site of the syngp120mn plasmid and tested for correct orientation. Supernatants of <sup>35</sup> S labeled cells were harvested 72 hours post transfection, precipitated with CD4:IgG fusion protein and protein A agarose, and run on a 7% reducing SDS-PAGE.</p>
    <p>FIG. 9B is a schematic diagram of the constructs used in the experiment depicted in FIG. 9A.</p>
    <p>FIG. 10A is a photograph of COS cells transfected with vector only showing no GFP fluorescence.</p>
    <p>FIG. 10B is a photograph of COS cells transfected with a CDM7 expression plasmid encoding native GFP engineered to include a consensus translational initiation sequence.</p>
    <p>FIG. 10C is a photograph of COS cells transfected with an expression plasmid having the same flanking sequences and initiation consensus as in FIG. 10B, but bearing a codon optimized gene sequence.</p>
    <p>FIG. 10D is a photograph of COS cells transfected with an expression plasmid as in FIG. 10C, but bearing a Thr at residue 65 in place of Ser.</p>
    <p>FIG. 11 depicts the sequence of a synthetic gene encoding green flourescent proteins (SEQ ID NO:40).</p>
    <p>FIG. 12 depicts the sequence of a native human Factor VIII gene lacking the central B domain (amino acids 760-1639, inclusive) (SEQ ID NO:41).</p>
    <p>FIG. 13 depicts the sequence of a synthetic human Factor VIII gene lacking the central B domain (amino acids 760-1639, inclusive) (SEQ ID NO:42).</p>
    <heading>DESCRIPTION OF THE PREFERRED EMBODIMENTS</heading> <heading>EXAMPLE 1</heading> <p>Construction of a Synthetic gp120 Gene Having Codons Found in Highly Expressed Human Genes</p>
    <p>A codon frequency table for the envelope precursor of the LAV subtype of HIV-1 was generated using software developed by the University of Wisconsin Genetics Computer Group. The results of that tabulation are contrasted in Table 1 with the pattern of codon usage by a collection of highly expressed human genes. For any amino acid encoded by degenerate codons, the most favored codon of the highly expressed genes is different from the most favored codon of the HIV envelope precursor. Moreover a simple rule describes the pattern of favored envelope codons wherever it applies: preferred codons maximize the number of adenine residues in the viral RNA. In all cases but one this means that the codon in which the third position is A is the most frequently used. In the special case of serine, three codons equally contribute one A residue to the mRNA; together these three comprise 85% of the serine codons actually used in envelope transcripts. A particularly striking example of the A bias is found in the codon choice for arginine, in which the AGA triplet comprises 88% of the arginine codons. In addition to the preponderance of A residues, a marked preference is seen for uridine among degenerate codons whose third residue must be a pyrimidine. Finally, the inconsistencies among the less frequently used variants can be accounted for by the observation that the dinucleotide CpG is under represented; thus the third position is less likely to be G whenever the second position is C, as in the codons for alanine, proline, serine and threonine; and the CGX triplets for arginine are hardly used at all.</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">              TABLE 1______________________________________Codon Frequency in the HIV-1 IIIb env gene and in highly expressed human genes.      High   Env               High Env______________________________________Ala                          Cys  GC C 53 27 TG C 68 16   T 17 18  T 32 84   A 13 50 Gln   G 17  5 CA A 12 55  Arg     G 88 45  CG C 37  0 Glu   T  7  4 GA A 25 67   A  6  0  G 75 33   G 21  0 Gly  AG A 10 88 GG C 50  6   G 18  8  T 12 13  Asn     A 14 53  AA C 78 30  G 24 28   T 22 70 His  Asp    CA C 79 25  GA C 75 33  T 21 75   T 25 67 Ile  Leu    AT C 77 25  CT C 26 10  T 18 31   T  5  7  A  5 44   A  3 17 Ser   G 58 17 TC C 28  8  TT A  2 30  T 13  8   G  6 20  A  5 22  Lys     G  9  0  AA A 18 68 AG C 34 22   G 82 32  T 10 41  Pro    Thr  CC C 48 27 AC C 57 20   T 19 14  T 14 22   A 16 55  A 14 51   G 17  5  G 15  7  Phe    Tyr  TT C 80 26 TA C 74  8   T 20 74  T 26 92 Val GT C 25 12  T  7  9  A  5 62  G 64 18______________________________________ Codon frequency was calculated using the GCG program established the University of Wisconsin Genetics Computer Group. Numbers represent the percentage of cases in which the particular codon is used. Codon usage frequencies of envelope genes of other HIV1 virus isolates are comparable and show a similar bias.</pre>
    
    <p>In order to produce a gp120 gene capable of high level expression in mammalian cells, a synthetic gene encoding the gp120 segment of HIV-1 was constructed (syngp120mn), based on the sequence of the most common North American subtype, HIV-1 MN (Shaw et al., Science 226:1165, 1984; Gallo et al., Nature 321:119, 1986). In this synthetic gp120 gene nearly all of the native codons have been systematically replaced with codons most frequently used in highly expressed human genes (FIG. 1). This synthetic gene was assembled from chemically synthesized oligonucleotides of 150 to 200 bases in length. If oligonucleotides exceeding 120 to 150 bases are chemically synthesized, the percentage of full-length product can be low, and the vast excess of material consists of shorter oligonucleotides. Since these shorter fragments inhibit cloning and PCR procedures, it can be very difficult to use oligonucleotides exceeding a certain length. In order to use crude synthesis material without prior purification, single-stranded oligonucleotide pools were PCR amplified before cloning. PCR products were purified in agarose gels and used as templates in the next PCR step. Two adjacent fragments could be co-amplified because of overlapping sequences at the end of either fragment. These fragments, which were between 350 and 400 bp in size, were subcloned into a pCDM7-derived plasmid containing the leader sequence of the CD5 surface molecule followed by a Nhe1/Pst1/Mlu1/EcoR1/BamH1 polylinker. Each of the restriction enzymes in this polylinker represents a site that is present at either the 5' or 3' end of the PCR-generated fragments. Thus, by sequential subcloning of each of the 4 long fragments, the whole gp120gene was assembled. For each fragment three to six different clones were subcloned and sequenced prior to assembly. A schematic drawing of the method used to construct the synthetic gp120 is shown in FIG. 2. The sequence of the synthetic gp120 gene (and a synthetic gp160 gene created using the same approach) is presented in FIG. 1.</p>
    <p>The mutation rate was considerable. The most commonly found mutations were short (1 nucleotide) and long (up to 30 nucleotides) deletions. In some cases it was necessary to exchange parts with either synthetic adapters or pieces from other subclones without mutation in that particular region. Some deviations from strict adherence to optimized codon usage were made to accommodate the introduction of restriction sites into the resulting gene to facilitate the replacement of various segments (FIG. 2). These unique restriction sites were introduced into the gene at approximately 100 bp intervals. The native HIV leader sequence was exchanged with the highly efficient leader peptide of the human CD5 antigen to facilitate secretion (Aruffo et al., Cell 61:1303, 1990) The plasmid used for construction is a derivative of the mammalian expression vector pCDM7 transcribing the inserted gene under the control of a strong human CMV immediate early promoter.</p>
    <p>To compare the wild-type and synthetic gp120 coding sequences, the synthetic gp120 coding sequence was inserted into a mammalian expression vector and tested in transient transfection assays. Several different native gp120 genes were used as controls to exclude variations in expression levels between different virus isolates and artifacts induced by distinct leader sequences. The gp120 HIV IIIb construct used as control was generated by PCR using a Sal1/Xho1 HIV-1 HXB2 envelope fragment as template. To exclude PCR induced mutations, a Kpn1/Ear1 fragment containing approximately 1.2 kb of the gene was exchanged with the respective sequence from the proviral clone. The wild-type gp120mn constructs used as controls were cloned by PCR from HIV-1 MN infected C8166 cells (AIDS Repository, Rockville, Md.) and expressed gp120 either with a native envelope or a CD5 leader sequence. Since proviral clones were not available in this case, two clones of each construct were tested to avoid PCR artifacts. To determine the amount of secreted gp120 semi-quantitatively supernatants of 293T cells transiently transfected by calcium phosphate co-precipitation were immunoprecipitated with soluble CD4:immunoglobulin fusion protein and protein A sepharose.</p>
    <p>The results of this analysis (FIG. 3) show that the synthetic gene product is expressed at a very high level compared to that of the native gp120 controls. The molecular weight of the synthetic gp120 gene was comparable to control proteins (FIG. 3) and appeared to be in the range of 100 to 110 kd. The slightly faster migration can be explained by the fact that in some tumor cell lines, e.g., 293T, glycosylation is either not complete or altered to some extent.</p>
    <p>To compare expression more accurately gp120 protein levels were quantitated using a gp120 ELISA with CD4 in the demobilized phase. This analysis shows (FIG. 4) that ELISA data were comparable to the immunoprecipitation data, with a gp120 concentration of approximately 125 ng/ml for the synthetic gp120 gene, and less than the background cutoff (5 ng/ml) for all the native gp120 genes. Thus, expression of the synthetic gp120 gene appears to be at least one order of magnitude higher than wild-type gp120 genes. In the experiment shown the increase was at least 25 fold.</p>
    <p>The Role of Rev in gp120 Expression</p>
    <p>Since rev appears to exert its effect at several steps in the expression of a viral transcript, the possible role of non-translational effects in the improved expression of the synthetic gp120 gene was tested. First, to rule out the possibility that negative signals elements conferring either increased mRNA degradation or nucleic retention were eliminated by changing the nucleotide sequence, cytoplasmic mRNA levels were tested. Cytoplasmic RNA was prepared by NP40 lysis of transiently transfected 293T cells and subsequent elimination of the nuclei by centrifugation. Cytoplasmic RNA was subsequently prepared from lysates by multiple phenol extractions and precipitation, spotted on nitrocellulose using a slot blot apparatus, and finally hybridized with an envelope-specific probe.</p>
    <p>Briefly, cytoplasmic mRNA 293 cells transfected with CDM&amp;, gp120 IIIB, or syngp120was isolated 36 hours post transfection. Cytoplasmic RNA of Hela cells infected with wild-type vaccinia virus or recombinant virus expressing gp120 IIIb or the synthetic gp120 gene was under the control of the 7.5 promoter was isolated 16 hours post infection. Equal amounts were spotted on nitrocellulose using a slot blot device and hybridized with randomly labeled 1.5 kb gp120IIIb and syngp120 fragments or human beta-actin. RNA expression levels were quantitated by scanning the hybridized membranes with a phospoimager. The procedures used are described in greater detail below.</p>
    <p>This experiment demonstrated that there was no significant difference in the mRNA levels of cells transfected with either the native or synthetic gp120 gene. In fact, in some experiments cytoplasmic mRNA level of the synthetic gp120 gene was even lower than that of the native gp120 gene.</p>
    <p>These data were confirmed by measuring expression from recombinant vaccinia viruses. Human 293 cells or Hela cells were infected with vaccinia virus expressing wild-type gp120 IIIb or syngp120mn at a multiplicity of infection of at least 10. Supernatants were harvested 24 hours post infection and immunoprecipitated with CD4:immunoglobin fusion protein and protein A sepharose. The procedures used in this experiment are described in greater detail below.</p>
    <p>This experiment showed that the increased expression of the synthetic gene was still observed when the endogenous gene product and the synthetic gene product were expressed from vaccinia virus recombinants under the control of the strong mixed early and late 7.5 k promoter. Because vaccinia virus mRNAs are transcribed and translated in the cytoplasm, increased expression of the synthetic envelope gene in this experiment cannot be attributed to improved export from the nucleus. This experiment was repeated in two additional human cell types, the kidney cancer cell line 293 and HeLa cells. As with transfected 293T cells, mRNA levels were similar in 293 cells infected with either recombinant vaccinia virus.</p>
    <p>Codon Usage in Lentivirus</p>
    <p>Because it appears that codon usage has a significant impact on expression in mammalian cells, the codon frequency in the envelope genes of other retroviruses was examined. This study found no clear pattern of codon preference between retroviruses in general. However, if viruses from the lentivirus genus, to which HIV-1 belongs to, were analyzed separately, codon usage bias almost identical to that of HIV-1 was found. A codon frequency table from the envelope glycoproteins of a variety of (predominantly type C) retroviruses excluding the lentiviruses was prepared, and compared a codon frequency table created from the envelope sequences of four lentiviruses not closely related to HIV-1 (caprine arthritis encephalitis virus, equine infectious anemia virus, feline immunodeficiency virus, and visna virus) (Table 2). The codon usage pattern for lentiviruses is strikingly similar to that of HIV-1, in all cases but one, the preferred codon for HIV-1 is the same as the preferred codon for the other lentiviruses. The exception is proline, which is encoded by CCT in 41% of non-HIV lentiviral envelope residues, and by CCA in 40% of residues, a situation which clearly also reflects a significant preference for the triplet ending in A. The pattern of codon usage by the non-lentiviral envelope proteins does not show a similar predominance of A residues, and is also not as skewed toward third position C and G residues as is the codon usage for the highly expressed human genes. In general non-lentiviral retroviruses appear to exploit the different codons more equally, a pattern they share with less highly expressed human genes.</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">              TABLE 2______________________________________Codon frequency in the envelope gene of  lentiviruses (lenti) and non-lentiviral  retroviruses (other)      Other  Lenti             Other                                    Lenti______________________________________Ala                          Cys  GC C 45 13 TG C 53 21   T 26 37  T 47 79   A 20 46 Gln   G  9  3 CA A 52 69  Arg     G 48 31  CG C 14  2 Glu   T  6  3 GA A 57 68   A 16  5  G 43 32   G 17  3 Gly  AG A 31 51 GG C 21  8   G 15 26  T 13  9  Asn     A 37 56  AA C 49 31  G 29 26   T 51 69 His  Asp    CA C 51 38  GA C 55 33  T 49 62   T 51 69 Ile  Leu    AT C 38 16  CT C 22  8  T 31 22   T 14  9  A 31 61   A 21 16 Ser   G 19 11 TC C 38 10  TT A 15 41  T 17 16   G 10 16  A 18 24  Lys     G  6  5  AA A 60 63 AG C 13 20   G 40 37  T  7 25  Pro    Thr  CC C 42 14 AC C 44 18   T 30 41  T 27 20   A 20 40  A 19 55   G  7  5  G 10  8  Phe Tyr  TT C 52 25 TA C 48 28   T 48 75  T 52 72 Val GT C 36  9  T 17 10  A 22 54  G 25 27______________________________________ Codon frequency was calculated using the GCG program established by the University of Wisconsin Genetics Computer Group. Numbers represent the percentage in which a particular codon is used. Codon usage of nonlentiviral retroviruses was compiled from the envelope precursor sequences of bovine leukemia virus feline leukemia virus, human Tcell leukemia virus type I, human Tcell lymphotropic virus type II, the mink cell focusforming isolate of murine leukemia virus (MuLV),  # the Rausche spleen focusforming isolate, the 10A1 isolate, the 4070A amphotropic isolate and the myeloproliferative leukemia virus isolate, and from rat leukemia virus, simian sarcoma virus, simian Tcell leukemia virus, leukemogenic retrovirus T1223/B and gibbon ape leukemia virus. The codon frequency tables for the nonHIV, nonSIV lentiviruses were compiled from the envelope precursor sequences for caprine arthritis encephalitis virus equine infectious anemia virus,  # feline immunodeficiency virus, and visna virus.</pre>
    
    <p>In addition to the prevalence of codons containing an A, lentiviral codons adhere to the HIV pattern of strong CpG under representation, so that the third position for alanine, proline, serine and threonine triplets is rarely G. The retroviral envelope triplets show a similar, but less pronounced, under representation of CpG. The most obvious difference between lentiviruses and other retroviruses with respect to CpG prevalence lies in the usage of the CGX variant of arginine triplets, which is reasonably frequently represented among the retroviral envelope coding sequences, but is almost never present among the comparable lentivirus sequences.</p>
    <p>Differences in Rev Dependence Between Native and Synthetic gp120</p>
    <p>To examine whether regulation by rev is connected to HIV-1 codon usage, the influence of rev on the expression of both native and synthetic gene was investigated. Since regulation by rev requires the rev-binding site RRE in cis, constructs were made in which this binding site was cloned into the 3' untranslated region of both the native and the synthetic gene. These plasmids were co-transfected with rev or a control plasmid in trans into 293T cells, and gp120 expression levels in supernatants were measured semiquantitatively by immunoprecipitation. The procedures used in this experiment are described in greater detail below.</p>
    <p>As shown in FIG. 5A and FIG. 5B, rev up regulates the native gp120 gene, but has no effect on the expression of the synthetic gp120gene. Thus, the action of rev is not apparent on a substrate which lacks the coding sequence of endogenous viral envelope sequences.</p>
    <p>Expression of a Synthetic ratTHY-1 Gene with HIV Envelope Codons</p>
    <p>The above-described experiment suggest that in fact "envelope sequences" have to be present for rev regulation. In order to test this hypothesis, a synthetic version of the gene encoding the small, typically highly expressed cell surface protein, ratTHY-1 antigen, was prepared. The synthetic version of the ratTHY-1 gene was designed to have a codon usage like that of HIV gp120. In designing this synthetic gene AUUUA sequences, which are associated with mRNA instability, were avoided. In addition, two restriction sites were introduced to simplify manipulation of the resulting gene (FIG. 6). This synthetic gene with the HIV envelope codon usage (rTHY-1env) was generated using three 150 to 170 mer oligonucleotides (FIG. 7). In contrast to the syngp120mn gene, PCR products were directly cloned and assembled in pUC12, and subsequently cloned into pCDM7.</p>
    <p>Expression levels of native rTHY-1 and rTHY-1 with the HIV envelope codons were quantitated by immunofluorescence of transiently transfected 293T cells. FIG. 8 shows that the expression of the native THY-1 gene is almost two orders of magnitude above the background level of the control transfected cells (pCDM7). In contrast, expression of the synthetic ratTHY-1 is substantially lower than that of the native gene (shown by the shift to of the peak towards a lower channel number).</p>
    <p>To prove that no negative sequence elements promoting mRNA degradation were inadvertently introduced, a construct was generated in which the rTHY-1env gene was cloned at the 3' end of the synthetic gp120 gene (FIG. 9B). In this experiment 293T cells were transfected with either the syngp120mn gene or the syngp120/ratTHY-1 env fusion gene (syngp120mn.rTHY-1env). Expression was measured by immunoprecipitation with CD4:IgG fusion protein and protein A agarose. The procedures used in this experiment are described in greater detail below.</p>
    <p>Since the synthetic gp120 gene has an UAG stop codon, rTHY-1env is not translated from this transcript. If negative elements conferring enhanced degradation were present in the sequence, gp120 protein levels expressed from this construct should be decreased in comparison to the syngp120mn construct without rTHY-1env. FIG. 9A, shows that the expression of both constructs is similar, indicating that the low expression must be linked to translation.</p>
    <p>Rev-dependent Expression of Synthetic ratTHY-1 Gene with Envelope Codons</p>
    <p>To explore whether rev is able to regulate expression of a ratTHY-1 gene having env codons, a construct was made with a rev-binding site in the 3' end of the rTHY1env open reading frame. To measure rev-responsiveness of the a ratTHY-1env construct having a 3' RRE, human 293T cells were cotransfected ratTHY-1envrre and either CDM7 or pCMVrev. At 60 hours post transfection cells were detached with 1 mM EDTA in PBS and stained with the OX-7 anti rTHY-1 mouse monoclonal antibody and a secondary FITC-conjugated antibody. Fluorescence intensity was measured using a EPICS XL cytofluorometer. These procedures are described in greater detail below.</p>
    <p>In repeated experiments, a slight increase of rTHY-1env expression was detected if rev was cotransfected with the rTHY-1env gene. To further increase the sensitivity of the assay system a construct expressing a secreted version of rTHY-1env was generated. This construct should produce more reliable data because the accumulated amount of secreted protein in the supernatant reflects the result of protein production over an extended period, in contrast to surface expressed protein, which appears to more closely reflect the current production rate. A gene capable of expressing a secreted form was prepared by PCR using forward and reverse primers annealing 3' of the endogenous leader sequence and 5' of the sequence motif required for phosphatidylinositol glycan anchorage respectively. The PCR product was cloned into a plasmid which already contained a CD5 leader sequence, thus generating a construct in which the membrane anchor has been deleted and the leader sequence exchanged by a heterologous (and probably more efficient) leader peptide.</p>
    <p>The rev-responsiveness of the secreted form ratTHY-1env was measured by immunoprecipitation of supernatants of human 293T cells cotransfected with a plasmid expressing a secreted form of ratTHY-1env and the RRE sequence in cis (rTHY-1envPI-rre) and either CDM7 or pCMVrev. The rTHY-1envPI-RRE construct was made by PCR using the oligonucleotide: cgcggggctagcgcaaagagtaataagtttaac (SEQ ID NO:38) as a forward primer, the oligonucleotide: cgcggatcccttgtattttgtactaata (SEQ ID NO:39) as reverse primer, and the synthetic rTHY-1env construct as a template. After digestion with Nhe1 and Not1 the PCR fragment was cloned into a plasmid containing CD5 leader and RRE sequences. Supernatants of <sup>35</sup> S labeled cells were harvested 72 hours post transfection, precipitated with a mouse monoclonal antibody OX7 against rTHY-1 and anti mouse IgG sepharose, and run on a 12% reducing SDS-PAGE.</p>
    <p>In this experiment the induction of rTHY-1env by rev was much more prominent and clear-cut than in the above-described experiment and strongly suggests that rev is able to translationally regulate transcripts that are suppressed by low-usage codons.</p>
    <p>Rev-independent Expression of a rTHY-1env:immunoglobulin Fusion Protein</p>
    <p>To test whether low-usage codons must be present throughout the whole coding sequence or whether a short region is sufficient to confer rev-responsiveness, a rTHY-1env:immunoglobulin fusion protein was generated. In this construct the rTHY-1env gene (without the sequence motif responsible for phosphatidylinositol glycan anchorage) is linked to the human IgG1 hinge, CH2 and CH3 domains. This construct was generated by anchor PCR using primers with Nhe1 and BamHI restriction sites and rTHY-1env as template. The PCR fragment was cloned into a plasmid containing the leader sequence of the CD5 surface molecule and the hinge, CH2 and CH3 parts of human IgG1 immunoglobulin. A Hind3/Eag1 fragment containing the rTHY-1enveg1 insert was subsequently cloned into a pCDM7-derived plasmid with the RRE sequence.</p>
    <p>To measure the response of the rTHY-1env/immunoglobin fusion gene (rTHY-1enveg1rre) to rev human 293T cells cotransfected with rTHY-1enveg1rre and either pCDM7 or pCMVrev. The rTHY-1enveg1rre construct was made by anchor PCR using forward and reverse primers with Nhe1 and BamH1 restriction sites respectively. The PCR fragment was cloned into a plasmid containing a CD5 leader and human IgG1 hinge, CH2 and CH3 domains. Supernatants of <sup>35</sup> S labeled cells were harvested 72 hours post transfection, precipitated with a mouse monoclonal antibody OX7 against rTHY-1 and anti mouse IgG sepharose, and run on a 12% reducing SDS-PAGE. The procedures used are described in greater detail below.</p>
    <p>As with the product of the rTHY-1envPI- gene, this rTHY-1env/immunoglobulin fusion protein is secreted into the supernatant. Thus, this gene should be responsive to rev-induction. However, in contrast to rTHY-1envPI-, cotransfection of rev in trans induced no or only a negligible increase of rTHY-1enveg1 expression.</p>
    <p>The expression of rTHY-1:immunoglobulin fusion protein with native rTHY-1 or HIV envelope codons was measured by immunoprecipitation. Briefly, human 293T cells transfected with either rTHY-1enveg1 (env codons) or rTHY-1wteg1 (native codons). The rTHY-1wteg1 construct was generated in manner similar to that used for the rTHY-1enveg1 construct, with the exception that a plasmid containing the native rTHY-1 gene was used as template. Supernatants of <sup>35</sup> S labeled cells were harvested 72 hours post transfection, precipitated with a mouse monoclonal antibody OX7 against rTHY-1 and anti mouse IgG sepharose, and run on a 12% reducing SDS-PAGE. THE procedures used in this experiment are described in greater detail below.</p>
    <p>Expression levels of rTHY-1enveg1 were decreased in comparison to a similar construct with wild-type rTHY-1 as the fusion partner, but were still considerably higher than rTHY-1env. Accordingly, both parts of the fusion protein influenced expression levels. The addition of rTHY-1env did not restrict expression to an equal level as seen for rTHY-1env alone. Thus, regulation by rev appears to be ineffective if protein expression is not almost completely suppressed.</p>
    <p>Codon Preference in HIV-1 Envelope Genes</p>
    <p>Direct comparison between codon usage frequency of HIV envelope and highly expressed human genes reveals a striking difference for all twenty amino acids. One simple measure of the statistical significance of this codon preference is the finding that among the nine amino acids with two fold codon degeneracy, the favored third residue is A or U in all nine. The probability that all nine of two equiprobable choices will be the same is approximately 0.004, and hence by any conventional measure the third residue choice cannot be considered random. Further evidence of a skewed codon preference is found among the more degenerate codons, where a strong selection for triplets bearing adenine can be seen. This contrasts with the pattern for highly expressed genes, which favor codons bearing C, or less commonly G, in the third position of codons with three or more fold degeneracy.</p>
    <p>The systematic exchange of native codons with codons of highly expressed human genes dramatically increased expression of gp120. A quantitative analysis by ELISA showed that expression of the synthetic gene was at least 25 fold higher in comparison to native gp120 after transient transfection into human 293 cells. The concentration levels in the ELISA experiment shown were rather low. Since an ELISA was used for quantification which is based on gp120 binding to CD4, only native, non-denatured material was detected. This may explain the apparent low expression. Measurement of cytoplasmic mRNA levels demonstrated that the difference in protein expression is due to translational differences and not mRNA stability.</p>
    <p>Retroviruses in general do not show a similar preference towards A and T as found for HIV. But if this family was divided into two subgroups, lentiviruses and non-lentiviral retroviruses, a similar preference to A and, less frequently, T, was detected at the third codon position for lentiviruses. Thus, the availing evidence suggests that lentiviruses retain a characteristic pattern of envelope codons not because of an inherent advantage to the reverse transcription or replication of such residues, but rather for some reason peculiar to the physiology of that class of viruses. The major difference between lentiviruses and non-complex retroviruses are additional regulatory and non-essentially accessory genes in lentiviruses, as already mentioned. Thus, one simple explanation for the restriction of envelope expression might be that an important regulatory mechanism of one of these additional molecules is based on it. In fact, it is known that one of these proteins, rev, which most likely has homologues in all lentiviruses. Thus codon usage in viral mRNA is used to create a class of transcripts which is susceptible to the stimulatory action of rev. This hypothesis was proved using a similar strategy as above, but this time codon usage was changed into the inverse direction. Codon usage of a highly expressed cellular gene was substituted with the most frequently used codons in the HIV envelope. As assumed, expression levels were considerably lower in comparison to the native molecule, almost two orders of magnitude when analyzed by immunofluorescence of the surface expressed molecule. If rev was coexpressed in trans and a RRE element was present in cis only a slight induction was found for the surface molecule. However, if THY-1 was expressed as a secreted molecule, the induction by rev was much more prominent, supporting the above hypothesis. This can probably be explained by accumulation of secreted protein in the supernatant, which considerably amplifies the rev effect. If rev only induces a minor increase for surface molecules in general, induction of HIV envelope by rev cannot have the purpose of an increased surface abundance, but rather of an increased intracellular gp160 level. It is completely unclear at the moment why this should be the case.</p>
    <p>To test whether small subtotal elements of a gene are sufficient to restrict expression and render it rev-dependent rTHY1env:immunoglobulin fusion proteins were generated, in which only about one third of the total gene had the envelope codon usage. Expression levels of this construct were on an intermediate level, indicating that the rTHY-1env negative sequence element is not dominant over the immunoglobulin part. This fusion protein was not or only slightly rev-responsive, indicating that only genes almost completely suppressed can be rev-responsive.</p>
    <p>Another characteristic feature that was found in the codon frequency tables is a striking under representation of CpG triplets. In a comparative study of codon usage in E. coli, yeast, drosophila and primates it was shown that in a high number of analyzed primate genes the 8 least used codons contain all codons with the CpG dinucleotide sequence. Avoidance of codons containing this dinucleotide motif was also found in the sequence of other retroviruses. It seems plausible that the reason for under representation of CpG-bearing triplets has something to do with avoidance of gene silencing by methylation of CpG cytosines. The expected number of CpG dinucleotides for HIV as a whole is about one fifth that expected on the basis of the base composition. This might indicate that the possibility of high expression is restored, and that the gene in fact has to be highly expressed at some point during viral pathogenesis.</p>
    <p>The results presented herein clearly indicate that codon preference has a severe effect on protein levels, and suggest that translational elongation is controlling mammalian gene expression. However, other factors may play a role. First, abundance of not maximally loaded mRNA's in eukaryotic cells indicates that initiation is rate limiting for translation in at least some cases, since otherwise all transcripts would be completely covered by ribosomes. Furthermore, if ribosome stalling and subsequent mRNA degradation were the mechanism, suppression by rare codons could most likely not be reversed by any regulatory mechanism like the one presented herein. One possible explanation for the influence of both initiation and elongation on translational activity is that the rate of initiation, or access to ribosomes, is controlled in part by cues distributed throughout the RNA, such that the lentiviral codons predispose the RNA to accumulate in a pool of poorly initiated RNAs. However, this limitation need not be kinetic; for example, the choice of codons could influence the probability that a given translation product, once initiated, is properly completed. Under this mechanism, abundance of less favored codons would incur a significant cumulative probability of failure to complete the nascent polypeptide chain. The sequestered RNA would then be lent an improved rate of initiation by the action of rev. Since adenine residues are abundant in rev-responsive transcripts, it could be that RNA adenine methylation mediates this translational suppression.</p>
    <p>Detailed Procedures</p>
    <p>The following procedures were used in the above-described experiments.</p>
    <p>Sequence Analysis</p>
    <p>Sequence analyses employed the software developed by the University of Wisconsin Computer Group.</p>
    <p>Plasmid Constructions</p>
    <p>Plasmid constructions employed the following methods. Vectors and insert DNA was digested at a concentration of 0.5 g/10 l in the appropriate restriction buffer for 1-4 hours (total reaction volume approximately 30 l). Digested vector was treated with 10% (v/v) of 1 g/ml calf intestine alkaline phosphatase for 30 min prior to gel electrophoresis. Both vector and insert digests (5 to 10 l each) were run on a 1.5% low melting agarose gel with TAE buffer. Gel slices containing bands of interest were transferred into a 1.5 ml reaction tube, melted at 65 C. and directly added to the ligation without removal of the agarose. Ligations were typically done in a total volume of 25 l in 1Low Buffer 1Ligation Additions with 200-400 U of ligase, 1 l of vector, and 4 l of insert. When necessary, 5' overhanging ends were filled by adding 1/10 volume of 250 M dNTPs and 2-5 U of Klenow polymerase to heat inactivated or phenol extracted digests and incubating for approximately 20 min at room temperature. When necessary, 3' overhanging ends were filled by adding 1/10 volume of 2.5 mM dNTPs and 5-10 U of T4 DNA polymerase to heat inactivated or phenol extracted digests, followed by incubation at 37 C. for 30 min. The following buffers were used in these reactions: 10Low buffer (60 mM Tris HCl, pH 7.5, 60 mM MgCl<sub>2</sub>, 50 mM NaCl, 4 mg/ml BSA, 70 mM -mercaptoethanol, 0.02% NaN<sub>3</sub>); 10Medium buffer (60 mM Tris HCl, pH 7.5, 60 mM MgCl<sub>2</sub>, 50 mM NaCl, 4 mg/ml BSA, 70 mM -mercaptoethanol, 0.02% NaN<sub>3</sub>); 10High buffer (60 mM Tris HCl, pH 7.5, 60 mM MgCl<sub>2</sub>, 50 mM NaCl, 4 mg/ml BSA, 70 mM -mercaptoethanol, 0.02% NaN<sub>3</sub>); 10Ligation additions (1 mM ATP, 20 mM DTT, 1 mg/ml BSA, 10 mM spermidine); 50TAE (2 M Tris acetate, 50 mM EDTA).</p>
    <p>Oligonucleotide Synthesis and Purification</p>
    <p>Oligonucleotides were produced on a Milligen 8750 synthesizer (Millipore). The columns were eluted with 1 ml of 30% ammonium hydroxide, and the eluted oligonucleotides were deblocked at 55 C. for 6 to 12 hours. After deblockiong, 150 l of oligonucleotide were precipitated with 10volume of unsaturated n-butanol in 1.5 ml reaction tubes, followed by centrifugation at 15,000 rpm in a microfuge. The pellet was washed with 70% ethanol and resuspended in 50 l of H<sub>2</sub> O. The concentration was determined by measuring the optical density at 260 nm in a dilution of 1:333 (1 OD<sub>260</sub> =30 g/ml).</p>
    <p>The following oligonucleotides were used for construction of the synthetic gp120 gene (all sequences shown in this text are in 5' to 3' direction). oligo 1 forward (Nhe1): cgc ggg cta gcc acc gag aag ctg (SEQ ID NO:1).</p>
    <p>oligo 1: acc gag aag ctg tgg gtg acc gtg tac tac ggc gtg ccc gtg tgg aag ag ag gcc acc acc acc ctg ttc tgc gcc agc gac gcc aag gcg tac gac acc gag gtg cac aac gtg tgg gcc acc cag gcg tgc gtg ccc acc gac ccc aac ccc cag gag gtg gag ctc gtg aac gtg acc gag aac ttc aac at (SEQ ID NO:2).</p>
    <p>oligo 1 reverse: cca cca tgt tgt tct tcc aca tgt tga agt tct c (SEQ ID NO:3).</p>
    <p>oligo 2 forward: gac cga gaa ctt caa cat gtg gaa gaa caa cat (SEQ ID NO:4)</p>
    <p>oligo 2: tgg aag aac aac atg gtg gag cag atg cat gag gac atc atc agc ctg tgg gac cag agc ctg aag ccc tgc gtg aag ctg acc cc ctg tgc gtg acc tg aac tgc acc gac ctg agg aac acc acc aac acc aac ac agc acc gcc aac aac aac agc aac agc gag ggc acc atc aag ggc ggc gag atg (SEQ ID NO:5).</p>
    <p>oligo 2 reverse (Pst1): gtt gaa gct gca gtt ctt cat ctc gcc gcc ctt (SEQ ID NO:6).</p>
    <p>oligo 3 forward (Pst1): gaa gaa ctg cag ctt caa cat cac cac cag c (SEQ ID NO:7).</p>
    <p>oligo 3: aac atc acc acc agc atc cgc gac aag atg cag aag gag tac gcc ctg ctg tac aag ctg gat atc gtg agc atc gac aac gac agc acc agc tac cgc ctg atc tcc tgc aac acc agc gtg atc acc cag gcc tgc ccc aag atc agc ttc gag ccc atc ccc atc cac tac tgc gcc ccc gcc ggc ttc gcc (SEQ ID NO:8).</p>
    <p>oligo 3 reverse: gaa ctt ctt gtc ggc ggc gaa gcc ggc ggg (SEQ ID NO:9).</p>
    <p>oligo 4 forward: gcg ccc ccg ccg gct tcg cca tcc tga agt gca acg aca aga agt tc (SEQ ID NO:10)</p>
    <p>oligo 4: gcc gac aag aag ttc agc ggc aag ggc agc tgc aag aac gtg agc acc gtg cag tgc acc cac ggc atc cgg ccg gtg gtg agc acc cag ctc ctg ctg aac ggc agc ctg gcc gag gag gag gtg gtg atc cgc agc gag aac ttc acc gac aac gcc aag acc atc atc gtg cac ctg aat gag agc gtg cag atc (SEQ ID NO:11)</p>
    <p>oligo 4 reverse (Mlu1): agt tgg gac gcg tgc agt tga tct gca cgc tct c (SEQ ID NO:12).</p>
    <p>oligo 5 forward (Mlu1): gag agc gtg cag atc aac tgc acg cgt ccc (SEQ ID NO:13).</p>
    <p>oligo 5: aac tgc acg cgt ccc aac tac aac aag cgc aag cgc atc cac atc ggc ccc ggg cgc gcc ttc tac acc acc aag aac atc atc ggc acc atc ctc cag gcc cac tgc aac atc tct aga (SEQ ID NO:14).</p>
    <p>oligo 5 reverse: gtc gtt cca ctt ggc tct aga gat gtt gca (SEQ ID NO:15).</p>
    <p>oligo 6 forward: gca aca tct cta gag cca agt gga acg ac (SEQ ID NO:16).</p>
    <p>oligo 6: gcc aag tgg aac gac acc ctg cgc cag atc gtg agc aag ctg aag gag cag ttc aag aac aag acc atc gtg ttc ac cag agc agc ggc ggc gac ccc gag atc gtg atg cac agc ttc aac tgc ggc ggc (SEQ ID NO:17).</p>
    <p>oligo 6 reverse (EcoR1): gca gta gaa gaa ttc gcc gcc gca gtt ga (SEQ ID NO:18).</p>
    <p>oligo 7 forward (EcoR1): tca act gcg gcg gcg aat tct tct act gc (SEQ ID NO:19).</p>
    <p>oligo 7: ggc gaa ttc ttc tac tgc aac acc agc ccc ctg ttc aac agc acc tgg aac ggc aac aac acc tgg aac aac acc acc ggc agc aac aac aat att acc ctc cag tgc aag atc aag cag atc atc aac atg tgg cag gag gtg ggc aag gcc atg tac gcc ccc ccc atc gag ggc cag atc cgg tgc agc agc (SEQ ID NO:20).</p>
    <p>oligo 7 reverse: gca gac cgg tga tgt tgc tgc tgc acc gga tct ggc cct c (SEQ ID NO:21).</p>
    <p>oligo 8 forward: cga ggg cca gat ccg gtg cag cag caa cat cac cgg tct g (SEQ ID NO:22).</p>
    <p>oligo 8: aac atc acc ggt ctg ctg ctg acc cgc gac ggc ggc aag gac acc gac acc aac gac acc gaa atc ttc cgc ccc ggc ggc ggc gac atg cgc gac aac tgg aga tct gag ctg tac aag tac aag gtg gtg acg atc gag ccc ctg ggc gtg gcc ccc acc aag gcc aag cgc cgc gtg gtg cag cgc gag aag cgc (SEQ ID NO:23).</p>
    <p>oligo 8 reverse (Not1): cgc ggg cgg ccg ctt tag cgc ttc tcg cgc tgc acc ac (SEQ ID NO:24).</p>
    <p>The following oligonucleotides were used for the construction of the ratTHY-1env gene.</p>
    <p>oligo 1 forward (BamH1/Hind3): cgc ggg gga tcc aag ctt acc atg att cca gta ata agt (SEQ ID NO:25).</p>
    <p>oligo 1: atg aat cca gta ata agt ata aca tta tta tta agt gta tta caa atg agt aga gga caa aga gta ata agt tta aca gca tct tta gta aat caa aat ttg aga tta gat tgt aga cat gaa aat aat aca aat ttg cca ata caa cat gaa ttt tca tta acg (SEQ ID NO:26).</p>
    <p>oligo 1 reverse (EcoR1/Mlu1): cgc ggg gaa ttc acg cgt taa tga aaa ttc atg ttg (SEQ ID NO:27).</p>
    <p>oligo 2 forward (BamH1/Mlu1): cgc gga tcc acg cgt gaa aaa aaa aaa cat (SEQ ID NO:28).</p>
    <p>oligo 2: cgt gaa aaa aaa aaa cat gta tta agt gga aca tta gga gta cca gaa cat aca tat aga agt aga gta aat ttg ttt agt gat aga ttc ata aaa gta tta aca tta gca aat ttt aca aca aaa gat gaa gga gat tat atg tgt gag (SEQ ID NO:29).</p>
    <p>oligo 2 reverse (EcoR1/Sac1): cgc gaa ttc gag ctc aca cat ata atc tcc (SEQ ID NO:30).</p>
    <p>oligo 3 forward (BamH1/Sac1): cgc gga tcc gag ctc aga gta agt gga caa (SEQ ID NO:31).</p>
    <p>oligo 3: ctc aga gta agt gga caa aat cca aca agt agt aat aaa aca ata aat gta ata aga gat aaa tta gta aaa tgt ga gga ata agt tta tta gta caa aat aca agt tgg tta tta tta tta tta tta agt tta agt ttt tta caa gca aca gat ttt ata agt tta tga (SEQ ID NO:32).</p>
    <p>oligo 3 reverse (EcoR1/Not1 ): cgc gaa ttc gcg gcc gct tca taa act tat aaa atc (SEQ ID NO:33).</p>
    <p>Polymerase Chain Reaction</p>
    <p>Short, overlapping 15 to 25 mer oligonucleotides annealing at both ends were used to amplify the long oligonuclotides by polymerase chain reaction (PCR). Typical PCR conditions were: 35 cycles, 55 C. annealing temperature, 0.2 sec extension time. PCR products were gel purified, phenol extracted, and used in a subsequent PCR to generate longer fragments consisting of two adjacent small fragments. These longer fragments were cloned into a CDM7-derived plasmid containing a leader sequence of the CD5 surface molecule followed by a Nhe1/Pst1/Mlu1/EcoR1/BamH1 polylinker.</p>
    <p>The following solutions were used in these reactions: 10PCR buffer (500 mM KCl, 100 mM Tris HCl, pH 7.5, 8 mM MgCl<sub>2</sub>, 2 mM each dNTP). The final buffer was complemented with 10% DMSO to increase fidelity of the Taq polymerase.</p>
    <p>Small scale DNA Preparation</p>
    <p>Transformed bacteria were grown in 3 ml LB cultures for more than 6 hours or overnight. Approximately 1.5 ml of each culture was poured into 1.5 ml microfuge tubes, spun for 20 seconds to pellet cells and resuspended in 200 l of solution I. Subsequently 400 l of solution II and 300 l of solution III were added. The microfuge tubes were capped, mixed and spun for &gt;30 sec. Supernatants were transferred into fresh tubes and phenol extracted once. DNA was precipitated by filling the tubes with isopropanol, mixing, and spinning in a microfuge for &gt;2 min. The pellets were rinsed in 70% ethanol and resuspended in 50 l dH20 containing 10 l of RNAse A. The following media and solutions were used in these procedures: LB medium (1.0% NaCl, 0.5% yeast extract, 1.0% trypton); solution I (10 mM EDTA pH 8.0); solution II (0.2 M NaOH, 1.0% SDS); solution III (2.5 M KOAc, 2.5 M glacial aceatic acid); phenol (pH adjusted to 6.0, overlaid with TE); TE (10 mM Tris HCl, pH 7.5, 1 mM EDTA pH 8.0).</p>
    <p>Large Scale DNA Preparation</p>
    <p>One liter cultures of transformed bacteria were grown 24 to 36 hours (MC1061p3 transformed with pCDM derivatives) or 12 to 16 hours (MC1061 transformed with pUC derivatives) at 37 C. in either M9 bacterial medium (pCDM derivatives) or LB (pUC derivatives). Bacteria were spun down in 1 liter bottles using a Beckman J6 centrifuge at 4,200 rpm for 20 min. The pellet was resuspended in 40 ml of solution I. Subsequently, 80 ml of solution II and 40 ml of solution III were added and the bottles were shaken semivigorously until lumps of 2 to 3 mm size developed. The bottle was spun at 4,200 rpm for 5 min and the supernatant was poured through cheesecloth into a 250 ml bottle.</p>
    <p>Isopropanol was added to the top and the bottle was spun at 4,200 rpm for 10 min. The pellet was resuspended in 4.1 ml of solution I and added to 4.5 g of cesium chloride, 0.3 ml of 10 mg/ml ethidium bromide, and 0.1 ml of 1% Triton X100 solution. The tubes were spun in a Beckman J2 high speed centrifuge at 10,000 rpm for 5 min. The supernatant was transferred into Beckman Quick Seal ultracentrifuge tubes, which were then sealed and spun in a Beckman ultracentrifuge using a NVT90 fixed angle rotor at 80,000 rpm for &gt;2.5 hours. The band was extracted by visible light using a 1 ml syringe and 20 gauge needle. An equal volume of dH<sub>2</sub> O was added to the extracted material. DNA was extracted once with n-butanol saturated with 1 M sodium chloride, followed by addition of an equal volume of 10 M ammonium acetate/1 mM EDTA. The material was poured into a 13 ml snap tube which was tehn filled to the top with absolute ethanol, mixed, and spun in a Beckman J2 centrifuge at 10,000 rpm for 10 min. The pellet was rinsed with 70% ethanol and resuspended in 0.5 to 1 ml of H<sub>2</sub> O. The DNA concentration was determined by measuring the optical density at 260 nm in a dilution of 1:200 (1 OD<sub>260</sub> =50 g/ml).</p>
    <p>The following media and buffers were used in these procedures: M9 bacterial medium (10 g M9 salts, 10 g casamino acids (hydrolyzed), 10 ml M9 additions, 7.5 g/ml tetracycline (500 l of a 15 mg/ml stock solution), 12.5 g/ml ampicillin (125 l of a 10 mg/ml stock solution); M9 additions (10 mM CaCl<sub>2</sub>, 100 mM MgSO<sub>4</sub>, 200, g/ml thiamine, 70% glycerol); LB medium (1.0% NaCl, 0.5% yeast extract, 1.0% trypton); Solution I (10 mM EDTA pH 8.0); Solution II (0.2 M NaOH 1.0% SDS); Solution III (2.5 M KOAc 2.5 M HOAc)</p>
    <p>Sequencing</p>
    <p>Synthetic genes were sequenced by the Sanger dideoxynucleotide method. In brief, 20 to 50 g double-stranded plasmid DNA were denatured in 0.5 M NaOH for 5 min. Subsequently the DNA was precipitated with 1/10 volume of sodium acetate (pH 5.2) and 2 volumes of ethanol and centrifuged for 5 min. The pellet was washed with 70% ethanol and resuspended at a concentration of 1 g/l . The annealing reaction was carried out with 4 g of template DNA and 40 ng of primer in 1annealing buffer in a final volume of 10 l. The reaction was heated to 65 C. and slowly cooled to 37 C.</p>
    <p>In a separate tube 1 l of 0.1 M DTT, 2 l of labeling mix, 0.75 l of dH<sub>2</sub> O, 1 l of [<sup>35</sup> S] DATP (10 Ci), and 0.25 l of Sequenase (12 U/l) were added for each reaction. Five l of this mix were added to each annealed primer-template tube and incubated for 5 min at room temperature. For each labeling reaction 2.5 l of each of the 4 termination mixes were added on a Terasaki plate and prewarmed at 37 C. At the end of the incubation period 3.5 l of labeling reaction were added to each of the 4 termination mixes. After 5 min, 4 l of stop solution were added to each reaction and the Terasaki plate was incubated at 80 C. for 10 min in an oven. The sequencing reactions were run on 5% denaturing polyacrylamide gel. An acrylamide solution was prepared by adding 200 ml of 10TBE buffer and 957 ml of dH<sub>2</sub> O to 100 g of acrylamide:bisacrylamide (29:1). 5% polyacrylamide 46% urea and 1TBE gel was prepared by combining 38 ml of acrylamide solution and 28 g urea. Polymerization was initiated by the addition of 400 l of 10% ammonium peroxodisulfate and 60 l of TEMED. Gels were poured using silanized glass plates and sharktooth combs and run in 1TBE buffer at 60 to 100 W for 2 to 4 hours (depending on the region to be read). Gels were transferred to Whatman blotting paper, dried at 80 C. for about 1 hour, and exposed to x-ray film at room temperature. Typically exposure time was 12 hours. The following solutions were used in these procedures: 5Annealing buffer (200 mM Tris HCl, pH 7.5, 100 mM MgCl<sub>2</sub>, 250 mM NaCl); Labelling Mix (7.5 M each dCTP, dGTP, and dTTP); Termination Mixes (80 M each DNTP, 50 mM NaCl, 8 M ddNTP (one each)); Stop solution (95% formamide, 20 mM EDTA, 0.05% bromphenol blue, 0.05% xylencyanol); 5TBE (0.9 M Tris borate, 20 mM EDTA); Polyacrylamide solution (96.7 g polyacrylamide, 3.3 g bisacrylamide, 200 ml 1TBE, 957 ml dH<sub>2</sub> O).</p>
    <p>RNA Isolation</p>
    <p>Cytoplasmic RNA was isolated from calcium phosphate transfected 293T cells 36 hours post transfection and from vaccinia infected Hela cells 16 hours post infection essentially as described by Gilman. (Gilman Preparation of cytoplasmic RNA from tissue culture cells. In Current Protocols in Molecular Biology, Ausubel et al., eds., Wiley &amp; Sons, New York, 1992). Briefly, cells were lysed in 400 l lysis buffer, nuclei were spun out, and SDS and proteinase K were added to 0.2% and 0.2 mg/ml respectively. The cytoplasmic extracts were incubated at 37 C. for 20 min, phenol/chloroform extracted twice, and precipitated. The RNA was dissolved in 100 l buffer I and incubated at 37 C. for 20 min. The reaction was stopped by adding 25 l stop buffer and precipitated again.</p>
    <p>The following solutions were used in this procedure: Lysis Buffer (TRUSTEE containing with 50 mM Tris pH 8.0, 100 mM NaCl, 5 mM MgCl<sub>2</sub>, 0.5% NP40); Buffer I (TRUSTEE buffer with 10 mM MgCl<sub>2</sub>, 1 mM DTT, 0.5 U/l placental RNAse inhibitor, 0.1 U/l RNAse free DNAse I); Stop buffer (50 mM EDTA 1.5 M NaOAc 1.0% SDS).</p>
    <p>Slot Blot Analysis</p>
    <p>For slot blot analysis 10 g of cytoplasmic RNA was dissolved in 50 l dH<sub>2</sub> O to which 150 l of 10SSC/18% formaldehyde were added. The solubilized RNA was then incubated at 65 C. for 15 min and spotted onto with a slot blot apparatus. Radioactively labeled probes of 1.5 kb gp120IIIb and syngp120mn fragments were used for hybridization. Each of the two fragments was random labeled in a 50 l reaction with 10 l of 5oligo-labeling buffer, 8 l of 2.5 mg/ml BSA, 4 l of [<sup>32</sup> P]-dCTP (20 uCi/l; 6000 Ci/mmol), and 5 U of Klenow fragment. After 1 to 3 hours incubation at 37 C. 100 l of TRUSTEE were added and unincorporated [<sup>32</sup> P]-dCTP was eliminated using G50 spin column. Activity was measured in a Beckman beta-counter, and equal specific activities were used for hybridization. Membranes were pre-hybridized for 2 hours and hybridized for 12 to 24 hours at 42 C. with 0.510<sup>6</sup> cpm probe per ml hybridization fluid. The membrane was washed twice (5 min) with washing buffer I at room temperature, for one hour in washing buffer II at 65 C., and then exposed to x-ray film. Similar results were obtained using a 1.1 kb Not1 /Sfi1 fragment of pCDM7 containing the 3 untranslated region. Control hybridizations were done in parallel with a random-labeled human beta-actin probe. RNA expression was quantitated by scanning the hybridized nitrocellulose membranes with a Magnetic Dynamics phosphorimager.</p>
    <p>The following solutions were used in this procedure: 5Oligo-labeling buffer (250 mM Tris HCl, pH 8.0, 25 mM MgCl<sub>2</sub>, 5 mM -mercaptoethanol, 2 mM DATP, 2 mM dGTP, mM dTTP, 1 M Hepes pH 6.6, 1 mg/ml hexanucleotides [dNTP]6); Hybridization Solution (0.05 M sodium phosphate, 250 mM NaCl, 7% SDS, 1 mM EDTA, 5% dextrane sulfate, 50% formamide, 100 g/ml denatured salmon sperm DNA); Washing buffer I (2SSC, 0.1% SDS); Washing buffer II (0.5SSC, 0.1% SDS); 20SSC (3 M NaCl, 0.3 M Na<sub>3</sub> citrate, pH adjusted to 7.0).</p>
    <p>Vaccinia Recombination</p>
    <p>Vaccinia recombination used a modification of the of the method described by Romeo and Seed (Romeo and Seed, Cell, 64: 1037, 1991). Briefly, CV1 cells at 70 to 90% confluency were infected with 1 to 3 l of a wild-type vaccinia stock WR (210<sup>8</sup> pfu/ml) for 1 hour in culture medium without calf serum. After 24 hours, the cells were transfected by calcium phosphate with 25 g TKG plasmid DNA per dish. After an additional 24 to 48 hours the cells were scraped off the plate, spun down, and resuspended in a volume of 1 ml. After 3 freeze/thaw cycles trypsin was added to 0.05 mg/ml and lysates were incubated for 20 min. A dilution series of 10, 1 and 0.1 l of this lysate was used to infect small dishes (6 cm) of CV1 cells, that had been pretreated with 12.5 g/ml mycophenolic acid, 0.25 mg/ml xanthin and 1.36 mg/ml hypoxanthine for 6 hours.</p>
    <p>Infected cells were cultured for 2 to 3 days, and subsequently stained with the monoclonal antibody NEA9301 against gp120 and an alkaline phosphatase conjugated secondary antibody. Cells were incubated with 0.33 mg/ml NBT and 0.16 mg/ml BCIP in AP-buffer and finally overlaid with 1% agarose in PBS. Positive plaques were picked and resuspended in 100 l Tris pH 9.0. The plaque purification was repeated once. To produce high titer stocks the infection was slowly scaled up. Finally, one large plate of Hela cells was infected with half of the virus of the previous round. Infected cells were detached in 3 ml of PBS, lysed with a Dounce homogenizer and cleared from larger debris by centrifugation. VPE-8 recombinant vaccinia stocks were kindly provided by the AIDS repository, Rockville, Md., and express HIV-1 IIIB gp120 under the 7.5 mixed early/late promoter (Ear1 et al., J. Virol., 65:31, 1991). In all experiments with recombinant vaccina cells were infected at a multiplicity of infection of at least 10.</p>
    <p>The following solution was used in this procedure: AP buffer (100 mM Tris HCl, pH 9.5, 100 mM NaCl, 5 mM MgCl<sub>2</sub>)</p>
    <p>Cell Culture</p>
    <p>The monkey kidney carcinoma cell lines CV1 and Cos7, the human kidney carcinoma cell line 293T, and the human cervix carcinoma cell line Hela were obtained from the American Tissue Typing Collection and were maintained in supplemented IMDM. They were kept on 10 cm tissue culture plates and typically split 1:5 to 1:20 every 3 to 4 days. The following medium was used in this procedure: Supplemented IMDM (90% Iscove's modified Dulbecco Medium, 10% calf serum, iron-complemented, heat inactivated 30 min 56 C., 0.3 mg/ml L-glutamine, 25 82 g/ml gentamycin 0.5 mM -mercaptoethanol (pH adjusted with 5 M NaOH, 0.5 ml)).</p>
    <p>Transfection</p>
    <p>Calcium phosphate transfection of 293T cells was performed by slowly adding and under vortexing 10 g plasmid DNA in 250 l 0.25 M CaCl<sub>2</sub> to the same volume of 2HEBS buffer while vortexing. After incubation for 10 to 30 min at room temperature the DNA precipitate was added to a small dish of 50 to 70% confluent cells. In cotransfection experiments with rev, cells were transfected with 10 g gp120IIIb, gp120IIIbrre, syngp120mnrre or rTHY-1enveg1rre and 10 g of pCMVrev or CDM7 plasmid DNA.</p>
    <p>The following solutions were used in this procedure: 2HEBS buffer (280 mM NaCl, 10 mM KCl, 1.5 mM sterile filtered); 0.25 mM CaCl<sub>2</sub> (autoclaved).</p>
    <p>Immunoprecipitation</p>
    <p>After 48 to 60 hours medium was exchanged and cells were incubated for additional 12 hours in Cys/Met-free medium containing 200 Ci of <sup>35</sup> S-translabel. Supernatants were harvested and spun for 15 min at 3000 rpm to remove debris. After addition of protease inhibitors leupeptin, aprotinin and PMSF to 2.5 g/ml, 50 g/ml, 100 g/ml respectively, 1 ml of supernatant was incubated with either 10 l of packed protein A sepharose alone (rTHY-1enveg1rre) or with protein A sepharose and 3 g of a purified CD4/immunoglobulin fusion protein (kindly provided by Behring) (all gp120 constructs) at 4 C. for 12 hours on a rotator. Subsequently the protein A beads were washed 5 times for 5 to 15 min each time. After the final wash 10 l of loading buffer containing was added, samples were boiled for 3 min and applied on 7% (all gp120 constructs) or 10% (rTHY-1enveg1rre) SDS polyacrylamide gels (TRIS pH 8.8 buffer in the resolving, TRIS pH 6.8 buffer in the stacking gel, TRIS-glycin running buffer, Maniatis et al., supra 1989). Gels were fixed in 10% acetic acid and 10% methanol, incubated with Amplify for 20 min, dried and exposed for 12 hours.</p>
    <p>The following buffers and solutions were used in this procedure: Wash buffer (100 mM Tris, pH 7.5, 150 mM NaCl, 5 mM CaCl<sub>2</sub>, 1% NP-40); 5Running Buffer (125 mM Tris, 1.25 M Glycin, 0.5% SDS); Loading buffer (10% glycerol, 4% SDS, 4% -mercaptoethanol, 0.02% bromphenol blue).</p>
    <p>Immunofluorescence</p>
    <p>293T cells were transfected by calcium phosphate coprecipitation and analyzed for surface THY-1 expression after 3 days. After detachment with 1 mM EDTA/PBS, cells were stained with the monoclonal antibody OX-7 in a dilution of 1:250 at 4 C. for 20 min, washed with PBS and subsequently incubated with a 1:500 dilution of a FITC-conjugated goat anti-mouse immunoglobulin antiserum. Cells were washed again, resuspended in 0.5 ml of a fixing solution, and analyzed on a EPICS XL cytofluorometer (Coulter).</p>
    <p>The following solutions were used in this procedure: PBS (137 mM NaCl, 2.7 mM KCl, 4.3 mM Na<sub>2</sub> HPO<sub>4</sub>, 1.4 mM KH<sub>2</sub> PO<sub>4</sub>, pH adjusted to 7.4); Fixing solution (2% formaldehyde in PBS).</p>
    <p>ELISA</p>
    <p>The concentration of gp120 in culture supernatants was determined using CD4-coated ELISA plates and goat anti-gp120 antisera in the soluble phase. Supernatants of 293T cells transfected by calcium phosphate were harvested after 4 days, spun at 3000 rpm for 10 min to remove debris and incubated for 12 hours at 4 C. on the plates. After 6 washes with PBS 100 l of goat anti-gp120 antisera diluted 1:200 were added for 2 hours. The plates were washed again and incubated for 2 hours with a peroxidase-conjugated rabbit anti-goat IgG antiserum 1:1000. Subsequently the plates were washed and incubated for 30 min with 100 l of substrate solution containing 2 mg/ml o-phenylenediamine in sodium citrate buffer. The reaction was finally stopped with 100 l of 4 M sulfuric acid. Plates were read at 490 nm with a Coulter microplate reader. Purified recombinant gp120IIIb was used as a control. The following buffers and solutions were used in this procedure: Wash buffer (0.1% NP40 in PBS); Substrate solution (2 mg/ml o-phenylenediamine in sodium citrate buffer).</p>
    <heading>EXAMPLE 2</heading> <p>A Synthetic Green Fluorescent Protein Gene</p>
    <p>The efficacy of codon replacement for gp120 suggests that replacing non-preferred codons with less preferred codons or preferred codons (and replacing less preferred codons with preferred codons) will increase expression in mammalian cells of other proteins, e.g., other eukaryotic proteins.</p>
    <p>The green fluorescent protein (GFP) of the jellyfish Aequorea victoria (Ward, Photochem. Photobiol. 4:1, 1979; Prasher et al., Gene 111:229, 1992; Cody et al., Biochem. 32:1212, 1993) has attracted attention recently for its possible utility as a marker or reporter for transfection and lineage studies (Chalfie et al., Science 263:802, 1994).</p>
    <p>Examination of a codon usage table constructed from the native coding sequence of GFP showed that the GFP codons favored either A or U in the third position. The bias in this case favors A less than does the bias of gp120, but is substantial. A synthetic gene was created in which the natural GFP sequence was re-engineered in much the same manner as for gp120 (FIG. 11; SEQ ID NO:40). In addition, the translation initiation sequence of GFP was replaced with sequences corresponding to the translational initiation consensus. The expression of the resulting protein was contrasted with that of the wild type sequence, similarly engineered to bear an optimized translational initiation consensus (FIG. 10B and FIG. 10C). In addition, the effect of inclusion of the mutation Ser 65Thr, reported to improve excitation efficiency of GFP at 490 nm and hence preferred for fluorescence microscopy (Heim et al., Nature 373:663, 1995), was examined (FIG. 10D). Codon engineering conferred a significant increase in expression efficiency (an concomitant percentage of cells apparently positive for transfection), and the combination of the Ser 65Thr mutation and codon optimization resulted in a DNA segment encoding a highly visible mammalian marker protein (FIG. 10D).</p>
    <p>The above-described synthetic green fluorescent protein coding sequence was assembled in a similar manner as for gp120 from six fragments of approximately 120 bp each, using a strategy for assembly that relied on the ability of the restriction enzymes BsaI and BbsI to cleave outside of their recognition sequence. Long oligonucleotides were synthesized which contained portions of the coding sequence for GFP embedded in flanking sequences encoding EcoRI and BsaI at one end, and BamHI and BbsI at the other end. Thus, each oligonucleotide has the configuration EcoRI/BsaI/GFP fragment/BbsI/BamHI. The restriction site ends generated by the BsaI and BbsI sites were designed to yield compatible ends that could be used to join adjacent GFP fragments. Each of the compatible ends were designed to be unique and non-selfcomplementary. The crude synthetic DNA segments were amplified by PCR, inserted between EcoRI and BamHI in pUC9, and sequenced. Subsequently the intact coding sequence was assembled in a six fragment ligation, using insert fragments prepared with BsaI and BbsI. Two of six plasmids resulting from the ligation bore an insert of correct size, and one contained the desired full length sequence. Mutation of Ser65 to Thr was accomplished by standard PCR based mutagenesis, using a primer that overlapped a unique BssSI site in the synthetic GFP.</p>
    <p>Codon Optimization as a Strategy for Improved Expression in Mammalian Cells</p>
    <p>The data presented here suggest that coding sequence re-engineering may have general utility for the improvement of expression of mammalian and non-mammalian eukaryotic genes in mammalian cells. The results obtained here with three unrelated proteins: HIV gp120, the rat cell surface antigen Thy-1 and green fluorescent protein from Aequorea victoria, and human Factor VIII (see below) suggest that codon optimization may prove to be a fruitful strategy for improving the expression in mammalian cells of a wide variety of eukaryotic genes.</p>
    <heading>EXAMPLE III</heading> <p>Design of a Codon-Optimized Gene Expressing Human Factor VIII Lacking the Central B Domain</p>
    <p>A synthetic gene was designed that encodes mature human Factor VIII lacking amino acid residues 760 to 1639, inclusive (residues 779 to 1658, inclusive, of the precursor). The synthetic gene was created by choosing codons corresponding to those favored by highly expressed human genes. Some deviation from strict adherence to the favored residue pattern was made to allow unique restriction enzyme cleavage sites to be introduced throughout the gene to facilitate future manipulations. For preparation of the synthetic gene the sequence was then divided into 28 segments of 150 basepairs, and a 29th segment of 161 basepairs.</p>
    <p>The a synthetic gene expressing human Factor VIII lacking the central B domain was constructed as follows. Twenty-nine pairs of template oligonucleotides (see below) were synthesized. The 5' template oligos were 105 bases long and the 3' oligos were 104 bases long (except for the last 3' oligo, which was 125 residues long). The template oligos were designed so that each annealing pair composed of one 5' oligo and one 3' oligo, created a 19 basepair double-stranded regions.</p>
    <p>To facilitate the PCR and subsequent manipulations, the 5' ends of the oligo pairs were designed to be invariant over the first 18 residues, allowing a common pair of PCR primers to be used for amplification, and allowing the same PCR conditions to be used for all pairs. The first 18 residues of each 5' member of the template pair were cgc gaa ttc gga aga ccc (SEQ ID NO:110) and the first 18 residues of each 3' member of the template pair were: ggg gat cct cac gtc tca (SEQ ID NO:43).</p>
    <p>Pairs of oligos were annealed and then extended and amplified by PCR in a reaction mixture as follows: templates were annealed at 200 g/ml each in PCR buffer (10 mM Tris-HCl, 1.5 mM MgCl<sub>2</sub>, 50 mM KCl, 100 g/ml gelatin, pH 8.3). The PCR reactions contained 2 ng of the annealed template oligos, 0.5 g of each of the two 18-mer primers (described below), 200 M of each of the deoxynucleoside triphosphates, 10% by volume of DMSO and PCR buffer as supplied by Boehringer Mannheim Biochemicals, in a final volume of 50 l. After the addition of Taq polymerase (2.5 units, 0.5 l; Boehringer Mannheim Biochemicals) amplifications were conducted on a Perkin-Elmer Thermal Cycler for 25 cycles (94 C. for 30 sec, 55 C. for 30 sec, and 72 C. for 30 sec). The final cycle was followed by a 10 minute extension at 72 C.</p>
    <p>The amplified fragments were digested with EcoRI and BamHI (cleaving at the 5' and 3' ends of the fragments respectively) and ligated to a pUC9 derivative cut with EcoRI and BamHI.</p>
    <p>Individual clones were sequenced and a collection of plasmids corresponding to the entire desired sequence was identified. The clones were then assembled by multifragment ligation taking advantage of restriction sites at the 3' ends of the PCR primers, immediately adjacent to the amplified sequence. The 5' PCR primer contained a BbsI site, and the 3' PCR primer contained a BsmBI site, positioned so that cleavage by the respective enzymes preceded the first nucleotide of the amplified portion and left a 4 base 5' overhang created by the first 4 bases of the amplified portion. Simultaneous digestion with BbsI and BsmBI thus liberated the amplified portion with unique 4 base 5' overhangs at each end which contained none of the primer sequences. In general these overhangs were not self-complementary, allowing multifragment ligation reactions to produce the desired product with high efficiency. The unique portion of the first 28 amplified oligonucleotide pairs was thereby 154 basepairs, and after digestion each gave rise to a 150 bp fragment with unique ends. The first and last fragments were not manipulated in this manner, however, since they had other restriction sites designed into them to facilitate insertion of the assembled sequence into an appropriate mammalian expression vector. The actual assembly process proceded as follows.</p>
    <p>Assembly of the Synthetic Factor VIII Gene</p>
    <p>Step 1: 29 Fragments Assembled to Form 10 Fragments.</p>
    <p>The 29 pairs of oligonucleotides, which formed segments 1 to 29 when base-paired, are described below.</p>
    <p>Plasmids carrying segments 1, 5, 9, 12, 16, 20, 24 and 27 were digested with EcoR1and BsmBI and the 170 bp fragments were isolated; plasmids bearing segments 2, 3, 6, 7, 10, 13, 17, 18, 21, 25, and 28 were digested with BbsI and BsmBI and the 170 bp fragments were isolated; and plasmids bearing segments 4, 8, 11, 14, 19, 22, 26 and 29 were digested with EcoRI and BbsI and the 2440 bp vector fragment was isolated. Fragments bearing segments 1, 2, 3 and 4 were then ligated to generate segment "A"; fragments bearing segments 5, 6, 7 and 8 were ligated to generate segment "B"; fragments bearing segments 9, 10 and 11 were ligated to generate segment "C"; fragments bearing segments 12, 13, and 14 were ligated to generate segment "D"; fragments bearing segments 16, 17, 18 and 19 were ligated to generate segment "F"; fragments bearing segments 20, 21 and 22 were ligated to generate segment "G"; fragments bearing segments 24, 25 and 26 were ligated to generate segment "I"; and fragments bearing segments 27, 28 and 29 were ligated to generate segment "J".</p>
    <p>Step 2: Assembly of the 10 Resulting Fragments from Step 1 to Three Fragments.</p>
    <p>Plasmids carrying the segments "A", "D" and "G" were digested with EcoRI and BsmBI, plasmids carrying the segments B, 15, 23, and I were digested with BbsI and BsmBI, and plasmids carrying the segments C, F, and J were digested with EcoRI and BbsI. Fragments bearing segments A, B, and C were ligated to generate segment "K"; fragments bearing segments D, 15, and F were ligated to generate segment "O"; and fragments bearing segments G, 23, I, and J were ligated to generate segment "P".</p>
    <p>Step 3: Assembly of the Final Three Pieces.</p>
    <p>The plasmid bearing segment K was digested with EcoRI and BsmBI, the plasmid bearing segment O was digested with BbsI and BsmBI, and the plasid bearing segment P was digested with EcoRI and BbsI. The three resulting fragments were ligated to generate segments.</p>
    <p>Step 4: Insertion of the Synthetic Gene in a Mammalian Expression Vector.</p>
    <p>The plasmid bearing segment S was digested with NheI and NotI and inserted between NheI and EagI sites of plasmid CD5lNEg1to generate plasmid cd5lsf8b-.</p>
    <p>Sequencing and Correction of the Synthetic Factor VIII Gene</p>
    <p>After assembly of the synthetic gene it was discovered that there were two undesired residues encoded in the sequence. One was an Arg residue at 749, which is present in the GenBank sequence entry originating from Genentech but is not in the sequence reported by Genentech in the literature. The other was an Ala residue at 146, which should have been Pro. This mutation arose at an unidentified step subsequent to the sequencing of the 29 constituent fragments. The Pro749Arg mutation was corrected by incorporating the desired change in a PCR primer (ctg ctt ctg acg cgt gct ggg gtg gcg gga gtt; SEQ ID NO:44) that included the MluI site at position 2335 of the sequence below (sequence of HindIII to NotI segment) and amplifying between that primer and a primer (ctg ctg aaa gtc tcc agc tgc; SEQ ID NO:44) 5' to the SgrAI site at 2225. The SgrAI to MluI fragment was then inserted into the expression vector at the cognate sites in the vector, and the resulting correct sequence change verified by sequencing. The Pro146Ala mutation was corrected by incorporating the desired sequence change in an oligonucleotide (ggc agg tgc tta agg aga acg gcc cta tgg cca; SEQ ID NO:46) bearing the AflII site at residue 504, and amplifying the fragment resulting from PCR reaction between that oligo and the primer having sequence cgt tgt tct tca tac gcg tct ggg gct cct cgg ggc (SEQ ID NO:109), cutting the resulting PCR fragment with AflII and AvrII at (residue 989), inserting the corrected fragment into the expression vector and confirming the construction by sequencing.</p>
    <p>Construction of a Matched Native Gene Expressing Human Factor VIII Lacking the Central B Domain</p>
    <p>A matched Factor VIII B domain deletion expression plasmid having the native codon sequence was constructed by introducing NheI at the 5' end of the mature coding sequence using primer cgc caa ggg cta gcc gcc acc aga aga tac tac ctg ggt (SEQ ID NO:47), amplifying between that primer and the primer att cgt agt tgg ggt tcc tct gga cag (corresponding to residues 1067 to 1093 of the sequence shown below), cutting with NheI and AflII (residue 345 in the sequence shown below) and inserting the resulting fragment into an appropriately cleaved plasmid bearing native Factor VIII. The B domain deletion was created by overlap PCR using ctg tat ttg atg aga acc g, (corresponding to residues 1813 to 1831 below) and caa gac tgg tgg ggt ggc att aaa ttg ctt t (SEQ ID NO:48) (2342 to 2372 on complement below) for the 5' end of the overlap, and aat gcc acc cca cca gtc ttg aaa cgc ca (SEQ ID NO:49) (2352 to 2380 on sequence below) and cat ctg gat att gca ggg ag (SEQ ID NO:50) (3145 to 3164). The products of the two individual PCR reactions were then mixed and reamplified by use of the outermost primers, the resulting fragment cleaved by Asp718 (KpnI isoschizomer, 1837 on sequence below) and PflMI (3100 on sequence below), and inserted into the appropriately cleaved expression plasmid bearing native Factor VIII.</p>
    <p>The complete sequence (SEQ ID NO:41) of the native human factor VIII gene deleted for the central B region is presented in FIG. 12. The complete sequence (SEQ ID NO:42) of the synthetic Factor VIII gene deleted for the central B region is presented in FIG. 13.</p>
    <p>Preparation and Assay of Expression Plasmids</p>
    <p>Two independent plasmid isolates of the native, and four independent isolates of the synthetic Factor VIII expression plasmid were separately propagated in bacteria and their DNA prepared by CsCl buoyant density centrifugation followed by phenol extraction. Analysis of the supernatants of COS cells transfected with the plasmids showed that the synthetic gene gave rise to approximately four times as much Factor VIII as did the native gene.</p>
    <p>COS cells were then transfected with 5 g of each factor VIII construct per 6 cm dish using the DEAE-dextran method. At 72 hours post-transfection, 4 ml of fresh medium containing 10% calf serum was added to each plated. A sample of media was taken from each plate 12 hr later. Samples were tested by ELISA using mouse anti-human factor VIII light chain monoclonal antibody and peroxidase-conjugated goat anti-human factor VIII polyclonal antibody. Purified human plasma factor VIII was used as a standard. Cells transfected with the synthetic Factor VIII gene construct expressed 13820.2 ng/ml (equivalent ng/ml non-deleted Factor VIII) of Factor VIII (n=4) while the cells transfected with the native Factor VIII gene expressed 33.50.7 ng/ml (equivalent ng/ml non-deleted Factor VIII) of Factor VIII (n=2).</p>
    <p>The following template oligonucleotides were used for construction of the synthetic Factor VIII gene.</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">                    r1 bbs  1 for (gcta)  cgc gaa ttc gga aga ccc gct agc cgc cac         1 r1  ccg ccg cta cta cct ggg cgc cgt gga gct  gtc ctg gga cta cat gca gag cga cct ggg  cga gct ccc cgt gga (SEQ ID NO:51)   - ggg gat cct cac gtc tca ggt ttt ctt gta         1 bam  cac cac gct ggt gtt gaa ggg gaa gct ctt  ggg cac gcg ggg ggg gaa gcg ggc gtc cac  ggg gag ctc gcc ca  (SEQ ID NO:52)   -                     r1 bbs  2 for (aacc)  cgc gaa ttc gga aga ccc aac cct gtt cgt         2 r1  gga gtt cac cga cca cct gtt caa cat tgc  caa gcc gcg ccc ccc ctg gat ggg cct gct  ggg ccc cac cat cca (SEQ ID NO:53)   - ggg gat cct cac gtc tca gtg cag gct gac         2 bam  ggg gtg gct ggc cat gtt ctt cag ggt gat  cac cac ggt gtc gta cac ctc ggc ctg gat  ggt ggg gcc cag ca  (SEQ ID NO:54)   -                     r1 bbs  3 for (gcac)  cgc gaa ttc gga aga ccc gca cgc cgt ggg         3 r1  cgt gag cta ctg gaa ggc cag cga ggg cgc  cga gta cga cga cca gac gtc cca gcg cga  gaa gga gga cga caa (SEQ ID NO:55)   - ggg gat cct cac gtc tca gct ggc cat agg         3 bam  gcc gtt ctc ctt aag cac ctg cca cac gta  ggt gtg gct ccc ccc cgg gaa cac ctt gtc  gtc ctc ctt ctc gc  (SEQ ID NO:56)   -                     r1 bbs  4 for (cagc)  cgc gaa ttc gga aga ccc cag cga ccc cct         4 r1  gtg cct gac cta cag cta cct gag cca cgt  gga cct ggt gaa gga tct gaa cag cgg gct  gat cgg cgc cct gct (SEQ ID NO:57)   - ggg gat cct cac gtc tca gaa cag cag gat         4 bam  gaa ctt gtg cag ggt ctg ggt ttt ctc ctt  ggc cag gct gcc ctc gcg aca cac cag cag  ggc gcc gat cag cc  (SEQ ID NO:58)   -                     r1 bbs  5 for (gttc)  cgc gaa ttc gga aga ccc gtt cgc cgt gtt         5 r1  cga cga ggg gaa gag ctg gca cag cga gac  taa gaa cag cct gat gca gga ccg cga cgc  cgc cag cgc ccg cgc (SEQ ID NO:59)   - ggg gat cct cac gtc tca gtg gca gcc gat         5 bam  cag gcc ggg cag gct gcg gtt cac gta gcc  gtt aac ggt gtg cat ctt ggg cca ggc gcg  ggc gct ggc ggc gt  (SEQ ID NO:60)   -                     r1 bbs  6 for (ccac)  cgc gaa ttc gga aga ccc cca ccg caa gag         6 r1  cgt gta ctg gca cgt cat cgg cat ggg cac  cac ccc tga ggt gca cag cat ctt cct gga  ggg cca cac ctt cct (SEQ ID NO:61)   - ggg gat cct cac gtc tca cag ggt ctg ggc         6 bam  agt cag gaa ggt gat ggg gct gat ctc cag  gct ggc ctg gcg gtg gtt gcg cac cag gaa  ggt gtg gcc ctc ca  (SEQ ID NO:62)   -                     r1 bbs  7 for (cctg)  cgc gaa ttc gga aga ccc cct gct gat gga         7 r1  cct agg cca gtt cct gct gtt ctg cca cat  cag cag cca cca gca cga cgg cat gga ggc  tta cgt gaa ggt gga (SEQ ID NO:63)   - ggg gat cct cac gtc tca gtc gtc gtc gta         7 bam  gtc ctc ggc ctc ctc gtt gtt ctt cat gcg  cag ctg ggg ctc ctc ggg gca gct gtc cac  ctt cac gta agc ct  (SEQ ID NO:64)   -                     r1 bbs  8 for (cgac)  cgc gaa ttc gga aga ccc cga cct gac cga         8 r1  cag cga gat gga tgt cgt acg ctt cga cga  cga caa cag ccc cag ctt cat cca gat ccg  cag cgt ggc caa gaa (SEQ ID NO:65)   - ggg gat cct cac gtc tca tac tag cgg ggc         8 bam  gta gtc cca gtc ctc ctc ctc ggc ggc gat  gta gtg cac cca ggt ctt agg gtg ctt ctt  ggc cac gct gcg ga  (SEQ ID NO:66)   -                     r1 bbs  9 for (agta)  cgc gaa ttc gga aga ccc agt act ggc ccc         9 r1  cga cga ccg cag cta caa gag cca gta cct  gaa caa cgg ccc cca gcg cat cgg ccg caa  gta caa gaa ggt gcg (SEQ ID NO:67)   - ggg gat cct cac gtc tca gag gat gcc gga         9 bam  ctc gtg ctg gat ggc ctc gcg ggt ctt gaa  agt ctc gtc ggt gta ggc cat gaa gcg cac  ctt ctt gta ctt gc  (SEQ ID NO:68)   -                     r1 bbs 10 for (cctc)  cgc gaa ttc gga aga ccc cct cgg ccc cct        10 r1  gct gta cgg cga ggt ggg cga cac cct gct  gat cat ctt caa gaa cca ggc cag cag gcc  cta caa cat cta ccc (SEQ ID NO:69)   - ggg gat cct cac gtc tca ctt cag gtg ctt        10 bam  cac gcc ctt ggg cag gcg gcg gct gta cag  ggg gcg cac gtc ggt gat gcc gtg ggg gta  gat gtt gta ggg cc (SEQ ID NO:70)   -                     r1 bbs 11 for (gaag)  cgc gaa ttc gga aga ccc gaa gga ctt ccc        11 r1  cat cct gcc cgg cga gat ctt caa gta caa  gtg gac cgt gac cgt gga gga cgg ccc cac  caa gag cga ccc ccg (SEQ ID NO:71)   - ggg gat cct cac gtc tca gcc gat cag tcc        11 bam  gga ggc cag gtc gcg ctc cat gtt cac gaa  gct gct gta gta gcg ggt cag gca gcg ggg  gtc gct ctt ggt gg  (SEQ ID NO:72)   -                     r1 bbs 12 for (cggc)  cgc gaa ttc gga aga ccc cgg ccc cct gct        12 r1  gat ctg cta caa gga gag cgt gga cca gcg  cgg caa cca gat cat gag cga caa gcg caa  cgt gat cct gtt cag (SEQ ID NO:73)   - ggg gat cct cac gtc tca agc ggg gtt ggg        12 bam  cag gaa gcg ctg gat gtt ctc ggt cag ata  cca gct gcg gtt ctc gtc gaa cac gct gaa  cag gat cac gtt gc (SEQ ID NO:74)   -                     r1 bbs 13 for (cgct)  cgc gaa ttc gga aga ccc cgc tgg cgt gca        13 r1  gct gga aga tcc cga gtt cca ggc cag caa  cat cat gca cag cat caa cgg cta cgt gtt  cga cag cct gca gct (SEQ ID NO:75)   - ggg gat cct cac gtc tca cag gaa gtc ggt        13 bam  ctg ggc gcc gat gct cag gat gta cca gta  ggc cac ctc atg cag gca cac gct cag ctg  cag gct gtc gaa ca  (SEQ ID NO:76)   -                     r1 bbs 14 for (cctg)  cgc gaa ttc gga aga ccc cct gag cgt gtt        14 r1  ctt ctc cgg gta tac ctt caa gca caa gat  ggt gta cga gga cac cct gac cct gtt ccc  ctt ctc cgg cga gac (SEQ ID NO:77)   - ggg gat cct cac gtc tca gtt gcg gaa gtc        14 bam  gct gtt gtg gca gcc cag aat cca cag gcc  ggg gtt ctc cat aga cat gaa cac agt ctc  gcc gga gaa ggg ga  (SEQ ID NO:78)   -                     r1 bbs 15 for (caac)  cgc gaa ttc gga aga ccc caa ccg cgg cat        15 r1  gac tgc cct gct gaa agt ctc cag ctg cga  caa gaa cac cgg cga cta cta cga gga cag  cta cga gga cat ctc (SEQ ID NO:79)   - ggg gat cct cac gtc tca gcg gtg gcg gga        15 bam  gtt ttg gga gaa gga gcg ggg ctc gat ggc  gtt gtt ctt gga cag cag gta ggc gga gat  gtc ctc gta gct gt  (SEQ ID NO:80)   -                     r1 bbs 16 fgr (ccgc)  cgc gaa ttc gga aga ccc ccg cag cac gcg        16 r1  tca gaa gca gtt caa cgc cac ccc ccc cgt  gct gaa gcg cca cca gcg cga gat cac ccg  cac cac cct gca aag (SEQ ID NO:81)   - ggg gat cct cac gtc tca gat gtc gaa gtc        16 bam  ctc ctt ctt cat ctc cac gct gat ggt gtc  gtc gta gtc gat ctc ctc ctg gtc gct ttg  cag ggt ggt gcg gg  (SEQ ID NO:82)   -                     r1 bbs 17 for (catc)  cgc gaa ttc gga aga ccc cat cta cga cga        17 r1  gga cga gaa cca gag ccc ccg ctc ctt cca  aaa gaa aac ccg cca cta ctt cat cgc cgc  cgt gga gcg cct gtg (SEQ ID NO:83)   - ggg gat cct cac gtc tca ctg ggg cac gct        17 bam  gcc gct ctg ggc gcg gtt gcg cag gac gtg  ggg gct gct gct cat gcc gta gtc cca cag  gcg ctc cac ggc gg  (SEQ ID NO:84)   -                     r1 bbs 18 for (ccag)  cgc gaa ttc gga aga ccc cca gtt caa gaa        18 r1  ggt ggt gtt cca gga gtt cac cga cgg cag  ctt cac cca gcc cct gta ccg cgg cga gct  gaa cga gca cct ggg (SEQ ID NO:85)   - ggg gat cct cac gtc tca ggc ttg gtt gcg        18 bam  gaa ggt cac cat gat gtt gtc ctc cac ctc  ggc gcg gat gta ggg gcc gag cag gcc cag  gtg ctc gtt cag ct  (SEQ ID NO:86)   -                     r1 bbs 19 for (agcc)  cgc gaa ttc gga aga ccc agc ctc ccg gcc        19 r1  cta ctc ctt cta ctc ctc cct gat cag cta  cga gga gga cca gcg cca ggg cgc cga gcc  ccg caa gaa ctt cgt (SEQ ID NO:87)   - ggg gat cct cac gtc tca ctc gtc ctt ggt        19 bam  ggg ggc cat gtg gtg ctg cac ctt cca gaa  gta ggt ctt agt ctc gtt ggg ctt cac gaa  gtt ctt gcg ggg ct  (SEQ ID NO:88)   -                     r1 bbs 20 for (cgag)  cgc gaa ttc gga aga ccc cga gtt cga ctg        20 r1  caa ggc ctg ggc cta ctt cag cga cgt gga  cct gga gaa gga cgt gca cag cgg cct gat  cgg ccc cct gct ggt (SEQ ID NO:89)   - ggg gat cct cac gtc tca gaa cag ggc aaa        20 bam  ttc ctg cac agt cac ctg cct ccc gtg ggg  ggg gtt cag ggt gtt ggt gtg gca cac cag  cag ggg gcc gat ca  (SEQ ID NO:90)   -                     r1 bbs 21 for (gttc)  cgc gaa ttc gga aga ccc gtt ctt cac cat        21 r1  ctt cga cga gac taa gag ctg gta ctt cac  cga gaa cat gga gcg caa ctg ccg cgc ccc  ctg caa cat cca gat (SEQ ID NO:91)   - ggg gat cct cac gtc tca cag ggt gtc cat        21 bam  gat gta gcc gtt gat ggc gtg gaa gcg gta  gtt ctc ctt gaa ggt ggg atc ttc cat ctg  gat gtt gca ggg gg  (SEQ ID NO:92)   -                     r1 bbs 22 for (cctg)  cgc gaa ttc gga aga ccc cct gcc cgg cct        22 r1  ggt gat ggc cca gga cca gcg cat ccg ctg  gta cct gct gtc tat ggg cag caa cga gaa  cat cca cag cat cca (SEQ ID NO:93)   - ggg gat cct cac gtc tca gta cag gtt gta        22 bam  cag ggc cat ctt gta ctc ctc ctt ctt gcg  cac ggt gaa aac gtg gcc gct gaa gtg gat  gct gtg gat gtt ct  (SEQ ID NO:94)   -                     r1 bbs 23 for (gtac)  cgc gaa ttc gga aga ccc gta ccc cgg cgt        23 r1  gtt cga gac tgt gga gat gct gcc cag caa  ggc cgg gat ctg gcg cgt gga gtg cct gat  cgg cga gca cct gca (SEQ ID NO:95)   - ggg gat cct cac gtc tca gct ggc cat gcc        23 bam  cag ggg ggt ctg gca ctt gtt gct gta cac  cag gaa cag ggt gct cat gcc ggc gtg cag  gtg ctc gcc gat ca  (SEQ ID NO:96)   -                     r1 bbs 24 for (cagc)  cgc gaa ttc gga aga ccc cag cgg cca cat        24 r1  ccg cga ctt cca gat cac cgc cag cgg cca  gta cgg cca gtg ggc tcc caa gct ggc ccg  cct gca cta cag cgg (SEQ ID NO:97)   - ggg gat cct cac gtc tca cat ggg ggc cag        24 bam  cag gtc cac ctt gat cca gga gaa ggg ctc  ctt ggt cga cca ggc gtt gat gct gcc gct  gta gtg cag gcg gg  (SEQ ID NO:98)   -                     r1 bbs 25 for (catg)  cgc gaa ttc gga aga ccc cat gat cat cca        25 r1  cgg cat caa gac cca ggg cgc ccg cca gaa  gtt cag cag cct gta cat cag cca gtt cat  cat cat gta ctc tct (SEQ ID NO:99)   - ggg gat cct cac gtc tca gtt gcc gaa gaa        25 bam  cac cat cag ggt gcc ggt gct gtt gcc gcg  gta ggt ctg cca ctt ctt gcc gtc tag aga  gta cat gat gat ga  (SEQ ID NO:100)   -                     r1 bbs 26 for (caac)  cgc gaa ttc gga aga ccc caa cgt gga cag        26 r1  cag cgg cat caa gca caa cat ctt caa ccc  ccc cat cat cgc ccg cta cat ccg cct gca  ccc cac cca cta cag (SEQ ID NO:101)   - ggg gat cct cac gtc tca gcc cag ggg cat        26 bam  gct gca gct gtt cag gtc gca gcc cat cag  ctc cat gcg cag ggt gct gcg gat gct gta  gtg ggt ggg gtg ca  (SEQ ID NO:102)   -                     r1 bbs 27 for (gggc)  cgc gaa ttc gga aga ccc ggg cat gga gag        27 r1  caa ggc cat cag cga cgc cca gat cac cgc  ctc cag cta ctt cac caa cat gtt cgc cac  ctg gag ccc cag caa (SEQ ID NO:103)   - ggg gat cct cac gtc tca cca ctc ctt ggg        27 bam  gtt gtt cac ctg ggg gcg cca ggc gtt gct  gcg gcc ctg cag gtg cag gcg ggc ctt gct  ggg gct cca ggt gg  (SEQ ID NO:104)   -                     r1 bbs 28 for (gtgg)  cgc gaa ttc gga aga ccc gtg gct gca ggt        28 r1  gga ctt cca gaa aac cat gaa ggt gac tgg  cgt gac cac cga ggg cgt caa gag cct gct  gac cag cat gta cgt (SEQ ID NO:105)   - ggg gat cct cac gtc tca ctt gcc gtt ttg        28 bam  gaa gaa cag ggt cca ctg gtg gcc gtc ctg  gct gct gct gat cag gaa ctc ctt cac gta  cat gct ggt cag ca  (SEQ ID NO:106)   -                     r1 bbs 29 for (caag)  cgc gaa ttc gga aga ccc caa ggt gaa ggt        29 r1  gtt cca ggg caa cca gga cag ctt cac acc  ggt cgt gaa cag cct gga ccc ccc cct gct  gac ccg cta cct gcg (SEQ ID NO:107)   - ggg gat cct cac gtc tca gcg gcc gct tca        29 bam  gta cag gtc ctg ggc ctc gca gcc cag cac  ctc cat gcg cag ggc gat ctg gtg cac cca  gct ctg ggg gtg gat gcg cag gta gcg ggt  cag ca  (SEQ ID NO:108)</pre>
    
    <p>The codon usage for the native and synthetic genes described above are presented in Tables 3 and 4, respectively.</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">              TABLE 3______________________________________Codon Frequency of the Synthetic  Factor VIII B Domain Deleted Gene  AA        Codon   Number    /1000 Fraction______________________________________Gly     GGG     7.00        4.82  0.09  Gly    GGA        1.00     0.69      0.01  Gly    GGT        0.00     0.00      0.00  Gly    GGC       74.00    50.93      0.90   - Glu    GAG       81.00    55.75      6.96  Glu    GAA        3.00     2.06      0.04  Asp    GAT        4.00     2.75      0.05  Asp    GAC       78.00    53.68      0.95   - Val    GTG       77.00    52.99      0.88  Val    GTA        2.00     1.38      0.02  Val    GTT        2.00     1.38      0.02  Val    GTC        7.00     4.82      0.08   - Ala    GCG        0.00     0.00      0.00  Ala    GCA        0.00     0.00      0.00  Ala    GCT        3.00     2.06      0.04  Ala    GCC       67.00    46.11      0.96   - Arg    AGG        2.00     1.38      0.03  Arg    AGA        0.00     0.00      0.00  Ser    AGT        0.00     0.00      0.00  Ser    AGC       97.00    66.76      0.81   - Lys    AAG       75.00    51.62      0.94  Lys    AAA        5.00     3.44      0.06  Asn    AAT        0.00     0.00      0.00  Asn    AAC       63.00    43.36      1.00   - Met    ATG       43.00    29.59      1.00  Ile    ATA        0.00     0.00      0.00  Ile    ATT        2.00     1.38      0.03  Ile    ATC       72.00    49.55      0.97   - Thr    ACG        2.00     1.38      0.02  Thr    ACA        1.00     0.69      0.01  Thr    ACT       10.00     6.88      0.12  Thr    ACC       70.00    48.18      0.84   - Trp    TGG       28.00    19.27      1.00  End    TGA        1.00     0.69      1.00  Cys    TGT        1.00     0.69      0.05  Cys    TGC       18.00    12.39      0.95   - End    TAG        0.00     0.00      0.00  End    TAA        0.00     0.00      0.00  Tyr    TAT        2.00     1.38      0.03  Tyr    TAC       66.00    45.42      0.97   - Leu    TTG        0.00     0.00      0.00  Leu    TTA        0.00     0.00      0.00  Phe    TTT        1.00     0.69      0.01  Phe    TTC       76.00    52.31      0.99   - Ser    TCG        1.00     0.69      0.01  Ser    TCA        0.00     0.00      0.00  Ser    TCT        3.00     2.06      0.03  Ser    TCC       19.00    13.08      0.16   - Arg    CGG        1.00     0.69      0.01  Arg    CGA        0.00     0.00      0.00  Arg    CGT        1.00     0.69      0.01  Arg    CGC       69.00    47.49      0.95   - Gln    CAG       62.00    42.67      0.93  Gln    CAA        5.00     3.44      0.07  His    CAT        1.00     0.69      0.02  His    CAC       50.00    34.41      0.98   - Leu    CTG      118.00    81.21      0.94  Leu    CTA        3.00     2.06      0.02  Leu    CTT        1.00     0.69      0.01  Leu    CTC        3.00     2.06      0.02   - Pro    CCG        4.00     2.75      0.05  Pro    CCA        0.00     0.00      0.00  Pro    CCT        3.00     2.06      0.04  Pro    CCC       68.00    46.80      0.91______________________________________</pre>
    
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">              TABLE 4______________________________________Codon Frequency Table of the Native  Factor VIII B Domain Deleted Gene  AA        Codon   Number    /1000 Fraction______________________________________Gly     GGG     12.00       8.26  0.15  Gly     GGA      34.00     23.40      0.41  Gly     GGT      16.00     11.01      0.20  Gly     GGC      20.00     13.76      0.24   - Glu     GAG      33.00     22.71      0.39  Glu     GAA      51.00     35.10      0.61  Asp     GAT      55.00     37.85      0.67  Asp     GAC      27.00     18.58      0.33   - Val     GTG      29.00     19.96      0.33  Val     GTA      19.00     13.08      0.22  Val     GTT      17.00     11.70      0.19  Val     GTC      23.00     15.83      0.26   - Ala     GCG       2.00      1.38      0.03  Ala     GCA      18.00     12.39      0.25  Ala     GCT      31.00     21.34      0.44  Ala     GCC      20.00     13.76      0.28   - Arg     AGG      18.00     12.39      0.25  Arg     AGA      22.00     15.14      0.30  Ser     AGT      22.00     15.14      0.18  Ser     AGC      24.00     16.52      0.20   - Lys     AAG      32.00     22.02      0.40  Lys     AAA      48.00     33.04      0.60  Asn     AAT      38.00     26.15      0.60  Asn     AAC      25.00     17.21      0.40   - Met     ATG      43.00     29.59      1.00  Ile     ATA      13.00      8.95      0.18  Ile     ATT      36.00     24.78      0.49  Ile     ATC      25.00     17.21      0.34   - Thr     ACG       1.00      0.69      0.01  Thr     ACA      23.00     15.83      0.28  Thr     ACT      36.00     24.78      0.43  Thr     ACC      23.00     15.83      0.28   - Trp     TGG      28.00     19.27      1.00  End     TGA       1.00      0.69      1.00  Cys     TGT       7.00      4.82      0.37  Cys     TGC      12.00      8.26      0.63   - End     TAG       0.00      0.00      0.00  End     TAA       0.00      0.00      0.00  Tyr     TAT      41.00     28.22      0.60  Tyr     TAC      27.00     18.58      0.40   - Leu     TTG      20.00     13.76      0.16  Leu     TTA      10.00      6.88      0.08  Phe     TTT      45.00     30.97      0.58  Phe     TTC      32.00     22.02      0.42   - Ser     TCG       2.00      1.38      0.02  Ser     TCA      27.00     18.58      0.22  Ser     TCT      27.00     18.58      0.22  Ser     TCC      18.00     12.39      0.15   - Arg     CGG       6.00      4.13      0.08  Arg     CGA      10.00      6.88      0.14  Arg     CGT       7.00      4.82      0.10  Arg     CGC      10.00      6.88      0.14   - Gln     CAG      42.00     28.91      0.63  Gln     CAA      25.00     17.21      0.37  His     CAT      28.00     19.27      0.55  His     CAC      23.00     15.83      0.45   - Leu     CTG      36.00     24.78      0.29  Leu     CTA      15.00     10.32      0.12  Leu     CTT      24.00     16.52      0.19  Leu     CTC      20.00     13.76      0.16   - Pro     CCG       1.00      0.69      0.01  Pro     CCA      32.00     22.02      0.43  Pro     CCT      26.00     17.89      0.35  Pro     CCC      15.00     10.32      0.20______________________________________</pre>
    
    <p>Use</p>
    <p>The synthetic genes of the invention are useful for expressing the a protein normally expressed in mammalian cells in cell culture (e.g. for commercial production of human proteins such as hGH, TPA, Factor VIII, and Factor IX). The synthetic genes of the invention are also useful for gene therapy. For example, a synthetic gene encoding a selected protein can be introduced in to a cell which can express the protein to create a cell which can be administered to a patient in need of the protein. Such cell-based gene therapy techniques are well known to those skilled in the art, see, e.g., Anderson, et al., U.S. Pat. No. 5,399,349; Mulligan and Wilson, U.S. Pat. No. 5,460,959.</p>
    <p>
      </p> <pre xml:space="preserve" listing-type="tabular">__________________________________________________________________________#             SEQUENCE LISTING   - -  - - (1) GENERAL INFORMATION:   - -    (iii) NUMBER OF SEQUENCES: 110   - -  - - (2) INFORMATION FOR SEQ ID NO:1:   - -      (i) SEQUENCE CHARACTERISTICS:       (A) LENGTH: 24 base - #pairs       (B) TYPE: nucleic acid       (C) STRANDEDNESS: single       (D) TOPOLOGY: linear   - -     (ii) MOLECULE TYPE: Other   - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:1:   - - CGCGGGCTAG CCACCGAGAA GCTG          - #                  - #    24  - -  - - (2) INFORMATION FOR SEQ ID NO:2:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 195 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:2:  - - ACCGAGAAGC TGTGGGTGAC CGTGTACTAC GGCGTGCCCG TGTGGAAGAG AG -#GCCACCAC     60   - - CACCCTGTTC TGCGCCAGCG ACGCCAAGGC GTACGACACC GAGGTGCACA AC -#GTGTGGGC    120   - - CACCCAGGCG TGCGTGCCCA CCGACCCCAA CCCCCAGGAG GTGGAGCTCG TG -#AACGTGAC    180   - - CGAGAACTTC AACAT              - #                  - #- #   195  - -  - - (2) INFORMATION FOR SEQ ID NO:3:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 34 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:3:  - - CCACCATGTT GTTCTTCCAC ATGTTGAAGT TCTC       - #                  -#        34   - -  - - (2) INFORMATION FOR SEQ ID NO:4:   - -      (i) SEQUENCE CHARACTERISTICS:       (A) LENGTH: 33 base - #pairs       (B) TYPE: nucleic acid       (C) STRANDEDNESS: single       (D) TOPOLOGY: linear   - -     (ii) MOLECULE TYPE: Other   - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:4:   - - GACCGAGAAC TTCAACATGT GGAAGAACAA CAT       - #                  -#         33   - -  - - (2) INFORMATION FOR SEQ ID NO:5:   - -      (i) SEQUENCE CHARACTERISTICS:       (A) LENGTH: 192 base - #pairs       (B) TYPE: nucleic acid       (C) STRANDEDNESS: single       (D) TOPOLOGY: linear   - -     (ii) MOLECULE TYPE: Other   - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:5:   - - TGGAAGAACA ACATGGTGGA GCAGATGCAT GAGGACATCA TCAGCCTGTG GG -#ACCAGAGC     60   - - CTGAAGCCCT GCGTGAAGCT GACCCCCTGT GCGTGACCTG AACTGCACCG AC -#CTGAGGAA    120   - - CACCACCAAC ACCAACACAG CACCGCCAAC AACAACAGCA ACAGCGAGGG CA -#CCATCAAG    180   - - GGCGGCGAGA TG              - #                  - #  - #      192  - -  - - (2) INFORMATION FOR SEQ ID NO:6:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 33 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:6:  - - GTTGAAGCTG CAGTTCTTCA TCTCGCCGCC CTT       - #                  - #   33  - -  - - (2) INFORMATION FOR SEQ ID NO:7:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 31 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:7:  - - GAAGAACTGC AGCTTCAACA TCACCACCAG C        - #                  - #   31  - -  - - (2) INFORMATION FOR SEQ ID NO:8:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 195 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:8:  - - AACATCACCA CCAGCATCCG CGACAAGATG CAGAAGGAGT ACGCCCTGCT GT -#ACAAGCTG     60   - - GATATCGTGA GCATCGACAA CGACAGCACC AGCTACCGCC TGATCTCCTG CA -#ACACCAGC    120   - - GTGATCACCC AGGCCTGCCC CAAGATCAGC TTCGAGCCCA TCCCCATCCA CT -#ACTGCGCC    180   - - CCCGCCGGCT TCGCC              - #                  - #- #   195  - -  - - (2) INFORMATION FOR SEQ ID NO:9:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 30 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:9:  - - GAACTTCTTG TCGGCGGCGA AGCCGGCGGG         - #                  - #   30  - -  - - (2) INFORMATION FOR SEQ ID NO:10:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 47 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:10:  - - GCGCCCCCGC CGGCTTCGCC ATCCTGAAGT GCAACGACAA GAAGTTC   - #47  - -  - - (2) INFORMATION FOR SEQ ID NO:11:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 198 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:11:  - - GCCGACAAGA AGTTCAGCGG CAAGGGCAGC TGCAAGAACG TGAGCACCGT GC -#AGTGCACC     60   - - CACGGCATCC GGCCGGTGGT GAGCACCCAG CTCCTGCTGA ACGGCAGCCT GG -#CCGAGGAG    120   - - GAGGTGGTGA TCCGCAGCGA GAACTTCACC GACAACGCCA AGACCATCAT CG -#TGCACCTG    180   - - AATGAGAGCG TGCAGATC             - #                  - #  - # 198  - -  - - (2) INFORMATION FOR SEQ ID NO:12:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 34 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:12:  - - AGTTGGGACG CGTGCAGTTG ATCTGCACGC TCTC       - #                  -#        34   - -  - - (2) INFORMATION FOR SEQ ID NO:13:   - -      (i) SEQUENCE CHARACTERISTICS:       (A) LENGTH: 30 base - #pairs       (B) TYPE: nucleic acid       (C) STRANDEDNESS: single       (D) TOPOLOGY: linear   - -     (ii) MOLECULE TYPE: Other   - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:13:   - - GAGAGCGTGC AGATCAACTG CACGCGTCCC         - #                  - #    30  - -  - - (2) INFORMATION FOR SEQ ID NO:14:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 120 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:14:  - - AACTGCACGC GTCCCAACTA CAACAAGCGC AAGCGCATCC ACATCGGCCC CG -#GGCGCGCC     60   - - TTCTACACCA CCAAGAACAT CATCGGCACC ATCCTCCAGG CCCACTGCAA CA -#TCTCTAGA    120   - -  - - (2) INFORMATION FOR SEQ ID NO:15:   - -      (i) SEQUENCE CHARACTERISTICS:       (A) LENGTH: 30 base - #pairs       (B) TYPE: nucleic acid       (C) STRANDEDNESS: single       (D) TOPOLOGY: linear   - -     (ii) MOLECULE TYPE: Other   - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:15:   - - GTCGTTCCAC TTGGCTCTAG AGATGTTGCA         - #                  - #    30  - -  - - (2) INFORMATION FOR SEQ ID NO:16:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 29 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:16:  - - GCAACATCTC TAGAGCCAAG TGGAACGAC         - #                  - #   29  - -  - - (2) INFORMATION FOR SEQ ID NO:17:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 131 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:17:  - - GCCAAGTGGA ACGACACCCT GCGCCAGATC GTGAGCAAGC TGAAGGAGCA GT -#TCAAGAAC     60   - - AAGACCATCG TGTTCACCAG AGCAGCGGCG GCGACCCCGA GATCGTGATG CA -#CAGCTTCA    120   - - ACTGCGGCGG C               - #                  - #  - #      131  - -  - - (2) INFORMATION FOR SEQ ID NO:18:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 29 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:18:  - - GCAGTAGAAG AATTCGCCGC CGCAGTTGA         - #                  - #   29  - -  - - (2) INFORMATION FOR SEQ ID NO:19:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 29 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:19:  - - TCAACTGCGG CGGCGAATTC TTCTACTGC         - #                  - #   29  - -  - - (2) INFORMATION FOR SEQ ID NO:20:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 195 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:20:  - - GGCGAATTCT TCTACTGCAA CACCAGCCCC CTGTTCAACA GCACCTGGAA CG -#GCAACAAC     60   - - ACCTGGAACA ACACCACCGG CAGCAACAAC AATATTACCC TCCAGTGCAA GA -#TCAAGCAG    120   - - ATCATCAACA TGTGGCAGGA GGTGGGCAAG GCCATGTACG CCCCCCCCAT CG -#AGGGCCAG    180   - - ATCCGGTGCA GCAGC              - #                  - #- #   195  - -  - - (2) INFORMATION FOR SEQ ID NO:21:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 40 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:21:  - - GCAGACCGGT GATGTTGCTG CTGCACCGGA TCTGGCCCTC     - # - #    40  - -  - - (2) INFORMATION FOR SEQ ID NO:22:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 40 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:22:  - - CGAGGGCCAG ATCCGGTGCA GCAGCAACAT CACCGGTCTG     - # - #    40  - -  - - (2) INFORMATION FOR SEQ ID NO:23:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 198 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:23:  - - AACATCACCG GTCTGCTGCT GACCCGCGAC GGCGGCAAGG ACACCGACAC CA -#ACGACACC     60   - - GAAATCTTCC GCCCCGGCGG CGGCGACATG CGCGACAACT GGAGATCTGA GC -#TGTACAAG    120   - - TACAAGGTGG TGACGATCGA GCCCCTGGGC GTGGCCCCCA CCAAGGCCAA GC -#GCCGCGTG    180   - - GTGCAGCGCG AGAAGCGC             - #                  - #  - # 198  - -  - - (2) INFORMATION FOR SEQ ID NO:24:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 38 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:24:  - - CGCGGGCGGC CGCTTTAGCG CTTCTCGCGC TGCACCAC      - #- #     38  - -  - - (2) INFORMATION FOR SEQ ID NO:25:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 39 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:25:  - - CGCGGGGGAT CCAAGCTTAC CATGATTCCA GTAATAAGT      - # - #    39  - -  - - (2) INFORMATION FOR SEQ ID NO:26:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 165 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:26:  - - ATGAATCCAG TAATAAGTAT AACATTATTA TTAAGTGTAT TACAAATGAG TA -#GAGGACAA     60   - - AGAGTAATAA GTTTAACAGC ATCTTTAGTA AATCAAAATT TGAGATTAGA TT -#GTAGACAT    120   - - GAAAATAATA CAAATTTGCC AATACAACAT GAATTTTCAT TAACG   - #165  - -  - - (2) INFORMATION FOR SEQ ID NO:27:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 36 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:27:  - - CGCGGGGAAT TCACGCGTTA ATGAAAATTC ATGTTG      - #                  -#       36  - -  - - (2) INFORMATION FOR SEQ ID NO:28:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 30 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:28:  - - CGCGGATCCA CGCGTGAAAA AAAAAAACAT         - #                  - #   30  - -  - - (2) INFORMATION FOR SEQ ID NO:29:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 150 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:29:  - - CGTGAAAAAA AAAAACATGT ATTAAGTGGA ACATTAGGAG TACCAGAACA TA -#CATATAGA     60   - - AGTAGAGTAA ATTTGTTTAG TGATAGATTC ATAAAAGTAT TAACATTAGC AA -#ATTTTACA    120   - - ACAAAAGATG AAGGAGATTA TATGTGTGAG         - #                  - #   150  - -  - - (2) INFORMATION FOR SEQ ID NO:30:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 30 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:30:  - - CGCGAATTCG AGCTCACACA TATAATCTCC         - #                  - #   30  - -  - - (2) INFORMATION FOR SEQ ID NO:31:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 30 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:31:  - - CGCGGATCCG AGCTCAGAGT AAGTGGACAA         - #                  - #   30  - -  - - (2) INFORMATION FOR SEQ ID NO:32:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 170 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:32:  - - CTCAGAGTAA GTGGACAAAA TCCAACAAGT AGTAATAAAA CAATAAATGT AA -#TAAGAGAT     60   - - AAATTAGTAA AATGTGAGGA ATAAGTTTAT TAGTACAAAA TACAAGTTGG TT -#ATTATTAT    120   - - TATTATTAAG TTTAAGTTTT TTACAAGCAA CAGATTTTAT AAGTTTATGA  - #170  - -  - - (2) INFORMATION FOR SEQ ID NO:33:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 36 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:33:  - - CGCGAATTCG CGGCCGCTTC ATAAACTTAT AAAATC      - #                  -#       36  - -  - - (2) INFORMATION FOR SEQ ID NO:34:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 1632 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: double      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: cDNA  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:34:  - - CTCGAGATCC ATTGTGCTCT AAAGGAGATA CCCGGCCAGA CACCCTCACC TG -#CGGTGCCC     60   - - AGCTGCCCAG GCTGAGGCAA GAGAAGGCCA GAAACCATGC CCATGGGGTC TC -#TGCAACCG    120   - - CTGGCCACCT TGTACCTGCT GGGGATGCTG GTCGCTTCCG TGCTAGCCAC CG -#AGAAGCTG    180   - - TGGGTGACCG TGTACTACGG CGTGCCCGTG TGGAAGGAGG CCACCACCAC CC -#TGTTCTGC    240   - - GCCAGCGACG CCAAGGCGTA CGACACCGAG GTGCACAACG TGTGGGCCAC CC -#AGGCGTGC    300   - - GTGCCCACCG ACCCCAACCC CCAGGAGGTG GAGCTCGTGA ACGTGACCGA GA -#ACTTCAAC    360   - - ATGTGGAAGA ACAACATGGT GGAGCAGATG CATGAGGACA TCATCAGCCT GT -#GGGACCAG    420   - - AGCCTGAAGC CCTGCGTGAA GCTGACCCCC CTGTGCGTGA CCCTGAACTG CA -#CCGACCTG    480   - - AGGAACACCA CCAACACCAA CAACAGCACC GCCAACAACA ACAGCAACAG CG -#AGGGCACC    540   - - ATCAAGGGCG GCGAGATGAA CAACTGCAGC TTCAACATCA CCACCAGCAT CC -#GCGACAAG    600   - - ATGCAGAAGG AGTACGCCCT GCTGTACAAG CTGGATATCG TGAGCATCGA CA -#ACGACAGC    660   - - ACCAGCTACC GCCTGATCTC CTGCAACACC AGCGTGATCA CCCAGGCCTG GC -#CCAAGATC    720   - - AGCTTCGAGC CCATCCCCAT CCACTACTGC GCCCCCGCCG GCTTCGCCAT CC -#TGAAGTGC    780   - - AACGACAAGA AGTTCAGCGG CAAGGGCAGC TGCAAGAACG TGAGCACCGT GC -#AGTGCACC    840   - - CACGGCATCC GGCCGGTGGT GAGCACCCAG CTCCTGCTGA ACGGCAGCCT GG -#CCGAGGAG    900   - - GAGGTGGTGA TCCGCAGCGA GAACTTCACC GACAACGCCA AGACCATCAT CG -#TGCACCTG    960   - - AATGAGAGCG TGCAGATCAA CTGCACGCGT CCCAACTACA ACAAGCGCAA GC -#GCATCCAC   1020   - - ATCGGCCCCG GGCGCGCCTT CTACACCACC AAGAACATCA TCGGCACCAT CC -#GCCAGGCC   1080   - - CACTGCAACA TCTCTAGAGC CAAGTGGAAC GACACCCTGC GCCAGATCGT GA -#GCAAGCTG   1140   - - AAGGAGCAGT TCAAGAACAA GACCATCGTG TTCAACCAGA GCAGCGGCGG CG -#ACCCCGAG   1200   - - ATCGTGATGC ACAGCTTCAA CTGCGGCGGC GAATTCTTCT ACTGCAACAC CA -#GCCCCCTG   1260   - - TTCAACAGCA CCTGGAACGG CAACAACACC TGGAACAACA CCACCGGCAG CA -#ACAACAAT   1320   - - ATTACCCTCC AGTGCAAGAT CAAGCAGATC ATCAACATGT GGCAGGAGGT GG -#GCAAGGCC   1380   - - ATGTACGCCC CCCCCATCGA GGGCCAGATC CGGTGCAGCA GCAACATCAC CG -#GTCTGCTG   1440   - - CTGACCCGCG ACGGCGGCAA GGACACCGAC ACCAACGACA CCGAAATCTT CC -#GCCCCGGC   1500   - - GGCGGCGACA TGCGCGACAA CTGGAGATCT GAGCTGTACA AGTACAAGGT GG -#TGACGATC   1560   - - GAGCCCCTGG GCGTGGCCCC CACCAAGGCC AAGCGCCGCG TGGTGCAGCG CG -#AGAAGCGC   1620   - - TAAAGCGGCC GC              - #                  - #  - #     1632  - -  - - (2) INFORMATION FOR SEQ ID NO:35:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 2481 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: double      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: cDNA  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:35:  - - ACCGAGAAGC TGTGGGTGAC CGTGTACTAC GGCGTGCCCG TGTGGAAGGA GG -#CCACCACC     60   - - ACCCTGTTCT GCGCCAGCGA CGCCAAGGCG TACGACACCG AGGTGCACAA CG -#TGTGGGCC    120   - - ACCCAGGCGT GCGTGCCCAC CGACCCCAAC CCCCAGGAGG TGGAGCTCGT GA -#ACGTGACC    180   - - GAGAACTTCA ACATGTGGAA GAACAACATG CTGGAGCAGA TGCATGAGGA CA -#TCATCAGC    240   - - CTGTGGGACC AGAGCCTGAA GCCCTGCGTG AAGCTGACCC CCCTGTGCGT GA -#CCCTGAAC    300   - - TGCACCGACC TGAGGAACAC CACCAACACC AACAACAGCA CCGCCAACAA CA -#ACAGCAAC    360   - - AGCGAGGGCA CCATCAAGGG CGGCGAGATG AAGAACTGCA GCTTCAACAT CA -#CCACCAGC    420   - - ATCCGCGACA AGATGCAGAA GGAGTACGCC CTGCTGTACA AGCTGGATAT CG -#TGAGCATC    480   - - CACAACGACA GCACCAGCTA CCGCCTGATC TCCTGCAACA CCAGCGTGAT CA -#CCCAGGCC    540   - - TGCCCCAAGA TCAGCTTCGA GCCCATCCCC ATCCACTACT GCGCCCCCGC CG -#GCTTCGCC    600   - - ATCCTGAAGT GCAACGACAA GAAGTTCAGC GGCAAGGGCA GCTGCAAGAA CG -#TGACCACC    660   - - GTGCAGTGCA CCCACGGCAT CCGGCCGGTG GTGAGCACCC AGCTCCTGCT GA -#ACGGCAGC    720   - - CTGGCCGAGG AGGAGGTGGT GATCCGCAGC GAGAACTTCA CCGACAACGC CA -#AGACCATC    780   - - ATCGTGCACC TGAATGAGAG CGTGCAGATC AACTGCACGC GTCCCAACTA CA -#ACAAGCGC    840   - - AAGCGCATCC ACATCGGCCC CGGGCGCGCC TTCTACACCA CCAAGAACAT CA -#TCGGCACC    900   - - ATCCGCCAGG CCCACTGCAA CATCTCTAGA GCCAAGTGGA ACGACACCCT GC -#GCCAGATC    960   - - GTGAGCAAGC TGAAGGAGCA GTTCAAGAAC AAGACCATCG TGTTCAACCA GA -#GCAGCGGC   1020   - - GGCGACCCCG AGATCGTGAT GCACAGCTTC AACTGCGGCG GCGAATTCTT CT -#ACTGCAAC   1080   - - ACCAGCCCCC TGTTCAACAG CACCTGGAAC GGCAACAACA CCTGGAACAA CA -#CCACCGGC   1140   - - AGCAACAACA ATATTACCCT CCAGTGCAAG ATCAAGCAGA TCATCAACAT GT -#GGCAGGAG   1200   - - GTGGGCAAGG CCATGTACGC CCCCCCCATC GAGGGCCAGA TCCGGTGCAG CA -#GCAACATC   1260   - - ACCGGTCTGC TGCTGACCCG CGACGGCGGC AAGGACACCG ACACCAACGA CA -#CCGAAATC   1320   - - TTCCGCCCCG GCGGCGGCGA CATGCGCGAC AACTGGAGAT CTGAGCTGTA CA -#AGTACAAG   1380   - - GTGGTGACGA TCGAGCCCCT GGGCGTGGCC CCCACCAAGG CCAAGCGCCG CG -#TGGTGCAG   1440   - - CGCGAGAAGC GGGCCGCCAT CGGCGCCCTG TTCCTGGGCT TCCTGGGGGC GG -#CGGGCAGC   1500   - - ACCATGGGGG CCGCCAGCGT GACCCTGACC GTGCAGGCCC GCCTGCTCCT GA -#GCGGCATC   1560   - - GTGCAGCAGC AGAACAACCT CCTCCGCGCC ATCGAGGCCC AGCAGCATAT GC -#TCCAGCTC   1620   - - ACCGTGTGGG GCATCAAGCA GCTCCAGGCC CGCGTGCTGG CCGTGGAGCG CT -#ACCTGAAG   1680   - - GACCAGCAGC TCCTGGGCTT CTGGGGCTGC TCCGGCAAGC TGATCTGCAC CA -#CCACGGTA   1740   - - CCCTGGAACG CCTCCTGGAG CAACAAGAGC CTGGACGACA TCTGGAACAA CA -#TGACCTGG   1800   - - ATGCAGTGGG AGCGCGAGAT CGATAACTAC ACCAGCCTGA TCTACAGCCT GC -#TGGAGAAG   1860   - - AGCCAGACCC AGCAGGAGAA GAACGAGCAG GAGCTGCTGG AGCTGGACAA CT -#GGGCGAGC   1920   - - CTGTGGAACT GGTTCGACAT CACCAACTGG CTGTGGTACA TCAAAATCTT CA -#TCATGATT   1980   - - GTGGGCGGCC TGGTGGGCCT CCGCATCGTG TTCGCCGTGC TGAGCATCGT GA -#ACCGCGTG   2040   - - CGCCAGGGCT ACAGCCCCCT GAGCCTCCAG ACCCGGCCCC CCGTGCCGCG CG -#GGCCCGAC   2100   - - CGCCCCGAGG GCATCGAGGA GGAGGGCGGC GAGCGCGACC GCGACACCAG CG -#GCAGGCTC   2160   - - GTGCACGGCT TCCTGGCGAT CATCTGGGTC GACCTCCGCA GCCTGTTCCT GT -#TCAGCTAC   2220   - - CACCACCGCG ACCTGCTGCT GATCGCCGCC CGCATCGTGG AACTCCTAGG CC -#GCCGCGGC   2280   - - TGGGAGGTGC TGAAGTACTG GTGGAACCTC CTCCAGTATT GGAGCCAGGA GC -#TGAAGTCC   2340   - - AGCGCCGTGA GCCTGCTGAA CGCCACCGCC ATCGCCGTGG CCGAGGGCAC CG -#ACCGCGTG   2400   - - ATCGAGGTGC TCCAGAGGGC CGGGAGGGCG ATCCTGCACA TCCCCACCCG CA -#TCCGCCAG   2460   - - GGGCTCGAGA GGGCGCTGCT G           - #                  - #  2481  - -  - - (2) INFORMATION FOR SEQ ID NO:36:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 486 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: double      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: cDNA  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:36:  - - ATGAATCCAG TAATAAGTAT AACATTATTA TTAAGTGTAT TACAAATGAG TA -#GAGGACAA     60   - - AGAGTAATAA GTTTAACAGC ATGTTTAGTA AATCAAAATT TGAGATTAGA TT -#GTAGACAT    120   - - GAAAATAATA CACCTTTGCC AATACAACAT GAATTTTCAT TAACGCGTGA AA -#AAAAAAAA    180   - - CATGTATTAA GTGGAACATT AGGAGTACCA GAACATACAT ATAGAAGTAG AG -#TAAATTTG    240   - - TTTAGTGATA GATTCATAAA AGTATTAACA TTAGCAAATT TTACAACAAA AG -#ATGAAGGA    300   - - GATTATATGT GTGAGCTCAG AGTAAGTGGA CAAAATCCAA CAAGTAGTAA TA -#AAACAATA    360   - - AATGTAATAA GAGATAAATT AGTAAAATGT GGAGGAATAA GTTTATTAGT AC -#AAAATACA    420   - - AGTTGGTTAT TATTATTATT ATTAAGTTTA AGTTTTTTAC AAGCAACAGA TT -#TTATAAGT    480   - - TTATGA                 - #                  - #                  -#          486  - -  - - (2) INFORMATION FOR SEQ ID NO:37:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 485 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: double      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: cDNA  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:37:  - - ATGAACCCAG TCATCAGCAT CACTCTCCTG CTTTCAGTCT TGCAGATGTC CC -#GAGGACAG     60   - - AGGGTGATCA GCCTGACAGC CTGCCTGGTG AACAGAACCT TCGACTGGAC TG -#CCGTCATG    120   - - AGAATAACAC CAACTTGCCC ATCCAGCATG AGTTCAGCCT GACCCGAGAG AA -#GAAGAAGC    180   - - ACGTGCTGTC AGGCACCCTG GGGGTTCCCG AGCACACTTA CCGCTCCCGC GT -#CAACCTTT    240   - - TCAGTGACCG CTTTATCAAG GTCCTTACTC TAGCCAACTT GACCACCAAG GA -#TGAGGGCG    300   - - ACTACATGTG TGAACTTCGA GTCTCGGGCC AGAATCCCAC AAGCTCCAAT AA -#AACTATCA    360   - - ATGTGATCAG AGACAAGCTG GTCAAGTGTG GTGGCATAAG CCTGCTGGTT CA -#AAACACTT    420   - - CCTGGCTGCT GCTGCTCCTG CTTTCCCTCT CCTTCCTCCA AGCCACGGAC TT -#CATTTCTC    480   - - TGTGA                 - #                  - #                  -#           485   - -  - - (2) INFORMATION FOR SEQ ID NO:38:   - -      (i) SEQUENCE CHARACTERISTICS:       (A) LENGTH: 33 base - #pairs       (B) TYPE: nucleic acid       (C) STRANDEDNESS: single       (D) TOPOLOGY: linear   - -     (ii) MOLECULE TYPE: Other   - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:38:   - - CGCGGGGCTA GCGCAAAGAG TAATAAGTTT AAC       - #                  -#         33   - -  - - (2) INFORMATION FOR SEQ ID NO:39:   - -      (i) SEQUENCE CHARACTERISTICS:       (A) LENGTH: 28 base - #pairs       (B) TYPE: nucleic acid       (C) STRANDEDNESS: single       (D) TOPOLOGY: linear   - -     (ii) MOLECULE TYPE: Other   - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:39:   - - CGCGGATCCC TTGTATTTTG TACTAATA         - #                  - #    28  - -  - - (2) INFORMATION FOR SEQ ID NO:40:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 762 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:40:  - - GAATTCACGC GTAAGCTTGC CGCCACCATG GTGAGCAAGG GCGAGGAGCT GT -#TCACCGGG     60   - - GTGGTGCCCA TCCTGGTCGA GCTGGACGGC GACGTGAACG GCCACAAGTT CA -#GCGTGTCC    120   - - GGCGAGGGCG AGGGCGATGC CACCTACGGC AAGCTGACCC TGAAGTTCAT CT -#GCACCACC    180   - - GGCAAGCTGC CCGTGCCCTG GCCCACCCTC GTGACCACCT TCAGCTACGG CG -#TGCAGTGC    240   - - TTCAGCCGCT ACCCCGACCA CATGAAGCAG CACGACTTCT TCAAGTCCGC CA -#TGCCCGAA    300   - - GGCTACGTCC AGGAGCGCAC CATCTTCTTC AAGGACGACG GCAACTACAA GA -#CCCGCGCC    360   - - GAGGTGAAGT TCGAGGGCGA CACCCTGGTG AACCGCATCG AGCTGAAGGG CA -#TCGACTTC    420   - - AAGGAGGACG GCAACATCCT GGGGCACAAG CTGGAGTACA ACTACAACAG CC -#ACAACGTC    480   - - TATATCATGG CCGACAAGCA GAAGAACGGC ATCAAGGTGA ACTTCAAGAT CC -#GCCACAAC    540   - - ATCGAGGACG GCAGCGTGCA GCTCGCCGAC CACTACCAGC AGAACACCCC CA -#TCGGCGAC    600   - - GGCCCCGTGC TGCTGCCCGA CAACCACTAC CTGAGCACCC AGTCCGCCCT GA -#GCAAAGAC    660   - - CCCAACGAGA AGCGCGATCA CATGGTCCTG CTGGAGTTCG TGACCGCCGC CG -#GGATCACT    720   - - CACGGCATGG ACGAGCTGTA CAAGTAAAGC GGCCGCGGAT CC    - #    - # 762  - -  - - (2) INFORMATION FOR SEQ ID NO:41:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 4670 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: double      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: cDNA  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:41:  - - AAGCTTAAAC CATGCCCATG GGGTCTCTGC AACCGCTGGC CACCTTGTAC CT -#GCTGGGGA     60   - - TGCTGGTCGC TTCCGTGCTA GCCGCCACCA GAAGATACTA CCTGGGTGCA GT -#GGAACTGT    120   - - CATGGGACTA TATGCAAAGT GATCTCGGTG AGCTGCCTGT GGACGCAAGA TT -#TCCTCCTA    180   - - GAGTGCCAAA ATCTTTTCCA TTCAACACCT CAGTCGTGTA CAAAAAGACT CT -#GTTTGTAG    240   - - AATTCACGGA TCACCTTTTC AACATCGCTA AGCCAAGGCC ACCCTGGATG GG -#TCTGCTAG    300   - - GTCCTACCAT CCAGGCTGAG GTTTATGATA CAGTGGTCAT TACACTTAAG AA -#CATGGCTT    360   - - CCCATCCTGT CAGTCTTCAT GCTGTTGGTG TATCCTACTG GAAAGCTTCT GA -#GGGAGCTG    420   - - AATATGATGA TCAGACCAGT CAAAGGGAGA AAGAAGATGA TAAAGTCTTC CC -#TGGTGGAA    480   - - GCCATACATA TGTCTGGCAG GTCCTGAAAG AGAATGGTCC AATGGCCTCT GA -#CCCACTGT    540   - - GCCTTACCTA CTCATATCTT TCTCATGTGG ACCTGGTAAA AGACTTGAAT TC -#AGGCCTCA    600   - - TTGGAGCCCT ACTAGTATGT AGAGAAGGGA GTCTGGCCAA GGAAAAGACA CA -#GACCTTGC    660   - - ACAAATTTAT ACTACTTTTT GCTGTATTTG ATGAAGGGAA AAGTTGGCAC TC -#AGAAACAA    720   - - AGAACTCCTT GATGCAGGAT AGGGATGCTG CATCTGCTCG GGCCTGGCCT AA -#AATGCACA    780   - - CAGTCAATGG TTATGTAAAC AGGTCTCTGC CAGGTCTGAT TGGATGCCAC AG -#GAAATCAG    840   - - TCTATTGGCA TGTGATTGGA ATGGGCACCA CTCCTGAAGT GCACTCAATA TT -#CCTCGAAG    900   - - GTCACACATT TCTTGTGAGG AACCATCGCC AGGCGTCCTT GGAAATCTCG CC -#AATAACTT    960   - - TCCTTACTGC TCAAACACTC TTGATGGACC TTGGACAGTT TCTACTGTTT TG -#TCATATCT   1020   - - CTTCCCACCA ACATGATGGC ATGGAAGCTT ATGTCAAAGT AGACAGCTGT CC -#AGAGGAAC   1080   - - CCCAACTACG AATGAAAAAT AATGAAGAAG CGGAAGACTA TGATGATGAT CT -#TACTGATT   1140   - - CTGAAATGGA TGTGGTCAGG TTTGATGATG ACAACTCTCC TTCCTTTATC CA -#AATTCGCT   1200   - - CAGTTGCCAA GAAGCATCCT AAAACTTGGG TACATTACAT TGCTGCTGAA GA -#GGAGGACT   1260   - - GGGACTATGC TCCCTTAGTC CTCGCCCCCG ATGACAGAAG TTATAAAAGT CA -#ATATTTGA   1320   - - ACAATGGCCC TCAGCGGATT GGTAGGAAGT ACAAAAAAGT CCGATTTATG GC -#ATACACAG   1380   - - ATGAAACCTT TAAGACTCGT GAAGCTATTC AGCATGAATC AGGAATCTTG GG -#ACCTTTAC   1440   - - TTTATGGGGA AGTTGGAGAC ACACTGTTGA TTATATTTAA GAATCAAGCA AG -#CAGACCAT   1500   - - ATAACATCTA CCCTCACGGA ATCACTGATG TCCGTCCTTT GTATTCAAGG AG -#ATTACCAA   1560   - - AAGGTGTAAA ACATTTGAAG GATTTTCCAA TTCTGCCAGG AGAAATATTC AA -#ATATAAAT   1620   - - GGACAGTGAC TGTAGAAGAT GGGCCAACTA AATCAGATCC TCGGTGCCTG AC -#CCGCTATT   1680   - - ACTCTAGTTT CGTTAATATG GAGAGAGATC TAGCTTCAGG ACTCATTGGC CC -#TCTCCTCA   1740   - - TCTGCTACAA AGAATCTGTA GATCAAAGAG GAAACCAGAT AATGTCAGAC AA -#GAGGAATG   1800   - - TCATCCTGTT TTCTGTATTT GATGAGAACC GAAGCTGGTA CCTCACAGAG AA -#TATACAAC   1860   - - GCTTTCTCCC CAATCCAGCT GGAGTGCAGC TTGAGGATCC AGAGTTCCAA GC -#CTCCAACA   1920   - - TCATGCACAG CATCAATGGC TATGTTTTTG ATAGTTTGCA GTTGTCAGTT TG -#TTTGCATG   1980   - - AGGTGGCATA CTGGTACATT CTAAGCATTG GAGCACAGAC TGACTTCCTT TC -#TGTCTTCT   2040   - - TCTCTGGATA TACCTTCAAA CACAAAATGG TCTATGAAGA CACACTCACC CT -#ATTCCCAT   2100   - - TCTCAGGAGA AACTGTCTTC ATGTCGATGG AAAACCCAGG TCTATGGATT CT -#GGGGTGCC   2160   - - ACAACTCAGA CTTTCGGAAC AGAGGCATGA CCGCCTTACT GAAGGTTTCT AG -#TTGTGACA   2220   - - AGAACACTGG TGATTATTAC GAGGACAGTT ATGAAGATAT TTCAGCATAC TT -#GCTGAGTA   2280   - - AAAACAATGC CATTGAACCA AGAAGCTTCT CCCAGAATTC AAGACACCCT AG -#CACTAGGC   2340   - - AAAAGCAATT TAATGCCACC CCACCAGTCT TGAAACGCCA TCAACGGGAA AT -#AACTCGTA   2400   - - CTACTCTTCA GTCAGATCAA GAGGAAATTG ACTATGATGA TACCATATCA GT -#TGAAATGA   2460   - - AGAAGGAAGA TTTTGACATT TATGATGAGG ATGAAAATCA GAGCCCCCGC AG -#CTTTCAAA   2520   - - AGAAAACACG ACACTATTTT ATTGCTGCAG TGGAGAGGCT CTGGGATTAT GG -#GATGAGTA   2580   - - GCTCCCCACA TGTTCTAAGA AACAGGGCTC AGAGTGGCAG TGTCCCTCAG TT -#CAAGAAAG   2640   - - TTGTTTTCCA GGAATTTACT GATGGCTCCT TTACTCAGCC CTTATACCGT GG -#AGAACTAA   2700   - - ATGAACATTT GGGACTCCTG GGGCCATATA TAAGAGCAGA AGTTGAAGAT AA -#TATCATGG   2760   - - TAACTTTCAG AAATCAGGCC TCTCGTCCCT ATTCCTTCTA TTCTAGCCTT AT -#TTCTTATG   2820   - - AGGAAGATCA GAGGCAAGGA GCAGAACCTA GAAAAAACTT TGTCAAGCCT AA -#TGAAACCA   2880   - - AAACTTACTT TTGGAAAGTG CAACATCATA TGGCACCCAC TAAAGATGAG TT -#TGACTGCA   2940   - - AAGCCTGGGC TTATTTCTCT GATGTTGACC TGGAAAAAGA TGTGCACTCA GG -#CCTGATTG   3000   - - GACCCCTTCT GGTCTGCCAC ACTAACACAC TGAACCCTGC TCATGGGAGA CA -#AGTGACAG   3060   - - TACAGGAATT TGCTCTGTTT TTCACCATCT TTGATGAGAC CAAAAGCTGG TA -#CTTCACTG   3120   - - AAAATATGGA AAGAAACTGC AGGGCTCCCT GCAATATCCA GATGGAAGAT CC -#CACTTTTA   3180   - - AAGAGAATTA TCGCTTCCAT GCAATCAATG GCTACATAAT GGATACACTA CC -#TGGCTTAG   3240   - - TAATGGCTCA GGATCAAAGG ATTCGATGGT ATCTGCTCAG CATGGGCAGC AA -#TGAAAACA   3300   - - TCCATTCTAT TCATTTCAGT GGACATGTGT TCACTGTACG AAAAAAAGAG GA -#GTATAAAA   3360   - - TGGCACTGTA CAATCTCTAT CCAGGTGTTT TTGAGACAGT GGAAATGTTA CC -#ATCCAAAG   3420   - - CTGGAATTTG GCGGGTGGAA TGCCTTATTG GCGAGCATCT ACATGCTGGG AT -#GAGCACAC   3480   - - TTTTTCTGGT GTACAGCAAT AAGTGTCAGA CTCCCCTGGG AATGGCTTCT GG -#ACACATTA   3540   - - GAGATTTTCA GATTACAGCT TCAGGACAAT ATGGACAGTG GGCCCCAAAG CT -#GGCCAGAC   3600   - - TTCATTATTC CGGATCAATC AATGCCTGGA GCACCAAGGA GCCCTTTTCT TG -#GATCAAGG   3660   - - TGGATCTGTT GGCACCAATG ATTATTCACG GCATCAAGAC CCAGGGTGCC CG -#TCAGAAGT   3720   - - TCTCCAGCCT CTACATCTCT CAGTTTATCA TCATGTATAG TCTTGATGGG AA -#GAAGTGGC   3780   - - AGACTTATCG AGGAAATTCC ACTGGAACCT TAATGGTCTT CTTTGGCAAT GT -#GGATTCAT   3840   - - CTGGGATAAA ACACAATATT TTTAACCCTC CAATTATTGC TCGATACATC CG -#TTTGCACC   3900   - - CAACTCATTA TAGCATTCGC AGCACTCTTC GCATGGAGTT GATGGGCTGT GA -#TTTAAATA   3960   - - GTTGCAGCAT GCCATTGGGA ATGGAGAGTA AAGCAATATC AGATGCACAG AT -#TACTGCTT   4020   - - CATCCTACTT TACCAATATG TTTGCCACCT GGTCTCCTTC AAAAGCTCGA CT -#TCACCTCC   4080   - - AAGGGAGGAG TAATGCCTGG AGACCTCAGG TGAATAATCC AAAAGAGTGG CT -#GCAAGTGG   4140   - - ACTTCCAGAA GACAATGAAA GTCACAGGAG TAACTACTCA GGGAGTAAAA TC -#TCTGCTTA   4200   - - CCAGCATGTA TGTGAAGGAG TTCCTCATCT CCAGCAGTCA AGATGGCCAT CA -#GTGGACTC   4260   - - TCTTTTTTCA GAATGGCAAA GTAAAGGTTT TTCAGGGAAA TCAAGACTCC TT -#CACACCTG   4320   - - TGGTGAACTC TCTAGACCCA CCGTTACTGA CTCGCTACCT TCGAATTCAC CC -#CCAGAGTT   4380   - - GGGTGCACCA GATTGCCCTG AGGATGGAGG TTCTGGGCTG CGAGGCACAG GA -#CCTCTACT   4440   - - GAGGGTGGCC ACTGCAGCAC CTGCCACTGC CGTCACCTCT CCCTCCTCAG CT -#CCAGGGCA   4500   - - GTGTCCCTCC CTGGCTTGCC TTCTACCTTT GTGCTAAATC CTAGCAGACA CT -#GCCTTGAA   4560   - - GCCTCCTGAA TTAACTATCA TCAGTCCTGC ATTTCTTTGG TGGGGGGCCA GG -#AGGGTGCA   4620   - - TCCAATTTAA CTTAACTCTT ACCGTCGACC TGCAGGCCCA ACGCGGCCGC  - #    4670  - -  - - (2) INFORMATION FOR SEQ ID NO:42:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 4451 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: double      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: cDNA  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:42:  - - AAGCTTAAAC CATGCCCATG GGGTCTCTGC AACCGCTGGC CACCTTGTAC CT -#GCTGGGGA     60   - - TGCTGGTCGC TTCCGTGCTA GCCGCCACCC GCCGCTACTA CCTGGGCGCC GT -#GGAGCTGT    120   - - CCTGGGACTA CATGCAGAGC GACCTGGGCG AGCTCCCCGT GGACGCCCGC TT -#CCCCCCCC    180   - - GCGTGCCCAA GAGCTTCCCC TTCAACACCA GCGTGGTGTA CAAGAAAACC CT -#GTTCGTGG    240   - - AGTTCACCGA CCACCTGTTC AACATTGCCA AGCCGCGCCC CCCCTGGATG GG -#CCTGCTGG    300   - - GCCCCACCAT CCAGGCCGAG GTGTACGACA CCGTGGTGAT CACCCTGAAG AA -#CATGGCCA    360   - - GCCACCCCGT CAGCCTGCAC GCCGTGGGCG TGAGCTACTG GAAGGCCAGC GA -#GGGCGCCG    420   - - AGTACGACGA CCAGACGTCC CAGCGCGAGA AGGAGGACGA CAAGGTGTTC CC -#GGGGGGGA    480   - - GCCACACCTA CGTGTGGCAG GTGCTTAAGG AGAACGGCCC TATGGCCAGC GA -#CCCCCTGT    540   - - GCCTGACCTA CAGCTACCTG AGCCACGTGG ACCTGGTGAA GGATCTGAAC AG -#CGGGCTGA    600   - - TCGGCGCCCT GCTGGTGTGT CGCGAGGGCA GCCTGGCCAA GGAGAAAACC CA -#GACCCTGC    660   - - ACAAGTTCAT CCTGCTGTTC GCCGTGTTCG ACGAGGGGAA GAGCTGGCAC AG -#CGAGACTA    720   - - AGAACAGCCT GATGCAGGAC CGCGACGCCG CCAGCGCCCG CGCCTGGCCC AA -#GATGCACA    780   - - CCGTTAACGG CTACGTGAAC CGCAGCCTGC CCGGCCTGAT CGGCTGCCAC CG -#CAAGAGCG    840   - - TGTACTGGCA CGTCATCGGC ATGGGCACCA CCCCTGAGGT GCACAGCATC TT -#CCTGGAGG    900   - - GCCACACCTT CCTGGTGCGC AACCACCGCC AGGCCAGCCT GGAGATCAGC CC -#CATCACCT    960   - - TCCTGACTGC CCAGACCCTG CTGATGGACC TAGGCCAGTT CCTGCTGTTC TG -#CCACATCA   1020   - - GCAGCCACCA GCACGACGGC ATGGAGGCTT ACGTGAAGGT GGACAGCTGC CC -#CGAGGAGC   1080   - - CCCAGCTGCG CATGAAGAAC AACGAGGAGG CCGAGGACTA CGACGACGAC CT -#GACCGACA   1140   - - GCGAGATGGA TGTCGTACGC TTCGACGACG ACAACAGCCC CAGCTTCATC CA -#GATCCGCA   1200   - - GCGTGGCCAA GAAGCACCCT AAGACCTGGG TGCACTACAT CGCCGCCGAG GA -#GGAGGACT   1260   - - GGGACTACGC CCCGCTAGTA CTGGCCCCCG ACGACCGCAG CTACAAGAGC CA -#GTACCTGA   1320   - - ACAACGGCCC CCAGCGCATC GGCCGCAAGT ACAAGAAGGT GCGCTTCATG GC -#CTACACCG   1380   - - ACGAGACTTT CAAGACCCGC GAGGCCATCC AGCACGAGTC CGGCATCCTC GG -#CCCCCTGC   1440   - - TGTACGGCGA GGTGGGCGAC ACCCTGCTGA TCATCTTCAA GAACCAGGCC AG -#CAGGCCCT   1500   - - ACAACATCTA CCCCCACGGC ATCACCGACG TGCGCCCCCT GTACAGCCGC CG -#CCTGCCCA   1560   - - AGGGCGTGAA GCACCTGAAG GACTTCCCCA TCCTGCCCGG CGAGATCTTC AA -#GTACAAGT   1620   - - GGACCGTGAC CGTGGAGGAC GGCCCCACCA AGAGCGACCC CCGCTGCCTG AC -#CCGCTACT   1680   - - ACAGCAGCTT CGTGAACATG GAGCGCGACC TGGCCTCCGG ACTGATCGGC CC -#CCTGCTGA   1740   - - TCTGCTACAA GGAGAGCGTG GACCAGCGCG GCAACCAGAT CATGAGCGAC AA -#GCGCAACG   1800   - - TGATCCTGTT CAGCGTGTTC GACGAGAACC GCAGCTGGTA TCTGACCGAG AA -#CATCCAGC   1860   - - GCTTCCTGCC CAACCCCGCT GGCGTGCAGC TGGAAGATCC CGAGTTCCAG GC -#CAGCAACA   1920   - - TCATGCACAG CATCAACGGC TACGTGTTCG ACAGCCTGCA GCTGAGCGTG TG -#CCTGCATG   1980   - - AGGTGGCCTA CTGGTACATC CTGAGCATCG GCGCCCAGAC CGACTTCCTG AG -#CGTGTTCT   2040   - - TCTCCGGGTA TACCTTCAAG CACAAGATGG TGTACGAGGA CACCCTGACC CT -#GTTCCCCT   2100   - - TCTCCGGCGA GACTGTGTTC ATGTCTATGG AGAACCCCGG CCTGTGGATT CT -#GGGCTGCC   2160   - - ACAACAGCGA CTTCCGCAAC CGCGGCATGA CTGCCCTGCT GAAAGTCTCC AG -#CTGCGACA   2220   - - AGAACACCGG CGACTACTAC GAGGACAGCT ACGAGGACAT CTCCGCCTAC CT -#GCTGTCCA   2280   - - AGAACAACGC CATCGAGCCC CGCTCCTTCT CCCAAAACTC CCGCCACCCC AG -#CACGCGTC   2340   - - AGAAGCAGTT CAACGCCACC CCCCCCGTGC TGAAGCGCCA CCAGCGCGAG AT -#CACCCGCA   2400   - - CCACCCTGCA AAGCGACCAG GAGGAGATCG ACTACGACGA CACCATCAGC GT -#GGAGATGA   2460   - - AGAAGGAGGA CTTCGACATC TACGACGAGG ACGAGAACCA GAGCCCCCGC TC -#CTTCCAAA   2520   - - AGAAAACCCG CCACTACTTC ATCGCCGCCG TGGAGCGCCT GTGGGACTAC GG -#CATGAGCA   2580   - - GCAGCCCCCA CGTCCTGCGC AACCGCGCCC AGAGCGGCAG CGTGCCCCAG TT -#CAAGAAGG   2640   - - TGGTGTTCCA GGAGTTCACC GACGGCAGCT TCACCCAGCC CCTGTACCGC GG -#CGAGCTGA   2700   - - ACGAGCACCT GGGCCTGCTC GGCCCCTACA TCCGCGCCGA GGTGGAGGAC AA -#CATCATGG   2760   - - TGACCTTCCG CAACCAAGCC TCCCGGCCCT ACTCCTTCTA CTCCTCCCTG AT -#CAGCTACG   2820   - - AGGAGGACCA GCGCCAGGGC GCCGAGCCCC GCAAGAACTT CGTGAAGCCC AA -#CGAGACTA   2880   - - AGACCTACTT CTGGAAGGTG CAGCACCACA TGGCCCCCAC CAAGGACGAG TT -#CGACTGCA   2940   - - AGGCCTGGGC CTACTTCAGC GACGTGGACC TGGAGAAGGA CGTGCACAGC GG -#CCTGATCG   3000   - - GCCCCCTGCT GGTGTGCCAC ACCAACACCC TGAACCCCCC CCACGGGAGG CA -#GGTGACTG   3060   - - TGCAGGAATT TGCCCTGTTC TTCACCATCT TCGACGAGAC TAAGAGCTGG TA -#CTTCACCG   3120   - - AGAACATGGA GCGCAACTGC CGCGCCCCCT GCAACATCCA GATGGAAGAT CC -#CACCTTCA   3180   - - AGGAGAACTA CCGCTTCCAC GCCATCAACG GCTACATCAT GGACACCCTG CC -#CGGCCTGG   3240   - - TGATGGCCCA GGACCAGCGC ATCCGCTGGT ACCTGCTGTC TATGGGCAGC AA -#CGAGAACA   3300   - - TCCACAGCAT CCACTTCAGC GGCCACGTTT TCACCGTGCG CAAGAAGGAG GA -#GTACAAGA   3360   - - TGGCCCTGTA CAACCTGTAC CCCGGCGTGT TCGAGACTGT GGAGATGCTG CC -#CAGCAAGG   3420   - - CCGGGATCTG GCGCGTGGAG TGCCTGATCG GCGAGCACCT GCACGCCGGC AT -#GAGCACCC   3480   - - TGTTCCTGGT GTACAGCAAC AAGTGCCAGA CCCCCCTGGG CATGGCCAGC GG -#CCACATCC   3540   - - GCGACTTCCA GATCACCGCC AGCGGCCAGT ACGGCCAGTG GGCTCCCAAG CT -#GGCCCGCC   3600   - - TGCACTACAG CGGCAGCATC AACGCCTGGT CGACCAAGGA GCCCTTCTCC TG -#GATCAAGG   3660   - - TGGACCTGCT GGCCCCCATG ATCATCCACG GCATCAAGAC CCAGGGCGCC CG -#CCAGAAGT   3720   - - TCAGCAGCCT GTACATCAGC CAGTTCATCA TCATGTACTC TCTAGACGGC AA -#GAAGTGGC   3780   - - AGACCTACCG CGGCAACAGC ACCGGCACCC TGATGGTGTT CTTCGGCAAC GT -#GGACAGCA   3840   - - GCGGCATCAA GCACAACATC TTCAACCCCC CCATCATCGC CCGCTACATC CG -#CCTGCACC   3900   - - CCACCCACTA CAGCATCCGC AGCACCCTGC GCATGGAGCT GATGGGCTGC GA -#CCTGAACA   3960   - - GCTGCAGCAT GCCCCTGGGC ATGGAGAGCA AGGCCATCAG CGACGCCCAG AT -#CACCGCCT   4020   - - CCAGCTACTT CACCAACATG TTCGCCACCT GGAGCCCCAG CAAGGCCCGC CT -#GCACCTGC   4080   - - AGGGCCGCAG CAACGCCTGG CGCCCCCAGG TGAACAACCC CAAGGAGTGG CT -#GCAGGTGG   4140   - - ACTTCCAGAA AACCATGAAG GTGACTGGCG TGACCACCCA GGGCGTCAAG AG -#CCTGCTGA   4200   - - CCAGCATGTA CGTGAAGGAG TTCCTGATCA GCAGCAGCCA GGACGGCCAC CA -#GTGGACCC   4260   - - TGTTCTTCCA AAACGGCAAG GTGAAGGTGT TCCAGGGCAA CCAGGACAGC TT -#CACACCGG   4320   - - TCGTGAACAG CCTGGACCCC CCCCTGCTGA CCCGCTACCT GCGCATCCAC CC -#CCAGAGCT   4380   - - GGGTGCACCA GATCGCCCTG CGCATGGAGG TGCTGGGCTG CGAGGCCCAG GA -#CCTGTACT   4440   - - GAAGCGGCCG C               - #                  - #  - #     4451  - -  - - (2) INFORMATION FOR SEQ ID NO:43:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 18 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:43:  - - GGGGATCCTC ACGTCTCA             - #                  - # - #  18  - -  - - (2) INFORMATION FOR SEQ ID NO:44:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 33 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:44:  - - CTGCTTCTGA CGCGTGCTGG GGTGGCGGGA GTT       - #                  - #   33  - -  - - (2) INFORMATION FOR SEQ ID NO:45:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 21 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:45:  - - CTGCTGAAAG TCTCCAGCTG C           - #                  - #   - #21  - -  - - (2) INFORMATION FOR SEQ ID NO:46:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 33 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:46:  - - GGCAGGTGCT TAAGGAGAAC GGCCCTATGG CCA       - #                  - #   33  - -  - - (2) INFORMATION FOR SEQ ID NO:47:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 39 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:47:  - - CGCCAAGGGC TAGCCGCCAC CAGAAGATAC TACCTGGGT      - # - #    39  - -  - - (2) INFORMATION FOR SEQ ID NO:48:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 31 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:48:  - - CAAGACTGGT GGGGTGGCAT TAAATTGCTT T        - #                  - #   31  - -  - - (2) INFORMATION FOR SEQ ID NO:49:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 29 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:49:  - - AATGCCACCC CACCAGTCTT GAAACGCCA         - #                  - #   29  - -  - - (2) INFORMATION FOR SEQ ID NO:50:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 20 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:50:  - - CATCTGGATA TTGCAGGGAG            - #                  - #  - # 20  - -  - - (2) INFORMATION FOR SEQ ID NO:51:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 105 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:51:  - - CGCGAATTCG GAAGACCCGC TAGCCGCCAC CCGCCGCTAC TACCTGGGCG CC -#GTGGAGCT     60   - - GTCCTGGGAC TACATGCAGA GCGACCTGGG CGAGCTCCCC GTGGA   - #105  - -  - - (2) INFORMATION FOR SEQ ID NO:52:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 104 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:52:  - - GGGGATCCTC ACGTCTCAGG TTTTCTTGTA CACCACGCTG GTGTTGAAGG GG -#AAGCTCTT     60   - - GGGCACGCGG GGGGGGAAGC GGGCGTCCAC GGGGAGCTCG CCCA   - #- #104  - -  - - (2) INFORMATION FOR SEQ ID NO:53:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 105 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:53:  - - CGCGAATTCG GAAGACCCAA CCCTGTTCGT GGAGTTCACC GACCACCTGT TC -#AACATTGC     60   - - CAAGCCGCGC CCCCCCTGGA TGGGCCTGCT GGGCCCCACC ATCCA   - #105  - -  - - (2) INFORMATION FOR SEQ ID NO:54:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 104 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:54:  - - GGGGATCCTC ACGTCTCAGT GCAGGCTGAC GGGGTGGCTG GCCATGTTCT TC -#AGGGTGAT     60   - - CACCACGGTG TCGTACACCT CGGCCTGGAT GGTGGGGCCC AGCA   - #- #104  - -  - - (2) INFORMATION FOR SEQ ID NO:55:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 105 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:55:  - - CGCGAATTCG GAAGACCCGC ACGCCGTGGG CGTGAGCTAC TGGAAGGCCA GC -#GAGGGCGC     60   - - CGAGTACGAC GACCAGACGT CCCAGCGCGA GAAGGAGGAC GACAA   - #105  - -  - - (2) INFORMATION FOR SEQ ID NO:56:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 104 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:56:  - - GGGGATCCTC ACGTCTCAGC TGGCCATAGG GCCGTTCTCC TTAAGCACCT GC -#CACACGTA     60   - - GGTGTGGCTC CCCCCCGGGA ACACCTTGTC GTCCTCCTTC TCGC   - #- #104  - -  - - (2) INFORMATION FOR SEQ ID NO:57:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 105 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:57:  - - CGCGAATTCG GAAGACCCCA GCGACCCCCT GTGCCTGACC TACAGCTACC TG -#AGCCACGT     60   - - GGACCTGGTG AAGGATCTGA ACAGCGGGCT GATCGGCGCC CTGCT   - #105  - -  - - (2) INFORMATION FOR SEQ ID NO:58:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 104 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:58:  - - GGGGATCCTC ACGTCTCAGA ACAGCAGGAT GAACTTGTGC AGGGTCTGGG TT -#TTCTCCTT     60   - - GGCCAGGCTG CCCTCGCGAC ACACCAGCAG GGCGCCGATC AGCC   - #- #104  - -  - - (2) INFORMATION FOR SEQ ID NO:59:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 105 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:59:  - - CGCGAATTCG GAAGACCCGT TCGCCGTGTT CGACGAGGGG AAGAGCTGGC AC -#AGCGAGAC     60   - - TAAGAACAGC CTGATGCAGG ACCGCGACGC CGCCAGCGCC CGCGC   - #105  - -  - - (2) INFORMATION FOR SEQ ID NO:60:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 104 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:60:  - - GGGGATCCTC ACGTCTCAGT GGCAGCCGAT CAGGCCGGGC AGGCTGCGGT TC -#ACGTAGCC     60   - - GTTAACGGTG TGCATCTTGG GCCAGGCGCG GGCGCTGGCG GCGT   - #- #104  - -  - - (2) INFORMATION FOR SEQ ID NO:61:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 105 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:61:  - - CGCGAATTCG GAAGACCCCC ACCGCAAGAG CGTGTACTGG CACGTCATCG GC -#ATGGGCAC     60   - - CACCCCTGAG GTGCACAGCA TCTTCCTGGA GGGCCACACC TTCCT   - #105  - -  - - (2) INFORMATION FOR SEQ ID NO:62:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 104 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:62:  - - GGGGATCCTC ACGTCTCACA GGGTCTGGGC AGTCAGGAAG GTGATGGGGC TG -#ATCTCCAG     60   - - GCTGGCCTGG CGGTGGTTGC GCACCAGGAA GGTGTGGCCC TCCA   - #- #104  - -  - - (2) INFORMATION FOR SEQ ID NO:63:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 105 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:63:  - - CGCGAATTCG GAAGACCCCC TGCTGATGGA CCTAGGCCAG TTCCTGCTGT TC -#TGCCACAT     60   - - CAGCAGCCAC CAGCACGACG GCATGGAGGC TTACGTGAAG GTGGA   - #105  - -  - - (2) INFORMATION FOR SEQ ID NO:64:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 104 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:64:  - - GGGGATCCTC ACGTCTCAGT CGTCGTCGTA GTCCTCGGCC TCCTCGTTGT TC -#TTCATGCG     60   - - CAGCTGGGGC TCCTCGGGGC AGCTGTCCAC CTTCACGTAA GCCT   - #- #104  - -  - - (2) INFORMATION FOR SEQ ID NO:65:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 105 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:65:  - - CGCGAATTCG GAAGACCCCG ACCTGACCGA CAGCGAGATG GATGTCGTAC GC -#TTCGACGA     60   - - CGACAACAGC CCCAGCTTCA TCCAGATCCG CAGCGTGGCC AAGAA   - #105  - -  - - (2) INFORMATION FOR SEQ ID NO:66:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 104 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:66:  - - GGGGATCCTC ACGTCTCATA CTAGCGGGGC GTAGTCCCAG TCCTCCTCCT CG -#GCGGCGAT     60   - - GTAGTGCACC CAGGTCTTAG GGTGCTTCTT GGCCACGCTG CGGA   - #- #104  - -  - - (2) INFORMATION FOR SEQ ID NO:67:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 105 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:67:  - - CGCGAATTCG GAAGACCCAG TACTGGCCCC CGACGACCGC AGCTACAAGA GC -#CAGTACCT     60   - - GAACAACGGC CCCCAGCGCA TCGGCCGCAA GTACAAGAAG GTGCG   - #105  - -  - - (2) INFORMATION FOR SEQ ID NO:68:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 104 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:68:  - - GGGGATCCTC ACGTCTCAGA GGATGCCGGA CTCGTGCTGG ATGGCCTCGC GG -#GTCTTGAA     60   - - AGTCTCGTCG GTGTAGGCCA TGAAGCGCAC CTTCTTGTAC TTGC   - #- #104  - -  - - (2) INFORMATION FOR SEQ ID NO:69:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 105 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:69:  - - CGCGAATTCG GAAGACCCCC TCGGCCCCCT GCTGTACGGC GAGGTGGGCG AC -#ACCCTGCT     60   - - GATCATCTTC AAGAACCAGG CCAGCAGGCC CTACAACATC TACCC   - #105  - -  - - (2) INFORMATION FOR SEQ ID NO:70:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 104 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:70:  - - GGGGATCCTC ACGTCTCACT TCAGGTGCTT CACGCCCTTG GGCAGGCGGC GG -#CTGTACAG     60   - - GGGGCGCACG TCGGTGATGC CGTGGGGGTA GATGTTGTAG GGCC   - #- #104  - -  - - (2) INFORMATION FOR SEQ ID NO:71:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 105 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:71:  - - CGCGAATTCG GAAGACCCGA AGGACTTCCC CATCCTGCCC GGCGAGATCT TC -#AAGTACAA     60   - - GTGGACCGTG ACCGTGGAGG ACGGCCCCAC CAAGAGCGAC CCCCG   - #105  - -  - - (2) INFORMATION FOR SEQ ID NO:72:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 104 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:72:  - - GGGGATCCTC ACGTCTCAGC CGATCAGTCC GGAGGCCAGG TCGCGCTCCA TG -#TTCACGAA     60   - - GCTGCTGTAG TAGCGGGTCA GGCAGCGGGG GTCGCTCTTG GTGG   - #- #104  - -  - - (2) INFORMATION FOR SEQ ID NO:73:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 105 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:73:  - - CGCGAATTCG GAAGACCCCG GCCCCCTGCT GATCTGCTAC AAGGAGAGCG TG -#GACCAGCG     60   - - CGGCAACCAG ATCATGAGCG ACAAGCGCAA CGTGATCCTG TTCAG   - #105  - -  - - (2) INFORMATION FOR SEQ ID NO:74:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 104 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:74:  - - GGGGATCCTC ACGTCTCAAG CGGGGTTGGG CAGGAAGCGC TGGATGTTCT CG -#GTCAGATA     60   - - CCAGCTGCGG TTCTCGTCGA ACACGCTGAA CAGGATCACG TTGC   - #- #104  - -  - - (2) INFORMATION FOR SEQ ID NO:75:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 105 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:75:  - - CGCGAATTCG GAAGACCCCG CTGGCGTGCA GCTGGAAGAT CCCGAGTTCC AG -#GCCAGCAA     60   - - CATCATGCAC AGCATCAACG GCTACGTGTT CGACAGCCTG CAGCT   - #105  - -  - - (2) INFORMATION FOR SEQ ID NO:76:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 104 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:76:  - - GGGGATCCTC ACGTCTCACA GGAAGTCGGT CTGGGCGCCG ATGCTCAGGA TG -#TACCAGTA     60   - - GGCCACCTCA TGCAGGCACA CGCTCAGCTG CAGGCTGTCG AACA   - #- #104  - -  - - (2) INFORMATION FOR SEQ ID NO:77:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 105 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:77:  - - CGCGAATTCG GAAGACCCCC TGAGCGTGTT CTTCTCCGGG TATACCTTCA AG -#CACAAGAT     60   - - GGTGTACGAG GACACCCTGA CCCTGTTCCC CTTCTCCGGC GAGAC   - #105  - -  - - (2) INFORMATION FOR SEQ ID NO:78:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 104 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:78:  - - GGGGATCCTC ACGTCTCAGT TGCGGAAGTC GCTGTTGTGG CAGCCCAGAA TC -#CACAGGCC     60   - - GGGGTTCTCC ATAGACATGA ACACAGTCTC GCCGGAGAAG GGGA   - #- #104  - -  - - (2) INFORMATION FOR SEQ ID NO:79:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 105 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:79:  - - CGCGAATTCG GAAGACCCCA ACCGCGGCAT GACTGCCCTG CTGAAAGTCT CC -#AGCTGCGA     60   - - CAAGAACACC GGCGACTACT ACGAGGACAG CTACGAGGAC ATCTC   - #105  - -  - - (2) INFORMATION FOR SEQ ID NO:80:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 104 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:80:  - - GGGGATCCTC ACGTCTCAGC GGTGGCGGGA GTTTTGGGAG AAGGAGCGGG GC -#TCGATGGC     60   - - GTTGTTCTTG GACAGCAGGT AGGCGGAGAT GTCCTCGTAG CTGT   - #- #104  - -  - - (2) INFORMATION FOR SEQ ID NO:81:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 105 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:81:  - - CGCGAATTCG GAAGACCCCC GCAGCACGCG TCAGAAGCAG TTCAACGCCA CC -#CCCCCCGT     60   - - GCTGAAGCGC CACCAGCGCG AGATCACCCG CACCACCCTG CAAAG   - #105  - -  - - (2) INFORMATION FOR SEQ ID NO:82:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 104 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:82:  - - GGGGATCCTC ACGTCTCAGA TGTCGAAGTC CTCCTTCTTC ATCTCCACGC TG -#ATGGTGTC     60   - - GTCGTAGTCG ATCTCCTCCT GGTCGCTTTG CAGGGTGGTG CGGG   - #- #104  - -  - - (2) INFORMATION FOR SEQ ID NO:83:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 105 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:83:  - - CGCGAATTCG GAAGACCCCA TCTACGACGA GGACGAGAAC CAGAGCCCCC GC -#TCCTTCCA     60   - - AAAGAAAACC CGCCACTACT TCATCGCCGC CGTGGAGCGC CTGTG   - #105  - -  - - (2) INFORMATION FOR SEQ ID NO:84:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 104 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:84:  - - GGGGATCCTC ACGTCTCACT GGGGCACGCT GCCGCTCTGG GCGCGGTTGC GC -#AGGACGTG     60   - - GGGGCTGCTG CTCATGCCGT AGTCCCACAG GCGCTCCACG GCGG   - #- #104  - -  - - (2) INFORMATION FOR SEQ ID NO:85:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 105 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:85:  - - CGCGAATTCG GAAGACCCCC AGTTCAAGAA GGTGGTGTTC CAGGAGTTCA CC -#GACGGCAG     60   - - CTTCACCCAG CCCCTGTACC GCGGCGAGCT GAACGAGCAC CTGGG   - #105  - -  - - (2) INFORMATION FOR SEQ ID NO:86:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 104 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:86:  - - GGGGATCCTC ACGTCTCAGG CTTGGTTGCG GAAGGTCACC ATGATGTTGT CC -#TCCACCTC     60   - - GGCGCGGATG TAGGGGCCGA GCAGGCCCAG GTGCTCGTTC AGCT   - #- #104  - -  - - (2) INFORMATION FOR SEQ ID NO:87:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 105 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:87:  - - CGCGAATTCG GAAGACCCAG CCTCCCGGCC CTACTCCTTC TACTCCTCCC TG -#ATCAGCTA     60   - - CGAGGAGGAC CAGCGCCAGG GCGCCGAGCC CCGCAAGAAC TTCGT   - #105  - -  - - (2) INFORMATION FOR SEQ ID NO:88:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 104 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:88:  - - GGGGATCCTC ACGTCTCACT CGTCCTTGGT GGGGGCCATG TGGTGCTGCA CC -#TTCCAGAA     60   - - GTAGGTCTTA GTCTCGTTGG GCTTCACGAA GTTCTTGCGG GGCT   - #- #104  - -  - - (2) INFORMATION FOR SEQ ID NO:89:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 105 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:89:  - - CGCGAATTCG GAAGACCCCG AGTTCGACTG CAAGGCCTGG GCCTACTTCA GC -#GACGTGGA     60   - - CCTGGAGAAG GACGTGCACA GCGGCCTGAT CGGCCCCCTG CTGGT   - #105  - -  - - (2) INFORMATION FOR SEQ ID NO:90:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 104 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:90:  - - GGGGATCCTC ACGTCTCAGA ACAGGGCAAA TTCCTGCACA GTCACCTGCC TC -#CCGTGGGG     60   - - GGGGTTCAGG GTGTTGGTGT GGCACACCAG CAGGGGGCCG ATCA   - #- #104  - -  - - (2) INFORMATION FOR SEQ ID NO:91:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 105 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:91:  - - CGCGAATTCG GAAGACCCGT TCTTCACCAT CTTCGACGAG ACTAAGAGCT GG -#TACTTCAC     60   - - CGAGAACATG GAGCGCAACT GCCGCGCCCC CTGCAACATC CAGAT   - #105  - -  - - (2) INFORMATION FOR SEQ ID NO:92:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 104 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:92:  - - GGGGATCCTC ACGTCTCACA GGGTGTCCAT GATGTAGCCG TTGATGGCGT GG -#AAGCGGTA     60   - - GTTCTCCTTG AAGGTGGGAT CTTCCATCTG GATGTTGCAG GGGG   - #- #104  - -  - - (2) INFORMATION FOR SEQ ID NO:93:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 105 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:93:  - - CGCGAATTCG GAAGACCCCC TGCCCGGCCT GGTGATGGCC CAGGACCAGC GC -#ATCCGCTG     60   - - GTACCTGCTG TCTATGGGCA GCAACGAGAA CATCCACAGC ATCCA   - #105  - -  - - (2) INFORMATION FOR SEQ ID NO:94:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 104 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:94:  - - GGGGATCCTC ACGTCTCAGT ACAGGTTGTA CAGGGCCATC TTGTACTCCT CC -#TTCTTGCG     60   - - CACGGTGAAA ACGTGGCCGC TGAAGTGGAT GCTGTGGATG TTCT   - #- #104  - -  - - (2) INFORMATION FOR SEQ ID NO:95:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 106 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:95:  - - CGCGAATTCG GAAGACCCGT ACCCCGGCGT RGTTCGAGAC TGTGGAGATG CT -#GCCCAGCA     60   - - AGGCCGGGAT CTGGCGCGTG GAGTGCCTGA TCGGCGAGCA CCTGCA   - #106  - -  - - (2) INFORMATION FOR SEQ ID NO:96:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 104 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:96:  - - GGGGATCCTC ACGTCTCAGC TGGCCATGCC CAGGGGGGTC TGGCACTTGT TG -#CTGTACAC     60   - - CAGGAACAGG GTGCTCATGC CGGCGTGCAG GTGCTCGCCG ATCA   - #- #104  - -  - - (2) INFORMATION FOR SEQ ID NO:97:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 106 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:97:  - - CGCGAATTCG GAAGACCCCA GCGGCCACAT RCCGCGACTT CCAGATCACC GC -#CAGCGGCC     60   - - AGTACGGCCA GTGGGCTCCC AAGCTGGCCC GCCTGCACTA CAGCGG   - #106  - -  - - (2) INFORMATION FOR SEQ ID NO:98:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 104 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:98:  - - GGGGATCCTC ACGTCTCACA TGGGGGCCAG CAGGTCCACC TTGATCCAGG AG -#AAGGGCTC     60   - - CTTGGTCGAC CAGGCGTTGA TGCTGCCGCT GTAGTGCAGG CGGG   - #- #104  - -  - - (2) INFORMATION FOR SEQ ID NO:99:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 105 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:99:  - - CGCGAATTCG GAAGACCCCA TGATCATCCA CGGCATCAAG ACCCAGGGCG CC -#CGCCAGAA     60   - - GTTCAGCAGC CTGTACATCA GCCAGTTCAT CATCATGTAC TCTCT   - #105  - -  - - (2) INFORMATION FOR SEQ ID NO:100:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 104 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:100:  - - GGGGATCCTC ACGTCTCAGT TGCCGAAGAA CACCATCAGG GTGCCGGTGC TG -#TTGCCGCG     60   - - GTAGGTCTGC CACTTCTTGC CGTCTAGAGA GTACATGATG ATGA   - #- #104  - -  - - (2) INFORMATION FOR SEQ ID NO:101:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 105 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:101:  - - CGCGAATTCG GAAGACCCCA ACGTGGACAG CAGCGGCATC AAGCACAACA TC -#TTCAACCC     60   - - CCCCATCATC GCCCGCTACA TCCGCCTGCA CCCCACCCAC TACAG   - #105  - -  - - (2) INFORMATION FOR SEQ ID NO:102:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 104 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:102:  - - GGGGATCCTC ACGTCTCAGC CCAGGGGCAT GCTGCAGCTG TTCAGGTCGC AG -#CCCATCAG     60   - - CTCCATGCGC AGGGTGCTGC GGATGCTGTA GTGGGTGGGG TGCA   - #- #104  - -  - - (2) INFORMATION FOR SEQ ID NO:103:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 105 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:103:  - - CGCGAATTCG GAAGACCCGG GCATGGAGAG CAAGGCCATC AGCGACGCCC AG -#ATCACCGC     60   - - CTCCAGCTAC TTCACCAACA TGTTCGCCAC CTGGAGCCCC AGCAA   - #105  - -  - - (2) INFORMATION FOR SEQ ID NO:104:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 104 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:104:  - - GGGGATCCTC ACGTCTCACC ACTCCTTGGG GTTGTTCACC TGGGGGCGCC AG -#GCGTTGCT     60   - - GCGGCCCTGC AGGTGCAGGC GGGCCTTGCT GGGGCTCCAG GTGG   - #- #104  - -  - - (2) INFORMATION FOR SEQ ID NO:105:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 105 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:105:  - - CGCGAATTCG GAAGACCCGT GGCTGCAGGT GGACTTCCAG AAAACCATGA AG -#GTGACTGG     60   - - CGTGACCACC CAGGGCGTCA AGAGCCTGCT GACCAGCATG TACGT   - #105  - -  - - (2) INFORMATION FOR SEQ ID NO:106:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 104 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:106:  - - GGGGATCCTC ACGTCTCACT TGCCGTTTTG GAAGAACAGG GTCCACTGGT GG -#CCGTCCTG     60   - - GCTGCTGCTG ATCAGGAACT CCTTCACGTA CATGCTGGTC AGCA   - #- #104  - -  - - (2) INFORMATION FOR SEQ ID NO:107:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 105 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:107:  - - CGCGAATTCG GAAGACCCCA AGGTGAAGGT GTTCCAGGGC AACCAGGACA GC -#TTCACACC     60   - - GGTCGTGAAC AGCCTGGACC CCCCCCTGCT GACCCGCTAC CTGCG   - #105  - -  - - (2) INFORMATION FOR SEQ ID NO:108:  - -      (i) SEQUENCE CHARACTERISTICS:      (A) LENGTH: 125 base - #pairs      (B) TYPE: nucleic acid      (C) STRANDEDNESS: single      (D) TOPOLOGY: linear  - -     (ii) MOLECULE TYPE: Other  - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:108:  - - GGGGATCCTC ACGTCTCAGC GGCCGCTTCA GTACAGGTCC TGGGCCTCGC AG -#CCCAGCAC     60   - - CTCCATGCGC AGGGCGATCT GGTGCACCCA GCTCTGGGGG TGGATGCGCA GG -#TAGCGGGT    120   - - CAGCA                 - #                  - #                  -#           125   - -  - - (2) INFORMATION FOR SEQ ID NO:109:   - -      (i) SEQUENCE CHARACTERISTICS:       (A) LENGTH: 36 base - #pairs       (B) TYPE: nucleic acid       (C) STRANDEDNESS: single       (D) TOPOLOGY: linear   - -     (ii) MOLECULE TYPE: Other   - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:109:   - - CGTTGTTCTT CATACGCGTC TGGGGCTCCT CGGGGC      - #- #       36   - -  - - (2) INFORMATION FOR SEQ ID NO:110:   - -      (i) SEQUENCE CHARACTERISTICS:       (A) LENGTH: 18 base - #pairs       (B) TYPE: nucleic acid       (C) STRANDEDNESS: single       (D) TOPOLOGY: linear   - -     (ii) MOLECULE TYPE: Other   - -     (xi) SEQUENCE DESCRIPTION: SEQ ID NO:110:   - - CGCGAATTCG GAAGACCC             - #                  - #  - #  18__________________________________________________________________________</pre>
    
    </div></div></div><div class="patent-section patent-tabular-section"><a id="backward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Cited Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4652639">US4652639</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">May 6, 1982</td><td class="patent-data-table-td patent-date-value">Mar 24, 1987</td><td class="patent-data-table-td ">Amgen</td><td class="patent-data-table-td ">Manufacture and expression of structural genes</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5082767">US5082767</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Feb 27, 1989</td><td class="patent-data-table-td patent-date-value">Jan 21, 1992</td><td class="patent-data-table-td ">Hatfield G Wesley</td><td class="patent-data-table-td ">Codon pair utilization</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5270171">US5270171</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jun 18, 1990</td><td class="patent-data-table-td patent-date-value">Dec 14, 1993</td><td class="patent-data-table-td ">Boris Cercek</td><td class="patent-data-table-td ">Cancer-associated SCM-recognition factor, preparation and method of use</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5276268">US5276268</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jul 29, 1991</td><td class="patent-data-table-td patent-date-value">Jan 4, 1994</td><td class="patent-data-table-td ">Hoechst Aktiengesellschaft</td><td class="patent-data-table-td ">Phosphinothricin-resistance gene, and its use</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5405776">US5405776</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jan 24, 1992</td><td class="patent-data-table-td patent-date-value">Apr 11, 1995</td><td class="patent-data-table-td ">Life Technologies, Inc.</td><td class="patent-data-table-td ">Genetic engineering</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5464774">US5464774</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Dec 31, 1987</td><td class="patent-data-table-td patent-date-value">Nov 7, 1995</td><td class="patent-data-table-td ">The Salk Institute For Biological Studies</td><td class="patent-data-table-td ">DNA sequence</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5786464">US5786464</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Sep 19, 1994</td><td class="patent-data-table-td patent-date-value">Jul 28, 1998</td><td class="patent-data-table-td ">The General Hospital Corporation</td><td class="patent-data-table-td ">Overexpression of mammalian and viral proteins</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP0335635A1?cl=en">EP0335635A1</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Mar 28, 1989</td><td class="patent-data-table-td patent-date-value">Oct 4, 1989</td><td class="patent-data-table-td ">The Board Of Trustees Of The Leland Stanford Junior University</td><td class="patent-data-table-td ">Mutated HIV envelope protein</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP0345242A2?cl=en">EP0345242A2</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jun 2, 1989</td><td class="patent-data-table-td patent-date-value">Dec 6, 1989</td><td class="patent-data-table-td ">Smithkline Biologicals S.A.</td><td class="patent-data-table-td ">Expression of gag proteins from retroviruses in eucaryotic cells</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1996009378A1?cl=en">WO1996009378A1</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Sep 8, 1995</td><td class="patent-data-table-td patent-date-value">Mar 28, 1996</td><td class="patent-data-table-td ">Gen Hospital Corp</td><td class="patent-data-table-td ">Overexpression of mammalian and viral proteins</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1996027675A1?cl=en">WO1996027675A1</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Mar 4, 1996</td><td class="patent-data-table-td patent-date-value">Sep 12, 1996</td><td class="patent-data-table-td ">James Phillip Haseloff</td><td class="patent-data-table-td ">Jellyfish green fluorescent protein (gfp) expression in plants</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1997026333A1?cl=en">WO1997026333A1</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jan 17, 1997</td><td class="patent-data-table-td patent-date-value">Jul 24, 1997</td><td class="patent-data-table-td ">William W Hauswirth</td><td class="patent-data-table-td ">Humanized green fluorescent protein genes and methods</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="npl-citations"></a><div class="patent-section-header"><span class="patent-section-title">Non-Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th colspan="3"class="patent-data-table-th">Reference</th></tr></thead><tr><td class="patent-data-table-td ">1</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Bosch et al., Journal of Virology 68:7566 7569 (1994).</td></tr><tr><td class="patent-data-table-td ">2</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Bosch et al., Journal of Virology 68:7566-7569 (1994).</td></tr><tr><td class="patent-data-table-td ">3</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Chiu et al., "<a href='http://scholar.google.com/scholar?q="Engineered+GFP+as+a+Vital+Reporter+in+Plants"'>Engineered GFP as a Vital Reporter in Plants</a>" Current Biology 6:325-330 (1996).</td></tr><tr><td class="patent-data-table-td ">4</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Chiu et al., Engineered GFP as a Vital Reporter in Plants Current Biology 6:325 330 (1996).</td></tr><tr><td class="patent-data-table-td ">5</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Chou et al., AIDS Research and Human Retroviruses 8:1967 1976 (1992).</td></tr><tr><td class="patent-data-table-td ">6</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Chou et al., AIDS Research and Human Retroviruses 8:1967-1976 (1992).</td></tr><tr><td class="patent-data-table-td ">7</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Cochrane et al., J. of Virology 65 (10):5305 5314 (1991).</td></tr><tr><td class="patent-data-table-td ">8</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Cochrane et al., J. of Virology 65 (10):5305-5314 (1991).</td></tr><tr><td class="patent-data-table-td ">9</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Cohen et al. (abstract), in Modern Approaches New Vaccines, 64 (1988).</td></tr><tr><td class="patent-data-table-td ">10</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Coulombe and Skup, Gene 46:89 95 (1986).</td></tr><tr><td class="patent-data-table-td ">11</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Coulombe and Skup, Gene 46:89-95 (1986).</td></tr><tr><td class="patent-data-table-td ">12</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">D Agostino et al., Mol. And Cell. Biol. 12(3):1375 1386 (1992).</td></tr><tr><td class="patent-data-table-td ">13</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">D'Agostino et al., Mol. And Cell. Biol. 12(3):1375-1386 (1992).</td></tr><tr><td class="patent-data-table-td ">14</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Feinberg et al., Cell 46:807 817 (1986).</td></tr><tr><td class="patent-data-table-td ">15</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Feinberg et al., Cell 46:807-817 (1986).</td></tr><tr><td class="patent-data-table-td ">16</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Fortkamp et al., DNA 5:511 517 (1986).</td></tr><tr><td class="patent-data-table-td ">17</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Fortkamp et al., DNA 5:511-517 (1986).</td></tr><tr><td class="patent-data-table-td ">18</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Grantham et al., Nature 319:727 728 (1986).</td></tr><tr><td class="patent-data-table-td ">19</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Grantham et al., Nature 319:727-728 (1986).</td></tr><tr><td class="patent-data-table-td ">20</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Haas et al., "<a href='http://scholar.google.com/scholar?q="Codon+Usage+Limitation+in+the+Expression+of+HIV-1+Envelope+Glycoprotein"'>Codon Usage Limitation in the Expression of HIV-1 Envelope Glycoprotein</a>" Current Biology 6:315-324 (1996).</td></tr><tr><td class="patent-data-table-td ">21</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Haas et al., Codon Usage Limitation in the Expression of HIV 1 Envelope Glycoprotein Current Biology 6:315 324 (1996).</td></tr><tr><td class="patent-data-table-td ">22</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Hammarskjold et al. J. of Virol. 63(5):1959 1966 (1989).</td></tr><tr><td class="patent-data-table-td ">23</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Hammarskjold et al. J. of Virol. 63(5):1959-1966 (1989).</td></tr><tr><td class="patent-data-table-td ">24</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Haseloff et al., "<a href='http://scholar.google.com/scholar?q="GFP+in+Plants"'>GFP in Plants</a>" Trends in Genetics 11:328-329 (1995).</td></tr><tr><td class="patent-data-table-td ">25</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Haseloff et al., GFP in Plants Trends in Genetics 11:328 329 (1995).</td></tr><tr><td class="patent-data-table-td ">26</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Hernan et al., Biochemistry 31:8619 8628 (1992).</td></tr><tr><td class="patent-data-table-td ">27</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Hernan et al., Biochemistry 31:8619-8628 (1992).</td></tr><tr><td class="patent-data-table-td ">28</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Holler et al., Gene 136:323 328 (1993).</td></tr><tr><td class="patent-data-table-td ">29</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Holler et al., Gene 136:323-328 (1993).</td></tr><tr><td class="patent-data-table-td ">30</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Inouye et al., FEBS Letters 341:277 280 (1994).</td></tr><tr><td class="patent-data-table-td ">31</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Inouye et al., FEBS Letters 341:277-280 (1994).</td></tr><tr><td class="patent-data-table-td ">32</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Kamiya et al., Jpn. J. Cancer Res. 80:200 203 (1989).</td></tr><tr><td class="patent-data-table-td ">33</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Kamiya et al., Jpn. J. Cancer Res. 80:200-203 (1989).</td></tr><tr><td class="patent-data-table-td ">34</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Kypr et al., Nature 327:20 (1987).</td></tr><tr><td class="patent-data-table-td ">35</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">McCarrey, Nucleic Acids Res. 18:949 955 (1990).</td></tr><tr><td class="patent-data-table-td ">36</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">McCarrey, Nucleic Acids Res. 18:949-955 (1990).</td></tr><tr><td class="patent-data-table-td ">37</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Nakamura et al., FEBS Letters 289(1):123 125 (1991).</td></tr><tr><td class="patent-data-table-td ">38</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Nakamura et al., FEBS Letters 289(1):123-125 (1991).</td></tr><tr><td class="patent-data-table-td ">39</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Newgard et al., Proc. Natl. Acad. Sci. USA 83:8132 8136 (1986).</td></tr><tr><td class="patent-data-table-td ">40</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Newgard et al., Proc. Natl. Acad. Sci. USA 83:8132-8136 (1986).</td></tr><tr><td class="patent-data-table-td ">41</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Rangwala et al., Gene 122:263 269 (1992).</td></tr><tr><td class="patent-data-table-td ">42</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Rangwala et al., Gene 122:263-269 (1992).</td></tr><tr><td class="patent-data-table-td ">43</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Robinson et al., Nucl. Acids. Res. 12(17):6663 6671 (1984).</td></tr><tr><td class="patent-data-table-td ">44</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Robinson et al., Nucl. Acids. Res. 12(17):6663-6671 (1984).</td></tr><tr><td class="patent-data-table-td ">45</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Scorer et al., Gene 136:111 119 (1993).</td></tr><tr><td class="patent-data-table-td ">46</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Scorer et al., Gene 136:111-119 (1993).</td></tr><tr><td class="patent-data-table-td ">47</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Seetharam et al., "<a href='http://scholar.google.com/scholar?q="Mistranslation+in+IGF-1+During+Over-Expression+of+the+Protein+in+Escherichia+Coli+Using+a+Synthetic+Gene+Containing+Low+Frequency+Codons%2C"'>Mistranslation in IGF-1 During Over-Expression of the Protein in Escherichia Coli Using a Synthetic Gene Containing Low Frequency Codons,</a>" Biochem. Biophys. Res. Comm. 155:518-523 (1988).</td></tr><tr><td class="patent-data-table-td ">48</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Seetharam et al., Mistranslation in IGF 1 During Over Expression of the Protein in Escherichia Coli Using a Synthetic Gene Containing Low Frequency Codons, Biochem. Biophys. Res. Comm. 155:518 523 (1988).</td></tr><tr><td class="patent-data-table-td ">49</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Sharp et al., Nucleic Acids Res. 16:8207 8211 (1988).</td></tr><tr><td class="patent-data-table-td ">50</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Sharp et al., Nucleic Acids Res. 16:8207-8211 (1988).</td></tr><tr><td class="patent-data-table-td ">51</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Sharp, Nature 324:114 (1986).</td></tr><tr><td class="patent-data-table-td ">52</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">van Hemert et al., Journal of Molecular Evolution 41:132 140 (1995).</td></tr><tr><td class="patent-data-table-td ">53</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">van Hemert et al., Journal of Molecular Evolution 41:132-140 (1995).</td></tr><tr><td class="patent-data-table-td ">54</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Williams et al., Nucleic Acids Research 16/22:10453 10467 (1988).</td></tr><tr><td class="patent-data-table-td ">55</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Williams et al., Nucleic Acids Research 16/22:10453-10467 (1988).</td></tr><tr><td class="patent-data-table-td ">56</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Zhang et al., Gene 105(1):61 72 (1991).</td></tr><tr><td class="patent-data-table-td ">57</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Zhang et al., Gene 105(1):61-72 (1991).</td></tr><tr><td class="patent-data-table-td ">58</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Zhang et al., J. of Protein Chem. 12(3):329 335 (1993).</td></tr><tr><td class="patent-data-table-td ">59</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Zhang et al., J. of Protein Chem. 12(3):329-335 (1993).</td></tr><tr><td class="patent-data-table-td ">60</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Zolotukhin et al., "<a href='http://scholar.google.com/scholar?q="A+%22Humanized%22+Green+Fluorescent+Protein+cDNA+Adapted+for+High-Level+Expression+in+Mammalian+Cells"'>A "Humanized" Green Fluorescent Protein cDNA Adapted for High-Level Expression in Mammalian Cells</a>" Journal of Virology 70:4646-4654 (1996).</td></tr><tr><td class="patent-data-table-td ">61</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Zolotukhin et al., A Humanized Green Fluorescent Protein cDNA Adapted for High Level Expression in Mammalian Cells Journal of Virology 70:4646 4654 (1996).</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="forward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Referenced by</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Citing Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6534312">US6534312</a></td><td class="patent-data-table-td patent-date-value">Jun 28, 1999</td><td class="patent-data-table-td patent-date-value">Mar 18, 2003</td><td class="patent-data-table-td ">Merck &amp; Co., Inc.</td><td class="patent-data-table-td ">Vaccines comprising synthetic genes</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6602705">US6602705</a></td><td class="patent-data-table-td patent-date-value">Dec 30, 1999</td><td class="patent-data-table-td patent-date-value">Aug 5, 2003</td><td class="patent-data-table-td ">Chiron Corporation</td><td class="patent-data-table-td ">Expression of HIV polypeptides and production of virus-like particles</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6656706">US6656706</a></td><td class="patent-data-table-td patent-date-value">Jun 1, 2001</td><td class="patent-data-table-td patent-date-value">Dec 2, 2003</td><td class="patent-data-table-td ">The United States Of America As Represented By The Department Of Health And Human Services</td><td class="patent-data-table-td ">Molecular clones with mutated HIV gag/pol, SIV gag and SIV env genes</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6686684">US6686684</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Dec 22, 2000</td><td class="patent-data-table-td patent-date-value">Feb 3, 2004</td><td class="patent-data-table-td ">Koninklijke Philips Electronics N.V.</td><td class="patent-data-table-td ">Color selection electrode implementing a temperature based tension reduction of a shadow mask</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6689879">US6689879</a></td><td class="patent-data-table-td patent-date-value">Dec 30, 1999</td><td class="patent-data-table-td patent-date-value">Feb 10, 2004</td><td class="patent-data-table-td ">Chiron Corporation</td><td class="patent-data-table-td ">Nucleotide sequences coding nvelope protein for use in the prevention of viral diseases</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6911538">US6911538</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Feb 19, 2002</td><td class="patent-data-table-td patent-date-value">Jun 28, 2005</td><td class="patent-data-table-td ">Washington University</td><td class="patent-data-table-td ">Alternative (wild type) open reading frames are optimized for codon usage and expression of transcription factor and tumor supressor protein p53 proteins; useful in cancer therapy</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7132262">US7132262</a></td><td class="patent-data-table-td patent-date-value">Jul 30, 2003</td><td class="patent-data-table-td patent-date-value">Nov 7, 2006</td><td class="patent-data-table-td ">Smithkline Beecham Corp.</td><td class="patent-data-table-td ">administering codon optimized polynucleotides that code human papillomavirus polypeptides comprising early proteins or fragments to cells of mammals; gene therapy; DNA vaccines</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7205278">US7205278</a></td><td class="patent-data-table-td patent-date-value">Jun 14, 2002</td><td class="patent-data-table-td patent-date-value">Apr 17, 2007</td><td class="patent-data-table-td ">The Scripps Research Institute</td><td class="patent-data-table-td ">Introducing one or more cysteine residues into a polypeptide which permit the stabilization of the polypeptide by formation of at least one bond, preferably a disulfide bond, between different domains of the polypeptide.</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7211659">US7211659</a></td><td class="patent-data-table-td patent-date-value">Jul 5, 2002</td><td class="patent-data-table-td patent-date-value">May 1, 2007</td><td class="patent-data-table-td ">Chiron Corporation</td><td class="patent-data-table-td ">Polynucleotides encoding antigenic HIV type C polypeptides, polypeptides and uses thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7282364">US7282364</a></td><td class="patent-data-table-td patent-date-value">Oct 29, 2004</td><td class="patent-data-table-td patent-date-value">Oct 16, 2007</td><td class="patent-data-table-td ">Novartis Vaccines And Diagnostics, Inc.</td><td class="patent-data-table-td ">Expression cassette generated from these sequences; generating an immune response in a subject</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7316925">US7316925</a></td><td class="patent-data-table-td patent-date-value">Jul 15, 2003</td><td class="patent-data-table-td patent-date-value">Jan 8, 2008</td><td class="patent-data-table-td ">Vgx Pharmaceuticals, Inc.</td><td class="patent-data-table-td ">Codon optimized synthetic plasmids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7348177">US7348177</a></td><td class="patent-data-table-td patent-date-value">Mar 11, 2003</td><td class="patent-data-table-td patent-date-value">Mar 25, 2008</td><td class="patent-data-table-td ">Novartis Vaccines And Diagnostics, Inc.</td><td class="patent-data-table-td ">Expression of HIV polypeptides and production of virus-like particles</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7354594">US7354594</a></td><td class="patent-data-table-td patent-date-value">May 27, 2005</td><td class="patent-data-table-td patent-date-value">Apr 8, 2008</td><td class="patent-data-table-td ">Gtc Biotherapeutics, Inc.</td><td class="patent-data-table-td ">Merozoite surface protein 1 lacking glycosylation sites</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7501553">US7501553</a></td><td class="patent-data-table-td patent-date-value">Feb 20, 2002</td><td class="patent-data-table-td patent-date-value">Mar 10, 2009</td><td class="patent-data-table-td ">Gtc Biotherapeutics, Inc.</td><td class="patent-data-table-td ">Non-human transgenic mammal comprising a modified MSP-1 nucleic acid</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7541446">US7541446</a></td><td class="patent-data-table-td patent-date-value">Jun 5, 2003</td><td class="patent-data-table-td patent-date-value">Jun 2, 2009</td><td class="patent-data-table-td ">Tet Systems Holding Gmbh &amp; Co. Kg</td><td class="patent-data-table-td ">Tet repressor-based transcriptional regulatory proteins</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7561972">US7561972</a></td><td class="patent-data-table-td patent-date-value">Jul 31, 2008</td><td class="patent-data-table-td patent-date-value">Jul 14, 2009</td><td class="patent-data-table-td ">Dna Twopointo, Inc.</td><td class="patent-data-table-td ">designing a polynucleotide sequence that encodes a polypeptide sequence; When the respective sequence element encodes a portion of polypeptide sequence, the sequence element is incorporated into the polynucleotide at a frequency of occurrence that is within the frequency range specified; computer medium</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7561973">US7561973</a></td><td class="patent-data-table-td patent-date-value">Jul 31, 2008</td><td class="patent-data-table-td patent-date-value">Jul 14, 2009</td><td class="patent-data-table-td ">Dna Twopointo, Inc.</td><td class="patent-data-table-td ">designing nucleic acids with desired properties, particularly for the expression of encoded proteins; drug design; reliable algorithm with which to assess the likely level of protein expressed from a synthetic gene</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7608422">US7608422</a></td><td class="patent-data-table-td patent-date-value">Aug 19, 2003</td><td class="patent-data-table-td patent-date-value">Oct 27, 2009</td><td class="patent-data-table-td ">The United States Of America As Represented By The Secretary Of The Department Of Health And Human Services</td><td class="patent-data-table-td ">Gene therapy; vaccines; inducing antibodies and/or cytotoxic or helper T lymphocytes</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7632980">US7632980</a></td><td class="patent-data-table-td patent-date-value">Oct 20, 1998</td><td class="patent-data-table-td patent-date-value">Dec 15, 2009</td><td class="patent-data-table-td ">Gtc Biotherapeutics, Inc.</td><td class="patent-data-table-td ">Modified nucleic acid sequences and methods for increasing mRNA levels and protein expression in cell systems</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7662916">US7662916</a></td><td class="patent-data-table-td patent-date-value">Jul 25, 2003</td><td class="patent-data-table-td patent-date-value">Feb 16, 2010</td><td class="patent-data-table-td ">Novartis Vaccines &amp; Diagnostics, Inc</td><td class="patent-data-table-td ">Modified HIV Env polypeptides</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7718401">US7718401</a></td><td class="patent-data-table-td patent-date-value">Feb 6, 2008</td><td class="patent-data-table-td patent-date-value">May 18, 2010</td><td class="patent-data-table-td ">Novartis Vaccines And Diagnostics, Inc.</td><td class="patent-data-table-td ">Expression of HIV polypeptides and production of virus-like particles</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7728118">US7728118</a></td><td class="patent-data-table-td patent-date-value">Sep 17, 2004</td><td class="patent-data-table-td patent-date-value">Jun 1, 2010</td><td class="patent-data-table-td ">Promega Corporation</td><td class="patent-data-table-td ">Expression vector comprising modified luciferase for use as gene transfer and report agent in heterologous gene expression</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7825233">US7825233</a></td><td class="patent-data-table-td patent-date-value">Jun 9, 2005</td><td class="patent-data-table-td patent-date-value">Nov 2, 2010</td><td class="patent-data-table-td ">Allergan, Inc.</td><td class="patent-data-table-td ">Optimizing expression of active Botulinum Toxin type E</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7846720">US7846720</a></td><td class="patent-data-table-td patent-date-value">Jan 26, 2006</td><td class="patent-data-table-td patent-date-value">Dec 7, 2010</td><td class="patent-data-table-td ">Vgx Pharmaceuticals, Inc.</td><td class="patent-data-table-td ">Optimized high yield synthetic plasmids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7879540">US7879540</a></td><td class="patent-data-table-td patent-date-value">Aug 24, 2000</td><td class="patent-data-table-td patent-date-value">Feb 1, 2011</td><td class="patent-data-table-td ">Promega Corporation</td><td class="patent-data-table-td ">Nucleotide sequences modified to present reduced transcriptional characteristics subsequent to gene expression</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7906282">US7906282</a></td><td class="patent-data-table-td patent-date-value">Apr 12, 2007</td><td class="patent-data-table-td patent-date-value">Mar 15, 2011</td><td class="patent-data-table-td ">Promega Corporation</td><td class="patent-data-table-td ">Synthetic nucleic acid molecule compositions and methods of preparation</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7928199">US7928199</a></td><td class="patent-data-table-td patent-date-value">Jan 25, 2007</td><td class="patent-data-table-td patent-date-value">Apr 19, 2011</td><td class="patent-data-table-td ">The Scripps Research Institute</td><td class="patent-data-table-td ">Stabilized proteins with engineered disulfide bonds</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7935805">US7935805</a></td><td class="patent-data-table-td patent-date-value">Jul 5, 2000</td><td class="patent-data-table-td patent-date-value">May 3, 2011</td><td class="patent-data-table-td ">Novartis Vaccines &amp; Diagnostics, Inc</td><td class="patent-data-table-td ">Polynucleotides encoding antigenic HIV Type C polypeptides, polypeptides and uses thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7943375">US7943375</a></td><td class="patent-data-table-td patent-date-value">Jun 25, 2008</td><td class="patent-data-table-td patent-date-value">May 17, 2011</td><td class="patent-data-table-td ">Novartis Vaccines &amp; Diagnostics, Inc</td><td class="patent-data-table-td ">Polynucleotides encoding antigenic HIV type C polypeptides, polypeptides and uses thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8008006">US8008006</a></td><td class="patent-data-table-td patent-date-value">Jul 5, 2007</td><td class="patent-data-table-td patent-date-value">Aug 30, 2011</td><td class="patent-data-table-td ">Promega Corporation</td><td class="patent-data-table-td ">Synthetic nucleic acid molecule compositions and methods of preparation</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8030017">US8030017</a></td><td class="patent-data-table-td patent-date-value">Jul 31, 2009</td><td class="patent-data-table-td patent-date-value">Oct 4, 2011</td><td class="patent-data-table-td ">Promega Corporation</td><td class="patent-data-table-td ">Thermostable luciferases and methods of production</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8076100">US8076100</a></td><td class="patent-data-table-td patent-date-value">Oct 2, 2002</td><td class="patent-data-table-td patent-date-value">Dec 13, 2011</td><td class="patent-data-table-td ">The United States Of America As Represented By The Department Of Health And Human Services</td><td class="patent-data-table-td ">Genetic engineering; vaccine; induction antibodies</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8106176">US8106176</a></td><td class="patent-data-table-td patent-date-value">Oct 3, 2006</td><td class="patent-data-table-td patent-date-value">Jan 31, 2012</td><td class="patent-data-table-td ">Instituto di Richerche di Biologia Molecolare P. Angeletti SpA</td><td class="patent-data-table-td ">Matrix metalloproteinase 11 vaccine</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8126653">US8126653</a></td><td class="patent-data-table-td patent-date-value">Jul 31, 2008</td><td class="patent-data-table-td patent-date-value">Feb 28, 2012</td><td class="patent-data-table-td ">Dna Twopointo, Inc.</td><td class="patent-data-table-td ">Synthetic nucleic acids for expression of encoded proteins</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8133494">US8133494</a></td><td class="patent-data-table-td patent-date-value">Mar 13, 2007</td><td class="patent-data-table-td patent-date-value">Mar 13, 2012</td><td class="patent-data-table-td ">Novartis Vaccine &amp; Diagnostics Inc</td><td class="patent-data-table-td ">For generating an immune response in a subject; synthetic expression cassettes generated from these and other HIV sequences; vaccines; immunogenic HIV Gag polypeptide</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8168418">US8168418</a></td><td class="patent-data-table-td patent-date-value">Apr 30, 2010</td><td class="patent-data-table-td patent-date-value">May 1, 2012</td><td class="patent-data-table-td ">Susan W Barnett</td><td class="patent-data-table-td ">Expression of HIV polypeptides and production of virus-like particles</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8263394">US8263394</a></td><td class="patent-data-table-td patent-date-value">Jan 23, 2008</td><td class="patent-data-table-td patent-date-value">Sep 11, 2012</td><td class="patent-data-table-td ">Novartis Vaccines &amp; Diagnostics Inc.</td><td class="patent-data-table-td ">Polynucleotides encoding antigenic HIV type B polypeptides, polypeptides, and uses thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8401798">US8401798</a></td><td class="patent-data-table-td patent-date-value">Jul 31, 2008</td><td class="patent-data-table-td patent-date-value">Mar 19, 2013</td><td class="patent-data-table-td ">Dna Twopointo, Inc.</td><td class="patent-data-table-td ">Systems and methods for constructing frequency lookup tables for expression systems</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8492522">US8492522</a></td><td class="patent-data-table-td patent-date-value">Dec 21, 2011</td><td class="patent-data-table-td patent-date-value">Jul 23, 2013</td><td class="patent-data-table-td ">Istituto Di Ricerche Di Biologia Molecolare P. Angeletti Spa</td><td class="patent-data-table-td ">Matrix metalloproteinase 11 vaccine</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8541230">US8541230</a></td><td class="patent-data-table-td patent-date-value">May 16, 2011</td><td class="patent-data-table-td patent-date-value">Sep 24, 2013</td><td class="patent-data-table-td ">Novartis Vaccines And Diagnostics, Inc</td><td class="patent-data-table-td ">Polynucleotides encoding antigenic HIV type C polypeptides, polypeptides and uses thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8563302">US8563302</a></td><td class="patent-data-table-td patent-date-value">Nov 17, 2010</td><td class="patent-data-table-td patent-date-value">Oct 22, 2013</td><td class="patent-data-table-td ">Vgx Pharmaceuticals, Inc.</td><td class="patent-data-table-td ">Optimized high yield synthetic plasmids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2110385A1?cl=en">EP2110385A1</a></td><td class="patent-data-table-td patent-date-value">Jun 14, 2002</td><td class="patent-data-table-td patent-date-value">Oct 21, 2009</td><td class="patent-data-table-td ">The Scripps Research Institute</td><td class="patent-data-table-td ">Stabilized factor VIII with engineered disulfide bonds</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2385065A1?cl=en">EP2385065A1</a></td><td class="patent-data-table-td patent-date-value">Oct 15, 2008</td><td class="patent-data-table-td patent-date-value">Nov 9, 2011</td><td class="patent-data-table-td ">Perseid Therapeutics LLC</td><td class="patent-data-table-td ">Immunosuppressive polypeptides and nucleic acids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2612867A1?cl=en">EP2612867A1</a></td><td class="patent-data-table-td patent-date-value">Oct 15, 2008</td><td class="patent-data-table-td patent-date-value">Jul 10, 2013</td><td class="patent-data-table-td ">Perseid Therapeutics LLC</td><td class="patent-data-table-td ">Immunosuppressive polypeptides and nucleic acids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2612868A1?cl=en">EP2612868A1</a></td><td class="patent-data-table-td patent-date-value">Oct 15, 2008</td><td class="patent-data-table-td patent-date-value">Jul 10, 2013</td><td class="patent-data-table-td ">Perseid Therapeutics LLC</td><td class="patent-data-table-td ">Immunosuppressive polypeptides and nucleic acids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2002038769A2?cl=en">WO2002038769A2</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Sep 19, 2001</td><td class="patent-data-table-td patent-date-value">May 16, 2002</td><td class="patent-data-table-td ">Sophia Biemelt</td><td class="patent-data-table-td ">Dna sequences, which code for optimised eukaryotic hpv 16-l1 and hpv 16-l2</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2005007808A2?cl=en">WO2005007808A2</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">May 17, 2004</td><td class="patent-data-table-td patent-date-value">Jan 27, 2005</td><td class="patent-data-table-td ">Susan W Barnett</td><td class="patent-data-table-td ">Hiv polynucleotides and polypeptides derived from botswana mj4</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2005052171A2?cl=en">WO2005052171A2</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Oct 28, 2004</td><td class="patent-data-table-td patent-date-value">Jun 9, 2005</td><td class="patent-data-table-td ">Susan Kingsman</td><td class="patent-data-table-td ">Retroviral and lentiviral vectors</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2009058564A2?cl=en">WO2009058564A2</a></td><td class="patent-data-table-td patent-date-value">Oct 15, 2008</td><td class="patent-data-table-td patent-date-value">May 7, 2009</td><td class="patent-data-table-td ">Maxygen Inc</td><td class="patent-data-table-td ">Immunosuppressive polypeptides and nucleic acids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2012084923A1?cl=en">WO2012084923A1</a></td><td class="patent-data-table-td patent-date-value">Dec 20, 2011</td><td class="patent-data-table-td patent-date-value">Jun 28, 2012</td><td class="patent-data-table-td ">Geneart Ag</td><td class="patent-data-table-td ">Method for producing reading-frame-corrected fragment libraries</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2012098111A1?cl=en">WO2012098111A1</a></td><td class="patent-data-table-td patent-date-value">Jan 17, 2012</td><td class="patent-data-table-td patent-date-value">Jul 26, 2012</td><td class="patent-data-table-td ">Philip Morris Products S.A.</td><td class="patent-data-table-td ">Vectors for nucleic acid expression in plants</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2012098119A2?cl=en">WO2012098119A2</a></td><td class="patent-data-table-td patent-date-value">Jan 17, 2012</td><td class="patent-data-table-td patent-date-value">Jul 26, 2012</td><td class="patent-data-table-td ">Philip Morris Products S.A.</td><td class="patent-data-table-td ">Protein expression in plants</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2013049227A2?cl=en">WO2013049227A2</a></td><td class="patent-data-table-td patent-date-value">Sep 26, 2012</td><td class="patent-data-table-td patent-date-value">Apr 4, 2013</td><td class="patent-data-table-td ">Geneart Ag</td><td class="patent-data-table-td ">High efficiency, small volume nucleic acid synthesis</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="classifications"></a><div class="patent-section-header"><span class="patent-section-title">Classifications</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th"> </th><th class="patent-data-table-th"> </th></tr></thead><tr><td class="patent-data-table-td ">U.S. Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=6yZRBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S091100">435/91.1</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=6yZRBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S091400">435/91.4</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=6yZRBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S440000">435/440</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=6yZRBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S325000">435/325</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=6yZRBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S091410">435/91.41</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=6yZRBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S069100">435/69.1</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=6yZRBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S069600">435/69.6</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=6yZRBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S252300">435/252.3</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=6yZRBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S320100">435/320.1</a></span></td></tr><tr><td class="patent-data-table-td ">International Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=6yZRBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C07K0014160000">C07K14/16</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=6yZRBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0005100000">C12N5/10</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=6yZRBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C07K0014755000">C07K14/755</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=6yZRBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12P0021020000">C12P21/02</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=6yZRBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0015090000">C12N15/09</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=6yZRBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12R0001910000">C12R1/91</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=6yZRBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0015670000">C12N15/67</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=6yZRBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0015850000">C12N15/85</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=6yZRBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C07K0014435000">C07K14/435</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=6yZRBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0015330000">C12N15/33</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=6yZRBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0005070000">C12N5/07</a></span></td></tr><tr><td class="patent-data-table-td ">Cooperative Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=6yZRBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C07K14/755">C07K14/755</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=6yZRBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N2740/16122">C12N2740/16122</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=6yZRBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C07K14/005">C07K14/005</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=6yZRBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12P21/02">C12P21/02</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=6yZRBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C07K14/43595">C07K14/43595</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=6yZRBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C07K2319/00">C07K2319/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=6yZRBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N15/67">C12N15/67</a></span></td></tr><tr><td class="patent-data-table-td ">European Classification</td><td class="patent-data-table-td "><span class="nested-value">C07K14/005</span>, <span class="nested-value">C12N15/67</span>, <span class="nested-value">C12P21/02</span>, <span class="nested-value">C07K14/435A5</span>, <span class="nested-value">C07K14/755</span></td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="legal-events"></a><div class="patent-section-header"><span class="patent-section-title">Legal Events</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Date</th><th class="patent-data-table-th">Code</th><th class="patent-data-table-th">Event</th><th class="patent-data-table-th">Description</th></tr></thead><tr><td class="patent-data-table-td patent-date-value">Oct 23, 2012</td><td class="patent-data-table-td ">CC</td><td class="patent-data-table-td ">Certificate of correction</td><td class="patent-data-table-td "></td></tr><tr><td class="patent-data-table-td patent-date-value">Jul 17, 2012</td><td class="patent-data-table-td ">RR</td><td class="patent-data-table-td ">Request for reexamination filed</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20120601</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">May 1, 2012</td><td class="patent-data-table-td ">B1</td><td class="patent-data-table-td ">Reexamination certificate first reexamination</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">CLAIM 8 IS CANCELLED. CLAIMS 1 AND 18 ARE DETERMINED TO BE PATENTABLE AS AMENDED. CLAIMS 2-14 AND 16, DEPENDENT ON AN AMENDED CLAIM, ARE DETERMINED TO BE PATENTABLE. NEW CLAIMS 19-23 ARE ADDED AND DETERMINED TO BE PATENTABLE. CLAIMS 15 AND 17 WERE NOT REEXAMINED.</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Mar 5, 2012</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">12</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Feb 1, 2011</td><td class="patent-data-table-td ">RR</td><td class="patent-data-table-td ">Request for reexamination filed</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20101015</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Jan 18, 2011</td><td class="patent-data-table-td ">RR</td><td class="patent-data-table-td ">Request for reexamination filed</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20101019</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Mar 5, 2008</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">8</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Aug 24, 2004</td><td class="patent-data-table-td ">CC</td><td class="patent-data-table-td ">Certificate of correction</td><td class="patent-data-table-td "></td></tr><tr><td class="patent-data-table-td patent-date-value">Sep 29, 2003</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">4</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Apr 10, 2000</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">GENERAL HOSPITAL CORPORATION, THE, MASSACHUSETTS</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNOR:SEED, BRIAN;REEL/FRAME:010740/0811</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">19961115</span></div><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">GENERAL HOSPITAL CORPORATION, THE 55 FRUIT STREET</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Jul 26, 1999</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">GENERAL HOSPITAL CORPORATION, THE, MASSACHUSETTS</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNOR:HAAS, JURGEN;REEL/FRAME:010126/0059</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">19990519</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Aug 28, 1997</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">NATIONAL INSTITUTES OF HEALTH, THE, MARYLAND</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">CONFIRMATORY LICENSE;ASSIGNOR:MASSACHUSETTS GENERAL HOSPITAL, THE;REEL/FRAME:008692/0481</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">19970807</span></div></td></tr></table><div class="patent-section-footer"></div></div><div class="modal-dialog" id="patent-images-lightbox"><div class="patent-lightbox-controls"><div class="patent-lightbox-rotate-controls"><div class="patent-lightbox-rotation-text">Rotate</div><div class="rotate-icon rotate-ccw-icon"></div><div class="rotate-icon rotate-cw-icon"></div></div><div class="patent-lightbox-index-counter"></div><a class="patent-lightbox-fullsize-link" target="_blank">Original Image</a><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_right.png" alt="Next page"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_left.png" alt="Previous page"width="21" height="21" /></div></div><div class="modal-dialog-content"><div class="patent-lightbox-image-holder"><div class="patent-lightbox-placeholder"></div></div></div></div><script>_OC_initPatentsAtb({image_not_available_html: " Image not available"});</script></div></div></div></td></tr></table><script>(function() {var href = window.location.href;if (href.indexOf('?') !== -1) {var parameters = href.split('?')[1].split('&');for (var i = 0; i < parameters.length; i++) {var param = parameters[i].split('=');if (param[0] == 'focus') {var elem = document.getElementById(param[1]);if (elem) {elem.focus();}}}}})();</script><script>_OC_addFlags({LockSrc:"/books/javascript/lock_8a2b04e7bf975d5171d8e4c0b6365c7a.js", Host:"http://www.google.com/", IsBooksRentalEnabled:1, IsWebstoreDisplayCaseEnabled:1, IsObfuscationEnabled:1, IsBrowsingHistoryEnabled:1, IsWebReaderSvgEnabled:0, IsGeoLayerEnabled:1, IsImageModeNotesEnabled:1, IsCopyMenuItemEnabled:1, IsGiftingEnabled:0, IsWebReaderUniversalPaginatorEnabled:0, IsOfflineBubbleEnabled:1, IsReaderEnabledForPlayRequests:1, IsFutureOnSaleVolumesEnabled:1, IsOfflineRestrictedCopyEnabled:1, IsBooksUnifiedLeftNavEnabled:1, IsRestrictedCopyEnabled:1, IsZipitFolderCollectionEnabled:1, IsEndOfSampleRecommendationsEnabled:1, IsRatingsOnBookcardsEnabled:1, IsAdsDisabled:0, IsIframePageDisplayEnabled:0, IsEmbeddedMediaEnabled:1, IsImageModeAnnotationsEnabled:1, IsMyLibraryGooglePlusEnabled:1, IsImagePageProviderEnabled:0, IsBookcardListPriceSmall:0, IsInternalUser:0, IsBooksShareButtonEnabled:0, IsPreOrdersEnabled:0, IsDisabledRandomBookshelves:0, WebstoreDisplayCasePosition:3});_OC_Run({"enable_p13n":false,"add_vol_to_collection_base_url":"http://www.google.com/patents?op=add\u0026sig=ACfU3U1_xjXgL1hndWpxu_97B89qAnSXRw\u0026id=6yZRBAABERAJ","remove_vol_from_collection_base_url":"http://www.google.com/patents?op=remove\u0026sig=ACfU3U0YwKNJR8KZEWPkiA6u_vYoTWVloQ\u0026id=6yZRBAABERAJ","logged_in":false,"p13n_save_user_settings_url":"http://www.google.com/patents?op=edit_user_settings\u0026sig=ACfU3U3YNyJZHWGtlwp_vzvBz6a9oz92AQ","is_cobrand":false,"sign_in_url":"https://www.google.com/accounts/Login?service=\u0026continue=http://www.google.com/patents%3Fhl%3Den\u0026hl=en","is_play_enabled":true}, {"volume_id":"","is_ebook":true,"volumeresult":{"has_flowing_text":false,"has_scanned_text":true,"can_download_pdf":false,"can_download_epub":false,"is_pdf_drm_enabled":false,"is_epub_drm_enabled":false,"download_pdf_url":"http://www.google.com/patents/download/High_level_expression_of_proteins.pdf?id=6yZRBAABERAJ\u0026output=pdf\u0026sig=ACfU3U1hoNvIQwNub-i_HDAwMr5NCSLCsA"},"sample_url":"http://www.google.com/patents/reader?id=6yZRBAABERAJ\u0026printsec=frontcover\u0026output=reader\u0026source=gbs_atb_hover","is_browsable":true,"is_public_domain":true}, {});</script><div id="footer_table" style="font-size:83%;text-align:center;position:relative;top:20px;height:4.5em;margin-top:2em"><div style="margin-bottom:8px"><a href=http://www.google.com/><nobr>Google&nbsp;Home</nobr></a> - <a href=//www.google.com/patents/sitemap/><nobr>Sitemap</nobr></a> - <a href=http://www.google.com/googlebooks/uspto.html><nobr>USPTO Bulk Downloads</nobr></a> - <a href=/intl/en/privacy/><nobr>Privacy Policy</nobr></a> - <a href=/intl/en/policies/terms/><nobr>Terms of Service</nobr></a> - <a href=https://support.google.com/faqs/answer/2539193?hl=en><nobr>About Google Patents</nobr></a> - <a href="http://www.google.com/tools/feedback/intl/en/error.html" onclick="try{_OC_startFeedback({productId: '72792',locale: 'en'});return false;}catch(e){}"><nobr>Send Feedback</nobr></a></div><span>Data provided by IFI CLAIMS Patent Services</span><br><span >&copy;2012 Google</span></div> <script type="text/javascript">var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));</script><script type="text/javascript">var pageTracker = _gat._getTracker("UA-27188110-1");pageTracker._setCookiePath("/patents/");pageTracker._trackPageview();</script> </body></html>