<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN""http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html><head><title>Patent US6258569 - Amplifying using polymerase with 5&#39; to 3&#39; nuclease activity, a primer, and ... - Google Patents</title><script>(function(){(function(){function e(a){this.t={};this.tick=function(a,c,b){var d=void 0!=b?b:(new Date).getTime();this.t[a]=[d,c];if(void 0==b)try{window.console.timeStamp("CSI/"+a)}catch(e){}};this.tick("start",null,a)}var a;window.performance&&(a=window.performance.timing);var f=a?new e(a.responseStart):new e;window.jstiming={Timer:e,load:f};if(a){var c=a.navigationStart,d=a.responseStart;0<c&&d>=c&&(window.jstiming.srt=d-c)}if(a){var b=window.jstiming.load;0<c&&d>=c&&(b.tick("_wtsrt",void 0,c),b.tick("wtsrt_",
"_wtsrt",d),b.tick("tbsd_","wtsrt_"))}try{a=null,window.chrome&&window.chrome.csi&&(a=Math.floor(window.chrome.csi().pageT),b&&0<c&&(b.tick("_tbnd",void 0,window.chrome.csi().startE),b.tick("tbnd_","_tbnd",c))),null==a&&window.gtbExternal&&(a=window.gtbExternal.pageT()),null==a&&window.external&&(a=window.external.pageT,b&&0<c&&(b.tick("_tbnd",void 0,window.external.startE),b.tick("tbnd_","_tbnd",c))),a&&(window.jstiming.pt=a)}catch(g){}})();})();
</script><link rel="stylesheet" href="/patents/css/_8a2b04e7bf975d5171d8e4c0b6365c7a/kl_intl_patents_bundle.css" type="text/css" /><script src="/books/javascript/atb_8a2b04e7bf975d5171d8e4c0b6365c7a__en.js"></script><script>function googleTranslateElementInit() {new google.translate.TranslateElement({pageLanguage: "en",gaTrack: true,gaId: "UA-27188110-1",multilanguagePage: true});}</script><script src="//translate.google.com/translate_a/element.js?cb=googleTranslateElementInit"></script><meta name="DC.type" content="Patent"><meta name="DC.title" content="Amplifying using polymerase with 5&#39; to 3&#39; nuclease activity, a primer, and an oligonucleotide probe with fluorescent reporter and quencher attached at opposite ends; digestion of probe by polymerase increases signal"><meta name="DC.contributor" content="Kenneth J. Livak" scheme="inventor"><meta name="DC.contributor" content="Susan J. A. Flood" scheme="inventor"><meta name="DC.contributor" content="Jeffrey Mamoro" scheme="inventor"><meta name="DC.contributor" content="Khairuzzaman Bashar Mullah" scheme="inventor"><meta name="DC.contributor" content="The Perkin-Elmer Corporation" scheme="assignee"><meta name="DC.date" content="1999-11-8" scheme="dateSubmitted"><meta name="DC.description" content="Provided is a method of nucleic acid amplification. In one embodiment, the method comprises performing nucleic acid amplification on a target polynucleotide using a nucleic acid polymerase having 5′ to 3′ nuclease activity, a primer capable of hybridizing to the target polynucleotide, and an oligonucleotide probe under amplification conditions such that the probe hybridizes to the target polynucleotide 3′ relative to the primer and the probe does not hybridize with itself to form a hairpin structure. The oligonucleotide probe has at one end a fluorescent reporter and at the other end a quencher that quenches the fluorescence of the reporter molecule when both the fluorescent reporter and quencher are attached to the probe. Digestion of the oligonucleotide probe by the polymerase during amplification is effective to separate the reporter from the quencher, whereby a fluorescence signal of the reporter is increased."><meta name="DC.date" content="2001-7-10" scheme="issued"><meta name="DC.relation" content="AU:640089" scheme="references"><meta name="DC.relation" content="EP:0229943" scheme="references"><meta name="DC.relation" content="EP:0232967:A2" scheme="references"><meta name="DC.relation" content="EP:0343955:A2" scheme="references"><meta name="DC.relation" content="EP:0420102:A2" scheme="references"><meta name="DC.relation" content="EP:0457213:A2" scheme="references"><meta name="DC.relation" content="EP:0523557:A1" scheme="references"><meta name="DC.relation" content="EP:0601889:A2" scheme="references"><meta name="DC.relation" content="JP:H05123195" scheme="references"><meta name="DC.relation" content="US:4220450" scheme="references"><meta name="DC.relation" content="US:4996143" scheme="references"><meta name="DC.relation" content="US:5210015" scheme="references"><meta name="DC.relation" content="US:5332659" scheme="references"><meta name="DC.relation" content="US:5487972" scheme="references"><meta name="DC.relation" content="US:5491063" scheme="references"><meta name="DC.relation" content="US:5538848" scheme="references"><meta name="DC.relation" content="US:5565554" scheme="references"><meta name="DC.relation" content="US:5607834" scheme="references"><meta name="DC.relation" content="US:5654419" scheme="references"><meta name="DC.relation" content="US:5688648" scheme="references"><meta name="DC.relation" content="US:5723591" scheme="references"><meta name="DC.relation" content="US:5804375" scheme="references"><meta name="DC.relation" content="US:5876930" scheme="references"><meta name="DC.relation" content="US:5925517" scheme="references"><meta name="DC.relation" content="US:6008373" scheme="references"><meta name="DC.relation" content="US:6030787" scheme="references"><meta name="DC.relation" content="WO:1990003446:A1" scheme="references"><meta name="DC.relation" content="WO:1992002638:A1" scheme="references"><meta name="DC.relation" content="WO:1993013224:A1" scheme="references"><meta name="DC.relation" content="WO:1995003429:A1" scheme="references"><meta name="DC.relation" content="WO:1995021266:A1" scheme="references"><meta name="DC.relation" content="WO:1996030540:A2" scheme="references"><meta name="citation_reference" content="Agrawal and Zamecnik, &quot;Site Specific Functionalization of Olignucleotides for Attaching Two Different Reporter Groups,&quot; Nucl. Acids Res. 18(18):5419-5423 (1990)."><meta name="citation_reference" content="Cardullo et al., &quot;Detection of Nucleic Acid Hybridization by Nonradiative Fluorescence Resonance Energy Transfer,&quot; Proc. Natl. Acad. Sci. USA 85:8790-8794 (1988)."><meta name="citation_reference" content="Clegg et al., &quot;Observing the Helical Geometry of Double-Stranded DNA in Solution by Fluorescence Resonance Energy Transfer,&quot; Proc. Natl. Acad. Sci. USA 90:2994-2998 (1993)."><meta name="citation_reference" content="Clegg, &quot;Fluorescence Resonance Energy Transfer and Nucleic Acids,&quot; Methods of Enzymology 211:353-389 (1992)."><meta name="citation_reference" content="Database WPI, Sect. Ch., Wk. 8608, Derwent Publ. Ltd., London, GB Jan. 1986, JP 5015439 (1993) &quot;Determine Polynucleotide Single Strand Label Luminous Polynucleotide Substance&quot;."><meta name="citation_reference" content="Guo et al., &quot;Direct Fluores. Analy. of Genetic Polymor. by Hybrid. with oligonucl. arrays on glass supports,&quot; Nucl. Acids Res., 22(24) :5456-5465 (1994)."><meta name="citation_reference" content="Heller et al., &quot;Fluorescent Energy Transfer Oligonucleotide Probes,&quot; Abstract 248, Fed. Proc. 46:1968 (1987)."><meta name="citation_reference" content="Higuchi et al., &quot;Kinetic PCR Analysis: Real-Time Monitoring of DNA Amplification Reactions,&quot; Biotechnology 11:1026-1030 (1993)."><meta name="citation_reference" content="Higuchi et al., &quot;Simultaneous Amplification and Detection of Specific DNA Sequences,&quot; Biotechnology 10:413-417 (1992)."><meta name="citation_reference" content="Holland et al., &quot;Detection of Specific Polymerase Chain Reaction Product by Utilizing the 5&#39;-3&#39; Exonuclease Activity of Thermus aquaticus DNA Polymerase,&quot; Proc. Natl. Acad. Sci. USA 88:7276-7280 (1991)."><meta name="citation_reference" content="Holland et al., “Detection of Specific Polymerase Chain Reaction Product by Utilizing the 5′-3′ Exonuclease Activity of Thermus aquaticus DNA Polymerase,” Proc. Natl. Acad. Sci. USA 88:7276-7280 (1991)."><meta name="citation_reference" content="Ju et al., &quot;Design and Synthesis of Fluorescence Energy Transfer Dye-Labeled Primers and their Application for DNA Sequencing and Analysis,&quot; Anal. Biochem. 231:131-140 (1995)."><meta name="citation_reference" content="Lee et al. &quot;Allelic discrimination by nick-translation PCR with fluorogemic probes&quot; Nucleic acids research pp. 3761-3766, 1993.*"><meta name="citation_reference" content="Lee et al., &quot;DNA sequencing with dye-labeled terminators and T7 DNA polymerase: effect of dyes and dNTPs on incorporation of dye-terminators and probability analysis of termination fragments,&quot; Nucl. Acids Res. 20(10):2471-2483 (1992)."><meta name="citation_reference" content="Lindsey et al., &quot;Visible light-harvesting in covalently-linked porphyrin-cyanine dyes,&quot; Tetrahedron 45(15) :4845-4866 (1989)."><meta name="citation_reference" content="Livak et al. &quot;Oligonucleotides with fluorescent dyes at opposote ends provide a quenched probe system useful for detecting PCR product and nucleic acid hybridization&quot; PCR Methods and Applications pp. 357-362, 1995.*"><meta name="citation_reference" content="Mergny et al., &quot;Fluorescent Energy Transfer as a Probe for Nucleic Acid Structures and Sequences,&quot; Nucl. Acids Res. 22(6) :920-928 (1994)."><meta name="citation_reference" content="Ozaki et al., &quot;The estimation of distances between specific backbone-labeled sites in DNA using fluorescence resonance energy transfer,&quot; Nucl. Acids Res. 20(19):5205-5214 (1992)."><meta name="citation_reference" content="Parkhurst &amp; Parkhurst, &quot;Changes in the end-to-end distance distribution in an oligonucleotide following hybridization,&quot; Time-Resolved Laser Spectroscopy in Biochemistry (Ladowicz, J.R. Ed.), Proc SPIE 2137:475-485 (1994)."><meta name="citation_reference" content="Parkhurst &amp; Parkhurst, &quot;Donor-Acceptor Distance Distributions in a Double-Labeled Fluorescent Oligonucleotide Both as a Single Strand and in Duplexes,&quot; Biochemistry 34:293-300 (1995)."><meta name="citation_reference" content="Parkhurst &amp; Parkhurst, &quot;Fluorescence Studies of Oligonucleotide-DNA Hybridization in Solution and of Oligonucleotide End to End Distance Distributions,&quot; Abstracts, 11th International Congress on Photobiology, Kyoto, Japan, p. 258, Photobiology Association of Japan, Kyoto, Japan (1992)."><meta name="citation_reference" content="Parkhurst &amp; Parkhurst, &quot;Kinetic Studies of Oligonucleotide-DNA Hybridization in Solution by Fluorescence Resonance Energy Transfer,&quot; Abstr. Biophys. J. 64:A266 (1993)."><meta name="citation_reference" content="Parkhurst et al., &quot;Kinetic Studies by fluorescence resonance energy transfer employing a double-labeled oligonucleotide: hybridization to the oligonucleotide complement and to single-stranded DNA,&quot; Biochemistry 34:285-292 (1995)."><meta name="citation_reference" content="Search Report of WO 96/15270 by Livak, (PCT/US95/14882)."><meta name="citation_reference" content="Stryer et al., &quot;Energy Transfer: A Spectroscopic Ruler,&quot; Proc. Natl. Acad. Sci. USA 58:719-726 (1967)."><meta name="citation_reference" content="Tyagi et al., Molecular Beacons: Probes that fluoresce upon hybridization, Nature Biotechnology 14:303-308 (1996)."><meta name="citation_reference" content="Wu et al., &quot;Resonance Energy Transfer: Methods and Applications,&quot; Anal. Biochem. 218:1-13 (1994)."><meta name="citation_patent_number" content="US:6258569"><meta name="citation_patent_application_number" content="US:09/436,454"><link rel="canonical" href="http://www.google.com/patents/US6258569"/><meta property="og:url" content="http://www.google.com/patents/US6258569"/><meta name="title" content="Patent US6258569 - Amplifying using polymerase with 5&#39; to 3&#39; nuclease activity, a primer, and an oligonucleotide probe with fluorescent reporter and quencher attached at opposite ends; digestion of probe by polymerase increases signal"/><meta name="description" content="Provided is a method of nucleic acid amplification. In one embodiment, the method comprises performing nucleic acid amplification on a target polynucleotide using a nucleic acid polymerase having 5′ to 3′ nuclease activity, a primer capable of hybridizing to the target polynucleotide, and an oligonucleotide probe under amplification conditions such that the probe hybridizes to the target polynucleotide 3′ relative to the primer and the probe does not hybridize with itself to form a hairpin structure. The oligonucleotide probe has at one end a fluorescent reporter and at the other end a quencher that quenches the fluorescence of the reporter molecule when both the fluorescent reporter and quencher are attached to the probe. Digestion of the oligonucleotide probe by the polymerase during amplification is effective to separate the reporter from the quencher, whereby a fluorescence signal of the reporter is increased."/><meta property="og:title" content="Patent US6258569 - Amplifying using polymerase with 5&#39; to 3&#39; nuclease activity, a primer, and an oligonucleotide probe with fluorescent reporter and quencher attached at opposite ends; digestion of probe by polymerase increases signal"/><meta property="og:type" content="book"/><meta property="og:site_name" content="Google Books"/><meta property="og:image" content="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><link rel="image_src" href="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><script>(function(){try{var aa=function(a,b,c,d){d=d||{};d._sn=["cfg",b,c].join(".");window.gbar.logger.ml(a,d)};var g=window.gbar=window.gbar||{},l=window.gbar.i=window.gbar.i||{},m={},n;function _tvn(a,b){var c=parseInt(a,10);return isNaN(c)?b:c}function _tvf(a,b){var c=parseFloat(a);return isNaN(c)?b:c}function _tvv(a){return!!a}function p(a,b,c){(c||g)[a]=b}g.bv={n:_tvn("2",0),r:"",f:".67.",e:"0",m:_tvn("0",1)};
function q(a,b,c){var d="on"+b;if(a.addEventListener)a.addEventListener(b,c,!1);else if(a.attachEvent)a.attachEvent(d,c);else{var f=a[d];a[d]=function(){var a=f.apply(this,arguments),b=c.apply(this,arguments);return void 0==a?b:void 0==b?a:b&&a}}}var s=function(a){return function(){return g.bv.m==a}},ba=s(1),ca=s(2);p("sb",ba);p("kn",ca);l.a=_tvv;l.b=_tvf;l.c=_tvn;l.i=aa;var da=window.gbar.i.i;var t,u,v,w;function ea(a){v=a}function fa(a){var b;if(b=v&&window.encodeURIComponent)b=a.href,b=!b.match(/^http[s]?:\/\/accounts\.google\.[^/]*\/ClearSID/i)&&!b.match(/^http[s]?:\/\/[^/]*\/accounts\/ClearSID/i);if(b=b&&encodeURIComponent(v()))a.href=a.href.replace(/([?&]continue=)[^&]*/,"$1"+b)}function ga(a){window.gApplication&&(a.href=window.gApplication.getTabUrl(a.href))}
function ha(a){var b=document.forms[0].q,c=window.encodeURIComponent&&b&&b.value,b=b&&b.placeholder;c&&c!=b&&(a.href=a.href.replace(/([?&])q=[^&]*|$/,function(a,b){return(b||"&")+"q="+encodeURIComponent(c)}))}n=l.a("")?ga:ha;
function x(a,b,c,d,f,e){var h=document.getElementById(a);if(h){var k=h.style;k.left=d?"auto":b+"px";k.right=d?b+"px":"auto";k.top=c+"px";k.visibility=u?"hidden":"visible";f&&e?(k.width=f+"px",k.height=e+"px"):(x(t,b,c,d,h.offsetWidth,h.offsetHeight),u=u?"":a)}}
var y=[],ia=function(a,b){y.push(b)},ja=function(a){a=a||window.event;var b=a.target||a.srcElement;a.cancelBubble=!0;null==t&&(a=document.createElement(Array.every||window.createPopup?"iframe":"div"),a.frameBorder="0",t=a.id="gbs",a.src="javascript:''",b.parentNode.appendChild(a),q(document,"click",z));var c=b,b=0;"gb3"!=c.className&&(c=c.parentNode);a=c.getAttribute("aria-owns")||"gbi";var d=c.offsetWidth,f=20<c.offsetTop?46:24;document.getElementById("tphdr")&&(f-=3);var e=!1;do b+=c.offsetLeft||
0;while(c=c.offsetParent);var c=(document.documentElement.clientWidth||document.body.clientWidth)-b-d,h,d=document.body,k=document.defaultView;k&&k.getComputedStyle?(d=k.getComputedStyle(d,""))&&(h=d.direction):h=d.currentStyle?d.currentStyle.direction:d.style.direction;h="rtl"==h;if("gbi"==a){for(d=0;k=y[d++];)k();A(null,window.navExtra);h&&(b=c,e=!0)}else h||(b=c,e=!0);u!=a&&z();x(a,b,f,e)},z=function(){u&&x(u,0,0)},A=function(a,b){var c,d=document.getElementById("gbi"),f=a;f||(f=d.firstChild);
for(;b&&(c=b.pop());){var e=d,h=c,k=f;w||(w="gb2");e.insertBefore(h,k).className=w}},ka=function(a,b,c){if((b=document.getElementById(b))&&a){a.className="gb4";var d=document.createElement("span");d.appendChild(a);d.appendChild(document.createTextNode(" | "));d.id=c;b.appendChild(d)}},la=function(){return document.getElementById("gb_70")},ma=function(){return!!u};p("qs",n);p("setContinueCb",ea);p("pc",fa);p("tg",ja);p("close",z);p("addLink",ka);p("almm",A);p("si",la);p("adh",ia);p("op",ma);var B=function(){},C=function(){},F=function(a){var b=new Image,c=D;b.onerror=b.onload=b.onabort=function(){try{delete E[c]}catch(a){}};E[c]=b;b.src=a;D=c+1},E=[],D=0;p("logger",{il:C,ml:B,log:F});var G=window.gbar.logger;var H={},na={},I=[],oa=l.b("0.1",.1),pa=l.a("1",!0),qa=function(a,b){I.push([a,b])},ra=function(a,b){H[a]=b},sa=function(a){return a in H},J={},K=function(a,b){J[a]||(J[a]=[]);J[a].push(b)},ta=function(a){K("m",a)},L=function(a,b){var c=document.createElement("script");c.src=a;c.async=pa;Math.random()<oa&&(c.onerror=function(){c.onerror=null;B(Error("Bundle load failed: name="+(b||"UNK")+" url="+a))});(document.getElementById("xjsc")||document.getElementsByTagName("body")[0]||
document.getElementsByTagName("head")[0]).appendChild(c)},N=function(a){for(var b=0,c;(c=I[b])&&c[0]!=a;++b);!c||c[1].l||c[1].s||(c[1].s=!0,M(2,a),c[1].url&&L(c[1].url,a),c[1].libs&&m.d&&m.d(c[1].libs))},O=function(a){K("gc",a)},P=null,ua=function(a){P=a},M=function(a,b,c){if(P){a={t:a,b:b};if(c)for(var d in c)a[d]=c[d];try{P(a)}catch(f){}}};p("mdc",H);p("mdi",na);p("bnc",I);p("qGC",O);p("qm",ta);p("qd",J);p("lb",N);p("mcf",ra);p("bcf",qa);p("aq",K);p("mdd","");p("has",sa);
p("trh",ua);p("tev",M);var Q=l.b("0.1",.001),R=0;
function _mlToken(a,b){try{if(1>R){R++;var c,d=a,f=b||{},e=encodeURIComponent,h=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&jexpid=",e("17483"),"&srcpg=",e("prop=22"),"&jsr=",Math.round(1/Q),"&ogev=",e("7knsU7fxEo3CggS4s4KYCg"),"&ogf=",g.bv.f,"&ogrp=",e("1"),"&ogv=",e("1407723702.0"),"&oggv="+e("es_plusone_gc_20140723.0_p0"),"&ogd=",e("com"),"&ogc=",e("BRA"),"&ogl=",e("en")];f._sn&&(f._sn="og."+
f._sn);for(var k in f)h.push("&"),h.push(e(k)),h.push("="),h.push(e(f[k]));h.push("&emsg=");h.push(e(d.name+":"+d.message));var r=h.join("");S(r)&&(r=r.substr(0,2E3));c=r;var Aa=window.gbar.logger._aem(a,c);F(Aa)}}catch(Na){}}var S=function(a){return 2E3<=a.length},va=function(a,b){return b};function T(a){B=a;p("_itl",S,G);p("_aem",va,G);p("ml",B,G);a={};H.er=a}l.a("")?T(function(a){throw a;}):l.a("1")&&Math.random()<Q&&T(_mlToken);I.push(["m",{url:"//ssl.gstatic.com/gb/js/scm_7385cc5883250b43a39405734c1bea59.js"}]);g.mcf("c",{});g.sg={c:""};if(l.a("1")){var wa=l.a("");I.push(["gc",{auto:wa,url:"//ssl.gstatic.com/gb/js/abc/gci_91f30755d6a6b787dcc2a4062e6e9824.js",libs:"googleapis.client:plusone:gapi.iframes"}]);var xa={version:"gci_91f30755d6a6b787dcc2a4062e6e9824.js",index:"",lang:"en"};H.gc=xa;var U=function(a){window.googleapis&&window.iframes?a&&a():(a&&O(a),N("gc"))};p("lGC",U);l.a("1")&&p("lPWF",U)};window.__PVT="";if(l.a("1")&&l.a("1")){var V=function(a){U(function(){K("pw",a);N("pw")})};p("lPW",V);I.push(["pw",{url:"//ssl.gstatic.com/gb/js/abc/pwm_45f73e4df07a0e388b0fa1f3d30e7280.js"}]);var W=[],ya=function(a){W[0]=a},za=function(a,b){var c=b||{};c._sn="pw";B(a,c)},Ba={signed:W,elog:za,base:"https://plusone.google.com/u/0",loadTime:(new Date).getTime()};H.pw=Ba;var X=function(a,b){for(var c=b.split("."),d=function(){var b=arguments;a(function(){for(var a=g,d=0,e=c.length-1;d<e;++d)a=a[c[d]];a[c[d]].apply(a,b)})},f=g,e=0,h=c.length-1;e<h;++e)f=
f[c[e]]=f[c[e]]||{};return f[c[e]]=d};X(V,"pw.clk");X(V,"pw.hvr");p("su",ya,g.pw)};function Ca(){function a(){for(var b;(b=e[h++])&&"m"!=b[0]&&!b[1].auto;);b&&(M(2,b[0]),b[1].url&&L(b[1].url,b[0]),b[1].libs&&m.d&&m.d(b[1].libs));h<e.length&&setTimeout(a,0)}function b(){0<f--?setTimeout(b,0):a()}var c=l.a("1"),d=l.a(""),f=3,e=I,h=0,k=window.gbarOnReady;if(k)try{k()}catch(r){da(r,"ml","or")}d?p("ldb",a):c?q(window,"load",b):b()}p("rdl",Ca);var Da={D:1,H:2,da:3,p:4,W:5,M:6,F:7,g:8,ha:9,U:10,L:11,T:12,S:13,N:14,Q:15,P:16,fa:17,w:18,O:19,ga:20,ea:21,u:22,G:23,ja:24,ka:25,ia:26,A:27,j:28,o:29,k:30,ca:31,Z:32,$:33,J:34,K:35,ba:36,aa:37,Y:38,B:39,R:40,v:41,X:42,V:43,h:48,C:49,I:500},Y=[1,2,3,4,5,6,9,10,11,13,14,28,29,30,34,35,37,38,39,40,41,42,43,48,49,500];var Z=l.b("0.001",1E-4),Ea=l.b("1",1),Fa=!1,Ga=!1;if(l.a("1")){var Ha=Math.random();Ha<=Z&&(Fa=!0);Ha<=Ea&&(Ga=!0)}var Ia=Da,$=null;function Ja(){var a=0,b=function(b,d){l.a(d)&&(a|=b)};b(1,"");b(2,"");b(4,"");b(8,"");return a}
function Ka(a,b){var c=Z,d=Fa,f;f=a;if(!$){$={};for(var e=0;e<Y.length;e++){var h=Y[e];$[h]=!0}}if(f=!!$[f])c=Ea,d=Ga;if(d){d=encodeURIComponent;g.rp?(f=g.rp(),f="-1"!=f?f:"1"):f="1";c=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&oge=",a,"&ogex=",d("17483"),"&ogev=",d("7knsU7fxEo3CggS4s4KYCg"),"&ogf=",g.bv.f,"&ogp=",d("22"),"&ogrp=",d(f),"&ogsr=",Math.round(1/c),"&ogv=",d("1407723702.0"),"&oggv="+
d("es_plusone_gc_20140723.0_p0"),"&ogd=",d("com"),"&ogl=",d("en"),"&ogc=",d("BRA"),"&ogus=",Ja()];if(b){"ogw"in b&&(c.push("&ogw="+b.ogw),delete b.ogw);var k;f=b;e=[];for(k in f)0!=e.length&&e.push(","),e.push(La(k)),e.push("."),e.push(La(f[k]));k=e.join("");""!=k&&(c.push("&ogad="),c.push(d(k)))}F(c.join(""))}}function La(a){"number"==typeof a&&(a+="");return"string"==typeof a?a.replace(".","%2E").replace(",","%2C"):a}C=Ka;p("il",C,G);var Ma={};H.il=Ma;setTimeout(function(){C(Ia.g)},0);}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var b=window.gbar.i.i;var c=window.gbar;var f=function(d){try{var a=document.getElementById("gbom");a&&d.appendChild(a.cloneNode(!0))}catch(e){b(e,"omas","aomc")}};c.aomc=f;}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var a=window.gbar;a.mcf("pm",{p:""});}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{window.gbar.rdl();}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
if (window['_OC_timingAction']) {window['_OC_timingAction']('patents_refpage');}</script><style>#gbar,#guser{font-size:13px;padding-top:1px !important;}#gbar{float:left;height:22px}#guser{padding-bottom:7px !important;text-align:right}.gbh,.gbd{border-top:1px solid #c9d7f1;font-size:1px}.gbh{height:0;position:absolute;top:24px;width:100%}#gbs,.gbm{background:#fff;left:0;position:absolute;text-align:left;visibility:hidden;z-index:1000}.gbm{border:1px solid;border-color:#c9d7f1 #36c #36c #a2bae7;z-index:1001}.gb1{margin-right:.5em}.gb1,.gb3{zoom:1}.gb2{display:block;padding:.2em .5em}.gb2,.gb3{text-decoration:none !important;border-bottom:none}a.gb1,a.gb4{text-decoration:underline !important}a.gb1,a.gb2,a.gb3,a.gb4{color:#00c !important}.gbi .gb3,.gbi .gb2,.gbi .gb4{color:#dd8e27 !important}.gbf .gb3,.gbf .gb2,.gbf .gb4{color:#900 !important}a.gb2:hover{background:#36c;color:#fff !important}#gbar .gbz0l{color:#000 !important;cursor:default;font-weight:bold;text-decoration:none !important}
#gbar { padding:.3em .6em !important;}</style></head><body  topmargin="3" marginheight="3"><div id=gbar><nobr><a onclick=gbar.qs(this);gbar.logger.il(1,{t:1}); class=gb1 id=gb_1 href="https://www.google.com/search?sa=N&tab=tw">Search</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:2}); class=gb1 id=gb_2 href="http://www.google.com/search?hl=en&tbm=isch&source=og&sa=N&tab=ti">Images</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:8}); class=gb1 id=gb_8 href="http://maps.google.com/maps?hl=en&sa=N&tab=tl">Maps</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:78}); class=gb1 id=gb_78 href="https://play.google.com/?hl=en&sa=N&tab=t8">Play</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:36}); class=gb1 id=gb_36 href="http://www.youtube.com/results?sa=N&tab=t1">YouTube</a> <a onclick=gbar.logger.il(1,{t:5}); class=gb1 id=gb_5 href="http://news.google.com/nwshp?hl=en&tab=tn">News</a> <a onclick=gbar.logger.il(1,{t:23}); class=gb1 id=gb_23 href="https://mail.google.com/mail/?tab=tm">Gmail</a> <a onclick=gbar.logger.il(1,{t:25}); class=gb1 id=gb_25 href="https://drive.google.com/?tab=to">Drive</a> <a class=gb3 href="http://www.google.com/intl/en/options/" onclick="this.blur();gbar.tg(event);return !1" aria-haspopup=true><u>More</u> <small>&#9660;</small></a><div class=gbm id=gbi><a onclick=gbar.logger.il(1,{t:24}); class=gb2 id=gb_24 href="https://www.google.com/calendar?tab=tc">Calendar</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:51}); class=gb2 id=gb_51 href="http://translate.google.com/?hl=en&sa=N&tab=tT">Translate</a><a onclick=gbar.logger.il(1,{t:17}); class=gb2 id=gb_17 href="http://www.google.com/mobile/?hl=en&tab=tD">Mobile</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:10}); class=gb2 id=gb_10 href="http://www.google.com/search?hl=en&tbo=u&tbm=bks&source=og&sa=N&tab=tp">Books</a><a onclick=gbar.logger.il(1,{t:212}); class=gb2 id=gb_212 href="https://wallet.google.com/manage/?tab=ta">Wallet</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:6}); class=gb2 id=gb_6 href="http://www.google.com/search?hl=en&tbo=u&tbm=shop&source=og&sa=N&tab=tf">Shopping</a><a onclick=gbar.logger.il(1,{t:30}); class=gb2 id=gb_30 href="http://www.blogger.com/?tab=tj">Blogger</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:27}); class=gb2 id=gb_27 href="http://www.google.com/finance?sa=N&tab=te">Finance</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:31}); class=gb2 id=gb_31 href="https://plus.google.com/photos?sa=N&tab=tq">Photos</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:12}); class=gb2 id=gb_12 href="http://www.google.com/search?hl=en&tbo=u&tbm=vid&source=og&sa=N&tab=tv">Videos</a><div class=gb2><div class=gbd></div></div><a onclick=gbar.logger.il(1,{t:66}); href="http://www.google.com/intl/en/options/" class=gb2>Even more &raquo;</a></div></nobr></div><div id=guser width=100%><nobr><span id=gbn class=gbi></span><span id=gbf class=gbf></span><span id=gbe></span><a target=_top id=gb_70 href="https://www.google.com/accounts/Login?service=&continue=http://www.google.com/patents%3Fhl%3Den&hl=en" class=gb4>Sign in</a><div style="display: none"><div class=gbm id=gbd5 aria-owner=gbg5><div class=gbmc><ol id=gbom class=gbmcc></ol></div></div></div></nobr></div><div class=gbh style=left:0></div><div class=gbh style=right:0></div><div role="alert" style="position: absolute; left: 0; right: 0;"><a href="http://www.google.com/patents/us6258569?hl=en&amp;output=html_text" title="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."><img border="0" src="http://www.google.com/images/cleardot.gif"alt="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."></a></div><div id="guser"><nobr></nobr></div><div style="clear:both;"></div><div id="gb-top-search-box" class="gb-top-search-box-small gb-reset"><table><tr><td class="logo"><a href="http://www.google.com/patents" class="logo-link"><img class="logo-img" src="/intl/en/images/logos/google_logo_41.png" alt="Go to Google Books Home" height="41"/></a></td><td><form action="http://www.google.com/search" name="f" id="vheadf" method="get"><span id="hf"></span><input type="hidden" name="tbm" value="pts"/><input type="hidden" name="tbo" value="1"/><input type="hidden" name="hl" value="en"/><table><tr><td><div class="inputs"><table><tr><td><div class="text-input"><input type="text" name="q" id="vheadq" class="text" maxlength="2048" size="31" value="" title="Search Patents" accesskey="s" autocomplete="off"/><script>window._OC_autoDir &&window._OC_autoDir('vheadq', 'tia-vheadq');</script></div></td><td><div class="submit-input"><input name="btnG" class="submit" type="submit" value=""/></div></td></tr></table></div></td><td class="col-ext-links"><div class="ext-links"><a href="http://www.google.com/advanced_patent_search">&lt;nobr&gt;Advanced Patent Search&lt;/nobr&gt;</a></div></td></tr></table></form></td></tr></table></div><div class="kd-appbar"><h2 class="kd-appname"><a href="/patents">Patents</a></h2><div class="kd-buttonbar left" id="left-toolbar-buttons"><a id="appbar-write-review-link" href=""></a><a id="appbar-view-print-sample-link" href=""></a><a id="appbar-view-ebook-sample-link" href=""></a><a id="appbar-patents-prior-art-finder-link" href="https://www.google.com/patents/related/US6258569"></a><a id="appbar-patents-discuss-this-link" href="http://www.google.com/url?id=sLFVBAABERAJ&amp;q=http://patents.stackexchange.com/redirect/google-patents%3Fpatent%3DUS6258569&amp;usg=AFQjCNGZyJKnQEoO094gYdwH0fuU94L07g" data-is-grant="true"></a><a id="appbar-read-patent-link" href="//docs.google.com/viewer?url=patentimages.storage.googleapis.com/pdfs/US6258569.pdf"></a><a id="appbar-download-pdf-link" href="//patentimages.storage.googleapis.com/pdfs/US6258569.pdf"></a></div><div class="kd-buttonbar right" id="right-toolbar-buttons"></div></div><div id="books-microdata" itemscope=""itemtype="http://schema.org/Book"itemid="http://www.google.com/patents/US6258569" style="display:none"><span itemprop="description">Provided is a method of nucleic acid amplification. In one embodiment, the method comprises performing nucleic acid amplification on a target polynucleotide using a nucleic acid polymerase having 5′ to 3′ nuclease activity, a primer capable of hybridizing to the target polynucleotide, and an oligonucleotide...</span><span itemprop="url">http://www.google.com/patents/US6258569?utm_source=gb-gplus-share</span><span class="main-title" itemprop="name">Patent US6258569 - Amplifying using polymerase with 5&#39; to 3&#39; nuclease activity, a primer, and an oligonucleotide probe with fluorescent reporter and quencher attached at opposite ends; digestion of probe by polymerase increases signal</span><img itemprop="image" src="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"alt="Patent US6258569 - Amplifying using polymerase with 5&#39; to 3&#39; nuclease activity, a primer, and an oligonucleotide probe with fluorescent reporter and quencher attached at opposite ends; digestion of probe by polymerase increases signal" title="Patent US6258569 - Amplifying using polymerase with 5&#39; to 3&#39; nuclease activity, a primer, and an oligonucleotide probe with fluorescent reporter and quencher attached at opposite ends; digestion of probe by polymerase increases signal"></div><div style="display: none"><ol id="ofe-gear-menu-contents" class="gbmcc"><li class="gbe gbmtc"><a class="gbmt goog-menuitem-content" id="" href="http://www.google.com/advanced_patent_search">Advanced Patent Search</a></li></ol></div><table id="viewport_table" cellpadding="0" style="clear:both" cellspacing="0"><tr><td id="viewport_td"><div class=vertical_module_list_row><div id=intl_patents class=about_content><div id=intl_patents_v><table class="patent-bibdata"><tr><td class="patent-bibdata-heading">Publication number</td><td class="single-patent-bibdata">US6258569 B1</td></tr><tr><td class="patent-bibdata-heading">Publication type</td><td class="single-patent-bibdata">Grant</td></tr><tr><td class="patent-bibdata-heading">Application number</td><td class="single-patent-bibdata">US 09/436,454</td></tr><tr><td class="patent-bibdata-heading">Publication date</td><td class="single-patent-bibdata">Jul 10, 2001</td></tr><tr><td class="patent-bibdata-heading">Filing date</td><td class="single-patent-bibdata">Nov 8, 1999</td></tr><tr><td class="patent-bibdata-heading">Priority date<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The priority date is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the date listed."></span></td><td class="single-patent-bibdata">Nov 16, 1994</td></tr><tr><td class="patent-bibdata-heading">Fee status<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The fee status is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the status or dates listed."></span></td><td class="single-patent-bibdata">Paid</td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Also published as</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/CA2201756A1">CA2201756A1</a>, </span><span class="patent-bibdata-value"><a href="/patents/CA2201756C">CA2201756C</a>, </span><span class="patent-bibdata-value"><a href="/patents/DE69519940D1">DE69519940D1</a>, </span><span class="patent-bibdata-value"><a href="/patents/DE69519940T2">DE69519940T2</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP0792374A1">EP0792374A1</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP0792374B1">EP0792374B1</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP0972848A2">EP0972848A2</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP0972848A3">EP0972848A3</a>, </span><span class="patent-bibdata-value"><a href="/patents/US5538848">US5538848</a>, </span><span class="patent-bibdata-value"><a href="/patents/US5723591">US5723591</a>, </span><span class="patent-bibdata-value"><a href="/patents/US5876930">US5876930</a>, </span><span class="patent-bibdata-value"><a href="/patents/US6030787">US6030787</a>, </span><span class="patent-bibdata-value"><a href="/patents/WO1996015270A1">WO1996015270A1</a></span></span></td></tr><tr class="patent-bibdata-list-row alternate-patent-number"><td class="patent-bibdata-heading">Publication number</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value">09436454, </span><span class="patent-bibdata-value">436454, </span><span class="patent-bibdata-value">US 6258569 B1, </span><span class="patent-bibdata-value">US 6258569B1, </span><span class="patent-bibdata-value">US-B1-6258569, </span><span class="patent-bibdata-value">US6258569 B1, </span><span class="patent-bibdata-value">US6258569B1</span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Inventors</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Kenneth+J.+Livak%22">Kenneth J. Livak</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Susan+J.+A.+Flood%22">Susan J. A. Flood</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Jeffrey+Mamoro%22">Jeffrey Mamoro</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Khairuzzaman+Bashar+Mullah%22">Khairuzzaman Bashar Mullah</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Original Assignee</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=inassignee:%22The+Perkin-Elmer+Corporation%22">The Perkin-Elmer Corporation</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Export Citation</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/US6258569.bibtex">BiBTeX</a>, </span><span class="patent-bibdata-value"><a href="/patents/US6258569.enw">EndNote</a>, </span><span class="patent-bibdata-value"><a href="/patents/US6258569.ris">RefMan</a></span></span></td></tr><tr class="patent-internal-links"><td colspan=2><span class="patent-bibdata-value"><a href="#backward-citations">Patent Citations</a> (32),</span> <span class="patent-bibdata-value"><a href="#npl-citations">Non-Patent Citations</a> (27),</span> <span class="patent-bibdata-value"><a href="#forward-citations">Referenced by</a> (44),</span> <span class="patent-bibdata-value"><a href="#classifications">Classifications</a> (18),</span> <span class="patent-bibdata-value"><a href="#legal-events">Legal Events</a> (13)</span> </td></tr><tr><td colspan=2 class="patent-bibdata-external-link-spacer-top"></td></tr><tr class="patent-bibdata-external-link-spacer-bottom"></tr><tr><td colspan=2><span class="patent-bibdata-heading">External Links:&nbsp;</span><span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=sLFVBAABERAJ&q=http://patft.uspto.gov/netacgi/nph-Parser%3FSect2%3DPTO1%26Sect2%3DHITOFF%26p%3D1%26u%3D/netahtml/PTO/search-bool.html%26r%3D1%26f%3DG%26l%3D50%26d%3DPALL%26RefSrch%3Dyes%26Query%3DPN/6258569&usg=AFQjCNEMNH4zNEnytmbKSAEczBZUBvIFPQ">USPTO</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=sLFVBAABERAJ&q=http://assignments.uspto.gov/assignments/q%3Fdb%3Dpat%26pat%3D6258569&usg=AFQjCNHtY0LRe4q98ElUJaD5fw1udWuFPg">USPTO Assignment</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=sLFVBAABERAJ&q=http://worldwide.espacenet.com/publicationDetails/biblio%3FCC%3DUS%26NR%3D6258569B1%26KC%3DB1%26FT%3DD&usg=AFQjCNGvyv0udQL0m4usSZ8I2s8bVTgQQg">Espacenet</a></span></span></td></tr><tr class="patent-bibdata-group-spacer"></tr></table><div class="number-and-title"><span class="patent-title"><invention-title mxw-id="PT129042566" lang="EN" load-source="ifi">Amplifying using polymerase with 5' to 3' nuclease activity, a primer, and an oligonucleotide probe with fluorescent reporter and quencher attached at opposite ends; digestion of probe by polymerase increases signal</invention-title></span><br><span class="patent-number">US 6258569 B1</span></div><div class="patent-section patent-abstract-section"><div class="patent-section-header"><span class="patent-section-title">Abstract</span></div><div class="patent-text"><abstract mxw-id="PA72583495" lang="EN" load-source="patent-office"> <div class="abstract">Provided is a method of nucleic acid amplification. In one embodiment, the method comprises performing nucleic acid amplification on a target polynucleotide using a nucleic acid polymerase having 5′ to 3′ nuclease activity, a primer capable of hybridizing to the target polynucleotide, and an oligonucleotide probe under amplification conditions such that the probe hybridizes to the target polynucleotide 3′ relative to the primer and the probe does not hybridize with itself to form a hairpin structure. The oligonucleotide probe has at one end a fluorescent reporter and at the other end a quencher that quenches the fluorescence of the reporter molecule when both the fluorescent reporter and quencher are attached to the probe. Digestion of the oligonucleotide probe by the polymerase during amplification is effective to separate the reporter from the quencher, whereby a fluorescence signal of the reporter is increased.</div>
  </abstract></div></div><div class="patent-section patent-drawings-section"><div class="patent-section-header"><span class="patent-section-title">Images<span class="patent-section-count">(3)</span></span></div><div class="patent-drawings-body"><div class="patent-drawings-carousel"><div class="drawings"><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6258569B1/US06258569-20010710-D00000.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6258569B1/US06258569-20010710-D00000.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6258569B1/US06258569-20010710-D00001.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6258569B1/US06258569-20010710-D00001.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6258569B1/US06258569-20010710-D00002.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6258569B1/US06258569-20010710-D00002.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div></div></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img" alt="Previous page"src="/googlebooks/images/kennedy/page_left.png"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img" alt="Next page"src="/googlebooks/images/kennedy/page_right.png"width="21" height="21" /></div></div></div><div class="patent-post-drawings"></div><div class="patent-section patent-claims-section"><div class="patent-section-header"><span class="patent-section-title">Claims<span class="patent-section-count">(36)</span></span></div><div class="patent-text"><div mxw-id="PCLM28668302" lang="EN" load-source="patent-office" class="claims">
    <claim-statement>What is claimed is: </claim-statement> <div class="claim"> <div num="1" id="US-6258569-B1-CLM-00001" class="claim">
      <div class="claim-text">1. A method of nucleic acid amplification comprising:</div>
      <div class="claim-text">performing nucleic acid amplification on a target polynucleotide using a nucleic acid polymerase having 5′ to 3′ nuclease activity, a primer capable of hybridizing to the target polynucleotide, and an oligonucleotide probe under amplification conditions such that the probe hybridizes to the target polynucleotide 3′ relative to the primer and the probe does not hybridize with itself to form a hairpin structure, </div>
      <div class="claim-text">the oligonucleotide probe having at one end a fluorescent reporter and at the other end a quencher that quenches the fluorescence of the reporter molecule when both the fluorescent reporter and quencher are attached to the probe, </div>
      <div class="claim-text">under conditions such that digestion of the oligonucleotide probe by the polymerase during amplification is effective to separate the reporter from the quencher, whereby a fluorescence signal of the reporter is increased. </div>
    </div>
    </div> <div class="claim-dependent"> <div num="2" id="US-6258569-B1-CLM-00002" class="claim">
      <div class="claim-text">2. The method of claim <b>1</b>, wherein the polymerase is a thermostable nucleic acid polymerase.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="3" id="US-6258569-B1-CLM-00003" class="claim">
      <div class="claim-text">3. The method of claim <b>1</b>, wherein the reporter is a fluorescein dye and the quencher is a rhodamine dye.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="4" id="US-6258569-B1-CLM-00004" class="claim">
      <div class="claim-text">4. The method of claim <b>3</b>, wherein the fluorescent reporter is attached to the 5′ end of the probe.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="5" id="US-6258569-B1-CLM-00005" class="claim">
      <div class="claim-text">5. The method of claim <b>4</b>, wherein the probe is 15 to 60 nucleotides in length.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="6" id="US-6258569-B1-CLM-00006" class="claim">
      <div class="claim-text">6. The method of claim <b>4</b>, wherein the probe is 18 to 30 nucleotides in length.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="7" id="US-6258569-B1-CLM-00007" class="claim">
      <div class="claim-text">7. The method of claim <b>3</b>, wherein the fluorescent reporter is attached to the 3′ end of the probe.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="8" id="US-6258569-B1-CLM-00008" class="claim">
      <div class="claim-text">8. The method of claim <b>7</b>, wherein the probe is 15 to 60 nucleotides in length.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="9" id="US-6258569-B1-CLM-00009" class="claim">
      <div class="claim-text">9. The method of claim <b>7</b>, wherein the probe is 18 to 30 nucleotides in length.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="10" id="US-6258569-B1-CLM-00010" class="claim">
      <div class="claim-text">10. The method of claim <b>1</b>, wherein the fluorescent reporter is attached to the 5′ end of the probe.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="11" id="US-6258569-B1-CLM-00011" class="claim">
      <div class="claim-text">11. The method of claim <b>10</b>, wherein the probe is 15 to 60 nucleotides in length.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="12" id="US-6258569-B1-CLM-00012" class="claim">
      <div class="claim-text">12. The method of claim <b>10</b>, wherein the probe is 18 to 30 nucleotides in length.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="13" id="US-6258569-B1-CLM-00013" class="claim">
      <div class="claim-text">13. The method of claim <b>1</b>, wherein the fluorescent reporter is attached to the 3′ end of the probe.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="14" id="US-6258569-B1-CLM-00014" class="claim">
      <div class="claim-text">14. The method of claim <b>13</b>, wherein the probe is 15 to 60 nucleotides in length.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="15" id="US-6258569-B1-CLM-00015" class="claim">
      <div class="claim-text">15. The method of claim <b>13</b>, wherein the probe is 18 to 30 nucleotides in length.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="16" id="US-6258569-B1-CLM-00016" class="claim">
      <div class="claim-text">16. The metod of claim <b>1</b>, wherein the repoter or the quencher comprises a rhodamine dye.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="17" id="US-6258569-B1-CLM-00017" class="claim">
      <div class="claim-text">17. The method of claim <b>1</b>, wherein the reporter or the quencher comprises a fluorescein dye.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="18" id="US-6258569-B1-CLM-00018" class="claim">
      <div class="claim-text">18. The method of claim <b>1</b>, wberein the reporter or quencher is selected from the group consisting of a xanthene dye and a naphthylamine dye.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="19" id="US-6258569-B1-CLM-00019" class="claim">
      <div class="claim-text">19. The method of claim <b>1</b>, wherein the oligonucleotide probe is not capable of extension by the nucleic acid polymerase.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="20" id="US-6258569-B1-CLM-00020" class="claim">
      <div class="claim-text">20. The method of claim <b>19</b>, wherein the polymerase is a thermostable nucleic acid polymerase.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="21" id="US-6258569-B1-CLM-00021" class="claim">
      <div class="claim-text">21. The method of claim <b>19</b>, wherein the reporter is a fluorescein dye and the quencher is a rhodamine dye.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="22" id="US-6258569-B1-CLM-00022" class="claim">
      <div class="claim-text">22. The method of claim <b>21</b>, wherein the fluorescent reporter is attached to the 5′ end of the probe.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="23" id="US-6258569-B1-CLM-00023" class="claim">
      <div class="claim-text">23. The method of claim <b>22</b>, wherein the probe is 15 to 60 nucleotides in length.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="24" id="US-6258569-B1-CLM-00024" class="claim">
      <div class="claim-text">24. The method of claim <b>22</b>, wherein the probe is 18 to 30 nucleotides in length.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="25" id="US-6258569-B1-CLM-00025" class="claim">
      <div class="claim-text">25. The method of claim <b>21</b>, wherein the fluorescent reporter is attached to the 3′ end of the probe.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="26" id="US-6258569-B1-CLM-00026" class="claim">
      <div class="claim-text">26. The method of claim <b>25</b>, wherein the probe is 15 to 60 nucleotides in length.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="27" id="US-6258569-B1-CLM-00027" class="claim">
      <div class="claim-text">27. The method of claim <b>25</b>, wherein the probe is 18 to 30 nucleotides in length.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="28" id="US-6258569-B1-CLM-00028" class="claim">
      <div class="claim-text">28. The method of claim <b>19</b>, wherein the fluorescent reporter is attached to the 5′ end of the probe.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="29" id="US-6258569-B1-CLM-00029" class="claim">
      <div class="claim-text">29. The method of claim <b>28</b>, wherein the probe is 15 to 60 nucleotides in length.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="30" id="US-6258569-B1-CLM-00030" class="claim">
      <div class="claim-text">30. The method of claim <b>28</b>, wherein the probe is 18 to 30 nucleotides in length.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="31" id="US-6258569-B1-CLM-00031" class="claim">
      <div class="claim-text">31. The method of claim <b>19</b>, wherein the fluorescent reporter is attached to the 3′ end of tbe probe.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="32" id="US-6258569-B1-CLM-00032" class="claim">
      <div class="claim-text">32. The method of claim <b>31</b>, wherein the probe is 15 to 60 nucleotides in length.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="33" id="US-6258569-B1-CLM-00033" class="claim">
      <div class="claim-text">33. The method of claim <b>31</b>, wherein the probe is 18 to 30 nucleotides in length.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="34" id="US-6258569-B1-CLM-00034" class="claim">
      <div class="claim-text">34. The method of claim <b>19</b>, wherein the fluorescent reporter or the quencher comprises a rhodamine dye.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="35" id="US-6258569-B1-CLM-00035" class="claim">
      <div class="claim-text">35. The method of claim <b>19</b>, wherein the fluorescent reporter or the quencher comprises a fluorescein dye.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="36" id="US-6258569-B1-CLM-00036" class="claim">
      <div class="claim-text">36. The method of claim <b>19</b>, wherei dte reporter or quencher is selected from the group consisting of a xanthene dye and a naphthylamine dye.</div>
    </div>
  </div> </div></div></div><div class="patent-section patent-description-section"><div class="patent-section-header"><span class="patent-section-title">Description</span></div><div class="patent-text"><div mxw-id="PDES54599903" lang="EN" load-source="patent-office" class="description">
    <p>This application is a continuation of Ser. No. 09/207,170 filed Dec. 7, 1998, now U.S. Pat. No. 6,030,787, which is a continuation of Ser. No. 08/558,303 filed Nov. 15, 1995, now U.S. Pat. No. 5,876,930, which is a continuation-in-part of Ser. No. 08/340,558 filed Nov. 16, 1994, now U.S. Pat. No. 5,538,848, each of which is incorporated herein by reference in its entirety.</p>
    <heading>BACKGROUND OF THE INVENTION</heading> <p>1. Field of the Invention</p>
    <p>The invention relates generally to fluorescent probes which include a fluorescent reporter molecule and a fluorescent quencher molecule. More specifically, the invention relates to fluorescent probes which include a fluorescent reporter molecule and a fluorescent quencher molecule which may be used in hybridization assays and in nucleic acid amplification reactions, especially polymerase chain reactions (PCR).</p>
    <p>2. Description of Related Art</p>
    <p>Fluorescent reporter molecule-quencher molecule pairs have been incorporated onto oligonucleotide probes in order to monitor biological events based on the fluorescent reporter molecule and quencher molecule being separated or brought within a minimum quenching distance of each other. For example, probes have been developed where the intensity of the reporter molecule fluorescence increases due to the separation of the reporter molecule from the quencher molecule. Probes have also been developed which lose their fluorescence because the quencher molecule is brought into proximity with the reporter molecule. These reporter-quencher molecule pair probes have been used to monitor hybridization assays and nucleic acid amplification reactions. especially polymerase chain reactions (PCR), by monitoring either the appearance or disappearance of the fluorescence signal generated by the reporter molecule.</p>
    <p>As used herein. a reporter molecule is a molecule capable of generating a fluorescence signal. A quencher molecule is a molecule capable of absorbing the fluorescence energy of an excited reporter molecule, thereby quenching the fluorescence signal that would otherwise be released from the excited reporter molecule. In order for a quencher molecule to quench an excited fluorophore, the quencher molecule must be within a minimum quenching distance of the excited reporter molecule at some time prior to the reporter molecule releasing the stored fluorescence energy.</p>
    <p>Probes containing a reporter molecule-quencher molecule pair have been developed for hybridization assays where the probe forms a hairpin structure, i.e., where the probe hybridizes to itself to form a loop such that the quencher molecule is brought into proximity with the reporter molecule in the absence of a complementary nucleic acid sequence to prevent the formation of the hairpin structure. WO 90/03446; European Patent Application No. 0 601 889 A2. When a complementary target sequence is present, hybridization of the probe to the complementary target sequence disrupts the hairpin structure and causes the probe to adopt a conformation where the quencher molecule is no longer close enough to the reporter molecule to quench the reporter molecule. As a result. the probes provide an increased fluorescent signal when hybridized to a target sequence than when unhybridized. Probes including a hairpin structure have the disadvantage that they can be difficult to design and may interfere with the hybridization of the probe to the target sequence.</p>
    <p>Assays have also been developed for identifying the presence of a hairpin structure using two separate probes. one containing a reporter molecule and the other a quencher molecule. Mergney, et al., Nucleic Acids Research, 22:6 920-928 (1994). In these assays, the fluorescence signal of the reporter molecule decreases when hybridized to the target sequence due to the quencher molecule being brought into proximity with the reporter molecule.</p>
    <p>One particularly important application for probes including a reporter-quencher molecule pair is their use in nucleic acid amplification reactions, such as polymerase chain reactions (PCR), to detect the presence and amplification of a target nucleic acid sequence. In general, nucleic acid amplification techniques have opened broad new approaches to genetic testing and DNA analysis. Arnheim and Erlich, Ann. Rev. Biochem., 61: 131-156 (1992). PCR, in particular, has become a research tool of major importance with applications in, for example, cloning, analysis of genetic expression, DNA sequencing, genetic mapping and drug discovery. Amheim and Erlich, Ann. Rev. Biochem., 61: 131-156 (1992); Gilliland et al., Proc. Nati. Acad. Sci., 87: 2725-2729 (1990); Bevan et al., PCR Methods and Applications. 1: 222-228 (1992); Green et al., PCR Methods and Applications, 1: 77-90 (1991); Blackwell et al., Science. 250: 1104-1110 (1990).</p>
    <p>The widespread applications of nucleic acid amplification techniques has driven the development of instrumentation for carrying out the amplification reactions under a variety of circumstances. Important design goals for such instrument development have included fine temperature control. minimization of sample-to-sample variability in multi-sample thermal cycling, automation of pre- and post-reaction processing steps, high speed temperature cycling, minimization of sample volumes, real time measurement of amplification products and minimization of cross contamination, for example, due to “sample carryover”. In particular. the design of instruments permitting amplification to be carried out in closed reaction chambers and monitored in real time would be highly desirable for preventing cross-contamination. Higuchi et al., Biotechnology, 10: 413-417 (1992) and 11: 1026-1030 (1993); and Holland et al., Proc. Natl. Acad. Sci., 88: 7276-7280 (1991). Clearly, the successful realization of such a design goal would be especially desirable in the analysis of diagnostic samples, where a high frequency of false positives and false negatives, for example caused by “sample carryover”, would severely reduce the value of an amplification procedure. Moreover, real time monitoring of an amplification reaction permits far more accurate quantification of starting target DNA concentrations in multiple-target amplifications, as the relative values of close concentrations can be resolved by taking into account the history of the relative concentration values during the reaction. Real time monitoring also permits the efficiency of the amplification reaction to be evaluated, which can indicate whether reaction inhibitors are present in a sample.</p>
    <p>Holland et al. (cited above), U.S. Pat. No. 5,210,015 to Gelfand, et al. and others have proposed fluorescence-based approaches to provide real time measurements of amplification products during PCR. Such approaches have either employed intercalating dyes (such as ethidium bromide) to indicate the amount of double-stranded DNA present, or they have employed probes containing fluorescence-quencher pairs (also referred to as the “Taq-Man” approach) where the probe is cleaved during amplification to release a fluorescent molecule whose concentration is proportional to the amount of double-stranded DNA present. During amplification, the probe is digested by the nuclease activity of a polymerase when hybridized to the target sequence to cause the fluorescent molecule to be separated from the quencher molecule, thereby causing fluorescence from the reporter molecule to appear.</p>
    <p>The Taq-Man approach, illustrated in FIG. 1, uses an oligonucleotide probe containing a reporter molecule-quencher molecule pair that specifically anneals to a region of a target polynucleotide “downstream”, i.e. in the direction of extension of primer binding sites. The reporter molecule and quencher molecule are positioned on the probe sufficiently close to each other such that whenever the reporter molecule is excited, the energy of the excited state nonradiatively transfers to the quencher molecule where it either dissipates nonradiatively or is emitted at a different emission frequency than that of the reporter molecule. During strand extension by a DNA polymerase, the probe anneals to the template where it is digested by the 5′→3′ exonuclease activity of the polymerase. As a result of the probe being digested, the reporter molecule is effectively separated from the quencher molecule such that the quencher molecule is no longer close enough to the reporter molecule to quench the reporter molecule's fluorescence. Thus, as more and more probes are digested during amplification, the number of reporter molecules in solution increases, thus resulting in an increasing number of unquenched reporter molecules which produce a stronger and stronger fluorescent signal.</p>
    <p>Three main factors influence the utility of reporter-quencher molecule pair probes in hybridization and amplification assays. The first factor is the effectiveness of the quencher molecule on the probe to quench the reporter molecule. This first factor. herein designated “RQ”, can be characterized by the ratio of the fluorescent emissions of the reporter molecule to the quencher molecule when the probe is not hybridized to a complementary polynucleotide. That is RQ is the ratio of the fluorescent emissions of the reporter molecule to the fluorescence of the quencher molecule when the oligonucleotide probe is in a single-stranded state. Influences on the value of RQ include, for example, the particular reporter and quencher molecules used, the spacing between the reporter and quencher molecules, nucleotide sequence-specific effects, and the degree of flexibility of structures, e.g., linkers, to which the reporter and quencher molecules are attached, and the presence of impurities. Wo et al., Anal. Biochem., 218:1-13 (1994); and Clegg, Meth. Enzymol., 211: 353-388 (1992). A related quantity RQ<sup>+</sup>, refers to the ratio of fluorescent emissions of the reporter molecule to the quencher molecule when the oligonucleotide probe is hybridized to a complementary polynucleotide.</p>
    <p>A second factor is the efficiency of the probe to hybridize to a complementary polynucleotide. This second factor depends on the probe's melting temperature, T<sub>m</sub>, the presence of a secondary structure in the probe or target polynucleotide, the annealing temperature, and other reaction conditions.</p>
    <p>A third factor is the efficiency with which the DNA polymerase 5′→3′ exonuclease activity cleaves the bound probe between the reporter molecule and quencher molecule. This efficiency depends on such factors as the proximity of the reporter or quencher to the 5′ end of the probe. the “bulkiness” of the reporter or quencher. and the degree of complementarity between the probe and target polynucleotide. Lee et al., Nucleic Acids Research. 21: 3761≧3766 (1993).</p>
    <p>Since quenching depends on the physical proximity of the reporter molecule to the quencher molecule. it was previously assumed that the quencher and reporter molecules must be attached to the probe such that the quencher molecule remains at all times within the maximum distance at which the quencher molecule can quench the reporter molecule. this distance generally being a separation of about 6-16 nucleotides. Lee et al. Nucleic Acids Research. 21: 3761-3766 (1993); Mergny et al., Nucleic Acids Research 22: 920-928 (1994); Cardullo et al., Proc. Natl. Acad. Sci., 85: 8790-8794 (1988); Clegg et al., Proc. Natl. Acad. Sci., 90: 2994-2998 (1993); and Ozaki et al., Nucleic Acids Research, 20: 5205-5214 (1992). This short separation between the reporter molecule and the quencher molecule is typically achieved by attaching one member of the reporter-quencher pair to the 3′ or 5′ end of the probe and the other member to an internal base <b>6-16 nucleotides away. </b> </p>
    <p>There are at least two significant disadvantages associated with attaching a reporter or quencher molecule to an internal base. Attaching a reporter or quencher molecule to an internal nucleotide typically involves more difficult chemistry than the chemistry required to attach the molecule to a terminal nucleotide. In addition, attachment of a reporter or quencher molecule to an internal nucleotide can adversely affect the hybridization efficiency of the probe. Ward et al., U.S. Pat. No. 5,328,824; and Ozaki et al. Nucleic Acids Research, 20: 5205-5214 (1992).</p>
    <p>A need currently exists for effective oligonucleotide probes containing a fluorescent reporter molecule and a quencher molecule for use in hybridization and nucleic acid amplification assays. Accordingly, a need exists for probes which exhibit distinguishable fluorescence characteristics when hybridized and not hybridized to a target nucleic acid sequence. A further need exists for probes where the reporter molecule and quencher molecule are positioned on the probe such that the quencher molecule can effectively quench the fluorescence of the reporter molecule. A further need exists for probes which are efficiently synthesized. Yet a further need exists for the reporter molecule and quencher molecule to be positionable on the probe such that the reporter and quencher molecules do not adversely impact the hybridization efficiency of probe. These and further objectives are provided by the probes and methods of the present invention.</p>
    <heading>SUMMARY OF THE INVENTION</heading> <p>The present invention relates to an oligonucieotide probe which includes a fluorescent reporter molecule and a quencher molecule capable of quenching the fluorescence of the reporter molecule. According to the present invention, the oligonucleotide probe is constructed such that the probe exists in at least one single-stranded conformation when unhybridized where the quencher molecule is near enough to the reporter molecule to quench the fluorescence of the reporter molecule. The oligonucleotide probe also exists in at least one conformation when hybridized to a target polynucleotide where the quencher molecule is not positioned close enough to the reporter molecule to quench the fluorescence of the reporter molecule. By adopting these hybridized and unhybridized conformations. the reporter molecule and quencher molecule on the probe exhibit different fluorescence signal intensities when the probe is hybridized and unhybridized. As a result, it is possible to determine whether the probe is hybridized or unhybridized based on a change in the fluorescence intensity of the reporter molecule, the quencher molecule, or a combination thereof In addition. because the probe can be designed such that the quencher molecule quenches the reporter molecule when the probe is not hybridized. the probe can be designed such that the reporter molecule exhibits limited fluorescence until the probe is either hybridized or digested.</p>
    <p>According to the present invention. the fluorescence intensity of the reporter molecule is preferably greater than the fluorescence intensity of the quencher molecule when the probe is hybridized to the target polynucleotide. The fluorescence intensity of the reporter molecule is more preferably at least about a factor of 3.5 greater than the fluorescence intensity of the quencher molecule when the probe is hybridized to the target polynucleotide.</p>
    <p>The fluorescence intensity of the oligonucleotide probe hybridized to the target polynucieotide is also preferably at least about a factor of 6 greater than the fluorescence intensity of the oligonucleotide probe when not hybridized to the target polynucieotide.</p>
    <p>The reporter molecule is preferably separated from the quencher molecule by at least about 15 nucleotides, more preferably at least about 18 nucleotides. The reporter molecule is preferably separated from the quencher molecule by between about 15 and 60 nucleotides, more preferably between about 18 and 30 nucleotides.</p>
    <p>The reporter molecule is preferably attached to either the 3′ or 5′ terminal nucleotides of the probe. The quencher molecule is also preferably attached to either the 3′ or 5′ terminal nucleotides of the probe. More preferably, the reporter and quencher molecules are attached to the 3′ and 5′ or 5′ and 3′ terminal nucieotides of the probe respectively.</p>
    <p>The reporter molecule is preferably a fluorescein dye and the quencher molecule is preferably a rhodamine dye.</p>
    <p>In one embodiment, the oligonucleotide probe of the present invention is immobilized on a solid support. The oligonucleotide probe may be attached directly to the solid support. for example by attachment of the 3′ or 5′ terminal nucleotide of the probe to the solid support. More preferably, however. the probe is attached to the solid support by a linker. The linker serves to distance the probe from the solid support. The linker is most preferably at least 30 atoms in length, more preferably at least 50 atoms in length.</p>
    <p>A wide variety of linkers are known in the art which may be used to attach the oligonudeotide probe to the solid support. The linker most preferably includes a functionalized polyethylene glycol because it does not significantly interfere with the hybridization of probe to the target oligonucleotide, is commercially available, soluble in both organic and aqueous media, easy to functionalize, and completely stable under oligonucleotide synthesis and post-synthesis conditions.</p>
    <p>The linkages between the solid support, the linker and the probe are preferably not cleaved during removal of base protecting groups under basic conditions at high temperature. Examples of preferred linkages include carbamate and amide linkages.</p>
    <p>The present invention also relates to the use of the oligonucleotide probe as a hybridization probe to detect target polynucleotides. Accordingly, the present invention relates to a hybridization assay for detecting the presence of a target polynucleotide in a sample. In one embodiment of the method, the hybridization probe is immobilized on a solid support.</p>
    <p>According to the method, an oligonucleotide probe of the present invention is contacted with a sample of polynucleotides under conditions favorable for hybridization. The fluorescence signal of the reporter molecule before and after being contacted with the sample is compared. Since the reporter molecule on the probe exhibits a greater fluorescence signal when hybridized to a target sequence, an increase in the fluorescence signal after the probe is contacted with the sample indicates the hybridization of the probe to target sequences in the sample. thereby indicating the pressure of target sequences in the sample. Quantification of the change in fluorescence intensity as a result of the probe being contacted with the sample can be used to quantify the amount of target sequences present in the sample.</p>
    <p>The present invention also relates to the use of the oligonucteotide probe for monitoring nucleic acid amplification. Accordingly, the present invention relates to a method for monitoring nucleic acid amplification by performing nucieic acid amplification on a target sequence using a nucleic acid polymerase having 5′→3′ nuclease activity, a primer capable of hybridizing to the target sequence and an oligonucleotide probe according to the present invention which is capable of hybridizing to the target sequence 3′ relative to the primer. According to the method, the nucleic acid polymerase digests the oligonucleotide probe during amplification when it is hybridized to the target sequence, thereby separating the reporter molecule from the quencher molecule. As the amplification is conducted, the fluorescence of the reporter molecule is monitored, the generation of fluorescence corresponding to the occurrence of nucleic acid amplification. Accordingly, the amount of amplification performed can be quantified based on the change in fluorescence observed. It is noted that the fluorescence of the quencher molecule may also be monitored, either separately or in combination with the reporter molecule, to detect amplification.</p>
    <p>BRIEF DESCRIPTlON OF THE FIGURES</p>
    <p>FIG. 1 illustrates a method for real-time monitoring nucleic acid amplification utilizing a probe which is degraded by the 5′→3′ exonuclease activity of a nucleic acid polymerase.</p>
    <p>FIG. 2 illustrates a probe according to the present invention immobilized to a solid support in hybridized and unhybridized conformations.</p>
    <heading>DETAILED DESCRIPTION</heading> <p>The present invention relates to an oligonucleotide probe which includes a fluorescent reporter molecule and a quencher molecule capable of quenching the fluorescence of the reporter molecule. According to the present invention, the oligonucleotide probe is constructed such that the probe exists in at least one single-stranded conformation when unhybridized where the quencher molecule is near enough to the reporter molecule to quench the fluorescence of the reporter molecule. The oligonucleotide probe also exists in at least one conformation when hybridized to a target polynucleotide such that the quencher molecule is not positioned close enough to the reporter molecule to quench the fluorescence of the reporter molecule. By adopting these hybridized and unhybridized conformations, the reporter molecule and quencher molecule on the probe exhibit different fluorescence signal intensities when the probe is hybridized and unhybridized. As a result, it is possible to determine whether the probe is hybridized or unhybridized based on a change in the fluorescence intensity of the reporter molecule, the quencher molecule, or a combination thereof. In addition, because the probe can be designed such that the quencher molecule quenches the reporter molecule when the probe is not hybridized, the probe can be designed such that the reporter molecule exhibits limited fluorescence unless the probe is either hybridized or digested.</p>
    <p>According to the present invention. the fluorescence intensity of the reporter molecule is preferably greater than the fluorescence intensity of the quencher molecule when the probe is hybridized to the target polynucleotide. The fluorescence intensity of the reporter molecule is more preferably at least about a factor of 3.5 greater than the fluorescence intensity of the quencher molecule when the probe is hybridized to the target polynucleotide.</p>
    <p>The fluorescence intensity of the oligonucleotide probe hybridized to the target polynucleotide is also preferably at least about a factor of 6 greater than the fluorescence intensity of the oligonucleotide probe when not hybridized to the target polynucleotide.</p>
    <p>The reporter molecule is preferably separated from the quencher molecule by at least about 15 nucleotides, more preferably at least about 18 nucleotides. The reporter molecule is preferably separated from the quencher molecule by between about 15 and 60 nucleotides, more preferably between about 18 and 30 nucleotides.</p>
    <p>The reporter molecule is preferably attached to either the 3′ or 5′ terminal nucleotides of the probe. The quencher molecule is also preferably attached to either the 3′ or 5′ terminal nucieotides of the probe. More preferably, the reporter and quencher molecules are attached to the 3′ and 5′ or 5′ and 3′ terminal nucleotides of the probe respectively.</p>
    <p>The reporter molecule is preferably a fluorescein dye and the quencher molecule is preferably a rhodamine dye.</p>
    <p>In one embodiment, the oligonucleotide probe is attached to a solid support. As illustrated in FIG. 2, when the probe is unhybridized, the probe is able to adopt at least one singlestranded conformation such that the quencher molecule is near enough to the reporter molecule to quench the fluorescence of the reporter molecule. As further illustrated in FIG. 2, when the probe is hybridized to a target sequence. the probe adopts at least one conformation where the quencher molecule is not positioned close enough to the reporter molecule to quench the fluorescence of the reporter molecule. As a result. the fluorescence intensity of the reporter molecule increases when the probe hybridizes to a target sequence.</p>
    <p>As illustrated in FIG. <b>2</b>. different probes may be attached to the solid support and may be used to simultaneously detect different target sequences in a sample. Reporter molecules having different fluorescence wavelengths can be used on the different probes, thus enabling hybridization to the different probes to be separately detected.</p>
    <p>Examples of preferred types of solid supports for immobilization of the oligonucleotide probe include controlled pore glass, glass plates, polystyrene, avidin coated polystyrene beads, cellulose, nylon, acrylamide gel and activated dextran. CPG, glass plates and high cross-linked polystyrene. These solid supports are preferred for hybridization and diagnostic studies because of their chemical stability, ease of functionalization and well defined surface area. Solid supports such as controlled pore glass (CPG, 500 Å, 1000 Å) and non-swelling high cross-linked polystyrene (1000 Å) are particularly preferred in view of their compatibility with oligonucleotide synthesis.</p>
    <p>The oligonucleotide probe may be attached to the solid support in a variety of manners. For example, the probe may be attached to the solid support by attachment of the 3′ or 5′ terminal nucleotide of the probe to the solid support. More preferably, however, the probe is attached to the solid support by a linker which serves to distance the probe from the solid support. The linker is most preferably at least 30 atoms in length, more preferably at least 50 atoms in length.</p>
    <p>The length and chemical stability of linker between solid support and the first 3′ unit of oligonucleotides play an important role in efficient synthesis and hybridization of support bound oligonucleotides. The linker arm should be sufficiently long so that a high yield (&gt;97%) can be achieved during automated synthesis. The required length of the linker will depend on the particular solid support used. For example. a six atom linker is generally sufficient to achieve a &gt;97% yield during automated synthesis of oligonucleotides when high cross-linked polystyrene is used as the solid support. The linker arm is preferably at least 20 atoms long in order to attain a high yield (&gt;97%) during automated synthesis when CPG is used as the solid support.</p>
    <p>Hybridization of a probe immobilized to a solid support generally requires that the probe be separated from the solid support by at least 30 atoms, more preferably at least 50 atoms. In order to achieve this separation, the linker generally includes a spacer positioned between the linker and the 3′ nuclieoside. For oligonucleotide synthesis, the linker arm is usually attached to the 3′ -OH of the 3′ nucleoside by an ester linkage which can be cleaved with basic reagents to free the oligonucleotide from the solid support.</p>
    <p>A wide variety of linkers are known in the art which may be used to attach the oligonucleotide probe to the solid support. The linker may be formed of any compound which does not significantly interfere with the hybridization of the target sequence to the probe attached to the solid support. The linker may be formed of a homopolymeric oligonucleotide which can be readily added on to the linker by automated synthesis. Alternatively, polymers such as functionalized polyethylene glycol can be used as the linker. Such polymers are preferred over homopolymeric oligonucleotides because they do not significantly interfere with the hybridization of probe to the target oligonucleotide. Polyethylene glycol is particularly preferred because it is commercially available, soluble in both organic and aqueous media, easy to functionalize, and completely stable under oligonucleotide synthesis and post-synthesis conditions.</p>
    <p>The linkages between the solid support. the linker and the probe are preferably not cleaved during removal of base protecting groups under basic conditions at high temperature. Examples of preferred linkages include carbamate and amide linkages.</p>
    <p>The oligonucleotide probe of the present invention may be used as a hybridization probe to detect target polynucleotides. Accordingly, the present invention relates to a hybridization assay for detecting the presence of a target polynucleotide in a sample. According to the method. an oligonucleotide probe of the present invention is contacted with a sample of nucleic acids under conditions favorable for hybridization. The fluorescence signal of the reporter molecule is measured before and after being contacted with the sample. Since the reporter molecule on the probe exhibits a greater fluorescence signal when hybridized to a target sequence, an increase in the fluorescence signal after the probe is contacted with the sample indicates the hybridization of the probe to target sequences in the sample and hence the presence of target sequences in the sample. Further, by quantifying the change in fluorescence intensity as a result of the probe being contacted with the sample, the amount of target sequences in the sample can be quantified.</p>
    <p>According to one embodiment of the method, the hybridization probe is immobilized on a solid support. The oligonucleotide probe is contacted with a sample of nucleic acids under conditions favorable for hybridization. The fluorescence signal of the reporter molecule is measured before and after being contacted with the sample. Since the reporter molecule on the probe exhibits a greater fluorescence signal when hybridized to a target sequence, an increase in the fluorescence signal after the probe is contacted with the sample indicates the hybridization of the probe to target sequences in the sample. Immobilization of the hybridization probe to the solid support enables the target sequence hybridized to the probe to be readily isolated from the sample. In later steps, the isolated target sequence may be separated from the solid support and processed (e.g., purified. amplified) according to methods well known in the art depending on the particular needs of the researcher.</p>
    <p>The oligonucleotide probe of the present invention may also be used as a probe for monitoring nucleic acid amplification. Accordingly. the present invention relates to a method for monitoring nucleic acid amplification using an oligonucleotide probe according to the present invention which is capable of hybridizing to the target sequence 3′ relative to an amplification primer. According to the method, nucleic acid amplification is performed on a target polynucleotide using a nucleic acid polymerase having 5′→3′ nuclease activity, a primer capable of hybridizing to the target polynucleotide. and an oligonucleotide probe according to the present invention capable of hybridizing to the target polynucleotide 3′ relative to the primer. During amplification, the nucleic acid polymerase digests the oligonucleotide probe when it is hybridized to the target sequence, thereby separating the reporter molecule from the quencher molecule. As the amplification is conducted, the fluorescence of the reporter molecule is monitored, the generation of fluorescence corresponding to the occurrence of nucleic acid amplification.</p>
    <p>Use of a reporter-quencher pair probe generally in conjunction with the amplification of a target polynucleotide, for example, by PCR, e.g., is described in many references, such as Innis et al., editors, PCR Protocols (Academic Press, New York, 1989); Sambrook et al., Molecular Cloning, Second Edition (Cold Spring Harbor Laboratory, New York, 1989), each of which are incorporated herein by reference. The binding site of the oligonucleotide probe is located between the PCR primers used to amplify the target polynucleotide. Preferably, PCR is carried out using Taq DNA polymerase. e.g., Amplitaq™ (Perkin-Elmer, Norwalk, Conn.), or an equivalent thermostable DNA polymerase, and the annealing temperature of the PCR is about 5-10° C. below the melting temperature of the oligonucleotide probes empioyed.</p>
    <p>Use of an oligonucleotide probe according to the present invention for monitoring nucleic acid amplification provides several advantages over the use of prior art reporter-quencher pair probes. For example. prior art probes required that the reporter and quencher molecules be positioned on the probe such that the quencher molecule remained within a minimum quenching distance of the reporter molecule. However, by realizing that the probe need only be designed such that the probe be able to adopt a conformation where the quencher molecule is within a minimum quenching distance of the reporter molecule, a far wider array of probes are enabled. For example, dually labelled probes having the reporter and quencher molecules at the 5′ and 3′ ends can be designed. Such probes are far easier to synthesize than probes where the reporter molecule or the quencher molecule is attached to an internal nucleotide. Positioning of the reporter and quencher molecules on terminal nucleotides also enhances the hybridization efficiency of the probes.</p>
    <p>As used in this application, the term “oligonucleotide”, includes linear oligomers of natural or modified monomers or linkages, including deoxyribonucleosides, ribonucleosides, and the like; capable of specifically binding a target polynucleotide by way of a regular pattern of monomer-to-monomer interactions, such as Watson-Crick type of basepairing, or the like. Usually monomers are linked by phosphodiester bonds or analogs thereof to form oligonucleotides ranging in size from a few monomeric units, e.g., 3-4, to several tens of monomeric units. Whenever an oligonucleotide is represented by a sequence of letters, such as “ATGCCTG”, it will be understood that the nucleotides are in 5′→<b>3</b>′ order from left to right and that “A” denotes deoxyadenosine, “C” denotes deoxycytidine, “G” denotes deoxyguanosine, and “T” denotes thymidine, unless otherwise noted. Analogs of phosphodiester linkages include phosphorothioate, phosphorodithioate, phosphoranilidate, phosphoramidate, and the like. Generally, oligonucleotide probes of the invention will have a sufficient number of phosphodiester linkages adjacent to its 5′ end so that the 5′→3′ exonuclease activity employed can efficiently degrade the bound probe to separate the reporter and quencher molecules. “Perfectly matched” in reference to a duplex means that the poly- or oligonucleotide strands making up the duplex form a double-stranded structure with one other such that every nucleotide in each strand undergoes Watson-Crick basepairing with a nucleotide in the other strand. The term also comprehends the pairing of nucleoside analogs, such as deoxyinosine, nucleosides with 2-aminopurine bases, and the like, that may be employed. Conversely, a “mismatch” in a duplex between a target polynucleotide and an oligonucleotide probe or primer means that a pair of nucleotides in the duplex fails to undergo Watson-Crick bonding.</p>
    <p>As used in the application, “nucleoside” includes the natural nucleosides, including 2′ -deoxy and 2′ -hydroxyl forms, e.g., as described in Kornberg and Baker. DNA Replication, 2nd Ed. (Freeman, San Francisco, 1992). “Analogs” in reference to nucleosides includes synthetic nucleosides having modified base moieties and/or modified sugar moieties, e.g., described by Scheit, Nucleotide Analogs (John Wiley, New York, 1980); Uhlman and Peyman, Chemical Reviews, 90: 543-584 (1990), or the like, with the only proviso that they are capable of specific hybridization. Such analogs include synthetic nucleosides designed to enhance binding properties. reduce degeneracy, increase specificity, and the like.</p>
    <p>Oligonucleotide probes of the invention can be synthesized by a number of approaches, e.g., Ozaki et al., Nucleic Acids Research, 20:</p>
    <p>5205-5214 (1992); Agrawal et al., Nucleic Acids Research, 18: 5419-5423 (1990); or the like. The oligonucleotide probes of the invention are conveniently synthesized on an automated DNA synthesizer. e.g., an Applied Biosystems. Inc. (Foster City, Calif.) model 392 or 394 DNA/RNA Synthesizer. using standard chemistries. such as phosphoramidite chemistry, e.g., disclosed in the following references: Beaucage and Iyer, Tetrahedron, 48: 2223-2311 (1992); Molko et al., U.S. Pat. No. 4,980,460: Koster et al., U.S. Pat. No. 4,725,677; Caruthers et al., U.S. Pat. Nos. 4,415,732; 4,458,066; and 4,973,679; and the like. Alternative chemistries, e.g., resulting in non-natural backbone groups, such as phosphorothioate, phosphoramidate, and the like, may also be employed provided that the hybridization efficiencies of the resulting oligonucleotides and/or cleavage efficiency of the exonuclease employed are not adversely affected.</p>
    <p>Preferably, the oligonucleotide probe is in the range of 15-60 nucleotides in length. More preferably, the oligonucleotide probe is in the range of 18-30 nucleotides in length. The precise sequence and length of an oligonucleotide probe of the invention depends in part on the nature of the target polynucleotide to which it binds. The binding location and length may be varied to achieve appropriate annealing and melting properties for a particular embodiment. Guidance for making such design choices can be found in many of the above-cited references describing the “Taq-man” type of assays.</p>
    <p>Preferably, the 3′ terminal nucleotide of the oligonucteotide probe is blocked or rendered incapable of extension by a nucleic acid polymerase. Such blocking is conveniently carried out by the attachment of a reporter or quencher molecule to the terminal 3′ carbon of the oligonucleotide probe by a linking moiety.</p>
    <p>Preferably, reporter molecules are fluorescent organic dyes derivatized for attachment to the terminal 3′ carbon or terminal 5′ carbon of the probe via a linking moiety. Preferably, quencher molecules are also organic dyes, which may or may not be fluorescent, depending on the embodiment of the invention. For example, in a preferred embodiment of the invention, the quencher molecule is fluorescent. Generally whether the quencher molecule is fluorescent or simply releases the transferred energy from the reporter by non-radiative decay, the absorption band of the quencher should substantially overlap the fluorescent emission band of the reporter molecule. Non-fluorescent quencher molecules that absorb energy from excited reporter molecules, but which do not release the energy radiatively, are referred to in the application as chromogenic molecules.</p>
    <p>There is a great deal of practical guidance available in the literature for selecting appropriate reporterluencher pairs for particular probes, as exemplified by the following references: Clegg (cited above); Wu et at. (cited above); Pesce et al., editors, Fluorescence Spectroscopy (Marcel Dekker, New York, 1971); White et al., Fluorescence Analysis: A Practical Approach (Marcel Dekker, New York, 1970); and the like. The literature also includes references providing exhaustive lists of fluorescent and chromogenic molecules and their relevant optical properties for choosing reporter-quencher pairs. e.g., Berlman, Handbook of Fluorescence Spectra of Aromatic Molecules, 2nd Edition (Academic Press, New York, 1971); Griffiths, Colour and Constitution of Organic Molecules (Academic Press, New York, 1976); Bishop, editor, Indicators (Pergamon Press, Oxford, 1972); Haugland, Handbook of Fluorescent Probes and Research Chemicals (Molecular Probes. Eugene, 1992) Pringsheim, Fluorescence and Phosphorescence (Interscience Publishers. New York, 1949); and the like. Further, there is extensive guidance in the literature for derivatizing reporter and quencher molecules for covalent attachment via common reactive groups that can be added to an oligonucleotide, as exemplified by the following references: Haugland (cited above); Ullman et al., U.S. Pat. No. 3,996,345; Khanna et al., U.S. Pat. No. 4,351,760; and the like.</p>
    <p>Exemplary reporter-quencher pairs may be selected from xanthene dyes, including fluoresceins, and rhodamine dyes. Many suitable forms of these compounds are widely available commercially with substituents on their phenyl moieties which can be used as the site for bonding or as the bonding functionality for attachment to an oligonucleotide. Another group of fluorescent compounds are the naphthylamines. having an amino group in the alpha or beta position. Included among such naphthylamino compounds are 1-dimethylaminonaphthyl-5-sulfonate. 1 -anilino-8-naphthalene sulfonate and 2-p-touidinyl-6-naphthalene sulfonate. Other dyes include 3-phenyl-7-isocyanatocoumarin, acridines, such as 9-isothiocyanatoacridine and acridine orange; N-(p-(2-benzoxazolyl)phenyl)maleimide; benzoxadiazoles, stilbenes, pyrenes, and the like.</p>
    <p>Preferably, reporter and quencher molecules are selected from fluorescein and rhodamine dyes. These dyes and appropriate linking methodologies for attachment to oligonucleotides are described in many references, e.g., Khanna et al. (cited above); Marshall, Histochemical J., 7: 299-303 (1975); Menchen et al., U.S. Pat. No. 5,188,934; Menchen etal., European Patent Application 87310256.0; and Bergot et al., International Application PCT/US90/05565. The latter four documents are hereby incorporated by reference.</p>
    <p>There are many linking moieties and methodologies for attaching reporter or quencher molecules to the 5′ or 3′ termini of oligonucleotides, as exemplified by the following references: Eckstein, editor, Oligonucleotides and Analogues: A Practical Approach (IRL Press, Oxford, 1991); Zuckerman et al., Nucleic Acids Research, 15: 5305-5321 (1987) (3′ thiol group on oligonucleotide); Sharma et al., Nucleic Acids Research, 19: 3019 (1991) (3′ sulfhydryl); Giusti et al., PCR Methods and Applications, 2: 223-227 (1993) and Fung et al., U.S. Pat. No. 4,757,141 (5′ phosphoamino group via Aminolink™ II available from Applied Biosystems, Foster City, Calif.) Stabinsky, U.S. Pat. No. 4,739,044 (3′ aminoaikyiphosphoryl group); Agrawal et al., Tetrahedron Letters. 31: 1543-1546 (1990) (attachment via phosphoramidate linkages); Sproat et al., Nucleic Acids Research, 15: 4837 (1987) (5′ mercapto group); Nelson et al., Nucleic Acids Research, 17: 7187-7194 (1989) (3′ amino group); and the like.</p>
    <p>Preferably, commercially available linking moieties are employed that can be attached to an oligonucleotide during synthesis, e.g., available from Clontech Laboratories (Palo Alto, Calif.).</p>
    <p>Rhodamine and fluorescein dyes are also conveniently attached to the 5′ hydroxyl of an oligonucleotide at the conclusion of solid phase synthesis by way of dyes derivatized with a phosphoramidite moiety, e.g. Woo et al., U.S. Pat. No. 5,231,191; and Hobbs, Jr., U.S. Pat. No. 4,997,928.</p>
    <p>The following examples set forth probes and methods for using the probes according to the present invention. It is understood that the specific probes, probe constructs and steps of the methods described in these examples are not intended to be limiting. Further objectives and advantages of the present invention other than those set forth above will become apparent from the examples which are not intended to limit the scope of the present invention.</p>
    <heading>EXAMPLES</heading> <heading>1. Synthesis of Oligonucleotide Probes</heading> <p>The following example describes the synthesis of the oligonucleotides shown in Table 1. Linker arm nucleotide (“LAN”) phosphoramidite was obtained from Glen Research. Standard DNA phosphoramidites. 6-carboxyfluorescein (“6-FAM”) phosphoramidite. 6-carboxytetramethylrhodamine succinimidyl ester (“TAMRA NHS ester”), and Phosphalink™ for attaching a 3′ blocking phosphate were obtained from Perkin-Elmer, Applied Biosystems Division. Oligonucleotide synthesis was performed on a model 394 DNA Synthesizer (Applied Biosystems). Primer and complement oligonucieotides were purified using Oligo Purification Cartridges (Applied Biosystems). Doubly labeled probes were synthesized with 6-FAM-labeled phosphoramidite at the 5′ end, LAN replacing one of the T's in the oligonucleotide sequence, and Phosphalink™ at the 3′ end. Following deprotection and ethanol precipitation. TAMRA NHS ester was coupled to the LAN-containing oligonucleotide in 250 mM Na-bicarbonate buffer (pH 9.0) at room temperature. Unreacted dye was removed by passage over a PD-10 Sephadex column. Finally, the doubly labeled probe was purified by preparative HPLC using standard protocols. Below, probes are named by designating the sequence from Table 1 and the position of the LAN-TAMRA moiety. For example, probe A1-7 has sequence of A1 with LAN-TAMRA at nucleoside position 7 from the 5′ end.</p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup cols="1" colsep="0" rowsep="0" align="left"> <colspec colname="1" align="center" colwidth="217PT"> </colspec> <thead valign="bottom"> <tr class="description-tr"> <td namest="1" nameend="1" morerows="0" rowsep="1" valign="top" class="description-td" colspan="1">TABLE 1</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" morerows="0" rowsep="1" valign="top" align="center" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td">Sequences of oligonucleotides</td>
              </tr> </tbody> </tgroup> <tgroup cols="3" colsep="0" rowsep="0" align="left"> <colspec colname="1" align="left" colwidth="21PT"> </colspec> <colspec colname="2" align="left" colwidth="42PT"> </colspec> <colspec colname="3" align="left" colwidth="154PT"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td">Name</td>
                <td morerows="0" valign="top" class="description-td">Type</td>
                <td morerows="0" valign="top" class="description-td">Sequence</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="3" morerows="0" rowsep="1" valign="top" align="center" class="description-td" colspan="3"> </td>
              </tr> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td">F119</td>
                <td morerows="0" valign="top" class="description-td">primer</td>
                <td morerows="0" valign="top" class="description-td">ACCCACAGGAACTGATCACCACTC</td>
              </tr> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td"> </td>
                <td morerows="0" valign="top" class="description-td"> </td>
                <td morerows="0" valign="top" class="description-td">[SEQ. ID. No.: 1]</td>
              </tr> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td">R119</td>
                <td morerows="0" valign="top" class="description-td">primer</td>
                <td morerows="0" valign="top" class="description-td">ATGTCGCGTTCCGGCTGACGTTCTGC</td>
              </tr> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td"> </td>
                <td morerows="0" valign="top" class="description-td"> </td>
                <td morerows="0" valign="top" class="description-td">[SEQ. ID. No.: 2]</td>
              </tr> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td">P2</td>
                <td morerows="0" valign="top" class="description-td">probe</td>
                <td morerows="0" valign="top" class="description-td">TCGCAT<u>T</u>ACTGATCGTTGCCAACCAG<u>Tp</u> </td>
              </tr> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td"> </td>
                <td morerows="0" valign="top" class="description-td"> </td>
                <td morerows="0" valign="top" class="description-td">[SEQ. ID. No.: 3]</td>
              </tr> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td">P2C</td>
                <td morerows="0" valign="top" class="description-td">complement</td>
                <td morerows="0" valign="top" class="description-td">GTACTGGTTGGCAACGATCAGTAATGCGATG</td>
              </tr> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td"> </td>
                <td morerows="0" valign="top" class="description-td"> </td>
                <td morerows="0" valign="top" class="description-td">[SEQ. ID. No.: 4]</td>
              </tr> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td">P5</td>
                <td morerows="0" valign="top" class="description-td">probe</td>
                <td morerows="0" valign="top" class="description-td">CGGATTTGC<u>T</u>GGTATCTATGACAAGGA<u>Tp</u> </td>
              </tr> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td"> </td>
                <td morerows="0" valign="top" class="description-td"> </td>
                <td morerows="0" valign="top" class="description-td">[SEQ. ID. No.: 5]</td>
              </tr> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td">P5C</td>
                <td morerows="0" valign="top" class="description-td">complement</td>
                <td morerows="0" valign="top" class="description-td">TTCATCCTTGTCATAGATACCAGCAAATCCG</td>
              </tr> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td"> </td>
                <td morerows="0" valign="top" class="description-td"> </td>
                <td morerows="0" valign="top" class="description-td">[SEQ. ID. No.: 6]</td>
              </tr> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td">AFP</td>
                <td morerows="0" valign="top" class="description-td">primer</td>
                <td morerows="0" valign="top" class="description-td">TCACCCACACTGTGCCCATCTACGA</td>
              </tr> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td"> </td>
                <td morerows="0" valign="top" class="description-td"> </td>
                <td morerows="0" valign="top" class="description-td">[SEQ. ID. No.: 7]</td>
              </tr> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td">ARP</td>
                <td morerows="0" valign="top" class="description-td">primer</td>
                <td morerows="0" valign="top" class="description-td">CAGCGGAACCGCTCATTGCCAATGG</td>
              </tr> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td"> </td>
                <td morerows="0" valign="top" class="description-td"> </td>
                <td morerows="0" valign="top" class="description-td">[SEQ. ID. No.: 8]</td>
              </tr> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td">A1</td>
                <td morerows="0" valign="top" class="description-td">probe</td>
                <td morerows="0" valign="top" class="description-td">A<u>T</u>GCCC<u>T</u>CCCCCA<u>T</u>GCCA<u>T</u>CC<u>T</u>GCG<u>Tp</u> </td>
              </tr> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td"> </td>
                <td morerows="0" valign="top" class="description-td"> </td>
                <td morerows="0" valign="top" class="description-td">[SEQ. ID. No.: 9]</td>
              </tr> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td">A1C</td>
                <td morerows="0" valign="top" class="description-td">complement</td>
                <td morerows="0" valign="top" class="description-td">AGACGCAGGATGGCATGGGGGAGGGCATAC</td>
              </tr> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td"> </td>
                <td morerows="0" valign="top" class="description-td"> </td>
                <td morerows="0" valign="top" class="description-td">[SEQ. ID. No.: 10]</td>
              </tr> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td">A3</td>
                <td morerows="0" valign="top" class="description-td">probe</td>
                <td morerows="0" valign="top" class="description-td">CGCCC<u>T</u>GGACTTCGAGCAAGAGA<u>Tp</u> </td>
              </tr> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td"> </td>
                <td morerows="0" valign="top" class="description-td"> </td>
                <td morerows="0" valign="top" class="description-td">[SEQ. ID. No.: 11]</td>
              </tr> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td">A3C</td>
                <td morerows="0" valign="top" class="description-td">complement</td>
                <td morerows="0" valign="top" class="description-td">CCATCTCTTGCTCGAAGTCCAGGGCGAC</td>
              </tr> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td"> </td>
                <td morerows="0" valign="top" class="description-td"> </td>
                <td morerows="0" valign="top" class="description-td">[SEQ. ID. No.: 12]</td>
              </tr> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td">G1</td>
                <td morerows="0" valign="top" class="description-td">probe</td>
                <td morerows="0" valign="top" class="description-td">CAAGCTTCCCGTTCTCAGCCT</td>
              </tr> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td"> </td>
                <td morerows="0" valign="top" class="description-td"> </td>
                <td morerows="0" valign="top" class="description-td">[SEQ. ID. No.: 13]</td>
              </tr> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td">G1C</td>
                <td morerows="0" valign="top" class="description-td">complement</td>
                <td morerows="0" valign="top" class="description-td">ACCGTCAAGGCTGAGAACGGGAAGCTTGTC</td>
              </tr> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td"> </td>
                <td morerows="0" valign="top" class="description-td"> </td>
                <td morerows="0" valign="top" class="description-td">[SEQ. ID. No.: 14]</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="3" morerows="0" rowsep="1" valign="top" align="center" class="description-td" colspan="3"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup cols="1" colsep="0" rowsep="0" align="left"> <colspec colname="1" align="left" colwidth="217PT"> </colspec> <thead valign="bottom"> <tr class="description-tr"> <td namest="1" nameend="1" morerows="0" rowsep="1" valign="top" class="description-td" colspan="1">TABLE 2</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="1" morerows="0" rowsep="1" valign="top" align="center" class="description-td" colspan="1"> </td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td">
                  <chemistry> <div class="patent-image small-patent-image"> <a href="//patentimages.storage.googleapis.com/US6258569B1/US06258569-20010710-C00001.png"> <img id="EMI-C00001" file="US06258569-20010710-C00001.TIF" img-content="chem" img-format="tif" src="//patentimages.storage.googleapis.com/US6258569B1/US06258569-20010710-C00001.png" class="patent-full-image" alt="Figure US06258569-20010710-C00001"> </a> </div>
                    <attachments> <attachment idref="CHEMCDX-00001" attachment-type="cdx" file="US06258569-20010710-C00001.CDX"> </attachment> <attachment idref="CHEMMOL-00001" attachment-type="mol" file="US06258569-20010710-C00001.MOL"> </attachment> </attachments> </chemistry> </td>
              </tr> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td">
                  <chemistry> <div class="patent-image small-patent-image"> <a href="//patentimages.storage.googleapis.com/US6258569B1/US06258569-20010710-C00002.png"> <img id="EMI-C00002" file="US06258569-20010710-C00002.TIF" img-content="chem" img-format="tif" src="//patentimages.storage.googleapis.com/US6258569B1/US06258569-20010710-C00002.png" class="patent-full-image" alt="Figure US06258569-20010710-C00002"> </a> </div>
                    <attachments> <attachment idref="CHEMCDX-00002" attachment-type="cdx" file="US06258569-20010710-C00002.CDX"> </attachment> <attachment idref="CHEMMOL-00002" attachment-type="mol" file="US06258569-20010710-C00002.MOL"> </attachment> </attachments> </chemistry> </td>
              </tr> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td">
                  <chemistry> <div class="patent-image small-patent-image"> <a href="//patentimages.storage.googleapis.com/US6258569B1/US06258569-20010710-C00003.png"> <img id="EMI-C00003" file="US06258569-20010710-C00003.TIF" img-content="chem" img-format="tif" src="//patentimages.storage.googleapis.com/US6258569B1/US06258569-20010710-C00003.png" class="patent-full-image" alt="Figure US06258569-20010710-C00003"> </a> </div>
                    <attachments> <attachment idref="CHEMCDX-00003" attachment-type="cdx" file="US06258569-20010710-C00003.CDX"> </attachment> <attachment idref="CHEMMOL-00003" attachment-type="mol" file="US06258569-20010710-C00003.MOL"> </attachment> </attachments> </chemistry> </td>
              </tr> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td">
                  <chemistry> <div class="patent-image small-patent-image"> <a href="//patentimages.storage.googleapis.com/US6258569B1/US06258569-20010710-C00004.png"> <img id="EMI-C00004" file="US06258569-20010710-C00004.TIF" img-content="chem" img-format="tif" src="//patentimages.storage.googleapis.com/US6258569B1/US06258569-20010710-C00004.png" class="patent-full-image" alt="Figure US06258569-20010710-C00004"> </a> </div>
                    <attachments> <attachment idref="CHEMCDX-00004" attachment-type="cdx" file="US06258569-20010710-C00004.CDX"> </attachment> <attachment idref="CHEMMOL-00004" attachment-type="mol" file="US06258569-20010710-C00004.MOL"> </attachment> </attachments> </chemistry> </td>
              </tr> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td">
                  <chemistry> <div class="patent-image small-patent-image"> <a href="//patentimages.storage.googleapis.com/US6258569B1/US06258569-20010710-C00005.png"> <img id="EMI-C00005" file="US06258569-20010710-C00005.TIF" img-content="chem" img-format="tif" src="//patentimages.storage.googleapis.com/US6258569B1/US06258569-20010710-C00005.png" class="patent-full-image" alt="Figure US06258569-20010710-C00005"> </a> </div>
                    <attachments> <attachment idref="CHEMCDX-00005" attachment-type="cdx" file="US06258569-20010710-C00005.CDX"> </attachment> <attachment idref="CHEMMOL-00005" attachment-type="mol" file="US06258569-20010710-C00005.MOL"> </attachment> </attachments> </chemistry> </td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="1" morerows="0" rowsep="1" valign="top" align="center" class="description-td" colspan="1"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" pgwide="1" class="description-table"> <tgroup cols="1" colsep="0" rowsep="0" align="left"> <colspec colname="1" align="center" colwidth="441PT"> </colspec> <thead valign="bottom"> <tr class="description-tr"> <td namest="1" nameend="1" morerows="0" rowsep="1" valign="top" class="description-td" colspan="1">TABLE 3</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="1" morerows="0" rowsep="1" valign="top" align="center" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td">TAMRA Dye Labeled High Cross-Linked Polystyrene Support</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="1" morerows="0" rowsep="1" valign="top" align="center" class="description-td" colspan="1"> </td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td"> </td>
              </tr> </tbody> </tgroup> <tgroup cols="1" colsep="0" rowsep="0" align="left"> <colspec colname="1" align="left" colwidth="441PT"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td">
                  <chemistry> <div class="patent-image small-patent-image"> <a href="//patentimages.storage.googleapis.com/US6258569B1/US06258569-20010710-C00006.png"> <img id="EMI-C00006" file="US06258569-20010710-C00006.TIF" img-content="chem" img-format="tif" src="//patentimages.storage.googleapis.com/US6258569B1/US06258569-20010710-C00006.png" class="patent-full-image" alt="Figure US06258569-20010710-C00006"> </a> </div>
                    <attachments> <attachment idref="CHEMCDX-00006" attachment-type="cdx" file="US06258569-20010710-C00006.CDX"> </attachment> <attachment idref="CHEMMOL-00006" attachment-type="mol" file="US06258569-20010710-C00006.MOL"> </attachment> </attachments> </chemistry> </td>
              </tr> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td">
                  <chemistry> <div class="patent-image small-patent-image"> <a href="//patentimages.storage.googleapis.com/US6258569B1/US06258569-20010710-C00007.png"> <img id="EMI-C00007" file="US06258569-20010710-C00007.TIF" img-content="chem" img-format="tif" src="//patentimages.storage.googleapis.com/US6258569B1/US06258569-20010710-C00007.png" class="patent-full-image" alt="Figure US06258569-20010710-C00007"> </a> </div>
                    <attachments> <attachment idref="CHEMCDX-00007" attachment-type="cdx" file="US06258569-20010710-C00007.CDX"> </attachment> <attachment idref="CHEMMOL-00007" attachment-type="mol" file="US06258569-20010710-C00007.MOL"> </attachment> </attachments> </chemistry> </td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="1" morerows="0" rowsep="1" valign="top" align="center" class="description-td" colspan="1"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" pgwide="1" class="description-table"> <tgroup cols="1" colsep="0" rowsep="0" align="left"> <colspec colname="1" align="center" colwidth="441PT"> </colspec> <thead valign="bottom"> <tr class="description-tr"> <td namest="1" nameend="1" morerows="0" rowsep="1" valign="top" class="description-td" colspan="1">TABLE 4</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="1" morerows="0" rowsep="1" valign="top" align="center" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td">TAMRA Dye Labeled Controlled Pore Glass (CPG) Support</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="1" morerows="0" rowsep="1" valign="top" align="center" class="description-td" colspan="1"> </td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td"> </td>
              </tr> </tbody> </tgroup> <tgroup cols="1" colsep="0" rowsep="0" align="left"> <colspec colname="1" align="left" colwidth="441PT"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td">
                  <chemistry> <div class="patent-image small-patent-image"> <a href="//patentimages.storage.googleapis.com/US6258569B1/US06258569-20010710-C00008.png"> <img id="EMI-C00008" file="US06258569-20010710-C00008.TIF" img-content="chem" img-format="tif" src="//patentimages.storage.googleapis.com/US6258569B1/US06258569-20010710-C00008.png" class="patent-full-image" alt="Figure US06258569-20010710-C00008"> </a> </div>
                    <attachments> <attachment idref="CHEMCDX-00008" attachment-type="cdx" file="US06258569-20010710-C00008.CDX"> </attachment> <attachment idref="CHEMMOL-00008" attachment-type="mol" file="US06258569-20010710-C00008.MOL"> </attachment> </attachments> </chemistry> </td>
              </tr> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td">
                  <chemistry> <div class="patent-image small-patent-image"> <a href="//patentimages.storage.googleapis.com/US6258569B1/US06258569-20010710-C00009.png"> <img id="EMI-C00009" file="US06258569-20010710-C00009.TIF" img-content="chem" img-format="tif" src="//patentimages.storage.googleapis.com/US6258569B1/US06258569-20010710-C00009.png" class="patent-full-image" alt="Figure US06258569-20010710-C00009"> </a> </div>
                    <attachments> <attachment idref="CHEMCDX-00009" attachment-type="cdx" file="US06258569-20010710-C00009.CDX"> </attachment> <attachment idref="CHEMMOL-00009" attachment-type="mol" file="US06258569-20010710-C00009.MOL"> </attachment> </attachments> </chemistry> </td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="1" morerows="0" rowsep="1" valign="top" align="center" class="description-td" colspan="1"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <heading>2. Synthesis of Oligonucleotide Probes Attached To A Solid Support</heading> <p>Both high cross-linked polystyrene (1000 Å) and controlled pore glass (CPG) (500 Å) are used as solid support matrices. The functionalization of a spacer (compound 5) is illustrated in Table 2. The attachment of the spacer to polystyrene and CPG supports. and the labelling of the solid supports with TAMRA dye is shown in Tables 3 and 4 respectively.</p>
    <p>Table 2 illustrates a reaction scheme for the synthesis of a spacer, compound 5, which is used to derivatize CPG and polystyrene supports. As shown in Table 2, N-Fmoc-ε-aminocaproic acid was reacted with DL-homoserine in presence of HOBT/HBTU/DIPEA (Knorr, et al., <i>Tetrahedron Lett. </i>1989, 30, 1927) in DMF to give compound 2 in 65% yield. Compound 2 was reacted with dimethoxytrityl chloride in presence of DMAP in pyridine to give compound 3 in 72% yield after chromatography. Treatment of compound 3 with a large excess of PEG-diamine (Buckmann, et al., <i>Biotech. Appl. Biochem. </i>1987, 9, 258) in presence of HOBT/HBTU/DIPEA in DMF afforded amine 4 in 60% yield. The amine 4 was then converted to succinate 5 by treating amine 4 with succinic anhydride/Et<sub>3</sub>N/DMAP in CH<sub>2</sub>Cl<sub>2 </sub>in 90% yield. The succinate 5 was then attached to polystyrene and CPG support as illustrated in Tables 3 and 4 respectively without further purification.</p>
    <p>As illustrated in Tables 3 and 4. succinate 5 was separately reacted with polystyrene and CPG support in presence of HOBT/HBTU/DIPEA in DMF to provide functionalized support 6 (5 μmol/g loading) and functionalized support 8 (15 μmol/g loading) respectively. The Fmoc group was removed from support bound spacer by treating supports 6 and 8 with 20% piperidine in DMF (Fields, et al., <i>J. Peptide Res. </i>1990. 35. 161) to give amine which was reacted with TAMRA NHS ester to give TAMRA labeled supports 7 and 9 respectively. The polystyrene and CPG supports showed a final loading of 4.8 μmol/g and 14 μmol/g respectively by trityl cation assay.</p>
    <p>Double labeled Taqman probe was synthesized using both TAMRA labeled supports 7 and 9, FastPhoramidites (User Bulletin Number 85, Perkin Elmer Corporation 1994) and FAM phosphoramidite (User Bulletin Number 78. Perkin Elmer Corporation 1994) in 40 nanomol scale. The support bound oligonucleotides were deprotected by treating with MeOH:t-BuNH<sub>2</sub>:H<sub>2</sub>O (1:1:2) at 65° C. for 3 hours (Woo, et al., U.S. Pat. No. 5,231,191). Liquid was removed and the support containing probes were washed with H<sub>2</sub>O:MeOH (3:1) and MeOH. The support was then dried under vacuum and used in a hybridization assay.</p>
    <heading>Experimental</heading> <p>Compound 2: N,N-Diisopropylethylamine (1.1 g, 1.48 mL, 8.52 mmol), 1-hydroxybenzotriazol (420 mg, 3.1 mmol) and (2-(1H-benzotriazol-1-yl)-1,1,3,3-tetramethyluronium hexafluorophosphate (1.17 g, 3.1 mmol) were added to a stirred solution of Nfmoc-ε-aminocaproic acid (1 g, 2.84 mmol) in DMF (30 mL) at room temperature. After 15 min DL-homoserine (1.35 g, 11.36 mmol) was added to the reaction mixture. After 3 hours, DMF was removed under reduced pressure. The residue was dissolved in CHCl<sub>3 </sub>(100 mL) and washed with 5% aqueous HCl (2×50 mL). The organic layer was dried over MgSO<sub>4 </sub>and evaporated to give a thick oil which was trituated with ether to give a colorless solid (840 mg, 65%). The product was left under high vacuum for 24 hours and used in the next step without further purification.</p>
    <p>Compound 3: 4,4′ -Dimethoxytrityl chloride (484 mg, 1.43 mmol) and 4-dimethyaminopyridine (25 mg, 0.2 mmol) were added to a stirred solution of compound 2 (500 mg, 1.1 mmol) in pyridine (15 mL) at room temperature under nitrogen atmosphere. After 14 hours, pyridine was removed and the residue was dissolved in CHCl<sub>3 </sub>(70 mL). The organic layer was extracted with 5% aqueous citric acid (1×50 mL), H<sub>2</sub>O (1×50 mL) and saturated brine (1×50 mL). The organic layer was dried over MgSO<sub>4 </sub>and evaporated to give a yellow foam. The product was purified by a silica gel column eluting with CHCl<sub>3</sub>—MeOH gradient (0-10% MeOH). The appropriate fractions were combined and evaporated to give Compound 3 as a colorless foam (600 mg, 72%).</p>
    <p>Compound 4: Poly(ethylene glycol) bis(2-aminoethyl ether) (3.16 g, 5.3 mmol), N,N-diisopropylethylamine (205 mg, 0.27 mL, 1.59 mmoi), 1-hydroxybenzotriazol (78 mg, 0.58 mmol) and (2-(1H-benzotriazol-1-yl)-1,1,3,3-tetramethyluonium hexafluorophosphate (220 mg, 0.58 mmol) were added to a stirred solution of compound 3 (400 mg, 0.53 mmol) in DMF (10 mL) at room temperature. The reaction mixture was stirred at room temperature for 3 hours. DMF was removed under reduced pressure and, the residue was dissolved in CHCl<sub>3 </sub>(70 mL) and washed with H<sub>2</sub>O (1×50 mL) and saturated brine (2×50 mL). The organic layer was dried over MgSO<sub>4 </sub>and evaporated to give a thick oil. Compound 4 was purified by a silica gel column eluting with a CHCl<sub>3</sub>—MeOH gradient (5-15% MeOH) as a colorless glass (423 mg, 60%).</p>
    <p>Compound 5: Succinic anhydride (22 mg, 0.22 mmol), Et<sub>3</sub>N (23 mg, 0.31 μL, 0.22 mmol), 4-dimethylaminopyridine (14 mg, 0.11 mmol) were added to a solution of compound 4 (300 mg, 0.22 mmol) in CH<sub>2</sub>Cl<sub>2 </sub>(15 mL). The reaction mixture was stirred at room temperature for 3 hours. The reaction mixture was diluted with CHCl<sub>3 </sub>(30 mL) and washed with 5% aqueous citric acid (1×50 mL) and saturated brine (2×50 mL). The organic layer was dried over MgSO<sub>4 </sub>and evaporated to a colorless foam (284 mg, 90%) which was used for derivatization of the solid support without further purification.</p>
    <p>Derivatization of Polystyrene support with TAMRA dye: High cross linked polystyrene 1000 Å, 10 μmol/g amine loading, 1 g, 10 μmol), was treated with compound 5 (17 mg, 12 μmol, 1-hydroxybenzotriazol (1.3 mg, 12 μmol), (2-(1H-benzotriazol-1-yl)-1,1,3,3-tetramethyluonium hexafluorophosphate (4.8 mg, 12 μmol), N,N-diisopropylethylamine (6 μL, 30 μmol) in DMF (10 mL) on a wrist action shaker for 4 hours at room temperature. The support was washed with DMF (3×10 mL), CH<sub>3</sub>CN (2×10 mL) and CH<sub>2</sub>Cl<sub>2 </sub>(1×10 mL) and dried under high vacuum overnight. The ninhydrin assay showed 1 μmol/g amine left. The trityl cation assay gave 5 μmol/g loading of compound 5. The support was capped with acetic anhydride/lutidine in THF (10% solution, 5 mL) and 1-methylimidazol in THF (16% solution, 5 mL) for 2 hours at room temperature. The support was washed with CH<sub>3</sub>CN (3×10 mL) and CH<sub>2</sub>Cl<sub>2 </sub>(1×10 mL). The support was treated with 20% piperidine in DMF (3×10 mL) to remove the Fmoc protecting group. The removal of the Fmoc group was monitored by measuring UV of the solution at 302 nm. The support was washed with DMF (3×10 mL) and, then treated with TAMRA NHS ester (15 mg, 27 μmol) and Et<sub>3</sub>N (50 μmol) in DMF (10 mL) for 42 hours on a shaker. The support was washed with DMF (3×10 mL) CH<sub>3</sub>CN (2×10 mL) and CH<sub>2</sub>Cl<sub>2 </sub>(1×10 mL) and dried under high vacuum for 24 hours. Ninhydrin test showed less than 0.5 μmol/g amine left. The support was capped with acetic anhydride/lutidine in THF (10% solution, 5 mL) and 1-methylimidazol in THF (16% solution, 5 mL) for 1 hour and then washed with CH<sub>3</sub>CN (3×10 mL), CH<sub>2</sub>Cl<sub>2 </sub>(2×10 mL) and dried under high vacuum for 24 hour. The trityl cation assay showed a final loading of 4.8 μmol/g.</p>
    <p>Derivatization of CPG support with TAMRA dye: A mixture of CPG (500 Å, 40 μmol/g amine loading, 500 mg, 20 μmol), compound 5 (31 mg, 22 μmol), 1-hydroxybenzotriazol (5.9 mg, 22 μmol), (2-(1H-benzotriazol-1-yl)-1,1,3,3-tetramethyluronium hexaflurophosphate (8.4 mg, 22 μmol), N,N-diisopropylethylamine (10.4 μL, 60 μmol) in DMF (10 mL) was shaken on a wrist action shaker for 4 hours at room temperature. The support was washed with DMF (3×10 mL), CH<sub>2</sub>CN (2×10 mL) and CH<sub>2</sub>Cl<sub>2 </sub>(1×10 mL) and dried under high vacuum overnight. The ninhydrin assay showed 4 μmol/g amine left. The trityl assay gave 15 μmol/g loading of compound 5 on CPG support. The support was capped with acetic anhydride/lutidine in THF (10% solution, 5 mL) and 1-methylimidazol in THF (16% solution, 5 mL) for 2 hours at room temperature. The support was washed with CH<sub>3</sub>CN (3×10 mL) and CH<sub>2</sub>Cl<sub>2 </sub>(1×10 mL). The support was treated with 20% piperidine in DMF (3×10 mL) to remove the Fmoc protecting group. Removal of the Fmoc group was monitored by measuring UV of the solution at 302 nm. The support was washed with DMF (3×10 mL). The support was then treated with TAMRA NHS ester (25 mg, 45 μmol) and Et<sub>3</sub>N (90 μmol) in DMF (5 mL) for 42 hours on a shaker. The support was washed with DMF (3×10 mL), CH<sub>3</sub>CN (2×10 mL) and CH<sub>2</sub>Cl<sub>2 </sub>(1×10 mL) and dried under high vacuum for 24 hours. Ninhydrn test showed less than 1 μmol/g amine left. The support was capped with acetic anhydride/lutidine in THF (10% solution. 5 mL) and 1-methylimidazol in THF (16% solution, 5 mL) for 1 hour and then washed with CH<sub>3</sub>CN (3×10 mL), CH<sub>2</sub>Cl<sub>2 </sub>(2×10 mL) and dried under high vacuum for 24 hours. The trtyl cation assay showed a final loading of 14 μmol/g.</p>
    <p>Synthesis of FAM and TAMRA Doubled Labeled Probes: Doubled dye labeled oligonucleotide probe were synthesized by using TAMRA labelled supports 7 and 9, DNA FastPhosphoramidite and FAM amidite in 40 nmol scale. After completion of synthesis, supports containing probes were transferred to 4 mL glass vials and treated with a mixture of MeOH:t-BuNH<sub>2</sub>:H<sub>2</sub>O (1:1:2) at 65° C. for 3 hours. Liquid was removed by a syringe and the support was washed with H<sub>2</sub>O:MeOH (3:1) and MeOH. The support was dried under vacuum and used in the hybridization assay.</p>
    <p>3. Hybridization Assay Using Oligonucleotide Probe</p>
    <p>A 295 basepair segment of exon 3 of human beta-actin gene (nucleotides 2141-2435 as disclosed in Nakajima-Iijima, S., <i>Proc. Natl Acad. Sci. USA </i>82: 6133-6137 (1985) can be amplified using 50 ul reactions that contain 10 mM Tris—HCl (pH 8.3), 50 mM KCl, 4 mM MgCl<sub>2</sub>, 300 nM primer AFP [SEQ. I.D. No. 7], 300 nM primer biotin-ARP [SEQ. I.D. No. 8 with biotin attached to the 5′ end], 200 μM dATP, 200 μM dCTP, 200 μM dGTP, 200 μM TTP, and 1.25 units AmpliTaq (Perkin-Elmer). The reactions are performed with (+template) or without (no template) 20 ng human genomic DNA.</p>
    <p>After thermal cycling at 50° C. (2 min); 95° C. (10 min); and 40 cycles of 95° C. (20 sec) followed by 60 IC (1 min), each sample is diluted by adding 200 μl Hybridization Buffer (5X SSC, 8% (v/v) formamide, 8% (v/v) Triton X-100). The resulting samples are transferred to a streptavidin-coated 96-well microtiter plate (Xenopore Corp., Saddle Brook, N.J.) and incubated at 37° C. for 30 min in order to capture the amplified beta-actin DNA segment. Each well is then washed with 350 μl phosphate buffered saline/0.05% TWEEN-20. Any unbiotinylated DNA strands are removed by adding 100 μl 0.1 M NaOH/1 mM EDTA, incubating at room temperature for 5 min, and washing with 350 ul phosphate buffered saline/0.05% TWEEN-20. 50 ul of Hybridization Buffer containing 100 nM of probe A1-26 [SEQ. I.D. No. 9, nucleotides 1-26 (A1-26), labeled with reporter FAM and quencher TAMRA) is then added and incubate at 37° C. for 30 min.</p>
    <p>Fluorescence is then measured at 518 nm and 582 nm using a Perkin-Elmer TaqMan LS-50B System. The ΔRQ is then calculated as described in Example 5. The magnitude of ΔRQ indicates the level of hybridization of the A1-26 probe and thus is a measure of the amount of amplified beta-actin DNA segment captured in each well.</p>
    <heading>4. Hybridization Assay Using Oligonucleotide Probe Attached To Solid Support</heading> <p>Three probe/solid support combinations were examined: A1-PS: A1 [SEQ. I.D. No. 9] attached to polystyrene support: A1-CPG: A1 [SEQ. ID. No. 9] attached to glass support: and G1-PS: G1 [SEQ. I.D. No. 13] attached to polystyrene support.</p>
    <p>All three probes have FAM attached to the 5′ end of the sequence and TAMRA attached to the 3′ end. No template reactions were prepared by suspending each probe/solid support sample in 50 μl 1X PCR Buffer (10 mM Tris-HCl (pH 8.3), 50 mM KCl. 3.5 mM MgCl<sub>2</sub>). For plus template reactions, A1-PS and A1-CPG were suspended in 50 μl 1X PCR Buffer +1 μM A1C; G1-PS was suspended in 50 μl 1X PCR Buffer +1 μM G1C.</p>
    <p>Reactions were incubated at 95° C. for 1 min, then allowed to cool slowly to room temperature. A portion of each suspension was placed on a microscope slide. Each sample was excited with 488 nm light and a fluorescence image was captured on a CCD array using either a 518 nm or 583 nm interference filter. The images were analyzed by finding a peak pixel value on the 518 nm image and then finding the 583 nm value for the same pixel. Pixel values were corrected by subtracting the background readings observed with buffer. Table 5 gives the results of fluorescence measurements of the indicated probes.</p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup cols="4" colsep="0" rowsep="0" align="left"> <colspec colname="OFFSET" align="left" colwidth="35PT"> </colspec> <colspec colname="1" align="center" colwidth="49PT"> </colspec> <colspec colname="2" align="center" colwidth="49PT"> </colspec> <colspec colname="3" align="center" colwidth="84PT"> </colspec> <thead valign="bottom"> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td"> </td>
                <td namest="OFFSET" nameend="3" morerows="0" rowsep="1" valign="top" class="description-td" colspan="4">TABLE 5</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td"> </td>
                <td namest="OFFSET" nameend="3" morerows="0" rowsep="1" valign="top" align="center" class="description-td" colspan="4"> </td>
              </tr> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td"> </td>
                <td morerows="0" valign="top" class="description-td">518</td>
                <td morerows="0" valign="top" class="description-td">582</td>
                <td morerows="0" valign="top" class="description-td"> </td>
              </tr> </tbody> </tgroup> <tgroup cols="8" colsep="0" rowsep="0" align="left"> <colspec colname="1" align="left" colwidth="35PT"> </colspec> <colspec colname="2" align="center" colwidth="21PT"> </colspec> <colspec colname="3" align="center" colwidth="28PT"> </colspec> <colspec colname="4" align="center" colwidth="21PT"> </colspec> <colspec colname="5" align="center" colwidth="28PT"> </colspec> <colspec colname="6" align="center" colwidth="28PT"> </colspec> <colspec colname="7" align="center" colwidth="21PT"> </colspec> <colspec colname="8" align="center" colwidth="35PT"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td"> </td>
                <td morerows="0" valign="top" class="description-td">no</td>
                <td morerows="0" valign="top" class="description-td"> </td>
                <td morerows="0" valign="top" class="description-td">no</td>
                <td morerows="0" valign="top" class="description-td"> </td>
                <td morerows="0" valign="top" class="description-td"> </td>
                <td morerows="0" valign="top" class="description-td"> </td>
                <td morerows="0" valign="top" class="description-td"> </td>
              </tr> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td">PROBE</td>
                <td morerows="0" valign="top" class="description-td">temp.</td>
                <td morerows="0" valign="top" class="description-td">+temp.</td>
                <td morerows="0" valign="top" class="description-td">temp.</td>
                <td morerows="0" valign="top" class="description-td">+temp.</td>
                <td morerows="0" valign="top" class="description-td">RQ−</td>
                <td morerows="0" valign="top" class="description-td">RQ+</td>
                <td morerows="0" valign="top" class="description-td">ΔRQ</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="8" morerows="0" rowsep="1" valign="top" align="center" class="description-td" colspan="8"> </td>
              </tr> </tbody> </tgroup> <tgroup cols="8" colsep="0" rowsep="0" align="left"> <colspec colname="1" align="left" colwidth="35PT"> </colspec> <colspec colname="2" align="char" char="." colwidth="21PT"> </colspec> <colspec colname="3" align="char" char="." colwidth="28PT"> </colspec> <colspec colname="4" align="char" char="." colwidth="21PT"> </colspec> <colspec colname="5" align="char" char="." colwidth="28PT"> </colspec> <colspec colname="6" align="char" char="." colwidth="28PT"> </colspec> <colspec colname="7" align="char" char="." colwidth="21PT"> </colspec> <colspec colname="8" align="char" char="." colwidth="35PT"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td">A1-PS</td>
                <td morerows="0" valign="top" class="description-td">149</td>
                <td morerows="0" valign="top" class="description-td">354</td>
                <td morerows="0" valign="top" class="description-td">253</td>
                <td morerows="0" valign="top" class="description-td">379</td>
                <td morerows="0" valign="top" class="description-td">0.42</td>
                <td morerows="0" valign="top" class="description-td">0.67</td>
                <td morerows="0" valign="top" class="description-td">0.25</td>
              </tr> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td">A1-CPG</td>
                <td morerows="0" valign="top" class="description-td">494</td>
                <td morerows="0" valign="top" class="description-td">437</td>
                <td morerows="0" valign="top" class="description-td">1500</td>
                <td morerows="0" valign="top" class="description-td">616</td>
                <td morerows="0" valign="top" class="description-td">1.13</td>
                <td morerows="0" valign="top" class="description-td">2.44</td>
                <td morerows="0" valign="top" class="description-td">1.31</td>
              </tr> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td">G1-PS</td>
                <td morerows="0" valign="top" class="description-td">75</td>
                <td morerows="0" valign="top" class="description-td">166</td>
                <td morerows="0" valign="top" class="description-td">178</td>
                <td morerows="0" valign="top" class="description-td">245</td>
                <td morerows="0" valign="top" class="description-td">0.45</td>
                <td morerows="0" valign="top" class="description-td">0.73</td>
                <td morerows="0" valign="top" class="description-td">0.28</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="8" morerows="0" rowsep="1" valign="top" align="center" class="description-td" colspan="8"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <heading>5. Method For Monitoring PCR Amplification Usina Oligonucleotide Probe</heading> <p>All PCR amplifications were performed in a Perkin-Elmer Thermocycler 9600 using 50 μl reactions that contained 10 mM Tris-HCl (pH 8.3), 50 mM KCl, 200 μM dATP, 200 μM dCTP, 200 μM dGTP, 400 μM dUTP, 0.5 units AmpErase™ uracil N-glycolyase (Perkin-Elmer), and 1.25 units AmpliTaq™ (Perkin-Elmer). A 295 basepair segment of exon 3 of human β-actin gene (nucleotides 2141-2435 disclosed by Nakajima-Iijima. S., <i>Proc. Nati. Acad. Sci, USA </i>82: 6133-6137 (1985) was amplified using the AFP and ARP primers listed below. The amplification reactions contained 4 mM MgCl<sub>2</sub>, 20 ng human genomic DNA, 50 nM A1 or A3 probe, and 300 nM of each primer. Thermal regimen was 50° C. (2 min); 95° C. (10 min); 40 cycles of 95° C. (20 sec); 60° C. (1 min); and hold at 72° C. A 515 basepair segment was amplified from a plasmid that consists of a segment of k DNA (nucleotides 32, 220-32, 747) inserted into the Sma I site of vector pUC119. These reactions contained 3.5 mM MgCl<sub>2</sub>, 1 ng plasmid DNA, 50 nMP2 or P5 probe, 200 nM primer F 119, and 200 nM primer R119. The thermal regimen was 50° C. (2 min); 95° C. (10 min); 25 cycles of 95° C. (20 sec), 57° C. (1 min); and hold at 72° C.</p>
    <p>For each amplification reaction, 40 μl was transferred to an individual well of a white 96-well microtiter plate (Perkin-Elmer). Fluorescence was measured on a Perkin-Elmer TaqMan™ LS-50B System, which consists of a luminescence spectrometer with a plate reader assembly, a 485 nm excitation filter, and a 515 nm emission filter. Excitation was carried out at 488 nm using a 5 nm slit width. Emission was measured at 518 nm for 6-FAM (the reporter, or R Valve) and 582 nm for TAMRA (the quencher, or Q value) using a 10 nm slit width. In order to determine the increase in reporter emission that is due to cleavage of the probe during PCR. three normalizations are applied to the raw emission data. First, emission intensity of a buffer blank is subtracted for each wavelength. Second, emission intensity of the reporter is divided by the emission intensity of the quencher to give an RQ ratio for each reaction tube. This normalizes for well-to-well variation in probe concentration and fluorescence measurement. Finally, ΔRQ is calculated by subtracting the RQ value of the no template control (RQ<sup>−</sup>) from the RQ value for the complete reaction including a template (RQ<sup>+</sup>).</p>
    <p>Three pairs of probes were tested in PCR assays. For each pair, one probe has TAMRA attached to an internal nucleotide and the other has TAMRA attached to the 3′ end nucleotide. Results are shown in Table 6. For all three sets, the probe with the 3′ quencher exhibits a ΔRQ value that is considerable higher than for the probe with the internal quencher.</p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup cols="4" colsep="0" rowsep="0" align="left"> <colspec colname="OFFSET" align="left" colwidth="35PT"> </colspec> <colspec colname="1" align="center" colwidth="49PT"> </colspec> <colspec colname="2" align="center" colwidth="49PT"> </colspec> <colspec colname="3" align="center" colwidth="84PT"> </colspec> <thead valign="bottom"> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td"> </td>
                <td namest="OFFSET" nameend="3" morerows="0" rowsep="1" valign="top" class="description-td" colspan="4">TABLE 6</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td"> </td>
                <td namest="OFFSET" nameend="3" morerows="0" rowsep="1" valign="top" align="center" class="description-td" colspan="4"> </td>
              </tr> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td"> </td>
                <td morerows="0" valign="top" class="description-td">518</td>
                <td morerows="0" valign="top" class="description-td">582</td>
                <td morerows="0" valign="top" class="description-td"> </td>
              </tr> </tbody> </tgroup> <tgroup cols="8" colsep="0" rowsep="0" align="left"> <colspec colname="1" align="left" colwidth="35PT"> </colspec> <colspec colname="2" align="center" colwidth="21PT"> </colspec> <colspec colname="3" align="center" colwidth="28PT"> </colspec> <colspec colname="4" align="center" colwidth="21PT"> </colspec> <colspec colname="5" align="center" colwidth="28PT"> </colspec> <colspec colname="6" align="center" colwidth="28PT"> </colspec> <colspec colname="7" align="center" colwidth="21PT"> </colspec> <colspec colname="8" align="center" colwidth="35PT"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td"> </td>
                <td morerows="0" valign="top" class="description-td">no</td>
                <td morerows="0" valign="top" class="description-td"> </td>
                <td morerows="0" valign="top" class="description-td">no</td>
                <td morerows="0" valign="top" class="description-td"> </td>
                <td morerows="0" valign="top" class="description-td"> </td>
                <td morerows="0" valign="top" class="description-td"> </td>
                <td morerows="0" valign="top" class="description-td"> </td>
              </tr> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td">PROBE</td>
                <td morerows="0" valign="top" class="description-td">temp.</td>
                <td morerows="0" valign="top" class="description-td">+temp.</td>
                <td morerows="0" valign="top" class="description-td">temp.</td>
                <td morerows="0" valign="top" class="description-td">+temp.</td>
                <td morerows="0" valign="top" class="description-td">RQ−</td>
                <td morerows="0" valign="top" class="description-td">RQ+</td>
                <td morerows="0" valign="top" class="description-td">ΔRQ</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="8" morerows="0" rowsep="1" valign="top" align="center" class="description-td" colspan="8"> </td>
              </tr> </tbody> </tgroup> <tgroup cols="8" colsep="0" rowsep="0" align="left"> <colspec colname="1" align="left" colwidth="35PT"> </colspec> <colspec colname="2" align="char" char="." colwidth="21PT"> </colspec> <colspec colname="3" align="char" char="." colwidth="28PT"> </colspec> <colspec colname="4" align="char" char="." colwidth="21PT"> </colspec> <colspec colname="5" align="char" char="." colwidth="28PT"> </colspec> <colspec colname="6" align="char" char="." colwidth="28PT"> </colspec> <colspec colname="7" align="char" char="." colwidth="21PT"> </colspec> <colspec colname="8" align="char" char="." colwidth="35PT"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td">A3-6</td>
                <td morerows="0" valign="top" class="description-td">34.06</td>
                <td morerows="0" valign="top" class="description-td">50.1</td>
                <td morerows="0" valign="top" class="description-td">73.78</td>
                <td morerows="0" valign="top" class="description-td">70.8</td>
                <td morerows="0" valign="top" class="description-td">0.5</td>
                <td morerows="0" valign="top" class="description-td">0.71</td>
                <td morerows="0" valign="top" class="description-td">0.25</td>
              </tr> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td">A3-24</td>
                <td morerows="0" valign="top" class="description-td">58.85</td>
                <td morerows="0" valign="top" class="description-td">202</td>
                <td morerows="0" valign="top" class="description-td">69.66</td>
                <td morerows="0" valign="top" class="description-td">78.8</td>
                <td morerows="0" valign="top" class="description-td">0.8</td>
                <td morerows="0" valign="top" class="description-td">2.57</td>
                <td morerows="0" valign="top" class="description-td">1.72</td>
              </tr> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td">P2-7</td>
                <td morerows="0" valign="top" class="description-td">67.58</td>
                <td morerows="0" valign="top" class="description-td">341</td>
                <td morerows="0" valign="top" class="description-td">85.78</td>
                <td morerows="0" valign="top" class="description-td">87.9</td>
                <td morerows="0" valign="top" class="description-td">0.8</td>
                <td morerows="0" valign="top" class="description-td">3.89</td>
                <td morerows="0" valign="top" class="description-td">3.1</td>
              </tr> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td">P2-27</td>
                <td morerows="0" valign="top" class="description-td">124.6</td>
                <td morerows="0" valign="top" class="description-td">722</td>
                <td morerows="0" valign="top" class="description-td">152.6</td>
                <td morerows="0" valign="top" class="description-td">118</td>
                <td morerows="0" valign="top" class="description-td">0.8</td>
                <td morerows="0" valign="top" class="description-td">6.1</td>
                <td morerows="0" valign="top" class="description-td">5.28</td>
              </tr> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td">P5-10</td>
                <td morerows="0" valign="top" class="description-td">77.32</td>
                <td morerows="0" valign="top" class="description-td">156</td>
                <td morerows="0" valign="top" class="description-td">75.41</td>
                <td morerows="0" valign="top" class="description-td">67</td>
                <td morerows="0" valign="top" class="description-td">1</td>
                <td morerows="0" valign="top" class="description-td">2.33</td>
                <td morerows="0" valign="top" class="description-td">1.3</td>
              </tr> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td">P5-28</td>
                <td morerows="0" valign="top" class="description-td">73.23</td>
                <td morerows="0" valign="top" class="description-td">507</td>
                <td morerows="0" valign="top" class="description-td">106.6</td>
                <td morerows="0" valign="top" class="description-td">96.3</td>
                <td morerows="0" valign="top" class="description-td">0.7</td>
                <td morerows="0" valign="top" class="description-td">5.28</td>
                <td morerows="0" valign="top" class="description-td">4.59</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="8" morerows="0" rowsep="1" valign="top" align="center" class="description-td" colspan="8"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup cols="1" colsep="0" rowsep="0" align="left"> <colspec colname="1" align="center" colwidth="217PT"> </colspec> <thead valign="bottom"> <tr class="description-tr"> <td namest="1" nameend="1" morerows="0" rowsep="1" valign="top" class="description-td" colspan="1">TABLE 7</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" morerows="0" rowsep="1" valign="top" align="center" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td">Fluorescence In Single And Double-stranded States</td>
              </tr> </tbody> </tgroup> <tgroup cols="4" colsep="0" rowsep="0" align="left"> <colspec colname="OFFSET" align="left" colwidth="21PT"> </colspec> <colspec colname="1" align="center" colwidth="70PT"> </colspec> <colspec colname="2" align="center" colwidth="70PT"> </colspec> <colspec colname="3" align="center" colwidth="56PT"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td"> </td>
                <td morerows="0" valign="top" class="description-td">518</td>
                <td morerows="0" valign="top" class="description-td">582</td>
                <td morerows="0" valign="top" class="description-td">RQ</td>
              </tr> </tbody> </tgroup> <tgroup cols="7" colsep="0" rowsep="0" align="left"> <colspec colname="1" align="left" colwidth="28PT"> </colspec> <colspec colname="2" align="center" colwidth="28PT"> </colspec> <colspec colname="3" align="center" colwidth="35PT"> </colspec> <colspec colname="4" align="center" colwidth="28PT"> </colspec> <colspec colname="5" align="center" colwidth="42PT"> </colspec> <colspec colname="6" align="center" colwidth="21PT"> </colspec> <colspec colname="7" align="center" colwidth="35PT"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td">Probe</td>
                <td morerows="0" valign="top" class="description-td">ss</td>
                <td morerows="0" valign="top" class="description-td">ds</td>
                <td morerows="0" valign="top" class="description-td">ss</td>
                <td morerows="0" valign="top" class="description-td">ds</td>
                <td morerows="0" valign="top" class="description-td">ss</td>
                <td morerows="0" valign="top" class="description-td">ds</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="7" morerows="0" rowsep="1" valign="top" align="center" class="description-td" colspan="7"> </td>
              </tr> </tbody> </tgroup> <tgroup cols="7" colsep="0" rowsep="0" align="left"> <colspec colname="1" align="left" colwidth="28PT"> </colspec> <colspec colname="2" align="char" char="." colwidth="28PT"> </colspec> <colspec colname="3" align="char" char="." colwidth="35PT"> </colspec> <colspec colname="4" align="char" char="." colwidth="28PT"> </colspec> <colspec colname="5" align="char" char="." colwidth="42PT"> </colspec> <colspec colname="6" align="char" char="." colwidth="21PT"> </colspec> <colspec colname="7" align="char" char="." colwidth="35PT"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td">P2-7</td>
                <td morerows="0" valign="top" class="description-td">63.81</td>
                <td morerows="0" valign="top" class="description-td">84.07</td>
                <td morerows="0" valign="top" class="description-td">96.52</td>
                <td morerows="0" valign="top" class="description-td">142.97</td>
                <td morerows="0" valign="top" class="description-td">0.66</td>
                <td morerows="0" valign="top" class="description-td">0.59</td>
              </tr> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td">P2-27</td>
                <td morerows="0" valign="top" class="description-td">92.31</td>
                <td morerows="0" valign="top" class="description-td">557.53</td>
                <td morerows="0" valign="top" class="description-td">165.13</td>
                <td morerows="0" valign="top" class="description-td">89.47</td>
                <td morerows="0" valign="top" class="description-td">0.56</td>
                <td morerows="0" valign="top" class="description-td">6.23</td>
              </tr> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td">P5-10</td>
                <td morerows="0" valign="top" class="description-td">266.30</td>
                <td morerows="0" valign="top" class="description-td">366.37</td>
                <td morerows="0" valign="top" class="description-td">437.97</td>
                <td morerows="0" valign="top" class="description-td">491.00</td>
                <td morerows="0" valign="top" class="description-td">0.61</td>
                <td morerows="0" valign="top" class="description-td">0.75</td>
              </tr> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td">P5-28</td>
                <td morerows="0" valign="top" class="description-td">51.91</td>
                <td morerows="0" valign="top" class="description-td">782.80</td>
                <td morerows="0" valign="top" class="description-td">141.20</td>
                <td morerows="0" valign="top" class="description-td">154.07</td>
                <td morerows="0" valign="top" class="description-td">0.37</td>
                <td morerows="0" valign="top" class="description-td">5.08</td>
              </tr> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td">A1-7</td>
                <td morerows="0" valign="top" class="description-td">18.40</td>
                <td morerows="0" valign="top" class="description-td">60.45</td>
                <td morerows="0" valign="top" class="description-td">105.53</td>
                <td morerows="0" valign="top" class="description-td">218.83</td>
                <td morerows="0" valign="top" class="description-td">0.17</td>
                <td morerows="0" valign="top" class="description-td">0.28</td>
              </tr> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td">A1-26</td>
                <td morerows="0" valign="top" class="description-td">87.75</td>
                <td morerows="0" valign="top" class="description-td">734.37</td>
                <td morerows="0" valign="top" class="description-td">90.91</td>
                <td morerows="0" valign="top" class="description-td">118.57</td>
                <td morerows="0" valign="top" class="description-td">0.97</td>
                <td morerows="0" valign="top" class="description-td">6.19</td>
              </tr> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td">A3-6</td>
                <td morerows="0" valign="top" class="description-td">44.77</td>
                <td morerows="0" valign="top" class="description-td">104.80</td>
                <td morerows="0" valign="top" class="description-td">90.80</td>
                <td morerows="0" valign="top" class="description-td">177.87</td>
                <td morerows="0" valign="top" class="description-td">0.49</td>
                <td morerows="0" valign="top" class="description-td">0.59</td>
              </tr> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td">A3-24</td>
                <td morerows="0" valign="top" class="description-td">45.57</td>
                <td morerows="0" valign="top" class="description-td">857.57</td>
                <td morerows="0" valign="top" class="description-td">100.15</td>
                <td morerows="0" valign="top" class="description-td">191.43</td>
                <td morerows="0" valign="top" class="description-td">0.46</td>
                <td morerows="0" valign="top" class="description-td">3.47</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="7" morerows="0" rowsep="1" valign="top" align="center" class="description-td" colspan="7"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p>Table 7 gives the results of fluorescence measurements of the indicated probes in single and double-stranded states. For probes having reporter and quencher at opposite ends of the oligonucleotide, hybridization caused a dramatic increase in RQ.</p>
    <p>The foregoing description of a preferred embodiment of the invention has been presented for purposes of illustration and description. It is not intended to be exhaustive or to limit the invention to the precise forms disclosed. Obviously, many modifications and variations will be apparent to practitioners skilled in this art. It is intended that the scope of the invention be defined by the following claims and their equivalents.</p>
    <p>
      <sequence-cwu id="SEQLST-1"> <sequence-list-old-rules> <s-1> <s-1-iii>14</s-1-iii> </s-1> <s-2 seq-no="1"> <s-2-i> <s-2-i-a>24 nucleotides</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-vi> <s-2-vi-a>not provided</s-2-vi-a> </s-2-vi> <s-2-xi> 1
ACCCACAGGA  ACTGATCACC  ACTC                                          24</s-2-xi> </s-2> <s-2 seq-no="2"> <s-2-i> <s-2-i-a>26 nucleotides</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-vi> <s-2-vi-a>not provided</s-2-vi-a> </s-2-vi> <s-2-xi> 2
ATGTCGCGTT  CCGGCTGACG  TTCTGC                                        26</s-2-xi> </s-2> <s-2 seq-no="3"> <s-2-i> <s-2-i-a>27 nucleotides</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-vi> <s-2-vi-a>not provided</s-2-vi-a> </s-2-vi> <s-2-xi> 3
TCGCATTACT  GATCGTTGCC  AACCAGT                                       27</s-2-xi> </s-2> <s-2 seq-no="4"> <s-2-i> <s-2-i-a>31 nucleotides</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-vi> <s-2-vi-a>not provided</s-2-vi-a> </s-2-vi> <s-2-xi> 4
GTACTGGTTG  GCAACGATCA  GTAATGCGAT  G                                 31</s-2-xi> </s-2> <s-2 seq-no="5"> <s-2-i> <s-2-i-a>28 nucleotides</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-vi> <s-2-vi-a>not provided</s-2-vi-a> </s-2-vi> <s-2-xi> 5
CGGATTTGCT  GGTATCTATG  ACAAGGAT                                      28</s-2-xi> </s-2> <s-2 seq-no="6"> <s-2-i> <s-2-i-a>31 nucleotides</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-vi> <s-2-vi-a>not provided</s-2-vi-a> </s-2-vi> <s-2-xi> 6
TTCATCCTTG  TCATAGATAC  CAGCAAATCC  G                                 31</s-2-xi> </s-2> <s-2 seq-no="7"> <s-2-i> <s-2-i-a>25  nucleotides</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-vi> <s-2-vi-a>not provided</s-2-vi-a> </s-2-vi> <s-2-xi> 7
TCACCCACAC  TGTGCCCATC  TACGA                                         25</s-2-xi> </s-2> <s-2 seq-no="8"> <s-2-i> <s-2-i-a>25 nucleotides</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-vi> <s-2-vi-a>not provided</s-2-vi-a> </s-2-vi> <s-2-xi> 8
CAGCGGAACC  GCTCATTGCC  AATGG                                         25</s-2-xi> </s-2> <s-2 seq-no="9"> <s-2-i> <s-2-i-a>26 nucleotides</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-vi> <s-2-vi-a>not provided</s-2-vi-a> </s-2-vi> <s-2-xi> 9
ATGCCCTCCC  CCATGCCATC  CTGCGT                                        26</s-2-xi> </s-2> <s-2 seq-no="10"> <s-2-i> <s-2-i-a>30 nucleotides</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-vi> <s-2-vi-a>not provided</s-2-vi-a> </s-2-vi> <s-2-xi> 10
AGACGCAGGA  TGGCATGGGG  GAGGGCATAC                                    30</s-2-xi> </s-2> <s-2 seq-no="11"> <s-2-i> <s-2-i-a>24 nucleotides</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-vi> <s-2-vi-a>not provided</s-2-vi-a> </s-2-vi> <s-2-xi> 11
CGCCCTGGAC  TTCGAGCAAG  AGAT                                          24</s-2-xi> </s-2> <s-2 seq-no="12"> <s-2-i> <s-2-i-a>28 nucleotides</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-vi> <s-2-vi-a>not provided</s-2-vi-a> </s-2-vi> <s-2-xi> 12
CCATCTCTTG  CTCGAAGTCC  AGGGCGAC                                      28</s-2-xi> </s-2> <s-2 seq-no="13"> <s-2-i> <s-2-i-a>21 nucleotides</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-vi> <s-2-vi-a>not provided</s-2-vi-a> </s-2-vi> <s-2-xi> 13
CAAGCTTCCC  GTTCTCAGCC  T                                             21</s-2-xi> </s-2> <s-2 seq-no="14"> <s-2-i> <s-2-i-a>30 nucleotides</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-vi> <s-2-vi-a>not provided</s-2-vi-a> </s-2-vi> <s-2-xi> 14
ACCGTCAAGG  CTGAGAACGG  GAAGCTTGTC                                    30</s-2-xi> </s-2> </sequence-list-old-rules> </sequence-cwu> </p>
    </div></div></div><div class="patent-section patent-tabular-section"><a id="backward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Cited Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4220450">US4220450</a></td><td class="patent-data-table-td patent-date-value">Apr 5, 1978</td><td class="patent-data-table-td patent-date-value">Sep 2, 1980</td><td class="patent-data-table-td ">Syva Company</td><td class="patent-data-table-td ">Chemically induced fluorescence immunoassay</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4996143">US4996143</a></td><td class="patent-data-table-td patent-date-value">Apr 13, 1990</td><td class="patent-data-table-td patent-date-value">Feb 26, 1991</td><td class="patent-data-table-td ">Syngene, Inc.</td><td class="patent-data-table-td ">Fluorescent stokes shift probes for polynucleotide hybridization</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5210015">US5210015</a></td><td class="patent-data-table-td patent-date-value">Aug 6, 1990</td><td class="patent-data-table-td patent-date-value">May 11, 1993</td><td class="patent-data-table-td ">Hoffman-La Roche Inc.</td><td class="patent-data-table-td ">Polymerase chain reaction technique</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5332659">US5332659</a></td><td class="patent-data-table-td patent-date-value">Jan 15, 1993</td><td class="patent-data-table-td patent-date-value">Jul 26, 1994</td><td class="patent-data-table-td ">The United States Of America As Represented By The Secretary Of The Navy</td><td class="patent-data-table-td ">Light emission-or absorbance-based binding assays for polynucleic acids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5487972">US5487972</a></td><td class="patent-data-table-td patent-date-value">Jan 5, 1993</td><td class="patent-data-table-td patent-date-value">Jan 30, 1996</td><td class="patent-data-table-td ">Hoffmann-La Roche Inc.</td><td class="patent-data-table-td ">Nucleic acid detection by the 5&#39;-3&#39;exonuclease activity of polymerases acting on adjacently hybridized oligonucleotides</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5491063">US5491063</a></td><td class="patent-data-table-td patent-date-value">Sep 1, 1994</td><td class="patent-data-table-td patent-date-value">Feb 13, 1996</td><td class="patent-data-table-td ">Hoffmann-La Roche Inc.</td><td class="patent-data-table-td ">Methods for in-solution quenching of fluorescently labeled oligonucleotide probes</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5538848">US5538848</a></td><td class="patent-data-table-td patent-date-value">Nov 16, 1994</td><td class="patent-data-table-td patent-date-value">Jul 23, 1996</td><td class="patent-data-table-td ">Applied Biosystems Division, Perkin-Elmer Corp.</td><td class="patent-data-table-td ">Method for detecting nucleic acid amplification using self-quenching fluorescence probe</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5565554">US5565554</a></td><td class="patent-data-table-td patent-date-value">Jul 29, 1994</td><td class="patent-data-table-td patent-date-value">Oct 15, 1996</td><td class="patent-data-table-td ">The Regents Of The University Of California</td><td class="patent-data-table-td ">Dna-staining fluorecencent dyes, used in gel electrophoresis and for labels</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5607834">US5607834</a></td><td class="patent-data-table-td patent-date-value">Apr 10, 1995</td><td class="patent-data-table-td patent-date-value">Mar 4, 1997</td><td class="patent-data-table-td ">Maine Medical Center Research Institute</td><td class="patent-data-table-td ">Xanthene dyes for dna or rna analyzing</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5654419">US5654419</a></td><td class="patent-data-table-td patent-date-value">Feb 1, 1994</td><td class="patent-data-table-td patent-date-value">Aug 5, 1997</td><td class="patent-data-table-td ">The Regents Of The University Of California</td><td class="patent-data-table-td ">Fluorescent labels and their use in separations</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5688648">US5688648</a></td><td class="patent-data-table-td patent-date-value">Dec 19, 1995</td><td class="patent-data-table-td patent-date-value">Nov 18, 1997</td><td class="patent-data-table-td ">The Regents Of The University Of California</td><td class="patent-data-table-td ">Identificaiton and detection of nucleic acids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5723591">US5723591</a></td><td class="patent-data-table-td patent-date-value">Nov 15, 1995</td><td class="patent-data-table-td patent-date-value">Mar 3, 1998</td><td class="patent-data-table-td ">Perkin-Elmer Corporation</td><td class="patent-data-table-td ">Self-quenching fluorescence probe</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5804375">US5804375</a></td><td class="patent-data-table-td patent-date-value">Apr 25, 1995</td><td class="patent-data-table-td patent-date-value">Sep 8, 1998</td><td class="patent-data-table-td ">Roche Molecular Systems, Inc.</td><td class="patent-data-table-td ">Reaction mixtures for detection of target nucleic acids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5876930">US5876930</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Nov 15, 1995</td><td class="patent-data-table-td patent-date-value">Mar 2, 1999</td><td class="patent-data-table-td ">Perkin-Elmer Corporation</td><td class="patent-data-table-td ">Detecting polynucleotide in sample using oligonucleotide probe containing fluorescence and quencher molecules; probe has different conformations allowing different signal intensities when hybridized or unhybridized</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5925517">US5925517</a></td><td class="patent-data-table-td patent-date-value">May 12, 1995</td><td class="patent-data-table-td patent-date-value">Jul 20, 1999</td><td class="patent-data-table-td ">The Public Health Research Institute Of The City Of New York, Inc.</td><td class="patent-data-table-td ">Hybridization probe of single-stranded sequence complementary to target, flanked by oligonucleotide arms with interacting labels capable of forming stem duplex; hybridization with target alters level of signal at least ten percent</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6008373">US6008373</a></td><td class="patent-data-table-td patent-date-value">Jun 7, 1995</td><td class="patent-data-table-td patent-date-value">Dec 28, 1999</td><td class="patent-data-table-td ">Carnegie Mellon University</td><td class="patent-data-table-td ">Fluorescent labeling complexes with large stokes shift formed by coupling together cyanine and other fluorochromes capable of resonance energy transfer</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6030787">US6030787</a></td><td class="patent-data-table-td patent-date-value">Dec 7, 1998</td><td class="patent-data-table-td patent-date-value">Feb 29, 2000</td><td class="patent-data-table-td ">Pe Corporation</td><td class="patent-data-table-td ">Hybridization assay using self-quenching fluorescence probe</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="http://www.google.com/url?id=sLFVBAABERAJ&amp;q=http://worldwide.espacenet.com/publicationDetails/biblio%3FCC%3DAU%26NR%3D640089A%26KC%3DA%26FT%3DD&amp;usg=AFQjCNFh7uDJdexJnDOyQaGos5BfAVMwgQ">AU640089A</a></td><td class="patent-data-table-td patent-date-value"></td><td class="patent-data-table-td patent-date-value"></td><td class="patent-data-table-td "> </td><td class="patent-data-table-td citation-no-title">Title not available</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP0229943B1?cl=en">EP0229943B1</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Dec 1, 1986</td><td class="patent-data-table-td patent-date-value">Sep 4, 1991</td><td class="patent-data-table-td ">Molecular Biosystems, Inc.</td><td class="patent-data-table-td ">Fluorescent stokes shift probes for polynucleotide hybridization assays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP0232967A2?cl=en">EP0232967A2</a></td><td class="patent-data-table-td patent-date-value">Jan 9, 1987</td><td class="patent-data-table-td patent-date-value">Aug 19, 1987</td><td class="patent-data-table-td ">Amoco Corporation</td><td class="patent-data-table-td ">Competitive homogeneous Assay</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP0343955A2?cl=en">EP0343955A2</a></td><td class="patent-data-table-td patent-date-value">May 24, 1989</td><td class="patent-data-table-td patent-date-value">Nov 29, 1989</td><td class="patent-data-table-td ">Bioventures, Inc.</td><td class="patent-data-table-td ">Process for complex binding of targeted molecular species</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP0420102A2?cl=en">EP0420102A2</a></td><td class="patent-data-table-td patent-date-value">Sep 24, 1990</td><td class="patent-data-table-td patent-date-value">Apr 3, 1991</td><td class="patent-data-table-td ">Abbott Laboratories</td><td class="patent-data-table-td ">Stabilization of monoclonal antibody for use in fluorescent polarization techniques</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP0457213A2?cl=en">EP0457213A2</a></td><td class="patent-data-table-td patent-date-value">May 10, 1991</td><td class="patent-data-table-td patent-date-value">Nov 21, 1991</td><td class="patent-data-table-td ">Abbott Laboratories</td><td class="patent-data-table-td ">Barbiturate assay, tracers, immunogens, antibodies and kit</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP0523557A1?cl=en">EP0523557A1</a></td><td class="patent-data-table-td patent-date-value">Jul 10, 1992</td><td class="patent-data-table-td patent-date-value">Jan 20, 1993</td><td class="patent-data-table-td ">Boehringer Mannheim Gmbh</td><td class="patent-data-table-td ">Immobilisation of nucleic acids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP0601889A2?cl=en">EP0601889A2</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Dec 10, 1993</td><td class="patent-data-table-td patent-date-value">Jun 15, 1994</td><td class="patent-data-table-td ">Maine Medical Center Research Institute</td><td class="patent-data-table-td ">Nucleic acid probes</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="http://www.google.com/url?id=sLFVBAABERAJ&amp;q=http://worldwide.espacenet.com/publicationDetails/biblio%3FCC%3DJP%26NR%3DH05123195A%26KC%3DA%26FT%3DD&amp;usg=AFQjCNHygrazHtg2Ar726zOc-QghFnNBkg">JPH05123195A</a></td><td class="patent-data-table-td patent-date-value"></td><td class="patent-data-table-td patent-date-value"></td><td class="patent-data-table-td "> </td><td class="patent-data-table-td citation-no-title">Title not available</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1990003446A1?cl=en">WO1990003446A1</a></td><td class="patent-data-table-td patent-date-value">Sep 29, 1989</td><td class="patent-data-table-td patent-date-value">Apr 5, 1990</td><td class="patent-data-table-td ">New York Health Res Inst</td><td class="patent-data-table-td ">Nucleic acid probes containing improved molecular switch, and assays and kits incorporating same</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1992002638A1?cl=en">WO1992002638A1</a></td><td class="patent-data-table-td patent-date-value">Aug 6, 1991</td><td class="patent-data-table-td patent-date-value">Feb 7, 1992</td><td class="patent-data-table-td ">Cetus Corp</td><td class="patent-data-table-td ">Homogeneous assay system</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1993013224A1?cl=en">WO1993013224A1</a></td><td class="patent-data-table-td patent-date-value">Dec 22, 1992</td><td class="patent-data-table-td patent-date-value">Jul 8, 1993</td><td class="patent-data-table-td ">Chiron Corp</td><td class="patent-data-table-td ">Process for immobilizing nucleic acid probes on polystyrene surfaces</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1995003429A1?cl=en">WO1995003429A1</a></td><td class="patent-data-table-td patent-date-value">Jul 22, 1994</td><td class="patent-data-table-td patent-date-value">Feb 2, 1995</td><td class="patent-data-table-td ">Univ Utah Res Found</td><td class="patent-data-table-td ">Immunoassay procedure utilizing fluorogenic tracer antigens</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1995021266A1?cl=en">WO1995021266A1</a></td><td class="patent-data-table-td patent-date-value">Jan 30, 1995</td><td class="patent-data-table-td patent-date-value">Aug 10, 1995</td><td class="patent-data-table-td ">Univ California</td><td class="patent-data-table-td ">Probes labelled with energy transfer coupled dyes</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1996030540A2?cl=en">WO1996030540A2</a></td><td class="patent-data-table-td patent-date-value">Mar 20, 1996</td><td class="patent-data-table-td patent-date-value">Oct 3, 1996</td><td class="patent-data-table-td ">Univ California</td><td class="patent-data-table-td ">Substrates for beta-lactamase and uses thereof</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="npl-citations"></a><div class="patent-section-header"><span class="patent-section-title">Non-Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th colspan="3"class="patent-data-table-th">Reference</th></tr></thead><tr><td class="patent-data-table-td ">1</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Agrawal and Zamecnik, "<a href='http://scholar.google.com/scholar?q="Site+Specific+Functionalization+of+Olignucleotides+for+Attaching+Two+Different+Reporter+Groups%2C"'>Site Specific Functionalization of Olignucleotides for Attaching Two Different Reporter Groups,</a>" Nucl. Acids Res. 18(18):5419-5423 (1990).</td></tr><tr><td class="patent-data-table-td ">2</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Cardullo et al., "<a href='http://scholar.google.com/scholar?q="Detection+of+Nucleic+Acid+Hybridization+by+Nonradiative+Fluorescence+Resonance+Energy+Transfer%2C"'>Detection of Nucleic Acid Hybridization by Nonradiative Fluorescence Resonance Energy Transfer,</a>" Proc. Natl. Acad. Sci. USA 85:8790-8794 (1988).</td></tr><tr><td class="patent-data-table-td ">3</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Clegg et al., "<a href='http://scholar.google.com/scholar?q="Observing+the+Helical+Geometry+of+Double-Stranded+DNA+in+Solution+by+Fluorescence+Resonance+Energy+Transfer%2C"'>Observing the Helical Geometry of Double-Stranded DNA in Solution by Fluorescence Resonance Energy Transfer,</a>" Proc. Natl. Acad. Sci. USA 90:2994-2998 (1993).</td></tr><tr><td class="patent-data-table-td ">4</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Clegg, "<a href='http://scholar.google.com/scholar?q="Fluorescence+Resonance+Energy+Transfer+and+Nucleic+Acids%2C"'>Fluorescence Resonance Energy Transfer and Nucleic Acids,</a>" Methods of Enzymology 211:353-389 (1992).</td></tr><tr><td class="patent-data-table-td ">5</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Database WPI, Sect. Ch., Wk. 8608, Derwent Publ. Ltd., London, GB Jan. 1986, JP 5015439 (1993) "<a href='http://scholar.google.com/scholar?q="Determine+Polynucleotide+Single+Strand+Label+Luminous+Polynucleotide+Substance"'>Determine Polynucleotide Single Strand Label Luminous Polynucleotide Substance</a>".</td></tr><tr><td class="patent-data-table-td ">6</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Guo et al., "<a href='http://scholar.google.com/scholar?q="Direct+Fluores.+Analy.+of+Genetic+Polymor.+by+Hybrid.+with+oligonucl.+arrays+on+glass+supports%2C"'>Direct Fluores. Analy. of Genetic Polymor. by Hybrid. with oligonucl. arrays on glass supports,</a>" Nucl. Acids Res., 22(24) :5456-5465 (1994).</td></tr><tr><td class="patent-data-table-td ">7</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Heller et al., "<a href='http://scholar.google.com/scholar?q="Fluorescent+Energy+Transfer+Oligonucleotide+Probes%2C"'>Fluorescent Energy Transfer Oligonucleotide Probes,</a>" Abstract 248, Fed. Proc. 46:1968 (1987).</td></tr><tr><td class="patent-data-table-td ">8</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Higuchi et al., "<a href='http://scholar.google.com/scholar?q="Kinetic+PCR+Analysis%3A+Real-Time+Monitoring+of+DNA+Amplification+Reactions%2C"'>Kinetic PCR Analysis: Real-Time Monitoring of DNA Amplification Reactions,</a>" Biotechnology 11:1026-1030 (1993).</td></tr><tr><td class="patent-data-table-td ">9</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Higuchi et al., "<a href='http://scholar.google.com/scholar?q="Simultaneous+Amplification+and+Detection+of+Specific+DNA+Sequences%2C"'>Simultaneous Amplification and Detection of Specific DNA Sequences,</a>" Biotechnology 10:413-417 (1992).</td></tr><tr><td class="patent-data-table-td ">10</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Holland et al., "<a href='http://scholar.google.com/scholar?q="Detection+of+Specific+Polymerase+Chain+Reaction+Product+by+Utilizing+the+5%27-3%27+Exonuclease+Activity+of+Thermus+aquaticus+DNA+Polymerase%2C"'>Detection of Specific Polymerase Chain Reaction Product by Utilizing the 5'-3' Exonuclease Activity of Thermus aquaticus DNA Polymerase,</a>" Proc. Natl. Acad. Sci. USA 88:7276-7280 (1991).</td></tr><tr><td class="patent-data-table-td ">11</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Holland et al., "<a href='http://scholar.google.com/scholar?q="Detection+of+Specific+Polymerase+Chain+Reaction+Product+by+Utilizing+the+5%E2%80%B2-3%E2%80%B2+Exonuclease+Activity+of+Thermus+aquaticus+DNA+Polymerase%2C"'>Detection of Specific Polymerase Chain Reaction Product by Utilizing the 5′-3′ Exonuclease Activity of Thermus aquaticus DNA Polymerase,</a>" Proc. Natl. Acad. Sci. USA 88:7276-7280 (1991).</td></tr><tr><td class="patent-data-table-td ">12</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Ju et al., "<a href='http://scholar.google.com/scholar?q="Design+and+Synthesis+of+Fluorescence+Energy+Transfer+Dye-Labeled+Primers+and+their+Application+for+DNA+Sequencing+and+Analysis%2C"'>Design and Synthesis of Fluorescence Energy Transfer Dye-Labeled Primers and their Application for DNA Sequencing and Analysis,</a>" Anal. Biochem. 231:131-140 (1995).</td></tr><tr><td class="patent-data-table-td ">13</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Lee et al. "<a href='http://scholar.google.com/scholar?q="Allelic+discrimination+by+nick-translation+PCR+with+fluorogemic+probes"'>Allelic discrimination by nick-translation PCR with fluorogemic probes</a>" Nucleic acids research pp. 3761-3766, 1993.*</td></tr><tr><td class="patent-data-table-td ">14</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Lee et al., "<a href='http://scholar.google.com/scholar?q="DNA+sequencing+with+dye-labeled+terminators+and+T7+DNA+polymerase%3A+effect+of+dyes+and+dNTPs+on+incorporation+of+dye-terminators+and+probability+analysis+of+termination+fragments%2C"'>DNA sequencing with dye-labeled terminators and T7 DNA polymerase: effect of dyes and dNTPs on incorporation of dye-terminators and probability analysis of termination fragments,</a>" Nucl. Acids Res. 20(10):2471-2483 (1992).</td></tr><tr><td class="patent-data-table-td ">15</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Lindsey et al., "<a href='http://scholar.google.com/scholar?q="Visible+light-harvesting+in+covalently-linked+porphyrin-cyanine+dyes%2C"'>Visible light-harvesting in covalently-linked porphyrin-cyanine dyes,</a>" Tetrahedron 45(15) :4845-4866 (1989).</td></tr><tr><td class="patent-data-table-td ">16</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Livak et al. "<a href='http://scholar.google.com/scholar?q="Oligonucleotides+with+fluorescent+dyes+at+opposote+ends+provide+a+quenched+probe+system+useful+for+detecting+PCR+product+and+nucleic+acid+hybridization"'>Oligonucleotides with fluorescent dyes at opposote ends provide a quenched probe system useful for detecting PCR product and nucleic acid hybridization</a>" PCR Methods and Applications pp. 357-362, 1995.*</td></tr><tr><td class="patent-data-table-td ">17</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Mergny et al., "<a href='http://scholar.google.com/scholar?q="Fluorescent+Energy+Transfer+as+a+Probe+for+Nucleic+Acid+Structures+and+Sequences%2C"'>Fluorescent Energy Transfer as a Probe for Nucleic Acid Structures and Sequences,</a>" Nucl. Acids Res. 22(6) :920-928 (1994).</td></tr><tr><td class="patent-data-table-td ">18</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Ozaki et al., "<a href='http://scholar.google.com/scholar?q="The+estimation+of+distances+between+specific+backbone-labeled+sites+in+DNA+using+fluorescence+resonance+energy+transfer%2C"'>The estimation of distances between specific backbone-labeled sites in DNA using fluorescence resonance energy transfer,</a>" Nucl. Acids Res. 20(19):5205-5214 (1992).</td></tr><tr><td class="patent-data-table-td ">19</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Parkhurst &amp; Parkhurst, "<a href='http://scholar.google.com/scholar?q="Changes+in+the+end-to-end+distance+distribution+in+an+oligonucleotide+following+hybridization%2C"'>Changes in the end-to-end distance distribution in an oligonucleotide following hybridization,</a>" Time-Resolved Laser Spectroscopy in Biochemistry (Ladowicz, J.R. Ed.), Proc SPIE 2137:475-485 (1994).</td></tr><tr><td class="patent-data-table-td ">20</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Parkhurst &amp; Parkhurst, "<a href='http://scholar.google.com/scholar?q="Donor-Acceptor+Distance+Distributions+in+a+Double-Labeled+Fluorescent+Oligonucleotide+Both+as+a+Single+Strand+and+in+Duplexes%2C"'>Donor-Acceptor Distance Distributions in a Double-Labeled Fluorescent Oligonucleotide Both as a Single Strand and in Duplexes,</a>" Biochemistry 34:293-300 (1995).</td></tr><tr><td class="patent-data-table-td ">21</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Parkhurst &amp; Parkhurst, "<a href='http://scholar.google.com/scholar?q="Fluorescence+Studies+of+Oligonucleotide-DNA+Hybridization+in+Solution+and+of+Oligonucleotide+End+to+End+Distance+Distributions%2C"'>Fluorescence Studies of Oligonucleotide-DNA Hybridization in Solution and of Oligonucleotide End to End Distance Distributions,</a>" Abstracts, 11th International Congress on Photobiology, Kyoto, Japan, p. 258, Photobiology Association of Japan, Kyoto, Japan (1992).</td></tr><tr><td class="patent-data-table-td ">22</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Parkhurst &amp; Parkhurst, "<a href='http://scholar.google.com/scholar?q="Kinetic+Studies+of+Oligonucleotide-DNA+Hybridization+in+Solution+by+Fluorescence+Resonance+Energy+Transfer%2C"'>Kinetic Studies of Oligonucleotide-DNA Hybridization in Solution by Fluorescence Resonance Energy Transfer,</a>" Abstr. Biophys. J. 64:A266 (1993).</td></tr><tr><td class="patent-data-table-td ">23</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Parkhurst et al., "<a href='http://scholar.google.com/scholar?q="Kinetic+Studies+by+fluorescence+resonance+energy+transfer+employing+a+double-labeled+oligonucleotide%3A+hybridization+to+the+oligonucleotide+complement+and+to+single-stranded+DNA%2C"'>Kinetic Studies by fluorescence resonance energy transfer employing a double-labeled oligonucleotide: hybridization to the oligonucleotide complement and to single-stranded DNA,</a>" Biochemistry 34:285-292 (1995).</td></tr><tr><td class="patent-data-table-td ">24</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Search Report of WO 96/15270 by Livak, (PCT/US95/14882).</td></tr><tr><td class="patent-data-table-td ">25</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Stryer et al., "<a href='http://scholar.google.com/scholar?q="Energy+Transfer%3A+A+Spectroscopic+Ruler%2C"'>Energy Transfer: A Spectroscopic Ruler,</a>" Proc. Natl. Acad. Sci. USA 58:719-726 (1967).</td></tr><tr><td class="patent-data-table-td ">26</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Tyagi et al., Molecular Beacons: Probes that fluoresce upon hybridization, Nature Biotechnology 14:303-308 (1996).</td></tr><tr><td class="patent-data-table-td ">27</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Wu et al., "<a href='http://scholar.google.com/scholar?q="Resonance+Energy+Transfer%3A+Methods+and+Applications%2C"'>Resonance Energy Transfer: Methods and Applications,</a>" Anal. Biochem. 218:1-13 (1994).</td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="forward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Referenced by</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Citing Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6635428">US6635428</a></td><td class="patent-data-table-td patent-date-value">Dec 4, 2001</td><td class="patent-data-table-td patent-date-value">Oct 21, 2003</td><td class="patent-data-table-td ">Quest Diagnostics Investments Incorporated</td><td class="patent-data-table-td ">Oligonucleotides and methods for detecting hepatitis B viral nucleic acids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6884583">US6884583</a></td><td class="patent-data-table-td patent-date-value">Mar 21, 2002</td><td class="patent-data-table-td patent-date-value">Apr 26, 2005</td><td class="patent-data-table-td ">Applera Corporation</td><td class="patent-data-table-td ">Detecting preferential nucleotide sequences in sample; obtain nucleotide sequence sample, incubate with primers and probes, amplify nucleotide sequences, detect hybridization signal from probes</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6946245">US6946245</a></td><td class="patent-data-table-td patent-date-value">Dec 4, 2001</td><td class="patent-data-table-td patent-date-value">Sep 20, 2005</td><td class="patent-data-table-td ">Quest Diagnostics Investments Incorporated</td><td class="patent-data-table-td ">Oligonucleotides and methods for detecting hepatitis C viral nucleic acids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7132239">US7132239</a></td><td class="patent-data-table-td patent-date-value">Jun 4, 2003</td><td class="patent-data-table-td patent-date-value">Nov 7, 2006</td><td class="patent-data-table-td ">Applera Corporation</td><td class="patent-data-table-td ">Determination of a genotype of an amplification product at multiple allelic sites</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7250506">US7250506</a></td><td class="patent-data-table-td patent-date-value">Jan 16, 2004</td><td class="patent-data-table-td patent-date-value">Jul 31, 2007</td><td class="patent-data-table-td ">Quest Diagnostics Investments Incorporated</td><td class="patent-data-table-td ">Conjugated nucleotides to a detectable fluorescent dye; trancribing primers/probes for replication; qualitative and quantitative analysis of detecting viral nucleic acids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7262292">US7262292</a></td><td class="patent-data-table-td patent-date-value">Oct 20, 2003</td><td class="patent-data-table-td patent-date-value">Aug 28, 2007</td><td class="patent-data-table-td ">Quest Diagnostics Investments Incorporated</td><td class="patent-data-table-td ">Oligonucleotides and methods for detecting hepatitis B viral nucleic acids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7291459">US7291459</a></td><td class="patent-data-table-td patent-date-value">Dec 10, 2002</td><td class="patent-data-table-td patent-date-value">Nov 6, 2007</td><td class="patent-data-table-td ">University Of Alabama At Huntsville</td><td class="patent-data-table-td ">Nucleic acid detector and method of detecting targets within a sample</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7575863">US7575863</a></td><td class="patent-data-table-td patent-date-value">Sep 21, 2004</td><td class="patent-data-table-td patent-date-value">Aug 18, 2009</td><td class="patent-data-table-td ">Applied Biosystems, Llc</td><td class="patent-data-table-td ">Methods, compositions, and kits comprising linker probes for quantifying polynucleotides</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7601495">US7601495</a></td><td class="patent-data-table-td patent-date-value">May 31, 2005</td><td class="patent-data-table-td patent-date-value">Oct 13, 2009</td><td class="patent-data-table-td ">Applied Biosystems, Llc</td><td class="patent-data-table-td ">MicroRNA; peptide nucleic acids; locked nucleic acids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7601498">US7601498</a></td><td class="patent-data-table-td patent-date-value">Mar 16, 2006</td><td class="patent-data-table-td patent-date-value">Oct 13, 2009</td><td class="patent-data-table-td ">Biotium, Inc.</td><td class="patent-data-table-td ">Use of bridged nucleic acid fluorescent dyes, such as phenanthridinium dyes, cyanine dyes, or acridinium dyes each having an ammonium group and anion; bridging group can be -CH2-CONH-(CH2-CH2-O)x-(CH2)2-NHCO-CH2- ; low background fluorescence in the absence of nucleic acids; noncarcinogenic, stability</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7642055">US7642055</a></td><td class="patent-data-table-td patent-date-value">Sep 21, 2005</td><td class="patent-data-table-td patent-date-value">Jan 5, 2010</td><td class="patent-data-table-td ">Applied Biosystems, Llc</td><td class="patent-data-table-td ">Quantifying a small target polyncucleotide; a pair of linker probes can be employed in two different reactions to query a particular species of target polynucleotide</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7667024">US7667024</a></td><td class="patent-data-table-td patent-date-value">Nov 19, 2004</td><td class="patent-data-table-td patent-date-value">Feb 23, 2010</td><td class="patent-data-table-td ">Allelogic Biosciences Corp.</td><td class="patent-data-table-td ">Oligonucleotides labeled with a plurality of fluorophores</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7776567">US7776567</a></td><td class="patent-data-table-td patent-date-value">Mar 16, 2006</td><td class="patent-data-table-td patent-date-value">Aug 17, 2010</td><td class="patent-data-table-td ">Biotium, Inc.</td><td class="patent-data-table-td ">performing a nucleic acid amplification reaction by providing a reaction mixture comprising the target nucleic acid, reagents necessary for amplifying the target, and a fluorescent nucleic acid dye; used in DNA quantitation in real-time PCR</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7803943">US7803943</a></td><td class="patent-data-table-td patent-date-value">Jun 15, 2009</td><td class="patent-data-table-td patent-date-value">Sep 28, 2010</td><td class="patent-data-table-td ">Biotium, Inc.</td><td class="patent-data-table-td ">Use of bridged nucleic acid fluorescent dyes, such as phenanthridinium dyes, cyanine dyes, or acridinium dyes each having an ammonium group and anion; bridging group can be -CH2-CONH-(CH2-CH2-O)x-(CH2)2-NHCO-CH2- ; low background fluorescence in the absence of nucleic acids; noncarcinogenic, stability</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7824858">US7824858</a></td><td class="patent-data-table-td patent-date-value">Jan 23, 2006</td><td class="patent-data-table-td patent-date-value">Nov 2, 2010</td><td class="patent-data-table-td ">Quest Diagnostics Investments Incorporated</td><td class="patent-data-table-td ">determining the presence of Mycobacterium avium complex nucleic acids in a biological sample by detecting the mig gene of M. avium and the DT1 gene of M. intracellulare and amplifying them</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7955802">US7955802</a></td><td class="patent-data-table-td patent-date-value">Dec 13, 2007</td><td class="patent-data-table-td patent-date-value">Jun 7, 2011</td><td class="patent-data-table-td ">Luminex Corporation</td><td class="patent-data-table-td ">Systems and methods for multiplex analysis of PCR in real time</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8084212">US8084212</a></td><td class="patent-data-table-td patent-date-value">Sep 21, 2010</td><td class="patent-data-table-td patent-date-value">Dec 27, 2011</td><td class="patent-data-table-td ">Quest Diagnostics Investments Incorporated</td><td class="patent-data-table-td ">Assay for Mycobacterium avium/intracellulare nucleic acid</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8232050">US8232050</a></td><td class="patent-data-table-td patent-date-value">Aug 11, 2010</td><td class="patent-data-table-td patent-date-value">Jul 31, 2012</td><td class="patent-data-table-td ">Biotium, Inc.</td><td class="patent-data-table-td ">Methods of using dyes in association with nucleic acid staining or detection and associated technology</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8263330">US8263330</a></td><td class="patent-data-table-td patent-date-value">Mar 8, 2005</td><td class="patent-data-table-td patent-date-value">Sep 11, 2012</td><td class="patent-data-table-td ">Quest Diagnostics Investments Incorporated</td><td class="patent-data-table-td ">Detection of Mycobacterium tuberculosis complex nucleic acids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8288105">US8288105</a></td><td class="patent-data-table-td patent-date-value">May 25, 2011</td><td class="patent-data-table-td patent-date-value">Oct 16, 2012</td><td class="patent-data-table-td ">Luminex Corporation</td><td class="patent-data-table-td ">Systems and methods for multiplex analysis of PCR in real time</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8440405">US8440405</a></td><td class="patent-data-table-td patent-date-value">Feb 29, 2008</td><td class="patent-data-table-td patent-date-value">May 14, 2013</td><td class="patent-data-table-td ">360 Genomics Limited</td><td class="patent-data-table-td ">Methods for detecting variant nucleic acids by extension-dependent degradation of primers</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8530195">US8530195</a></td><td class="patent-data-table-td patent-date-value">Jun 22, 2010</td><td class="patent-data-table-td patent-date-value">Sep 10, 2013</td><td class="patent-data-table-td ">Biotium, Inc.</td><td class="patent-data-table-td ">Dyes and labeled molecules</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8633015">US8633015</a></td><td class="patent-data-table-td patent-date-value">Sep 24, 2010</td><td class="patent-data-table-td patent-date-value">Jan 21, 2014</td><td class="patent-data-table-td ">Bio-Rad Laboratories, Inc.</td><td class="patent-data-table-td ">Flow-based thermocycling system with thermoelectric cooler</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8658776">US8658776</a></td><td class="patent-data-table-td patent-date-value">May 26, 2011</td><td class="patent-data-table-td patent-date-value">Feb 25, 2014</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Synthesis of 2′,3′-dideoxynucleosides for automated DNA synthesis and pyrophosphorolysis activated polymerization</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8663920">US8663920</a></td><td class="patent-data-table-td patent-date-value">Jul 30, 2012</td><td class="patent-data-table-td patent-date-value">Mar 4, 2014</td><td class="patent-data-table-td ">Bio-Rad Laboratories, Inc.</td><td class="patent-data-table-td ">Library characterization by digital assay</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8709762">US8709762</a></td><td class="patent-data-table-td patent-date-value">Mar 2, 2011</td><td class="patent-data-table-td patent-date-value">Apr 29, 2014</td><td class="patent-data-table-td ">Bio-Rad Laboratories, Inc.</td><td class="patent-data-table-td ">System for hot-start amplification via a multiple emulsion</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8730479">US8730479</a></td><td class="patent-data-table-td patent-date-value">Dec 30, 2011</td><td class="patent-data-table-td patent-date-value">May 20, 2014</td><td class="patent-data-table-td ">Bio-Rad Laboratories, Inc.</td><td class="patent-data-table-td ">Detection system for droplet-based assays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8753814">US8753814</a></td><td class="patent-data-table-td patent-date-value">Jul 3, 2012</td><td class="patent-data-table-td patent-date-value">Jun 17, 2014</td><td class="patent-data-table-td ">Biotium, Inc.</td><td class="patent-data-table-td ">Methods of using dyes in association with nucleic acid staining or detection and associated technology</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP1829964A1?cl=en">EP1829964A1</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Nov 24, 2005</td><td class="patent-data-table-td patent-date-value">Sep 5, 2007</td><td class="patent-data-table-td ">Takeshi Yamamoto</td><td class="patent-data-table-td ">Method of examining gene sequence</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2428586A1?cl=en">EP2428586A1</a></td><td class="patent-data-table-td patent-date-value">Mar 17, 2006</td><td class="patent-data-table-td patent-date-value">Mar 14, 2012</td><td class="patent-data-table-td ">Biotium Inc.</td><td class="patent-data-table-td ">Dimeric and trimeric nucleic acid dyes, and associated systems and methods</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2484781A2?cl=en">EP2484781A2</a></td><td class="patent-data-table-td patent-date-value">Jul 31, 2007</td><td class="patent-data-table-td patent-date-value">Aug 8, 2012</td><td class="patent-data-table-td ">Applied Biosystems, LLC</td><td class="patent-data-table-td ">Detection of analytes and nucleic acids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2730662A1?cl=en">EP2730662A1</a></td><td class="patent-data-table-td patent-date-value">Nov 12, 2009</td><td class="patent-data-table-td patent-date-value">May 14, 2014</td><td class="patent-data-table-td ">Caris Life Sciences Luxembourg Holdings</td><td class="patent-data-table-td ">Methods and systems of using exosomes for determining phenotypes</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2008104791A1?cl=en">WO2008104791A1</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Feb 29, 2008</td><td class="patent-data-table-td patent-date-value">Sep 4, 2008</td><td class="patent-data-table-td ">Oxitec Ltd</td><td class="patent-data-table-td ">Methods for detecting nucleic acid sequences</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2008104794A2?cl=en">WO2008104794A2</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Feb 29, 2008</td><td class="patent-data-table-td patent-date-value">Sep 4, 2008</td><td class="patent-data-table-td ">Oxitec Ltd</td><td class="patent-data-table-td ">Nucleic acid detection</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2011150277A2?cl=en">WO2011150277A2</a></td><td class="patent-data-table-td patent-date-value">May 26, 2011</td><td class="patent-data-table-td patent-date-value">Dec 1, 2011</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Synthesis of 2&#39;, 3&#39;-dideoxynucleosides for automated dna synthesis and pyrophosphorolysis activated polymerization</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2012042516A2?cl=en">WO2012042516A2</a></td><td class="patent-data-table-td patent-date-value">Sep 25, 2011</td><td class="patent-data-table-td patent-date-value">Apr 5, 2012</td><td class="patent-data-table-td ">Mor Research Applications Ltd.</td><td class="patent-data-table-td ">Prognostic methods, compositions and kits for prediction of acute lymphoblastic leukemia (all) relapse</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2012099896A2?cl=en">WO2012099896A2</a></td><td class="patent-data-table-td patent-date-value">Jan 17, 2012</td><td class="patent-data-table-td patent-date-value">Jul 26, 2012</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Workflow for detection of ligands using nucleic acids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2012142003A2?cl=en">WO2012142003A2</a></td><td class="patent-data-table-td patent-date-value">Apr 9, 2012</td><td class="patent-data-table-td patent-date-value">Oct 18, 2012</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Chemical ligation</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2012170907A2?cl=en">WO2012170907A2</a></td><td class="patent-data-table-td patent-date-value">Jun 8, 2012</td><td class="patent-data-table-td patent-date-value">Dec 13, 2012</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Polymerization of nucleic acids using proteins having low isoelectric points</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2012170908A1?cl=en">WO2012170908A1</a></td><td class="patent-data-table-td patent-date-value">Jun 8, 2012</td><td class="patent-data-table-td patent-date-value">Dec 13, 2012</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Design and development of novel detergents for use in pcr systems</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2013030841A1?cl=en">WO2013030841A1</a></td><td class="patent-data-table-td patent-date-value">Sep 3, 2012</td><td class="patent-data-table-td patent-date-value">Mar 7, 2013</td><td class="patent-data-table-td ">Yissum Research Development Company Of The Hebrew Universitiy Of Jerusalem Ltd.</td><td class="patent-data-table-td ">Prognostic methods and compositions for predicting interferon treatment eficacy in a subject</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2013168162A1?cl=en">WO2013168162A1</a></td><td class="patent-data-table-td patent-date-value">May 9, 2013</td><td class="patent-data-table-td patent-date-value">Nov 14, 2013</td><td class="patent-data-table-td ">Yissum Research Development Company Of The Hebrew University Of Jerusalem Ltd.</td><td class="patent-data-table-td ">Clustered single nucleotide polymorphisms in the human acetylcholinesterase gene and uses thereof in diagnosis and therapy</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2013188839A1?cl=en">WO2013188839A1</a></td><td class="patent-data-table-td patent-date-value">Jun 14, 2013</td><td class="patent-data-table-td patent-date-value">Dec 19, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Novel compositions, methods and kits for real time polymerase chain reaction (pcr)</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2014071315A1?cl=en">WO2014071315A1</a></td><td class="patent-data-table-td patent-date-value">Nov 4, 2013</td><td class="patent-data-table-td patent-date-value">May 8, 2014</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Novel compositions, methods and kits for enhancing pcr specificity</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="classifications"></a><div class="patent-section-header"><span class="patent-section-title">Classifications</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th"> </th><th class="patent-data-table-th"> </th></tr></thead><tr><td class="patent-data-table-td ">U.S. Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=sLFVBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S091100">435/91.1</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=sLFVBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S091200">435/91.2</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=sLFVBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S025320">536/25.32</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=sLFVBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S025300">536/25.3</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=sLFVBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S024320">536/24.32</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=sLFVBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S005000">435/5</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=sLFVBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S026600">536/26.6</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=sLFVBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S024300">536/24.3</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=sLFVBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S006110">435/6.11</a></span></td></tr><tr><td class="patent-data-table-td ">International Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=sLFVBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0015090000">C12N15/09</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=sLFVBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C07H0021000000">C07H21/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=sLFVBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0015000000">C12N15/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=sLFVBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12Q0001680000">C12Q1/68</a></span></td></tr><tr><td class="patent-data-table-td ">Cooperative Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=sLFVBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12Q1/6818">C12Q1/6818</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=sLFVBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12Q1/6837">C12Q1/6837</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=sLFVBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12Q1/6834">C12Q1/6834</a></span></td></tr><tr><td class="patent-data-table-td ">European Classification</td><td class="patent-data-table-td "><span class="nested-value">C12Q1/68B2B</span>, <span class="nested-value">C12Q1/68B10</span></td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="legal-events"></a><div class="patent-section-header"><span class="patent-section-title">Legal Events</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Date</th><th class="patent-data-table-th">Code</th><th class="patent-data-table-th">Event</th><th class="patent-data-table-th">Description</th></tr></thead><tr><td class="patent-data-table-td patent-date-value">Apr 9, 2013</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20100528</span></div><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">APPLIED BIOSYSTEMS, INC., CALIFORNIA</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">LIEN RELEASE;ASSIGNOR:BANK OF AMERICA, N.A.;REEL/FRAME:030182/0677</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Mar 12, 2013</td><td class="patent-data-table-td ">B1</td><td class="patent-data-table-td ">Reexamination certificate first reexamination</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">THE PATENTABILITY OF CLAIMS 1-36 IS CONFIRMED.</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Jan 10, 2013</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">12</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Aug 28, 2012</td><td class="patent-data-table-td ">RR</td><td class="patent-data-table-td ">Request for reexamination filed</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20120718</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Feb 26, 2010</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">APPLIED BIOSYSTEMS INC.,CALIFORNIA</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">CHANGE OF NAME;ASSIGNOR:APPLERA CORPORATION;US-ASSIGNMENT DATABASE UPDATED:20100226;REEL/FRAME:23994/538</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20080701</span></div><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">APPLIED BIOSYSTEMS, LLC,CALIFORNIA</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">MERGER;ASSIGNOR:APPLIED BIOSYSTEMS INC.;US-ASSIGNMENT DATABASE UPDATED:20100302;REEL/FRAME:23994/587</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20081121</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">CHANGE OF NAME;ASSIGNOR:APPLERA CORPORATION;US-ASSIGNMENT DATABASE UPDATED:20100302;REEL/FRAME:23994/538</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">MERGER;ASSIGNOR:APPLIED BIOSYSTEMS INC.;US-ASSIGNMENT DATABASE UPDATED:20100226;REEL/FRAME:23985/801</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">MERGER;ASSIGNOR:APPLIED BIOSYSTEMS INC.;US-ASSIGNMENT DATABASE UPDATED:20100226;REEL/FRAME:23994/587</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">MERGER;ASSIGNOR:APPLIED BIOSYSTEMS INC.;US-ASSIGNMENT DATABASE UPDATED:20100316;REEL/FRAME:23985/801</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">CHANGE OF NAME;ASSIGNOR:APPLERA CORPORATION;US-ASSIGNMENT DATABASE UPDATED:20100329;REEL/FRAME:23994/538</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">CHANGE OF NAME;ASSIGNOR:APPLERA CORPORATION;US-ASSIGNMENT DATABASE UPDATED:20100413;REEL/FRAME:23994/538</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">MERGER;ASSIGNOR:APPLIED BIOSYSTEMS INC.;US-ASSIGNMENT DATABASE UPDATED:20100329;REEL/FRAME:23994/587</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">MERGER;ASSIGNOR:APPLIED BIOSYSTEMS INC.;US-ASSIGNMENT DATABASE UPDATED:20100330;REEL/FRAME:23985/801</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">MERGER;ASSIGNOR:APPLIED BIOSYSTEMS INC.;US-ASSIGNMENT DATABASE UPDATED:20100413;REEL/FRAME:23994/587</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">CHANGE OF NAME;ASSIGNOR:APPLERA CORPORATION;US-ASSIGNMENT DATABASE UPDATED:20100420;REEL/FRAME:23994/538</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">CHANGE OF NAME;ASSIGNOR:APPLERA CORPORATION;US-ASSIGNMENT DATABASE UPDATED:20100504;REEL/FRAME:23994/538</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">MERGER;ASSIGNOR:APPLIED BIOSYSTEMS INC.;US-ASSIGNMENT DATABASE UPDATED:20100420;REEL/FRAME:23994/587</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">MERGER;ASSIGNOR:APPLIED BIOSYSTEMS INC.;US-ASSIGNMENT DATABASE UPDATED:20100504;REEL/FRAME:23994/587</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">MERGER;ASSIGNOR:APPLIED BIOSYSTEMS INC.;REEL/FRAME:23985/801</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">MERGER;ASSIGNOR:APPLIED BIOSYSTEMS INC.;REEL/FRAME:23994/587</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">CHANGE OF NAME;ASSIGNOR:APPLERA CORPORATION;REEL/FRAME:23994/538</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">MERGER;ASSIGNOR:APPLIED BIOSYSTEMS INC.;REEL/FRAME:023994/0587</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">CHANGE OF NAME;ASSIGNOR:APPLERA CORPORATION;REEL/FRAME:023994/0538</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">MERGER;ASSIGNOR:APPLIED BIOSYSTEMS INC.;REEL/FRAME:023985/0801</span></div><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">APPLIED BIOSYSTEMS, LLC, CALIFORNIA</span></div><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">APPLIED BIOSYSTEMS INC., CALIFORNIA</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Jul 20, 2009</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">APPLIED BIOSYSTEMS INC., CALIFORNIA</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">MERGER;ASSIGNOR:ATOM ACQUISITION CORPORATION;REEL/FRAME:022973/0320</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20081121</span></div><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">APPLIED BIOSYSTEMS, LLC, CALIFORNIA</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">MERGER;ASSIGNOR:ATOM ACQUISITION, LLC AND APPLIED BIOSYSTEMS INC.;REEL/FRAME:022973/0327</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">CHANGE OF NAME;ASSIGNOR:APPLERA CORPORATION;REEL/FRAME:022973/0313</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20080630</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Jan 12, 2009</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">8</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Dec 5, 2008</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">BANK OF AMERICA, N.A, AS COLLATERAL AGENT, WASHING</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">SECURITY AGREEMENT;ASSIGNOR:APPLIED BIOSYSTEMS, LLC;REEL/FRAME:021976/0001</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20081121</span></div><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">BANK OF AMERICA, N.A, AS COLLATERAL AGENT,WASHINGT</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">SECURITY AGREEMENT;ASSIGNOR:APPLIED BIOSYSTEMS, LLC;US-ASSIGNMENT DATABASE UPDATED:20100216;REEL/FRAME:21976/1</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">SECURITY AGREEMENT;ASSIGNOR:APPLIED BIOSYSTEMS, LLC;US-ASSIGNMENT DATABASE UPDATED:20100209;REEL/FRAME:21976/1</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">SECURITY AGREEMENT;ASSIGNOR:APPLIED BIOSYSTEMS, LLC;US-ASSIGNMENT DATABASE UPDATED:20100225;REEL/FRAME:21976/1</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">SECURITY AGREEMENT;ASSIGNOR:APPLIED BIOSYSTEMS, LLC;US-ASSIGNMENT DATABASE UPDATED:20100302;REEL/FRAME:21976/1</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">SECURITY AGREEMENT;ASSIGNOR:APPLIED BIOSYSTEMS, LLC;US-ASSIGNMENT DATABASE UPDATED:20100304;REEL/FRAME:21976/1</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">SECURITY AGREEMENT;ASSIGNOR:APPLIED BIOSYSTEMS, LLC;US-ASSIGNMENT DATABASE UPDATED:20100309;REEL/FRAME:21976/1</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">SECURITY AGREEMENT;ASSIGNOR:APPLIED BIOSYSTEMS, LLC;US-ASSIGNMENT DATABASE UPDATED:20100316;REEL/FRAME:21976/1</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">SECURITY AGREEMENT;ASSIGNOR:APPLIED BIOSYSTEMS, LLC;US-ASSIGNMENT DATABASE UPDATED:20100323;REEL/FRAME:21976/1</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">SECURITY AGREEMENT;ASSIGNOR:APPLIED BIOSYSTEMS, LLC;US-ASSIGNMENT DATABASE UPDATED:20100329;REEL/FRAME:21976/1</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">SECURITY AGREEMENT;ASSIGNOR:APPLIED BIOSYSTEMS, LLC;US-ASSIGNMENT DATABASE UPDATED:20100330;REEL/FRAME:21976/1</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">SECURITY AGREEMENT;ASSIGNOR:APPLIED BIOSYSTEMS, LLC;US-ASSIGNMENT DATABASE UPDATED:20100406;REEL/FRAME:21976/1</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">SECURITY AGREEMENT;ASSIGNOR:APPLIED BIOSYSTEMS, LLC;US-ASSIGNMENT DATABASE UPDATED:20100413;REEL/FRAME:21976/1</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">SECURITY AGREEMENT;ASSIGNOR:APPLIED BIOSYSTEMS, LLC;US-ASSIGNMENT DATABASE UPDATED:20100415;REEL/FRAME:21976/1</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">SECURITY AGREEMENT;ASSIGNOR:APPLIED BIOSYSTEMS, LLC;US-ASSIGNMENT DATABASE UPDATED:20100420;REEL/FRAME:21976/1</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">SECURITY AGREEMENT;ASSIGNOR:APPLIED BIOSYSTEMS, LLC;US-ASSIGNMENT DATABASE UPDATED:20100427;REEL/FRAME:21976/1</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">SECURITY AGREEMENT;ASSIGNOR:APPLIED BIOSYSTEMS, LLC;US-ASSIGNMENT DATABASE UPDATED:20100504;REEL/FRAME:21976/1</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">SECURITY AGREEMENT;ASSIGNOR:APPLIED BIOSYSTEMS, LLC;US-ASSIGNMENT DATABASE UPDATED:20100511;REEL/FRAME:21976/1</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">SECURITY AGREEMENT;ASSIGNOR:APPLIED BIOSYSTEMS, LLC;US-ASSIGNMENT DATABASE UPDATED:20100513;REEL/FRAME:21976/1</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">SECURITY AGREEMENT;ASSIGNOR:APPLIED BIOSYSTEMS, LLC;REEL/FRAME:21976/1</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Jan 31, 2005</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">APPLERA CORPORATION, CALIFORNIA</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNOR:FLOOD, SUSAN J. A.;REEL/FRAME:015621/0426</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20050128</span></div><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">APPLERA CORPORATION 850 LINCOLN CENTRE DRIVE APPLI</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNOR:FLOOD, SUSAN J. A. /AR;REEL/FRAME:015621/0426</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Jan 10, 2005</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">4</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Dec 17, 2004</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">APPLERA CORPORATION, CALIFORNIA</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNORS:LIVAK, KENNETH J.;MARMARO, JEFFREY;MULLAH, KHAIRUZZAMAN BASHAR;REEL/FRAME:015469/0076;SIGNING DATES FROM 20041015 TO 20041027</span></div><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">APPLERA CORPORATION 850 LINCOLN CENTRE DRIVE APPLI</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNORS:LIVAK, KENNETH J. /AR;REEL/FRAME:015469/0076;SIGNING DATES FROM 20041015 TO 20041027</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Apr 9, 2002</td><td class="patent-data-table-td ">CC</td><td class="patent-data-table-td ">Certificate of correction</td><td class="patent-data-table-td "></td></tr><tr><td class="patent-data-table-td patent-date-value">Nov 8, 1999</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">PERKIN-ELMER CORPORATION, CALIFORNIA</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNORS:LIVAK, KENNETH J.;FLOOD, SUSAN J.A.;MARMARO, JEFFREY;ANDOTHERS;REEL/FRAME:010381/0652;SIGNING DATES FROM 19960307 TO 19960314</span></div><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">PERKIN-ELMER CORPORATION 850 LINCOLN CENTER DRIVE</span></div></td></tr></table><div class="patent-section-footer"></div></div><div class="modal-dialog" id="patent-images-lightbox"><div class="patent-lightbox-controls"><div class="patent-lightbox-rotate-controls"><div class="patent-lightbox-rotation-text">Rotate</div><div class="rotate-icon rotate-ccw-icon"></div><div class="rotate-icon rotate-cw-icon"></div></div><div class="patent-lightbox-index-counter"></div><a class="patent-lightbox-fullsize-link" target="_blank">Original Image</a><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_right.png" alt="Next page"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_left.png" alt="Previous page"width="21" height="21" /></div></div><div class="modal-dialog-content"><div class="patent-lightbox-image-holder"><div class="patent-lightbox-placeholder"></div></div></div></div><script>_OC_initPatentsAtb({image_not_available_html: " Image not available"});</script></div></div></div></td></tr></table><script>(function() {var href = window.location.href;if (href.indexOf('?') !== -1) {var parameters = href.split('?')[1].split('&');for (var i = 0; i < parameters.length; i++) {var param = parameters[i].split('=');if (param[0] == 'focus') {var elem = document.getElementById(param[1]);if (elem) {elem.focus();}}}}})();</script><script>_OC_addFlags({LockSrc:"/books/javascript/lock_8a2b04e7bf975d5171d8e4c0b6365c7a.js", Host:"http://www.google.com/", IsBooksRentalEnabled:1, IsWebstoreDisplayCaseEnabled:1, IsObfuscationEnabled:1, IsBrowsingHistoryEnabled:1, IsWebReaderSvgEnabled:0, IsGeoLayerEnabled:1, IsImageModeNotesEnabled:1, IsCopyMenuItemEnabled:1, IsGiftingEnabled:0, IsWebReaderUniversalPaginatorEnabled:0, IsOfflineBubbleEnabled:1, IsReaderEnabledForPlayRequests:1, IsFutureOnSaleVolumesEnabled:1, IsOfflineRestrictedCopyEnabled:1, IsBooksUnifiedLeftNavEnabled:1, IsRestrictedCopyEnabled:1, IsZipitFolderCollectionEnabled:1, IsEndOfSampleRecommendationsEnabled:1, IsRatingsOnBookcardsEnabled:1, IsAdsDisabled:0, IsIframePageDisplayEnabled:0, IsEmbeddedMediaEnabled:1, IsImageModeAnnotationsEnabled:1, IsMyLibraryGooglePlusEnabled:1, IsImagePageProviderEnabled:0, IsBookcardListPriceSmall:0, IsInternalUser:0, IsBooksShareButtonEnabled:0, IsPreOrdersEnabled:0, IsDisabledRandomBookshelves:0, WebstoreDisplayCasePosition:3});_OC_Run({"enable_p13n":false,"add_vol_to_collection_base_url":"http://www.google.com/patents?op=add\u0026sig=ACfU3U0WzxagdIjalV3v7mLaqaZmeU9kLw\u0026id=sLFVBAABERAJ","remove_vol_from_collection_base_url":"http://www.google.com/patents?op=remove\u0026sig=ACfU3U1yzx1ycQpKRlImQwHBac47yUYu6Q\u0026id=sLFVBAABERAJ","logged_in":false,"p13n_save_user_settings_url":"http://www.google.com/patents?op=edit_user_settings\u0026sig=ACfU3U12qACbISNu3Vk59pV6z6kC1xvzCg","is_cobrand":false,"sign_in_url":"https://www.google.com/accounts/Login?service=\u0026continue=http://www.google.com/patents%3Fhl%3Den\u0026hl=en","is_play_enabled":true}, {"volume_id":"","is_ebook":true,"volumeresult":{"has_flowing_text":false,"has_scanned_text":true,"can_download_pdf":false,"can_download_epub":false,"is_pdf_drm_enabled":false,"is_epub_drm_enabled":false,"download_pdf_url":"http://www.google.com/patents/download/Amplifying_using_polymerase_with_5_to_3.pdf?id=sLFVBAABERAJ\u0026output=pdf\u0026sig=ACfU3U0A3tvNbJiD2O4lub5vZuZs7msN4A"},"sample_url":"http://www.google.com/patents/reader?id=sLFVBAABERAJ\u0026printsec=frontcover\u0026output=reader\u0026source=gbs_atb_hover","is_browsable":true,"is_public_domain":true}, {});</script><div id="footer_table" style="font-size:83%;text-align:center;position:relative;top:20px;height:4.5em;margin-top:2em"><div style="margin-bottom:8px"><a href=http://www.google.com/><nobr>Google&nbsp;Home</nobr></a> - <a href=//www.google.com/patents/sitemap/><nobr>Sitemap</nobr></a> - <a href=http://www.google.com/googlebooks/uspto.html><nobr>USPTO Bulk Downloads</nobr></a> - <a href=/intl/en/privacy/><nobr>Privacy Policy</nobr></a> - <a href=/intl/en/policies/terms/><nobr>Terms of Service</nobr></a> - <a href=https://support.google.com/faqs/answer/2539193?hl=en><nobr>About Google Patents</nobr></a> - <a href="http://www.google.com/tools/feedback/intl/en/error.html" onclick="try{_OC_startFeedback({productId: '72792',locale: 'en'});return false;}catch(e){}"><nobr>Send Feedback</nobr></a></div><span>Data provided by IFI CLAIMS Patent Services</span><br><span >&copy;2012 Google</span></div> <script type="text/javascript">var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));</script><script type="text/javascript">var pageTracker = _gat._getTracker("UA-27188110-1");pageTracker._setCookiePath("/patents/");pageTracker._trackPageview();</script> </body></html>