<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN""http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html><head><title>Patent US6265196 - Preparing delivery vehicle containing nuclease having a dna binding domain ... - Google Patents</title><script>(function(){(function(){function e(a){this.t={};this.tick=function(a,c,b){var d=void 0!=b?b:(new Date).getTime();this.t[a]=[d,c];if(void 0==b)try{window.console.timeStamp("CSI/"+a)}catch(e){}};this.tick("start",null,a)}var a;window.performance&&(a=window.performance.timing);var f=a?new e(a.responseStart):new e;window.jstiming={Timer:e,load:f};if(a){var c=a.navigationStart,d=a.responseStart;0<c&&d>=c&&(window.jstiming.srt=d-c)}if(a){var b=window.jstiming.load;0<c&&d>=c&&(b.tick("_wtsrt",void 0,c),b.tick("wtsrt_",
"_wtsrt",d),b.tick("tbsd_","wtsrt_"))}try{a=null,window.chrome&&window.chrome.csi&&(a=Math.floor(window.chrome.csi().pageT),b&&0<c&&(b.tick("_tbnd",void 0,window.chrome.csi().startE),b.tick("tbnd_","_tbnd",c))),null==a&&window.gtbExternal&&(a=window.gtbExternal.pageT()),null==a&&window.external&&(a=window.external.pageT,b&&0<c&&(b.tick("_tbnd",void 0,window.external.startE),b.tick("tbnd_","_tbnd",c))),a&&(window.jstiming.pt=a)}catch(g){}})();})();
</script><link rel="stylesheet" href="/patents/css/_8a2b04e7bf975d5171d8e4c0b6365c7a/kl_intl_patents_bundle.css" type="text/css" /><script src="/books/javascript/atb_8a2b04e7bf975d5171d8e4c0b6365c7a__en.js"></script><script>function googleTranslateElementInit() {new google.translate.TranslateElement({pageLanguage: "en",gaTrack: true,gaId: "UA-27188110-1",multilanguagePage: true});}</script><script src="//translate.google.com/translate_a/element.js?cb=googleTranslateElementInit"></script><meta name="DC.type" content="Patent"><meta name="DC.title" content="Preparing delivery vehicle containing nuclease having a dna binding domain or gene encoding the nuclease, delivering into cells, then deactivating target"><meta name="DC.contributor" content="Srinivasan Chandrasegaran" scheme="inventor"><meta name="DC.contributor" content="Johns Hopkins University" scheme="assignee"><meta name="DC.date" content="1999-3-31" scheme="dateSubmitted"><meta name="DC.description" content="The present invention reveals a method for enzymatically inactivating a target DNA, a method for detecting conformational change in a nucleic acid, and a method for detecting the presence of a target DNA molecule. The method for enzymatically inactivating a target DNA involves preparing a plasmid, phage, virus, or any other delivery vehicle such as a liposome containing a gene encoding a nuclease. The delivery vehicle is then delivered into cells. The cells are induced to produce the nuclease and the target DNA is then enzymatically inactivated. Alternatively, the nuclease protein is delivered directly to cells and used to enzymatically inactivate the target DNA. The method for detecting conformational change in a nucleic acid requires contacting a nucleic acid with a hybrid restriction nuclease, determining whether the hybrid restriction nuclease has interacted with the nucleic acid, and detecting the conformational change in the nucleic acid. The method for detecting the presence of a target DNA entails contacting a target DNA with a fusion protein, comprising a DNA binding protein joined to a detection domain such as the constant region of an immunoglogulin heavy chain molecule."><meta name="DC.date" content="2001-7-24" scheme="issued"><meta name="DC.relation" content="US:5792640" scheme="references"><meta name="citation_patent_number" content="US:6265196"><meta name="citation_patent_application_number" content="US:09/281,792"><link rel="canonical" href="http://www.google.com/patents/US6265196"/><meta property="og:url" content="http://www.google.com/patents/US6265196"/><meta name="title" content="Patent US6265196 - Preparing delivery vehicle containing nuclease having a dna binding domain or gene encoding the nuclease, delivering into cells, then deactivating target"/><meta name="description" content="The present invention reveals a method for enzymatically inactivating a target DNA, a method for detecting conformational change in a nucleic acid, and a method for detecting the presence of a target DNA molecule. The method for enzymatically inactivating a target DNA involves preparing a plasmid, phage, virus, or any other delivery vehicle such as a liposome containing a gene encoding a nuclease. The delivery vehicle is then delivered into cells. The cells are induced to produce the nuclease and the target DNA is then enzymatically inactivated. Alternatively, the nuclease protein is delivered directly to cells and used to enzymatically inactivate the target DNA. The method for detecting conformational change in a nucleic acid requires contacting a nucleic acid with a hybrid restriction nuclease, determining whether the hybrid restriction nuclease has interacted with the nucleic acid, and detecting the conformational change in the nucleic acid. The method for detecting the presence of a target DNA entails contacting a target DNA with a fusion protein, comprising a DNA binding protein joined to a detection domain such as the constant region of an immunoglogulin heavy chain molecule."/><meta property="og:title" content="Patent US6265196 - Preparing delivery vehicle containing nuclease having a dna binding domain or gene encoding the nuclease, delivering into cells, then deactivating target"/><meta property="og:type" content="book"/><meta property="og:site_name" content="Google Books"/><meta property="og:image" content="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><link rel="image_src" href="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><script>(function(){try{var aa=function(a,b,c,d){d=d||{};d._sn=["cfg",b,c].join(".");window.gbar.logger.ml(a,d)};var g=window.gbar=window.gbar||{},l=window.gbar.i=window.gbar.i||{},m={},n;function _tvn(a,b){var c=parseInt(a,10);return isNaN(c)?b:c}function _tvf(a,b){var c=parseFloat(a);return isNaN(c)?b:c}function _tvv(a){return!!a}function p(a,b,c){(c||g)[a]=b}g.bv={n:_tvn("2",0),r:"",f:".67.",e:"0",m:_tvn("0",1)};
function q(a,b,c){var d="on"+b;if(a.addEventListener)a.addEventListener(b,c,!1);else if(a.attachEvent)a.attachEvent(d,c);else{var f=a[d];a[d]=function(){var a=f.apply(this,arguments),b=c.apply(this,arguments);return void 0==a?b:void 0==b?a:b&&a}}}var s=function(a){return function(){return g.bv.m==a}},ba=s(1),ca=s(2);p("sb",ba);p("kn",ca);l.a=_tvv;l.b=_tvf;l.c=_tvn;l.i=aa;var da=window.gbar.i.i;var t,u,v,w;function ea(a){v=a}function fa(a){var b;if(b=v&&window.encodeURIComponent)b=a.href,b=!b.match(/^http[s]?:\/\/accounts\.google\.[^/]*\/ClearSID/i)&&!b.match(/^http[s]?:\/\/[^/]*\/accounts\/ClearSID/i);if(b=b&&encodeURIComponent(v()))a.href=a.href.replace(/([?&]continue=)[^&]*/,"$1"+b)}function ga(a){window.gApplication&&(a.href=window.gApplication.getTabUrl(a.href))}
function ha(a){var b=document.forms[0].q,c=window.encodeURIComponent&&b&&b.value,b=b&&b.placeholder;c&&c!=b&&(a.href=a.href.replace(/([?&])q=[^&]*|$/,function(a,b){return(b||"&")+"q="+encodeURIComponent(c)}))}n=l.a("")?ga:ha;
function x(a,b,c,d,f,e){var h=document.getElementById(a);if(h){var k=h.style;k.left=d?"auto":b+"px";k.right=d?b+"px":"auto";k.top=c+"px";k.visibility=u?"hidden":"visible";f&&e?(k.width=f+"px",k.height=e+"px"):(x(t,b,c,d,h.offsetWidth,h.offsetHeight),u=u?"":a)}}
var y=[],ia=function(a,b){y.push(b)},ja=function(a){a=a||window.event;var b=a.target||a.srcElement;a.cancelBubble=!0;null==t&&(a=document.createElement(Array.every||window.createPopup?"iframe":"div"),a.frameBorder="0",t=a.id="gbs",a.src="javascript:''",b.parentNode.appendChild(a),q(document,"click",z));var c=b,b=0;"gb3"!=c.className&&(c=c.parentNode);a=c.getAttribute("aria-owns")||"gbi";var d=c.offsetWidth,f=20<c.offsetTop?46:24;document.getElementById("tphdr")&&(f-=3);var e=!1;do b+=c.offsetLeft||
0;while(c=c.offsetParent);var c=(document.documentElement.clientWidth||document.body.clientWidth)-b-d,h,d=document.body,k=document.defaultView;k&&k.getComputedStyle?(d=k.getComputedStyle(d,""))&&(h=d.direction):h=d.currentStyle?d.currentStyle.direction:d.style.direction;h="rtl"==h;if("gbi"==a){for(d=0;k=y[d++];)k();A(null,window.navExtra);h&&(b=c,e=!0)}else h||(b=c,e=!0);u!=a&&z();x(a,b,f,e)},z=function(){u&&x(u,0,0)},A=function(a,b){var c,d=document.getElementById("gbi"),f=a;f||(f=d.firstChild);
for(;b&&(c=b.pop());){var e=d,h=c,k=f;w||(w="gb2");e.insertBefore(h,k).className=w}},ka=function(a,b,c){if((b=document.getElementById(b))&&a){a.className="gb4";var d=document.createElement("span");d.appendChild(a);d.appendChild(document.createTextNode(" | "));d.id=c;b.appendChild(d)}},la=function(){return document.getElementById("gb_70")},ma=function(){return!!u};p("qs",n);p("setContinueCb",ea);p("pc",fa);p("tg",ja);p("close",z);p("addLink",ka);p("almm",A);p("si",la);p("adh",ia);p("op",ma);var B=function(){},C=function(){},F=function(a){var b=new Image,c=D;b.onerror=b.onload=b.onabort=function(){try{delete E[c]}catch(a){}};E[c]=b;b.src=a;D=c+1},E=[],D=0;p("logger",{il:C,ml:B,log:F});var G=window.gbar.logger;var H={},na={},I=[],oa=l.b("0.1",.1),pa=l.a("1",!0),qa=function(a,b){I.push([a,b])},ra=function(a,b){H[a]=b},sa=function(a){return a in H},J={},K=function(a,b){J[a]||(J[a]=[]);J[a].push(b)},ta=function(a){K("m",a)},L=function(a,b){var c=document.createElement("script");c.src=a;c.async=pa;Math.random()<oa&&(c.onerror=function(){c.onerror=null;B(Error("Bundle load failed: name="+(b||"UNK")+" url="+a))});(document.getElementById("xjsc")||document.getElementsByTagName("body")[0]||
document.getElementsByTagName("head")[0]).appendChild(c)},N=function(a){for(var b=0,c;(c=I[b])&&c[0]!=a;++b);!c||c[1].l||c[1].s||(c[1].s=!0,M(2,a),c[1].url&&L(c[1].url,a),c[1].libs&&m.d&&m.d(c[1].libs))},O=function(a){K("gc",a)},P=null,ua=function(a){P=a},M=function(a,b,c){if(P){a={t:a,b:b};if(c)for(var d in c)a[d]=c[d];try{P(a)}catch(f){}}};p("mdc",H);p("mdi",na);p("bnc",I);p("qGC",O);p("qm",ta);p("qd",J);p("lb",N);p("mcf",ra);p("bcf",qa);p("aq",K);p("mdd","");p("has",sa);
p("trh",ua);p("tev",M);var Q=l.b("0.1",.001),R=0;
function _mlToken(a,b){try{if(1>R){R++;var c,d=a,f=b||{},e=encodeURIComponent,h=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&jexpid=",e("17483"),"&srcpg=",e("prop=22"),"&jsr=",Math.round(1/Q),"&ogev=",e("-4_tU4bEB6TTsATIoIG4Ag"),"&ogf=",g.bv.f,"&ogrp=",e("1"),"&ogv=",e("1407723702.0"),"&oggv="+e("es_plusone_gc_20140723.0_p0"),"&ogd=",e("com"),"&ogc=",e("USA"),"&ogl=",e("en")];f._sn&&(f._sn="og."+
f._sn);for(var k in f)h.push("&"),h.push(e(k)),h.push("="),h.push(e(f[k]));h.push("&emsg=");h.push(e(d.name+":"+d.message));var r=h.join("");S(r)&&(r=r.substr(0,2E3));c=r;var Aa=window.gbar.logger._aem(a,c);F(Aa)}}catch(Na){}}var S=function(a){return 2E3<=a.length},va=function(a,b){return b};function T(a){B=a;p("_itl",S,G);p("_aem",va,G);p("ml",B,G);a={};H.er=a}l.a("")?T(function(a){throw a;}):l.a("1")&&Math.random()<Q&&T(_mlToken);I.push(["m",{url:"//ssl.gstatic.com/gb/js/scm_7385cc5883250b43a39405734c1bea59.js"}]);g.mcf("c",{});g.sg={c:""};if(l.a("1")){var wa=l.a("");I.push(["gc",{auto:wa,url:"//ssl.gstatic.com/gb/js/abc/gci_91f30755d6a6b787dcc2a4062e6e9824.js",libs:"googleapis.client:plusone:gapi.iframes"}]);var xa={version:"gci_91f30755d6a6b787dcc2a4062e6e9824.js",index:"",lang:"en"};H.gc=xa;var U=function(a){window.googleapis&&window.iframes?a&&a():(a&&O(a),N("gc"))};p("lGC",U);l.a("1")&&p("lPWF",U)};window.__PVT="";if(l.a("1")&&l.a("1")){var V=function(a){U(function(){K("pw",a);N("pw")})};p("lPW",V);I.push(["pw",{url:"//ssl.gstatic.com/gb/js/abc/pwm_45f73e4df07a0e388b0fa1f3d30e7280.js"}]);var W=[],ya=function(a){W[0]=a},za=function(a,b){var c=b||{};c._sn="pw";B(a,c)},Ba={signed:W,elog:za,base:"https://plusone.google.com/u/0",loadTime:(new Date).getTime()};H.pw=Ba;var X=function(a,b){for(var c=b.split("."),d=function(){var b=arguments;a(function(){for(var a=g,d=0,e=c.length-1;d<e;++d)a=a[c[d]];a[c[d]].apply(a,b)})},f=g,e=0,h=c.length-1;e<h;++e)f=
f[c[e]]=f[c[e]]||{};return f[c[e]]=d};X(V,"pw.clk");X(V,"pw.hvr");p("su",ya,g.pw)};function Ca(){function a(){for(var b;(b=e[h++])&&"m"!=b[0]&&!b[1].auto;);b&&(M(2,b[0]),b[1].url&&L(b[1].url,b[0]),b[1].libs&&m.d&&m.d(b[1].libs));h<e.length&&setTimeout(a,0)}function b(){0<f--?setTimeout(b,0):a()}var c=l.a("1"),d=l.a(""),f=3,e=I,h=0,k=window.gbarOnReady;if(k)try{k()}catch(r){da(r,"ml","or")}d?p("ldb",a):c?q(window,"load",b):b()}p("rdl",Ca);var Da={D:1,H:2,da:3,p:4,W:5,M:6,F:7,g:8,ha:9,U:10,L:11,T:12,S:13,N:14,Q:15,P:16,fa:17,w:18,O:19,ga:20,ea:21,u:22,G:23,ja:24,ka:25,ia:26,A:27,j:28,o:29,k:30,ca:31,Z:32,$:33,J:34,K:35,ba:36,aa:37,Y:38,B:39,R:40,v:41,X:42,V:43,h:48,C:49,I:500},Y=[1,2,3,4,5,6,9,10,11,13,14,28,29,30,34,35,37,38,39,40,41,42,43,48,49,500];var Z=l.b("0.001",1E-4),Ea=l.b("1",1),Fa=!1,Ga=!1;if(l.a("1")){var Ha=Math.random();Ha<=Z&&(Fa=!0);Ha<=Ea&&(Ga=!0)}var Ia=Da,$=null;function Ja(){var a=0,b=function(b,d){l.a(d)&&(a|=b)};b(1,"");b(2,"");b(4,"");b(8,"");return a}
function Ka(a,b){var c=Z,d=Fa,f;f=a;if(!$){$={};for(var e=0;e<Y.length;e++){var h=Y[e];$[h]=!0}}if(f=!!$[f])c=Ea,d=Ga;if(d){d=encodeURIComponent;g.rp?(f=g.rp(),f="-1"!=f?f:"1"):f="1";c=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&oge=",a,"&ogex=",d("17483"),"&ogev=",d("-4_tU4bEB6TTsATIoIG4Ag"),"&ogf=",g.bv.f,"&ogp=",d("22"),"&ogrp=",d(f),"&ogsr=",Math.round(1/c),"&ogv=",d("1407723702.0"),"&oggv="+
d("es_plusone_gc_20140723.0_p0"),"&ogd=",d("com"),"&ogl=",d("en"),"&ogc=",d("USA"),"&ogus=",Ja()];if(b){"ogw"in b&&(c.push("&ogw="+b.ogw),delete b.ogw);var k;f=b;e=[];for(k in f)0!=e.length&&e.push(","),e.push(La(k)),e.push("."),e.push(La(f[k]));k=e.join("");""!=k&&(c.push("&ogad="),c.push(d(k)))}F(c.join(""))}}function La(a){"number"==typeof a&&(a+="");return"string"==typeof a?a.replace(".","%2E").replace(",","%2C"):a}C=Ka;p("il",C,G);var Ma={};H.il=Ma;setTimeout(function(){C(Ia.g)},0);}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var b=window.gbar.i.i;var c=window.gbar;var f=function(d){try{var a=document.getElementById("gbom");a&&d.appendChild(a.cloneNode(!0))}catch(e){b(e,"omas","aomc")}};c.aomc=f;}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var a=window.gbar;a.mcf("pm",{p:""});}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{window.gbar.rdl();}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
if (window['_OC_timingAction']) {window['_OC_timingAction']('patents_refpage');}</script><style>#gbar,#guser{font-size:13px;padding-top:1px !important;}#gbar{float:left;height:22px}#guser{padding-bottom:7px !important;text-align:right}.gbh,.gbd{border-top:1px solid #c9d7f1;font-size:1px}.gbh{height:0;position:absolute;top:24px;width:100%}#gbs,.gbm{background:#fff;left:0;position:absolute;text-align:left;visibility:hidden;z-index:1000}.gbm{border:1px solid;border-color:#c9d7f1 #36c #36c #a2bae7;z-index:1001}.gb1{margin-right:.5em}.gb1,.gb3{zoom:1}.gb2{display:block;padding:.2em .5em}.gb2,.gb3{text-decoration:none !important;border-bottom:none}a.gb1,a.gb4{text-decoration:underline !important}a.gb1,a.gb2,a.gb3,a.gb4{color:#00c !important}.gbi .gb3,.gbi .gb2,.gbi .gb4{color:#dd8e27 !important}.gbf .gb3,.gbf .gb2,.gbf .gb4{color:#900 !important}a.gb2:hover{background:#36c;color:#fff !important}#gbar .gbz0l{color:#000 !important;cursor:default;font-weight:bold;text-decoration:none !important}
#gbar { padding:.3em .6em !important;}</style></head><body  topmargin="3" marginheight="3"><div id=gbar><nobr><a onclick=gbar.qs(this);gbar.logger.il(1,{t:1}); class=gb1 id=gb_1 href="https://www.google.com/search?sa=N&tab=tw">Search</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:2}); class=gb1 id=gb_2 href="http://www.google.com/search?hl=en&tbm=isch&source=og&sa=N&tab=ti">Images</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:8}); class=gb1 id=gb_8 href="http://maps.google.com/maps?hl=en&sa=N&tab=tl">Maps</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:78}); class=gb1 id=gb_78 href="https://play.google.com/?hl=en&sa=N&tab=t8">Play</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:36}); class=gb1 id=gb_36 href="http://www.youtube.com/results?sa=N&tab=t1">YouTube</a> <a onclick=gbar.logger.il(1,{t:5}); class=gb1 id=gb_5 href="http://news.google.com/nwshp?hl=en&tab=tn">News</a> <a onclick=gbar.logger.il(1,{t:23}); class=gb1 id=gb_23 href="https://mail.google.com/mail/?tab=tm">Gmail</a> <a onclick=gbar.logger.il(1,{t:25}); class=gb1 id=gb_25 href="https://drive.google.com/?tab=to">Drive</a> <a class=gb3 href="http://www.google.com/intl/en/options/" onclick="this.blur();gbar.tg(event);return !1" aria-haspopup=true><u>More</u> <small>&#9660;</small></a><div class=gbm id=gbi><a onclick=gbar.logger.il(1,{t:24}); class=gb2 id=gb_24 href="https://www.google.com/calendar?tab=tc">Calendar</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:51}); class=gb2 id=gb_51 href="http://translate.google.com/?hl=en&sa=N&tab=tT">Translate</a><a onclick=gbar.logger.il(1,{t:17}); class=gb2 id=gb_17 href="http://www.google.com/mobile/?hl=en&tab=tD">Mobile</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:10}); class=gb2 id=gb_10 href="http://www.google.com/search?hl=en&tbo=u&tbm=bks&source=og&sa=N&tab=tp">Books</a><a onclick=gbar.logger.il(1,{t:212}); class=gb2 id=gb_212 href="https://wallet.google.com/manage/?tab=ta">Wallet</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:6}); class=gb2 id=gb_6 href="http://www.google.com/search?hl=en&tbo=u&tbm=shop&source=og&sa=N&tab=tf">Shopping</a><a onclick=gbar.logger.il(1,{t:30}); class=gb2 id=gb_30 href="http://www.blogger.com/?tab=tj">Blogger</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:27}); class=gb2 id=gb_27 href="http://www.google.com/finance?sa=N&tab=te">Finance</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:31}); class=gb2 id=gb_31 href="https://plus.google.com/photos?sa=N&tab=tq">Photos</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:12}); class=gb2 id=gb_12 href="http://www.google.com/search?hl=en&tbo=u&tbm=vid&source=og&sa=N&tab=tv">Videos</a><div class=gb2><div class=gbd></div></div><a onclick=gbar.logger.il(1,{t:66}); href="http://www.google.com/intl/en/options/" class=gb2>Even more &raquo;</a></div></nobr></div><div id=guser width=100%><nobr><span id=gbn class=gbi></span><span id=gbf class=gbf></span><span id=gbe></span><a target=_top id=gb_70 href="https://www.google.com/accounts/Login?service=&continue=http://www.google.com/patents%3Fhl%3Den&hl=en" class=gb4>Sign in</a><div style="display: none"><div class=gbm id=gbd5 aria-owner=gbg5><div class=gbmc><ol id=gbom class=gbmcc></ol></div></div></div></nobr></div><div class=gbh style=left:0></div><div class=gbh style=right:0></div><div role="alert" style="position: absolute; left: 0; right: 0;"><a href="http://www.google.com/patents/us6265196?hl=en&amp;output=html_text" title="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."><img border="0" src="http://www.google.com/images/cleardot.gif"alt="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."></a></div><div id="guser"><nobr></nobr></div><div style="clear:both;"></div><div id="gb-top-search-box" class="gb-top-search-box-small gb-reset"><table><tr><td class="logo"><a href="http://www.google.com/patents" class="logo-link"><img class="logo-img" src="/intl/en/images/logos/google_logo_41.png" alt="Go to Google Books Home" height="41"/></a></td><td><form action="http://www.google.com/search" name="f" id="vheadf" method="get"><span id="hf"></span><input type="hidden" name="tbm" value="pts"/><input type="hidden" name="tbo" value="1"/><input type="hidden" name="hl" value="en"/><table><tr><td><div class="inputs"><table><tr><td><div class="text-input"><input type="text" name="q" id="vheadq" class="text" maxlength="2048" size="31" value="" title="Search Patents" accesskey="s" autocomplete="off"/><script>window._OC_autoDir &&window._OC_autoDir('vheadq', 'tia-vheadq');</script></div></td><td><div class="submit-input"><input name="btnG" class="submit" type="submit" value=""/></div></td></tr></table></div></td><td class="col-ext-links"><div class="ext-links"><a href="http://www.google.com/advanced_patent_search">&lt;nobr&gt;Advanced Patent Search&lt;/nobr&gt;</a></div></td></tr></table></form></td></tr></table></div><div class="kd-appbar"><h2 class="kd-appname"><a href="/patents">Patents</a></h2><div class="kd-buttonbar left" id="left-toolbar-buttons"><a id="appbar-write-review-link" href=""></a><a id="appbar-view-print-sample-link" href=""></a><a id="appbar-view-ebook-sample-link" href=""></a><a id="appbar-patents-prior-art-finder-link" href="https://www.google.com/patents/related/US6265196"></a><a id="appbar-patents-discuss-this-link" href="http://www.google.com/url?id=DZhVBAABERAJ&amp;q=http://patents.stackexchange.com/redirect/google-patents%3Fpatent%3DUS6265196&amp;usg=AFQjCNHMsYxyrrC5O8_J_cZXXndbZ9Z60w" data-is-grant="true"></a><a id="appbar-read-patent-link" href="//docs.google.com/viewer?url=patentimages.storage.googleapis.com/pdfs/US6265196.pdf"></a><a id="appbar-download-pdf-link" href="//patentimages.storage.googleapis.com/pdfs/US6265196.pdf"></a></div><div class="kd-buttonbar right" id="right-toolbar-buttons"></div></div><div id="books-microdata" itemscope=""itemtype="http://schema.org/Book"itemid="http://www.google.com/patents/US6265196" style="display:none"><span itemprop="description">The present invention reveals a method for enzymatically inactivating a target DNA, a method for detecting conformational change in a nucleic acid, and a method for detecting the presence of a target DNA molecule. The method for enzymatically inactivating a target DNA involves preparing a plasmid, phage,...</span><span itemprop="url">http://www.google.com/patents/US6265196?utm_source=gb-gplus-share</span><span class="main-title" itemprop="name">Patent US6265196 - Preparing delivery vehicle containing nuclease having a dna binding domain or gene encoding the nuclease, delivering into cells, then deactivating target</span><img itemprop="image" src="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"alt="Patent US6265196 - Preparing delivery vehicle containing nuclease having a dna binding domain or gene encoding the nuclease, delivering into cells, then deactivating target" title="Patent US6265196 - Preparing delivery vehicle containing nuclease having a dna binding domain or gene encoding the nuclease, delivering into cells, then deactivating target"></div><div style="display: none"><ol id="ofe-gear-menu-contents" class="gbmcc"><li class="gbe gbmtc"><a class="gbmt goog-menuitem-content" id="" href="http://www.google.com/advanced_patent_search">Advanced Patent Search</a></li></ol></div><table id="viewport_table" cellpadding="0" style="clear:both" cellspacing="0"><tr><td id="viewport_td"><div class=vertical_module_list_row><div id=intl_patents class=about_content><div id=intl_patents_v><table class="patent-bibdata"><tr><td class="patent-bibdata-heading">Publication number</td><td class="single-patent-bibdata">US6265196 B1</td></tr><tr><td class="patent-bibdata-heading">Publication type</td><td class="single-patent-bibdata">Grant</td></tr><tr><td class="patent-bibdata-heading">Application number</td><td class="single-patent-bibdata">US 09/281,792</td></tr><tr><td class="patent-bibdata-heading">Publication date</td><td class="single-patent-bibdata">Jul 24, 2001</td></tr><tr><td class="patent-bibdata-heading">Filing date</td><td class="single-patent-bibdata">Mar 31, 1999</td></tr><tr><td class="patent-bibdata-heading">Priority date<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The priority date is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the date listed."></span></td><td class="single-patent-bibdata">May 7, 1996</td></tr><tr><td class="patent-bibdata-heading">Fee status<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The fee status is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the status or dates listed."></span></td><td class="single-patent-bibdata">Paid</td></tr><tr class="patent-bibdata-list-row alternate-patent-number"><td class="patent-bibdata-heading">Publication number</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value">09281792, </span><span class="patent-bibdata-value">281792, </span><span class="patent-bibdata-value">US 6265196 B1, </span><span class="patent-bibdata-value">US 6265196B1, </span><span class="patent-bibdata-value">US-B1-6265196, </span><span class="patent-bibdata-value">US6265196 B1, </span><span class="patent-bibdata-value">US6265196B1</span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Inventors</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Srinivasan+Chandrasegaran%22">Srinivasan Chandrasegaran</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Original Assignee</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=inassignee:%22Johns+Hopkins+University%22">Johns Hopkins University</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Export Citation</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/US6265196.bibtex">BiBTeX</a>, </span><span class="patent-bibdata-value"><a href="/patents/US6265196.enw">EndNote</a>, </span><span class="patent-bibdata-value"><a href="/patents/US6265196.ris">RefMan</a></span></span></td></tr><tr class="patent-internal-links"><td colspan=2><span class="patent-bibdata-value"><a href="#backward-citations">Patent Citations</a> (1),</span> <span class="patent-bibdata-value"><a href="#forward-citations">Referenced by</a> (29),</span> <span class="patent-bibdata-value"><a href="#classifications">Classifications</a> (10),</span> <span class="patent-bibdata-value"><a href="#legal-events">Legal Events</a> (8)</span> </td></tr><tr><td colspan=2 class="patent-bibdata-external-link-spacer-top"></td></tr><tr class="patent-bibdata-external-link-spacer-bottom"></tr><tr><td colspan=2><span class="patent-bibdata-heading">External Links:&nbsp;</span><span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=DZhVBAABERAJ&q=http://patft.uspto.gov/netacgi/nph-Parser%3FSect2%3DPTO1%26Sect2%3DHITOFF%26p%3D1%26u%3D/netahtml/PTO/search-bool.html%26r%3D1%26f%3DG%26l%3D50%26d%3DPALL%26RefSrch%3Dyes%26Query%3DPN/6265196&usg=AFQjCNGD8mWXrXzWHHN8bU0bM1wA9NAaQw">USPTO</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=DZhVBAABERAJ&q=http://assignments.uspto.gov/assignments/q%3Fdb%3Dpat%26pat%3D6265196&usg=AFQjCNFxH03_ujN9cu2LN6Xs_S53QXFt7g">USPTO Assignment</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=DZhVBAABERAJ&q=http://worldwide.espacenet.com/publicationDetails/biblio%3FCC%3DUS%26NR%3D6265196B1%26KC%3DB1%26FT%3DD&usg=AFQjCNFdfOsLtuO5eclaP4c5xf87ajOGRg">Espacenet</a></span></span></td></tr><tr class="patent-bibdata-group-spacer"></tr></table><div class="number-and-title"><span class="patent-title"><invention-title mxw-id="PT129044068" lang="EN" load-source="ifi">Preparing delivery vehicle containing nuclease having a dna binding domain or gene encoding the nuclease, delivering into cells, then deactivating target</invention-title></span><br><span class="patent-number">US 6265196 B1</span></div><div class="patent-section patent-abstract-section"><div class="patent-section-header"><span class="patent-section-title">Abstract</span></div><div class="patent-text"><abstract mxw-id="PA72589651" lang="EN" load-source="patent-office"> <div class="abstract">The present invention reveals a method for enzymatically inactivating a target DNA, a method for detecting conformational change in a nucleic acid, and a method for detecting the presence of a target DNA molecule. The method for enzymatically inactivating a target DNA involves preparing a plasmid, phage, virus, or any other delivery vehicle such as a liposome containing a gene encoding a nuclease. The delivery vehicle is then delivered into cells. The cells are induced to produce the nuclease and the target DNA is then enzymatically inactivated. Alternatively, the nuclease protein is delivered directly to cells and used to enzymatically inactivate the target DNA. The method for detecting conformational change in a nucleic acid requires contacting a nucleic acid with a hybrid restriction nuclease, determining whether the hybrid restriction nuclease has interacted with the nucleic acid, and detecting the conformational change in the nucleic acid. The method for detecting the presence of a target DNA entails contacting a target DNA with a fusion protein, comprising a DNA binding protein joined to a detection domain such as the constant region of an immunoglogulin heavy chain molecule.</div>
  </abstract></div></div><div class="patent-section patent-drawings-section"><div class="patent-section-header"><span class="patent-section-title">Images<span class="patent-section-count">(22)</span></span></div><div class="patent-drawings-body"><div class="patent-drawings-carousel"><div class="drawings"><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6265196-0.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6265196-0.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6265196-1.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6265196-1.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6265196-2.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6265196-2.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6265196-3.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6265196-3.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6265196-4.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6265196-4.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6265196-5.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6265196-5.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6265196-6.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6265196-6.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6265196-7.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6265196-7.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6265196-8.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6265196-8.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6265196-9.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6265196-9.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6265196-10.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6265196-10.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6265196-11.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6265196-11.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6265196-12.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6265196-12.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6265196-13.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6265196-13.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6265196-14.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6265196-14.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6265196-15.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6265196-15.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6265196-16.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6265196-16.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6265196-17.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6265196-17.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6265196-18.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6265196-18.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6265196-19.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6265196-19.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6265196-20.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6265196-20.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6265196-21.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6265196-21.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div></div></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img" alt="Previous page"src="/googlebooks/images/kennedy/page_left.png"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img" alt="Next page"src="/googlebooks/images/kennedy/page_right.png"width="21" height="21" /></div></div></div><div class="patent-post-drawings"></div><div class="patent-section patent-claims-section"><div class="patent-section-header"><span class="patent-section-title">Claims<span class="patent-section-count">(8)</span></span></div><div class="patent-text"><div mxw-id="PCLM28675082" lang="EN" load-source="patent-office" class="claims">
    <claim-statement>What is claimed is: </claim-statement> <div class="claim"> <div num="1" id="US-6265196-B1-CLM-00001" class="claim">
      <div class="claim-text">1. A method for enzymatically inactivating a target DNA, comprising the steps of:</div>
      <div class="claim-text">a) preparing a delivery vehicle containing a nuclease or a gene encoding a nuclease, </div>
      <div class="claim-text">i) wherein said nuclease specifically recognizes and enzymatically inactivates said target DNA and </div>
      <div class="claim-text">ii) wherein said nuclease encodes a DNA binding domain that specifically binds to a DNA sequence; </div>
      <div class="claim-text">b) delivering the delivery vehicle containing a nuclease or a gene encoding a nuclease into cells; and </div>
      <div class="claim-text">c) enzymatically inactivating said target DNA. </div>
    </div>
    </div> <div class="claim-dependent"> <div num="2" id="US-6265196-B1-CLM-00002" class="claim">
      <div class="claim-text">2. The method of claim <b>1</b>, wherein said nuclease encodes a DNA binding protein selected from the group consisting of naturally occurring DNA-binding proteins and engineered or designed DNA-binding proteins.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="3" id="US-6265196-B1-CLM-00003" class="claim">
      <div class="claim-text">3. The method of claim <b>2</b>, wherein said naturally occurring DNA-binding proteins and engineered or designed DNA-binding proteins are selected from the group consisting of transcription repressor proteins, transcription activator proteins, telomere binding proteins, and DNA origin binding proteins.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="4" id="US-6265196-B1-CLM-00004" class="claim">
      <div class="claim-text">4. The method of claim <b>3</b>, wherein said DNA binding protein is an ori binding protein.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="5" id="US-6265196-B1-CLM-00005" class="claim">
      <div class="claim-text">5. The method of claim <b>3</b>, wherein said DNA binding protein is a telomere binding protein.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="6" id="US-6265196-B1-CLM-00006" class="claim">
      <div class="claim-text">6. The method of claim <b>4</b>, wherein said ori binding protein is selected from the group consisting of SV40 T antigen, HSV-I UL9 gene product, Varicella-Zoster gene 51 product, human herpes 6B CH6R gene product, Epstein-Barr virus EBNA-1 gene product, human papilloma virus E1 gene product, and human papilloma virus E2 gene product.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="7" id="US-6265196-B1-CLM-00007" class="claim">
      <div class="claim-text">7. The method of claim <b>1</b>, wherein said target DNA is selected from the group consisting of human immunodeficiency virus, hepatitis B, herpesviruses, polyoma viruses, and papilloma viruses.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="8" id="US-6265196-B1-CLM-00008" class="claim">
      <div class="claim-text">8. The method of claim <b>1</b>, wherein said target DNA is a gene selected from the group consisting of human, animal, viral, and bacterial genes.</div>
    </div>
  </div> </div></div></div><div class="patent-section patent-description-section"><div class="patent-section-header"><span class="patent-section-title">Description</span></div><div class="patent-text"><div mxw-id="PDES54606683" lang="EN" load-source="patent-office" class="description">
    <p>This is a division of application Ser. No. 08/647,449, filed May 7. 1996.</p>
    <p>This patent application was supported in part by grant GM 42140 from the National Institutes of Health and by grant MCB-9415861 from the National Science Foundation. The Government has certain rights in the invention.</p>
    <heading>BACKGROUND OF THE INVENTION</heading> <p>1. Field of the Invention</p>
    <p>The present invention relates to hybrid genes which encode hybrid restriction endonucleases. The hybrid restriction endonucleases are designed to specifically recognize DNA at given base sites and to enzymatically cleave the DNA at distant sites.</p>
    <p>More specifically, the present invention relates to a method for enzymatically inactivating a target DNA, to a method for detecting conformational change in a nucleic acid and to hybrid molecules comprised of a sequence-specific nucleic acid binding domain joined to a detection domain. 2. Description of the Related Art</p>
    <p>Since their discovery nearly 25 years ago (1), Type II restriction enzymes have played a crucial role in the development of the recombinant DNA technology and the field of molecular biology. The Type II restriction (R) endonucleases and modification (M) methylases are relatively simple bacterial enzymes that recognize specific sequences in duplex DNA. While the former cleave DNA, the latter methylate adenine or cytosine residues within the recognition site so as to protect the host-genome against cleavage by the former. So far, over 2500 restriction and modification enzymes have been identified and these are found in widely diverse organisms (2). These enzymes fall into numerous “isoschizomer” (identically cleaving) groups with about 200 sequence-specificities.</p>
    <p>Discovery of new enzymes involves tedious and time-consuming effort that requires extensive screening of bacteria and other microorganisms (3). Even when one finds a new enzyme, more often than not, it falls into the already-discovered isoschizomer groups. Furthermore, most naturally occurring restriction enzymes recognize sequences that are 4-6 bp long. Although these enzymes are very useful in manipulating recombinant DNA, they are not suitable for producing large DNA segments. For example, restriction enzymes that recognize DNA sequences 6 bp long, result in cuts as often as every 4096 bases. In many instances, it is preferable to have fewer but longer DNA strands, especially during genome mapping. Rare cutters like NotI, that recognizes 8 bp-long sequences, cut human DNA (which contains about 3 billion bp) every 65536 bases on average. So far, only a few endonucleases with recognition sequences longer than 6 bp (rare cutters) have been identified (New England Biolabs catalog).</p>
    <p>R-M (restriction-modification) systems appear, to have a single biological function—namely, to protect cells from infection by foreign DNA that would otherwise destroy them. The phage genomes are usually small. It stands to reason, then, that bacteria select for R-M systems with small recognition sites (4-6 bp) because these sites occur more frequently in the phages. Therefore, a long term goal in the field of restriction-modification enzymes has been to generate restriction endonucleases with longer recognition sites by mutating or engineering existing enzymes (3).</p>
    <p>The FokI restriction endonuclease from <i>Flavobacterium okeanokoites </i>belongs to the Type IIS class of endonucleases. FokI recognizes the asymmetric sequence 5′-GGATG-3′ and cleaves double-stranded DNA at staggered sites 9 and 13 nucleotides away from the recognition site. The cloning and sequencing of the FokI restriction-modification system have been reported. Several research groups have purified FokI endonuclease and characterized its properties. Previous reports by the present inventor on proteolytic fragments of FokI endonuclease using trypsin have revealed an N-terminal DNA-binding domain and a C-terminal catalytic domain with non-specific DNA cleavage activity (4-7). These reports have suggested that the two domains are connected by a linker region which is susceptible to cleavage by trypsin. The present inventor has also shown that insertion of four (or seven) codons between the recognition and cleavage domains of FokI can alter the cleavage distance of FokI within its substrate.</p>
    <p>Recently, Waugh and Sauer have shown that single amino acid substitutions uncouple the DNA-binding and strand scission activities of FokI endonuclease (28). Furthermore, they have obtained a novel class of FokI restriction mutants that cleave hemi-methylated DNA substrates (29). The modular structure of FokI suggested that it may be feasible to construct hybrid endonucleases with novel sequence-specificity by linking other DNA-binding proteins to the cleavage domain of FokI endonuclease. Recently, the present inventor reported the construction of the first “chimeric” restriction endonuclease by linking the Ubx homeo domain to the cleavage domain of FokI (8).</p>
    <p>To further probe the linker region, the present inventor constructed several insertion and deletion mutants of FokI endonuclease. A detailed description of the process for making and using and the properties of these mutants are disclosed in U.S. patent application Ser. No. 08/346,293, allowed, the entire contents of which are hereby incorporated by reference and relied upon.</p>
    <p>Unlike the Ubx homeo domain, zinc finger proteins, because of their modular structure, offer a better framework for designing chimeric restriction enzymes with tailor-made sequence-specificities. The Cys<sub>2</sub>His<sub>2 </sub>zinc finger proteins are a class of DNA-binding proteins that contain sequences of the form (Tyr,Phe)-Xaa-Cys-Xaa<sub>2-4</sub>-Cys-Xaa<sub>3</sub>-Phe-Xaa<sub>5</sub>-Leu-Xaa<sub>2</sub>-His-Xaa<sub>3-5</sub>-His (SEQ ID NOS:1-18) usually in tandem arrays (9). Each of these sequences binds a zinc(ii) ion to form the structural domain termed a zinc finger. These proteins, like many sequence-specific DNA-binding proteins, bind to the DNA by inserting an α-helix into the major groove of the double helix (10).</p>
    <p>The crystallographic structure of the three zinc finger domain of zif268 bound to a cognate oligonucleotide reveals that each finger interacts with a triplet within the DNA substrate. Each finger, because of variations of certain key amino acids from one zinc finger to the next, makes its own unique contribution to DNA-binding affinity and specificity.</p>
    <p>The zinc fingers, because they appear to bind as independent modules, can be linked together in a peptide designed to bind a predetermined DNA site. Although, more recent studies suggest that the zinc finger—DNA recognition is more complex than originally perceived (11,12), it still appears that zinc finger motifs will provide an excellent framework for designing DNA-binding proteins with a variety of new sequence-specificities.</p>
    <p>In theory, one can design a zinc finger for each of the 64 possible triplet codons and, using a combination of these fingers, one could design a protein for sequence-specific recognition of any segment of DNA. Studies to understand the rules relating to zinc finger sequences/DNA-binding preferences and redesigning of DNA-binding specificities of zinc finger proteins are well underway (13-15).</p>
    <p>An alternative approach to the design of zinc finger proteins with new specificities involves the selection of desirable mutants from a library of randomized fingers displayed on phage (16-20). The ability to design or select zinc fingers with desired specificity implies that DNA-binding proteins containing zinc fingers will be made to order. Therefore, we reasoned that one could design “artificial” nucleases that will cut DNA at any preferred site by making fusions of zinc finger proteins to the cleavage domain of FokI endonuclease. We thus undertook the deliberate creation of zinc finger hybrid restriction enzymes, the cloning of the hybrid enzymes, and the characterization of their DNA cleavage properties.</p>
    <p>One of the main difficulties in cloning or overproducing restriction enzymes is their potential lethality. The restriction enzymes can enzymatically attack and destroy the host DNA. This is circumvented by first cloning a methylase gene (M). The methylase gene modifies the restriction enzyme sites and provides protection against chromosomal cleavage. A restriction endonuclease gene (R) is then introduced into the host on a separate compatible plasmid.</p>
    <p>Our work on hybrid restriction endonuclease genes has indicated that they are likewise lethal, since there are no corresponding methylase genes available to protect the host genome from cleavage by the hybrid endonuclease. We now report on a method for cloning the genes for hybrid restriction endonucleases and on a method for using nucleases to enzymatically destroy a target DNA. Furthermore, the method for cloning can be used to clone either mutant or wild type restriction endonucleases.</p>
    <heading>SUMMARY OF THE INVENTION</heading> <p>The present invention reveals a method for enzymatically inactivating a target DNA, a method for detecting conformational change in a nucleic acid, and hybrid molecules comprised of a sequence-specific nucleic acid binding domain joined to a detection domain, e.g., an immunoglobulin molecule.</p>
    <p>The method for enzymatically inactivating a target DNA comprising</p>
    <p>a) preparing a protein containing a nuclease domain,</p>
    <p>i) where the nuclease specifically recognizes the target DNA and enzymatically inactivates the DNA and</p>
    <p>ii) where the nuclease comprises a DNA binding domain that specifically binds to the target DNA,</p>
    <p>b) delivering the protein to a human, animal, or plant, and</p>
    <p>c) enzymatically inactivating the target DNA.</p>
    <p>The DNA binding protein can be either a naturally occurring DNA-binding protein or an engineered or designed DNA-binding protein.</p>
    <p>In addition, the naturally occurring DNA-binding protein or the engineered or designed DNA-binding protein can be selected, for example, from the group consisting of transcription repressor proteins, transcription activator proteins, and DNA origin binding proteins. Preferably, the DNA binding protein is ori binding protein. Furthermore, the ori binding protein can be selected from the group consisting of SV40 T antigen, HSV-I UL9 gene product, Varicella-Zoster gene 51 product, human herpes 6B CH6R gene product, Epstein-Barr virus EBNA-1 gene product, human papilloma virus E1 gene product, and human papilloma virus E2 gene product.</p>
    <p>In addition, the target DNA is preferably selected from the group consisting of human immunodeficiency virus, hepatitis B, herpesviruses, polyoma viruses, and papilloma viruses.</p>
    <p>The method for detecting conformational change in a nucleic acid comprises the steps of:</p>
    <p>a) contacting a nucleic acid with a hybrid restriction nuclease, where the hybrid restriction nuclease interacts with nucleic acids having a conformational change by binding to and cleaving the nucleic acids;</p>
    <p>b) determining whether the hybrid restriction nuclease has interacted with the nucleic acid; and</p>
    <p>c) detecting conformational change in the nucleic acid.</p>
    <p>Preferably, the hybrid restriction nuclease is MutS-F<sub>N </sub>and the conformational change is due to a mutation in the nucleic acid. The mutation can be a point mutation, a single or multiple base pair insertion, or a single or multibase deletion.</p>
    <p>The present invention also discloses a method for inactivating a DNA:RNA hybrid molecule comprising</p>
    <p>a) preparing a protein containing a nuclease domain,</p>
    <p>i) where the nuclease specifically recognizes a target DNA and enzymatically inactivates the DNA and</p>
    <p>ii) where the nuclease comprises a DNA binding domain that specifically binds to the target DNA,</p>
    <p>b) delivering the protein to a human, animal, or plant, and</p>
    <p>c) enzymatically inactivating the target DNA.</p>
    <p>Finally, the present invention discloses hybrid molecules comprising a sequence-specific or conformation-specific nucleic acid binding domain joined to a detection domain. Preferably, the sequence specific nucleic acid binding domain is a zinc-finger DNA binding protein or the conformational specific nucleic acid binding domain is the MutS protein. Preferably, the detection domain is a fluorescent molecule, or alternatively is a protein which can be assayed in an ELISA reader, or is an immunoglobulin heavy chain molecule.</p>
    <heading>DETAILED DESCRIPTION OF THE INVENTION</heading> <p>The present invention discloses a method for cloning hybrid restriction endonucleases and for a method for enzymatically inactivating a target DNA. The latter method involves the use of genes encoding nucleases, including site-specific hybrid restriction endonucleases. The hybrid restriction endonucleases are capable of specifically recognizing, binding to, inactivating, and cleaving the target DNA. The potential lethality of the hybrid restriction enzymes is initially circumvented by using <i>E. coli </i>DNA lig gene in the former method, i.e., for cloning hybrid restriction endonucleases.</p>
    <p>More specifically, the hybrid endonuclease genes are cloned into a tightly controlled expression system to lessen any deleterious effect to the cell and also to increase the level of DNA ligase by placing the <i>E.coli </i>DNA lig gene on a compatible plasmid. This vector expresses the DNA ligase constitutively. Later, in the method for enzymatically inactivating a target DNA, the cells are induced to produce the hybrid restriction endonucleases and to enzymatically inactivate the target DNA.</p>
    <p>The methods of the present invention are exemplified by the following non-limiting examples.</p>
    <heading>EXPERIMENTAL PROCEDURES</heading> <p>The complete nucleotide sequence of the FokI R-M (restriction-modification) system has been published (21,22). Experimental protocols for PCR have been described (4). The procedures for cell growth and purification of proteins using His-bind resin (23) was as outlined in the Novagen pET system manual. The protocol for SDS/PAGE was as described by Laemmli (24).</p>
    <p>1. Cell Transformation Assay</p>
    <p> <i>E. coli </i>strain RR1 and <i>E. coli </i>strain BL21 (DE3) were the host in all experiments. <i>E. coli </i>strain BL21 (DE3), as reported by Studier et al. (26), and <i>E. coli </i>strain RR1, as reported by Maniatis et al. (38), were transformed as described in Maniatis et al. (38). Briefly, The cells were grown to 0.2 OD and incubated with 100 mM CaCl<sub>2 </sub>for 16 hours at 4° C. to make them competent. These cells were then transfected with DNA as described in Maniatis et al. (38).</p>
    <p>2. Construction of the Clones Producing the Hybrid Enzymes ZF-F<sub>N </sub>Using PCR</p>
    <p>The PCR-generated DNAs using oligos 5′-CCCCTGAAGGAGATATACATATG-3′, (SEQ ID NO:19), start primer, and 5′-GGACTAGTCCCTTCTTATTCTGGTG-3′, (SEQ ID NO:20), stop primer, were digested with NdeI/SpeI and then ligated into NdeI/SpeI—cleaved pET-15b Ubx-F<sub>N </sub>vector which contains the FokI nuclease (F<sub>N</sub>) domain. This construct replaces the Ubx homeodomain with the genes coding for zinc finger proteins. The ligation mixture was used to transfect competent RR1 (pACYC184:lig) cells. The glycine linker (Gly<sub>4</sub>Ser)<sub>3 </sub>(SEQ ID NO:21) was inserted between the zinc finger motifs and the FokI nuclease domain using previously described procedures (16). The zinc finger fusion constructs were confirmed by Sanger's dideoxy sequencing method (25). The pET-15b:ZF-F<sub>N </sub>plasmids were then transferred to BL21 (DE3) that carries the compatible plasmid pACYC184:lig.</p>
    <p>3. Purification of ZF-F<sub>N </sub>Endonucleases</p>
    <p>The procedure for the purification of the zinc finger fusion proteins were as follows: 4 L of cells BL21 (DE3) (pACYC184:lig, pET-15:ZF-F<sub>N</sub>) were grown in LB containing 100 μg/ml of ampicillin and 20 μg/ml of tetracycline at 37° C. When OD<sub>600 </sub>reached 0.4, the growth temperature was shifted to 22° C. The cells were induced at OD<sub>600</sub>=0.5 with 0.7 mM of IPTG. After 4 hrs. of induction at 22° C., the cells were harvested by centrifugation. Induction at 22° C. maximizes the yield of soluble hybrid endonucleases in the crude extracts when compared to induction at 30° C. or 37° C.</p>
    <p>The cells were resuspended in Novagen's 1×bind buffer and then disrupted by sonication on ice. After centrifugation at 4° C. for 2 hrs, the crude extract was passed through a 0.45 micron filter and applied to the His-bind affinity column. The column was washed with 1×bind buffer (10 vol.) and 1×wash buffer (6 vol.) as described in Novagen's manual. In addition, the column was washed with 1×wash buffer (4 vol.) containing 100 mM imidazole. The column was eluted with 1×elute buffer containing 400 mM imidazole.</p>
    <p>Fractions containing the fusion proteins were identified by probing the immunoblots with rabbit polyclonal antibody against FokI endonuclease. The eluted fractions containing the hybrid proteins were diluted with 3 volumes of buffer A (10 mM Tris-base, 15 mM NaH<sub>2</sub>PO<sub>4</sub>.H<sub>2</sub>O, 10% glycerol, 100 μM ZnCl<sub>2</sub>, 3 mM DTT, pH 8.0) to reduce salt concentration to 125 mM NaCl and then applied to a SP-sepharose column and eluted with a 0.2 M-1 M linear salt gradient.</p>
    <p>Fractions containing the fusion proteins were concentrated using a SP-sepharose column and then loaded onto a S-100 HR gel-filtration column equilibrated with buffer A containing 0.5 M NaCl. Following the gel-filtration step, pure fractions were combined and the fusion proteins were stored in 50% glycerol at −20° C. or at −70° C. for long-time storage. After the final step of purification, the yield of each purified zinc finger fusion protein was greater than 100 μg per 10 gm of cell paste. The low yield can be attributed to the following: (1) the gene product is toxic to the cells and (2) a large portion of the fusion protein is lost as inclusion bodies.</p>
    <p>4. Construction of ZF-QNR Fusions with Different Linkers</p>
    <p>The three ZF-QNR-F<sub>N </sub>constructs with different linkers were prepared using synthetic oligomers as described below. The inserts for the linkers were made by annealing the appropriate oligomers. These include:</p>
    <p>
      <maths> <formula-text>5′-CTGACGGGGGCCAA-3′  (SEQ ID NO:22):</formula-text> </maths> </p>
    <p>
      <maths> <formula-text>3′-TGCCCCCGGTTGATC-5′  (SEQ ID NO:23)</formula-text> </maths> </p>
    <p>for (GlyGly) linker;</p>
    <p> 5′-CTAGACGGGGGAGGCGGCAGTCAA-3′  (SEQ ID NO:24):</p>
    <p>
      <maths> <formula-text>3′-TGCCCCCTCCGCCGTCAGTTGATC-5′  (SEQ ID NO:25)</formula-text> </maths> </p>
    <p>for (Gly<sub>4</sub>Ser) (SEQ ID NO:26) linker; and</p>
    <p>
      <maths> <formula-text>5′-CTAGACGGGGGAGGCGGCAGTGGAGGTGGCGGATCACAA-3′  (SEQ ID NO:27):</formula-text> </maths> </p>
    <p>
      <maths> <formula-text>3′-TGCCCCCTCCGCCGTCACCTCCACCGCCTAGTGTTGATC-5′  (SEQ ID NO:28)</formula-text> </maths> </p>
    <p>for (Gly<sub>4</sub>Ser)<sub>2 </sub>(SEQ ID NO:29) linker.</p>
    <p>The annealed oligonucleotide duplex made from each pair of primers are flanked by SpeI compatible 5′-overhangs at both ends. The oligonucleotide duplexes were phosphorylated by using T<sub>4 </sub>polynucleotide kinase and ATP. The plasmid pET-15b:ZF-QNR-F<sub>N </sub>was digested with SpeI, dephosphorylated using calf intestinal phosphatase and then gel-purified. The phosphorylated inserts were then ligated into the linearized plasmid.</p>
    <p>Several clones were screened for the appropriate inserts by restriction enzyme digestion. Plasmid with the right orientation of the inserts were further confirmed by DNA sequencing. The hybrid enzymes with different linkers were partially purified using a His-bind affinity column and a SP-sepharose column as described above. The DNA cleavage properties of the partially purified proteins were analyzed using the lambda DNA substrate as described above.</p>
    <heading>EXAMPLES</heading> <heading>1. Construction of Overproducer Clones of ZF-F<sub>N </sub>Using PCR</heading> <p>Two plasmids containing three zinc fingers each (ZF-QDR and ZF-QNR) were shown to preferentially bind to 5′-G(G/A)G G(C/T/A)G GC(T/A)-3′ and 5′-G(G/A)G GA(T/A) GG(G/T)-3′ sequences in double-stranded DNA, respectively (13-15). We used the PCR technique to link the zinc finger proteins to the cleavage domain—(F<sub>N</sub>) of FokI endonuclease (see U.S. patent application Ser. No. 08/575,362, pending, the entire contents of which are hereby incorporated by reference and relied upon). The hybrid gene, ZF-F<sub>N </sub>was cloned as a XhoI/NdeI fragment into pET-15b vector (26), which contains a T<sub>7 </sub>promoter for expression of the hybrid protein. We also inserted a glycine linker (Gly<sub>4</sub>Ser)<sub>3 </sub>(SEQ ID NO:21), between the domains of the fusion protein to confer added flexibility to the linker region (see U.S. patent application Ser. No. 08/575,362, pending).</p>
    <p>This construct links the zinc finger proteins through the glycine linker to the C-terminal 196-amino acids of FokI that constitute the FokI cleavage domain (8). This construct also tags the hybrid protein with six consecutive histidine residues at the N-terminus. These residues serve as the affinity tag for the purification of the hybrid proteins by metal chelation chromatography (23) with Novagen's His-bind resin. This histidine tag, if necessary, can be subsequently removed by thrombin. The hybrid endonucleases with His tag were used in all experiments described below.</p>
    <p>The clones carrying the hybrid genes may not be viable since there is no methylase available to protect the host genome from cleavage by the hybrid endonuclease. We have circumvented this problem as follows: (i) The hybrid genes were cloned into a tightly controlled expression system (26) to avoid any deleterious effect to the cell. (ii) In addition, we increased the level of DNA ligase within the cell by placing the <i>E. coli </i>lig gene on a compatible plasmid pACYC184, downstream of the chloramphenicol promoter. This vector expresses DNA ligase constitutively. BL21 (DE3) served as the host for these experiments. It contains a chromosomal copy of T<sub>7 </sub>RNA polymerase gene under lacUV5 control, the expression of which is induced by the addition of isopropyl, β-D-thio-galactoside (IPTG).</p>
    <p>After induction of the recombinant cells with 0.7 mM IPTG, the hybrid proteins were purified to homogeneity using His-bind resin, SP-sepharose column, and gel filtration chromatography (see U.S. patent application Ser. No. 08/575,362, pending). The size of the hybrid proteins was≅38 kDa and agrees well with that predicted for the fusion proteins. Identities of the hybrid proteins were further confirmed by probing the immunoblot with rabbit antiserum raised against FokI endonuclease (see U.S. patent application Ser. No. 08/575,362, pending).</p>
    <heading>2. Analysis of the Cleavage Activity of the ZF-F<sub>N </sub>Hybrid Enzymes</heading> <p>To determine whether the zinc finger fusion proteins cleave DNA, we used 48.5—kb λ DNA as the substrate. The DNA (30 μg/ml;≅10 nM) was incubated with the enzymes (≅10 nM) in 35 mM Tris.HCl (pH 8.5), 75 mM KCl, 100 μM ZnCl<sub>2</sub>, 3 mM DTT containing 5% (v/v) glycerol, 25 μg/ml yeast tRNA, and 50 μg/ml BSA for 20 min at room temperature in a total volume of 25 μl. MgCl<sub>2 </sub>was then added to a final concentration of 2 mM and the mixture incubated at room temperature for 4 more hrs. The reaction products were analyzed by 0.5% agarose gel electrophoresis.</p>
    <p>The ZF-QNR-F<sub>N </sub>fusion protein cleaves λ DNA into ≅9.5 kb and ˜39 kb fragments (see U.S. patent application Ser. No. 08/575,362, pending). The cleavage is highly specific and the reaction proceeds almost to completion. The ZF-QDR-F<sub>N </sub>fusion protein cleaves λ DNA primarily into a 5.5 kb and a 43 kb fragments (see U.S. patent application Ser. No. 08/575,362, pending). This appears to be the major site of cleavage. There are two other minor sites within the λ genome for this fusion protein. Addition of yeast RNA to the reaction mixture reduces cleavage at the minor site(s). Under these reaction conditions, there was no detectable random nonspecific cleavage as seen from the non-smearing of the agarose gels.</p>
    <p>The cleavage is sensitive to buffer conditions, pH and the purity of the DNA substrate. The kinetics of the cleavage of the lambda DNA substrate using ZF-QDR-F<sub>N </sub>and ZF-QNR-F<sub>N </sub>fusions are shown in U.S. patent application Ser. No. 08/575,362, pending. The cleavage occurs mainly at the major DNA binding site within the lambda genome at short incubation time. The cleavage at the secondary sites become more pronounced with longer incubation times in the case of ZF-QDR-F<sub>N </sub>fusion (see U.S. patent application Ser. No. 08/575,362, pending).</p>
    <p>The cleavage occurs predominantly at the major DNA binding site in the case of the ZF-QNR-F<sub>N </sub>fusion. Only a few weaker bands appear even after long incubation times suggesting that there is only one major DNA binding site for ZF-QNR-F<sub>N </sub>in the lambda DNA substrate (see U.S. patent application Ser. No. 08/575,362, pending).</p>
    <p>The reactions appear to proceed almost to completion (&gt;95% cleavage) within 4 hrs. The kinetics of the cleavage of the lambda DNA substrate by wild-type FokI are shown in U.S. patent application Ser. No. 08/575,362, pending. The cleavage reaction by FokI endonuclease proceeds to completion within 15 minutes. The rate and efficiency of cleavage by the hybrid endonucleases are much lower compared to wild-type FokI.</p>
    <p>We have also studied the effect of temperature and salt concentrations (KCl and MgCl<sub>2</sub>) on ZF-QNR-F<sub>N </sub>fusion protein cleavage activity using the lambda DNA as a substrate. The results of these experiments are shown in U.S. patent application Ser. No. 08/575,362, pending. The cleavage efficiency by ZF-QNR-F<sub>N </sub>appears to decrease with increasing temperatures (see U.S. patent application Ser. No. 08/575,362, pending). Room temperature (22° C.) appears to be the optimal temperature for the cleavage reaction. This may indicate the decreased binding of the ZF-QNR-F<sub>N </sub>fusion protein to the lambda DNA substrate at higher temperatures. The optimal salt concentration for cleavage appears to be 75 mM KCl. Under these conditions, the reaction proceeds to completion (see U.S. patent application Ser. No. 08/575,362, pending).</p>
    <p>The cleavage efficiency appears to drop off with increasing KCl concentration. This can be attributed to the instability of the protein-DNA complex at higher salt concentrations. The effect of increasing MgCl<sub>2 </sub>(co-factor) concentration on the cleavage reaction is shown in U.S. patent application Ser. No. 08/575,362, pending. The efficiency of cleavage increases with MgCl<sub>2 </sub>concentration and the reactions proceed to completion. However, with increasing MgCl<sub>2 </sub>the nonspecific cleavage by the FokI nuclease domain becomes more pronounced. The optimal MgCl<sub>2 </sub>concentration for the cleavage reaction appears to be between 2-3 mM.</p>
    <p>These experiments demonstrate that cleavage activity of the ZF-QDR-F<sub>N </sub>and ZF-QNR-F<sub>N </sub>fusions are quite reproducible. Furthermore, they also show that the reaction conditions can be optimized for site-specific cleavage as well as for the complete cleavage of the substrate.</p>
    <p>These results are consistent with what is known about zinc finger-DNA interactions. The zinc finger-DNA recognition appears to be by virtue of only two base contacts of the triplet per zinc finger (10). Therefore, zinc fingers may recognize more than one DNA sequence differing by one base in the central triplets. This may explain why the ZF-QDR-F<sub>N </sub>hybrid enzyme recognizes several DNA sites with different affinities, and then cuts these sites with different efficiencies. Thus, the subsite bindings of relatively moderate affinity may contribute to the degeneracy of cleavage. On the other hand, the ZF-QNR-F<sub>N </sub>fusion suggests that a hybrid restriction enzyme with a high sequence-specificity can be engineered by using the appropriate zinc finger motifs in the fusion constructs.</p>
    <heading>3. Analysis of the DNA-sequence Preference of the ZF-F<sub>N </sub>Hybrid Enzymes</heading> <p>Determination of the major DNA-binding sites of ZF-QDR-F<sub>N </sub>and ZF-QNR-F<sub>N </sub>fusion proteins were done in two steps: First, by using a series of known restriction enzyme digests of the λ DNA followed by cleavage with the fusion protein, the site was localized within a 1-2 Kb region of the genome. Second, a 300 bp λ DNA fragment containing the major cleavage site was isolated. This substrate was end-labeled with <sup>32</sup>p on the top DNA strand or the bottom DNA strand. The products of cleavage of each labeled substrate were analyzed by denaturing polyacrylamide gel electrophoresis (25) followed by autoradiography (see U.S. patent application Ser. No. 08/575,362, pending). More specifically, cleavage products of the <sup>32</sup>p-labeled DNA substrate containing a single binding-site by ZF-F<sub>N </sub>along with (G+A) sequencing reactions were separated by electrophoresis on a 8% polyacrylamide gel containing 6 M urea. The gel was dried and exposed to an x-ray film for 6 hrs.</p>
    <p>The map of the primary recognition and cleavage site(s) of the ZF-QDR-F<sub>N </sub>and ZF-QNR-F<sub>N </sub>fusion proteins found in the λ genome are shown in U.S. patent application Ser. No. 08/575,362, pending. The ZF-QDR-F<sub>N </sub>fusion protein preferentially binds to 5′-GAG GAG GCT-3′, which is one of the four predicted consensus sites that occur in the λ genome. The ZF-QNR-F<sub>N </sub>fusion does not bind to any of the four predicted consensus sites that are present in the λ genome. It preferentially binds to the 5′-GAG G<u>G</u>A <u>T</u>GT-3′ site that occurs only once in the genome. The two bases that are different from the reported consensus recognition site of ZF-QNR are underlined. The reported consensus DNA binding sites of the zinc finger proteins were determined by affinity-based screening (13-15). This method utilizes a library of DNA binding sites. Under representation of any of the possible sites within this library may lead to the identification of a subsite as the optimal DNA binding site. Alternatively, the fusion of the zinc finger proteins to the FokI cleavage domain may alter the DNA sequence-specificity. This is unlikely because the binding sites for the previously reported Ubx-F<sub>N </sub>and one of the two ZF-F<sub>N </sub>fusions described here agree with the reported consensus DNA sites. As many more zinc finger fusions are engineered and characterized, this apparent discrepancy may be resolved. If the sequence-specificity of the hybrids is indeed altered, then we need to develop a fast and efficient screening method to identify or select the DNA binding sites of the hybrid restriction enzymes.</p>
    <p>The specificity of the two hybrid restriction enzymes described here are different. More than likely, the specificity of these enzymes are determined solely by the DNA-binding properties of the zinc finger motifs. It appears that the hybrid endonucleases do turnover, that is, the fusion proteins come off the substrate after cleavage. Both enzymes cleave the top strand near the binding site; they cut the bottom strand at two distinct locations. Both fusions show multiple cuts on both strands of the DNA substrate (see U.S. patent application Ser. No. 08/575,362, pending). One possibility is that the cleavage domain is not optimally positioned for cutting. Naturally occurring Type IIS enzymes with multiple cut sites have been reported in the literature (27). The variations in the cleavage pattern of the two hybrid enzymes can be attributed to the differences in the mode of binding of the zinc finger motifs to their respective DNA-binding sites and to the orientation of the nuclease domain within the enzyme-DNA complex.</p>
    <heading>4. ZF-QNR Fusions with Different Linkers Between the Recognition ans Cleavage Domains</heading> <p>Five different ZF-QNR-F<sub>N </sub>hybrids containing different linkers were constructed using synthetic oligomers (see U.S. patent application Ser. No. 08/575,362, pending). The fusion protein from each construct was partially purified using His-bind affinity column and SP-sepharose column. The presence of the fusion proteins were confirmed by Western blots using polyclonal antisera raised against restriction FokI endonuclease. Only small amounts of intact fusion proteins were obtained in the case of the hybrids with (Gly<sub>4</sub>Ser) (SEQ ID NO:26) and (Gly<sub>4</sub>Ser)<sub>2 </sub>(SEQ ID NO:29) linkers and therefore, they were not tested for sequence-specific cleavage activity.</p>
    <p>The fusions with no linker, (GlyGly), and (Gly<sub>4</sub>Ser)<sub>3 </sub>(SEQ ID NO:21) were partially purified as described above; the hybrid enzymes from constructs with no linker or (GlyGly) linker showed only minimal sequence-specific cleavage (see U.S. patent application Ser. No. 08/575,362, pending). (Gly<sub>4</sub>Ser)<sub>3 </sub>(SEQ ID NO:21) appears to be the optimal spacer between the zinc finger and the FokI cleavage domain. This spacer appears to provide the added flexibility to the two functional domains of the zinc finger hybrids that is necessary for optimal DNA cleavage.</p>
    <heading>5. Cloning ZF-F<sub>N </sub>in the Presence of lig Gene</heading> <p>The <i>E. coli </i>lig gene was inserted into the NcoI site of plasmid pACYC184. The plasmid was prepared as described by Chang and Cohen (39) and carries the tetracycline drug marker. The recombinant plasmid carrying the lig gene in the same orientation as the chloramphenicol promoter was identified as pACYC lig (see U.S. patent application Ser. No. 08/575,362, pending). This recombinant was transfected into competent <i>E. coli </i>strain BL21 (DE3). The hybrid restriction endonuclease genes on a separate compatible plasmid, pET-15b were transfected into competent BL21 (DE3) (pACYC lig) as well as competent BL21 (DE3) cells.</p>
    <p>0.1 ml of the transformation mix was plated on LB-Amp-Tet plates (see U.S. patent application Ser. No. 08/575,362, pending). The plasmid pTZ19R that does not carry a hybrid endonuclease gene was used as a standard control to compare the efficiency of transformation of the competent cells. BL21 (DE3) (pACYC lig) transformed at about 5-10 fold lower efficiency as compared to the BL21 (DE3) cells (see U.S. patent application Ser. No. 08/575,362, pending).</p>
    <heading>6. Transformation Efficiency of Hybrid Genes</heading> <p>The transformation efficiency of two different endonuclease genes, pET-15b:ZFHD1-<sub>N </sub>and pET-15b:ZF-QQR-F<sub>N</sub>, into BL21 (DE3), with and without pACYC lig is shown in U.S. patent application Ser. No. 08/575,362, pending. BL21 (DE3) with pACYC lig transform about 2-fold better compared to BL21 (DE3) without the pACYC lig. Taking into consideration 5-10 fold lower efficiency of BL21 (DE3) (pACYC lig) as compared to BL21 (DE3), this translates into about 10-20 fold difference between <i>E. coli </i>strains with and without pACYC lig.</p>
    <heading>7. Induction of Hybrid Nuclease Activity</heading> <p>In a different example, pET-15b:ZF-QDR-F<sub>N </sub>(where the hybrid endonuclease gene is under the control of a T<sub>7 </sub>promoter) was transfected into two different <i>E. coli </i>strains, namely RR1 and BL21 (DE3) both without the plasmid, pACYC lig carrying the lig gene. While pET-15b:ZF-QDR-F<sub>N </sub>was stably maintained in RR1, it is unstable when it is transfected into BL21 (DE3), which has a copy of the T<sub>7 </sub>RNA polymerase gene in its chromosome. Only mutants of the hybrid endonuclease gene were obtained upon transfection of competent BL21 (DE3) cells. Plasmids from six different clones were isolated and analyzed by digestion with NdeI/XhoI. While three clones showed that the hybrid restriction endonuclease gene was deleted, the others contained inserts that were 0.9 kb larger than the original gene (see U.S. patent application Ser. No. 08/575,362, pending). The hybrid gene appears to be disrupted by the insertion of an IS1 element.</p>
    <p>We circumvented this problem by transfecting the pET-15:ZF-QDR-F<sub>N </sub>into <i>E. coli </i>BL21 (DE3) cells that carry the compatible plasmid (PACYC lig) which has the <i>E. coli </i>lig gene inserted downstream of the chloramphenicol promoter. This plasmid expresses the DNA ligase constitutively. The pET-15b:ZF-QDR-F<sub>N </sub>is stable within these cells. Induction of these clones with IPTG result in the production of the hybrid enzyme. This is an important finding and development since this implies that any hybrid endonuclease will be tolerated by the cells provided they can express the DNA ligase constitutively and thereby repair the damage. No methylase is needed to protect the host genome from cleavage by the hybrid endonuclease.</p>
    <p>We have cloned several hybrid endonucleases using this approach (see U.S. patent application Ser. No. 08/575,362, pending). ZF-Sp1C-F<sub>N </sub>is a specific abbreviation for a fusion product between the three zinc finger motif of the eukaryotic transcription factor Sp1 (a specific zinc finger) and the FokI endonuclease domain (F<sub>N</sub>).</p>
    <p>This patent application is for a method of cloning any hybrid endonuclease gene in any type of cell wherein any DNA ligase is produced at an increased level compared to the normal level of DNA ligase in the specific cell type. More specifically, the method of this patent application includes the cloning of any hybrid endonuclease gene in any prokaryotic (e.g., <i>E. coli</i>, mutants of <i>E. coli, </i>etc.) or eukaryotic (e.g., yeast, plant, or mammalian, etc.) cell that has been altered to produce increased levels of any type of DNA ligase (e.g., T<sub>4 </sub>ligase gene, etc.) within the cell.</p>
    <heading>8. Induction of Anti-bacterial Activity with Hybrid Nuclease</heading> <p>A specific application for these engineered sequence-specific endonucleases is in the cleavage, and thereby inactivation of genes in vivo. Several methods are currently available to express foreign genes in a number of bacterial, fungal, plant and animal species. These include transient expression via episomal or viral vectors or by microinjection. Such methods could be used for the delivery and expression of hybrid endonucleases within cells. Essentially any DNA intermediate is a potential target or substrate for cleavage by a hybrid endonuclease. These include RNA tumor viruses which replicate through a DNA intermediate. It should be possible to target one or more hybrid endonucleases against these specific DNA intermediates provided the gene sequences are known. Expression in vivo of such hybrid restriction enzymes would in effect destroy the corresponding gene. This targeted gene inactivation by the hybrid endonucleases could provide a basis for various anti-viral and anti-bacterial therapies and for a way to inactivate human, animal or plant genes.</p>
    <p>In another example, plasmids containing one of two different hybrid endonucleases, namely pET-15b:ZF-QDR-F<sub>N </sub>and pET-15b:ZF-QNR-F<sub>N</sub>, were separately transfected into <i>E. coli </i>BL21 (DE3) (pACYC lig) by standard CaCl<sub>2 </sub>procedure. The clones were then plated on LB-Amp-Tet plates with and without IPTG. Induction with IPTG turns on the production of T<sub>7 </sub>RNA polymerase, which lead to the production of the hybrid restriction enzymes. The constitutively produced ligase cannot cope and repair the damage resulting from the hybrid restriction enzymes. Therefore, the clones should not be viable upon induction with IPTG.</p>
    <p>Results obtained from such an experiment are shown in U.S. patent application Ser. No. 08/575,362, pending. BL21 (DE3) (pACYC lig) containing the hybrid endonuclease genes on a compatible plasmid grow well on LB-Amp-Tet plates without IPTG. No growth is observed when they are grown on LB-Amp-Tet plates containing 1 mM IPTG. Control BL21 (DE3) (pACYC lig) (pET-15b) strain that does not carry the hybrid restriction endonuclease gene grow well on LB-Amp-Tet plates with and without IPTG (see U.S. patent application Ser. No. 08/575,362, pending).</p>
    <p>This example shows that bacteria carrying the hybrid restriction enzymes gene can be forced to self-destruct by inducing the hybrid restriction enzymes. This example also provides proof of concept for potential use of hybrid restriction enzymes as therapeutic agents. Obviously, the hybrid restriction endonuclease genes could also be delivered into cells via a plasmid, virus, phage, or any other delivery vehicle that infects a particular type of bacterial or mammalian cells, including plant and animal cells; or the hybrid endonucleases could be introduced into cells by direct transfer via liposomes or by fusion to the translocating domains of bacterial toxins (Pastan et al., 1992).</p>
    <heading>9. Use of Hybrid Enxymes in the Treatment of Viral Diseases</heading> <p>Bacteriophages have been shown to be effective in the treatment of experimental <i>E. coli </i>infection (10,11). More recently, bacteriophage was shown to prevent destruction of skin grafts by <i>Pseudomonas aeruginosa </i>(12). These bacteriophages can be engineered to carry the lethal hybrid endonuclease genes targeted against their hosts. These bacteriophages will be more effective in the destruction of the bacteria they infect. The present invention specifically includes this concept as well. The present invention also contemplates the delivery of other normal as well as mutant site-specific restriction enzymes using a similar approach.</p>
    <p>One specific application of chimeric restriction enzymes is as therapeutics in the treatment of viral diseases caused by DNA viruses or retroviruses that replicate with a DNA intermediate. Chimeric restriction enzymes can be designed so that the DNA binding domain specifically targets viral-specific DNA sequences.</p>
    <p>The main goal in treating viral infection is reducing viral load in infected cells and within a patient. Anti-viral drugs available today are generally toxic and have little specificity. Certain drugs are designed to inhibit a component of the virus's replicative machinery such as the enzymes thymidine kinase or reverse transcriptase. These agents do not destroy viral DNA. Other anti-viral agents act to promote the host's immune response so that infected cells are killed more efficiently. This results in non-specific destruction of both viral and host cell DNA.</p>
    <p>At present there is a need for new therapeutic agents that specifically destroy viral DNA without destroying host cell DNA. Most viral DNA synthesis occurs within the cell's nucleus; thus it is important to generate therapeutic agents that can distinguish between viral and host cell DNA.</p>
    <p>The inventor of the present invention has identified strategies for generating chimeric restriction endonucleases that can be used to specifically cleave viral DNA. One example of this strategy involves identification of DNA sequences within the viral genome that are viral-specific, i.e., they are not present within the human genome. Once identified, DNA binding domains that specifically bind to such sequences with high affinity can be designed using zinc finger modules. Such DNA binding domains can be fused to the nuclease domain of FokI to create chimeric restriction enzymes.</p>
    <p>Another possible embodiment of the present invention is to take advantage of the fact that some viruses encode DNA binding proteins that specifically bind their own viral DNA sequences with high affinity and specificity. Such proteins include transcription repressor proteins, transcription activator proteins and DNA origin-binding proteins. The DNA origin is a specific site in the viral DNA where DNA synthesis initiates.</p>
    <p>Many eukaryotic DNA viruses such as herpesviruses and parvoviruses depend upon the host cell's DNA syntheses machinery to replicate their DNA. In particular, those viruses use the host cell's DNA polymerase to replicate their DNA. DNA synthesis initiates at a specific viral DNA sequence known as an origin of replication, or ori. The host cell DNA polymerase does not bind to or recognize the viral ori DNA sequence. The virally-encoded ori-binding protein also contains a second site that binds DNA polymerase. Thus, the ori binding protein serves to bring DNA polymerase in contact with ori to initiate DNA synthesis and viral DNA replication. The exact sequence of the ori region differs in different viruses.</p>
    <p>The most well characterized ori binding protein is T antigen of SV40 virus. In SV40 T antigen, the amino acids that bind ori and the amino acids that interact with DNA polymerase have been shown to reside in different domains of the protein. Other examples of ori binding molecules include, but are not limited to, the HSV-I UL9 gene product, Varicella-Zoster gene 51 product, human herpesvirus 6B CH6R gene product, the Epstein-Barr virus EBNA-1 gene product, and the human papilloma virus E1 and E2 gene products.</p>
    <p>Many of the ori-binding proteins possess a DNA helicase (ATPase) activity that has been localized to a region of the protein that is distinct from the DNA-binding region. For use of these proteins as chimeric restriction enzymes, it would be advantageous to delete this region from the protein or mutate it so that this enzymatic activity is no longer present.</p>
    <p>The chimeric restriction enzymes are constructed using the methods described in the previous examples. Such chimeric restriction enzymes can be introduced into cells using liposomes or by fusion to the translocating domains of bacterial toxins (Pastan et al., 1992) or by either stably or transiently having gene expression using vectors such as pcDNA3 (commercially available from Invitrogen). In some cases it may be advantageous to incorporate a nuclear localization signal into the recombinant protein to ensure that it is expressed within the cell nucleus. Transformed cells could be infected with appropriate viruses to determine if they prevent virus growth as measured by a reduction in plaques relative to cells not expressing the chimeric enzymes.</p>
    <p>Any virus that contains a double stranded DNA stage in it's life cycle can be targeted for destruction by creating a chimeric restriction enzyme that recognizes DNA sequences specific for the viral genome. Such viruses include human immunodeficiency virus (HIV), Hepatitis B, herpesviruses such as HSV, Cytomegalovirus (CMV), VZV, EBV, Herpesvirus 6B, polyoma viruses such as SV40, and papilloma viruses such as human papilloma virus (HPV). To assure that the targeted viral DNA sequences are not present in the host's genome, such DNA target sequences should be at least 15 nucleotides in length and preferably at least 18 nucleotides in length. DNA binding proteins with specificities for this length of DNA sequence can be designed using zinc finger modules. Zinc finger domains bind with high affinity and specificity to three consecutive nucleotides in a DNA helix. Zinc finger domains can be joined together to form a protein that recognizes 6 or 9 consecutive nucleotides. In the preceding examples, it was shown that DNA binding domains comprising three zinc fingers can be joined to the nuclease domain of FokI to create chimeric restriction enzymes that bind to and cleave specific DNA targets. There is considerable work on-going to understand the rules of zinc finger binding to DNA.</p>
    <p>In theory, it is possible to identify a zinc finger that binds each of the 64 possible triplet codons. Thus, it will be possible to design zinc finger proteins that bind any predetermined DNA sequence. The fact that individual zinc finger modules can be fused together using recombinant DNA techniques makes a very powerful technology for generating chimeric restriction endonucleases capable of cutting any pre-determined DNA sequence. Such molecules can be used to target viral genomes for cleavage.</p>
    <p>It is possible to increase the specificity of the target site to 15 or more base pairs by creating fusion proteins containing 5 or more zinc fingers within a single protein. The fusion proteins can be constructed by linking together two or more zinc finger proteins, each of which contains 2-3 zinc fingers, separated by spacer amino acids. It also is possible to create chimeric enzymes with 15 or more base pair specificity by creating dimeric zinc finger proteins, each half of which recognizes 6 or 9 nucleotides. The latter proteins can be constructed as described above. The former proteins can also be constructed as described above by linking together two or more 2-3 zinc finger domains separated by spacer amino acids.</p>
    <p>Zinc finger proteins can be designed to bind to specific viral DNA sequences of at least 15 and preferably at least 18 nucleotides. For a 15 nucleotide target site the target DNA stretches should be 6 and 9 nucleotides in length. For 18 nucleotides the target site should consist of at least two DNA stretches each of which is 9 nucleotides long. The 9 nucleotide target DNA stretches should be separated by spacer nucleotides that are not specific recognition sequences. The spacer nucleotides should be at least one and preferably less than 100 nucleotides in length. An 18 nucleotide recognition sequence also could be created by targeting three nucleotide stretches, each of which is 6 nucleotides long. Target DNA stretches longer than 18 nucleotides could be subdivided into stretches that are multiples of three nucleotides.</p>
    <p>Such chimeric enzymes can be expressed and purified from bacteria, preferably <i>E. coli</i>, as described in the preceding examples. The purified enzymes can be incubated with DNAs containing their target sequences to show that they cut the DNAs specifically at the target sites.</p>
    <p>Such chimeric enzymes can be introduced and stably expressed in mammalian cells using techniques well known to those in the art. Cells expressing the chimeric enzymes can be infected with virus to determine if they reduce growth of the virus as measured by any of a number of criteria such as a reduction in plaque number.</p>
    <p>Any DNA sequence in the viral genome can be targeted for cleavage by chimeric restriction enzymes using the above-described procedures. Preferred target sites include those sequences that are conserved between different strains of the virus or which lie within genes essential for virus propagation or infectivity. For HIV, a preferred target is within the TAT, REV or TAR genes. For HBV, a preferred target is a highly conserved 62 nucleotide sequence in the pre-core/core region (40).</p>
    <p>In some cases it would be advantageous to destroy viral DNA irrespective of whether host DNA also is destroyed. In these cases essentially any nucleotide sequence in the viral genome can serve as a target sequence. In these cases, a chimeric restriction endonuclease with little nucleotide specificity would be adequate.</p>
    <heading>10. Designer DNA Binding Proteins and Cleavage to Inactivate Genes in Vivo</heading> <p>A potential application for these engineered sequence-specific endonucleases is in cleavage, and thereby inactivation of genes in vivo. Several methods are currently available to express foreign genes in a number of bacterial, fungal, plant, and animal species. These include transient expression via episomal or viral vectors or by microinjection.</p>
    <p>Such methods could be used for the delivery and expression of hybrid endonucleases within cells. Essentially any double-stranded DNA molecule is a potential substrate for cleavage by a hybrid endonuclease. These include human, animal, and plant genes.</p>
    <p>It should be possible to target one or more hybrid endonucleases against specific DNA molecules provided the DNA sequences are known. Expression in vivo of such hybrid restriction enzymes would in effect destroy the corresponding gene.</p>
    <heading>11. Use of Hybrid Enzymes to Detect Mutations</heading> <p>A major goal of medicine in the next century is to detect and identify genetic variants in human population that predispose to disease. Diagnostic tools/kit are essential in molecular medicine to identify sequences of allelic variants in human population particularly those associated with clinical disease.</p>
    <p>The present invention can be used for screening patients to detect known, disease-causing gene mutations. The invention can also be used to provide a general method for identifying and mapping rare or previously unknown gene mutations.</p>
    <p>a) Detecting Known Mutations</p>
    <p>The β-globin and p53 genes can be used as model systems to test the applicability of using custom-designed restriction enzymes to detect known point mutations within genes.</p>
    <p>i) Sickle Cell Anemia</p>
    <p>Sickle cell anemia, a human genetic disease, is the result of a single base pair change (adenine to thymine) in the β-globin gene. This corresponds to the sixth amino acid residue (changing glutamic acid to valine) in the β-globin protein. <chemistry> <div class="patent-image small-patent-image"> <a href="//patentimages.storage.googleapis.com/US6265196B1/US06265196-20010724-C00001.png"> <img id="EMI-C00001" file="US06265196-20010724-C00001.TIF" img-content="chem" img-format="tif" src="//patentimages.storage.googleapis.com/US6265196B1/US06265196-20010724-C00001.png" class="patent-full-image" alt="Figure US06265196-20010724-C00001"> </a> </div> <attachments> <attachment idref="CHEMCDX-00001" attachment-type="cdx" file="US06265196-20010724-C00001.CDX"> </attachment> <attachment idref="CHEMMOL-00001" attachment-type="mol" file="US06265196-20010724-C00001.MOL"> </attachment> </attachments> </chemistry> </p>
    <p>The general approach that is used to detect the known gene mutation in sickle cell anemia involves three distinct steps:</p>
    <p>(1) a hybrid endonuclease that recognizes only the mutant gene and not the normal or wild-type gene is engineered;</p>
    <p>(2) PCR or other amplification technology is used to amplify the region of the gene in question from the biologic sample; and</p>
    <p>(3) the amplified PCR fragment is digested with the hybrid endonuclease and the cleavage products analyzed by agarose gel electrophoresis.</p>
    <p>If the person was homozygous for the sickle cell mutation, the amplified DNA fragment would be cut by the endonuclease, whereas if the person was homozygous for the wild type allele, the amplified DNA fragment would not be cut by the endonuclease. If the person was heterozygous for the sickle cell mutation, half the amplified DNA would be cut by the endonuclease. By determining whether the amplified DNA is cut by the endonuclease, for example by electrophoresing the digested DNA on an agarose gel and detecting the DNA bands by ethidium bromide staining, one can easily determine whether the person contains the sickle cell mutation.</p>
    <p>The crucial step in this protocol is the design or selection of a zinc finger protein that binds the mutated DNA site with high affinity and in a sequence-specific manner. Here, we design or select a zinc finger protein that will bind preferentially to the site: <chemistry> <div class="patent-image small-patent-image"> <a href="//patentimages.storage.googleapis.com/US6265196B1/US06265196-20010724-C00002.png"> <img id="EMI-C00002" file="US06265196-20010724-C00002.TIF" img-content="chem" img-format="tif" src="//patentimages.storage.googleapis.com/US6265196B1/US06265196-20010724-C00002.png" class="patent-full-image" alt="Figure US06265196-20010724-C00002"> </a> </div> <attachments> <attachment idref="CHEMCDX-00002" attachment-type="cdx" file="US06265196-20010724-C00002.CDX"> </attachment> <attachment idref="CHEMMOL-00002" attachment-type="mol" file="US06265196-20010724-C00002.MOL"> </attachment> </attachments> </chemistry> </p>
    <p>As indicated in Table I, the zinc fingers that preferentially bind to each of these triplets have been identified. Methods for joining those three zinc fingers together have been described in Desjarlais and Berg, 1993 (14). By using standard recombinant techniques, the three zinc fingers are fused together to form the protein that binds preferentially to the mutated DNA site. As discussed previously, the engineered zinc finger protein is linked to the cleavage domain of FokI to form the hybrid restriction enzyme. If this enzyme is not highly specific, the sequence of DNA sites encompassing the gene mutation is altered by moving the mutation to any one of the nine possible positions. Several different zinc finger proteins can be engineered and screened for their DNA site specificity until one that binds the mutated site in a sequence specific manner is found.</p>
    <p>Another approach uses the phase display system to select the desirable mutants from a library of randomized zinc fingers. Several laboratories have used random mutagenesis and phage display to alter the DNA-binding of Zif268, a transcription factor that contains three zinc finger domains. Within the crystal structure of the three reported zinc finger-DNA complexes, direct base contact has been observed between side chains of residues −1 to +6 with the exception of residue 4 of the α-helix. Randomization of only four positions (−1, +2, +3 and +6) appears to yield similar results to the work where more positions of the α-helix were randomized. Affinity selection using DNA sites with the base changes in the region recognized by the zinc finger, yields Zif268 variants that bind tightly and specifically to the new sites. These studies show it is possible to isolate zinc fingers by phage display that distinguish operator sequences that differ by only a single base change.</p>
    <p>The step of negatively selecting against the unmutated sites with original specificity can be incorporated in these procedures to improve the library of Zif268 variants to reduce the background. In the next step, these zinc fingers are linked to the cleavage domain of FokI to generate novel restriction enzymes and confirm their sequence-specificity. The limitations of phage display include the size of the library. Assuming full degeneracy of the eight variable positions of the α-helix, the number of transformants required is (16<sup>7</sup>×2<sup>1</sup>=) 5.4×10<sup>8</sup>. The practical limitation of these procedures is the efficiency of transformation in <i>E. coli. </i> </p>
    <p>A corollary experiment using a hybrid endonuclease that recognizes only the normal or wild-type β-globin gene sequence and not the mutant version is done to confirm the results. The zinc finger specific for the altered TGA triplet is known. As discussed above, a zinc finger protein that binds the normal DNA site 5′-TGA GGA GAA-3′ with high specificity is engineered or selected. This is then linked to the FokI cleavage domain to form the hybrid restriction enzyme. The method is generally applicable to detect all gene mutations associated with a wide variety of human diseases.</p>
    <p>i) Tumor Suppressor Gene p53</p>
    <p>Although many mutations have been described for the tumour suppressor gene p53, mutations affecting only 6 amino acid residues (R175, G245, R248, R249, R273 and K282) account for the majority of disease-causing mutations. The availability of simple diagnostic tests for these 6 common mutation “hotspots” would be of considerable value to the majority of patients and their relatives. A similar situation exists for the cystic fibrosis gene in which 70% of patients in the United States have the same mutation, a deletion of the 3 nucleotides encoding phenylalanine at position 508.</p>
    <p>Hybrid restriction enzymes that recognize the normal or wild-type amino acid sequence at each of the 6 mutational hotspots can be created. Since mutations R248 and R249 are adjacent, it is possible that a single hybrid restriction enzyme will suffice to recognize mutations at both sites. The methods to be used to create these hybrid enzymes is essentially the same as that described for the β-globin gene.</p>
    <p>b) Detecting Unknown Mutations</p>
    <p>Several different mutations in a given gene can result in the inactivation of that gene. The cystic fibrosis gene and p53 tumour suppressor gene are good examples of this phenomenon. More than a thousand p53 mutations have been identified in human tumors. The majority of these mutations occur in the central 200-amino acid portion of p53 and they are particularly common in the four conserved regions. Mutations at only 6 amino acid residues constitute the major portion of the observed mutations.</p>
    <p>In the case of cystic fibrosis, mutations affecting more than 220 different nucleotides have been described. However, approximately 70% of the cystic fibrosis cases in the United States are due to the same mutations, a 3 nucleotide deletion that deletes phenylalanine at amino acid position 508. Thus, in both the p53 and cystic fibrosis gene, a large number of different mutations have been described but a few mutational hotspots account for the majority of the disease incidence.</p>
    <p>The general approach used to detect these different mutations within p53 involves three distinct steps:</p>
    <p>First, a hybrid endonuclease that recognizes base pair mismatches in DNA is engineered. In one example, this is accomplished by the fusion of FokI nuclease domain to the MutS protein of <i>E. coli </i>that binds to all the eight possible mismatches (see below for specific details).</p>
    <p>Second, PCR or another amplification procedure is used to amplify the appropriate region of the gene in question from a patient's DNA. The amplification products are denatured and reannealed.</p>
    <p>Third, the reannealed product is then incubated with the hybrid endonuclease and the cleavage products analyzed by agarose gel electrophoresis. If there is a DNA base mismatch, i.e., a mutation in one of the patient's gene, the hybrid restriction enzyme recognizes and binds to the mismatch and cut near the site. If the patient has two copies of the wild type gene or two copies of the same mutation, there is no cleavage and a single DNA band is seen, the same as with wild-type, control DNA. If there is a base mismatch, the DNA is cleaved to two smaller fragments. From the sizes of the bands one can determine where the mutation occurs within the DNA. To determine if the patient has two copies of the wild type gene or two copies of the mutant gene, the amplified region of the patients gene is mixed with the amplified region of the gene from a control, wild type DNA sample. Asymmetric PCR or another amplification technology can be used to amplify either the (+) or (−) strands in each of these reactions, if desired. The amplified DNAs are mixed, denatured and then reannealed. The reannealed DNAs are incubated with the hybrid endonuclease and products analyzed as described above.</p>
    <p>The method is generally applicable to detect all the different gene mutations associated with a wide variety of human diseases. If can also be used to detect known gene mutations, for example, sickle cell anemia.</p>
    <p>i) Construction of the Clones Producing the Hybrid Endonucleases MutS-F<sub>N </sub> </p>
    <p>The crucial step of this method is the engineering of a hybrid restriction enzyme that binds to DNA mismatches and cleaves near these sites. The <i>E. coli </i>MutS protein binds to mismatch DNA base pairs. It displays variable affinity to all eight possible mismatches. The MutS protein can be linked to the cleavage domain of FokI endonuclease, thereby converting MutS into an hybrid restriction enzyme that will bind to DNA mismatches and cleave near these sites. By using standard recombinant DNA techniques, it is possible to fuse the MutS gene to the gene encoding the FokI cleavage domain. It is possible to construct the hybrid protein so that the FokI nuclease domain is at the N-terminal end of the hybrid protein or at the C-terminal end of the hybrid protein.</p>
    <p>c) Automation of the Technology</p>
    <p>An important aspect of diagnostic and detection methodology is how one measures the signal; what kinds of “reporter” systems can be used? Considerable progress has been made in the design of the reporter enzymes that can amplify the assay sensitivity by several fold. For example, fluorescent or light-producing enzyme reactions have been refined and are replacing radioactivity for a number of applications.</p>
    <p>The cleavage of the DNA substrate by hybrid restriction enzymes results in a majority of product with 5′-overhangs. These sticky ends can be filled-in by DNA polymerases and dNTPs carrying appropriate fluorescent tags or other reporter groups like biotin. This can be used to further amplify the signal from each target molecule for easy detection. Thus, our approach not only amplifies the number of target-DNA molecules (by PCR) but also has the potential for amplification of the signal from each target molecule, thereby increasing the sensitivity of the method several orders of magnitude. This approach makes the method suitable for automation. The signal detection can be done rapidly and efficiently using fluorescent enzymatic techniques. obviously, the unreacted dNTPs with the reporter groups have to be removed before the signal can be detected. This can be done in three ways:</p>
    <p>(1) The labelled mixtures can be spotted on a PEI (polyethylene imine) strip, dried and developed in a TLC chamber using 1N HCl. The unreacted dNTPs move with the solvent front to the top of the strip, while the larger substrate and the cleaved products remain at the bottom of the strip. After drying the PEI strip, each of the spots can be rapidly screened for fluorescence. This assay is very similar to the one that is used to detect DNA methylase activity in vitro.</p>
    <p>(2) The reaction mixture can also be passed through a centri-sep column (gel filtration column from Princeton Separations, Inc.) for fast efficient purification of large molecules from small molecules. The procedure is easy, rapid and very efficient. It is currently being used for the removal of excess dye terminators prior to DNA sequencing. The effluent from the column can be directly transferred to a spectrophotometer to detect the fluorescence.</p>
    <p>(3) Alternatively, the proper PCR product can be separated from unincorporated nucleotides by including a hybridization step. The PCR reaction mixture can be transferred to a multi-well plate precoated with oligonucleotides specific (common to both wild-type and mutant gene) for the amplified DNA sequence. After denaturation and hybridization steps, unbound nucleotides can be washed away and the signal remaining in each well quantified. Only the mutant sample show a positive fluorescent signal.</p>
    <p>The availability of these three assay methods makes this approach relatively easy and rapid. A large volume of samples can be screened quickly.</p>
    <p>d) Sequencing of the Mutation Site</p>
    <p>Samples that give a positive signal can be sequenced to confirm and validate the presence of the gene mutation. The size of the digested DNA products give the location of the gene mutation. By using appropriate primers, the presence of the gene mutation in the cleaved products can be confirmed by DNA sequencing.</p>
    <heading>12. Zinc Finger-antibody Fusions</heading> <p>The present invention also relates to use of sequence-specific DNA binding domains to create novel molecules that are compatible with simple detection methods. One such example of this would be a DNA sequence-specific DNA binding domain (without the DNA cleavage domain) joined to the heavy chain constant region of an immunoglobulin molecule using recombinant DNA technology.</p>
    <p>These sequence-specific DNA binding monoclonal antibodies can be used for the assays similar to the current use of protein-binding monoclonal antibodies. The DNA is mixed, and then reannealed. These molecules are mixed with DNA fragment containing a suspected DNA sequence of sufficient concentration, through either amplification or purification, and then experimentally assayed for the presence of DNA binding.</p>
    <p>In one example of gene mutation detection, the products of a DNA amplification reaction would be transferred to an ELISA plate pre-coated with a sequence-specific DNA monoclonal antibody. If the DNA in the mixture contains the proper sequence, the DNA fragment will bind the monoclonal antibody molecule and be retained in the ELISA plate. After washing, the bound DNA could be detected.</p>
    <p>One specific example of detection would be with the use of direct fluorescence. During the amplification step of the DNA, fluorescent molecules could be incorporated into the sequence. Another example of detection could be by enzymatic methods. The target DNA could be biotinylated and a secondary incubation with enzyme-conjugated avidin would cause a color change indicating the presence of bound DNA.</p>
    <p>Many variation of this general approach are possible. In one example, rather than using antibody domains, the sequence-specific DNA binding domains could be joined to proteins or peptides that are recognized by monoclonal antibodies. One example here would be to bind the sequence-specific DNA binding domains to protein A or G, which bind specifically to many monoclonal antibodies.</p>
    <p>A second example would be to bind the sequence-specific DNA binding domains to a peptide epitopes such as the FLAG epitope (Kodak) to which other monoclonal antibodies bind. In one example of detection, amplified DNA would be incubated with the sequence-specific DNA binding fusion protein, then transferred to an ELISA plate coated with an appropriate monoclonal antibody. After, washing, the absorbed DNA fragment could be detected as described above.</p>
    <p>Another example would be to use recombinant DNA methods to fuse the sequence-specific DNA binding domain to an enzyme such as alkaline phosphatase. The hybrid enzyme could be used to develop an ELISA assay detection system.</p>
    <p>The following scientific article have been cited throughout the present application and are hereby incorporated by reference in their entirety and relied upon:</p>
    <p>1. Smith H. O. &amp; Wilcox, K. W., <i>J. Mol. Biol. </i>51: 379-391, 1970.</p>
    <p>2. Wilson, G. G., <i>The NEB Transcript </i>5: 1, 1993.</p>
    <p>3. Roberts, R., <i>The NEB Transcript </i>5: 13, 1993.</p>
    <p>4. Li, L., Wu, L. P. &amp; Chandrasegaran, S., <i>Proc. Natl. Acad. Sci U.S.A. </i>89: 4279-4275, 1992.</p>
    <p>5. Li, L., Wu, L. P., Clark R. &amp; Chandrasegaran, S., <i>Gene </i>133: 79-84, 1993.</p>
    <p>6. Li, L. &amp; Chandrasegaran, S., <i>Proc. Natl. Acad. Sci U.S.A. </i>90: 2764-2768, 1993.</p>
    <p>7. Kim, Y.-G., Li, L. &amp; Chandrasegaran, S., <i>J. Biol. Chem. </i>269: 31978-31982, 1994.</p>
    <p>8. Kim, Y.-G. &amp; Chandrasegaran, S., <i>Proc. Natl. Acad. Sci. U.S.A. </i>91: 883-887, 1994.</p>
    <p>9. Berg, J. M., <i>Proc. Natl. Acad. Sci. U.S.A. </i>85: 99-102, 1988.</p>
    <p>10. Pavietich, N. P. &amp; Pabo, C. O., <i>Science </i>252: 809-817, 1992.</p>
    <p>11. Pavietich, N. P. &amp; Pabo, C. O., Science 261: 1701-1707, 1993.</p>
    <p>12. Fairall, L., Schwabe, J. W. R., Chapman, L., Finch, J. T. &amp; Rhodes, <i>Nature </i>(<i>London</i>) 366:483-487, 1993.</p>
    <p>13. Desjarlais, J. R. &amp; Berg, J. M., <i>Proc. Natl. Acad. Sci. U.S.A. </i>89: 7345 7349, 1992.</p>
    <p>14. Desjarlais, J. R. &amp; Berg, J. M., <i>Proc. Natl. Acad. Sci. U.S.A. </i>90: 2256-2260, 1993.</p>
    <p>15. Desjarlais, J. R. &amp; Berg, J. M., <i>Proc. Natl. Acad. Sci. U.S.A. </i>91: 11099-11103, 1994.</p>
    <p>16. Rebar, E. J. &amp; Pabo, C. O., <i>Science </i>263: 671-673, 1994.</p>
    <p>17. Choo, Y. &amp; Klug, A., <i>Proc. Natl. Acad. Sci. U.S.A. </i>91: 11163-11167, 1994.</p>
    <p>18. Choo, Y. &amp; Klug, A., <i>Proc. Natl. Acad. Sci. U.S.A. </i>91: 11168-11172, 1994.</p>
    <p>19. Jamieson, A. C., Kim, S.-H. &amp; Wells, J. A., <i>Biochemistry </i>33: 5689-5695, 1994.</p>
    <p>20. Wu, H., Yang, W.-P. &amp; Barbas III, C. F., <i>Proc. Natl. Acad. Sci. U.S.A. </i>92: 344-348, 1995.</p>
    <p>21. Kita, K., Kotani, H., Sugisaki H. &amp; Takanami, M., <i>J. Biol. Chem. </i>264: 5751-5756, 1989.</p>
    <p>22. Looney, M. C., Moran, L. S., Jack, W. E., Feehery, G. R., Benner, J. S., Slatko, B. E. &amp; Wilson, G. G., <i>Gene </i>80: 193-208, 1989.</p>
    <p>23. Hochuli, E., Dobeli, H. &amp; Schacher, A., <i>J. Chromatogr. </i>411: 177-184, 1987.</p>
    <p>24. Laemmli, U. K., <i>Nature </i>(<i>London</i>) 222: 680-685, 1970.</p>
    <p>25. Sanger, F., Nicklen S. &amp; Coulson, A. R., <i>Proc. Natl. Acad. Sci. U.S.A. </i>74: 5463-5467, 1977.</p>
    <p>26. Studier, F. W., Rosenberg, A. H., Dunn J. J. &amp; Dubendorff, J. W., <i>Methods in Enzymol. </i>185: 60-89, 1990.</p>
    <p>27. Szybalski, W., Kim, S. C. Hasan, N. &amp; Podhajska, A. J., <i>Gene </i>100: 13-26, 1991.</p>
    <p>28. Waugh, D. S. and Sauer, R. T., <i>Proc. Natl. Acad. Sci. U.S.A. </i>90: 9596-9600, 1993.</p>
    <p>29. Waugh, D. S. and Sauer, R. T., <i>J. Biol. Chem. </i>269: 12298-12303, 1994.</p>
    <p>30. Smith, W. H. &amp; Huggins, M. B., <i>J. Gen. Microbiol. </i>128: 307-318, 1982.</p>
    <p>31. Smith, W. H., Huggins, M. B. &amp; Shaw, K. M., <i>J. Gen Microbiol. </i>133: 1111-1126, 1987.</p>
    <p>32. Soothill, J. S., <i>Burns </i>20: 209-211, 1994.</p>
    <p>33. Rhodes, D. &amp; Klug, A., <i>Sci. Am. </i>268: 56-59, 1993.</p>
    <p>34. Shi, Y. &amp; Berg, J. M., <i>Science </i>268: 282-284, 1995.</p>
    <p>35. Hochuli, E., Dobeli, H. &amp; Schacher, A., <i>J. Chromatogr. </i>411: 177-184, 1987.</p>
    <p>36. Maxam, A. M. &amp; Gilbert, W., <i>Proc. Natl. Acad. Sci. U.S.A. </i>74: 560-564, 1977.</p>
    <p>37. Birkenbihl, R. P. &amp; Vielmetter, W., <i>Mol. Gen. Genet. </i>220: 147-153, 1989.</p>
    <p>38. Maniatis, T., Fritsch, E. F. and Sambrook, J., <i>Molecular Cloning, A Laboratory Manual, </i>Cold Spring Harbor Laboratory, New York, 1982.</p>
    <p>39. Chang A. C. Y. &amp; Cohen, S. N., <i>J. Bacteriol. </i>134: 1141-1156, 1978.</p>
    <p>40. Lau &amp; Wright, <i>Lancet </i>342: 1335, 1993.</p>
    <p>While the invention has been described in connection with what is presently considered to be the most practical and preferred embodiments, it is to be understood that the invention is not limited to the disclosed embodiments, but on the contrary is intended to cover various modifications and equivalent arrangements included within the spirit and scope of the appended claims.</p>
    <p>Thus, it is to be understood that variations in the present invention can be made without departing from the novel aspects of this invention as defined in the claims.</p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup cols="1" colsep="0" rowsep="0" align="left"> <colspec colname="1" align="center" colwidth="217PT"> </colspec> <thead valign="bottom"> <tr class="description-tr"> <td namest="1" nameend="1" morerows="0" rowsep="1" valign="top" class="description-td" colspan="1">TABLE I</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" morerows="0" rowsep="1" valign="top" align="center" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td">Zinc fingers specific for triplets</td>
              </tr> </tbody> </tgroup> <tgroup cols="3" colsep="0" rowsep="0" align="left"> <colspec colname="1" align="center" colwidth="91PT"> </colspec> <colspec colname="2" align="center" colwidth="98PT"> </colspec> <colspec colname="3" align="center" colwidth="28PT"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td"> </td>
                <td morerows="0" valign="top" class="description-td">Amino Acid Sequence of</td>
                <td morerows="0" valign="top" class="description-td"> </td>
              </tr> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td"> </td>
                <td morerows="0" valign="top" class="description-td">α-helical Region</td>
              </tr> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td">Triplet</td>
                <td morerows="0" valign="top" class="description-td">Involved in Recognition</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="3" morerows="0" rowsep="1" valign="top" align="center" class="description-td" colspan="3"> </td>
              </tr> </tbody> </tgroup> <tgroup cols="9" colsep="0" rowsep="0" align="left"> <colspec colname="1" align="center" colwidth="91PT"> </colspec> <colspec colname="2" align="center" colwidth="14PT"> </colspec> <colspec colname="3" align="center" colwidth="14PT"> </colspec> <colspec colname="4" align="center" colwidth="14PT"> </colspec> <colspec colname="5" align="center" colwidth="14PT"> </colspec> <colspec colname="6" align="center" colwidth="14PT"> </colspec> <colspec colname="7" align="center" colwidth="14PT"> </colspec> <colspec colname="8" align="center" colwidth="14PT"> </colspec> <colspec colname="9" align="center" colwidth="28PT"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td">TGT</td>
                <td morerows="0" valign="top" class="description-td">-1</td>
                <td morerows="0" valign="top" class="description-td">1</td>
                <td morerows="0" valign="top" class="description-td">2</td>
                <td morerows="0" valign="top" class="description-td">3</td>
                <td morerows="0" valign="top" class="description-td">4</td>
                <td morerows="0" valign="top" class="description-td">5</td>
                <td morerows="0" valign="top" class="description-td">6</td>
                <td morerows="0" valign="top" class="description-td"> </td>
              </tr> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td"> </td>
                <td morerows="0" valign="top" class="description-td">Q</td>
                <td morerows="0" valign="top" class="description-td">S</td>
                <td morerows="0" valign="top" class="description-td">T</td>
                <td morerows="0" valign="top" class="description-td">A</td>
                <td morerows="0" valign="top" class="description-td">S</td>
                <td morerows="0" valign="top" class="description-td">K</td>
                <td morerows="0" valign="top" class="description-td">A</td>
              </tr> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td"> </td>
                <td morerows="0" valign="top" class="description-td">P</td>
                <td morerows="0" valign="top" class="description-td">S</td>
                <td morerows="0" valign="top" class="description-td">T</td>
                <td morerows="0" valign="top" class="description-td">H</td>
                <td morerows="0" valign="top" class="description-td">L</td>
                <td morerows="0" valign="top" class="description-td">Q</td>
                <td morerows="0" valign="top" class="description-td">T</td>
              </tr> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td"> </td>
                <td morerows="0" valign="top" class="description-td">K</td>
                <td morerows="0" valign="top" class="description-td">T</td>
                <td morerows="0" valign="top" class="description-td">S</td>
                <td morerows="0" valign="top" class="description-td">H</td>
                <td morerows="0" valign="top" class="description-td">L</td>
                <td morerows="0" valign="top" class="description-td">R</td>
                <td morerows="0" valign="top" class="description-td">A</td>
              </tr> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td">TGA</td>
                <td morerows="0" valign="top" class="description-td">Q</td>
                <td morerows="0" valign="top" class="description-td">L</td>
                <td morerows="0" valign="top" class="description-td">A</td>
                <td morerows="0" valign="top" class="description-td">H</td>
                <td morerows="0" valign="top" class="description-td">L</td>
                <td morerows="0" valign="top" class="description-td">S</td>
                <td morerows="0" valign="top" class="description-td">T</td>
              </tr> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td"> </td>
                <td morerows="0" valign="top" class="description-td">Q</td>
                <td morerows="0" valign="top" class="description-td">K</td>
                <td morerows="0" valign="top" class="description-td">G</td>
                <td morerows="0" valign="top" class="description-td">H</td>
                <td morerows="0" valign="top" class="description-td">L</td>
                <td morerows="0" valign="top" class="description-td">T</td>
                <td morerows="0" valign="top" class="description-td">E</td>
              </tr> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td">GAA</td>
                <td morerows="0" valign="top" class="description-td">Q</td>
                <td morerows="0" valign="top" class="description-td">G</td>
                <td morerows="0" valign="top" class="description-td">G</td>
                <td morerows="0" valign="top" class="description-td">N</td>
                <td morerows="0" valign="top" class="description-td">L</td>
                <td morerows="0" valign="top" class="description-td">V</td>
                <td morerows="0" valign="top" class="description-td">R</td>
              </tr> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td"> </td>
                <td morerows="0" valign="top" class="description-td">L</td>
                <td morerows="0" valign="top" class="description-td">Q</td>
                <td morerows="0" valign="top" class="description-td">S</td>
                <td morerows="0" valign="top" class="description-td">N</td>
                <td morerows="0" valign="top" class="description-td">L</td>
                <td morerows="0" valign="top" class="description-td">V</td>
                <td morerows="0" valign="top" class="description-td">R</td>
              </tr> <tr class="description-tr"> <td morerows="0" valign="top" class="description-td">GGA</td>
                <td morerows="0" valign="top" class="description-td">Q</td>
                <td morerows="0" valign="top" class="description-td">S</td>
                <td morerows="0" valign="top" class="description-td">D</td>
                <td morerows="0" valign="top" class="description-td">N</td>
                <td morerows="0" valign="top" class="description-td">L</td>
                <td morerows="0" valign="top" class="description-td">Q</td>
                <td morerows="0" valign="top" class="description-td">R</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="9" morerows="0" rowsep="1" valign="top" align="center" class="description-td" colspan="9"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p>
      <sequence-cwu id="SEQLST-1"> <sequence-list-old-rules> <s-1> <s-1-iii>31</s-1-iii> </s-1> <s-2 seq-no="1"> <s-2-i> <s-2-i-a>23 amino acids</s-2-i-a> <s-2-i-b>amino acid</s-2-i-b> <s-2-i-c>&lt;Unknown&gt;</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>peptide</s-2-ii-mt> </s-2-ii> <s-2-vi> <s-2-vi-a>not provided</s-2-vi-a> </s-2-vi> <s-2-xi> 1
Tyr Xaa Cys Xaa Xaa Cys Xaa Xaa Xaa Phe Xaa Xaa Xaa Xaa Xaa Leu
1               5                   10                  15
Xaa Xaa His Xaa Xaa Xaa His
            20</s-2-xi> </s-2> <s-2 seq-no="2"> <s-2-i> <s-2-i-a>24 amino acids</s-2-i-a> <s-2-i-b>amino acid</s-2-i-b> <s-2-i-c>&lt;Unknown&gt;</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>peptide</s-2-ii-mt> </s-2-ii> <s-2-vi> <s-2-vi-a>not provided</s-2-vi-a> </s-2-vi> <s-2-xi> 2
Tyr Xaa Cys Xaa Xaa Cys Xaa Xaa Xaa Phe Xaa Xaa Xaa Xaa Xaa Leu
1               5                   10                  15
Xaa Xaa His Xaa Xaa Xaa Xaa His
            20</s-2-xi> </s-2> <s-2 seq-no="3"> <s-2-i> <s-2-i-a>25 amino acids</s-2-i-a> <s-2-i-b>amino acid</s-2-i-b> <s-2-i-c>&lt;Unknown&gt;</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>peptide</s-2-ii-mt> </s-2-ii> <s-2-vi> <s-2-vi-a>not provided</s-2-vi-a> </s-2-vi> <s-2-xi> 3
Tyr Xaa Cys Xaa Xaa Cys Xaa Xaa Xaa Phe Xaa Xaa Xaa Xaa Xaa Leu
1               5                   10                  15
Xaa Xaa His Xaa Xaa Xaa Xaa Xaa His
            20                  25</s-2-xi> </s-2> <s-2 seq-no="4"> <s-2-i> <s-2-i-a>24 amino acids</s-2-i-a> <s-2-i-b>amino acid</s-2-i-b> <s-2-i-c>&lt;Unknown&gt;</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>peptide</s-2-ii-mt> </s-2-ii> <s-2-vi> <s-2-vi-a>not provided</s-2-vi-a> </s-2-vi> <s-2-xi> 4
Tyr Xaa Cys Xaa Xaa Xaa Cys Xaa Xaa Xaa Phe Xaa Xaa Xaa Xaa Xaa
1               5                   10                  15
Leu Xaa Xaa His Xaa Xaa Xaa His
            20</s-2-xi> </s-2> <s-2 seq-no="5"> <s-2-i> <s-2-i-a>25 amino acids</s-2-i-a> <s-2-i-b>amino acid</s-2-i-b> <s-2-i-c>&lt;Unknown&gt;</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>peptide</s-2-ii-mt> </s-2-ii> <s-2-vi> <s-2-vi-a>not provided</s-2-vi-a> </s-2-vi> <s-2-xi> 5
Tyr Xaa Cys Xaa Xaa Xaa Cys Xaa Xaa Xaa Phe Xaa Xaa Xaa Xaa Xaa
1               5                   10                  15
Leu Xaa Xaa His Xaa Xaa Xaa Xaa His
            20                  25</s-2-xi> </s-2> <s-2 seq-no="6"> <s-2-i> <s-2-i-a>26 amino acids</s-2-i-a> <s-2-i-b>amino acid</s-2-i-b> <s-2-i-c>&lt;Unknown&gt;</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>peptide</s-2-ii-mt> </s-2-ii> <s-2-vi> <s-2-vi-a>not provided</s-2-vi-a> </s-2-vi> <s-2-xi> 6
Tyr Xaa Cys Xaa Xaa Xaa Cys Xaa Xaa Xaa Phe Xaa Xaa Xaa Xaa Xaa
1               5                   10                  15
Leu Xaa Xaa His Xaa Xaa Xaa Xaa Xaa His
            20                  25</s-2-xi> </s-2> <s-2 seq-no="7"> <s-2-i> <s-2-i-a>25 amino acids</s-2-i-a> <s-2-i-b>amino acid</s-2-i-b> <s-2-i-c>&lt;Unknown&gt;</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>peptide</s-2-ii-mt> </s-2-ii> <s-2-vi> <s-2-vi-a>not provided</s-2-vi-a> </s-2-vi> <s-2-xi> 7
Tyr Xaa Cys Xaa Xaa Xaa Xaa Cys Xaa Xaa Xaa Phe Xaa Xaa Xaa Xaa
1               5                   10                  15
Xaa Leu Xaa Xaa His Xaa Xaa Xaa His
            20                  25</s-2-xi> </s-2> <s-2 seq-no="8"> <s-2-i> <s-2-i-a>26 amino acids</s-2-i-a> <s-2-i-b>amino acid</s-2-i-b> <s-2-i-c>&lt;Unknown&gt;</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>peptide</s-2-ii-mt> </s-2-ii> <s-2-vi> <s-2-vi-a>not provided</s-2-vi-a> </s-2-vi> <s-2-xi> 8
Tyr Xaa Cys Xaa Xaa Xaa Xaa Cys Xaa Xaa Xaa Phe Xaa Xaa Xaa Xaa
1               5                   10                  15
Xaa Leu Xaa Xaa His Xaa Xaa Xaa Xaa His
            20                  25</s-2-xi> </s-2> <s-2 seq-no="9"> <s-2-i> <s-2-i-a>27 amino acids</s-2-i-a> <s-2-i-b>amino acid</s-2-i-b> <s-2-i-c>&lt;Unknown&gt;</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>peptide</s-2-ii-mt> </s-2-ii> <s-2-vi> <s-2-vi-a>not provided</s-2-vi-a> </s-2-vi> <s-2-xi> 9
Tyr Xaa Cys Xaa Xaa Xaa Xaa Cys Xaa Xaa Xaa Phe Xaa Xaa Xaa Xaa
1               5                   10                  15
Xaa Leu Xaa Xaa His Xaa Xaa Xaa Xaa Xaa His
            20                  25</s-2-xi> </s-2> <s-2 seq-no="10"> <s-2-i> <s-2-i-a>23 amino acids</s-2-i-a> <s-2-i-b>amino acid</s-2-i-b> <s-2-i-c>&lt;Unknown&gt;</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>peptide</s-2-ii-mt> </s-2-ii> <s-2-vi> <s-2-vi-a>not provided</s-2-vi-a> </s-2-vi> <s-2-xi> 10
Phe Xaa Cys Xaa Xaa Cys Xaa Xaa Xaa Phe Xaa Xaa Xaa Xaa Xaa Leu
1               5                   10                  15
Xaa Xaa His Xaa Xaa Xaa His
            20</s-2-xi> </s-2> <s-2 seq-no="11"> <s-2-i> <s-2-i-a>24 amino acids</s-2-i-a> <s-2-i-b>amino acid</s-2-i-b> <s-2-i-c>&lt;Unknown&gt;</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>peptide</s-2-ii-mt> </s-2-ii> <s-2-vi> <s-2-vi-a>not provided</s-2-vi-a> </s-2-vi> <s-2-xi> 11
Phe Xaa Cys Xaa Xaa Cys Xaa Xaa Xaa Phe Xaa Xaa Xaa Xaa Xaa Leu
1               5                   10                  15
Xaa Xaa His Xaa Xaa Xaa Xaa His
            20</s-2-xi> </s-2> <s-2 seq-no="12"> <s-2-i> <s-2-i-a>25 amino acids</s-2-i-a> <s-2-i-b>amino acid</s-2-i-b> <s-2-i-c>&lt;Unknown&gt;</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>peptide</s-2-ii-mt> </s-2-ii> <s-2-vi> <s-2-vi-a>not provided</s-2-vi-a> </s-2-vi> <s-2-xi> 12
Phe Xaa Cys Xaa Xaa Cys Xaa Xaa Xaa Phe Xaa Xaa Xaa Xaa Xaa Leu
1               5                   10                  15
Xaa Xaa His Xaa Xaa Xaa Xaa Xaa His
            20                  25</s-2-xi> </s-2> <s-2 seq-no="13"> <s-2-i> <s-2-i-a>24 amino acids</s-2-i-a> <s-2-i-b>amino acid</s-2-i-b> <s-2-i-c>&lt;Unknown&gt;</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>peptide</s-2-ii-mt> </s-2-ii> <s-2-vi> <s-2-vi-a>not provided</s-2-vi-a> </s-2-vi> <s-2-xi> 13
Phe Xaa Cys Xaa Xaa Xaa Cys Xaa Xaa Xaa Phe Xaa Xaa Xaa Xaa Xaa
1               5                   10                  15
Leu Xaa Xaa His Xaa Xaa Xaa His
            20</s-2-xi> </s-2> <s-2 seq-no="14"> <s-2-i> <s-2-i-a>25 amino acids</s-2-i-a> <s-2-i-b>amino acid</s-2-i-b> <s-2-i-c>&lt;Unknown&gt;</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>peptide</s-2-ii-mt> </s-2-ii> <s-2-vi> <s-2-vi-a>not provided</s-2-vi-a> </s-2-vi> <s-2-xi> 14
Phe Xaa Cys Xaa Xaa Xaa Cys Xaa Xaa Xaa Phe Xaa Xaa Xaa Xaa Xaa
1               5                   10                  15
Leu Xaa Xaa His Xaa Xaa Xaa Xaa His
            20                  25</s-2-xi> </s-2> <s-2 seq-no="15"> <s-2-i> <s-2-i-a>26 amino acids</s-2-i-a> <s-2-i-b>amino acid</s-2-i-b> <s-2-i-c>&lt;Unknown&gt;</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>peptide</s-2-ii-mt> </s-2-ii> <s-2-vi> <s-2-vi-a>not provided</s-2-vi-a> </s-2-vi> <s-2-xi> 15
Phe Xaa Cys Xaa Xaa Xaa Cys Xaa Xaa Xaa Phe Xaa Xaa Xaa Xaa Xaa
1               5                   10                  15
Leu Xaa Xaa His Xaa Xaa Xaa Xaa Xaa His
            20                  25</s-2-xi> </s-2> <s-2 seq-no="16"> <s-2-i> <s-2-i-a>25 amino acids</s-2-i-a> <s-2-i-b>amino acid</s-2-i-b> <s-2-i-c>&lt;Unknown&gt;</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>peptide</s-2-ii-mt> </s-2-ii> <s-2-vi> <s-2-vi-a>not provided</s-2-vi-a> </s-2-vi> <s-2-xi> 16
Phe Xaa Cys Xaa Xaa Xaa Xaa Cys Xaa Xaa Xaa Phe Xaa Xaa Xaa Xaa
1               5                   10                  15
Xaa Leu Xaa Xaa His Xaa Xaa Xaa His
            20                  25</s-2-xi> </s-2> <s-2 seq-no="17"> <s-2-i> <s-2-i-a>26 amino acids</s-2-i-a> <s-2-i-b>amino acid</s-2-i-b> <s-2-i-c>&lt;Unknown&gt;</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>peptide</s-2-ii-mt> </s-2-ii> <s-2-vi> <s-2-vi-a>not provided</s-2-vi-a> </s-2-vi> <s-2-xi> 17
Phe Xaa Cys Xaa Xaa Xaa Xaa Cys Xaa Xaa Xaa Phe Xaa Xaa Xaa Xaa
1               5                   10                  15
Xaa Leu Xaa Xaa His Xaa Xaa Xaa Xaa His
            20                  25</s-2-xi> </s-2> <s-2 seq-no="18"> <s-2-i> <s-2-i-a>27 amino acids</s-2-i-a> <s-2-i-b>amino acid</s-2-i-b> <s-2-i-c>&lt;Unknown&gt;</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>peptide</s-2-ii-mt> </s-2-ii> <s-2-vi> <s-2-vi-a>not provided</s-2-vi-a> </s-2-vi> <s-2-xi> 18
Phe Xaa Cys Xaa Xaa Xaa Xaa Cys Xaa Xaa Xaa Phe Xaa Xaa Xaa Xaa
1               5                   10                  15
Xaa Leu Xaa Xaa His Xaa Xaa Xaa Xaa Xaa His
            20                  25</s-2-xi> </s-2> <s-2 seq-no="19"> <s-2-i> <s-2-i-a>23 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA (genomic)</s-2-ii-mt> </s-2-ii> <s-2-vi> <s-2-vi-a>not provided</s-2-vi-a> </s-2-vi> <s-2-xi> 19
CCCCTGAAGG AGATATACAT ATG                                             23</s-2-xi> </s-2> <s-2 seq-no="20"> <s-2-i> <s-2-i-a>25 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA (genomic)</s-2-ii-mt> </s-2-ii> <s-2-vi> <s-2-vi-a>not provided</s-2-vi-a> </s-2-vi> <s-2-xi> 20
GGACTAGTCC CTTCTTATTC TGGTG                                           25</s-2-xi> </s-2> <s-2 seq-no="21"> <s-2-i> <s-2-i-a>15 amino acids</s-2-i-a> <s-2-i-b>amino acid</s-2-i-b> <s-2-i-c>&lt;Unknown&gt;</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>peptide</s-2-ii-mt> </s-2-ii> <s-2-vi> <s-2-vi-a>not provided</s-2-vi-a> </s-2-vi> <s-2-xi> 21
Gly Gly Gly Gly Ser Gly Gly Gly Gly Ser Gly Gly Gly Gly Ser
1               5                   10                  15</s-2-xi> </s-2> <s-2 seq-no="22"> <s-2-i> <s-2-i-a>14 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA (genomic)</s-2-ii-mt> </s-2-ii> <s-2-vi> <s-2-vi-a>not provided</s-2-vi-a> </s-2-vi> <s-2-xi> 22
CTGACGGGGG CCAA                                                       14</s-2-xi> </s-2> <s-2 seq-no="23"> <s-2-i> <s-2-i-a>15 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA (genomic)</s-2-ii-mt> </s-2-ii> <s-2-vi> <s-2-vi-a>not provided</s-2-vi-a> </s-2-vi> <s-2-xi> 23
TGCCCCCGGT TGATC                                                      15</s-2-xi> </s-2> <s-2 seq-no="24"> <s-2-i> <s-2-i-a>24 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA (genomic)</s-2-ii-mt> </s-2-ii> <s-2-vi> <s-2-vi-a>not provided</s-2-vi-a> </s-2-vi> <s-2-xi> 24
CTAGACGGGG GAGGCGGCAG TCAA                                            24</s-2-xi> </s-2> <s-2 seq-no="25"> <s-2-i> <s-2-i-a>24 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA (genomic)</s-2-ii-mt> </s-2-ii> <s-2-vi> <s-2-vi-a>not provided</s-2-vi-a> </s-2-vi> <s-2-xi> 25
TGCCCCCTCC GCCGTCAGTT GATC                                            24</s-2-xi> </s-2> <s-2 seq-no="26"> <s-2-i> <s-2-i-a>5 amino acids</s-2-i-a> <s-2-i-b>amino acid</s-2-i-b> <s-2-i-c>&lt;Unknown&gt;</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>peptide</s-2-ii-mt> </s-2-ii> <s-2-vi> <s-2-vi-a>not provided</s-2-vi-a> </s-2-vi> <s-2-xi> 26
Gly Gly Gly Gly Ser
1               5</s-2-xi> </s-2> <s-2 seq-no="27"> <s-2-i> <s-2-i-a>39 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA (genomic)</s-2-ii-mt> </s-2-ii> <s-2-vi> <s-2-vi-a>not provided</s-2-vi-a> </s-2-vi> <s-2-xi> 27
CTAGACGGGG GAGGCGGCAG TGGAGGTGGC GGATCACAA                            39</s-2-xi> </s-2> <s-2 seq-no="28"> <s-2-i> <s-2-i-a>39 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA (genomic)</s-2-ii-mt> </s-2-ii> <s-2-vi> <s-2-vi-a>not provided</s-2-vi-a> </s-2-vi> <s-2-xi> 28
TGCCCCCTCC GCCGTCACCT CCACCGCCTA GTGTTGATC                            39</s-2-xi> </s-2> <s-2 seq-no="29"> <s-2-i> <s-2-i-a>10 amino acids</s-2-i-a> <s-2-i-b>amino acid</s-2-i-b> <s-2-i-c>&lt;Unknown&gt;</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>peptide</s-2-ii-mt> </s-2-ii> <s-2-vi> <s-2-vi-a>not provided</s-2-vi-a> </s-2-vi> <s-2-xi> 29
Gly Gly Gly Gly Ser Gly Gly Gly Gly Ser
1               5                   10</s-2-xi> </s-2> <s-2 seq-no="30"> <s-2-i> <s-2-i-a>17 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA (genomic)</s-2-ii-mt> </s-2-ii> <s-2-vi> <s-2-vi-a>not provided</s-2-vi-a> </s-2-vi> <s-2-xi> 30
CTCCTGAGGA GAAGTCT                                                    17</s-2-xi> </s-2> <s-2 seq-no="31"> <s-2-i> <s-2-i-a>17 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA (genomic)</s-2-ii-mt> </s-2-ii> <s-2-vi> <s-2-vi-a>not provided</s-2-vi-a> </s-2-vi> <s-2-xi> 31
CTCCTGTGGA GAAGTCT                                                    17</s-2-xi> </s-2> </sequence-list-old-rules> </sequence-cwu> </p>
    </div></div></div><div class="patent-section patent-tabular-section"><a id="backward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Cited Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5792640">US5792640</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Dec 20, 1995</td><td class="patent-data-table-td patent-date-value">Aug 11, 1998</td><td class="patent-data-table-td ">The Johns Hopkins University</td><td class="patent-data-table-td ">General method to clone hybrid restriction endonucleases using lig gene</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="forward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Referenced by</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Citing Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7163824">US7163824</a></td><td class="patent-data-table-td patent-date-value">Aug 15, 2002</td><td class="patent-data-table-td patent-date-value">Jan 16, 2007</td><td class="patent-data-table-td ">Sangamo Biosciences, Inc.</td><td class="patent-data-table-td ">Cell containing nucleic acid molecule comprising polynucleotide sequence encoding engineered zinc finger protein operably linked to promoter, contacting target site in endogenous cellular gene with protein to inhibit transcription</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7407776">US7407776</a></td><td class="patent-data-table-td patent-date-value">Sep 17, 2004</td><td class="patent-data-table-td patent-date-value">Aug 5, 2008</td><td class="patent-data-table-td ">Sangamo Biosciences, Inc.</td><td class="patent-data-table-td ">Engineered zinc finger proteins for regulation of gene expression</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7888121">US7888121</a></td><td class="patent-data-table-td patent-date-value">Aug 6, 2004</td><td class="patent-data-table-td patent-date-value">Feb 15, 2011</td><td class="patent-data-table-td ">Sangamo Biosciences, Inc.</td><td class="patent-data-table-td ">Cleaving cellular chromatin and inserting expression vector comprising nucleotide sequences coding zinc finger-cleavage half domain genetic exchange; genome engineering and homologous recombination</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7972854">US7972854</a></td><td class="patent-data-table-td patent-date-value">Feb 3, 2005</td><td class="patent-data-table-td patent-date-value">Jul 5, 2011</td><td class="patent-data-table-td ">Sangamo Biosciences, Inc.</td><td class="patent-data-table-td ">Methods and compositions for targeted cleavage and recombination</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7985887">US7985887</a></td><td class="patent-data-table-td patent-date-value">Aug 17, 2006</td><td class="patent-data-table-td patent-date-value">Jul 26, 2011</td><td class="patent-data-table-td ">Sangamo Biosciences, Inc.</td><td class="patent-data-table-td ">Regulation of endogenous gene expression in cells using zinc finger proteins</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8021867">US8021867</a></td><td class="patent-data-table-td patent-date-value">Oct 18, 2006</td><td class="patent-data-table-td patent-date-value">Sep 20, 2011</td><td class="patent-data-table-td ">Duke University</td><td class="patent-data-table-td ">Rationally-designed meganucleases with altered sequence specificity and DNA-binding affinity</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8106255">US8106255</a></td><td class="patent-data-table-td patent-date-value">Jan 22, 2003</td><td class="patent-data-table-td patent-date-value">Jan 31, 2012</td><td class="patent-data-table-td ">Dana Carroll</td><td class="patent-data-table-td ">Targeted chromosomal mutagenasis using zinc finger nucleases</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8119361">US8119361</a></td><td class="patent-data-table-td patent-date-value">Sep 26, 2011</td><td class="patent-data-table-td patent-date-value">Feb 21, 2012</td><td class="patent-data-table-td ">Duke University</td><td class="patent-data-table-td ">Methods of cleaving DNA with rationally-designed meganucleases</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8119381">US8119381</a></td><td class="patent-data-table-td patent-date-value">Sep 26, 2011</td><td class="patent-data-table-td patent-date-value">Feb 21, 2012</td><td class="patent-data-table-td ">Duke University</td><td class="patent-data-table-td ">Rationally-designed meganucleases with altered sequence specificity and DNA-binding affinity</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8124369">US8124369</a></td><td class="patent-data-table-td patent-date-value">Sep 27, 2011</td><td class="patent-data-table-td patent-date-value">Feb 28, 2012</td><td class="patent-data-table-td ">Duke University</td><td class="patent-data-table-td ">Method of cleaving DNA with rationally-designed meganucleases</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8129134">US8129134</a></td><td class="patent-data-table-td patent-date-value">Sep 27, 2011</td><td class="patent-data-table-td patent-date-value">Mar 6, 2012</td><td class="patent-data-table-td ">Duke University</td><td class="patent-data-table-td ">Methods of cleaving DNA with rationally-designed meganucleases</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8133697">US8133697</a></td><td class="patent-data-table-td patent-date-value">Sep 27, 2011</td><td class="patent-data-table-td patent-date-value">Mar 13, 2012</td><td class="patent-data-table-td ">Duke University</td><td class="patent-data-table-td ">Methods of cleaving DNA with rationally-designed meganucleases</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8143015">US8143015</a></td><td class="patent-data-table-td patent-date-value">Sep 26, 2011</td><td class="patent-data-table-td patent-date-value">Mar 27, 2012</td><td class="patent-data-table-td ">Duke University</td><td class="patent-data-table-td ">Methods of cleaving DNA with rationally-designed meganucleases</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8143016">US8143016</a></td><td class="patent-data-table-td patent-date-value">Sep 27, 2011</td><td class="patent-data-table-td patent-date-value">Mar 27, 2012</td><td class="patent-data-table-td ">Duke University</td><td class="patent-data-table-td ">Methods of cleaving DNA with rationally-designed meganucleases</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8148098">US8148098</a></td><td class="patent-data-table-td patent-date-value">Sep 27, 2011</td><td class="patent-data-table-td patent-date-value">Apr 3, 2012</td><td class="patent-data-table-td ">Duke University</td><td class="patent-data-table-td ">Methods of cleaving DNA with rationally-designed meganucleases</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8163514">US8163514</a></td><td class="patent-data-table-td patent-date-value">Sep 27, 2011</td><td class="patent-data-table-td patent-date-value">Apr 24, 2012</td><td class="patent-data-table-td ">Duke University</td><td class="patent-data-table-td ">Methods of cleaving DNA with rationally-designed meganucleases</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8268618">US8268618</a></td><td class="patent-data-table-td patent-date-value">Aug 16, 2006</td><td class="patent-data-table-td patent-date-value">Sep 18, 2012</td><td class="patent-data-table-td ">Sangamo Biosciences, Inc.</td><td class="patent-data-table-td ">Repressing endogenous CCR5 gene expression in cells using engineered zinc finger proteins</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8304222">US8304222</a></td><td class="patent-data-table-td patent-date-value">Apr 26, 2012</td><td class="patent-data-table-td patent-date-value">Nov 6, 2012</td><td class="patent-data-table-td ">Duke University</td><td class="patent-data-table-td ">Rationally-designed meganucleases with altered sequence specificity and heterodimer formation</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8338157">US8338157</a></td><td class="patent-data-table-td patent-date-value">Sep 10, 2010</td><td class="patent-data-table-td patent-date-value">Dec 25, 2012</td><td class="patent-data-table-td ">Precision Biosciences, Inc.</td><td class="patent-data-table-td ">Rationally-designed meganuclease variants of lig-34 and I-crei for maize genome engineering</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8349810">US8349810</a></td><td class="patent-data-table-td patent-date-value">Jun 22, 2011</td><td class="patent-data-table-td patent-date-value">Jan 8, 2013</td><td class="patent-data-table-td ">Sangamo Biosciences, Inc.</td><td class="patent-data-table-td ">Methods for targeted cleavage and recombination of CCR5</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8377674">US8377674</a></td><td class="patent-data-table-td patent-date-value">Jun 25, 2012</td><td class="patent-data-table-td patent-date-value">Feb 19, 2013</td><td class="patent-data-table-td ">Duke University</td><td class="patent-data-table-td ">Method for producing genetically-modified cells with rationally-designed meganucleases with altered sequence specificity</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8409861">US8409861</a></td><td class="patent-data-table-td patent-date-value">Dec 15, 2005</td><td class="patent-data-table-td patent-date-value">Apr 2, 2013</td><td class="patent-data-table-td ">Sangamo Biosciences, Inc.</td><td class="patent-data-table-td ">Targeted deletion of cellular DNA sequences</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8524500">US8524500</a></td><td class="patent-data-table-td patent-date-value">Jul 16, 2010</td><td class="patent-data-table-td patent-date-value">Sep 3, 2013</td><td class="patent-data-table-td ">Sangamo Biosciences, Inc.</td><td class="patent-data-table-td ">Methods and compositions for targeted cleavage and recombination</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP1581610A2?cl=en">EP1581610A2</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Sep 5, 2003</td><td class="patent-data-table-td patent-date-value">Oct 5, 2005</td><td class="patent-data-table-td ">California Institute of Technology</td><td class="patent-data-table-td ">Use of chimeric nucleases to stimulate gene targeting</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2292274A1?cl=en">EP2292274A1</a></td><td class="patent-data-table-td patent-date-value">Sep 8, 2005</td><td class="patent-data-table-td patent-date-value">Mar 9, 2011</td><td class="patent-data-table-td ">Sangamo BioSciences, Inc.</td><td class="patent-data-table-td ">Compositions and methods for protein production</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2522723A1?cl=en">EP2522723A1</a></td><td class="patent-data-table-td patent-date-value">Jan 28, 2004</td><td class="patent-data-table-td patent-date-value">Nov 14, 2012</td><td class="patent-data-table-td ">Cellectis</td><td class="patent-data-table-td ">Custom-made meganuclease and use thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2559759A1?cl=en">EP2559759A1</a></td><td class="patent-data-table-td patent-date-value">Jan 28, 2004</td><td class="patent-data-table-td patent-date-value">Feb 20, 2013</td><td class="patent-data-table-td ">Cellectis</td><td class="patent-data-table-td ">Custom-made meganuclease and use thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2004067736A2?cl=en">WO2004067736A2</a></td><td class="patent-data-table-td patent-date-value">Jan 28, 2004</td><td class="patent-data-table-td patent-date-value">Aug 12, 2004</td><td class="patent-data-table-td ">Cellectis</td><td class="patent-data-table-td ">Custom-made meganuclease and use thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2005014791A2?cl=en">WO2005014791A2</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Aug 6, 2004</td><td class="patent-data-table-td patent-date-value">Feb 17, 2005</td><td class="patent-data-table-td ">Michael C Holmes</td><td class="patent-data-table-td ">Methods and compositions for targeted cleavage and recombination</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="classifications"></a><div class="patent-section-header"><span class="patent-section-title">Classifications</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th"> </th><th class="patent-data-table-th"> </th></tr></thead><tr><td class="patent-data-table-td ">U.S. Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=DZhVBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S199000">435/199</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=DZhVBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S023100">536/23.1</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=DZhVBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S193000">435/193</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=DZhVBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S023200">536/23.2</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=DZhVBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S069700">435/69.7</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=DZhVBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S091100">435/91.1</a></span></td></tr><tr><td class="patent-data-table-td ">International Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=DZhVBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12P0021040000">C12P21/04</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=DZhVBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12Q0001680000">C12Q1/68</a></span></td></tr><tr><td class="patent-data-table-td ">Cooperative Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=DZhVBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N9/22">C12N9/22</a></span></td></tr><tr><td class="patent-data-table-td ">European Classification</td><td class="patent-data-table-td "><span class="nested-value">C12N9/22</span></td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="legal-events"></a><div class="patent-section-header"><span class="patent-section-title">Legal Events</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Date</th><th class="patent-data-table-th">Code</th><th class="patent-data-table-th">Event</th><th class="patent-data-table-th">Description</th></tr></thead><tr><td class="patent-data-table-td patent-date-value">Jan 24, 2013</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">12</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Apr 19, 2011</td><td class="patent-data-table-td ">B1</td><td class="patent-data-table-td ">Reexamination certificate first reexamination</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">CLAIM 3 IS CANCELLED. CLAIMS 1-2 ARE DETERMINED TO BE PATENTABLE AS AMENDED. CLAIM 8, DEPENDENT ON AN AMENDED CLAIM, IS DETERMINED TO BE PATENTABLE. CLAIMS 4-7 WERE NOT REEXAMINED.</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Sep 29, 2009</td><td class="patent-data-table-td ">RR</td><td class="patent-data-table-td ">Request for reexamination filed</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20070420</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Dec 30, 2008</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">8</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Jul 21, 2005</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">4</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Jul 21, 2005</td><td class="patent-data-table-td ">SULP</td><td class="patent-data-table-td ">Surcharge for late payment</td><td class="patent-data-table-td "></td></tr><tr><td class="patent-data-table-td patent-date-value">Feb 9, 2005</td><td class="patent-data-table-td ">REMI</td><td class="patent-data-table-td ">Maintenance fee reminder mailed</td><td class="patent-data-table-td "></td></tr><tr><td class="patent-data-table-td patent-date-value">Mar 31, 1999</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">JOHNS HOPKINS UNIVERSITY, MARYLAND</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNOR:CHANDRASEGARAN, SRINIVASAN DR;REEL/FRAME:009878/0401</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">19960801</span></div></td></tr></table><div class="patent-section-footer"></div></div><div class="modal-dialog" id="patent-images-lightbox"><div class="patent-lightbox-controls"><div class="patent-lightbox-rotate-controls"><div class="patent-lightbox-rotation-text">Rotate</div><div class="rotate-icon rotate-ccw-icon"></div><div class="rotate-icon rotate-cw-icon"></div></div><div class="patent-lightbox-index-counter"></div><a class="patent-lightbox-fullsize-link" target="_blank">Original Image</a><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_right.png" alt="Next page"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_left.png" alt="Previous page"width="21" height="21" /></div></div><div class="modal-dialog-content"><div class="patent-lightbox-image-holder"><div class="patent-lightbox-placeholder"></div></div></div></div><script>_OC_initPatentsAtb({image_not_available_html: " Image not available"});</script></div></div></div></td></tr></table><script>(function() {var href = window.location.href;if (href.indexOf('?') !== -1) {var parameters = href.split('?')[1].split('&');for (var i = 0; i < parameters.length; i++) {var param = parameters[i].split('=');if (param[0] == 'focus') {var elem = document.getElementById(param[1]);if (elem) {elem.focus();}}}}})();</script><script>_OC_addFlags({LockSrc:"/books/javascript/lock_8a2b04e7bf975d5171d8e4c0b6365c7a.js", Host:"http://www.google.com/", IsBooksRentalEnabled:1, IsWebstoreDisplayCaseEnabled:1, IsObfuscationEnabled:1, IsBrowsingHistoryEnabled:1, IsWebReaderSvgEnabled:0, IsGeoLayerEnabled:1, IsImageModeNotesEnabled:1, IsCopyMenuItemEnabled:1, IsGiftingEnabled:0, IsWebReaderUniversalPaginatorEnabled:0, IsOfflineBubbleEnabled:1, IsReaderEnabledForPlayRequests:1, IsFutureOnSaleVolumesEnabled:1, IsOfflineRestrictedCopyEnabled:1, IsBooksUnifiedLeftNavEnabled:1, IsRestrictedCopyEnabled:1, IsZipitFolderCollectionEnabled:1, IsEndOfSampleRecommendationsEnabled:1, IsRatingsOnBookcardsEnabled:1, IsAdsDisabled:0, IsIframePageDisplayEnabled:0, IsEmbeddedMediaEnabled:1, IsImageModeAnnotationsEnabled:1, IsMyLibraryGooglePlusEnabled:1, IsImagePageProviderEnabled:0, IsBookcardListPriceSmall:0, IsInternalUser:0, IsBooksShareButtonEnabled:0, IsPreOrdersEnabled:0, IsDisabledRandomBookshelves:0, WebstoreDisplayCasePosition:3});_OC_Run({"enable_p13n":false,"add_vol_to_collection_base_url":"http://www.google.com/patents?op=add\u0026sig=ACfU3U1xozKReeJl8tT02rKeg-khA2G8mA\u0026id=DZhVBAABERAJ","remove_vol_from_collection_base_url":"http://www.google.com/patents?op=remove\u0026sig=ACfU3U1NPqFzTmH0yEJBAdcbsX__Qq9hRA\u0026id=DZhVBAABERAJ","logged_in":false,"p13n_save_user_settings_url":"http://www.google.com/patents?op=edit_user_settings\u0026sig=ACfU3U2lDIudq0wJmoZ4NedWrC_GjdRR7Q","is_cobrand":false,"sign_in_url":"https://www.google.com/accounts/Login?service=\u0026continue=http://www.google.com/patents%3Fhl%3Den\u0026hl=en","is_play_enabled":true}, {"volume_id":"","is_ebook":true,"volumeresult":{"has_flowing_text":false,"has_scanned_text":true,"can_download_pdf":false,"can_download_epub":false,"is_pdf_drm_enabled":false,"is_epub_drm_enabled":false,"download_pdf_url":"http://www.google.com/patents/download/Preparing_delivery_vehicle_containing_nu.pdf?id=DZhVBAABERAJ\u0026output=pdf\u0026sig=ACfU3U2Oen9pN4genN-SprMF_NOOi1t2ig"},"sample_url":"http://www.google.com/patents/reader?id=DZhVBAABERAJ\u0026printsec=frontcover\u0026output=reader\u0026source=gbs_atb_hover","is_browsable":true,"is_public_domain":true}, {});</script><div id="footer_table" style="font-size:83%;text-align:center;position:relative;top:20px;height:4.5em;margin-top:2em"><div style="margin-bottom:8px"><a href=http://www.google.com/><nobr>Google&nbsp;Home</nobr></a> - <a href=//www.google.com/patents/sitemap/><nobr>Sitemap</nobr></a> - <a href=http://www.google.com/googlebooks/uspto.html><nobr>USPTO Bulk Downloads</nobr></a> - <a href=/intl/en/privacy/><nobr>Privacy Policy</nobr></a> - <a href=/intl/en/policies/terms/><nobr>Terms of Service</nobr></a> - <a href=https://support.google.com/faqs/answer/2539193?hl=en><nobr>About Google Patents</nobr></a> - <a href="http://www.google.com/tools/feedback/intl/en/error.html" onclick="try{_OC_startFeedback({productId: '72792',locale: 'en'});return false;}catch(e){}"><nobr>Send Feedback</nobr></a></div><span>Data provided by IFI CLAIMS Patent Services</span><br><span >&copy;2012 Google</span></div> <script type="text/javascript">var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));</script><script type="text/javascript">var pageTracker = _gat._getTracker("UA-27188110-1");pageTracker._setCookiePath("/patents/");pageTracker._trackPageview();</script> </body></html>