<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN""http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html><head><title>Patent US6270954 - Contacting first cell that expresses confonnationally defective target ... - Google Patents</title><script>(function(){(function(){function e(a){this.t={};this.tick=function(a,c,b){var d=void 0!=b?b:(new Date).getTime();this.t[a]=[d,c];if(void 0==b)try{window.console.timeStamp("CSI/"+a)}catch(e){}};this.tick("start",null,a)}var a;window.performance&&(a=window.performance.timing);var f=a?new e(a.responseStart):new e;window.jstiming={Timer:e,load:f};if(a){var c=a.navigationStart,d=a.responseStart;0<c&&d>=c&&(window.jstiming.srt=d-c)}if(a){var b=window.jstiming.load;0<c&&d>=c&&(b.tick("_wtsrt",void 0,c),b.tick("wtsrt_",
"_wtsrt",d),b.tick("tbsd_","wtsrt_"))}try{a=null,window.chrome&&window.chrome.csi&&(a=Math.floor(window.chrome.csi().pageT),b&&0<c&&(b.tick("_tbnd",void 0,window.chrome.csi().startE),b.tick("tbnd_","_tbnd",c))),null==a&&window.gtbExternal&&(a=window.gtbExternal.pageT()),null==a&&window.external&&(a=window.external.pageT,b&&0<c&&(b.tick("_tbnd",void 0,window.external.startE),b.tick("tbnd_","_tbnd",c))),a&&(window.jstiming.pt=a)}catch(g){}})();})();
</script><link rel="stylesheet" href="/patents/css/_8a2b04e7bf975d5171d8e4c0b6365c7a/kl_intl_patents_bundle.css" type="text/css" /><script src="/books/javascript/atb_8a2b04e7bf975d5171d8e4c0b6365c7a__en.js"></script><script>function googleTranslateElementInit() {new google.translate.TranslateElement({pageLanguage: "en",gaTrack: true,gaId: "UA-27188110-1",multilanguagePage: true});}</script><script src="//translate.google.com/translate_a/element.js?cb=googleTranslateElementInit"></script><meta name="DC.type" content="Patent"><meta name="DC.title" content="Contacting first cell that expresses confonnationally defective target protein with protein stabilizing agent to improve conformational defect, thereby improving phenotypic defect of first cell where in congo red is not the stabilizer"><meta name="DC.contributor" content="William J. Welch" scheme="inventor"><meta name="DC.contributor" content="C. Randell Brown" scheme="inventor"><meta name="DC.contributor" content="Jörg Tatzelt" scheme="inventor"><meta name="DC.contributor" content="The Regents Of The University Of California" scheme="assignee"><meta name="DC.date" content="1999-4-13" scheme="dateSubmitted"><meta name="DC.description" content="A method of improving a phenotypic defect in a cell that contains a conformationally defective target protein wherein the conformational defect causes the phenotype defect, comprising contacting a first cell that expresses said confonnationally defective target protein with an amount of a protein stabilizing agent that is effective to improve the conformational defect, thereby improving the phenotypic defect of the first cell in comparison with a second cell having the same conformationally defective target protein and phenotypic defect, wherein the second cell is not contacted with the protein stabilizing agent."><meta name="DC.date" content="2001-8-7" scheme="issued"><meta name="DC.relation" content="US:5276059" scheme="references"><meta name="DC.relation" content="US:5900360" scheme="references"><meta name="citation_reference" content="Back, et al., Increased Thermal Stability of Proteins in the Presence of Sugars and Polyols, Biochemistry, 18:5191-5196 (1979)."><meta name="citation_reference" content="Bilsky, et al., Osmotic Reversal of Temperature Sensitivity in Escherichia coli, Journal of Bacteriology 113:76-81 (1973)."><meta name="citation_reference" content="Brown, et al., Chemical chaperones correct the mutant phenotype of the DELTAF508 cystic fibrosis transmembrane conductance regulator protein, Cell Stress &amp; Chaperones, 1(2), 117-125 (1996)."><meta name="citation_reference" content="Brown, et al., Chemical chaperones correct the mutant phenotype of the ΔF508 cystic fibrosis transmembrane conductance regulator protein, Cell Stress &amp; Chaperones, 1(2), 117-125 (1996)."><meta name="citation_reference" content="Brown, et al., Correcting Temperature-sensitive Protein Folding Defects, J. Clin. Invest., 99:1432-1444 (1997)."><meta name="citation_reference" content="Burg, Molecular basis of osmotic regulation, Walter B. Cannon Lecture, American Physiological Society F983-F996, (1995)."><meta name="citation_reference" content="Bychkova, et al., Folding intermediates are involved in genetic diseases?, Federation of European Biochemical Societies, 359:6-8 (1995)."><meta name="citation_reference" content="Cheng, et al., Functional activation of the cystic fibrosis trafficking mutantDELTAF508-CFTR by overexpression, American Physiological Society, L615-L624 (1995)."><meta name="citation_reference" content="Cheng, et al., Functional activation of the cystic fibrosis trafficking mutantΔF508-CFTR by overexpression, American Physiological Society, L615-L624 (1995)."><meta name="citation_reference" content="Chowdary, et al., Accumulation of p53 in a Mutant Cell Line Defective in the Ubiquitin Pathway, Molecular and Cellular Biology, 14:1997-2003 (1994)."><meta name="citation_reference" content="Denning, et al., Processing of mutant cystic fibrosis transmembrane conductance regulator is temperature-sensitive, Nature, 358:761-764 (1992)."><meta name="citation_reference" content="Edington, et al., Inhibition of Heat Shock (Stress) Protein Induction by Deuterium Oxide and Glycerol: Additional Support for the Abnormal Protein Hypothesis of Induction, Journal of Cellular Phsiology, 139:219-228, (1989)."><meta name="citation_reference" content="Egan, et al., Differential expression of ORCC and CFTR induced by low temperature in CF airway epithelial cells, American Physiological Society, C243-C251 (1995)."><meta name="citation_reference" content="Finley, et al., Thermolability of Ubiquitin-Activating Enzyme from the Mammalian Cell Cycle Mutant ts85, Cell, 37:43-55 (1984)."><meta name="citation_reference" content="Gekko, et al., Mechanism of Protein Stabilization by Glycerol: Preferential Hydration in Glycerol-Water Mixtures, Biochemistry, 20:4667-4676 (1981)."><meta name="citation_reference" content="Gekko, et al., Thermodynamic and Kinetic Examination of Protein Stabilization by Glycerol, Biochemistry, 20:4677-4686 (1981)."><meta name="citation_reference" content="Gerlsma, et al., The Effect of Polyhydric and Monohydric Alcohols on the Heat-Induced Reversible Denaturation of Lysozyme and Ribonuclease, Int. J. Peptide Protein Res., 4:377-383 (1972)."><meta name="citation_reference" content="Ginsberg, et al., Induction of Growth Arrest by a Temperature-Sensitive p53 Mutant Is Correlated with Increased Nuclear Localization and Decreased Stability of the Protein, Molecular and Cellular Biology, 582-585 (1991)."><meta name="citation_reference" content="Gordon, et al., Temperature-sensitive Mutations in the Phage P22 Coat Protein Which Interfere with Polypeptide Chain folding, The Journal of Biological Chemistry, 268:9358-9368 (1993)."><meta name="citation_reference" content="Hawthorne, et al., Osmotic-Remedial Mutants. A New Classification for Nutritional Mutants in Yeast, Genetics, 50:829-839 (1964)."><meta name="citation_reference" content="Henle, et al., Protection against Heat-induced Cell Killing by Polyols in Vitro, Cancer Research, 43:1624-1627 (1983)."><meta name="citation_reference" content="Lin, et al., Modification of Membrane Function, Protein Synthesis, and Heat Killing Effect in Cultured Chinese Hamster Cells by Glycerol and D20, Cancer Research, 44:5776-5784 (1984)."><meta name="citation_reference" content="Lin, et al., Why do Some Organisms Use a Urea-Methylamine Mixture as Osmolyte? Thermodynamic Compensation of Urea and Trimethylamine N -Oxide Interactions with Protein, Biochemistry, 33:12695-12701 (1994)."><meta name="citation_reference" content="Maroney, et al., Cloning and characterization of a thermolabile v-src gene for use in reversible transformation of mammalian cells, Onocogene, 7:1207-1214 (1992)."><meta name="citation_reference" content="Martinez, et al., Cellular localization and cell cycle regulation by a temperature-sensitive p53 protein, Genes &amp; Development, 5:151-159 (1991)."><meta name="citation_reference" content="Mitraki, et al., Global Suppression of Protein Folding Defects and Inclusion Body Formation, Science, 253:54-58 (1991)."><meta name="citation_reference" content="Russell, Temperature-Sensitive Osmotic Remedial Mutants of Escherichia col, Journal of Bacteriology, 112:661-665 (1972)."><meta name="citation_reference" content="Ryan, et al., Alteration of p53 Conformation and Induction of Apoptosis in a Murine Erythroleukemia Cell Line By Dimethylsulfoxide, Leukemia Research, 18:617-621 (1994)."><meta name="citation_reference" content="Santoro, et al., Increased Thermal Stability of Proteins in the Presence of Naturally Occurring Osmolytes, Biochemistry, 31:5278-5283 (1992)."><meta name="citation_reference" content="Sato, et al., Glycerol Reverses the Misfolding Phenotype of the Most Common Cystic Fibrosis Mutation, Journal of Biological Chemistry, 271:635-638 (1996)."><meta name="citation_reference" content="Schein, Solubility as a Function of Protein Structure and Solvent Components, Bio/Technology, 8:308-317 (1990)."><meta name="citation_reference" content="Somero, Protons, osmolytes, and fitness of internal milieu for protein function, American Physiological Society, R197-R213 (1986)."><meta name="citation_reference" content="Tatzelt, et al., Chemical chaperones interfere with the formation of scrapie prion protein, The EMBO Journal, 15:6363-6373 (1996)."><meta name="citation_reference" content="Th&#39;ng, et al., The FT210 Cell Line Is a Mouse G2 Phase Mutant with a Temperature-Sensitive CDC2 Gene Product, Cell, 63:313-324 (1990)."><meta name="citation_reference" content="Thomas, et al., Defective protein folding as a basis of human disease, TIBS 20:456-459 (1995)."><meta name="citation_reference" content="Welch, et al., Influence of molecular and chemical chaperones on protein folding, Cell Stress &amp; Chaperones, 1 (2), 109-115 (1996)."><meta name="citation_reference" content="Yancey, et al., Living with Water Stress: Evolution of Osmolyte Systems, Science, 217:1214-1222 (1982)."><meta name="citation_patent_number" content="US:6270954"><meta name="citation_patent_application_number" content="US:09/291,406"><link rel="canonical" href="http://www.google.com/patents/US6270954"/><meta property="og:url" content="http://www.google.com/patents/US6270954"/><meta name="title" content="Patent US6270954 - Contacting first cell that expresses confonnationally defective target protein with protein stabilizing agent to improve conformational defect, thereby improving phenotypic defect of first cell where in congo red is not the stabilizer"/><meta name="description" content="A method of improving a phenotypic defect in a cell that contains a conformationally defective target protein wherein the conformational defect causes the phenotype defect, comprising contacting a first cell that expresses said confonnationally defective target protein with an amount of a protein stabilizing agent that is effective to improve the conformational defect, thereby improving the phenotypic defect of the first cell in comparison with a second cell having the same conformationally defective target protein and phenotypic defect, wherein the second cell is not contacted with the protein stabilizing agent."/><meta property="og:title" content="Patent US6270954 - Contacting first cell that expresses confonnationally defective target protein with protein stabilizing agent to improve conformational defect, thereby improving phenotypic defect of first cell where in congo red is not the stabilizer"/><meta property="og:type" content="book"/><meta property="og:site_name" content="Google Books"/><meta property="og:image" content="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><link rel="image_src" href="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><script>(function(){try{var aa=function(a,b,c,d){d=d||{};d._sn=["cfg",b,c].join(".");window.gbar.logger.ml(a,d)};var g=window.gbar=window.gbar||{},l=window.gbar.i=window.gbar.i||{},m={},n;function _tvn(a,b){var c=parseInt(a,10);return isNaN(c)?b:c}function _tvf(a,b){var c=parseFloat(a);return isNaN(c)?b:c}function _tvv(a){return!!a}function p(a,b,c){(c||g)[a]=b}g.bv={n:_tvn("2",0),r:"",f:".67.",e:"0",m:_tvn("0",1)};
function q(a,b,c){var d="on"+b;if(a.addEventListener)a.addEventListener(b,c,!1);else if(a.attachEvent)a.attachEvent(d,c);else{var f=a[d];a[d]=function(){var a=f.apply(this,arguments),b=c.apply(this,arguments);return void 0==a?b:void 0==b?a:b&&a}}}var s=function(a){return function(){return g.bv.m==a}},ba=s(1),ca=s(2);p("sb",ba);p("kn",ca);l.a=_tvv;l.b=_tvf;l.c=_tvn;l.i=aa;var da=window.gbar.i.i;var t,u,v,w;function ea(a){v=a}function fa(a){var b;if(b=v&&window.encodeURIComponent)b=a.href,b=!b.match(/^http[s]?:\/\/accounts\.google\.[^/]*\/ClearSID/i)&&!b.match(/^http[s]?:\/\/[^/]*\/accounts\/ClearSID/i);if(b=b&&encodeURIComponent(v()))a.href=a.href.replace(/([?&]continue=)[^&]*/,"$1"+b)}function ga(a){window.gApplication&&(a.href=window.gApplication.getTabUrl(a.href))}
function ha(a){var b=document.forms[0].q,c=window.encodeURIComponent&&b&&b.value,b=b&&b.placeholder;c&&c!=b&&(a.href=a.href.replace(/([?&])q=[^&]*|$/,function(a,b){return(b||"&")+"q="+encodeURIComponent(c)}))}n=l.a("")?ga:ha;
function x(a,b,c,d,f,e){var h=document.getElementById(a);if(h){var k=h.style;k.left=d?"auto":b+"px";k.right=d?b+"px":"auto";k.top=c+"px";k.visibility=u?"hidden":"visible";f&&e?(k.width=f+"px",k.height=e+"px"):(x(t,b,c,d,h.offsetWidth,h.offsetHeight),u=u?"":a)}}
var y=[],ia=function(a,b){y.push(b)},ja=function(a){a=a||window.event;var b=a.target||a.srcElement;a.cancelBubble=!0;null==t&&(a=document.createElement(Array.every||window.createPopup?"iframe":"div"),a.frameBorder="0",t=a.id="gbs",a.src="javascript:''",b.parentNode.appendChild(a),q(document,"click",z));var c=b,b=0;"gb3"!=c.className&&(c=c.parentNode);a=c.getAttribute("aria-owns")||"gbi";var d=c.offsetWidth,f=20<c.offsetTop?46:24;document.getElementById("tphdr")&&(f-=3);var e=!1;do b+=c.offsetLeft||
0;while(c=c.offsetParent);var c=(document.documentElement.clientWidth||document.body.clientWidth)-b-d,h,d=document.body,k=document.defaultView;k&&k.getComputedStyle?(d=k.getComputedStyle(d,""))&&(h=d.direction):h=d.currentStyle?d.currentStyle.direction:d.style.direction;h="rtl"==h;if("gbi"==a){for(d=0;k=y[d++];)k();A(null,window.navExtra);h&&(b=c,e=!0)}else h||(b=c,e=!0);u!=a&&z();x(a,b,f,e)},z=function(){u&&x(u,0,0)},A=function(a,b){var c,d=document.getElementById("gbi"),f=a;f||(f=d.firstChild);
for(;b&&(c=b.pop());){var e=d,h=c,k=f;w||(w="gb2");e.insertBefore(h,k).className=w}},ka=function(a,b,c){if((b=document.getElementById(b))&&a){a.className="gb4";var d=document.createElement("span");d.appendChild(a);d.appendChild(document.createTextNode(" | "));d.id=c;b.appendChild(d)}},la=function(){return document.getElementById("gb_70")},ma=function(){return!!u};p("qs",n);p("setContinueCb",ea);p("pc",fa);p("tg",ja);p("close",z);p("addLink",ka);p("almm",A);p("si",la);p("adh",ia);p("op",ma);var B=function(){},C=function(){},F=function(a){var b=new Image,c=D;b.onerror=b.onload=b.onabort=function(){try{delete E[c]}catch(a){}};E[c]=b;b.src=a;D=c+1},E=[],D=0;p("logger",{il:C,ml:B,log:F});var G=window.gbar.logger;var H={},na={},I=[],oa=l.b("0.1",.1),pa=l.a("1",!0),qa=function(a,b){I.push([a,b])},ra=function(a,b){H[a]=b},sa=function(a){return a in H},J={},K=function(a,b){J[a]||(J[a]=[]);J[a].push(b)},ta=function(a){K("m",a)},L=function(a,b){var c=document.createElement("script");c.src=a;c.async=pa;Math.random()<oa&&(c.onerror=function(){c.onerror=null;B(Error("Bundle load failed: name="+(b||"UNK")+" url="+a))});(document.getElementById("xjsc")||document.getElementsByTagName("body")[0]||
document.getElementsByTagName("head")[0]).appendChild(c)},N=function(a){for(var b=0,c;(c=I[b])&&c[0]!=a;++b);!c||c[1].l||c[1].s||(c[1].s=!0,M(2,a),c[1].url&&L(c[1].url,a),c[1].libs&&m.d&&m.d(c[1].libs))},O=function(a){K("gc",a)},P=null,ua=function(a){P=a},M=function(a,b,c){if(P){a={t:a,b:b};if(c)for(var d in c)a[d]=c[d];try{P(a)}catch(f){}}};p("mdc",H);p("mdi",na);p("bnc",I);p("qGC",O);p("qm",ta);p("qd",J);p("lb",N);p("mcf",ra);p("bcf",qa);p("aq",K);p("mdd","");p("has",sa);
p("trh",ua);p("tev",M);var Q=l.b("0.1",.001),R=0;
function _mlToken(a,b){try{if(1>R){R++;var c,d=a,f=b||{},e=encodeURIComponent,h=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&jexpid=",e("17483"),"&srcpg=",e("prop=22"),"&jsr=",Math.round(1/Q),"&ogev=",e("DK7tU8nfNKixsQS-lYG4Bw"),"&ogf=",g.bv.f,"&ogrp=",e("1"),"&ogv=",e("1407723702.0"),"&oggv="+e("es_plusone_gc_20140723.0_p0"),"&ogd=",e("com"),"&ogc=",e("PAN"),"&ogl=",e("en")];f._sn&&(f._sn="og."+
f._sn);for(var k in f)h.push("&"),h.push(e(k)),h.push("="),h.push(e(f[k]));h.push("&emsg=");h.push(e(d.name+":"+d.message));var r=h.join("");S(r)&&(r=r.substr(0,2E3));c=r;var Aa=window.gbar.logger._aem(a,c);F(Aa)}}catch(Na){}}var S=function(a){return 2E3<=a.length},va=function(a,b){return b};function T(a){B=a;p("_itl",S,G);p("_aem",va,G);p("ml",B,G);a={};H.er=a}l.a("")?T(function(a){throw a;}):l.a("1")&&Math.random()<Q&&T(_mlToken);I.push(["m",{url:"//ssl.gstatic.com/gb/js/scm_7385cc5883250b43a39405734c1bea59.js"}]);g.mcf("c",{});g.sg={c:""};if(l.a("1")){var wa=l.a("");I.push(["gc",{auto:wa,url:"//ssl.gstatic.com/gb/js/abc/gci_91f30755d6a6b787dcc2a4062e6e9824.js",libs:"googleapis.client:plusone:gapi.iframes"}]);var xa={version:"gci_91f30755d6a6b787dcc2a4062e6e9824.js",index:"",lang:"en"};H.gc=xa;var U=function(a){window.googleapis&&window.iframes?a&&a():(a&&O(a),N("gc"))};p("lGC",U);l.a("1")&&p("lPWF",U)};window.__PVT="";if(l.a("1")&&l.a("1")){var V=function(a){U(function(){K("pw",a);N("pw")})};p("lPW",V);I.push(["pw",{url:"//ssl.gstatic.com/gb/js/abc/pwm_45f73e4df07a0e388b0fa1f3d30e7280.js"}]);var W=[],ya=function(a){W[0]=a},za=function(a,b){var c=b||{};c._sn="pw";B(a,c)},Ba={signed:W,elog:za,base:"https://plusone.google.com/u/0",loadTime:(new Date).getTime()};H.pw=Ba;var X=function(a,b){for(var c=b.split("."),d=function(){var b=arguments;a(function(){for(var a=g,d=0,e=c.length-1;d<e;++d)a=a[c[d]];a[c[d]].apply(a,b)})},f=g,e=0,h=c.length-1;e<h;++e)f=
f[c[e]]=f[c[e]]||{};return f[c[e]]=d};X(V,"pw.clk");X(V,"pw.hvr");p("su",ya,g.pw)};function Ca(){function a(){for(var b;(b=e[h++])&&"m"!=b[0]&&!b[1].auto;);b&&(M(2,b[0]),b[1].url&&L(b[1].url,b[0]),b[1].libs&&m.d&&m.d(b[1].libs));h<e.length&&setTimeout(a,0)}function b(){0<f--?setTimeout(b,0):a()}var c=l.a("1"),d=l.a(""),f=3,e=I,h=0,k=window.gbarOnReady;if(k)try{k()}catch(r){da(r,"ml","or")}d?p("ldb",a):c?q(window,"load",b):b()}p("rdl",Ca);var Da={D:1,H:2,da:3,p:4,W:5,M:6,F:7,g:8,ha:9,U:10,L:11,T:12,S:13,N:14,Q:15,P:16,fa:17,w:18,O:19,ga:20,ea:21,u:22,G:23,ja:24,ka:25,ia:26,A:27,j:28,o:29,k:30,ca:31,Z:32,$:33,J:34,K:35,ba:36,aa:37,Y:38,B:39,R:40,v:41,X:42,V:43,h:48,C:49,I:500},Y=[1,2,3,4,5,6,9,10,11,13,14,28,29,30,34,35,37,38,39,40,41,42,43,48,49,500];var Z=l.b("0.001",1E-4),Ea=l.b("1",1),Fa=!1,Ga=!1;if(l.a("1")){var Ha=Math.random();Ha<=Z&&(Fa=!0);Ha<=Ea&&(Ga=!0)}var Ia=Da,$=null;function Ja(){var a=0,b=function(b,d){l.a(d)&&(a|=b)};b(1,"");b(2,"");b(4,"");b(8,"");return a}
function Ka(a,b){var c=Z,d=Fa,f;f=a;if(!$){$={};for(var e=0;e<Y.length;e++){var h=Y[e];$[h]=!0}}if(f=!!$[f])c=Ea,d=Ga;if(d){d=encodeURIComponent;g.rp?(f=g.rp(),f="-1"!=f?f:"1"):f="1";c=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&oge=",a,"&ogex=",d("17483"),"&ogev=",d("DK7tU8nfNKixsQS-lYG4Bw"),"&ogf=",g.bv.f,"&ogp=",d("22"),"&ogrp=",d(f),"&ogsr=",Math.round(1/c),"&ogv=",d("1407723702.0"),"&oggv="+
d("es_plusone_gc_20140723.0_p0"),"&ogd=",d("com"),"&ogl=",d("en"),"&ogc=",d("PAN"),"&ogus=",Ja()];if(b){"ogw"in b&&(c.push("&ogw="+b.ogw),delete b.ogw);var k;f=b;e=[];for(k in f)0!=e.length&&e.push(","),e.push(La(k)),e.push("."),e.push(La(f[k]));k=e.join("");""!=k&&(c.push("&ogad="),c.push(d(k)))}F(c.join(""))}}function La(a){"number"==typeof a&&(a+="");return"string"==typeof a?a.replace(".","%2E").replace(",","%2C"):a}C=Ka;p("il",C,G);var Ma={};H.il=Ma;setTimeout(function(){C(Ia.g)},0);}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var b=window.gbar.i.i;var c=window.gbar;var f=function(d){try{var a=document.getElementById("gbom");a&&d.appendChild(a.cloneNode(!0))}catch(e){b(e,"omas","aomc")}};c.aomc=f;}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var a=window.gbar;a.mcf("pm",{p:""});}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{window.gbar.rdl();}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
if (window['_OC_timingAction']) {window['_OC_timingAction']('patents_refpage');}</script><style>#gbar,#guser{font-size:13px;padding-top:1px !important;}#gbar{float:left;height:22px}#guser{padding-bottom:7px !important;text-align:right}.gbh,.gbd{border-top:1px solid #c9d7f1;font-size:1px}.gbh{height:0;position:absolute;top:24px;width:100%}#gbs,.gbm{background:#fff;left:0;position:absolute;text-align:left;visibility:hidden;z-index:1000}.gbm{border:1px solid;border-color:#c9d7f1 #36c #36c #a2bae7;z-index:1001}.gb1{margin-right:.5em}.gb1,.gb3{zoom:1}.gb2{display:block;padding:.2em .5em}.gb2,.gb3{text-decoration:none !important;border-bottom:none}a.gb1,a.gb4{text-decoration:underline !important}a.gb1,a.gb2,a.gb3,a.gb4{color:#00c !important}.gbi .gb3,.gbi .gb2,.gbi .gb4{color:#dd8e27 !important}.gbf .gb3,.gbf .gb2,.gbf .gb4{color:#900 !important}a.gb2:hover{background:#36c;color:#fff !important}#gbar .gbz0l{color:#000 !important;cursor:default;font-weight:bold;text-decoration:none !important}
#gbar { padding:.3em .6em !important;}</style></head><body  topmargin="3" marginheight="3"><div id=gbar><nobr><a onclick=gbar.qs(this);gbar.logger.il(1,{t:1}); class=gb1 id=gb_1 href="https://www.google.com/search?sa=N&tab=tw">Search</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:2}); class=gb1 id=gb_2 href="http://www.google.com/search?hl=en&tbm=isch&source=og&sa=N&tab=ti">Images</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:8}); class=gb1 id=gb_8 href="http://maps.google.com/maps?hl=en&sa=N&tab=tl">Maps</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:78}); class=gb1 id=gb_78 href="https://play.google.com/?hl=en&sa=N&tab=t8">Play</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:36}); class=gb1 id=gb_36 href="http://www.youtube.com/results?sa=N&tab=t1">YouTube</a> <a onclick=gbar.logger.il(1,{t:5}); class=gb1 id=gb_5 href="http://news.google.com/nwshp?hl=en&tab=tn">News</a> <a onclick=gbar.logger.il(1,{t:23}); class=gb1 id=gb_23 href="https://mail.google.com/mail/?tab=tm">Gmail</a> <a onclick=gbar.logger.il(1,{t:25}); class=gb1 id=gb_25 href="https://drive.google.com/?tab=to">Drive</a> <a class=gb3 href="http://www.google.com/intl/en/options/" onclick="this.blur();gbar.tg(event);return !1" aria-haspopup=true><u>More</u> <small>&#9660;</small></a><div class=gbm id=gbi><a onclick=gbar.logger.il(1,{t:24}); class=gb2 id=gb_24 href="https://www.google.com/calendar?tab=tc">Calendar</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:51}); class=gb2 id=gb_51 href="http://translate.google.com/?hl=en&sa=N&tab=tT">Translate</a><a onclick=gbar.logger.il(1,{t:17}); class=gb2 id=gb_17 href="http://www.google.com/mobile/?hl=en&tab=tD">Mobile</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:10}); class=gb2 id=gb_10 href="http://www.google.com/search?hl=en&tbo=u&tbm=bks&source=og&sa=N&tab=tp">Books</a><a onclick=gbar.logger.il(1,{t:212}); class=gb2 id=gb_212 href="https://wallet.google.com/manage/?tab=ta">Wallet</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:6}); class=gb2 id=gb_6 href="http://www.google.com/search?hl=en&tbo=u&tbm=shop&source=og&sa=N&tab=tf">Shopping</a><a onclick=gbar.logger.il(1,{t:30}); class=gb2 id=gb_30 href="http://www.blogger.com/?tab=tj">Blogger</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:27}); class=gb2 id=gb_27 href="http://www.google.com/finance?sa=N&tab=te">Finance</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:31}); class=gb2 id=gb_31 href="https://plus.google.com/photos?sa=N&tab=tq">Photos</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:12}); class=gb2 id=gb_12 href="http://www.google.com/search?hl=en&tbo=u&tbm=vid&source=og&sa=N&tab=tv">Videos</a><div class=gb2><div class=gbd></div></div><a onclick=gbar.logger.il(1,{t:66}); href="http://www.google.com/intl/en/options/" class=gb2>Even more &raquo;</a></div></nobr></div><div id=guser width=100%><nobr><span id=gbn class=gbi></span><span id=gbf class=gbf></span><span id=gbe></span><a target=_top id=gb_70 href="https://www.google.com/accounts/Login?service=&continue=http://www.google.com/patents%3Fhl%3Den&hl=en" class=gb4>Sign in</a><div style="display: none"><div class=gbm id=gbd5 aria-owner=gbg5><div class=gbmc><ol id=gbom class=gbmcc></ol></div></div></div></nobr></div><div class=gbh style=left:0></div><div class=gbh style=right:0></div><div role="alert" style="position: absolute; left: 0; right: 0;"><a href="http://www.google.com/patents/us6270954?hl=en&amp;output=html_text" title="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."><img border="0" src="http://www.google.com/images/cleardot.gif"alt="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."></a></div><div id="guser"><nobr></nobr></div><div style="clear:both;"></div><div id="gb-top-search-box" class="gb-top-search-box-small gb-reset"><table><tr><td class="logo"><a href="http://www.google.com/patents" class="logo-link"><img class="logo-img" src="/intl/en/images/logos/google_logo_41.png" alt="Go to Google Books Home" height="41"/></a></td><td><form action="http://www.google.com/search" name="f" id="vheadf" method="get"><span id="hf"></span><input type="hidden" name="tbm" value="pts"/><input type="hidden" name="tbo" value="1"/><input type="hidden" name="hl" value="en"/><table><tr><td><div class="inputs"><table><tr><td><div class="text-input"><input type="text" name="q" id="vheadq" class="text" maxlength="2048" size="31" value="" title="Search Patents" accesskey="s" autocomplete="off"/><script>window._OC_autoDir &&window._OC_autoDir('vheadq', 'tia-vheadq');</script></div></td><td><div class="submit-input"><input name="btnG" class="submit" type="submit" value=""/></div></td></tr></table></div></td><td class="col-ext-links"><div class="ext-links"><a href="http://www.google.com/advanced_patent_search">&lt;nobr&gt;Advanced Patent Search&lt;/nobr&gt;</a></div></td></tr></table></form></td></tr></table></div><div class="kd-appbar"><h2 class="kd-appname"><a href="/patents">Patents</a></h2><div class="kd-buttonbar left" id="left-toolbar-buttons"><a id="appbar-write-review-link" href=""></a><a id="appbar-view-print-sample-link" href=""></a><a id="appbar-view-ebook-sample-link" href=""></a><a id="appbar-patents-prior-art-finder-link" href="https://www.google.com/patents/related/US6270954"></a><a id="appbar-patents-discuss-this-link" href="http://www.google.com/url?id=nwxWBAABERAJ&amp;q=http://patents.stackexchange.com/redirect/google-patents%3Fpatent%3DUS6270954&amp;usg=AFQjCNHJtv1V0sGD8BUXl2rmcNk-Yk1goQ" data-is-grant="true"></a><a id="appbar-read-patent-link" href="//docs.google.com/viewer?url=patentimages.storage.googleapis.com/pdfs/US6270954.pdf"></a><a id="appbar-download-pdf-link" href="//patentimages.storage.googleapis.com/pdfs/US6270954.pdf"></a></div><div class="kd-buttonbar right" id="right-toolbar-buttons"></div></div><div id="books-microdata" itemscope=""itemtype="http://schema.org/Book"itemid="http://www.google.com/patents/US6270954" style="display:none"><span itemprop="description">A method of improving a phenotypic defect in a cell that contains a conformationally defective target protein wherein the conformational defect causes the phenotype defect, comprising contacting a first cell that expresses said confonnationally defective target protein with an amount of a protein stabilizing...</span><span itemprop="url">http://www.google.com/patents/US6270954?utm_source=gb-gplus-share</span><span class="main-title" itemprop="name">Patent US6270954 - Contacting first cell that expresses confonnationally defective target protein with protein stabilizing agent to improve conformational defect, thereby improving phenotypic defect of first cell where in congo red is not the stabilizer</span><img itemprop="image" src="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"alt="Patent US6270954 - Contacting first cell that expresses confonnationally defective target protein with protein stabilizing agent to improve conformational defect, thereby improving phenotypic defect of first cell where in congo red is not the stabilizer" title="Patent US6270954 - Contacting first cell that expresses confonnationally defective target protein with protein stabilizing agent to improve conformational defect, thereby improving phenotypic defect of first cell where in congo red is not the stabilizer"></div><div style="display: none"><ol id="ofe-gear-menu-contents" class="gbmcc"><li class="gbe gbmtc"><a class="gbmt goog-menuitem-content" id="" href="http://www.google.com/advanced_patent_search">Advanced Patent Search</a></li></ol></div><table id="viewport_table" cellpadding="0" style="clear:both" cellspacing="0"><tr><td id="viewport_td"><div class=vertical_module_list_row><div id=intl_patents class=about_content><div id=intl_patents_v><table class="patent-bibdata"><tr><td class="patent-bibdata-heading">Publication number</td><td class="single-patent-bibdata">US6270954 B1</td></tr><tr><td class="patent-bibdata-heading">Publication type</td><td class="single-patent-bibdata">Grant</td></tr><tr><td class="patent-bibdata-heading">Application number</td><td class="single-patent-bibdata">US 09/291,406</td></tr><tr><td class="patent-bibdata-heading">Publication date</td><td class="single-patent-bibdata">Aug 7, 2001</td></tr><tr><td class="patent-bibdata-heading">Filing date</td><td class="single-patent-bibdata">Apr 13, 1999</td></tr><tr><td class="patent-bibdata-heading">Priority date<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The priority date is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the date listed."></span></td><td class="single-patent-bibdata">Apr 10, 1996</td></tr><tr><td class="patent-bibdata-heading">Fee status<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The fee status is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the status or dates listed."></span></td><td class="single-patent-bibdata">Lapsed</td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Also published as</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/US6541195">US6541195</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20010021500">US20010021500</a></span></span></td></tr><tr class="patent-bibdata-list-row alternate-patent-number"><td class="patent-bibdata-heading">Publication number</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value">09291406, </span><span class="patent-bibdata-value">291406, </span><span class="patent-bibdata-value">US 6270954 B1, </span><span class="patent-bibdata-value">US 6270954B1, </span><span class="patent-bibdata-value">US-B1-6270954, </span><span class="patent-bibdata-value">US6270954 B1, </span><span class="patent-bibdata-value">US6270954B1</span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Inventors</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22William+J.+Welch%22">William J. Welch</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22C.+Randell+Brown%22">C. Randell Brown</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22J%C3%B6rg+Tatzelt%22">Jörg Tatzelt</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Original Assignee</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=inassignee:%22The+Regents+Of+The+University+Of+California%22">The Regents Of The University Of California</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Export Citation</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/US6270954.bibtex">BiBTeX</a>, </span><span class="patent-bibdata-value"><a href="/patents/US6270954.enw">EndNote</a>, </span><span class="patent-bibdata-value"><a href="/patents/US6270954.ris">RefMan</a></span></span></td></tr><tr class="patent-internal-links"><td colspan=2><span class="patent-bibdata-value"><a href="#backward-citations">Patent Citations</a> (2),</span> <span class="patent-bibdata-value"><a href="#npl-citations">Non-Patent Citations</a> (37),</span> <span class="patent-bibdata-value"><a href="#forward-citations">Referenced by</a> (14),</span> <span class="patent-bibdata-value"><a href="#classifications">Classifications</a> (43),</span> <span class="patent-bibdata-value"><a href="#legal-events">Legal Events</a> (9)</span> </td></tr><tr><td colspan=2 class="patent-bibdata-external-link-spacer-top"></td></tr><tr class="patent-bibdata-external-link-spacer-bottom"></tr><tr><td colspan=2><span class="patent-bibdata-heading">External Links:&nbsp;</span><span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=nwxWBAABERAJ&q=http://patft.uspto.gov/netacgi/nph-Parser%3FSect2%3DPTO1%26Sect2%3DHITOFF%26p%3D1%26u%3D/netahtml/PTO/search-bool.html%26r%3D1%26f%3DG%26l%3D50%26d%3DPALL%26RefSrch%3Dyes%26Query%3DPN/6270954&usg=AFQjCNEILs2-ExLFjkvPotudo212Zy_XZg">USPTO</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=nwxWBAABERAJ&q=http://assignments.uspto.gov/assignments/q%3Fdb%3Dpat%26pat%3D6270954&usg=AFQjCNG0ZqhGvc4O4nkSQyzwi_4mgSnGwg">USPTO Assignment</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=nwxWBAABERAJ&q=http://worldwide.espacenet.com/publicationDetails/biblio%3FCC%3DUS%26NR%3D6270954B1%26KC%3DB1%26FT%3DD&usg=AFQjCNFhtBTSdQ9KRXQ1n8zEDCN3eLjFfA">Espacenet</a></span></span></td></tr><tr class="patent-bibdata-group-spacer"></tr></table><div class="number-and-title"><span class="patent-title"><invention-title mxw-id="PT129045426" lang="EN" load-source="ifi">Contacting first cell that expresses confonnationally defective target protein with protein stabilizing agent to improve conformational defect, thereby improving phenotypic defect of first cell where in congo red is not the stabilizer</invention-title></span><br><span class="patent-number">US 6270954 B1</span></div><div class="patent-section patent-abstract-section"><div class="patent-section-header"><span class="patent-section-title">Abstract</span></div><div class="patent-text"><abstract mxw-id="PA72597290" lang="EN" load-source="patent-office"> <div class="abstract">A method of improving a phenotypic defect in a cell that contains a conformationally defective target protein wherein the conformational defect causes the phenotype defect, comprising contacting a first cell that expresses said confonnationally defective target protein with an amount of a protein stabilizing agent that is effective to improve the conformational defect, thereby improving the phenotypic defect of the first cell in comparison with a second cell having the same conformationally defective target protein and phenotypic defect, wherein the second cell is not contacted with the protein stabilizing agent.</div>
  </abstract></div></div><div class="patent-section patent-drawings-section"><div class="patent-section-header"><span class="patent-section-title">Images<span class="patent-section-count">(35)</span></span></div><div class="patent-drawings-body"><div class="patent-drawings-carousel"><div class="drawings"><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6270954B1/US06270954-20010807-D00000.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6270954B1/US06270954-20010807-D00000.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6270954B1/US06270954-20010807-D00001.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6270954B1/US06270954-20010807-D00001.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6270954B1/US06270954-20010807-D00002.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6270954B1/US06270954-20010807-D00002.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6270954B1/US06270954-20010807-D00003.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6270954B1/US06270954-20010807-D00003.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6270954B1/US06270954-20010807-D00004.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6270954B1/US06270954-20010807-D00004.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6270954B1/US06270954-20010807-D00005.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6270954B1/US06270954-20010807-D00005.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6270954B1/US06270954-20010807-D00006.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6270954B1/US06270954-20010807-D00006.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6270954B1/US06270954-20010807-D00007.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6270954B1/US06270954-20010807-D00007.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6270954B1/US06270954-20010807-D00008.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6270954B1/US06270954-20010807-D00008.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6270954B1/US06270954-20010807-D00009.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6270954B1/US06270954-20010807-D00009.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6270954B1/US06270954-20010807-D00010.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6270954B1/US06270954-20010807-D00010.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6270954B1/US06270954-20010807-D00011.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6270954B1/US06270954-20010807-D00011.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6270954B1/US06270954-20010807-D00012.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6270954B1/US06270954-20010807-D00012.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6270954B1/US06270954-20010807-D00013.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6270954B1/US06270954-20010807-D00013.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6270954B1/US06270954-20010807-D00014.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6270954B1/US06270954-20010807-D00014.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6270954B1/US06270954-20010807-D00015.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6270954B1/US06270954-20010807-D00015.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6270954B1/US06270954-20010807-D00016.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6270954B1/US06270954-20010807-D00016.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6270954B1/US06270954-20010807-D00017.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6270954B1/US06270954-20010807-D00017.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6270954B1/US06270954-20010807-D00018.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6270954B1/US06270954-20010807-D00018.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6270954B1/US06270954-20010807-D00019.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6270954B1/US06270954-20010807-D00019.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6270954B1/US06270954-20010807-D00020.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6270954B1/US06270954-20010807-D00020.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6270954B1/US06270954-20010807-D00021.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6270954B1/US06270954-20010807-D00021.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6270954B1/US06270954-20010807-D00022.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6270954B1/US06270954-20010807-D00022.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6270954B1/US06270954-20010807-D00023.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6270954B1/US06270954-20010807-D00023.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6270954B1/US06270954-20010807-D00024.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6270954B1/US06270954-20010807-D00024.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6270954B1/US06270954-20010807-D00025.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6270954B1/US06270954-20010807-D00025.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6270954B1/US06270954-20010807-D00026.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6270954B1/US06270954-20010807-D00026.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6270954B1/US06270954-20010807-D00027.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6270954B1/US06270954-20010807-D00027.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6270954B1/US06270954-20010807-D00028.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6270954B1/US06270954-20010807-D00028.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6270954B1/US06270954-20010807-D00029.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6270954B1/US06270954-20010807-D00029.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6270954B1/US06270954-20010807-D00030.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6270954B1/US06270954-20010807-D00030.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6270954B1/US06270954-20010807-D00031.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6270954B1/US06270954-20010807-D00031.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6270954B1/US06270954-20010807-D00032.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6270954B1/US06270954-20010807-D00032.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6270954B1/US06270954-20010807-D00033.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6270954B1/US06270954-20010807-D00033.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6270954B1/US06270954-20010807-D00034.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6270954B1/US06270954-20010807-D00034.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div></div></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img" alt="Previous page"src="/googlebooks/images/kennedy/page_left.png"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img" alt="Next page"src="/googlebooks/images/kennedy/page_right.png"width="21" height="21" /></div></div></div><div class="patent-post-drawings"></div><div class="patent-section patent-claims-section"><div class="patent-section-header"><span class="patent-section-title">Claims<span class="patent-section-count">(22)</span></span></div><div class="patent-text"><div mxw-id="PCLM28683544" lang="EN" load-source="patent-office" class="claims">
    <claim-statement>What is claimed is: </claim-statement> <div class="claim"> <div num="1" id="US-6270954-B1-CLM-00001" class="claim">
      <div class="claim-text">1. A method of improving a phenotypic defect in a cell that contains a conformationally defective target protein wherein the conformational defect causes the phenotype defect, comprising contacting a first cell that expresses said conformationally defective target protein with an amount of a protein stabilizing agent that is effective to improve the conformational defect, thereby improving the phenotypic defect of the first cell in comparison with a second cell having the same conformationally defective target protein and phenotypic defect, wherein the second cell is not contacted with a protein stabilizing agent; wherein Congo Red is not the protein stabilizing agent.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="2" id="US-6270954-B1-CLM-00002" class="claim">
      <div class="claim-text">2. The method according to claim <b>1</b>, wherein the cell is selected from the group of cells consisting of bacterial and eukaryotic cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="3" id="US-6270954-B1-CLM-00003" class="claim">
      <div class="claim-text">3. The method according to claim <b>1</b>, wherein the defective target protein is the gene product of a naturally occurring mutant nucleic acid.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="4" id="US-6270954-B1-CLM-00004" class="claim">
      <div class="claim-text">4. The method according to claim <b>1</b>, wherein the defective target protein is the gene product of a heterologous nucleic acid.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="5" id="US-6270954-B1-CLM-00005" class="claim">
      <div class="claim-text">5. The method according to claim <b>1</b>, wherein the defective target protein is selected from the group consisting of the cystic fibrosis transmembrane conductance regulator (CFTR) protein, emphysema and chronic liver disease α-1 anti-trypsin inhibitor, LDL receptor (familial hypercholesterolinemia), β-hexylaminidase (Tay-sachs), fibrillin (Martan syndrome), superoxide dismutase (amyotropic lateral sclerosis), collagen (scurvy) α-ketoacid dehydrogenase complex (maple syrup urine disease), p53 (cancer), type I procollagen pro-α (osteogenesis imperfecta), β-amyloid (Alzheimer's disease), crystallins (cataracts), rhodopsin (retinitis pigmentosa), and insulin receptor (leprechaunism).</div>
    </div>
    </div> <div class="claim-dependent"> <div num="6" id="US-6270954-B1-CLM-00006" class="claim">
      <div class="claim-text">6. The method according to claim <b>1</b>, wherein the reference protein stabilizing agent is selected form the group consisting of dimethylsulfoxide (DMSO), deuterated water, polyols, sugars, and amino acids and derivatives thereof.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="7" id="US-6270954-B1-CLM-00007" class="claim">
      <div class="claim-text">7. The method according to claim <b>6</b>, wherein the protein stabilizing agent is selected from the group consisting of glycerol, erythritol, trehalose isofluoroside, sorbitol, and polyethylene glycol.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="8" id="US-6270954-B1-CLM-00008" class="claim">
      <div class="claim-text">8. The method according to claim <b>6</b>, wherein the protein stabilizing agent is selected from the group consisting of glycine, alanine, proline, taurine, betaine, octopine, glutamate, sarcosine, gamma-aminobutyric acid, and trimethylamine N-oxide (TMAO).</div>
    </div>
    </div> <div class="claim-dependent"> <div num="9" id="US-6270954-B1-CLM-00009" class="claim">
      <div class="claim-text">9. The method according to claim <b>1</b>, wherein the phenotypic defect is caused by a condition selected from the group consisting of improper folding, improper co- and post-translational modification, improper subcellular targeting, inability to bind biological ligands, aggregation, proteolytic degradation, and any combination thereof.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="10" id="US-6270954-B1-CLM-00010" class="claim">
      <div class="claim-text">10. The method according to claim <b>9</b>, wherein the condition that causes the phenotypic defect occurs in a part of the protein that is selected from the group consisting of pre-sequence, pro-sequence, and mature protein sequence.</div>
    </div>
    </div> <div class="claim"> <div num="11" id="US-6270954-B1-CLM-00011" class="claim">
      <div class="claim-text">11. A screening method for detecting a phenotypically defective cell whose phenotypic defect is due to the presence of a conformationally defective target protein, comprising the steps of</div>
      <div class="claim-text">contacting a test cell having a phenotypic defect with a protein stabilizing agent, and </div>
      <div class="claim-text">determining whether such contact is effective to improve the phenotypic defect of the cell. </div>
    </div>
    </div> <div class="claim-dependent"> <div num="12" id="US-6270954-B1-CLM-00012" class="claim">
      <div class="claim-text">12. The method according to claim <b>11</b>, wherein the reference protein stabilizing agent is selected from the group consisting of dimethylsulfoxide (DMSO), deuterated water, polyols, and amino acids and derivatives thereof.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="13" id="US-6270954-B1-CLM-00013" class="claim">
      <div class="claim-text">13. The method according to claim <b>9</b>, wherein the cell is selected from the group of cells consisting of bacterial and eukaryotic cells, in particular yeast, insect and mammalian cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="14" id="US-6270954-B1-CLM-00014" class="claim">
      <div class="claim-text">14. The method according to claim <b>11</b>, wherein the defective target protein is the gene product of a heterologous nucleic acid.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="15" id="US-6270954-B1-CLM-00015" class="claim">
      <div class="claim-text">15. A method according to claim <b>11</b>, wherein the defective target protein is selected from the group wherein the defective target protein is selected from the group consisting of the cystic fibrosis transmembrane conductance regulator (CFTR) protein, emphysema and chronic liver disease α-1 anti-trypsin inhibitor, LDL receptor (familial hypercholesterolinemia), β-hexylaminidase (Tay-sachs), fibrillin (Martan syndrome), superoxide dismutase (amyotropic lateral sclerosis), collagen (scurvy) α-ketoacid dehydrogenase complex (maple syrup urine disease), p53 (cancer), type I procollagen pro-α (osteogenesis imperfecta), β-amyloid (Alzheimer's disease), crystallins (cataracts), rhodopsin (retinitis pigmentosa), and insulin receptor (leprechaunism).</div>
    </div>
    </div> <div class="claim"> <div num="16" id="US-6270954-B1-CLM-00016" class="claim">
      <div class="claim-text">16. A method of detecting the relative proportions of PrP<sup>C </sup>and PrP<sup>Sc </sup>present in a composition, comprising:</div>
      <div class="claim-text">mixing a composition that comprises prion proteins with a solution wherein either PrP<sup>C </sup>or PrP<sup>Sc </sup>is insoluble; </div>
      <div class="claim-text">separating soluble PrP from the insoluble PrP; and </div>
      <div class="claim-text">determining the relative amounts of soluble and insoluble PrP. </div>
    </div>
    </div> <div class="claim-dependent"> <div num="17" id="US-6270954-B1-CLM-00017" class="claim">
      <div class="claim-text">17. The method according to claim <b>16</b>, wherein the PrP is mixed with a solution comprising about 1% Triton X-100 and about 1% DOC at 4 C.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="18" id="US-6270954-B1-CLM-00018" class="claim">
      <div class="claim-text">18. The method according to claim <b>16</b>, wherein the soluble and insoluble forms of PrP are separated by centrifugation.</div>
    </div>
    </div> <div class="claim"> <div num="19" id="US-6270954-B1-CLM-00019" class="claim">
      <div class="claim-text">19. A protein stabilizing agent to improve a phenotypic defect in a cell that contains a conformationally defective target protein wherein the conformational defect causes the phenotype defect, wherein the protein stabilizing agent is selected from the group consisting of dimethylsulfoxide (DMSO), deuterated water, polyols; and amino acids and derivatives thereof, comprising contacting a first cell that expresses said conformationally defective target protein with an amount of a protein stabilizing agent that is effective to improve the conformational defect thereby improving the phenotypic defect of the first cell in comparison with a second cell having the same conformationally defective target protein and phenotypic defect, wherein the second cell is not contacted with a protein stabilizing agent; wherein Congo Red is not the protein stabilizing agent.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="20" id="US-6270954-B1-CLM-00020" class="claim">
      <div class="claim-text">20. A protein stabilizing agent according to claim <b>19</b>, wherein the polyol is selected from the group consisting of glycerol, erythritol, trehalose isofluoroside; polyethylene glycol; and sorbitol.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="21" id="US-6270954-B1-CLM-00021" class="claim">
      <div class="claim-text">21. A protein stabilizing agent according to claim <b>20</b>, wherein the amino acid or derivative thereof is selected form the group consisting of glycine, alanine, proline, taurine, betaine, octopine, glutamate, sarcosine, gamma-aminobutyric acid, and trimethylamine N-oxide (TMAO).</div>
    </div>
    </div> <div class="claim-dependent"> <div num="22" id="US-6270954-B1-CLM-00022" class="claim">
      <div class="claim-text">22. A protein stabilizing agent according to claim <b>19</b>, wherein the defective target protein is selected from the group consisting of the cystic fibrosis transmembrane conductance regulator (CFTR) protein, emphysema and chronic liver disease α-1 anti-trypsin inhibitor, LDL receptor (familial hypercholesterolinemia), β-hexylaminidase (Tay-sachs), fibrillin (Martan syndrome), superoxide dismutase (amyotropic lateral sclerosis), collagen (scurvy), α-ketoacid dehydrogenase complex (maple syrup urine disease), p53 (cancer), type I procollagen pro-α (osteogenesis imperfecta), β-amyloid (Alzheimer's disease), crystallins (cataracts), rhodopsin (retinitis pigmentosa), and insulin receptor (leprechaunism).</div>
    </div>
  </div> </div></div></div><div class="patent-section patent-description-section"><div class="patent-section-header"><span class="patent-section-title">Description</span></div><div class="patent-text"><div mxw-id="PDES54707192" lang="EN" load-source="patent-office" class="description">
    <p>This application is a continuation in part of U.S application Ser. No. 08/838,691, filed Apr. 9, 1997 now U.S. Pat. No. 5,900,360 and it claims priority from U.S. provisional application Ser. No. 60/015,155, filed Apr. 10, 1996, both of which are incorporated by reference.</p>
    <p>The government may own certain rights in the present invention pursuant to grants from the Cystic Fibrosis Foundation (R613) and NIH (AG-10770.</p>
    <heading>BACKGROUND OF THE INVENTION</heading> <p>The present invention relates to the treatment of phenotypic defects that are caused by improper protein folding and processing.</p>
    <p>When a protein is synthesized, its amino acid side chains interact, causing the polypeptide backbone to fold into thermodynamically preferred three dimensional structures or “conformations.” The biological properties and proper localization (either within the cell or secreted out of the cell) of proteins are contingent on assuming certain biologically significant conformations. Proteins that for whatever reason do not assume “correct” biologically active conformations are inactive and/or misprocessed and/or mislocalized and/or degraded. Failure to assume a proper conformation can lead to disease, and can be fatal. Reviewed in Thomas, P. J. et al., <i>TIBS, </i>20:456-459 (1995).</p>
    <p>Abnormalities in protein folding constitute the molecular basis for a number of diseases. Thomas, P. J., et al., 1995, <i>TIBS </i>20:456-459; Welch, W. J. et al., 1996, <i>Cell Stress </i>&amp; <i>Chap. </i>1: 109-115. Oftentimes single point or deletion mutations give rise to subtle folding defects that result in either a loss of protein function, or a failure of the protein to be correctly localized. A number of pathological conditions are reportedly the result of improper folding. For example, in cystic fibrosis, a mutation (e.g., ΔF508) which results in improper folding leads to improper targeting of the cystic fibrosis transmembrane conductance regulator; mutant proteins are retained in the endoplasmic reticulum and not delivered to their normal site of action at the plasma membrane. Cheng, S. H. et al., <i>Cell, </i>63:827 (1990).; Denning, G. M. et al., <i>Nature, </i>358:761 (1992); Gregory et al., <i>Mol. Cell. Biol., </i>11:3886 (1991); Kartner et al., <i>Nature Genet., </i>1:321 1992); G. M. Denning, G. M. et al., <i>J. Cell Biol., </i>118:551 (1992). In emphysema and chronic liver diseases, conformational defects result in the failure to secrete alpha-1 antitrypsin inhibitor, leading to tissue damage. Lomas, D. A. et al., <i>Nature, </i>357:605-607 (1992); Lomas, D. A. et al., <i>Am. J. Physiol., </i>265:L211-219 (1993). In fanilial hypercholesterolemia, a mutation within the coding region of low density lipoprotein (LDL) receptor results in a failure of the protein to localize to the plasma membrane, leading to abnormal levels of serum cholesterol and heart disease. Amara, J. F. et al., <i>Trends Cell Biol., </i>2:145-149 (1992); Hobbs, H. H. et al., <i>Annu. Rev. Genet., </i>24:133-170 (1990); Yamamoto, T. et al., <i>Science, </i>232:1230-1237 (1986). In Tay Sachs disease, a mutation within the coding region of the alpha subunit of beta-hexosaminidase is not delivered to its normal lysosomal location; the mutant protein is instead retained in the endoplasmic reticulum. Amara, J. F. et al., supra; Lau, M. M. H. et al., <i>J. Biol. Chem., </i>264:21376-21380 (1989). Other diseased states that are due to improperly folded protein are known in the art. Bychkova, V. E. et al., <i>FEBS Let., </i>359:6-8 (1995); Thomas, P. J. et al., <i>Trends Biol. Sci., </i>20:456-459 (1995).</p>
    <p>Factors which influence a protein's preferred conformation include amino acid sequence, intra- and intermolecular charge interactions, hydrophobic interactions, steric interactions, Van der Waals forces, and disulphide bond linkages; reversible and irreversible post-translational modification (e.g., phosphorylation or glycosylation); degree of hydration; and nature and composition of the solvent medium.</p>
    <p>The primary driving force for assuming a biologically active conformation in vivo is thought to be the amino acid sequence. Certain proteins have been shown to spontaneously fold to assume their proper conformation, even after repeated cycles of denaturation and renaturation. Changes in sequence (e.g., mutations) may result in dramatic conformational alterations.</p>
    <p>A number of low molecular weight compounds are reportedly effective in stabilizing proteins in vitro against thermally induced denaturation. Germsla, et al., 1972, <i>Int. J. Pept. Proteins Res. </i>4:372-378; Back, J. F., et al., 1979, <i>Biochem. </i>18:5191-5199; Gekko, K. et al., 1983, <i>J. Biochem. </i>94:199-208. Representative compounds include polyols such as glycerol, solvents such as DMSO, and deuterated water (D<sub>2</sub>O). In addition to their effects in vitro some of these compounds appear to influence protein folding and/or stability in vivo. Lin, P. S. et al., 1981, <i>J. Cell. Physiol. </i>108:439-448; Henle, K. J. et al., 1983, <i>Cancer Res. </i>43:1624-1633; Edington, B. V. et al., 1989, <i>J. Cell. Physiol. </i>139:219-228. For example, animal cells incubated in the presence of either deuterated water or glycerol can withstand severe heat shock treatments that would otherwise be lethal to the cells. Here addition of the compounds to the cells helps to reduce the overall extent of thermal denaturation of intracellular proteins. In yeast and bacteria, the addition of glycerol into the growth medium not only protects the cells against thermal treatments, but in some cases also is effective in correcting protein folding abnormalities due to specific mutations. Hawthorne, D. C. et al., 1964, <i>Genetics </i>50:829-839. This type of “osmotic remediation” has been shown to be the most effective for those mutant proteins which exhibit a temperature sensitive protein folding defect.</p>
    <p>Some proteins appear to require interaction with other molecules in order to fold properly. Substances that aid proteins to assume their biologically active conformations have been identified in a variety of cell types and cell compartments. Fischer, G. et al., <i>Biochemistry, </i>29:2206-2212 (1990); Freedman, R. B., <i>Cell, </i>57:1069-1072 (1989); Ellis et al., <i>Annu. Rev. Biochem., </i>60:337-347 (1991). Among the best known are a class of proteins called “molecular chaperones” (Dingwall, C. K. et al., <i>Seminars in Cell Biol., </i>1:11-17 (1990)); (Hemmingsen, S. M. et al., <i>Nature, </i>333:330-334 (1988)), or “polypeptide chain binding proteins” (Rothman, J. E., <i>Cell, </i>59:591-601 (1989)). Chaperones stabilize newly synthesized polypeptides until they are assembled into their proper native structure or until they are transported to another cellular compartment, i.e., for secretion. Sambrook et al., <i>Nature, </i>342:224-225 (1989); U.S. Pat. No. 5,474,914, which issued to R. Spaete on Dec. 12, 1995. They reportedly prevent the formation of undesirable protein aggregates by binding to unfolded or partially denatured polypeptides.</p>
    <p>The heat-shock proteins of the hsp70 and hsp6o families are examples of chaperones. Langer, T. et al., <i>Curr. Topics in Microbiol. and Immun., </i>167:3-30 (1991); Pelham, H. R. B., <i>Nature, </i>332:776-777 (1988); and Harl, F., <i>Seminars in Immunol. </i>3, (1991). U.S. Ser. No. 07/261,573, filed Oct. 24, 1988, described the folding function of hsp60, isolated from yeast mitochondria. See also McMullin, T. W. et al., <i>Molec. Cell. Biol. </i>8:371-380 (1988); Reading, D. S. et al., <i>Nature, </i>337:655-659 (1989); Cheng, M. Y. et al., <i>Nature, </i>337:620-625 (1989); Ostermann, J. et al., <i>Nature, </i>341:125-130 (1989); and Cheng, M. Y. et al., <i>Nature, </i>348:455-458 (1990)). The essential role in protein folding of the members of the hsp60 family has since been demonstrated in vivo and in vitro. Other chaperones include the rubisco binding protein of chloroplasts, reviewed by Barraclough, R. et al., <i>Biochim. Biophys. Acta, </i>608:19-31 (1980); Musgrove, J. E. et al., <i>Eur. J. Biochem., </i>163:529-534 (1987); and Gatenby, A. A. et al., <i>Rev. Cell Biol., </i>6:125-149 (1990), and proteins such as GroEL, isolated from <i>E. coli </i>(Georgopoulos, C. et al., <i>J. Molec. Biol., </i>76:45-60 (1973); Stomborg, N. J., <i>Molec. Biol., </i>76:25-44 (1973); Hendrix, R. W. J., <i>Molec. Biol., </i>129:375-392 (1979); Bochkareva, E. S. et al., <i>Nature, </i>336:254-257 (1988); Goloubinoff, P. et al., <i>Nature, </i>342:884-889 (1989); Van Dyk, T. K. et al., <i>Nature, </i>342:451-453; Lecker, S. et al., <i>EMBO J, </i>8:2703-2709 (1989); Laminet, A. A. et al., <i>EMBO J, </i>9:2315-2319 (1990); Buchner, J. et al., <i>Biochemistry, </i>30:1586-1591 (1991)). U.S. Ser. No. 07/721,974 entitled “Chaperonin-Mediated Protein Folding” and filed on Jun. 27, 1991 by Franz-Ulrich Hartl and Arthur L. Horwich, described mechanisms and components required for chaperonin-dependent folding of proteins, using the groEL and groES proteins of <i>E. coli </i>to reconstitute dihydrofolate reductase (DHFR) and rhodanese. The folding reaction required Mg-ATP and the chaperonin proteins.</p>
    <heading>SUMMARY OF THE INVENTION</heading> <p>The present invention provides methods of improving phenotypic defects that are caused by conformationally defective target proteins. The methods of the invention comprise exposing a cell that expresses a conformationally defective target protein with an amount of a protein stabilizing agent that is effective to improve the phenotypic defect. The improvement in the phenotypic defect of treated cells is assessed by comparing them with cells having the same conformationally defective target protein and phenotypic defect that are not exposed to the protein stabilizing agent.</p>
    <p>Nonlimiting examples of “protein stabilizing agents” include dimethylsulfoxide (DMSO), deuterated water, trimethylamine N-oxide (TMAO), polyols and sugars such as glycerol, erythritol, trehalose (used by plants as an osmolyte) isofluoroside; inositol and sorbitol and polymers such as polyethylene glycol; amino acids and derivatives thereof such as glycine, alanine, proline, taurine, betaine, octopine, glutamate, sarcosine, and gamma-aminobutyric acid.</p>
    <p>Nonlimiting examples of defective target proteins to be treated (and, in parenthesis, their associated diseases) include the cystic fibrosis transmembrane conductance regulator (“CFTR”) protein (cystic fibrosis), α-1 anti-trypsin inhibitor (emphysema and chronic liver disease), LDL receptor (familial hypercholesterolinernia), β-hexylaminidase (Tay-sachs), fibrillin (Martan syndrome), superoxide dismutase (amyotropic lateral sclerosis), collagen (scurvy) α-ketoacid dehydrogenase complex (maple syrup urine disease), p53 (cancer), type I procollagen pro-α (osteogenesis imperfecta), crystallins (cataracts), rhodopsin (retinitis pigmentosa), insulin receptor (leprechaunism and/or insulin resistance), and prion proteins (e.g., kuru, Creutzfeld-Jakob disease, Gerstmann-Straussler Scheinker Syndrome, fatal family insomnia in humans, scrapie and bovine spongiform encephaly in animals). The cells used in the invention are selected from bacterial, yeast, insect and in particular animal or mammalian cells, including cells engineered to express a mutated form of CFTR called ΔF508.</p>
    <p>In one embodiment, treatment of cells expressing the ΔF508 CFTR mutant with glycerol causes the ΔF508 gene product to fold properly, and results in restoring wild-type CFTR activity to cells that express ΔF508. A variety of other chemicals propitiate proper folding, including dimethylsulfoxide (DMSO), deuterated water, and the naturally occurring osmolyte, trimethylamine N-oxide. All of these result in a restoration of normal chloride transport in mutant cells comparable to that of wild type cells.</p>
    <p>In another embodiment, we provide an assay for quantifying the conversion of the prion protein PrP from the cellular conformation PrP<sup>C </sup>to the pathogenic scrapie form PrP<sup>Sc</sup>. We demonstrate that glycerol, DMSO and TMAO interfere with the conversion of PrP<sup>C </sup>to PrP<sup>Sc</sup>.</p>
    <p>Cell lines expressing temperature sensitive mutants of: the tumor suppressor protein p53; the viral oncogene protein pp60; or a ubiquitin activating enzyme E1, were incubated at the nonpermissive temperature (39.5° C.) in the presence of glycerol, trimethylamine N-oxide or deuterated water. In each case, the cells now exhibited phenotypes similar to that observed when the cells were incubated at the permissive temperature (32.5° C.), indicative that the particular protein folding defect had been corrected. See also Brown et al., 1997, <i>J Clin. Invest. </i>99:1432-1444. Thus, protein stabilizing agents are effective in vivo for correcting protein folding abnormalities.</p>
    <p>In summary, the use of “chemical chaperones” is effective for the treatment of cystic fibrosis and other genetic diseases which involve defective protein folding and/or a failure in normal protein trafficking and maturation events.</p>
    <heading>BRIEF DESCRIPTION OF THE FIGURES</heading> <p>FIG. <b>1</b>. Effect of incubation with glycerol on the processing and maturation of the ΔF508 CFTR protein.</p>
    <p>FIG. <b>2</b>. Concentration and time dependence of the effect of glycerol on ΔF508 CFTR processing.</p>
    <p>FIG. <b>3</b>. Effect of glycerol on forskolin dependent chloride transport in ΔF508 CFTR expressing cells.</p>
    <p>FIG. <b>4</b>. Effect of deuterated water (dH2O) or trimethylamine N-oxide (TMAO) on forskolin dependent chloride transport in ΔF508 CFTR expressing cells.</p>
    <p>FIG. <b>5</b>. Differential detergent solubility assay for quantifying the relative proportions of PrP<sup>C </sup>and PrP<sup>Sc</sup>.</p>
    <p>FIG. <b>6</b>. Demonstration that glycerol treatment of ScN2a cells results in reduced levels of PrP<sup>Sc</sup>.</p>
    <p>FIG. <b>7</b>. Effect of different concentrations of glycerol, TMAO and DMSO on the conversion of PrP<sup>C </sup>to PrP<sup>Sc</sup>: demonstrating that glycerol, DMSO and TMAO interfere with the accumulation of insoluble PrP<sup>Sc </sup>in both a dose dependent and time dependent manner.</p>
    <p>FIG. <b>8</b>. Demonstration that glycerol treatment does not affect the cell surface localization of PrP<sup>C</sup>.</p>
    <p>FIG. <b>9</b>. Demonstration that the effect of glycerol on the interference of PrP<sup>C </sup>to PrP<sup>Sc </sup>conversion is reversible.</p>
    <p>FIG. <b>10</b>. Demonstration that detergent soluble PrP<sup>C </sup>added to ScN2a cells is converted to a detergent insoluble form in the absence, but not the presence of glycerol.</p>
    <p>FIG. <b>11</b>. The intracellular localization of temperature sensitive p53 in A1-5 cells incubated at the permissive (32.5° C.) and non-permissive (39.5° C.) temperatures.</p>
    <p>FIG. 12. A1-5 cells incubated at the non-permissive temperature in the presence of protein stabilizing agents exhibit nuclear p53 staining.</p>
    <p>FIG. 13. A1-5 cells treated with protein stabilizers at the nonpermissive temperature exhibit a wild type phenotype as determined by cell proliferation rates.</p>
    <p>FIG. <b>14</b>. The effects of the protein stabilizing agents on p53 protein folding are reversible.</p>
    <p>FIG. <b>15</b>. The presence of glycerol corrects temperature sensitive (ts) pp60<sup>src </sup>activities at the nonpermissive temperature.</p>
    <p>FIG. <b>16</b>. Effects of protein stabilizers on the morphology of ts 20 cells expressing a temperature sensitive enzyme involved in protein ubiquitination.</p>
    <p>FIG. <b>17</b>. ts20 cells incubated in the presence of the protein stabilizers proliferate at the nonpermissive temperature.</p>
    <p>FIG. <b>18</b>. Protein stabilizing chemicals correct the temperature sensitive phenotype of ts20 cells, as demonstrated by a lack of nuclear p53 staining at the nonpermissive temperature.</p>
    <p>FIG. <b>19</b>. ts20 cells incubated at 39.5° C. in the presence of protein stabilizing agents now exhibit low levels of the p53 protein as determined by Western blotting.</p>
    <p>FIG. <b>20</b>. Addition of protein stabilizing agents to cells already at the nonpermissive temperature is effective in restoring wild type E1 activities.</p>
    <p>FIG. <b>21</b>. Protein stabilizers correct the folding defect of the newly synthesized but not the mature form of the E1 enzyme.</p>
    <p>FIG. <b>22</b>. Treatment with mammalian cellular osmolytes promotes the proper processing and function of the F508 CFTR protein.</p>
    <p>FIG. 22A shows that cells expressing the DF508 mutant protein, when exposed to the different cellular osmolytes, now contain some of the mature protein (band C), similar to the wild type cells.</p>
    <p>FIG. 22B shows that DF508 cells (control) fail to transport any appreciable chloride ions in response to stimulation with forskolin (plotted as a percentage of chloride transport relative to cells expressing wild type protein). In contrast, cells treated with 4 different osmolytes now show partial chloride transport (note however that it never reaches the wild type levels, rather only about 10-30% of wild type).</p>
    <p>FIG. <b>22</b>C. Shows that the different treatments are not deleterious to the cells as indicated by continued cell proliferation.</p>
    <p>FIG. <b>23</b>. The effects of osmolytes on the processing of the F508 CFTR protein are concentration and time dependent.</p>
    <p>FIG. 23A shows a concentration curve of osmolyte treatment of the cells expressing the mutant protein-note that only the higher concentration (100 mM) appeared effective to elicit the appearance of the mature (band C) form of the CFTR protein.</p>
    <p>FIG. 23B shows that the positive effects of the osmolytes diminished over time.</p>
    <p>FIG. <b>24</b>. The effects of osmolyte treatment are reversible but can be reinitiated by a subsequent treatment.</p>
    <p>FIG. 25. 3T3 cells stably expressing the delta F508 CFTR protein can adapt to hyperosmotic conditions and accumulate cellular osmolytes.</p>
    <p>FIG. 25A shows that when cells expressing the mutant DF508 CFTR protein are incubated under hyperosmotic conditions (high salt) the cells begin to accumulate different osmolytes and to different evels.</p>
    <p>FIG. <b>26</b>. Osmotic adaptation results in a partial correction of F508 CFTR processing and function.</p>
    <p>FIG. <b>27</b>. Adaptation of kidney derived cells to high salt and urea conditions results in the accumulation of cellular osmolytes.</p>
    <p>FIG. <b>28</b>. Adaptation to high salt and urea conditions promotes the proper processing and function of the F508 CFTR protein.</p>
    <heading>DEFINITIONS</heading> <p>The term “wild type,” when applied to a protein, protein sequence conformation, and/or biological property, denotes a protein, protein sequence, conformation, or biological property predominantly found in nature. Mutant proteins occur less frequently and are often associated with a diseased state.</p>
    <p>“Biologically active conformation” denotes a conformation that enables a protein to perform a biological function (e.g., bind ligands, especially with characteristic binding parameters such as binding capacity or binding affinity; enzymatic activity) normally performed by corresponding wild type proteins.</p>
    <p>A “conformationally defective target protein” is a protein that is improperly folded in whole or in part at any point during its synthesis or post-translational modification, wherein the improper folding results in: improper targeting, and/or improper post-translational modification, and/or protein aggregation, and/or a detectable phenotypic defect, and/or an inability to perform biological activities performed by wild type proteins. The improper folding may take place within any region of the pre-sequence, pre-protein, pro-sequence, pro-protein or the mature protein. Defective target proteins include the cystic fibrosis transmembrane conductance regulator (CTFR) protein, α-1 anti-trypsin inhibitor (emphysema and chronic liver disease), LDL receptor (familial hypercholesterolinemia), β-hexylaminidase (Tay-sachs), prion protein (scrapies), etc. While any mutation may alter conformation to a degree, this term is specifically directed to mutations that are correctable by the addition of a protein stabilizing agent.</p>
    <p>“Phenotypic defect” denotes an observable and preferably readily quantifiable trait in a cell or organism, wherein the defect is caused by a “conformationally defective target protein,” that results in a medical condition or diseased state, as compared to wild type individuals whose protein is not conformationally defective.</p>
    <p>“Improving a phenotypic defect” denotes causing a phenotypic defect in a cell or organism to observably or measurably become more like the wild type.</p>
    <p>“Protein stabilizing agent” is a substance that stabilizes a protein in a biologically active conformation. For most proteins, it acts to stabilize the protein under conditions that can cause denaturation or aggregation. It stabilizes or induces a conformationally defective target protein to assume a biologically active conformation or compensates for its inability to assume a biologically active conformation, and causes a conformationally defective target protein to be processed properly, and/or to be targeted properly, and/or to acquire or retain a biological activity similar to that of the wild-type protein.</p>
    <p>An amount of a “protein stabilizing agent” that is effective to improve a phenotypic defect is an amount that, when administered to a cell or individual having a conformationally defective target protein that results in a phenotypic defect, causes the phenotypic defect that is caused by conformationally defective target protein to become more like the phenotypic trait of cells or individuals that express the wild type protein. Protein stabilizing agents usually are administered at concentrations wherein a defective target protein is induced to behave like the wild-type protein, usually 1 μM to 1 M. The precise quantity may vary as a function of the relative potency of the protein stabilizing agent.</p>
    <p>“Pre-sequence” refers to an amino acid sequence in a newly synthesized protein which serves as an identification, addressing and sorting “tag” for determining the subcellular destination of the tagged protein, and also as a handle for transporting the protein. This pre-sequence is usually removed as the protein matures. In addition, many proteins are synthesized as relatively inactive “pro-proteins.” Pro-proteins can also undergo proteolytic processing, usually involving the removal of a peptide sequence, to yield the mature protein. The “mature protein sequence” is the sequence of the protein after pre- and pro-sequences have been removed or modified.</p>
    <p>“Co-translational modification” and “post-translational modification” of a given immature protein to the mature protein includes covalent attachment of sugars, lipids or other functional groups (e.g., phosphate) and chemical modification of amino acid residues.</p>
    <p>The term “nucleic acid” refers to a deoxyribonucleotide or ribonucleotide polymer in either single- or double-stranded form, and unless specifically limited, encompasses known analogues of natural nucleotides that hybridize to nucleic acids in manner similar to naturally occurring nucleotides. Unless otherwise indicated, a particular nucleic acid sequence implicitly provides the complementary sequence thereof, as well as the sequence explicitly indicated. The term nucleic acid is used interchangeably with gene and cDNA.</p>
    <p>The phrase “heterologous nucleic acid” generally denotes a nucleic acid that has been isolated, cloned and introduced into and/or expressed in a cell or cellular environment other than the cell or cellular environment in which said nucleic acid or protein may be found in nature. The term encompasses both nucleic acids originally obtained from a different organism or cell type than the cell type in which it is expressed, and also nucleic acids that are obtained from the same cell line as the cell line in which it is expressed. Where two nucleic acid sequences not normally combined in nature are ligated together, each is heterologous to the other.</p>
    <p>The phrase “a nucleic acid sequence encoding” refers to a nucleic acid which contains sequence information that, if translated, yields the primary amino acid sequence of a specific protein or peptide. This phrase specifically encompasses degenerate codons (i.e., different codons which encode a single amino acid) of the native sequence or sequences which may be introduced to conform with codon preference in a specific host cell.</p>
    <p>The term “recombinant” or “engineered” when used with reference to a cell indicates that the cell replicates or expresses a nucleic acid or expresses a peptide or protein encoded by a nucleic acid, whose origin is exogenous to the cell. Recombinant cells can express nucleic acids that are not found within the native (nonrecombinant) form of the cell. Recombinant cells can also express nucleic acids found in the native form of the cell wherein the nucleic acids are re-introduced into the cell by artificial means.</p>
    <p>A “composition comprising prion proteins” denotes an organ, tissue, cell, subcellular preparation, or solution that contains either PrP<sup>C </sup>or a mixture of PrP<sup>C </sup>and PrP<sup>Sc</sup>.</p>
    <heading>DETAILED DESCRIPTION</heading> <p>The present invention is directed to “chemical chaperones” that improve a broad range of phenotypic defects which are the result of improper conformation, improper cotranslational or post-translational modification, inability to bind biological ligands, improper protein targeting, and/or loss of biological activity relative to the wild type protein. The chemical chaperones of the invention induce defective proteins to assume biologically active conformations, and/or exhibit correct subcellular targeting, and/or retain or acquire biological activity similar to that of the wild-type protein.</p>
    <p>The prior art focuses mainly on the in situ effects of naturally occurring protein chaperones. Since these are nonpermeant intracellular proteins, they are unsuitable for administration to cells to remedy phenotypic defects, unless special steps are taken to ensure that they penetrate into the cell. The protein stabilizing agents provided herein are vastly easier to obtain, chemically more stable, and easier to administer than prior art chaperones which are produced endogenously. Specifically, the chemical chaperones of the invention are readily taken up by the cell.</p>
    <p>In what follows, methods for identifying substances that function as protein stabilizing agents are discussed, using as the model a cell line expressing the mutant ΔF508 CFTR or a cell line which expresses PrP<sup>Sc</sup>. We then teach how to determine whether a given phenotypic defect is treatable with a protein stabilizing agent. Finally, the feasibility of practicing the invention is shown by providing specific examples of improving phenotypic defects using selected protein stabilizing agents and several conformationally defective target proteins: ΔF508 CFTR, prion proteins, the tumor suppressor protein p53; pp60<sup>src</sup>, the transforming protein encoded by Rous sarcoma; and an enzyme, E1 which catalyzes the first step in the pathway of protein ubiquitination.</p>
    <p>A. Protein Stabilizing Agents</p>
    <p>Specific protein stabilizing agents encompassed by the invention include dimethylsulfoxide (DMSO), deuterated water, trimethylamine N-oxide (TMAO), polyols and sugars such as glycerol, erythritol, inositol, trehalose (used by yeast as an osmolyte) isofluoroside; polymers such as polyethylene glycol; amino acids and derivatives thereof, such as glycine, alanine, proline, taurine, betaine, octopine, glutamate, sarcosine, gamma-aminobutyric acid, and trimethylamine N-oxide (TMAO). The invention is not limited to any specific agents. Indeed, any relatively nontoxic substance that functions as a protein stabilizing agent will do. This includes agents such as cellular osmolytes.</p>
    <p>B. Assays to Identify Protein Stabilizing Agents</p>
    <p>Protein stabilizing agents that are not specifically disclosed herein may be routinely identified by any of the following model methods. As a threshold test, candidate protein stabilizing agents can be identified by their ability to stabilize in vitro certain physical (e.g., reduce aggregation in response to chemical or thermal treatment) and functional (e.g., retain enzymatic activity in response to chemical or thermal treatment) properties of either wild type or conformationally defective proteins. Candidate agents can then be tested in cellular systems such as the mutant CFTR and prion protein expressing cell lines described below. These assays are also useful to determine the relative potency of different protein stabilizing agents, by comparing the ability of a putative protein stabilizing agent to stabilize a biologically active conformation or to induce a conformationally defective protein to become more like the wild type protein, relative to a known well characterized standard protein stabilizing agent (e.g., glycerol).</p>
    <p>1. Physical Property Assays</p>
    <p>One method of defining whether a test substance is a “protein stabilizing agent” is to determine its ability to stabilize a protein in vitro to aggregation induced by thermal treatment, changes in pH, or the addition of protein chaotropes (e.g. urea, guanidinium chloride). The assay is performed, for example, by (1) incubating 0.1 to 10 mg/ml of a test wild type protein in the presence or absence of preselected concentrations (ranging from about 0.1 μM to about 1 M) of a test compound, (2) subjecting the test protein to denaturing conditions, (3) measuring the alterations, if any, in the test protein's physical properties (e.g., light absorbance, aggregation, solubility in a specific solvent, α-helix or β sheet content, shape) in the presence or absence of the test compound, and (4) assessing the ability of the test protein to retain wild type behavior and physical properties in the presence and absence of the test substance. For example, the physical property may be a tendency to aggregate in a given solution. Aggregation may be measured by light scattering, velocity sedimentation, or other known techniques. Conditions which cause aggregation include thermal treatment (incubation at 10-70° C.), pH treatment (e.g., incubation at a pH selected from pH 2-10), or the addition of protein chaotropes (e.g. 1-8 M urea or guanidinium chloride). Many others are known in the art and are encompassed by the invention. Protein stabilizing agents are those test substances which reverse, reduce, or prevent the effects (e.g., denaturation, aggregation) of the protein denaturing agent (e.g., acid, detergent) or treatment (pH, temperature).</p>
    <p>Another method of testing whether a test substance is a “protein stabilizing agent” is to determine the ability of a specific concentration (or range of concentrations) of a test substance to cause a conformationally defective target protein having a physical property that is measurably different from that of the wild type protein to become more like the wild type. For example, the test protein may tend to aggregate under conditions in which the wild type protein does not aggregate. Aggregation is measured as described above. Test substances which reverse, reduce, or prevent the aggregation of the test protein and thus cause it to behave more like the wild type protein are protein stabilizing agents.</p>
    <p>2. Biological Activity Assays</p>
    <p>Another method of determining whether a test substance is a “protein stabilizing agent” is to measure the biological activity of a particular test protein (e.g. enzymatic activity of an enzyme such as β-galactosidase, luciferase, ribonuclease, lysozyme, etc.) in response to protein destabilizing conditions in the presence and absence of test substances.</p>
    <p>For example, a test wild type enzyme is exposed to protein destabilizing conditions as described in the previous section (thermal treatment, pH treatment, chaotrope or detergent treatment) in the presence and absence of a defined range of concentrations of a test substance, and the enzymatic activity of the test enzyme under either set of conditions is determined using methods known in the art. A protein stabilizing agent is a test substance that, when present in a defined concentration, results in at least 10% higher activity in the presence of the putative protein stabilizing agent as compared to the same protein exposed to the same destabilizing conditions in the absence of the putative agent.</p>
    <p>As a further example, cells expressing a test wild type enzyme are exposed to protein destabilizing conditions as described above, in the presence and absence of a defined range of concentrations of a test substance, and the enzymatic activity of the test enzyme at each concentration of the test substance is determined using methods known in the art. A protein stabilizing agent is a test substance that, when present in a defined concentration, results in at least 10% higher activity in the presence of the putative protein stabilizing agent as compared to the same test performed in the absence of said agent.</p>
    <p>As still another example, a cell line that expresses a phenotypic defect associated with the gene product of a homologous or heterologous gene is tested for the presence of a measurable biological activity, such as enzymatic activity, normally associated with the wild type protein but not with the conformationally defective target protein. For example, this activity may be temperature-sensitive. A temperature-sensitive protein or activity is one that is less active than the wild type protein or activity at a particular defined (“nonpermissive”) range of temperatures, but behaves more like the wild type at a second defined (“permissive”) range of temperatures. For example, temperature-sensitive proteins in mammalian cells are often active under 30° C. the “permissive” temperature), but are relatively inactive at 37° C. (the “non-permissive” temperature). A protein stabilizing agent is a test substance that, when present in a defined concentration, causes the enzyme to retain or acquire at least 10% of the wild type activity at the non-permissive temperature.</p>
    <p>3. Assays Utilizing ΔF508 CFTR-expressing Cell Lines</p>
    <p>Still another method of identifying a protein stabilizing agent involves the use of mutant ΔF508 CFTR-expressing cell lines as described below in greater detail in Example 1. Briefly, a ΔF508 CFTR-expressing line is produced by introducing either the wild type CFTR or a mutant ΔF508 CFTR-encoding nucleic acid (e.g., ΔF508) into a cell type which does not normally express the CFTR protein (e.g., NIH 3T3 fibroblasts). Different concentrations of a test substance, generally ranging from 0.1 μM to about 1 M, are dissolved in tissue culture medium. Equal numbers (about 10<sup>5 </sup>or more) of mutant ΔF508 CFTR-expressing cells and control (wild-type CFTR-expressing) cells are incubated at the selected concentration for about 1 hour to about several days, preferably 12-72 hours. The effects of the chemical chaperones are then examined by the biochemical methods described below. For example, the subcellular distribution of ΔF508 CFTR is determined using biochemical assays as described below, and/or immunofluorescent localization. In addition, the cultured cells are optionally analyzed for forskolin-dependent chloride efflux. A test substance that results in the proper folding and modification of the mutant ΔF508 CFTR protein, and its movement to the plasma membrane, which results in the appearance of functional forskolin sensitive chloride channels, and/or forskolin-dependent chloride effluxes that are at least 20% (and preferably at least 50%) of that observed for cells expressing wild type CFTR is a “protein stabilizing agent.”</p>
    <p>By way of background, mature CFTR is an approximately 160 kDa glycoprotein present in epithelial cells, where it functions as a forskolin dependent chloride channel. 20-25% of newly synthesized wild type CFTR protein exits from the ER and moves to the plasma membrane. The remainder is retained at the ER and eventually is targeted for degradation.</p>
    <p>A large number of mutations within the coding region of the gene encoding the CFTR protein have been described, many of which result in a failure of the newly synthesized protein to move out of the endoplasmic reticulum (ER) to the plasma membrane. Rommens, J. M. et al., <i>Science, </i>245:1059 (1989); Riordan, J. R. et al., <i>Science, </i>245:1066 (1989); Kerem, B. S. et al., <i>Science, </i>245:1073 (1989). Defective intracellular transport and processing of the CFTR protein appears to constitute the primary lesion associated with patients having cystic fibrosis. Cheng, S. H. et al., <i>Cell, </i>63:827 (1990).; Denning, G. M. et al., <i>Nature, </i>358:761 (1992); Gregory et al., <i>Mol. Cell. Biol., </i>11:3886 (1991); Kartner et al., <i>Nature Genet., </i>1:321 (1992); G. M. Denning, G. M. et al., <i>J. Cell Biol., </i>118:551 (1992). The ΔF508 mutant is but one example of mutations that are useful in the assays described below.</p>
    <p>In the mutant ΔF508 CFTR, a phenylalanine normally present at position 508 of the wild type CFTR is deleted. This deletion results in abnormal folding of the protein and retention at the ER. The vast majority (&gt;99%) of the newly synthesized ΔF508 mutant CFTR appears never to leave the ER and is degraded. Yang, Y., et al., <i>Proc. Natl. Acad. Sci., </i>90:9480 (1993); Pind, S., et al., <i>J. Biol. Chem., </i>269:12784 (1994).</p>
    <p>The folding defect associated with the mutant ΔF508 CFTR is temperature-sensitive. The defect is manifested in cells incubated at about 32-37° C. Cells expressing the ΔF508 mutant that are incubated at lower growth temperatures (e.g. below 30° C.) properly process at least a portion of the mutant protein and now exhibit forskolin stimulated chloride transport. Denning, G. M. et al., <i>Nature, </i>358:761 (1992).</p>
    <p>The wild type ΔF508 CFTR protein is normally core-glycosylated in the endoplasmic reticulum, and terminally glycosylated in the Golgi and the plasma membrane. The mature wild type CFTR has a molecular weight of about 160 kDa. Newly synthesized ΔF508 CFTR is retained at the level of the endoplasmic reticulum, and, therefore, is not terminally glycosylated. Accordingly, only core glycosylated, immature forms of the ΔF508 CFTR protein having a molecular weight of about 140 kDa are observed at the nonpermissive temperature (e.g., 37° C.) in NIH 3T3 cells transfected with the ΔF508 mutant (FIG. <b>1</b>). The presence of terminally glycosylated 160 kDa forms of the ΔF508 CFTR is indicative that the endoplasmic reticulum block of the mutant ΔF508 CFTR has been overcome.</p>
    <p>Nucleic acids encoding ΔF508 CFTR protein or other selected genes are introduced into cells by methods known in the art. (Although the following discussion focuses on ΔF508, it is applicable to any other gene introduced into and expressed in engineered cells.) Standard eukaryotic transduction methods are used to produce cell lines which express ΔF508 CFTR protein. It is expected that those of skill in the art are knowledgeable in the numerous systems available for cloning and expressing nucleic acids.</p>
    <p>The expression of natural or synthetic nucleic acids is typically achieved by operably linking a nucleic acid of interest (e.g., one encoding ΔF508) to a promoter (which is either constitutive or inducible) and incorporating the construct into an expression vector. The vectors are suitable for replication and integration in prokaryotes, eukaryotes, or preferably both. Typical cloning vectors contain transcription and translation terminators, transcription and translation initiation sequences, and promoters useful for regulation of the expression of the particular nucleic acid. The vectors optionally comprise generic expression cassettes containing at least one independent terminator sequence, sequences permitting replication of the cassette in eukaryotes, or prokaryotes, or both, (e.g., shuttle vectors) and selection markers for both prokaryotic and eukaryotic systems. See, Giliman, <i>Gene, </i>8:81-97 (1979); Roberts et al., <i>Nature, </i>328:731-734 (1987); Berger et al., <i>Guide to Molecular Cloning Techniques, Methods in Enzymology</i>, volume 152, Academic Press, Inc., San Diego, Calif. (Berger); Sambrook et al. (1989), Molecular Cloning—A Laboratory Manual (2nd ed.) Vol. 1-3, Cold Spring Harbor Laboratory, Cold Spring Harbor Press, N.Y., (Sambrook); and F. M. et al., Current Protocols in Molecular Biology, eds., Current Protocols, a joint venture between Greene Publishing Associates, Inc. and John Wiley &amp; Sons, Inc., (1994 Supplement) (Ausubel). Product information from manufacturers of biological reagents and experimental equipment also provide information useful in known biological methods. Such manufacturers include the SIGMA chemical company (Saint Louis, Mo.), R&amp;D systems (Minneapolis, Minn.), Pharmacia LKB Biotechnology (Piscataway, N.J.), CLONTECH Laboratories, Inc. (Palo Alto, Calif,), Chem Genes Corp., Aldrich Chemical Company (Milwaukee, Wis.), Glen Research, Inc., GIBCO BRL Life Technologies, Inc. (Gaithersburg, Md.), Fluka Chemica-Biochemika Analytika (Fluka Chemie AG, Buchs, Switzerland), and Applied Biosystems (Foster City, Calif.), as well as many other commercial sources known to one of skill.</p>
    <p>Once a nucleic acid is synthesized or isolated and inserted into a vector and cloned, one may express the nucleic acid in a variety of recombinantly engineered cells known to those of skill in the art. Expression of an exogenous nucleic acid can be enhanced by including multiple copies of, for example, a ΔF508 CFTR-encoding nucleic acid in an engineered cell, by selecting a vector known to reproduce in the host, thereby producing large quantities of protein from exogenous inserted DNA (such as pUC8, ptac12, or pIN-III-ompA1, 2, or 3), or by any other known means of enhancing protein expression. ΔF508 CFTR molecules will be expressed when the DNA sequence is functionally inserted into a vector. “Functionally inserted” means that it is inserted in proper reading frame and orientation and operably linked to proper regulatory elements. Typically, a ΔF508 CFTR gene will be inserted downstream from a promoter and will be followed by a stop codon (although production as a hybrid protein followed by cleavage may be used, if desired).</p>
    <p>A number of cell types are useful for the practice of this invention and are readily available. These include cells of fibroblastic or epithelial origin which do not express endogenous CFTR. These cells may be of rodent, feline, canine, bovine, equine, ovine, nonhuman primate, or human origin. The cells may be primary isolates, immortalized, or genetically modified using techniques known to those of skill in the art.</p>
    <p>The culture of cells is well known in the art. Freshney (1994) (<i>Culture of Animal Cells, a Manual of Basic Technique, third edition </i>Wiley-Liss, New York), Kuchler et al. (1977) <i>Biochemical Methods in Cell Culture and Virology</i>, Kuchler, R. J., Dowden, Hutchinson and Ross, Inc., and the references cited therein provides a general guide to the culture of cells. Cultured cell systems often will be in the form of monolayers of cells, although cell suspensions are also used.</p>
    <p>There are several well-known methods of introducing nucleic acids into animal cells, any of which may be used in the present invention. These include: calcium phosphate precipitation, fusion of the recipient cells with bacterial protoplasts containing the DNA, treatment of the recipient cells with liposomes containing the DNA, DEAE dextran, receptor-mediated endocytosis, electroporation, micro-injection of the DNA directly into the cells, infection with viral vectors, etc.</p>
    <p>In example 1 below, we have identified chemical compounds which elicit the proper folding and maturation of the ΔF508 CFTR mutant. These include low molecular weight compounds such as glycerol, dimethylsulfoxide (DMSO) and deuterated water, all of which stabilize proteins against thermal denaturation, both in vitro (Kresheck, G. C. et al., <i>J. Phys. Chem., </i>69:3132 (1965); Back, J. F. et al., <i>Biochem., </i>18:5191 (1979); Gerlsma, S. Y. et al., <i>Int. J Pept. Proteins Res., </i>4:372 (1972); Gekko, K. et al., <i>J. Biochem., </i>94:199 (1983)) as well as in vivo (Lin, P. S. et al., <i>J. Cell. Physiol., </i>108:439 (1981); Henle, K. J. et al., <i>Cancer Res., </i>43:1624 (1983); McIver, D. J. L. et al., <i>Physiol. Chem. Phys., </i>12:369 (1980); Edington, B. V. et al., <i>J. Cell. Physiol., </i>139:219 (1989)). Likewise, methylamines such as trimethylamine N-oxide (TMAO) protect proteins from denaturation by urea and high salt. Somero, G. N., <i>Am. J. Physiol., </i>251:197 (1986).</p>
    <p>4. Assays Utilizing ts20 Cells</p>
    <p>ts20 cells express a temperature sensitive mutant of the E1 enzyme, a component of the ubiquitin pathway. The E1 enzyme is rendered inactive at the non-permissive temperature (39.5° C.). As a consequence, these cells are unable to carry out ubiquitindependent protein degradation events when maintained at 39.5° C. (Kulka, R. G., et al., 1988, <i>J. Biol. Chem. </i>263:15726-15731; Chowdary, D. R., et al., 1994, <i>Mol. Cell. Biol. </i>14:1997-2003). ts20 cells do not proliferate when exposed to the non-permissive temperature.</p>
    <p>As demonstrated in Example 5 below, contact with chemical chaperones causes non-proliferating ts20 cells to proliferate. In example 5, ts20 cells growing at 32.5° C. were plated on 60 mm dishes and maintained at this temperature for 16 h. Cells expressing the ts E1 enzyme were then incubated at either 32.5° C. (the permissive temperature) or at 39.5° C. (the non-permissive temperature) in the presence or absence of protein stabilization agents. Cells incubated at 32.5° C. in regular culture medium displayed an exponential rate of growth, while those maintained at 39.5° C. failed to grow.</p>
    <p>Because the measure of effectiveness of a given test substance is the proliferation of ts20 cells, the ts20 system not only identifies substances that act as chemical chaperones, but it simultaneously screens out non-toxic compounds. Even if a test substance stabilizes favorable E1 conformations, if the test substance is otherwise toxic, the ts20 cell will not proliferate.</p>
    <p>Proliferation can be measured in a number of ways known in the art: rate of degree of thymidine incorporation, direct cell counts, mitotic indices, etc. Generally, test substances that, at 39.5° C. and a given concentration of the test substance, result in a rate of thymidine incorporation, a cell count, or a mitotic index that is at least 2 times higher, preferably at least ten times higher, and most preferably at least fifty times higher than that of ts20 cells maintained at 39.5° C. are classified as possible protein stabilization agents. Their ability to stabilize proteins can be directly tested as described above.</p>
    <p>5. Assays for Detecting Conformational Variants of Prion Proteins</p>
    <p>Another method for assessing conformational changes is useful to detect an interference in the conversion of prion proteins to pathogenic forms. The available evidence indicates that prion diseases (e.g., kuru, Creutzfeld-Jakob disease, Gerstmann-Straussler Scheinker Syndrome, fatal family insomnia in humans, scrapie and spongiform encephaly in animals) arise from a conformationally abnormal form of the prion protein (PrP<sup>C</sup>). Converging lines of evidence indicate that the conformational abnormality involves a transition of the α-helical form of PrP<sup>C </sup>into one predominantly consisting of a β-sheet, thereby giving rise to the pathogenic scrapie isoform (PrPsc). Prion-like proteins have been detected in yeast. Wickner, <i>Science </i>264:566 (1994).</p>
    <p>Conversion of PrP<sup>C </sup>to PrP<sup>Sc </sup>is reportedly accomplished in at least two ways. in infectious prion diseases, exogenously introduced PrP<sup>Sc </sup>is reported to interact with and convert cellular PrP<sup>C </sup>into the scrapie isoform. In inherited or familial cases, a mutation in the PrP gene likely causes a destabilization of PrP<sup>C </sup>which causes some of it to spontaneously convert into PrP<sup>Sc</sup>.</p>
    <p>In either case, conversion of PrP<sup>C </sup>to PrP<sup>Sc </sup>is accompanied by biochemical changes. For example, PrP<sup>C </sup>is soluble in buffers containing nonionic detergents and is sensitive to digestion by added proteases. In contrast, the majority of PrP<sup>Sc </sup>appears detergent soluble and is relatively more protease resistant.</p>
    <p>In Example 2 below, we have devised an assay for rapidly assessing the relative proportion of PrP<sup>C </sup>and PrP<sup>Sc </sup>within a cell or tissue lysate, and have used this assay to identify protein stabilizing agents that interfere with or prevent the conversion of PrP<sup>C </sup>to PrP<sup>Sc</sup>. In general, the method comprises (1) incubating a cell line or tissue capable of converting PrP<sup>C </sup>to PrP<sup>Sc</sup>for between one hour to several days in the presence and absence of a defined concentration (0.1-1 M) of a protein stabilizing agent; (2) mixing the cell line or tissue with a solution wherein only one form of PrP, either PrP<sup>C </sup>or PrP<sup>Sc</sup>, is soluble; (3) separating the form of PrP that is soluble from the form that is insoluble; (4) determining the relative amounts of each form of PrP; and (5) determining whether the test substance prevents the conversion of PrP<sup>C </sup>to PrP<sup>Sc</sup>. One example of a solution that solubilizes preferentially only one form of PrP is a cold (e.g., about 4° C.) solution that comprises about 1% Triton X-100 and about 1% DOC. The soluble and insoluble forms of PrP may be separated by a separation method, such as centrifugation at 15,000-50,000×g for 10-30 min. at 4° C. The PrP in each fraction (supernatant and pellet) may then be quantitated by methods known in the art, such as immunoblotting, ELISA, protein assays, etc. In the above example, most of PrP<sup>C </sup>partitions into the soluble phase, while most of the PrP<sup>Sc </sup>partitions into the insoluble phase.</p>
    <p>C. Methods for the Identification and Correction of Diseases Involving Improper Folding and Processing</p>
    <p>Cystic fibrosis is but one of a number of genetic diseases which arise due to specific mutations that ultimately lead to protein folding errors (reviewed in Thomas, P. J., et al., 1995, <i>TIBS </i>20:456-459; Welch, W. J. et al., 1996, <i>Cell Stress </i>&amp; <i>Chap. </i>1:109-115.); Brown et al., 1996, <i>Cell Stress </i>&amp; <i>Chap. </i>1:117-125.). In addition to demonstrating the feasibility of practicing the invention to treat CFTR and diagnose the presence and severity of prion disease, the present invention also provides methods for identifying and correcting phenotypic defects and medical conditions that are not specifically disclosed herein that are the result of improper protein folding and processing.</p>
    <p>In brief, a cell line that is believed to harbor an improperly folded and processed protein that results in a phenotypic defect is exposed to a defined range of concentrations of a known protein stabilizing agent. Candidate cells often (but not always) are cells that express a temperature-sensitive defect, many examples of which have been reported in the literature (e.g., the tumor suppressor gene p53, Martinez et al., <i>Genes and Development </i>5: 151-159 (1991); the G2 phase mutant CDC2, Th'ng et al., <i>Cell </i>63: 313-324 (1990); mutants of the low density lipoprotein (LDL) receptor, Hobbie et al., <i>J. Biol. Chem. </i>269: 20958-20970 (1994); the gene product of the v-src oncogene, Maroney et al., <i>Oncogene </i>7: 1207-1214 (1992); mutants of human P glycoprotein, Loo et al., <i>J. Biol. Chem. </i>269: 28683-28689 (1994); the oncogene protein pp60<sup>src</sup>; an enzyme involved in ubiquitination, etc.). The phenotypic trait is measured before and after exposure to the protein stabilizing agent. The goal of the treatment is to induce the cells to adopt a wild type phenotype for the trait in question. The degree to which the cells attain or approximate a wild type phenotype is a measure of the success of the treatment.</p>
    <p>How the phenotypic trait is measured varies as a function of the trait. Where the phenotypic defect results in a protein that retains biological activity (e.g. ability to bind a ligand with similar binding parameters as a wild type protein; enzymatic activity) but is not properly compartmentalized, the appropriate measurement is the extent to which the protein is correctly localized after exposure to a protein stabilizing agent. Where the phenotypic defect involves incorrect post-translational modification or loss of biological activity, a proper measure is, respectively, ability to be correctly processed, or recovery of wild type biological activity. Where the phenotypic defect involves a primarily visual trait (e.g, color, shape, size, cellular morphology, etc.), the appropriate measure is the degree to which the defective cell attains a wild-type color, shape, size or cellular morphology.</p>
    <p>For example, Alzheimer's disease involves the proteolytic cleavage of a ˜140 kDa amyloid precursor protein into a 40-43 kDA β-amyloid protein, and the subsequent deposition of β-amyloid protein into amyloid plaques. It is reported that plaque formation involves aggregation of the β-amyloid protein. (Kischner, D. A. et al., <i>PNAS, </i>83:503 (1986)). Other diseases involving deposition of “amyloid” proteins are known. It is expected that amyloid deposition is prevented by exposure to chemical chaperones. We note that U.S. Pat. No. 5,276,059 to Caughey et al. reported that Congo Red was useful to treat conditions involving amyloid deposition. We have been unable to repeat this observation.</p>
    <p>Protein stabilizing agents used to remedy phenotypic defect that result from folding include glycerol, DMSO, sorbitol and other “protein stabilizing agents” detected as described in the preceding section.</p>
    <heading>EXAMPLES</heading> <p>Examples 1 and 2 below demonstrate that protein stabilizing agents counter defects caused by certain folding variants of the CFTR gene product and the scrapies protein, without destroying the cells exposed to the agent. Examples 3-5 show that protein stabilizing agents also are effective for correcting other temperature sensitive protein folding mutants. For these studies, cell lines expressing 3 different temperature sensitive folding mutants were examined: the tumor suppressor protein p53; pp60<sup>src</sup>, the transforming protein encoded by Rous sarcoma; and an enzyme, E1, which catalyzes the first step in the pathway of protein ubiquitination. Treatment with the different protein stabilizing agents resulted in the cells adopting a wild type phenotype at the nonpermissive temperature, indicative that the particular protein folding defect had been corrected. Thus, mutations which result in temperature sensitive protein folding defects are amenable to correction in vivo via the use of protein stabilizing agents. These results have broad implications as they relate to the correction of protein folding defects associated with certain genetic diseases.</p>
    <heading>Example 1</heading> <p>Correction of ΔF508 CFTR Mutants with Chemical Chaperones</p>
    <p>Example 1 demonstrates the feasibility of practicing the invention, using as a model system a cell line genetically engineered to express a cystic fibrosis transmembrane chloride regulator protein (CFTR) mutant that exhibits folding defects. We provide four specific examples of protein stabilizing agents that improve phenotypic defects associated with the AF508 mutant.</p>
    <heading>Example 1A</heading> <p>ΔF508 CFTR and Glycerol</p>
    <p>Example 1A demonstrates that incubation of cells with protein stabilizing agent glycerol promotes the maturation and transport to the plasma membrane of the AF508 CFTR protein. The first part of this example is a description of the CFTR system, in particular the ΔF508 mutant The second part relates to the effect of glycerol on the ΔF508 CFTR.</p>
    <p>The effect of glycerol was measured by detecting either (1) the maturation of immature core-glycosylated endoplasmic reticulum forms of the ΔF508 CFTR to mature terminally glycosylated forms, or (2) the presence of forskolin-dependent chloride transport.</p>
    <p>To study glycosylation patterns, a rabbit polyclonal antibody specific for the R domain of the CFTR protein was produced. A GST fusion protein containing the R domain (amino acids 590-830) of CFTR was prepared by (1) excising from a full length CFTR cDNA sequence a BamH1-HindIII fragment encoding amino acids 590-830; and then (2) ligating this restriction fragment into plasmid pGEX-KG (Stratagene, San Diego) which, when expressed, produces a glutathione S transferase. The resulting ligated plasmid, when expressed in bacteria, yields a CFTR-GST fusion protein which contains a thrombin cleavage site between the GST affinity tag and the CFTR R domain.</p>
    <p>To purify the fusion protein, the cells were lysed in buffer containing lysozyme. Inclusion bodies enriched in the recombinant protein were separated via centrifugation at 6500×g for 10 min. at 4° C. The pellet containing inclusion bodies was washed with a buffer containing 0.5 M NaCl, 0.5% Triton X-100, 0.5% sodium deoxycholate, and 10 mM dithiotreitol. The material was repelleted via centrifugation and resuspended in 8 M urea. After dialysis to remove the bulk of the urea, the protein was partially purified by affinity chromatography using a glutathione column, followed by preparative SDS polyacrylamide electrophoresis.</p>
    <p>The protein was then eluted out of the gel, mixed with Freund's complete adjuvant, and injected into rabbits. After two subsequent boosts of the antigen (using Freund's incomplete adjuvant), serum was obtained, characterized, and shown to be specific for the CFTR protein or CFTR domain containing amino acids 590-830.</p>
    <p>The polyclonal antibody was initially characterized by its ability to immunoblot the GST-R fusion antigen, as well as immunoprecipitate the full length CFTR protein synthesized in vitro. The polyclonal antibody also immunoblotted CFTR expressed in various cell lines (T84, CaCo), in stable transfectants, or in transiently transfected SF9 cells. (Data not shown) For comparative purposes, a mouse monoclonal specific for the C terminus of the CFTR protein was purchased from Genzyme.</p>
    <p>Western blotting using a commercially available anti-CFTR monoclonal antibody (Panel 1A) or our polyclonal anti-CFTR antibody (Panel 1B) was performed using cell lysates from NIH 3T3 cells transfected with either the wild type or AF508 forms of CFTR Andersen, M. P. et al., <i>Science, </i>253:202 (1991). Both the polyclonal and monoclonal antibodies recognized the 140 kDa (core glycosylated, or B form) and 160 kDa (filly glycosylated, C form) forms of CFTR in lysates from wild type CFTR expressing cells. In contrast, only the 140 kDa core glycosylated CFTR was detected by the two antibodies in the lysates from cells transfected with the ΔF508 CFTR mutant.</p>
    <p>In panel 1C, cells expressing the ΔF508 mutant were examined by Western blotting using the rabbit polyclonal antibody following their treatment at 37° C. with 1 M glycerol for 3 days. As a positive control, the ΔF508 CFTR expressing cells were incubated at 26° C. for 3 days.</p>
    <p>As shown in FIG. 1, incubation of the ΔF508 mutant expressing cells at 26° C. resulted in the appearance of the mature form of the CFTR protein. Similar to the effects of lower temperature, cells incubated at 37° C. in the presence of 1 M glycerol for a period of 3 days now contained significant amounts of the mature ΔF508 CFTR protein (FIG. <b>1</b>C).</p>
    <p>The results of experiments to determine the optimal concentration of glycerol and the duration of glycerol treatment needed to elicit the highest levels of mutant ΔF508 CFTR maturation are summarized in FIG. <b>2</b>. In panel 2A, cells expressing ΔF508 CFTR were incubated at 37° C. for 3 days in culture medium containing various concentrations of glycerol. The cells were harvested and then analyzed by Western blotting using the rabbit polyclonal anti-CFTR antibody. In panel 2B, cells expressing ΔF508 CFTR were incubated with 1.25 M glycerol at 37° C. for 1, 2 or 3 days and then analyzed by Western blotting as described in A. Glycerol concentrations of 1-1.25 M gave the maximal amount of mutant protein which was properly processed. Lower doses resulted in less of the fully glycosylated protein and higher doses were inhibitory, perhaps due to adverse effects on the cells. At 1.25 M glycerol, a time course study showed maximal amounts of the fully glycosylated ΔF508 CFTR protein after 3 days of treatment.</p>
    <p>Forskolin-sensitive chloride transport was measured by the fluorescence assays method (FIG. <b>3</b>). Briefly, ΔF508-CFTR expressing cells were grown on glass coverslips at 37° C. in the absence or presence of 1.25 M glycerol for 3 days. These cells, along with cells expressing wild type CFTR, then were examined for forskolin stimulated chloride transport using SPQ fluorescence assays, as described in: Verkman, A. S., <i>Am. J. Physiol., </i>259:C375 (1990); Verkman, A. S. el al., in <i>Methods in Neurosciences</i>, Kraicer, J. et al., Eds. (Academic Press, 1995) vol. 27, 328. Plasma membrane chloride efflux was measured from the time course of increase in SPQ fluorescence in response to replacement of intracellular chloride by nitrate. Cells in the presence or absence of a protein stabilizing agent were incubated with 2 mM SPQ for 12-18 hrs. prior to measurements. The rate of chloride efflux was measured before and after stimulation of cAMP with forskolin. The reaction was quenched by the addition of 150 mM KSCN.</p>
    <p>In the absence of glycerol, ΔF508 CFTR-expressing cells showed little or no chloride transport. In contrast, after one day of incubation in the presence of 1.25 M glycerol, forskolin stimulated chloride transport was observed. Thus, despite an inability to observe, via Western blotting, any significant levels of the mature ΔF508 CFTR protein in the cells following 1 day of glycerol treatment, the more sensitive fluorescence assay indicated a partial restoration of chloride conductance after one day of glycerol treatment.</p>
    <p>The positive effect of glycerol treatment on chloride conductance was concentration and time dependent (FIG. <b>3</b>B). ΔF508 CFTR-expressing cells growing on glass coverslips were incubated with various concentrations of glycerol for 3 days and then examined for forskolin stimulated chloride transport. Alternatively, cells were incubated with 1.25 M glycerol for varying times (1, 2 or 3 days) and then analyzed for chloride transport following stimulation by forskolin. Increased chloride transport was observed for cells exposed to as little as 0.5 M glycerol for 3 days. Maximal rates of chloride efflux was observed in those cells incubated in 1.25 M glycerol for a period of 3 days, consistent with the results obtained using Western blotting. Glycerol exposure of the cells expressing the wild type CFTR protein had little or no effect on the amount of the mature protein (via Western blotting) or their forskolin dependent chloride conductance (data not shown). Mock-transfected 3T3 cells, either in the absence or presence of glycerol, did not display significant cAMP stimulated chloride movement (FIG. <b>3</b>B).</p>
    <heading>Example 1B</heading> <p>ΔF508 CFTR and Deuterated Water</p>
    <p>Applying essentially the same procedures as those used in Example IA, we demonstrated that deuterated water was a protein stabilizing agent, and that it was competent to induce CFTR processing and forskolin dependent chloride transport in ΔF508 CFTR-expressing cells.</p>
    <p>Powdered DMEM medium (Gibco) was reconstituted in 100% distilled water or in 100% dH<sub>2</sub>O (Sigma). Cells transfected with ΔF508 CFTR or mock transfected cells were incubated in either the distilled water or the dH<sub>2</sub>O containing culture medium for two days at 37° C., and then analyzed for forskolin dependent chloride efflux. As shown in FIG. 4, ΔF508 CFTR expressing cells incubated in the presence of deuterated water (dH<sub>2</sub>O) exhibited forskolin-dependent chloride transport.</p>
    <heading>Example 1C</heading> <p>AF508 CFTR and TMAO</p>
    <p>Trimethylamine N-oxide (TMAO) is a naturally occurring cellular osmolyte which accumulates in the cells of cartilaginous fish (e.g., sharks) in response to high concentrations of the protein chaotrope urea. Methylamines like TMAO are commonly used by a variety of organisms to withstand the protein denaturing effects of high concentrations of urea and/or salt. For example, elasmobrachs (sharks, rays) accumulate significant amounts of TMAO in their tissues (˜200 mM), to counteract the denaturing effects of high intracellular concentrations (˜400 mM) of urea (Somero, G. N., <i>Am. J. Physiol., </i>251:R197 (1986)). Thus, it appears that this class of compounds stabilizes proteins.</p>
    <p>Both mock transfected cells and the cells transfected with ΔF508 CFTR were incubated in regular DMEM culture medium containing 100 mM TMAO for two days at 37° C. and then analyzed for forskolin dependent chloride efflux. Forskolin dependent chloride transport was observed for the mutant cells treated with 100 mM TMAO for a period of 2 days (FIG. <b>5</b>).</p>
    <heading>Example 1D</heading> <p>ΔF508 CFTR and DMSO</p>
    <p>Similar results as those shown in FIGS. 1-3 were obtained when the cells were exposed for two days to DMEM medium supplemented with 2% dimethylsulfoxide (data not shown).</p>
    <heading>Example 1E</heading> <p>Further Confirmation that Treatment with Mammalian Cellular Osmolytes Promotes the Proper Processing and Function of the ΔF508 CFTR Protein.</p>
    <p>In Panel A of FIG. 22, 3T3 cells expressing the ΔF508 CFTR protein (via stable transfection) were treated with either 100 or 200 mM of the specified cellular osmolytes for 2 days. Cells were then lysed and examined by Western blot analysis. Wt represents cells expressing the wild type CFTR protein. B, G, T, S and M indicated cells treated with betaine, glucose, taurine, sorbitol and myoinositol, respectively.</p>
    <p>FIG. 22A shows that cells expressing the ΔF508 mutant protein, when exposed to the different cellular osmolytes, now contain some of the mature protein (band C), similar to the wild type cells. Note the C band in the wild type cells (wt), no C band in the mutants (indicated with a—over the lane), but now with the different osmolyte treatments we see some C band in the cells expressing the mutant protein.</p>
    <p>In FIG. 22B, the effects of osmolyte treatment on chloride conductance were determined following similar osmolyte treatments as described in panel A. FIG. 22B shows that AF508 cells (control) fail to transport any appreciable chloride ions in response to stimulation with forskolin (plotted as a percentage of chloride transport relative to cells expressing wild type protein). In contrast, cells treated with 4 different osmolytes now show partial chloride transport (note however that it never reaches the wild type levels, rather only about 10-30% of wild type).</p>
    <p>To determine the effects of osmotic treatment on cellular growth, cells growing on 60 mM dishes were maintained in the presence of absence of 200 mM taurine, sorbitol or myo-inositol. Some plates were counted prior to the addition of osmolytes, while the remainder were counted on a daily basis. FIG. 22C shows that the different treatments are not deleterious to the cells as indicated by continued cell proliferation.</p>
    <p>FIG. 23 shows that the effects of osmolytes on the processing of the ΔF508 CFTR protein are concentration and time dependent. In Panel A, cells expressing ΔF508 CFTR were incubated in the presence or absence of varying concentrations (25, 50, 100 mM) of osmolyte. After 2 days of growth, the cells were harvested and analyzed for their content of immature (band B) and mature (band C) forms of the CFTR protein by Western blotting. Cells expressing wild type CFTR (wt) were included as a positive control. In Panel B, cells expressing ΔF508 CFTR were incubated for 1 (1D), 2(2D) or 3 (3D) days in growth medium containing 150 mM of the indicated osmolyte. As a control, the same cells were incubated in normal growth medium. FIG. 23A shows a concentration curve of osmolyte treatment of the cells expressing the mutant protein-note that only the higher concentration (100 mM) appeared effective to elicit the appearance of the mature (band C) form of the CFTR protein. FIG. 23B shows that over time in the cells expressing the mutant protein that the positive effects of the osmolytes diminishes. After 3 days of continuous treatment with 150 mM osmolyte there is see a “drop-off” of the amount of the C band.</p>
    <p>In FIG. 24, cells expressing the ΔF508 CFTR protein were treated with the specified osmolyte for 1 D. At the end of this period, some cells were harvested (1), while others were washed and further incubated for 1 D in normal growth media (R). A third group of cells was treated for I day, washed and recovered for 1D and then given another 1D treatment with the specified osmolyte. FIG. 24A shows that the effects of the osmolytes are reversible.</p>
    <p>FIG. 25 shows that 3T3 cells stably expressing the delta ΔF508 CFTR protein can adapt to hyperosmotic conditions and accumulate cellular osmolytes. 3T3 cells were gradually adapted over a period of weeks to growth in hyperosmotic media (DMEM containing an additional 250 mM NaCl and 250 mM urea). Adapted cells (3T3a) and cells maintained at isosmotic conditions (3T3) were acid extracted and the extracts were examined by HPLC to determine intracellular osmolyte accumulation. FIG. 25A shows that when cells expressing the mutant AF508 CFTR protein are incubated under hyperosmotic conditions (high salt) the cells begin to accumulate different osmolytes and to different levels. This effect is not unique to the cells expressing the mutant protein but also occurs in those cells expressing the wild type protein (data not shown).</p>
    <p>FIG. 26 shows that osmotic adaptation results in a partial correction of ΔF508 CFTR processing and function In Panel A, 3T3 cells stably expressing the ΔF508 CFTR protein were osmotically adapted as described above. Both adapted and non-adapted control cells were lysed and examined by Western blot analysis to determine the processing status and content of immature (band B) and mature (band C) forms of the CFTR protein. As a positive control, cells expressing the wild type CFTR were also examined. In Panel B, chloride conductance analysis were conducted on control (cells maintained in isosmotic media) osmotically adapted cells, and cells treated for 2D with 1M glycerol. The results of these experiments are expressed as the percentage of forskolin stimulated chloride conductance as compared to cells expressing wild type CFTR.</p>
    <p>FIG. 26A shows that 3T3 cells expressing ΔF508 CFTR protein, when treated under hyperosmotic conditions as in FIG. 25A, now show some correct folding of the ΔF508 CFTR protein. Specifically, note that in the +NaCl and Urea the C band is seen (but only a fraction compared to the cells expressing the wild type protein). FIG. 26B shows that cells treated as described above in 26A now show some restoration of chloride transport (plotted again as % of that observed for wild type).</p>
    <p>FIG. 27 shows the adaptation of kidney derived cells to high salt and urea conditions results in the accumulation of cellular osmolytes. In Panel A, mlMCD cells were gradually adapted over a period of weeks to growth in hyperosmotic media. These cells (designated Ht500) were lysed and processed as described above to determine intracellular osmolyte concentrations. In Panel B, the effects of osmotic stress on cellular growth rates were determined using mlMCD cells maintained in either isomotic (mcd) or hyperosmotic (ht) conditions. As a comparison, the growth rates of 3T3 cells and adapted 3T3 (3T3a) cells were also examined under the same conditions. Control cells maintained in isomotic media and adapted cells were plated on 60 mM dishes and counted at daily intervals. FIG. 27A shows that kidney cells subjected to high salt conditions (hyperosmotic) accumulate cellular osmolytes. Med refers to the cells grown under normal conditions (no osmotic shock) while the ht refers to the same kidney cells, but grown under high salt conditions. FIG. 27B (top panel) shows that the kidney cells exhibit slower rate of proliferation when subjected to hyperosmotic conditions. The (bottom panel) shows that 3T3 cells exhibit a significant drop in cellular proliferation when subjected to hyperosmotic conditions.</p>
    <p>FIG. 28 demonstrates that adaptation to high salt and urea conditions promotes the proper processing and function of the =ΔF508 CFTR protein. mlMCD and Ht500 cells were transiently transfected with plasmids containing either the wild type or ΔF508 CFTR sequence. Following a 48 h recovery period, the cells were lysed and examined by Western blot analysis to determine the expression level and processing status of the CFTR protein. In Panel A, mlMCD cells and Ht500 cells that were either mock transfected, or transfected with the ΔF508 CFTR or wild type CFTR plasmid. In Panel B, chloride conductance rates of Ht500 cells that were mock transfected or transfected with the ΔF508 CFTR plasmid. In FIG. 28A, the kidney cells used above were maintained under normal conditions (MCD) or were adapted to high salt (HT500=hyperosmotic) conditions. Then the gene encoding, either the wild type or ΔF508 CFTR proteins, were transfected into the cells. Note that the ΔF508 protein expressed in the MCD cells is only the immature B band, while those cells expressing the wild type protein show the mature and functional C band. Now when adapted to high salt conditions, the cells transfected with the gene encoding ΔF508 CFTR now show the mature C band form of the protein, identical to that observed for the cells expressing the wild type protein. (This may explain why patients with the disease (ΔF508) show little or no abnormalities in the kidney. the natural high salt conditions found in the kidney may result in osmolyte accumulation and therefore correct folding of the mutant protein.</p>
    <p>In FIG. 28B, the salt adapted kidney cells (above) expressing ΔF508 CFTR protein are able to transport chloride ions. An increase in chloride transport was observed only in the salt adapted kidney cells expressing the ΔF508 CFTR protein as compared to the mock transfected cells.</p>
    <heading>Example 2</heading> <p>Prions</p>
    <p>This example illustrates that incubation of scrapie-infected mouse neuroblastoma cells with protein stabilizing agents used in Example 1 results in a significant reduction in the rate and extent of conversion of the prion protein to the infective scrapie form. See also, Tatzelt et al., 1996, <i>EMBO J. </i>15: 6363-6373.)</p>
    <p>Cell Culturing and Metabolic Labeling</p>
    <p>N2a and ScN2a cells were grown in MEM Eagles medium containing 10% fetal calf serum. For steady state labeling, cells were incubated with 200 μCi/ml [<sup>35</sup>S]-methionine/cysteine (Amersham) for 12-15 hours in medium consisting of methionine and cysteine free DME supplemented with 5% fetal calf serum and 5% complete MEM Eagles medium. For pulse-chase experiments cells were labeled with 400 μCi/ml [<sup>35</sup>S]-methionine/cysteine in methionine and cysteine free DME for the time periods indicated in the figure legends.</p>
    <p>Harvesting of the Cells and Detergent Fractionation</p>
    <p>Cells were harvested by removing the culture medium and subsequent washing with cold phosphate buffered saline (PBS). The cells were lysed in cold (4° C.) buffer A (1% Triton X-100, and 1% sodium deoxycholate in PBS). Following scraping of the cells off the culture dish, the lysate was centrifuged at 15,000×g for 20 min. at 4° C. The supernatant was removed and adjusted to a final 1% Sarkosyl, while the pellet was solubilized in buffer A supplemented with 1% Sarkosyl (buffer B). To an equal percentage of the supernatant and pellet, protease K at the concentrations indicated in FIG. 1, was added and the reactions performed at 37° C. for 30 min. Reactions were terminated by the addition of concentrated Laemmli sample buffer (such that the final concentration of SDS was 1%) and the samples immediately heated at 100° C. for 5 minutes.</p>
    <p>Antibodies Used</p>
    <p>Rabbit anti-PrP antibody (R073) was raised against sodium dodecyl sulfate-polyacrylamide gel electrophoresis (SDS-PAGE)-purified Syrian hamster PrP 27-30. Serban et al., <i>Neurology, </i>40:110-117 (1990). Mouse monoclonal anti PrP (3F4) is specific for hamster and human PrP. Kascsak et al., <i>J. Virol., </i>61:3688-3693 (1987). Although it does not recognize mouse PrP it does recognize the MHM2 chimeric form of PrP.</p>
    <p>Immunoprecipitation and Analysis by SDS-PAGE</p>
    <p>For immunoprecipitation, radiolabeled cells were lysed and fractionated as described above. The supernatants and pellets were adjusted to 1% Sarkosyl, and then incubated in the presence of immobilized Protein A for 30 min. After removal of the Protein A-sepharose, the primary antibody was added and the samples incubated for 1.5 hrs at 4° C. The antibody-antigen complexes were captured via the addition of immobilized Protein A and then washed 4 times with buffer B. The immunoprecipitated proteins were released from the Protein A-sepharose by the addition of Laemmli sample buffer containing 1% SDS and heating at 100° C. for 5 min.</p>
    <p>Proteins present within whole cell lysates or detergent fractionated cell lysates, as well as immunoprecipitated proteins, were analyzed by SDS-PAGE. Gels were impregnated with Amplify (NEN DuPont) and exposed to film at −70° C.</p>
    <p>Western Blotting</p>
    <p>Following SDS-PAGE, proteins were transferred to nitrocellulose. The filters were blocked by incubation in PBS containing 5% nonfat dry milk and 0.1% Tween-20 for 1 hr at room temperature. The blots were incubated with primary antibody diluted in PBS containing 0.1% Tween-20 (PBS/Tween) for 12 h at 4° C. After extensive washing with PBS/Tween, the blots were incubated with horseradish peroxidase conjugated secondary antibody (Amersham) for 30 min. at room temperature. Following extensive washing with PBS/Tween, the blots were developed by using an enhanced chemiluminescence substrate (Amersham).</p>
    <p>Preparation of Protein Extracts from Mice Brains and Inoculation of Cell Culture</p>
    <p>Brain homogenates (10% w/v) were prepared in ice cold PBS containing 1% Triton X-100 and centrifuged at 3000×g for 30 min. The pellet was dissolved in buffer A and the centrifigation step was repeated. Proteins present within the supernatant then were precipitated with cold methanol, resuspended in PBS containing 10% glycerol, quick frozen in liquid nitrogen and stored at −80° C. For inoculation experiments 50 μl MHM2 extract in 1 ml medium was added to a 25 cm<sup>2 </sup>dish of 30-50% confluent N2a or ScN2a cells. After 12 hours the cells were washed twice and analyzed with the detergent assay after various time points as indicated in the figure legends.</p>
    <p>Phospholipase Experiments.</p>
    <p>Cells were washed twice and subsequently incubated in serum free medium with 5 U/ml Phosphatidylinositol Phospholipase C (PIPLC, Boehringer Mannheim) for 4 hrs at 37° C. The cells were extensively washed with PBS, lysed with cold buffer A, and fractionated into a supernatant and pellet as described above. PrP present with the detergent soluble and insoluble fractions was analyzed by Western blotting.</p>
    <heading>Example 2A</heading> <p>Differential Detergent Solubility of PrP<sup>C </sup>and PrP<sup>Sc </sup>in Scrapie-Infected N2A Cells.</p>
    <p>In ScN2a cells, approximately 10% of PrP<sup>C </sup>is converted into infectious PrP<sup>Sc </sup>as assayed by its relative resistance to protease K digestion and by bioassays. Borchelt, <i>J. Cell Biol., </i>110: 743-752 (1990). Owing to the potential variability of results using protease digestion assays, along with both the high costs and long times needed for scrapie bioassays, we developed a new assay for monitoring the extent of PrP<sup>Sc </sup>formation in ScN2a cells. Specifically, we wanted an assay which, unlike the protease digestion approach, would also allow for the determination of the relative amounts of PrP<sup>C</sup>.</p>
    <p>Many glycolipidated proteins are relatively insoluble in cold (i.e., about 4° C.) buffers containing 1% Triton X-100 or Triton X-114. Hooper and Bashir, <i>Biochem J., </i>280:745-751 (1991). This is also true for PrP<sup>C</sup>, as illustrated in FIG. <b>5</b>A: N2a and ScN2a cells were solubilized in cold (i.e., about 4° C.) phosphate buffered saline (PBS) containing either 1% Triton X-100 or 1% Triton X-100 plus 1% sodium deoxycholate (DOC). The cell lysates were incubated for 10 min on ice and then centrifuged at 15,000×g for 20 min. The supernatant was removed and adjusted to a final Sarkosyl concentration of 1%. The pellet was resuspended in PBS containing 1% Triton X-100, 1% DOC and 1% Sarkosyl. An equal percentage of both the pellet (Pellet, lanes 1-4) and supernatant (Sup, lanes 5-8) fractions were analyzed by Western blotting using the α-PrP antibody R073.</p>
    <p>Analysis of the resultant supernatant and pellet revealed that the majority of PrP<sup>C</sup>, be it from the N2a or ScN2a cells, partitioned into the detergent insoluble fraction when the cells were lysed in cold buffer containing 1% Triton X-100. If, however, the cells were lysed at about 4° C. in PBS containing both 1% Triton X-100 and 1% sodium deoxycholate (DOC), markedly different results were obtained. Now the vast majority of PrP present within the N2a cells was observed to fractionate within the detergent soluble fraction. Relatively little PrP was found within the detergent insoluble pellet. In contrast, PrP from the ScN2a cells was observed to partition into both the detergent soluble and insoluble fractions. The faster migrating forms of PrP present were relatively enriched within the detergent insoluble fraction obtained from the ScN2a cells.</p>
    <p>To determine whether these faster migrating forms of PrP which were relatively enriched within the detergent insoluble fraction of the ScN2a cells might represent the scrapie isoform, protease digestion of the material present within both fractions was performed (FIG. <b>5</b>B). N2a and ScN2a cells were lysed in cold buffer A (PBS containing 1% Triton X-100 and 1% DOC) and centrifuged exactly as described above. The supernatant was removed and the pellet resuspended in buffer A containing 1% Sarkosyl. To the supernatant and pellet was added protease K (concentrations as indicated in the FIG. ). Following a 30 min. digestion at 37° C., the reactions were terminated via the addition of concentrated Laemmli sample buffer and immediate heating at 100° C. for 5 min. The samples then were examined for PrP via Western blotting using the R073 antibody. Varying amounts of PK were used to examine the supernatant (Sup) derived from either the N2a (lanes 1-4) and ScN2a cells (lanes 5-8), while only a single concentration of PK was used to examine the material present within the pellet (Pellet) of the N2a (lanes 9, 10) and ScN2a (lanes 11, 12) cells.</p>
    <p>In ScN2a cells, the majority of PrP present within the detergent soluble fraction was degraded by added protease K while the majority of PrP present within the detergent insoluble pellet was relatively unaffected by the protease K treatment. In N2a cells, the PrP present in either the detergent soluble or insoluble fraction was completely degraded by added protease.</p>
    <p>Owing to its relative resistance to protease K, we conclude that the detergent insoluble form of PrP present within the ScN2a cells represents primarily PrP<sup>Sc</sup>. These results indicate that PrP<sup>Sc</sup>, in contrast to PrP<sup>C</sup>, preferentially partitions into the detergent insoluble fraction when the cells are lysed in a cold (about 4° C.) physiological buffer containing both 1% Triton X-100 and 1% sodium deoxycholate.</p>
    <p>Using this detergent solubility assay, we examined the conversion of newly synthesized PrP<sup>C </sup>into PrP<sup>Sc </sup>in the ScN2a cells (FIG. <b>5</b>C). N2A (lanes 1,3) and ScN2A cells (lanes 2,4) were labeled with [<sup>35</sup>S]-methionine for 4 hrs. One plate of the radiolabeled cells was harvested immediately (chase 0). To the other radiolabeled cells, the medium was removed and the cells washed with and further incubated in fresh culture medium for 20 hrs (chase 24 h). Both the pulsed and pulsed-chased radiolabeled cells were harvested in buffer A, and the lysates centrifuged at 15,000×g for 20 min. PrP was immunoprecipitated from either the pellet (Pellet, lanes 1,2) or the supernatant (Sup, lanes 3,4) as described in the material and methods. The resultant immunoprecipitates were analyzed via SDS-PAGE and fluorography. Note that the exposure time of the films was 14 hrs for the pulse-labeled material, and 4 days for the pulsed and chased material. As a control in these experiments, the pulse and pulse-chase radiolabeling similarly was performed with the N2a cells. All of the radiolabeled cells then were lysed in PBS containing 1% Triton X-100 and 1% DOC and subjected to centrifugation as described above. An equal percentage of the resultant supernatants and pellets were analyzed for their distribution of PrP via immunoprecipitation.</p>
    <p>In both the 4 hr pulse-labeled N2a and ScN2a cells, all of the newly synthesized PrP was found within the detergent soluble fraction. Similarly, in the N2a cells pulse-labeled and then chased for 20 yrs. the portion of radiolabeled PrP still remaining was found exclusively within the detergent soluble fraction. In contrast, the remaining radiolabeled PrP produced in the ScN2a cells after the 20 hr. chase period was found to partition into both the detergent soluble and insoluble fractions. Again note that the forms of PrP present within the detergent insoluble fraction of the ScN2a cells migrated with a faster mobility relative to that observed for PrP present within the detergent soluble fraction. Results from several experiments indicated that approximately 10-20% of the PrP synthesized in the ScN2a cells was converted into the detergent insoluble PrP<sup>Sc</sup>-like isoform over the course of the 20 hr. chase period.</p>
    <heading>Example 2B</heading> <p>Glycerol Interferes with the Conversion of PrP<sup>C </sup>to PrP<sup>Sc </sup>in ScN2a Cells</p>
    <p>Using the simple, rapid assay for monitoring the in vivo conversion of PrP<sup>C </sup>to PrP<sup>Sc </sup>described in Example 2A, we set out to identify agents that interfere with the conversion event. The following demonstrates that glycerol is one such agent.</p>
    <p>A pulse-chase experiment similar to the one shown in FIG. 5 above was performed with the ScN2a cells, except that the cells first were incubated culture medium either lacking or supplemented with 1 M glycerol for 18 hrs. The cells were labeled with [<sup>35</sup>S]-methionine for 4 hrs, either in the presence or absence of glycerol. The medium containing the radiolabel was removed, and the cells then further incubated in complete medium (plus or minus glycerol) for an additional 20 hrs. The cells were harvested in Buffer A and fractionated into a supernatant and pellet, and analyzed for their distribution of radiolabeled PrP using the detergent solubility assay (FIG. <b>6</b>A). The distribution of PrP within each fraction was determined via immunoprecipitation using the R073 antibody.</p>
    <p>Shown in FIG. 6A is a fluorograph of the gel analyzing the resultant PrP imunoprecipitates. Again, in the absence of glycerol, in the ScN2a cells the PrP<sup>C </sup>was converted into PrP<sup>Sc</sup>, as evidenced by the presence of the radiolabeled protein within the detergent insoluble fraction. In contrast, relatively little of the radiolabeled PrP synthesized in the glycerol treated ScN2a cells was found within the detergent insoluble fraction. Instead, almost all of the remaining radiolabeled PrP now was observed within the detergent soluble fraction.</p>
    <p>We next incubated ScN2a cells in either the absence or presence of 1 M glycerol for a period of 3 days, and then analyzed the cells for their distribution of detergent soluble and insoluble PrP via Western blotting (FIG. <b>6</b>B). In those cells maintained in the presence of glycerol for 3 days, little or no detergent insoluble PrP was observed.</p>
    <p>Both the concentration of glycerol, as well as the duration of glycerol treatment needed to interfere with the conversion of PrP<sup>C </sup>to PrP<sup>Sc </sup>was examined (FIG. <b>7</b>). ScN2a were incubated for 3 days in medium containing differing concentrations (0, 250, 500, 750 and 1,000 mM) of glycerol. As a control, ScN2a cells, plated at the same density, were incubated for 3 days in culture medium not containing any additional components. The cells were harvested in cold buffer A and fractionated into a supernatant and pellet exactly as described earlier. The amount of detergent insoluble PrP was determined by comparative Western blotting using the R073 antibody. Results are expressed relative to the amount of PrP present within the detergent insoluble fraction of the ScN2a cells not exposed to glycerol (i.e., 100%). All values for insoluble PrP<sup>Sc </sup>were normalized to the expression of soluble PrP<sup>C</sup>. Shown are representative data from multiple experiments. (FIG. 7A, round symbols). While 1 M glycerol appeared to be the most effective, a significant reduction in the conversation of PrP<sup>C </sup>to PrP<sup>Sc </sup>also was observed at glycerol concentrations between 0.25-0.5 M.</p>
    <p>Analysis of the optimal time of glycerol treatment to reduce the extent of detergent insoluble PrP in the ScN2a cells is presented in FIG. <b>7</b>B. Here, the ScN2a cells were plated at approximately 10-20% confluency and then incubated in the presence or absence of two different concentrations of glycerol over a period of 1 to 3 days. After only 1 day of treatment with 0.5 M or 1 M glycerol, a vast reduction in the amount of detergent insoluble PrP was observed. In the case of those cells incubated with 0.5 M glycerol, by days 2 and 3 the relative levels of detergent insoluble PrP was found to be slightly on the increase. In contrast, the overall levels of detergent insoluble PrP observed within the ScN2a cells incubated with 1 M glycerol changed only little over the 3 day incubation period.</p>
    <p>A number of control experiments were performed in order to confirm that glycerol was in fact interfering with the conversion of PrP<sup>C </sup>to PrP<sup>Sc</sup>, rather than simply affecting some aspect of PrP<sup>C </sup>biogenesis. First, via metabolic labeling of the ScN2a cells in the presence and absence of glycerol, no significant differences were observed with respect to either the rate, or overall extent of PrP<sup>C </sup>synthesis.</p>
    <p>Next, we examined whether PrP<sup>C </sup>which is synthesized in the presence of glycerol still moves to the plasma membrane and can be released from the cells via treatment with phospholipase. Both N2a and ScN2a cells were cultured in either the absence or presence of glycerol for 24 hrs. The cells then were either mock incubated, or incubated with phospholipase C (PIPLC) (5 U/ml) for 4 hrs. at 37° C. Following their extensive washing with PBS, the cells were collected and analyzed by the detergent solubility assay and Western blotting (FIG. <b>8</b>).</p>
    <p>In both the N2a and ScN2a cells incubated in the absence of glycerol, phospholipase treatment resulted in a significant reduction, or even the complete absence of detergent soluble PrP (FIG. 8, lanes 2 and 6, respectively). This observation, consistent with the results of other studies, indicates that the majority of detergent soluble PrP likely is derived from that portion of PrP<sup>C </sup>which is present at the plasma membrane. Phospholipase treatment of both the N2a and ScN2a cells grown in the presence of glycerol similarly resulted in the disappearance of detergent soluble PrP (FIG. 8, lanes 4 and 8, respectively). With respect to the detergent insoluble PrP, again the ScN2a cells incubated in the presence of glycerol showed a marked reduction in the overall levels of insoluble PrP as compared to the non-glycerol treated cells (FIG. 8, lanes 15 and 13, respectively). Here treatment of the cells with phospholipase had little or no effect on the amounts of detergent insoluble PrP, be it in those cells incubated in the presence or absence of glycerol. Finally, it is interesting to note that the low levels of detergent insoluble PrP observed in the N2a cells (FIG. 8, lane 9 and also see FIG. SB, lane 10) no longer was observed when the cells had been incubated in the presence of glycerol (FIG. 8, lane 11) From these and a number of other control experiments, we conclude that the addition of glycerol to the ScN2a cells does not significantly affect either the overall rate or the absolute amounts of PrP<sup>C </sup>which is synthesized, nor the movement of the newly synthesized protein to the plasma membrane. Rather, we suspect that glycerol somehow is affecting directly the conversion process by which PrP<sup>C </sup>is driven into PrP<sup>Sc</sup>.</p>
    <p>We next examined whether incubation of the cells in glycerol for a period of 3 days might effectively “cure” the ScN2a cells of their ability to carry out the conversion process. Following 3 days of incubation in the presence of 1 M glycerol, the culture medium was removed, the cells extensively washed with fresh culture medium, and then further incubated in the absence of glycerol for 5 days (glycerol treated and reversed). The cells were harvested and analyzed via the detergent solubility assay and subsequent Western blotting (FIG. <b>9</b>A). As was shown in FIG. 6, we were unable to detect significant amounts of detergent insoluble PrP in the ScN2a cells incubated for 3 days in the presence of 1 M glycerol. However, when the 3 day glycerol treated ScN2a cells were subsequently incubated in the absence of glycerol for a period of 5 days, detergent insoluble PrP again was observed (FIG. 9A, lane 6). Note however, that relative to those ScN2a cells which never had been exposed to glycerol (FIG. 9A, lane 5), the amounts of detergent insoluble PrP still was markedly less in the glycerol treated and reversed ScN2a cells.</p>
    <p>Similar results were observed when the experiment was performed using metabolic pulse-chase labeling (FIG. <b>9</b>B). Utilizing a pulse-chase metabolic labeling protocol similar to that used earlier (i.e, the experiment presented in FIG. 6) the ScN2a cells first treated with glycerol for 3 days, and then incubated in the absence of the chemical for 5 days now had regained their ability to convert PrP<sup>C </sup>into PrP<sup>Sc</sup>. Note that relative to the control untreated ScN2a cells, the glycerol treated and reversed cells again exhibited a marked reduction in the overall extent of PrP<sup>C </sup>to PrP<sup>Sc </sup>conversion (FIG. <b>9</b>B).</p>
    <heading>Example 2C</heading> <p>Glycerol Treatment Interferes with the Conversion of Exogenously Added PrP<sup>C </sup>to PrP<sup>Sc </sup>in ScN2a Cells</p>
    <p>We next examined whether glycerol treatment of the ScN2a cells also might interfere with the conversion of exogenously added PrP<sup>C </sup>into PrP<sup>Sc</sup>. Brain extracts enriched in PrP<sup>C </sup>were prepared from transgenic mice expressing a chimeric mouse-hamster form of PrP<sup>C </sup>(MHM2, for Mouse-Syrian Hamster-Mouse chimera). Previous work has established that mice expressing this chimeric form of PrP<sup>C </sup>are susceptible to added mouse scrapie prions, and develop scrapie disease. Scott et al., <i>Cell </i>73:979-988 (1993). In the first experiment, the chimeric PrP<sup>C </sup>was added to the culture medium of both N2a and ScN2a cells incubated in the absence of glycerol. After 12 hrs of incubation, the cells were washed extensively with fresh culture medium to remove any remaining exogenous PrP<sup>C </sup>and the cells then further incubated for an additional 24, 42, or 66 hrs. At each time point the cells were collected and analyzed for their distribution of the exogenously added PrP via the detergent solubility assay and Western blotting (FIG. <b>10</b>A). To follow the distribution of only the exogenously added MHM2 form of PrP, we utilized the 3F4 monoclonal antibody which is specific for the Syrian hamster form of PrP, and does not cross-react with mouse PrP. Kascsak et al., <i>J. Virol., </i>61:3688-3693 (1987).</p>
    <p>After 24 hrs of incubation both the N2a and ScN2a cells had taken up the exogenously added MHM2 PrP<sup>C</sup>. Via the detergent solubility assay, the majority of the chimeric PrP<sup>C </sup>was present within the detergent insoluble fraction in both cell types. A portion of the chimeric PrP also was found within the detergent soluble fraction, but only in the ScN2a cells (FIG. 10<i>a</i>).</p>
    <p>In the N2a cells the relative amounts of chimeric PrP continued to decline over time and by 66 hrs 3F4 immunoreactive material no longer was observed. In contrast, a significant amount of the chimeric PrP was still present within the ScN2a cells incubated for an additional 42 or 66 hrs after their inoculation with the MHM2 PrP<sup>C </sup>(FIG. 10<i>a</i>). While the majority of the protein still was observed within the detergent insoluble fraction, a portion of the protein also was found within the detergent soluble fraction. We conclude that exogenously added PrP<sup>C </sup>can be taken up by both the N2a and ScN2a cells growing in culture, but that the fate of the internalized protein is somewhat different. Specifically, only in those cells already propagating the conversion of endogenously synthesized PrP<sup>C </sup>to PrP<sup>Sc </sup>(ie., the ScN2a cells) was the exogenously added PrP<sup>C </sup>converted into a form which appeared similar to that of PrP<sup>Sc</sup>.</p>
    <p>The effects of glycerol treatment on the ability of the ScN2a cells to convert the added MHM2 PrP<sup>C </sup>into a detergent insoluble form was examined (FIG. <b>10</b>B). N2a (lanes 1,4), ScN2a (lanes 2,5), and ScN2a cells cultured in the presence of I M glycerol for 24 hrs (gly, lanes 3,6), were incubated with brain extracts prepared from MHM2-expressing transgenic mice. After 40 hrs of incubation the culture medium was removed, the cells extensively washed with PBS, and then harvested. Following fractionation of the cell lysates the distribution of detergent soluble (Sup) and detergent insoluble (Pellet,) MHM2 PrP was determined via Western blotting using the 3F4 antibody. In addition duplicate blots were prepared and probed with the R073 antibody which recognizes both MHM2 PrP as well as endogenous mouse PrP (FIG. <b>10</b>B).</p>
    <p>Similar to our earlier results, the ScN2a cells incubated in the presence of glycerol exhibited significantly less detergent insoluble endogenous PrP relative to that observed for the cells incubated in the absence of glycerol. In the case of the added MHM2 PrP<sup>C</sup>, glycerol treatment also appeared to reduce the levels of the MHM2 protein in both the detergent soluble and insoluble fractions as compared to control, untreated ScN2a cells. Thus, similar to the situation with endogenously synthesized PrP, glycerol treatment appeared to effectively reduce the ability of the ScN2a cells to convert exogenously added PrP<sup>C </sup>into a form characteristic of PrP<sup>Sc</sup>.</p>
    <heading>Example 2D</heading> <p>DMSO Interferes with the Conversion of PrP<sup>C </sup>to PrP<sup>Sc </sup>in ScN2a Cells</p>
    <p>Using the methods discussed in Examples 2A and especially 2B, we demonstrated that DMSO was also very effective in reducing the extent of detergent insoluble PrP. 100 mM DMSO effectively reduced the amount of detergent insoluble PrP by over 50% (FIG. <b>7</b>A). At 200 mM and higher, DMSO appeared to adversely affect the morphology and growth rates of the cells.</p>
    <heading>Example 2E</heading> <p>TMAO Interferes with the Conversion of PrP<sup>C </sup>to PrP<sup>Sc </sup>in ScN2a Cells</p>
    <p>Using the methods discussed in Examples 2A and 2B, we demonstrated that TMAO also was very effective in reducing the extent of detergent insoluble PrP. 100 mM TMAO effectively reduced the amount of detergent insoluble PrP by over 50% (FIG. <b>7</b>A). At 200 mM and higher, TMAO appeared to adversely affect the morphology and growth rates of the cells.</p>
    <heading>Example 3</heading> <p>Correction of p53 Mutants with Chemical Chaperones</p>
    <p>A1-5 cells (a generous gift from A. Levine), a cell line expressing a temperature sensitive (ts) mutant p53 protein produced by transfection of primary rat fibroblasts with a p53 gene containing a missense mutation (ala to val at amino acid 135) (Finlay, C. A., et al., 1988, <i>Mol. Cell. Biol. </i>8:531-539), were used to study the effects of protein stabilizing agents on p53. As was shown previously, this mutation results in a temperature sensitive protein folding defect. Michalovitz, D., et al., 1990, <i>Cell </i>62:671-680; Martinez, J., et al., 1991, <i>Genes Dev. </i>5:151-159; Gannon, J. V. et al., 1991, <i>Nature </i>349:802-806.</p>
    <p>Effects of Chemical Treatments on Cell Growth</p>
    <p>A1-5 cells were plated in 60 mm dishes at an initial concentration of 5×10<sup>4 </sup>cells/plate. After allowing them to attach at 37° C. for 16 h, the cells were shifted to the appropriate temperature in the presence or absence of the various protein stabilizing agents. A few plates of the cells were collected at the time of chemical addition and cell number determined using a hemocytometer. Thereafter, the cells were counted at daily intervals. All time points were examined in duplicate with the values given representing the mean for replicate counts.</p>
    <p>Chemical Treatments</p>
    <p>Various concentrations of protein stabilizing agents were added to DMEM supplemented with 10% fetal bovine serum. For D<sub>2</sub>O experiments, powdered DMEM was reconstituted using 100% D<sub>2</sub>O (Sigma Chemical Co.) and supplemented with 10% fetal bovine serum. For each experiment, the culture medium was removed and replaced with fresh medium containing the chemical of interest. For recovery experiments (e.g., FIG. 14) cells were incubated with the various chemicals for 2 days, the medium then removed, and the cells washed with and further incubated in normal growth medium not containing the particular protein stabilizing agent.</p>
    <p>Western Blotting</p>
    <p>Following the particular experimental treatment, cells were lysed in Laemmli sample buffer and heated at 100° C. for 5 min. Lysates were clarified and the proteins separated by SDS-PAGE. The resolved proteins were transferred to nitrocellulose and subsequently immunoblotted using the mouse monoclonal anti p53 antibody, 421 (Oncogene Science) or the hsp73/hsp 72 specific antibody, N27 (StressGen Biotechnologies).</p>
    <p>A1-5 cells were grown on coverslips and incubated at either 32.5° C. or 39.5° C. for 2 days. The intracellular distribution of p53 was determined by indirect immunofluorescence using a monoclonal antibody which recognizes both the wild type and mutant forms of p53 (PAb 421). Shown in panels A &amp; B are the cells maintained at 39.5° C., and in panels C &amp; D the cells incubated at 32.5° C. Panels A &amp; C are the phase contrast photographs, and panels B and D are the corresponding immunofluorescence photographs.</p>
    <p>At temperatures around 32.5° C. (or less) the p53 protein adopts a wild type conformation and localizes within the cell nucleus (FIG. <b>11</b>D). In contrast, at temperatures of 39.5° C. (or greater) the p53 protein fails to fold properly and does not accumulate within the nucleus. Instead, the vast majority of the mutant protein is found predominantly within the cytoplasm (FIG. <b>11</b>B).</p>
    <p>For indirect immunofluorescence, cells were grown on glass coverslips. Following the particular experimental treatment (described in the figure legend), cells were fixed with 100% methanol and then subsequently rehydrated in PBS. The intracellular localization of p53 was determined by incubation of the fixed cells with the antibody 421 (Oncogene Science) which recognizes both the mutant and wild type forms of p53. Primary antibody was visualized by subsequent staining with a rhodamine conjugated goat anti-mouse antibody. All antibodies were diluted in 5 mg/ml BSA in PBS.</p>
    <p>We examined whether treatment of the A1-5 cells with different protein stabilizing agents would correct the temperature sensitive protein folding defect of mutant p53. Three different reagents were examined: the carbohydrate (or polyol) glycerol; a methylamine, trimethylamine N-oxide; and finally, deuterated water (D<sub>2</sub>O). Cells were plated in normal culture medium and then incubated at 39.5° C. where the p53 protein adopts the mutant conformation. The next day the culture medium was removed, and the cells then incubated in medium supplemented with either 0.6 M glycerol, or 75 mM trimethylamine N-oxide (TMAO). In the case of the deuterated water, powdered DMEM was reconstituted with 100% D<sub>2</sub>O, supplemented with serum, and added to the cells. After 2 days of incubation at 39.5° C., the cells were analyzed for their distribution of p53. The results are illustrated in FIG. 12. A1-5 cells were grown on glass coverslips and incubated for 1D at 39.5° C. Some of the coverslips then were transferred into medium supplemented with either D<sub>2</sub>O (100%), TMAO (75 mM), or glycerol (0.6M) and the cells frther incubated at 39.5° C. After 2D of incubation at 39.5° C., the cells were analyzed for the intracellular distribution of p53 by indirect immunofluorescence as described in FIG. <b>11</b>. Shown only are the immunofluorescent micrographs. Panel A, control cells incubated in normal medium. Panel B, cells incubated in medium prepared with 100% D<sub>2</sub>O. Panel C, cells incubated in the presence of 75 mM TMAO. Panel D, cells treated with 0.6 M glycerol.</p>
    <p>In the control cells maintained at 39.5° C. in normal growth medium, p53 was predominantly cytoplasmic (FIG. <b>12</b>A). In contrast, cells incubated at 39.5° C. in the presence of D<sub>2</sub>O (FIG. <b>12</b>B), TMAO (FIG. <b>12</b>C), or glycerol (FIG. 12D) now exhibited a nuclear locale of p53. Thus, incubation of the cells in the presence of the different protein stabilizing agents appeared to result in the mutant p53 protein adopting a wild type conformation at 39.5° C.</p>
    <p>As an alternative method to screen for the potential beneficial effects of these compounds on p53 function, the proliferative capacity of the A1-5 cells incubated at the nonpermissive temperature, in either the absence or presence of the different chemicals, was determined. As has been shown previously Martinez, J., et al., 1991, <i>Genes Dev. </i>5:151-159, the A1-5 cells become growth arrested when maintained at the permissive temperature (32.5° C.), ostensibly due to the high levels of the wild type p53 tumor suppressor protein (FIG. <b>13</b>A). In contrast, at 39.5° C. where the overexpressed p53 protein folds incorrectly and therefore is biologically inactive, the cells continue to proliferate normally. Hence, if treatment with the various protein stabilizing agents corrects the folding of the mutant p53 protein, the cells now should exhibit cell cycle arrest, even when incubated at the nonpermissive temperature. Indeed, treatment of the A1-5 cells maintained at 39.5° C. with any one of the 3 different compounds now resulted in a growth arrested phenotype, similar to that observed for the cells incubated at the permissive temperature where the p53 protein is functional (FIG. <b>13</b>A).</p>
    <p>In FIG. 13A, equal numbers of A1-5 cells, growing at 37° C., were plated on 60 mm dishes in DMEM containing 10% serum. After plating, the cells were moved to 39.5° C. The next day (day 0) the media was removed, and fresh medium supplemented with serum and containing either: nothing (con); 0.6 M glycerol (Gly); 100% D<sub>2</sub>O (D<sub>2</sub>O); or 75 mM TMAO (TMAO), was added to the cells. The cells were further incubated at 39.5° C. for 1, 2 or 3D. After each day, a plate of the cells was removed and cell number determined as described in the experimental methods. As a control [con (32)], one group of the cells were incubated at 32.5° C., the permissive temperature. In Panel B, equal numbers of cells were plated on 60 mm dishes and treated with the various chemicals as described above. Following 2D of treatment, the culture medium was removed and the cells extensively washed with and further incubated at 39.5° C. in fresh DMEM containing only 10% FBS. Cell numbers then were determined on day 1 and 2 after the changeover back into normal growth medium.</p>
    <p>The effects of the different protein stabilizing agents on the phenotype of the A1-5 cells were reversible (FIG. <b>14</b>). A1-5 cells growing on coverslips at 39.5° C. were incubated in media supplemented with either; D<sub>2</sub>O (panels A-C); TMAO (panels D-F); or glycerol (panels G-I) {concentrations as described in FIG. 3}. After 2D of incubation at 39.5° C. one coverslip from each group of cells was taken and processed for p53 indirect immunofluorescence. To the remaining coverslips, the media was removed, and the cells washed with and further incubated in fresh medium supplemented only with 10% serum. Following a further 1D, or 2D incubation at 39.5° C., the cells were examined for the distribution of p53. Panels A-C; cells after 0, 1 and 2 days respectively following removal of D<sub>2</sub>O: Panels D-F; cells after 0, 1, and 2 days respectively following removal of TMAO containing media; Panels G-I; cells after 0, 1, and 2 days respectively following removal of glycerol.</p>
    <p>As was shown earlier, cells incubated in the presence of the different protein stabilizing agents for 2 D at 39.5° C. exhibited a nuclear locale of p53, indicative of the protein being in its wild type conformation. However, when the culture medium containing the protein stabilizing agents was removed and replaced with fresh culture medium, p53 again began to accumulate within the cytoplasm. For example, after only one day following the removal of glycerol, the cells maintained at 39.5° C. no longer displayed a nuclear p53 distribution. In the case of either D<sub>2</sub>O or TMAO, after 2 days following their removal all of p53 now was found within the cytoplasm. Similar results were obtained when the analysis was performed by cell counts. As was shown in FIG. 13A, cells incubated at 39.5° C. in the present of the 3 different protein stabilizing agents for 2 days exhibited a growth arrested phenotype, consistent with a functional p53 protein. However, upon removal of the compounds and further incubation of the cells at 39.5° C. in normal culture medium, the cells exhibited a slow resumption in their normal growth rates (FIG. <b>13</b>B), indicative that p53 now was in the mutant conformation.</p>
    <heading>Example 4</heading> <p>Correction of pp60<sup>src </sup>Mutants with Chemical Chaperones</p>
    <p>Cells expressing a ts pp60<sup>src </sup>protein (kindly provided by D. T. Aftab) were routinely maintained under the same conditions.</p>
    <p>Rat fibroblasts transfected with a temperature sensitive form of pp60<sup>src</sup>, the transforming protein encoded by Rous sarcoma virus, exhibit a transformed phenotype when incubated at the permissive temperature of 32.5° C. (21). FIG. 15 shows that cells expressing a ts form of pp60<sup>src </sup>were incubated at 32.5° C., or at 39.5° C. in the presence or absence of 1M glycerol. Following three days of treatment, the cells were examined by phase-contrast microscopy. Panel A, cells incubated at 32.5° C. for 3 days. Panel B, cells incubated at 39.5° C. for 3 days. Panel C, cells incubated at 39.5° C. for 3 days in the presence of 1M glycerol.</p>
    <p>As shown in FIG. 15A, when incubated at 32.5° C. where pp60<sup>src </sup>is biologically active, the cells adhere weakly to the substratum. Instead, at this permissive temperature the cells grow on top of one another to form large foci. At 39.5° C. where pp60<sup>src </sup>is biologically inactive, the cells display a well spread morphology and exhibit contact-dependent growth arrest (FIG. <b>15</b>B). Inclusion of glycerol into the culture medium was sufficient to restore the transformed phenotype of the cells when incubated at the non-permissive temperature. For example, cells incubated at 39.5° C. in the presence of 1M glycerol for 3 days (FIG. 15C) now exhibited a morphology very similar to the control cells maintained at 32.5° C. where pp60<sup>src </sup>is active (FIG. <b>15</b>A). Interestingly the 2 other compounds, TMAO and deuterated water, were much less effective in restoring wild type pp60<sup>src </sup>activity at the nonpermissive temperature (data not shown).</p>
    <heading>Example 5</heading> <p>Correction of ts20 Mutants with Chemical Chaperones</p>
    <p>ts20 cells (kindly provided by H. Ozer) were maintained at 32.5° C. (the permissive temperature) in DMEM medium containing 10% fetal bovine serum. They were plated at subconfluency and then incubated at 32.5° C., or 39.5° C. in either the absence or presence of glycerol, TMAO or D<sub>2</sub>O. Two days later the cells were examined by phase-contrast microscopy (FIG. <b>16</b>). In FIG. 16, equal numbers of cells expressing a temperature sensitive mutant of the E1 enzyme were incubated at 32.5° C. (permissive temp.), or at 39.5° C. (nonpermissive temp.) in either the presence or absence of the different protein stabilizing agents. After two days of incubation the cells were examined by phase-contrast microscopy. Panel A, cells incubated at 32.5° C. Panel B, cells incubated at 39.50 C. Panel C, cells incubated at 39.5° C. in medium prepared with 100% D2O. Panel D, cells incubated at 39.5° C. in the presence of 75 mM TMAO. Panel E, cells incubated at 39.5° C. in the presence of 0.75 M glycerol.</p>
    <p>The untreated control cells maintained at 32.5° C. (where the E1 enzyme is active) had grown to near confluence and exhibited a well spread morphology (FIG. <b>16</b>A). In contrast, when incubated at the nonpermissive temperature (i.e. 39.5° C. where E1 is inactive) the cells did not grow at all (FIG. <b>16</b>B). Note the few cells that did survive at 39.5° C. exhibited a spindle-shaped morphology, markedly different than that observed for the cells grown at the permissive temperature. Interestingly, different morphological phenotypes and apparent growth rates were observed when the cells were incubated in the presence of the different protein stabilizing agents. For example, cells incubated at 39.5° C in culture medium prepared with 100% D<sub>2</sub>O exhibited an apparent growth rate only slightly higher than that observed for the untreated cells maintained at the nonpermissive temperature. In addition, the morphology of the cells appeared to reflect an “intermediate phenotype,” with both spindle-shaped and well rounded cells being observed (FIG. <b>16</b>C). Treatment of the ts20 cells with either TMAO (FIG. 16D) or glycerol (FIG. 16E) resulted in an apparent restoration of the wild type phenotype. Like the control cells maintained at 32.5° C., those cells incubated at 39.5° C. in the presence of TMAO or glycerol exhibited normal growth rates and a well-rounded morphology.</p>
    <p>A more quantitative comparison of the growth rates of the cells incubated at 39.5° C. in the presence of the 3 protein stabilizing compounds is shown in FIG. <b>17</b>. In FIG. 17, ts20 cells were plated on 60 mm dishes at low confluence and incubated for 24 hr at 32.5° C. Control cells (no added chemicals), were then placed at either 32.5° C. (permissive temp.) or 39.5° C. (nonpermissive temp.). In parallel, some of the cells were incubated in the presence of the 3 different compounds and then incubated at 39.5° C. After 1, 2, or 3 days of incubation, the cells were collected and cell number determined as described in the methods. Cell number as a function of the days (D) of incubation at either 32.5 or 39.5 are presented.</p>
    <p>Cells incubated at 32.5° C. in regular culture medium displayed an exponential rate of growth, while those maintained at 39.5° C. failed to grow. The actual reduction in cell number found for the cells maintained at 39.5° C. was observed over the course of many experiments, and likely represents cell death (although the mechanism, necrosis versus apoptosis, remains unclear). Similar to our observations using phase-contrast microscopy, cells incubated at 39.5° C. in culture medium prepared with D<sub>2</sub>O failed to exhibit any significant growth. Cells incubated with either glycerol or TMAO at 39.5° C. did exhibit a recovery of cell proliferation, albeit at rates less than that observed for the control cells incubated at 32.5° C.</p>
    <p>Growth inhibition of the ts2O cells at the non-permissive temperature has been suggested to be due, at least in part, to the failure of the cells to ubiquitinate and therefore degrade the normally short-lived p53 tumor suppressor protein. Chowdary, D. R., et al., 1994, <i>Mol. Cell. Biol. </i>14:1997-2003. To test this, ts20 cells were plated on coverslips and incubated at 32.5° C. (the permissive temperature). The next day some of the coverslips were maintained at 32.5° C., while the other coverslips were placed into media either lacking or containing glycerol or TMAO, and then incubated at 39.5° C. One day later the coverslips were fixed and the intracellular locale of p53 determined by indirect immunofluorescence analysis using the monoclonal antibody PAb 421. Shown in panels A, C, E, and G are phase-contrast micrographs, and in panels B, D, F, and H fluorescent micrographs. Panels A&amp;B, cells grown at 32.5° C. Panels C &amp; D, cells incubated at 39.5° C. Panels E&amp;F, cells incubated at 39.5° C. in the presence of 75 mM TMAO. Panels G &amp; H, cells incubated in the presence of 0.75 M glycerol.</p>
    <p>When the cells were maintained at the permissive temperature (at 32.5° C. where E1 is functional), p53 levels were undetectable as determined by indirect immunofluorescence analysis (FIG. <b>18</b>B). In contrast, p53 easily was observed within the nucleus of those cells maintained at the non-permissive temperature of 39.5° C. (FIG. <b>18</b>D). When the cells were incubated at 39.5° C. in the presence of either TMAO (FIG. 18F) or glycerol however (FIG. <b>18</b>H), the levels of p53 nuclear staining was significantly reduced, approaching that observed for the cells maintained at the permissive temperature (FIG. <b>18</b>B).</p>
    <p>Similar results were obtained when the levels of p53 in the ts20 cells were determined by Western blotting (FIG. <b>19</b>). In FIG. 19, ts20 cells were plated on 35 mm dishes and incubated at 32.5° C. for 1 day. One plate of cells was maintained at 32.5° C., while the other plates were incubated at 39.5° C. in the absence or presence of either TMAO (T) or glycerol (G). One day later the cells were harvested and the relative levels of p53 (Panel A) or the cytosolic chaperones hsp73/72 (Panel B) determined by Western blotting.</p>
    <p>Cells incubated at 32.5° C. showed relatively low levels of p53 while those incubated at 39.5° C. had much higher levels of the protein. When the cells were incubated at 39.5° C. in the presence of either TMAO (T) or glycerol (G) (FIG. <b>19</b>A), the levels of p53 were significantly reduced, now appearing similar to that observed for the cells maintained at 32.5° C. Again these observations are consistent with the idea that the different protein stabilizing agents are effective in restoring a functional ubiquitin pathway (thereby resulting in normal p53 degradation) even when the cells are maintained at the nonpermissive temperature. It should be noted that the chemical treatments did not have any obvious effects on general cellular metabolism, with the overall rates of protein synthesis, for example, being similar for each treatment (data not shown). Along these lines, we were also curious to know whether incubation of the ts20 cells with any of the different protein stabilizing agents might have an effect on the overall levels of the cytosolic hsp70 chaperones. As is shown in FIG. 19B the relative levels of hsp73 and hsp72 appeared identical in the cells regardless of the incubation temperature, or the inclusion of either glycerol or TMAO into the culture medium.</p>
    <p>We next examined whether the mutant phenotype was reversible in cells expressing the ts E1 protein. For these experiments, we again used the appearance of p53 staining (or lack thereof) as an assay for E1 enzyme activity. ts20 cells were plated on coverslips and incubated for 1 day at 32.5° C. All of the coverslips then were placed at 39.5° C. for 1 day to induce the mutant phenotype. The next day, fresh media containing either 0.75M glycerol or 75 mM TMAO was added and the cells further incubated at 39.5° C. As a control, one of the coverslips was placed back at 32.5° C. following the 1D incubation at 39.5° C. (no added chemicals). One day later all of the coverslips were collected and the distribution of p53 determined by indirect immunofluorescence as described earlier. Shown are the fluorescent micrographs. Panel A, cells maintained at 39.5° C. for 2D. Panel B, cells incubated at 39.5° C. for one day, and then placed at 32.5° C for one day. Panel C, cells incubated at 39.5° C. for one day, TMAO then added (to a final concentration of 75 mM) and the cells incubated further at 39.5° C. for one day. Panel D, cells incubated at 39.5° C. for one day, glycerol then added (to a final concentration of 0.75 M) and the cells incubated further at 39.5° C. for one day.</p>
    <p>As shown in FIG. 20A, cells maintained at 39.5° C. for 2 days in the absence of protein stabilizers exhibited strong nuclear p53 staining, indicative that the El enzyme (and therefore the ubiquitin pathway) was inactive. Switching the 2-day/39.5° C. cells back to 32.5° C. for 1 day resulted in a disappearance of nuclear p53 staining (FIG. <b>20</b>B), indicative of a restoration of the ubiquitin-dependent protein degradation pathway. In those cells incubated at 39.5° C. for 1 day in normal culture medium, and then transferred into culture medium containing either TMAO or glycerol for an additional day at 39.5° C., mixed results were obtained. Specifically, TMAO treatment resulted in a diminishment, but not complete abolition of nuclear p53 staining (FIG. <b>20</b>C). In the case of glycerol treatment however, p53 staining no longer was observed (FIG. <b>20</b>D).</p>
    <heading>Example 5A</heading> <p>Proper Folding of the Newly Synthesized, but not the Mature, Form of the E1 Protein is Corrected by the Chemical Treatments</p>
    <p>A set of experiments were performed to determine which population of the E1 enzyme was being corrected via treatment of the cells with the different chemicals. Specifically, were the chemical treatments correcting or “rescuing” the misfolding of the mature (already synthesized) E1 protein, or were they only efficacious as it pertained to the folding of the newly synthesized E1 protein? To address this question, the experiments presented in FIG. 20 were repeated. This time however, the protein synthesis inhibitor cycloheximide was added prior to shifting the cells (on day 2) into medium containing the different chemicals. In FIG. 21, ts20 cells were plated on coverslips and incubated at 39.5° C. for one day to induce the mutant phenotype. The cells then were incubated for an additional 24 h at 39.5° C. with fresh media containing either 0.75 M glycerol or 75 mM TMAO and supplemented with the protein synthesis inhibitor cycloheximide. As a positive control, one of the coverslips incubated at 39.5° C. for one day was returned to 32.5° C. and incubated for an additional 24 h in the presence of cycloheximide. Afterwards the cells were fixed and analyzed for the levels of p53 by indirect immunofluorescence. Panel A, cells incubated for 1 D at 39.5° C., and then for an additional day at 39.5° C. in the presence of cycloheximide. Panel B, cells incubated for 1D at 39.5° C., and then for an additional day at 32.5° C. in the presence of cycloheximide. Panel C, cells incubated for 1D at 39.5° C., and then for an additional day at 39.5° C. in the presence of cycloheximide and TMAO. Panel D, cells incubated for 1D at 39.5° C., and then for an additional day at 39.5° C. in the presence of cycloheximide and glycerol.</p>
    <p>Using this approach, the mature E1 enzyme could potentially be corrected in the presence of the protein stabilizing agents, especially considering the relatively long half life (˜20 h) of this protein. Stephen, A. G., et al., 1996, <i>J. Biol. Chem. </i>271:15608-15614. Instead, we observed that the addition of the protein synthesis inhibitor blocked the reversion of the cells back to the wild type phenotype. For example in each case, be it in the control cells returned to 32.5° C. (FIG. <b>21</b>B), or the cells left at 39.5° C. in the presence of either TMAO (FIG. 21C) or glycerol (FIG. 21D) strong nuclear staining of p53 was still observed. This is in marked contrast to the results shown in FIG. 20 where the nuclear staining of p53 no longer was observed upon temperature shift-down, or the addition of the different chemicals to the cells maintained at 39.5° C. Therefore, we conclude that in the case of the E1 enzyme, the different protein stabilizing agents apparently do not correct the already misfolded, mature protein. Instead, we suspect that the chemical chaperones are only effective in influencing the folding pathway of the newly synthesized E1 protein. Presumably, when synthesized at 39.5° C. in the presence of the chemicals, at least a portion of the newly synthesized E1 enzyme now adopts its properly folded and biologically active conformation.</p>
    <p>It should be noted that two types of temperature sensitive proteins folding defects have been described in the literature: Class 1, correctly folded (mature) proteins which are destabilized upon a temperature shift, thereby resulting in their loss of activity (Hawkes, R., et al., 1984, <i>J. Mol. Biol. </i>175:195-212) or Class 2, newly synthesized proteins which are able to fold properly at the permissive temperature, but which fold incorrectly at the nonpermissive temperature (Sturtevant, J. M., et al., 1989, <i>J. Biol. Chem. </i>264:10693-10698). The above results indicate that it is mainly this latter class of temperature sensitive mutants which are affected in the experiments presented here. For example, for each of the ts mutants examined above, the time needed to correct the mutant phenotype via exposure of the cells to the protein stabilizing agents was relatively long. Furthermore, at least for E1 enzyme, in the absence of ongoing protein synthesis, inclusion of the different chemicals into the culture medium did not result in the apparent restoration of mature E1 enzyme function (albeit as determined indirectly by monitoring p53 levels). This observation, along with a number of other unpublished findings, are consistent with the idea that mostly newly synthesized temperature sensitive protein folding mutants which are amenable to correction via the chemical chaperones. However, proteins do exist which manifest destabilizing temperature sensitive mutations (e.g., Class 1 mutants described above) and some of these may be amenable to correction via the strategies used here (currently under study).</p>
    <p>In the case of the Class 2 mutants it seems that the chemical chaperones affect a critical step in the folding pathway of the newly synthesized, temperature sensitive protein folding mutant. For example, at the nonpermissive temperature, the chemical chaperones may help in lowering a critical “energy barrier” which is rate limiting in the folding pathway. Alternatively (or in addition), the chemical chaperones might act to reduce the propensity of the newly synthesized mutant protein to go “off pathway” and end up in a denatured or aggregated state.</p>
    <p>Whatever the mechanism, the data herein indicate that once the particular temperature sensitive protein folding mutant successfully has acquired its properly folded and biologically active state, it remains correctly folded at the nonpermissive temperature, even when the chemical chaperon is subsequently removed.</p>
    <p>The preceding examples are merely illustrative and are not intended to limit the invention in any way. All references cited herein are incorporated by reference in their entirety.</p>
    </div></div></div><div class="patent-section patent-tabular-section"><a id="backward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Cited Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5276059">US5276059</a></td><td class="patent-data-table-td patent-date-value">Jul 10, 1992</td><td class="patent-data-table-td patent-date-value">Jan 4, 1994</td><td class="patent-data-table-td ">The United States Of America As Represented By The Department Of Health And Human Services</td><td class="patent-data-table-td ">Congo red</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5900360">US5900360</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Apr 9, 1997</td><td class="patent-data-table-td patent-date-value">May 4, 1999</td><td class="patent-data-table-td ">Welch; William J.</td><td class="patent-data-table-td ">Screening protein stabilizing agents (not congo red) by incubating cell containing conformationally defective protein, determining effective concentration to improve phenotype; treatment of cystic fibrosis, scrapie</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="npl-citations"></a><div class="patent-section-header"><span class="patent-section-title">Non-Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th colspan="3"class="patent-data-table-th">Reference</th></tr></thead><tr><td class="patent-data-table-td ">1</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Back, et al., Increased Thermal Stability of Proteins in the Presence of Sugars and Polyols, Biochemistry, 18:5191-5196 (1979).</td></tr><tr><td class="patent-data-table-td ">2</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Bilsky, et al., Osmotic Reversal of Temperature Sensitivity in Escherichia coli, Journal of Bacteriology 113:76-81 (1973).</td></tr><tr><td class="patent-data-table-td ">3</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Brown, et al., Chemical chaperones correct the mutant phenotype of the DELTAF508 cystic fibrosis transmembrane conductance regulator protein, Cell Stress &amp; Chaperones, 1(2), 117-125 (1996).</td></tr><tr><td class="patent-data-table-td ">4</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Brown, et al., Chemical chaperones correct the mutant phenotype of the ΔF508 cystic fibrosis transmembrane conductance regulator protein, Cell Stress &amp; Chaperones, 1(2), 117-125 (1996).</td></tr><tr><td class="patent-data-table-td ">5</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Brown, et al., Correcting Temperature-sensitive Protein Folding Defects, J. Clin. Invest., 99:1432-1444 (1997).</td></tr><tr><td class="patent-data-table-td ">6</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Burg, Molecular basis of osmotic regulation, Walter B. Cannon Lecture, American Physiological Society F983-F996, (1995).</td></tr><tr><td class="patent-data-table-td ">7</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Bychkova, et al., Folding intermediates are involved in genetic diseases?, Federation of European Biochemical Societies, 359:6-8 (1995).</td></tr><tr><td class="patent-data-table-td ">8</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Cheng, et al., Functional activation of the cystic fibrosis trafficking mutantDELTAF508-CFTR by overexpression, American Physiological Society, L615-L624 (1995).</td></tr><tr><td class="patent-data-table-td ">9</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Cheng, et al., Functional activation of the cystic fibrosis trafficking mutantΔF508-CFTR by overexpression, American Physiological Society, L615-L624 (1995).</td></tr><tr><td class="patent-data-table-td ">10</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Chowdary, et al., Accumulation of p53 in a Mutant Cell Line Defective in the Ubiquitin Pathway, Molecular and Cellular Biology, 14:1997-2003 (1994).</td></tr><tr><td class="patent-data-table-td ">11</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Denning, et al., Processing of mutant cystic fibrosis transmembrane conductance regulator is temperature-sensitive, Nature, 358:761-764 (1992).</td></tr><tr><td class="patent-data-table-td ">12</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Edington, et al., Inhibition of Heat Shock (Stress) Protein Induction by Deuterium Oxide and Glycerol: Additional Support for the Abnormal Protein Hypothesis of Induction, Journal of Cellular Phsiology, 139:219-228, (1989).</td></tr><tr><td class="patent-data-table-td ">13</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Egan, et al., Differential expression of ORCC and CFTR induced by low temperature in CF airway epithelial cells, American Physiological Society, C243-C251 (1995).</td></tr><tr><td class="patent-data-table-td ">14</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Finley, et al., Thermolability of Ubiquitin-Activating Enzyme from the Mammalian Cell Cycle Mutant ts85, Cell, 37:43-55 (1984).</td></tr><tr><td class="patent-data-table-td ">15</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Gekko, et al., Mechanism of Protein Stabilization by Glycerol: Preferential Hydration in Glycerol-Water Mixtures, Biochemistry, 20:4667-4676 (1981).</td></tr><tr><td class="patent-data-table-td ">16</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Gekko, et al., Thermodynamic and Kinetic Examination of Protein Stabilization by Glycerol, Biochemistry, 20:4677-4686 (1981).</td></tr><tr><td class="patent-data-table-td ">17</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Gerlsma, et al., The Effect of Polyhydric and Monohydric Alcohols on the Heat-Induced Reversible Denaturation of Lysozyme and Ribonuclease, Int. J. Peptide Protein Res., 4:377-383 (1972).</td></tr><tr><td class="patent-data-table-td ">18</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Ginsberg, et al., Induction of Growth Arrest by a Temperature-Sensitive p53 Mutant Is Correlated with Increased Nuclear Localization and Decreased Stability of the Protein, Molecular and Cellular Biology, 582-585 (1991).</td></tr><tr><td class="patent-data-table-td ">19</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Gordon, et al., Temperature-sensitive Mutations in the Phage P22 Coat Protein Which Interfere with Polypeptide Chain folding, The Journal of Biological Chemistry, 268:9358-9368 (1993).</td></tr><tr><td class="patent-data-table-td ">20</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Hawthorne, et al., Osmotic-Remedial Mutants. A New Classification for Nutritional Mutants in Yeast, Genetics, 50:829-839 (1964).</td></tr><tr><td class="patent-data-table-td ">21</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Henle, et al., Protection against Heat-induced Cell Killing by Polyols in Vitro, Cancer Research, 43:1624-1627 (1983).</td></tr><tr><td class="patent-data-table-td ">22</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Lin, et al., Modification of Membrane Function, Protein Synthesis, and Heat Killing Effect in Cultured Chinese Hamster Cells by Glycerol and D20, Cancer Research, 44:5776-5784 (1984).</td></tr><tr><td class="patent-data-table-td ">23</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Lin, et al., Why do Some Organisms Use a Urea-Methylamine Mixture as Osmolyte? Thermodynamic Compensation of Urea and Trimethylamine N -Oxide Interactions with Protein, Biochemistry, 33:12695-12701 (1994).</td></tr><tr><td class="patent-data-table-td ">24</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Maroney, et al., Cloning and characterization of a thermolabile v-src gene for use in reversible transformation of mammalian cells, Onocogene, 7:1207-1214 (1992).</td></tr><tr><td class="patent-data-table-td ">25</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Martinez, et al., Cellular localization and cell cycle regulation by a temperature-sensitive p53 protein, Genes &amp; Development, 5:151-159 (1991).</td></tr><tr><td class="patent-data-table-td ">26</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Mitraki, et al., Global Suppression of Protein Folding Defects and Inclusion Body Formation, Science, 253:54-58 (1991).</td></tr><tr><td class="patent-data-table-td ">27</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Russell, Temperature-Sensitive Osmotic Remedial Mutants of Escherichia col, Journal of Bacteriology, 112:661-665 (1972).</td></tr><tr><td class="patent-data-table-td ">28</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Ryan, et al., Alteration of p53 Conformation and Induction of Apoptosis in a Murine Erythroleukemia Cell Line By Dimethylsulfoxide, Leukemia Research, 18:617-621 (1994).</td></tr><tr><td class="patent-data-table-td ">29</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Santoro, et al., Increased Thermal Stability of Proteins in the Presence of Naturally Occurring Osmolytes, Biochemistry, 31:5278-5283 (1992).</td></tr><tr><td class="patent-data-table-td ">30</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Sato, et al., Glycerol Reverses the Misfolding Phenotype of the Most Common Cystic Fibrosis Mutation, Journal of Biological Chemistry, 271:635-638 (1996).</td></tr><tr><td class="patent-data-table-td ">31</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Schein, Solubility as a Function of Protein Structure and Solvent Components, Bio/Technology, 8:308-317 (1990).</td></tr><tr><td class="patent-data-table-td ">32</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Somero, Protons, osmolytes, and fitness of internal milieu for protein function, American Physiological Society, R197-R213 (1986).</td></tr><tr><td class="patent-data-table-td ">33</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Tatzelt, et al., Chemical chaperones interfere with the formation of scrapie prion protein, The EMBO Journal, 15:6363-6373 (1996).</td></tr><tr><td class="patent-data-table-td ">34</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Th'ng, et al., The FT210 Cell Line Is a Mouse G2 Phase Mutant with a Temperature-Sensitive CDC2 Gene Product, Cell, 63:313-324 (1990).</td></tr><tr><td class="patent-data-table-td ">35</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Thomas, et al., Defective protein folding as a basis of human disease, TIBS 20:456-459 (1995).</td></tr><tr><td class="patent-data-table-td ">36</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Welch, et al., Influence of molecular and chemical chaperones on protein folding, Cell Stress &amp; Chaperones, 1 (2), 109-115 (1996).</td></tr><tr><td class="patent-data-table-td ">37</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Yancey, et al., Living with Water Stress: Evolution of Osmolyte Systems, Science, 217:1214-1222 (1982).</td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="forward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Referenced by</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Citing Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6541195">US6541195</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Mar 30, 2001</td><td class="patent-data-table-td patent-date-value">Apr 1, 2003</td><td class="patent-data-table-td ">The Regents Of The University Of California</td><td class="patent-data-table-td ">Exposing a cell that expresses a conformationally defective target protein with an amount of protein stabilizer selected from amino acids, polyols, dimethylsulfoxide and deuterated water that is effective to improve phenotypic defects</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7238474">US7238474</a></td><td class="patent-data-table-td patent-date-value">Oct 6, 2000</td><td class="patent-data-table-td patent-date-value">Jul 3, 2007</td><td class="patent-data-table-td ">Florida State University Research Foundation</td><td class="patent-data-table-td ">Materials and methods for detecting interaction of CFTR polypeptides</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7695917">US7695917</a></td><td class="patent-data-table-td patent-date-value">Oct 8, 2002</td><td class="patent-data-table-td patent-date-value">Apr 13, 2010</td><td class="patent-data-table-td ">Oregon Health &amp; Science University</td><td class="patent-data-table-td ">Rescue of gonadotropin releasing hormone receptor mutants</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7794692">US7794692</a></td><td class="patent-data-table-td patent-date-value">Dec 1, 2006</td><td class="patent-data-table-td patent-date-value">Sep 14, 2010</td><td class="patent-data-table-td ">Amorfix Life Sciences Ltd.</td><td class="patent-data-table-td ">Methods and compositions for detecting amyotrophic lateral sclerosis</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7842470">US7842470</a></td><td class="patent-data-table-td patent-date-value">Feb 2, 2005</td><td class="patent-data-table-td patent-date-value">Nov 30, 2010</td><td class="patent-data-table-td ">Oregon Health &amp; Science University</td><td class="patent-data-table-td ">Method for pharmacoperones correction of GnRHR mutant protein misfolding</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7851143">US7851143</a></td><td class="patent-data-table-td patent-date-value">May 16, 2007</td><td class="patent-data-table-td patent-date-value">Dec 14, 2010</td><td class="patent-data-table-td ">Amicus Therapeutics, Inc.</td><td class="patent-data-table-td ">Lysosomal storage disorder; specific pharmacological chaperone is 1-deoxygalactonojirimycin; T cell stimulatory agent; broadly applicable, fast and efficient method for screening patients for responsiveness to SPC therapy</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7887803">US7887803</a></td><td class="patent-data-table-td patent-date-value">Sep 5, 2007</td><td class="patent-data-table-td patent-date-value">Feb 15, 2011</td><td class="patent-data-table-td ">Amorfix Life Sciences</td><td class="patent-data-table-td ">antibody, or binding fragment thereof, that binds selectively to an epitope presented by misfolded form of superoxide dismutase 1; use in diagnosis, prognosis and therapeutic intervention in neurodegenerative Alzheimer&#39;s disease, Parkinson&#39;s disease, Huntington&#39;s disease</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7977314">US7977314</a></td><td class="patent-data-table-td patent-date-value">Mar 5, 2007</td><td class="patent-data-table-td patent-date-value">Jul 12, 2011</td><td class="patent-data-table-td ">Amorfix Life Sciences Limited</td><td class="patent-data-table-td ">Using mutant superoxide dismutase 1 (SOD1) as tool in diagnosis, prevention and treatment of nervous system disorders</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8288107">US8288107</a></td><td class="patent-data-table-td patent-date-value">Jun 25, 2007</td><td class="patent-data-table-td patent-date-value">Oct 16, 2012</td><td class="patent-data-table-td ">Florida State University Research Foundation</td><td class="patent-data-table-td ">Incorporating polynucleotides encoding fusion proteins comprising CFTR peptide and DNA binding domain of transcriptional activator that binds to site on detectable gene into host cell ; gene expressing; drug screening, high throughput assay; enhances dimerization of CFTR protein; cystic fibrosis</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8513387">US8513387</a></td><td class="patent-data-table-td patent-date-value">Jul 21, 2010</td><td class="patent-data-table-td patent-date-value">Aug 20, 2013</td><td class="patent-data-table-td ">Amorfix Life Sciences Ltd.</td><td class="patent-data-table-td ">Methods and compositions for detecting amyotrophic lateral sclerosis</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8709422">US8709422</a></td><td class="patent-data-table-td patent-date-value">Dec 24, 2010</td><td class="patent-data-table-td patent-date-value">Apr 29, 2014</td><td class="patent-data-table-td ">Amorfix Life Sciences Ltd.</td><td class="patent-data-table-td ">Methods and compositions to treat and detect misfolded-SOD1 mediated diseases</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8778885">US8778885</a></td><td class="patent-data-table-td patent-date-value">Jun 8, 2011</td><td class="patent-data-table-td patent-date-value">Jul 15, 2014</td><td class="patent-data-table-td ">Amorfix Life Sciences Ltd.</td><td class="patent-data-table-td ">Methods and compositions to treat and detect misfolded-SOD1 mediated diseases</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2533050A1?cl=en">EP2533050A1</a></td><td class="patent-data-table-td patent-date-value">May 16, 2007</td><td class="patent-data-table-td patent-date-value">Dec 12, 2012</td><td class="patent-data-table-td ">Amicus Therapeutics, Inc.</td><td class="patent-data-table-td ">Treatment options for Fabry disease</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2006133098A2?cl=en">WO2006133098A2</a></td><td class="patent-data-table-td patent-date-value">Jun 5, 2006</td><td class="patent-data-table-td patent-date-value">Dec 14, 2006</td><td class="patent-data-table-td ">Amicus Therapeutics Inc</td><td class="patent-data-table-td ">Pharmacological chaperones for treating obesity</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="classifications"></a><div class="patent-section-header"><span class="patent-section-title">Classifications</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th"> </th><th class="patent-data-table-th"> </th></tr></thead><tr><td class="patent-data-table-td ">U.S. Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=nwxWBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S004000">435/4</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=nwxWBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S026000">435/26</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=nwxWBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S963000">435/963</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=nwxWBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S023000">435/23</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=nwxWBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S024000">435/24</a></span></td></tr><tr><td class="patent-data-table-td ">International Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=nwxWBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=A61K0031770000">A61K31/77</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=nwxWBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=A61K0031197000">A61K31/197</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=nwxWBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=A61K0031047000">A61K31/047</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=nwxWBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=A61K0031205000">A61K31/205</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=nwxWBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=G01N0033500000">G01N33/50</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=nwxWBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=A61K0031401000">A61K31/401</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=nwxWBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=A61K0031131000">A61K31/131</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=nwxWBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=A61K0031198000">A61K31/198</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=nwxWBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=A61K0031100000">A61K31/10</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=nwxWBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=A61K0031185000">A61K31/185</a></span></td></tr><tr><td class="patent-data-table-td ">Cooperative Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=nwxWBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=Y10S435/963">Y10S435/963</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=nwxWBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=G01N2800/382">G01N2800/382</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=nwxWBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=A61K31/185">A61K31/185</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=nwxWBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=A61K31/198">A61K31/198</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=nwxWBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=G01N33/5008">G01N33/5008</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=nwxWBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=A61K31/047">A61K31/047</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=nwxWBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=G01N33/502">G01N33/502</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=nwxWBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=A61K31/205">A61K31/205</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=nwxWBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=A61K31/10">A61K31/10</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=nwxWBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=A61K31/77">A61K31/77</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=nwxWBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=A61K31/197">A61K31/197</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=nwxWBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=A61K31/131">A61K31/131</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=nwxWBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=G01N33/5091">G01N33/5091</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=nwxWBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=A61K31/401">A61K31/401</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=nwxWBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=G01N33/5044">G01N33/5044</a></span></td></tr><tr><td class="patent-data-table-td ">European Classification</td><td class="patent-data-table-td "><span class="nested-value">A61K31/401</span>, <span class="nested-value">A61K31/205</span>, <span class="nested-value">A61K31/131</span>, <span class="nested-value">G01N33/50D4</span>, <span class="nested-value">A61K31/77</span>, <span class="nested-value">A61K31/185</span>, <span class="nested-value">A61K31/047</span>, <span class="nested-value">A61K31/197</span>, <span class="nested-value">A61K31/10</span>, <span class="nested-value">G01N33/50D2</span>, <span class="nested-value">A61K31/198</span>, <span class="nested-value">G01N33/50D2E</span>, <span class="nested-value">G01N33/50D2F</span></td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="legal-events"></a><div class="patent-section-header"><span class="patent-section-title">Legal Events</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Date</th><th class="patent-data-table-th">Code</th><th class="patent-data-table-th">Event</th><th class="patent-data-table-th">Description</th></tr></thead><tr><td class="patent-data-table-td patent-date-value">Sep 29, 2009</td><td class="patent-data-table-td ">FP</td><td class="patent-data-table-td ">Expired due to failure to pay maintenance fee</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20090807</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Aug 7, 2009</td><td class="patent-data-table-td ">LAPS</td><td class="patent-data-table-td ">Lapse for failure to pay maintenance fees</td><td class="patent-data-table-td "></td></tr><tr><td class="patent-data-table-td patent-date-value">Feb 16, 2009</td><td class="patent-data-table-td ">REMI</td><td class="patent-data-table-td ">Maintenance fee reminder mailed</td><td class="patent-data-table-td "></td></tr><tr><td class="patent-data-table-td patent-date-value">Aug 5, 2008</td><td class="patent-data-table-td ">B1</td><td class="patent-data-table-td ">Reexamination certificate first reexamination</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">CLAIMS 1-10 ARE CANCELLED. CLAIMS 11-22 WERE NOT REEXAMINED.</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Sep 25, 2007</td><td class="patent-data-table-td ">RR</td><td class="patent-data-table-td ">Request for reexamination filed</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20070703</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">May 9, 2005</td><td class="patent-data-table-td ">SULP</td><td class="patent-data-table-td ">Surcharge for late payment</td><td class="patent-data-table-td "></td></tr><tr><td class="patent-data-table-td patent-date-value">May 9, 2005</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">4</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Feb 23, 2005</td><td class="patent-data-table-td ">REMI</td><td class="patent-data-table-td ">Maintenance fee reminder mailed</td><td class="patent-data-table-td "></td></tr><tr><td class="patent-data-table-td patent-date-value">Mar 14, 2000</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">REGENTS OF THE UNIVERSITY OF CALIFORNIA, THE, CALI</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNORS:WELCH, WILLIAM J.;BROWN, C. RANDELL;TATZELT, JORG;REEL/FRAME:010695/0926;SIGNING DATES FROM 19991024 TO 19991122</span></div><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">REGENTS OF THE UNIVERSITY OF CALIFORNIA, THE 12TH</span></div></td></tr></table><div class="patent-section-footer"></div></div><div class="modal-dialog" id="patent-images-lightbox"><div class="patent-lightbox-controls"><div class="patent-lightbox-rotate-controls"><div class="patent-lightbox-rotation-text">Rotate</div><div class="rotate-icon rotate-ccw-icon"></div><div class="rotate-icon rotate-cw-icon"></div></div><div class="patent-lightbox-index-counter"></div><a class="patent-lightbox-fullsize-link" target="_blank">Original Image</a><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_right.png" alt="Next page"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_left.png" alt="Previous page"width="21" height="21" /></div></div><div class="modal-dialog-content"><div class="patent-lightbox-image-holder"><div class="patent-lightbox-placeholder"></div></div></div></div><script>_OC_initPatentsAtb({image_not_available_html: " Image not available"});</script></div></div></div></td></tr></table><script>(function() {var href = window.location.href;if (href.indexOf('?') !== -1) {var parameters = href.split('?')[1].split('&');for (var i = 0; i < parameters.length; i++) {var param = parameters[i].split('=');if (param[0] == 'focus') {var elem = document.getElementById(param[1]);if (elem) {elem.focus();}}}}})();</script><script>_OC_addFlags({LockSrc:"/books/javascript/lock_8a2b04e7bf975d5171d8e4c0b6365c7a.js", Host:"http://www.google.com/", IsBooksRentalEnabled:1, IsWebstoreDisplayCaseEnabled:1, IsObfuscationEnabled:1, IsBrowsingHistoryEnabled:1, IsWebReaderSvgEnabled:0, IsGeoLayerEnabled:1, IsImageModeNotesEnabled:1, IsCopyMenuItemEnabled:1, IsGiftingEnabled:0, IsWebReaderUniversalPaginatorEnabled:0, IsOfflineBubbleEnabled:1, IsReaderEnabledForPlayRequests:1, IsFutureOnSaleVolumesEnabled:1, IsOfflineRestrictedCopyEnabled:1, IsBooksUnifiedLeftNavEnabled:1, IsRestrictedCopyEnabled:1, IsZipitFolderCollectionEnabled:1, IsEndOfSampleRecommendationsEnabled:1, IsRatingsOnBookcardsEnabled:1, IsAdsDisabled:0, IsIframePageDisplayEnabled:0, IsEmbeddedMediaEnabled:1, IsImageModeAnnotationsEnabled:1, IsMyLibraryGooglePlusEnabled:1, IsImagePageProviderEnabled:0, IsBookcardListPriceSmall:0, IsInternalUser:0, IsBooksShareButtonEnabled:0, IsPreOrdersEnabled:0, IsDisabledRandomBookshelves:0, WebstoreDisplayCasePosition:3});_OC_Run({"enable_p13n":false,"add_vol_to_collection_base_url":"http://www.google.com/patents?op=add\u0026sig=ACfU3U1DLYH4PIxUIoVwwR_AO_PqFuSDyg\u0026id=nwxWBAABERAJ","remove_vol_from_collection_base_url":"http://www.google.com/patents?op=remove\u0026sig=ACfU3U1DPZ-1QbANuz57dhRoQ2XXRXpUMg\u0026id=nwxWBAABERAJ","logged_in":false,"p13n_save_user_settings_url":"http://www.google.com/patents?op=edit_user_settings\u0026sig=ACfU3U1-if-_u2OxkifV5n6XdjUOzhAziA","is_cobrand":false,"sign_in_url":"https://www.google.com/accounts/Login?service=\u0026continue=http://www.google.com/patents%3Fhl%3Den\u0026hl=en","is_play_enabled":true}, {"volume_id":"","is_ebook":true,"volumeresult":{"has_flowing_text":false,"has_scanned_text":true,"can_download_pdf":false,"can_download_epub":false,"is_pdf_drm_enabled":false,"is_epub_drm_enabled":false,"download_pdf_url":"http://www.google.com/patents/download/Contacting_first_cell_that_expresses_con.pdf?id=nwxWBAABERAJ\u0026output=pdf\u0026sig=ACfU3U0si522FruB9r43m-hjHZa67pZ56w"},"sample_url":"http://www.google.com/patents/reader?id=nwxWBAABERAJ\u0026printsec=frontcover\u0026output=reader\u0026source=gbs_atb_hover","is_browsable":true,"is_public_domain":true}, {});</script><div id="footer_table" style="font-size:83%;text-align:center;position:relative;top:20px;height:4.5em;margin-top:2em"><div style="margin-bottom:8px"><a href=http://www.google.com/><nobr>Google&nbsp;Home</nobr></a> - <a href=//www.google.com/patents/sitemap/><nobr>Sitemap</nobr></a> - <a href=http://www.google.com/googlebooks/uspto.html><nobr>USPTO Bulk Downloads</nobr></a> - <a href=/intl/en/privacy/><nobr>Privacy Policy</nobr></a> - <a href=/intl/en/policies/terms/><nobr>Terms of Service</nobr></a> - <a href=https://support.google.com/faqs/answer/2539193?hl=en><nobr>About Google Patents</nobr></a> - <a href="http://www.google.com/tools/feedback/intl/en/error.html" onclick="try{_OC_startFeedback({productId: '72792',locale: 'en'});return false;}catch(e){}"><nobr>Send Feedback</nobr></a></div><span>Data provided by IFI CLAIMS Patent Services</span><br><span >&copy;2012 Google</span></div> <script type="text/javascript">var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));</script><script type="text/javascript">var pageTracker = _gat._getTracker("UA-27188110-1");pageTracker._setCookiePath("/patents/");pageTracker._trackPageview();</script> </body></html>