<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN""http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html><head><title>Patent US6376217 - Fusion protein comprising gelonin amino acid sequence having enzymatic ... - Google Patents</title><script>(function(){(function(){function e(a){this.t={};this.tick=function(a,c,b){var d=void 0!=b?b:(new Date).getTime();this.t[a]=[d,c];if(void 0==b)try{window.console.timeStamp("CSI/"+a)}catch(e){}};this.tick("start",null,a)}var a;window.performance&&(a=window.performance.timing);var f=a?new e(a.responseStart):new e;window.jstiming={Timer:e,load:f};if(a){var c=a.navigationStart,d=a.responseStart;0<c&&d>=c&&(window.jstiming.srt=d-c)}if(a){var b=window.jstiming.load;0<c&&d>=c&&(b.tick("_wtsrt",void 0,c),b.tick("wtsrt_",
"_wtsrt",d),b.tick("tbsd_","wtsrt_"))}try{a=null,window.chrome&&window.chrome.csi&&(a=Math.floor(window.chrome.csi().pageT),b&&0<c&&(b.tick("_tbnd",void 0,window.chrome.csi().startE),b.tick("tbnd_","_tbnd",c))),null==a&&window.gtbExternal&&(a=window.gtbExternal.pageT()),null==a&&window.external&&(a=window.external.pageT,b&&0<c&&(b.tick("_tbnd",void 0,window.external.startE),b.tick("tbnd_","_tbnd",c))),a&&(window.jstiming.pt=a)}catch(g){}})();})();
</script><link rel="stylesheet" href="/patents/css/_8a2b04e7bf975d5171d8e4c0b6365c7a/kl_intl_patents_bundle.css" type="text/css" /><script src="/books/javascript/atb_8a2b04e7bf975d5171d8e4c0b6365c7a__en.js"></script><script>function googleTranslateElementInit() {new google.translate.TranslateElement({pageLanguage: "en",gaTrack: true,gaId: "UA-27188110-1",multilanguagePage: true});}</script><script src="//translate.google.com/translate_a/element.js?cb=googleTranslateElementInit"></script><meta name="DC.type" content="Patent"><meta name="DC.title" content="Fusion protein comprising gelonin amino acid sequence having enzymatic activity and antibody, antigen-binding fragment, hormone, lymphokine or growth factor targeting sequence allowing internalization of fusion protein"><meta name="DC.contributor" content="Marc D. Better" scheme="inventor"><meta name="DC.contributor" content="Stephen F. Carroll" scheme="inventor"><meta name="DC.contributor" content="Xoma Technology Ltd." scheme="assignee"><meta name="DC.date" content="2000-7-6" scheme="dateSubmitted"><meta name="DC.description" content="The present invention provides purified and isolated polynucleotides encoding Type I ribosome-inactivating proteins (RIPS) and analogs of the RIPs having a cysteine available for disulfide bonding to targeting molecules. Vectors comprising the polynucleotides and host cells transformed with the vectors are also provided. The RIPs and RIP analogs are particularly suited for use as components of cytotoxic therapeutic agents of the invention which include gene fusion products and immunoconjugates. Cytotoxic therapeutic agents or immunotoxins according to the present invention may be used to selectively eliminate any cell type to which the RIP component is targeted by the specific binding capacity of the second component of the agent, and are suited for treatment of diseases where the elimination of a particular cell type is a goal, such as autoimmune disease, cancer and graft-versus-host disease."><meta name="DC.date" content="2002-4-23" scheme="issued"><meta name="DC.relation" content="AU:2761788" scheme="references"><meta name="DC.relation" content="EP:0170697:A1" scheme="references"><meta name="DC.relation" content="EP:0192002:A1" scheme="references"><meta name="DC.relation" content="EP:0239400:A2" scheme="references"><meta name="DC.relation" content="EP:0438310:A1" scheme="references"><meta name="DC.relation" content="EP:0506124:A1" scheme="references"><meta name="DC.relation" content="EP:0519596:A1" scheme="references"><meta name="DC.relation" content="GB:2216891" scheme="references"><meta name="DC.relation" content="US:4675382" scheme="references"><meta name="DC.relation" content="US:4769326" scheme="references"><meta name="DC.relation" content="US:4853871" scheme="references"><meta name="DC.relation" content="US:4888415" scheme="references"><meta name="DC.relation" content="US:4894443" scheme="references"><meta name="DC.relation" content="US:4925673" scheme="references"><meta name="DC.relation" content="US:4946778" scheme="references"><meta name="DC.relation" content="US:5013653" scheme="references"><meta name="DC.relation" content="US:5028530" scheme="references"><meta name="DC.relation" content="US:5091513" scheme="references"><meta name="DC.relation" content="US:5093475" scheme="references"><meta name="DC.relation" content="US:5101025" scheme="references"><meta name="DC.relation" content="US:5169939" scheme="references"><meta name="DC.relation" content="US:5376546" scheme="references"><meta name="DC.relation" content="US:5416202" scheme="references"><meta name="DC.relation" content="US:5621083" scheme="references"><meta name="DC.relation" content="US:5624827" scheme="references"><meta name="DC.relation" content="US:5631348" scheme="references"><meta name="DC.relation" content="US:5744580" scheme="references"><meta name="DC.relation" content="US:5756699" scheme="references"><meta name="DC.relation" content="US:5837491" scheme="references"><meta name="DC.relation" content="US:6146631" scheme="references"><meta name="DC.relation" content="WO:1983003971:A1" scheme="references"><meta name="DC.relation" content="WO:1985003508:A1" scheme="references"><meta name="DC.relation" content="WO:1986000090:A1" scheme="references"><meta name="DC.relation" content="WO:1986005098:A1" scheme="references"><meta name="DC.relation" content="WO:1987002987:A1" scheme="references"><meta name="DC.relation" content="WO:1988001649:A1" scheme="references"><meta name="DC.relation" content="WO:1988007085:A1" scheme="references"><meta name="DC.relation" content="WO:1988009344:A1" scheme="references"><meta name="DC.relation" content="WO:1989000999:A1" scheme="references"><meta name="DC.relation" content="WO:1989006967:A1" scheme="references"><meta name="DC.relation" content="WO:1989006968:A1" scheme="references"><meta name="DC.relation" content="WO:1989009622:A1" scheme="references"><meta name="DC.relation" content="WO:1990002569:A1" scheme="references"><meta name="DC.relation" content="WO:1990010015:A1" scheme="references"><meta name="DC.relation" content="WO:1990012592:A1" scheme="references"><meta name="DC.relation" content="WO:1991002000:A1" scheme="references"><meta name="DC.relation" content="WO:1991019745:A1" scheme="references"><meta name="DC.relation" content="WO:1992003144:A1" scheme="references"><meta name="DC.relation" content="WO:1992008495:A1" scheme="references"><meta name="DC.relation" content="WO:1992009613:A1" scheme="references"><meta name="DC.relation" content="WO:1992011018:A1" scheme="references"><meta name="DC.relation" content="WO:1992014491:A1" scheme="references"><meta name="DC.relation" content="WO:1992015327:A1" scheme="references"><meta name="DC.relation" content="WO:1992022324:A1" scheme="references"><meta name="DC.relation" content="WO:1993005168:A1" scheme="references"><meta name="DC.relation" content="WO:1993009130:A1" scheme="references"><meta name="DC.relation" content="WO:2002006117:A1" scheme="references"><meta name="citation_reference" content="Antin, J.H., et al., &quot;Selective Depletion of Bone Marrow T Lymphoytes With Anti-CD5 Monoclonal Antibodies: Effective Prophylaxis for Graft-Versus-Host Disease in Patients With Hematologic Malignancies,&quot; Blood 78:2139-2149, W. B. Saunders (Oct. 1991)."><meta name="citation_reference" content="Asano, K., et al., &quot;Isolation and Characterization of Inhibitors of Animal Cell-Free Protein Synthesis from Barley Seeds,&quot; Carlsberg Res. Comm. 49:619-626, Springer-Verlag (1984)."><meta name="citation_reference" content="Bacha, P., et al., &quot;Interleukin 2 Receptor-Targeted Cytotoxicity,&quot; J. Exp. Med. 167:612-622, The Rockfeller University Press (Feb. 1988)."><meta name="citation_reference" content="Barry, B.W., ed., in Dermatological Formulations-Percutaneous Absorption, Marcel Decker, Inc., New York, N.Y., pp. 180-181 (1983)."><meta name="citation_reference" content="Barry, B.W., ed., in Dermatological Formulations—Percutaneous Absorption, Marcel Decker, Inc., New York, N.Y., pp. 180-181 (1983)."><meta name="citation_reference" content="Batra, J.K., et al., &quot;Anti-Tac(Fv)-PE40, a Single Chain Antibody Pseudomonas Fusion Protein Directed at Interleukin 2 Receptor Bearing Cells,&quot; J. Biol. Chem. 265:15198-15202, American Society for Biochemistry and Molecular Biology, Inc. (Sep. 1990)."><meta name="citation_reference" content="Batra, J.K., et al., &quot;Single-Chain Immunotoxins Directed at the Human Transferrin Receptor Containing Pseudomonas Exotoxin A or Diphtheria Toxin: Anti-TFR (Fv)-PE40 and DT388-Anti-TFR(Fv),&quot; Mol. Cell. Biol. 11:2200-2205, American Society for Microbiology (Apr. 1991)."><meta name="citation_reference" content="Benatti, L., et al., &quot;Nucleotide sequence of cDNA coding for saporin-6, a type-1, ribosome-inactivating protein from Saponaria officinalis,&quot; Eur. J. Biochem. 183:465-470, The Federation of American Societies for Experimental Biology (1989)."><meta name="citation_reference" content="Bernhard, S.L., et al., &quot;Cysteine Analogs of Recombinant Barley Ribosome Inactivating Protein Form Antibody Conjugates with Enhanced Stability and Potency in Vitro,&quot; Bioconjugate Chem. 5:126-132, American Chemical Society (Mar. 1994)."><meta name="citation_reference" content="Better, M., et al., &quot;Acitvity of Recombinant Mitogillin and Mitogillin Immunoconjugates,&quot; J. Biol. Chem. 267:16712-16718, American Society for Biochemistry and Molecular Biology, Inc. (Aug. 1992)."><meta name="citation_reference" content="Better, M., et al., &quot;Escherichia coli Secretion of an Active Chimeric Antibody Fragment,&quot; Science 240:1041-1043, Association for the Advancement of Science (May 1988)."><meta name="citation_reference" content="Better, M., et al., &quot;Gelonin Analogs with Engineered Cysteine Residues Form Antibody Immunoconjugates with Unique Properties,&quot; J. Biol. Chem. 269:9644-9650, American Society for Biochemistry and Molecular Biology, Inc. (Apr. 1994)."><meta name="citation_reference" content="Better, M., et al., &quot;Generation of Potent Immunoconjugates from Microbially Produced Fab&#39;, F(ab&#39;)2, and Recombinant Ribosome Inactivating Proteins,&quot; (Poster), Third Intl. Symposium on Immunotoxins, Orlando, FL (Jun. 1992)."><meta name="citation_reference" content="Better, M., et al., &quot;Potent anti-CD5 ricin A chain immunoconjugates from bacterially produced Fab&#39; and F(ab&#39;)2,&quot; Proc. Natl. Acad. Sci. USA 90:457-461, National Academy of Sciences of the USA (Jan. 1993)."><meta name="citation_reference" content="Better, M., et al., &quot;T Cell-targeted, Immunofusion Proteins from Escherichia coli,&quot; J. Biol. Chem. 270:14951-14957, American Society for Biochemistry and Molecular Biology, Inc. (Jun. 1995)."><meta name="citation_reference" content="Blakey, D.C., et al., &quot;Antibody Toxin Conjugates: A Perspective,&quot; Monoclonal Antibody Therapy Prog. Allergy 45:50-90, Karger (1988)."><meta name="citation_reference" content="Bolognesi, A., et al., &quot;A comparison of anti-lymphoctye immunotoxins containing different ribosome-inactivating proteins and antibodies,&quot; Clin. Exp. Immunol. 89:341-346, Blackwell Scientific Publications (Sep. 1992)."><meta name="citation_reference" content="Brown, D., et al., &quot;Customizing Lab Mice,&quot; The Washington Post:A1 and A4, The Washington Post (Aug. 21, 1992)."><meta name="citation_reference" content="Byers, V.S., et al., &quot;Use of an Anti-Pan T-Lymphocyte Ricin A Chain Immunotoxin in Steroid-Resistant Acute Graft-Versus-Host Disease,&quot; Blood 75:1426-1432, W. B. Saunder (Apr. 1990)."><meta name="citation_reference" content="Caron, P.C., et al., &quot;Engineered Humanized Dimeric Forms of IgG Are More Effective Antibodies,&quot; J. Exp. Med. 176:1191-1195, The Rockefeller University Press (Oct. 1992)."><meta name="citation_reference" content="Casellas, P., et al., &quot;Optimal Elimination of Leukemic T Cells from Human Bone Marrow With T101-Ricin A-Chain Immunotoxin,&quot; Blood 65:289-297, W. B. Saunders (Feb. 1985)."><meta name="citation_reference" content="Chaudhary, V.K., et al., &quot;A rapid method of cloning functional variable-region antibody genes in Escherichia coli as single-chain immunotoxins,&quot; Proc. Natl. Acad. Sci. USA 87:1066-1070, National Academy of Sciences of the USA (Feb. 1990)."><meta name="citation_reference" content="Chaudhary, V.K., et al., &quot;A recombinant immunotoxin consisting of two antibody variable domains fused to Pseudomonas exotoxin,&quot; Nature 339:394-397, Macmillan Publishers Ltd. (Jun. 1989)."><meta name="citation_reference" content="Cheadle, C., et al., &quot;Cloning and Expression of the Variable Regions of Mouse Myeloma Protein MOPC315 in E. Coli: Recovery of Active Fv Fragments,&quot; Mol. Immunol. 29:21-30, Pergammon Press (Jan. 1992)."><meta name="citation_reference" content="Chow, T., et al., &quot;Isolation and DNA Sequence of a Gene Encoding alpha-Trichosanthin, a Type 1 Ribosome-inactivating Protein,&quot; J. Biol. Chem. 265:8670-8674, American Society for Biochemistry and Molecular Biology, Inc. (May 1990)."><meta name="citation_reference" content="Chow, T., et al., “Isolation and DNA Sequence of a Gene Encoding α-Trichosanthin, a Type 1 Ribosome-inactivating Protein,” J. Biol. Chem. 265:8670-8674, American Society for Biochemistry and Molecular Biology, Inc. (May 1990)."><meta name="citation_reference" content="Co, M.S., et al., &quot;Humanized antibodies for antiviral therapy,&quot; Proc. Nat. Acad. Sci. USA 88:2869-2873, National Academy of Sciences of the USA (Apr. 1991)."><meta name="citation_reference" content="Colombatti, M., et al., &quot;Selective Killing of Target Cells By Antibody-Ricin A Chain or Antibody-Gelonin Hybrid Molecules: Comparison of Cytotoxic Potency and Use in Immunoselection Procedures,&quot; J. Immunol. 131:3091-3095, American Association of Immunologists (Dec. 1983)."><meta name="citation_reference" content="Delprino, L., et al., &quot;Toxin-Targeted Design for Anticancer Therapy. II: Preparation and Biological Comparison of Different Chemically Linked Gelonin-Antibody Conjugates,&quot; Journal of Pharm. Sci. 82:699-704, American Pharmaceutical Association (Jul. 1993)."><meta name="citation_reference" content="Descotes, G., et al., &quot;The Immunological Activity of Plant Toxins Used in the Preparation of Immunotoxins-II. The Immunodepressive Activity of Gelonin,&quot; J. Immunopharm. 7:455-463, International Society for Immunopharmacology (1985)."><meta name="citation_reference" content="Descotes, G., et al., “The Immunological Activity of Plant Toxins Used in the Preparation of Immunotoxins—II. The Immunodepressive Activity of Gelonin,” J. Immunopharm. 7:455-463, International Society for Immunopharmacology (1985)."><meta name="citation_reference" content="Dialog File 351, Derwent World Patents Index, English Lanugage abstract for EP 192002 A1, WPI Accession No. 86-226683."><meta name="citation_reference" content="Dower, W.J., et al., &quot;High efficiency transformation of E. coli by high voltage electroporation,&quot; Nucleic Acids Res. 16:6127-6145, IRL Press Limited (1988)."><meta name="citation_reference" content="Ebert, R.F., and Spryn, L.A., &quot;Immunotoxin Construction with a Ribosome-Inactivating Protein from Barley,&quot; Bioconjugate Chem. 1:331-336, American Chemical Society (Feb. 1990)."><meta name="citation_reference" content="Ellman, G.L., &quot;Tissue Sulfhydryl Groups,&quot; Arch. Biochem. Biophys. 82:70-77, Academic Press (1959)."><meta name="citation_reference" content="Fishwild, D.M., et al., &quot;Characterization of the increased cytotoxicity of gelonin anti-T cell immunoconjugates compared with ricin A chain immunoconjugates,&quot; Clin. Exp. Immunol. 97:10-18, Blackwell Scientific Publications (Jul. 1994)."><meta name="citation_reference" content="Fishwild, D.M., et al., &quot;Cytotoxicity against human peripheral blood mononuclear cells and T cell lines mediated by anti-t cell immunotoxins in the absence of added potentiator,&quot; Clin. Exp. Immunol. 86:506-513, Blackwell Scientific Publisher (Dec. 1991)."><meta name="citation_reference" content="Glockshuber, R., et al., &quot;A Comparison of Strategies To Stabliize Immunoglobulin Fv-Fragments,&quot; Biochem. 29:1362-1367, American Chemical Society (Feb. 1990)."><meta name="citation_reference" content="Goff, D.A., and Carrol. S.F., &quot;Substituted 2-Iminothiolanes: Reagents for the Preparation of Disulfide Cross-Linked Conjugates with Increased Stability,&quot; Bioconjugate Chem. 1:381-386, American Chemical Society (Jan. 1990)."><meta name="citation_reference" content="Habuka, N., et al., &quot;Amino Acid Sequence of Mirabilis Antiviral Proteins, Total Synthesis of Its Gene and Expression in Escherichia coli,&quot; J. Biol. Chem. 264:6629-6637, American Society for Biochemistry and Molecular Biology, Inc. (Apr. 1989)."><meta name="citation_reference" content="Halling, K.C., et al.,&quot;Genomic cloning and characterization of a ricin gene from Ricinus communis,&quot; Nucleic Acids Res. 13:8019-8033, IRL Press Limited (1985)."><meta name="citation_reference" content="Harris, P., et al., &quot;In Vitro Studies of the Effect of MAb NDA 4 Linked to Toxin on the Proliferation of a Human EBV-Transformed Lymphoblastoid B Cell Line and of Gibbon MLA Leukemia Cell Line,&quot; Immunol. 134:85-95, Academic Press, Inc. (Apr. 1991)."><meta name="citation_reference" content="Hertler, A. A., and Frankel, A.E., &quot;Immunotoxins: A Clinical Review of Their Use in the Treatment of Malignancies,&quot; J. Clin. Oncol. 7:1932-1943, American Society of Clinical Oncology (Dec. 1989)."><meta name="citation_reference" content="Ho, W.K.K., et al., &quot;Cloning of the cDNA of alpha-momorcharin: a ribosome inactivating protein,&quot; Biochem. Biophys. Acta 1088:311-314, Elsevier Science Publishers B.V. (Feb. 1991)."><meta name="citation_reference" content="Ho, W.K.K., et al., “Cloning of the cDNA of α-momorcharin: a ribosome inactivating protein,” Biochem. Biophys. Acta 1088:311-314, Elsevier Science Publishers B.V. (Feb. 1991)."><meta name="citation_reference" content="Huston, J.S., et al., &quot;Protein engineering of antibody binding sites: Recovery of specific activity in an anti-digoxin single-chain Fv analogue produced in Escherichia coli,&quot; Proc. Natl. Acad. Sci. USA 85:5879-5883, National Academy of Science USA (Aug. 1988)."><meta name="citation_reference" content="Islam, M.R., et al., &quot;Complete Amino Acid Sequence of Luffin-a, a Ribosome-inactivating Protein from the Seeds of Luffa cylindrica,&quot; Agric. Biol. Chem. 54:1343-1345, Agricultural Chemical Society of Japan (May 1990)."><meta name="citation_reference" content="Izzo, P., et al., &quot;Human aldolase A gene,&quot; Eur. J. Biochem. 174:569-578, The Federation of American Societies for Experimental Biology (1988)."><meta name="citation_reference" content="Jones, P.J., et al., &quot;Replacing the complmentarity-determining regions in a human antibody with those from a mouse,&quot; Nature 321:522-525, Macmillan Publishers Ltd. (May 1986)."><meta name="citation_reference" content="Junghans, R.P., et al., &quot;Anti-Tac-H, a Humanized Antibody to the Interleukin 2 Receptor with New Features for Immunotherapy in Malignant and Immune Disorders,&quot; Cancer Res. 50:1495-1502, American Association For Cancer Research (Mar. 1990)."><meta name="citation_reference" content="Katzin, B.J., et al., &quot;Structure of Ricin A-Chain at 2.5 Å, &quot; Proteins: Structure, Function and Genetics 10:251-259, Wiley-Liss, Inc. (Jun. 1991)."><meta name="citation_reference" content="Katzin, B.J., et al., “Structure of Ricin A-Chain at 2.5 Å, ” Proteins: Structure, Function and Genetics 10:251-259, Wiley-Liss, Inc. (Jun. 1991)."><meta name="citation_reference" content="Kernan, N.A., et al., &quot;Specific Inhibition of In Vitro Lymphocyte Transformation By An Anti-Pan T Cell (gp67) Ricin A Chain Immunotoxin,&quot; J. Immunol. 133:137-146, American Association of Immunologists (Jul. 1984)."><meta name="citation_reference" content="Kneble, K.D., et al., &quot;Quantitative Equilibrium Binding of Monoclonal Antibodies, Immunotoxins, and Radioimmunoimaging Agents by Flow Cytometry,&quot; Cytometry Suppl. 1:68, Alan R. Liss, Inc. (1987)."><meta name="citation_reference" content="Knowles, P.P., et al., &quot;Purification of Immunotoxins Containing Ricin A-Chain and Abrin A-Chain Using Blue Sepharose CL-6B,&quot; Anal. Biochem. 160:440-443, Academic Press, Inc. (1987)."><meta name="citation_reference" content="Kohn, F.R., et al., &quot;Efficacy of Anti-CD5 F(Ab&#39;)2 and FAB&#39; Immunoconjugates in Human Peripheral Blood Lymphocyte-Reconstituted Severe Combined Immunodeficient Mice,&quot; Int. J. Immunopharmac. 15:871-878, Pergamon Press, Ltd. (Nov. 1993)."><meta name="citation_reference" content="Kohr., W.J., et al., &quot;The Amino Acid Sequence of Gelonin.&quot; The Protein Society, 4th Symposium, San Diego, CA, Abstract No. T15 (Aug. 1990)."><meta name="citation_reference" content="Kong, K., et al., &quot;Subacute Intravenous Toxicity of a Ribosomal Inhibitory Protein, rGelonin, in Rats,&quot; (Poster) American College of Toxicology, San Francisco, CA (Oct. 1992)."><meta name="citation_reference" content="Kung, S.-S., &quot;The Complete Amino Acid Sequence of Antiviral Protein from the Seeds of Pokeweed (Phytolacca americana),&quot; Agric. Biol. Chem. 54:3301-3318, Agricultural Chemical Society of Japan (Sep. 1990)."><meta name="citation_reference" content="Lambert, J.M., et al., &quot;Purified Immunotoxins That Are Reactive with Human Lymphoid Cells,&quot; J. Biol. Chem. 260:12035-12041, American Society for Biochemistry and Molecular Biology, Inc. (Oct. 1985)."><meta name="citation_reference" content="Laurent, G., et al., &quot;Donor bone marrow treatment with T101 Fab fragment-ricin A-chain immunotoxin prevents graf-versus-host disease,&quot; Bone Morrow Transplantation 4:367-371, Macmillan Press Ltd. (1989)."><meta name="citation_reference" content="Lee-Huang, S.,et al., &quot;A new class of anti-HIV agents; GAP31, DAPs 30 and 32,&quot; FEBS 291:139-144, Elsevier Sicence Publishers B.V. (Oct. 1991)."><meta name="citation_reference" content="Li, S.S.-L., et al., &quot;Purification and partial characterization of two lectins from Momordica charantia,&quot; Experienta 36:524-527, Birkhäuser Verlag (1980)."><meta name="citation_reference" content="Madan, S., and Ghosh, P.C., &quot;Interaction of Gelonin With Macrophages: Effect of Lysosomotropic Amines,&quot; Exper. Cell Res. 198:52-58, Academic Press, Inc. (Jan. 1992)."><meta name="citation_reference" content="Montecucchi, P.C., et al., &quot;N-terminal sequence of some ribosome-inactivating proteins,&quot; Int. J. Peptide Protein Res. 33:263-267, Munksgaard (1989)."><meta name="citation_reference" content="Morishima, Y., et al., &quot;Functionally Different T Lymphocyte Subpopulations Determined By Their Sensitivity to Complement-Dependent Cell Lysis With the Monoclonal Antibody 4A,&quot; J. Immunol. 129:1091-1098, American Association of Immunologists (Sep. 1982)."><meta name="citation_reference" content="Mountain, A., and Adair, J.A., &quot;Engineering Antibodies for Therapy,&quot; in Biotechnology and Genetic Engineering Reviews 10:1-142, Intercept Ltd., Andover, U.K. (Dec. 1992)."><meta name="citation_reference" content="Munson, P.J., et al., &quot;Ligand: A Versatile Computerized Approach for Characterization of Ligand-Binding Systems,&quot; Anal. Biochem. 107:220-239, Academic Press, Inc. (1980)."><meta name="citation_reference" content="Myers, E.W., and Miller, W., &quot;Optimal alignments in linear space,&quot; Cabios Commun. 4:11-17, IRL Press Limited (1988)."><meta name="citation_reference" content="Nolan, P.A., et al., &quot;Cloning and expression of a gene encoding gleonin, a ribosome-inactivating protein from Gelonium multiflorum,&quot; Gene 134:223-227, Elsevier Science Publishers B.V. (Dec. 1993)."><meta name="citation_reference" content="Ogata, M., et al., &quot;Processing of Pseudomonas Exotoxin by a Cellular Protease Results in the Generation of a 37,000-Da Toxin Fragment That is Translocated to the Cytosol,&quot; J. Biol. Chem. 265:20678-20685, American Society for Biochemistry and Molecular Biology, Inc. (Nov. 1990)."><meta name="citation_reference" content="O&#39;Hare, M., et al., &quot;Cytotoxicity of a recombinant ricin-A-chain fusion protein containing a proteolytically-cleavable spacer sequence,&quot; FEBS Lett. 273:200-204, Elsevier Science Publishers B.V. (Oct. 1990)."><meta name="citation_reference" content="Oldham, R.K., et al., &quot;Adriamycin custom-tailored immunoconjugates in the treatment of human malignancies,&quot; Mol. Biother. 1:103-113, Butterworth Publishers (1988)."><meta name="citation_reference" content="Pastan, I., and FitzGerald, D., &quot;Recombinant Toxins for Cancer Treatment,&quot; Science 254:1173-1177, Association for the Advancement of Science (Nov. 1991)."><meta name="citation_reference" content="Pending Non-Provisional United States Patent Application No. 09/711,485, Better et al., filed Nov. 13, 2000."><meta name="citation_reference" content="Potter, H., et al., &quot;Enhancer-dependent expression of human kappa immunoglobulin genes introduced into mouse pre-B lymphocytes by electroporation,&quot; Proc. Natl. Acad. Sci. USA 81:7161-7165, National Academy of Sciences of the USA (Nov. 1984)."><meta name="citation_reference" content="Potter, H., et al., “Enhancer-dependent expression of human κ immunoglobulin genes introduced into mouse pre-B lymphocytes by electroporation,” Proc. Natl. Acad. Sci. USA 81:7161-7165, National Academy of Sciences of the USA (Nov. 1984)."><meta name="citation_reference" content="Preijers, F.W.M.B., et al., &quot;Autologous Transplantation of Bone Marrow Purged In Vitro With Anti-CD7-(WT1-) Ricin A Immunotoxin in T-Cell Lymphoblastic Leukemia and Lymphoma,&quot; Blood 74:1152-1158, Grune &amp; Stratton, Inc. (Aug. 1989)."><meta name="citation_reference" content="Press, O.W., et al., &quot;A simplified microassay for inhibition of protein synthesis in reticulocyte lysates by immunotoxins,&quot; Immunol. Letters 14:37-41, Elsevier Science Publishers B.V. (1986)."><meta name="citation_reference" content="Queen, C., et al., &quot;A humanized antibody that binds to the interleukin 2 receptor,&quot; Proc. Natl. Acad. Sci. USA 86:10029-10033, National Academy of Sciences of the USA (Dec. 1989)."><meta name="citation_reference" content="Ready, M., et al., &quot;Ricin-like Plant Toxins are Evolutionarily Related to Single-chain Ribosome-inhibiting Proteins from Phytolacca,&quot; J. Biol. Chem. 259:15252-15256, American Society of Biological Chemists, Inc. (Dec. 1984)."><meta name="citation_reference" content="Reimann, K. A., et al., &quot;In Vivo Administration of Lymphocyte-specific Monoclonal Antibodies in Nonhuman Primates,&quot; J. Clin. Invest. 82:129-138, American Society for Clinical Investigation, Inc. (Jul. 1988)."><meta name="citation_reference" content="Riechmann, L., et al., &quot;Reshaping human antibodies for therapy,&quot; Nature 322:323-327, Macmillan Publishers Ltd. (Mar. 1988)."><meta name="citation_reference" content="Robinson, R.R., et al., &quot;Chimeric mouse-human anti-carcinoma antibodies that mediate different anti-tumor cell biological activities,&quot; Human Antibod. and Hybridomas 2:84-93, Butterworth-Heinemann (Apr. 1991)."><meta name="citation_reference" content="Rosenblum, M.G., et al., &quot;A specific and potent immunotoxin composed of antibody ZME-018 and the plant toxin gelonin,&quot; Mol. Biother. 3:6-13, Butterworth-Heinemann (Mar. 1991)."><meta name="citation_reference" content="Rostaing-Capaillon, O., and Casellas, P., &quot;In vivo cytotoxic efficacy of immunotoxins prepared from anti-CD5 antibody linked to ricin A-chain,&quot; Cancer Immunol. Immunother. 34:24-30, Springer-Verlag (Jan. 1991)."><meta name="citation_reference" content="Rowley, S.D., et al., &quot;Efficacy of Ex Vivo Purging for Autologous Bone Marrow Transplantation in the Treatment of Acute Nonlymphoblastic Leukemia,&quot; Blood 74:501-506, Grune &amp; Stratton, Inc. (Jul. 1989)."><meta name="citation_reference" content="Royston, I., et al., &quot;Human T Cell Antigens Defined By Monoclonal Antibodies: The 65,000-Dalton Antigen of T Cells (T65) Is Also Found On Chronic Lymphocytic Leukemia Cells Bearing Surface Immunoglobulin,&quot; J. Immunol. 125:725-731, The Williams &amp; Wilkins Co. (Aug. 1980)."><meta name="citation_reference" content="Scott, C.F., et al., &quot;The antileukemic efficacy of an immunotoxin composed of a monoclonal anti-Thy-1 antibody disulfide linked to the ribosome-inactivating protein gelonin,&quot; Cancer Immunol. Immunother 25:31-40, Springer-Verlag (Jul. 1987)."><meta name="citation_reference" content="Shaw, P.-C., et al., &quot;Cloning of trichosanthin cDNA and its expression in Escherichai coli,&quot; Gene 97:267-272, Elsevier (Jan. 1991)."><meta name="citation_reference" content="Singh, V., and Curtiss III, R., &quot;Hormonotoxins: Abrogation of Ribosome Inactivatinb Property of Gelonin in the Disulfide Linked Ovine Lutenizing Hormone-Gelonin Conjugates,&quot; Biochem. Intl. 25:531-536, Academic Press Australia (Oct. 1991)."><meta name="citation_reference" content="Sivam, G., et al., &quot;Immjnotoxins to a Human Melanoma-associated Antigen: Comparison of Gelonin With Ricin and Other A Chain Conjugates,&quot; Cancer Res. 47:3169-3173, American Association For Cancer Research (Jun. 1987)."><meta name="citation_reference" content="Sperti, S., et al., &quot;Requirements for the inactivation of ribosomes by gelonin,&quot; Biochem J. 227:281-284, London Portland Press On Behalf Of The Biochemical Society (Jul. 1991)."><meta name="citation_reference" content="Stirpe, F., et al., &quot;Gelonin, a New Inhibitor of Protein Synthesis, Nontoxic to Intact Cells,&quot; J. Biol. Chem. 255:6947-6953, American Society for Biochemistry and Molecular Biology, Inc. (Jul. 1980)."><meta name="citation_reference" content="Stirpe, F., et al., &quot;Modification of ribosomal RNA by ribosome-inactivating proteins from plants,&quot; Nucleic Acids Research, 16:1349-1357, IRL Press Limited (Feb. 1988)."><meta name="citation_reference" content="Stirpe, F., et al., &quot;Ribosome-inactivating proteins up to date,&quot; FEBS Lett. 195:1-8, Elsevier Science Publishers B.V. (Jan. 1986)."><meta name="citation_reference" content="Strockbine, N.A., et al., &quot;Cloning and Sequencing of the Genes for Shiga Toxin from Shigella dysenteria Type 1,&quot; J. Bacteriol. 170:1116-1122, American Society for Microbiology (1988)."><meta name="citation_reference" content="Thorpe, P.E., et al., &quot;Improved Antitumor Effects of Immunotoxins Prepared with Deglycosylated Ricin A-Chain and Hindered Disulfide Linkages,&quot; Cancer Res. 48:6396-6403, American Association For Cancer Research (Nov. 1988)."><meta name="citation_reference" content="Thorpe, P.E., et al., &quot;New Coupling Agents for the Synthesis of Immunotoxins Containing a Hindered Disulfide Bond with Improved Stability in Vivo,&quot; Cancer Res. 47:5924-5931 (1987)."><meta name="citation_reference" content="Tolan, D.R., et al., &quot;The Complete Nucleotide Sequence for Rabbit Muscle Aldolase A Messenger RNA,&quot; J. Biol. Chem. 259:1127-1130, American Society for Biochemistry and Molecular Biology, Inc. (1984)."><meta name="citation_reference" content="Vallera, D.A., &quot;Immunotoxins: Will Their Clinical Promise Be Fulfilled?&quot; Blood 83:309-317, American Society for Hematology (Jan. 1994)."><meta name="citation_reference" content="Wawrzynczak, E.J., et al., &quot;Pharmacokinetics in the Rat of a Panel of Immunotoxins Made With Abrin A Chain, Ricin A Chain, Gelonin, and Momordin,&quot; Cancer Res. 50:7519-7526, American Association For Cancer Research (Dec. 1990)."><meta name="citation_reference" content="Westby, M., et al., &quot;Preparation and Characterization of Recombinant Proricin Containing an Alternative Protease-Sensitive Linker Sequence,&quot; Bioconjugate Chem. 3:375-381, American Chemical Society (Sep. 1992)."><meta name="citation_reference" content="Williams, D.P., et al., &quot;Diphtheria toxin receptor binding domain substitution with interleukin-2: genetic construction and properties of a diphtheria toxin-related interleukin-2 fusion protein,&quot; Protein Engineering 1(6):493-498, Oxford University Press (1987)."><meta name="citation_patent_number" content="US:6376217"><meta name="citation_patent_application_number" content="US:09/610,838"><link rel="canonical" href="http://www.google.com/patents/US6376217"/><meta property="og:url" content="http://www.google.com/patents/US6376217"/><meta name="title" content="Patent US6376217 - Fusion protein comprising gelonin amino acid sequence having enzymatic activity and antibody, antigen-binding fragment, hormone, lymphokine or growth factor targeting sequence allowing internalization of fusion protein"/><meta name="description" content="The present invention provides purified and isolated polynucleotides encoding Type I ribosome-inactivating proteins (RIPS) and analogs of the RIPs having a cysteine available for disulfide bonding to targeting molecules. Vectors comprising the polynucleotides and host cells transformed with the vectors are also provided. The RIPs and RIP analogs are particularly suited for use as components of cytotoxic therapeutic agents of the invention which include gene fusion products and immunoconjugates. Cytotoxic therapeutic agents or immunotoxins according to the present invention may be used to selectively eliminate any cell type to which the RIP component is targeted by the specific binding capacity of the second component of the agent, and are suited for treatment of diseases where the elimination of a particular cell type is a goal, such as autoimmune disease, cancer and graft-versus-host disease."/><meta property="og:title" content="Patent US6376217 - Fusion protein comprising gelonin amino acid sequence having enzymatic activity and antibody, antigen-binding fragment, hormone, lymphokine or growth factor targeting sequence allowing internalization of fusion protein"/><meta property="og:type" content="book"/><meta property="og:site_name" content="Google Books"/><meta property="og:image" content="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><link rel="image_src" href="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><script>(function(){try{var aa=function(a,b,c,d){d=d||{};d._sn=["cfg",b,c].join(".");window.gbar.logger.ml(a,d)};var g=window.gbar=window.gbar||{},l=window.gbar.i=window.gbar.i||{},m={},n;function _tvn(a,b){var c=parseInt(a,10);return isNaN(c)?b:c}function _tvf(a,b){var c=parseFloat(a);return isNaN(c)?b:c}function _tvv(a){return!!a}function p(a,b,c){(c||g)[a]=b}g.bv={n:_tvn("2",0),r:"",f:".67.",e:"0",m:_tvn("0",1)};
function q(a,b,c){var d="on"+b;if(a.addEventListener)a.addEventListener(b,c,!1);else if(a.attachEvent)a.attachEvent(d,c);else{var f=a[d];a[d]=function(){var a=f.apply(this,arguments),b=c.apply(this,arguments);return void 0==a?b:void 0==b?a:b&&a}}}var s=function(a){return function(){return g.bv.m==a}},ba=s(1),ca=s(2);p("sb",ba);p("kn",ca);l.a=_tvv;l.b=_tvf;l.c=_tvn;l.i=aa;var da=window.gbar.i.i;var t,u,v,w;function ea(a){v=a}function fa(a){var b;if(b=v&&window.encodeURIComponent)b=a.href,b=!b.match(/^http[s]?:\/\/accounts\.google\.[^/]*\/ClearSID/i)&&!b.match(/^http[s]?:\/\/[^/]*\/accounts\/ClearSID/i);if(b=b&&encodeURIComponent(v()))a.href=a.href.replace(/([?&]continue=)[^&]*/,"$1"+b)}function ga(a){window.gApplication&&(a.href=window.gApplication.getTabUrl(a.href))}
function ha(a){var b=document.forms[0].q,c=window.encodeURIComponent&&b&&b.value,b=b&&b.placeholder;c&&c!=b&&(a.href=a.href.replace(/([?&])q=[^&]*|$/,function(a,b){return(b||"&")+"q="+encodeURIComponent(c)}))}n=l.a("")?ga:ha;
function x(a,b,c,d,f,e){var h=document.getElementById(a);if(h){var k=h.style;k.left=d?"auto":b+"px";k.right=d?b+"px":"auto";k.top=c+"px";k.visibility=u?"hidden":"visible";f&&e?(k.width=f+"px",k.height=e+"px"):(x(t,b,c,d,h.offsetWidth,h.offsetHeight),u=u?"":a)}}
var y=[],ia=function(a,b){y.push(b)},ja=function(a){a=a||window.event;var b=a.target||a.srcElement;a.cancelBubble=!0;null==t&&(a=document.createElement(Array.every||window.createPopup?"iframe":"div"),a.frameBorder="0",t=a.id="gbs",a.src="javascript:''",b.parentNode.appendChild(a),q(document,"click",z));var c=b,b=0;"gb3"!=c.className&&(c=c.parentNode);a=c.getAttribute("aria-owns")||"gbi";var d=c.offsetWidth,f=20<c.offsetTop?46:24;document.getElementById("tphdr")&&(f-=3);var e=!1;do b+=c.offsetLeft||
0;while(c=c.offsetParent);var c=(document.documentElement.clientWidth||document.body.clientWidth)-b-d,h,d=document.body,k=document.defaultView;k&&k.getComputedStyle?(d=k.getComputedStyle(d,""))&&(h=d.direction):h=d.currentStyle?d.currentStyle.direction:d.style.direction;h="rtl"==h;if("gbi"==a){for(d=0;k=y[d++];)k();A(null,window.navExtra);h&&(b=c,e=!0)}else h||(b=c,e=!0);u!=a&&z();x(a,b,f,e)},z=function(){u&&x(u,0,0)},A=function(a,b){var c,d=document.getElementById("gbi"),f=a;f||(f=d.firstChild);
for(;b&&(c=b.pop());){var e=d,h=c,k=f;w||(w="gb2");e.insertBefore(h,k).className=w}},ka=function(a,b,c){if((b=document.getElementById(b))&&a){a.className="gb4";var d=document.createElement("span");d.appendChild(a);d.appendChild(document.createTextNode(" | "));d.id=c;b.appendChild(d)}},la=function(){return document.getElementById("gb_70")},ma=function(){return!!u};p("qs",n);p("setContinueCb",ea);p("pc",fa);p("tg",ja);p("close",z);p("addLink",ka);p("almm",A);p("si",la);p("adh",ia);p("op",ma);var B=function(){},C=function(){},F=function(a){var b=new Image,c=D;b.onerror=b.onload=b.onabort=function(){try{delete E[c]}catch(a){}};E[c]=b;b.src=a;D=c+1},E=[],D=0;p("logger",{il:C,ml:B,log:F});var G=window.gbar.logger;var H={},na={},I=[],oa=l.b("0.1",.1),pa=l.a("1",!0),qa=function(a,b){I.push([a,b])},ra=function(a,b){H[a]=b},sa=function(a){return a in H},J={},K=function(a,b){J[a]||(J[a]=[]);J[a].push(b)},ta=function(a){K("m",a)},L=function(a,b){var c=document.createElement("script");c.src=a;c.async=pa;Math.random()<oa&&(c.onerror=function(){c.onerror=null;B(Error("Bundle load failed: name="+(b||"UNK")+" url="+a))});(document.getElementById("xjsc")||document.getElementsByTagName("body")[0]||
document.getElementsByTagName("head")[0]).appendChild(c)},N=function(a){for(var b=0,c;(c=I[b])&&c[0]!=a;++b);!c||c[1].l||c[1].s||(c[1].s=!0,M(2,a),c[1].url&&L(c[1].url,a),c[1].libs&&m.d&&m.d(c[1].libs))},O=function(a){K("gc",a)},P=null,ua=function(a){P=a},M=function(a,b,c){if(P){a={t:a,b:b};if(c)for(var d in c)a[d]=c[d];try{P(a)}catch(f){}}};p("mdc",H);p("mdi",na);p("bnc",I);p("qGC",O);p("qm",ta);p("qd",J);p("lb",N);p("mcf",ra);p("bcf",qa);p("aq",K);p("mdd","");p("has",sa);
p("trh",ua);p("tev",M);var Q=l.b("0.1",.001),R=0;
function _mlToken(a,b){try{if(1>R){R++;var c,d=a,f=b||{},e=encodeURIComponent,h=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&jexpid=",e("17483"),"&srcpg=",e("prop=22"),"&jsr=",Math.round(1/Q),"&ogev=",e("Ga3tU8_VKsW4sASeoYLICw"),"&ogf=",g.bv.f,"&ogrp=",e("1"),"&ogv=",e("1407723702.0"),"&oggv="+e("es_plusone_gc_20140723.0_p0"),"&ogd=",e("com"),"&ogc=",e("PAN"),"&ogl=",e("en")];f._sn&&(f._sn="og."+
f._sn);for(var k in f)h.push("&"),h.push(e(k)),h.push("="),h.push(e(f[k]));h.push("&emsg=");h.push(e(d.name+":"+d.message));var r=h.join("");S(r)&&(r=r.substr(0,2E3));c=r;var Aa=window.gbar.logger._aem(a,c);F(Aa)}}catch(Na){}}var S=function(a){return 2E3<=a.length},va=function(a,b){return b};function T(a){B=a;p("_itl",S,G);p("_aem",va,G);p("ml",B,G);a={};H.er=a}l.a("")?T(function(a){throw a;}):l.a("1")&&Math.random()<Q&&T(_mlToken);I.push(["m",{url:"//ssl.gstatic.com/gb/js/scm_7385cc5883250b43a39405734c1bea59.js"}]);g.mcf("c",{});g.sg={c:""};if(l.a("1")){var wa=l.a("");I.push(["gc",{auto:wa,url:"//ssl.gstatic.com/gb/js/abc/gci_91f30755d6a6b787dcc2a4062e6e9824.js",libs:"googleapis.client:plusone:gapi.iframes"}]);var xa={version:"gci_91f30755d6a6b787dcc2a4062e6e9824.js",index:"",lang:"en"};H.gc=xa;var U=function(a){window.googleapis&&window.iframes?a&&a():(a&&O(a),N("gc"))};p("lGC",U);l.a("1")&&p("lPWF",U)};window.__PVT="";if(l.a("1")&&l.a("1")){var V=function(a){U(function(){K("pw",a);N("pw")})};p("lPW",V);I.push(["pw",{url:"//ssl.gstatic.com/gb/js/abc/pwm_45f73e4df07a0e388b0fa1f3d30e7280.js"}]);var W=[],ya=function(a){W[0]=a},za=function(a,b){var c=b||{};c._sn="pw";B(a,c)},Ba={signed:W,elog:za,base:"https://plusone.google.com/u/0",loadTime:(new Date).getTime()};H.pw=Ba;var X=function(a,b){for(var c=b.split("."),d=function(){var b=arguments;a(function(){for(var a=g,d=0,e=c.length-1;d<e;++d)a=a[c[d]];a[c[d]].apply(a,b)})},f=g,e=0,h=c.length-1;e<h;++e)f=
f[c[e]]=f[c[e]]||{};return f[c[e]]=d};X(V,"pw.clk");X(V,"pw.hvr");p("su",ya,g.pw)};function Ca(){function a(){for(var b;(b=e[h++])&&"m"!=b[0]&&!b[1].auto;);b&&(M(2,b[0]),b[1].url&&L(b[1].url,b[0]),b[1].libs&&m.d&&m.d(b[1].libs));h<e.length&&setTimeout(a,0)}function b(){0<f--?setTimeout(b,0):a()}var c=l.a("1"),d=l.a(""),f=3,e=I,h=0,k=window.gbarOnReady;if(k)try{k()}catch(r){da(r,"ml","or")}d?p("ldb",a):c?q(window,"load",b):b()}p("rdl",Ca);var Da={D:1,H:2,da:3,p:4,W:5,M:6,F:7,g:8,ha:9,U:10,L:11,T:12,S:13,N:14,Q:15,P:16,fa:17,w:18,O:19,ga:20,ea:21,u:22,G:23,ja:24,ka:25,ia:26,A:27,j:28,o:29,k:30,ca:31,Z:32,$:33,J:34,K:35,ba:36,aa:37,Y:38,B:39,R:40,v:41,X:42,V:43,h:48,C:49,I:500},Y=[1,2,3,4,5,6,9,10,11,13,14,28,29,30,34,35,37,38,39,40,41,42,43,48,49,500];var Z=l.b("0.001",1E-4),Ea=l.b("1",1),Fa=!1,Ga=!1;if(l.a("1")){var Ha=Math.random();Ha<=Z&&(Fa=!0);Ha<=Ea&&(Ga=!0)}var Ia=Da,$=null;function Ja(){var a=0,b=function(b,d){l.a(d)&&(a|=b)};b(1,"");b(2,"");b(4,"");b(8,"");return a}
function Ka(a,b){var c=Z,d=Fa,f;f=a;if(!$){$={};for(var e=0;e<Y.length;e++){var h=Y[e];$[h]=!0}}if(f=!!$[f])c=Ea,d=Ga;if(d){d=encodeURIComponent;g.rp?(f=g.rp(),f="-1"!=f?f:"1"):f="1";c=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&oge=",a,"&ogex=",d("17483"),"&ogev=",d("Ga3tU8_VKsW4sASeoYLICw"),"&ogf=",g.bv.f,"&ogp=",d("22"),"&ogrp=",d(f),"&ogsr=",Math.round(1/c),"&ogv=",d("1407723702.0"),"&oggv="+
d("es_plusone_gc_20140723.0_p0"),"&ogd=",d("com"),"&ogl=",d("en"),"&ogc=",d("PAN"),"&ogus=",Ja()];if(b){"ogw"in b&&(c.push("&ogw="+b.ogw),delete b.ogw);var k;f=b;e=[];for(k in f)0!=e.length&&e.push(","),e.push(La(k)),e.push("."),e.push(La(f[k]));k=e.join("");""!=k&&(c.push("&ogad="),c.push(d(k)))}F(c.join(""))}}function La(a){"number"==typeof a&&(a+="");return"string"==typeof a?a.replace(".","%2E").replace(",","%2C"):a}C=Ka;p("il",C,G);var Ma={};H.il=Ma;setTimeout(function(){C(Ia.g)},0);}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var b=window.gbar.i.i;var c=window.gbar;var f=function(d){try{var a=document.getElementById("gbom");a&&d.appendChild(a.cloneNode(!0))}catch(e){b(e,"omas","aomc")}};c.aomc=f;}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var a=window.gbar;a.mcf("pm",{p:""});}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{window.gbar.rdl();}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
if (window['_OC_timingAction']) {window['_OC_timingAction']('patents_refpage');}</script><style>#gbar,#guser{font-size:13px;padding-top:1px !important;}#gbar{float:left;height:22px}#guser{padding-bottom:7px !important;text-align:right}.gbh,.gbd{border-top:1px solid #c9d7f1;font-size:1px}.gbh{height:0;position:absolute;top:24px;width:100%}#gbs,.gbm{background:#fff;left:0;position:absolute;text-align:left;visibility:hidden;z-index:1000}.gbm{border:1px solid;border-color:#c9d7f1 #36c #36c #a2bae7;z-index:1001}.gb1{margin-right:.5em}.gb1,.gb3{zoom:1}.gb2{display:block;padding:.2em .5em}.gb2,.gb3{text-decoration:none !important;border-bottom:none}a.gb1,a.gb4{text-decoration:underline !important}a.gb1,a.gb2,a.gb3,a.gb4{color:#00c !important}.gbi .gb3,.gbi .gb2,.gbi .gb4{color:#dd8e27 !important}.gbf .gb3,.gbf .gb2,.gbf .gb4{color:#900 !important}a.gb2:hover{background:#36c;color:#fff !important}#gbar .gbz0l{color:#000 !important;cursor:default;font-weight:bold;text-decoration:none !important}
#gbar { padding:.3em .6em !important;}</style></head><body  topmargin="3" marginheight="3"><div id=gbar><nobr><a onclick=gbar.qs(this);gbar.logger.il(1,{t:1}); class=gb1 id=gb_1 href="https://www.google.com/search?sa=N&tab=tw">Search</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:2}); class=gb1 id=gb_2 href="http://www.google.com/search?hl=en&tbm=isch&source=og&sa=N&tab=ti">Images</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:8}); class=gb1 id=gb_8 href="http://maps.google.com/maps?hl=en&sa=N&tab=tl">Maps</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:78}); class=gb1 id=gb_78 href="https://play.google.com/?hl=en&sa=N&tab=t8">Play</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:36}); class=gb1 id=gb_36 href="http://www.youtube.com/results?sa=N&tab=t1">YouTube</a> <a onclick=gbar.logger.il(1,{t:5}); class=gb1 id=gb_5 href="http://news.google.com/nwshp?hl=en&tab=tn">News</a> <a onclick=gbar.logger.il(1,{t:23}); class=gb1 id=gb_23 href="https://mail.google.com/mail/?tab=tm">Gmail</a> <a onclick=gbar.logger.il(1,{t:25}); class=gb1 id=gb_25 href="https://drive.google.com/?tab=to">Drive</a> <a class=gb3 href="http://www.google.com/intl/en/options/" onclick="this.blur();gbar.tg(event);return !1" aria-haspopup=true><u>More</u> <small>&#9660;</small></a><div class=gbm id=gbi><a onclick=gbar.logger.il(1,{t:24}); class=gb2 id=gb_24 href="https://www.google.com/calendar?tab=tc">Calendar</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:51}); class=gb2 id=gb_51 href="http://translate.google.com/?hl=en&sa=N&tab=tT">Translate</a><a onclick=gbar.logger.il(1,{t:17}); class=gb2 id=gb_17 href="http://www.google.com/mobile/?hl=en&tab=tD">Mobile</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:10}); class=gb2 id=gb_10 href="http://www.google.com/search?hl=en&tbo=u&tbm=bks&source=og&sa=N&tab=tp">Books</a><a onclick=gbar.logger.il(1,{t:212}); class=gb2 id=gb_212 href="https://wallet.google.com/manage/?tab=ta">Wallet</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:6}); class=gb2 id=gb_6 href="http://www.google.com/search?hl=en&tbo=u&tbm=shop&source=og&sa=N&tab=tf">Shopping</a><a onclick=gbar.logger.il(1,{t:30}); class=gb2 id=gb_30 href="http://www.blogger.com/?tab=tj">Blogger</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:27}); class=gb2 id=gb_27 href="http://www.google.com/finance?sa=N&tab=te">Finance</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:31}); class=gb2 id=gb_31 href="https://plus.google.com/photos?sa=N&tab=tq">Photos</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:12}); class=gb2 id=gb_12 href="http://www.google.com/search?hl=en&tbo=u&tbm=vid&source=og&sa=N&tab=tv">Videos</a><div class=gb2><div class=gbd></div></div><a onclick=gbar.logger.il(1,{t:66}); href="http://www.google.com/intl/en/options/" class=gb2>Even more &raquo;</a></div></nobr></div><div id=guser width=100%><nobr><span id=gbn class=gbi></span><span id=gbf class=gbf></span><span id=gbe></span><a target=_top id=gb_70 href="https://www.google.com/accounts/Login?service=&continue=http://www.google.com/patents%3Fhl%3Den&hl=en" class=gb4>Sign in</a><div style="display: none"><div class=gbm id=gbd5 aria-owner=gbg5><div class=gbmc><ol id=gbom class=gbmcc></ol></div></div></div></nobr></div><div class=gbh style=left:0></div><div class=gbh style=right:0></div><div role="alert" style="position: absolute; left: 0; right: 0;"><a href="http://www.google.com/patents/us6376217?hl=en&amp;output=html_text" title="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."><img border="0" src="http://www.google.com/images/cleardot.gif"alt="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."></a></div><div id="guser"><nobr></nobr></div><div style="clear:both;"></div><div id="gb-top-search-box" class="gb-top-search-box-small gb-reset"><table><tr><td class="logo"><a href="http://www.google.com/patents" class="logo-link"><img class="logo-img" src="/intl/en/images/logos/google_logo_41.png" alt="Go to Google Books Home" height="41"/></a></td><td><form action="http://www.google.com/search" name="f" id="vheadf" method="get"><span id="hf"></span><input type="hidden" name="tbm" value="pts"/><input type="hidden" name="tbo" value="1"/><input type="hidden" name="hl" value="en"/><table><tr><td><div class="inputs"><table><tr><td><div class="text-input"><input type="text" name="q" id="vheadq" class="text" maxlength="2048" size="31" value="" title="Search Patents" accesskey="s" autocomplete="off"/><script>window._OC_autoDir &&window._OC_autoDir('vheadq', 'tia-vheadq');</script></div></td><td><div class="submit-input"><input name="btnG" class="submit" type="submit" value=""/></div></td></tr></table></div></td><td class="col-ext-links"><div class="ext-links"><a href="http://www.google.com/advanced_patent_search">&lt;nobr&gt;Advanced Patent Search&lt;/nobr&gt;</a></div></td></tr></table></form></td></tr></table></div><div class="kd-appbar"><h2 class="kd-appname"><a href="/patents">Patents</a></h2><div class="kd-buttonbar left" id="left-toolbar-buttons"><a id="appbar-write-review-link" href=""></a><a id="appbar-view-print-sample-link" href=""></a><a id="appbar-view-ebook-sample-link" href=""></a><a id="appbar-patents-prior-art-finder-link" href="https://www.google.com/patents/related/US6376217"></a><a id="appbar-patents-discuss-this-link" href="http://www.google.com/url?id=CsRZBAABERAJ&amp;q=http://patents.stackexchange.com/redirect/google-patents%3Fpatent%3DUS6376217&amp;usg=AFQjCNHUu5KYtOzIOlo_8_3bU4KfAl9PUQ" data-is-grant="true"></a><a id="appbar-read-patent-link" href="//docs.google.com/viewer?url=patentimages.storage.googleapis.com/pdfs/US6376217.pdf"></a><a id="appbar-download-pdf-link" href="//patentimages.storage.googleapis.com/pdfs/US6376217.pdf"></a></div><div class="kd-buttonbar right" id="right-toolbar-buttons"></div></div><div id="books-microdata" itemscope=""itemtype="http://schema.org/Book"itemid="http://www.google.com/patents/US6376217" style="display:none"><span itemprop="description">The present invention provides purified and isolated polynucleotides encoding Type I ribosome-inactivating proteins (RIPS) and analogs of the RIPs having a cysteine available for disulfide bonding to targeting molecules. Vectors comprising the polynucleotides and host cells transformed with the vectors...</span><span itemprop="url">http://www.google.com/patents/US6376217?utm_source=gb-gplus-share</span><span class="main-title" itemprop="name">Patent US6376217 - Fusion protein comprising gelonin amino acid sequence having enzymatic activity and antibody, antigen-binding fragment, hormone, lymphokine or growth factor targeting sequence allowing internalization of fusion protein</span><img itemprop="image" src="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"alt="Patent US6376217 - Fusion protein comprising gelonin amino acid sequence having enzymatic activity and antibody, antigen-binding fragment, hormone, lymphokine or growth factor targeting sequence allowing internalization of fusion protein" title="Patent US6376217 - Fusion protein comprising gelonin amino acid sequence having enzymatic activity and antibody, antigen-binding fragment, hormone, lymphokine or growth factor targeting sequence allowing internalization of fusion protein"></div><div style="display: none"><ol id="ofe-gear-menu-contents" class="gbmcc"><li class="gbe gbmtc"><a class="gbmt goog-menuitem-content" id="" href="http://www.google.com/advanced_patent_search">Advanced Patent Search</a></li></ol></div><table id="viewport_table" cellpadding="0" style="clear:both" cellspacing="0"><tr><td id="viewport_td"><div class=vertical_module_list_row><div id=intl_patents class=about_content><div id=intl_patents_v><table class="patent-bibdata"><tr><td class="patent-bibdata-heading">Publication number</td><td class="single-patent-bibdata">US6376217 B1</td></tr><tr><td class="patent-bibdata-heading">Publication type</td><td class="single-patent-bibdata">Grant</td></tr><tr><td class="patent-bibdata-heading">Application number</td><td class="single-patent-bibdata">US 09/610,838</td></tr><tr><td class="patent-bibdata-heading">Publication date</td><td class="single-patent-bibdata">Apr 23, 2002</td></tr><tr><td class="patent-bibdata-heading">Filing date</td><td class="single-patent-bibdata">Jul 6, 2000</td></tr><tr><td class="patent-bibdata-heading">Priority date<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The priority date is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the date listed."></span></td><td class="single-patent-bibdata">Nov 4, 1991</td></tr><tr><td class="patent-bibdata-heading">Fee status<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The fee status is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the status or dates listed."></span></td><td class="single-patent-bibdata">Lapsed</td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Also published as</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/US6146850">US6146850</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20030166196">US20030166196</a></span></span></td></tr><tr class="patent-bibdata-list-row alternate-patent-number"><td class="patent-bibdata-heading">Publication number</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value">09610838, </span><span class="patent-bibdata-value">610838, </span><span class="patent-bibdata-value">US 6376217 B1, </span><span class="patent-bibdata-value">US 6376217B1, </span><span class="patent-bibdata-value">US-B1-6376217, </span><span class="patent-bibdata-value">US6376217 B1, </span><span class="patent-bibdata-value">US6376217B1</span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Inventors</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Marc+D.+Better%22">Marc D. Better</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Stephen+F.+Carroll%22">Stephen F. Carroll</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Original Assignee</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=inassignee:%22Xoma+Technology+Ltd.%22">Xoma Technology Ltd.</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Export Citation</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/US6376217.bibtex">BiBTeX</a>, </span><span class="patent-bibdata-value"><a href="/patents/US6376217.enw">EndNote</a>, </span><span class="patent-bibdata-value"><a href="/patents/US6376217.ris">RefMan</a></span></span></td></tr><tr class="patent-internal-links"><td colspan=2><span class="patent-bibdata-value"><a href="#backward-citations">Patent Citations</a> (57),</span> <span class="patent-bibdata-value"><a href="#npl-citations">Non-Patent Citations</a> (104),</span> <span class="patent-bibdata-value"><a href="#forward-citations">Referenced by</a> (4),</span> <span class="patent-bibdata-value"><a href="#classifications">Classifications</a> (35),</span> <span class="patent-bibdata-value"><a href="#legal-events">Legal Events</a> (8)</span> </td></tr><tr><td colspan=2 class="patent-bibdata-external-link-spacer-top"></td></tr><tr class="patent-bibdata-external-link-spacer-bottom"></tr><tr><td colspan=2><span class="patent-bibdata-heading">External Links:&nbsp;</span><span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=CsRZBAABERAJ&q=http://patft.uspto.gov/netacgi/nph-Parser%3FSect2%3DPTO1%26Sect2%3DHITOFF%26p%3D1%26u%3D/netahtml/PTO/search-bool.html%26r%3D1%26f%3DG%26l%3D50%26d%3DPALL%26RefSrch%3Dyes%26Query%3DPN/6376217&usg=AFQjCNHtw-6O2y_xJEk5QF4gMYJSK27WKQ">USPTO</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=CsRZBAABERAJ&q=http://assignments.uspto.gov/assignments/q%3Fdb%3Dpat%26pat%3D6376217&usg=AFQjCNEsb8eDC_JofSMeCfT_AkXuLei6Eg">USPTO Assignment</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=CsRZBAABERAJ&q=http://worldwide.espacenet.com/publicationDetails/biblio%3FCC%3DUS%26NR%3D6376217B1%26KC%3DB1%26FT%3DD&usg=AFQjCNHO7465Mctloj-cNT2WLBD14o9zoA">Espacenet</a></span></span></td></tr><tr class="patent-bibdata-group-spacer"></tr></table><div class="number-and-title"><span class="patent-title"><invention-title mxw-id="PT129084590" lang="EN" load-source="ifi">Fusion protein comprising gelonin amino acid sequence having enzymatic activity and antibody, antigen-binding fragment, hormone, lymphokine or growth factor targeting sequence allowing internalization of fusion protein</invention-title></span><br><span class="patent-number">US 6376217 B1</span></div><div class="patent-section patent-abstract-section"><div class="patent-section-header"><span class="patent-section-title">Abstract</span></div><div class="patent-text"><abstract mxw-id="PA50306315" lang="EN" load-source="patent-office"> <div class="abstract">The present invention provides purified and isolated polynucleotides encoding Type I ribosome-inactivating proteins (RIPS) and analogs of the RIPs having a cysteine available for disulfide bonding to targeting molecules. Vectors comprising the polynucleotides and host cells transformed with the vectors are also provided. The RIPs and RIP analogs are particularly suited for use as components of cytotoxic therapeutic agents of the invention which include gene fusion products and immunoconjugates. Cytotoxic therapeutic agents or immunotoxins according to the present invention may be used to selectively eliminate any cell type to which the RIP component is targeted by the specific binding capacity of the second component of the agent, and are suited for treatment of diseases where the elimination of a particular cell type is a goal, such as autoimmune disease, cancer and graft-versus-host disease.</div>
  </abstract></div></div><div class="patent-section patent-drawings-section"><div class="patent-section-header"><span class="patent-section-title">Images<span class="patent-section-count">(15)</span></span></div><div class="patent-drawings-body"><div class="patent-drawings-carousel"><div class="drawings"><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6376217B1/US06376217-20020423-D00001.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6376217B1/US06376217-20020423-D00001.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6376217B1/US06376217-20020423-D00002.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6376217B1/US06376217-20020423-D00002.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6376217B1/US06376217-20020423-D00003.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6376217B1/US06376217-20020423-D00003.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6376217B1/US06376217-20020423-D00004.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6376217B1/US06376217-20020423-D00004.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6376217B1/US06376217-20020423-D00005.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6376217B1/US06376217-20020423-D00005.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6376217B1/US06376217-20020423-D00006.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6376217B1/US06376217-20020423-D00006.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6376217B1/US06376217-20020423-D00007.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6376217B1/US06376217-20020423-D00007.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6376217B1/US06376217-20020423-D00008.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6376217B1/US06376217-20020423-D00008.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6376217B1/US06376217-20020423-D00009.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6376217B1/US06376217-20020423-D00009.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6376217B1/US06376217-20020423-D00010.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6376217B1/US06376217-20020423-D00010.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6376217B1/US06376217-20020423-D00011.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6376217B1/US06376217-20020423-D00011.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6376217B1/US06376217-20020423-D00012.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6376217B1/US06376217-20020423-D00012.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6376217B1/US06376217-20020423-D00013.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6376217B1/US06376217-20020423-D00013.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6376217B1/US06376217-20020423-D00014.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6376217B1/US06376217-20020423-D00014.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6376217B1/US06376217-20020423-D00015.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6376217B1/US06376217-20020423-D00015.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div></div></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img" alt="Previous page"src="/googlebooks/images/kennedy/page_left.png"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img" alt="Next page"src="/googlebooks/images/kennedy/page_right.png"width="21" height="21" /></div></div></div><div class="patent-post-drawings"></div><div class="patent-section patent-claims-section"><div class="patent-section-header"><span class="patent-section-title">Claims<span class="patent-section-count">(15)</span></span></div><div class="patent-text"><div mxw-id="PCLM8278184" lang="EN" load-source="patent-office" class="claims">
    <claim-statement>What is claimed is: </claim-statement> <div class="claim"> <div num="1" id="US-6376217-B1-CLM-00001" class="claim">
      <div class="claim-text">1. A fusion protein comprising (a) a gelonin amino acid sequence that has enzymatic activity and (b) a targeting sequence that allows the internalization of said fusion protein, wherein said targeting sequence is an antibody, an antigen-binding portion of an antibody, a hormone, a lymphokine or a growth factor.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="2" id="US-6376217-B1-CLM-00002" class="claim">
      <div class="claim-text">2. The fusion protein of <claim-ref idref="US-6376217-B1-CLM-00001">claim 1</claim-ref>, further comprising a linker sequence between said gelonin sequence and said targeting sequence, wherein said gelonin possesses enzymatic activity, said antibody is capable of recognizing antigen and said hormone, lymphokine or growth factor is capable of binding to a cell that has a receptor for said hormone lymphokine or growth factor.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="3" id="US-6376217-B1-CLM-00003" class="claim">
      <div class="claim-text">3. The fusion protein of <claim-ref idref="US-6376217-B1-CLM-00002">claim 2</claim-ref>, wherein said linker sequence is that of SEQ ID NO. 56 or SEQ ID NO. 57.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="4" id="US-6376217-B1-CLM-00004" class="claim">
      <div class="claim-text">4. The fusion protein of any one of claims <b>1</b>, <b>2</b> and <b>3</b>, wherein said targeting sequence is an antibody.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="5" id="US-6376217-B1-CLM-00005" class="claim">
      <div class="claim-text">5. The fusion protein of any one of claims <b>1</b>, <b>2</b> and <b>3</b>, wherein said targeting sequence is an antigen-binding portion of an antibody.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="6" id="US-6376217-B1-CLM-00006" class="claim">
      <div class="claim-text">6. The fusion protein of <claim-ref idref="US-6376217-B1-CLM-00005">claim 5</claim-ref>, wherein said antigen-binding portion of said antibody is an Fab.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="7" id="US-6376217-B1-CLM-00007" class="claim">
      <div class="claim-text">7. The fusion protein of <claim-ref idref="US-6376217-B1-CLM-00005">claim 5</claim-ref>, wherein, wherein said antigen-binding portion of said antibody is an Fab′.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="8" id="US-6376217-B1-CLM-00008" class="claim">
      <div class="claim-text">8. The fusion protein of <claim-ref idref="US-6376217-B1-CLM-00005">claim 5</claim-ref>, wherein said antigen-binding portion of said antibody is an F(ab′)<sub>2</sub>.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="9" id="US-6376217-B1-CLM-00009" class="claim">
      <div class="claim-text">9. The fusion protein of <claim-ref idref="US-6376217-B1-CLM-00005">claim 5</claim-ref>, wherein said antigen-binding portion of said antibody is an Fv.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="10" id="US-6376217-B1-CLM-00010" class="claim">
      <div class="claim-text">10. The fusion protein of <claim-ref idref="US-6376217-B1-CLM-00005">claim 5</claim-ref>, wherein said antigen-binding portion of said antibody has a single variable domain.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="11" id="US-6376217-B1-CLM-00011" class="claim">
      <div class="claim-text">11. The fusion protein of <claim-ref idref="US-6376217-B1-CLM-00005">claim 5</claim-ref>, wherein said antibody is a single-chain antibody.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="12" id="US-6376217-B1-CLM-00012" class="claim">
      <div class="claim-text">12. The fusion protein of <claim-ref idref="US-6376217-B1-CLM-00005">claim 5</claim-ref>, wherein said fusion protein is multivalent.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="13" id="US-6376217-B1-CLM-00013" class="claim">
      <div class="claim-text">13. The fusion protein of any one of claims <b>1</b>, <b>2</b> and <b>3</b> wherein said targeting sequence is a hormone.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="14" id="US-6376217-B1-CLM-00014" class="claim">
      <div class="claim-text">14. The fusion protein of any one of claims <b>1</b>, <b>2</b> and <b>3</b>, wherein said targeting sequence is an antibody.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="15" id="US-6376217-B1-CLM-00015" class="claim">
      <div class="claim-text">15. The fusion protein of any one of claims <b>1</b>, <b>2</b> and <b>3</b>, wherein said targeting sequence is a growth factor.</div>
    </div>
  </div> </div></div></div><div class="patent-section patent-description-section"><div class="patent-section-header"><span class="patent-section-title">Description</span></div><div class="patent-text"><div mxw-id="PDES53551874" lang="EN" load-source="patent-office" class="description">
    <p>This is a continuation of U.S. application Ser. No. 09/136,389, filed Aug. 18, 1998, now U.S. Pat. No. 6,146,850, which is a continuation of U.S. application Ser. No. 08/646,360, filed May 13, 1996, now U.S. Pat. No. 5,837,491, which is the U.S. National Phase of PCT/US94/05348, internationally filed May 12, 1994, which is a continuation-in-part of U.S. application Ser. No. 08/064,691, filed May 12, 1993 (now abandoned), which is a continuation-in-part of U.S. application Ser. No. 07/988,430, filed Dec. 9, 1992 (now U.S. Pat. No. 5,416,202), which is a continuation-in-part of U.S. application Ser. No. 07/901,707 filed Jun. 19, 1992 (now U.S. Pat. No. 5,376,546), which is a continuation-in-part of U.S. application Ser. No. 07/787,567, filed Nov. 4, 1991 (now abandoned).</p>
    <heading>FIELD OF THE INVENTION</heading> <p>The present invention generally relates to materials useful as components of cytotoxic therapeutic agents. More particularly, the invention relates to ribosome-inactivating proteins, to analogs of ribosome-inactivating proteins, to polynucleotides encoding such proteins and analogs, some of which are specifically modified for conjugation to targeting molecules, and to gene fusions of polynucleotides encoding ribosome-inactivating proteins to polynucleotides encoding targeting molecules.</p>
    <heading>BACKGROUND</heading> <p>Ribosome-inactivating proteins (RIPs) comprise a class of proteins which is ubiquitous in higher plants. However, such proteins have also been isolated from bacteria. RIPs are potent inhibitors of eukaryotic protein synthesis. The N-glycosidic bond of a specific adenine base is hydrolytically cleaved by RIPs in a highly conserved loop region of the 28S rRNA of eukaryotic ribosomes thereby inactivating translation.</p>
    <p>Plant RIPs have been divided into two types. Stirpe et al., <i>FEBS Lett., </i>195(1,2):1-8 (1986). Type I proteins each consist of a single peptide chain having ribosome-inactivating activity, while Type II proteins each consist of an A-chain, essentially equivalent to a Type I protein, disulfide-linked to a B-chain having cell-binding properties. Gelonin, dodecandrin, tricosanthin, tricokirin, bryodin, Mirabilis antiviral protein (MAP), barley ribosome-inactivating protein (BRIP), pokeweed antiviral proteins (PAPS), saporins, luffins, and momordins are examples of Type I RIPs; whereas Ricin and abrin are examples of Type II RIPs.</p>
    <p>Amino acid sequence information is reported for various ribosome-inactivating proteins. It appears that at least the tertiary structure of RIP active sites is conserved among Type I RIPs, bacterial RIPs, and A-chains of Type II RIPs. In many cases, primary structure homology is also found. Ready et al., <i>J. Biol. Chem., </i>259(24):15252-15256 (1984) and other reports suggest that the two types of RIPs are evolutionarily related.</p>
    <p>Type I plant ribosome-inactivating proteins may be particularly suited for use as components of cytotoxic therapeutic agents. A RIP may be conjugated to a targeting agent which will deliver the RIP to a particular cell type in vivo in order to selectively kill those cells. Typically, the targeting agent (e.g., an antibody) is linked to the toxin by a disulfide bond which is reduced in vivo allowing the protein toxin to separate from the delivery antibody and become active intracellularly. Another strategy for producing targeted cytotoxic proteins is to express a gene encoding a cytotoxic protein fused to a gene encoding a targeting moiety. The resulting protein product is composed of one or more polypeptides containing the cytotoxic protein linked to, for example, at least one chain of an antibody.</p>
    <p>A variety of such gene fusions are discussed in Pastan et al., <i>Science, </i>254;1173-1177 (1991). However, these fusion proteins have been constructed with sequences from diphtheria toxin or <i>Pseudomonas aeruginosa </i>exotoxin A, both of which are ADP-ribosyltransferases of bacterial origin. These protein toxins are reported to intoxicate cells and inhibit protein synthesis by mechanisms which differ from those of the RIPs. Moreover, diphtheria toxin and exotoxin A are structurally different from, and show little amino acid sequence similarity with, RIPs. In general, fusion proteins made with diphtheria toxin or exotoxin A have been immunogenic and toxic in animals, and are produced intracellularly in relatively low yield. Another strategy for producing a cytotoxic agent is to express a gene encoding a RIP fused to a gene encoding a targeting moiety. The resulting protein product is a single polypeptide containing a RIP linked to, for example, at least one chain of an antibody.</p>
    <p>Because some RIPs, such as the Type I RIP gelonin, are primarily available from scarce plant materials, it is desirable to clone the genes encoding the RIPs to enable recombinant production of the proteins. It is also desirable to develop analogs of the natural proteins which may be easily conjugated to targeting molecules while retaining their natural biological activity because most Type I RIPs have no natural sites (i.e. available cysteine residues) for conjugation to targeting agents. Alternatively, it is desirable to develop gene fusion products including Type I RIPs as a toxic moiety and antibody substances as a targeting moiety.</p>
    <p>The present invention also provides novel humanized or human-engineered antibodies and methods for producing such antibodies which may be conjugated or fused to various toxins. Such conjugations or fusions are useful in the treatment of various disease states, including autoimmune diseases and cancer.</p>
    <p>There are several reports relating to replacement of amino acids in a mouse antibody with amino acids normally occurring at the analogous position in the human form of the antibody. see, e.g., Junghaus, et al., <i>Cancer Res., </i>50: 1495-1502 (1990) and other publications which describe genetically-engineered mouse/human chimeric antibodies. Also by genetic engineering techniques, the genetic information from murine hypervariable complementarity determining regions (hereinafter referred to as “CDRs”) may be inserted in place of the DNA encoding the CDRs of a human monoclonal antibody to generate a construct encoding a human antibody with murine CDRS. See, e.g., Jones, et al., <i>Nature, </i>321: 522-525 (1986).</p>
    <p>Protein structure analysis on such “CDR-grafted” antibodies may be used to “add back” murine residues in order to restore lost antigen-binding capability, as described in Queen, et al, <i>Proc. Natl. Acad. Sci. </i>(<i>USA</i>), 86:10029-10033 (1989); Co, et al., <i>Proc. Nat. Acad. Sci. </i>(<i>USA</i>), 88: 2869-2873 (1991). However, a frequent result of CDR-grafting is that the specific binding acitvity of the resulting humanized antibodies may be diminished or completely abolished.</p>
    <p>As demonstrated by the foregoing, there exists a need in the art for cloned genes encoding Type I RIPs, for analogs of Type I RIPs which may be easily conjugated to targeting molecules, and for gene fusion products comprising Type I RIPs, and especially wherein such gene fusions also comprise an humanized antibody portion.</p>
    <heading>SUMMARY OF THE INVENTION</heading> <p>The present invention provides purified and isolated polynucleotides encoding Type I RIPs, Type I RIPs having a cysteine available for disulfide bonding to targeting molecules and fusion products comprising Type I RIPs. Vectors comprising the polynucleotides and host calls transformed with the vectors are also provided.</p>
    <p>A purified and isolated polynucleotide encoding natural sequence gelonin (SEQ ID NO: 11), and a host cell including a vector encoding gelonin of the type deposited as ATCC Accession No. 68721 are provided. Further provided are a purified and isolated polynucleotide encoding natural sequence barley ribosome-inactivating protein and a purified and isolated polynucleotide encoding momordin II.</p>
    <p>Some of the polynucleotides mentioned above encode fusion proteins of the present invention comprising gelonin (SEQ ID NO: 2) or another RIP and an antibody or a fragment comprising an antigen-binding portion thereof. Several alternate forms of fusion proteins comprising gelonin are contemplated herein. For example, the fusion protein may contain a single RIP fused to a monovalent antibody binding moiety, such as a Fab or single chain antibody. Alternatively, multivalent forms of the fusion proteins may be made and may have greater activity than the monovalent forms. In preferred embodiments of the invention, gelonin may be fused to either the carboxy or the amino terminus of the antibody or antigen-binding portion of thereof. Also in a preferred embodiment of the invention, the antibody or fragment thereof comprising an antigen-binding portion may be an he3 antibody, an he3-Fab, an he3 Fd, single-chain antibody, or an he3 kappa fragment. The antibody or antigen-binding portion thereof may be fused to gelonin by means of a linker peptide, preferably a peptide segment of shiga-like toxin as shown in SEQ ID NO: 56 or a peptide segment of Rabbit muscle aldolase as shown in SEQ ID NO: 57 or a human muscle aldolase, an example of which is reported in Izzo, et al., <i>Eur. J. Biochem, </i>174: 569-578 (1988), incorporated by reference herein.</p>
    <p>Analogs of a Type I plant RIP are defined herein as non-naturally occurring polypeptides that share the ribosome-inactivating activity of the natural protein but that differ in amino acid sequence from the natural type I RIP protein in some degree but less than they differ from the amino acid sequences of other Type I plant RIP. Preferred analogs according to the present invention are analogs of Type I plant RIPs each having a cysteine available for disulfide bonding located at a position in its amino acid sequence from the position corresponding to position 251 in SEQ ID NO: 1 to the carboxyl terminal position of the analog. SEQ ID NO: 1 represents the amino acid sequence of ricin A-chain. Other preferred analogs according to the invention are Type I RIPs each having a cysteine available for disulfide bonding at a position in the analog that is on the surface of the protein in its natural conformation and that does not impair native folding or biological activity of the ribosome-inactivating protein. Analogs of bacterial RIPs are also contemplated by the present invention.</p>
    <p>The present invention provides an analog of a Type I ribosome-inactivating protein, which analog has a cysteine available for intermolecular disulfide bonding at an amino acid position corresponding to a position not naturally available for intermolecular disulfide bonding in the Type I ribosome-inactivating protein and which cysteine is located at a position in the amino acid sequence of the analog corresponding to position 259 in SEQ ID NO: 1 or at a position in the amino acid sequence in the analog corresponding to a position from position 251 in SEQ ID NO: 1 to the carboxyl terminal position of the analog.</p>
    <p>An analog according to the present invention may be an analog of gelonin. In an analog of gelonin according to the present invention, the cysteine may be at a position in the analog from position 244 to the carboxyl terminal position of the analog, more preferably at a position in the analog from position 247 to the carboxyl terminal position of the analog, and most preferably at position 244, at position 247, or at position 248 of the amino acid sequence of the analog. In these analogs, it is preferred that the gelonin cysteine residues at positions 44 and 50 be replaced with non-cysteine residues such as alanine.</p>
    <p>An analog according to the present invention may be an analog of barley ribosome-inactivating protein. Preferably, a cysteine in such an analog is at a position in the analog from position 256 to the carboxyl terminal position, and more preferably the cysteine is at a position in the amino acid sequence of the analog from position 260 to the carboxyl terminal position of the analog. Most preferably, in these regions, the cysteine is at position 256, at position 270, or at position 277 of the amino acid sequence of the analog.</p>
    <p>An analog according to the present invention may be an analog of momordin II.</p>
    <p>Analogs according to the present invention may have a cysteine in the amino acid sequence of the analog at a position which corresponds to a position within one amino acid of position 259 of SEQ ID NO: 1. Such an analog may be an analog of gelonin, of barley ribosome-inactivating protein, or of momordin II.</p>
    <p>The present invention also provides a polynucleotide encoding an analog of a Type I ribosome-inactivating protein, which analog has a cysteine available for intermolecular disulfide bonding at an amino acid position corresponding to a position not naturally available for intermolecular disulfide bonding in the Type I ribosome-inactivating protein, and which cysteine is located at a position in the amino acid sequence of the analog from the position corresponding to position 251 in SEQ ID NO: 1 to the carboxyl terminal position of the analog. The polynucleotide may encode an analog of gelonin, preferably an analog wherein the cysteine is at a position in the amino acid sequence of the analog from position 244 to the carboxyl terminal position of the analog, more preferably wherein the cysteine is at a position in the analog from position 247 to the carboxyl terminal position of the analog, and most preferably the cysteine is at position 244, at position 247 or at position 248 of the amino acid sequence of the analog. It is preferred that a polynucleotide according to the present invention encode a gelonin analog wherein the native gelonin cysteine residues at positions 44 and 50 are replaced with non-cysteine residues, such as alanine.</p>
    <p>A polynucleotide according to the present invention may encode an analog of barley ribosome-inactivating protein, preferably an analog wherein the cysteine is at a position in the analog from position 256 to the carboxyl terminal position of the analog, more preferably wherein the cysteine is at a position in the analog from position 260 to the carboxyl terminal position of the analog, and most preferably wherein the cysteine is at position 256, at position 270 or at position 277 of the amino acid sequence of the analog.</p>
    <p>A polynucleotide according to the present invention may encode an analog of mormordin II.</p>
    <p>The present invention provides a vector including a polynucleotide encoding an analog of a Type I ribosome-inactivating protein, which analog has a cysteine available for intermolecular disulfide bonding at a amino acid position corresponding to a position not naturally available for intermolecular disulfide bonding in the Type I ribosome-inactivating protein and which cysteine is located at a position in the amino acid sequence of the analog from the position corresponding to position 251 in SEQ ID NO: 1 to the carboxyl terminal position of the analog.</p>
    <p>The present invention further provides a host cell including a DNA vector encoding an analog of a Type I ribosome-inactivating protein, which analog has a cysteine available for intermolecular disulfide bonding at an amino acid position corresponding to a position not naturally available for intermolecular disulfide bonding in the Type I ribosome-inactivating protein and which cysteine is located at a position in the amino acid sequence of the analog from the position corresponding to position 251 in SEQ ID NO: 1 to the carboxyl terminal position of the analog. In such a host cell the vector may encode an analog of gelonin, especially an analog wherein the cysteine is at position 247 of the amino acid sequence of the analog, such as in the host cell deposited as ATCC Accession No. 69009.</p>
    <p>A host cell according to the present invention may include a vector encoding barley ribosome-inactivating protein, especially preferred is a host cell containing a BRIP analog wherein the cysteine is at position 277, such as in the host cell deposited on Oct. 2, 1991 with the American Type Culture Collection, 12301 Parklawn Drive, Rockville, Md. 20852 as ATCC Accession No. 68722. Particularly preferred are prokaryotic host cells because such cells may be less sensitive to the action or RIPs as compared to eukaryotic cells.</p>
    <p>The present invention also provides an agent toxic to a cell including an analog of a Type I ribosome-inactivating protein linked by a disulfide bond through a cysteine to a molecule which specifically binds to the cell, which cysteine is at an amino acid position in the analog corresponding to a position not naturally available for intermolecular disulfide bonding in the Type I ribosome-inactivating protein and which cysteine is located in the amino acid sequence of the analog from the position corresponding to position 251 in SEQ ID NO: 1 to the carboxyl terminal position of the analog. The agent may include an analog of gelonin, preferably an analog wherein the cysteine is at a position in the analog from position 247 to the carboxyl terminal position of the analog, and more preferably wherein the cysteine is at position 247 or 248 of the amino acid sequence of analog. An agent including an analog wherein the native gelonin cysteine residues at positions 44 and 50 are replaced with non-cysteine residues, such as alanine is preferred.</p>
    <p>An agent according to the present invention may include an analog of barley ribosome-inactivating protein, preferably an analog wherein the cysteine is at a position in the analog from position 260 to the carboxyl terminal position of the analog, more preferably wherein the cysteine is at a position in the analog from position 270 to the carboxyl terminal position of the analog, and most preferably wherein the cysteine is at position 256, at position 270 or at position 277 of the amino acid sequence of the analog.</p>
    <p>An agent according to the present invention may include an analog of momordin II.</p>
    <p>The present invention provides an agent wherein the Type I ribosome-inactivating protein is linked to an antibody, particularly to an H65 antibody or to an antibody fragment, more particularly to an antibody fragment selected from the group consisting of chimeric and human engineered antibody fragments, and most particularly to a Fab antibody fragment, a Fab′ antibody fragment or a F(ab′)<sub>2 </sub>antibody fragment. It is highly preferred that an agent according to the present invention include a chimeric or human engineered antibody fragment selected from the group consisting of a Fab antibody fragment, a Fab′ antibody fragment and a F(ab′)<sub>2 </sub>antibody fragment.</p>
    <p>A method according to the present invention for preparing an analog of a Type I ribosome-inactivating protein includes the step of expressing in a suitable host cell a polynucleotide encoding a Type I ribosome-inactivating fusion protein or type I RIP (especially gelonin) having a cysteine available for intermolecular disulfide bonding substituted (e.g., by site-directed mutagenesis of the natural DNA sequence encoding the RIP or by chemical synthesis of a DNA sequence encoding the RIP analog) at an amino acid position corresponding to a position not naturally available for intermolecular disulfide bonding in the Type I ribosome-inactivating protein, which cysteine is located at a position in the amino acid sequence of the analog from the position corresponding to position 251 in SEQ ID NO: 1 to the carboxyl terminal position of the analog.</p>
    <p>A product according to the present invention may be a product of a method including the step of expressing in a suitable host cell a polynucleotide encoding a Type I ribosome-inactivating protein having a cysteine available for intermolecular disulfide bonding substituted at an amino acid position corresponding to a position not naturally available for intermolecular disulfide bonding in the Type I ribosome-inactivating protein, which cysteine is located at a position in the amino acid sequence of the analog from the position corresponding to position 251 in SEQ ID NO: 1 to the carboxyl terminal position of the analog.</p>
    <p>The present invention provides a method for preparing an agent toxic to a cell including the step of linking an analog of a Type I ribosome-inactivating protein through a cysteine to a molecule which specifically binds to the cell, which analog has the cysteine at an amino acid position corresponding to a position not naturally available for intermolecular disulfide bonding in the Type I ribosome-inactivating protein and which cysteine is located at a position in the amino acid sequence of the analog from the position corresponding to position 251 in SEQ ID NO: 1 to the carboxyl terminal position of the analog.</p>
    <p>According to the present invention, a method for treating a disease in which elimination of particular cells is a goal may include the step of administering to a patient having the disease a therapeutically effective amount of an agent toxic to the cells including a type I RIP (especially gelonin fused to or an analog of a Type I ribosome-inactivating protein linked through a cysteine to a molecule which specifically binds to the cell, the analog having the cysteine at an amino acid position corresponding to a position not naturally available for intermolecular disulfide bonding in the Type I ribosome-inactivating protein and the cysteine being located at a position in the amino acid sequence of the analog from the position corresponding to position 251 in SEQ ID NO: 1 to the carboxyl terminal position of the analog.</p>
    <p>Useful target cells for immunotoxins of the present invention include, but are not limited to, cells which are pathogenic, such as cancer cells, autoimmune cells, and virally-infected cells. Such pathogenic cells may be targeted by antibodies or other targeting agents of the invention which are joined, either by genetic engineering techniques or by chemical cross-linking, to an RIP. Specifically useful targets include tumor-associated antigens (e.g., on cancer cells), cell differentiation markers (e.g., on autoimmune cells), parasite-specific antigens, bacteria-specific antigens, and virus-specific antigens.</p>
    <p>The present invention also provides an analog of a Type I ribosome-inactivating protein, wherein the analog has a cysteine available for intermolecular disulfide bonding located at an amino acid position corresponding to a position not naturally available for intermolecular disulfide bonding in the Type I ribosome-inactivating protein and corresponding to a position on the surface of ricin A-chain in its natural conformation, and wherein the analog retains the ribosome-inactivating activity of the Type I ribosome-inactivating protein.</p>
    <p>Such a fusion protein or an analog may be a fusion protein or an analog wherein the Type I ribosome inactivating protein is gelonin, and the analog is preferably an analog of gelonin wherein the cysteine is at position 10 of the amino acid sequence of the analog as encoded in a vector in a host cell deposited with the American Type Culture Collection, 12301 Parklawn Drive, Rockville, Md. 20852 as ATCC Accession No. 69008 on Jun. 9, 1992. Other such gelonin analogs include those wherein the cysteine is at a position 60, 103, 146, 184 or 215 in the amino acid sequence of the gelonin analog. It is preferred that the gelonin cysteine residues at positions 44 and 50 be replaced with non-cysteine residues such as alanine in these analogs.</p>
    <p>The present invention further provides an analog of a Type I ribosome-inactivating protein wherein the analog includes only a single cysteine. Such an analog may be an analog of gelonin and is preferably an analog wherein the single cysteine is at position 10, position 44, position 50 or position 247 in the amino acid sequence of the analog, but the cysteine may be located at other positions defined by the invention as well.</p>
    <p>The present invention provides a polynucleotide encoding an analog of a Type I ribosome-inactivating protein, wherein the analog has a cysteine available for intermolecular disulfide bonding located at an amino acid position corresponding to a position not naturally available for intermolecular disulfide bonding in the Type I ribosome-inactivating protein and corresponding to a position on the surface of ricin A-chain in its natural conformation, and wherein the analog retains ribosome-inactivating activity of the Type I ribosome-inactivating protein.</p>
    <p>According to the present invention, a method for preparing an analog of a Type I ribosome-inactivating protein may include the step of expressing in suitable host cell a polynucleotide encoding a Type I ribosome-inactivating protein having a cysteine available for intermolecular disulfide bonding substituted at an amino acid position corresponding to a position not naturally available for disulfide bonding in the Type I ribosome-inactivating protein, the cysteine is located at a position corresponding to an amino acid position on the surface of ricin A-chain in its natural conformation and which analog retains ribosome-inactivating activity of the Type I ribosome-inactivating protein.</p>
    <p>The present invention provides an agent toxic to a cell including an analog of a Type I ribosome-inactivating protein linked by a disulfide bond through a cysteine to a molecule which specifically binds to the cell, wherein the analog has a cysteine available for intermolecular disulfide bonding located at an amino acid position corresponding to a position not naturally available for intermolecular disulfide bonding in the Type I ribosome-inactivating protein and corresponding to a position on the surface of ricin A-chain in its natural conformation, and wherein the analog retains ribosome-inactivating activity of the Type I ribosome-inactivating protein.</p>
    <p>A method according to the present invention for preparing an agent toxic to a cell may include the step of linking an analog of a Type I ribosome-inactivating protein through a cysteine to a molecule which specifically binds to the cell, which analog has a cysteine available for intermolecular disulfide bonding located at an amino acid position corresponding to a position not naturally available for intermolecular disulfide bonding in the Type I ribosome-inactivating protein and corresponding to a position on the surface of ricin A-chain in its natural conformation, and which analog retains ribosome-inactivating activity of the Type I ribosome-inactivating protein.</p>
    <p>A method according to the present invention for treating a disease in which elimination of particular cells is a goal includes the step of administering to a patient having the disease a therapeutically effective amount of an agent toxic to the cells wherein the agent includes a type I RIP fused to or an analog of a Type I ribosome-inactivating protein linked by a disulfide bond through a cysteine to a molecule which specifically binds to the cell, which analog has a cysteine available for intermolecular disulfide bonding located at an amino acid position corresponding to a position not naturally available for intermolecular disulfide bonding in the Type I ribosome-inactivating protein and corresponding to a position on the surface of ricin A-chain in its natural conformation, and which analog retains ribosome-inactivating activity of the Type I ribosome-inactivating protein.</p>
    <p>The RIP analogs of the invention are particularly suited for use as components of cytotoxic therapeutic agents. Cytotoxic agents according to the present invention may be used in viva to selectively eliminate any cell type to which the RIP component is targeted by the specific binding capacity of the second component. To form cytotoxic agents, RIP analogs may be conjugated to monoclonal antibodies, including chimeric and CDR-grafted antibodies, and antibody domains/fragments (e.g., Fab, Fab′, F(ab′)<sub>2</sub>, single chain antibodies, and Fv or single variable domains). Analogs of RIPs conjugated to monoclonal antibodies genetically engineered to include free cysteine residues are also within the scope of the present invention. Examples of Fab′ and F(ab′)<sub>2 </sub>fragments useful in the present invention are described in co-pending, co-owned U.S. patent application Ser. No. 07/714,175, filed Jun. 14, 1991 and in International Publication No. WO 89/00999 published on Feb. 9, 1989, which are incorporated by reference herein.</p>
    <p>The RIP analogs of the invention may preferably be conjugated or fused to humanized or human engineered antibodies, such as the he3 antibody described herein. Such humanized antibodies may be constructed from mouse antibody variable domains, such as the mouse antibody H65 (SEQ ID NOS: 123 and 124). Specifically RIP analogs according to the present invention may be conjugated or fused to he3 human-engineered antibody light and heavy chain variable regions (SEQ ID NO: 125 and 126, respectively) or fragments thereof. A cell line producing an intact he3 immunoglobulin was deposited as ATCC Accession No. HB11206 with the American Type Culture Collection, 12301 Parklawn Drive, Rockville, Md. 20852.</p>
    <p>RIPs according to the present invention may also be conjugated to targeting agents other than antibodies, for example, lectins which bind to cells having particular surface carbohydrates, hormones, lymphokines, growth factors or other polypeptides which bind specifically to cells having particular receptors. Immunoconjugates including RIPs may be described as immunotoxins. An immunotoxin may also consist of a fusion protein rather than an immunoconjugate.</p>
    <p>The present invention provides gene fusions of an antigen-binding portion of an antibody (e.g., an antibody light chain or truncated heavy chain, or a single chain antibody) or any targeting agent listed in the foregoing paragraph, linked to a Type I RIP. Preferably, the antigen-binding portion of an antibody or fragment thereof comprises a single chain antibody, a Fab fragment, or a F(ab′)<sub>2 </sub>fragment. Active antibodies generally have a conserved three-dimensional folding pattern and it is expected that any antibody which maintains that folding pattern will retain binding specificity. Such antibodies are expected to retain target enzymatic activity when incorporated into a fusion protein according to the present invention.</p>
    <p>It is sometimes necessary that immunotoxins comprising cytotoxic components, such as RIPs, be attached to targeting agents via cleavable linkers (i.e., disulfides, acid-sensitive linkers, and the like) in order to allow intracellular release of the cytotoxic component. Such intracellular release allows the cytotoxic component to function unhindered by possible negative kinetic or steric effects of the attached antibody. Accordingly, gene fusions of the present invention may comprise a RIP gene fused, via a DNA segment encoding a linker protein as described above, to either the 5′ or the 3′ end of a gene encoding an antibody. If a linker is used, it preferably encodes a polypeptide which contains two cysteine residues participating in a disulfide bond and forming a loop which includes a protease-sensitive amino acid sequence (e.g., a segment of <i>E. coli </i>shiga-like toxin as in SEQ ID NO: 56) or a segment which contains several cathepsin cleavage sites (e.g., a segment of rabbit muscle aldolase as in SEQ ID NO: 57; a segment of human muscle aldolase; or a synthetic peptide including a cathepsin cleavage sites such as in SEQ ID NOs: 141 or 142). A linker comprising cathepsin cleavage sites as exemplified herein comprises the C-terminal 20 amino acids of RMA. However, that sequence differs by only one amino acid from human muscle aldolase and it is contemplated that muscle aldolase from human or other sources may be used as a linker in the manner described below. The Type I RIP portion of the fused genes preferably encodes gelonin, BRIP or momordin II. Also preferably, the antibody portion of the fused genes comprises sequences encoding one of the chains of an antibody Fab fragment (i.e., kappa or Fd) and the fused gene is co-expressed in a host cell with the other Fab gene, or the antibody portion comprises sequences encoding a single chain antibody.</p>
    <p>Alternatively, since fusion proteins of the present invention may be of low (approximately 55 kDa) molecular weight while maintaining full enzymatic activity, such fusions may be constructed without a linker and still possess cytotoxic activity. Such low-molecular weight fusions are not as susceptible to kinetic and steric hinderance as are the larger fusions, such as fusions involving IgG molecules. Therefore, cleavage of the RIP away from the fusion may not be necessary to facilitate activity of the RIP.</p>
    <p>The present invention also provides a method for purifying a protein or immunotoxin comprising a ribosome-inactivating protein and a portion of an antibody including the steps of passing a solution containing the protein through an anion exchange column; applying the flow-through to a protein G column; and eluting the protein from the protein G column. The method may further comprise the steps of introducing the flow-through of the anion exchange column into a cation exchange column; exposing the cation exchange column to an eluent effective to elute said protein; and then applying the eluted protein to a protein G column, rather than applying the anion exchange column flow-through directly to a protein G column.</p>
    <p>Immunotoxins according to the present invention, including immunoconjugates and fusion proteins (immunofusions), are suited for treatment of diseases where the elimination of a particular cell type is a goal, such as autoimmune disease, cancer, and graft-versus-host disease. The immunotoxins are also suited for use in causing immunosuppression and in treatment of infections by viruses such as the Human Immunodeficiency Virus.</p>
    <p>Specifically illustrating polynucleotide sequences according to the present invention are the inserts in the plasmid pING3731 in <i>E. coli </i>MC1061 (designated strain G274) and in the plasmid pING3803 in <i>E. coli </i>E104 (designated strain G275), both deposited with the American Type Culture Collection (ATCC), 12301 Parklawn Drive, Rockville, Md., on Oct. 2, 1991, and assigned ATCC Accession Nos. 68721 and 68722, respectively. Additional polynucleotide sequences illustrating the invention are the inserts in the plasmid pING3746 in <i>E. coli </i>E104 (designated strain G277) and in the plasmid pING3737 in <i>E. coli </i>E104 (designated strain G276), which were both deposited with the ATCC on Jun. 9, 1992, and were respectively assigned Accession Nos. 69008 and 69009. Still other polynucleotide sequences illustrating the invention are the inserts in the plasmid pING3747 in <i>E. coli </i>E104 (designated strain G278), in the plasmid pING3754 in <i>E. coli </i>E104 (designated strain G279), in the plasmid pING3758 in <i>E. coli </i>E104 (designated strain G280) and in the plasmid pING3759 in <i>E. coli </i>E104 (designated strain G281), which plasmids were all deposited with the ATCC on Oct. 27, 1992 and were assigned ATCC Accession Nos. 69101, 69102, 69103 and 69104, respectively.</p>
    <p>As noted above, RIPs may preferably be conjugated or fused to humanized or human-engineered antibodies, such as he3. Thus, the present invention also provides novel proteins comprising a humanized antibody variable domain which is specifically reactive with an human CD5 cell differentiation marker. Specifically, the present invention provides proteins comprising the he3 light and heavy chain variable regions as shown in SEQ ID NOS:125 or 126, respectively. DNA encoding certain he3 proteins is shown in SEQ ID NOS:72 and 71.</p>
    <p>In a preferred embodiment of the present invention, the protein comprising an humanized antibody variable region is an intact he3 immunoglobulin deposited as ATCC HB 11206.</p>
    <p>Also in a preferred embodiment of the invention, the protein comprising a humanized antibody variable region is a Fab or F(ab′)<sub>2 </sub>or Fab fragment.</p>
    <p>Proteins according to the present invention may be made by methods taught herein and in co-pending, co-owned U.S. patent application Ser. No. 07/808,464 by Studnicka et al. incorporated by reference herein; and modified antibody variable domains made by such methods may be used in therapeutic administration to humans either alone or as part of an immunoconjugate as taught in co-owned, co-pending U.S. patent application Ser. No. 07/787,567 by Better et al.</p>
    <p>The present invention also provides methods for preparing a modified antibody variable domain useful in preparing immunotoxins and immunofusions by determining the amino acids of a subject antibody variable domain which may be modified without diminishing the native affinity of the domain for antigen while reducing its immunogenicity with respect to a heterologous species. As used herein, the term “subject antibody variable domain” refers to the antibody upon which determinations are made. The method includes the following steps: determining the amino acid sequence of a subject light chain and a subject heavy chain of a subject antibody variable domain to be modified; aligning by homology the subject light and heavy chains with a plurality of human light and heavy chain amino acid sequences; identifying the amino acids in the subject light and heavy chain sequences which are least likely to diminish the native affinity of the subject variable domain for antigen while, at the same time, reducing its immunogenicity by selecting each amino acid which is not in an interface region of the subject antibody variable domain and which is not in a complementarity-determining region or in an antigen-binding region of the subject antibody variable domain, but which amino acid is in a position exposed to a solvent containing the antibody; changing each residue identified above which aligns with a highly or a moderately conserved residue in the plurality of human light and heavy chain amino acid sequences if said identified amino acid is different from the amino acid in the plurality.</p>
    <p>Another group of sequences, such as those in FIGS. 1A and 1B may be used to determine an alignment from which the skilled artisan may determine appropriate changes to make.</p>
    <p>The present invention provides a further method wherein the plurality of human light and heavy chain amino acid sequences is selected from the human consensus sequences in FIGS. 10A and 10B.</p>
    <p>In general, human engineering according to the above methods may be used to treat various diseases against which monoclonal antibodies generally may be effective. However, humanized antibodies possess the additional advantage of reducing the immunogenic response in the treated patient.</p>
    <p>Additional aspects and applications of the present invention will become apparent to the skilled artisan upon consideration of the detailed description of the invention which follows.</p>
    <heading>BRIEF DESCRIPTION OF THE DRAWINGS</heading> <p>FIG. 1 is a computer-generated alignment of the amino acid sequence of the ricin A-chain (RTA) (SEQ ID NO: 1) with the amino acid sequence of the Type I ribosome-inactivating protein gelonin (SEQ ID NO: 2), wherein starred positions indicate amino acids invariant among the ricin A-chain and the Type I RIPs;</p>
    <p>FIG. 2 is a computer-generated alignment of the amino acid sequence of the ricin A-chain (SEQ ID NO: 1) with the amino acid sequence of the Type I ribosome-inactivating protein BRIP (SEQ ID NO: 3), wherein starred positions indicate amino acids invariant among the ricin A-chain and the Type I RIPs;</p>
    <p>FIG. 3 is a computer-generated alignment of the amino acid sequence of the ricin A-chain (SEQ ID NO: 1) with the amino acid sequence of the Type I ribosome-inactivating protein momordin II (MOMOII) (SEQ ID NO: 4), wherein starred positions indicate amino acids invariant among the ricin A-chain and the Type I RIPs;</p>
    <p>FIG. 4 is a computer-generated alignment of the amino acid sequence of the ricin A-chain (SEQ ID NO: 1) with the amino acid sequence of the Type I ribosome-inactivating protein luffin (SEQ ID NO: 5), wherein starred positions indicate amino acids invariant among the ricin A-chain and the Type I RIPs;</p>
    <p>FIG. 5 is a computer-generated alignment of the amino acid sequence of the ricin A-chain (SEQ ID NO: 1) with the amino acid sequence of the Type I ribosome-inactivating protein αtrichosanthin (TRICHO) (SEQ ID NO: 6), wherein starred positions indicate amino acids invariant among the ricin A-chain and the Type I RIPs;</p>
    <p>FIG. 6 is a computer-generated alignment of the amino acid sequence of the ricin A-chain (SEQ ID NO: 1) with the amino acid sequence of the Type I ribosome-inactivating protein momordin I (MOMOI) (SEQ ID NO: 7), wherein starred positions indicate amino acids invariant among the ricin A-chain and the Type I RIPs;</p>
    <p>FIG. 7 is a computer-generated alignment of the amino acid sequence of the ricin A-chain (SEQ ID NO: 1) with the amino acid sequence of the Type I ribosome-inactivating protein Mirabilis anti-viral protein (MAP) (SEQ ID NO: 8), wherein starred positions indicate amino acids invariant among the ricin A-chain and the Type I RIPs;</p>
    <p>FIG. 8 is a computer-generated alignment of the amino acid sequence of the ricin A-chain (SEQ ID NO: 1) with the amino acid sequence of the Type I ribosome-inactivating protein pokeweed antiviral protein from seeds (PAPS) (SEQ ID NO: 9), wherein starred positions indicate amino acids invariant among the ricin A-chain and the Type I RIPs;</p>
    <p>FIG. 9 is a computer-generated alignment of the amino acid sequence of the ricin A-chain (SEQ ID NO: 1) with the amino acid sequence of the Type I ribosomeinactivating protein saporin 6 (SAP6) (SEQ ID NO: 10), wherein starred positions indicate amino acids invariant among the ricin A-chain and the Type I RIPs;</p>
    <p>FIGS. 10A and 10B are alignments of the consensus amino acid sequences for the subgroups of light [hK1 (SEQ ID NO: 149) (human kappa light chain subgroup 1), hK3 (SEQ ID NO: 150) (human kappa light chain subgroup 3), hK2 (SEQ ID NO: 151) (human kappa light chain subgroup 2), hL1 (SEQ ID NO: 152) (human lambda light chain subgroup 1), hL2 (SEQ ID NO: 153) (human lambda light chain subgroup 2), hL3 (SEQ ID NO:154) (human lambda light chain subgroup 3), hL6 (SEQ ID NO: 155) (human lambda light chain subgroup 6), hK4 (SEQ ID NO: 156) (human kappa light chain subgroup 4), hL4 (SEQ ID NO: 157) (human lambda light chain subgroup 4) and hL5 (SEQ ID NO: 158) (human lambda light chain subgroup 5)] and heavy chains [hH3 (SEQ ID NO: 159) (human heavy chain subgroup 3), hH1 (SEQ ID NO: 160) (human heavy chain subgroup 1) and hH2 (SEQ ID NO: 161) (human heavy chain subgroup 2)], respectively, of human antibody variable domains;</p>
    <p>FIGS. 11A and 11B set out the nucleotide sequences of the oligonucleotides utilized in the construction of the genes encoding modified V/J-regions of the light and heavy chains of the H65 mouse monoclonal antibody variable domain sequence $H65K-1:SEQ ID No. 117; HUH-K1:SEQ ID No. 141; HUH-K2:SEQ ID No. 142; HUH-K3:SEQ ID No. 143; HUH-K4:SEQ ID No. 121; HUH-K5:SEQ ID No. 122; HUH-G1:SEQ ID No. 144; HUH-G2:SEQ ID No. 145; HUH-G3:SEQ ID No. 137; HUH-G4:SEQ ID No. 138; HUH-G5:SEQ ID No. 139; HUH-G6:SEQ ID No. 140; H65G-2S:SEQ ID No. 146; H65-G2:SEQ ID No. 85; H65K-2S:SEQ ID No. 116; JK1-HindIII:SEQ ID No. 87; and</p>
    <p>FIGS. 12A and 12B are alignments of human light chain consensus hK1 (SEQ ID No. 149) and heavy chain consensus hH1 (SEQ ID No. 160) with the light and heavy chain sequences, respectively, of the variable domain of human antibody EU (SEQ ID Nos. 162 and 166), human antibody TAC (SEQ ID Nos. 163 and 167), human antibody TAC modified according to the present invention (prop) (SEQ ID Nos. 164 and 168) and human antibody TAC modified according to a different method (Que) (SEQ ID Nos. 165 and 169).</p>
    <heading>DETAILED DESCRIPTION</heading> <p>Nucleotide sequences of genes encoding three plant Type I RIPs and expression vectors containing the genes are provided by the present invention. A first plant RIP, gelonin, is produced by seeds of <i>Gelonium multiflorum, </i>a plant of the Euphorbiaceae family native to the tropical forests of eastern Asia, while a second plant RIP, BRIP, is synthesized by the common cereal grain barley. Momordin II, a third plant RIP, is produced in <i>Momordica balsamina </i>seeds. Analogs of BRIP are also provided by the present invention. The analogs were genetically engineered to include a cysteine free to participate in a intermolecular disulfide bond and were conjugated to antibody molecules without non-specific chemical derivatization of the RIP with crosslinking agents.</p>
    <p>Type I RIP analogs of the present invention offer distinct advantages over the natural proteins for use as components of immunotoxins. Chemical treatment to introduce free sulfhydryl groups in the natural proteins lacking free cysteines typically involves the non-selective modification of amino acid side chains. This non-selectivity often results in antibodies conjugated to different sites on different RIP molecules (i.e., a heterogeneous population of conjugates) and also in a decrease in RIP activity if antibodies are conjugated in or near important regions of the RIP (e.g., the active site or regions involved in translocation across cell membranes). In contrast, RIP analogs according to the present invention may be conjugated to a single antibody through a disulfide bond to a specific residue of the analog resulting in reduced batch to batch variation of the immunoconjugates and, in some cases, immunoconjugates with enhanced properties (e.g., greater cytotoxicity or solubility).</p>
    <p>Type I plant RIPs, as well as bacterial RIPs such as shiga and shiga-like toxin A-chains, are homologous to the ricin A-chain and are useful in the present invention.</p>
    <p>Type I RIPs may be defined and sites for substitution of a cysteine in a RIP may be identified by comparing the primary amino acid sequence of the RIP to the natural ricin A-chain amino acid sequence, the tertiary structure of which has been described in Katzin et al., <i>Proteins, </i>10:251-259 (1991), which is incorporated by reference herein.</p>
    <p>Amino acid sequence alignment defines Type I RIPs in that the ricin A-chain and the Type I plant RIPs have nine invariant amino acids in common. Based on the ricin sequence the invariant amino acids are tyrosine<sub>21</sub>, arginine<sub>29</sub>, tyrosine<sub>80</sub>, tyrosine<sub>123</sub>, leucine<sub>144</sub>, glutamic acid<sub>177</sub>, alanine<sub>178</sub>, arginine<sub>180</sub>, and tryptophan<sub>211</sub>. The ricin A-chain may be used as a model for the three-dimensional structure of Type I RIPs. A protein lacking a cysteine available for conjugation while having ribosome-inactivating activity and having the nine invariant amino acids when its primary sequence is compared to the primary sequence of the ricin A-chain [according to the alignment algorithm of Myers et al., <i>CABIOS COMMUNICATIONS, </i>4(1):11-17 (1988), implemented by the PC/GENE program PALIGN (Intelligenetics, Inc., Mountain View, Calif.) and utilizing the Dayhoff Mutation Data Matrix (MDM-78) as described in Schwartz et al., pp. 353-358 in <i>Atlas of Protein Sequence and Structure, </i>5 Supp. 3, National Biomedical Research Foundation, Washington, D.C. (1978)] is defined as a Type I RIP herein and is expected to be useful in the present invention. “Corresponding” refers herein to amino acid positions which align when two amino acid sequences are compared by the strategy of Myers et al., supra.</p>
    <p>The primary amino acid sequences of the Type I RIPs:gelonin, BRIP, momordin II, luffin [see Islam et al., <i>Agricultural Biological Chem., </i>54(5):1343-1345 (199)], αtrichosanthin [see Chow et al., <i>J. Biol. Chem., </i>265:8670-8674 (1990)], momordin I [see Ho et al., BBA, 1088:311-314 (1991)], Mirabilis anti-viral protein [see Habuka et al., <i>J. Biol. Chem., </i>264(12):6629-6637 (1989)], pokeweed antiviral protein isolated from seeds [see Kung et al., <i>Agric. Biol. Chem., </i>54(12):3301-3318 (1990)] and saporin [see Benatti et al., <i>Eur. J. Biochem., </i>183:465-470 (1989)] are individually aligned with the primary sequence of the ricin A-chain [see Halling et al., <i>Nucleic Acids Res., </i>13:8019-8033 (1985)] in FIGS. 1-9, respectively, according to the algorithm of Myers et al., supra, as specified above.</p>
    <p>FIGS. 1-9 may be utilized to predict the amino acid positions of the Type I RIPs where cysteine residues may be substituted. Preferred amino acids for cysteine substitution are on the surface of the molecule and include any solvent accessible amino acids which will not interfere with proper folding of the protein if replaced with a cysteine. A region of the ricin A-chain comprising such amino acids is the carboxyl terminal region. Amino acids that should be avoided for replacement are those critical for proper protein folding, such as proline, and those that are solvent inaccessible. Also to be avoided are the nine amino acids invariant among RIPs, and the amino acids in or near regions comprising the active site of ricin A-chain as depicted in FIG. 6 of Katzin et al., supra.</p>
    <p>Therefore, a preferred region of substitution for Type I RIPs is their carboxyl terminal region which is solvent accessible and corresponds to the carboxyl terminal region where Type II RIP A-chains and B-chains are naturally linked by a disulfide bond. As shown in the examples, a cysteine may be substituted in positions in the amino acid sequence of a Type I RIP from the position corresponding to position 251 in SEQ ID NO: 1 to the carboxyl terminal position of said Type I RIP, resulting in RIP analogs which retain enzymatic activity and gain disulfide cross-linking capability. One preferred cysteine substitution position is near the position which corresponds to the cysteine at position 259 in the ricin A-chain.</p>
    <p>For purposes of the present invention, immunotoxins comprise a class of compounds of which toxin-antibody fusions and immunoconjugates are examples. Immunotoxins are particularly suited for use in treatment of human autoimmune diseases and in the treatment of diseases in which depletion of a particular cell type is a goal, such as cancer. For example, treatment of autoimmune diseases with immunotoxins is described in International Publication No. WO89/06968 published Aug. 10, 1989, which is incorporated by reference herein.</p>
    <p>In any treatment regimen, the immunotoxins may be administered to a patient either singly or in a cocktail containing two or more immunotoxins, other therapeutic agents, compositions, or the like, including, but not limited to, immunosuppressive agents, tolerance-inducing agents, potentiators and side-effect relieving agents. Particularly preferred are immunosuppressive agents useful in suppressing allergic reactions of a host. Preferred immunosuppressive agents include prednisone, prednisolone, DECADRON (Merck, Sharp &amp; Dohme, West Point, Pa.), cyclophosphamide, cyclosporine, 6-mercaptopurine, methotrexate, azathioprine and i.v. gamma globulin or their combination. Preferred potentiators include monensin, ammonium chloride, perhexiline, verapamil, amantadine and chloroquine. All of these agents are administered in generally-accepted efficacious dose ranges such as those disclosed in the <i>Physician's Desk Reference, </i>41st Ed., Publisher Edward R. Barnhart, New Jersey (1987). Patent Cooperation Treaty (PCT) U.S. patent application WO 89/069767 published on Aug. 10, 1989, discloses administration of an immunotoxin as an immunosuppressive agent and is incorporated by reference herein.</p>
    <p>Immunotoxins of the present invention may be formulated into either an injectable or topical preparation. Parenteral formulations are known and are suitable for use in the invention, preferably for intramuscular or intravenous administration. The formulations containing therapeutically-effective amounts of immunotoxins are either sterile liquid solutions, liquid suspensions, or lyophilized versions, and optionally contain stabilizers or excipients. Lyophilized compositions are reconstituted with suitable diluents, e.g., water for injection, saline, 0.3% glycine and the like, at a level of about from 0.01 mg/kg of host body weight to 10 mg/kg where the biological activity is less than or equal to 20 ng/ml when measured in a reticulocyte lysate assay. Typically, the pharmaceutical compositions containing immunotoxins of the present invention are administered in a therapeutically effective dose in a range of from about 0.01 mg/kg to about 5 mg/kg of the patient. A preferred, therapeutically effective dose of the pharmaceutical composition containing immunotoxins of the invention is in a range of from about 0.01 mg/kg to about 0.5 mg/kg body weight of the patient administered over several days to two weeks by daily intravenous infusion, each given over a one hour period, in a sequential patient dose-escalation regimen.</p>
    <p>Immunotoxin compositions according to the invention may be formulated into topical preparations for local therapy by including a therapeutically effective concentration of immunotoxin in a dermatological vehicle. The amount of immunotoxin to be administered, and the immunotoxin concentration in the topical formulations, depend upon the vehicle selected, the clinical condition of the patient, the systemic toxicity and the stability of the immunotoxin in the formulation. Thus, a physician knows to employ the appropriate preparation containing the appropriate concentration of immunotoxins in the formulation, as well as the appropriate amount of formulation to administer depending upon clinical experience with the patient in question or with similar patients. The concentration of immunotoxin for topical formulations is in the range of greater than from about 0.1 mg/ml to about 25 mg/ml. Typically, the concentration of immunotoxin for topical formulations is in the range of greater than from about 1 mg/ml to about 20 mg/ml. Solid dispersions of immunotoxins according to the invention, as well as solubilized preparations, may be used. Thus, the precise concentration to be used in the vehicle is subject to modest experimental manipulation in order to optimize the therapeutic response. For example, greater than about 10 mg immunotoxin/100 grams of vehicle may be useful with 1% w/w hydrogel vehicles in the treatment of skin inflammation. Suitable vehicles, in addition to gels, are oil-in-water or water-in-oil emulsions using mineral oils, petroleum and the like.</p>
    <p>Immunotoxins according to the invention may be optionally administered topically by the use of a transdermal therapeutic system [Barry, <i>Dermatological Formulations, </i>p. 181 (1983) and literature cited therein]. While such topical delivery systems may be designed for transdermal administration of low molecular weight drugs, they are capable of percutaneous delivery. Further, such systems may be readily adapted to administration of immunotoxin or derivatives thereof and associated therapeutic proteins by appropriate selection of the rate-controlling microporous membrane.</p>
    <p>Topical preparations of immunotoxin either for systemic or local delivery may be employed and may contain excipients as described above for parenteral administration and other excipients used in a topical preparation such as cosolvents, surfactants, oils, humectants, emollients, preservatives, stabilizers and antioxidants. Pharmacologically-acceptable buffers may be used, e.g., Tris or phosphate buffers. The topical formulations may also optionally include one or more agents variously termed enhancers, surfactants, accelerants, adsorption promoters or penetration enhancers, such as an agent for enhancing percutaneous penetration of the immunotoxin or other agents. Such agents should desirably possess some or all of the following features as would be known to the ordinarily skilled artisan: pharmacological inertness, non-promotive of body fluid or electrolyte loss, compatible with immunotoxin (non-inactivating), and capable of formulation into creams, gels or other topical delivery systems as desired.</p>
    <p>Immunotoxins according to the present invention may also be administered by aerosol to achieve localized delivery to the lungs. This is accomplished by preparing an aqueous aerosol, liposomal preparation or solid particles containing immunotoxin. Ordinarily, an aqueous aerosol is made by formulating an aqueous solution or suspension of immunotoxin together with conventional pharmaceutically acceptable carriers and stabilizers. The carriers and stabilizers vary depending upon the requirements for the particular immunotoxin, but typically include: nonionic surfactants (Tweens, Pluronics, or polyethylene glycol); innocuous proteins like serum albumin, sorbitan esters, oleic acid, lecithin; amino acids such as glycine; and buffers, salts, sugars or sugar alcohols. The formulations may also include mucolytic agents as well as bronchodilating agents. The formulations are sterile. Aerosols generally are prepared from isotonic solutions. The particles optionally include normal lung surfactants.</p>
    <p>Alternatively, immunotoxins of the invention may be administered orally by delivery systems such as proteinoid encapsulation as described by Steiner, et al., U.S. Pat. No. 4,925,673, incorporated by reference herein. Typically, a therapeutically-effective oral dose of an immunotoxin according to the invention is in the range from about 0.05 mg/kg body weight to about 50 mg/kg body weight per day. A preferred effective dose is in the range from about 0.05 mg/kg body weight to about 5 mg/kg body weight per day.</p>
    <p>Immunotoxins according to the present invention may be administered systemically, rather than topically, by injection intramuscularly, subcutaneously, intrathecally or intraperitoneally or into vascular spaces, particularly into the joints, e.g., intraarticular injection at a dosage of greater than about 1 μg/cc joint fluid/day. The dose will be dependent upon the properties of the specific immunotoxin employed, e.g., its activity and biological half-life, the concentration of immunotoxin in the formulation, the site and rate of dosage, the clinical tolerance of the patient involved, the disease afflicting the patient and the like, as is well within the skill of the physician.</p>
    <p>The immunotoxins of the present invention may be administered in solution. The pH of the solution should be in the range of pH 5 to 9.5, preferably pH 6.5 to 7.5. The immunotoxin or derivatives thereof should be in a solution having a suitable pharmaceutically-acceptable buffer such as phosphate, Tris(hydroxymethyl)aminomethane-HCl or citrate and the like. Buffer concentrations should be in the range of 1 to 100 mM. The immunotoxin solution may also contain a salt, such as sodium chloride or potassium chloride in a concentration of 50 to 150 mM. An effective amount of a stabilizing agent such as an albumin, a globulin, a gelatin, a protamine or a salt of protamine may also be included, and may be added to a solution containing immunotoxin or to the composition from which the solution is prepared.</p>
    <p>Systemic administration of immunotoxin may be made daily and is generally by intramuscular injection, although intravascular infusion is acceptable. Administration may also be intranasal or by other nonparenteral routes. Immunotoxins of the present invention may also be administered via microspheres, liposomes or other microparticulate delivery systems placed in certain tissues including blood. Topical preparations are applied daily directly to the skin or mucosa and are then preferably occluded, i.e., protected by overlaying a bandage, polyolefin film or other barrier impermeable to the topical preparation.</p>
    <p>The following Examples are illustrative of practice of the invention but are not to be construed as limiting the invention. The present application is broadly organized as follows. The first portion of the application broadly teaches the preparation, expression and properties of an exemplary RIP, gelonin. A second portion of the application teaches the preparation of human-engineered antibodies. A third portion of the application teaches the construction and properties of immunoconjugates, comprising an RIP and an antibody or fragment thereof comprising an antigen-binding portion. A forth portion of the application relates to the preparation and properties of immunofusion proteins comprising an RIP and an antibody or fragment thereof comprising an antigen-binding portion. A fifth portion of the application teaches the preparation and properties of the RIP Barley ribosome-inactivating protein and a final aspect of the invention provides the preparation and properties of the RIP momordin.</p>
    <p>Specifically, Example 1 relates to the preparation of the RIP gelonin. Construction of expression vector, comprising the gelonin gene, including expression and purification of gelonin, is taught in Example 2. The assembly of gelonin genes with cysteine residues available for conjugation is taught in Example 3 and Example 4 provides results of a reticulocyte lysate assay performed on gelonin.</p>
    <p>Example 5 teaches the construction of human-engineered antibodies for use in immunotoxins of the invention and Example 6 demonstrates transfection of he3 genes, expression of those genes, and purification of the products thereof.</p>
    <p>Example 7 next teaches the preparation of gelonin immunoconjugates. The procedures and results of whole cell kill assays are next presented in Example 8. Various properties of gelonin immunoconjugates are taught in Example 9 and Examples 10 and 11 teach the pharmacokinetics of two types of immunoconjugates. Examples 12 and 13 teach the immunogenicity of immunoconjugates of the invention and the in vivo efficacy of those immunoconjugates, respectively.</p>
    <p>The construction of genes encoding gelonin immunofusions is taught in Examples 14, 15, 16, 17 and 18. Example 19 teaches alternative cathepsin cleavable linkers for use in the immunofusions of the invention. The expression and purification of various genes encoding immunoconjugates are presented in Example 20 and their activity properties are presented in Example 21.</p>
    <p>The construction of genes encoding the RIP, BRIP, and its expression and properties are taught in Examples 22, 23, and 24.</p>
    <p>Finally, construction of genes encoding momordin and properties of momordin on expression are taught in Example 25.</p>
    <heading>EXAMPLE 1</heading> <p>Preparation of Gelonin</p>
    <p>The cloning of the gelonin gene according to the present invention obviates the requirement of purifying the RIP gene product from its relatively scarce natural source, <i>G. multiflorum </i>seeds. Cloning also allows development of gelonin analogs which may be conjugated to antibodies without prior chemical derivatization and also allows development of gelonin gene fusion products.</p>
    <p>A. Preparation of RNA from <i>G. Multiflorum </i>Seeds</p>
    <p>Total RNA was prepared from Gelonium seeds (Dr. Michael Rosenblum, M.D. Anderson Cancer Center, Houston, Tex.) by a modification of the procedure for preparation of plant RNA described in Ausubel et al., eds., <i>Current Protocols in Molecular Biology, </i>Wiley &amp; Sons, 1989. Briefly, 4.0 grams of seeds were ground to a fine powder in a pre-cooled (−70° C.) mortar and pestle with liquid N<sub>2</sub>. The powder was added to 25 ml Grinding buffer (0.18M Tris, 0.09M LiCl, 4.5 mM EDTA, 1% SDS, pH 8.2) along with 8.5 ml of phenol equilibrated with TLE (0.2M Tris, 0.1M LiCl, 5 mM EDTA pH8.2). The mixture was homogenized using a Polytron PT-1035 (#5 setting). 8.5 ml of chloroform was added, mixed and incubated at 50° C. for 20 minutes. The mixture was centrifuged at 3000 g for 20 minutes in a rotor precooled to 4° C. and the aqueous phase was transferred to a new tube. 8.5 ml of phenol was added followed by 8.5 ml of chloroform and the mixture was recentrifuged. This extraction was repeated 3 times. The RNA in the aqueous phase was then precipitated by adding ⅓ volume 8M LiCl, and incubated at 4° C. for 16 hours. Next, the RNA was pelleted by centrifugation for 20 minutes at 4° C. The pellet was washed with 5 ml of 2M LiCl, recentrifuged and resuspended in 2 ml of water. The RNA was precipitated by addition of NaOAc to 0.3M and 2 volumes of ethanol. The RNA was stored in 70% ethanol at −70° C.</p>
    <p>B. cDNA Preparation</p>
    <p>cDNA was prepared from total Gelonium RNA by two methods. The first method involved making a cDNA library in the bacterial expression plasmid pcDNAII using the Librarian II cDNA Library Construction System kit (Invitrogen). Approximately 5 μg of total RNA was converted to first strand cDNA with a 1:1 mixture of random primers and oligo-dT. Second strand synthesis with DNA polymerase I was performed as described by the system manufacturer. Double stranded cDNA was ligated to BstX1 linkers and size fractionated. Pieces larger than about 500 bp were ligated into the expression vector provided in the kit. Individual vectors were introduced into <i>E. coli </i>either by transformation into high-efficiency competent cells or by electroporation into electrocompetent cells. Electroporation was performed with a BTX100 unit (BTX, San Diego, Calif.) in 0.56μ Flatpack cells as recommended by BTX based on the method of Dower et al., <i>Nucleic Acids Res., </i>16:6127-6145 (1988), at a voltage amplitude of 850 V and a pulse length of 5 mS. The resulting library consisted of approximately 150,000 colonies.</p>
    <p>The second method involved generating cDNA using the RNA-PCR kit sold by Perkin-Elmer-Cetus. About 100 ng of total Gelonium RNA was used as template for cDNA synthesis.</p>
    <p>C. Determination of the Gelonin Protein Sequence</p>
    <p>The partial sequence of the native gelonin protein was determined by direct amino acid sequence analysis using automated Edman degradation as recommended by the manufacturer using an Applied Biosystems model 470A protein sequencer. Proteolytic peptide fragments of gelonin (isolated from the same batch of seeds as the total RNA) were sequenced.</p>
    <p>D. Cloning of the Gelonin Gene</p>
    <p>Three overlapping gelonin cDNA fragments were cloned and a composite gelonin gene was assembled from the three fragments.</p>
    <p>1. Cloning Of The Fragment Encoding The Middle Amino Acids Of Gelonin In Vector pING3823</p>
    <p>Degenerate DNA primers based on the gelonin partial amino acid sequences were used to PCR-amplify segments of the cDNA generated with Perkin-Elmor-Cetus kit. Six primers were designed based on regions of the gelonin amino acid sequence where degeneracy of the primers could be minimized. Appropriate pairs of primers were tested for amplification of a gelonin gene fragment. Products of the expected DNA size were identified as ethidium bromide-stained DNA bands on agarose gels that DNA was treated with T4 DNA polymerase and then purified from an agarose gel. Only the primer pair consisting of primers designated gelo-7 and gelo-5 yielded a relatively pure product of the expected size. The sequences of degenerate primers gelo-7 and gelo-5 are set out below using IUPAC nucleotide symbols.</p>
    <p>Gelo-7 (SEQ ID NO: 14) 5′ TTYAARGAYGCNCCNGAYGCNGCNTAYGARGG 3′</p>
    <p>Gelo-5 (SEQ ID NO: 15) 3′ TTYTTYATRATRCANTGNCGNCANCTRGTYCA 5′</p>
    <p>Primer gelo-7 corresponds to amino acids 87-97 of gelonin while primer gelo-5 corresponds to amino acids 226-236. The blunt-ended DNA fragment (corresponding to amino acids 87 to 236 of gelonin) generated with primers gelo-7 and gelo-5 was cloned into pUC18 (BRL, Gaithersburg, Md.). The DNA sequence of the insert was determined, and the deduced amino acid sequence based on the resulting DNA sequence matched the experimentally determined gelonin amino acid sequence. The clone containing this gelonin segment was denoted pING3726.</p>
    <p>The insert of clone pING3726 was labeled with <sup>32</sup>P and used as a probe to screen the 150,000-member Gelonium cDNA library. Only one clone hybridized to the library plated in duplicate. This clone was purified from the library and its DNA sequence was determined. The clone contains a fragment encoding 185 of the 270 amino acids of gelonin (residues 25-209) and is denoted pING3823.</p>
    <p>2. Cloning Of The Fragment Encoding The N-Terminal Amino Acids Of Gelonin</p>
    <p>Based on the sequence determined for the gelonin gene segment in pING3726, exact oligonucleotide primers were designed as PCR amplification primers to be used in conjunction with a degenerate primer to amplify a 5′ gelonin gene fragment and with a nonspecific primer to amplify a 3′ gelonin gene fragment. cDNA generated using the Perkin-Elmer-Cetus RNA-PCR kit was amplified.</p>
    <p>To amplify the 5′-end of the gelonin gene, PCR amplification with a degenerate primer gelo-1 and an exact primer gelo-10 was performed. The sequences of the primers are set out below.</p>
    <p>Gelo-1 (SEQ ID NO: 16) 5′ GGNYTNGAYACNGTNWSNTTYWSNACNAARGG 3′</p>
    <p>Gelo-10 (SEQ ID NO: 17) 3′ TGTCTGAACCCGTAACTTGGTAA 5′</p>
    <p>Primer gelo-1 corresponds to amino acids 1-11 of the gelonin gene while primer gelo-10 corresponds to amino acids 126-133. The product from the reaction was re-amplified with gelo-1 (SEQ ID NO: 16) and gelo-11 (an exact primer comprising sequences encoding amino acids 119-125 of gelonin) to confer specificity to the reaction product. The sequence of primer gelo-11 is listed below.</p>
    <p>Gelo-11 (SEQ ID No: 18) 3′ CACTCTTCCGTATATCTCTCTGT 5′</p>
    <p>Hybridization with an internal probe confirmed that the desired specific gelonin DNA fragment was amplified. That fragment was cloned into pUC18 and the vector generated was designated pING3727. The fragment was sequenced, revealing that the region of the fragment (the first 27 nucleotides) corresponding to part of the degenerate primer gelo-1 could not be translated to yield the amino acid sequence upon which primer gelo-1 was originally based. This was not unexpected considering the degeneracy of the primer. The fragment was reamplified from the Gelonium cDNA with exact primers gelo-11 (SEQ ID NO: 18) and gelo-5′ (which extends upstream of the 5′ end of the gelonin gene in addition to encoding the first 16 amino acids of gelonin). The sequence of primer gelo-5′ is set out below.</p>
    <p>Gelo-5′ (SEQ ID NO: 19) 5′ TCAACCCGGGCTAGATACCGTGTCAT TCTCAACCAAAGGTGCCACTTATATTA 3′</p>
    <p>The resulting DNA fragment encodes the first 125 amino acids of gelonin. While the majority of the sequence is identical to the natural gelonin gene, the first 32 nucleotides of the DNA fragment may be different. For the purposes of this application this N-terminal fragment is referred to as fragment GEL1-125.</p>
    <p>3. Cloning Of The Fragment Encoding The C-Terminal Amino Acids Of Gelonin</p>
    <p>To amplify the 3′-end of the gelonin gene as well as 3′ untranslated sequences, PCR amplification with exact primers gelo-9 and XE-dT was performed. The sequence of each of the primers is set out below.</p>
    <p>Gelo-9 (SEQ ID NO: 20) 5′ CTTCATTTTGGCGGCACGTATCC 3′</p>
    <p>XE-dT (SEQ ID NO: 21) 3′ TTTTTTTTTTTTTTTTTTTTTAG GGTGCATTCGAACGTCGGAGCTC 5′</p>
    <p>Primer gelo-9 corresponds to amino acids 107-113 of gelonin. Primer XE-dT consists of a 3′ oligo-dT portion and a 5′ portion containing the restriction sites HindIII and XhoI, and will prime any poly A-containing cDNA. The reaction product was reamplified with exact primers gelo-8 and XE. The sequences of primers gelo-8 and XE are set out below.</p>
    <p>Gelo-8 (SEQ ID NO: 22) 5′ CTCGCTGGAAGGTGAGAA 3′</p>
    <p>XE (SEQ ID NO: 23) 3′ AGGGTGCATTCGAACGTCGGAGCTC 5′</p>
    <p>Primer gelo-8 consists of sequences encoding amino acids 115-120 of gelonin while the primer XE corresponds to the 5′ portion of the XE-dT primer which contains HindIII and XhoI restriction sites. Hybridization with internal probes confirmed that the desired gelonin gene fragment was amplified. That fragment was then cloned into pUC18 by two different methods. First, it was cloned as a blunt-ended fragment into the SmaI site of pUC18 (the resulting vector was designated pING3728) and, second, it was cloned as an EcoRI to HindIII fragment into pUC18 (this vector was designated pING3729). Both vector inserts were sequenced. The insert of pING3728 encodes amino acids 114-270 of gelonin, while the insert of pING3729 encodes amino acids 184-270 of gelonin plus other 3′ sequences.</p>
    <p>4. Assembly Of The Overlapping Gelonin DNA Fragments Into A Composite Gelonin Gene</p>
    <p>To reassemble the C-terminal two-thirds of the gelonin gene, vector pING3729 was cut with SspI (one SspI site is located within the vector and the second is located about 80 bp downstream of the termination codon of the insert in the vector) and an XhoI linker (8 bp, New England Biolabs) was ligated to the resulting free ends. The DNA was then cut with XhoI and EcoRI, and the 350 bp fragment generated, encoding amino acids 185-270 of gelonin, was isolated. This 350 bp fragment was ligated adjacent to a NcoI to EcoRI fragment from pING3823 encoding amino acids 37-185 of gelonin in a intermediate vector denoted pING3730, thus reassembling the terminal 87% of the gelonin gene (amino acids 37-270).</p>
    <p>Next, fragment GEL1-125 was cut with SmaI and NcoI, resulting in a fragment encoding amino acids 1-36 of gelonin which was ligated along with the NcoI to XhoI fragment of pING3730 into the vector pIC100. [pIC100 is identical to pING1500 described in Better, et al., <i>Science, </i>240:1041-1043 (1988) incorporated by reference herein], except that it lacks 37 bp upstream of the pelB leader sequence. The 37 bp were eliminated by digestion of pING1500 with SphI and ECORI, treatment with T4 polymerase, and religation of the vector. This manipulation regenerated an EcoRI site in the vector while eliminating other undesirable restriction sites.] Before ligation, the vector pIC100 had previously been digested with SstI, treated with T4 polymerase, and cut with XhoI. The ligation generated a new vector containing a complete gelonin gene which was designated plasmid pING3731 and deposited with The American Type Culture Collection, 12301 Parklawn Drive, Rockville, Md. 20852 on Oct. 2, 1991 as Accession No. 68721. The complete DNA sequence of the gelonin gene is set out in SEQ ID NO: 11.</p>
    <heading>EXAMPLE 2</heading> <p>A. Construction of Expression Vectors Containing the Gelonin Gene</p>
    <p>A first <i>E. coli </i>expression vector was constructed containing the gelonin gene linked to the <i>Erwinia carotovora </i>pelB leader sequence, and to the <i>Salmonella typhimurium </i>araB promoter. A basic vector containing the arab promoter is described in co-owned U.S. Pat. No. 5,028,530 issued Jul. 2, 1991 which is incorporated by reference herein. The vector containing the araB promoter was cut with EcoRI and XhoI. Two DNA fragments were then ligated in tandem immediately downstream of the promoter. The fragment ligated adjacent to the promoter was a 131 bp fragment derived from SstI digestion, T4 polymerase treatment and digestion with EcoRI of the pIC100 vector which includes the leader sequence of the <i>E. carotovora </i>pe1B gene. The translated leader sequence is a signal for secretion of the respective protein through the cytoplasmic membrane. The fragment ligated downstream of the leader sequence was a SmaI to XhoI fragment from pING3731 which contains the complete gelonin gene. Thus, the expression vector contains the gelonin gene linked to the pelB leader sequence and the arab promoter. This plasmid is designated pING3733.</p>
    <p>A second expression vector may be constructed that is identical to the first except that the gelonin gene sequences encoding the nineteen C-terminal amino acids of gelonin are not included. The cDNA sequence of the gelonin gene predicted a 19 residue C-terminal segment that was not detected in any peptide fragments generated for determination of the gelonin amino acid sequence. These 19 amino acids may represent a peptide segment that is cleaved from the mature toxin post-translationally, i.e. that is not present in the native protein. A similar C-terminal amino acid segment was identified in the plant toxin α-trichosanthin [Chow et al., <i>J. Biol. Chem., </i>265:8670-8674 (1990)]. Therefore, the expression product without the C-terminal fragment is of interest.</p>
    <p>For construction of a gelonin expression vector without the 19 C-terminal amino acids of gelonin, PCR was used to amplify and alter the 3′-end of the gene. pING3728 was amplified with primers gelo-14 and gelo-9 (SEQ ID NO: 20). The sequence of primer gelo-14 is set out below.</p>
    <p>Gelo-14 (SEQ ID NO: 24) 5′ TGATCTCGAGTACTATTTAGGATCTTTATCGACGA 3′</p>
    <p>Primer gelo-14, which corresponds to gelonin amino acids 245-256, introduces a termination codon (underlined in the primer sequence) in the gelonin gene sequence which stops transcription of the gene before the sequences encoding the terminal 19 amino acids of the gelonin and also introduces a XhoI site immediately downstream of the termination codon. The PCR product was cut with XhoI and EcoRI, and the resulting 208 bp fragment encoding amino acids 185-251 of gelonin was purified from an agarose gel. This fragment was ligated adjacent to the NcoI to EcoRI fragment from pING3823 encoding amino acids 37-185 of gelonin to generate plasmid pING3732. A final expression vector, pING3734, containing a gelonin gene with an altered 3′-end was generated by substituting an NcoI to XhoI fragment encoding amino acids 37-251 of gelonin from pING3732 into pING3733.</p>
    <p>B. Identification of the Native Gelonin 5′-End</p>
    <p>Inverse PCR was used to identify a cDNA clone encoding the 5′-end of the mature gelonin gene. 5 μg of total <i>G. multiflorum </i>RNA was converted to cDNA using the Superscript Plasmid System (BRL, Gaithersburg, Md.) with Gelo-11 (SEQ ID NO: 18) as a primer. Gelonin cDNA was self-ligated to generate covalent circular DNA and the ligated DNA was amplified by PCR with oligonucleotides Gelo-9 (SEQ ID NO: 20) and Gelo-16. The sequence of primer Gelo-16 is set out below.</p>
    <p>Gelo-16 (SEQ ID NO: 25) 5′ GTAAGCAGCATCTGGAGCATCT 3′</p>
    <p>The PCR product was size-fractionated on an agarose gel and DNAs larger than 300 bp were cloned into SmaI cut pUC18. Several clones were sequenced with the primer Gelo-18, the sequence of which is set out below.</p>
    <p>Gelo-18 (SEQ ID NO: 26) 5′ CATTCAAGAAATTCACGTAGG 3′</p>
    <p>A clone identified as having the largest gelonin-specific insert was designated pING3826. The DNA sequence of pING3826 included the first 32 nucleotides of the natural, mature gelonin gene not necessarily present in gelonin expression plasmids pING3733 and pING3734. The complete DNA sequence of the natural gelonin gene is set out in SEQ ID NO: 11.</p>
    <p>C. Construction of Expression Vectors Containing a Gelonin Gene with a Natural 5′ End</p>
    <p>Derivatives of expression vectors pING3733 and pING3734 (described above) containing a gelonin gene with the natural 5′ sequence were generated as follows. The 5′-end of gelonin was amplified from pING3826 with the PCR primers Gelo-16 (SEQ ID NO: 24) and Gelo-17, the sequence of which is set out below.</p>
    <p>Gelo-17 (SEQ ID NO: 27) 5′ GGCCTGGACACCGTGAGCTTTAG 3′</p>
    <p>The 285 bp PCR product was treated with T4 polymerase and cut with NcoI. The resulting 100 bp 5′-end DNA fragment was isolated from an agarose gel and ligated adjacent to the 120 bp pelB leader fragment from plC100 (cut with SstI, treated with T4 polymerase and cut with PstI) into either pING3733 or pING3734 digested with PstI and NcoI. The resulting plasmids pING3824 and pING3825 contain the entire native gelonin gene and the native gelonin gene minus the nineteen amino acid carboxyl extension, respectively, linked to the pelB leader and under the transcriptional control of the araB promoter. The gene construct without the nineteen amino acid carboxyl extension in both pING3734 and pING3825 encodes a protein product referred to in this application as “recombinant gelonin”.</p>
    <p>D. Purification of Recombinant Gelonin</p>
    <p>Recombinant gelonin was purified by the following procedure: <i>E. coli </i>fermentation broth was concentrated and buffer-exchanged to 10 mM sodium phosphate at pH 7.0 by using an S10Y10 cartridge over a DC10 unit (Amicon) the concentrated and buffer-exchanged material was applied to a CM52 column (100 g, 5×10 cm). The column was washed with 1 L of starting buffer and eluted with a 0 to 300 mM NaCl gradient in starting buffer (750 ml total volume). The pure gelonin containing fractions were pooled (elution was from 100-250 mM NaCl), concentrated over an Amicon YM10 membrane, equilibrated with 10 mM sodium phosphate buffer, pH 7.0, and stored frozen at −20° C. A further purification step was attempted using Blue Toyopearl chromatography. However, this procedure did not result in an increased purity of material and resulted in an approximate 50% loss of the starting material.</p>
    <heading>EXAMPLE 3</heading> <p>Assembly of Gelonin Genes with Cysteine Residues Available for Conjugation</p>
    <p>The wild-type gelonin protein has two cysteine residues at positions 44 and 50 which are linked by an endogenous disulfide bond. The protein contains no free cysteine residue directly available for conjugation to antibodies or other proteins. Analogs of gelonin which contain a free cysteine residue available for conjugation were generated by three different approaches. In one approach, various residues along the primary sequence of the gelonin were replaced with a cysteine residue, creating a series of analogs which contain an odd number of cysteine residues. In another approach, one of the two endogenous cysteines was replaced by alanine, creating a molecule which lacks an intrachain disulfide bond but contains a single, unpaired cysteine. In yet another approach both endogenous cysteines were replaced by alanines and a third non-cysteine residue was replaced by a cysteine, creating an analog with a single, unpaired cysteine.</p>
    <p>Fifteen analogs of gelonin were constructed. Ten non-cysteine residues of gelonin were targeted for substitution with a cysteine residue. Comparison of the amino acid sequence of gelonin to the natural amino acid sequence and tertiary structure of the ricin A-chain (see FIG. 1) suggested that these positions would be at the surface of the molecule and available for conjugation. Each of the ten gelonin analogs include a cysteine substituted in place of one of the following residues: lysine<sub>10</sub>, asparagine<sub>60</sub>, isoleucine<sub>103</sub>, aspartic acid<sub>146</sub>, arginine<sub>184</sub>, serine<sub>215</sub>, asparagine<sub>239</sub>, lysine<sub>244</sub>, aspartic acid<sub>247</sub>, and lysine<sub>248</sub>, and the analogs have respectively been designated Gel<sub>C10</sub>, Gel<sub>C60</sub>, Gel<sub>C103</sub>, Gel<sub>C146</sub>, Gel<sub>C184</sub>, Gel<sub>C215</sub>, Gel<sub>C239</sub>, Gel<sub>C244</sub>, Gel<sub>C247</sub>, and Gel<sub>248</sub>.</p>
    <p>Two analogs of gelonin were constructed in which one of the native gelonin cysteines that participates in an endogenous disulfide bond was replaced with a non-cysteine residue. Specifically, the cysteine at position 50 was replaced with an alanine residue, creating a gelonin analog (designated Gel<sub>A50(C44)</sub>, shown in SEQ ID NO: 99) which has a cysteine available for disulfide bonding at position 44. The Gel<sub>A50(C44) </sub>analog has been referred to previously as Gel<sub>C44 </sub>(see, e.g., co-owned, co-pending U.S. patent application Ser. No. 07/988,430, incorporated by reference herein). Conversely, the cysteine at position 44 was replaced with an alanine residue, resulting in an analog (designated Gel<sub>A44(C50)</sub>, shown in SEQ ID NO: 100) which has a cysteine available for disulfide bonding at position 50. The Gel<sub>A44(C50) </sub>analog has been referred to previously as Gel<sub>C50 </sub>(see, e.g., co-owned, co-pending U.S. patent application Ser. No. 07/988,430, incorporated by reference herein). The combined series of the foregoing twelve analogs thus spans the entire length of the mature gelonin protein.</p>
    <p>Another gelonin analog (Gel<sub>A44A50 </sub>SEQ ID NO: 101) was constructed in which both native gelonin cysteines were replaced with alanines. The Gel<sub>A44A50 </sub>analog has been referred to previously as Gel<sub>C44AC30A </sub>(see, e.g., co-owned, co-pending U.S. patent application Ser. No. 07/988,430, incorporated by: reference herein). Two additional analogs were constructed which have alanine residues substituted in place of both native cysteines and have either a cysteine residue substituted in place of the native lysine at position 10 (Gel<sub>C10A44A50</sub>, shown in SEQ ID NO: 110) or a cysteine residue substituted in place of the native aspartate at position 247 (Gel<sub>C247A44A50</sub>, shown in SEQ ID NO: 111).</p>
    <p>The variants of recombinant gelonin were constructed by restriction fragment manipulation or by overlap extension PCR with synthetic oligonucleotides. The sequences of the primers used for PCR are set out below. In each mutagenic primer sequence, the nucleotides corresponding to the changed amino acid, either a cysteine or an alanine residue, are underlined.</p>
    <p>Gelo-9 (SEQ ID NO: 20)</p>
    <p>Gelo-11 (SEQ ID NO: 18)</p>
    <p>Gelo-16 (SEQ ID NO: 25)</p>
    <p>Gelo-17 (SEQ ID NO: 27)</p>
    <p>Gelo-18 (SEQ ID NO: 26)</p>
    <p>Gelo-19 (SEQ ID NO: 58) 5′ CAGCCATGGAATCCCATTGCTG 3′</p>
    <p>GeloC-1 (SEQ ID NO: 28) 5′ TCGAT<u>TGC</u>GATCCTAAATAGTACTC 3′</p>
    <p>GeloC-2 (SEQ ID NO: 29) 5′ TTTAGGATC<u>GCA</u>ATCGACGAACTTCAAG 3′</p>
    <p>GeloC-3-2 (SEQ ID NO: 30) 5′ GTTCGTC<u>TGT</u>AAAGATCCTAAATAGTACTCGA 3′</p>
    <p>GeloC-4 (SEQ ID NO: 31) 5′ GGATCTTT<u>ACA</u>GACGAACTTCAAGAGT 3′</p>
    <p>GeloC-5 (SEQ ID NO: 32) 5′ TCTTG<u>TGC</u>TTCGTCGATAAAGATCC 3′</p>
    <p>GeloC-6 (SEQ ID NO: 33) 5′ ATCGACGAA<u>GCA</u>CAAGAGTGCTATTTT 3′</p>
    <p>GeloC-9 (SEQ ID NO: 34) 5′ GTAAAACCA<u>TGC</u>ATAGCACTCTTGAAGTTCGT 3′</p>
    <p>GeloC-10 (SEQ ID NO: 35) 5′ AGTGCTAT<u>GCA</u>TGGTTTTACTTGATCAACTGC 3′</p>
    <p>GeloC-13 (SEQ ID NO: 36) 5′ AGCACA<u>TGT</u>GGTGCCACTTATATTACCTA 3′</p>
    <p>GeloC-14 (SEQ ID NO: 37) 5′ TAAGTGGCACC<u>ACA</u>TGTGCTAAAGCTCACGGTG 3′</p>
    <p>GeloC-15 (SEQ ID NO: 38) 5′ TGAC<u>TGT</u>GGACAGTTGGCGGAAATA 3′</p>
    <p>GeloC-16 (SEQ ID NO: 39) 5′ GCCAACTGTCC<u>ACA</u>GTCATTTGAAAGCGCTACC 3′</p>
    <p>GeloC-17 (SEQ ID NO: 40) 5′ GATGATCCTGGAAAG<u>GCT</u>TTCGTTTTGGTAGCGCTT3′</p>
    <p>GeloC-18 (SEQ ID NO: 41) 5′ A<u>AGC</u>CTTTCCAGGATCATC<u>AGC</u>TTTTTTGCGCAGCAATGGG 3′</p>
    <p>GeloC-19 (SEQ ID NO: 42) 5′ A<u>AGC</u>CTTTCCAGGATCATCACAT 3′</p>
    <p>GeloC-20 (SEQ ID NO: 59) 5′ CACA<u>TGT</u>AAAACAAGACTTCATTTTGGC 3′</p>
    <p>GeloC-21 (SEQ ID NO: 60) 5′ TGAAGTCTTGTTTT<u>AGA</u>TGTGTTTTTGAAGAGGCCT 3′</p>
    <p>GeloC-22 (SEQ ID NO: 61) 5′ ATGCCATA<u>TGC</u>AATTATAAACCAACGGAGA 3′</p>
    <p>GeloC-23 (SEQ ID NO: 62) 5′ GGTTTATAATT<u>GCA</u>TATGG CATTTTCATCAAGTTTCTTG 3′</p>
    <p>GeloC-24 (SEQ ID NO: 63) 5′ CTTTCAACAA<u>TGC</u>ATTCGCCCGGCGAATAATAC 3′</p>
    <p>GeloC-25 (SEQ ID NO: 64) 5′ GCGAAT<u>GCA</u>TTGTTGAAAGTTATTTCTAATTTG 3′</p>
    <p>GeloC-26 (SEQ ID NO: 65) 5′ GTTT<u>TGT</u>GAGGCAGTTGAATTGGAAC 3′</p>
    <p>GeloC-27 (SEQ ID NO: 66) 5′ TTCAACTGCCTC<u>ACA</u>AAACATTCCATTTGCACCT 3′</p>
    <p>GeloC-28 (SEQ ID NO: 67) 5′ AAAA<u>GCT</u>GATGATCCTGGAAAGTG 3′</p>
    <p>GeloC-29 (SEQ ID NO: 68) 5′ TCCAGGATCATC<u>AGC</u>TTTTTTGCGCAGCAATGGGA 3′</p>
    <p>araB2 (SEQ ID NO: 43) 5′ GCGACTCTCTACTGTTTC 3′</p>
    <p>HINDIII-2 (SEQ ID NO: 44) 5′ CGTTAGCAATTTAACTGTGAT 3′</p>
    <p>(1) Specifically, a cysteine was introduced at amino acid 247 of gelonin (which is normally occupied by an aspartic acid which corresponds to the cysteine at position 259 in the ricin A-chain) by PCR with mutagenic primers GeloC-3-2 and GeloC-4 in conjunction with primers HINDIII-2 (a primer located in the vector portion of pING3734 or pING3825), Gelo-9 and Gelo-8. Template DNA (pING3734) was amplified with GeloC-3-2 and HINDIII-2 and in a concurrent reaction with GeloC-4 and Gelo-9. The products of these reactions were mixed and amplified with the outside primers Gelo-8 and HINDIII-2. The reaction product was cut with EcoRI and XhoI, purified, and was inserted into plasmid pING3825 in a three-piece ligation. The DNA sequence of the Gel<sub>C247 </sub>variant (SEQ ID NO: 102) was then verified. The plasmid containing the sequence encoding Gel<sub>C247 </sub>was designated pING3737 and was deposited with the American Type Culture Collection, 12301 Parklawn Drive, Rockville, Md. 20852 on Jun. 9, 1992 as ATCC Accession No. 69009.</p>
    <p>(2-3) In the same manner, a cysteine residue was introduced in place of the amino acid at position 248 (a lysine) of gelonin with the mutagenic oligonucleotides GeloC-1 and GeloC-2 to generate analog Gel<sub>C248 </sub>(SEQ ID NO: 103) in plasmid pING3741, and a cysteine residue was introduced at amino acid position 239 (normally occupied by a lysine) with primers GeloC-9 and GeloC-10 to generate analog Gel<sub>239 </sub>(SEQ ID NO: 104) in plasmid pING3744.</p>
    <p>(4) Also in the same manner, a cysteine residue was introduced at amino acid 244 (a lysine) of gelonin with mutagenic primers GeloC-5 and GeloC-6 to generate analog Gel<sub>C244 </sub>(SEQ ID NO: 105) in a plasmid designated pING3736. This variant was prepared by PCR using plasmid pING3734 as template DNA rather than pING3825. It therefore encodes the same N-terminal gelonin amino acid sequence as plasmids pING3737, pING3741, and pING3744, but includes the PCR primer-derived 5′-end 32 nucleotides instead of the native gelonin 5′-end nucleotides.</p>
    <p>(5) A cysteine residue was introduced in place of the amino acid (normally occupied by a lysine) at position 10 of gelonin by a similar procedure. A cysteine was introduced with mutagenic primers GeloC-13 and GeloC-14 by amplifying pING3824 with araB2 (a vector primer) and GeloC-14, and in a separate reaction, with GeloC-13 and Gelo-11. These reaction products were mixed and amplified with the outside primers araB2 and Gelo-11. The PCR product was cut with PstI and NcoI, purified, and cloned back into pING3825 to generate analog Gel<sub>C10 </sub>(SEQ ID NO: 106) in the plasmid designated pING3746 and deposited with the American Type Culture Collection, 12301 Parklawn Drive, Rockville, Md. 20852 on Jun. 9, 1992 as ATCC Accession No. 69008.</p>
    <p>(6) The asparagine at position 60 of gelonin was replaced with a cysteine residue using two mutagenic oligos, GeloC-15 and GeloC-16, in conjunction with oligos araB2 and Gelo-11 in the same manner as for the Gel<sub>C10 </sub>variant. The plasmid encoding the Gel<sub>C60 </sub>(SEQ ID NO: 107) analog was designated pING3749.</p>
    <p>(7) A cysteine was introduced at amino acid 103 (an isoleucine) by PCR with mutagenic primers GeloC-20 and GeloC-21 in conjunction with primers araB2 and HINDIII-2. Template DNA (pING3733) was amplified with GeloC-21 and araB2 and separately with GeloC-20 and HINDIII-2. The products of these reactions were mixed and amplified with the outside primers araB2 and HINDIII-2. The reaction product was cut with NcoI and BclI, purified, and inserted into pING3825 digested with NcoI and BclI. The oligonucleotides used to place a cysteine at residue 103 also introduced an AflIII restriction site which was verified in the cloned gene. The plasmid containing the Gel<sub>C103 </sub>(SEQ ID NO: 108) analog was designated pING3760.</p>
    <p>(8) A cysteine was introduced at position 146 (an aspartic acid) by a similar strategy. Template DNA (pING3733) was amplified with mutagenic primer GeloC-22 and Gelo-14 and separately with mutagenic primer GeloC-23 and Gelo-19. The products of these reactions were mixed, and amplified with Gelo-19 and Gelo-14. The reaction product was cut with BglII and EcoRI, and can be inserted into pING3825 in a three-piece ligation. The oligonucleotides used to place a cysteine at residue 146 also introduced a NdeI restriction site which can be verified in the cloned gene.</p>
    <p>(9) To introduce a cysteine at position 184 (normally occupied by an arginine) of gelonin, template DNA (pING3733) was amplified with mutagenic primer GeloC-25 and araB-2 and separately with mutagenic primer GeloC-24 and HINDIII-2. The products of these reactions were mixed, and amplified with araB2 and Gelo-14. The reaction product was cut with NcoI and BclI, and inserted into pING3825 previously digested with NcoI and BclI. The oligonucleotides used to place a cysteine at residue 184 also introduced an NsiI restriction site which was verified in the cloned gene. The plasmid containing the sequence encoding the Gel<sub>C184 </sub>(SEQ ID NO: 109) variant was designated pING3761.</p>
    <p>(10) A cysteine may be introduced at position 215 (a serine) by a similar strategy. Template DNA (pING3733) was amplified with mutagenic primer GeloC-27 and araB2 and separately with mutagenic primer GeloC-26 and HINDIII-2. The products of these reactions were mixed, and amplified with araB2 and HINDIII-2. The reaction product was cut with EcoRI and BclI, and may be inserted into pING3825 in a three-piece ligation.</p>
    <p>(11) Another gelonin variant with a free cysteine residue was generated by replacing on* of the two naturally occurring gelonin cysteine residues, the cysteine a position 50, with an alanine. Plasmid pING3824 was amplified with primers GeloC-17 and Gelo-11, and concurrently in a separate reaction with primers GeloC-19 and araB2. The reaction products were mixed and amplified with araB2 and Gelo-11. This product was cut with NcoI and BglII, and cloned back into the vector portion of pING3825 to generate pING3747 (ATCC 69101). This analog was designated Gel<sub>A50(C44) </sub>and it contains a cysteine available for disulfide bonding at amino acid position 44. Non-cysteine residues, other than alanine, which do not disrupt the activity of gelonin, also may be inserted at position 50 in natural gelonin in order to generate a gelonin analog with a single cysteine at position 44.</p>
    <p>(12) A gelonin variant in which the natural cysteine at position 44 was changed to alanine was constructed by amplifying pING3733 using the mutagenic oligonucleotides GeloC-28 and GeloC-29 in conjunction with primers araB2 and HINDIII-2. The amplified DNA was cut with NcoI and BglII and cloned into a gelonin vector, generating pING3756. That variant generated was designated Gel<sub>A44(C50)</sub>. Non-cysteine residues, other than alanine, which do not disrupt gelonin activity, also may be inserted at position 44 in order to generate a gelonin analog with a single cysteine at position 50.</p>
    <p>(13) A gelonin variant in which both the cysteine at position 44 and the cysteine at position 50 of gelonin were changed to alanine residues was constructed by overlap PCR of pING3824 using the mutagenic oligonucleotides GeloC-17 and GeloC-18 in conjunction with primers araB2 and Gelo-11. This analog, like the native gelonin protein, has no cysteine residues available for conjugation. The plasmid encoding the analog was designated pING3750. The analog generated was designated Gel<sub>A44A50 </sub>(SEQ ID NO: 101). Non-cysteine residues, other than alanine, which do not disrupt gelonin activity, also may be substituted at both positions 44 and 50 in order to generate a gelonin analog with no cysteine residues.</p>
    <p>(14) The triple mutant Gelonin<sub>C247A44A50 </sub>(SEQ ID NO: 111) was constructed from the plasmids pING3824, pING3750 and pING3737. This variant contains an introduced cysteine at position 247 while both of the naturally occurring cysteine residues at positions 44 and 50 have been replaced with alanine. The analog is desirable because, in this analog, disulfide linkage to an antibody is only assured at a single cysteine residue. Plasmid pING3824 was cut with NcoI and XhoI and the vector fragment was purified in an agarose gel. pING3750 was cut with NcoI and EcoRI and pING3737 was cut with EcoRI and XhoI. The NcoI-EcoRI fragment encodes the alanines at positions 44 and 50 while the EcoRI-XhoI fragment encodes the cysteine at position 247. Each of these fragments was purified and ligated to the NcoI to XhoI vector fragment. The resulting plasmid is named pING3752.</p>
    <p>(15) The triple mutant Gelonin<sub>C10A44A50 </sub>(SEQ ID NO: 110) was also constructed by assembly from previously assembled plasmids. In this case, pING3746 was cut with PstI and NcoI, while pING3750 was cut with NcoI and XhoI. Each of the insert fragments were purified by electrophoresis in an agarose gel, and the fragments were ligated into a PstI and XhoI digested vector fragment. The resulting vector was designated pING3753. The Gel<sub>C10A44A50 </sub>analog has been referred to previously as Gel<sub>C10C44AC50A </sub>(see, e.g., co-owned, co-pending U.S. patent application Ser. No. 07/988,430, incorporated by reference herein).</p>
    <p>Each of the gelonin variants constructed was transformed into <i>E. coli </i>strain E104. Upon induction of bacterial cultures with arabinose, gelonin polypeptide could be detected in the culture supernatants with gelonin-specific antibodies. There were no significant differences detected in the expression levels of gelonin from plasmids pING3734 and pING3825, or in the levels from any of the gelonin variants. Each protein was produced in <i>E. coli </i>at levels of approximately 1 g/l.</p>
    <heading>EXAMPLE 4</heading> <p>Reticulocyte Lysate Assay</p>
    <p>The ability of gelonin and recombinant gelonin analogs to inhibit protein synthesis in vitro was tested using a reticulocyte lysate assay (RLA) described in Press et al., <i>Immunol. Letters, </i>14:37-41 (1986). The assay measures the inhibition of protein synthesis in a cell-free system using endogenous globin mRNA from a rabbit red blood cell lysate. Decreased incorporation of tritiated leucine (<sup>3</sup>H-Leu) was measured as a function of toxin concentration. Serial log dilutions of standard toxin (the 30 kD form of ricin A-chain, abbreviated as RTA 30), native gelonin, recombinant gelonin (rGelonin or rGel) and gelonin analogs were tested over a range of 1 μg/ml to 1 pg/ml. Samples were tested in triplicate, prepared on ice, incubated for 30 minutes at 37° C., and then counted on an Inotec Trace 96 cascade ionization counter. By comparison with an uninhibited sample, the picomolar concentration of toxin (pM) which corresponds to 50% inhibition of protein synthesis (IC<sub>50</sub>) was calculated. As is shown in Table 1 below, recombinant gelonin and most of its analogs exhibit activity in the RLA comparable to that of native gelonin. For some of the analogs (such as Gel<sub>C239</sub>), RLA activity was diminished.</p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="offset" colwidth="42pt" align="left"> </colspec> <colspec colname="1" colwidth="49pt" align="left"> </colspec> <colspec colname="2" colwidth="126pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="OFFSET" nameend="2" rowsep="1" class="description-td" colspan="3">TABLE 1</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="OFFSET" nameend="2" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Toxin</td>
                <td class="description-td">IC<sub>50 </sub>(pM)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="OFFSET" nameend="2" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="offset" colwidth="42pt" align="left"> </colspec> <colspec colname="1" colwidth="49pt" align="left"> </colspec> <colspec colname="2" colwidth="126pt" align="char" char="."> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">RTA 30</td>
                <td class="description-td">2.5</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Gelonin</td>
                <td class="description-td">15</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">rGelonin</td>
                <td class="description-td">11</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Gel<sub>C10</sub> </td>
                <td class="description-td">60</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Gel<sub>A50(C44)</sub> </td>
                <td class="description-td">20</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Gel<sub>A44(C50)</sub> </td>
                <td class="description-td">47</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Gel<sub>C60</sub> </td>
                <td class="description-td">26</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Gel<sub>C239</sub> </td>
                <td class="description-td">955</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Gel<sub>C244</sub> </td>
                <td class="description-td">32</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Gel<sub>C247</sub> </td>
                <td class="description-td">12</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Gel<sub>C248</sub> </td>
                <td class="description-td">47</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Gel<sub>A44A50</sub> </td>
                <td class="description-td">16</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Gel<sub>C10A4450A</sub> </td>
                <td class="description-td">7</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Gel<sub>C247A44A50</sub> </td>
                <td class="description-td">20</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="OFFSET" nameend="2" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <heading>EXAMPLE 5</heading> <p>Human-Engineered Antibodies for Construction of Immunotoxins</p>
    <p>Antibodies for use in constructing immunotoxins according to the present invention may be humanized antibodies, such as he3 and fragments thereof which display increased content of human amino acids and a high affinity for human CD5 cell differentiation marker. he3 is a humanized form of a mouse H65 antibody (H65 is a preferred monoclonal antibody for use in preparing humanized antibodies according to the present invention and is produced by hybridoma cell line XMMLY-H65 (H65) deposited with the American Type Culture Collection in Rockville, Md. (A.T.C.C.) and given the Accession No. HB9286).</p>
    <p>Humanized antibodies for use in the present invention are prepared as disclosed herein using the humanized forms of the murine H65 antibody in which both low and moderate risk changes described below were made in both variable regions. Such humanized antibodies should have less immunogenicity and have therapeutic utility in the treatment of autoimmune diseases in humans. For example, because of their increased affinity over existing therapeutic monoclonal antibodies such as H65, he3 antibodies of the invention may be administered in lower doses than H65 anti-CD5 antibodies in order to obtain the same therapeutic effect.</p>
    <p>Humanized antibodies, such as he3, are useful in reducing the immunogenicity of foreign antibodies and also results in increased potency when used as a portion of an immunoconjugate.</p>
    <p>Construction of humanized antibody variable domains according to the present invention and for use as components of immunotoxins may be based on a method which includes the steps of: (1) identification of the amino acid residues of an antibody variable domain which may be modified without diminishing the native affinity of the domain for antigen while reducing its immunogenicity with respect to a heterologous species; and (2) the preparation of antibody variable domains having modifications at the identified residues which are useful for administration to heterologous species. The methods of the invention are based on a model of the antibody variable domain described herein and in U.S. patent application Ser. No. 07/808,464 by Studnicka, et al., which predicts the involvement of each amino acid in the structure of the domain.</p>
    <p>Unlike other methods for humanization of antibodies, which advocate replacement of the entire classical antibody framework regions with those from a human antibody, the methods described herein and in U.S. patent application Ser. No. 07/808,464 by Studnicka, et al., now abandoned, introduce human residues into the variable domain of an antibody only in positions which are not critical for antigen-binding activity and which are likely to be exposed to immunogenicity-stimulating factors. The present methods are designed to retain sufficient natural internal structure of the variable domain so that the antigen-binding capacity of the modified domain is not diminished in comparison to the natural domain.</p>
    <p>The human consensus sequences in which moderate risk residues are converted from mouse residues to human residues are represented in FIGS. 10A and 10B as lines labelled hK1 (i.e., subgroup 1 of the human kappa chain) and hH3 (i.e., subgroup 3 of the human heavy chain). Symbols in the figures for conservation and for risk in “bind” and “bury” lines are as follows:</p>
    <p>First Symbol in Pair (Ligand Binding)</p>
    <p>+ Little or not direct influence on antigen-binding loops, low risk if substituted</p>
    <p>∘ Indirectly involved in antigen-binding loop structure, moderate risk if changed</p>
    <p>− Directly involved in antigen-binding loop conformation or antigen contact, great risk if modified</p>
    <p>Second Symbol in Pair (Immunogenicity/Struture)</p>
    <p>+ Highly accessible to solvent, high immunogenicity, low risk if substituted</p>
    <p>∘ Partially buried, moderate immunogenicity, moderate risk if altered</p>
    <p>− Completely buried in subunit's hydrophobic core, low immunogenicity, high risk if changed</p>
    <p>= Completely buried in interface between subunits, low immunogenicity, high risk if modified</p>
    <p>Significance of Pairs</p>
    <p>++ Low risk</p>
    <p>Highly accessible to solvent and high immunogenicity, but little or no effect on specific antigen binding</p>
    <p>∘+, +∘, ∘∘ Moderate Risk</p>
    <p>Slight immunogenicity or indirect involvment with antigen binding</p>
    <p>any − or = High risk</p>
    <p>Buried within the subunit core/interface or strongly involved in antigen binding, but little immunogenic potential</p>
    <p>In the line labelled “mod”, a dot (.) represents a residue which may be mutated from “mouse” to “human” at moderate risk. There are 29 such moderate risk positions.</p>
    <p>The mouse residue matches the human consensus residue more than 50% of the time at 131 positions (102 positions match 90%-100% and 29 positions match 50% to 90%). These positions were not changed.</p>
    <p>The lines labelled M/H in FIGS. 12A and 12B indicate the 91 positions which differed significantly between the mouse and human sequences (i.e., where the human sequences have the mouse residue less than 50% of the time). Moderate risk positions, designated m in the M/H line, were kept “mouse”; whereas those designated H or h were changed to human. The 25 low risk positions which were already human-like or which were previously humanized (as described supra in Example 2) are designated “{circumflex over ( )}” in the M/H line. Finally, the 54 high risk positions in which the mouse and human residues did not match are designated M and are kept “mouse”.</p>
    <p>Fifteen differences occur at moderate risk positions at which the mouse and human sequences differ. At ten of those positions (designated “H” on the M/H lines of FIG. 6) the mouse residue aligns with a human consensus amino acid which is highly conserved. Therefore, the mouse residue at that position is identified as one to be changed to the conserved human-residue.</p>
    <p>At moderate risk positions (designated “m”) in which the mouse and the human sequences differ, the mouse residue aligns with a human consensus amino acid which is moderately conserved. However, since the mouse residue is found at that position in other actual sequences of human antibodies [See Kabat, et al., sequences of Proteins of Immunoglobulin Interest,Fourth Edition, U.S. Department of Health and Human Services, Public Health Service, National Institutes of Health (1987)] the positions are identified as ones to be kept “mouse.” Although there are no such positions in this particular sequence, such positions may occur in other antibodies.</p>
    <p>At four moderate risk positions (designated “h”), the mouse residue aligns with a human consensus amino acid which is moderately conserved but the mouse residue is not found at that position in an actual human antibody sequence in Kabat, et al. <i>Sequences of Proteins of Immunoglobulin Interest, </i>supra. Therefore, that position is identified as ones to be changed to “human.”</p>
    <p>At one moderate risk position (designated “m”) in which the mouse and human sequences differ, the mouse residue aligns with a human consensus amino acid which is poorly conserved. Therefore, that position is identified as one to be kept “mouse.”</p>
    <p>A. Assembly of Moderate Risk Heavy Chain Expression Vectors</p>
    <p>The humanized H65 heavy chain containing the moderate risk residues was assembled by the following strategy. The moderate-risk expression vector was assembled from intermediate vectors. The six oligonucleotide sequences (oligos), disclosed in FIG. <b>12</b> and labelled HUH-G11, HUH-G12, HUH-G3, HUH-G4, HUH-G5, and HUH-G6 (the sequences of HUH-G11 and HUH-G12 are set out in SEQ ID Nos. 131 and 132 and HUH-G3, HUH-G4, HUH-G5, and HUH-G6 are set out in SEQ ID NOS: 137-140) were assembled by PCR. Oligonucleotides containing the synthetic humanized antibody gene were mixed in pairs (HUH-G11+HUH-G12, HUH-G3+HUH-G4, and HUH-G5+HUH-G6) in a 100 μl reaction with 1 μg of each DNA and filled in as described above. A portion of each reaction product was mixed in pairs (HUH-G11, 12+HUH-G3, 4; HUH-G3, 4+HUH-G5, 6), 2.5 U Taq was added and samples were reincubated as described above. The V-J region was assembled by mixing equal amounts of the HUH-G11, 12, 3, 4 reaction product with the HUH-G3, 4, 5, 6 product, followed by PCR with 0.5 ug of primers H65G-2S and H65-G2 as described above. The reaction product was cut with SalI and BstEII and cloned into the expression vector, similar to that described for heavy chain in Robinson et al., <i>Hum. Antibod. Hybridomas </i>2:84 (1991), generating pING4617. That plasmid was sequenced with Sequenase (USB, Cleveland), revealing that two residues were altered (a G-A at position 288 and a A-T at position 312, numbered from the beginning of the leader sequence). The correct variable region was restored by substitution of this region from pING4612, generating the expected V-region sequence in pING4619.</p>
    <p>An intermediate vector containing the other moderate-risk changes was constructed by PCR assembly of the oligos HUH-G13, HUH-G14, HUH-G15, and HUH-G16 (FIG. <b>11</b> and SEQ ID Nos: 133-136). Oligos HUH-G13+HUH-G14 and HUH-G15+HUH-G16 were mixed-and filled in with Vent polymerase (New England Biotabs) in a reaction containing 10 mM KCl, 20 mM TRIS pH 8.8, 10 mM (NH<sub>4</sub>)<sub>2</sub>SO<sub>2</sub>, 2 mM MgSO<sub>4</sub>, 0.1% Triton X-100, 100 ng/ml BSA, 200 uM of each dNTP, and 2 units of Vent polymerase in a total volume of 100 μl. The reaction mix was incubated at 94° C. for 1 minute, followed by 2 minutes at 50° C. and 20 minutes at 72° C. The reaction products (40 μl) were mixed and amplified with the oligonucleotides H65-G13 and H65-G2 with Vent polymerase in the same reaction buffer and amplified for 25 cycles with denaturation at 94° C. for 1 minute, annealing at 50° C. for 2 minutes and polymerization at 72° C. for 3 minutes. The reaction product was treated with T4 polymerase and then digested with AccI. The 274 base pair (bp) fragment was purified on an agarose gel and ligated along with the 141 bp SalI to AccI fragment from pING4619 into pUC18 cut with SalI and SmaI to generate pING4620. pING4620 contains the entire signal sequence, V-region, and J-region of the moderate-risk H65 heavy chain.</p>
    <p>The final expression vector for the moderate-risk H65 heavy chain, pING4621, was assembled by cloning the SalI to BstEII fragment from pING4620 into the same expression vector described above.</p>
    <p>B. Assembly of Moderate-Risk Light Chain Expression Vectors</p>
    <p>The moderate-risk humanized V- and J-segments of the light chain were assembled from six oligonucleotides, $H65K-1 (SEQ ID NO: 117), HUH-K7 (SEQ ID NO: 119), HUH-K6 (SEQ ID NO: 118), HUH-K8 (SEQ ID NO: 120), HUH-K4 (SEQ ID NO: 121 and HUH-K5 (SEQ ID NO: 122). The oligonucleotides were amplified with PCR primers H65K-2S and JK1-HindIII. Oligonucleotides containing the synthetic humanized antibody gene were mixed in pairs ($H65-K1+HUH-K7, HUH-K6+HUH-K4+HUH-K5) and incubated with Vent polymerase as described for the moderate-risk heavy chain. A portion of each reaction product (40 ul) was mixed in pairs ($H65H-K1/HUH-K7+HUH-K6, 8; HUH-K6, 8+HUH-K4, 5) and filled in as above. The light chain gene was then assembled by amplifying the full length gene with the PCR primers H65K-2S and JK1-HindIII with Vent polymerase for 25 cycles as outlined above. The assembled V/J region was cut with SalI and HindIII, purified by electrophoresis on an agarose gel, and assembled into a light chain antibody expression vector, pING4630.</p>
    <heading>EXAMPLE 6</heading> <p>Transfection of he3 Genes and Purification of Expression Products</p>
    <p>A. Stable Transfection of Mouse Lymphoid Cells for the Production of he3 Antibody</p>
    <p>The cell line Sp2/0 (American Type culture Collection Accession No. CRL1581) was grown in Dulbecco's Modified Eagle Medium plus 4.5 g/l glucose (DMEM, Gibco) plus 10% fetal bovine serum. Media were supplemented with glutamine/penicillin/streptomycin (Irvine Scientific, Irvine, Calif.).</p>
    <p>The electroporation method of Potter, H., et al., <i>Proc. Natl. Acad. Sci., USA, </i>81:7161 (1984) was used. After transfection, calls were allowed to recover in complete DMEM for 24-48 hours, and then seeded at 10,000 to 50,000 calls per well in 96-well culture plates in the presence of selective medium. Histidinol (Sigma) selection was at 1.71 μg/ml, and mycophenolic acid (Calbiochem) was at 6 μg/ml plus 0.25 mg/ml xanthine (Sigma). The electroporation technique gave a transfection frequency of 1-10×10<sup>−5 </sup>for the Sp2/0 cells.</p>
    <p>The he3 light chain expression plasmid pING4630 was linearized by digestion with PvuI restriction endonuclease and transfected into Sp2/0 cells, giving mycophenolic acid—resistant clones which were screened for light chain synthesis.</p>
    <p>Four of the top-producing subclones, secreting 4.9-7.5 μg/ml were combined into two pools (2 clones/pool) and each pool was transfected with plasmid pING42621, containing the moderate-risk heavy chain. After selection with histidinol, the clones producing the most light plus heavy chain, Sp2/0-4630 and 4621 Clones C1705 and C1718, secreted antibody at approximately 15 and 22 μg/ul respectively in the presence of 10<sup>−7 </sup>M dexamethasone in an overgrown culture. in a T25 flask. Clone C1718 was deposited with the American Type Culture Collection, 1230 Parklawn Drive, Rockville, Md., 20852 on Dec. 1, 1992 as ATCC HB 11206. The best producer is a subclone of Clone C1718 which is produced by limiting dilution subcloning of Clone C1718.</p>
    <p>B. Purification of he3 Antibody Secreted in Tissue Culture</p>
    <p>Sp2/0-4630+4621 Clone C1705cells were grown in culture medium HB101 (Hana Biologics)+1% Fetal Bovine Serum, supplemented with 10 mM HEPES, 1×Glutamine-Pen-Strep (Irvine Scientific #9316). The spent medium was centrifuged at about 5,000×g for 20 minutes. The antibody level was measured by ELISA. Approximately 200 ml of cell culture supernatant was loaded onto a 2 ml Protein A-column (Sigma Chemicals), equilibrated with PBS (buffer 0.15 M NaCl, 5 mM sodium phosphate, 1 mM potassium phosphate, buffer pH 7.2). The he3 antibody was eluted with a step pH gradient (pH 5.5, 4.5 and 2.5). A fraction containing he3 antibody (9% yield) but not bovine antibody, was neutralized with 1 M Tris pH 8.5, and then concentrated 10-fold by Centricon 30 (Amicon) diluted 10-fold with PBS, reconcentrated 10-fold by Centricon 30, diluted 10-fold with PBS, and finally reconcentrated 10-fold. The antibody was stored in 0.25 ml aliquots at −20° C.</p>
    <p>C. Affinity Measurements of he3 IgG for CD5</p>
    <p>The affinity of he3 IgG for CD5 was determined using Molt-4M cells, which express CD5 on their surface, and I<sup>125</sup>-labeled chimeric H65 IgG in a competitive binding assay. Culture supernatants from Clone C1705 and C1718 and purified IgG from C1705 were used as the sources of he3 IgG.</p>
    <p>For this assay, 20 μg of chimeric H65 IgG (cH65 IgG) was iodinated by exposure to 100 μl lactoperoxidase-glucose oxidase immobilized beads (Enzymobeads, BioRad), 100 μl of PBS, 1.0 mCi I<sup>125 </sup>(Amersham, IMS30), 50 μl of 55 mM b-D-glucose for 45 minutes at 23° C. The reaction was quenched by the addition of 20 μl of 105 mM sodium metabisulfite and 120 mM potassium iodine followed by centrifugation for 1 minute to pellet the beads. <sup>125</sup>I-cH65 IgG was purified by gel filtration using 7 mls of sephadex G25, using PBS (137 mM NaCl, 1.47 mM KH<sub>2</sub>PO<sub>4</sub>, 8.1 mM Na<sub>2</sub>HPO<sub>4</sub>, 2.68 mM KCl at pH 7.2-7.4) plus 0.1% BSA. <sup>125</sup>I-cH65 IgG recovery and specific activity were determined by TCA precipitation.</p>
    <p>Competitive binding was performed as follows: 100 μl of Molt-4M cells were washed two times in ice-cold DHB binding buffer (Dubellco's modified Eagle's medium (Gibco, 320-1965PJ), 1.0% BSA and 10 mM Hepes at pH 7.2.-7.4). Cells were resuspended in the same buffer, plated into 96 v-bottomed wells (Costar) at 3×10<sup>5 </sup>cells per well and pelleted at 4° C. by centrifugation for 5 min at 1,000 rpm using a Beckman JS 4.2 rotor; 50 μl of 2×-concentrated 0.1 nM <sup>125</sup>I-cH65 IgG in DHB was then added to each well and competed with 50 μl of 2×—concentrated cH65 IgG or humanized antibody in DHB at final antibody concentrations from 100 nM to 0.0017 nM. Humanized antibody was obtained from culture supernatants of Sp2/0 clone C1718 which expresses he3 IgG. The concentration of the antibody in the supernatants was established by ELISA using a chimeric antibody as a standard. The concentration of the antibody in the purified preparation was determined by binding was allowed to proceed at 4° C. for 5 hrs and was terminated by washing cells three times with 200 μl of DHB binding buffer by centrifugation for 5 min at 1,000 rpm. All buffers and operations were at 4° C. Radioactivity was determined by solubilizing cells in 100 μl of 1.0 M NaOH and counting in a Cobra II auto gamma counter (Packard) Data from binding experiments were analyzed by the weighted nonlinear least squares curve fitting program, MacLigand, a Macintosh version of the computer program “Ligand” from Munson, <i>Analyt. Biochem., </i>107:220 (1980). Objective statistical criteria (F, test, extra sum squares principle) were used to evaluate goodness of fit and for discriminating between models. Nonspecific binding was treated as a parameter subject to error and was fitted simultaneously with other parameters.</p>
    <p>Data showing relative binding of he3 and CE65 to CD5 on molt-4M cells in a competition binding assay demonstrate that the moderate-risk changes made in he3 IgG result in an antibody with a higher affinity than the chimeric mouse-human form of this antibody (cH65) for its target, CD5.</p>
    <heading>EXAMPLE 7</heading> <p>Preparation of Gelonin Immunoconjugates</p>
    <p>Gelonin analogs of the invention were variously conjugated to murine (ATCC HB9286) and chimeric H65 (cH65) antibody, cH65 antibody domains (including cFab, cFab′ and cF(ab′)<sub>2 </sub>fragments), and humanized antibodies and antibody domains, all of which are specifically reactive with the human T cell determinant CD5. H65 antibody was prepared and purified by methods described in U.S. patent application Ser. No. 07/306,433, supra and International Publication No. WO 89/06968, supra. Chimeric H65 antibody was prepared according to methods similar to those described in Robinson et al., <i>Human Antibodies and Hybridomas, </i>2:84-93 (1991), incorporated by reference herein. Chimeric H65 Fab, Fab′, and F(ab′)<sub>2 </sub>proteins were prepared as described in Better, et al., <i>Proc. Nat. Acad. Sci. </i>(<i>USA</i>), 90: 457-461 (1993), incorporated by reference herein. Finally, he3 humanized antibodies were prepared according to the procedures described in U.S. patent application Ser. No. 07/808,464, incorporated by reference herein.</p>
    <p>A. Conjugation to H65 Antibodies</p>
    <p>To expose a reactive sulfhydryl, the unpaired cysteine residues of the gelonin analogs were first reduced by incubation with 0.1 to 2 mM DTT (30-60 minutes at room temperature), and then were desalted by size-exclusion chromatography.</p>
    <p>Specifically, the Gel<sub>C248 </sub>analog (3.8 mg/ml) was treated with 2 mM DTT for 60 minutes in 0.1 M Naphosphate, 0.25 M NaCl, pH 7.5 buffer. The Gel<sub>C244 </sub>variant (7.6 mg/ml) was treated with 2 mM DTT for 30 minutes in 0.1 M Naphosphate, 0.25 M NaCl, pH 7.5 buffer. The Gel<sub>C247 </sub>analog (4 mg/ml) was treated with 2 mM DTT for 30 minutes in 0.1 M Naphosphate, 0.5 M NaCl, pH 7.5 buffer with 0.5 mM EDTA. The Gel<sub>C239 </sub>variant (3.2 mg/ml) was treated with 2 mM DTT for 30 minutes in 0.1 m Naphosphate, 0.5 M NaCl, pH 7.5 buffer with 0.5 mM EDTA. The Gel<sub>A50(C44) </sub>analog (4.2 mg/ml) was treated with 0.1 mM DTT for 30 minutes in 0.1 M Naphosphate, 0.1 M NaCl, pH 7.5 buffer with 0.5 mM EDTA. Lastly, the Gel<sub>C10 </sub>variant (3.1 mg/ml) was treated with 1 mM DTT for 20 minutes in 0.1 M Naphosphate, 0.1 M NaCl, pH 7.5 buffer with 1 mM EDTA.</p>
    <p>The presence of a free sulfhydryl was verified by reaction with DTNB and the average value obtained was 1.4±0.65 SH/molecule. No free thiols were detected in the absence of reduction.</p>
    <p>H65 antibody and chimeric H65 antibody were chemically modified with the hindered linker 5-methyl-2-iminothiolane (M2IT) at lysine residues to introduce a reactive sulfhydryl group as described in Goff et al., <i>Bioconjugate Chem., </i>1:381-386 (1990) and co-owned Carroll et al., U.S. Pat. No. 5,093,475, incorporated by reference herein.</p>
    <p>Specifically, for conjugation with Gel<sub>C248 </sub>and Gel<sub>C244</sub>, murine H65 antibody at 4 mg/mL was derivitized with 18× M2IT and 2.5 mM DTNB in 25 mM TEOA, 150 mM NaCl, pH 8 buffer for 1 hour at 23° C. The reaction gave 1.9 linkers per antibody as determined by DTNB assay.</p>
    <p>For conjugation with Gel<sub>C247 </sub>and Gel<sub>C239</sub>, H65 antibody at 4.7 mg/mL was derivitized with 20× M2IT and 2.5 mM DTNB in 25 mM TEOA 150 mM NaCl, pH 8 buffer for 50 minutes at 23° C. The reaction gave 1.6 linkers per antibody as determined by DTNB assay.</p>
    <p>Before reaction with Gel<sub>A50(C44)</sub>, H65 antibody at 5.8 mg/mL was derivitized with 20× m2IT and 2.5 mM DTNB in 25 mM TEOA, 150 mM NaCl, pH 8 buffer for 30 minutes at 23° C. The reaction gave 1.5 linkers per antibody as determined by DTNB assay.</p>
    <p>For conjugation with Ge<sub>C10</sub>, H65 antibody at 2.2 mg/mL was derivitized with 10× m2IT and 2.5 mM DTNB in 25 mM TEOA, 150 mM NaCl, pH 8 buffer for 1 hour at 23° C. The reaction gave 1.4 linkers per antibody as determined by DTNB assay.</p>
    <p>Chimeric H65 antibody was prepared for conjugation in a similar manner to murine H65 antibody.</p>
    <p>Two methods were initially compared for their effectiveness in preparing immunoconjugates with recombinant gelonin. First, the native disulfide bond in recombinant gelonin was reduced by the addition of 2 mM DTT at room temperature for 30 minutes. The reduced gelonin was recovered by size-exclusion chromatography on a column of Sephadex GF-05LS and assayed for the presence of free sulfhydryls by the DTNB assay. 1.4 free SH groups were detected. This reduced gelonin was then reacted with H65-(M2IT)-S-S-TNB (1.8 TNB groups/H65). Under these experimental conditions, little or no conjugate was prepared between reduced gelonin and thiol-activated H65 antibody.</p>
    <p>In contrast, when both the recombinant gelonin and the H65 antibody were first derivitized with the crosslinker M2IT (creating gelonin-(M2IT)-SH and H65-(M2IT)-S-S-TNB) and then mixed together, H65-(M2IT)-S-S-(M2IT)-gelonin conjugate was prepared in good yield (toxin/antibody ratio of 1.6). The starting materials for this conjugation (gelonin-(M2IT)-SH and H65-(M2IT)-S-S-TNB) contained linker/protein ratios of 1.2 and 1.4, respectively. Native gelonin was derivatized in a similar manner prior to conjugation to murine or chimeric H65 antibody.</p>
    <p>The reduced gelonin analogs were mixed with H65-(M2IT)-S-S-TNB to allow conjugation. The following conjugation reactions were set up for each analog: 23 mg (in 7.2 ml) of H65-M2IT-TNB were mixed with a 5-fold molar excess of Gel<sub>C248 </sub>(23 mg in 6 ml) for 2 hours at room temperature, then for 18 hours overnight at 4° C.; 23 mg (in 7.3 ml) of H65-m2IT-TNB were mixed with a 5-fold molar excess of Gel<sub>C244 </sub>(23 mg in 3 ml) for 3 hours at room temperature, then for 18 hours overnight at 4° C.; 9 mg (in 2.8 mL) of H65-m2IT-TNB were mixed with a 5-fold molar excess of Gel<sub>C247 </sub>(9 mg in 2.25 mL) for 2 hours at room temperature, then for 5 nights at 4° C.; 9 mg (in 2.8 mL) of H65-m2IT-TNB were mixed with a 5-fold molar excess of Gel<sub>C239 </sub>(9 mg in 2.6 mL) for 2 hours at room temperature, then at 4° C. for 3 days; 12 mg (in 1.9 mL) of H65-m2IT-TNB were mixed with a 5.6-fold molar excess of Gel<sub>A50(C44) </sub>(13.44 mg in 3.2 mL) for 4.5 hours at room temperature, then 4° C. overnight; and 11 mg of H65-m2IT-TNB were mixed with a 5-fold molar excess of Gel<sub>C10 </sub>(11 mg in 3.5 mL) for 4 hours at room temperature, then at 4° C. overnight.</p>
    <p>Following conjugation, unreacted M2IT linkers on the antibody were quenched with 1:1 mole cysteamine to linker for 15 minutes at room temperature. The quenched reaction solution was then loaded onto a gel filtration column [Sephadex G-150 (Pharmacia) in the case of Gel<sub>C248</sub>, Gel<sub>C247</sub>, Gel<sub>C244 </sub>and Gel<sub>C239 </sub>and an AcA-44 column (IBF Biotecnics, France) in the case of Gel<sub>A50(C44) </sub>and Gel<sub>C10</sub>]. The reactions were run over the gel filtration columns and eluted with 10 mM Tris, 0.15M NaCl pH 7. The first peak off each column was loaded onto Blue Toyopearl® resin (ToSoHaas, Philadelphia, Pa.) in 10 mM Tris, 30 mM NaCl, pH 7 and the product was eluted with 10 mM Tris, 0.5 M NaCl, pH 7.5.</p>
    <p>Samples of the final conjugation products were run on 5% non-reduced SDS PAGE, Coomassie stained and scanned with a Shimadzu laser densitometer to quantitate the number of toxins per antibody (T/A ratio). The yield of final product for each analog conjugate was as follows: Gel<sub>C248</sub>, 17 mg with a T/A ration of 1.6; Gel<sub>C247</sub>, 1.1 mg with a T/A ratio of 1; Gel<sub>C244</sub>, 4.5 mgs with a T/A ratio of 1.46; Gel<sub>C239</sub>, 2.9 mg with a T/A ratio of 2.4; Gel<sub>A50(C44)</sub>, 7.3 mg with a T/A ratio of 1.22; and Gel<sub>C10</sub>, 6.2 mg with a T/A ratio of 1.37. Conjugation efficiency (i.e., conversion of free antibody to immunoconjugate) was significantly greater (˜80%) for some analogs (Gel<sub>C10</sub>, Gel<sub>A50(C44)</sub>, Gel<sub>C239</sub>, Gel<sub>C247</sub>, and Gel<sub>C248</sub>) than for others (˜10%, Gel<sub>C244</sub>).</p>
    <p>B. Gelonin Immunoconjugates with Chimeric and Humanized Antibodies</p>
    <p>Analogs Gel<sub>C247</sub>, and Gel<sub>A50(C44)</sub>, were also conjugated to various chimeric [cH65Fab, cH65Fab′ and cH65F(ab′)<sub>2</sub>], and “human engineered” [he1 Fab, he2-Fab, he3-Fab, he1 Fab′ and he1 F(ab′)<sub>2</sub>], antibody fragments. Chimeric H65 antibody fragments may be prepared according to the methods described in International Publication No. WO 89/00999, supra. The DNA sequences encoding the variable regions of H65 antibody fragments that were human engineered (referring to the replacement of selected murine-encoded amino acids to make the H65 antibody sequences less immunogenic to humans) according to the methods described above in Example 5, are set out in SEQ ID NO: 69 (variable region of the kappa chain of he1 and he2), SEQ ID NO: 70 (variable region of the gamma chain of he1), SEQ ID NO: 71 (variable region of the gamma chain of he2 and he3) and SEQ ID NO: 72 (variable region of the kappa chain of he3).</p>
    <p>The chimeric H65 antibody fragments were conjugated to the Gel<sub>C247 </sub>analog in the same manner as described below for conjugation of human engineered Fab and Fab′ fragments to Gel<sub>C247 </sub>and Gel<sub>A50(C44)</sub>.</p>
    <p>(i) he1 Fab-Gel<sub>C247 </sub> </p>
    <p>The he1 Fab was dialyzed into 25 mM TEOA buffer, 250 mM NaCl, pH 8 and then concentrated to 6.8 mg/mL prior to derivitization with the M2IT crosslinker. For the linker reaction, M2IT was used at 20-fold molar excess, in the presence of 2.5 mM DTNB. The reaction was allowed to proceed for 30 minutes at room temperature, then desalted on GF05 (gel filtration resin) and equilibrated in 0.1 M Na phosphate, 0.2M NaCl, pH 7.5. A linker number of 1.8 linkers per Fab was calculated based on the DTNB assay. The he1 Fab-M2IT-TNB was concentrated to 3.7 mg/mL prior to conjugation with Gel<sub>C247</sub>.</p>
    <p>Gel<sub>C247 </sub>at 12.8 mg/mL in 10 mM Na phosphate, 0.3M NaCl, was treated with 1 mM DTT, 0.5 mM EDTA for 20 minutes at room temperature to expose a reactive sulfhydryl for conjugation and then was desalted on GF05 and equilibrated in 0.1 M Na phosphate, 0.2 M NaCl, pH 7.5. Free thiol content was determined to be 0.74 moles of free SH per mole of Gel<sub>C247 </sub>using the DTNB assay. The gelonin was concentrated to 8.3 mg/mL prior to conjugation with activated antibody.</p>
    <p>The conjugation reaction between the free thiol on Gel<sub>C247 </sub>and the derivitized he1 Fab-M2IT-TNB, conditions were as follows. A 5-fold excess of the gelonin analog was added to activated he1 Fab-M2IT-TNB (both proteins were in 0.1M Na phosphate, 0.2M NaCl, pH7.5) and the reaction mixture was incubated for 3.5 hours at room temperature and then overnight at 4° C. Following conjugation, untreated M2IT linkers were quenched with 1:1 mole cysteamine to linker for 15 minutes at room temperature. The quenched reaction solution was loaded onto a gel filtration column (G-75) equilibrated with 10 mM Tris, 150 mM NaCl, pH 7. The first peak off this column was diluted to 30 mM NaCl with 10 mM Tris, pH7 and loaded on Blue Toyopearl®. The product was eluted with 10 mM Tris, 0.5 M NaCl, pH 7.5.</p>
    <p>(ii) he1 Fab′-Gel<sub>C247 </sub> </p>
    <p>Similarly, the H65 he1 Fab′ fragment was dialyzed into 25 mM TEOA buffer, 400 mM NaCl, pH 8 at 2.9 mg/mL prior to derivitization with the M2IT crosslinker. For the linker reaction, M2IT was used at 20-fold molar excess, in the presence of 2.5 mM DTNB. The reaction was allowed to proceed for 1 hour at room temperature then it was desalted on GF05 (gel filtration resin) and equilibrated in 0.1 M Na phosphate, 0.2 M NaCl, pH 7.5. A linker number of 1.6 linkers per Fab′ was calculated based on the DTNB assay. The he1 Fab′-M2IT-TNB was concentrated to 3.7 mg/mL prior to conjugation with Gel<sub>C247</sub>.</p>
    <p>The Ge<sub>C247 </sub>at 77 mg/mL was diluted with 10 mM Na phosphate, 0.1 M NaCl to a concentration of 5 mg/mL, treated with 1 mM DTT, 0.5 mM EDTA for 30 minutes at room temperature to expose a free thiol for conjugation and then was desalted on GF05 and equilibrated in 0.1 M Na phosphate, 0.2 M NaCl, pH 7.5. Free thiol content was determined to be 1.48 moles of free SH per mole of Gel<sub>C247 </sub>using the DTNB assay. The Gel<sub>C247 </sub>was concentrated to 10 mg/mL prior to conjugation with activated he1 Fab′-M2IT-TNB.</p>
    <p>For the reaction between the free thiol on Gel<sub>C247 </sub>and the derivitized he1 Fab′-M2IT-TNB, conditions were as follows. A 5.7-fold molar excess of gelonin was added to activated hel Fab′-M2IT-TNB and the final salt concentration was adjusted to 0.25 M. The reaction mix was incubated for 1.5 hours at room temperature and then over the weekend at 4° C. Following conjugation, unreacted M2IT linkers were quenched with 1:1 mole cysteamine to linker for 15 minutes at room temperature. The quenched reaction solution was loaded onto a gel filtration column (AcA54) equilibrated with 10 mM Tris, 250 mM NaCl, pH 7.5. The first peak off this column was diluted to 20 mM NaCl with 10 mM Tris, pH 7 and loaded on Blue Toyopearl® which was equilibrated in 10 mM Tris, 20 mM NaCl, pH 7. The column was then washed with 10 mM Tris, 30 mM NaCl, pH 7.5. The product was eluted with 10 mM Tris, 1 M NaCl, pH 7.5.</p>
    <p>(iii) he2-Fab Gel<sub>A50(C44) </sub> </p>
    <p>The he2-Fab was dialyzed overnight into 25 mM TEOA, 0.25 M NaCl, pH 8 buffer and then concentrated to 13.3 mg/mL prior to derivitization with the M2IT crosslinker. For the linker reaction, M2IT was used in a 20-fold molar excess in the presence of 2.5 mM DTNB. The reaction was allowed to proceed for 20 minutes at room temperature and was then desalted on a GF05-LS (gel filtration) column, equilibrated in 0.1 M Na phosphate, 0.2 M NaCl with 0.02% Na azide. A linker number of 1.7 linkers per Fab-M2IT-TNB was calculated based on the DTNB assay. After derivitization and gel filtration, the he2-Fab concentration was 5.2 mg/mL.</p>
    <p>Gel<sub>A50(C44) </sub>at 8.33 mg/mL in 10 mM Na phosphate, pH 7.2 was treated with 5 mM DTT and 0.5 mM EDTA for 30 minutes at room temperature to expose a reactive thiol for conjugation and then was desalted on GF05-LS resin equilibrated in 0.1 M Na phosphate, 0.1 M NaCl with 0.5 mM EDTA plus 0.02% Na azide, pH 7.5. Free thiol content was determined to be 0.83 moles of free SH per mole of Gel<sub>A50(C44) </sub>using the DTNB assay. The gelonin was concentrated to 11.4 mg/mL prior to conjugation with activated he2-Fab.</p>
    <p>The conjugation reaction conditions between the free thiol on Gel<sub>A50(C44) </sub>and the derivitized he2-Fab-M2IT-TNB were as follows. A 3-fold excess of the gelonin analog was added to activated he2-Fab-M2IT-TNB (both proteins were in 0.1 M Na phosphate, 0.1 M NaCl, pH 7.5 but the gelonin solution contained 0.5 mM EDTA as well). The reaction mixture was concentrated to half its original volume, then the mixture was incubated for 4 hours at room temperature followed by 72 hours at 4° C. Following the incubation period the efficiency of conjugation was estimated at 70-75% by examination of SDS PAGE.</p>
    <p>Following conjugation the excess M2IT linkers were quenched by incubation with 1:1 mole cysteamine to linker for 15 minutes at room temperature. The quenched reaction as loaded onto a gel filtration column (G-75) equilibrated in 10 mM Tris, 0.15 M NaCl, pH 7. The first peak off this column was diluted to 30 mM NaCl with 10 mM Tris, pH 7 and loaded onto a Blue Toyopearl® (TosoHaas) column. The product was eluted with 10 mM Tris, 1 M NaCl, pH 7.5.</p>
    <p>(iv) he3-Fab Gel<sub>A50(C44) </sub> </p>
    <p>Similarly, the he3-Fab was dialyzed overnight into 25 mM TEOA, 0.25 M NaCl, pH 8 buffer and then concentrated to 5 mg/mL prior to derivitization with the M2IT crosslinker. For the linker reaction, M2IT was used in a 10-fold molar excess in the presence of 2.5 mM DTNB. The reaction was allowed to proceed for 45 minutes at room temperature and was then desalted on a GF05-LS (gel filtration) column, equilibrated in 0.1 M Na phosphate, 0.2 M NaCl with 0.02% Na azide. A linker number of 1 M2IT per Fab-M2IT-TNB was calculated based on the DTNB assay. After derivitization and gel filtration, the he3-Fab concentration was 5.3 mg/mL.</p>
    <p>Gel<sub>A50(C44) </sub>at 7.8 mg/mL in 0.1 M Na phosphate, 0.1 M NaCl, pH 7.5 was treated with 1.5 mM DTT and 1 mM EDTA for 30 minutes at room temperature to expose a reactive thiol for conjugation and then was desalted on GF05-LS resin equilibrated in 0.1 M Na phosphate, 0.1 M NaCl plus 0.02% Na azide, pH 7.5. Free thiol content was determined to be 0.66 moles of free SH per mole of Gel<sub>A50(C44) </sub>using the DTNB assay. The gelonin was concentrated to 5.2 mg/mL prior to conjugation with activated he3-Fab.</p>
    <p>The conjugation reaction conditions between the free thiol on Gel<sub>A50(C44) </sub>and the derivitized he3-Fab-M2IT-TN were as follows. A 5-fold excess of the gelonin analog was added to activated he3-Fab-M2IT-TNB (both proteins were in 0.1 M Na phosphate 0.1 M NaCl, pH 7.5). The reaction mixture was incubated for 2 hours at room temperature followed by 72 hours at 4° C. Following the incubated period the efficiency of conjugation was estimated at 70-75% by examination of SDS PAGE.</p>
    <p>Following conjugation, the excess M2IT linkers were quenched by incubation with 1:1 mole cysteamine to linker for 15 minutes at room temperature. The quenched reaction was loaded onto a GammaBind G (immobilized protein G affinity resin, obtained from Genex, Gaithersburg, Md.) equilibrated in 10 mM Na phosphate, 0.15 M NaCl, pH 7. It was eluted with 0.5 M NaOAc, pH 3 and neutralized with Tris. It was dialyzed into 10 mM Tris, 0.15 M NaCl, pH 7 overnight, then diluted to 30 mM NaCl with 10 mM Tris, pH 7 and loaded onto a blue Toyopearl® (TosoHaas) column. The product was eluted with 10 mM Tris, 1 M NaCl, pH 7.5.</p>
    <heading>EXAMPLE 8</heading> <p>Whole Cell Kill Assays</p>
    <p>Immunoconjugates prepared with gelonin and gelonin analogs were tested for cytotoxicity against an acute lymphoblastoid leukemia T cell line (HSB2 cells) and against human peripheral blood mononuclear cells (PBMCs). Immunoconjugates of ricin A-chain with H65 antibody (H65-RTA) and antibody fragments were also tested. The ricin A-chain (RTA) as well as the H65-RTA immunoconjugates were prepared and purified according to methods described in U.S. patent application Ser. No. 07/306,433, supra and in International Publication No. WO 89/06968, supra.</p>
    <p>Briefly, HSB2 cells were incubated with immunotoxin and the inhibition of protein synthesis in the presence of immunotoxin was measured relative to untreated control cells. The standard immunoconjugates H65-RTA (H65 derivitized with SPDP linked to RTA), H65-Gelonin and H65-rGelonin, H65 fragment immunoconjugate, and gelonin immunoconjugate samples were diluted with RPMI without leucine at half-log concentrations ranging from 2000 to 0.632 ng/ml. All dilutions were added in triplicate to wells of microtiter plates containing 1×10<sup>5 </sup>HSB2 cells per well. HSB2 plates were incubated for 20 hours at 37° C. and then pulsed with <sup>3</sup>H-Leu for 4 hours before harvesting. Samples were counted on the Inotec Trace 96 cascade ionization counter. By comparison with an untreated sample, the picomolar concentration (pM) of immunotoxin which resulted in a 50% inhibition of protein synthesis (IC<sub>50</sub>) was calculated. In order to normalize for conjugates containing differing amounts of toxin or toxin analog, the cytotoxicity data were converted to picomolar toxin (pM T) by multiplying the conjugate IC<sub>50 </sub>(in pM) by the toxin/antibody ratio which is unique to each conjugate preparation.</p>
    <p>The PMBC assays were performed as described by Fishwild et al., <i>Clin. and Exp. Immunol., </i>86:506-513 (1991) and involved the incubation of immunoconjugates with PBMCs for a total of 90 hours. During the final 16 hours of incubation, <sup>3</sup>H-thymidine was added; upon completion, immunoconjugate-induced inhibition of DNA synthesis was quantified. The activities of the H65 and chimeric H65 antibody conjugates against HSB2 cells and PBMC cells are listed in Table 2 below.</p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 2</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">IC<sub>50 </sub>(pM T)</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="4"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="119pt" align="left"> </colspec> <colspec colname="2" colwidth="42pt" align="center"> </colspec> <colspec colname="3" colwidth="42pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Conjugate</td>
                <td class="description-td">HSB2 Cells</td>
                <td class="description-td">PBMCs</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="OFFSET" nameend="3" align="center" rowsep="1" class="description-td" colspan="4"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="4"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="119pt" align="left"> </colspec> <colspec colname="2" colwidth="42pt" align="char" char="."> </colspec> <colspec colname="3" colwidth="42pt" align="char" char="."> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">H65-RTA</td>
                <td class="description-td">143</td>
                <td class="description-td">459</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">H65-(M2IT)-S-S-(M2IT)-Gelonin</td>
                <td class="description-td">1770</td>
                <td class="description-td">81</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">H65-(M2IT)-S-S-(M2IT)-rGelonin</td>
                <td class="description-td">276</td>
                <td class="description-td">75</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">H65-(M2IT)-S-S-Gel<sub>C10</sub> </td>
                <td class="description-td">140</td>
                <td class="description-td">28</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">H65-(M2IT)-S-S-Gel<sub>A50(C44)</sub> </td>
                <td class="description-td">99</td>
                <td class="description-td">51</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">H65-(M2IT)-S-S-Gel<sub>C239</sub> </td>
                <td class="description-td">2328</td>
                <td class="description-td">180</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">H65-(M2IT)-S-S-Gel<sub>C244</sub> </td>
                <td class="description-td">&gt;5000</td>
                <td class="description-td">&gt;2700</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">H65-(M2IT)-S-S-Gel<sub>C247</sub> </td>
                <td class="description-td">41</td>
                <td class="description-td">35</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">H65-(M2IT)-S-S-Gel<sub>C248</sub> </td>
                <td class="description-td">440</td>
                <td class="description-td">203</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">cH65-RTA<sub>30</sub> </td>
                <td class="description-td">60</td>
                <td class="description-td">400</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">cH65-(M2IT)-S-S-(M2IT)-Gelonin</td>
                <td class="description-td">1770</td>
                <td class="description-td">140</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">cH65-(M2IT)-S-S-(M2IT)-rGelonin</td>
                <td class="description-td">153</td>
                <td class="description-td">120</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">cH65-(M2IT)-S-S-Gel<sub>C239</sub> </td>
                <td class="description-td">&gt;7000</td>
                <td class="description-td">290</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">cH65-(M2IT)-S-S-Gel<sub>C247</sub> </td>
                <td class="description-td">34</td>
                <td class="description-td">60</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">cH65-(M2IT)-S-S-Gel<sub>C248</sub> </td>
                <td class="description-td">238</td>
                <td class="description-td">860</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">H65-(M2IT)-S-S-Gel<sub>A44(C50)</sub> </td>
                <td class="description-td">338</td>
                <td class="description-td">ND*</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">H65-(M2IT)-S-S-Gel<sub>C247A44A50</sub> </td>
                <td class="description-td">71</td>
                <td class="description-td">ND*</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="OFFSET" nameend="3" align="center" rowsep="1" class="description-td" colspan="4"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="OFFSET" nameend="3" align="left" class="description-td" colspan="4">*Not determined. </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p>Against HSB2 cells, many of the gelonin analog immunoconjugates were significantly more potent than conjugates prepared with native gelonin or recombinant, unmodified gelonin, both in terms of a low IC<sub>50 </sub>value, but also in terms of a greater extent of cell kill. Against human PBMCs, the gelonin analog conjugates were at least as active as native and recombinant gelonin conjugates. Importantly, however, some of the conjugates (for example, Gel<sub>C10</sub>, Gel<sub>A50(C44) </sub>and Gel<sub>C247</sub>) exhibited an enhanced potency against PBMCs compared to native and recombinant gelonin conjugates, and also exhibited an enhanced level of cell kill.</p>
    <p>The activities of the H65 antibody fragment conjugates against HSB2 cells and PBMC cells are listed in Tables 3 and 4 below, wherein extent of kill in Table 3 refers to the percentage of protein synthesis inhibited in HSB2 cells at the highest immunotoxin concentration tested (1 μg/ml).</p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 3</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">IC<sub>50 </sub>(pM T)</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="4"> <colspec colname="offset" colwidth="21pt" align="left"> </colspec> <colspec colname="1" colwidth="84pt" align="left"> </colspec> <colspec colname="2" colwidth="42pt" align="center"> </colspec> <colspec colname="3" colwidth="70pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Conjugate</td>
                <td class="description-td">HSB2 Cells</td>
                <td class="description-td">PBMCs</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="4"> <colspec colname="offset" colwidth="21pt" align="left"> </colspec> <colspec colname="1" colwidth="84pt" align="left"> </colspec> <colspec colname="2" colwidth="42pt" align="char" char="."> </colspec> <colspec colname="3" colwidth="70pt" align="char" char="."> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">cH65Fab′-RTA 30</td>
                <td class="description-td">530</td>
                <td class="description-td">1800</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">cH65Fab′-rGelonin</td>
                <td class="description-td">135</td>
                <td class="description-td">160</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">cH65Fab′-Gel<sub>C247</sub> </td>
                <td class="description-td">48</td>
                <td class="description-td">64</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">cH65F(ab′)<sub>2</sub>-RTA 30</td>
                <td class="description-td">33</td>
                <td class="description-td">57</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">cH65F(ab′)<sub>2</sub>-rGelonin</td>
                <td class="description-td">55</td>
                <td class="description-td">34</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">cH65F(ab′)<sub>2</sub>-Gel<sub>C247</sub> </td>
                <td class="description-td">23</td>
                <td class="description-td">20</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">cH65F(ab′)<sub>2</sub>-Gel<sub>C248</sub> </td>
                <td class="description-td">181</td>
                <td class="description-td">95</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="OFFSET" nameend="3" align="center" rowsep="1" class="description-td" colspan="4"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 4</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">IC<sub>50 </sub>(pM T)</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="4"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="84pt" align="left"> </colspec> <colspec colname="2" colwidth="42pt" align="center"> </colspec> <colspec colname="3" colwidth="77pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Conjugate</td>
                <td class="description-td">HSB2 Cells</td>
                <td class="description-td">Extent of Kill</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="OFFSET" nameend="3" align="center" rowsep="1" class="description-td" colspan="4"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="4"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="84pt" align="left"> </colspec> <colspec colname="2" colwidth="42pt" align="char" char="."> </colspec> <colspec colname="3" colwidth="77pt" align="char" char="."> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">he1 Fab′-Gel<sub>C247</sub> </td>
                <td class="description-td">57.7</td>
                <td class="description-td">93%</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">he1 Fab-Gel<sub>C247</sub> </td>
                <td class="description-td">180.0</td>
                <td class="description-td">94%</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">he2-Fab-Gel<sub>A50(C44)</sub> </td>
                <td class="description-td">363.0</td>
                <td class="description-td">91%</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">he3-Fab-Gel<sub>A50(C44)</sub> </td>
                <td class="description-td">191.0</td>
                <td class="description-td">93%</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">cH65Fab′-Gel<sub>C247</sub> </td>
                <td class="description-td">47.5</td>
                <td class="description-td">93%</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">cH65F(ab′)<sub>2</sub>-rGelonin</td>
                <td class="description-td">45.4</td>
                <td class="description-td">85%</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">cH65F(ab′)<sub>2</sub>-Gel<sub>C247</sub> </td>
                <td class="description-td">77.5</td>
                <td class="description-td">83%</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">cH65F(ab′)<sub>2</sub>-Gel<sub>C247</sub> </td>
                <td class="description-td">23.2</td>
                <td class="description-td">85%</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="OFFSET" nameend="3" align="center" rowsep="1" class="description-td" colspan="4"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p>The data in Table 3 show that monovalent (Fab or Fab′) fragments conjugated to various forms of gelonin are more potent than RTA conjugates. Table 4 shows that the human-engineered gelonin-Fab conjugates exhibit a very high degree of extent of kill.</p>
    <heading>EXAMPLE 9</heading> <p>Properties of Gelonin Immunoconjugates</p>
    <p>A. Solubility</p>
    <p>Recombinant gelonin and the gelonin analogs exhibited enhanced solubility in comparison to both native gelonin and RTA30. In addition, recombinant gelonin and gelonin analog immunoconjugates exhibited enhanced solubility relative to immunoconjugates prepared with native gelonin and RTA30. This enhanced solubility was particularly noteworthy for recombinant gelonin and analog conjugates prepared with chimeric Fab fragments.</p>
    <p>B. Disulfide Bond Stability Assay</p>
    <p>The stability of the disulfide bond linking a RIP to a targeting molecule (such as an antibody) is known to influence the lifespan of immunoconjugates in vivo [See Thorpe et al., <i>Cancer Res., </i>47:5924-5931 (1987), incorporated by reference herein]. For example, conjugates in which the disulfide bond is easily broken by reduction in vitro are less stable and less efficacious in animal models [See Thorpe at al., <i>Cancer Res., </i>48:6396-6403 (1988), incorporated by reference herein].</p>
    <p>Immunoconjugates prepared with native gelonin, recombinant gelonin and gelonin analogs were therefore examined in an in vitro disulfide bond stability assay similar to that described in Wawrzynczak et al., <i>Cancer Res., </i>50:7519-7526 (1990), incorporated by reference herein. Conjugates were incubated with increasing concentrations of glutathione for 1 hour at 37° C. and, after terminating the reaction with iodoacetamide, the amount of RIP released was quantitated by size-exclusion HPLC on a TosoHaas TSK-G2000SW column.</p>
    <p>By comparison with the amount of RIP released by high concentrations of 2-mercaptoethanol (to determine 100% release), the concentration of glutathione required to release 50% of the RIP (the RC<sub>50</sub>) was calculated. The results of assays for H65 antibody conjugates are set out in Table 5 below.</p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="offset" colwidth="21pt" align="left"> </colspec> <colspec colname="1" colwidth="112pt" align="left"> </colspec> <colspec colname="2" colwidth="84pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="OFFSET" nameend="2" rowsep="1" class="description-td" colspan="3">TABLE 5</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="OFFSET" nameend="2" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Conjugate</td>
                <td class="description-td">RC<sub>50 </sub>(mM)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="OFFSET" nameend="2" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="offset" colwidth="21pt" align="left"> </colspec> <colspec colname="1" colwidth="112pt" align="left"> </colspec> <colspec colname="2" colwidth="84pt" align="char" char="."> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">H65-RTA 30</td>
                <td class="description-td">3.2</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">H65-(M2IT)-S-S-(M2IT)-gelonin</td>
                <td class="description-td">11.1</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">H65-(M2IT)-S-S-(M2IT)-rGelonin</td>
                <td class="description-td">3.0</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">H65-(M2IT)-S-S-Gel<sub>C10</sub> </td>
                <td class="description-td">2.5</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">H65-(M2IT)-S-S-Gel<sub>A50(C44)</sub> </td>
                <td class="description-td">0.6</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">H65-(M2IT)-S-S-Gel<sub>C239</sub> </td>
                <td class="description-td">774.0</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">H65-(M2IT)-S-S-Gel<sub>C244</sub> </td>
                <td class="description-td">1.2</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">H65-(M2IT)-S-S-Gel<sub>C247</sub> </td>
                <td class="description-td">0.1</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">H65-(M2IT)-S-S-Gel<sub>C248</sub> </td>
                <td class="description-td">0.4</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">cH65-RTA 30</td>
                <td class="description-td">2.50</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">cH65-(M21T)-S-S-(M2IT)-rGelonin</td>
                <td class="description-td">2.39</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">cH65-(M21T)-S-S-Gel<sub>C247</sub> </td>
                <td class="description-td">0.11</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">cH65-(M2IT)-S-S-Gel<sub>C248</sub> </td>
                <td class="description-td">0.32</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">H65-(M2IT)-S-S-Gel<sub>A44(C50)</sub> </td>
                <td class="description-td">9.2</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">H65-(M2IT)-S-S-Gel<sub>C247A44A50</sub> </td>
                <td class="description-td">0.3</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="OFFSET" nameend="2" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p>The foregoing results indicate that the stability of the bonds between the different gelonin proteins and H65 antibody varied greatly. With the exception of Gel<sub>C10 </sub>and Gel<sub>C239</sub>, most of the gelonin analogs resulted in conjugates with linkages that were somewhat less stable in the in vitro assay than the dual-linker chemical conjugate. The stability of the Gel<sub>C239 </sub>analog, however, was particularly enhanced.</p>
    <p>The results of the assay for H65 antibody fragment conjugates are set out in Table 6 below.</p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="offset" colwidth="42pt" align="left"> </colspec> <colspec colname="1" colwidth="63pt" align="left"> </colspec> <colspec colname="2" colwidth="112pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="OFFSET" nameend="2" rowsep="1" class="description-td" colspan="3">TABLE 6</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="OFFSET" nameend="2" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Conjugate</td>
                <td class="description-td">RC<sub>50 </sub>(mM)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="OFFSET" nameend="2" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="offset" colwidth="42pt" align="left"> </colspec> <colspec colname="1" colwidth="63pt" align="left"> </colspec> <colspec colname="2" colwidth="112pt" align="char" char="."> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">he1 Fab′-Gel<sub>C247</sub> </td>
                <td class="description-td">0.07</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">cFab′-Gelonin</td>
                <td class="description-td">1.27</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">cFab′-Gel<sub>C247</sub> </td>
                <td class="description-td">0.08</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">cF(ab′)<sub>2</sub>-RTA 30</td>
                <td class="description-td">1.74</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">cF(ab′)<sub>2</sub>-rGelonin</td>
                <td class="description-td">2.30</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">cF(ab′)<sub>2</sub>-Gel<sub>C247</sub> </td>
                <td class="description-td">0.09</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">cF(ab′)<sub>2</sub>-Gel<sub>C248</sub> </td>
                <td class="description-td">0.32</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">he2-Fab-Gel<sub>A50(C44)</sub> </td>
                <td class="description-td">0.46</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">he3-Fab-Gel<sub>A50(C44)</sub> </td>
                <td class="description-td">0.58</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="OFFSET" nameend="2" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p>From the RC<sub>50 </sub>results presented in Tables 5 and 6, it appears that the particular RIP analog component of each immunotoxin dictates the stability of the immunotoxin disulfide bond in vitro.</p>
    <heading>EXAMPLE 10</heading> <p>Pharmacokinetics of Conjugates to H65 Antibody</p>
    <p>The pharmacokinetics of gelonin analogs Gel<sub>C247</sub>, Gel<sub>A50(C44)</sub>, and Gel<sub>C10 </sub>linked to whole H65 antibody was investigated in rats. An IV bolus of 0.1 mg/kg of <sup>125</sup>I-labelled immunoconjugate H65-(M2IT)-S-S-Gel<sub>C247</sub>, H65-(M2IT)-S-S-Gel<sub>A50(C44) </sub>or H65-(M2IT)-S-S-Gel<sub>C10 </sub>was administered to male Sprague-Dawley rats weighing 134-148 grams. Serum samples were collected from the rats at 3, 15, 30 and 45 minutes, and at 1.5, 2, 4, 6, 8, 18, 24, 48, 72, and 96 hours. Radioactivity (cpm/ml) of each sample was measured, and SDS-PAGE was performed to determine the fraction of radioactivity associated with whole immunoconjugate. Immunoconjugate-associated serum radioactivity was analyzed using the computer program PCNONLIN (SCI Software, Lexington, Ky.). Table 7 below lists the pharmacokinetic parameters of the immunoconjugates. In that table, the standard error for each value is indicated and a one way analysis of variance is presented, IC is the immunoconjugate (specified by the abbreviation for the gelonin variant that is part of the immunoconjugate), n is the number of animals in the study, Vc is the central volume of distribution, Cl is the clearance, MRT is the total body mean residence time, Alpha is the α half-life and Beta is the β half-life of the immunoconjugate.</p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="6"> <colspec colname="1" colwidth="56pt" align="left"> </colspec> <colspec colname="2" colwidth="28pt" align="left"> </colspec> <colspec colname="3" colwidth="35pt" align="left"> </colspec> <colspec colname="4" colwidth="28pt" align="left"> </colspec> <colspec colname="5" colwidth="35pt" align="left"> </colspec> <colspec colname="6" colwidth="35pt" align="left"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="6" rowsep="1" class="description-td" colspan="6">TABLE 7</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="6" align="center" rowsep="1" class="description-td" colspan="6"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Vc</td>
                <td class="description-td">Cl</td>
                <td class="description-td">MRT</td>
                <td class="description-td">Alpha</td>
                <td class="description-td">Beta</td>
              </tr> <tr class="description-tr"> <td class="description-td">IC</td>
                <td class="description-td">(ml/kg)</td>
                <td class="description-td">(ml/hr/kg)</td>
                <td class="description-td">(hours)</td>
                <td class="description-td">(hours)</td>
                <td class="description-td">(hours)</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="6" align="center" rowsep="1" class="description-td" colspan="6"> </td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">H65 Gel<sub>C247</sub> </td>
                <td class="description-td">65.3 ±</td>
                <td class="description-td">11.0 ± 0.4</td>
                <td class="description-td">16.5 ±</td>
                <td class="description-td">2.3 ± 0.2</td>
                <td class="description-td">20.5 ± 3.0</td>
              </tr> <tr class="description-tr"> <td class="description-td">n = 32</td>
                <td class="description-td">3.4</td>
                <td class="description-td"> </td>
                <td class="description-td">1.9</td>
              </tr> <tr class="description-tr"> <td class="description-td">H65 Gel<sub>A50(C44)</sub> </td>
                <td class="description-td">61.9 ±</td>
                <td class="description-td">4.1 ± 0.1</td>
                <td class="description-td">22.7 ±</td>
                <td class="description-td">3.0 ± 0.7</td>
                <td class="description-td">17.8 ± 0.8</td>
              </tr> <tr class="description-tr"> <td class="description-td">n = 38</td>
                <td class="description-td">2.4</td>
                <td class="description-td"> </td>
                <td class="description-td">0.7</td>
              </tr> <tr class="description-tr"> <td class="description-td">H65 Gel<sub>C10</sub> </td>
                <td class="description-td">59.2 ±</td>
                <td class="description-td">2.5 ± 0.04</td>
                <td class="description-td">42.7 ±</td>
                <td class="description-td">3.3 ± 0.3</td>
                <td class="description-td">32.9 ± 1.1</td>
              </tr> <tr class="description-tr"> <td class="description-td">n = 45</td>
                <td class="description-td">1.3</td>
                <td class="description-td"> </td>
                <td class="description-td">1.1</td>
              </tr> <tr class="description-tr"> <td class="description-td">p-value</td>
                <td class="description-td">0.176</td>
                <td class="description-td">&lt;0.0001</td>
                <td class="description-td">&lt;0.0001</td>
                <td class="description-td">0.303</td>
                <td class="description-td">&lt;0.0001</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="6" align="center" rowsep="1" class="description-td" colspan="6"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p>The Gel<sub>C247 </sub>immunoconjugate was found to have α and β half lives of 2.3 and 20 hours, with a total mean residence time of 17 hours. The 72 and 96 hour time points were excluded from analysis because of the poor resolution of immunoconjugate associated radioactivity on the SDS-PAGE gel for these serum samples.</p>
    <p>Because in vitro studies suggested that the Gel<sub>C10 </sub>immunoconjugate had greater disulfide bond stability it was anticipated that its half lives in vivo would be longer relative to the cys<sub>247 </sub>form of the immunoconjugate. The β half life of the immunoconjugate was about 33 hours compared to 20 hours for the Gel<sub>C247 </sub>conjugate. The total mean residence time was also much greater for the Gel<sub>C10 </sub>immunoconjugate (42 hours versus 42 hours for the Gel<sub>247 </sub>conjugate). In addition, the clearance of the Gel<sub>C10 </sub>immunoconjugate was 2.5 ml/hr/kg, about four times less than that of the Gel<sub>C247 </sub>immunoconjugate (11 ml/hr/kg). As also predicted from the in vitro disulfide stability data, the clearance of the Gel<sub>A50(C44) </sub>immunoconjugate was intermediate between those of the Gel<sub>C10 </sub>and Gel<sub>C247 </sub>immunoconjugates.</p>
    <p>Based on these studies, the Gel<sub>C10 </sub>analog conjugated to H65 antibody has greater in vivo stability than the Gel<sub>A50(C44) </sub>and Gel<sub>C247 </sub>analogs conjugated to H65 antibody (as determined by the longer mean residence time and clearance rates), although the properties of the Gel<sub>A50(C44) </sub>immunoconjugate more closely resembled those of the Gel<sub>C10 </sub>immunoconjugate than the Gel<sub>C247 </sub>immunoconjugate.</p>
    <heading>EXAMPLE 11</heading> <p>Pharmacokinetics of Conjugates to H65 Antibody Fragments</p>
    <p>The pharmacokinetics of Gel<sub>C247 </sub>and Gel<sub>A50(C44) </sub>analogs linked to human engineered H65 Fab fragments were also investigated in rats. An IV bolus of 0.1 mg/kg of <sup>125</sup>l-labelled he1 H65 Fab-Gel<sub>C247</sub>, he2 H65 Fab-Gel<sub>A50(C44) </sub>or he3 H65 Fab-Gel<sub>A50(C44) </sub>was administered to male Sprague-Dawley rats weighing 150-180 grams. Serum samples were collected at 3, 5, 15, 20, 30, and 40 minutes, and 1, 1.5, 3, 6, 8, 18, 24, 32, 48, and 72 hours, and were analyzed by ELISA using rabbit anti-Gelonin antibody as the capture antibody and biotin-labelled goat anti-human kappa light chain antibody as the secondary antibody. Results of the analysis are presented in Table 8 below. In the table, the standard error for each value is shown, and IC is the immunoconjugate, n is the number of animals in the study, Vc is the central volume of distribution, Vss is the steady state volume of distribution, Cl is the clearance, MRT is the total body mean residence time, Alpha is the α half-life and Beta is the β half-life of the indicated conjugate.</p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="7"> <colspec colname="1" colwidth="42pt" align="left"> </colspec> <colspec colname="2" colwidth="28pt" align="left"> </colspec> <colspec colname="3" colwidth="28pt" align="left"> </colspec> <colspec colname="4" colwidth="28pt" align="left"> </colspec> <colspec colname="5" colwidth="28pt" align="left"> </colspec> <colspec colname="6" colwidth="35pt" align="left"> </colspec> <colspec colname="7" colwidth="28pt" align="left"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="7" rowsep="1" class="description-td" colspan="7">TABLE 8</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="7" align="center" rowsep="1" class="description-td" colspan="7"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">Vss</td>
                <td class="description-td">Cl</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">VC</td>
                <td class="description-td">(ml/</td>
                <td class="description-td">(ml/</td>
                <td class="description-td">MRT</td>
                <td class="description-td">Alpha</td>
                <td class="description-td">Beta</td>
              </tr> <tr class="description-tr"> <td class="description-td">IC</td>
                <td class="description-td">(ml/kg)</td>
                <td class="description-td">hr/kg)</td>
                <td class="description-td">hr/kg)</td>
                <td class="description-td">(hours)</td>
                <td class="description-td">(hours)</td>
                <td class="description-td">(hours)</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="7" align="center" rowsep="1" class="description-td" colspan="7"> </td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">hel Gel<sub>C247</sub> </td>
                <td class="description-td">48 ± 3</td>
                <td class="description-td">133 ± 7</td>
                <td class="description-td">62 ± 3</td>
                <td class="description-td">2.1 ±</td>
                <td class="description-td">0.33 ±</td>
                <td class="description-td">3.0</td>
              </tr> <tr class="description-tr"> <td class="description-td">n = 27</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">0.1</td>
                <td class="description-td">0.03</td>
                <td class="description-td">fixed</td>
              </tr> <tr class="description-tr"> <td class="description-td">he2</td>
                <td class="description-td">54 ± 5</td>
                <td class="description-td">141 ± 8</td>
                <td class="description-td">53 ± 3</td>
                <td class="description-td">2.7 ±</td>
                <td class="description-td">0.37 ±</td>
                <td class="description-td">3.1</td>
              </tr> <tr class="description-tr"> <td class="description-td">Gel<sub>A50(C44)</sub> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">0.2</td>
                <td class="description-td">0.04</td>
                <td class="description-td">fixed</td>
              </tr> <tr class="description-tr"> <td class="description-td">n = 28</td>
              </tr> <tr class="description-tr"> <td class="description-td">he3</td>
                <td class="description-td">77 ± 6</td>
                <td class="description-td">140 ±</td>
                <td class="description-td">57 ± 3</td>
                <td class="description-td">2.5 ±</td>
                <td class="description-td">0.58 ±</td>
                <td class="description-td">3.0 ±</td>
              </tr> <tr class="description-tr"> <td class="description-td">Gel<sub>A50(C44)</sub> </td>
                <td class="description-td"> </td>
                <td class="description-td"> 20</td>
                <td class="description-td"> </td>
                <td class="description-td">0.4</td>
                <td class="description-td">0.11</td>
                <td class="description-td">1.0</td>
              </tr> <tr class="description-tr"> <td class="description-td">n = 33</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="7" align="center" rowsep="1" class="description-td" colspan="7"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p>Comparing the three immunoconjugates, the pharmacokinetics of he1 H65 Fab-Gel<sub>C247</sub>, he2 H65 Fab-Gel<sub>A50(C44) </sub>and he3-Fab-Gel<sub>A50(C44) </sub>were very similar, having similar alpha and beta half-lives, mean residence times, and clearance, particularly when comparing parameters obtained from the ELISA assayed curves. This is in contrast to their whole antibody immunoconjugate counterparts, where the clearance of Gel<sub>C247 </sub>immunoconjugate (11 ml/kg/hr) was three-fold greater than that of Gel<sub>A50(C44) </sub>immunoconjugate (4 ml/kg/hr). This suggests that cleavage of the disulfide bond linking the Fab fragment and gelonin is not as important for the serum clearance of Fab immunoconjugates as for whole antibody immunoconjugates.</p>
    <heading>EXAMPLE 12</heading> <p>Immunogenicity of Immunoconjugates</p>
    <p>Outbred Swiss/Webster mice were injected repeatedly (0.2 mg/kg each injection) with murine H65 antibody conjugates prepared with RTA, RTA30 and recombinant gelonin. The cycle was such that each animal was injected on days 1 and 2, and then the injections were repeated 28 and 29 days later. The animals received 5 such cycles of injections. One week and three weeks following each series of injections, blood was collected and the amount of anti-RIP antibodies present was determined by ELISA; peak titers for each cycle are shown in Table 9. RTA and RTA30 generated strong responses which began immediately following the first cycle of injections and remained high throughout the experiment. In contrast, no immune response was detected for the gelonin conjugate, even after 5 cycles of injections. When the conjugates were mixed with Complete Freund Adjuvant and injected i.p. into mice, anti-RTA and RTA-30 antibodies were readily detected after several weeks. These data indicate that anti-gelonin antibodies, if generated, would have been detected by the ELISA assay, and suggest that recombinant gelonin may be much less immunogenic in animals than is RTA.</p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="5"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="35pt" align="left"> </colspec> <colspec colname="2" colwidth="63pt" align="center"> </colspec> <colspec colname="3" colwidth="42pt" align="center"> </colspec> <colspec colname="4" colwidth="63pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="OFFSET" nameend="4" rowsep="1" class="description-td" colspan="5">TABLE 9</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="OFFSET" nameend="4" align="center" rowsep="1" class="description-td" colspan="5"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Cycle</td>
                <td class="description-td">H65-RTA</td>
                <td class="description-td">H65-RTA30</td>
                <td class="description-td">H65-rGel</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="OFFSET" nameend="4" align="center" rowsep="1" class="description-td" colspan="5"> </td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="5"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="35pt" align="left"> </colspec> <colspec colname="2" colwidth="63pt" align="char" char="."> </colspec> <colspec colname="3" colwidth="42pt" align="char" char="."> </colspec> <colspec colname="4" colwidth="63pt" align="char" char="."> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Prebleed</td>
                <td class="description-td">100</td>
                <td class="description-td">100</td>
                <td class="description-td">100</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Cycle 1</td>
                <td class="description-td">168</td>
                <td class="description-td">117</td>
                <td class="description-td">100</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Cycle 2</td>
                <td class="description-td">4208</td>
                <td class="description-td">1008</td>
                <td class="description-td">100</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Cycle 3</td>
                <td class="description-td">7468</td>
                <td class="description-td">3586</td>
                <td class="description-td">100</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Cycle 4</td>
                <td class="description-td">5707</td>
                <td class="description-td">3936</td>
                <td class="description-td">100</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Cycle 5</td>
                <td class="description-td">4042</td>
                <td class="description-td">2505</td>
                <td class="description-td">100</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="OFFSET" nameend="4" align="center" rowsep="1" class="description-td" colspan="5"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <heading>EXAMPLE 13</heading> <p>In vivo Efficacy of Immunoconjugates</p>
    <p>A human peripheral blood lymphocyte (PBL)-reconstituted, severe combined immunodeficient mouse model was utilized to evaluate the in vivo efficacy of various Immunoconjugates comprising the gelonin analogs Gel<sub>C247 </sub>and Gel<sub>A50(C44)</sub>. Immunoconjugates were tested for the capacity to deplete human blood cells expressing the CD5 antigen.</p>
    <p>A. Human PBL Donors and Cell Isolation</p>
    <p>Human peripheral blood cells were obtained from lymphapheresis samples (HemaCare Corporation, Sherman Oaks, Calif.) or venous blood samples (Stanford University Blood Bank, Palo Alto, Calif.) collected from healthy donors. Blood cells were enriched for PBLs using Ficoll-Hypaque density gradient centrifugation (Ficoll-Paque®; Pharmacia, Piscataway, N.J.) and subsequently washed 4 times with PBS. Residual erythrocytes were lysed with RBC lysing buffer (16 μM ammonium chloride, 1 mM potassium bicarbonate, 12.5 μM EDTA) during the second wash. Cell viability in the final suspension was &gt;95% as assessed by trypan blue dye exclusion.</p>
    <p>B. Animals and Human PBL Transfer</p>
    <p>CB.17 scid/scid (SCID) mice were purchased from Taconic (Germantown, N.Y.) or were bred under sterile conditions in a specific pathogen-free animal facility (original breeding pairs were obtained from Hana Biologics, Alameda, Calif.). Animals were housed in filter-top cages and were not administered prophylactic antibiotic treatment. Cages, bedding, food and water were autoclaved before use. All manipulations with animals were performed in a laminar flow hood.</p>
    <p>Untreated SCID mice were bled for determination of mouse Ig levels. Human PBL-injected mice were bled at various intervals for quantitation of human Ig and sIL-2R. Blood collection was from the retro-orbital sinus into heparinized tubes. Blood samples were centrifuged at 300×g for 10 min, and plasma was collected and stored at −70° C. Mouse and human Ig were quantified using standard sandwich ELISAs. Briefly, flat-bottom microtiter plates (MaxiSorp Immuno-Plates, Nunc, Roskilde, Denmark) were coated overnight at 4° C. with goat anti-mouse IgG+IgA+IgM (Zymed Laboratories, Inc., South San Francisco, Calif.) or goat anti-human Igs (Tago, Inc., Burlingame, Calif.) in bicarbonate buffer, pH 9.6. Plates were blocked for 2 hours at room temperature with 1% BSA in Tris-buffered saline, pH 7.5 (TBS), and then incubated at 37° C. for 1 hour with standards or samples serially-diluted in TBS/1% BSA/0.05% Tween 20. Standards used were a monoclonal mouse IgG2a (IND1 anti-melanoma; XOMA Corporation, Berkeley, Calif.) and polyclonal human Ig (Sigma Chemical Co., St. Louis, Mo.). Subsequently, plates were washed with TBS/Tween 20 and incubated at 37° C. for 1 hour with alkaline phosphatase-conjugated goat anti-mouse IgG+IgA+IgM or goat anti-human Igs (Caltag Laboratories, South San Francisco, Calif.). Detection was by measurement of absorbance at 405 nm following incubation with 1 mg/ml p-nitro-phenylphosphate (Sigma) in 10% diethanolamine buffer, pH 9.8. Plasma from a normal BALB/c mouse was used as a positive control in the mouse Ig ELISA. Plasma samples from naive SCID mice or normal BALB/c mice did not have detectable levels of human Ig. Human sIL-2R was quantified using an ELISA kit (Immunotech S.A., Marseille, France) as per the manufacturer's instructions.</p>
    <p>Five-to-seven week old mice with low plasma levels of mouse Ig (&lt;10 μg/ml) were preconditioned with an i.p. injection of cyclophosphamide (Sigma) at 200 mg/kg. Two days later, they were injected i.p. with 25-40×10<sup>6 </sup>freshly-isolated human PBL suspended in 0.8 ml PBS.</p>
    <p>C. Immunoconjugate Treatment</p>
    <p>SCID mice were bled at approximately 2 weeks after human PBL transplantation. Mice with undetectable (&lt;10 pM) or low plasma levels of human sIL-2R were eliminated from the study. The cut-off for exclusion of mice with detectable, but low, levels of human sIL-2R was empirically determined for each study and was generally 20 pM. The remaining mice were divided into groups and were administered vehicle or immunoconjugate as an i.v. bolus (0.2 mg/kg) daily for 5 consecutive days. Animals were sacrificed 1 day after cessation of treatment for quantitation of human T cells in tissues and human sIL-2R in plasma.</p>
    <p>D. Collection of Tissues and Analysis of PBL Depletion</p>
    <p>Blood was collected from the retro-orbital sinus into heparinized tubes. Mice were then killed by cervical dislocation and spleens were removed aseptically. Single cell suspensions of splenocytes were prepared in HBSS by pressing the spleens between the frosted ends of sterile glass microscope slides. Collected cells were washed twice with PBS. Erythrocytes were eliminated from blood and splenocyte suspensions using RBC lysing buffer. Subsequently, cells were resuspended in PBS for enumeration. Recovered cells were then assayed for Ag expression using flow cytometry.</p>
    <p>Two to five hundred thousand cells in 100 μl of PBS/1% BSA/0.1% sodium azide were incubated on ice for 30 min. with saturating amounts of various FITC- or phycoerythrin (PE)-conjugated Abs (Becton-Dickinson, Mountain View, Calif.) Abs used for staining included: HLe-1-FITC (IgG1 anti-CD45), Leu 2-FITC (IgG1 anti-CD8), Leu 3 PE (IgG1 anti-CD4), and Leu M3-PE (IgG2a anti-CD14). Cells were then washed in cold buffer and fixed in 0.37% formaldehyde in PBS. Samples were analyzed on a FACscan (Becton-Dickinson) using log amplifiers. Regions to quantify positive cells were set based on staining of cells obtained from naive SCID mice. The absolute numbers of human Ag-positive cells recovered from SCID tissues were determined by multiplying the percent positive cells by the total number of cells recovered from each tissue sample. The total number of leukocytes in blood was calculated using a theoretical blood volume of 1.4 ml/mouse. The detection limit for accurate quantitation of human cells in SCID mouse tissues was 0.05%. All statistical comparison between treatment groups were made using the Mann-Whitney U test. Treatment groups were determined to be significantly different from buffer control groups when the p value was &lt;0.05. Results are presented in Table 10 below, wherein + indicates a significant difference from controls, − indicates an insignificant difference and NT means the conjugate was not tested. CD5 Plus (XOMA Corporation, Berkeley, Calif.), is mouse H65 antibody chemically linked to RTA and is a positive control. OX19 Fab-Gel<sub>C247 </sub>is a negative control immunoconjugate. The OX19 antibody (European Collection of Animal Cell Cultures #84112012) is a mouse anti-rat CD5 antibody that does not cross react with human CD5.</p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="offset" colwidth="105pt" align="left"> </colspec> <colspec colname="1" colwidth="91pt" align="center"> </colspec> <colspec colname="2" colwidth="21pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="OFFSET" nameend="2" rowsep="1" class="description-td" colspan="3">TABLE 10</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="OFFSET" nameend="2" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Human T Cell Depletion</td>
                <td class="description-td"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="4"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="91pt" align="left"> </colspec> <colspec colname="2" colwidth="56pt" align="left"> </colspec> <colspec colname="3" colwidth="56pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Test Article</td>
                <td class="description-td">Spleen</td>
                <td class="description-td">Blood</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="OFFSET" nameend="3" align="center" rowsep="1" class="description-td" colspan="4"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">CD5 Plus</td>
                <td class="description-td">+</td>
                <td class="description-td">+</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">cH65 F(ab′)<sub>2</sub> </td>
                <td class="description-td">−</td>
                <td class="description-td">−</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">cH65 Fab′</td>
                <td class="description-td">−</td>
                <td class="description-td">−</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">H65-rGEL</td>
                <td class="description-td">+</td>
                <td class="description-td">+</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">cH65 F(ab′)<sub>2</sub>-rGel</td>
                <td class="description-td">+</td>
                <td class="description-td">+</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">cH65 Fab′-rGel</td>
                <td class="description-td">+</td>
                <td class="description-td">+</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">cH65 F(ab′)<sub>2</sub>-Gel<sub>c247</sub> </td>
                <td class="description-td">+</td>
                <td class="description-td">NT</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">cH65 Fab′-Gel<sub>c247</sub> </td>
                <td class="description-td">+</td>
                <td class="description-td">+</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">he1H65 Fab′-Gel<sub>c247</sub> </td>
                <td class="description-td">+</td>
                <td class="description-td">NT</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">cH65 Fab′-Gel<sub>A50(C44)</sub> </td>
                <td class="description-td">+</td>
                <td class="description-td">+</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">OX19 Fab-Gel<sub>c247</sub> </td>
                <td class="description-td">−</td>
                <td class="description-td">−</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="OFFSET" nameend="3" align="center" rowsep="1" class="description-td" colspan="4"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p>All the gelonin immunoconjugates were capable of depleting human cells in the SCID mouse model.</p>
    <heading>EXAMPLE 14</heading> <p>Construction of Gelonin Immunofusions with Chimeric Antibodies</p>
    <p>Several genetic constructs were assembled which included a natural sequence gelonin gene fused to an H65 truncated heavy chain gene (Fd or Fd′), or an H65 light chain gene (kappa). In this Example, H65 Fd, Fd′, and H65 light chain refer to chimeric constructs. The H65 Fd sequence consists of the nucleotides encoding the murine H65 heavy chain variable (V), joining (J) and human IgG<sub>1</sub>, constant (C) domain 1 regions, including the cysteine bound to light chain IgG<sub>1 </sub>and has the carboxyl terminal sequence SCDKTHT (SEQ ID NO: 130). The H65 Fd′ sequence has the H65 Fd sequence with the addition of the residues CPP from the hinge region of human IgG<sub>1 </sub>heavy chain, including a cysteine residue which is bound to the other human IgG<sub>1 </sub>heavy chain and its F(ab′)<sub>2 </sub>fragment. See Better, et al., <i>Proc. Nat. Acad. Sci. </i>(<i>USA</i>), 90: 457-461 (1993), incorporated by reference herein.</p>
    <p>The H65 light chain sequence consists of the nucleotides encoding the murine H65 light chain variable (V), joining (J), and human kappa (C<sub>k</sub>) regions. The DNA sequences of the V and J regions of the H65 Fd and kappa fragment genes linked to the pelB leader can be obtained from GenBank (Los Alamos National Laboratories, Los Alamos, N. Mex.) under Accession Nos. M90468 and M90467, respectively. Several of the gene fusions included a gelonin gene linked at the 5′ end of an H65 Fab fragment gene while the others included a gelonin gene linked at the 3′ end of an H65 Fab fragment gene. A DNA linker encoding a peptide segment of the <i>E. coli </i>shiga-like toxin (SLT) (SEQ ID NO: 56), which contains two cysteine residues participating in a disulfide bond and forming a loop that includes a protease sensitive amino acid sequence) or of rabbit muscle aldolase [(RMA) as in SEQ ID NO: 57, which contains several potential cathepsin cleavage sites] was inserted between the gelonin gene and the antibody gene in the constructs. Alternatively, a direct fusion was made between a gelonin gene and an H65 Fab fragment gene without a peptide linker segment. Table 11 below sets out a descriptive name of each gene fusion and indicates the expression plasmid containing the gene fusion and the section of the application in which each is designated. Each plasmid also includes the Fab fragment gene (shown in parentheses in Table 11) with which each particular gene fusion was co-expressed. The inclusion of a cysteine from a hinge region (Fd′) allows potential formation of either monovalent Fab′ or bivalent F(ab′)<sub>2 </sub>forms of the expression product of the gene fusion.</p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="4"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="49pt" align="left"> </colspec> <colspec colname="2" colwidth="49pt" align="left"> </colspec> <colspec colname="3" colwidth="105pt" align="left"> </colspec> <thead> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="OFFSET" nameend="3" rowsep="1" class="description-td" colspan="4">TABLE 11</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="OFFSET" nameend="3" align="center" rowsep="1" class="description-td" colspan="4"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Section</td>
                <td class="description-td">Plasmid</td>
                <td class="description-td">Description</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="OFFSET" nameend="3" align="center" rowsep="1" class="description-td" colspan="4"> </td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">B(i)</td>
                <td class="description-td">pING3754</td>
                <td class="description-td">Gelonin::SLT::Fd′ (kappa)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">B(ii)</td>
                <td class="description-td">pING3757</td>
                <td class="description-td">Gelonin::SLT::kappa (Fd′)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">B(iii)</td>
                <td class="description-td">pING3759</td>
                <td class="description-td">Gelonin::RMA::Fd′ (kappa)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">B(iv)</td>
                <td class="description-td">pING3758</td>
                <td class="description-td">Gelonin::RMA::kappa (Fd′)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">A(i)</td>
                <td class="description-td">pING4406</td>
                <td class="description-td">Fd::SLT::Gelonin (kappa)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">A(ii)</td>
                <td class="description-td">pING4407</td>
                <td class="description-td">kappa::SLT::Gelonin (Fd)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">A(iii)</td>
                <td class="description-td">pING4408</td>
                <td class="description-td">Fd::RMA::Gelonin (kappa)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">A(iv)</td>
                <td class="description-td">pING4410</td>
                <td class="description-td">kappa::RMA::Gelonin (Fd)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">C(i)</td>
                <td class="description-td">pING3334</td>
                <td class="description-td">Gelonin::Fd (kappa)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="OFFSET" nameend="3" align="center" rowsep="1" class="description-td" colspan="4"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p>A. Fusions of Gelonin at the Carboxyl-Terminus of Antibody Genes</p>
    <p>(i) Fd::SLT::Gelonin (kappa)</p>
    <p>A gelonin gene fusion to the 3′-end of the H65 Fd chain with the 23 amino acid SLT linker sequence was assembled in a three piece ligation from plasmids pVK1, pING3731 (ATCC 68721) and pING4000. Plasmid pVK1 contains the Fd gene linked in-frame to the SLT linker sequence and some H65 Fd′ and kappa gene modules as in pING3217, shown in Better, et al., <i>Proc. Nat. Acad. Sci. </i>(<i>USA</i>): 457-461 (1993), except that the kappa and Fd′ regions are reversed. Plasmid pING3731 contains the gelonin gene, and pING4000 contains the H65 kappa and Fd′ genes each linked to the pelB leader sequence under the control of the araB promoter as a dicistronic message.</p>
    <p>Plasmid pVK1 was designed to link the 3′-end of a human IgG Fd constant region in-frame to a protease-sensitive segment of the SLT gene bounded by two cysteine residues which form an intra-chain disulfide bond. The SLT gene segment (20 amino acids from SLT bounded by cysteine residues, plus three amino acids introduced to facilitate cloning) was assembled from two oligonucleotides, SLT Linker 1 and SLT Linker 2.</p>
    <p>SLT Linker 1 (SEQ ID NO: 73) 5′ TGTCATCATCATGCATCGCGAGTTGCCAGAATGGCATCT GATGAGTTTCCTTCTATGTGCGCAAGTACTC 3′</p>
    <p>SLT Linker 2 (SEQ ID NO: 74) 5′ TCGAGAGTACTTGCGCACATAGAAGGAAACTCATCAGAT GCCATTCTGGCAACTCGCGATGCATGATGATGACATGCA 3′</p>
    <p>The two oligonucleotides were annealed and ligated into a vector (pING3185) containing PstI and XhoI cohesive ends, destroying the PstI site and maintaining the XhoI site. Plasmid pING3185 contained an engineered PstI site at the 3′-end of the Fd gene, and contained an XhoI site downstream of the Fd gene. The product of this ligation, pVK1, contained the H65 Fd gene (fused to the pelB leader) in frame with the SLT linker segment, and contained two restriction sites, FspI and ScaI, at the 3′-end of the SLT linker.</p>
    <p>Plasmid pVK1 was digested with SauI and ScaI, and the 217 bp fragment containing a portion of the Fd constant domain and the entire SLT gene segment was purified by electrophoresis on an agarose gel. pING3731 was digested with SmaI and XhoI and the 760 bp gelonin gene was similarly purified. Plasmid pING4000 was digested with SauI and XhoI and the vector segment containing the entire kappa gene and a portion of the Fd gene was also purified. Ligation of these three DNA fragments resulted in pING4406 containing the Fd::SLT::Gelonin (kappa) gene fusion vector.</p>
    <p>(ii) kappa::SLT::Gelonin (Fd)</p>
    <p>A gelonin gene fusion to the 3′-end of the H65 kappa chain with the 25 amino acid SLT linker sequence (20 amino acids from SLT bounded by cysteine residues, plus 5 amino acids introduced to facilitate cloning) was assembled from the DNA segments in pING3731 (ATCC 68721) and pING3713.</p>
    <p>Plasmid pING3713 is an Fab expression vector where the H65 Fd and kappa genes are linked in a dicistronic transcription unit containing the SLT linker segment cloned in-frame at the 3′-end of the kappa gene. The plasmid was constructed as follows. In a source plasmid containing the H65 Fd and kappa genes, an EagI site was positioned at the 3′-end of the kappa gene by site directed mutagenesis without altering the encoded amino acid sequence. The SLT gene segment from pVK1 was amplified with primers SLT-EagI-5′ and SalI for in frame linkage to the EagI site at the 3′-end of the kappa gene.</p>
    <p>SLT-Eag-5′ (SEQ ID NO: 75) 5′ TGTTCGGCCGCATGTCATCATCATGCATCG 3′</p>
    <p>SalI (SEQ ID NO: 76) 5′ AGTCATGCCCCGCGC 3′</p>
    <p>The 140 bp PCR product was digested with EagI and XhoI, and the 75 bp fragment containing the SLT gene segment was cloned adjacent to the Fd and kappa genes in the source plasmid to generate pING3713.</p>
    <p>For construction of gene fusion to gelonin, pING3713 was cut with ScaI and XhoI, and the vector fragment containing the Fd gene and kappa::SLT fusion was purified. pING3731 was digested with SmaI and XhoI and the DNA fragment containing the gelonin gene was also purified. The product of the ligation of these two fragments, pING4407, contains the Fd and kappa::SLT::gelonin genes.</p>
    <p>(iii) Fd::RMA::Gelonin (kappa)</p>
    <p>A gelonin gene fusion to the 3′-end of the H65 Fd chain with the 21 amino acid RMA linker sequence (20 amino acids from RMA, plus 1 amino acid introduced to facilitate cloning) was assembled in a three piece ligation from plasmids pSH4, pING3731 (ATCC 68721) and pING4000.</p>
    <p>Plasmid pSH4 contains an Fd gene linked in frame to the RMA linker sequence. The RNA gene segment was linked to the 3′-end of Fd by overlap extension PCR as follows. The 3′-end (constant region) of the Fd gene was amplified by PCR from a source plasmid with the primers KBA-γ2 and RMAG-1. Any Fd constant region may be used because constant regions of all human IgG<sub>1 </sub>antibodies are identical in this region.</p>
    <p>KBA-γ2 (SEQ ID NO: 77) 5′ TCCCGGCTGTCCTACAGT 3′</p>
    <p>RHAG-1 (SEQ ID NO: 78) 5′ TCCAGCCTGTCCAGATGGTGTGTGAGTTTTGTCACAA 3′</p>
    <p>The product of this reaction was mixed with primer RMA-76, which annealed to the amplified product of the first reaction, and the mixture was amplified with primers KBA-γ2 and RMAK-2.</p>
    <p>RMA-76 (SEQ ID NO: 79) 5′ CTAACTCGAGAGTACTGTATGCATGGTTCGAGATGAACA AAGATTCTGAGGCTGCAGCTCCAGCCTGTCCAGATGG 3′</p>
    <p>RMAK-2 (SEQ ID NO: 80) 5′ CTAACTCGAGAGTACTGTAT 3′</p>
    <p>The PCR product contained a portion of the Fd constant region linked in-frame to the RMA gene segment. The product also contained a ScaI restriction site useful for in-frame fusion to a protein such as gelonin, and an XhoI site for subsequent cloning.</p>
    <p>This PCR product was cut with SauI and XhoI and ligated adjacent to the remainder of the Fd gene to generate pSH4.</p>
    <p>For assembly of the gene fusion vector containing the Fd::RMA::Gelonin, kappa genes, pSH4 was cut with SauI and ScaI and the Fd::RMA segment was purified. Plasmid PING3731 was cut with SmaI and XhoI and the 760 bp DNA fragment containing the gelonin gene was purified, and pING4000 was cut with SauI and XhoI and the vector was purified. The product of the ligation of these fragments, pING4408, contained the Fd::RMA::Gelonin and kappa genes.</p>
    <p>(iv) kaDpa::RMA::Gelonin (Fd)</p>
    <p>A gelonin gene fusion to the 3′-end of the H65 kappa chain with the 21 amino acid RMA linker sequence was assembled in a three piece ligation from plasmids pSH6, pING4408 (see the foregoing paragraph) and pING3713.</p>
    <p>Plasmid pSH6 contains a kappa gene linked in-frame to the RMA linker sequence. The RMA gene segment was linked to the 3′-end of kappa by overlap extension PCR as follows. The 3′-end (constant region) of the kappa gene was amplified by PCR from a source plasmid with the primers KBA-K2 and RMAK-1.</p>
    <p>RMAK-1 (SEQ ID NO: 81) 5′ TCCAGCCTGTCCAGATGGACACTCTCCCCTGTTGAA 3′</p>
    <p>KBA-K2 (SEQ ID NO: 82) 5′ GTACAGTGGAAGGTGGAT 3′</p>
    <p>The product of this reaction was mixed with primer RMA-76 (SEQ ID NO: 81), which annealed to the amplified product of the first reaction, and the mixture was amplified with primers KBA-K2 and RMAK-2. The PCR product contained a portion of the kappa constant region linked in-frame to the RMA gene segment. The product also contained a ScaI restriction site useful for in-frame fusion to a protein such as gelonin, and an XhoI site for subsequent cloning. This PCR product was cut with SstI and XhoI and ligated adjacent to the remainder of the kappa gene to generate pSH6.</p>
    <p>For assembly of the gene fusion vector containing the kappa::RMA::Gelonin and Fd genes, pSH6 was cut with HindIII and PstI and the DNA fragment containing the kappa constant region and a portion of the RMA linker (the PstI RMA linker segment contains a PstI site) segment was purified. Plasmid pING4408 was cut with PstI and SalI and the DNA fragment containing a segment of the RMA linker, the gelonin gene and a portion of the tetracycline resistance gene in the vector segment was purified. pING3713 was cut with SalI and HindIII and the vector was purified. The product of the ligation of these three fragments, pING4410, contained the kappa::RMA::Gelonin and Fd genes.</p>
    <p>B. Fusions of Gelonin at the Amino-Terminus of Antibody Genes</p>
    <p>(i) Gelonin::SLT::Fd′ (kappa)</p>
    <p>A gelonin gene fusion to the 5′-end of the H65 Fd′ chain with a 25 amino acid SLT linker sequence (20 amino acids from SLT bounded by cysteine residues, plus five amino acids introduced to facilitate cloning) was assembled in a three piece ligation from plasmids pING3748, pING3217, and a PCR fragment encoding the H65 heavy chain variable region (V<sub>H</sub>) gene segment which is the variable region of the Fd′ gene in pING3217. Plasmid pING3748 contains the gelonin gene linked in-frame to the SLT linker sequence, and pING3217 contains the H65 Fd′ and kappa genes in a dicistronic transcription unit.</p>
    <p>Plasmid pING3825 (see Example 2) was amplified with PCR primers gelo3′-Eag and gelo-9 to introduce an EagI restriction site at the 3′-end of the gelonin gene by PCR mutagenesis.</p>
    <p>gelo3′-Eag (SEQ ID NO: 83)</p>
    <p>5′ CATGCGGCCGATTTAGGATCTTTATCGACGA 3′</p>
    <p>The PCR product was cut with BclI and EagI and the 56 bp DNA fragment was purified. Plasmid pING3713 was cut with EagI and XhoI, and the 77 bp DNA fragment containing the SLT linker was purified. The 56 bp BclI to EagI fragment and the 77 bp EagI to XhoI fragment were ligated into pING3825 which had been digested with BclI and XhoI to generate pING3748 which contains the gelonin gene linked in-frame to the SLT linker sequence.</p>
    <p>For assembly of the gene fusion vector containing the Gelonin::SLT::Fd′ and kappa genes, the H65 V<sub>H </sub>was amplified by PCR from pING3217 with primers H65-G1 and H65-G2, and the product was treated with T4 polymerase followed by digestion with NdeI.</p>
    <p>H65-G1 (SEQ ID NO: 84) 5′ AACATCCAGTTGGTGCAGTCTG 3′</p>
    <p>H65-G2 (SEQ ID NO: 85) 5′ GAGGAGACGGTGACCGTGGT 3′</p>
    <p>The 176 bp fragment containing the 5′-end of the H65 heavy chain V-region was purified. Concurrently, pING3217 was digested with NdeI and XhoI, and the 1307 bp DNA fragment containing a portion of the Fd′ gene and all of the kappa gene was purified. The two fragments were ligated to pING3748 which had been digested with ScaI and XhoI in a three piece ligation yielding pING3754 (ATCC 69102), which contains the Gelonin::SLT::Fd′ and kappa genes.</p>
    <p>(ii) Gelonin::SLT::kappa (Fd′)</p>
    <p>A gelonin gene fusion to the 5′-end of the H65 kappa chain with the 25 amino acid SLT linker sequence was assembled in a three piece ligation from plasmids pING3748 (see the foregoing section), pING4000, and a PCR fragment encoding the H65 light chain variable region (V<sub>L</sub>) gene segment.</p>
    <p>For assembly of the gene fusion vector containing the Gelonin::SLT::kappa and Fd′ genes, an H65 V<sub>L </sub>fragment was amplified by PCR from pING3217 with primers H65-K1 and JK1-HindIII, and the product was treated with T4 polymerase followed by digestion with Hind III.</p>
    <p>H65-K1 (SEQ ID NO: 86) 5′ GACATCAAGATGACCCAGT 3′</p>
    <p>JK1-HindIII (SEQ ID NO: 87) 5′ GTTTGATTTCAAGCTTGGTGC 3′</p>
    <p>The 306 bp fragment containing the light chain V-region was purified. Concurrently, pING4000 was digested with HindIII and XhoI, and the 1179 bp DNA fragment containing the kappa constant region and all of the Fd′ gene was purified. The two fragments were ligated to pING3748 which had been digested with ScaI and XhoI in a three piece ligation yielding pING3757, which contains the Gelonin::SLT::kappa and Fd genes.</p>
    <p>(iii) Gelonin::RMA::Fd′ (kappa)</p>
    <p>A gelonin gene fusion to the 5′-end of the H65 Fd′ chain with the 24 amino acid RMA linker sequence (20 amino acids from RMA, plus 4 amino acids introduced to facilitate cloning) was assembled in a three piece ligation from plasmids pING3755, pING3217 and a PCR fragment encoding the H65 V<sub>H </sub>gene segment. Plasmid pING3755 contains the gelonin gene linked in-frame to the RMA linker sequence, and pING3217 contains the H65 Fd′ and kappa genes in a dicistronic transcription unit.</p>
    <p>Plasmid pING3755 was assembled to contain the gelonin gene linked to the RMA linker gene segment. The RMA linker gene segment was amplified by PCR from pSH4 with primers RMA-EagI and HINDIII-2.</p>
    <p>RMA-EagI (SEQ ID NO: 88) 5′ ACTTCGGCCGCACCATCTGGACAGGCTGGAG 3′</p>
    <p>HINDIII-2 (SEQ ID NO: 44) 5′ CGTTAGCAATTTAACTGTGAT 3′</p>
    <p>The 198 bp PCR product was cut with EagI and HindIII, and the resulting 153 bp DNA fragment was purified. This RMA gene segment was cloned adjacent to gelonin using an PstI to EagI fragment from pING3748 and the PstI to HindIII vector fragment from pING3825. The product of this three piece ligation was pING3755.</p>
    <p>For assembly of the gene fusion vector containing the Gelonin::RMA::Fd′, kappa genes, the H65 V<sub>H </sub>was amplified by PCR from pING3217 with primers H65-G1 (SEQ ID NO: 84) and H65-G2 (SEQ ID NO: 85), and the product was treated with T4 polymerase followed by digestion with NdeI. The 186 bp fragment containing the 5′-end of the heavy chain V-region was purified. Concurrently, pING3217 was digested with NdeI and XhoI, and the 1307 bp DNA fragment containing a portion of the Fd′ gene and all of the kappa gene was purified. These two fragments were ligated to pING3755 which had been digested with ScaI and XhoI in a three piece ligation yielding pING3759 (ATCC 69104), which contains the Gelonin::RMA::Fd′ and kappa genes.</p>
    <p>(iv) Gelonin::RMA::kappa (Fd′)</p>
    <p>A gelonin gene fusion to the 5′-end of the H65 kappa chain with the 24 amino acid RMA linker sequence was assembled in a three piece ligation from plasmids pING3755, pING4000, and a PCR fragment encoding the H65 V<sub>L </sub>gene segment.</p>
    <p>For assembly of the gene fusion vector containing the Gelonin::RMA::kappa and Fd′ genes, an H65 V<sub>L </sub>segment was amplified by PCR from pING3217 with primers H65K-1 (SEQ ID NO: 86) and JK1-HindIII, and the product was treated with T4 polymerase followed by digestion with HindIII. The 306 bp fragment containing the 5′-end of the light chain V-region was purified. Concurrently, pING4000 was digested with HindIII and XhoI, and the 1179 bp DNA fragment containing the kappa constant region and all of the Fd′ gene was purified. These two fragments were ligated to pING3755 which had been digested with ScaI and XhoI in a three piece ligation yielding pING3758 (ATCC 69103), which contains the Gelonin::RMA::kappa and Fd′ genes.</p>
    <p>C. Direct Fusions of Gelonin at the Amino Terminus of Antibody Genes</p>
    <p>(i) Gelonin::Fd′ (Kappa)</p>
    <p>A direct gelonin gene fusion was constructed from pING3754. pING3754 was digested with BgIII and XhoI and the vector segment was purified. Concurrently, pING3754 was digested with EagI, treated with T4 polymerase, cut with BglII, and the gelonin gene segment was purified. pING3754 was also cut with FspI and XhoI, and the Fd and kappa gene segment was purified. These fragments were assembled in a three-piece ligation to generate pING3334, which contains a direct gene fusion of gelonin to Fd′ in association with a kappa gene.</p>
    <heading>EXAMPLE 15</heading> <p>Preparation of he3 Fab and Gelonin he3Fab Immunofusions</p>
    <p>The sections below detail the construction of human-engineering he3Fab protein and immunofusions of gelonin to he3 Fd and kappa chains.</p>
    <p>A. he3-Fab Expression Plasmids</p>
    <p>The he3 heavy chain V-region was PCR-amplified from plasmid pING4621 (pING4621 is fully described above in Example 5 above), with primers H65-G3, GAGATCCAGTTGGTGCAGTCTG (SEQ ID NO: 116) and H65G2 (SEQ ID NO: 85). Amplification was carried at using vent polymerase (New England Biolabs) for 25 cycles, including a 94° C. denaturation for 1 minute, annealing at 50° C. for 2 minutes, and polymerization for 3 minutes at 72° C. The PCR product was treated with polynucleotide kinase and digested with BstEII and the V-region DNA was purified. The purified DNA fragment was then ligated into pIC100, which had been digested with SstI, treated with T4 polymerase, and cut with BstEII. The resulting fragment was then ligated with the BstEII fragment from pING3218 (containing Fab′ genes) to make pING4623 which contained the he3 Fd gene linked to the pelB leader sequence.</p>
    <p>The he3 kappa V-region was next assembled as described above in Example 5 and in co-owned, co-pending U.S. patent application Ser. No. 07/808,464, incorporated by reference herein, using six oligonucleotide primers,</p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" pgwide="1" tabstyle="monospace" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="1" colwidth="252pt" align="left"> </colspec> <colspec colname="2" colwidth="70pt" align="left"> </colspec> <colspec colname="3" colwidth="-329pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">           $H65k-1, AGT CGT CGA CAC GAT GGA CAT GAG GAC CCC</td>
                <td class="description-td">(SEQ ID NO: 117)</td>
              </tr> <tr class="description-tr"> <td class="description-td">TGC TCA GTT TCT TGG CAT CCT CCT ACT CTG GTT TCC AGG TAT CAA</td>
              </tr> <tr class="description-tr"> <td class="description-td">ATG TGA CAT CCA GAT GAC TCA GT;</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">            HUH-K6, TCA CTT GCC GGG CGA ATC AGG ACA TTA ATA</td>
                <td class="description-td">(SEQ ID NO: 118)</td>
              </tr> <tr class="description-tr"> <td class="description-td">GCT ATT TAA GCT GGT TCC AGC AGA AAC CAG GGA AAG CTC CTA AGA</td>
              </tr> <tr class="description-tr"> <td class="description-td">CCC T;</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">            HUH-K7, TGA CTC GCC CGG CAA GTG ATA GTG ACT CTG</td>
                <td class="description-td">(SEQ ID NO: 119)</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCT CCT ACA GAT GCA GAC AGG GAA GAT GGA GAC TGA GTC ATC TGG</td>
              </tr> <tr class="description-tr"> <td class="description-td">ATG TC;</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">            HUH-K8, GAT CCA CTG CCA CTG AAC CTT GAT GGG ACC</td>
                <td class="description-td">(SEQ ID NO: 120)</td>
              </tr> <tr class="description-tr"> <td class="description-td">CCA GAT TCC AAT CTG TTT GCA CGA TAG ATC AGG GTC TTA GGA GCT</td>
              </tr> <tr class="description-tr"> <td class="description-td">TTC C;</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">            HUH-K4, GGT TCA GTG GCA GTG GAT CTG GGA CAG ATT</td>
                <td class="description-td">(SEQ ID NO: 121)</td>
              </tr> <tr class="description-tr"> <td class="description-td">ATA CTC TCA CCA TCA GCA GCC TGC AAT ATG AAG ATT TTG GAA TTT</td>
              </tr> <tr class="description-tr"> <td class="description-td">ATT ATT G; and</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">            HUH-K5, GTT TGA TTT CAA GCT TGG TGC CTC CAC CGA</td>
                <td class="description-td">(SEQ ID NO: 122)</td>
              </tr> <tr class="description-tr"> <td class="description-td">ACG TCC ACG GAG ACT CAT CAT ACT GTT GAC AAT AAT AAA TTC CAA</td>
              </tr> <tr class="description-tr"> <td class="description-td">AAT CTT C</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">and amplified with primers HUK-7 (SEQ ID NO: 92) and JK1-</td>
              </tr> <tr class="description-tr"> <td class="description-td">HindIII (SEQ ID NO: 87).</td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p>The resulting PCR product was treated with T4 polymerase, digested with HindIII, and purified. The purified fragment was then cloned into pIC100, which had first been cut with SstI, treated with T4 polymerase, and digested with XhoI, along with the 353 bp HindIII-XhoI fragment encoding the kappa constant region from pING3217. The resulting plasmid was pING4627 which contains the he3 kappa sequence linked in frame to the pelB leader.</p>
    <p>Plasmid pING4628, containing the pelB-linked he3 kappa and Fd genes under transcriptional control of the araB promoter, was assembled from pING4623 and pING4627 as follows.</p>
    <p>An expression vector for unrelated kappa and Fd genes, pNRX-2, was first cut with SauI and EcoRI, leaving a vector fragment which contains all the features relevant to plasmid replication, a tetracycline resistance marker, araB transcriptional control, and the 3′ end of the Fd constant region. [Plasmid pNRX-2 comprises an EcoRI to XhoI DNA segment from pING 3104 (described in WO 90/02569, incorporated by reference herein). That segment contains the replication, resistance and transcription control features of pING3104 and is joined to an XhoI to SauI DNA segment from pING1444 (described in WO 89/00999, incorporated by reference herein) which contains the 3′ end of an Fd constant region.] Next pING4623 was cut with PstI, treated with T4 polymerase, digested with SauI and the pelB::Fd gene segment was then isolated. Plasmid pING4627 was cut with XhoI, treated with T4 polymerase, cut with EcoRI and ligated to the pelB::Fd gene segment and the pNRX-2 vector fragment to generate the he3-Fab expression vector pING4628. That plasmid contains two XhoI sites, one located between the kappa and Fd genes, and another 4 bp downstream of the termination codon for the Fd gene.</p>
    <p>A vector, pING4633, which lacks the XhoI site between the kappa and Fd genes was constructed. To assemble pING4633, pING4623 was cut with EcoRI, treated with T4 polymerase, digested with SauI. The pelB::kappa gene segment was then isolated and purified. The pNRX-2 vector fragment and the pelB::Fd gene segment were then ligated to the purified pelB::kappa gene segment to form pING4633.</p>
    <p>Both pING4633 and pING4628 are bacterial expression vectors for he3-Fab and each comprises the he3 Fd and Kappa genes which are expressed as a dicistronic message upon induction of the host cell with L-arabinose. Moreover, pING4628 contains two XhoI restriction sites, one located 4 bp past the Fd termination codon and one in the intergenic region between the 3′ end of the Kappa gene and the 5′ end of the Fd gene. Plasmid pING4633 lacks the XhoI site in the intergenic region.</p>
    <p>B. Purification of he3Fab</p>
    <p>Plasmids pING4628 and pING4633 were transformed into <i>E. coli </i>E104. Bacterial cultures were induced with arabinose and cell-free supernatant comprising the he3Fab was concentrated and filtered into 20 mm HEPES, pH 6.8. The sample was then loaded onto a CM Spheradex column (2.5×3 cm), equilibrated in 20 mM HEPEs, 1.5 mM NaCl, pH 6.8. The column was washed with the same buffer and eluted with 20 mm HEPES, 27 mM NaCl, pH 6.8. The eluate was split into 2 aliquots and each was loaded onto and eluted from a protein G (Bioprocessing) column (2.5×2.5 cm) separately. The protein G column was equilibrated in 20 mM HEPES, 75 MM NaCl, pH 6.8 and the sample was eluted with 100 mM glycine, 100 mM NaCl, pH 3.0. The two eluates were combined and diluted two times with 20 mM HEPES, 3 M ammonium sulfate, pH 6.8. The diluted eluates were loaded onto phenyl sepharose high substitution Fast Flow (Pharmacia) column (2.5×3.3 cm), equilibrated n 20 mM HEPES, 1.5 M ammonium sulfate, pH 6.8. The column was then eluted with 20 mM HEPES, 0.6 M ammonium sulfate, pH 6.8.</p>
    <p>C. Gelonin::RMA::he3Kappa, he3Fd Fusions</p>
    <p>Other genetic constructs were assembled which included a natural sequence gelonin gene fused to an he3-Fab via a linker.</p>
    <p>A fusion comprising Gelonin::RMA::he3Kappa, Fd was assembled from DNA from plasmids pING3755, pING4633, and pING4628. Both pING4633 and pING4628 were assembled in a series of steps whereby the he3 heavy and light V-regions were individually linked in-frame to the pelB leader. The heavy and light V-regions were then placed together in a dicistronic expression vector under the control of the araB promoter in a bacterial expression vector.</p>
    <p>Assembly of the Gelonin::RMA::he3Kappa, he3Fd fusions was accomplished by constructing three DNA fragments from plasmids pING3755, pING4633, and pING4628. The first such fragment was made by digesting pING3755 with ScaI and XhoI which excises the 4 bp between those sites. The resulting vector fragment was purified. The second fragment for use in constructing the above fusions was obtained from plasmid pING4633, which was cut with AseI (which cuts in V<sub>L</sub>) and XhoI and the resulting 1404 bp fragment, containing the 3′ end segment of the Kappa and Fd genes, was purified. The third fragment, comprising the 5′ end of the Kappa variable region coding sequence, was prepared from the PCR amplified V<sub>L </sub>gene contained in pING4628 using the oligonucleotide primers, Huk-7 and jk1-HindIII. The resulting 322 bp PCR-amplified V<sub>L </sub>fragment was treated with T4 polymerase, digested with AseI, and the 86 bp fragments containing the 5′ end of V<sub>L </sub>was purified. The three fragment produced above were ligated together to form pING3764. The DNA sequence of the PCR amplified V-region was verified by direct DNA sequencing of pING3764.</p>
    <p>D. Gelonin::SLT::he3Kappa, he3Fd Fusion</p>
    <p>A Gelonin::SLT::he3Kappa, he3Fd fusion was constructed by ligating the pING4633 and pING4628 fragments described in section A immediately above with a fragment produced from pING3748 which contains Gelonin::SLT. The pING3748 fragment was produced using ScaI and XhoI as described immediately above for pING3755. The resulting vector was designated pING3763.</p>
    <p>E. Construction of Expression Vector Containing Gelonin::SLT::he3Fd, he3kappa Fusions</p>
    <p>An expression vector containing the Gelonin::SLT::he3Fd, he3kappa fusion was constructed in two steps form DNA segments from plasmids pING3825, pING4628, pING4639, pING3217 [described in Better, et al., <i>Proc. Natl. Acad. Sci. </i>(<i>USA</i>), 90:457-461 (1993), incorporated by reference herein], and pING4627. pING3825 was digested with NcoI and XhoI, generating a 654 bp fragment containing the 3′ end of the gelonin gene and a fragment containing the 5′ end of the gelonin gene which were purified. Next, pING4639 was digested with NcoI and NdeI and the 903 bp fragment containing the 3′ end of the Gelonin gene, the SLT linker, and the 5′ end of V<sub>E </sub>which resulted was purified. Finally, pING4628 was cut with NdeI and XhoI, resulting in a 523 bp fragment containing the 3′ end of the Fd gene which was purified. The three fragments were then ligated to form plasmid pING3765 which contains a gene encoding a gelonin::SLT::he3Fd fusion.</p>
    <p>Three vector fragments were used to assemble the final expression vector (containing the gelonin::SLT::he3Fd and he3 kappa segments). Plasmid pING3765 was digested with XhoI, treated with T4 polymerase, cut with NheI (which releases a 265 bp fragment encoding the tetracycline resistent marker), and the resulting vector fragment was purified. Plasmid pING4627, which contains the he3Kappa gene linked in-frame to the pelB leader was used for the construction of pING4628. Plasmid pING4627 was cut with PstI, treated with T4 polymerase, and further digested with SstI. The resulting 726 bp fragment, containing the Kappa gene (except 40 bp at the 3′ end) was purified. Plasmid pING3217 was then cut with SstI and NheI, resulting in a 318 bp fragment containing the 3′ end of the Kappa gene and downstream portion, including a portion of the tetracycline resistance gene, which was purified. Ligation of the foregoing three fragments produced the final expression vector, pING3767.</p>
    <p>F. Construction of Expression Vector Containing Gelonin::RMA::he3Fd Fusions</p>
    <p>Gelonin::RMA:he3Fd, he3Kappa fusion expression vectors was constructed in two steps from plasmids pING3825, pING4628, pING3217, and pING4627. The cloning scheme used was identical to that used to generate pING3767 except that pING4638 was substituted for pING4639. Plasmid pING4638 differs from pING4639 as described below in Example 16. The intermediate vector encoding the Gelonin::RMA::Fd fusion was designated pING3766 and the final expression vector was designated pING3768.</p>
    <heading>EXAMPLE 16</heading> <p>Gelonin-Single Chain Antibody Fusions</p>
    <p>The natural sequence gelonin gene was also fused to a single chain form of the human engineered he3 H65 variable region. The gelonin gene was positioned at either the N-terminus or the C-terminus of the fusion gene and the SLT or RMA linker peptide was positioned between the gelonin and antibody domains to allow intracellular processing of the fusion protein with subsequent cytosolic release of gelonin.</p>
    <p>A. Construction of Gel:: MA::SCA(V<sub>L</sub>-V<sub>H</sub>), Gel::SLT::SCA (V<sub>L</sub>-V<sub>H</sub>), Gel::RMA:;SCA(V<sub>H</sub>-V<sub>L</sub>), and Gel::SLT::SCA(V<sub>H</sub>-V<sub>L</sub>)</p>
    <p>A single chain antibody (SCA) form of the he3 H65 variable domain was assembled from previously constructed genes. This SCA segment consisted of the entire V and J region of the one chain (heavy or light) linked to the entire V and J segment of the other chain (heavy or light) via a 15 amino acid flexible peptide: [(Gly)<sub>4 </sub>Ser]<sub>3</sub>. This peptide is identical to that described in Huston et al., <i>Proc. Natl. Acad. Sci. USA, </i>85:5879-5883 (1988); Glockshuber et al., <i>Biochemistry, </i>29:1362-1367 (1990); and Cheadle et al., <i>Molecular Immunol., </i>29:21-30 (1992). The SCA was assembled in two orientations: V-J<sub>kappa</sub>::[(Gly)<sub>4</sub>Ser]<sub>3</sub>::V-J<sub>Gamma </sub>and V-J<sub>Gamma</sub>::[(Gly)<sub>4</sub>Ser]<sub>3</sub>::V-J<sub>kappa</sub>. Each SCA segment was assembled and subsequently fused to gelonin.</p>
    <p>For assembly of the SCA segment V-J<sub>kappa</sub>::[(Gly)<sub>4</sub>Ser]<sub>3</sub>:: V-J<sub>Gamma</sub>, primers HUK-7 and SCFV-1 were used to amplify a 352 bp DNA fragment containing the he3 V/J kappa sequences from pING4627 by PCR in a reaction containing 10 mM KCl, 20 mM TRIS pH 8.8, 10 mM (NH<sub>4</sub>)<sub>2</sub>SO<sub>2</sub>, 2 mM MgSO<sub>4</sub>, 0.1% Triton X-100, 100 ng/ml BSA, 200 uM of each dNTP, and 2 Units of Vent polymerase (New England Biolabs, Beverley, Mass.) in a total volume of 100 μl.</p>
    <p>SCFV-1 (SEQ ID NO:91) 5′ CGGACCCACCTCCACCAGATCCACCGC CACCTTTCATCTCAAGCTTGGTGC 3′</p>
    <p>HUK-7 (SEQ ID NO: 92) 5′ GACATCCAGATGACTCAGT 3′</p>
    <p>concurrently, primers SCFV-2 and SCFV-3 were used to amplify a he3 heavy chain V/J gamma segment from pING4623, generating a 400 bp fragment.</p>
    <p>SCFV-2 (SEQ ID NO: 93) 5′ GGTGGAGGTGGGTCCGGAGGTGGAGGATCTGA GATCCAGTTGGTGCAGT 3′</p>
    <p>SCFV-3 (SEQ ID NO: 94) 5′ TGTACTCGAGCCCATCATGAGGAGACGGTGACCGT 3′</p>
    <p>The products from these reactions were mixed and amplified with the outside primers HUK-7 and SCFV-3. The product of this reaction was treated with T4 polymerase and then cut with XhoI. The resulting 728 bp fragment was then purified by electrophoresis on an agarose gel. This fragment was ligated into the vectors pING3755 and pING3748 (see Example 10), each digested with ScaI and XhoI. The resulting vectors pING4637 and pING4412 contain the Gelonin::RMA::SCA V-J<sub>kappa</sub>::[(Gly)<sub>4</sub>Ser]<sub>3</sub>::V-J<sub>Gamma </sub>and Gelonin::SLT::SCA V-J<sub>kappa</sub>::[(Gly)<sub>4</sub>Ser]<sub>3</sub>::V-J<sub>Gamma </sub>fusion genes, respectively. The 728 bp fragment was also ligated into pIC100 previously digested with SstI, treated with T4 polymerase and digested with XhoI, to generate pING4635. This plasmid contains the pelB leader sequence linked in-frame to the V-J<sub>kappa</sub>::[(Gly)<sub>4</sub>Ser]<sub>3</sub>::V-JJ<sub>gamma </sub>gene. The pelB::SCA gene in pING4635 was excised as an EcoRI-XhoI restriction fragment and cloned into the bacterial expression vector to generate pING4640.</p>
    <p>Similarly, the SCA V-J<sub>Gamma</sub>::[(Gly)<sub>4</sub>Ser]<sub>3</sub>::V-J<sub>kappa </sub>was assembled by amplification of pING4627 with primers SCFV-5 and SCFV-6 generating a 367 bp fragment containing he3 V/J kappa sequences,</p>
    <p>SCFV-5 (SEQ ID NO: 95) 5 GGTGGAGGTGGGTCCGGAGGTGGAGGATCT GACATCCAGATGACTCAGT 3′</p>
    <p>SCFV-6 (SEQ ID NO: 96) 5′ TGTACTCGAGCCCATCATTTCATCTCAAGCTTGGTGC 3′</p>
    <p>and pING4623 with primers H65-G3 and SCFV-4 generating a 385 bp fragment containing he3 gamma V/J sequences by PCR with Vent polymerase.</p>
    <p>H65-G3 (SEQ ID NO: 97) 5′ GAGATCCAGTTGGTGCAGTCTG 3′</p>
    <p>SCFV-4 (SEQ ID NO: 98) 5′ CGGACCCACCTCCACCAGATCC ACCGCCACCTGAGGAGACGGTGACCGT 3′</p>
    <p>The products from these reactions were mixed and amplified with H65-G3 and SCFV-6. The 737 bp product was treated with T4 polymerase and cut with XhoI. Ligation into pING3755 and pING3748 (digested with ScaI and XhoI) resulted in assembly of the Gelonin::RMA::SCA V-J<sub>Gamma</sub>::[(Gly)<sub>4</sub>Ser]<sub>3</sub>::V-J<sub>kappa </sub>gene fusion in pING4638 and Gelonin::SLT::SCA V-J<sub>Gamma</sub>::[(Gly)<sub>4</sub>Ser]<sub>3</sub>::V-J<sub>kappa </sub>gene fusion in pING4639, respectively.</p>
    <p>The vectors pING4637, pING4412, pING4638 and pING4639 were each transformed into <i>E. coli </i>strain E104 and induced with arabinose. Protein products of the predicted molecular weight were identified by Western blot with gelonin-specific antibodies.</p>
    <p>B. Construction of SCA(V<sub>L</sub>-V<sub>H</sub>::SLT::Gelonin Vectors</p>
    <p>The expression vector containing SCA (V<sub>L-V</sub> <sub>H</sub>)::SLT::Gelonin fusions was assembled using restriction fragments from previously-constructed plasmids pING4640 (containing SCA(V<sub>L</sub>-V<sub>H</sub>) ) pING4407 (containing Kappa::SLT::Gelonin, Fd), and pING3197. Plasmid pING4640 was first cut with BspHI, filled in with T4 polymerase in the presence of only dCTP, treated with mung bean nuclease (MBN) to remove the overhang and to generate a blunt end, and cut with EcoRI. The resulting 849 bp fragment was purified. The SLT-containing fragment from pING4407 was excised by cutting with EagI, blunted with T4 polymerase, cut with XhoI, and the approximately 850 bp fragment which resulted was purified. The two fragments were ligated together into pING3197, which had been treated with EcoRI and XhoI to generate pING4642. The DNA sequence at the BspHI-T4-MBN/EagI junction revealed that two of the expected codons were missing but that the fusion protein was in frame.</p>
    <p>C. Construction of SCA(V<sub>H</sub>-V<sub>L</sub>)::SLT::Gelonin Vectors</p>
    <p>The expression vector containing the SCA(V<sub>H</sub>-V<sub>L</sub>)::SLT::Gelonin fusions was assembled using DNA from plasmids pING4636, (the <i>E. coli </i>expression vector for SCA(V<sub>H</sub>-V<sub>L</sub>)) and pING4407. Plasmid pING4636 was cut with BstEII and XhoI and the resulting vector fragment was purified. Concurrently, pINg4636 was used as a template for PCR with primers SCFV-7, 5′TGATGCGGCCGACATCTCAAGCTTGGTGC (SEQ ID NO: 112) and H65-G13, TGATGCGGCCGACATCTCAAGCTTGGTGC3′ (SEQ ID NO: 113). The amplified product was digested with EagI and BstEII and the resulting approximately 380 bp fragment was purified. Plasmid pING4407 was then cut with EagI and XhoI, resulting in an approximately 850 bp fragment, which was purified. The three above fragments were ligated together to produce pING4643.</p>
    <p>D. Construction of SCA(V<sub>L</sub>-V<sub>H</sub>::RMA::Gelonin Vectors</p>
    <p>Expression vectors containing SCA(<sub>L</sub>-V<sub>H</sub>)::RMA::Gelonin fusions were assembled using DNA from pING4640, pING4408 [Example 14A(iii)], and pING3825 (Example 2C). Plasmid pING4640 was cut with SalI and BstEII and the resulting approximately 700 bp vector fragment (containing the tetracycline resistance matter) was purified. Next, pING3825 was digested with NcoI and SalI, resulting in an approximately 1344 bp fragment containing the 3′ end of the gelonin gene and adjacent vector sequences. That fragment was purified. Plasmid pING4408 was then PCR amplified with oligonucleotide primers, RMA-G3 5′TCTAGGTCACCGTCTCCTCACCATCTGGACAGGCTGGA3′ (SEQ ID NO: 114), and gelo-10. The resulting PCR product was cut with BstEII and NcoI to generate an approximately 180 bp fragment containing the 3′ end of V<sub>H</sub>, RMA, and the 5′ end of the Gelonin gene which was purified. The above three fragments were ligated to generate the final expression vector, pING4644.</p>
    <p>E. Construction of SCA (V<sub>H</sub>-V<sub>L</sub>)::RMA::Gelonin Vectors</p>
    <p>Expression vectors containing SCA(V<sub>H</sub>-V<sub>L</sub>)::RMA::Gelonin were constructed using DNA from pING 4636, pING4410, and pING3825. Plasmid pING4636 was digested with SalI and HindIlI and the resulting vector fragment was purified. Next, pING3825 was cut with NcoI and SalI and the 1344 bp fragment which resulted contained the 3′ end of the gelonin gene and adjacent vector sequences encoding tetracycline resistance was purified. Finally, pING4410 was PCR amplified with primers RMA-G4, 5′TTCGAAGCTTGAGATGAAACCATCTGGACAGGCTGGA3′ (SEQ ID NO: 115) and gelo-10. The PCR product was cut with HindIII and NcoI, resulting in a 180 bp fragment containing the 3′ end of V<sub>L, </sub>RMA, and the 5′ end of Gelonin and was purified. The three above fragments were ligated together to generate the final expression vector, pING4645.</p>
    <p>Gelonin::SCA fusions without a cleavable linker may be constructed by deletion of the SLT linker in pING4412 using the restriction enzymes EagI and FspI. Digestion at these sites and religation of the plasmid results in an in-frame deletion of the SLT sequence.</p>
    <heading>EXAMPLE 17</heading> <p>Multivalent Immunofusions</p>
    <p>Multivalent forms of the immunofusions may be constructed.</p>
    <p>A. Construction of (Gel::RMA::kappa, Fd′)<sub>2 </sub>and (Gel::RMA::Fd′, kappa)<sub>2 </sub>Expression Vectors</p>
    <p>Bacterial Fab expression vectors can result in the production of F(ab′)<sub>2 </sub>if the two cysteine residues from the human IgG1 hinge region are included into the carboxyl-terminus of the Fd protein [Better et al., <i>Proc. Natl. Acad. Sci. USA, </i>90:457-461 (1993)]. To express a gelonin fusion protein that could form a bi-valent structure like an F(ab′)<sub>2</sub>, the he3 Fd′ (2C) hinge region (Better et al., supra) was cloned into the expression vector pING3764 (Example 15C) encoding the fusion protein Gel::RMA::kappa, Fd.</p>
    <p>Plasmid pING3764 was cut with XhoI and Bsu36I and the approximately 7500 bp fragment containing the immunofusion gene and vector sequences was purified. Plasmid pING4629, which encodes he3 F(ab′)<sub>2</sub>, was also cut with Bsu36I and XhoI, and the approximately 200 bp DNA fragment containing the he3 Fd′ (2C) gene segment was purified. These two DNA fragments were ligated to generate pING3775 encoding (Gel::RMA::kappa, Fd′)<sub>2</sub>. An expression vector encoding the fusion protein (Gel::RMA::Fd′, kappa)<sub>2 </sub>was also made.</p>
    <p>B. Construction of Vectors Containing Both Gel::RMA::Fd and Gel::RMA::K Fusions</p>
    <p>In order to construct a plasmid comprising Gel::RMA::Fd and Gel::RMA::k fusions, plasmid pING3764 [described above in Example 15(b)] was digested with BsgI and SauI and a 5.7 kb vector fragment containing plasmid replication functions, Gel::RMA::k, and the 3′ end of Fd was isolated and purified. Plasmid pING3768 [described above in Example 15(E)] was digested with SauI and partially digested with PstI and a 1.5 kb fragment containing Gel::RMA::Fd was purified. Finally, pING4000 [described above in Example 14] was digested with BsgI and PstI, generating a 350 bp fragment containing the 3′ end of the kappa gene. That fragment was purified and the 5.7 kb, 1.5 kb, and 350 kb fragments described above were ligated together to form pING3770, containing the gelonin::RMA::k and gelonin:RMA::Fd fusions.</p>
    <p>C. Construction of Vectors Containing Both Gel::SLT::Fd and Gel::SLT::k Fusions</p>
    <p>Plasmid pING3772 contains the above-entitled fusions and was constructed as follows. Plasmid pING3763 [described above in Example 15(D)] was digested with BsgI and SauI and a 5.7 kb fragment containing the replication functions, the 5′ end of Gel::SLT::k and the 3′ end of Fd was generated and purified. Next, plasmid pING3767 [described in Example 15(D) above] was digested with SauI and PstI, generating a 1.5 kb fragment containing the 5′ end of the gel::SLT::Fd fusion. That fragment was purified and pING4000 [described in Example 14 above] was digested with BsgI and PstI. The resulting 350 bp fragment was purified and the above-described 5.7 kb, 1.5 kb, and 350 bp fragments were ligated to form pING3772.</p>
    <p>D. Expression of Multivalent Fusions</p>
    <p>Both pING3770 and pING3772 were transformed into <i>E. coli </i>(E104) cells by techniques known to those of ordinary skill in the art and induced with arabinose. Concentrated supernatants from the transformed call cultures were analyzed by Western blot analysis with rabbit anti-gelonin antiserum. Transformants from both plasmids generated a reactive band on the gel at the size expected for a Fab molecule carrying two gelonins (approximately 105 kD). These results are consistent with the production of fusion proteins comprising monovalent Fab, with both Fd and kappa chains separately fused to gelonin.</p>
    <p> <i>E. coli </i>strains containing plasmids pING3775, pING3770 and pING3772 were grown in fermenters and the fusion protein products were purified. The (Gel::RMA::kappa,Fd′)<sub>2 </sub>expressed from pING3775 was purified as described in Better et al., supra.</p>
    <heading>EXAMPLE 18</heading> <p>Construction of Expression Vectors Encoding Immunofusions without Linkers</p>
    <p>Expression vectors encoding direct fusions of gelonin and dicistronic he3 Fab protein or single chain antibody were constructed as follows.</p>
    <p>A. V<sub>H</sub>V<sub>L</sub>::Gel</p>
    <p>Plasmid pING4642 (Example 16B) which encodes the V<sub>L</sub>V<sub>H</sub>::SLT::Gel fusion protein was cut with FspI and NcoI, and the approximately 100 bp DNA fragment containing the 5′-end of the gelonin gene was purified. Plasmid pING4643 (Example 16C), which encodes the V<sub>H</sub>V<sub>L</sub>::SLT::Gel fusion protein, was cut with EagI, treated with T4 polymerase and cut with PstI. The approximately 850 bp DNA fragment encoding the V<sub>H</sub>V<sub>L </sub>gene segment was purified. The DNA fragments from pING4642 and pING4643 were ligated into the vector DNA fragment from pING4644 (Example 16D) that had been cut with PstI and NcoI to generate pING3781, which encodes the V<sub>H</sub>V<sub>L</sub>::Gel direct gene fusion.</p>
    <p>B. V<sub>L</sub>V<sub>H</sub>::Gel</p>
    <p>Plasmid pING4640 which encodes the he3 SCA gene V<sub>L</sub>V<sub>H </sub>was cut with BspHI, treated with T4 polymerase in the presence of the nucleotide dCTP only, treated with mung bean nuclease to remove the remaining 5′ overhang, and then cut with EcoRI. The approximately 800 bp DNA fragment containing the he3 V<sub>L</sub>V<sub>H </sub>gene was then purified on an agarose gel.</p>
    <p>Plasmid pING3781 which encodes the direct fusion V<sub>H</sub>V<sub>L</sub>::Gel was digested with EagI, treated with T4 polymerase, and then digested with XhoI. The approximately 800 bp DNA fragment encoding the gelonin gene was then purified on an agarose gel. The two DNA fragments from pING4640 and pING3781 were ligated into the vector DNA from pING3767 which had been digested EcoRI and XhoI and purified on an agaraose gel. The resultant plasmid, pING3348, encoded the V<sub>L</sub>V<sub>H</sub>::Gel fusion protein. The DNA sequence at the fusion junction was verified by direct DNA sequencing.</p>
    <p>C. Gel::V<sub>H</sub>V<sub>L </sub> </p>
    <p>The plasmid pING3755 [Example 14B(iii)], which contains the gelonin gene with an engineered EagI site at its 3′-end, was cut with EagI, treated with T4 polymerase, and digested with NcoI. The approximately 650 bp DNA fragment containing the 3′-end of the gelonin gene was purified on an agarose gel. The plasmid pING4639 (Example 16A) encoding the fusion Gel::SLT::V<sub>H</sub>V<sub>L </sub>was cut with XhoI and then partially digested with FspI. The approximately 730 bp DNA fragment containing all of the he3 V<sub>H</sub>V<sub>L </sub>gene was then purified in an agarose gel (a single FspI restriction site occurs in the V<sub>H </sub>gene segment, and the purified he3 V<sub>H</sub>V<sub>L </sub>gene was separated from the incomplete gene segment which was approximately 660 bp). The two DNA fragments from pING3755 and pING4639 were ligated into the vector pING3825 that had been digested with NcoI plus XhoI and purified on an agarose gel. The plasmid pING3350 was generated which encoded the Gel::V<sub>H</sub>V<sub>L </sub>fusion protein. The DNA sequence at the fusion junction was verified by direct DNA sequencing.</p>
    <p>D. Gel::V<sub>L</sub>V<sub>H </sub> </p>
    <p>Plasmid pING3336 which encodes the he3 V<sub>L</sub>V<sub>H </sub>single chain antibody gene was cut with SstI and AseI, and the approximately 5500 bp DNA fragment containing the 3′-end of V<sub>L</sub>V<sub>H </sub>and downstream vector sequences was purified. (pING3336 is identical to pING4640 except that the V<sub>L</sub>V<sub>H </sub>gene encodes six histidine residues in frame at the carboxyl-terminus). Plasmid PING4627 (Example 15A) served as a substrate for PCR amplification of the V<sub>H </sub>gene segment. Plasmid pING4627 was amplified with the two oligonucleotide primers HUK-7 (SEQ ID NO: 92) and JK1-HindIII (SEQ ID NO: 87), the resultant product was treated with T4 polymerase and cut with Asel, and the 86 bp DNA fragment containing the 5′-end of the VL was purified. The DNa fragments from pING3336 and pING4627 were ligated to the approximately 2350 bp DNA fragment of pING3755 generated by digestion with EagI, treatment with T4 polymerase and subsequent digestion with SstI. The resultant vector containing the Gel::V<sub>L</sub>V<sub>H </sub>gene fusion was named pING4652. The DNA sequence of pING4652 was verified at ligation juctions.</p>
    <p>E. Gel::kappa, Fd</p>
    <p>The direct gene fusion which encodes Gel::kappa, Fd was also assembled from DNA segments from three plasmids. Plasmid pING3764 (Example 15C) was digested with HindIII and XhoI, and the approximately 1200 bp DNA fragment encoding the 3′-end of the kappa gene and the Fd gene was purified. Plasmid pING4652, which encodes a direct gene fusion of gelonin to the he3 SCA gene V<sub>L</sub>V<sub>H</sub>, was cut with BglII and HindIII, and the approximately 850 bp DNA fragment encoding the 3′-end of the gelonin gene and the V<sub>L </sub>region of kappa was purified. The DNA fragments from pING3764 and pING4652 were ligated into the vector fragment from pING3825 (Example 2C) that had been digested with BglII and XhoI to generate pING3784 encoding Gel::kappa, Fd.</p>
    <p>F. Gel::Fd, kappa</p>
    <p>Plasmid pING3768 (Example 15F), which encodes the fusion protein Gel::RMA::Fd, kappa, was cut with NdeI and NheI, and the DNA segment containing the majority of the he3 Fd gene, the he3 kappa gene and a portion of the tetracycline resistance gene of the vector was purified. Plasmid pING3350, which is described in section C above, was cut with NdeI and PstI, and the DNA fragment containing the 5′-end of the he3 Fd gene linked to the gelonin gene was purified. The DNA fragments from pING3350 and pING3768 were ligated into the vector fragment from pING4633 (Example 16D) that had been cut with Nhel and Pstl to generate pING3789. Plasmid pING3789 encodes the fusion protein Gel::Fd, kappa.</p>
    <heading>EXAMPLE 19</heading> <p>Alternative Cathepsin Cleavable Linkers</p>
    <p>The segment of rabbit muscle aldolase chosen for the RMA linker described herein is known to contain peptide sequences susceptible to digestion with cathepsins. Other cathepsin-cleavable protein segments are effective targets for intracellular cleavage, and two particular amino acid sequences were included as cleavable linkers in additional immunofusions of the invention. These are the amino acid sequence KPAKFFRL (SEQ ID NO: 141 (“CCF”) and KPAKFLRL (SEQ ID NO: 142) (“CCL”). Two oligonucleotides were synthesized that encode these peptide segments. Degeneracy was introduced at one nucleotide position in each synthetic primer to allow the appropriate amino acid to be encoded at the particular amino acid position in which CCF and CCL differ. The two oligonucleotides 5′-GGCCGCAAAGCCGGCTAAGTTCTT(A/C)CGTCTGAGT-3′ (SEQ ID NO: 143) and 5′-ACTCAGACG(G/T)AAGAACTTAGCCGGCTTTGC-3′ (SEQ ID NO: 144). The oligonucleotide linkers were then used to assemble a family of fusion gene expression vectors encoding: Gel::CCL::kappa, Fd; Gel::CCF::kappa, Fd; Gel::CCF::V<sub>L</sub>V<sub>H</sub>; and Gel::CCL::V<sub>H</sub>V<sub>L</sub>.</p>
    <p>The CCL and CCF linkers were also included in fusion vectors where the antigen-binding domain of the fusion protein was at the N-terminus of the fusion to generate expression vectors encoding immunofusions such as V<sub>L</sub>V<sub>H</sub>::CCL::Gel.</p>
    <p>Several of the fusion proteins with the CCL and CCF linkers were tested for cytotoxicity on the T cell lines HSB2 and PBMC and were comparable in activity to the fusion proteins containing the RMA linker.</p>
    <heading>EXAMPLE 20</heading> <p>Expression and Purification of Gelonin Immunofusions</p>
    <p>A. Expression of Gelonin Immunofusions</p>
    <p>Each of the gelonin gene fusions whose construction is described in Example 15 was co-expressed with its pair H65 Fab gene in arabinose-induced <i>E. coli </i>strain E104.</p>
    <p>Expression products of the gene fusions were detected in the supernatant of induced cultures by ELISA. Typically, a plate was coated with antibody recognizing gelonin. Culture supernatant was applied and bound Fab was detected with antibody recognizing human kappa coupled to horseradish peroxidase. H65 Fab fragment chemically conjugated to gelonin was used a standard. Alternative ELISA protocols involving coating a plate with antibody recognizing either the kappa or Fd or involving a detection step with anti-human Fd rather than anti-human kappa yielded similar results. Only properly assembled fusion protein containing gelonin, kappa and Pd was detected by this assay. Unassociated chains were not detected.</p>
    <p>The fusion protein produced from induced cultures containing expression vectors pING4406, 4407, 4408, and 4410 in <i>E. coli </i>E104 accumulated at about 20-50 ng/ml. The fusion proteins expressed upon induction of pING3754, 3334, 3758 and 3759 (but not pING3757) were expressed at much higher levels, at about 100 to 500 ng/ml. A fusion protein of about 70,000 Kd was detected in the concentrated <i>E. coli </i>culture supernatant by immunostaining of Western blots with either anti-human kappa or anti-gelonin antibodies.</p>
    <p>The Gelonin::SLT::Fd′ (kappa) fusion protein from pING3754 (ATCC 69102) was purified from induced 10 L fermentation broth. The 10 L fermentation broth was concentrated and buffer exchanged into 10 mM phosphate buffer at pH 7.0, using an S10Y10 cartridge (Amicon) and a DC10 concentrator. The supernatant was purified by passing the concentrated supernatant through a DE52 column (20×5 cm) equilibrated with 10 mM sodium phosphate buffer at pH 7.0. The flow-through was then further purified and concentrated by column chromatography on CM52 (5×10 cm) in 10 mM phosphate buffer. A 0-0.2 M linear gradient of NaCl was used to the elute the fusion protein, and fractions containing the fusion protein were pooled and loaded onto a Protein G column (1 ml). The fusion protein was eluted from protein G with 0.2 M sodium citrate, pH 5.5 and then 0.2 M sodium acetate, pH 4.5, and finally, 0.2 M glycine, pH 2.5. The Gelonin::RMA::Fd′ (kappa) and Gelonin::RMA::kappa (Fd′) fusions proteins were purified from fermentation broths by similar methods except that the CM52 column step was eliminated, and the DE52 column was equilibrated with 100 mM sodium phosphate buffer at pH 7.0. The fusion proteins were not purified to homogeneity.</p>
    <p>Each of the three purified fusion proteins was then assayed for activity in the RLA assay and for cytotoxicity against the T-cell line HSB2. (T cells express the CD5 antigen which is recognized by H65 antibody.) The RLA assay was performed as described in Example 4 and results of the assay are presented below in Table 12.</p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="offset" colwidth="35pt" align="left"> </colspec> <colspec colname="1" colwidth="84pt" align="left"> </colspec> <colspec colname="2" colwidth="98pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="OFFSET" nameend="2" rowsep="1" class="description-td" colspan="3">TABLE 12</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="OFFSET" nameend="2" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Fusion Protein</td>
                <td class="description-td">IC50 (pM)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="OFFSET" nameend="2" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">rGelonin</td>
                <td class="description-td">11</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Gelonin::SLT::Fd (kappa)</td>
                <td class="description-td">19</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Gelonin::RMA::Fd (kappa)</td>
                <td class="description-td">28</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Gelonin::RMA::kappa (Fd)</td>
                <td class="description-td">10</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="OFFSET" nameend="2" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p>Two fusion proteins were tested in whole cell cytotoxicity assays performed as described in Example 6 (Table 13). As shown in Table 13, the fusion proteins were active. Gelonin::SLT::Fd(kappa) killed two T cell lines, HSB2 and CEM, with respective IC<sub>50</sub>s 2-fold (HSB2) or 10-fold (CEM) higher than that of the gelonin chemically linked to H65. See Table 13 below for results wherein IC<sub>50 </sub>values were adjusted relative to the amount of fusion protein in each sample.</p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 13</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">IC<sub>50 </sub>(pMT)</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="4"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="84pt" align="left"> </colspec> <colspec colname="2" colwidth="63pt" align="center"> </colspec> <colspec colname="3" colwidth="56pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Fusion Protein</td>
                <td class="description-td">HSB2 Cells</td>
                <td class="description-td">CEM Cells</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="OFFSET" nameend="3" align="center" rowsep="1" class="description-td" colspan="4"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">he3Fab-Gel<sub>A50(C44)</sub> </td>
                <td class="description-td">165</td>
                <td class="description-td"> 173</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Gelonin::SLT::Fd (kappa)</td>
                <td class="description-td">180</td>
                <td class="description-td">1007</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Gelonin::RKA::Fd (kappa)</td>
                <td class="description-td">150</td>
                <td class="description-td">NT</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="OFFSET" nameend="3" align="center" rowsep="1" class="description-td" colspan="4"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="OFFSET" nameend="3" align="left" class="description-td" colspan="4">These fusion protein showed similar activity on peripheral blood mononuclear cells (data not shown). </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p>B. Purification of Immunofusions</p>
    <p>(i) Immunofusions Comprising cH65Fab′</p>
    <p>Immunofusions comprising a cH65Fab′ fragment were purified from cell-free supernatants by passing the supernatant through a CM Spheradex (Sepacor) column (5 cm×3 cm), equilibrated in 10 Mm Na phosphate at pH 7.0. Immunofusion proteins bind to the column and are eluted with 10 mM Na phosphate, 200 mM NaCl, pH 7.0. The eluate was diluted two-fold with 20 Mm HEPES, 3 M ammonium sulfate, pH 7.6 and loaded onto a phenyl sepharose fast flow (Pharmacia) column (2.5×3.5 cm), equilibrated in 20 mM HEPES, 1.2 M ammonium sulfate, pH 7.0. The column was next washed with 20 mM Hepes, 1.2 ammonium sulfate, pH 7.0 and eluted with 20 mM HEPES, 0.9 M ammonium sulfate, pH 7.0. The phenyl sepharose eluate was concentrated to a volume of 2-4 ml in an Amicon stirred cell fitted with a YM10 membrane. The concentrated sample was loaded onto an S-200 (Pharmacia) column (3.2×38 cm), equilibrated in 10 mm Na phosphate, 150 mm NaCl, pH 7.0. The column was run in the same buffer and fractions were collected. Fractions containing the fusion protein of desired molecular weight were combined. For example, by selection of appropriate column fractions, both monovalent (gelonin-Fab′) and bivalent (gelonin<sub>2</sub>-F(ab′)<sub>2 </sub>forms encoded by pING3758 were purified.</p>
    <p>(ii) Immunofusions Comprising he3Fab</p>
    <p>Immunofusions comprising he3Fab were purified as in the preceding section with the exception that the phenyl sepharose column was eluted with 20 mM HEPES, 1.0 M ammonium sulfate, pH 7.0.</p>
    <p>(iii) Immunofusions Comprising SCA</p>
    <p>Cell-free supernatant was passed through a CM spheradex column (5×3 cm), equilibrated with 10 mM Na phosphate, pH 7.0. Single-chain antibody binds to the column which is then washed with 10 mM Na phosphate, 45 mM NaCl, pH 7.0. The fusion protein was then eluted with 10 mM Naphosphate, 200 mM NaCl, pH 7.0. The eluate was diluted two-fold with 20 HEPES, 3 M ammonium sulfate, pH 7.0 and loaded onto a butyl sepharose Fast Flow (Pharmacia) column (2.5×4.1 cm) equilibrated in 20 mM HEPES, 1.5 M ammonium sulfate, pH 7.0. The column was then washed with 20 mM HEPES, 1.0 M ammonium sulfate, pH 7.0 and eluted with 20 mM HEPES pH 7.0. The butyl sepharose eluate was concentrated to a volume of 2-4 ml in an Amicon stirred cell fitted with a YM10 membrane. The concentrated sample was loaded onto an S-200 (Pharmacia) column (3.2×38 cm) equilibrated in 10 mM Na phosphate, 150 mM NaCl, pH 7.0. The column was then run in the same buffer and the fractions were collected. Some of the fractions were analyzed by SDS-PAGE to determine which fractions to pool together for the final product.</p>
    <heading>EXAMPLE 21</heading> <p>Activity of Gelonin Immunofusions</p>
    <p>A concern in constructing immunofusions comprising any RIP is that the targeting and enzymatic activities of the components of the fusion protein may be lost as a result of the fusion. For example, attachment of an RIP to the amino terminus of an antibody may affect the antigen-binding (complementarity-determining regions) of the antibody and may also result in steric hinderance at the active site. Similarly, the activity of an RIP may be hindered by attachment of an antibody or antibody portion. For example, RIPs chemically conjugated to antibodies via a disulfide bridge are typically inactive in the absence of reducing agents. In order to assess the foregoing in immunofusions of the present invention, such proteins were subjected to assays to determine their enzymatic, binding, and cytotoxic activities.</p>
    <p>A. Reticulocyte Lysate Assay</p>
    <p>The enzymatic activity of immunofusions comprising gelonin was assayed using the reticulocyte lysate assay (RLA) describe above. As noted in Example 4, the RLA assay measures the inhibition of protein synthesis in a cell-free system using endogenous globin mRNA from a rabbit red blood cell lysate. Decreased incorporation of tritiated leucine (<sup>3</sup>H-Leu) was measured as a function of toxin concentration. Serial log dilutions of standard toxin (the 30 kD form of ricin A-chain, abbreviated as RTA 30), native gelonin, recombinant gelonin (rGelonin or rGel) and gelonin analogs were tested over a range of 1 μg/ml to 1 pg/ml. Samples were tested in triplicate, prepared on ice, incubated for 30 minutes at 37° C., and then counted on an Inotec Trace 96 cascade ionization counter. By comparison with an uninhibited sample, the picomolar concentration of toxin (pM) which corresponds to 50% inhibition of protein synthesis (IC<sub>50</sub>) was calculated.</p>
    <p>Representative data for various immunotoxins of the invention are shown below in Table 14.</p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="4"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="112pt" align="left"> </colspec> <colspec colname="2" colwidth="28pt" align="center"> </colspec> <colspec colname="3" colwidth="63pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="OFFSET" nameend="3" rowsep="1" class="description-td" colspan="4">TABLE 14</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="OFFSET" nameend="3" align="center" rowsep="1" class="description-td" colspan="4"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Immunotoxin</td>
                <td class="description-td">Lot No.</td>
                <td class="description-td">IC<sub>50 </sub>(pM)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="OFFSET" nameend="3" align="center" rowsep="1" class="description-td" colspan="4"> </td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">rGel::RMA::SCA(V<sub>E</sub>-V<sub>L</sub>)</td>
                <td class="description-td">AB1136</td>
                <td class="description-td">12</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">rGel::RMA::SCA(V<sub>L</sub>-V<sub>E</sub>)</td>
                <td class="description-td">AB1137</td>
                <td class="description-td">18</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">rGel::SLT::SCA(V<sub>E</sub>-V<sub>L</sub>)</td>
                <td class="description-td">AB1133</td>
                <td class="description-td">26</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">rGel::RMA::SCA(V<sub>L</sub>-V<sub>E</sub>)</td>
                <td class="description-td">AB1124</td>
                <td class="description-td">33</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">rGel::RMA::K + Fd′ (cH65Fab′)</td>
                <td class="description-td">AB1122</td>
                <td class="description-td">54</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">rGel::SLT::K + Fd (he3Fab)</td>
                <td class="description-td">AB1160</td>
                <td class="description-td">40</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">rGel::RMA::K + Fd (he3Fab)</td>
                <td class="description-td">AB1149</td>
                <td class="description-td">33</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">rGel::RMA::Fd + K (he3Fab)</td>
                <td class="description-td">AB1163</td>
                <td class="description-td">14</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">rGel::Fd′ + K (cH65Fab′)</td>
                <td class="description-td">AB1123</td>
                <td class="description-td">45</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="OFFSET" nameend="3" align="center" rowsep="1" class="description-td" colspan="4"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p>Contrary to the expectations discussed above, gelonin immunofusions of the invention exhibit enzymatic activity which is comparable to the activities of native and recombinant gelonin shown in Example 4. This was true for fusions made with either the reducible (SLT) or non-reducible (RMA) linkers.</p>
    <p>B. Binding Activity of Immunofusions</p>
    <p>Several immunofusions according to the present invention were assayed for their ability to compete with labelled antibody for binding to CD5-positive cells. The Kd of the immunofusions was estimated by three different means as shown in Table 15. The first Kd estimation (Kd<sub>1 </sub>in Table 15) was obtained by competition with fluorescein-labelled H65 IgG for binding to MOLT-4X cells (ATCC CRL 1582) according to the procedure reported in Knebel, et al., <i>Cytometry Suppl., </i>1: 68 (1987), incorporated by reference herein.</p>
    <p>The second Kd measurement (Kd<sub>2 </sub>in Table 15) was obtained by Scatchard analysis of competition of the immunofusion with <sup>125</sup>I-cH65 IgG for binding on MOLT-4M cells as follows. A 20 μg aliquot of chimeric H65 IgG (cH65 IgG) was iodinated by exposure to 100 μl lactoperoxidase-glucose oxidase immobilized beads (Enzymobeads, BioRad), 100 μl of PBS, 1.0 mCi I<sup>125 </sup>(Amersham, IMS30), 50 μl of 55 mM b-D-glucose for 45 minutes at 23° C. The reaction was quenched by the addition of 20 μl of 105 mM sodium metabisulfite and 120 mM potassium iodine followed by centrifugation for 1 minute to pellet the beads. <sup>125</sup>I-cH65 IgG was purified by gel filtration using a 7 ml column of sephadex G25, eluted with PBS (137 mM NaCl, 1.47 mM KH<sub>2</sub>PO<sub>4</sub>, 8.1 mM Na<sub>2</sub>HPO<sub>4</sub>, 2.68 mM KCl at pH 7.2-7.4) plus 0.1% BSA. <sup>125</sup>I-cH65 IgG recovery and specific activity were determined by TCA precipitation.</p>
    <p>Competitive binding was performed as follows: 100 μl of Molt-4M cells were washed two times in ice-cold DHB binding buffer (Dubellco's modified Eagle's medium (Gibco, 320-1965PJ), 1.0% BSA and 10 mM Hepes at pH 7.2-7.4). Cells were resuspended in the same buffer, plated into 96 v-bottomed wells (Costar) at 3×10<sup>5 </sup>cells per well and pelleted at 4° C. by centrifugation for 5 min at 1,000 rpm using a Beckman JS 4.2 rotor; 50 μl of 2×-concentrated 0.1 nM <sup>125</sup>I-cH65 IgG in DHB was then added to each well and competed with 50 μl of 2×-concentrated cH65 IgG in DHB at final protein concentrations from 100 nM to 0.0017 nM. The concentrations of assayed proteins were determined by measuring absorbance (A<sub>280 </sub>and using an extinction coefficient of 1.0 for fusion proteins, 1.3 for Fab, and 1.22 for Fab conjugated to gelonin. Also, protein concentrations were determined by BCA assay (Pierce Chemical) with bovine serum albumin as the standard. Binding was allowed to proceed at 4° C. for 5 hrs and was terminated by washing cells three times with 200 μl of DHB binding buffer by centrifugation for 5 min. at 1,000 rpm. All buffers and operations were at 4° C. Radioactivity was determined by solubilizing cells in 100 μl of 1.0 M NaOH and counting in a Cobra II auto gamma counter (Packard). Data from binding experiments were analyzed by the weighted nonlinear least squares curve fitting program, MacLigand, a Macintosh version of the computer program “Ligand” from Munson, <i>Analyt. Biochem., </i>107:220 (1980), incorporated by reference herein.</p>
    <p>Finally, the Kd (Kd<sub>3 </sub>in the Table) was estimated by examination of the ED<sub>50 </sub>values obtained from separate competition binding assays, performed as described in the previous paragraph. All three measurements are shown in Table 15 below:</p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="4"> <colspec colname="1" colwidth="98pt" align="left"> </colspec> <colspec colname="2" colwidth="49pt" align="center"> </colspec> <colspec colname="3" colwidth="28pt" align="center"> </colspec> <colspec colname="4" colwidth="42pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="4" rowsep="1" class="description-td" colspan="4">TABLE 15</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="4" align="center" rowsep="1" class="description-td" colspan="4"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Molecule Type</td>
                <td class="description-td">Kd<sub>1</sub> </td>
                <td class="description-td">Kd<sub>2</sub> </td>
                <td class="description-td">Kd<sub>3</sub> </td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="4" align="center" rowsep="1" class="description-td" colspan="4"> </td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">H65 IgG</td>
                <td class="description-td">1.6 </td>
                <td class="description-td">ND</td>
                <td class="description-td">ND</td>
              </tr> <tr class="description-tr"> <td class="description-td">cH65 IgG</td>
                <td class="description-td">ND</td>
                <td class="description-td">3.0</td>
                <td class="description-td">  2.5</td>
              </tr> <tr class="description-tr"> <td class="description-td">cH65Fab′</td>
                <td class="description-td">4.0 </td>
                <td class="description-td">14.0 </td>
                <td class="description-td">ND</td>
              </tr> <tr class="description-tr"> <td class="description-td">cH65Fab′-rGel<sub>A50(C44)</sub> </td>
                <td class="description-td">3.5 </td>
                <td class="description-td">13.0 </td>
                <td class="description-td">ND</td>
              </tr> <tr class="description-tr"> <td class="description-td">rGel::RMA::K + Fd′ (cH65Fab′)</td>
                <td class="description-td">16.0 </td>
                <td class="description-td">ND</td>
                <td class="description-td">100 </td>
              </tr> <tr class="description-tr"> <td class="description-td">he3Fab</td>
                <td class="description-td">1.20</td>
                <td class="description-td"> 2.60</td>
                <td class="description-td">ND</td>
              </tr> <tr class="description-tr"> <td class="description-td">he3Fab-rGel<sub>A50(C44)</sub> </td>
                <td class="description-td">1.10</td>
                <td class="description-td"> 2.70</td>
                <td class="description-td">ND</td>
              </tr> <tr class="description-tr"> <td class="description-td">rGel::RMA::K + Fd′ (he3Fab)</td>
                <td class="description-td">2.60</td>
                <td class="description-td">ND</td>
                <td class="description-td">  5.0</td>
              </tr> <tr class="description-tr"> <td class="description-td">rGel::SLT::K + Fd (he3Fab)</td>
                <td class="description-td">ND</td>
                <td class="description-td">ND</td>
                <td class="description-td">30</td>
              </tr> <tr class="description-tr"> <td class="description-td">SCA(V<sub>L</sub>-V<sub>E</sub>)</td>
                <td class="description-td">2.20</td>
                <td class="description-td">ND</td>
                <td class="description-td">30</td>
              </tr> <tr class="description-tr"> <td class="description-td">rGel::RMA::SCA(V<sub>E</sub>-V<sub>L</sub>)</td>
                <td class="description-td">3.50</td>
                <td class="description-td">ND</td>
                <td class="description-td">20</td>
              </tr> <tr class="description-tr"> <td class="description-td">rGel::RMA::SCA(V<sub>L</sub>-V<sub>E</sub>)</td>
                <td class="description-td">4.70</td>
                <td class="description-td">ND</td>
                <td class="description-td">30</td>
              </tr> <tr class="description-tr"> <td class="description-td">SCA(V<sub>L</sub>-V<sub>E</sub>)</td>
                <td class="description-td">ND</td>
                <td class="description-td">ND</td>
                <td class="description-td">20</td>
              </tr> <tr class="description-tr"> <td class="description-td">rGel::RMA::SCA(V<sub>L</sub>-V<sub>E</sub>)</td>
                <td class="description-td">2.30</td>
                <td class="description-td">ND</td>
                <td class="description-td">ND</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="4" align="center" rowsep="1" class="description-td" colspan="4"> </td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="4" align="left" class="description-td" colspan="4">ND = not determined </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p>The results presented in Table 15 suggest that Fab and SCA antibody forms may retain substantial binding activity even when fused to an RIP.</p>
    <p>C. Comparative Cytotoxicity Assays</p>
    <p>Fusion proteins and immunoconjugates according to the present invention were used in a comparative cytoxicity assay. Two types of assays were conducted, one targeting T cell line HSB2, and the other targeting lectin-activated peripheral blood mononuclear cells (PBMC) according to procedures in Example 6. The results of the assays are presented below in Tables 16a, 16b and 16c.</p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 16a</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">CYTOTOXIC POTENCIES:</td>
              </tr> <tr class="description-tr"> <td class="description-td">CHEMICAL VS. GENE-FUSED CONJUGATES</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="4"> <colspec colname="1" colwidth="56pt" align="left"> </colspec> <colspec colname="2" colwidth="35pt" align="left"> </colspec> <colspec colname="3" colwidth="35pt" align="left"> </colspec> <colspec colname="4" colwidth="91pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">HSB2</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="6"> <colspec colname="1" colwidth="56pt" align="left"> </colspec> <colspec colname="2" colwidth="35pt" align="left"> </colspec> <colspec colname="3" colwidth="35pt" align="left"> </colspec> <colspec colname="4" colwidth="35pt" align="left"> </colspec> <colspec colname="5" colwidth="35pt" align="left"> </colspec> <colspec colname="6" colwidth="21pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">Fusion</td>
                <td class="description-td">IC<sub>50</sub>,</td>
                <td class="description-td">IC<sub>50</sub>,</td>
                <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Immunotoxin</td>
                <td class="description-td">Lot #</td>
                <td class="description-td">Plasmid</td>
                <td class="description-td">pM Toxin</td>
                <td class="description-td">pM Toxin</td>
                <td class="description-td">N</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="6" align="center" rowsep="1" class="description-td" colspan="6"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="6"> <colspec colname="1" colwidth="56pt" align="left"> </colspec> <colspec colname="2" colwidth="35pt" align="left"> </colspec> <colspec colname="3" colwidth="35pt" align="left"> </colspec> <colspec colname="4" colwidth="35pt" align="left"> </colspec> <colspec colname="5" colwidth="35pt" align="char" char="."> </colspec> <colspec colname="6" colwidth="21pt" align="char" char="."> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">CD5 Plus</td>
                <td class="description-td">HF002002</td>
                <td class="description-td">—</td>
                <td class="description-td">   148</td>
                <td class="description-td">24</td>
                <td class="description-td">8</td>
              </tr> <tr class="description-tr"> <td class="description-td">H65-M-</td>
                <td class="description-td">999</td>
                <td class="description-td">—</td>
                <td class="description-td">   68*</td>
                <td class="description-td">NA</td>
                <td class="description-td">1</td>
              </tr> <tr class="description-tr"> <td class="description-td">rGel<sub>A50(C44)</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td">cH65-MM-rGel</td>
                <td class="description-td">807</td>
                <td class="description-td">—</td>
                <td class="description-td">   183*</td>
                <td class="description-td">NA</td>
                <td class="description-td">1</td>
              </tr> <tr class="description-tr"> <td class="description-td">cH65Fab′-M-</td>
                <td class="description-td">941</td>
                <td class="description-td">—</td>
                <td class="description-td">   99</td>
                <td class="description-td">6</td>
                <td class="description-td">2</td>
              </tr> <tr class="description-tr"> <td class="description-td">rGel<sub>A50(C44)</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td">He2Fab-M-</td>
                <td class="description-td">970</td>
                <td class="description-td">—</td>
                <td class="description-td">   468</td>
                <td class="description-td">195</td>
                <td class="description-td">4</td>
              </tr> <tr class="description-tr"> <td class="description-td">rGel<sub>A50(C44)</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td">he3Fab-M-</td>
                <td class="description-td">1012/1047</td>
                <td class="description-td">—</td>
                <td class="description-td">   190</td>
                <td class="description-td">70</td>
                <td class="description-td">12</td>
              </tr> <tr class="description-tr"> <td class="description-td">rGel<sub>A50(C44)</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td">he3Fab-SMCC-</td>
                <td class="description-td">1086</td>
                <td class="description-td">—</td>
                <td class="description-td"> 5,904</td>
                <td class="description-td">2,442</td>
                <td class="description-td">2</td>
              </tr> <tr class="description-tr"> <td class="description-td">rGel<sub>A50(C44)</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td">rGel::SLT::Fd′ +</td>
                <td class="description-td">AB1095</td>
                <td class="description-td">pING3754</td>
                <td class="description-td">   320</td>
                <td class="description-td">25</td>
                <td class="description-td">2</td>
              </tr> <tr class="description-tr"> <td class="description-td">K(1)<sup>+</sup> </td>
              </tr> <tr class="description-tr"> <td class="description-td">rGel::SLT::Fd′ +</td>
                <td class="description-td">AB1095</td>
                <td class="description-td">pING3754</td>
                <td class="description-td">   374*</td>
                <td class="description-td">NA</td>
                <td class="description-td">1</td>
              </tr> <tr class="description-tr"> <td class="description-td">K(3)<sup>+</sup> </td>
              </tr> <tr class="description-tr"> <td class="description-td">rGel::SLT::K +</td>
                <td class="description-td">AB1147</td>
                <td class="description-td">pING3763</td>
                <td class="description-td">   495*</td>
                <td class="description-td">NA</td>
                <td class="description-td">1</td>
              </tr> <tr class="description-tr"> <td class="description-td">Fd(he3)</td>
              </tr> <tr class="description-tr"> <td class="description-td">rGel::SLT::K +</td>
                <td class="description-td">AB1160</td>
                <td class="description-td">pING3763</td>
                <td class="description-td">   746*</td>
                <td class="description-td">NA</td>
                <td class="description-td">1</td>
              </tr> <tr class="description-tr"> <td class="description-td">Fd(he3)</td>
              </tr> <tr class="description-tr"> <td class="description-td">rGel::SLT::</td>
                <td class="description-td">AB1133</td>
                <td class="description-td">pING4639</td>
                <td class="description-td">   422</td>
                <td class="description-td">31</td>
                <td class="description-td">5</td>
              </tr> <tr class="description-tr"> <td class="description-td">SCA(Vh-Vl)</td>
              </tr> <tr class="description-tr"> <td class="description-td">rGel::SLT::</td>
                <td class="description-td">AB1124</td>
                <td class="description-td">pING4412</td>
                <td class="description-td">   776</td>
                <td class="description-td">347</td>
                <td class="description-td">3</td>
              </tr> <tr class="description-tr"> <td class="description-td">SCA(Vl-Vh)</td>
              </tr> <tr class="description-tr"> <td class="description-td">rGel::RMA::K +</td>
                <td class="description-td">AB1122</td>
                <td class="description-td">pING3758</td>
                <td class="description-td"> 1,506</td>
                <td class="description-td">1,033</td>
                <td class="description-td">2</td>
              </tr> <tr class="description-tr"> <td class="description-td">Fd′</td>
              </tr> <tr class="description-tr"> <td class="description-td">rGel::RMA::K +</td>
                <td class="description-td">AB1141</td>
                <td class="description-td">pING3758</td>
                <td class="description-td"> 5,833*</td>
                <td class="description-td">NA</td>
                <td class="description-td">1</td>
              </tr> <tr class="description-tr"> <td class="description-td">Fd′</td>
              </tr> <tr class="description-tr"> <td class="description-td">rGel::RMA::K +</td>
                <td class="description-td">AB1149</td>
                <td class="description-td">pING3764</td>
                <td class="description-td"> 9,154*</td>
                <td class="description-td">NA</td>
                <td class="description-td">1</td>
              </tr> <tr class="description-tr"> <td class="description-td">Fd(he3)</td>
              </tr> <tr class="description-tr"> <td class="description-td">rGel::RMA::K +</td>
                <td class="description-td">AB1161</td>
                <td class="description-td">pING3764</td>
                <td class="description-td"> 5,974*</td>
                <td class="description-td">NA</td>
                <td class="description-td">1</td>
              </tr> <tr class="description-tr"> <td class="description-td">Fd(he3)</td>
              </tr> <tr class="description-tr"> <td class="description-td">rGel::RMA::Fd′ +</td>
                <td class="description-td">RF524(1)</td>
                <td class="description-td">pING3759</td>
                <td class="description-td"> 1,955*</td>
                <td class="description-td">NA</td>
                <td class="description-td">1</td>
              </tr> <tr class="description-tr"> <td class="description-td">K</td>
              </tr> <tr class="description-tr"> <td class="description-td">rGel::RMA::Fd′ +</td>
                <td class="description-td">AB1121</td>
                <td class="description-td">pING3759</td>
                <td class="description-td"> 32,051*</td>
                <td class="description-td">NA</td>
                <td class="description-td">1</td>
              </tr> <tr class="description-tr"> <td class="description-td">K</td>
              </tr> <tr class="description-tr"> <td class="description-td">rGel::RMA::Fd +</td>
                <td class="description-td">AB1163</td>
                <td class="description-td">pING3768</td>
                <td class="description-td"> 3,256*</td>
                <td class="description-td">NA</td>
                <td class="description-td">1</td>
              </tr> <tr class="description-tr"> <td class="description-td">K(he3)</td>
              </tr> <tr class="description-tr"> <td class="description-td">rGel::RMA::</td>
                <td class="description-td">AB1136</td>
                <td class="description-td">pING4638</td>
                <td class="description-td"> 3,687</td>
                <td class="description-td">1,144</td>
                <td class="description-td">6</td>
              </tr> <tr class="description-tr"> <td class="description-td">SCA(Vh-Vl)</td>
              </tr> <tr class="description-tr"> <td class="description-td">rGel::RMA::</td>
                <td class="description-td">AB1152</td>
                <td class="description-td">pING4638</td>
                <td class="description-td"> 41,218*</td>
                <td class="description-td">NA</td>
                <td class="description-td">1</td>
              </tr> <tr class="description-tr"> <td class="description-td">SCA(Vh-Vl)</td>
              </tr> <tr class="description-tr"> <td class="description-td">rGel::RMA::</td>
                <td class="description-td">AB1137</td>
                <td class="description-td">pING4637</td>
                <td class="description-td"> 11,979*</td>
                <td class="description-td">NA</td>
                <td class="description-td">1</td>
              </tr> <tr class="description-tr"> <td class="description-td">SCA(Vl-Vh)</td>
              </tr> <tr class="description-tr"> <td class="description-td">rGel::RMA::</td>
                <td class="description-td">AB1164</td>
                <td class="description-td">pING4637</td>
                <td class="description-td"> 1,146*</td>
                <td class="description-td">NA</td>
                <td class="description-td">1</td>
              </tr> <tr class="description-tr"> <td class="description-td">SCA(Vl-Vh)</td>
              </tr> <tr class="description-tr"> <td class="description-td">rGel::Fd′ +</td>
                <td class="description-td">AB1123</td>
                <td class="description-td">pING3334</td>
                <td class="description-td"> 6,346*</td>
                <td class="description-td">NA</td>
                <td class="description-td">1</td>
              </tr> <tr class="description-tr"> <td class="description-td">K</td>
              </tr> <tr class="description-tr"> <td class="description-td">K::RMA::rGel +</td>
                <td class="description-td">AB1140</td>
                <td class="description-td">pING4410</td>
                <td class="description-td"> 10,090*</td>
                <td class="description-td">NA</td>
                <td class="description-td">1</td>
              </tr> <tr class="description-tr"> <td class="description-td">Fd</td>
              </tr> <tr class="description-tr"> <td class="description-td">rGel</td>
                <td class="description-td">1056</td>
                <td class="description-td">—</td>
                <td class="description-td"> 46,600</td>
                <td class="description-td">34,600</td>
                <td class="description-td">3</td>
              </tr> <tr class="description-tr"> <td class="description-td">B72.3Fab-M-</td>
                <td class="description-td">1057</td>
                <td class="description-td">—</td>
                <td class="description-td">129,032*</td>
                <td class="description-td">NA</td>
                <td class="description-td">1</td>
              </tr> <tr class="description-tr"> <td class="description-td">rGel<sub>A50(C44)</sub> </td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="6" align="center" rowsep="1" class="description-td" colspan="6"> </td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="6" align="left" class="description-td" colspan="6">*Results represent single values and not a mean value. </td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="6" align="left" class="description-td" colspan="6"> <sup>+</sup>rGel::SLT::Fd′ + k(1) and rGel::SLT::Fd′ + k(3) are separate fractions from the final purification column. </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 16b</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">CYTOTOXIC POTENCIES:</td>
              </tr> <tr class="description-tr"> <td class="description-td">CHEMICAL VS. GENE-FUSED CONJUGATES</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="1" colwidth="56pt" align="left"> </colspec> <colspec colname="2" colwidth="35pt" align="left"> </colspec> <colspec colname="3" colwidth="126pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">PBMC</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="6"> <colspec colname="1" colwidth="56pt" align="left"> </colspec> <colspec colname="2" colwidth="35pt" align="left"> </colspec> <colspec colname="3" colwidth="35pt" align="center"> </colspec> <colspec colname="4" colwidth="35pt" align="center"> </colspec> <colspec colname="5" colwidth="35pt" align="center"> </colspec> <colspec colname="6" colwidth="21pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">IC<sub>50</sub>,</td>
                <td class="description-td">IC<sub>50</sub>,</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Immunotoxin</td>
                <td class="description-td">Lot #</td>
                <td class="description-td">pM Toxin</td>
                <td class="description-td">pM Toxin</td>
                <td class="description-td">SD</td>
                <td class="description-td">N</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="6" align="center" rowsep="1" class="description-td" colspan="6"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="6"> <colspec colname="1" colwidth="56pt" align="left"> </colspec> <colspec colname="2" colwidth="35pt" align="left"> </colspec> <colspec colname="3" colwidth="35pt" align="char" char="."> </colspec> <colspec colname="4" colwidth="35pt" align="char" char="."> </colspec> <colspec colname="5" colwidth="35pt" align="char" char="."> </colspec> <colspec colname="6" colwidth="21pt" align="char" char="."> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">CD5 Plus</td>
                <td class="description-td">HF002002</td>
                <td class="description-td">1,095</td>
                <td class="description-td">1,236</td>
                <td class="description-td">908</td>
                <td class="description-td">18</td>
              </tr> <tr class="description-tr"> <td class="description-td">H65-m-</td>
                <td class="description-td">999</td>
                <td class="description-td">133</td>
                <td class="description-td">133</td>
                <td class="description-td">129</td>
                <td class="description-td">2</td>
              </tr> <tr class="description-tr"> <td class="description-td">rGel<sub>A50(C44)</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td">cH65-m2-rGel</td>
                <td class="description-td">807</td>
                <td class="description-td">143</td>
                <td class="description-td">308</td>
                <td class="description-td">492</td>
                <td class="description-td">8</td>
              </tr> <tr class="description-tr"> <td class="description-td">cFab′-rGel<sub>A50(C44)</sub> </td>
                <td class="description-td">941</td>
                <td class="description-td">434</td>
                <td class="description-td">405</td>
                <td class="description-td">280</td>
                <td class="description-td">4</td>
              </tr> <tr class="description-tr"> <td class="description-td">He2Fab-</td>
                <td class="description-td">970</td>
                <td class="description-td">397</td>
                <td class="description-td">397</td>
                <td class="description-td">146</td>
                <td class="description-td">2</td>
              </tr> <tr class="description-tr"> <td class="description-td">rGel<sub>A50(C44)</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td">he3Fab-</td>
                <td class="description-td">1012/1047</td>
                <td class="description-td">206</td>
                <td class="description-td">307</td>
                <td class="description-td">274</td>
                <td class="description-td">18</td>
              </tr> <tr class="description-tr"> <td class="description-td">rGel<sub>A50(C44)</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td">he3Fab-smcc-</td>
                <td class="description-td">1086</td>
                <td class="description-td">335</td>
                <td class="description-td">638</td>
                <td class="description-td">538</td>
                <td class="description-td">3</td>
              </tr> <tr class="description-tr"> <td class="description-td">rGel<sub>A50(C44)</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td">rGel::SLT::Fd′ +</td>
                <td class="description-td">AB1095</td>
                <td class="description-td">15,840</td>
                <td class="description-td">15,840</td>
                <td class="description-td">15,783</td>
                <td class="description-td">2</td>
              </tr> <tr class="description-tr"> <td class="description-td">K(1)<sup>+</sup> </td>
              </tr> <tr class="description-tr"> <td class="description-td">rGel::SLT;:Fd′ +</td>
                <td class="description-td">AB1095</td>
                <td class="description-td">2,350</td>
                <td class="description-td">4,322</td>
                <td class="description-td">4,159</td>
                <td class="description-td">9</td>
              </tr> <tr class="description-tr"> <td class="description-td">K(3)<sup>+</sup> </td>
              </tr> <tr class="description-tr"> <td class="description-td">rGel::SLT::K +</td>
                <td class="description-td">AB1147</td>
                <td class="description-td">1,890</td>
                <td class="description-td">1,407</td>
                <td class="description-td">1,015</td>
                <td class="description-td">5</td>
              </tr> <tr class="description-tr"> <td class="description-td">Fd(he3)</td>
              </tr> <tr class="description-tr"> <td class="description-td">rGel::SLT::K +</td>
                <td class="description-td">AB1160</td>
                <td class="description-td">2,910</td>
                <td class="description-td">4,584</td>
                <td class="description-td">5,100</td>
                <td class="description-td">3</td>
              </tr> <tr class="description-tr"> <td class="description-td">Fd(he3)</td>
              </tr> <tr class="description-tr"> <td class="description-td">rGel::SLT::</td>
                <td class="description-td">AB1133</td>
                <td class="description-td">1,125</td>
                <td class="description-td">1,870</td>
                <td class="description-td">1,637</td>
                <td class="description-td">6</td>
              </tr> <tr class="description-tr"> <td class="description-td">SCA(Vh-Vl)</td>
              </tr> <tr class="description-tr"> <td class="description-td">rGel::SLT::</td>
                <td class="description-td">AB1124</td>
                <td class="description-td">2,725</td>
                <td class="description-td">2,815</td>
                <td class="description-td">743</td>
                <td class="description-td">4</td>
              </tr> <tr class="description-tr"> <td class="description-td">SCA(Vl-Vh)</td>
              </tr> <tr class="description-tr"> <td class="description-td">rGel::RMA::K +</td>
                <td class="description-td">AB1122</td>
                <td class="description-td">211</td>
                <td class="description-td">307</td>
                <td class="description-td">250</td>
                <td class="description-td">14</td>
              </tr> <tr class="description-tr"> <td class="description-td">Fd′</td>
              </tr> <tr class="description-tr"> <td class="description-td">rGel::RMA::K +</td>
                <td class="description-td">AB1141</td>
                <td class="description-td">4,400</td>
                <td class="description-td">4,041</td>
                <td class="description-td">2,691</td>
                <td class="description-td">4</td>
              </tr> <tr class="description-tr"> <td class="description-td">Fd′</td>
              </tr> <tr class="description-tr"> <td class="description-td">rGel::RMA::K +</td>
                <td class="description-td">RF-532</td>
                <td class="description-td">15,000</td>
                <td class="description-td">9,114</td>
                <td class="description-td">8,325</td>
                <td class="description-td">3</td>
              </tr> <tr class="description-tr"> <td class="description-td">Fd′</td>
              </tr> <tr class="description-tr"> <td class="description-td">rGel::RMA::K +</td>
                <td class="description-td">AB1149</td>
                <td class="description-td">7,124</td>
                <td class="description-td">10,764</td>
                <td class="description-td">14,081</td>
                <td class="description-td">5</td>
              </tr> <tr class="description-tr"> <td class="description-td">Fd(he3)</td>
              </tr> <tr class="description-tr"> <td class="description-td">rGel::RMA::K +</td>
                <td class="description-td">AB1161</td>
                <td class="description-td">1,854</td>
                <td class="description-td">2,990</td>
                <td class="description-td">3,324</td>
                <td class="description-td">3</td>
              </tr> <tr class="description-tr"> <td class="description-td">Fd(he3)</td>
              </tr> <tr class="description-tr"> <td class="description-td">rGel::RMA::Fd′ +</td>
                <td class="description-td">RF524(1)</td>
                <td class="description-td">1,760</td>
                <td class="description-td">1,893</td>
                <td class="description-td">1,049</td>
                <td class="description-td">5</td>
              </tr> <tr class="description-tr"> <td class="description-td">K</td>
              </tr> <tr class="description-tr"> <td class="description-td">rGel::RMA::Fd′ +</td>
                <td class="description-td">AB1121</td>
                <td class="description-td">2,090</td>
                <td class="description-td">1,664</td>
                <td class="description-td">1,553</td>
                <td class="description-td">6</td>
              </tr> <tr class="description-tr"> <td class="description-td">K</td>
              </tr> <tr class="description-tr"> <td class="description-td">rGel::RMA::Fd +</td>
                <td class="description-td">AB1163</td>
                <td class="description-td">854</td>
                <td class="description-td">567</td>
                <td class="description-td">406</td>
                <td class="description-td">2</td>
              </tr> <tr class="description-tr"> <td class="description-td">K(he3)</td>
              </tr> <tr class="description-tr"> <td class="description-td">rGel::RMA::</td>
                <td class="description-td">AB1136</td>
                <td class="description-td">393</td>
                <td class="description-td">567</td>
                <td class="description-td">510</td>
                <td class="description-td">7</td>
              </tr> <tr class="description-tr"> <td class="description-td">SCA(Vh-Vl)</td>
              </tr> <tr class="description-tr"> <td class="description-td">rGel::RMA::</td>
                <td class="description-td">AB1152</td>
                <td class="description-td">9,650</td>
                <td class="description-td">9,170</td>
                <td class="description-td">6,483</td>
                <td class="description-td">3</td>
              </tr> <tr class="description-tr"> <td class="description-td">SCA(Vh-Vl)</td>
              </tr> <tr class="description-tr"> <td class="description-td">rGel::RMA::</td>
                <td class="description-td">AB1137</td>
                <td class="description-td">4,040</td>
                <td class="description-td">4,554</td>
                <td class="description-td">4,310</td>
                <td class="description-td">7</td>
              </tr> <tr class="description-tr"> <td class="description-td">SCA(Vl-Vh)</td>
              </tr> <tr class="description-tr"> <td class="description-td">rGel::RMA::</td>
                <td class="description-td">AB1164</td>
                <td class="description-td">1,598</td>
                <td class="description-td">1,598</td>
                <td class="description-td">1,144</td>
                <td class="description-td">2</td>
              </tr> <tr class="description-tr"> <td class="description-td">SCA(Vl-Vh)</td>
              </tr> <tr class="description-tr"> <td class="description-td">rGel::Fd′ +</td>
                <td class="description-td">AB1123</td>
                <td class="description-td">2,606</td>
                <td class="description-td">2,777</td>
                <td class="description-td">2,167</td>
                <td class="description-td">4</td>
              </tr> <tr class="description-tr"> <td class="description-td">K</td>
              </tr> <tr class="description-tr"> <td class="description-td">K::RMA::rGel +</td>
                <td class="description-td">AB1140</td>
                <td class="description-td">1,545</td>
                <td class="description-td">1,545</td>
                <td class="description-td">417</td>
                <td class="description-td">2</td>
              </tr> <tr class="description-tr"> <td class="description-td">Fd</td>
              </tr> <tr class="description-tr"> <td class="description-td">rGel</td>
                <td class="description-td">1056</td>
                <td class="description-td">13,350</td>
                <td class="description-td">40,233</td>
                <td class="description-td">43,048</td>
                <td class="description-td">6</td>
              </tr> <tr class="description-tr"> <td class="description-td">8B2.3Fab-m-</td>
                <td class="description-td">1057</td>
                <td class="description-td">12,400</td>
                <td class="description-td">13,174</td>
                <td class="description-td">14,339</td>
                <td class="description-td">11</td>
              </tr> <tr class="description-tr"> <td class="description-td">rGel<sub>A50(C44)</sub> </td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="6" align="center" rowsep="1" class="description-td" colspan="6"> </td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="6" align="left" class="description-td" colspan="6">*Results represent single values and not a mean value. </td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="6" align="left" class="description-td" colspan="6"> <sup>+</sup>rGel::SLT::Fd′ + k(1) and rGel::SLT::Fd′ + k(3) are separate fractions from the final purification column. </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="6"> <colspec colname="1" colwidth="35pt" align="left"> </colspec> <colspec colname="2" colwidth="84pt" align="left"> </colspec> <colspec colname="3" colwidth="35pt" align="center"> </colspec> <colspec colname="4" colwidth="14pt" align="center"> </colspec> <colspec colname="5" colwidth="28pt" align="center"> </colspec> <colspec colname="6" colwidth="21pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="6" rowsep="1" class="description-td" colspan="6">TABLE 16c</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="6" align="center" rowsep="1" class="description-td" colspan="6"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">PBMC</td>
                <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">HSB2</td>
                <td class="description-td"> </td>
                <td class="description-td">IC50,</td>
              </tr> <tr class="description-tr"> <td class="description-td">Fusion</td>
                <td class="description-td"> </td>
                <td class="description-td">IC50,</td>
                <td class="description-td"> </td>
                <td class="description-td">pM</td>
              </tr> <tr class="description-tr"> <td class="description-td">Plasmid</td>
                <td class="description-td">Immunotoxin</td>
                <td class="description-td">pM Toxin</td>
                <td class="description-td">n =</td>
                <td class="description-td">Toxin</td>
                <td class="description-td">n =</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="6" align="center" rowsep="1" class="description-td" colspan="6"> </td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="6"> <colspec colname="1" colwidth="35pt" align="left"> </colspec> <colspec colname="2" colwidth="84pt" align="left"> </colspec> <colspec colname="3" colwidth="35pt" align="char" char="."> </colspec> <colspec colname="4" colwidth="14pt" align="center"> </colspec> <colspec colname="5" colwidth="28pt" align="char" char="."> </colspec> <colspec colname="6" colwidth="21pt" align="char" char="."> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">pING4644</td>
                <td class="description-td">V<sub>L</sub>V<sub>H</sub>::RMA::Gel</td>
                <td class="description-td">1933</td>
                <td class="description-td">4</td>
                <td class="description-td">1513</td>
                <td class="description-td">29</td>
              </tr> <tr class="description-tr"> <td class="description-td">pING3784</td>
                <td class="description-td">Gel::kappa, Fd</td>
                <td class="description-td">&gt;12,500</td>
                <td class="description-td">3</td>
                <td class="description-td">2645</td>
                <td class="description-td">7</td>
              </tr> <tr class="description-tr"> <td class="description-td">pING3789</td>
                <td class="description-td">Gel::Fd, kappa</td>
                <td class="description-td">1212</td>
                <td class="description-td">1</td>
                <td class="description-td">3665</td>
                <td class="description-td">1</td>
              </tr> <tr class="description-tr"> <td class="description-td">pING3348</td>
                <td class="description-td">V<sub>L</sub>V<sub>H</sub>::Gel</td>
                <td class="description-td">2158</td>
                <td class="description-td">4</td>
                <td class="description-td">1264</td>
                <td class="description-td">9</td>
              </tr> <tr class="description-tr"> <td class="description-td">pING3350</td>
                <td class="description-td">Gel::V<sub>H</sub>V<sub>L</sub> </td>
                <td class="description-td">8056</td>
                <td class="description-td">3</td>
                <td class="description-td">2729</td>
                <td class="description-td">4</td>
              </tr> <tr class="description-tr"> <td class="description-td">pING3775</td>
                <td class="description-td">(Gel::RMA::kappa, Fd′)<sub>2</sub> </td>
                <td class="description-td">175</td>
                <td class="description-td">1</td>
                <td class="description-td">44</td>
                <td class="description-td">22</td>
              </tr> <tr class="description-tr"> <td class="description-td">pING3770</td>
                <td class="description-td">Gel::RMA::k,</td>
                <td class="description-td">3548</td>
                <td class="description-td">2</td>
                <td class="description-td">519</td>
                <td class="description-td">9</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Gel::RMA::Fd</td>
              </tr> <tr class="description-tr"> <td class="description-td">pING3772</td>
                <td class="description-td">Gel::SLT::k, Gel::SLT::Fd</td>
                <td class="description-td">—</td>
                <td class="description-td">—</td>
                <td class="description-td">663</td>
                <td class="description-td">6</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="6" align="center" rowsep="1" class="description-td" colspan="6"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p>The results presented in Tables 16a, 16b and 16c demonstrate that gelonin immunofusions may vary in their activity. In general, immunofusions of the invention which have IC<sub>50 </sub>median or mean values of less than 2000 pM Toxin display strong activity; whereas those with IC<sub>50 </sub>values equal to or less than 500 pM Toxin are considered highly active. In sum, the results in Tables 16a, 16b and 16c demonstrate that the optimum fusion protein for killing a particular cell line may vary depending upon the targeted call.</p>
    <heading>EXAMPLE 22</heading> <p>Preparation of BRIP</p>
    <p>BRIP possesses characteristics which make it an attractive candidate for a component of immunotoxins. BRIP is a naturally unglycosylated protein that may have reduced uptake in the liver and enhanced circulatory residence time in vivo. Additionally, BRIP is less toxic and less immunogenic in animals than the A-chain of ricin. Cloning of the BRIP gene and expression of recombinant BRIP in an <i>E. coli </i>expression system obviates the need to purify native BRIP directly from barley, and enables the development of analogs of BRIP which may be conjugated with an available cysteine residue for conjugation to antibodies.</p>
    <p>A. Purification of BRIP and Generation of Polyclonal Antibodies to BRIP</p>
    <p>Native BRIP was purified from pearled barley flour. Four kilograms of flour was extracted with 16 liters of extraction buffer (10 mM NaPO4, 25 mM NaCl, pH 7.2) for 20 hours at 4° C. The sediment was removed by centrifugation, and 200 ml of packed S-Sepharose (Pharmacia, Piscataway, N.J.) was added to absorb BRIP. After mixing for 20 hours at 4° C., the resin was allowed to settle out, rinsed several times with extraction buffer and then packed into a 2.6×40 cm column. Once packed, the column was washed with extraction buffer (150 ml/h) until the absorbance of the effluent approached zero. BRIP was then eluted with a linear gradient of 0.025 to 0.3 M NaCl in extraction buffer and 5 ml fractions were collected. BRIP-containing peaks (identified by Western analysis of column fractions) were pooled, concentrated to about 20 ml, and then chromatographed on a 2.6×100 cm Sephacryl S-200HR (Pharmacia) column equilibrated in 10 mM NaPO<sub>4</sub>, 125 mM NaCl, pH 7.4 (10 ml/hr). BRIP-containing peaks were pooled again, concentrated, and stored at −70° C.</p>
    <p>The resulting purified BRIP protein had a molecular weight of about 30,000 Daltons, based upon the mobility of Coomassie-stained protein bands following SDS-PAGE. The amino acid composition was consistent with that published by Asano et al., <i>Carlsberg Res. Comm., </i>49:619-626 (1984).</p>
    <p>Rabbits were immunized with purified BRIP to generate polyclonal antisera.</p>
    <p>B. Cloning of the BRIP Gene</p>
    <p>A cDNA expression library prepared from germinating barley seeds in the phage λ expression vector λZAPII was purchased from Stratagene, La Jolla, Calif. Approximately 700,000 phage plaques were screened with anti-BRIP polyclonal antisera and 6 immunoreactive plaques were identified. One plaque was chosen, and the CDNA contained therein was excised from λZAPII with EcoRI and subcloned into pUC18 generating the vector pBS1. The cDNA insert was sequenced with Sequenase (United States Biochemical, Cleveland, Ohio). The DNA sequence of the native BRIP gene is set out in SEQ ID NO: 12. To confirm that cDNA encoded the native BRIP gene, the cDNA was expressed in the <i>E. coli </i>plasmid pKX233-2 (Pharmacia). BRIP protein was detected in IPTG-induced cells transformed with the plasmid by Western analysis with above-described rabbit anti-BRIP antisera.</p>
    <p>C. Construction of an <i>E. coli </i>Expression Vector Containing the BRIP Gene</p>
    <p>Barley cDNA containing the BRIP gene was linked to a pelB leader sequence and placed under control of an araB promoter in a bacterial secretion vector.</p>
    <p>An intermediate vector containing the BRIP gene linked to the pelB leader sequence was generated. Plasmid pBS1 was cut with NcoI, treated with Mung Bean Nuclease, cut with BamHI and the 760 bp fragment corresponding to amino acids 1-256 of BRIP was purified from an agarose gel. Concurrently, a unique XhoI site was introduced downstream of the 3′-end of the BRIP gene in pBS1 by PCR amplification with a pUC18 vector primer (identical to the Reverse® primer sold by NEB or BRL but synthesized on a Cyclone Model 8400 DNA synthesizer) and the specific primer BRIP 3′Xho. The sequence of each of the primers is set out below.</p>
    <p>Reverse (SEQ ID NO: 45) 5′ AACAGCTATGACCATG 3′</p>
    <p>BRIP 3′Xho (SEQ ID NO: 46) 5′ TGAACTCGAGGAAAACTACCTATTTCCCAC 3′</p>
    <p>Primer BRIP 3′Xho includes a portion corresponding to the last 8 bp of the BRIP gene, the termination codon and several base pairs downstream of the BRIP gene, and an additional portion that introduces a XhoI site in the resulting PCR fragment. The PCR reaction product was digested with BamHI and XhoI, and an 87 bp fragment containing the 3′-end of the BRIP gene was purified on a 5% acrylamide gel. The 760 and 87 bp purified BRIP fragments were ligated in the vector pING1500 adjacent to the pelB leader sequence. pING1500 had previously been cut with SstI, treated with T4 polymerase, cut with XhoI, and purified. The DNA sequence at the junction of the pelB leader and the 5′-end of the BRIP gene was verified by DNA sequence analysis. This vector was denoted pING3321-1.</p>
    <p>The final expression vector was assembled by placing the BRIP gene under the control of the inducible araB promoter. Plasmid pING3321-1 was cut with PstI and XhoI, and the BRIP gene linked to the pelB leader was purified from an agarose gel. The expression vector pING3217, containing the araB promoter, was cut with PstI and XhoI and ligated to the BRIP gene. The expression vector was denoted pING3322.</p>
    <p>Arabinose induction of <i>E. coli </i>cells containing the plasmid pING3322 in a fermenter resulted in the production of about 100 mg per liter of recombinant BRIP. <i>E. coli</i>-produced BRIP displays properties identical to BRIP purified directly from barley seeds.</p>
    <p>D. Construction of BRIP Analogs with a Free Cysteine Residue</p>
    <p>The BRIP protein contains no cysteine residues, and therefore contains no residues directly available which may form a disulfide linkage to antibodies or other proteins. Analogs of recombinant BRIP were generated which contain a free cysteine residue near the C-terminus of the protein. Three residues of the BRIP protein were targets for amino acid substitutions. Comparison of the amino acid sequence of BRIP to the known tertiary structure of the ricin A-chain (see FIG. 2) suggested that the three positions would be available near the surface of the molecule. The three BRIP analogs include cysteines substituted in place of serine<sub>277</sub>, alanine<sub>270</sub>, and leucine<sub>256 </sub>of the native protein, and were designated BRIP<sub>C256</sub>, (SEQ ID NO: 127), BRIP<sub>C270 </sub>(SEQ ID NO: 128) and BRIP<sub>C256 </sub>(SEQ ID NO: 129), respectively.</p>
    <p>(1) A plasmid vector capable of expressing the BRIP<sub>C277 </sub>analog was constructed by replacing the 3′-end of the BRIP gene with a DNA segment conferring the amino acid change. The EcoRI fragment containing the BRIP gene from pBS1 was subcloned into M13mp18, and single-stranded DNA (anti-sense strand) was amplified by PCR with primers OBM2 (corresponding nucleotides −11 to +8 of the BRIP gene) and OMB4 (corresponding to amino acids 264-280 of BRIP and the termination codon of BRIP, and incorporating the substitution of a cysteine codon for the native codon for serine<sub>277 </sub>of native BRIP). The sequences of primers OBM2 and OMB4, wherein the underlined nucleotides encode the substituted cysteine, are set out below.</p>
    <p>OBM2 (SEQ ID NO: 47) 5′ GCATTACATCCATGGCGGC 3′</p>
    <p>OMB4 (SEQ ID NO: 48) 5′ GATATCTCGAGTTAACTATTTCCCACC<u>ACA</u>CG CATGGAACAGCTCCAGCGCCTTGGCCACCGTC 3′</p>
    <p>A fragment containing a BRIP gene in which the codon for the amino acid at position 277 was changed to a cysteine codon was amplified. The fragment was cloned into the SmaI site of pUC19 (BRL) and the plasmid generated was denoted pMB22. pMB22 was digested with EcoRI and an EcoRI-XhoI linker (Clonetech, Palo Alto, Calif.) was ligated into the vector. Subsequent digestion with XhoI and religation generated vector pINGMB2X. A BamHI to XhoI fragment encoding the 3′-end of BRIP with the altered amino acid was excised from pMB2X and the fragment was purified on a 5% acrylamide gel. This fragment along with an EcoRI to BamHI fragment containing the pelB leader sequence and sequences encoding the first 256 amino acids of BRIP were substituted in a three piece ligation into pING3322 cut with ZcoRI and XhoI. The resulting vector containing the BRIP<sub>C277 </sub>analog was designated pING3803 (ATCC Accession No. 68722).</p>
    <p>(2) A BRIP analog with a free cysteine at position 256 was constructed using PCR to introduce the amino acid substitution. A portion of the expression plasmid pING3322 was amplified with primers BRIP-256 and HINDIII-2. The sequence of each primer is set out below.</p>
    <p>BRIP-256 (SEQ ID NO: 49) 5′ <u>TGT</u>CTGTTCGTGGAGGTGCCG 3′</p>
    <p>HINDIII-2 (SEQ ID NO: 44) 5′ CGTTAGCAATTTAACTGTGAT 3′</p>
    <p>Nucleotides 4-21 of primer BRIP-256 encode amino acids 256-262 of BRIP while the underlined nucleotides specify the cysteine to be substituted for the leucine at the corresponding position of the native BRIP protein. Primer HINDIII-2 corresponds to a portion of the plasmid. The PCR product, which encodes the carboxyl terminal portion of the BRIP analog, was treated with T4 polymerase, cut with XhoI, and the resulting fragment was purified on a 5% acrylamide gel. Concurrently, plasmid pING3322 was cut with BamHI, treated with T4 polymerase, cut with EcoRI, and the fragment containing the pelB leader sequence and sequences encoding the first 256 amino acids of BRIP was purified. The two fragments were then assembled back into pING3322 to generate the gene encoding the analog BRIP<sub>C256</sub>. This plasmid is denoted pING3801.</p>
    <p>(3) A BRIP analog with a cysteine at position 270 was also generated using PCR. A portion of the expression plasmid pING3322 was amplified with primers BRIP-270 and the HINDIII-2 primer (SEQ ID NO: 44). The sequence of primer BRIP-270 is set out below.</p>
    <p>BRIP-270 (SEQ ID NO: 50) 5′ CCAAGTGTCTGGAGCTGTTCCATGCGA 3′</p>
    <p>Primer BRIP-270 corresponds to amino acids 268-276 of BRIP with the exception of residue 270. The codon of the primer corresponding to position 270 specifies a cysteine instead of the alanine present in the corresponding position in native BRIP. The PCR product was treated with T4 polymerase, cut with XhoI, and the 51 bp fragment, which encodes the carboxyl terminal portion of the analog, was purified on a 5% acrylamide gel. The fragment (corresponding to amino acids 268-276 of BRIP<sub>C270</sub>) was cloned in a three piece ligation along with the internal 151 bp BRIP restriction fragment from SstII to MscI (corresponding to BRIP amino acids 217-267) from plasmid pING3322, and restriction fragment from SstII to XhoI from pING3322 containing the remainder of the BRIP gene. The plasmid generated contains the gene encoding the BRIPC270 analog and is designated pING3802.</p>
    <p>E. Purification of Recombinant BRIP and the BRIP Analogs</p>
    <p>Recombinant BRIP (rBRIP) and the BRIP analogs with free cysteine residues were purified essentially as described for native BRIP except they were prepared from concentrated fermentation broths. For rBRIP, concentrated broth from a 10 liter fermentation batch was exchanged into 10 mM Tris, 20 mM NaCl pH 7.5, loaded onto a Sephacryl S-200 column, and eluted with a 20 to 500 mM NaCl linear gradient. Pooled rBRIP was further purified on a Blue Toyopearl® column (TosoHaas) loaded in 20 mM NaCl and eluted in a 20 to 500 mM NaCl gradient in 10 mM Tris, pH 7.5. For BRIP analogs, concentrated fermentation broths were loaded onto a CM52 column (Whatman) in 10 mM phosphate buffer, pH 7.5, and eluted with a 0 to 0.3M NaCl linear gradient. Further purification was by chromatography on a Blue Toyopearl® column.</p>
    <p>F. Reticulocyte Lysate Assay</p>
    <p>The ability of the rBRIP and the BRIP analogs to inhibit protein synthesis in vitro was tested by reticulocyte lysate assay as described in Example 1. Serial log dilutions of standard toxin (RTA 30), native BRIP, rBRIP and BRIP analogs were tested over a range of 1 μg/ml to 1 pg/ml. By comparison with an uninhibited sample, the picomolar concentration of toxin (pM) which corresponds to 50% inhibition of protein synthesis (IC<sub>50</sub>) was calculated. The results of the assays are presented below in Table 17.</p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="offset" colwidth="49pt" align="left"> </colspec> <colspec colname="1" colwidth="42pt" align="left"> </colspec> <colspec colname="2" colwidth="126pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="OFFSET" nameend="2" rowsep="1" class="description-td" colspan="3">TABLE 17</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="OFFSET" nameend="2" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Toxin</td>
                <td class="description-td">IC<sub>50 </sub>(pM)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="OFFSET" nameend="2" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="offset" colwidth="49pt" align="left"> </colspec> <colspec colname="1" colwidth="42pt" align="left"> </colspec> <colspec colname="2" colwidth="126pt" align="char" char="."> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">RTA 30</td>
                <td class="description-td">3.1</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Native BRIP</td>
                <td class="description-td">15</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">rBRIP</td>
                <td class="description-td">18</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">BRIP<sub>C256</sub> </td>
                <td class="description-td">23</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">BRIP<sub>C270</sub> </td>
                <td class="description-td">20</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">BRIP<sub>C277</sub> </td>
                <td class="description-td">24</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="OFFSET" nameend="2" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p>The RLA results indicate that the BRIP analogs exhibit ribosome-inactivating activity comparable to that of the recombinant and native BRIP toxin. All the analogs retained the natural ability of native BRIP to inhibit protein synthesis, suggesting that amino acid substitution at these positions does not affect protein folding and activity.</p>
    <heading>EXAMPLE 23</heading> <p>Construction of BRIP Immunoconjugates</p>
    <p>Immunoconjugates of native BRIP (SEQ ID NO: 3) with 4A2 (described in Morishima at al., <i>J. Immunol., </i>129:1091 (1982) and H65 antibody (obtained from hybridoma ATCC HB9286) which recognize the T-cell determinants CD7 and CD5, respectively, were constructed. Immunoconjugates of ricin A-chains (RTAs) with 4A2 and H65 antibody were constructed as controls. The H65 antibody and ricin A-chains as well as the RTA immunoconjugates were prepared and purified according to methods described in U.S. patant application Ser. No. 07/306,433 supra and in International Publication No. WO 89/06968.</p>
    <p>To prepare immunoconjugates of native BRIP, both the antibody (4A2 or H65) and native BRIP were chemically modified with the hindered linker 5-methyl-2-iminothiolane (M2IT) at lysine residues to introduce a reactive sulfhydryl group as described in Goff et al., <i>Bioconjugate Chem., </i>1:381-386 (1990). BRIP (3 mg/ml) was first incubated with 0.5 mM M2IT and 1 mM DTNB in 25 mM triethanolamine, 150 mM NaCl, pH 8.0, for 3 hours at 25° C. The derivitized BRIP-(M2IT)-S-S-TNB was then desalted on a column of Sephadex GF-05LS and the number of thiol groups introduced was quantitated by the addition of 0.1 mM DTT. On average, each BRIP molecule contained 0.7 SH/mol.</p>
    <p>4A2 or H65 antibody (4 mg/ml) in triethanolamine buffer was similarly incubated with M2IT (0.3 mM) and DTNB (1 mM) for 3 hours at 25° C. Antibody-(M2IT)-S-S-TNB was then desalted and the TNB:antibody ratio was determined. To prepare the conjugate, the BRIP-(M2IT)-S-S-TNB was first reduced to BRIP—(M2IT)-SH by treatment with 0.5 mM DTT for 1 hour at 25° C., desalted by gel filtration of Sephadex® GF-05LS to remove the reducing agent, and then mixed with antibody-(M2IT)-S-S-TNB.</p>
    <p>Following a 3 hour incubation at 25° C., and an additional 18 hours at 4° C., the conjugate was purified by sequential chromatography on AcA44 (IBF) and Blue Toyopearl®. Samples of the final product were run on 5% non-reducing SDS PAGE, Coomassie stained, and scanned with a Shimadzu laser densitometer to quantitate the number of toxins per antibody.</p>
    <p>The BRIP analogs containing a free cysteine were also conjugated to 4A2 and H65 antibodies. The analogs were treated with 50 mM DTT either for 2 hours at 25° C. or for 18 hours at 4° C. to expose the reactive sulfhydryl group of the cysteine and desalted. The presence of a free sulfhydryl was verified by reaction with DTNB [Ellman et al., <i>Arch. Biochem. Biophys, </i>82:70-77 (1959)]. 4A2 or H65 antibody derivatized as described above with M2IT was incubated with the reduced BRIP analogs at a ratio of 1:5 at room temperature for 3 hours and then overnight at 4° C. Immunoconjugates H65-BRIP<sub>C256</sub>, 4A2-BRIP<sub>C256</sub>, H65-BRIP<sub>C277 </sub>were prepared in 25 mM triethanolamine, 150 mM NaCl pH 8, while immunoconjugates H65-BRIP<sub>C270</sub>, 4A2-BRIP<sub>C270 </sub>and 4A2-BRIP<sub>C277 </sub>were prepared in 0.1 M sodium phosphate, 150 mM NaCl pH 7.5. Following conjugation, 10 μM mercaptoethylamine was added for 15 minutes at 25° C. to quenched any unreacted m2IT linkers on the antibody. The quenched reaction solution was promptly loaded onto a gel filtration column (AcA44) to remove unconjugated ribosome-inactivating protein. Purification was completed using soft gel affinity chromatography on Blue Toyopearl® resin using a method similar to Knowles et al., <i>Analyt. Biochem., </i>160:440 (1987). Samples of the final product were run on 5% non-reduced SDS PAGE, Coomassie stained, and scanned with a Shimadzu laser densitometer to quantitate the number of toxins per antibody. The conjugation efficiency was substantially greater for BRIP<sub>C277 </sub>(78%) than for either of the other two analogs, BRIP<sub>C270 </sub>and BRIP<sub>C256 </sub>(each of these was about 10%). Additionally, the BRIP<sub>C277 </sub>product was a polyconjugate, i.e., several BRIP molecules conjugated to a single antibody, in contrast to the BRIP<sub>C270 </sub>and BRIP<sub>C256 </sub>products which were monoconjugates.</p>
    <heading>EXAMPLE 24</heading> <p>Properties of BRIP Immunoconjugates</p>
    <p>A. Whole Cell Kill Assay</p>
    <p>Immunoconjugates of native BRIP and of the BRIP analogs were tested for the ability to inhibit protein synthesis in HSB2 cells by the whole cell kill assay described in Example 1. Standard immunoconjugates H65-RTA (H65 derivatized with SPDP linked to RTA) and 4MRTA (4A2 antibody derivatized with M2IT linked to RTA) and BRIP immunoconjugate samples were diluted with RPMI without leucine at half-log concentrations ranging from 2000 to 0.632 ng/ml. All dilutions were added in triplicate to microtiter plates containing 1×10<sup>5 </sup>HSB2 cells. HSB2 plates were incubated for 20 hours at 37° C. and then pulsed with <sup>3</sup>H-Leu for 4 hours before harvesting. Samples were counted on the Inotec Trace 96 cascade ionization counter. By comparison with an untreated sample, the picomolar toxin concentration (pM T) of immunoconjugate which resulted in a 50% inhibition of protein synthesis (IC<sub>50</sub>) was calculated. The assay results are presented below in Table 18.</p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="offset" colwidth="42pt" align="left"> </colspec> <colspec colname="1" colwidth="49pt" align="left"> </colspec> <colspec colname="2" colwidth="126pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="OFFSET" nameend="2" rowsep="1" class="description-td" colspan="3">TABLE 18</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="OFFSET" nameend="2" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Conjugate</td>
                <td class="description-td">IC<sub>50 </sub>(pM T)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="OFFSET" nameend="2" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="offset" colwidth="42pt" align="left"> </colspec> <colspec colname="1" colwidth="49pt" align="left"> </colspec> <colspec colname="2" colwidth="126pt" align="char" char="."> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">4A2-BRIP</td>
                <td class="description-td">122.45</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">4A2-BRIP<sub>C270</sub> </td>
                <td class="description-td">46.3</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">4A2-BRIP<sub>C277</sub> </td>
                <td class="description-td">57.5</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">4A2-BRIP<sub>C256</sub> </td>
                <td class="description-td">1116</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">H65-BRIP</td>
                <td class="description-td">&gt;5000</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">H65-BRIP<sub>C277</sub> </td>
                <td class="description-td">1176</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="OFFSET" nameend="2" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p>The BRIP analog conjugates were less potent than the ricin conjugate control (data not shown). The immunotoxins containing antibody 4A2 and either the BRIP<sub>C270 </sub>or the BRIP<sub>C277 </sub>analog exhibited comparable to increased specific cytotoxicity toward target cells as compared to immunotoxin containing native BRIP. While 4A2-BRIP<sub>C256 </sub>is less active than 4A2-BRIP, 4A2-BRIP<sub>C270 </sub>and 4A2-BRIP<sub>C277 </sub>are between 3 and 4 times more active. Similarly, the immunoconjugate of H65 to BRIP<sub>C277 </sub>shows greater toxicity toward target cells than the immunoconjugate of H65 to native BRIP. Thus, linkage of antibody to BRIP derivatives which have an available cysteine residue in an appropriate location results in immunotoxins with enhanced specific toxicity toward target cells relative to conjugates with native BRIP.</p>
    <p>B. Disulfide Bond Stability Assay</p>
    <p>Immunoconjugates prepared with native BRIP and the BRIP analogs were examined by the disulfide bond stability assay described in Example 1. Briefly, conjugates were incubated with increasing concentrations of glutathione for 1 hour at 37° C. and, after terminating the reaction with iodoacetamide, the amount of RIP released was quantitated by size-exclusion HPLC on a TosoHaas TSK-G2000SW column.</p>
    <p>By comparisons with the amount of RIP released by high concentrations of 2-mercaptoethanol (to determine 100% release), the concentration of glutathione required to release 50% of the RIP (the RC<sub>50</sub>) was calculated. As shown below in Table 19, the conjugates prepared with BRIP<sub>C270 </sub>or BRIP<sub>C277 </sub>were significantly more stable than either the RTA conjugates or those prepared with native BRIP.</p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="offset" colwidth="42pt" align="left"> </colspec> <colspec colname="1" colwidth="56pt" align="left"> </colspec> <colspec colname="2" colwidth="119pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="OFFSET" nameend="2" rowsep="1" class="description-td" colspan="3">TABLE 19</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="OFFSET" nameend="2" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Conjugate</td>
                <td class="description-td">RC<sub>50 </sub>(mM)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="OFFSET" nameend="2" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="offset" colwidth="42pt" align="left"> </colspec> <colspec colname="1" colwidth="56pt" align="left"> </colspec> <colspec colname="2" colwidth="119pt" align="char" char="."> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">H65-RTA</td>
                <td class="description-td">7.0</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">H65-BRIP</td>
                <td class="description-td">2.8</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">H65-BRIPC277</td>
                <td class="description-td">196.0</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">4A2-RTA</td>
                <td class="description-td">4.4</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">4A2-BRIP</td>
                <td class="description-td">3.3</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">42-BRIP<sub>C270</sub> </td>
                <td class="description-td">53.0</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">4A2-BRIP<sub>C277</sub> </td>
                <td class="description-td">187.0</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="OFFSET" nameend="2" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p>These unexpected results suggest that conjugates prepared with Type I RIP analogs according to the present invention may have enhanced stability and efficacy in vivo.</p>
    <heading>EXAMPLE 25</heading> <p>Preparation of Momordin and Analogs Thereof</p>
    <p>Plants of the genus Momordica produce a number of related proteins known as momordins or momorcharins which are Type I RIPs. The gene encoding momordin II was cloned from <i>Momordica balsamina </i>seeds.</p>
    <p>A. Preparation of <i>M. balsamina </i>RNA</p>
    <p>Total RNA was prepared from 4 g of <i>M. balsamina </i>seeds as described in Ausubel et al., supra. PolyA containing RNA was prepared from 1 mg of total RNA by chromatography on oligo-(dT)-cellulose. 40 mg of oligo-(dT)-cellulose Type 7 (Pharmacia) was added to 0.1 N NaOH and poured into a disposable column (Biorad). The column was washed with water until the eluate was pH 5.5, and then was washed with 1×loading buffer (50 mM NaCitrate, 0.5 M NaCl, 1 mM EDTA, 0.1% SDS, pH 7.0) until the eluate was pH 7.0. 1 mg of total RNA was suspended in 300 μl of water, heated to 65° C. for 5 minutes, and 300 μl of 2×loading buffer was added (100 mM Na Citrate, 1 M NaCl, 2 mM EDTA, and 0. 2% SDS) The RNA was loaded onto the column, and the flow through was reheated to 65° C., cooled to room temperature, and reloaded onto the column. Column-bound mRNA was washed 5 times with 0.5 ml of 1×loading buffer, and two times with 0.5 ml of 0.05M NaCitrate, 0.1 M NaCl, 1 mM EDTA, 0.1% SDS. PolyA-containing RNA was eluted two times from the column with 0.5 ml of 25 mM NaCitrate, 1 mM EDTA, and 0.05% SDS.</p>
    <p>B. Library Preparation</p>
    <p>A cDNA library from the polyA-containing <i>M. balsamina </i>RNA was prepared in a bacterial expression plasmid with the SuperScript Plasmid System (BRL, Gaithersburg, Md.). The cDNA was synthesized from 2 μg of poly A-containing RNA, size fractionated, digested with NotI, and ligated into the expression vector pSPORT as recommended by the manufacturer of the vector, BRL.</p>
    <p>C. Cloning of the Momordin II Gene</p>
    <p>A DNA fragment encoding the first 27 amino acids of momordin II was amplified from <i>M. balsamina </i>cDNA by PCR. First strand cDNA was prepared from 100 ng of polyA containing RNA with an RNA-PCR Kit (Perkin Elmer Cetus). Two partially degenerate primers were synthesized based on the amino acid sequence of the first 27 amino acids of momordin II described in Li et al., <i>Experientia, </i>36:524-527 (1980). Because the amino acid sequence of amino acids 1-27 of momordin II is 52% homologous to amino acids 1-17 of momordin I [Ho et al., <i>BBA, </i>1088:311-314 (1991)], some codon assignments in the degenerate primers were based on homology to the corresponding amino acid as well as codon preference in the momordin I gene. The sequences of primers momo-3 and momo-4 are set out below using IUPAC nucleotide symbols.</p>
    <p>momo-3 (SEQ ID NO: 51) 5′ GATGTTAAYTTYGAYTTGTCNACDGCTAC 3′</p>
    <p>momo-4 (SEQ ID NO: 52) 5′ ATTGGNAGDGTAGCCCTRAARTCYTCDAT 3′</p>
    <p>The resulting 81 bp PCR product was purified on a 5% acrylamide gel and cloned into the Smal site of pUC18. Three candidate clones were sequenced, and one clone, pMO110, was identified which encoded the N-terminal 27 amino acids of momordin II.</p>
    <p>A hybridization probe was designed for screening of the momordin II cDNA library based on the sequence of the pMO100 momordin II DNA fragment. The sequence of the primer momo-5 is shown below.</p>
    <p>momo-5 (SEQ ID NO: 53) 5′ GCCAC<u>TGCAAAAACCTACACAAAATTT</u>ATTGA 3′</p>
    <p>Primer momo-5 corresponds to amino acids 9-18 of mature momordin II. The underlined nucleotides of the primer were expected to match the DNA sequence of the momordin II gene exactly. Since this sequence is highly A/T-rich and may hybridize to the momordin II gene weakly, the additional adjacent nucleotides were included in the primer. Bases 3 and 30 (overlined) were in the “wobble” position (i.e., the third nucleotide in a codon) of amino acids 9 (alanine) and 18 (isoleucine), respectively, of momordin II and may not be identical to the nucleotide bases in the native gene.</p>
    <p>A 90,000 member cDNA library in pSPORT was screened with <sup>32</sup>P-kinased momo-5, and eight potential candidate clones were identified. One clone, pING3619, was sequenced and contains an open reading frame corresponding in part to the expected N-terminal 27 residues of Momordin II. The complete momordin gene contains 286 amino acids, the first 23 of which are a presumed leader signal (mature momordin II is 263 residues). The DNA sequence of the momordin II gene is set out in SEQ ID NO: 13.</p>
    <p>D. Construction of an Expression Vector Containing the Momordin II Gene</p>
    <p>A bacterial expression vector for the momordin II gene was constructed. Two PCR primers were synthesized, one (momo-9) which primes from the +1 residue of the mature momordin II amino acid sequence, and one at the C-terminus (momo-10) of momordin II which introduces an XhoI restriction site:</p>
    <p>momo-9 (SEQ ID NO: 54) 5′ GATGTTAACTTCGATTTGTCGA 3′</p>
    <p>momo-10 (SEQ ID NO: 55) 5′ TCAACTCGAGGTACTCAATTCACAACAGATTCC 3′</p>
    <p>pING3619 was amplified with momo-9 and momo-10, and the product was treated with T4 polymerase, cut with XhoI, and purified on an agarose gel. This gene fragment was ligated along with the 131 bp pelB leader fragment from pIC100 which has been generated by SstI digestion, T4-polymerase treatment, and EcoRI digestion, into the araB expression vector cleaved with EcoRI and XhoI. The product of this three piece ligation was sequenced to verify that the pelB junction and momordin II coding sequence were correct. Arabinose induction of calls containing the momordin II expression plasmid pING3621 results in production of momordin II in <i>E. coli. </i> </p>
    <p>E. Analogs of Mormordin II</p>
    <p>Mormordin II has no natural cysteines available for conjugation to antibody. Analogs of momordin which have a free cysteine for conjugation to an antibody may be constructed. Positions likely to be appropriate for substitution of a cysteine residue may be identified from FIG. 3 as positions near the ricin A-chain cysteine<sub>259 </sub>and as positions including the last 26 amino acids of momordin II that are accessible to solvent. For example, the arginine at position 242 of momordin II aligns with the ricin A-chain cysteine at position 259 and is a preferred target for substitution. Additional preferred substitution positions for momordin II include the serine at position 241 and the alanine at position 243.</p>
    <p>While the present invention has been described in terms of preferred embodiments, it is understood that variations and improvements will occur to those skilled in the art. Therefore, it is intended that the appended claims cover all such equivalent variations which come within the scope of the invention as claimed.</p>
    <p>
      <sequence-cwu id="SEQLST-1"> <sequence-list-old-rules> <s-1> <s-1-iii>173</s-1-iii> </s-1> <s-2 seq-no="1"> <s-2-i> <s-2-i-a>267 amino acids</s-2-i-a> <s-2-i-b>amino acid</s-2-i-b> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>protein</s-2-ii-mt> </s-2-ii> <s-2-xi>1
Ile Phe Pro Lys Gln Tyr Pro Ile Ile Asn Phe Thr Thr Ala Gly Ala
1               5                   10                  15
Thr Val Gln Ser Tyr Thr Asn Phe Ile Arg Ala Val Arg Gly Arg Leu
            20                  25                  30
Thr Thr Gly Ala Asp Val Arg His Glu Ile Pro Val Leu Pro Asn Arg
        35                  40                  45
Val Gly Leu Pro Ile Asn Gln Arg Phe Ile Leu Val Glu Leu Ser Asn
    50                  55                  60
His Ala Glu Leu Ser Val Thr Leu Ala Leu Asp Val Thr Asn Ala Tyr
65                  70                  75                  80
Val Val Gly Tyr Arg Ala Gly Asn Ser Ala Tyr Phe Phe His Pro Asp
                85                  90                  95
Asn Gln Glu Asp Ala Glu Ala Ile Thr His Leu Phe Thr Asp Val Gln
            100                 105                 110
Asn Arg Tyr Thr Phe Ala Phe Gly Gly Asn Tyr Asp Arg Leu Glu Gln
        115                 120                 125
Leu Ala Gly Asn Leu Arg Glu Asn Ile Glu Leu Gly Asn Gly Pro Leu
    130                 135                 140
Glu Glu Ala Ile Ser Ala Leu Tyr Tyr Tyr Ser Thr Gly Gly Thr Gln
145                 150                 155                 160
Leu Pro Thr Leu Ala Arg Ser Phe Ile Ile Cys Ile Gln Met Ile Ser
                165                 170                 175
Glu Ala Ala Arg Phe Gln Tyr Ile Glu Gly Glu Met Arg Thr Arg Ile
            180                 185                 190
Arg Tyr Asn Arg Arg Ser Ala Pro Asp Pro Ser Val Ile Thr Leu Glu
        195                 200                 205
Asn Ser Trp Gly Arg Leu Ser Thr Ala Ile Gln Glu Ser Asn Gln Gly
    210                 215                 220
Ala Phe Ala Ser Pro Ile Gln Leu Gln Arg Arg Asn Gly Ser Lys Phe
225                 230                 235                 240
Ser Val Tyr Asp Val Ser Ile Leu Ile Pro Ile Ile Ala Leu Met Val
                245                 250                 255
Tyr Arg Cys Ala Pro Pro Pro Ser Ser Gln Phe
            260                 265
</s-2-xi> </s-2> <s-2 seq-no="2"> <s-2-i> <s-2-i-a>251 amino acids</s-2-i-a> <s-2-i-b>amino acid</s-2-i-b> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>protein</s-2-ii-mt> </s-2-ii> <s-2-xi>2
Gly Leu Asp Thr Val Ser Phe Ser Thr Lys Gly Ala Thr Tyr Ile Thr
1               5                   10                  15
Tyr Val Asn Phe Leu Asn Glu Leu Arg Val Lys Leu Lys Pro Glu Gly
            20                  25                  30
Asn Ser His Gly Ile Pro Leu Leu Arg Lys Lys Cys Asp Asp Pro Gly
        35                  40                  45
Lys Cys Phe Val Leu Val Ala Leu Ser Asn Asp Asn Gly Gln Leu Ala
    50                  55                  60
Glu Ile Ala Ile Asp Val Thr Ser Val Tyr Val Val Gly Tyr Gln Val
65                  70                  75                  80
Arg Asn Arg Ser Tyr Phe Phe Lys Asp Ala Pro Asp Ala Ala Tyr Glu
                85                  90                  95
Gly Leu Phe Lys Asn Thr Ile Lys Thr Arg Leu His Phe Gly Gly Ser
            100                 105                 110
Tyr Pro Ser Leu Glu Gly Glu Lys Ala Tyr Arg Glu Thr Thr Asp Leu
        115                 120                 125
Gly Ile Glu Pro Leu Arg Ile Gly Ile Lys Lys Leu Asp Glu Asn Ala
    130                 135                 140
Ile Asp Asn Tyr Lys Pro Thr Glu Ile Ala Ser Ser Leu Leu Val Val
145                 150                 155                 160
Ile Gln Met Val Ser Glu Ala Ala Arg Phe Thr Phe Ile Glu Asn Gln
                165                 170                 175
Ile Arg Asn Asn Phe Gln Gln Arg Ile Arg Pro Ala Asn Asn Thr Ile
            180                 185                 190
Ser Leu Glu Asn Lys Trp Gly Lys Leu Ser Phe Gln Ile Arg Thr Ser
        195                 200                 205
Gly Ala Asn Gly Met Phe Ser Glu Ala Val Glu Leu Glu Arg Ala Asn
    210                 215                 220
Gly Lys Lys Tyr Tyr Val Thr Ala Val Asp Gln Val Lys Pro Lys Ile
225                 230                 235                 240
Ala Leu Leu Lys Phe Val Asp Lys Asp Pro Lys
                245                 250
</s-2-xi> </s-2> <s-2 seq-no="3"> <s-2-i> <s-2-i-a>280 amino acids</s-2-i-a> <s-2-i-b>amino acid</s-2-i-b> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>protein</s-2-ii-mt> </s-2-ii> <s-2-xi>3
Ala Ala Lys Met Ala Lys Asn Val Asp Lys Pro Leu Phe Thr Ala Thr
1               5                   10                  15
Phe Asn Val Gln Ala Ser Ser Ala Asp Tyr Ala Thr Phe Ile Ala Gly
            20                  25                  30
Ile Arg Asn Lys Leu Arg Asn Pro Ala His Phe Ser His Asn Arg Pro
        35                  40                  45
Val Leu Pro Pro Val Glu Pro Asn Val Pro Pro Ser Arg Trp Phe His
    50                  55                  60
Val Val Leu Lys Ala Ser Pro Thr Ser Ala Gly Leu Thr Leu Ala Ile
65                  70                  75                  80
Arg Ala Asp Asn Ile Tyr Leu Glu Gly Phe Lys Ser Ser Asp Gly Thr
                85                  90                  95
Trp Trp Glu Leu Thr Pro Gly Leu Ile Pro Gly Ala Thr Tyr Val Gly
            100                 105                 110
Phe Gly Gly Thr Tyr Arg Asp Leu Leu Gly Asp Thr Asp Lys Leu Thr
        115                 120                 125
Asn Val Ala Leu Gly Arg Gln Gln Leu Ala Asp Ala Val Thr Ala Leu
    130                 135                 140
His Gly Arg Thr Lys Ala Asp Lys Ala Ser Gly Pro Lys Gln Gln Gln
145                 150                 155                 160
Ala Arg Glu Ala Val Thr Thr Leu Val Leu Met Val Asn Glu Ala Thr
                165                 170                 175
Arg Phe Gln Thr Val Ser Gly Phe Val Ala Gly Leu Leu His Pro Lys
            180                 185                 190
Ala Val Glu Lys Lys Ser Gly Lys Ile Gly Asn Glu Met Lys Ala Gln
        195                 200                 205
Val Asn Gly Trp Gln Asp Leu Ser Ala Ala Leu Leu Lys Thr Asp Val
    210                 215                 220
Lys Pro Pro Pro Gly Lys Ser Pro Ala Lys Phe Ala Pro Ile Glu Lys
225                 230                 235                 240
Met Gly Val Arg Thr Ala Glu Gln Ala Ala Asn Thr Leu Gly Ile Leu
                245                 250                 255
Leu Phe Val Glu Val Pro Gly Gly Leu Thr Val Ala Lys Ala Leu Glu
            260                 265                 270
Leu Phe His Ala Ser Gly Gly Lys
        275                 280
</s-2-xi> </s-2> <s-2 seq-no="4"> <s-2-i> <s-2-i-a>263 amino acids</s-2-i-a> <s-2-i-b>amino acid</s-2-i-b> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>protein</s-2-ii-mt> </s-2-ii> <s-2-xi>4
Asp Val Asn Phe Asp Leu Ser Thr Ala Thr Ala Lys Thr Tyr Thr Lys
1               5                   10                  15
Phe Ile Glu Asp Phe Arg Ala Thr Leu Pro Phe Ser His Lys Val Tyr
            20                  25                  30
Asp Ile Pro Leu Leu Tyr Ser Thr Ile Ser Asp Ser Arg Arg Phe Ile
        35                  40                  45
Leu Leu Asp Leu Thr Ser Tyr Ala Tyr Glu Thr Ile Ser Val Ala Ile
    50                  55                  60
Asp Val Thr Asn Val Tyr Val Val Ala Tyr Arg Thr Arg Asp Val Ser
65                  70                  75                  80
Tyr Phe Phe Lys Glu Ser Pro Pro Glu Ala Tyr Asn Ile Leu Phe Lys
                85                  90                  95
Gly Thr Arg Lys Ile Thr Leu Pro Tyr Thr Gly Asn Tyr Glu Asn Leu
            100                 105                 110
Gln Thr Ala Ala His Lys Ile Arg Glu Asn Ile Asp Leu Gly Leu Pro
        115                 120                 125
Ala Leu Ser Ser Ala Ile Thr Thr Leu Phe Tyr Tyr Asn Ala Gln Ser
    130                 135                 140
Ala Pro Ser Ala Leu Leu Val Leu Ile Gln Thr Thr Ala Glu Ala Ala
145                 150                 155                 160
Arg Phe Lys Tyr Ile Glu Arg His Val Ala Lys Tyr Val Ala Thr Asn
                165                 170                 175
Phe Lys Pro Asn Leu Ala Ile Ile Ser Leu Glu Asn Gln Trp Ser Ala
            180                 185                 190
Leu Ser Lys Gln Ile Phe Leu Ala Gln Asn Gln Gly Gly Lys Phe Arg
        195                 200                 205
Asn Pro Val Asp Leu Ile Lys Pro Thr Gly Glu Arg Phe Gln Val Thr
    210                 215                 220
Asn Val Asp Ser Asp Val Val Lys Gly Asn Ile Lys Leu Leu Leu Asn
225                 230                 235                 240
Ser Arg Ala Ser Thr Ala Asp Glu Asn Phe Ile Thr Thr Met Thr Leu
                245                 250                 255
Leu Gly Glu Ser Val Val Asn
            260
</s-2-xi> </s-2> <s-2 seq-no="5"> <s-2-i> <s-2-i-a>248 amino acids</s-2-i-a> <s-2-i-b>amino acid</s-2-i-b> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>protein</s-2-ii-mt> </s-2-ii> <s-2-xi>5
Asp Val Arg Phe Ser Leu Ser Gly Ser Ser Ser Thr Ser Tyr Ser Lys
1               5                   10                  15
Phe Ile Gly Asp Leu Arg Lys Ala Leu Pro Ser Asn Gly Thr Val Tyr
            20                  25                  30
Asn Leu Thr Ile Leu Leu Ser Ser Ala Ser Gly Ala Ser Arg Tyr Thr
        35                  40                  45
Leu Met Thr Leu Ser Asn Tyr Asp Gly Lys Ala Ile Thr Val Ala Val
    50                  55                  60
Asp Val Ser Gln Leu Tyr Ile Met Gly Tyr Leu Val Asn Ser Thr Ser
65                  70                  75                  80
Tyr Phe Phe Asn Glu Ser Asp Ala Lys Leu Ala Ser Gln Tyr Val Phe
                85                  90                  95
Lys Gly Ser Thr Ile Val Thr Leu Pro Tyr Ser Gly Asn Tyr Glu Lys
            100                 105                 110
Leu Gln Thr Ala Ala Gly Lys Ile Arg Glu Lys Ile Pro Leu Gly Phe
        115                 120                 125
Pro Ala Leu Asp Ser Ala Leu Thr Thr Ile Phe His Tyr Asp Ser Thr
    130                 135                 140
Ala Ala Ala Ala Ala Phe Leu Val Ile Leu Gln Thr Thr Ala Glu Ala
145                 150                 155                 160
Ser Arg Phe Lys Tyr Ile Glu Gly Gln Ile Ile Glu Arg Ile Ser Lys
                165                 170                 175
Asn Gln Val Pro Ser Leu Ala Thr Ile Ser Leu Glu Asn Ser Leu Trp
            180                 185                 190
Ser Ala Leu Ser Lys Gln Ile Gln Leu Ala Gln Thr Asn Asn Gly Thr
        195                 200                 205
Phe Lys Thr Pro Val Val Ile Thr Asp Asp Lys Gln Gln Arg Val Glu
    210                 215                 220
Ile Thr Asn Val Thr Ser Lys Val Val Thr Lys Asn Ile Gln Leu Leu
225                 230                 235                 240
Leu Asn Tyr Lys Gln Asn Val Ala
                245
</s-2-xi> </s-2> <s-2 seq-no="6"> <s-2-i> <s-2-i-a>247 amino acids</s-2-i-a> <s-2-i-b>amino acid</s-2-i-b> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>protein</s-2-ii-mt> </s-2-ii> <s-2-xi>6
Asp Val Ser Phe Arg Leu Ser Gly Ala Thr Ser Ser Ser Tyr Gly Val
1               5                   10                  15
Phe Ile Ser Asn Leu Arg Lys Ala Leu Pro Asn Glu Arg Lys Leu Tyr
            20                  25                  30
Asp Ile Pro Leu Leu Arg Ser Ser Leu Pro Gly Ser Gln Arg Tyr Ala
        35                  40                  45
Leu Ile His Leu Thr Asn Tyr Ala Asp Glu Thr Ile Ser Val Ala Ile
    50                  55                  60
Asp Val Thr Asn Val Tyr Ile Met Gly Tyr Arg Ala Gly Asp Thr Ser
65                  70                  75                  80
Tyr Phe Phe Asn Glu Ala Ser Ala Thr Glu Ala Ala Lys Tyr Val Phe
                85                  90                  95
Lys Asp Ala Met Arg Lys Val Thr Leu Pro Tyr Ser Gly Asn Tyr Glu
            100                 105                 110
Arg Leu Gln Thr Ala Ala Gly Lys Ile Arg Glu Asn Ile Pro Leu Gly
        115                 120                 125
Leu Pro Ala Leu Asp Ser Ala Ile Thr Thr Leu Phe Tyr Tyr Asn Ala
    130                 135                 140
Asn Ser Ala Ala Ser Ala Leu Met Val Leu Ile Gln Ser Thr Ser Glu
145                 150                 155                 160
Ala Ala Arg Tyr Lys Phe Ile Glu Gln Gln Ile Gly Lys Arg Val Asp
                165                 170                 175
Lys Thr Phe Leu Pro Ser Leu Ala Ile Ile Ser Leu Glu Asn Ser Trp
            180                 185                 190
Ser Ala Leu Ser Lys Gln Ile Gln Ile Ala Ser Thr Asn Asn Gly Gln
        195                 200                 205
Phe Glu Ser Pro Val Val Leu Ile Asn Ala Gln Asn Gln Val Ala Thr
    210                 215                 220
Ile Thr Asn Val Asp Ala Gly Val Val Thr Ser Asn Ile Ala Leu Leu
225                 230                 235                 240
Leu Asn Arg Asn Asn Met Ala
                245
</s-2-xi> </s-2> <s-2 seq-no="7"> <s-2-i> <s-2-i-a>263 amino acids</s-2-i-a> <s-2-i-b>amino acid</s-2-i-b> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>protein</s-2-ii-mt> </s-2-ii> <s-2-xi>7
Asp Val Ser Phe Arg Leu Ser Gly Ala Asp Pro Arg Ser Tyr Gly Met
1               5                   10                  15
Phe Ile Lys Asp Leu Arg Asn Ala Leu Pro Phe Arg Glu Lys Val Tyr
            20                  25                  30
Asn Ile Pro Leu Leu Leu Pro Ser Val Ser Gly Ala Gly Arg Tyr Leu
        35                  40                  45
Leu Met His Leu Phe Asn Tyr Asp Gly Lys Thr Ile Thr Val Ala Val
    50                  55                  60
Asp Val Thr Asn Val Tyr Ile Met Gly Tyr Leu Ala Asp Thr Thr Ser
65                  70                  75                  80
Tyr Phe Phe Asn Glu Pro Ala Ala Glu Leu Ala Ser Gln Tyr Val Phe
                85                  90                  95
Arg Asp Ala Arg Arg Lys Ile Thr Leu Pro Tyr Ser Gly Asn Tyr Glu
            100                 105                 110
Arg Leu Gln Ile Ala Ala Gly Lys Pro Arg Glu Lys Ile Pro Ile Gly
        115                 120                 125
Leu Pro Ala Leu Asp Ser Ala Ile Ser Thr Leu Leu His Tyr Asp Ser
    130                 135                 140
Thr Ala Ala Ala Gly Ala Leu Leu Val Leu Ile Gln Thr Thr Ala Glu
145                 150                 155                 160
Ala Ala Arg Phe Lys Tyr Ile Glu Gln Gln Ile Gln Glu Arg Ala Tyr
                165                 170                 175
Arg Asp Glu Val Pro Ser Leu Ala Thr Ile Ser Leu Glu Asn Ser Trp
            180                 185                 190
Ser Gly Leu Ser Lys Gln Ile Gln Leu Ala Gln Gly Asn Asn Gly Ile
        195                 200                 205
Phe Arg Thr Pro Ile Val Leu Val Asp Asn Lys Gly Asn Arg Val Gln
    210                 215                 220
Ile Thr Asn Val Thr Ser Lys Val Val Thr Ser Asn Ile Gln Leu Leu
225                 230                 235                 240
Leu Asn Thr Arg Asn Ile Ala Glu Gly Asp Asn Gly Asp Val Ser Thr
                245                 250                 255
Thr His Gly Phe Ser Ser Thr
            260
</s-2-xi> </s-2> <s-2 seq-no="8"> <s-2-i> <s-2-i-a>250 amino acids</s-2-i-a> <s-2-i-b>amino acid</s-2-i-b> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>protein</s-2-ii-mt> </s-2-ii> <s-2-xi>8
Ala Pro Thr Leu Glu Thr Ile Ala Ser Leu Asp Leu Asn Asn Pro Thr
1               5                   10                  15
Thr Tyr Leu Ser Phe Ile Thr Asn Ile Arg Thr Lys Val Ala Asp Lys
            20                  25                  30
Thr Glu Gln Cys Thr Ile Gln Lys Ile Ser Lys Thr Phe Thr Gln Arg
        35                  40                  45
Tyr Ser Tyr Ile Asp Leu Ile Val Ser Ser Thr Gln Lys Ile Thr Leu
    50                  55                  60
Ala Ile Asp Met Ala Asp Leu Tyr Val Leu Gly Tyr Ser Asp Ile Ala
65                  70                  75                  80
Asn Asn Lys Gly Arg Ala Phe Phe Phe Lys Asp Val Thr Glu Ala Val
                85                  90                  95
Ala Asn Asn Phe Phe Pro Gly Ala Thr Gly Thr Asn Arg Ile Lys Leu
            100                 105                 110
Thr Phe Thr Gly Ser Tyr Gly Asp Leu Glu Lys Asn Gly Gly Leu Arg
        115                 120                 125
Lys Asp Asn Pro Leu Gly Ile Phe Arg Leu Glu Asn Ser Ile Val Asn
    130                 135                 140
Ile Tyr Gly Lys Ala Gly Asp Val Lys Lys Gln Ala Lys Phe Phe Leu
145                 150                 155                 160
Leu Ala Ile Gln Met Val Ser Glu Ala Ala Arg Phe Lys Tyr Ile Ser
                165                 170                 175
Asp Lys Ile Pro Ser Glu Lys Tyr Glu Glu Val Thr Val Asp Glu Tyr
            180                 185                 190
Met Thr Ala Leu Glu Asn Asn Trp Ala Lys Leu Ser Thr Ala Val Tyr
        195                 200                 205
Asn Ser Lys Pro Ser Thr Thr Thr Ala Thr Lys Cys Gln Leu Ala Thr
    210                 215                 220
Ser Pro Val Thr Ile Ser Pro Trp Ile Phe Lys Thr Val Glu Glu Ile
225                 230                 235                 240
Lys Leu Val Met Gly Leu Leu Lys Ser Ser
                245                 250
</s-2-xi> </s-2> <s-2 seq-no="9"> <s-2-i> <s-2-i-a>261 amino acids</s-2-i-a> <s-2-i-b>amino acid</s-2-i-b> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>protein</s-2-ii-mt> </s-2-ii> <s-2-xi>9
Ile Asn Thr Ile Thr Phe Asp Ala Gly Asn Ala Thr Ile Asn Lys Tyr
1               5                   10                  15
Ala Thr Phe Met Glu Ser Leu Arg Asn Glu Ala Lys Asp Pro Ser Leu
            20                  25                  30
Lys Cys Tyr Gly Ile Pro Met Leu Pro Asn Thr Asn Ser Thr Ile Lys
        35                  40                  45
Tyr Leu Leu Val Lys Leu Gln Gly Ala Ser Leu Lys Thr Ile Thr Leu
    50                  55                  60
Met Leu Arg Arg Asn Asn Leu Tyr Val Met Gly Tyr Ser Asp Pro Tyr
65                  70                  75                  80
Asp Asn Lys Cys Arg Tyr His Ile Phe Asn Asp Ile Lys Gly Thr Glu
                85                  90                  95
Tyr Ser Asp Val Glu Asn Thr Leu Cys Pro Ser Ser Asn Pro Arg Val
            100                 105                 110
Ala Lys Pro Ile Asn Tyr Asn Gly Leu Tyr Pro Thr Leu Glu Lys Lys
        115                 120                 125
Ala Gly Val Thr Ser Arg Asn Glu Val Gln Leu Gly Ile Gln Ile Leu
    130                 135                 140
Ser Ser Asp Ile Gly Lys Ile Ser Gly Gln Gly Ser Phe Thr Glu Lys
145                 150                 155                 160
Ile Glu Ala Asp Phe Leu Leu Val Ala Ile Gln Met Val Ser Glu Ala
                165                 170                 175
Ala Arg Phe Lys Tyr Ile Glu Asn Gln Val Lys Thr Asn Phe Asn Arg
            180                 185                 190
Asp Phe Ser Pro Asn Asp Lys Val Leu Asp Leu Glu Glu Asn Trp Gly
        195                 200                 205
Lys Ile Ser Thr Ala Ile His Asn Ser Lys Asn Gly Ala Leu Pro Lys
    210                 215                 220
Pro Leu Glu Leu Lys Asn Ala Asp Gly Thr Lys Trp Ile Val Leu Arg
225                 230                 235                 240
Val Asp Glu Ile Lys Pro Asp Val Gly Leu Leu Asn Tyr Val Asn Gly
                245                 250                 255
Thr Cys Gln Ala Thr
            260
</s-2-xi> </s-2> <s-2 seq-no="10"> <s-2-i> <s-2-i-a>259 amino acids</s-2-i-a> <s-2-i-b>amino acid</s-2-i-b> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>protein</s-2-ii-mt> </s-2-ii> <s-2-xi>10
Val Thr Ser Ile Thr Leu Asp Leu Val Asn Pro Thr Ala Gly Gln Tyr
1               5                   10                  15
Ser Ser Phe Val Asp Lys Ile Arg Asn Asn Val Lys Asp Pro Asn Leu
            20                  25                  30
Lys Tyr Gly Gly Thr Asp Ile Ala Val Ile Gly Pro Pro Ser Lys Glu
        35                  40                  45
Lys Phe Leu Arg Ile Asn Phe Gln Ser Ser Arg Gly Thr Val Ser Leu
    50                  55                  60
Gly Leu Lys Arg Asp Asn Leu Tyr Val Val Ala Tyr Leu Ala Met Asp
65                  70                  75                  80
Asn Thr Asn Val Asn Arg Ala Tyr Tyr Phe Arg Ser Glu Ile Thr Ser
                85                  90                  95
Ala Glu Ser Thr Ala Leu Phe Pro Glu Ala Thr Thr Ala Asn Gln Lys
            100                 105                 110
Ala Leu Glu Tyr Thr Glu Asp Tyr Gln Ser Ile Glu Lys Asn Ala Gln
        115                 120                 125
Ile Thr Gln Gly Asp Gln Ser Arg Lys Glu Leu Gly Leu Gly Ile Asp
    130                 135                 140
Leu Leu Ser Thr Ser Met Glu Ala Val Asn Lys Lys Ala Arg Val Val
145                 150                 155                 160
Lys Asp Glu Ala Arg Phe Leu Leu Ile Ala Ile Gln Met Thr Ala Glu
                165                 170                 175
Ala Ala Arg Phe Arg Tyr Ile Gln Asn Leu Val Ile Lys Asn Phe Pro
            180                 185                 190
Asn Lys Phe Asn Ser Glu Asn Lys Val Ile Gln Phe Glu Val Asn Trp
        195                 200                 205
Lys Lys Ile Ser Thr Ala Ile Tyr Gly Asp Ala Lys Asn Gly Val Phe
    210                 215                 220
Asn Lys Asp Tyr Asp Phe Gly Phe Gly Lys Val Arg Gln Val Lys Asp
225                 230                 235                 240
Leu Gln Met Gly Leu Leu Met Tyr Leu Gly Lys Pro Lys Ser Ser Asn
                245                 250                 255
Glu Ala Asn
</s-2-xi> </s-2> <s-2 seq-no="11"> <s-2-i> <s-2-i-a>813 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>cDNA</s-2-ii-mt> </s-2-ii> <s-2-xi>11
GGGCTAGATA CCGTGTCATT CTCAACCAAA GGTGCCACTT ATATTACCTA CGTGAATTTC     60
TTGAATGAGC TACGAGTTAA ATTGAAACCC GAAGGTAACA GCCATGGAAT CCCATTGCTG    120
CGCAAAAAAT GTGATGATCC TGGAAAGTGT TTCGTTTTGG TAGCGCTTTC AAATGACAAT    180
GGACAGTTGG CGGAAATAGC TATAGATGTT ACAAGTGTTT ATGTGGTGGG CTATCAAGTA    240
AGAAACAGAT CTTACTTCTT TAAAGATGCT CCAGATGCTG CTTACGAAGG CCTCTTCAAA    300
AACACAATTA AAACAAGACT TCATTTTGGC GGCAGCTATC CCTCGCTGGA AGGTGAGAAG    360
GCATATAGAG AGACAACAGA CTTGGGCATT GAACCATTAA GGATTGGCAT CAAGAAACTT    420
GATGAAAATG CGATAGACAA TTATAAACCA ACGGAGATAG CTAGTTCTCT ATTGGTTGTT    480
ATTCAAATGG TGTCTGAAGC AGCTCGATTC ACCTTTATTG AGAACCAAAT TAGAAATAAC    540
TTTCAACAGA GAATTCGCCC GGCGAATAAT ACAATCAGCC TTGAGAATAA ATGGGGTAAA    600
CTCTCGTTCC AGATCCGGAC ATCAGGTGCA AATGGAATGT TTTCGGAGGC AGTTGAATTG    660
GAACGTGCAA ATGGCAAAAA ATACTATGTC ACCGCAGTTG ATCAAGTAAA ACCCAAAATA    720
GCACTCTTGA AGTTCGTCGA TAAAGATCCT AAAACGAGCC TTGCTGCTGA ATTGATAATC    780
CAGAACTATG AGTCATTAGT GGGCTTTGAT TAG                                 813
</s-2-xi> </s-2> <s-2 seq-no="12"> <s-2-i> <s-2-i-a>846 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>cDNA</s-2-ii-mt> </s-2-ii> <s-2-xi>12
ATGGCGGCAA AGATGGCGAA GAACGTGGAC AAGCCGCTCT TCACCGCGAC GTTCAACGTC     60
CAGGCCAGCT CCGCCGACTA CGCCACCTTC ATCGCCGGCA TCCGCAACAA GCTCCGCAAC    120
CCGGCGCACT TCTCCCACAA CCGCCCCGTG CTGCCGCCGG TCGAGCCCAA CGTCCCGCCG    180
AGCAGGTGGT TCCACGTCGT GCTCAAGGCC TCGCCGACCA GCGCCGGGCT CACGCTGGCC    240
ATCCGCGCGG ACAACATCTA CCTGGAGGGC TTCAAGAGCA GCGACGGCAC CTGGTGGGAG    300
CTCACCCCGG GCCTCATCCC CGGCGCCACC TACGTCGGGT TCGGCGGCAC CTACCGCGAC    360
CTCCTCGGCG ACACCGACAA GCTAACCAAC GTCGCTCTCG GCCGACAGCA GCTGGCGGAC    420
GCGGTGACCG CGCTCCACGG GCGCACCAAG GCCGACAAGG CCTCCGGCCC GAAGCAGCAG    480
CAGGCGAGGG AGGCGGTGAC GACGCTGGTC CTCATGGTGA ACGAGGCCAC GCGGTTCCAG    540
ACGGTGTCTG GGTTCGTGGC CGGGTTGCTG CACCCCAAGG CGGTGGAGAA GAAGAGCGGG    600
AAGATCGGCA ATGAGATGAA GGCCCAGGTG AACGGGTGGC AGGACCTGTC CGCGGCGCTG    660
CTGAAGACGG ACGTGAAGCC TCCGCCGGGA AAGTCGCCAG CGAAGTTCGC GCCGATCGAG    720
AAGATGGGCG TGAGGACGGC TGAACAGGCC GCCAACACGC TGGGGATCCT GCTGTTCGTG    780
GAGGTGCCGG GTGGGTTGAC GGTGGCCAAG GCGCTGGAGC TGTTCCATGC GAGTGGTGGG    840
AAATAG                                                               846
</s-2-xi> </s-2> <s-2 seq-no="13"> <s-2-i> <s-2-i-a>913 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>cDNA</s-2-ii-mt> </s-2-ii> <s-2-xi>13
CGTCCGAAAA TGGTGAAATG CTTACTACTT TCTTTTTTAA TTATCGCCAT CTTCATTGGT     60
GTTCCTACTG CCAAAGGCGA TGTTAACTTC GATTTGTCGA CTGCCACTGC AAAAACCTAC    120
ACAAAATTTA TCGAAGATTT CAGGGCGACT CTTCCATTTA GCCATAAAGT GTATGATATA    180
CCTCTACTGT ATTCCACTAT TTCCGACTCC AGACGTTTCA TACTCCTCGA TCTTACAAGT    240
TATGCATATG AAACCATCTC GGTGGCCATA GATGTGACGA ACGTTTATGT TGTGGCGTAT    300
CGCACCCGCG ATGTATCCTA CTTTTTTAAA GAATCTCCTC CTGAAGCTTA TAACATCCTA    360
TTCAAAGGTA CGCGGAAAAT TACACTGCCA TATACCGGTA ATTATGAAAA TCTTCAAACT    420
GCTGCACACA AAATAAGAGA GAATATTGAT CTTGGACTCC CTGCCTTGAG TAGTGCCATT    480
ACCACATTGT TTTATTACAA TGCCCAATCT GCTCCTTCTG CATTGCTTGT ACTAATCCAG    540
ACGACTGCAG AAGCTGCAAG ATTTAAGTAT ATCGAGCGAC ACGTTGCTAA GTATGTTGCC    600
ACTAACTTTA AGCCAAATCT AGCCATCATA AGCTTGGAAA ATCAATGGTC TGCTCTCTCC    660
AACAAATCTT TTTGGCGCAG AATCAAGGAG GAAAATTTAG AAATCCTGTC GACCTTATAA    720
AACCTACCGG GGAACGGTTT CAAGTAACCA ATGTTGATTC AGATGTTGTA AAAGGTAATA    780
TCAAACTCCT GCTGAACTCC AGAGCTAGCA CTGCTGATGA AAACTTTATC ACAACCATGA    840
CTCTACTTGG GGAATCTGTT GTGAATTGAA AGTTTAATAA TCCACCCATA TCGAAATAAG    900
GCATGTTCAT GAC                                                       913
</s-2-xi> </s-2> <s-2 seq-no="14"> <s-2-i> <s-2-i-a>32 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>14
TTYAARGAYG CNCCNGAYGC NGCNTAYGAR GG                                   32
</s-2-xi> </s-2> <s-2 seq-no="15"> <s-2-i> <s-2-i-a>32 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>15
ACYTGRTCNA CNGCNGTNAC RTARTAYTTY TT                                   32
</s-2-xi> </s-2> <s-2 seq-no="16"> <s-2-i> <s-2-i-a>32 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>16
GGNYTNGAYA CNGTNWSNTT YWSNACNAAR GG                                   32
</s-2-xi> </s-2> <s-2 seq-no="17"> <s-2-i> <s-2-i-a>23 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>17
AATGGTTCAA TGCCCAAGTC TGT                                             23
</s-2-xi> </s-2> <s-2 seq-no="18"> <s-2-i> <s-2-i-a>23 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>18
TGTCTCTCTA TATGCCTTCT CAC                                             23
</s-2-xi> </s-2> <s-2 seq-no="19"> <s-2-i> <s-2-i-a>53 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>19
TCAACCCGGG CTAGATACCG TGTCATTCTC AACCAAAGGT GCCACTTATA TTA            53
</s-2-xi> </s-2> <s-2 seq-no="20"> <s-2-i> <s-2-i-a>23 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>20
CTTCATTTTG GCGGCACGTA TCC                                             23
</s-2-xi> </s-2> <s-2 seq-no="21"> <s-2-i> <s-2-i-a>46 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>21
CTCGAGGCTG CAAGCTTACG TGGGATTTTT TTTTTTTTTT TTTTTT                    46
</s-2-xi> </s-2> <s-2 seq-no="22"> <s-2-i> <s-2-i-a>18 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>22
CTCGCTGGAA GGTGAGAA                                                   18
</s-2-xi> </s-2> <s-2 seq-no="23"> <s-2-i> <s-2-i-a>25 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>23
CTCGAGGCTG CAAGCTTACG TGGGA                                           25
</s-2-xi> </s-2> <s-2 seq-no="24"> <s-2-i> <s-2-i-a>35 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>24
TGATCTCGAG TACTATTTAG GATCTTTATC GACGA                                35
</s-2-xi> </s-2> <s-2 seq-no="25"> <s-2-i> <s-2-i-a>22 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>25
GTAAGCAGCA TCTGGAGCAT CT                                              22
</s-2-xi> </s-2> <s-2 seq-no="26"> <s-2-i> <s-2-i-a>21 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>26
CATTCAAGAA ATTCACGTAG G                                               21
</s-2-xi> </s-2> <s-2 seq-no="27"> <s-2-i> <s-2-i-a>23 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>27
GGCCTGGACA CCGTGAGCTT TAG                                             23
</s-2-xi> </s-2> <s-2 seq-no="28"> <s-2-i> <s-2-i-a>25 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>28
TCGATTGCGA TCCTAAATAG TACTC                                           25
</s-2-xi> </s-2> <s-2 seq-no="29"> <s-2-i> <s-2-i-a>28 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>29
TTTAGGATCG CAATCGACGA ACTTCAAG                                        28
</s-2-xi> </s-2> <s-2 seq-no="30"> <s-2-i> <s-2-i-a>32 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>30
GTTCGTCTGT AAAGATCCTA AATAGTACTC GA                                   32
</s-2-xi> </s-2> <s-2 seq-no="31"> <s-2-i> <s-2-i-a>27 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>31
GGATCTTTAC AGACGAACTT CAAGAGT                                         27
</s-2-xi> </s-2> <s-2 seq-no="32"> <s-2-i> <s-2-i-a>25 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>32
TCTTGTGCTT CGTCGATAAA GATCC                                           25
</s-2-xi> </s-2> <s-2 seq-no="33"> <s-2-i> <s-2-i-a>27 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>33
ATCGACGAAG CACAAGAGTG CTATTTT                                         27
</s-2-xi> </s-2> <s-2 seq-no="34"> <s-2-i> <s-2-i-a>32 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>34
GTAAAACCAT GCATAGCACT CTTGAAGTTC GT                                   32
</s-2-xi> </s-2> <s-2 seq-no="35"> <s-2-i> <s-2-i-a>32 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>35
AGTGCTATGC ATGGTTTTAC TTGATCAACT GC                                   32
</s-2-xi> </s-2> <s-2 seq-no="36"> <s-2-i> <s-2-i-a>29 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>36
AGCACATGTG GTGCCACTTA TATTACCTA                                       29
</s-2-xi> </s-2> <s-2 seq-no="37"> <s-2-i> <s-2-i-a>33 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>37
TAAGTGGCAC CACATGTGCT AAAGCTCACG GTG                                  33
</s-2-xi> </s-2> <s-2 seq-no="38"> <s-2-i> <s-2-i-a>25 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>38
TGACTGTGGA CAGTTGGCGG AAATA                                           25
</s-2-xi> </s-2> <s-2 seq-no="39"> <s-2-i> <s-2-i-a>33 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>39
GCCAACTGTC CACAGTCATT TGAAAGCGCT ACC                                  33
</s-2-xi> </s-2> <s-2 seq-no="40"> <s-2-i> <s-2-i-a>36 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>40
GATGATCCTG GAAAGGCTTT CGTTTTGGTA GCGCTT                               36
</s-2-xi> </s-2> <s-2 seq-no="41"> <s-2-i> <s-2-i-a>41 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>41
AAGCCTTTCC AGGATCATCA GCTTTTTTGC GCAGCAATGG G                         41
</s-2-xi> </s-2> <s-2 seq-no="42"> <s-2-i> <s-2-i-a>23 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>42
AAGCCTTTCC AGGATCATCA CAT                                             23
</s-2-xi> </s-2> <s-2 seq-no="43"> <s-2-i> <s-2-i-a>18 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>43
GCGACTCTCT ACTGTTTC                                                   18
</s-2-xi> </s-2> <s-2 seq-no="44"> <s-2-i> <s-2-i-a>21 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>44
CGTTAGCAAT TTAACTGTGA T                                               21
</s-2-xi> </s-2> <s-2 seq-no="45"> <s-2-i> <s-2-i-a>16 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>45
AACAGCTATG ACCATG                                                     16
</s-2-xi> </s-2> <s-2 seq-no="46"> <s-2-i> <s-2-i-a>30 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>46
TGAACTCGAG GAAAACTACC TATTTCCCAC                                      30
</s-2-xi> </s-2> <s-2 seq-no="47"> <s-2-i> <s-2-i-a>19 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>47
GCATTACATC CATGGCGGC                                                  19
</s-2-xi> </s-2> <s-2 seq-no="48"> <s-2-i> <s-2-i-a>64 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>48
GATATCTCGA GTTAACTATT TCCCACCACA CGCATGGAAC AGCTCCAGCG CCTTGGCCAC     60
CGTC                                                                  64
</s-2-xi> </s-2> <s-2 seq-no="49"> <s-2-i> <s-2-i-a>21 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>49
TGTCTGTTCG TGGAGGTGCC G                                               21
</s-2-xi> </s-2> <s-2 seq-no="50"> <s-2-i> <s-2-i-a>27 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>50
CCAAGTGTCT GGAGCTGTTC CATGCGA                                         27
</s-2-xi> </s-2> <s-2 seq-no="51"> <s-2-i> <s-2-i-a>29 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>51
GATGTTAAYT TYGAYTTGTC NACDGCTAC                                       29
</s-2-xi> </s-2> <s-2 seq-no="52"> <s-2-i> <s-2-i-a>29 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>52
ATTGGNAGDG TAGCCCTRAA RTCYTCDAT                                       29
</s-2-xi> </s-2> <s-2 seq-no="53"> <s-2-i> <s-2-i-a>32 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>53
GCCACTGCAA AAACCTACAC AAAATTTATT GA                                   32
</s-2-xi> </s-2> <s-2 seq-no="54"> <s-2-i> <s-2-i-a>22 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>54
GATGTTAACT TCGATTTGTC GA                                              22
</s-2-xi> </s-2> <s-2 seq-no="55"> <s-2-i> <s-2-i-a>33 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>55
TCAACTCGAG GTACTCAATT CACAACAGAT TCC                                  33
</s-2-xi> </s-2> <s-2 seq-no="56"> <s-2-i> <s-2-i-a>20 amino acids</s-2-i-a> <s-2-i-b>amino acid</s-2-i-b> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>peptide</s-2-ii-mt> </s-2-ii> <s-2-xi>56
Cys His His His Ala Ser Arg Val Ala Arg Met Ala Ser Asp Glu Phe
1               5                   10                  15
Pro Ser Met Cys
            20
</s-2-xi> </s-2> <s-2 seq-no="57"> <s-2-i> <s-2-i-a>20 amino acids</s-2-i-a> <s-2-i-b>amino acid</s-2-i-b> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>peptide</s-2-ii-mt> </s-2-ii> <s-2-xi>57
Pro Ser Gly Gln Ala Gly Ala Ala Ala Ser Glu Ser Leu Phe Ile Ser
1               5                   10                  15
Asn His Ala Tyr
            20
</s-2-xi> </s-2> <s-2 seq-no="58"> <s-2-i> <s-2-i-a>22 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>58
CAGCCATGGA ATCCCATTGC TG                                              22
</s-2-xi> </s-2> <s-2 seq-no="59"> <s-2-i> <s-2-i-a>28 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>59
CACATGTAAA ACAAGACTTC ATTTTGGC                                        28
</s-2-xi> </s-2> <s-2 seq-no="60"> <s-2-i> <s-2-i-a>36 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>60
TGAAGTCTTG TTTTAGATGT GTTTTTGAAG AGGCCT                               36
</s-2-xi> </s-2> <s-2 seq-no="61"> <s-2-i> <s-2-i-a>30 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>61
ATGCCATATG CAATTATAAA CCAACGGAGA                                      30
</s-2-xi> </s-2> <s-2 seq-no="62"> <s-2-i> <s-2-i-a>39 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>62
GGTTTATAAT TGCATATGGC ATTTTCATCA AGTTTCTTG                            39
</s-2-xi> </s-2> <s-2 seq-no="63"> <s-2-i> <s-2-i-a>33 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>63
CTTTCAACAA TGCATTCGCC CGGCGAATAA TAC                                  33
</s-2-xi> </s-2> <s-2 seq-no="64"> <s-2-i> <s-2-i-a>33 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>64
GCGAATGCAT TGTTGAAAGT TATTTCTAAT TTG                                  33
</s-2-xi> </s-2> <s-2 seq-no="65"> <s-2-i> <s-2-i-a>26 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>65
GTTTTGTGAG GCAGTTGAAT TGGAAC                                          26
</s-2-xi> </s-2> <s-2 seq-no="66"> <s-2-i> <s-2-i-a>34 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>66
TTCAACTGCC TCACAAAACA TTCCATTTGC ACCT                                 34
</s-2-xi> </s-2> <s-2 seq-no="67"> <s-2-i> <s-2-i-a>24 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>67
AAAAGCTGAT GATCCTGGAA AGTG                                            24
</s-2-xi> </s-2> <s-2 seq-no="68"> <s-2-i> <s-2-i-a>35 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>68
TCCAGGATCA TCAGCTTTTT TGCGCAGCAA TGGGA                                35
</s-2-xi> </s-2> <s-2 seq-no="69"> <s-2-i> <s-2-i-a>321 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>69
GACATCCAGA TGACTCAGTC TCCATCTTCC ATGTCTGCAT CTCTGGGAGA CAGAGTCACT     60
ATCACTTGCC GGGCGAGTCA GGACATTAAT AGCTATTTAA GCTGGTTCCA GCAGAAACCA    120
GGGAAATCTC CTAAGACCCT GATCTATCGT GCAAACAGAT TGGTAGATGG GGTCCCATCA    180
AGGTTCAGTG GCAGTGGATC TGGGACAGAT TATACTCTCA CCATCAGCAG CCTGCAATAT    240
GAAGATTTTG GAATTTATTA TTGTCAACAG TATGATGAGT CTCCGTGGAC GTTCGGTGGA    300
GGCACCAAGC TTGAAATCAA A                                              321
</s-2-xi> </s-2> <s-2 seq-no="70"> <s-2-i> <s-2-i-a>354 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>70
CAGATCCAGT TGGTGCAGTC TGGACCTGGC CTGAAGAAGC CTGGAGGGTC CGTCAGAATC     60
TCCTGCGCAG CTTCTGGGTA TACCTTCACA AACTATGGAA TGAACTGGGT GAAGCAGGCT    120
CCAGGAAAGG GTTTAAGGTG GATGGGCTGG ATAAACACCC ACACTGGAGA GCCAACATAT    180
GCTGATGACT TCAAGGGACG GTTTACCTTC TCTTTGGACA CGTCTAAGAG CACTGCCTAT    240
TTACAGATCA ACAGCCTCAG AGCCGAGGAC ACGGCTACAT ATTTCTGTAC AAGACGGGGT    300
TACGACTGGT ACTTCGATGT CTGGGGCCAA GGGACCACGG TCACCGTCTC CTCC          354
</s-2-xi> </s-2> <s-2 seq-no="71"> <s-2-i> <s-2-i-a>354 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>71
GAGATCCAGT TGGTGCAGTC TGGAGGAGGC CTGGTGAAGC CTGGAGGGTC CGTCAGAATC     60
TCCTGCGCAG CTTCTGGGTA TACCTTCACA AACTATGGAA TGAACTGGGT GCGCCAGGCT    120
CCAGGAAAGG GTTTAGAGTG GATGGGCTGG ATAAACACCC ACACTGGAGA GCCAACATAT    180
GCTGATTCTT TCAAGGGACG GTTTACCTTC TCTTTGGACG ATTCTAAGAA CACTGCCTAT    240
TTACAGATCA ACAGCCTCAG AGCCGAGGAC ACGGCTGTGT ATTTCTGTAC AAGACGGGGT    300
TACGACTGGT ACTTCGATGT CTGGGGCCAA GGGACCACGG TCACCGTCTC CTCC          354
</s-2-xi> </s-2> <s-2 seq-no="72"> <s-2-i> <s-2-i-a>321 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>72
GACATCCAGA TGACTCAGTC TCCATCTTCC CTGTCTGCAT CTGTAGGAGA CAGAGTCACT     60
ATCACTTGCC GGGCGAGTCA GGACATTAAT AGCTATTTAA GCTGGTTCCA GCAGAAACCA    120
GGGAAAGCTC CTAAGACCCT GATCTATCGT GCAAACAGAT TGGAATCTGG GGTCCCATCA    180
AGGTTCAGTG GCAGTGGATC TGGGACAGAT TATACTCTCA CCATCAGCAG CCTGCAATAT    240
GAAGATTTTG GAATTTATTA TTGTCAACAG TATGATGAGT CTCCGTGGAC GTTCGGTGGA    300
GGCACCAAGC TTGAAATCAA A                                              321
</s-2-xi> </s-2> <s-2 seq-no="73"> <s-2-i> <s-2-i-a>70 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>73
TGTCATCATC ATGCATCGCG AGTTGCCAGA ATGGCATCTG ATGAGTTTCC TTCTATGTGC     60
GCAAGTACTC                                                            70
</s-2-xi> </s-2> <s-2 seq-no="74"> <s-2-i> <s-2-i-a>78 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>74
TCGAGAGTAC TTGCGCACAT AGAAGGAAAC TCATCAGATG CCATTCTGGC AACTCGCGAT     60
GCATGATGAT GACATGCA                                                   78
</s-2-xi> </s-2> <s-2 seq-no="75"> <s-2-i> <s-2-i-a>30 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>75
TGTTCGGCCG CATGTCATCA TCATGCATCG                                      30
</s-2-xi> </s-2> <s-2 seq-no="76"> <s-2-i> <s-2-i-a>15 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>76
AGTCATGCCC CGCGC                                                      15
</s-2-xi> </s-2> <s-2 seq-no="77"> <s-2-i> <s-2-i-a>18 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>77
TCCCGGCTGT CCTACAGT                                                   18
</s-2-xi> </s-2> <s-2 seq-no="78"> <s-2-i> <s-2-i-a>37 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>78
TCCAGCCTGT CCAGATGGTG TGTGAGTTTT GTCACAA                              37
</s-2-xi> </s-2> <s-2 seq-no="79"> <s-2-i> <s-2-i-a>76 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>79
CTAACTCGAG AGTACTGTAT GCATGGTTCG AGATGAACAA AGATTCTGAG GCTGCAGCTC     60
CAGCCTGTCC AGATGG                                                     76
</s-2-xi> </s-2> <s-2 seq-no="80"> <s-2-i> <s-2-i-a>20 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>80
CTAACTCGAG AGTACTGTAT                                                 20
</s-2-xi> </s-2> <s-2 seq-no="81"> <s-2-i> <s-2-i-a>36 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>81
TCCAGCCTGT CCAGATGGAC ACTCTCCCCT GTTGAA                               36
</s-2-xi> </s-2> <s-2 seq-no="82"> <s-2-i> <s-2-i-a>18 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>82
GTACAGTGGA AGGTGGAT                                                   18
</s-2-xi> </s-2> <s-2 seq-no="83"> <s-2-i> <s-2-i-a>31 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>83
CATGCGGCCG ATTTAGGATC TTTATCGACG A                                    31
</s-2-xi> </s-2> <s-2 seq-no="84"> <s-2-i> <s-2-i-a>22 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>84
AACATCCAGT TGGTGCAGTC TG                                              22
</s-2-xi> </s-2> <s-2 seq-no="85"> <s-2-i> <s-2-i-a>20 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>85
GAGGAGACGG TGACCGTGGT                                                 20
</s-2-xi> </s-2> <s-2 seq-no="86"> <s-2-i> <s-2-i-a>19 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>86
GACATCAAGA TGACCCAGT                                                 19
</s-2-xi> </s-2> <s-2 seq-no="87"> <s-2-i> <s-2-i-a>21 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>87
GTTTGATTTC AAGCTTGGTG C                                               21
</s-2-xi> </s-2> <s-2 seq-no="88"> <s-2-i> <s-2-i-a>31 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>88
ACTTCGGCCG CACCATCTGG ACAGGCTGGA G                                    31
</s-2-xi> </s-2> <s-2 seq-no="89"> <s-2-i> <s-2-i-a>723 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>89
GACATCCAGA TGACTCAGTC TCCATCTTCC CTGTCTGCAT CTGTAGGAGA CAGAGTCACT     60
ATCACTTGCC GGGCGAGTCA GGACATTAAT AGCTATTTAA GCTGGTTCCA GCAGAAACCA    120
GGGAAAGCTC CTAAGACCCT GATCTATCGT GCAAACAGAT TGGAATCTGG GGTCCCATCA    180
AGGTTCAGTG GCAGTGGATC TGGGACAGAT TATACTCTCA CCATCAGCAG CCTGCAATAT    240
GAAGATTTTG GAATTTATTA TTGTCAACAG TATGATGAGT CTCCGTGGAC GTTCGGTGGA    300
GGCACCAAGC TTGAGATGAA AGGTGGCGGT GGATCTGGTG GAGGTGGGTC CGGAGGTGGA    360
GGATCTGAGA TCCAGTTGGT GCAGTCTGGA GGAGGCCTGG TGAAGCCTGG AGGGTCCGTC    420
AGAATCTCCT GCGCAGCTTC TGGGTATACC TTCACAAACT ATGGAATGAA CTGGGTGCGC    480
CAGGCTCCAG GAAAGGGTTT AGAGTGGATG GGCTGGATAA ACACCCACAC TGGAGAGCCA    540
ACATATGCTG ATTCTTTCAA GGGACGGTTT ACCTTCTCTT TGGACGATTC TAAGAACACT    600
GCCTATTTAC AGATCAACAG CCTCAGAGCC GAGGACACGG CTGTGTATTT CTGTACAAGA    660
CGGGGTTACG ACTGGTACTT CGATGTCTGG GGCCAAGGGA CCACGGTCAC CGTCTCCTCA    720
TGA                                                                  723
</s-2-xi> </s-2> <s-2 seq-no="90"> <s-2-i> <s-2-i-a>723 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>90
GAGATCCAGT TGGTGCAGTC TGGAGGAGGC CTGGTGAAGC CTGGAGGGTC CGTCAGAATC     60
TCCTGCGCAG CTTCTGGGTA TACCTTCACA AACTATGGAA TGAACTGGGT GCGCCAGGCT    120
CCAGGAAAGG GTTTAGAGTG GATGGGCTGG ATAAACACCC ACACTGGAGA GCCAACATAT    180
GCTGATTCTT TCAAGGGACG GTTTACCTTC TCTTTGGACG ATTCTAAGAA CACTGCCTAT    240
TTACAGATCA ACAGCCTCAG AGCCGAGGAC ACGGCTGTGT ATTTCTGTAC AAGACGGGGT    300
TACGACTGGT ACTTCGATGT CTGGGGCCAA GGGACCACGG TCACCGTCTC CTCAGGTGGC    360
GGTGGATCTG GTGGAGGTGG GTCCGGAGGT GGAGGATCTG ACATCCAGAT GACTCAGTCT    420
CCATCTTCCC TGTCTGCATC TGTAGGAGAC AGAGTCACTA TCACTTGCCG GGCGAGTCAG    480
GACATTAATA GCTATTTAAG CTGGTTCCAG CAGAAACCAG GGAAAGCTCC TAAGACCCTG    540
ATCTATCGTG CAAACAGATT GGAATCTGGG GTCCCATCAA GGTTCAGTGG CAGTGGATCT    600
GGGACAGATT ATACTCTCAC CATCAGCAGC CTGCAATATG AAGATTTTGG AATTTATTAT    660
TGTCAACAGT ATGATGAGTC TCCGTGGACG TTCGGTGGAG GCACCAAGCT TGAGATGAAA    720
TGA                                                                  723
</s-2-xi> </s-2> <s-2 seq-no="91"> <s-2-i> <s-2-i-a>51 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>91
CGGACCCACC TCCACCAGAT CCACCGCCAC CTTTCATCTC AAGCTTGGTG C              51
</s-2-xi> </s-2> <s-2 seq-no="92"> <s-2-i> <s-2-i-a>19 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>92
GACATCCAGA TGACTCAGT                                                  19
</s-2-xi> </s-2> <s-2 seq-no="93"> <s-2-i> <s-2-i-a>49 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>93
GGTGGAGGTG GGTCCGGAGG TGGAGGATCT GAGATCCAGT TGGTGCAGT                 49
</s-2-xi> </s-2> <s-2 seq-no="94"> <s-2-i> <s-2-i-a>35 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>94
TGTACTCGAG CCCATCATGA GGAGACGGTG ACCGT                                35
</s-2-xi> </s-2> <s-2 seq-no="95"> <s-2-i> <s-2-i-a>49 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>95
GGTGGAGGTG GGTCCGGAGG TGGAGGATCT GACATCCAGA TGACTCAGT                 49
</s-2-xi> </s-2> <s-2 seq-no="96"> <s-2-i> <s-2-i-a>37 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>96
TGTACTCGAG CCCATCATTT CATCTCAAGC TTGGTGC                              37
</s-2-xi> </s-2> <s-2 seq-no="97"> <s-2-i> <s-2-i-a>22 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>97
GAGATCCAGT TGGTGCAGTC TG                                              22
</s-2-xi> </s-2> <s-2 seq-no="98"> <s-2-i> <s-2-i-a>49 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>98
CGGACCCACC TCCACCAGAT CCACCGCCAC CTGAGGAGAC GGTGACCGT                 49
</s-2-xi> </s-2> <s-2 seq-no="99"> <s-2-i> <s-2-i-a>251 amino acids</s-2-i-a> <s-2-i-b>amino acid</s-2-i-b> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>protein</s-2-ii-mt> </s-2-ii> <s-2-xi>99
Gly Leu Asp Thr Val Ser Phe Ser Thr Lys Gly Ala Thr Tyr Ile Thr
1               5                   10                  15
Tyr Val Asn Phe Leu Asn Glu Leu Arg Val Lys Leu Lys Pro Glu Gly
            20                  25                  30
Asn Ser His Gly Ile Pro Leu Leu Arg Lys Lys Cys Asp Asp Pro Gly
        35                  40                  45
Lys Ala Phe Val Leu Val Ala Leu Ser Asn Asp Asn Gly Gln Leu Ala
    50                  55                  60
Glu Ile Ala Ile Asp Val Thr Ser Val Tyr Val Val Gly Tyr Gln Val
65                  70                  75                  80
Arg Asn Arg Ser Tyr Phe Phe Lys Asp Ala Pro Asp Ala Ala Tyr Glu
                85                  90                  95
Gly Leu Phe Lys Asn Thr Ile Lys Thr Arg Leu His Phe Gly Gly Ser
            100                 105                 110
Tyr Pro Ser Leu Glu Gly Glu Lys Ala Tyr Arg Glu Thr Thr Asp Leu
        115                 120                 125
Gly Ile Glu Pro Leu Arg Ile Gly Ile Lys Lys Leu Asp Glu Asn Ala
    130                 135                 140
Ile Asp Asn Tyr Lys Pro Thr Glu Ile Ala Ser Ser Leu Leu Val Val
145                 150                 155                 160
Ile Gln Met Val Ser Glu Ala Ala Arg Phe Thr Phe Ile Glu Asn Gln
                165                 170                 175
Ile Arg Asn Asn Phe Gln Gln Arg Ile Arg Pro Ala Asn Asn Thr Ile
            180                 185                 190
Ser Leu Glu Asn Lys Trp Gly Lys Leu Ser Phe Gln Ile Arg Thr Ser
        195                 200                 205
Gly Ala Asn Gly Met Phe Ser Glu Ala Val Glu Leu Glu Arg Ala Asn
    210                 215                 220
Gly Lys Lys Tyr Tyr Val Thr Ala Val Asp Gln Val Lys Pro Lys Ile
225                 230                 235                 240
Ala Leu Leu Lys Phe Val Asp Lys Asp Pro Lys
                245                 250
</s-2-xi> </s-2> <s-2 seq-no="100"> <s-2-i> <s-2-i-a>251 amino acids</s-2-i-a> <s-2-i-b>amino acid</s-2-i-b> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>protein</s-2-ii-mt> </s-2-ii> <s-2-xi>100
Gly Leu Asp Thr Val Ser Phe Ser Thr Lys Gly Ala Thr Tyr Ile Thr
1               5                   10                  15
Tyr Val Asn Phe Leu Asn Glu Leu Arg Val Lys Leu Lys Pro Glu Gly
            20                  25                  30
Asn Ser His Gly Ile Pro Leu Leu Arg Lys Lys Ala Asp Asp Pro Gly
        35                  40                  45
Lys Cys Phe Val Leu Val Ala Leu Ser Asn Asp Asn Gly Gln Leu Ala
    50                  55                  60
Glu Ile Ala Ile Asp Val Thr Ser Val Tyr Val Val Gly Tyr Gln Val
65                  70                  75                  80
Arg Asn Arg Ser Tyr Phe Phe Lys Asp Ala Pro Asp Ala Ala Tyr Glu
                85                  90                  95
Gly Leu Phe Lys Asn Thr Ile Lys Thr Arg Leu His Phe Gly Gly Ser
            100                 105                 110
Tyr Pro Ser Leu Glu Gly Glu Lys Ala Tyr Arg Glu Thr Thr Asp Leu
        115                 120                 125
Gly Ile Glu Pro Leu Arg Ile Gly Ile Lys Lys Leu Asp Glu Asn Ala
    130                 135                 140
Ile Asp Asn Tyr Lys Pro Thr Glu Ile Ala Ser Ser Leu Leu Val Val
145                 150                 155                 160
Ile Gln Met Val Ser Glu Ala Ala Arg Phe Thr Phe Ile Glu Asn Gln
                165                 170                 175
Ile Arg Asn Asn Phe Gln Gln Arg Ile Arg Pro Ala Asn Asn Thr Ile
            180                 185                 190
Ser Leu Glu Asn Lys Trp Gly Lys Leu Ser Phe Gln Ile Arg Thr Ser
        195                 200                 205
Gly Ala Asn Gly Met Phe Ser Glu Ala Val Glu Leu Glu Arg Ala Asn
    210                 215                 220
Gly Lys Lys Tyr Tyr Val Thr Ala Val Asp Gln Val Lys Pro Lys Ile
225                 230                 235                 240
Ala Leu Leu Lys Phe Val Asp Lys Asp Pro Lys
                245                 250
</s-2-xi> </s-2> <s-2 seq-no="101"> <s-2-i> <s-2-i-a>251 amino acids</s-2-i-a> <s-2-i-b>amino acid</s-2-i-b> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>protein</s-2-ii-mt> </s-2-ii> <s-2-xi>101
Gly Leu Asp Thr Val Ser Phe Ser Thr Lys Gly Ala Thr Tyr Ile Thr
1               5                   10                  15
Tyr Val Asn Phe Leu Asn Glu Leu Arg Val Lys Leu Lys Pro Glu Gly
            20                  25                  30
Asn Ser His Gly Ile Pro Leu Leu Arg Lys Lys Ala Asp Asp Pro Gly
        35                  40                  45
Lys Ala Phe Val Leu Val Ala Leu Ser Asn Asp Asn Gly Gln Leu Ala
    50                  55                  60
Glu Ile Ala Ile Asp Val Thr Ser Val Tyr Val Val Gly Tyr Gln Val
65                  70                  75                  80
Arg Asn Arg Ser Tyr Phe Phe Lys Asp Ala Pro Asp Ala Ala Tyr Glu
                85                  90                  95
Gly Leu Phe Lys Asn Thr Ile Lys Thr Arg Leu His Phe Gly Gly Ser
            100                 105                 110
Tyr Pro Ser Leu Glu Gly Glu Lys Ala Tyr Arg Glu Thr Thr Asp Leu
        115                 120                 125
Gly Ile Glu Pro Leu Arg Ile Gly Ile Lys Lys Leu Asp Glu Asn Ala
    130                 135                 140
Ile Asp Asn Tyr Lys Pro Thr Glu Ile Ala Ser Ser Leu Leu Val Val
145                 150                 155                 160
Ile Gln Met Val Ser Glu Ala Ala Arg Phe Thr Phe Ile Glu Asn Gln
                165                 170                 175
Ile Arg Asn Asn Phe Gln Gln Arg Ile Arg Pro Ala Asn Asn Thr Ile
            180                 185                 190
Ser Leu Glu Asn Lys Trp Gly Lys Leu Ser Phe Gln Ile Arg Thr Ser
        195                 200                 205
Gly Ala Asn Gly Met Phe Ser Glu Ala Val Glu Leu Glu Arg Ala Asn
    210                 215                 220
Gly Lys Lys Tyr Tyr Val Thr Ala Val Asp Gln Val Lys Pro Lys Ile
225                 230                 235                 240
Ala Leu Leu Lys Phe Val Asp Lys Asp Pro Lys
                245                 250
</s-2-xi> </s-2> <s-2 seq-no="102"> <s-2-i> <s-2-i-a>251 amino acids</s-2-i-a> <s-2-i-b>amino acid</s-2-i-b> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>protein</s-2-ii-mt> </s-2-ii> <s-2-xi>102
Gly Leu Asp Thr Val Ser Phe Ser Thr Lys Gly Ala Thr Tyr Ile Thr
1               5                   10                  15
Tyr Val Asn Phe Leu Asn Glu Leu Arg Val Lys Leu Lys Pro Glu Gly
            20                  25                  30
Asn Ser His Gly Ile Pro Leu Leu Arg Lys Lys Cys Asp Asp Pro Gly
        35                  40                  45
Lys Cys Phe Val Leu Val Ala Leu Ser Asn Asp Asn Gly Gln Leu Ala
    50                  55                  60
Glu Ile Ala Ile Asp Val Thr Ser Val Tyr Val Val Gly Tyr Gln Val
65                  70                  75                  80
Arg Asn Arg Ser Tyr Phe Phe Lys Asp Ala Pro Asp Ala Ala Tyr Glu
                85                  90                  95
Gly Leu Phe Lys Asn Thr Ile Lys Thr Arg Leu His Phe Gly Gly Ser
            100                 105                 110
Tyr Pro Ser Leu Glu Gly Glu Lys Ala Tyr Arg Glu Thr Thr Asp Leu
        115                 120                 125
Gly Ile Glu Pro Leu Arg Ile Gly Ile Lys Lys Leu Asp Glu Asn Ala
    130                 135                 140
Ile Asp Asn Tyr Lys Pro Thr Glu Ile Ala Ser Ser Leu Leu Val Val
145                 150                 155                 160
Ile Gln Met Val Ser Glu Ala Ala Arg Phe Thr Phe Ile Glu Asn Gln
                165                 170                 175
Ile Arg Asn Asn Phe Gln Gln Arg Ile Arg Pro Ala Asn Asn Thr Ile
            180                 185                 190
Ser Leu Glu Asn Lys Trp Gly Lys Leu Ser Phe Gln Ile Arg Thr Ser
        195                 200                 205
Gly Ala Asn Gly Met Phe Ser Glu Ala Val Glu Leu Glu Arg Ala Asn
    210                 215                 220
Gly Lys Lys Tyr Tyr Val Thr Ala Val Asp Gln Val Lys Pro Lys Ile
225                 230                 235                 240
Ala Leu Leu Lys Phe Val Cys Lys Asp Pro Lys
                245                 250
</s-2-xi> </s-2> <s-2 seq-no="103"> <s-2-i> <s-2-i-a>251 amino acids</s-2-i-a> <s-2-i-b>amino acid</s-2-i-b> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>protein</s-2-ii-mt> </s-2-ii> <s-2-xi>103
Gly Leu Asp Thr Val Ser Phe Ser Thr Lys Gly Ala Thr Tyr Ile Thr
1               5                   10                  15
Tyr Val Asn Phe Leu Asn Glu Leu Arg Val Lys Leu Lys Pro Glu Gly
            20                  25                  30
Asn Ser His Gly Ile Pro Leu Leu Arg Lys Lys Cys Asp Asp Pro Gly
        35                  40                  45
Lys Cys Phe Val Leu Val Ala Leu Ser Asn Asp Asn Gly Gln Leu Ala
    50                  55                  60
Glu Ile Ala Ile Asp Val Thr Ser Val Tyr Val Val Gly Tyr Gln Val
65                  70                  75                  80
Arg Asn Arg Ser Tyr Phe Phe Lys Asp Ala Pro Asp Ala Ala Tyr Glu
                85                  90                  95
Gly Leu Phe Lys Asn Thr Ile Lys Thr Arg Leu His Phe Gly Gly Ser
            100                 105                 110
Tyr Pro Ser Leu Glu Gly Glu Lys Ala Tyr Arg Glu Thr Thr Asp Leu
        115                 120                 125
Gly Ile Glu Pro Leu Arg Ile Gly Ile Lys Lys Leu Asp Glu Asn Ala
    130                 135                 140
Ile Asp Asn Tyr Lys Pro Thr Glu Ile Ala Ser Ser Leu Leu Val Val
145                 150                 155                 160
Ile Gln Met Val Ser Glu Ala Ala Arg Phe Thr Phe Ile Glu Asn Gln
                165                 170                 175
Ile Arg Asn Asn Phe Gln Gln Arg Ile Arg Pro Ala Asn Asn Thr Ile
            180                 185                 190
Ser Leu Glu Asn Lys Trp Gly Lys Leu Ser Phe Gln Ile Arg Thr Ser
        195                 200                 205
Gly Ala Asn Gly Met Phe Ser Glu Ala Val Glu Leu Glu Arg Ala Asn
    210                 215                 220
Gly Lys Lys Tyr Tyr Val Thr Ala Val Asp Gln Val Lys Pro Lys Ile
225                 230                 235                 240
Ala Leu Leu Lys Phe Val Asp Cys Asp Pro Lys
                245                 250
</s-2-xi> </s-2> <s-2 seq-no="104"> <s-2-i> <s-2-i-a>251 amino acids</s-2-i-a> <s-2-i-b>amino acid</s-2-i-b> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>protein</s-2-ii-mt> </s-2-ii> <s-2-xi>104
Gly Leu Asp Thr Val Ser Phe Ser Thr Lys Gly Ala Thr Tyr Ile Thr
1               5                   10                  15
Tyr Val Asn Phe Leu Asn Glu Leu Arg Val Lys Leu Lys Pro Glu Gly
            20                  25                  30
Asn Ser His Gly Ile Pro Leu Leu Arg Lys Lys Cys Asp Asp Pro Gly
        35                  40                  45
Lys Cys Phe Val Leu Val Ala Leu Ser Asn Asp Asn Gly Gln Leu Ala
    50                  55                  60
Glu Ile Ala Ile Asp Val Thr Ser Val Tyr Val Val Gly Tyr Gln Val
65                  70                  75                  80
Arg Asn Arg Ser Tyr Phe Phe Lys Asp Ala Pro Asp Ala Ala Tyr Glu
                85                  90                  95
Gly Leu Phe Lys Asn Thr Ile Lys Thr Arg Leu His Phe Gly Gly Ser
            100                 105                 110
Tyr Pro Ser Leu Glu Gly Glu Lys Ala Tyr Arg Glu Thr Thr Asp Leu
        115                 120                 125
Gly Ile Glu Pro Leu Arg Ile Gly Ile Lys Lys Leu Asp Glu Asn Ala
    130                 135                 140
Ile Asp Asn Tyr Lys Pro Thr Glu Ile Ala Ser Ser Leu Leu Val Val
145                 150                 155                 160
Ile Gln Met Val Ser Glu Ala Ala Arg Phe Thr Phe Ile Glu Asn Gln
                165                 170                 175
Ile Arg Asn Asn Phe Gln Gln Arg Ile Arg Pro Ala Asn Asn Thr Ile
            180                 185                 190
Ser Leu Glu Asn Lys Trp Gly Lys Leu Ser Phe Gln Ile Arg Thr Ser
        195                 200                 205
Gly Ala Asn Gly Met Phe Ser Glu Ala Val Glu Leu Glu Arg Ala Asn
    210                 215                 220
Gly Lys Lys Tyr Tyr Val Thr Ala Val Asp Gln Val Lys Pro Cys Ile
225                 230                 235                 240
Ala Leu Leu Lys Phe Val Asp Lys Asp Pro Lys
                245                 250
</s-2-xi> </s-2> <s-2 seq-no="105"> <s-2-i> <s-2-i-a>251 amino acids</s-2-i-a> <s-2-i-b>amino acid</s-2-i-b> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>protein</s-2-ii-mt> </s-2-ii> <s-2-xi>105
Gly Leu Asp Thr Val Ser Phe Ser Thr Lys Gly Ala Thr Tyr Ile Thr
1               5                   10                  15
Tyr Val Asn Phe Leu Asn Glu Leu Arg Val Lys Leu Lys Pro Glu Gly
            20                  25                  30
Asn Ser His Gly Ile Pro Leu Leu Arg Lys Lys Cys Asp Asp Pro Gly
        35                  40                  45
Lys Cys Phe Val Leu Val Ala Leu Ser Asn Asp Asn Gly Gln Leu Ala
    50                  55                  60
Glu Ile Ala Ile Asp Val Thr Ser Val Tyr Val Val Gly Tyr Gln Val
65                  70                  75                  80
Arg Asn Arg Ser Tyr Phe Phe Lys Asp Ala Pro Asp Ala Ala Tyr Glu
                85                  90                  95
Gly Leu Phe Lys Asn Thr Ile Lys Thr Arg Leu His Phe Gly Gly Ser
            100                 105                 110
Tyr Pro Ser Leu Glu Gly Glu Lys Ala Tyr Arg Glu Thr Thr Asp Leu
        115                 120                 125
Gly Ile Glu Pro Leu Arg Ile Gly Ile Lys Lys Leu Asp Glu Asn Ala
    130                 135                 140
Ile Asp Asn Tyr Lys Pro Thr Glu Ile Ala Ser Ser Leu Leu Val Val
145                 150                 155                 160
Ile Gln Met Val Ser Glu Ala Ala Arg Phe Thr Phe Ile Glu Asn Gln
                165                 170                 175
Ile Arg Asn Asn Phe Gln Gln Arg Ile Arg Pro Ala Asn Asn Thr Ile
            180                 185                 190
Ser Leu Glu Asn Lys Trp Gly Lys Leu Ser Phe Gln Ile Arg Thr Ser
        195                 200                 205
Gly Ala Asn Gly Met Phe Ser Glu Ala Val Glu Leu Glu Arg Ala Asn
    210                 215                 220
Gly Lys Lys Tyr Tyr Val Thr Ala Val Asp Gln Val Lys Pro Lys Ile
225                 230                 235                 240
Ala Leu Leu Cys Phe Val Asp Lys Asp Pro Lys
                245                 250
</s-2-xi> </s-2> <s-2 seq-no="106"> <s-2-i> <s-2-i-a>251 amino acids</s-2-i-a> <s-2-i-b>amino acid</s-2-i-b> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>protein</s-2-ii-mt> </s-2-ii> <s-2-xi>106
Gly Leu Asp Thr Val Ser Phe Ser Thr Cys Gly Ala Thr Tyr Ile Thr
1               5                   10                  15
Tyr Val Asn Phe Leu Asn Glu Leu Arg Val Lys Leu Lys Pro Glu Gly
            20                  25                  30
Asn Ser His Gly Ile Pro Leu Leu Arg Lys Lys Cys Asp Asp Pro Gly
        35                  40                  45
Lys Cys Phe Val Leu Val Ala Leu Ser Asn Asp Asn Gly Gln Leu Ala
    50                  55                  60
Glu Ile Ala Ile Asp Val Thr Ser Val Tyr Val Val Gly Tyr Gln Val
65                  70                  75                  80
Arg Asn Arg Ser Tyr Phe Phe Lys Asp Ala Pro Asp Ala Ala Tyr Glu
                85                  90                  95
Gly Leu Phe Lys Asn Thr Ile Lys Thr Arg Leu His Phe Gly Gly Ser
            100                 105                 110
Tyr Pro Ser Leu Glu Gly Glu Lys Ala Tyr Arg Glu Thr Thr Asp Leu
        115                 120                 125
Gly Ile Glu Pro Leu Arg Ile Gly Ile Lys Lys Leu Asp Glu Asn Ala
    130                 135                 140
Ile Asp Asn Tyr Lys Pro Thr Glu Ile Ala Ser Ser Leu Leu Val Val
145                 150                 155                 160
Ile Gln Met Val Ser Glu Ala Ala Arg Phe Thr Phe Ile Glu Asn Gln
                165                 170                 175
Ile Arg Asn Asn Phe Gln Gln Arg Ile Arg Pro Ala Asn Asn Thr Ile
            180                 185                 190
Ser Leu Glu Asn Lys Trp Gly Lys Leu Ser Phe Gln Ile Arg Thr Ser
        195                 200                 205
Gly Ala Asn Gly Met Phe Ser Glu Ala Val Glu Leu Glu Arg Ala Asn
    210                 215                 220
Gly Lys Lys Tyr Tyr Val Thr Ala Val Asp Gln Val Lys Pro Lys Ile
225                 230                 235                 240
Ala Leu Leu Lys Phe Val Asp Lys Asp Pro Lys
                245                 250
</s-2-xi> </s-2> <s-2 seq-no="107"> <s-2-i> <s-2-i-a>251 amino acids</s-2-i-a> <s-2-i-b>amino acid</s-2-i-b> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>protein</s-2-ii-mt> </s-2-ii> <s-2-xi>107
Gly Leu Asp Thr Val Ser Phe Ser Thr Lys Gly Ala Thr Tyr Ile Thr
1               5                   10                  15
Tyr Val Asn Phe Leu Asn Glu Leu Arg Val Lys Leu Lys Pro Glu Gly
            20                  25                  30
Asn Ser His Gly Ile Pro Leu Leu Arg Lys Lys Cys Asp Asp Pro Gly
        35                  40                  45
Lys Cys Phe Val Leu Val Ala Leu Ser Asn Asp Cys Gly Gln Leu Ala
    50                  55                  60
Glu Ile Ala Ile Asp Val Thr Ser Val Tyr Val Val Gly Tyr Gln Val
65                  70                  75                  80
Arg Asn Arg Ser Tyr Phe Phe Lys Asp Ala Pro Asp Ala Ala Tyr Glu
                85                  90                  95
Gly Leu Phe Lys Asn Thr Ile Lys Thr Arg Leu His Phe Gly Gly Ser
            100                 105                 110
Tyr Pro Ser Leu Glu Gly Glu Lys Ala Tyr Arg Glu Thr Thr Asp Leu
        115                 120                 125
Gly Ile Glu Pro Leu Arg Ile Gly Ile Lys Lys Leu Asp Glu Asn Ala
    130                 135                 140
Ile Asp Asn Tyr Lys Pro Thr Glu Ile Ala Ser Ser Leu Leu Val Val
145                 150                 155                 160
Ile Gln Met Val Ser Glu Ala Ala Arg Phe Thr Phe Ile Glu Asn Gln
                165                 170                 175
Ile Arg Asn Asn Phe Gln Gln Arg Ile Arg Pro Ala Asn Asn Thr Ile
            180                 185                 190
Ser Leu Glu Asn Lys Trp Gly Lys Leu Ser Phe Gln Ile Arg Thr Ser
        195                 200                 205
Gly Ala Asn Gly Met Phe Ser Glu Ala Val Glu Leu Glu Arg Ala Asn
    210                 215                 220
Gly Lys Lys Tyr Tyr Val Thr Ala Val Asp Gln Val Lys Pro Lys Ile
225                 230                 235                 240
Ala Leu Leu Lys Phe Val Asp Lys Asp Pro Lys
                245                 250
</s-2-xi> </s-2> <s-2 seq-no="108"> <s-2-i> <s-2-i-a>251 amino acids</s-2-i-a> <s-2-i-b>amino acid</s-2-i-b> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>protein</s-2-ii-mt> </s-2-ii> <s-2-xi>108
Gly Leu Asp Thr Val Ser Phe Ser Thr Lys Gly Ala Thr Tyr Ile Thr
1               5                   10                  15
Tyr Val Asn Phe Leu Asn Glu Leu Arg Val Lys Leu Lys Pro Glu Gly
            20                  25                  30
Asn Ser His Gly Ile Pro Leu Leu Arg Lys Lys Cys Asp Asp Pro Gly
        35                  40                  45
Lys Cys Phe Val Leu Val Ala Leu Ser Asn Asp Asn Gly Gln Leu Ala
    50                  55                  60
Glu Ile Ala Ile Asp Val Thr Ser Val Tyr Val Val Gly Tyr Gln Val
65                  70                  75                  80
Arg Asn Arg Ser Tyr Phe Phe Lys Asp Ala Pro Asp Ala Ala Tyr Glu
                85                  90                  95
Gly Leu Phe Lys Asn Thr Cys Lys Thr Arg Leu His Phe Gly Gly Ser
            100                 105                 110
Tyr Pro Ser Leu Glu Gly Glu Lys Ala Tyr Arg Glu Thr Thr Asp Leu
        115                 120                 125
Gly Ile Glu Pro Leu Arg Ile Gly Ile Lys Lys Leu Asp Glu Asn Ala
    130                 135                 140
Ile Asp Asn Tyr Lys Pro Thr Glu Ile Ala Ser Ser Leu Leu Val Val
145                 150                 155                 160
Ile Gln Met Val Ser Glu Ala Ala Arg Phe Thr Phe Ile Glu Asn Gln
                165                 170                 175
Ile Arg Asn Asn Phe Gln Gln Arg Ile Arg Pro Ala Asn Asn Thr Ile
            180                 185                 190
Ser Leu Glu Asn Lys Trp Gly Lys Leu Ser Phe Gln Ile Arg Thr Ser
        195                 200                 205
Gly Ala Asn Gly Met Phe Ser Glu Ala Val Glu Leu Glu Arg Ala Asn
    210                 215                 220
Gly Lys Lys Tyr Tyr Val Thr Ala Val Asp Gln Val Lys Pro Lys Ile
225                 230                 235                 240
Ala Leu Leu Lys Phe Val Asp Lys Asp Pro Lys
                245                 250
</s-2-xi> </s-2> <s-2 seq-no="109"> <s-2-i> <s-2-i-a>251 amino acids</s-2-i-a> <s-2-i-b>amino acid</s-2-i-b> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>protein</s-2-ii-mt> </s-2-ii> <s-2-xi>109
Gly Leu Asp Thr Val Ser Phe Ser Thr Lys Gly Ala Thr Tyr Ile Thr
1               5                   10                  15
Tyr Val Asn Phe Leu Asn Glu Leu Arg Val Lys Leu Lys Pro Glu Gly
            20                  25                  30
Asn Ser His Gly Ile Pro Leu Leu Arg Lys Lys Cys Asp Asp Pro Gly
        35                  40                  45
Lys Cys Phe Val Leu Val Ala Leu Ser Asn Asp Asn Gly Gln Leu Ala
    50                  55                  60
Glu Ile Ala Ile Asp Val Thr Ser Val Tyr Val Val Gly Tyr Gln Val
65                  70                  75                  80
Arg Asn Arg Ser Tyr Phe Phe Lys Asp Ala Pro Asp Ala Ala Tyr Glu
                85                  90                  95
Gly Leu Phe Lys Asn Thr Ile Lys Thr Arg Leu His Phe Gly Gly Ser
            100                 105                 110
Tyr Pro Ser Leu Glu Gly Glu Lys Ala Tyr Arg Glu Thr Thr Asp Leu
        115                 120                 125
Gly Ile Glu Pro Leu Arg Ile Gly Ile Lys Lys Leu Asp Glu Asn Ala
    130                 135                 140
Ile Asp Asn Tyr Lys Pro Thr Glu Ile Ala Ser Ser Leu Leu Val Val
145                 150                 155                 160
Ile Gln Met Val Ser Glu Ala Ala Arg Phe Thr Phe Ile Glu Asn Gln
                165                 170                 175
Ile Arg Asn Asn Phe Gln Gln Cys Ile Arg Pro Ala Asn Asn Thr Ile
            180                 185                 190
Ser Leu Glu Asn Lys Trp Gly Lys Leu Ser Phe Gln Ile Arg Thr Ser
        195                 200                 205
Gly Ala Asn Gly Met Phe Ser Glu Ala Val Glu Leu Glu Arg Ala Asn
    210                 215                 220
Gly Lys Lys Tyr Tyr Val Thr Ala Val Asp Gln Val Lys Pro Lys Ile
225                 230                 235                 240
Ala Leu Leu Lys Phe Val Asp Lys Asp Pro Lys
                245                 250
</s-2-xi> </s-2> <s-2 seq-no="110"> <s-2-i> <s-2-i-a>251 amino acids</s-2-i-a> <s-2-i-b>amino acid</s-2-i-b> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>protein</s-2-ii-mt> </s-2-ii> <s-2-xi>110
Gly Leu Asp Thr Val Ser Phe Ser Thr Cys Gly Ala Thr Tyr Ile Thr
1               5                   10                  15
Tyr Val Asn Phe Leu Asn Glu Leu Arg Val Lys Leu Lys Pro Glu Gly
            20                  25                  30
Asn Ser His Gly Ile Pro Leu Leu Arg Lys Lys Ala Asp Asp Pro Gly
        35                  40                  45
Lys Ala Phe Val Leu Val Ala Leu Ser Asn Asp Asn Gly Gln Leu Ala
    50                  55                  60
Glu Ile Ala Ile Asp Val Thr Ser Val Tyr Val Val Gly Tyr Gln Val
65                  70                  75                  80
Arg Asn Arg Ser Tyr Phe Phe Lys Asp Ala Pro Asp Ala Ala Tyr Glu
                85                  90                  95
Gly Leu Phe Lys Asn Thr Ile Lys Thr Arg Leu His Phe Gly Gly Ser
            100                 105                 110
Tyr Pro Ser Leu Glu Gly Glu Lys Ala Tyr Arg Glu Thr Thr Asp Leu
        115                 120                 125
Gly Ile Glu Pro Leu Arg Ile Gly Ile Lys Lys Leu Asp Glu Asn Ala
    130                 135                 140
Ile Asp Asn Tyr Lys Pro Thr Glu Ile Ala Ser Ser Leu Leu Val Val
145                 150                 155                 160
Ile Gln Met Val Ser Glu Ala Ala Arg Phe Thr Phe Ile Glu Asn Gln
                165                 170                 175
Ile Arg Asn Asn Phe Gln Gln Arg Ile Arg Pro Ala Asn Asn Thr Ile
            180                 185                 190
Ser Leu Glu Asn Lys Trp Gly Lys Leu Ser Phe Gln Ile Arg Thr Ser
        195                 200                 205
Gly Ala Asn Gly Met Phe Ser Glu Ala Val Glu Leu Glu Arg Ala Asn
    210                 215                 220
Gly Lys Lys Tyr Tyr Val Thr Ala Val Asp Gln Val Lys Pro Lys Ile
225                 230                 235                 240
Ala Leu Leu Lys Phe Val Asp Lys Asp Pro Lys
                245                 250
</s-2-xi> </s-2> <s-2 seq-no="111"> <s-2-i> <s-2-i-a>251 amino acids</s-2-i-a> <s-2-i-b>amino acid</s-2-i-b> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>protein</s-2-ii-mt> </s-2-ii> <s-2-xi>111
Gly Leu Asp Thr Val Ser Phe Ser Thr Cys Gly Ala Thr Tyr Ile Thr
1               5                   10                  15
Tyr Val Asn Phe Leu Asn Glu Leu Arg Val Lys Leu Lys Pro Glu Gly
            20                  25                  30
Asn Ser His Gly Ile Pro Leu Leu Arg Lys Lys Ala Asp Asp Pro Gly
        35                  40                  45
Lys Ala Phe Val Leu Val Ala Leu Ser Asn Asp Asn Gly Gln Leu Ala
    50                  55                  60
Glu Ile Ala Ile Asp Val Thr Ser Val Tyr Val Val Gly Tyr Gln Val
65                  70                  75                  80
Arg Asn Arg Ser Tyr Phe Phe Lys Asp Ala Pro Asp Ala Ala Tyr Glu
                85                  90                  95
Gly Leu Phe Lys Asn Thr Ile Lys Thr Arg Leu His Phe Gly Gly Ser
            100                 105                 110
Tyr Pro Ser Leu Glu Gly Glu Lys Ala Tyr Arg Glu Thr Thr Asp Leu
        115                 120                 125
Gly Ile Glu Pro Leu Arg Ile Gly Ile Lys Lys Leu Asp Glu Asn Ala
    130                 135                 140
Ile Asp Asn Tyr Lys Pro Thr Glu Ile Ala Ser Ser Leu Leu Val Val
145                 150                 155                 160
Ile Gln Met Val Ser Glu Ala Ala Arg Phe Thr Phe Ile Glu Asn Gln
                165                 170                 175
Ile Arg Asn Asn Phe Gln Gln Arg Ile Arg Pro Ala Asn Asn Thr Ile
            180                 185                 190
Ser Leu Glu Asn Lys Trp Gly Lys Leu Ser Phe Gln Ile Arg Thr Ser
        195                 200                 205
Gly Ala Asn Gly Met Phe Ser Glu Ala Val Glu Leu Glu Arg Ala Asn
    210                 215                 220
Gly Lys Lys Tyr Tyr Val Thr Ala Val Asp Gln Val Lys Pro Lys Ile
225                 230                 235                 240
Ala Leu Leu Lys Phe Val Cys Lys Asp Pro Lys
                245                 250
</s-2-xi> </s-2> <s-2 seq-no="112"> <s-2-i> <s-2-i-a>29 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>112
TGATGCGGCC GACATCTCAA GCTTGGTGC                                       29
</s-2-xi> </s-2> <s-2 seq-no="113"> <s-2-i> <s-2-i-a>29 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>113
TGATGCGGCC GACATCTCAA GCTTGGTGC                                       29
</s-2-xi> </s-2> <s-2 seq-no="114"> <s-2-i> <s-2-i-a>38 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>114
TCTAGGTCAC CGTCTCCTCA CCATCTGGAC AGGCTGGA                             38
</s-2-xi> </s-2> <s-2 seq-no="115"> <s-2-i> <s-2-i-a>37 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>115
TTCGAAGCTT GAGATGAAAC CATCTGGACA GGCTGGA                              37
</s-2-xi> </s-2> <s-2 seq-no="116"> <s-2-i> <s-2-i-a>27 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>116
AGTCGTCGAC ACGATGGACA TGAGGAC                                         27
</s-2-xi> </s-2> <s-2 seq-no="117"> <s-2-i> <s-2-i-a>98 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>117
AGTCGTCGAC ACGATGGACA TGAGGACCCC TGCTCAGTTT CTTGGCATCC TCCTACTCTG     60
GTTTCCAGGT ATCAAATGTG ACATCCAGAT GACTCAGT                             98
</s-2-xi> </s-2> <s-2 seq-no="118"> <s-2-i> <s-2-i-a>79 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>118
TCACTTGCCG GGCGAATCAG GACATTAATA GCTATTTAAG CTGGTTCCAG CAGAAACCAG     60
GGAAAGCTCC TAAGACCCT                                                  79
</s-2-xi> </s-2> <s-2 seq-no="119"> <s-2-i> <s-2-i-a>80 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>119
TGACTCGCCC GGCAAGTGAT AGTGACTCTG TCTCCTACAG ATGCAGACAG GGAAGATGGA     60
GACTGAGTCA TCTGGATGTC                                                 80
</s-2-xi> </s-2> <s-2 seq-no="120"> <s-2-i> <s-2-i-a>79 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>120
GATCCACTGC CACTGAACCT TGATGGGACC CCAGATTCCA ATCTGTTTGC ACGATAGATC     60
AGGGTCTTAG GAGCTTTCC                                                  79
</s-2-xi> </s-2> <s-2 seq-no="121"> <s-2-i> <s-2-i-a>82 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>121
GGTTCAGTGG CAGTGGATCT GGGACAGATT ATACTCTCAC CATCAGCAGC CTGCAATATG     60
AAGATTTTGG AATTTATTAT TG                                              82
</s-2-xi> </s-2> <s-2 seq-no="122"> <s-2-i> <s-2-i-a>82 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>122
GTTTGATTTC AAGCTTGGTG CCTCCACCGA ACGTCCACGG AGACTCATCA TACTGTTGAC     60
AATAATAAAT TCCAAAATCT TC                                              82
</s-2-xi> </s-2> <s-2 seq-no="123"> <s-2-i> <s-2-i-a>107 amino acids</s-2-i-a> <s-2-i-b>amino acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>protein</s-2-ii-mt> </s-2-ii> <s-2-xi>123
Asp Ile Lys Met Thr Gln Ser Pro Ser Ser Met Tyr Ala Ser Leu Gly
1               5                   10                  15
Glu Arg Val Thr Ile Thr Cys Lys Ala Ser Gln Asp Ile Asn Ser Tyr
            20                  25                  30
Leu Ser Trp Phe His His Lys Pro Gly Lys Ser Pro Lys Thr Leu Ile
        35                  40                  45
Tyr Arg Ala Asn Arg Leu Val Asp Gly Val Pro Ser Arg Phe Ser Gly
    50                  55                  60
Ser Gly Ser Gly Gln Asp Tyr Ser Leu Thr Ile Ser Ser Leu Asp Tyr
65                  70                  75                  80
Glu Asp Met Gly Ile Tyr Tyr Cys Gln Gln Tyr Asp Glu Ser Pro Trp
                85                  90                  95
Thr Phe Gly Gly Gly Thr Lys Leu Glu Ile Lys
            100                 105
</s-2-xi> </s-2> <s-2 seq-no="124"> <s-2-i> <s-2-i-a>118 amino acids</s-2-i-a> <s-2-i-b>amino acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>protein</s-2-ii-mt> </s-2-ii> <s-2-xi>124
Gln Ile Gln Leu Val Gln Ser Gly Pro Glu Leu Lys Lys Pro Gly Glu
1               5                   10                  15
Thr Val Lys Ile Ser Cys Lys Ala Ser Gly Tyr Thr Phe Thr Asn Tyr
            20                  25                  30
Gly Met Asn Trp Val Lys Gln Ala Pro Gly Lys Gly Leu Arg Trp Met
        35                  40                  45
Gly Trp Ile Asn Thr His Thr Gly Glu Pro Thr Tyr Ala Asp Asp Phe
    50                  55                  60
Lys Gly Arg Phe Ala Phe Ser Leu Glu Thr Ser Ala Ser Thr Ala Tyr
65                  70                  75                  80
Leu Gln Ile Asn Asn Leu Lys Asn Glu Asp Thr Ala Thr Tyr Phe Cys
                85                  90                  95
Thr Arg Arg Gly Tyr Asp Trp Tyr Phe Asp Val Trp Gly Ala Gly Thr
            100                 105                 110
Thr Val Thr Val Ser Ser
        115
</s-2-xi> </s-2> <s-2 seq-no="125"> <s-2-i> <s-2-i-a>107 amino acids</s-2-i-a> <s-2-i-b>amino acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>protein</s-2-ii-mt> </s-2-ii> <s-2-xi>125
Asp Ile Gln Met Thr Gln Ser Pro Ser Ser Leu Ser Ala Ser Val Gly
1               5                   10                  15
Asp Arg Val Thr Ile Thr Cys Arg Ala Ser Gln Asp Ile Asn Ser Tyr
            20                  25                  30
Leu Ser Trp Phe Gln Gln Lys Pro Gly Lys Ala Pro Lys Thr Leu Ile
        35                  40                  45
Tyr Arg Ala Asn Arg Leu Glu Ser Gly Val Pro Ser Arg Phe Ser Gly
    50                  55                  60
Ser Gly Ser Gly Thr Asp Tyr Thr Leu Thr Ile Ser Ser Leu Gln Tyr
65                  70                  75                  80
Glu Asp Phe Gly Ile Tyr Tyr Cys Gln Gln Tyr Asp Glu Ser Pro Trp
                85                  90                  95
Thr Phe Gly Gly Gly Thr Lys Leu Glu Ile Lys
            100                 105
</s-2-xi> </s-2> <s-2 seq-no="126"> <s-2-i> <s-2-i-a>118 amino acids</s-2-i-a> <s-2-i-b>amino acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>protein</s-2-ii-mt> </s-2-ii> <s-2-xi>126
Glu Ile Gln Leu Val Gln Ser Gly Gly Gly Leu Val Lys Pro Gly Gly
1               5                   10                  15
Ser Val Arg Ile Ser Cys Ala Ala Ser Gly Tyr Thr Phe Thr Asn Tyr
            20                  25                  30
Gly Met Asn Trp Val Arg Gln Ala Pro Gly Lys Gly Leu Glu Trp Met
        35                  40                  45
Gly Trp Ile Asn Thr His Thr Gly Glu Pro Thr Tyr Ala Asp Ser Phe
    50                  55                  60
Lys Gly Arg Phe Thr Phe Ser Leu Asp Asp Ser Lys Asn Thr Ala Tyr
65                  70                  75                  80
Leu Gln Ile Asn Ser Leu Arg Ala Glu Asp Thr Ala Val Tyr Phe Cys
                85                  90                  95
Thr Arg Arg Gly Tyr Asp Trp Tyr Phe Asp Val Trp Gly Gln Gly Thr
            100                 105                 110
Thr Val Thr Val Ser Ser
        115
</s-2-xi> </s-2> <s-2 seq-no="127"> <s-2-i> <s-2-i-a>280 amino acids</s-2-i-a> <s-2-i-b>amino acid</s-2-i-b> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>protein</s-2-ii-mt> </s-2-ii> <s-2-xi>127
Ala Ala Lys Met Ala Lys Asn Val Asp Lys Pro Leu Phe Thr Ala Thr
1               5                   10                  15
Phe Asn Val Gln Ala Ser Ser Ala Asp Tyr Ala Thr Phe Ile Ala Gly
            20                  25                  30
Ile Arg Asn Lys Leu Arg Asn Pro Ala His Phe Ser His Asn Arg Pro
        35                  40                  45
Val Leu Pro Pro Val Glu Pro Asn Val Pro Pro Ser Arg Trp Phe His
    50                  55                  60
Val Val Leu Lys Ala Ser Pro Thr Ser Ala Gly Leu Thr Leu Ala Ile
65                  70                  75                  80
Arg Ala Asp Asn Ile Tyr Leu Glu Gly Phe Lys Ser Ser Asp Gly Thr
                85                  90                  95
Trp Trp Glu Leu Thr Pro Gly Leu Ile Pro Gly Ala Thr Tyr Val Gly
            100                 105                 110
Phe Gly Gly Thr Tyr Arg Asp Leu Leu Gly Asp Thr Asp Lys Leu Thr
        115                 120                 125
Asn Val Ala Leu Gly Arg Gln Gln Leu Ala Asp Ala Val Thr Ala Leu
    130                 135                 140
His Gly Arg Thr Lys Ala Asp Lys Ala Ser Gly Pro Lys Gln Gln Gln
145                 150                 155                 160
Ala Arg Glu Ala Val Thr Thr Leu Val Leu Met Val Asn Glu Ala Thr
                165                 170                 175
Arg Phe Gln Thr Val Ser Gly Phe Val Ala Gly Leu Leu His Pro Lys
            180                 185                 190
Ala Val Glu Lys Lys Ser Gly Lys Ile Gly Asn Glu Met Lys Ala Gln
        195                 200                 205
Val Asn Gly Trp Gln Asp Leu Ser Ala Ala Leu Leu Lys Thr Asp Val
    210                 215                 220
Lys Pro Pro Pro Gly Lys Ser Pro Ala Lys Phe Ala Pro Ile Glu Lys
225                 230                 235                 240
Met Gly Val Arg Thr Ala Glu Gln Ala Ala Asn Thr Leu Gly Ile Leu
                245                 250                 255
Leu Phe Val Glu Val Pro Gly Gly Leu Thr Val Ala Lys Ala Leu Glu
            260                 265                 270
Leu Phe His Ala Cys Gly Gly Lys
        275                 280
</s-2-xi> </s-2> <s-2 seq-no="128"> <s-2-i> <s-2-i-a>280 amino acids</s-2-i-a> <s-2-i-b>amino acid</s-2-i-b> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>protein</s-2-ii-mt> </s-2-ii> <s-2-xi>128
Ala Ala Lys Met Ala Lys Asn Val Asp Lys Pro Leu Phe Thr Ala Thr
1               5                   10                  15
Phe Asn Val Gln Ala Ser Ser Ala Asp Tyr Ala Thr Phe Ile Ala Gly
            20                  25                  30
Ile Arg Asn Lys Leu Arg Asn Pro Ala His Phe Ser His Asn Arg Pro
        35                  40                  45
Val Leu Pro Pro Val Glu Pro Asn Val Pro Pro Ser Arg Trp Phe His
    50                  55                  60
Val Val Leu Lys Ala Ser Pro Thr Ser Ala Gly Leu Thr Leu Ala Ile
65                  70                  75                  80
Arg Ala Asp Asn Ile Tyr Leu Glu Gly Phe Lys Ser Ser Asp Gly Thr
                85                  90                  95
Trp Trp Glu Leu Thr Pro Gly Leu Ile Pro Gly Ala Thr Tyr Val Gly
            100                 105                 110
Phe Gly Gly Thr Tyr Arg Asp Leu Leu Gly Asp Thr Asp Lys Leu Thr
        115                 120                 125
Asn Val Ala Leu Gly Arg Gln Gln Leu Ala Asp Ala Val Thr Ala Leu
    130                 135                 140
His Gly Arg Thr Lys Ala Asp Lys Ala Ser Gly Pro Lys Gln Gln Gln
145                 150                 155                 160
Ala Arg Glu Ala Val Thr Thr Leu Val Leu Met Val Asn Glu Ala Thr
                165                 170                 175
Arg Phe Gln Thr Val Ser Gly Phe Val Ala Gly Leu Leu His Pro Lys
            180                 185                 190
Ala Val Glu Lys Lys Ser Gly Lys Ile Gly Asn Glu Met Lys Ala Gln
        195                 200                 205
Val Asn Gly Trp Gln Asp Leu Ser Ala Ala Leu Leu Lys Thr Asp Val
    210                 215                 220
Lys Pro Pro Pro Gly Lys Ser Pro Ala Lys Phe Ala Pro Ile Glu Lys
225                 230                 235                 240
Met Gly Val Arg Thr Ala Glu Gln Ala Ala Asn Thr Leu Gly Ile Leu
                245                 250                 255
Leu Phe Val Glu Val Pro Gly Gly Leu Thr Val Ala Lys Cys Leu Glu
            260                 265                 270
Leu Phe His Ala Ser Gly Gly Lys
        275                 280
</s-2-xi> </s-2> <s-2 seq-no="129"> <s-2-i> <s-2-i-a>280 amino acids</s-2-i-a> <s-2-i-b>amino acid</s-2-i-b> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>protein</s-2-ii-mt> </s-2-ii> <s-2-xi>129
Ala Ala Lys Met Ala Lys Asn Val Asp Lys Pro Leu Phe Thr Ala Thr
1               5                   10                  15
Phe Asn Val Gln Ala Ser Ser Ala Asp Tyr Ala Thr Phe Ile Ala Gly
            20                  25                  30
Ile Arg Asn Lys Leu Arg Asn Pro Ala His Phe Ser His Asn Arg Pro
        35                  40                  45
Val Leu Pro Pro Val Glu Pro Asn Val Pro Pro Ser Arg Trp Phe His
    50                  55                  60
Val Val Leu Lys Ala Ser Pro Thr Ser Ala Gly Leu Thr Leu Ala Ile
65                  70                  75                  80
Arg Ala Asp Asn Ile Tyr Leu Glu Gly Phe Lys Ser Ser Asp Gly Thr
                85                  90                  95
Trp Trp Glu Leu Thr Pro Gly Leu Ile Pro Gly Ala Thr Tyr Val Gly
            100                 105                 110
Phe Gly Gly Thr Tyr Arg Asp Leu Leu Gly Asp Thr Asp Lys Leu Thr
        115                 120                 125
Asn Val Ala Leu Gly Arg Gln Gln Leu Ala Asp Ala Val Thr Ala Leu
    130                 135                 140
His Gly Arg Thr Lys Ala Asp Lys Ala Ser Gly Pro Lys Gln Gln Gln
145                 150                 155                 160
Ala Arg Glu Ala Val Thr Thr Leu Val Leu Met Val Asn Glu Ala Thr
                165                 170                 175
Arg Phe Gln Thr Val Ser Gly Phe Val Ala Gly Leu Leu His Pro Lys
            180                 185                 190
Ala Val Glu Lys Lys Ser Gly Lys Ile Gly Asn Glu Met Lys Ala Gln
        195                 200                 205
Val Asn Gly Trp Gln Asp Leu Ser Ala Ala Leu Leu Lys Thr Asp Val
    210                 215                 220
Lys Pro Pro Pro Gly Lys Ser Pro Ala Lys Phe Ala Pro Ile Glu Lys
225                 230                 235                 240
Met Gly Val Arg Thr Ala Glu Gln Ala Ala Asn Thr Leu Gly Ile Cys
                245                 250                 255
Leu Phe Val Glu Val Pro Gly Gly Leu Thr Val Ala Lys Ala Leu Glu
            260                 265                 270
Leu Phe His Ala Ser Gly Gly Lys
        275                 280
</s-2-xi> </s-2> <s-2 seq-no="130"> <s-2-i> <s-2-i-a>7 amino acids</s-2-i-a> <s-2-i-b>amino acid</s-2-i-b> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>protein</s-2-ii-mt> </s-2-ii> <s-2-xi>130
Ser Cys Asp Lys Thr His Thr
1               5
</s-2-xi> </s-2> <s-2 seq-no="131"> <s-2-i> <s-2-i-a>85 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>131
TGTCGACATC ATGGCTTGGG TGTGGACCTT GCTATTCCTG ATGGCAGCTG CCCAAAGTGC     60
CCAAGCAGAG ATCCAGTTGG TGCAG                                           85
</s-2-xi> </s-2> <s-2 seq-no="132"> <s-2-i> <s-2-i-a>86 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>132
AAGGTATACC CAGAAGCTGC GCAGGAGATT CTGACGGACC CTCCAGGCTT CACCAGGCCT     60
CCTCCAGACT GCACCAACTG GATCTC                                          86
</s-2-xi> </s-2> <s-2 seq-no="133"> <s-2-i> <s-2-i-a>84 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>133
GCAGCTTCTG GGTATACCTT CACAAACTAT GGAATGAACT GGGTGCGCCA GGCTCCAGGA     60
AAGAATTTAG AGTGGATGGG CTGG                                            84
</s-2-xi> </s-2> <s-2 seq-no="134"> <s-2-i> <s-2-i-a>85 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>134
AAAGAGAAGG TAAACCGTCC CTTGAAAGAA TCAGCATATG TTGGCTCTCC AGTGTGGGTG     60
TTTATCCAGC CCATCCACTC TAAAC                                           85
</s-2-xi> </s-2> <s-2 seq-no="135"> <s-2-i> <s-2-i-a>87 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>135
GACGGTTTAC CTTCTCTTTG GACGATTCTA AGAACACTGC CTATTTACAG ATCAACAGCC     60
TCAGAGCCGA GGACACGGCT GTGTATT                                         87
</s-2-xi> </s-2> <s-2 seq-no="136"> <s-2-i> <s-2-i-a>92 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>136
GAGGAGACGG TGACCGTGGT CCCTTGGCCC CAGACATCGA AGTACCAGTC GTAACCCCGT     60
CTTGTACAGA AATACACAGC CGTGTCCTCG GC                                   92
</s-2-xi> </s-2> <s-2 seq-no="137"> <s-2-i> <s-2-i-a>84 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>137
GCAGCTTCTG GGTATACCTT CACAAACTAT GGAATGAACT GGGTGAAGCA GGCTCCAGGA     60
AAGGGTTTAA GGTGGATGGG CTGG                                            84
</s-2-xi> </s-2> <s-2 seq-no="138"> <s-2-i> <s-2-i-a>85 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>138
AAAGAGAAGG TAAACCGTCC CTTGAAGTCA TCAGCATATG TTGGCTCTCC AGTGTGGGTG     60
TTTATCCAGC CCATCCACCT TAAAC                                           85
</s-2-xi> </s-2> <s-2 seq-no="139"> <s-2-i> <s-2-i-a>84 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>139
GACGGTTTAC CTTCTCTTTG GACACGTCTA AGTGCACTGC CTATTTACAG ATCAACAGCC     60
TCAGAGCCGA GGACACGGCT ACAT                                            84
</s-2-xi> </s-2> <s-2 seq-no="140"> <s-2-i> <s-2-i-a>91 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>140
AGGAGACGGT GACCGTGGTC CCTTGGCCCC AGACATCGAA GTACCAGTCG TAACCCCGTC     60
TTGTACAGAA ATATGTAGCC GTGTCCTCGG C                                    91
</s-2-xi> </s-2> <s-2 seq-no="141"> <s-2-i> <s-2-i-a>8 amino acids</s-2-i-a> <s-2-i-b>amino acid</s-2-i-b> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>protein</s-2-ii-mt> </s-2-ii> <s-2-xi>141
Lys Pro Ala Lys Phe Phe Arg Leu
1               5
</s-2-xi> </s-2> <s-2 seq-no="142"> <s-2-i> <s-2-i-a>8 amino acids</s-2-i-a> <s-2-i-b>amino acid</s-2-i-b> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>protein</s-2-ii-mt> </s-2-ii> <s-2-xi>142
Lys Pro Ala Lys Phe Leu Arg Leu
1               5
</s-2-xi> </s-2> <s-2 seq-no="143"> <s-2-i> <s-2-i-a>34 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>143
GGCCGCAAAG CCGGCTAAGT TCTTMCGTCT GAGT                                 34
</s-2-xi> </s-2> <s-2 seq-no="144"> <s-2-i> <s-2-i-a>30 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>144
ACTCAGACGK AAGAACTTAG CCGGCTTTGC                                     30
</s-2-xi> </s-2> <s-2 seq-no="145"> <s-2-i> <s-2-i-a>85 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>145
AAGGTATACC CAGAAGCTGC GCAGGAGATT CTGACGGACC CTCCAGGCTT CTTCAGGCCA     60
GGTCCAGACT GCACCAACTG GATCT                                           85
</s-2-xi> </s-2> <s-2 seq-no="146"> <s-2-i> <s-2-i-a>26 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>146
ACTAGTGTCG ACATCATGGC TTGGGT                                          26
</s-2-xi> </s-2> <s-2 seq-no="147"> <s-2-i> <s-2-i-a>240 amino acids</s-2-i-a> <s-2-i-b>amino acid</s-2-i-b> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>protein</s-2-ii-mt> </s-2-ii> <s-2-xi>147
Asp Ile Gln Met Thr Gln Ser Pro Ser Ser Leu Ser Ala Ser Val Gly
  1               5                  10                  15
Asp Arg Val Thr Ile Thr Cys Arg Ala Ser Gln Asp Ile Asn Ser Tyr
             20                  25                  30
Leu Ser Trp Phe Gln Gln Lys Pro Gly Lys Ala Pro Lys Thr Leu Ile
         35                  40                  45
Tyr Arg Ala Asn Arg Leu Glu Ser Gly Val Pro Ser Arg Phe Ser Gly
     50                  55                  60
Ser Gly Ser Gly Thr Asp Tyr Thr Leu Thr Ile Ser Ser Leu Gln Tyr
 65                  70                  75                  80
Glu Asp Phe Gly Ile Tyr Tyr Cys Gln Gln Tyr Asp Glu Ser Pro Trp
                 85                  90                  95
Thr Phe Gly Gly Gly Thr Lys Leu Glu Met Lys Gly Gly Gly Gly Ser
            100                 105                 110
Gly Gly Gly Gly Ser Gly Gly Gly Gly Ser Glu Ile Gln Leu Val Gln
        115                 120                 125
Ser Gly Gly Gly Leu Val Lys Pro Gly Gly Ser Val Arg Ile Ser Cys
    130                 135                 140
Ala Ala Ser Gly Tyr Thr Phe Thr Asn Tyr Gly Met Asn Trp Val Arg
145                 150                 155                 160
Gln Ala Pro Gly Lys Gly Leu Glu Trp Met Gly Trp Ile Asn Thr His
                165                 170                 175
Thr Gly Glu Pro Thr Tyr Ala Asp Ser Phe Lys Gly Arg Phe Thr Phe
            180                 185                 190
Ser Leu Asp Asp Ser Lys Asn Thr Ala Tyr Leu Gln Ile Asn Ser Leu
        195                 200                 205
Arg Ala Glu Asp Thr Ala Val Tyr Phe Cys Thr Arg Arg Gly Tyr Asp
    210                 215                 220
Trp Tyr Phe Asp Val Trp Gly Gln Gly Thr Thr Val Thr Val Ser Ser
225                 230                 235                 240
</s-2-xi> </s-2> <s-2 seq-no="148"> <s-2-i> <s-2-i-a>240 amino acids</s-2-i-a> <s-2-i-b>amino acid</s-2-i-b> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>protein</s-2-ii-mt> </s-2-ii> <s-2-xi>148
Glu Ile Gln Leu Val Gln Ser Gly Gly Gly Leu Val Lys Pro Gly Gly
  1               5                  10                  15
Ser Val Arg Ile Ser Cys Ala Ala Ser Gly Tyr Thr Phe Thr Asn Tyr
             20                  25                  30
Gly Met Asn Trp Val Arg Gln Ala Pro Gly Lys Gly Leu Glu Trp Met
         35                  40                  45
Gly Trp Ile Asn Thr His Thr Gly Glu Pro Thr Tyr Ala Asp Ser Phe
     50                  55                  60
Lys Gly Arg Phe Thr Phe Ser Leu Asp Asp Ser Lys Asn Thr Ala Tyr
 65                  70                  75                  80
Leu Gln Ile Asn Ser Leu Arg Ala Glu Asp Thr Ala Val Tyr Phe Cys
                 85                  90                  95
Thr Arg Arg Gly Tyr Asp Trp Tyr Phe Asp Val Trp Gly Gln Gly Thr
            100                 105                 110
Thr Val Thr Val Ser Ser Gly Gly Gly Gly Ser Gly Gly Gly Gly Ser
        115                 120                 125
Gly Gly Gly Gly Ser Asp Ile Gln Met Thr Gln Ser Pro Ser Ser Leu
    130                 135                 140
Ser Ala Ser Val Gly Asp Arg Val Thr Ile Thr Cys Arg Ala Ser Gln
145                 150                 155                 160
Asp Ile Asn Ser Tyr Leu Ser Trp Phe Gln Gln Lys Pro Gly Lys Ala
                165                 170                 175
Pro Lys Thr Leu Ile Tyr Arg Ala Asn Arg Leu Glu Ser Gly Val Pro
            180                 185                 190
Ser Arg Phe Ser Gly Ser Gly Ser Gly Thr Asp Tyr Thr Leu Thr Ile
        195                 200                 205
Ser Ser Leu Gln Tyr Glu Asp Phe Gly Ile Tyr Tyr Cys Gln Gln Tyr
    210                 215                 220
Asp Glu Ser Pro Trp Thr Phe Gly Gly Gly Thr Lys Leu Glu Met Lys
225                 230                 235                 240
</s-2-xi> </s-2> <s-2 seq-no="149"> <s-2-i> <s-2-i-a>107 amino acids</s-2-i-a> <s-2-i-b>amino acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-xi>149
Asp Ile Gln Met Thr Gln Ser Pro Ser Ser Leu Ser Ala Ser Val Gly
1               5                   10                  15
Asp Arg Val Thr Ile Thr Cys Arg Ala Ser Gln Xaa Ile Ser Xaa Tyr
            20                  25                  30
Leu Xaa Trp Tyr Gln Gln Lys Pro Gly Lys Ala Pro Lys Leu Leu Ile
        35                  40                  45
Tyr Ala Ala Ser Xaa Leu Xaa Ser Gly Val Pro Ser Arg Phe Ser Gly
    50                  55                  60
Ser Gly Ser Gly Thr Xaa Phe Thr Leu Thr Ile Ser Ser Leu Gln Pro
65                  70                  75                  80
Glu Asp Phe Ala Thr Tyr Tyr Cys Gln Gln Tyr Xaa Xaa Xaa Pro Xaa
                85                  90                  95
Thr Phe Gly Gln Gly Thr Lys Val Glu Ile Lys
            100                 105
</s-2-xi> </s-2> <s-2 seq-no="150"> <s-2-i> <s-2-i-a>108 amino acids</s-2-i-a> <s-2-i-b>amino acid</s-2-i-b> <s-2-i-c>double</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>protein</s-2-ii-mt> </s-2-ii> <s-2-xi>150
Glu Ile Val Leu Thr Gln Ser Pro Gly Thr Leu Ser Leu Ser Pro Gly
1               5                   10                  15
Glu Arg Ala Thr Leu Ser Cys Arg Ala Ser Gln Ser Val Ser Ser Ser
            20                  25                  30
Tyr Leu Ala Trp Tyr Gln Gln Lys Pro Gly Gln Ala Pro Arg Leu Leu
        35                  40                  45
Ile Tyr Gly Ala Ser Ser Arg Ala Thr Gly Ile Pro Asp Arg Phe Ser
    50                  55                  60
Gly Ser Gly Ser Gly Thr Asp Phe Thr Leu Thr Ile Ser Arg Leu Glu
65                  70                  75                  80
Pro Gly Asp Phe Ala Val Tyr Tyr Cys Gln Gln Tyr Gly Ser Ser Pro
                85                  90                  95
Xaa Thr Phe Gly Gln Gly Thr Lys Val Glu Ile Lys
            100                 105
</s-2-xi> </s-2> <s-2 seq-no="151"> <s-2-i> <s-2-i-a>108 amino acids</s-2-i-a> <s-2-i-b>amino acid</s-2-i-b> <s-2-i-c>double</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-xi>151
Asp Ile Val Met Thr Gln Ser Pro Leu Ser Leu Pro Val Thr Pro Gly
1               5                   10                  15
Glu Pro Ala Ser Ile Ser Cys Arg Ser Ser Gln Ser Leu Leu Asn Asn
            20                  25                  30
Tyr Leu Asn Trp Tyr Leu Gln Lys Pro Gly Gln Ser Pro Gln Leu Leu
        35                  40                  45
Ile Tyr Leu Gly Ser Asn Arg Ala Ser Gly Val Pro Asp Arg Phe Ser
    50                  55                  60
Gly Ser Gly Ser Gly Thr Asp Phe Thr Leu Lys Ile Ser Arg Val Glu
65                  70                  75                  80
Ala Glu Asp Val Gly Val Tyr Tyr Cys Met Gln Ala Leu Gln Xaa Pro
                85                  90                  95
Xaa Thr Phe Gly Gln Gly Thr Lys Xaa Glu Ile Lys
            100                 105
</s-2-xi> </s-2> <s-2 seq-no="152"> <s-2-i> <s-2-i-a>106 amino acids</s-2-i-a> <s-2-i-b>amino acid</s-2-i-b> <s-2-i-c>double</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>protein</s-2-ii-mt> </s-2-ii> <s-2-xi>152
Xaa Ser Val Leu Thr Gln Pro Pro Ser Ala Ser Gly Thr Pro Gly Gln
1               5                   10                  15
Arg Val Thr Ile Ser Cys Ser Gly Ser Ser Ser Ile Gly Xaa Asn Xaa
            20                  25                  30
Val Xaa Trp Tyr Gln Gln Leu Pro Gly Thr Ala Pro Lys Leu Leu Ile
        35                  40                  45
Tyr Asn Asn Arg Pro Ser Gly Val Pro Asp Arg Phe Ser Gly Ser Lys
    50                  55                  60
Ser Gly Thr Ser Ala Ser Leu Ala Ile Ser Gly Leu Gln Ser Glu Asp
65                  70                  75                  80
Glu Ala Asp Tyr Tyr Cys Ala Thr Trp Asp Asp Ser Leu Asp Pro Val
                85                  90                  95
Phe Gly Gly Gly Thr Lys Thr Val Leu Gly
            100                 105
</s-2-xi> </s-2> <s-2 seq-no="153"> <s-2-i> <s-2-i-a>104 amino acids</s-2-i-a> <s-2-i-b>amino acid</s-2-i-b> <s-2-i-c>double</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>protein</s-2-ii-mt> </s-2-ii> <s-2-xi>153
Xaa Ser Ala Leu Thr Gln Pro Ala Ser Val Ser Gly Ser Pro Gly Gln
1               5                   10                  15
Ser Ile Thr Ile Ser Cys Thr Gly Thr Ser Ser Val Gly Tyr Asn Xaa
            20                  25                  30
Val Ser Trp Tyr Gln Gln His Pro Gly Lys Ala Pro Lys Leu Ile Tyr
        35                  40                  45
Asp Val Arg Pro Ser Gly Val Arg Phe Ser Gly Ser Lys Ser Gly Asn
    50                  55                  60
Thr Ala Ser Leu Thr Ile Ser Gly Leu Gln Ala Glu Asp Glu Ala Asp
65                  70                  75                  80
Tyr Tyr Cys Ser Ser Tyr Xaa Gly Xaa Xaa Xaa Xaa Val Phe Gly Gly
                85                  90                  95
Gly Thr Lys Leu Thr Val Leu Gly
            100
</s-2-xi> </s-2> <s-2 seq-no="154"> <s-2-i> <s-2-i-a>100 amino acids</s-2-i-a> <s-2-i-b>amino acid</s-2-i-b> <s-2-i-c>double</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>protein</s-2-ii-mt> </s-2-ii> <s-2-xi>154
Ser Tyr Glu Leu Thr Gln Pro Pro Ser Val Ser Val Ser Pro Gly Gln
1               5                   10                  15
Thr Ala Ile Thr Cys Ser Gly Asp Xaa Leu Xaa Xaa Xaa Tyr Val Xaa
            20                  25                  30
Trp Tyr Gln Gln Lys Pro Gly Gln Ala Pro Val Leu Val Ile Tyr Asp
        35                  40                  45
Arg Pro Ser Gly Ile Pro Gln Arg Phe Ser Gly Ser Ser Thr Thr Ala
    50                  55                  60
Thr Leu Thr Ile Ser Gly Val Gln Ala Asp Glu Ala Asp Tyr Tyr Cys
65                  70                  75                  80
Gln Xaa Trp Asp Xaa Xaa Xaa Val Val Phe Gly Gly Gly Thr Lys Leu
                85                  90                  95
Thr Val Leu Gly
            100
</s-2-xi> </s-2> <s-2 seq-no="155"> <s-2-i> <s-2-i-a>106 amino acids</s-2-i-a> <s-2-i-b>amino acid</s-2-i-b> <s-2-i-c>double</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>protein</s-2-ii-mt> </s-2-ii> <s-2-xi>155
Asn Phe Met Leu Thr Gln Pro His Ser Val Ser Glu Ser Pro Gly Lys
1               5                   10                  15
Thr Val Thr Ile Ser Cys Thr Xaa Ser Xaa Gly Ile Ala Ser Xaa Tyr
            20                  25                  30
Val Gln Trp Tyr Gln Gln Arg Pro Gly Ser Ala Pro Thr Thr Val Ile
        35                  40                  45
Tyr Glu Asp Asn Arg Pro Ser Gly Val Pro Asp Arg Phe Ser Gly Ser
    50                  55                  60
Ser Ser Asn Ser Ala Ser Leu Thr Ile Ser Gly Leu Lys Thr Glu Asp
65                  70                  75                  80
Glu Ala Asp Tyr Tyr Cys Gln Ser Tyr Asp Ser Xaa Xaa Trp Val Phe
                85                  90                  95
Gly Gly Gly Thr Lys Leu Thr Val Leu Gly
            100                 105
</s-2-xi> </s-2> <s-2 seq-no="156"> <s-2-i> <s-2-i-a>107 amino acids</s-2-i-a> <s-2-i-b>amino acid</s-2-i-b> <s-2-i-c>double</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>protein</s-2-ii-mt> </s-2-ii> <s-2-xi>156
Asp Ile Val Met Thr Gln Ser Pro Asp Ser Leu Ala Val Ser Leu Gly
1               5                   10                  15
Glu Arg Ala Thr Ile Asn Cys Lys Ser Ser Gln Ser Val Leu Lys Asn
            20                  25                  30
Tyr Leu Ala Trp Tyr Gln Gln Lys Pro Gly Gln Pro Pro Lys Leu Leu
        35                  40                  45
Ile Tyr Trp Ala Ser Arg Glu Ser Gly Val Pro Asp Arg Phe Ser Gly
    50                  55                  60
Ser Gly Ser Gly Thr Asp Phe Thr Leu Thr Ile Ser Ser Leu Gln Ala
65                  70                  75                  80
Gln Asp Val Ala Val Tyr Tyr Cys Gln Gln Tyr Tyr Ser Thr Pro Xaa
                85                  90                  95
Thr Phe Gly Gln Gly Thr Lys Xaa Gly Ile Lys
            100                 105
</s-2-xi> </s-2> <s-2 seq-no="157"> <s-2-i> <s-2-i-a>105 amino acids</s-2-i-a> <s-2-i-b>amino acid</s-2-i-b> <s-2-i-c>double</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>protein</s-2-ii-mt> </s-2-ii> <s-2-xi>157
Ser Glu Leu Thr Gln Pro Pro Ser Val Ser Val Ala Pro Gly Gln Thr
1               5                   10                  15
Arg Ile Thr Cys Ser Gly Asp Xaa Leu Gly Xaa Tyr Asp Ala Xaa Trp
            20                  25                  30
Tyr Gln Gln Lys Pro Gly Gln Ala Pro Leu Leu Val Ile Tyr Gly Arg
        35                  40                  45
Asn Arg Pro Ser Gly Ile Pro Asp Arg Phe Ser Gly Ser Ser Ser Gly
    50                  55                  60
His Thr Ala Ser Leu Thr Ile Thr Gly Ala Gln Ala Glu Asp Glu Ala
65                  70                  75                  80
Asp Tyr Tyr Cys Asn Ser Arg Asp Ser Ser Gly Lys Val Leu Phe Gly
                85                  90                  95
Gly Gly Thr Lys Leu Thr Val Leu Gly
            100                 105
</s-2-xi> </s-2> <s-2 seq-no="158"> <s-2-i> <s-2-i-a>96 amino acids</s-2-i-a> <s-2-i-b>amino acid</s-2-i-b> <s-2-i-c>double</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>protein</s-2-ii-mt> </s-2-ii> <s-2-xi>158
Ser Ala Leu Thr Gln Pro Pro Ser Ala Ser Gly Ser Pro Gly Gln Ser
1               5                   10                  15
Val Thr Ile Ser Cys Thr Gly Thr Ser Ser Val Gly Xaa Xaa Tyr Val
            20                  25                  30
Ser Trp Tyr Gln Gln His Gly Ala Pro Lys Ile Glu Val Arg Pro Ser
        35                  40                  45
Gly Val Pro Asp Arg Phe Ser Gly Ser Lys Ser Asn Thr Ala Ser Leu
    50                  55                  60
Thr Val Ser Gly Leu Ala Glu Asp Glu Ala Asp Tyr Tyr Cys Ser Ser
65                  70                  75                  80
Tyr Xaa Xaa Xaa Xaa Xaa Phe Val Phe Gly Gly Thr Lys Thr Val Leu
                85                  90                  95
</s-2-xi> </s-2> <s-2 seq-no="159"> <s-2-i> <s-2-i-a>119 amino acids</s-2-i-a> <s-2-i-b>amino acid</s-2-i-b> <s-2-i-c>double</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>protein</s-2-ii-mt> </s-2-ii> <s-2-xi>159
Glu Val Gln Leu Val Glu Ser Gly Gly Gly Leu Val Gln Pro Gly Gly
1               5                   10                  15
Ser Leu Arg Leu Ser Cys Ala Ala Ser Gly Phe Thr Phe Ser Xaa Xaa
            20                  25                  30
Xaa Met Xaa Trp Val Arg Gln Ala Pro Gly Lys Gly Leu Glu Trp Val
        35                  40                  45
Xaa Xaa Ile Xaa Xaa Xaa Xaa Xaa Gly Xaa Xaa Tyr Ala Asp Ser Val
    50                  55                  60
Lys Gly Arg Phe Thr Ile Ser Arg Asp Asp Ser Lys Asn Thr Leu Tyr
65                  70                  75                  80
Leu Gln Met Asn Ser Leu Arg Ala Glu Asp Thr Ala Val Tyr Tyr Cys
                85                  90                  95
Ala Arg Xaa Xaa Xaa Xaa Xaa Xaa Xaa Xaa Xaa Xaa Trp Gly Gln Gly
            100                 105                 110
Thr Leu Val Thr Val Ser Ser
        115
</s-2-xi> </s-2> <s-2 seq-no="160"> <s-2-i> <s-2-i-a>119 amino acids</s-2-i-a> <s-2-i-b>amino acid</s-2-i-b> <s-2-i-c>double</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-xi>160
Gln Val Gln Leu Val Gln Ser Gly Ala Glu Val Lys Lys Pro Gly Xaa
1               5                   10                  15
Ser Val Xaa Val Ser Cys Lys Xaa Ser Gly Tyr Tyr Phe Xaa Xaa Tyr
            20                  25                  30
Xaa Ile Xaa Trp Val Arg Gln Ala Pro Gly Xaa Gly Leu Glu Trp Val
        35                  40                  45
Gly Xaa Ile Xaa Pro Xaa Xaa Gly Xaa Thr Xaa Tyr Ala Pro Xaa Phe
    50                  55                  60
Gln Gly Arg Val Thr Xaa Thr Arg Asp Xaa Ser Xaa Asn Thr Ala Tyr
65                  70                  75                  80
Met Glu Leu Xaa Ser Leu Arg Ser Glu Asp Thr Ala Val Tyr Tyr Cys
                85                  90                  95
Ala Arg Xaa Xaa Xaa Xaa Xaa Xaa Xaa Xaa Xaa Xaa Trp Gly Gln Gly
            100                 105                 110
Thr Leu Val Thr Val Ser Ser
        115
</s-2-xi> </s-2> <s-2 seq-no="161"> <s-2-i> <s-2-i-a>117 amino acids</s-2-i-a> <s-2-i-b>amino acid</s-2-i-b> <s-2-i-c>double</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>protein</s-2-ii-mt> </s-2-ii> <s-2-xi>161
Xaa Val Thr Leu Xaa Glu Ser Gly Pro Xaa Leu Val Leu Pro Thr Gln
1               5                   10                  15
Thr Leu Thr Leu Thr Cys Thr Val Ser Gly Xaa Ser Leu Ser Xaa Xaa
            20                  25                  30
Xaa Val Xaa Trp Ile Arg Gln Pro Pro Gly Lys Xaa Leu Glu Trp Leu
        35                  40                  45
Ala Xaa Ile Xaa Xaa Asp Asp Asp Xaa Tyr Xaa Thr Ser Leu Arg Ser
    50                  55                  60
Arg Leu Thr Ile Ser Lys Asp Thr Ser Lys Asn Gln Val Val Leu Xaa
65                  70                  75                  80
Xaa Xaa Xaa Xaa Asp Pro Xaa Asp Thr Ala Thr Tyr Tyr Cys Ala Arg
                85                  90                  95
Xaa Xaa Xaa Xaa Xaa Xaa Xaa Xaa Asp Val Trp Gly Gln Gly Thr Thr
            100                 105                 110
Val Thr Val Ser Ser
        115
</s-2-xi> </s-2> <s-2 seq-no="162"> <s-2-i> <s-2-i-a>107 amino acids</s-2-i-a> <s-2-i-b>amino acid</s-2-i-b> <s-2-i-c>Not Relevant</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>protein</s-2-ii-mt> </s-2-ii> <s-2-xi>162
Asp Ile Gln Met Thr Gln Ser Pro Ser Thr Leu Ser Ala Ser Val Gly
1               5                   10                  15
Asp Arg Val Thr Ile Thr Cys Arg Ala Ser Gln Ser Ile Asn Thr Trp
            20                  25                  30
Leu Ala Trp Tyr Gln Gln Lys Pro Gly Lys Ala Pro Lys Leu Leu Met
        35                  40                  45
Tyr Lys Ala Ser Ser Leu Glu Ser Gly Val Pro Ser Arg Phe Ile Gly
    50                  55                  60
Ser Gly Ser Gly Thr Glu Phe Thr Leu Thr Ile Ser Ser Leu Gln Pro
65                  70                  75                  80
Asp Asp Phe Ala Thr Tyr Tyr Cys Gln Gln Tyr Asn Ser Asp Ser Lys
                85                  90                  95
Met Phe Gly Gln Gly Thr Lys Val Glu Val Lys
            100                 105
</s-2-xi> </s-2> <s-2 seq-no="163"> <s-2-i> <s-2-i-a>106 amino acids</s-2-i-a> <s-2-i-b>amino acid</s-2-i-b> <s-2-i-c>double</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>protein</s-2-ii-mt> </s-2-ii> <s-2-xi>163
Gln Ile Val Leu Thr Gln Ser Pro Ala Ile Met Ser Ala Ser Pro Gly
1               5                   10                  15
Glu Lys Val Thr Ile Thr Cys Ser Ala Ser Ser Ser Ile Ser Tyr Met
            20                  25                  30
His Trp Phe Gln Gln Lys Pro Gly Thr Ser Pro Lys Leu Trp Ile Tyr
        35                  40                  45
Thr Thr Ser Asn Leu Ala Ser Gly Val Pro Ala Arg Phe Ser Gly Ser
    50                  55                  60
Gly Ser Gly Thr Ser Tyr Ser Leu Thr Ile Ser Arg Met Glu Ala Glu
65                  70                  75                  80
Asp Ala Ala Thr Tyr Tyr Cys His Gln Arg Ser Thr Tyr Pro Leu Thr
                85                  90                  95
Phe Gly Ser Gly Thr Lys Leu Glu Leu Lys
            100                 105
</s-2-xi> </s-2> <s-2 seq-no="164"> <s-2-i> <s-2-i-a>106 amino acids</s-2-i-a> <s-2-i-b>amino acid</s-2-i-b> <s-2-i-c>double</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>protein</s-2-ii-mt> </s-2-ii> <s-2-xi>164
Asp Ile Gln Leu Thr Gln Ser Pro Ser Ser Met Ser Ala Ser Pro Gly
1               5                   10                  15
Asp Arg Val Thr Ile Thr Cys Arg Ala Ser Ser Ser Ile Ser Tyr Met
            20                  25                  30
His Trp Phe Gln Gln Lys Pro Gly Lys Ser Pro Lys Leu Trp Ile Tyr
        35                  40                  45
Thr Thr Ser Asn Leu Ala Ser Gly Val Pro Ser Arg Phe Ser Gly Ser
    50                  55                  60
Gly Ser Gly Thr Ser Tyr Thr Leu Thr Ile Ser Ser Met Gln Ala Glu
65                  70                  75                  80
Asp Phe Ala Thr Tyr Tyr Cys His Gln Arg Ser Thr Tyr Pro Leu Thr
                85                  90                  95
Phe Gly Gln Gly Thr Lys Leu Glu Leu Lys
            100                 105
</s-2-xi> </s-2> <s-2 seq-no="165"> <s-2-i> <s-2-i-a>106 amino acids</s-2-i-a> <s-2-i-b>amino acid</s-2-i-b> <s-2-i-c>double</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>protein</s-2-ii-mt> </s-2-ii> <s-2-xi>165
Asp Ile Gln Met Thr Gln Ser Pro Ser Thr Leu Ser Ala Ser Val Gly
1               5                   10                  15
Asp Arg Val Thr Ile Thr Cys Ser Ala Ser Ser Ser Ile Ser Tyr Met
            20                  25                  30
His Trp Tyr Gln Gln Lys Pro Gly Lys Ala Pro Lys Leu Leu Ile Tyr
        35                  40                  45
Thr Thr Ser Asn Leu Ala Ser Gly Val Pro Ala Arg Phe Ser Gly Ser
    50                  55                  60
Gly Ser Gly Thr Glu Phe Thr Leu Thr Ile Ser Ser Leu Gln Pro Asp
65                  70                  75                  80
Asp Phe Ala Thr Tyr Tyr Cys His Gln Arg Ser Thr Tyr Pro Leu Thr
                85                  90                  95
Phe Gly Gln Gly Thr Lys Val Glu Val Lys
            100                 105
</s-2-xi> </s-2> <s-2 seq-no="166"> <s-2-i> <s-2-i-a>117 amino acids</s-2-i-a> <s-2-i-b>amino acid</s-2-i-b> <s-2-i-c>Not Relevant</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>protein</s-2-ii-mt> </s-2-ii> <s-2-xi>166
Gln Val Gln Leu Val Gln Ser Gly Ala Glu Val Lys Lys Pro Gly Ser
1               5                   10                  15
Ser Val Lys Val Ser Cys Lys Ala Ser Gly Gly Thr Phe Ser Arg Ser
            20                  25                  30
Ala Ile Ile Trp Val Arg Gln Ala Pro Gly Gln Gly Leu Glu Trp Met
        35                  40                  45
Gly Gly Ile Val Pro Met Phe Gly Pro Pro Asn Tyr Ala Gln Lys Phe
    50                  55                  60
Gln Gly Arg Val Thr Ile Thr Ala Asp Glu Ser Thr Asn Thr Ala Tyr
65                  70                  75                  80
Met Glu Leu Ser Ser Leu Arg Ser Glu Asp Thr Ala Phe Tyr Phe Cys
                85                  90                  95
Ala Gly Gly Tyr Gly Ile Tyr Ser Pro Glu Glu Tyr Asn Gly Gly Leu
            100                 105                 110
Val Thr Val Ser Ser
        115
</s-2-xi> </s-2> <s-2 seq-no="167"> <s-2-i> <s-2-i-a>116 amino acids</s-2-i-a> <s-2-i-b>amino acid</s-2-i-b> <s-2-i-c>double</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>protein</s-2-ii-mt> </s-2-ii> <s-2-xi>167
Gln Val Gln Leu Gln Gln Ser Gly Ala Glu Leu Ala Lys Pro Gly Ala
1               5                   10                  15
Ser Val Lys Met Ser Cys Lys Ala Ser Gly Tyr Thr Phe Thr Ser Tyr
            20                  25                  30
Arg Met His Trp Val Lys Gln Arg Pro Gly Gln Gly Leu Glu Trp Ile
        35                  40                  45
Gly Tyr Ile Asn Pro Ser Thr Gly Tyr Thr Glu Tyr Asn Gln Lys Phe
    50                  55                  60
Lys Asp Lys Ala Thr Leu Thr Ala Asp Lys Ser Ser Ser Thr Ala Tyr
65                  70                  75                  80
Met Gln Leu Ser Ser Leu Thr Phe Glu Asp Ser Ala Val Tyr Tyr Cys
                85                  90                  95
Ala Arg Gly Gly Gly Val Phe Asp Tyr Trp Gly Gln Gly Thr Thr Leu
            100                 105                 110
Thr Val Ser Ser
        115
</s-2-xi> </s-2> <s-2 seq-no="168"> <s-2-i> <s-2-i-a>116 amino acids</s-2-i-a> <s-2-i-b>amino acid</s-2-i-b> <s-2-i-c>double</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>protein</s-2-ii-mt> </s-2-ii> <s-2-xi>168
Gln Val Gln Leu Gln Gln Ser Gly Ala Glu Val Ala Lys Pro Gly Ala
1               5                   10                  15
Ser Val Lys Met Ser Cys Lys Ala Ser Gly Tyr Thr Phe Thr Ser Tyr
            20                  25                  30
Arg Met His Trp Val Lys Gln Ala Pro Gly Gln Gly Leu Glu Trp Ile
        35                  40                  45
Gly Tyr Ile Asn Pro Ser Thr Gly Tyr Thr Glu Tyr Asn Gln Lys Phe
    50                  55                  60
Lys Gly Lys Ala Thr Leu Thr Ala Asp Lys Ser Ser Ser Thr Ala Tyr
65                  70                  75                  80
Met Glu Leu Ser Ser Leu Arg Ser Glu Asp Thr Ala Val Tyr Tyr Cys
                85                  90                  95
Ala Arg Gly Gly Gly Val Phe Asp Tyr Trp Gly Gln Gly Thr Thr Leu
            100                 105                 110
Thr Val Ser Ser
        115
</s-2-xi> </s-2> <s-2 seq-no="169"> <s-2-i> <s-2-i-a>116 amino acids</s-2-i-a> <s-2-i-b>amino acid</s-2-i-b> <s-2-i-c>double</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-xi>169
Gln Val Gln Leu Val Gln Ser Gly Ala Glu Val Lys Lys Pro Gly Ser
1               5                   10                  15
Ser Val Lys Val Ser Cys Lys Ala Ser Gly Tyr Thr Phe Thr Ser Tyr
            20                  25                  30
Arg Met His Trp Val Arg Gln Ala Pro Gly Gln Gly Leu Glu Trp Ile
        35                  40                  45
Gly Tyr Ile Asn Pro Ser Thr Gly Tyr Thr Glu Tyr Asn Gln Lys Phe
    50                  55                  60
Lys Asp Lys Ala Thr Ile Thr Ala Asp Glu Ser Thr Asn Thr Ala Tyr
65                  70                  75                  80
Met Glu Leu Ser Ser Leu Arg Ser Glu Asp Thr Ala Val Tyr Tyr Cys
                85                  90                  95
Ala Arg Gly Gly Gly Val Phe Asp Tyr Trp Gly Gln Gly Thr Leu Val
            100                 105                 110
Thr Val Ser Ser
        115
</s-2-xi> </s-2> <s-2 seq-no="170"> <s-2-i> <s-2-i-a>80 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>170
TGACTCGCCC GGCAAGTGAT AGTGACTCTG TCTCCCAGAC ATGCAGACAT GGAAGATGAG     60
GACTGAGTCA TCTGGATGTC                                                 80
</s-2-xi> </s-2> <s-2 seq-no="171"> <s-2-i> <s-2-i-a>79 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>171
TCACTTGCCG GGCGAGTCAG GACATTAATA GCTATTTAAG CTGGTTCCAG CAGAAACCAG     60
GGAAATCTCC TAAGACCCT                                                  79
</s-2-xi> </s-2> <s-2 seq-no="172"> <s-2-i> <s-2-i-a>79 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>172
GATCCACTGC CACTGAACCT TGATGGGACC CCATCTACCA ATCTGTTTGC ACGATAGATC     60
AGGGTCTTAG GAGATTTCC                                                  79
</s-2-xi> </s-2> <s-2 seq-no="173"> <s-2-i> <s-2-i-a>85 base pairs</s-2-i-a> <s-2-i-b>nucleic acid</s-2-i-b> <s-2-i-c>single</s-2-i-c> <s-2-i-d>linear</s-2-i-d> </s-2-i> <s-2-ii> <s-2-ii-mt>DNA</s-2-ii-mt> </s-2-ii> <s-2-xi>173
TGTCGACATC ATGGCTTGGG TGTGGACCTT GCTATTCCTG ATGGCAGCTG CCCAAAGTGC     60
CCAAGCACAG ATCCAGTTGG TGCAG                                           85
</s-2-xi> </s-2> </sequence-list-old-rules> </sequence-cwu> </p>
    </div></div></div><div class="patent-section patent-tabular-section"><a id="backward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Cited Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4675382">US4675382</a></td><td class="patent-data-table-td patent-date-value">Nov 6, 1985</td><td class="patent-data-table-td patent-date-value">Jun 23, 1987</td><td class="patent-data-table-td ">President And Fellows Of Harvard College</td><td class="patent-data-table-td ">Anticarcinogenic agents; polypeptide linkages</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4769326">US4769326</a></td><td class="patent-data-table-td patent-date-value">Apr 12, 1984</td><td class="patent-data-table-td patent-date-value">Sep 6, 1988</td><td class="patent-data-table-td ">The Regents Of The University Of California</td><td class="patent-data-table-td ">Genetic engineering</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4853871">US4853871</a></td><td class="patent-data-table-td patent-date-value">Apr 6, 1987</td><td class="patent-data-table-td patent-date-value">Aug 1, 1989</td><td class="patent-data-table-td ">Genex Corporation</td><td class="patent-data-table-td ">Computer-based method for designing stablized proteins</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4888415">US4888415</a></td><td class="patent-data-table-td patent-date-value">Mar 7, 1988</td><td class="patent-data-table-td patent-date-value">Dec 19, 1989</td><td class="patent-data-table-td ">Dana-Farber Cancer Institute, Inc.</td><td class="patent-data-table-td ">Anticarcinogenic agents</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4894443">US4894443</a></td><td class="patent-data-table-td patent-date-value">Sep 7, 1984</td><td class="patent-data-table-td patent-date-value">Jan 16, 1990</td><td class="patent-data-table-td ">Cetus Corporation</td><td class="patent-data-table-td ">Polypeptides, antibodies</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4925673">US4925673</a></td><td class="patent-data-table-td patent-date-value">Aug 14, 1987</td><td class="patent-data-table-td patent-date-value">May 15, 1990</td><td class="patent-data-table-td ">Clinical Technologies Associates, Inc.</td><td class="patent-data-table-td ">Delivery systems for pharmacological agents encapsulated with proteinoids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4946778">US4946778</a></td><td class="patent-data-table-td patent-date-value">Jan 19, 1989</td><td class="patent-data-table-td patent-date-value">Aug 7, 1990</td><td class="patent-data-table-td ">Genex Corporation</td><td class="patent-data-table-td ">Single polypeptide chain binding molecules</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5013653">US5013653</a></td><td class="patent-data-table-td patent-date-value">Dec 28, 1989</td><td class="patent-data-table-td patent-date-value">May 7, 1991</td><td class="patent-data-table-td ">Creative Biomolecules, Inc.</td><td class="patent-data-table-td ">Product and process for introduction of a hinge region into a fusion protein to facilitate cleavage</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5028530">US5028530</a></td><td class="patent-data-table-td patent-date-value">Feb 5, 1990</td><td class="patent-data-table-td patent-date-value">Jul 2, 1991</td><td class="patent-data-table-td ">Xoma Corporation</td><td class="patent-data-table-td ">AraB promoters and method of producing polypeptides, including cecropins, by microbiological techniques</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5091513">US5091513</a></td><td class="patent-data-table-td patent-date-value">Jan 2, 1991</td><td class="patent-data-table-td patent-date-value">Feb 25, 1992</td><td class="patent-data-table-td ">Creative Biomolecules, Inc.</td><td class="patent-data-table-td ">Biosynthetic antibody binding sites</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5093475">US5093475</a></td><td class="patent-data-table-td patent-date-value">Dec 21, 1989</td><td class="patent-data-table-td patent-date-value">Mar 3, 1992</td><td class="patent-data-table-td ">Xoma Corporation</td><td class="patent-data-table-td ">Hindered linking agents derived from 2-iminothiolanes and methods</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5101025">US5101025</a></td><td class="patent-data-table-td patent-date-value">Sep 7, 1989</td><td class="patent-data-table-td patent-date-value">Mar 31, 1992</td><td class="patent-data-table-td ">Genelabs Incorporated</td><td class="patent-data-table-td ">Recombinant trichosanthin and coding sequence</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5169939">US5169939</a></td><td class="patent-data-table-td patent-date-value">Jan 9, 1989</td><td class="patent-data-table-td patent-date-value">Dec 8, 1992</td><td class="patent-data-table-td ">Massachusetts Institute Of Technology &amp; Pres. &amp; Fellows Of Harvard College</td><td class="patent-data-table-td ">Chimeric antibodies</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5376546">US5376546</a></td><td class="patent-data-table-td patent-date-value">Jun 19, 1992</td><td class="patent-data-table-td patent-date-value">Dec 27, 1994</td><td class="patent-data-table-td ">Xoma Corporation</td><td class="patent-data-table-td ">Analogs of ribosome-inactivating proteins</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5416202">US5416202</a></td><td class="patent-data-table-td patent-date-value">Dec 9, 1992</td><td class="patent-data-table-td patent-date-value">May 16, 1995</td><td class="patent-data-table-td ">Xoma Corporation</td><td class="patent-data-table-td ">Such as gelonin, cysteine substitution</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5621083">US5621083</a></td><td class="patent-data-table-td patent-date-value">Apr 18, 1995</td><td class="patent-data-table-td patent-date-value">Apr 15, 1997</td><td class="patent-data-table-td ">Xoma Corporation</td><td class="patent-data-table-td ">Immunotoxins comprising ribosome-inactivating proteins</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5624827">US5624827</a></td><td class="patent-data-table-td patent-date-value">Oct 21, 1994</td><td class="patent-data-table-td patent-date-value">Apr 29, 1997</td><td class="patent-data-table-td ">Research Development Foundation</td><td class="patent-data-table-td ">DNA sequences encoding the plant toxin gelonin</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5631348">US5631348</a></td><td class="patent-data-table-td patent-date-value">Jun 6, 1994</td><td class="patent-data-table-td patent-date-value">May 20, 1997</td><td class="patent-data-table-td ">The Research Development Foundation</td><td class="patent-data-table-td ">Protein sequence of the plant toxin gelonin</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5744580">US5744580</a></td><td class="patent-data-table-td patent-date-value">Jun 7, 1995</td><td class="patent-data-table-td patent-date-value">Apr 28, 1998</td><td class="patent-data-table-td ">Xoma Corporation</td><td class="patent-data-table-td ">Fusion protein</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5756699">US5756699</a></td><td class="patent-data-table-td patent-date-value">Jun 7, 1995</td><td class="patent-data-table-td patent-date-value">May 26, 1998</td><td class="patent-data-table-td ">Xoma Corporation</td><td class="patent-data-table-td ">Immunotoxins comprising ribosome-inactivating proteins</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5837491">US5837491</a></td><td class="patent-data-table-td patent-date-value">May 12, 1994</td><td class="patent-data-table-td patent-date-value">Nov 17, 1998</td><td class="patent-data-table-td ">Xoma Corporation</td><td class="patent-data-table-td ">Polynucleotides encoding gelonin sequences</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6146631">US6146631</a></td><td class="patent-data-table-td patent-date-value">Apr 15, 1997</td><td class="patent-data-table-td patent-date-value">Nov 14, 2000</td><td class="patent-data-table-td ">Xoma Corporation</td><td class="patent-data-table-td ">Administering a fusion protein comprising a gelonin prefered amino acid sequeance, a targeting sequence that allows internalization of fusion protein in the cell, internizing the fusion protein in the cell, eliminating the cell</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="http://www.google.com/url?id=CsRZBAABERAJ&amp;q=http://worldwide.espacenet.com/publicationDetails/biblio%3FCC%3DAU%26NR%3D2761788A%26KC%3DA%26FT%3DD&amp;usg=AFQjCNHVag2gtR5lF1P2gZqiGZLDdvHYTQ">AU2761788A</a></td><td class="patent-data-table-td patent-date-value"></td><td class="patent-data-table-td patent-date-value"></td><td class="patent-data-table-td "> </td><td class="patent-data-table-td citation-no-title">Title not available</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP0170697A1?cl=en">EP0170697A1</a></td><td class="patent-data-table-td patent-date-value">Feb 7, 1985</td><td class="patent-data-table-td patent-date-value">Feb 12, 1986</td><td class="patent-data-table-td ">Cetus Corp</td><td class="patent-data-table-td ">Toxin conjugates.</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP0192002A1?cl=en">EP0192002A1</a></td><td class="patent-data-table-td patent-date-value">Dec 4, 1985</td><td class="patent-data-table-td patent-date-value">Aug 27, 1986</td><td class="patent-data-table-td ">Elf Sanofi</td><td class="patent-data-table-td ">Prolonged acting immunotoxins comprising a ribosome inactivating glycopeptide constituent having modified polyssaccharide moieties</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP0239400A2?cl=en">EP0239400A2</a></td><td class="patent-data-table-td patent-date-value">Mar 26, 1987</td><td class="patent-data-table-td patent-date-value">Sep 30, 1987</td><td class="patent-data-table-td ">Medical Research Council</td><td class="patent-data-table-td ">Recombinant antibodies and methods for their production</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP0438310A1?cl=en">EP0438310A1</a></td><td class="patent-data-table-td patent-date-value">Jan 17, 1991</td><td class="patent-data-table-td patent-date-value">Jul 24, 1991</td><td class="patent-data-table-td ">Merck &amp;amp; Co., Inc.</td><td class="patent-data-table-td ">Method for producing recombinant immunoglobuline</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP0506124A1?cl=en">EP0506124A1</a></td><td class="patent-data-table-td patent-date-value">Mar 27, 1992</td><td class="patent-data-table-td patent-date-value">Sep 30, 1992</td><td class="patent-data-table-td ">Tanox Biosystems, Inc.</td><td class="patent-data-table-td ">Methods and substances for recruiting therapeutic agents to solid issues comprising two single chain VH-VL bifunctional binding molecules</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP0519596A1?cl=en">EP0519596A1</a></td><td class="patent-data-table-td patent-date-value">May 12, 1992</td><td class="patent-data-table-td patent-date-value">Dec 23, 1992</td><td class="patent-data-table-td ">National Institutes Of Health</td><td class="patent-data-table-td ">A method for reducing the immunogenicity of antibody variable domains</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="http://www.google.com/url?id=CsRZBAABERAJ&amp;q=http://worldwide.espacenet.com/publicationDetails/biblio%3FCC%3DGB%26NR%3D2216891A%26KC%3DA%26FT%3DD&amp;usg=AFQjCNESljPHD2abVEgz0LkToW9gID6lIw">GB2216891A</a></td><td class="patent-data-table-td patent-date-value"></td><td class="patent-data-table-td patent-date-value"></td><td class="patent-data-table-td "> </td><td class="patent-data-table-td citation-no-title">Title not available</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1983003971A1?cl=en">WO1983003971A1</a></td><td class="patent-data-table-td patent-date-value">May 12, 1983</td><td class="patent-data-table-td patent-date-value">Nov 24, 1983</td><td class="patent-data-table-td ">Harvard College</td><td class="patent-data-table-td ">Hybrid proteins</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1985003508A1?cl=en">WO1985003508A1</a></td><td class="patent-data-table-td patent-date-value">Feb 7, 1985</td><td class="patent-data-table-td patent-date-value">Aug 15, 1985</td><td class="patent-data-table-td ">Cetus Corp</td><td class="patent-data-table-td ">Toxin conjugates</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1986000090A1?cl=en">WO1986000090A1</a></td><td class="patent-data-table-td patent-date-value">Jun 7, 1985</td><td class="patent-data-table-td patent-date-value">Jan 3, 1986</td><td class="patent-data-table-td ">John R Murphy</td><td class="patent-data-table-td ">Hybrid protein and fused gene encoding gene</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1986005098A1?cl=en">WO1986005098A1</a></td><td class="patent-data-table-td patent-date-value">Jul 8, 1985</td><td class="patent-data-table-td patent-date-value">Sep 12, 1986</td><td class="patent-data-table-td ">Dana Farber Cancer Inst Inc</td><td class="patent-data-table-td ">Immunotoxin and method of making</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1987002987A1?cl=en">WO1987002987A1</a></td><td class="patent-data-table-td patent-date-value">Nov 13, 1986</td><td class="patent-data-table-td patent-date-value">May 21, 1987</td><td class="patent-data-table-td ">John R Murphy</td><td class="patent-data-table-td ">Cys codon-modified dna</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1988001649A1?cl=en">WO1988001649A1</a></td><td class="patent-data-table-td patent-date-value">Sep 2, 1987</td><td class="patent-data-table-td patent-date-value">Mar 10, 1988</td><td class="patent-data-table-td ">Genex Corp</td><td class="patent-data-table-td ">Single polypeptide chain binding molecules</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1988007085A1?cl=en">WO1988007085A1</a></td><td class="patent-data-table-td patent-date-value">Mar 4, 1988</td><td class="patent-data-table-td patent-date-value">Sep 22, 1988</td><td class="patent-data-table-td ">Creative Biomolecules Inc</td><td class="patent-data-table-td ">Process for the purification of recombinant polypeptides</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1988009344A1?cl=en">WO1988009344A1</a></td><td class="patent-data-table-td patent-date-value">May 19, 1988</td><td class="patent-data-table-td patent-date-value">Dec 1, 1988</td><td class="patent-data-table-td ">Creative Biomolecules Inc</td><td class="patent-data-table-td ">Targeted multifunctional proteins</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1989000999A1?cl=en">WO1989000999A1</a></td><td class="patent-data-table-td patent-date-value">Jul 25, 1988</td><td class="patent-data-table-td patent-date-value">Feb 9, 1989</td><td class="patent-data-table-td ">Int Genetic Eng</td><td class="patent-data-table-td ">Modular assembly of antibody genes, antibodies prepared thereby and use</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1989006967A1?cl=en">WO1989006967A1</a></td><td class="patent-data-table-td patent-date-value">Feb 3, 1989</td><td class="patent-data-table-td patent-date-value">Aug 10, 1989</td><td class="patent-data-table-td ">Xoma Corp</td><td class="patent-data-table-td ">Immunosupression with anti-pan t-cell immunotoxin compositions</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1989006968A1?cl=en">WO1989006968A1</a></td><td class="patent-data-table-td patent-date-value">Feb 3, 1989</td><td class="patent-data-table-td patent-date-value">Aug 10, 1989</td><td class="patent-data-table-td ">Xoma Corp</td><td class="patent-data-table-td ">Therapeutic use of anti-t cell immunotoxin for autoimmune diseases</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1989009622A1?cl=en">WO1989009622A1</a></td><td class="patent-data-table-td patent-date-value">Apr 13, 1989</td><td class="patent-data-table-td patent-date-value">Oct 19, 1989</td><td class="patent-data-table-td ">Protein Design Labs Inc</td><td class="patent-data-table-td ">Il-2 receptor-specific chimeric antibodies</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1990002569A1?cl=en">WO1990002569A1</a></td><td class="patent-data-table-td patent-date-value">Sep 6, 1989</td><td class="patent-data-table-td patent-date-value">Mar 22, 1990</td><td class="patent-data-table-td ">Int Genetic Eng</td><td class="patent-data-table-td ">Production of chimeric mouse-human antibodies with specificity to human tumor antigens</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1990010015A1?cl=en">WO1990010015A1</a></td><td class="patent-data-table-td patent-date-value">Feb 1, 1990</td><td class="patent-data-table-td patent-date-value">Sep 7, 1990</td><td class="patent-data-table-td ">Seragen Inc</td><td class="patent-data-table-td ">Chimeric toxin</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1990012592A1?cl=en">WO1990012592A1</a></td><td class="patent-data-table-td patent-date-value">Apr 20, 1990</td><td class="patent-data-table-td patent-date-value">Oct 22, 1990</td><td class="patent-data-table-td ">Protein Design Labs Inc</td><td class="patent-data-table-td ">Recombinant antibody-toxin fusion protein</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1991002000A1?cl=en">WO1991002000A1</a></td><td class="patent-data-table-td patent-date-value">Jul 30, 1990</td><td class="patent-data-table-td patent-date-value">Feb 3, 1991</td><td class="patent-data-table-td ">Seragen Inc</td><td class="patent-data-table-td ">Il-2 deletion mutants</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1991019745A1?cl=en">WO1991019745A1</a></td><td class="patent-data-table-td patent-date-value">Jun 12, 1991</td><td class="patent-data-table-td patent-date-value">Dec 14, 1991</td><td class="patent-data-table-td ">Univ Hospital</td><td class="patent-data-table-td ">Chimeric toxins with improved inter-domain geometry</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1992003144A1?cl=en">WO1992003144A1</a></td><td class="patent-data-table-td patent-date-value">Aug 14, 1991</td><td class="patent-data-table-td patent-date-value">Feb 15, 1992</td><td class="patent-data-table-td ">Res Dev Foundation</td><td class="patent-data-table-td ">Protein structure of the plant toxin gelonin</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1992008495A1?cl=en">WO1992008495A1</a></td><td class="patent-data-table-td patent-date-value">Nov 7, 1991</td><td class="patent-data-table-td patent-date-value">May 10, 1992</td><td class="patent-data-table-td ">Abbott Biotech Inc</td><td class="patent-data-table-td ">Cytokine immunoconjugates</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1992009613A1?cl=en">WO1992009613A1</a></td><td class="patent-data-table-td patent-date-value">Dec 3, 1991</td><td class="patent-data-table-td patent-date-value">Jun 4, 1992</td><td class="patent-data-table-td ">Us Health</td><td class="patent-data-table-td ">Recombinant immunotoxin composed of a single chain antibody reacting with the human transferrin receptor and diphtheria toxin</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1992011018A1?cl=en">WO1992011018A1</a></td><td class="patent-data-table-td patent-date-value">Dec 19, 1991</td><td class="patent-data-table-td patent-date-value">Jun 20, 1992</td><td class="patent-data-table-td ">Protein Design Labs Inc</td><td class="patent-data-table-td ">Improved humanized immunoglobulins</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1992014491A1?cl=en">WO1992014491A1</a></td><td class="patent-data-table-td patent-date-value">Feb 18, 1992</td><td class="patent-data-table-td patent-date-value">Sep 3, 1992</td><td class="patent-data-table-td ">Italfarmaco Spa</td><td class="patent-data-table-td ">Immunotoxin from anti-cd5 monoclonal antibodies</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1992015327A1?cl=en">WO1992015327A1</a></td><td class="patent-data-table-td patent-date-value">Mar 6, 1992</td><td class="patent-data-table-td patent-date-value">Sep 17, 1992</td><td class="patent-data-table-td ">Protein Design Labs Inc</td><td class="patent-data-table-td ">Recombinant double chain immunotoxins</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1992022324A1?cl=en">WO1992022324A1</a></td><td class="patent-data-table-td patent-date-value">Jun 15, 1992</td><td class="patent-data-table-td patent-date-value">Dec 23, 1992</td><td class="patent-data-table-td ">Xoma Corp</td><td class="patent-data-table-td ">Microbially-produced antibody fragments and their conjugates</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1993005168A1?cl=en">WO1993005168A1</a></td><td class="patent-data-table-td patent-date-value">Aug 21, 1992</td><td class="patent-data-table-td patent-date-value">Mar 18, 1993</td><td class="patent-data-table-td ">Res Dev Foundation</td><td class="patent-data-table-td ">Dna sequences encoding gelonin polypeptide</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1993009130A1?cl=en">WO1993009130A1</a></td><td class="patent-data-table-td patent-date-value">Nov 4, 1992</td><td class="patent-data-table-td patent-date-value">May 13, 1993</td><td class="patent-data-table-td ">Xoma Corp</td><td class="patent-data-table-td ">Materials comprising and methods of preparation and use for ribosome-inactivating proteins</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2002006117A1?cl=en">WO2002006117A1</a></td><td class="patent-data-table-td patent-date-value">Jul 13, 2001</td><td class="patent-data-table-td patent-date-value">Jan 24, 2002</td><td class="patent-data-table-td ">Juliet Claire Blackburn</td><td class="patent-data-table-td ">Convertible seating and bedding apparatus</td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="npl-citations"></a><div class="patent-section-header"><span class="patent-section-title">Non-Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th colspan="3"class="patent-data-table-th">Reference</th></tr></thead><tr><td class="patent-data-table-td ">1</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Antin, J.H., et al., "<a href='http://scholar.google.com/scholar?q="Selective+Depletion+of+Bone+Marrow+T+Lymphoytes+With+Anti-CD5+Monoclonal+Antibodies%3A+Effective+Prophylaxis+for+Graft-Versus-Host+Disease+in+Patients+With+Hematologic+Malignancies%2C"'>Selective Depletion of Bone Marrow T Lymphoytes With Anti-CD5 Monoclonal Antibodies: Effective Prophylaxis for Graft-Versus-Host Disease in Patients With Hematologic Malignancies,</a>" Blood 78:2139-2149, W. B. Saunders (Oct. 1991).</td></tr><tr><td class="patent-data-table-td ">2</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Asano, K., et al., "<a href='http://scholar.google.com/scholar?q="Isolation+and+Characterization+of+Inhibitors+of+Animal+Cell-Free+Protein+Synthesis+from+Barley+Seeds%2C"'>Isolation and Characterization of Inhibitors of Animal Cell-Free Protein Synthesis from Barley Seeds,</a>" Carlsberg Res. Comm. 49:619-626, Springer-Verlag (1984).</td></tr><tr><td class="patent-data-table-td ">3</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Bacha, P., et al., "<a href='http://scholar.google.com/scholar?q="Interleukin+2+Receptor-Targeted+Cytotoxicity%2C"'>Interleukin 2 Receptor-Targeted Cytotoxicity,</a>" J. Exp. Med. 167:612-622, The Rockfeller University Press (Feb. 1988).</td></tr><tr><td class="patent-data-table-td ">4</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Barry, B.W., ed., in Dermatological Formulations-Percutaneous Absorption, Marcel Decker, Inc., New York, N.Y., pp. 180-181 (1983).</td></tr><tr><td class="patent-data-table-td ">5</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Barry, B.W., ed., in Dermatological Formulations—Percutaneous Absorption, Marcel Decker, Inc., New York, N.Y., pp. 180-181 (1983).</td></tr><tr><td class="patent-data-table-td ">6</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Batra, J.K., et al., "<a href='http://scholar.google.com/scholar?q="Anti-Tac%28Fv%29-PE40%2C+a+Single+Chain+Antibody+Pseudomonas+Fusion+Protein+Directed+at+Interleukin+2+Receptor+Bearing+Cells%2C"'>Anti-Tac(Fv)-PE40, a Single Chain Antibody Pseudomonas Fusion Protein Directed at Interleukin 2 Receptor Bearing Cells,</a>" J. Biol. Chem. 265:15198-15202, American Society for Biochemistry and Molecular Biology, Inc. (Sep. 1990).</td></tr><tr><td class="patent-data-table-td ">7</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Batra, J.K., et al., "<a href='http://scholar.google.com/scholar?q="Single-Chain+Immunotoxins+Directed+at+the+Human+Transferrin+Receptor+Containing+Pseudomonas+Exotoxin+A+or+Diphtheria+Toxin%3A+Anti-TFR+%28Fv%29-PE40+and+DT388-Anti-TFR%28Fv%29%2C"'>Single-Chain Immunotoxins Directed at the Human Transferrin Receptor Containing Pseudomonas Exotoxin A or Diphtheria Toxin: Anti-TFR (Fv)-PE40 and DT388-Anti-TFR(Fv),</a>" Mol. Cell. Biol. 11:2200-2205, American Society for Microbiology (Apr. 1991).</td></tr><tr><td class="patent-data-table-td ">8</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Benatti, L., et al., "<a href='http://scholar.google.com/scholar?q="Nucleotide+sequence+of+cDNA+coding+for+saporin-6%2C+a+type-1%2C+ribosome-inactivating+protein+from+Saponaria+officinalis%2C"'>Nucleotide sequence of cDNA coding for saporin-6, a type-1, ribosome-inactivating protein from Saponaria officinalis,</a>" Eur. J. Biochem. 183:465-470, The Federation of American Societies for Experimental Biology (1989).</td></tr><tr><td class="patent-data-table-td ">9</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Bernhard, S.L., et al., "<a href='http://scholar.google.com/scholar?q="Cysteine+Analogs+of+Recombinant+Barley+Ribosome+Inactivating+Protein+Form+Antibody+Conjugates+with+Enhanced+Stability+and+Potency+in+Vitro%2C"'>Cysteine Analogs of Recombinant Barley Ribosome Inactivating Protein Form Antibody Conjugates with Enhanced Stability and Potency in Vitro,</a>" Bioconjugate Chem. 5:126-132, American Chemical Society (Mar. 1994).</td></tr><tr><td class="patent-data-table-td ">10</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Better, M., et al., "<a href='http://scholar.google.com/scholar?q="Acitvity+of+Recombinant+Mitogillin+and+Mitogillin+Immunoconjugates%2C"'>Acitvity of Recombinant Mitogillin and Mitogillin Immunoconjugates,</a>" J. Biol. Chem. 267:16712-16718, American Society for Biochemistry and Molecular Biology, Inc. (Aug. 1992).</td></tr><tr><td class="patent-data-table-td ">11</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Better, M., et al., "<a href='http://scholar.google.com/scholar?q="Escherichia+coli+Secretion+of+an+Active+Chimeric+Antibody+Fragment%2C"'>Escherichia coli Secretion of an Active Chimeric Antibody Fragment,</a>" Science 240:1041-1043, Association for the Advancement of Science (May 1988).</td></tr><tr><td class="patent-data-table-td ">12</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Better, M., et al., "<a href='http://scholar.google.com/scholar?q="Gelonin+Analogs+with+Engineered+Cysteine+Residues+Form+Antibody+Immunoconjugates+with+Unique+Properties%2C"'>Gelonin Analogs with Engineered Cysteine Residues Form Antibody Immunoconjugates with Unique Properties,</a>" J. Biol. Chem. 269:9644-9650, American Society for Biochemistry and Molecular Biology, Inc. (Apr. 1994).</td></tr><tr><td class="patent-data-table-td ">13</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Better, M., et al., "<a href='http://scholar.google.com/scholar?q="Generation+of+Potent+Immunoconjugates+from+Microbially+Produced+Fab%27%2C+F%28ab%27%292%2C+and+Recombinant+Ribosome+Inactivating+Proteins%2C"'>Generation of Potent Immunoconjugates from Microbially Produced Fab', F(ab')2, and Recombinant Ribosome Inactivating Proteins,</a>" (Poster), Third Intl. Symposium on Immunotoxins, Orlando, FL (Jun. 1992).</td></tr><tr><td class="patent-data-table-td ">14</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Better, M., et al., "<a href='http://scholar.google.com/scholar?q="Potent+anti-CD5+ricin+A+chain+immunoconjugates+from+bacterially+produced+Fab%27+and+F%28ab%27%292%2C"'>Potent anti-CD5 ricin A chain immunoconjugates from bacterially produced Fab' and F(ab')2,</a>" Proc. Natl. Acad. Sci. USA 90:457-461, National Academy of Sciences of the USA (Jan. 1993).</td></tr><tr><td class="patent-data-table-td ">15</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Better, M., et al., "<a href='http://scholar.google.com/scholar?q="T+Cell-targeted%2C+Immunofusion+Proteins+from+Escherichia+coli%2C"'>T Cell-targeted, Immunofusion Proteins from Escherichia coli,</a>" J. Biol. Chem. 270:14951-14957, American Society for Biochemistry and Molecular Biology, Inc. (Jun. 1995).</td></tr><tr><td class="patent-data-table-td ">16</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Blakey, D.C., et al., "<a href='http://scholar.google.com/scholar?q="Antibody+Toxin+Conjugates%3A+A+Perspective%2C"'>Antibody Toxin Conjugates: A Perspective,</a>" Monoclonal Antibody Therapy Prog. Allergy 45:50-90, Karger (1988).</td></tr><tr><td class="patent-data-table-td ">17</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Bolognesi, A., et al., "<a href='http://scholar.google.com/scholar?q="A+comparison+of+anti-lymphoctye+immunotoxins+containing+different+ribosome-inactivating+proteins+and+antibodies%2C"'>A comparison of anti-lymphoctye immunotoxins containing different ribosome-inactivating proteins and antibodies,</a>" Clin. Exp. Immunol. 89:341-346, Blackwell Scientific Publications (Sep. 1992).</td></tr><tr><td class="patent-data-table-td ">18</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Brown, D., et al., "<a href='http://scholar.google.com/scholar?q="Customizing+Lab+Mice%2C"'>Customizing Lab Mice,</a>" The Washington Post:A1 and A4, The Washington Post (Aug. 21, 1992).</td></tr><tr><td class="patent-data-table-td ">19</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Byers, V.S., et al., "<a href='http://scholar.google.com/scholar?q="Use+of+an+Anti-Pan+T-Lymphocyte+Ricin+A+Chain+Immunotoxin+in+Steroid-Resistant+Acute+Graft-Versus-Host+Disease%2C"'>Use of an Anti-Pan T-Lymphocyte Ricin A Chain Immunotoxin in Steroid-Resistant Acute Graft-Versus-Host Disease,</a>" Blood 75:1426-1432, W. B. Saunder (Apr. 1990).</td></tr><tr><td class="patent-data-table-td ">20</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Caron, P.C., et al., "<a href='http://scholar.google.com/scholar?q="Engineered+Humanized+Dimeric+Forms+of+IgG+Are+More+Effective+Antibodies%2C"'>Engineered Humanized Dimeric Forms of IgG Are More Effective Antibodies,</a>" J. Exp. Med. 176:1191-1195, The Rockefeller University Press (Oct. 1992).</td></tr><tr><td class="patent-data-table-td ">21</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Casellas, P., et al., "<a href='http://scholar.google.com/scholar?q="Optimal+Elimination+of+Leukemic+T+Cells+from+Human+Bone+Marrow+With+T101-Ricin+A-Chain+Immunotoxin%2C"'>Optimal Elimination of Leukemic T Cells from Human Bone Marrow With T101-Ricin A-Chain Immunotoxin,</a>" Blood 65:289-297, W. B. Saunders (Feb. 1985).</td></tr><tr><td class="patent-data-table-td ">22</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Chaudhary, V.K., et al., "<a href='http://scholar.google.com/scholar?q="A+rapid+method+of+cloning+functional+variable-region+antibody+genes+in+Escherichia+coli+as+single-chain+immunotoxins%2C"'>A rapid method of cloning functional variable-region antibody genes in Escherichia coli as single-chain immunotoxins,</a>" Proc. Natl. Acad. Sci. USA 87:1066-1070, National Academy of Sciences of the USA (Feb. 1990).</td></tr><tr><td class="patent-data-table-td ">23</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Chaudhary, V.K., et al., "<a href='http://scholar.google.com/scholar?q="A+recombinant+immunotoxin+consisting+of+two+antibody+variable+domains+fused+to+Pseudomonas+exotoxin%2C"'>A recombinant immunotoxin consisting of two antibody variable domains fused to Pseudomonas exotoxin,</a>" Nature 339:394-397, Macmillan Publishers Ltd. (Jun. 1989).</td></tr><tr><td class="patent-data-table-td ">24</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Cheadle, C., et al., "<a href='http://scholar.google.com/scholar?q="Cloning+and+Expression+of+the+Variable+Regions+of+Mouse+Myeloma+Protein+MOPC315+in+E.+Coli%3A+Recovery+of+Active+Fv+Fragments%2C"'>Cloning and Expression of the Variable Regions of Mouse Myeloma Protein MOPC315 in E. Coli: Recovery of Active Fv Fragments,</a>" Mol. Immunol. 29:21-30, Pergammon Press (Jan. 1992).</td></tr><tr><td class="patent-data-table-td ">25</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Chow, T., et al., "<a href='http://scholar.google.com/scholar?q="Isolation+and+DNA+Sequence+of+a+Gene+Encoding+alpha-Trichosanthin%2C+a+Type+1+Ribosome-inactivating+Protein%2C"'>Isolation and DNA Sequence of a Gene Encoding alpha-Trichosanthin, a Type 1 Ribosome-inactivating Protein,</a>" J. Biol. Chem. 265:8670-8674, American Society for Biochemistry and Molecular Biology, Inc. (May 1990).</td></tr><tr><td class="patent-data-table-td ">26</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Chow, T., et al., "<a href='http://scholar.google.com/scholar?q="Isolation+and+DNA+Sequence+of+a+Gene+Encoding+%CE%B1-Trichosanthin%2C+a+Type+1+Ribosome-inactivating+Protein%2C"'>Isolation and DNA Sequence of a Gene Encoding α-Trichosanthin, a Type 1 Ribosome-inactivating Protein,</a>" J. Biol. Chem. 265:8670-8674, American Society for Biochemistry and Molecular Biology, Inc. (May 1990).</td></tr><tr><td class="patent-data-table-td ">27</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Co, M.S., et al., "<a href='http://scholar.google.com/scholar?q="Humanized+antibodies+for+antiviral+therapy%2C"'>Humanized antibodies for antiviral therapy,</a>" Proc. Nat. Acad. Sci. USA 88:2869-2873, National Academy of Sciences of the USA (Apr. 1991).</td></tr><tr><td class="patent-data-table-td ">28</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Colombatti, M., et al., "<a href='http://scholar.google.com/scholar?q="Selective+Killing+of+Target+Cells+By+Antibody-Ricin+A+Chain+or+Antibody-Gelonin+Hybrid+Molecules%3A+Comparison+of+Cytotoxic+Potency+and+Use+in+Immunoselection+Procedures%2C"'>Selective Killing of Target Cells By Antibody-Ricin A Chain or Antibody-Gelonin Hybrid Molecules: Comparison of Cytotoxic Potency and Use in Immunoselection Procedures,</a>" J. Immunol. 131:3091-3095, American Association of Immunologists (Dec. 1983).</td></tr><tr><td class="patent-data-table-td ">29</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Delprino, L., et al., "<a href='http://scholar.google.com/scholar?q="Toxin-Targeted+Design+for+Anticancer+Therapy.+II%3A+Preparation+and+Biological+Comparison+of+Different+Chemically+Linked+Gelonin-Antibody+Conjugates%2C"'>Toxin-Targeted Design for Anticancer Therapy. II: Preparation and Biological Comparison of Different Chemically Linked Gelonin-Antibody Conjugates,</a>" Journal of Pharm. Sci. 82:699-704, American Pharmaceutical Association (Jul. 1993).</td></tr><tr><td class="patent-data-table-td ">30</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Descotes, G., et al., "<a href='http://scholar.google.com/scholar?q="The+Immunological+Activity+of+Plant+Toxins+Used+in+the+Preparation+of+Immunotoxins-II.+The+Immunodepressive+Activity+of+Gelonin%2C"'>The Immunological Activity of Plant Toxins Used in the Preparation of Immunotoxins-II. The Immunodepressive Activity of Gelonin,</a>" J. Immunopharm. 7:455-463, International Society for Immunopharmacology (1985).</td></tr><tr><td class="patent-data-table-td ">31</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Descotes, G., et al., "<a href='http://scholar.google.com/scholar?q="The+Immunological+Activity+of+Plant+Toxins+Used+in+the+Preparation+of+Immunotoxins%E2%80%94II.+The+Immunodepressive+Activity+of+Gelonin%2C"'>The Immunological Activity of Plant Toxins Used in the Preparation of Immunotoxins—II. The Immunodepressive Activity of Gelonin,</a>" J. Immunopharm. 7:455-463, International Society for Immunopharmacology (1985).</td></tr><tr><td class="patent-data-table-td ">32</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Dialog File 351, Derwent World Patents Index, English Lanugage abstract for EP 192002 A1, WPI Accession No. 86-226683.</td></tr><tr><td class="patent-data-table-td ">33</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Dower, W.J., et al., "<a href='http://scholar.google.com/scholar?q="High+efficiency+transformation+of+E.+coli+by+high+voltage+electroporation%2C"'>High efficiency transformation of E. coli by high voltage electroporation,</a>" Nucleic Acids Res. 16:6127-6145, IRL Press Limited (1988).</td></tr><tr><td class="patent-data-table-td ">34</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Ebert, R.F., and Spryn, L.A., "<a href='http://scholar.google.com/scholar?q="Immunotoxin+Construction+with+a+Ribosome-Inactivating+Protein+from+Barley%2C"'>Immunotoxin Construction with a Ribosome-Inactivating Protein from Barley,</a>" Bioconjugate Chem. 1:331-336, American Chemical Society (Feb. 1990).</td></tr><tr><td class="patent-data-table-td ">35</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Ellman, G.L., "<a href='http://scholar.google.com/scholar?q="Tissue+Sulfhydryl+Groups%2C"'>Tissue Sulfhydryl Groups,</a>" Arch. Biochem. Biophys. 82:70-77, Academic Press (1959).</td></tr><tr><td class="patent-data-table-td ">36</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Fishwild, D.M., et al., "<a href='http://scholar.google.com/scholar?q="Characterization+of+the+increased+cytotoxicity+of+gelonin+anti-T+cell+immunoconjugates+compared+with+ricin+A+chain+immunoconjugates%2C"'>Characterization of the increased cytotoxicity of gelonin anti-T cell immunoconjugates compared with ricin A chain immunoconjugates,</a>" Clin. Exp. Immunol. 97:10-18, Blackwell Scientific Publications (Jul. 1994).</td></tr><tr><td class="patent-data-table-td ">37</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Fishwild, D.M., et al., "<a href='http://scholar.google.com/scholar?q="Cytotoxicity+against+human+peripheral+blood+mononuclear+cells+and+T+cell+lines+mediated+by+anti-t+cell+immunotoxins+in+the+absence+of+added+potentiator%2C"'>Cytotoxicity against human peripheral blood mononuclear cells and T cell lines mediated by anti-t cell immunotoxins in the absence of added potentiator,</a>" Clin. Exp. Immunol. 86:506-513, Blackwell Scientific Publisher (Dec. 1991).</td></tr><tr><td class="patent-data-table-td ">38</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Glockshuber, R., et al., "<a href='http://scholar.google.com/scholar?q="A+Comparison+of+Strategies+To+Stabliize+Immunoglobulin+Fv-Fragments%2C"'>A Comparison of Strategies To Stabliize Immunoglobulin Fv-Fragments,</a>" Biochem. 29:1362-1367, American Chemical Society (Feb. 1990).</td></tr><tr><td class="patent-data-table-td ">39</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Goff, D.A., and Carrol. S.F., "<a href='http://scholar.google.com/scholar?q="Substituted+2-Iminothiolanes%3A+Reagents+for+the+Preparation+of+Disulfide+Cross-Linked+Conjugates+with+Increased+Stability%2C"'>Substituted 2-Iminothiolanes: Reagents for the Preparation of Disulfide Cross-Linked Conjugates with Increased Stability,</a>" Bioconjugate Chem. 1:381-386, American Chemical Society (Jan. 1990).</td></tr><tr><td class="patent-data-table-td ">40</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Habuka, N., et al., "<a href='http://scholar.google.com/scholar?q="Amino+Acid+Sequence+of+Mirabilis+Antiviral+Proteins%2C+Total+Synthesis+of+Its+Gene+and+Expression+in+Escherichia+coli%2C"'>Amino Acid Sequence of Mirabilis Antiviral Proteins, Total Synthesis of Its Gene and Expression in Escherichia coli,</a>" J. Biol. Chem. 264:6629-6637, American Society for Biochemistry and Molecular Biology, Inc. (Apr. 1989).</td></tr><tr><td class="patent-data-table-td ">41</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Halling, K.C., et al.,"<a href='http://scholar.google.com/scholar?q="Genomic+cloning+and+characterization+of+a+ricin+gene+from+Ricinus+communis%2C"'>Genomic cloning and characterization of a ricin gene from Ricinus communis,</a>" Nucleic Acids Res. 13:8019-8033, IRL Press Limited (1985).</td></tr><tr><td class="patent-data-table-td ">42</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Harris, P., et al., "<a href='http://scholar.google.com/scholar?q="In+Vitro+Studies+of+the+Effect+of+MAb+NDA+4+Linked+to+Toxin+on+the+Proliferation+of+a+Human+EBV-Transformed+Lymphoblastoid+B+Cell+Line+and+of+Gibbon+MLA+Leukemia+Cell+Line%2C"'>In Vitro Studies of the Effect of MAb NDA 4 Linked to Toxin on the Proliferation of a Human EBV-Transformed Lymphoblastoid B Cell Line and of Gibbon MLA Leukemia Cell Line,</a>" Immunol. 134:85-95, Academic Press, Inc. (Apr. 1991).</td></tr><tr><td class="patent-data-table-td ">43</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Hertler, A. A., and Frankel, A.E., "<a href='http://scholar.google.com/scholar?q="Immunotoxins%3A+A+Clinical+Review+of+Their+Use+in+the+Treatment+of+Malignancies%2C"'>Immunotoxins: A Clinical Review of Their Use in the Treatment of Malignancies,</a>" J. Clin. Oncol. 7:1932-1943, American Society of Clinical Oncology (Dec. 1989).</td></tr><tr><td class="patent-data-table-td ">44</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Ho, W.K.K., et al., "<a href='http://scholar.google.com/scholar?q="Cloning+of+the+cDNA+of+alpha-momorcharin%3A+a+ribosome+inactivating+protein%2C"'>Cloning of the cDNA of alpha-momorcharin: a ribosome inactivating protein,</a>" Biochem. Biophys. Acta 1088:311-314, Elsevier Science Publishers B.V. (Feb. 1991).</td></tr><tr><td class="patent-data-table-td ">45</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Ho, W.K.K., et al., "<a href='http://scholar.google.com/scholar?q="Cloning+of+the+cDNA+of+%CE%B1-momorcharin%3A+a+ribosome+inactivating+protein%2C"'>Cloning of the cDNA of α-momorcharin: a ribosome inactivating protein,</a>" Biochem. Biophys. Acta 1088:311-314, Elsevier Science Publishers B.V. (Feb. 1991).</td></tr><tr><td class="patent-data-table-td ">46</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Huston, J.S., et al., "<a href='http://scholar.google.com/scholar?q="Protein+engineering+of+antibody+binding+sites%3A+Recovery+of+specific+activity+in+an+anti-digoxin+single-chain+Fv+analogue+produced+in+Escherichia+coli%2C"'>Protein engineering of antibody binding sites: Recovery of specific activity in an anti-digoxin single-chain Fv analogue produced in Escherichia coli,</a>" Proc. Natl. Acad. Sci. USA 85:5879-5883, National Academy of Science USA (Aug. 1988).</td></tr><tr><td class="patent-data-table-td ">47</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Islam, M.R., et al., "<a href='http://scholar.google.com/scholar?q="Complete+Amino+Acid+Sequence+of+Luffin-a%2C+a+Ribosome-inactivating+Protein+from+the+Seeds+of+Luffa+cylindrica%2C"'>Complete Amino Acid Sequence of Luffin-a, a Ribosome-inactivating Protein from the Seeds of Luffa cylindrica,</a>" Agric. Biol. Chem. 54:1343-1345, Agricultural Chemical Society of Japan (May 1990).</td></tr><tr><td class="patent-data-table-td ">48</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Izzo, P., et al., "<a href='http://scholar.google.com/scholar?q="Human+aldolase+A+gene%2C"'>Human aldolase A gene,</a>" Eur. J. Biochem. 174:569-578, The Federation of American Societies for Experimental Biology (1988).</td></tr><tr><td class="patent-data-table-td ">49</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Jones, P.J., et al., "<a href='http://scholar.google.com/scholar?q="Replacing+the+complmentarity-determining+regions+in+a+human+antibody+with+those+from+a+mouse%2C"'>Replacing the complmentarity-determining regions in a human antibody with those from a mouse,</a>" Nature 321:522-525, Macmillan Publishers Ltd. (May 1986).</td></tr><tr><td class="patent-data-table-td ">50</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Junghans, R.P., et al., "<a href='http://scholar.google.com/scholar?q="Anti-Tac-H%2C+a+Humanized+Antibody+to+the+Interleukin+2+Receptor+with+New+Features+for+Immunotherapy+in+Malignant+and+Immune+Disorders%2C"'>Anti-Tac-H, a Humanized Antibody to the Interleukin 2 Receptor with New Features for Immunotherapy in Malignant and Immune Disorders,</a>" Cancer Res. 50:1495-1502, American Association For Cancer Research (Mar. 1990).</td></tr><tr><td class="patent-data-table-td ">51</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Katzin, B.J., et al., "<a href='http://scholar.google.com/scholar?q="Structure+of+Ricin+A-Chain+at+2.5+%C3%85%2C+"'>Structure of Ricin A-Chain at 2.5 Å, </a>" Proteins: Structure, Function and Genetics 10:251-259, Wiley-Liss, Inc. (Jun. 1991).</td></tr><tr><td class="patent-data-table-td ">52</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Katzin, B.J., et al., "<a href='http://scholar.google.com/scholar?q="Structure+of+Ricin+A-Chain+at+2.5+%E2%84%AB%2C+"'>Structure of Ricin A-Chain at 2.5 Å, </a>" Proteins: Structure, Function and Genetics 10:251-259, Wiley-Liss, Inc. (Jun. 1991).</td></tr><tr><td class="patent-data-table-td ">53</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Kernan, N.A., et al., "<a href='http://scholar.google.com/scholar?q="Specific+Inhibition+of+In+Vitro+Lymphocyte+Transformation+By+An+Anti-Pan+T+Cell+%28gp67%29+Ricin+A+Chain+Immunotoxin%2C"'>Specific Inhibition of In Vitro Lymphocyte Transformation By An Anti-Pan T Cell (gp67) Ricin A Chain Immunotoxin,</a>" J. Immunol. 133:137-146, American Association of Immunologists (Jul. 1984).</td></tr><tr><td class="patent-data-table-td ">54</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Kneble, K.D., et al., "<a href='http://scholar.google.com/scholar?q="Quantitative+Equilibrium+Binding+of+Monoclonal+Antibodies%2C+Immunotoxins%2C+and+Radioimmunoimaging+Agents+by+Flow+Cytometry%2C"'>Quantitative Equilibrium Binding of Monoclonal Antibodies, Immunotoxins, and Radioimmunoimaging Agents by Flow Cytometry,</a>" Cytometry Suppl. 1:68, Alan R. Liss, Inc. (1987).</td></tr><tr><td class="patent-data-table-td ">55</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Knowles, P.P., et al., "<a href='http://scholar.google.com/scholar?q="Purification+of+Immunotoxins+Containing+Ricin+A-Chain+and+Abrin+A-Chain+Using+Blue+Sepharose+CL-6B%2C"'>Purification of Immunotoxins Containing Ricin A-Chain and Abrin A-Chain Using Blue Sepharose CL-6B,</a>" Anal. Biochem. 160:440-443, Academic Press, Inc. (1987).</td></tr><tr><td class="patent-data-table-td ">56</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Kohn, F.R., et al., "<a href='http://scholar.google.com/scholar?q="Efficacy+of+Anti-CD5+F%28Ab%27%292+and+FAB%27+Immunoconjugates+in+Human+Peripheral+Blood+Lymphocyte-Reconstituted+Severe+Combined+Immunodeficient+Mice%2C"'>Efficacy of Anti-CD5 F(Ab')2 and FAB' Immunoconjugates in Human Peripheral Blood Lymphocyte-Reconstituted Severe Combined Immunodeficient Mice,</a>" Int. J. Immunopharmac. 15:871-878, Pergamon Press, Ltd. (Nov. 1993).</td></tr><tr><td class="patent-data-table-td ">57</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Kohr., W.J., et al., "<a href='http://scholar.google.com/scholar?q="The+Amino+Acid+Sequence+of+Gelonin."'>The Amino Acid Sequence of Gelonin.</a>" The Protein Society, 4th Symposium, San Diego, CA, Abstract No. T15 (Aug. 1990).</td></tr><tr><td class="patent-data-table-td ">58</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Kong, K., et al., "<a href='http://scholar.google.com/scholar?q="Subacute+Intravenous+Toxicity+of+a+Ribosomal+Inhibitory+Protein%2C+rGelonin%2C+in+Rats%2C"'>Subacute Intravenous Toxicity of a Ribosomal Inhibitory Protein, rGelonin, in Rats,</a>" (Poster) American College of Toxicology, San Francisco, CA (Oct. 1992).</td></tr><tr><td class="patent-data-table-td ">59</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Kung, S.-S., "<a href='http://scholar.google.com/scholar?q="The+Complete+Amino+Acid+Sequence+of+Antiviral+Protein+from+the+Seeds+of+Pokeweed+%28Phytolacca+americana%29%2C"'>The Complete Amino Acid Sequence of Antiviral Protein from the Seeds of Pokeweed (Phytolacca americana),</a>" Agric. Biol. Chem. 54:3301-3318, Agricultural Chemical Society of Japan (Sep. 1990).</td></tr><tr><td class="patent-data-table-td ">60</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Lambert, J.M., et al., "<a href='http://scholar.google.com/scholar?q="Purified+Immunotoxins+That+Are+Reactive+with+Human+Lymphoid+Cells%2C"'>Purified Immunotoxins That Are Reactive with Human Lymphoid Cells,</a>" J. Biol. Chem. 260:12035-12041, American Society for Biochemistry and Molecular Biology, Inc. (Oct. 1985).</td></tr><tr><td class="patent-data-table-td ">61</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Laurent, G., et al., "<a href='http://scholar.google.com/scholar?q="Donor+bone+marrow+treatment+with+T101+Fab+fragment-ricin+A-chain+immunotoxin+prevents+graf-versus-host+disease%2C"'>Donor bone marrow treatment with T101 Fab fragment-ricin A-chain immunotoxin prevents graf-versus-host disease,</a>" Bone Morrow Transplantation 4:367-371, Macmillan Press Ltd. (1989).</td></tr><tr><td class="patent-data-table-td ">62</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Lee-Huang, S.,et al., "<a href='http://scholar.google.com/scholar?q="A+new+class+of+anti-HIV+agents%3B+GAP31%2C+DAPs+30+and+32%2C"'>A new class of anti-HIV agents; GAP31, DAPs 30 and 32,</a>" FEBS 291:139-144, Elsevier Sicence Publishers B.V. (Oct. 1991).</td></tr><tr><td class="patent-data-table-td ">63</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Li, S.S.-L., et al., "<a href='http://scholar.google.com/scholar?q="Purification+and+partial+characterization+of+two+lectins+from+Momordica+charantia%2C"'>Purification and partial characterization of two lectins from Momordica charantia,</a>" Experienta 36:524-527, Birkhäuser Verlag (1980).</td></tr><tr><td class="patent-data-table-td ">64</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Madan, S., and Ghosh, P.C., "<a href='http://scholar.google.com/scholar?q="Interaction+of+Gelonin+With+Macrophages%3A+Effect+of+Lysosomotropic+Amines%2C"'>Interaction of Gelonin With Macrophages: Effect of Lysosomotropic Amines,</a>" Exper. Cell Res. 198:52-58, Academic Press, Inc. (Jan. 1992).</td></tr><tr><td class="patent-data-table-td ">65</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Montecucchi, P.C., et al., "<a href='http://scholar.google.com/scholar?q="N-terminal+sequence+of+some+ribosome-inactivating+proteins%2C"'>N-terminal sequence of some ribosome-inactivating proteins,</a>" Int. J. Peptide Protein Res. 33:263-267, Munksgaard (1989).</td></tr><tr><td class="patent-data-table-td ">66</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Morishima, Y., et al., "<a href='http://scholar.google.com/scholar?q="Functionally+Different+T+Lymphocyte+Subpopulations+Determined+By+Their+Sensitivity+to+Complement-Dependent+Cell+Lysis+With+the+Monoclonal+Antibody+4A%2C"'>Functionally Different T Lymphocyte Subpopulations Determined By Their Sensitivity to Complement-Dependent Cell Lysis With the Monoclonal Antibody 4A,</a>" J. Immunol. 129:1091-1098, American Association of Immunologists (Sep. 1982).</td></tr><tr><td class="patent-data-table-td ">67</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Mountain, A., and Adair, J.A., "<a href='http://scholar.google.com/scholar?q="Engineering+Antibodies+for+Therapy%2C"'>Engineering Antibodies for Therapy,</a>" in Biotechnology and Genetic Engineering Reviews 10:1-142, Intercept Ltd., Andover, U.K. (Dec. 1992).</td></tr><tr><td class="patent-data-table-td ">68</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Munson, P.J., et al., "<a href='http://scholar.google.com/scholar?q="Ligand%3A+A+Versatile+Computerized+Approach+for+Characterization+of+Ligand-Binding+Systems%2C"'>Ligand: A Versatile Computerized Approach for Characterization of Ligand-Binding Systems,</a>" Anal. Biochem. 107:220-239, Academic Press, Inc. (1980).</td></tr><tr><td class="patent-data-table-td ">69</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Myers, E.W., and Miller, W., "<a href='http://scholar.google.com/scholar?q="Optimal+alignments+in+linear+space%2C"'>Optimal alignments in linear space,</a>" Cabios Commun. 4:11-17, IRL Press Limited (1988).</td></tr><tr><td class="patent-data-table-td ">70</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Nolan, P.A., et al., "<a href='http://scholar.google.com/scholar?q="Cloning+and+expression+of+a+gene+encoding+gleonin%2C+a+ribosome-inactivating+protein+from+Gelonium+multiflorum%2C"'>Cloning and expression of a gene encoding gleonin, a ribosome-inactivating protein from Gelonium multiflorum,</a>" Gene 134:223-227, Elsevier Science Publishers B.V. (Dec. 1993).</td></tr><tr><td class="patent-data-table-td ">71</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Ogata, M., et al., "<a href='http://scholar.google.com/scholar?q="Processing+of+Pseudomonas+Exotoxin+by+a+Cellular+Protease+Results+in+the+Generation+of+a+37%2C000-Da+Toxin+Fragment+That+is+Translocated+to+the+Cytosol%2C"'>Processing of Pseudomonas Exotoxin by a Cellular Protease Results in the Generation of a 37,000-Da Toxin Fragment That is Translocated to the Cytosol,</a>" J. Biol. Chem. 265:20678-20685, American Society for Biochemistry and Molecular Biology, Inc. (Nov. 1990).</td></tr><tr><td class="patent-data-table-td ">72</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">O'Hare, M., et al., "<a href='http://scholar.google.com/scholar?q="Cytotoxicity+of+a+recombinant+ricin-A-chain+fusion+protein+containing+a+proteolytically-cleavable+spacer+sequence%2C"'>Cytotoxicity of a recombinant ricin-A-chain fusion protein containing a proteolytically-cleavable spacer sequence,</a>" FEBS Lett. 273:200-204, Elsevier Science Publishers B.V. (Oct. 1990).</td></tr><tr><td class="patent-data-table-td ">73</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Oldham, R.K., et al., "<a href='http://scholar.google.com/scholar?q="Adriamycin+custom-tailored+immunoconjugates+in+the+treatment+of+human+malignancies%2C"'>Adriamycin custom-tailored immunoconjugates in the treatment of human malignancies,</a>" Mol. Biother. 1:103-113, Butterworth Publishers (1988).</td></tr><tr><td class="patent-data-table-td ">74</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Pastan, I., and FitzGerald, D., "<a href='http://scholar.google.com/scholar?q="Recombinant+Toxins+for+Cancer+Treatment%2C"'>Recombinant Toxins for Cancer Treatment,</a>" Science 254:1173-1177, Association for the Advancement of Science (Nov. 1991).</td></tr><tr><td class="patent-data-table-td ">75</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Pending Non-Provisional United States Patent Application No. 09/711,485, Better et al., filed Nov. 13, 2000.</td></tr><tr><td class="patent-data-table-td ">76</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Potter, H., et al., "<a href='http://scholar.google.com/scholar?q="Enhancer-dependent+expression+of+human+kappa+immunoglobulin+genes+introduced+into+mouse+pre-B+lymphocytes+by+electroporation%2C"'>Enhancer-dependent expression of human kappa immunoglobulin genes introduced into mouse pre-B lymphocytes by electroporation,</a>" Proc. Natl. Acad. Sci. USA 81:7161-7165, National Academy of Sciences of the USA (Nov. 1984).</td></tr><tr><td class="patent-data-table-td ">77</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Potter, H., et al., "<a href='http://scholar.google.com/scholar?q="Enhancer-dependent+expression+of+human+%CE%BA+immunoglobulin+genes+introduced+into+mouse+pre-B+lymphocytes+by+electroporation%2C"'>Enhancer-dependent expression of human κ immunoglobulin genes introduced into mouse pre-B lymphocytes by electroporation,</a>" Proc. Natl. Acad. Sci. USA 81:7161-7165, National Academy of Sciences of the USA (Nov. 1984).</td></tr><tr><td class="patent-data-table-td ">78</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Preijers, F.W.M.B., et al., "<a href='http://scholar.google.com/scholar?q="Autologous+Transplantation+of+Bone+Marrow+Purged+In+Vitro+With+Anti-CD7-%28WT1-%29+Ricin+A+Immunotoxin+in+T-Cell+Lymphoblastic+Leukemia+and+Lymphoma%2C"'>Autologous Transplantation of Bone Marrow Purged In Vitro With Anti-CD7-(WT1-) Ricin A Immunotoxin in T-Cell Lymphoblastic Leukemia and Lymphoma,</a>" Blood 74:1152-1158, Grune &amp; Stratton, Inc. (Aug. 1989).</td></tr><tr><td class="patent-data-table-td ">79</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Press, O.W., et al., "<a href='http://scholar.google.com/scholar?q="A+simplified+microassay+for+inhibition+of+protein+synthesis+in+reticulocyte+lysates+by+immunotoxins%2C"'>A simplified microassay for inhibition of protein synthesis in reticulocyte lysates by immunotoxins,</a>" Immunol. Letters 14:37-41, Elsevier Science Publishers B.V. (1986).</td></tr><tr><td class="patent-data-table-td ">80</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Queen, C., et al., "<a href='http://scholar.google.com/scholar?q="A+humanized+antibody+that+binds+to+the+interleukin+2+receptor%2C"'>A humanized antibody that binds to the interleukin 2 receptor,</a>" Proc. Natl. Acad. Sci. USA 86:10029-10033, National Academy of Sciences of the USA (Dec. 1989).</td></tr><tr><td class="patent-data-table-td ">81</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Ready, M., et al., "<a href='http://scholar.google.com/scholar?q="Ricin-like+Plant+Toxins+are+Evolutionarily+Related+to+Single-chain+Ribosome-inhibiting+Proteins+from+Phytolacca%2C"'>Ricin-like Plant Toxins are Evolutionarily Related to Single-chain Ribosome-inhibiting Proteins from Phytolacca,</a>" J. Biol. Chem. 259:15252-15256, American Society of Biological Chemists, Inc. (Dec. 1984).</td></tr><tr><td class="patent-data-table-td ">82</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Reimann, K. A., et al., "<a href='http://scholar.google.com/scholar?q="In+Vivo+Administration+of+Lymphocyte-specific+Monoclonal+Antibodies+in+Nonhuman+Primates%2C"'>In Vivo Administration of Lymphocyte-specific Monoclonal Antibodies in Nonhuman Primates,</a>" J. Clin. Invest. 82:129-138, American Society for Clinical Investigation, Inc. (Jul. 1988).</td></tr><tr><td class="patent-data-table-td ">83</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Riechmann, L., et al., "<a href='http://scholar.google.com/scholar?q="Reshaping+human+antibodies+for+therapy%2C"'>Reshaping human antibodies for therapy,</a>" Nature 322:323-327, Macmillan Publishers Ltd. (Mar. 1988).</td></tr><tr><td class="patent-data-table-td ">84</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Robinson, R.R., et al., "<a href='http://scholar.google.com/scholar?q="Chimeric+mouse-human+anti-carcinoma+antibodies+that+mediate+different+anti-tumor+cell+biological+activities%2C"'>Chimeric mouse-human anti-carcinoma antibodies that mediate different anti-tumor cell biological activities,</a>" Human Antibod. and Hybridomas 2:84-93, Butterworth-Heinemann (Apr. 1991).</td></tr><tr><td class="patent-data-table-td ">85</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Rosenblum, M.G., et al., "<a href='http://scholar.google.com/scholar?q="A+specific+and+potent+immunotoxin+composed+of+antibody+ZME-018+and+the+plant+toxin+gelonin%2C"'>A specific and potent immunotoxin composed of antibody ZME-018 and the plant toxin gelonin,</a>" Mol. Biother. 3:6-13, Butterworth-Heinemann (Mar. 1991).</td></tr><tr><td class="patent-data-table-td ">86</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Rostaing-Capaillon, O., and Casellas, P., "<a href='http://scholar.google.com/scholar?q="In+vivo+cytotoxic+efficacy+of+immunotoxins+prepared+from+anti-CD5+antibody+linked+to+ricin+A-chain%2C"'>In vivo cytotoxic efficacy of immunotoxins prepared from anti-CD5 antibody linked to ricin A-chain,</a>" Cancer Immunol. Immunother. 34:24-30, Springer-Verlag (Jan. 1991).</td></tr><tr><td class="patent-data-table-td ">87</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Rowley, S.D., et al., "<a href='http://scholar.google.com/scholar?q="Efficacy+of+Ex+Vivo+Purging+for+Autologous+Bone+Marrow+Transplantation+in+the+Treatment+of+Acute+Nonlymphoblastic+Leukemia%2C"'>Efficacy of Ex Vivo Purging for Autologous Bone Marrow Transplantation in the Treatment of Acute Nonlymphoblastic Leukemia,</a>" Blood 74:501-506, Grune &amp; Stratton, Inc. (Jul. 1989).</td></tr><tr><td class="patent-data-table-td ">88</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Royston, I., et al., "<a href='http://scholar.google.com/scholar?q="Human+T+Cell+Antigens+Defined+By+Monoclonal+Antibodies%3A+The+65%2C000-Dalton+Antigen+of+T+Cells+%28T65%29+Is+Also+Found+On+Chronic+Lymphocytic+Leukemia+Cells+Bearing+Surface+Immunoglobulin%2C"'>Human T Cell Antigens Defined By Monoclonal Antibodies: The 65,000-Dalton Antigen of T Cells (T65) Is Also Found On Chronic Lymphocytic Leukemia Cells Bearing Surface Immunoglobulin,</a>" J. Immunol. 125:725-731, The Williams &amp; Wilkins Co. (Aug. 1980).</td></tr><tr><td class="patent-data-table-td ">89</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Scott, C.F., et al., "<a href='http://scholar.google.com/scholar?q="The+antileukemic+efficacy+of+an+immunotoxin+composed+of+a+monoclonal+anti-Thy-1+antibody+disulfide+linked+to+the+ribosome-inactivating+protein+gelonin%2C"'>The antileukemic efficacy of an immunotoxin composed of a monoclonal anti-Thy-1 antibody disulfide linked to the ribosome-inactivating protein gelonin,</a>" Cancer Immunol. Immunother 25:31-40, Springer-Verlag (Jul. 1987).</td></tr><tr><td class="patent-data-table-td ">90</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Shaw, P.-C., et al., "<a href='http://scholar.google.com/scholar?q="Cloning+of+trichosanthin+cDNA+and+its+expression+in+Escherichai+coli%2C"'>Cloning of trichosanthin cDNA and its expression in Escherichai coli,</a>" Gene 97:267-272, Elsevier (Jan. 1991).</td></tr><tr><td class="patent-data-table-td ">91</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Singh, V., and Curtiss III, R., "<a href='http://scholar.google.com/scholar?q="Hormonotoxins%3A+Abrogation+of+Ribosome+Inactivatinb+Property+of+Gelonin+in+the+Disulfide+Linked+Ovine+Lutenizing+Hormone-Gelonin+Conjugates%2C"'>Hormonotoxins: Abrogation of Ribosome Inactivatinb Property of Gelonin in the Disulfide Linked Ovine Lutenizing Hormone-Gelonin Conjugates,</a>" Biochem. Intl. 25:531-536, Academic Press Australia (Oct. 1991).</td></tr><tr><td class="patent-data-table-td ">92</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Sivam, G., et al., "<a href='http://scholar.google.com/scholar?q="Immjnotoxins+to+a+Human+Melanoma-associated+Antigen%3A+Comparison+of+Gelonin+With+Ricin+and+Other+A+Chain+Conjugates%2C"'>Immjnotoxins to a Human Melanoma-associated Antigen: Comparison of Gelonin With Ricin and Other A Chain Conjugates,</a>" Cancer Res. 47:3169-3173, American Association For Cancer Research (Jun. 1987).</td></tr><tr><td class="patent-data-table-td ">93</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Sperti, S., et al., "<a href='http://scholar.google.com/scholar?q="Requirements+for+the+inactivation+of+ribosomes+by+gelonin%2C"'>Requirements for the inactivation of ribosomes by gelonin,</a>" Biochem J. 227:281-284, London Portland Press On Behalf Of The Biochemical Society (Jul. 1991).</td></tr><tr><td class="patent-data-table-td ">94</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Stirpe, F., et al., "<a href='http://scholar.google.com/scholar?q="Gelonin%2C+a+New+Inhibitor+of+Protein+Synthesis%2C+Nontoxic+to+Intact+Cells%2C"'>Gelonin, a New Inhibitor of Protein Synthesis, Nontoxic to Intact Cells,</a>" J. Biol. Chem. 255:6947-6953, American Society for Biochemistry and Molecular Biology, Inc. (Jul. 1980).</td></tr><tr><td class="patent-data-table-td ">95</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Stirpe, F., et al., "<a href='http://scholar.google.com/scholar?q="Modification+of+ribosomal+RNA+by+ribosome-inactivating+proteins+from+plants%2C"'>Modification of ribosomal RNA by ribosome-inactivating proteins from plants,</a>" Nucleic Acids Research, 16:1349-1357, IRL Press Limited (Feb. 1988).</td></tr><tr><td class="patent-data-table-td ">96</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Stirpe, F., et al., "<a href='http://scholar.google.com/scholar?q="Ribosome-inactivating+proteins+up+to+date%2C"'>Ribosome-inactivating proteins up to date,</a>" FEBS Lett. 195:1-8, Elsevier Science Publishers B.V. (Jan. 1986).</td></tr><tr><td class="patent-data-table-td ">97</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Strockbine, N.A., et al., "<a href='http://scholar.google.com/scholar?q="Cloning+and+Sequencing+of+the+Genes+for+Shiga+Toxin+from+Shigella+dysenteria+Type+1%2C"'>Cloning and Sequencing of the Genes for Shiga Toxin from Shigella dysenteria Type 1,</a>" J. Bacteriol. 170:1116-1122, American Society for Microbiology (1988).</td></tr><tr><td class="patent-data-table-td ">98</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Thorpe, P.E., et al., "<a href='http://scholar.google.com/scholar?q="Improved+Antitumor+Effects+of+Immunotoxins+Prepared+with+Deglycosylated+Ricin+A-Chain+and+Hindered+Disulfide+Linkages%2C"'>Improved Antitumor Effects of Immunotoxins Prepared with Deglycosylated Ricin A-Chain and Hindered Disulfide Linkages,</a>" Cancer Res. 48:6396-6403, American Association For Cancer Research (Nov. 1988).</td></tr><tr><td class="patent-data-table-td ">99</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Thorpe, P.E., et al., "<a href='http://scholar.google.com/scholar?q="New+Coupling+Agents+for+the+Synthesis+of+Immunotoxins+Containing+a+Hindered+Disulfide+Bond+with+Improved+Stability+in+Vivo%2C"'>New Coupling Agents for the Synthesis of Immunotoxins Containing a Hindered Disulfide Bond with Improved Stability in Vivo,</a>" Cancer Res. 47:5924-5931 (1987).</td></tr><tr><td class="patent-data-table-td ">100</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Tolan, D.R., et al., "<a href='http://scholar.google.com/scholar?q="The+Complete+Nucleotide+Sequence+for+Rabbit+Muscle+Aldolase+A+Messenger+RNA%2C"'>The Complete Nucleotide Sequence for Rabbit Muscle Aldolase A Messenger RNA,</a>" J. Biol. Chem. 259:1127-1130, American Society for Biochemistry and Molecular Biology, Inc. (1984).</td></tr><tr><td class="patent-data-table-td ">101</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Vallera, D.A., "<a href='http://scholar.google.com/scholar?q="Immunotoxins%3A+Will+Their+Clinical+Promise+Be+Fulfilled%3F"'>Immunotoxins: Will Their Clinical Promise Be Fulfilled?</a>" Blood 83:309-317, American Society for Hematology (Jan. 1994).</td></tr><tr><td class="patent-data-table-td ">102</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Wawrzynczak, E.J., et al., "<a href='http://scholar.google.com/scholar?q="Pharmacokinetics+in+the+Rat+of+a+Panel+of+Immunotoxins+Made+With+Abrin+A+Chain%2C+Ricin+A+Chain%2C+Gelonin%2C+and+Momordin%2C"'>Pharmacokinetics in the Rat of a Panel of Immunotoxins Made With Abrin A Chain, Ricin A Chain, Gelonin, and Momordin,</a>" Cancer Res. 50:7519-7526, American Association For Cancer Research (Dec. 1990).</td></tr><tr><td class="patent-data-table-td ">103</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Westby, M., et al., "<a href='http://scholar.google.com/scholar?q="Preparation+and+Characterization+of+Recombinant+Proricin+Containing+an+Alternative+Protease-Sensitive+Linker+Sequence%2C"'>Preparation and Characterization of Recombinant Proricin Containing an Alternative Protease-Sensitive Linker Sequence,</a>" Bioconjugate Chem. 3:375-381, American Chemical Society (Sep. 1992).</td></tr><tr><td class="patent-data-table-td ">104</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Williams, D.P., et al., "<a href='http://scholar.google.com/scholar?q="Diphtheria+toxin+receptor+binding+domain+substitution+with+interleukin-2%3A+genetic+construction+and+properties+of+a+diphtheria+toxin-related+interleukin-2+fusion+protein%2C"'>Diphtheria toxin receptor binding domain substitution with interleukin-2: genetic construction and properties of a diphtheria toxin-related interleukin-2 fusion protein,</a>" Protein Engineering 1(6):493-498, Oxford University Press (1987).</td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="forward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Referenced by</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Citing Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6649742">US6649742</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Nov 13, 2000</td><td class="patent-data-table-td patent-date-value">Nov 18, 2003</td><td class="patent-data-table-td ">Xoma Technology Ltd.</td><td class="patent-data-table-td ">Fusion protein comprising gelonin and a antibody fused thereto; cytolysis; use in treatment of cancer, autoimmune and graft-versus-host disease</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7153932">US7153932</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Nov 18, 2003</td><td class="patent-data-table-td patent-date-value">Dec 26, 2006</td><td class="patent-data-table-td ">Research Development Foundation</td><td class="patent-data-table-td ">Immunotoxins comprising ribosome-inactivating proteins</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7754211">US7754211</a></td><td class="patent-data-table-td patent-date-value">Oct 13, 2004</td><td class="patent-data-table-td patent-date-value">Jul 13, 2010</td><td class="patent-data-table-td ">Research Development Foundation</td><td class="patent-data-table-td ">Immunotoxins directed against c-erbB-2(HER-2/neu) related surface antigens</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2008052322A1?cl=en">WO2008052322A1</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Oct 30, 2007</td><td class="patent-data-table-td patent-date-value">May 8, 2008</td><td class="patent-data-table-td ">Jeannick Cizeau</td><td class="patent-data-table-td ">Immunotoxγn fusions comprising an antibody fragment and a plant toxin linked by protease cleavable linkers</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="classifications"></a><div class="patent-section-header"><span class="patent-section-title">Classifications</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th"> </th><th class="patent-data-table-th"> </th></tr></thead><tr><td class="patent-data-table-td ">U.S. Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=CsRZBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S069100">435/69.1</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=CsRZBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc530/defs530.htm&usg=AFQjCNEOd3srpZKCSH0W93D5fiJ4_0S25w#C530S326000">530/326</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=CsRZBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S023600">536/23.6</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=CsRZBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc530/defs530.htm&usg=AFQjCNEOd3srpZKCSH0W93D5fiJ4_0S25w#C530S350000">530/350</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=CsRZBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S023200">536/23.2</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=CsRZBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc530/defs530.htm&usg=AFQjCNEOd3srpZKCSH0W93D5fiJ4_0S25w#C530S391900">530/391.9</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=CsRZBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S023400">536/23.4</a></span></td></tr><tr><td class="patent-data-table-td ">International Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=CsRZBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0001210000">C12N1/21</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=CsRZBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0015290000">C12N15/29</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=CsRZBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C07K0016000000">C07K16/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=CsRZBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C07K0016160000">C07K16/16</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=CsRZBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C07K0014415000">C07K14/415</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=CsRZBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=A61K0038000000">A61K38/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=CsRZBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C07K0016280000">C07K16/28</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=CsRZBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=A61K0047480000">A61K47/48</a></span></td></tr><tr><td class="patent-data-table-td ">Cooperative Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=CsRZBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C07K16/16">C07K16/16</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=CsRZBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C07K2317/54">C07K2317/54</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=CsRZBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C07K14/415">C07K14/415</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=CsRZBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=A61K38/00">A61K38/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=CsRZBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=A61K47/48561">A61K47/48561</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=CsRZBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C07K16/00">C07K16/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=CsRZBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C07K2317/55">C07K2317/55</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=CsRZBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C07K16/2896">C07K16/2896</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=CsRZBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C07K2317/24">C07K2317/24</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=CsRZBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=A61K47/48469">A61K47/48469</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=CsRZBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C07K2319/00">C07K2319/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=CsRZBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C07K16/28">C07K16/28</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=CsRZBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C07K2317/622">C07K2317/622</a></span></td></tr><tr><td class="patent-data-table-td ">European Classification</td><td class="patent-data-table-td "><span class="nested-value">C07K14/415</span>, <span class="nested-value">C07K16/28Z</span>, <span class="nested-value">A61K47/48T2C12P2F</span>, <span class="nested-value">C07K16/16</span>, <span class="nested-value">A61K47/48T4B28</span>, <span class="nested-value">C07K16/00</span>, <span class="nested-value">C07K16/28</span></td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="legal-events"></a><div class="patent-section-header"><span class="patent-section-title">Legal Events</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Date</th><th class="patent-data-table-th">Code</th><th class="patent-data-table-th">Event</th><th class="patent-data-table-th">Description</th></tr></thead><tr><td class="patent-data-table-td patent-date-value">Jun 10, 2014</td><td class="patent-data-table-td ">FP</td><td class="patent-data-table-td ">Expired due to failure to pay maintenance fee</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20140423</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Apr 23, 2014</td><td class="patent-data-table-td ">LAPS</td><td class="patent-data-table-td ">Lapse for failure to pay maintenance fees</td><td class="patent-data-table-td "></td></tr><tr><td class="patent-data-table-td patent-date-value">Nov 29, 2013</td><td class="patent-data-table-td ">REMI</td><td class="patent-data-table-td ">Maintenance fee reminder mailed</td><td class="patent-data-table-td "></td></tr><tr><td class="patent-data-table-td patent-date-value">Oct 23, 2009</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">8</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Oct 24, 2005</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">4</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Oct 12, 2005</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">RESEARCH DEVELOPMENT FOUNDATION, NEVADA</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNOR:XOMA TECHNOLOGY LTD.;REEL/FRAME:016634/0728</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20051003</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Jun 17, 2003</td><td class="patent-data-table-td ">RR</td><td class="patent-data-table-td ">Request for reexamination filed</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20030521</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Nov 20, 2001</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">XOMA LTD., BERMUDA</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">MERGER AND CHANGE OF NAME;ASSIGNORS:XOMA CORPORATION (MERGED INTO);XOMA ARIZONA, INC. (CHANGED TO);XOMA CORPORATION (CHANGED TO);REEL/FRAME:012312/0259;SIGNING DATES FROM 19981212 TO 19981229</span></div><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">XOMA TECHNOLOGY LTD., BERMUDA</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNOR:XOMA LTD.;REEL/FRAME:012312/0254</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">19991022</span></div><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">XOMA LTD. 2 CHURCH STREET CLARENDON HOUSE HAMILTON</span></div><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">XOMA LTD. 2 CHURCH STREET CLARENDON HOUSEHAMILTON,</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">MERGER AND CHANGE OF NAME;ASSIGNORS:XOMA CORPORATION (MERGED INTO) /AR;REEL/FRAME:012312/0259;SIGNING DATES FROM 19981212 TO 19981229</span></div><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">XOMA TECHNOLOGY LTD. CLARENDON HOUSE 2 CHURCH STRE</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNOR:XOMA LTD. /AR;REEL/FRAME:012312/0254</span></div></td></tr></table><div class="patent-section-footer"></div></div><div class="modal-dialog" id="patent-images-lightbox"><div class="patent-lightbox-controls"><div class="patent-lightbox-rotate-controls"><div class="patent-lightbox-rotation-text">Rotate</div><div class="rotate-icon rotate-ccw-icon"></div><div class="rotate-icon rotate-cw-icon"></div></div><div class="patent-lightbox-index-counter"></div><a class="patent-lightbox-fullsize-link" target="_blank">Original Image</a><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_right.png" alt="Next page"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_left.png" alt="Previous page"width="21" height="21" /></div></div><div class="modal-dialog-content"><div class="patent-lightbox-image-holder"><div class="patent-lightbox-placeholder"></div></div></div></div><script>_OC_initPatentsAtb({image_not_available_html: " Image not available"});</script></div></div></div></td></tr></table><script>(function() {var href = window.location.href;if (href.indexOf('?') !== -1) {var parameters = href.split('?')[1].split('&');for (var i = 0; i < parameters.length; i++) {var param = parameters[i].split('=');if (param[0] == 'focus') {var elem = document.getElementById(param[1]);if (elem) {elem.focus();}}}}})();</script><script>_OC_addFlags({LockSrc:"/books/javascript/lock_8a2b04e7bf975d5171d8e4c0b6365c7a.js", Host:"http://www.google.com/", IsBooksRentalEnabled:1, IsWebstoreDisplayCaseEnabled:1, IsObfuscationEnabled:1, IsBrowsingHistoryEnabled:1, IsWebReaderSvgEnabled:0, IsGeoLayerEnabled:1, IsImageModeNotesEnabled:1, IsCopyMenuItemEnabled:1, IsGiftingEnabled:0, IsWebReaderUniversalPaginatorEnabled:0, IsOfflineBubbleEnabled:1, IsReaderEnabledForPlayRequests:1, IsFutureOnSaleVolumesEnabled:1, IsOfflineRestrictedCopyEnabled:1, IsBooksUnifiedLeftNavEnabled:1, IsRestrictedCopyEnabled:1, IsZipitFolderCollectionEnabled:1, IsEndOfSampleRecommendationsEnabled:1, IsRatingsOnBookcardsEnabled:1, IsAdsDisabled:0, IsIframePageDisplayEnabled:0, IsEmbeddedMediaEnabled:1, IsImageModeAnnotationsEnabled:1, IsMyLibraryGooglePlusEnabled:1, IsImagePageProviderEnabled:0, IsBookcardListPriceSmall:0, IsInternalUser:0, IsBooksShareButtonEnabled:0, IsPreOrdersEnabled:0, IsDisabledRandomBookshelves:0, WebstoreDisplayCasePosition:3});_OC_Run({"enable_p13n":false,"add_vol_to_collection_base_url":"http://www.google.com/patents?op=add\u0026sig=ACfU3U1GF_LljpmcOx0YvA6mkwz__snpuw\u0026id=CsRZBAABERAJ","remove_vol_from_collection_base_url":"http://www.google.com/patents?op=remove\u0026sig=ACfU3U2dG7s2hXRqJDBRacnx3-2t82lSpQ\u0026id=CsRZBAABERAJ","logged_in":false,"p13n_save_user_settings_url":"http://www.google.com/patents?op=edit_user_settings\u0026sig=ACfU3U0n8qgg8sGX121EwQK79iC94wurMg","is_cobrand":false,"sign_in_url":"https://www.google.com/accounts/Login?service=\u0026continue=http://www.google.com/patents%3Fhl%3Den\u0026hl=en","is_play_enabled":true}, {"volume_id":"","is_ebook":true,"volumeresult":{"has_flowing_text":false,"has_scanned_text":true,"can_download_pdf":false,"can_download_epub":false,"is_pdf_drm_enabled":false,"is_epub_drm_enabled":false,"download_pdf_url":"http://www.google.com/patents/download/Fusion_protein_comprising_gelonin_amino.pdf?id=CsRZBAABERAJ\u0026output=pdf\u0026sig=ACfU3U1VwSVvtpff3ATqtW8diFQho0Iqgw"},"sample_url":"http://www.google.com/patents/reader?id=CsRZBAABERAJ\u0026printsec=frontcover\u0026output=reader\u0026source=gbs_atb_hover","is_browsable":true,"is_public_domain":true}, {});</script><div id="footer_table" style="font-size:83%;text-align:center;position:relative;top:20px;height:4.5em;margin-top:2em"><div style="margin-bottom:8px"><a href=http://www.google.com/><nobr>Google&nbsp;Home</nobr></a> - <a href=//www.google.com/patents/sitemap/><nobr>Sitemap</nobr></a> - <a href=http://www.google.com/googlebooks/uspto.html><nobr>USPTO Bulk Downloads</nobr></a> - <a href=/intl/en/privacy/><nobr>Privacy Policy</nobr></a> - <a href=/intl/en/policies/terms/><nobr>Terms of Service</nobr></a> - <a href=https://support.google.com/faqs/answer/2539193?hl=en><nobr>About Google Patents</nobr></a> - <a href="http://www.google.com/tools/feedback/intl/en/error.html" onclick="try{_OC_startFeedback({productId: '72792',locale: 'en'});return false;}catch(e){}"><nobr>Send Feedback</nobr></a></div><span>Data provided by IFI CLAIMS Patent Services</span><br><span >&copy;2012 Google</span></div> <script type="text/javascript">var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));</script><script type="text/javascript">var pageTracker = _gat._getTracker("UA-27188110-1");pageTracker._setCookiePath("/patents/");pageTracker._trackPageview();</script> </body></html>