<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN""http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html><head><title>Patent US6406848 - Microparticles and polynucleotides - Google Patents</title><script>(function(){(function(){function e(a){this.t={};this.tick=function(a,c,b){var d=void 0!=b?b:(new Date).getTime();this.t[a]=[d,c];if(void 0==b)try{window.console.timeStamp("CSI/"+a)}catch(e){}};this.tick("start",null,a)}var a;window.performance&&(a=window.performance.timing);var f=a?new e(a.responseStart):new e;window.jstiming={Timer:e,load:f};if(a){var c=a.navigationStart,d=a.responseStart;0<c&&d>=c&&(window.jstiming.srt=d-c)}if(a){var b=window.jstiming.load;0<c&&d>=c&&(b.tick("_wtsrt",void 0,c),b.tick("wtsrt_",
"_wtsrt",d),b.tick("tbsd_","wtsrt_"))}try{a=null,window.chrome&&window.chrome.csi&&(a=Math.floor(window.chrome.csi().pageT),b&&0<c&&(b.tick("_tbnd",void 0,window.chrome.csi().startE),b.tick("tbnd_","_tbnd",c))),null==a&&window.gtbExternal&&(a=window.gtbExternal.pageT()),null==a&&window.external&&(a=window.external.pageT,b&&0<c&&(b.tick("_tbnd",void 0,window.external.startE),b.tick("tbnd_","_tbnd",c))),a&&(window.jstiming.pt=a)}catch(g){}})();})();
</script><link rel="stylesheet" href="/patents/css/_8a2b04e7bf975d5171d8e4c0b6365c7a/kl_intl_patents_bundle.css" type="text/css" /><script src="/books/javascript/atb_8a2b04e7bf975d5171d8e4c0b6365c7a__en.js"></script><script>function googleTranslateElementInit() {new google.translate.TranslateElement({pageLanguage: "en",gaTrack: true,gaId: "UA-27188110-1",multilanguagePage: true});}</script><script src="//translate.google.com/translate_a/element.js?cb=googleTranslateElementInit"></script><meta name="DC.type" content="Patent"><meta name="DC.title" content="Microparticles and polynucleotides"><meta name="DC.contributor" content="John Bridgham" scheme="inventor"><meta name="DC.contributor" content="Kevin Corcoran" scheme="inventor"><meta name="DC.contributor" content="George Golda" scheme="inventor"><meta name="DC.contributor" content="Michael C. Pallas" scheme="inventor"><meta name="DC.contributor" content="Sydney Brenner" scheme="inventor"><meta name="DC.contributor" content="Lynx Therapeutics, Inc." scheme="assignee"><meta name="DC.date" content="1998-5-22" scheme="dateSubmitted"><meta name="DC.description" content="An apparatus and system are provided for simultaneously analyzing a plurality of analytes anchored to microparticles. Microparticles each having a uniform population of a single kind of analyte attached are disposed as a substantially immobilized planar array inside of a flow chamber where steps of an analytical process are carried out by delivering a sequence of processing reagents to the microparticles by a fluidic system under microprocessor control. In response to such process steps, an optical signal is generated at the surface of each microparticle which is characteristic of the interaction between the analyte carried by the microparticle and the delivered processing reagent. The plurality of analytes are simultaneously analyzed by collecting and recording images of the optical signals generated by all the microparticles in the planar array. A key feature of the invention is the correlation of the sequence of optical signals generated by each microparticle in the planar array during the analytical process."><meta name="DC.date" content="2002-6-18" scheme="issued"><meta name="DC.relation" content="US:5429807" scheme="references"><meta name="DC.relation" content="US:5708153" scheme="references"><meta name="citation_patent_number" content="US:6406848"><meta name="citation_patent_application_number" content="US:09/424,028"><link rel="canonical" href="http://www.google.com/patents/US6406848"/><meta property="og:url" content="http://www.google.com/patents/US6406848"/><meta name="title" content="Patent US6406848 - Microparticles and polynucleotides"/><meta name="description" content="An apparatus and system are provided for simultaneously analyzing a plurality of analytes anchored to microparticles. Microparticles each having a uniform population of a single kind of analyte attached are disposed as a substantially immobilized planar array inside of a flow chamber where steps of an analytical process are carried out by delivering a sequence of processing reagents to the microparticles by a fluidic system under microprocessor control. In response to such process steps, an optical signal is generated at the surface of each microparticle which is characteristic of the interaction between the analyte carried by the microparticle and the delivered processing reagent. The plurality of analytes are simultaneously analyzed by collecting and recording images of the optical signals generated by all the microparticles in the planar array. A key feature of the invention is the correlation of the sequence of optical signals generated by each microparticle in the planar array during the analytical process."/><meta property="og:title" content="Patent US6406848 - Microparticles and polynucleotides"/><meta property="og:type" content="book"/><meta property="og:site_name" content="Google Books"/><meta property="og:image" content="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><link rel="image_src" href="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><script>(function(){try{var aa=function(a,b,c,d){d=d||{};d._sn=["cfg",b,c].join(".");window.gbar.logger.ml(a,d)};var g=window.gbar=window.gbar||{},l=window.gbar.i=window.gbar.i||{},m={},n;function _tvn(a,b){var c=parseInt(a,10);return isNaN(c)?b:c}function _tvf(a,b){var c=parseFloat(a);return isNaN(c)?b:c}function _tvv(a){return!!a}function p(a,b,c){(c||g)[a]=b}g.bv={n:_tvn("2",0),r:"",f:".67.",e:"0",m:_tvn("0",1)};
function q(a,b,c){var d="on"+b;if(a.addEventListener)a.addEventListener(b,c,!1);else if(a.attachEvent)a.attachEvent(d,c);else{var f=a[d];a[d]=function(){var a=f.apply(this,arguments),b=c.apply(this,arguments);return void 0==a?b:void 0==b?a:b&&a}}}var s=function(a){return function(){return g.bv.m==a}},ba=s(1),ca=s(2);p("sb",ba);p("kn",ca);l.a=_tvv;l.b=_tvf;l.c=_tvn;l.i=aa;var da=window.gbar.i.i;var t,u,v,w;function ea(a){v=a}function fa(a){var b;if(b=v&&window.encodeURIComponent)b=a.href,b=!b.match(/^http[s]?:\/\/accounts\.google\.[^/]*\/ClearSID/i)&&!b.match(/^http[s]?:\/\/[^/]*\/accounts\/ClearSID/i);if(b=b&&encodeURIComponent(v()))a.href=a.href.replace(/([?&]continue=)[^&]*/,"$1"+b)}function ga(a){window.gApplication&&(a.href=window.gApplication.getTabUrl(a.href))}
function ha(a){var b=document.forms[0].q,c=window.encodeURIComponent&&b&&b.value,b=b&&b.placeholder;c&&c!=b&&(a.href=a.href.replace(/([?&])q=[^&]*|$/,function(a,b){return(b||"&")+"q="+encodeURIComponent(c)}))}n=l.a("")?ga:ha;
function x(a,b,c,d,f,e){var h=document.getElementById(a);if(h){var k=h.style;k.left=d?"auto":b+"px";k.right=d?b+"px":"auto";k.top=c+"px";k.visibility=u?"hidden":"visible";f&&e?(k.width=f+"px",k.height=e+"px"):(x(t,b,c,d,h.offsetWidth,h.offsetHeight),u=u?"":a)}}
var y=[],ia=function(a,b){y.push(b)},ja=function(a){a=a||window.event;var b=a.target||a.srcElement;a.cancelBubble=!0;null==t&&(a=document.createElement(Array.every||window.createPopup?"iframe":"div"),a.frameBorder="0",t=a.id="gbs",a.src="javascript:''",b.parentNode.appendChild(a),q(document,"click",z));var c=b,b=0;"gb3"!=c.className&&(c=c.parentNode);a=c.getAttribute("aria-owns")||"gbi";var d=c.offsetWidth,f=20<c.offsetTop?46:24;document.getElementById("tphdr")&&(f-=3);var e=!1;do b+=c.offsetLeft||
0;while(c=c.offsetParent);var c=(document.documentElement.clientWidth||document.body.clientWidth)-b-d,h,d=document.body,k=document.defaultView;k&&k.getComputedStyle?(d=k.getComputedStyle(d,""))&&(h=d.direction):h=d.currentStyle?d.currentStyle.direction:d.style.direction;h="rtl"==h;if("gbi"==a){for(d=0;k=y[d++];)k();A(null,window.navExtra);h&&(b=c,e=!0)}else h||(b=c,e=!0);u!=a&&z();x(a,b,f,e)},z=function(){u&&x(u,0,0)},A=function(a,b){var c,d=document.getElementById("gbi"),f=a;f||(f=d.firstChild);
for(;b&&(c=b.pop());){var e=d,h=c,k=f;w||(w="gb2");e.insertBefore(h,k).className=w}},ka=function(a,b,c){if((b=document.getElementById(b))&&a){a.className="gb4";var d=document.createElement("span");d.appendChild(a);d.appendChild(document.createTextNode(" | "));d.id=c;b.appendChild(d)}},la=function(){return document.getElementById("gb_70")},ma=function(){return!!u};p("qs",n);p("setContinueCb",ea);p("pc",fa);p("tg",ja);p("close",z);p("addLink",ka);p("almm",A);p("si",la);p("adh",ia);p("op",ma);var B=function(){},C=function(){},F=function(a){var b=new Image,c=D;b.onerror=b.onload=b.onabort=function(){try{delete E[c]}catch(a){}};E[c]=b;b.src=a;D=c+1},E=[],D=0;p("logger",{il:C,ml:B,log:F});var G=window.gbar.logger;var H={},na={},I=[],oa=l.b("0.1",.1),pa=l.a("1",!0),qa=function(a,b){I.push([a,b])},ra=function(a,b){H[a]=b},sa=function(a){return a in H},J={},K=function(a,b){J[a]||(J[a]=[]);J[a].push(b)},ta=function(a){K("m",a)},L=function(a,b){var c=document.createElement("script");c.src=a;c.async=pa;Math.random()<oa&&(c.onerror=function(){c.onerror=null;B(Error("Bundle load failed: name="+(b||"UNK")+" url="+a))});(document.getElementById("xjsc")||document.getElementsByTagName("body")[0]||
document.getElementsByTagName("head")[0]).appendChild(c)},N=function(a){for(var b=0,c;(c=I[b])&&c[0]!=a;++b);!c||c[1].l||c[1].s||(c[1].s=!0,M(2,a),c[1].url&&L(c[1].url,a),c[1].libs&&m.d&&m.d(c[1].libs))},O=function(a){K("gc",a)},P=null,ua=function(a){P=a},M=function(a,b,c){if(P){a={t:a,b:b};if(c)for(var d in c)a[d]=c[d];try{P(a)}catch(f){}}};p("mdc",H);p("mdi",na);p("bnc",I);p("qGC",O);p("qm",ta);p("qd",J);p("lb",N);p("mcf",ra);p("bcf",qa);p("aq",K);p("mdd","");p("has",sa);
p("trh",ua);p("tev",M);var Q=l.b("0.1",.001),R=0;
function _mlToken(a,b){try{if(1>R){R++;var c,d=a,f=b||{},e=encodeURIComponent,h=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&jexpid=",e("17483"),"&srcpg=",e("prop=22"),"&jsr=",Math.round(1/Q),"&ogev=",e("_4ntU6-2HcutsASZtYH4CA"),"&ogf=",g.bv.f,"&ogrp=",e("1"),"&ogv=",e("1407723702.0"),"&oggv="+e("es_plusone_gc_20140723.0_p0"),"&ogd=",e("com"),"&ogc=",e("FRA"),"&ogl=",e("en")];f._sn&&(f._sn="og."+
f._sn);for(var k in f)h.push("&"),h.push(e(k)),h.push("="),h.push(e(f[k]));h.push("&emsg=");h.push(e(d.name+":"+d.message));var r=h.join("");S(r)&&(r=r.substr(0,2E3));c=r;var Aa=window.gbar.logger._aem(a,c);F(Aa)}}catch(Na){}}var S=function(a){return 2E3<=a.length},va=function(a,b){return b};function T(a){B=a;p("_itl",S,G);p("_aem",va,G);p("ml",B,G);a={};H.er=a}l.a("")?T(function(a){throw a;}):l.a("1")&&Math.random()<Q&&T(_mlToken);I.push(["m",{url:"//ssl.gstatic.com/gb/js/scm_7385cc5883250b43a39405734c1bea59.js"}]);g.mcf("c",{});g.sg={c:""};if(l.a("1")){var wa=l.a("");I.push(["gc",{auto:wa,url:"//ssl.gstatic.com/gb/js/abc/gci_91f30755d6a6b787dcc2a4062e6e9824.js",libs:"googleapis.client:plusone:gapi.iframes"}]);var xa={version:"gci_91f30755d6a6b787dcc2a4062e6e9824.js",index:"",lang:"en"};H.gc=xa;var U=function(a){window.googleapis&&window.iframes?a&&a():(a&&O(a),N("gc"))};p("lGC",U);l.a("1")&&p("lPWF",U)};window.__PVT="";if(l.a("1")&&l.a("1")){var V=function(a){U(function(){K("pw",a);N("pw")})};p("lPW",V);I.push(["pw",{url:"//ssl.gstatic.com/gb/js/abc/pwm_45f73e4df07a0e388b0fa1f3d30e7280.js"}]);var W=[],ya=function(a){W[0]=a},za=function(a,b){var c=b||{};c._sn="pw";B(a,c)},Ba={signed:W,elog:za,base:"https://plusone.google.com/u/0",loadTime:(new Date).getTime()};H.pw=Ba;var X=function(a,b){for(var c=b.split("."),d=function(){var b=arguments;a(function(){for(var a=g,d=0,e=c.length-1;d<e;++d)a=a[c[d]];a[c[d]].apply(a,b)})},f=g,e=0,h=c.length-1;e<h;++e)f=
f[c[e]]=f[c[e]]||{};return f[c[e]]=d};X(V,"pw.clk");X(V,"pw.hvr");p("su",ya,g.pw)};function Ca(){function a(){for(var b;(b=e[h++])&&"m"!=b[0]&&!b[1].auto;);b&&(M(2,b[0]),b[1].url&&L(b[1].url,b[0]),b[1].libs&&m.d&&m.d(b[1].libs));h<e.length&&setTimeout(a,0)}function b(){0<f--?setTimeout(b,0):a()}var c=l.a("1"),d=l.a(""),f=3,e=I,h=0,k=window.gbarOnReady;if(k)try{k()}catch(r){da(r,"ml","or")}d?p("ldb",a):c?q(window,"load",b):b()}p("rdl",Ca);var Da={D:1,H:2,da:3,p:4,W:5,M:6,F:7,g:8,ha:9,U:10,L:11,T:12,S:13,N:14,Q:15,P:16,fa:17,w:18,O:19,ga:20,ea:21,u:22,G:23,ja:24,ka:25,ia:26,A:27,j:28,o:29,k:30,ca:31,Z:32,$:33,J:34,K:35,ba:36,aa:37,Y:38,B:39,R:40,v:41,X:42,V:43,h:48,C:49,I:500},Y=[1,2,3,4,5,6,9,10,11,13,14,28,29,30,34,35,37,38,39,40,41,42,43,48,49,500];var Z=l.b("0.001",1E-4),Ea=l.b("1",1),Fa=!1,Ga=!1;if(l.a("1")){var Ha=Math.random();Ha<=Z&&(Fa=!0);Ha<=Ea&&(Ga=!0)}var Ia=Da,$=null;function Ja(){var a=0,b=function(b,d){l.a(d)&&(a|=b)};b(1,"");b(2,"");b(4,"");b(8,"");return a}
function Ka(a,b){var c=Z,d=Fa,f;f=a;if(!$){$={};for(var e=0;e<Y.length;e++){var h=Y[e];$[h]=!0}}if(f=!!$[f])c=Ea,d=Ga;if(d){d=encodeURIComponent;g.rp?(f=g.rp(),f="-1"!=f?f:"1"):f="1";c=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&oge=",a,"&ogex=",d("17483"),"&ogev=",d("_4ntU6-2HcutsASZtYH4CA"),"&ogf=",g.bv.f,"&ogp=",d("22"),"&ogrp=",d(f),"&ogsr=",Math.round(1/c),"&ogv=",d("1407723702.0"),"&oggv="+
d("es_plusone_gc_20140723.0_p0"),"&ogd=",d("com"),"&ogl=",d("en"),"&ogc=",d("FRA"),"&ogus=",Ja()];if(b){"ogw"in b&&(c.push("&ogw="+b.ogw),delete b.ogw);var k;f=b;e=[];for(k in f)0!=e.length&&e.push(","),e.push(La(k)),e.push("."),e.push(La(f[k]));k=e.join("");""!=k&&(c.push("&ogad="),c.push(d(k)))}F(c.join(""))}}function La(a){"number"==typeof a&&(a+="");return"string"==typeof a?a.replace(".","%2E").replace(",","%2C"):a}C=Ka;p("il",C,G);var Ma={};H.il=Ma;setTimeout(function(){C(Ia.g)},0);}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var b=window.gbar.i.i;var c=window.gbar;var f=function(d){try{var a=document.getElementById("gbom");a&&d.appendChild(a.cloneNode(!0))}catch(e){b(e,"omas","aomc")}};c.aomc=f;}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var a=window.gbar;a.mcf("pm",{p:""});}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{window.gbar.rdl();}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
if (window['_OC_timingAction']) {window['_OC_timingAction']('patents_refpage');}</script><style>#gbar,#guser{font-size:13px;padding-top:1px !important;}#gbar{float:left;height:22px}#guser{padding-bottom:7px !important;text-align:right}.gbh,.gbd{border-top:1px solid #c9d7f1;font-size:1px}.gbh{height:0;position:absolute;top:24px;width:100%}#gbs,.gbm{background:#fff;left:0;position:absolute;text-align:left;visibility:hidden;z-index:1000}.gbm{border:1px solid;border-color:#c9d7f1 #36c #36c #a2bae7;z-index:1001}.gb1{margin-right:.5em}.gb1,.gb3{zoom:1}.gb2{display:block;padding:.2em .5em}.gb2,.gb3{text-decoration:none !important;border-bottom:none}a.gb1,a.gb4{text-decoration:underline !important}a.gb1,a.gb2,a.gb3,a.gb4{color:#00c !important}.gbi .gb3,.gbi .gb2,.gbi .gb4{color:#dd8e27 !important}.gbf .gb3,.gbf .gb2,.gbf .gb4{color:#900 !important}a.gb2:hover{background:#36c;color:#fff !important}#gbar .gbz0l{color:#000 !important;cursor:default;font-weight:bold;text-decoration:none !important}
#gbar { padding:.3em .6em !important;}</style></head><body  topmargin="3" marginheight="3"><div id=gbar><nobr><a onclick=gbar.qs(this);gbar.logger.il(1,{t:1}); class=gb1 id=gb_1 href="https://www.google.com/search?sa=N&tab=tw">Search</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:2}); class=gb1 id=gb_2 href="http://www.google.com/search?hl=en&tbm=isch&source=og&sa=N&tab=ti">Images</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:8}); class=gb1 id=gb_8 href="http://maps.google.com/maps?hl=en&sa=N&tab=tl">Maps</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:78}); class=gb1 id=gb_78 href="https://play.google.com/?hl=en&sa=N&tab=t8">Play</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:36}); class=gb1 id=gb_36 href="http://www.youtube.com/results?sa=N&tab=t1">YouTube</a> <a onclick=gbar.logger.il(1,{t:5}); class=gb1 id=gb_5 href="http://news.google.com/nwshp?hl=en&tab=tn">News</a> <a onclick=gbar.logger.il(1,{t:23}); class=gb1 id=gb_23 href="https://mail.google.com/mail/?tab=tm">Gmail</a> <a onclick=gbar.logger.il(1,{t:25}); class=gb1 id=gb_25 href="https://drive.google.com/?tab=to">Drive</a> <a class=gb3 href="http://www.google.com/intl/en/options/" onclick="this.blur();gbar.tg(event);return !1" aria-haspopup=true><u>More</u> <small>&#9660;</small></a><div class=gbm id=gbi><a onclick=gbar.logger.il(1,{t:24}); class=gb2 id=gb_24 href="https://www.google.com/calendar?tab=tc">Calendar</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:51}); class=gb2 id=gb_51 href="http://translate.google.com/?hl=en&sa=N&tab=tT">Translate</a><a onclick=gbar.logger.il(1,{t:17}); class=gb2 id=gb_17 href="http://www.google.com/mobile/?hl=en&tab=tD">Mobile</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:10}); class=gb2 id=gb_10 href="http://www.google.com/search?hl=en&tbo=u&tbm=bks&source=og&sa=N&tab=tp">Books</a><a onclick=gbar.logger.il(1,{t:212}); class=gb2 id=gb_212 href="https://wallet.google.com/manage/?tab=ta">Wallet</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:6}); class=gb2 id=gb_6 href="http://www.google.com/search?hl=en&tbo=u&tbm=shop&source=og&sa=N&tab=tf">Shopping</a><a onclick=gbar.logger.il(1,{t:30}); class=gb2 id=gb_30 href="http://www.blogger.com/?tab=tj">Blogger</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:27}); class=gb2 id=gb_27 href="http://www.google.com/finance?sa=N&tab=te">Finance</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:31}); class=gb2 id=gb_31 href="https://plus.google.com/photos?sa=N&tab=tq">Photos</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:12}); class=gb2 id=gb_12 href="http://www.google.com/search?hl=en&tbo=u&tbm=vid&source=og&sa=N&tab=tv">Videos</a><div class=gb2><div class=gbd></div></div><a onclick=gbar.logger.il(1,{t:66}); href="http://www.google.com/intl/en/options/" class=gb2>Even more &raquo;</a></div></nobr></div><div id=guser width=100%><nobr><span id=gbn class=gbi></span><span id=gbf class=gbf></span><span id=gbe></span><a target=_top id=gb_70 href="https://www.google.com/accounts/Login?service=&continue=http://www.google.com/patents%3Fhl%3Den&hl=en" class=gb4>Sign in</a><div style="display: none"><div class=gbm id=gbd5 aria-owner=gbg5><div class=gbmc><ol id=gbom class=gbmcc></ol></div></div></div></nobr></div><div class=gbh style=left:0></div><div class=gbh style=right:0></div><div role="alert" style="position: absolute; left: 0; right: 0;"><a href="http://www.google.com/patents/us6406848?hl=en&amp;output=html_text" title="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."><img border="0" src="http://www.google.com/images/cleardot.gif"alt="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."></a></div><div id="guser"><nobr></nobr></div><div style="clear:both;"></div><div id="gb-top-search-box" class="gb-top-search-box-small gb-reset"><table><tr><td class="logo"><a href="http://www.google.com/patents" class="logo-link"><img class="logo-img" src="/intl/en/images/logos/google_logo_41.png" alt="Go to Google Books Home" height="41"/></a></td><td><form action="http://www.google.com/search" name="f" id="vheadf" method="get"><span id="hf"></span><input type="hidden" name="tbm" value="pts"/><input type="hidden" name="tbo" value="1"/><input type="hidden" name="hl" value="en"/><table><tr><td><div class="inputs"><table><tr><td><div class="text-input"><input type="text" name="q" id="vheadq" class="text" maxlength="2048" size="31" value="" title="Search Patents" accesskey="s" autocomplete="off"/><script>window._OC_autoDir &&window._OC_autoDir('vheadq', 'tia-vheadq');</script></div></td><td><div class="submit-input"><input name="btnG" class="submit" type="submit" value=""/></div></td></tr></table></div></td><td class="col-ext-links"><div class="ext-links"><a href="http://www.google.com/advanced_patent_search">&lt;nobr&gt;Advanced Patent Search&lt;/nobr&gt;</a></div></td></tr></table></form></td></tr></table></div><div class="kd-appbar"><h2 class="kd-appname"><a href="/patents">Patents</a></h2><div class="kd-buttonbar left" id="left-toolbar-buttons"><a id="appbar-write-review-link" href=""></a><a id="appbar-view-print-sample-link" href=""></a><a id="appbar-view-ebook-sample-link" href=""></a><a id="appbar-patents-prior-art-finder-link" href="https://www.google.com/patents/related/US6406848"></a><a id="appbar-patents-discuss-this-link" href="http://www.google.com/url?id=OGJaBAABERAJ&amp;q=http://patents.stackexchange.com/redirect/google-patents%3Fpatent%3DUS6406848&amp;usg=AFQjCNHSktpwvw9uLjKuCCb-4svzrQds4A" data-is-grant="true"></a><a id="appbar-read-patent-link" href="//docs.google.com/viewer?url=patentimages.storage.googleapis.com/pdfs/US6406848.pdf"></a><a id="appbar-download-pdf-link" href="//patentimages.storage.googleapis.com/pdfs/US6406848.pdf"></a></div><div class="kd-buttonbar right" id="right-toolbar-buttons"></div></div><div id="books-microdata" itemscope=""itemtype="http://schema.org/Book"itemid="http://www.google.com/patents/US6406848" style="display:none"><span itemprop="description">An apparatus and system are provided for simultaneously analyzing a plurality of analytes anchored to microparticles. Microparticles each having a uniform population of a single kind of analyte attached are disposed as a substantially immobilized planar array inside of a flow chamber where steps of an...</span><span itemprop="url">http://www.google.com/patents/US6406848?utm_source=gb-gplus-share</span><span class="main-title" itemprop="name">Patent US6406848 - Microparticles and polynucleotides</span><img itemprop="image" src="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"alt="Patent US6406848 - Microparticles and polynucleotides" title="Patent US6406848 - Microparticles and polynucleotides"></div><div style="display: none"><ol id="ofe-gear-menu-contents" class="gbmcc"><li class="gbe gbmtc"><a class="gbmt goog-menuitem-content" id="" href="http://www.google.com/advanced_patent_search">Advanced Patent Search</a></li></ol></div><table id="viewport_table" cellpadding="0" style="clear:both" cellspacing="0"><tr><td id="viewport_td"><div class=vertical_module_list_row><div id=intl_patents class=about_content><div id=intl_patents_v><table class="patent-bibdata"><tr><td class="patent-bibdata-heading">Publication number</td><td class="single-patent-bibdata">US6406848 B1</td></tr><tr><td class="patent-bibdata-heading">Publication type</td><td class="single-patent-bibdata">Grant</td></tr><tr><td class="patent-bibdata-heading">Application number</td><td class="single-patent-bibdata">US 09/424,028</td></tr><tr><td class="patent-bibdata-heading">PCT number</td><td class="single-patent-bibdata">PCT/US1998/011224</td></tr><tr><td class="patent-bibdata-heading">Publication date</td><td class="single-patent-bibdata">Jun 18, 2002</td></tr><tr><td class="patent-bibdata-heading">Filing date</td><td class="single-patent-bibdata">May 22, 1998</td></tr><tr><td class="patent-bibdata-heading">Priority date<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The priority date is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the date listed."></span></td><td class="single-patent-bibdata">May 23, 1997</td></tr><tr><td class="patent-bibdata-heading">Fee status<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The fee status is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the status or dates listed."></span></td><td class="single-patent-bibdata">Paid</td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Also published as</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/CA2291180A1">CA2291180A1</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP0985142A2">EP0985142A2</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP0985142A4">EP0985142A4</a>, </span><span class="patent-bibdata-value"><a href="/patents/US6806052">US6806052</a>, </span><span class="patent-bibdata-value"><a href="/patents/US6831994">US6831994</a>, </span><span class="patent-bibdata-value"><a href="/patents/US8361713">US8361713</a>, </span><span class="patent-bibdata-value"><a href="/patents/US8728729">US8728729</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20020051992">US20020051992</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20030077615">US20030077615</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20090143244">US20090143244</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20130184162">US20130184162</a>, </span><span class="patent-bibdata-value"><a href="/patents/WO1998053300A2">WO1998053300A2</a>, </span><span class="patent-bibdata-value"><a href="/patents/WO1998053300A3">WO1998053300A3</a></span></span></td></tr><tr class="patent-bibdata-list-row alternate-patent-number"><td class="patent-bibdata-heading">Publication number</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value">09424028, </span><span class="patent-bibdata-value">424028, </span><span class="patent-bibdata-value">PCT/1998/11224, </span><span class="patent-bibdata-value">PCT/US/1998/011224, </span><span class="patent-bibdata-value">PCT/US/1998/11224, </span><span class="patent-bibdata-value">PCT/US/98/011224, </span><span class="patent-bibdata-value">PCT/US/98/11224, </span><span class="patent-bibdata-value">PCT/US1998/011224, </span><span class="patent-bibdata-value">PCT/US1998/11224, </span><span class="patent-bibdata-value">PCT/US1998011224, </span><span class="patent-bibdata-value">PCT/US199811224, </span><span class="patent-bibdata-value">PCT/US98/011224, </span><span class="patent-bibdata-value">PCT/US98/11224, </span><span class="patent-bibdata-value">PCT/US98011224, </span><span class="patent-bibdata-value">PCT/US9811224, </span><span class="patent-bibdata-value">US 6406848 B1, </span><span class="patent-bibdata-value">US 6406848B1, </span><span class="patent-bibdata-value">US-B1-6406848, </span><span class="patent-bibdata-value">US6406848 B1, </span><span class="patent-bibdata-value">US6406848B1</span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Inventors</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22John+Bridgham%22">John Bridgham</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Kevin+Corcoran%22">Kevin Corcoran</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22George+Golda%22">George Golda</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Michael+C.+Pallas%22">Michael C. Pallas</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Sydney+Brenner%22">Sydney Brenner</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Original Assignee</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=inassignee:%22Lynx+Therapeutics,+Inc.%22">Lynx Therapeutics, Inc.</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Export Citation</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/US6406848.bibtex">BiBTeX</a>, </span><span class="patent-bibdata-value"><a href="/patents/US6406848.enw">EndNote</a>, </span><span class="patent-bibdata-value"><a href="/patents/US6406848.ris">RefMan</a></span></span></td></tr><tr class="patent-internal-links"><td colspan=2><span class="patent-bibdata-value"><a href="#backward-citations">Patent Citations</a> (2),</span> <span class="patent-bibdata-value"><a href="#forward-citations">Referenced by</a> (113),</span> <span class="patent-bibdata-value"><a href="#classifications">Classifications</a> (47),</span> <span class="patent-bibdata-value"><a href="#legal-events">Legal Events</a> (8)</span> </td></tr><tr><td colspan=2 class="patent-bibdata-external-link-spacer-top"></td></tr><tr class="patent-bibdata-external-link-spacer-bottom"></tr><tr><td colspan=2><span class="patent-bibdata-heading">External Links:&nbsp;</span><span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=OGJaBAABERAJ&q=http://patft.uspto.gov/netacgi/nph-Parser%3FSect2%3DPTO1%26Sect2%3DHITOFF%26p%3D1%26u%3D/netahtml/PTO/search-bool.html%26r%3D1%26f%3DG%26l%3D50%26d%3DPALL%26RefSrch%3Dyes%26Query%3DPN/6406848&usg=AFQjCNEqcUKFcw0r7ZbXr4mWPurpEnfp4g">USPTO</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=OGJaBAABERAJ&q=http://assignments.uspto.gov/assignments/q%3Fdb%3Dpat%26pat%3D6406848&usg=AFQjCNH-4LVhR_gwunxD7zvdb6NkyUZDuQ">USPTO Assignment</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=OGJaBAABERAJ&q=http://worldwide.espacenet.com/publicationDetails/biblio%3FCC%3DUS%26NR%3D6406848B1%26KC%3DB1%26FT%3DD&usg=AFQjCNE65ZiaD6XRkrvecWxWqUDn529SiA">Espacenet</a></span></span></td></tr><tr class="patent-bibdata-group-spacer"></tr></table><div class="number-and-title"><span class="patent-title"><invention-title mxw-id="PT129091534" lang="EN" load-source="ifi">Microparticles and polynucleotides</invention-title></span><br><span class="patent-number">US 6406848 B1</span></div><div class="patent-section patent-abstract-section"><div class="patent-section-header"><span class="patent-section-title">Abstract</span></div><div class="patent-text"><abstract mxw-id="PA50335934" lang="EN" load-source="patent-office"> <div class="abstract">An apparatus and system are provided for simultaneously analyzing a plurality of analytes anchored to microparticles. Microparticles each having a uniform population of a single kind of analyte attached are disposed as a substantially immobilized planar array inside of a flow chamber where steps of an analytical process are carried out by delivering a sequence of processing reagents to the microparticles by a fluidic system under microprocessor control. In response to such process steps, an optical signal is generated at the surface of each microparticle which is characteristic of the interaction between the analyte carried by the microparticle and the delivered processing reagent. The plurality of analytes are simultaneously analyzed by collecting and recording images of the optical signals generated by all the microparticles in the planar array. A key feature of the invention is the correlation of the sequence of optical signals generated by each microparticle in the planar array during the analytical process.</div>
  </abstract></div></div><div class="patent-section patent-drawings-section"><div class="patent-section-header"><span class="patent-section-title">Images<span class="patent-section-count">(10)</span></span></div><div class="patent-drawings-body"><div class="patent-drawings-carousel"><div class="drawings"><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6406848B1/US06406848-20020618-D00001.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6406848B1/US06406848-20020618-D00001.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6406848B1/US06406848-20020618-D00002.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6406848B1/US06406848-20020618-D00002.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6406848B1/US06406848-20020618-D00003.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6406848B1/US06406848-20020618-D00003.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6406848B1/US06406848-20020618-D00004.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6406848B1/US06406848-20020618-D00004.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6406848B1/US06406848-20020618-D00005.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6406848B1/US06406848-20020618-D00005.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6406848B1/US06406848-20020618-D00006.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6406848B1/US06406848-20020618-D00006.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6406848B1/US06406848-20020618-D00007.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6406848B1/US06406848-20020618-D00007.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6406848B1/US06406848-20020618-D00008.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6406848B1/US06406848-20020618-D00008.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6406848B1/US06406848-20020618-D00009.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6406848B1/US06406848-20020618-D00009.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6406848B1/US06406848-20020618-D00010.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6406848B1/US06406848-20020618-D00010.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div></div></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img" alt="Previous page"src="/googlebooks/images/kennedy/page_left.png"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img" alt="Next page"src="/googlebooks/images/kennedy/page_right.png"width="21" height="21" /></div></div></div><div class="patent-post-drawings"></div><div class="patent-section patent-claims-section"><div class="patent-section-header"><span class="patent-section-title">Claims<span class="patent-section-count">(4)</span></span></div><div class="patent-text"><div mxw-id="PCLM8310742" lang="EN" load-source="patent-office" class="claims">
    <claim-statement>We claim: </claim-statement> <div class="claim"> <div num="1" id="US-6406848-B1-CLM-00001" class="claim">
      <div class="claim-text">1. An array of polynucleotides, comprising:</div>
      <div class="claim-text">a closely packed planar array of microparticles, the closely packed planar array having a number of microparticles per unit area in the planar array is at least eighty percent of the number of microparticles in a hexagonal array of equal area; and </div>
      <div class="claim-text">a plurality of different polynucleotides attached to the microparticles, such that each different polynucleotide is attached to a different microparticle. </div>
    </div>
    </div> <div class="claim-dependent"> <div num="2" id="US-6406848-B1-CLM-00002" class="claim">
      <div class="claim-text">2. The array of <claim-ref idref="US-6406848-B1-CLM-00001">claim 1</claim-ref>, wherein the diameter of each of the microparticles is between about 0.1 μm and 100 μm.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="3" id="US-6406848-B1-CLM-00003" class="claim">
      <div class="claim-text">3. The array of <claim-ref idref="US-6406848-B1-CLM-00001">claim 1</claim-ref>, wherein the plurality of different polynucleotides comprises a cDNA library.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="4" id="US-6406848-B1-CLM-00004" class="claim">
      <div class="claim-text">4. The array of <claim-ref idref="US-6406848-B1-CLM-00001">claim 1</claim-ref>, wherein the planar array of microparticles is disposed in a flow chamber.</div>
    </div>
  </div> </div></div></div><div class="patent-section patent-description-section"><div class="patent-section-header"><span class="patent-section-title">Description</span></div><div class="patent-text"><div mxw-id="PDES53584431" lang="EN" load-source="patent-office" class="description">
    <p>This application claims priority to application No. PCT/US98/11224 filed May 22, 1998, which is a continuation-in-part to application Ser. No. 08/862,610 filed May 23, 1997, now abandoned, which are both incorporated herein by reference.</p>
    <heading>FIELD OF THE INVENTION</heading> <p>The invention relates generally to systems and apparatus for carrying out large scale parallel reactions on solid phase supports, and more particularly, to systems and apparatus for monitoring and carrying out reactions on arrays of microparticles.</p>
    <heading>BACKGROUND</heading> <p>The desire to understand and analyze complex chemical and biological systems has led to the development of analytical techniques that employ parallelization and miniaturization of analyte processing, e.g. Graber et al, Current Opinion in Biotechnology, 9: 14-18 (1998); Fodor et al, Nature, 364: 555-556 (1993); Meier-Ewert et al, Nature, 361: 375-376 (1993); Taylor et al, Nucleic Acids Research, 25: 3164-3168 (1997); Garner et al, BioTechniques, 14: 112-115 (1993); Lam et al, Nature, 354: 82-84 (1991); Ohlmeyer et al, Proc. Natl. Acad. Sci., 90: 10922-10926 (1993); DeRisi et al, Science, 278: 680-686 (1997); Wodicka et al, Nature Biotechnology, 15: 1359-1367 (1997); and the like.</p>
    <p>Many of these techniques employ microparticles for synthesizing analytes or for capturing analytes for subsequent analysis, e.g. Lam et al (cited above); Benkovic et al, International patent application PCT/US95/03355; Gavin et al, International patent application PCT/EP97/02039; Brenner et al, International patent application PCT/US96/09513, and the like. Even though the properties of different types of microparticles can vary widely, microparticles generally facilitate the construction and manipulation of large repertoires of analytes with minimal reagent and/or sample consumption. However, handling and manipulating large numbers of microparticles, e.g. tens to hundreds of thousands, for carrying out specific chemical and/or biochemical analyses gives rise to many difficulties, including whether sufficient signal is generated on individual microparticles for detection, how to track individual microparticles through multiple steps of a process, mechanical strength of microparticles under pressure or flow conditions, the ability to uniformly deliver reagents to microparticles for carrying out steps of an analytical process, whether clumping or other inappropriate interaction of microparticles and/or reagents occurs, the degree to which analytes and/or processing reagents adsorb onto vessel walls, whether protein reagents or analytes denature causing a disruption of reagent distribution and access, whether adjacent microparticles will interact, e.g. to degrade or obscure a signal or to inhibit reagent access, and the like.</p>
    <p>In view of these difficulties, it would be desirable to provide a system and apparatus for handling and processing multiple solid phase supports, such as populations of microparticles. It would be especially desirable if such system and apparatus permitted the tracking and analysis of multiple analytes anchored to separate microparticles through a sequence of several processing and/or analysis steps.</p>
    <heading>SUMMARY OF THE INVENTION</heading> <p>Accordingly, objects of our invention include, but are not limited to, providing a system and apparatus for sequentially delivering reagents to a population of analytes anchored to separate microparticles; providing an apparatus for simultaneously monitoring the interactions of processing reagents and analytes on the surfaces of microparticles disposed in a planar array; providing an apparatus for detecting optical signals generated by, or as the result of, interactions of processing reagents and analytes on the surfaces of microparticles disposed in a planar array; providing an apparatus for detecting pluralities of optical signals, each such plurality being generated at the surface of the same microparticle as a result of interactions between processing reagents and an analyte anchored to the surface of such microparticle; providing an apparatus for simultaneously tracking the positions of individual microparticles in a population of microparticles disposed in a flow chamber as a closely packed planar array; and providing a system and apparatus for simultaneously analyzing the nucleotide sequences of a population of polynucleotides anchored to microparticles disposed in a planar array in a flow chamber.</p>
    <p>Our invention achieves these and other objects with an apparatus comprising a flow chamber for disposing a population of microparticles in a planar array; fluidic means for sequentially delivering processing reagents from one or more reagent reservoirs to the flow chamber; and detection means for detecting a sequence of optical signals from each of the microparticles of the population. Preferably, the sequences of optical signals are generated as a result of a multi-step analytical process, such as nucleic acid sequence analysis.</p>
    <p>In one aspect, the invention provides a system for simultaneously monitoring a population of analytes which includes the apparatus of the invention, microparticles carrying the analytes, and software means for processing images of, and/or optical signals generated by, the microparticles when disposed in a planar array. Preferably, the flow chamber includes constraining means for restricting the movement of microparticles during cycles of reagent delivery.</p>
    <p>In another aspect, the invention includes a system for simultaneously analyzing the nucleotide sequences of a population of polynucleotides. Copies of each kind of polynucleotide in the population are sorted onto and anchored to one or more microparticles so that a population of loaded microparticles is formed. Loaded microparticles are disposed in a planar array in a flow chamber through which processing reagents are sequentially delivered to the loaded microparticles from one or more reagent reservoirs by a fluidic means. Optical signals generated by, or produced as a result of, the interaction of processing reagents and polynucleotides on the microparticles are imaged by a detection means. Preferably, when analysis includes determining the nucleotide sequence of a portion of each polynucleotide on the different microparticles, massively parallel signature sequencing (MPSS) analysis is employed, e.g. as described in Albrecht et al, International patent application PCT/US97/09472.</p>
    <heading>BRIEF DESCRIPTION OF THE DRAWINGS</heading> <p>FIG. 1<i>a </i>is a schematic representation of a flow chamber and fluidics and detection systems for observing a planar array of microparticles loaded with analyte molecules, such as cDNA molecules for sequencing.</p>
    <p>FIG. 1<i>b </i>is a schematic of a preferred holder for a flow chamber.</p>
    <p>FIG. 2<i>a </i>is bilateral cut away view of a flow chamber.</p>
    <p>FIG. 2<i>b </i>is a top view of a flow chamber.</p>
    <p>FIG. 2<i>c </i>is an illustration of microparticles being loaded into a flow chamber.</p>
    <p>FIGS. 3<i>a </i>through <b>3</b> <i>d </i>schematically illustrate microparticle constraining means for a flow chamber.</p>
    <p>FIG. 4 is a schematic representation of a device for loading microparticles into a flow chamber.</p>
    <p>FIG. 5 is a schematic representation of a fluidics system for use with the invention.</p>
    <p>FIGS. 6<i>a </i>and <b>6</b> <i>b </i>schematically illustrate top-lighting and back-lighting approaches for determining microparticle centers in an array.</p>
    <p>FIG. 7 schematically illustrates the assignment of pixels to microparticles for data processing.</p>
    <p>FIG. 8 is a flow chart summarizing operation of the system of the invention.</p>
    <heading>DEFINITIONS</heading> <p>“Complement” or “tag complement” as used herein in reference to oligonucleotide tags refers to an oligonucleotide to which a oligonucleotide tag specifically hybridizes to form a perfectly matched duplex or triplex. In embodiments where specific hybridization results in a triplex, the oligonucleotide tag may be selected to be either double stranded or single stranded. Thus, where triplexes are formed, the term “complement” is meant to encompass either a double stranded complement of a single stranded oligonucleotide tag or a single stranded complement of a double stranded oligonucleotide tag.</p>
    <p>The term “oligonucleotide” as used herein includes linear oligomers of natural or modified monomers or linkages, including deoxyribonucleosides, ribonucleosides, anomeric forms thereof, peptide nucleic acids (PNAs), and the like, capable of specifically binding to a target polynucleotide by way of a regular pattern of monomer-to-monomer interactions, such as Watson-Crick type of base pairing, base stacking, Hoogsteen or reverse Hoogsteen types of base pairing, or the like. Usually monomers are linked by phosphodiester bonds or analogs thereof to form oligonucleotides ranging in size from a few monomeric units, e.g. 3-4, to several tens of monomeric units, e.g. 40-60. Whenever an oligonucleotide is represented by a sequence of letters, such as “ATGCCTG,” it will be understood that the nucleotides are in 5′→3′ order from left to right and that “A” denotes deoxyadenosine, “C” denotes deoxycytidine, “G” denotes deoxyguanosine, and “T” denotes thymidine, unless otherwise noted. Usually oligonucleotides of the invention comprise the four natural nucleotides; however, they may also comprise non-natural nucleotide analogs. It is clear to those skilled in the art when oligonucleotides having natural or non-natural nucleotides may be employed, e.g. where processing by enzymes is called for, usually oligonucleotides consisting of natural nucleotides are required.</p>
    <p>“Perfectly matched” in reference to a duplex means that the poly- or oligonucleotide strands making up the duplex form a double stranded structure with one other such that every nucleotide in each strand undergoes Watson-Crick basepairing with a nucleotide in the other strand. The term also comprehends the pairing of nucleoside analogs, such as deoxyinosine, nucleosides with 2-aminopurine bases, and the like, that may be employed. In reference to a triplex, the term means that the triplex consists of a perfectly matched duplex and a third strand in which every nucleotide undergoes Hoogsteen or reverse Hoogsteen association with a basepair of the perfectly matched duplex. Conversely, a “mismatch” in a duplex between a tag and an oligonucleotide means that a pair or triplet of nucleotides in the duplex or triplex fails to undergo Watson-Crick and/or Hoogsteen and/or reverse Hoogsteen bonding.</p>
    <p>As used herein, “nucleoside” includes the natural nucleosides, including 2′-deoxy and 2′-hydroxyl forms, e.g. as described in Komberg and Baker, DNA Replication, 2nd Ed. (Freeman, San Francisco, 1992). “Analogs” in reference to nucleosides includes synthetic nucleosides having modified base moieties and/or modified sugar moieties, e.g. described by Scheit, Nucleotide Analogs (John Wiley, New York, 1980); Uhlman and Peyman, Chemical Reviews, 90: 543-584 (1990), or the like, with the only proviso that they are capable of specific hybridization. Such analogs include synthetic nucleosides designed to enhance binding properties, reduce complexity, increase specificity, and the like.</p>
    <p>As used herein “sequence determination” or “determining a nucleotide sequence” in reference to polynucleotides includes determination of partial as well as full sequence information of the polynucleotide. That is, the term includes sequence comparisons, fingerprinting, and like levels of information about a target polynucleotide, as well as the express identification and ordering of nucleosides, usually each nucleoside, in a target polynucleotide. The term also includes the determination of the identification, ordering, and locations of one, two, or three of the four types of nucleotides within a target polynucleotide. For example, in some embodiments sequence determination may be effected by identifying the ordering and locations of a single type of nucleotide, e.g. cytosines, within the target polynucleotide “CATCGC . . .” so that its sequence is represented as a binary code, e.g. “100101 . . .” for “C-(not C)(not C)-C-(not C)-C . . .” and the like.</p>
    <p>As used herein, the term “complexity” in reference to a population of polynucleotides means the number of different species of molecule present in the population.</p>
    <heading>DETAILED DESCRIPTION OF THE INVENTION</heading> <p>The system and apparatus of the invention is particularly applicable to the analysis of molecules that can be anchored in populations of duplicate copies to particulate solid phase supports. That is, in accordance with the invention, each analyte of a population is present on at least one microparticle in a quantity sufficient for the type of analysis being performed. For example, if combinatorially synthesized peptides on the microparticles are screened against a soluble receptor protein for detecting those that form stable complexes, the number of peptides available for binding on the surface of the microparticles must be large enough to generate a detectable signal when a binding event occurs. Of course, many additional factors well known in the art will present additional design constraints, such as the nature of the system for generating optical signals, the concentration of receptors, pH, salt concentration, the density and accessibility of the peptides on the microparticle surface, the solvent system employed, and the like. Analyte populations particularly relevant for use with the present apparatus include combinatorial libraries synthesized on microparticle supports, e.g as disclosed in Lam et al, Chem. Rev., 97: 411-448 (1997); or Dower et al, U.S. Pat. No. 5,708,153, and polynucleotide libraries sorted onto microparticle supports, e.g. as disclosed in Brenner (cited above).</p>
    <p>FIG. 1<i>a </i>is a schematic representation of an embodiment of the invention for detecting fluorescent signals. Flow chamber (<b>100</b>) having inlet (<b>102</b>), outlet (<b>104</b>) and planar cavity (<b>106</b>) holds microparticles in a planar array from which optical signals (<b>108</b>) generated by analytes and/or reactants on microparticles can be collected and imaged. Flow chamber (<b>100</b>) is operationally associated with fluidic system (<b>112</b>) and detection system (<b>114</b>), so that delivery of fluids and collection of signals is under control of computer (<b>116</b>). Preferably, optical signals are collected by microscope (<b>118</b>) and are imaged onto a solid state imaging device, such as charge-coupled device (CCD) (<b>120</b>) which is capable of generating a digital image of the physical image of the microparticle array with sufficient resolution for individual microparticles to be distinguished. For fluorescent signals, detection system (<b>114</b>) usually includes appropriate bandpass filter (<b>122</b>) for optical signal (<b>108</b>), bandpass filter (<b>124</b>) for excitation beam (<b>128</b>) generated by light source (<b>126</b>), and other standard components. As illustrated, a conventional fluorescence microscope is preferred which is configured for epiillumination. There is a great deal of guidance in the art for selecting appropriate fluorescence microscopes, e.g Wang and Taylor, editors, Fluroescence Microscopy of Living Cells in Culture, Parts A and B, Methods in Cell Biology, Vols. 29 and 30 (Academic Press, New York, 1989).</p>
    <p>A key feature of the invention is flow chamber (<b>100</b>). Body (<b>130</b>) of flow chamber (<b>100</b>) preferably comprised inlet (<b>102</b>), outlet (<b>104</b>) and planar cavity (<b>106</b>) which are formed by standard micromachining techniques, e.g. Ekstrom et al, International patent application PCT/SE91/00327; Brown, U.S. Pat. No. 4,911,782; Harrison et al, Anal. Chem. 64: 1926-1932 (1992); and the like. Transparent plate (<b>132</b>) is sealingly attached to body (<b>130</b>) to form an operational flow chamber (<b>100</b>). Body (<b>130</b>) may be constructed from any of several different materials including glass, silicon, polyethylene, polyester, teflon, other plastics, and the like. Preferably, transparent plate (<b>132</b>) is glass or quartz; and, when body (<b>130</b>) and transparent plate (<b>132</b>) are glass or silicon, transparent plate (<b>132</b>) is preferably attached to body (<b>130</b>) by anodic bonding, e.g. Pomerantz, U.S. Pat. No. 3,397,279. Key functions of the flow chamber include i) holding a population of microparticles in a substantially immobilized planar array, or monolayer, during a sequence of processing steps, ii) ensuring that processing reagents can access each microparticle during each step of a process, and iii) minimizing processing reagent usage. The degree of immobilization required may vary among different embodiments. Generally, more movement of microparticles within a planar array increases the computational and measurement burden of tracking positions of microparticles by image processing software. Design trade-offs therefore exist between the use of image processing software and the use of physical and/or chemical means for constraining microparticle movement. Preferably, physical and/or chemical means are employed to constrain microparticle movement within the planar array of microparticles in flow chamber (<b>100</b>). Such means are referred to herein as “movement constraining means.” Most preferably, physical, or mechanical, movement constraining means are employed.</p>
    <p>Preferably, microparticles are disposed in flow chamber (<b>100</b>) in a closely packed planar array. As used herein, “closely packed” in reference to a planar array means either that the number of microparticles per unit area of a planar array is at least eighty percent of the number of microparticles in a hexagonal array of equal area, or that the average distance between centers of adjacent microparticles is less than two microparticle diameters. As used herein, a “hexagonal” array of microparticles means a planar array of microparticles in which every microparticle in the array contacts at least six other adjacent microparticles, as shown in FIG. 3<i>a. </i> </p>
    <p>Additions features of flow chamber (<b>100</b>) of a preferred embodiment are illustrated in FIGS. 2<i>a </i>through <b>2</b> <i>c</i>. FIG. 2<i>a </i>is a cross sectional view along a longitudinal plane that bisects flow chamber (<b>100</b>). The same view, in a more abstracted rendition, is shown in FIG. 2<i>c</i>. In both Figures, inlet (<b>102</b>) fluidly communicates with planar cavity (<b>106</b>) and outlet (<b>104</b>). Microparticles (<b>200</b>) carrying analytes enter inlet (<b>102</b>) and are carried by a suspending buffer to planar cavity (<b>106</b>) where they become packed against dam (<b>202</b>) which prevents the microparticles from exiting the flow chamber through outlet (<b>104</b>). Structurally, dam (<b>202</b>) may be formed by a sudden reduction of the vertical dimension of planar cavity (<b>106</b>). Preferably, vertical dimension (<b>204</b>) of planar cavity (<b>106</b>) is selected so that microparticles (<b>200</b>) are constrained to a plane, i.e. a monolayer, when they pack against dam (<b>202</b>). More preferably, vertical dimension (<b>204</b>) is selected to be between about 120 to 150 percent of the diameter of the microparticles employed. For example, when microparticles are employed that have diameters of 5 μm, vertical dimension (<b>204</b>) may be 7 μm. Magnetic microparticles may be constrained to a plane and constrained from movement by applying a magnetic field so that the microparticles are attracted to the ceiling or to the floor of planar cavity (<b>106</b>). Width (<b>206</b>) of planar cavity (<b>106</b>) is not a critical dimension; however, for convenience and efficiency, width (<b>206</b>) may be selected to correspond to the dimensions of the signal collection region of detection system (<b>114</b>). Such regions labeled 1 through k in FIG. 2<i>b </i>are referred to herein as “tiles.” That is, the region of planar cavity (<b>106</b>) occupied by microparticles may be divided into non-overlapping areas, referred to as “tiles,” that cover the entire occupied region. FIG. 2<i>b</i>, which is a top view of the flow chamber of FIG. 2<i>a</i>, also shows inlet (<b>102</b>), planar cavity (<b>106</b>), dam (<b>202</b>), and outlet (<b>104</b>) that lie in sequence along axis (<b>217</b>) of flow chamber (<b>100</b>).</p>
    <p>Many movement constraining means may be selected for use with the flow chamber, either alone or in combination. Such means include loading microparticles with trace amounts of a chemically reactive species which may be activated and cross-linked; providing physical, or mechanical structures, such as ridges, within the flow chamber; providing magnetically responsive microparticles which may be immobilized by an external magnetic field; providing a second population of microparticles that are loaded into a flow chamber after the analyte-containing population, which forces the analyte-containing population against dam (<b>202</b>); and the like. Exemplary chemically reactive species for use with nucleic acid analytes are disclosed in Summerton et al, U.S. Pat. No. 4,123,610; Gamper et al, J. Mol. Biol., 197: 349-362 (1987); Hearst, Ann. Rev. Phys. Chem. 39: 291-315 (1988); Pieles et al, Nucleic Acids Research, 17: 8967-8978 (1989); and the like.</p>
    <p>Preferably, microparticle movement is constrained by providing a flow chamber with planar cavity (<b>106</b>) containing a plurality of ridges running parallel to axis (<b>217</b>) of the flow chamber, i.e. parallel to the direction of reagent flow, so that microparticles are arranged into rows, which may be single-file, or several microparticles wide, as shown in FIGS. 3<i>a </i>and <b>3</b> <i>b</i>. The particular selection may depend on several factors, including the degree of immobilization desired, constraints imposed by the fabrication technique used to construct the flow chamber, the amount of reagent access desired, the degree to which flow resistance or back-pressure can be tolerated, and the like. FIGS. 3<i>a </i>and <b>3</b> <i>b </i>illustrate two possible distances between parallel ridges.</p>
    <p>In FIG. 3<i>a</i>, the distance is selected to permit maximal packing of microparticles into a hexagonal array, and in FIG. 3<i>b</i>, the distance is selected for less efficient packing, but for increased reagent access to microparticle surfaces. FIGS. 3<i>c </i>and <b>3</b> <i>d </i>are axial views of the flow chamber showing the microparticle arrangements of FIGS. 3<i>a </i>and <b>3</b> <i>b</i>, respectively.</p>
    <p>In some embodiments, such as those employing enzymatic processes, the inner surfaces of flow chamber (<b>100</b>) may be passivated, that is, treated to render such surfaces inert and/or non-adsorbing with respect to enzymes. The type of treatment depends on the sensitivity of the enzymes used in the process, and their affinity for the surfaces. Surface treatments include silanization, e.g. with commercially available reagents (Pierce, Rockford, Ill.); and/or adsorption of various blocking polymers, such as poly-a-alanine, polyglycine, polyadenylic acid, polymaleimide, polyvinylpyrrolidone, or the like, e.g. Shoffner et al, Nucleic Acids Research, 24:375-379 (1996). Preferably, glass inner surfaces of flow chamber (<b>100</b>) are covalently coated with a neutral coating, such as allyl methacrylate, using the technique disclosed in Sandoval et al, U.S. Pat. No. 5,326,738, which is incorporated by reference.</p>
    <p>FIG. 1<i>b </i>illustrates flow chamber (<b>100</b>) mounted between holders (<b>140</b>) and (<b>142</b>) which sealingly connect inlet (<b>102</b>) to inlet tubing (<b>144</b>) and outlet (<b>104</b>) to outlet tubing (<b>146</b>), respectively. Preferably, holder (<b>140</b>) contains a rotary valve (not shown) operated by actuator (<b>148</b>) that shunts fluid flowing through inlet tubing (<b>144</b>) to inlet (<b>102</b>) or to waste line (<b>150</b>). Such a valve minimizes the amount of process reagent from a previous step that must be passed through flow chamber (<b>100</b>) prior to the initiation of the next process step. That is, such a rotary valve permits reagent in inlet tubing (<b>144</b>) to be shunted to waste and replaced by processing reagent required for the next step in the process being executed. Preferably, for use in DNA analysis, peltier block (<b>152</b>) is employed to control temperature in flow chamber (<b>100</b>) and the entire assembly including flow chamber (<b>100</b>) and peltier block (<b>152</b>) is mounted on xyz-stage (<b>154</b>) which is under control of computer (<b>116</b>).</p>
    <p>Preferably, microparticles are loaded into flow chamber (<b>100</b>) prior to attachment of holders (<b>140</b>) and (<b>142</b>) and the initiation of processing steps. FIG. 4 illustrates a microparticle loader for loading microparticles into flow chamber (<b>100</b>). Flow chamber (<b>100</b>) is mounted between holders (<b>400</b>), (<b>402</b>), (<b>404</b>), and (<b>406</b>). Holders (<b>400</b>) and (<b>402</b>) sealingly clamp onto the inlet end (<b>101</b>) of flow chamber (<b>100</b>) and holders (<b>404</b>) and (<b>406</b>) sealingly clamp onto the outlet end (<b>103</b>) of flow chamber (<b>100</b>) so that inlet tubing (<b>408</b>) is in fluid communication with outlet tubing (<b>410</b>) when the microparticle loader is assembled. Inlet tubing (<b>408</b>) is connected to syringe (<b>416</b>) which is used to drive fluid through flow chamber (<b>100</b>). Holder (<b>400</b>) is constructed to have conical passage (<b>412</b>) which narrows to match the diameter of inlet (<b>102</b>) of flow chamber (<b>100</b>). After assembly of holders (<b>400</b>), (<b>402</b>), (<b>404</b>), and (<b>406</b>) a suspension of microparticles is placed in the conical passage after which fitting (<b>414</b>) is sealingly connected to holder (<b>400</b>). Fluid pressure and flow generated by syringe (<b>416</b>) then drives the microparticles into planar cavity (<b>106</b>) and against dam (<b>202</b>). In a preferred embodiment which employs 5 μm diameter GMA microparticles carrying DNA, approximately 500 thousand microparticles are loaded into flow chamber (<b>100</b>) by placing 5 μL of a 100 thousand microparticle/μL solution (TE buffer, pH 8.0, Sambrook et al, Molecular Cloning, Second Edition (Cold Spring Harbor Laboratory, New York, 1989)) in conical passage (<b>412</b>), attaching fitting (<b>414</b>), and using syringe (<b>416</b>) to drive the microparticles through inlet (<b>102</b>) and into planar cavity (<b>106</b>). After loading, holders (<b>400</b>), (<b>402</b>), (<b>404</b>), and (<b>406</b>) are removed from flow chamber (<b>100</b>), which is then mounted on the apparatus as shown in FIG. 1<i>b. </i> </p>
    <p>Preferably, process reagents are delivered to flow chamber (<b>100</b>) by the fluidic system illustrated in FIG. 5 which has the capacity to handle many different reagents for complex analytical processes. In the illustrated embodiment, which is used in connection with DNA sequencing, the fluidics system may accommodate up to 38 reagents, including wash buffers, rinses, enzymes, hybridization probes, adaptors, and the like. Preferably, the function of the fluidics system is the sequential metering of selected processing reagents to flow chamber (<b>100</b>). Inlet (<b>102</b>) of flow chamber(<b>100</b>) is sealingly connected to holder (<b>140</b>) which contains rotary valve (actuator shown as <b>148</b>) (not shown in FIG. <b>5</b>). The function of the rotary valve is described above. A variety of means may be employed for moving processing reagents from reservoirs, through tubing, and into flow chamber (<b>100</b>), including gravity feed, pressure feed, and pumps, e.g. peristaltic, syringe, and the like. Preferably, common syringe pump (<b>500</b>) is employed for removing predetermined amounts processing reagents from reservoirs and for forcing such reagents through flow chamber (<b>100</b>) at a predetermined flow rate. Under control of computer (<b>116</b>), pump (<b>500</b>) in operational association with valve block (<b>502</b>) and rotary valve (<b>504</b>) removes a predetermined amount of processing reagent from a selected reservoir by siphoning reagent out of the reservoir on the out-stroke of plunger (<b>501</b>) of pump (<b>500</b>). On the in-stroke of plunger (<b>501</b>), rotary valve (<b>504</b>) directs processing reagent from tubing (<b>503</b>) to reservoir (<b>505</b>) of pump (<b>500</b>). On the out-stroke of plunger (<b>501</b>), state of rotary valve (<b>504</b>) is changed to direct processing reagent from reservoir (<b>505</b>) to inlet tubing (<b>144</b>). Tubing (<b>503</b>) connects rotary valve (<b>504</b>) with manifold (<b>508</b>) which, in turn, is connected to a plurality (five shown) of banks of zero dead volume valves (<b>506</b>). Zero dead volume valves (<b>506</b>) connect individual reservoirs holding processing reagents to a common passageway (not shown in FIG. 5) that runs through each of the banks of valves connecting to manifold (<b>508</b>).</p>
    <p>A preferred zero dead volume valve is described in U.S. Pat. Nos. 4,558,845 and 4,703,913, which are incorporated by reference. Process reagents from reservoirs (<b>514</b>) are distributed to the banks of dead volume valves by way of manifold (<b>510</b>). Alternative valve blocks for controlling delivery of process reagents to flow chamber (<b>100</b>) include the valve matrix disclosed in U.S. Pat. No. 5,203,368.</p>
    <p>An important feature of detection means (<b>114</b>) of the invention is the ability to keep track of individual microparticles through multiple process steps and/or cycles. In connection with such tracking, detection means (<b>114</b>) periodically records optical characteristics of individual microparticles that provide a close approximation microparticle centers. Preferably, when trans-illumination, or “back lighting” of flow chamber (<b>100</b>) is possible, the optical characteristic is the focused back light from the microparticles. That is, in reference to FIG. 6<i>a</i>, back light (<b>600</b>) passes vertically through flow chamber (<b>100</b>) where it is focused by microparticles (<b>602</b>) onto focal plane (<b>604</b>). The image of focal plane (<b>604</b>) in this configuration appears as a field of bright points, where each point is located at the approximate center of its corresponding microparticle. In an epiillumination system, light from above flow chamber (<b>100</b>), i.e. “top light (<b>610</b>),” is directed from a vertical direction onto microparticles (<b>602</b>) where it scatters from the top surface of the microparticles. In this configuration, the optical characteristic is the scatter center of a microparticle. Thus, an image is collected from the plane containing scatter centers (<b>612</b>) resulting from such top lighting. As with focused back lighting, the image of the scatter centers provides a convenient way to readily determine the approximate centers of the microparticles.</p>
    <p>In the preferred image processing approach, once microparticle centers (<b>700</b>) are determined, pixels (<b>702</b>) are assigned for determining characteristics, e.g. intensity, of an optical signal generated at each microparticle (<b>602</b>). The size of microparticle (<b>602</b>) and pixel area determine how many pixels are assigned to each microparticle. In making such an assignment, important factors include the degree to which the calculated center of a microparticle (as described above) is likely to deviate from the geometric center, the extent to which optical signal collected from the edge of an image contains spurious information (e.g. signal from an overlapping or adjacent microparticle), the uniformity of microparticle diameter and shape, and the like. In the preferred apparatus of the invention, 5 μm diameter microparticles are employed and the pixel dimensions of the CCD detector are about 0.9 μm×0.9 μm. Thus, nine pixels fit easily within the interior of a microparticle image with a margin of at least about 1 μm between any pixel and the edge of the microparticle image. In the preferred embodiment, an initial pixel is assigned which encloses the computed center of a microparticle, e.g. pixel “5” in FIG. <b>7</b>. Thereafter, additional pixels are assigned, usually the immediately adjacent pixels. Preferably, the value of the optical signal generated by a process at the surface of a microparticle is the average value of the optical signals collected by pixels assigned to that microparticle.</p>
    <p>The general operation of the system of the preferred embodiment is summarized by the flow chart of FIG. <b>8</b>. At the start (<b>800</b>) of an analysis, microparticles with anchored analytes have been loaded into flow chamber (<b>100</b>) which has been operationally mounted in holders <b>140</b> and <b>142</b>. The initial operation is the calibration of the microparticle focal plane (<b>802</b>). That is, the vertical, or “z”, position of the xyz-stage is determined which optimizes the focus of either the scatter centers of the microparticles, i.e. the microparticle tops for top-lighting, or the focus points of the microparticles for back-lighting. The optimization is carried out by a conventional autofocusing algorithm which provides an image contrast function constructed from a predetermined sample of regions within a collected image. For example, the contrast function may be evaluated iteratively for sequence of z-positions so that the differences of successive values of the contrast function can be determined. These are tested until a difference is found below a predetermined threshold, which is taken as the maximum of the contrast function. Focal plane location is taken as the z position which maximizes the image contrast function. Such calibration is carried out for each tile, if more than one tile is employed, so that a correction table is constructed of changes in stage setting values with respect to the settings of the first tile that are required to bring the system into focus upon translation to subsequent tiles. These values are stored by computer (<b>116</b>).</p>
    <p>After calibration, process steps are initiated (<b>804</b>) by way of a fluidics controller operationally associated with computer (<b>116</b>). After process steps (<b>804</b>) are completed, stage settings are adjusted to place the first tile into focus using the autofocus algorithm (<b>806</b>), which places the focal plane of the microscope objective approximately at the tops of the microparticles. Stage settings are then adjusted (<b>808</b>) to bring the focal plane of the microscope objective to the approximate centers of the microparticles, as illustrated (<b>606</b>) in FIGS. 6<i>a </i>and <b>6</b> <i>b</i>. The amount of stage movement in this re-focusing depends on the diameter of the microparticles being used. After appropriate selection of filters (<b>124</b>) and (<b>122</b>), a fluorescent image of the first tile is collected (<b>810</b>) and transferred to data server (<b>812</b>). Fluorescent images are collected on the plane of the microparticle centers because of imperfections in the planar array. That is, microparticles in planar cavity (<b>106</b>) do not lie in a perfect planar array for a variety of reasons. For example, some microparticles are elevated above others as a result of packing into the flow chamber; there is some variability in the size and shape of the microparticles; and, the floor of planar cavity (<b>106</b>) may be uneven. After the fluorescent image is collected, the focal plane of the microscope objective is returned (<b>814</b>) to the microparticle focal plane, where another image is collected (<b>816</b>) for the purpose of computing microparticle centers as described above. The image of microparticle centers is transferred to data server (<b>812</b>) where data processor (<b>818</b>) assigns pixels of the fluorescent image to each microparticle center, as described above. After the image of microparticle centers is collected (<b>816</b>), the stage is moved so that an image of the next tile can be collected (<b>822</b>). If there are no further tiles of microparticles (<b>820</b>), then the next steps and/or cycles of the process are executed (<b>826</b>). If there are no further process steps (<b>824</b>), then the process is complete and the apparatus is placed in a holding mode.</p>
    <p>Optical signals collected in the course of analysis may be generated by a variety of mechanisms, including absorption and fluorescence, chemiluminescence, electrochemiluminescence, or bioluminescence emission. Extensive guidance is available for selecting appropriate optical signaling means, e.g. Kessler, editor, Nonradioactive Labeling and Detection of Biomolecules (Springer-Verlag, Berlin); Keller and Manak, DNA Probes, Second Edition (Stockton Press, New York, 1993); and the like. Preferably, optical signals generated in processing steps are fluorescence emissions.</p>
    <heading>Microparticles</heading> <p>An important feature of the system of the invention is the use of microparticles for carrying analytes. A variety of microparticles may be employed depending on particular applications. Generally, microparticles must consist of a material compatible with the reagents and chemistry of the process steps being carried out and microparticle must be substantially mechanically rigid so that they retain their shape and size during process steps. Preferably, as used herein, the term “substantially mechanically rigid” means that microparticles neither swell nor contract by more than ten percent (as measure by diameter) in any process solvent or reagent. Preferably, microparticles are microspheres of uniform size, i.e. microparticles are monodisperse. More preferably, the diameters of spherical microparticles have a coefficient of variation less than five percent, and most preferably, less than two percent. Microparticle diameters are in the range of from 0.1 μm to 100 μm. Preferably, microparticle diameters range from 1 μm to 20 μm. Most preferably, microparticle diameters are in the range of 1 to 5 μm. Suitable microparticle materials include inorganic support materials such as glass, e.g. controlled-pore glass, Balltoni beads; silica, zirconia, and the like, e.g. Weetall, Methods in Enzymology, 44: 134-148 (<b>1976</b>); and organic support materials such as highly cross-linked polystyrene, polyacrylate, polymethylmethacrylate, glycidylmethacrylate (GMA), Dynabeads (Dynal, Oslo, Norway), and the like, Rembaum et al, U.S. Pat. No. 4,046,720; Hodge and Sherrington, editors, pages 435-456, Polymer-supported Reactions in Organic Synthesis (Wiley &amp; Sons, New York, 1980); Andrus et al, U.S. Pat. No. 5,047,524; and the like.</p>
    <heading>Attaching Identical Copies of Polynucleotides to Microparticles by Solid Phase Cloning</heading> <p>In a preferred embodiment of the invention, identical copies of polynucleotides from a population are anchored to separate microparticles by solid phase cloning, i.e. the use of oligonucleotide tags for sorting polynucleotides onto microparticles such that only the same kind of polynucleotide will be attached to the same microparticle, e.g. Brenner, U.S. Pat. No. 5,604,097, which is incorporated by reference. This condition is accomplished by taking a sample of the full ensemble of tag-polynucleotide conjugates. (It is acceptable that identical polynucleotides have different tags, as it merely results in the same polynucleotide being operated on or analyzed twice in two different locations.) Such sampling can be carried out either overtly—for example, by taking a small volume from a larger mixture—after the tags have been attached to the polynucleotides, it can be carried out inherently as a secondary effect of the techniques used to process the polynucleotides and tags, or sampling can be carried out both overtly and as an inherent part of processing steps.</p>
    <p>Oligonucleotide tags for use with the invention are members of a minimally cross-hybridizing set of oligonucleotides. The sequences of oligonucleotides of such a set differ from the sequences of every other member of the same set by at least two nucleotides. Thus, each member of such a set cannot form a duplex (or triplex) with the complement of any other member with less than two mismatches. Complements of oligonucleotide tags of the invention, referred to herein as “tag complements,” may comprise natural nucleotides or non-natural nucleotide analogs. Tag complements are attached to microparticles.</p>
    <p>Minimally cross-hybridizing sets of oligonucleotide tags and tag complements may be synthesized either combinatorially or individually depending on the size of the set desired and the degree to which cross-hybridization is sought to be minimized (or stated another way, the degree to which specificity is sought to be enhanced). For example, a minimally cross-hybridizing set may consist of a set of individually synthesized 10-mer sequences that differ from each other by at least 4 nucleotides, such set having a maximum size of 332 (when composed of 3 kinds of nucleotides and counted using a computer program such as disclosed in Appendix Ic of International patent application PCT/US96/09513). Alternatively, a minimally cross-hybridizing set of oligonucleotide tags may also be assembled combinatorially from subunits which themselves are selected from a minimally cross-hybridizing set. For example, a set of minimally cross-hybridizing 12-mers differing from one another by at least three nucleotides may be synthesized by assembling 3 subunits selected from a set of minimally cross-hybridizing 4-mers that each differ from one another by three nucleotides. Such an embodiment gives a maximally sized set of 9<sup>3</sup>, or 729, 12-mers, “9” is number of oligonucleotides generated by the computer program of Appendix Ia of International patent application PCT/US96/09513, which assumes, as with the 10-mers, that only 3 of the 4 different types of nucleotides are used. The set is described as “maximal” because the computer programs disclosed in International patent application PCT/US96/09513 provide the largest set for a given input (e.g. length, composition, difference in number of nucleotides between members). Additional minimally cross-hybridizing sets may be formed from subsets of such calculated sets.</p>
    <p>When synthesized combinatorially, an oligonucleotide tag of the invention preferably consists of a plurality of subunits, each subunit consisting of an oligonucleotide of 3 to 9 nucleotides in length wherein each subunit is selected from the same minimally cross-hybridizing set. In such embodiments, the number of oligonucleotide tags available depends on the number of subunits per tag and on the length of the subunits.</p>
    <p>As used herein in reference to oligonucleotide tags and tag complements, the term “repertoire” means the set of minimally cross-hybridizing set of oligonucleotides that make up the tags in a particular embodiment or the corresponding set of tag complements.</p>
    <p>Preferably, in constructing a cDNA library where substantially all different cDNAs have different tags, a tag repertoire is employed whose complexity, or number of distinct tags, greatly exceeds the total number of mRNAs extracted from a cell or tissue sample. Preferably, the complexity of the tag repertoire is at least 10 times that of the polynucleotide population; and more preferably, the complexity of the tag repertoire is at least 100 times that of the polynucleotide population. Below, a protocol is disclosed for cDNA library construction using a primer mixture that contains a full repertoire of exemplary 9-word tags. Such a mixture of tag-containing primers has a complexity of 8<sup>9</sup>, or about 1.34×10<sup>8</sup>. As indicated by Winslow et al, Nucleic Acids Research, 19: 3251-3253 (1991), mRNA for library construction can be extracted from as few as 10-100 mammalian cells. Since a single mammalian cell contains about 5×10<sup>5 </sup>copies of mRNA molecules of about 3.4×10<sup>4 </sup>different kinds, by standard techniques one can isolate the mRNA from about 100 cells, or (theoretically) about 5×10<sup>7 </sup>mRNA molecules. Comparing this number to the complexity of the primer mixture shows that without any additional steps, and even assuming that mRNAs are converted into cDNAs with perfect efficiency (1% efficiency or less is more accurate), the cDNA library construction protocol results in a population containing no more than 37% of the total number of different tags. That is, without any overt sampling step at all, the protocol inherently generates a sample that comprises 37%, or less, of the tag repertoire. The probability of obtaining a double under these conditions is about 5%, which is within the preferred range. With mRNA from 10 cells, the fraction of the tag repertoire sampled is reduced to only 3.7%, even assuming that all the processing steps take place at 100% efficiency. In fact, the efficiencies of the processing steps for constructing cDNA libraries are very low, a “rule of thumb” being that good library should contain about 10<sup>8 </sup>cDNA clones from mRNA extracted from 10<sup>6 </sup>mammalian cells.</p>
    <p>Use of larger amounts of mRNA in the above protocol, or for larger amounts of polynucleotides in general, where the number of such molecules exceeds the complexity of the tag repertoire, a tag-polynucleotide conjugate mixture potentially contains every possible pairing of tags and types of mRNA or polynucleotide. In such cases, overt sampling may be implemented by removing a sample volume after a serial dilution of the starting mixture of tag-polynucleotide conjugates. The amount of dilution required depends on the amount of starting material and the efficiencies of the processing steps, which are readily estimated.</p>
    <p>If mRNA were extracted from 10<sup>6 </sup>cells (which would correspond to about 0.5 μg of poly(A)<sup>+</sup>RNA), and if primers were present in about 10-100 fold concentration excess—as is called for in a typical protocol, e.g. Sambrook et al, Molecular Cloning, Second Edition, page 8.61 [10 μL 1.8 kb mRNA at 1 mg/mL equals about 1.68×10<sup>−11 </sup>moles and 10 μL 18-mer primer at 1 mg/mL equals about 1.68×10<sup>−9 </sup>moles], then the total number of tag-polynucleotide conjugates in a cDNA library would simply be equal to or less than the starting number of mRNAs, or about 5×10<sup>11 </sup>vectors containing tag-polynucleotide conjugates—again this assumes that each step in cDNA construction—first strand synthesis, second strand synthesis, ligation into a vector—occurs with perfect efficiency, which is a very conservative estimate. The actual number is significantly less.</p>
    <p>If a sample of n tag-polynucleotide conjugates are randomly drawn from a reaction mixture—as could be effected by taking a sample volume, the probability of drawing conjugates having the same tag is described by the Poisson distribution, P(r)=e <sup>−λ</sup>(λ)<sup>r</sup>/r, where r is the number of conjugates having the same tag and λ=np, where p is the probability of a given tag being selected. If n=10<sup>6 </sup>and p=1/(1.34×10<sup>8</sup>), then λ=0.00746 and P(<b>2</b>)=2.76×10<sup>−5</sup>. Thus, a sample of one million molecules gives rise to an expected number of doubles well within the preferred range. Such a sample is readily obtained as follows: Assume that the 5×10<sup>11 </sup>mRNAs are perfectly converted into 5×10<sup>11 </sup>vectors with tag-cDNA conjugates as inserts and that the 5×10<sup>11 </sup>vectors are in a reaction solution having a volume of 100 μl. Four 10-fold serial dilutions may be carried out by transferring 10 μl from the original solution into a vessel containing 90 μl of an appropriate buffer, such as TE. This process may be repeated for three additional dilutions to obtain a 100 μl solution containing 5×10<sup>5 </sup>vector molecules per μl. A 2 μl aliquot from this solution yields 10<sup>6 </sup>vectors containing tag-cDNA conjugates as inserts. This sample is then amplified by straight forward transformation of a competent host cell followed by culturing.</p>
    <p>Of course, as mentioned above, no step in the above process proceeds with perfect efficiency. In particular, when vectors are employed to amplify a sample of tag-polynucleotide conjugates, the step of transforming a host is very inefficient. Usually, no more than 1% of the vectors are taken up by the host and replicated. Thus, for such a method of amplification, even fewer dilutions would be required to obtain a sample of 10<sup>6 </sup>conjugates.</p>
    <p>A repertoire of oligonucleotide tags can be conjugated to a population of polynucleotides in a number of ways, including direct enzymatic ligation, amplification, e.g. via PCR, using primers containing the tag sequences, and the like. The initial ligating step produces a very large population of tag-polynucleotide conjugates such that a single tag is generally attached to many different polynucleotides. However, as noted above, by taking a sufficiently small sample of the conjugates, the probability of obtaining “doubles,” i.e. the same tag on two different polynucleotides, can be made negligible. Generally, the larger the sample the greater the probability of obtaining a double. Thus, a design trade-off exists between selecting a large sample of tag-polynucleotide conjugates—which, for example, ensures adequate coverage of a target polynucleotide in a shotgun sequencing operation or adequate representation of a rapidly changing mRNA pool, and selecting a small sample which ensures that a minimal number of doubles will be present. In most embodiments, the presence of doubles merely adds an additional source of noise or, in the case of sequencing, a minor complication in scanning and signal processing, as microparticles giving multiple fluorescent signals can simply be ignored.</p>
    <p>As used herein, the term “substantially all” in reference to attaching tags to molecules, especially polynucleotides, is meant to reflect the statistical nature of the sampling procedure employed to obtain a population of tag-molecule conjugates essentially free of doubles. The meaning of substantially all in terms of actual percentages of tag-molecule conjugates depends on how the tags are being employed. Preferably, for nucleic acid sequencing, substantially all means that at least eighty percent of the polynucleotides have unique tags attached. More preferably, it means that at least ninety percent of the polynucleotides have unique tags attached. Still more preferably, it means that at least ninety-five percent of the polynucleotides have unique tags attached. And, most preferably, it means that at least ninety-nine percent of the polynucleotides have unique tags attached.</p>
    <p>Tags can be conjugated to cDNAs of existing libraries by standard cloning methods. cDNAs are excised from their existing vector, isolated, and then ligated into a vector containing a repertoire of tags. Preferably, the tag-containing vector is linearized by cleaving with two restriction enzymes so that the excised cDNAs can be ligated in a predetermined orientation. The concentration of the linearized tag-containing vector is in substantial excess over that of the cDNA inserts so that ligation provides an inherent sampling of tags.</p>
    <p>A general method for exposing the single stranded tag after amplification involves digesting a target polynucleotide-containing conjugate with the 5′3′ exonuclease activity of T4 DNA polymerase, or a like enzyme, e.g. as described in Kuijper et al, Gene, 112: 147-155 (1992). When used in the presence of a single deoxynucleoside triphosphate, such a polymerase will cleave nucleotides from 3′ recessed ends present on the non-template strand of a double stranded fragment until a complement of the single deoxynucleoside triphosphate is reached on the template strand. When such a nucleotide is reached the 5′3′ digestion effectively ceases, as the polymerase's extension activity adds nucleotides at a higher rate than the excision activity removes nucleotides. Consequently, single stranded tags constructed with three nucleotides are readily prepared for loading onto solid phase supports.</p>
    <p>After the oligonucleotide tags are prepared for specific hybridization, e.g. by rendering them single stranded as described above, the polynucleotides are mixed with microparticles containing the complementary sequences of the tags under conditions that favor the formation of perfectly matched duplexes between the tags and their complements. There is extensive guidance in the literature for creating these conditions. Exemplary references providing such guidance include Wetmur, Critical Reviews in Biochemistry and Molecular Biology, 26: 227-259 (1991); Sambrook et al, Molecular Cloning: A Laboratory Manual, 2nd Edition (Cold Spring Harbor Laboratory, New York, 1989); and the like. Preferably, the hybridization conditions are sufficiently stringent so that only perfectly matched sequences form stable duplexes. Under such conditions the polynucleotides specifically hybridized through their tags may be ligated to the complementary sequences attached to the microparticles. Finally, the microparticles are washed to remove polynucleotides with unligated and/or mismatched tags.</p>
    <p>Preferably, for sequencing applications, standard CPG beads of diameter in the range of 20-50 μm are loaded with about 10<sup>5 </sup>polynucleotides, and glycidalmethacrylate (GMA) beads available from Bangs Laboratories (Carmel, Ind.) of diameter in the range of 5-10 μm are loaded with a few tens of thousand polynucleotide, e.g. 4×10<sup>4 </sup>to 6×10<sup>4</sup>, to a hundred thousand polynucleotides.</p>
    <heading>DNA Sequencing</heading> <p>Polynucleotides loaded onto microparticles may be simultaneously sequenced in the instant apparatus using a “base-by-base” DNA sequencing methodology. Such sequencing methodology permits the stepwise identification of a sequence of nucleotides in a target polynucleotide, usually one base at a time, through successive cycles of treatment and detection. Base-by-base approaches are disclosed in the following references: Cheeseman, U.S. Pat. No. 5,302,509; Tsien et al, International application WO 91/06678; Rosenthal et al, International application WO 93/21340; Canard et al, Gene, 148: 1-6 (1994); Metzker et al, Nucleic Acids Research, 22: 4259-4267 (1994); and the like. Preferably, the base-by-base approach disclosed by Brenner in U.S. Pat. No. 5,599,675 is used with the apparatus of the invention to sequence polynucleotides on a population of loaded microparticles disposed as a planar array in the flow chamber. Accordingly, Brenner, U.S. Pat. No. 5,599,675 is incorporated by reference. Preferably, the a population of loaded microparticles for sequencing includes at least ten thousand loaded microparticles; more preferably, such a population includes at least fifty thousand loaded microparticles; and still more preferably, such a population includes at least one hundred thousand loaded microparticles.</p>
    <p>Preferably, the sequencing method of Brenner (cited above) is employed in the embodiment disclosed in Albrecht et al International patent application PCT/US97/09472 which discloses the use of encoded adaptors. An encoded adaptor is a double stranded oligonucleotide comprising a protruding strand and an oligonucleotide tag selected from a minimally cross-hybridizing set of oligonucleotides. Encoded adaptors whose protruding strands form perfectly matched duplexes with the complementary protruding strands of the target polynucleotide are ligated. After ligation, the identity and ordering of the nucleotides in the protruding strands are determined, or “decoded,” by specifically hybridizing a labeled tag complement to its corresponding tag on the ligated adaptor. Encoded adaptors may be used in an adaptor-based method of DNA sequencing that includes repeated cycles of ligation, identification, and cleavage, such as the method described in Brenner (cited above). Briefly, such a method comprises the following steps: (a) ligating an encoded adaptor to an end of a polynucleotide, the encoded adaptor having a nuclease recognition site of a nuclease whose cleavage site is separate from its recognition site; (b) identifying one or more nucleotides at the end of the polynucleotide by the identity of the encoded adaptor ligated thereto; (c) cleaving the polynucleotide with a nuclease recognizing the nuclease recognition site of the encoded adaptor such that the polynucleotide is shortened by one or more nucleotides; and (d) repeating said steps (a) through (c) until said nucleotide sequence of the polynucleotide is determined. In the identification step, successive sets of tag complements are specifically hybridized to the respective tags carried by encoded adaptors ligated to the ends of the target polynucleotides, as described above. The type and sequence of nucleotides in the protruding strands of the polynucleotides are identified by the label carried by the specifically hybridized tag complement and the set from which the tag complement came.</p>
    <heading>Construction and Sorting of cDNA Library for Signature Sequencing with Encoded Adaptors</heading> <p>In this example, a cDNA library is constructed in which an oligonucleotide tag consisting of 8 four-nucleotide “words” is attached to each cDNA. As described above, the repertoire of oligonucleotide tags of this size is sufficiently large (about 10<sup>8</sup>) so that if the cDNAs are synthesized from a population of about 10<sup>6 </sup>mRNAs, then there is a high probability that each cDNA will have a unique tag for sorting. After mRNA extraction, first strand synthesis is carried out in the presence of 5-Me-dCTP (to block certain cDNA restriction sites) and a biotinylated primer mixture containing the oligonucleotide tags. After conventional second strand synthesis, the tag-cDNA conjugates are cleaved with Dpn II (which is unaffected by the 5-Me-deoxycytosines), the biotinylated portions are separated from the reaction mixture using streptavidin-coated magnetic beads, and the tag-cDNA conjugates are recovered by cleaving them from the magnetic beads via a Bsm BI site carried by the biotinylated primer. The Bsm BI-Dpn II fragment containing the tag-cDNA conjugate is then inserted into a plasmid and amplified. After isolation of the plasmids, tag-cDNA conjugates are amplified out of the plasmids by PCR in the presence of 5-Me-dCTP, using biotinylated and fluorescently labeled primers containing pre-defined restriction endonuclease sites. After affinity purification with streptavidin coated magnetic beads, the tag-cDNA conjugates are cleaved from the beads, treated with T4 DNA polymerase in the presence of dGTP to render the tags single stranded, and then combined with a repertoire of GMA beads having tag complements attached. After stringent hybridization and ligation, the GMA beads are sorted via FACS to produce an enriched population of GMA beads loaded with cDNAs. The enriched population of loaded GMA beads are immobilized in a planar array in a flow chamber where base-by-base sequence takes place using encoded adaptors, as disclosed in Albrecht et al, International patent application PCT/US97/09472.</p>
    <p>Approximately 5 μg of poly(A<sup>+</sup>) mRNA is extracted from DBY746 yeast cells using conventional protocols. First and second strand cDNA synthesis is carried out by combining 100-150 pmoles of the following primer (SEQ ID NO: 1):</p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" tabstyle="monospace" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">5′-biotin-</td>
              </tr> <tr class="description-tr"> <td class="description-td">ACTAAT<u>CGTCTC</u>ACTAT<u style="double">TTAATTAA</u> [W,W,W,G]<sub>8</sub>CC(T)<sub>18</sub>V-3′</td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p>with the poly(A<sup>+</sup>) mRNA using a Stratagene (La Jolla, Calif.) cDNA Synthesis Kit in accordance with the manufacturer's protocol. This results in cDNAs whose first stand deoxycytosines are methylated at the 5-carbon position. In the above formula, “V” is G, C, or A, “[W,W,W,G]” is a four-nucleotide word selected from Table II of Brenner, International patent application PCT/US96/09513, the single underlined portion is a Bsm BI recognition site, and the double underlined portion is a Pac I recognition site. After size fractionation (GIBCO-BRL cDNA Size Fractionation Kit) using conventional protocols, the cDNAs are digested with Dpn II (New England Bioscience, Beverly, Me.) using manufacturer's protocol and affinity purified with streptavidin-coated magnetic beads (M-280 beads, Dynal A. S., Oslo, Norway). The DNA captured by the beads is digested with Bsm BI to release the tag-cDNA conjugates for cloning into a modified pBCSK<sup>−</sup>vector (Stratagene, La Jolla, Calif.) using standard protocols. The pBCSK<sup>−</sup>vector is modified by adding a Bbs I site by inserting the following fragment (SEQ ID NO: 2) into the Kpn I/Eco RV digested vector.</p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" tabstyle="monospace" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">      CGAAGACCC</td>
              </tr> <tr class="description-tr"> <td class="description-td">3′-CATGGCTTCTGGGGATA-5′</td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p>Bsm BI/Dpn II digested tag-cDNA conjugate is inserted in the pBCSK<sup>−</sup>which is previously digested with Bbs I and Bam HI. After ligation, the vector is transfected into the manufacturer's recommended host for amplification.</p>
    <p>After isolating the above pBCSK- vector from a standard plasmid miniprep, the tag-cDNA conjugates are amplified by PCR in the presence of 5-Me-dCTP using 20-mer primers complementary to vector sequences flanking the tag-cDNA insert. The “upstream” primer, i.e. adjacent to the tag, is biotinylated and the “downstream” primer, i.e. adjacent to the cDNA, is labeled with fluorescein. After amplification, the PCR product is affinity purified then cleaved with Pac I to release fluorescently labeled tag-cDNA conjugates. The tags of the conjugates are rendered single stranded by treating them with T4 DNA polymerase in the presence of dGTP. After the reaction is quenched, the tag-cDNA conjugate is purified by phenol-chloroform extraction and combined with 5.5 mm GMA beads carrying tag complements, each tag complement having a 5′ phosphate. Hybridization is conducted under stringent conditions in the presence of a thermal stable ligase so that only tags forming perfectly matched duplexes with their complements are ligated. The GMA beads are washed and the loaded beads are concentrated by FACS sorting, using the fluorescently labeled cDNAs to identify loaded GMA beads. The tag-cDNA conjugates attached to the GMA beads are digested with Dpn II to remove the fluorescent label and treated with alkaline phosphatase to prepare the cDNAs for sequencing. That is, phasphatase is used to remove the 5′ phosphate from the ends of the cDNAs to prevent unwanted cDNA-cDNA ligations by way of the palindromic Dpn II site.</p>
    <p>The following cleavage adaptor (SEQ ID NO: 3) is ligated to the Dpn II-digested and phosphatase treated cDNAs:</p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" tabstyle="monospace" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">5′-pGATCAGCTGCTGCAAATTT</td>
              </tr> <tr class="description-tr"> <td class="description-td">       pTCGACGACGTTTAAA</td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p>After ligation, the 3′ phosphate is removed by alkaline phosphatase, the 5′ strand of the cDNA is treated with T4 DNA kinase, and the nick between the cleavage adaptor and cDNA is ligated. After cleavage by Bbv 1, encoded adaptors are ligated to the ends of the cDNAs and the beads are ready for loading into the flow chamber.</p>
    <p>Ligation of the adaptors to the target polynucleotide is carried out in a mixture consisting of 5 μl beads (20 mg), 3 μL NEB 10× ligase buffer, 5 μL adaptor mix (25 nM), 2.5 μL NEB T4 DNA ligase (2000 units/μL), and 14.5 μL distilled water. The mixture is incubated at 16° C. for 30 minutes, after which the beads are washed 3 times in TE (pH 8.0).</p>
    <p>After centrifugation and removal of TE, the 3′ phosphates of the ligated adaptors are removed by treating the polynucleotide-bead mixture with calf intestinal alkaline phosphatase (CIP) (New England Biolabs, Beverly, Me.), using the manufacturer's protocol. After removal of the 3′ phosphates, the CIP may be inactivated by proteolytic digestion, e.g. using Pronase™ (available form Boeringer Mannhiem, Indianapolis, Ind.), or an equivalent protease, with the manufacturer's protocol. The polynucleotide-bead mixture is then washed, treated with a mixture of T4 polynucleotide kinase and T4 DNA ligase (New England Biolabs, Beverly, Me.) to add a 5′ phosphate at the gap between the target polynucleotide and the adaptor, and to complete the litigation of the adaptors to the target polynucleotide. The bead-polynucleotide mixture is then washed in TE, diluted to a concentration of approximately 100 thousand beads per μL, and 5 μL of the resulting solution is loaded into a flow chamber with the help of the holders of FIG. <b>4</b>.</p>
    <p>The top strands of the following 16 sets of 64 encoded adaptors (SEQ ID NO: 4 through SEQ ID NO: 19) are each separately synthesized on an automated DNA synthesizer (model 392 Applied Biosystems, Foster City) using standard methods. The bottom strand, which is the same for all adaptors, is synthesized separately then hybridized to the respective top strands:</p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" tabstyle="monospace" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="1" colwidth="56pt" align="center"> </colspec> <colspec colname="2" colwidth="161pt" align="left"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="2" align="center" rowsep="1" class="description-td" colspan="2"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">SEQ ID NO.</td>
                <td class="description-td">Encoded Adaptor</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="2" align="center" rowsep="1" class="description-td" colspan="2"> </td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="1" colwidth="49pt" align="char" char="."> </colspec> <colspec colname="2" colwidth="168pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">4</td>
                <td class="description-td">5′-pANNNTACAGCTGCATCCCttggcgctgagg</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">       pATGCACGCGTAGGG-5′</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">5</td>
                <td class="description-td">5′-pNANNTACAGCTGCATCCCtgggcctgtaag</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">       pATGCACGCGTAGGG-5′</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">6</td>
                <td class="description-td">5′-pCNNNTACAGCTGCATCCCttgacgggtctc</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">       pATGCACGCGTAGGG-5′</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">7</td>
                <td class="description-td">5′-pNCNNTACAGCTGCATCCCtgCcCgCacagt</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">       pATGCACGCGTAGGG-5′</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">8</td>
                <td class="description-td">5′-pGNNNTACAGCTGCATCCCttcgcctcggac</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">       pATGCACGCGTAGGG-5′</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">9</td>
                <td class="description-td">5′-pNGNNTACAGCTGCATCCCtgatcCgctagc</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">       pATGCACGCGTAGGG-5′</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">10</td>
                <td class="description-td">5′-pTNNNTACAGCTGCATCCCttCcgaaccCgc</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">       pATGCACGCGTAGGG-5′</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">11</td>
                <td class="description-td">5′-pNTNNTACAGCTGCATCCCtgagggggatag</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">       pATGCACGCGTAGGG-5′</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">12</td>
                <td class="description-td">5′-pNNANTACAGCTGCATCCCttcccgctacac</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">       pATGCACGCGTAGGG-5′</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">13</td>
                <td class="description-td">5′-pNNNATACAGCTGCATCCCtgactccccgag</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">       pATGCACGCGTAGGG-5′</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">14</td>
                <td class="description-td">5′-pNNCNTACAGCTGCATCCCtgtgttgcgcgg</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">       pATGCACGCGTAGGG-5′</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">15</td>
                <td class="description-td">5′-pNNNCTACAGCTGCATCCCtOtaCagcagcg</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">       pATGCACGCGTAGGG-5′</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">16</td>
                <td class="description-td">5′-pNNGNTACAGCTGCATCCCtgtcgcgtcgtt</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">       pATGCACGCGTAGGG-5′</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">17</td>
                <td class="description-td">5′-pNNNGTACAGCTGCATCCCtcggagcaacct</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">       pATGCACGCGTAGGG-5′</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">18</td>
                <td class="description-td">5′-pNNTNTACAGCTGCATCCCtggtgaccgtag</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">       pATGCACGCGTAGGG-5′</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">19</td>
                <td class="description-td">5′-pNNNTTACAGCTGCATCCCtcccctgtcgga</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">       pATGCACGCGTAGGG-5′</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="2" align="center" rowsep="1" class="description-td" colspan="2"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p>where N and p are as defined above, and the nucleotides indicated in lower case letters are the 12-mer oligonucleotide tags. Each tag differs from every other by 6 nucleotides. Equal molar quantities of each adaptor are combined in NEB #2 restriction buffer (New England Biolabs, Beverly, Me.) to form a mixture at a concentration of 1000 pmol/μL.</p>
    <p>Each of the 16 tag complements are separately synthesized as amino-derivatized oligonucleotides and are each labeled with a fluorescein molecule (using an NHS-ester of fluorescein, available from Molecular Probes, Eugene, Oreg.) which is attached to the 5′ end of the tag complement through a polyethylene glycol linker (Clonetech Laboratories, Palo Alto, Calif.). The sequences of the tag complements are simply the 12-mer complements of the tags listed above.</p>
    <p>A flow chamber of the design shown in FIGS. 2<i>a </i>and <b>2</b> <i>b </i>is employed in association with an Olympus Optical Co., Ltd. (Tokyo, Japan) model BX60MF5 fluorescent microscope fitted with a model U-ULS75XE 75 watt Xenon arc lamp, a motorized filter wheel, a Ludl Electronic Products, Ltd. computer-controlled stage, and a Photometrics, Ltd. (Tucson, Ariz.) PXL CCD camera with a 2000×2000 pixel array. Appropriate bandpass filters (<b>122</b>) and (<b>124</b>) are employed for exciting fluorescein and transmitting fluorescent signal to CCD camera (<b>120</b>). Microparticle positions are determined by top-lighting with broadband light from Xenon lamp (<b>126</b>) reduced by a factor of about 10<sup>−4 </sup>with a neutral density filter. Fluorescent images are collected with about 2 minute exposure times.</p>
    <p>Height (<b>204</b>) of flow chamber (<b>201</b>) is selected to be 7 μm, or approximately 140% of the diameter of the GMA beads. Width (<b>210</b>) of flow chamber (<b>201</b>) is selected so as to ensure that a 3x3 array of 9 image pixels will cover approximately 40-60% of a bead's image after 10× magnification (as illustrated in FIG. <b>7</b>). Thus, in order to capture images of tiles of about 100 thousand 5 sum GMA beads, width (<b>210</b>) is selected to have a value of 1.7 mm. Length (<b>212</b>) is selected so that the flow chamber can hold from 1 to 10 tiles of about one hundred thousand 5 μm diameter beads each. The cross section (<b>220</b>) of inlet passage (<b>214</b>) matches that of the inlet tubing and gradually enlarges to match that of flow chamber (<b>201</b>) in the region of the planar cavity, i.e. the region holding the GMA beads on which analysis is performed. It is desirable to have a constant cross section through the planar cavity of flow chamber (<b>201</b>) to minimize the creation of non-uniform flow patterns, as might occur with sudden constrictions and/or expansions in cross section. Both body (<b>218</b>) and cover (<b>216</b>) of flow chamber (<b>201</b>) are glass, and the planar cavity and channels of body (<b>218</b>) are formed by standard chemical etching techniques. Cross section (<b>222</b>) of outlet passage (<b>224</b>) is selected to match the cross section of flow chamber (<b>201</b>) at dam (<b>202</b>).</p>
    <p>The fluidics system of FIG. 5<i>a </i>which includes all valves, syringe pump (<b>500</b>), and Peltier block (<b>152</b>), is controlled by code written in LabVIEW 5.0 (National Instruments, Austin, Tex.) and run on a Compact Deskpro Pentium-based microprocessor, which is connected to the various components of the fluidics system by standard I/O circuit boards. Detection system (<b>114</b>) and overall control of the instrument is effected through a Sun Microsystems (Mountain View, Calif.) Sparcstation 5.</p>
    <p>Three cycles of ligation, identification, and cleavage are carried out in flow chamber (<b>201</b>) to give the sequences of 12 nucleotides at the termini of each of appoximately 500,000 cDNAs. That is, five tiles of GMA beads are analyzed in the following series of process steps:</p>
    <p>1. Calibrate focal plane of GMA beads.</p>
    <p>2. Hybridize decoder.</p>
    <p>3. Autofocus on tile 1.</p>
    <p>4. Set focus to bead centers.</p>
    <p>5. Collect fluorescent image.</p>
    <p>6. Set focus to bead focal plane (scatter centers).</p>
    <p>7. Collect image.</p>
    <p>8. Repeat steps 4-7 for remaining tiles.</p>
    <p>9. Wash.</p>
    <p>10. Repeat steps 2-9 for remaining decoders.</p>
    <p>11. Cleave encoded adaptor.</p>
    <p>12. Wash.</p>
    <p>13. Ligate top strand of next encoded adaptor.</p>
    <p>14. Wash.</p>
    <p>15. Repeat steps 13-14.</p>
    <p>16. Kinase bottom strand of encoded adaptor.</p>
    <p>17. Wash.</p>
    <p>18. Ligate bottom strand of encoded adaptor.</p>
    <p>19. Wash.</p>
    <p>20. Repeat steps 2-9.</p>
    <p>21. Repeat steps 11-19 for next encoded adaptor.</p>
    <p>In steps 2-9, nucleotides of the cDNAs are identified by hybridizing tag complements to the encoded adaptors. Specifically hybridized tag complements are detected by exciting their fluorescent labels with illumination beam (<b>110</b>) from Xenon arc lamp (<b>126</b>). In step 13, encoded adaptors and T4 DNA ligase (Promega, Madison, Wis.) at about 0.75 units per μL are passed through the flow chamber at a flow rate of about 1-2 μL per minute for about 20-30 minutes at 16° C., after which wash of step 14 is executed by flowing, in succession, a solution of Pronase™ (Boehringer Mannheim, Indianapolis, Ind.), a salt wash solution, and an ethanol wash solution through the flow chamber, all with the same flow rate of 1-2 μL per minute and for durations of 15, 10, and 10 minutes, respectively. The salt wash solution is 150 mM NaCl and 10 mM Tris-HCl (pH 8.5), and the ethanol wash solution is 3:1 (v/v) solution of the salt wash solution and ethanol. The ligation and wash steps 13 and 14 are repeated once, after which the adaptors and the cDNAs are prepared for second strand ligation by passing T4 DNA kinase (New England Bioscience. Beverly, Me.) at 7 units per μL through the flow chamber at 37° C. with a flow rate of 1-2 μL per minute for 15-20 minutes. Ligation of the second strand is carried out by flowing T4 DNA ligase (0.75 units per mL, Promega) through the flow chamber for 20-30 minutes at a rate of 1-2 μL per minute, followed by Pronase™ treatment and washing as described above. Tag complements at 25 nM concentration are passed through the flow chamber at a flow rate of 1-2 μL per minute for 10 minutes at 20° C., after which the fluorescent labels carried by the tag complements are illuminated and fluorescence is collected. The tag complements are melted from the encoded adaptors by passing NEB #2 restriction buffer with 3 mM MgCl<sub>2 </sub>through the flow chamber at a flow rate of 1-2 μL per minute at 55° C. for 10 minutes. Encoded adaptors are cleaved from the cDNAs by passing Bbv I (New England Biosciences, Beverly, Me.) at 1 unit/μL at a flow rate of 1-2 μL per minute for 20 minutes at 37° C., followed by Pronase™ treatment and washing, as described above.</p>
    </div></div></div><div class="patent-section patent-tabular-section"><a id="backward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Cited Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5429807">US5429807</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Oct 28, 1993</td><td class="patent-data-table-td patent-date-value">Jul 4, 1995</td><td class="patent-data-table-td ">Beckman Instruments, Inc.</td><td class="patent-data-table-td ">Method and apparatus for creating biopolymer arrays on a solid support surface</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5708153">US5708153</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jun 7, 1995</td><td class="patent-data-table-td patent-date-value">Jan 13, 1998</td><td class="patent-data-table-td ">Affymax Technologies N.V.</td><td class="patent-data-table-td ">Preparing a library</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="forward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Referenced by</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Citing Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6806052">US6806052</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Apr 18, 2002</td><td class="patent-data-table-td patent-date-value">Oct 19, 2004</td><td class="patent-data-table-td ">Lynx Therapeutics, Inc.</td><td class="patent-data-table-td ">Planar arrays of microparticle-bound polynucleotides</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7175810">US7175810</a></td><td class="patent-data-table-td patent-date-value">May 19, 2004</td><td class="patent-data-table-td patent-date-value">Feb 13, 2007</td><td class="patent-data-table-td ">Eksigent Technologies</td><td class="patent-data-table-td ">Processing of particles</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7220592">US7220592</a></td><td class="patent-data-table-td patent-date-value">Nov 15, 2002</td><td class="patent-data-table-td patent-date-value">May 22, 2007</td><td class="patent-data-table-td ">Eksigent Technologies, Llc</td><td class="patent-data-table-td ">Particulate processing system</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7282370">US7282370</a></td><td class="patent-data-table-td patent-date-value">Aug 18, 2005</td><td class="patent-data-table-td patent-date-value">Oct 16, 2007</td><td class="patent-data-table-td ">Solexa, Inc.</td><td class="patent-data-table-td ">System and apparatus for sequential processing of analytes</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7316899">US7316899</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jan 31, 2001</td><td class="patent-data-table-td patent-date-value">Jan 8, 2008</td><td class="patent-data-table-td ">The Board Of Regents Of The University Of Texas System</td><td class="patent-data-table-td ">Apparatus for high speed characterization and discrimination of toxins and/or bacteria in medical, food/beverage and environmental solutions</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7365179">US7365179</a></td><td class="patent-data-table-td patent-date-value">Sep 2, 2004</td><td class="patent-data-table-td patent-date-value">Apr 29, 2008</td><td class="patent-data-table-td ">Compass Genetics, Llc</td><td class="patent-data-table-td ">Solid phase support for use in genomic analysis; gene expression analysis; high throughput assay; genotyping</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7537897">US7537897</a></td><td class="patent-data-table-td patent-date-value">Jan 22, 2007</td><td class="patent-data-table-td patent-date-value">May 26, 2009</td><td class="patent-data-table-td ">Population Genetics Technologies, Ltd.</td><td class="patent-data-table-td ">Molecular counting</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7553620">US7553620</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">May 11, 2001</td><td class="patent-data-table-td patent-date-value">Jun 30, 2009</td><td class="patent-data-table-td ">Gnothis Holdings S.A.</td><td class="patent-data-table-td ">Method for determining polynucleotides in a sample without attaching these to a support, and using detection probes</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7622281">US7622281</a></td><td class="patent-data-table-td patent-date-value">May 19, 2005</td><td class="patent-data-table-td patent-date-value">Nov 24, 2009</td><td class="patent-data-table-td ">The Board Of Trustees Of The Leland Stanford Junior University</td><td class="patent-data-table-td ">Methods and compositions for clonal amplification of nucleic acid</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7635566">US7635566</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jun 27, 2008</td><td class="patent-data-table-td patent-date-value">Dec 22, 2009</td><td class="patent-data-table-td ">Population Genetics Technologies Ltd.</td><td class="patent-data-table-td ">Methods and compositions for isolating nucleic acid sequence variants</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7948015">US7948015</a></td><td class="patent-data-table-td patent-date-value">Dec 14, 2007</td><td class="patent-data-table-td patent-date-value">May 24, 2011</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for measuring analytes using large scale FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7978326">US7978326</a></td><td class="patent-data-table-td patent-date-value">Dec 3, 2009</td><td class="patent-data-table-td patent-date-value">Jul 12, 2011</td><td class="patent-data-table-td ">Applied Biosystems, Llc</td><td class="patent-data-table-td ">DNA sequencing system</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8058030">US8058030</a></td><td class="patent-data-table-td patent-date-value">Dec 7, 2009</td><td class="patent-data-table-td patent-date-value">Nov 15, 2011</td><td class="patent-data-table-td ">Applied Biosystems, Llc</td><td class="patent-data-table-td ">Methods of producing and sequencing modified polynucleotides</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8169610">US8169610</a></td><td class="patent-data-table-td patent-date-value">Jun 6, 2011</td><td class="patent-data-table-td patent-date-value">May 1, 2012</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">DNA sequencing system</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8198028">US8198028</a></td><td class="patent-data-table-td patent-date-value">Jul 2, 2009</td><td class="patent-data-table-td patent-date-value">Jun 12, 2012</td><td class="patent-data-table-td ">Illumina Cambridge Limited</td><td class="patent-data-table-td ">Using populations of beads for the fabrication of arrays on surfaces</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8217433">US8217433</a></td><td class="patent-data-table-td patent-date-value">Mar 15, 2012</td><td class="patent-data-table-td patent-date-value">Jul 10, 2012</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">One-transistor pixel array</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8222040">US8222040</a></td><td class="patent-data-table-td patent-date-value">Aug 28, 2007</td><td class="patent-data-table-td patent-date-value">Jul 17, 2012</td><td class="patent-data-table-td ">Lightspeed Genomics, Inc.</td><td class="patent-data-table-td ">Nucleic acid sequencing by selective excitation of microparticles</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8241850">US8241850</a></td><td class="patent-data-table-td patent-date-value">Nov 9, 2009</td><td class="patent-data-table-td patent-date-value">Aug 14, 2012</td><td class="patent-data-table-td ">Population Genetics Technologies Ltd.</td><td class="patent-data-table-td ">Methods and compositions for isolating nucleic acid sequence variants</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8247849">US8247849</a></td><td class="patent-data-table-td patent-date-value">Mar 15, 2012</td><td class="patent-data-table-td patent-date-value">Aug 21, 2012</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Two-transistor pixel array</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8262900">US8262900</a></td><td class="patent-data-table-td patent-date-value">Dec 17, 2007</td><td class="patent-data-table-td patent-date-value">Sep 11, 2012</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for measuring analytes using large scale FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8263336">US8263336</a></td><td class="patent-data-table-td patent-date-value">May 31, 2011</td><td class="patent-data-table-td patent-date-value">Sep 11, 2012</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for measuring analytes</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8264014">US8264014</a></td><td class="patent-data-table-td patent-date-value">Feb 2, 2011</td><td class="patent-data-table-td patent-date-value">Sep 11, 2012</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for measuring analytes using large scale FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8269261">US8269261</a></td><td class="patent-data-table-td patent-date-value">Feb 8, 2011</td><td class="patent-data-table-td patent-date-value">Sep 18, 2012</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for measuring analytes using large scale FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8293082">US8293082</a></td><td class="patent-data-table-td patent-date-value">Feb 28, 2011</td><td class="patent-data-table-td patent-date-value">Oct 23, 2012</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for measuring analytes using large scale FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8306757">US8306757</a></td><td class="patent-data-table-td patent-date-value">Mar 10, 2010</td><td class="patent-data-table-td patent-date-value">Nov 6, 2012</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for measuring analytes using large scale FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8313625">US8313625</a></td><td class="patent-data-table-td patent-date-value">Feb 28, 2011</td><td class="patent-data-table-td patent-date-value">Nov 20, 2012</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for measuring analytes using large scale FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8313639">US8313639</a></td><td class="patent-data-table-td patent-date-value">Feb 28, 2011</td><td class="patent-data-table-td patent-date-value">Nov 20, 2012</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for measuring analytes using large scale FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8317999">US8317999</a></td><td class="patent-data-table-td patent-date-value">May 31, 2011</td><td class="patent-data-table-td patent-date-value">Nov 27, 2012</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for measuring analytes using large scale FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8329404">US8329404</a></td><td class="patent-data-table-td patent-date-value">Mar 2, 2012</td><td class="patent-data-table-td patent-date-value">Dec 11, 2012</td><td class="patent-data-table-td ">Applied Biosystems Llc</td><td class="patent-data-table-td ">Reagents, methods, and libraries for bead-based sequencing</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8349167">US8349167</a></td><td class="patent-data-table-td patent-date-value">Jun 26, 2009</td><td class="patent-data-table-td patent-date-value">Jan 8, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for detecting molecular interactions using FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8384898">US8384898</a></td><td class="patent-data-table-td patent-date-value">Apr 9, 2012</td><td class="patent-data-table-td patent-date-value">Feb 26, 2013</td><td class="patent-data-table-td ">Applied Biosystems, Llc</td><td class="patent-data-table-td ">DNA sequencing system</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8384899">US8384899</a></td><td class="patent-data-table-td patent-date-value">Apr 9, 2012</td><td class="patent-data-table-td patent-date-value">Feb 26, 2013</td><td class="patent-data-table-td ">Applied Biosystems, Llc</td><td class="patent-data-table-td ">DNA sequencing system</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8399192">US8399192</a></td><td class="patent-data-table-td patent-date-value">Feb 3, 2012</td><td class="patent-data-table-td patent-date-value">Mar 19, 2013</td><td class="patent-data-table-td ">Illumina Cambridge Limited</td><td class="patent-data-table-td ">Using populations of beads for the fabrication of arrays on surfaces</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8415176">US8415176</a></td><td class="patent-data-table-td patent-date-value">Jun 30, 2011</td><td class="patent-data-table-td patent-date-value">Apr 9, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">One-transistor pixel array</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8415177">US8415177</a></td><td class="patent-data-table-td patent-date-value">Jun 30, 2011</td><td class="patent-data-table-td patent-date-value">Apr 9, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Two-transistor pixel array</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8415716">US8415716</a></td><td class="patent-data-table-td patent-date-value">Jul 23, 2012</td><td class="patent-data-table-td patent-date-value">Apr 9, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Chemically sensitive sensors with feedback circuits</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8421437">US8421437</a></td><td class="patent-data-table-td patent-date-value">Mar 15, 2012</td><td class="patent-data-table-td patent-date-value">Apr 16, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Array column integrator</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8426898">US8426898</a></td><td class="patent-data-table-td patent-date-value">Jan 22, 2010</td><td class="patent-data-table-td patent-date-value">Apr 23, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for measuring analytes using large scale FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8426899">US8426899</a></td><td class="patent-data-table-td patent-date-value">Feb 2, 2011</td><td class="patent-data-table-td patent-date-value">Apr 23, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for measuring analytes using large scale FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8431691">US8431691</a></td><td class="patent-data-table-td patent-date-value">Jul 21, 2008</td><td class="patent-data-table-td patent-date-value">Apr 30, 2013</td><td class="patent-data-table-td ">Applied Biosystems Llc</td><td class="patent-data-table-td ">Reagents, methods, and libraries for bead-based sequencing</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8432149">US8432149</a></td><td class="patent-data-table-td patent-date-value">Jun 30, 2011</td><td class="patent-data-table-td patent-date-value">Apr 30, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Array column integrator</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8432150">US8432150</a></td><td class="patent-data-table-td patent-date-value">Mar 15, 2012</td><td class="patent-data-table-td patent-date-value">Apr 30, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods for operating an array column integrator</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8435395">US8435395</a></td><td class="patent-data-table-td patent-date-value">Sep 14, 2012</td><td class="patent-data-table-td patent-date-value">May 7, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for measuring analytes using large scale FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8441044">US8441044</a></td><td class="patent-data-table-td patent-date-value">Jul 20, 2012</td><td class="patent-data-table-td patent-date-value">May 14, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods for manufacturing low noise chemically-sensitive field effect transistors</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8445945">US8445945</a></td><td class="patent-data-table-td patent-date-value">Jul 20, 2012</td><td class="patent-data-table-td patent-date-value">May 21, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Low noise chemically-sensitive field effect transistors</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8450781">US8450781</a></td><td class="patent-data-table-td patent-date-value">Sep 14, 2012</td><td class="patent-data-table-td patent-date-value">May 28, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for measuring analytes using large scale FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8455927">US8455927</a></td><td class="patent-data-table-td patent-date-value">Jun 30, 2011</td><td class="patent-data-table-td patent-date-value">Jun 4, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">One-transistor pixel array with cascoded column circuit</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8470164">US8470164</a></td><td class="patent-data-table-td patent-date-value">Jun 25, 2009</td><td class="patent-data-table-td patent-date-value">Jun 25, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for measuring analytes using large scale FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8487790">US8487790</a></td><td class="patent-data-table-td patent-date-value">Jun 30, 2011</td><td class="patent-data-table-td patent-date-value">Jul 16, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Chemical detection circuit including a serializer circuit</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8492799">US8492799</a></td><td class="patent-data-table-td patent-date-value">May 31, 2011</td><td class="patent-data-table-td patent-date-value">Jul 23, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for detecting molecular interactions using FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8492800">US8492800</a></td><td class="patent-data-table-td patent-date-value">Jul 20, 2012</td><td class="patent-data-table-td patent-date-value">Jul 23, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Chemically sensitive sensors with sample and hold capacitors</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8496802">US8496802</a></td><td class="patent-data-table-td patent-date-value">Jul 23, 2012</td><td class="patent-data-table-td patent-date-value">Jul 30, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods for operating chemically-sensitive sample and hold sensors</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8502278">US8502278</a></td><td class="patent-data-table-td patent-date-value">Jul 23, 2012</td><td class="patent-data-table-td patent-date-value">Aug 6, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Chemically-sensitive sample and hold sensors</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8502867">US8502867</a></td><td class="patent-data-table-td patent-date-value">Mar 19, 2010</td><td class="patent-data-table-td patent-date-value">Aug 6, 2013</td><td class="patent-data-table-td ">Lightspeed Genomics, Inc.</td><td class="patent-data-table-td ">Synthetic aperture optics imaging method using minimum selective excitation patterns</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8519448">US8519448</a></td><td class="patent-data-table-td patent-date-value">Jul 23, 2012</td><td class="patent-data-table-td patent-date-value">Aug 27, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Chemically-sensitive array with active and reference sensors</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8524057">US8524057</a></td><td class="patent-data-table-td patent-date-value">Jul 28, 2011</td><td class="patent-data-table-td patent-date-value">Sep 3, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for measuring analytes using large scale FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8524487">US8524487</a></td><td class="patent-data-table-td patent-date-value">Mar 15, 2012</td><td class="patent-data-table-td patent-date-value">Sep 3, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">One-transistor pixel array with cascoded column circuit</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8530941">US8530941</a></td><td class="patent-data-table-td patent-date-value">Feb 8, 2011</td><td class="patent-data-table-td patent-date-value">Sep 10, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for measuring analytes using large scale FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8535513">US8535513</a></td><td class="patent-data-table-td patent-date-value">Sep 14, 2012</td><td class="patent-data-table-td patent-date-value">Sep 17, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for measuring analytes using large scale FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8540865">US8540865</a></td><td class="patent-data-table-td patent-date-value">Sep 8, 2011</td><td class="patent-data-table-td patent-date-value">Sep 24, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for detecting molecular interactions using FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8540866">US8540866</a></td><td class="patent-data-table-td patent-date-value">Sep 9, 2011</td><td class="patent-data-table-td patent-date-value">Sep 24, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for detecting molecular interactions using FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8540867">US8540867</a></td><td class="patent-data-table-td patent-date-value">Sep 9, 2011</td><td class="patent-data-table-td patent-date-value">Sep 24, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for detecting molecular interactions using FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8540868">US8540868</a></td><td class="patent-data-table-td patent-date-value">Sep 9, 2011</td><td class="patent-data-table-td patent-date-value">Sep 24, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for detecting molecular interactions using FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8552771">US8552771</a></td><td class="patent-data-table-td patent-date-value">Apr 22, 2013</td><td class="patent-data-table-td patent-date-value">Oct 8, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">System for reducing noise in a chemical sensor array</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8558288">US8558288</a></td><td class="patent-data-table-td patent-date-value">Feb 28, 2011</td><td class="patent-data-table-td patent-date-value">Oct 15, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for measuring analytes using large scale FET arrays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8575664">US8575664</a></td><td class="patent-data-table-td patent-date-value">Jul 23, 2012</td><td class="patent-data-table-td patent-date-value">Nov 5, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Chemically-sensitive sensor array calibration circuitry</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8592153">US8592153</a></td><td class="patent-data-table-td patent-date-value">Sep 5, 2012</td><td class="patent-data-table-td patent-date-value">Nov 26, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods for manufacturing high capacitance microwell structures of chemically-sensitive sensors</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8592154">US8592154</a></td><td class="patent-data-table-td patent-date-value">Sep 7, 2012</td><td class="patent-data-table-td patent-date-value">Nov 26, 2013</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for high speed operation of a chemically-sensitive sensor array</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8625094">US8625094</a></td><td class="patent-data-table-td patent-date-value">Sep 14, 2012</td><td class="patent-data-table-td patent-date-value">Jan 7, 2014</td><td class="patent-data-table-td ">Applied Biosystems, Llc</td><td class="patent-data-table-td ">DNA sequencing system</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8653567">US8653567</a></td><td class="patent-data-table-td patent-date-value">Jun 30, 2011</td><td class="patent-data-table-td patent-date-value">Feb 18, 2014</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Chemically sensitive sensor with lightly doped drains</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8658017">US8658017</a></td><td class="patent-data-table-td patent-date-value">Jul 20, 2012</td><td class="patent-data-table-td patent-date-value">Feb 25, 2014</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods for operating an array of chemically-sensitive sensors</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8673627">US8673627</a></td><td class="patent-data-table-td patent-date-value">May 24, 2010</td><td class="patent-data-table-td patent-date-value">Mar 18, 2014</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Apparatus and methods for performing electrochemical reactions</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8685230">US8685230</a></td><td class="patent-data-table-td patent-date-value">Jul 24, 2012</td><td class="patent-data-table-td patent-date-value">Apr 1, 2014</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for high-speed operation of a chemically-sensitive sensor array</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8685324">US8685324</a></td><td class="patent-data-table-td patent-date-value">Jun 30, 2011</td><td class="patent-data-table-td patent-date-value">Apr 1, 2014</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Matched pair transistor circuits</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8685678">US8685678</a></td><td class="patent-data-table-td patent-date-value">Mar 28, 2013</td><td class="patent-data-table-td patent-date-value">Apr 1, 2014</td><td class="patent-data-table-td ">Population Genetics Technologies Ltd</td><td class="patent-data-table-td ">Increasing confidence of allele calls with molecular counting</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8692298">US8692298</a></td><td class="patent-data-table-td patent-date-value">May 7, 2013</td><td class="patent-data-table-td patent-date-value">Apr 8, 2014</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Chemical sensor array having multiple sensors per well</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8697004">US8697004</a></td><td class="patent-data-table-td patent-date-value">May 19, 2010</td><td class="patent-data-table-td patent-date-value">Apr 15, 2014</td><td class="patent-data-table-td ">Applied Biosystems, Llc</td><td class="patent-data-table-td ">Sequencing system with memory</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8698212">US8698212</a></td><td class="patent-data-table-td patent-date-value">Aug 30, 2012</td><td class="patent-data-table-td patent-date-value">Apr 15, 2014</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Active chemically-sensitive sensors</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8715967">US8715967</a></td><td class="patent-data-table-td patent-date-value">Mar 29, 2013</td><td class="patent-data-table-td patent-date-value">May 6, 2014</td><td class="patent-data-table-td ">Population Genetics Technologies Ltd.</td><td class="patent-data-table-td ">Method for accurately counting starting molecules</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8722368">US8722368</a></td><td class="patent-data-table-td patent-date-value">Mar 29, 2013</td><td class="patent-data-table-td patent-date-value">May 13, 2014</td><td class="patent-data-table-td ">Population Genetics Technologies Ltd.</td><td class="patent-data-table-td ">Method for preparing a counter-tagged population of nucleic acid molecules</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8728766">US8728766</a></td><td class="patent-data-table-td patent-date-value">Mar 29, 2013</td><td class="patent-data-table-td patent-date-value">May 20, 2014</td><td class="patent-data-table-td ">Population Genetics Technologies Ltd.</td><td class="patent-data-table-td ">Method of adding a DBR by primer extension</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8731847">US8731847</a></td><td class="patent-data-table-td patent-date-value">Jun 30, 2011</td><td class="patent-data-table-td patent-date-value">May 20, 2014</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Array configuration and readout scheme</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8741571">US8741571</a></td><td class="patent-data-table-td patent-date-value">Mar 13, 2013</td><td class="patent-data-table-td patent-date-value">Jun 3, 2014</td><td class="patent-data-table-td ">Illumina Cambridge Limited</td><td class="patent-data-table-td ">Using populations of beads for the fabrication of arrays on surfaces</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8741606">US8741606</a></td><td class="patent-data-table-td patent-date-value">Mar 29, 2013</td><td class="patent-data-table-td patent-date-value">Jun 3, 2014</td><td class="patent-data-table-td ">Population Genetics Technologies Ltd.</td><td class="patent-data-table-td ">Method of tagging using a split DBR</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8741680">US8741680</a></td><td class="patent-data-table-td patent-date-value">Mar 22, 2013</td><td class="patent-data-table-td patent-date-value">Jun 3, 2014</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Two-transistor pixel array</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8742469">US8742469</a></td><td class="patent-data-table-td patent-date-value">Aug 30, 2012</td><td class="patent-data-table-td patent-date-value">Jun 3, 2014</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Active chemically-sensitive sensors with correlated double sampling</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8742471">US8742471</a></td><td class="patent-data-table-td patent-date-value">Mar 21, 2013</td><td class="patent-data-table-td patent-date-value">Jun 3, 2014</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Chemical sensor array with leakage compensation circuit</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8742472">US8742472</a></td><td class="patent-data-table-td patent-date-value">Jun 19, 2013</td><td class="patent-data-table-td patent-date-value">Jun 3, 2014</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Chemically sensitive sensors with sample and hold capacitors</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8747748">US8747748</a></td><td class="patent-data-table-td patent-date-value">Jan 19, 2012</td><td class="patent-data-table-td patent-date-value">Jun 10, 2014</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Chemical sensor with conductive cup-shaped sensor surface</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8748947">US8748947</a></td><td class="patent-data-table-td patent-date-value">Aug 30, 2012</td><td class="patent-data-table-td patent-date-value">Jun 10, 2014</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Active chemically-sensitive sensors with reset switch</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8753812">US8753812</a></td><td class="patent-data-table-td patent-date-value">Mar 29, 2012</td><td class="patent-data-table-td patent-date-value">Jun 17, 2014</td><td class="patent-data-table-td ">The Board Of Trustees Of The Leland Stanford Junior University</td><td class="patent-data-table-td ">Charge perturbation detection method for DNA and other molecules</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8759077">US8759077</a></td><td class="patent-data-table-td patent-date-value">Aug 28, 2007</td><td class="patent-data-table-td patent-date-value">Jun 24, 2014</td><td class="patent-data-table-td ">Lightspeed Genomics, Inc.</td><td class="patent-data-table-td ">Apparatus for selective excitation of microparticles</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8764969">US8764969</a></td><td class="patent-data-table-td patent-date-value">Jul 20, 2012</td><td class="patent-data-table-td patent-date-value">Jul 1, 2014</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods for operating chemically sensitive sensors with sample and hold capacitors</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8766327">US8766327</a></td><td class="patent-data-table-td patent-date-value">Aug 30, 2012</td><td class="patent-data-table-td patent-date-value">Jul 1, 2014</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Active chemically-sensitive sensors with in-sensor current sources</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8766328">US8766328</a></td><td class="patent-data-table-td patent-date-value">Jun 20, 2013</td><td class="patent-data-table-td patent-date-value">Jul 1, 2014</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Chemically-sensitive sample and hold sensors</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8772698">US8772698</a></td><td class="patent-data-table-td patent-date-value">Jun 30, 2011</td><td class="patent-data-table-td patent-date-value">Jul 8, 2014</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">CCD-based multi-transistor active pixel sensor array</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8776573">US8776573</a></td><td class="patent-data-table-td patent-date-value">Dec 21, 2011</td><td class="patent-data-table-td patent-date-value">Jul 15, 2014</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">Methods and apparatus for measuring analytes</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8786331">US8786331</a></td><td class="patent-data-table-td patent-date-value">Mar 13, 2013</td><td class="patent-data-table-td patent-date-value">Jul 22, 2014</td><td class="patent-data-table-td ">Life Technologies Corporation</td><td class="patent-data-table-td ">System for reducing noise in a chemical sensor array</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2003214A2?cl=en">EP2003214A2</a></td><td class="patent-data-table-td patent-date-value">Feb 1, 2006</td><td class="patent-data-table-td patent-date-value">Dec 17, 2008</td><td class="patent-data-table-td ">AB Advanced Genetic Analysis Corporation</td><td class="patent-data-table-td ">Reagents, methods, and libraries for bead-based sequencing</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2230315A1?cl=en">EP2230315A1</a></td><td class="patent-data-table-td patent-date-value">Feb 1, 2006</td><td class="patent-data-table-td patent-date-value">Sep 22, 2010</td><td class="patent-data-table-td ">AB Advanced Genetic Analysis Corporation</td><td class="patent-data-table-td ">Nucleic acid sequencing by performing successive cycles of duplex extension</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2230316A1?cl=en">EP2230316A1</a></td><td class="patent-data-table-td patent-date-value">Feb 1, 2006</td><td class="patent-data-table-td patent-date-value">Sep 22, 2010</td><td class="patent-data-table-td ">AB Advanced Genetic Analysis Corporation</td><td class="patent-data-table-td ">Nucleic acid sequencing by performing successive cycles of duplex extension</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2233581A1?cl=en">EP2233581A1</a></td><td class="patent-data-table-td patent-date-value">Feb 1, 2006</td><td class="patent-data-table-td patent-date-value">Sep 29, 2010</td><td class="patent-data-table-td ">AB Advanced Genetic Analysis Corporation</td><td class="patent-data-table-td ">Nucleic acid sequencing by performing successive cycles of duplex extension</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2233582A1?cl=en">EP2233582A1</a></td><td class="patent-data-table-td patent-date-value">Feb 1, 2006</td><td class="patent-data-table-td patent-date-value">Sep 29, 2010</td><td class="patent-data-table-td ">AB Advanced Genetic Analysis Corporation</td><td class="patent-data-table-td ">Nucleic acid sequencing by performing successive cycles of duplex extension</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2233583A1?cl=en">EP2233583A1</a></td><td class="patent-data-table-td patent-date-value">Feb 1, 2006</td><td class="patent-data-table-td patent-date-value">Sep 29, 2010</td><td class="patent-data-table-td ">AB Advanced Genetic Analysis Corporation</td><td class="patent-data-table-td ">Nucleic acid sequencing by performing successive cycles of duplex extension</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2236628A2?cl=en">EP2236628A2</a></td><td class="patent-data-table-td patent-date-value">Feb 1, 2006</td><td class="patent-data-table-td patent-date-value">Oct 6, 2010</td><td class="patent-data-table-td ">AB Advanced Genetic Analysis Corporation</td><td class="patent-data-table-td ">Reagents, methods and libraries for bead-based sequencing</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2239342A2?cl=en">EP2239342A2</a></td><td class="patent-data-table-td patent-date-value">Feb 1, 2006</td><td class="patent-data-table-td patent-date-value">Oct 13, 2010</td><td class="patent-data-table-td ">AB Advanced Genetic Analysis Corporation</td><td class="patent-data-table-td ">Reagents, methods and libraries for bead-based sequencing</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2241637A1?cl=en">EP2241637A1</a></td><td class="patent-data-table-td patent-date-value">Feb 1, 2006</td><td class="patent-data-table-td patent-date-value">Oct 20, 2010</td><td class="patent-data-table-td ">AB Advanced Genetic Analysis Corporation</td><td class="patent-data-table-td ">Nucleic acid sequencing by performing successive cycles of duplex extension</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2272983A1?cl=en">EP2272983A1</a></td><td class="patent-data-table-td patent-date-value">Feb 1, 2006</td><td class="patent-data-table-td patent-date-value">Jan 12, 2011</td><td class="patent-data-table-td ">AB Advanced Genetic Analysis Corporation</td><td class="patent-data-table-td ">Reagents, methods and libraries for bead-based sequencing</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2316977A1?cl=en">EP2316977A1</a></td><td class="patent-data-table-td patent-date-value">Feb 1, 2006</td><td class="patent-data-table-td patent-date-value">May 4, 2011</td><td class="patent-data-table-td ">AB Advanced Genetic Analysis Corporation</td><td class="patent-data-table-td ">Reagents, methods and libraries for bead-based amflication</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2657869A2?cl=en">EP2657869A2</a></td><td class="patent-data-table-td patent-date-value">Aug 28, 2008</td><td class="patent-data-table-td patent-date-value">Oct 30, 2013</td><td class="patent-data-table-td ">Applied Biosystems, LLC</td><td class="patent-data-table-td ">Alternative nucleic acid sequencing methods</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2004046695A2?cl=en">WO2004046695A2</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Nov 14, 2003</td><td class="patent-data-table-td patent-date-value">Jun 3, 2004</td><td class="patent-data-table-td ">Eksigent Technologies Llc</td><td class="patent-data-table-td ">Method and system for processing and measuring microparticles</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2005026686A2?cl=en">WO2005026686A2</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Sep 2, 2004</td><td class="patent-data-table-td patent-date-value">Mar 24, 2005</td><td class="patent-data-table-td ">Sydney Brenner</td><td class="patent-data-table-td ">Multiplexed analytical platform</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2005116642A2?cl=en">WO2005116642A2</a></td><td class="patent-data-table-td patent-date-value">May 18, 2005</td><td class="patent-data-table-td patent-date-value">Dec 8, 2005</td><td class="patent-data-table-td ">Eksigent Technologies Llc</td><td class="patent-data-table-td ">Processing of particles</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="classifications"></a><div class="patent-section-header"><span class="patent-section-title">Classifications</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th"> </th><th class="patent-data-table-th"> </th></tr></thead><tr><td class="patent-data-table-td ">U.S. Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=OGJaBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S006110">435/6.11</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=OGJaBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S288300">435/288.3</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=OGJaBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S297500">435/297.5</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=OGJaBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S299100">435/299.1</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=OGJaBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S022100">536/22.1</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=OGJaBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S006100">435/6.1</a></span></td></tr><tr><td class="patent-data-table-td ">International Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=OGJaBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=B01L0003000000">B01L3/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=OGJaBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=G01N0021640000">G01N21/64</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=OGJaBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C40B0060140000">C40B60/14</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=OGJaBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=G01N0033566000">G01N33/566</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=OGJaBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=G01N0021250000">G01N21/25</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=OGJaBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=G01N0015000000">G01N15/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=OGJaBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=G01N0015140000">G01N15/14</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=OGJaBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12Q0001680000">C12Q1/68</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=OGJaBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=G01N0037000000">G01N37/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=OGJaBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=B01J0019000000">B01J19/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=OGJaBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=G01N0033530000">G01N33/53</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=OGJaBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=G01N0021050000">G01N21/05</a></span></td></tr><tr><td class="patent-data-table-td ">Cooperative Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=OGJaBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=G01N2015/0015">G01N2015/0015</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=OGJaBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=G01N15/1484">G01N15/1484</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=OGJaBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=B01L2400/0644">B01L2400/0644</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=OGJaBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=B01L2400/0403">B01L2400/0403</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=OGJaBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=G01N15/1475">G01N15/1475</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=OGJaBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=B01L2400/0622">B01L2400/0622</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=OGJaBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=B01J2219/00659">B01J2219/00659</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=OGJaBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=B01J2219/00274">B01J2219/00274</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=OGJaBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=B01L2300/0877">B01L2300/0877</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=OGJaBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=B01L2400/0475">B01L2400/0475</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=OGJaBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=B01J2219/00648">B01J2219/00648</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=OGJaBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=G01N21/253">G01N21/253</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=OGJaBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=B01L3/502715">B01L3/502715</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=OGJaBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=B01J2219/00317">B01J2219/00317</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=OGJaBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=G01N15/1456">G01N15/1456</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=OGJaBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=B01J19/0046">B01J19/0046</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=OGJaBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=B01L2200/0647">B01L2200/0647</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=OGJaBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=G01N2015/149">G01N2015/149</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=OGJaBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=B01L3/502761">B01L3/502761</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=OGJaBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12Q1/6874">C12Q1/6874</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=OGJaBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=B01L2200/10">B01L2200/10</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=OGJaBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C40B60/14">C40B60/14</a></span></td></tr><tr><td class="patent-data-table-td ">European Classification</td><td class="patent-data-table-td "><span class="nested-value">C12Q1/68E4</span>, <span class="nested-value">B01L3/5027B</span>, <span class="nested-value">B01L3/5027H</span>, <span class="nested-value">G01N15/14H3</span>, <span class="nested-value">B01J19/00C</span>, <span class="nested-value">G01N21/25B2</span>, <span class="nested-value">G01N15/14M</span></td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="legal-events"></a><div class="patent-section-header"><span class="patent-section-title">Legal Events</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Date</th><th class="patent-data-table-th">Code</th><th class="patent-data-table-th">Event</th><th class="patent-data-table-th">Description</th></tr></thead><tr><td class="patent-data-table-td patent-date-value">Nov 20, 2013</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">12</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Oct 4, 2011</td><td class="patent-data-table-td ">FPB1</td><td class="patent-data-table-td ">Expired due to reexamination which canceled all claims</td><td class="patent-data-table-td "></td></tr><tr><td class="patent-data-table-td patent-date-value">Nov 12, 2010</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">MERGER;ASSIGNOR:SOLEXA, INC.;REEL/FRAME:025357/0553</span></div><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">ILLUMINA, INC., CALIFORNIA</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20100208</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Feb 2, 2010</td><td class="patent-data-table-td ">RR</td><td class="patent-data-table-td ">Request for reexamination filed</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20091104</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Dec 18, 2009</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">8</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">May 15, 2007</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">SOLEXA, INC., CALIFORNIA</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">MERGER;ASSIGNOR:LYNX THERAPEUTICS, INC.;REEL/FRAME:019297/0520</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20050304</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Dec 19, 2005</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">4</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Aug 10, 1998</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">LYNX THERAPEUTICS, INC., CALIFORNIA</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNORS:BRIDGHAM, JOHN;CORCORAN, KEVIN;GOLDA, GEORGE;AND OTHERS;REEL/FRAME:009383/0367</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">19980729</span></div></td></tr></table><div class="patent-section-footer"></div></div><div class="modal-dialog" id="patent-images-lightbox"><div class="patent-lightbox-controls"><div class="patent-lightbox-rotate-controls"><div class="patent-lightbox-rotation-text">Rotate</div><div class="rotate-icon rotate-ccw-icon"></div><div class="rotate-icon rotate-cw-icon"></div></div><div class="patent-lightbox-index-counter"></div><a class="patent-lightbox-fullsize-link" target="_blank">Original Image</a><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_right.png" alt="Next page"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_left.png" alt="Previous page"width="21" height="21" /></div></div><div class="modal-dialog-content"><div class="patent-lightbox-image-holder"><div class="patent-lightbox-placeholder"></div></div></div></div><script>_OC_initPatentsAtb({image_not_available_html: " Image not available"});</script></div></div></div></td></tr></table><script>(function() {var href = window.location.href;if (href.indexOf('?') !== -1) {var parameters = href.split('?')[1].split('&');for (var i = 0; i < parameters.length; i++) {var param = parameters[i].split('=');if (param[0] == 'focus') {var elem = document.getElementById(param[1]);if (elem) {elem.focus();}}}}})();</script><script>_OC_addFlags({LockSrc:"/books/javascript/lock_8a2b04e7bf975d5171d8e4c0b6365c7a.js", Host:"http://www.google.com/", IsBooksRentalEnabled:1, IsWebstoreDisplayCaseEnabled:1, IsObfuscationEnabled:1, IsBrowsingHistoryEnabled:1, IsWebReaderSvgEnabled:0, IsGeoLayerEnabled:1, IsImageModeNotesEnabled:1, IsCopyMenuItemEnabled:1, IsGiftingEnabled:0, IsWebReaderUniversalPaginatorEnabled:0, IsOfflineBubbleEnabled:1, IsReaderEnabledForPlayRequests:1, IsFutureOnSaleVolumesEnabled:1, IsOfflineRestrictedCopyEnabled:1, IsBooksUnifiedLeftNavEnabled:1, IsRestrictedCopyEnabled:1, IsZipitFolderCollectionEnabled:1, IsEndOfSampleRecommendationsEnabled:1, IsRatingsOnBookcardsEnabled:1, IsAdsDisabled:0, IsIframePageDisplayEnabled:0, IsEmbeddedMediaEnabled:1, IsImageModeAnnotationsEnabled:1, IsMyLibraryGooglePlusEnabled:1, IsImagePageProviderEnabled:0, IsBookcardListPriceSmall:0, IsInternalUser:0, IsBooksShareButtonEnabled:0, IsPreOrdersEnabled:0, IsDisabledRandomBookshelves:0, WebstoreDisplayCasePosition:3});_OC_Run({"enable_p13n":false,"add_vol_to_collection_base_url":"http://www.google.com/patents?op=add\u0026sig=ACfU3U0wbbSuW9yzVh9oCzyYfcnsyyHMqw\u0026id=OGJaBAABERAJ","remove_vol_from_collection_base_url":"http://www.google.com/patents?op=remove\u0026sig=ACfU3U2F7tfV9iR0tUEq09rmBHk7l-l8_w\u0026id=OGJaBAABERAJ","logged_in":false,"p13n_save_user_settings_url":"http://www.google.com/patents?op=edit_user_settings\u0026sig=ACfU3U329jb8NyHyWAm6AaRxvEiLhtk0bg","is_cobrand":false,"sign_in_url":"https://www.google.com/accounts/Login?service=\u0026continue=http://www.google.com/patents%3Fhl%3Den\u0026hl=en","is_play_enabled":true}, {"volume_id":"","is_ebook":true,"volumeresult":{"has_flowing_text":false,"has_scanned_text":true,"can_download_pdf":false,"can_download_epub":false,"is_pdf_drm_enabled":false,"is_epub_drm_enabled":false,"download_pdf_url":"http://www.google.com/patents/download/Microparticles_and_polynucleotides.pdf?id=OGJaBAABERAJ\u0026output=pdf\u0026sig=ACfU3U3JULbCRfCpqMbv997C911-mLoCFA"},"sample_url":"http://www.google.com/patents/reader?id=OGJaBAABERAJ\u0026printsec=frontcover\u0026output=reader\u0026source=gbs_atb_hover","is_browsable":true,"is_public_domain":true}, {});</script><div id="footer_table" style="font-size:83%;text-align:center;position:relative;top:20px;height:4.5em;margin-top:2em"><div style="margin-bottom:8px"><a href=http://www.google.com/><nobr>Google&nbsp;Home</nobr></a> - <a href=//www.google.com/patents/sitemap/><nobr>Sitemap</nobr></a> - <a href=http://www.google.com/googlebooks/uspto.html><nobr>USPTO Bulk Downloads</nobr></a> - <a href=/intl/en/privacy/><nobr>Privacy Policy</nobr></a> - <a href=/intl/en/policies/terms/><nobr>Terms of Service</nobr></a> - <a href=https://support.google.com/faqs/answer/2539193?hl=en><nobr>About Google Patents</nobr></a> - <a href="http://www.google.com/tools/feedback/intl/en/error.html" onclick="try{_OC_startFeedback({productId: '72792',locale: 'en'});return false;}catch(e){}"><nobr>Send Feedback</nobr></a></div><span>Data provided by IFI CLAIMS Patent Services</span><br><span >&copy;2012 Google</span></div> <script type="text/javascript">var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));</script><script type="text/javascript">var pageTracker = _gat._getTracker("UA-27188110-1");pageTracker._setCookiePath("/patents/");pageTracker._trackPageview();</script> </body></html>