<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN""http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html><head><title>Patent US6410278 - Process for synthesizing nucleic acid - Google Patents</title><script>(function(){(function(){function e(a){this.t={};this.tick=function(a,c,b){var d=void 0!=b?b:(new Date).getTime();this.t[a]=[d,c];if(void 0==b)try{window.console.timeStamp("CSI/"+a)}catch(e){}};this.tick("start",null,a)}var a;window.performance&&(a=window.performance.timing);var f=a?new e(a.responseStart):new e;window.jstiming={Timer:e,load:f};if(a){var c=a.navigationStart,d=a.responseStart;0<c&&d>=c&&(window.jstiming.srt=d-c)}if(a){var b=window.jstiming.load;0<c&&d>=c&&(b.tick("_wtsrt",void 0,c),b.tick("wtsrt_",
"_wtsrt",d),b.tick("tbsd_","wtsrt_"))}try{a=null,window.chrome&&window.chrome.csi&&(a=Math.floor(window.chrome.csi().pageT),b&&0<c&&(b.tick("_tbnd",void 0,window.chrome.csi().startE),b.tick("tbnd_","_tbnd",c))),null==a&&window.gtbExternal&&(a=window.gtbExternal.pageT()),null==a&&window.external&&(a=window.external.pageT,b&&0<c&&(b.tick("_tbnd",void 0,window.external.startE),b.tick("tbnd_","_tbnd",c))),a&&(window.jstiming.pt=a)}catch(g){}})();})();
</script><link rel="stylesheet" href="/patents/css/_8a2b04e7bf975d5171d8e4c0b6365c7a/kl_intl_patents_bundle.css" type="text/css" /><script src="/books/javascript/atb_8a2b04e7bf975d5171d8e4c0b6365c7a__en.js"></script><script>function googleTranslateElementInit() {new google.translate.TranslateElement({pageLanguage: "en",gaTrack: true,gaId: "UA-27188110-1",multilanguagePage: true});}</script><script src="//translate.google.com/translate_a/element.js?cb=googleTranslateElementInit"></script><meta name="DC.type" content="Patent"><meta name="DC.title" content="Process for synthesizing nucleic acid"><meta name="DC.contributor" content="Tsugunori Notomi" scheme="inventor"><meta name="DC.contributor" content="Tetsu Hase" scheme="inventor"><meta name="DC.contributor" content="Eiken Kagaku Kabushiki Kaisha" scheme="assignee"><meta name="DC.date" content="1999-11-8" scheme="dateSubmitted"><meta name="DC.description" content="The present invention relates to an oligonucleotide having a novel structure and a method of synthesizing nucleic acid by using the same as a primer. This oligonucleotide is provided at the 5′-side of the primer with a nucleotide sequence substantially the same as a region synthesized with this primer as the origin of synthesis. The present invention realizes synthesis of nucleic acid based on an isothermal reaction with a simple constitution of reagents. Further, the present invention provides a method of synthesizing highly specific nucleic acid on the basis of this method of synthesizing nucleic acid."><meta name="DC.date" content="2002-6-25" scheme="issued"><meta name="DC.relation" content="EP:0549107:A1" scheme="references"><meta name="DC.relation" content="EP:0971039:A2" scheme="references"><meta name="DC.relation" content="US:4683202" scheme="references"><meta name="DC.relation" content="US:5270184" scheme="references"><meta name="DC.relation" content="US:5525462" scheme="references"><meta name="DC.relation" content="US:5595891" scheme="references"><meta name="DC.relation" content="US:5612199" scheme="references"><meta name="DC.relation" content="US:5874260" scheme="references"><meta name="DC.relation" content="US:6025139" scheme="references"><meta name="DC.relation" content="US:6033881" scheme="references"><meta name="DC.relation" content="WO:1996001327:A1" scheme="references"><meta name="DC.relation" content="WO:1999066071:A1" scheme="references"><meta name="citation_reference" content="Lizardi et al., &quot;Mutation Detection and Single-Molecule Counting Using Isothermal Rolling-Circle Amplification,&quot; Nature Genetics 19:225-232 (1998)."><meta name="citation_reference" content="Stratagene Catalog, &quot;Gene Characterization kits&quot;, p. 39, column 1, Feb. 1988.*"><meta name="citation_reference" content="Whitcombe et al., &quot;Detection of PCR Products Using Self-probing Amplicons and Fluorescence,&quot; Nature Biotechnology 17:804-807 (1999)."><meta name="citation_patent_number" content="US:6410278"><meta name="citation_patent_application_number" content="US:09/530,061"><link rel="canonical" href="http://www.google.com/patents/US6410278"/><meta property="og:url" content="http://www.google.com/patents/US6410278"/><meta name="title" content="Patent US6410278 - Process for synthesizing nucleic acid"/><meta name="description" content="The present invention relates to an oligonucleotide having a novel structure and a method of synthesizing nucleic acid by using the same as a primer. This oligonucleotide is provided at the 5′-side of the primer with a nucleotide sequence substantially the same as a region synthesized with this primer as the origin of synthesis. The present invention realizes synthesis of nucleic acid based on an isothermal reaction with a simple constitution of reagents. Further, the present invention provides a method of synthesizing highly specific nucleic acid on the basis of this method of synthesizing nucleic acid."/><meta property="og:title" content="Patent US6410278 - Process for synthesizing nucleic acid"/><meta property="og:type" content="book"/><meta property="og:site_name" content="Google Books"/><meta property="og:image" content="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><link rel="image_src" href="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><script>(function(){try{var aa=function(a,b,c,d){d=d||{};d._sn=["cfg",b,c].join(".");window.gbar.logger.ml(a,d)};var g=window.gbar=window.gbar||{},l=window.gbar.i=window.gbar.i||{},m={},n;function _tvn(a,b){var c=parseInt(a,10);return isNaN(c)?b:c}function _tvf(a,b){var c=parseFloat(a);return isNaN(c)?b:c}function _tvv(a){return!!a}function p(a,b,c){(c||g)[a]=b}g.bv={n:_tvn("2",0),r:"",f:".67.",e:"0",m:_tvn("0",1)};
function q(a,b,c){var d="on"+b;if(a.addEventListener)a.addEventListener(b,c,!1);else if(a.attachEvent)a.attachEvent(d,c);else{var f=a[d];a[d]=function(){var a=f.apply(this,arguments),b=c.apply(this,arguments);return void 0==a?b:void 0==b?a:b&&a}}}var s=function(a){return function(){return g.bv.m==a}},ba=s(1),ca=s(2);p("sb",ba);p("kn",ca);l.a=_tvv;l.b=_tvf;l.c=_tvn;l.i=aa;var da=window.gbar.i.i;var t,u,v,w;function ea(a){v=a}function fa(a){var b;if(b=v&&window.encodeURIComponent)b=a.href,b=!b.match(/^http[s]?:\/\/accounts\.google\.[^/]*\/ClearSID/i)&&!b.match(/^http[s]?:\/\/[^/]*\/accounts\/ClearSID/i);if(b=b&&encodeURIComponent(v()))a.href=a.href.replace(/([?&]continue=)[^&]*/,"$1"+b)}function ga(a){window.gApplication&&(a.href=window.gApplication.getTabUrl(a.href))}
function ha(a){var b=document.forms[0].q,c=window.encodeURIComponent&&b&&b.value,b=b&&b.placeholder;c&&c!=b&&(a.href=a.href.replace(/([?&])q=[^&]*|$/,function(a,b){return(b||"&")+"q="+encodeURIComponent(c)}))}n=l.a("")?ga:ha;
function x(a,b,c,d,f,e){var h=document.getElementById(a);if(h){var k=h.style;k.left=d?"auto":b+"px";k.right=d?b+"px":"auto";k.top=c+"px";k.visibility=u?"hidden":"visible";f&&e?(k.width=f+"px",k.height=e+"px"):(x(t,b,c,d,h.offsetWidth,h.offsetHeight),u=u?"":a)}}
var y=[],ia=function(a,b){y.push(b)},ja=function(a){a=a||window.event;var b=a.target||a.srcElement;a.cancelBubble=!0;null==t&&(a=document.createElement(Array.every||window.createPopup?"iframe":"div"),a.frameBorder="0",t=a.id="gbs",a.src="javascript:''",b.parentNode.appendChild(a),q(document,"click",z));var c=b,b=0;"gb3"!=c.className&&(c=c.parentNode);a=c.getAttribute("aria-owns")||"gbi";var d=c.offsetWidth,f=20<c.offsetTop?46:24;document.getElementById("tphdr")&&(f-=3);var e=!1;do b+=c.offsetLeft||
0;while(c=c.offsetParent);var c=(document.documentElement.clientWidth||document.body.clientWidth)-b-d,h,d=document.body,k=document.defaultView;k&&k.getComputedStyle?(d=k.getComputedStyle(d,""))&&(h=d.direction):h=d.currentStyle?d.currentStyle.direction:d.style.direction;h="rtl"==h;if("gbi"==a){for(d=0;k=y[d++];)k();A(null,window.navExtra);h&&(b=c,e=!0)}else h||(b=c,e=!0);u!=a&&z();x(a,b,f,e)},z=function(){u&&x(u,0,0)},A=function(a,b){var c,d=document.getElementById("gbi"),f=a;f||(f=d.firstChild);
for(;b&&(c=b.pop());){var e=d,h=c,k=f;w||(w="gb2");e.insertBefore(h,k).className=w}},ka=function(a,b,c){if((b=document.getElementById(b))&&a){a.className="gb4";var d=document.createElement("span");d.appendChild(a);d.appendChild(document.createTextNode(" | "));d.id=c;b.appendChild(d)}},la=function(){return document.getElementById("gb_70")},ma=function(){return!!u};p("qs",n);p("setContinueCb",ea);p("pc",fa);p("tg",ja);p("close",z);p("addLink",ka);p("almm",A);p("si",la);p("adh",ia);p("op",ma);var B=function(){},C=function(){},F=function(a){var b=new Image,c=D;b.onerror=b.onload=b.onabort=function(){try{delete E[c]}catch(a){}};E[c]=b;b.src=a;D=c+1},E=[],D=0;p("logger",{il:C,ml:B,log:F});var G=window.gbar.logger;var H={},na={},I=[],oa=l.b("0.1",.1),pa=l.a("1",!0),qa=function(a,b){I.push([a,b])},ra=function(a,b){H[a]=b},sa=function(a){return a in H},J={},K=function(a,b){J[a]||(J[a]=[]);J[a].push(b)},ta=function(a){K("m",a)},L=function(a,b){var c=document.createElement("script");c.src=a;c.async=pa;Math.random()<oa&&(c.onerror=function(){c.onerror=null;B(Error("Bundle load failed: name="+(b||"UNK")+" url="+a))});(document.getElementById("xjsc")||document.getElementsByTagName("body")[0]||
document.getElementsByTagName("head")[0]).appendChild(c)},N=function(a){for(var b=0,c;(c=I[b])&&c[0]!=a;++b);!c||c[1].l||c[1].s||(c[1].s=!0,M(2,a),c[1].url&&L(c[1].url,a),c[1].libs&&m.d&&m.d(c[1].libs))},O=function(a){K("gc",a)},P=null,ua=function(a){P=a},M=function(a,b,c){if(P){a={t:a,b:b};if(c)for(var d in c)a[d]=c[d];try{P(a)}catch(f){}}};p("mdc",H);p("mdi",na);p("bnc",I);p("qGC",O);p("qm",ta);p("qd",J);p("lb",N);p("mcf",ra);p("bcf",qa);p("aq",K);p("mdd","");p("has",sa);
p("trh",ua);p("tev",M);var Q=l.b("0.1",.001),R=0;
function _mlToken(a,b){try{if(1>R){R++;var c,d=a,f=b||{},e=encodeURIComponent,h=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&jexpid=",e("17483"),"&srcpg=",e("prop=22"),"&jsr=",Math.round(1/Q),"&ogev=",e("8lPsU_3BO-vJsQSS3IG4CA"),"&ogf=",g.bv.f,"&ogrp=",e("1"),"&ogv=",e("1407723702.0"),"&oggv="+e("es_plusone_gc_20140723.0_p0"),"&ogd=",e("com"),"&ogc=",e("CZE"),"&ogl=",e("en")];f._sn&&(f._sn="og."+
f._sn);for(var k in f)h.push("&"),h.push(e(k)),h.push("="),h.push(e(f[k]));h.push("&emsg=");h.push(e(d.name+":"+d.message));var r=h.join("");S(r)&&(r=r.substr(0,2E3));c=r;var Aa=window.gbar.logger._aem(a,c);F(Aa)}}catch(Na){}}var S=function(a){return 2E3<=a.length},va=function(a,b){return b};function T(a){B=a;p("_itl",S,G);p("_aem",va,G);p("ml",B,G);a={};H.er=a}l.a("")?T(function(a){throw a;}):l.a("1")&&Math.random()<Q&&T(_mlToken);I.push(["m",{url:"//ssl.gstatic.com/gb/js/scm_7385cc5883250b43a39405734c1bea59.js"}]);g.mcf("c",{});g.sg={c:""};if(l.a("1")){var wa=l.a("");I.push(["gc",{auto:wa,url:"//ssl.gstatic.com/gb/js/abc/gci_91f30755d6a6b787dcc2a4062e6e9824.js",libs:"googleapis.client:plusone:gapi.iframes"}]);var xa={version:"gci_91f30755d6a6b787dcc2a4062e6e9824.js",index:"",lang:"en"};H.gc=xa;var U=function(a){window.googleapis&&window.iframes?a&&a():(a&&O(a),N("gc"))};p("lGC",U);l.a("1")&&p("lPWF",U)};window.__PVT="";if(l.a("1")&&l.a("1")){var V=function(a){U(function(){K("pw",a);N("pw")})};p("lPW",V);I.push(["pw",{url:"//ssl.gstatic.com/gb/js/abc/pwm_45f73e4df07a0e388b0fa1f3d30e7280.js"}]);var W=[],ya=function(a){W[0]=a},za=function(a,b){var c=b||{};c._sn="pw";B(a,c)},Ba={signed:W,elog:za,base:"https://plusone.google.com/u/0",loadTime:(new Date).getTime()};H.pw=Ba;var X=function(a,b){for(var c=b.split("."),d=function(){var b=arguments;a(function(){for(var a=g,d=0,e=c.length-1;d<e;++d)a=a[c[d]];a[c[d]].apply(a,b)})},f=g,e=0,h=c.length-1;e<h;++e)f=
f[c[e]]=f[c[e]]||{};return f[c[e]]=d};X(V,"pw.clk");X(V,"pw.hvr");p("su",ya,g.pw)};function Ca(){function a(){for(var b;(b=e[h++])&&"m"!=b[0]&&!b[1].auto;);b&&(M(2,b[0]),b[1].url&&L(b[1].url,b[0]),b[1].libs&&m.d&&m.d(b[1].libs));h<e.length&&setTimeout(a,0)}function b(){0<f--?setTimeout(b,0):a()}var c=l.a("1"),d=l.a(""),f=3,e=I,h=0,k=window.gbarOnReady;if(k)try{k()}catch(r){da(r,"ml","or")}d?p("ldb",a):c?q(window,"load",b):b()}p("rdl",Ca);var Da={D:1,H:2,da:3,p:4,W:5,M:6,F:7,g:8,ha:9,U:10,L:11,T:12,S:13,N:14,Q:15,P:16,fa:17,w:18,O:19,ga:20,ea:21,u:22,G:23,ja:24,ka:25,ia:26,A:27,j:28,o:29,k:30,ca:31,Z:32,$:33,J:34,K:35,ba:36,aa:37,Y:38,B:39,R:40,v:41,X:42,V:43,h:48,C:49,I:500},Y=[1,2,3,4,5,6,9,10,11,13,14,28,29,30,34,35,37,38,39,40,41,42,43,48,49,500];var Z=l.b("0.001",1E-4),Ea=l.b("1",1),Fa=!1,Ga=!1;if(l.a("1")){var Ha=Math.random();Ha<=Z&&(Fa=!0);Ha<=Ea&&(Ga=!0)}var Ia=Da,$=null;function Ja(){var a=0,b=function(b,d){l.a(d)&&(a|=b)};b(1,"");b(2,"");b(4,"");b(8,"");return a}
function Ka(a,b){var c=Z,d=Fa,f;f=a;if(!$){$={};for(var e=0;e<Y.length;e++){var h=Y[e];$[h]=!0}}if(f=!!$[f])c=Ea,d=Ga;if(d){d=encodeURIComponent;g.rp?(f=g.rp(),f="-1"!=f?f:"1"):f="1";c=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&oge=",a,"&ogex=",d("17483"),"&ogev=",d("8lPsU_3BO-vJsQSS3IG4CA"),"&ogf=",g.bv.f,"&ogp=",d("22"),"&ogrp=",d(f),"&ogsr=",Math.round(1/c),"&ogv=",d("1407723702.0"),"&oggv="+
d("es_plusone_gc_20140723.0_p0"),"&ogd=",d("com"),"&ogl=",d("en"),"&ogc=",d("CZE"),"&ogus=",Ja()];if(b){"ogw"in b&&(c.push("&ogw="+b.ogw),delete b.ogw);var k;f=b;e=[];for(k in f)0!=e.length&&e.push(","),e.push(La(k)),e.push("."),e.push(La(f[k]));k=e.join("");""!=k&&(c.push("&ogad="),c.push(d(k)))}F(c.join(""))}}function La(a){"number"==typeof a&&(a+="");return"string"==typeof a?a.replace(".","%2E").replace(",","%2C"):a}C=Ka;p("il",C,G);var Ma={};H.il=Ma;setTimeout(function(){C(Ia.g)},0);}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var b=window.gbar.i.i;var c=window.gbar;var f=function(d){try{var a=document.getElementById("gbom");a&&d.appendChild(a.cloneNode(!0))}catch(e){b(e,"omas","aomc")}};c.aomc=f;}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var a=window.gbar;a.mcf("pm",{p:""});}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{window.gbar.rdl();}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
if (window['_OC_timingAction']) {window['_OC_timingAction']('patents_refpage');}</script><style>#gbar,#guser{font-size:13px;padding-top:1px !important;}#gbar{float:left;height:22px}#guser{padding-bottom:7px !important;text-align:right}.gbh,.gbd{border-top:1px solid #c9d7f1;font-size:1px}.gbh{height:0;position:absolute;top:24px;width:100%}#gbs,.gbm{background:#fff;left:0;position:absolute;text-align:left;visibility:hidden;z-index:1000}.gbm{border:1px solid;border-color:#c9d7f1 #36c #36c #a2bae7;z-index:1001}.gb1{margin-right:.5em}.gb1,.gb3{zoom:1}.gb2{display:block;padding:.2em .5em}.gb2,.gb3{text-decoration:none !important;border-bottom:none}a.gb1,a.gb4{text-decoration:underline !important}a.gb1,a.gb2,a.gb3,a.gb4{color:#00c !important}.gbi .gb3,.gbi .gb2,.gbi .gb4{color:#dd8e27 !important}.gbf .gb3,.gbf .gb2,.gbf .gb4{color:#900 !important}a.gb2:hover{background:#36c;color:#fff !important}#gbar .gbz0l{color:#000 !important;cursor:default;font-weight:bold;text-decoration:none !important}
#gbar { padding:.3em .6em !important;}</style></head><body  topmargin="3" marginheight="3"><div id=gbar><nobr><a onclick=gbar.qs(this);gbar.logger.il(1,{t:1}); class=gb1 id=gb_1 href="https://www.google.com/search?sa=N&tab=tw">Search</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:2}); class=gb1 id=gb_2 href="http://www.google.com/search?hl=en&tbm=isch&source=og&sa=N&tab=ti">Images</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:8}); class=gb1 id=gb_8 href="http://maps.google.com/maps?hl=en&sa=N&tab=tl">Maps</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:78}); class=gb1 id=gb_78 href="https://play.google.com/?hl=en&sa=N&tab=t8">Play</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:36}); class=gb1 id=gb_36 href="http://www.youtube.com/results?sa=N&tab=t1">YouTube</a> <a onclick=gbar.logger.il(1,{t:5}); class=gb1 id=gb_5 href="http://news.google.com/nwshp?hl=en&tab=tn">News</a> <a onclick=gbar.logger.il(1,{t:23}); class=gb1 id=gb_23 href="https://mail.google.com/mail/?tab=tm">Gmail</a> <a onclick=gbar.logger.il(1,{t:25}); class=gb1 id=gb_25 href="https://drive.google.com/?tab=to">Drive</a> <a class=gb3 href="http://www.google.com/intl/en/options/" onclick="this.blur();gbar.tg(event);return !1" aria-haspopup=true><u>More</u> <small>&#9660;</small></a><div class=gbm id=gbi><a onclick=gbar.logger.il(1,{t:24}); class=gb2 id=gb_24 href="https://www.google.com/calendar?tab=tc">Calendar</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:51}); class=gb2 id=gb_51 href="http://translate.google.com/?hl=en&sa=N&tab=tT">Translate</a><a onclick=gbar.logger.il(1,{t:17}); class=gb2 id=gb_17 href="http://www.google.com/mobile/?hl=en&tab=tD">Mobile</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:10}); class=gb2 id=gb_10 href="http://www.google.com/search?hl=en&tbo=u&tbm=bks&source=og&sa=N&tab=tp">Books</a><a onclick=gbar.logger.il(1,{t:212}); class=gb2 id=gb_212 href="https://wallet.google.com/manage/?tab=ta">Wallet</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:6}); class=gb2 id=gb_6 href="http://www.google.com/search?hl=en&tbo=u&tbm=shop&source=og&sa=N&tab=tf">Shopping</a><a onclick=gbar.logger.il(1,{t:30}); class=gb2 id=gb_30 href="http://www.blogger.com/?tab=tj">Blogger</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:27}); class=gb2 id=gb_27 href="http://www.google.com/finance?sa=N&tab=te">Finance</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:31}); class=gb2 id=gb_31 href="https://plus.google.com/photos?sa=N&tab=tq">Photos</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:12}); class=gb2 id=gb_12 href="http://www.google.com/search?hl=en&tbo=u&tbm=vid&source=og&sa=N&tab=tv">Videos</a><div class=gb2><div class=gbd></div></div><a onclick=gbar.logger.il(1,{t:66}); href="http://www.google.com/intl/en/options/" class=gb2>Even more &raquo;</a></div></nobr></div><div id=guser width=100%><nobr><span id=gbn class=gbi></span><span id=gbf class=gbf></span><span id=gbe></span><a target=_top id=gb_70 href="https://www.google.com/accounts/Login?service=&continue=http://www.google.com/patents%3Fhl%3Den&hl=en" class=gb4>Sign in</a><div style="display: none"><div class=gbm id=gbd5 aria-owner=gbg5><div class=gbmc><ol id=gbom class=gbmcc></ol></div></div></div></nobr></div><div class=gbh style=left:0></div><div class=gbh style=right:0></div><div role="alert" style="position: absolute; left: 0; right: 0;"><a href="http://www.google.com/patents/us6410278?hl=en&amp;output=html_text" title="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."><img border="0" src="http://www.google.com/images/cleardot.gif"alt="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."></a></div><div id="guser"><nobr></nobr></div><div style="clear:both;"></div><div id="gb-top-search-box" class="gb-top-search-box-small gb-reset"><table><tr><td class="logo"><a href="http://www.google.com/patents" class="logo-link"><img class="logo-img" src="/intl/en/images/logos/google_logo_41.png" alt="Go to Google Books Home" height="41"/></a></td><td><form action="http://www.google.com/search" name="f" id="vheadf" method="get"><span id="hf"></span><input type="hidden" name="tbm" value="pts"/><input type="hidden" name="tbo" value="1"/><input type="hidden" name="hl" value="en"/><table><tr><td><div class="inputs"><table><tr><td><div class="text-input"><input type="text" name="q" id="vheadq" class="text" maxlength="2048" size="31" value="" title="Search Patents" accesskey="s" autocomplete="off"/><script>window._OC_autoDir &&window._OC_autoDir('vheadq', 'tia-vheadq');</script></div></td><td><div class="submit-input"><input name="btnG" class="submit" type="submit" value=""/></div></td></tr></table></div></td><td class="col-ext-links"><div class="ext-links"><a href="http://www.google.com/advanced_patent_search">&lt;nobr&gt;Advanced Patent Search&lt;/nobr&gt;</a></div></td></tr></table></form></td></tr></table></div><div class="kd-appbar"><h2 class="kd-appname"><a href="/patents">Patents</a></h2><div class="kd-buttonbar left" id="left-toolbar-buttons"><a id="appbar-write-review-link" href=""></a><a id="appbar-view-print-sample-link" href=""></a><a id="appbar-view-ebook-sample-link" href=""></a><a id="appbar-patents-prior-art-finder-link" href="https://www.google.com/patents/related/US6410278"></a><a id="appbar-patents-discuss-this-link" href="http://www.google.com/url?id=c-BaBAABERAJ&amp;q=http://patents.stackexchange.com/redirect/google-patents%3Fpatent%3DUS6410278&amp;usg=AFQjCNFO0ygKwh4unymaMQ7DPp8To_8Whg" data-is-grant="true"></a><a id="appbar-read-patent-link" href="//docs.google.com/viewer?url=patentimages.storage.googleapis.com/pdfs/US6410278.pdf"></a><a id="appbar-download-pdf-link" href="//patentimages.storage.googleapis.com/pdfs/US6410278.pdf"></a></div><div class="kd-buttonbar right" id="right-toolbar-buttons"></div></div><div id="books-microdata" itemscope=""itemtype="http://schema.org/Book"itemid="http://www.google.com/patents/US6410278" style="display:none"><span itemprop="description">The present invention relates to an oligonucleotide having a novel structure and a method of synthesizing nucleic acid by using the same as a primer. This oligonucleotide is provided at the 5′-side of the primer with a nucleotide sequence substantially the same as a region synthesized with this primer...</span><span itemprop="url">http://www.google.com/patents/US6410278?utm_source=gb-gplus-share</span><span class="main-title" itemprop="name">Patent US6410278 - Process for synthesizing nucleic acid</span><img itemprop="image" src="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"alt="Patent US6410278 - Process for synthesizing nucleic acid" title="Patent US6410278 - Process for synthesizing nucleic acid"></div><div style="display: none"><ol id="ofe-gear-menu-contents" class="gbmcc"><li class="gbe gbmtc"><a class="gbmt goog-menuitem-content" id="" href="http://www.google.com/advanced_patent_search">Advanced Patent Search</a></li></ol></div><table id="viewport_table" cellpadding="0" style="clear:both" cellspacing="0"><tr><td id="viewport_td"><div class=vertical_module_list_row><div id=intl_patents class=about_content><div id=intl_patents_v><table class="patent-bibdata"><tr><td class="patent-bibdata-heading">Publication number</td><td class="single-patent-bibdata">US6410278 B1</td></tr><tr><td class="patent-bibdata-heading">Publication type</td><td class="single-patent-bibdata">Grant</td></tr><tr><td class="patent-bibdata-heading">Application number</td><td class="single-patent-bibdata">US 09/530,061</td></tr><tr><td class="patent-bibdata-heading">PCT number</td><td class="single-patent-bibdata">PCT/JP1999/006213</td></tr><tr><td class="patent-bibdata-heading">Publication date</td><td class="single-patent-bibdata">Jun 25, 2002</td></tr><tr><td class="patent-bibdata-heading">Filing date</td><td class="single-patent-bibdata">Nov 8, 1999</td></tr><tr><td class="patent-bibdata-heading">Priority date<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The priority date is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the date listed."></span></td><td class="single-patent-bibdata">Nov 9, 1998</td></tr><tr><td class="patent-bibdata-heading">Fee status<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The fee status is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the status or dates listed."></span></td><td class="single-patent-bibdata">Paid</td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Also published as</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/DE1020534T1">DE1020534T1</a>, </span><span class="patent-bibdata-value"><a href="/patents/DE69937223D1">DE69937223D1</a>, </span><span class="patent-bibdata-value"><a href="/patents/DE69937223T2">DE69937223T2</a>, </span><span class="patent-bibdata-value"><a href="/patents/DE69937223T3">DE69937223T3</a>, </span><span class="patent-bibdata-value"><a href="/patents/DE69940623D1">DE69940623D1</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP1020534A1">EP1020534A1</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP1020534A4">EP1020534A4</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP1020534B1">EP1020534B1</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP1020534B2">EP1020534B2</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP1873260A1">EP1873260A1</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP1873260B1">EP1873260B1</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP2045337A1">EP2045337A1</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP2045337B1">EP2045337B1</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP2287338A1">EP2287338A1</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP2287338B1">EP2287338B1</a>, </span><span class="patent-bibdata-value"><a href="/patents/US6974670">US6974670</a>, </span><span class="patent-bibdata-value"><a href="/patents/US7494790">US7494790</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20020168676">US20020168676</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20040132144">US20040132144</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20090098566">US20090098566</a>, </span><span class="patent-bibdata-value"><a href="/patents/WO2000028082A1">WO2000028082A1</a></span></span></td></tr><tr class="patent-bibdata-list-row alternate-patent-number"><td class="patent-bibdata-heading">Publication number</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value">09530061, </span><span class="patent-bibdata-value">530061, </span><span class="patent-bibdata-value">PCT/1999/6213, </span><span class="patent-bibdata-value">PCT/JP/1999/006213, </span><span class="patent-bibdata-value">PCT/JP/1999/06213, </span><span class="patent-bibdata-value">PCT/JP/99/006213, </span><span class="patent-bibdata-value">PCT/JP/99/06213, </span><span class="patent-bibdata-value">PCT/JP1999/006213, </span><span class="patent-bibdata-value">PCT/JP1999/06213, </span><span class="patent-bibdata-value">PCT/JP1999006213, </span><span class="patent-bibdata-value">PCT/JP199906213, </span><span class="patent-bibdata-value">PCT/JP99/006213, </span><span class="patent-bibdata-value">PCT/JP99/06213, </span><span class="patent-bibdata-value">PCT/JP99006213, </span><span class="patent-bibdata-value">PCT/JP9906213, </span><span class="patent-bibdata-value">US 6410278 B1, </span><span class="patent-bibdata-value">US 6410278B1, </span><span class="patent-bibdata-value">US-B1-6410278, </span><span class="patent-bibdata-value">US6410278 B1, </span><span class="patent-bibdata-value">US6410278B1</span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Inventors</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Tsugunori+Notomi%22">Tsugunori Notomi</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Tetsu+Hase%22">Tetsu Hase</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Original Assignee</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=inassignee:%22Eiken+Kagaku+Kabushiki+Kaisha%22">Eiken Kagaku Kabushiki Kaisha</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Export Citation</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/US6410278.bibtex">BiBTeX</a>, </span><span class="patent-bibdata-value"><a href="/patents/US6410278.enw">EndNote</a>, </span><span class="patent-bibdata-value"><a href="/patents/US6410278.ris">RefMan</a></span></span></td></tr><tr class="patent-internal-links"><td colspan=2><span class="patent-bibdata-value"><a href="#backward-citations">Patent Citations</a> (12),</span> <span class="patent-bibdata-value"><a href="#npl-citations">Non-Patent Citations</a> (3),</span> <span class="patent-bibdata-value"><a href="#forward-citations">Referenced by</a> (93),</span> <span class="patent-bibdata-value"><a href="#classifications">Classifications</a> (17),</span> <span class="patent-bibdata-value"><a href="#legal-events">Legal Events</a> (9)</span> </td></tr><tr><td colspan=2 class="patent-bibdata-external-link-spacer-top"></td></tr><tr class="patent-bibdata-external-link-spacer-bottom"></tr><tr><td colspan=2><span class="patent-bibdata-heading">External Links:&nbsp;</span><span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=c-BaBAABERAJ&q=http://patft.uspto.gov/netacgi/nph-Parser%3FSect2%3DPTO1%26Sect2%3DHITOFF%26p%3D1%26u%3D/netahtml/PTO/search-bool.html%26r%3D1%26f%3DG%26l%3D50%26d%3DPALL%26RefSrch%3Dyes%26Query%3DPN/6410278&usg=AFQjCNHygOOehw3Y-X1AECTyeI0eUmH_Fw">USPTO</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=c-BaBAABERAJ&q=http://assignments.uspto.gov/assignments/q%3Fdb%3Dpat%26pat%3D6410278&usg=AFQjCNGVJVG3BosAeWjfCyqlQN-SCAREEw">USPTO Assignment</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=c-BaBAABERAJ&q=http://worldwide.espacenet.com/publicationDetails/biblio%3FCC%3DUS%26NR%3D6410278B1%26KC%3DB1%26FT%3DD&usg=AFQjCNEKT7g8OaBX38T7JpDKy740Tq7-Qw">Espacenet</a></span></span></td></tr><tr class="patent-bibdata-group-spacer"></tr></table><div class="number-and-title"><span class="patent-title"><invention-title mxw-id="PT72384891" lang="EN" load-source="patent-office">Process for synthesizing nucleic acid</invention-title></span><br><span class="patent-number">US 6410278 B1</span></div><div class="patent-section patent-abstract-section"><div class="patent-section-header"><span class="patent-section-title">Abstract</span></div><div class="patent-text"><abstract mxw-id="PA50340120" lang="EN" load-source="patent-office"> <div class="abstract">The present invention relates to an oligonucleotide having a novel structure and a method of synthesizing nucleic acid by using the same as a primer. This oligonucleotide is provided at the 5′-side of the primer with a nucleotide sequence substantially the same as a region synthesized with this primer as the origin of synthesis. The present invention realizes synthesis of nucleic acid based on an isothermal reaction with a simple constitution of reagents. Further, the present invention provides a method of synthesizing highly specific nucleic acid on the basis of this method of synthesizing nucleic acid.</div>
  </abstract></div></div><div class="patent-section patent-drawings-section"><div class="patent-section-header"><span class="patent-section-title">Images<span class="patent-section-count">(18)</span></span></div><div class="patent-drawings-body"><div class="patent-drawings-carousel"><div class="drawings"><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6410278B1/US06410278-20020625-D00001.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6410278B1/US06410278-20020625-D00001.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6410278B1/US06410278-20020625-D00002.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6410278B1/US06410278-20020625-D00002.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6410278B1/US06410278-20020625-D00003.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6410278B1/US06410278-20020625-D00003.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6410278B1/US06410278-20020625-D00004.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6410278B1/US06410278-20020625-D00004.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6410278B1/US06410278-20020625-D00005.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6410278B1/US06410278-20020625-D00005.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6410278B1/US06410278-20020625-D00006.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6410278B1/US06410278-20020625-D00006.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6410278B1/US06410278-20020625-D00007.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6410278B1/US06410278-20020625-D00007.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6410278B1/US06410278-20020625-D00008.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6410278B1/US06410278-20020625-D00008.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6410278B1/US06410278-20020625-D00009.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6410278B1/US06410278-20020625-D00009.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6410278B1/US06410278-20020625-D00010.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6410278B1/US06410278-20020625-D00010.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6410278B1/US06410278-20020625-D00011.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6410278B1/US06410278-20020625-D00011.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6410278B1/US06410278-20020625-D00012.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6410278B1/US06410278-20020625-D00012.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6410278B1/US06410278-20020625-D00013.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6410278B1/US06410278-20020625-D00013.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6410278B1/US06410278-20020625-D00014.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6410278B1/US06410278-20020625-D00014.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6410278B1/US06410278-20020625-D00015.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6410278B1/US06410278-20020625-D00015.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6410278B1/US06410278-20020625-D00016.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6410278B1/US06410278-20020625-D00016.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6410278B1/US06410278-20020625-D00017.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6410278B1/US06410278-20020625-D00017.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6410278B1/US06410278-20020625-D00018.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6410278B1/US06410278-20020625-D00018.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div></div></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img" alt="Previous page"src="/googlebooks/images/kennedy/page_left.png"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img" alt="Next page"src="/googlebooks/images/kennedy/page_right.png"width="21" height="21" /></div></div></div><div class="patent-post-drawings"></div><div class="patent-section patent-claims-section"><div class="patent-section-header"><span class="patent-section-title">Claims<span class="patent-section-count">(19)</span></span></div><div class="patent-text"><div mxw-id="PCLM8315446" lang="EN" load-source="patent-office" class="claims">
    <claim-statement>What is claimed is: </claim-statement> <div class="claim"> <div num="1" id="US-6410278-B1-CLM-00001" class="claim">
      <div class="claim-text">1. A method of amplifying a nucleic acid comprising:</div>
      <div class="claim-text">A) providing a template having (i) a 3′ end portion comprising a first region located 3′ terminal and a first complementary region which, under suitable conditions, anneal to one another to form a first loop, (ii) a 5′ end portion comprising a second region located 5′ terminal and a second complementary region which, under suitable conditions, anneal to one another to form a second loop, and (iii) a region connecting the 3′ end portion and the 5′ end portion; </div>
      <div class="claim-text">B) extending the 3′ terminal of the template to the 5′ end of the template by means of a polymerase having strand displacement activity, when the first region and first complementary region are annealed to one another to form the first loop, to form a template extension which includes a third region located 3′ terminal and a third complementary region which arc substantially the same as the second complementary region and second region, respectively, and which, under suitable conditions, anneal to one another to form a third loop; </div>
      <div class="claim-text">C) annealing to the first loop of the extended template an oligonucleotide primer comprising at the 3′ terminal a nucleotide sequence complementary to at least part of the first loop and at the 5′ terminal a nucleotide sequence complementary to the first region of the template; </div>
      <div class="claim-text">D) extending the oligonucleotide primer along the extended template, by means of a polymerase having strand displacement activity, to form a new template complementary to the template, thereby displacing the template extension formed during said extending in step B); </div>
      <div class="claim-text">E) Further extending the 3′ terminal of the extended template to the 5′ end of the extended template by means of a polymerase having strand displacement activity, when the third region and the third complementary region are annealed to one another to form the third loop, thereby displacing the new template from the extended template; and </div>
      <div class="claim-text">F) repeating steps A)-E) using the new template as the template in step A), thereby amplifying the nucleic acid. </div>
    </div>
    </div> <div class="claim-dependent"> <div num="2" id="US-6410278-B1-CLM-00002" class="claim">
      <div class="claim-text">2. The method according to <claim-ref idref="US-6410278-B1-CLM-00001">claim 1</claim-ref>, wherein each said extending is carried out in the presence of a melting temperature regulator.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="3" id="US-6410278-B1-CLM-00003" class="claim">
      <div class="claim-text">3. The method according to <claim-ref idref="US-6410278-B1-CLM-00002">claim 2</claim-ref>, wherein the melting temperature regulator is betaine.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="4" id="US-6410278-B1-CLM-00004" class="claim">
      <div class="claim-text">4. The method according to <claim-ref idref="US-6410278-B1-CLM-00003">claim 3</claim-ref>, wherein 0.2 to 3.0 M betaine is allowed to be present in the reaction solution.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="5" id="US-6410278-B1-CLM-00005" class="claim">
      <div class="claim-text">5. The method according to <claim-ref idref="US-6410278-B1-CLM-00001">claim 1</claim-ref>, wherein said providing in step A) comprises:</div>
      <div class="claim-text">A1) annealing a first oligonucleotide primer to a sample single-stranded nucleic acid molecule, the first oligonucleotide primer comprising a 3′ terminal portion which anneals to the sample single-stranded nucleic acid molecule and a 5′ terminal portion comprising substantially the same nucleotide sequence as an arbitrary region of the sample single-stranded nucleic acid molecule; </div>
      <div class="claim-text">A2) extending the first oligonucleotide primer from its 3′ terminal, using a suitable polymerase, to form a first single-stranded nucleic acid molecule comprising (i) a region complementary to at least a portion of the sample single-stranded nucleic acid molecule, and (ii) a 5′ end portion comprising the 5′ terminal portion of the first oligonucleotide primer; </div>
      <div class="claim-text">A3) displacing the first single-stranded nucleic acid molecule from the sample single-stranded nucleic acid molecule; </div>
      <div class="claim-text">A4) annealing a second oligonucleotide primer to the first single-stranded nucleic acid molecule, the second oligonucleotide primer comprising a 3′ terminal portion which anneals to the first single-stranded nucleic acid molecule and a 5′ terminal portion comprising substantially the same nucleotide sequence as an arbitrary region of the first single-stranded nucleic acid molecule; </div>
      <div class="claim-text">A5) extending the second oligonucleotide primer from its 3′ end, using a suitable polymerase, to form the template; and </div>
      <div class="claim-text">A6) displacing the template from the first single-stranded nucleic acid molecule. </div>
    </div>
    </div> <div class="claim-dependent"> <div num="6" id="US-6410278-B1-CLM-00006" class="claim">
      <div class="claim-text">6. The method according to <claim-ref idref="US-6410278-B1-CLM-00005">claim 5</claim-ref>, wherein the first oligonucleotide primer used during said annealing in step A1) or the second oligonucleotide primer used during said annealing in step A4) is the same as the oligonucleotide primer used during said annealing in step C).</div>
    </div>
    </div> <div class="claim-dependent"> <div num="7" id="US-6410278-B1-CLM-00007" class="claim">
      <div class="claim-text">7. A method of detecting a target nucleotide sequence in a sample comprising:</div>
      <div class="claim-text">performing the method of amplifying according to <claim-ref idref="US-6410278-B1-CLM-00001">claim 1</claim-ref>, wherein the template comprises the target nucleotide sequence, and </div>
      <div class="claim-text">determining whether the target nucleotide sequence is present in the product of the method of amplifying. </div>
    </div>
    </div> <div class="claim-dependent"> <div num="8" id="US-6410278-B1-CLM-00008" class="claim">
      <div class="claim-text">8. The method according to <claim-ref idref="US-6410278-B1-CLM-00007">claim 7</claim-ref>, wherein said determining comprises:</div>
      <div class="claim-text">combining the product of the method of amplifying with a probe containing a nucleotide sequence complementary to the first loop or the second loop; and </div>
      <div class="claim-text">observing whether hybridization occurs between the probe and the product of the method of amplifying. </div>
    </div>
    </div> <div class="claim-dependent"> <div num="9" id="US-6410278-B1-CLM-00009" class="claim">
      <div class="claim-text">9. The method according to <claim-ref idref="US-6410278-B1-CLM-00008">claim 8</claim-ref>, wherein the probe is labeled on particles and said observing comprises:</div>
      <div class="claim-text">detecting whether an aggregation reaction occurs. </div>
    </div>
    </div> <div class="claim-dependent"> <div num="10" id="US-6410278-B1-CLM-00010" class="claim">
      <div class="claim-text">10. The method according to <claim-ref idref="US-6410278-B1-CLM-00007">claim 7</claim-ref>, wherein said performing the amplification method is conducted in the presence of a detector for nucleic acids, and wherein said determining comprises:</div>
      <div class="claim-text">detecting a change in a signal of the detector. </div>
    </div>
    </div> <div class="claim"> <div num="11" id="US-6410278-B1-CLM-00011" class="claim">
      <div class="claim-text">11. A method comprising:</div>
      <div class="claim-text">A) providing a template having (i) a 3′ end portion comprising a first region located 3′ terminal and a first complementary region which, under suitable conditions, anneal to one another to form a first loop, (ii) a 5′ end portion comprising a second region located 5′ terminal and a second complementary region which, under suitable conditions, anneal to one another to form a second loop, and (iii) a region connecting the 3′ end portion and the 5′ end portion; </div>
      <div class="claim-text">B) extending the 3′ terminal of the template to the 5′ end of the template by means of a polymerase having strand displacement activity, when the first region and first complementary region are annealed to one another to form the first loop, to form a template extension which includes a third region located 3′ terminal and a third complementary region that are substantially the same as the second complementary region and second region, respectively, and which, under suitable conditions, anneal to one another to form a third loop; </div>
      <div class="claim-text">C) annealing to the first loop of the extended template an oligonucleotide primer comprising at the 3′ terminal a nucleotide sequence complementary to at least a part of the first loop and at the 5′ terminal a nucleotide sequence complementary to the first region of the template; </div>
      <div class="claim-text">D) extending the oligonucleotide primer along the extended template, by means of a polymerase having strand displacement activity, to form a new template complementary to the template; and </div>
      <div class="claim-text">E) displacing the new template from the extended template. </div>
    </div>
    </div> <div class="claim-dependent"> <div num="12" id="US-6410278-B1-CLM-00012" class="claim">
      <div class="claim-text">12. The method according to <claim-ref idref="US-6410278-B1-CLM-00011">claim 11</claim-ref>, wherein the new template has (i) a 3′ end portion comprising the second region and the second complementary region located 3′ terminal which, under suitable conditions, anneal to one another to form the second loop, (ii) a 5′ end portion comprising the first region and the first complementary region located 5′ terminal which, under suitable conditions, anneal to one another to form the first loop, and (iii) a region connecting the 3′ end portion and the 5′ end portion, said method further comprising:</div>
      <div class="claim-text">F) extending the 3′ terminal of the new template to the 5′ end of the new template by means of a polymerase having strand displacement activity, when the second region and second complementary region are annealed to one another to form the second loop, to form a template extension which includes a third region and a third complementary region that are substantially the same as the first complementary region and first region, respectively, and which, under suitable conditions, anneal to one another to form a third loop; </div>
      <div class="claim-text">G) annealing to the second loop of the extended new template a second oligonucleotide primer comprising at the 3′ terminal a nucleotide sequence complementary to at least a part of the second loop and at the 5′ terminal a nucleotide sequence complementary to the second complementary region of the template; </div>
      <div class="claim-text">H) extending the second oligonucleotide primer along the extended new template, by means of a polymerase having strand displacement activity, to form a third template which is substantially the same as the template. </div>
    </div>
    </div> <div class="claim-dependent"> <div num="13" id="US-6410278-B1-CLM-00013" class="claim">
      <div class="claim-text">13. The method according to <claim-ref idref="US-6410278-B1-CLM-00012">claim 12</claim-ref> further comprising:</div>
      <div class="claim-text">J) displacing the third template from the new template. </div>
    </div>
    </div> <div class="claim-dependent"> <div num="14" id="US-6410278-B1-CLM-00014" class="claim">
      <div class="claim-text">14. The method according to <claim-ref idref="US-6410278-B1-CLM-00011">claim 11</claim-ref> further comprising:</div>
      <div class="claim-text">repeating steps B) through I) using the third template. </div>
    </div>
    </div> <div class="claim-dependent"> <div num="15" id="US-6410278-B1-CLM-00015" class="claim">
      <div class="claim-text">15. The method according to <claim-ref idref="US-6410278-B1-CLM-00011">claim 11</claim-ref>, wherein said extending in step D) displaces the sequence in the template extension which is complementary to the 5′ end portion of the template, allowing the third region and the third complementary region to anneal to one another to form the third loop, said method further comprising:</div>
      <div class="claim-text">further extending the 3′ terminal of the extended template to the 5′ end of the template, thereby displacing the new template in step E). </div>
    </div>
    </div> <div class="claim-dependent"> <div num="16" id="US-6410278-B1-CLM-00016" class="claim">
      <div class="claim-text">16. The method according to <claim-ref idref="US-6410278-B1-CLM-00015">claim 15</claim-ref> further comprising:</div>
      <div class="claim-text">annealing to the third loop a second oligonucleotide primer comprising at the 3′ terminal a nucleotide sequence complementary to at least a part of the third loop and at the 5′ terminal a nucleotide sequence complementary to the third region of the template; and </div>
      <div class="claim-text">extending the 3′ terminal of the second oligonucleotide primer by means of a polymerase having strand displacement activity. </div>
    </div>
    </div> <div class="claim-dependent"> <div num="17" id="US-6410278-B1-CLM-00017" class="claim">
      <div class="claim-text">17. The method according to <claim-ref idref="US-6410278-B1-CLM-00011">claim 11</claim-ref> wherein each said extending is carried out in the presence of a melting temperature regulator.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="18" id="US-6410278-B1-CLM-00018" class="claim">
      <div class="claim-text">18. The method according to <claim-ref idref="US-6410278-B1-CLM-00017">claim 17</claim-ref>, wherein the melting temperature regulator is betaine.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="19" id="US-6410278-B1-CLM-00019" class="claim">
      <div class="claim-text">19. The method according to <claim-ref idref="US-6410278-B1-CLM-00018">claim 18</claim-ref>, wherein 0.2 to 3.0 M betaine is allowed to be present in the reaction solution.</div>
    </div>
  </div> </div></div></div><div class="patent-section patent-description-section"><div class="patent-section-header"><span class="patent-section-title">Description</span></div><div class="patent-text"><div mxw-id="PDES53589390" lang="EN" load-source="patent-office" class="description">
    <heading>TECHNICAL FIELD</heading> <p>The present invention relates to a method of synthesizing nucleic acid composed of a specific nucleotide sequence, which is useful as a method of amplifying nucleic acid.</p>
    <heading>BACKGROUND ART</heading> <p>An analysis method based on complementarity of a nucleic acid nucleotide sequence can analyze genetic traits directly. Accordingly, this analysis is a very powerful means for identification of genetic diseases, canceration, microorganisms etc. Further, a gene itself is the object of detection, and thus time-consuming and cumbersome procedures such as in culture can be omitted in some cases.</p>
    <p>Nevertheless, the detection of a target gene present in a very small amount in a sample is not easy in general so that amplification of a target gene it self or its detection signal is necessary. As a method of amplifying a target gene, the PCR (polymerase chain reaction) method is known (Science, 230, 1350-1354, 1985). Currently, the PCR method is the most popular method as a technique of amplifying nucleic acid in vitro. This method was established firmly as an excellent detection method by virtue of high sensitivity based on the effect of exponential amplification. Further, since the amplification product can be recovered as DNA, this method is applied widely as an important tool supporting genetic engineering techniques such as gene cloning and structural determination. In the PCR method, however, there are the following noted problems: a special temperature controller is necessary for practice; the exponential progress of the amplification reaction causes a problem in quantification; and samples and reaction solutions are easily contaminated from the outside to permit nucleic acid mixed in error to function as a template.</p>
    <p>As genomic information is accumulated, analysis of single nucleotide polymorphism (SNPs) comes to attract attention. Detection of SNPs by means of PCR is feasible by designing a primer such that its nucleotide sequence contains SNPS. That is, whether a nucleotide sequence complementary to the primer is present or not can be inferred by determining whether a reaction product is present or not. However, once a complementary chain is synthesized in error in PCR by any chance, this product functions as a template in subsequent reaction, thus causing an erroneous result. In practice, it is said that strict control of PCR is difficult with the difference of only one base given at the terminal of the primer. Accordingly, it is necessary to improve specificity in order to apply PCR to detection of SNPs.</p>
    <p>On one hand, a method of synthesizing nucleic acid by a ligase is also practically used. The LCR method (ligase chain reaction, Laffler TG; Garrino JJ; Marshall RL; Ann. Biol. Clin. (Paris), 51:9, 821-6, 1993) is based on the reaction in which two adjacent probes are hybridized with a target sequence and ligated to each other by a ligase. The two probes could not be ligated in the absence of the target nucleotide sequence, and thus the presence of the ligated product is indicative of the target nucleotide sequence. Because the LCR method also requires control of temperature for separation of a complementary chain from a template, there arises the same problem as in the PCR method. For LCR, there is also a report on a method of improving specificity by adding the step of providing a gap between adjacent probes and filling the gap by a DNA polymerase. However, what can be expected in this modified method is specificity only, and there still remains a problem in that control of temperature is required. Furthermore, use of the additional enzyme leads to an increase in cost.</p>
    <p>A method called the SDA method (strand displacement amplification) [Proc. Natl. Acad. Sci. USA, 89, 392-396, 1992] [Nucleic Acid. Res., 20, 1691-1696, 1992] is also known as a method of amplifying DNA having a sequence complementary to a target sequence as a template. In the SDA method, a special DNA polymerase is used to synthesize a complementary chain starting from a primer complementary to the 3′-side of a certain nucleotide sequence while displacing a double-stranded chain if any at the 5′-side of the sequence. In the present specification, the simple expression “5′-side” or “3′-side” refers to that of a chain serving as a template. Because a double-stranded chain at the 5′-side is displaced by a newly synthesized complementary chain, this technique is called the SDA method. The temperature-changing step essential in the PCR method can be eliminated in the SDA method by previously inserting a-restriction enzyme recognition sequence into an annealed sequence as a primer. That is, a nick generated by a restriction enzyme gives a 3′—OH group acting as the origin of synthesis of complementary chain, and the previously synthesized complementary chain is released as a single-stranded chain by strand displacement synthesis and then utilized again as a template for subsequent synthesis of complementary chain. In this manner, the complicated control of temperature essential in the PCR method is not required in the SDA method.</p>
    <p>In the SDA method, however, the restriction enzyme generating a nick should be used in addition to the strand displacement-type DNA polymerase. This requirement for the additional enzyme is a major cause for higher cost. Further, because the restriction enzyme is to be used not for cleavage of both double-stranded chains but for introduction of a nick (that is, cleavage of only one of the chains), a dNTP derivative such as α-thio dNTP should be used as a substrate for synthesis to render the other chain resistant to digestion with the enzyme. Accordingly, the amplification product by SDA has a different structure from that of natural nucleic acid, and there is a limit to cleavage with restriction enzymes or application of the amplification product to gene cloning. In this respect too, there is a major cause for higher cost. In addition, when the SDA method is applied to an unknown sequence, there is the possibility that the same nucleotide sequence as the restriction enzyme recognition sequence used for introducing a nick may be present in a region to be synthesized. In this case, it is possible that a complete complementary chain is prevented from being synthesized.</p>
    <p>NASBA (nucleic acid sequence-based amplification, also called the TMA/transcription mediated amplification method) is known as a method of amplifying nucleic acid wherein the complicated control of temperature is not necessary. NASBA is a reaction system wherein DNA is synthesized by DNA polymerase in the presence of target RNA as a template with a probe having a T7 promoter added thereto, and the product is formed with a second probe into a double-stranded chain, followed by transcription by T7 RNA polymerase with the formed double-stranded chain as a template to amplify a large amount of RNA (Nature, 350, 91-92, 1991). NASBA requires some heat denaturation steps until double-stranded DNA is completed, but the subsequent transcriptional reaction by T7 RNA polymerase proceeds under isothermal conditions. However, a combination of plural enzymes such as reverse transcriptase, RNase H, DNA polymerase and T7 RNA polymerase is essential, and this is unfavorable for cost similarly to SDA. Further, because it is complicated to set up conditions for a plurality of enzyme reaction, this method is hardly widespread as a general analytical method. In the known reactions of amplification of nucleic acid, there remain problems such as complicated control of temperature and the necessity for plural enzymes as described above</p>
    <p>For these known reactions of synthesizing nucleic acid, there are few reports on an attempt for further improving the efficiency of synthesis of nucleic acid without sacrificing specificity or cost. For example, in a method called RCA (rolling-circle amplification), it was shown that single-stranded DNA having a series of nucleotide sequences complementary to a padlock probe can be synthesized continuously in the presence of a target nucleotide sequence (Paul M. Lizardi et al., Nature Genetics, 19, 225-232, July, 1998). In RCA, a padlock probe having a special structure wherein each of the 5′- and 3′-terminals of a single oligonucleotide constitutes an adjacent probe in LCR is utilized. Then, the continuous reaction of synthesizing complementary chain with the padlock probe as a template which was ligated and cyclized in the presence of a target nucleotide sequence is triggered by combination with a polymerase catalyzing the strand displacement-type reaction of synthesizing complementary chain. Single-stranded nucleic acid having a structure of a series of regions each consisting of the same nucleotide sequence is thus formed. A primer is further annealed to this single-stranded nucleic acid to synthesize its complementary chain and a high degree of amplification is thus realized. However, there still remains the problem of the necessity for a plurality of enzymes. Further, triggering of synthesis of complementary chain depends on the reaction of ligating two adjacent regions, and its specificity is basically the same as in LCR.</p>
    <p>For the object of supplying 3′—OH, there is a known method in which a nucleotide sequence is provided at the 3-terminal with a sequence complementary thereto and a hair pin loop is formed at the terminal (Gene, 71, 29-40, 1988). Synthesis of complementary chain with a target sequence itself as a template starts at the hairpin loop to form single-stranded nucleic acid composed of the complementary nucleotide sequence. For example, a structure in which annealing occurs in the same chain at the terminal to which a complementary nucleotide sequence has been linked is realized in PCT/FR95/00891. In this method, however, the step in which the terminal cancels base pairing with the complementary chain and base pairing is constituted again in the same chain is essential. It is estimated that this step proceeds depending on a subtle equilibrium state at the terminal of mutually complementary nucleotide sequences involving base pairing. That is, an equilibrium state maintained between base pairing with a complementary chain and base pairing in the same chain is utilized and the only chain annealing to the nucleotide sequence in the same chain serves as the origin of synthesis of a complementary chain. Accordingly, it is considered that strict reaction conditions should be set to achieve high reaction efficiency. Further, in this prior art, the primer itself forms a loop structure. Accordingly, once a primer dimer is formed, amplification reaction is automatically initiated regardless of whether a target nucleotide sequence is present or not, and an unspecific synthetic product is thus formed. This can be a serious problem. Further, formation of the primer dimer and subsequent consumption of the primer by unspecific synthetic reaction lead to a reduction in the amplification efficiency of the desired reaction.</p>
    <p>Besides, there is a report that a region not serving as a template for DNA polymerase was utilized to realize a 3′-terminal structure annealing to the same chain (EP713922). This report also has the same problem as in PCT/FR95/00891 supra in respect of the utilization of dynamic equilibrium at the terminal or the possibility of unspecific synthetic reaction due to formation of a dimer primer. Further, a special region not serving as a template for DNA polymerase should be prepared as a primer.</p>
    <p>Further, in various signal amplification reactions to which the principle of NASBA described above is applied, an oligonucleotide having a hairpin structure at the terminal thereof is often utilized to supply a double-stranded promoter region (JP-A 5-211873). However, these techniques are not those permitting successive supply of 3′—OH for synthesis of a complementary chain. Further, a hairpin loop structure having a 3′-terminal annealed in the same chain is utilized for the purpose of obtaining a DNA template transcribed by RNA polymerase is utilized in JP-A 10-510161 (W096/17079). In this method, the template is amplified by using transcription into RNA and reverse transcription from RNA to DNA. In this method, however, the reaction system cannot be constituted without a combination of a plurality of enzymes.</p>
    <heading>DISCLOSURE OF THE INVENTION</heading> <p>The object of the present invention is to provide a method of synthesizing nucleic acid based on a novel principle. A more specific object is to provide a method capable of realizing the synthesis of nucleic acid depending on sequence efficiently at low costs. That is, an object of the present invention is to provide a method capable of achieving the synthesis and amplification of nucleic acid by a single enzyme even under isothermal reaction conditions. Another object of the present invention is to provide a method of synthesizing nucleic acid which can realize high specificity difficult to achieve in the known reaction principle of nucleic acid synthesis, as well as a method of amplifying nucleic acid by applying said synthetic method.</p>
    <p>The present inventors focused their attention on the fact that the utilization. of a polymerase catalyzing strand displacement-type synthesis of complementary chain is useful for nucleic acid synthesis not depending on complicated control of temperature. Such a DNA polymerase is an enzyme utilized in SDA and RCA. However, even if such an enzyme is used, another enzyme reaction is always required for supplying 3′—OH as the origin of synthesis in the known means based on primers, such as SDA.</p>
    <p>Under these circumstances, the present inventors examined supply of 3′—OH from a completely different viewpoint from the known approach. As a result, the present inventors found that by utilizing an oligonucleotide having a special structure, 3′—OH can be supplied without any additional enzyme reaction, thereby completing the present invention. That is, the present invention relates to a method of synthesizing nucleic acid, a method of amplifying nucleic acid by applying said method of synthesizing nucleic acid and a novel oligonucleotide enabling said methods, as follows:</p>
    <p>1. A method of synthesizing nucleic acid having complementary nucleotide sequences linked alternately in a single-stranded chain, comprising:</p>
    <p>a) the step of giving nucleic acid which is provided at the 3′-terminal thereof with a region F1 capable of annealing to a part F1c in the same chain and which upon annealing of the region F1 to F1c, is capable of forming a loop containing a region F2c capable of base pairing,</p>
    <p>b) the step of performing synthesis of a complementary chain wherein the 3′-terminal of F1 having annealed to F1c serves as the origin of synthesis,</p>
    <p>c) the step of annealing, to a region F2c, of an oligonucleotide provided with the 3′-terminal thereof with F2 consisting of a sequence complementary to the region F2c, followed by synthesis, with said oligonucleotide as the origin of synthesis, of a complementary chain by a polymerase catalyzing the strand displacement reaction of synthesizing a complementary chain to displace the complementary chain synthesized in step b), and</p>
    <p>d) the step of annealing, to the complementary chain displaced in step c) to be ready for base pairing, of a polynucleotide provided at the 3′-terminal thereof with a sequence complementary to an arbitrary region in said chain synthesized in step c), followed by synthesis, with said 3′-terminal as the origin of synthesis, of a complementary chain by a polymerase catalyzing the strand displacement reaction of synthesizing a complementary chain to displace the complementary chain synthesized in step c).</p>
    <p>2. The method according to item 1, wherein in step d), the origin of synthesis is a region R1 present at the 3′-terminal in the same chain and capable of annealing to a region R1c, and a loop containing the region R2c capable of base pairing is formed by annealing R1 to R1c.</p>
    <p>3. An oligonucleotide composed of at least two regions X2 and X1c below, and X1c is linked to the 5′-side of X2,</p>
    <p>X2: a region having a nucleotide sequence complementary to an arbitrary region X2c in nucleic acid having a specific nucleotide sequence, and</p>
    <p>X1c: a region having substantially the same nucleotide sequence as in a region X1c located at the 5′-side of the region X2c in nucleic acid having a specific nucleotide sequence.</p>
    <p>4. The method according to item 1, wherein the nucleic acid in step a) is. second nucleic acid provided by the following steps:</p>
    <p>i) the step of annealing, to a region F2c in nucleic acid serving as a template, of a region F2 in the oligonucleotide described in item 3 wherein the region X2 is a region F2 and the region X1c is a region F1c,</p>
    <p>ii) the step of synthesizing first nucleic acid having a nucleotide sequence complementary to the template wherein F2 in the oligonucleotide serves as the origin of synthesis,</p>
    <p>iii) the step of rendering an arbitrary region in the first nucleic acid synthesized in step ii) ready for base pairing, and</p>
    <p>iv) the step of annealing an oligonucleotide having a nucleotide sequence complementary to the region made ready for base pairing in the first nucleic acid in step iii), followed by synthesizing second nucleic acid with said oligonucleotide as the origin of synthesis and rendering F1 at the 3′-terminal thereof ready for base pairing.</p>
    <p>5. The method according to item 4, wherein the region enabling base pairing in step iii) is R2c, and the oligonucleotide in step iv) is the oligonucleotide described in item 3 wherein the region X2c is a region R2c and the region X1c is a region R1c.</p>
    <p>6. The method according to item 4 or 5, wherein the step of rendering base pairing ready in steps iii) and iv) is conducted by the strand displacement synthesis of complementary chain by a polymerase catalyzing the strand displacement reaction of synthesizing complementary chain wherein an outer primer annealing to the 3′-side of F2c in the template and an outer primer annealing to the 3′-side of the region used as the origin of- synthesis in step iv) for the first nucleic acid serve as the origin of synthesis.</p>
    <p>7. The method according to item 6, wherein the melting temperature of each oligonucleotide and its complementary region in the template used in the reaction is in the following relationship under the same stringency:</p>
    <p>(outer primer/region at the 3′-side in the template)≦(F2c/F2 and R2c/R2)≦(F1c/F1 and R1c/R1).</p>
    <p>8. The method according to any one of items 4 to 7, wherein the nucleic acid serving as the template is RNA, and the synthesis of complementary chain in step ii) is conducted by an enzyme having a reverse transcriptase activity.</p>
    <p>9. A method of amplifying nucleic acid having complementary nucleotide sequences linked alternately in a single-stranded chain by repeatedly conducting the following steps:</p>
    <p>A) the step of providing a template which is provided at the 3′- and 5′-terminals thereof with a region consisting of a nucleotide sequence complementary to each terminal region in the same chain and which upon annealing of these mutually complementary nucleotide sequences, forms a loop capable of base pairing therebetween,</p>
    <p>B) the step of performing the synthesis of complementary chain wherein the 3′-terminal of said template annealed to the same chain serves as the origin of synthesis,</p>
    <p>C) the step of annealing, to the loop portion, of an oligonucleotide provided at the 3′-terminal thereof with a complementary nucleotide sequence to a loop which among said loops, is located at the 3′-terminal site, followed by synthesis, with the oligonucleotide as the origin of synthesis, of a complementary chain by a polymerase catalyzing the strand displacement reaction of synthesizing a complementary chain to displace the complementary chain synthesized in step B) to make the 3′-terminal thereof ready for base pairing, and</p>
    <p>D) the step wherein the chain with the 3′-terminal made ready for base pairing in step C) serves as a new template.</p>
    <p>10. The method according to item 9, wherein the oligonucleotide in step C) is provided at the 5′-terminal thereof with a nucleotide sequence complementary to the 3′-terminal serving as the origin of synthesis in step B).</p>
    <p>11. The method according to item 10, further comprising the step where a complementary chain synthesized with the oligonucleotide in step C) as the origin of synthesis is used as a template in step A).</p>
    <p>12. The method according to item 9, wherein the template in step A) is synthesized by the method described in item 5.</p>
    <p>13. The method according to item 1 or 9, wherein the strand displacement reaction of synthesizing complementary chain is carried out in the presence of a melting temperature regulator.</p>
    <p>14. The method according to item 13, wherein the melting temperature regulator is betaine.</p>
    <p>15. The method according to item 14, wherein 0.2 to 3.0 M betaine is allowed to be present in the reaction solution.</p>
    <p>16. A method of detecting a target nucleotide sequence in a sample, which comprises performing an amplification method described in any one of items 9 to 15 and observing whether an amplification reaction product is generated or not.</p>
    <p>17. The method according to item 16, wherein a probe containing a nucleotide sequence complementary to the loop is added to the amplification reaction product and hybridization therebetween is observed.</p>
    <p>18. The method according to item 17, wherein the probe is labeled on particles and aggregation reaction occurring upon hybridization is observed.</p>
    <p>19. The method according to item 16, wherein an amplification method described in any one of items 9 to 15 is conducted in the presence of a detector for nucleic acid, and whether an amplification reaction product is generated or not is observed on the basis of a change in the signal of the detector.</p>
    <p>20. A method of detecting a mutation in a target nucleotide sequence by the detection method described in item 16, wherein a mutation in a nucleotide sequence as the subject of amplification prevents synthesis of any one of complementary chains constituting the amplification method.</p>
    <p>21. A kit for synthesis of nucleic acid having complementary chains alternately linked in a single-stranded chain, comprising the following elements:</p>
    <p>i) the oligonucleotide described in item 3 wherein the region F2c in nucleic acid as a template is X2c, and F1c located at the 5′-side of F2c is X1c;</p>
    <p>ii) an oligonucleotide containing a nucleotide sequence complementary to an arbitrary region in a complementary chain synthesized with the oligonucleotide in (i) as a primer;</p>
    <p>iii) an oligonucleotide having a nucleotide sequence complementary to a region F3c located at the 3′-side of the region F2c in the nucleic acid serving as a template;</p>
    <p>iv) a DNA polymerase catalyzing the strand displacement-type reaction of synthesizing complementary chain; and</p>
    <p>v) a nucleotide serving as a substrate for the element iv). 22. The kit according to item 21, wherein the oligonucleotide in ii) is the oligonucleotide described in item 3 wherein an arbitrary region R2c in a complementary chain synthesized with the oligonucleotide in i) as the origin of synthesis is X2c, and R1c located at the 5′ of R2c is X1c.</p>
    <p>23. The kit according to item 22, further comprising:</p>
    <p>vi) an oligonucleotide having a nucleotide sequence complementary to a region R3c located at the 3′-side of the arbitrary R2c in a complementary chain synthesized with the oligonucleotide in i) as the origin of synthesis.</p>
    <p>24. A kit for detection of a target nucleotide sequence, comprising a detector for detection of a product of nucleic acid synthetic reaction additionally in a kit described in any one of items 21 to 23.</p>
    <p>The nucleic acid having complementary nucleotide sequences linked alternately in a single-stranded chain as the object of synthesis in the present invention means nucleic acid having mutually complementary nucleotide sequences linked side by side in a single-stranded chain. Further, in the present invention, it should contain a nucleotide sequence for forming a loop between the complementary chains. In the present invention, this sequence is called the loop-forming sequence. The nucleic acid synthesized by the present invention is composed substantially of mutually complementary chains linked via the loop-forming sequence. In general, a strand not separated into 2 or more molecules upon dissociation of base pairing is called a single-stranded chain regardless of whether it partially involves base pairing or not. The complementary nucleotide sequence can form base pairing in the same chain. An intramolecular base-paired product, which can be obtained by permitting the nucleic acid having complementary nucleotide sequences linked alternately in a single-stranded chain according to the present invention to be base-paired in the same chain, gives a region constituting an apparently double-stranded chain and a loop not involving base pairing.</p>
    <p>That is, the nucleic acid having complementary nucleotide sequences linked alternately in a single-stranded chain according to the present invention contains complementary nucleotide sequences capable of annealing in the same chain, and its annealed product can be defined as single-stranded nucleic acid constituting a loop not involving base pairing at a bent hinged portion. A nucleotide having a nucleotide sequence complementary thereto can anneal to the loop not involving base pairing. The loop-forming sequence can be an arbitrary nucleotide sequence. The loop-forming sequence is capable of base pairing so as to initiate the synthesis of a complementary chain for displacement, and is provided preferably with a sequence distinguishable from a nucleotide sequence located in the other region in order to achieve specific annealing. For example, in a preferred embodiment, the loop-forming sequence contains substantially the same nucleotide sequence as a region F2c (or R2c) located at the 3′-side of a region (i.e. F1c or R1c) derived from nucleic acid as a template and annealed in the same chain.</p>
    <p>In the present invention, substantially the same nucleotide sequence is defined as follows. That is, when a complementary chain synthesized with a certain sequence as a template anneals to a target nucleotide sequence to give the origin of synthesizing a complementary chain, this certain sequence is substantially the same as the target nucleotide sequence. For example, substantially the same sequence as F2 includes not only absolutely the same nucleotide sequence as F2 but also a nucleotide sequence capable of functioning as a template giving a nucleotide sequence capable of annealing to F2 and acting as the origin of synthesizing complementary chain. The term “anneal” in the present invention means formation of a double-stranded structure of nucleic acid through base pairing based on the law of Watson-Crick. Accordingly, even if a nucleic acid chain constituting base pairing is a single-stranded chain, annealing occurs if intramolecular complementary nucleotide sequences are base-paired. In the present invention, annealing and hybridization have the same meaning in that the nucleic acid constitutes a double-stranded structure through base pairing.</p>
    <p>The number of pairs of complementary nucleotide sequences constituting the nucleic acid according to the present invention is at least 1. According to a desired mode of the present invention, it may be 2 or more. In this case, there is theoretically no upper limit to the number of pairs of complementary nucleotide sequences constituting the nucleic acid. When the nucleic acid as the synthetic product of the present invention is constituted of plural sets of complementary nucleotide sequences, this nucleic acid is composed of repeated identical nucleotide sequences.</p>
    <p>The nucleic acid having complementary nucleotide sequences linked alternately in a single-stranded chain synthesized by the present invention may not have the same structure as naturally occurring nucleic acid. It is known that if a nucleotide derivative is used as a substrate when nucleic acid is synthesized by the action of a DNA polymerase, a nucleic acid derivative can be synthesized. The nucleotide derivative used includes nucleotides labeled with a radioisotope or nucleotide derivatives labeled with a binding ligand such as biotin or digoxin. These nucleotide derivatives can be used to label nucleic acid derivatives as the product. Alternatively, if fluorescent nucleotides are used as a substrate, the nucleic acid as the product can be a fluorescent derivative. Further, this product may be either DNA or RNA. Which one is formed is determined by a combination of the structure of a primer, the type of a substrate for polymerization and polymerization reagents for carrying out polymerization of nucleic acid.</p>
    <p>Synthesis of the nucleic acid having the structure described above can be initiated by use of a DNA polymerase having the strand displacement activity and nucleic acid which is provided at the 3-terminal thereof with a region F1 capable of annealing to a part F1c in the same chain and which upon annealing of the region F1 to F1c, is capable of forming a loop containing a region F2c capable of base pairing. There are many reports on the reaction of synthesizing complementary chain wherein a hairpin loop is formed and a sample sequence itself is used as a template, while in the present invention the portion of the hairpin loop is provided with a region capable of base pairing, and there is a novel feature on utilization of this region in synthesizing complementary chain. By use of this region as the origin of synthesis, a complementary chain previously synthesized with a sample sequence itself as a template is displaced. Then, a region R1c (arbitrary region) located at the 3′-terminal of the displaced chain is in a state ready for base-pairing. A region having a complementary sequence to this R1c is annealed thereto, resulting in formation of the nucleic acid (2 molecules) having a nucleotide sequence extending from F1 to R1c and its complementary chain linked alternately via the loop-forming sequence. In the present invention, the arbitrary region such as R1c above can be selected arbitrarily provided that it can be annealed to a polynucleotide having a nucleotide sequence complementary to that region, and that a complementary chain synthesized with the polynucleotide as the origin of synthesis has necessary functions for the present invention.</p>
    <p>In the present invention, the term “nucleic acid” is used. The nucleic acid in the present invention generally includes both DNA and RNA. However, nucleic acid whose nucleotide is replaced by an artificial derivative or modified nucleic acid from natural DNA or RNA is also included in the nucleic acid of the present invention insofar as it functions as a template for synthesis of complementary chain. The nucleic acid of the present invention is generally contained in a biological sample. The biological sample includes animal, plant or microbial tissues, cells, cultures and excretions, or extracts therefrom. The biological sample of the present invention includes intracellular parasitic genomic DNA or RNA such as virus or mycoplasma. The nucleic acid of the present invention may be derived from nucleic acid contained in said biological sample. For example, cDNA synthesized from mRNA, or nucleic acid amplified on the basis of nucleic acid derived from the biological sample, is a typical example of the nucleic acid of the present invention.</p>
    <p>The nucleic acid characteristic of the present invention which is provided at the 3′-terminal thereof with a region F1 capable of annealing to a part F1c in the same chain and which upon annealing of the region F1 to F1c, is capable of forming a loop containing a region F2c capable of base pairing can be obtained in various methods. In the most preferable embodiment, the reaction of synthesizing complementary chain utilizing an oligonucleotide having the following structure can be used to give the structure.</p>
    <p>That is, the useful oligonucleotide in the present invention consists of at least two regions X2 and X1c below wherein X1c is ligated to the 5′-side of X2.</p>
    <p>X2: a region having a nucleotide sequence complementary to a region X2c in nucleic acid having a specific nucleotide-sequence.</p>
    <p>X1c: a region having substantially the same nucleotide sequence as a region X1c located at the 5′-side of the region X2c in nucleic acid having a specific nucleotide sequence.</p>
    <p>Here, the nucleic acid having a specific nucleotide sequence by which the structure of the oligonucleotide of the invention is determined refers to nucleic acid serving as a template when the oligonucleotide of the present invention is used as a primer. In the case of detection of nucleic acid based on the synthetic method of the present invention, the nucleic acid having a specific nucleotide sequence is a detection target or nucleic acid derived from the detection target. The nucleic acid having a specific nucleotide sequence refers to nucleic acid wherein at least a part of the nucleotide sequence is revealed or predictable. The part of the nucleotide sequence revealed is the region X2c and the region X1c located at the 5′-side thereof. It can be supposed that these 2 regions are contiguous or located apart from each other. By the relative positional relationship of the two, the state of a loop formed upon self-annealing of nucleic acid as the product is determined. The distance between the two is preferably not very apart from each other in order that nucleic acid as the product is subjected to self-annealing preferentially over intermolecular annealing. Accordingly, the positional relationship of the two is preferably that they are contiguous via a distance of usually 0 to 100 bases. However, in the formation of a loop by self-annealing described below, there can be the case where it would be disadvantageous for formation of a loop in a desired state that the two are too close to each other. In the loop, there is a need for a structure for annealing of a new oligonucleotide and for readily initiating the strand-displacement reaction of synthesizing a complementary chain with said oligonucleotide as the origin of synthesis. More preferably, the distance between the region X2c and the region X1c located at the 5′-side of X2c is designed to be 0 to 100 bases, more desirably 10 to 70 bases. This numerical value shows a length excluding X1c and X2. The number of bases constituting the part of a loop is that of this length plus a region corresponding to X2.</p>
    <p>Both the terms “same” and “complementary” used for characterization of the nucleotide sequence constituting the oligonucleotide based on the present invention do not mean being absolutely the same or absolutely complementary. That is, the same sequence as a certain sequence includes sequences complementary to nucleotide sequences capable of annealing to a certain sequence. On the other hand, the complementary sequence means a sequence capable of annealing under stringent conditions to provide a 3′-terminal serving as the origin of synthesis of complementary chain.</p>
    <p>Usually, the regions X2 and X1c constituting the oligonucleotide of the present invention for the nucleic acid having a specific nucleotide sequence are located contiguously without being overlapped. If there is a common part in both the nucleotide sequences, the two can be partially overlaid. Because X2 should function as a primer, it should always be a 3-terminal. On the other hand, X1c should give the function of a primer as described blow to the 3′-terminal of a complementary chain synthesized with the nucleic acid as a template, and thus it shall be arranged at the 5′-terminal. The complementary chain obtained with this oligonucleotide as the origin of synthesis serves as a template for synthesis of complementary chain in the reverse direction in the next step, and finally the part of the oligonucleotide of the present invention is copied as a template into a complementary chain. The 3′-terminal generated by copying has the nucleotide sequence X1, which anneals to X1c in the same chain to form a loop.</p>
    <p>In the present invention, the oligonucleotide means the one that satisfies the 2 requirements, that is, it must be able to form complementary base pairing and give an —OH group serving as the origin of synthesis of complementary chain at the 3′-terminal. Accordingly, its backbone is not necessarily limited to the one via phosphodiester linkages. For example, it may be composed of a phosphothioate derivative having S in place of O as a backbone or a peptide nucleic acid based on peptide linkages. The bases may be those capable of complementary base pairing. In the nature, there are 5 bases, that is, A, C, T, G and U, but the base can be an analogue such as bromodeoxyuridine. The oligonucleotide used in the present invention functions preferably not only as the origin of synthesis but also as a template for synthesis of complementary chain. The term polynucleotide in the present invention includes oligonucleotides. The term “polynucleotide” is used in the case where the chain length is not limited, while the term “oligonucleotide” is used to refer to a nucleotide polymer having a relatively short chain length.</p>
    <p>The oligonucleotide according to the present invention has such a chain length as to be capable of base pairing with a complementary chain and to maintain necessary specificity under the given environment in the various reactions of synthesizing nucleic acid described below. Specifically, it is composed of 5 to 200 base pairs, more preferably 10 to 50 base pairs. The chain length of a primer recognizing the known polymerase catalyzing the sequence-dependent nucleic acid synthetic reaction is at least about 5 bases, so the chain length of the annealing part should be longer than that. In addition, a length of 10 bases or more is desired statistically in order to expect specificity as the nucleotide sequence. On the other hand, preparation of a too long nucleotide sequence by chemical synthesis is difficult, and thus the chain length described above is exemplified as a desired range. The chain length exemplified here refers to the chain length of a part annealing to a complementary chain. As described below, the oligonucleotide according to the present invention can anneal finally to at least 2 regions-individually. Accordingly, it should be understood that the chain length exemplified here is the chain length of each region constituting the oligonucleotide.</p>
    <p>Further, the oligonucleotide according to the present invention can be labeled with a known labeling substance. The labeling substance includes binding ligands such as digoxin and biotin, enzymes, fluorescent substances and luminescent substances, and radioisotopes. The techniques of replacing a base constituting an oligonucleotide by a fluorescent analogue are also known (W095/05391, Proc. Natl. Acad. Sci. USA, 91, 6644-6648, 1994).</p>
    <p>Other oligonucleotides according to the present invention can also have been bound to a solid phase. Alternatively, an arbitrary part of the oligonucleotide may be labeled with a binding ligand such as biotin, and it can be immobilized indirectly via a binding partner such as immobilized avidin. When the immobilized oligonucleotide is used as the origin of synthesis, nucleic acid as the synthetic reaction product is captured by the solid phase, thus facilitating its separation. The separated product can be detected by a nucleic acid-specific indicator or by hybridization with a labeling probe. The target nucleic acid fragments can also be recovered by digesting the product with arbitrary restriction enzymes.</p>
    <p>The term “template” used in the present invention means nucleic acid serving as a template for synthesizing a complementary chain. A complementary chain having a nucleotide sequence complementary to the template has a meaning as a chain corresponding to the template, but the relationship between the two is merely relative. That is, a chain synthesized as the complementary chain can function again as a template. That is, the complementary chain can become a template.</p>
    <p>The oligonucleotide useful in the present invention is not limited to the 2 regions described above and can contain an additional region. While X2 and X1c are arranged at the 3′-and 5′-terminals respectively, an arbitrary sequence can be interposed therebetween. For example, it can be a restriction enzyme recognition site, a promoter recognized by RNA polymerase, or DNA coding for ribozyme. By using it as a restriction enzyme recognition sequence, the nucleic acid having a complementary sequence alternately linked in a single-stranded chain as the synthetic product of the present invention can be cleaved into double-stranded nucleic acids of the same length. By arranging a promoter sequence recognized by RNA polymerase, the synthetic product of the present invention serves as the template to permit further transcription into RNA. By further arranging DNA coding for ribozyme, a system where the transcriptional product is self-cleaved is realized. These additional nucleotide sequences are those functioning after formed into a double-stranded chain. Accordingly, when the single-stranded nucleic acid according to the present invention has formed a loop, these sequences do not function. They do not function until the nucleic acid is elongated and annealed in the absence of a loop to a chain having a complementary nucleotide sequence.</p>
    <p>When a promoter is combined with the oligonucleotide based on the present invention in such a direction as to permit transcription of the synthesized region, the reaction product based on the present invention where the same nucleotide sequence is repeated realizes a highly efficient transcriptional system. By combining this system with a suitable expression system, translation into a protein is also feasible. That is, the system can be utilized for transcription and translation into protein in bacteria or animal cells or in vitro.</p>
    <p>The oligonucleotide of the present invention having the structure described above can be chemically synthesized. Alternatively, natural nucleic acid may be cleaved with e.g. restriction enzymes and modified so as to be composed of, or ligated into, the nucleotide sequence described above.</p>
    <p>The basic principle of the reaction for performing synthesis by utilizing the useful oligonucleotide described above in combination with DNA polymerase having the strand displacement activity in the reaction of synthesizing nucleic acid according to the present invention is described by reference to FIGS. 5 to <b>6</b>. The oligonucleotide described above (FA in FIG. 5) anneals at X2 (corresponding to F2) to nucleic acid as a template, to provide the origin of synthesis of complementary chain. In FIG. 5, a complementary chain synthesized from FA as the origin of synthesis is displaced by synthesis of complementary chain (described below) from an outer primer (F3), to form a single-stranded chain (FIG. <b>5</b>-A). When synthesis of complementary chain to the resulting complementary chain is further conducted, the 3′-terminal of nucleic acid synthesized as complementary chain in FIG. 5-A has a nucleotide sequence complementary to the oligonucleotide of the present invention. That is, because the 5-terminal of the oligonucleotide of the present invention has the same sequence as a region X1c (corresponding to F1c), the 3′-terminal of the nucleic acid thus synthesized has a complementary sequence X1 (F1). FIG. 5 shows that the complementary chain synthesized from R1 as the origin of synthesis is displaced by synthesis of complementary chain by primer R3 as the origin of synthesis. Once the 3′-terminal portion is made ready for base pairing by this displacement, X1 (F1) at the 3′-terminal anneals to X1c (F1c) in the same chain, and elongation reaction with itself as a template proceeds (FIG. <b>5</b>-B). Then, X2c (F2c) located at the 3′-terminal thereof is left as a loop not involving base pairing. X2 (F2) in the oligonucleotide according to the present invention anneals to this loop, and a complementary chain is synthesized with said oligonucleotide as the origin of synthesis (FIG. <b>5</b>-B). A product of complementary chain synthetic reaction with the previously synthesized product as a template is displaced by the strand displacement reaction so that it is made ready for base pairing.</p>
    <p>By the basic constitution using one kind of oligonucleotide according to the present invention and an arbitrary reverse primer capable of conducting nucleic acid synthesis where a complementary chain synthesized with said oligonucleotide as a primer is used as a template, a plurality of nucleic acid synthetic products as shown in FIG. 6 can be obtained. As can be seen from FIG. 6, (D) is the desired nucleic acid product of the invention having complementary nucleotide sequence alternately linked in a single-stranded chain. Once converted into a single-stranded chain by treatment such as heat denaturation, the other product (E) serves again as a template for forming (D). If the product (D) as nucleic acid in the form of a double-stranded chain is converted into a single-stranded chain by heat denaturation, annealing occurs within the same chain at high probability without forming the original double-stranded chain. This is because a complementary chain having the same melting temperature (Tm) undergoes intramolecular reaction preferentially over intermolecular reaction. Each single-stranded chain derived from the product (D) annealed in the same chain is annealed in the same chain and returned to the state of (B), and each chain further gives one molecule of (D) and (E) respectively. By repeating these steps, it is possible to successively synthesize the nucleic acid having complementary nucleotide sequences linked alternately in a single-stranded chain. The template and the product formed in 1 cycle are increased exponentially, thus making the reaction very efficient.</p>
    <p>To realize the state of FIG. <b>5</b>(A), the initially synthesized complementary chain should, in at least the portion to which the reverse primer anneals, should be ready for base pairing. This step can be achieved by an arbitrary method. That is, an outer primer (F3), which anneals to the first template at a region F3c at the 3′-side of the region F2c to which the oligonucleotide of the present invention anneals, is separately prepared. If this outer primer is used as the origin synthesis to synthesize a complementary chain by a polymerase catalyzing the. strand displacement-type synthesis of complementary chain, the complementary chain synthesized from the F2c as the origin of synthesis in the invention is displaced, and as a result the region R1c to be annealed by R1 is made ready for base pairing (FIG. <b>5</b>). By utilization of the strand displacement reaction, the reaction up to now can proceed under isothermal conditions.</p>
    <p>When an outer primer is used, synthesis from the outer primer (F3) should be initiated after synthesis from F2c. In the most simple method, the concentration of the inner primer is made higher than the concentration of the outer primer. Specifically, the primers are used at usually 2- to 50-fold, preferably 4- to 10-fold different concentrations, whereby the reaction can proceed as expected. Further, the melting temperature (Tm) of the outer primer is set to be lower than the Tm of the X1 (corresponding to F1 and R1) in the inner primer whereby the timing of synthesis can be controlled. That is, (outer primer F3:F3c)≦F2c/F2)≦(F1c/F1) or (outer primer/region at the 3′-side in the template)≦(X2c:X2)≦(X1c:X1). Here, the reason for (F2c/F2)≦(F1c/F1) is for annealing between F1c/F1 prior to annealing of F2 to the loop. The annealing between F1c/F1 is an intramolecular reaction and may thus proceed preferentially at high probability. However, it is meaningful to consider Tm in order to give more desired reaction conditions. As a matter of course, similar conditions should be considered even in the design of a reverse primer. By using such a relationship, statistically ideal reaction conditions can be achieved. If other conditions are fixed, melting temperature (Tm) can be theoretically calculated by a combination of the length of an annealing complementary chain and bases constituting base-pairing. Accordingly, those skilled in the art can derive preferable conditions on the basis of the disclosure of this specification.</p>
    <p>Further, the phenomenon called contiguous stacking can also be applied for controlling timing of annealing of the outer primer. contiguous stacking is a phenomenon in which an oligonucleotide not capable of annealing independently is made capable of annealing upon being contiguous to the part of a double-stranded chain (Chiara Borghesi-Nicoletti et al., Bio Techniques, 12, 474-477 (1992)). That is, the outer primer is designed so as to be contiguous to F2c (X2c) and not to be able to anneal independently. By doing so, annealing of the outer primer does not occur until F2c (X2c) anneals, and thus the annealing of F2c (X2c) occurs preferentially. On the basis of this principle, the Examples show setting of the nucleotide sequence of an oligonucleotide necessary as a primer for a series of reactions. This step can also be achieved by denaturation under heating or with a DNA helicase.</p>
    <p>If the template nucleic acid having F2c (X2c) is RNA, the state of FIG. <b>5</b>-(A) can also be realized by a different method. For example, if this RNA chain is decomposed, R1 is made ready for base pairing. That is, F2 is annealed to F2c in RNA and a complementary chain is synthesized as DNA by a reverse transcriptase. The RNA serving as a template is decomposed by alkali denaturation or by enzymatic treatment with a ribonuclease acting on RNA in a double-stranded chain of DNA/RNA whereby the DNA synthesized from F2 is formed into a single-stranded chain. For the enzyme selectively decomposing RNA in a double-stranded chain of DNA/RNA, the ribonuclease activity of RNase H or some reverse transcriptases can be utilized. In this manner, the reverse primer can be annealed to R1c made capable of base pairing. Accordingly, the outer primer for rendering R1c ready for base pairing becomes unnecessary.</p>
    <p>Alternatively, the strand displacement activity of reverse transcriptase can be utilized for the strand displacement by an outer primer as described above. In this case, a reaction system can be constituted by a reverse transcriptase only. That is, using RNA as a template, it is made possible by a reverse transcriptase to synthesize a complementary chain from F2 annealing to F2c in the template and to synthesize a complementary chain from the outer primer F3 as the origin of synthesis annealing to F3c located at the 3′-side of F2c and to simultaneously displace the previously synthesized complementary chain. When the reverse transcriptase performs the reaction of synthesizing a complementary chain with DNA as the template, all the reactions of synthesizing complementary chains including the synthesis of a complementary chain with R1 as the origin of synthesis annealing to R1c in the displaced complementary chain as the template, the synthesis of a complementary chain with R3 as the origin of synthesis annealing to R3c located at the 3′-side of R1c and the simultaneous displacement reaction, proceed by the reverse transcriptase. If it is not possible to expect that the reverse transcriptase exhibits the DNA/RNA strand displacement activity under given reaction conditions, a DNA polymerase having the strand displacement activity described above may be combined. The mode of obtaining a first single-stranded nucleic acid with RNA as a template as described above constitutes a preferable mode of the present invention. On the other hand, if a DNA polymerase such as Bca DNA polymerase having both strand displacement activity and reverse transcriptase activity is used, not only synthesis of a first single-stranded nucleic acid from RNA but also subsequent reaction with DNA as a template can proceed similarly by the same enzyme.</p>
    <p>The reaction system described above brings about various variations inherent in the present invention by utilization of the reverse primer having a specific structure. The most effective variation is described below. That is, the oligonucleotide constituted as described in [<b>5</b>] is used as the reverse primer in the most advantageous mode of the present invention. The oligonucleotide in [<b>5</b>] is an oligonucleotide wherein arbitrary regions R2c and R1c in a complementary chain synthesized with F2 as a primer are X2c and X1c respectively. By use of such a reverse primer, a series of reactions for forming a loop and for synthesizing and displacing a complementary chain from this loop occur in both the sense and antisense chains (forward side and reverse side). As a result, the reaction efficiency for synthesis of the nucleic acid having complementary nucleotide sequences linked alternately in a single-stranded chain according to the present invention is greatly improved while a series of these reactions are feasible under isothermal conditions. Hereinafter, this mode is described in more detail by reference to FIGS. 1 to <b>3</b> where this mode is summarized.</p>
    <p>In the following mode, 2 kinds of oligonucleotides based on the present invention are prepared. For explanation, these are designated FA and RA. The regions constituting FA and RA are as follows:</p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="offset" colwidth="98pt" align="left"> </colspec> <colspec colname="1" colwidth="56pt" align="left"> </colspec> <colspec colname="2" colwidth="63pt" align="left"> </colspec> <thead> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="OFFSET" nameend="2" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">X2</td>
                <td class="description-td">Xlc</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="OFFSET" nameend="2" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="4"> <colspec colname="offset" colwidth="42pt" align="left"> </colspec> <colspec colname="1" colwidth="56pt" align="left"> </colspec> <colspec colname="2" colwidth="56pt" align="left"> </colspec> <colspec colname="3" colwidth="63pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">FA</td>
                <td class="description-td">F2</td>
                <td class="description-td">Flc</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">RA</td>
                <td class="description-td">R2</td>
                <td class="description-td">Rlc</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="OFFSET" nameend="3" align="center" rowsep="1" class="description-td" colspan="4"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p>Here, F2 is a complementary nucleotide sequence to a region F2c in nucleic acid as the template. R2 is a nucleotide sequence complementary to an arbitrary region R2c contained in a complementary chain synthesized with F2 as a primer. F1c and R1c are arbitrary nucleotide sequences located downward from F2c and R2c respectively. The distance between F2 and R2 may be arbitrary. Even if its length is about 1 kbp, sufficient synthesis is feasible under suitable conditions, though depending on the synthetic ability of DNA polymerase to perform the synthesis of a complementary chain. Specifically, when Bst DNA polymerase is used, the desired product is certainly synthesized if the distance between F2 and R2c is 800 bp, preferably 500 bp or less. In PCR involving temperature cycle, the reduction in the enzyme activity by the stress of temperature change is considered to reduce the efficiency of synthesis of a long nucleotide sequence. In a preferable mode of the present invention, the temperature cycle in the step of amplifying nucleic acid is not required, and thus the synthesis and amplification of an even long nucleotide sequence can be certainly achieved.</p>
    <p>First, F2 in FA is annealed to nucleic acid as a template and used as the origin of synthesis of a complementary chain. The subsequent. reaction steps until FIG. <b>1</b>(<b>4</b>) are the same as in the previously described basic mode (FIG. 5) in the present invention. The sequence annealed as F3 in FIG. <b>1</b>(<b>2</b>) is the outer primer described above. A DNA polymerase for conducting the strand displacement-type synthesis of a complementary chain with this primer as the origin of synthesis is used so that the complementary chain synthesized from FA is displaced and made ready for base pairing.</p>
    <p>When R2c is made ready for base pairing in (<b>4</b>), RA as a reverse primer anneals thereto in the combination of R2c/R2. Synthesis of a complementary chain with this site as the origin of synthesis proceeds until the chain reaches F1c at the 5′-terminal of FA. Following this reaction of synthesizing a complementary chain, the outer primer R3 for displacement anneals thereto to synthesize a complementary chain, during which strand displacement also proceeds so that the complementary chain synthesized from RA as the origin of synthesis is displaced. In the complementary chain thus displaced, RA is located at the 5′-side thereof and a sequence complementary to FA is located at the 3′-terminal thereof.</p>
    <p>At the 3′-side of the single-stranded nucleic acid thus displaced, there is a sequence F1 complementary to F1c in the same chain. F1 rapidly anneals to F1c in the same molecule to initiate synthesis of a complementary chain. When the 3′-terminal (F1) anneals to F1c in the same chain, a loop containing F2c (i.e., a first loop) is formed. (As illustrated in FIG. <b>2</b>-(<b>7</b>), the same single-stranded nucleic acid also contains at the 5′-side a sequence R1 complementary to R1c in the same chain, which can likewise anneal to form a loop containing R2, i.e., a second loop.) As is also evident from FIG. <b>2</b>-(<b>7</b>), the part of this loop containing F2c remains ready for base pairing. The oligonucleotide FA of the invention having a nucleotide sequence complementary to F2c anneals to the part of this loop (i.e., the first loop) and acts as the origin of synthesis of a complementary chain (<b>7</b>). Synthesis of a complementary chain from the loop proceeds while the reaction product in the previously initiated complementary chain synthesis from F1 is displaced. As a result, the complementary chain synthesized with itself as the template is made ready for base pairing again at the 3′-terminal. This 3′-terminal is provided with a region R1 capable of annealing to R1c in the same chain, and the two are annealed preferentially due to the rapid intramolecular reaction (i.e., forming a third loop). The same reaction as the above-described reaction starting from the 3′-terminal synthesized with FA as a template proceeds in this region as well. As a result, the nucleic acid having complementary nucleotide sequences linked alternately in the same single-stranded chain according to the present invention is continued to be extended from R1 as the starting point at the 3′-terminal by successive synthesis of a complementary chain and subsequent displacement thereof. Because R2c is always contained in the loop formed by intramolecular annealing of the 3′-terminal R1, the oligonucleotide (RA) provided with R2 anneals to the loop at the 3′-terminal in the subsequent reaction.</p>
    <p>When attention is paid to nucleic acid synthesized as complementary chain from the oligonuclcotide annealing to the loop in the single-stranded nucleic acid elongated with itself as the template, synthesis of the nucleic acid having complementary nucleotide sequences linked alternately in the same single-stranded chain according to the present invention also proceeds here That is, synthesis of a complementary chain from the loop (i.e., the first loop) is completed when it reached RA in e.g. FIG. <b>2</b>-(<b>7</b>). Then, when the nucleic acid displaced by this nucleic acid synthesis (i.e., forming the third loop) initiates synthesis of complementary chain (FIG. <b>3</b>-(<b>8</b>)), the reaction reaches the loop which was once the origin of synthesis (i.e., the first loop), and displacement is initiated again. In this manner, the nucleic acid initiated to be synthesized from the loop is also displaced, and as a result, the 3′-terminal R1 capable of annealing in the same chain is obtained (FIG. <b>3</b>-(<b>10</b>)). This 3′-terminal R1 anneals to R1 c in the same chain to initiate synthesis of complementary chain. This reaction is the same as in FIG. <b>2</b>-(<b>7</b>) except that F is used in place of R. Accordingly, the structure shown in FIG. <b>3</b>-(<b>10</b>) can function as a new nucleic acid which continues self-elongation and new nucleic acid formation.</p>
    <p>The reaction of synthesizing nucleic acid, initiated from the nucleic acid shown in FIG. <b>3</b>-(<b>10</b>), causes elongation from the 3′-terminal F1 as the origin of synthesis, as opposed to the reaction described above. That is, in the present invention, as one nucleic acid is elongated, the reaction of continuing to supply a new nucleic acid initiating elongation separately proceeds. Further, as the chain is elongated, a plurality of loop-forming sequences are brought about not only at the terminal but also in the same chain. When these loop-forming sequences are made ready for base pairing by the strand displacement synthetic reaction, an oligonucleotide anneals thereto to serve as a base for the reaction of forming a new nucleic acid. Further efficient amplification is achieved by the synthetic reaction starting not only at the terminal but also in the chain. The oligonucleotide RA based on the present invention is combined as the reverse primer as described above whereby elongation and subsequent formation of a new nucleic acid occur. Further, in the present invention, this newly formed nucleic acid itself is elongated and brings about subsequent formation of a new nucleic acid. A series of these reactions continue theoretically permanently to achieve very efficient amplification of nucleic acid. In addition, the reaction in the present invention can be conducted under isothermal conditions.</p>
    <p>The reaction products thus accumulated possess a structure having a nucleotide sequence between F1 and R1 and its complementary sequence linked alternately therein. However, both the terminals of the repeating unit have a region consisting of the successive nucleotide sequences F2-F1 (F2c-F1c) and R2-R1 (R2c-R1c). For example, in FIG. <b>3</b>-(<b>9</b>), the sequences (R2-F2c)-(F1-R2c)-(R1-F1c)-(F2-R2c) are linked in this order from the 5′-side. This is because the amplification reaction based on the present invention proceeds on the principle that the reaction is initiated from F2 (or R2) with an oligonucleotide as the origin of synthesis and then a complementary chain is elongated by the synthetic reaction from F1 (or R1) with the 3′-terminal as the origin of synthesis.</p>
    <p>Here, in the most preferable mode, oligonucleotides FA and RA according to the present invention were used as oligonucleotides annealing to the part of a loop. However, even if these oligonucleotide having a limited structure are not used, the amplification reaction according to the present invention can be carried out by use of an oligonucleotide capable of initiating the synthesis of a complementary chain from the loop. That is, the elongating 3′-terminal, once displaced by a complementary chain synthesized from the loop, gives the part of a loop again. Because the nucleic acid having complementary nucleotide sequences linked alternately in a single-stranded chain is always used as a template in the complementary chain synthesis starting from the loop, it is evident that the nucleic acid desired in the present invention can be synthesized. However, the nucleic acid thus synthesized performs synthesis of a complementary chain by forming a loop after displacement, but there is no 3′-terminal available for subsequent formation of a loop, and thus it cannot function as a new template. Accordingly, the product in this case, unlike nucleic acid initiated to be synthesized by FA or RA, cannot be expected to be exponentially amplified. From this reason, an oligonucleotide having the structure of FA or RA is useful for highly efficient synthesis of nucleic acid based on the present invention.</p>
    <p>A series of these reactions proceed by adding the following components to single-stranded nucleic acid as a template and then incubating the mixture at such a temperature that the nucleotide sequence constituting FA and RA can form stable base pairing with its complementary nucleotide sequence while the enzyme activity can be maintained.</p>
    <p>4 kinds of oligonucleotides:</p>
    <p>FA,</p>
    <p>RA,</p>
    <p>outer primer F3, and</p>
    <p>outer primer R3,</p>
    <p>DNA polymerase for performing the strand displacement-type synthesis of complementary chain,</p>
    <p>an oligonucleotide serving as a substrate for DNA polymerase.</p>
    <p>Accordingly, temperature cycle such as in PCR is not necessary. The stable base pairing referred to herein means a state in which at least a part of an oligonucleotide present in the reaction system can give the origin of synthesis of complementary chain. For example, the desired condition for bringing about stable base pairing is to set lower than melting temperature (Tm). Generally, melting temperature (Tm) is regarded as the temperature at which 50% of nucleic acids having mutually complementary nucleotide sequences are base-paired. Setting at melting temperature (Tm) or less is not an essential condition in the present invention, but is one of the reaction conditions to be considered for attaining high efficiency of synthesis. If nucleic acid to be used as a template is a double-stranded chain, the nucleic acid should, in at least a region to which the oligonucleotide anneals, be made ready for base pairing. For this, heat denaturation is generally conducted, and this may be conducted only once as pretreatment before the reaction is initiated.</p>
    <p>This reaction is conducted in the presence of a buffer giving suitable pH to the enzyme reaction, salts necessary for annealing or for maintaining the catalytic activity of the enzyme, a protective agent for the enzyme, and as necessary a regulator for melting temperature (Tm). As the buffer, e.g. Tris-HCl having a buffering action in a neutral to weakly alkaline range is used. The pH is adjusted depending on the DNA polymerase used. As the salts, KCl, NaCl, (NH<sub>4</sub>)<sub>2</sub>SO<sub>4 </sub>etc. are suitably added to maintain the activity of the enzyme and to regulate the melting temperature (Tm) of nucleic acid. The protective agent for the enzyme makes use of bovine serum albumin or sugars. Further, dimethyl sulfoxide (DMSO) or formamide is generally used as the regulator for melting temperature (Tm). By use of the regulator for melting temperature (Tm), annealing of the oligonucleotide can be regulated under limited temperature conditions. Further, betaine (N,N,N-trimethylglycine) or a tetraalkyl ammonium salt is also effective for improving the efficiency of strand displacement by virtue of its isostabilization. By adding betaine in an amount of 0.2 to 3.0 M, preferably 0.5 to 1.5 M to the reaction solution, its promoting action on the nucleic acid amplification of the present invention can be expected. Because these regulators for melting temperature act for lowering melting temperature, those conditions giving suitable stringency and reactivity are empirically determined in consideration of the concentration of salts, reaction temperature etc.</p>
    <p>An important feature in the present invention is that a series of reactions do not proceed unless the positional relationship of a plurality of regions is maintained. By this feature, unspecific synthetic reaction accompanied by unspecific synthesis of complementary chain is effectively prevented. That is, even if a certain unspecific reaction occurs, the possibility for the product to serve as a starting material in the subsequent amplification step is minimized. Further, the regulation of the progress of reactions by many regions brings about the possibility that a detection system capable of strict identification of the desired product in analogous nucleotide sequences can be arbitrarily constituted.</p>
    <p>This feature can be utilized for detection of mutations in a gene. In the mode of the invention where the outer primer is used, 4 primers, that is, 2 outer primers and 2 primers consisting of the oligonucleotides of the present invention, are used. That is, unless the 6 regions contained in the 4 oligonucleotides work as designed, the synthetic reaction of the present invention do not proceed. In particular, the sequences of the 3′-terminal of each oligonucleotide as the origin of synthesis of complementary chain and of the 5′-terminal of the X1c region where the complementary chain serves as the origin of synthesis are important. Hence, these important sequences is designed so as to correspond to a mutation to be detected, and the synthetic reaction product of the present invention is observed whereby the presence or absence of a mutation such as base deletion or insertion, or genetic polymorphism such as SNPs can be comprehensively analyzed. Specifically, bases estimated to have a mutation or polymorphism are designed so as to correspond to the vicinity of the 3′-terminal of an oligonucleotide as the origin of synthesis of complementary chain, or of 5′-terminal thereof when a complementary chain is the origin of synthesis. If a mismatch is present at the 3′-terminal as the origin of synthesis of complementary chain or in its vicinity, the reaction of synthesizing a complementary chain to nucleic acid is significantly inhibited. In the present invention, a high degree of amplification reaction is not achieved unless the structure of the terminals of a product in the initial reaction brings about repeated reactions. Accordingly, even if erroneous synthesis occurs, complementary chain synthesis constituting amplification reaction is always interrupted in some of the steps, and thus a high degree of amplification reaction does not occur in the presence of a mismatch. As a result, the mismatch effectively inhibits amplification reaction, and an accurate result is finally brought about. That is, it can be said that the amplification reaction of nucleic acid based on the present invention has a highly completed mechanism for checking the nucleotide sequence. These features are an advantage hardly expectable in e.g. the PCR method where amplification reaction is performed in mere 2 regions.</p>
    <p>The region X1c characterizing the oligonucleotide used in the present invention can serve as the origin of synthesis after a complementary sequence is synthesized, and this complementary sequence anneals to the sequence X1 in the same newly synthesized chain whereby synthetic reaction with itself as a template proceeds. Therefore, even if the so-called primer dimer which is often problematic in the prior art is formed, this oligonucleotide does not form a loop. Accordingly, unspecific amplification attributable to the primer dimer cannot occur theoretically, and thus the present oligonucleotide contributes to an improvement in the specificity of the reaction.</p>
    <p>Further, according to the present invention, the outer primers shown as F3 (FIG. <b>1</b>-(<b>2</b>)) or R3 (FIG. <b>2</b>-(<b>5</b>)) are combined whereby a series of the reactions described above can be conducted under isothermal conditions. That is, the present invention provides a method of amplifying nucleic acid having complementary sequences linked alternately in a single-stranded chain, which comprises the steps shown in item 9 above. In this method, temperature conditions where stable annealing occurs between F2c/F2, between R2c/R2, between F1c/F1, and between R1c/R1 are selected, and preferably F3c/F3 and R3c/R3 are set up to be annealed by the phenomenon of contiguous stacking facilitated by annealing of F2c/F2 and R2c/R2, respectively.</p>
    <p>In the present invention, the terms “synthesis” and “amplification” of nucleic acid are used. The synthesis of nucleic acid in the present invention means the elongation of nucleic acid from an oligonucleotide serving as the origin of synthesis. If not only this synthesis but also the formation of other nucleic acid and the elongation reaction of this formed nucleic acid occur continuously, a series of these reactions is comprehensively called amplification.</p>
    <p>The single-stranded nucleic acid which is provided at the 3′-terminal thereof with a region F1 capable of annealing to a part F1c in the same chain and which upon annealing of the region F1 to F1c in the same chain, is capable of forming a loop containing a region F2c capable of base pairing is an important element of the present invention. Such a single-stranded nucleic acid can also be supplied on the following principle. That is, the synthesis of a complementary chain is allowed to proceed on the basis of a primer having the following structure. 5′-[region X1 annealing to region X1c located in primer]-[loop forming sequence ready for base pairing]-[region X1c]-[region having a sequence complementary to a template]-3′</p>
    <p>As the region having a sequence complementary to a template, two nucleotide sequences, that is, a nucleotide sequence (primer FA) complementary to F1 and a nucleotide sequence (primer RA) complementary to R1c, are prepared. The nucleotide sequence constituting nucleic acid to be synthesized contains a nucleotide sequence extending from the region F1 to the region R1c and a nucleotide sequence extending from the region R1 having a nucleotide sequence complementary to this nucleotide sequence to the region F1c. X1c and X1 capable of annealing in the inside of the primer can be arbitrary sequences. However, in a region between primers FA and RF, the sequence of the region X1c/X1 is made preferably different.</p>
    <p>First, the synthesis of a complementary chain by the primer FA from the region F1 in template nucleic acid is conducted. Then, the region R1c in the synthesized complementary chain is made ready for base pairing, to which the other primer is annealed to form the origin of synthesis of complementary chain. The 3′-terminal of the complementary chain synthesized in this step has a nucleotide sequence complementary to the primer FA constituting the 5′-terminal of the initially synthesized chain, so it has been provided at the 3′-terminal thereof with the region X1 which anneals to the region X1c in the same chain to form a loop. The characteristic 3′-terminal structure according to the present invention is thus provided, and the subsequent reaction constitutes the reaction system shown previously as the most preferable mode. The oligonucleotide annealing to the portion of the loop is provided at the 3′-terminal thereof with the region X2 complementary to the region X2c located in the loop and at the 5′-terminal thereof with the region X1. In the previous reaction system, primers FA and RA were used to synthesize a chain complementary to template nucleic acid thereby giving a loop structure to the 3 ′-terminal of the nucleic acid. In this method, the terminal structure characteristic of the present invention is provided by the short primers. In this mode, on the other hand, the whole of a nucleotide sequence constituting a loop is provided as a primer, and synthesis of this longer primer is necessary.</p>
    <p>If a nucleotide sequence containing restriction enzyme recognition regions is used as a reverse primer, a different mode according to the present invention can be constituted. The reaction with a reverse primer containing a restriction enzyme recognition sequence is specifically described by reference to FIG. <b>6</b>. When FIG. <b>6</b>-(D) is completed, a nick is generated by a restriction enzyme corresponding to a restriction enzyme recognition site in the reverse primer. The strand displacement-type reaction of synthesizing complementary chain is initiated from this nick as the origin of synthesis. Because the reverse primers are located at both the terminals of a double-stranded nucleic acid constituting (D), the reaction of synthesizing complementary chain is also initiated from both the terminals. Though basically based on the SDA method described as the prior art, the nucleotide sequence serving as a template has a structure having complementary nucleotide sequences alternately linked according to the present invention so that the nucleic acid synthetic system unique to the present invention is constituted. A part serving as a complementary chain of the reverse primer to be nicked should be designed to incorporate a dNTP derivative such that it is rendered nuclease resistance to prevent cleavage of the double-stranded chain by the restriction enzyme.</p>
    <p>It is also possible to insert a promoter for RNA polymerase into the reverse primer. Transcription from both the terminals in FIG. <b>6</b>-(D) is performed by a RNA polymerase recognizing this promoter in this case too similar to the previous mode where the SDA method was applied.</p>
    <p>The nucleic acid synthesized in the present invention is a single-stranded chain but is composed of partial complementary sequences, and thus the majority of these sequences are base-paired. By use of this feature, the synthesized product can be detected. By carrying out the method of synthesizing nucleic acid according to the present invention in the presence of a fluorescent pigment as a double-stranded chain-specific intercalater such as ethidium bromide, SYBR Green I or Pico Green, the increased density of fluorescence is observed as the product is increased. By monitoring it, it is possible to trace the real-time synthetic reaction in a closed system. Application of this type of detection system to the PCR method is also considered, but it is deemed that there are many problems because the signal from the product cannot be distinguished from signals from primer dimers etc. However, when this system is applied to this invention, the possibility of increasing unspecific base pairing is very low, and thus high sensitivity and low noises can be simultaneously expected. Similar to use of the double-stranded chain-specific intercalater, the transfer of fluorescent energy can be utilized for a method of realizing a detection system in a uniform system.</p>
    <p>The method of synthesizing nucleic acid according to the present invention is supported by the DNA polymerase catalyzing the strand displacement-type reaction for synthesis of complementary chain. During the reaction described above, a reaction step not necessarily requiring the strand displacement-type polymerase is also contained. However, for simplification of a constitutional reagent and in an economical viewpoint, it is advantageous to use one kind of DNA polymerase. As this kind of DNA polymerase, the following enzymes are known. Further, various mutants of these enzymes can be utilized in the present invention insofar as they have both the sequence-dependent activity for synthesis of complementary chain and the strand displacement activity. The mutants referred to herein include those having only a structure bringing about the catalytic activity required of the enzyme or those with modifications to catalytic activity, stability or thermostability by e.g. mutations in amino acids.</p>
    <p>Bst DNA polymerase</p>
    <p>Bca (exo-)DNA polymerase</p>
    <p>DNA polymerase I Klenow fragment</p>
    <p>Vent DNA polymerase</p>
    <p>Vent (exo-)DNA polymerase (Vent DNA polymerase deficient in exonuclease activity)</p>
    <p>Deep Vent DNA polymerase</p>
    <p>Deep Vent(exo-)DNA polymerase (Deep Vent DNA polymerase deficient in exonuclease activity)</p>
    <p>Φ29 phage DNA polymerase</p>
    <p>MS-2 phage DNA polymerase</p>
    <p>Z-Taq DNA polymerase (Takara Shuzo Co., Ltd.)</p>
    <p>KOD DNA polymerase (Toyobo Co., Ltd.)</p>
    <p>Among these enzymes, Bst DNA polymerase and Bca (exo-) DNA polymerase are particularly desired enzymes because they have a certain degree of thermostability and high catalytic activity. The reaction of this invention can be carried isothermally in a preferred embodiment, but because of the adjustment of melting temperature (Tm) etc., it is not always possible to utilize temperature conditions desired for the stability of the enzyme. Accordingly, it is one of the desired conditions that the enzyme is thermostable. Although the isothermal reaction is feasible, heat denaturation may be conducted to provide nucleic acid as a first template, and in this respect too, utilization of a thermostable enzyme broadens selection of assay protocol.</p>
    <p>Vent (exo-) DNA polymerase is an enzyme having both strand displacement activity and a high degree of thermostability. It is known that the complementary chain synthetic reaction involving strand displacement by DNA polymerase is promoted by adding a single strand binding protein (Paul M. Lizardi et al., Nature Genetics, 19, 225-232, July, 1998). This action is applied to the present invention, and by adding the single strand binding protein, the effect of promoting the synthesis of complementary chain can be expected. For example, T4 gene 32 is effective as a single strand binding protein for Vent (exo-) DNA polymerase.</p>
    <p>For DNA polymerase free of 3′-5′ exonuclease activity, there is a known phenomenon where the synthesis of complementary chain does not stop at the 5′-terminal of a template, resulting in generation of a one-base protrusion. In the present invention, this phenomenon is not preferable because when synthesis of the complementary chain reaches the terminal, the sequence of the 3′-terminal leads to initiation of next synthesis of complementary chain. However, because a base “A” is added at high probability to the 3′-terminal by the DNA polymerase, the sequence may be selected such that the synthesis from the 3′-terminal starts at “A”, so that there is no problem if an additional base is added erroneously by dATP. Further, even if the 3′-terminal is protruded during synthesis of complementary chain, the 3′→5′ exonuclease activity can be utilized for digesting the protrusion to make it blunt-ended. For example, since natural Vent DNA polymerase has this activity, this enzyme may be used as a mixture with Vent (exo-) DNA polymerase in order to solve this problem.</p>
    <p>Various reagents necessary for the method of synthesizing or amplifying nucleic acid according to the present invention may be previously packaged and provided as a kit. Specifically, a kit is provided for the present invention, comprising various kinds of oligonucleotides necessary as primers for synthesis of complementary chain and as outer primers for displacement, dNTP as a substrate for synthesis of complementary chain, a DNA polymerase for carrying out the strand displacement-type synthesis of complementary chain, a buffer giving suitable conditions to the enzyme reaction, and as necessary regents necessary for detection of synthetic reaction products. In particular, the addition of reagents is necessary during the reaction in a preferable mode of the present invention, and thus the reagents necessary for one reaction are supplied after pipetted into reaction vessel, whereby the reaction can be initiated by adding only a sample. By constituting a system in which the reaction product can be detected in situ in a reaction vessel by utilizing a luminescent signal or a fluorescent signal, it is not necessary to open and shut the vessel after reaction. This is very desirable for prevention of contamination.</p>
    <p>The nucleic acid having complementary nucleotide sequences alternately linked in a single-stranded chain, synthesized according to the present invention, has e.g. the following usefulness. The first feature is use of an advantage resulting from the special structure having complementary sequences in one molecule. This feature can be expected to facilitate detection. That is, there is a known system for detecting nucleic acid wherein its signal is varied depending on base pairing with a complementary nucleotide sequence. For example, by combination with the method of utilizing a double-stranded chain-specific intercalater as a detector as described above, a detection system making full use of the characteristics of the synthetic product of the present invention can be realized. If the synthetic reaction product of the present invention is once heat-denatured in said detection system and returned to the original temperature, intramolecular annealing occurs preferentially thus permitting complementary sequences to be rapidly base-paired. If there are unspecific reaction products, they have not complementary sequences in the molecule so that after separated by heat denaturation into 2 or more molecules, they cannot immediately be returned to the original double-stranded chain. By providing the step of heat denaturation before detection, noises accompanying the unspecific reaction can be reduced. If the DNA polymerase not resistant to heat is used, the step of heat denaturation has the meaning of termination of the reaction and is thus advantageous for the control of reaction temperature.</p>
    <p>The second feature is to always form a loop capable of base pairing. The structure of a loop capable of base pairing is shown in FIG. <b>4</b>. As can be seen from FIG. 4, the loop is composed of the nucleotide sequence F2c (X2c) which can be annealed by the primer and a nucleotide sequence intervening between F2c-F1c (X1c). The sequence between F2c-F1c (or between X2c-X1c in a more universal form) is a nucleotide derived sequence derived from the template. Accordingly, if a probe having a complementary nucleotide sequence is hybridized with this region, template-specific detection is feasible. In addition, this region is always ready for base pairing, and therefore, heat denaturation prior to hybridization is not necessary. The nucleotide sequence constituting a loop in the amplification reaction product in the present invention may have an arbitrary length. Accordingly, if hybridization with a probe is desired, a region to be annealed by the primer and a region to be hybridized by the probe are arranged separately to prevent their competition, whereby ideal reaction conditions can be constituted.</p>
    <p>According to a preferable mode of the present invention, a large number of loops capable of base pairing are given in a single strand of nucleic acid. This means that a large number of probes can be hybridized with one molecule of nucleic acid to permit highly sensitive detection. It is thus possible to realize not only the improvement of sensitivity but also a method of detecting nucleic acid based on a special reaction principle such as aggregation. For example, a probe immobilized onto fine particles such as polystyrene latex is added to the reaction product of the present invention, the aggregation of latex particles is observed as the hybridization of the product with the probe proceeds. Highly sensitive and quantitative observation is feasible by optically measuring the strength of the aggregation. Because the aggregation can also be observed with the naked eyes, a reaction system not using an optical measuring device can also be constituted.</p>
    <p>Further, the reaction product of the present invention permitting many labels to be bound thereto per nucleic acid molecule enables chromatographic detection. In the field of immunoassay, an analytical method (immunochromatography) using a chromatographic medium utilizing a visually detectable label is used practically. This method is based on the principle that an analyte is sandwiched between an antibody immobilized on a chromatographic medium and a labeled antibody, and the unreacted labeled component is washed away. The reaction product of the present invention makes this principle applicable to analysis of nucleic acid. That is, a labeled probe toward the part of a loop is prepared and immobilized onto a chromatographic medium to prepare a capturing probe for trapping thereby permitting analysis in the chromatographic medium. As the capturing probe, a sequence complementary to the part of the loop can be utilized. Since the reaction product of the present invention has a large number of loops, the product binds to a large number of labeled probes to give a visually recognizable signal.</p>
    <p>The reaction product according to the present invention always giving a region as a loop capable of base pairing enables a wide variety of other detection systems. For example, a detection system utilizing surface plasmon resonance using an immobilized probe for this loop portion is feasible. Further, if a probe for the loop portion is labeled with a double-stranded chain-specific intercalater, more sensitive fluorescent analysis can be conducted. Alternatively, it is also possible to positively utilize the ability of the nucleic acid synthesized by the present invention to form a loop capable of base pairing at both the 3′- and 5′-sides. For example, one loop is designed to have a common nucleotide sequence between a normal type and an abnormal type, while the other loop is designed to generate a difference therebetween. It is possible to constitute a characteristic analytic system in which the presence of a gene is confirmed by the probe for the common portion while the presence of an abnormality is confirmed in the other region. Because the reaction of synthesizing nucleic acid according to the present invention can also proceed isothermally, it is a very important advantage that real-time analysis can be effected by a general fluorescent photometer. Heretofore, the structure of nucleic acid to be annealed in the same chain is known. However, the nucleic acid having complementary nucleotide sequences linked alternately in a single-stranded chain obtained by the present invention is novel in that it contains a large number of loops capable of base pairing with other oligonucleotides.</p>
    <p>On the other hand, a large number of loops themselves given by the reaction product according to the present invention can be used as probes. For example, in a DNA chip, probes should be accumulated at high density in a limited area. In present technology, however, the number of oligonucleotides which can be immobilized in a certain area is limited. Hence, by use of the reaction product of the present invention, a large number of probes capable of annealing can be immobilized at high density. That is, the reaction product according to the present invention may be immobilized as probes on a DNA chip. After amplification, the reaction product may be immobilized by any techniques known in the art, or the immobilized oligonucleotide is utilized as the oligonucleotide in the amplification reaction of the present invention, resulting in generating the immobilized reaction product. By use of the probe thus immobilized, a large number of sample DNAs can be hybridized in a limited area, and as a result, high signals can be expected.</p>
    <heading>BRIEF DESCRIPTION OF THE DRAWINGS</heading> <p>FIG. 1 is an illustration of a part (<b>1</b>) to (<b>4</b>) of the reaction principle in a preferable mode of the present invention.</p>
    <p>FIG. 2 is an illustration of a part (<b>5</b>) to (<b>7</b>) of the reaction principle in a preferable mode of the present invention.</p>
    <p>FIG. 3 is an illustration of a part (<b>8</b>) to (<b>10</b>) of the reaction principle in a preferable mode of the present invention.</p>
    <p>FIG. 4 is an illustration of the structure of a loop formed by the single-stranded nucleic acid according to the present invention.</p>
    <p>FIG. 5 is an illustration of a part (A) to (B) in a basic mode of the present invention.</p>
    <p>FIG. 6 is an illustration of a part (C) to (D) in a basic mode of the present invention.</p>
    <p>FIG. 7 is a drawing showing the positional relationship of each nucleotide sequence constituting an oligonucleotide in the target nucleotide sequence of M13mp18.</p>
    <p>FIG. 8 is a photograph showing the result of agarose electrophoresis of a product obtained by the method of synthesizing single-stranded nucleic acid with M13mp18 as a template according to the present invention.</p>
    <p>Lane 1: XIV size marker</p>
    <p>Lane 2: 1 fmol M13mp18 dsDNA</p>
    <p>Lane 3: No target</p>
    <p>FIG. 9 is a photograph showing the result of agarose gel electrophoresis of a restriction enzyme-digested product obtained in Example 1 by the nucleic acid synthetic reaction according to the present invention.</p>
    <p>Lane 1: XIV size marker</p>
    <p>Lane 2: BamHI digest of the purified product</p>
    <p>Lane 3: PvuII digest of the purified product</p>
    <p>Lane 4: HindIII digest of the purified product</p>
    <p>FIG. 10 is a photograph showing the result of agarose gel electrophoresis of a product obtained by the method of synthesizing single-stranded nucleic acid according to the present invention using M13mp18 as a template in the presence of betaine. 0, 0.5, 1 and 2 indicate the concentration (M) of betaine added to the reaction solution. N indicates the negative control, and −21 indicates the concentration 10<sub>−21 </sub>mol of template DNA.</p>
    <p>FIG. 11 is a drawing showing the positional relationship of each nucleotide sequence constituting an oligonucleotide in a target nucleotide sequence derived from HVB.</p>
    <p>FIG. 12 is a photograph showing the result of agarose gel electrophoresis of a product obtained by the method of synthesizing single-stranded nucleic acid according to the present invention wherein HBV-M13mp18 integrated in M13mp18 was used as a template.</p>
    <p>Lane 1: XIV size marker</p>
    <p>Lane 2: 1 fmol HBV-M13mp18 dsDNA</p>
    <p>Lane 3: No target</p>
    <p>FIG. 13 is a photograph showing the result of gel electrophoresis of an alkali-denatured product obtained by the method of synthesizing single-stranded nucleic acid according to the present invention.</p>
    <p>Lane 1: HindIII-digested fragment from lambda-phage</p>
    <p>Lane 2: The reaction product in Example 1.</p>
    <p>Lane 3: The reaction product in Example 3.</p>
    <p>FIG. 14 is a photograph showing the result of agarose gel electrophoresis of a product obtained by the method of synthesizing single-stranded nucleic acid according to the present invention wherein the concentration of M13mp18 as a target was varied. The upper and lower photographs show the result of the reaction for 1 and 3 hours respectively.</p>
    <p>Lane 1: M13mp18 dsDNA 1×10<sub>−15 </sub>mol/tube</p>
    <p>Lane 2: M13mp18 dSDNA 1×10<sub>−16 </sub>mol/tube</p>
    <p>Lane 3: M13mp18 dsDNA 1×10<sub>−17 </sub>mol/tube</p>
    <p>Lane 4: M13mp18 dsDNA 1×10<sub>−18 </sub>mol/tube</p>
    <p>Lane 5: M13mp18 dsDNA 1×10<sub>−19 </sub>mol/tube</p>
    <p>Lane 6: M13mp18 dsDNA 1×10<sub>−20 </sub>mol/tube</p>
    <p>Lane 7: M13mp18 dsDNA 1×10<sub>−21 </sub>mol/tube</p>
    <p>Lane 8: M13mp18 dsDNA 1×10<sub>−22 </sub>mol/tube</p>
    <p>Lane 9: No target</p>
    <p>Lane 10: XIV size marker</p>
    <p>FIG. 15 is a drawing showing the position of a mutation and the positional relationship of each region toward a target nucleotide sequence (target). Underlined guanine is replaced by adenine in the mutant.</p>
    <p>FIG. 16 is a photograph showing the result of agarose gel electrophoresis of a product according to the amplification reaction of the present invention.</p>
    <p>M: 100 bp ladder (New England Biolabs)</p>
    <p>N: No template (purified water)</p>
    <p>WT: 1 fmol wild-type template M13mp18</p>
    <p>MT: 1 fmol mutant template M13mp18FM</p>
    <p>FIG. 17 is a drawing showing the positional relationship of each nucleotide sequence constituting an oligonucleotide in a nucleotide sequence coding for target mRNA.</p>
    <p>FIG. 18 is a photograph showing the result of agarose electrophoresis of a product obtained by the method of synthesizing single-stranded nucleic acid according to the present invention using mRNA as a target.</p>
    <heading>BEST MODE FOR CARRYING OUT THE INVENTION</heading> <heading>EXAMPLE 1</heading> <heading>Amplification of a Region in M13mp18</heading> <p>The method of synthesizing the nucleic acid having complementary chains alternately linked in a single-stranded chain according to the present invention was attempted using M13mp18 as a template. Four kinds of primers, that is, M13FA, M13RA, M13F3, and M13R3, were used in the experiment. M13F3 and M13R3 were outer primers for displacing the first nucleic acid obtained respectively with M13FA and M13RA as the origin of synthesis. Because the outer primers are primers serving as the origin of synthesis of complementary chain after synthesis with M13FA (or M13RA), these were designed to anneal to a region contiguous to M13FA (or M13RA) by use of the phenomenon of contiguous stacking. Further, the concentrations of these primers were set high such that annealing of M13FA (or M13RA) occurred preferentially.</p>
    <p>The nucleotide sequence constituting each primer is as shown in the Sequence Listing. The structural characteristics of the primers are summarized below. Further, the positional relationship of each region toward the target nucleotide sequence (target) is shown in FIG. <b>7</b>.</p>
    <p>Primer Region at the 5′-side/region at the 3′-side</p>
    <p>M13FA The same as region F1c in complementary chain synthesized by M13FA/complementary to region F2c in M13mp18</p>
    <p>M13RA The same as region R1c in complementary chain synthesized by M13RA/complementary to region R2c in complementary chain synthesized by M13FA</p>
    <p>M13F3 Complementary to F3c contiguous to the 3′-side of region F2c in M13mp18</p>
    <p>M13R3 Complementary to R3c contiguous to the 3′-side of region F2c in complementary chain synthesized by M13FA</p>
    <p>By such primers, nucleic acid wherein a region extending from F1c to R1c in M13mp18, and its complementary nucleotide sequence, are alternately linked via a loop-forming sequence. containing F2c in a single-stranded chain, is synthesized. The composition of a reaction solution for the method of synthesizing nucleic acid by these primers according to the present invention is shown below.</p>
    <p>Composition of the reaction solution (in 25 μL)</p>
    <p>20 mM Tris-HCl pH8.8</p>
    <p>10 mM KCl</p>
    <p>10 mM (NH<sub>4</sub>)<sub>2</sub>SO<sub>4 </sub> </p>
    <p>6 mM MgSO<sub>4 </sub> </p>
    <p>0.1% Triton X-100</p>
    <p>5% dimethyl sulfoxide (DMSO)</p>
    <p>0.4 mM dNTP</p>
    <p>Primers:</p>
    <p>800 nM M13FA/SEQ ID NO:1</p>
    <p>800 nM M13RA/SEQ ID NO:2</p>
    <p>200 nM M13F3/SEQ ID NO:3</p>
    <p>200 nM M13R3/SEQ ID NO:4</p>
    <p>Target: M13mp18 dsDNA/SEQ ID NO:5</p>
    <p>Reaction: The above reaction solution was heated at 95° C. for 5 minutes, and the target was denatured into a single-stranded chain. The reaction solution was transferred to ice-cold water, and 4 U of Bst DNA polymerase (NEW ENGLAND Biolabs) was added thereto and the mixture was reacted at 65° C. for 1 hour. After reaction, the reaction was terminated at 80° C. for 10 minutes and transferred again to ice-cold water.</p>
    <p>Confirmation of the reaction: 1 μl loading buffer was added to 5 μl of the above reaction solution, and the sample was electrophoresed for 1 hour at 80 mV on 2% agarose gel (0.5% TBE). As a molecular-weight marker, XIV (100 bp ladder, Boehringer Mannheim) was used. The gel after electrophoresis was stained with SYBR Green I (Molecular Probes, Inc.) to confirm the nucleic acid. The results are shown in FIG. <b>8</b>. The respective lanes correspond to the following samples.</p>
    <p>1. XIV size marker.</p>
    <p>2. 1 fmol M13mp18 dsDNA.</p>
    <p>3. No target.</p>
    <p>In lane 3, no band was confirmed except that the unreacted primers were stained. In lane 2 in the presence of the target, the products were confirmed as a low size band ladder, as smeared staining at high size and as a band hardly electrophoresed in the gel. Among the low-size bands, bands in the vicinity of 290 bp and 450 bp agree with the products estimated in the synthetic reaction of this invention, that is, double-stranded chains of SEQ ID NOS:11 and 12 (corresponding to double-stranded chains formed as shown in FIGS. <b>2</b>-(<b>7</b>) and <b>2</b>-(<b>10</b>)) and a single-stranded chain of SEQ ID NO:13 (corresponding to the long single-stranded chain in FIG. <b>3</b>-(<b>9</b>)), and it was thus confirmed that the reaction proceeds as expected. It was estimated that the electrophoresis results of the smeared pattern at high size and the band not electrophoresed were brought about because this reaction was basically a continuous reaction to permit varying molecular weights of the reaction product and further because the product has a complicated structure having a partially single-stranded chain and a double-stranded complex.</p>
    <heading>EXAMPLE 2</heading> <heading>Confirmation of the Reaction Products by Digestion with Restriction Enzymes</heading> <p>For the purpose of clarifying the structure of the nucleic acid having complementary nucleotide sequences linked alternately in a single-stranded chain obtained in Example 1 according to the present invention, the digestion of the products with restriction enzymes was conducted. If fragments are theoretically generated by digestion thereof with restriction enzymes and simultaneously the smear pattern at high size and the band not electrophoresed as observed in Example 1 disappear, then it can be estimated that any of these products are the nucleic acid having complementary sequences linked alternately in a single-stranded chain synthesized according to the present invention.</p>
    <p>The reaction solution (200 μl) from 8 tubes in Example 1 was pooled and purified by treatment with phenol and precipitation with ethanol. The resulting precipitates were recovered and dissolved again in 200 μl TE buffer, and 10 μl aliquot was digested at 37° C. for 2 hours with restriction enzymes BamHI, PvuII, and HindIII respectively. The digest was electrophoresed for 1 hour at 80 mV on 2% agarose gel (0.5% TBE). As a molecular marker, Super Ladder-Low (100 bp ladder) (Gensura Laboratories, Inc.) was used. The gel after electrophoresis was stained with SYBR Green I (Molecular Probes, Inc.) to confirm the nucleic acid. The results are shown in FIG. <b>9</b>. The respective lanes correspond to the following samples.</p>
    <p>1. XIV size marker</p>
    <p>2. BamHI digest of the purified product</p>
    <p>3. PvuII digest of the purified product</p>
    <p>4. HindIII digest of the purified product</p>
    <p>It is estimated that nucleotide sequences constituting relatively short amplification products are those of SEQ ID NOS:13, 14, 15 and 16. From these nucleotide sequences, the estimated size of each fragment digested with the restriction as shown in Table 1. “L” in the table indicates that its position in electrophoresis is not established because L is a fragment containing a loop (single-stranded chain).</p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 1</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Restriction enzyme-digested fragments of the</td>
              </tr> <tr class="description-tr"> <td class="description-td">amplification products according to the present invention</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="4"> <colspec colname="1" colwidth="49pt" align="center"> </colspec> <colspec colname="2" colwidth="70pt" align="center"> </colspec> <colspec colname="3" colwidth="28pt" align="center"> </colspec> <colspec colname="4" colwidth="70pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">SEQ ID NO</td>
                <td class="description-td">BamHI</td>
                <td class="description-td">PvuII</td>
                <td class="description-td">HindIII</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="4" align="center" rowsep="1" class="description-td" colspan="4"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">13</td>
                <td class="description-td">177 +L</td>
                <td class="description-td">56 +L</td>
                <td class="description-td">147 +L</td>
              </tr> <tr class="description-tr"> <td class="description-td">14</td>
                <td class="description-td">15 + 101 +L</td>
                <td class="description-td">—</td>
                <td class="description-td">142 +L</td>
              </tr> <tr class="description-tr"> <td class="description-td">15</td>
                <td class="description-td">171 + 101 +L</td>
                <td class="description-td">56 +L</td>
                <td class="description-td">147 + 161 +L</td>
              </tr> <tr class="description-tr"> <td class="description-td">16</td>
                <td class="description-td">11 + 101 + 230 +L</td>
                <td class="description-td">237 +L</td>
                <td class="description-td">142 + 170 +L</td>
              </tr> <tr class="description-tr"> <td class="description-td">Summary</td>
                <td class="description-td">101, 177, 230</td>
                <td class="description-td">56, 237</td>
                <td class="description-td">142, 147, 161, 170</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="4" align="center" rowsep="1" class="description-td" colspan="4"> </td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="4" align="left" class="description-td" colspan="4">(11,15; not confirmed) </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p>Because almost all bands before digestion were changed into estimated bands, it was confirmed that the object reaction products were amplified. Further, it was also shown that there were no or less unspecific products.</p>
    <heading>EXAMPLE 3</heading> <heading>Promotion of Amplification Reaction by Addition of Betaine</heading> <p>An experiment for examining the effect of betaine (N,N,N-trimethylglycine, Sigma) added to the amplification reaction solution on the amplification reaction of nucleic acid was conducted. Synthesis of the nucleic acid having complementary chains alternately linked in a single-stranded chain according to the present invention was conducted using M13mp18 as a template similarly in Example 1 in the presence of betaine at various concentrations. The primers used in the experiment were identical to those used in Example 1. The amount of the template DNA was 10<sup>−21 </sup>mol (M13mp18) and water was used as the negative control. Betaine was added at concentrations of 0, 0.5, 1 and 2. M to the reaction solution. The composition of the reaction solution is shown below.</p>
    <p>Composition of the reaction solution (in 25 μL)</p>
    <p>20 mM Tris-HCl pH8.8</p>
    <p>4 mM MgSO<sub>4 </sub> </p>
    <p>0.4 mM dNTPs</p>
    <p>10 mM KCl</p>
    <p>10 mM (NH<sub>4</sub>)<sub>2</sub>SO<sub>4 </sub> </p>
    <p>0.1% Triton X-100</p>
    <p>Primers:</p>
    <p>800 nM M13FA/SEQ ID NO:1</p>
    <p>800 nM M13RA/SEQ ID NO:2</p>
    <p>200 nM M13F3/SEQ ID N<b>0</b>:3</p>
    <p>200 nM M13R3/SEQ ID NO:4</p>
    <p>Target: M13mp18 dsDNA/SEQ ID NO:5</p>
    <p>The polymerase, reaction conditions, and conditions for electrophoresis after the reaction were identical to those described in Example 1.</p>
    <p>The results are shown in FIG. <b>10</b>. In the reaction in the presence of betaine at a concentration of 0.5 or 1.0 M, the amount of the amplification product was increased. Further, if its concentration was increased to 2.0 M, no amplification was observed. It was thus shown that the amplification reaction was promoted in the presence of betaine at a suitable concentration. The estimated reason that the amount of the amplification product was decreased when the concentration of betaine was 2 M was that Tm was lowered too much.</p>
    <heading>EXAMPLE 4</heading> <heading>Amplification of HBV Gene Sequence</heading> <p>The method of synthesizing nucleic acid according to the present invention was attempted wherein M13mp18 dsDNA having a partial sequence of HBV gene integrated therein was used as a template. Four kinds of primers, HB65FA (SEQ ID NO:6), HB65RA (SEQ ID NO:7), HBF3 (SEQ ID NO:8) and HBR3 (SEQ ID NO:9), were used in the experiment. HBF3 and HBR3 were outer primers for displacement of the first nucleic acid obtained respectively with HB65FA and HB65RA as the origin of synthesis. Because the outer primers are primers serving as the origin of synthesis of complementary chain after synthesis with HB65FA (or HB65RA), these were designed to anneal to a region contiguous to HB65FA (or HB65RA) by use of the phenomenon of contiguous stacking. Further, the concentrations of these primers were set high such that annealing of HB65FA (or HB65RA) occurred preferentially. The target sequence (430 bp) in this example, derived from HBV integrated in M13mp18, is shown in SEQ ID NO:10.</p>
    <p>The nucleotide sequence constituting each primer is shown in the Sequence Listing. The structural feature of each primer is summarized below. Further, the positional relationship of each region toward the target nucleotide sequence (target) is shown in FIG. <b>11</b>.</p>
    <p>Primer Region at the 5′-side/region at the 3′-side</p>
    <p>HB65FA The same as region F1c in complementary chain synthesized by HB65FA/complementary to region F2c in HBV-M13mp18</p>
    <p>HB65RA The same as region R1c in complementary chain synthesized by HB65RA/complementary to region R2c in complementary chain synthesized by HB65FA</p>
    <p>HBF3 Complementary to F3c contiguous to the 3′-side of region F2c in HBV-M13mp18</p>
    <p>HBR3 Complementary to R3c contiguous to the 3′-side of region F2c in complementary chain synthesized by HB65FA</p>
    <p>By such primers, nucleic acid wherein a region extending from F1c to R1c in M13mp18 (HBV-M13mp18) having a partial sequence of HBV gene integrated therein, and its complementary nucleotide sequence, are alternately linked via a loop-forming sequence containing F2c in a single-stranded chain, is synthesized. The reaction was conducted under the same conditions as in Example 1 except that the primers described above were used, and the reaction solution was analyzed by agarose electrophoresis. The results are shown in FIG. <b>12</b>. The respective lanes correspond to the following samples.</p>
    <p>1. XIV size marker</p>
    <p>2. 1 fmol HBV-M13mp18 dsDNA.</p>
    <p>3. No target</p>
    <p>Similar to Example 1, the products were confirmed only in the presence of the target as a low size band ladder, as smeared staining at high size and as a band hardly electrophoresed in the gel (lane 2). Among the low-size bands, bands in the vicinity of 310 bp and 480 bp agree with the products estimated in the synthetic reaction of this invention, that is, double-stranded chains of SEQ ID NOS:17 and 18, and it was thus confirmed that the reaction proceeds as expected. As described in the results in Example 1, it was estimated that the smeared pattern at high size and the band not electrophoresed were caused by the structure of the synthetic product characteristic of the present invention. From this experiment, it was confirmed that the present invention can be practiced even if a different sequence (target) is used for amplification.</p>
    <heading>EXAMPLE 5.</heading> <heading>Confirmation of the Sizes of the Synthetic Reaction Products</heading> <p>To confirm the structure of the nucleic acid synthesized according to the present invention, its length was analyzed by electrophoresis under alkali-denaturing conditions. 1 μl alkaline loading buffer was added to 5 μl of each reaction solution in the presence of the target in Example 1 or 4 and electrophoresed at 50 mA in 0.7% agarose gel (50 mM NaOH, 1 mM EDTA) for 14 hours. As the molecular-weight size marker, HindIII-digested lambda-phage fragments were used. The gel after electrophoresis was neutralized with 1 M Tris, pH 8 and stained with SYBR Green I (Molecular Probes, Inc.) to confirm the nucleic acid. The results are shown in FIG. <b>13</b>. The respective lanes correspond to the following samples.</p>
    <p>1. HindIII-digested fragments from lambda-phage.</p>
    <p>2. The reaction product in Example 1.</p>
    <p>3. The reaction product in Example 4.</p>
    <p>When the reaction product was electrophoresed under alkali-denaturing conditions, its size in a single-stranded state could be confirmed. It was confirmed that the sizes of the major products in both Example 1 (lane 2) and Example 4 (lane 3) were within 2 kbase. Further, it was revealed that the product according to the present invention had been extended to have a size of at least 6 kbase or more within the range capable of confirmation by this analysis. In addition, it was confirmed again that bands not electrophoresed under non-denaturing conditions in Examples 1 and 4 were separated in a denatured state into individual single-stranded chains of smaller size.</p>
    <heading>EXAMPLE 6</heading> <heading>Confirmation of Amplification Depending on the Concentration of a Target in the Amplification of a Region in M-13mp13</heading> <p>The influence of a varying concentration of a target on the method of synthesizing nucleic acid according to the present invention was observed. The method of synthesizing nucleic acid according to the present invention was carried out under the same conditions as in Example 1 except that the amount of M13mp18 dsDNA as the target was 0 to 1 fmol and the reaction time was 1 hour or 3 hours. Similar to Example 1, the sample was electrophoresed in 2% agarose gel (0.5% TBE) and stained with SYBR Green I (Molecular Probes, Inc.) to confirm the nucleic acid. As a molecular size marker, XIV (100 bp ladder, Boehringer Mannheim) was used. The results are shown in FIG. 14 (upper: 1-hour reaction, below: 3-hour reaction). The respective lanes correspond to the following samples:</p>
    <p>1. M13mp18 dSDNA 1×10<sub>−15 </sub>mol/tube.</p>
    <p>2. M13mp18 dsDNA 1×10<sub>−16 </sub>mol/tube.</p>
    <p>3. M13mp18 dsDNA 1×10<sub>−17 </sub>mol/tube.</p>
    <p>4. M13mp18 dsDNA 1×10<sub>−18 </sub>mol/tube.</p>
    <p>5. M13mp18 dsDNA 1×10<sub>−19 </sub>mol/tube.</p>
    <p>6. M13mp18 dsDNA 1×10<sub>−20 </sub>mol/tube.</p>
    <p>7. M13mp18 dsDNA 1×10<sub>−21 </sub>mol/tube.</p>
    <p>8. M13mp18 dsDNA 1×10<sub>−22 </sub>mol/tube.</p>
    <p>9. No target.</p>
    <p>10. XIV size marker.</p>
    <p>A common band among the respective lanes appears in a lower part in the electrophoretic profile and shows the unreacted stained primers. Regardless of the reaction time, no amplification product is observed in the absence of the target. A staining pattern, depending on the concentration of the target, of the amplification product was obtained only in the presence of the target. Further, the amplification product could be confirmed at lower concentration as the reaction time was increased.</p>
    <heading>EXAMPLE 7</heading> <heading>Detection of a Point Mutation</heading> <p>(1) Preparation of M13mp18FM (Mutant)</p>
    <p>The target DNA used was M13mp18 (wild-type) and M13mp18FM (mutant). For the construction of the mutant M13mp18FM, LA PCR™ in vitro Mutagenesis Kit (Takara Shuzo Co., Ltd.) was used to replace one nucleotide for mutation. Thereafter, the sequence was confirmed by sequencing. The sequence of the F1 region is shown below:</p>
    <p>Wild-type: CCGGGGATCCTCTAGAGTCG (SEQ ID NO:19)</p>
    <p>Mutant: CCGGGGATCCTCTAGAGTCA (SEQ ID NO:20)</p>
    <p>(2) Design of Primers</p>
    <p>The FA primers used for the wild-type and the mutant were provided at the 5′-terminal of the F1c region thereof with different nucleotide sequences, respectively. The location of the mutation and the positional relationship of each region toward the target nucleotide sequence (target) are shown in FIG. <b>15</b>.</p>
    <p>(3) Amplification Reaction</p>
    <p>An experiment was conducted to examine whether template-specific amplification reaction occurs using a combination of specific primers shown below by use of M13mp18 (wild-type) and M13mp18FM (mutant) as primers.</p>
    <p>Primer set for wild-type amplification: FAd4, RAd4, F3, R3</p>
    <p>Primer set for mutant amplification: FAMd4, RAd4, F3, R3</p>
    <p>The nucleotide sequence of each primer is as follows:</p>
    <p>FAd4: CGACTCTAGAGGATCCCCGGTTTTTGTTGTGTGGAATTGTGAGCGGAT (SEQ ID NO:21)</p>
    <p>FAMd4: TGACTCTAGAGGATCCCCGGTTTTTGTTGTGTGGAATTGTGAGCGGAT (SEQ ID NO:22)</p>
    <p>RAd4: CGTCGTGACTGGGAAAACCCTTTTTGTGCGGGCCTCTTCGCTATTAC (SEQ ID NO:23)</p>
    <p>F3: ACTTTATGCTTCCGGCTCGTA (SEQ ID NO:24)</p>
    <p>R3: GTTGGGAAGGGCGATCG (SEQ ID NO:25)</p>
    <p>(4) Detection of the Point Mutation in M13mp18</p>
    <p>The composition of the reaction solution is as follows:</p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="140pt" align="left"> </colspec> <colspec colname="1" colwidth="77pt" align="left"> </colspec> <thead> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="OFFSET" nameend="1" align="center" rowsep="1" class="description-td" colspan="2"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Final concentration</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="OFFSET" nameend="1" align="center" rowsep="1" class="description-td" colspan="2"> </td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="1" colwidth="112pt" align="left"> </colspec> <colspec colname="2" colwidth="28pt" align="left"> </colspec> <colspec colname="3" colwidth="77pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">D2W</td>
                <td class="description-td">3.75 μL</td>
                <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">10X Thermo pol buffer (NEB)</td>
                <td class="description-td">2.5 μL</td>
                <td class="description-td">20 mM Tris-HCl pH 8.8</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">10 mM KCl</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">10 mM (NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">6 mM MgSO<sub>4</sub> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">0.1% TritonX-100</td>
              </tr> <tr class="description-tr"> <td class="description-td">2.5 mM dNTP</td>
                <td class="description-td">4 μL</td>
                <td class="description-td">400 μM</td>
              </tr> <tr class="description-tr"> <td class="description-td">100 mM MgSO<sub>4</sub> </td>
                <td class="description-td">0.5 μL</td>
              </tr> <tr class="description-tr"> <td class="description-td">4 M Betaine</td>
                <td class="description-td">6.25 μL</td>
                <td class="description-td">1 M</td>
              </tr> <tr class="description-tr"> <td class="description-td">M13FAd4 primer (10 pmol/μL) or</td>
              </tr> <tr class="description-tr"> <td class="description-td">M13FAMd4 primer (10 pmol/μL)</td>
                <td class="description-td">2 μL</td>
                <td class="description-td">800 nM</td>
              </tr> <tr class="description-tr"> <td class="description-td">M13RAd4 primer (10 pmol/μL)</td>
                <td class="description-td">2 μL</td>
                <td class="description-td">800 nM</td>
              </tr> <tr class="description-tr"> <td class="description-td">M13F3 primer (10 pmol/μL)</td>
                <td class="description-td">0.5 μL</td>
                <td class="description-td">200 nM</td>
              </tr> <tr class="description-tr"> <td class="description-td">M13R3 primer (10 pmol/μL)</td>
                <td class="description-td">0.5 μL</td>
                <td class="description-td">200 nM</td>
              </tr> <tr class="description-tr"> <td class="description-td">Total amount</td>
                <td class="description-td">22 μL</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="3" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p>1 fmol (2 μl) of the target M13mp18 or M13mp18FM was added to the reaction solution and heated at 95° C. for 5 minutes whereby the target was denatured into a single-stranded chain. The reaction solution was transferred to ice-cold water, and 1 μl (8 U) of Bst DNA polymerase (NEW ENGLAND Biolab) was added thereto and reacted for 1 hour at 68° C. or 68.5° C. After reaction, the reaction was terminated at 80° C. for 10 minutes, and the reaction solution was transferred again to ice-cold water.</p>
    <p>As shown in FIG. 16, when FAd4 for wild type was used as the FA primer, effective amplification was observed only in the presence of the wild-type template. On the other hand, when FAMd4 for mutant was used as the FA primer, effective amplification was observed only in the presence of the wild-type [sic.] template.</p>
    <p>From the results described above, it was shown that the point mutation could be detected efficiently by use of the amplification reaction of the present invention.</p>
    <heading>EXAMPLE 8</heading> <heading>Amplification Reaction of mRNA as a Target</heading> <p>The method of synthesizing nucleic acid according to the present invention was attempted using mRNA as the target nucleic acid. To prepare the target mRNA, prostate cancer cell line LNCaP cells (ATCC No. CRL-1740) expressing prostate specific antigen (PSA) were mixed with chronic myeloid leukemia cell line K562 cells (ATCC No. CCL-243) as non-expressing cells at 1:10<sub>6 </sub>to 100:10<sub>6</sub>, followed by extraction of the total RNA by use of an RNeasy Mini kit from Qiagen (Germany). Four kinds of primers, that is, PSAFA, PSARA, PSAF3 and PSAR3, were used in the experiment. PSAF3 and PSAR3 are outer primers for displacing the first nucleic acid obtained respectively with PSAFA and PSARA as the origin of synthesis. Further, the concentrations of these primers were set high such that annealing of PSAFA (or PSARA) occurred preferentially. The nucleotide sequences constituting the respective primers are as follows.</p>
    <p>Primer:</p>
    <p>PSAFA: TGTTCCTGATGCAGTGGGCAGCTTTAGTCTGCGGCGGTGTTCTG (SEQ ID NO: 26)</p>
    <p>PSARA: TGCTGGGTCGGCACAGCCTGAAGCTGACCTGAAATACCTGGCCTG (SEQ ID NO: 27)</p>
    <p>PSAF3: TGCTTGTGGCCTCTCGTG (SEQ ID NO: 28)</p>
    <p>PSAR3: GGGTGTGGGAAGCTGTG (SEQ ID NO: 29)</p>
    <p>The structural features of the primers are summarized below. Further, the positional relationship of each primer toward the DNA nucleotide sequence coding for the target mRNA and recognition sites of restriction enzyme Sau3AI are shown in FIG. <b>17</b>.</p>
    <p>Primer Region at the 5′-side/region at the 3′-side</p>
    <p>PSAFA The same as region F1c in complementary chain synthesized by PSAFA/complementary to region F2c in the target nucleotide sequence</p>
    <p>PSARA The same as region R1c in complementary chain synthesized by PSARA/complementary to region R2c in complementary chain synthesized by PSAFA</p>
    <p>PSAF3 Complementary to F3c contiguous to the 3′-side of region F2c in the target nucleotide sequence</p>
    <p>PSAR3 Complementary to R3c contiguous to the 3′-side of region R2c in complementary chain synthesized by PSAFA</p>
    <p>The composition of a reaction solution for the method of synthesizing nucleic acid according to the present invention is as follows:</p>
    <p>Composition of the reaction solution (in 25 μL)</p>
    <p>20 mM Tris-HCl pH 8.8</p>
    <p>4 mM MgSO<sub>4 </sub> </p>
    <p>0.4 mM dNTPs</p>
    <p>10 mM KCl</p>
    <p>10 MM (NH<sub>4</sub>)<sub>2</sub>SO<sub>4 </sub> </p>
    <p>0.1% Triton X-100</p>
    <p>0.8 M betaine</p>
    <p>5 mM DTT</p>
    <p>1600 nM PSAFA &amp; PSARA primer</p>
    <p>200 nM PSAF3 &amp; PSAR3 primer</p>
    <p>8 U Bst DNA polymerase</p>
    <p>100 U Rever Tra Ace (Toyobo Co., Ltd., Japan)</p>
    <p>5 μg total RNA</p>
    <p>All ingredients were mixed on ice. In this experiment, MRNA (single-stranded chain) is used as a target, and thus the step of making single-stranded chain by heat denaturation is not necessary. The reaction was conducted at 65° C. for 45 minutes, and the reaction was terminated at 85° C. for 5 minutes. After reaction, 5 μl of the reaction solution was electrophoresed in 2% agarose and detected by SYBR Green I.</p>
    <p>The results are shown in Table 18. The respective lanes correspond to the following samples.</p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="5"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="21pt" align="center"> </colspec> <colspec colname="2" colwidth="28pt" align="center"> </colspec> <colspec colname="3" colwidth="14pt" align="center"> </colspec> <colspec colname="4" colwidth="140pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="OFFSET" nameend="4" align="center" rowsep="1" class="description-td" colspan="5"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Lane</td>
                <td class="description-td">Bst</td>
                <td class="description-td">RT</td>
                <td class="description-td">Number of LNCaP cells/10<sup>6 </sup>K562 cells</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="OFFSET" nameend="4" align="center" rowsep="1" class="description-td" colspan="5"> </td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="5"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="21pt" align="center"> </colspec> <colspec colname="2" colwidth="28pt" align="center"> </colspec> <colspec colname="3" colwidth="14pt" align="center"> </colspec> <colspec colname="4" colwidth="140pt" align="char" char="."> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">1</td>
                <td class="description-td">−</td>
                <td class="description-td">+</td>
                <td class="description-td">0</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">2</td>
                <td class="description-td">−</td>
                <td class="description-td">+</td>
                <td class="description-td">10</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">3</td>
                <td class="description-td">+</td>
                <td class="description-td">−</td>
                <td class="description-td">0</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">4</td>
                <td class="description-td">+</td>
                <td class="description-td">−</td>
                <td class="description-td">10</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">5</td>
                <td class="description-td">+</td>
                <td class="description-td">+</td>
                <td class="description-td">0</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">6</td>
                <td class="description-td">+</td>
                <td class="description-td">+</td>
                <td class="description-td">1</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">7</td>
                <td class="description-td">+</td>
                <td class="description-td">+</td>
                <td class="description-td">10</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="OFFSET" nameend="4" align="center" rowsep="1" class="description-td" colspan="5"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p>8 Sau3AI digest of 1 μL aliquot of the reaction solution in lane 6</p>
    <p>9 Sau3AI digest of 1 μL aliquot of the reaction solution in lane 7</p>
    <p>10 Size maker, 100 bp ladder (New England Biolabs)</p>
    <p>In the absence of either Bst DNA polymerase or Rever Tra Ace, no amplification product could be obtained (lanes 1 to 4). In the presence of both the enzymes, an amplification product was detected (lanes 5 to 7) if RNA derived from LNCaP was present. RNA extracted from one LNCaP cell/one million K562 cells could be detected (lane 6). When the amplification product was digested at the restriction enzyme site Sau3AI located in the inside of the target, the product was digested into a fragment of estimated size (lanes 8 and 9).</p>
    <p>From the results described above, it was confirmed that the desired reaction product can be obtained in the method of synthesizing nucleic acid according to the present invention even if RNA is used as a target.</p>
    <heading>INDUSTRIAL APPLICABILITY</heading> <p>According to the novel oligonucleotide according to the present invention and the method of synthesizing nucleic acid by using said oligonucleotide, there is provided a method of synthesizing nucleic acid having complementary nucleotide sequences linked alternately in a single-stranded chain, without requiring any complicated control of temperature. A complementary chain synthesized with the oligonucleotide as a primer based on the present invention serves as the origin of synthesizing a new complementary chain with the 3′-terminal of said synthesized chain as a template. This is accompanied by formation of a loop causing annealing of a new primer, and a product of the reaction of synthesizing complementary chain with the previously synthesized chain as a template is displaced again by synthesis of complementary chain from the loop and made ready for base pairing. The thus obtained nucleic acid synthesized with itself as a template is combined with e.g. a known nucleic acid synthesizing method such as SDA, to contribute the improvement of efficiency of nucleic acid synthesis.</p>
    <p>According to an additional preferred mode of the present invention, there is provided a novel method of synthesizing nucleic acid, which achieves the improvement of efficiency of the known method of synthesizing nucleic acid, does not require complicated control of temperature, can be expected to attain high efficiency of amplification and can achieve high specificity. That is, the oligonucleotide based on the present invention is applied to a template chain and its complementary chain whereby nucleic acid having complementary sequences linked alternately in a single-stranded chain can be successively synthesized. This reaction continues theoretically until the starting materials necessary for synthesis are exhausted, during which new nucleic acid initiated to be synthesized from the loop continues to be formed. The elongation from the oligonucleotide having annealed to the loop performs strand displacement for supplying 3′—OH for elongation of long single-stranded nucleic acid (that is, nucleic acid having plural pairs of complementary chains linked therein). On the other hand, the 3′—OH of the long single-stranded chain performs the reaction of synthesizing complementary chain with itself as a template whereby its elongation is achieved, during which a new complementary chain whose synthesis is initiated from the loop is displaced. Such an amplification reaction step proceeds under isothermal conditions while maintaining high specificity.</p>
    <p>The oligonucleotides in the present invention can, when two contiguous regions are arranged as designed, function as primers for the reaction of synthesizing nucleic acid according to the present invention. This contributes significantly to the preservation of specificity. By comparison with e.g. PCR where unspecific amplification reaction is initiated by unspecific missannealing regardless of the intended positional relationship of 2 primers, it can be easily explained that high specificity can be expected in the present invention. This feature can be utilized to detect SNPs highly sensitively and accurately.</p>
    <p>The characterizing feature of the present invention lies in that such reaction can be easily achieved by a very simple constitution of reagents. For example, the oligonucleotide according to the present invention has a special structure, but this is a matter of selection of nucleotide sequence, and it is a simple oligonucleotide as substance. Further, in a preferred mode, the reaction can proceed by only a DNA polymerase catalyzing the strand displacement-type reaction of synthesizing complementary chain. Further, if the present invention is carried out with RNA as a template, a DNA polymerase such as Bca DNA polymerase having both reverse transcriptase activity and strand displacement-type DNA polymerase activity is used so that all enzyme reactions can be conducted by the single enzyme. The reaction principle of realizing a high degree of nucleic acid amplification reaction by such simple enzyme reaction is not known. Even for the application of the present invention to a known nucleic acid synthesizing reaction such as SDA, no additional enzyme is necessary for their combination, and such a simple combination with the oligonucleotide based on the present invention can be applied to various reaction systems. Accordingly, it can be said that the method of synthesizing nucleic acid according to the present invention is also advantageous in respect of cost.</p>
    <p>As described above, the method of synthesizing nucleic acid according to the present invention and the oligonucleotide therefor provide a new principle of simultaneously solving a plurality of difficult problems such as operativeness (temperature control is not necessary), improvement of efficiency of synthesis, economization, and high specificity.</p>
    </div></div></div><div class="patent-section patent-tabular-section"><a id="backward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Cited Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4683202">US4683202</a></td><td class="patent-data-table-td patent-date-value">Oct 25, 1985</td><td class="patent-data-table-td patent-date-value">Nov 27, 1990</td><td class="patent-data-table-td ">Cetus Corp</td><td class="patent-data-table-td citation-no-title">Title not available</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5270184">US5270184</a></td><td class="patent-data-table-td patent-date-value">Nov 19, 1991</td><td class="patent-data-table-td patent-date-value">Dec 14, 1993</td><td class="patent-data-table-td ">Becton, Dickinson And Company</td><td class="patent-data-table-td ">Amplification</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5525462">US5525462</a></td><td class="patent-data-table-td patent-date-value">Apr 28, 1992</td><td class="patent-data-table-td patent-date-value">Jun 11, 1996</td><td class="patent-data-table-td ">Toyo Boseki Kabushiki Kaisha</td><td class="patent-data-table-td ">For detection of a specific nucleic acid sequence in a sample</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5595891">US5595891</a></td><td class="patent-data-table-td patent-date-value">Jul 19, 1990</td><td class="patent-data-table-td patent-date-value">Jan 21, 1997</td><td class="patent-data-table-td ">Behringwerke Ag</td><td class="patent-data-table-td ">Method for producing a polynucleotide for use in single primer amplification</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5612199">US5612199</a></td><td class="patent-data-table-td patent-date-value">Apr 1, 1994</td><td class="patent-data-table-td patent-date-value">Mar 18, 1997</td><td class="patent-data-table-td ">Behringwerke Ag</td><td class="patent-data-table-td ">Method for producing a polynucleotide for use in single primer amplification</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5874260">US5874260</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Oct 27, 1995</td><td class="patent-data-table-td patent-date-value">Feb 23, 1999</td><td class="patent-data-table-td ">Bio Merieux</td><td class="patent-data-table-td ">Stem loop structure for oligonucleotides with blocking agents to prevent replication with polymerase for use as primers</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6025139">US6025139</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Aug 28, 1996</td><td class="patent-data-table-td patent-date-value">Feb 15, 2000</td><td class="patent-data-table-td ">Visible Genetics Inc.</td><td class="patent-data-table-td ">Hybridizing sample with oligonucleotides contiguously complementary to wild-type sequence including labeled 3&#39; and 5&#39; probes and at least one intermediate probe, ligating oligonucleotides together, evalutating length and quantity</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6033881">US6033881</a></td><td class="patent-data-table-td patent-date-value">Jun 13, 1996</td><td class="patent-data-table-td patent-date-value">Mar 7, 2000</td><td class="patent-data-table-td ">Himmler; Gottfried</td><td class="patent-data-table-td ">Method for one step isothermal non-transcription based amplification of nucleic acids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP0549107A1?cl=en">EP0549107A1</a></td><td class="patent-data-table-td patent-date-value">Oct 9, 1992</td><td class="patent-data-table-td patent-date-value">Jun 30, 1993</td><td class="patent-data-table-td ">BEHRINGWERKE Aktiengesellschaft</td><td class="patent-data-table-td ">Method for producing a polynucleotide for use in single primer amplification and phosphorothioate-containing oligonucleotides as primers in nucleic acid amplification</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP0971039A2?cl=en">EP0971039A2</a></td><td class="patent-data-table-td patent-date-value">Jun 24, 1999</td><td class="patent-data-table-td patent-date-value">Jan 12, 2000</td><td class="patent-data-table-td ">Enzo Diagnostics, Inc.</td><td class="patent-data-table-td ">Processes useful for nucleic acid amplification and sequencing, and for the production of nucleic acid having decreased thermodynamic stability</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1996001327A1?cl=en">WO1996001327A1</a></td><td class="patent-data-table-td patent-date-value">Jul 4, 1995</td><td class="patent-data-table-td patent-date-value">Jan 18, 1996</td><td class="patent-data-table-td ">Fabrice David</td><td class="patent-data-table-td ">Nucleic acid sequence amplification method</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1999066071A1?cl=en">WO1999066071A1</a></td><td class="patent-data-table-td patent-date-value">Nov 25, 1998</td><td class="patent-data-table-td patent-date-value">Dec 23, 1999</td><td class="patent-data-table-td ">Zeneca Ltd</td><td class="patent-data-table-td ">Pcr primer constructs for use in an integrated signalling system</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="npl-citations"></a><div class="patent-section-header"><span class="patent-section-title">Non-Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th colspan="3"class="patent-data-table-th">Reference</th></tr></thead><tr><td class="patent-data-table-td ">1</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Lizardi et al., "<a href='http://scholar.google.com/scholar?q="Mutation+Detection+and+Single-Molecule+Counting+Using+Isothermal+Rolling-Circle+Amplification%2C"'>Mutation Detection and Single-Molecule Counting Using Isothermal Rolling-Circle Amplification,</a>" Nature Genetics 19:225-232 (1998).</td></tr><tr><td class="patent-data-table-td ">2</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Stratagene Catalog, "<a href='http://scholar.google.com/scholar?q="Gene+Characterization+kits"'>Gene Characterization kits</a>", p. 39, column 1, Feb. 1988.*</td></tr><tr><td class="patent-data-table-td ">3</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Whitcombe et al., "<a href='http://scholar.google.com/scholar?q="Detection+of+PCR+Products+Using+Self-probing+Amplicons+and+Fluorescence%2C"'>Detection of PCR Products Using Self-probing Amplicons and Fluorescence,</a>" Nature Biotechnology 17:804-807 (1999).</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="forward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Referenced by</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Citing Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6974670">US6974670</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Apr 24, 2002</td><td class="patent-data-table-td patent-date-value">Dec 13, 2005</td><td class="patent-data-table-td ">Eiken Kagaku Kabushiki Kaisha</td><td class="patent-data-table-td ">Method of synthesizing nucleic acid</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7056671">US7056671</a></td><td class="patent-data-table-td patent-date-value">Aug 20, 2002</td><td class="patent-data-table-td patent-date-value">Jun 6, 2006</td><td class="patent-data-table-td ">Takara Bio Inc.</td><td class="patent-data-table-td ">mixing deoxyribonucleotide triphosphates, polymerase, complementary sequences and ribonucleases, then incubating the mixture to generate reaction products; biosynthesis; genetic engineering</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7175985">US7175985</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Aug 2, 2002</td><td class="patent-data-table-td patent-date-value">Feb 13, 2007</td><td class="patent-data-table-td ">Eiken Kagaku Kabushiki Kaisha</td><td class="patent-data-table-td ">within specific regions of target nucleotide sequences comprising incubating nucleic acids, DNA polymerases, primers and nucleotide substrates, under conditions that allow complementary strand biosynthesis, and analyzing for mutations</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7208295">US7208295</a></td><td class="patent-data-table-td patent-date-value">Nov 19, 2002</td><td class="patent-data-table-td patent-date-value">Apr 24, 2007</td><td class="patent-data-table-td ">Affymetrix, Inc.</td><td class="patent-data-table-td ">Joining nucleic acid to oligonucleotide sequences; obtain oligonucleotide, incubate with primer, ligate, denature, amplify, recover chain extended oligonucleotide</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7256020">US7256020</a></td><td class="patent-data-table-td patent-date-value">Feb 3, 2003</td><td class="patent-data-table-td patent-date-value">Aug 14, 2007</td><td class="patent-data-table-td ">Third Wave Technologies, Inc.</td><td class="patent-data-table-td ">Using rolling circle amplification of propagate and detect preferential nucleotide sequences</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7344048">US7344048</a></td><td class="patent-data-table-td patent-date-value">Jan 21, 2005</td><td class="patent-data-table-td patent-date-value">Mar 18, 2008</td><td class="patent-data-table-td ">Sysmex Corporation</td><td class="patent-data-table-td ">Dispensing device and analyzing apparatus</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7355194">US7355194</a></td><td class="patent-data-table-td patent-date-value">Jan 21, 2005</td><td class="patent-data-table-td patent-date-value">Apr 8, 2008</td><td class="patent-data-table-td ">Sysmex Corporation</td><td class="patent-data-table-td ">Optical device and turbidity detection apparatus using same</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7402386">US7402386</a></td><td class="patent-data-table-td patent-date-value">Apr 14, 2004</td><td class="patent-data-table-td patent-date-value">Jul 22, 2008</td><td class="patent-data-table-td ">Nugen Technologies, Inc.</td><td class="patent-data-table-td ">Global amplification using random priming by a composite primer</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7413857">US7413857</a></td><td class="patent-data-table-td patent-date-value">Nov 21, 2003</td><td class="patent-data-table-td patent-date-value">Aug 19, 2008</td><td class="patent-data-table-td ">Epicentre Technologies</td><td class="patent-data-table-td ">Methods for using riboprimers for strand displacement replication of target sequences</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7468245">US7468245</a></td><td class="patent-data-table-td patent-date-value">Nov 27, 2002</td><td class="patent-data-table-td patent-date-value">Dec 23, 2008</td><td class="patent-data-table-td ">Enzo Life Sciences, Inc.</td><td class="patent-data-table-td ">Methods for detecting nucleic acid sequences</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7485417">US7485417</a></td><td class="patent-data-table-td patent-date-value">Nov 22, 2002</td><td class="patent-data-table-td patent-date-value">Feb 3, 2009</td><td class="patent-data-table-td ">Enzo Life Sciences, Inc.</td><td class="patent-data-table-td ">Primers for use as amplification and diagnostic tools in genetic engineering</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7488581">US7488581</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Mar 20, 2006</td><td class="patent-data-table-td patent-date-value">Feb 10, 2009</td><td class="patent-data-table-td ">Kabushiki Kaisha Toshiba</td><td class="patent-data-table-td ">Reacting the nucleic acid for measurement with the probe nucleic acid to specifically hybridize the target sequence portion of the nucleic acid for measurement to the probe nucleic acid; detecting presence or absence of the nucleic acid for measurement hybridized to the probe nucleic acid</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7494790">US7494790</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Feb 19, 2004</td><td class="patent-data-table-td patent-date-value">Feb 24, 2009</td><td class="patent-data-table-td ">Eiken Kagaku Kabushiki Kaisha</td><td class="patent-data-table-td ">Preparing single stranded nucleotide sequences capable of forming stem-loop structures; for use as tools in gene expression analysis and amplification</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7510829">US7510829</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Nov 19, 2002</td><td class="patent-data-table-td patent-date-value">Mar 31, 2009</td><td class="patent-data-table-td ">Affymetrix, Inc.</td><td class="patent-data-table-td ">Primers for use in the amplification of preferential nucleotide sequences</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7615625">US7615625</a></td><td class="patent-data-table-td patent-date-value">Jun 29, 2007</td><td class="patent-data-table-td patent-date-value">Nov 10, 2009</td><td class="patent-data-table-td ">Replicon, Inc.</td><td class="patent-data-table-td ">In vitro amplification of nucleic acid molecules via circular replicons</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7638280">US7638280</a></td><td class="patent-data-table-td patent-date-value">Dec 22, 2006</td><td class="patent-data-table-td patent-date-value">Dec 29, 2009</td><td class="patent-data-table-td ">Eiken Kagaku Kabushiki Kaisha</td><td class="patent-data-table-td ">Method for detecting mutations and/or polymorphisms</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7659072">US7659072</a></td><td class="patent-data-table-td patent-date-value">Aug 5, 2008</td><td class="patent-data-table-td patent-date-value">Feb 9, 2010</td><td class="patent-data-table-td ">Epicentre Technologies Corporation</td><td class="patent-data-table-td ">Amplifying a target sequence by strand displacement replication; uses primers that have only ribonucleotides or purine ribonucleotides and at least one 2&#39;-substituted pyrimidine-2&#39;-deoxyribonucleotide</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7713691">US7713691</a></td><td class="patent-data-table-td patent-date-value">Nov 27, 2002</td><td class="patent-data-table-td patent-date-value">May 11, 2010</td><td class="patent-data-table-td ">Enzo Life Sciences, Inc.</td><td class="patent-data-table-td ">Modified nucleic acid polymers and methods for their production</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7728119">US7728119</a></td><td class="patent-data-table-td patent-date-value">Jan 17, 2008</td><td class="patent-data-table-td patent-date-value">Jun 1, 2010</td><td class="patent-data-table-td ">Kabushiki Kaisha Toshiba</td><td class="patent-data-table-td ">Nucleotide primer set and nucleotide probe for detecting genotype of methylene tetrahydrofolate reductase (MTHFR)</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7754451">US7754451</a></td><td class="patent-data-table-td patent-date-value">Apr 24, 2007</td><td class="patent-data-table-td patent-date-value">Jul 13, 2010</td><td class="patent-data-table-td ">Affymetrix, Inc.</td><td class="patent-data-table-td ">Multiplex oligonucleotide addition and target amplification</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7803579">US7803579</a></td><td class="patent-data-table-td patent-date-value">Oct 29, 2003</td><td class="patent-data-table-td patent-date-value">Sep 28, 2010</td><td class="patent-data-table-td ">Riken</td><td class="patent-data-table-td ">Strand displacement amplification is used to synthesize a nucleic acid having a sequence complementary to a target sequence by using a primer designed so that it is capable of forming a stem-loop structure by the extension of the primer and satisfies additional specific requirements</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7851150">US7851150</a></td><td class="patent-data-table-td patent-date-value">Dec 18, 2003</td><td class="patent-data-table-td patent-date-value">Dec 14, 2010</td><td class="patent-data-table-td ">Third Wave Technologies, Inc.</td><td class="patent-data-table-td ">Using circular oligonucleotide to identify micro-RNA sequences; RNA interference</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7888014">US7888014</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">May 16, 2005</td><td class="patent-data-table-td patent-date-value">Feb 15, 2011</td><td class="patent-data-table-td ">Nihon University</td><td class="patent-data-table-td ">distinguishing H. influenzae Type b from other capsular serotype and non-encapsulated type H. influenzae rapidly, simply, and accurately</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7906075">US7906075</a></td><td class="patent-data-table-td patent-date-value">Oct 27, 2005</td><td class="patent-data-table-td patent-date-value">Mar 15, 2011</td><td class="patent-data-table-td ">Sysmex Corporation</td><td class="patent-data-table-td ">Pipette tip rack and pipette tip assembly</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7910719">US7910719</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jun 6, 2008</td><td class="patent-data-table-td patent-date-value">Mar 22, 2011</td><td class="patent-data-table-td ">Kabushiki Kaisha Toshiba</td><td class="patent-data-table-td ">Method of detecting human papilloma virus by using nucleic acid amplification method and nucleic acid chain-immobilized carrier</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7919252">US7919252</a></td><td class="patent-data-table-td patent-date-value">Dec 22, 2008</td><td class="patent-data-table-td patent-date-value">Apr 5, 2011</td><td class="patent-data-table-td ">Kabushiki Kaisha Toshiba</td><td class="patent-data-table-td ">Method for detecting a target nucleic acid sequence</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7919611">US7919611</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jan 15, 2008</td><td class="patent-data-table-td patent-date-value">Apr 5, 2011</td><td class="patent-data-table-td ">Kabushiki Kaisha Toshiba</td><td class="patent-data-table-td ">Nucleotide primer set and nucleotide probe for detecting genotype of N-acetyltransferase-2 (NAT2)</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7943347">US7943347</a></td><td class="patent-data-table-td patent-date-value">May 30, 2008</td><td class="patent-data-table-td patent-date-value">May 17, 2011</td><td class="patent-data-table-td ">Hitachi High-Technologies Corporation</td><td class="patent-data-table-td ">Nucleic acid amplification method</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7977055">US7977055</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jun 20, 2007</td><td class="patent-data-table-td patent-date-value">Jul 12, 2011</td><td class="patent-data-table-td ">Nishikawa Rubber Co., Ltd.</td><td class="patent-data-table-td ">Method for amplification of nucleotide sequence</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8017357">US8017357</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Mar 30, 2001</td><td class="patent-data-table-td patent-date-value">Sep 13, 2011</td><td class="patent-data-table-td ">Eiken Kagaku Kabushiki Kaisha</td><td class="patent-data-table-td ">Involves placing a region, to which a primer capable of isothermally amplifying the template nucleic acid will anneal, in a condition that ensures base pairing, using an arbitrary primer</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8076080">US8076080</a></td><td class="patent-data-table-td patent-date-value">May 20, 2003</td><td class="patent-data-table-td patent-date-value">Dec 13, 2011</td><td class="patent-data-table-td ">Sysmex Corporation</td><td class="patent-data-table-td ">Nucleic acid amplification primers for detecting cytokeratins and examination method with the use of the primers</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8078409">US8078409</a></td><td class="patent-data-table-td patent-date-value">Jul 25, 2006</td><td class="patent-data-table-td patent-date-value">Dec 13, 2011</td><td class="patent-data-table-td ">Sysmex Corporation</td><td class="patent-data-table-td ">External quality control method, external quality control computer, preprocessing device, and nucleic acid detecting device</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8133989">US8133989</a></td><td class="patent-data-table-td patent-date-value">Nov 27, 2002</td><td class="patent-data-table-td patent-date-value">Mar 13, 2012</td><td class="patent-data-table-td ">Enzo Diagnostics, Inc.</td><td class="patent-data-table-td ">Nucleic acid primer/construct compositions</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8137911">US8137911</a></td><td class="patent-data-table-td patent-date-value">Nov 21, 2003</td><td class="patent-data-table-td patent-date-value">Mar 20, 2012</td><td class="patent-data-table-td ">Cellscript, Inc.</td><td class="patent-data-table-td ">Using promoter sequences and RNA polymerase to generate nucleotide sequences corresponding to target gene sequences; gene expression inhibition and diagnostic probe generation</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8143006">US8143006</a></td><td class="patent-data-table-td patent-date-value">Aug 4, 2008</td><td class="patent-data-table-td patent-date-value">Mar 27, 2012</td><td class="patent-data-table-td ">Igor Kutyavin</td><td class="patent-data-table-td ">Accelerated cascade amplification (ACA) of nucleic acids comprising strand and sequence specific DNA nicking</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8206902">US8206902</a></td><td class="patent-data-table-td patent-date-value">Dec 24, 2004</td><td class="patent-data-table-td patent-date-value">Jun 26, 2012</td><td class="patent-data-table-td ">Riken</td><td class="patent-data-table-td ">Method of amplifying nucleic acid and method of detecting mutated nucleic acid using the same</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8206904">US8206904</a></td><td class="patent-data-table-td patent-date-value">Jun 1, 2007</td><td class="patent-data-table-td patent-date-value">Jun 26, 2012</td><td class="patent-data-table-td ">Third Wave Technologies, Inc.</td><td class="patent-data-table-td ">Detection of nucleic acids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8252913">US8252913</a></td><td class="patent-data-table-td patent-date-value">Aug 17, 2010</td><td class="patent-data-table-td patent-date-value">Aug 28, 2012</td><td class="patent-data-table-td ">Baxter International Inc.</td><td class="patent-data-table-td ">Aptamers to tissue factor pathway inhibitor and their use as bleeding disorder therapeutics</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8288092">US8288092</a></td><td class="patent-data-table-td patent-date-value">Oct 26, 2007</td><td class="patent-data-table-td patent-date-value">Oct 16, 2012</td><td class="patent-data-table-td ">Enzo Life Sciences, Inc.</td><td class="patent-data-table-td ">Modified nucleic acid polymers and methods for their production</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8334099">US8334099</a></td><td class="patent-data-table-td patent-date-value">Jul 30, 2007</td><td class="patent-data-table-td patent-date-value">Dec 18, 2012</td><td class="patent-data-table-td ">Wanli Bi</td><td class="patent-data-table-td ">Nucleic acid amplification using a reversibly modified oligonucleotide</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8349556">US8349556</a></td><td class="patent-data-table-td patent-date-value">Apr 30, 2007</td><td class="patent-data-table-td patent-date-value">Jan 8, 2013</td><td class="patent-data-table-td ">Igor Kutyavin</td><td class="patent-data-table-td ">Use of base-modified deoxynucleoside triphosphates to improve nucleic acid detection</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8409805">US8409805</a></td><td class="patent-data-table-td patent-date-value">Feb 13, 2009</td><td class="patent-data-table-td patent-date-value">Apr 2, 2013</td><td class="patent-data-table-td ">Asuragen, Inc.</td><td class="patent-data-table-td ">Method of amplification of GC-rich DNA templates</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8435738">US8435738</a></td><td class="patent-data-table-td patent-date-value">Sep 26, 2011</td><td class="patent-data-table-td patent-date-value">May 7, 2013</td><td class="patent-data-table-td ">Theranos, Inc.</td><td class="patent-data-table-td ">Systems and methods for multi-analysis</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8445238">US8445238</a></td><td class="patent-data-table-td patent-date-value">Oct 3, 2007</td><td class="patent-data-table-td patent-date-value">May 21, 2013</td><td class="patent-data-table-td ">Third Wave Technologies, Inc.</td><td class="patent-data-table-td ">Snap-back primers and detectable hairpin structures</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8445664">US8445664</a></td><td class="patent-data-table-td patent-date-value">Oct 20, 2005</td><td class="patent-data-table-td patent-date-value">May 21, 2013</td><td class="patent-data-table-td ">Enzo Diagnostics, Inc.</td><td class="patent-data-table-td ">Apparatus which utilizes strand displacement replication to propagate preferential nucleotide sequences; gene expression analysis</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8461318">US8461318</a></td><td class="patent-data-table-td patent-date-value">Nov 3, 2011</td><td class="patent-data-table-td patent-date-value">Jun 11, 2013</td><td class="patent-data-table-td ">Baxter International Inc.</td><td class="patent-data-table-td ">Aptamers to tissue factor pathway inhibitor and their use as bleeding disorder thereapeutics</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8475739">US8475739</a></td><td class="patent-data-table-td patent-date-value">Sep 26, 2011</td><td class="patent-data-table-td patent-date-value">Jul 2, 2013</td><td class="patent-data-table-td ">Theranos, Inc.</td><td class="patent-data-table-td ">Systems and methods for fluid handling</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8598327">US8598327</a></td><td class="patent-data-table-td patent-date-value">Feb 11, 2011</td><td class="patent-data-table-td patent-date-value">Dec 3, 2013</td><td class="patent-data-table-td ">Baxter International Inc.</td><td class="patent-data-table-td ">Aptamers to tissue factor pathway inhibitor and their use as bleeding disorder therapeutics</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8609336">US8609336</a></td><td class="patent-data-table-td patent-date-value">Feb 7, 2012</td><td class="patent-data-table-td patent-date-value">Dec 17, 2013</td><td class="patent-data-table-td ">Third Wave Technologies, Inc.</td><td class="patent-data-table-td ">Detection of small nucleic acids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8637250">US8637250</a></td><td class="patent-data-table-td patent-date-value">May 13, 2010</td><td class="patent-data-table-td patent-date-value">Jan 28, 2014</td><td class="patent-data-table-td ">Great Basin Scientific</td><td class="patent-data-table-td ">Systems and methods for point-of-care amplification and detection of polynucleotides</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8663557">US8663557</a></td><td class="patent-data-table-td patent-date-value">Jan 19, 2005</td><td class="patent-data-table-td patent-date-value">Mar 4, 2014</td><td class="patent-data-table-td ">Sysmex Corporation</td><td class="patent-data-table-td ">Analyzer</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8679757">US8679757</a></td><td class="patent-data-table-td patent-date-value">Feb 16, 2010</td><td class="patent-data-table-td patent-date-value">Mar 25, 2014</td><td class="patent-data-table-td ">Asuragen, Inc.</td><td class="patent-data-table-td ">PCR methods for characterizing the 5′ untranslated region of the FMR1 and FMR2 genes</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8697377">US8697377</a></td><td class="patent-data-table-td patent-date-value">Jun 12, 2013</td><td class="patent-data-table-td patent-date-value">Apr 15, 2014</td><td class="patent-data-table-td ">Theranos, Inc.</td><td class="patent-data-table-td ">Modular point-of-care devices, systems, and uses thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20100285473">US20100285473</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jan 27, 2010</td><td class="patent-data-table-td patent-date-value">Nov 11, 2010</td><td class="patent-data-table-td ">John Wolff</td><td class="patent-data-table-td ">Thermophilic helicase dependent amplification technology with endpoint homogenous fluorescent detection</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20120141989">US20120141989</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Sep 9, 2011</td><td class="patent-data-table-td patent-date-value">Jun 7, 2012</td><td class="patent-data-table-td ">National Cheng Kung University</td><td class="patent-data-table-td ">Kit and method for rapidly detecting a target nucleic acid fragment</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP1741489A1?cl=en">EP1741489A1</a></td><td class="patent-data-table-td patent-date-value">Aug 20, 2004</td><td class="patent-data-table-td patent-date-value">Jan 10, 2007</td><td class="patent-data-table-td ">Sysmex Corporation</td><td class="patent-data-table-td ">Container for analyzer, detection container, reaction container and packing container for storing detection container</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP1909108A2?cl=en">EP1909108A2</a></td><td class="patent-data-table-td patent-date-value">Mar 10, 2006</td><td class="patent-data-table-td patent-date-value">Apr 9, 2008</td><td class="patent-data-table-td ">Gen-Probe Incorporated</td><td class="patent-data-table-td ">Systems and methods to perform assays for detecting or quantifiying analytes</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP1921454A2?cl=en">EP1921454A2</a></td><td class="patent-data-table-td patent-date-value">Mar 10, 2006</td><td class="patent-data-table-td patent-date-value">May 14, 2008</td><td class="patent-data-table-td ">Gen-Probe Incorporated</td><td class="patent-data-table-td ">Systems and methods to perform assays for detecting or quantifying analytes</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP1930730A2?cl=en">EP1930730A2</a></td><td class="patent-data-table-td patent-date-value">Mar 10, 2006</td><td class="patent-data-table-td patent-date-value">Jun 11, 2008</td><td class="patent-data-table-td ">Gen-Probe Incorporated</td><td class="patent-data-table-td ">Systems and methods to perform assays for detecting or quantifying analytes</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP1944379A1?cl=en">EP1944379A1</a></td><td class="patent-data-table-td patent-date-value">Jan 7, 2008</td><td class="patent-data-table-td patent-date-value">Jul 16, 2008</td><td class="patent-data-table-td ">Sysmex Corporation</td><td class="patent-data-table-td ">Method of judging lymph node metastasis of stomach cancer using cytokeratin 19</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP1964934A1?cl=en">EP1964934A1</a></td><td class="patent-data-table-td patent-date-value">Feb 28, 2008</td><td class="patent-data-table-td patent-date-value">Sep 3, 2008</td><td class="patent-data-table-td ">Sysmex Corporation</td><td class="patent-data-table-td ">Sample treatment solution and reagent kit for preparing sample for detecting DNA methylation</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2017356A2?cl=en">EP2017356A2</a></td><td class="patent-data-table-td patent-date-value">Jun 6, 2007</td><td class="patent-data-table-td patent-date-value">Jan 21, 2009</td><td class="patent-data-table-td ">Gen-Probe Incorporated</td><td class="patent-data-table-td ">Tagged oliggonucleotides and their use in nucleic acid amplification methods</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2191897A1?cl=en">EP2191897A1</a></td><td class="patent-data-table-td patent-date-value">Jun 20, 2008</td><td class="patent-data-table-td patent-date-value">Jun 2, 2010</td><td class="patent-data-table-td ">Gen-Probe Incorporated</td><td class="patent-data-table-td ">Instrument and receptacles for performing processes</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2286845A1?cl=en">EP2286845A1</a></td><td class="patent-data-table-td patent-date-value">Mar 4, 2005</td><td class="patent-data-table-td patent-date-value">Feb 23, 2011</td><td class="patent-data-table-td ">Gen-Probe Incorporated</td><td class="patent-data-table-td ">Reagents methodsand kits for use in deactivating nucleic acids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2322940A1?cl=en">EP2322940A1</a></td><td class="patent-data-table-td patent-date-value">Mar 10, 2006</td><td class="patent-data-table-td patent-date-value">May 18, 2011</td><td class="patent-data-table-td ">Gen-Probe Incorporated</td><td class="patent-data-table-td ">Systems amd methods to perform assays for detecting or quantifing analytes within samples</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2333561A2?cl=en">EP2333561A2</a></td><td class="patent-data-table-td patent-date-value">Mar 10, 2006</td><td class="patent-data-table-td patent-date-value">Jun 15, 2011</td><td class="patent-data-table-td ">Gen-Probe Incorporated</td><td class="patent-data-table-td ">System for performing multi-formatted assays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2345740A1?cl=en">EP2345740A1</a></td><td class="patent-data-table-td patent-date-value">Jun 6, 2007</td><td class="patent-data-table-td patent-date-value">Jul 20, 2011</td><td class="patent-data-table-td ">Gen-Probe Incorporated</td><td class="patent-data-table-td ">Tagged oligonucleotides and their use in nucleic acid amplification methods</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2348320A2?cl=en">EP2348320A2</a></td><td class="patent-data-table-td patent-date-value">Mar 10, 2006</td><td class="patent-data-table-td patent-date-value">Jul 27, 2011</td><td class="patent-data-table-td ">Gen-Probe Incorporated</td><td class="patent-data-table-td ">Systems and methods for detecting multiple optical signals</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2348321A2?cl=en">EP2348321A2</a></td><td class="patent-data-table-td patent-date-value">Mar 10, 2006</td><td class="patent-data-table-td patent-date-value">Jul 27, 2011</td><td class="patent-data-table-td ">Gen-Probe Incorporated</td><td class="patent-data-table-td ">System and methods to perform assays for detecting or quantifying analytes within samples</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2397564A1?cl=en">EP2397564A1</a></td><td class="patent-data-table-td patent-date-value">Jun 14, 2011</td><td class="patent-data-table-td patent-date-value">Dec 21, 2011</td><td class="patent-data-table-td ">Sysmex Corporation</td><td class="patent-data-table-td ">Method of determining lymph node metastasis in a lung cancer, apparatus for determining lymph node metastasis in a lung cancer, and computer program product</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2425894A1?cl=en">EP2425894A1</a></td><td class="patent-data-table-td patent-date-value">Jun 20, 2008</td><td class="patent-data-table-td patent-date-value">Mar 7, 2012</td><td class="patent-data-table-td ">Gen-Probe Incorporated</td><td class="patent-data-table-td ">Instruments and method for exposing a receptacle to multiple thermal zones</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2465609A1?cl=en">EP2465609A1</a></td><td class="patent-data-table-td patent-date-value">Jun 20, 2008</td><td class="patent-data-table-td patent-date-value">Jun 20, 2012</td><td class="patent-data-table-td ">Gen-Probe Incorporated</td><td class="patent-data-table-td ">Instruments and method for mixing the contents of a detection chamber</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2479287A1?cl=en">EP2479287A1</a></td><td class="patent-data-table-td patent-date-value">May 13, 2009</td><td class="patent-data-table-td patent-date-value">Jul 25, 2012</td><td class="patent-data-table-td ">Gen-Probe Incorporated</td><td class="patent-data-table-td ">Inactivatable target capture oligomers for use in the selective hybridization and capture of target nucleic acid sequences</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2508529A1?cl=en">EP2508529A1</a></td><td class="patent-data-table-td patent-date-value">Oct 24, 2009</td><td class="patent-data-table-td patent-date-value">Oct 10, 2012</td><td class="patent-data-table-td ">Epicentre Technologies Corporation</td><td class="patent-data-table-td ">Transposon end compositions and methods for modifying nucleic acids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2664678A1?cl=en">EP2664678A1</a></td><td class="patent-data-table-td patent-date-value">Oct 24, 2009</td><td class="patent-data-table-td patent-date-value">Nov 20, 2013</td><td class="patent-data-table-td ">Epicentre Technologies Corporation</td><td class="patent-data-table-td ">Transposon end compositions and methods for modifying nucleic acids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2738264A1?cl=en">EP2738264A1</a></td><td class="patent-data-table-td patent-date-value">Nov 22, 2013</td><td class="patent-data-table-td patent-date-value">Jun 4, 2014</td><td class="patent-data-table-td ">Sysmex Corporation</td><td class="patent-data-table-td ">A method and system for determining behavior of thyroid tumor</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2004013354A1?cl=en">WO2004013354A1</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Aug 6, 2003</td><td class="patent-data-table-td patent-date-value">Feb 12, 2004</td><td class="patent-data-table-td ">Douglas John Oultram</td><td class="patent-data-table-td ">Amplification of nucleic acids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2004016755A2?cl=en">WO2004016755A2</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Aug 14, 2003</td><td class="patent-data-table-td patent-date-value">Feb 26, 2004</td><td class="patent-data-table-td ">Donald M Crothers</td><td class="patent-data-table-td ">Amplification of target nucleotide sequence without polymerase chain reaction</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2004048596A2?cl=en">WO2004048596A2</a></td><td class="patent-data-table-td patent-date-value">Nov 21, 2003</td><td class="patent-data-table-td patent-date-value">Jun 10, 2004</td><td class="patent-data-table-td ">Gary A Dahl</td><td class="patent-data-table-td ">Methods for using primers that encode one strand of a double-stranded promoter</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2005087951A2?cl=en">WO2005087951A2</a></td><td class="patent-data-table-td patent-date-value">Mar 4, 2005</td><td class="patent-data-table-td patent-date-value">Sep 22, 2005</td><td class="patent-data-table-td ">Gen Probe Inc</td><td class="patent-data-table-td ">Reagents, methods and kits for use in deactivating nucleic acids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2008021290A2?cl=en">WO2008021290A2</a></td><td class="patent-data-table-td patent-date-value">Aug 9, 2007</td><td class="patent-data-table-td patent-date-value">Feb 21, 2008</td><td class="patent-data-table-td ">Homestead Clinical Corp</td><td class="patent-data-table-td ">Organ-specific proteins and methods of their use</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2008089286A1?cl=en">WO2008089286A1</a></td><td class="patent-data-table-td patent-date-value">Jan 16, 2008</td><td class="patent-data-table-td patent-date-value">Jul 24, 2008</td><td class="patent-data-table-td ">Meridian Bioscience Inc</td><td class="patent-data-table-td ">Stable reagents and kits useful in loop-mediated isothermal amplification (lamp)</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2009119331A1?cl=en">WO2009119331A1</a></td><td class="patent-data-table-td patent-date-value">Mar 12, 2009</td><td class="patent-data-table-td patent-date-value">Oct 1, 2009</td><td class="patent-data-table-td ">Sysmex Corporation</td><td class="patent-data-table-td ">REAGENT COMPRISING PRIMER FOR DETECTION OF mRNA FOR CYTOKERATIN-7</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2009145303A1?cl=en">WO2009145303A1</a></td><td class="patent-data-table-td patent-date-value">May 29, 2009</td><td class="patent-data-table-td patent-date-value">Dec 3, 2009</td><td class="patent-data-table-td ">Sysmex Corporation</td><td class="patent-data-table-td ">PRIMER FOR DETECTION OF TYROSINASE mRNA</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2010088496A1?cl=en">WO2010088496A1</a></td><td class="patent-data-table-td patent-date-value">Jan 29, 2010</td><td class="patent-data-table-td patent-date-value">Aug 5, 2010</td><td class="patent-data-table-td ">Gen-Probe Incorporated</td><td class="patent-data-table-td ">Systems and methods for detecting a signal and applying thermal energy to a signal transmission element</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2010093552A1?cl=en">WO2010093552A1</a></td><td class="patent-data-table-td patent-date-value">Feb 4, 2010</td><td class="patent-data-table-td patent-date-value">Aug 19, 2010</td><td class="patent-data-table-td ">Asuragen, Inc.</td><td class="patent-data-table-td ">Method of amplification of gc-rich dna templates</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2011022427A1?cl=en">WO2011022427A1</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Aug 17, 2010</td><td class="patent-data-table-td patent-date-value">Feb 24, 2011</td><td class="patent-data-table-td ">Archemix Corp.</td><td class="patent-data-table-td ">Aptamers to tissue factor pathway inhibitor and their use as bleeding disorder therapeutics</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2011026139A1?cl=en">WO2011026139A1</a></td><td class="patent-data-table-td patent-date-value">Aug 31, 2010</td><td class="patent-data-table-td patent-date-value">Mar 3, 2011</td><td class="patent-data-table-td ">Gen-Probe Incorporated</td><td class="patent-data-table-td ">Dengue virus assay</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2012003289A1?cl=en">WO2012003289A1</a></td><td class="patent-data-table-td patent-date-value">Jun 30, 2011</td><td class="patent-data-table-td patent-date-value">Jan 5, 2012</td><td class="patent-data-table-td ">Gen-Probe Incorporated</td><td class="patent-data-table-td ">Method and apparatus for identifying analyte-containing samples using single-read determination of analyte and process control signals</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2012061283A1?cl=en">WO2012061283A1</a></td><td class="patent-data-table-td patent-date-value">Oct 31, 2011</td><td class="patent-data-table-td patent-date-value">May 10, 2012</td><td class="patent-data-table-td ">The United States Of America, As Represented By The Secretary, Department Of Health And Human Services</td><td class="patent-data-table-td ">Rapid molecular assays for specific detection and quantitation of loa loa microfilaremia</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2012142516A1?cl=en">WO2012142516A1</a></td><td class="patent-data-table-td patent-date-value">Apr 13, 2012</td><td class="patent-data-table-td patent-date-value">Oct 18, 2012</td><td class="patent-data-table-td ">Becton, Dickinson And Company</td><td class="patent-data-table-td ">Scanning real-time microfluidic thermo-cycler and methods for synchronized thermocycling and scanning optical detection</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2013044097A1?cl=en">WO2013044097A1</a></td><td class="patent-data-table-td patent-date-value">Sep 21, 2012</td><td class="patent-data-table-td patent-date-value">Mar 28, 2013</td><td class="patent-data-table-td ">Gen-Probe Incorporated</td><td class="patent-data-table-td ">Methods for amplifying nucleic acid using tag-mediated displacement</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2013154734A1?cl=en">WO2013154734A1</a></td><td class="patent-data-table-td patent-date-value">Mar 13, 2013</td><td class="patent-data-table-td patent-date-value">Oct 17, 2013</td><td class="patent-data-table-td ">Becton, Dickinson And Company, Inc.</td><td class="patent-data-table-td ">Reflex testing of samples using residual materials from a prior test</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="classifications"></a><div class="patent-section-header"><span class="patent-section-title">Classifications</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th"> </th><th class="patent-data-table-th"> </th></tr></thead><tr><td class="patent-data-table-td ">U.S. Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=c-BaBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S091200">435/91.2</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=c-BaBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S023100">536/23.1</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=c-BaBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S024330">536/24.33</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=c-BaBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S024300">536/24.3</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=c-BaBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S006100">435/6.1</a></span></td></tr><tr><td class="patent-data-table-td ">International Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=c-BaBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12Q0001680000">C12Q1/68</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=c-BaBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0015100000">C12N15/10</a></span></td></tr><tr><td class="patent-data-table-td ">Cooperative Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=c-BaBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12Q2527/125">C12Q2527/125</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=c-BaBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12Q1/6844">C12Q1/6844</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=c-BaBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12Q2527/101">C12Q2527/101</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=c-BaBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12Q1/6853">C12Q1/6853</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=c-BaBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12Q2531/119">C12Q2531/119</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=c-BaBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N15/10">C12N15/10</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=c-BaBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12Q2525/161">C12Q2525/161</a></span></td></tr><tr><td class="patent-data-table-td ">European Classification</td><td class="patent-data-table-td "><span class="nested-value">C12Q1/68D2E</span>, <span class="nested-value">C12Q1/68D</span>, <span class="nested-value">C12N15/10</span></td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="legal-events"></a><div class="patent-section-header"><span class="patent-section-title">Legal Events</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Date</th><th class="patent-data-table-th">Code</th><th class="patent-data-table-th">Event</th><th class="patent-data-table-th">Description</th></tr></thead><tr><td class="patent-data-table-td patent-date-value">Dec 19, 2013</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">12</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">May 22, 2012</td><td class="patent-data-table-td ">B1</td><td class="patent-data-table-td ">Reexamination certificate first reexamination</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">THE PATENTABILITY OF CLAIMS 1-13 AND 15-19 IS CONFIRMED. CLAIM 14 IS DETERMINED TO BE PATENTABLE ASAMENDED. NEW CLAIMS 20-26 ARE ADDED AND DETERMINED TO BE PATENTABLE.</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Dec 29, 2009</td><td class="patent-data-table-td ">RR</td><td class="patent-data-table-td ">Request for reexamination filed</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20091001</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Dec 21, 2009</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">8</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Apr 7, 2009</td><td class="patent-data-table-td ">RR</td><td class="patent-data-table-td ">Request for reexamination filed</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20090211</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Jan 5, 2009</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">EIKEN KAGAKU KABUSHIKI KAISHA, JAPAN</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">CORRECTIVE ASSIGNMENT TO CORRECT THE ASSIGNEE ADDRESS PREVIOUSLY RECORDED ON  REEL 011078 FRAME 0411;ASSIGNORS:NOTOMI, TSUGUNORI;HASE, TETSU;REEL/FRAME:022058/0116</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20000615</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Dec 16, 2005</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">4</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Aug 12, 2003</td><td class="patent-data-table-td ">CC</td><td class="patent-data-table-td ">Certificate of correction</td><td class="patent-data-table-td "></td></tr><tr><td class="patent-data-table-td patent-date-value">Sep 6, 2000</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">EIKEN KAGAKU KABUSHIKI KAISHA, JAPAN</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNORS:NOTOMI, TSUGUNORI;HASE, TETSU;REEL/FRAME:011078/0411</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20000615</span></div><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">EIKEN KAGAKU KABUSHIKI KAISHA 33-8, HONGO 1-CHOME</span></div></td></tr></table><div class="patent-section-footer"></div></div><div class="modal-dialog" id="patent-images-lightbox"><div class="patent-lightbox-controls"><div class="patent-lightbox-rotate-controls"><div class="patent-lightbox-rotation-text">Rotate</div><div class="rotate-icon rotate-ccw-icon"></div><div class="rotate-icon rotate-cw-icon"></div></div><div class="patent-lightbox-index-counter"></div><a class="patent-lightbox-fullsize-link" target="_blank">Original Image</a><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_right.png" alt="Next page"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_left.png" alt="Previous page"width="21" height="21" /></div></div><div class="modal-dialog-content"><div class="patent-lightbox-image-holder"><div class="patent-lightbox-placeholder"></div></div></div></div><script>_OC_initPatentsAtb({image_not_available_html: " Image not available"});</script></div></div></div></td></tr></table><script>(function() {var href = window.location.href;if (href.indexOf('?') !== -1) {var parameters = href.split('?')[1].split('&');for (var i = 0; i < parameters.length; i++) {var param = parameters[i].split('=');if (param[0] == 'focus') {var elem = document.getElementById(param[1]);if (elem) {elem.focus();}}}}})();</script><script>_OC_addFlags({LockSrc:"/books/javascript/lock_8a2b04e7bf975d5171d8e4c0b6365c7a.js", Host:"http://www.google.com/", IsBooksRentalEnabled:1, IsWebstoreDisplayCaseEnabled:1, IsObfuscationEnabled:1, IsBrowsingHistoryEnabled:1, IsWebReaderSvgEnabled:0, IsGeoLayerEnabled:1, IsImageModeNotesEnabled:1, IsCopyMenuItemEnabled:1, IsGiftingEnabled:0, IsWebReaderUniversalPaginatorEnabled:0, IsOfflineBubbleEnabled:1, IsReaderEnabledForPlayRequests:1, IsFutureOnSaleVolumesEnabled:1, IsOfflineRestrictedCopyEnabled:1, IsBooksUnifiedLeftNavEnabled:1, IsRestrictedCopyEnabled:1, IsZipitFolderCollectionEnabled:1, IsEndOfSampleRecommendationsEnabled:1, IsRatingsOnBookcardsEnabled:1, IsAdsDisabled:0, IsIframePageDisplayEnabled:0, IsEmbeddedMediaEnabled:1, IsImageModeAnnotationsEnabled:1, IsMyLibraryGooglePlusEnabled:1, IsImagePageProviderEnabled:0, IsBookcardListPriceSmall:0, IsInternalUser:0, IsBooksShareButtonEnabled:0, IsPreOrdersEnabled:0, IsDisabledRandomBookshelves:0, WebstoreDisplayCasePosition:3});_OC_Run({"enable_p13n":false,"add_vol_to_collection_base_url":"http://www.google.com/patents?op=add\u0026sig=ACfU3U2WT9oI66kLYr1QpVTWbwq49RbS7A\u0026id=c-BaBAABERAJ","remove_vol_from_collection_base_url":"http://www.google.com/patents?op=remove\u0026sig=ACfU3U2qmftkoOnj-GGEfzqeb_6pal_bpg\u0026id=c-BaBAABERAJ","logged_in":false,"p13n_save_user_settings_url":"http://www.google.com/patents?op=edit_user_settings\u0026sig=ACfU3U0MPFfLKOu3hhl8Qr3yCuYUWeWIQg","is_cobrand":false,"sign_in_url":"https://www.google.com/accounts/Login?service=\u0026continue=http://www.google.com/patents%3Fhl%3Den\u0026hl=en","is_play_enabled":true}, {"volume_id":"","is_ebook":true,"volumeresult":{"has_flowing_text":false,"has_scanned_text":true,"can_download_pdf":false,"can_download_epub":false,"is_pdf_drm_enabled":false,"is_epub_drm_enabled":false,"download_pdf_url":"http://www.google.com/patents/download/Process_for_synthesizing_nucleic_acid.pdf?id=c-BaBAABERAJ\u0026output=pdf\u0026sig=ACfU3U0bUCOD2ctSHz1TnIJDfUOisasV5w"},"sample_url":"http://www.google.com/patents/reader?id=c-BaBAABERAJ\u0026printsec=frontcover\u0026output=reader\u0026source=gbs_atb_hover","is_browsable":true,"is_public_domain":true}, {});</script><div id="footer_table" style="font-size:83%;text-align:center;position:relative;top:20px;height:4.5em;margin-top:2em"><div style="margin-bottom:8px"><a href=http://www.google.com/><nobr>Google&nbsp;Home</nobr></a> - <a href=//www.google.com/patents/sitemap/><nobr>Sitemap</nobr></a> - <a href=http://www.google.com/googlebooks/uspto.html><nobr>USPTO Bulk Downloads</nobr></a> - <a href=/intl/en/privacy/><nobr>Privacy Policy</nobr></a> - <a href=/intl/en/policies/terms/><nobr>Terms of Service</nobr></a> - <a href=https://support.google.com/faqs/answer/2539193?hl=en><nobr>About Google Patents</nobr></a> - <a href="http://www.google.com/tools/feedback/intl/en/error.html" onclick="try{_OC_startFeedback({productId: '72792',locale: 'en'});return false;}catch(e){}"><nobr>Send Feedback</nobr></a></div><span>Data provided by IFI CLAIMS Patent Services</span><br><span >&copy;2012 Google</span></div> <script type="text/javascript">var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));</script><script type="text/javascript">var pageTracker = _gat._getTracker("UA-27188110-1");pageTracker._setCookiePath("/patents/");pageTracker._trackPageview();</script> </body></html>