<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN""http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html><head><title>Patent US6410516 - Nuclear factors associated with transcriptional regulation - Google Patents</title><script>(function(){(function(){function e(a){this.t={};this.tick=function(a,c,b){var d=void 0!=b?b:(new Date).getTime();this.t[a]=[d,c];if(void 0==b)try{window.console.timeStamp("CSI/"+a)}catch(e){}};this.tick("start",null,a)}var a;window.performance&&(a=window.performance.timing);var f=a?new e(a.responseStart):new e;window.jstiming={Timer:e,load:f};if(a){var c=a.navigationStart,d=a.responseStart;0<c&&d>=c&&(window.jstiming.srt=d-c)}if(a){var b=window.jstiming.load;0<c&&d>=c&&(b.tick("_wtsrt",void 0,c),b.tick("wtsrt_",
"_wtsrt",d),b.tick("tbsd_","wtsrt_"))}try{a=null,window.chrome&&window.chrome.csi&&(a=Math.floor(window.chrome.csi().pageT),b&&0<c&&(b.tick("_tbnd",void 0,window.chrome.csi().startE),b.tick("tbnd_","_tbnd",c))),null==a&&window.gtbExternal&&(a=window.gtbExternal.pageT()),null==a&&window.external&&(a=window.external.pageT,b&&0<c&&(b.tick("_tbnd",void 0,window.external.startE),b.tick("tbnd_","_tbnd",c))),a&&(window.jstiming.pt=a)}catch(g){}})();})();
</script><link rel="stylesheet" href="/patents/css/_8a2b04e7bf975d5171d8e4c0b6365c7a/kl_intl_patents_bundle.css" type="text/css" /><script src="/books/javascript/atb_8a2b04e7bf975d5171d8e4c0b6365c7a__en.js"></script><script>function googleTranslateElementInit() {new google.translate.TranslateElement({pageLanguage: "en",gaTrack: true,gaId: "UA-27188110-1",multilanguagePage: true});}</script><script src="//translate.google.com/translate_a/element.js?cb=googleTranslateElementInit"></script><meta name="DC.type" content="Patent"><meta name="DC.title" content="Nuclear factors associated with transcriptional regulation"><meta name="DC.contributor" content="David Baltimore" scheme="inventor"><meta name="DC.contributor" content="Ranjan Sen" scheme="inventor"><meta name="DC.contributor" content="Phillip A. Sharp" scheme="inventor"><meta name="DC.contributor" content="Harinder Singh" scheme="inventor"><meta name="DC.contributor" content="Louis Staudt" scheme="inventor"><meta name="DC.contributor" content="Jonathan H. LeBowitz" scheme="inventor"><meta name="DC.contributor" content="Albert S. Baldwin, Jr." scheme="inventor"><meta name="DC.contributor" content="Roger G. Clerc" scheme="inventor"><meta name="DC.contributor" content="Lynn M. Corcoran" scheme="inventor"><meta name="DC.contributor" content="Patrick A. Baeuerle" scheme="inventor"><meta name="DC.contributor" content="Michael J. Lenardo" scheme="inventor"><meta name="DC.contributor" content="Chen-Ming Fan" scheme="inventor"><meta name="DC.contributor" content="Thomas P. Maniatis" scheme="inventor"><meta name="DC.contributor" content="President &amp; Fellows Of Harvard College" scheme="assignee"><meta name="DC.contributor" content="Massachusetts Institute Of Technology" scheme="assignee"><meta name="DC.contributor" content="Whitehead Instittue For Biomedical Research" scheme="assignee"><meta name="DC.date" content="1995-6-5" scheme="dateSubmitted"><meta name="DC.description" content="Constitutive and tissue-specific protein factors which bind to transcriptional regulatory elements of Ig genes (promoter and enhancer) are described. The factors were identified and isolated by an improved assay for protein-DNA binding. Genes encoding factors which positively regulate transcription can be isolated and employed to enhance transription of Ig genes. In particular, NF-kB, the gene encoding NF-kB, IkB and the gene encoding IkB and uses therefor."><meta name="DC.date" content="2002-6-25" scheme="issued"><meta name="DC.relation" content="WO:1987004170:A1" scheme="references"><meta name="citation_reference" content="Baeuerle, P.A and Baltimore, D., &quot;IkappaB: A Specific Inhibitor of the NF-kappaB Transcription Factor,&quot; Science, 242:540-546 (1988)."><meta name="citation_reference" content="Baeuerle, P.A and Baltimore, D., “IκB: A Specific Inhibitor of the NF-κB Transcription Factor,” Science, 242:540-546 (1988)."><meta name="citation_reference" content="Baeuerle, P.A. and Baltimore, D., &quot;Activation of DNA-Binding Activity in an Apparently Cytoplasmic Precursor of the NF-kappaB Transcription Factor,&quot; Cell, 53:211-217 (1988)."><meta name="citation_reference" content="Baeuerle, P.A. and Baltimore, D., “Activation of DNA-Binding Activity in an Apparently Cytoplasmic Precursor of the NF-κB Transcription Factor,” Cell, 53:211-217 (1988)."><meta name="citation_reference" content="Baeurle, P.A. and Baltimore, D., &quot;Activation of NF-kappaB: A Transcription Factor Controlling Expression of the Immunoglobuli kappa Light-chain Gene and of HIV,&quot; The Control of Human Retrovirus Gene Expression, Banbury Conference, Cold Spring Harbor, NY, pp.: 217-226 (1988)."><meta name="citation_reference" content="Baeurle, P.A. and Baltimore, D., “Activation of NF-κB: A Transcription Factor Controlling Expression of the Immunoglobuli κ Light-chain Gene and of HIV,” The Control of Human Retrovirus Gene Expression, Banbury Conference, Cold Spring Harbor, NY, pp.: 217-226 (1988)."><meta name="citation_reference" content="Baldwin, A. and Sharp, P., et al., &quot;Binding of a Nuclear Factor to a Regulatory Sequence in the Promoter of the Mouse H-2Kb Class I Major Histocompatibility Gene,&quot; Mol. &amp; Cell. Biol., 7(1):305-313 (1987)."><meta name="citation_reference" content="Baldwin, Jr., A.S., and Sharp, P.A., &quot;Two transcription factors, NF-kB and H2TF1, interact with a single regulatory sequence in the class I major histocompatability complex promotor,&quot; Proc. Natl. Acad. Sci, USA, 85:723-727 (1988)."><meta name="citation_reference" content="Ballard, D.W., et al., &quot;HTLV-I Tax Induces Cellular Proteins That Activate the kB Element in the IL-2 Receptor a Gene,&quot; Science, 241:1652-1657 (1988)."><meta name="citation_reference" content="Banerji, J., et al., &quot;A Lymphocyte-Specific Cellular Enhancer Is Located Downstream of the Joining Region in Immunoglobulin Heavy Chain Genes,&quot; Cell, 33:729-740 (1983)."><meta name="citation_reference" content="Bergman, Y., et al., &quot;Two regulatory elements for immunoglobulin kappa light chain gene expression,&quot; Proc. Natl. Acad. Sci. USA, 81:7041-7045 (1984)."><meta name="citation_reference" content="Bergman, Y., et al., “Two regulatory elements for immunoglobulin κ light chain gene expression,” Proc. Natl. Acad. Sci. USA, 81:7041-7045 (1984)."><meta name="citation_reference" content="Blanar, M.A., et al., &quot;Nf-kB Binds within a Region Required for B-Cell-Specific Expression of the Major Histocompatibility Complex Class II Gene Ead,&quot; Mol. &amp; Cell. Biol., 9 (2):844-846."><meta name="citation_reference" content="Böhnlein, E.,, et al., &quot;The Same Inducible Nuclear Proteins Regulates Mitogen Activation of Both the Interleukin-2 Receptor-Alpha Gene and Type 1 HIV,&quot; Cell, 53:827-836 (1988)."><meta name="citation_reference" content="Church, G. M., et al., &quot;Cell-type-specific contacts to immunoglobulin enhancers in nuclei,&quot; Nature, 313:798-801 (1985)."><meta name="citation_reference" content="Cross, S. L., et al., &quot;Functionally Distinct NF-kappaB Binding Sites in the Immunoglobulin kappa and IL-2 Receptor alpha Chain Genes,&quot; Science, 244:466-469 (1989)."><meta name="citation_reference" content="Cross, S. L., et al., “Functionally Distinct NF-κB Binding Sites in the Immunoglobulin κ and IL-2 Receptor α Chain Genes,” Science, 244:466-469 (1989)."><meta name="citation_reference" content="Davis, N., et al., &quot;Rel-Associated pp40: An Inhibitor of the Rel Family of Transcription Factors,&quot; Science 253:1268-1271 (1991)."><meta name="citation_reference" content="Fletcher, C., et al., &quot;Purification and Characterizaiton of OTF-1, a Transcription Factor Regulating Cell Cycle Expression of a Human Histone H2b Gene,&quot; Cell, 51:773-781 (1987)."><meta name="citation_reference" content="Foster, J., et al., &quot;An immunoglobulin promoter displays cell-type specificity independently of the enhancer,&quot; Nature, 315:423-425 (1985)."><meta name="citation_reference" content="Fried, M. and Crothers, D. M., &quot;Equilibria and kinetics of lac repressor-operator interactions by polyacrylamide gel electrophoresis,&quot; Nucleic Acids Research, 9(23):6505-6524 (1981)."><meta name="citation_reference" content="Garner, M. M. and Revzin, A., &quot;A gel electrophoresis method for quantifying the binding of proteins to specific DNA regions: application to components of the Escherichia coli lactose operon regulatory system,&quot; Nucleic Acids Research, 9(13):3047-3060 (1981)."><meta name="citation_reference" content="Gerster, T., et al., &quot;Cell type-specificity elements of the immunoglobulin heavy chain gene enhancer,&quot; EMBO Journal, 6(5):1323-1330 (1987)."><meta name="citation_reference" content="Goodbourn, S., et al., &quot;Human beta-Interferon Gene Expression Is Regulated by an Inducible Enhancer Element,&quot; Cell, 41:509-520 (1985)."><meta name="citation_reference" content="Goodbourn, S., et al., “Human β-Interferon Gene Expression Is Regulated by an Inducible Enhancer Element,” Cell, 41:509-520 (1985)."><meta name="citation_reference" content="Gosh, S. and Baltimore, D., &quot;Activation in vitro of NF-kB by phosphorylation of its inhibitor IkB,&quot; Nature, 344(6267): 678-682 (1990)."><meta name="citation_reference" content="Grosschedi, R. and Baltimore, D., &quot;Cell-Type Specificity of Immunoglobulin Gene Expression is Regulated by at Least Three DNA Sequence Elements,&quot; Cell, 41:885-897 (1985)."><meta name="citation_reference" content="Haskill, S., et al., &quot;Characterization of an Immediate-Early Gene Induced in Adherent Monocytes That Encodes IkB-like Activity,&quot; Cell, 65:1281-1289 (1991)."><meta name="citation_reference" content="Johnston et al. (1993) Present Status and future prospects for HIV therapies. Science 260:1286-1293."><meta name="citation_reference" content="Karin, M., et al., &quot;Activation of a Heterologous Promoter in Response to Dexamethasone and Cadmium by Metallothionein Gene 5&#39;Flanking DNA,&quot; Cell, 36:371-379 (1984)."><meta name="citation_reference" content="Karin, M., et al., “Activation of a Heterologous Promoter in Response to Dexamethasone and Cadmium by Metallothionein Gene 5′Flanking DNA,” Cell, 36:371-379 (1984)."><meta name="citation_reference" content="Kawakami, K., et al., &quot;Identification and purification of a human immunoglobulin-enhancer-binding protein (NF-kappaB) that activates transcription from a human immunodeficiency virus type 1 promoter in vitro,&quot; Proc. Natl. Acad. Sci. USA, 85:4700-4704 (1988)."><meta name="citation_reference" content="Kawakami, K., et al., “Identification and purification of a human immunoglobulin-enhancer-binding protein (NF-κB) that activates transcription from a human immunodeficiency virus type 1 promoter in vitro,” Proc. Natl. Acad. Sci. USA, 85:4700-4704 (1988)."><meta name="citation_reference" content="Ko, H.-S., et al., &quot;A Human Protein Specific for the Immunoglobulin Octamer DNA Motif Contains a Functional Homeobox Domain,&quot; Cell, 55:135-144 (1988)."><meta name="citation_reference" content="Landolfi, N. F., et al., &quot;Interaction of cell-type-specific nuclear proteins with immunoglobulin VH promoter region sequences,&quot; Nature, 323:548-551 (1986)."><meta name="citation_reference" content="Lenardo, J.J., et al., &quot;NF-kB protein purification from bovine spleen: Nucleotide stimulation and binding site specificity,&quot; Prod. Natl. Acad. Sci. USA, 85:8825-8829 (1988)."><meta name="citation_reference" content="Lenardo, M., et al., &quot;Protein-Binding Sites in Ig Gene Enhancers Determine Transcriptional Activity and Inducibility,&quot; Science, 236:1573-1577 (1987)."><meta name="citation_reference" content="Leonard et al. (1985) Interleukin 2 receptor gene expression in normal human T lymphocytes. Proc. Natl. Acad. Sci. USA 82:6281-6285.*"><meta name="citation_reference" content="Leung, K. and Nable, G.J., &quot;HTLV-1 transactivator induces interleukin-2 receptor expression through an NF-kB-like factor,&quot; Nature, 333:776-778 (1988)."><meta name="citation_reference" content="Mason, J. O., et al., &quot;Transcription Cell Type Specificity Is Conferred by an Immunoglobulin VH Gene Promoter That Includes a Functional Consensus Sequence,&quot; Cell, 41:479-487 (1985)."><meta name="citation_reference" content="Mercola, M., et al., &quot;Immunoglobulin Heavy-Chain Enhancer Requires One or More Tissue-Specific Factors,&quot; Science, 227:266-270 (1985)."><meta name="citation_reference" content="Mercola, M., et al., &quot;Transcriptional Enhancer Elements in the Mouse Immunoglobulin Heavy Chain Locus,&quot; Science, 221:663-665 (1983)."><meta name="citation_reference" content="Nabel, G. and Baltimore, D., An inducible transcription factor activates expression of human immunodeficiency virus in T cells, Nature, 326:711-713 (1987)."><meta name="citation_reference" content="Nelsen, B., et al., &quot;The NF-kB-Binding Site Mediates Phorbol Ester-Inducible Transcription in Nonlymphoid Cells,&quot; Mol. &amp; Cell Biol., 8:3526-3531 (1988)."><meta name="citation_reference" content="Nelson, K. J., et al., &quot;Inducible transcription of the unrearranged kappa constant region locus is a common feature of pre-B cells and does not require DNA or protein synthesis,&quot; Proc. Natl. Acad. Sci. USA, 82:5305-5309 (1985)."><meta name="citation_reference" content="Nelson, K. J., et al., “Inducible transcription of the unrearranged κ constant region locus is a common feature of pre-B cells and does not require DNA or protein synthesis,” Proc. Natl. Acad. Sci. USA, 82:5305-5309 (1985)."><meta name="citation_reference" content="Picard, D. and Schaffner, W., &quot;A lymphocyte-specific enhancer in the mouse immunoglobulin kappa gene,&quot; Nature, 307:80-82 (1984)."><meta name="citation_reference" content="Picard, D. and Schaffner, W., “A lymphocyte-specific enhancer in the mouse immunoglobulin κ gene,” Nature, 307:80-82 (1984)."><meta name="citation_reference" content="Queen, C. and Baltimore, D., &quot;Immunoglobulin Gene Transcription Is Activated by Downstream Sequence Elements,&quot; Cell, 33:741-748 (1983)."><meta name="citation_reference" content="Queen, C. and Stafford, J., &quot;Fine Mapping of an Immunoglobulin Gene Activator,&quot; Mol. Cel. Biol., 4(6):1042-1049 (1984)."><meta name="citation_reference" content="Ruben, S., et al., &quot;Cellular Transcription Factors and Regulation of IL-2 Receptor Gene Expression by HTLV-1 tax Gene Product,&quot; Science, 241:89-92 (1988)."><meta name="citation_reference" content="Sassone-Corsi, P., et al., &quot;A trans-acting factor is responsible for the simian virus 40 enhancer activity in vitro,&quot; Nature, 313:458-463 (1985)."><meta name="citation_reference" content="Scheidereit, C., et al., &quot;Identification and Purification of a Human Lymphoid-Specific Octamer-Binding Protein (OTF-2) That Activates Transcription of an Immunoglobulin Promoter In Vitro,&quot; Cell, 51:783-793 (1987)."><meta name="citation_reference" content="Sen, R. and Baltimore, D., &quot;Multiple Nuclear Factors Interact with the Immunoglobulin Enhancer Sequences,&quot; Cell, 46:705-716 (1986)."><meta name="citation_reference" content="Sen, R. and Baltimore, D., Inducibility of kappa Immunoglobulin Enhancer-Binding Protein NF-kappaB by a Posttranslational Mechanism, Cell, 47:921-928 (1986)."><meta name="citation_reference" content="Sen, R. and Baltimore, D., Inducibility of κ Immunoglobulin Enhancer-Binding Protein NF-κB by a Posttranslational Mechanism, Cell, 47:921-928 (1986)."><meta name="citation_reference" content="Singh, H., et al., &quot;A nuclear factor that binds to a conserved sequence motif in transcriptional control elements of immunoglobulin genes,&quot; Nature, 319:154-158 (1986)."><meta name="citation_reference" content="Staudt, L. M., et al., &quot;A Lymphoid-specific protein binding to the octamer motif of immunoglobulin genes,&quot; Nature, 323:640-643 (1986)."><meta name="citation_reference" content="Staudt, L., et al., &quot;Cloning of a Lymphoid-Specific cDNA Encoding a Protein Binding the Regulatory Octamer DNA Motif,&quot; Science, 241:577-580 (1988)."><meta name="citation_reference" content="Strauss, F. and Varshavsky, A., &quot;A Protein Binds to a Satellite DNA Repeat at Three Specific Sites That Would be Brought into Mutualy Proximity by DNA Folding in the Nucleosome,&quot; Cell, 37:889-901 (1984)."><meta name="citation_reference" content="Treisman, R., &quot;Transient Accumulation of c-fos RNA Following Serum Stimulation Requires a Conserved 5&#39;Element and c-fos 3&#39; Sequences,&quot; Cell, 42:889-902 (1985)."><meta name="citation_reference" content="Treisman, R., “Transient Accumulation of c-fos RNA Following Serum Stimulation Requires a Conserved 5′Element and c-fos 3′ Sequences,” Cell, 42:889-902 (1985)."><meta name="citation_reference" content="Wall, R., et al., &quot;A labile inhibitor blocks immunoglobulin kappa-light-chain-gene transcription in a pre-B leukemic cell line,&quot; Proc. Natl. Acad. Sci. USA, 83:295-298 (1986)."><meta name="citation_reference" content="Wall, R., et al., “A labile inhibitor blocks immunoglobulin κ-light-chain-gene transcription in a pre-B leukemic cell line,” Proc. Natl. Acad. Sci. USA, 83:295-298 (1986)."><meta name="citation_reference" content="Wirth, T. and Baltimore, D., &quot;Nuclear factor NF-kB can interact functionally with its cognate binding site to provide lymphoid-specific promotor function,&quot; The EMBO Journal, 7 (10):3109-3113 (1988)."><meta name="citation_reference" content="Wu et al. (1988) Purification of the human immunodeficiency virus type 1 enhancer and TAR binding protiens EBP-1 and UBP-1, EMBO J. 7:2117-2129.*"><meta name="citation_reference" content="Zabel, U. and Baeurle, P., &quot;Purified Human IkB Can Rapidly Dissociate the Complex of the NF kB Transcription Factor with it Cognate DNA,&quot; Cell, 61:255-265 (1990)."><meta name="citation_patent_number" content="US:6410516"><meta name="citation_patent_application_number" content="US:08/464,364"><link rel="canonical" href="http://www.google.com/patents/US6410516"/><meta property="og:url" content="http://www.google.com/patents/US6410516"/><meta name="title" content="Patent US6410516 - Nuclear factors associated with transcriptional regulation"/><meta name="description" content="Constitutive and tissue-specific protein factors which bind to transcriptional regulatory elements of Ig genes (promoter and enhancer) are described. The factors were identified and isolated by an improved assay for protein-DNA binding. Genes encoding factors which positively regulate transcription can be isolated and employed to enhance transription of Ig genes. In particular, NF-kB, the gene encoding NF-kB, IkB and the gene encoding IkB and uses therefor."/><meta property="og:title" content="Patent US6410516 - Nuclear factors associated with transcriptional regulation"/><meta property="og:type" content="book"/><meta property="og:site_name" content="Google Books"/><meta property="og:image" content="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><link rel="image_src" href="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><script>(function(){try{var aa=function(a,b,c,d){d=d||{};d._sn=["cfg",b,c].join(".");window.gbar.logger.ml(a,d)};var g=window.gbar=window.gbar||{},l=window.gbar.i=window.gbar.i||{},m={},n;function _tvn(a,b){var c=parseInt(a,10);return isNaN(c)?b:c}function _tvf(a,b){var c=parseFloat(a);return isNaN(c)?b:c}function _tvv(a){return!!a}function p(a,b,c){(c||g)[a]=b}g.bv={n:_tvn("2",0),r:"",f:".67.",e:"0",m:_tvn("0",1)};
function q(a,b,c){var d="on"+b;if(a.addEventListener)a.addEventListener(b,c,!1);else if(a.attachEvent)a.attachEvent(d,c);else{var f=a[d];a[d]=function(){var a=f.apply(this,arguments),b=c.apply(this,arguments);return void 0==a?b:void 0==b?a:b&&a}}}var s=function(a){return function(){return g.bv.m==a}},ba=s(1),ca=s(2);p("sb",ba);p("kn",ca);l.a=_tvv;l.b=_tvf;l.c=_tvn;l.i=aa;var da=window.gbar.i.i;var t,u,v,w;function ea(a){v=a}function fa(a){var b;if(b=v&&window.encodeURIComponent)b=a.href,b=!b.match(/^http[s]?:\/\/accounts\.google\.[^/]*\/ClearSID/i)&&!b.match(/^http[s]?:\/\/[^/]*\/accounts\/ClearSID/i);if(b=b&&encodeURIComponent(v()))a.href=a.href.replace(/([?&]continue=)[^&]*/,"$1"+b)}function ga(a){window.gApplication&&(a.href=window.gApplication.getTabUrl(a.href))}
function ha(a){var b=document.forms[0].q,c=window.encodeURIComponent&&b&&b.value,b=b&&b.placeholder;c&&c!=b&&(a.href=a.href.replace(/([?&])q=[^&]*|$/,function(a,b){return(b||"&")+"q="+encodeURIComponent(c)}))}n=l.a("")?ga:ha;
function x(a,b,c,d,f,e){var h=document.getElementById(a);if(h){var k=h.style;k.left=d?"auto":b+"px";k.right=d?b+"px":"auto";k.top=c+"px";k.visibility=u?"hidden":"visible";f&&e?(k.width=f+"px",k.height=e+"px"):(x(t,b,c,d,h.offsetWidth,h.offsetHeight),u=u?"":a)}}
var y=[],ia=function(a,b){y.push(b)},ja=function(a){a=a||window.event;var b=a.target||a.srcElement;a.cancelBubble=!0;null==t&&(a=document.createElement(Array.every||window.createPopup?"iframe":"div"),a.frameBorder="0",t=a.id="gbs",a.src="javascript:''",b.parentNode.appendChild(a),q(document,"click",z));var c=b,b=0;"gb3"!=c.className&&(c=c.parentNode);a=c.getAttribute("aria-owns")||"gbi";var d=c.offsetWidth,f=20<c.offsetTop?46:24;document.getElementById("tphdr")&&(f-=3);var e=!1;do b+=c.offsetLeft||
0;while(c=c.offsetParent);var c=(document.documentElement.clientWidth||document.body.clientWidth)-b-d,h,d=document.body,k=document.defaultView;k&&k.getComputedStyle?(d=k.getComputedStyle(d,""))&&(h=d.direction):h=d.currentStyle?d.currentStyle.direction:d.style.direction;h="rtl"==h;if("gbi"==a){for(d=0;k=y[d++];)k();A(null,window.navExtra);h&&(b=c,e=!0)}else h||(b=c,e=!0);u!=a&&z();x(a,b,f,e)},z=function(){u&&x(u,0,0)},A=function(a,b){var c,d=document.getElementById("gbi"),f=a;f||(f=d.firstChild);
for(;b&&(c=b.pop());){var e=d,h=c,k=f;w||(w="gb2");e.insertBefore(h,k).className=w}},ka=function(a,b,c){if((b=document.getElementById(b))&&a){a.className="gb4";var d=document.createElement("span");d.appendChild(a);d.appendChild(document.createTextNode(" | "));d.id=c;b.appendChild(d)}},la=function(){return document.getElementById("gb_70")},ma=function(){return!!u};p("qs",n);p("setContinueCb",ea);p("pc",fa);p("tg",ja);p("close",z);p("addLink",ka);p("almm",A);p("si",la);p("adh",ia);p("op",ma);var B=function(){},C=function(){},F=function(a){var b=new Image,c=D;b.onerror=b.onload=b.onabort=function(){try{delete E[c]}catch(a){}};E[c]=b;b.src=a;D=c+1},E=[],D=0;p("logger",{il:C,ml:B,log:F});var G=window.gbar.logger;var H={},na={},I=[],oa=l.b("0.1",.1),pa=l.a("1",!0),qa=function(a,b){I.push([a,b])},ra=function(a,b){H[a]=b},sa=function(a){return a in H},J={},K=function(a,b){J[a]||(J[a]=[]);J[a].push(b)},ta=function(a){K("m",a)},L=function(a,b){var c=document.createElement("script");c.src=a;c.async=pa;Math.random()<oa&&(c.onerror=function(){c.onerror=null;B(Error("Bundle load failed: name="+(b||"UNK")+" url="+a))});(document.getElementById("xjsc")||document.getElementsByTagName("body")[0]||
document.getElementsByTagName("head")[0]).appendChild(c)},N=function(a){for(var b=0,c;(c=I[b])&&c[0]!=a;++b);!c||c[1].l||c[1].s||(c[1].s=!0,M(2,a),c[1].url&&L(c[1].url,a),c[1].libs&&m.d&&m.d(c[1].libs))},O=function(a){K("gc",a)},P=null,ua=function(a){P=a},M=function(a,b,c){if(P){a={t:a,b:b};if(c)for(var d in c)a[d]=c[d];try{P(a)}catch(f){}}};p("mdc",H);p("mdi",na);p("bnc",I);p("qGC",O);p("qm",ta);p("qd",J);p("lb",N);p("mcf",ra);p("bcf",qa);p("aq",K);p("mdd","");p("has",sa);
p("trh",ua);p("tev",M);var Q=l.b("0.1",.001),R=0;
function _mlToken(a,b){try{if(1>R){R++;var c,d=a,f=b||{},e=encodeURIComponent,h=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&jexpid=",e("17483"),"&srcpg=",e("prop=22"),"&jsr=",Math.round(1/Q),"&ogev=",e("QZHtU7zZDqu2sQSunYLgBQ"),"&ogf=",g.bv.f,"&ogrp=",e("1"),"&ogv=",e("1407723702.0"),"&oggv="+e("es_plusone_gc_20140723.0_p0"),"&ogd=",e("com"),"&ogc=",e("USA"),"&ogl=",e("en")];f._sn&&(f._sn="og."+
f._sn);for(var k in f)h.push("&"),h.push(e(k)),h.push("="),h.push(e(f[k]));h.push("&emsg=");h.push(e(d.name+":"+d.message));var r=h.join("");S(r)&&(r=r.substr(0,2E3));c=r;var Aa=window.gbar.logger._aem(a,c);F(Aa)}}catch(Na){}}var S=function(a){return 2E3<=a.length},va=function(a,b){return b};function T(a){B=a;p("_itl",S,G);p("_aem",va,G);p("ml",B,G);a={};H.er=a}l.a("")?T(function(a){throw a;}):l.a("1")&&Math.random()<Q&&T(_mlToken);I.push(["m",{url:"//ssl.gstatic.com/gb/js/scm_7385cc5883250b43a39405734c1bea59.js"}]);g.mcf("c",{});g.sg={c:""};if(l.a("1")){var wa=l.a("");I.push(["gc",{auto:wa,url:"//ssl.gstatic.com/gb/js/abc/gci_91f30755d6a6b787dcc2a4062e6e9824.js",libs:"googleapis.client:plusone:gapi.iframes"}]);var xa={version:"gci_91f30755d6a6b787dcc2a4062e6e9824.js",index:"",lang:"en"};H.gc=xa;var U=function(a){window.googleapis&&window.iframes?a&&a():(a&&O(a),N("gc"))};p("lGC",U);l.a("1")&&p("lPWF",U)};window.__PVT="";if(l.a("1")&&l.a("1")){var V=function(a){U(function(){K("pw",a);N("pw")})};p("lPW",V);I.push(["pw",{url:"//ssl.gstatic.com/gb/js/abc/pwm_45f73e4df07a0e388b0fa1f3d30e7280.js"}]);var W=[],ya=function(a){W[0]=a},za=function(a,b){var c=b||{};c._sn="pw";B(a,c)},Ba={signed:W,elog:za,base:"https://plusone.google.com/u/0",loadTime:(new Date).getTime()};H.pw=Ba;var X=function(a,b){for(var c=b.split("."),d=function(){var b=arguments;a(function(){for(var a=g,d=0,e=c.length-1;d<e;++d)a=a[c[d]];a[c[d]].apply(a,b)})},f=g,e=0,h=c.length-1;e<h;++e)f=
f[c[e]]=f[c[e]]||{};return f[c[e]]=d};X(V,"pw.clk");X(V,"pw.hvr");p("su",ya,g.pw)};function Ca(){function a(){for(var b;(b=e[h++])&&"m"!=b[0]&&!b[1].auto;);b&&(M(2,b[0]),b[1].url&&L(b[1].url,b[0]),b[1].libs&&m.d&&m.d(b[1].libs));h<e.length&&setTimeout(a,0)}function b(){0<f--?setTimeout(b,0):a()}var c=l.a("1"),d=l.a(""),f=3,e=I,h=0,k=window.gbarOnReady;if(k)try{k()}catch(r){da(r,"ml","or")}d?p("ldb",a):c?q(window,"load",b):b()}p("rdl",Ca);var Da={D:1,H:2,da:3,p:4,W:5,M:6,F:7,g:8,ha:9,U:10,L:11,T:12,S:13,N:14,Q:15,P:16,fa:17,w:18,O:19,ga:20,ea:21,u:22,G:23,ja:24,ka:25,ia:26,A:27,j:28,o:29,k:30,ca:31,Z:32,$:33,J:34,K:35,ba:36,aa:37,Y:38,B:39,R:40,v:41,X:42,V:43,h:48,C:49,I:500},Y=[1,2,3,4,5,6,9,10,11,13,14,28,29,30,34,35,37,38,39,40,41,42,43,48,49,500];var Z=l.b("0.001",1E-4),Ea=l.b("1",1),Fa=!1,Ga=!1;if(l.a("1")){var Ha=Math.random();Ha<=Z&&(Fa=!0);Ha<=Ea&&(Ga=!0)}var Ia=Da,$=null;function Ja(){var a=0,b=function(b,d){l.a(d)&&(a|=b)};b(1,"");b(2,"");b(4,"");b(8,"");return a}
function Ka(a,b){var c=Z,d=Fa,f;f=a;if(!$){$={};for(var e=0;e<Y.length;e++){var h=Y[e];$[h]=!0}}if(f=!!$[f])c=Ea,d=Ga;if(d){d=encodeURIComponent;g.rp?(f=g.rp(),f="-1"!=f?f:"1"):f="1";c=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&oge=",a,"&ogex=",d("17483"),"&ogev=",d("QZHtU7zZDqu2sQSunYLgBQ"),"&ogf=",g.bv.f,"&ogp=",d("22"),"&ogrp=",d(f),"&ogsr=",Math.round(1/c),"&ogv=",d("1407723702.0"),"&oggv="+
d("es_plusone_gc_20140723.0_p0"),"&ogd=",d("com"),"&ogl=",d("en"),"&ogc=",d("USA"),"&ogus=",Ja()];if(b){"ogw"in b&&(c.push("&ogw="+b.ogw),delete b.ogw);var k;f=b;e=[];for(k in f)0!=e.length&&e.push(","),e.push(La(k)),e.push("."),e.push(La(f[k]));k=e.join("");""!=k&&(c.push("&ogad="),c.push(d(k)))}F(c.join(""))}}function La(a){"number"==typeof a&&(a+="");return"string"==typeof a?a.replace(".","%2E").replace(",","%2C"):a}C=Ka;p("il",C,G);var Ma={};H.il=Ma;setTimeout(function(){C(Ia.g)},0);}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var b=window.gbar.i.i;var c=window.gbar;var f=function(d){try{var a=document.getElementById("gbom");a&&d.appendChild(a.cloneNode(!0))}catch(e){b(e,"omas","aomc")}};c.aomc=f;}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var a=window.gbar;a.mcf("pm",{p:""});}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{window.gbar.rdl();}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
if (window['_OC_timingAction']) {window['_OC_timingAction']('patents_refpage');}</script><style>#gbar,#guser{font-size:13px;padding-top:1px !important;}#gbar{float:left;height:22px}#guser{padding-bottom:7px !important;text-align:right}.gbh,.gbd{border-top:1px solid #c9d7f1;font-size:1px}.gbh{height:0;position:absolute;top:24px;width:100%}#gbs,.gbm{background:#fff;left:0;position:absolute;text-align:left;visibility:hidden;z-index:1000}.gbm{border:1px solid;border-color:#c9d7f1 #36c #36c #a2bae7;z-index:1001}.gb1{margin-right:.5em}.gb1,.gb3{zoom:1}.gb2{display:block;padding:.2em .5em}.gb2,.gb3{text-decoration:none !important;border-bottom:none}a.gb1,a.gb4{text-decoration:underline !important}a.gb1,a.gb2,a.gb3,a.gb4{color:#00c !important}.gbi .gb3,.gbi .gb2,.gbi .gb4{color:#dd8e27 !important}.gbf .gb3,.gbf .gb2,.gbf .gb4{color:#900 !important}a.gb2:hover{background:#36c;color:#fff !important}#gbar .gbz0l{color:#000 !important;cursor:default;font-weight:bold;text-decoration:none !important}
#gbar { padding:.3em .6em !important;}</style></head><body  topmargin="3" marginheight="3"><div id=gbar><nobr><a onclick=gbar.qs(this);gbar.logger.il(1,{t:1}); class=gb1 id=gb_1 href="https://www.google.com/search?sa=N&tab=tw">Search</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:2}); class=gb1 id=gb_2 href="http://www.google.com/search?hl=en&tbm=isch&source=og&sa=N&tab=ti">Images</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:8}); class=gb1 id=gb_8 href="http://maps.google.com/maps?hl=en&sa=N&tab=tl">Maps</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:78}); class=gb1 id=gb_78 href="https://play.google.com/?hl=en&sa=N&tab=t8">Play</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:36}); class=gb1 id=gb_36 href="http://www.youtube.com/results?sa=N&tab=t1">YouTube</a> <a onclick=gbar.logger.il(1,{t:5}); class=gb1 id=gb_5 href="http://news.google.com/nwshp?hl=en&tab=tn">News</a> <a onclick=gbar.logger.il(1,{t:23}); class=gb1 id=gb_23 href="https://mail.google.com/mail/?tab=tm">Gmail</a> <a onclick=gbar.logger.il(1,{t:25}); class=gb1 id=gb_25 href="https://drive.google.com/?tab=to">Drive</a> <a class=gb3 href="http://www.google.com/intl/en/options/" onclick="this.blur();gbar.tg(event);return !1" aria-haspopup=true><u>More</u> <small>&#9660;</small></a><div class=gbm id=gbi><a onclick=gbar.logger.il(1,{t:24}); class=gb2 id=gb_24 href="https://www.google.com/calendar?tab=tc">Calendar</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:51}); class=gb2 id=gb_51 href="http://translate.google.com/?hl=en&sa=N&tab=tT">Translate</a><a onclick=gbar.logger.il(1,{t:17}); class=gb2 id=gb_17 href="http://www.google.com/mobile/?hl=en&tab=tD">Mobile</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:10}); class=gb2 id=gb_10 href="http://www.google.com/search?hl=en&tbo=u&tbm=bks&source=og&sa=N&tab=tp">Books</a><a onclick=gbar.logger.il(1,{t:212}); class=gb2 id=gb_212 href="https://wallet.google.com/manage/?tab=ta">Wallet</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:6}); class=gb2 id=gb_6 href="http://www.google.com/search?hl=en&tbo=u&tbm=shop&source=og&sa=N&tab=tf">Shopping</a><a onclick=gbar.logger.il(1,{t:30}); class=gb2 id=gb_30 href="http://www.blogger.com/?tab=tj">Blogger</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:27}); class=gb2 id=gb_27 href="http://www.google.com/finance?sa=N&tab=te">Finance</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:31}); class=gb2 id=gb_31 href="https://plus.google.com/photos?sa=N&tab=tq">Photos</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:12}); class=gb2 id=gb_12 href="http://www.google.com/search?hl=en&tbo=u&tbm=vid&source=og&sa=N&tab=tv">Videos</a><div class=gb2><div class=gbd></div></div><a onclick=gbar.logger.il(1,{t:66}); href="http://www.google.com/intl/en/options/" class=gb2>Even more &raquo;</a></div></nobr></div><div id=guser width=100%><nobr><span id=gbn class=gbi></span><span id=gbf class=gbf></span><span id=gbe></span><a target=_top id=gb_70 href="https://www.google.com/accounts/Login?service=&continue=http://www.google.com/patents%3Fhl%3Den&hl=en" class=gb4>Sign in</a><div style="display: none"><div class=gbm id=gbd5 aria-owner=gbg5><div class=gbmc><ol id=gbom class=gbmcc></ol></div></div></div></nobr></div><div class=gbh style=left:0></div><div class=gbh style=right:0></div><div role="alert" style="position: absolute; left: 0; right: 0;"><a href="http://www.google.com/patents/us6410516?hl=en&amp;output=html_text" title="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."><img border="0" src="http://www.google.com/images/cleardot.gif"alt="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."></a></div><div id="guser"><nobr></nobr></div><div style="clear:both;"></div><div id="gb-top-search-box" class="gb-top-search-box-small gb-reset"><table><tr><td class="logo"><a href="http://www.google.com/patents" class="logo-link"><img class="logo-img" src="/intl/en/images/logos/google_logo_41.png" alt="Go to Google Books Home" height="41"/></a></td><td><form action="http://www.google.com/search" name="f" id="vheadf" method="get"><span id="hf"></span><input type="hidden" name="tbm" value="pts"/><input type="hidden" name="tbo" value="1"/><input type="hidden" name="hl" value="en"/><table><tr><td><div class="inputs"><table><tr><td><div class="text-input"><input type="text" name="q" id="vheadq" class="text" maxlength="2048" size="31" value="" title="Search Patents" accesskey="s" autocomplete="off"/><script>window._OC_autoDir &&window._OC_autoDir('vheadq', 'tia-vheadq');</script></div></td><td><div class="submit-input"><input name="btnG" class="submit" type="submit" value=""/></div></td></tr></table></div></td><td class="col-ext-links"><div class="ext-links"><a href="http://www.google.com/advanced_patent_search">&lt;nobr&gt;Advanced Patent Search&lt;/nobr&gt;</a></div></td></tr></table></form></td></tr></table></div><div class="kd-appbar"><h2 class="kd-appname"><a href="/patents">Patents</a></h2><div class="kd-buttonbar left" id="left-toolbar-buttons"><a id="appbar-write-review-link" href=""></a><a id="appbar-view-print-sample-link" href=""></a><a id="appbar-view-ebook-sample-link" href=""></a><a id="appbar-patents-prior-art-finder-link" href="https://www.google.com/patents/related/US6410516"></a><a id="appbar-patents-discuss-this-link" href="http://www.google.com/url?id=ottaBAABERAJ&amp;q=http://patents.stackexchange.com/redirect/google-patents%3Fpatent%3DUS6410516&amp;usg=AFQjCNG_QKDnb431Qj0DjbBs-ux54r3P9g" data-is-grant="true"></a><a id="appbar-read-patent-link" href="//docs.google.com/viewer?url=patentimages.storage.googleapis.com/pdfs/US6410516.pdf"></a><a id="appbar-download-pdf-link" href="//patentimages.storage.googleapis.com/pdfs/US6410516.pdf"></a></div><div class="kd-buttonbar right" id="right-toolbar-buttons"></div></div><div id="books-microdata" itemscope=""itemtype="http://schema.org/Book"itemid="http://www.google.com/patents/US6410516" style="display:none"><span itemprop="description">Constitutive and tissue-specific protein factors which bind to transcriptional regulatory elements of Ig genes (promoter and enhancer) are described. The factors were identified and isolated by an improved assay for protein-DNA binding. Genes encoding factors which positively regulate transcription can...</span><span itemprop="url">http://www.google.com/patents/US6410516?utm_source=gb-gplus-share</span><span class="main-title" itemprop="name">Patent US6410516 - Nuclear factors associated with transcriptional regulation</span><img itemprop="image" src="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"alt="Patent US6410516 - Nuclear factors associated with transcriptional regulation" title="Patent US6410516 - Nuclear factors associated with transcriptional regulation"></div><div style="display: none"><ol id="ofe-gear-menu-contents" class="gbmcc"><li class="gbe gbmtc"><a class="gbmt goog-menuitem-content" id="" href="http://www.google.com/advanced_patent_search">Advanced Patent Search</a></li></ol></div><table id="viewport_table" cellpadding="0" style="clear:both" cellspacing="0"><tr><td id="viewport_td"><div class=vertical_module_list_row><div id=intl_patents class=about_content><div id=intl_patents_v><table class="patent-bibdata"><tr><td class="patent-bibdata-heading">Publication number</td><td class="single-patent-bibdata">US6410516 B1</td></tr><tr><td class="patent-bibdata-heading">Publication type</td><td class="single-patent-bibdata">Grant</td></tr><tr><td class="patent-bibdata-heading">Application number</td><td class="single-patent-bibdata">US 08/464,364</td></tr><tr><td class="patent-bibdata-heading">Publication date</td><td class="single-patent-bibdata">Jun 25, 2002</td></tr><tr><td class="patent-bibdata-heading">Filing date</td><td class="single-patent-bibdata">Jun 5, 1995</td></tr><tr><td class="patent-bibdata-heading">Priority date<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The priority date is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the date listed."></span></td><td class="single-patent-bibdata">Jan 9, 1986</td></tr><tr><td class="patent-bibdata-heading">Fee status<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The fee status is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the status or dates listed."></span></td><td class="single-patent-bibdata">Paid</td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Also published as</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/US6150090">US6150090</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20040214757">US20040214757</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20060063709">US20060063709</a></span></span></td></tr><tr class="patent-bibdata-list-row alternate-patent-number"><td class="patent-bibdata-heading">Publication number</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value">08464364, </span><span class="patent-bibdata-value">464364, </span><span class="patent-bibdata-value">US 6410516 B1, </span><span class="patent-bibdata-value">US 6410516B1, </span><span class="patent-bibdata-value">US-B1-6410516, </span><span class="patent-bibdata-value">US6410516 B1, </span><span class="patent-bibdata-value">US6410516B1</span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Inventors</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22David+Baltimore%22">David Baltimore</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Ranjan+Sen%22">Ranjan Sen</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Phillip+A.+Sharp%22">Phillip A. Sharp</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Harinder+Singh%22">Harinder Singh</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Louis+Staudt%22">Louis Staudt</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Jonathan+H.+LeBowitz%22">Jonathan H. LeBowitz</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Albert+S.+Baldwin,+Jr.%22">Albert S. Baldwin, Jr.</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Roger+G.+Clerc%22">Roger G. Clerc</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Lynn+M.+Corcoran%22">Lynn M. Corcoran</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Patrick+A.+Baeuerle%22">Patrick A. Baeuerle</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Michael+J.+Lenardo%22">Michael J. Lenardo</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Chen-Ming+Fan%22">Chen-Ming Fan</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Thomas+P.+Maniatis%22">Thomas P. Maniatis</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Original Assignee</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=inassignee:%22President+%26+Fellows+Of+Harvard+College%22">President & Fellows Of Harvard College</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=inassignee:%22Massachusetts+Institute+Of+Technology%22">Massachusetts Institute Of Technology</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=inassignee:%22Whitehead+Instittue+For+Biomedical+Research%22">Whitehead Instittue For Biomedical Research</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Export Citation</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/US6410516.bibtex">BiBTeX</a>, </span><span class="patent-bibdata-value"><a href="/patents/US6410516.enw">EndNote</a>, </span><span class="patent-bibdata-value"><a href="/patents/US6410516.ris">RefMan</a></span></span></td></tr><tr class="patent-internal-links"><td colspan=2><span class="patent-bibdata-value"><a href="#backward-citations">Patent Citations</a> (1),</span> <span class="patent-bibdata-value"><a href="#npl-citations">Non-Patent Citations</a> (67),</span> <span class="patent-bibdata-value"><a href="#forward-citations">Referenced by</a> (66),</span> <span class="patent-bibdata-value"><a href="#classifications">Classifications</a> (37),</span> <span class="patent-bibdata-value"><a href="#legal-events">Legal Events</a> (7)</span> </td></tr><tr><td colspan=2 class="patent-bibdata-external-link-spacer-top"></td></tr><tr class="patent-bibdata-external-link-spacer-bottom"></tr><tr><td colspan=2><span class="patent-bibdata-heading">External Links:&nbsp;</span><span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=ottaBAABERAJ&q=http://patft.uspto.gov/netacgi/nph-Parser%3FSect2%3DPTO1%26Sect2%3DHITOFF%26p%3D1%26u%3D/netahtml/PTO/search-bool.html%26r%3D1%26f%3DG%26l%3D50%26d%3DPALL%26RefSrch%3Dyes%26Query%3DPN/6410516&usg=AFQjCNGsmQM-0AB7XvyRZDy5kuSlDAckDg">USPTO</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=ottaBAABERAJ&q=http://assignments.uspto.gov/assignments/q%3Fdb%3Dpat%26pat%3D6410516&usg=AFQjCNHeqUU_b-WgysPhatCc7765qLreSQ">USPTO Assignment</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=ottaBAABERAJ&q=http://worldwide.espacenet.com/publicationDetails/biblio%3FCC%3DUS%26NR%3D6410516B1%26KC%3DB1%26FT%3DD&usg=AFQjCNEbYbLXls9TfNNqD9TREZv-ShWIxg">Espacenet</a></span></span></td></tr><tr class="patent-bibdata-group-spacer"></tr></table><div class="number-and-title"><span class="patent-title"><invention-title mxw-id="PT54944899" lang="EN" load-source="patent-office">Nuclear factors associated with transcriptional regulation</invention-title></span><br><span class="patent-number">US 6410516 B1</span></div><div class="patent-section patent-abstract-section"><div class="patent-section-header"><span class="patent-section-title">Abstract</span></div><div class="patent-text"><abstract mxw-id="PA50339004" lang="EN" load-source="patent-office"> <div class="abstract">Constitutive and tissue-specific protein factors which bind to transcriptional regulatory elements of Ig genes (promoter and enhancer) are described. The factors were identified and isolated by an improved assay for protein-DNA binding. Genes encoding factors which positively regulate transcription can be isolated and employed to enhance transription of Ig genes. In particular, NF-kB, the gene encoding NF-kB, IkB and the gene encoding IkB and uses therefor.</div>
  </abstract></div></div><div class="patent-section patent-drawings-section"><div class="patent-section-header"><span class="patent-section-title">Images<span class="patent-section-count">(58)</span></span></div><div class="patent-drawings-body"><div class="patent-drawings-carousel"><div class="drawings"><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6410516B1/US06410516-20020625-D00001.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6410516B1/US06410516-20020625-D00001.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6410516B1/US06410516-20020625-D00002.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6410516B1/US06410516-20020625-D00002.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6410516B1/US06410516-20020625-D00003.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6410516B1/US06410516-20020625-D00003.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6410516B1/US06410516-20020625-D00004.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6410516B1/US06410516-20020625-D00004.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6410516B1/US06410516-20020625-D00005.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6410516B1/US06410516-20020625-D00005.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6410516B1/US06410516-20020625-D00006.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6410516B1/US06410516-20020625-D00006.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6410516B1/US06410516-20020625-D00007.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6410516B1/US06410516-20020625-D00007.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6410516B1/US06410516-20020625-D00008.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6410516B1/US06410516-20020625-D00008.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6410516B1/US06410516-20020625-D00009.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6410516B1/US06410516-20020625-D00009.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6410516B1/US06410516-20020625-D00010.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6410516B1/US06410516-20020625-D00010.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6410516B1/US06410516-20020625-D00011.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6410516B1/US06410516-20020625-D00011.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6410516B1/US06410516-20020625-D00012.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6410516B1/US06410516-20020625-D00012.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6410516B1/US06410516-20020625-D00013.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6410516B1/US06410516-20020625-D00013.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6410516B1/US06410516-20020625-D00014.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6410516B1/US06410516-20020625-D00014.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6410516B1/US06410516-20020625-D00015.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6410516B1/US06410516-20020625-D00015.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6410516B1/US06410516-20020625-D00016.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6410516B1/US06410516-20020625-D00016.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6410516B1/US06410516-20020625-D00017.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6410516B1/US06410516-20020625-D00017.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6410516B1/US06410516-20020625-D00018.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6410516B1/US06410516-20020625-D00018.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6410516B1/US06410516-20020625-D00019.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6410516B1/US06410516-20020625-D00019.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6410516B1/US06410516-20020625-D00020.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6410516B1/US06410516-20020625-D00020.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6410516B1/US06410516-20020625-D00021.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6410516B1/US06410516-20020625-D00021.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6410516B1/US06410516-20020625-D00022.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6410516B1/US06410516-20020625-D00022.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6410516B1/US06410516-20020625-D00023.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6410516B1/US06410516-20020625-D00023.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6410516B1/US06410516-20020625-D00024.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6410516B1/US06410516-20020625-D00024.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6410516B1/US06410516-20020625-D00025.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6410516B1/US06410516-20020625-D00025.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6410516B1/US06410516-20020625-D00026.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6410516B1/US06410516-20020625-D00026.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6410516B1/US06410516-20020625-D00027.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6410516B1/US06410516-20020625-D00027.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6410516B1/US06410516-20020625-D00028.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6410516B1/US06410516-20020625-D00028.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6410516B1/US06410516-20020625-D00029.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6410516B1/US06410516-20020625-D00029.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6410516B1/US06410516-20020625-D00030.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6410516B1/US06410516-20020625-D00030.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6410516B1/US06410516-20020625-D00031.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6410516B1/US06410516-20020625-D00031.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6410516B1/US06410516-20020625-D00032.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6410516B1/US06410516-20020625-D00032.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6410516B1/US06410516-20020625-D00033.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6410516B1/US06410516-20020625-D00033.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6410516B1/US06410516-20020625-D00034.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6410516B1/US06410516-20020625-D00034.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6410516B1/US06410516-20020625-D00035.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6410516B1/US06410516-20020625-D00035.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6410516B1/US06410516-20020625-D00036.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6410516B1/US06410516-20020625-D00036.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6410516B1/US06410516-20020625-D00037.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6410516B1/US06410516-20020625-D00037.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6410516B1/US06410516-20020625-D00038.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6410516B1/US06410516-20020625-D00038.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6410516B1/US06410516-20020625-D00039.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6410516B1/US06410516-20020625-D00039.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6410516B1/US06410516-20020625-D00040.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6410516B1/US06410516-20020625-D00040.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6410516B1/US06410516-20020625-D00041.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6410516B1/US06410516-20020625-D00041.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6410516B1/US06410516-20020625-D00042.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6410516B1/US06410516-20020625-D00042.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6410516B1/US06410516-20020625-D00043.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6410516B1/US06410516-20020625-D00043.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6410516B1/US06410516-20020625-D00044.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6410516B1/US06410516-20020625-D00044.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6410516B1/US06410516-20020625-D00045.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6410516B1/US06410516-20020625-D00045.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6410516B1/US06410516-20020625-D00046.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6410516B1/US06410516-20020625-D00046.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6410516B1/US06410516-20020625-D00047.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6410516B1/US06410516-20020625-D00047.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6410516B1/US06410516-20020625-D00048.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6410516B1/US06410516-20020625-D00048.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6410516B1/US06410516-20020625-D00049.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6410516B1/US06410516-20020625-D00049.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6410516B1/US06410516-20020625-D00050.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6410516B1/US06410516-20020625-D00050.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6410516B1/US06410516-20020625-D00051.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6410516B1/US06410516-20020625-D00051.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6410516B1/US06410516-20020625-D00052.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6410516B1/US06410516-20020625-D00052.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6410516B1/US06410516-20020625-D00053.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6410516B1/US06410516-20020625-D00053.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6410516B1/US06410516-20020625-D00054.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6410516B1/US06410516-20020625-D00054.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6410516B1/US06410516-20020625-D00055.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6410516B1/US06410516-20020625-D00055.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6410516B1/US06410516-20020625-D00056.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6410516B1/US06410516-20020625-D00056.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6410516B1/US06410516-20020625-D00057.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6410516B1/US06410516-20020625-D00057.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6410516B1/US06410516-20020625-D00058.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6410516B1/US06410516-20020625-D00058.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div></div></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img" alt="Previous page"src="/googlebooks/images/kennedy/page_left.png"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img" alt="Next page"src="/googlebooks/images/kennedy/page_right.png"width="21" height="21" /></div></div></div><div class="patent-post-drawings"></div><div class="patent-section patent-claims-section"><div class="patent-section-header"><span class="patent-section-title">Claims<span class="patent-section-count">(203)</span></span></div><div class="patent-text"><div mxw-id="PCLM8314330" lang="EN" load-source="patent-office" class="claims">
    <claim-statement>What is claimed is: </claim-statement> <div class="claim"> <div num="1" id="US-6410516-B1-CLM-00001" class="claim">
      <div class="claim-text">1. A method for inhibiting expression, in a eukaryotic cell, of a gene whose transcription is regulated by NF-κKB, the method comprising reducing NF-κKB activity in the cell such that expression of said gene is inhibited.</div>
    </div>
    </div> <div class="claim"> <div num="2" id="US-6410516-B1-CLM-00002" class="claim">
      <div class="claim-text">2. A method for selectively inhibiting expression, in a eukaryotic cell, of genes whose transcription is regulated by NF-κB, the method comprising reducing NF-κB activity in the cell such that expression of said genes is inhibited.</div>
    </div>
    </div> <div class="claim"> <div num="3" id="US-6410516-B1-CLM-00003" class="claim">
      <div class="claim-text">3. A method for reducing the level of expression of a viral gene whose transcription is regulated by NF-κB in a eukaryotic cell, the method comprising reducing NF-κB activity in the cell such that expression of said viral gene is reduced.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="4" id="US-6410516-B1-CLM-00004" class="claim">
      <div class="claim-text">4. The method of <claim-ref idref="US-6410516-B1-CLM-00003">claim 3</claim-ref>, wherein the viral gene is a cytomegalovirus (CMV), human immunodeficiency virus (HIV) or simian virus 40 (SV40) gene.</div>
    </div>
    </div> <div class="claim"> <div num="5" id="US-6410516-B1-CLM-00005" class="claim">
      <div class="claim-text">5. A method for reducing the level of expression of a cytokine gene whose transcription is regulated by NF-κB in a eukaryotic cell comprising reducing NF-κB activity in the cell such that expression of said cytokine gene is reduced.</div>
    </div>
    </div> <div class="claim"> <div num="6" id="US-6410516-B1-CLM-00006" class="claim">
      <div class="claim-text">6. A method for diminishing induced NF-κB-mediated intracellular signaling comprising reducing NF-κB activity in cells such that NF-κB-mediated intracellular signaling is diminished.</div>
    </div>
    </div> <div class="claim"> <div num="7" id="US-6410516-B1-CLM-00007" class="claim">
      <div class="claim-text">7. A method for modifying effects of external influences on a eukaryotic cell, which external influences induce NF-κB-mediated intracellular signaling, the method comprising altering NF-κB activity in the cells such that NF-κB-mediated effects of external influences are modified.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="8" id="US-6410516-B1-CLM-00008" class="claim">
      <div class="claim-text">8. The method of <claim-ref idref="US-6410516-B1-CLM-00007">claim 7</claim-ref>, wherein NF-κB activity in the cell is reduced.</div>
    </div>
    </div> <div class="claim"> <div num="9" id="US-6410516-B1-CLM-00009" class="claim">
      <div class="claim-text">9. A method for reducing, in eukaryotic cells, the level of expression of genes which are activated by extracellular influences which induce NF-κB-mediated intracellular signaling, the method comprising reducing NF-κB activity in the cells such that expression of said genes is reduced.</div>
    </div>
    </div> <div class="claim"> <div num="10" id="US-6410516-B1-CLM-00010" class="claim">
      <div class="claim-text">10. A method for reducing the effects of bacterial infection on mammalian cells comprising reducing NF-κB activity in mammalian cells so as to reduce bacterial lipopolysaccharide-induced gene expression in the mammalian cells.</div>
    </div>
    </div> <div class="claim"> <div num="11" id="US-6410516-B1-CLM-00011" class="claim">
      <div class="claim-text">11. A method for reducing the effects of viral infection on mammalian cells comprising reducing NF-κB activity in mammalian cells so as to reduce virus-mediated gene expression in the mammalian cells.</div>
    </div>
    </div> <div class="claim"> <div num="12" id="US-6410516-B1-CLM-00012" class="claim">
      <div class="claim-text">12. A method for reducing the effects of bacterial infection on mammalian immune cells comprising reducing NF-κB activity in mammalian immune cells so as to reduce bacterial lipopolysaccharide-mediated stimulation of the immune cells.</div>
    </div>
    </div> <div class="claim"> <div num="13" id="US-6410516-B1-CLM-00013" class="claim">
      <div class="claim-text">13. A method for reducing the effects of bacterial lipopolysaccharide on mammalian cells comprising reducing NF-κB activity in the cells so as to reduce bacterial lipopolysaccharide-induced gene expression in the cells.</div>
    </div>
    </div> <div class="claim"> <div num="14" id="US-6410516-B1-CLM-00014" class="claim">
      <div class="claim-text">14. A method for reducing bacterial lipopolysaccharide-induced expression of cytokines in mammalian cells, which method comprises reducing NF-κB activity in the cells so as to reduce bacterial lipopolysaccharide-induced expression of said cytokines in the cells.</div>
    </div>
    </div> <div class="claim"> <div num="15" id="US-6410516-B1-CLM-00015" class="claim">
      <div class="claim-text">15. A method for reducing bacterial lipopolysaccharide-induced expression of Tumor Necrosis Factor-α in mammalian cells, which method comprises reducing NF-κB activity in the cells so as to reduce bacterial lipopolysaccharide-induced expression of Tumor Necrosis Factor-α in the cells.</div>
    </div>
    </div> <div class="claim"> <div num="16" id="US-6410516-B1-CLM-00016" class="claim">
      <div class="claim-text">16. A method for reducing bacterial lipopolysaccharide-mediated stimulation of immune cells, which method comprises reducing NF-κB activity in the cells so as to reduce bacterial lipopolysaccharide-mediated stimulation of the immune cells.</div>
    </div>
    </div> <div class="claim"> <div num="17" id="US-6410516-B1-CLM-00017" class="claim">
      <div class="claim-text">17. A method for reducing bacterial-induced NF-κB signaling in cells, which method comprises reducing NF-κB activity in the cells so as to reduce bacterial-induced NF-κB signaling in the cells.</div>
    </div>
    </div> <div class="claim"> <div num="18" id="US-6410516-B1-CLM-00018" class="claim">
      <div class="claim-text">18. A method for reducing Interleukin-1 or Tumor Necrosis Factor-α activity in mammalian cells comprising reducing NF-κB activity in the cells so as to reduce intracellular signaling caused by Interleukin-1 or Tumor Necrosis Factor-α in the cells.</div>
    </div>
    </div> <div class="claim"> <div num="19" id="US-6410516-B1-CLM-00019" class="claim">
      <div class="claim-text">19. A method for reducing bacterial lipopolysaccharide-induced nuclear translocation of NF-κB in eukaryotic cells comprising inhibiting one or more of: (a) modification of an IκB protein which reduces binding to NF-κB, (b) degradation of an IκB protein, or (c) dissociation of NF-κB-IκB complexes so as to reduce nuclear translocation of NF-κB in the cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="20" id="US-6410516-B1-CLM-00020" class="claim">
      <div class="claim-text">20. The method of <claim-ref idref="US-6410516-B1-CLM-00001">claim 1</claim-ref> wherein NF-κB activity is reduced by decreasing the level of NF-κB not bound in an NF-κB:IκB complex.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="21" id="US-6410516-B1-CLM-00021" class="claim">
      <div class="claim-text">21. The method of <claim-ref idref="US-6410516-B1-CLM-00001">claim 1</claim-ref> wherein NF-κB activity is reduced by inhibiting the passage of NF-κB into the nucleus of cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="22" id="US-6410516-B1-CLM-00022" class="claim">
      <div class="claim-text">22. The method of <claim-ref idref="US-6410516-B1-CLM-00001">claim 1</claim-ref>, wherein NF-κB activity is reduced by inhibiting modification of an IκB protein, which modification otherwise reduces IκB binding to NF-κB.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="23" id="US-6410516-B1-CLM-00023" class="claim">
      <div class="claim-text">23. The method of <claim-ref idref="US-6410516-B1-CLM-00001">claim 1</claim-ref>, wherein NF-κB activity is reduced by inhibiting degradation of an IκB protein.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="24" id="US-6410516-B1-CLM-00024" class="claim">
      <div class="claim-text">24. The method of <claim-ref idref="US-6410516-B1-CLM-00001">claim 1</claim-ref>, wherein NF-κB activity is reduced by inhibiting dissociation of NF-κB:IκB complexes.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="25" id="US-6410516-B1-CLM-00025" class="claim">
      <div class="claim-text">25. The method of <claim-ref idref="US-6410516-B1-CLM-00001">claim 1</claim-ref> wherein reducing NF-κB activity comprises reducing binding of NF-κB to NF-κB recognition sites on genes which are transcriptionally regulated by NF-κB.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="26" id="US-6410516-B1-CLM-00026" class="claim">
      <div class="claim-text">26. The method of <claim-ref idref="US-6410516-B1-CLM-00001">claim 1</claim-ref>, carried out on mammalian cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="27" id="US-6410516-B1-CLM-00027" class="claim">
      <div class="claim-text">27. The method of <claim-ref idref="US-6410516-B1-CLM-00001">claim 1</claim-ref>, carried out on human cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="28" id="US-6410516-B1-CLM-00028" class="claim">
      <div class="claim-text">28. The method of <claim-ref idref="US-6410516-B1-CLM-00026">claim 26</claim-ref> or <claim-ref idref="US-6410516-B1-CLM-00027">27</claim-ref>, carried out on immune cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="29" id="US-6410516-B1-CLM-00029" class="claim">
      <div class="claim-text">29. The-method of <claim-ref idref="US-6410516-B1-CLM-00026">claim 26</claim-ref> or <claim-ref idref="US-6410516-B1-CLM-00027">27</claim-ref> carried out on lymphoid cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="30" id="US-6410516-B1-CLM-00030" class="claim">
      <div class="claim-text">30. The method of <claim-ref idref="US-6410516-B1-CLM-00026">claim 26</claim-ref> or <claim-ref idref="US-6410516-B1-CLM-00027">27</claim-ref>, carried out on liver cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="31" id="US-6410516-B1-CLM-00031" class="claim">
      <div class="claim-text">31. The method of <claim-ref idref="US-6410516-B1-CLM-00002">claim 2</claim-ref> wherein NF-κB activity is reduced by decreasing the level of NF-κB not bound in an NF-κB:IκB complex.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="32" id="US-6410516-B1-CLM-00032" class="claim">
      <div class="claim-text">32. The method of <claim-ref idref="US-6410516-B1-CLM-00002">claim 2</claim-ref> wherein NF-κB activity is reduced by inhibiting the passage of NF-κB into the nucleus of cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="33" id="US-6410516-B1-CLM-00033" class="claim">
      <div class="claim-text">33. The method of <claim-ref idref="US-6410516-B1-CLM-00002">claim 2</claim-ref>, wherein NF-κB activity is reduced by inhibiting modification of an I-κB protein, which modification otherwise reduces IκB binding to NF-κB.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="34" id="US-6410516-B1-CLM-00034" class="claim">
      <div class="claim-text">34. The method of <claim-ref idref="US-6410516-B1-CLM-00002">claim 2</claim-ref>, wherein NF-κB activity is reduced by inhibiting degradation of an IκB protein.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="35" id="US-6410516-B1-CLM-00035" class="claim">
      <div class="claim-text">35. The method of <claim-ref idref="US-6410516-B1-CLM-00002">claim 2</claim-ref>, wherein NF-κB activity is reduced by inhibiting dissociation of NF-κB:IκB complexes.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="36" id="US-6410516-B1-CLM-00036" class="claim">
      <div class="claim-text">36. The method of <claim-ref idref="US-6410516-B1-CLM-00002">claim 2</claim-ref> wherein reducing NF-κB activity comprises reducing binding of NF-κB to NF-κB recognition sites on genes which are transcriptionally regulated by NF-κB.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="37" id="US-6410516-B1-CLM-00037" class="claim">
      <div class="claim-text">37. The method of <claim-ref idref="US-6410516-B1-CLM-00002">claim 2</claim-ref>, carried out on mammalian cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="38" id="US-6410516-B1-CLM-00038" class="claim">
      <div class="claim-text">38. The method of <claim-ref idref="US-6410516-B1-CLM-00002">claim 2</claim-ref>, carried out on human cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="39" id="US-6410516-B1-CLM-00039" class="claim">
      <div class="claim-text">39. The method of <claim-ref idref="US-6410516-B1-CLM-00037">claim 37</claim-ref> or <claim-ref idref="US-6410516-B1-CLM-00038">38</claim-ref>, carried out on immune cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="40" id="US-6410516-B1-CLM-00040" class="claim">
      <div class="claim-text">40. The method of <claim-ref idref="US-6410516-B1-CLM-00037">claim 37</claim-ref> or <claim-ref idref="US-6410516-B1-CLM-00038">38</claim-ref>, carried out on lymphoid cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="41" id="US-6410516-B1-CLM-00041" class="claim">
      <div class="claim-text">41. The method of <claim-ref idref="US-6410516-B1-CLM-00037">claim 37</claim-ref> or <claim-ref idref="US-6410516-B1-CLM-00038">38</claim-ref>, carried out on liver cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="42" id="US-6410516-B1-CLM-00042" class="claim">
      <div class="claim-text">42. The method of <claim-ref idref="US-6410516-B1-CLM-00003">claim 3</claim-ref> wherein NF-κB activity is reduced by decreasing the level of NF-κB not bound in an NF-κB:IκB complex.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="43" id="US-6410516-B1-CLM-00043" class="claim">
      <div class="claim-text">43. The method of <claim-ref idref="US-6410516-B1-CLM-00003">claim 3</claim-ref> wherein NF-κB activity is reduced by inhibiting the passage of NF-κB into the nucleus of cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="44" id="US-6410516-B1-CLM-00044" class="claim">
      <div class="claim-text">44. The method of <claim-ref idref="US-6410516-B1-CLM-00003">claim 3</claim-ref>, wherein NF-κB activity is reduced by inhibiting modification of an I-κB protein, which modification otherwise reduces IκB binding to NF-κB.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="45" id="US-6410516-B1-CLM-00045" class="claim">
      <div class="claim-text">45. The method of <claim-ref idref="US-6410516-B1-CLM-00003">claim 3</claim-ref>, wherein NF-κB activity is reduced by inhibiting degradation of an IκB protein.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="46" id="US-6410516-B1-CLM-00046" class="claim">
      <div class="claim-text">46. The method of <claim-ref idref="US-6410516-B1-CLM-00003">claim 3</claim-ref>, wherein NF-κB activity is reduced by inhibiting dissociation of NF-κB:IκB complexes.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="47" id="US-6410516-B1-CLM-00047" class="claim">
      <div class="claim-text">47. The method of <claim-ref idref="US-6410516-B1-CLM-00003">claim 3</claim-ref> wherein reducing NF-κB activity comprises reducing binding of NF-κB to NF-κB recognition sites on genes which are transcriptionally regulated by NF-κB.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="48" id="US-6410516-B1-CLM-00048" class="claim">
      <div class="claim-text">48. The method of <claim-ref idref="US-6410516-B1-CLM-00003">claim 3</claim-ref>, carried out on mammalian cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="49" id="US-6410516-B1-CLM-00049" class="claim">
      <div class="claim-text">49. The method of <claim-ref idref="US-6410516-B1-CLM-00003">claim 3</claim-ref>, carried out on human cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="50" id="US-6410516-B1-CLM-00050" class="claim">
      <div class="claim-text">50. The method of <claim-ref idref="US-6410516-B1-CLM-00048">claim 48</claim-ref> or <claim-ref idref="US-6410516-B1-CLM-00049">49</claim-ref>, carried out on immune cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="51" id="US-6410516-B1-CLM-00051" class="claim">
      <div class="claim-text">51. The method of <claim-ref idref="US-6410516-B1-CLM-00048">claim 48</claim-ref> or <claim-ref idref="US-6410516-B1-CLM-00049">49</claim-ref>, carried out on lymphoid cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="52" id="US-6410516-B1-CLM-00052" class="claim">
      <div class="claim-text">52. The method of <claim-ref idref="US-6410516-B1-CLM-00048">claim 48</claim-ref> or <claim-ref idref="US-6410516-B1-CLM-00049">49</claim-ref>, carried out on liver cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="53" id="US-6410516-B1-CLM-00053" class="claim">
      <div class="claim-text">53. The method of <claim-ref idref="US-6410516-B1-CLM-00005">claim 5</claim-ref> wherein NF-κB activity is reduced by decreasing the level of NF-κB not bound in an NF-κB:IκB complex.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="54" id="US-6410516-B1-CLM-00054" class="claim">
      <div class="claim-text">54. The method of <claim-ref idref="US-6410516-B1-CLM-00005">claim 5</claim-ref> wherein NF-κB activity is reduced by inhibiting the passage of NF-κB into the nucleus of cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="55" id="US-6410516-B1-CLM-00055" class="claim">
      <div class="claim-text">55. The method of <claim-ref idref="US-6410516-B1-CLM-00005">claim 5</claim-ref>, wherein NF-κB activity is reduced by inhibiting modification of an IκB protein, which modification otherwise reduces IκB binding to NF-κB.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="56" id="US-6410516-B1-CLM-00056" class="claim">
      <div class="claim-text">56. The method of <claim-ref idref="US-6410516-B1-CLM-00005">claim 5</claim-ref>, wherein NF-κB activity is reduced by inhibiting degradation of an IκB protein.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="57" id="US-6410516-B1-CLM-00057" class="claim">
      <div class="claim-text">57. The method of <claim-ref idref="US-6410516-B1-CLM-00005">claim 5</claim-ref>, wherein NF-κB activity is reduced by inhibiting dissociation of NF-κB:IκB complexes.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="58" id="US-6410516-B1-CLM-00058" class="claim">
      <div class="claim-text">58. The method of <claim-ref idref="US-6410516-B1-CLM-00005">claim 5</claim-ref> wherein reducing NF-κB activity comprises reducing binding of NF-κB to NF-κB recognition sites on genes which are transcriptionally regulated by NF-κB.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="59" id="US-6410516-B1-CLM-00059" class="claim">
      <div class="claim-text">59. The method of <claim-ref idref="US-6410516-B1-CLM-00005">claim 5</claim-ref>, carried out on mammalian cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="60" id="US-6410516-B1-CLM-00060" class="claim">
      <div class="claim-text">60. The method of <claim-ref idref="US-6410516-B1-CLM-00005">claim 5</claim-ref>, carried out on human cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="61" id="US-6410516-B1-CLM-00061" class="claim">
      <div class="claim-text">61. The method of <claim-ref idref="US-6410516-B1-CLM-00059">claim 59</claim-ref> or <claim-ref idref="US-6410516-B1-CLM-00060">60</claim-ref>, carried out on immune cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="62" id="US-6410516-B1-CLM-00062" class="claim">
      <div class="claim-text">62. The method of <claim-ref idref="US-6410516-B1-CLM-00059">claim 59</claim-ref> or <claim-ref idref="US-6410516-B1-CLM-00060">60</claim-ref>, carried out on lymphoid cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="63" id="US-6410516-B1-CLM-00063" class="claim">
      <div class="claim-text">63. The method of <claim-ref idref="US-6410516-B1-CLM-00059">claim 59</claim-ref> or <claim-ref idref="US-6410516-B1-CLM-00060">60</claim-ref>, carried out on liver cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="64" id="US-6410516-B1-CLM-00064" class="claim">
      <div class="claim-text">64. The method of <claim-ref idref="US-6410516-B1-CLM-00006">claim 6</claim-ref> wherein NF-κB activity is reduced by decreasing the level of NF-κB not bound in an NF-κB:IκB complex.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="65" id="US-6410516-B1-CLM-00065" class="claim">
      <div class="claim-text">65. The method of <claim-ref idref="US-6410516-B1-CLM-00006">claim 6</claim-ref> wherein NF-κB activity is reduced by inhibiting the passage of NF-κB into the nucleus of cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="66" id="US-6410516-B1-CLM-00066" class="claim">
      <div class="claim-text">66. The method of <claim-ref idref="US-6410516-B1-CLM-00006">claim 6</claim-ref>, wherein NF-κB activity is reduced by inhibiting modification of an IκB protein, which modification otherwise reduces IκB binding to NF-κB.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="67" id="US-6410516-B1-CLM-00067" class="claim">
      <div class="claim-text">67. The method of <claim-ref idref="US-6410516-B1-CLM-00006">claim 6</claim-ref>, wherein NF-κB activity is reduced by inhibiting degradation of an IκB protein.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="68" id="US-6410516-B1-CLM-00068" class="claim">
      <div class="claim-text">68. The method of <claim-ref idref="US-6410516-B1-CLM-00006">claim 6</claim-ref>, wherein NF-κB activity is reduced by inhibiting dissociation of NF-κB:IκB complexes.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="69" id="US-6410516-B1-CLM-00069" class="claim">
      <div class="claim-text">69. The method of <claim-ref idref="US-6410516-B1-CLM-00006">claim 6</claim-ref> wherein reducing NF-κB activity comprises reducing binding of NF-κB to NF-κB recognition sites on genes which are transcriptionally regulated by NF-κB.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="70" id="US-6410516-B1-CLM-00070" class="claim">
      <div class="claim-text">70. The method of <claim-ref idref="US-6410516-B1-CLM-00006">claim 6</claim-ref>, carried out on mammalian cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="71" id="US-6410516-B1-CLM-00071" class="claim">
      <div class="claim-text">71. The method of <claim-ref idref="US-6410516-B1-CLM-00006">claim 6</claim-ref>, carried out on human cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="72" id="US-6410516-B1-CLM-00072" class="claim">
      <div class="claim-text">72. The method of <claim-ref idref="US-6410516-B1-CLM-00070">claim 70</claim-ref> or <claim-ref idref="US-6410516-B1-CLM-00071">71</claim-ref>, carried out on immune cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="73" id="US-6410516-B1-CLM-00073" class="claim">
      <div class="claim-text">73. The method of <claim-ref idref="US-6410516-B1-CLM-00070">claim 70</claim-ref> or <claim-ref idref="US-6410516-B1-CLM-00071">71</claim-ref>, carried out on lymphoid cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="74" id="US-6410516-B1-CLM-00074" class="claim">
      <div class="claim-text">74. The method of <claim-ref idref="US-6410516-B1-CLM-00070">claim 70</claim-ref> or <claim-ref idref="US-6410516-B1-CLM-00071">71</claim-ref>, carried out on liver cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="75" id="US-6410516-B1-CLM-00075" class="claim">
      <div class="claim-text">75. The method of <claim-ref idref="US-6410516-B1-CLM-00008">claim 8</claim-ref> wherein NF-κB activity is reduced by decreasing the level of NF-κB not bound in an NF-κB:IκB complex.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="76" id="US-6410516-B1-CLM-00076" class="claim">
      <div class="claim-text">76. The method of <claim-ref idref="US-6410516-B1-CLM-00008">claim 8</claim-ref> wherein NF-κKB activity is reduced by inhibiting the passage of NF-κB into the nucleus of cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="77" id="US-6410516-B1-CLM-00077" class="claim">
      <div class="claim-text">77. The method of <claim-ref idref="US-6410516-B1-CLM-00008">claim 8</claim-ref>, wherein NF-κB activity is reduced by inhibiting modification of an IκB protein, which modification otherwise reduces IκB binding to NF-κB.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="78" id="US-6410516-B1-CLM-00078" class="claim">
      <div class="claim-text">78. The method of <claim-ref idref="US-6410516-B1-CLM-00008">claim 8</claim-ref>, wherein NF-κB activity is reduced by inhibiting degradation of an IκB protein.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="79" id="US-6410516-B1-CLM-00079" class="claim">
      <div class="claim-text">79. The method of <claim-ref idref="US-6410516-B1-CLM-00008">claim 8</claim-ref>, wherein NF-κB activity is reduced by inhibiting dissociation of NF-κB:IκB complexes.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="80" id="US-6410516-B1-CLM-00080" class="claim">
      <div class="claim-text">80. The method of <claim-ref idref="US-6410516-B1-CLM-00008">claim 8</claim-ref> wherein reducing NF-κB activity comprises reducing binding of NF-κB to NF-κB recognition sites on genes which are transcriptionally regulated by NF-κB.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="81" id="US-6410516-B1-CLM-00081" class="claim">
      <div class="claim-text">81. The method of <claim-ref idref="US-6410516-B1-CLM-00007">claim 7</claim-ref>, carried out on mammalian cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="82" id="US-6410516-B1-CLM-00082" class="claim">
      <div class="claim-text">82. The method of <claim-ref idref="US-6410516-B1-CLM-00008">claim 8</claim-ref>, carried out on mammalian cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="83" id="US-6410516-B1-CLM-00083" class="claim">
      <div class="claim-text">83. The method of <claim-ref idref="US-6410516-B1-CLM-00007">claim 7</claim-ref>, carried out on human cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="84" id="US-6410516-B1-CLM-00084" class="claim">
      <div class="claim-text">84. The method of <claim-ref idref="US-6410516-B1-CLM-00008">claim 8</claim-ref>, carried out on human cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="85" id="US-6410516-B1-CLM-00085" class="claim">
      <div class="claim-text">85. The method of any of claims <b>81</b>-<b>84</b>, carried out on immune cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="86" id="US-6410516-B1-CLM-00086" class="claim">
      <div class="claim-text">86. The method of any of claims <b>81</b>-<b>84</b>, carried out on lymphoid cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="87" id="US-6410516-B1-CLM-00087" class="claim">
      <div class="claim-text">87. The method of any of claims <b>81</b>-<b>84</b>, carried out on liver cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="88" id="US-6410516-B1-CLM-00088" class="claim">
      <div class="claim-text">88. The method of <claim-ref idref="US-6410516-B1-CLM-00009">claim 9</claim-ref> wherein NF-κB activity is reduced by decreasing the level of NF-κB not bound in an NF-κB:IκB complex.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="89" id="US-6410516-B1-CLM-00089" class="claim">
      <div class="claim-text">89. The method of <claim-ref idref="US-6410516-B1-CLM-00009">claim 9</claim-ref> wherein NF-κB activity is reduced by inhibiting the passage of NF-κB into the nucleus of cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="90" id="US-6410516-B1-CLM-00090" class="claim">
      <div class="claim-text">90. The method of <claim-ref idref="US-6410516-B1-CLM-00009">claim 9</claim-ref>, wherein NF-κB activity is reduced by inhibiting modification of an IκB protein, which modification otherwise reduces IκB binding to NF-κB.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="91" id="US-6410516-B1-CLM-00091" class="claim">
      <div class="claim-text">91. The method of <claim-ref idref="US-6410516-B1-CLM-00009">claim 9</claim-ref>, wherein NF-κB activity is reduced by inhibiting degradation of an IκB protein.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="92" id="US-6410516-B1-CLM-00092" class="claim">
      <div class="claim-text">92. The method of <claim-ref idref="US-6410516-B1-CLM-00009">claim 9</claim-ref>, wherein NF-κB activity is reduced by inhibiting dissociation of NF-κB:IκB complexes.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="93" id="US-6410516-B1-CLM-00093" class="claim">
      <div class="claim-text">93. The method of <claim-ref idref="US-6410516-B1-CLM-00009">claim 9</claim-ref> wherein reducing NF-κB activity comprises reducing binding of NF-κB to NF-κB recognition sites on genes which are transcriptionally regulated by NF-κB.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="94" id="US-6410516-B1-CLM-00094" class="claim">
      <div class="claim-text">94. The method of <claim-ref idref="US-6410516-B1-CLM-00009">claim 9</claim-ref>, carried out on mammalian cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="95" id="US-6410516-B1-CLM-00095" class="claim">
      <div class="claim-text">95. The method of <claim-ref idref="US-6410516-B1-CLM-00009">claim 9</claim-ref>, carried out on human cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="96" id="US-6410516-B1-CLM-00096" class="claim">
      <div class="claim-text">96. The method of <claim-ref idref="US-6410516-B1-CLM-00094">claim 94</claim-ref> or <claim-ref idref="US-6410516-B1-CLM-00095">95</claim-ref>, carried out on immune cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="97" id="US-6410516-B1-CLM-00097" class="claim">
      <div class="claim-text">97. The method of <claim-ref idref="US-6410516-B1-CLM-00094">claim 94</claim-ref> or <claim-ref idref="US-6410516-B1-CLM-00095">95</claim-ref>, carried out on lymphoid cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="98" id="US-6410516-B1-CLM-00098" class="claim">
      <div class="claim-text">98. The method of <claim-ref idref="US-6410516-B1-CLM-00094">claim 94</claim-ref> or <claim-ref idref="US-6410516-B1-CLM-00095">95</claim-ref>, carried out on liver cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="99" id="US-6410516-B1-CLM-00099" class="claim">
      <div class="claim-text">99. The method of <claim-ref idref="US-6410516-B1-CLM-00010">claim 10</claim-ref> wherein NF-κB activity is reduced by decreasing the level of NF-κB not bound in an NF-κB:IκB complex.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="100" id="US-6410516-B1-CLM-00100" class="claim">
      <div class="claim-text">100. The method of <claim-ref idref="US-6410516-B1-CLM-00010">claim 10</claim-ref> wherein NF-κB activity is reduced by inhibiting the passage of NF-κB into the nucleus of cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="101" id="US-6410516-B1-CLM-00101" class="claim">
      <div class="claim-text">101. The method of <claim-ref idref="US-6410516-B1-CLM-00010">claim 10</claim-ref>, wherein NF-κB activity is reduced by inhibiting modification of an IκB protein, which modification otherwise reduces IκB binding to NF-κB.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="102" id="US-6410516-B1-CLM-00102" class="claim">
      <div class="claim-text">102. The method of <claim-ref idref="US-6410516-B1-CLM-00010">claim 10</claim-ref>, wherein NF-κB activity is reduced by inhibiting degradation of an IκB protein.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="103" id="US-6410516-B1-CLM-00103" class="claim">
      <div class="claim-text">103. The method of <claim-ref idref="US-6410516-B1-CLM-00010">claim 10</claim-ref>, wherein NF-κB activity is reduced by inhibiting dissociation of NF-κB:IκB complexes.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="104" id="US-6410516-B1-CLM-00104" class="claim">
      <div class="claim-text">104. The method of <claim-ref idref="US-6410516-B1-CLM-00010">claim 10</claim-ref> wherein reducing NF-κB activity comprises reducing binding of NF-κB to NF-κB recognition sites on genes which are transcriptionally regulated by NF-κB.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="105" id="US-6410516-B1-CLM-00105" class="claim">
      <div class="claim-text">105. The method of <claim-ref idref="US-6410516-B1-CLM-00010">claim 10</claim-ref>, carried out on human cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="106" id="US-6410516-B1-CLM-00106" class="claim">
      <div class="claim-text">106. The method of <claim-ref idref="US-6410516-B1-CLM-00011">claim 11</claim-ref>, carried out on immune cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="107" id="US-6410516-B1-CLM-00107" class="claim">
      <div class="claim-text">107. The method of <claim-ref idref="US-6410516-B1-CLM-00011">claim 11</claim-ref>, carried out on lymphoid cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="108" id="US-6410516-B1-CLM-00108" class="claim">
      <div class="claim-text">108. The method of <claim-ref idref="US-6410516-B1-CLM-00011">claim 11</claim-ref>, carried out on liver cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="109" id="US-6410516-B1-CLM-00109" class="claim">
      <div class="claim-text">109. The method of <claim-ref idref="US-6410516-B1-CLM-00011">claim 11</claim-ref> wherein NF-κB activity is reduced by decreasing the level of NF-κB not bound in an NF-κB:IκB complex.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="110" id="US-6410516-B1-CLM-00110" class="claim">
      <div class="claim-text">110. The method of <claim-ref idref="US-6410516-B1-CLM-00011">claim 11</claim-ref> wherein NF-κB activity is reduced by inhibiting the passage of NF-κB into the nucleus of cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="111" id="US-6410516-B1-CLM-00111" class="claim">
      <div class="claim-text">111. The method of <claim-ref idref="US-6410516-B1-CLM-00011">claim 11</claim-ref>, wherein NF-κB activity is reduced by inhibiting modification of an IκB protein, which modification otherwise reduces IκB binding to NF-κB.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="112" id="US-6410516-B1-CLM-00112" class="claim">
      <div class="claim-text">112. The method of <claim-ref idref="US-6410516-B1-CLM-00011">claim 11</claim-ref>, wherein NF-κB activity is reduced by inhibiting degradation of an IκB protein.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="113" id="US-6410516-B1-CLM-00113" class="claim">
      <div class="claim-text">113. The method of <claim-ref idref="US-6410516-B1-CLM-00011">claim 11</claim-ref>, wherein NF-κB activity is reduced by inhibiting dissociation of NF-κB:IκB complexes.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="114" id="US-6410516-B1-CLM-00114" class="claim">
      <div class="claim-text">114. The method of <claim-ref idref="US-6410516-B1-CLM-00011">claim 11</claim-ref> wherein reducing NF-κB activity comprises reducing binding of NF-κB to NF-κB recognition sites on genes which are transcriptionally regulated by NF-κB.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="115" id="US-6410516-B1-CLM-00115" class="claim">
      <div class="claim-text">115. The method of <claim-ref idref="US-6410516-B1-CLM-00011">claim 11</claim-ref>, carried out on human cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="116" id="US-6410516-B1-CLM-00116" class="claim">
      <div class="claim-text">116. The method of <claim-ref idref="US-6410516-B1-CLM-00011">claim 11</claim-ref> or <claim-ref idref="US-6410516-B1-CLM-00115">115</claim-ref>, carried out on immune cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="117" id="US-6410516-B1-CLM-00117" class="claim">
      <div class="claim-text">117. The method of <claim-ref idref="US-6410516-B1-CLM-00011">claim 11</claim-ref> or <claim-ref idref="US-6410516-B1-CLM-00115">115</claim-ref>, carried out on lymphoid cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="118" id="US-6410516-B1-CLM-00118" class="claim">
      <div class="claim-text">118. The method of <claim-ref idref="US-6410516-B1-CLM-00011">claim 11</claim-ref> or <claim-ref idref="US-6410516-B1-CLM-00115">115</claim-ref>, carried out on liver cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="119" id="US-6410516-B1-CLM-00119" class="claim">
      <div class="claim-text">119. The method of <claim-ref idref="US-6410516-B1-CLM-00012">claim 12</claim-ref> wherein NF-κB activity is reduced by decreasing the level of NF-κB not bound in an NF-κB:IκB complex.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="120" id="US-6410516-B1-CLM-00120" class="claim">
      <div class="claim-text">120. The method of <claim-ref idref="US-6410516-B1-CLM-00012">claim 12</claim-ref> wherein NF-κB activity is reduced by inhibiting the passage of NF-κB into the nucleus of cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="121" id="US-6410516-B1-CLM-00121" class="claim">
      <div class="claim-text">121. The method of <claim-ref idref="US-6410516-B1-CLM-00012">claim 12</claim-ref>, wherein NF-κB activity is reduced by inhibiting modification of an IκB protein, which modification otherwise reduces IκB binding to NF-κB.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="122" id="US-6410516-B1-CLM-00122" class="claim">
      <div class="claim-text">122. The method of <claim-ref idref="US-6410516-B1-CLM-00012">claim 12</claim-ref>, wherein NF-κB activity is reduced by inhibiting degradation of an IκB protein.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="123" id="US-6410516-B1-CLM-00123" class="claim">
      <div class="claim-text">123. The method of <claim-ref idref="US-6410516-B1-CLM-00012">claim 12</claim-ref>, wherein NF-κB activity is reduced by inhibiting dissociation of NF-κB:IκB complexes.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="124" id="US-6410516-B1-CLM-00124" class="claim">
      <div class="claim-text">124. The method of <claim-ref idref="US-6410516-B1-CLM-00012">claim 12</claim-ref> wherein reducing NF-κB activity comprises reducing binding of NF-κB to NF-κB recognition sites on genes which are transcriptionally regulated by NF-κB.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="125" id="US-6410516-B1-CLM-00125" class="claim">
      <div class="claim-text">125. The method of <claim-ref idref="US-6410516-B1-CLM-00012">claim 12</claim-ref>, carried out on human immune cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="126" id="US-6410516-B1-CLM-00126" class="claim">
      <div class="claim-text">126. The method of <claim-ref idref="US-6410516-B1-CLM-00012">claim 12</claim-ref>, carried out on lymphoid cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="127" id="US-6410516-B1-CLM-00127" class="claim">
      <div class="claim-text">127. The method of <claim-ref idref="US-6410516-B1-CLM-00125">claim 125</claim-ref>, carried out on lymphoid cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="128" id="US-6410516-B1-CLM-00128" class="claim">
      <div class="claim-text">128. The method of <claim-ref idref="US-6410516-B1-CLM-00013">claim 13</claim-ref> wherein NF-κB activity is reduced by decreasing the level of NF-κB not bound in an NF-κB:IκB complex.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="129" id="US-6410516-B1-CLM-00129" class="claim">
      <div class="claim-text">129. The method of <claim-ref idref="US-6410516-B1-CLM-00013">claim 13</claim-ref> wherein NF-κB activity is reduced by inhibiting the passage of NF-κB into the nucleus of cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="130" id="US-6410516-B1-CLM-00130" class="claim">
      <div class="claim-text">130. The method of <claim-ref idref="US-6410516-B1-CLM-00013">claim 13</claim-ref>, wherein NF-κB activity is reduced by inhibiting modification of an IκB protein, which modification otherwise reduces IκB binding to NF-κB.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="131" id="US-6410516-B1-CLM-00131" class="claim">
      <div class="claim-text">131. the method of <claim-ref idref="US-6410516-B1-CLM-00013">claim 13</claim-ref>, wherein NF-κB activity is reduced by inhibiting degradation of an IκB protein.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="132" id="US-6410516-B1-CLM-00132" class="claim">
      <div class="claim-text">132. The method of <claim-ref idref="US-6410516-B1-CLM-00013">claim 13</claim-ref>, wherein NF-κB activity is reduced by inhibiting dissociation of NF-κB:IκB complexes.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="133" id="US-6410516-B1-CLM-00133" class="claim">
      <div class="claim-text">133. The method of <claim-ref idref="US-6410516-B1-CLM-00013">claim 13</claim-ref> wherein reducing NF-κB activity comprises reducing binding of NF-κB to NF-κB recognition sites on genes which are transcriptionally regulated by NF-κB.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="134" id="US-6410516-B1-CLM-00134" class="claim">
      <div class="claim-text">134. The method of <claim-ref idref="US-6410516-B1-CLM-00013">claim 13</claim-ref>, carried out on human cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="135" id="US-6410516-B1-CLM-00135" class="claim">
      <div class="claim-text">135. The method of <claim-ref idref="US-6410516-B1-CLM-00013">claim 13</claim-ref>, carried out on immune cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="136" id="US-6410516-B1-CLM-00136" class="claim">
      <div class="claim-text">136. The method of <claim-ref idref="US-6410516-B1-CLM-00134">claim 134</claim-ref>, carried out on immune cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="137" id="US-6410516-B1-CLM-00137" class="claim">
      <div class="claim-text">137. The method of <claim-ref idref="US-6410516-B1-CLM-00013">claim 13</claim-ref> or <claim-ref idref="US-6410516-B1-CLM-00134">134</claim-ref>, carried out on lymphoid cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="138" id="US-6410516-B1-CLM-00138" class="claim">
      <div class="claim-text">138. The method of <claim-ref idref="US-6410516-B1-CLM-00013">claim 13</claim-ref> or <claim-ref idref="US-6410516-B1-CLM-00134">134</claim-ref>, carried out on liver cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="139" id="US-6410516-B1-CLM-00139" class="claim">
      <div class="claim-text">139. The method of <claim-ref idref="US-6410516-B1-CLM-00014">claim 14</claim-ref> wherein NF-κB activity is reduced by decreasing the level of NF-κB not bound in an NF-κB:IκB complex.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="140" id="US-6410516-B1-CLM-00140" class="claim">
      <div class="claim-text">140. The method of <claim-ref idref="US-6410516-B1-CLM-00014">claim 14</claim-ref> wherein NF-κB activity is reduced by inhibiting the passage of NF-κB into the nucleus of cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="141" id="US-6410516-B1-CLM-00141" class="claim">
      <div class="claim-text">141. The method of <claim-ref idref="US-6410516-B1-CLM-00014">claim 14</claim-ref>, wherein NF-κB activity is reduced by inhibiting modification of an IκB protein, which modification otherwise reduces IκB binding to NF-κB.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="142" id="US-6410516-B1-CLM-00142" class="claim">
      <div class="claim-text">142. The method of <claim-ref idref="US-6410516-B1-CLM-00014">claim 14</claim-ref>, wherein NF-κB activity is reduced by inhibiting degradation of an IκB protein.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="143" id="US-6410516-B1-CLM-00143" class="claim">
      <div class="claim-text">143. The method of <claim-ref idref="US-6410516-B1-CLM-00014">claim 14</claim-ref>, wherein NF-κB activity is reduced by inhibiting dissociation of NF-κB:IκB complexes.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="144" id="US-6410516-B1-CLM-00144" class="claim">
      <div class="claim-text">144. The method of <claim-ref idref="US-6410516-B1-CLM-00014">claim 14</claim-ref> wherein reducing NF-κB activity comprises reducing binding of NF-κB to NF-κB recognition sites on genes which are transcriptionally regulated by NF-κB.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="145" id="US-6410516-B1-CLM-00145" class="claim">
      <div class="claim-text">145. The method of <claim-ref idref="US-6410516-B1-CLM-00014">claim 14</claim-ref>, carried out on human cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="146" id="US-6410516-B1-CLM-00146" class="claim">
      <div class="claim-text">146. The method of <claim-ref idref="US-6410516-B1-CLM-00014">claim 14</claim-ref> or <claim-ref idref="US-6410516-B1-CLM-00145">145</claim-ref>, carried out on immune cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="147" id="US-6410516-B1-CLM-00147" class="claim">
      <div class="claim-text">147. The method of <claim-ref idref="US-6410516-B1-CLM-00014">claim 14</claim-ref> or <claim-ref idref="US-6410516-B1-CLM-00145">145</claim-ref>, carried out on lymphoid cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="148" id="US-6410516-B1-CLM-00148" class="claim">
      <div class="claim-text">148. The method of <claim-ref idref="US-6410516-B1-CLM-00014">claim 14</claim-ref> or <claim-ref idref="US-6410516-B1-CLM-00145">145</claim-ref>, carried out on liver cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="149" id="US-6410516-B1-CLM-00149" class="claim">
      <div class="claim-text">149. The method of <claim-ref idref="US-6410516-B1-CLM-00015">claim 15</claim-ref> wherein NF-κB activity is reduced by decreasing the level of NF-κB not bound in an NF-κB:IκB complex.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="150" id="US-6410516-B1-CLM-00150" class="claim">
      <div class="claim-text">150. The method of <claim-ref idref="US-6410516-B1-CLM-00015">claim 15</claim-ref> wherein NF-κB activity is reduced by inhibiting the passage of NF-κB into the nucleus of cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="151" id="US-6410516-B1-CLM-00151" class="claim">
      <div class="claim-text">151. The method of <claim-ref idref="US-6410516-B1-CLM-00015">claim 15</claim-ref>, wherein NF-κB activity is reduced by inhibiting modification of an IκB protein, which modification otherwise reduces IκB binding to NF-κB.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="152" id="US-6410516-B1-CLM-00152" class="claim">
      <div class="claim-text">152. The method of <claim-ref idref="US-6410516-B1-CLM-00015">claim 15</claim-ref>, wherein NF-κB activity is reduced by inhibiting degradation of an IκB protein.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="153" id="US-6410516-B1-CLM-00153" class="claim">
      <div class="claim-text">153. The method of <claim-ref idref="US-6410516-B1-CLM-00015">claim 15</claim-ref>, wherein NF-κB activity is reduced by inhibiting dissociation of NF-κB:IκB complexes.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="154" id="US-6410516-B1-CLM-00154" class="claim">
      <div class="claim-text">154. The method of <claim-ref idref="US-6410516-B1-CLM-00015">claim 15</claim-ref> wherein reducing NF-κB activity comprises reducing binding of NF-κB to NF-κB recognition sites on genes which are transcriptionally regulated by NF-κB.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="155" id="US-6410516-B1-CLM-00155" class="claim">
      <div class="claim-text">155. The method of <claim-ref idref="US-6410516-B1-CLM-00015">claim 15</claim-ref>, carried out on human cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="156" id="US-6410516-B1-CLM-00156" class="claim">
      <div class="claim-text">156. The method of <claim-ref idref="US-6410516-B1-CLM-00015">claim 15</claim-ref> of <b>155</b>, carried out on immune cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="157" id="US-6410516-B1-CLM-00157" class="claim">
      <div class="claim-text">157. The method of <claim-ref idref="US-6410516-B1-CLM-00015">claim 15</claim-ref> or <claim-ref idref="US-6410516-B1-CLM-00155">155</claim-ref>, carried out on lymphoid cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="158" id="US-6410516-B1-CLM-00158" class="claim">
      <div class="claim-text">158. The method of <claim-ref idref="US-6410516-B1-CLM-00015">claim 15</claim-ref> or <claim-ref idref="US-6410516-B1-CLM-00155">155</claim-ref>, carried out on liver cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="159" id="US-6410516-B1-CLM-00159" class="claim">
      <div class="claim-text">159. The method of <claim-ref idref="US-6410516-B1-CLM-00016">claim 16</claim-ref> wherein NF-κB activity is reduced by decreasing the level of NF-κB not bound in an NF-κB:IκB complex.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="160" id="US-6410516-B1-CLM-00160" class="claim">
      <div class="claim-text">160. The method of <claim-ref idref="US-6410516-B1-CLM-00016">claim 16</claim-ref> wherein NF-κB activity is reduced by inhibiting the passage of NF-κB into the nucleus of cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="161" id="US-6410516-B1-CLM-00161" class="claim">
      <div class="claim-text">161. The method of <claim-ref idref="US-6410516-B1-CLM-00016">claim 16</claim-ref>, wherein NF-κB activity is reduced by inhibiting modification of an IκB protein, which modification otherwise reduces IκB binding to NF-κB.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="162" id="US-6410516-B1-CLM-00162" class="claim">
      <div class="claim-text">162. The method of <claim-ref idref="US-6410516-B1-CLM-00016">claim 16</claim-ref>, wherein NF-κB activity is reduced by inhibiting degradation of an IκB protein.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="163" id="US-6410516-B1-CLM-00163" class="claim">
      <div class="claim-text">163. The method of <claim-ref idref="US-6410516-B1-CLM-00016">claim 16</claim-ref>, wherein NF-κB activity is reduced by inhibiting dissociation of NF-κB:IκB complexes.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="164" id="US-6410516-B1-CLM-00164" class="claim">
      <div class="claim-text">164. The method of <claim-ref idref="US-6410516-B1-CLM-00016">claim 16</claim-ref> wherein reducing NF-κB activity comprises reducing binding of NF-κB to NF-κB recognition sites on genes which are transcriptionally regulated by NF-κB.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="165" id="US-6410516-B1-CLM-00165" class="claim">
      <div class="claim-text">165. The method of <claim-ref idref="US-6410516-B1-CLM-00016">claim 16</claim-ref>, carried out on human cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="166" id="US-6410516-B1-CLM-00166" class="claim">
      <div class="claim-text">166. The method of <claim-ref idref="US-6410516-B1-CLM-00016">claim 16</claim-ref> or <claim-ref idref="US-6410516-B1-CLM-00165">165</claim-ref>, carried out on lymphoid cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="167" id="US-6410516-B1-CLM-00167" class="claim">
      <div class="claim-text">167. The method of <claim-ref idref="US-6410516-B1-CLM-00017">claim 17</claim-ref> wherein NF-κB activity is reduced by decreasing the level of NF-κB not bound in an NF-κB:IκB complex.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="168" id="US-6410516-B1-CLM-00168" class="claim">
      <div class="claim-text">168. The method of <claim-ref idref="US-6410516-B1-CLM-00017">claim 17</claim-ref> wherein NF-κB activity is reduced by inhibiting the passage of NF-κB into the nucleus of cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="169" id="US-6410516-B1-CLM-00169" class="claim">
      <div class="claim-text">169. The method of <claim-ref idref="US-6410516-B1-CLM-00017">claim 17</claim-ref>, wherein NF-κB activity is reduced by inhibiting modification of an IκB protein, which modification otherwise reduces IκB binding to NF-κB.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="170" id="US-6410516-B1-CLM-00170" class="claim">
      <div class="claim-text">170. The method of <claim-ref idref="US-6410516-B1-CLM-00017">claim 17</claim-ref>, wherein NF-κB activity is reduced by inhibiting degradation of an IκB protein.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="171" id="US-6410516-B1-CLM-00171" class="claim">
      <div class="claim-text">171. The method of <claim-ref idref="US-6410516-B1-CLM-00017">claim 17</claim-ref>, wherein NF-κB activity is reduced by inhibiting dissociation of NF-κB:IκB complexes.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="172" id="US-6410516-B1-CLM-00172" class="claim">
      <div class="claim-text">172. The method of <claim-ref idref="US-6410516-B1-CLM-00017">claim 17</claim-ref> wherein reducing NF-κB activity comprises reducing binding of NF-κB to NF-κB recognition sites on genes which are transcriptionally regulated by NF-κB.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="173" id="US-6410516-B1-CLM-00173" class="claim">
      <div class="claim-text">173. The method of <claim-ref idref="US-6410516-B1-CLM-00017">claim 17</claim-ref>, carried out on human cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="174" id="US-6410516-B1-CLM-00174" class="claim">
      <div class="claim-text">174. The method of <claim-ref idref="US-6410516-B1-CLM-00017">claim 17</claim-ref> or <claim-ref idref="US-6410516-B1-CLM-00173">173</claim-ref>, carried out on immune cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="175" id="US-6410516-B1-CLM-00175" class="claim">
      <div class="claim-text">175. The method of <claim-ref idref="US-6410516-B1-CLM-00017">claim 17</claim-ref> or <claim-ref idref="US-6410516-B1-CLM-00173">173</claim-ref>, carried out on lymphoid cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="176" id="US-6410516-B1-CLM-00176" class="claim">
      <div class="claim-text">176. The method of <claim-ref idref="US-6410516-B1-CLM-00017">claim 17</claim-ref> or <claim-ref idref="US-6410516-B1-CLM-00173">173</claim-ref>, carried out on liver cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="177" id="US-6410516-B1-CLM-00177" class="claim">
      <div class="claim-text">177. The method of <claim-ref idref="US-6410516-B1-CLM-00018">claim 18</claim-ref> wherein NF-κB activity is reduced by decreasing the level of NF-κB not bound in an NF-κB:IκB complex.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="178" id="US-6410516-B1-CLM-00178" class="claim">
      <div class="claim-text">178. The method of <claim-ref idref="US-6410516-B1-CLM-00018">claim 18</claim-ref> wherein NF-κB activity is reduced by inhibiting the passage of NF-κB into the nucleus of cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="179" id="US-6410516-B1-CLM-00179" class="claim">
      <div class="claim-text">179. The method of <claim-ref idref="US-6410516-B1-CLM-00018">claim 18</claim-ref>, wherein NF-κB activity is reduced by inhibiting modification of an IκB protein, which modification otherwise reduces IκB binding to NF-κB.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="180" id="US-6410516-B1-CLM-00180" class="claim">
      <div class="claim-text">180. The method of <claim-ref idref="US-6410516-B1-CLM-00018">claim 18</claim-ref>, wherein NF-κB activity is reduced by inhibiting degradation of an IκB protein.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="181" id="US-6410516-B1-CLM-00181" class="claim">
      <div class="claim-text">181. The method of <claim-ref idref="US-6410516-B1-CLM-00018">claim 18</claim-ref>, wherein NF-κB activity is reduced by inhibiting dissociation of NF-κB:IκB complexes.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="182" id="US-6410516-B1-CLM-00182" class="claim">
      <div class="claim-text">182. The method of <claim-ref idref="US-6410516-B1-CLM-00018">claim 18</claim-ref> wherein reducing NF-κB activity comprises reducing binding of NF-κB to NF-κB recognition sites on genes which are transcriptionally regulated by NF-κB.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="183" id="US-6410516-B1-CLM-00183" class="claim">
      <div class="claim-text">183. The method of <claim-ref idref="US-6410516-B1-CLM-00018">claim 18</claim-ref>, carried out on human cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="184" id="US-6410516-B1-CLM-00184" class="claim">
      <div class="claim-text">184. The method of <claim-ref idref="US-6410516-B1-CLM-00018">claim 18</claim-ref> or <claim-ref idref="US-6410516-B1-CLM-00183">183</claim-ref>, carried out on immune cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="185" id="US-6410516-B1-CLM-00185" class="claim">
      <div class="claim-text">185. The method of <claim-ref idref="US-6410516-B1-CLM-00018">claim 18</claim-ref> or <claim-ref idref="US-6410516-B1-CLM-00183">183</claim-ref>, carried out on lymphoid cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="186" id="US-6410516-B1-CLM-00186" class="claim">
      <div class="claim-text">186. The method of <claim-ref idref="US-6410516-B1-CLM-00018">claim 18</claim-ref> or <claim-ref idref="US-6410516-B1-CLM-00183">183</claim-ref>, carried out on liver cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="187" id="US-6410516-B1-CLM-00187" class="claim">
      <div class="claim-text">187. The method of <claim-ref idref="US-6410516-B1-CLM-00019">claim 19</claim-ref>, carried out on mammalian cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="188" id="US-6410516-B1-CLM-00188" class="claim">
      <div class="claim-text">188. The method of <claim-ref idref="US-6410516-B1-CLM-00019">claim 19</claim-ref>, carried out on human cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="189" id="US-6410516-B1-CLM-00189" class="claim">
      <div class="claim-text">189. The method of <claim-ref idref="US-6410516-B1-CLM-00187">claim 187</claim-ref> or <claim-ref idref="US-6410516-B1-CLM-00188">188</claim-ref>, carried out on immune cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="190" id="US-6410516-B1-CLM-00190" class="claim">
      <div class="claim-text">190. The method of <claim-ref idref="US-6410516-B1-CLM-00187">claim 187</claim-ref> or <claim-ref idref="US-6410516-B1-CLM-00188">188</claim-ref>, carried out on lymphoid cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="191" id="US-6410516-B1-CLM-00191" class="claim">
      <div class="claim-text">191. The method of <claim-ref idref="US-6410516-B1-CLM-00193">claim 193</claim-ref> or <claim-ref idref="US-6410516-B1-CLM-00194">194</claim-ref>, carried out on liver cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="192" id="US-6410516-B1-CLM-00192" class="claim">
      <div class="claim-text">192. The method of <claim-ref idref="US-6410516-B1-CLM-00004">claim 4</claim-ref> wherein NF-κB activity is reduced by decreasing the level of NF-κB not bound in an NF-κB:IκB complex.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="193" id="US-6410516-B1-CLM-00193" class="claim">
      <div class="claim-text">193. The method of <claim-ref idref="US-6410516-B1-CLM-00004">claim 4</claim-ref> wherein NF-κB activity is reduced by inhibiting the passage of NF-κB into the nucleus of cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="194" id="US-6410516-B1-CLM-00194" class="claim">
      <div class="claim-text">194. The method of <claim-ref idref="US-6410516-B1-CLM-00004">claim 4</claim-ref>, wherein NF-κB activity is reduced by inhibiting modification of an IκB protein, which modification otherwise reduces IκB binding to NF-κB.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="195" id="US-6410516-B1-CLM-00195" class="claim">
      <div class="claim-text">195. The method of <claim-ref idref="US-6410516-B1-CLM-00004">claim 4</claim-ref>, wherein NF-κB activity is reduced by inhibiting degradation of an IκB protein.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="196" id="US-6410516-B1-CLM-00196" class="claim">
      <div class="claim-text">196. The method of <claim-ref idref="US-6410516-B1-CLM-00004">claim 4</claim-ref>, wherein NF-κB activity is reduced by inhibiting dissociation of NF-κB:IκB complexes.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="197" id="US-6410516-B1-CLM-00197" class="claim">
      <div class="claim-text">197. The method of <claim-ref idref="US-6410516-B1-CLM-00004">claim 4</claim-ref> wherein reducing NF-κB activity comprises reducing binding of NF-κB to NF-κB recognition sites on genes which are transcriptionally regulated by NF-κB.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="198" id="US-6410516-B1-CLM-00198" class="claim">
      <div class="claim-text">198. The method of <claim-ref idref="US-6410516-B1-CLM-00004">claim 4</claim-ref>, carried out on mammalian cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="199" id="US-6410516-B1-CLM-00199" class="claim">
      <div class="claim-text">199. The method of <claim-ref idref="US-6410516-B1-CLM-00004">claim 4</claim-ref>, carried out on human cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="200" id="US-6410516-B1-CLM-00200" class="claim">
      <div class="claim-text">200. The method of <claim-ref idref="US-6410516-B1-CLM-00198">claim 198</claim-ref> or <claim-ref idref="US-6410516-B1-CLM-00199">199</claim-ref>, carried out on immune cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="201" id="US-6410516-B1-CLM-00201" class="claim">
      <div class="claim-text">201. The method of <claim-ref idref="US-6410516-B1-CLM-00198">claim 198</claim-ref> or <claim-ref idref="US-6410516-B1-CLM-00199">199</claim-ref>, carried out on lymphoid cells.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="202" id="US-6410516-B1-CLM-00202" class="claim">
      <div class="claim-text">202. The method of <claim-ref idref="US-6410516-B1-CLM-00198">claim 198</claim-ref> or <claim-ref idref="US-6410516-B1-CLM-00199">199</claim-ref>, carried out on liver cells.</div>
    </div>
    </div> <div class="claim"> <div num="203" id="US-6410516-B1-CLM-00203" class="claim">
      <div class="claim-text">203. A method of inhibiting expression, in a mammalian cell, of a gene whose transcriptional activity is activated by binding of NF-κB to said gene, comprising introducing a nucleic acid decoy molecule into the cell in an amount sufficient to inhibit expression of the gene, which decoy includes a NF-κB binding site that binds to NF-κB.</div>
    </div>
  </div> </div></div></div><div class="patent-section patent-description-section"><div class="patent-section-header"><span class="patent-section-title">Description</span></div><div class="patent-text"><div mxw-id="PDES53588161" lang="EN" load-source="patent-office" class="description">
    <heading>RELATED APPLICATIONS</heading> <p>This application is a division of application Ser. No. 08/418,266 filed Apr. 6, 1995, U.S. Pat. No. 5,804,374 which is a continuation of U.S. Ser. No. 07/791,898, filed Nov. 13, 1991, abandoned which is a continuation-in-part of U.S. Ser. No. 06/946,365, filed Dec. 24, 1986, abandoned and of U.S. Ser. No. 07/318,901, filed Mar. 3, 1989, abandoned and of U.S. Ser. No. 07/162,680, filed Mar. 1, 1988, abandoned and of U.S. Ser. No. 07/341,436, filed Apr. 21, 1989, abandoned and of U.S. Ser. No. 06/817,441, filed Jan. 9, 1986, abandoned and of U.S. Ser. No. 07/155,207, filed Feb. 12, 1988, abandoned and of U.S. Ser. No. 07/280,173, filed Dec. 5, 1988, abandoned. All of the above applications are incorporated herein by reference in their entirety.</p>
    <heading>GOVERNMENT SUPPORT</heading> <p>The work leading to this invention was supported in part by a grant from the National Cancer Institute. The Government has certain rights in this invention.</p>
    <heading>BACKGROUND OF THE INVENTION</heading> <p>Trans-acting factors that mediate B cell specific transcription of immunoglobulin (Ig) genes have been postulated based on an analysis of the expression of exogenously introduced Ig gene recombinants in lymphoid and non-lymphoid cells. Two B cell-specific, cis-acting transcriptional regulatory elements have been identified. One element is located in the intron between the variable and constant regions of both heavy and kappa light chain genes and acts as a transcriptional enhancer. The second element is found upstream of both heavy chain and kappa light chain gene promoters. This element directs lymphoid-specific transcription even in the presence of viral enhancers.</p>
    <p>Mouse and human light chain promoters contain the octamer sequence ATTTGCAT approximately 70 base pairs upstream from the site of initiation. Heavy chain gene promoters contain the identical sequence in inverted orientation, ATGCAAAT, at the same position. This element appears to be required for the efficient utilization of Ig promoters in B cells. The high degree of sequence and positional conservation of this element as well as its apparent functional requirement suggests its interaction with a sequence-specific transcription factor but no such factor has been identified.</p>
    <heading>DISCLOSURE OF THE INVENTION</heading> <p>This invention pertains to human lymphoid-cell nuclear factors which bind to gene elements associated with regulation of the transcription of Ig genes and to methods for identification and for isolation of such factors. The factors are involved in the regulation of transcription of Ig genes. The invention also pertains to the nucleic acid encoding the regulatory factors, to methods of cloning factor-encoding genes and to methods of altering transcription of Ig genes in lymphoid cells or lymphoid derived cells, such as hybridoma cells, by transfecting or infecting cells with nucleic acid encoding the factors.</p>
    <p>Four different factors which bind to transciptional regulatory DNA elements of Ig genes were identified and isolated in nuclear extracts of lymphoid cells. Two of the factors, IgNF-A and E, are constitutive; two IgNF-B and κ-3 (hereinafter NF-κB) are lymphoid cell specific. Each factor is described below.</p>
    <p>IgNF-A (NF-A1)</p>
    <p>IgNF-A binds to DNA sequences in the upstream regions of both the murine heavy and kappa light chain gene promoters and also to the murine heavy chain gene enhancer. The binding is sequence specific and is probably mediated by a highly conserved sequence motif, ATTTGCAT, present in all three transcriptional elements. A factor with binding specificity similar to IgNF-A is also present in human HeLa cells indicating that IgNF-A may not be tissue specific.</p>
    <p>E Factors</p>
    <p>The E factors are expressed in all cell types and bind to the light and heavy chain enhancers.</p>
    <p>IgNF-B (NF-A2)</p>
    <p>IgNF-B exhibits the same sequence-specificity as IgNF-A; it binds to upstream regions of murine heavy and kappa light chain gene promoters and to murine heavy chain gene enhancer. This factor, however, is lymphoid specific; it is restricted to B and T cells.</p>
    <p>NF-κB (Previously Kappa-3)</p>
    <p>NF-κB binds exclusively to the kappa light chain gene enhancer (the sequence TGGGGATTCCCA). Initial work provided evidence that NF-kB is specific to B-lymphocytes (B-cells) and also to be B-cell stage specific. NF-kB was originally defected because it stimulates transcription of genes encoding kappa immunoglobulins in B lymphocytes. As described herein, it has subsequently been shown that transcription factor NF-kB, previously thought to be limited in its cellular distribution, is, in fact, present and inducible in many, if not all, cell types and that it acts as an intracellular messenger capable of playing a broad role in gene regulation as a mediator of inducible signal transduction. It has now been demonstrated that NF-kB has a central role in regulation of intercellular signals in many cell types. For example, NF-kB has not been shown to positively regulate the human β-interferon (β-IFN) gene in many, if not all, cell types. As described below, it is now clear not only that NF-kB is not tissue specific in nature, but also that in the wide number of types of cells in which it is present, it serves the important function of acting as an intracellular transducer of external influences. NF-kB has been shown to interact with a virus inducible element, called PRDII, in the β-IFN gene and to be highly induced by virus infection or treatment of cells with double-stranded RNA. In addition, NF-kB controls expression of the human immunodeficiency virus (HIV).</p>
    <p>As further described, it has been shown that a precursor of NF-KB is present in a variety of cells, that the NF-KB precursor in cytosolic fractions is inhibited in its DNA binding activity and that inhibition can be removed by appropriate stimulation, which also results in translocation of NF-KB to the nucleus. A protein inhibitor of NF-KB, designated IkB, has been shown to be present in the cytosol and to convert NF-KB into an inactive form in a reversible, saturable and specific reaction. Release of active NF-kB from the IkB-NF-kB complex has been shown to result from stimulation of cells by a variety of agents, such as bacterial lipopolysaccharide, extracellular polypeptides and chemical agents, such as phorbel esters, which stimulate intracellular phosphokinases. IkB and NF-KB appear to be present in a stoichiometric complex and dissociation of the two complex components results in two events: activation (appearance of NF-KB binding activity) and translocation of NF-KB to the nucleus.</p>
    <p>Identification and Isolation of the Transcriptional Regulatory Factors</p>
    <p>The transcription regulatory factors of the present invention were identified and isolated by means of a modified DNA binding assay. The assay has general applicability for analysis of protein DNA interactions in eukaryotic cells. In performing the assay, DNA probes embodying the relevant DNA elements or segments thereof are incubated with cellular nuclear extracts. The incubation is performed under conditions which allows the formation of protein-DNA complexes. Protein-DNA complexes are resolved from uncomplexed DNA by electrophoresis through polyacrylamide gels in low ionic strength buffers. In order to minimize binding of protein in a sequence nonspecific fashion, a competitor DNA species can be added to the incubation mixture of the extract and DNA probe. In the present work with eukaryotic cells the addition of alternating copolymer duplex poly(dI-dC)-poly(dI-dC) as a competitor DNA species provided for an enhancement of sensitivity in the detection of specific protein-DNA complexes and facilitated detection of the regulatory factors described herein.</p>
    <p>This invention pertains to the transcriptional regulatory factors, the genes encoding the four factors associated with transcriptional regulation, reagents (e.g., oligonucleotide probes, antibodies) which include or are reactive with the genes or the encoded factors and uses for the genes, factors and reagents. It further relates to NF-KB inhibitors, including isolated IkB, the gene encoding IkB and agents or drugs which enhance or block the activity of NF-KB or of the NF-KB inhibitor (e.g., IkB).</p>
    <p>The invention also pertains to a method of cloning DNA encoding the transcriptional regulatory factors or other related transcriptional regulatory factors. The method involves screening for expression of the part of the binding protein with binding-site DNA probes. Identification and cloning of the genes can also be accomplished by conventional techniques. For example, the desired factor can be purified from crude cellular nuclear extracts. A portion of the protein can then be sequenced and with the sequence information, oligonucleotide probes can be constructed and used to identify the gene coding the factor in a cDNA library. Alternatively, the polymerase chain reaction (PCR) can be used to identify genes encoding transcriptional regulatory factors.</p>
    <p>The present invention further relates to a method of inducing expression of a gene in a cell. In the method, a gene of interest (i.e., one to be expressed) is linked to the enhancer sequence containing the NF-KB binding site in such a manner that expression of the gene of interest is under the influence of the enhancer sequence. The resulting construct includes the kappa enhancer or a kappa enhancer portion containing at least the NF-KB binding site, the gene of interest, and a promoter appropriate for the gene of interest. Cells are transfected with the construct and, at an appropriate time, exposed to an appropriate inducer of NF-KB, resulting in induction of NF-KB and expression of the gene of interest.</p>
    <p>The subject invention further relates to methods of regulating (inducing or preventing) activation of NF-KB, controlling expression of the immunoglobulin kappa light chain gene and of other genes whose expression is controlled by NF-KB (e.g., HIV).</p>
    <p>As a result of this finding, it is now possible to alter or modify the activity of NF-κB as an intracellular messenger and, as a result, to alter or modify the effect of a variety of external influences, referred to as inducing substances, whose messages are transduced within cells through NF-κB activity. Alteration or modification, whether to enhance or reduce NF-κB activity or to change its binding activity (e.g., affinity, specificity), is referred to herein as regulation of NF-κB activity. The present invention relates to a method of regulating or influencing transduction, by NF-κB, of extracellular signals into specific patterns of gene expression and, thus, of regulating NF-κB-mediated gene expression in the cells and systems in which it occurs.</p>
    <p>In particular, the present invention relates to a method of regulating (enhancing or diminishing) the activity of NF-κB in cells in which it is present and capable of acting as an intracellular messenger, as well as to substances or composition useful in such a method. Such methods and compositions are designed to make use of the role of NF-κB as a mediator in the expression of genes in a variety of cell types. The expression of a gene having a NF-κB binding recognition sequence can be regulated, either positively or negatively, to provide for increased or decreased production of the protein whose expression is mediated by NF-κB. NF-κB-mediated gene expression can also be selectively regulated by altering the binding domain of NF-κB in such a manner that binding specificity and/or affinity are modified. In addition, genes which do not normally possess NF-κB binding recognition sequences can be placed under the control of NF-κB by inserting an NF-κB binding site in an appropriate position, to produce a construct which is then regulated by NF-κB. As a result of the present invention, cellular interactions between NF-κB and a gene or genes whose expression is mediated by NF-κB activity and which have, for example, medical implications (e.g., NF-κB/cytokine interactions; NF-κB/HTLV-I tax gene product interactions) can be altered or modified.</p>
    <p>Genes encoding the regulatory factors can be used to alter cellular transcription. For example, positive acting lymphoid specific factors involved in Ig gene transcription can be inserted into Ig-producing cells in multiple copies to enhance Ig production. Genes encoding tissue specific factors can be used in conjunction with genes encoding constitutive factors, where such combinations are determined necessary or desirable. Modified genes, created by, for example, mutagenesis techniques, may also be used. Further, the sequence-specific DNA binding domain of the factors can be used to direct a hybrid or altered protein to the specific binding site.</p>
    <p>DNA sequences complementary to regions of the factor-encoding genes can be used as DNA probes to determine the presence of DNA encoding the factors for diagnostic purposes and to help identify other genes encoding transcriptional regulatory factors. Antibodies can be raised against the factors and used as probes for factor expression. In addition, the cloned genes permit development of assays to screen for agonists or antagonists of gene expression and/or of the factors themselves. Further, because the binding site for NF-kB in the kappa gene is clearly defined, an assay for blockers or inhibitors of binding is available, as is an assay to determinte whether active NF-kB is present.</p>
    <heading>BRIEF DESCRIPTION OF THE DRAWINGS</heading> <p>FIG. 1A is a schematic depiction of the 5′ region of the MOPC 41 V<sub>κ</sub> gene segment; FIG. 1B is an autoradiograph of gel electrophoresis DNA binding assays with the SfaNI-SfaNI κ promoter fragment of the MOPC 41 V<sub>κ</sub> gene; and FIG. 1C is an autoradiograph of gel electrophoresis DNA binding assay with overlapping κ promoter fragments.</p>
    <p>FIGS. 2A-2B show autoradiographs of binding competition analysis in nuclear extracts of human (a) EW and (b) HeLa nuclear extracts.</p>
    <p>FIG. 3 shows the results of DNase I foot printing analysis of factor-DNA complexes.</p>
    <p>FIG. 4A shows the nucleotide sequences of actual and putative binding sites of IgNF-A; FIG. 4B is an autoradiograph of binding assays with various DNA probes of three Ig transcriptional control elements.</p>
    <p>FIG. 5A shows the DNA sequence of the promoter region of MODC41; FIG. 5B shows an autoradiograph of RNA transcript generalized in whole cell extracts made from human B lymphoma cell lines RAMOS and EW and from HeLa cells from the indicated templates.</p>
    <p>FIG. 6 shows an autoradiograph of RNA transcripts from templates containing an upstream deletion.</p>
    <p>FIG. 7 is a radioautograph of the binding of B cell nuclear extract to the MOPC-41 κ promoter region showing the IgNF-A and IgNF-B complexes.</p>
    <p>FIG. 8 shows the binding of T cell and nonlymphoid cell nuclear extracts to the MOPC-41 κ promoter region.</p>
    <p>FIG. 9A shows a restriction map of the μ-enhancer; FIG. 9B shows an autoradiograph binding assay carried out with μ-enhancer fragments.</p>
    <p>FIG. 10A shows a restriction map of the μ300 fragment; FIG. 10B shows complexes formed by various subfragments of μ300; FIG. 10C is a restriction map of the relevant region; FIG. 10E and 10D show competition binding assays with the subfragment μ70.</p>
    <p>FIG. 11A and 11B show location of binding sites in μ50 and μ70 by the methylation interference technique; FIG. 11C provides a summary of these results.</p>
    <p>FIG. 12A and 12B show an autoradiograph of binding complexes formed with μ50 and μ70 in B-cell and non B-cell extracts.</p>
    <p>FIG. 13A is a restriction map of κ enhancer; FIG. 13B shows an autoradiograph of binding assays with κ-enhancer fragments; FIGS. 13C and 13D show an autoradiograph of competition assays with κ-enhancer fragments.</p>
    <p>FIG. 14 shows location of NK-κB binding by methylation interference experiments.</p>
    <p>FIG. 15A shows binding analysis of NK-κB in various lymphoid and non-lymphoid cells; FIG. 15B shows the binding analysis of NK-κB in cells at various stages of B-cell differentiation.</p>
    <p>FIG. 16 shows the λgt11-EBNA-1 (λEB) recombinant and the oriP probe.</p>
    <p>FIG. 17 shows the sequence of the DNA probe used to screen for an H2TF1 and NF-κB expression.</p>
    <p>FIG. 18A shows the nucleotide sequence of the oct-2 gene derived from cDNA and the predicted amino acid sequence of encoded proteins.</p>
    <p>FIG. 18B shows the nucleotide sequence of the 3′ terminus and predicted the amino acid sequence of the C-terminus derived from clone pass-3.</p>
    <p>FIG. 18C is a schematic representation of the amino acid sequence deduced from oct-2 gene derived cDNA.</p>
    <p>FIG. 19 is a schematic representation of expression plasmid pBS-ATG-oct-2.</p>
    <p>FIG. 20 shows amino acid sequence alignment of the DNA binding domain of oct-2 factor with homeo-boxes of several other genes.</p>
    <p>FIGS. 21A-21B show the electrophoretic mobility shift analysis of (A) extracts derived from 10Z/3 cells before and after simulation with bacterial lipopolysaccharide (LPS) and (B) extracts derived from PD, an Abelson murine leukemia virus transformed pre-B cell line before and after stimulation with LPS.</p>
    <p>FIGS. 22A-22B show the effect (A) of cycloheximide on LPS stimulation of 70Z/3 cells and (B) of anisomycin on LPS stimulation of 70Z/3 cells.</p>
    <p>FIGS. 23A-23B show the effect of phorbol 12-myristate-13-acetate (PMA) on NF-B in 70Z/3 cells.</p>
    <p>FIGS. 24A-24C shows the induction of NFKB in a human lymphoma and in HeLa cells.</p>
    <p>FIG. 25 is a representation of binding sites for the NF-kB transcription factor in the immunoglobulin kappa light chain enhancer and the HIV enhancer. Boxes indicate the binding sites for NF-kB (B); other regulatory sites are referred to as E1, E2 and E3 and Spl. Dots indicate guanosine residues in the kappa enhancer whose methylation interfered with binding of NF-kB.</p>
    <p>FIGS. 26A-26B are a characterization of the NF-kB protein. FIG. 26A represents determination of the molecular weight of NF-kB. Nuclear extract (300 ug of protein) from TPA-stimulated 70Z/3 pre-B cells was denatured and subjected to reducing SDS-polyacrylamide gel electrophoresis (SDS-PAGE). Protein in the molecular weight fractions indicated by dashed lines was eluted and renatured prior to mobility shift assays as described. A fluorogram of a native gel is shown. The filled arrowhead indicates the position of a specific protein DNA-complex only detected in the 62-55 kDa fraction with a wild type (wt) but not with a mutant (mu) kappa enhancer fragment. The open arrowhead indicates the position of unbound DNA-fragments. FIG. 26B is a representation of glycerol gradient centrifugation of NF-kB. Nuclear extract (400 ug of protein) from TPA-stimulated 70Z/3 cells was subjected to ultracentrifugation on a continuous 10-30% glycerol gradient for 20 hours at 150,000×g in buffer D(+). Co-sedimented molecular weight standards (ovalbumin, 45 kDa; bovine serum albumin, 67 kDa; immunoglobulin G, 158 kDa; thyroglobulin monomer, 330 kDa and dimer 660 kDa) were detected in the fractions by SDS-PAGE, followed by Coomassie Blue staining. The distribution of NF-kB activity was determined by electrophoretic mobility shift assays using an end-labelled kappa enhancer fragment. Fluorograms of native gels are shown. The specificity of binding was tested using a kappa enhancer fragment with a mutation in the NF-kB binding site.</p>
    <p>FIGS <b>27</b>A-<b>27</b>C represent detection of a cytosolic precursor of NF-kB. FIG. 27A represents analysis of subcellular fractions for NF-kB DNA-binding activity. Nuclear extracts (N), cytosolic (C) and postnuclear membrane fractions (P) from control and TPA-stimulated 70Z/3 pre-B cells were analyzed by gel-shift assays. The filled arrowhead indicates the position of the specific protein-DNA complex seen only with a wild type but not with a mutant kappa enhancer fragment. FIG. 27B represents activation of a cytosolic NF-kB precursor after treatment with dissociating agents. Subcellular fractions were treated with 25% formamide followed by dilution and addition of 0.2% sodium desoxycholate as described. FIG. 27C represents detection of a cytosolic NF-kB precursor after denaturation, SDS-PAGE and renaturation of protein. Nuclear extract (N) and cytosolic fraction (C) from unstimulated (control) 70Z/3 cells was subjected to the treatment outlined in FIG. <b>26</b>A. For details of illustration, see FIG. <b>27</b>A.</p>
    <p>FIG. 28 represents analysis of subcellular fractions for DNA-binding activity of the TPA-inducible transcription factor AP-1. Equal cell-equivalents of nuclear extracts (N) and cytosolic fractions (C) from 70Z/3 and HeLa cells were used in mobility shift assays. AP-1 specific DNA-binding activity was detected using an end-labeled EcoRI-HindIII fragment from the yeast HIS 4 promoter containing three binding sites for GCN4 recognized by mammalian AP-1. The three protein-DNA complexes seen on shorter exposures of the fluorogram are indicated by filled arrowheads and the position of unbound DNA-fragment by an open arrowhead.</p>
    <p>FIG. 29 represents results of electrophoretic mobility shift analysis of subcellular fraction of 70Z/3 cells.</p>
    <p>FIG. 30 shows the effect of denaturation and renaturation of kB-specific DNA-binding activity in nuclear extracts and cytosolic fractions of 70Z/3 cells.</p>
    <p>FIGS. 31A-31B show the effects of dissociating agents on the activity of NF-kB in subcellular fractions of 70Z/3 cells. FIG. <b>31</b>A: cell-free activation of a NF-kB precursor in the cytosolic fraction by desoxycholate. FIG. <b>31</b>B: cell-free activation of a NF-kB precursor in the cytosolic fraction by formamide and by a combined treatment with formamide and desoxycholate.</p>
    <p>FIG. 32 shows the effect of TPA stimulation on the subcellular distribution of NF-kB in 70Z/3 cells.</p>
    <p>FIG. 33 shows the effect of TPA stimulation on the subcellular distribution of NF-kB in HeLa cells.</p>
    <p>FIGS. 34A-34B show results of DNA-cellulose chromatography of DOC-treated cytosol. Cytosol was prepared from unstimulated 70Z/3 pre-B cells and protein concentrations determined. In the fluorograms of native gels shown, the filled arrowheads indicate the position of the NF-kB-k enhancer fragment complex and the open arrowheads the position of unbound DNA probe. FIG. <b>34</b>A: Release of DOC-independent NF-kB activity. Equal proportions of load, flow-through (FT), washings, and eluates were analyzed by EMSA, with (+) or without (−) excess DOC. The <sup>32</sup>P-radioactivity in the NF-kB-DNA complexes was counted by liquid scintillation and the percentage of NF-kB activity recovered in the various fractions was calculated. FIG. <b>34</b>B: Release of an inhibitory activity. NF-kB contained in the 0.2M NaCl fraction (31 ng of protein) or NF-kB in a nuclear extract from TPA-treated 70Z/3 cells (1.1 μg of protein) was incubated under non dissociating conditions with the indicated amounts (in microliters) of either cytosol which was DOC-treated but not passed over DNA-cellulose (lanes 4 to 6 and 13 to 15) or the flow-through fraction (referred to as NF-kB-depleted cytosol; lanes 7 to 9 and 16 to 18).</p>
    <p>FIGS. 35A-35C shows-characterization of IkB and its complex with NF-kB. In the fluorograms shown, the filled arrowheads indicate the position of the NF-kB-DNA complex and the open arrowheads the position of free DNA probe. FIG. <b>35</b>A: For size determination of IkB, the flow-through from the DNA-cellulose column was passed over a G-200 Sephadex column. Portions of fractions were incubated with NF-kB contained in nuclear extracts from TPA-stimulated 70Z/3 cells (N TPA), and analyzed by EMSA. v, void volume; P, fraction where remaining NF-kB precursor (FIG. 34A, lane 4) peaked after gel filtration as assayed with excess DOC in the absence of added NF-kB; I, fraction where the inhibiting activity peaked. FIG. <b>35</b>B: The effect of trypsin treatment on the inhibiting activity of IkB. NF-kB in a nuclear extract (lane 1) was incubated with a fraction containing inhibitor (lane 2) without any addition (−; lane 3) or with bovine pancreas trypsin inhibitor (TI; lane 4), trypsin that had been incubated with BPTI (T+TI; lane 5), or with trypsin alone (T; lane 6). Samples were then used in the inhibitor assay. FIG. <b>35</b>C: Glycerol gradient sedimentation of NF-kB and its complex with IkB. Nuclear extract from TPA-stimulated 70Z/3 cells (N TPA) and cytosol from unstimulated cells (C Co) were subjected to sedimentation through a glycerol gradient. Cosedimented size markers were ovalbumin (45 kD), BSA (67 kD), immunoglobulin G (158 kD) and thyroglobulin (330 and 660 kD). NF-kB activity was detected in the fractions by EMSA with a wild type k enhancer fragment (kB wt, left panels). The specificity was tested with a mutant fragment (kB mu, right panels). The inactive cytosolic NF-κB precursor (lower panel) was activated by formamide treatment (Fa; middle panel).</p>
    <p>FIGS. 36A-36B show the reversibility and kinetics of the inactivation of NF-kB. FIG. <b>36</b>A: The effect of DOC treatment on in vitro inactivated NF-kB. NF-kB contained in nuclear extracts from TPA-stimulated 70Z/3 cells (N TPA; 1.1 μg of protein) was inactivated by addition of a gel filtration fraction containing IkB (2.5 μg of protein). A duplicate sample was treated after the inhibition reaction with 0.8% DOC followed by addition of DNA binding reaction mixture containing 0.7% NP-40. Samples were analyzed by EMSA. In the fluorograms shown, the filled arrowhead indicates the position of the NF-kB-DNA complex and the open arrowhead the position of unbound DNA probe. FIG. <b>36</b>B: A titration and kinetic analysis of the in vitro inactivation of NF-kB. NF-kB contained in nuclear extracts from TPA-treated 70Z/3 cells (2.2 μg of protein) was incubated with increasing amounts (0.25 to 2.25 μg of protein) of a gel filtration fraction containing IkB. After the DNA binding reaction, samples were analyzed by EMSA. The <sup>32</sup>P-radioactivity in the NF-kB-DNA complexes visualized by fluorography was determined by liquid scintillation counting. All reactions were performed in triplicates. The bars represent standard deviations.</p>
    <p>FIGS. 37A-37B show the specificity of IkB. Nuclear extracts from unstimulated (Co) or TPA-treated cells were incubated with 5 μl of buffer G (−) or with 5 μl of a gel filtration fraction containing IkB (+) (A, in the presence of 150 mM NaCl). After DNA binding reactions, samples were analyzed by EMSA. FIG. <b>37</b>A: Influence of IkB on the DNA binding activity of various nuclear factors. The probes were: NF-kB; H2TF1, an oligonucleotide subcloned into pUC containing the H2TF1 binding site from the H-2 promoter; OCTA, an oligonucleotide subcloned into pUC containing the common binding site for the ubiquitous (upper filled arrowhead) and lymphoid-specific (lower filled arrowhead) octamer-binding proteins; NF-μE1; NF-kE2; and AP-1, EcoRI-HindIII fragment of the yeast HIS4 promoter containing three binding sites recognized by mammalian AP-1/jun. In the fluorograms shown, filled arrowheads indicate the positions of specific protein-DNA complexes. Open arrowheads indicate the positions of uncomplexed DNA fragments. FIG. <b>37</b>B: Interaction of IkB with NF-kB from different cell lines. The filled arrowheads indicate the positions of the NF-kB-DNA complexes from the various cell lines and the open arrowhead indicates the position of uncomplexed DNA probe.</p>
    <p>FIGS. 38A-38B show the presence of NF-kB in enucleated cells. FIG. <b>38</b>A: Phase contrast and fluoresence microscopy of enucleated HeLa cells. From 612 cells counted on photographic prints, 63 showed nuclear staining. A representative micrograph is shown. The arrow indicates a cell that retained its nucleus. FIG. <b>38</b>B: Analysis of complete and enucleated cells for NF-kB activity. Total cell extracts (1.2 μg of protein) from control (Co) and TPA-treated complete and enucleated cells were analyzed by EMSA with a labeled k enhancer fragment (kB) or HIS4 promoter fragment (AP-1), 3 μg of poly(dI-dC), 1 μg of BSA, 1.2% NP-40 and the binding buffer in a final volume of 20 μl. In lanes 5 to 8, extracts were treated with DOC followed by the addition of the DNA binding mixture to give final concentrations of 0.8% DOC and 1.2% NP-40. Samples were analyzed by EMSA. In the fluorograms shown, the filled arrowheads indicate the positions of specific protein-DNA complexes and the open arrowheads the positions of uncomplexed DNA probe.</p>
    <p>FIG. 39 is a diagram showing the location of positive regulatory domain II (PRDII) within the interferon gene regulatory element (IRE) and a comparison of the nucleotide sequences of the PRDII site, κB site, and the H2TF1 site.</p>
    <p>FIGS. 40A-40B show the results of assays demonstrating that NF-κB binds to PRDII in vitro.</p>
    <p>FIG. 40A shows the results of mobility shift electrophoresis assays using as radiolabelled probe an IRE DNA fragment (IRE, lanes 1-3, 10-14, 20-24, 30 and 31), an oligonucleotide containing two copies of the PRDII sequence (PRDII<sub>2</sub>, lanes 4-6) or a κB site oligonucleotide (κB, lanes 7-9, 15-19, 25-29, 32 and 33). Assays contained either no protein (φ, lanes 1, 4 and 7); 5 μg of unstimulated Jurkat nuclear extract (−, lanes 2, 5 and 8) or 5 μg of nuclear extract from Jurkat cells stimulated with PHA and PMA (+, lanes 3, 6, 9 and 10-29). Competitions used either the κB oligonucleotide (κB; lanes 10-19) or the PRDII oligonucleotide (PRD II<sub>2</sub>, lanes 20-29) in the ng amounts shown above each lane. Cytosol (8 μg) from unstimulated Jurkat cells was tested either before (−, lanes 30 and 32) or following treatment with 0.8% deoxycholate (DC, lanes 31 and 33).</p>
    <p>FIG. 40B shows that mutations within PRDII that reduce β-IFN induction in vivo decrease the affinity of NF-κB for PRDII in vitro. IRE sequences bearing the mutations indicated (for positions refer to FIG. 39) were tested. The mutations were previously shown to have either high (+) or low (−) inducibility. Goodbourn and Maniatis, <i>Proc. Natl. Acad. Sci. USA, </i>85:1447-1451 (1988). Binding and competition were carried out, as described for FIG. 2A, using 5 μg of nuclear extract from virus infected 70Z/3 cells. Competitor was 10 ng of either the wild-type (WT) or mutant (MUT) κB oligonucleotide (lanes 7,8).</p>
    <p>FIGS. 41A-41B demonstrate the functional interchangeability of PRDII and NF-κB in vivo.</p>
    <p>FIG. 41A is an autoradiogram showing the results of CAT assays of extracts prepared from L929 and S194 myeloma cells transfected with the reporter genes illustrated in FIG. <b>41</b>B.</p>
    <p>FIG. 41B is a diagram of the reporter genes containing multiple copies of PRDII or κB. Two or four PRDII sites [(P)<sub>2 </sub>and (P)<sub>4</sub>, respectively] were inserted upstream of the truncated −41 human β-globin promoter/CAT fusion gene (−41β) using an oligonucleotide containing two copies of PRDII (PRDII×2, as described in the Exemplification). Two copies of a synthetic wild-type KB site, or mutant κB site (B and B<sup>−</sup>, respectively) were inserted upstream of the mouse c-fos promoter/CAT fusion gene in which the promoter was truncated to nucleotide −56 (Δ56).</p>
    <p>FIGS. 42A-42C demonstrate that virus infection activates binding of NF-κB and gene expression in B lymphocytes and fibroblasts.</p>
    <p>FIG. 42A represents the results of a binding assay which shows complexes formed with the Ig κB site using 5 μg of nuclear extract from unstimulated cells (lanes 1, 9 and 17) and 5 μg (lanes 2-6, 10-14 and 18-22) or 1 μg (lanes 7, 8, 15, 16, 23 and 24) of nuclear extract from cells after virus infection. Extracts were prepared from Namalwa cells (lanes 1-8), 70Z/3 cells (lanes 7-16), and L929 cells (lanes 17-28). Competitions used either 5 or 20 ng of the wild-type (WT) or mutant (MUT)κB oligonucleotide. GTP stimulation was tested by addition directly to the binding assay to a final concentration of 3 mM. Cytosol was obtained from L929 cells either prior to (lanes 25, 26) or after (lanes 27, 28) viral treatment and tested before (−) and after (DC) treatment with deoxycholate.</p>
    <p>FIG. 42B presents a comparison of methylation interference footprints of virus-induced complexes from Namalwa and L929 cell extracts with NF-κB complexes derived from PHA/PMA-stimulated Jurkat cells. The cleavage pattern resulting from methylation of guanine or adenine residues is shown for DNA extracted from the free probe (F) or the DNA-protein complex (B) observed following mobility shift electrophoresis. The sequence of the κB site presented at the sides and methylated residues that interfere with binding are indicated by a solid circle.</p>
    <p>FIG. 42C presents results of Northern blot analyses of 70Z/3 cells treated with various inducers. The upper panel shows the induction (from 0 to 20 hr as indicated) of κ (κ) mRNA by treatment with 50 ng/ml PMA (PMA, lanes 1-4), 15 μg/ml LPS (LPS, lanes 5-8) or Sendai virus (V, lanes 9-12). The lower panel shows the same blot hybridized to a β-IFN DNA fragment.</p>
    <p>FIG. 43 is the nucleotide sequence and the amino acid sequence of IkB-α.</p>
    <heading>CLONE DEPOSITS</heading> <p>Clones λh3 and λ3-1 were deposited (Feb. 12, 1988) at the AMerican Type Culture Collection (12301 Parklawn Drive; Rockville, Md. 20852), under the terms of the Budapest Treaty. They were assigned ATCC Designationals 67629 and 67630, respectively. Upon issue of a U.S. patent from the subject application, all restrictions upon the availability of these clones will be irrevocably removed.</p>
    <heading>DETAILED DESCRIPTION OF THE INVENTION</heading> <p>The present invention relates to the identification, isolation and characterization of human transcriptional regulatory factors, to genes encoding the factors, methods of isolating DNA encoding transcriptional regulatory factors and the encoded factors, uses of the DNA, encoded factors, and antibodies against the encoded factors, inhibitors of the transcriptional regulatory factors. In particular, it relates to the transcriptional regulatory factor NF-kB (previously designated Kappa-3); its inhibitor; IkB, DNA encoding each, methods of altering interactions of NF-kB and IkB and methods of regulating the activity of NF-kB. As described herein, NF-kB, was initially thought to be a B-cell specific factor involved in immunoglobulin gene regulation and has since been shown to be inducible in many, if not all, cell types and to act as an intracellular transducer or mediator of a variety of external influences. The following is a description of the discovery and characterization of four transcriptional regulatory factors, assessment of the function of NF-kB and its role, in many cell types, as an intracellular mediator or transducer of a variety of external influences, discovery of the NF-kB inhibitor IkB and demonstration that NF-kB and IkB exist in the cytoplasm as a NF-kB-IkB complex whose dissociation results in activation of NF-kB and its translocation into the nucleus. The following is also a description of the uses of the genes, regulatory factors and related products and reagents.</p>
    <p>The transcriptional regulatory factors described herein can be broadly classified as constitutive (non-lymphoid) or tissue (lymphoid) specific. All factors are believed to play a role in transcription of immunoglobulin (Ig) genes. Constitutive factors, which are present in non-lymphoid cells, may have a role in regulating transcription of genes other than Ig genes; lymphoid-specific factors might also play a role in regulating transcription of genes in addition to Ig genes.</p>
    <p>Four transcriptional regulatory factors were identified, as described below and in the Examples. The presence of constitutive factors rendered the detection of tissue specific factors more difficult. A sensitive DNA binding assay, described below, was employed in all studies to facilitate detection of tissue specific factors.</p>
    <p>The characteristics of the transcriptional regulatory factors IgNF-A, E, IgNFB and Kappa-3 (or NF-κB) are summarized in Table 1 below.</p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 1</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">CHARACTERISTICS OF FOUR</td>
              </tr> <tr class="description-tr"> <td class="description-td">TRANSCRIPTIONAL REGULATORY FACTORS</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="42pt" align="left"> </colspec> <colspec colname="1" colwidth="175pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Ig Regulatory Sequence</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="4"> <colspec colname="1" colwidth="42pt" align="center"> </colspec> <colspec colname="2" colwidth="42pt" align="center"> </colspec> <colspec colname="3" colwidth="42pt" align="center"> </colspec> <colspec colname="4" colwidth="91pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Factor</td>
                <td class="description-td">Promoter</td>
                <td class="description-td">Enhancer</td>
                <td class="description-td"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="7"> <colspec colname="1" colwidth="42pt" align="center"> </colspec> <colspec colname="2" colwidth="21pt" align="center"> </colspec> <colspec colname="3" colwidth="21pt" align="center"> </colspec> <colspec colname="4" colwidth="21pt" align="center"> </colspec> <colspec colname="5" colwidth="21pt" align="center"> </colspec> <colspec colname="6" colwidth="42pt" align="center"> </colspec> <colspec colname="7" colwidth="49pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Designation</td>
                <td class="description-td">V<sub>H</sub> </td>
                <td class="description-td">V<sub>κ</sub> </td>
                <td class="description-td">U<sub>E</sub> </td>
                <td class="description-td">K<sub>E</sub> </td>
                <td class="description-td">Lymphoid</td>
                <td class="description-td">Nonlymphoid</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="7" align="center" rowsep="1" class="description-td" colspan="7"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">IgNF-A</td>
                <td class="description-td">+</td>
                <td class="description-td">+</td>
                <td class="description-td">+</td>
                <td class="description-td">−</td>
                <td class="description-td">+</td>
                <td class="description-td">+</td>
              </tr> <tr class="description-tr"> <td class="description-td">(NF-A1)</td>
              </tr> <tr class="description-tr"> <td class="description-td">E factors</td>
                <td class="description-td">−</td>
                <td class="description-td">−</td>
                <td class="description-td">+</td>
                <td class="description-td">+</td>
                <td class="description-td">+</td>
                <td class="description-td">+</td>
              </tr> <tr class="description-tr"> <td class="description-td">IgNF-B</td>
                <td class="description-td">+</td>
                <td class="description-td">+</td>
                <td class="description-td">+</td>
                <td class="description-td">−</td>
                <td class="description-td">+</td>
                <td class="description-td">−</td>
              </tr> <tr class="description-tr"> <td class="description-td">(NF-A2)</td>
              </tr> <tr class="description-tr"> <td class="description-td">Kappa-3</td>
                <td class="description-td">−</td>
                <td class="description-td">−</td>
                <td class="description-td">−</td>
                <td class="description-td">+</td>
                <td class="description-td">+</td>
                <td class="description-td">+</td>
              </tr> <tr class="description-tr"> <td class="description-td">(NF-κB)</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="7" align="center" rowsep="1" class="description-td" colspan="7"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p>Factor Ig NF-A</p>
    <p>As indicated in Table 1, IgNF-A binds to Ig regulatory DNA elements in the region of mouse heavy and kappa light chain gene promoters and also to mouse heavy chain gene enhancer. DNAase I footprint analysis indicates that the binding is mediated by the octamer sequence (ATTTGCAT) which occurs in mouse and human light chain gene promoters approximately 70 base pairs upstream from the site of initation and in heavy chain gene promoters at about the same position (in inverted sequence).</p>
    <p>Deletion or disruption of the IgNF-A binding site in Ig promoters significantly reduces the level of accurately initiated transcripts in vivo. See, e.g., Bergman, Y. et al. PNAS USA 81 7041-7045 (1984); Mason, J. O. et al. Cell 41 479-487 (1985). As demonstrated in Example 2, this also occurs in an in vitro transcription system. IgNF-A appears to be a positive transacting factor.</p>
    <p>The IgNF-A binding site appears to be a functional component of the B-cell-specific Ig promoter. For example, sequences from this promoter containing the IgNF-A binding site specify accurate transcription in B-cells but not in Hela cells. IgNF-A however, may not be restricted to B-cells because a factor was detected in Hela cell extracts which generated complexes with similar mobilities and sequence specificity (as tested by competition analysis). Interestingly, the Ig octamer motif in the IgNF-A binding site has recently been shown to be present in the upstream region (about 225 bp) of vertebrate U1 and U2 snRNA genes. More importantly, this element dramatically stimulates (20 to 50 fold) transcription of U2 snRNA genes in Xenopus oocytes. Therefore, IgNF-A may be a constitutive activator protein that also functions in the high level expression of U1 and U2 snRNA genes in vertebrate cells.</p>
    <p>The presence of an IgNF-A binding site in the mouse heavy chain enhancer suggests the additional involvement of IgNF-A in enhancer function. It is known that deletion of an 80 bp region of the enhancer containing the putative binding site reduces activity approximately tenfold. The occupation of the binding site, in vivo, has been inferred from the fact that the G residue in the enhancer octamer is protected from dimethyl sulfate modification only in cell of the B lineage. Furthermore, IgNF-A also binds in a sequence-specific manner to the SV40 enhancer (J. Weinberger, personal communication), which contains the Ig octamer motif, thereby strengthening the notion that the factor participates in enhancer function.</p>
    <p>E Factors</p>
    <p>The E factors are constitutive factors which bind to the Ig light and heavy chain enhancer.</p>
    <p>Factor Ig NF-B</p>
    <p>Factor IgNF-B binds to the same regulatory elements as IgNF-A. Indeed, the binding site for IgNF-B appears to be the octamer motif. In contrast to IgNF-A, IgNF-B is lymphoid cell specific. It was. found in nuclear extracts from pre-B, mature B and myeloma cell lines and in nuclear extracts from some T cell lymphomas. IgNF-B was undetectable in nuclear extracts of several non-lymphoid cells. The gene encoding Ig NF-B has been cloned (oct-2 clone below) and its nucleotide sequence has been determined (See FIG. 18<i>a</i>).</p>
    <p>Factor NF-κB</p>
    <p>NF-κB (previously referred to as Kappa-3) binds only to the Ig light chain enhancer. The binding is mediated by the sequence TGGGATTCCCA. The factor initially was characterized as lymphoid cell specific and also as lymphoid stage specific; that is, work showed that it is expressed only by mature B-cells. Thus, it is a marker of B cell maturation (e.g. the factor can be used to type B cell lymphomas). Additional work, described in Examples 8-15 in particular, has shown that NF-kB is an inducible factor in cells, both pre-B and non pre-B, in which it is not constitutively present (Example 8), that it is present in the cytoplasm as an inactive precursor (Examples 10 and 11), and that the inactive precursor is a complex of NF-kB and an inhibitor, referred to as IkB, which converts NF-kB to an inactive form in a reversible saturable and specific reaction. Dissociation of the complex results in activation of NF-kB (appearance of NF-kB binding activity) and translocation of the NF-kB into the nucleus.</p>
    <p>As discussed below, it is now evident that this DNA binding protein, initially thought to be a B-cell specific factor and subsequently implicated in gene regulation in T lymphocytes, is present in many, if not all, cell types and that it acts as an important intracellular transducer or mediator of a variety of external influences. That is, NF-κB is now known to be involved in a variety of induction processes in essentially all types of cells and is thought to participate in a system through which multiple induction pathways work, in much the same manner as “second messengers” (e.g., cAMP, IP<sub>3</sub>) act, resulting in transduction of a variety of extra-cellular signals into specific patterns of gene expression. Different cell types and different genes respond to this one signal, which serves as a central “control”, whose activity can be altered by means of the present invention. As used, the terms altering and modifying mean changing the activity or function of NF-κB in such a manner that it differs from the naturally-occurring activity of NF-κB under the same conditions (e.g., is greater than or less than, including no activity, the naturally-occurring NF-κB activity; is of different specificity in terms of binding, etc.).</p>
    <p>It has been shown that NF-κB participates in gene expression (e.g., cytokine gene expression) which is activated by a specific influence or extracellular signal (e.g., infection by a virus) in many, if not all types of cells. In particular, it has now been demonstrated that NF-κB has a central role in virus induction of human β-interferon (β-IFN) gene expression. Virus infection has been shown to potently activate the binding and nuclear localization of NF-κB and, in pre-B lymphocytes, to result in expression of both the β-IFN gene and the Ig kappa gene. The wide variety of cell types in which β-interferon can be induced and the divergent set of gene induction processes which involve NF-κB provide evidence that NF-κB plays a broad role in gene regulation as a mediator of inducible signal transduction.</p>
    <p>The following is a description and exemplification of work (Example 15) which clearly demonstrates the role of NF-κB in virus-induced human β-IFN gene expression; of the evidence that there is a single NF-κB which serves many roles in many different cell types and how it acts as an intracellular messenger in a variety of different gene induction processes, particularly several which have important effects on cell physiology in health and disease; and of the use of methods and compositions of the present invention.</p>
    <p>Role of NF-κB in Cytokine Gene Regulation</p>
    <p>The role of NFκB as a mediator or messenger in cytokine gene regulation has been demonstrated, as explained in greater detail in the Exemplification, through assessment of the viral induction of human β-IFN gene expression. The human β-IFN gene has been shown to be positively regulated by NF-κB, which was, in turn, shown to interact with a virus inducible element, called PRDII, in the β-IFN gene. As described below, NF-κB has been shown to be highly induced in lymphoid and non-lymphoid cells by either virus infection or treatment of cells with double-stranded RNA [poly (rI:rC)]. It has also been shown to bind specifically to PRDII, which is one of two positive regulatory domains of the interferon gene regulatory element (IRE) which, together with the release of a negative influence over a site called NRDI, are necessary and sufficient for virus induction of the β-IFN gene.</p>
    <p>It is known that the human β-interferon (β-IFN) gene is highly inducible by virus or synthetic double-stranded RNA poly(rI:rC) in many, if not all, cell types. DeMaeyer, E. and J. DeMaeyer-Guignard, “Interferons and Other Regulatory Cytokines”, John Wiley and Sons, New York (1988). Extensive characterization of the β-IFN gene promoter has revealed a complex arrangement of positive and negative regulatory elements. Taniguchi, T., <i>Ann. Rev. Immunol., </i>6:439-464 (1988). A 40 base pair DNA sequence designated the IRE (Interferon gene Regulatory Element) is both necessary and sufficient for virus induction. Goodbourn et al., <i>Cell, </i>41: 509-520 (1985). The IRE contains two distinct positive regulatory domains (PRDI and PRDII) and one negative regulatory domain (NRDI). Goodbourn et al., <i>Cell, </i>45:601-610 (1986); Goodbourn, S. and T. Maniatis, <i>Proc. Natl. Acad. Sci. USA, </i>85:1447-1451 (1988). Virus induction apparently requires cooperative interactions between PRDI and PRDII. Goodbourn, S. and T. Maniatis, <i>Proc. Natl. Acad. Sci. USA, </i>85:1447-1451 (1988). Single copies of PRDI or PRDII alone are not sufficient for virus or poly(rI:rC) induction, but two or more copies of PRDI (Fujita et al., <i>Cell, </i>49:357-367 (1987)) or PRDII (Fan, C. M. and T. Maniatis, <i>EMBO J., </i>8:101-110 (1989)) confer inducibility on heterologous promoters.</p>
    <p>The PRDII sequence binds a nuclear factor, designated PRDII-BF, that is present in extracts from both uninduced and induced MG63 cells. Keller, A. and T. Maniatis, <i>Proc. Natl. Acad. Sci. USA, </i>85:3309-3313 (1988). A cDNA clone encoding a PRDII binding factor (designated PRDII-BF1) was isolated. DNA sequence analysis revealed that PRDII-BF1 is similar, if not identical, to a cDNA clone encoding a protein that binds to related sites in both the MHC class I H-2K<sup>b </sup>gene and the Ig κ enhancer. Singh et al., <i>Cell </i>52:415-423 (1988). This observation suggested that PDRII might be functionally related to the H2-K<sup>b </sup>and κ enhancer sites.</p>
    <p>The site in the H-2K<sup>b </sup>promoter is required for its constitutive and interferon-induced expression and binds a factor designated H2TF1 and possibly similar factors KBF1 and EBP-1 which are constitutively expressed in most cell types. (Baldwin, A. S. and P. A. Sharp, <i>Proc. Natl. Acad. Sci. USA, </i>85:723-727 (1988); Yano et al., <i>EMBO J., </i>6:3317-3324 (1988); Clark et al., <i>Genes </i>&amp; <i>Dev., </i>2:991-1002 (1988)). The Ig κ enhancer site, termed κB, binds NF-κB , which is required for κ enhancer function. Sen, R. and D. Baltimore, <i>Cell, </i>46:705-716 (1986); Atchison, M. and R. P. Perry, <i>Cell, </i>48:121-128 (1987); Lenardo, M. et al., <i>Science, </i>236:1573-1577 (1987). The transcriptional activities and in vitro binding of the κB site and PRDII were compared and results showed that the two regulatory sequences are interchangeable in vivo, and that PRDII specifically binds NF-κB in vitro. A binding activity indistinguishable from NF-κB in nuclear extracts from virus-infected cells was also identified. Viral treatment of 70Z/3 pre-B lymphocytes induced κ gene expression as well as β-IFN gene expression. These results show that NF-κB plays an important role in the virus induction of the β-IFN gene and indicate that NF-κB acts similarly to second messenger systems in that it transduces a variety of extracellular signals into specific patterns of gene expression.</p>
    <p>It has been shown, by all available criteria, that the κB and the PRDII DNA elements—one from the Ig κ light chain gene and one from the β-IFN gene—are interchangeable. They drive transcription of reporter genes in response to the same set of inducers, cross-compete for binding in vitro and have closely-related DNA sequences. Another indication of the identity of the two elements is that release of NF-κB from a complex with its inhibitor, I-κB, correlates with the induction of β-IFN in L929 cells and that, conversely, a β-IFN inducer (Sendai virus) induces κ gene transcription in 70Z/3 cells. This relationship is strengthened by the correlation between the ability of mutations in PRDII to decrease β-IFN gene inducibility in vivo and reduce binding to NF-κB in vitro. Evidence that double-stranded RNA induces a factor resembling NF-κB has also been recently obtained by Visvanathan, K. V. and S. Goodbourn, <i>EMBO J., </i>8:1129-1138 (1989).</p>
    <p>Results described in the Exemplification strongly imply that β-IFN gene expression is activated, at least in part, by induction of NF-κB. The ability of NF-κB to be activated by a protein synthesis-independent pathway is consistent with the fact that induction of β-IFN is not blocked by cycloheximide. In fact, the β-IFN gene, like the κ gene, can be induced by cycloheximide. Ringold et al., <i>Proc. Natl. Acad. Sci. USA, </i>81:3964-3968 (1984); Enoch et al., <i>Mol. Cell Biol., </i>6:801-810 (1987); and Wall et al., <i>Proc. Natl. Acad. Sci. USA, </i>83:295-298 (1986). In addition to the interaction between NF-κB and PRDII, virus induction of β-IFN involves activation through PRDI and the release of repression at NRDI. The present data revealing a role for NF-κB in β-IFN regulation is a striking example of how it is used in many, if not all, cell types.</p>
    <p>Evidence for the Existence of a Single NF-κB</p>
    <p>As shown in Table 1, sites present in a variety of genes form a mobility shift electrophoretic complex which resembles NF-κB , as reported by Sen, and Baltimore, upon incubation of the Ig κ enhancer with B-cell extracts, Sen, R. and D. Baltimore, (<i>Cell </i>46:705-716 (1986)). The biochemical evidence suggests the involvement of a single NF-κB in all cell types and not a family of factors in which individual members are specifically inducible in particular cell types.</p>
    <p>This evidence includes the fact that purification of NF-κB to homogeneity from both human and bovine sources yields a single polypeptide chain of approximately 44 to 50 kD (although this could be a fragment of a larger protein). Kawakami et al., <i>Proc. Natl. Acad. Sci. USA </i>85:4700-4704 (1988); and Lenardo et al., <i>Proc. Natl. Acad. Sci. USA, </i>85:8825-8829 (1988). NF-κB adopts an oligomeric structure in solution; based on the size of the complex, it exists either as a homodimer or associates with a heterologous subunit of approximately equal molecular weight. Baeuerle, P. et al., <i>Cold Spring Harbor Symposium, </i>53:789-798 (1988); Lenardo et al., <i>Proc. Natl. Acad. Sci. USA, </i>85:8825-8829 (1988). NF-κB has the unique property that nucleoside triphosphates dramatically stimulate its ability to bind DNA in vitro. Lenardo et al., <i>Proc. Natl. Acad. Sci. USA, </i>85:8825-8829 (1988). NF-κB is further distinguished by the fact that it can be released as an active binding species from an inactive cytosolic form that is completed with IκB. Baeuerle, P. and D. Baltimore, <i>Cell, </i>53:211-217 (1988); Baeuerle, P. and D. Baltimore, <i>Science, </i>242:540-545 (1988). All of these features are shared by the NF-κB complex irrespective of the cell-type from which it is derived.</p>
    <p>More importantly, no differences in binding specificity have been detected between the NF-κB complexes from different cell types. That is, the NF-κB complex induced in T cells has no preference for sites from genes activated in T cells rather than those from genes activated in B cells and vice-versa. Lenardo et al., <i>Proc. Natl. Acad. Sci. USA, </i>85:8825-8829 (1988). An identical pattern of base contacts is characteristic of complexes between DNA and NF-κB from different cell types, further decreasing the possibility that the NF-κB complex in different cell types is due to heterogeneous proteins.</p>
    <p>It is clear that NF-κB binding sites are recognized by other obviously distinct transcription factors. The best examples are the H2-TF1 and KBF-1 proteins, which bind to an NF-κB-like site in the H2-K<sup>b </sup>MHC class I gene (Baldwin, A. S. and P. A. Sharp, <i>Mol. Cell. Biol., </i>7:305-313 (1987); and Yano et al., <i>EMBO J., </i>6:3317-3324 (1987)). However, these factors are constitutively active nuclear binding proteins in many different cell types and no evidence implicates them in inducible gene expression. Other examples include the factor EBP-1 which binds to the SV40 κB site but has a different molecular size than NF-κB and is also not inducible (Clark et al., <i>Genes </i>&amp; <i>Development, </i>2:991-1002 (1988); HIVEN86A, an 86 kD factor identified in activated T cell extracts by DNA affinity chromatography (Franza et al., <i>Nature, </i>330:391-395 (1987)); and finally, a protein encoded by a cDNA (λh3 or PRDII-BF1) selected from λgt11 expression libraries (Singh et al., <i>Cell, </i>52:415-523 (1988)). Recent evidence has made it unlikely that the λh3 clone encodes NF-κB because several cell types that have abundant expression of NF-κB lack the transcript for λh3. Taken together these findings indicate that there is only one NF-κB that serves multiple roles in many different cell types.</p>
    <p>NF-κB Acts as an Intracellular Messenger</p>
    <p>A salient feature of the induction of NF-κB is that it takes place in the absence of new protein synthesis. Sen, R. and D. Baltimore, <i>Cell, </i>47:921-928 (1986). In fact, the protein synthesis inhibitor cycloheximide can alone activate NF-κB. Sen, R. and D. Baltimore, <i>Cell, </i>47:921-928 (1986). It appears, therefore, that NF-κB induction involves the conversion of a pre-existing precursor into an active form.</p>
    <p>Inactive NF-κB is complexed with a labile inhibitor protein, I-κB. Cytosolic extracts from uninduced cells can be treated in vitro with dissociating agents such as formamide and deoxycholate to unmask very high levels of NF-κB activity. Baeuerle, P. and D. Baltimore, <i>Cell, </i>53:211-217 (1988). These treatments by and large do not work on nuclear extracts from uninduced cells. Conversely, NF-κB activated normally in the cell is detected in nuclear but not cytosolic extracts implying a nuclear translocation step following activation in vivo. The inhibitory activity has been shown to be due to a protein of 68 kD that can be separated chromatographically from NF-κB. Baeuerle, P. and D. Baltimore, <i>Science, </i>242:540-545 (1988). This protein is able to inhibit the binding of NF-κB but not other DNA-binding proteins and has therefore been named “I-κB” (Inhibitor-κB).</p>
    <p>Notably, crude preparations of I-κB efficiently inhibit binding of NF-κB derived from mature B cells or other cell-types that have been induced. Baeuerle, P. and D. Baltimore, <i>Science, </i>242:540-545 (1988). The implicaton is that activation of NF-κB involves a modification of I-κB and not NF-κB. This distinguishes NF-κB activation from a similar phenomenon involving the glucocorticoid receptor. In the latter, a direct interaction of glucocorticoid with the receptor is required to release it from a cytoplasmic complex with the heat shock protein, hsp90. Picard, D. and K. R. Yamamoto, <i>EMBO J., </i>6:3333-3340 (1987).</p>
    <p>The model which ties together these observations is that NF-κB is initially located in the cytoplasm in a form unable to bind DNA because it is complexed with I-κB. Various inducers then cause an alteration in I-κB allowing NF-κB to be released from the complex. Free NF-κB then travels to the nucleus and interacts with its DNA recognition sites to facilitate gene transcription. The complex formation of NF-κB with I-κB appears to be rapidly and efficiently reversible in vitro which lends itself well to the shut-off as well as turn-on of NF-κB binding. Moreover, this model resolves a major question in signal transduction: NF-κB, like the glucocorticoid receptor, acts as a messenger to transmit the gene induction signal from the plasma membrane to the nucleus.</p>
    <p>The model presented above is not unlike the well known role of cAMP as a second messenger in the action of many hormones; in the case of cAMP, the first messenger is the hormone itself. (see, eg., Pastan, <i>Sci. Amer., </i>227:97-105 (1972)). The essential features of the cAMP model are that cells contain receptors for hormones in the plasma membrane. The combination of a hormone with its specific membrane receptor stimulates the enzyme adenylate cyclase which is also bound to the plasma membrane. The concomitant increase in adenylate cyclase activity increases the amount of cAMP inside the cell which serves to alter the rate of one or more cellular processes. An important feature of this second messenger or mediator model is that the hormone (the first messenger) need not enter the cell.</p>
    <p>The participation of NF-κB in gene expression that is activated in specific cells by specific influences calls for a level of regulation in addition to the inducibility of NF-κB binding. How is “cross-talk” between the various paths employing NF-κB avoided? Factors acting upon other sequences within a transcriptional control element appear to govern the response to the NF-κB signal, as described herein for β-IFN. Studies of β-IFN expression have shown that virus induction works through three events: two virus-inducible positive signals, one of which is NF-κB, and the release of a single negative regulator. The two positive signals work through distinct DNA sites (PRDI and PRDII), but must act together to facilitate transcription. Either site alone is not inducible.</p>
    <p>The theme of multiple signals that generate specificity is further supported by studies of the Ig κ gene and the IL-2 receptor gene. The NF-κB site from the κ light chain enhancer alone on a short oligonucleotide will stimulate transcription in B and T lymphocytes as well as in non-lymphoid cells. Pierce, J. W. et al., <i>Proc. Natl. Acad. Sci. USA, </i>85:1482-1486 (1988). Its function depends solely on the presence of NF-κB. By contrast, the entire κ enhancer is inducible only in B lymphocytes and is unresponsive to NF-κB in other cell types. The restricted response to NF-κB by the κ enhancer has now been attributed to a repressor sequence. The repressor sequence resides in the enhancer some distance away from the NF-κB binding site and acts to suppress transcriptional effects of NF-κB in non-B cell types. The activation of the IL-2 receptor gene specifically in T lymphocytes is attained by a slightly different means. Full induction of this gene depends on NF-κB as well as a positively-acting sequence immediately downstream. The downstream element has now been found to bind a T cell specific protein called NF-ILT. Though NF-ILT is not itself inducible, its presence only in T cells seems to contribute to T cell specific induction of the IL-2 receptor gene.</p>
    <p>Role of NF-κB in Other Inducible Systems</p>
    <p>Recently, NF-κB has been implicated in several other inducible systems. For example, NF-κB is induced in T-cells by a trans-activator (tax) of HTLV-1 or by PMA/PHA treatment and thereby activates the IL-2 receptor α gene and possibly the IL-2 gene. Bohnlein et al., <i>Cell, </i>53:827-836 (1988); Leung, K. and G. Nabel, <i>Nature, </i>333:776-778 (1988); Ruben et al., <i>Science, </i>241:89-92 (1988); Cross et al., <i>Science, (</i>1989); and Lenardo et al., <i>Proc. Natl. Acad. Sci. USA, </i>85:8825-8829 (1988). NF-κB also appears to take part in gene activation during the acute phase response of the liver. Edbrooke et al., (1989). Results described here suggest that inducibility of NF-κB plays a prominent role in interactions between cytokines. IL-1 and TNF-α activate NF-κB binding and both have been shown known to induce β-IFN. Osborn et al., <i>Proc. Natl. Acad. Sci. USA, </i>86:2336-2340 (1989); and DeMaeyer, E. and J. DeMaeyer-Guignard, “Interferons and Other Regulatory Cytokines”, John Wiley and Sons, New York (1988). Finally, NF-κB has been shown to play a role in the transcription of human immunodeficiency virus (HIV). Nabel, G. and D. Baltimore, <i>Nature, </i>326:711-713 (1987). Significantly, herpes simplex virus has recently been shown to increase HIV LTR transcription through NF-κB/core sequences. Gimble et al., <i>J. Virol., </i>62:4104-4112 (1988). Thus, NF-κB induction may lead to the propagation of HIV in cells infected with other viruses.</p>
    <p>NF-κB is unique among transcription regulatory proteins in its role as a major intracellular transducer of a variety of external influences in many cell types. In the cases studied thus far, it appears that the actual target of induction is I-κB, which becomes modified to a form that no longer binds to NF-κB. Baeuerle, P. and D. Baltimore, <i>Science, </i>242:540-545 (1988). The released NF-κB then displays DNA binding activity and translocates to the nucleus.</p>
    <p>Role of NF-kB in HIV Expression</p>
    <p>Treatment of latently HIV-infected T-cells with phorbol ester (12-O-tetradecanoylphorbol 13-acetate; TPA) and with phytohaemaglutinin (PHA) results in the onset of virus production. Harada, S. et al., <i>Virology, </i>154:249-258 (1986); Zagury, D. J. et al., <i>Science, </i>232:755-759 (1986). The same treatments induce NF-kB activity in the human T-lymphoma cell line Jurkat. Sen, R. and D. Baltimore, <i>Cell, </i>47:921-928 (1896). This correlation and the finding that two NF-kB binding sites are present upstream of the transcriptional start site in the HIV enhancer, (FIG. 25) suggested a direct role for NF-kB in the activation of the viral enhancer, an event ultimately leading to the production of virus. Nabel, G. and D. Baltimore, <i>Nature, </i>326:711-713 (1987). This possibility was tested by transient transfection of a plasmid containing an HIV LTR-controlled CAT gene into a human T-lymphoma cell line. Nabel, G. and D. Baltimore, <i>Nature, </i>326:711-713 (1987). The viral cis-acting elements rendered the transcriptional activity of the CAT gene responsive to TPA/PHA treatment of cells. This inducible transcriptional stimulation of the CAT gene was completely dependent on intact binding sites for NF-kB in the HIV enhancer because mutation of the two binding sites abolished inducibility. A protein-DNA complex with a fragment of the HIV enhancer containing the two NF-kB binding sites was observed in mobility shift assays only with nuclear extracts from TPA/PHA-stimulated T-cells and not with control extracts. These observations provided strong evidence that HIV expression in latently infected T-cells is induced by the same transcription factor that regulates kappa gene expression, NF-kB. A precursor of NF-kB is constitutively present in T-cells. Its activity can be induced by a treatment that mimicks antigenic T-cell activation and, after induction, NF-kB is able to bind to and subsequently enhance the activity of HIV transcriptional control elements. Thus, it is reasonable to conclude that NF-kB is the physiological transactivator responsible for initial expression of dormant HIV-DNA following stimulation of T-lymphocytes.</p>
    <p>Other factors have also been implicated in the control of HIV expression including the HIV-encoded tat-III protein, the cellular transcription factor Spl, and viral proteins encoded by the ElA gene of adenovirus and the ICPO gene of the Herpes Simplex Virus. Muesing, M. A. et al., <i>Cell, </i>48:691-701 (1987); Jones, K. A. et al., <i>Science, </i>232:755-759 (1986); Gendelman, H. E. et al., <i>Proc. of the Natl. Acad. of Sc., USA, </i>83:9759-9763 (1986); Nabel, G. J. et al., <i>Science </i>(1988); Rando, R. F. et al., <i>Oncogene, </i>1:13-19 (1987); Mosca, J. D. et al., <i>Nature, </i>325:67-70 (1987). It is doubtful whether the tat-III and Spl proteins are responsible for an initial induction of HIV expression. Although the tat-III protein functions as a strong positive feedback regulator of HIV expression, full expression of the tat-III protein appears to depend on NF-kB. Muesing, M. A. et al., <i>Cell, </i>48:691-701 (1987); Nabel, G. and D. Baltimore, <i>Nature, </i>326:711-713 (1987). It is unlikely that Spl initiates HIV expression because it is constitutively active. Dynan, W. S. and R. Tjian, <i>Cell, </i>32:669-680 (1983). The viral ElA and ICPO gene products might lead to induction of HIV expression. This, however, is independent of T-cell activation by antigenic stimulation and of NF-kB, as shown by cotransfection experiments into human T-lymphoma cells of plasmids with an HIV enhancer-controlled CAT gene and plasmids encoding the viral genes. The increase in CAT activity induced by the viral gene products was unchanged when the NF-kB binding sites in the HIV enhancer were inactivated by mutation.</p>
    <p>Improved DNA Binding Assay with Enhanced Sensitivity for Identification of Regulatory Factors</p>
    <p>The transcriptional regulatory factors described above were identified in extracts of cellular nuclear protein by means of an improved gel electrophoresis DNA binding assay with enhanced sensitivity. This improved assay is a modification of an original assay based on the altered mobility of protein-DNA complexes during gel electrophoresis. In the improved assay of this invention, the simple alternating copolymer, duplex poly(dI-dC)-poly(dI-dC) was used as the competitor DNA species. The use of this copolymer as competitor resulted in an enhancement of sensitivity for detection of specific protein-DNA complexes. The original assay has been extensively employed in equilibrium and kinetic analyses of purified prokaryotic gene regulatory proteins. See, e.g., Fried, M. and Crothers, D. M., <i>Nucleic Acid Res. </i>9 6505-6525 (1981); Garner, M. M. and Revzin A., <i>Nucleic Acids Res. </i>9 3047-3060 (1981). More recently it has been used to identify and isolate a protein that binds to satellite DNA from a nuclear extract of eukaryotic cells (monkey cells). See Strauss, R. and Varshavsky, A., <i>Cell </i>37 889-901 (1984). In the latter study an excess of heterologous competitor DNA (<i>E. coli</i>) was included with the specific probe fragment to bind the more abundant, sequence non-specific DNA binding proteins in the extract.</p>
    <p>The assay is performed essentially as described by Strauss and Varshavky, supra, except for the addition of the poly(dI-dC)-poly(dI-dC). An extract of nuclear protein is prepared, for example, by the method of Dingnam, J. D. et al., <i>Nucleic Acids Research </i>11:1475-1489 (1983). The extract is incubated with a radiolabelled DNA probe (such as an end-labeled DNA probe) that is to be tested for binding to nuclear protein present in the extract. Incubation is carried out in the presence of the poly(dI-dC)-poly(dI-dC) competitor in a pysiological buffer. DNA protein complexes are resolved from (separated from) free DNA probes by electrophoresis through a polyacrylamide gel in a low ionic strength buffer and visualized by autoradiography.</p>
    <p>In a preferred embodiment of the method, protein samples (about 10 μg protein) are incubated with approximately 10,000 cpm (about 0.5 ηg ) of an end-labeled <sup>32</sup>P double-stranded DNA probe fragment in the presence of about 0.8-4 ηg poly(dI-dC)-poly(dI-dC) (Pharmacia) in a final volume of about 25 μl. Incubations are carried out at 30° for 30-60 minutes in 10 mM Tris HCl (pH 7.5), 50 mM NaCl, 1 mM DTT, 1 mM EDTA. Protein-DNA complexes are resolved on low-ionic strength polyacrylamide gels. Samples are layered onto low ionic-strength 4% polyacrylamide gels (0.15×16 cm; acrylamide:bisacrylamide weight ratio of 30:1). Gels are pre-electrophoresed for about 30 min at 11 V/cm in buffer consisting of 6.7 mM TrisHCl, (pH 7.5), 3.3 mM NaOAc, and 1 mM EDTA. Buffer is recirculated between compartments. Gels are electrophoresed at the same voltage at room temperature, transferred to Whatman 3MM, dried and autoradiographed.</p>
    <p>The enhanced sensitivity of the assay of the present invention is evident in the initial work which led to identification of the factor IgNF-A. A radiolabelled SfaNI-SfaNI DNA fragment derived from the upstream region of the MOPC 41 κ light chain gene (FIG. 1<i>a</i>) was incubated with a nuclear extract of a human B cell line, in the absence or in the presence of <i>E. coli </i>chromosomal DNA or poly(dI-dC)-poly(dI-dC). The resulting complexes were resolved from the free fragment by electrophoresis through a low ionic strength, non-denaturing polyacrylamide gel and visualized by autoradiography (FIG. 1<i>b</i>). In the absence of competitor DNA, all of the labeled fragment was retained at the top of the gel (lane 1), probably due to the binding of an excess of non sequence-specific proteins. With addition of increasing amounts of either poly(dI-dC)-poly(dI-dC) (lanes 2-6) or <i>E. coli </i>chromosomal DNA (lanes 7-11) as competitors, putative protein-DNA complexes which migrated slower than the free fragment were detected. The relative abundance of the major species of complex (B) as well as that of minor species was significantly greater in the presence of the alternating copolymer competitor DNA.</p>
    <p>The use of a sensitive gel electrophoresis DNA binding assay in conjunction with the copolymer competitor poly(dI-dC)-poly(dI-dC) facilitated the identification of the regulatory factors described herein. The simple alternating copolymer probably competes less effectively than heterologous DNA sequences for binding of a sequence-specific factor, thereby significantly increasing the sensitivity of the assay. The assay has general applicability for elucidation of mammalian gene regulatory proteins.</p>
    <p>A further increase in sensitivity in this assay is obtained by the use of small DNA probes (about 100 bp or less) which minimize non-specific binding interactions in a crude extract. (See Example 1).</p>
    <p>Employing this assay, binding competition tests can be performed to analyze the sequence specificity of protein-DNA interactions. For this purpose, an unlabeled DNA fragment to be examined for competitive binding to the protein factor can be added to the incubation mixture of protein extract and labeled DNA probe (along with the poly(dI-dC)-poly(dI-dC)). The disappearance of protein-DNA probe complex, or its diminishment, indicates that the unlabeled fragments compete for binding of the protein factor. In addition, relative binding affinity of the protein to a probe sequence can be assessed by examining the ability of a competitor to displace the protein at varying concentrations.</p>
    <p>In conjunction with the competition assays, DNase I footprint analysis (See Galas, D. and Schmitz A., <i>Nucl. Acids Res. </i>5 3157-3170 (1978) and Example 1) and methylation interference experiments (See, e.g., Ephrussi, A. et al., <i>Science </i>227:134-140 (1985) can be used to refine analysis of the binding domain of the protein factors.</p>
    <p>Assessment of the Functional Role of Factors Described Herein in Regulation of Transcription</p>
    <p>The functional role of the factors in the regulation of the transcription can be assessed in several ways. A preferred technique for lymphoid cell factors entails the use of the in vitro transcription system developed from cells of lymphoid cell lineage. This system is described in detail in the Example 2. The function of a factor can be indirectly assessed in this system by employing as templates for transcription, nucleotide sequences from which the binding domain of the factor has been deleted. As has been noted above, deletion of the upstream sequence located between −44 and −79 bp from the cap site of the MOPC41 κ gene disrupts transcriptions in this system (This has also been noted in in vivo systems). The deleted region includes the IgNF-A binding site. This indicates that transcription of the template is dependent upon the factor—binding site and, inferentially, upon the factor itself.</p>
    <p>A direct way to assess the function of the factors is to show that transcription can be modulated by removal and replacement of the factor in the in vitro transcription system with an appropriate template. For example, the intact MOPC41 κ promoter gene can be used as a template in the in vitro system described and transcription of this template can be assessed in the presence and absence of a factor (for instance, NF-κB, a lymphoid specific factor). The factor can be removed from the lymphoid cell extract by chromatographic fractionation and then replaced. If the level of transcription is diminished in the absence and restored by replacement of the factor, a direct indication of the factors involvement in transcription is provided.</p>
    <p>In an alternative approach, antisera or monoclonal antibody can be raised against a purified or enriched preparation of the factor. The antibody can be used to probe for expression of the factor in a library of cDNA of cells known to express the factor.</p>
    <p>Cloning of Genes Encoding Sequence-Specific DNA Binding Proteins, Particularly Genes Encoding Transcriptional Regulatory Factors</p>
    <p>Genes encoding transcriptional regulatory factors can be isolated by a novel method for cloning genes that encode sequence-specific DNA binding proteins. The method involves screening a library of recombinant expression vectors for expression of the factor with a DNA probe comprising the recognition (binding) site for the factor. Expression of the factor is identified by the presence of complex between the DNA probe and the expressed binding protein. The approach has general applicability to the cloning of sequence-specific DNA binding proteins.</p>
    <p>According to the method, an expression library is created by inserting DNA (e.g., cDNA from a cell which expresses the sequence specific binding protein) into an appropriate expression vector to establish an expression library. A preferred expression vector is the bacteriophage λgt11 which is capable of expressing foreign DNA inserts within <i>E. coli</i>. See e.g., Young, R. A. and Davis, R. W. in <i>Genetic Engineering: Principles and Techniques</i>, vol 7 (eds Setlow, J. &amp; Hollaender, A.) 29-41 (Plenum, New York 1985). Alternatively, plasmid vectors may be used.</p>
    <p>The expression library is screened with a binding-site DNA probe. The probe comprises the DNA sequence recognized by the binding protein, such as an appropriate transcriptional regulatory element (e.g., the octamer or κ-element). In preferred embodiments, the probe is less than 150 bp in length, to reduce nonspecific binding. The probe can be an oligomer of the binding site. Multiple copies of the site provide for multiple protein binding to the probe. The DNA probe is generally detectably labeled DNA. A particularly useful label is <sup>32</sup>P.</p>
    <p>In the present method, the binding site probe is incubated with host cell protein under conditions which allow the probe to complex with the any cognate binding protein expressed in the cell. The formation of such complexes is determined by detecting label associated with the protein. In a preferred mode, the screening is performed by generating a replica of host cell lysates and by screening the replicated protein with the probe. For example, when the bacteriophage λgt11 is used, recombinant viruses are plated in arrays onto a lawn of <i>E. coli </i>and a replica of the resulting viral plaques is made by transferring plaque protein onto an appropriate adsorbtive surface (e.g. protein replica filters). The adsorbed plaque protein is contacted with the probe under conditions which permit the formation of complexes between adsorbed protein and the probe. The replica is then washed to remove unbound probe and then examined for associated label. The protein can be examined autoradiograghically for the presence of label.</p>
    <p>In other embodiments, a nonspecific competitor DNA can be used along with the recognition site probe, to reduce nonspecific binding to the probe. Examples of such nonspecific competitor DNA include poly (dI-dC)-poly(dI-dC) and denatured calf thymus DNA. In addition, the protein-probe complexes can be stabilized covalently for detection, for example, by uv irradiation.</p>
    <p>This method of screening for sequence specific binding proteins is dependent, inter alia, upon:</p>
    <p>i) the functional expression of the binding domain of the desired binding protein in the host cell;</p>
    <p>ii) a strong and selective interaction between the binding domain and the DNA probe; and</p>
    <p>iii) a sufficiently high level of expression of the binding protein.</p>
    <p>These parameters can be optimimized for different proteins by routine experimentation. Some factors relevant to such optimization are discussed in detail in the exemplification of the cloning of transcriptional regulatory factor NF-κB given below.</p>
    <p>Other modes of cloning genes encoding sequence-specific DNA binding proteins, such as genes encoding transcriptional regulatory factors, may be used. For example, the factor can be purified chromatographically by, for example, ion exchange, gel filtration and affinity chromatography or combinations thereof. Once the factor is sufficiently purified, it can be partially sequenced and from the sequence information, oligodeoxy-nucleotide probes can be made and used to identify the gene encoding the factor in a cDNA library.</p>
    <p>Occurrence and Activation of NF-kB and Demonstration of the Role of an NF-kB Inhibitor (IkB)</p>
    <p>The following is a description of the occurrence and activation of NF-kB in cells which do not express k immunoglobulin light chain genes (and, in which NF-kB is not evident in either cytoplasmic or nuclear fractions). In particular, the following is a description of localization of NF-kB in the cytosolic fraction; of activation of NF-kB in cytosolic fractions by dissociating agents; of redistribution of NF-kB into the nuclear fraction upon TPA stimulation; of demonstration of the appearance of NF-kB binding ability; and of the occurrence and characterization of an NF-kB inhibitor.</p>
    <p>NF-kB Occurrence and Activation in 70Z/3 Cells</p>
    <p>NF-kB is Virtually Undetectable in Unstimulated 70Z/3 Cells</p>
    <p>To determine where in the cell NF-kB or its inactive precursor are located, subcellular fractions from control and TPA-stimulated 70Z/3 cells were analyzed for kB-specific DNA-binding activity. Nuclear extracts, cytosolic and postnuclear membrane fractions were analyzed at equal amounts of protein in an electrophoretic mobility shift assay, described in Example 1, followed by fluorography. (Sen, R. and D. Baltimore, <i>Cell, </i>46:705-716 (1986). The specificity of protein-binding to a fragment of the kappa light chain enhancer was controlled by using a fragment with a mutation in the binding motif for NF-kB. This mutation has been shown to prevent binding of NF-kB. Lenardo, M. et al., <i>Science, </i>236:1573-1577 (1987). Thus, any complexes formed on the wild type, but not on the mutant fragment, are considered specific for the NF-kB site.</p>
    <p>Nuclear extracts from control cells contained very little kB-specific binding activity (FIG. 29, compare lanes 1 and 7). This is in agreement with results reported previously by Sen and Baltimore. Sen, R. and D. Baltimore, <i>Cell, </i>46:705-716 (1986); Sen, R. and D. Baltimore, <i>Cell, </i>47:921-928 (1986). Similarly, the ctyosolic fraction produced only a faint, but specific and reproducible, signal co-migrating with the signal from the nuclear extract (FIG. 29, compare lanes 2 and 8). The fraction containing postnuclear membranes did not exhibit any detectable DNA-binding activity (FIG. 29, lane 3).</p>
    <p>Upon treatment of cells with TPA for 30 minutes, the nuclear NF-kB activity was dramatically increased (FIG. 8, compare lanes 4 and 10). Almost no increase of the specific signal in the cytosolic fraction was observed (FIG. 29, compare lanes 5 and 11). The post-nuclear membrane fraction gave raise to an apparently kB-specific complex with a mobility higher than that formed by nuclear NF-kB (FIG. 29, compare lanes 6 and 12). None of the fractions had inhibitors of binding because added authentic NF-kB was fully recovered in all fractions, indicating that the results reflect a true activation of binding specificity.</p>
    <p>NF-kB is Detectable in the Cytosolic Fraction after Denaturation and Renaturation</p>
    <p>To examine whether active NF-kB might be present but masked in fractions from unstimulated 70Z/3 cells, proteins from nuclear extracts and cytosolic fractions of control and TPA-stimulated cells were precipitated, denatured by boiling in SDS plus 2-mercaptoethanol and fractionated by electrophoresis through SDS-polyacrylamide gels. 300 ug of protein of nuclear extracts (N) and cytosolic fractions (C) from control (Co) and TPA-stimulated cells (TPA) were subjected to reducing SDS-polyacrylamide gel electrophoresis. Proteins eluted from different molecular weight fractions of the gel (i.e., corresponding to approximately 70-62 kDa (gel slice No. 6), 62-55 kDa (gel slice No. 7) and 55-48 kDa (gel slice No. 8)) were subjected to a renaturation protocol after removal of SDS. Hager, D. A. and R. R. Burgess, <i>Anal. Biochem., </i>109:76-86 (1980) and Example 10. Renatured fractions were tested for kB-specific DNA-binding activity in mobility shift assays using wild type and mutant kappa light chain enhancer fragments. DNA-binding reactions were performed with 11 ul of the renatured fractions in the presence of 80 ng poly(d[I-c]) in a final volume of 15 ul. Assays with wild type (WT) and mutant (mu) k enhancer fragments were loaded in adjacent lanes.</p>
    <p>In nuclear extracts from TPA-stimulated cells, NF-kB activity was exclusively found in a molecular weight region of 62-55 kDa. The efficiency of renaturation of the nuclear NF-kB activity was about one percent. In FIG. 30, the active and two adjacent fractions are shown for the nuclear extract from TPA-stimulated cells (lanes 13 to 18). In nuclear extracts from control cells, much less NF-kB activity was found in the same molecular weight fraction after renaturation (FIG. 30, lanes 3 and 4). Both the cytosolic fractions from control and TPA-stimulated cells, however, gave rise to a strong NF-kB-specific signal (FIG. 30, lanes 9, 10 and 21, 22). The specificity of the signal was shown by several criteria. First, it was only present when the wild type, but not the mutant, DNA fragment was used in mobility shift assays (FIG. 30, lanes 9, 10 and 21, 22). Second, it was generated with protein eluted from the same molecular weight fraction that contained authentic nuclear NF-kB. Third, upon mixing, the complex formed by the putative cytoplasmic NF-kB co-migrated exactly in native polyacrylamide gels with the complex formed by interaction of the nuclear form of NF-kB with its cognate DNA.</p>
    <p>Assuming that NF-kB from the various fractions had a similar recovery and efficiency of renaturation, the data suggest that significant amounts of NF-kB can be activated in unstimulated 70Z/3 cells by denaturation, followed by fractionation and renaturation. Furthermore, in unstimulated cells, the in vitro activated NF-kB activity was almost exclusively recovered in the cytosolic fraction.</p>
    <p>The subcellular distribution of two noninducible DNA-binding proteins, NF-uE3 and the octamer-binding protein were also examined in mobility shift assays, in order to determine whether other DNA-binding factors also partition into cytoplasmic fractions. Sen, R. and D. Baltimore, <i>Cell, </i>46:705-716 (1986); Singh, H. et al., <i>Nature, </i>319:154-158 (1986); and Staudt, L. M. et al., <i>Nature, </i>323:640-643 (1986). The vast majority of both DNA-binding activities was found in nuclear extracts; cytosolic and postnuclear membrane fractions contained only very little activities (FIG. 29, lanes 13 to 24). No significant change in the complex formation by the two factors was observed when fractions from control and TPA-stimulated cells were compared. Thus, although subcellular fractionation can produce artificial redistribution of proteins, the fractions used in this study do well reflect nuclear localization of a number of DNA-binding proteins.</p>
    <p>NF-kB in the Cytosolic Fraction Can be Activated by Dissociating Agents</p>
    <p>The ability to reveal cytosolic NF-kB by simply denaturation and renaturation suggested that NF-kB might be bound to an inhibitor and therefore several compounds that might dissociate protein complexes were tested for their ability to directly activate kB-specific DNA-binding activity in fractions of 70Z/3 cells. The cytosolic fraction from unstimulated cells and, as a control, the nuclear extract from TPA-treated cells were incubated with the compounds prior to electrophoretic separation of the protein-DNA complexes. Incubation of the cytosolic fraction with 0.2% sodium desoxycholate (DOC) (in the presence of 0.2% NP-40) resulted in the activation of DNA-binding activity (FIG. 31, lane 2). The induced complex had the same mobility in native gels as the one formed by nuclear NF-kB. It appeared to be specific for the kB site of the kappa light chain enhancer because it was not formed when the mutant fragment was used in the mobility shift assay (FIG. 31, lane 21). Higher concentrations of DOC led to the inactivation of the newly activated kB-binding activity (FIG. 31, lanes 3 to 5) as well as of the authentic nuclear factor (FIG. 31, lanes 13 to 15).</p>
    <p>DOC can be sequestered out of a solution by the addition of excess nonionic detergent, presumably by inclusion of the DOC into micelles formed by the nonionic detergent. When treatment of the cytosolic fraction with up to 0.8% DOC was followed by the addition of 1% of the nonionic detergent NP-40, a quite efficient activation of the cytosolic kB-binding activity was achieved (FIG. 31, lanes 7 to 9). The DNA-binding activity of in vivo activated NF-kB from nuclear extracts was not significantly increased at low concentrations of DOC (FIG. 31, lanes 11, 12 and 16, 17). Elevated concentrations of DOC showed inhibitory effects on the DNA-binding activity of TPA-activated NF-kB that paralleled those observed for the in vitro activated kB-binding activity (FIG. 31, compare lanes 3, 4, 5 with lanes 13, 14, 15 and lanes 9, 10 with lanes 19, 20).</p>
    <p>A partial activation of the cytosolic kB-binding activity was observed after treatment of the cytosolic fraction with 27% formamide followed by dilution (FIG. 31, lane 7). With the further addition of 0.2% DOC—a condition that alone also leads only to partial activation (FIG. 31, lane 3)—a very potent activation was observed (FIG. 31, lane 11). A titration showed that formamide and DOC activated in a synergistic manner (FIG. 31, lanes 2 to 11). The DNA-binding activity of in vivo activated NF-kB from nuclear extracts was not enhanced by any of the treatments (FIG. 31, lanes 13 to 22). On the contrary, partial inhibition of DNA-binding of NF-kB was observed under some conditions.</p>
    <p>No in vitro activation of NF-kB was achieved by treatment with guanidinium hydrochloride (between 0.3 and 3M), urea (between 0.5 and 5M), and SDS (between 0.1 and 1%), in the presence of 0.2% NP-40). Exhaustive dialysis of the cytosolic fraction using dialysis membranes with a cut-off of 25 kDa did not lead to an activation of DNA-binding activity. In the dialyzed fraction, NF-kB-activity could still be efficiently induced by formamide/DOC treatment, suggesting that no freely diffusible cofactors smaller than 25 kDa were required for the in vitro activation.</p>
    <p>TPA Stimulation Causes Redistribution of NF-kB into the Nuclear Fraction</p>
    <p>To examine whether the form of NF-kB detected after in vitro activation in the cytosolic fraction could quantitatively account for the NF-kB found in nuclear extracts after TPA stimulation of cells, subcellular fractions of 70Z/3 cells were reinvestigated in mobility shift assays after treatment with formamide and DOC using equal cell-equivalents of subcellular fractions. Equal cell-equivalents of nuclear extracts (N) and cytosolic (C) and post-nuclear membrane fractions (P) from control (Co) and TPA-stimulated cells (TPA) were left untreated (lanes 1-6 and 13-18) or subjected to a formamide/desoxycholate treatment (Fa+DOC; lanes 7-12 and 19-24; for conditions see FIG. 31, lane 11). This treatment was preferred over the DOC/NP-40 chase treatment because it gave a higher resolution of bands in mobility shift assays. DNA-binding reactions were performed in the presence of 3.2 ug poly(d[I-C]) using 4.4 ug of protein from nuclear extracts, 8.8 ug of protein from cytosolic fractions or 2.2 ug of protein from postnuclear membrane fractions (all in 4 ul buffer D(+)). Fluorograms of native gels are shown. The specificity of protein-DNA complexes was controlled using wild type (kB wt) and mutant kappa enhancer fragments (kB mutant; see legend to FIG. <b>29</b>). The filled arrowhead indicates the position of kB-specific protein-DNA complexes and the open arrowhead the positions of unbound DNA-fragments.</p>
    <p>Densitometric scanning of fluorograms showed that in control cells, more than 92% of the total cellular kB-specific DNA-binding activity was recovered in the cytosolic fraction following treatment with formamide and COD (FIG. 32, lanes 7 to 9). In TPA-stimulated cells, 80% of the kB-specific DNA-binding activity was found in nuclear extracts (FIG. 32, lanes 10 to 12). The remaining activity was largely recovered in the cytosolic fraction (FIG. 32, lane 11). All DNA-binding activities described were specific for the kB site, as shown by their absence when the mutant kappa enhancer fragment was used in the mobility shift assays (FIG. 32, lanes 13 to 24).</p>
    <p>When the total cellular NF-kB activity that was activated in vitro in control cells was compared to the total cellular activity found in TPA-stimulated cells after the same treatment, virtually identical amounts of activity were observed. The equal amounts of NF-kB activity found in control and TPA-treated cells suggest that the treatment with formamide and DOC resulted in the complete conversion of an inactive precursor of NF-kB into a form of NF-kB with high DNA-binding affinity. Furthermore, these results provide evidence for a TPA-inducible translocation of NF-kB from the cytosol into the nucleus.</p>
    <p>NF-kB Occurrence and Activation in HeLa Cells</p>
    <p>NF-kB activity can also be induced in HeLa cells after TPA treatment, as shown by the appearance of a kB-specific DNA-binding activity in nuclear extracts. Sen, R. and D. Baltimore, <i>Cell, </i>47:921-928 (1986). Therefore, induction of NF-kB in the cytosolic fraction of HeLa cells was tested by treatment with formamide and DOC. To equal cell-equivalents of fractions, 17% formamide was added and diluted to 10% by the addition of the DNA-binding reaction mixture containing 4 ug poly(d[I-C]). DOC was then added to a final concentration of 0.6% to give a reaction volume of 20 ul. Assays contained either 1.35 ug of protein from nuclear extracts, 9 ug of protein from the cytosolic fractions of 0.9 ug of protein from the postnuclear membrane fractions (all in 10 ul buffer D(+)).</p>
    <p>Redistribution of NF-kB activity in the subcellular fractions upon TPA stimulation of cells, was also assessed, using the procedure described for 70Z/3 cells. Mobility shift assays were performed with equal cell-equivalents of the subcellular fractions. Because HeLa cells had about ten times as much cytosolic protein as nuclear protein—as opposed to the 2:1 ratio in 70Z/3 cells—the use of equal cell-equivalents of fractions gave very different quantitative results from those obtained with equal amounts of protein. Without any treatment, only traces of a kB-specific DNA-binding activity were detected in the nuclear and cytosolic fractions of HeLa cells and no activity was observed in the postnuclear membrane fraction (FIG. 33, lanes 1 to 3). Upon TPA stimulation of cells under the same conditions as for 70Z/3 cells, NF-kB activity was strongly increased in the nuclear extract (FIG. 33, lane 4). Also, in the cytosolic fraction, a significant increase of NF-kB activity was found (FIG. 33, lane 5). This was not an artifact of fractionation because the activity of AP-1, another nuclear factor (Lee, W. et al., <i>Cell, </i>49:741-752 (1987), was highly enriched in nuclear extracts and almost not detectable in the cytosolic fraction of HeLa cells before and after TPA stimulation.</p>
    <p>The treatment of control fractions of HeLa cells with formamide and DOC revealed large amounts of kB-specific DNA-binding activity in the cytosolic fraction (FIG. 33, compare lanes 8 and 20). The concentrations of formamide and DOC required for an optimal in vitro activation of NF-kB in HeLa cells were different from those required for 70Z/3 cells; less formamide and more DOC was needed. All DNA-binding activities described were specific for the kB-binding site in the kappa enhancer fragment (FIG. 33, lanes 13 to 24).</p>
    <p>Almost no activity was detected in the HeLa nuclear extract and the postnuclear membrane fraction after in vitro activation (FIG. 33, lanes 7 and 9). Large amounts of NF-kB activity could still be activated in the cytosolic fraction of TPA-stimulated HeLa cells (FIG. 33, lane 11). This suggests that in vivo in HeLa cells—as contrasted to 70Z/3 cells—only a minor portion of the total cellular NF-kB is activated upon a TPA stimulus. The NF-kB activity in formamide/DOC-treated nuclear extracts of TPA-stimulated cells was less, compared to untreated nuclear extracts (FIG. 33, compare lanes 4 and 10), reflecting a partial inhibition of the DNA-binding activity of in vivo activated NF-kB. As in 70Z/3 cells, the total cellular NF-kB activity in HeLa cells, as revealed after in vitro activation, remained constant before and after TPA treatment of cells. These data imply that NF-kB is activated by the same mechanism in HeLa cells as it is in the pre-B cell line 70Z/3. However, in HeLa cells, TPA is much less complete in its activation than it is in 70Z/3 cells.</p>
    <p>NF-kB Occurrence and Activation in Other Cell Types</p>
    <p>NF-kB occurrence and activation in several additional cell types, including two T cell lines (H9, Jurkat) and fibroblasts, and in tissues, including human placenta and mice kidney, liver, spleen, lung, muscle and brain, were also assessed, as described above. In each case, NF-kB in a DOC-activatable form was shown to be present in the cytosolic fraction.</p>
    <p>Appearance of Binding Activity</p>
    <p>Results described above suggested that the appearance of binding activity may be due to separation of NF-kB from an inhibitor. Size fractionation and denaturing agents were both shown to be capable of separating NF-kB from such an inhibitor, which was apparently of low molecular weight. This provides a reasonable explanation for how NF-kB is induced in pre-B cells, HeLa cells and other inducible cells, such as T cells.</p>
    <p>Whether the DOC-dependence of cytosolic NF-kB results from its association with an inhibitor, was investigated by probing for activity in cytosolic fractions that would specifically prevent DNA binding to NF-kB in electrophoretic mobility shift assays (EMSA). This work demonstrated the existence of a protein inhibitor, called IkB, in cytosolic fractions of unstimulated pre-B cells, that can convert NF-kB into an inactive DOC-dependent form by a reversible, saturable, and specific reaction. The inhibitory activity becomes evident after selective removal of the endogenous cytosolic NF-kB under dissociating conditions, suggesting that NF-kB and IkB were present in a stoichiometric complex. Enucleation experiments showed that the complex of NF-kB and IkB is truly cytoplasmic. The data are consistent with a molecular mechanism of inducible gene expression by which a cytoplasmic transcription factor-inhibitor complex is dissociated by the action of TPA, presumably through activation of protein kinase C. The dissociation event results in activation and apparent nuclear translocation of the transcription factor. It would appear that IkB is the target for the TPA-induced dissociation reaction. The following is a description of this investigation, which is described in greater detail in Example 12.</p>
    <p>Separation of an Inhibitor from NF-kB</p>
    <p>Cytosolic fractions from unstimulated 70Z/3 pre-B cells were examined for an activity that would impair the DNA binding activity of added NF-kB in an EMSA. Baeuerle, P. A., and D. Baltimore, <i>Cell </i>53: 211 (1988). Increasing amounts of cytosol from unstimulated cells did not significantly influence the formation of a protein-DNA complex between NF-kB and a k enhancer fragment (FIG. 34, lanes 13 to 15). This indicated the absence of free inhibitor, presumably because all of it is complexed with endogenous NF-kB. DNA-cellulose was used to selectively remove the endogenous NF-kB from DOC-treated cytosol, in an attempt to liberate the inhibitor. Almost all NF-kB was present in a DOC-dependent form, prior to DOC activation and chromatography (FIG. 34A, lanes 1 and 2). In the presence of excess DOC, about 80% of the NF-kB activity was retained by DNA-cellulose (FIG. 34A, compare lanes 2 and 4), most of which eluted from the DNA-cellulose between 0.15 and 0.35M NaCl (FIG. 34A, lanes 8 to 10 and 16 to 18). The NF-kB activity eluting at high salt was detectable in mobility shift assays in the absence of excess DOC (FIG. 34A, lanes 8 to 11), indicating that NF-kB had been separated from an activity that caused its DOC-dependent DNA binding activity. In contrast, the small percentage of NF-kB activity contained in the washings was still dependent on DOC (FIG. 34A, compare lanes 5 to 7 and 13 to 15). These results show that affinity chromatography is sufficient to convert DOC-dependent NF-kB precursor into DOC-independent active NF-kB, similar to that found in nuclear extracts from TPA-stimulated cells.</p>
    <p>The flow-through fraction from the DNA-cellulose was assayed for an activity that, after neutralization of DOC by non-ionic detergent, would inactivate added NF-kB from the 0.2M NaCl-fraction from nuclear extracts of TPA-stimulated cells. Increasing amounts of cytosol from which the endogenous NF-kB was removed inhibited the formation of an NF-kB—DNA complex as monitored by EMSA (FIG. 34B, lanes 7 to 9 and 16 to 18). DOC-treated cytosol that was not passed over DNA-cellulose had no effect (FIG. 34B, lanes 4 to 6 and 13 to 15), even if cells had been treated with TPA. The fact that, after DNA-cellulose chromatography of DOC-treated cytosol, both DOC-independent NF-kB and an inhibitory activity were observed made it reasonable to believe that NF-kB had been separated from an inhibitor. This inhibitor is referred to as IkB.</p>
    <p>IkB Characterization</p>
    <p>IkB fractionates as a 60 to 70 kD protein. The flow-through fraction from the DNA-cellulose column was subjected to gel filtration through G-200 Sephadex and the fractions were assayed for an activity that would interfere with the DNA binding activity of added NF-kB contained in a nuclear extract from TPA-stimulated 70Z/3 cells (FIG. <b>35</b>A). The 67 kD fraction had the highest activity: it virtually completely prevented interaction of NF-kB and DNA (FIG. 35A, lanes 6). In fractions from a G-75 Sephadex column, no additional inhibitor of low molecular size was detectable indicating that NF-kB was inactivated by a macromolecule of defined size. No significant inhibitory activity could be demonstrated after gel filtration of a DNA-cellulose flow-through of DOC-treated cytosol from TPA-stimulated 70Z/3 cells, implying that TPA treatment of cells inactivated IkB.</p>
    <p>The inhibitor fraction was treated with trypsin to test whether IkB is a protein (FIG. <b>35</b>B). Tryptic digestion was stopped by the addition of bovine pancreas trypsin inhibitor (BPTI) and samples were analyzed for NF-kB inhibition. Trypsin treatment interfered with the activity of IkB, as shown by the complete inability of the treated sample to inhibit NF-kB activity (FIG. 35B, compare lanes 1 and 6). Trypsin that had been treated with BPTI had no effect (FIG. 35B, lane 5), demonstrating that the inactivation of IkB was specifically caused by the proteolytic activity of trypsin. It appears that IkB requires an intact polypeptide structure for its activity. The nucleotide sequence of the IkB-α gene and the amino acid sequence of IkB-α are shown in FIG. <b>43</b>.</p>
    <p>The cytosolic complex of IkB and NF-kB showed an apparent size of about 120 to 130 kD, both after gel filtration (FIG. 35A, lane 3) and after sedimentation through a glycerol gradient (FIG. 35C, lanes 6 and 7). For both methods, cytosol from unstimulated cells was analyzed under non-dissociating conditions. NF-kB was activated in fractions by either DOC (FIG. 35A) or formamide (FIG. 35C, middle panel) prior to analysis by EMSA. Baeuerle, P. A. and D. Baltimore, <i>Cell, </i>53:211 (1988). The specificity of complexes was tested with a mutant DNA probe (FIG. 35C, right panels). Lenardo, M. et al., <i>Science, </i>236:1573 (1987). The apparent release of a 60 to 70 kD inhibitory protein from the cytosolic NF-kB precursor, its sedimentation velocity in glycerol gradients, and its size seen by gel filtration suggest that the inactive NF-kB precursor is a heterodimer composed of a 55 to 62 kD NF-kB molecule and a 60 to 70 kD IkB molecule. Nuclear NF-kB was found to cosediment with the cytosolic complex of IkB and NF-kB (FIG. 35C, upper panel). Native gel electrophoresis, a method that allows resolution of size differences of protein-DNA complexes, provided evidence that the 120 kD form of nuclear NF-kB seen in glycerol gradients comes from the formation of a homodimer. Hope, I. A. and K. Struhl, <i>EMBO J., </i>6:2781 (1987). By these interpretations, activation of NF-kB would include an additional step (i.e., formation of a NF-kB homodimer). This is consistent with the observation that the protein-DNA complexes formed with in vitro-activated NF-kB have the same mobility in native gels as those formed with nuclear NF-kB. Baeuerle, P. A. and D. Baltimore, <i>Cell, </i>53:211 (1988).</p>
    <p>The inactivation of NF-kB by IkB is reversible, saturable and specific. Incubation with the inhibitor fraction can inhibit the DNA binding activity of NF-kB by more than 90% (FIG. 36A, lanes 1 and 3). Treatment of a duplicate sample with DOC after the inhibition reaction reactivated 66% of the added NF-kB activity (FIG. 36A; compare lanes 3, 4 and 6). This showed that a DOC-dependent form of NF-kB can be reconstituted in vitro by the addition of a fraction containing IkB to nuclear NF-kB. The incomplete activation of NF-kB by DOC might be due to the DOC-neutralizing effect of non-ionic detergent which was still present in the sample from the preceding inhibition reaction.</p>
    <p>A titration and kinetic analysis showed that IkB stoichiometrically interacts with NF-kB (FIG. <b>36</b>B). Increasing amounts of inhibitor fraction were added to an excess amount of NF-kB and incubated for 20 or 60 minutes. After the DNA binding reaction, NF-kB-DNA complexes were separated on native gels and quantified by liquid scintillation counting. The relationship between amount of IkB fraction added and extent of inhibition was linear. The amount of NF-kB inactivated after 20 minutes of incubation was not increased after 60 minutes (FIG. <b>36</b>B). These kinetics were probably not the result of a rapid decay of a catalytically active inhibitor because the fractions were incubated prior to the reaction. The data are consistent with rapid formation of an inactive complex by addition of IkB to NF-kB. The fraction containing IkB does not appear to catalytically or covalently inactivate NF-kB: neither the reversibility nor the kinetics support such a model.</p>
    <p>IkB was tested for its influence on the DNA binding activity of other defined nuclear factors (FIG. <b>13</b>A). These factors were contained in nuclear extracts that had essentially no active NF-kB, which otherwise could have inactivated IkB by complex formation. The DNA binding activity of H2TF1, a transcription factor thought to be related to NF-kB, was not affected by the inhibitor fraction. Baeuerle, P. A. et al., unpublished observation). Ubiquituous and lymphoid-specific octamer-binding proteins (OCTA) (Sive, H. L. and R. G. Roeder, <i>Proc. Natl. Acad. Sci. USA, </i>83:6382 (1986) and Staudt, L. M. et al., <i>Nature, </i>323:640 (1986)) were unaffected in their DNA binding activities, as were two E-box binding factor, NF-μE1 (Weinberger, J. et al., <i>Nature, </i>322:846 (1986)) and NF-kE2 (Lenardo, M. et al., <i>Science, </i>236:1573 (1987)), interacting with μ heavy chain and k light chain enhancers, respectively. AP-1, another TPA-inducible transcription factor (Lee, W. et al., <i>Cell, </i>49:741 (1987); Angel, P. et al., <i>Cell, </i>49:729 (1987)), also showed equal complex formation after incubation in the presence and absence of the inhibitor fraction. Furthermore, none of the undefined DNA binding activities seen in the EMSA showed any inactivation by IkB. These results show that IkB is a specific inhibitor of the DNA binding activity of NF-kB.</p>
    <p>In vivo activated NF-κB is responsive to IkB. IkB prepared from the mouse pre-B cell line 70Z/3 was tested for inactivation of NF-kB contained in nuclear extracts from other cell lines. Human NF-kB contained in nuclear extracts from TPA-stimulated HeLa cells and H-9 T-lymphoma cells was efficiently inactivated (FIG. <b>37</b>B). When excess amounts of the various NF-kB activities were used in the inhibitor assay, the extent of reduction of NF-kB activities by a fixed amount of IkB was very similar, as quantified by liquid scintillation counting. NF-kB from nuclear extracts of TPA-stimulated Madin-Darby bovine kidney (MDBK) cells was also inactivated suggesting that the control of NF-kB activity by IkB is conserved among different mammalian species.</p>
    <p>NF-kB is constitutively active in cell lines derived from mature B cells. Sen, R. and D. Baltimore, <i>Cell, </i>46:705 (1986). Nuclear extracts from the mouse B cell line WEHI 231 were tested in the inhibitor assay to examine whether NF-kB has undergone a modification in those cell lines that prevented its inactivation by IkB. NF-kB from B cells was as efficiently inactivated as NF-kB from pre-B cells (FIG. <b>37</b>B), suggesting that NF-kB is not stably modified in B cells (or in other cells after TPA stimulation) in such a way that it cannot respond to inactivation by IkB.</p>
    <p>The NF-kB—IkB complex is present in enucleated cells. The NF-kB—IkB complex shows a cytosolic localization on subcellular fractionation (FIG. <b>38</b>A). This procedure may, however, cause artifacts. Hypotonic lysis of cells may result in partitioning of nuclear proteins into the cytosol, especially, when they are not tightly associated with nuclear components. Li, J. J. and T. J. Kelly, <i>Proc. Natl. Acad. Sci, USA, </i>81:6973 (1984). Detection of the complex of IkB and NF-kB in enucleated cells was attempted. Enucleation is performed with living cells at 37° C. and should therefore not interfere with active nuclear import of proteins, which is ATP-dependent and blocked at low temperature. Prescott, D. M. and J. B. Kirkpatrick, In: <i>Methods Cell Biol</i>., D. M. Prescott, ed. (Academic Press, New York, 1973), p. 189; Newmeyer, D. D. and D. J. Forbes, <i>Cell, </i>52:641 (1987); Richardson, W. D. et al., <i>Cell, </i>52:655 (1988).</p>
    <p>Using cytochalasin B-treated HeLa cells, an enucleation efficiency of about 90% was obtained (FIG. <b>38</b>A). Enucleated and cytochalasin B-treated complete cells were incubated in the absence and presence of TPA, solubilized by detergent and proteins were extracted with high salt. Because of the small number of cells analyzed, this procedure is different from the standard one. Total cell extracts were analyzed for NF-kB specific DNA binding activity by EMSA (FIG. <b>38</b>B). In both enucleated and complete cells, similar amounts of NF-kB activity were found after TPA stimulation (FIG. 38B, lanes 1 to 4). The activity was specific for NF-kB because it was not observed with a mutant k enhancer fragment. Lenardo, M. et al., <i>Science, </i>236:1573 (1987); These results suggest that TPA-inducible NF-kB in HeLa cells is predominantly cytoplasmic because it was still present in enucleated cells. The NF-kB activity seen under control conditions (FIG. 38B, lanes 1 and 3) was most likely activated by the lysis conditions used because it was also observed in extracts from HeLa cells that were not treated with cytochalasin B, but not in fractions obtained after hypotonic lysis. Baeuerle, P. A. and D. Baltimore, <i>Cell, </i>53:211 (1988). It was still evident, however, that TPA could activate NF-kB in enucleated cells (FIG. 38B, lanes 3 and 4).</p>
    <p>After treatment with DOC, total extracts from complete and enucleated control cells showed about a 2-fold increase in the amount of NF-kB activity (FIG. 38B, compare lanes 1 and 3 with 5 and 7). The demonstration of DOC-activatable NF-kB in enucleated cells, as well as the presence of similar amounts of total NF-kB in enucleated and complete cells (FIG. 38B, compare lanes 5 to 8), shows that a substantial amount of the total cellular NF-kB—IkB complex was cytoplasmic. In contrast to NF-kB, most of the DNA binding activity of AP-1, a bona fide nuclear protein, was apparently lost by enucleation of cells (FIG. 38B, lanes 9 to 12). Lee, W. et al., <i>Cell, </i>49:741 (1987); Angel, P. et al., <i>Cell, </i>49:729 (1987).</p>
    <p>Mechanism of NF-kB Activation</p>
    <p>Thus, it has been shown that the NF-kB nuclear transcription factor exists in unstimulated pre-B cells in a cytoplasmic complex with a specific inhibitory protein, IκB. In this complex, NF-kB does not exhibit DNA binding activity in EMSA and partitions upon subcellular fractionation into the cytosol. The complex is apparently a heterodimer consisting of about a 60 kD NF-kB molecule and a 60 to 70 kD IkB molecule. Upon TPA stimulation of cells, or after treatment with dissociating agents in vitro, the NF-kB—IkB complex dissociates. This releases NF-kB, which appears now to form a homodimer and can translocate into the nucleus. Whether dimerization is required for activation of NF-kB is not known.</p>
    <p>The inhibitory effect of IkB on the DNA binding activity and nuclear localization properties of NF-kB appears to arise from a simple physical affinity of the two proteins. The complex freely dissociates and the components readily associate under in vitro conditions. Even in vivo, dissociation by short-term TPA treatment and reassociation after long-term TPA treatment is evident. The latter presumably results from the degradation of protein kinase C after TPA activation and implies that NF-kB can move back to the cytoplasm after being active in the nucleus.</p>
    <p>The effect of TPA appears to involve an alteration of IkB, but not of NF-kB. After TPA stimulation, no active IkB was found—implying its alteration—while the nuclear NF-κB remained sensitive to unmodified IkB when tested in vitro. Whether inactive IkB can be regenerated is unclear; in experiments using cycloheximide (Baeuerle, P. A. et al., <i>Cold Spring Harbor Symp. Quant. Biol., </i>53, In Press), irreversible loss of IkB activity was the only demonstrable effect after 8 hours of TPA treatment. Given the ability of TPA to activate protein kinase C, it is a reasonable hypothesis that direct or indirect phosphorylation of IkB results in its dissociation from NF-kB.</p>
    <p>It had previously been found that the NF-kB—IkB complex is recovered in the cytosol. It is now shown directly that the complex is not removed from the cell by enucleation and, therefore, is truly cytoplasmic. Welshons, W. V. et al., <i>Nature, </i>307:747 (1984). Because active protein kinase C is bound to the plasma membrane (Kraft, A. S. et al., <i>J. Biol. Chem., </i>257:13193 (1983); Wolf, M. et al., <i>Nature, </i>317:546 (1985); Kikkawa, U. and Y. Nishizuka, <i>Ann. Rev. Cell. Biol., </i>2:149 (1986)), it becomes increasingly attractive to suggest that the cytoplasmic complex interacts in the cytoplasm (maybe near the plasma membrane) with protein kinase C and the liberated NF-kB carries the signal from cytoplasm to nucleus. Under a number of conditions, active NF-kB is found in the cytoplasm. This fact and the reversibility of NF-kB activation in vivo suggests that the protein may freely move in and out of the nucleus, bringing to the nucleus information reflecting the cytoplasmic activation state of protein kinase C and possibly of other signalling systems.</p>
    <p>The response of NF-kB to activated protein kinase C occurs apparently indirectly through modification and subsequent release of associated IkB. The inducibility of NF-kB by TPA is thus dependent on the presence and state of activity of IkB. Changes in amount or activity of IkB should therefore influence the TPA inducibility of NF-kB. NF-kB can indeed exist not only in TPA-inducible but also in constitutively active form (e.g., in mature B cells; Sen, R. and D. Baltimore, <i>Cell, </i>46:705 (1986). Because constitutive NF-kB from B cells is still responsive to IkB in vitro, it is thought that IkB, and not NF-kB, is altered during differentiation of pre-B into B cells.</p>
    <p>IkB is apparently unstable when not complexed with NF-kB. This is suggested by the absence of excess active inhibitor in the cytosol from unstimulated cells. In a situation where the production of new inhibitor is impaired, the decay of occasionally released inhibitor could activate NF-kB. This would explain the partial activation of NF-kB seen after treatment with the protein synthesis inhibitors cycloheximide and anisomycin. Sen R. and D. Baltimore, <i>Cell, </i>47:921 (1987). The demonstration of a specific inhibitory protein of NF-kB and the interpretation that cycloheximide treatment can activate NF-kB, presumably because cells become depleted of inhibitor, suggest that IkB is the putative labile repressor of k gene expression (Wall, R. et al., <i>Proc. Natl. Acad. Sci. USA, </i>83:295 (1986)) and of NF-kB activity. Sen, R. and D. Baltimore, <i>Cell, </i>47:921 (1987).</p>
    <p>As a result of the work described herein, the IκB gene is now available, as is IκB itself, antibodies specific for the IκB gene-encoded product, and probes which include all or a portion of the IκB gene sequence. Also available are methods of using the IκB gene, the encoded protein and IκB-specific antibodies for such purposes as identifying and isolating other IκB genes, IκB “like” genes, and IκB-encoded products. Altering NF-κB activity and altering NF-κB-mediated gene expression. In particular, it is now possible, through the method of the present invention, to block or inhibit NF-κB passage into the nucleus of cells in which it occurs and, thus, block (partially or totally) binding of NF-κB to NF-κB binding sites on genes which include such recognition sites. Such a method is useful for altering expression of genes which is mediated by NF-κB; such genes include cellular genes (e.g., cytokine genes) and genes introduced into host cells (e.g., viral genes, such as cytomegalovirus gene, HIV-1 genes (e.g., the tax gene), and the SV40 gene). This method of altering NF-κB-mediated gene expression is useful, for example, for inhibiting viral gene expression in infected cells, such as in an individual infected with the HIV-1 or cytomegalovirus.</p>
    <p>The IκB gene and the encoded IκB protein can be used to negatively regulate NF-κB activity in cells. For example, the IκB gene can be incorporated into an appropriate vector (e.g., a retroviral vector or capable of expressing the IκB gene and introduced into cells in which NF-κB activity is to be inhibited (partially or totally). For example, a vector capable of expressing IκB can be introduced into HIV-1 infected cells (e.g, T cells) in order to inhibit HIV-1 gene expression and activity in the cells. IκB expressed in the cells binds NF-κB (e.g., free NF-κB such as that released from its inactive complex with Iκ-β) and limits its ability to act as a messenger by inhibiting its translocation into the nucleus. For this purpose, all or a portion of the IκB-encoding DNA or DNA encoding an IκB-like protein is used. If a portion is used, it must encode at least that region of the IκB (or other rel-associated protein) molecule sufficient to bind NF-κB and prevent it from passing into the cell nucleus. The IκB-encoding DNA or DNA encoding an IκB-like protein used can be obtained from a source in which it naturally occurs, can be produced by genetic engineering or recombinant techniques or can be synthesized using known chemical methods. For convenience, DNA from all three types of sources is referred to herein as “essentially pure”. The DNA used can have all or a portion of the DNA sequence of clone MAD-3, all or a portion of pp40 or all or a portion of another sequence which encodes a rel-associated or IκB-like protein capable of inhibiting NF-κB. In a similar manner, DNA encoding a rel-associated or IκB-like protein can be introduced into cells to inhibit a rel-related protein other than NF-κB.</p>
    <p>Cells in which IκB (or other rel-associated protein) is to be expressed in this manner to inhibit NF-κB (or other rel-related protein) can be removed from the body, the IκB-expressing vector can be introduced, using known methods, and the resulting cells, which contain the IκB-expressing vector, then reintroduced into the body. For example, T-cells or bone marrow cells can be removed from an HIV-1 infected individual, IκB-expressing vectors can be introduced into them, and they can then be replaced in the individual. Alternatively, the expression vector containing IκB-encoding DNA can be introduced into an individual, using known techniques, by any of a variety of routes, such as intramuscular, intravenous, intraperitoneal administration. IκB itself (or other rel-associated protein) can also be introduced into cells to inhibit NF-κB (or other rel-related protein). The entire IκB molecule or a portion sufficient to bind NF-κB and prevent its passage into the nucleus can be used for this purpose. IκB or an appropriate IκB portion can be obtained from naturally-occurring sources, can be produced using known genetic engineering methods or, particularly in the case of an IκB portion, can be synthesized chemically. For convenience, proteins (or portions thereof) of all three types are referred to herein as “essentially pure”.</p>
    <p>Uses of Genes Encoding Transcriptional Regulatory Factors, the Encoded Factors and Related Products</p>
    <p>The genes encoding positive transcriptional regulatory factors provide a means for enhancing gene expression. Lymphoid-specific factors involved in positive regulation of Ig gene transcription provide a method for enhancing immunoglobulin production in lymphoid cells. Lymphoid cells, such as monoclonal antibody-producing hybridomas or myelomas, can be transfected with multiple copies of a gene encoding a regulatory factory to induce greater production of Ig. For this purpose, the gene encoding a regulatory factor can be linked to a strong promoter. In addition, the construct can include DNA encoding a selectable marker. Multiple copies of the contruct can be inserted into the cell, using known transfection procedures, such as electroporation. The cell can be transfected with multiple regulatory factors, including constitutive factors; this is particularly useful in the case of factors determined to act in conjunction, possibly synergistically. Amplification of genes encoding transcriptional regulatory factors in this manner results in enhanced or increased production of the regulatory factors and, consequently, production of immunoglobulin is enhanced in these cells.</p>
    <p>The present invention also relates to a method for transiently expressing a gene product in a eukaryotic cell, in which the inducibility of the NF-kB factor is used to advantage. This phenomenon can be exploited to provide for the transient overexpression of a gene product produced by a transfected gene in a eukaryotic cell at a chosen time.</p>
    <p>According to the method of this invention, a gene of interest is placed under influence of the κ enhancer sequence containing the binding site for NF-kB (i.e., the entire enhancer sequence or a portion containing at least the NF-κB site). The κ-enhancer sequence is linked to a structural gene of interest to provide an gene inducible by NF-kB. A gene construct is thus provided comprising 1) a κ-enhancer sequence or a portion of the κ enhancer sequence containing at least the sequence to which the factor NF-kB binds; 2) a promoter; and in 3) structural gene of interest.</p>
    <p>Conventional recombinant DNA techniques can be used to prepare the construct. The κ enhancer sequence can be obtained from lymphoid cells which express the κ-light chain. The κ enhancer can also be obtained from clones containing the sequence. The construct can be prepared in or inserted into a transfection vehicle such as a plasmid.</p>
    <p>The structural gene can be any gene or gene segment which encodes a useful protein for which transient overexpression is desired. Such proteins are, for example, those that are damaging to cells when produced constitutively. The structural gene can be used with its endogenous promoter or other eukaryotic promoter.</p>
    <p>Cells for transfection can be any eukaryotic cells used for the expression of eukaryotic proteins. Transfection procedures, such as the calcium precipitation technique are well known in the art.</p>
    <p>As the desired time, the transfected cells can be stimulated with the appropriate inducer in an amount sufficient to induce production of NF-kB. The preferred inducer is a phorbol ester which acts rapidly and directly to activate protein kinase C and induces production of NF-kB. If the transfected cell is a lymphoid cell (e.g., B cell) responsive to a mitogen such as LPS or PHA the mitogen may be used alone or in combination with phorbol ester.</p>
    <p>Genes encoding transcriptional regulatory factors can be modified for a variety of purposes, such as to encode factors with activity equivalent to the naturally-occurring factor, factors with enhanced ability to regulate transcription (e.g., to cause enhanced transcription of genes, relative to transcription resulting from regulation by or the effects of the normal/unmodified factor), or factors with decreased ability to regulate transcription. This can be carried out, for example, by mutagenesis of factor-encoding DNA or by producing DNA (e.g., by recombinant DNA methods or synthetic techniques) which encodes a modified or mutant transcriptional regulatory factor (i.e., a transcriptional regulatory factor with an amino acid sequence different from the normal or naturally-occurring transcription factor amino acid sequence). These modified DNA sequences and encoded modified factors are intended to be encompassed by the present invention.</p>
    <p>The gene encoding IgNF-b, for example, has been cloned and sequenced and the nucleotide sequence is shown in FIG. <b>18</b>A. For the various utilities discussed below, the modified nucleotide sequence can be obtained either naturally (e.g., polymorphic variants) or by mutagenesis to yield substantially complementary sequences having comparable or improved biological activity. Fragments of the sequence may also be used. This invention encompasses nucleic acid sequences to which the sequence of FIG. 18A hybridizes in a specific fashion.</p>
    <p>In addition, the DNA binding domain of the factors, which is responsible for the binding sequence-specificity, can be combined with different “activators” (responsible for the effect on transcription) to provide modified or hybrid proteins for transcriptional regulation. For example, with recombinant DNA techniques, DNA sequences encoding the binding domain can be linked to DNA sequences encoding the activator to form a gene encoding a hybrid protein. The activator portion can be derived from one of the factors or from other molecules. The DNA binding region of the hybrid protein serves to direct the protein to the cognate DNA sequence. For example, in this way, stronger activators of RNA polymerases can be designed and linked to the appropriate DNA binding domain to provide for stronger enhancement of transcription.</p>
    <p>DNA probes for the genes encoding the regulatory factors can be used to determine the presence, absence or copy number of regulatory factor-encoding genes or to identify related genes, by hybridization techniques or a polymerase chain amplification method (e.g., PCR). The ability to detect and quantify genes encoding transcriptional regulatory factors can be used in diagnostic applications, such as to assess conditions relating to aberrant expression of a regulatory factor. Cells can be typed as positive or negative for the occurrence of a particular gene and, in addition, can be analyzed for the copy number of the gene. The DNA probes are labeled DNA sequences complementary to at least a portion of a nucleic acid encoding a transcriptional regulatory factor. The labeled probe is contacted with a sample to be tested (e.g., a cell lysate) and incubated under stringent hybridization conditions which permit the labeled probe to hybridize with only DNA or RNA containing the sequence to which the probe is substantially complementary. The unhybridized probe is then removed and the sample is analyzed for hybridized probe.</p>
    <p>The DNA probes can also be used to identify genes encoding related transcriptional regulatory factors. For this purpose, hybridization conditions may be relaxed in order to make it possible to detect related DNA sequences which are not completely homologous to the probe.</p>
    <p>Antibodies can be raised against the transcriptional regulatory factors of this invention. The antibodies can be polyclonal or monoclonal and they can be used as diagnostic reagents in assays to determine whether a factor is expressed by particular cells or to quantitate expression levels of a factor.</p>
    <p>A gene encoding a transcriptional regulatory factors can also be used to develop in vivo or in vitro assays to screen for agonists or antagonists of a factor-encoding gene or of the factor encoded by the gene. For example, genetic constructs can be created in which a reporter gene (e.g., the CAT gene) is made dependent upon the activity of a factor-encoding gene. These constructs introduced into host cells provide a means to screen for agonists or antagonists of the factor-encoding gene. The antagonists may be used to decrease the activity of the factors and thus may be useful in the therapy of diseases associated with overactivity of a transcriptional regulatory factor. Such agonists or antagonists identified by assays employing the factor-encoding genes of this invention are within the scope of this invention.</p>
    <p>The present invention is useful as a means of controlling activation in a host cell of an NF-kB precursor, which results in formation of activated NF-kB, which, in turn, plays a key role in transcriptional activation of other sequences, such as the k light chain enhancer, the HIV enhancer and the interleukin-2 receptor α-chain gene. NF-kB has been shown to be a ubiquitous inducible transcription factor; it has been shown, as described herein, to be present in many types of cells (i.e., all cell types assessed to date). It serves to make immediate early responses which it is capable of effecting because it is post-translationally activated. As a result, the method and composition of the present invention can be used to control transcriptional activation of genes encoding a selected cellular protein. Changes in expression of genes transcribed by RNA polymerase II in response to agents, such as steroid hormones, growth factors, interferon, tumor promoters, heavy metal ions and heat shock are mediated through cis-acting DNA sequence elements such as enhancers. Binding of NF-kB transcription factor has been shown to confer transcriptional activity on several genes. Expression of these genes and others similarly affected can be controlled by the present method. For example, it has been shown that expression of one of the two elements of the cell surface receptor specific for IL-2 is controlled by NF-kB. Thus, in T cells, which produce IL-2, production can be controlled (enhanced, reduced) by controlling activation of NF-kB. In a similar manner, the method of the present invention can be used to control expression of human immunodeficiency virus in infected host cells.</p>
    <p>Methods and compositions of the present invention are based on use of the role of NF-κB as a second messenger, or mediator, in the expression of genes in a wide variety of cell types. The expression of a gene having an NF-κB binding recognition sequence can be positively or negatively regulated to provide, respectively, for increased or decreased production of the protein whose expression is mediated by NF-κB . Furthermore, genes which do not, in their wild type form, have NF-κB,recognition sequences can be placed under the control of NF-κB by inserting NF-κB binding site in an appropriate position, using techniques known to those skilled in the art.</p>
    <p>DNA sequences known to contain NF-κB binding domains are shown in Table 2. According to the methods described herein, the expression of genes under the control of one of these elements is subject to modulation by alteration of the concentration or availability of NF-κB. This can also be carried out, according to the present method, for any gene which contains an NF-κB binding site. Furthermore, genes which do not naturally contain NF-κB binding sites can be modified, using known techniques, to subject these genes to NF-κB modulation. First, an appropriate expression vector is selected for use in a biological system of interest, the vector having a gene of interest and restriction enzyme recognition sequences to facilitate the insertion of a DNA fragment carrying the NF-κB binding sites.</p>
    <p>For example, the sequences of the κ immunoglobulin enhancer, the SV40 70 base pair repeat, the HIV long terminal repeat, the MHC class I H2-kb gene and the interferon β PRDII gene, all possess NF-κB binding sites (Table 2). By comparing sequences to which NF-κB binds specifically, a consensus sequence has been determined:</p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" tabstyle="monospace" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="84pt" align="left"> </colspec> <colspec colname="1" colwidth="133pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">    C   C</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">GGGRATYYAC.</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">    T   T</td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p>DNA sequences which flank the binding site are scanned for convenient restriction enzyme recognition sequences to facilitate removal of the fragment from the longer sequence in which occurs and its subsequent insertion into the expression vector. If such sequences are present, the transfer of the fragment carrying the binding site, to the expression vector, is straight forward. If convenient sites do not exist, fragment transfer is facilitated through the introduction of such restriction enzyme recognition sequences using well known, site-directed mutagenic techniques. The construct, prepared as described, can then be introduced into a biological system of interest.</p>
    <p>The expression of such constructs in a biological system is subject to modulation by NF-κB. For example, purified NF-κB could be introduced into the system in an effective amount such that any inhibitory molecule present in the system would be titrated out and uninhibited NF-κB could interact with its binding recognition sequence, thereby increasing the rate of transcription. This is an example of positive regulation.</p>
    <p>Similarly, a copy of the NF-κB gene, cloned in an appropriate expression vector, could be introduced into the biological system, thereby providing for internal expression of the NF-κB molecule, preferably at relatively high levels. Again, high levels of NF-κB would function to titrate out any inhibitor molecule present, and also to increase the rate of transcription from a gene possessing a NF-κB binding site.</p>
    <p>A level of discrimination among members of a related family of NF-κB binding sites, by a modified NF-κB molecule, can also be introduced. Referring to Table 1, for example, there are apparent differences among the various NF-κB binding sites from various genes. A copy of a cloned NF-κB gene can be mutagenized to alter the binding domain by well known techniques, such as site or region directed mutagenesis. Alternatively, DNA fragments encoding a modified NF-κB binding domain can be made synthetically, having appropriate cohesive or blunt termini to facilitate insertion into the NF-κB gene to replace the existing sequences encoding the corresponding portion of the binding domain. Such restriction fragments can be synthesized having any desired nucleotide changes. Mutated proteins encoded by such genes can be expressed and assayed for preferential binding to, for example, one of the 10 different DNA binding sites shown in Table 1 or related members of the family of NF-κB binding sites. An example of an assay which can be used to screen large numbers of recombinant clones in order to identify binding domain mutants is that described by Singh et al., (<i>Cell, </i>52:415-423 (1988)). Once such a mutant is identified, a DNA expression vector encoding this mutant protein can be introduced into a cell. The mutant protein will preferentially bind to the selected member or members of the family of DNA binding sites, such as those shown in Table 2, thereby preferentially enhancing transcription from only those genes which contain that particular binding site.</p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" tabstyle="monospace" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 2</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Sequences recognized by NF-κB.</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="offset" colwidth="21pt" align="left"> </colspec> <colspec colname="1" colwidth="119pt" align="left"> </colspec> <colspec colname="2" colwidth="77pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Gene</td>
                <td class="description-td">Sequence</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="OFFSET" nameend="2" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Ig κ enhancer - mouse</td>
                <td class="description-td">GGGGACTTTCC</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">SV40 enhancer</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">HIV-1 (−91)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">CMV (4)<sup>1,2</sup> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">HIV-1 (−105)</td>
                <td class="description-td">AGGGACTTTCC</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">HIV-2</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">CMV (1)<sup>1</sup> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">β2-microglobulin</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">serum amyloid A -g9</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Ig κ enhancer - human</td>
                <td class="description-td">GGGGATTTCC</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">CMV (3)<sup>1</sup> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Interferon-β- PRDII</td>
                <td class="description-td">GGGAAATTCC</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">CMV(2)<sup>1</sup> </td>
                <td class="description-td">GGGACTTTCC</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">MHC class II-E<sub>α</sub> <sup>d</sup> </td>
                <td class="description-td">GGGACTTCCC</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">IL-2 lymphokine</td>
                <td class="description-td">GGGATTTCAC</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">mouse IL-2Rα</td>
                <td class="description-td">GGGGATTCCT</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">human IL-2Rα</td>
                <td class="description-td">GGGAATCTCC</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">MHC class I - H2 - K<sup>b</sup> </td>
                <td class="description-td">GGGATTCCCC</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">HLA - A2, A11, B7</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">B27, B51</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">    C   C</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">CONSENSUS<sup>3</sup>:</td>
                <td class="description-td">GGGRATYYAC</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">    T   T</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="OFFSET" nameend="2" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="OFFSET" nameend="2" align="left" class="description-td" colspan="3"> <sup>1</sup>In this particular element, the sequence has not been tested in a binding assay. All others have been proven by direct binding and usually by inhibition of binding to the Ig κ sequence. </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="OFFSET" nameend="2" align="left" class="description-td" colspan="3"> <sup>2</sup>Since there are four putative NF-KB recognition sites in the cytomegalovirus enhancer, these have been numbered 1-4 as they are found from 5′ to 3′ on the coding strand. </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="OFFSET" nameend="2" align="left" class="description-td" colspan="3"> <sup>3</sup>Consensus is based on all sequences though the assignments of the sixth and tenth positions ignore one deviant. </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p>Negative regulation can be effected in an analogous manner. For example, a specific inhibitor molecule which is able to block (reduce or eliminate) NF-κB binding can be added to the biological system in an effective amount. Preferably, this inhibitor is specific for NF-κB and does not interact with other cell constituents. An example of such a molecule is I-κB.</p>
    <p>Alternatively, negative regulation can be effected using “decoy” molecules, which are designed to mimic a region of the gene whose expression would normally be induced by NF-κB. In this case, NF-κB would bind the decoy and, thus, not be available to bind its natural target.</p>
    <p>Furthermore, in the case of an inhibitor molecule which is also a protein, the gene encoding the inhibitor molecule can be identified, isolated, and cloned into an appropriate expression vector using common methodology. When introduced into an appropriate biological system, the inhibitor molecule is synthesized and functions to interact with NF-κB with its binding site and as a consequence reducing the level of transcription of the gene containing the NF-κB binding site.</p>
    <p>Yet another method for negatively regulating the expression of a gene containing an NF-κB binding domain involves the introduction of an effective amount of a decoy sequence encoding the NF-κB binding domain. The decoy sequence serves as an unproductive binding domain with which the NF-κB molecule binds. As the finite number of NF-κB molecules bind to the decoy sequences, the number which bind productively (result in increased transcription) with an intact gene, decreases.</p>
    <p>Negative regulation can also be effected by the introduction of “dominantly interfering” molecules (see e.g., Friedman et al., <i>Nature, </i>335:452-454 (1988). For example, if the DNA binding domain and the DNA polymerase activating domain of NF-κB are spatially distinct in the molecule, a truncated form of the NF-κB molecule can be synthesized, using well known techniques. A preferred embodiment would be a truncated molecule retaining the DNA binding domain, but lacking the RNA polymerase activating domain. Such a “dominantly interfering” molecule would recognize and bind to the NF-κB binding site, however, the binding would be non-productive. Because the activation portion of NF-κB would be required for enhanced transcription, the truncated molecule would exert no positive effect. Furthermore, its occupation of the NF-κB binding site effectively blocks access to any intact NF-κB molecule which may be present in the cell.</p>
    <p>The invention is further illustrated by the following examples, which are not intended to be limiting in any way.</p>
    <heading>EXAMPLES</heading> <heading>Example 1</heading> <heading>Identification of Nuclear Factor IgNF-A</heading> <heading>Methods</heading> <p>1. Gel Electrophoresis DNA Binding Assays with the SfaNI-SfaNI X Promoter Fragment</p>
    <p>The SfaNI fragment was subcloned into the SmaI site of pS64 (pSPIgV<sub>κ</sub>, provided by N. Speck). For binding analysis this fragment was excised from pSPIgV<sub>κ</sub> by digesting with Hind III and Eco RI. These latter sites flank the Sma I site in the polylinker of pSP64. After end-labeling with [α-<sup>32</sup>P]dATP and the large fragment of <i>E. coli </i>DNA polymerase I, the radiolabeled fragment was isolated by polyacrylamide gel electrophoresis. Binding reactions were performed and the reaction mixtures resolved by electrophoresis (FIG. 1<i>b</i>). The <sup>32</sup>Plabeled fragment (about 0.5 ng, 10,000 cpm) was incubated with a nuclear extract of a human B lymphoma cell line (EW)(prepared by the method of Dignam, J. D. et al. <i>Nucl. Acids Res. </i>11 1475-1489 (1983)) in the absence (lane 1) or presence of two different non-specific competitor DNAs (lanes 2-11). Binding reactions (25 μl) contained 10 mM Tris.HCl (pH 7.5), 50 mM NaCl, 1 mM DTT, 1 mM EDTA, 5% glycerol and 8 μg EW nuclear extract protein. Reactions 2-6 additionally contained 800, 1600, 2400, 3200 and 4000 ηg, respectively, of poly(dI-dC)-poly(dI-dC). Reactions 7-11 contained 300, 600, 900, 1200 and 1500 ηg, respectively, of Hinf I digested <i>E. coli </i>chromosomal DNA. After a 30 min incubation at room temperature, the resulting complexes were resolved in a low ionic strength 4% polyacrylamide gel (acrylamide:bisacrylamide weight ratio of 30:1) containing 6.7 mM Tris.HCl (pH 7.5), 3.3 mM Na-acetate and 1 mM Na-EDTA. See Strauss, F. and Varshavsky, A. Cell 37 889-901 (1984). The gel was preelectrophoresed for 30 min at 11V/cm. Electrophoresis was carried out at the same voltage gradient for 90 min at room temperature with buffer recirculation. The gel was then dried and auto-radiographed at −70° C. with a screen. In FIG. 1<i>b</i>, F and B indicate positions of free and bound fragments respectively.</p>
    <p>Binding assays were performed as detailed above using 2400 ng poly(dI-dC)-poly(dI-dC)) and the following DNA fragments: κ SfaNI-SfaNI (˜0.5 ηg, 10,000 cpm, lane 1), κ PvuII-KpnI (˜0.5 ηg, 10,000 cpm, lane 2), κ PvuII-SfaNI (˜0.1 ηg, 5000 cpm, lane 3) and pSP64 PvuII-EcoRI (˜0.2 ng, 5000 cpm, lane 4). The κ PvuII-SfaNI fragment was derived from the plasmid pSPIgV<sub>κ</sub> by digesting with PvuII and EcoRI. The EcoRI site is in the polylinker and therefore this fragment contains 16 bp of polylinker sequence.</p>
    <p>2. Binding Competition Analysis in Nuclear Extracts of Human EW and HeLa Cells</p>
    <p>EW nuclear extract, FIG. 2<i>a</i>. Binding assays were performed as detailed above using radiolabeled K PvuII-SfaNI fragment (˜0.1 ηg, 5000 cpm) and 2400 ηg poly(dI-dC)-poly(dI-dC). Reactions 2-4 additionally contained 50, 100 and 200 ηg, respectively, of the bacterial plasmid pSP64 whereas 5-7 contained 50, 100 and 200 ηg, respectively, of the recombinant plasmid pSPIgV<sub>κ</sub>. Assuming that a molecule of pSPIgV<sub>κ</sub> contains a single high affinity site whereas a molecule of pSP64 (3000 bp) contains 6000 non-specific sites, the apparent affinity ratio of the factor for these two types of sites is greater than 6000×200/50=12.4×10<sup>4 </sup>Hela nuclear extract, FIG. 2<i>b</i>. Binding assays were performed as detailed above using radiolabeled κ PvuII-SfaNI fragment (about 0.1 ng, 5000 cpm), 2400 ng poly(dI-dC)-poly(dI-dC) and 6 μg Hela nuclear extract protein (provided by P. Grabowski). Reactions 1 and 2 additionally contained 100 ηg of pSP64 and pSPIgV<sub>κ</sub>, respectively.</p>
    <p>3. DNase Footprinting of Factor-DNA Complexes. (FIG. 3)</p>
    <p>The B cell nuclear extract was applied to a heparin-sepharose column equilibrated with 10 mM Hepes pH 7.9, 20% glycerol, 1 mM DTT, 1 mM EDTA, 5 mM MgCl<sub>2 </sub>and 0.1 M KCl. The κ-promoter binding factor was eluted with a 0.25 M KCl step. Binding reactions with this fraction (30 μl) contained 2.5 mM MgCl<sub>2</sub>, κ PvuII-SfaNI (˜1 ηg, 100,000 cpm) and 4800 ηg poly(dI-dC)-poly(dI-dC) in addition to components detailed above. The coding strand of the κ promoter probe was 3′ end-labeled (Eco RI site) with [α-<sup>32</sup>P] dATP using the large fragment of <i>E. coli </i>DNA polymerase. Reaction 1 was digested with DNase I (5 μg/ml) for 2.5 min at room temperature in the absence of B cell nuclear protein. Reaction 2 was initially incubated with the heparin Sepharose fraction of the EW nuclear factor (14 μg protein) for 15 min at room temperature and then digested with DNase I as above. Each reaction was stopped with EDTA (5 mM) and the products separated by native polyacrylamide gel electrophoresis as detailed above. After autoradiography to visualize the various species, DNA was eluted from the free (reaction 1) and bound (B1 and B2, reaction 2) fragment bands by incubating gel slices in 0.5 M ammonium acetate (ph 7.5), 0.1% SDS and 1 mM EDTA with shaking at 37° C. overnight. The supernatants were extracted sequentially with phenol-chloroformisoamylalcohol (25:24:1 v/v) and chloroform-isoamylalcohol (24:1 v/v) and precipitated with 2 volumes of ethanol in the presence of carrier tRNA. After a reprecipitation step the products were analyzed by separation in a 10% polyacrylamide gel (20:1) in the presence of 8M urea followed by autoradiography at −70° C. with a screen. Lane 1 contains products of free fragment digestion from reaction 1. Lanes 2 and 3 contain digestion products eluted from bound bands B1 and B2, respectively, from reaction 1. Lanes 1′, 2′ and 3′ corresponds to 1, 2 and 3, respectively, with the exception that the former set was digested with DNase .1 for 5 min. A-G chemical cleavage ladders of the K promoter probe were coelectrophoresed to map the binding domain. See Maxam, A. and Gilbert, W. Meth. Enzymol. 65, 499-525 (1980).</p>
    <p>4. Binding of a Common Nuclear Factor to Three Ig Transcriptional Control Elements</p>
    <p>Nucleotide sequences of actual and putative binding sites, FIG. 4<i>a</i>. The V<sub>L </sub>binding site is defined by the DNase I protection assay (* indicates boundaries of the protected region). The V<sub>H </sub>and J<sub>H</sub>-C<sub>U </sub>sequences are putative binding sites in the V<sub>17.2.25 </sub>promoter and the mouse heavy chain enhancer, respectively. Numbers in brackets indicate start coordinated of octamer motif. Binding competitions. Binding assays (10 μl) were performed as detailed above using 1600 ng poly(dI-dC)-poly(dI-dC) and the heparin Sepharose fraction of the EW nuclear factor (1.5 μg protein). V<sub>L </sub>probe (about 0.1 ηg, 5000 cpm) lanes 1-3, 10, 11. V<sub>H </sub>probe (˜0.2 ηg, 5000 cpm) lanes 4-6, 12—12. J<sub>H</sub>-C<sub>μ</sub> probe (˜0.2 ηg, 5000 cpm), lanes 7-9, 14, 15. Lanes 2, 5, 8 additionally contained 5 ng of a V<sub>L </sub>promoter oligomer (36 bp, spanning positions −81 to −44 of the MOPC-41 V<sub>κ</sub> gene segment) whereas lanes 3, 6, 9 contained 50 ng of the same oligomer. Lanes 11, 13, 15 additionally contained 50 ηg of a J<sub>H</sub>-C<sub>H </sub>oligomer (41 bp, spanning positions −1 to 40 of the heavy chain enhancer). Complementary single-stranded synthetic oligonucleotides were kindly made by Dr. Ronald Mertz, Genzentrum der Universitat Munchen and Dr. E. L. Winnacker, Institut fur Biochemie der Universitat Munchen. They were annealed prior to use as competition substrates in the binding assay.</p>
    <heading>Results</heading> <p>A radiolabeled SfaNI-SfaNI DNA fragment derived from the upstream region of the MOPC 41 κ light chain gene (FIG. 1<i>a</i>) was incubated with a nuclear extract of a human B cell line in the absence or-presence of two different competitor DNAs. The resulting complexes were resolved from the free fragment by electrophoresis through a low ionic strength, non-denaturing polyacrylamide gel and visualized by autoradiography (FIG. 1<i>b</i>). In the absence of competitor DNA all of the labeled fragment was retained at the top of the gel (lane 1) probably due to the binding of an excess of non sequence-specific proteins. With addition of increasing amounts of either poly(dI-dC)-poly(dI-dC) (lanes 2-6) or <i>E. coli </i>chromosomal DNA (lanes 7-11) as competitors, putative protein-DNA complexes which migrated more slowly than the free fragment were detected. The relative abundance of the major species of complex (B) as well as that of minor species was significantly greater in the presence of the alternating copolymer competitor DNA. Because substitution of the simple copolymer considerably increased the sensitivity of the assay, it was used in all subsequent binding analyses.</p>
    <p>To test the sequence-specificity of the major species detected in the binding assay, a set of mutually overlapping Kappa promoter fragments (See FIG. 1<i>a</i>, SfaNI-SfaNI, PvuII-KpnI and PvuII-SfaNI) and a similar length fragment derived from the bacterial plasmid SP64 (EcoRI-PvuII) were individually assayed (FIG. 1<i>c</i>). Whereas the bacterial DNA fragment showed no appreciable binding (lane 4) all of the κ promoter fragments yielded major discrete complexes of similar mobilities (lanes 1-3). The mobilities of a series of complexes formed with different length fragment probes (75-300 bp) are approximately the same (data not shown). These data therefore suggested the binding of a specific nuclear factor within the region of overlap of the Kappa promoter fragments. This region includes the conserved octameric sequence but not the TATA element. Note that with the smallest 75 bp Kappa promoter fragment (lane 3) no appreciable label was retained at the top of the gel. Thus, as has been noted recently, the use of small probe fragments further enhances the sensitivity of detection of specific protein-DNA complexes.</p>
    <p>The sensitivity gained by use of both the copolymer and a small fragment probe permitted the detection of two complexes, B1 and B2 (FIG. 2<i>a</i>, lane 1). The major species B1 corresponds to complex B in the earlier figure. The relative affinity of the factor(s) for κ promoter DNA was estimated by a competition assay (FIG. 2<i>a</i>). Whereas a control plasmid (pSP64), when added in the above incubation, failed to compete for binding in the concentration range tested (lanes 2-4), the recombinant plasmid (pSPIgV<sub>κ</sub>) effectively reduced the formation of both species B1 and B2 in the same range (lanes 5-7). The latter plasmid was constructed by insertion of the upstream region of the Kappa promoter into pSP64. Assuming that the pSPIgV<sub>κ</sub> plasmid contains a single high affinity binding site these results suggest that the nuclear factor(s) responsible for B1 and B2 has at least a 10<sup>4</sup>-fold higher affinity for its cognate sequence than for heterologous plasmid DNA (see Methods section 2 above).</p>
    <p>To determine if the factor(s) responsible for formation of B1 and B2 was specific to B lymphocytes, a nuclear extract derived from human HeLa cells was assayed for binding to the κ promoter probe (FIG. 2<i>b</i>). Both species B1 and B2 were generated at similar levels as that observed with B cell extracts, by the HeLa extract (lane 1). Furthermore, both were specifically competed by the pSPIgV<sub>κ</sub> plasmid (lane 2). Thus HeLa cells also contain a factor(s) which binds specifically to the κ promoter upstream region.</p>
    <p>DNase I footprint analysis was used to delineate at a higher resolution the binding domain(s) of factor(s) present in complexes B1 and B2. To facilitate these studies, the binding factor(s) from B cells was partially purified by chromatography of nuclear extract protein on a heparin sepharose column. Most of the binding activity eluted in a 0.25 M KCl step fraction, giving a purification of approximately 5-fold (data not shown; see legend to FIG. <b>3</b>). For footprint analysis, DNase I was added for a partial digestion after incubation of the partially purified factor(s) with the κ promoter probe B1 and B2 species were then resolved from free fragment by polyacrylamide gel electrophoresis. Bound DNA was eluted from both B1 and B2 bands and examined by denaturing polyacrylamide gel electrophoresis (FIG. 3, lanes 2, 3 and 2′, 3′). DNase I digests of the κ promoter probe in the absence of B cell protein (lanes 1 and 1′) and A+C chemical cleavage ladders were coelectrophoresed to map the binding domain. Factor(s) in the B1 complexes (lanes 2 and 2′) appeared to protect a 19 nucleotide region on the coding strand. The 5′ and 3′ boundaries of the protected region map to positions −72 and −52, respectively, from the site of transcriptional initiation. The region of DNase I protection was centered about the conserved octanucleotide sequence ATTTGCAT suggesting its importance in the recognition of the Ig promoter by the nuclear factor. B2 complexes showed a virtually identical DNase I protection pattern as B1 complexes and therefore do not appear to involve additional DNA contacts (lanes 3 and 3′). The simplest interpretation of this observation is that the B2 complex is generated by dimerization through protein-protein interactions of the factor responsible for the B1 complex. Alternatively, the B2 complex could be formed either by the binding of another protein to the factor responsible for the B1 complex or by recognition of the same set of sequences by a distinct DNA binding protein.</p>
    <p>Because the octamer sequence motif is present in both light and heavy chain gene promoters as well as in the enhancer elements of both mouse and human heavy chain genes, assays were performed for factor binding to fragments from a mouse heavy chain promoter (V<sub>H</sub>) and the mouse heavy chain enhancer. The V<sub>H </sub>promoter fragment was derived from the 5′ region of the V<sub>17.2.25 </sub>gene and included nucleotides between positions −154 and +57 relative to the transcriptional start site. Grosscheal, R. and Baltimore, D. Cell 41 885-897 (1985). In this promoter the conserved octanucleotide spans positions −57 to −50 (FIG. 4<i>a</i>). The heavy chain enhancer fragment was derived from the germline J<sub>H </sub>C<sub>μ</sub> region and spanned positions 81 to 251 within a 313 bp region implicated in enhancer function. Banerji, T. et al. Cell 33 729-740 (1983). The conserved octanucleotide is positioned between coordinates 166 to 173 in the above fragment (FIG. 4<i>a</i>). The B-cell heparin sepharose fraction (purified on the basis of binding to the Kappa promoter sequence, FIG. 4<i>b</i>, lane 1) evidenced binding to both the V<sub>H </sub>promoter fragment (lane 4) and to the enhancer fragment (lane 7). The mobilities of the complexes formed with the three fragments were very similar consistent with the binding of a common factor. Furthermore, binding to these fragments was specifically competed by a synthetic duplex 40-mer that spanned the octanucleotide motif of the MOPC 41 κ light chain gene promoter (lanes 1-9). An oligomer of equivalent size containing a sequence from a region of the mouse heavy chain enhancer lacking the octanucleotide motif (FIG. 4<i>b</i>) failed to compete for binding in the same concentration range (lanes 10-15). This competition analysis further strengthens the suggestion that a common nuclear factor (IgNF-A) binds to all three Ig transcriptional elements. As has been mentioned previously, these three transcriptional elements share an identical sequence motif ATTTGCAT (FIG. 4<i>a</i>). Thus, the binding of a common nuclear factor is almost certainly mediated by this motif.</p>
    <heading>Example 2</heading> <heading>Dependency of in Vitro Transcription of Ig Genes on an Upstream Sequence</heading> <heading>Methods</heading> <p>FIG. 5<i>a</i>: Templates. The deletions 5′Δ5 and 5′Δ7 have been described before. See Bergman Y. et al PNAS USA 81:7041 (1981). The highly conserved octanucleotide sequence which is found upstream of all sequenced heavy and light chain variable region genes is boxed (labelled “OCTA”). It is located approximately 30 base pairs upstream from the “TATA” box. The plasmids pκ and pΔκ were constructed by converting the 5′-ends of 5′Δ5 and 5′Δ7 into a Hind III site by means of synthetic linkers followed by cloning the fragment up to the Bgl II site in the J<sub>κ</sub>-C<sub>κ</sub> major intron into Hind III, Bam HI digested pUC-13. pκE<sub>κ</sub> and pκE<sub>κ</sub> represent plasmids containing either the kappa enhancer of the heavy chain enhaner cloned into the unique Hind III site of pK. The segments used as the enhancers are an 800 bp Hind III-Mbo II fragment from the J<sub>κ</sub>-C<sub>κ</sub> intron (Max, E. E. et al. (1981) J. Biol. Chem. 2565116) and a 700 bp Xba I-Eco RI fragment from the J<sub>H</sub>-C<sub>μ</sub> intron. Gillies, S. D. et al. Cell 33:717 (1983); Banjerji, J. et al. Cell 33:729 (1983). FIG. 5<i>b</i>; transcription in whole cell extracts made from the human B lymphoma cell lines RAMOS (lanes 1,2) and EW (lanes 3,4); transcription in a HeLa whole cell extract (from A Fire (lanes 5,6)). The expected 2.3 kb run off transcript is indicated.</p>
    <p>The cell lines RAMOS and EW were grown in RPMI medium containing 10% inactivated fetal calf serum to a density of 5-8×10<sup>5 </sup>cells per ml. Whole cell extracts were generated according to the procedure of Manley et al., PNAS USA 77:3855 (1980), and had a final protein concentration of approximately 18 mg/ml. Run off transcription reactions were carried out at 30° for 60′ in a reaction volume of 20 μl. A typical reaction mix contained 9 ul (160 μg) of whole cell extract, 50 uM each of ATP, CTP and GTP, 0.5 uM UTP, 10 μCi of α-<sup>32</sup>P UTP (NEG 007×, 7600 Ci/mM) 5 mM creatine phosphate, 0.3 mg/ml creatine phosphokinase (Sigma), 12 mM Hepes 7.9, 12% glycerol, 60 mM KC1, 5 mM MG<sup>++</sup>, 1 mM EDTA, 0.6 mM DTT, linearized template (about 50 ηg) and poly (dI-dC)-poly(dI-dc) as a non-specific carrier (about 400 ηg). The reactions were terminated by adding 200 μl of stop buffer (7M urea, 100 mM LiCl, 0.5% SDS, 10 mM EDTA, 250 μg/ml tRNA, 10 mM Tris (pH 7.9), followed by two extractions with phenol: chloroform: isoamyl alcohol (1:1:0.05), one with chloroform and precipitation with ethanol. The RNA's were treated with glyoxal and analyzed by electrophoresis through a 1.4% agarose gel in 10 mM sodium phosphate (pH 6.8), 1 mM EDTA. See Manley et al. supra. The gel was then dried for autoradiography with an intensifying screen at −70° C.</p>
    <p>FIG. <b>6</b>: Effect of the upstream deletion 5′Δ7 on in vitro transcription in B cell extracts utilizing a pre-incubation pulse chase protocol. Run off transcripts obtained utilizing templates containing either the wild type promoter (lane 1) or the truncated Kappa promoter (lanes 2,3). Lanes 4-6: In vitro transcription using closed circular templates containing the wild type promoter (lane 4) or the truncated κ promoter (lanes 5-6). In these reactions 50 ηg of a closed circular template containing the adenovirus major late promoter (MLP) was included as an internal control. The transcripts specific to the κ template or the adenovirus template are indicated as κ and MLP, respectively. For a template containing the major late promoter the plasmid pFLBH was used. The plasmid contains sequences from 14.7 to 17.0 map units of adenovirus inserted between the Bam HI and Hind III sites of pBR322 and was the kind gift of A. Fire and M. Samuels.</p>
    <p>Either the linearized or the supercoiled template (50 ηg) was incubated in a volume of 20 μl with 9 μl (about 150 μg) of EW extract, 6% (wt/vol) polyethylene glycerol 20,000 and all other components described for FIG. 5<i>b </i>except the nucleotides for a period of 60 minutes at 30° C. Transcription was initiated by the addition of nucleotides and radioactive UTP to the following final concentrations: 60 uM each of ATP, CTP and GTP and 1 μM UTP and 10 μCi α-<sup>32</sup>P UTP (NEG 007×, 600 Ci/mM). The initiating pulse was maintained for 10′ at 30° followed by a 10′ chase with a vast excess of non-radioactive nucleotides. Final concentrations during the chase were as follows: 330 μM ATP, CTP, GTP and 1 mM UTP. The reactions were quenched, worked up and the run off transcripts analyzed as described above. Mapping of the initiation site of the transcript was conducted as follows: Transcripts generated from closed circular templates were taken up in 20 μl of HE (50 mM Hepes, pH 0.7, 1 mM EDTA) and 10 μl was used for hybridization selection. A hybridization template complementary to the κ RNA was constructed by cloning the Pva II-Sau 3A fragment which contains the cap site of the MOPC41 gene (Queen and Baltimore, Cell 33:741 (1983)) into the M13 phage MP9. Single stranded phage DNA was prepared and purified by density gradient centrifugation through cesium chloride. MLP specific transcripts were detected using the M13 clone XH11 provided by A. Fire and M. Samuels. Hybridizations were done in a final volume of 15 μl in the presence of 750 mM NaCl and 100-200 ug of single stranded complementary DNA at 50° C. for 2 hrs. The reactions were then diluted with 200 μl of cold quench solution (0.2 M NaCl, 10 mM Hepes pH 7.5, 1 mM EDTA) and 2 U of ribonuclease T1 added. Digestion of single stranded RNA was allowed to proceed for 30′ at 30° C. after which the reactions were extracted once with phenol chloroform isoamyl alcohol (1:1:0.05) and precipitated with carrier tRNA. The pellet was washed once with cold 70% ETOH, dried and resuspended in 80% v/v formamide, 50 mM Tris borate, pH 8.3 and 1 mM EDTA. The RNA was denatured at 95° C. for 3 min and then electrophoresed through a 6% polyacrylamide 8.3 M urea sequencing gel. The upper of 2 bands (κ) derived form the immunoglobulin promoter represent the correct start for κ transcription. The lower band is seen at variable intensities and probably does not represent a different cap site, as explained below.</p>
    <heading>Results</heading> <p>Whole cell extracts were made from two human Burkitt lymphoma lines, EW and RAMOS, by the procedure of Manley et al. supra. The templates used for in vitro transcription reactions are diagrammed in FIG. 5<i>a</i>. The template representing the wild type gene (pK) was derived from the MOPC41 κ gene and contained sequences from approximately 100 bp upstream from the transcription initiation site (end point 5′Δ5, FIG. 5<i>a</i>) to the Bgl II site in the major J<sub>κ</sub>-C<sub>κ</sub> intron Max, E. E. J. Biol. Chem. 256:5116. This fragment retains the complete variable region which is rearranged to J<sub>κ</sub> l, but not the κ enhancer which is further downstream of the Bgl II site. See, e.g., Queen and Stafford, Mol. Cell. Biol. 4:1042 (1984). This short 5′ flank has been shown to be sufficient for accurate initiation and high level of transcription in a transient transfection assay. Bergman, Y. et al. PNAS USA 81:7041 (1984). Deletion analysis of the κ promoter showed previously that important regulatory sequences are present between nucleotides −79 and −44 because deletion 5′Δ7 completely abolished transcriptional competence of the gene while deletion 5′Δ5 had no effect. See Bergman et al. supra. The template representing an inactive promoter mutant (pΔκ) was constructed by engineering a Hind III site into the 5′ -end of 5′Δ7 and cloning the segment of the gene up to the Bgl II site into pUC13 cut with Hind III and Bam HI.</p>
    <p>To examine transcriptional activity in B cell extracts, a linear template truncated at the Sac I site in the polylinker was used and transcripts ending at this site (run off transcripts) were examined by electrophoretic separation. A run off transcript of 2.3 kb was evident when RAMOS, EW or HeLa cell extracts were used (FIG. 5<i>b</i>, lanes 2, 4 and 6). When a κ chain enhancer sequence was added to the construct, no effect was evident implying that transcription in these extracts is enhancer independent (FIG. 5<i>b </i>lanes 1, 3 and 5). (In EW and HeLa, the enhancer appears to cause a slight increase in the background radioactivity but not in the 2.3 kb band.) The band at 2.3 kb could be abolished by not adding the template or by transcribing in the presence of 0.5 μg/ml amanitin. Thus it represents a template-specific, RNA polymerase II transcript. The band just below 2.3 kb is not decreased by α-amanitin and presumably reflects end-labeling of endogenous 18S rRNA.</p>
    <p>To assess whether initiation of transcription occurred at the natural cap site, a second assay was used. See Hansen, U. and P. A. Sharp (1983) <i>EMBOJ </i>2:2293. For this assay, the uniformly labeled RNA was hybridized to a single stranded DNA probe spanning the transcription initiation site (generated by cloning the Pvu II-Sau 3A fragment of the κ gene into phage M13). The resulting complex was digested with ribonuclease T1 and the ribonuclease-resistant RNA fragments were analyzed by electrophoresis through a 6% polyacrylamide gel with 8.3 M urea. Analysis of in vitro synthesized RNA by this method is shown in FIG. 6 (lane4). The upper band (labeled κ) represents the correct cap site. The band just below it was seen at variable intensities and probably does not represent a different cap site but rather arises from cleavage with ribonuclease T1 at the next G residue from the 3′-end of the protected region. (Examination of the sequence near the Sau 3A1 site shows that the second set of G residues on the RNA lies 19 bp upstream from the end of the region of homology with the single stranded DNA probe). Thus the extracts generated from B cells were capable of correctly initiating and transcribing the immunoglobulin promoter in vitro with approximately the same efficiency as a HeLa cell extract.</p>
    <p>To analyze the effect of 5′ flanking sequences in vitro, we examined the transcription of the deleted gene, pΔK. Because many regulatory effects act upon the rate of initiation of transcription, we chose to use a preincubation, pulse-chase protocol which measures the initiation rates. See Fire, A. et al. J. Biol. Chem. 259:2509 (1984). The template DNA was first preincubated with the extract to form a pre-initiation complex. Transcription was then initiated by the addition of nucleotides and radiolabeled UTP. The initiated transcripts were completed during a chase period with unlabelled nucleotides and analyzed by electrophoretic separation. Incorporated radioactivity in this assay is proportional to the number of correct initiations occurring during the pulse.</p>
    <p>In FIG. 6, comparison of lanes 2 and 3 with lane 1 shows that the template pκ, which contains about 100 bp upstream of the initiation site, initiated approximately 10-fold more efficiently than did the deletion mutant, pκ. Again, the presence of the heavy chain enhancer placed at −44 bp to the truncated promoter did not alter the level of transcription. When closed circular templates were used, a similar effect of the promoter truncation was observed (FIG. 6, lanes 4-6). In these reactions a template containing the major late promoter of adenovirus was included as an internal control; the expected protected RNA fragment of 180 bp is labeled MLP. Comparison of lanes 5 and 6 with lane 4 shows that there was a 10-fold decrease in the efficiency of transcription from the mutant promoter, whereas the transcript of the major late promoter remained constant. The reason for the apparent decrease in the amount of transcription from the supercoiled template containing the heavy chain enhancer has not been further addressed. It is evident, however, that the dependence of transcription on an upstream sequence between −44 and −79 is observed whether the effect described above was specific to B cell extracts, the same templates were transcribed in the heterologous HeLa whole cell extract. A 4- to 5-fold decrease in transcription was seen with the deleted template when compared with the wild type template (data not shown). Thus, the effect of the deletion is at best, only modestly tissue-specific.</p>
    <p>We have reported here the development of transcriptionally competent whole cell and nuclear extracts from two independent human B cell lymphomas. In such extracts, transcription from the promoter of the MOPC41 κ gene was correctly initiated and a promoter deletion significantly reduced the level of initiated RNA. In vitro, the effect of the deletion used here is several hundredfold when analyzed by a transient transfection assay. However, the effect observed in vitro is only about 10- to 15-fold. Although there are now several examples of upstream sequence requirements for in vitro transcriptions, See, e.g., Groschedl R. and Birsteil M. L. (1982) PNAS USA 79:297; Hen, R. et al. (1982) PNAS USA 79:7132, the effects have been smaller than the corresponding one in vivo. This is possibly due to the dominance of the TATA box and associated factors in determining the level of transcription in vitro Miyamoto, N. G. et al., Nucl. Acids Res. 12:8779 (1984).</p>
    <heading>Example 3</heading> <heading>Discovery and Characterization of IgNF-B</heading> <p>The mobility shift gel electrophoresis assay was used to screen nuclear extracts from a variety of cell lines for octamer binding proteins. The band corresponding to IgNF-A was found in all extracts but a second band with distinct mobility from IgNF-A was found only in nuclear extracts from lymphoid cells. This lymphoid-specific octamer binding protein, termed IgNF-B, was found in nuclear extracts from all pre-B, mature B and myeloma cell lines tested and in nuclear extracts from some T cell lymphomas (see FIG. <b>7</b> and <b>8</b>). IgNFB was not detected in nuclear extracts from the non-lymphoid cell lines, Hela, ψ2, Cos and Mel (see FIG. <b>8</b>). IgNF-B was shown to be specific for the same octamer sequence as IgNF-A by competition experiments n which the IgNFB band was selectively competed by unlabelled DNA fragments sharing only the octamer sequence and not by DNA fragments lacking the octamer sequence.</p>
    <p>The octamer sequence is found at approximately position −70 upstream of the transcription start site of all immunoglobulin (Ig) variable genes which is in the region that has been shown to control the lymphoid specificity of the Ig promoter. Thus, IgNF-B binds to the upstream octamer sequence in lymphoid cells and activate transcription.</p>
    <heading>Example 4</heading> <heading>Factors Binding to μ-Enhancer: E Factor</heading> <p>The fully functional μ enhancer has been localized to a 700 bp XbaI EcoRI fragment from the major intron between J<sub>H </sub>and C<sub>μ</sub>. This fragment can be further sub-divided by cleaving at the PstI site to generate a 400 bp Xbal-PstI fragment (μ400) and a 300 bp PstI-EcoRI fragment (μ300). It has been shown by transient transfections that 30-50% of the tissue specific enhancer activity is retained in μ300, whereas there is no detectable activity of μ400. The gel binding assay was employed to investigate what protein factors may interact with the u-enhancer. Briefly, end-labelled DNA fragments were incubated with nuclear extracts made from tissue culture cells. After 20 min at room temperature the mixture was loaded on a low ionic strength polyacrylamide gel and electrophoresis carried out at 120V for 2 hrs. The gels were then dried for autoradiography. When the functional 300 bp (μ300) enhancer fragment was used in such an assay a DNA-protein complex was observed in extracts derived from the human B lymphoma cell line EW (FIG. 9<i>b</i>, lane 2). To show that this new band represented a specific complex binding reactions were carried out in the presence of varying amounts of non-radioactive competitor fragments (FIG. 9<i>b</i>, lanes 3-11). It is easily seen that when μ300 is added as the competitor fragment, the complex band is completely lost. In contrast, the adjacent u400 fragment (lanes 6,7,8) or a 450 bp fragment containing the κ light chain enhancer (lanes 9,10,11), cause only a minor effect even at the highest concentrations used. It is interesting to note that there appears to be a slight increase in the amount of specific complex in the presence of the κ enhancer fragment (compare lanes 9 and 2). As demonstrated below, both the μ and the κ enhancers interact with at least one common protein and this is not the factor being detected by binding the u300 fragment. The increase in the specific complex in the presence of the κ enhancer is probably due to the removal of factors common to both the enhancers from the reaction mix, thus leaving more of the labelled fragment available to bind to the μ specific factor being detected by it.</p>
    <p>In order to be able to detect binding sites for less abundant proteins and also to more precisely define the complex detected with μ300, this fragment was further dissected. Each of the smaller fragments generated were analyzed for their ability to serve as binding sites for nuclear proteins. FIG. 10A shows a partial restriction map of the relevant region of the μ enhancer. μ300 was digested with AluI, HinfI and DdeI to generate a number of 50-70 bp fragments labelled μ50, μ(60)<sub>2 </sub>(a mixture of AluI-DdeI and HinfI to AluI) and μ70 (AluI-AluI). Binding reactions were carried out with each of these fragments with nuclear extracts of EW lymphoma cells in the presence of increasing amounts of the non-specific competitor poly (dI-dC)-poly(dI-dC). The results are shown in FIG. <b>10</b>B.</p>
    <p>Fragment μ50 forms a major complex band (lanes 2,3,4) that is barely decreased even in the presence of 4 ug of poly(dI-dC)-poly(dI-dC) (lane 4). The mixture of the two 60 bp fragments does not give rise to a discrete complex band (lanes 6,7,8). Finally the μ70 fragment gave 2 faint, but discrete, nucleoprotein complex bands (lanes 10,11,12) of which the lower one is again barely affected by 3 ugm of non-specific carrier poly(dI-C)-(dI-C) (lane 12).</p>
    <p>Specificity of the complexes observed were shown by competition experiments using a variety of DNA fragments, FIG. <b>10</b>C. Thus, the complex generated with μ50 is specifically competed away in the presence of μ300 (of which μ50 is a part), or a κ promoter fragment, but not by corresponding amounts of μ400 or a κ enhancer fragment, consistent with the complex being generated by the interaction of the previously described factor IgNF-A with its cognate sequence. (This factor recognizes a conserved octanucleotide, ATTTGCAT, found in the promoters of all sequenced immunoglobulin genes and within this subfragment of the heavy chain enhancer.)</p>
    <p>The complex observed with μ70 was specifically competed away by itself (lane 9) and to some extent with the κ enhancer (lanes 5,6 FIG. 10C) but not at all by either the Moloney murine leukemia virus enhancer (lanes 3,4) or by μ400 (lanes 7,8, FIG. <b>10</b>D). Further competition analysis showed that this complex could not be competed away by either (μ60)<sub>2 </sub>(FIG. 2D, lanes 7,8), μ50 (FIG. 2D, lanes 5,6) or μ170 (central AluI-AluI fragment) (FIG. 10D, lanes 11,12). The binding we have observed is therefore specific to this small fragment and was detected only upon further dissecting μ300 which separated the major observable interaction of IgNF-A with the enhancer sequence to another fragment (μ50).</p>
    <p>Ephrussi et al. and Church et al. have used methylation protection experiments to define a set of G residues within the heavy chain enhancer that are specifically resistant to methylation by DMS in B cells. This result lead to the proposal that tissue-specific DNA binding proteins were responsible for this decreased accessibility of the reagent to DNA. The protection was observed in 4 clusters, the DNA sequences of which were sufficiently homologous to derive a consensus sequence for the binding site of a putative factor. All four postulated binding sites (E1-E4) are found within the 700 bp fragment; however μ300 retains only 2 complete binding domains (E3 and E4) for this factor and the octamer (0) sequence. The Alu-Alu fragment that shows that specific nucleoprotein complex described above contains the complete E3 domain and the factor we detect in vitro presumably is the same as that detected in vivo. Thus, it was unexpected that the HinfI-Dde fragment (μ50) containing E4 and 0 should not compete for binding to μ70 (FIG. 10D, lanes 5,6).</p>
    <p>In case this was due to the fragment predominantly binding IgNF-A at the octamer site and thus making it unavailable as a competitor for μ70, binding reactions and competition assay were done for a fraction generated by chromatography of the crude extract over a heparin-sepharose column, that contained μ70 binding activity and was significantly depleted of IgNF-A. When μ50 or μ170 was endlabeled and incubated with the column fraction, no specific nucleoprotein complexes were seen upon electrophoretic analysis. Even in this fraction, μ50 and μ170 failed to compete successfully for the interaction between μ70 and its binding protein (data not shown), thus implying strongly that the binding site defined as E4 perhaps does not bind the same factor that binds at E3. Similarly, the El domain (isolated as a Hinf-PstI fragment) does not compete as effectively as μ70 itself for the binding of the factor to μ70.</p>
    <p>To determine the location of the binding sites within individual fragments (μ70 and μ50), the technique of methylation interference was employed. End-labelled DNA fragments were partially methylated on guanines and adenines using dimethyl sulfate (DMS). Methylated DNA was then used for binding reactions with crude extracts and the complex was resolved from the free fragment by electrophoresis. Both complex and free fragment bands were then excised from the gel, and the DNA was recovered by electroelution. Piperidine cleavage of the recovered fragments was followed by electrophoresis through 12% polyacylamide urea sequencing gels. In principle, if any of methyl groups introduced by reaction with DMS interfere with the binding of a specific protein then that molecule of DNA will be selectively missing in the complex formed and subsequently the corresponding ladder. The method therefore allows identification of G residues making intimate contacts with the protein. [We have found that the use of DNaseI footprinting via the gel binding assay to be complicated in the case of some of these less abundant factors because of the short half lives of the complexes themselves. Thus, if a binding incubation is followed by partial DNaseI digestion, it is possible that in the course of time required to load the sample and have the complex enter the gel, DNA fragments that were in complex form may exchange with the larger amounts of free fragment in the binding reaction. Thus not leading to any distinction in the DNase patterns seen with wither the complex or the free DNA (e.g. the half life of the nucleoprotein complex in μ70 is less than 1 minute,)].</p>
    <p>The result of carrying out such an interference experiment using nuclear extracts and on the u50 DNA fragment shows that the complex observed arises via interaction of the IgNF-A protein at the conserved octameric sequence (FIG. <b>3</b>A). The free fragment generates a characteristic G ladder (FIG. <b>11</b>A), lane 2,3) and the complex form (lane 1) is specifically depleted in DNA molecules carrying a methyl group at the G residue indicated by the asterisk which lies in the middle of the conserved octamer. Presumably, modification at this residue seriously impedes the formation of a stable complex between the protein and its cognate sequence. This residue was also shown to be protected against methylation of DMS in vivo. Interestingly, however, none of the other G residues observed to be protected in vivo in this region of the μ enhancers appear to be affected in our in vitro interference experiment. Therefore, if these protections in vivo are due to the binding of a protein, this factor is different from IgNF-A or B and is not binding to fragment in vitro.</p>
    <p>On the μ70 fragment several G residues were identified as being important in forming intimate contacts with the binding protein (E) (FIG. <b>11</b>B). On the coding strand bands the 3 G's are significantly reduced in intensity in the complex as compared to the free DNA fragment (FIG. 11B, compare lanes 1,2), and on the non-coding strand 2, G's are significantly affected (FIG. 11B, compare lanes 3 and 4).</p>
    <p>The results of both the in vivo DMS protection experiment and the in vitro methylation interference experiments are summarized in FIG. <b>3</b>C. The open and closed circles above the sequence were the residues identified by Ephrussi et al. to be protected against methylation in vivo whereas the encircled G's are the ones identified by us in vitro. The pattern of protection and interference on the μ70 fragment over the consensus sequence is strikingly similar in vivo and in vitro, which indicated strongly that the protein identified here by means of the gel bind assay is the one that interacts with this sequence in vivo. Analogous to μ50, however, the second set of protections seen in this region in vivo was not observed in vitro. Tissue specificity of the factors detected: In order to determined whether the proteins identified are limited to expression only in B cells, a large number of extracts made from B cells and non-B cells were screened (FIG. <b>12</b>). Complexes that co-migrate with the ones generated and characterized (by competition and methylation interference experiments) in the B cell line EW, were observed on both the fragments μ50 and μ70 (FIG. 12; μ70) in all the cell lines examined. Although the complex generated in each extract has not been further characterized, we interpret this data as indicating that both these factors are non-tissue specific. A second complex (NF-κB) was observed with the μ50 fragment that was restricted to B and T cells only.</p>
    <p>A point to note is that although the amount of protein in each lane has been held constant at between 9 and 11 μg, the extent of complex generated was found to vary considerably from extract to extract. Thus, showing that quantitive estimations of the abundance of proteins in different cell lines using this assay is not very meaningful at this stage. (This is presumably due to subtle variations in the state of the cells and the extraction conditions for the different cell lines).</p>
    <p>In summary, analysis of the functional 300 bp PstI-EcoRI fragment of the μ enhancer reveals that:</p>
    <p>(i) at least 2 different proteins bind within this DNA sequence. One protein (IgNF-A) interacts with an octamer sequence (ATTTGCAT) that is highly conserved upstream of all heavy and light chain variable region genes and is also found in the u enhancer. The second protein interacts with a sequence shown by Ephrussi and Church to be protected in a tissue specific manner against methylation by DMS in whole cells;</p>
    <p>(ii) both factors can be detected in nuclear extracts from a variety of cell lines and are therefore not B-cell specific;</p>
    <p>(iii) both E1 and E4 sequences hardly compete for the binding of the factor to μ70 (which corresponds to E3), thus implying that these sequences do not interact with the same factor, although the sequence homology amongst the sites would have lead one to expect that they should.</p>
    <heading>Example 5</heading> <heading>Identification of Factors Binding to Kappa-light Chain Enhancer</heading> <p>An enhancer element has also been identified in the major intron of the κ light chain gene. Picard and Schaffner showed that the enhancement activity can be localized to a ˜500 bp AluI-AluI fragment and Queen and Stafford have further refined the 5′ and 3′ boundaries so that the enhancer may be considered restricted to 275 base pairs within the larger fragment. We have dissected this region into a number of smaller fragments and assayed each of these by means of the gel binding assay for the location of protein binding sites.</p>
    <p>A restriction map of the relevant region of the κ enhancer is shown in FIG. 13<i>a</i>. The black boxes represent sequences identified by Church et al. to be homologous to the putative protein binding domains detected in the μ enhancer in vivo.</p>
    <p>Fragments generated by cutting with Dde and HaeIII (κ1, κ2, κ3, κ4 and κ5; FIG. 13<i>a</i>) were assayed for binding in the presence of increasing amounts of poly(dI-dC)-(dI-dc) as a non-specific carrier, κ4 and κ5 appeared to be obviously negative (FIG. 13<i>b</i>, lanes 1-8) while κ3 and κ2 appeared to be positive (FIG. 13<i>b</i>, lanes 10-12 and 14-16). Preliminary results show that the internal fragment does not contain any specific binding sites either. The nucleoprotein complex bands generated with 0.5-1 ηg of radiolabelled probe could be detected even in the presence of 3 μg of the carrier (lanes 12, 16, FIG. 13<i>b</i>).</p>
    <p>To show that the bands detected represented a specific interaction between a protein and DNA, we carried out competition experiments (FIG. 13<i>c </i>and <b>13</b> <i>d</i>). The competition pattern for κ2 was strikingly similar to what had been earlier observed with the μ70 fragment; relatively large amounts of u400, the Moloney leukemia virus enhancer, the SV40 enhancer or the κ promoter (containing the conserved octa) to κ2 did not compete for binding, although u300 and the κ enhancer did. Since κ2 contains a putative E box identified by sequence comparison (as does μ70) we competed its binding with smaller fragments from μ300 (FIG. <b>5</b>C). The complex is specifically competed away by the addition of unlabelled μ70 during the incubation (compare lanes 3 and 4 with lane 2), but not by μ60 (lanes 5,6), μ170 (lanes 7,8) or the SV40 enhancer (lanes 9,10). Further, the protein that binds to this sequence co-fractionates with the μ70 binding activity through two sequential chromatographic steps (Heparin agarose and DEAE Sepharose). Thus, we conclude that the same sequence specific protein binds to both the fragments μ70 and κ2 and that there is at least one common protein interacting with both the μ and the κ enhancers.</p>
    <p>The κ3 complex (indicated by the arrowhead, FIG. 13<i>d</i>) failed to be competed away by μ300 (compare lanes 3 and 4 with lane 2), μ400 (compare lanes 5 and 6 with lane 2) as a κ promoter containing fragment (compare lanes 7 and 8 with lane 2). However, the complex was specifically competed away with both the complete κ enhancer (lanes 9,10) and the SV40 enhancer (lanes 11,12). The band below the major κ3 complex was seen at variable intensities in different experiments and failed to compete even with the complete κ enhancer in this experiment and has not been further investigated at this stage. The observation that the SV40 enhancer specifically competes for binding of this factor is not altogether surprising, since this fragment and the SV40 enhancer share an identical stretch of 11 nucleotides.</p>
    <p>The binding site of this factor on the κ3 fragment was localized by methylation interference experiments. In two different extracts, methylation at three of a stretch of 4 residues within this sequence completely abolished binding (FIG. 14, compare lane 1 [complex] and 2 [free]; and lanes 3 [complex] and 4 [free]). This stretch of G's forms a part of the conserved region (GGGGACTTTCC) between the SV40 enhancer and κ3. Thus, the binding site was localized towards one end of the κ3 fragment. The results also served to explain the specific competition observed earlier with the SV40 Enhancer. Interestingly, deletion mapping of the κ enhancer shows that sequences within the κ3 fragment are extremely important for enhancer function.</p>
    <p>The tissue range of this factor was examined by carrying out binding analysis with κ3 in extracts from a variety of cell lines. Nucleoprotein complex formation κ3 was detected in a mouse B cell line (FIG. 15<i>a</i>, lane 2), but not in 5 other non-B cell lines (FIG. 15<i>a</i>, odd numbered lanes from 5-11). Even numbered lanes show that the ubiquitous factor detected by μ50 is present in all these cell lines and served as a positive control for the experiment. The factor κ3 therefore appears to be restricted to expression to B lymphoid cells.</p>
    <p>We then examined extracts made from cells at various stages of B cell differentiation (FIG. <b>15</b>B). Interestingly κ3 binding protein can be detected in the Abelson murine leukemia virus transformed pre-B cell line PD, in two mouse B cell lines (WEH1231 and AJ9, FIG. 15B, lanes 12,14), one human B cell line (EW, FIG. 15B, lane 16) mouse myeloma line (MPC22, FIG. 15B, lane 18) and 2 human myelomas (KR12 and 8226, FIG. 15B, lanes 20,22). However, it does not appear to be present in a pre-preB cell line (C5, FIG. 15B, lane 4) and in mouse pre-B cell lines (HATFL, 38B9, 70Z, FIG. 15B, lanes 6,8,10). Thus, this factor appears to be not only tissue-specific, i.e., limited to cells of the B lymphoid lineage, but also stage-specific within that lineage. In the series of extracts examined, the presence of this factor bears a striking correlation with κ expression.</p>
    <p>The results with the Kappa enhancer can be summarized follows: Dissection of the κ enhancer enabled detection of two distinct binding proteins with this DNA. One of these proteins appears to be ubiquitous and interacts with the u heavy chain enhancer as well. The second protein appears to be highly expressed in a stage-specific manner within the B cell lineage and can be detected only in those cell lines where the endogenous κ gene is active. There does not appear to be a binding site for this factor in the heavy chain enhancer, although there is one in the SV40 enhancer. Examples 6 and 7 describe cloning of two transcriptional factors: NF-κB and IgNFB.</p>
    <heading>Example 6</heading> <heading>Cloning of Putative NF-κB</heading> <heading>Experimental Procedures</heading> <p>λgt11-EBNA-1 Recombinant</p>
    <p>A HinfiI-AhaII DNA fragment of the EBV genome (coordinates 107,946-109,843), that contains the EBNA-1 open reading frame, was subcloned using BamHI linkers into the BamHI site of pUC13 (pUCEBNA-<b>1</b>). The λgt11-EBNA-1 recombinant was constructed by inserting the 600 bp SamI-BamHI fragment of pUCEBNA-l (EBV coordinates 109,298-109,893) into the EcoRI site of λgt11 using an EcoRI linker (GGAATTCC). A phage recombinant containing the EBNA-1 insert in the sense orientation was isolated by immunoscreening with EBNA-1 antibodies (see below). In this recombinant, the carboxy-terminal region of EBNA-1 (191 amino acids) is fused in frame to the carboxy-terminus of β-galactosidase.</p>
    <p>λgt11 cDNA Expression Library</p>
    <p>The human B cells (RPMI 4265) cDNA library constructed in the expression vector λgt11 was purchased from Clontech Laboratories, Inc. The library contains approximately 9×10<sup>5 </sup>independent clones and has an average insert size of 1.2 kb.</p>
    <p> <i>E. Coli </i>Strains</p>
    <p>The standard pair of λgt11 host strains, Y1090 and Y1098, were employed. The former was used to screen λgt11 recombinants and the latter to generate λlysogens for the analysis of β-gal fusion proteins.</p>
    <p>Plasmids</p>
    <p>The plasmid pUCoriP1 was constructed by subcloning the EcoRI-Ncol fragment from the oriP region of the EBV genome into the SmaI site of pUC13. This fragment contains 20 high affinity binding sites for EBNA-1. pUCoriP2 was derived from pUCoriP1 by subcloning of an oriP fragment (EcoRI-BstXI) of the latter into the SmaI site of pUC13. pUCoriP2 contains 11 high affinity binding sites for EBNA-1. pUCORIλ2 was made by insertion of a synthetic binding site for the bacteriophase λO protein (AAATCCCCTAAAACGAGGGATAAA) into the SmaI site of pUC13. The complementary oligonucleotides were a gift of R. MacMacken. pUCMHCI and pUCmhcI were constructed by insertion of the following oligonucleotides:</p>
    <p>GATCCGGCTGGGGATTCCCCATCT GATCCGGCTGcGGATTCCCaATCT</p>
    <p>GCCGACCCCTAAGGGGTAGACTAG GCCGACgCCTAAGGGtTAGACTAG</p>
    <p>into the BamHI site of pUC13. The wild type sequence is a binding site for H2TF1 and NF-κB. pUCOCTA is a similarly constructed pUC18 derivative that contains a synthetic recognition site (ATGCAAAT) for the mammalian octamer binding protein(s). The plasmids p190H2KCAT (−190 to +5) and p138H2KCAT (−138 to +5) contain 5′-deletions of the H-2K<sup>b </sup>gene promoter fused to the coding sequence for chloramphenicol acetyl transferase. All plasmid DNAs were purified by an alkaline lysis protocol followed by two bandings in CsCl-EtBr gradients.</p>
    <p>Binding Site Probes Competitor DNAs</p>
    <p>The MHC, mhc1, ori and OCTA probes were generated by digesting the corresponding pUC plasmids with EcoRI and HindIII. The resulting products were end-labeled with [α-<sup>32</sup>P]dATP using the large fragment of <i>E. coli </i>DNA polymerase I. dCTP, dGTP and dTTP were included in these reactions so as to fill in the ends of the restriction fragments. The labeled fragments were separated by native polyacrylamide gel electrophoresis. The binding site fragments (60-75 bp) were eluted from the gel and purified by ELUTIP™ (Schleicher and Schuell) chromatography. Using high specific activity [α-<sup>32</sup>P]dATP (5000 Ci/mmol), typical labelings yielded DNA probes with specific activities of 2-4×10<sup>7 </sup>cpm/pmol.</p>
    <p>To generate the oriP probe, pUCoriP2 was digested with EcoRI and HindIII, and the oriP fragment (˜400 bp) isolated by low melt agarose gel electrophoresis. This DNA fragment was then digested with HpaII and the products labeled as detailed above. The smaller of the two HpaII fragments (˜90 bp) was isolated for use as the oriP probe. The MHCg probe was prepared by digesting p190H2KCAT with XhoI was labeling as before. The labeled DNA was then digested with HincII and the 90 bp probe fragment purified as before. This probe contains sequence from −190 to −100 of the upstream region of the H−2K<sup>b </sup>gene.</p>
    <p>The Δ6MHCg (−190 to +270) and Δ11MHCg (−138 to +270) competitor DNAs were prepared by digesting the plasmids, p190H2KCAT and p138H2KCAT, with XhoI and EcoRI. The H2KCAT fragments were isolated by low melt agarose gel electrophoresis.</p>
    <heading>RESULTS</heading> <p>Specific Detection of a λ Recombinant Expressing EBNA-1</p>
    <p>A model system was used to test the notion that a recombinant clone encoding a sequence-specific DNA binding protein could be specifically detected with a recognition site probe. The Epstein-Barr virus nuclear antigen (EBNA-1) was selected as the model protein. EBNA-1 is required for maintenance of the EBV genome as an autonomously replicating plasmid in human cell lines. It is also a transactivator of viral gene expression. The carboxy-terminal region of EBNA-1 (191 amino acids) has been expressed in <i>E. coli </i>as a fusion protein and shown to encode a sequence-specific DNA binding domain. The fusion protein binds to multiple high affinity sites at three different loci in the EBV genome. Two of these loci consitute a cis-acting element required for maintenance of the plasmid state (oriP). In the λgt11-EBNA-1 (λEB) recombinant the carboxy-terminal region of ENBA-1 was fused in frame to the carboxy-terminus of β-galactosidase (FIG. <b>16</b>). A lysogen harboring the λEB phage conditionally expressed a βgal-EBNA-1 fusion protein of expected size (approximately M.W. 145,000) that accumulated to a level of about 1%. The DNA binding activity of the fusion protein was assayed with a segment of oriP DNA that contained two high affinity sites for EBNA-1 (FIG. <b>16</b>). Extracts of λgt11 and λEB-lysogens were incubated with labeled oriP DNA and the products resolved by native polyacrylamide gel electrophoresis. With the λEB extract, a distinct set of protein-DNA complexes was observed. The formation of these complexes was specifically competed by an excess of plasmid DNA containing EBNA-1 binding sites. Thus, the β-gal-ENBA-1 fusion protein has the expected sequence-specific DNA binding activity.</p>
    <p>To establish conditions for detection of EB plaques with probes of oriP DNA, protein replica filters were generated from platings of the phage. These filters were screened with a variety of protocols using oriP or control DNAs. Under a defined set of conditions (see Experimental Procedures), λEB plaques can be specifically detected using radiolabeled oriP DNA. The control probe (ori) contains a high affinity binding site for the bacteriophage λO protein. The specific array of spots generated by the oriP probe corresponded to plaques on the master plate as well as to spots that reacted with antiserum to β-gal on the replica filter. Furthermore, in a similar experiment the oriP probe did not detect control λgt11 plaques. From a series of such experiments the following conclusions were drawn; (i) the specific detection of λEB plaques requires a DNA probe with at least one binding site for EBNA-1 (a duplex 30-mer with a consensus binding site sequence gave a signal comparable to a probe containing two or more binding sites), (ii) DNA probes longer than 150 bp yield higher non-specific signals, (iii) the addition of an excess of non-specific competitor DNA [poly(dI-dC)-poly(dI-dC)] to the binding solution reduces the non-specific signal, and (iv) both specific and non-specific interactions of the DNA probe with proteins on the replica filter are reversible. In view of this latter point and the fact that non-specific interactions typically have much shorter half-lives than the specific interactions, sequence-specific binding proteins can be detected after a suitable wash time.</p>
    <p>Given the ability to specifically detect λEB plaques with oriP DNA, reconstruction experiments were carried out to test the sensitivity of the screen. In these experiments the λEB phage was mixed with an excess of control λgt11 recombinants. Relica filters generated from such mixed platings were screened initially with oriP DNA and subsequently with antibodies to EBNA-1. In an experiment where approximately 5,000 phage were plated, with λEB being present at a frequency of 10<sup>−2</sup>, a identical number of positives (approximately 50) were detected with both oriP DNA and antibody probes. In fact, the two patterns are superimposable. Furthermore the signal to noise ratio of the DNA binding site probe was better than that of the antibody probe. Thus it is possible to screen for the λEB phage with an oriP DNA probe.</p>
    <p>Screening for Mammalian Clones Encoding Sequence-Specific DNA Binding Proteins</p>
    <p>A λgt11 library of cDNAs prepared with mRNA from human B cells was screened using the conditions developed with λEB. The DNA probe used in the screen contained a regulatory element from a mouse MHC class I gene (H-2K<sup>b</sup>. FIG. <b>17</b>). This sequence (MHC) was synthesized and cloned into the pUC polylinker. The mammalian transcriptional regulatory factors H2TF1 and NF-κB bind with high affinity to this MHC element. In a screen of 2.5×10<sup>5 </sup>recombinants, two positive phage, designated λh3 and λh4, were isolated. In an autoradiogram of a filter from the primary screen, a positive spot resulted in the isolation of λh3. Partially purified λh3 and λh4 phage were challenged with other DNA probes to determine if their detection was specific for the MHC probe. λh3 and λh4 were not detected by the ori probe. These phage were also not detected by labeled pUC polylinker DNA or by a related probe (OCTA) containing a recognition site for the immunoglobulin octamer binding protein(s). A mutant MHC binding site probe (mhcl FIG. 17) was used to more stringently test the sequence-specificity of the presumptive fusion proteins. The mhcl probe did not detect either λh3 or λh4 plaques. These data strongly suggested that the two phage express proteins that bind specifically to the MHC element.</p>
    <p>Characterization of the DNA Binding Proteins Encoded by λh3 and λh4</p>
    <p>Direct evidence that the β-gal fusion proteins encoded by λh3 and λh4 are responsible for the sequence-specific DNA binding activities was obtained by screening Western blots with DNA and antibody probes. Lysogens of λgt11, λh3 and λh4 were isolated and induced to generate high levels of their respective β-gal proteins. Western blots of proteins from induced lysogens were prepared and the immobilized proteins were briefly denatured with 6M guanidine and then allowed to renature (see Experimental Procedures above). This treatment increased the recovery of active molecules. Two equivalent transfers were initially probed with either the MHC element or the OCTA control DNA. A set of four bands specific to the MHC probe and the λh3, λh4 tracks was observed. The two largest species of this set are labeled P1 and P2. The same transfers were then probed with antibodies to β-gal. A pair of novel fusion protein bands was observed with each of the two recombinant lysogens. These bands corresponded to the species P1 and P2 detected with the MHC probe. This shows that λh3 and λh4 encode β-gal fusion proteins which bind specifically to the MHC element DNA. The two phage may be identical since they encode the same size fusion proteins. P1 (approximate m.w. 160,000) probably represents the full length fusion protein whereas P2 is a presumptive proteolytic cleavage product. Since the β-gal portion of this fusion polypeptide has a molecular weight of approximately 120,000, the cDNA encoded portion must have a molecular weight of 40,000.</p>
    <p>A gel electrophoresis DNA binding assay was used to confirm the sequence specificity of the λh3 and λh4 fusion proteins as well as to better define their recognition properties. Extracts derived from the λgt11, λh3 and λh4 lysogens were assayed, with the MHC probe. A novel DNA binding activity was detected specifically in extracts of the λh3 and λh4 lysogens. This activity was IPTG inducible indicating that it was a product of the lacZ fusion gene. A competition assay indicated that the activity represented a sequence-specific DNA binding protein. Two 5′ deletion mutants of the H−2K<sup>b </sup>genomic sequence was used as competitor DNAs. The segment 6MHCg extends to 190 nucleotides upstream of the transcription start site and contains the MHC sequence element. The segment Δ11MHCg, on the other hand, only contains 138 nucleotides of sequences upstream of the initiation site and therefore lacks the MHC element. Increasing amounts of Δ6MHCg specifically competed for the binding of the λh3 fusion protein to the MHC element oligonucleotide probe while the control Δ11MHCg did not compete. It should be noted that the sequences flanking the MHC element in the probe used for the initial screening, the cloned oligonucleotide, are totally difference from the sequences flanking the same element in the genomic probe, Δ6MHCg. Therefore, the fusion protein appears to exclusively recognize the common MHC element. This was confirmed by a direct DNA binding assay with a genomic sequence probe (MHCg) containing the MHC element. Both the oligonucleotide (MHC) and genomic (MHCg) probes gave rise to similarly migrating complexes. Furthermore, a double base substitution mutant (mhc1, FIG. 17) abolished recognition by the fusion protein. The mutant sequence contains a transverion in each half of the symmetric MHC element. These changes destroy the symmetry of the element and abolish binding by either H2TF1 or NF-κB.</p>
    <p>The immunoglobulin κ chain gene enhancer contains a binding site ( EN) for NF-κB. This site is related in sequence to the MHC element but is recognized by H2TF1 with a 10 to 20 fold lower affinity (FIG. <b>17</b>). A mutant κ enhancer (κEN) has been characterized both in vivo and in vitro. This mutant sequence has no B cell specific enhancer activity and is not bound by NF-κB. The mutant contains clustered base substitutions and an insertion of a base pair in one of the two symmetric half sites (FIG. <b>17</b>). The binding of the λh3 fusion protein to the wild type κ-element and the mutant version was tested. The κEN probe generated a complex with a mobility similar to those obtained with the MHC probes. No specific complex was formed with the mutant κ-enhancer DNA. Experiments in which the MHC and κ-enhancer binding sites were tested for competition with binding of the MHC probe showed that the fusion protein bind with 2-5 fold higher affinity to the MHC site (data not shown). The κEN site differs, in part, from the MHC site by the substitution of two adenine residues for guanine residues. As discussed below, these guanine residues are probably contacted by the fusion.</p>
    <p>The contacts of the fusion protein with the MHC element were probed chemically by modification of the DNA with dimethylsulfate. After partial methylation at purine residues, the modified probe was used in the gel electrophoresis DNA binding assay. Free (F) and bound (B) probe DNA was recovered, subjected to chemical cleavage at methylated interference experiment. On both the coding and non-coding strands strong interference was detected when any of central guanine residues of each putative half site was modified at the N-7 position in the major groove. Weaker interference was observed when the external guanine residue in either putative half site was similarly modified. Thus the fusion protein appears to symmetrically contact the MHC element in a manner similar to both H2TF1 and NF-κB.</p>
    <p>Hybridization Analysis with the cDNA Segment of the Recombinant Phage</p>
    <p>The recombinant phage λh3 and λh4 contain cross-hybridizing and equivalent size (approximately 1 b) cDNA segments. The inserts also have indistinguishable restriction maps and therefore appear to be identical. Southern blot hybridization confirmed that these cDNA segments are homologous to sequences in the human genome. The patterns of hybridization to restriction digests of genomic DNAs of various human cell lines are identical. Furthermore, the fact that restriction digests with Bam HI (no site in cDNA) and Pst I (on site in cDNA) both generate two prominent bands suggests that the cDNAs are derived from a single copy gene. A similarly simple hybridization pattern is observed on probing the mouse and rat genomes.</p>
    <p>The expression of the human gene was analyzed by Northern blot hybridization. A single, large transcript (approximately 10 kb) was observed with polyA(+) RNA from both B (X50-7) and non-B human cells (HeLa). This transcript is moderately abundant in both cell types. Since the cDNA library was constructed by oligo dT priming, we were probably fortunate to obtain the coding region for the DNA binding domain within the 1 kb segments of the recombinant phage. However, this only illustrates the power of the screening strategy for the isolation of clones encoding sequence-specific DNA binding domains.</p>
    <p>Discussion</p>
    <p>A novel strategy is disclosed for the molecular cloning of genes encoding sequence-specific DNA binding proteins. This strategy can be used to isolate genes specifying mammalian transcription regulatory proteins. An important step in this approach is the detection of bacterial clones synthesizing significant levels of a sequence-specific DNA binding protein by screening with a labeled DNA binding site probe. This approach is similar to that previously developed for the isolation of genes by screening recombinant libraries with antibodies specific for a given protein. In fact, the phage expression vector, λgt11, developed previously for immunological screening can be in this approach.</p>
    <p>The feasibility of the strategy was established by the specific detection of a phage recombinant, λEB, encoding a β-gal-EBNA-1 fusion polypeptide with oriP DNA. Conditions have also been developed for the selective detection of <i>E. coli </i>colonies expressing high levels of EBNA-1 or the bacteriophage λO protein with their respective binding site DNAs. In these cases, a plasmid expression vector was employed. Using the conditions developed with λEB, we have screened phage cDNA libraries with three difference DNA probes. Screening with a probe containing the H2TF1 site in the MHC class I gene H-2K<sup>b </sup>led to the isolation of two identical clones that specify a putative transcription regulatory protein (properties discussed below). In similar screens with two other DNA probes, positive recombinant phage were also isolated at a frequency of approximately 1/100,000. However, the DNA binding proteins encoded by these phage do not appear to recognize specific sequence elements but rather to bind sequence nonspecifically to either single strand or double strand DNA. Although detection of these types of clones represented a troublesome background in this study their isolation suggests that recombinants encoding different types of DNA binding proteins can be detected by such functional screens of expression libraries. In future screens for recombinants encoding site-specific DNA binding proteins, the detection of these other types of clones might be selectively suppressed by inclusion of a non-specific competitor DNA that is structurally more similar to the probe than poly(dI-dC)-poly(dI-dC).</p>
    <p>The prospects for the isolation of other cDNAs encoding sequence-specific binding protein by this strategy can be assessed by examining the three assumptions on which it is based: (i) functional expression of the DNA binding domain of the desired protein in <i>E. coli</i>, (ii) a strong and selective interaction of the binding domain and its recognition site, and (iii) high level expression of the DNA binding domain. A number of eukaryotic sequence-specific DNA binding proteins have been functionally expressed in <i>E. coli</i>. These include the proteins GAL4, GCN4 and MAT 2 of yeast, ftz of Drosophila, TFIIIA of Xenopus, E2 of the bovine papilloma virus and EBNA-1 of the Epstein Barr Virus. In most cases, the functional DNA binding domain is contained within a short tract of amino acids. Thus, it is reasonable to expect the functional expression in <i>E. coli </i>of the sequence-specific DNA binding domain of most eukaryotic regulatory proteins. The equilibrium association constants of site-specific DNA binding proteins range over many orders of magnitude (10<sup>7</sup>-0<sup>12 </sup>M). The following analysis suggests that successful screening may be restricted to proteins with relatively high binding constants. If a regulatory protein has an association constant of 10<sup>10 </sup>M, then under the screening conditions (the DNA probe is in excess and at a concentration of (˜10<sup>10 </sup>M) approximately half of the active molecules on the filter will have DNA bound. Since the filters are subsequently washed for 30 minutes, the fraction of protein-DNA complexes that remain will be determined by their dissociation rate constant. Assuming a diffusion limited association rate constant of 10<sup>7 </sup>M<sup>−1 </sup>S<sup>−1</sup>, the dissociation rate constant will be 10<sup>−3 </sup>S<sup>−1</sup>. Such a protein-DNA complex will have a half life of approximately 15 minutes. Thus only a quarter of the protein-DNA complexes will survive the 30 minute wash. For a binding constant of 10<sup>9 </sup>M, only about a tenth of the active protein molecules will have DNA bound and much of this signal will be lost, since the half-life of these complexes is approximately 1.5 minutes. Isolation of recombinants encoding proteins with binding constants of 10<sup>9 </sup>or lower may be possible given that the binding of probe to less than 1% of the total fusion protein within a plaque can be detected. The sensitivity of the current methodology for low affinity proteins could be significantly enhanced by covalent stabilization of protein-DNA complexes. This might be accomplished by procedures such as UV-irradiation of pre-formed complexes. Since the binding constants of regulatory proteins are dependent on ionic strength, temperature and pH, these factors might also be manipulated to enhance detection.</p>
    <p>The successful detection of λEB and λh3 recombinants with DNA binding site probes required high level expression of their fusion proteins. In both cases, the fusion proteins accumulate, after induction, to a level of about 1% of total cellular protein. This level of recombinant protein expression is typical of λgt11 as well as other <i>E. coli </i>vectors. The strategy of cloning a gene on the basis of specific detection of its functional recombinant product in <i>E. coli </i>has considered potential. Indeed, while our work was in progress, this approach was used by other to isolate clones encoding a peptide acetyltransferase and a calmodulin-binding protein. Direct screening of clones encoding recombinant protein products has also been used to isolate ras GTP-binding mutants.</p>
    <p>The λh3 recombinant expresses a β-gal fusion protein that recognizes related transcription control elements in the enhancers of the MHC class I and immunoglobulin κ-chain genes (see FIG. 17 for sequences). This protein also binds a similar element in the SV40 enhancer 72 bp repeat. Furthermore, there are two putative binding sites in the long terminal repeat (LRT) of the HIV genome (FIG. <b>17</b>). One of these is identical to the site in the SV40 enhancer and therefore should be recognized by the fusion protein. The existence of a clone such as λh3 was anticipated since it had previously been shown that a common factor, NF-κB, binds to the three related elements in the enhancer, the SV40 72 bp repeat an the HIV-LTR. Interestingly, these three binding sites are more closely related to one another than they are to the MHC site (FIG. <b>17</b>). The former set can be viewed as variants of the MHC site which exhibits perfect two-fold symmetry. It should be noted that the pUC polylinker contains the sequence, CGGGGA, which is a variant of one of the symmetric halves (TGGGGA) of the MHC element. The fusion protein does not bind with detectably affinity to the pUC polylinker. Thus, a high affinity interacter appears to require both symmetric halves.</p>
    <p>Even though the above control elements represent quite similar sequences, they function in very different regulatory capacities. The MHC element is a component of an enhancer that functions in a variety of cell types that express MHC class I genes. The κ-element, on the other hand, is a component of a cell-type specific enhancer that functions only in B cells. The activity of this enhancer is induced in pre-B cells upon their differentiation into mature B lymphocytes. Such differentiation, in vitro, is accompanied by transcriptional activation of the chain gene. The κ-element appears to dictate the B cell specificity of the κ-enhancer. The different modes of functioning of the MHC and κ-elements are correlated with the properties of their corresponding recognition factors, H2TF1 and NF-κB. H2TF1 activity is detected in a variety of differentiated cell types and this protein appears to stimulate MHC class I gene transcription approximately 10-fold. On the other hand, NF-κB activity is detected only a mature B cells. In addition, this activity is induced during differentiation of pre-B cells to mature lymphocytes. Finally, NF-κB activity is also induced by phorbol ester treatment of non-B cell lines (HeLa, Jurkat). In the case of Jurkat cells, a T4<sup>+ </sup>human T cell line, NF-κB appears to stimulate the transcriptional activity of the HIV-LTR. It should be noted that induction of NF-κB in non-B cells does not require new protein synthesis. Thus the protein for NF-κB must exist in cells before induction and the activated by a post-translational modification.</p>
    <p>The DNA binding properties of the fusion protein encoded by the recombinant λh3 overlap those of H2TF1 or NF-κB. Mutants of the MHC and κ-elements that are not recognized by H2TF1 or NF-κB are also not bound by the fusion protein. The recombinant protein binds the MHC element DNA with 2-5 fold higher affinity than the κ-element. In this regard, the fusion protein has relative affinities intermediate between those of H2TF1 and NF-κB. H2TF1 binds the MHC element with 10- to 20-fold higher affinity than the κ-element while NF-κB recognizes both elements with roughly equivalent affinity. This intermediate relationship is also observed in the comparison of the methylation interference patterns of the three DNA binding activities. Methylation of any of the central six guanine residues in the MHC site strongly interfers with the binding of all three activities. Methylation at either of the two external guanines partially interferes with recognition by the fusion protein. In contrast, H2TF1 binding is strongly suppressed upon methylation of either of these residues while NF-κB binding shows little perturbation upon this modification. This analysis of the three DNA binding activities is limited by the use of cell extracts and not purified proteins. Furthermore, the properties of a recombinant protein may be different from those of its native counterpart. Thus, it is not possible to be definitively relate the protein encoded by λh3 to either H2TF1 or NF-κB.</p>
    <p>Antibodies raised against the λh3 fusion protein will be useful in clarifying its structural relationship with H2TF1 and NF-κB. A definitive relationship will emerge from a comparison of the deduced amino acid sequence of the cDNA and the protein sequences of H2TF1 and NF-κB. It should be noted that in terms of protein expression, both H2TF1 and NF-κB are present in a wide variety of mammalian cells. Furthermore, the DNA binding specificities of these two factors are remarkably similar. These facts as well as the observations that the cDNA in λh3 hybridizes to a single copy gene and to a single mRNA in both B and non-B cells suggest that all three binding activities may be products of the same gene. This hypothesis would imply that H2TF1 and NF-κB represent alternative modifications of a common protein.</p>
    <heading>Example 7</heading> <heading>Cloning of the IgNFB Gene</heading> <p>Methods</p>
    <p>DNA Sequencing</p>
    <p>DNA sequencing was performed on double stranded plasmid DNA templates according to the Sanger dideoxy-nucleotide protocol as modified by United States Biochemical for use with bacteriophage T7 DNA polymerase (Sequenase). The entire sequence was confirmed by sequencing the opposite strand and in the GC-rich regions by sequencing according to Maxam and Gilbert (<i>Methods Enzymol., </i>65: 449-560, (1980).</p>
    <p>Plasmid Constructions</p>
    <p>cDNA's were subcloned from λgt11 to pGEM4 (Promega), and these plasmids were used for DNA sequence analysis and in vitro transcription. Plasmid pBS-ATG was kindly provided by H. Singh and K. LeClair and was constructed by ligating a 27 bp long oligonucleotide containing an ATG codon surrounded by the appropriate boxes for efficient initiation, TGCACACCATGGCCATCGATATCGATC, into the Pstl site of pBS−/+Bluescropt plasmid (Stratagene). The expression vector pBS-ATG-oct-2 depicted in FIG. 19A was designed for transcription and translation in vitro and was constructed by cleaving pBS-ATG with Smal and ligating the blunt-ended EcoRI 1.2 kb cDNA fragment from plasmid 3-1 (position 655 to 1710 in FIG. <b>18</b>A).</p>
    <p>In Vitro Transcription/Translation</p>
    <p>In vitro transcription and translation reactoins were performed as recommended by the manufacturer (Promega).</p>
    <p>DNA Binding Assay</p>
    <p>The EcoRI/HindIII 50 bp fragment containing the wild type octanucleotide sequence ATGCAAAT in the BamHI site of pUC18 polylinker was <sup>32</sup>P-labeled (50,000 cpm/ng) and 1 ng DNA probe was incubated with 1 μl of the reacted/unreacted reticulocyte lysate. The bindign reactions were incubated at room temperature for 30 min. and contained 10 mM Tris HCl pH7.5, 50 mM NaCl, 1 mM DTT, 1 mM EDTA pH8, 5% glycerol, 25 μl/ml sonicated denatured calf thymus DNA in 2.5 μl/ml sonicated native calf thymus DNA as nonspecific competitors. The complexes were resolved by electrophoresis in 4% polyacrylamide gel (acrylamide: bisacrylamide weight ratio of 20:1), containing as buffer 25 mM Tris HCl pH8.5, 190 mM glycine, 1 mM EDTA buffer as previouly described (Singh et al., Nature 319: 154-158 (1986)).</p>
    <p>Purification of NF-A2</p>
    <p>NF-A2 was purified to &gt;90% homogeneity from nuclear extracts derived from the human Burkett's lymphoma cell line, BJAB. Purification was accomplished by sequential fractionation on Zetachrom QAE discs (Cuno Inc.), heparin sepharose (Pharmacia), ssDNA cellulose (Pharmacia), and on a DNA affinity column which contained an immobilized double straded (ds) segment containing the octanucleotide sequence. In vitro translated, <sup>35</sup>S-methionine-labeled, oct-2 protein was purified by chromatography on dsDNA cellulose followed by affinity chromatography on the octanucleotide DNA affinity column.</p>
    <p>Tryptic Digestions of NF-A1 and oct-2 Protein</p>
    <p>Tryptic digests were performed at room temperature in a buffer consisting of 20 mM Hepes,KOH, pH 7.9, 20% glycerol, 0.5 M KCl, 0.2 mM EDTA, 0.5 MM DTT. Aliquots of purified NF-A2 (˜250 ng) or of affinity purified oct-2 protein (90,000 cpm) were incubated with varying amounts of trypsin (affinity incubated with varying amounts of trypsin (affinity purified trypsin was a gift of Dan Doering). After 60 minutes, reactions were terminated by the addition of 2.5 volumes of SDS-PAGE sample buffer and were boiled for 5 minutes. The tryptic digestes were resolved on 10% polyacrylamide gels. Tryptic fragments of NF-A2 were visualized by silver-staining. Tryptic fragments of <sup>35</sup>S-methionine labeled oct-2 protein were visualized by autoradiography after treatment of the gel with En<sup>3</sup>hance (Dupont).</p>
    <p>To clone the gene encoding the lymphoid-specific octamer binding protein, IgNF-B (NF-A2), a randomly primed, non-size selected cDNA library in λgt11 was constructed using cytoplasmic poly (A)-containing mRNA from a human B cell lymphoma cell line, BJAB. We had previously observed that this cell line contained a particularly large amount of NF-A2 when 28 lymphoid cell lines were surveyed. By randomly priming the cDNA synthesis we expected to obtain recombinant phage encoding the octamer motif binding domain even if that domain was encoded by the 5′ end of a long mRNA. The randomly primed cDNA library in λgt11 was generated by standard methods (Gubler, U. and Hoffman, B. J., Genes 25:263-269 (1983)). Random hexamers (Pharmacia) were used to prime the first strand cDNA synthesis. The unamplified library contained 500,000 recombinants. This library was screened by the method described above using a radiolabelled DNA probe consisting of four copies, in direct orientation, of a 26 bp oligonucleotide derived from the Vk41 promoter. The probe was constructed by cloning four copies of the oligonucleotide in direct orientation into the BamH1 site of the pUC polylinker and radiolabelling the 112 bp Sma1-Xba1 fragment. The library was screened with the tetramer probe (at 1×10<sup>6 </sup>cpm/ml) as described above for the cloning of NF-κB with the following modification. Previous screens using poly(dI-dC)-poly(dI-dC) as the nonspecific competitor DNA yielded recombinant phage encoding single stranded DNA binding proteins. The signal from these phage but not phage encoding sequence-specific DNA binding proteins could be efficiently competed with denatured calf thymas DNA (5 μg/ml) and therefore this nonspecific competitor was substituted for poly(dI-dC)-poly(dI-dC) in all subsequent screens.</p>
    <p>From a primary screen of 450,000 phage plaques, three plaques were isolated which bound this tetramer probe. Two of these phage, phage 3 and phage 5, were found to give plaques that bound specifically to the tetramer probe in that they did not bind DNA probes which lacked the octamer motif. These two phage bound probes containing one copy of the κ promoter octamer motif with a much lower affinity than they bound the tetramer probe. Even when four-fold more monomer probe was used then tetramer probe, the tetramer probe still gave a greater signal suggesting that the better binding of the tetramer probe was not merely a result of increasing the molar concentration of binding sites in the screen. Certainly in the case of phage 5, which showed dramatically better binding to the tetramer probe, it seems most likely that the tetramer probe was able to bind simultaneously to multiple phage fusion proteins on the filter. This multipoint attachment would be expected to dramatically decrease the dissociation rate and thus, increase the avidity of the interaction. Genes encoding DNA binding proteins with relatively low binding affinities could be cloned by screening λgt11 expression libraries with such multimer probes.</p>
    <p>The specificity of the DNA binding proteins encoded by the recombinant phage was investigated by preparing extracts of induced phage lysogens. Lysogen extracts from both phages bound to the tetramer probe in a mobility shift assay whereas lysogen extracts from non-recombinant λgt11 showed no binding to this probe. Only the phage 3 extract bound strongly to the κ promoter probe. Because the inserts of phage 3 and phage 5 (1.2 kb and 0.45 kb in size, respectively) were found to cross-hybridize by Southern blotting analysis, phage 3 was chosen for further analysis.</p>
    <p>Phage 3 encoded an octamer binding protein as demonstrated by a competition mobility shift assay in which the lysogen extract was bound to the κ promoter probe in the presence of competing unlabelled DNA fragments containing either the wild type or mutant octamer motifs. Phage lysogen extracts were prepared as described above for NK-κB cloning. The extracts were assayed in a mobility shift assay as described above using the octamer-containing PvuII-EcoR1 fragment from pSPIgVk as the radiolabelled probe. Binding reactions were carried out in the absence or presence of 24 ηg of cold competitor DNA containing no octamer motif, the wild type octamer motif or mutant octamer motifs as described.</p>
    <p>The wild type octamer motif competed efficiently for binding but the octamer motifs containing point mutations either did not compete or competed less well than the wild type motif. In fact, the two mutants which showed slight competition for the binding of the lysogen protein, TCATTT<u>C</u>CAT and <u>AT</u>ATTGCAT, were the only mutants which somewhat competed the binding of NF-A1 and NF-A2 in a WEHI 231 nuclear extract.</p>
    <p>The phage-encoded octamer binding protein was further compared to NF-A1 and NF-A2 using a methylation interference footprinting assay. Methylation interference was performed as described using the non-coding strand of the octamer-containing PvuII-EcoR1 fragment of pSPlgVk as radiolabelled probes. The probes were partially methylated and used in preparative mobility shift DNA binding assays. DNA present in the bound bands (NF-A1 and NF-A2 bands from a nuclear extract from the BJAB cell line (or phage 3 lysogen extract bound band or free bands) was isolated, cleaved at the modified purine residues and subjected to denaturing polyacrylamide gel electrophoresis. The footprint obtained using the lysogen extract was centered over the octamer motif and was very similar to the footprints of NF-A1 and NF-A2 from a BJAB nuclear extract and from a WEHI 231 nuclear extract (see above). Minor differences were seen between the footprints of the lysogen and nuclear extract proteins which could reflect changes in affinity and/or specificty of DNA binding as a result of fusion of the insert-encoded octamer binding protein with β-galactosidase. Alternatively, the phage insert could encode an octamer binding protein distinct from NF-A1 and NF-A2.</p>
    <p>The phage-encoded β-galactosidase fusion protein was directly shown to be the octamer binding protein in the phage lysogen extracts. Phage lysogen extracts were subjected to SDS polyacrylamide gel electrophoresis and transferred to nitrocellulose filters. After a denaturation/renaturation procedure (Celenza, J. L. and Carlson, M. <i>Science </i>233:1175-1180 (1986)), the filters were probed with either the radiolabelled octamer-containing tetramer probe (OCTA) or a non-specific DNA probe (pUC). The OCTA probe specifically bound to the β-galactosidase fusion proteins of phage 3 and phage 5 to a much greater extent than the pUC probe, thus formally showing that the octamer binding activity was encoded by the phage inserts. The apparent molecular weights of the largest fusion proteins of phage 3 and phage 5 lysogens are consistent with the entire phage inserts contributing coding sequences to the fusion proteins. Prototeolysis was presumed to account for the heterogeneity in apparent molecular weight of the fusion proteins.</p>
    <p>The insert of phage 3, which defines what we term the OCT-2 gene, was used in a Southern blot analysis to probe human and mouse genomic DNA digested with several restriction enzymes. Restriction enzyme digested genomic DNA was electrophoresed through a 1% agarose gel and transferred to Zetabind (CUNO Laboratory, Inc.) by standard techniques (Maniatis, T., Frisch, E. F. and Sambrook, J. Molecular Cloning. A Laboratory Manual. Cold Spring Harbor Laboratory Press, NY. (1982)). The phage 3 insert was radiolabelled by randomly primed synthesis using hexanucleotides (Pharmacia). Following standard prehybridization high-stringency hybridization (Maniatis, supra.) with the OCT-2 probe the filters were washed with 0.2×SSC, 0.1% SDS or 2×SSC, 0.1% SDS.</p>
    <p>One or two bands were observed in each restriction enzyme digest which is consistent with OCT-2 being a single genetic locus. No rearrangements or amplifications of the gene were observed in a survey of 8 lymphoid and non-lymphoid cells lines including BJAB. The strength of the signal on the mouse Southern blot at high stringency suggested that the gene is highly conserved between human and mouse.</p>
    <p>The oct-2 cDNA segment (1.2 kb) of phage 3 was used to identify additional overlapping recombiants in the same library. One of these phage (pass-3) contained a 1.8 kb DNA insert. Sequence analysis of the cDNA segment in the original λgt11 phage (3-1) revealed a long open reading frame (ORF) which was ended with multiple nonsense codons at its 3′ terminus. Sequence analysis of the pass-3 segment yielded an identical sequence through the open reading frame but an abrupt transition to a novel sequence occurred at the C-terminue (FIG. 18B; see below). The N terminus of the open reading frame in both of these cDNA segments was not represented in the cDNA inserts. Additional recombinats from the λgt11 library were identified by screening with a probe from the N0terminal portion of the pass-3 segment. This resulted in the isolation of a 0.75 kb cDNA segment (pass-5.5) whose sequence extended the N-terminal portion o the previously identified open readign frame. In this cDNA segment, a nonsense codon is found 36 pb upstream of the first AUG in the open frame. The sequence context of this AUG conforms well to that expected for an initiation codon (Kozak, Cell 44: 283-292 (1986)). Two other AUG codons occur at positions 6 and 13 in the reading frame. Each of these also has an excellent context for initiation. The N terminus of the protein has been arbitrarily assigned to the 5′-most AUG codon. The cDNA sequence extends 66 bases 5′ from this position but the total length of the 5′ untranslated region has not been determined.</p>
    <p>The sequences of pass-5.5, pass-3 and 3-1 were combined to form an open reading frame encoding a protein of 466 amino acids in length as shown in FIG. 46 amino acids in length as shown in FIG. <b>18</b>. FIG. 18 shows the amino-acid sequence of oct-2 protein depicted in plain capital letters. cDNA-clone pass-5.5 spans from position 1 (5′ end) to position 750 (3′ end). cDNA clone pass-3 5′ end and 3′ end are respectively at position 92 in FIG. 18<i>a </i>and 1847 in FIG. 18<i>b</i>. cDNA clone 3-1 starts at position 650 and ends at position 1710. The nucleotide sequence shown in panel A was reconstructed by merging the DNA sequences from clone pass-5.5 from position 1 to 100, from clone pass-3 from 100 to 660 and from clone 3-1 from position 660-1710. Extensive nucleotide sequence overlaps were available to allow unequivocal merges. Sequence of protein encoded by the long overlappingopen reading frame (LORF, 277aa) is shown in italic letters. Wavy arrows delimit the glutamine (Q)-rich, glutamic and aspartic (E/D)-rich and glycine (G)-rich regions, respectively. Solid arrows delimt the helix-turn-helix motif. Boxed leucine (L) residues are spaced exactly by seven residues. Vertical arrow indicates the position where the nucleotide sequence diverges with that shown in panel B. Stars indicate stop codons.</p>
    <p>FIG. 18<i>b </i>shows the nucleotide sequence of the 3′ terminus and redicted amino acid sequence of the C-terminus derived from clone pass-3. The code is the same as in A and the vertical arrow denote sthe divergence point.</p>
    <p>FIG. 18<i>c </i>is a schematic representation of the amino acid sequence deduced from oct-2 gene derived cDNA. The code is as in panel A. The DNA binding domain is depicted as DNA and the region containing the four regularly spaced L residues is boxed-in. LORF stands for long open reading frame, N stands for N-terminus and C for COOH-terminus.</p>
    <p>Data presented below suggests that this ORF encodes one form of NF-A2 (oct-2). The amino acid sequence of oct-2 has several interesting features (FIG. <b>18</b>C). It contains three glutamine (Q) rich blocks (ranging from 50% of Q content) in the N-terminal part of the polypeptide, beginning at nucleotide positions 376, 448 and 502, and a comparably acidic region [aspartic (E) or glutamic (D) amino acids] between positions 648 and 678. Clusters of Q resideus as well as E or D amino acids have been described previously in many transcription factors. Such acidic regions in other factors have been shown to be important in activation of transcription (Gill and Ptashne, <i>Cell </i>51: 121-126 1987; Hope et al., Nature 333: 635-640 (1988)).</p>
    <p>The region of oct-2 responsible for sequence-specific DNA binding, depicted “DNA”, is discussed below. Downstream of this position is a series of four leucine residues separated by exactly seven amino acids (position 1227 to 1293 in FIG. <b>18</b>A). A similar configuration of leucine residues in the transcription factor C/EBP has been suggested to form an amphipathic α-helical structure where the leucine residues are arranged along one side of the helix. Two such helices are throught to interact by a “leucine zipper” mechanism generating a dimeric protein (Handschultz et al., Science 240: 1759-1764 (1988); Landschultz et al., Genes &amp; Development 2: 786-800 (1988)).</p>
    <p>Consistent with this sugestion, no helix disrupting proline residue is present in oct-2 in the 22 amino acid tract defined by the four leucines. However, unlike the first example of a leucine zipper”, protein C/EBP, the potential α-helical region in oct-2 does not possess a high density of paried charged residues which could stabilize the structure. Also, unlike the C/EBP protein, which binds DNA specifically as a homodimer probably by pairing through the “leucine zipper”, the oct-2 protein appears to specifically bind DNA as a monomer. It is interesting to speculate that the “leucine zipper” region of oct-2 might be important for interaction with other proteins as there is no obvious reason to restrict the binding of such a structure to self-recognition.</p>
    <p>Searches for sequence similarities in the GenBank library revealed that a region of the oct-2 protein from position 952 to 1135 was distantly related to a family of proteins containing homeoboxes. The 60-residue homeobox domain is highly conserved among 16 examples in different Drosophila genes (Gehering, Science 236: 1245-1252 (1987)). This level of conservation extends to homeobox sequences found in vertebrates and worms. Among this total family, nine of the 60 residues are invariant. The oct-2 protein only contains six of these nine residues and four of these six sites are clustered in the sub-region of the homeobox thought to be related to the helix-turn-helix structure (see FIG. <b>20</b>). As shown in FIG. 20A, a 60 amino acid region of oct-2 contains 30% identity with the prototype homeobox sequence in the Antennapedia (Antp) protein.</p>
    <p>FIG. 20 shows the amino acid sequence alignment of the DNA binding domain of oct-2 factor with homeoboxes from Antp. (Schneuwly et al., <i>EMBO J. </i>5: 733-739 (1986), cut (Blochlinger, <i>Nature </i>333:629-635 (1988), en (Poole et al., <i>Cell </i>40: 37-43 (1985), proteins (boxed-in amino acid sequences from the <i>S. cerevisae </i>proteins Matal (Miller, <i>EMBO J. #: </i>1061-1065 (1984), Matα2 (Astell et al., <i>Cell </i>53: 339-340 (1988) and <i>C. elegans </i>protein mec-3 (Way and Chalfie, <i>Cell </i>54: 5-16 (1988). THe nine invariant residues in canonical homeobox sequences Atnp, cut, and en are listed below the boxed-in amino acid sequences and shown in bold print if present in the amino acid sequences. The stars indicate the hydrophobic amino acids that are critical for the protein to maintain the helix-turn-helix structure (Pabo and Sauer, 1984). Solid arrows delimit the helix-turn-helix domain.</p>
    <p>That the homeobox specifies a sequence-specific DNA binding domain is most strongly argued by its homology with the DNA binding domain of the yeast mating regulatory protein, MATα (Astell et al., Cell 27: 15-23 (1981); Scott and Weiner, <i>Proc. Natl. Acad. Sci. USA </i>81: 4115-4119 (1984)), which also has homology through this. subregion of the homeobox but does not conserve the other invariant of the homeobox. The homologous regions in these proteins can be folded into a helix-turn-helix-structure similar to that first identified in the structural analysis of phage λ repressor (for a review see Pabo and Sauer, Ann. Rev. Biochem. 53: 293-321 (1984)). A prediction of the most probable secondary structure of oct-2 also revealed a helix-turn-helix structure between the residues of isoleucine (position 1041) and cysteine (position 1090). Thus, by analogy, we propose that this region of oct-2 specifies the sequence-specified binding of the protein.</p>
    <p>As mentioned above, sequences at the 3′ end of the pass-3 recombinant abruptly diverged from that of recombinant 3-1 at the position (1463) of its termination codon (see vertical arrow in FIG. <b>18</b>B). The substituted sequences in the second recombinant, pass-3, extended the reading frame of the oct-2 realted protein by an additional 16 amino acids. To rule out a possible artifactual sequence generated by the insertion of fragment during construction of the cDNA library, total polyA(+) RNA from the BJAB cell line was analyzed by Northern blot with a DNA fragment from the novel 3′ terminal portion of the pass-3 cDNA. This specific probe hydribized only to the two fastest migrating mRNAs of the total family of six mRNAs which were detected by hybridization with the total 3-1 cDNA. A similar specific probe was excised from the 3′ terminus of the 3-1 cDNA. In contracts, this probe only hybridized to the two slowest migrating mRNAs in the total family of six. This suggests that the two cDNA segments correspond to different populations of oct-2 mRNAs.</p>
    <p>The proteins encoded by the two cDNAs should only differ at their C terminus by 16 amino acids or approximately 1.5 kD. In vitro transcription/translation of subfragments of the 3-1 and pass-3 recombinants was used to confirm this prediction. Fragments representing the 3′ portions of 3-1 and pass-3 were subcloned into the expression plasmid PBS-ATG. The resulting plasmid DNAs were transcribed with bacteriophage T7 RNApolymerase and were subsequently translated in a reticulocyte system. The resulting polypeptides migrated with the mobilities of the anticipated molecular weights 34 kD and 32.4 kD. the polypeptide from the pass-3 cDNA was 1.6 kD larger than that from the 3-1 cDNA. Both polypeptides specifically bound a probe containing the octanucleotide sequence, producing a readily detectable DNA-protein complex in the gel mobility assay. This suggests that the oct-2 gene is expressed as a family of polypeptides in B-cells.</p>
    <p>The potential significance of these additional 16 amino acids is unclear. These two cDNAs almost certainly differ by alternative splicing patterns of RNA transcribed from the oct-2 gene. Furthermore, it is likely that the oct-2 gene encodes a more diverse set of mRNAs than those partially defined by these two cDNAs. Six different length mRNAs are produced at significant levels in mature B cells. The relative amounts of these mRNAs vary between pre-B, B and plasma cell lines (Staudt et al., Science 241: 577-580 (1988)). This population could reflect variations in sites of initiation of transcription and of polyadenylation as well as further differences in splicing patterns.</p>
    <p>The expression of the OCT-2 gene was assessed by Northern blot analysis of mRNA from 13 lymphoid and non-lymphoid cell lines and was found to be predominantly restricted to lymphoid cells. Poly(A)-containing mRNA (3 μg, or 20 μg) or total mRNA (30 μg) was analyzed from the following cell lines. 1. NIH 3T3: mouse fibroblast; 2. 38B9: mouse pre-B cell line; 3. WEHI 231: mouse mature B cell line; 4. A431: human epidermal cell line; 5. U1242: human glioma cell line; 6. RB27: human retinoblastoma cell line; 7. Jurkat: human T cell line; 8. Namalwa: human mature B cell line; 9. BJAB: human mature B cell line (poly(A)-containing mRNA); 10. BJAB (total mRNA); 11. Hut78: human T cell line; 12. HeLa: human cervical carcinoma cell line; 13. EL4: mouse T cell line. mRNA was electrophoresed through a formaldehyde-containing 1.3% agarose gel and transfered to a nitrocellulose filter by standard techniques (Maniatis, supra.). Following prehybridization, the filter was hybridized at high stringency with radio-labelled OCT-2 probe (above). The filter was washed in 0.2×SSC, 0.1% SDS at 68° C. and autoradiographed with an intensifying screen at −70° C. for 24 hrs. The filter was stripped by washing in 50% formamide, 10 mM Tris (pH 7.4), 1 mM EDTA at 68° C. for 1 hr. and rehybridized with a radiolabelled rat alpha tubulin cDNA probe (Lemischka, I. R., Farmer, S., Racaniello, V. R. and Sharp, P. A., <i>J. Mol. Biol. </i>151:101-120 (1981)) to control for the amount of mRNA loaded.</p>
    <p>All five B lymphoma cell lines, including pre-B and mature B cell lines, and one of three T lymphoma cell lines expressed a family of 6 transcripts. Of the five non-lymphoid cell lines tested, only a glioma cell line, U1242( ), showed detectable expression of this gene. Even at low stringency we were unable to detect a transcript present in all cell lines which might correspond to NF-A1. The various transcripts, estimated to be 7.2 kb, 5.8 kb, 5.4 kb, 3.7 kb, 3.1 kb and 1.2 kb long, were expressed in somewhat varying amounts relative to each other in the positive cell lines. Whether these transcripts represent alternative mRNA splicing or highly specific mRNA degradation remains to be determined. In this regard, it is interesting that highly purified preparations of NF-A2 consist of three or more major polypeptides with distinct molecular weights which could be the products of the family of transcripts that we have observed.</p>
    <p>Previously, we and others (See above and Gerster, T. et al. <i>EMBO J. </i>6:1323-1330 (1987); Landolfi et al., <i>Nature </i>323:548-51 (1986)) showed that the octamer binding protein NF-A2 varied considerably in expression among lymphoid cell lines. We therefore investigated the relationship between levels of expression of the OCT-2 gene and levels of NF-A2 as judged by mobility shift analysis. BJAB, the cell line which expressed the largest amount of transcript showed the largest amount of NF-A2. Nuclear extracts from the pre-B cell lines, 38B9 and 70Z, showed very little NF-A2 and, correspondingly, expressed very little transcript (more poly(A)-containing mRNA from these two cell lines was loaded to see a readily detectable signal). Of the three T lymphoma cell lines tested, Jurkat, HUT78 and EL4, EL4 was the only line that showed large amounts of NF-A2. Although NF-A2 was previously believed to be expressed only in lymphoid cells we found that nuclear extracts from the glioma cell line that expressed the OCT-2 gene contained an octamer binding protein which comigrated with NF-A2 in the mobility shift assay. Nuclear extracts from two glioma cell lines which were negative for OCT-2 expression did not contain NF-A2. We have at present no explanation for this apparent non-lymphoid expression of NF-A2 and the cloned octamer binding protein gene. Previously, we had shown that NF-A2 but not NF-A1 was inducible in pre-B cells by treatment of the cells with bacterial lipopolysaccharide (LPS) and that this induction required new protein synthesis. Therefore, we prepared poly(A)-containing mRNA from the pre-B cell line 70Z/3 before and after LPS treatment and observed that LPS increased the expression of the OCT-2 gene. Thus, in every instance, the expression of the OCT-2 gene correlated with the presence of NF-A2 and is thus a good candidate for the gene which encodes NF-A2.</p>
    <p>Further evidence that the oct-2 gene encodes NF-A2 was discovered when the NF-A2 factor was purified from nuclear extracts of BJAB cells by conventional chromatography followed by multiple passages over an affinity column containing immobilized oligomers of the octanucleotide sequence. The purified NF-A2 consisted of three bands, as resolved by gel electrophoresis: a major band and two minor bands with deduced molecular weights of 61 kD and 58 kD, and 63 kD, respectively. A cDNA (pass-3) for the oct-2 gene was inserted into the polylinker of the pGEM (Promega) expression vector. Translation of RNA transcribed from the SP6 promoter-pass-3 cDNA construct yielded a major polypeptide from the purified sample of NF-A2.</p>
    <p>The nobility of a DNA-protein complex in the gel assay is primarily determined by the molecular weight of the protein. Complexes were generated with the affinity purified NF-A2 and the products of translational in vitro of RNA from the oct-2 cDNA. These complexes co-migrated during electrophoresis in a native gel, again suggesting that the oct-2 cDNA encodes the major form of the NF-A2 factor.</p>
    <p>The affinity purified NF-A2 protein and the polypeptide translated in vitro from the oct-2 cDNA were also compared by partial tryptic digestion. Samples from different digestion times of NF-A2 were resolved by denaturing gel electrophoresis and detected by staining with silver. The mobility of these partial fragments was compared with those observed after a parallel analysis of <sup>35</sup>S-methionine labelled polypeptide from transcription/translation of the pass-3 cDNA in vitro. The two samples generated a similar set of digestion fragments, again suggesting that NF-A2 is encoded by the oct-2 cDNA.</p>
    <p>Protein sequence comparisons suggested that the DNA binding domain of oct-2 was specified by a domain (positions 952-1135) that was distantly related to both the helix-turn-helix structure of bacterial repressors and the homeobox-proteins. To directly test this analogy a fragment of the cDNA encompassing this region (655 to 1710) was inserted into the expression vector pBS-ATG so that RNA could be transcribed from the truncated templates by bacteriophage T7 RNA polymerase as indicated in FIG. <b>19</b>A. The polypeptides translated in vitro from these RNAs were tested for specific DNA binding by addition of the total translation mix to the DNA-protein gel assay. Polypeptides produced from RNAs terminating at positions 1710 (Kpnl), 1443 (Stul), and 1134 (Pstl) specifically bound the octanucleotide containing probe, while the polypeptide translated from RNA terminating at the 945 (Eagl) site did not specifically bind. The region containing the helix-turn-helix portion of oct-2 is deleted in the latter protein. Since the truncated polypeptide encoded by RNA from the latter template was efficiently translated in the reticulocyte reaction, this suggests that the specific binding of the oct-2-protein requires the helix-turn-helix structure.</p>
    <p>Two distinct but similarly migrating protein-DNA complexes were detected in the sample generated by translation of RNA from the Stul cleaved template. Faint slower migrating complex comigrated with the complex generated with templates cleaved by Kpnl. The presence of the two complexes in the Stul-sample is due to a partial digestion of the plasmid DNA. The slower migrating complex is probably produced by protein terminated at the stop codon TAA located at position 1465. The faster migrating complex probably results from molecules terminated at the Stul site. This interpretation was supported by the resolution of two <sup>35</sup>S-labeled polypeptides during gel electrophoresis of the Stul sample and confirms the position of the termination codon of oct-2.</p>
    <p>Many sequence-specific binding proteins have an oligomeric structure. For example, bacterial repressor proteins typically bind sites with two-fold rotational symmetry by forming a similarly symmetric dimer (Ptashne, Cell Press and Blackwell Scientific Publications, (1986)). It should be noted that the binding site sequence of the oct-2 protein is not symmetric but oligomeric proteins could bind to non-symmetric sites. Other examples of oligomerization of sequence-specific bindingprotiens are the GCN4 protein of yeast (Hope and Struhl, EMBO J. 6: 2781-2784, (1987)) and the C/EBP protein of mammals. In the latter case, an α-helical region with an amphipathic character reflected in the spacing of four leucine residues by exactly seven residues is thought to be responsible for dimer formation (Landschultz et al., Science 240: 1759-1764 (1988); Landschultz et al., Genes &amp; Development 2: 786-800 (1988)). A convenient assay for detection of dimerization of sequence-specific bindignproteins is to co-translate RNAs encoding two different size forms of the protein and test whether protein-DNA complexes with novel mobilities are generated (Hope and Struhl, <i>EMBO J. </i>6: 2781-2784 (1987). If only monomers bind to the probe, the sample containing the co-translated polypeptides will generated only the complexes detected when either RNA is assayed singularly. This was the case with combinations of different length RNAs transcribed from the oct-2 cDNA segment. Specifically, cotranslation of RNAs from templates cleaved at Stul (1443) and Pstl (1134) did not generate novel bands in the gel mobility assay. Thus, on the basis of this negative evidence, we suggest that a single molecule of the oct-2 protein is present in the resolved DNA-protein complexes and that it does not require dimerization for binding to DNA.</p>
    <p>Anti-sera raised in rabbits against a bacterial fusion protein containing oct-2 encoded sequence (prepared employing the vestor pRIT2T (Pharmacia)) recognized the native oct-2 protein in metabolically labeled (<sup>35</sup>S-methionine) human B cells.</p>
    <p>The molecular cloning of a lymphoid-restricted octamer binding protein gene demonstrates that higher eukaryotes have adopted a strategy of genetic diversification of transcriptional regulatory proteins which bind a common regulatory motif. The ubiquitous and lymphoid-specific octamer binding proteins have indistinguishable DNA binding sites, yet appear to have distinct functional properties (Staudt, L. M. et al, <i>Nature </i>323:640-643 (1986)). Structure-function analysis of cloned yeast transcription factors (Petkovich, M. et al., <i>Cell </i>330:444-450 (1987); Giguere, V. et al., <i>Nature </i>330:625-629 (1987)) and steroid receptor related transcription regulatory activity of a transcription factor often reside in discrete protein domains that can be experimentally interchanged. The present findings suggest that similar diversification of function among proteins which bind the octamer motif has occurred during evolution. The octamer motif has been shown to be necessary and sufficient for lymphoid-specific promoter activity (Fletcher, C. et al., <i>Cell </i>51:773-781 (1987)) and NF-A2 has been shown to function as a transcription factor using octamer containing templates in vitro (Scheidereit, C. et al., <i>Cell </i>51:783-793 (1987)). A further understanding of the lymphoid-specific activity of immunoglobulin promoters may now come from an understanding of the mechanisms underlying the lymphoid-specific expression of the OCT-2 gene.</p>
    <heading>Example 8</heading> <heading>Induction of NF-KB in Cells in which it is not Constitutively Present</heading> <p>The following work demonstrates that NF-KB is inducible in cells other than B (lymphoid) cells. As described below, it has now been shown that NF-KB is inducible in pre-B cells and in non-lymphoid cells. In particular, the following work demonstrates that: 1) NF-kB factor can be induced by the mitogen lipopolysaccharide (LPS) in two cell lines representing a pre-B stage of B cell differentiation; 2) induction of this factor involves a post-translational modification of a pre-existing protein because the induction takes place even in the presence of translational inhibitors like cycloheximide and anisomycin; 3) these translational inhibitors by themselves can at least partially induce NF-kB and synergize with LPS to produce a superinduction; 4) an active phorbol ester like PMA can induce NF-kB by itself, and the time-course of this activation is more rapid than that with LPS alone; and 5) it is also possible to induce this factor in cell lines other than those having a pre-B phenotype by means of an appropriate stimulus (e.g., in the human T cell line, Jurkat, by PHA and/or PMA or in HeLa cells with PMA). Thus, B cells and plasma cells appear to support constitutive presence of this factor whereas in other cell types it can be induced transiently by an appropriate stimulus.</p>
    <p>Experimental Procedures</p>
    <p>Cell lines and Extracts: 70Z/3 and PD cells were grown in RPMI 1640 medium supplemented with 10% inactivated fetal calf serum, 50 μM β-mercaptoethanol and penicillin and streptomycin sulfate (pen-strep) antibiotics. LPS (GIBCO) stimulation was carried out with 10-15 μg/ml. For experiments using protein synthesis inhibitors and LPS, cell cultures were treated with inhibitors approximately 20 min prior to addition of LPS. Cycloheximide (Sigma) was used to 10 μg/mo which causes greater than 95% inhibition of protein synthesis in 70Z/3 cells (Wall, R., et al., <i>Proc. Natl. Acad. Sci. USA </i>83:295-298, (1986). Anisomycin (Sigma) was used at 10 which causes approximately 99% inhibition of protein synthesis in HeLa cells (Grollman, A. P., <i>J. Biol. Chem. </i>242:3226-3233, 1967). Phorbol ester activation of 70Z/3 cells was carried out using the active ester phorbol 12-myristate-13-acetate (PMA) or the inactive ester phorbol 12,13-didecanoate at a concentration of 25 ng/ml for the times indicated in the text. All treatments were carried out at cell densities varying between 5×10<sup>5</sup>-10<sup>6 </sup>cells/ml. Jurkat cells were grown in RPMI 1640 medium with 10% inactivated fetal calf serum and pen-strep antibiotics. Phytohemmagglutinin (PHA) treatment was done at 5 μg/ml and PMA treatment at 50 μ/ml. HeLa cells were grown in MEM medium with 5% horse serum and pen-strep antibiotics. Phorbol ester (PMA treatment was at 50 μg/ml with cell density varying between 7×10<sup>5</sup>-10<sup>6 </sup>cells/ml.</p>
    <p>Nuclear extracts were generated essentially according to the protocol of Dignam, J. D. et al., <i>Nucl. Acids Res. </i>11:1475-1489 (1983) and protein concentration were determined using a Bradford assay with serum albumin standards.</p>
    <p>Gel Binding Analysis: Gel binding analyses were carried out as described earlier using a radioactive DdeI to HaeIII fragment (k3) derived from the enhancer (Sen and Baltimore, 1986). Levels of NF-KB induced by various stimuli were normalized to total protein present in the extracts. Further, analysis with a different fragment that contains a binding site for the ubiquitous factor NF-A, shows that this nuclear protein remains at approximately constant levels in all of the extracts reported here. Thus, the modulation of NF-KB activity is not a reflection of variability of nuclear factors in general under these conditions. For competition experiments, the specific and non-specific competitors DNA's were included in the mixture (in amounts shown in FIG. 24C) prior to addition of the protein. The competitor fragments μ300, μ400, KE and SV40E which have been described earlier (Sen and Balitmore, 1986) were isolated from low melting point agarose gels and quantitated by spotting onto ethidium bromide-containing agarose plates.</p>
    <p>NF-KB can be Induced in Pre-B Cell Lines with Bacterial Lipopolysaccharide</p>
    <p>To examine whether NF-KB might be inducible in 70Z/3 cells, cells were stimulated with LPS for 20 hr and nuclear extracts derived from these cells were assayed for the presence of NF-KB using the electrophoretic mobility shift assay described previously. Singh, H. et al., <i>Nature, </i>319:154-158 (1986). U.S. patent application Ser. No. 817,441, To assay for NF-KB, a DNA fragment containing its binding site (K3 fragment; Sen and Baltimore, supra,) was end-labelled and incubated with extracts derived from either unstimulated 70Z/3 (11 s (FIG. 21A, lanes 2,3) or LPS-stimulated 70Z/3 cells (FIG. 21A, lanes 4,5) in the presence of increasing amounts of the carrier poly d(IC). Binding reactions were carried out for 15-30 minutes at room temperature in a final volume of 15 μl containing 9 μg of total protein 3.5 μg (lanes 2,4) or 4.5 μg (lanes 3,5) of nonspecific carrier DNA poly d(IC) and 0.2-0.5 μg of probe. Reaction products were fractionated by electrophoresis through low ionic strength polyacrylamide gels and visualized by autoradiography. Lane 1: free DNA fragments; lane 6: nucleoprotein complex generated by interaction of NF KB with the fragment 3 in a nuclear extract derived from the B cell line WEHI 231.</p>
    <p>Unstimulated 70Z/3 cell extracts lacked a major band evident with B cell extracts (FIG. 21A, lane 6; arrow). This nucleoprotein complex band was induced in the 70Z/3 cells after LPS treatment for 20 hr. The band was not competed away even with 4.5 ugm of poly d(IC) (lane 5). This induction phenomenon was not restricted to the 70Z/3 cell line; another pre-B cell line, PD (Lewis S., et al., Cell 30:807-816 (1982)), was weakly positive for the factor prior to induction (FIG. 21B, lane 3; Sen and Baltimore, 1986) but was strongly induced by LPS (FIG. 21B lanes 5,6). A number of other minor bands could be seen in the binding assay, some of which were inducible and others not. The major inducible band comigrated with the major band produced by B cell and plasma cell extracts (typified by WEHI 231 extracts in FIG. 21A, lane 6 and FIG. 21B, lane 2). We have earlier characterized this band by competition experiments and localized the binding site of the factor by methylation interference experiments defining the band as one produced by interaction of the NF-KB factor with the B site within the enhancer (a site containing the sequence GGGGACTTTCC). Thus two pre-B cell lines, one with a rearranged K gene (70Z/3) and the other in the process of undergoing rearrangement (PD), are clearly inducible by LPS for NF-KB activity.</p>
    <p>Induction of NF-KB by LPS does not Require Protein Synthesis</p>
    <p>Recently it has been reported that induction of transcription in 70Z/3 does not require new protein synthesis. Nelson, K. J. et al., <i>Proc. Nat. Acad. Sci. </i>USA 82:5305-5309 (1985). Thus, induction of gene expression was evident in cells pretreated (10 min) with the translation inhibitors cycloheximide or anisomysin followed by stimulation with LPS. Further, Wall et al. reported that expression could be induced in the presence of cycloheximide alone which led them to argue in favor of a labile repressor blocking the activation of genes in this cell line. See Wall, R. et al. <i>Proc. Nat. Acad. Sci. USA </i>83:295-298 (1986). To determine if these characteristics of transcriptional activation were paralleled by changes in the levels of NF-KB, we analyzed extracts derived from 70Z/3 cells which had been treated either with LPS alone, or with a translation inhibitor alone or with both together. To be able to make direct correlations with the published reports concerning the effects of translational inhibitors on expression in pre-B cells, we examined a 4 hr time point in these experiments, although maximal stimulation of expression by LPS takes 14-20 hr. Binding reactions were carried out as detailed in FIG. 21A legend and contained 2.5, 3.5 or 4.5 μg poly dIG) with each set of extracts. End-labelled K3 fragment was the proble lane 1) and was incubated with 9-11 μg or protein from extracts derived from: untreated 70Z/3 cells (lanes 2,3,4), 70Z/3 cells treated for 4 hr with 10 μg/ml of LPS (lanes 5,6,7), 70Z/3 cells treated for 4 hr with 10 μg/ml of LPS and 10 μg/ml cycloheximide (lanes 8,9,10); 70Z/3 cells treated with 10 μg/ml of cyclheximide alone (lanes 11,12,13) and WEHI 231 cells (lane 14). The characteristic nucleoprotein complex is indicated by the arrow. In accord with the transcriptional analyses, uninduced 70Z/3 cells were negative for NF-KB (FIG. 22A, lanes 2-4), and treatment with either LPS alone (FIG. 22A, lanes 5-7) or with cycloheximide alone (FIG. 22A, lanes 11-13) for 4 hrs induced the factor. Unexpectedly, stimulation of 70Z/3 with LPS in the presence of cycloheximide for 4 hr gave a superinduction of NF-KB (FIG. 22A, lanes 8-10), increasing it to a level above that seen after a 20 hr induction.</p>
    <p>Qualitatively, the same result was observed when anisomycin was used as a translation inhibitor (FIG. <b>22</b>B). Binding reactions were as detailed in FIG. 21A legend using 2.5 and 3.5 μg of poly d(IC) and protein from untreated 70Z/3 cells (lanes 2,3); 70Z/3 cells after induction with LPS alone (lanes 4,5); 70Z/3 cells with LPS induction in the presence of anisomycin (lanes 6,7); 70Z/3 cells treated with anisomycin by itself (lanes 8,9) and the B cell WEHI 231 as a positive control (lane 10). The characteristic nucleoprotein complex is indicated by the arrow. Thus, the presence of anisomycin (10 uM) during a 4 hr stimulation with LPS gave a superinduction of NF-KB (FIG. 22B, lanes 6,7) relative to either LPS alone (FIG. 22B, lanes 4,5) or anisomycin alone (FIG. 22B lanes 8-9). Once again, prior to LPS treatment there was no detectable NF-KB activity in 70Z/3 (FIG. 22B lanes 2,3). Although treatment of 70Z/3 with cycloheximide alone or with LPS alone gave approximately equivalent amounts of NF-KB (FIG. 22A compare lanes 5-7 with lanes 11-13), the level of NF-KB induced with anisomycin alone appeared to be much less (FIG. 22B, compare lanes 8,9 with lanes 4,5). This is probably due to drug toxicity because, even after a short exposure to anisomycin, the cells looked quite unhealthy. Presumably this also accounts for lesser degree of superinduction seen with LPS and anisomycin. Thus the enhancer binding factor NF-KB appears to be inducible in 70Z/3 cells in the absence of protein synthesis. Further, it appears to be inducible by either of 2 different translation inhibitors alone and is superinduced when the cells are stimulated with LPS and the inhibitor.</p>
    <p>Phorbol Ester can Induce NF-KB in 70Z/3</p>
    <p>The tumor promoting phorbol ester, phorbol 12-myristate-13-acetate (PMA), has been shown to induce surface immunoglobulin in 70Z/3, presumably via activation of K transcription and transport of complete immunoglobulin to the cell surface (Rosoff P. M. et al., <i>J. Biol. Chem. </i>259:7056-7060 1984; Rosoff, P. M. and Cantley, L. C., <i>J. Biol. Chem. </i>260 9209-9215, (1985). To determine if this activation is reflected in an increase of NF-KB, we analyzed extracts derived from 70Z/3 cells after a 4 hr stimulation with PMA at 50 ng/ml. Binding reactions using K3 as a problem (lane 1) were carried out as detailed in FIG. 21A legend with protein from untreated 70Z/3 cells (lane 2) or 70Z/3 cells that had been treated with PMA at 50 ng/ml for 4 hr (lanes 3,4). Lane 5 is the positive control for NF-KB in extracts from WEHI 231. There was a striking induction of NF-KB activity in these extracts (FIG. 23A, compare lanes 3,4 with lane 2). Thus an active phorbol ester by itself is capable of inducing NF-B activity in 70Z/3 cells, implicating protein kinase C as a possible intermediate in the post-translational modification reaction that produces NF-B in these cells [(Bell, R. M. <i>Cell </i>45:631-632 (1986); Nishizuka, Y. <i>Nature </i>308:693-697, (1984)]. An inactive phorbol ester (phorbol 12, 13 didecanoate) did not cause induction of NF-KB under similar conditions (data not shown).</p>
    <p>Time Course of Activation of NF-B by LPS and PMA are Different</p>
    <p>LPS-mediated stimulation of surface Ig expression of mRNA accumulation reaches a maximum after at least one cell cycle, i.e., in 14-18 hr. Recent work has shown that LPS stimulation of RNA synthesis, as measured by nuclear run on assays [Nelson, K. J., et al., <i>Proc. Natl. Acad. Sci USA </i>82:5305-5309 (1985); Wall et al. supra, (1986)] can be seen as early as 4 hr after stimulation and that the DNAse I hypersensitive site associated with the enhancer can be detected as early as 1 hr post-stimulation. To examine the time-course of NF-KB induction, we generated 70Z/3 cell extracts after stimulation either by LPS or PMA for varying lengths of time. Analysis for NF-KB activity using the binding assay showed that the time course of activation of NF-KB by these two agents was quite different (FIG. <b>23</b>B). Binding reactions were carried out with extracts derived from 70Z/3 cells that had been treated with LPS at 10 μg/ml (lanes 3-7) or PMA at 25 ng/ml (lanes 8-12) for various lengths of time as shown above each lane in the figure. Lane 2 is a positive control for NF-KB in WEHI 231 extracts. With LPS alone, a nucleoprotein complex band reflecting the presence of NF-KB increased until 2 hr post-stimulation after which a slight decrease occurred and then the level remained constant. By contrast, in PMA-stimulated cells, NF-KB was detected at maximal levels within 0.5 hr after stimulation, remained at this level for 2-3 hours and then began to drop off rapidly, such that by 8 hr it was barely detectable. Because prolonged exposure of cells to phorbol esters is known to result in desensitization of endogenous protein kinase C (Rodriquez-Pena, A. and Rozengurt, E., <i>Biochem Biophys. Res. Comm. </i>120:1053-1009, 1984; <i>EMBO J, </i>5:77-83 1986), a possible explanation for the rapid decline of NF-KB may be that its maintenance as a binding factor requires continuous activity of protein kinase C. A similar phenomenon has been described recently by Blemis and Erikson where S6 kinase activity assayed by phosphorylation of S6 protein) first rises and then falls during prolonged exposure to PMA. See Blemis, J. and Erikson, R. L. <i>Proc. Natl. Acad. Sci. USA </i>83:1733-1737 (1986). Although it has been reported that LPS may directly activate protein kinase C (Wightman, P. D. and Raetz, C. R. H., <i>J. Biol. Chem. </i>259:10048-10052, 1984) the different kinetics of induction of NF-KB by LPS and PMA implies that these activators feed into a common pathway through distinguishable sites of activation.</p>
    <p>Non Pre-B Cell Lines can also Be Activated to Produce NF-KB</p>
    <p>In our previous analysis we have shown that NF-KB is present only in cell lines representing the B cell or plasma cell stages of B lymphoid differentiation, but was undetectable in a variety of non B cells, pre-B cells and T cells (Sen and Baltimore, 1986). However, as shown above, this factor may be induced to high levels in pre-B cells upon stimulation with LPS. To check if this inducibility was restricted to cells having a pre-B pheno-type only or was a general characteristic of the other constitutively negative cell lines we have taken representative examples of cell types (T cells and non lymphoid cells) and examined then for induction of NF-KB after appropriate stimulation.</p>
    <p>The human T leukemia cell line, Jurkat, can be stimulated to produce interleukin-2 (IL-2) by the combined influence of phytohemagglutinin (PHA) and phorbol ester (PMA) (Gillis, S. and Watson, J., <i>J. Exp. Med., </i>152:1709-1719, 1980; Weiss et al., <i>J. Immunol. </i>133:123-128, 1984). Nuclear extracts were prepared from Jurkat cells that had been stimulated with either PHA alone or PMA alone or both together and analyzed for the presence of NF-KB (FIG. <b>24</b>A). The human T lymphoma Jurkat was stimulated with phytohemagglutinin (PHA) and phorbol 12-myristate-13-acetate (PMA) individually or together for 20 hr. Nuclear extracts made after treatment were analyzed by the mobility shift assay using K-3 fragment as the labelled probe. Binding reactions typically contained 6 g of protein, 2.5-3.5 g of poly d(IC) and 0.3-0.5 ng of end-labelled DNA probe. Lane 1: no protein added; lane 2: WEHI 231 extract (positive control); lane 3: extract from uninduced Jurkat cells: lane 4: Jurkat cells stimulated with PHA alone; lane 5: Jurkat cells stimulated with pHA and PMA; Lane 6: Jurkat cells stimulated with PMA alone. The arrow shows the position of the expected nucleoprotein complex generated by interaction of NF-KB with K-3 fragment. As originally observed, extracts derived from uninduced Jurkat cells were negative for NF-KB activity (FIG. 24A, lane 3). However, extracts made from Jurkat cells which had been stimulated either with PHA or PMA contained detectable levels of NF-KB (FIG. 24A, lanes 4,6) and the extracts from the co-stimulated cells showed higher levels of the factor (FIG. 24A, lane 5). Thus a factor with the properties of NF-KB can be induced in a T cell lines after appropriate activation.</p>
    <p>As an example of a non-lymphoid line we used the human HeLa cell lines which is constitutively negative for NF-KB (Sen and Baltimore, 1986). These cells were induced with PMA for 2 hr and extracts derived from treated and untreated cells were analyzed for NF-KB activity (FIG. <b>24</b>B). HeLa cells were treated with PMA (50 ng/ml) for 2 hrs. and the extracts derived thereafter were analyzed for induction of NF-KB. Binding reactions contained 15-18 μg of protein, 3.5 μg of poly d(IC) and 0.3-0.5 μg of end-labelled DNA probe. Lane 1: 3 fragment/no protein added; lane 2: 3 fragment incubated with extracts derived from the human B lymphoma EW; lane 3: K3 fragment incubated with uninduced HeLa cell nuclear extract; lane 4: p50 fragment (derived from the μ-heavy chain enhancer and containing a copy of the conserved octamer sequence ATTTGCAT) incubated with uninduced HeLa cell extracts; lane 5: K3 fragment incubated with induced HeLa cell extracts. The untreated HeLa extract (FIG. 24B, lane 3) did not show a nucleoprotein complex which comigrated with the complex generated in B cell extracts. However treatment with PMA induced a factor that generated the characteristic DNA-protein complex produced by NF-KB (FIG. 24B, lane 5). As a control, both the uninduced and induced extracts showed equivalent levels of the ubiquitous NF-A1 DNA binding protein when analyzed using a probe containing the sequence ATTTGCAT (FIG. 24B, lanes 4,6; Singh, H. et al., <i>Nature </i>319:154-158, 1986). Therefore treatment of HeLa cells with PMA induces a factor that can form a nucleoprotein complex with the K3 fragment.</p>
    <p>To further characterize the DNA-protein complex formed in the PMA-treated HeLa cell extracts, we carried out competition experiments. Binding reactions were carried out using end-labelled K3 fragment, 3.5 μg of poly d(IC) and 15-18 μg of nuclear extract in the present of 50 ng of unlabelled competitor DNA derived from various immunoglobulin and viral regulatory sequences (lanes 5-9). The complex generated in PMA-induced HeLa cell extracts (FIG. 24C, lane 4) was specifically competed away by the inclusion of 50 ng of unlabelled DNA in the binding reaction containing either the enhanced (FIG. 24C, lane 7) or the SV40 enhanced (FIG. 24<i>c, </i>lane 8) but was unaffected by two DNA fragments that together span the K enhancer (FIG. 24C, lane 5,6), or by a 250 bp fragment containing the K promoter (FIG. 24C, lane 9). This pattern of competition exactly parallels the pattern observed earlier using the K3 fragment in binding experiments with B cell derived extracts (Sen and Baltimore, 1986). These results further strengthen the conclusion that the NF-KB factor can be induced in non-lymphoid cells as well as lymphoid cells following appropriate stimulation.</p>
    <heading>Example 9</heading> <heading>Characterization of the NF-kB Protein and Electrophoretic Mobility Shift Analysis of Subcellular Fractions of 70Z/3 Cells</heading> <p>Characterization of the NF-kB protein</p>
    <p>Mouse NF-kB is a polypeptide with a molecular weight around 60 kDa. This has been determined by a DNA-binding renaturation experiment using eluates from different molecular weight fractions of a reducing SDS-gel (FIG. <b>26</b>A). The size of the native NF-kB protein was determined in the following manner: Nuclear extract from TPA-stimulated cells was subjected to ultracentrifugation on a continuous glycerol gradient. The fractions were assayed for DNA-binding activity of NF-kB by electrophoretic mobility shift assays (FIG. <b>26</b>B). NF-kB activity was found highest between the co-sedimented bovine serum albumin (67 kDa) and IgG (158 kDa) standards (FIG. 26B, lanes 6 to 8). The specificity of binding was shown by the absence of a complex when a DNA probe with a mutation in the NF-kB binding sequence was used to assay the fractions (FIG. 26B, right lanes 4 to 10). Lenardo, M. et al., <i>Science, </i>236:1573-1577 (1987). Little NF-KB activity was contained in the fractions where a 60 kDa protein would be expected to sediment (FIG. 26B, lanes 4 and 5). If the sedimentation of NF-kB is not highly abnormal, the results from the glycerol gradient centrifugation suggest that NF-kB is associated with another protein of approximately the same size. Presumably NF-kB forms a homodimer because the protein-DNA complex formed in native gels using whole nuclear extract is of the same mobility as the complex formed with renatured NF-kB protein from a single spot of a two-dimensional gel.</p>
    <p>70Z/3 cell cultures were incubated in the absence (Co) and presence of phorbol ester (TPA), followed by subcellular fractionation of cells. In the DNA-binding reactions, 8.8 ug of protein of nuclear extracts (N), cytosolic fractions (C), and post-nuclear membrane fractions (P) in 4 ul buffer D(+) were used. The end-labeled DNA-fragments were incubated in the presence of 3.2 ug poly(d[I-C]) with the subcellular fractions in a final volume of 20 ul for 15 to 30 minutes followed by separation of protein-DNA complexes and unbound DNA on native 4% polyacrylamide gels. Fluorograms of native gels are shown. To detect kB-specific DNA-binding activity a DdeI-HaeIII wild type fragment of the kappa light chain enhancer (kB wt; lanes 1-6) was used. Sen, R. and D. Baltimore, <i>Cell, </i>46:705-716 (1986). kB-unspecific activities binding to the kappa enhancer fragment were detected using a fragment mutated in NF-kB binding site that was otherwise identical to the wild type fragment (lanes 7-12). Lenardo, M. et al., <i>Science, </i>236:1573-1577 (1987). NF-uE3-binding activity and octamer binding protein activity were assayed with a HaeIII-DdeI kappa enhancer fragment (uE3; lanes 13-18) and a PvuII-EcoRI kappa heavy chain promoter fragment (OCTA; lanes 19-24), respectively. Sen, R. and D. Baltimore, <i>Cell, </i>46:705-716 (1986); Singh, H. et al., <i>Nature, </i>319:154-158 (1986,). Specific protein-DNA complexes are indicated by filled arrowheads and the positions of unbound DNA-fragments by open arrowheads.</p>
    <heading>Example 10</heading> <heading>Renaturation of NF-kB</heading> <p>70Z/3 cells were grown in spinner cultures with RPMI 1640 medium supplemented with 10% newborn calf serum and 50 uM 2-mercaptoethanol. HeLa cells were also grown in spinner cultures with MEM medium supplemented with 10% horse serum. Cell cultures were treated with 25 ng/ml 12-O-tetradecanoylphorbol 13-acetate (TPA; Sigma) for 30 minutes at cell densities between 7×10<sup>5 </sup>and 2×10<sup>6</sup>/ml.</p>
    <p>Subcellular Fractionation</p>
    <p>Cells were collected by centrifugation for 10 minutes at 150×g. Cell pellets were resuspended in ice-cold phosphate-buffered saline and collected again by centrifugation. All following steps were carried out at 4° C.. Washed cells were resuspended in four packed cell volumes of a hypotonic lysis buffer (buffer A; Dignam, J. P. et al., <i>Nucleic Acid Research, </i>11:1475-1489 (1983)). After 20 minutes, cells were homogenized by 15 (HeLa) or 20 strokes (70Z/3 cells) with a loose fitting Dounce homogenizer. Nuclei were collected by centrifugation for 6 minutes at 4300×g, resuspended in five volumes of buffer A and washed once by centrifugation. Proteins were extracted from washed nuclei by high salt, followed by centrifugation of the nuclear extracts and dialysis against buffer D as described. Dignam, J. P. et al., <i>Nucleic Acids Research, </i>11:1475-1489 (1983). One percent NP-40 (v/v) was added to the dialyzed nuclear extracts. The postnuclear supernatant was centrifuged for 6 minutes at 4300×g and the resulting supernatant ultracentrifuged for 1 hour at 150,000×g. The pellet after ultracentrifugation containing postnuclear membranes was dissolved in buffer D containing 1% (v/v) NP-40 (referred to as buffer D(+)). Insoluble material was removed by centrifugation for 10 minutes in a Microfuge. The supernatant after ultracentrifugation (referred to as cytosolic fraction) was adjusted to buffer D(+) conditions by the addition of stock solutions. Fractions were stored at −70° C.</p>
    <p>Protein concentrations were determined by an assay using bicinchoninic acid. Smith, P. K. et al., <i>Anals of Biochemistry, </i>150:76-85 (1985). The ratio of the total protein recovered during a fractionation experiment in nuclear extracts, cytosolic fractions and postnuclear membrane fractions was 2:4:1 for 70Z/3 cells and 1:10:1.5 for HeLa cells. These ratios were used to adjust the fractions to protein concentrations reflecting equal cell-equivalents of subcellular fractions.</p>
    <p>Electrophoretic Mobility Shift Assays and Treatments with Dissociating Agents</p>
    <p>DNA-binding reactions were carried out as described above. The DNA-binding reaction mixture contained poly(d[I-C]) (Pharmacia), 3000-6000 cpm of [<sup>32</sup>P]end-labeled DNA-fragments and a buffer composed of 10 mM Tris-HCl, pH 7.5, 50 mM NaCl, 1 mM dithiothreitol (DTT), 1 mM EDTA and 5% glycerol. Binding reactions and subsequent analysis on native 4% polyacrylamide gels were performed at room temperature as described. Sen, R. and D. Baltimore, <i>Cell, </i>46:705-716 (1986). Subcellular fractions were treated with formamide (DNA grade; American Bioanalytical) prior to the addition of the DNA-binding reaction mixture. Sodium desoxycholate (Fisher Scientific Company) was added after the DNA-binding reaction.</p>
    <p>Renaturation of NF-kB</p>
    <p>Protein in the subcellular fractions was precipitated at −20° C. by the addition of four volumes of acetone. Pellets were dissolved in SDS-sample buffer containing 3.3% 2-mercaptoethanol and boiled for 5 minutes. Laemmli, U. K., <i>Nature., </i>227:680-685 (1970). After SDS-polyacrylamide gel electrophoresis, gel pieces from different molecular weight regions were cut out, ground, and proteins eluted overnight at 4 ° C. in 500 ul of a buffer containing 50 mM Tris-HCl, pH 7.9, 0.1% SDS, 0.1 mg/ml bovine serum albumin, 1 mM DTT, 0.2 mM EDTA, 0.1 mM phenylmethylsulfonyl fluoride (PMSF) and 2.5% glycerol. After centrifugation for 2 minutes in a Microfuge, the supernatant was removed and recentrifuged for 10 minutes to remove gel debris.</p>
    <p>To 200 ul of the supernatant four volumes of acetone were added and proteins were allowed to precipitate for 2 hours at −20° C. The precipitate was collected by centrifugation for 10 minutes in a Microfuge, washed once with 1 ml methanol at −20° C. and dried for 30 minutes in the inverted tube. The dried pellet was dissolved in 2.5 ul of a saturated solution of urea (ultrapure; American Bioanalytical) and dilued with 125 ul of a buffer containing 20 mM Tris-HCl, pH 7.6, 10 mM KCl, 2 mM DTT and 10 uM PMSF. Renaturation was allowed for a minimum of 18 hours at 4° C. kB-specific DNA-binding activity was detectable in mobility shift assays for at least 48 hours after storage of renatured fractions at 4° C. without appreciable loss of activity.</p>
    <heading>Example 11</heading> <heading>Subcellular Localization of the NF-kB Precursor</heading> <p>Because NF-kB is a DNA-binding protein, it is expected to reside in the nucleus. This is certainly true for NF-kB of phorbol ester-treated cells and mature B-cells where the activity is detectable in nuclear extracts. It is however not mandatory for a precursor of NF-kB especially, in view of the fact that the precursor is activated by protein kinase C, a cytosolic protein that is associated in its active state with the plasma membrane.</p>
    <p>The precursor of NF-kB was analyzed by an investigation of subcellular fractions of unstimulated pre-B cells for the presence of NF-kB activity, using electrophoretic mobility shift assays (FIG. <b>27</b>A). Little DNA-binding activity was detected in the subcellular fractions, indicating that the precursor must exist in a form of low affinity for its cognate DNA (FIG. 27A, lanes 1 to 3). In fractions from TPA-stimulated cells, the newly activated NF-kB was almost exclusively contained in the nuclear extract (FIG. 27A, lanes 4 to 6).</p>
    <p>In an attempt to activate the DNA-binding activity of the NF-kB precursor, the various subcellular fractions were treated with agents known to gently dissociate protein-protein interactions. Low concentrations of desoxycholate, formamide or a combination of both included in the mobility shift assay mixture led to the activation of an NF-kB-specific DNA-binding activity (FIG. <b>27</b>B). The fraction containing the bulk of the in vitro activatable NF-kB precursor was the cytosol (FIG. 27B, lanes 1 to 3). When fractions from TPA-stimulated cells were subjected to the same treatment, the amount of precursor in the cytosolic fraction was found strongly reduced, apparently because of redistribution of activated NF-kB into the nuclear extract fraction (FIG. 27B, lanes 4 and 5). In both control and TPA-stimulated cells, the amount of total cellular NF-kB activity revealed after treatment with dissociating agents was equal, suggesting a complete conversion of the NF-kB precursor into active NF-kB. Cytosolic fractions from HeLa cells and from calf spleen also contained NF-kB precursor which could be demonstrated after activation with dissociating agents. These observations strongly suggest that NF-kB is localized as an inactive precursor in the cytosol. Activation of protein kinase C by phorbol ester then would result in two events: induction of DNA-binding activity and nuclear translocation of NF-kB.</p>
    <p>Using subcellular fractions from HeLa cells, another TPA-inducible transcription factor, AP-1, was tested to determine whether it also exhibits activation and sub-cellular redistribution upon TPA-stimulation. As detected by mobility shift assays, AP-1 from nuclear extracts did not show an increase in DNA-binding activity after TPA-stimulation nor were there significant amounts of AP-1 activity present in the cytosolic fractions from control and stimulated cells (FIG. <b>28</b>). This showed that the mechanism by which the transcription factor activity of NF-kB is induced is fundamentally different from that of AP-1 although the initial signal—activation of protein kinase C by phorbol ester—is the same.</p>
    <heading>Example 12</heading> <heading>Investigation of the DOC-dependence of Cytosolic NF-kB</heading> <p>Cytosol from unstimulated 70Z/3 pre-B cells in buffer A (Dignam, J. P. et al., <i>Nucl. Acids. Res., </i>11:1475 (1983); Baeuerle, P. A. and D. Baltimore, <i>Cell, </i>53:211 (1988)) was adjusted to a final concentration of 50 mM NaCl, 20 mM (HEPES) (pH 7.9), 1.5 mM EDTA, 5% glycerol and 0.2% NP-40. Cytosolic protein (45 mg) was mixed to a final volume of 4 ml with 0.6% DOC, 0.75 g calf thymus (wet weight) DNA-cellulose [Sigma; equilibrated in buffer G: 10 mM Tris-HCl (pH 7.5), 5 mM NaCl, 1 mM EDTA, 1 mM dithiothreitol (DTT), 5% glycerol, 0.2% DOC, 0.2% NP-40, and 0.5 mM phenylmethyl sulfonylfluoride (PMSF)] and 1.2% NP-40. The suspension was incubated in a mini column for 1 hour at room temperature on a rotary shaker. The flow-through fraction was used for gel filtration. DNA-cellulose was washed with buffer G and eluted with a NaCl step gradient in buffer G. Equal proportions of fractions were assayed by EMSA (Sen, R. and D. Baltimore, <i>Cell, </i>46:705 (1986); Baeuerle, P. A. and D. Baltimore, <i>Cell, </i>53:211 (1988)) at a final concentration of 1.2% NP-40 in the presence of either 0.03% DOC (non dissociating condition) or 0.6% DOC (dissociating condition) and with 10 μg of bovine serum albumin (BSA) as carrier. Results of this investigation are represented in FIG. <b>34</b> and described above.</p>
    <heading>Example 13</heading> <heading>Characterization of IkB and its Complex with NF-kB</heading> <p>The flow-through fraction from the DNA-cellulose column (1.55 mg of protein in 250 μl described in Example 4) was subjected to a G-200 Sephadex column (280 by 7 mm) with a flow rate of 0.15 ml/min in buffer G at room temperature. A mix of size markers (dextran blue; immunoglobulin G, 158 kDa, BSA, 67 kDa, ovalbumin, 45 kDa, myoglobin, 17 kDa; Biorad) was run separately on the column prior to sample runs. Markers were detected in fractions by their color and using SDS-polyacrylamide gel electrophoresis (SDS-PAGE) followed by Coomassie Blue staining.</p>
    <p>To detect inhibiting activity, portions of fractions (5 μl; in buffer G) were mixed with 1 μl of nuclear extracts [in buffer D(+)] and 0.5 μl 10% NP-40. Dignam, J. P. et al., <i>Nucl. Acids Res., </i>11:1475 (1983). After 30 minutes at room temperature, the reaction volume was brought to 20 μl by the addition of a DNA binding reaction mixture containing 3.2 μg of poly(dI-dC) (Pharmacia), 5 to 20 fmoles of <sup>32</sup>P-end labeled k enhancer fragment, 75 mM NaCl, 15 mM Tris-HCl (pH 7.5), 1.5 mM EDTA, 1.5 mM DTT, 7.5% glycerol, 0.3% NP-40 and 20 μg BSA. After a 20-minute DNA binding reaction, samples were analyzed by EMSA.</p>
    <p>Gel filtration fractions containing IkB (25 μg of protein) were incubated for 1 hour at room temperature in buffer G without any addition or with 2 μg of TPCK-treated trypsin (Sigma), 8 μg of BPTI (Sigma), or with 2 μg of trypsin that had been incubated with 8 μg of BPTI. Tryptic digestion was stopped by a 10-minute incubation with 8 μg of BPTI and samples analyzed as described above.</p>
    <p>Nuclear extract from TPA-stimulated 70Z/3 cells and cytosol from untreated cells (both 220 μg of protein) were sedimented through 5 ml of a continuous 10 to 30% glycerol gradient in buffer D(+) and 150,000 g (SW 50.1 rotor; Beckman) for 20 hours at 4° C. Cosedimented size markers were detected in fractions by SDS-PAGE and Coomassie Blue staining. Portions of glycerol gradient fractions (4 μl) were analyzed by EMSA with 10 μg of BSA as carrier and 0.5 μg of poly(dI-dC). NF-kB precursor was activated by treating 4 μl of fractions with 1.5 μl of formamide before the DNA binding reaction mixture was added.</p>
    <heading>Example 14</heading> <heading>Demonstration of the Presence of the NF-kB—IkB Complex in Enucleated Cells</heading> <p>HeLa cells were grown in Eagle's Minimum Essential Medium supplemented with 10% horse serum, penicillin (50 I.U./ml) and streptomycin (50 μg/ml) (referred to as MEM-medium) on discs (1.8 cm in diameter) cut from cell culture plastic ware. For enucleation, discs were placed upside down into centrifuge tubes filled with 10 ml of MEM-medium of 37° C. containing cytochalasin B (10 μg/ml) and held for the same time in the incubator. To estimate the enucleation efficiency, enucleated cells on one disc were fixed with formaldehyde (3.7%) in phosphate-buffered saline (PBS) for 20 minutes, stained for 4 minutes with 4′,6-diamidino-2-phenylindole (DAPI, 1 μg/ml; Sigma) in PBS, and washed in PBS. Fluorescence microscopy under UV light and phase contrast microscopy were performed with a Zeiss Photomicroscope III. Control and enucleated cells were allowed to recover in cytochalasin B-free MEM-medium for 30 minutes before a 2-hour incubation in the absence or presence of TPA (50 ng/ml). Cells were then washed in ice-cold PBS, scraped off the discs in 100 μl of a buffer containing 20 mM HEPES (pH 7.9), 0.35M NaCl, 20% glycerol, 1% NP-40, 1 mM MgCl<sub>2</sub>, 1 mM DTT, 0.5 mM EDTA, 0.1 mM EGTA, 1% aprotinin (Sigma) and 1 mM PMSF. After lysis and extraction for 10 minutes on ice, particulate material was removed by centrifugation (Microfuge) for 15 minutes at 4° C. and the resulting supernatants were analyzed by EMSA.</p>
    <heading>Example 15</heading> <heading>Demonstration of the Role of NF-KB as Mediator in Regulation of a Gene in Non-Lymphoid Cells</heading> <p>The following demonstrates that NF-κB has the role of mediator in cytokine gene regulation (in this case, positive regulation of β-IFN gene expression). NF-κB has been shown to interact with a virus-inducible element (PRDII) in the β-IFN gene and to be highly induced by either virus infection or treatment of cells with double-standard RNA.</p>
    <heading>A. Experimental Procedure</heading> <p>Cell Culture and Transfection</p>
    <p>Mouse L929 fibroblasts were maintained in MEM medium (Gibco) with 5% serum (Gibco). Jurkat (human T lymphocytes), Namalwa (human Burkitt lymphoma), S194 (mouse myeloma), and 70Z/3 (mouse pre-B lymphocyte) cells were grown in RPMI 1640 medium supplemented with 10% fetal calf serum (Life Science) and 50 μM β-mercaptoethanol. Sendai virus (SPAFAS) or poly(rI:rC) (Pharmacia) inductions were either 6 hours in length for protein extracts (Zinn et al., <i>Cell, </i>34:865-879 (1983)) or 12 hours for transfections. Phorbol myristate acetate (Sigma) and phytohemagglutinin (PHA) induction was carried out as described by Sen, R. and D. Baltimore (<i>Cell, </i>47:921-928 (1986)).</p>
    <p>Transient transfections of L929 cells were performed according to Kuhl et al., <i>Cell, </i>50:1057-1069 (1987); and for S194 cells, according to Pierce et al., <i>Proc. Natl. Acad. Sci. USA, </i>85:1482-1486 (1988). A β-galactosidse (β-gal) expression plasmid (Edlund et al., <i>Science, </i>230:912-916 (1985)) was co-transfected to monitor the transfection efficiency. CAT assays were described by Gorman et al., <i>Mol. Cell. Biol., </i>2:1044-1051 (1982), and the amount of protein assayed was normalized to a constant amount of β-gal activity. An et al., <i>Mol. Cell. Biol., </i>2:1628-1632 (1982).</p>
    <p>Plasmid Constructions</p>
    <p>The plasmids p-41βCAT (-41β), p-41PII4r (-41β(P)<sub>4</sub>), and p-41PII2r (-41β(P)<sub>2</sub>) were constructed as follows: in Fan, C. M. and T. Maniatis, <i>EMBO J., </i>in press (1989). The nucleotide sequence of the PRDII×2 (PRDII<sub>2</sub>) is 5′-GATCTGTGGGAAATTCCGTGGGAAATTCCGGATC-3′. The construction of Δ56 (c-fosCAT), Δ56(B)<sub>2 </sub>(J16), and Δ56(B<sup>−</sup>) (J32) were described by Pierce et al., <i>Proc. Natl. Acad. Sci. USA, </i>85:1482-1486 (1988). The κB oligonucleotides were: Wild-type: 5′-TCGACAGAGGGGACTTTCCGAGAGGCTCGA-3′ and mutant: 5′-TCGACAGAATTCACTTTCCAGGAGGCTCGA-3′. The IRE was isolated as a BglII-BamHI fragment (Goodbourn et al., <i>Cell, </i>45:601-610 (1986)) and cloned into pSP73. Mutant PRDII sites were described in Goodbourn, S. and T. Maniatis, <i>Proc. Natl. Acad. Sci. USA, </i>85:1447-1451 (1988).</p>
    <p>Preparation of Subcellular Protein Fractions and Mobility Shift Electrophoresis</p>
    <p>Buffers A, C and D are those described by Dignam et al., <i>Nucl. Acids Res., </i>11:1475-1489 (1983). Frozen pellets containing from 1×10<sup>6 </sup>to 1×10<sup>7 </sup>cells were thawed in the presence of an equal volume of buffer A. The suspension was mixed using 10 strokes of a Dounce homogenizer and the nuclei were pelleted for 20 minutes at 4° in a microcentrifuge. The supernatant, termed cytosol, was ultracentrifuged for 1 hour at 100,000×g and adjusted to 20% glycerol, 10 mM HEPES, pH 7.9, 1 mM EDTA, and 0.1 M KCl. The nuclei were extracted with 2 volumes of the buffer C for 20 minutes and the nuclear extract was cleared by centrifugation and dialyzed against buffer D. To minimize proteolysis, all buffers included 0.5 mM PMSF, 0.3 μg/ml leupeptin, and 0.3 μg/ml antipain and buffers A and C included 0.3 TIU/ml aprotinin, 0.5 mg/ml benzamidine, 0.1 μg/ml chymostatin, and 0.7 μg/ml pepstatin.</p>
    <p>Binding assays were carried out as described in Lenardo et al., <i>Science, </i>236:1573-1577 (1987) and Lenardo et al., <i>Proc. Natl. Acad. Sci. USA, </i>85:8825-8829 (1988). Assay samples of 20 μl contained nuclear extract incubated with 0.25 ng <sup>32</sup>P-labeled DNA fragment (5,000 CPM), 10 mM Tris:Cl, pH 7.5, 1 mM dithiothreitol, 1 mM EDTA, 0.5 mM MgCl<sub>2</sub>, 3 mM GTP (omitted in experiments varying amounts of added GTP), 2 μg poly(dI-dC), and 5% glycerol for 20 minutes at room temperature. Cytosol was activated in vitro using 0.8% sodium deoxycholate followed by addition of the binding mixture including 0.75% NP-40. Methylation interference assays were performed using the procedure of Gilman et al., <i>Mol. Cell. Biol., </i>6:4305-4316 (1986).</p>
    <p>RNA Analysis</p>
    <p>RNA preparation and Northern blot analysis were carried out as previously described by Zinn et al., <i>Cell, </i>34, 865-879 (1983).</p>
    <p>B. Results</p>
    <p>NF-κB binds specifically to PRDII</p>
    <p>The protein encoded by the PRDII-BF1 cDNA binds to the PRDII site and to the H2-K<sup>b </sup>and κB binding sites (FIG. 39, Singh et al., <i>Cell, </i>52:415-423 (1988); Fan and Maniatis, unpublished). Thus, the ability of NF-κB to bind to PRDII and to the κB site was compared, using an electrophoretic mobility shift assay. Fried, M. and Crothers, D. M., <i>Nucleic Acid Res., </i>9:6505-6525 (1981). NF-κB is present in the human T lymphocytic line Jurkat in an inactive form, but its binding is inducible by phorbol myristate acetate (PMA) and phytohemagglutinin (PHA). Sen, R. and D. Baltimore, <i>Cell, </i>47:921-928 (1986); and Nabel, G. and D. Baltimore, <i>Nature, </i>326:711-713 (1987). The entire interferon gene regulatory element (IRE) or an oligonucleotide comprised of two copies of the PRDII sequences (PRDII<sub>2 </sub>or (P)<sub>2</sub>) was used. A complex with PMA/PHA-induced Jurkat nuclear extracts that migrated identically to that formed with the κB site was detected. It was assumed that the additional slower migrating complex observed with the PRDII<sub>2 </sub>oligonucleotide corresponds to DNA molecules in which both of the PRDII sites are bound to protein. Specific κB complexes were undetectable in extracts from unstimulated Jurkat cells.</p>
    <p>To determine whether the same protein binds to PRDII and κB, competition experiments were carried out. Increasing amounts of unlabeled κB oligonucleotide inhibited complex formation with either the IRE or the κB site. The ability of PRDII and κB to compete for NF-κB binding was also reciprocal; an unlabeled fragment prevented complex formation with either labeled fragment. Quantitatively, both sites competed equally for NF-κB. Furthermore, the IRE and the κB sites both formed an identical complex using cytosol from unstimulated Jurkat cells after treatment with the detergent deoxycholate. This observation is in agreement with the finding that NF-κB binding activity can be unmasked in cytosolic extracts by deoxycholate. Baeuerle, P. and D. Baltimore, <i>Cell, </i>53:211-217 (1988).</p>
    <p>A number of single base mutations in PRDII decrease the level of virus induction of the β-IFN gene. Goodbourn, S. and T. Maniatis, <i>Proc. Natl. Acad. Sci. USA, </i>85:1447-1451 (1988). Therefore, these mutations were examined to assess whether they also affect in vitro binding to NF-κB. Four point mutations that impair inducibility of the β-IFN gene were shown to reduce binding of NF-κB. (64G→A, 62G→A, 60A→G and 56C→T). A single point mutation that has no effect on inducibility allows specific NF-κB binding (65T→C). Taken together, these results strongly suggest the NF-κB plays a direct role in β-IFN gene regulation.</p>
    <p>The in Vivo Activities of PRDIII and the κB Site are Indistinguishable</p>
    <p>To determine whether the PRDII and κB sites function similarly in vivo, we compared their transcriptional activities were compared, using chloramphenicol acetyl-transferase gene (CAT) reporter plasmids in virus-induced mouse L929 fibroblasts and in S194 mouse myeloma cells. The structures of the reporter genes are illustrated in FIG. <b>41</b>B. The -41 β-globin/CAT gene was not expressed in L929 cells before or after induction by inactivated Sendai virus or poly(rI:rC) (FIG. 41A, lanes 1-3). However, the reporter gene linked to four copies of PRDII (-41β(P)<sub>4</sub>) was highly inducible (FIG. 41A, lanes 4-6). Remarkably, two tandemly-repeated κB sites also conferred virus and poly(rI:rC) inducibility on a c-fos promoter/CAT fusion gene in L929 cells (FIG. 41A, compare lanes 7-9 to 10-12). Mutations in the κB site that eliminated binding of NF-κB also abolished virus inducibility. Thus, in L929 cells, the same inducible factor may interact with PRDII and the κB site to stimulate transcription.</p>
    <p>The B-cell specific activities of PRDII and the κB site were compared by transfecting the reporter enes into S194 mouse myeloma cells. As previously demonstrated, wild-type, but not mutant, κB sites were highly active in mature B-cells which constitutively express NF-κB (FIG. 40A, lanes 13-15; Pierce et al., <i>Proc. Natl. Acad. Sci. USA, </i>85:1482-1486 (1988)). Significantly, multiple PRDII elements were also highly active in S194 cells (lanes 16-18). These results further suggest that the same factor, NF-κB, interacts productively with PRDII and the κB site.</p>
    <p>Virus Induction Stimulates NF-κB Binding in Lymphoid and Non-lymphoid Cells</p>
    <p>The ability of PRDII to bind NF-κB suggested that virus infection might activate NF-κB. Therefore, nuclear extracts prepared from cells before and after virus induction were analyzed. NF-κB binding activity was virus-inducible in Namalwa human mature B lymphocytes, 70Z/3 murine pre-B lymphocytes, and murine L929 fibroblasts. Virus Induction of NF-κB in Namalwa cells was unexpected because these cells display significant constitutive NF-κB binding activity in the nucleus. Thus, only a fraction of the NF-κB in Namalwa cells is in the active state, and the remaining molecules can be activated by virus. Virus-induced complexes in all three cell types appeared to contain NF-κB because they could be eliminated by competition with wild-type (WT) but not mutant (MUT) κB sites. Moreover, complex formation could be stimulated by GTP, a biochemical property of NF-κB. Lenardo et al., <i>Proc. Natl. Acad. Sci. USA, </i>85:8825-8829 (1988). The NF-κB complex was also induced by poly(rI:rC) in 70Z/3 cells, but the effect was less dramatic.</p>
    <p>Additional evidence that the virus-induced complexes contain NF-κB was provided by comparisons of the methylation interference patterns of the virus-induced complexes from Namalwa and L929 cells with the PMA/PHA-induced complex in Jurkat cells. All of the interference patterns were identical and exhibited the close base contacts that are distinctive of the interaction between NF-κB and its cognate binding site in the κ enhancer. Sen, R. and D. Baltimore, <i>Cell, </i>46:705-716 (1986); and Baldwin, A. S. and P. A. Sharp, <i>Mol. Cell. Biol., </i>7:305-313 (1988). Finally, the level of NF-κB revealed by deoxycholate in the cytosol of L929 cells was diminished after virus induction. Therefore, like phorbol ester treatment, virus infection apparently releases NF-κB from an active cytosolic form and allows translocation to the nucleus. Baeuerle, P. and D. Baltimore, <i>Cell, </i>53:211-217 (1988).</p>
    <p>Endogenous β-IFN and Ig Kappa Gene Expression are Activated by Virus in Pre-B Lymphocytes</p>
    <p>As shown above, virus infection dramatically increased the levels of nuclear NF-κB and induced reporter genes containing PRDII or κB sites. Therefore, the ability of virus to induce the transcription of an endogenous κ gene was also assessed. The pre-B cell line 70Z/3, which produces cytoplasmic Ig μ heavy chains, but not light chains, was used for this purpose. Paige et al., <i>J. Immunol., </i>121:641-647 (1978). The κ gene in 70Z/3 cells is functionally rearranged and can be transcriptionally induced by lipopolysaccharide (LPS) and phorbol myristate acetate (PMA), conditions which powerfully induce NF-κB. Nelson et al., <i>Nucl. Acids Res., </i>12: 1911-1923 (1984); Rosoff and Cantley, <i>J. Biol. Chem., </i>259:7056-7060 (1985); and Sen, R. and D. Baltimore, <i>Cell, </i>47:921-928 (1986). As expected, treatment of 70Z/3 cells with phorbol esters or, more strikingly, with LPS resulted in the activation of the endogenous κ gene. Surprisingly, virus infection also induced κ gene expression to a level comparable to that observed with PMA induction. Under the same conditions, endogenous β-IFN mRNA was induced by virus, but not by PMA or LPS. Thus, NF-κB is necessary and sufficient for expression of the endogenous κ gene in 70Z/3 cells, but not for the β-IFN gene. These results indicate the virus-induced complex has all the in vitro binding properties and in vivo transcriptional properties of NF-κB.</p>
    <heading>EQUIVALENTS</heading> <p>Those skilled in the art will recognize or be able to ascertain, using no more than routine experimentation, many equivalents of the specific embodiments of the invention described therein. Such equivalents are intended to be encompassed by the following claims.</p>
    </div></div></div><div class="patent-section patent-tabular-section"><a id="backward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Cited Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1987004170A1?cl=en">WO1987004170A1</a></td><td class="patent-data-table-td patent-date-value">Jan 9, 1987</td><td class="patent-data-table-td patent-date-value">Jul 16, 1987</td><td class="patent-data-table-td ">Massachusetts Inst Technology</td><td class="patent-data-table-td ">Nuclear factors associated with transcriptional regulation</td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="npl-citations"></a><div class="patent-section-header"><span class="patent-section-title">Non-Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th colspan="3"class="patent-data-table-th">Reference</th></tr></thead><tr><td class="patent-data-table-td ">1</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Baeuerle, P.A and Baltimore, D., "<a href='http://scholar.google.com/scholar?q="IkappaB%3A+A+Specific+Inhibitor+of+the+NF-kappaB+Transcription+Factor%2C"'>IkappaB: A Specific Inhibitor of the NF-kappaB Transcription Factor,</a>" Science, 242:540-546 (1988).</td></tr><tr><td class="patent-data-table-td ">2</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Baeuerle, P.A and Baltimore, D., "<a href='http://scholar.google.com/scholar?q="I%CE%BAB%3A+A+Specific+Inhibitor+of+the+NF-%CE%BAB+Transcription+Factor%2C"'>IκB: A Specific Inhibitor of the NF-κB Transcription Factor,</a>" Science, 242:540-546 (1988).</td></tr><tr><td class="patent-data-table-td ">3</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Baeuerle, P.A. and Baltimore, D., "<a href='http://scholar.google.com/scholar?q="Activation+of+DNA-Binding+Activity+in+an+Apparently+Cytoplasmic+Precursor+of+the+NF-kappaB+Transcription+Factor%2C"'>Activation of DNA-Binding Activity in an Apparently Cytoplasmic Precursor of the NF-kappaB Transcription Factor,</a>" Cell, 53:211-217 (1988).</td></tr><tr><td class="patent-data-table-td ">4</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Baeuerle, P.A. and Baltimore, D., "<a href='http://scholar.google.com/scholar?q="Activation+of+DNA-Binding+Activity+in+an+Apparently+Cytoplasmic+Precursor+of+the+NF-%CE%BAB+Transcription+Factor%2C"'>Activation of DNA-Binding Activity in an Apparently Cytoplasmic Precursor of the NF-κB Transcription Factor,</a>" Cell, 53:211-217 (1988).</td></tr><tr><td class="patent-data-table-td ">5</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Baeurle, P.A. and Baltimore, D., "<a href='http://scholar.google.com/scholar?q="Activation+of+NF-kappaB%3A+A+Transcription+Factor+Controlling+Expression+of+the+Immunoglobuli+kappa+Light-chain+Gene+and+of+HIV%2C"'>Activation of NF-kappaB: A Transcription Factor Controlling Expression of the Immunoglobuli kappa Light-chain Gene and of HIV,</a>" The Control of Human Retrovirus Gene Expression, Banbury Conference, Cold Spring Harbor, NY, pp.: 217-226 (1988).</td></tr><tr><td class="patent-data-table-td ">6</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Baeurle, P.A. and Baltimore, D., "<a href='http://scholar.google.com/scholar?q="Activation+of+NF-%CE%BAB%3A+A+Transcription+Factor+Controlling+Expression+of+the+Immunoglobuli+%CE%BA+Light-chain+Gene+and+of+HIV%2C"'>Activation of NF-κB: A Transcription Factor Controlling Expression of the Immunoglobuli κ Light-chain Gene and of HIV,</a>" The Control of Human Retrovirus Gene Expression, Banbury Conference, Cold Spring Harbor, NY, pp.: 217-226 (1988).</td></tr><tr><td class="patent-data-table-td ">7</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Baldwin, A. and Sharp, P., et al., "<a href='http://scholar.google.com/scholar?q="Binding+of+a+Nuclear+Factor+to+a+Regulatory+Sequence+in+the+Promoter+of+the+Mouse+H-2Kb+Class+I+Major+Histocompatibility+Gene%2C"'>Binding of a Nuclear Factor to a Regulatory Sequence in the Promoter of the Mouse H-2Kb Class I Major Histocompatibility Gene,</a>" Mol. &amp; Cell. Biol., 7(1):305-313 (1987).</td></tr><tr><td class="patent-data-table-td ">8</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Baldwin, Jr., A.S., and Sharp, P.A., "<a href='http://scholar.google.com/scholar?q="Two+transcription+factors%2C+NF-kB+and+H2TF1%2C+interact+with+a+single+regulatory+sequence+in+the+class+I+major+histocompatability+complex+promotor%2C"'>Two transcription factors, NF-kB and H2TF1, interact with a single regulatory sequence in the class I major histocompatability complex promotor,</a>" Proc. Natl. Acad. Sci, USA, 85:723-727 (1988).</td></tr><tr><td class="patent-data-table-td ">9</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Ballard, D.W., et al., "<a href='http://scholar.google.com/scholar?q="HTLV-I+Tax+Induces+Cellular+Proteins+That+Activate+the+kB+Element+in+the+IL-2+Receptor+a+Gene%2C"'>HTLV-I Tax Induces Cellular Proteins That Activate the kB Element in the IL-2 Receptor a Gene,</a>" Science, 241:1652-1657 (1988).</td></tr><tr><td class="patent-data-table-td ">10</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Banerji, J., et al., "<a href='http://scholar.google.com/scholar?q="A+Lymphocyte-Specific+Cellular+Enhancer+Is+Located+Downstream+of+the+Joining+Region+in+Immunoglobulin+Heavy+Chain+Genes%2C"'>A Lymphocyte-Specific Cellular Enhancer Is Located Downstream of the Joining Region in Immunoglobulin Heavy Chain Genes,</a>" Cell, 33:729-740 (1983).</td></tr><tr><td class="patent-data-table-td ">11</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Bergman, Y., et al., "<a href='http://scholar.google.com/scholar?q="Two+regulatory+elements+for+immunoglobulin+kappa+light+chain+gene+expression%2C"'>Two regulatory elements for immunoglobulin kappa light chain gene expression,</a>" Proc. Natl. Acad. Sci. USA, 81:7041-7045 (1984).</td></tr><tr><td class="patent-data-table-td ">12</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Bergman, Y., et al., "<a href='http://scholar.google.com/scholar?q="Two+regulatory+elements+for+immunoglobulin+%CE%BA+light+chain+gene+expression%2C"'>Two regulatory elements for immunoglobulin κ light chain gene expression,</a>" Proc. Natl. Acad. Sci. USA, 81:7041-7045 (1984).</td></tr><tr><td class="patent-data-table-td ">13</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Blanar, M.A., et al., "<a href='http://scholar.google.com/scholar?q="Nf-kB+Binds+within+a+Region+Required+for+B-Cell-Specific+Expression+of+the+Major+Histocompatibility+Complex+Class+II+Gene+Ead%2C"'>Nf-kB Binds within a Region Required for B-Cell-Specific Expression of the Major Histocompatibility Complex Class II Gene Ead,</a>" Mol. &amp; Cell. Biol., 9 (2):844-846.</td></tr><tr><td class="patent-data-table-td ">14</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Böhnlein, E.,, et al., "<a href='http://scholar.google.com/scholar?q="The+Same+Inducible+Nuclear+Proteins+Regulates+Mitogen+Activation+of+Both+the+Interleukin-2+Receptor-Alpha+Gene+and+Type+1+HIV%2C"'>The Same Inducible Nuclear Proteins Regulates Mitogen Activation of Both the Interleukin-2 Receptor-Alpha Gene and Type 1 HIV,</a>" Cell, 53:827-836 (1988).</td></tr><tr><td class="patent-data-table-td ">15</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Church, G. M., et al., "<a href='http://scholar.google.com/scholar?q="Cell-type-specific+contacts+to+immunoglobulin+enhancers+in+nuclei%2C"'>Cell-type-specific contacts to immunoglobulin enhancers in nuclei,</a>" Nature, 313:798-801 (1985).</td></tr><tr><td class="patent-data-table-td ">16</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Cross, S. L., et al., "<a href='http://scholar.google.com/scholar?q="Functionally+Distinct+NF-kappaB+Binding+Sites+in+the+Immunoglobulin+kappa+and+IL-2+Receptor+alpha+Chain+Genes%2C"'>Functionally Distinct NF-kappaB Binding Sites in the Immunoglobulin kappa and IL-2 Receptor alpha Chain Genes,</a>" Science, 244:466-469 (1989).</td></tr><tr><td class="patent-data-table-td ">17</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Cross, S. L., et al., "<a href='http://scholar.google.com/scholar?q="Functionally+Distinct+NF-%CE%BAB+Binding+Sites+in+the+Immunoglobulin+%CE%BA+and+IL-2+Receptor+%CE%B1+Chain+Genes%2C"'>Functionally Distinct NF-κB Binding Sites in the Immunoglobulin κ and IL-2 Receptor α Chain Genes,</a>" Science, 244:466-469 (1989).</td></tr><tr><td class="patent-data-table-td ">18</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Davis, N., et al., "<a href='http://scholar.google.com/scholar?q="Rel-Associated+pp40%3A+An+Inhibitor+of+the+Rel+Family+of+Transcription+Factors%2C"'>Rel-Associated pp40: An Inhibitor of the Rel Family of Transcription Factors,</a>" Science 253:1268-1271 (1991).</td></tr><tr><td class="patent-data-table-td ">19</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Fletcher, C., et al., "<a href='http://scholar.google.com/scholar?q="Purification+and+Characterizaiton+of+OTF-1%2C+a+Transcription+Factor+Regulating+Cell+Cycle+Expression+of+a+Human+Histone+H2b+Gene%2C"'>Purification and Characterizaiton of OTF-1, a Transcription Factor Regulating Cell Cycle Expression of a Human Histone H2b Gene,</a>" Cell, 51:773-781 (1987).</td></tr><tr><td class="patent-data-table-td ">20</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Foster, J., et al., "<a href='http://scholar.google.com/scholar?q="An+immunoglobulin+promoter+displays+cell-type+specificity+independently+of+the+enhancer%2C"'>An immunoglobulin promoter displays cell-type specificity independently of the enhancer,</a>" Nature, 315:423-425 (1985).</td></tr><tr><td class="patent-data-table-td ">21</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Fried, M. and Crothers, D. M., "<a href='http://scholar.google.com/scholar?q="Equilibria+and+kinetics+of+lac+repressor-operator+interactions+by+polyacrylamide+gel+electrophoresis%2C"'>Equilibria and kinetics of lac repressor-operator interactions by polyacrylamide gel electrophoresis,</a>" Nucleic Acids Research, 9(23):6505-6524 (1981).</td></tr><tr><td class="patent-data-table-td ">22</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Garner, M. M. and Revzin, A., "<a href='http://scholar.google.com/scholar?q="A+gel+electrophoresis+method+for+quantifying+the+binding+of+proteins+to+specific+DNA+regions%3A+application+to+components+of+the+Escherichia+coli+lactose+operon+regulatory+system%2C"'>A gel electrophoresis method for quantifying the binding of proteins to specific DNA regions: application to components of the Escherichia coli lactose operon regulatory system,</a>" Nucleic Acids Research, 9(13):3047-3060 (1981).</td></tr><tr><td class="patent-data-table-td ">23</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Gerster, T., et al., "<a href='http://scholar.google.com/scholar?q="Cell+type-specificity+elements+of+the+immunoglobulin+heavy+chain+gene+enhancer%2C"'>Cell type-specificity elements of the immunoglobulin heavy chain gene enhancer,</a>" EMBO Journal, 6(5):1323-1330 (1987).</td></tr><tr><td class="patent-data-table-td ">24</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Goodbourn, S., et al., "<a href='http://scholar.google.com/scholar?q="Human+beta-Interferon+Gene+Expression+Is+Regulated+by+an+Inducible+Enhancer+Element%2C"'>Human beta-Interferon Gene Expression Is Regulated by an Inducible Enhancer Element,</a>" Cell, 41:509-520 (1985).</td></tr><tr><td class="patent-data-table-td ">25</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Goodbourn, S., et al., "<a href='http://scholar.google.com/scholar?q="Human+%CE%B2-Interferon+Gene+Expression+Is+Regulated+by+an+Inducible+Enhancer+Element%2C"'>Human β-Interferon Gene Expression Is Regulated by an Inducible Enhancer Element,</a>" Cell, 41:509-520 (1985).</td></tr><tr><td class="patent-data-table-td ">26</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Gosh, S. and Baltimore, D., "<a href='http://scholar.google.com/scholar?q="Activation+in+vitro+of+NF-kB+by+phosphorylation+of+its+inhibitor+IkB%2C"'>Activation in vitro of NF-kB by phosphorylation of its inhibitor IkB,</a>" Nature, 344(6267): 678-682 (1990).</td></tr><tr><td class="patent-data-table-td ">27</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Grosschedi, R. and Baltimore, D., "<a href='http://scholar.google.com/scholar?q="Cell-Type+Specificity+of+Immunoglobulin+Gene+Expression+is+Regulated+by+at+Least+Three+DNA+Sequence+Elements%2C"'>Cell-Type Specificity of Immunoglobulin Gene Expression is Regulated by at Least Three DNA Sequence Elements,</a>" Cell, 41:885-897 (1985).</td></tr><tr><td class="patent-data-table-td ">28</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Haskill, S., et al., "<a href='http://scholar.google.com/scholar?q="Characterization+of+an+Immediate-Early+Gene+Induced+in+Adherent+Monocytes+That+Encodes+IkB-like+Activity%2C"'>Characterization of an Immediate-Early Gene Induced in Adherent Monocytes That Encodes IkB-like Activity,</a>" Cell, 65:1281-1289 (1991).</td></tr><tr><td class="patent-data-table-td ">29</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Johnston et al. (1993) Present Status and future prospects for HIV therapies. Science 260:1286-1293.</td></tr><tr><td class="patent-data-table-td ">30</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Karin, M., et al., "<a href='http://scholar.google.com/scholar?q="Activation+of+a+Heterologous+Promoter+in+Response+to+Dexamethasone+and+Cadmium+by+Metallothionein+Gene+5%27Flanking+DNA%2C"'>Activation of a Heterologous Promoter in Response to Dexamethasone and Cadmium by Metallothionein Gene 5'Flanking DNA,</a>" Cell, 36:371-379 (1984).</td></tr><tr><td class="patent-data-table-td ">31</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Karin, M., et al., "<a href='http://scholar.google.com/scholar?q="Activation+of+a+Heterologous+Promoter+in+Response+to+Dexamethasone+and+Cadmium+by+Metallothionein+Gene+5%E2%80%B2Flanking+DNA%2C"'>Activation of a Heterologous Promoter in Response to Dexamethasone and Cadmium by Metallothionein Gene 5′Flanking DNA,</a>" Cell, 36:371-379 (1984).</td></tr><tr><td class="patent-data-table-td ">32</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Kawakami, K., et al., "<a href='http://scholar.google.com/scholar?q="Identification+and+purification+of+a+human+immunoglobulin-enhancer-binding+protein+%28NF-kappaB%29+that+activates+transcription+from+a+human+immunodeficiency+virus+type+1+promoter+in+vitro%2C"'>Identification and purification of a human immunoglobulin-enhancer-binding protein (NF-kappaB) that activates transcription from a human immunodeficiency virus type 1 promoter in vitro,</a>" Proc. Natl. Acad. Sci. USA, 85:4700-4704 (1988).</td></tr><tr><td class="patent-data-table-td ">33</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Kawakami, K., et al., "<a href='http://scholar.google.com/scholar?q="Identification+and+purification+of+a+human+immunoglobulin-enhancer-binding+protein+%28NF-%CE%BAB%29+that+activates+transcription+from+a+human+immunodeficiency+virus+type+1+promoter+in+vitro%2C"'>Identification and purification of a human immunoglobulin-enhancer-binding protein (NF-κB) that activates transcription from a human immunodeficiency virus type 1 promoter in vitro,</a>" Proc. Natl. Acad. Sci. USA, 85:4700-4704 (1988).</td></tr><tr><td class="patent-data-table-td ">34</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Ko, H.-S., et al., "<a href='http://scholar.google.com/scholar?q="A+Human+Protein+Specific+for+the+Immunoglobulin+Octamer+DNA+Motif+Contains+a+Functional+Homeobox+Domain%2C"'>A Human Protein Specific for the Immunoglobulin Octamer DNA Motif Contains a Functional Homeobox Domain,</a>" Cell, 55:135-144 (1988).</td></tr><tr><td class="patent-data-table-td ">35</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Landolfi, N. F., et al., "<a href='http://scholar.google.com/scholar?q="Interaction+of+cell-type-specific+nuclear+proteins+with+immunoglobulin+VH+promoter+region+sequences%2C"'>Interaction of cell-type-specific nuclear proteins with immunoglobulin VH promoter region sequences,</a>" Nature, 323:548-551 (1986).</td></tr><tr><td class="patent-data-table-td ">36</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Lenardo, J.J., et al., "<a href='http://scholar.google.com/scholar?q="NF-kB+protein+purification+from+bovine+spleen%3A+Nucleotide+stimulation+and+binding+site+specificity%2C"'>NF-kB protein purification from bovine spleen: Nucleotide stimulation and binding site specificity,</a>" Prod. Natl. Acad. Sci. USA, 85:8825-8829 (1988).</td></tr><tr><td class="patent-data-table-td ">37</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Lenardo, M., et al., "<a href='http://scholar.google.com/scholar?q="Protein-Binding+Sites+in+Ig+Gene+Enhancers+Determine+Transcriptional+Activity+and+Inducibility%2C"'>Protein-Binding Sites in Ig Gene Enhancers Determine Transcriptional Activity and Inducibility,</a>" Science, 236:1573-1577 (1987).</td></tr><tr><td class="patent-data-table-td ">38</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Leonard et al. (1985) Interleukin 2 receptor gene expression in normal human T lymphocytes. Proc. Natl. Acad. Sci. USA 82:6281-6285.*</td></tr><tr><td class="patent-data-table-td ">39</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Leung, K. and Nable, G.J., "<a href='http://scholar.google.com/scholar?q="HTLV-1+transactivator+induces+interleukin-2+receptor+expression+through+an+NF-kB-like+factor%2C"'>HTLV-1 transactivator induces interleukin-2 receptor expression through an NF-kB-like factor,</a>" Nature, 333:776-778 (1988).</td></tr><tr><td class="patent-data-table-td ">40</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Mason, J. O., et al., "<a href='http://scholar.google.com/scholar?q="Transcription+Cell+Type+Specificity+Is+Conferred+by+an+Immunoglobulin+VH+Gene+Promoter+That+Includes+a+Functional+Consensus+Sequence%2C"'>Transcription Cell Type Specificity Is Conferred by an Immunoglobulin VH Gene Promoter That Includes a Functional Consensus Sequence,</a>" Cell, 41:479-487 (1985).</td></tr><tr><td class="patent-data-table-td ">41</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Mercola, M., et al., "<a href='http://scholar.google.com/scholar?q="Immunoglobulin+Heavy-Chain+Enhancer+Requires+One+or+More+Tissue-Specific+Factors%2C"'>Immunoglobulin Heavy-Chain Enhancer Requires One or More Tissue-Specific Factors,</a>" Science, 227:266-270 (1985).</td></tr><tr><td class="patent-data-table-td ">42</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Mercola, M., et al., "<a href='http://scholar.google.com/scholar?q="Transcriptional+Enhancer+Elements+in+the+Mouse+Immunoglobulin+Heavy+Chain+Locus%2C"'>Transcriptional Enhancer Elements in the Mouse Immunoglobulin Heavy Chain Locus,</a>" Science, 221:663-665 (1983).</td></tr><tr><td class="patent-data-table-td ">43</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Nabel, G. and Baltimore, D., An inducible transcription factor activates expression of human immunodeficiency virus in T cells, Nature, 326:711-713 (1987).</td></tr><tr><td class="patent-data-table-td ">44</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Nelsen, B., et al., "<a href='http://scholar.google.com/scholar?q="The+NF-kB-Binding+Site+Mediates+Phorbol+Ester-Inducible+Transcription+in+Nonlymphoid+Cells%2C"'>The NF-kB-Binding Site Mediates Phorbol Ester-Inducible Transcription in Nonlymphoid Cells,</a>" Mol. &amp; Cell Biol., 8:3526-3531 (1988).</td></tr><tr><td class="patent-data-table-td ">45</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Nelson, K. J., et al., "<a href='http://scholar.google.com/scholar?q="Inducible+transcription+of+the+unrearranged+kappa+constant+region+locus+is+a+common+feature+of+pre-B+cells+and+does+not+require+DNA+or+protein+synthesis%2C"'>Inducible transcription of the unrearranged kappa constant region locus is a common feature of pre-B cells and does not require DNA or protein synthesis,</a>" Proc. Natl. Acad. Sci. USA, 82:5305-5309 (1985).</td></tr><tr><td class="patent-data-table-td ">46</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Nelson, K. J., et al., "<a href='http://scholar.google.com/scholar?q="Inducible+transcription+of+the+unrearranged+%CE%BA+constant+region+locus+is+a+common+feature+of+pre-B+cells+and+does+not+require+DNA+or+protein+synthesis%2C"'>Inducible transcription of the unrearranged κ constant region locus is a common feature of pre-B cells and does not require DNA or protein synthesis,</a>" Proc. Natl. Acad. Sci. USA, 82:5305-5309 (1985).</td></tr><tr><td class="patent-data-table-td ">47</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Picard, D. and Schaffner, W., "<a href='http://scholar.google.com/scholar?q="A+lymphocyte-specific+enhancer+in+the+mouse+immunoglobulin+kappa+gene%2C"'>A lymphocyte-specific enhancer in the mouse immunoglobulin kappa gene,</a>" Nature, 307:80-82 (1984).</td></tr><tr><td class="patent-data-table-td ">48</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Picard, D. and Schaffner, W., "<a href='http://scholar.google.com/scholar?q="A+lymphocyte-specific+enhancer+in+the+mouse+immunoglobulin+%CE%BA+gene%2C"'>A lymphocyte-specific enhancer in the mouse immunoglobulin κ gene,</a>" Nature, 307:80-82 (1984).</td></tr><tr><td class="patent-data-table-td ">49</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Queen, C. and Baltimore, D., "<a href='http://scholar.google.com/scholar?q="Immunoglobulin+Gene+Transcription+Is+Activated+by+Downstream+Sequence+Elements%2C"'>Immunoglobulin Gene Transcription Is Activated by Downstream Sequence Elements,</a>" Cell, 33:741-748 (1983).</td></tr><tr><td class="patent-data-table-td ">50</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Queen, C. and Stafford, J., "<a href='http://scholar.google.com/scholar?q="Fine+Mapping+of+an+Immunoglobulin+Gene+Activator%2C"'>Fine Mapping of an Immunoglobulin Gene Activator,</a>" Mol. Cel. Biol., 4(6):1042-1049 (1984).</td></tr><tr><td class="patent-data-table-td ">51</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Ruben, S., et al., "<a href='http://scholar.google.com/scholar?q="Cellular+Transcription+Factors+and+Regulation+of+IL-2+Receptor+Gene+Expression+by+HTLV-1+tax+Gene+Product%2C"'>Cellular Transcription Factors and Regulation of IL-2 Receptor Gene Expression by HTLV-1 tax Gene Product,</a>" Science, 241:89-92 (1988).</td></tr><tr><td class="patent-data-table-td ">52</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Sassone-Corsi, P., et al., "<a href='http://scholar.google.com/scholar?q="A+trans-acting+factor+is+responsible+for+the+simian+virus+40+enhancer+activity+in+vitro%2C"'>A trans-acting factor is responsible for the simian virus 40 enhancer activity in vitro,</a>" Nature, 313:458-463 (1985).</td></tr><tr><td class="patent-data-table-td ">53</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Scheidereit, C., et al., "<a href='http://scholar.google.com/scholar?q="Identification+and+Purification+of+a+Human+Lymphoid-Specific+Octamer-Binding+Protein+%28OTF-2%29+That+Activates+Transcription+of+an+Immunoglobulin+Promoter+In+Vitro%2C"'>Identification and Purification of a Human Lymphoid-Specific Octamer-Binding Protein (OTF-2) That Activates Transcription of an Immunoglobulin Promoter In Vitro,</a>" Cell, 51:783-793 (1987).</td></tr><tr><td class="patent-data-table-td ">54</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Sen, R. and Baltimore, D., "<a href='http://scholar.google.com/scholar?q="Multiple+Nuclear+Factors+Interact+with+the+Immunoglobulin+Enhancer+Sequences%2C"'>Multiple Nuclear Factors Interact with the Immunoglobulin Enhancer Sequences,</a>" Cell, 46:705-716 (1986).</td></tr><tr><td class="patent-data-table-td ">55</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Sen, R. and Baltimore, D., Inducibility of kappa Immunoglobulin Enhancer-Binding Protein NF-kappaB by a Posttranslational Mechanism, Cell, 47:921-928 (1986).</td></tr><tr><td class="patent-data-table-td ">56</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Sen, R. and Baltimore, D., Inducibility of κ Immunoglobulin Enhancer-Binding Protein NF-κB by a Posttranslational Mechanism, Cell, 47:921-928 (1986).</td></tr><tr><td class="patent-data-table-td ">57</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Singh, H., et al., "<a href='http://scholar.google.com/scholar?q="A+nuclear+factor+that+binds+to+a+conserved+sequence+motif+in+transcriptional+control+elements+of+immunoglobulin+genes%2C"'>A nuclear factor that binds to a conserved sequence motif in transcriptional control elements of immunoglobulin genes,</a>" Nature, 319:154-158 (1986).</td></tr><tr><td class="patent-data-table-td ">58</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Staudt, L. M., et al., "<a href='http://scholar.google.com/scholar?q="A+Lymphoid-specific+protein+binding+to+the+octamer+motif+of+immunoglobulin+genes%2C"'>A Lymphoid-specific protein binding to the octamer motif of immunoglobulin genes,</a>" Nature, 323:640-643 (1986).</td></tr><tr><td class="patent-data-table-td ">59</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Staudt, L., et al., "<a href='http://scholar.google.com/scholar?q="Cloning+of+a+Lymphoid-Specific+cDNA+Encoding+a+Protein+Binding+the+Regulatory+Octamer+DNA+Motif%2C"'>Cloning of a Lymphoid-Specific cDNA Encoding a Protein Binding the Regulatory Octamer DNA Motif,</a>" Science, 241:577-580 (1988).</td></tr><tr><td class="patent-data-table-td ">60</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Strauss, F. and Varshavsky, A., "<a href='http://scholar.google.com/scholar?q="A+Protein+Binds+to+a+Satellite+DNA+Repeat+at+Three+Specific+Sites+That+Would+be+Brought+into+Mutualy+Proximity+by+DNA+Folding+in+the+Nucleosome%2C"'>A Protein Binds to a Satellite DNA Repeat at Three Specific Sites That Would be Brought into Mutualy Proximity by DNA Folding in the Nucleosome,</a>" Cell, 37:889-901 (1984).</td></tr><tr><td class="patent-data-table-td ">61</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Treisman, R., "<a href='http://scholar.google.com/scholar?q="Transient+Accumulation+of+c-fos+RNA+Following+Serum+Stimulation+Requires+a+Conserved+5%27Element+and+c-fos+3%27+Sequences%2C"'>Transient Accumulation of c-fos RNA Following Serum Stimulation Requires a Conserved 5'Element and c-fos 3' Sequences,</a>" Cell, 42:889-902 (1985).</td></tr><tr><td class="patent-data-table-td ">62</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Treisman, R., "<a href='http://scholar.google.com/scholar?q="Transient+Accumulation+of+c-fos+RNA+Following+Serum+Stimulation+Requires+a+Conserved+5%E2%80%B2Element+and+c-fos+3%E2%80%B2+Sequences%2C"'>Transient Accumulation of c-fos RNA Following Serum Stimulation Requires a Conserved 5′Element and c-fos 3′ Sequences,</a>" Cell, 42:889-902 (1985).</td></tr><tr><td class="patent-data-table-td ">63</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Wall, R., et al., "<a href='http://scholar.google.com/scholar?q="A+labile+inhibitor+blocks+immunoglobulin+kappa-light-chain-gene+transcription+in+a+pre-B+leukemic+cell+line%2C"'>A labile inhibitor blocks immunoglobulin kappa-light-chain-gene transcription in a pre-B leukemic cell line,</a>" Proc. Natl. Acad. Sci. USA, 83:295-298 (1986).</td></tr><tr><td class="patent-data-table-td ">64</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Wall, R., et al., "<a href='http://scholar.google.com/scholar?q="A+labile+inhibitor+blocks+immunoglobulin+%CE%BA-light-chain-gene+transcription+in+a+pre-B+leukemic+cell+line%2C"'>A labile inhibitor blocks immunoglobulin κ-light-chain-gene transcription in a pre-B leukemic cell line,</a>" Proc. Natl. Acad. Sci. USA, 83:295-298 (1986).</td></tr><tr><td class="patent-data-table-td ">65</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Wirth, T. and Baltimore, D., "<a href='http://scholar.google.com/scholar?q="Nuclear+factor+NF-kB+can+interact+functionally+with+its+cognate+binding+site+to+provide+lymphoid-specific+promotor+function%2C"'>Nuclear factor NF-kB can interact functionally with its cognate binding site to provide lymphoid-specific promotor function,</a>" The EMBO Journal, 7 (10):3109-3113 (1988).</td></tr><tr><td class="patent-data-table-td ">66</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Wu et al. (1988) Purification of the human immunodeficiency virus type 1 enhancer and TAR binding protiens EBP-1 and UBP-1, EMBO J. 7:2117-2129.*</td></tr><tr><td class="patent-data-table-td ">67</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Zabel, U. and Baeurle, P., "<a href='http://scholar.google.com/scholar?q="Purified+Human+IkB+Can+Rapidly+Dissociate+the+Complex+of+the+NF+kB+Transcription+Factor+with+it+Cognate+DNA%2C"'>Purified Human IkB Can Rapidly Dissociate the Complex of the NF kB Transcription Factor with it Cognate DNA,</a>" Cell, 61:255-265 (1990).</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="forward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Referenced by</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Citing Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6528482">US6528482</a></td><td class="patent-data-table-td patent-date-value">Dec 17, 1999</td><td class="patent-data-table-td patent-date-value">Mar 4, 2003</td><td class="patent-data-table-td ">Immunex Corporation</td><td class="patent-data-table-td ">Cytokine receptor/ligand pairs having immunoregulatory activity;RANK antagonist</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6537763">US6537763</a></td><td class="patent-data-table-td patent-date-value">May 31, 2001</td><td class="patent-data-table-td patent-date-value">Mar 25, 2003</td><td class="patent-data-table-td ">Immunex Corporation</td><td class="patent-data-table-td ">Receptor activator of NF-κB</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6545027">US6545027</a></td><td class="patent-data-table-td patent-date-value">Jun 7, 1995</td><td class="patent-data-table-td patent-date-value">Apr 8, 2003</td><td class="patent-data-table-td ">Eli Lilly And Company</td><td class="patent-data-table-td ">Methods of modulating NF-kB transcription factor</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6562948">US6562948</a></td><td class="patent-data-table-td patent-date-value">May 30, 2001</td><td class="patent-data-table-td patent-date-value">May 13, 2003</td><td class="patent-data-table-td ">Immunex Corporation</td><td class="patent-data-table-td ">RANK (receptor activator of neurotrophic factor-kappa B) polypeptides; immunoregulatory activity</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6566553">US6566553</a></td><td class="patent-data-table-td patent-date-value">Aug 10, 2001</td><td class="patent-data-table-td patent-date-value">May 20, 2003</td><td class="patent-data-table-td ">Millennium Pharmaceuticals, Inc.</td><td class="patent-data-table-td ">Synthetic pathway relies upon stereospecific synthesis of oxazoline intermediate and stereoselective addition of formyl amide to oxazoline</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6649164">US6649164</a></td><td class="patent-data-table-td patent-date-value">Jun 8, 2001</td><td class="patent-data-table-td patent-date-value">Nov 18, 2003</td><td class="patent-data-table-td ">Immunex Corporation</td><td class="patent-data-table-td ">(RANK); RANK is a member of the TNF superfamily; used to regulate an immune response</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6660268">US6660268</a></td><td class="patent-data-table-td patent-date-value">Mar 18, 1994</td><td class="patent-data-table-td patent-date-value">Dec 9, 2003</td><td class="patent-data-table-td ">The President And Fellows Of Harvard College</td><td class="patent-data-table-td ">Proteasome regulation of NF-KB activity</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6696453">US6696453</a></td><td class="patent-data-table-td patent-date-value">May 17, 2002</td><td class="patent-data-table-td patent-date-value">Feb 24, 2004</td><td class="patent-data-table-td ">Alcon, Inc.</td><td class="patent-data-table-td ">Use of NF-κB inhibitors to treat dry eye disorders</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6696498">US6696498</a></td><td class="patent-data-table-td patent-date-value">May 10, 2002</td><td class="patent-data-table-td patent-date-value">Feb 24, 2004</td><td class="patent-data-table-td ">Consiglio Nazionale Della Richerche</td><td class="patent-data-table-td ">2-cyclopenten-1-one and its derivatives as inhibitors of the NF-kB factor</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6831057">US6831057</a></td><td class="patent-data-table-td patent-date-value">Oct 28, 1997</td><td class="patent-data-table-td patent-date-value">Dec 14, 2004</td><td class="patent-data-table-td ">The University Of North Carolina At Chapel Hill</td><td class="patent-data-table-td ">Use of NF-κB inhibition in combination therapy for cancer</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6878735">US6878735</a></td><td class="patent-data-table-td patent-date-value">Jan 17, 2003</td><td class="patent-data-table-td patent-date-value">Apr 12, 2005</td><td class="patent-data-table-td ">Board Of Trustees Of Michigan State University</td><td class="patent-data-table-td ">Multi-substituted imidazolines and method of use thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6884598">US6884598</a></td><td class="patent-data-table-td patent-date-value">Sep 20, 2001</td><td class="patent-data-table-td patent-date-value">Apr 26, 2005</td><td class="patent-data-table-td ">Immunex Corporation</td><td class="patent-data-table-td ">Screening for modulator of receptor binding activity; obtain cells, incubate with modulator, monitor cell response</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6890909">US6890909</a></td><td class="patent-data-table-td patent-date-value">Jul 3, 1998</td><td class="patent-data-table-td patent-date-value">May 10, 2005</td><td class="patent-data-table-td ">Fujisawa Pharmaceutical Co., Ltd.</td><td class="patent-data-table-td ">Containing an NF-kappa B decoy; compound which specifically antagonizes the NF-kappa B binding sites; encephalopathy; diminishing a cerebral vasospasm associated with a subarachnoid hemorrhage; liposomal delivery of an</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6890946">US6890946</a></td><td class="patent-data-table-td patent-date-value">Jun 21, 2002</td><td class="patent-data-table-td patent-date-value">May 10, 2005</td><td class="patent-data-table-td ">Indiana University Research And Technology Corporation</td><td class="patent-data-table-td ">Or conditions characterized by abnormal angiogenesis or to treat non-tumorigenic angiogenesis-dependent diseases that are characterized by abnormal growth of blood vessels</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7081343">US7081343</a></td><td class="patent-data-table-td patent-date-value">Jun 18, 2001</td><td class="patent-data-table-td patent-date-value">Jul 25, 2006</td><td class="patent-data-table-td ">The Regents Of The University Of California</td><td class="patent-data-table-td ">Methods for identifying modulators of NF-KB activity</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7097834">US7097834</a></td><td class="patent-data-table-td patent-date-value">Dec 14, 1998</td><td class="patent-data-table-td patent-date-value">Aug 29, 2006</td><td class="patent-data-table-td ">Amgen Inc.</td><td class="patent-data-table-td ">Osteoprotegerin binding proteins</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7186503">US7186503</a></td><td class="patent-data-table-td patent-date-value">Sep 19, 2003</td><td class="patent-data-table-td patent-date-value">Mar 6, 2007</td><td class="patent-data-table-td ">Signal Pharmaceuticals Llc</td><td class="patent-data-table-td ">Compounds and methods for modulating activation of NF-κB</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7262274">US7262274</a></td><td class="patent-data-table-td patent-date-value">Apr 1, 2003</td><td class="patent-data-table-td patent-date-value">Aug 28, 2007</td><td class="patent-data-table-td ">Immunex Corporation</td><td class="patent-data-table-td ">Kit which detects ligand associated with nuclear factor kappa beta receptor for treating toxic shock, sepsis, inflammation and graft vs host disease</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7285654">US7285654</a></td><td class="patent-data-table-td patent-date-value">Jan 8, 2003</td><td class="patent-data-table-td patent-date-value">Oct 23, 2007</td><td class="patent-data-table-td ">Signal Pharmaceuticals</td><td class="patent-data-table-td ">Stimulus-inducible protein kinase complex and methods of use therefor</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7314936">US7314936</a></td><td class="patent-data-table-td patent-date-value">Jan 7, 2003</td><td class="patent-data-table-td patent-date-value">Jan 1, 2008</td><td class="patent-data-table-td ">Eisai Co., Ltd.</td><td class="patent-data-table-td ">antiinflammatory agents; 1H-imidazo[4,5-b]pyridin-7-indole-2-yl- derivatives; autoimmune and proliferative disorders; inhibitors of cell adhesion molecule expression and inflammatory cytokine signal transduction; rheumatoid arthritis, ulcerative colitis/Crohn&#39;s disease, central nervous system diseases</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7371411">US7371411</a></td><td class="patent-data-table-td patent-date-value">Dec 27, 2006</td><td class="patent-data-table-td patent-date-value">May 13, 2008</td><td class="patent-data-table-td ">Novelos Therapeutics, Inc.</td><td class="patent-data-table-td ">oxidized glutathione-based compound having a stabilized disulfide bond</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7411050">US7411050</a></td><td class="patent-data-table-td patent-date-value">Mar 16, 2004</td><td class="patent-data-table-td patent-date-value">Aug 12, 2008</td><td class="patent-data-table-td ">Immunex Corporation</td><td class="patent-data-table-td ">Bioassay for detection of receptor activated nuclear factor kappa beta ligand (RANKL); identifying modulator immune and inflammatory response</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7414120">US7414120</a></td><td class="patent-data-table-td patent-date-value">Jan 25, 2007</td><td class="patent-data-table-td patent-date-value">Aug 19, 2008</td><td class="patent-data-table-td ">Signal Pharmaceuticals</td><td class="patent-data-table-td ">enhances ubiquitination of phosphorylated I kappa B; human E3 ubiquitin ligase polypeptides; inflammatory diseases, autoimmune diseases, cancer and viral infections</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7528161">US7528161</a></td><td class="patent-data-table-td patent-date-value">Dec 3, 2003</td><td class="patent-data-table-td patent-date-value">May 5, 2009</td><td class="patent-data-table-td ">Michigan State University</td><td class="patent-data-table-td ">NF-kappaB inhibitors and uses thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7572594">US7572594</a></td><td class="patent-data-table-td patent-date-value">Apr 4, 2005</td><td class="patent-data-table-td patent-date-value">Aug 11, 2009</td><td class="patent-data-table-td ">Immunex Corporation</td><td class="patent-data-table-td ">Screening assays for agonists or antagonists or receptor activator of NF-κB</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7585848">US7585848</a></td><td class="patent-data-table-td patent-date-value">Jan 11, 2005</td><td class="patent-data-table-td patent-date-value">Sep 8, 2009</td><td class="patent-data-table-td ">Rush University Medical Center</td><td class="patent-data-table-td ">Administration of NF- kappa B inhibitor into intervertebral disc to treat intervertebral disc degeneration, wherein NF- kappa B inhibitor is selected from decoy oligonucleotide capable of binding to and acting on DNA binding site of NF- kappa B, antisense NF- kappa B nucleic acid, and NF- kappa B siRNA</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7612114">US7612114</a></td><td class="patent-data-table-td patent-date-value">Jun 22, 2005</td><td class="patent-data-table-td patent-date-value">Nov 3, 2009</td><td class="patent-data-table-td ">Eisai R&amp;D Management Co., Ltd.</td><td class="patent-data-table-td ">Selective estrogen receptor modulator</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7628989">US7628989</a></td><td class="patent-data-table-td patent-date-value">Jun 17, 2005</td><td class="patent-data-table-td patent-date-value">Dec 8, 2009</td><td class="patent-data-table-td ">Agensys, Inc.</td><td class="patent-data-table-td ">Methods of inducing an immune response</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7641905">US7641905</a></td><td class="patent-data-table-td patent-date-value">Jun 15, 2005</td><td class="patent-data-table-td patent-date-value">Jan 5, 2010</td><td class="patent-data-table-td ">Agensys, Inc.</td><td class="patent-data-table-td ">Methods of inducing an immune response</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7652056">US7652056</a></td><td class="patent-data-table-td patent-date-value">Apr 6, 2005</td><td class="patent-data-table-td patent-date-value">Jan 26, 2010</td><td class="patent-data-table-td ">Board Of Trustees Of Michigan State University</td><td class="patent-data-table-td ">Using 4-imidazoline carboxylic acid or ester; antiinflammatory agents; microbiocides; bactericides; antitumor agenst0</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7700073">US7700073</a></td><td class="patent-data-table-td patent-date-value">Sep 27, 2006</td><td class="patent-data-table-td patent-date-value">Apr 20, 2010</td><td class="patent-data-table-td ">The University Of North Carolina At Chapel Hill</td><td class="patent-data-table-td ">Use of NF-κB inhibition in combination therapy for cancer</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7700543">US7700543</a></td><td class="patent-data-table-td patent-date-value">Nov 2, 2004</td><td class="patent-data-table-td patent-date-value">Apr 20, 2010</td><td class="patent-data-table-td ">The University Of North Carolina At Chapel Hill</td><td class="patent-data-table-td ">Use NF-κB inhibition in combination therapy for cancer</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7700545">US7700545</a></td><td class="patent-data-table-td patent-date-value">Sep 27, 2006</td><td class="patent-data-table-td patent-date-value">Apr 20, 2010</td><td class="patent-data-table-td ">The University Of North Carolina At Chapel Hill</td><td class="patent-data-table-td ">Use of NF-κB inhibition in combination therapy for cancer</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7718776">US7718776</a></td><td class="patent-data-table-td patent-date-value">Apr 7, 2003</td><td class="patent-data-table-td patent-date-value">May 18, 2010</td><td class="patent-data-table-td ">Amgen Inc.</td><td class="patent-data-table-td ">Comprises monoclonal antibody which specifically binds osteoprotegerin ligand (OPGL) for diagnosis, prevention and treatment of bone disorders</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7736654">US7736654</a></td><td class="patent-data-table-td patent-date-value">Apr 10, 2002</td><td class="patent-data-table-td patent-date-value">Jun 15, 2010</td><td class="patent-data-table-td ">Agensys, Inc.</td><td class="patent-data-table-td ">Proteins, polynucleotides, antibodies, transgenic animals, hybridoma; humoral or cellular immune response; pancreatic</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7744886">US7744886</a></td><td class="patent-data-table-td patent-date-value">Jun 24, 2008</td><td class="patent-data-table-td patent-date-value">Jun 29, 2010</td><td class="patent-data-table-td ">Immunex Corporation</td><td class="patent-data-table-td ">Methods for interfering with rank signaling</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7759476">US7759476</a></td><td class="patent-data-table-td patent-date-value">Oct 22, 2007</td><td class="patent-data-table-td patent-date-value">Jul 20, 2010</td><td class="patent-data-table-td ">Signal Pharmaceuticals</td><td class="patent-data-table-td ">Double-stranded RNAS directed to IKK-2</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7790684">US7790684</a></td><td class="patent-data-table-td patent-date-value">Jun 11, 2008</td><td class="patent-data-table-td patent-date-value">Sep 7, 2010</td><td class="patent-data-table-td ">Immunex Corporation</td><td class="patent-data-table-td ">Method of inhibiting osteoclast activity</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7807795">US7807795</a></td><td class="patent-data-table-td patent-date-value">Dec 14, 1998</td><td class="patent-data-table-td patent-date-value">Oct 5, 2010</td><td class="patent-data-table-td ">Amgen Inc.</td><td class="patent-data-table-td ">Antibodies to osteoprotegerin binding proteins</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7868002">US7868002</a></td><td class="patent-data-table-td patent-date-value">Sep 28, 2007</td><td class="patent-data-table-td patent-date-value">Jan 11, 2011</td><td class="patent-data-table-td ">Eisai R&amp;D Management Co., Ltd.</td><td class="patent-data-table-td ">Deazapurines and uses thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7871983">US7871983</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Feb 14, 2003</td><td class="patent-data-table-td patent-date-value">Jan 18, 2011</td><td class="patent-data-table-td ">Anges Mg Inc.</td><td class="patent-data-table-td ">For therapy and prophylaxis of ischemic diseases, inflammatory diseases, autoimmune diseases, cancer metastasis and invasion, and cachexia</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7875274">US7875274</a></td><td class="patent-data-table-td patent-date-value">Dec 13, 2006</td><td class="patent-data-table-td patent-date-value">Jan 25, 2011</td><td class="patent-data-table-td ">The Board Of Trustees Of The Leland Stanford Junior University</td><td class="patent-data-table-td ">identifying therapeutically responsive phenotype in brain tumor cells by measuring expression of TNFAIP3 gene (tumor necrosis factor alpha-induced protein 3)</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7915009">US7915009</a></td><td class="patent-data-table-td patent-date-value">Dec 22, 2000</td><td class="patent-data-table-td patent-date-value">Mar 29, 2011</td><td class="patent-data-table-td ">The Mathilda And Terence Kennedy Institute Of Rheumatology Trust</td><td class="patent-data-table-td ">Modulating NF-kappa B by administering antigens or polynucleotide encoding antigens, antibody; treating cancer, tumor, Alzheimer&#39;s disease or a spongiform encephalopathy; DNA vaccine in kits</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7923008">US7923008</a></td><td class="patent-data-table-td patent-date-value">Jan 19, 2006</td><td class="patent-data-table-td patent-date-value">Apr 12, 2011</td><td class="patent-data-table-td ">Amgen Inc.</td><td class="patent-data-table-td ">monoclonal antibodies; treat bone diseases characterized by changes in bone resorption, such as osteoporosis, hypercalcemia, bone loss due to arthritis metastasis, immobilization or periodontal disease, Paget&#39;s disease; denosumab</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7927597">US7927597</a></td><td class="patent-data-table-td patent-date-value">Apr 30, 2004</td><td class="patent-data-table-td patent-date-value">Apr 19, 2011</td><td class="patent-data-table-td ">Agensys, Inc.</td><td class="patent-data-table-td ">Proteins, polynucleotides, antibodies, transgenic animals, hybridoma; tissue specific expression; pancreatic</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7932375">US7932375</a></td><td class="patent-data-table-td patent-date-value">Jul 30, 2007</td><td class="patent-data-table-td patent-date-value">Apr 26, 2011</td><td class="patent-data-table-td ">Immunex Corporation</td><td class="patent-data-table-td ">Kits for detecting rank nucleic acids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7943591">US7943591</a></td><td class="patent-data-table-td patent-date-value">May 12, 2008</td><td class="patent-data-table-td patent-date-value">May 17, 2011</td><td class="patent-data-table-td ">Adynxx, Inc.</td><td class="patent-data-table-td ">Gene expression and pain</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7951375">US7951375</a></td><td class="patent-data-table-td patent-date-value">Jun 17, 2005</td><td class="patent-data-table-td patent-date-value">May 31, 2011</td><td class="patent-data-table-td ">Agensys, Inc.</td><td class="patent-data-table-td ">Methods of inducing an immune response</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7960412">US7960412</a></td><td class="patent-data-table-td patent-date-value">Aug 20, 2009</td><td class="patent-data-table-td patent-date-value">Jun 14, 2011</td><td class="patent-data-table-td ">Eisai R&amp;D Management Co., Ltd.</td><td class="patent-data-table-td ">Selective estrogen receptor modulator</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8093225">US8093225</a></td><td class="patent-data-table-td patent-date-value">Mar 15, 2011</td><td class="patent-data-table-td patent-date-value">Jan 10, 2012</td><td class="patent-data-table-td ">Adynxx, Inc.</td><td class="patent-data-table-td ">Gene expression and pain</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8153775">US8153775</a></td><td class="patent-data-table-td patent-date-value">Apr 22, 2011</td><td class="patent-data-table-td patent-date-value">Apr 10, 2012</td><td class="patent-data-table-td ">Immunex Corporation</td><td class="patent-data-table-td ">Kits for detecting rank</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8247438">US8247438</a></td><td class="patent-data-table-td patent-date-value">Jun 10, 2008</td><td class="patent-data-table-td patent-date-value">Aug 21, 2012</td><td class="patent-data-table-td ">Neuropill, Inc.</td><td class="patent-data-table-td ">Methods for treating schizophrenia</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8263326">US8263326</a></td><td class="patent-data-table-td patent-date-value">Aug 8, 2008</td><td class="patent-data-table-td patent-date-value">Sep 11, 2012</td><td class="patent-data-table-td ">Signal Pharmaceuticals, Llc</td><td class="patent-data-table-td ">Compounds and methods for modulating activation of NF-κB</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8333963">US8333963</a></td><td class="patent-data-table-td patent-date-value">Aug 4, 2010</td><td class="patent-data-table-td patent-date-value">Dec 18, 2012</td><td class="patent-data-table-td ">Immunex Corporation</td><td class="patent-data-table-td ">Method of inhibiting osteoclast activity</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8367063">US8367063</a></td><td class="patent-data-table-td patent-date-value">May 18, 2010</td><td class="patent-data-table-td patent-date-value">Feb 5, 2013</td><td class="patent-data-table-td ">Amgen, Inc.</td><td class="patent-data-table-td ">Human anti-OPGL neutralizing antibodies as selective OPGL pathway inhibitors</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8377690">US8377690</a></td><td class="patent-data-table-td patent-date-value">Jun 15, 2010</td><td class="patent-data-table-td patent-date-value">Feb 19, 2013</td><td class="patent-data-table-td ">Immunex Corporation</td><td class="patent-data-table-td ">Cells and methods for producing blocking antibodies to human RANKL</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8399520">US8399520</a></td><td class="patent-data-table-td patent-date-value">Mar 15, 2011</td><td class="patent-data-table-td patent-date-value">Mar 19, 2013</td><td class="patent-data-table-td ">Eisai R&amp;D Management Co., Ltd.</td><td class="patent-data-table-td ">Selective estrogen receptor modulator</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8455629">US8455629</a></td><td class="patent-data-table-td patent-date-value">Jan 27, 2012</td><td class="patent-data-table-td patent-date-value">Jun 4, 2013</td><td class="patent-data-table-td ">Amgen Inc.</td><td class="patent-data-table-td ">Human anti-OPGL neutralizing antibodies as selective OPGL pathway inhibitors</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8552206">US8552206</a></td><td class="patent-data-table-td patent-date-value">Apr 3, 2009</td><td class="patent-data-table-td patent-date-value">Oct 8, 2013</td><td class="patent-data-table-td ">Board Of Trustees Of Michigan State University</td><td class="patent-data-table-td ">NF-κB inhibitors and uses thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8569456">US8569456</a></td><td class="patent-data-table-td patent-date-value">Mar 19, 2012</td><td class="patent-data-table-td patent-date-value">Oct 29, 2013</td><td class="patent-data-table-td ">Immunex Corporation</td><td class="patent-data-table-td ">Receptor activator of NF-kappaB</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8592466">US8592466</a></td><td class="patent-data-table-td patent-date-value">Jul 18, 2012</td><td class="patent-data-table-td patent-date-value">Nov 26, 2013</td><td class="patent-data-table-td ">Neuropill, Inc.</td><td class="patent-data-table-td ">Methods for treating conditions caused by higher-than-normal dopaminergic activity in basal ganglia</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8691814">US8691814</a></td><td class="patent-data-table-td patent-date-value">Nov 2, 2010</td><td class="patent-data-table-td patent-date-value">Apr 8, 2014</td><td class="patent-data-table-td ">Eisai R&amp;D Management Co., Ltd.</td><td class="patent-data-table-td ">Deazapurines and uses thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8715683">US8715683</a></td><td class="patent-data-table-td patent-date-value">Feb 12, 2013</td><td class="patent-data-table-td patent-date-value">May 6, 2014</td><td class="patent-data-table-td ">Immunex Corporation</td><td class="patent-data-table-td ">RANK ligand polypeptides</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8741864">US8741864</a></td><td class="patent-data-table-td patent-date-value">Dec 9, 2011</td><td class="patent-data-table-td patent-date-value">Jun 3, 2014</td><td class="patent-data-table-td ">Adynxx, Inc</td><td class="patent-data-table-td ">Gene expression and pain</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2486924A2?cl=en">EP2486924A2</a></td><td class="patent-data-table-td patent-date-value">Apr 23, 2007</td><td class="patent-data-table-td patent-date-value">Aug 15, 2012</td><td class="patent-data-table-td ">Harbor BioSciences, Inc.</td><td class="patent-data-table-td ">Drugs and uses</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2007134386A1?cl=en">WO2007134386A1</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">May 18, 2007</td><td class="patent-data-table-td patent-date-value">Nov 29, 2007</td><td class="patent-data-table-td ">Univ Queensland</td><td class="patent-data-table-td ">Diagnostic agents and uses therefor</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="classifications"></a><div class="patent-section-header"><span class="patent-section-title">Classifications</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th"> </th><th class="patent-data-table-th"> </th></tr></thead><tr><td class="patent-data-table-td ">U.S. Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=ottaBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc514/defs514.htm&usg=AFQjCNG3QPK1Vhbpe6jO1mfvD3pMcfCuxA#C514S04400R">514/44.00R</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=ottaBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S372300">435/372.3</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=ottaBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S370000">435/370</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=ottaBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S372000">435/372</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=ottaBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S455000">435/455</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=ottaBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S366000">435/366</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=ottaBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S372200">435/372.2</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=ottaBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S325000">435/325</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=ottaBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S006130">435/6.13</a></span></td></tr><tr><td class="patent-data-table-td ">International Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=ottaBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C07K0014160000">C07K14/16</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=ottaBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12Q0001680000">C12Q1/68</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=ottaBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0015850000">C12N15/85</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=ottaBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=G01N0033680000">G01N33/68</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=ottaBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C07K0014470000">C07K14/47</a></span></td></tr><tr><td class="patent-data-table-td ">Cooperative Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=ottaBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N2830/15">C12N2830/15</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=ottaBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N2800/108">C12N2800/108</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=ottaBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=G01N2500/02">G01N2500/02</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=ottaBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=G01N33/6872">G01N33/6872</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=ottaBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C07K14/4702">C07K14/4702</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=ottaBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12Q1/6813">C12Q1/6813</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=ottaBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N2830/00">C12N2830/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=ottaBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C07K14/005">C07K14/005</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=ottaBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N15/85">C12N15/85</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=ottaBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12Q1/6897">C12Q1/6897</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=ottaBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C07K2319/00">C07K2319/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=ottaBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N2740/16022">C12N2740/16022</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=ottaBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N2830/008">C12N2830/008</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=ottaBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N2830/30">C12N2830/30</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=ottaBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12Q1/68">C12Q1/68</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=ottaBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N2830/85">C12N2830/85</a></span></td></tr><tr><td class="patent-data-table-td ">European Classification</td><td class="patent-data-table-td "><span class="nested-value">C07K14/005</span>, <span class="nested-value">C12Q1/68P</span>, <span class="nested-value">C12Q1/68</span>, <span class="nested-value">C07K14/47A1</span>, <span class="nested-value">C12Q1/68B</span>, <span class="nested-value">C12N15/85</span>, <span class="nested-value">G01N33/68F</span></td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="legal-events"></a><div class="patent-section-header"><span class="patent-section-title">Legal Events</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Date</th><th class="patent-data-table-th">Code</th><th class="patent-data-table-th">Event</th><th class="patent-data-table-th">Description</th></tr></thead><tr><td class="patent-data-table-td patent-date-value">Jan 31, 2014</td><td class="patent-data-table-td ">REMI</td><td class="patent-data-table-td ">Maintenance fee reminder mailed</td><td class="patent-data-table-td "></td></tr><tr><td class="patent-data-table-td patent-date-value">May 17, 2011</td><td class="patent-data-table-td ">B1</td><td class="patent-data-table-td ">Reexamination certificate first reexamination</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">THE PATENTABILITY OF CLAIMS 19, 30, 41, 44-46, 52, 63, 74, 87, 98, 101-103, 108, 111-113, 118, 121-123, 130-132, 138, 141-143, 148, 151-153, 158, 161-163, 169-171, 176, 179-181, 186-191, 194-196 AND 202 IS CONFIRMED. CLAIMS 1-18, 20-29, 31-40, 42-43, 47-51, 53-62, 64-73, 75-86, 88-97, 99-100, 104-107, 109-110, 114-117, 119-120, 124-129, 133-137, 139-140, 144-147, 149-150, 154-157, 159-160, 164-168, 172-175, 177-178, 182-185, 192-193, 197-201 AND 203-211 ARE CANCELLED.</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Nov 25, 2009</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">8</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Oct 31, 2006</td><td class="patent-data-table-td ">RR</td><td class="patent-data-table-td ">Request for reexamination filed</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20060407</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Apr 25, 2006</td><td class="patent-data-table-td ">RR</td><td class="patent-data-table-td ">Request for reexamination filed</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20051202</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Sep 7, 2005</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">4</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">May 31, 2005</td><td class="patent-data-table-td ">RR</td><td class="patent-data-table-td ">Request for reexamination filed</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20050404</span></div></td></tr></table><div class="patent-section-footer"></div></div><div class="modal-dialog" id="patent-images-lightbox"><div class="patent-lightbox-controls"><div class="patent-lightbox-rotate-controls"><div class="patent-lightbox-rotation-text">Rotate</div><div class="rotate-icon rotate-ccw-icon"></div><div class="rotate-icon rotate-cw-icon"></div></div><div class="patent-lightbox-index-counter"></div><a class="patent-lightbox-fullsize-link" target="_blank">Original Image</a><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_right.png" alt="Next page"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_left.png" alt="Previous page"width="21" height="21" /></div></div><div class="modal-dialog-content"><div class="patent-lightbox-image-holder"><div class="patent-lightbox-placeholder"></div></div></div></div><script>_OC_initPatentsAtb({image_not_available_html: " Image not available"});</script></div></div></div></td></tr></table><script>(function() {var href = window.location.href;if (href.indexOf('?') !== -1) {var parameters = href.split('?')[1].split('&');for (var i = 0; i < parameters.length; i++) {var param = parameters[i].split('=');if (param[0] == 'focus') {var elem = document.getElementById(param[1]);if (elem) {elem.focus();}}}}})();</script><script>_OC_addFlags({LockSrc:"/books/javascript/lock_8a2b04e7bf975d5171d8e4c0b6365c7a.js", Host:"http://www.google.com/", IsBooksRentalEnabled:1, IsWebstoreDisplayCaseEnabled:1, IsObfuscationEnabled:1, IsBrowsingHistoryEnabled:1, IsWebReaderSvgEnabled:0, IsGeoLayerEnabled:1, IsImageModeNotesEnabled:1, IsCopyMenuItemEnabled:1, IsGiftingEnabled:0, IsWebReaderUniversalPaginatorEnabled:0, IsOfflineBubbleEnabled:1, IsReaderEnabledForPlayRequests:1, IsFutureOnSaleVolumesEnabled:1, IsOfflineRestrictedCopyEnabled:1, IsBooksUnifiedLeftNavEnabled:1, IsRestrictedCopyEnabled:1, IsZipitFolderCollectionEnabled:1, IsEndOfSampleRecommendationsEnabled:1, IsRatingsOnBookcardsEnabled:1, IsAdsDisabled:0, IsIframePageDisplayEnabled:0, IsEmbeddedMediaEnabled:1, IsImageModeAnnotationsEnabled:1, IsMyLibraryGooglePlusEnabled:1, IsImagePageProviderEnabled:0, IsBookcardListPriceSmall:0, IsInternalUser:0, IsBooksShareButtonEnabled:0, IsPreOrdersEnabled:0, IsDisabledRandomBookshelves:0, WebstoreDisplayCasePosition:3});_OC_Run({"enable_p13n":false,"add_vol_to_collection_base_url":"http://www.google.com/patents?op=add\u0026sig=ACfU3U3x1L_9d1LQBEeDR6Dv348se0oLpQ\u0026id=ottaBAABERAJ","remove_vol_from_collection_base_url":"http://www.google.com/patents?op=remove\u0026sig=ACfU3U33_ROnRUHZv_filYr5UAaWldJpxw\u0026id=ottaBAABERAJ","logged_in":false,"p13n_save_user_settings_url":"http://www.google.com/patents?op=edit_user_settings\u0026sig=ACfU3U0ao-Z8pgwDkVY99G2f5jv25UAqrA","is_cobrand":false,"sign_in_url":"https://www.google.com/accounts/Login?service=\u0026continue=http://www.google.com/patents%3Fhl%3Den\u0026hl=en","is_play_enabled":true}, {"volume_id":"","is_ebook":true,"volumeresult":{"has_flowing_text":false,"has_scanned_text":true,"can_download_pdf":false,"can_download_epub":false,"is_pdf_drm_enabled":false,"is_epub_drm_enabled":false,"download_pdf_url":"http://www.google.com/patents/download/Nuclear_factors_associated_with_transcri.pdf?id=ottaBAABERAJ\u0026output=pdf\u0026sig=ACfU3U33jZNANNS_5TZ0arEy-O2Cnna4mQ"},"sample_url":"http://www.google.com/patents/reader?id=ottaBAABERAJ\u0026printsec=frontcover\u0026output=reader\u0026source=gbs_atb_hover","is_browsable":true,"is_public_domain":true}, {});</script><div id="footer_table" style="font-size:83%;text-align:center;position:relative;top:20px;height:4.5em;margin-top:2em"><div style="margin-bottom:8px"><a href=http://www.google.com/><nobr>Google&nbsp;Home</nobr></a> - <a href=//www.google.com/patents/sitemap/><nobr>Sitemap</nobr></a> - <a href=http://www.google.com/googlebooks/uspto.html><nobr>USPTO Bulk Downloads</nobr></a> - <a href=/intl/en/privacy/><nobr>Privacy Policy</nobr></a> - <a href=/intl/en/policies/terms/><nobr>Terms of Service</nobr></a> - <a href=https://support.google.com/faqs/answer/2539193?hl=en><nobr>About Google Patents</nobr></a> - <a href="http://www.google.com/tools/feedback/intl/en/error.html" onclick="try{_OC_startFeedback({productId: '72792',locale: 'en'});return false;}catch(e){}"><nobr>Send Feedback</nobr></a></div><span>Data provided by IFI CLAIMS Patent Services</span><br><span >&copy;2012 Google</span></div> <script type="text/javascript">var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));</script><script type="text/javascript">var pageTracker = _gat._getTracker("UA-27188110-1");pageTracker._setCookiePath("/patents/");pageTracker._trackPageview();</script> </body></html>