<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN""http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html><head><title>Patent US6414133 - Complementary to breakpoint region of dna but not whole chromosome ... - Google Patents</title><script>(function(){(function(){function e(a){this.t={};this.tick=function(a,c,b){var d=void 0!=b?b:(new Date).getTime();this.t[a]=[d,c];if(void 0==b)try{window.console.timeStamp("CSI/"+a)}catch(e){}};this.tick("start",null,a)}var a;window.performance&&(a=window.performance.timing);var f=a?new e(a.responseStart):new e;window.jstiming={Timer:e,load:f};if(a){var c=a.navigationStart,d=a.responseStart;0<c&&d>=c&&(window.jstiming.srt=d-c)}if(a){var b=window.jstiming.load;0<c&&d>=c&&(b.tick("_wtsrt",void 0,c),b.tick("wtsrt_",
"_wtsrt",d),b.tick("tbsd_","wtsrt_"))}try{a=null,window.chrome&&window.chrome.csi&&(a=Math.floor(window.chrome.csi().pageT),b&&0<c&&(b.tick("_tbnd",void 0,window.chrome.csi().startE),b.tick("tbnd_","_tbnd",c))),null==a&&window.gtbExternal&&(a=window.gtbExternal.pageT()),null==a&&window.external&&(a=window.external.pageT,b&&0<c&&(b.tick("_tbnd",void 0,window.external.startE),b.tick("tbnd_","_tbnd",c))),a&&(window.jstiming.pt=a)}catch(g){}})();})();
</script><link rel="stylesheet" href="/patents/css/_8a2b04e7bf975d5171d8e4c0b6365c7a/kl_intl_patents_bundle.css" type="text/css" /><script src="/books/javascript/atb_8a2b04e7bf975d5171d8e4c0b6365c7a__en.js"></script><script>function googleTranslateElementInit() {new google.translate.TranslateElement({pageLanguage: "en",gaTrack: true,gaId: "UA-27188110-1",multilanguagePage: true});}</script><script src="//translate.google.com/translate_a/element.js?cb=googleTranslateElementInit"></script><meta name="DC.type" content="Patent"><meta name="DC.title" content="Complementary to breakpoint region of dna but not whole chromosome; detecting translocations; cancer and genetic disorder diagnosis and monitoring"><meta name="DC.contributor" content="Jeanne Dietz-Band" scheme="inventor"><meta name="DC.contributor" content="Wang-Ting Hsieh" scheme="inventor"><meta name="DC.contributor" content="John F. Connaughton" scheme="inventor"><meta name="DC.contributor" content="Ventana Medical Systems, Inc." scheme="assignee"><meta name="DC.date" content="1998-10-13" scheme="dateSubmitted"><meta name="DC.description" content="The invention is directed to a DNA probe set, the probe set comprising a first probe set and a second probe set, the first probe set being sufficient in length and substantially complementary to an entire breakpoint region of a first DNA and nucleotides breakpoint region but less than an entire chromosome such that the first probe set will hybridize to both sides of the breakpoint region regardless of whether the first DNA has been broken in the breakpoint region and either end fused to another DNA, and the second probe set being sufficient in length and substantially complementary to an entire breakpoint region of a second DNA and nucleotides on both sides of the breakpoint region but less than an entire chromosome such that the second probe set will hybridize to both sides of the breakpoint region regardless of whether the second DNA has been broken in the breakpoint region and either end fused to another DNA. Diagnostic kits utilizing the probe sets of the invention are also claimed."><meta name="DC.date" content="2002-7-2" scheme="issued"><meta name="DC.relation" content="US:4999290" scheme="references"><meta name="DC.relation" content="US:5487970" scheme="references"><meta name="DC.relation" content="US:5538869" scheme="references"><meta name="DC.relation" content="US:5567586" scheme="references"><meta name="citation_reference" content="1996/1997 Catalog, Vysis, Inc. 3100 Woodcreek Drive, Downers Grove, IL pp. 25 and 30."><meta name="citation_reference" content="Dewald et al, Blood, 91(9); pp. 3357-3365 (May 1, 1998)."><meta name="citation_reference" content="Dewald et al. Highly sensitive fluorescence in situ hybridization method to detect double BCR/ABL fusion and monitor response to therapy in chronic myeloid leukemia. Blood. vol. 91, No. 9, pp. 3357-3365, May 1998.*"><meta name="citation_reference" content="Fischer et al. Design and Validation of DNA Probe Sets for a Comprehensive Interphase Cytogenetic Analysis of Acute Myeloid Leukemia. Blood. vol. 88, pp. 3962-3971, Nov. 1996.*"><meta name="citation_reference" content="Oncor 1992-1993 Product Catalog, p. 11 Oncor, Inc. 209 Perry Pky. Gaithersburg, MD."><meta name="citation_reference" content="Paskulin et al, Genes, Chromosomes &amp; Cancer 21: 144-151 (1998)."><meta name="citation_reference" content="Paskulin et al. Pre-Clinical Evaluation of Probes to detect t(8;21) AML minimal residual disease by fluorescence in situ hybridization. Genes, Chromosome and Cancer vol. 21, pp. 144-151, Feb. 1998."><meta name="citation_reference" content="TKachuk et al. Detection of BCR-ABL Fusion in Chronic Myelogeneous Leukemia by in situ Hybridization. Science. vol. 250, pp. 559-562, Nov. 1990.*"><meta name="citation_reference" content="TKachuk et al. Detection of BCR-ABL Fusion in Chronic Myelogeneous Leukemia by in situ Hybridization. Science. vol. 250, pp. 559-562, Oct. 1990.*"><meta name="citation_patent_number" content="US:6414133"><meta name="citation_patent_application_number" content="US:09/170,630"><link rel="canonical" href="http://www.google.com/patents/US6414133"/><meta property="og:url" content="http://www.google.com/patents/US6414133"/><meta name="title" content="Patent US6414133 - Complementary to breakpoint region of dna but not whole chromosome; detecting translocations; cancer and genetic disorder diagnosis and monitoring"/><meta name="description" content="The invention is directed to a DNA probe set, the probe set comprising a first probe set and a second probe set, the first probe set being sufficient in length and substantially complementary to an entire breakpoint region of a first DNA and nucleotides breakpoint region but less than an entire chromosome such that the first probe set will hybridize to both sides of the breakpoint region regardless of whether the first DNA has been broken in the breakpoint region and either end fused to another DNA, and the second probe set being sufficient in length and substantially complementary to an entire breakpoint region of a second DNA and nucleotides on both sides of the breakpoint region but less than an entire chromosome such that the second probe set will hybridize to both sides of the breakpoint region regardless of whether the second DNA has been broken in the breakpoint region and either end fused to another DNA. Diagnostic kits utilizing the probe sets of the invention are also claimed."/><meta property="og:title" content="Patent US6414133 - Complementary to breakpoint region of dna but not whole chromosome; detecting translocations; cancer and genetic disorder diagnosis and monitoring"/><meta property="og:type" content="book"/><meta property="og:site_name" content="Google Books"/><meta property="og:image" content="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><link rel="image_src" href="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><script>(function(){try{var aa=function(a,b,c,d){d=d||{};d._sn=["cfg",b,c].join(".");window.gbar.logger.ml(a,d)};var g=window.gbar=window.gbar||{},l=window.gbar.i=window.gbar.i||{},m={},n;function _tvn(a,b){var c=parseInt(a,10);return isNaN(c)?b:c}function _tvf(a,b){var c=parseFloat(a);return isNaN(c)?b:c}function _tvv(a){return!!a}function p(a,b,c){(c||g)[a]=b}g.bv={n:_tvn("2",0),r:"",f:".67.",e:"0",m:_tvn("0",1)};
function q(a,b,c){var d="on"+b;if(a.addEventListener)a.addEventListener(b,c,!1);else if(a.attachEvent)a.attachEvent(d,c);else{var f=a[d];a[d]=function(){var a=f.apply(this,arguments),b=c.apply(this,arguments);return void 0==a?b:void 0==b?a:b&&a}}}var s=function(a){return function(){return g.bv.m==a}},ba=s(1),ca=s(2);p("sb",ba);p("kn",ca);l.a=_tvv;l.b=_tvf;l.c=_tvn;l.i=aa;var da=window.gbar.i.i;var t,u,v,w;function ea(a){v=a}function fa(a){var b;if(b=v&&window.encodeURIComponent)b=a.href,b=!b.match(/^http[s]?:\/\/accounts\.google\.[^/]*\/ClearSID/i)&&!b.match(/^http[s]?:\/\/[^/]*\/accounts\/ClearSID/i);if(b=b&&encodeURIComponent(v()))a.href=a.href.replace(/([?&]continue=)[^&]*/,"$1"+b)}function ga(a){window.gApplication&&(a.href=window.gApplication.getTabUrl(a.href))}
function ha(a){var b=document.forms[0].q,c=window.encodeURIComponent&&b&&b.value,b=b&&b.placeholder;c&&c!=b&&(a.href=a.href.replace(/([?&])q=[^&]*|$/,function(a,b){return(b||"&")+"q="+encodeURIComponent(c)}))}n=l.a("")?ga:ha;
function x(a,b,c,d,f,e){var h=document.getElementById(a);if(h){var k=h.style;k.left=d?"auto":b+"px";k.right=d?b+"px":"auto";k.top=c+"px";k.visibility=u?"hidden":"visible";f&&e?(k.width=f+"px",k.height=e+"px"):(x(t,b,c,d,h.offsetWidth,h.offsetHeight),u=u?"":a)}}
var y=[],ia=function(a,b){y.push(b)},ja=function(a){a=a||window.event;var b=a.target||a.srcElement;a.cancelBubble=!0;null==t&&(a=document.createElement(Array.every||window.createPopup?"iframe":"div"),a.frameBorder="0",t=a.id="gbs",a.src="javascript:''",b.parentNode.appendChild(a),q(document,"click",z));var c=b,b=0;"gb3"!=c.className&&(c=c.parentNode);a=c.getAttribute("aria-owns")||"gbi";var d=c.offsetWidth,f=20<c.offsetTop?46:24;document.getElementById("tphdr")&&(f-=3);var e=!1;do b+=c.offsetLeft||
0;while(c=c.offsetParent);var c=(document.documentElement.clientWidth||document.body.clientWidth)-b-d,h,d=document.body,k=document.defaultView;k&&k.getComputedStyle?(d=k.getComputedStyle(d,""))&&(h=d.direction):h=d.currentStyle?d.currentStyle.direction:d.style.direction;h="rtl"==h;if("gbi"==a){for(d=0;k=y[d++];)k();A(null,window.navExtra);h&&(b=c,e=!0)}else h||(b=c,e=!0);u!=a&&z();x(a,b,f,e)},z=function(){u&&x(u,0,0)},A=function(a,b){var c,d=document.getElementById("gbi"),f=a;f||(f=d.firstChild);
for(;b&&(c=b.pop());){var e=d,h=c,k=f;w||(w="gb2");e.insertBefore(h,k).className=w}},ka=function(a,b,c){if((b=document.getElementById(b))&&a){a.className="gb4";var d=document.createElement("span");d.appendChild(a);d.appendChild(document.createTextNode(" | "));d.id=c;b.appendChild(d)}},la=function(){return document.getElementById("gb_70")},ma=function(){return!!u};p("qs",n);p("setContinueCb",ea);p("pc",fa);p("tg",ja);p("close",z);p("addLink",ka);p("almm",A);p("si",la);p("adh",ia);p("op",ma);var B=function(){},C=function(){},F=function(a){var b=new Image,c=D;b.onerror=b.onload=b.onabort=function(){try{delete E[c]}catch(a){}};E[c]=b;b.src=a;D=c+1},E=[],D=0;p("logger",{il:C,ml:B,log:F});var G=window.gbar.logger;var H={},na={},I=[],oa=l.b("0.1",.1),pa=l.a("1",!0),qa=function(a,b){I.push([a,b])},ra=function(a,b){H[a]=b},sa=function(a){return a in H},J={},K=function(a,b){J[a]||(J[a]=[]);J[a].push(b)},ta=function(a){K("m",a)},L=function(a,b){var c=document.createElement("script");c.src=a;c.async=pa;Math.random()<oa&&(c.onerror=function(){c.onerror=null;B(Error("Bundle load failed: name="+(b||"UNK")+" url="+a))});(document.getElementById("xjsc")||document.getElementsByTagName("body")[0]||
document.getElementsByTagName("head")[0]).appendChild(c)},N=function(a){for(var b=0,c;(c=I[b])&&c[0]!=a;++b);!c||c[1].l||c[1].s||(c[1].s=!0,M(2,a),c[1].url&&L(c[1].url,a),c[1].libs&&m.d&&m.d(c[1].libs))},O=function(a){K("gc",a)},P=null,ua=function(a){P=a},M=function(a,b,c){if(P){a={t:a,b:b};if(c)for(var d in c)a[d]=c[d];try{P(a)}catch(f){}}};p("mdc",H);p("mdi",na);p("bnc",I);p("qGC",O);p("qm",ta);p("qd",J);p("lb",N);p("mcf",ra);p("bcf",qa);p("aq",K);p("mdd","");p("has",sa);
p("trh",ua);p("tev",M);var Q=l.b("0.1",.001),R=0;
function _mlToken(a,b){try{if(1>R){R++;var c,d=a,f=b||{},e=encodeURIComponent,h=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&jexpid=",e("17483"),"&srcpg=",e("prop=22"),"&jsr=",Math.round(1/Q),"&ogev=",e("WqTtU66XCon9oAS4qoHYCA"),"&ogf=",g.bv.f,"&ogrp=",e("1"),"&ogv=",e("1407723702.0"),"&oggv="+e("es_plusone_gc_20140723.0_p0"),"&ogd=",e("com"),"&ogc=",e("NZL"),"&ogl=",e("en")];f._sn&&(f._sn="og."+
f._sn);for(var k in f)h.push("&"),h.push(e(k)),h.push("="),h.push(e(f[k]));h.push("&emsg=");h.push(e(d.name+":"+d.message));var r=h.join("");S(r)&&(r=r.substr(0,2E3));c=r;var Aa=window.gbar.logger._aem(a,c);F(Aa)}}catch(Na){}}var S=function(a){return 2E3<=a.length},va=function(a,b){return b};function T(a){B=a;p("_itl",S,G);p("_aem",va,G);p("ml",B,G);a={};H.er=a}l.a("")?T(function(a){throw a;}):l.a("1")&&Math.random()<Q&&T(_mlToken);I.push(["m",{url:"//ssl.gstatic.com/gb/js/scm_7385cc5883250b43a39405734c1bea59.js"}]);g.mcf("c",{});g.sg={c:""};if(l.a("1")){var wa=l.a("");I.push(["gc",{auto:wa,url:"//ssl.gstatic.com/gb/js/abc/gci_91f30755d6a6b787dcc2a4062e6e9824.js",libs:"googleapis.client:plusone:gapi.iframes"}]);var xa={version:"gci_91f30755d6a6b787dcc2a4062e6e9824.js",index:"",lang:"en"};H.gc=xa;var U=function(a){window.googleapis&&window.iframes?a&&a():(a&&O(a),N("gc"))};p("lGC",U);l.a("1")&&p("lPWF",U)};window.__PVT="";if(l.a("1")&&l.a("1")){var V=function(a){U(function(){K("pw",a);N("pw")})};p("lPW",V);I.push(["pw",{url:"//ssl.gstatic.com/gb/js/abc/pwm_45f73e4df07a0e388b0fa1f3d30e7280.js"}]);var W=[],ya=function(a){W[0]=a},za=function(a,b){var c=b||{};c._sn="pw";B(a,c)},Ba={signed:W,elog:za,base:"https://plusone.google.com/u/0",loadTime:(new Date).getTime()};H.pw=Ba;var X=function(a,b){for(var c=b.split("."),d=function(){var b=arguments;a(function(){for(var a=g,d=0,e=c.length-1;d<e;++d)a=a[c[d]];a[c[d]].apply(a,b)})},f=g,e=0,h=c.length-1;e<h;++e)f=
f[c[e]]=f[c[e]]||{};return f[c[e]]=d};X(V,"pw.clk");X(V,"pw.hvr");p("su",ya,g.pw)};function Ca(){function a(){for(var b;(b=e[h++])&&"m"!=b[0]&&!b[1].auto;);b&&(M(2,b[0]),b[1].url&&L(b[1].url,b[0]),b[1].libs&&m.d&&m.d(b[1].libs));h<e.length&&setTimeout(a,0)}function b(){0<f--?setTimeout(b,0):a()}var c=l.a("1"),d=l.a(""),f=3,e=I,h=0,k=window.gbarOnReady;if(k)try{k()}catch(r){da(r,"ml","or")}d?p("ldb",a):c?q(window,"load",b):b()}p("rdl",Ca);var Da={D:1,H:2,da:3,p:4,W:5,M:6,F:7,g:8,ha:9,U:10,L:11,T:12,S:13,N:14,Q:15,P:16,fa:17,w:18,O:19,ga:20,ea:21,u:22,G:23,ja:24,ka:25,ia:26,A:27,j:28,o:29,k:30,ca:31,Z:32,$:33,J:34,K:35,ba:36,aa:37,Y:38,B:39,R:40,v:41,X:42,V:43,h:48,C:49,I:500},Y=[1,2,3,4,5,6,9,10,11,13,14,28,29,30,34,35,37,38,39,40,41,42,43,48,49,500];var Z=l.b("0.001",1E-4),Ea=l.b("1",1),Fa=!1,Ga=!1;if(l.a("1")){var Ha=Math.random();Ha<=Z&&(Fa=!0);Ha<=Ea&&(Ga=!0)}var Ia=Da,$=null;function Ja(){var a=0,b=function(b,d){l.a(d)&&(a|=b)};b(1,"");b(2,"");b(4,"");b(8,"");return a}
function Ka(a,b){var c=Z,d=Fa,f;f=a;if(!$){$={};for(var e=0;e<Y.length;e++){var h=Y[e];$[h]=!0}}if(f=!!$[f])c=Ea,d=Ga;if(d){d=encodeURIComponent;g.rp?(f=g.rp(),f="-1"!=f?f:"1"):f="1";c=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&oge=",a,"&ogex=",d("17483"),"&ogev=",d("WqTtU66XCon9oAS4qoHYCA"),"&ogf=",g.bv.f,"&ogp=",d("22"),"&ogrp=",d(f),"&ogsr=",Math.round(1/c),"&ogv=",d("1407723702.0"),"&oggv="+
d("es_plusone_gc_20140723.0_p0"),"&ogd=",d("com"),"&ogl=",d("en"),"&ogc=",d("NZL"),"&ogus=",Ja()];if(b){"ogw"in b&&(c.push("&ogw="+b.ogw),delete b.ogw);var k;f=b;e=[];for(k in f)0!=e.length&&e.push(","),e.push(La(k)),e.push("."),e.push(La(f[k]));k=e.join("");""!=k&&(c.push("&ogad="),c.push(d(k)))}F(c.join(""))}}function La(a){"number"==typeof a&&(a+="");return"string"==typeof a?a.replace(".","%2E").replace(",","%2C"):a}C=Ka;p("il",C,G);var Ma={};H.il=Ma;setTimeout(function(){C(Ia.g)},0);}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var b=window.gbar.i.i;var c=window.gbar;var f=function(d){try{var a=document.getElementById("gbom");a&&d.appendChild(a.cloneNode(!0))}catch(e){b(e,"omas","aomc")}};c.aomc=f;}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var a=window.gbar;a.mcf("pm",{p:""});}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{window.gbar.rdl();}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
if (window['_OC_timingAction']) {window['_OC_timingAction']('patents_refpage');}</script><style>#gbar,#guser{font-size:13px;padding-top:1px !important;}#gbar{float:left;height:22px}#guser{padding-bottom:7px !important;text-align:right}.gbh,.gbd{border-top:1px solid #c9d7f1;font-size:1px}.gbh{height:0;position:absolute;top:24px;width:100%}#gbs,.gbm{background:#fff;left:0;position:absolute;text-align:left;visibility:hidden;z-index:1000}.gbm{border:1px solid;border-color:#c9d7f1 #36c #36c #a2bae7;z-index:1001}.gb1{margin-right:.5em}.gb1,.gb3{zoom:1}.gb2{display:block;padding:.2em .5em}.gb2,.gb3{text-decoration:none !important;border-bottom:none}a.gb1,a.gb4{text-decoration:underline !important}a.gb1,a.gb2,a.gb3,a.gb4{color:#00c !important}.gbi .gb3,.gbi .gb2,.gbi .gb4{color:#dd8e27 !important}.gbf .gb3,.gbf .gb2,.gbf .gb4{color:#900 !important}a.gb2:hover{background:#36c;color:#fff !important}#gbar .gbz0l{color:#000 !important;cursor:default;font-weight:bold;text-decoration:none !important}
#gbar { padding:.3em .6em !important;}</style></head><body  topmargin="3" marginheight="3"><div id=gbar><nobr><a onclick=gbar.qs(this);gbar.logger.il(1,{t:1}); class=gb1 id=gb_1 href="https://www.google.com/search?sa=N&tab=tw">Search</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:2}); class=gb1 id=gb_2 href="http://www.google.com/search?hl=en&tbm=isch&source=og&sa=N&tab=ti">Images</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:8}); class=gb1 id=gb_8 href="http://maps.google.com/maps?hl=en&sa=N&tab=tl">Maps</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:78}); class=gb1 id=gb_78 href="https://play.google.com/?hl=en&sa=N&tab=t8">Play</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:36}); class=gb1 id=gb_36 href="http://www.youtube.com/results?sa=N&tab=t1">YouTube</a> <a onclick=gbar.logger.il(1,{t:5}); class=gb1 id=gb_5 href="http://news.google.com/nwshp?hl=en&tab=tn">News</a> <a onclick=gbar.logger.il(1,{t:23}); class=gb1 id=gb_23 href="https://mail.google.com/mail/?tab=tm">Gmail</a> <a onclick=gbar.logger.il(1,{t:25}); class=gb1 id=gb_25 href="https://drive.google.com/?tab=to">Drive</a> <a class=gb3 href="http://www.google.com/intl/en/options/" onclick="this.blur();gbar.tg(event);return !1" aria-haspopup=true><u>More</u> <small>&#9660;</small></a><div class=gbm id=gbi><a onclick=gbar.logger.il(1,{t:24}); class=gb2 id=gb_24 href="https://www.google.com/calendar?tab=tc">Calendar</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:51}); class=gb2 id=gb_51 href="http://translate.google.com/?hl=en&sa=N&tab=tT">Translate</a><a onclick=gbar.logger.il(1,{t:17}); class=gb2 id=gb_17 href="http://www.google.com/mobile/?hl=en&tab=tD">Mobile</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:10}); class=gb2 id=gb_10 href="http://www.google.com/search?hl=en&tbo=u&tbm=bks&source=og&sa=N&tab=tp">Books</a><a onclick=gbar.logger.il(1,{t:212}); class=gb2 id=gb_212 href="https://wallet.google.com/manage/?tab=ta">Wallet</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:6}); class=gb2 id=gb_6 href="http://www.google.com/search?hl=en&tbo=u&tbm=shop&source=og&sa=N&tab=tf">Shopping</a><a onclick=gbar.logger.il(1,{t:30}); class=gb2 id=gb_30 href="http://www.blogger.com/?tab=tj">Blogger</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:27}); class=gb2 id=gb_27 href="http://www.google.com/finance?sa=N&tab=te">Finance</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:31}); class=gb2 id=gb_31 href="https://plus.google.com/photos?sa=N&tab=tq">Photos</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:12}); class=gb2 id=gb_12 href="http://www.google.com/search?hl=en&tbo=u&tbm=vid&source=og&sa=N&tab=tv">Videos</a><div class=gb2><div class=gbd></div></div><a onclick=gbar.logger.il(1,{t:66}); href="http://www.google.com/intl/en/options/" class=gb2>Even more &raquo;</a></div></nobr></div><div id=guser width=100%><nobr><span id=gbn class=gbi></span><span id=gbf class=gbf></span><span id=gbe></span><a target=_top id=gb_70 href="https://www.google.com/accounts/Login?service=&continue=http://www.google.com/patents%3Fhl%3Den&hl=en" class=gb4>Sign in</a><div style="display: none"><div class=gbm id=gbd5 aria-owner=gbg5><div class=gbmc><ol id=gbom class=gbmcc></ol></div></div></div></nobr></div><div class=gbh style=left:0></div><div class=gbh style=right:0></div><div role="alert" style="position: absolute; left: 0; right: 0;"><a href="http://www.google.com/patents/us6414133?hl=en&amp;output=html_text" title="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."><img border="0" src="http://www.google.com/images/cleardot.gif"alt="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."></a></div><div id="guser"><nobr></nobr></div><div style="clear:both;"></div><div id="gb-top-search-box" class="gb-top-search-box-small gb-reset"><table><tr><td class="logo"><a href="http://www.google.com/patents" class="logo-link"><img class="logo-img" src="/intl/en/images/logos/google_logo_41.png" alt="Go to Google Books Home" height="41"/></a></td><td><form action="http://www.google.com/search" name="f" id="vheadf" method="get"><span id="hf"></span><input type="hidden" name="tbm" value="pts"/><input type="hidden" name="tbo" value="1"/><input type="hidden" name="hl" value="en"/><table><tr><td><div class="inputs"><table><tr><td><div class="text-input"><input type="text" name="q" id="vheadq" class="text" maxlength="2048" size="31" value="" title="Search Patents" accesskey="s" autocomplete="off"/><script>window._OC_autoDir &&window._OC_autoDir('vheadq', 'tia-vheadq');</script></div></td><td><div class="submit-input"><input name="btnG" class="submit" type="submit" value=""/></div></td></tr></table></div></td><td class="col-ext-links"><div class="ext-links"><a href="http://www.google.com/advanced_patent_search">&lt;nobr&gt;Advanced Patent Search&lt;/nobr&gt;</a></div></td></tr></table></form></td></tr></table></div><div class="kd-appbar"><h2 class="kd-appname"><a href="/patents">Patents</a></h2><div class="kd-buttonbar left" id="left-toolbar-buttons"><a id="appbar-write-review-link" href=""></a><a id="appbar-view-print-sample-link" href=""></a><a id="appbar-view-ebook-sample-link" href=""></a><a id="appbar-patents-prior-art-finder-link" href="https://www.google.com/patents/related/US6414133"></a><a id="appbar-patents-discuss-this-link" href="http://www.google.com/url?id=z89aBAABERAJ&amp;q=http://patents.stackexchange.com/redirect/google-patents%3Fpatent%3DUS6414133&amp;usg=AFQjCNH8Aa2MRXf1C4twhpSRc40VH61yfA" data-is-grant="true"></a><a id="appbar-read-patent-link" href="//docs.google.com/viewer?url=patentimages.storage.googleapis.com/pdfs/US6414133.pdf"></a><a id="appbar-download-pdf-link" href="//patentimages.storage.googleapis.com/pdfs/US6414133.pdf"></a></div><div class="kd-buttonbar right" id="right-toolbar-buttons"></div></div><div id="books-microdata" itemscope=""itemtype="http://schema.org/Book"itemid="http://www.google.com/patents/US6414133" style="display:none"><span itemprop="description">The invention is directed to a DNA probe set, the probe set comprising a first probe set and a second probe set, the first probe set being sufficient in length and substantially complementary to an entire breakpoint region of a first DNA and nucleotides breakpoint region but less than an entire chromosome...</span><span itemprop="url">http://www.google.com/patents/US6414133?utm_source=gb-gplus-share</span><span class="main-title" itemprop="name">Patent US6414133 - Complementary to breakpoint region of dna but not whole chromosome; detecting translocations; cancer and genetic disorder diagnosis and monitoring</span><img itemprop="image" src="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"alt="Patent US6414133 - Complementary to breakpoint region of dna but not whole chromosome; detecting translocations; cancer and genetic disorder diagnosis and monitoring" title="Patent US6414133 - Complementary to breakpoint region of dna but not whole chromosome; detecting translocations; cancer and genetic disorder diagnosis and monitoring"></div><div style="display: none"><ol id="ofe-gear-menu-contents" class="gbmcc"><li class="gbe gbmtc"><a class="gbmt goog-menuitem-content" id="" href="http://www.google.com/advanced_patent_search">Advanced Patent Search</a></li></ol></div><table id="viewport_table" cellpadding="0" style="clear:both" cellspacing="0"><tr><td id="viewport_td"><div class=vertical_module_list_row><div id=intl_patents class=about_content><div id=intl_patents_v><table class="patent-bibdata"><tr><td class="patent-bibdata-heading">Publication number</td><td class="single-patent-bibdata">US6414133 B1</td></tr><tr><td class="patent-bibdata-heading">Publication type</td><td class="single-patent-bibdata">Grant</td></tr><tr><td class="patent-bibdata-heading">Application number</td><td class="single-patent-bibdata">US 09/170,630</td></tr><tr><td class="patent-bibdata-heading">Publication date</td><td class="single-patent-bibdata">Jul 2, 2002</td></tr><tr><td class="patent-bibdata-heading">Filing date</td><td class="single-patent-bibdata">Oct 13, 1998</td></tr><tr><td class="patent-bibdata-heading">Priority date<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The priority date is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the date listed."></span></td><td class="single-patent-bibdata">Oct 13, 1998</td></tr><tr><td class="patent-bibdata-heading">Fee status<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The fee status is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the status or dates listed."></span></td><td class="single-patent-bibdata">Paid</td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Also published as</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/CA2314426A1">CA2314426A1</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP1044213A1">EP1044213A1</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP1044213A4">EP1044213A4</a>, </span><span class="patent-bibdata-value"><a href="/patents/US7250254">US7250254</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20020182628">US20020182628</a>, </span><span class="patent-bibdata-value"><a href="/patents/WO2000021975A1">WO2000021975A1</a>, </span><span class="patent-bibdata-value"><a href="/patents/WO2000021975A9">WO2000021975A9</a></span></span></td></tr><tr class="patent-bibdata-list-row alternate-patent-number"><td class="patent-bibdata-heading">Publication number</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value">09170630, </span><span class="patent-bibdata-value">170630, </span><span class="patent-bibdata-value">US 6414133 B1, </span><span class="patent-bibdata-value">US 6414133B1, </span><span class="patent-bibdata-value">US-B1-6414133, </span><span class="patent-bibdata-value">US6414133 B1, </span><span class="patent-bibdata-value">US6414133B1</span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Inventors</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Jeanne+Dietz-Band%22">Jeanne Dietz-Band</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Wang-Ting+Hsieh%22">Wang-Ting Hsieh</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22John+F.+Connaughton%22">John F. Connaughton</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Original Assignee</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=inassignee:%22Ventana+Medical+Systems,+Inc.%22">Ventana Medical Systems, Inc.</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Export Citation</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/US6414133.bibtex">BiBTeX</a>, </span><span class="patent-bibdata-value"><a href="/patents/US6414133.enw">EndNote</a>, </span><span class="patent-bibdata-value"><a href="/patents/US6414133.ris">RefMan</a></span></span></td></tr><tr class="patent-internal-links"><td colspan=2><span class="patent-bibdata-value"><a href="#backward-citations">Patent Citations</a> (4),</span> <span class="patent-bibdata-value"><a href="#npl-citations">Non-Patent Citations</a> (9),</span> <span class="patent-bibdata-value"><a href="#forward-citations">Referenced by</a> (13),</span> <span class="patent-bibdata-value"><a href="#classifications">Classifications</a> (12),</span> <span class="patent-bibdata-value"><a href="#legal-events">Legal Events</a> (9)</span> </td></tr><tr><td colspan=2 class="patent-bibdata-external-link-spacer-top"></td></tr><tr class="patent-bibdata-external-link-spacer-bottom"></tr><tr><td colspan=2><span class="patent-bibdata-heading">External Links:&nbsp;</span><span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=z89aBAABERAJ&q=http://patft.uspto.gov/netacgi/nph-Parser%3FSect2%3DPTO1%26Sect2%3DHITOFF%26p%3D1%26u%3D/netahtml/PTO/search-bool.html%26r%3D1%26f%3DG%26l%3D50%26d%3DPALL%26RefSrch%3Dyes%26Query%3DPN/6414133&usg=AFQjCNES_87xjymrw3zjIkun6Jv8czarFw">USPTO</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=z89aBAABERAJ&q=http://assignments.uspto.gov/assignments/q%3Fdb%3Dpat%26pat%3D6414133&usg=AFQjCNHda6dMoWbTB5ZG1KMNV3Dt_4tiZA">USPTO Assignment</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=z89aBAABERAJ&q=http://worldwide.espacenet.com/publicationDetails/biblio%3FCC%3DUS%26NR%3D6414133B1%26KC%3DB1%26FT%3DD&usg=AFQjCNFDgIVbQ7lqhqF3hGMkI8Sv9T64cQ">Espacenet</a></span></span></td></tr><tr class="patent-bibdata-group-spacer"></tr></table><div class="number-and-title"><span class="patent-title"><invention-title mxw-id="PT129032086" lang="EN" load-source="ifi">Complementary to breakpoint region of dna but not whole chromosome; detecting translocations; cancer and genetic disorder diagnosis and monitoring</invention-title></span><br><span class="patent-number">US 6414133 B1</span></div><div class="patent-section patent-abstract-section"><div class="patent-section-header"><span class="patent-section-title">Abstract</span></div><div class="patent-text"><abstract mxw-id="PA50343083" lang="EN" load-source="patent-office"> <div class="abstract">The invention is directed to a DNA probe set, the probe set comprising a first probe set and a second probe set, the first probe set being sufficient in length and substantially complementary to an entire breakpoint region of a first DNA and nucleotides breakpoint region but less than an entire chromosome such that the first probe set will hybridize to both sides of the breakpoint region regardless of whether the first DNA has been broken in the breakpoint region and either end fused to another DNA, and the second probe set being sufficient in length and substantially complementary to an entire breakpoint region of a second DNA and nucleotides on both sides of the breakpoint region but less than an entire chromosome such that the second probe set will hybridize to both sides of the breakpoint region regardless of whether the second DNA has been broken in the breakpoint region and either end fused to another DNA. Diagnostic kits utilizing the probe sets of the invention are also claimed.</div>
  </abstract></div></div><div class="patent-section patent-drawings-section"><div class="patent-section-header"><span class="patent-section-title">Images<span class="patent-section-count">(10)</span></span></div><div class="patent-drawings-body"><div class="patent-drawings-carousel"><div class="drawings"><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6414133B1/US06414133-20020702-D00001.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6414133B1/US06414133-20020702-D00001.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6414133B1/US06414133-20020702-D00002.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6414133B1/US06414133-20020702-D00002.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6414133B1/US06414133-20020702-D00003.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6414133B1/US06414133-20020702-D00003.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6414133B1/US06414133-20020702-D00004.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6414133B1/US06414133-20020702-D00004.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6414133B1/US06414133-20020702-D00005.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6414133B1/US06414133-20020702-D00005.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6414133B1/US06414133-20020702-D00006.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6414133B1/US06414133-20020702-D00006.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6414133B1/US06414133-20020702-D00007.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6414133B1/US06414133-20020702-D00007.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6414133B1/US06414133-20020702-D00008.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6414133B1/US06414133-20020702-D00008.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6414133B1/US06414133-20020702-D00009.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6414133B1/US06414133-20020702-D00009.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6414133B1/US06414133-20020702-D00010.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6414133B1/US06414133-20020702-D00010.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div></div></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img" alt="Previous page"src="/googlebooks/images/kennedy/page_left.png"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img" alt="Next page"src="/googlebooks/images/kennedy/page_right.png"width="21" height="21" /></div></div></div><div class="patent-post-drawings"></div><div class="patent-section patent-claims-section"><div class="patent-section-header"><span class="patent-section-title">Claims<span class="patent-section-count">(19)</span></span></div><div class="patent-text"><div mxw-id="PCLM8318698" lang="EN" load-source="patent-office" class="claims">
    <claim-statement>What is claimed is: </claim-statement> <div class="claim"> <div num="1" id="US-6414133-B1-CLM-00001" class="claim">
      <div class="claim-text">1. A DNA probe set, said probe set comprising a first probe set and a second probe set,</div>
      <div class="claim-text">said first probe set being sufficient in length and substantially complementary to an entire breakpoint region of a first DNA and nucleotides on both sides of the breakpoint region but less than an entire chromosome such that said first probe set will hybridize to both sides of the breakpoint region regardless of whether the first DNA has been broken in the breakpoint region and either end fused to another DNA, and </div>
      <div class="claim-text">said second probe set being sufficient in length and substantially complementary to an entire breakpoint region of a second DNA and nucleotides on both sides of the breakpoint region but less than an entire chromosome such that said second probe set will hybridize to both sides of the breakpoint region regardless of whether the second DNA has been broken in the breakpoint region and either end fused to another DNA. </div>
    </div>
    </div> <div class="claim-dependent"> <div num="2" id="US-6414133-B1-CLM-00002" class="claim">
      <div class="claim-text">2. The probe set of <claim-ref idref="US-6414133-B1-CLM-00001">claim 1</claim-ref>, wherein said probes are detectably labelled.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="3" id="US-6414133-B1-CLM-00003" class="claim">
      <div class="claim-text">3. The probe set of <claim-ref idref="US-6414133-B1-CLM-00002">claim 2</claim-ref>, wherein said first DNA is part of the ABL1 gene on chromosome 9 and the second DNA is part of the BCR gene on chromosome 22.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="4" id="US-6414133-B1-CLM-00004" class="claim">
      <div class="claim-text">4. The probe set of <claim-ref idref="US-6414133-B1-CLM-00002">claim 2</claim-ref>, wherein said first DNA is part of the AML1 gene on chromosome 21 and the second DNA is part of the ETO gene on chromosome 8.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="5" id="US-6414133-B1-CLM-00005" class="claim">
      <div class="claim-text">5. A diagnostic kit for detecting a structural abnormality caused by chromosomal breakage and rearrangement containing a reagent comprising at least one probe set of the probe set according to <claim-ref idref="US-6414133-B1-CLM-00001">claim 1</claim-ref>, and a container containing said reagent.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="6" id="US-6414133-B1-CLM-00006" class="claim">
      <div class="claim-text">6. A diagnostic kit according to <claim-ref idref="US-6414133-B1-CLM-00005">claim 5</claim-ref> further comprising at least two containers, wherein a first container contains a reagent comprising said first probe set and a second container contains a reagent comprising said second probe set.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="7" id="US-6414133-B1-CLM-00007" class="claim">
      <div class="claim-text">7. A diagnostic kit according to <claim-ref idref="US-6414133-B1-CLM-00006">claim 6</claim-ref> wherein said reagent comprises said first and said second probe set.</div>
    </div>
    </div> <div class="claim"> <div num="8" id="US-6414133-B1-CLM-00008" class="claim">
      <div class="claim-text">8. A DNA probe set, said probe set comprising a first probe set and a second probe set,</div>
      <div class="claim-text">said first probe set being sufficient in length and substantially complementary to an entire breakpoint region of a first DNA and nucleotides on both sides of the breakpoint region but less than an entire chromosome such that said first probe set will hybridize to both sides of the breakpoint region regardless of whether a second DNA from a region other than the breakpoint region has been inserted in the breakpoint region, and </div>
      <div class="claim-text">said second probe set being sufficient in length and substantially complementary to a 3′ end and a 5′ end of a second DNA but less than an entire chromosome such that said second probe set will hybridize to both ends of the second DNA regardless of whether the second DNA is inserted in the first DNA. </div>
    </div>
    </div> <div class="claim-dependent"> <div num="9" id="US-6414133-B1-CLM-00009" class="claim">
      <div class="claim-text">9. The probe set of <claim-ref idref="US-6414133-B1-CLM-00008">claim 8</claim-ref>, wherein said probes are detectably labelled.</div>
    </div>
    </div> <div class="claim"> <div num="10" id="US-6414133-B1-CLM-00010" class="claim">
      <div class="claim-text">10. A DNA probe set, said probe set comprising a first probe set and a second probe set,</div>
      <div class="claim-text">said first probe set being sufficient in length and substantially complementary to nucleotides on both sides of the breakpoint region of a first DNA but less than an entire chromosome such that said first probe set will hybridize to both sides of the breakpoint region regardless of whether the first DNA has been broken in the breakpoint region and either end fused to another DNA, and </div>
      <div class="claim-text">said second probe set being sufficient in length and substantially complementary to nucleotides on both sides of the breakpoint region of a second DNA but less than an entire chromosome such that said second probe set will hybridize to both sides of the breakpoint region regardless of whether the second DNA has been broken in the breakpoint region and either end fused to another DNA. </div>
    </div>
    </div> <div class="claim-dependent"> <div num="11" id="US-6414133-B1-CLM-00011" class="claim">
      <div class="claim-text">11. The probe set of <claim-ref idref="US-6414133-B1-CLM-00010">claim 10</claim-ref>, wherein said probes are detectably labeled.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="12" id="US-6414133-B1-CLM-00012" class="claim">
      <div class="claim-text">12. The probe set of <claim-ref idref="US-6414133-B1-CLM-00011">claim 11</claim-ref>, wherein said first DNA is part of the ABL1 gene on chromosome 9 and the second DNA is part of the BCR gene on chromosome 22.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="13" id="US-6414133-B1-CLM-00013" class="claim">
      <div class="claim-text">13. The probe set of <claim-ref idref="US-6414133-B1-CLM-00011">claim 11</claim-ref>, wherein said first DNA is part of the AML1 gene on chromosome 21 and the second DNA is part of the ETO gene on chromosome 8.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="14" id="US-6414133-B1-CLM-00014" class="claim">
      <div class="claim-text">14. A diagnostic kit for detecting a structural abnormality caused by chromosomal breakage and rearrangement containing a reagent comprising at least one probe set of the probe set according to <claim-ref idref="US-6414133-B1-CLM-00010">claim 10</claim-ref>, and a container containing said reagent.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="15" id="US-6414133-B1-CLM-00015" class="claim">
      <div class="claim-text">15. A diagnostic kit according to <claim-ref idref="US-6414133-B1-CLM-00014">claim 14</claim-ref> further comprising at least two containers, wherein a first container contains a reagent comprising said first probe set and a second container contains a reagent comprising said second probe set.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="16" id="US-6414133-B1-CLM-00016" class="claim">
      <div class="claim-text">16. A diagnostic kit according to <claim-ref idref="US-6414133-B1-CLM-00015">claim 15</claim-ref> wherein said reagent comprises said first and said second probe sets.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="17" id="US-6414133-B1-CLM-00017" class="claim">
      <div class="claim-text">17. A diagnostic kit for detecting a structural abnormality caused by chromosomal breakage and rearrangement containing a reagent comprising at least one probe set of the probe set according to <claim-ref idref="US-6414133-B1-CLM-00008">claim 8</claim-ref>, and a container containing said reagent.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="18" id="US-6414133-B1-CLM-00018" class="claim">
      <div class="claim-text">18. A diagnostic kit according to <claim-ref idref="US-6414133-B1-CLM-00017">claim 17</claim-ref> further comprising at least two containers, wherein a first container contains a reagent comprising said first probe set and a second container contains a reagent comprising said second probe set.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="19" id="US-6414133-B1-CLM-00019" class="claim">
      <div class="claim-text">19. A diagnostic kit according to <claim-ref idref="US-6414133-B1-CLM-00018">claim 18</claim-ref> wherein said reagent comprises said first and said second probe sets.</div>
    </div>
  </div> </div></div></div><div class="patent-section patent-description-section"><div class="patent-section-header"><span class="patent-section-title">Description</span></div><div class="patent-text"><div mxw-id="PDES53592444" lang="EN" load-source="patent-office" class="description">
    <heading>FIELD OF THE INVENTION</heading> <p>The invention relates to improved polynucleotide probe configurations for detecting structural abnormalities that result from chromosome breakage and rearrangement, particularly as used in the detection of several types of genetic disorders related to cancer and other diseases. The invention further relates to an improved method of detecting translocations using probe sets which span each breakpoint region associated with a translocation and the regions on both sides beyond the 3′ and 5′ ends of each breakpoint region.</p>
    <heading>BACKGROUND OF THE INVENTION</heading> <p>A number of inherited genetic diseases and types of cancer have been linked to chromosomal translocation events which result in the fusion of two genes which do not occur together in the normal genome. Certain conditions involve translocations which frequently occur at the same or very near location. The chromosome regions where frequent breaks occur are called breakpoint regions.</p>
    <p>One of the best known examples of a clinically important translocation is the Philadelphia Chromosome which results from a break in the ABL1 gene on distal chromosome 9q and the BCR gene on proximal chromosome 22q {t(9;22)} (FIG. <b>1</b>). The breakpoints within the ABL1 gene may occur throughout a region spanning more than 175 kb upstream from exon II while the breaks in chromosome 22 are clustered into two areas of the BCR gene, termed the major breakpoint cluster region (m-bcr) and the minor breakpoint cluster region (M-bcr) (Kurzrock et al, <i>New England Journal of Medicine</i>, 319:990 (1988)). The Philadelphia Chromosome occurs in most cases of Chronic Myelogenous Leukemia (CML) and some cases of Acute Lymphocytic Leukemia (ALL). Other important translocations include, but are not limited to, t(8;21) in Acute Myelogenous Leukemia, t(8:14) in Burkett's Lymphoma and pre-B-cell Acute Lymphoblastic Leukemia, t(1:14), t(7:9), t(7:19), t(11:14), t(10:14) and t(7:9) in T-acute Lymphoblastic Leukemia, t(15;17) in Acute Myelogenous Leukemia (AML) and t(15:17) Acute Promyelocytic Leukemia (PML). Solid tumors include, t(9;22) in Ewing's Sarcoma, t(15:16), and hereditary diseases associated with translocations include a number of mental retardation associated syndromes. It is likely that other conditions are caused by subcriptic translocations or other structural aberrations which are yet to be determined and are too small to be noticed by standard cytogenetics.</p>
    <p>Multiple genetic testing methods have been developed for use in diagnosis, monitoring of minimal residual disease and/or response to therapy during clinical practice. However, no single technique has been developed that can accurately detect and quantify disease at diagnosis and throughout treatment. Conventional quantitative cytogenetics and G-banding analysis is cumbersome and can only be applied to cycling cells (Lion, Leukemia 10: 896 (1996)). In practice, the sensitivity of conventional cytogenetics is dependent upon the number of good metaphase cells which can be evaluated. In the example of cancers caused by neoplastic cells in the bone marrow, obtaining large numbers of good metaphase cells from bone marrows of patients is difficult.</p>
    <p>More recently, the assay technique in situ hybridization (ISH), particularly fluorescent in situ hybridization (FISH) (Pinkel et al, <i>Proc. Natl. Acad. Sci., U.S.A</i>. 83:2934-2938 (1986)) has been of assistance in detecting translocations. FISH allows the analysis of individual metaphase or interphase cells, thereby eliminating the need to obtain and assay cycling cells. It is therefore possible to use nondividing tissue, including bone marrow and peripheral blood cells in a diagnostic or prognostic analysis.</p>
    <p>In the field of detecting the Philadelphia Chromosome, a commonly used method for detection of ABL1/BCR fusion utilizes differently labeled probes for BCR and ABL1, and detects a single ABL1/BCR fusion (or closely linked) signal in cells with a Ph chromosome. (This method is referred to for convenience as S-FISH.) An example of this technique is Tkachuk et al, <i>Science </i>250: p. 559-562 (1990) where one fluorescently labeled probe hybridized to part of the ABL1 gene and a second fluorescently labeled probe hybridized to part of the BCR gene.</p>
    <p>The probes in commercial single FISH test kits do not span the entire length of each translocation breakpoint but rather are designed to bind to one portion of each gene, i.e. sometimes overlapping or adjacent to a breakpoint region, sometimes many kilobases away and sometimes both (See FIG. 1 of Tkachuk et al for example). Normal chromosomes 9 and 22 each bind one probe, which is specific to that chromosome. The Philadelphia Chromosome, both probes hybridize at the fusion site bringing both labels in close proximity so as to usually form a color shift or fusion near proximity/signal. Because the exact breakpoint may vary, the two probe labels may not come sufficiently close to form a fusion label. Likewise for probes useable to detect the t(8;21) translocation in Acute Myelogenous Leukemia (AML).</p>
    <p>Using the probe configuration above, the following detection method for the Philadelphia Chromosome using FISH has been used: the ABL1 gene probe is labeled using a probe containing one hapten or fluorophore (for example, FITC) and the BCR gene probe is labeled using a probe containing another hapten or fluorophore (for example Rhodamine). After hybridization and detection, a normal chromosome 9 shows the green signal and a normal chromosome 22 shows a red signal. A normal cell would therefore exhibit two red signals and two green signals. A cell containing a Philadelphia chromosome has one red and one green signal for the unaffected homologues of chromosomes 9 and 22 and one white, yellow or closely linked pair of signals that results from the close proximity of the labeled probes hybridized to the translocated BCR and ABL1 genes, the so-called fusion signal.</p>
    <p>However, the probes used heretofore in this method have not been constructed so as to specifically bind and detect the second fusion site for the reciprocal translocation event. Thus, the S-FISH method detects only one of the abnormal chromosomes resulting from the translocation event, the Philadelphia chromosome.</p>
    <p>In another method using labelled probes to detect ALL gene rearrangements in solid tumors, a probe set was designed so that the two probes lie adjacent to each other on the normal chromosome, but split apart and move to the two different abnormal chromosomes if the translocation has occurred (Croce, U.S. Pat. No. 5,567,586, hereby incorporated by reference). In this method the probes are designed to be complementary to sequences in the translocation region on one chromosome. In this method, the fluorescent probes produce a single spot on the normal chromosome, but appear as two distinct spots when translocation has occurred.</p>
    <p>The same format has been used for other assays for detecting other translocations such as t(8:21) in Acute Myeloid Leukemia (AML). For example, Le Beau, Blood 81: 1979-1983 (1993), and Sacchi et al, Cancer Genetics and Cytogenetics 79: 97-103 (1995) and Fischer et al, Blood 88: 3962-3971 (1996).</p>
    <heading>SUMMARY OF THE INVENTION</heading> <p>It is an object of the invention to provide methods with increased sensitivity and accuracy for detecting chromosome translocations and other structural rearrangements which result in more than one abnormal fusion site in the genome.</p>
    <p>It is a further object of the invention to provide probes and probe sets which are useful in detecting reciprocal genetic translocations according to the methods of the invention.</p>
    <p>It is another object of the present invention to detect cancer, inherited disease, susceptibility to inherited disease or a carrier of a fused gene for an inherited disease wherein the condition results from a chromosomal translocation in one or more cells. This is particularly beneficial when the diagnosis, prognosis, monitoring for residual disease and response to therapy in cancer or other disease is dependant upon the quantity of abnormal cells as an indicia of the disease state and/or response to treatment.</p>
    <p>It is also an object of the invention to provide a means of constructing such probes and probe sets, which will detect reciprocal fusions resulting from chromosomal translocations and will accordingly be useful in diagnosis, prognosis, monitoring of residual disease and response to therapy when reciprocal chromosome translocations are present.</p>
    <p>It is still another object of the present invention to provide diagnostic test kits which can be used by any cytogenetist or other trained individual to detect multiple fusion events which result from structural rearrangement of the genome.</p>
    <p>Probes and probe sets of the present invention have the characteristic of encompassing the entire breakpoint region and a region on each side of the breakpoint region on each chromosome for the reciprocal translocation event of interest and are capable of detecting such translocations with much greater sensitivity than the probes and probe sets which were previously known.</p>
    <p>A particularly preferred probe set and method is used for detecting the Philadelphia chromosome and its corresponding derivative chromosome as companion indicators of CML and some other cancers such as ALL. One functional probe is designated P5161-DC, described hereinbelow. Another example is for detecting the AML1/ETO gene fusion in AML.</p>
    <p>The use of specifically designed probe sets by the method of the present invention has allowed the clinician to assess physical information regarding all fusion events associated with a defined structural rearrangement in a cell. For example, using the standard detection method of fluorescence in situ hybridization (FISH) it has been demonstrated that these probe sets provide the following advantages over traditional testing methodologies for detecting the same translocation.</p>
    <p>1. Unlike traditional single fusion probe sets, probe sets which detect multiple, derivatives of a structural rearrangement have the ability to detect much lower copy numbers of abnormal cells thereby providing greater improved diagnostics using FISH assays</p>
    <p>2. The ability of the probe sets to derive necessary information from cells in interphase, Thereby rivaling the sensitivity of metaphase cells in conventional cytogenetics.</p>
    <p>3. Specifically, increased sensitivity has been demonstrated with multiple fusion probes used in interphase FISH analysis which is at least as sensitive as Q-cytogenetics (the previous gold standard) for monitoring bone marrow or peripheral blood cell populations for minimal residual disease and response to therapy.</p>
    <p>4. Greater sensitivity allows the use of peripheral blood instead of invasive and painful bone marrow samples from patients for routine testing, to monitor for minimal residual disease and response to therapy.</p>
    <p>5. By detecting high and low copy numbers of gene fusions, the present invention can be used for diagnosis and monitoring throughout the course of the disease thereby avoiding traditional multiple assay-type testing methodologies.</p>
    <p>6. Simplified sample requirements and testing provides further benefits in cost and patient well being.</p>
    <heading>BRIEF DESCRIPTION OF THE DRAWINGS</heading> <p>The file of this patent contains at least one drawing executed in color. Copies of this patent with color drawing will be provided by the Patent and Trademark Office upon request and payment of the necessary fee.</p>
    <p>FIG. 1 shows a schematic drawing of a BCR/ABL1 translocation and probes constructed according to the invention.</p>
    <p>FIG. 2A shows the appearance of a normal cell after testing with probe P5161-DC using the methods of the invention. The two red signals designate the BCR genes on chromosome 22 and the two green signals designate the ABL1 genes on chromosome 9.</p>
    <p>FIG. 2B shows the appearance of a cell containing a BCR/ABL1 translocation after testing with probe P5161-DC using the methods of the invention. One red, one green and two fused signals denoting both of the reciprocal translocation events are present.</p>
    <p>FIG. 2C also shows the appearance of a cell containing a BCR/ABL1 translocation after testing with probe P5161-DC using the methods of the invention. Two red, two green and one fused signals are present in this example. While two fusion signals are usually detected, because of the physical configuration of the gene and the relaxation of the heterochromatin in interphase, a red and a green signal may appear to be closely linked but not quite overlapping. Note the two signals at the lower end of the field which are not quite fused. This configuration is believed to represent the fused portion of a translocation event.</p>
    <p>FIG. 2D shows the appearance of a cell containing a BCR/ABL1 translocation after testing with probe P5161-DC using the methods of the invention. One red, one green and three fused signals are present. This cell contains an additional Philadelphia chromosome.</p>
    <p>FIG. <b>3</b>. Percentage of Ph positive cells (Y-axis) prior to therapy and during treatment at approximately 4 month sampling intervals (X-axis in days) in bone marrow by Q-cytogenetics and D-FISH, and blood by D-FISH.</p>
    <p>FIG. <b>4</b>. Relationship between the percentage of Ph positive cells for paired-sets of bone marrow (Y-axis) and peripheral blood (X-axis).</p>
    <p>FIG. <b>5</b>. Linear regression analysis of the (transformed) proportion of abnormal cells from bone marrow on the (transformed) proportion from peripheral blood from FIG. <b>4</b>. Dashed lines are the 95% prediction interval.</p>
    <p>FIG. <b>6</b>. Results of linear regression analysis but transformed to original scale of proportions of abnormal cells for bone narrow (Y-axis) versus peripheral blood (X-axis). Dashed lines represent the 95 prediction internal for a bone narrow prediction given a specific peripheral blood score.</p>
    <p>FIG. <b>7</b>. BCR map and summary of probe listed as Table 1.</p>
    <p>FIG. <b>8</b>. ABL1 map and summary of probe listed as Table 2.</p>
    <p>FIG. <b>9</b>. Data and comparison of different techniques for assaying for the Philadelphia chromosome, listed as Table 3.</p>
    <p>FIG. <b>10</b>. Data comparing bone marrow and blood samples for monitoring the disease state and response to therapy, listed as Table 4.</p>
    <heading>DETAILED DESCRIPTION OF THE INVENTION</heading> <p>The present invention is an assay and reagents therefore which may be packaged into a simple test kit. The reagent comprises two probes, the first complementary to and encompassing the entire breakpoint region on a first chromosome as well as both upstream and downstream regions from the breakpoint region. As such, the first probe will hybridize to a normal first chromosome as well as both fragments of the first chromosome which have undergone a translocation or break and may be rearranged elsewhere. The second probe is the same as the first except for spanning the entire breakpoint region, as well as both upstream and downstream regions on the second chromosome. Kits may also be constructed for multiple translocation events or having more than two chromosomes involved according to the methods of the present invention.</p>
    <p>Each probe is detectable when hybridized to the target DNA, preferably by being labeled with a unique detectable label that can be either directly or indirectly detected. The labeling may be by covalent bonding or other affinity attachment. Each polynucleotide comprising a probe is labeled with the same label and each probe has a detectably different label from other probes in the mixture. As such, one can easily detect each normal chromosome potentially involved in the translocation as well as fusions between the two or more breakpoint regions which are detected as a fusion signal.</p>
    <p>The different construction of multiple fusion probes provides numerous advantages over two similarly labeled single fusion probes hybridizing to different locations. Some of these advantages were not apparent until after testing the probes on biological samples. Thus, by constructing probes complementary to the entire breakpoint regions, as in D-FISH, one achieves certain advantages over conventional S-FISH.</p>
    <p>In the present invention's improved method, designated D-FISH, fusion signals can be detected in each cell as an indicator of the presence of a reciprocal translocation. The sensitivity of the method using the available S-FISH probes has not been sufficient in the past to detect very low levels of translocations which are found in the peripheral blood cells or bone marrows of many patients. Specifically, commercially available S-FISH detects about 70-75% of patients actually positive for the Philadelphia chromosome. Actual positive results were necessarily determined by sequencing, PCR amplification or Southern blotting. By contrast, using D-FISH with the probes of the present invention detects greater than 99% of patients actually positive. This data rivals or exceeds standard Q-cytogenetics (See Dewald et al data). The improved detection indicates that the present invention should be accepted as the “gold standard” by which all other assays are compared.</p>
    <p>This improved sensitivity is accomplished by using probes which are specifically developed to cover the entire breakpoint region of each chromosome involved in the translocation. In such a translocation, the labeled probe for a first chromosome is immediately adjacent to the labeled probe for a second chromosome thereby producing multiple fusion signals. By contrast, S-FISH employs probes which are designed to be complementary to one side of each of two breakpoint regions and therefore relies upon the detection of a single fusion event.</p>
    <p>The new method presented here involves novel DNA probe constructs which are designed to target the length of each breakpoint region in a translocation event and additional DNA sequence beyond both the 3′ and the 5′ ends of each breakpoint region. When the breakpoint region occurs in a gene, it is preferable for the probes to encompass the entire gene and additional DNA sequence beyond both ends also. When used together, all reciprocal translocation events involving the target gene regions can be detected simultaneously in an interphase or metaphase cell. The probes are further designed to give easily visualized balanced signals in interphase cells.</p>
    <p>The effectiveness of dual or multi-fusion probes is perhaps best demonstrated when the DNA probes are fluorescently labeled in different colors and hybridized to cellular DNA using the standard assay technique of fluorescence in situ hybridization (FISH) (Pinkel et al, <i>Proc. Natl. Acad. Sci., U.S.A</i>. 83:2934-2938 (1986)). Various types of DNA probe configurations have been used with FISH technology in an attempt to find a reliable and sensitive assay for interphase cell analysis. However all of these techniques provide either too many false positives, false negatives or simply lack the sensitivity to determine the presence of the translocation in interphase detection of minimal residual disease and/or response to therapy. Additionally, other techniques are cumbersome or expensive to test or require large quantities of hard-to-obtain biological material. By comparison, the Examples below show obtaining a result from a simple blood sample using conventional cytogenetic equipment with a high sensitivity and low error rate.</p>
    <p>Balanced and non-reciprocal translocations may also be detected using the probe strategy and method of the present invention. Even inversions within the same chromosome may be detected as double fusions with the probe sets developed for the two breakpoint regions on the same chromosome. In such situations, one still has two breakpoint regions forming at least one fusion site for detection of a fusion signal. The same general principles apply either way. In accordance with the present invention, probes constructed in accordance with the general instructions provided herein are used to produce reagents and a method for detecting multiple breaks as well as any resulting fusions thereby determining the presence of such multiple breakage events simultaneously. The present invention may also be used for screening for chromosome breakage at multiple genome sites due to environmental factors, chemicals, radiation (diagnostic X-rays or radiation therapy or radiation exposure), biological agents etc.</p>
    <p>The source of cells may be highly variable. If a cancer is being diagnosed or monitored, cells from the tumor site or removed from the tumor site may be used. For inherited diseases, readily available cells from tissues, blood, urine, feces, buccal scrapings, cervical and vaginal scrapings (PAP smear), body fluids, etc. may be used. For prenatal testing, fetal, amniotic, placental, cord, chorionic villus, and “cells” including sperm or egg for the situations of gamete “donation” or in vitro fertilization may be used.</p>
    <p>The cells being tested may be in any phase, but metaphase and interphase are preferred.</p>
    <p>While this application generally refers to humans and human diseases, persons of skill will appreciate the invention is useful in other settings. The present invention is equally applicable to other animals for agricultural or veterinary purposes as useful in the diagnosis, prognosis and monitoring of disease. If so desired, the present invention is applicable to determining translocations in plants as well. The present invention is also applicable to non-disease conditions where determining the presence of a translocation is important for plant and animal breeding such as to follow the presence of a trait throughout generations.</p>
    <p>One standard method for plant breeding involves infecting the plant with Agrobacterium tumefaciens carrying a Ti plasmid which will integrate T-DNA into the plant chromosome. Transfection of the plasmid per se may also be used. A desired gene is typically incorporated into the T-DNA region, especially in a hormone gene if not already deleted. In such a situation, the plant chromosome has a breakpoint region for inserted T-DNA. Probes encompassing the plant chromosome breakpoint region and the two ends of the T-DNA or the desired gene may be used to determine whether plant cells contain the desired gene. The present invention results in considerable time savings compared to tissue culture and cultivation of the plant to maturity and testing for a trait caused by the desired gene.</p>
    <p>Along the same procedural lines, the present invention may be used to assay for insertion of any other DNA into a specific chromosome site such as is desirable in gene therapy. During certain forms of gene therapy, added DNA is incorporated into the host chromosomes at specific locations. The present invention permits monitoring and provides proof of integration.</p>
    <p>Viruses which integrate are biologically significantly different when their DNA is integrated into a host chromosome. Such an integration involves a breakage of the chromosome and a fusion of the viral DNA into the chromosome. Monitoring the integration is an important step in assaying for antiviral therapeutics, determining prognosis, etc. If the integration site or region is known, probes to that site and to the virus (or viruses if two are coinfecting) may be prepared and used according to the present invention.</p>
    <p>False positives can occur in normal cells, for example when the BCR locus of chromosome 22 coincidently occurs very close to, behind or in front of the ABL1 locus of chromosome 9. Since the cell being viewed is three dimensional and usually in interphase, the chromosomes are freely moving within the cell nucleus permitting a random juxtaposition of signal. About 4% of normal slides have the two loci sufficiently superimposed on each other to cause the signals to appear fused using S-FISH.</p>
    <p>However, when using D-FISH, two fusion signals typically occur as well as two normal signals. The percentage of normal cells with both ABL1 and both BCR loci coincidently superimposed is very small. Thus, the false positive rate in D-FISH is lower. Still further, normal cells displaying two fusion signals by chance would not display two normal signals as well, providing a further check against false positives.</p>
    <p>As for reducing the false negative rate and increasing sensitivity, one potentially has twice as many fusion signals per cell which makes it easier to detect an abnormal cell.</p>
    <p>The advantages of the present invention depends upon a number of factors, including the unique probe configuration, the number or percentage of affected cells, which may vary with individuals and disease states. For example, the methods described above typically require that about 1% of the cells be affected for accurate signal detection. This compares to a S-FISH assay requiring about 30% of the cells positive. If fewer cells are affected, an abnormal condition may not be detected.</p>
    <p>While the Examples use microscopic identification of normal and abnormal chromosomes, other techniques may be used. For example, cells may be observed and determined to contain or not contain a translocation during flow cytometry or extracts may be taken and conventional DNA hybridization assays performed.</p>
    <p>Several factors determine how large a probe construct should be. In the example of the BCR gene, for example, the probe would be sufficiently long to include both the major breakpoint cluster region and the minor breakpoint cluster region, as well as sequences beyond the gene. For other genes exemplified below, the breakpoint region is widely variable in size and requires probes of sufficient size unique to each application and may be determined by routine optimization. Generally, the probes will have a considerable length complementary to the adjacent non-breakpoint region for a normal or translocated fusion configuration. The length will depend on the particular translocation being detected. The length of each probe will further be manipulated to make visually balanced signals and/or enough to routinely cause a color shift when the signal is fused to a different label's signal. The length must accommodate all breaks, regardless of where in the breakpoint region the actual breakage and fusion occurs. Preferably, the length is also be sufficient to provide fusion signals of similar size throughout the target clinical population of cells, thereby increasing reliability and ease of interpretation.</p>
    <p>Generally, the length of the probe sets will correlate to the length of the largest breakpoint region involved in a translocation event. Thereby, the balance of fusion signals in interphase cells is assured. The length may also be affected by the amount of label which can be incorporated on the probe. Considerable variance is acceptable, if there is optimization of labeling conditions for each probe being developed.</p>
    <p>In one preferred embodiment of the invention, one probe of a probe pair is designed to be complementary to the ABL1 sequence (600 Kb) and the other probe is designed to be complementary to the BCR sequence (500 Kb). Individual cloned human DNA probes of varying lengths complementary to the ABL1 and BCR breakpoint regions were used collectively to make probes of appropriate length. A single clone may be used; however, if the insert is of sufficient size. In practice, probes are developed from overlapping “probe sets”, consisting of several cloned DNA sequences which hybridize to and span the breakpoints on the relevant chromosomes.</p>
    <p>Alternative techniques may be used other than FISH for probes of the present invention. For example, during the use of conventional blot assays, Southern and Northern, probes of the present invention may be optimized to be used in lieu of other labeling techniques. The probes of the present invention may also be used in developing assays in aqueous solution.</p>
    <p>The probes of the invention may be detected after it is hybridized to the target DNA or RNA. This may be done by any technique which detects a probe containing double stranded DNA within the biological sample. If the remainder of the sample lacks significant double stranded regions, one may use chemicals which specifically bind to double stranded but not single stranded DNA or DNA/RNA. Examples include a labeled antibody to double stranded DNA or RNA/DNA followed by detecting the label, ethidium bromide, SYBR green, an acridine dye (e.g. acridine orange), a protein or enzyme, etc.</p>
    <p>The more preferred option is to have the probes labeled in order to provide a means of detection. Suitable labels include, but are not limited to, haptens and fluorophores, such as, FITC, Rhodamine and Texas Red as well as radioactive, chemiluminescent, bioluminescent, a metal chelator, quencher, enzyme, chemical modifications rendering the DNA detectable immunochemically or by affinity reactions, and other known labels. Many such suitable detection labels are known to persons of skill in the art of binding assays such as nucleic acid hybridization assays and immunoassays. When the label is a hapten, a receptor labeled directly or indirectly with an easily detectable substance, is employed before, with or after hybridization of the hapten labeled probe. When the label is a quencher, the absence of or reduced signal indicates the presence of the quencher.</p>
    <p>Common ways to incorporate the label into the probe include nick translation, random priming or PCR amplification using a derivitized dNTP or NTP. Also post probe synthesis labeling and end labeling may be performed. The amount of label varies from one probe to another and the various uses for the probes. Too much labeling may actually cause a quenching effect. Typically about 1-25% of a nucleotide (A, G, C, or T) will be modified to incorporate a label into a DNA probe.</p>
    <p>One of ordinary skill can choose appropriate labeling techniques, other colors or detection strategies which may vary depending on the particular translocation or other fusions being detected.</p>
    <heading>DEFINITIONS</heading> <p>As used herein, the term “probe” is intended to mean one or more polynucleic acids which hybridize specifically to a particular region of chromosome which is of interest. Depending on the size of the region, multiple polynucleotide molecules may be combined to comprise the probe. The number of polynucleotides will also be determined by whether the polynucleotides are synthesized chemically, by PCR, by plasmid, by cosmid, by yeast artificial chromosome (YAC) etc. Individual molecules comprising the probe may hybridize to overlapping portions of the chromosome of interest or may hybridize to physically linked regions separated from each other. These gaps may be sizable but should not be so large that upon hybridizing to a translocation locus in a cell, the probes are so far apart that they appear as non-associated signals and no fusion event can be reliably detected. For example, a 100 base pair gap is probably insignificant whereas a 1 Mb gap is too much to be acceptable. Note that the break may occur anywhere in the breakpoint region and therefore construction of the polynucleotide molecules composing the probe should be designed to accommodate breaks at the worst possible locations.</p>
    <p>A probe need not have exact complementarity to the desired target, but should have sufficient complementarity to bind to the region of interest using the methods of the invention. To achieve this generally requires a matching sequence with at least 80%, preferably 95%, and most preferably about 100% complementarity to the target. Occasional polymorphisms may preclude true 100% complementarity in some individuals, particularly when the breakpoint does not occur in a coding sequence.</p>
    <p>Accordingly, as used to refer to probes herein, the term “complementary” includes “substantially complementary” which is intended to refer to a probe which will specifically bind to the region of interest on a chromosome under the test conditions which are employed, and thus be useful for detecting and localizing the region. Complementarity will be extensive enough so that the probes will form specific and stable hybrids with the target DNA under the hybridization conditions used. Persons of skill in the art will be able to determine suitable sequences through the general knowledge available in the art, and by routine experimentation, using the examples set forth hereinbelow as guidelines.</p>
    <p>A “cell” as used herein includes biological samples which were derived from cells. “Biological sample” includes all nucleic acid containing compositions where the nucleic acid (RNA or DNA, chromosome, viral, vector, mitochondrial . . . ) was obtained from an individual organism or amplified from a nucleic acid obtained from an individual organism. The slide preparation procedure used in the Examples actually kills the cell and removes some of its components. However, the DNA remains. The term “cell” as used herein includes cellular components, extracts and other partial cellular components provided that they contain the nucleic acids. It is preferred that a reasonably complete set of the chromosomes remains or at least the DNA of the breakpoint regions and adjacent regions remains such that one can determine normal untranslocated DNA sequences from fused DNA sequences resulting from a translocation.</p>
    <p>A “translocation” is the exchange of genetic material between two or more non-homologous chromosomes. This is frequently a reciprocal event where two chromosomes are simultaneously broken and the fragments are exchanged between the two chromosomes. Two new chromosome derivatives are created.</p>
    <p>A piece of a chromosome may be broken twice and reincorporated in the same region in reversed order. This is called a inversion and is a subset of structural abnormalities caused by chromosomal breakage and rearrangement.</p>
    <p>The present invention has many uses other than detecting reciprocal translocations such as detecting other chromosomal abnormalities caused by chromosomal breakage and rearrangement such as insertions, inversions, derivative chromosomes and possibly duplications and ring formations.</p>
    <p>As used herein, the phrase “the entire breakpoint region” is intended to refer to a sequence or probe of sufficient length to include the entire region in which a particular break may occur. This region will vary with the particular structural aberration one wishes to detect. In rare instances where the boundaries of the breakpoint region may not be completely known or unclear, the breakpoint region is the region encompassing the distribution of two standard deviations of known breakpoints.</p>
    <p>A “contig” is a collection of two or more overlapping cloned DNA fragments that when used together will extend the target region beyond that of using a singular cloned fragment. A contig refers to “contiguous” DNA fragments.</p>
    <heading>EXAMPLE 1: CONSTRUCTION OF BCR/ABL1 DUAL FUSION PROBES</heading> <p>The BCR/ABL1 dual fusion probes were assembled by screening through several different human libraries cloned into PAC, P1, BAC, and YAC vectors available from commercial sources, e.g. a CEPH library. The procedure included several rounds of sequencing and walking. These methods are known to persons of skill in the art and are described in various molecular procedure manuals such as PCR Protocols, A Guide to Methods and Applications, Innis et al, Academic Press, Inc. (1990) incorporated herein by reference.</p>
    <p>Each round of screening included the following steps:</p>
    <p>1. Synthesizing new PCR primers based on sequence information.</p>
    <p>2. Establishing PCR conditions for the new primers.</p>
    <p>3. Screening the libraries by either PCR (using primers) or DNA hybridization (by amplified fragments).</p>
    <p>4. Selecting the positive clones.</p>
    <p>5. Evaluating the positive clones by FISH. Verifying that the positive clone hybridizes to the correct region and does not show any cross hybridization.</p>
    <p>6. Obtaining the end sequences of the insert of new clones by either direct sequencing or by sequencing the purified end fragment amplified by using a combination of Alu or other primers and vector end primers.</p>
    <p>7. Comparing the new sequence to the existing sequence to establish the relative location of the new clone. New primers were then made from the new sequence outside the existing sequence.</p>
    <p>8. Repeating steps 2-7 until the probes reached the appropriate length to include the entire breakpoint region and achieve the desired FISH signal intensity.</p>
    <p>9. Establishing the relative locations of all clones in the final contig by STS mapping and estimating the size of the contig.</p>
    <p>To obtain multiple fusion probes according to the invention, it is preferred that the probes cover both sides of the breakpoint and show not only good but also balanced signals in affected cells. For both BCR and ABL1 probes, screening was done for clones which collectively hybridize to the entire breakpoint region and both sides of the breakpoint region containing normal chromosomal DNA.</p>
    <p>BCR:</p>
    <p>The BCR dual fusion probe set is composed of 5 human PAC clones which are shown in Table 1, FIG. <b>7</b>.</p>
    <p>The BCR region contains a 152141 bp sequence published by GenBank. Three primer pairs were initially made, BCR a/b, BCR c/d, and BCR e/f, which correspond to the gene sequence at the −15 kb, −123 kb and −152 kb, 5′ to 3′ positions respectively. These primer sets were used to screen a P1 library by PCR and the amplified fragments were isolated and pooled to screen a PAC library by hybridization. Several positive P1 and PAC clones containing BCR gene sequences were obtained.</p>
    <p>P1 Clone OC2001 was scored positive using primers BCR a/b. The end sequences of the insert were obtained. This clone has one end of the insert located in the BCR known sequence and one end outside the 5′ end of known sequence. Primer set BCR 13/14 was synthesized using the new sequence information. Both PAC OCB1001 and OCB1002 were obtained by screening using BCR 13/14. The next round of screening was done by first sequencing the end sequences of the insert in PAC OCB1001, establishing the 5′ and 3′ positions of the ends and primers BCR 26/27 were made. PAC OCB1003 was acquired by screening the PAC library using the new primers BCR 26/27. This PAC is on the most 5′ end of the contig.</p>
    <p>PAC OCB1004 was obtained from the hybridization of PAC library using the pooled amplified DNA fragments generated by the BCR a-f primers described above. This clone covers almost all the BCR known sequence and also extends in the 3′ direction.</p>
    <p>From the PCR screening of the P1 library using primers BCR e/f, P1 clone OC2002 was obtained on the 3′ end of the gene. Both ends of the insert were sequenced. This clone contains the BCR gene sequences from the 3′ position, 109 kb into the gene, and extends further in the 3′ direction from the end of the BCR gene. A new primer pair BCR 13/14 was made using the new 3′ end sequence. PAC OCB1005 was obtained from the new screening which became the furthest 3′ clone in the contig.</p>
    <p>The size of the inserts of these individual clones are estimated by adding up all the EcoR1 restriction fragments found on agarose gel as compared to commercially available molecular weight DNA markers. The relative locations of all the clones are established by whether the clones are positive or negative to all the PCR primer sets tested. Because the entirety of the clones were not sequenced, the extent of overlap or gaps (if any) present in the clones has not been characterized. However, the clones are known to contain sequences in common to other clones within the BCR probe set. The total size of the BCR contig is approximately 500 kb.</p>
    <p>ABL1</p>
    <p>The ABL1 dual fusion probe set consists of 1 BAC, 1 P1, 4 PAC and 1 YAC clone as shown in Table 2, (FIG. <b>8</b>).</p>
    <p>The ABL1 region contains 3 segments of published GenBank sequences: HSALBGR1, 35,692 bp, covering the 5′ ABL1 exon 1b and part of intron 1b, HSABLGR2, 59,012 bp containing portions of intron 1b and HSABLGR3, 84,539 bp extending from the end of intron 1b to the end of exon 10 and poly A region. The intron 1b is about 200 kb in length.</p>
    <p>The initial screening was done in a similar way to screening for five BCR probes. Three primer sets were synthesized, ABL1 a/b, ABL1 c/d, and ABL1 e/f. ABL1 a/b is located &gt;2000 bp in from the 5′ end of the HSALBGR1 sequence, see table 2, FIG. <b>8</b>. ABL1 c/d is ˜79,000 bp in from the 5′ end of the HSABLGR3 sequence, and ABL1 e/f is located ˜31,000 bp in from the 5′ end of HSABLGR2 sequence. The ABL1 a-f primers were used to screen a P1 library directly by PCR and the amplified fragments from these primers were used to screen a PAC library by DNA hybridization. Several positive P1 and PAC clones were identified.</p>
    <p>The P1 clone OC3001 was obtained from PCR screening using primers ABL1 a/b. The clone covers a small segment of the HSABLGR1 sequence and extends further in the 5′ end of ABL1. A new primer set ABL1 5/6 was made after sequencing the end of the OC3001 insert. ABL1 5/6 was used to screen a PAC library and the PAC clone OCA1001 was acquired. The OCA1001 clone contains the most 5′ end of the contig. The P1 clone OC3002 was obtained by PCR screening using primers ABL1 e/f. This clone contains most of the HSABLGR1 and HSABLGR2 sequence regions.</p>
    <p>PAC clone OCA1002 was obtained by hybridization screening using the pooled amplified fragments generated by the ABL 1 a-f primers. This PAC clone also extends outside the 5′ end of ABL1 gene. The end fragments of the insert were sequenced and primer set ABL1 3/4 was made. ABL1 3/4 was used to screen a BAC library. The BAC clone OCA1003 was identified.</p>
    <p>YAC clone OCA1004 was obtained from the commercially available library. OCA1004 contains a portion of HSABLGR2 sequence and extends beyond the 3′ end of the HSABLGR3 region. The end fragments of OCA1004 were isolated and sequenced. Primer pair ABL1 7/8 was made and used to screen a PAC library. PAC clone OCA1005 was obtained. A new primer set, ABL1 19/20, was synthesized using sequence information obtained from clone OCA1005. Both PAC OCA1006 and OCA1007 were identified by library screening using ABL1 19/20.</p>
    <p>The sizes of the inserts of the clones in the ABL1 probe set, except for the YAC, were estimated by summing up EcoR1 restriction fragments visualized on an agarose gel. The size of YAC clone was determined by comparing to known size standards on a gel. The relative positions of all the clones were determined from using the primer sets developed for screening DNA bands as physical map anchor sites throughout the ABL1 region. The total length of this contig is approximately 600 kb.</p>
    <p>The combination of the BCR and ABL1 probe sets described above defines a dual fusion probe set for t(9;22). It has been designated P5161-DC. The skilled artisan will appreciate that by using these and other techniques known in the art, additional suitable probe sets would be constructed for the ABL1/BCR translocation and for other translocations of interest.</p>
    <heading>EXAMPLE 2: USING THE PROBE SET FOR CML D-FISH ASSAYS</heading> <p>The P5161-DC probe set was used in standard FISH protocols to devaluate the usefulness of using dual fusion probes (D-FISH) FOR DETECTION. The study of Philadelphia chromosome in a CML clinical population included 37 paired-sets of bone marrow and peripheral blood specimens from 10 patients undergoing treatment for CML, 10 normal peripheral blood specimens, 10 normal bone marrow specimens and four serial dilutions with known percentages of Ph positive nuclei.</p>
    <p>Each patient with CML was a participant of the CML National Study Group clinical trial and was randomly receiving treatment with interferon α-2b with or without ara-C. Each patient was known to have cells with a Ph chromosome that produced a typical D-FISH pattern (two fusion signals, two normal signals) for t(9;22)(q34;q11.2). For each patient a paired-set of bone marrow and peripheral blood specimens were collected prior to treatment and at two or more times at approximately 4-month intervals during treatment. Each paired-set of peripheral blood and bone marrow specimens was obtained on the same day except for specimens collected prior to treatment in patients 3 (blood and bone marrow were collected 1 day apart), 5 and 8 (blood and bone marrow were collected 4 days apart).</p>
    <p>Uncultured bone marrow and peripheral blood specimens were processed by conventional procedures for cytogenetic and FISH studies. These specimens were stored as fixed pellets at −70° C. in methanol:acetic acid (3:1) until FISH studies could be performed. The D-FISH specimens were prepared by being washed twice with fresh fixative and cells were placed on microscope slides and allowed to air-dry in a CDS-5 cytogenetic drying chamber (Thermotron, Holland, Mich.) adjusted to 50% relative humidity and 25° C. Slides were further dried for 1 hr in a 65° C. oven and then treated with 2× standard saline citrate solution (SSC) (300 mmol/L sodium chloride, 30 mmol/L sodium citrate) for 1 hr at 37° C. Slides were then dehydrated with 70-85-100% cold ethanol (stored at −20° C.) for 2 minutes each, and air-dried.</p>
    <p>The FISH hybridization and detection procedure was carried out as follows. Chromosomal DNA (in the form of cells on a slide) was denatured in 70% formamide/2×SSC for 2 min at 70° C. Slides were dehydrated with an ethanol series (70%, 85% and 100%) for 2 min each and air-dried. The probe (Oncor product #P5161-DC) was denatured in a water bath at 70° C. for 5 min. Then 10 μl of stock solution BCR/ABL1 probes were added to each slide, a 22×22 mm coverslip placed on the slide and sealed with rubber cement. Slides were hybridized for 18-20 hrs at 37° C. in a humidified chamber. After the coverslips were removed, slides were washed for 2 min in 0.4×SSC at 70° C., and then in 1× PBD (phosphate-buffered non-ionic detergent) for 2 min. Chromatin was counterstained in blue with 10 μl of 1% solution of 4′,6′-diamidine-2-phenylindole (DAPI) in Vectashield antifade. Representative cells were captured using a computer-based imaging system (Quips XL Genetics Workstation, Vysis, Inc., Downers Grove, Ill.).</p>
    <p>Q-cytogenetic studies were performed on each bone marrow specimen by analyzing 25 consecutive G-banded or Q-banded metaphases in which chromosomes 9 and 22 could be observed using the methods of Dewald et al, <i>Cancer Cytogenet</i>. 94:59 (1997). Hypermetaphase studies using single fusion probes for BCR and ABL1 (S-FISH) were done on many of these specimens using the methods of Seong et al, <i>Blood </i>86:2343 (1995). D-FISH was performed using the directly labeled P5161-DC probe set to reveal two BCR/ABL1 fusion signals in cells with a t(9;22)(q34;q11.2); one on the abnormal chromosome 9 and the other on the abnormal chromosome 22. The ABL1 (600 kb) probe was directly labeled with Rhodamine Green (green signal) and included several DNA sequences that hybridized to 9 q34 and spanned the 200-Kb breakpoint region of ABL1 including additional normal chromosome sequence on each side of the breakpoint region. The BCR (500 Kb) probe was directly labeled with Texas Red (red signal) and included several DNA sequences that hybridized to 22q11.2 and spanned the common breakpoints in both the major and minor BCR as well as normal chromosome sequences on each side of the BCR gene breakpoint regions.</p>
    <p>The specimens were studied in random order and in a blind fashion by two microscopists using strict scoring criteria for D-FISH. Dewald et al, <i>Blood </i>31(9): 3357-3365 (1998). As referred to hereinafter, red BCR signals are referred to as R, green ABL1 signals as G, and BCR/ABL1 fusion signals as F. For scoring purposes, fusion signals were defined as merging or touching R and G signals. The scoring process was limited to normal nuclei with 2 R2 G, and abnormal nuclei with 1R1G2 F or 2R2G1F (one Ph chromosome), and 1R1G3F or 2R2G2F (two Ph chromosomes). For each specimen, each microscopist scored 250 consecutive qualifying interphase nuclei from different areas of the same slide. At the conclusion of the study, the inter-microscopist agreement was sufficient to pool their results on each specimen in subsequent analyses of the data. Thus, the final statistical analyses were based on 500 nuclei per specimen.</p>
    <p>The normal range for D-FISH was calculated for peripheral blood specimens collected from 10 patients without any malignant hematologic disorder and for bone marrow specimens collected from 10 normal bone marrow transplant donors. The four serial dilutions were prepared by mixing cells from a normal individual and a Ph positive specimen to create a series of specimens determined by repeated blind studies to contain specified mean percentages of Ph positive nuclei.</p>
    <p>The D-FISH results for each patient's specimens from both peripheral blood and bone marrow samples were calculated as the proportion of abnormal cells (number of abnormal cells per 500 scored cells). Since the proportion (p) of abnormal cells among the specimens ranged from 0 to 1 (i.e. 0-100%), a sin<sup>1</sup>({square root over (p)}) transformation was used to stabilize variances and provide a more nearly Gaussian distribution of values. Then, the differences (delta value) between bone marrow and peripheral blood in transformed proportions were computed for each patient's specimens. The proportion (p) of abnormal cells by Q-cytogenetics was also transformed to sin<sup>1</sup>({square root over (p)}).</p>
    <p>The delta value for each paired-set of bone marrow and blood specimens was then analyzed using a repeated measures regression analysis (PROC MIXED in SAS) (19). For purposes of this statistical analysis, the approximate 4 month sampling intervals relative to commencement of therapy was considered a nominal predictor variable and the transformed proportion from Q-cytogenetics was included as a covariate. The within-patient correlation of delta values among respective specimen collection times was specified as an auto-correlation structure depending on the actual number of days between sampling times i.e., smaller correlations between sequential values for longer times between sampling episodes.</p>
    <p>The classification scheme for response to therapy was based on Q-cytogenetics and was similar to the Italian Cooperative Group (Italian Cooperative Study Group on Chronic Myeloid Leukemia <i>New England Journal of Medicine </i>30:820 (1994)) i.e., no response, minimal, minor, major and complete remission when 100%, 99-67%, 66-33%, 32-1% and 0% of metaphases are Ph positive, respectively.</p>
    <heading>Probe Sets in a D-FISH Assay Demonstrate Higher Sensitivity Than Standard Cytogenetic Testing</heading> <p>The goal was to study the effectiveness of the P5161-DC probes using 500 nuclei for each bone marrow and blood specimen. The goal for Q-cytogenetics was to study 25 metaphases from each bone marrow specimen. The goal for hypermetaphase studies was to study 200 metaphases from bone marrow. D-FISH was successful on 37/37 blood specimens and 37/37 bone marrow specimens. Q-cytogenetic was successful in 32/37 bone marrow specimens. Hypermetaphase was successful in 14/24 bone marrow specimens.</p>
    <heading>Very Low False Positive Rate (&lt;1.0%)</heading> <p>Based on 500 nuclei from each of 10 normal bone marrow specimens, the mean percentage and standard deviation of nuclei with false BCR/ABL1 fusion was 0.1%±0.1 (range 0 to 1 per 500 nuclei). Based on 500 nuclei from each of 10 normal peripheral blood specimens, the mean percentage and standard deviation of nuclei with false BCR/ABL1 fusion was 0.04%±0.08. Based on this data, the upper bound of a one-sided 95% confidence interval for observing 1 to 500 (0.2%) neoplastic cells in either bone marrow or peripheral blood was calculated using the binomial distribution. For both bone marrow and peripheral blood, this calculation implied a cutoff greater than 4/500 (&gt;0.8%) nuclei with BCR/ABL1 fusion to classify any specimen as abnormal.</p>
    <heading>Abnormal Reference Range for D-FISH in Untreated CML</heading> <p>The results of D-FISH for specimens from seven patients (nos. 2-7, 9) that were collected prior to treatment and that were not mosaic by Q-cytogenetic studies were used to establish an abnormal reference range. These specimens generally represent patients with untreated CML in clinical practice. Among these seven specimens, the mean percentage of abnormal cells was 97.6%±1.38 (range 95.4 to 99.0) for bone marrow, and 86.1%±13.59 (range 61.6 to 98.5) for blood.</p>
    <heading>Serial Dilutions</heading> <p>The observed percentage of neoplastic cells in each of the four serial dilution specimens was 97.6, 49.2, 8.2 and 1.8. The expected mean percentage of neoplastic cells in these specimens was 98.2, 49.1, 10.7, and 2.8, respectively. The difference between observed and expected values for each of these specimens was 0.6%, 0.1%, 2.5% and 1.0%, respectively.</p>
    <heading>Results of Using the Probe Set in a D-FISH Assay With Clinical Specimens</heading> <p>Results for Q-cytogenetic studies for bone marrow, and D-FISH for bone marrow and blood for each patient specimen are shown in FIG. <b>3</b>. Based on Q-cytogenetics, three patients (nos. 4, 5 and 6) achieved a complete cytogenetic remission, one patient (no. 3) briefly achieved a major response and the rest of the patients were classified as minimal, minor or non-responders.</p>
    <p>Each bone marrow specimen that had any abnormal metaphases by Q-cytogenetics was also abnormal for interphase nuclei by D-FISH in blood and bone marrow. Six specimens from three patients (nos. 4, 5 and 6) had only normal metaphases by Q-cytogenetics. For patient 6, D-FISH results were abnormal at 357 days in both bone marrow (4.8% abnormal nuclei) and blood (3.0% abnormal nuclei). For patient 5 at 262 days, the peripheral blood was marginally abnormal (1.0% abnormal nuclei) but bone marrow was within normal limits (0.6% abnormal nuclei). Each of the remaining four specimens with only normal metaphases by Q-cytogenetics were within normal limits for D-FISH in both bone marrow and blood.</p>
    <heading>Detection of Minimal Residual Disease States and Tracking Response to Therapy using the Probe Sets in FISH</heading> <p>Additional studies on the paired-sets of bone marrow and blood specimens that were normal by Q-cytogenetics and D-FISH were done to look for minimal residual disease. In a blind study, D-FISH was used to score 6,000 nuclei from four of the bone marrow specimens and five of the peripheral blood specimens in this series (Table 3, FIG. <b>9</b>), and 3 blood and bone marrow specimens from normal individuals. In a separate study, the normal range for D-FISH for 6,000 nuclei was calculated to be &lt;0.079%. Based on this cutoff, each of the normal blood and bone marrow specimens was correctly classified as normal. Three of the four patient bone marrow specimens and each of the patient peripheral blood specimens had minimal residual disease. It was not possible to do further studies on bone marrow no. 5 from patient 4 as this specimen had no leftover cells. The paired-blood specimen for this sampling time was in the abnormal range for D-FISH when 6,000 nuclei were studied and the bone marrow and one Ph positive metaphase among 169 metaphases that were examined by hypermetaphase FISH studies.</p>
    <p>The actual proportions of neoplastic cells from bone marrow specimens were plotted against the corresponding proportions from peripheral blood samples (FIG. <b>2</b>). The results imply that the proportion of abnormal cells from bone marrow specimens was typically greater (above y=x line) than for peripheral blood.</p>
    <p>For D-FISH, the mean 4 month inter-sample differences in percentage of abnormal nuclei between paired-sets of bone marrow and peripheral blood were not statistically different (p&gt;0.3)(Table 4, FIG. <b>10</b>). The deltas for D-FISH for peripheral blood were associated (p&lt;0.05) with the transformed proportion of abnormal cells based on Q-cytogenetics of the paired bone marrow specimen. This is important because Q-cytogenetics of bone marrow is widely recognized as the “gold standard” for monitoring response to interferon therapy.</p>
    <p>Based on these results, an additional regression analysis was done to develop a model for estimating the proportion of abnormal cells that would be obtained from bone marrow specimens using D-FISH results from peripheral blood samples. This is regression analysis of the data displayed in FIG. 4, but used the transformed values of the proportions (FIG. <b>3</b>). In FIG. 5, the dashed lines represent an approximate 95% confidence interval for a new predicted observation given a (new) peripheral blood value (prediction interval). This analysis indicated a significant (p&lt;0.001) linear relationship and yields the following equation for estimating the proportion of abnormal cells in bone marrow</p>
    <p>
      <maths> <formula-text>specimens (P<sub>BM</sub>), {circumflex over (P)}<sub>BM</sub>=[sin{0.1494+1.0324*sin<sup>1</sup>({square root over (P<sub>PB</sub>+L )})}]<sup>2</sup>,</formula-text> </maths> </p>
    <p>where P<sub>PB </sub>is the proportion of abnormal cells based on D-FISH results in peripheral blood samples. This relationship is displayed in FIG. 6, and the numeric results for several choices of P<sub>PB </sub>is listed in Table 5.</p>
    <heading>Discussion</heading> <p>The 4-month inter-sample changes in percentage of neoplastic nuclei in blood agreed closely with the corresponding intersample changes in percentage of neoplastic metaphases and nuclei in bone marrow over the course of interferon α-2b therapy. The reduction in percentage of Ph positive metaphases correlates with a prolonged chronic phase and increased survival in CML and the results of D-FISH on blood correlates with Q-cytogenetics. This demonstrates that using probes according to the present invention in a FISH assay is efficacious to test periodic peripheral blood specimens from patients with CML to monitor the effectiveness of interferon therapy. The analysis of 500 nuclei with the P5161-DC probe set in a D-FISH in bone marrow and peripheral blood detects &lt;1% disease and is at least as sensitive as Q-cytogenetics. Thus, D-FISH analyses of interphase nuclei using probe constructs according to the present invention could substitute for Q-cytogenetics for purposes of monitoring response to therapy for CML. By analyzing 6,000 nuclei in specimens that were normal by Q-cytogenetics and by D-FISH based on analysis of 500 nuclei revealed evidence of residual disease was found (Table 4, FIG. <b>10</b>). Thus, the methods and probe sets of the invention have the potential to detect very low levels of minimal disease in both blood and bone marrow.</p>
    <p>In one other experiment that compares the results of FISH studies of paired-sets of bone marrow and peripheral blood to monitor therapy in CML, Muhlmann et al, <i>Genes, Chromosomes and Cancer </i>21:90 (1998) used S-FISH to study 49 peripheral blood smears and 30 bone marrow specimens from 36 patients in chronic phase CML at different stages of cytogenetic remission. This experiment establishes that one can use whole blood as a comparative measure for events in the bone marrow.</p>
    <p>The present invention precisely predicts the percentage of neoplastic nuclei in bone marrow based on data from blood. This should allow one to use blood to monitor therapy in clinical practice. The results presented in the present specification indicate that it is best to assess response to therapy based on changes in percentage of neoplastic nuclei using the same tissue over time. In other words, to compare D-FISH results among blood studies or among bone marrow studies, but not between blood and bone marrow studies. This is important because the percentage of abnormal nuclei in blood and bone marrow differs in most patients at most times before and after therapy (FIG. <b>4</b>).</p>
    <p>The results show a strong correlation between changes in the percentage of Ph positive metaphases by Q-cytogenetic studies over the course of therapy and changes in the percentage of interphase nuclei with BCR/ABL1 fusion in both blood and bone marrow. D-FISH using the probes of the invention was also useful to identify residual disease in both bone marrow and peripheral blood specimens for patients in complete cytogenetic remission. For patients on therapy, D-FISH could then be performed on peripheral blood at periodic intervals to assess the effectiveness of therapy. Consequently, bone marrow would not need to be collected to monitor therapy as frequently or at all as it is in current practice.</p>
    <p>More details regarding scoring and correlation to clinical patients may be found in Dewald et al, Blood 31(9): 3357-3365 (1998).</p>
    <heading>EXAMPLE 3: CONSTRUCTION OF AML1/ETO DUAL FUSION PROBES</heading> <p>The AML1/ETO also called MTG8/CDR dual probes were assembled using the same method as in EXAMPLE 1 above. The highlights being illustrated below. The breakpoints are known to be clustered, Miyoshi et al (1991), Erickson et al (1992), Shimizu et al (1992), and Tighe et al (1993). The translocation has traditionally been detected using reverse transcriptase mediated polymerase chain reaction.</p>
    <p>Two overlapping YACs, 902G10 and 903A9 were isolated from a total human library using an ETO cDNA probe. The YACs spanned the entire 8q22 breakpoint region. YAC C14B2 is predominantly located proximal to the 21q22 breakpoint region. YAC 925E1 was obtained from a total human library and includes a region located immediately distal to the breakpoint region.</p>
    <p>The YAC DNAs 902G10 and 903A9 were labeled by nick translation with digoxigenin and C14B2 and 925E1 were labeled with biotin. FITC was used to detect biotin labeled probe molecules and rhodamine was used to detect digoxigenin labeled probe molecules using detection kits (Oncor, Inc.)</p>
    <heading>EXAMPLE 4: D-FISH FOR THE AML1/ETO TRANSLOCATION</heading> <p>The methods of Example 2 were repeated using the probe set of Example 3 with AML cell line Kasumi-1, lymphoblastoid cell line GM09948, bone marrow. Excellent results were obtained either two clear fusion signals being seen in a large percentage of cells. Details may be seen in Paskulin et al, Genes, Chromosomes &amp; Cancer 21:144-151 (1998). The method of Example 2 is also performed on peripheral blood cells and correlated to the bone marrow data.</p>
    <p>References cited herein are hereby incorporated by reference, and are listed below for convenience:</p>
    <heading>REFERENCES</heading> <p>Bentz, M., Cabot, G., Moos, M., Speicher, M. R., Ganser, A., Lichter, P. and Dohner, H. (1994) <i>Blood </i>83: 1922.</p>
    <p>Chumakov et al, <i>Nature </i>377:175-197 (1995).</p>
    <p>Dewald, GW, Juneau, AL, Schad, CR, Tefferi, A <i>Cancer Cytogenet</i>. 94:59 (1997).</p>
    <p>Dewald et al, <i>Blood </i>31(9): 3357-3365 (1998).</p>
    <p>Fischer et al, <i>Blood </i>88: 3962-3971 (1996).</p>
    <p>Italian Cooperative Study Group on Chronic Myeloid Leukemia <i>New England Journal of Medicine </i>30:820 (1994).</p>
    <p>Kurzrock et al, <i>New England Journal of Medicine </i>319:990 (1988).</p>
    <p>Le Beau, <i>Blood </i>81: 1979-1983 (1993).</p>
    <p>Lion, T, Monitoring of Residual Disease in Chronic Myelogenous Leukaemia: Methodological Approaches and Clinical Aspects <i>Leukaemia </i>10: 896 (1996).</p>
    <p>Muhlmann, J., Thaler, H. Hilbe, W, Bechter, O, Erdel, M., Utermann, G. Duba, HC <i>Genes, Chromosomes and Cancer </i>21:90.(1988).</p>
    <p>Nakane et al, <i>Acta Histochem. Cytochem</i>. 20(2):229 (1987).</p>
    <p>Pinkel et al, <i>Proc. Natl. Acad. Sci., U.S.A</i>. 83:2934-2938 (1986).</p>
    <p>Paskulin et al, <i>Genes, Chromosomes &amp; Cancer </i>21:144-151 (1998).</p>
    <p>Rudkin et al, <i>Nature </i>265:472-3 (1977).</p>
    <p>Sacchi et al, <i>Cancer Genetics and Cytogenetics </i>79: 97-103 (1995)</p>
    <p>Sinclair, P. B., Green, A. R., Grace, C., Nacheva, E. P., <i>Blood </i>90:1395 (1997),</p>
    <p>Seong, D C, Kantarjian, H M, Ro, J Y, Ralpaz, M, Xu, J. Robinson, J R, Deisseroth, A B, Champlin, R E, Siciliano, M J, <i>Blood </i>86:2343 (1995).</p>
    <p>The present invention is not to be limited in scope by the specific embodiments described herein. Indeed, various modifications of the invention in addition to those described herein will become apparent to those skilled in the art from the foregoing description and accompanying drawings. Such modifications are intended to fall within the scope of the appended claims.</p>
    <p>Various publications are cited herein, the disclosures of which are incorporated by reference in their entireties.</p>
    </div></div></div><div class="patent-section patent-tabular-section"><a id="backward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Cited Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4999290">US4999290</a></td><td class="patent-data-table-td patent-date-value">Mar 31, 1988</td><td class="patent-data-table-td patent-date-value">Mar 12, 1991</td><td class="patent-data-table-td ">The Board Of Regents, The University Of Texas System</td><td class="patent-data-table-td ">Detection of genomic abnormalities with unique aberrant gene transcripts</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5487970">US5487970</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jun 17, 1993</td><td class="patent-data-table-td patent-date-value">Jan 30, 1996</td><td class="patent-data-table-td ">Arch Development Corp.</td><td class="patent-data-table-td ">Compositions and methods for detecting gene rearrangements and translocations</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5538869">US5538869</a></td><td class="patent-data-table-td patent-date-value">Jun 1, 1993</td><td class="patent-data-table-td patent-date-value">Jul 23, 1996</td><td class="patent-data-table-td ">Board Of Regents, The University Of Texas System</td><td class="patent-data-table-td ">In-situ hybridization probes for identification and banding of specific human chromosomes and regions</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5567586">US5567586</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">May 18, 1995</td><td class="patent-data-table-td patent-date-value">Oct 22, 1996</td><td class="patent-data-table-td ">Thomas Jefferson University</td><td class="patent-data-table-td ">Obtaining sample of solid tumor and detecting presence of all-1 gene rearrangement or mutation</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="npl-citations"></a><div class="patent-section-header"><span class="patent-section-title">Non-Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th colspan="3"class="patent-data-table-th">Reference</th></tr></thead><tr><td class="patent-data-table-td ">1</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">1996/1997 Catalog, Vysis, Inc. 3100 Woodcreek Drive, Downers Grove, IL pp. 25 and 30.</td></tr><tr><td class="patent-data-table-td ">2</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Dewald et al, Blood, 91(9); pp. 3357-3365 (May 1, 1998).</td></tr><tr><td class="patent-data-table-td ">3</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Dewald et al. Highly sensitive fluorescence in situ hybridization method to detect double BCR/ABL fusion and monitor response to therapy in chronic myeloid leukemia. Blood. vol. 91, No. 9, pp. 3357-3365, May 1998.*</td></tr><tr><td class="patent-data-table-td ">4</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Fischer et al. Design and Validation of DNA Probe Sets for a Comprehensive Interphase Cytogenetic Analysis of Acute Myeloid Leukemia. Blood. vol. 88, pp. 3962-3971, Nov. 1996.*</td></tr><tr><td class="patent-data-table-td ">5</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Oncor 1992-1993 Product Catalog, p. 11 Oncor, Inc. 209 Perry Pky. Gaithersburg, MD.</td></tr><tr><td class="patent-data-table-td ">6</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Paskulin et al, Genes, Chromosomes &amp; Cancer 21: 144-151 (1998).</td></tr><tr><td class="patent-data-table-td ">7</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Paskulin et al. Pre-Clinical Evaluation of Probes to detect t(8;21) AML minimal residual disease by fluorescence in situ hybridization. Genes, Chromosome and Cancer vol. 21, pp. 144-151, Feb. 1998.</td></tr><tr><td class="patent-data-table-td ">8</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">TKachuk et al. Detection of BCR-ABL Fusion in Chronic Myelogeneous Leukemia by in situ Hybridization. Science. vol. 250, pp. 559-562, Nov. 1990.*</td></tr><tr><td class="patent-data-table-td ">9</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">TKachuk et al. Detection of BCR-ABL Fusion in Chronic Myelogeneous Leukemia by in situ Hybridization. Science. vol. 250, pp. 559-562, Oct. 1990.*</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="forward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Referenced by</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Citing Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7306916">US7306916</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">May 3, 2005</td><td class="patent-data-table-td patent-date-value">Dec 11, 2007</td><td class="patent-data-table-td ">Dako Denmark A/S</td><td class="patent-data-table-td ">Methods for detecting chromosome aberrations</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7695929">US7695929</a></td><td class="patent-data-table-td patent-date-value">Nov 1, 2007</td><td class="patent-data-table-td patent-date-value">Apr 13, 2010</td><td class="patent-data-table-td ">Ventana Medical Systems, Inc.</td><td class="patent-data-table-td ">Haptens, hapten conjugates, compositions thereof and method for their preparation and use</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7985557">US7985557</a></td><td class="patent-data-table-td patent-date-value">May 22, 2008</td><td class="patent-data-table-td patent-date-value">Jul 26, 2011</td><td class="patent-data-table-td ">Ventana Medical Systems, Inc.</td><td class="patent-data-table-td ">Polymeric carriers for immunohistochemistry and in situ hybridization</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8076074">US8076074</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Nov 29, 2005</td><td class="patent-data-table-td patent-date-value">Dec 13, 2011</td><td class="patent-data-table-td ">Quest Diagnostics Investments Incorporated</td><td class="patent-data-table-td ">Balanced translocation in comparative hybridization</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8124351">US8124351</a></td><td class="patent-data-table-td patent-date-value">Nov 14, 2006</td><td class="patent-data-table-td patent-date-value">Feb 28, 2012</td><td class="patent-data-table-td ">Board Of Regents, The University Of Texas System</td><td class="patent-data-table-td ">Quantification of fusion proteins and their activity from chromosomal translocation</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8415464">US8415464</a></td><td class="patent-data-table-td patent-date-value">Jan 22, 2001</td><td class="patent-data-table-td patent-date-value">Apr 9, 2013</td><td class="patent-data-table-td ">The Regents Of The University Of California</td><td class="patent-data-table-td ">Chromosome-specific staining to detect genetic rearrangements</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8445191">US8445191</a></td><td class="patent-data-table-td patent-date-value">May 2, 2011</td><td class="patent-data-table-td patent-date-value">May 21, 2013</td><td class="patent-data-table-td ">Ventana Medical Systems, Inc.</td><td class="patent-data-table-td ">Polymeric carriers for immunohistochemistry and in situ hybridization</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8486620">US8486620</a></td><td class="patent-data-table-td patent-date-value">Apr 29, 2011</td><td class="patent-data-table-td patent-date-value">Jul 16, 2013</td><td class="patent-data-table-td ">Ventana Medical Systems, Inc.</td><td class="patent-data-table-td ">Polymeric carriers for immunohistochemistry and in situ hybridization</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8592155">US8592155</a></td><td class="patent-data-table-td patent-date-value">Jun 30, 2003</td><td class="patent-data-table-td patent-date-value">Nov 26, 2013</td><td class="patent-data-table-td ">The Regents Of The University Of California</td><td class="patent-data-table-td ">Method of detecting genetic deletions identified with chromosomal abnormalities</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8618265">US8618265</a></td><td class="patent-data-table-td patent-date-value">Mar 2, 2010</td><td class="patent-data-table-td patent-date-value">Dec 31, 2013</td><td class="patent-data-table-td ">Ventana Medical Systems, Inc.</td><td class="patent-data-table-td ">Haptens, hapten conjugates, compositions thereof and method for their preparation and use</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8703490">US8703490</a></td><td class="patent-data-table-td patent-date-value">Jun 1, 2009</td><td class="patent-data-table-td patent-date-value">Apr 22, 2014</td><td class="patent-data-table-td ">Ventana Medical Systems, Inc.</td><td class="patent-data-table-td ">Compositions comprising nanomaterials and method for using such compositions for histochemical processes</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2006048265A2?cl=en">WO2006048265A2</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Nov 3, 2005</td><td class="patent-data-table-td patent-date-value">May 11, 2006</td><td class="patent-data-table-td ">Roche Diagnostics Gmbh</td><td class="patent-data-table-td ">Classifying leukemia with translocation (9;22)</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2007064534A1?cl=en">WO2007064534A1</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Nov 21, 2006</td><td class="patent-data-table-td patent-date-value">Jun 7, 2007</td><td class="patent-data-table-td ">Quest Diagnostics Invest Inc</td><td class="patent-data-table-td ">Balanced translocation in comparative hybridization</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="classifications"></a><div class="patent-section-header"><span class="patent-section-title">Classifications</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th"> </th><th class="patent-data-table-th"> </th></tr></thead><tr><td class="patent-data-table-td ">U.S. Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=z89aBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S024300">536/24.3</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=z89aBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S024330">536/24.33</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=z89aBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S023100">536/23.1</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=z89aBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S024320">536/24.32</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=z89aBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S091100">435/91.1</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=z89aBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S024310">536/24.31</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=z89aBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S006140">435/6.14</a></span></td></tr><tr><td class="patent-data-table-td ">International Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=z89aBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12Q0001680000">C12Q1/68</a></span></td></tr><tr><td class="patent-data-table-td ">Cooperative Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=z89aBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12Q1/6841">C12Q1/6841</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=z89aBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12Q1/6827">C12Q1/6827</a></span></td></tr><tr><td class="patent-data-table-td ">European Classification</td><td class="patent-data-table-td "><span class="nested-value">C12Q1/68B6</span>, <span class="nested-value">C12Q1/68B14</span></td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="legal-events"></a><div class="patent-section-header"><span class="patent-section-title">Legal Events</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Date</th><th class="patent-data-table-th">Code</th><th class="patent-data-table-th">Event</th><th class="patent-data-table-th">Description</th></tr></thead><tr><td class="patent-data-table-td patent-date-value">Dec 30, 2013</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">12</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Dec 22, 2009</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">8</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Apr 14, 2009</td><td class="patent-data-table-td ">B1</td><td class="patent-data-table-td ">Reexamination certificate first reexamination</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">CLAIMS 1-19 ARE CANCELLED. NEW CLAIMS 20-29 ARE ADDED AND DETERMINED TO BE PATENTABLE.</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Jan 3, 2006</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">4</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Feb 22, 2005</td><td class="patent-data-table-td ">DI</td><td class="patent-data-table-td ">Adverse decision in interference</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20041222</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">May 18, 2004</td><td class="patent-data-table-td ">RR</td><td class="patent-data-table-td ">Request for reexamination filed</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20040407</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Jul 29, 2003</td><td class="patent-data-table-td ">CC</td><td class="patent-data-table-td ">Certificate of correction</td><td class="patent-data-table-td "></td></tr><tr><td class="patent-data-table-td patent-date-value">Jan 24, 2002</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">VENTANA MEDICAL SYSTEMS, INC., ARIZONA</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNORS:DIETZ-BAND, JEANNE;HSIEH, WANG-TING;CONNAUGHTON, JOHN F.;REEL/FRAME:012552/0450</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20011221</span></div><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">VENTANA MEDICAL SYSTEMS, INC. 1910 INNOVATION PARK</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNORS:DIETZ-BAND, JEANNE /AR;REEL/FRAME:012552/0450</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Jan 25, 1999</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">VENTANA MEDICAL SYSTEMS, INC., ARIZONA</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNOR:ONCOR, INC.;REEL/FRAME:009719/0214</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">19981120</span></div></td></tr></table><div class="patent-section-footer"></div></div><div class="modal-dialog" id="patent-images-lightbox"><div class="patent-lightbox-controls"><div class="patent-lightbox-rotate-controls"><div class="patent-lightbox-rotation-text">Rotate</div><div class="rotate-icon rotate-ccw-icon"></div><div class="rotate-icon rotate-cw-icon"></div></div><div class="patent-lightbox-index-counter"></div><a class="patent-lightbox-fullsize-link" target="_blank">Original Image</a><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_right.png" alt="Next page"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_left.png" alt="Previous page"width="21" height="21" /></div></div><div class="modal-dialog-content"><div class="patent-lightbox-image-holder"><div class="patent-lightbox-placeholder"></div></div></div></div><script>_OC_initPatentsAtb({image_not_available_html: " Image not available"});</script></div></div></div></td></tr></table><script>(function() {var href = window.location.href;if (href.indexOf('?') !== -1) {var parameters = href.split('?')[1].split('&');for (var i = 0; i < parameters.length; i++) {var param = parameters[i].split('=');if (param[0] == 'focus') {var elem = document.getElementById(param[1]);if (elem) {elem.focus();}}}}})();</script><script>_OC_addFlags({LockSrc:"/books/javascript/lock_8a2b04e7bf975d5171d8e4c0b6365c7a.js", Host:"http://www.google.com/", IsBooksRentalEnabled:1, IsWebstoreDisplayCaseEnabled:1, IsObfuscationEnabled:1, IsBrowsingHistoryEnabled:1, IsWebReaderSvgEnabled:0, IsGeoLayerEnabled:1, IsImageModeNotesEnabled:1, IsCopyMenuItemEnabled:1, IsGiftingEnabled:0, IsWebReaderUniversalPaginatorEnabled:0, IsOfflineBubbleEnabled:1, IsReaderEnabledForPlayRequests:1, IsFutureOnSaleVolumesEnabled:1, IsOfflineRestrictedCopyEnabled:1, IsBooksUnifiedLeftNavEnabled:1, IsRestrictedCopyEnabled:1, IsZipitFolderCollectionEnabled:1, IsEndOfSampleRecommendationsEnabled:1, IsRatingsOnBookcardsEnabled:1, IsAdsDisabled:0, IsIframePageDisplayEnabled:0, IsEmbeddedMediaEnabled:1, IsImageModeAnnotationsEnabled:1, IsMyLibraryGooglePlusEnabled:1, IsImagePageProviderEnabled:0, IsBookcardListPriceSmall:0, IsInternalUser:0, IsBooksShareButtonEnabled:0, IsPreOrdersEnabled:0, IsDisabledRandomBookshelves:0, WebstoreDisplayCasePosition:3});_OC_Run({"enable_p13n":false,"add_vol_to_collection_base_url":"http://www.google.com/patents?op=add\u0026sig=ACfU3U1rTs8KkocXZNgcoQut7-22j0ByoA\u0026id=z89aBAABERAJ","remove_vol_from_collection_base_url":"http://www.google.com/patents?op=remove\u0026sig=ACfU3U3STl5gzjzqcKHJFLjbed31I3jqoQ\u0026id=z89aBAABERAJ","logged_in":false,"p13n_save_user_settings_url":"http://www.google.com/patents?op=edit_user_settings\u0026sig=ACfU3U2BVASBDCD6yC-sZ_E6ybSTR8D9NQ","is_cobrand":false,"sign_in_url":"https://www.google.com/accounts/Login?service=\u0026continue=http://www.google.com/patents%3Fhl%3Den\u0026hl=en","is_play_enabled":true}, {"volume_id":"","is_ebook":true,"volumeresult":{"has_flowing_text":false,"has_scanned_text":true,"can_download_pdf":false,"can_download_epub":false,"is_pdf_drm_enabled":false,"is_epub_drm_enabled":false,"download_pdf_url":"http://www.google.com/patents/download/Complementary_to_breakpoint_region_of_dn.pdf?id=z89aBAABERAJ\u0026output=pdf\u0026sig=ACfU3U1erokDGivWFBgGFP2Vl_URndzATw"},"sample_url":"http://www.google.com/patents/reader?id=z89aBAABERAJ\u0026printsec=frontcover\u0026output=reader\u0026source=gbs_atb_hover","is_browsable":true,"is_public_domain":true}, {});</script><div id="footer_table" style="font-size:83%;text-align:center;position:relative;top:20px;height:4.5em;margin-top:2em"><div style="margin-bottom:8px"><a href=http://www.google.com/><nobr>Google&nbsp;Home</nobr></a> - <a href=//www.google.com/patents/sitemap/><nobr>Sitemap</nobr></a> - <a href=http://www.google.com/googlebooks/uspto.html><nobr>USPTO Bulk Downloads</nobr></a> - <a href=/intl/en/privacy/><nobr>Privacy Policy</nobr></a> - <a href=/intl/en/policies/terms/><nobr>Terms of Service</nobr></a> - <a href=https://support.google.com/faqs/answer/2539193?hl=en><nobr>About Google Patents</nobr></a> - <a href="http://www.google.com/tools/feedback/intl/en/error.html" onclick="try{_OC_startFeedback({productId: '72792',locale: 'en'});return false;}catch(e){}"><nobr>Send Feedback</nobr></a></div><span>Data provided by IFI CLAIMS Patent Services</span><br><span >&copy;2012 Google</span></div> <script type="text/javascript">var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));</script><script type="text/javascript">var pageTracker = _gat._getTracker("UA-27188110-1");pageTracker._setCookiePath("/patents/");pageTracker._trackPageview();</script> </body></html>