<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN""http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html><head><title>Patent US6451534 - KVLQT1—a long QT syndrome gene - Google Patents</title><script>(function(){(function(){function e(a){this.t={};this.tick=function(a,c,b){var d=void 0!=b?b:(new Date).getTime();this.t[a]=[d,c];if(void 0==b)try{window.console.timeStamp("CSI/"+a)}catch(e){}};this.tick("start",null,a)}var a;window.performance&&(a=window.performance.timing);var f=a?new e(a.responseStart):new e;window.jstiming={Timer:e,load:f};if(a){var c=a.navigationStart,d=a.responseStart;0<c&&d>=c&&(window.jstiming.srt=d-c)}if(a){var b=window.jstiming.load;0<c&&d>=c&&(b.tick("_wtsrt",void 0,c),b.tick("wtsrt_",
"_wtsrt",d),b.tick("tbsd_","wtsrt_"))}try{a=null,window.chrome&&window.chrome.csi&&(a=Math.floor(window.chrome.csi().pageT),b&&0<c&&(b.tick("_tbnd",void 0,window.chrome.csi().startE),b.tick("tbnd_","_tbnd",c))),null==a&&window.gtbExternal&&(a=window.gtbExternal.pageT()),null==a&&window.external&&(a=window.external.pageT,b&&0<c&&(b.tick("_tbnd",void 0,window.external.startE),b.tick("tbnd_","_tbnd",c))),a&&(window.jstiming.pt=a)}catch(g){}})();})();
</script><link rel="stylesheet" href="/patents/css/_8a2b04e7bf975d5171d8e4c0b6365c7a/kl_intl_patents_bundle.css" type="text/css" /><script src="/books/javascript/atb_8a2b04e7bf975d5171d8e4c0b6365c7a__en.js"></script><script>function googleTranslateElementInit() {new google.translate.TranslateElement({pageLanguage: "en",gaTrack: true,gaId: "UA-27188110-1",multilanguagePage: true});}</script><script src="//translate.google.com/translate_a/element.js?cb=googleTranslateElementInit"></script><meta name="DC.type" content="Patent"><meta name="DC.title" content="KVLQT1—a long QT syndrome gene"><meta name="DC.contributor" content="Mark T. Keating" scheme="inventor"><meta name="DC.contributor" content="Michael C. Sanguinetti" scheme="inventor"><meta name="DC.contributor" content="Mark E. Curran" scheme="inventor"><meta name="DC.contributor" content="Gregory M. Landes" scheme="inventor"><meta name="DC.contributor" content="Timothy D. Connors" scheme="inventor"><meta name="DC.contributor" content="Timothy C. Burn" scheme="inventor"><meta name="DC.contributor" content="Igor Splawski" scheme="inventor"><meta name="DC.contributor" content="University Of Utah Research Foundation" scheme="assignee"><meta name="DC.contributor" content="Genzyme Corporation" scheme="assignee"><meta name="DC.date" content="2000-6-19" scheme="dateSubmitted"><meta name="DC.description" content="The genomic structure including the sequence of the intron/exon junctions is disclosed for KVLQT1 and KCNE1 which are genes associated with long QT syndrome. Additional sequence data for the two genes ARE also disclosed. Also disclosed are newly found mutations in KVLQT1 which result in long QT syndrome. The intron/exon junction sequence data allow for the design of primer pairs to amplify and sequence across all of the exons of the two genes. This can be used to screen persons for the presence of mutations which cause long QT syndrome. Assays can be performed to screen persons for the presence of mutations in either the DNA or proteins. The DNA and proteins may also be used in assays to screen for drugs which will be useful in treating or preventing the occurrence of long QT syndrome."><meta name="DC.date" content="2002-9-17" scheme="issued"><meta name="DC.relation" content="WO:1997023598:A2" scheme="references"><meta name="citation_reference" content="Ackerman, M.J. &quot;The Long QT Syndrome: Ion Channel Diseases of the Heart&quot;, Mayo Clin. Proc., 1998; 73:250-269."><meta name="citation_reference" content="Barhanin, J. et al. (1997). GenBank Accession No. AF000571.1."><meta name="citation_reference" content="Benhorin,J. et al. &quot;Evidence of Genetic Heterogeneity in the Long QT Syndrome&quot;, Science, Jun. 25, 1993; 260:1960-1962."><meta name="citation_reference" content="Bulman &quot;Phenotype variation and newcomers in ion channel disorders&quot;, Human Molecular Genetics, 1997; 6(10) Review:1679-1685."><meta name="citation_reference" content="Chouabe, C. et al. &quot;Properties of KVLQT1 K+ channel mutations in Romano-Ward and Jervell and Lange-Nielsen inherited cardiac arrhythmias&quot;, The EMBO Journal, 1997; 16(17):5472-5479."><meta name="citation_reference" content="Coonar, et al. &quot;Molecular Genetics of Familial Cardiomyopathies&quot;, Advances in Genetics, _; 35:285-324."><meta name="citation_reference" content="Curran, M. et al. &quot;Locus Heterogeneity of Autosomal Dominant Long QT Syndrome&quot;, J. Clin. Invest., Aug. 1993; 92:799-803."><meta name="citation_reference" content="de Jager, et al. &quot;Evidence of a long QT founder gene with varying phenotypic expression in South African families&quot;, J. Med. Genet., 1996; 33:567-573."><meta name="citation_reference" content="Donger, C. et al. &quot;KVLQT1 C-Terminal Missense Mutation Causes a Forme Fruste Long-QT Syndrome&quot;, Circulation, 1997; 96:2778-2781."><meta name="citation_reference" content="GenBank Accession No. U86146 (1997)."><meta name="citation_reference" content="Hoffman, et al. &quot;Ion Channels-Molecular Divining Rods Hit Their Clinical Mark&quot;, New England Journal of Medicine, May 29, 1997; 336(22):1599-1600."><meta name="citation_reference" content="Hoffman, et al. “Ion Channels—Molecular Divining Rods Hit Their Clinical Mark”, New England Journal of Medicine, May 29, 1997; 336(22):1599-1600."><meta name="citation_reference" content="Itoh, T. et al. &quot;Genomic organization and mutational analysis of KVLQT1, a gene responsible for familial long QT syndrome&quot;, Hum. Genet., 1998; 103:290-294."><meta name="citation_reference" content="Keating, et al. &quot;Consistent Linkage of the Long-QT Syndrome to the Harvey Ras-I Locus on Chromosome II&quot;, Am. J. Hum. Genet., 1991; 49:1335-1339."><meta name="citation_reference" content="Keating, et al. &quot;Linkage of a Cardiac Arrhythmia, the Long QT Syndrome, and the Harvey ras-1 Gene&quot;, Science, May 3, 1991; 252:704-706."><meta name="citation_reference" content="Keating, M. &quot;Linkage Analysis and Long QT Syndrome Using Genetics to Study Cardiovascular Disease&quot;, Circulation, 1992; 85:1973-1986."><meta name="citation_reference" content="Santana et al. Bacil;lus subtilis FOF1 ATPase: DNA Sequence of the atp Operon and Characterization of atp Mutants. Nov. 1994. J. of Bacteriol. 176(22):6802-6811.*"><meta name="citation_reference" content="van den Berg et al. &quot;The long QT Syndrome: a novel missense mutation in the S6 region of the KVLQT1 gene&quot;, Hum. Genet., 1997; 100:356-361."><meta name="citation_reference" content="Vincent &quot;Genetics and Molecular Biology of the Inherited Long QT Syndrome&quot;, Annals of Medicine, 1994; 26:419-425."><meta name="citation_reference" content="Vincent &quot;The Molecular Genetics of the Long QT Syndrome: Genes causing fainting and sudden death&quot;, Annu. Rev. Med., 1998; 49:263-274."><meta name="citation_reference" content="Wang, Q. et al. &quot;Molecular genetics of long QT syndrome from genes to patients&quot;, Current Opinion in Cardiology, 1997; 12:310-320."><meta name="citation_reference" content="Wang, Q. et al. &quot;Positional cloning of a novel potassium channel gene: KVLQT1 mutations cause cardiac arrhythmias&quot;, Nature Genetics, Jan. 1996; 12:17-23."><meta name="citation_reference" content="Wollnik, et al. &quot;Pathophysiological mechanisms of dominant and recessive KVLQT1 K+ channel mutations found in inherited cardiac arrhythmias&quot;, Human Molecular Genetics, 1997; 6(11):1943-1949."><meta name="citation_reference" content="Yang, et al. &quot;KVLQT1, a voltage-gated potassium channel responsible for human cardiac arrhythmias&quot;, Proc. Natl. Acad. Sci. USA, Apr. 1997; 94:4017-4021."><meta name="citation_patent_number" content="US:6451534"><meta name="citation_patent_application_number" content="US:09/597,732"><link rel="canonical" href="http://www.google.com/patents/US6451534"/><meta property="og:url" content="http://www.google.com/patents/US6451534"/><meta name="title" content="Patent US6451534 - KVLQT1—a long QT syndrome gene"/><meta name="description" content="The genomic structure including the sequence of the intron/exon junctions is disclosed for KVLQT1 and KCNE1 which are genes associated with long QT syndrome. Additional sequence data for the two genes ARE also disclosed. Also disclosed are newly found mutations in KVLQT1 which result in long QT syndrome. The intron/exon junction sequence data allow for the design of primer pairs to amplify and sequence across all of the exons of the two genes. This can be used to screen persons for the presence of mutations which cause long QT syndrome. Assays can be performed to screen persons for the presence of mutations in either the DNA or proteins. The DNA and proteins may also be used in assays to screen for drugs which will be useful in treating or preventing the occurrence of long QT syndrome."/><meta property="og:title" content="Patent US6451534 - KVLQT1—a long QT syndrome gene"/><meta property="og:type" content="book"/><meta property="og:site_name" content="Google Books"/><meta property="og:image" content="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><link rel="image_src" href="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><script>(function(){try{var aa=function(a,b,c,d){d=d||{};d._sn=["cfg",b,c].join(".");window.gbar.logger.ml(a,d)};var g=window.gbar=window.gbar||{},l=window.gbar.i=window.gbar.i||{},m={},n;function _tvn(a,b){var c=parseInt(a,10);return isNaN(c)?b:c}function _tvf(a,b){var c=parseFloat(a);return isNaN(c)?b:c}function _tvv(a){return!!a}function p(a,b,c){(c||g)[a]=b}g.bv={n:_tvn("2",0),r:"",f:".67.",e:"0",m:_tvn("0",1)};
function q(a,b,c){var d="on"+b;if(a.addEventListener)a.addEventListener(b,c,!1);else if(a.attachEvent)a.attachEvent(d,c);else{var f=a[d];a[d]=function(){var a=f.apply(this,arguments),b=c.apply(this,arguments);return void 0==a?b:void 0==b?a:b&&a}}}var s=function(a){return function(){return g.bv.m==a}},ba=s(1),ca=s(2);p("sb",ba);p("kn",ca);l.a=_tvv;l.b=_tvf;l.c=_tvn;l.i=aa;var da=window.gbar.i.i;var t,u,v,w;function ea(a){v=a}function fa(a){var b;if(b=v&&window.encodeURIComponent)b=a.href,b=!b.match(/^http[s]?:\/\/accounts\.google\.[^/]*\/ClearSID/i)&&!b.match(/^http[s]?:\/\/[^/]*\/accounts\/ClearSID/i);if(b=b&&encodeURIComponent(v()))a.href=a.href.replace(/([?&]continue=)[^&]*/,"$1"+b)}function ga(a){window.gApplication&&(a.href=window.gApplication.getTabUrl(a.href))}
function ha(a){var b=document.forms[0].q,c=window.encodeURIComponent&&b&&b.value,b=b&&b.placeholder;c&&c!=b&&(a.href=a.href.replace(/([?&])q=[^&]*|$/,function(a,b){return(b||"&")+"q="+encodeURIComponent(c)}))}n=l.a("")?ga:ha;
function x(a,b,c,d,f,e){var h=document.getElementById(a);if(h){var k=h.style;k.left=d?"auto":b+"px";k.right=d?b+"px":"auto";k.top=c+"px";k.visibility=u?"hidden":"visible";f&&e?(k.width=f+"px",k.height=e+"px"):(x(t,b,c,d,h.offsetWidth,h.offsetHeight),u=u?"":a)}}
var y=[],ia=function(a,b){y.push(b)},ja=function(a){a=a||window.event;var b=a.target||a.srcElement;a.cancelBubble=!0;null==t&&(a=document.createElement(Array.every||window.createPopup?"iframe":"div"),a.frameBorder="0",t=a.id="gbs",a.src="javascript:''",b.parentNode.appendChild(a),q(document,"click",z));var c=b,b=0;"gb3"!=c.className&&(c=c.parentNode);a=c.getAttribute("aria-owns")||"gbi";var d=c.offsetWidth,f=20<c.offsetTop?46:24;document.getElementById("tphdr")&&(f-=3);var e=!1;do b+=c.offsetLeft||
0;while(c=c.offsetParent);var c=(document.documentElement.clientWidth||document.body.clientWidth)-b-d,h,d=document.body,k=document.defaultView;k&&k.getComputedStyle?(d=k.getComputedStyle(d,""))&&(h=d.direction):h=d.currentStyle?d.currentStyle.direction:d.style.direction;h="rtl"==h;if("gbi"==a){for(d=0;k=y[d++];)k();A(null,window.navExtra);h&&(b=c,e=!0)}else h||(b=c,e=!0);u!=a&&z();x(a,b,f,e)},z=function(){u&&x(u,0,0)},A=function(a,b){var c,d=document.getElementById("gbi"),f=a;f||(f=d.firstChild);
for(;b&&(c=b.pop());){var e=d,h=c,k=f;w||(w="gb2");e.insertBefore(h,k).className=w}},ka=function(a,b,c){if((b=document.getElementById(b))&&a){a.className="gb4";var d=document.createElement("span");d.appendChild(a);d.appendChild(document.createTextNode(" | "));d.id=c;b.appendChild(d)}},la=function(){return document.getElementById("gb_70")},ma=function(){return!!u};p("qs",n);p("setContinueCb",ea);p("pc",fa);p("tg",ja);p("close",z);p("addLink",ka);p("almm",A);p("si",la);p("adh",ia);p("op",ma);var B=function(){},C=function(){},F=function(a){var b=new Image,c=D;b.onerror=b.onload=b.onabort=function(){try{delete E[c]}catch(a){}};E[c]=b;b.src=a;D=c+1},E=[],D=0;p("logger",{il:C,ml:B,log:F});var G=window.gbar.logger;var H={},na={},I=[],oa=l.b("0.1",.1),pa=l.a("1",!0),qa=function(a,b){I.push([a,b])},ra=function(a,b){H[a]=b},sa=function(a){return a in H},J={},K=function(a,b){J[a]||(J[a]=[]);J[a].push(b)},ta=function(a){K("m",a)},L=function(a,b){var c=document.createElement("script");c.src=a;c.async=pa;Math.random()<oa&&(c.onerror=function(){c.onerror=null;B(Error("Bundle load failed: name="+(b||"UNK")+" url="+a))});(document.getElementById("xjsc")||document.getElementsByTagName("body")[0]||
document.getElementsByTagName("head")[0]).appendChild(c)},N=function(a){for(var b=0,c;(c=I[b])&&c[0]!=a;++b);!c||c[1].l||c[1].s||(c[1].s=!0,M(2,a),c[1].url&&L(c[1].url,a),c[1].libs&&m.d&&m.d(c[1].libs))},O=function(a){K("gc",a)},P=null,ua=function(a){P=a},M=function(a,b,c){if(P){a={t:a,b:b};if(c)for(var d in c)a[d]=c[d];try{P(a)}catch(f){}}};p("mdc",H);p("mdi",na);p("bnc",I);p("qGC",O);p("qm",ta);p("qd",J);p("lb",N);p("mcf",ra);p("bcf",qa);p("aq",K);p("mdd","");p("has",sa);
p("trh",ua);p("tev",M);var Q=l.b("0.1",.001),R=0;
function _mlToken(a,b){try{if(1>R){R++;var c,d=a,f=b||{},e=encodeURIComponent,h=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&jexpid=",e("17483"),"&srcpg=",e("prop=22"),"&jsr=",Math.round(1/Q),"&ogev=",e("HaLtU7LQOtSysQTD_4CQAg"),"&ogf=",g.bv.f,"&ogrp=",e("1"),"&ogv=",e("1407723702.0"),"&oggv="+e("es_plusone_gc_20140723.0_p0"),"&ogd=",e("com"),"&ogc=",e("USA"),"&ogl=",e("en")];f._sn&&(f._sn="og."+
f._sn);for(var k in f)h.push("&"),h.push(e(k)),h.push("="),h.push(e(f[k]));h.push("&emsg=");h.push(e(d.name+":"+d.message));var r=h.join("");S(r)&&(r=r.substr(0,2E3));c=r;var Aa=window.gbar.logger._aem(a,c);F(Aa)}}catch(Na){}}var S=function(a){return 2E3<=a.length},va=function(a,b){return b};function T(a){B=a;p("_itl",S,G);p("_aem",va,G);p("ml",B,G);a={};H.er=a}l.a("")?T(function(a){throw a;}):l.a("1")&&Math.random()<Q&&T(_mlToken);I.push(["m",{url:"//ssl.gstatic.com/gb/js/scm_7385cc5883250b43a39405734c1bea59.js"}]);g.mcf("c",{});g.sg={c:""};if(l.a("1")){var wa=l.a("");I.push(["gc",{auto:wa,url:"//ssl.gstatic.com/gb/js/abc/gci_91f30755d6a6b787dcc2a4062e6e9824.js",libs:"googleapis.client:plusone:gapi.iframes"}]);var xa={version:"gci_91f30755d6a6b787dcc2a4062e6e9824.js",index:"",lang:"en"};H.gc=xa;var U=function(a){window.googleapis&&window.iframes?a&&a():(a&&O(a),N("gc"))};p("lGC",U);l.a("1")&&p("lPWF",U)};window.__PVT="";if(l.a("1")&&l.a("1")){var V=function(a){U(function(){K("pw",a);N("pw")})};p("lPW",V);I.push(["pw",{url:"//ssl.gstatic.com/gb/js/abc/pwm_45f73e4df07a0e388b0fa1f3d30e7280.js"}]);var W=[],ya=function(a){W[0]=a},za=function(a,b){var c=b||{};c._sn="pw";B(a,c)},Ba={signed:W,elog:za,base:"https://plusone.google.com/u/0",loadTime:(new Date).getTime()};H.pw=Ba;var X=function(a,b){for(var c=b.split("."),d=function(){var b=arguments;a(function(){for(var a=g,d=0,e=c.length-1;d<e;++d)a=a[c[d]];a[c[d]].apply(a,b)})},f=g,e=0,h=c.length-1;e<h;++e)f=
f[c[e]]=f[c[e]]||{};return f[c[e]]=d};X(V,"pw.clk");X(V,"pw.hvr");p("su",ya,g.pw)};function Ca(){function a(){for(var b;(b=e[h++])&&"m"!=b[0]&&!b[1].auto;);b&&(M(2,b[0]),b[1].url&&L(b[1].url,b[0]),b[1].libs&&m.d&&m.d(b[1].libs));h<e.length&&setTimeout(a,0)}function b(){0<f--?setTimeout(b,0):a()}var c=l.a("1"),d=l.a(""),f=3,e=I,h=0,k=window.gbarOnReady;if(k)try{k()}catch(r){da(r,"ml","or")}d?p("ldb",a):c?q(window,"load",b):b()}p("rdl",Ca);var Da={D:1,H:2,da:3,p:4,W:5,M:6,F:7,g:8,ha:9,U:10,L:11,T:12,S:13,N:14,Q:15,P:16,fa:17,w:18,O:19,ga:20,ea:21,u:22,G:23,ja:24,ka:25,ia:26,A:27,j:28,o:29,k:30,ca:31,Z:32,$:33,J:34,K:35,ba:36,aa:37,Y:38,B:39,R:40,v:41,X:42,V:43,h:48,C:49,I:500},Y=[1,2,3,4,5,6,9,10,11,13,14,28,29,30,34,35,37,38,39,40,41,42,43,48,49,500];var Z=l.b("0.001",1E-4),Ea=l.b("1",1),Fa=!1,Ga=!1;if(l.a("1")){var Ha=Math.random();Ha<=Z&&(Fa=!0);Ha<=Ea&&(Ga=!0)}var Ia=Da,$=null;function Ja(){var a=0,b=function(b,d){l.a(d)&&(a|=b)};b(1,"");b(2,"");b(4,"");b(8,"");return a}
function Ka(a,b){var c=Z,d=Fa,f;f=a;if(!$){$={};for(var e=0;e<Y.length;e++){var h=Y[e];$[h]=!0}}if(f=!!$[f])c=Ea,d=Ga;if(d){d=encodeURIComponent;g.rp?(f=g.rp(),f="-1"!=f?f:"1"):f="1";c=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&oge=",a,"&ogex=",d("17483"),"&ogev=",d("HaLtU7LQOtSysQTD_4CQAg"),"&ogf=",g.bv.f,"&ogp=",d("22"),"&ogrp=",d(f),"&ogsr=",Math.round(1/c),"&ogv=",d("1407723702.0"),"&oggv="+
d("es_plusone_gc_20140723.0_p0"),"&ogd=",d("com"),"&ogl=",d("en"),"&ogc=",d("USA"),"&ogus=",Ja()];if(b){"ogw"in b&&(c.push("&ogw="+b.ogw),delete b.ogw);var k;f=b;e=[];for(k in f)0!=e.length&&e.push(","),e.push(La(k)),e.push("."),e.push(La(f[k]));k=e.join("");""!=k&&(c.push("&ogad="),c.push(d(k)))}F(c.join(""))}}function La(a){"number"==typeof a&&(a+="");return"string"==typeof a?a.replace(".","%2E").replace(",","%2C"):a}C=Ka;p("il",C,G);var Ma={};H.il=Ma;setTimeout(function(){C(Ia.g)},0);}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var b=window.gbar.i.i;var c=window.gbar;var f=function(d){try{var a=document.getElementById("gbom");a&&d.appendChild(a.cloneNode(!0))}catch(e){b(e,"omas","aomc")}};c.aomc=f;}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var a=window.gbar;a.mcf("pm",{p:""});}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{window.gbar.rdl();}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
if (window['_OC_timingAction']) {window['_OC_timingAction']('patents_refpage');}</script><style>#gbar,#guser{font-size:13px;padding-top:1px !important;}#gbar{float:left;height:22px}#guser{padding-bottom:7px !important;text-align:right}.gbh,.gbd{border-top:1px solid #c9d7f1;font-size:1px}.gbh{height:0;position:absolute;top:24px;width:100%}#gbs,.gbm{background:#fff;left:0;position:absolute;text-align:left;visibility:hidden;z-index:1000}.gbm{border:1px solid;border-color:#c9d7f1 #36c #36c #a2bae7;z-index:1001}.gb1{margin-right:.5em}.gb1,.gb3{zoom:1}.gb2{display:block;padding:.2em .5em}.gb2,.gb3{text-decoration:none !important;border-bottom:none}a.gb1,a.gb4{text-decoration:underline !important}a.gb1,a.gb2,a.gb3,a.gb4{color:#00c !important}.gbi .gb3,.gbi .gb2,.gbi .gb4{color:#dd8e27 !important}.gbf .gb3,.gbf .gb2,.gbf .gb4{color:#900 !important}a.gb2:hover{background:#36c;color:#fff !important}#gbar .gbz0l{color:#000 !important;cursor:default;font-weight:bold;text-decoration:none !important}
#gbar { padding:.3em .6em !important;}</style></head><body  topmargin="3" marginheight="3"><div id=gbar><nobr><a onclick=gbar.qs(this);gbar.logger.il(1,{t:1}); class=gb1 id=gb_1 href="https://www.google.com/search?sa=N&tab=tw">Search</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:2}); class=gb1 id=gb_2 href="http://www.google.com/search?hl=en&tbm=isch&source=og&sa=N&tab=ti">Images</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:8}); class=gb1 id=gb_8 href="http://maps.google.com/maps?hl=en&sa=N&tab=tl">Maps</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:78}); class=gb1 id=gb_78 href="https://play.google.com/?hl=en&sa=N&tab=t8">Play</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:36}); class=gb1 id=gb_36 href="http://www.youtube.com/results?sa=N&tab=t1">YouTube</a> <a onclick=gbar.logger.il(1,{t:5}); class=gb1 id=gb_5 href="http://news.google.com/nwshp?hl=en&tab=tn">News</a> <a onclick=gbar.logger.il(1,{t:23}); class=gb1 id=gb_23 href="https://mail.google.com/mail/?tab=tm">Gmail</a> <a onclick=gbar.logger.il(1,{t:25}); class=gb1 id=gb_25 href="https://drive.google.com/?tab=to">Drive</a> <a class=gb3 href="http://www.google.com/intl/en/options/" onclick="this.blur();gbar.tg(event);return !1" aria-haspopup=true><u>More</u> <small>&#9660;</small></a><div class=gbm id=gbi><a onclick=gbar.logger.il(1,{t:24}); class=gb2 id=gb_24 href="https://www.google.com/calendar?tab=tc">Calendar</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:51}); class=gb2 id=gb_51 href="http://translate.google.com/?hl=en&sa=N&tab=tT">Translate</a><a onclick=gbar.logger.il(1,{t:17}); class=gb2 id=gb_17 href="http://www.google.com/mobile/?hl=en&tab=tD">Mobile</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:10}); class=gb2 id=gb_10 href="http://www.google.com/search?hl=en&tbo=u&tbm=bks&source=og&sa=N&tab=tp">Books</a><a onclick=gbar.logger.il(1,{t:212}); class=gb2 id=gb_212 href="https://wallet.google.com/manage/?tab=ta">Wallet</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:6}); class=gb2 id=gb_6 href="http://www.google.com/search?hl=en&tbo=u&tbm=shop&source=og&sa=N&tab=tf">Shopping</a><a onclick=gbar.logger.il(1,{t:30}); class=gb2 id=gb_30 href="http://www.blogger.com/?tab=tj">Blogger</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:27}); class=gb2 id=gb_27 href="http://www.google.com/finance?sa=N&tab=te">Finance</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:31}); class=gb2 id=gb_31 href="https://plus.google.com/photos?sa=N&tab=tq">Photos</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:12}); class=gb2 id=gb_12 href="http://www.google.com/search?hl=en&tbo=u&tbm=vid&source=og&sa=N&tab=tv">Videos</a><div class=gb2><div class=gbd></div></div><a onclick=gbar.logger.il(1,{t:66}); href="http://www.google.com/intl/en/options/" class=gb2>Even more &raquo;</a></div></nobr></div><div id=guser width=100%><nobr><span id=gbn class=gbi></span><span id=gbf class=gbf></span><span id=gbe></span><a target=_top id=gb_70 href="https://www.google.com/accounts/Login?service=&continue=http://www.google.com/patents%3Fhl%3Den&hl=en" class=gb4>Sign in</a><div style="display: none"><div class=gbm id=gbd5 aria-owner=gbg5><div class=gbmc><ol id=gbom class=gbmcc></ol></div></div></div></nobr></div><div class=gbh style=left:0></div><div class=gbh style=right:0></div><div role="alert" style="position: absolute; left: 0; right: 0;"><a href="http://www.google.com/patents/us6451534?hl=en&amp;output=html_text" title="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."><img border="0" src="http://www.google.com/images/cleardot.gif"alt="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."></a></div><div id="guser"><nobr></nobr></div><div style="clear:both;"></div><div id="gb-top-search-box" class="gb-top-search-box-small gb-reset"><table><tr><td class="logo"><a href="http://www.google.com/patents" class="logo-link"><img class="logo-img" src="/intl/en/images/logos/google_logo_41.png" alt="Go to Google Books Home" height="41"/></a></td><td><form action="http://www.google.com/search" name="f" id="vheadf" method="get"><span id="hf"></span><input type="hidden" name="tbm" value="pts"/><input type="hidden" name="tbo" value="1"/><input type="hidden" name="hl" value="en"/><table><tr><td><div class="inputs"><table><tr><td><div class="text-input"><input type="text" name="q" id="vheadq" class="text" maxlength="2048" size="31" value="" title="Search Patents" accesskey="s" autocomplete="off"/><script>window._OC_autoDir &&window._OC_autoDir('vheadq', 'tia-vheadq');</script></div></td><td><div class="submit-input"><input name="btnG" class="submit" type="submit" value=""/></div></td></tr></table></div></td><td class="col-ext-links"><div class="ext-links"><a href="http://www.google.com/advanced_patent_search">&lt;nobr&gt;Advanced Patent Search&lt;/nobr&gt;</a></div></td></tr></table></form></td></tr></table></div><div class="kd-appbar"><h2 class="kd-appname"><a href="/patents">Patents</a></h2><div class="kd-buttonbar left" id="left-toolbar-buttons"><a id="appbar-write-review-link" href=""></a><a id="appbar-view-print-sample-link" href=""></a><a id="appbar-view-ebook-sample-link" href=""></a><a id="appbar-patents-prior-art-finder-link" href="https://www.google.com/patents/related/US6451534"></a><a id="appbar-patents-discuss-this-link" href="http://www.google.com/url?id=dGBcBAABERAJ&amp;q=http://patents.stackexchange.com/redirect/google-patents%3Fpatent%3DUS6451534&amp;usg=AFQjCNFLJt53QiK5rcKrz3nA8_KoIYNUPQ" data-is-grant="true"></a><a id="appbar-read-patent-link" href="//docs.google.com/viewer?url=patentimages.storage.googleapis.com/pdfs/US6451534.pdf"></a><a id="appbar-download-pdf-link" href="//patentimages.storage.googleapis.com/pdfs/US6451534.pdf"></a></div><div class="kd-buttonbar right" id="right-toolbar-buttons"></div></div><div id="books-microdata" itemscope=""itemtype="http://schema.org/Book"itemid="http://www.google.com/patents/US6451534" style="display:none"><span itemprop="description">The genomic structure including the sequence of the intron/exon junctions is disclosed for KVLQT1 and KCNE1 which are genes associated with long QT syndrome. Additional sequence data for the two genes ARE also disclosed. Also disclosed are newly found mutations in KVLQT1 which result in long QT syndrome....</span><span itemprop="url">http://www.google.com/patents/US6451534?utm_source=gb-gplus-share</span><span class="main-title" itemprop="name">Patent US6451534 - KVLQT1—a long QT syndrome gene</span><img itemprop="image" src="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"alt="Patent US6451534 - KVLQT1—a long QT syndrome gene" title="Patent US6451534 - KVLQT1—a long QT syndrome gene"></div><div style="display: none"><ol id="ofe-gear-menu-contents" class="gbmcc"><li class="gbe gbmtc"><a class="gbmt goog-menuitem-content" id="" href="http://www.google.com/advanced_patent_search">Advanced Patent Search</a></li></ol></div><table id="viewport_table" cellpadding="0" style="clear:both" cellspacing="0"><tr><td id="viewport_td"><div class=vertical_module_list_row><div id=intl_patents class=about_content><div id=intl_patents_v><table class="patent-bibdata"><tr><td class="patent-bibdata-heading">Publication number</td><td class="single-patent-bibdata">US6451534 B1</td></tr><tr><td class="patent-bibdata-heading">Publication type</td><td class="single-patent-bibdata">Grant</td></tr><tr><td class="patent-bibdata-heading">Application number</td><td class="single-patent-bibdata">US 09/597,732</td></tr><tr><td class="patent-bibdata-heading">Publication date</td><td class="single-patent-bibdata">Sep 17, 2002</td></tr><tr><td class="patent-bibdata-heading">Filing date</td><td class="single-patent-bibdata">Jun 19, 2000</td></tr><tr><td class="patent-bibdata-heading">Priority date<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The priority date is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the date listed."></span></td><td class="single-patent-bibdata">Dec 22, 1995</td></tr><tr><td class="patent-bibdata-heading">Fee status<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The fee status is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the status or dates listed."></span></td><td class="single-patent-bibdata">Paid</td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Also published as</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/CA2336263A1">CA2336263A1</a>, </span><span class="patent-bibdata-value"><a href="/patents/DE69936781D1">DE69936781D1</a>, </span><span class="patent-bibdata-value"><a href="/patents/DE69936781T2">DE69936781T2</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP1100538A2">EP1100538A2</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP1100538A4">EP1100538A4</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP1100538B1">EP1100538B1</a>, </span><span class="patent-bibdata-value"><a href="/patents/US6277978">US6277978</a>, </span><span class="patent-bibdata-value"><a href="/patents/US6420124">US6420124</a>, </span><span class="patent-bibdata-value"><a href="/patents/US6582913">US6582913</a>, </span><span class="patent-bibdata-value"><a href="/patents/US6972176">US6972176</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20030170708">US20030170708</a>, </span><span class="patent-bibdata-value"><a href="/patents/WO2000006199A1">WO2000006199A1</a>, </span><span class="patent-bibdata-value"><a href="/patents/WO2000006199A8">WO2000006199A8</a></span></span></td></tr><tr class="patent-bibdata-list-row alternate-patent-number"><td class="patent-bibdata-heading">Publication number</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value">09597732, </span><span class="patent-bibdata-value">597732, </span><span class="patent-bibdata-value">US 6451534 B1, </span><span class="patent-bibdata-value">US 6451534B1, </span><span class="patent-bibdata-value">US-B1-6451534, </span><span class="patent-bibdata-value">US6451534 B1, </span><span class="patent-bibdata-value">US6451534B1</span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Inventors</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Mark+T.+Keating%22">Mark T. Keating</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Michael+C.+Sanguinetti%22">Michael C. Sanguinetti</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Mark+E.+Curran%22">Mark E. Curran</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Gregory+M.+Landes%22">Gregory M. Landes</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Timothy+D.+Connors%22">Timothy D. Connors</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Timothy+C.+Burn%22">Timothy C. Burn</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Igor+Splawski%22">Igor Splawski</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Original Assignee</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=inassignee:%22University+Of+Utah+Research+Foundation%22">University Of Utah Research Foundation</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=inassignee:%22Genzyme+Corporation%22">Genzyme Corporation</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Export Citation</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/US6451534.bibtex">BiBTeX</a>, </span><span class="patent-bibdata-value"><a href="/patents/US6451534.enw">EndNote</a>, </span><span class="patent-bibdata-value"><a href="/patents/US6451534.ris">RefMan</a></span></span></td></tr><tr class="patent-internal-links"><td colspan=2><span class="patent-bibdata-value"><a href="#backward-citations">Patent Citations</a> (1),</span> <span class="patent-bibdata-value"><a href="#npl-citations">Non-Patent Citations</a> (24),</span> <span class="patent-bibdata-value"><a href="#forward-citations">Referenced by</a> (4),</span> <span class="patent-bibdata-value"><a href="#classifications">Classifications</a> (34),</span> <span class="patent-bibdata-value"><a href="#legal-events">Legal Events</a> (7)</span> </td></tr><tr><td colspan=2 class="patent-bibdata-external-link-spacer-top"></td></tr><tr class="patent-bibdata-external-link-spacer-bottom"></tr><tr><td colspan=2><span class="patent-bibdata-heading">External Links:&nbsp;</span><span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=dGBcBAABERAJ&q=http://patft.uspto.gov/netacgi/nph-Parser%3FSect2%3DPTO1%26Sect2%3DHITOFF%26p%3D1%26u%3D/netahtml/PTO/search-bool.html%26r%3D1%26f%3DG%26l%3D50%26d%3DPALL%26RefSrch%3Dyes%26Query%3DPN/6451534&usg=AFQjCNET1BGCEkfYGhlIG6uNMXaQ6iCpgw">USPTO</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=dGBcBAABERAJ&q=http://assignments.uspto.gov/assignments/q%3Fdb%3Dpat%26pat%3D6451534&usg=AFQjCNFINd4nMNHxgQ6qs9zDa98PehAoFw">USPTO Assignment</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=dGBcBAABERAJ&q=http://worldwide.espacenet.com/publicationDetails/biblio%3FCC%3DUS%26NR%3D6451534B1%26KC%3DB1%26FT%3DD&usg=AFQjCNGS33tY0sFSuFIAI_MR4drSxAaczQ">Espacenet</a></span></span></td></tr><tr class="patent-bibdata-group-spacer"></tr></table><div class="number-and-title"><span class="patent-title"><invention-title mxw-id="PT72387473" lang="EN" load-source="patent-office">KVLQT1—a long QT syndrome gene</invention-title></span><br><span class="patent-number">US 6451534 B1</span></div><div class="patent-section patent-abstract-section"><div class="patent-section-header"><span class="patent-section-title">Abstract</span></div><div class="patent-text"><abstract mxw-id="PA50382888" lang="EN" load-source="patent-office"> <div class="abstract">The genomic structure including the sequence of the intron/exon junctions is disclosed for KVLQT1 and KCNE1 which are genes associated with long QT syndrome. Additional sequence data for the two genes ARE also disclosed. Also disclosed are newly found mutations in KVLQT1 which result in long QT syndrome. The intron/exon junction sequence data allow for the design of primer pairs to amplify and sequence across all of the exons of the two genes. This can be used to screen persons for the presence of mutations which cause long QT syndrome. Assays can be performed to screen persons for the presence of mutations in either the DNA or proteins. The DNA and proteins may also be used in assays to screen for drugs which will be useful in treating or preventing the occurrence of long QT syndrome.</div>
  </abstract></div></div><div class="patent-section patent-drawings-section"><div class="patent-section-header"><span class="patent-section-title">Images<span class="patent-section-count">(26)</span></span></div><div class="patent-drawings-body"><div class="patent-drawings-carousel"><div class="drawings"><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6451534B1/US06451534-20020917-D00001.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6451534B1/US06451534-20020917-D00001.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6451534B1/US06451534-20020917-D00002.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6451534B1/US06451534-20020917-D00002.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6451534B1/US06451534-20020917-D00003.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6451534B1/US06451534-20020917-D00003.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6451534B1/US06451534-20020917-D00004.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6451534B1/US06451534-20020917-D00004.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6451534B1/US06451534-20020917-D00005.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6451534B1/US06451534-20020917-D00005.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6451534B1/US06451534-20020917-D00006.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6451534B1/US06451534-20020917-D00006.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6451534B1/US06451534-20020917-D00007.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6451534B1/US06451534-20020917-D00007.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6451534B1/US06451534-20020917-D00008.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6451534B1/US06451534-20020917-D00008.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6451534B1/US06451534-20020917-D00009.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6451534B1/US06451534-20020917-D00009.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6451534B1/US06451534-20020917-D00010.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6451534B1/US06451534-20020917-D00010.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6451534B1/US06451534-20020917-D00011.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6451534B1/US06451534-20020917-D00011.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6451534B1/US06451534-20020917-D00012.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6451534B1/US06451534-20020917-D00012.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6451534B1/US06451534-20020917-D00013.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6451534B1/US06451534-20020917-D00013.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6451534B1/US06451534-20020917-D00014.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6451534B1/US06451534-20020917-D00014.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6451534B1/US06451534-20020917-D00015.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6451534B1/US06451534-20020917-D00015.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6451534B1/US06451534-20020917-D00016.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6451534B1/US06451534-20020917-D00016.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6451534B1/US06451534-20020917-D00017.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6451534B1/US06451534-20020917-D00017.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6451534B1/US06451534-20020917-D00018.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6451534B1/US06451534-20020917-D00018.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6451534B1/US06451534-20020917-D00019.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6451534B1/US06451534-20020917-D00019.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6451534B1/US06451534-20020917-D00020.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6451534B1/US06451534-20020917-D00020.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6451534B1/US06451534-20020917-D00021.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6451534B1/US06451534-20020917-D00021.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6451534B1/US06451534-20020917-D00022.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6451534B1/US06451534-20020917-D00022.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6451534B1/US06451534-20020917-D00023.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6451534B1/US06451534-20020917-D00023.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6451534B1/US06451534-20020917-D00024.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6451534B1/US06451534-20020917-D00024.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6451534B1/US06451534-20020917-D00025.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6451534B1/US06451534-20020917-D00025.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6451534B1/US06451534-20020917-D00026.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6451534B1/US06451534-20020917-D00026.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div></div></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img" alt="Previous page"src="/googlebooks/images/kennedy/page_left.png"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img" alt="Next page"src="/googlebooks/images/kennedy/page_right.png"width="21" height="21" /></div></div></div><div class="patent-post-drawings"></div><div class="patent-section patent-claims-section"><div class="patent-section-header"><span class="patent-section-title">Claims<span class="patent-section-count">(12)</span></span></div><div class="patent-text"><div mxw-id="PCLM8361730" lang="EN" load-source="patent-office" class="claims">
    <claim-statement>What is claimed is: </claim-statement> <div class="claim"> <div num="1" id="US-6451534-B1-CLM-00001" class="claim">
      <div class="claim-text">1. An isolated nucleic acid selected from the group consisting of SEQ ID NO:41, SEQ ID NO:42, SEQ ID NO:43, SEQ ID NO:44, SEQ ID NO:45, SEQ ID NO:46, SEQ ID NO:47, SEQ ID NO:48, SEQ D NO:49, SEQ ID NO:50, SEQ ID NO:51, SEQ ID NO:52, SEQ ID NO:53, SEQ ID NO:54, SEQ ID NO:55, SEQ ID NO:56, SEQ ID NO:57, SEQ ID NO:58, SEQ ID NO:59, SEQ ID NO:60, SEQ ID NO:61, SEQ ID NO:62, SEQ ID NO:63, SEQ ID NO:64, SEQ ID NO:65, SEQ ID NO:66, SEQ ID NO:67, SEQ ID NO:68, SEQ ID NO:69, SEQ ID NO:70, SEQ ID NO:71, SEQ ID NO:72, SEQ ID NO:73 and SEQ ID NO:74.</div>
    </div>
    </div> <div class="claim"> <div num="2" id="US-6451534-B1-CLM-00002" class="claim">
      <div class="claim-text">2. A pair of nucleic acid primers wherein said primers are:</div>
      <div class="claim-text">a) SEQ ID NOs:41 and 42; </div>
      <div class="claim-text">b) SEQ ID NOs:43 and 44; </div>
      <div class="claim-text">c) SEQ ID NOs:45 and 46; </div>
      <div class="claim-text">d) SEQ ID NOs:47 and 48; </div>
      <div class="claim-text">e) SEQ ID NOs:49 and 50; </div>
      <div class="claim-text">f) SEQ ID NOs:51 and 52; </div>
      <div class="claim-text">g) SEQ ID NOs:53 and 54; </div>
      <div class="claim-text">h) SEQ ID NOs:55 and 56; </div>
      <div class="claim-text">i) SEQ ID NOs:57 and 58; </div>
      <div class="claim-text">j) SEQ ID NOs:59 and 60; </div>
      <div class="claim-text">k) SEQ ID NOs:61 and 62; </div>
      <div class="claim-text">l) SEQ ID NOs:63 and 64; </div>
      <div class="claim-text">m) SEQ ID NOs:65 and 66; </div>
      <div class="claim-text">n) SEQ ID NOs:67 and 68; </div>
      <div class="claim-text">o) SEQ ID NOs:69 and 70; </div>
      <div class="claim-text">p) SEQ ID NOs:71 and 72; or </div>
      <div class="claim-text">q) SEQ ID NOs:73 and 74. </div>
    </div>
    </div> <div class="claim"> <div num="3" id="US-6451534-B1-CLM-00003" class="claim">
      <div class="claim-text">3. An isolated nucleic acid comprising any 15 consecutive nucleotides of SEQ ID NO:1 or its complement wherein SEQ ID NO:1 comprises one or more mutations selected from the group consisting of: an A at base 664, an A at base 1102, a G at base 1106, a C at base 1116, a C at base 1220, a T at base 1258, a deletion of bases 662-664, a C at base 694, an A at base 727, an A at base 731, an A at base 922, a T at base 979, an A at base 1078, a T at base 1097, an A at base 1184, a T at base 1184 and an A at base 1196, and wherein said isolated nucleic acid comprises one or more of said mutations.</div>
    </div>
    </div> <div class="claim"> <div num="4" id="US-6451534-B1-CLM-00004" class="claim">
      <div class="claim-text">4. An isolated nucleic acid comprising any 12 consecutive nucleotides of SEQ ID NO:1 or its complement wherein SEQ ID NO:1 comprises one or more mutations selected from the group consisting of: an A at base 664, an A at base 1102, a G at base 1106, a C at base 1116, a C at base 1220, a T at base 1258, a deletion of bases 662-664, a C at base 694, an A at base 727, an A at base 731, an A at base 922, a T at base 979, an A at base 1078, a T at base 1097, an A at base 1184, a T at base 1184 and an A at base 1196, and wherein said isolated nucleic acid comprises one or more of said mutations.</div>
    </div>
    </div> <div class="claim"> <div num="5" id="US-6451534-B1-CLM-00005" class="claim">
      <div class="claim-text">5. An isolated nucleic acid comprising any 23 consecutive nucleotides of a nucleic acid encoding a Xenopus KVLQT1 polypeptide having the amino acid sequence set forth int SEQ ID NO:113 or its complement.</div>
    </div>
    </div> <div class="claim"> <div num="6" id="US-6451534-B1-CLM-00006" class="claim">
      <div class="claim-text">6. An isolated nucleic acid comprising any 25 consecutive nucleotides of a nucleic acid encoding a Xenopus KVLQT1 polypeptide having the amino acid sequence set forth int SEQ ID NO:113 or its complement.</div>
    </div>
    </div> <div class="claim"> <div num="7" id="US-6451534-B1-CLM-00007" class="claim">
      <div class="claim-text">7. An isolated nucleic acid comprising any 15 consecutive nucleotides of SEQ ID NO:115 or its complement.</div>
    </div>
    </div> <div class="claim"> <div num="8" id="US-6451534-B1-CLM-00008" class="claim">
      <div class="claim-text">8. An isolated nucleic acid comprising any 12 consecutive nucleotides of SEQ ID NO:115 or its complement.</div>
    </div>
    </div> <div class="claim"> <div num="9" id="US-6451534-B1-CLM-00009" class="claim">
      <div class="claim-text">9. An isolated nucleic acid comprising any 15 consecutive nucleotides of SEQ ID NO:115 or its complement wherein SEQ ID NO:115 comprises one or more mutations selected from the group consisting of: an A at base 304, an A at base 742, a G at base 746, a C at base 756, a C at base 860, a T at base 898, a deletion of bases 302-304, a C at base 334, an A at base 367, an A at base 371, an A at base 562, a T at base 619, an A at base 718, a T at base 737, an A at base 824, a T at base 824 and an A at base 836, and wherein said isolated nucleic acid comprises one or more of said mutations.</div>
    </div>
    </div> <div class="claim"> <div num="10" id="US-6451534-B1-CLM-00010" class="claim">
      <div class="claim-text">10. An isolated nucleic acid comprising any 12 consecutive nucleotides of SEQ ID NO:115 or its complement wherein SEQ ID NO:115 comprises one or more mutations selected from the group consisting of: an A at base 304, an A at base 742, a G at base 746, a C at base 756, a C at base 860, a T at base 898, a deletion of bases 302-304, a C at base 334, an A at base 367, an A at base 371, an A at base 562, a T at base 619, an A at base 718, a T at base 737, an A at base 824, a T at base 824 and an A at base 836, and wherein said isolated nucleic acid comprises one or more of said mutations.</div>
    </div>
    </div> <div class="claim"> <div num="11" id="US-6451534-B1-CLM-00011" class="claim">
      <div class="claim-text">11. A method of amplifying an exon of KVLQT1 wherein said method comprises:</div>
      <div class="claim-text">a) contacting a test sample with a pair of primers, wherein said pair of primers is useful for amplifying an exon of KVLQT1 and each of said pair of primers comprises a nucleotide sequence at an intron/exon boundary; </div>
      <div class="claim-text">b) conducting an amplification reaction to amplify an exon of KVLQT1. </div>
    </div>
    </div> <div class="claim-dependent"> <div num="12" id="US-6451534-B1-CLM-00012" class="claim">
      <div class="claim-text">12. The method of <claim-ref idref="US-6451534-B1-CLM-00011">claim 11</claim-ref>, wherein said pair of primers are selected from the group consisting of the primer pairs</div>
      <div class="claim-text">a) SEQ ID NOs:41 and 42; </div>
      <div class="claim-text">b) SEQ ID NOs:43 and 44; </div>
      <div class="claim-text">c) SEQ ID NOs:45 and 46; </div>
      <div class="claim-text">d) SEQ ID NOs:47 and 48; </div>
      <div class="claim-text">e) SEQ ID NOs:49 and 50; </div>
      <div class="claim-text">f) SEQ ID NOs:51 and 52; </div>
      <div class="claim-text">g) SEQ ID NOs:53 and 54; </div>
      <div class="claim-text">h) SEQ ID NOs:55 and 56; </div>
      <div class="claim-text">i) SEQ ID NOs:57 and 58; </div>
      <div class="claim-text">j) SEQ ID NOs:59 and 60; </div>
      <div class="claim-text">k) SEQ ID NOs:61 and 62; </div>
      <div class="claim-text">l) SEQ ID NOs:63 and 64; </div>
      <div class="claim-text">m) SEQ ID NOs:65 and 66; </div>
      <div class="claim-text">n) SEQ ID NOs:67 and 68; </div>
      <div class="claim-text">o) SEQ ID NOs:69 and 70; </div>
      <div class="claim-text">p) SEQ ID NOs:71 and 72; and </div>
      <div class="claim-text">q) SEQ ID NOs:73 and 74.</div>
    </div>
  </div> </div></div></div><div class="patent-section patent-description-section"><div class="patent-section-header"><span class="patent-section-title">Description</span></div><div class="patent-text"><div mxw-id="PDES53638105" lang="EN" load-source="patent-office" class="description">
    <heading>CROSS REFERENCE TO RELATED APPLICATIONS</heading> <p>The present invention is a divisional of application Ser. No. 09/135,010, filed Aug. 17, 1998, now U.S. Pat. No. 6,277,978, which is a continuation-in-part of application Ser. No. 08/921,068 filed Aug. 29, 1997, abandoned which is a continuation-in-part of application Ser. No. 08/739,383 filed Oct. 29, 1996, abandoned, which is related to provisional patent application Serial No. 60/019,014 filed Dec. 22, 1995, and the present invention is related to provisional patent application Ser. No. 60/094,477 filed Jul. 29, 1998, all of which are incorporated herein by reference.</p>
    <p>This application was made with Government support under Grant No., and P50-HL52338-02 (SCOR), funded by the National Institutes of Health, Bethesda. The federal government may have certain rights in this invention.</p>
    <heading>BACKGROUND OF THE INVENTION</heading> <p>The present invention is directed to genes and gene products associated with long QT syndrome (LQT) and to a process for the diagnosis of LQT. LQT is diagnosed in accordance with the present invention by analyzing the DNA sequence of the KVLQT1 or KCNE1 gene of an individual to be tested and comparing the respective DNA sequence to the known DNA sequence of a normal KVLQT1 or KCNE1 gene. Alternatively, the, KVLQT1 or KCNE1 gene of an individual to be tested can be screened for mutations which cause LQT. Prediction of LQT will enable practitioners to prevent this disorder using existing medical therapy. This invention is further directed to the discovery that the KVLQT1 and KCNE1 (also known as minK) proteins coassemble to form a cardiac I<sub>Ks </sub>potassium channel. This knowledge can be used to coexpress these two proteins in a cell and such a transformed cell can be used for screening for drugs which will be useful in treating or preventing LQT. The invention is further directed to mutations in the human KCNE1 gene (which gene encodes human minK protein) which have been discovered in families with LQT.</p>
    <p>The publications and other materials used herein to illuminate the background of the invention or provide additional details respecting the practice, are incorporated by reference, and for convenience are respectively grouped in the appended List of References.</p>
    <p>Cardiac arrhythmias are a common cause of morbidity and mortality, accounting for approximately 11% of all natural deaths (Kannel, 1987; Willich et al., 1987). In general, presymptomatic diagnosis and treatment of individuals with life-threatening ventricular tachyarrhythmias is poor, and in some cases medical management actually increases the risk of arrhythmia and death (Cardiac Arrhythmia Suppression Trial II Investigators, 1992). These factors make early detection of individuals at risk for cardiac arrhythmias and arrhythmia prevention high priorities.</p>
    <p>Both genetic and acquired factors contribute to the risk of developing cardiac arrhythmias. Long QT syndrome (LQT) is an inherited cardiac arrhythmia that causes abrupt loss of consciousness, syncope, seizures and sudden death from ventricular tachyarrhythmias, specifically torsade de pointes and ventricular fibrillation (Ward, 1964; Romano, 1965; Schwartz et al., 1975; Moss et al., 1991). This disorder usually occurs in young, otherwise healthy individuals (Ward, 1964; Romano, 1965; Schwartz et al., 1975). Most LQT gene carriers manifest prolongation of the QT interval on electrocardiograms, a sign of abnormal cardiac repolarization (Vincent et al., 1992). The clinical features of LQT result from episodic cardiac arrhythmias, specifically repolarization-related ventricular tachyarrhythmias like torsade de pointes, named for the characteristic undulating nature of the electrocardiogram in this arrhythmia and ventricular fibrillation (Schwartz et al., 1975; Moss and McDonald, 1971). Torsade de pointes may degenerate into ventricular fibrillation, a particularly lethal arrhythmia. Although LQT is not a common diagnosis, ventricular arrhythmias are very common; more than 300,000 United States citizens die suddenly every year (Kannel, et al., 1987; Willich et al., 1987) and, in many cases, the underlying mechanism may be aberrant cardiac repolarization. LQT, therefore, provides a unique opportunity to study life-threatening cardiac arrhythmias at the molecular level.</p>
    <p>Both inherited and acquired forms of LQT have been defined. Acquired LQT and secondary arrhythmias can result from cardiac ischemia, bradycardia and metabolic abnormalities such as low serum potassium or calcium concentration (Zipes, 1987). LQT can also result from treatment with certain medications, including antibiotics, antihistamines, general anesthetics, and, most commonly, antiarrhythmic medications (Zipes, 1987). Inherited forms of LQT can result from mutations in at least five different genes. In previous studies, LQT loci were mapped to chromosome 11 p15.5 (KVLQT1 or LQT1) (Keating et al., 1991a; Keating et al., 1991b), 7q35-36 (HERG or LQT2), 3p21-24 (SCN5A or LQT3) (Jiang et al., 1994). Of these, the most common cause of inherited LQT is KVLQT1. Our data indicate that mutations in this gene are responsible for more than 50% of inherited LQT. Recently, a fourth LQT locus (LQT4) was mapped to 4q25-27 (Schott et al., 1995). Also, KCNE1 (LQT5) has been associated with long QT syndrome (Splawski et al., 1997b; Duggal et al., 1998). These genes encode ion channels involved in generation of the cardiac action potential. Mutations can lead to channel dysfunction and delayed myocellular repolarization. Because of regional heterogeneity of channel expression with the myocardium, the aberrant cardiac repolarization creates a substrate for arrhythmia. KVLQT1 and KCNE1 are also expressed in the inner ear (Neyroud et al., 1997; Vetter et al., 1996). We and others demonstrated that homozygous or compound heterozygous mutations in each of these genes can cause deafness and the severe cardiac phenotype of the Jervell and Lange-Nielsen syndrome (Neyroud et al., 1997; Splawski et al., 1997a; Schultze-Bahr et al., 1997; Tyson et al., 1997). Loss of functional channels in the ear apparently disrupts the production of endolymph, leading to deafness.</p>
    <p>Presymptomatic diagnosis of LQT is currently based on prolongation of the QT interval on electrocardiograms. QTc (QT interval corrected for heart rate; Bazzett, 1920) greater than 0.44 second has traditionally classified an individual as affected. Most LQT patients, however, are young, otherwise healthy individuals, who do not have electrocardiograms. Moreover, genetic studies have shown that QTc is neither sensitive nor specific (Vincent et al., 1992). The spectrum of QTc intervals for gene carriers and non-carriers overlaps, leading to misclassifications. Non-carriers can have prolonged QTc intervals and be diagnosed as affected. Conversely, some LQT gene carriers have QTc intervals of ≦0.44 second but are still at increased risk for arrhythmia. Correct presymptomatic diagnosis is important for effective, gene-specific treatment of LQT.</p>
    <p>Autosomal dominant and autosomal recessive forms of this disorder have been reported. Autosomal recessive LQT (also known as Jervell and Lange-Nielsen syndrome) has been associated with congenital neural deafnless; this form of LQT is rare (Jervell and Lange-Nielsen, 1957). Autosomal dominant LQT (Romano-Ward syndrome) is more common, and is not associated with other phenotypic abnormalities (Romano et al., 1963; Ward, 1964). A disorder very similar to inherited LQT can also be acquired, usually as a result of pharmacologic therapy (Schwartz et al., 1975; Zipes, 1987).</p>
    <p>The data have implications for the mechanism of arrhythmias in LQT. Two hypotheses for LQT have previously been proposed (Schwartz et al., 1994). One suggests that a predominance of left autonomic innervation causes abnormal cardiac repolarization and arrhythmias. This hypothesis is supported by the finding that arrhythmias can be induced in dogs by removal of the right stellate ganglion. In addition, anecdotal evidence suggests that some LQT patients are effectively treated by β-adrenergic blocking agents and by left stellate ganglionectomy (Schwartz et al., 1994). The second hypothesis for LQT-related arrhythmias suggests that mutations in cardiac-specific ion channel genes, or genes that modulate cardiac ion channels, cause delayed myocellular repolarization. Delayed myocellular repolarization could promote reactivation of L-type calcium channels, resulting in secondary depolarizations (January and Riddle, 1989). These secondary depolarizations are the likely cellular mechanism of torsade de pointes arrhythmias (Surawicz, 1989). This hypothesis is supported by the observation that pharmacologic block of potassium channels can induce QT prolongation and repolarization-related arrhythmias in humans and animal models (Antzelevitch and Sicouri, 1994). The discovery that one form of LQT results from mutations in a cardiac potassium channel gene supports the myocellular hypothesis.</p>
    <p>In theory, mutations in a cardiac sodium channel gene could cause LQT. Voltage-gated sodium channels mediate rapid depolarization in ventricular myocytes, and also conduct a small current during the plateau phase of the action potential (Attwell et al., 1979). Subtle abnormalities of sodium channel finction (e.g., delayed sodium channel inactivation or altered voltage-dependence of channel inactivation) could delay cardiac repolarization, leading to QT prolongation and arrhythmias. In 1992, Gellens and colleagues cloned and characterized a cardiac sodium channel gene, SCN5A (Gellens et al., 1992). The structure of this gene was similar to other, previously characterized sodium channels, encoding a large protein of 2016 amino acids. These channel proteins contain four homologous domains (DI-DIV), each of which contains six putative membrane spanning segments (S1-S6). SCN5A was recently mapped to chromosome 3p21, making it an excellent candidate gene for LQT3 (George et al., 1995), and this gene was then proved to be associated with LQT3 (Wang et al., 1995a).</p>
    <p>In 1994, Warmke and Ganetzky identified a novel human cDNA, human ether a-go-go related gene (HERG, Warmke and Ganetzky, 1994). HERG was localized to human chromosome 7 by PCR analysis of a somatic cell hybrid panel (Warnike and Ganetzky, 1994) making it a candidate for LQT2. It has predicted amino acid sequence homology to potassium channels. HERG was isolated from a hippocarnpal cDNA library by homology to the Drosophila ether a-go-go gene (eag), which encodes a calcium-modulated potassium channel (Bruggemann et al., 1993). HERG is not the human homolog of eag, however, sharing only ˜50% amino acid sequence homology. HERG has been shown to be associated with LQT2 (Curran et al., 1995).</p>
    <p>LQT1 was found to be linked with the gene KVLQT1 (Q. Wang et al., 1996). Sixteen families with mutations in KVLQT1 were identified and characterized and it was shown that in all sixteen families there was complete linkage between LQT1 and KVLQT1. KVLQT1 was mapped to chromosome 11p15.5 making it a candidate gene for LQT1. KVLQT1 encodes a protein with structural characteristics of potassium channels, and expression of the gene as measured by Northern blot analysis demonstrated that KVLQT1 is most strongly expressed in the heart. One intragenic deletion and ten different missense mutations which cause LQT were identified in KVLQT1. These data define KVLQT1 as a novel cardiac potassium channel gene and show that mutations in this gene cause susceptibility to ventricular tachyarrhythmias and sudden death.</p>
    <p>It was known that one component of the I<sub>Ks </sub>channel is minK, a 130 amino acid protein with a single putative transmembrane domain (Takumi et al., 1988; Goldstein and Miller, 1991; Hausdorff et al., 1991; Takumi et al., 1991; Busch et al., 1992; Wang and Goldstein, 1995; K W Wang et al., 1996). The size and structure of this protein made it unlikely that minK alone forms functional channels (Attali et al., 1993; Lesage et al., 1993). Evidence is presented that KVLQT1 and minK coassemble to form the cardiac I<sub>Ks </sub>potassium channel. This was published by Sanguinetti et al. (1996b). I<sub>Ks </sub>dysfunction is a cause of cardiac arrhythmia. It was later shown that mutations in KCNE1 (which encodes minK) also can result in LQT (Splawski et al., 1997b).</p>
    <heading>SUMMARY OF THE INVENTION</heading> <p>The present invention teaches the genomic structure of the LQT genes KVLQT1 and KCNE1. This includes a teaching of the intron/exon boundaries. Also disclosed are additional sequence data not previously reported for both genes as well as mutations in KVLQT1 and KCNE1 which are associated with LQT. Analysis of the KVLQT1 or KCNE1 gene will provide an early diagnosis of subjects with LQT. The diagnostic method comprises analyzing the DNA sequence of the KVLQT1 and/or KCNE1 gene of an individual to be tested and comparing it with the DNA sequence of the native, non-variant gene. In a second embodiment, the KVLQT1 or KCNE1 gene of an individual to be tested is screened for mutations which cause LQT. The ability to predict LQT will enable physicians to prevent the disease with medical therapy such as beta blocking agents.</p>
    <p>It is further demonstrated that KVLQT1 and KCNE1 (minK) coassemble to form a cardiac I<sub>Ks </sub>potassium channel. I<sub>Ks </sub>dysfunction is a cause of cardiac arrhythmia. The knowledge that these two proteins coassemble to form the I<sub>Ks </sub>channel is useful for developing an assay to screen for drugs which are useful in treating or preventing LQT1. By coexpressing both genes in a cell such as an oocyte it is possible to screen for drugs which have an effect on the I<sub>Ks </sub>channel, both in its wild-type and in its mutated forms. This knowledge is also useful for the analysis of the KCNE1 gene for an early diagnosis of subjects with LQT. The diagnostic methods are performed as noted above for KVLQT1 and/or KCNE1.</p>
    <heading>BRIEF DESCRIPTION OF THE FIGURES</heading> <p>FIG. <b>1</b>. Pedigree structure for a portion of LQT kindred 1532. Affected individuals are shown as filled circles (females) or squares (males), unaffected individuals as empty symbols and individuals with equivocal phenotypes are stippled. Genotypes for chromosome 11 markers are indicated beneath each symbol and are shown as haplotypes. Marker order (top to bottom) is: Tel-HRAS-D11S922-TH-D11S1318-D11S454-D11S860-D11S12-Cen. The accuracy of haplotypes was ensured using genotypes from additional chromosome 11p15.5 markers. Inferred genotypes are shown in brackets. Disease chromosomes are indicated by boxes and recombination events are indicated with solid horizontal lines. Recombination events affecting disease chromosomes occur in individuals: IV-22, IV-25, V-6, V-17, V-24, V-34, VI-13, VI-14 and VI-16. Recombination events occurring in non-disease chromosomes are not indicated. KVLQT1 is an SSCP conformer within KVLQT1 identified by primers 5 and 6; this conformer was only identified in K1532 and represents a disease-associated mutation (allele 2 is the mutant allele). Haplotype analyses indicate that KVLQT1 is located between flanking markers D11S922 and D11S454.</p>
    <p>FIG. <b>2</b>. Physical map of the LQT1 region. Ideogram of chromosome 11 indicates the approximate location of LQT1 (11p15.5). The location of polymorphic markers and some cosmids are indicated by vertical lines on the map. Refined genetic mapping places LQT1 between TH and D11S454. The distance between TH and D11S454 was estimated by pulsed field gel analyses as &lt;700 kb. A physical map of the minimal set of overlapping YAC and P1 clones is shown. The locations of the KVLQT1 cDNA and trapped exons are indicated. Dashed lines in YACs indicate chimerism.</p>
    <p>FIG. <b>3</b>. Alignment of the S1-S6 region of KVLQT1 with Drosophila Shaker potassium channel, DMSHAKE1 (SHA) (Pongs et al., 1988). Identity (|) and similarity (:) are indicated. The 3 separate fragments of KVLQT1 are in order: SEQ ID NO:107, SEQ ID NO:108 and SEQ ID NO: 109. The 3 separate fragments of DMSHAKE1 are in order: SEQ ID NO: 110, SEQ ID NO:111 and SEQ ID NO:112.</p>
    <p>FIG. <b>4</b>. Northern analysis indicating expression of KVLQT1 in human heart, placenta, lung, kidney and pancreas.</p>
    <p>FIGS. 5A-5B. Genomic organization of KVLQT1 coding and 5′ and 3′ untranslated regions. Positions of the introns are indicated with arrowheads. The six putative transmembrane segments (S6 to S6) and the putative pore region (Pore) are underlined. The stop codon is denoted by an asterisk. The nucleotide sequence of FIGS. 5A-5B is SEQ ID NO: 1. The amino acid sequence of FIGS. 5A-5B is SEQ ID NO:2.</p>
    <p>FIG. <b>6</b>. Physical map and exon organization of KVLQT1. The genomic region of KVLQT1 encompasses approximately 400 kilobases. Physical map of the minimal contig of overlapping P1 clones and the cosmid containing exon 1 is shown. The location of KVLQT1 exons relative to genomic clones is indicated. Sizes of exons and distances are not drawn to scale.</p>
    <p>FIGS. 7A-7E. KVLQT1 and hminK coexpression in CHO cells induces a current nearly identical to cardiac I<sub>Ks</sub>. A) KVLQT1 currents recorded during 1 sec depolarizing pulses to membrane potentials of −50 to +40 mV, applied from a holding potential of −80 mV. Tail currents were measured at −70 mV. B) Normalized isochronal activation curves for cells transfected with KVLQT1 (n=6; 1 sec pulses) or KVLQT1 and KCNE1 (n=7; 7.5 sec pulses). C-E) Currents recorded during 7.5 sec pulses to −40, −20, −10, 0, +20 and +40 mV in cells transfected with KCNE1 (C), KVLQT1 (D) or KVLQT1 and KCNE1 (E). Tail currents were measured at −70 mV in D, and at −50 mV in C and E. The amplitude of steady state KVLQT1 current at +40 mV was 0.37±0.14 nA (n=6). In cells cotransfected with KVLQT1 and KCNE1, time-dependent current during a 7.5-s pulse to +40 mV was 1.62±0.39 nA (n=7).</p>
    <p>FIGS. 8A-8C. Expression of KVLQT1 in Xenopus oocytes. A) Currents recorded in an oocyte injected with 12.5 ng KVLQT1 cRNA. Pulses were applied in 10 mV increments from −70 to +40 mV. B) Isochronal (1s) activation curve for KVLQT1 current. The V<sub>½ </sub>was −14.0±0.2 mV and the slope factor was 11.2±0.2 mV (n=9). C) The relationship of E<sub>rev </sub>versus log[K<sup>+</sup>]<sub>e </sub>was fit with a linear function and had a slope of 49.9±0.4 mV (n=6-7 oocytes per point). Tail currents were measured at several voltages after 1.6 sec prepulses to +10 mV.</p>
    <p>FIGS. 9A-9E. Coexpression of KVLQT1 and hminK suggests the presence of a KVLQT1 homologue in Xenopus oocytes. Currents were recorded at −40, −20, 0, +20 and +40 mV in oocytes injected with either 5.8 ng KVLQT1 (FIG. <b>9</b>A), 1 ng KCNE1 (FIG. <b>9</b>B), or co-injected with both cRNAs (FIG. <b>9</b>C). FIG. 9D shows current-voltage relationships measured using 2 sec pulses for KVLQT1, and 7.5 sec pulses for hminK, or KVLQT1 and hminK (n=20 cells for each condition). For oocytes injected with 60 pg or 1 ng of KCNE1 cRNA, I<sub>sK </sub>at +40 mV was 2.11±0.12 μA and 2.20±0.18 μA. FIG. 9E shows normalized isochronal activation curves for oocytes injected with KCNE1 (V<sub>½</sub>=2.4±0.3 mV; slope=11.4±0.3 mV; n=16) or co-injected with KVLQT1 and KCNE1 cRNA (V<sub>½</sub>=6.2±0.3 mV; slope=12.3±0.2 mV; n=20).</p>
    <p>FIG. <b>10</b>. Comparison of a partial human and a partial Xenopus KVLQT1 amino acid sequence. Vertical lines indicate identical residues. The Xenopus amino acid sequence is SEQ ID NO:113 and the human amino acid sequence is SEQ ID NO:114.</p>
    <p>FIGS. 11A-11D. KVLQT1 missense mutations cosegregate with LQT in kindreds K1532 (FIG. <b>11</b>A), K2605 (FIG. <b>11</b>B), K1723 (FIG. 11C) and K1807 (FIG. <b>11</b>D). The results of SSCP analyses with primer pair 5-6 (K1532), primer pair 9-10 (K1723, K1807), and primer pair 11-12 (K2605) are shown below each pedigree. Aberrant SSCP conformers (indicated by *) cosegregate with LQT in each kindred. For K1532, only eight of the 217 individuals are shown. Because aberrant SSCP conformers cosegregating with LQT in K161 and K162 were identical to the aberrant conformer defined in K1807, results for these kindreds are not shown. Results of DNA sequence analyses of the normal (left) and aberrant (right) conformers are shown below each pedigree.</p>
    <p>FIGS. 12A-12O. KVLQT1 intragenic deletions and missense mutations associated with LQT in kindreds K13216 (FIG. <b>12</b>A), K1777 (FIG. <b>12</b>B), K20925 (FIG. <b>12</b>C), K2557 (FIG. <b>12</b>D), K13119 (FIG. <b>12</b>E), K20926 (FIG. <b>12</b>F), K15019 (FIG. <b>12</b>G), K2625 (FIG. <b>12</b>H), K2673 (FIG. <b>12</b>I), K3698 (FIG. <b>12</b>J), K19187 (FIG. <b>12</b>K), K22709 (FIG. <b>12</b>L), K2762 (FIG. <b>12</b>M), K3401 (FIG. 12N) and K2824 (FIG. <b>12</b>O). Affected individuals are indicated by filled circles (females) and squares (males). Unaffected individuals are indicated with empty symbols and uncertain individuals are either gray or stippled. The results of SSCP analyses with primer pair 1-2 (K13216, K2557, K13119, K15019), primer pair 7-8 (K1777, K20926), and primer pair 9-10 (K20925) are shown below each pedigree in FIGS. 12A-12G (see Table 5 for primer pairs). Because aberrant SSCP conformers cosegregating with LQT in K2050, K163 and K164 were identical to the aberrant conformers defined in K1723 and K1807, results for these kindreds are not shown. For FIGS. 12A-12G, results of DNA sequence analyses of the normal (left) and aberrant (right) conformers are shown below each pedigree and the sequences shown are on the antisense strand. For FIGS. 12H-12O the aberrant SSCP conformers are indicated by an arrow.</p>
    <p>FIGS. 13A-13C. KCNE1 mutations associated with LQT. Pedigree structure for LQT kindreds 1789 (FIG. 13A) and 1754 (FIG. <b>13</b>B). Affected individuals are indicated by filled circles (females) or squares (males). Unaffected individuals are indicated by open symbols. Deceased individuals are identified by a diagonal slash. Aberrant SSCP conformers that cosegregate with the disease are shown below each pedigree. A common polymorphism (G38S) that is not related to LQT is also detected by these primers. The effect of mutations on hminK protein sequence is indicated. FIG. 13C is a schematic representation of hminK protein showing the location of LQT-associated mutations.</p>
    <p>FIGS. 14A-14B. Magnitude of I<sub>Ks </sub>varies as a function of injected KCNE1 cRNA. A) Representative current tracings elicited by 7.5 second pulses to +40 mV following injection of oocytes with 6 ng/oocyte KVLQT1 and a variable amount of KCNE1 cRNA, as indicated. Note the presence of KvLQT1 current, and the absence of I<sub>Ks </sub>in the oocyte injected with 0.01 ng KCNE1. B) Current amplitude following a 7.5 second pulse to +40 mV was normalized to peak current obtained by injection of 1.2 ng KCNE1. Values represent mean+S.E.M. N=8 oocytes/group.</p>
    <p>FIGS. 15A-15D. Functional effects of D76N KCNE1 mutation. A) I<sub>Ks </sub>was elicited by 7.5 second pulses from a holding potential of −80 mV to test potentials of −40 to +40 mV. Deactivating tail currents were elicited by returning membrane potential to −50 mV. B) Isochronal current-voltage relation of I<sub>Ks-WT </sub>(n=14) and I<sub>Ks-D76N </sub>(n=14), demonstrating dominant negative suppressionof I<sub>Ks </sub>by D76N (p&lt;0.0001). C) The voltage dependence of I<sub>Ks-D76N </sub>activation, using a 7.5 second test pulse, is shifted by +16 mV compared to I<sub>Ks-WT</sub>. Smooth curves are best fits of normalized tail currents to a Boltzmann function (V<sub>½</sub>=10.8±0.8 mV, slope factor=12.1±0.3 mV for I<sub>Ks-WT</sub>; for I<sub>Ks-D76N </sub>V<sub>½</sub>=25.7±1.0 mV [p&lt;0.0001, compared to I<sub>Ks-WT</sub>], slope factor=12.0±0.2 mV; n=14). D) I<sub>Ks-D76N </sub>deactivates faster than I<sub>Ks-WT</sub>. I<sub>Ks </sub>was activated by a 5 second pulse to +20 mV, and tail currents were measured at the indicated potentials. Tail currents were fit to a single exponential function. Inset shows normalized deactivating tail currents at −50 mV, after a voltage step to +20 mV.</p>
    <p>FIGS. 16A-16D. Functional effects of S74L KCNE1 mutation. A) I<sub>Ks-WT </sub>and I<sub>Ks-S74L </sub>recorded during 7.5 second depolarizations to −40, −20, 0, +20 and +40 mV. Note the faster rate of deactivating I<sub>Ks-S74L </sub>tail currents compared to I<sub>Ks-WT</sub>. B) Isochronal current-voltage relation for I<sub>Ks-WT </sub>and I<sub>Ks-S74L </sub>(n=15). C) Voltage dependence of I<sub>Ks-S74L </sub>activation is shifted by +19 mV relative to I<sub>Ks-WT</sub>. Smooth curves are best fits of normalized tail currents to a Boltzmann function (V<sub>½</sub>=13.7±0.6 mV, slope factor=16.0±0.3 mV for I<sub>Ks-WT</sub>; for I<sub>Ks-S74L </sub>V<sub>½</sub>=33.6±0.8 mV, slope factor=13.3±mV [both p&lt;0.0001 relative to I<sub>Ks-WT</sub>]). D) I<sub>Ks-S74L </sub>deactivates faster than I<sub>Ks-WT</sub>.</p>
    <p>FIG. <b>17</b>. Physical map and exon organization of KCNE1. The two cosmid clones spanning the entire KCNE1 transcript are shown. Cosmid 1 does not extend to the end of exon 3 and cosmid 2 does not include exons 1 and 2. Sizes of the exons and distances are not drawn to scale.</p>
    <p>FIG. <b>18</b>. Genomic organization of the KCNE1 coding and 5′ and 3′ untranslated regions. Positions of the introns are indicated with arrowheads. Note that both introns are within the 5′-untranslated region. The asterisk indicates the stop codon. The nucleotide sequence of FIG. 18 is SEQ ID NO:3. The amino acid sequence of FIG. 18 is SEQ ID NO:4.</p>
    <heading>BRIEF DESCRIPTION OF THE SEQUENCE LISTING</heading> <p>SEQ ID NO:1 is human KVLQT1 cDNA.</p>
    <p>SEQ ID NO:2 is human KVLQT1 protein.</p>
    <p>SEQ ID NO:3 is human KCNE1 cDNA.</p>
    <p>SEQ ID NO:4 is human KCNE1 protein.</p>
    <p>SEQ ID NOs:5-6 are hypothetical nucleic acids used to demonstrate calculation of homology.</p>
    <p>SEQ ID NOs:7-8 are oligonucleotides used to capture and repair human KVLQT1 cDNA (see Example 4).</p>
    <p>SEQ ID NOs:9-40 are the intron/exon boundaries of human KVLQT1 (Table 3).</p>
    <p>SEQ ID NOs:41-74 are primers used to amplify KVLQT1 exons (Table 4).</p>
    <p>SEQ ID NOs:75-86 are primers used to define KVLQT1 mutations (Table 5).</p>
    <p>SEQ ID NOs:87-92 are primer pairs used to amplify genomic KCNE1.</p>
    <p>SEQ ID NOs:93-94 are primers used to amplify KCNE1 cDNA.</p>
    <p>SEQ ID NOs:95-100 are intron/exon boundaries of KCNE1 (Table 8).</p>
    <p>SEQ ID NOs:101-106 are primers to amplify KCNE1 exons (Table 9).</p>
    <p>SEQ ID NOs:107-109 are fragments of KVLQT1 shown in FIG. <b>3</b>.</p>
    <p>SEQ ID NOs:110-112 are fragments of DMSHAKE shown in FIG. <b>3</b>.</p>
    <p>SEQ ID NO:113 is a partial Xenopus KVLQT1 shown in FIG. <b>10</b>.</p>
    <p>SEQ ID NO:114 is a partial human KVLQT1 shown in FIG. <b>10</b>.</p>
    <p>SEQ ID NO:115 is a splice form of human KVLQT1 cDNA.</p>
    <p>SEQ ID NO:116 is a human KVLQT1 encoded by SEQ ID NO:115.</p>
    <heading>DETAILED DESCRIPTION OF THE INVENTION</heading> <p>The present invention is directed to the determination of the genomic structure of KVLQT1 and KCNE1 and to molecular variants of these genes which cause or are involved in the pathogenesis of LQT. It is also directed to the determination that KVLQT1 and minK coassemble to form cardiac I<sub>Ks </sub>potassium channels. More specifically, the present invention relates to mutations in the KVLQT1 gene and also in the KCNE1 gene and their use in the diagnosis of LQT. The present invention is further directed to methods of screening humans for the presence of KVLQT1 and/or KCNE1 gene variants which cause LQT. Since LQT can now be detected earlier (i.e., before symptoms appear) and more definitively, better treatment options will be available in those individuals identified as having LQT. The present invention is also directed to methods for screening for drugs useful in treating or preventing LQT1.</p>
    <p>The present invention provides methods of screening the KVLQT1 and/or KCNE1 gene to identify mutations. Such methods may further comprise the step of amplifying a portion of the KVLQT1 or KCNE1 gene, and may further include a step of providing a set of polynucleotides which are primers for amplification of said portion of the KVLQT1 or KCNE1 gene. The method is useful for identifying mutations for use in either diagnosis of LQT or prognosis of LQT.</p>
    <p>The present invention further demonstrates that KCNE1 (encoding KCNE1 which is also referred to in the literature as minK) on chromosome 21 is also involved in LQT. The minK protein and KVLQT1 coassemble to form a K<sup>+ </sup>channel. The present invention thus provides methods of screening the KCNE1 gene to identify mutations. Such methods may further comprise the step of amplifying a portion of the KCNE1 gene, and may further include a step of providing a set of polynucleotides which are primers for amplification of said portion of the KCNE1 gene. The method is useful for identifying mutations for use in either diagnosis of LQT or prognosis of LQT.</p>
    <p>Finally, the present invention is directed to a method for screening drug candidates to identify drugs useful for treating or preventing LQT. Drug screening is performed by coexpressing mutant KVLQT1 and/or KCNE1 genes in cells, such as oocytes, mammalian cells or transgenic animals, and assaying the effect of a drug candidate on the I<sub>Ks </sub>channel. The effect is compared to the I<sub>Ks </sub>channel activity of the wild-type KVLQT1 and KCNE1 genes.</p>
    <p>Proof that the KVLQT1 or KCNE1 gene is involved in causing LQT is obtained by finding sequences in DNA extracted from affected kindred members which create abnormal KVLQT1 or KCNE1 gene products or abnormal levels of the gene products. Such LQT susceptibility alleles will co-segregate with the disease in large kindreds. They will also be present at a much higher frequency in non-kindred individuals with LQT than in individuals in the general population. The key is to find mutations which are serious enough to cause obvious disruption to the normal function of the gene product. These mutations can take a number of forms. The most severe forms would be frame shift mutations or large deletions which would cause the gene to code for an abnormal protein or one which would significantly alter protein expression. Less severe disruptive mutations would include small in-frame deletions and nonconservative base pair substitutions which would have a significant effect on the protein produced, such as changes to or from a cysteine residue, from a basic to an acidic amino acid or vice versa, from a hydrophobic to hydrophilic amino acid or vice versa, or other mutations which would affect secondary or tertiary protein structure. Silent mutations or those resulting in conservative amino acid substitutions would not generally be expected to disrupt protein function.</p>
    <p>According to the diagnostic and prognostic method of the present invention, alteration of the wild-type KVLQT1 or KCNE1 gene is detected. In addition, the method can be performed by detecting the wild-type KVLQT1 or KCNE1 gene and confirming the lack of a cause of LQT as a result of this locus. “Alteration of a wild-type gene” encompasses all forms of mutations including deletions, insertions and point mutations in the coding and noncoding regions. Deletions may be of the entire gene or of only a portion of the gene. Point mutations may result in stop codons, frameshift mutations or amino acid substitutions. Somatic mutations are those which occur only in certain tissues and are not inherited in the germline. Germline mutations can be found in any of a body's tissues and are inherited. Point mutational events may occur in regulatory regions, such as in the promoter of the gene, leading to loss or diminution of expression of the mRNA. Point mutations may also abolish proper RNA processing, leading to loss of expression of the KVLQT1 or KCNE1 gene product, or to a decrease in mRNA stability or translation efficiency.</p>
    <p>Useful diagnostic techniques include, but are not limited to fluorescent in situ hybridization (FISH), direct DNA sequencing, PFGE analysis, Southern blot analysis, single stranded conformation analysis (SSCA), RNase protection assay, allele-specific oligonucleotide (ASO), dot blot analysis and PCR-SSCP, as discussed in detail further below. Also useful is the recently developed technique of DNA microchip technology.</p>
    <p>The presence of LQT may be ascertained by testing any tissue of a human for mutations of the KVLQT1 gene or the KCNE1 gene. For example, a person who has inherited a germline KVLQT1 or KCNE1 mutation would be prone to develop LQT. This can be determined by testing DNA from any tissue of the person's body. Most simply, blood can be drawn and DNA extracted from the cells of the blood. In addition, prenatal diagnosis can be accomplished by testing fetal cells, placental cells or amniotic cells for mutations of the KVLQT1 or KCNE1 gene. Alteration of a wild-type KVLQT1 or KCNE1 allele, whether, for example, by point mutation or deletion, can be detected by any of the means discussed herein.</p>
    <p>There are several methods that can be used to detect DNA sequence variation. Direct DNA sequencing, either manual sequencing or automated fluorescent sequencing can detect sequence variation. Another approach is the single-stranded conformation polymorphism assay (SSCP) (Orita et al., 1989). This method does not detect all sequence changes, especially if the DNA fragment size is greater than 200 bp, but can be optimized to detect most DNA sequence variation. The reduced detection sensitivity is a disadvantage, but the increased throughput possible with SSCP makes it an attractive, viable alternative to direct sequencing for mutation detection on a research basis. The fragments which have shifted mobility on SSCP gels are then sequenced to determine the exact nature of the DNA sequence variation. Other approaches based on the detection of mismatches between the two complementary DNA strands include clamped denaturing gel electrophoresis (CDGE) (Sheffield et al., 1991), heteroduplex analysis (HA) (White et al., 1992) and chemical mismatch cleavage (CMC) (Grompe et al., 1989). None of the methods described above will detect large deletions, duplications or insertions, nor will they detect a regulatory mutation which affects transcription or translation of the protein. Other methods which might detect these classes of mutations such as a protein truncation assay or the asymmetric assay, detect only specific types of mutations and would not detect missense mutations. A review of currently available methods of detecting DNA sequence variation can be found in a recent review by Grompe (1993). Once a mutation is known, an allele specific detection approach such as allele specific oligonucleotide (ASO) hybridization can be utilized to rapidly screen large numbers of other samples for that same mutation. Such a technique can utilize probes which are labeled with gold nanoparticles to yield a visual color result (Elghanian et al., 1997).</p>
    <p>A rapid preliminary analysis to detect polymorphisms in DNA sequences can be performed by looking at a series of Southern blots of DNA cut with one or more restriction enzymes, preferably with a large number of restriction enzymes. Each blot contains a series of normal individuals and a series of LQT cases. Southern blots displaying hybridizing fragments (differing in length from control DNA when probed with sequences near or including the KVLQT1 locus) indicate a possible mutation. If restriction enzymes which produce very large restriction fragments are used, then pulsed field gel electrophoresis (PFGE) is employed.</p>
    <p>Detection of point mutations may be accomplished by molecular cloning of the KVLQT1 or KCNE1 alleles and sequencing the alleles using techniques well known in the art. Also, the gene or portions of the gene may be amplified, e.g., by PCR or other amplification technique, and the amplified gene or amplified portions of the gene may be sequenced.</p>
    <p>There are six well known methods for a more complete, yet still indirect, test for confirming the presence of a susceptibility allele: 1) single stranded conformation analysis (SSCP) (Orita et al., 1989); 2) denaturing gradient gel electrophoresis (DGGE) (Wartell et al., 1990; Sheffield et al., 1989); 3) RNase protection assays (Finkelstein et al., 1990; Kinszler et al., 1991); 4) allele-specific oligonucleotides (ASOs) (Conner et al., 1983); 5) the use of proteins which recognize nucleotide mismatches, such as the <i>E. coli </i>mutS protein (Modrich, 1991); and 6) allele-specific PCR (Ruano and Kidd, 1989). For allele-specific PCR, primers are used which hybridize at their 3' ends to a particular KVLQT1 or KCNE1 mutation. If the particular mutation is not present, an amplification product is not observed. Amplification Refractory Mutation System (ARMS) can also be used, as disclosed in European Patent Application Publication No. 0332435 and in Newton et al., 1989. Insertions and deletions of genes can also be detected by cloning, sequencing and amplification. In addition, restriction fragment length polymorphism (RFLP) probes for the gene or surrounding marker genes can be used to score alteration of an allele or an insertion in a polymorphic fragment. Such a method is particularly useful for screening relatives of an affected individual for the presence of the mutation found in that individual. Other techniques for detecting insertions and deletions as known in the art can be used.</p>
    <p>In the first three methods (SSCP, DGGE and RNase protection assay), a new electrophoretic band appears. SSCP detects a band which migrates differentially because the sequence change causes a difference in single-strand, intramolecular base pairing. RNase protection involves cleavage of the mutant polynucleotide into two or more smaller fragments. DGGE detects differences in migration rates of mutant sequences compared to wild-type sequences, using a denaturing gradient gel. In an allele-specific oligonucleotide assay, an oligonucleotide is designed which detects a specific sequence, and the assay is performed by detecting the presence or absence of a hybridization signal. In the mutS assay, the protein binds only to sequences that contain a nucleotide mismatch in a heteroduplex between mutant and wild-type sequences.</p>
    <p>Mismatches, according to the present invention, are hybridized nucleic acid duplexes in which the two strands are not 100% complementary. Lack of total homology may be due to deletions, insertions, inversions or substitutions. Mismatch detection can be used to detect point mutations in the gene or in its mRNA product. While these techniques are less sensitive than sequencing, they are simpler to perform on a large number of samples. An example of a mismatch cleavage technique is the RNase protection method. In the practice of the present invention, the method involves the use of a labeled riboprobe which is complementary to the human wild-type KVLQT1 or KCNE1 gene coding sequence. The riboprobe and either mRNA or DNA isolated from the person are annealed (hybridized) together and subsequently digested with the enzyme RNase A which is able to detect some mismatches in a duplex RNA structure. If a mismatch is detected by RNase A, it cleaves at the site of the mismatch. Thus, when the annealed RNA preparation is separated on an electrophoretic gel matrix, if a mismatch has been detected and cleaved by RNase A, an RNA product will be seen which is smaller than the full length duplex RNA for the riboprobe and the mRNA or DNA. The riboprobe need not be the full length of the mRNA or gene but can be a segment of either. If the riboprobe comprises only a segment of the mRNA or gene, it will be desirable to use a number of these probes to screen the whole mRNA sequence for mismatches.</p>
    <p>In similar fashion, DNA probes can be used to detect mismatches, through enzymatic or chemical cleavage. See, e.g., Cotton et al., 1988; Shenk et al., 1975; Novack et al., 1986. Alternatively, mismatches can be detected by shifts in the electrophoretic mobility of mismatched duplexes relative to matched duplexes. See, e.g., Cariello, 1988. With either riboprobes or DNA probes, the cellular mRNA or DNA which might contain a mutation can be amplified using PCR (see below) before hybridization. Changes in DNA of the KVLQT1 or KCNE1 gene can also be detected using Southern hybridization, especially if the changes are gross rearrangements, such as deletions and insertions.</p>
    <p>DNA sequences of the KVLQT1 or KCNE1 gene which have been amplified by use of PCR may also be screened using allele-specific probes. These probes are nucleic acid oligomers, each of which contains a region of the gene sequence harboring a known mutation. For example, one oligomer may be about 30 nucleotides in length, corresponding to a portion of the gene sequence. By use of a battery of such allele-specific probes, PCR amplification products can be screened to identify the presence of a previously identified mutation in the gene. Hybridization of allele-specific probes with amplified KVLQT1 or KCNE1 sequences can be performed, for example, on a nylon filter. Hybridization to a particular probe under high stringency hybridization conditions indicates the presence of the same mutation in the tissue as in the allele-specific probe.</p>
    <p>The newly developed technique of nucleic acid analysis via microchip technology is also applicable to the present invention. In this technique, literally thousands of distinct oligonucleotide probes are built up in an array on a silicon chip. Nucleic acid to be analyzed is fluorescently labeled and hybridized to the probes on the chip. It is also possible to study nucleic acid-protein interactions using these nucleic acid microchips. Using this technique one can determine the presence of mutations or even sequence the nucleic acid being analyzed or one can measure expression levels of a gene of interest. The method is one of parallel processing of many, even thousands, of probes at once and can tremendously increase the rate of analysis. Several papers have been published which use this technique. Some of these are Hacia et al., 1996; Shoemaker et al., 1996; Chee et al., 1996; Lockhart et al., 1996; DeRisi et al., 1996; Lipshutz et al., 1995. This method has already been used to screen people for mutations in the breast cancer gene BRCA1 (Hacia et al., 1996). This new technology has been reviewed in a news article in Chemical and Engineering News (Borman, 1996) and been the subject of an editorial (Editorial, Nature Genetics, 1996). Also see Fodor (1997).</p>
    <p>The most definitive test for mutations in a candidate locus is to directly compare genomic KVLQT1 or KCNE1 sequences from patients with those from a control population. Alternatively, one could sequence messenger RNA after amplification, e.g., by PCR, thereby eliminating the necessity of determining the exon structure of the candidate gene.</p>
    <p>Mutations from patients falling outside the coding region of KVLQT1 or KCNE1 can be detected by examining the non-coding regions, such as introns and regulatory sequences near or within the genes. An early indication that mutations in noncoding regions are important may come from Northern blot experiments that reveal messenger RNA molecules of abnormal size or abundance in patients as compared to control individuals.</p>
    <p>Alteration of KVLQT1 or KCNE1 mRNA expression can be detected by any techniques known in the art. These include Northern blot analysis, PCR amplification and RNase protection. Diminished mRNA expression indicates an alteration of the wild-type gene. Alteration of wild-type genes can also be detected by screening for alteration of wild-type KVLQT1 or KCNE1 protein. For example, monoclonal antibodies immunoreactive with KVLQT1 or KCNE1 can be used to screen a tissue. Lack of cognate antigen would indicate a mutation. Antibodies specific for products of mutant alleles could also be used to detect mutant gene product. Such immunological assays can be done in any convenient formats known in the art. These include Western blots, immunohistochemical assays and ELISA assays. Any means for detecting an altered KVLQT1 or KCNE1 protein can be used to detect alteration of the wild-type KVLQT1 or KCNE1 gene. Functional assays, such as protein binding determinations, can be used. In addition, assays can be used which detect KVLQT1 or KCNE1 biochemical function. Finding a mutant KVLQT1 or KCNE1 gene product indicates alteration of a wild-type KVLQT1 or KCNE1 gene.</p>
    <p>A mutant KVLQT1 or KCNE1 gene or gene product can also be detected in other human body samples, such as serum, stool, urine and sputum. The same techniques discussed above for detection of mutant genes or gene products in tissues can be applied to other body samples. By screening such body samples, a simple early diagnosis can be achieved for LQT.</p>
    <p>The primer pairs of the present invention are useful for determination of the nucleotide sequence of a particular KVLQT1 or KCNE1 allele using PCR. The pairs of single-stranded DNA primers for KVLQT1 can be annealed to sequences within or surrounding the KVLQT1 gene on chromosome 11 in order to prime amplifying DNA synthesis of the gene itself. The pairs of single-stranded DNA primers for KCNE1 can be annealed to sequences within or surrounding the KCNE1 gene on chromosome 21 in order to prime amplifying DNA synthesis of the gene itself. A complete set of these primers allows synthesis of all of the nucleotides of the gene coding sequences, i.e., the exons. The set of primers preferably allows synthesis of both intron and exon sequences. Allele-specific primers can also be used. Such primers anneal only to particular KVLQT1 or KCNE1 mutant alleles, and thus will only amplify a product in the presence of the mutant allele as a template.</p>
    <p>In order to facilitate subsequent cloning of amplified sequences, primers may have restriction enzyme site sequences appended to their 5′ ends. Thus, all nucleotides of the primers are derived from KVLQT1 or KCNE1 sequence or sequences adjacent to KVLQT1 or KCNE1, except for the few nucleotides necessary to form a restriction enzyme site. Such enzymes and sites are well known in the art. The primers themselves can be synthesized using techniques which are well known in the art. Generally, the primers can be made using oligonucleotide synthesizing machines which are commercially available. Given the sequence of KVLQT1 and KCNE1, design of particular primers is well within the skill of the art. The present invention adds to this by presenting data on the intron/exon boundaries thereby allowing one to design primers to amplify and sequence all of the exonic regions completely.</p>
    <p>The nucleic acid probes provided by the present invention are useful for a number of purposes. They can be used in Southern hybridization to genomic DNA and in the RNase protection method for detecting point mutations already discussed above. The probes can be used to detect PCR amplification products. They may also be used to detect mismatches with the KVLQT1 or KCNE1 gene or mRNA using other techniques.</p>
    <p>It has been discovered that individuals with the wild-type KVLQT1 or KCNE1 gene do not have LQT. However, mutations which interfere with the function of the KVLQT1 or KCNE1 gene product are involved in the pathogenesis of LQT. Thus, the presence of an altered (or a mutant) KVLQT1 or KCNE1 gene which produces a protein having a loss of function, or altered function, directly causes LQT which increases the risk of cardiac arrhythmias. In order to detect a KVLQT1 or KCNE1 gene mutation, a biological sample is prepared and analyzed for a difference between the sequence of the allele being analyzed and the sequence of the wild-type allele. Mutant KVLQT1 or KCNE1 alleles can be initially identified by any of the techniques described above. The mutant alleles are then sequenced to identify the specific mutation of the particular mutant allele. Alternatively, mutant alleles can be initially identified by identifying mutant (altered) proteins, using conventional techniques. The mutant alleles are then sequenced to identify the specific mutation for each allele. The mutations, especially those which lead to an altered function of the protein, are then used for the diagnostic and prognostic methods of the present invention.</p>
    <p>It has also been discovered that the KVLQT1 protein coassembles with the minK protein. Thus, mutations in KCNE1 (which encodes minK) which interfere in the function of the KCNE1 gene product are involved in the pathogenesis of LQT. Thus, the presence of an altered (or a mutant) KCNE1 gene which produces a protein having a loss of function, or altered function, directly causes LQT which increases the risk of cardiac arrhythmias. In order to detect a KCNE1 gene mutation, a biological sample is prepared and analyzed for a difference between the sequence of the allele being analyzed and the sequence of the wild-type allele. Mutant KCNE1 alleles can be initially identified by any of the techniques described above. The mutant alleles are then sequenced to identify the specific mutation of the particular mutant (altered) proteins, using conventional techniques. The mutant alleles are then sequenced to identify the specific mutation for each allele. The mutations, especially those which lead to an altered function of the protein, are then used for the diagnostic and prognostic methods of the present invention.</p>
    <p>Definitions</p>
    <p>The present invention employs the following definitions:</p>
    <p>“Amplification of Polynucleotides” utilizes methods such as the polymerase chain reaction (PCR), ligation amplification (or ligase chain reaction, LCR) and amplification methods based on the use of Q-beta replicase. Also useful are strand displacement amplification (SDA), thermophilic SDA, and nucleic acid sequence based amplification (3SR or NASBA). These methods are well known and widely practiced in the art. See, e.g., U.S. Pat. Nos. 4,683,195 and 4,683,202 and Innis et al., 1990 (for PCR); Wu and Wallace, 1989 (for LCR); U.S. Pat. Nos. 5,270,184 and 5,455,166 and Walker et al., 1992 (for SDA); Spargo et al., 1996 (for thermophilic SDA) and U.S. Pat. No. 5,409,818, Fahy et al., 1991 and Compton, 1991 for 3SR and NASBA. Reagents and hardware for conducting PCR are commercially available. Primers useful to amplify sequences from the KVLQT1 or KCNE1 region are preferably complementary to, and hybridize specifically to sequences in the KVLQT1 or KCNE1 region or in regions that flank a target region therein. KVLQT1 or KCNE1 sequences generated by amplification may be sequenced directly. Alternatively, but less desirably, the amplified sequence(s) may be cloned prior to sequence analysis. A method for the direct cloning and sequence analysis of enzymatically amplified genomic segments has been described by Scharf et al., 1986.</p>
    <p>“Analyte polynucleotide” and “analyte strand” refer to a single- or double-stranded polynucleotide which is suspected of containing a target sequence, and which may be present in a variety of types of samples, including biological samples.</p>
    <p>“Antibodies.” The present invention also provides polyclonal and/or monoclonal antibodies and fragments thereof, and immunologic binding equivalents thereof, which are capable of specifically binding to the KVLQT1 or KCNE1 polypeptide and fragments thereof or to polynucleotide sequences from the KVLQT1 or KCNE1 region. The term “antibody” is used both to refer to a homogeneous molecular entity, or a mixture such as a serum product made up of a plurality of different molecular entities. Polypeptides may be prepared synthetically in a peptide synthesizer and coupled to a carrier molecule (e.g., keyhole limpet hemocyanin) and injected over several months into rabbits. Rabbit sera is tested for immunoreactivity to the KVLQT1 or KCNE1 polypeptide or fragment. Monoclonal antibodies may be made by injecting mice with the protein polypeptides, fusion proteins or fragments thereof. Monoclonal antibodies will be screened by ELISA and tested for specific immunoreactivity with KVLQT1 or KCNE1 polypeptide or fragments thereof. See, Harlow and Lane, 1988. These antibodies will be useful in assays as well as pharmaceuticals.</p>
    <p>Once a sufficient quantity of desired polypeptide has been obtained, it may be used for various purposes. A typical use is the production of antibodies specific for binding. These antibodies may be either polyclonal or monoclonal, and may be produced by in vitro or in vivo techniques well known in the art. For production of polyclonal antibodies, an appropriate target immune system, typically mouse or rabbit, is selected. Substantially purified antigen is presented to the immune system in a fashion determined by methods appropriate for the animal and by other parameters well known to immunologists. Typical sites for injection are in footpads, intramuscularly, intraperitoneally, or intradermally. Of course, other species may be substituted for mouse or rabbit. Polyclonal antibodies are then purified using techniques known in the art, adjusted for the desired specificity.</p>
    <p>An immunological response is usually assayed with an immunoassay. Normally, such immunoassays involve some purification of a source of antigen, for example, that produced by the same cells and in the same fashion as the antigen. A variety of immunoassay methods are well known in the art. See, e.g., Harlow and Lane, 1988, or Goding, 1986.</p>
    <p>Monoclonal antibodies with affinities of 10<sup>−8 </sup>M<sup>−1 </sup>or preferably 10<sup>−9 </sup>to 10<sup>−10 </sup>M<sup>−1 </sup>or stronger will typically be made by standard procedures as described, e.g., in Harlow and Lane, 1988 or Goding, 1986. Briefly, appropriate animals will be selected and the desired immunization protocol followed. After the appropriate period of time, the spleens of such animals are excised and individual spleen cells fused, typically, to immortalized myeloma cells under appropriate selection conditions. Thereafter, the cells are clonally separated and the supernatants of each clone tested for their production of an appropriate antibody specific for the desired region of the antigen.</p>
    <p>Other suitable techniques involve in vitro exposure of lymphocytes to the antigenic polypeptides, or alternatively, to selection of libraries of antibodies in phage or similar vectors. See Huse et al., 1989. The polypeptides and antibodies of the present invention may be used with or without modification. Frequently, polypeptides and antibodies will be labeled by joining, either covalently or non-covalently, a substance which provides for a detectable signal. A wide variety of labels and conjugation techniques are known and are reported extensively in both the scientific and patent literature. Suitable labels include radionuclides, enzymes, substrates, cofactors, inhibitors, fluorescent agents, chemiluminescent agents, magnetic particles and the like. Patents teaching the use of such labels include U.S. Pat. Nos. 3,817,837; 3,850,752; 3,939,350; 3,996,345; 4,277,437; 4,275,149 and 4,366,241. Also, recombinant immunoglobulins may be produced (see U.S. Pat. No. 4,816,567).</p>
    <p>“Binding partner” refers to a molecule capable of binding a ligand molecule with high specificity, as for example, an antigen and an antigen-specific antibody or an enzyme and its inhibitor. In general, the specific binding partners must bind with sufficient affinity to immobilize the analyte copy/complementary strand duplex (in the case of polynucleotide hybridization) under the isolation conditions. Specific binding partners are known in the art and include, for example, biotin and avidin or streptavidin, IgG and protein A, the numerous, known receptor-ligand couples, and complementary polynucleotide strands. In the case of complementary polynucleotide binding partners, the partners are normally at least about 15 bases in length, and may be at least 40 bases in length. It is well recognized by those of skill in the art that lengths shorter than 15 (e.g., 8 bases), between 15 and 40, and greater than 40 bases may also be used. The polynucleotides may be composed of DNA, RNA, or synthetic nucleotide analogs. Further binding partners can be identified using, e.g., the two-hybrid yeast screening assay as described herein.</p>
    <p>A “biological sample” refers to a sample of tissue or fluid suspected of containing an analyte polynucleotide or polypeptide from an individual including, but not limited to, e.g., plasma, serum, spinal fluid, lymph fluid, the external sections of the skin, respiratory, intestinal, and genitourinary tracts, tears, saliva, blood cells, tumors, organs, tissue and samples of in vitro cell culture constituents.</p>
    <p>“Encode”. A polynucleotide is said to “encode” a polypeptide if, in its native state or when manipulated by methods well known to those skilled in the art, it can be transcribed and/or translated to produce the mRNA for and/or the polypeptide or a fragment thereof. The anti-sense strand is the complement of such a nucleic acid, and the encoding sequence can be deduced therefrom.</p>
    <p>“Isolated” or “substantially pure”. An “isolated” or “substantially pure” nucleic acid (e.g., an RNA, DNA or a mixed polymer) is one which is substantially separated from other cellular components which naturally accompany a native human sequence or protein, e.g., ribosomes, polymerases, many other human genome sequences and proteins. The term embraces a nucleic acid sequence or protein which has been removed from its naturally occurring environment, and includes recombinant or cloned DNA isolates and chemically synthesized analogs or analogs biologically synthesized by heterologous systems.</p>
    <p>“KVLQT1 or KCNE1 Allele” refers, respectively, to normal alleles of the KVLQT1 or KCNE1 locus as well as alleles of KVLQT1 or KCNE1 carrying variations that cause LQT.</p>
    <p>“KVLQT1 or KCNE1 Locus”, “KVLQT1 or KCNE1 Gene”, “KVLQT1 or KCNE1 Nucleic Acids” or “KVLQT1 or KCNE1 Polynucleotide” each refer to polynucleotides, all of which are in the KVLQT1 or KCNE1 region, respectively, that are likely to be expressed in normal tissue, certain alleles of which result in LQT. The KVLQT1 or KCNE1 locus is intended to include coding sequences, intervening sequences and regulatory elements controlling transcription and/or translation. The KVLQT1 or KCNE1 locus is intended to include all allelic variations of the DNA sequence. The terms “KCNE1 ” and “minK” may be used interchangeably.</p>
    <p>These terms, when applied to a nucleic acid, refer to a nucleic acid which encodes a human KVLQT1 or KCNE1 polypeptide, fragment, homolog or variant, including, e.g., protein fusions or deletions. The nucleic acids of the present invention will possess a sequence which is either derived from, or substantially similar to a natural KVLQT1- or KCNE1-encoding gene or one having substantial homology with a natural KVLQT1- or KCNE1-encoding gene or a portion thereof.</p>
    <p>The KVLQT1 or KCNE1 gene or nucleic acid includes normal alleles of the KVLQT1 or KCNE1 gene, respectively, including silent alleles having no effect on the amino acid sequence of the KVLQT1 or KCNE1 polypeptide as well as alleles leading to amino acid sequence variants of the KVLQT1 or KCNE1 polypeptide that do not substantially affect its function. These terms also include alleles having one or more mutations which adversely affect the function of the KVLQT1 or KCNE1 polypeptide. A mutation may be a change in the KVLQT1 or KCNE1 nucleic acid sequence which produces a deleterious change in the amino acid sequence of the KVLQT1 or KCNE1 polypeptide, resulting in partial or complete loss of KVLQT1 or KCNE1 function, respectively, or may be a change in the nucleic acid sequence which results in the loss of effective KVLQT1 or KCNE1 expression or the production of aberrant forms of the KVLQT1 or KCNE1 polypeptide.</p>
    <p>The KVLQT1 or KCNE1 nucleic acid may be that shown in SEQ ID NO:1 (KVLQT1) or SEQ ID NO:3 (KCNE1) or it may be an allele as described above or a variant or derivative differing from that shown by a change which is one or more of addition, insertion, deletion and substitution of one or more nucleotides of the sequence shown. Changes to the nucleotide sequence may result in an amino acid change at the protein level, or not, as determined by the genetic code.</p>
    <p>Thus, nucleic acid according to the present invention may include a sequence different from the sequence shown in SEQ ID NOs:1 and 3 yet encode a polypeptide with the same amino acid sequence as shown in SEQ ID NOs:2 (KVLQT1) and 4 (KCNE1). That is, nucleic acids of the present invention include sequences which are degenerate as a result of the genetic code. On the other hand, the encoded polypeptide may comprise an amino acid sequence which differs by one or more amino acid residues from the amino acid sequence shown in SEQ ID NOs:2 and 4. Nucleic acid encoding a polypeptide which is an amino acid sequence variant, derivative or allele of the amino acid sequence shown in SEQ ID NOs:2 and 4 is also provided by the present invention.</p>
    <p>The KVLQT1 or KCNE1 gene, respectively, also refers to (a) any DNA sequence that (i) hybridizes to the complement of the DNA sequences that encode the amino acid sequence set forth in SEQ ID NO:2 or SEQ ID NO:4 under highly stringent conditions (Ausubel et al., 1992) and (ii) encodes a gene product functionally equivalent to KVLQT1 or KCNE1, or (b) any DNA sequence that (i) hybridizes to the complement of the DNA sequences that encode the amino acid sequence set forth in SEQ ID NO:2 or SEQ ID NO:4 under less stringent conditions, such as moderately stringent conditions (Ausubel et al., 1992) and (ii) encodes a gene product functionally equivalent to KVLQT1 or KCNE1. The invention also includes nucleic acid molecules that are the complements of the sequences described herein.</p>
    <p>The polynucleotide compositions of this invention include RNA, cDNA, genomic DNA, synthetic forms, and mixed polymers, both sense and antisense strands, and may be chemically or biochemically modified or may contain non-natural or derivatized nucleotide bases, as will be readily appreciated by those skilled in the art. Such modifications include, for example, labels, methylation, substitution of one or more of the naturally occurring nucleotides with an analog, internucleotide modifications such as uncharged linkages (e.g., methyl phosphonates, phosphotriesters, phosphoramidates, carbamates, etc.), charged linkages (e.g., phosphorothioates, phosphorodithioates, etc.), pendent moieties (e.g., polypeptides), intercalators (e.g., acridine, psoralen, etc.), chelators, alkylators, and modified linkages (e.g., alpha anomeric nucleic acids, etc.). Also included are synthetic molecules that mimic polynucleotides in their ability to bind to a designated sequence via hydrogen bonding and other chemical interactions. Such molecules are known in the art and include, for example, those in which peptide linkages substitute for phosphate linkages in the backbone of the molecule.</p>
    <p>The present invention provides recombinant nucleic acids comprising all or part of the KVLQT1 or KCNE1 region. The recombinant construct may be capable of replicating autonomously in a host cell. Alternatively, the recombinant construct may become integrated into the chromosomal DNA of the host cell. Such a recombinant polynucleotide comprises a polynucleotide of genomic, cDNA, semi-synthetic, or synthetic origin which, by virtue of its origin or manipulation, 1) is not associated with all or a portion of a polynucleotide with which it is associated in nature; 2) is linked to a polynucleotide other than that to which it is linked in nature; or 3) does not occur in nature. Where nucleic acid according to the invention includes RNA, reference to the sequence shown should be construed as reference to the RNA equivalent, with U substituted for T.</p>
    <p>Therefore, recombinant nucleic acids comprising sequences otherwise not naturally occurring are provided by this invention. Although the wild-type sequence may be employed, it will often be altered, e.g., by deletion, substitution or insertion. cDNA or genomic libraries of various types may be screened as natural sources of the nucleic acids of the present invention, or such nucleic acids may be provided by amplification of sequences resident in genomic DNA or other natural sources, e.g., by PCR. The choice of cDNA libraries normally corresponds to a tissue source which is abundant in mRNA for the desired proteins. Phage libraries are normally preferred, but other types of libraries may be used. Clones of a library are spread onto plates, transferred to a substrate for screening, denatured and probed for the presence of desired sequences.</p>
    <p>The DNA sequences used in this invention will usually comprise at least about five codons (15 nucleotides), more usually at least about 7-15 codons, and most preferably, at least about 35 codons. One or more introns may also be present. This number of nucleotides is usually about the minimal length required for a successful probe that would hybridize specifically with a KVLQT1- or KCNE1-encoding sequence. In this context, oligomers of as low as 8 nucleotides, more generally 8-17 nucleotides, can be used for probes, especially in connection with chip technology.</p>
    <p>Techniques for nucleic acid manipulation are described generally, for example, in Sambrook et al., 1989 or Ausubel et al., 1992. Reagents useful in applying such techniques, such as restriction enzymes and the like, are widely known in the art and commercially available from such vendors as New England BioLabs, Boehringer Mannheim, Amersham, Promega, U.S. Biochemicals, New England Nuclear, and a number of other sources. The recombinant nucleic acid sequences used to produce fusion proteins of the present invention may be derived from natural or synthetic sequences. Many natural gene sequences are obtainable from various cDNA or from genomic libraries using appropriate probes. See, GenBank, National Institutes of Health.</p>
    <p>As used herein, a “portion” of the KVLQT1 or KCNE1 locus or region or allele is defined as having a minimal size of at least about eight nucleotides, or preferably about 15 nucleotides, or more preferably at least about 25 nucleotides, and may have a minimal size of at least about 40 nucleotides. This definition includes all sizes in the range of 8-40 nucleotides as well as greater than 40 nucleotides. Thus, this definition includes nucleic acids of 8, 12, 15, 20, 25, 40, 60, 80, 100, 200, 300, 400, 500 nucleotides, or nucleic acids having any number of nucleotides within these ranges of values (e.g., 9, 10, 11, 16, 23, 30, 38, 50, 72, 121, etc., nucleotides), or nucleic acids having more than 500 nucleotides. The present invention includes all novel nucleic acids having at least 8 nucleotides derived from SEQ ID NO:1 or SEQ ID NO:3, its complement or functionally equivalent nucleic acid sequences. The present invention does not include nucleic acids which exist in the prior art. That is, the present invention includes all nucleic acids having at least 8 nucleotides derived from SEQ ID NO:1 or SEQ ID NO:3 with the proviso that it does not include nucleic acids existing in the prior art.</p>
    <p>“KVLQT1 or KCNE1 protein” or “KVLQT1 or KCNE1 polypeptide” refers to a protein or polypeptide encoded by the KVLQT1 or KCNE1 locus, variants or fragments thereof. The terms “KCNE1” and “minK” are used interchangeably. The term “polypeptide” refers to a polymer of amino acids and its equivalent and does not refer to a specific length of the product; thus, peptides, oligopeptides and proteins are included within the definition of a polypeptide. This term also does not refer to, or exclude modifications of the polypeptide, for example, glycosylations, acetylations, phosphorylations, and the like. Included within the definition are, for example, polypeptides containing one or more analogs of an amino acid (including, for example, unnatural amino acids, etc.), polypeptides with substituted linkages as well as other modifications known in the art, both naturally and non-naturally occurring. Ordinarily, such polypeptides will be at least about 50% homologous to the native KVLQT1 or KCNE1 sequence, preferably in excess of about 90%, and more preferably at least about 95% homologous. Also included are proteins encoded by DNA which hybridize under high or low stringency conditions, to KVLQT1-or KCNE1-encoding nucleic acids and closely related polypeptides or proteins retrieved by antisera to the KVLQT1 or KCNE1 protein(s).</p>
    <p>The KVLQT1 or KCNE1 polypeptide may be that shown in SEQ ID NO:2 or SEQ ID NO:4 which may be in isolated and/or purified form, free or substantially free of material with which it is naturally associated. The polypeptide may, if produced by expression in a prokaryotic cell or produced synthetically, lack native post-translational processing, such as glycosylation. Alternatively, the present invention is also directed to polypeptides which are sequence variants, alleles or derivatives of the KVLQT1 or KCNE1 polypeptide. Such polypeptides may have an amino acid sequence which differs from that set forth in SEQ ID NO:2 or SEQ ID NO:4 by one or more of addition, substitution, deletion or insertion of one or more amino acids. Preferred such polypeptides have KVLQT1 or KCNE1 function.</p>
    <p>Substitutional variants typically contain the exchange of one amino acid for another at one or more sites within the protein, and may be designed to modulate one or more properties of the polypeptide, such as stability against proteolytic cleavage, without the loss of other functions or properties. Amino acid substitutions may be made on the basis of similarity in polarity, charge, solubility, hydrophobicity, hydrophilicity, and/or the amphipathic nature of the residues involved. Preferred substitutions are ones which are conservative, that is, one amino acid is replaced with one of similar shape and charge. Conservative substitutions are well known in the art and typically include substitutions within the following groups: glycine, alanine; valine, isoleucine, leucine; aspartic acid, glutamic acid; asparagine, glutamine; serine, threonine; lysine, arginine; and tyrosine, phenylalanine.</p>
    <p>Certain amino acids may be substituted for other amino acids in a protein structure without appreciable loss of interactive binding capacity with structures such as, for example, antigen-binding regions of antibodies or binding sites on substrate molecules or binding sites on proteins interacting with the KVLQT1 or KCNE1 polypeptide. Since it is the interactive capacity and nature of a protein which defines that protein's biological functional activity, certain amino acid substitutions can be made in a protein sequence, and its underlying DNA coding sequence, and nevertheless obtain a protein with like properties. In making such changes, the hydropathic index of amino acids may be considered. The importance of the hydrophobic amino acid index in conferring interactive biological function on a protein is generally understood in the art (Kyte and Doolittle, 1982). Alternatively, the substitution of like amino acids can be made effectively on the basis of hydrophilicity. The importance of hydrophilicity in conferring interactive biological function of a protein is generally understood in the art (U.S. Pat. No. 4,554,101). The use of the hydrophobic index or hydrophilicity in designing polypeptides is further discussed in U.S. Pat. No. 5,691,198.</p>
    <p>The length of polypeptide sequences compared for homology will generally be at least about 16 amino acids, usually at least about 20 residues, more usually at least about 24 residues, typically at least about 28 residues, and preferably more than about 35 residues.</p>
    <p>“Operably linked” refers to a juxtaposition wherein the components so described are in a relationship permitting them to function in their intended manner. For instance, a promoter is operably linked to a coding sequence if the promoter affects its transcription or expression.</p>
    <p>The term peptide mimetic or mimetic is intended to refer to a substance which has the essential biological activity of the KVLQT1 or KCNE1 polypeptide. A peptide mimetic may be a peptide-containing molecule that mimics elements of protein secondary structure (Johnson et al., 1993). The underlying rationale behind the use of peptide mimetics is that the peptide backbone of proteins exists chiefly to orient amino acid side chains in such a way as to facilitate molecular interactions, such as those of antibody and antigen, enzyme and substrate or scaffolding proteins. A peptide mimetic is designed to permit molecular interactions similar to the natural molecule. A mimetic may not be a peptide at all, but it will retain the essential biological activity of natural KVLQT1 or KCNE1 polypeptide.</p>
    <p>“Probes” Polynucleotide polymorphisms associated with KVLQT1 or KCNE1 alleles which predispose to LQT are detected by hybridization with a polynucleotide probe which forms a stable hybrid with that of the target sequence, under stringent to moderately stringent hybridization and wash conditions. If it is expected that the probes will be perfectly complementary to the target sequence, high stringency conditions will be used. Hybridization stringency may be lessened if some mismatching is expected, for example, if variants are expected with the result that the probe will not be completely complementary. Conditions are chosen which rule out nonspecific/adventitious bindings, that is, which minimize noise. (It should be noted that throughout this disclosure, if it is simply stated that “stringent” conditions are used that is meant to be read as “high stringency” conditions are used.) Since such indications identify neutral DNA polymorphisms as well as mutations, these indications need further analysis to demonstrate detection of a KVLQT1 or KCNE1 susceptibility allele.</p>
    <p>Probes for KVLQT1 or KCNE1 alleles may be derived from the sequences of the KVLQT1 or KCNE1 region, its cDNA, functionally equivalent sequences, or the complements thereof. The probes may be of any suitable length, which span all or a portion of the KVLQT1 or KCNE1 region, and which allow specific hybridization to the region. If the target sequence contains a sequence identical to that of the probe, the probes may be short, e.g., in the range of about 8-30 base pairs, since the hybrid will be relatively stable under even stringent conditions. If some degree of mismatch is expected with the probe, i.e., if it is suspected that the probe will hybridize to a variant region, a longer probe may be employed which hybridizes to the target sequence with the requisite specificity.</p>
    <p>The probes will include an isolated polynucleotide attached to a label or reporter molecule and may be used to isolate other polynucleotide sequences, having sequence similarity by standard methods. For techniques for preparing and labeling probes see, e.g., Sambrook et al., 1989 or Ausubel et al., 1992. Other similar polynucleotides may be selected by using homologous polynucleotides. Alternatively, polynucleotides encoding these or similar polypeptides may be synthesized or selected by use of the redundancy in the genetic code. Various codon substitutions may be introduced, e.g., by silent changes (thereby producing various restriction sites) or to optimize expression for a particular system. Mutations may be introduced to modify the properties of the polypeptide, perhaps to change the polypeptide degradation or turnover rate.</p>
    <p>Probes comprising synthetic oligonucleotides or other polynucleotides of the present invention may be derived from naturally occurring or recombinant single- or double-stranded polynucleotides, or be chemically synthesized. Probes may also be labeled by nick translation, Klenow fill-in reaction, or other methods known in the art.</p>
    <p>Portions of the polynucleotide sequence having at least about eight nucleotides, usually at least about 15 nucleotides, and fewer than about 9 kb, usually fewer than about 1.0 kb, from a polynucleotide sequence encoding KVLQT1 or KCNE1 are preferred as probes. This definition therefore includes probes of sizes 8 nucleotides through 9000 nucleotides. Thus, this definition includes probes of 8, 12, 15, 20, 25, 40, 60, 80, 100, 200, 300, 400 or 500 nucleotides or probes having any number of nucleotides within these ranges of values (e.g., 9, 10, 11, 16, 23, 30, 38, 50, 72, 121, etc., nucleotides), or probes having more than 500 nucleotides. The probes may also be used to determine whether MnRNA encoding KVLQT1 or KCNE1 is present in a cell or tissue. The present invention includes all novel probes having at least 8 nucleotides derived from SEQ ID NO:1 or SEQ ID NO:3, its complement or functionally equivalent nucleic acid sequences. The present invention does not include probes which exist in the prior art. That is, the present invention includes all probes having at least 8 nucleotides derived from SEQ ID NO:1 or SEQ ID NO:3 with the proviso that they do not include probes existing in the prior art.</p>
    <p>Similar considerations and nucleotide lengths are also applicable to primers which may be used for the amplification of all or part of the KVLQT1 or KCNE1 gene. Thus, a definition for primers includes primers of 8, 12, 15, 20, 25, 40, 60, 80, 100, 200, 300,400, 500 nucleotides, or primers having any number of nucleotides within these ranges of values (e.g., 9, 10, 11, 16, 23, 30, 38, 50, 72, 121, etc. nucleotides), or primers having more than 500 nucleotides, or any number of nucleotides between 500 and 9000. The primers may also be used to determine whether mRNA encoding KVLQT1 or KCNE1 is present in a cell or tissue. The present invention includes all novel primers having at least 8 nucleotides derived from the KVLQT1 or KCNE1 locus for amplifying the KVLQT1 or KCNE1 gene, its complement or functionally equivalent nucleic acid sequences. The present invention does not include primers which exist in the prior art. That is, the present invention includes all primers having at least 8 nucleotides with the proviso that it does not include primers existing in the prior art.</p>
    <p>“Protein modifications or fragments” are provided by the present invention for KVLQT1 or KCNE1 polypeptides or fragments thereof which are substantially homologous to primary structural sequence but which include, e.g., in vivo or in vitro chemical and biochemical modifications or which incorporate unusual amino acids. Such modifications include, for example, acetylation, carboxylation, phosphorylation, glycosylation, ubiquitination, labeling, e.g., with radionuclides, and various enzymatic modifications, as will be readily appreciated by those well skilled in the art. A variety of methods for labeling polypeptides and of substituents or labels useful for such purposes are well known in the art, and include radioactive isotopes such as <sup>32</sup>P, ligands which bind to labeled antiligands (e.g., antibodies), fluorophores, chemiluminescent agents, enzymes, and antiligands which can serve as specific binding pair members for a labeled ligand. The choice of label depends on the sensitivity required, ease of conjugation with the primer, stability requirements, and available instrumentation. Methods of labeling polypeptides are well known in the art. See Sambrook et al., 1989 or Ausubel et al., 1992.</p>
    <p>Besides substantially full-length polypeptides, the present invention provides for biologically active fragments of the polypeptides. Significant biological activities include ligand-binding, immunological activity and other biological activities characteristic of KVLQT1 or KCNE1 polypeptides. Immunological activities include both immunogenic function in a target immune system, as well as sharing of immunological epitopes for binding, serving as either a competitor or substitute antigen for an epitope of the KVLQT1 or KCNE1 protein. As used herein, “epitope” refers to an antigenic determinant of a polypeptide. An epitope could comprise three amino acids in a spatial conformation which is unique to the epitope. Generally, an epitope consists of at least five such amino acids, and more usually consists of at least 8-10 such amino acids. Methods of determining the spatial conformation of such amino acids are known in the art.</p>
    <p>For immunological purposes, tandem-repeat polypeptide segments may be used as immunogens, thereby producing highly antigenic proteins. Alternatively, such polypeptides will serve as highly efficient competitors for specific binding. Production of antibodies specific for KVLQT1 or KCNE 1 polypeptides or fragments thereof is described below.</p>
    <p>The present invention also provides for fusion polypeptides, comprising KVLQT1 or KCNE1 polypeptides and fragments. Homologous polypeptides may be fusions between two or more KVLQT1 or KCNE1 polypeptide sequences or between the sequences of KVLQT1 or KCNE1 and a related protein. Likewise, heterologous fusions may be constructed which would exhibit a combination of properties or activities of the derivative proteins. For example, ligand-binding or other domains may be “swapped” between different new fusion polypeptides or fragments. Such homologous or heterologous fusion polypeptides may display, for example, altered strength or specificity of binding. Fusion partners include immunoglobulins, bacterial β-galactosidase, trpE, protein A, β-lactamase, alpha amylase, alcohol dehydrogenase and yeast alpha mating factor. See Godowski et al., 1988.</p>
    <p>Fusion proteins will typically be made by either recombinant nucleic acid methods, as described below, or may be chemically synthesized. Techniques for the synthesis of polypeptides are described, for example, in Merrifield (1963).</p>
    <p>“Protein purification” refers to various methods for the isolation of the KVLQT1 or KCNE1 polypeptides from other biological material, such as from cells transformed with recombinant nucleic acids encoding KVLQT1 or KCNE1, and are well known in the art. For example, such polypeptides may be purified by immunoaffinity chromatography employing, e.g., the antibodies provided by the present invention. Various methods of protein purification are well known in the art, and include those described in Deutscher, 1990 and Scopes, 1982.</p>
    <p>The terms “isolated”, “substantially pure”, and “substantially homogeneous” are used interchangeably to describe a protein or polypeptide which has been separated from components which accompany it in its natural state. A monomeric protein is substantially pure when at least about 60 to 75% of a sample exhibits a single polypeptide sequence. A substantially pure protein will typically comprise about 60 to 90% W/W of a protein sample, more usually about 95%, and preferably will be over about 99% pure. Protein purity or homogeneity may be indicated by a number of means well known in the art, such as polyacrylamide gel electrophoresis of a protein sample, followed by visualizing a single polypeptide band upon staining the gel. For certain purposes, higher resolution may be provided by using HPLC or other means well known in the art which are utilized for purification.</p>
    <p>A KVLQT1 or KCNE1 protein is substantially free of naturally associated components when it is separated from the native contaminants which accompany it in its natural state. Thus, a polypeptide which is chemically synthesized or synthesized in a cellular system different from the cell from which it naturally originates will be substantially free from its naturally associated components. A protein may also be rendered substantially free of naturally associated components by isolation, using protein purification techniques well known in the art.</p>
    <p>A polypeptide produced as an expression product of an isolated and manipulated genetic sequence is an “isolated polypeptide”, as used herein, even if expressed in a homologous cell type. Synthetically made forms or molecules expressed by heterologous cells are inherently isolated molecules.</p>
    <p>“Recombinant nucleic acid” is a nucleic acid which is not naturally occurring, or which is made by the artificial combination of two otherwise separated segments of sequence. This artificial combination is often accomplished by either chemical synthesis means, or by the artificial manipulation of isolated segments of nucleic acids, e.g., by genetic engineering techniques. Such is usually done to replace a codon with a redundant codon encoding the same or a conservative amino acid, while typically introducing or removing a sequence recognition site. Alternatively, it is performed to join together nucleic acid segments of desired functions to generate a desired combination of functions.</p>
    <p>“Regulatory sequences” refers to those sequences normally within 100 kb of the coding region of a locus, but they may also be more distant from the coding region, which affect the expression of the gene (including transcription of the gene, and translation, splicing, stability or the like of the messenger RNA).</p>
    <p>“Substantial homology or similarity”. A nucleic acid or fragment thereof is “substantially homologous” (“or substantially similar”) to another if, when optimally aligned (with appropriate nucleotide insertions or deletions) with the other nucleic acid (or its complementary strand), there is nucleotide sequence identity in at least about 60% of the nucleotide bases, usually at least about 70%, more usually at least about 80%, preferably at least about 90%, and more preferably at least about 95-98% of the nucleotide bases.</p>
    <p>To determine homology between two different nucleic acids, the percent homology is to be determined using the BLASTN program “BLAST 2 sequences”. This program is available for public use from the National Center for Biotechnology Information (NCBI) over the Internet (http://www.ncbi.nlm.nih.gov/gorf/b12.html) (Altschul et al., 1997). The parameters to be used are whatever combination of the following yields the highest calculated percent homology (as calculated below) with the default parameters shown in parentheses:</p>
    <p>Program—blastn</p>
    <p>Matrix—0 BLOSUM62</p>
    <p>Reward for a match—0 or 1 (1)</p>
    <p>Penalty for a mismatch—0, −1, −2 or −3 (−2)</p>
    <p>Open gap penalty—0, 1, 2, 3, 4 or 5 (5)</p>
    <p>Extension gap penalty—0 or 1 (1)</p>
    <p>Gap x_dropoff—0 or 50 (50)</p>
    <p>Expect—10</p>
    <p>Along with a variety of other results, this program shows a percent identity across the complete strands or across regions of the two nucleic acids being matched. The program shows as part of the results an alignment and identity of the two strands being compared. If the strands are of equal length then the identity will be calculated across the complete length of the nucleic acids. If the strands are of unequal lengths, then the length of the shorter nucleic acid is to be used. If the nucleic acids are quite similar across a portion of their sequences but different across the rest of their sequences, the blastn program “BLAST 2 Sequences” will show an identity across only the similar portions, and these portions are reported individually. For purposes of determining homology herein, the percent homology refers to the shorter of the two sequences being compared. If any one region is shown in different alignments with differing percent identities, the alignments which yield the greatest homology are to be used. The averaging is to be performed as in this example of SEQ ID NOs:5 and 6.</p>
    <p>
      <maths> <formula-text>5′-ACCGTAGCTACGTACGTATATAGAAAGGGCGCGATCGTCGTCGCGTATGACGAC TTAGCATGC-3′  (SEQ ID NO:5)</formula-text> </maths> </p>
    <p>
      <maths> <formula-text>5′-ACCGGTAGCTACGTACGTTATTTAGAAAGGGGTGTGTGTGTGTGTGTAAACCGGG GTTTTCGGGATCGTCCGTCGCGTATGACGACTTAGCCATGCACGGTATATCGTATT AGGACTAGCGATTGACTAG-3′  (SEQ ID NO:6)</formula-text> </maths> </p>
    <p>The program “BLAST 2 Sequences” shows differing alignments of these two nucleic acids depending upon the parameters which are selected. As examples, four sets of parameters were selected for comparing SEQ ID NOs:5 and 6 (gap x_dropoff was 50 for all cases), with the results shown in Table 1. It is to be noted that none of the sets of parameters selected as shown in Table 1 is necessarily the best set of parameters for comparing these sequences. The percent homology is calculated by multiplying for each region showing identity the fraction of bases of the shorter strand within a region times the percent identity for that region and adding all of these together. For example, using the first set of parameters shown in Table 1, SEQ ID NO:5 is the short sequence (63 bases), and two regions of identity are shown, the first encompassing bases 4-29 (26 bases) of SEQ ID NO:5 with 92% identity to SEQ ID NO:6 and the second encompassing bases 39-59 (21 bases) of SEQ ID NO:5 with 100% identity to SEQ ID NO:6. Bases 1-3, 30-38 and 60-63 (16 bases) are not shown as having any identity with SEQ ID NO:6. Percent homology is calculated as: (26/63)(92)+(21/63)(100)+(16/63)(0)=71.3% homology. The percents of homology calculated using each of the four sets of parameters shown are listed in Table 1. Several other combinations of parameters are possible, but they are not listed for the sake of brevity. It is seen that each set of parameters resulted in a different calculated percent homology. Because the result yielding the highest percent homology is to be used, based solely on these four sets of parameters one would state that SEQ ID NOs:5 and 6 have 87.1% homology. Again it is to be noted that use of other parameters may show an even higher homology for SEQ ID NOs:5 and 6, but for brevity not all the possible results are shown.</p>
    <p>Alternatively, substantial homology or (similarity) exists when a nucleic acid or fragment thereof will hybridize to another nucleic acid (or a complementary strand thereof) under selective hybridization conditions, to a strand, or to its complement. Selectivity of hybridization exists</p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="1" colwidth="112pt" align="center"> </colspec> <colspec colname="2" colwidth="70pt" align="center"> </colspec> <colspec colname="3" colwidth="35pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="3" rowsep="1" class="description-td" colspan="3">TABLE 1</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="3" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Parameter Values</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="6"> <colspec colname="1" colwidth="28pt" align="center"> </colspec> <colspec colname="2" colwidth="28pt" align="center"> </colspec> <colspec colname="3" colwidth="21pt" align="center"> </colspec> <colspec colname="4" colwidth="35pt" align="center"> </colspec> <colspec colname="5" colwidth="70pt" align="center"> </colspec> <colspec colname="6" colwidth="35pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Mis-</td>
                <td class="description-td">Open</td>
                <td class="description-td">Extension</td>
                <td class="description-td">Regions of identity</td>
                <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Match</td>
                <td class="description-td">match</td>
                <td class="description-td">Gap</td>
                <td class="description-td">Gap</td>
                <td class="description-td">(%)</td>
                <td class="description-td">Homology</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="6" align="center" rowsep="1" class="description-td" colspan="6"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="7"> <colspec colname="1" colwidth="28pt" align="center"> </colspec> <colspec colname="2" colwidth="28pt" align="center"> </colspec> <colspec colname="3" colwidth="21pt" align="center"> </colspec> <colspec colname="4" colwidth="35pt" align="center"> </colspec> <colspec colname="5" colwidth="35pt" align="center"> </colspec> <colspec colname="6" colwidth="35pt" align="center"> </colspec> <colspec colname="7" colwidth="35pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">1</td>
                <td class="description-td">−2</td>
                <td class="description-td">5</td>
                <td class="description-td">1</td>
                <td class="description-td">4-29 of 5</td>
                <td class="description-td">39-59 of 5</td>
                <td class="description-td">71.3</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">and 5-31</td>
                <td class="description-td">and 71-91</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">of 6 (92%)</td>
                <td class="description-td">of 6</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">(100%)</td>
              </tr> <tr class="description-tr"> <td class="description-td">1</td>
                <td class="description-td">−2</td>
                <td class="description-td">2</td>
                <td class="description-td">1</td>
                <td class="description-td">4-29 of 5</td>
                <td class="description-td">33‘63 of 5</td>
                <td class="description-td">83.7</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">and 5-31</td>
                <td class="description-td">and 64-96</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">of 6 (92%)</td>
                <td class="description-td">of 6 (93%)</td>
              </tr> <tr class="description-tr"> <td class="description-td">1</td>
                <td class="description-td">−1</td>
                <td class="description-td">5</td>
                <td class="description-td">1</td>
                <td class="description-td">—</td>
                <td class="description-td">30-59 of 5</td>
                <td class="description-td">44.3</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">and 61-91</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">of 6 (93%)</td>
              </tr> <tr class="description-tr"> <td class="description-td">1</td>
                <td class="description-td">−1</td>
                <td class="description-td">2</td>
                <td class="description-td">1</td>
                <td class="description-td">4-29 of 5</td>
                <td class="description-td">30-63 of 5</td>
                <td class="description-td">87.1</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">and 5-31</td>
                <td class="description-td">and 61-96</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">of 6 (92%)</td>
                <td class="description-td">of 6 (91%)</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="7" align="center" rowsep="1" class="description-td" colspan="7"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p>when hybridization which is substantially more selective than total lack of specificity occurs. Typically, selective hybridization will occur when there is at least about 55% homology over a stretch of at least about 14 nucleotides, preferably at least about 65%, more preferably at least about 75%, and most preferably at least about 90%. See, Kanehisa, 1984. The length of homology comparison, as described, may be over longer stretches, and in certain embodiments will often be over a stretch of at least about nine nucleotides, usually at least about 20 nucleotides, more usually at least about 24 nucleotides, typically at least about 28 nucleotides, more typically at least about 32 nucleotides, and preferably at least about 36 or more nucleotides.</p>
    <p>Nucleic acid hybridization will be affected by such conditions as salt concentration, temperature, or organic solvents, in addition to the base composition, length of the complementary strands, and the number of nucleotide base mismatches between the hybridizing nucleic acids, as will be readily appreciated by those skilled in the art. Stringent temperature conditions will generally include temperatures in excess of 30° C., typically in excess of 37° C., and preferably in excess of 45° C. Stringent salt conditions will ordinarily be less than 1000 mM, typically less than 500 mM, and preferably less than 200 mM. However, the combination of parameters is much more important than the measure of any single parameter. The stringency conditions are dependent on the length of the nucleic acid and the base composition of the nucleic acid and can be determined by techniques well known in the art. See, e.g., Wetmur and Davidson, 1968.</p>
    <p>Probe sequences may also hybridize specifically to duplex DNA under certain conditions to form triplex or other higher order DNA complexes. The preparation of such probes and suitable hybridization conditions are well known in the art.</p>
    <p>The terms “substantial homology” or “substantial identity”, when referring to polypeptides, indicate that the polypeptide or protein in question exhibits at least about 30% identity with an entire naturally-occurring protein or a portion thereof, usually at least about 70% identity, more usually at least about 80% identity, preferably at least about 90% identity, and more preferably at least about 95% identity.</p>
    <p>Homology, for polypeptides, is typically measured using sequence analysis software. See, e.g., the Sequence Analysis Software Package of the Genetics Computer Group, University of Wisconsin Biotechnology Center, 910 University Avenue, Madison, Wis. 53705. Protein analysis software matches similar sequences using measures of homology assigned to various substitutions, deletions and other modifications. Conservative substitutions typically include substitutions within the following groups: glycine, alanine; valine, isoleucine, leucine; aspartic acid, glutamic acid; asparagine, glutamine; serine, threonine; lysine, arginine; and phenylalanine, tyrosine.</p>
    <p>“Substantially similar function” refers to the function of a modified nucleic acid or a modified protein, with reference to the wild-type KVLQT1 or KCNE1 nucleic acid or wild-type KVLQT1 or KCNE1 polypeptide. The modified polypeptide will be substantially homologous to the wild-type KVLQT1 or KCNE1 polypeptide and will have substantially the same function. The modified polypeptide may have an altered amino acid sequence and/or may contain modified amino acids. In addition to the similarity of function, the modified polypeptide may have other useful properties, such as a longer half-life. The similarity of function (activity) of the modified polypeptide may be substantially the same as the activity of the wild-type KVLQT1 or KCNE1 polypeptide. Alternatively, the similarity of function (activity) of the modified polypeptide may be higher than the activity of the wild-type KVLQT1 or KCNE1 polypeptide. The modified polypeptide is synthesized using conventional techniques, or is encoded by a modified nucleic acid and produced using conventional techniques. The modified nucleic acid is prepared by conventional techniques. A nucleic acid with a function substantially similar to the wild-type KVLQT1 or KCNE1 gene function produces the modified protein described above.</p>
    <p>A polypeptide “fragment”, “portion” or “segment” is a stretch of amino acid residues of at least about five to seven contiguous amino acids, often at least about seven to nine contiguous amino acids, typically at least about nine to 13 contiguous amino acids and, most preferably, at least about 20 to 30 or more contiguous amino acids.</p>
    <p>The polypeptides of the present invention, if soluble, may be coupled to a solid-phase support, e.g., nitrocellulose, nylon, column packing materials (e.g., Sepharose beads), magnetic beads, glass wool, plastic, metal, polymer gels, cells, or other substrates. Such supports may take the form, for example, of beads, wells, dipsticks, or membranes.</p>
    <p>“Target region” refers to a region of the nucleic acid which is amplified and/or detected. The term “target sequence” refers to a sequence with which a probe or primer will form a stable hybrid under desired conditions.</p>
    <p>The practice of the present invention employs, unless otherwise indicated, conventional techniques of chemistry, molecular biology, microbiology, recombinant DNA, genetics, and immunology. See, e.g., Maniatis et al., 1982; Sambrook et al., 1989; Ausubel et al., 1992; Glover, 1985; Anand, 1992; Guthrie and Fink, 1991. A general discussion of techniques and materials for human gene mapping, including mapping of human chromosome 1, is provided, e.g., in White and Lalouel, 1988.</p>
    <p>Preparation of Recombinant or Chemically Synthesized Nucleic Acids: Vectors, Transformation, Host Cells</p>
    <p>Large amounts of the polynucleotides of the present invention may be produced by replication in a suitable host cell. Natural or synthetic polynucleotide fragments coding for a desired fragment will be incorporated into recombinant polynucleotide constructs, usually DNA constructs, capable of introduction into and replication in a prokaryotic or eukaryotic cell. Usually the polynucleotide constructs will be suitable for replication in a unicellular host, such as yeast or bacteria, but may also be intended for introduction to (with and without integration within the genome) cultured mammalian or plant or other eukaryotic cell lines. The purification of nucleic acids produced by the methods of the present invention are described, e.g., in Sambrook et al., 1989 or Ausubel et al., 1992.</p>
    <p>The polynucleotides of the present invention may also be produced by chemical synthesis, e.g., by the phosphoramidite method described by Beaucage and Caruthers (1981) or the triester method according to Matteucci and Caruthers (1981) and may be performed on commercial, automated oligonucleotide synthesizers. A double-stranded fragment may be obtained from the single-stranded product of chemical synthesis either by synthesizing the complementary strand and annealing the strand together under appropriate conditions or by adding the complementary strand using DNA polymerase with an appropriate primer sequence.</p>
    <p>Polynucleotide constructs prepared for introduction into a prokaryotic or eukaryotic host may comprise a replication system recognized by the host, including the intended polynucleotide fragment encoding the desired polypeptide, and will preferably also include transcription and translational initiation regulatory sequences operably linked to the polypeptide encoding segment. Expression vectors may include, for example, an origin of replication or autonomously replicating sequence (ARS) and expression control sequences, a promoter, an enhancer and necessary processing information sites, such as ribosome-binding sites, RNA splice sites, polyadenylation sites, transcriptional terminator sequences, and mRNA stabilizing sequences. Such vectors may be prepared by means of standard recombinant techniques well known in the art and discussed, for example, in Sambrook et al., 1989 or Ausubel et al., 1992.</p>
    <p>An appropriate promoter and other necessary vector sequences will be selected so as to be functional in the host, and may include, when appropriate, those naturally associated with the KVLQT1 or KCNE1 gene. Examples of workable combinations of cell lines and expression vectors are described in Sambrook et al., 1989 or Ausubel et al., 1992; see also, e.g., Metzger et al., 1988. Many useful vectors are known in the art and may be obtained from such vendors as Stratagene, New England Biolabs, Promega Biotech, and others. Promoters such as the trp, lac and phage promoters, tRNA promoters and glycolytic enzyme promoters may be used in prokaryotic hosts. Useful yeast promoters include promoter regions for metallothionein, 3-phosphoglycerate kinase or other glycolytic enzymes such as enolase or glyceraldehyde-3-phosphate dehydrogenase, enzymes responsible for maltose and galactose utilization, and others. Vectors and promoters suitable for use in yeast expression are further described in Hitzeman et al., EP 73,675A. Appropriate non-native mammalian promoters might include the early and late promoters from SV40 (Fiers et al., 1978) or promoters derived from murine Molony leukemia virus, mouse tumor virus, avian sarcoma viruses, adenovirus II, bovine papilloma virus or polyoma. Insect promoters may be derived from baculovirus. In addition, the construct may be joined to an amplifiable gene (e.g., DHFR) so that multiple copies of the gene may be made. For appropriate enhancer and other expression control sequences, see also <i>Enhancers and Eukaryotic Gene Expression</i>, Cold Spring Harbor Press, Cold Spring Harbor, N.Y. (1983). See also, e.g., U.S. Pat. Nos. 5,691,198; 5,735,500; 5,747,469 and 5,436,146.</p>
    <p>While such expression vectors may replicate autonomously, they may also replicate by being inserted into the genome of the host cell, by methods well known in the art.</p>
    <p>Expression and cloning vectors will likely contain a selectable marker, a gene encoding a protein necessary for survival or growth of a host cell transformed with the vector. The presence of this gene ensures growth of only those host cells which express the inserts. Typical selection genes encode proteins that a) confer resistance to antibiotics or other toxic substances, e.g. ampicillin, neomycin, methotrexate, etc., b) complement auxotrophic deficiencies, or c) supply critical nutrients not available from complex media, e.g., the gene encoding D-alanine racemase for Bacilli. The choice of the proper selectable marker will depend on the host cell, and appropriate markers for different hosts are well known in the art.</p>
    <p>The vectors containing the nucleic acids of interest can be transcribed in vitro, and the resulting RNA introduced into the host cell by well-known methods, e.g., by injection (see, Kubo et al., 1988), or the vectors can be introduced directly into host cells by methods well known in the art, which vary depending on the type of cellular host, including electroporation; transfection employing calcium chloride, rubidium chloride calcium phosphate, DEAE-dextran, or other substances; microprojectile bombardment; lipofection; infection (where the vector is an infectious agent, such as a retroviral genome); and other methods. See generally, Sambrook et al., 1989 and Ausubel et al., 1992. The introduction of the polynucleotides into the host cell by any method known in the art, including, inter alia, those described above, will be referred to herein as “transformation.” The cells into which have been introduced nucleic acids described above are meant to also include the progeny of such cells.</p>
    <p>Large quantities of the nucleic acids and polypeptides of the present invention may be prepared by expressing the KVLQT1 or KCNE1 nucleic acid or portions thereof in vectors or other expression vehicles in compatible prokaryotic or eukaryotic host cells. The most commonly used prokaryotic hosts are strains of <i>Escherichia coli</i>, although other prokaryotes, such as <i>Bacillus subtilis </i>or Pseudomonas may also be used.</p>
    <p>Mammalian or other eukaryotic host cells, such as those of yeast, filamentous fungi, plant, insect, or amphibian or avian species, may also be useful for production of the proteins of the present invention. Propagation of mammalian cells in culture is per se well known. See, Jakoby and Pastan (eds.) (1979). Examples of commonly used mammalian host cell lines are VERO and HeLa cells, Chinese hamster ovary (CHO) cells, and W138, BHK, and COS cell lines, although it will be appreciated by the skilled practitioner that other cell lines may be appropriate, e.g., to provide higher expression, desirable glycosylation patterns, or other features. An example of a commonly used insect cell line is SF9.</p>
    <p>Clones are selected by using markers depending on the mode of the vector construction. The marker may be on the same or a different DNA molecule, preferably the same DNA molecule. In prokaryotic hosts, the transfornant may be selected, e.g., by resistance to ampicillin, tetracycline or other antibiotics. Production of a particular product based on temperature sensitivity may also serve as an appropriate marker.</p>
    <p>Prokaryotic or eukaryotic cells transformed with the polynucleotides of the present invention will be useful not only for the production of the nucleic acids and polypeptides of the present invention, but also, for example, in studying the characteristics of KVLQT1 or KCNE1 polypeptides.</p>
    <p>The probes and primers based on the KVLQT1 or KCNE1 gene sequence disclosed herein are used to identify homologous KVLQT1 or KCNE1 gene sequences and proteins in other species. These gene sequences and proteins are used in the diagnostic/prognostic, therapeutic and drug screening methods described herein for the species from which they have been isolated.</p>
    <p>Methods of Use: Drug Screening</p>
    <p>The invention is particularly useful for screening compounds by using KVLQT1 and KCNE1 proteins in transformed cells, transfected oocytes or transgenic animals. Since mutations in either the KVLQT1 or KCNE1 protein can alter the functioning of the cardiac I<sub>Ks </sub>potassium channel, candidate drugs are screened for effects on the channel using cells containing either a normal KVLQT1 or KCNE1 protein and a mutant KCNE1 or KVLQT1 protein, respectively, or a mutant KVLQT1 and a mutant KCNE1 protein. The drug is added to the cells in culture or administered to a transgenic animal and the effect on the induced current of the I<sub>Ks </sub>potassium channel is compared to the induced current of a cell or animal containing the wild-type KVLQT1 and minK. Drug candidates which alter the induced current to a more normal level are useful for treating or preventing LQT.</p>
    <p>This invention is particularly useful for screening compounds by using the KVLQT1 or KCNE1 polypeptide or binding fragment thereof in any of a variety of drug screening techniques.</p>
    <p>The KVLQT1 or KCNE1 polypeptide or fragment employed in such a test may either be free in solution, affixed to a solid support, or borne on a cell surface. One method of drug screening utilizes eucaryotic or procaryotic host cells which are stably transformed with recombinant polynucleotides expressing the polypeptide or fragment, preferably in competitive binding assays. Such cells, either in viable or fixed form, can be used for standard binding assays. One may measure, for example, for the formation of complexes between a KVLQT1 or KCNE1 polypeptide or fragment and the agent being tested, or examine the degree to which the formation of a complex between a KVLQT1 or KCNE1 polypeptide or fragment and a known ligand is interfered with by the agent being tested.</p>
    <p>Thus, the present invention provides methods of screening for drugs comprising contacting such an agent with a KVLQT1 or KCNE1 polypeptide or fragment thereof and assaying (i) for the presence of a complex between the agent and the KVLQT1 or KCNE1 polypeptide or fragment, or (ii) for the presence of a complex between the KVLQT1 or KCNE1 polypeptide or fragment and a ligand, by methods well known in the art. In such competitive binding assays the KVLQT1 or KCNE1 polypeptide or fragment is typically labeled. Free KVLQT1 or KCNE1 polypeptide or fragment is separated from that present in a protein:protein complex, and the amount of free (i.e., uncomplexed) label is a measure of the binding of the agent being tested to KVLQT1 or KCNE1 or its interference with KVLQT1 or KCNE1:ligand binding, respectively. One may also measure the amount of bound, rather than free, KVLQT1 or KCNE1. It is also possible to label the ligand rather than the KVLQT1 or KCNE1 and to measure the amount of ligand binding to KVLQT1 or KCNE1 in the presence and in the absence of the drug being tested.</p>
    <p>Another technique for drug screening provides high throughput screening for compounds having suitable binding affinity to the KVLQT1 or KCNE1 polypeptides and is described in detail in Geysen (published PCT application WO 84/03564). Briefly stated, large numbers of different small peptide test compounds are synthesized on a solid substrate, such as plastic pins or some other surface. The peptide test compounds are reacted with KVLQT1 or KCNE1 polypeptide and washed. Bound KVLQT1 or KCNE1 polypeptide is then detected by methods well known in the art.</p>
    <p>Purified KVLQT1 or KCNE1 can be coated directly onto plates for use in the aforementioned drug screening techniques. However, non-neutralizing antibodies to the polypeptide can be used to capture antibodies to immobilize the KVLQT1 or KCNE1 polypeptide on the solid phase.</p>
    <p>This invention also contemplates the use of competitive drug screening assays in which neutralizing antibodies capable of specifically binding the KVLQT1 or KCNE1 polypeptide compete with a test compound for binding to the KVLQT1 or KCNE1 polypeptide or fragments thereof. In this manner, the antibodies can be used to detect the presence of any peptide which shares one or more antigenic determinants of the KVLQT1 or KCNE1 polypeptide.</p>
    <p>The above screening methods are not limited to assays employing only KVLQT1 or KCNE1 but are also applicable to studying KVLQT1- or KCNE1-protein complexes. The effect of drugs on the activity of this complex is analyzed.</p>
    <p>In accordance with these methods, the following assays are examples of assays which can be used for screening for drug candidates.</p>
    <p>A mutant KVLQT1 or KCNE1 (per se or as part of a fusion protein) is mixed with a wild-type protein (per se or as part of a fusion protein) to which wild-type KVLQT1 or KCNE1 binds. This mixing is performed in both the presence of a drug and the absence of the drug, and the amount of binding of the mutant KVLQT1 or KCNE1 with the wild-type protein is measured. If the amount of the binding is more in the presence of said drug than in the absence of said drug, the drug is a drug candidate for treating LQT resulting from a mutation in KVLQT1 or KCNE1.</p>
    <p>A wild-type KVLQTb <b>1</b> or KCNE1 (per se or as part of a fusion protein) is mixed with a wild-type protein (per se or as part of a fusion protein) to which wild-type KVLQT1 or KCNE1 binds. This mixing is performed in both the presence of a drug and the absence of the drug, and the amount of binding of the wild-type KVLQT1 or KCNE1 with the wild-type protein is measured. If the amount of the binding is more in the presence of said drug than in the absence of said drug, the drug is a drug candidate for treating LQT resulting from a mutation in KVLQT1 or KCNE1.</p>
    <p>A mutant protein, which as a wild-type protein binds to KVLQT1 or KCNE1 (per se or as part of a fusion protein) is mixed with a wild-type KVLQT1 or KCNE1 (per se or as part of a fusion protein). This mixing is performed in both the presence of a drug and the absence of the drug, and the amount of binding of the mutant protein with the wild-type KVLQT1 or KCNE1 is measured. If the amount of the binding is more in the presence of said drug than in the absence of said drug, the drug is a drug candidate for treating LQT resulting from a mutation in the gene encoding the protein.</p>
    <p>The polypeptide of the invention may also be used for screening compounds developed as a result of combinatorial library technology. Combinatorial library technology provides an efficient way of testing a potential vast number of different substances for ability to modulate activity of a polypeptide. Such libraries and their use are known in the art. The use of peptide libraries is preferred. See, for example, WO 97/02048.</p>
    <p>Briefly, a method of screening for a substance which modulates activity of a polypeptide may include contacting one or more test substances with the polypeptide in a suitable reaction medium, testing the activity of the treated polypeptide and comparing that activity with the activity of the polypeptide in comparable reaction medium untreated with the test substance or substances. A difference in activity between the treated and untreated polypeptides is indicative of a modulating effect of the relevant test substance or substances.</p>
    <p>Prior to or as well as being screened for modulation of activity, test substances may be screened for ability to interact with the polypeptide, e.g., in a yeast two-hybrid system (e.g., Bartel et al., 1993; Fields and Song, 1989; Chevray and Nathans, 1992; Lee et al., 1995). This system may be used as a coarse screen prior to testing a substance for actual ability to modulate activity of the polypeptide. Alternatively, the screen could be used to screen test substances for binding to an KVLQT1 or KCNE1 specific binding partner, or to find mimetics of the KVLQT1 or KCNE1 polypeptide.</p>
    <p>Following identification of a substance which modulates or affects polypeptide activity, the substance may be investigated further. Furthermore, it may be manufactured and/or used in preparation, i.e., manufacture or formulation, or a composition such as a medicament, pharmaceutical composition or drug. These may be administered to individuals.</p>
    <p>Thus, the present invention extends in various aspects not only to a substance identified using a nucleic acid molecule as a modulator of polypeptide activity, in accordance with what is disclosed herein, but also a pharmaceutical composition, medicament, drug or other composition comprising such a substance, a method comprising administration of such a composition comprising such a substance, a method comprising administration of such a composition to a patient, e.g., for treatment (which may include preventative treatment) of LQT, use of such a substance in the manufacture of a composition for administration, e.g., for treatment of LQT, and a method of making a pharmaceutical composition comprising admixing such a substance with a pharmaceutically acceptable excipient, vehicle or carrier, and optionally other ingredients.</p>
    <p>A substance identified as a modulator of polypeptide function may be peptide or non-peptide in nature. Non-peptide “small molecules” are often preferred for many in vivo pharmaceutical uses. Accordingly, a mirmetic or mimic of the substance (particularly if a peptide) may be designed for pharmaceutical use.</p>
    <p>The designing of mimetics to a known pharmaceutically active compound is a known approach to the development of pharmaceuticals based on a “lead” compound. This might be desirable where the active compound is difficult or expensive to synthesize or where it is unsuitable for a particular method of administration, e.g., pure peptides are unsuitable active agents for oral compositions as they tend to be quickly degraded by proteases in the alimentary canal. Mimetic design, synthesis and testing is generally used to avoid randomly screening large numbers of molecules for a target property.</p>
    <p>There are several steps commonly taken in the design of a mimetic from a compound having a given target property. First, the particular parts of the compound that are critical and/or important in determining the target property are determined. In the case of a peptide, this can be done by systematically varying the amino acid residues in the peptide, e.g., by substituting each residue in turn. Alanine scans of peptide are commonly used to refine such peptide motifs. These parts or residues constituting the active region of the compound are known as its “pharmacophore”.</p>
    <p>Once the pharmacophore has been found, its structure is modeled according to its physical properties, e.g., stereochemistry, bonding, size and/or charge, using data from a range of sources, e.g., spectroscopic techniques, x-ray diffraction data and NMR. Computational analysis, similarity mapping (which models the charge and/or volume of a pharmacophore, rather than the bonding between atoms) and other techniques can be used in this modeling process.</p>
    <p>In a variant of this approach, the three-dimensional structure of the ligand and its binding partner are modeled. This can be especially useful where the ligand and/or binding partner change conformation on binding, allowing the model to take account of this in the design of the mimetic.</p>
    <p>A template molecule is then selected onto which chemical groups which mimic the pharmacophore can be grafted. The template molecule and the chemical groups grafted onto it can conveniently be selected so that the mimetic is easy to synthesize, is likely to be pharmacologically acceptable, and does not degrade in vivo, while retaining the biological activity of the lead compound. Alternatively, where the mimetic is peptide-based, further stability can be achieved by cyclizing the peptide, increasing its rigidity. The mimetic or mimetics found by this approach can then be screened to see whether they have the target property, or to what extent they exhibit it. Further optimization or modification can then be carried out to arrive at one or more final mimetics for in vivo or clinical testing.</p>
    <p>Methods of Use: Nucleic Acid Diagnosis and Diagnostic Kits</p>
    <p>In order to detect the presence of a KVLQT1 or KCNE1 allele predisposing an individual to LQT, a biological sample such as blood is prepared and analyzed for the presence or absence of susceptibility alleles of KVLQT1 or KCNE1. In order to detect the presence of LQT or as a prognostic indicator, a biological sample is prepared and analyzed for the presence or absence of mutant alleles of KVLQT1 or KCNE1. Results of these tests and interpretive information are returned to the health care provider for communication to the tested individual. Such diagnoses may be performed by diagnostic laboratories, or, alternatively, diagnostic kits are manufactured and sold to health care providers or to private individuals for self-diagnosis.</p>
    <p>Initially, the screening method involves amplification of the relevant KVLQT1 or KCNE1 sequences. In another preferred embodiment of the invention, the screening method involves a non-PCR based strategy. Such screening methods include two-step label amplification methodologies that are well known in the art. Both PCR and non-PCR based screening strategies can detect target sequences with a high level of sensitivity.</p>
    <p>The most popular method used today is target amplification. Here, the target nucleic acid sequence is amplified with polymerases. One particularly preferred method using polymerase-driven amplification is the polymerase chain reaction (PCR). The polymerase chain reaction and other polymerase-driven amplification assays can achieve over a million-fold increase in copy number through the use of polymerase-driven amplification cycles. Once amplified, the resulting nucleic acid can be sequenced or used as a substrate for DNA probes.</p>
    <p>When the probes are used to detect the presence of the target sequences the biological sample to be analyzed, such as blood or serum, may be treated, if desired, to extract the nucleic acids. The sample nucleic acid may be prepared in various ways to facilitate detection of the target sequence, e.g. denaturation, restriction digestion, electrophoresis or dot blotting. The targeted region of the analyte nucleic acid usually must be at least partially single-stranded to form hybrids with the targeting sequence of the probe. If the sequence is naturally single-stranded, denaturation will not be required. However, if the sequence is double-stranded, the sequence will probably need to be denatured. Denaturation can be carried out by various techniques known in the art.</p>
    <p>Analyte nucleic acid and probe are incubated under conditions which promote stable hybrid formation of the target sequence in the probe with the putative targeted sequence in the analyte. The region of the probes which is used to bind to the analyte can be made completely complementary to the targeted region of human chromosome 11 for KVLQT1 or chromosome 21 for KCNE1. Therefore, high stringency conditions are desirable in order to prevent false positives. However, conditions of high stringency are used only if the probes are complementary to regions of the chromosome which are unique in the genome. The stringency of hybridization is determined by a number of factors during hybridization and during the washing procedure, including temperature, ionic strength, base composition, probe length, and concentration of formamide. These factors are outlined in, for example, Maniatis et al., 1982 and Sambrook et al., 1989. Under certain circumstances, the formation of higher order hybrids, such as triplexes, quadraplexes, etc., may be desired to provide the means of detecting target sequences.</p>
    <p>Detection, if any, of the resulting hybrid is usually accomplished by the use of labeled probes. Alternatively, the probe may be unlabeled, but may be detectable by specific binding with a ligand which is labeled, either directly or indirectly. Suitable labels, and methods for labeling probes and ligands are known in the art, and include, for example, radioactive labels which may be incorporated by known methods (e.g., nick translation, random priming or kinasing), biotin, fluorescent groups, chemiluminescent groups (e.g., dioxetanes, particularly triggered dioxetanes), enzymes, antibodies, gold nanoparticles and the like. Variations of this basic scheme are known in the art, and include those variations that facilitate separation of the hybrids to be detected from extraneous materials and/or that amplify the signal from the labeled moiety. A number of these variations are reviewed in, e.g., Matthews and Kricka, 1988; Landegren et al., 1988; Mifflin, 1989; U.S. Pat. No. 4,868,105; and in EPO Publication No. 225,807.</p>
    <p>As noted above, non-PCR based screening assays are also contemplated in this invention. This procedure hybridizes a nucleic acid probe (or an analog such as a methyl phosphonate backbone replacing the normal phosphodiester), to the low level DNA target. This probe may have an enzyme covalently linked to the probe, such that the covalent linkage does not interfere with the specificity of the hybridization. This enzyme-probe-conjugate-target nucleic acid complex can then be isolated away from the free probe enzyme conjugate and a substrate is added for enzyme detection. Enzymatic activity is observed as a change in color development or luminescent output resulting in a 10<sup>3</sup>-10<sup>6 </sup>increase in sensitivity. For an example relating to the preparation of oligodeoxynucleotide-alkaline phosphatase conjugates and their use as hybridization probes, see Jablonski et al. (1986).</p>
    <p>Two-step label amplification methodologies are known in the art. These assays work on the principle that a small ligand (such as digoxigenin, biotin, or the like) is attached to a nucleic acid probe capable of specifically binding KVLQT1. Allele specific probes are also contemplated within the scope of this example and exemplary allele specific probes include probes encompassing the predisposing mutations of this patent application.</p>
    <p>In one example, the small ligand attached to the nucleic acid probe is specifically recognized by an antibody-enzyme conjugate. In one embodiment of this example, digoxigenin is attached to the nucleic acid probe. Hybridization is detected by an antibody-alkaline phosphatase conjugate which turns over a chemiluminescent substrate. For methods for labeling nucleic acid probes according to this embodiment see Martin et al., 1990. In a second example, the small ligand is recognized by a second ligand-enzyme conjugate that is capable of specifically complexing to the first ligand. A well known embodiment of this example is the biotin-avidin type of interactions. For methods for labeling nucleic acid probes and their use in biotin-avidin based assays see Rigby et al., 1977 and Nguyen et al., 1992.</p>
    <p>It is also contemplated within the scope of this invention that the nucleic acid probe assays of this invention will employ a cocktail of nucleic acid probes capable of detecting KVLQT1 or KCNE1. Thus, in one example to detect the presence of KVLQT1 or KCNE1 in a cell sample, more than one probe complementary to the gene is employed and in particular the number of different probes is alternatively two, three, or five different nucleic acid probe sequences. In another example, to detect the presence of mutations in the KVLQT1 or KCNE1 gene sequence in a patient, more than one probe complementary to these genes is employed where the cocktail includes probes capable of binding to the allele-specific mutations identified in populations of patients with alterations in KVLQT1 or KCNE1. In this embodiment, any number of probes can be used, and will preferably include probes corresponding to the major gene mutations identified as predisposing an individual to LQT.</p>
    <p>Methods of Use: Peptide Diagnosis and Diagnostic Kits</p>
    <p>The presence of LQT can also be detected on the basis of the alteration of wild-type KVLQT1 or KCNE1 polypeptide. Such alterations can be determined by sequence analysis in accordance with conventional techniques. More preferably, antibodies (polyclonal or monoclonal) are used to detect differences in, or the absence of KVLQT1 or KCNE1 peptides. Techniques for raising and purifying antibodies are well known in the art and any such techniques may be chosen to achieve the preparations claimed in this invention. In a preferred embodiment of the invention, antibodies will immunoprecipitate KVLQT1 or KCNE1 proteins from solution as well as react with these proteins on Western or immunoblots of polyacrylamide gels. In another preferred embodiment, antibodies will detect KVLQT1 or KCNE1 proteins in paraffin or frozen tissue sections, using immunocytochemical techniques.</p>
    <p>Preferred embodiments relating to methods for detecting KVLQT1 or KCNE1 or their mutations include enzyme linked immunosorbent assays (ELISA), radioimmunoassays (RIA), immunoradiometric assays (IRMA) and immunoenzymatic assays (IEMA), including sandwich assays using monoclonal and/or polyclonal antibodies. Exemplary sandwich assays are described by David et al., in U.S. Pat. Nos. 4,376,110 and 4,486,530, hereby incorporated by reference.</p>
    <p>Methods of Use: Rational Drug Design</p>
    <p>The goal of rational drug design is to produce structural analogs of biologically active polypeptides of interest or of small molecules with which they interact (e.g., agonists, antagonists, inhibitors) in order to fashion drugs which are, for example, more active or stable forms of the polypeptide, or which, e.g., enhance or interfere with the function of a polypeptide in vivo. See, e.g., Hodgson, 1991. In one approach, one first determines the three-dimensional structure of a protein of interest (e.g., KVLQT1 or KCNE1 polypeptide) by x-ray crystallography, by computer modeling or most typically, by a combination of approaches. Less often, useful information regarding the structure of a polypeptide may be gained by modeling based on the structure of homologous proteins. An example of rational drug design is the development of HIV protease inhibitors (Erickson et al., 1990). In addition, peptides (e.g., KVLQT1 or KCNE1 polypeptide) are analyzed by ah alanine scan (Wells, 1991). Inthis technique, an amino acid residue is replaced by Ala, and its effect on the peptide's activity is determined. Each of the amino acid residues of the peptide is analyzed in this manner to determine the important regions of the peptide.</p>
    <p>It is also possible to isolate a target-specific antibody, selected by a functional assay, and then to solve its crystal structure. In principle, this approach yields a pharmacore upon which subsequent drug design can be based. It is possible to bypass protein crystallography altogether by generating anti-idiotypic antibodies (anti-ids) to a functional, pharmacologically active antibody. As a mirror image of a mirror image, the binding site of the anti-ids would be expected to be an analog of the original receptor. The anti-id could then be used to identify and isolate peptides from banks of chemically or biologically produced banks of peptides. Selected peptides would then act as the pharmacore.</p>
    <p>Thus, one may design drugs which have, e.g., improved KVLQT1 or KCNE1 polypeptide activity or stability or which act as inhibitors, agonists, antagonists, etc. of KVLQT1 or KCNE1 polypeptide activity. By virtue of the availability of cloned KVLQT1 or KCNE1 sequences, sufficient amounts of the KVLQT1 or KCNE1 polypeptide may be made available to perform such analytical studies as x-ray crystallography. In addition, the knowledge of the KVLQT1 or KCNE1 protein sequences provided herein will guide those employing computer modeling techniques in place of, or in addition to x-ray crystallography.</p>
    <p>Methods of Use: Gene Therapy</p>
    <p>According to the present invention, a method is also provided of supplying wild-type KVLQT1 or KCNE1 function to a cell which carries a mutant KVLQT1 or KCNE1 allele, respectively. Supplying such a function should allow normal functioning of the recipient cells. The wild-type gene or a part of the gene may be introduced into the cell in a vector such that the gene remains extrachromosomal. In such a situation, the gene will be expressed by the cell from the extrachromosomal location. More preferred is the situation where the wild-type gene or a part thereof is introduced into the mutant cell in such a way that it recombines with the endogenous mutant gene present in the cell. Such recombination requires a double recombination event which results in the correction of the gene mutation. Vectors for introduction of genes both for recombination and for extrachromosomal maintenance are known in the art, and any suitable vector may be used. Methods for introducing DNA into cells such as electroporation, calcium phosphate co-precipitation and viral transduction are known in the art, and the choice of method is within the competence of the practitioner.</p>
    <p>As generally discussed above, the KVLQT1 or KCNE1 gene or fragment, where applicable, may be employed in gene therapy methods in order to increase the amount of the expression products of such gene in cells. It may also be useful to increase the level of expression of a given LQT gene even in those heart cells in which the mutant gene is expressed at a “normal” level, but the gene product is not fully functional.</p>
    <p>Gene therapy would be carried out according to generally accepted methods, for example, as described by Friedman (1991) or Culver (1996). Cells from apatient would be first analyzed by the diagnostic methods described above, to ascertain the production of KVLQT1 or KCNE1 polypeptide in the cells. A virus or plasmid vector (see further details below), containing a copy of the KVLQT1 or KCNE1 gene linked to expression control elements and capable of replicating inside the cells, is prepared. The vector may be capable of replicating inside the cells. Alternatively, the vector may be replication deficient and is replicated in helper cells for use in gene therapy. Suitable vectors are known, such as disclosed in U.S. Pat. No. 5,252,479 and PCT published application WO 93/07282 and U.S. Pat. Nos. 5,691,198; 5,747,469; 5,436,146 and 5,753,500. The vector is then injected into the patient. If the transfected gene is not permanently incorporated into the genome of each of the targeted cells, the treatment may have to be repeated periodically.</p>
    <p>Gene transfer systems known in the art may be useful in the practice of the gene therapy methods of the present invention. These include viral and nonviral transfer methods. A number of viruses have been used as gene transfer vectors or as the basis for repairing gene transfer vectors, including papovaviruses (e.g., SV40, Madzak et al., 1992), adenovirus (Berkner, 1992; Berkner et al., 1988; Gorziglia and Kapikian, 1992; Quantin et al., 1992; Rosenfeld et al., 1992; Wilkinson and Akrigg, 1992; Stratford-Perricaudet et al., 1990; Schneider et al., 1998), vaccinia virus (Moss, 1992; Moss, 1996), adeno-associated virus (Muzyczka, 1992; Ohi et al., 1990; Russell and Hirata, 1998), herpesviruses including HSV and EBV (Margolskee, 1992; Johnson et al., 1992; Fink et al., 1992; Breakefield and Geller, 1987; Freese et al., 1990; Fink et al., 1996), lentiviruses (Naldini et al., 1996), Sindbis and Semliki Forest virus (Berglund et al., 1993), and retroviruses of avian (Bandyopadhyay and Temin, 1984; Petropoulos et al., 1992), murine (Miller, 1992; Miller et al., 1985; Sorge et al., 1984; Mann and Baltimore, 1985; Miller et al., 1988), and human origin (Shimada et al., 1991; Helseth et al., 1990; Page et al., 1990; Buchschacher and Panganiban, 1992). Most human gene therapy protocols have been based on disabled murine retroviruses, although adenovirus and adeno-associated virus are also being used.</p>
    <p>Nonviral gene transfer methods known in the art include chemical techniques such as calcium phosphate coprecipitation (Graham and van der Eb, 1973; Pellicer et al., 1980); mechanical techniques, for example microinjection (Anderson et al., 1980; Gordon et al., 1980; Brinster et al., 1981; Costantini and Lacy, 1981); membrane fusion-mediated transfer via liposomes (Felgner et al., 1987; Wang and Huang, 1989; Kaneda et al., 1989; Stewart et al., 1992; Nabel et al., 1990; Lim et al., 1991); and direct DNA uptake and receptor-mediated DNA transfer (Wolff et al., 1990; Wu et al., 1991; Zenke et al., 1990; Wu et al., 1989; Wolffet al., 1991; Wagner et al., 1990; Wagner et al., 1991; Cotten et al., 1990; Curiel et al., 1992; Curiel et al., 1991). Viral-mediated gene transfer can be combined with direct in vivo gene transfer using liposome delivery, allowing one to direct the viral vectors to the tumor cells and not into the surrounding nondividing cells. Alternatively, the retroviral vector producer cell line can be injected into tumors (Culver et al., 1992). Injection of producer cells would then provide a continuous source of vector particles. This technique has been approved for use in humans with inoperable brain tumors.</p>
    <p>In an approach which combines biological and physical gene transfer methods, plasmid DNA of any size is combined with a polylysine-conjugated antibody specific to the adenovirus hexon protein, and the resulting complex is bound to an adenovirus vector. The trimolecular complex is then used to infect cells. The adenovirus vector permits efficient binding, internalization, and degradation of the endosome before the coupled DNA is damaged. For other techniques for the delivery of adenovirus based vectors see Schneider et al. (1998) and U.S. Pat. Nos. 5,691,198; 5,747,469; 5,436,146 and 5,753,500.</p>
    <p>Liposome/DNA complexes have been shown to be capable of mediating direct in vivo gene transfer. While in standard liposome preparations the gene transfer process is nonspecific, localized in vivo uptake and expression have been reported in tumor deposits, for example, following direct in situ administration (Nabel, 1992).</p>
    <p>Expression vectors in the context of gene therapy are meant to include those constructs containing sequences sufficient to express a polynucleotide that has been cloned therein. In viral expression vectors, the construct contains viral sequences sufficient to support packaging of the construct. If the polynucleotide encodes KVLQT1 or KCNE1, expression will produce KVLQT1 or KCNE1. If the polynucleotide encodes an antisense polynucleotide or a ribozyme, expression will produce the antisense polynucleotide or ribozyme. Thus in this context, expression does not require that a protein product be synthesized. In addition to the polynucleotide cloned into the expression vector, the vector also contains a promoter functional in eukaryotic cells. The cloned polynucleotide sequence is under control of this promoter. Suitable eukaryotic promoters include those described above. The expression vector may also include sequences, such as selectable markers and other sequences described herein.</p>
    <p>Gene transfer techniques which target DNA directly to heart tissue is preferred. Receptor-mediated gene transfer, for example, is accomplished by the conjugation of DNA (usually in the form of covalently closed supercoiled plasmid) to a protein ligand via polylysine. Ligands are chosen on the basis of the presence of the corresponding ligand receptors on the cell surface of the target cell/tissue type. These ligand-DNA conjugates can be injected directly into the blood if desired and are directed to the target tissue where receptor binding and internalization of the DNA-protein complex occurs. To overcome the problem of intracellular destruction of DNA, coinfection with adenovirus can be included to disrupt endosome function.</p>
    <p>The therapy is as follows: patients who carry a KVLQT1 or KCNE1 susceptibility allele are treated with a gene delivery vehicle such that some or all of their heart precursor cells receive at least one additional copy of a functional normal KVLQT1 or KCNE1 allele. In this step, the treated individuals have reduced risk of LQT to the extent that the effect of the susceptible allele has been countered by the presence of the normal allele.</p>
    <p>Methods of Use: Peptide Therapy</p>
    <p>Peptides which have KVLQT1 or KCNE1 activity can be supplied to cells which carry a mutant or missing KVLQT1 or KCNE1 allele. Protein can be produced by expression of the cDNA sequence in bacteria, for example, using known expression vectors. Alternatively, KVLQT1 or KCNE1 polypeptide can be extracted from KVLQT1- or KCNE1-producing mammalian cells. In addition, the techniques of synthetic chemistry can be employed to synthesize KVLQT1 or KCNE1 protein. Any of such techniques can provide the preparation of the present invention which comprises the KVLQT1 or KCNE1 protein. The preparation is substantially free of other human proteins. This is most readily accomplished by synthesis in a microorganism or in vitro.</p>
    <p>Active KVLQT1 or KCNE1 molecules can be introduced into cells by microinjection or by use of liposomes, for example. Alternatively, some active molecules may be taken up by cells, actively or by diffusion. Supply of molecules with KVLQT1 or KCNE1 activity should lead to partial reversal of LQT. Other molecules with KVLQT1 or KCNE1 activity (for example, peptides, drugs or organic compounds) may also be used to effect such a reversal. Modified polypeptides having substantially similar function are also used for peptide therapy.</p>
    <p>Methods of Use: Transformed Hosts</p>
    <p>Animals for testing therapeutic agents can be selected after mutagenesis of whole animals or after treatment of germline cells or zygotes. Such treatments include insertion of mutant KVLQT1 and/or KCNE1 alleles, usually from a second animal species, as well as insertion of disrupted homologous genes. Alternatively, the endogenous KVLQT1 or KCNE1 gene of the animals may be disrupted by insertion or deletion mutation or other genetic alterations using conventional techniques (Capecchi, 1989; Valancius and Smithies, 1991; Hasty et al., 1991; Shinkai et al., 1992; Mombaerts et al., 1992; Philpott et al., 1992; Snouwaert et al., 1992; Donehower et al., 1992). After test substances have been administered to the animals, the presence of LQT must be assessed. If the test substance prevents or suppresses the appearance of LQT, then the test substance is a candidate therapeutic agent for treatment of LQT. These animal models provide an extremely important testing vehicle for potential therapeutic products.</p>
    <p>Two strategies had been utilized herein to identify LQT genes, a candidate gene approach and positional cloning. Positional information is now available for three LQT loci with LQT1 having been mapped to chromosome 11p15.5 (Keating et al., 1991 a; Keating et al., 1991b), LQT2 to 7q35-36 and LQT3 to 3p21-24 (Jiang et al., 1994). The present invention has also identified minK, on chromosome 21, as an LQT gene. The candidate gene approach relies on likely mechanistic hypotheses based on physiology. Although little is known about the physiology of LQT, the disorder is associated with prolongation of the QT interval on electrocardiograms, a sign of abnormal cardiac repolarization. This association suggests that genes encoding ion channels, or their modulators, are reasonable candidates for LQT. This hypothesis is now supported by the discovery that chromosome 7-linked LQT results from mutations in HERG, a putative cardiac potassium channel gene. A neuroendocrine calcium channel gene (CACNL1A2; Chin et al., 1991; Seino et al., 1992) and a gene encoding a GTP-binding protein that modulates potassium channels (GNAI2; Weinstein et al., 1988; Magovcevic et al., 1992) became candidates for LQT3 based on their chromosomal location. Subsequent linkage analyses, however, have excluded these genes. It has now been shown that LQT3 is associated with SCN5A (Wang et al., 1995a). Despite considerable effort, however, a candidate gene approach to chromosome 11-linked LQT has not been successful. Two potassium channel genes (KCNA4 and KCNC1) were mapped to the short arm of chromosome 11 (Wymore et al., 1994), but both were excluded as candidates for LQT1 by linkage analyses (Russell et al., 1995; the present study). All other previously characterized cardiac potassium, chloride, sodium and calcium channel genes were similarly excluded based on their chromosomal locations. The present study has used positional cloning and mutational analyses to identify LQT1.</p>
    <p>The present invention has used genotypic analyses to show that KVLQT1 is tightly linked to LQT1 in 16 unrelated families (details provided in the Examples). KVLQT1 is a putative cardiac potassium channel gene and causes the chromosome 11-linked form of LQT. Genetic analyses suggested that KVLQT1 encodes a voltage-gated potassium channel with functional importance in cardiac repolarization and it is now shown that KVLQT1 coassembles with KCNE1 to form a cardiac I<sub>Ks </sub>potassium channel. If correct, the mechanism of chromosome 11-linked LQT probably involves reduced repolarizing KVLQT1 current. Since potassium channels with six transmembrane domains are thought to be formed from homo- or hetero-tetramers (MacKinnon, 1991; MacKinnon et al., 1993; Covarrubias et al., 1991), it is possible that LQT-associated mutations of KVLQT1 act through a dominant-negative mechanism. The type and location of KVLQT1 mutations described here are consistent with this hypothesis. The resultant suppression of potassium channel function, in turn, would likely lead to abnormal cardiac repolarization and increased risk of ventricular tachyarrhythmias. The mutations identified in HERG, and the biophysics of potassium channel alpha subunits, suggest that chromosome 7-linked LQT results from dominant-negative mutations and a resultant reduction in functional channels. In chromosome 3-linked LQT, by contrast, the LQT-associated deletions identified in SCN5A are likely to result in functional cardiac sodium channels with altered properties, such as delayed inactivation or altered voltage-dependence of channel inactivation. Delayed sodium channel inactivation would increase inward sodium current, depolarizing the membrane. This effect is similar to the altered membrane potential expected from HERG mutations where outward potassium current is decreased. It is unlikely that more deleterious mutations of SCN5A would cause LQT. A reduction of the total number of cardiac sodium channels, for example, would be expected to reduce action potential duration, a phenotype opposite that of LQT.</p>
    <p>Presymptomatic diagnosis of LQT has depended on identification of QT prolongation on electrocardiograms. Unfortunately, electrocardiograms are rarely performed in young, healthy individuals. In addition, many LQT gene carriers have relatively normal QT intervals, and the first sign of disease can be a fatal cardiac arrhythmia (Vincent et al., 1992). Now that more LQT genes (KVLQT1 and KCNE1) have been identified and have been associated with LQT, genetic testing for this disorder can be contemplated. This will require continued mutational analyses and identification of additional LQT genes. With more detailed phenotypic analyses, phenotypic differences between the varied forms of LQT may be discovered. These differences may be useful for diagnosis and treatment.</p>
    <p>The identification of the association between the KVLQT1 and KCNE1 gene mutations and LQT permits the early presymptomatic screening of individuals to identify those at risk for developing LQT. To identify such individuals, the KVLQT1 and/or KCNE1 alleles are screened for mutations either directly or after cloning the alleles. The alleles are tested for the presence of nucleic acid sequence differences from the normal allele using any suitable technique, including but not limited to, one of the following methods: fluorescent in situ hybridization (FISH), direct DNA sequencing, PFGE analysis, Southern blot analysis, single stranded conformation analysis (SSCP), linkage analysis, RNase protection assay, allele specific oligonucleotide (ASO), dot blot analysis and PCR-SSCP analysis. Also useful is the recently developed technique of DNA microchip technology. For example, either (1) the nucleotide sequence of both the cloned alleles and normal KVLQT1 or KCNE1 gene or appropriate fragment (coding sequence or genomic sequence) are determined and then compared, or (2) the RNA transcripts of the KVLQT1 or KCNE1 gene or gene fragment are hybridized to single stranded whole genomic DNA from an individual to be tested, and the resulting heteroduplex is treated with Ribonuclease A (RNase A) and run on a denaturing gel to detect the location of any mismatches. Two of these methods can be carried out according to the following procedures.</p>
    <p>The alleles of the KVLQT1 or KCNE1 gene in an individual to be tested are cloned using conventional techniques. For example, a blood sample is obtained from the individual. The genomic DNA isolated from the cells in this sample is partially digested to an average fragment size of approximately 20 kb. Fragments in the range from 18-21 kb are isolated. The resulting fragments are ligated into an appropriate vector. The sequences of the clones are then determined and compared to the normal KVLQT1 or KCNE1 gene.</p>
    <p>Alternatively, polymerase chain reactions (PCRs) are performed with primer pairs for the 5′ region or the exons of the KVLQT1 or KCNE1 gene. PCRs can also be performed with primer pairs based on any sequence of the normal KVLQT1 or KCNE1 gene. For example, primer pairs for one of the introns can be prepared and utilized. Finally, RT-PCR can also be performed on the mRNA. The amplified products are then analyzed by single stranded conformation polymorphisms (SSCP) using conventional techniques to identify any differences and these are then sequenced and compared to the normal gene sequence.</p>
    <p>Individuals can be quickly screened for common KVLQT1 or KCNE1 gene variants by amplifying the individual's DNA using suitable primer pairs and analyzing the amplified product, e.g., by dot-blot hybridization using allele-specific oligonucleotide probes.</p>
    <p>The second method employs RNase A to assist in the detection of differences between the normal KVLQT1 or KCNE1 gene and defective genes. This comparison is performed in steps using small (˜500 bp) restriction fragments of the KVLQT1 or KCNE1 gene as the probe. First, the KVLQT1 or KCNE1 gene is digested with a restriction enzyme(s) that cuts the gene sequence into fragments of approximately 500 bp. These fragments are separated on an electrophoresis gel, purified from the gel and cloned individually, in both orientations, into an SP6 vector (e.g., pSP64 or pSP65). The SP6-based plasmids containing inserts of the KVLQT1 or KCNE1 gene fragments are transcribed in vitro using the SP6 transcription system, well known in the art, in the presence of [α-<sup>32</sup>P]GTP, generating radiolabeled RNA transcripts of both strands of the gene.</p>
    <p>Individually, these RNA transcripts are used to form heteroduplexes with the allelic DNA using conventional techniques. Mismatches that occur in the RNA:DNA heteroduplex, owing to sequence differences between the KVLQT1 or KCNE1 fragment and the KVLQT1 or KCNE1 allele subclone from the individual, result in cleavage in the RNA strand when treated with RNase A. Such mismatches can be the result of point mutations or small deletions in the individual's allele. Cleavage of the RNA strand yields two or more small RNA fragments, which run faster on the denaturing gel than the RNA probe itself.</p>
    <p>Any differences which are found, will identify an individual as having a molecular variant of the KVLQT1 or KCNE1 gene and the consequent presence of long QT syndrome. These variants can take a number of forms. The most severe forms would be frame shift mutations or large deletions which would cause the gene to code for an abnormal protein or one which would significantly alter protein expression. Less severe disruptive mutations would include small in-frame deletions and nonconservative base pair substitutions which would have a significant effect on the protein produced, such as changes to or from a cysteine residue, from a basic to an acidic amino acid or vice versa, from a hydrophobic to hydrophilic amino acid or vice versa, or other mutations which would affect secondary or tertiary protein structure. Silent mutations or those resulting in conservative amino acid substitutions would not generally be expected to disrupt protein function.</p>
    <p>Genetic testing will enable practitioners to identify individuals at risk for LQT at, or even before, birth. Presymptomatic diagnosis of LQT will enable prevention of these disorders. Existing medical therapies, including beta adrenergic blocking agents, may prevent and delay the onset of problems associated with the disease. Finally, this invention changes our understanding of the cause and treatment of common heart disease like cardiac arrhythmias which account for 11% of all natural deaths. Existing diagnosis has focused on measuring the QT interval from electrocardiograms. This method is not a fully accurate indicator of the presence of long QT syndrome. The present invention is a more accurate indicator of the presence of the disease. Genetic testing and improved mechanistic understanding of LQT provide the opportunity for prevention of life-threatening arrhythmias through rational therapies. It is possible, for example, that potassium channel opening agents will reduce the risk of arrhythmias in patients with KVLQT1 or KCNE1 mutations; sodium channel blocking agents, by contrast, may be a more effective treatment for patients with mutations that alter the function of SCN5A. Finally, these studies may provide insight into mechanisms underlying common arrhythmias, as these arrhythmias are often associated with abnormal cardiac repolarization and may result from a combination of inherited and acquired factors.</p>
    <p>Pharmaceutical Compositions and Routes of Administration</p>
    <p>The KVLQT1 and KCNE1 polypeptides, antibodies, peptides and nucleic acids of the present invention can be formulated in pharmaceutical compositions, which are prepared according to conventional pharmaceutical compounding techniques. See, for example, <i>Remington's Pharmaceutical Sciences, </i>18th Ed. (1990, Mack Publishing Co., Easton, Pa.). The composition may contain the active agent or pharmaceutically acceptable salts of the active agent. These compositions may comprise, in addition to one of the active substances, a pharmaceutically acceptable excipient, carrier, buffer, stabilizer or other materials well known in the art. Such materials should be non-toxic and should not interfere with the efficacy of the active ingredient. The carrier may take a wide variety of forms depending on the form of preparation desired for administration, e.g., intravenous, oral, intrathecal, epineural or parenteral.</p>
    <p>For oral administration, the compounds can be formulated into solid or liquid preparations such as capsules, pills, tablets, lozenges, melts, powders, suspensions or emulsions. In preparing the compositions in oral dosage form, any of the usual pharmaceutical media may be employed, such as, for example, water, glycols, oils, alcohols, flavoring agents, preservatives, coloring agents, suspending agents, and the like in the case of oral liquid preparations (such as, for example, suspensions, elixirs and solutions); or carriers such as starches, sugars, diluents, granulating agents, lubricants, binders, disintegrating agents and the like in the case of oral solid preparations (such as, for example, powders, capsules and tablets). Because of their ease in administration, tablets and capsules represent the most advantageous oral dosage unit form, in which case solid pharmaceutical carriers are obviously employed. If desired, tablets may be sugar-coated or enteric-coated by standard techniques. The active agent can be encapsulated to make it stable to passage through the gastrointestinal tract while at the same time allowing for passage across the blood brain barrier. See for example, WO 96/11698.</p>
    <p>For parenteral administration, the compound may be dissolved in a pharmaceutical carrier and administered as either a solution or a suspension. Illustrative of suitable carriers are water, saline, dextrose solutions, fructose solutions, ethanol, or oils of animal, vegetative or synthetic origin. The carrier may also contain other ingredients, for example, preservatives, suspending agents, solubilizing agents, buffers and the like. When the compounds are being administered intrathecally, they may also be dissolved in cerebrospinal fluid.</p>
    <p>The active agent is preferably administered in a therapeutically effective amount. The actual amount administered, and the rate and time-course of administration, will depend on the nature and severity of the condition being treated. Prescription of treatment, e.g. decisions on dosage, timing, etc., is within the responsibility of general practitioners or specialists, and typically takes account of the disorder to be treated, the condition of the individual patient, the site of delivery, the method of administration and other factors known to practitioners. Examples of techniques and protocols can be found in <i>Remington's Pharmaceutical Sciences. </i> </p>
    <p>Alternatively, targeting therapies may be used to deliver the active agent more specifically to certain types of cell, by the use of targeting systems such as antibodies or cell specific ligands. Targeting may be desirable for a variety of reasons, e.g. if the agent is unacceptably toxic, or if it would otherwise require too high a dosage, or if it would not otherwise be able to enter the target cells.</p>
    <p>Instead of administering these agents directly, they could be produced in the target cell, e.g. in a viral vector such as described above or in a cell based delivery system such as described in U.S. Pat. No. 5,550,050 and published PCT application Nos. WO 92/19195, WO 94/25503, WO 95/01203, WO 95/05452, WO 96/02286, WO 96/02646, WO 96/40871, WO 96/40959 and WO 97/12635, designed for implantation in a patient. The vector could be targeted to the specific cells to be treated, or it could contain regulatory elements which are more tissue specific to the target cells. The cell based delivery system is designed to be implanted in a patient's body at the desired target site and contains a coding sequence for the active agent. Alternatively, the agent could be administered in a precursor form for conversion to the active form by an activating agent produced in, or targeted to, the cells to be treated. See for example, EP 425,731A and WO 90/07936.</p>
    <p>The present invention is further detailed in the following Examples, which are offered by way of illustration and are not intended to limit the invention in any manner. Standard techniques well known in the art or the techniques specifically described below are utilized.</p>
    <heading>EXAMPLE 1</heading> <p>Methods for Phenotypic Evaluation</p>
    <p>For these studies, six large LQT kindreds (K1532, K1723, K2605, K1807, K161 and K162) as well as some small kindreds and sporadic cases were studied. LQT patients were identified from medical clinics throughout North America and Europe. Two factors were considered for phenotyping: 1) historical data (the presence of syncope, the number of syncopal episodes, the presence of seizures, the age of onset of symptoms, and the occurrence of sudden death); and 2) the QT interval on electrocardiograms corrected for heart rate (QT<sub>c</sub>) (Bazzett, 1920). To avoid misclassifying individuals, the same conservative approach to phenotypic assignment that was successful in previous studies was used (Keating et al., 1991a; Keating et al., 1991b; Jiang et al., 1994). Informed consent was obtained from each individual, or their guardians, in accordance with local institutional review board guidelines. Phenotypic data were interpreted without knowledge of genotype. Symptomatic individuals with a corrected QT interval (QT<sub>c</sub>) of 0.45 seconds or greater and asymptomatic individuals with a QT<sub>c </sub>of 0.47 seconds or greater were classified as affected. Asymptomatic individuals with a QT<sub>c </sub>of 0.41 seconds or less were classified as unaffected. Asymptomatic individuals with QT<sub>c </sub>between 0.41 and 0.47 seconds and symptomatic individuals with QT<sub>c </sub>of 0.44 seconds or less were classified as uncertain.</p>
    <heading>EXAMPLE 2</heading> <p>Genotyping and Linkage Analysis</p>
    <p>Genomic DNA was prepared from peripheral blood lymphocytes or cell lines derived from Epstein-Barr virus transformed lymphocytes using standard procedures (Anderson and Gusella, 1984). For genotypic analyses, four small tandem repeat (STR) polymorphisms were used that were previously mapped to chromosome 11p15.5: D11S922, TH, D11S1318 and D11S860 (Gyapay et al., 1994). Genotyping of RFLP markers (HRAS1, D11S454 and D11S12) was performed as previously described (Keating et al., 1991a).</p>
    <p>Pairwise linkage analysis was performed using MLINK in LINKAGE v5.1 (Lathrop et al., 1985). Assumed values of 0.90 for penetrance and 0.001 for LQT gene frequency were used. Gene frequency was assumed to be equal between males and females. Male and female recombination frequencies were considered to be equal. STR allele frequencies were 1/n where n=number of observed alleles. Although the maximum LOD score for D11S454 was identified at a recombination fraction of 0, the presence of one non-obligate recombinant (individual VI-14, FIG. 1) places this LQT gene telomeric of D11S454.</p>
    <heading>EXAMPLE 3</heading> <p>Physical Mapping</p>
    <p>Primers were designed based on sequences from TH-INS-IGFII and D11S454 loci and used to identify and isolate clones from CEPH YAC libraries using the PCR based technique (Green and Olson, 1990; Kwiatkowski et al., 1990). YAC terminal sequences were determined by inverse PCR as described (Ochman et al., 1988) and used as STSs.</p>
    <p>P1 clones were isolated using single copy probes from previously identified cosmids cosQW22 (this study), cCI11-469 (D11S679), cCI11-385 (D11S551), cCI11-565 (D11S601), cCI11-237 (D11S454) (Tanigami et al., 1992; Tokino et al., 1991; Sternberg, 1990). Newly isolated P1s were mapped to chromosome 11p15 by FISH or Southern analyses. End-specific riboprobes were generated from newly isolated P1s and used to identify additional adjacent clones (Riboprobe Gemini Core System Kit; Promega). DNA for P1 and cosmid clones was prepared using alkaline lysis plasmid isolation and purified by equilibrium centrifugation in CsC1-ethidium bromide gradients as described (Sambrook et al., 1989). P1 insert end sequences were determined by cycle sequencing as described (Wang and Keating, 1994). STSs were generated based on these insert end sequences. Overlap between P1s and cosmids was calculated by summing the restriction fragments in common.</p>
    <heading>EXAMPLE 4</heading> <p>Isolation and Characterization of KVLQT1 Clones</p>
    <p>An adult human cardiac cDNA library (Stratagene) was plated, and 1×10<sup>6 </sup>plaques were screened using trapped exon 4181A as the probe. Sequences of trapped exon 4181A were used to design oligonucleotide probes for cDNA library screening. The GENETRAPPER™ cDNA Positive Selection System was used to screen 1×10<sup>11 </sup>clones from a human heart cDNA library (Life Technologies, Inc.). The sequences of the capture and repair oligonucleotides were 5′-CAGATCCTGAGGATGCT-3′ (SEQ ID NO:7) and 5′-GTACCTGGCTGAGAAGG-3′ (SEQ ID NO:8).</p>
    <p>Composite cDNA sequences for KVLQT1 were obtained by end sequencing of overlapping cDNA clones and by primer walking. Sequencing was performed either automatically, using Pharmacia A.L.F. automated sequencers, or manually, using a Sequenase Version 2.0 DNA Sequencing Kit (United States Biochemical, Inc.). Database analyses and sequence analyses were carried out using the GCG software package, IG software package, and the BLAST network service from the National Center for Biotechnology Information.</p>
    <p>The partial genomic structure (from transmembrane domain S2 to S6) of KVLQT1 was determined by cycle sequencing of P1 18B12 as described (Wang and Keating, 1994). Primers were designed based on KVLQT1 cDNA sequence and used for cycle sequencing.</p>
    <heading>EXAMPLE 5</heading> <p>Mutation Analyses</p>
    <p>SSCP was carried out as previously described (Wang et al., 1995a; Wang et al., 1995b). Normal and aberrant SSCP products were isolated sequenced directly as described (Wang and Keating, 1994) or subdloned into pBluescript (SK<sup>+</sup>; Stratagene) using the T-vector method (Marchuk et al., 1991). When the latter method was used, several clones were sequenced by the dideoxy chain termination method using Sequenase™ Version 2.0 (United States Biochemicals, Inc.).</p>
    <heading>EXAMPLE 6</heading> <p>Northern Analyses</p>
    <p>A multiple tissue Northern filter (Human MTN blot 1, Clontech) was probed with a <sup>32</sup>P-labeled KVLQT1 cDNA probe as previously described (Curran et al., 1995).</p>
    <heading>EXAMPLE 7</heading> <p>Refined Genetic and Physical Localization of LQT1</p>
    <p>The precise location of LQT1 was determined by genotypic analyses in kindred 1532 (K1532), a large Utah family of northern European descent (FIG. <b>1</b>). This kindred had been used in the initial study linking the first LQT gene, LQT1, to chromosome 11p15.5 (Keating et al., 1991a; Keating et al., 1991b). Additional family members were identified and phenotyped for a total sample size of 217 individuals. Phenotypic determination was performed as previously described (Keating et al., 1991a; Keating et al., 1991b; Jiang et al., 1994). Preliminary genotypic analyses using markers at HRAS, TH, D11S454, and D11S2 included all ascertained members ofK1532. These experiments identified informative branches of this family. Additional genotypic analyses were performed using three highly polymorphic markers from chromosome 11p15.5: D11S922, D11S1318, and D11S860 (Gyapay et al., 1994). Genotypes and pairwise LOD scores for each marker are shown in FIG. <b>1</b> and Table 2. Of these markers, TH and D11S1318 were completely linked. Recombination was identified with all other markers tested, including HRAS, but in each case a statistically significant positive LOD score (+3 or greater) was identified. These data indicate that LQT1 is completely linked to TH and D11S1318 in this kindred and that the disease gene is located centromeric of HRAS.</p>
    <p>To refine localization of LQT1, haplotype analyses of K1532 were performed (see FIG. <b>1</b>). Nine chromosomes bearing informative recombination events were identified. Telomeric recombination events were observed in unaffected individual IV-22 (between D11S922 and TH), affected individual IV-25 (between D11S922 and TH), unaffected individual V-6 (between HRAS and D11S922), and affected individual V-24 (between HRAS and D11S922). Centromeric recombination events were identified in unaffected individual V-17 (between D11S860 and D11S454), affected individual V-24 (between D11S860 and D11S454), unaffected individual V-34 (between D11S860 and D11S454), unaffected individual VI-13 (between D11S860 and D11S454), unaffected individual VI-14 (between D11S454 and D11S1318), and affected</p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 2</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Pairwise LOD Scores Between LQT1 and 11p15.5 Markers</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="42pt" align="left"> </colspec> <colspec colname="1" colwidth="175pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Recombination Fraction (θ)</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="9"> <colspec colname="offset" colwidth="42pt" align="left"> </colspec> <colspec colname="1" colwidth="21pt" align="center"> </colspec> <colspec colname="2" colwidth="28pt" align="center"> </colspec> <colspec colname="3" colwidth="21pt" align="center"> </colspec> <colspec colname="4" colwidth="21pt" align="center"> </colspec> <colspec colname="5" colwidth="21pt" align="center"> </colspec> <colspec colname="6" colwidth="21pt" align="center"> </colspec> <colspec colname="7" colwidth="21pt" align="center"> </colspec> <colspec colname="8" colwidth="21pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">0.0</td>
                <td class="description-td">0.001</td>
                <td class="description-td">0.01</td>
                <td class="description-td">0.05</td>
                <td class="description-td">0.1</td>
                <td class="description-td">0.2</td>
                <td class="description-td">Z<sub>max</sub>*</td>
                <td class="description-td">θ<sub>max</sub> <sup>†</sup> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="OFFSET" nameend="8" align="center" rowsep="1" class="description-td" colspan="9"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="9"> <colspec colname="1" colwidth="42pt" align="left"> </colspec> <colspec colname="2" colwidth="21pt" align="char" char="."> </colspec> <colspec colname="3" colwidth="28pt" align="char" char="."> </colspec> <colspec colname="4" colwidth="21pt" align="char" char="."> </colspec> <colspec colname="5" colwidth="21pt" align="char" char="."> </colspec> <colspec colname="6" colwidth="21pt" align="char" char="."> </colspec> <colspec colname="7" colwidth="21pt" align="char" char="."> </colspec> <colspec colname="8" colwidth="21pt" align="char" char="."> </colspec> <colspec colname="9" colwidth="21pt" align="char" char="."> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">HRAS</td>
                <td class="description-td">9.67</td>
                <td class="description-td">9.94</td>
                <td class="description-td">10.50</td>
                <td class="description-td">10.38</td>
                <td class="description-td">9.62</td>
                <td class="description-td">7.57</td>
                <td class="description-td">10.59</td>
                <td class="description-td">0.021</td>
              </tr> <tr class="description-tr"> <td class="description-td">D11S922</td>
                <td class="description-td">10.05</td>
                <td class="description-td">13.05</td>
                <td class="description-td">13.85</td>
                <td class="description-td">13.59</td>
                <td class="description-td">12.59</td>
                <td class="description-td">10.01</td>
                <td class="description-td">13.92</td>
                <td class="description-td">0.019</td>
              </tr> <tr class="description-tr"> <td class="description-td">TH</td>
                <td class="description-td">11.01</td>
                <td class="description-td">10.99</td>
                <td class="description-td">10.82</td>
                <td class="description-td">10.06</td>
                <td class="description-td">9.07</td>
                <td class="description-td">6.96</td>
                <td class="description-td">11.01</td>
                <td class="description-td">0.0</td>
              </tr> <tr class="description-tr"> <td class="description-td">D11S1318</td>
                <td class="description-td">10.30</td>
                <td class="description-td">10.29</td>
                <td class="description-td">10.13</td>
                <td class="description-td">9.40</td>
                <td class="description-td">8.47</td>
                <td class="description-td">6.50</td>
                <td class="description-td">10.30</td>
                <td class="description-td">0.0</td>
              </tr> <tr class="description-tr"> <td class="description-td">KVLQT1</td>
                <td class="description-td">14.19</td>
                <td class="description-td">14.17</td>
                <td class="description-td">13.94</td>
                <td class="description-td">12.89</td>
                <td class="description-td">11.54</td>
                <td class="description-td">8.68</td>
                <td class="description-td">14.19</td>
                <td class="description-td">0.0</td>
              </tr> <tr class="description-tr"> <td class="description-td">D11S454</td>
                <td class="description-td">11.06</td>
                <td class="description-td">11.05</td>
                <td class="description-td">10.89</td>
                <td class="description-td">10.16</td>
                <td class="description-td">9.17</td>
                <td class="description-td">7.01</td>
                <td class="description-td">11.06</td>
                <td class="description-td">0.0</td>
              </tr> <tr class="description-tr"> <td class="description-td">D11S860</td>
                <td class="description-td">5.77</td>
                <td class="description-td">6.92</td>
                <td class="description-td">8.32</td>
                <td class="description-td">9.14</td>
                <td class="description-td">8.92</td>
                <td class="description-td">7.46</td>
                <td class="description-td">9.15</td>
                <td class="description-td">0.058</td>
              </tr> <tr class="description-tr"> <td class="description-td">D11S12</td>
                <td class="description-td">1.50</td>
                <td class="description-td">2.26</td>
                <td class="description-td">3.12</td>
                <td class="description-td">3.46</td>
                <td class="description-td">3.27</td>
                <td class="description-td">2.49</td>
                <td class="description-td">3.46</td>
                <td class="description-td">0.047</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="9" align="center" rowsep="1" class="description-td" colspan="9"> </td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="9" align="left" class="description-td" colspan="9">LOD scores were computed with the assumption of 90% penetrance and gene frequency of 0.001 (Lathrop et al., 1985). </td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="9" align="left" class="description-td" colspan="9">*Z<sub>max </sub>indicates maximum LOD score. </td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="9" align="left" class="description-td" colspan="9"> <sup>†</sup>θ<sub>max </sub>indicates estimated recombination fraction at Z<sub>max</sub>. </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p>individual VI-16 (between D11S860 and D11S454). These data indicate that LQT1 is located between D11S922 and D11S454. Together with recent studies placing LQT1 centromeric of TH (Russell et al., 1995), these data place LQT1 in the interval between TH and D11S454.</p>
    <p>The size of the region containing LQT1 was estimated using pulsed-field gel analyses with genomic probes from chromosome 11P15.5. Probes from TH, D11S551 and D11S454 hybridized to a 700 kb Mlu I restriction fragment (FIG. <b>2</b>). These data suggested that the region containing LQT1 is less than 700 kb. Physical representation of this region was achieved by screening yeast artificial chromosome (YAC) and P1 libraries with probes from the region (Tanigami et al., 1992; Tokino et al., 1991). The order of these clones was confirmed using fluorescent in situ hybridization (FISH) analyses as: telomere-TH-D11S551-D11S679-D11S601-D11S454-centromere. The clones identified in initial experiments were then used for identification of adjacent, overlapping clones. The minimum set of clones from the LQT1 interval is shown in FIG. <b>2</b>.</p>
    <heading>EXAMPLE 8</heading> <p>Identification and Characterization of KVLQT1</p>
    <p>Exon amplification with clones from the physical map was performed to identify candidate genes for LQT1. Exon trapping was performed using pSPL3B (Burn et al., 1995) on genomic P1 clones as previously described (Buckler et al., 1991; Church et al., 1994). A minimum of 128 trapped exons from each P1 clone were initially characterized by sizing the PCR products. From these, 400 clones were further analyzed by dideoxy sequencing using an A.L.F. automated sequencer (Pharmacia). DNA sequence and database analyses revealed eight possible exons with predicted amino acid sequence similarity to ion channels. The highest similarity was obtained for a 238 base pair trapped exon (4181A), with 53% similarity to potassium channel proteins from multiple species, including similarity to a portion of a putative pore region. PCR analyses were used to map 4181A to the short arm of chromosome 11 and to two P1s from the physical map (118A10, 18B12). These data suggested that 4181A was part of a potassium channel gene on chromosome 11p15.5.</p>
    <p>Two different cDNA library screening methods were used to determine if trapped exon 4181A was part of a gene. Traditional plaque filter hybridization with an adult human cardiac cDNA library led to the identification of a single positive clone. A variation of cDNA selection was used to screen a second cardiac cDNA library (the GENETRAPPER™ cDNA Positive Selection System, Life Technologies, Inc.), and twelve independent clones were recovered. DNA sequence analyses revealed complete alignment with sequences derived from 4181A and the other trapped exons described above. The longest open reading frame spanned 1654 base pairs. Two consensus polyadenylation signals were identified upstream of the poly(A) tail in the 3′ untranslated region. The complete cDNA was not obtained at this stage of the study.</p>
    <p>The partial cDNA predicted a protein with structural characteristics of potassium channels. Hydropathy analyses suggested a topology of six major hydrophobic regions that may represent membrane-spanning α-helices. These regions share sequence similarity with potassium channel transmembrane domains S1-S6. A comparison of the predicted amino acid sequence derived from the identified gene and the Shaker (SHA) potassium channel (Pongs et al., 1988) is shown in FIG. <b>3</b>. In the region containing S1-S6, the amino acid sequence identity was 30% and similarity was 59%. The sequence located 3′ of S1-S6 did not have significant similarity to any known protein. Because this gene has high similarity to voltage-gated potassium channel genes and became a strong candidate for LQT1, it was named KVLQT1.</p>
    <p>Northern blot analyses were used to determine the tissue distribution of KVLQT1 mRNA. KVLQT1 cDNA probes detected a 3.2 kb transcript in human pancreas, heart, kidney, lung, and placenta, but not in skeletal muscle, liver, or brain (FIG. <b>4</b>). The heart showed highest levels of KVLQT1 mRNA. The Northern analyses were performed using a multiple tissue Northern filter (Human MTN blot 1, Clontech) as described by Curran et al., 1995.</p>
    <heading>EXAMPLE 9</heading> <p>Characterization of the Complete KVLQT1 cDNA</p>
    <p>The studies described above resulted in the cloning and characterization of an incomplete cDNA for KVLQT1. The sequence of this incomplete cDNA predicted a protein with six hydrophobic membrane-spanning α-helices (S1-S6) and a typical K<sup>+ </sup>channel pore signature sequence (Heginbotham et al., 1994). However, this cDNA appeared to be missing the amino terminal domain and did not functionally express. To define the complete sequence of KVLQT1, several cDNA libraries were screened and a new clone was isolated. A cDNA probe containing exons 3 through 6 was used to isolate three full length KVLQT1 cDNA clones from an adult heart cDNA library prepared in the laboratory using SuperScript Choice system (GIBCO BRL). The complete cDNA sequence and the encoded protein are shown in FIGS. 5A-5B.</p>
    <heading>EXAMPLE 10</heading> <p>Genomic Structure of KVLQT1</p>
    <p>The genomic DNA of KVLQT1 was examined and the exon/intron boundaries determined for all exons.</p>
    <p>A. Isolation of cDNA Clones</p>
    <p>A cDNA probe containing exons 3 through 6 was used to isolate three full length KVLQT1 cDNA clones from an adult heart cDNA library prepared in the laboratory using SuperScript Choice system (GIBCO BRL).</p>
    <p>B. Isolation of Genomic Clones</p>
    <p>KVLQT1 P1 clones were isolated as described (Wang et al., 1996). The cosmid containing exon 1 was isolated screening a human genomic cosmid library (Stratagene) with a cDNA probe from exon 1.</p>
    <p>C. Exon/Intron Boundary Determination</p>
    <p>All genomic clones were sequenced using primers designed to the cDNA sequences. The KVLQT1 P1 clones were cycle sequenced using ThermoSequenase (Amersham Life Science). The KVLQT1 cosmids were sequenced by the dideoxy chain termination method on an Applied Biosystems model 373A DNA sequencer. The exact exon/intron boundaries were determined by comparison of cDNA, genomic sequences, and known splice site consensus sequences.</p>
    <p>D. Design of PCR Primers and PCR Reaction Conditions</p>
    <p>Primers to amplify exons of the two genes were designed empirically or using OLIGO 4.0 (NBI). Amplification conditions were:</p>
    <p>(1) 94° C. for 3 minutes followed by 30 cycles of 94° C. for 10 seconds, 58° C. for 20 seconds and 72° C. for 20 seconds and a 5 minute extension at 72° C.</p>
    <p>(2) same as conditions in (1) except that the reactions had final concentrations of 10% glycerol and 4% formamide and were overlaid with mineral oil.</p>
    <p>(3) 94° C. for 3 minutes followed by 5 cycles of 94° C. for 10 seconds, 64° C. for 20 seconds and 72° C. for 20 seconds and 30 cycles of 94° C. for 10 seconds, 62° C. for 20 seconds and 72° C. for 20 seconds and a 5 minute extension at 72° C.</p>
    <p>E. KVLQT1 Genomic Structure and Primer Sets</p>
    <p>Full length cDNA clones were isolated from an adult heart cDNA library. A 5′-cDNA probe generated from one of these clones was used to isolate cos1, a genomic cosmid clone containing exon 1. P1 genomic clones encompassing the rest of the KVLQT1 cDNA were previously isolated (Wang et al., 1996). These genomic clones span approximately 400 kb on chromosome 11p15.5 (FIG. <b>6</b>). To determine the exon structure and exon/intron boundaries, cos1 and P1 clones 118A 10, 112E3, 46F10 and 49E5 were sequenced using primers designed to the cDNA. Comparison of the genomic and cDNA sequences of KVLQT1 revealed the presence of 16 exons (FIGS. 5A-5B and Table 3). Exon size ranged from 47 bp (exon 14) to 1122 bp (exon 16). All intronic sequences contained the invariant GT and AG at the donor and acceptor splice sites, respectively (Table 3). One pair of PCR primers was designed for each of intron sequences flanking exons 2 through 16 and two pairs of primers with overlapping products were designed for exon 1 due to its large size (Table 4). These primers can be used to screen all KVLQT1 exons.</p>
    <heading>EXAMPLE 11</heading> <p>Characterization of KVLQT1 Function</p>
    <p>To define the function of KVLQT1, Chinese hamster ovary (CHO) cells were transfected with the complete cDNA described above in Example 9. The KVLQT1 cDNA was subcloned into pCEP4 (InVitrogen). CHO cells were cultured in Ham's F-12 medium and transiently transfected using Lipofectamine (Gibco BRL). Cells were transfected for 18 hours in 35 mm dishes containing 6 μL lipofectamine, 0.5 μg green fluorescent protein (pGreen Lantern-1, Gibco BRL), and 1.5 μg of KVLQT1 in pCEP4. Fluorescent cells were voltage-clamped using an Axopatch 200 patch clamp amplifier (Axon Instruments) 48 to 78 hours after transfection. The bathing solution contained, in mM: 142 NaCl, 2 KCl, 1.2 MgCl<sub>2</sub>, 1.8 CaCl<sub>2</sub>, 11.1 glucose, 5.5 HEPES buffer (pH 7.4, 22-25° C.). The pipette solution contained, in mM: 110 potassium glutamate, 20 KCl, 1.0 MgCl<sub>2</sub>, 5 EGTA, 5 K<sub>2</sub>ATP, 10 HEPES (pH 7.3). Data acquisition and analyses were done using pCLAMP6 (Axon Instruments). The voltage dependence of current activation was determined by fitting the relationship between tail currents (determined by extrapolation of deactivating phase of current to the end of the test pulse) and test potential with a Boltzmann function. Tail currents were normalized relative to the largest value for each oocyte.</p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" pgwide="1" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="266pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 3</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Intron/Exon Boundaries in KVLQT1</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="4"> <colspec colname="1" colwidth="21pt" align="center"> </colspec> <colspec colname="2" colwidth="98pt" align="left"> </colspec> <colspec colname="3" colwidth="42pt" align="center"> </colspec> <colspec colname="4" colwidth="105pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Exon</td>
                <td class="description-td"> </td>
                <td class="description-td">EXON</td>
                <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">No.</td>
                <td class="description-td">intron/EXON<sup>a</sup> </td>
                <td class="description-td">(total bases)</td>
                <td class="description-td">EXON/intron<sup>a</sup> </td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="4" align="center" rowsep="1" class="description-td" colspan="4"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="4"> <colspec colname="1" colwidth="21pt" align="char" char="."> </colspec> <colspec colname="2" colwidth="98pt" align="left"> </colspec> <colspec colname="3" colwidth="42pt" align="center"> </colspec> <colspec colname="4" colwidth="105pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">1</td>
                <td class="description-td">5′UTR . . . ATGGCCGCGG (9)</td>
                <td class="description-td">386+  </td>
                <td class="description-td">ACTTCGCCGTgtgagtatcg (10)</td>
              </tr> <tr class="description-tr"> <td class="description-td">2</td>
                <td class="description-td">tgtcttgcagCTTCCTCATC (11)</td>
                <td class="description-td">91</td>
                <td class="description-td">CTTCTGGATGgtacgtagca (12)</td>
              </tr> <tr class="description-tr"> <td class="description-td">3</td>
                <td class="description-td">gtccctgcagGAGATCGTGC (13)</td>
                <td class="description-td">127</td>
                <td class="description-td">TCCATCATCGgtgagtcatg (14)</td>
              </tr> <tr class="description-tr"> <td class="description-td">4</td>
                <td class="description-td">cactccacagACCTCATCGT (15)</td>
                <td class="description-td">79</td>
                <td class="description-td">GGGCCATCAGgtgcgtctgt (16)</td>
              </tr> <tr class="description-tr"> <td class="description-td">5</td>
                <td class="description-td">tccttcgcagGGGCATCCGC (17)</td>
                <td class="description-td">97</td>
                <td class="description-td">CCACCGCCAGgtgggtggcc (18)</td>
              </tr> <tr class="description-tr"> <td class="description-td">6</td>
                <td class="description-td">tctggcctagGAGCTGATAA (19)</td>
                <td class="description-td">141</td>
                <td class="description-td">GTGGGGGGTGgtaagtcgga (20)</td>
              </tr> <tr class="description-tr"> <td class="description-td">7</td>
                <td class="description-td">ctccctgcagGTCACAGTCA (21)</td>
                <td class="description-td">111</td>
                <td class="description-td">GCTCCCAGCGgtaggtgccc (22)</td>
              </tr> <tr class="description-tr"> <td class="description-td">8</td>
                <td class="description-td">tccttcccagGGGATTCTTG (23)</td>
                <td class="description-td">96</td>
                <td class="description-td">ACTCATTCAGgtgcggtgcc (24)</td>
              </tr> <tr class="description-tr"> <td class="description-td">9</td>
                <td class="description-td">cccacctcagACCGCATGGA (25)</td>
                <td class="description-td">123</td>
                <td class="description-td">GTCTGTGGTGgtgagtagcc (26)</td>
              </tr> <tr class="description-tr"> <td class="description-td">10</td>
                <td class="description-td">ttttttttagGTAAAGAAAA (27)</td>
                <td class="description-td">142</td>
                <td class="description-td">GACAGTTCTGgtgagaaccc (28)</td>
              </tr> <tr class="description-tr"> <td class="description-td">11</td>
                <td class="description-td">ttctcctcagTAAGGAAGAG (29)</td>
                <td class="description-td">121</td>
                <td class="description-td">ACATCTCACAgtgagtgcct (30)</td>
              </tr> <tr class="description-tr"> <td class="description-td">12</td>
                <td class="description-td">tccactgcagGCTGCGGGAA (31)</td>
                <td class="description-td">76</td>
                <td class="description-td">GAAATTCCAGgtaagccctg (32)</td>
              </tr> <tr class="description-tr"> <td class="description-td">13</td>
                <td class="description-td">tgtcccgcagCAAGCGCGGA (33)</td>
                <td class="description-td">95</td>
                <td class="description-td">TGCAGAGGAGgtgggcacgg (34)</td>
              </tr> <tr class="description-tr"> <td class="description-td">14</td>
                <td class="description-td">ttctctccagGCTGGACCAG (35)</td>
                <td class="description-td">47</td>
                <td class="description-td">TCCGTCTCAGgtgggtttct (36)</td>
              </tr> <tr class="description-tr"> <td class="description-td">15</td>
                <td class="description-td">tcccccatagAAAAGAGCAA (37)</td>
                <td class="description-td">62</td>
                <td class="description-td">AGAAGACAAGgtaggctcac (38)</td>
              </tr> <tr class="description-tr"> <td class="description-td">16</td>
                <td class="description-td">gtccccgcagGTGACGCAGC (39)</td>
                <td class="description-td">237+  </td>
                <td class="description-td">GGGGTCCTGA . . . 3′UTR (40)</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="4" align="center" rowsep="1" class="description-td" colspan="4"> </td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="4" align="left" class="description-td" colspan="4"> <sup>a</sup>SEQ ID NO is shown in parentheses following each sequence. </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" pgwide="1" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="350pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 4</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Primers Used to Amplify KVLQT1 Exons</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="5"> <colspec colname="1" colwidth="21pt" align="center"> </colspec> <colspec colname="2" colwidth="147pt" align="left"> </colspec> <colspec colname="3" colwidth="147pt" align="left"> </colspec> <colspec colname="4" colwidth="21pt" align="left"> </colspec> <colspec colname="5" colwidth="14pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Exon</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">No.</td>
                <td class="description-td">Forward Primer<sup>a</sup> </td>
                <td class="description-td">Reverse Primer<sup>a</sup> </td>
                <td class="description-td">Size</td>
                <td class="description-td">C<sup>b</sup> </td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="5" align="center" rowsep="1" class="description-td" colspan="5"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="7"> <colspec colname="1" colwidth="21pt" align="char" char="."> </colspec> <colspec colname="2" colwidth="126pt" align="left"> </colspec> <colspec colname="3" colwidth="21pt" align="left"> </colspec> <colspec colname="4" colwidth="126pt" align="left"> </colspec> <colspec colname="5" colwidth="21pt" align="left"> </colspec> <colspec colname="6" colwidth="21pt" align="left"> </colspec> <colspec colname="7" colwidth="14pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> 1</td>
                <td class="description-td">CTCGCCTTCGCTGCAGCTC</td>
                <td class="description-td">(41)</td>
                <td class="description-td">GCGCGGGTCTAGGCTCACC</td>
                <td class="description-td">(42)</td>
                <td class="description-td">334</td>
                <td class="description-td">2</td>
              </tr> <tr class="description-tr"> <td class="description-td">1</td>
                <td class="description-td">CGCCGCGCCCCCAGTTGC</td>
                <td class="description-td">(43)</td>
                <td class="description-td">CAGAGCTCCCCCACACCAG</td>
                <td class="description-td">(44)</td>
                <td class="description-td">224</td>
                <td class="description-td">2</td>
              </tr> <tr class="description-tr"> <td class="description-td">2</td>
                <td class="description-td">ATGGGCAGAGGCCGTGATGCTGAC</td>
                <td class="description-td">(45)</td>
                <td class="description-td">ATCCAGCCATGCCCTCAGATGC</td>
                <td class="description-td">(46)</td>
                <td class="description-td">165</td>
                <td class="description-td">3</td>
              </tr> <tr class="description-tr"> <td class="description-td">3</td>
                <td class="description-td">GTTCAAACAGGTTGCAGGGTCTGA</td>
                <td class="description-td">(47)</td>
                <td class="description-td">CTTCCTGGTCTGGAAACCTGG</td>
                <td class="description-td">(48)</td>
                <td class="description-td">256</td>
                <td class="description-td">3</td>
              </tr> <tr class="description-tr"> <td class="description-td">4</td>
                <td class="description-td">CTCTTCCCTGGGGCCCTGGC</td>
                <td class="description-td">(49)</td>
                <td class="description-td">TGCGGGGGAGCTTGTGGCACAG</td>
                <td class="description-td">(50)</td>
                <td class="description-td">170</td>
                <td class="description-td">3</td>
              </tr> <tr class="description-tr"> <td class="description-td">5</td>
                <td class="description-td">TCAGCCCCACACCATCTCCTTC</td>
                <td class="description-td">(51)</td>
                <td class="description-td">CTGGGCCCCTACCCTAACCC</td>
                <td class="description-td">(52)</td>
                <td class="description-td">154</td>
                <td class="description-td">3</td>
              </tr> <tr class="description-tr"> <td class="description-td">6</td>
                <td class="description-td">TCCTGGAGCCCGACACTGTGTGT</td>
                <td class="description-td">(53)</td>
                <td class="description-td">TGTCCTGCCCACTCCTCAGCCT</td>
                <td class="description-td">(54)</td>
                <td class="description-td">238</td>
                <td class="description-td">2</td>
              </tr> <tr class="description-tr"> <td class="description-td">7</td>
                <td class="description-td">TGGCTGACCACTGTCCCTCT</td>
                <td class="description-td">(55)</td>
                <td class="description-td">CCCCAGGACCCCAGCTGTCCAA</td>
                <td class="description-td">(56)</td>
                <td class="description-td">195</td>
                <td class="description-td">3</td>
              </tr> <tr class="description-tr"> <td class="description-td">8</td>
                <td class="description-td">GCTGGCAGTGGCCTGTGTGGA</td>
                <td class="description-td">(57)</td>
                <td class="description-td">AACAGTGACCAAAATGACAGTGAC</td>
                <td class="description-td">(58)</td>
                <td class="description-td">191</td>
                <td class="description-td">3</td>
              </tr> <tr class="description-tr"> <td class="description-td">9</td>
                <td class="description-td">TGGCTCAGCAGGTGACAGC</td>
                <td class="description-td">(59)</td>
                <td class="description-td">TGGTGGCAGGTGGGCTACT</td>
                <td class="description-td">(60)</td>
                <td class="description-td">185</td>
                <td class="description-td">1</td>
              </tr> <tr class="description-tr"> <td class="description-td">10</td>
                <td class="description-td">GCCTGGCAGACGATGTCCA</td>
                <td class="description-td">(61)</td>
                <td class="description-td">CAACTGCCTGAGGGGTTCT</td>
                <td class="description-td">(62)</td>
                <td class="description-td">216</td>
                <td class="description-td">1</td>
              </tr> <tr class="description-tr"> <td class="description-td">11</td>
                <td class="description-td">CTGTCCCCACACTTTCTCCT</td>
                <td class="description-td">(63)</td>
                <td class="description-td">TGAGCTCCAGTCCCCTCCAG</td>
                <td class="description-td">(64)</td>
                <td class="description-td">195</td>
                <td class="description-td">1</td>
              </tr> <tr class="description-tr"> <td class="description-td">12</td>
                <td class="description-td">TGGCCACTCACAATCTCCT</td>
                <td class="description-td">(65)</td>
                <td class="description-td">GCCTTGACACCCTCCACTA</td>
                <td class="description-td">(66)</td>
                <td class="description-td">222</td>
                <td class="description-td">1</td>
              </tr> <tr class="description-tr"> <td class="description-td">13</td>
                <td class="description-td">GGCACAGGGAGGAGAAGTG</td>
                <td class="description-td">(67)</td>
                <td class="description-td">CGGCACCGCTGATCATGCA</td>
                <td class="description-td">(68)</td>
                <td class="description-td">216</td>
                <td class="description-td">1</td>
              </tr> <tr class="description-tr"> <td class="description-td">14</td>
                <td class="description-td">CCAGGGCCAGGTGTGACTG</td>
                <td class="description-td">(69)</td>
                <td class="description-td">TGGGCCCAGAGTAACTGACA</td>
                <td class="description-td">(70)</td>
                <td class="description-td">119</td>
                <td class="description-td">2</td>
              </tr> <tr class="description-tr"> <td class="description-td">15</td>
                <td class="description-td">GGCCCTGATTTGGGTGTTTTA</td>
                <td class="description-td">(71)</td>
                <td class="description-td">GGACGCTAACCAGAACCAC</td>
                <td class="description-td">(72)</td>
                <td class="description-td">135</td>
                <td class="description-td">2</td>
              </tr> <tr class="description-tr"> <td class="description-td">16</td>
                <td class="description-td">CACCACTGACTCTCTCGTCT</td>
                <td class="description-td">(73)</td>
                <td class="description-td">CCATCCCCCAGCCCCATC</td>
                <td class="description-td">(74)</td>
                <td class="description-td">297</td>
                <td class="description-td">2</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="7" align="center" rowsep="1" class="description-td" colspan="7"> </td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="7" align="left" class="description-td" colspan="7"> <sup>a</sup>SEQ ID NO is shown in parentheses following each sequence </td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="7" align="left" class="description-td" colspan="7"> <sup>b</sup>conditions of the PCR as described in Example 10D </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p>A voltage-dependent, outward K<sup>+ </sup>current was observed after membrane depolarization to potentials above −60 mV (FIG. <b>7</b>A). This current reached a steady state within 1 second at +40 mV. Activation of the current was preceded by a brief delay, and repolarization to −70 mV elicited a tail current with an initial increase in amplitude (a hook) before deactivation. Similar tail current hooks were previously observed for HERG K<sup>+ </sup>channels, and were attributed to recovery of channels from inactivation at a rate faster than deactivation (Sanguinetti et al., 1995; Smith et al., 1996; Spector et al., 1996). The activation curve for KVLQT1 current was half-maximal maximal (V<sub>½</sub>) at −11.6±0.6 mV, and had a slope factor of 12.6±0.5 mV (n=6; FIG. <b>7</b>B).</p>
    <p>The biophysical properties of KVLQT1 were unlike other known cardiac K<sup>+ </sup>currents. It was hypothesized that KVLQT1 might coassemble with another subunit to form a known cardiac channel. The slowly activating delayed rectifier K<sup>+ </sup>current, I<sub>Ks</sub>, modulates repolarization of cardiac action potentials. Despite intensive study, the molecular structure of the I<sub>Ks </sub>channel is not understood. Physiological data suggest that one component of the I<sub>Ks </sub>channel is minK (Goldstein and Miller, 1991; Hausdorff et al., 1991; Takumi et al., 1991; Busch et al., 1992; Wang and Goldstein, 1995; Wang et al., 1996), a 130 amino acid protein with a single putative transmembrane domain (Takumi et al., 1988). The size and structure of this protein, however, have led to doubt that minK alone forms functional channels (Attali et al., 1993; Lesage et al., 1993).</p>
    <p>To test this hypothesis, CHO cells were cotransfected with KVLQT1 and human KCNE1 cDNAs. A KCNE1 cDNA was subcloned in pCEP4 (InVitrogen) and transfection was performed as described above for KVLQT1 alone. For the cotransfection of KVLQT1 and KCNE1, 0.75 μg of each cDNA was used. As reported previously (Lesage et al., 1993), transfection of CHO cells with KCNE1 alone did not induce detectable current (n=10, FIG. <b>7</b>C). Cotransfection of KCNE1 with KVLQT1 induced a slowly activating delayed-rectifier current that was much larger than the current in cells transfected with KVLQT1 alone (FIGS. <b>7</b>D and <b>7</b>E). The slow activation of current in cotransfected CHO cells was preceded by a delay that lasted several hundred msec, indicating that no significant homomeric KVLQT1 channel current was present. Current did not saturate during long depolarizing pulses, and required a three-exponential function to best describe the initial delay and two phases of current activation. During a 30 sec depolarizing pulse to +40 mV, current was activated with time constants of 0.68±0.18, 1.48±0.16, and 8.0±0.6 sec (n=4). The isochronal (7.5 sec) activation curve for current had a V<sub>½ </sub>of 7.5±0.9 mV, and a slope factor of 16.5±0.8 mV (n=7; FIG. <b>9</b>B). By comparison, the V<sub>½ </sub>and slope of the activation curve for human cardiac I<sub>Ks </sub>are 9.4 mV and 11.8 mV (Li et al., 1996). Like KVLQT1 and hminK coexpressed in CHO cells, activation of cardiac I<sub>Ks </sub>is extremely slow and was best described by a three-exponential function (Balser et al., 1990; Sanguinetti and Jurkiewicz, 1990). Quinidine (50 μM) blocked tail currents in cotransfected CHO cells by 30±8% (n=5), similar to its effect (40-50% block) on I<sub>Ks </sub>in isolated myocytes (Balser et al, 1991). Thus, coexpression of KVLQT1 and hminK in CHO cells induced a K<sup>+ </sup>current with biophysical properties nearly identical to cardiac I<sub>Ks</sub>.</p>
    <p>To characterize the properties of hminK and KVLQT1 further, these channels were expressed separately and together in Xenopus oocytes. <i>Xenopus laevis </i>oocytes were isolated and injected with cRNA as described by Sanguinetti et al. (1995). KVLQT1 cDNA was subcloned into pSP64 (Promega). KCNE1 cDNA was a gift from R. Swanson. Roughly equimolar concentrations of KVLQT1 cRNA (5.8 ng per oocyte) and KCNE1 (1 ng per oocyte) cRNA were used for the co-injection experiments. The bathing solution contained, in mM: 98 NaCl, 2 KCl, 2 MgCl<sub>2</sub>, 0.1 CaCl<sub>2</sub>, and 5 HEPES (pH 7.6, 22-25° C.). For reversal-potential experiments, osmolarity was maintained by equimolar substitution of external NaCl for KCl. Currents were recorded using standard two-microelectrode voltage clamp techniques 3 days after injection of oocytes with cRNA (Sanguinetti et al., 1995). Currents were filtered at 0.5 kHz and digitized at 2 kHz. Data are presented as mean±s.e.m.</p>
    <p>Oocytes injected with KVLQT1 complementary RNA expressed a rapidly activating outward K<sup>+ </sup>current with a voltage dependence of activation nearly identical to CHO cells transfected with KVLQT1 cDNA (FIGS. <b>8</b>A and <b>8</b>B). The K<sup>+ </sup>selectivity of KVLQT1 channels was determined by measuring the reversal potential (E<sub>rev</sub>) of tail currents in different concentrations of extracellular K ([K<sup>+</sup>]<sub>e</sub>). The slope of the relationship between E<sub>rev </sub>and log[K<sup>+</sup>]<sub>e </sub>was 49.9±0.4 mV (n=7; FIG. <b>8</b>C), significantly less than predicted by the Nernst equation (58 mV) for a perfectly selective K<sup>+ </sup>channel. Co-injection of oocytes with KVLQT1 and KCNE1 cRNA induced a current similar to I<sub>Ks </sub>(FIG. <b>9</b>C). The slope of the relationship between E<sub>rev </sub>and log[K<sup>+</sup>]<sub>e </sub>for co-injected oocytes was 49.9±4 mV (n=6), similar to KVLQT1 alone and to guinea pig cardiac I<sub>Ks </sub>(49 mV) (Matsuura et al., 1987). The isochronal (7.5 sec) activation curve for co-injected oocytes had a V<sub>½ </sub>of 6.2 mV and a slope of 12.3 mV (FIG. <b>9</b>E), similar to cardiac I<sub>Ks</sub>.</p>
    <heading>EXAMPLE 12</heading> <p>Identification of a KVLQT1 Gene in Xenopus</p>
    <p>By contrast with CHO cells, KCNE1 was able to undergo functional expression in Xenopus oocytes (FIG. <b>9</b>B). The induced current (I<sub>sK</sub>) was smaller than the current induced in co-injected oocytes, but the kinetics and voltage dependence of activation were similar (FIGS. 9 A-E). Two observations have led to the hypothesis that I<sub>sK </sub>in Xenopus oocytes results from channels formed by coassembly of minK with an unidentified, constitutively expressed subunit. First, the magnitude of I<sub>sK </sub>saturates after injection of very small amounts of KCNE1 cRNA (FIG. <b>9</b>D), suggesting that an endogenous component of limited quantity is required for functional expression (Wang and Goldstein, 1995; Cui et al., 1994). Second, heterologous expression of minK in mammalian cells does not induce detectable current (Lesage et al., 1993) (FIG. <b>7</b>C), suggesting that minK is not sufficient to form functional channels. It was hypothesized that this unidentified subunit might be a homologue of KVLQT1. To test this hypothesis, a Xenopus oocyte cDNA library (Clontech) was screened with a KVLQT1 cDNA clone spanning the S3-S5 domains. A 1.6 kb partial clone (XKVLQT1, FIG. 10A) was isolated. XKVLQT1 is 88% identical at the amino acid level with the corresponding region of KVLQT1 (FIG. <b>10</b>A). These data suggest that I<sub>sK </sub>results from the coassembly of the XKVLQT1 and minK proteins.</p>
    <p>It was concluded that KVLQT1 and hminK coassemble to form the cardiac I<sub>Ks </sub>channel. Two delayed-rectifier K<sup>+ </sup>currents, I<sub>Kr </sub>and I<sub>Ks</sub>, modulate action-potential duration in cardiac myocytes (Li et al., 1996; Sanguinetti and Jurkiewicz, 1990). Previous studies have implicated dysfunction of I<sub>Kr </sub>channels in long QT syndrome (Sanguinetti et al., 1995; Curran et al., 1995; Sanguinetti et al., 1996a). The observation that KVLQT1 mutations also cause this disorder (Wang et al., 1996), and the discovery that KVLQT1 forms part of the I<sub>Ks </sub>channel, indicate that dysfunction of both cardiac delayed-rectifier K<sup>+ </sup>channels contribute to risk of sudden death from cardiac arrhythmia.</p>
    <heading>EXAMPLE 13</heading> <p>Cosegregation of KVLQT1 Missense Mutations with LQT in Six Large Families</p>
    <p>To test the hypothesis that KVLQT1 is LQT1, single-strand conformational polymorphism (SSCP) analyses were used to screen for functional mutations in affected members of K1532, the largest LQT family that showed linkage to chromosome 11. SSCP was carried out as previously described (Wang et al., 1995a; Wang et al., 1995b). Normal and aberrant SSCP products were isolated and sequenced directly as described (Wang and Keating, 1994) or subcloned into pBluescript (SK<sup>+</sup>) (Stratagene) using the T-vector method (Marchuk et al., 1991). When the latter method was used, several clones were sequenced by the dideoxy chain termination method using Sequenase™ Version 2.0 (United States Biochemicals, Inc.). Analyses were focused on the region between S2 and S6 since these regions might be important for KVLQT1 function. We designed oligonucleotide primers based on cDNA sequences and used these primers for cycle sequencing reactions with the KVLQT1-containing P1, 18B12 (Wang and Keating, 1994). These experiments defined intronic sequences flanking exons encoding S2-S6. Additional primers were then generated from these intronic sequences and used for SSCP analyses (Table 5).</p>
    <p>SSCP analyses identified an anomalous conformer in the 70 affected members of K1532 (FIG. <b>11</b>A). This aberrant conformer was not observed in the 147 unaffected members of this kindred or in genomic DNA from more than 200 unrelated control individuals. The two-point LOD score for linkage between this anomaly and LQT was 14.19 at a recombination fraction of 0 (Table 2). No recombination was observed between KVLQT1 and LQT1, indicating that these loci are completely linked. DNA sequence analyses of the normal and aberrant SSCP conformers revealed a single base substitution, a G to A transition, at the first nucleotide of codon Val-125 (FIG. <b>11</b>A and Table 6). This mutation results in a valine to methionine substitution in the predicted intracellular domain between S4 and S5.</p>
    <p>To further test the hypothesis that mutations in KVLQT1 cause LQT, DNA samples from affected members of five additional large LQT kindreds were studied. Linkage analyses with polymorphic markers from this region had shown that the disease phenotype was linked to chromosome 11 in these families. Aberrant SSCP conformers were identified in affected members of K2605, K1723, K1807 (FIGS. <b>11</b>B-D), K161 and K162. The SSCP anomalies identified in K161 and K162 were identical to that observed in K1807. The aberrant SSCP conformer was not seen in unaffected members of these kindreds or in DNA samples from more than 200 unrelated control individuals. The normal and aberrant conformers identified in each family were sequenced. The nucleotide change, coding effect, and location of each mutation are summarized in Table 6.</p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" pgwide="1" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="259pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 5</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">PCR Primers Used to Define KVLQT1 Mutations</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="4"> <colspec colname="1" colwidth="35pt" align="center"> </colspec> <colspec colname="2" colwidth="126pt" align="left"> </colspec> <colspec colname="3" colwidth="56pt" align="center"> </colspec> <colspec colname="4" colwidth="42pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Primer</td>
                <td class="description-td">Sequence</td>
                <td class="description-td">Region Amplified</td>
                <td class="description-td">SEQ ID NO:</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="4" align="center" rowsep="1" class="description-td" colspan="4"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="4"> <colspec colname="1" colwidth="35pt" align="char" char="."> </colspec> <colspec colname="2" colwidth="126pt" align="left"> </colspec> <colspec colname="3" colwidth="56pt" align="center"> </colspec> <colspec colname="4" colwidth="42pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">1</td>
                <td class="description-td">GAGATCGTGCTGGTGGTGTTCT</td>
                <td class="description-td">S2-S3</td>
                <td class="description-td">75</td>
              </tr> <tr class="description-tr"> <td class="description-td">2</td>
                <td class="description-td">CTTCCTGGTCTGGAAACCTGG</td>
                <td class="description-td"> </td>
                <td class="description-td">76</td>
              </tr> <tr class="description-tr"> <td class="description-td">3</td>
                <td class="description-td">CTCTTCCCTGGGGCCCTGGC</td>
                <td class="description-td">S3-S4</td>
                <td class="description-td">77</td>
              </tr> <tr class="description-tr"> <td class="description-td">4</td>
                <td class="description-td">TGCGGGGGAGCTTGTGGCACAG</td>
                <td class="description-td"> </td>
                <td class="description-td">78</td>
              </tr> <tr class="description-tr"> <td class="description-td">5</td>
                <td class="description-td">GGGCATCCGCTTCCTGCAGA</td>
                <td class="description-td">S4</td>
                <td class="description-td">79</td>
              </tr> <tr class="description-tr"> <td class="description-td">6</td>
                <td class="description-td">CTGGGCCCCTACCCTAACCC</td>
                <td class="description-td"> </td>
                <td class="description-td">80</td>
              </tr> <tr class="description-tr"> <td class="description-td">7</td>
                <td class="description-td">TCCTGGAGCCCGAACTGTGTGT</td>
                <td class="description-td">S5-Pore</td>
                <td class="description-td">81</td>
              </tr> <tr class="description-tr"> <td class="description-td">8</td>
                <td class="description-td">TGTCCTGCCCACTCCTCAGCCT</td>
                <td class="description-td"> </td>
                <td class="description-td">82</td>
              </tr> <tr class="description-tr"> <td class="description-td">9</td>
                <td class="description-td">CCCCAGGACCCCAGCTGTCCAA</td>
                <td class="description-td">Pore-S6</td>
                <td class="description-td">83</td>
              </tr> <tr class="description-tr"> <td class="description-td">10</td>
                <td class="description-td">AGGCTGACCACTGTCCCTCT</td>
                <td class="description-td"> </td>
                <td class="description-td">84</td>
              </tr> <tr class="description-tr"> <td class="description-td">11</td>
                <td class="description-td">GCTGGCAGTGGCCTGTGTGGA</td>
                <td class="description-td">S6</td>
                <td class="description-td">85</td>
              </tr> <tr class="description-tr"> <td class="description-td">12</td>
                <td class="description-td">AACAGTGACCAAAATGACAGTGAC</td>
                <td class="description-td"> </td>
                <td class="description-td">86</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="4" align="center" rowsep="1" class="description-td" colspan="4"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" pgwide="1" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="266pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 6</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Summary of KVLQT1 Mutations</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="7"> <colspec colname="1" colwidth="35pt" align="center"> </colspec> <colspec colname="2" colwidth="49pt" align="center"> </colspec> <colspec colname="3" colwidth="35pt" align="center"> </colspec> <colspec colname="4" colwidth="49pt" align="center"> </colspec> <colspec colname="5" colwidth="42pt" align="center"> </colspec> <colspec colname="6" colwidth="28pt" align="left"> </colspec> <colspec colname="7" colwidth="28pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Nucleotide</td>
                <td class="description-td">Coding</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">No. of</td>
              </tr> <tr class="description-tr"> <td class="description-td">Codon</td>
                <td class="description-td">change</td>
                <td class="description-td">effect</td>
                <td class="description-td">Mutation</td>
                <td class="description-td">Region</td>
                <td class="description-td">Kindred</td>
                <td class="description-td">affected</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="7" align="center" rowsep="1" class="description-td" colspan="7"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="7"> <colspec colname="1" colwidth="35pt" align="center"> </colspec> <colspec colname="2" colwidth="49pt" align="center"> </colspec> <colspec colname="3" colwidth="35pt" align="center"> </colspec> <colspec colname="4" colwidth="49pt" align="center"> </colspec> <colspec colname="5" colwidth="42pt" align="center"> </colspec> <colspec colname="6" colwidth="28pt" align="left"> </colspec> <colspec colname="7" colwidth="28pt" align="char" char="."> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">167-168</td>
                <td class="description-td">ΔTCG</td>
                <td class="description-td">Deletion</td>
                <td class="description-td">F167W/G168Δ</td>
                <td class="description-td">S2</td>
                <td class="description-td">K13216</td>
                <td class="description-td">1</td>
              </tr> <tr class="description-tr"> <td class="description-td">178</td>
                <td class="description-td"> <u>G</u>CC to <u>C</u>CC</td>
                <td class="description-td">Missense</td>
                <td class="description-td">A178P</td>
                <td class="description-td">S2-S3</td>
                <td class="description-td">Kl3119</td>
                <td class="description-td">1</td>
              </tr> <tr class="description-tr"> <td class="description-td">189</td>
                <td class="description-td"> <u>G</u>GG to <u>A</u>GG</td>
                <td class="description-td">Missense</td>
                <td class="description-td">G189R</td>
                <td class="description-td">S2-S3</td>
                <td class="description-td">K2557</td>
                <td class="description-td">3</td>
              </tr> <tr class="description-tr"> <td class="description-td">190</td>
                <td class="description-td">C<u>G</u>G to C<u>A</u>G</td>
                <td class="description-td">Missense</td>
                <td class="description-td">R190Q</td>
                <td class="description-td">S2-S3</td>
                <td class="description-td">K15019</td>
                <td class="description-td">2</td>
              </tr> <tr class="description-tr"> <td class="description-td">254</td>
                <td class="description-td"> <u>G</u>TG to <u>A</u>TG</td>
                <td class="description-td">Missense</td>
                <td class="description-td">V254M</td>
                <td class="description-td">S4-S5</td>
                <td class="description-td">K1532</td>
                <td class="description-td">70</td>
              </tr> <tr class="description-tr"> <td class="description-td">273</td>
                <td class="description-td"> <u>C</u>TC to <u>T</u>TC</td>
                <td class="description-td">Missense</td>
                <td class="description-td">L273F</td>
                <td class="description-td">S5</td>
                <td class="description-td">K1777</td>
                <td class="description-td">2</td>
              </tr> <tr class="description-tr"> <td class="description-td">306</td>
                <td class="description-td"> <u>G</u>GG to <u>A</u>GG</td>
                <td class="description-td">Missense</td>
                <td class="description-td">G306R</td>
                <td class="description-td">Pore</td>
                <td class="description-td">K20926</td>
                <td class="description-td">1</td>
              </tr> <tr class="description-tr"> <td class="description-td">312</td>
                <td class="description-td">A<u>C</u>C to A<u>T</u>C</td>
                <td class="description-td">Missense</td>
                <td class="description-td">T312I</td>
                <td class="description-td">Pore</td>
                <td class="description-td">K20925</td>
                <td class="description-td">1</td>
              </tr> <tr class="description-tr"> <td class="description-td">341</td>
                <td class="description-td">G<u>C</u>G to G<u>A</u>G</td>
                <td class="description-td">Missense</td>
                <td class="description-td">A341E</td>
                <td class="description-td">S6</td>
                <td class="description-td">K1723</td>
                <td class="description-td">6</td>
              </tr> <tr class="description-tr"> <td class="description-td">341</td>
                <td class="description-td">G<u>C</u>G to G<u>A</u>G</td>
                <td class="description-td">Missense</td>
                <td class="description-td">A341E</td>
                <td class="description-td">S6</td>
                <td class="description-td">K2050</td>
                <td class="description-td">2</td>
              </tr> <tr class="description-tr"> <td class="description-td">341</td>
                <td class="description-td">G<u>C</u>G to G<u>T</u>G</td>
                <td class="description-td">Missense</td>
                <td class="description-td">A341V</td>
                <td class="description-td">S6</td>
                <td class="description-td">K1807</td>
                <td class="description-td">6</td>
              </tr> <tr class="description-tr"> <td class="description-td">341</td>
                <td class="description-td">G<u>C</u>G to G<u>T</u>G</td>
                <td class="description-td">Missense</td>
                <td class="description-td">A341V</td>
                <td class="description-td">S6</td>
                <td class="description-td">K161</td>
                <td class="description-td">18</td>
              </tr> <tr class="description-tr"> <td class="description-td">341</td>
                <td class="description-td">G<u>C</u>G to G<u>T</u>G</td>
                <td class="description-td">Missense</td>
                <td class="description-td">A341V</td>
                <td class="description-td">S6</td>
                <td class="description-td">K162</td>
                <td class="description-td">18</td>
              </tr> <tr class="description-tr"> <td class="description-td">341</td>
                <td class="description-td">G<u>C</u>G to G<u>T</u>G</td>
                <td class="description-td">Missense</td>
                <td class="description-td">A341V</td>
                <td class="description-td">S6</td>
                <td class="description-td">K163</td>
                <td class="description-td">3</td>
              </tr> <tr class="description-tr"> <td class="description-td">341</td>
                <td class="description-td">G<u>C</u>G to G<u>T</u>G</td>
                <td class="description-td">Missense</td>
                <td class="description-td">A341V</td>
                <td class="description-td">S6</td>
                <td class="description-td">K164</td>
                <td class="description-td">2</td>
              </tr> <tr class="description-tr"> <td class="description-td">345</td>
                <td class="description-td">G<u>G</u>G to G<u>A</u>G</td>
                <td class="description-td">Missense</td>
                <td class="description-td">G345E</td>
                <td class="description-td">S6</td>
                <td class="description-td">K2605</td>
                <td class="description-td">11</td>
              </tr> <tr class="description-tr"> <td class="description-td">168</td>
                <td class="description-td"> <u>G</u>GG to <u>A</u>GG</td>
                <td class="description-td">Missense</td>
                <td class="description-td">G168R</td>
                <td class="description-td">S2</td>
                <td class="description-td">K2625</td>
                <td class="description-td">—</td>
              </tr> <tr class="description-tr"> <td class="description-td">168</td>
                <td class="description-td"> <u>G</u>GG to <u>A</u>GG</td>
                <td class="description-td">Missense</td>
                <td class="description-td">G168R</td>
                <td class="description-td">S2</td>
                <td class="description-td">K2673</td>
                <td class="description-td">—</td>
              </tr> <tr class="description-tr"> <td class="description-td">168</td>
                <td class="description-td"> <u>G</u>GG to <u>A</u>GG</td>
                <td class="description-td">Missense</td>
                <td class="description-td">G168R</td>
                <td class="description-td">S2</td>
                <td class="description-td">K3698</td>
                <td class="description-td">—</td>
              </tr> <tr class="description-tr"> <td class="description-td">314</td>
                <td class="description-td"> <u>G</u>GC to <u>A</u>GC</td>
                <td class="description-td">Missense</td>
                <td class="description-td">G314S</td>
                <td class="description-td">Pore</td>
                <td class="description-td">K19187</td>
                <td class="description-td">—</td>
              </tr> <tr class="description-tr"> <td class="description-td">315</td>
                <td class="description-td">T<u>A</u>T to T<u>G</u>T</td>
                <td class="description-td">Missense</td>
                <td class="description-td">Y315C</td>
                <td class="description-td">Pore</td>
                <td class="description-td">K22709</td>
                <td class="description-td">—</td>
              </tr> <tr class="description-tr"> <td class="description-td">318</td>
                <td class="description-td">AA<u>G</u> to AA<u>C</u> </td>
                <td class="description-td">Missense</td>
                <td class="description-td">K318N</td>
                <td class="description-td">Pore</td>
                <td class="description-td">K2762</td>
                <td class="description-td">—</td>
              </tr> <tr class="description-tr"> <td class="description-td">353</td>
                <td class="description-td">C<u>T</u>G to C<u>C</u>G</td>
                <td class="description-td">Missense</td>
                <td class="description-td">L353P</td>
                <td class="description-td">S6</td>
                <td class="description-td">K3401</td>
                <td class="description-td">—</td>
              </tr> <tr class="description-tr"> <td class="description-td">366</td>
                <td class="description-td"> <u>C</u>GG to <u>T</u>GG</td>
                <td class="description-td">Missense</td>
                <td class="description-td">R366W</td>
                <td class="description-td">C-terminus</td>
                <td class="description-td">K2824</td>
                <td class="description-td">—</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="7" align="center" rowsep="1" class="description-td" colspan="7"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <heading>EXAMPLE 14</heading> <p>A KVLQT1 Intragenic Deletion and Fifteen Missense Mutations Associated with LQT in Small Families and Sporadic Cases</p>
    <p>To identify additional LQT-associated mutations in KVLQT1, further SSCP analyses were performed for small kindreds and sporadic cases. SSCP revealed an aberrant conformer in kindred 13216 (FIG. <b>12</b>A). Analyses of more than 200 unrelated control individuals failed to show this anomaly. This aberrant conformer was cloned and sequenced, revealing a three base pair deletion encompassing codons 167 and 168. This mutation results in a phenylalanine to tryptophan substitution and deletion of a glycine in the putative S2 domain (Table 6).</p>
    <p>Aberrant SSCP conformers were identified in affected members of additional kindreds. An aberrant SSCP conformer identified in K2050 was identical to that in K1723, and aberrant conformers identified in K161, K162, K163 and K164 were identical to that observed in K1807. Also kindreds 2625, 2673 and 3698 had the identical mutation. None of the aberrant conformers was identified in DNA samples from more than 200 control individuals. In each case, the normal and aberrant conformers were sequenced. These data are shown in FIGS. 12A-O and summarized in Table 6. In total, KVLQT1 mutations associated with LQT in 24 families or sporadic cases were identified, providing strong molecular genetic evidence that mutations in KVLQT1 cause the chromosome 11-linked form of LQT.</p>
    <heading>EXAMPLE 15</heading> <p>KCNE1 Variations Which Result in LQT</p>
    <p>Separate studies on different individuals were performed in finding variants of minK. These studies were performed using the following methods.</p>
    <p>A. Phenotypic Analyses</p>
    <p>Individuals were phenotypically characterized based on the QT interval corrected for heart rate. Individuals were characterized as affected if QTc≧0.46 second. Individuals were assigned as unaffected if QTc≦0.42 second. Informed consent was obtained from all individuals or their guardians in accordance with local institutional review board guidelines. Phenotypic data were interpreted without knowledge of genotype.</p>
    <p>B. Mutation Analyses</p>
    <p>Genomic samples were amplified by PCR using the following primer pairs:</p>
    <p>
      <maths> <formula-text>MINK1F-5′-CTGCAGCAGTGGAACCTTAATG-3′  (SEQ ID NO:87)</formula-text> </maths> </p>
    <p>and</p>
    <p> MINK1R-5′-GTTCGAGTGCTCCAGCTTCTTG-3′  (SEQ ID NO:88);</p>
    <p>
      <maths> <formula-text>MINK2F-5′-AGGGCATCATGCTGAGCTACAT-3′  (SEQ ID NO:89)</formula-text> </maths> </p>
    <p>and</p>
    <p>
      <maths> <formula-text>MINK2R-5′-TTTAGCCAGTGGTGGGGTTCA-3′  (SEQ ID NO:90);</formula-text> </maths> </p>
    <p>
      <maths> <formula-text>MINK3F-5′-GTTCAGCAGGGTGGCAACAT-3′  (SEQ ID NO:91)</formula-text> </maths> </p>
    <p>and</p>
    <p>
      <maths> <formula-text>MINK3R-5′-GCCAGATGGTTTTCAACGACA-3′  (SEQ ID NO:92).</formula-text> </maths> </p>
    <p>PCR products were used in SSCP analysis as described (K W Wang et al., 1996). PCR was completed with 75 ng DNA in a volume of 10 μL using a Perkin-Elmer Cetus 9600 thermocycler. Amplification conditions were 94° C. for 3 minutes followed by 30 cycles of 94° C. for 10 seconds, 58° C. for 20 seconds, 72° C. for 20 seconds and a 5 minute extension at 72° C. Reactions were diluted with 40 μL of 0.1% SDS/10 mM EDTA and with 30 μL of 95% formamide load dye. The mixture was denatured at 94° C. for 5 minutes and placed on ice. Three microliters of each sample were separated on 5% and 10% non-denaturing polyacrylamide gels acrylamide:bisacrylamide 49:1) at 4° C. and on 0.5× and 1×MDE (mutation detection enhancement) gels (FMC BioProducts) at room temperature. Electrophoreses on the 5% and 10% gels were completed at 40 W for 3-5 hours; electrophoreses on 0.5× and 1×MDE gels were completed overnight, respectively, at 350 V and 600 V. Gels were dried on 3 MM filter paper and exposed to film for 18 hours at −70° C.</p>
    <p>SSCP bands were cut out of the gel and eluted in 100 μL double distilled water at 65° C. for 30 minutes. Ten microliters of eluted DNA was reamplified using the original primer pair. Products were separated on 1% low melting temperature agarose gels (FMC), phenol-chloroform extracted and ethanol precipitated. DNA was sequenced in both directions by the dideoxy chain termination method on an Applied Biosystems model 373A DNA sequencer.</p>
    <p>C. Functional Expression</p>
    <p>KCNE1 cDNA expression constructs were amplified by PCR from total human DNA and cloned in pSP64 transcription vector (Promega) using the following primers:</p>
    <p>
      <maths> <formula-text>MINKF-5′-CAGTGG<u>AAGCTT</u>AATGCCCAGGATGATC-3′  (SEQ ID NO:93)</formula-text> </maths> </p>
    <p>and</p>
    <p>
      <maths> <formula-text>MINKR-5′-CAGGA<u>GGATCC</u>AGTTTAGCCAGTGGTGGGGGTTCA-3′  (SEQ ID NO:94).</formula-text> </maths> </p>
    <p>Nucleotides in bold denote the changes made to create Hind III and BamH I restriction sites (underlined). A full-length KVLQT1 cDNA clone (identical to that reported by Yang et al. (1997)) was isolated from a human cardiac cDNA library and subcloned into the pSP64 plasmid expression vector. All constructs were confirmed by DNA sequence analyses. Complementary RNAs were synthesized using the mCAP RNA capping kit (Stratagene).</p>
    <p>Isolation of <i>Xenopus laevis </i>oocytes and cRNA injection were performed as described (Sanguinetti et al., 1995). Voltage clamp data were acquired and analyzed using PCLAMP v6.0 software (Axon Instruments). Isochronal (7.5 seconds) rather than steady state measurements were used to estimate the voltage dependence of I<sub>Ks </sub>activation. The voltage-dependence of I<sub>Ks </sub>activation was determined by fitting peak tail currents to a Boltzmann function. V<sub>½</sub>, the voltage at which the current was half-activated using this pulse protocol, and the slope factor, were calculated from these data. Activating current was fitted to a biexponential function to obtain slow and fast time constants of activation. Deactivation time constants were obtained by fitting decaying tail currents at various test potentials to a single exponential function.</p>
    <p>All data are mean±S.E.M. Statistical analyses were performed using repeated measures analysis of variance, with the Fisher's Least Significance post hoc test and the unpaired Student's T-test. A p value&lt;0.05 was considered statistically significant.</p>
    <p>D. Results</p>
    <p>Ion channel β subunits are ancillary proteins that coassemble with α subunits to modulate the gating kinetics and enhance stability of multimeric channel complexes (Rettig et al., 1994; Shi et al., 1996). Despite their functional importance, dysfunction of potassium β subunits has not been associated with disease. Recent physiologic studies suggest that KCNE1 encodes β subunits that coassemble with KvLQT1 α subunits to form the slowly activating delayed rectifier K<sup>+ </sup>(I<sub>Ks</sub>) channel (Sanguinetti et al., 1996b; Barhanin et al., 1996). Because KVLQT1 mutations cause arrhythmia susceptibility in the long QT syndrome (LQT) (Q. Wang et al., 1996; Neyroud et al., 1997; Splawski et al., 1997a), we hypothesized that mutations in KCNE1 also cause this disorder. Here KCNE1 missense mutations are defined in affected members of two LQT families. Both mutations (S74L, D76N) reduced I<sub>Ks </sub>by shifting the voltage dependence of activation and accelerating channel deactivation. D76N hminK also had a dominant negative effect. The functional consequences of these mutations would be delayed cardiac repolarization and an increased risk of arrhythmia. These data establish KCNE1 as an LQT gene and confirm that hminK is an integral protein of the I<sub>Ks </sub>channel.</p>
    <p>Individuals with LQT have been ascertained and phenotypically characterized (Keating et al., 1991 a; Jiang et al., 1994). Single strand conformation polymorphism (SSCP) analyses using primers that span KCNE1 led to the identification of an anomalous conformer in affected members of kindred 1789 (FIG. <b>13</b>A). This conformer was not observed in unaffected family members or in 200 unrelated control individuals (400 chromosomes). DNA sequence analysis revealed a G to A transition at the first nucleotide of codon 76, causing an Asp to Asn substitution (D76N) (FIG. <b>13</b>C). The sequences for KCNE1 cDNA and its protein product are listed here as SEQ ID NO:3 and SEQ ID NO:4, respectively. The first nucleotide of codon 76 is base 418 of SEQ ID NO:3.</p>
    <p>Further SSCP analyses defined a second anomaly that cosegregated with the disease in kindred 1754 (FIG. <b>13</b>B). This anomaly was not observed in unaffected members of the family or in 200 controls. DNA sequence analysis revealed a C to T transition in the second nucleotide of codon 74 (base 413 of SEQ ID NO:3), leading to substitution of Ser to Leu (S74L) (FIG. <b>13</b>C). Analyses of further DNA samples obtained from unrelated individuals with LQT revealed additional KCNE1 mutations. Table 7 lists the KCNE1 mutations found in LQT families.</p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 7</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Summary of KCNE1 Mutations</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="5"> <colspec colname="1" colwidth="28pt" align="center"> </colspec> <colspec colname="2" colwidth="70pt" align="left"> </colspec> <colspec colname="3" colwidth="49pt" align="left"> </colspec> <colspec colname="4" colwidth="42pt" align="left"> </colspec> <colspec colname="5" colwidth="28pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Codon</td>
                <td class="description-td">Nucleotide change</td>
                <td class="description-td">Coding effect</td>
                <td class="description-td">Mutation</td>
                <td class="description-td">Kindred</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="5" align="center" rowsep="1" class="description-td" colspan="5"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="5"> <colspec colname="1" colwidth="28pt" align="char" char="."> </colspec> <colspec colname="2" colwidth="70pt" align="left"> </colspec> <colspec colname="3" colwidth="49pt" align="left"> </colspec> <colspec colname="4" colwidth="42pt" align="left"> </colspec> <colspec colname="5" colwidth="28pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">28</td>
                <td class="description-td">T<u>C</u>G to T<u>T</u>G</td>
                <td class="description-td">Missense</td>
                <td class="description-td">S28L</td>
                <td class="description-td">1789</td>
              </tr> <tr class="description-tr"> <td class="description-td">32</td>
                <td class="description-td">C<u>G</u>C to C<u>A</u>C</td>
                <td class="description-td">Missense</td>
                <td class="description-td">R32H</td>
                <td class="description-td">2521</td>
              </tr> <tr class="description-tr"> <td class="description-td">74</td>
                <td class="description-td">T<u>C</u>G to T<u>T</u>G</td>
                <td class="description-td">Missense</td>
                <td class="description-td">S74L</td>
                <td class="description-td">1754</td>
              </tr> <tr class="description-tr"> <td class="description-td">76</td>
                <td class="description-td"> <u>G</u>AC to <u>A</u>AC</td>
                <td class="description-td">Missense</td>
                <td class="description-td">D76N</td>
                <td class="description-td">1789</td>
              </tr> <tr class="description-tr"> <td class="description-td">98</td>
                <td class="description-td"> <u>C</u>GG to <u>T</u>GG</td>
                <td class="description-td">Missense</td>
                <td class="description-td">R98W</td>
                <td class="description-td">2016</td>
              </tr> <tr class="description-tr"> <td class="description-td">127</td>
                <td class="description-td"> <u>C</u>CT to <u>G</u>CT</td>
                <td class="description-td">Missense</td>
                <td class="description-td">P127A</td>
                <td class="description-td">2016</td>
              </tr> <tr class="description-tr"> <td class="description-td">127</td>
                <td class="description-td"> <u>C</u>CT to <u>A</u>CT</td>
                <td class="description-td">Missense</td>
                <td class="description-td">P127T</td>
                <td class="description-td">2819</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="5" align="center" rowsep="1" class="description-td" colspan="5"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p>To determine the functional consequences of these KCNE1 mutations, we expressed mutant and wild-type (WT) proteins in Xenopus oocytes. Because the stoichiometry of KVLQT1 and minK interaction is not known, varying amounts of KCNE1 cRNA (0.01-2.5 ng/oocyte) were coinjected with a fixed quantity of KVLQT1 cRNA (6 ng/oocyte) and the resultant currents recorded. I<sub>Ks </sub>amplitude increased as a function of injected KCNE1, and saturated at KCNE1 cRNA levels ≧0.6 ng/oocyte (FIGS. <b>14</b>A-<b>14</b>B). Subsequent coexpression experiments were performed using 1.2 ng/oocyte KCNE1 and 6 ng/oocyte KVLQT1 cRNA, to insure that KCNE1 was not a limiting factor for expression of heteromultimeric channels.</p>
    <p>Coinjection of D76N KCNE1 and KVLQT1 cRNA failed to induce detectable K<sup>+ </sup>currents (n=13). Because LQT is inherited as an autosomal dominant trait, affected individuals possess one normal and one mutant KCNE1 allele. Therefore, mutant KCNE1 cRNA was coinjected with WT KCNE1 and KVLQT1 cRNA. The current (I<sub>Ks-D76</sub>N) induced by coinjection of D76N KCNE1 (0.6 ng/oocyte), WT KCNE1 (0.6 ng/oocyte) and KVLQT1 cRNA (6 ng/oocyte) was 91% smaller than the current (I<sub>Ks-WT</sub>) induced by WT KCNE1 (1.2 ng/oocyte) and KVLQT1 (6 ng/oocyte) cRNA at +40 mV (FIGS. <b>15</b>A and <b>15</b>B). Thee data indicate that D76N hminK subunits form heteromultimeric channels with WT hminK and KVLQT1, and reduce I<sub>Ks </sub>by a strong dominant-negative mechanism.</p>
    <p>To compare the biophysical properties of wild-type and mutant channels, the voltage dependence of activation and the kinetics of deactivation for I<sub>Ks-D76N </sub>and I<sub>Ks-WT </sub>were characterized. The magnitude of I<sub>Ks </sub>does not reach steady state even when elicited with pulses of 100 second duration (Swanson et al., 1993). Therefore, tail current amplitude following 7.5 second test pulses was used as an empirical measure of the voltage dependence of I<sub>Ks</sub>. I<sub>Ks-D76N </sub>tail currents were half-maximal at +28 mV, a +16 mV shift relative to I<sub>Ks-WT </sub>(FIG. <b>15</b>C). A shift in channel gating was confirmed by the voltage dependence of current deactivation. The rate of I<sub>Ks-D76N </sub>channel closure (deactivation) was faster than I<sub>Ks-WT </sub>at voltages ≧−80 mV (FIG. <b>15</b>D). The voltage dependence of the time constants of deactivation were shifted by approximately +30 mV. Thus, D76N hminK reduces I<sub>Ks </sub>by three mechanisms: a dominant negative suppression of channel function, an increased rate of channel deactivation and a positive shift in the voltage dependence of channel activation. These effects would reduce outward current during the repolarization phase and lengthen the duration of a cardiac action potential.</p>
    <p>Unlike D76N hminK, S74L hminK formed I<sub>Ks </sub>channels when coexpressed with KVLQT1, albeit with altered function. Current induced by injection of S74L KCNE1 (1.2 ng/oocyte) and KVLQT1 (6.0 ng/oocyte) cRNA had a threshold for activation that was approximately 40 mV higher than I<sub>Ks-WT</sub>. The resultant current was 66% smaller than I<sub>Ks-WT </sub>after 7.5 second pulses to +60 mV (n=15). When S74L KCNE1 (0.6 ng/oocyte) and WT KCNE1 (0.6 ng/oocyte) were coinjected with KVLQT1 (6.0 ng/oocyte) cRNA, the resultant current (I<sub>Ks-S74L</sub>) was reduced by approximately 33% at +60 mV compared to I<sub>Ks-WT </sub>(FIGS. <b>16</b>A-<b>16</b>B). As shown in FIG. 16C, this reduction was due primarily to a positive shift in the voltage dependence of current activation. The voltage dependence of deactivation was shifted approximately +40 mV (FIG. <b>16</b>D). This shift caused a marked increase in the rate of I<sub>Ks-S74L </sub>deactivation. Thus, S74L hminK subunits form heteromultimeric channels with WT hminK and KVLQT1, and reduce I<sub>Ks </sub>by a shift in the voltage dependence of channel activation and an increased rate of channel deactivation. Because I<sub>Ks-S74L </sub>did not equal I<sub>Ks-WT </sub>at +60 mV (as expected for a simple shift in gating), it is possible tat S74L mutant subunits also reduce the number of functional I<sub>Ks </sub>channels and/or single channel conductance.</p>
    <p>The observation that LQT-associated mutations of KCNE1 alter gating kinetics provides compelling evidence that hminK forms an integral part of the I<sub>Ks </sub>channel, rather than simply serving as a chaperone. Earlier studies of minK, performed before the discovery of KVLQT1, also support this conclusion (Takumi et al., 1991; Goldstein and Miller, 1991; Wang and Goldstein, 1995; K W Wang et al., 1996). In oe of these studies, a mutant rat minK subunit (D77N), analogous to D76N hminK, coassembled with WT minK and suppressed I<sub>Ks </sub>function, a dominant-lethal effect (Wang and Goldstein, 1995).</p>
    <p>It is concluded that mutations in KCNE1, the gene that encodes β subunits of I<sub>Ks </sub>channels, cause arrhythmia susceptibility by reducing I<sub>Ks </sub>and thereby delaying myocellular repolarization. Because regional heterogeneity in I<sub>Ks </sub>exists within the myocardium (Liu and Antzelevitch, 1995), mutations in KCNE1 would cause abnormal regional disparity in action potential duration, creating a substrate for arrhythmia. The discovery of LQT-associated mutations in KCNE1 will facilitate presymptomatic diagnosis of this disorder and may have implications for therapy.</p>
    <heading>EXAMPLE 16</heading> <p>Genomic Structure of KCNE1</p>
    <p>The genomic DNA of KCNE1 was examined and the exon/intron boundaries determined for all exons essentially as done for KVLQT1. An adult heart cDNA library was screened with a PCR product amplified from total human DNA and containing the entire coding sequence to isolate two identical 1.7 kb KCNE1 clones. Two overlapping cosmid clones encompassing the entire KCNE1 cDNA were also isolated using full length KCNE1 as a probe (FIG. <b>17</b>). The cosmids were sequenced by a dideoxy chain termination method on an Applied Biosystems model 373A DNA sequencer to define the genomic structure of the KCNE1 gene. Three exons comprise KCNE1 cDNA (FIG. <b>18</b> and Table 8). The two introns were located in the 5′-UTR. The donor and acceptor splice sites for both introns were GT and AG, respectively. Three pairs of primers were designed for screening KCNE1 (Table 9). The first and second pair overlap and cover the entire coding sequence. The third pair amplifies part of the coding region including the putative transmembrane domain and some of the flanking sequences.</p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" pgwide="1" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="259pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 8</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Intron/Exon Boundaries in KCNE1</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="4"> <colspec colname="1" colwidth="21pt" align="center"> </colspec> <colspec colname="2" colwidth="98pt" align="left"> </colspec> <colspec colname="3" colwidth="35pt" align="center"> </colspec> <colspec colname="4" colwidth="105pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Exon</td>
                <td class="description-td"> </td>
                <td class="description-td">EXON</td>
                <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">No.</td>
                <td class="description-td">Intron/EXON<sup>a</sup> </td>
                <td class="description-td">SIZE (bp)</td>
                <td class="description-td">EXON/Intron<sup>a</sup> </td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="4" align="center" rowsep="1" class="description-td" colspan="4"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">1</td>
                <td class="description-td">5′UTR . . . CCACACCCG (95)</td>
                <td class="description-td"> 33</td>
                <td class="description-td">TCAGACCCGGgtgagttagg (96)</td>
              </tr> <tr class="description-tr"> <td class="description-td">2</td>
                <td class="description-td">caatcaccagGAAAAATCCC (97)</td>
                <td class="description-td">111</td>
                <td class="description-td">GGATATTCAGgtaggacctg (98)</td>
              </tr> <tr class="description-tr"> <td class="description-td">3</td>
                <td class="description-td">ttcctttaagAGGT . . . ATG (99)</td>
                <td class="description-td">437</td>
                <td class="description-td">TTCCCCATGA . . . 3′UTR (100)</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="4" align="center" rowsep="1" class="description-td" colspan="4"> </td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="4" align="left" class="description-td" colspan="4"> <sup>a</sup>SEQ ID NO is shown in parentheses following each sequence </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" pgwide="1" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="315pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 9</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Primers Used to Amplify KCNE1 Coding Sequence</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="5"> <colspec colname="1" colwidth="21pt" align="center"> </colspec> <colspec colname="2" colwidth="133pt" align="left"> </colspec> <colspec colname="3" colwidth="126pt" align="left"> </colspec> <colspec colname="4" colwidth="21pt" align="center"> </colspec> <colspec colname="5" colwidth="14pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Exon</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">Size</td>
                <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">No.</td>
                <td class="description-td">Forward Primer<sup>a</sup> </td>
                <td class="description-td">Reverse Primer<sup>a</sup> </td>
                <td class="description-td">(bp)</td>
                <td class="description-td">C<sup>b</sup> </td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="5" align="center" rowsep="1" class="description-td" colspan="5"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">3</td>
                <td class="description-td">CTGCAGCAGTGGAACCTTAATG (101)</td>
                <td class="description-td">GTTCGAGTGCTCCAGCTTCTTG (102)</td>
                <td class="description-td">264</td>
                <td class="description-td">1</td>
              </tr> <tr class="description-tr"> <td class="description-td">3</td>
                <td class="description-td">GGGCATCATGCTGAGCTACAT (103)</td>
                <td class="description-td">TTTAGCCAGTGGTGGGGTTCA (104)</td>
                <td class="description-td">231</td>
                <td class="description-td">1</td>
              </tr> <tr class="description-tr"> <td class="description-td">3</td>
                <td class="description-td">GTTCAGCAGGGTGGCAACAT (105)</td>
                <td class="description-td">GCCAGATGGTTTTCAACGACA (106)</td>
                <td class="description-td">281</td>
                <td class="description-td">1</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="5" align="center" rowsep="1" class="description-td" colspan="5"> </td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="5" align="left" class="description-td" colspan="5"> <sup>a</sup>SEQ ID NO is shown in parentheses following each sequence. </td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="5" align="left" class="description-td" colspan="5"> <sup>b</sup>Conditions of the PCR as described in Example 10D. </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <heading>EXAMPLE 17</heading> <p>Generation of Polyclonal Antibody Against KVLQT1 or KCNE1</p>
    <p>Segments of KVLQT1 or KCNE1 coding sequence are expressed as fusion protein in <i>E. coli</i>. The overexpressed protein is purified by gel elution and used to immunize rabbits and mice using a procedure similar to the one described by Harlow and Lane (1988). This procedure has been shown to generate Abs against various other proteins (for example, see Kraemer et al., 1993).</p>
    <p>Briefly, a stretch of KVLQT1 or KCNE1 coding sequence is cloned as a fusion protein in plasmid PET5A (Novagen, Inc., Madison, Wis.). After induction with IPTG, the overexpression of a fusion protein with the expected molecular weight is verified by SDS/PAGE. Fusion protein is purified from the gel by electroelution. Identification of the protein as the KVLQT1 or KCNE1 fusion product is verified by protein sequencing at the N-terminus. Next, the purified protein is used as immunogen in rabbits. Rabbits are immunized with 100 μg of the protein in complete Freund's adjuvant and boosted twice in 3 week intervals, first with 100 μg of immunogen in incomplete Freund's adjuvant followed by 100 μg of immunogen in PBS. Antibody containing serum is collected two weeks thereafter.</p>
    <p>This procedure is repeated to generate antibodies against the mutant forms of the KVLQT1 or KCNE1 gene product. These antibodies, in conjunction with antibodies to wild type KVLQT1 or KCNE1, are used to detect the presence and the relative level of the mutant forms in various tissues and biological fluids.</p>
    <heading>EXAMPLE 18</heading> <p>Generation of Monoclonal Antibodies Specific for KVLQT1 or KCNE1</p>
    <p>Monoclonal antibodies are generated according to the following protocol. Mice are immunized with immunogen comprising intact KVLQT1, KCNE1, KVLQT1 peptides or KCNE1 peptides (wild type or mutant) conjugated to keyhole limpet hemocyanin using glutaraldehyde or EDC as is well known.</p>
    <p>The immunogen is mixed with an adjuvant. Each mouse receives four injections of 10 to 100 μg of immunogen and after the fourth injection blood samples are taken from the mice to determine if the serum contains antibody to the immunogen. Serum titer is determined by ELISA or RIA. Mice with sera indicating the presence of antibody to the immunogen are selected for hybridoma production.</p>
    <p>Spleens are removed from immune mice and a single cell suspension is prepared (see Harlow and Lane, 1988). Cell fusions are performed essentially as described by Kohler and Milstein (1975). Briefly, P3.65.3 myeloma cells (American Type Culture Collection, Rockville, Md.) are fused with immune spleen cells using polyethylene glycol as described by Harlow and Lane (1988). Cells are plated at a density of 2×10<sup>5 </sup>cells/well in 96 well tissue culture plates. Individual wells are examined for growth and the supernatants of wells with growth are tested for the presence of KVLQT1 or KCNE1 specific antibodies by ELISA or RIA using wild type or mutant KVLQT1 or KCNE1 target protein. Cells in positive wells are expanded and subcloned to establish and confirm monoclonality.</p>
    <p>Clones with the desired specificities are expanded and grown as ascites in mice or in a hollow fiber system to produce sufficient quantities of antibody for characterization and assay development.</p>
    <heading>EXAMPLE 19</heading> <p>Sandwich Assay for KVLQT1 or KCNE1</p>
    <p>Monoclonal antibody is attached to a solid surface such as a plate, tube, bead or particle. Preferably, the antibody is attached to the well surface of a 96-well ELISA plate. 100 μL sample (e.g., serum, urine, tissue cytosol) containing the KVLQT1 or KCNE1 peptide/protein (wild-type or mutants) is added to the solid phase antibody. The sample is incubated for 2 hrs at room temperature. Next the sample fluid is decanted, and the solid phase is washed with buffer to remove unbound material. 100 μL of a second monoclonal antibody (to a different determinant on the KVLQT1 or KCNE1 peptide/protein) is added to the solid phase. This antibody is labeled with a detector molecule (e.g., <sup>125</sup>I, enzyme, fluorophore, or a chromophore) and the solid phase with the second antibody is incubated for two hrs at room temperature. The second antibody is decanted and the solid phase is washed with buffer to remove unbound material.</p>
    <p>The amount of bound label, which is proportional to the amount of KVLQT1 or KCNE1 peptide/protein present in the sample, is quantified. Separate assays are performed using monoclonal antibodies which are specific for the wild-type KVLQT1 or KCNE1 as well as monoclonal antibodies specific for each of the mutations identified in KVLQT1 or KCNE1.</p>
    <heading>EXAMPLE 20</heading> <p>Assay to Screen Drugs Affecting the KVLQT1 and KCNE1 K<sup>+ </sup>Channel</p>
    <p>With the knowledge that KVLQT1 and KCNE1 coassemble to form a cardiac I<sub>Ks </sub>potassium channel, it is now possible to devise an assay to screen for drugs which will have an effect on this channel. The two genes, KVLQT1 and KCNE1, are cotransfected into oocytes or mammalian cells and coexpressed as described above. The cotransfection is performed using any combination of wild-type or specifically mutated KVLQT1 and KCNE1. When one of the genes used for cotransfection contains a mutation which causes LQT a change in the induced current is seen as compared to cotransfection with wild-type genes only. A drug candidate is added to the bathing solution of the transfected cells to test the effects of the drug candidates upon the induced current. A drug candidate which alters the induced current such that it is closer to the current seen with cells cotransfected with wild-type KVLQT1 and KCNE1 is useful for treating LQT.</p>
    <p>While the invention has been disclosed in this patent application by reference to the details of preferred embodiments of the invention, it is to be understood that the disclosure is intended in an illustrative rather than in a limiting sense, as it is contemplated that modifications will readily occur to those skilled in the art, within the spirit of the invention and the scope of the appended claims.</p>
    <heading>LIST OF REFERENCES</heading> <p>Altschul S F, et al. (1997). <i>Nucl. Acids Res. </i>25:3389-3402.</p>
    <p>Anand, R (1992). <i>Techniques for the Analysis of Complex Genomes</i>, (Academic Press).</p>
    <p>Anderson W F, et al. (1980). <i>Proc. Natl. Acad. Sci. USA </i>77:5399-5403.</p>
    <p>Anderson M A and Gussella J F (1984). <i>In Vitro </i>20:856-858.</p>
    <p>Antzelevitch C and Sicouri S (1994). <i>J. Am. Col. Card. </i>23:259-277.</p>
    <p>Attali B, et al. (1993). <i>Nature </i>365:850-852.</p>
    <p>Attwell D, et al. (1979). <i>Pflugers Arch. </i>379:137-142.</p>
    <p>Ausubel F M, et al. (1992). <i>Current Protocols in Molecular Biology</i>, (John Wiley and Sons, New York, N.Y.).</p>
    <p>Balser J R, et al. (1990). <i>J. Gen. Physiol. </i>96:835-863.</p>
    <p>Balser J R, et al. (1991). <i>Circ. Res. </i>69:519-529.</p>
    <p>Bandyopadhyay P K and Temin H M (1984). <i>Mol. Cell. Biol. </i>4:749-754.</p>
    <p>Barhanin J, et al. (1996). <i>Nature </i>384:78-80.</p>
    <p>Bartel P L, et al. (1993). “Using the 2-hybrid system to detect protein-protein interactions.” In <i>Cellular Interactions in Development: A Practical Approach</i>, Oxford University Press, pp. 153-179.</p>
    <p>Bazzett H (1920). <i>Heart </i>7:353-370.</p>
    <p>Beaucage S L and Caruthers M H (1981). <i>Tetra. Letts. </i>22:1859-1862.</p>
    <p>Berglund P, et al. (1993). <i>Biotechnology </i>11:916-920.</p>
    <p>Berkner K L, et al. (1988). <i>BioTechniques </i>6:616-629.</p>
    <p>Berkner K L (1992). <i>Curr. Top. Microbiol. Immunol. </i>158:39-66.</p>
    <p>Borman S (1996). <i>Chemical </i>&amp; <i>Engineering News</i>, December 9 issue, pp.42-43.</p>
    <p>Breakefield X O and Geller A I (1987). <i>Mol. Neurobiol. </i>1:337-371.</p>
    <p>Brinster R L, et al. (1981). <i>Cell </i>27:223-231.</p>
    <p>Bruggemann A, et al. (1993). <i>Nature </i>365:445-448.</p>
    <p>Buchschacher G L and Panganiban A T (1992). <i>J. Virol. </i>66:2731-2739.</p>
    <p>Buckler A J, et al. (1991). <i>Proc. Natl. Acad. Sci. USA </i>88:4005-4009.</p>
    <p>Burn T C, et al. (1995). <i>Gene </i>161:183-187.</p>
    <p>Busch A E, et al. (1992). <i>Science </i>255:1705-1707.</p>
    <p>Capecchi M R (1989). <i>Science </i>244:1288.</p>
    <p>Cardiac Arrhythmia Suppression Trial II Investigators (1992). <i>N. Engl. J. Med. </i>327:227-233</p>
    <p>Cariello N F (1988). <i>Am. J. Human Genetics </i>42:726-734.</p>
    <p>Chee M, et al. (1996). Science 274:610-614 .</p>
    <p>Chevray P M and Nathans D N (1992). <i>Proc. Natl. Acad. Sci. USA </i>89:5789-5793.</p>
    <p>Chin H M, et al. (1991). <i>Genomics </i>11:914-919.</p>
    <p>Church D M, et al. (1994). <i>Nat. Genet. </i>6:98-105.</p>
    <p>Compton J (1991). <i>Nature </i>350:91-92.</p>
    <p>Conner B J, et al. (1983). <i>Proc. Natl. Acad. Sci. USA </i>80:278-282.</p>
    <p>Costantini F and Lacy E (1981). <i>Nature </i>294:92-94.</p>
    <p>Cotten M, et al. (1990). <i>Proc. Natl. Acad. Sci. USA </i>87:4033-4037.</p>
    <p>Cotton R G, et al. (1988). <i>Proc. Natl. Acad. Sci. USA </i>85:4397-4401.</p>
    <p>Covarrubias M, et al. (1991). <i>Neuron </i>7:763-773.</p>
    <p>Cui J, et al. (1994). J. Gen. Physiol. 104:87-105.</p>
    <p>Culver K W, et al. (1992). <i>Science </i>256:1550-1552.</p>
    <p>Culver K (1996). <i>Gene Therapy: A Primer for Physicians, </i>2nd Ed., Mary Ann Liebert.</p>
    <p>Curiel D T, et al. (1991). <i>Proc. Natl. Acad. Sci. USA </i>88:8850-8854.</p>
    <p>Curiel D T, et al. (1992). <i>Hum. Gene Ther. </i>3:147-154.</p>
    <p>Curran M E, et al. (1995). <i>Cell </i>80:795-804.</p>
    <p>DeRisi J, et al. (1996). <i>Nat. Genet. </i>14:457-460.</p>
    <p>Deutscher M (1990). <i>Meth. Enzymology </i>182:83-89 (Academic Press, San Diego, Calif.).</p>
    <p>Donehower L A, et al. (1992). <i>Nature </i>356:215.</p>
    <p>Duggal P et al. (1998). <i>Circulation </i>97:142-146.</p>
    <p>Editorial (1996). <i>Nature Genetics </i>14:367-370.</p>
    <p>Elghanian R, et al. (1997). <i>Science </i>277:1078-1081.</p>
    <p> <i>Enhancers and Eukaryotic Gene Expression</i>, Cold Spring Harbor Press, Cold Spring Harbor, N.Y. (1983).</p>
    <p>Erickson J, et al. (1990). <i>Science </i>249:527-533.</p>
    <p>Fahy E, et al. (1991). <i>PCR Methods Appl. </i>1:25-33.</p>
    <p>Felgner P L, et al. (1987). <i>Proc. Natl. Acad. Sci. USA </i>84:7413-7417.</p>
    <p>Fields S and Song O-K (1989). <i>Nature </i>340:245-246.</p>
    <p>Fiers W. et al. (1978). <i>Nature </i>273:113-120.</p>
    <p>Fink D J, et al. (1992). <i>Hum. Gene Ther. </i>3:11-19.</p>
    <p>Fink D J, et al. (1996). <i>Ann. Rev. Neurosci. </i>19:265-287.</p>
    <p>Finkelstein J, et al. (1990). <i>Genomics </i>7:167-172.</p>
    <p>Fodor SPA (1997). <i>Science </i>277:393-395.</p>
    <p>Freese A, et al. (1990). <i>Biochem. Pharmacol. </i>40:2189-2199.</p>
    <p>Friedman T (1991). In <i>Therapy for Genetic Diseases</i>, T. Friedman, ed., Oxford University Press, pp. 105-121.</p>
    <p>Gellens M, et al. (1992). <i>Proc. Natl. Acad. Sci. USA </i>89:554-558.</p>
    <p>George A L, et al. (1995). <i>Cytogenet. Cell. Genet. </i>68:67-70.</p>
    <p>Glover D (1985). <i>DNA Cloning</i>, I and II (Oxford Press).</p>
    <p>Goding (1986). <i>Monoclonal Antibodies: Principles and Practice, </i>2d ed. (Academic Press, N.Y.).</p>
    <p>Godowski P J, et al. (1988). <i>Science </i>241:812-816.</p>
    <p>Goldstein S A N and Miller C (1991). <i>Neuron </i>7:403-408.</p>
    <p>Gordon J W, et al. (1980). <i>Proc. Natl. Acad. Sci. USA </i>77:7380-7384.</p>
    <p>Gorziglia M and Kapikian A Z (1992). <i>J. Virol. </i>66:4407-4412.</p>
    <p>Graham F L and van der Eb A J (1973). <i>Virology </i>52:456-467.</p>
    <p>Green E D and Olson M V (1990). <i>Proc. Natl. Acad. Sci. USA </i>87:1213-1217.</p>
    <p>Grompe M (1993). <i>Nature Genetics </i>5:111-117.</p>
    <p>Grompe M, et al. (1989). <i>Proc. Natl. Acad. Sci. USA </i>86:5855-5892.</p>
    <p>Guthrie G and Fink G R (1991). <i>Guide to Yeast Genetics and Molecular Biology </i>(Academic Press).</p>
    <p>Gyapay G, et al. (1994). <i>Nat. Genet. </i>7:246-339.</p>
    <p>Hacia J G, et al. (1996). <i>Nature Genetics </i>14:441-447.</p>
    <p>Harlow E and Lane D (1988). <i>Antibodies: A Laboratory Manual </i>(Cold Spring Harbor Laboratory, Cold Spring Harbor, N.Y.).</p>
    <p>Hasty P K, et al. (1991). <i>Nature </i>350:243.</p>
    <p>Hausdorff S F, et al. (1991). <i>Biochem. </i>30:3341-3346.</p>
    <p>Heginbotham L, et al. (1994). <i>Biophys. J. </i>66:1061-1067.</p>
    <p>Helseth E, et al. (1990). <i>J. Virol. </i>64:2416-2420.</p>
    <p>Hodgson J (1991). <i>Bio/Technology </i>9:19-21.</p>
    <p>Huse W D, et al. (1989). <i>Science </i>246:1275-1281.</p>
    <p>Innis M A, et al. (1990). <i>PCR Protocols: A Guide to Methods and Applications </i>(Academic Press, San Diego).</p>
    <p>Jablonski E, et al. (1986). <i>Nucl. Acids Res. </i>14:6115-6128.</p>
    <p>Jakoby W B and Pastan I H (eds.) (1979). <i>Cell Culture. Methods in Enzymology </i>volume 58 (Academic Press, Inc., Harcourt Brace Jovanovich (New York)).</p>
    <p>January C T and Riddle J M (1989). <i>Circ. Res. </i>64:977-990.</p>
    <p>Jervell A and Lange-Nielsen F (1957). <i>Am. Heart J. </i>54:59-68.</p>
    <p>Jiang C, et al. (1994). Nat. Genet. 8:141-147.</p>
    <p>Johnson P A, et al. (1992). <i>J. Virol. </i>66:2952-2965.</p>
    <p>Johnson, et al. (1993). “Peptide Turn Mimetics” in <i>Biotechnology and Pharmacy</i>, Pezzuto et al., eds., Chapman and Hall, New York.</p>
    <p>Kaneda Y, et al. (1989). <i>J. Biol. Chem. </i>264:12126-12129.</p>
    <p>Kanehisa M (1984). <i>Nucl. Acids Res. </i>12:203-213.</p>
    <p>Kannel W B, et al. (1987). <i>Am. Heart J. </i>113:799-804.</p>
    <p>Keating M T, et al. (1991a). <i>Science </i>252:704-706.</p>
    <p>Keating M T, et al. (1991b). <i>Am. J. Hum. Genet. </i>49:1335-1339.</p>
    <p>Kinszler K W, et al. (1991). <i>Science </i>251:1366-1370.</p>
    <p>Kohler G and Milstein C (1975). <i>Nature </i>256:495-497.</p>
    <p>Kraemer F B, et al. (1993). <i>J. Lipid Res. </i>34:663-672.</p>
    <p>Kubo T, et al. (1988). <i>FEBS Lett. </i>241:119.</p>
    <p>Kwiatkowski T J, et al. (1990). <i>Nucl. Acids Res. </i>18:7191-7192.</p>
    <p>Kyte J and Doolittle R F (1982). <i>J. Mol. Biol. </i>157:105-132.</p>
    <p>Landegren U, et al. (1988). <i>Science </i>242:229-237.</p>
    <p>Lathrop, G M, et al. (1985). <i>Am. J. Hum. Genet. </i>37:482-498.</p>
    <p>Lee J E, et al. (1995). <i>Science </i>268:836-844.</p>
    <p>Lesage F, et al. (1993). <i>Receptors and Channels </i>1:143-152.</p>
    <p>Li G-R, et al. (1996). <i>Circ. Res. </i>78:689-696.</p>
    <p>Lim C S, et al. (1991). <i>Circulation </i>83:2007-2011.</p>
    <p>Lipshutz R J, et al. (1995). <i>BioTechniques </i>19:442-447.</p>
    <p>Liu D W and Antzelevitch C (1995). <i>Circ. Res. </i>76:351-365.</p>
    <p>Lockhart D J, et al. (1996). <i>Nature Biotechnology </i>14:1675-1680.</p>
    <p>MacKinnon R (1991). <i>Nature </i>350:232-235.</p>
    <p>MacKinnon R, et al. (1993). <i>Science </i>262:757-759.</p>
    <p>Madzak C, et al. (1992). <i>J. Gen. Virol. </i>73:1533-1536.</p>
    <p>Magovcevic I, et al. (1992). <i>Genomics </i>12:125-129.</p>
    <p>Maniatis T, et al. (1982). <i>Molecular Cloning: A Laboratory Manual </i>(Cold Spring Harbor Laboratory, Cold Spring Harbor, N.Y.).</p>
    <p>Mann R and Baltimore D (1985). <i>J. Virol. </i>54:401-407.</p>
    <p>Marchuk D, et al. (1991). <i>Nucl. Acids Res. </i>19:1154.</p>
    <p>Margolskee R F (1992). <i>Curr. Top. Microbiol. Immunol. </i>158:67-95.</p>
    <p>Martin R, et al. (1990). <i>BioTechniques </i>9:762-768.</p>
    <p>Matsuura H, et al. (1987). <i>Pflugers Arch. </i>410:596-603.</p>
    <p>Matteucci M D and Caruthers M H (1981). <i>J. Am. Chem. Soc. </i>103:3185.</p>
    <p>Matthews J A and Kricka L J (1988). <i>Anal. Biochem. </i>169:1.</p>
    <p>Merrifield B (1963). <i>J. Am. Chem. Soc. </i>85:2149-2156.</p>
    <p>Metzger D, et al. (1988). <i>Nature </i>334:31-36.</p>
    <p>Mifflin T E (1989). <i>Clinical Chem. </i>35:1819-1825.</p>
    <p>Miller A D (1992). <i>Curr. Top. Microbiol. Immunol. </i>158:1-24.</p>
    <p>Miller A D, et al. (1985). <i>Mol. Cell. Biol. </i>5:431-437.</p>
    <p>Miller A D, et al. (1988). <i>J. Virol. </i>62:4337-4345.</p>
    <p>Modrich P (1991). <i>Ann. Rev. Genet. </i>25:229-253.</p>
    <p>Mombaerts P, et al. (1992). <i>Cell </i>68:869.</p>
    <p>Moss A J and McDonald J (1971). <i>N. Engl. J. Med. </i>285:903-904.</p>
    <p>Moss A J, et al. (1991). <i>Circulation </i>84:1136-1144.</p>
    <p>Moss B (1992). <i>Curr. Top. Microbiol. Immunol. </i>158:25-38.</p>
    <p>Moss B (1996). <i>Proc. Natl. Acad. Sci. USA </i>93:11341-11348.</p>
    <p>Muzyczka N (1992). <i>Curr. Top. Microbiol. Immunol. </i>158:97-129.</p>
    <p>Nabel E G, et al. (1990). <i>Science </i>249:1285-1288.</p>
    <p>Nabel (1992). <i>Hum. Gene Ther. </i>3:399-410.</p>
    <p>Naldini L, et al. (1996). <i>Science </i>272:263-267.</p>
    <p>Newton C R, et al. (1989). <i>Nucl. Acids Res. </i>17:2503-2516.</p>
    <p>Neyroud N, et al. (1997). <i>Nat. Genet. </i>15:186-189.</p>
    <p>Nguyen Q, et al. (1992). <i>BioTechniques </i>13:116-123.</p>
    <p>Novack D F, et al. (1986). <i>Proc. Natl. Acad. Sci. USA </i>83:586-590.</p>
    <p>Ochman H, et al. (1988). <i>Genetics </i>120:621-623.</p>
    <p>Ohi S, et al. (1990). <i>Gene </i>89:279-282.</p>
    <p>Orita M, et al. (1989). <i>Proc. Natl. Acad. Sci. USA </i>86:2766-2770.</p>
    <p>Page K A, et al. (1990). <i>J. Virol. </i>64:5270-5276.</p>
    <p>Pellicer A, et al. (1980). <i>Science </i>209:1414-1422.</p>
    <p>Petropoulos C J, et al. (1992). <i>J. Virol. </i>66:3391-3397.</p>
    <p>Philpott K L, et al. (1992). <i>Science </i>256:1448.</p>
    <p>Pongs O, et al. (1988). <i>EMBO J. </i>7:1087-1095.</p>
    <p>Quantin B, et al. (1992). <i>Proc. Natl. Acad. Sci. USA </i>89:2581-2584.</p>
    <p> <i>Remington's Pharmaceutical Sciences, </i>18th Ed. (1990, Mack Publishing Co., Easton, Pa.).</p>
    <p>Rettig J, et al. (1994). <i>Nature </i>369:289-294.</p>
    <p>Rigby P W J, et al. (1977). <i>J. Mol. Biol. </i>113:237-251.</p>
    <p>Romano C (1965). <i>Lancet I</i>658-659.</p>
    <p>Romano C, et al. (1963). <i>Clin. Pediatr. </i>45:656-683.</p>
    <p>Rosenfeld M A, et al. (1992). <i>Cell </i>68:143-155.</p>
    <p>Ruano G and Kidd K K (1989). <i>Nucl. Acids Res. </i>17:8392.</p>
    <p>Russell M W, et al. (1995). <i>Am. J. Hum. Genet. </i>57:503-507.</p>
    <p>Russell D and Hirata R (1998). <i>Nature Genetics </i>18:323-328.</p>
    <p>Sambrook J, et al. (1989). <i>Molecular Cloning: A Laboratory Manual, </i>2nd Ed. (Cold Spring Harbor Laboratory, Cold Spring Harbor, N.Y.).</p>
    <p>Sanguinetti M C and Jurkiewicz N K (1990). <i>J. Gen. Physiol. </i>96:195-215.</p>
    <p>Sanguinetti M C, et al. (1995). <i>Cell </i>81:299-307.</p>
    <p>Sanguinetti M C, et al. (1996a). <i>Proc. Natl. Acad, Sci. USA </i>93:2208-2212.</p>
    <p>Sanguinetti M C, et al. (1996b). <i>Nature </i>384:80-83.</p>
    <p>Scharf S J, et al. (1986). <i>Science </i>233:1076-1078.</p>
    <p>Schneider G, et al. (1998). <i>Nature Genetics </i>18:180-183.</p>
    <p>Schott J, et al. (1995). <i>Am. J. Hum. Genet. </i>57:1114-1122.</p>
    <p>Schultze-Bahr E, et al. (1997). <i>Nat. Genet. </i>17:267-268.</p>
    <p>Schwartz P J, et al. (1975). <i>Am. Heart J. </i>109:378-390.</p>
    <p>Schwartz P J, et al. (1994). “The long QT syndrome.” In <i>Cardiac Electrophysiology: from cell to bedside</i>. D. P. Zipes and J. Jalife eds. (W. B. Sanders Company) pp.788-811.</p>
    <p>Scopes R (1982). <i>Protein Purification: Principles and Practice</i>, (Springer-Verlag, N.Y.).</p>
    <p>Seino S, et al. (1992). <i>Genomics </i>13:1375-1377.</p>
    <p>Sheffield V C, et al. (1989). <i>Proc. Natl. Acad. Sci. USA </i>86:232-236.</p>
    <p>Sheffield V C, et al. (1991). <i>Am. J. Hum. Genet. </i>49:699-706.</p>
    <p>Shenk T E, et al. (1975). <i>Proc. Natl. Acad. Sci. USA </i>72:989-993.</p>
    <p>Shi G, et al. (1996). <i>Neuron </i>16:843-852.</p>
    <p>Shimada T, et al. (1991). <i>J. Clin. Invest. </i>88:1043-1047.</p>
    <p>Shinkai Y, et al. (1992). <i>Cell </i>68:855.</p>
    <p>Shoemaker D D, et al. (1996). <i>Nature Genetics </i>14:450-456.</p>
    <p>Smith P L, et al. (1996). <i>Nature </i>379:833-836.</p>
    <p>Snouwaert J N, et al. (1992). <i>Science </i>257:1083.</p>
    <p>Sorge J, et al. (1984). <i>Mol. Cell. Biol. </i>4:1730-1737.</p>
    <p>Spargo C A, et al. (1996). <i>Mol. Cell. Probes </i>10:247-256.</p>
    <p>Spector P S, et al. (1996). <i>J. Gen. Physiol. </i>107:611-619.</p>
    <p>Splawski I, et al. (1997a). <i>N. Engl. J. Med </i>336:1562-1567.</p>
    <p>Splawski I, et al. (1997b). <i>Nat. Genet. </i>17:338-340.</p>
    <p>Stemberg N (1990). <i>Proc. Natl. Acad. Sci. USA </i>87:103-107.</p>
    <p>Stewart M J, et al. (1992). <i>Hum. Gene Ther. </i>3:267-275.</p>
    <p>Stratford-Perricaudet L D, et al. (1990). <i>Hum. Gene Ther. </i>1:241-256.</p>
    <p>Surawicz B (1989). <i>J. Am. Coll. Cardiol. </i>14:172-184.</p>
    <p>Swanson R, et al. (1993). <i>Seminars in the Neurosciences </i>5:117-124.</p>
    <p>Takumi T, et al. (1988). <i>Science </i>242:1042-1045.</p>
    <p>Takumi T, et al. (1991). <i>J. Biol. Chem. </i>266:22192-22198.</p>
    <p>Tanigami A, et al. (1992). <i>Am. J. Hum. Genet. </i>50:56-64.</p>
    <p>Tokino T, et al. (1991). <i>Am. J. Hum. Genet. </i>48:258-268.</p>
    <p>Tyson J, et al. (1997). <i>Hum. Mol. Genet. </i>6:2179-2185.</p>
    <p>Valancius V and Smithies O (1991). <i>Mol. Cell Biol. </i>11:1402.</p>
    <p>Vetter D E, et al. (1996). <i>Neuron </i>17:1251-1264.</p>
    <p>Vincent G M, et al. (1992). <i>N Engl. J. Med. </i>327:846-852.</p>
    <p>Wagner E, et al. (1991). <i>Proc. Natl. Acad. Sci. USA </i>88:4255-4259.</p>
    <p>Wagner E, et al. (1990). <i>Proc. Natl. Acad. Sci. USA </i>87:3410-3414.</p>
    <p>Walker G T, et al., (1992). <i>Nucl. Acids Res. </i>20:1691-1696.</p>
    <p>Wang K W and Goldstein S A (1995). <i>Neuron </i>14:1303-1309.</p>
    <p>Wang K W, et al. (1996). <i>Neuron </i>16:571-577.</p>
    <p>Wang C Y and Huang L (1989). <i>Biochemistry </i>28:9508-9514.</p>
    <p>Wang Q and Keating M T (1994). <i>BioTechniques </i>17:282-284.</p>
    <p>Wang Q, et al. (1995a). <i>Cell </i>80:805-811.</p>
    <p>Wang Q, et al. (1995b). <i>Hum. Mol. Genet. </i>4:1603-1607.</p>
    <p>Wang Q, et al. (1996). <i>Nat. Genet. </i>12:17-23.</p>
    <p>Ward O C (1964). <i>J. Ir. Med. Assoc. </i>54:103-106.</p>
    <p>Warmke J E and Ganetzky B (1994). <i>Proc. Natl. Acad. Sci. </i>91:3438-3442.</p>
    <p>Wartell R M, et al. (1990). <i>Nucl. Acids Res. </i>18:2699-2705.</p>
    <p>Weinstein L S et al. (1988). <i>FEBS Letters </i>232:333-340.</p>
    <p>Wells J A (1991). <i>Methods Enzymol. </i>202:390-411.</p>
    <p>Wetmur J G and Davidson N (1968). <i>J. Mol. Biol. </i>31:349-370.</p>
    <p>White M B, et al. (1992). <i>Genomics </i>12:301-306.</p>
    <p>White R and Lalouel J M (1988). <i>Annu. Rev. Genet. </i>22:259-279.</p>
    <p>Wilkinson G W and Akrigg A (1992). <i>Nucleic Acids Res. </i>20:2233-2239.</p>
    <p>Willich S N, et al. (1987). <i>Am. J. Cardiol. </i>60:801-806.</p>
    <p>Wolff J A, et al. (1990). <i>Science </i>247:1465-1468.</p>
    <p>Wolff J A, et al. (1991). <i>BioTechniques </i>11:474-485.</p>
    <p>Wu D Y and Wallace R B (1989). <i>Genomics </i>4:560-569.</p>
    <p>Wu C H, et al. (1989). <i>J. Biol. Chem. </i>264:16985-16987.</p>
    <p>Wu G Y, et al. (1991). <i>J. Biol. Chem. </i>266:14338-14342.</p>
    <p>Wymore R S, et al. (1994). <i>Genomics </i>20:191202.</p>
    <p>Yang W P, et al. (1997). <i>Proc. Natl. Acad. Sci. USA </i>94:4017-4021.</p>
    <p>Zenke M, et al. (1990). <i>Proc. Natl. Acad Sci USA </i>87:3655-3659.</p>
    <p>Zipes D P (1987). <i>Am. J. Cardiol. </i>59:26E-31E.</p>
    <heading>PATENTS AND PATENT APPLICATIONS</heading> <p>European Patent Application Publication No. 0332435.</p>
    <p>EPO Publication No. 225,807.</p>
    <p>Hitzeman et al., EP 73,675A.</p>
    <p>EP 425,731A.</p>
    <p>WO 84/03564.</p>
    <p>WO 90/07936.</p>
    <p>WO 92/19195.</p>
    <p>WO 93/07282.</p>
    <p>WO 94/25503.</p>
    <p>WO 95/01203.</p>
    <p>WO 95/05452.</p>
    <p>WO 96/02286.</p>
    <p>WO 96/02646.</p>
    <p>WO 96/11698.</p>
    <p>WO 96/40871.</p>
    <p>WO 96/40959.</p>
    <p>WO 97/02048.</p>
    <p>WO 97/12635.</p>
    <p>U.S. Pat. No. 3,817,837.</p>
    <p>U.S. Pat. No. 3,850,752.</p>
    <p>U.S. Pat. No. 3,939,350.</p>
    <p>U.S. Pat. No. 3,996,345.</p>
    <p>U.S. Pat. No. 4,275,149.</p>
    <p>U.S. Pat. No. 4,277,437.</p>
    <p>U.S. Pat. No. 4,366,241.</p>
    <p>U.S. Pat. No. 4,376,110.</p>
    <p>U.S. Pat. No. 4,486,530.</p>
    <p>U.S. Pat. No. 4,554,101.</p>
    <p>U.S. Pat. No. 4,683,195.</p>
    <p>U.S. Pat. No. 4,683,202.</p>
    <p>U.S. Pat. No. 4,816,567.</p>
    <p>U.S. Pat. No. 4,868,105.</p>
    <p>U.S. Pat. No. 5,252,479.</p>
    <p>U.S. Pat. No. 5,270,184.</p>
    <p>U.S. Pat. No. 5,409,818.</p>
    <p>U.S. Pat. No. 5,436,146.</p>
    <p>U.S. Pat. No. 5,455,166.</p>
    <p>U.S. Pat. No. 5,550,050.</p>
    <p>U.S. Pat. No. 5,691,198.</p>
    <p>U.S. Pat. No. 5,735,500.</p>
    <p>U.S. Pat. No. 5,747,469.</p>
    <p>
      <sequence-cwu id="SEQLST-1"> <sequence-list-new-rules> <s100> <s160>116</s160> </s100> <s200> <s210>1</s210> <s211>3181</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> <s220> <s221>CDS</s221> <s222>(163)..(2190)</s222> </s220> </s200> <s400> 1
ctgccccctc cggccccgcc ccgagcgccc gggctgggcc ggcagcggcc ccccgcggcg     60
gggctggcag cagtggctgc ccgcactgcg cccgggcgct cgccttcgct gcagctcccg    120
gtgccgccgc tcgggccggc cccccggcag gccctcctcg tt atg gcc gcg gcc       174
                                               Met Ala Ala Ala
                                                 1
tcc tcc ccg ccc agg gcc gag agg aag cgc tgg ggt tgg ggc cgc ctg      222
Ser Ser Pro Pro Arg Ala Glu Arg Lys Arg Trp Gly Trp Gly Arg Leu
  5                  10                  15                  20
cca ggc gcc cgg cgg ggc agc gcg ggc ctg gcc aag aag tgc ccc ttc      270
Pro Gly Ala Arg Arg Gly Ser Ala Gly Leu Ala Lys Lys Cys Pro Phe
                 25                  30                  35
tcg ctg gag ctg gcg gag ggc ggc ccg gcg ggc ggc gcg ctc tac gcg      318
Ser Leu Glu Leu Ala Glu Gly Gly Pro Ala Gly Gly Ala Leu Tyr Ala
             40                  45                  50
ccc atc gcg ccc ggc gcc cca ggt ccc gcg ccc cct gcg tcc ccg gcc      366
Pro Ile Ala Pro Gly Ala Pro Gly Pro Ala Pro Pro Ala Ser Pro Ala
         55                  60                  65
gcg ccc gcc gcg ccc cca gtt gcc tcc gac ctt ggc ccg cgg ccg ccg      414
Ala Pro Ala Ala Pro Pro Val Ala Ser Asp Leu Gly Pro Arg Pro Pro
     70                  75                  80
gtg agc cta gac ccg cgc gtc tcc atc tac agc acg cgc cgc ccg gtg      462
Val Ser Leu Asp Pro Arg Val Ser Ile Tyr Ser Thr Arg Arg Pro Val
 85                  90                  95                 100
ttg gcg cgc acc cac gtc cag ggc cgc gtc tac aac ttc ctc gag cgt      510
Leu Ala Arg Thr His Val Gln Gly Arg Val Tyr Asn Phe Leu Glu Arg
                105                 110                 115
ccc acc ggc tgg aaa tgc ttc gtt tac cac ttc gcc gtc ttc ctc atc      558
Pro Thr Gly Trp Lys Cys Phe Val Tyr His Phe Ala Val Phe Leu Ile
            120                 125                 130
gtc ctg gtc tgc ctc atc ttc agc gtg ctg tcc acc atc gag cag tat      606
Val Leu Val Cys Leu Ile Phe Ser Val Leu Ser Thr Ile Glu Gln Tyr
        135                 140                 145
gcc gcc ctg gcc acg ggg act ctc ttc tgg atg gag atc gtg ctg gtg      654
Ala Ala Leu Ala Thr Gly Thr Leu Phe Trp Met Glu Ile Val Leu Val
    150                 155                 160
gtg ttc ttc ggg acg gag tac gtg gtc cgc ctc tgg tcc gcc ggc tgc      702
Val Phe Phe Gly Thr Glu Tyr Val Val Arg Leu Trp Ser Ala Gly Cys
165                 170                 175                 180
cgc agc aag tac gtg ggc ctc tgg ggg cgg ctg cgc ttt gcc cgg aag      750
Arg Ser Lys Tyr Val Gly Leu Trp Gly Arg Leu Arg Phe Ala Arg Lys
                185                 190                 195
ccc att tcc atc atc gac ctc atc gtg gtc gtg gcc tcc atg gtg gtc      798
Pro Ile Ser Ile Ile Asp Leu Ile Val Val Val Ala Ser Met Val Val
            200                 205                 210
ctc tgc gtg ggc tcc aag ggg cag gtg ttt gcc acg tcg gcc atc agg      846
Leu Cys Val Gly Ser Lys Gly Gln Val Phe Ala Thr Ser Ala Ile Arg
        215                 220                 225
ggc atc cgc ttc ctg cag atc ctg agg atg cta cac gtc gac cgc cag      894
Gly Ile Arg Phe Leu Gln Ile Leu Arg Met Leu His Val Asp Arg Gln
    230                 235                 240
gga ggc acc tgg agg ctc ctg ggc tcc gtg gtc ttc atc cac cgc cag      942
Gly Gly Thr Trp Arg Leu Leu Gly Ser Val Val Phe Ile His Arg Gln
245                 250                 255                 260
gag ctg ata acc acc ctg tac atc ggc ttc ctg ggc ctc atc ttc tcc      990
Glu Leu Ile Thr Thr Leu Tyr Ile Gly Phe Leu Gly Leu Ile Phe Ser
                265                 270                 275
tcg tac ttt gtg tac ctg gct gag aag gac gcg gtg aac gag tca ggc     1038
Ser Tyr Phe Val Tyr Leu Ala Glu Lys Asp Ala Val Asn Glu Ser Gly
            280                 285                 290
cgc gtg gag ttc ggc agc tac gca gat gcg ctg tgg tgg ggg gtg gtc     1086
Arg Val Glu Phe Gly Ser Tyr Ala Asp Ala Leu Trp Trp Gly Val Val
        295                 300                 305
aca gtc acc acc atc ggc tat ggg gac aag gtg ccc cag acg tgg gtc     1134
Thr Val Thr Thr Ile Gly Tyr Gly Asp Lys Val Pro Gln Thr Trp Val
    310                 315                 320
ggg aag acc atc gcc tcc tgc ttc tct gtc ttt gcc atc tcc ttc ttt     1182
Gly Lys Thr Ile Ala Ser Cys Phe Ser Val Phe Ala Ile Ser Phe Phe
325                 330                 335                 340
gcg ctc cca gcg ggg att ctt ggc tcg ggg ttt gcc ctg aag gtg cag     1230
Ala Leu Pro Ala Gly Ile Leu Gly Ser Gly Phe Ala Leu Lys Val Gln
                345                 350                 355
cag aag cag agg cag aag cac ttc aac cgg cag atc ccg gcg gca gcc     1278
Gln Lys Gln Arg Gln Lys His Phe Asn Arg Gln Ile Pro Ala Ala Ala
            360                 365                 370
tca ctc att cag acc gca tgg agg tgc tat gct gcc gag aac ccc gac     1326
Ser Leu Ile Gln Thr Ala Trp Arg Cys Tyr Ala Ala Glu Asn Pro Asp
        375                 380                 385
tcc tcc acc tgg aag atc tac atc cgg aag gcc ccc cgg agc cac act     1374
Ser Ser Thr Trp Lys Ile Tyr Ile Arg Lys Ala Pro Arg Ser His Thr
    390                 395                 400
ctg ctg tca ccc agc ccc aaa ccc aag aag tct gtg gtg gta aag aaa     1422
Leu Leu Ser Pro Ser Pro Lys Pro Lys Lys Ser Val Val Val Lys Lys
405                 410                 415                 420
aaa aag ttc aag ctg gac aaa gac aat ggg gtg act cct gga gag aag     1470
Lys Lys Phe Lys Leu Asp Lys Asp Asn Gly Val Thr Pro Gly Glu Lys
                425                 430                 435
atg ctc aca gtc ccc cat atc acg tgc gac ccc cca gaa gag cgg cgg     1518
Met Leu Thr Val Pro His Ile Thr Cys Asp Pro Pro Glu Glu Arg Arg
            440                 445                 450
ctg gac cac ttc tct gtc gac ggc tat gac agt tct gta agg aag agc     1566
Leu Asp His Phe Ser Val Asp Gly Tyr Asp Ser Ser Val Arg Lys Ser
        455                 460                 465
cca aca ctg ctg gaa gtg agc atg ccc cat ttc atg aga acc aac agc     1614
Pro Thr Leu Leu Glu Val Ser Met Pro His Phe Met Arg Thr Asn Ser
    470                 475                 480
ttc gcc gag gac ctg gac ctg gaa ggg gag act ctg ctg aca ccc atc     1662
Phe Ala Glu Asp Leu Asp Leu Glu Gly Glu Thr Leu Leu Thr Pro Ile
485                 490                 495                 500
acc cac atc tca cag ctg cgg gaa cac cat cgg gcc acc att aag gtc     1710
Thr His Ile Ser Gln Leu Arg Glu His His Arg Ala Thr Ile Lys Val
                505                 510                 515
att cga cgc atg cag tac ttt gtg gcc aag aag aaa ttc cag caa gcg     1758
Ile Arg Arg Met Gln Tyr Phe Val Ala Lys Lys Lys Phe Gln Gln Ala
            520                 525                 530
cgg aag cct tac gat gtg cgg gac gtc att gag cag tac tcg cag ggc     1806
Arg Lys Pro Tyr Asp Val Arg Asp Val Ile Glu Gln Tyr Ser Gln Gly
        535                 540                 545
cac ctc aac ctc atg gtg cgc atc aag gag ctg cag agg agg ctg gac     1854
His Leu Asn Leu Met Val Arg Ile Lys Glu Leu Gln Arg Arg Leu Asp
    550                 555                 560
cag tcc att ggg aag ccc tca ctg ttc atc tcc gtc tca gaa aag agc     1902
Gln Ser Ile Gly Lys Pro Ser Leu Phe Ile Ser Val Ser Glu Lys Ser
565                 570                 575                 580
aag gat cgc ggc agc aac acg atc ggc gcc cgc ctg aac cga gta gaa     1950
Lys Asp Arg Gly Ser Asn Thr Ile Gly Ala Arg Leu Asn Arg Val Glu
                585                 590                 595
gac aag gtg acg cag ctg gac cag agg ctg gca ctc atc acc gac atg     1998
Asp Lys Val Thr Gln Leu Asp Gln Arg Leu Ala Leu Ile Thr Asp Met
            600                 605                 610
ctt cac cag ctg ctc tcc ttg cac ggt ggc agc acc ccc ggc agc ggc     2046
Leu His Gln Leu Leu Ser Leu His Gly Gly Ser Thr Pro Gly Ser Gly
        615                 620                 625
ggc ccc ccc aga gag ggc ggg gcc cac atc acc cag ccc tgc ggc agt     2094
Gly Pro Pro Arg Glu Gly Gly Ala His Ile Thr Gln Pro Cys Gly Ser
    630                 635                 640
ggc ggc tcc gtc gac cct gag ctc ttc ctg ccc agc aac acc ctg ccc     2142
Gly Gly Ser Val Asp Pro Glu Leu Phe Leu Pro Ser Asn Thr Leu Pro
645                 650                 655                 660
acc tac gag cag ctg acc gtg ccc agg agg ggc ccc gat gag ggg tcc     2190
Thr Tyr Glu Gln Leu Thr Val Pro Arg Arg Gly Pro Asp Glu Gly Ser
                665                 670                 675
tgaggagggg atggggctgg gggatgggcc tgagtgagag gggaggccaa gagtggcccc   2250
acctggccct ctctgaagga ggccacctcc taaaaggccc agagagaaga gccccactct   2310
cagaggcccc aataccccat ggaccatgct gtctggcaca gcctgcactt gggggctcag   2370
caaggccacc tcttcctggc cggtgtgggg gccccgtctc aggtctgagt tgttacccca   2430
agcgccctgg cccccacatg gtgatgttga catcactggc atggtggttg ggacccagtg   2490
gcagggcaca gggcctggcc catgtatggc caggaagtag cacaggctga gtgcaggccc   2550
accctgcttg gcccaggggg cttcctgagg ggagacagag caacccctgg accccagcct   2610
caaatccagg accctgccag gcacaggcag ggcaggacca gcccacgctg actacagggc   2670
caccggcaat aaaagcccag gagcccattt ggagggcctg ggcctggctc cctcactctc   2730
aggaaatgct gacccatggg caggagactg tggagactgc tcctgagccc ccagcttcca   2790
gcaggaggga cagtctcacc atttccccag ggcacgtggt tgagtggggg gaacgcccac   2850
ttccctgggt tagactgcca gctcttccta gctggagagg agccctgcct ctccgcccct   2910
gagcccactg tgcgtggggc tcccgcctcc aacccctcgc ccagtcccag cagccagcca   2970
aacacacaga aggggactgc cacctcccct tgccagctgc tgagccgcag agaagtgacg   3030
gttcctacac aggacagggg ttccttctgg gcattacatc gcatagaaat caataatttg   3090
tggtgatttg gatctgtgtt ttaatgagtt tcacagtgtg attttgatta ttaattgtgc   3150
aagcttttcc taataaacgt ggagaatcac a                                  3181
</s400> <s200> <s210>2</s210> <s211>676</s211> <s212>PRT</s212> <s213>Homo sapiens</s213> </s200> <s400> 2
Met Ala Ala Ala Ser Ser Pro Pro Arg Ala Glu Arg Lys Arg Trp Gly
  1               5                  10                  15
Trp Gly Arg Leu Pro Gly Ala Arg Arg Gly Ser Ala Gly Leu Ala Lys
             20                  25                  30
Lys Cys Pro Phe Ser Leu Glu Leu Ala Glu Gly Gly Pro Ala Gly Gly
         35                  40                  45
Ala Leu Tyr Ala Pro Ile Ala Pro Gly Ala Pro Gly Pro Ala Pro Pro
     50                  55                  60
Ala Ser Pro Ala Ala Pro Ala Ala Pro Pro Val Ala Ser Asp Leu Gly
 65                  70                  75                  80
Pro Arg Pro Pro Val Ser Leu Asp Pro Arg Val Ser Ile Tyr Ser Thr
                 85                  90                  95
Arg Arg Pro Val Leu Ala Arg Thr His Val Gln Gly Arg Val Tyr Asn
            100                 105                 110
Phe Leu Glu Arg Pro Thr Gly Trp Lys Cys Phe Val Tyr His Phe Ala
        115                 120                 125
Val Phe Leu Ile Val Leu Val Cys Leu Ile Phe Ser Val Leu Ser Thr
    130                 135                 140
Ile Glu Gln Tyr Ala Ala Leu Ala Thr Gly Thr Leu Phe Trp Met Glu
145                 150                 155                 160
Ile Val Leu Val Val Phe Phe Gly Thr Glu Tyr Val Val Arg Leu Trp
                165                 170                 175
Ser Ala Gly Cys Arg Ser Lys Tyr Val Gly Leu Trp Gly Arg Leu Arg
            180                 185                 190
Phe Ala Arg Lys Pro Ile Ser Ile Ile Asp Leu Ile Val Val Val Ala
        195                 200                 205
Ser Met Val Val Leu Cys Val Gly Ser Lys Gly Gln Val Phe Ala Thr
    210                 215                 220
Ser Ala Ile Arg Gly Ile Arg Phe Leu Gln Ile Leu Arg Met Leu His
225                 230                 235                 240
Val Asp Arg Gln Gly Gly Thr Trp Arg Leu Leu Gly Ser Val Val Phe
                245                 250                 255
Ile His Arg Gln Glu Leu Ile Thr Thr Leu Tyr Ile Gly Phe Leu Gly
            260                 265                 270
Leu Ile Phe Ser Ser Tyr Phe Val Tyr Leu Ala Glu Lys Asp Ala Val
        275                 280                 285
Asn Glu Ser Gly Arg Val Glu Phe Gly Ser Tyr Ala Asp Ala Leu Trp
    290                 295                 300
Trp Gly Val Val Thr Val Thr Thr Ile Gly Tyr Gly Asp Lys Val Pro
305                 310                 315                 320
Gln Thr Trp Val Gly Lys Thr Ile Ala Ser Cys Phe Ser Val Phe Ala
                325                 330                 335
Ile Ser Phe Phe Ala Leu Pro Ala Gly Ile Leu Gly Ser Gly Phe Ala
            340                 345                 350
Leu Lys Val Gln Gln Lys Gln Arg Gln Lys His Phe Asn Arg Gln Ile
        355                 360                 365
Pro Ala Ala Ala Ser Leu Ile Gln Thr Ala Trp Arg Cys Tyr Ala Ala
    370                 375                 380
Glu Asn Pro Asp Ser Ser Thr Trp Lys Ile Tyr Ile Arg Lys Ala Pro
385                 390                 395                 400
Arg Ser His Thr Leu Leu Ser Pro Ser Pro Lys Pro Lys Lys Ser Val
                405                 410                 415
Val Val Lys Lys Lys Lys Phe Lys Leu Asp Lys Asp Asn Gly Val Thr
            420                 425                 430
Pro Gly Glu Lys Met Leu Thr Val Pro His Ile Thr Cys Asp Pro Pro
        435                 440                 445
Glu Glu Arg Arg Leu Asp His Phe Ser Val Asp Gly Tyr Asp Ser Ser
    450                 455                 460
Val Arg Lys Ser Pro Thr Leu Leu Glu Val Ser Met Pro His Phe Met
465                 470                 475                 480
Arg Thr Asn Ser Phe Ala Glu Asp Leu Asp Leu Glu Gly Glu Thr Leu
                485                 490                 495
Leu Thr Pro Ile Thr His Ile Ser Gln Leu Arg Glu His His Arg Ala
            500                 505                 510
Thr Ile Lys Val Ile Arg Arg Met Gln Tyr Phe Val Ala Lys Lys Lys
        515                 520                 525
Phe Gln Gln Ala Arg Lys Pro Tyr Asp Val Arg Asp Val Ile Glu Gln
    530                 535                 540
Tyr Ser Gln Gly His Leu Asn Leu Met Val Arg Ile Lys Glu Leu Gln
545                 550                 555                 560
Arg Arg Leu Asp Gln Ser Ile Gly Lys Pro Ser Leu Phe Ile Ser Val
                565                 570                 575
Ser Glu Lys Ser Lys Asp Arg Gly Ser Asn Thr Ile Gly Ala Arg Leu
            580                 585                 590
Asn Arg Val Glu Asp Lys Val Thr Gln Leu Asp Gln Arg Leu Ala Leu
        595                 600                 605
Ile Thr Asp Met Leu His Gln Leu Leu Ser Leu His Gly Gly Ser Thr
    610                 615                 620
Pro Gly Ser Gly Gly Pro Pro Arg Glu Gly Gly Ala His Ile Thr Gln
625                 630                 635                 640
Pro Cys Gly Ser Gly Gly Ser Val Asp Pro Glu Leu Phe Leu Pro Ser
                645                 650                 655
Asn Thr Leu Pro Thr Tyr Glu Gln Leu Thr Val Pro Arg Arg Gly Pro
            660                 665                 670
Asp Glu Gly Ser
        675
</s400> <s200> <s210>3</s210> <s211>1703</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> <s220> <s221>CDS</s221> <s222>(193)..(579)</s222> </s220> </s200> <s400> 3
acacccggct ctctcggcat ctcagacccg ggaaaaatcc ctctgctttc tctggccagt     60
ttcacacaat catcaggtga gccgaggatc cattggagga aggcattatc tgtatccaga    120
ggaaatagcc aaggatattc agaggtgtgc ctgggaagtt tgagctgcag cagtggaacc    180
ttaatgccca gg atg atc ctg tct aac acc aca gcg gtg acg ccc ttt ctg    231
              Met Ile Leu Ser Asn Thr Thr Ala Val Thr Pro Phe Leu
                1               5                  10
acc aag ctg tgg cag gag aca gtt cag cag ggt ggc aac atg tcg ggc      279
Thr Lys Leu Trp Gln Glu Thr Val Gln Gln Gly Gly Asn Met Ser Gly
     15                  20                  25
ctg gcc cgc agg tcc ccc cgc agc ggt gac ggc aag ctg gag gcc ctc      327
Leu Ala Arg Arg Ser Pro Arg Ser Gly Asp Gly Lys Leu Glu Ala Leu
 30                  35                  40                  45
tac gtc ctc atg gta ctg gga ttc ttc ggc ttc ttc acc ctg ggc atc      375
Tyr Val Leu Met Val Leu Gly Phe Phe Gly Phe Phe Thr Leu Gly Ile
                 50                  55                  60
atg ctg agc tac atc cgc tcc aag aag ctg gag cac tcg aac gac cca      423
Met Leu Ser Tyr Ile Arg Ser Lys Lys Leu Glu His Ser Asn Asp Pro
             65                  70                  75
ttc aac gtc tac atc gag tcc gat gcc tgg caa gag aag gac aag gcc      471
Phe Asn Val Tyr Ile Glu Ser Asp Ala Trp Gln Glu Lys Asp Lys Ala
         80                  85                  90
tat gtc cag gcc cgg gtc ctg gag agc tac agg tcg tgc tat gtc gtt      519
Tyr Val Gln Ala Arg Val Leu Glu Ser Tyr Arg Ser Cys Tyr Val Val
     95                 100                 105
gaa aac cat ctg gcc ata gaa caa ccc aac aca cac ctt cct gag acg      567
Glu Asn His Leu Ala Ile Glu Gln Pro Asn Thr His Leu Pro Glu Thr
110                 115                 120                 125
aag cct tcc cca tgaaccccac cactggctaa actggacacc tcctgctggn          619
Lys Pro Ser Pro
nnnnagattt tctaatcaca ttcctctcat actctttatt gtgatggata ccactggatt    679
tctttttggc tgttgtaang ggtgaggggt ggattaatga cactgtttca ctgtttctct    739
aaaatcacgt tcttttgtga tagactgtca gtggttcccc catatctgtc cctgccttgc    799
taaatttagc agaatccctg aggacatggc ctctgagaat agcagctgca tttcccagac    859
tcccttgcag ctagcaaggt tgtgtgacta agccctggcc agtaggcatg gaagtgaaga    919
ctgtaatgtc caagtaatcc ttggaaagaa aagaacgtgc ccttaactaa ctttgtcctg    979
cttcccagtg gctggatgtg gaggaggtgg agagcagtta tgagactggg aaagttcggg   1039
gcactcaaag agccacacac atctgggcct gggcgacgtg gatcctcctt accacccacc   1099
aggccagatt tacaggagag agaaatccac tccactcttc cttaagccac tgttattctg   1159
atctctgtta aggtcgcaga atcaatgccc ttactgatac acctacctta taggactgaa   1219
cctaaaggca tgacatttcc atacttgtca caagcacaca ctgattctgc ccttgtcact   1279
tctgtgctca ctcttgtggc tctatcctcc tcctgccctt ccgccttcca ctcctccctt   1339
gcacccatcc tgcacacatc tccctgaaaa cacacaggca catacactca tatacataga   1399
cacacataca cacctcaatc tagaaagaac ttgctttgta cagggctgag atggaggaga   1459
aaaaaatgcc cccttcagaa tgcataccaa ggggaaggtg ctcggtcact gtgggagcag   1519
ggaaaggtgc ccccactccc cgagagccag gggaaggagt ggctctgggc agagagggac   1579
acatagcact ggggtggcag gtccttttga ggtgatgggc cggttttgtg agatgaattg   1639
tatcccccaa aaagacaggt accttcaatg tgacctaatt gggaaataga gtctttgcag   1699
atga                                                                1703
</s400> <s200> <s210>4</s210> <s211>129</s211> <s212>PRT</s212> <s213>Homo sapiens</s213> </s200> <s400> 4
Met Ile Leu Ser Asn Thr Thr Ala Val Thr Pro Phe Leu Thr Lys Leu
  1               5                  10                  15
Trp Gln Glu Thr Val Gln Gln Gly Gly Asn Met Ser Gly Leu Ala Arg
             20                  25                  30
Arg Ser Pro Arg Ser Gly Asp Gly Lys Leu Glu Ala Leu Tyr Val Leu
         35                  40                  45
Met Val Leu Gly Phe Phe Gly Phe Phe Thr Leu Gly Ile Met Leu Ser
     50                  55                  60
Tyr Ile Arg Ser Lys Lys Leu Glu His Ser Asn Asp Pro Phe Asn Val
 65                  70                  75                  80
Tyr Ile Glu Ser Asp Ala Trp Gln Glu Lys Asp Lys Ala Tyr Val Gln
                 85                  90                  95
Ala Arg Val Leu Glu Ser Tyr Arg Ser Cys Tyr Val Val Glu Asn His
            100                 105                 110
Leu Ala Ile Glu Gln Pro Asn Thr His Leu Pro Glu Thr Lys Pro Ser
        115                 120                 125
Pro
</s400> <s200> <s210>5</s210> <s211>63</s211> <s212>DNA</s212> <s213>Artificial Sequence</s213> <s220> <s223>Description of Artificial SequenceHypothetical
      sequence to demonstrate calculation of percent
      homology or identity.</s223> </s220> </s200> <s400> 5
accgtagcta cgtacgtata tagaaagggc gcgatcgtcg tcgcgtatga cgacttagca     60
tgc                                                                   63
</s400> <s200> <s210>6</s210> <s211>130</s211> <s212>DNA</s212> <s213>Artificial Sequence</s213> <s220> <s223>Description of Artificial SequenceHypothetical
      sequence to demonstrate calculation of percent
      homology or identity.</s223> </s220> </s200> <s400> 6
accggtagct acgtacgtta tttagaaagg ggtgtgtgtg tgtgtgtaaa ccggggtttt     60
cgggatcgtc cgtcgcgtat gacgacttag ccatgcacgg tatatcgtat taggactagc    120
gattgactag                                                           130
</s400> <s200> <s210>7</s210> <s211>17</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 7
cagatcctga ggatgct                                                    17
</s400> <s200> <s210>8</s210> <s211>17</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 8
gtacctggct gagaagg                                                    17
</s400> <s200> <s210>9</s210> <s211>10</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 9
atggccgcgg                                                            10
</s400> <s200> <s210>10</s210> <s211>20</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 10
acttcgccgt gtgagtatcg                                                 20
</s400> <s200> <s210>11</s210> <s211>20</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 11
tgtcttgcag cttcctcatc                                                 20
</s400> <s200> <s210>12</s210> <s211>20</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 12
cttctggatg gtacgtagca                                                 20
</s400> <s200> <s210>13</s210> <s211>20</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 13
gtccctgcag gagatcgtgc                                                 20
</s400> <s200> <s210>14</s210> <s211>20</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 14
tccatcatcg gtgagtcatg                                                 20
</s400> <s200> <s210>15</s210> <s211>20</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 15
cactccacag acctcatcgt                                                 20
</s400> <s200> <s210>16</s210> <s211>20</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 16
gggccatcag gtgcgtctgt                                                 20
</s400> <s200> <s210>17</s210> <s211>20</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 17
tccttcgcag gggcatccgc                                                 20
</s400> <s200> <s210>18</s210> <s211>20</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 18
ccaccgccag gtgggtggcc                                                 20
</s400> <s200> <s210>19</s210> <s211>20</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 19
tctggcctag gagctgataa                                                 20
</s400> <s200> <s210>20</s210> <s211>20</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 20
gtggggggtg gtaagtcgga                                                 20
</s400> <s200> <s210>21</s210> <s211>20</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 21
ctccctgcag gtcacagtca                                                 20
</s400> <s200> <s210>22</s210> <s211>20</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 22
gctcccagcg gtaggtgccc                                                 20
</s400> <s200> <s210>23</s210> <s211>20</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 23
tccttcccag gggattcttg                                                 20
</s400> <s200> <s210>24</s210> <s211>20</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 24
actcattcag gtgcggtgcc                                                 20
</s400> <s200> <s210>25</s210> <s211>20</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 25
cccacctcag accgcatgga                                                 20
</s400> <s200> <s210>26</s210> <s211>20</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 26
gtctgtggtg gtgagtagcc                                                 20
</s400> <s200> <s210>27</s210> <s211>20</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 27
ttttttttag gtaaagaaaa                                                 20
</s400> <s200> <s210>28</s210> <s211>20</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 28
gacagttctg gtgagaaccc                                                 20
</s400> <s200> <s210>29</s210> <s211>20</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 29
ttctcctcag taaggaagag                                                 20
</s400> <s200> <s210>30</s210> <s211>20</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 30
acatctcaca gtgagtgcct                                                 20
</s400> <s200> <s210>31</s210> <s211>20</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 31
tccactgcag gctgcgggaa                                                 20
</s400> <s200> <s210>32</s210> <s211>20</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 32
gaaattccag gtaagccctg                                                 20
</s400> <s200> <s210>33</s210> <s211>20</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 33
tgtcccgcag caagcgcgga                                                 20
</s400> <s200> <s210>34</s210> <s211>20</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 34
tgcagaggag gtgggcacgg                                                 20
</s400> <s200> <s210>35</s210> <s211>20</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 35
ttctctccag gctggaccag                                                 20
</s400> <s200> <s210>36</s210> <s211>20</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 36
tccgtctcag gtgggtttct                                                 20
</s400> <s200> <s210>37</s210> <s211>20</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 37
tcccccatag aaaagagcaa                                                 20
</s400> <s200> <s210>38</s210> <s211>20</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 38
agaagacaag gtaggctcac                                                 20
</s400> <s200> <s210>39</s210> <s211>20</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 39
gtccccgcag gtgacgcagc                                                 20
</s400> <s200> <s210>40</s210> <s211>10</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 40
ggggtcctga                                                            10
</s400> <s200> <s210>41</s210> <s211>19</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 41
ctcgccttcg ctgcagctc                                                  19
</s400> <s200> <s210>42</s210> <s211>19</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 42
gcgcgggtct aggctcacc                                                  19
</s400> <s200> <s210>43</s210> <s211>18</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 43
cgccgcgccc ccagttgc                                                   18
</s400> <s200> <s210>44</s210> <s211>19</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 44
cagagctccc ccacaccag                                                  19
</s400> <s200> <s210>45</s210> <s211>24</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 45
atgggcagag gccgtgatgc tgac                                            24
</s400> <s200> <s210>46</s210> <s211>22</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 46
atccagccat gccctcagat gc                                              22
</s400> <s200> <s210>47</s210> <s211>24</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 47
gttcaaacag gttgcagggt ctga                                            24
</s400> <s200> <s210>48</s210> <s211>21</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 48
cttcctggtc tggaaacctg g                                               21
</s400> <s200> <s210>49</s210> <s211>20</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 49
ctcttccctg gggccctggc                                                 20
</s400> <s200> <s210>50</s210> <s211>22</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 50
tgcgggggag cttgtggcac ag                                              22
</s400> <s200> <s210>51</s210> <s211>22</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 51
tcagccccac accatctcct tc                                              22
</s400> <s200> <s210>52</s210> <s211>20</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 52
ctgggcccct accctaaccc                                                 20
</s400> <s200> <s210>53</s210> <s211>23</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 53
tcctggagcc cgacactgtg tgt                                             23
</s400> <s200> <s210>54</s210> <s211>22</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 54
tgtcctgccc actcctcagc ct                                              22
</s400> <s200> <s210>55</s210> <s211>20</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 55
tggctgacca ctgtccctct                                                 20
</s400> <s200> <s210>56</s210> <s211>22</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 56
ccccaggacc ccagctgtcc aa                                              22
</s400> <s200> <s210>57</s210> <s211>21</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 57
gctggcagtg gcctgtgtgg a                                               21
</s400> <s200> <s210>58</s210> <s211>24</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 58
aacagtgacc aaaatgacag tgac                                            24
</s400> <s200> <s210>59</s210> <s211>19</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 59
tggctcagca ggtgacagc                                                  19
</s400> <s200> <s210>60</s210> <s211>19</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 60
tggtggcagg tgggctact                                                  19
</s400> <s200> <s210>61</s210> <s211>19</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 61
gcctggcaga cgatgtcca                                                  19
</s400> <s200> <s210>62</s210> <s211>19</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 62
caactgcctg aggggttct                                                  19
</s400> <s200> <s210>63</s210> <s211>20</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 63
ctgtccccac actttctcct                                                 20
</s400> <s200> <s210>64</s210> <s211>20</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 64
tgagctccag tcccctccag                                                 20
</s400> <s200> <s210>65</s210> <s211>19</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 65
tggccactca caatctcct                                                  19
</s400> <s200> <s210>66</s210> <s211>19</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 66
gccttgacac cctccacta                                                  19
</s400> <s200> <s210>67</s210> <s211>19</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 67
ggcacaggga ggagaagtg                                                  19
</s400> <s200> <s210>68</s210> <s211>19</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 68
cggcaccgct gatcatgca                                                  19
</s400> <s200> <s210>69</s210> <s211>19</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 69
ccagggccag gtgtgactg                                                  19
</s400> <s200> <s210>70</s210> <s211>20</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 70
tgggcccaga gtaactgaca                                                 20
</s400> <s200> <s210>71</s210> <s211>21</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 71
ggccctgatt tgggtgtttt a                                               21
</s400> <s200> <s210>72</s210> <s211>19</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 72
ggacgctaac cagaaccac                                                  19
</s400> <s200> <s210>73</s210> <s211>20</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 73
caccactgac tctctcgtct                                                 20
</s400> <s200> <s210>74</s210> <s211>18</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 74
ccatccccca gccccatc                                                   18
</s400> <s200> <s210>75</s210> <s211>22</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 75
gagatcgtgc tggtggtgtt ct                                              22
</s400> <s200> <s210>76</s210> <s211>21</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 76
cttcctggtc tggaaacctg g                                               21
</s400> <s200> <s210>77</s210> <s211>20</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 77
ctcttccctg gggccctggc                                                 20
</s400> <s200> <s210>78</s210> <s211>22</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 78
tgcgggggag cttgtggcac ag                                              22
</s400> <s200> <s210>79</s210> <s211>20</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 79
gggcatccgc ttcctgcaga                                                 20
</s400> <s200> <s210>80</s210> <s211>20</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 80
ctgggcccct accctaaccc                                                 20
</s400> <s200> <s210>81</s210> <s211>22</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 81
tcctggagcc cgaactgtgt gt                                              22
</s400> <s200> <s210>82</s210> <s211>22</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 82
tgtcctgccc actcctcagc ct                                              22
</s400> <s200> <s210>83</s210> <s211>22</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 83
ccccaggacc ccagctgtcc aa                                              22
</s400> <s200> <s210>84</s210> <s211>20</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 84
aggctgacca ctgtccctct                                                 20
</s400> <s200> <s210>85</s210> <s211>21</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 85
gctggcagtg gcctgtgtgg a                                               21
</s400> <s200> <s210>86</s210> <s211>24</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 86
aacagtgacc aaaatgacag tgac                                            24
</s400> <s200> <s210>87</s210> <s211>22</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 87
ctgcagcagt ggaaccttaa tg                                              22
</s400> <s200> <s210>88</s210> <s211>22</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 88
gttcgagtgc tccagcttct tg                                              22
</s400> <s200> <s210>89</s210> <s211>22</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 89
agggcatcat gctgagctac at                                              22
</s400> <s200> <s210>90</s210> <s211>21</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 90
tttagccagt ggtggggttc a                                               21
</s400> <s200> <s210>91</s210> <s211>20</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 91
gttcagcagg gtggcaacat                                                 20
</s400> <s200> <s210>92</s210> <s211>21</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 92
gccagatggt tttcaacgac a                                               21
</s400> <s200> <s210>93</s210> <s211>28</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> <s220> <s221>misc_difference</s221> <s222>(9)</s222> <s223>Base change made to create a restriction enzyme
      site.</s223> </s220> </s200> <s400> 93
cagtggaagc ttaatgccca ggatgatc                                        28
</s400> <s200> <s210>94</s210> <s211>35</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> <s220> <s221>misc_difference</s221> <s222>(7)..(8)</s222> <s223>Base changes made to create a restriction
      enzyme site.</s223> </s220> </s200> <s400> 94
caggaggatc cagtttagcc agtggtgggg gttca                                35
</s400> <s200> <s210>95</s210> <s211>9</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 95
ccacacccg                                                              9
</s400> <s200> <s210>96</s210> <s211>20</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 96
tcagacccgg gtgagttagg                                                 20
</s400> <s200> <s210>97</s210> <s211>20</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 97
caatcaccag gaaaaatccc                                                 20
</s400> <s200> <s210>98</s210> <s211>20</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 98
ggatattcag gtaggacctg                                                 20
</s400> <s200> <s210>99</s210> <s211>14</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 99
ttcctttaag aggt                                                       14
</s400> <s200> <s210>100</s210> <s211>10</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 100
ttccccatga                                                            10
</s400> <s200> <s210>101</s210> <s211>22</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 101
ctgcagcagt ggaaccttaa tg                                              22
</s400> <s200> <s210>102</s210> <s211>22</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 102
gttcgagtgc tccagcttct tg                                              22
</s400> <s200> <s210>103</s210> <s211>21</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 103
gggcatcatg ctgagctaca t                                               21
</s400> <s200> <s210>104</s210> <s211>21</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 104
tttagccagt ggtggggttc a                                               21
</s400> <s200> <s210>105</s210> <s211>20</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 105
gttcagcagg gtggcaacat                                                 20
</s400> <s200> <s210>106</s210> <s211>21</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> </s200> <s400> 106
gccagatggt tttcaacgac a                                               21
</s400> <s200> <s210>107</s210> <s211>26</s211> <s212>PRT</s212> <s213>Homo sapiens</s213> </s200> <s400> 107
Phe Leu Ile Val Leu Val Cys Leu Ile Phe Ser Val Leu Ser Thr Ile
  1               5                  10                  15
Glu Gln Tyr Ala Ala Leu Ala Thr Gly Thr
             20                  25
</s400> <s200> <s210>108</s210> <s211>61</s211> <s212>PRT</s212> <s213>Homo sapiens</s213> </s200> <s400> 108
Leu Phe Trp Met Glu Ile Val Leu Val Val Phe Phe Gly Thr Glu Tyr
  1               5                  10                  15
Val Val Arg Leu Trp Ser Ala Gly Cys Arg Ser Lys Tyr Val Gly Leu
             20                  25                  30
Trp Gly Arg Leu Arg Phe Ala Arg Lys Pro Ile Ser Ile Ile Asp Leu
         35                  40                  45
Ile Val Val Val Ala Ser Met Val Val Leu Cys Val Gly
     50                  55                  60
</s400> <s200> <s210>109</s210> <s211>137</s211> <s212>PRT</s212> <s213>Homo sapiens</s213> </s200> <s400> 109
Ser Lys Gly Gln Val Phe Ala Thr Ser Ala Ile Arg Gly Ile Arg Phe
  1               5                  10                  15
Leu Gln Ile Leu Arg Met Leu His Val Asp Arg Gln Gly Gly Thr Trp
             20                  25                  30
Arg Leu Leu Gly Ser Val Val Phe Ile His Arg Gln Glu Leu Ile Thr
         35                  40                  45
Thr Leu Tyr Ile Gly Phe Leu Gly Leu Ile Phe Ser Ser Tyr Phe Val
     50                  55                  60
Tyr Leu Ala Glu Lys Asp Ala Val Asn Glu Ser Gly Arg Val Glu Phe
 65                  70                  75                  80
Gly Ser Tyr Ala Asp Ala Leu Trp Trp Gly Val Val Thr Val Thr Thr
                 85                  90                  95
Ile Gly Tyr Gly Asp Lys Val Pro Gln Thr Trp Val Gly Lys Thr Ile
            100                 105                 110
Ala Ser Cys Phe Ser Val Phe Ala Ile Ser Phe Phe Ala Leu Pro Ala
        115                 120                 125
Gly Ile Leu Gly Ser Gly Phe Ala Leu
    130                 135
</s400> <s200> <s210>110</s210> <s211>66</s211> <s212>PRT</s212> <s213>Drosophila melanogaster</s213> </s200> <s400> 110
Ile Leu Leu Ser Ile Val Ile Phe Cys Leu Glu Thr Leu Pro Glu Phe
  1               5                  10                  15
Lys His Tyr Lys Val Phe Asn Thr Thr Thr Asn Gly Thr Lys Ile Glu
             20                  25                  30
Glu Asp Glu Val Pro Asp Ile Thr Asp Pro Phe Phe Leu Ile Glu Thr
         35                  40                  45
Leu Cys Ile Ile Trp Phe Thr Phe Glu Leu Thr Val Arg Phe Leu Ala
     50                  55                  60
Cys Pro
 65
</s400> <s200> <s210>111</s210> <s211>123</s211> <s212>PRT</s212> <s213>Drosophila melanogaster</s213> </s200> <s400> 111
Asn Lys Leu Asn Phe Cys Arg Asp Val Met Asn Val Ile Asp Ile Ile
  1               5                  10                  15
Ala Ile Ile Pro Tyr Phe Ile Thr Leu Ala Thr Val Val Ala Glu Glu
             20                  25                  30
Glu Asp Thr Leu Asn Leu Pro Lys Ala Pro Val Ser Pro Gln Asp Lys
         35                  40                  45
Ser Ser Asn Gln Ala Met Ser Leu Ala Ile Leu Arg Val Ile Arg Leu
     50                  55                  60
Val Arg Val Phe Arg Ile Phe Lys Leu Ser Arg His Ser Lys Gly Leu
 65                  70                  75                  80
Gln Ile Leu Gly Arg Thr Leu Lys Ala Ser Met Arg Glu Leu Gly Leu
                 85                  90                  95
Leu Ile Phe Phe Leu Phe Ile Gly Val Val Leu Phe Ser Ser Ala Val
            100                 105                 110
Tyr Phe Ala Glu Ala Gly Ser Glu Asn Ser Phe
        115                 120
</s400> <s200> <s210>112</s210> <s211>58</s211> <s212>PRT</s212> <s213>Drosophila melanogaster</s213> </s200> <s400> 112
Phe Lys Ser Ile Pro Asp Ala Phe Trp Trp Ala Val Val Thr Met Thr
  1               5                  10                  15
Thr Val Gly Tyr Gly Asp Met Thr Pro Val Gly Phe Trp Gly Lys Ile
             20                  25                  30
Val Gly Ser Leu Cys Val Val Ala Gly Val Leu Thr Ile Ala Leu Pro
         35                  40                  45
Val Pro Val Ile Val Ser Asn Phe Asn Tyr
     50                  55
</s400> <s200> <s210>113</s210> <s211>376</s211> <s212>PRT</s212> <s213>Xenopus laevis</s213> </s200> <s400> 113
Met Asn Glu Asn Ala Ile Asn Ser Leu Tyr Glu Ala Ile Pro Leu Pro
  1               5                  10                  15
Gln Asp Gly Ser Ser Asn Gly Gln Arg Gln Glu Asp Arg Gln Ala Asn
             20                  25                  30
Ser Phe Glu Leu Lys Arg Glu Thr Leu Val Ala Thr Asp Pro Pro Arg
         35                  40                  45
Pro Thr Ile Asn Leu Asp Pro Arg Val Ser Ile Tyr Ser Gly Arg Arg
     50                  55                  60
Pro Leu Phe Ser Arg Thr Asn Ile Gln Gly Arg Val Tyr Asn Phe Leu
 65                  70                  75                  80
Glu Arg Pro Thr Gly Trp Lys Cys Phe Val Tyr His Phe Thr Val Phe
                 85                  90                  95
Leu Ile Val Leu Ile Cys Leu Ile Phe Ser Val Leu Ser Thr Ile Gln
            100                 105                 110
Gln Tyr Asn Asn Leu Ala Thr Glu Thr Leu Phe Trp Met Glu Ile Val
        115                 120                 125
Leu Val Val Phe Phe Gly Ala Glu Tyr Val Val Arg Leu Trp Ser Ala
    130                 135                 140
Gly Cys Arg Ser Lys Tyr Val Gly Val Trp Gly Arg Leu Arg Phe Ala
145                 150                 155                 160
Arg Lys Pro Ile Ser Val Ile Asp Leu Ile Val Val Val Ala Ser Val
                165                 170                 175
Ile Val Leu Cys Val Gly Ser Asn Gly Gln Val Phe Ala Thr Ser Ala
            180                 185                 190
Ile Arg Gly Ile Arg Phe Leu Gln Ile Leu Arg Met Leu His Val Asp
        195                 200                 205
Arg Gln Gly Gly Thr Trp Arg Leu Leu Gly Ser Val Val Phe Ile His
    210                 215                 220
Arg Gln Glu Leu Ile Thr Thr Leu Tyr Ile Gly Phe Leu Gly Leu Ile
225                 230                 235                 240
Phe Ser Ser Tyr Phe Val Tyr Leu Ala Glu Lys Asp Ala Ile Asp Ser
                245                 250                 255
Ser Gly Glu Tyr Gln Phe Gly Ser Tyr Ala Asp Ala Leu Trp Trp Gly
            260                 265                 270
Val Val Thr Val Thr Thr Ile Gly Tyr Gly Asp Lys Val Pro Gln Thr
        275                 280                 285
Trp Ile Gly Lys Thr Ile Ala Ser Cys Phe Ser Val Phe Ala Ile Ser
    290                 295                 300
Phe Phe Ala Leu Pro Ala Gly Ile Leu Gly Ser Gly Phe Ala Leu Lys
305                 310                 315                 320
Val Gln Gln Lys Gln Arg Gln Lys His Phe Asn Arg Gln Ile Pro Ala
                325                 330                 335
Ala Ala Ser Leu Ile Gln Thr Ala Trp Arg Cys Tyr Ala Ala Glu Asn
            340                 345                 350
Pro Asp Ser Ala Thr Trp Lys Ile Tyr Ile Arg Lys Gln Ser Arg Asn
        355                 360                 365
His His Ile Met Ser Pro Ser Pro
    370                 375
</s400> <s200> <s210>114</s210> <s211>570</s211> <s212>PRT</s212> <s213>Homo sapiens</s213> </s200> <s400> 114
Gln Gly Arg Val Tyr Asn Phe Leu Glu Arg Pro Thr Gly Trp Lys Cys
  1               5                  10                  15
Phe Val Tyr His Phe Ala Val Phe Leu Ile Val Leu Val Cys Leu Ile
             20                  25                  30
Phe Ser Val Leu Ser Thr Ile Glu Gln Tyr Ala Ala Leu Ala Thr Gly
         35                  40                  45
Thr Leu Phe Trp Met Glu Ile Val Leu Val Val Phe Phe Gly Thr Glu
     50                  55                  60
Tyr Val Val Arg Leu Trp Ser Ala Gly Cys Arg Ser Lys Tyr Val Gly
 65                  70                  75                  80
Leu Trp Gly Arg Leu Arg Phe Ala Arg Lys Pro Ile Ser Ile Ile Asp
                 85                  90                  95
Leu Ile Val Val Val Ala Ser Met Val Val Leu Cys Val Gly Ser Lys
            100                 105                 110
Gly Gln Val Phe Ala Thr Ser Ala Ile Arg Gly Ile Arg Phe Leu Gln
        115                 120                 125
Ile Leu Arg Met Leu His Val Asp Arg Gln Gly Gly Thr Trp Arg Leu
    130                 135                 140
Leu Gly Ser Val Val Phe Ile His Arg Gln Glu Leu Ile Thr Thr Leu
145                 150                 155                 160
Tyr Ile Gly Phe Leu Gly Leu Ile Phe Ser Ser Tyr Phe Val Tyr Leu
                165                 170                 175
Ala Glu Lys Asp Ala Val Asn Glu Ser Gly Arg Val Glu Phe Gly Ser
            180                 185                 190
Tyr Ala Asp Ala Leu Trp Trp Gly Val Val Thr Val Thr Thr Ile Gly
        195                 200                 205
Tyr Gly Asp Lys Val Pro Gln Thr Trp Val Gly Lys Thr Ile Ala Ser
    210                 215                 220
Cys Phe Ser Val Phe Ala Ile Ser Phe Phe Ala Leu Pro Ala Gly Ile
225                 230                 235                 240
Leu Gly Ser Gly Phe Ala Leu Lys Val Gln Gln Lys Gln Arg Gln Lys
                245                 250                 255
His Phe Asn Arg Gln Ile Pro Ala Ala Ala Ser Leu Ile Gln Thr Ala
            260                 265                 270
Trp Arg Cys Tyr Ala Ala Glu Asn Pro Asp Ser Ser Thr Trp Lys Ile
        275                 280                 285
Tyr Ile Arg Lys Ala Pro Arg Ser His Thr Leu Leu Ser Pro Ser Pro
    290                 295                 300
Lys Pro Lys Lys Ser Val Val Val Lys Lys Lys Lys Phe Lys Leu Asp
305                 310                 315                 320
Lys Asp Asn Gly Val Thr Pro Gly Glu Lys Met Leu Thr Val Pro His
                325                 330                 335
Ile Thr Cys Asp Pro Pro Glu Glu Arg Arg Leu Asp His Phe Ser Val
            340                 345                 350
Asp Gly Tyr Asp Ser Ser Val Arg Lys Ser Pro Thr Leu Leu Glu Val
        355                 360                 365
Ser Met Pro His Phe Met Arg Thr Asn Ser Phe Ala Glu Asp Leu Asp
    370                 375                 380
Leu Glu Gly Glu Thr Leu Leu Thr Pro Ile Thr His Ile Ser Gln Leu
385                 390                 395                 400
Arg Glu His His Arg Ala Thr Ile Lys Val Ile Arg Arg Met Gln Tyr
                405                 410                 415
Phe Val Ala Lys Lys Lys Phe Gln Gln Ala Arg Lys Pro Tyr Asp Val
            420                 425                 430
Arg Asp Val Ile Glu Gln Tyr Ser Gln Gly His Leu Asn Leu Met Arg
        435                 440                 445
Val Ile Lys Glu Leu Gln Arg Arg Leu Asp Gln Ser Ile Gly Lys Pro
    450                 455                 460
Ser Leu Phe Ile Ser Val Ser Glu Lys Ser Lys Asp Arg Gly Ser Asn
465                 470                 475                 480
Thr Ile Gly Ala Arg Leu Asn Arg Val Glu Asp Lys Val Thr Gln Leu
                485                 490                 495
Asp Gln Arg Leu Ala Leu Ile Thr Asp Met Leu His Gln Leu Leu Ser
            500                 505                 510
Leu His Gly Gly Ser Thr Pro Gly Ser Gly Gly Pro Pro Arg Glu Gly
        515                 520                 525
Gly Ala His Ile Thr Gln Pro Cys Gly Ser Gly Gly Ser Val Asp Pro
    530                 535                 540
Glu Leu Phe Leu Pro Ser Asn Thr Leu Pro Thr Tyr Glu Gln Leu Thr
545                 550                 555                 560
Val Pro Arg Arg Gly Pro Asp Glu Gly Ser
                565                 570
</s400> <s200> <s210>115</s210> <s211>2821</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> <s220> <s221>CDS</s221> <s222>(88)..(1830)</s222> </s220> </s200> <s400> 115
ggcttcctcg agcgtcccac cggctggaag ttgtagacgc ggccctggac gtgggtgcgc     60
gccaacaccg ggcggcgcgt gctgtag atg gag acg cgc ggg tct agg ctc acc    114
                              Met Glu Thr Arg Gly Ser Arg Leu Thr
                                1               5
ggc ggc cag ggc cgc gtc tac aac ttc ctc gag cgt ccc acc ggc tgg      162
Gly Gly Gln Gly Arg Val Tyr Asn Phe Leu Glu Arg Pro Thr Gly Trp
 10                  15                  20                  25
aaa tgc ttc gtt tac cac ttc gcc gtc ttc ctc atc gtc ctg gtc tgc      210
Lys Cys Phe Val Tyr His Phe Ala Val Phe Leu Ile Val Leu Val Cys
                 30                  35                  40
ctc atc ttc agc gtg ctg tcc acc atc gag cag tat gcc gcc ctg gcc      258
Leu Ile Phe Ser Val Leu Ser Thr Ile Glu Gln Tyr Ala Ala Leu Ala
             45                  50                  55
acg ggg act ctc ttc tgg atg gag atc gtg ctg gtg gtg ttc ttc ggg      306
Thr Gly Thr Leu Phe Trp Met Glu Ile Val Leu Val Val Phe Phe Gly
         60                  65                  70
acg gag tac gtg gtc cgc ctc tgg tcc gcc ggc tgc cgc agc aag tac      354
Thr Glu Tyr Val Val Arg Leu Trp Ser Ala Gly Cys Arg Ser Lys Tyr
     75                  80                  85
gtg ggc ctc tgg ggg cgg ctg cgc ttt gcc cgg aag ccc att tcc atc      402
Val Gly Leu Trp Gly Arg Leu Arg Phe Ala Arg Lys Pro Ile Ser Ile
 90                  95                 100                 105
atc gac ctc atc gtg gtc gtg gcc tcc atg gtg gtc ctc tgc gtg ggc      450
Ile Asp Leu Ile Val Val Val Ala Ser Met Val Val Leu Cys Val Gly
                110                 115                 120
tcc aag ggg cag gtg ttt gcc acg tcg gcc atc agg ggc atc cgc ttc      498
Ser Lys Gly Gln Val Phe Ala Thr Ser Ala Ile Arg Gly Ile Arg Phe
            125                 130                 135
ctg cag atc ctg agg atg cta cac gtc gac cgc cag gga ggc acc tgg      546
Leu Gln Ile Leu Arg Met Leu His Val Asp Arg Gln Gly Gly Thr Trp
        140                 145                 150
agg ctc ctg ggc tcc gtg gtc ttc atc cac cgc cag gag ctg ata acc      594
Arg Leu Leu Gly Ser Val Val Phe Ile His Arg Gln Glu Leu Ile Thr
    155                 160                 165
acc ctg tac atc ggc ttc ctg ggc ctc atc ttc tcc tcg tac ttt gtg      642
Thr Leu Tyr Ile Gly Phe Leu Gly Leu Ile Phe Ser Ser Tyr Phe Val
170                 175                 180                 185
tac ctg gct gag aag gac gcg gtg aac gag tca ggc cgc gtg gag ttc      690
Tyr Leu Ala Glu Lys Asp Ala Val Asn Glu Ser Gly Arg Val Glu Phe
                190                 195                 200
ggc agc tac gca gat gcg ctg tgg tgg ggg gtg gtc aca gtc acc acc      738
Gly Ser Tyr Ala Asp Ala Leu Trp Trp Gly Val Val Thr Val Thr Thr
            205                 210                 215
atc ggc tat ggg gac aag gtg ccc cag acg tgg gtc ggg aag acc atc      786
Ile Gly Tyr Gly Asp Lys Val Pro Gln Thr Trp Val Gly Lys Thr Ile
        220                 225                 230
gcc tcc tgc ttc tct gtc ttt gcc atc tcc ttc ttt gcg ctc cca gcg      834
Ala Ser Cys Phe Ser Val Phe Ala Ile Ser Phe Phe Ala Leu Pro Ala
    235                 240                 245
ggg att ctt ggc tcg ggg ttt gcc ctg aag gtg cag cag aag cag agg      882
Gly Ile Leu Gly Ser Gly Phe Ala Leu Lys Val Gln Gln Lys Gln Arg
250                 255                 260                 265
cag aag cac ttc aac cgg cag atc ccg gcg gca gcc tca ctc att cag      930
Gln Lys His Phe Asn Arg Gln Ile Pro Ala Ala Ala Ser Leu Ile Gln
                270                 275                 280
acc gca tgg agg tgc tat gct gcc gag aac ccc gac tcc tcc acc tgg      978
Thr Ala Trp Arg Cys Tyr Ala Ala Glu Asn Pro Asp Ser Ser Thr Trp
            285                 290                 295
aag atc tac atc cgg aag gcc ccc cgg agc cac act ctg ctg tca ccc     1026
Lys Ile Tyr Ile Arg Lys Ala Pro Arg Ser His Thr Leu Leu Ser Pro
        300                 305                 310
agc ccc aaa ccc aag aag tct gtg gtg gta aag aaa aaa aag ttc aag     1074
Ser Pro Lys Pro Lys Lys Ser Val Val Val Lys Lys Lys Lys Phe Lys
    315                 320                 325
ctg gac aaa gac aat ggg gtg act cct gga gag aag atg ctc aca gtc     1122
Leu Asp Lys Asp Asn Gly Val Thr Pro Gly Glu Lys Met Leu Thr Val
330                 335                 340                 345
ccc cat atc acg tgc gac ccc cca gaa gag cgg cgg ctg gac cac ttc     1170
Pro His Ile Thr Cys Asp Pro Pro Glu Glu Arg Arg Leu Asp His Phe
                350                 355                 360
tct gtc gac ggc tat gac agt tct gta agg aag agc cca aca ctg ctg     1218
Ser Val Asp Gly Tyr Asp Ser Ser Val Arg Lys Ser Pro Thr Leu Leu
            365                 370                 375
gaa gtg agc atg ccc cat ttc atg aga acc aac agc ttc gcc gag gac     1266
Glu Val Ser Met Pro His Phe Met Arg Thr Asn Ser Phe Ala Glu Asp
        380                 385                 390
ctg gac ctg gaa ggg gag act ctg ctg aca ccc atc acc cac atc tca     1314
Leu Asp Leu Glu Gly Glu Thr Leu Leu Thr Pro Ile Thr His Ile Ser
    395                 400                 405
cag ctg cgg gaa cac cat cgg gcc acc att aag gtc att cga cgc atg     1362
Gln Leu Arg Glu His His Arg Ala Thr Ile Lys Val Ile Arg Arg Met
410                 415                 420                 425
cag tac ttt gtg gcc aag aag aaa ttc cag caa gcg cgg aag cct tac     1410
Gln Tyr Phe Val Ala Lys Lys Lys Phe Gln Gln Ala Arg Lys Pro Tyr
                430                 435                 440
gat gtg cgg gac gtc att gag cag tac tcg cag ggc cac ctc aac ctc     1458
Asp Val Arg Asp Val Ile Glu Gln Tyr Ser Gln Gly His Leu Asn Leu
            445                 450                 455
atg gtg cgc atc aag gag ctg cag agg agg ctg gac cag tcc att ggg     1506
Met Val Arg Ile Lys Glu Leu Gln Arg Arg Leu Asp Gln Ser Ile Gly
        460                 465                 470
aag ccc tca ctg ttc atc tcc gtc tca gaa aag agc aag gat cgc ggc     1554
Lys Pro Ser Leu Phe Ile Ser Val Ser Glu Lys Ser Lys Asp Arg Gly
    475                 480                 485
agc aac acg atc ggc gcc cgc ctg aac cga gta gaa gac aag gtg acg     1602
Ser Asn Thr Ile Gly Ala Arg Leu Asn Arg Val Glu Asp Lys Val Thr
490                 495                 500                 505
cag ctg gac cag agg ctg gca ctc atc acc gac atg ctt cac cag ctg     1650
Gln Leu Asp Gln Arg Leu Ala Leu Ile Thr Asp Met Leu His Gln Leu
                510                 515                 520
ctc tcc ttg cac ggt ggc agc acc ccc ggc agc ggc ggc ccc ccc aga     1698
Leu Ser Leu His Gly Gly Ser Thr Pro Gly Ser Gly Gly Pro Pro Arg
            525                 530                 535
gag ggc ggg gcc cac atc acc cag ccc tgc ggc agt ggc ggc tcc gtc     1746
Glu Gly Gly Ala His Ile Thr Gln Pro Cys Gly Ser Gly Gly Ser Val
        540                 545                 550
gac cct gag ctc ttc ctg ccc agc aac acc ctg ccc acc tac gag cag     1794
Asp Pro Glu Leu Phe Leu Pro Ser Asn Thr Leu Pro Thr Tyr Glu Gln
    555                 560                 565
ctg acc gtg ccc agg agg ggc ccc gat gag ggg tcc tgaggagggg          1840
Leu Thr Val Pro Arg Arg Gly Pro Asp Glu Gly Ser
570                 575                 580
atggggctgg gggatgggcc tgagtgagag gggaggccaa gagtggcccc acctggccct   1900
ctctgaagga ggccacctcc taaaaggccc agagagaaga gccccactct cagaggcccc   1960
aataccccat ggaccatgct gtctggcaca gcctgcactt gggggctcag caaggccacc   2020
tcttcctggc cggtgtgggg gccccgtctc aggtctgagt tgttacccca agcgccctgg   2080
cccccacatg gtgatgttga catcactggc atggtggttg ggacccagtg gcagggcaca   2140
gggcctggcc catgtatggc caggaagtag cacaggctga gtgcaggccc accctgcttg   2200
gcccaggggg cttcctgagg ggagacagag caacccctgg accccagcct caaatccagg   2260
accctgccag gcacaggcag ggcaggacca gcccacgctg actacagggc caccggcaat   2320
aaaagcccag gagcccattt ggagggcctg ggcctggctc cctcactctc aggaaatgct   2380
gacccatggg caggagactg tggagactgc tcctgagccc ccagcttcca gcaggaggga   2440
cagtctcacc atttccccag ggcacgtggt tgagtggggg gaacgcccac ttccctgggt   2500
tagactgcca gctcttccta gctggagagg agccctgcct ctccgcccct gagcccactg   2560
tgcgtggggc tcccgcctcc aacccctcgc ccagtcccag cagccagcca aacacacaga   2620
aggggactgc cacctcccct tgccagctgc tgagccgcag agaagtgacg gttcctacac   2680
aggacagggg ttccttctgg gcattacatc gcatagaaat caataatttg tggtgatttg   2740
gatctgtgtt ttaatgagtt tcacagtgtg attttgatta ttaattgtgc aagcttttcc   2800
taataaacgt ggagaatcac a                                             2821
</s400> <s200> <s210>116</s210> <s211>581</s211> <s212>PRT</s212> <s213>Homo sapiens</s213> </s200> <s400> 116
Met Glu Thr Arg Gly Ser Arg Leu Thr Gly Gly Gln Gly Arg Val Tyr
  1               5                  10                  15
Asn Phe Leu Glu Arg Pro Thr Gly Trp Lys Cys Phe Val Tyr His Phe
             20                  25                  30
Ala Val Phe Leu Ile Val Leu Val Cys Leu Ile Phe Ser Val Leu Ser
         35                  40                  45
Thr Ile Glu Gln Tyr Ala Ala Leu Ala Thr Gly Thr Leu Phe Trp Met
     50                  55                  60
Glu Ile Val Leu Val Val Phe Phe Gly Thr Glu Tyr Val Val Arg Leu
 65                  70                  75                  80
Trp Ser Ala Gly Cys Arg Ser Lys Tyr Val Gly Leu Trp Gly Arg Leu
                 85                  90                  95
Arg Phe Ala Arg Lys Pro Ile Ser Ile Ile Asp Leu Ile Val Val Val
            100                 105                 110
Ala Ser Met Val Val Leu Cys Val Gly Ser Lys Gly Gln Val Phe Ala
        115                 120                 125
Thr Ser Ala Ile Arg Gly Ile Arg Phe Leu Gln Ile Leu Arg Met Leu
    130                 135                 140
His Val Asp Arg Gln Gly Gly Thr Trp Arg Leu Leu Gly Ser Val Val
145                 150                 155                 160
Phe Ile His Arg Gln Glu Leu Ile Thr Thr Leu Tyr Ile Gly Phe Leu
                165                 170                 175
Gly Leu Ile Phe Ser Ser Tyr Phe Val Tyr Leu Ala Glu Lys Asp Ala
            180                 185                 190
Val Asn Glu Ser Gly Arg Val Glu Phe Gly Ser Tyr Ala Asp Ala Leu
        195                 200                 205
Trp Trp Gly Val Val Thr Val Thr Thr Ile Gly Tyr Gly Asp Lys Val
    210                 215                 220
Pro Gln Thr Trp Val Gly Lys Thr Ile Ala Ser Cys Phe Ser Val Phe
225                 230                 235                 240
Ala Ile Ser Phe Phe Ala Leu Pro Ala Gly Ile Leu Gly Ser Gly Phe
                245                 250                 255
Ala Leu Lys Val Gln Gln Lys Gln Arg Gln Lys His Phe Asn Arg Gln
            260                 265                 270
Ile Pro Ala Ala Ala Ser Leu Ile Gln Thr Ala Trp Arg Cys Tyr Ala
        275                 280                 285
Ala Glu Asn Pro Asp Ser Ser Thr Trp Lys Ile Tyr Ile Arg Lys Ala
    290                 295                 300
Pro Arg Ser His Thr Leu Leu Ser Pro Ser Pro Lys Pro Lys Lys Ser
305                 310                 315                 320
Val Val Val Lys Lys Lys Lys Phe Lys Leu Asp Lys Asp Asn Gly Val
                325                 330                 335
Thr Pro Gly Glu Lys Met Leu Thr Val Pro His Ile Thr Cys Asp Pro
            340                 345                 350
Pro Glu Glu Arg Arg Leu Asp His Phe Ser Val Asp Gly Tyr Asp Ser
        355                 360                 365
Ser Val Arg Lys Ser Pro Thr Leu Leu Glu Val Ser Met Pro His Phe
    370                 375                 380
Met Arg Thr Asn Ser Phe Ala Glu Asp Leu Asp Leu Glu Gly Glu Thr
385                 390                 395                 400
Leu Leu Thr Pro Ile Thr His Ile Ser Gln Leu Arg Glu His His Arg
                405                 410                 415
Ala Thr Ile Lys Val Ile Arg Arg Met Gln Tyr Phe Val Ala Lys Lys
            420                 425                 430
Lys Phe Gln Gln Ala Arg Lys Pro Tyr Asp Val Arg Asp Val Ile Glu
        435                 440                 445
Gln Tyr Ser Gln Gly His Leu Asn Leu Met Val Arg Ile Lys Glu Leu
    450                 455                 460
Gln Arg Arg Leu Asp Gln Ser Ile Gly Lys Pro Ser Leu Phe Ile Ser
465                 470                 475                 480
Val Ser Glu Lys Ser Lys Asp Arg Gly Ser Asn Thr Ile Gly Ala Arg
                485                 490                 495
Leu Asn Arg Val Glu Asp Lys Val Thr Gln Leu Asp Gln Arg Leu Ala
            500                 505                 510
Leu Ile Thr Asp Met Leu His Gln Leu Leu Ser Leu His Gly Gly Ser
        515                 520                 525
Thr Pro Gly Ser Gly Gly Pro Pro Arg Glu Gly Gly Ala His Ile Thr
    530                 535                 540
Gln Pro Cys Gly Ser Gly Gly Ser Val Asp Pro Glu Leu Phe Leu Pro
545                 550                 555                 560
Ser Asn Thr Leu Pro Thr Tyr Glu Gln Leu Thr Val Pro Arg Arg Gly
                565                 570                 575
Pro Asp Glu Gly Ser
            580
</s400> </sequence-list-new-rules> </sequence-cwu> </p>
    </div></div></div><div class="patent-section patent-tabular-section"><a id="backward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Cited Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1997023598A2?cl=en">WO1997023598A2</a></td><td class="patent-data-table-td patent-date-value">Dec 20, 1996</td><td class="patent-data-table-td patent-date-value">Jul 3, 1997</td><td class="patent-data-table-td ">Univ Utah Res Found</td><td class="patent-data-table-td ">A long qt syndrome gene which encodes kvlqt1 and its association with mink</td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="npl-citations"></a><div class="patent-section-header"><span class="patent-section-title">Non-Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th colspan="3"class="patent-data-table-th">Reference</th></tr></thead><tr><td class="patent-data-table-td ">1</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Ackerman, M.J. "<a href='http://scholar.google.com/scholar?q="The+Long+QT+Syndrome%3A+Ion+Channel+Diseases+of+the+Heart"'>The Long QT Syndrome: Ion Channel Diseases of the Heart</a>", Mayo Clin. Proc., 1998; 73:250-269.</td></tr><tr><td class="patent-data-table-td ">2</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Barhanin, J. et al. (1997). GenBank Accession No. AF000571.1.</td></tr><tr><td class="patent-data-table-td ">3</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Benhorin,J. et al. "<a href='http://scholar.google.com/scholar?q="Evidence+of+Genetic+Heterogeneity+in+the+Long+QT+Syndrome"'>Evidence of Genetic Heterogeneity in the Long QT Syndrome</a>", Science, Jun. 25, 1993; 260:1960-1962.</td></tr><tr><td class="patent-data-table-td ">4</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Bulman "<a href='http://scholar.google.com/scholar?q="Phenotype+variation+and+newcomers+in+ion+channel+disorders"'>Phenotype variation and newcomers in ion channel disorders</a>", Human Molecular Genetics, 1997; 6(10) Review:1679-1685.</td></tr><tr><td class="patent-data-table-td ">5</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Chouabe, C. et al. "<a href='http://scholar.google.com/scholar?q="Properties+of+KVLQT1+K%2B+channel+mutations+in+Romano-Ward+and+Jervell+and+Lange-Nielsen+inherited+cardiac+arrhythmias"'>Properties of KVLQT1 K+ channel mutations in Romano-Ward and Jervell and Lange-Nielsen inherited cardiac arrhythmias</a>", The EMBO Journal, 1997; 16(17):5472-5479.</td></tr><tr><td class="patent-data-table-td ">6</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Coonar, et al. "<a href='http://scholar.google.com/scholar?q="Molecular+Genetics+of+Familial+Cardiomyopathies"'>Molecular Genetics of Familial Cardiomyopathies</a>", Advances in Genetics, _; 35:285-324.</td></tr><tr><td class="patent-data-table-td ">7</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Curran, M. et al. "<a href='http://scholar.google.com/scholar?q="Locus+Heterogeneity+of+Autosomal+Dominant+Long+QT+Syndrome"'>Locus Heterogeneity of Autosomal Dominant Long QT Syndrome</a>", J. Clin. Invest., Aug. 1993; 92:799-803.</td></tr><tr><td class="patent-data-table-td ">8</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">de Jager, et al. "<a href='http://scholar.google.com/scholar?q="Evidence+of+a+long+QT+founder+gene+with+varying+phenotypic+expression+in+South+African+families"'>Evidence of a long QT founder gene with varying phenotypic expression in South African families</a>", J. Med. Genet., 1996; 33:567-573.</td></tr><tr><td class="patent-data-table-td ">9</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Donger, C. et al. "<a href='http://scholar.google.com/scholar?q="KVLQT1+C-Terminal+Missense+Mutation+Causes+a+Forme+Fruste+Long-QT+Syndrome"'>KVLQT1 C-Terminal Missense Mutation Causes a Forme Fruste Long-QT Syndrome</a>", Circulation, 1997; 96:2778-2781.</td></tr><tr><td class="patent-data-table-td ">10</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">GenBank Accession No. U86146 (1997).</td></tr><tr><td class="patent-data-table-td ">11</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Hoffman, et al. "<a href='http://scholar.google.com/scholar?q="Ion+Channels-Molecular+Divining+Rods+Hit+Their+Clinical+Mark"'>Ion Channels-Molecular Divining Rods Hit Their Clinical Mark</a>", New England Journal of Medicine, May 29, 1997; 336(22):1599-1600.</td></tr><tr><td class="patent-data-table-td ">12</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Hoffman, et al. "<a href='http://scholar.google.com/scholar?q="Ion+Channels%E2%80%94Molecular+Divining+Rods+Hit+Their+Clinical+Mark"'>Ion Channels—Molecular Divining Rods Hit Their Clinical Mark</a>", New England Journal of Medicine, May 29, 1997; 336(22):1599-1600.</td></tr><tr><td class="patent-data-table-td ">13</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Itoh, T. et al. "<a href='http://scholar.google.com/scholar?q="Genomic+organization+and+mutational+analysis+of+KVLQT1%2C+a+gene+responsible+for+familial+long+QT+syndrome"'>Genomic organization and mutational analysis of KVLQT1, a gene responsible for familial long QT syndrome</a>", Hum. Genet., 1998; 103:290-294.</td></tr><tr><td class="patent-data-table-td ">14</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Keating, et al. "<a href='http://scholar.google.com/scholar?q="Consistent+Linkage+of+the+Long-QT+Syndrome+to+the+Harvey+Ras-I+Locus+on+Chromosome+II"'>Consistent Linkage of the Long-QT Syndrome to the Harvey Ras-I Locus on Chromosome II</a>", Am. J. Hum. Genet., 1991; 49:1335-1339.</td></tr><tr><td class="patent-data-table-td ">15</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Keating, et al. "<a href='http://scholar.google.com/scholar?q="Linkage+of+a+Cardiac+Arrhythmia%2C+the+Long+QT+Syndrome%2C+and+the+Harvey+ras-1+Gene"'>Linkage of a Cardiac Arrhythmia, the Long QT Syndrome, and the Harvey ras-1 Gene</a>", Science, May 3, 1991; 252:704-706.</td></tr><tr><td class="patent-data-table-td ">16</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Keating, M. "<a href='http://scholar.google.com/scholar?q="Linkage+Analysis+and+Long+QT+Syndrome+Using+Genetics+to+Study+Cardiovascular+Disease"'>Linkage Analysis and Long QT Syndrome Using Genetics to Study Cardiovascular Disease</a>", Circulation, 1992; 85:1973-1986.</td></tr><tr><td class="patent-data-table-td ">17</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Santana et al. Bacil;lus subtilis FOF1 ATPase: DNA Sequence of the atp Operon and Characterization of atp Mutants. Nov. 1994. J. of Bacteriol. 176(22):6802-6811.*</td></tr><tr><td class="patent-data-table-td ">18</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">van den Berg et al. "<a href='http://scholar.google.com/scholar?q="The+long+QT+Syndrome%3A+a+novel+missense+mutation+in+the+S6+region+of+the+KVLQT1+gene"'>The long QT Syndrome: a novel missense mutation in the S6 region of the KVLQT1 gene</a>", Hum. Genet., 1997; 100:356-361.</td></tr><tr><td class="patent-data-table-td ">19</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Vincent "<a href='http://scholar.google.com/scholar?q="Genetics+and+Molecular+Biology+of+the+Inherited+Long+QT+Syndrome"'>Genetics and Molecular Biology of the Inherited Long QT Syndrome</a>", Annals of Medicine, 1994; 26:419-425.</td></tr><tr><td class="patent-data-table-td ">20</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Vincent "<a href='http://scholar.google.com/scholar?q="The+Molecular+Genetics+of+the+Long+QT+Syndrome%3A+Genes+causing+fainting+and+sudden+death"'>The Molecular Genetics of the Long QT Syndrome: Genes causing fainting and sudden death</a>", Annu. Rev. Med., 1998; 49:263-274.</td></tr><tr><td class="patent-data-table-td ">21</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Wang, Q. et al. "<a href='http://scholar.google.com/scholar?q="Molecular+genetics+of+long+QT+syndrome+from+genes+to+patients"'>Molecular genetics of long QT syndrome from genes to patients</a>", Current Opinion in Cardiology, 1997; 12:310-320.</td></tr><tr><td class="patent-data-table-td ">22</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Wang, Q. et al. "<a href='http://scholar.google.com/scholar?q="Positional+cloning+of+a+novel+potassium+channel+gene%3A+KVLQT1+mutations+cause+cardiac+arrhythmias"'>Positional cloning of a novel potassium channel gene: KVLQT1 mutations cause cardiac arrhythmias</a>", Nature Genetics, Jan. 1996; 12:17-23.</td></tr><tr><td class="patent-data-table-td ">23</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Wollnik, et al. "<a href='http://scholar.google.com/scholar?q="Pathophysiological+mechanisms+of+dominant+and+recessive+KVLQT1+K%2B+channel+mutations+found+in+inherited+cardiac+arrhythmias"'>Pathophysiological mechanisms of dominant and recessive KVLQT1 K+ channel mutations found in inherited cardiac arrhythmias</a>", Human Molecular Genetics, 1997; 6(11):1943-1949.</td></tr><tr><td class="patent-data-table-td ">24</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Yang, et al. "<a href='http://scholar.google.com/scholar?q="KVLQT1%2C+a+voltage-gated+potassium+channel+responsible+for+human+cardiac+arrhythmias"'>KVLQT1, a voltage-gated potassium channel responsible for human cardiac arrhythmias</a>", Proc. Natl. Acad. Sci. USA, Apr. 1997; 94:4017-4021.</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="forward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Referenced by</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Citing Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7537928">US7537928</a></td><td class="patent-data-table-td patent-date-value">Aug 23, 2004</td><td class="patent-data-table-td patent-date-value">May 26, 2009</td><td class="patent-data-table-td ">Masonic Medical Research Laboratory</td><td class="patent-data-table-td ">Expression vector comprising nucleotide sequences coding potassium chanel protein for use as tools in identifying modulator for prevention and treatment of cardiovascular/myocardial disorders</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7745213">US7745213</a></td><td class="patent-data-table-td patent-date-value">Mar 30, 2009</td><td class="patent-data-table-td patent-date-value">Jun 29, 2010</td><td class="patent-data-table-td ">Masonic Medical Research Laboratory</td><td class="patent-data-table-td ">nucleotide sequences coding potassium chanel protein for use in identifying modulator for prevention and treatment of cardiovascular/myocardial disorders; utilized to diagnose and screen for short QT syndrome, long QT syndrome, Brugada syndrome and progressive conduction disease</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7833718">US7833718</a></td><td class="patent-data-table-td patent-date-value">Nov 6, 2007</td><td class="patent-data-table-td patent-date-value">Nov 16, 2010</td><td class="patent-data-table-td ">Masonic Medical Research Laboratory</td><td class="patent-data-table-td ">CACNA1C nucleic acid mutations as indicators of shorter than normal QT interval and ST segment elevation associated with sudden cardiac death</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2004057030A2?cl=en">WO2004057030A2</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Dec 10, 2003</td><td class="patent-data-table-td patent-date-value">Jul 8, 2004</td><td class="patent-data-table-td ">Amy Regina Mank-Seymour</td><td class="patent-data-table-td ">Methods and compositions relating to drug-induced arrhythmia</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="classifications"></a><div class="patent-section-header"><span class="patent-section-title">Classifications</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th"> </th><th class="patent-data-table-th"> </th></tr></thead><tr><td class="patent-data-table-td ">U.S. Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=dGBcBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S006170">435/6.17</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=dGBcBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S024330">536/24.33</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=dGBcBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S023500">536/23.5</a></span></td></tr><tr><td class="patent-data-table-td ">International Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=dGBcBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0015120000">C12N15/12</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=dGBcBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C07K0014460000">C07K14/46</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=dGBcBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=A01K0067027000">A01K67/027</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=dGBcBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0015090000">C12N15/09</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=dGBcBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12Q0001020000">C12Q1/02</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=dGBcBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=G01N0033150000">G01N33/15</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=dGBcBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0005100000">C12N5/10</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=dGBcBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=G01N0033530000">G01N33/53</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=dGBcBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12P0021080000">C12P21/08</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=dGBcBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=G01N0033680000">G01N33/68</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=dGBcBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=A61P0009060000">A61P9/06</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=dGBcBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C07K0016180000">C07K16/18</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=dGBcBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=G01N0033500000">G01N33/50</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=dGBcBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0001190000">C12N1/19</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=dGBcBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=G01N0033580000">G01N33/58</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=dGBcBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0001150000">C12N1/15</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=dGBcBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C07K0014470000">C07K14/47</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=dGBcBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0001210000">C12N1/21</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=dGBcBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=G01N0033566000">G01N33/566</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=dGBcBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=G01N0033577000">G01N33/577</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=dGBcBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C07K0014705000">C07K14/705</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=dGBcBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12Q0001680000">C12Q1/68</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=dGBcBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=A61K0048000000">A61K48/00</a></span></td></tr><tr><td class="patent-data-table-td ">Cooperative Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=dGBcBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=A61K48/00">A61K48/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=dGBcBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=A01K2217/05">A01K2217/05</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=dGBcBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C07K14/705">C07K14/705</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=dGBcBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C07K14/47">C07K14/47</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=dGBcBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12Q1/6883">C12Q1/6883</a></span></td></tr><tr><td class="patent-data-table-td ">European Classification</td><td class="patent-data-table-td "><span class="nested-value">C07K14/47</span>, <span class="nested-value">C07K14/705</span>, <span class="nested-value">C12Q1/68M6</span></td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="legal-events"></a><div class="patent-section-header"><span class="patent-section-title">Legal Events</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Date</th><th class="patent-data-table-th">Code</th><th class="patent-data-table-th">Event</th><th class="patent-data-table-th">Description</th></tr></thead><tr><td class="patent-data-table-td patent-date-value">Feb 25, 2014</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">12</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Jan 19, 2011</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">ESOTERIX GENETIC LABORATORIES, LLC, NORTH CAROLINA</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20101130</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNOR:GENZYME CORPORATION;REEL/FRAME:025656/0581</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Mar 4, 2010</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">8</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Nov 17, 2009</td><td class="patent-data-table-td ">B1</td><td class="patent-data-table-td ">Reexamination certificate first reexamination</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">CLAIMS 7-8 ARE DETERMINED TO BE PATENTABLE AS AMENDED. NEW CLAIMS 13-15 ARE ADDED AND DETERMINED TOBE PATENTABLE. CLAIMS 1-6 AND 9-12 WERE NOT REEXAMINED.</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Apr 7, 2009</td><td class="patent-data-table-td ">RR</td><td class="patent-data-table-td ">Request for reexamination filed</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20090115</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Feb 27, 2006</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">4</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Mar 23, 2005</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">HOWARD HUGHES MEDICAL INSTITUTE, MARYLAND</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNOR:WANG, QING, PH.D.;REEL/FRAME:015953/0355</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20040707</span></div><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">UNIVERSITY OF UTAH RESEARCH FOUNDATION, UTAH</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNOR:UNIVERSITY OF UTAH;REEL/FRAME:015953/0362</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20041006</span></div><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">UTAH, UNIVERISTY OF, UTAH</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNOR:HOWARD HUGHES MEDICAL INSTITUTE;REEL/FRAME:015953/0359</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20010415</span></div><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">HOWARD HUGHES MEDICAL INSTITUTE 4000 JONES BRIDGE</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNOR:WANG, QING, PH.D. /AR;REEL/FRAME:015953/0355</span></div><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">UNIVERSITY OF UTAH RESEARCH FOUNDATION 615 ARAPEEN</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNOR:UNIVERSITY OF UTAH /AR;REEL/FRAME:015953/0362</span></div><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">UTAH, UNIVERISTY OF 615 ARAPEEN DRIVE SUITE 310SAL</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNOR:HOWARD HUGHES MEDICAL INSTITUTE /AR;REEL/FRAME:015953/0359</span></div></td></tr></table><div class="patent-section-footer"></div></div><div class="modal-dialog" id="patent-images-lightbox"><div class="patent-lightbox-controls"><div class="patent-lightbox-rotate-controls"><div class="patent-lightbox-rotation-text">Rotate</div><div class="rotate-icon rotate-ccw-icon"></div><div class="rotate-icon rotate-cw-icon"></div></div><div class="patent-lightbox-index-counter"></div><a class="patent-lightbox-fullsize-link" target="_blank">Original Image</a><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_right.png" alt="Next page"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_left.png" alt="Previous page"width="21" height="21" /></div></div><div class="modal-dialog-content"><div class="patent-lightbox-image-holder"><div class="patent-lightbox-placeholder"></div></div></div></div><script>_OC_initPatentsAtb({image_not_available_html: " Image not available"});</script></div></div></div></td></tr></table><script>(function() {var href = window.location.href;if (href.indexOf('?') !== -1) {var parameters = href.split('?')[1].split('&');for (var i = 0; i < parameters.length; i++) {var param = parameters[i].split('=');if (param[0] == 'focus') {var elem = document.getElementById(param[1]);if (elem) {elem.focus();}}}}})();</script><script>_OC_addFlags({LockSrc:"/books/javascript/lock_8a2b04e7bf975d5171d8e4c0b6365c7a.js", Host:"http://www.google.com/", IsBooksRentalEnabled:1, IsWebstoreDisplayCaseEnabled:1, IsObfuscationEnabled:1, IsBrowsingHistoryEnabled:1, IsWebReaderSvgEnabled:0, IsGeoLayerEnabled:1, IsImageModeNotesEnabled:1, IsCopyMenuItemEnabled:1, IsGiftingEnabled:0, IsWebReaderUniversalPaginatorEnabled:0, IsOfflineBubbleEnabled:1, IsReaderEnabledForPlayRequests:1, IsFutureOnSaleVolumesEnabled:1, IsOfflineRestrictedCopyEnabled:1, IsBooksUnifiedLeftNavEnabled:1, IsRestrictedCopyEnabled:1, IsZipitFolderCollectionEnabled:1, IsEndOfSampleRecommendationsEnabled:1, IsRatingsOnBookcardsEnabled:1, IsAdsDisabled:0, IsIframePageDisplayEnabled:0, IsEmbeddedMediaEnabled:1, IsImageModeAnnotationsEnabled:1, IsMyLibraryGooglePlusEnabled:1, IsImagePageProviderEnabled:0, IsBookcardListPriceSmall:0, IsInternalUser:0, IsBooksShareButtonEnabled:0, IsPreOrdersEnabled:0, IsDisabledRandomBookshelves:0, WebstoreDisplayCasePosition:3});_OC_Run({"enable_p13n":false,"add_vol_to_collection_base_url":"http://www.google.com/patents?op=add\u0026sig=ACfU3U2XgEEOl2OBNSu0yrJRsEdTg7SYbQ\u0026id=dGBcBAABERAJ","remove_vol_from_collection_base_url":"http://www.google.com/patents?op=remove\u0026sig=ACfU3U3XPNPFFuI-wZ2Fe1wFeWX51MN0xg\u0026id=dGBcBAABERAJ","logged_in":false,"p13n_save_user_settings_url":"http://www.google.com/patents?op=edit_user_settings\u0026sig=ACfU3U0WfsUW7hcJro-DzS-xNa2nkecw8A","is_cobrand":false,"sign_in_url":"https://www.google.com/accounts/Login?service=\u0026continue=http://www.google.com/patents%3Fhl%3Den\u0026hl=en","is_play_enabled":true}, {"volume_id":"","is_ebook":true,"volumeresult":{"has_flowing_text":false,"has_scanned_text":true,"can_download_pdf":false,"can_download_epub":false,"is_pdf_drm_enabled":false,"is_epub_drm_enabled":false,"download_pdf_url":"http://www.google.com/patents/download/KVLQT1_a_long_QT_syndrome_gene.pdf?id=dGBcBAABERAJ\u0026output=pdf\u0026sig=ACfU3U03oxbHzTALeZCT0eksUoimiPUx_A"},"sample_url":"http://www.google.com/patents/reader?id=dGBcBAABERAJ\u0026printsec=frontcover\u0026output=reader\u0026source=gbs_atb_hover","is_browsable":true,"is_public_domain":true}, {});</script><div id="footer_table" style="font-size:83%;text-align:center;position:relative;top:20px;height:4.5em;margin-top:2em"><div style="margin-bottom:8px"><a href=http://www.google.com/><nobr>Google&nbsp;Home</nobr></a> - <a href=//www.google.com/patents/sitemap/><nobr>Sitemap</nobr></a> - <a href=http://www.google.com/googlebooks/uspto.html><nobr>USPTO Bulk Downloads</nobr></a> - <a href=/intl/en/privacy/><nobr>Privacy Policy</nobr></a> - <a href=/intl/en/policies/terms/><nobr>Terms of Service</nobr></a> - <a href=https://support.google.com/faqs/answer/2539193?hl=en><nobr>About Google Patents</nobr></a> - <a href="http://www.google.com/tools/feedback/intl/en/error.html" onclick="try{_OC_startFeedback({productId: '72792',locale: 'en'});return false;}catch(e){}"><nobr>Send Feedback</nobr></a></div><span>Data provided by IFI CLAIMS Patent Services</span><br><span >&copy;2012 Google</span></div> <script type="text/javascript">var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));</script><script type="text/javascript">var pageTracker = _gat._getTracker("UA-27188110-1");pageTracker._setCookiePath("/patents/");pageTracker._trackPageview();</script> </body></html>