<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN""http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html><head><title>Patent US6455275 - DNA construct for producing proteinaceous materials in eucaryotic cells - Google Patents</title><script>(function(){(function(){function e(a){this.t={};this.tick=function(a,c,b){var d=void 0!=b?b:(new Date).getTime();this.t[a]=[d,c];if(void 0==b)try{window.console.timeStamp("CSI/"+a)}catch(e){}};this.tick("start",null,a)}var a;window.performance&&(a=window.performance.timing);var f=a?new e(a.responseStart):new e;window.jstiming={Timer:e,load:f};if(a){var c=a.navigationStart,d=a.responseStart;0<c&&d>=c&&(window.jstiming.srt=d-c)}if(a){var b=window.jstiming.load;0<c&&d>=c&&(b.tick("_wtsrt",void 0,c),b.tick("wtsrt_",
"_wtsrt",d),b.tick("tbsd_","wtsrt_"))}try{a=null,window.chrome&&window.chrome.csi&&(a=Math.floor(window.chrome.csi().pageT),b&&0<c&&(b.tick("_tbnd",void 0,window.chrome.csi().startE),b.tick("tbnd_","_tbnd",c))),null==a&&window.gtbExternal&&(a=window.gtbExternal.pageT()),null==a&&window.external&&(a=window.external.pageT,b&&0<c&&(b.tick("_tbnd",void 0,window.external.startE),b.tick("tbnd_","_tbnd",c))),a&&(window.jstiming.pt=a)}catch(g){}})();})();
</script><link rel="stylesheet" href="/patents/css/_8a2b04e7bf975d5171d8e4c0b6365c7a/kl_intl_patents_bundle.css" type="text/css" /><script src="/books/javascript/atb_8a2b04e7bf975d5171d8e4c0b6365c7a__en.js"></script><script>function googleTranslateElementInit() {new google.translate.TranslateElement({pageLanguage: "en",gaTrack: true,gaId: "UA-27188110-1",multilanguagePage: true});}</script><script src="//translate.google.com/translate_a/element.js?cb=googleTranslateElementInit"></script><meta name="DC.type" content="Patent"><meta name="DC.title" content="DNA construct for producing proteinaceous materials in eucaryotic cells"><meta name="DC.contributor" content="Richard Axel" scheme="inventor"><meta name="DC.contributor" content="Michael H. Wigler" scheme="inventor"><meta name="DC.contributor" content="Saul J. Silverstein" scheme="inventor"><meta name="DC.contributor" content="The Trustees Of Columbia University In The City Of New York" scheme="assignee"><meta name="DC.date" content="1995-6-7" scheme="dateSubmitted"><meta name="DC.description" content="The invention relates to processes for inserting into eucaryotic cells a multiplicity of DNA molecules which includes genes coding for desired proteinaceous materials. The insertion of multiple copies of desired genes is accomplished by cotransformation with the desired genes and with amplifiable genes for a dominant selectable marker in the presence of successively higher amounts of an inhibitor. Alternatively, the insertion of multiple copies of desired genes is accomplished by transformation using DNA molecules formed by ligating a DNA molecule including the desired gene to a DNA molecule which includes an amplifiable gene coding for a dominant selectable phenotype such as a gene associated with resistance to a drug in the presence of successively higher amounts of an agent such as a drug against which the gene confers resistance so that only those eucaryotic cells into which multiple copies of the amplifiable gene have been inserted survive. Eucaryotic cells into which multiple copies of the amplifiable gene have been inserted additionally include multiple copies of the desired gene and may be used to produce multiple copies of proteinaceous molecules. In this way otherwise rare proteinaceous materials may be obtained in higher concentrations than are obtainable using conventional techniques."><meta name="DC.date" content="2002-9-24" scheme="issued"><meta name="DC.relation" content="EP:0022685:A1" scheme="references"><meta name="DC.relation" content="EP:0037723:A2" scheme="references"><meta name="DC.relation" content="EP:0038765" scheme="references"><meta name="DC.relation" content="GB:1521032" scheme="references"><meta name="DC.relation" content="GB:2007675" scheme="references"><meta name="DC.relation" content="GB:2007676" scheme="references"><meta name="DC.relation" content="GB:2008123" scheme="references"><meta name="DC.relation" content="GB:2010847" scheme="references"><meta name="DC.relation" content="US:3800035" scheme="references"><meta name="DC.relation" content="US:4195125" scheme="references"><meta name="DC.relation" content="US:4237224" scheme="references"><meta name="DC.relation" content="US:4264731" scheme="references"><meta name="DC.relation" content="US:4342832" scheme="references"><meta name="DC.relation" content="US:4356270" scheme="references"><meta name="DC.relation" content="US:4363877" scheme="references"><meta name="DC.relation" content="US:4366246" scheme="references"><meta name="DC.relation" content="US:4399216" scheme="references"><meta name="DC.relation" content="US:4411994" scheme="references"><meta name="DC.relation" content="US:4634665" scheme="references"><meta name="DC.relation" content="US:5179017" scheme="references"><meta name="citation_reference" content="Alt, F.W. et al. (1978) &quot;Selective multiplication of dihydrofolate reductase genes in methotrexate-resistant variant of cultured murine cells.&quot; The Journal of Biological Chemistry, 253:1357-1370 (Exhibit 22)."><meta name="citation_reference" content="Anderson, R.P. and Roth, J.R. (1977) &quot;Tandem genetic duplications in phage and bacteria.&quot; Ann. Rev. Microbiol., 31:473-505 (Exhibit 24)."><meta name="citation_reference" content="Anderson, R.P. et al. (1976) &quot;Tandem duplications of the histidine operon observed following generalized transduction in salmonella typhimurium.&quot; J. Mol. Biol., 105:201-218 (Exhibit 23)."><meta name="citation_reference" content="Anderson, W.F. et al. (1981) &quot;Genetic engineering in mammalian cells.&quot; Scientific American, 245:106-121."><meta name="citation_reference" content="Anonymous (1974) &quot;Cellular transformation by D.N.A.&quot; The Lancet 1205-1206 (Exhibit 25)."><meta name="citation_reference" content="Biedler J.L. and Spengler, B.A. (1976b) &quot;Quantitive relationship between a chromosome abnormality (HSR) and antifolate resistance associated with enzyme overproduction.&quot; Abs. Int&#39;l Congress on Cell Biol., 117a (Exhibit 28)."><meta name="citation_reference" content="Biedler, J.L. and Spengler, B.A. (1976a) &quot;Metaphase chromosome anomaly: Association with drug resistance and cell-specific products.&quot; Science, 191:185-187 (Exhibit 27)."><meta name="citation_reference" content="Biedler, J.L. et al. (1972) &quot;Drug response, dihydrofolate reductase, and cytogenetics of amethopterin-resistant Chinese hamster cells in vitro.&quot; Cancer Research, 32:153-161."><meta name="citation_reference" content="Bostock, C.J. and Clark, E.M. (1980) &quot;Satellite DNA in large marker chromosomes of methotrexate-resistant mouse cells.&quot; Cell, 19:709-715 (Exhibit 30)."><meta name="citation_reference" content="Breathnach, R. et al. (1980) &quot;Correct splicing of a chicken ovalbumin gene transcript in mouse L cells.&quot; Proc. Natl. Acad. Sci., 77:740-744 (Exhibit 31)."><meta name="citation_reference" content="Burch, J.W. and McBride, O.W. (1975) &quot;Human gene expression in rodent cells after uptake of isolated metaphase chromosomes.&quot; Proc. Nat. Acad. Sci. 72:1797-1801 (Exhibit 32)."><meta name="citation_reference" content="Camacho, A. and Spear, P.G. (1978) &quot;Transformation of hamster embryo fibroblasts by a specific fragment of the herpes simplex virus genome.&quot; Cell, 15:993-1002 (Exhibit 33)."><meta name="citation_reference" content="Chang, A., et al. (1978) &quot;Phenotypic expression in E. coli of a DNA sequence coding for mouse dihydrofolate reductase.&quot; Nature, 275:617-624 (Exhibit 34)."><meta name="citation_reference" content="Chang, S.E. and Littlefield, J.W. (1976) &quot;Elevated dihydrofolate reductase messenger RNA levels in methotrexate-resistant BHK cells.&quot; Cell, 7:391-396 (Exhibit 35)."><meta name="citation_reference" content="Chasin et al., &quot;Isolation of Chinese hamster cell mutants deficient in dihydrofolate reductase activity&quot; Proc. Natl. Acad. Sci., Jul. 1980 p. 4216-4220 (Exhibit 1)."><meta name="citation_reference" content="Coon, H.G. and Ho, C. (1977) &quot;Transformation of cultured cells to chloramphenicol resistance by purified mammalian mitochondrial DNA.&quot; Brookhaven Symp. Biol., 29:166-177 (Exhibit 36)."><meta name="citation_reference" content="Davis, B.D. (1980) &quot;Gene transfer in bacteria.&quot; Microbiology: Including Immunology and Molecular Genetics, Third Edition 138-151, Harper &amp; Row, Publishers, Inc. (Exhibit 37)."><meta name="citation_reference" content="Degnen, G.E. et al. (1977) &quot;Overexpression of an unstably inherited gene in cultured mouse cells.&quot; Proc. Natl. Acad. Sci., 74:3956-3959 (Exhibit 38)."><meta name="citation_reference" content="Dolnick, B.J. et al. (1979) &quot;Correlation of dihydrofolate reductase elevation with gene amplification in a homogeneously staining chromosomal region in L5178Y cells.&quot; J. Cell Biology, 83:394-402 (Exhibit 39)."><meta name="citation_reference" content="Ehrlich, M. et al. (1976) &quot;Interaction of microbial DNA with cultured mammalian cells. Binding of the donor DNA to the cell surface.&quot; Biochima et Biophysica Acta, 454:397-409 (Exhibit 40).*"><meta name="citation_reference" content="Erickson, R.J. and Braun, W. (1968) &quot;Apparent dependence of transformation on the stage of deoxyribonucleic acid replication of recipient cells.&quot; Bacteriological Reviews, 32:291-296 (Exhibit 41).*"><meta name="citation_reference" content="Fischer, G.A. (1962) &quot;Defective transport of amethopterin (methotrexate) as a mechanism of resistance to the antimetabolite in L5178Y leukemic cells.&quot; Biochemical Pharmacology, 11:1233-1234 (Exhibit 42).*"><meta name="citation_reference" content="Fling, M. et al. (1979) &quot;Cloning and amplification of DNA sequences encoding trimethoprim-resistant dihydrofolate reductase gene.&quot; Chemical Abstracts, published by The American Chemical Society, 90:51265f (Exhibit 43).*"><meta name="citation_reference" content="Flintoff, W.F. et al., (1976) &quot;Isolation and partial characterization of three methotrexate-resistant phenotypes from chinese hamster ovary cells.&quot; Somatic Cell Genetics, 2:245-261 (Exhibit 44)."><meta name="citation_reference" content="Fournier, R.E.K. and Ruddle, F.H. (1977a) &quot;Microcell-mediated transfer of murine chromosomes into mouse, Chinese hamster, and human somatic cells.&quot; Proc. Natl. Acad. Sci., 74:319-323 (Exhibit 45)."><meta name="citation_reference" content="Fournier, R.E.K. and Ruddle, F.H. (1977b) &quot;Stable association of the human transgenome and host murine chromosomes demonstrated with trispecific microcell hybrids.&quot; Proc. Natl. Acad. Sci., 74:3937-3941 (Exhibit 46)."><meta name="citation_reference" content="Fox, J. (1980) &quot;Questions muddy gene transplant successes.&quot; C &amp; E News, pp. 35, 36 and 42."><meta name="citation_reference" content="Graham, F.L. and Van Der Eb, A.J. (1973) &quot;A new technique for the assay of infectivity of human adenovirus 5 DNA.&quot; Virology 52:456-467 (Exhibit 47)."><meta name="citation_reference" content="Graham, F.L. et al. (1975) &quot;Studies on in vitro transformation by DNA and DNA fragments of human adenoviruses and simian virus 40.&quot; Cold Spring Harbor Symp. Quant. Biol., 39:637-649 (Exhibit 48)."><meta name="citation_reference" content="Kaufman, R.J. et al. (1979) &quot;Amplified dihydrofolate reductase genes in unstably methotrexate-resistant cells are associated with double minute chromosomes.&quot; Proc. Natl. Acad. Sci., 76:5669-5673 (Exhibit 49)."><meta name="citation_reference" content="Lehninger, A.L. (1975) &quot;The molecular basis of cell structure and function.&quot; Biochemistry, Second Edition 118-119 (Exhibit 51)."><meta name="citation_reference" content="Lewis, W.H. (1980) &quot;Parameters governing the transfer of the genes for thymidine kinase and dihydrofolate reductase into mouse cells using metaphase chromosomes or DNA.&quot; Somatic Cell and Molecular Genetics, 6:333-348 (Exhibit 52)."><meta name="citation_reference" content="Maitland, N.J. and McDougall, J.K. (1977) &quot;Biochemical transformation of mouse cells by fragments of herpes simplex virus DNA.&quot; Cell, 11:233-241 (Exhibit 53)."><meta name="citation_reference" content="Majumdar, A. and Bose, S.K. (1968) &quot;DNA mediated genetic transformation of a human cancerous cell line cultured in vitro.&quot; Br. J. Cancer, 22:603-613 (Exhibit 54)."><meta name="citation_reference" content="McBride, O.W. and Ozer, H.L. (1973) &quot;Transfer of genetic information by purified metaphase chromosomes.&quot; Proc. Nat. Acad. Sci., 70:1258-1262 (Exhibit 56)."><meta name="citation_reference" content="McBride, O.W. and Peterson, J.L. (1980) &quot;Chromosome-mediated gene transfer in mammalian cells.&quot; Ann. Rev. Genet., 14:321-345 (Exhibit 57)."><meta name="citation_reference" content="Merck &amp; Co., Inc. (1968) The Merck Index, p. 568 (Exhibit 58)."><meta name="citation_reference" content="Minson, A.C. et al. (1978) &quot;Introduction of the herpes simplex virus thymidine kinase gene into mouse cells using virus DNA or transformed cell DNA.&quot; Cell, 13:581-587 (Exhibit 59)."><meta name="citation_reference" content="Mulligan, R.C. et al. (1979) &quot;Synthesis of rabbit beta-globin in cultured monkey kidney cells following infection with a SV40 beta-globin recombinant genome.&quot; Nature, 277:108-114 (Exhibit 60)."><meta name="citation_reference" content="Mulligan, R.C. et al. (1979) “Synthesis of rabbit β-globin in cultured monkey kidney cells following infection with a SV40 β-globin recombinant genome.” Nature, 277:108-114 (Exhibit 60)."><meta name="citation_reference" content="Nunberg, J.H. et al. (1978) &quot;Amplified dihydrofolate reductase genes are localized to a homogeneously staining region of a single chromosome in a methotrexate-resistant chinese hamster ovary cell line.&quot; PNAS, 75:5553-5556."><meta name="citation_reference" content="Ottolenghi-Nightingale, E. (1974) &quot;DNA-mediated transformation in mammalian cells.&quot; Cell Commun., 233-254 (Exhibit 63)."><meta name="citation_reference" content="Perlman, D. et al. (1975) &quot;Denaturation mapping of R factor deoxyribonucleic acid.&quot; J. Bacteriology, 123:1035-1042 (Exhibit 64)."><meta name="citation_reference" content="Peterson, J.L. and McBride, O.W. (1980) &quot;Cotransfer of linked eukaryotic genes and efficient transfer of hypoxanthine phosphoribosyltransferase by DNA-mediated gene transfer.&quot; PNAS, 77:1583-1587 (Exhibit 65)."><meta name="citation_reference" content="Roosa, R.A. and Bailey, E. (1970) &quot;DNA-mediated transformation of mammalian cells in culture. Increased transforming efficiency following sonication.&quot; J. Cell. Physiol., 75:137-150 (Exhibit 68)."><meta name="citation_reference" content="Scangos, G.A. et al. (1979) &quot;Molecular analysis of chromosome-mediated gene transfer.&quot; Proc. Natl. Acad. Sci., 76:3987-3990 (Exhibit 69)."><meta name="citation_reference" content="Schimke, R.T. et al. (1978a) &quot;Gene amplification and drug resistance in cultured murine cells.&quot; Science 202:1051-1055 (Exhibit 70)."><meta name="citation_reference" content="Schimke, R.T. et al. (1978b) &quot;Amplification of dihydrofolate reductase genes in methotrexate-resistant cultured mouse cells.&quot; Cold Spring Harbor Symp. Quant. Biol., 42:649-657 (Exhibit 71)."><meta name="citation_reference" content="Schimke, R.T. et al. (1979a) &quot;Studies on the amplification of dihydrofolate reductase genes in methotrexate-resistant cultured mouse cells.&quot; Cold Spring Harbor Symp. Quant. Biol., 43:1297-1303 (Exhibit 72)."><meta name="citation_reference" content="Schimke, R.T. et al. (1979b) &quot;Structure and localization of dihydrofolate reductase genes in methotrexate-resistant cultured cells.&quot; Eucaryotic Gene Regulation, ICN-UCLA Symposia, R. Axel and T. Maniatis, Editors, Academic Press, 499-510 (Exhibit 73)."><meta name="citation_reference" content="Sirotnak, F.M. et al. (1968) &quot;On the nature of a transport alteration determining resistance to amethopterin in the L1210 leukemia.&quot; Cancer Research, 28:75-80 (Exhibit 74)."><meta name="citation_reference" content="Spandidos, D.A. and Siminovitch, L. (1977a) &quot;Linkage of markers controlling consecutive biochemical steps in CHO cells as demonstrated by chromosome transfer.&quot; Cell, 12:235-242 (Exhibit 75)."><meta name="citation_reference" content="Spandidos, D.A. and Siminovitch, L. (1977b) &quot;Transfer of codominant markers by isolated metaphase chromosomes in Chinese hamster ovary cells.&quot; Proc. Natl. Acad. Sci., 74:3480-3484 (Exhibit 76)."><meta name="citation_reference" content="Spradling, A.C. and Mahowald, A.P. (1980) &quot;Amplification of genes for chorion proteins during oogenesis in Drosophila melanogaster.&quot; Proc. Natl. Acad. Sci., 77:1096-1100."><meta name="citation_reference" content="Stebbing, N. (1979) &quot;Cellular uptake and in vivo fate of polynucleotides.&quot; Cell Biology International Reports, 3:485-502 (Exhibit 78)."><meta name="citation_reference" content="Sugden, B. et al. (1979) &quot;Epstein-Barr virus DNA is amplified in transformed lymphocytes.&quot; Journal of Virology, 31:590-595 (Exhibit 79)."><meta name="citation_reference" content="Sullivan, D. et al. (1973) &quot;Synthesis of a deoxyribonucleic acid sequence complementary to ovalbumin messenger ribonucleic acid and quantification of ovalbumin genes.&quot; J. Biological Chemistry 248: 7530-7539 (Exhibit 80)."><meta name="citation_reference" content="Trust. Of Columbia Univ. v. Roche Diagnostics GMBH, 150 F. Supp.2d 191 (D. Mass 2001) (Exhibit 3)."><meta name="citation_reference" content="Trust. Of Columbia Univ. v. Roche Diagnostics, 126 F. Supp.2d 16 (D. Mass 2000) (Exhibit 4)."><meta name="citation_reference" content="U.S. patent application Ser. No. 08/477159, Axel et al., filed Jun. 5, 1995."><meta name="citation_reference" content="Van den Berg, J. et al. (1978) &quot;Comparison of cloned rabbit and mouse beta-globin genes showing strong evolutionary divergence of two homologous pairs of introns.&quot; Nature 276:37-44 (Exhibit 81)."><meta name="citation_reference" content="Van den Berg, J. et al. (1978) “Comparison of cloned rabbit and mouse β-globin genes showing strong evolutionary divergence of two homologous pairs of introns.” Nature 276:37-44 (Exhibit 81)."><meta name="citation_reference" content="Werkheiser, W.C. (1961) &quot;Specific binding of 4-amino folic acid analogues by folic acid reductase.&quot; The Journal of Biological Chemistry, 236:888-893 (Exhibit 83)."><meta name="citation_reference" content="Wiessmann, C. et al. (1979) &quot;Expression of cloned viral and chromosomal plasmid-linked DNA in cognate host cells.&quot; From Gene to Protein Information Transfer in Normal and Abnormal Cells: Miami Winter Symposium, 16:99-132 (Russell, T.R. et al. eds., Academic Press) (Exhibit 82)."><meta name="citation_reference" content="Wigler, M. et al. (1977) &quot;Transfer of purified herpes virus thymidine kinase gene to cultured mouse cells.&quot; Cell, 11:223-232."><meta name="citation_reference" content="Wigler, M. et al. (1978) &quot;Biochemical transfer of single-copy eucaryotic genes using total cellular DNA as donor.&quot; Cell, 14:725-731."><meta name="citation_reference" content="Wilkie, N.M. et al. (1979) &quot;Hybrid plasmids containing an active thymidine kinase gene of herpes simplex virus 1.&quot; Nucleic Acids Research 7:859-877 (Exhibit 87)."><meta name="citation_reference" content="Willecke, K. (1978) &quot;Results and prospects of chromosomal gene transfer between cultured mammalian cells.&quot; Theor. Appl. Genet. 52:97-104 (Exhibit 89)."><meta name="citation_reference" content="Willecke, K. and Ruddle, F.H. (1975) &quot;Transfer of the human gene for hypoxanthine-guanine phosphoribosyltransferase via isolated human metaphase chromosomes into mouse L-Cells.&quot; Proc. Nat. Acad. Sci., 72:1792-1796 (Exhibit 88)."><meta name="citation_reference" content="Zakai, N. et al. (1977) &quot;Membrane ultrastructural changes during calcium phosphate-induced fusion of human erythrocyte ghosts.&quot; Proc. Natl. Acad. Sci., 74:2417-2421 (Exhibit 91)."><meta name="citation_patent_number" content="US:6455275"><meta name="citation_patent_application_number" content="US:08/484,136"><link rel="canonical" href="http://www.google.com/patents/US6455275"/><meta property="og:url" content="http://www.google.com/patents/US6455275"/><meta name="title" content="Patent US6455275 - DNA construct for producing proteinaceous materials in eucaryotic cells"/><meta name="description" content="The invention relates to processes for inserting into eucaryotic cells a multiplicity of DNA molecules which includes genes coding for desired proteinaceous materials. The insertion of multiple copies of desired genes is accomplished by cotransformation with the desired genes and with amplifiable genes for a dominant selectable marker in the presence of successively higher amounts of an inhibitor. Alternatively, the insertion of multiple copies of desired genes is accomplished by transformation using DNA molecules formed by ligating a DNA molecule including the desired gene to a DNA molecule which includes an amplifiable gene coding for a dominant selectable phenotype such as a gene associated with resistance to a drug in the presence of successively higher amounts of an agent such as a drug against which the gene confers resistance so that only those eucaryotic cells into which multiple copies of the amplifiable gene have been inserted survive. Eucaryotic cells into which multiple copies of the amplifiable gene have been inserted additionally include multiple copies of the desired gene and may be used to produce multiple copies of proteinaceous molecules. In this way otherwise rare proteinaceous materials may be obtained in higher concentrations than are obtainable using conventional techniques."/><meta property="og:title" content="Patent US6455275 - DNA construct for producing proteinaceous materials in eucaryotic cells"/><meta property="og:type" content="book"/><meta property="og:site_name" content="Google Books"/><meta property="og:image" content="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><link rel="image_src" href="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><script>(function(){try{var aa=function(a,b,c,d){d=d||{};d._sn=["cfg",b,c].join(".");window.gbar.logger.ml(a,d)};var g=window.gbar=window.gbar||{},l=window.gbar.i=window.gbar.i||{},m={},n;function _tvn(a,b){var c=parseInt(a,10);return isNaN(c)?b:c}function _tvf(a,b){var c=parseFloat(a);return isNaN(c)?b:c}function _tvv(a){return!!a}function p(a,b,c){(c||g)[a]=b}g.bv={n:_tvn("2",0),r:"",f:".67.",e:"0",m:_tvn("0",1)};
function q(a,b,c){var d="on"+b;if(a.addEventListener)a.addEventListener(b,c,!1);else if(a.attachEvent)a.attachEvent(d,c);else{var f=a[d];a[d]=function(){var a=f.apply(this,arguments),b=c.apply(this,arguments);return void 0==a?b:void 0==b?a:b&&a}}}var s=function(a){return function(){return g.bv.m==a}},ba=s(1),ca=s(2);p("sb",ba);p("kn",ca);l.a=_tvv;l.b=_tvf;l.c=_tvn;l.i=aa;var da=window.gbar.i.i;var t,u,v,w;function ea(a){v=a}function fa(a){var b;if(b=v&&window.encodeURIComponent)b=a.href,b=!b.match(/^http[s]?:\/\/accounts\.google\.[^/]*\/ClearSID/i)&&!b.match(/^http[s]?:\/\/[^/]*\/accounts\/ClearSID/i);if(b=b&&encodeURIComponent(v()))a.href=a.href.replace(/([?&]continue=)[^&]*/,"$1"+b)}function ga(a){window.gApplication&&(a.href=window.gApplication.getTabUrl(a.href))}
function ha(a){var b=document.forms[0].q,c=window.encodeURIComponent&&b&&b.value,b=b&&b.placeholder;c&&c!=b&&(a.href=a.href.replace(/([?&])q=[^&]*|$/,function(a,b){return(b||"&")+"q="+encodeURIComponent(c)}))}n=l.a("")?ga:ha;
function x(a,b,c,d,f,e){var h=document.getElementById(a);if(h){var k=h.style;k.left=d?"auto":b+"px";k.right=d?b+"px":"auto";k.top=c+"px";k.visibility=u?"hidden":"visible";f&&e?(k.width=f+"px",k.height=e+"px"):(x(t,b,c,d,h.offsetWidth,h.offsetHeight),u=u?"":a)}}
var y=[],ia=function(a,b){y.push(b)},ja=function(a){a=a||window.event;var b=a.target||a.srcElement;a.cancelBubble=!0;null==t&&(a=document.createElement(Array.every||window.createPopup?"iframe":"div"),a.frameBorder="0",t=a.id="gbs",a.src="javascript:''",b.parentNode.appendChild(a),q(document,"click",z));var c=b,b=0;"gb3"!=c.className&&(c=c.parentNode);a=c.getAttribute("aria-owns")||"gbi";var d=c.offsetWidth,f=20<c.offsetTop?46:24;document.getElementById("tphdr")&&(f-=3);var e=!1;do b+=c.offsetLeft||
0;while(c=c.offsetParent);var c=(document.documentElement.clientWidth||document.body.clientWidth)-b-d,h,d=document.body,k=document.defaultView;k&&k.getComputedStyle?(d=k.getComputedStyle(d,""))&&(h=d.direction):h=d.currentStyle?d.currentStyle.direction:d.style.direction;h="rtl"==h;if("gbi"==a){for(d=0;k=y[d++];)k();A(null,window.navExtra);h&&(b=c,e=!0)}else h||(b=c,e=!0);u!=a&&z();x(a,b,f,e)},z=function(){u&&x(u,0,0)},A=function(a,b){var c,d=document.getElementById("gbi"),f=a;f||(f=d.firstChild);
for(;b&&(c=b.pop());){var e=d,h=c,k=f;w||(w="gb2");e.insertBefore(h,k).className=w}},ka=function(a,b,c){if((b=document.getElementById(b))&&a){a.className="gb4";var d=document.createElement("span");d.appendChild(a);d.appendChild(document.createTextNode(" | "));d.id=c;b.appendChild(d)}},la=function(){return document.getElementById("gb_70")},ma=function(){return!!u};p("qs",n);p("setContinueCb",ea);p("pc",fa);p("tg",ja);p("close",z);p("addLink",ka);p("almm",A);p("si",la);p("adh",ia);p("op",ma);var B=function(){},C=function(){},F=function(a){var b=new Image,c=D;b.onerror=b.onload=b.onabort=function(){try{delete E[c]}catch(a){}};E[c]=b;b.src=a;D=c+1},E=[],D=0;p("logger",{il:C,ml:B,log:F});var G=window.gbar.logger;var H={},na={},I=[],oa=l.b("0.1",.1),pa=l.a("1",!0),qa=function(a,b){I.push([a,b])},ra=function(a,b){H[a]=b},sa=function(a){return a in H},J={},K=function(a,b){J[a]||(J[a]=[]);J[a].push(b)},ta=function(a){K("m",a)},L=function(a,b){var c=document.createElement("script");c.src=a;c.async=pa;Math.random()<oa&&(c.onerror=function(){c.onerror=null;B(Error("Bundle load failed: name="+(b||"UNK")+" url="+a))});(document.getElementById("xjsc")||document.getElementsByTagName("body")[0]||
document.getElementsByTagName("head")[0]).appendChild(c)},N=function(a){for(var b=0,c;(c=I[b])&&c[0]!=a;++b);!c||c[1].l||c[1].s||(c[1].s=!0,M(2,a),c[1].url&&L(c[1].url,a),c[1].libs&&m.d&&m.d(c[1].libs))},O=function(a){K("gc",a)},P=null,ua=function(a){P=a},M=function(a,b,c){if(P){a={t:a,b:b};if(c)for(var d in c)a[d]=c[d];try{P(a)}catch(f){}}};p("mdc",H);p("mdi",na);p("bnc",I);p("qGC",O);p("qm",ta);p("qd",J);p("lb",N);p("mcf",ra);p("bcf",qa);p("aq",K);p("mdd","");p("has",sa);
p("trh",ua);p("tev",M);var Q=l.b("0.1",.001),R=0;
function _mlToken(a,b){try{if(1>R){R++;var c,d=a,f=b||{},e=encodeURIComponent,h=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&jexpid=",e("17483"),"&srcpg=",e("prop=22"),"&jsr=",Math.round(1/Q),"&ogev=",e("hknsU6DoIZCMNvangrgF"),"&ogf=",g.bv.f,"&ogrp=",e("1"),"&ogv=",e("1407723702.0"),"&oggv="+e("es_plusone_gc_20140723.0_p0"),"&ogd=",e("com"),"&ogc=",e("BRA"),"&ogl=",e("en")];f._sn&&(f._sn="og."+
f._sn);for(var k in f)h.push("&"),h.push(e(k)),h.push("="),h.push(e(f[k]));h.push("&emsg=");h.push(e(d.name+":"+d.message));var r=h.join("");S(r)&&(r=r.substr(0,2E3));c=r;var Aa=window.gbar.logger._aem(a,c);F(Aa)}}catch(Na){}}var S=function(a){return 2E3<=a.length},va=function(a,b){return b};function T(a){B=a;p("_itl",S,G);p("_aem",va,G);p("ml",B,G);a={};H.er=a}l.a("")?T(function(a){throw a;}):l.a("1")&&Math.random()<Q&&T(_mlToken);I.push(["m",{url:"//ssl.gstatic.com/gb/js/scm_7385cc5883250b43a39405734c1bea59.js"}]);g.mcf("c",{});g.sg={c:""};if(l.a("1")){var wa=l.a("");I.push(["gc",{auto:wa,url:"//ssl.gstatic.com/gb/js/abc/gci_91f30755d6a6b787dcc2a4062e6e9824.js",libs:"googleapis.client:plusone:gapi.iframes"}]);var xa={version:"gci_91f30755d6a6b787dcc2a4062e6e9824.js",index:"",lang:"en"};H.gc=xa;var U=function(a){window.googleapis&&window.iframes?a&&a():(a&&O(a),N("gc"))};p("lGC",U);l.a("1")&&p("lPWF",U)};window.__PVT="";if(l.a("1")&&l.a("1")){var V=function(a){U(function(){K("pw",a);N("pw")})};p("lPW",V);I.push(["pw",{url:"//ssl.gstatic.com/gb/js/abc/pwm_45f73e4df07a0e388b0fa1f3d30e7280.js"}]);var W=[],ya=function(a){W[0]=a},za=function(a,b){var c=b||{};c._sn="pw";B(a,c)},Ba={signed:W,elog:za,base:"https://plusone.google.com/u/0",loadTime:(new Date).getTime()};H.pw=Ba;var X=function(a,b){for(var c=b.split("."),d=function(){var b=arguments;a(function(){for(var a=g,d=0,e=c.length-1;d<e;++d)a=a[c[d]];a[c[d]].apply(a,b)})},f=g,e=0,h=c.length-1;e<h;++e)f=
f[c[e]]=f[c[e]]||{};return f[c[e]]=d};X(V,"pw.clk");X(V,"pw.hvr");p("su",ya,g.pw)};function Ca(){function a(){for(var b;(b=e[h++])&&"m"!=b[0]&&!b[1].auto;);b&&(M(2,b[0]),b[1].url&&L(b[1].url,b[0]),b[1].libs&&m.d&&m.d(b[1].libs));h<e.length&&setTimeout(a,0)}function b(){0<f--?setTimeout(b,0):a()}var c=l.a("1"),d=l.a(""),f=3,e=I,h=0,k=window.gbarOnReady;if(k)try{k()}catch(r){da(r,"ml","or")}d?p("ldb",a):c?q(window,"load",b):b()}p("rdl",Ca);var Da={D:1,H:2,da:3,p:4,W:5,M:6,F:7,g:8,ha:9,U:10,L:11,T:12,S:13,N:14,Q:15,P:16,fa:17,w:18,O:19,ga:20,ea:21,u:22,G:23,ja:24,ka:25,ia:26,A:27,j:28,o:29,k:30,ca:31,Z:32,$:33,J:34,K:35,ba:36,aa:37,Y:38,B:39,R:40,v:41,X:42,V:43,h:48,C:49,I:500},Y=[1,2,3,4,5,6,9,10,11,13,14,28,29,30,34,35,37,38,39,40,41,42,43,48,49,500];var Z=l.b("0.001",1E-4),Ea=l.b("1",1),Fa=!1,Ga=!1;if(l.a("1")){var Ha=Math.random();Ha<=Z&&(Fa=!0);Ha<=Ea&&(Ga=!0)}var Ia=Da,$=null;function Ja(){var a=0,b=function(b,d){l.a(d)&&(a|=b)};b(1,"");b(2,"");b(4,"");b(8,"");return a}
function Ka(a,b){var c=Z,d=Fa,f;f=a;if(!$){$={};for(var e=0;e<Y.length;e++){var h=Y[e];$[h]=!0}}if(f=!!$[f])c=Ea,d=Ga;if(d){d=encodeURIComponent;g.rp?(f=g.rp(),f="-1"!=f?f:"1"):f="1";c=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&oge=",a,"&ogex=",d("17483"),"&ogev=",d("hknsU6DoIZCMNvangrgF"),"&ogf=",g.bv.f,"&ogp=",d("22"),"&ogrp=",d(f),"&ogsr=",Math.round(1/c),"&ogv=",d("1407723702.0"),"&oggv="+
d("es_plusone_gc_20140723.0_p0"),"&ogd=",d("com"),"&ogl=",d("en"),"&ogc=",d("BRA"),"&ogus=",Ja()];if(b){"ogw"in b&&(c.push("&ogw="+b.ogw),delete b.ogw);var k;f=b;e=[];for(k in f)0!=e.length&&e.push(","),e.push(La(k)),e.push("."),e.push(La(f[k]));k=e.join("");""!=k&&(c.push("&ogad="),c.push(d(k)))}F(c.join(""))}}function La(a){"number"==typeof a&&(a+="");return"string"==typeof a?a.replace(".","%2E").replace(",","%2C"):a}C=Ka;p("il",C,G);var Ma={};H.il=Ma;setTimeout(function(){C(Ia.g)},0);}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var b=window.gbar.i.i;var c=window.gbar;var f=function(d){try{var a=document.getElementById("gbom");a&&d.appendChild(a.cloneNode(!0))}catch(e){b(e,"omas","aomc")}};c.aomc=f;}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var a=window.gbar;a.mcf("pm",{p:""});}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{window.gbar.rdl();}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
if (window['_OC_timingAction']) {window['_OC_timingAction']('patents_refpage');}</script><style>#gbar,#guser{font-size:13px;padding-top:1px !important;}#gbar{float:left;height:22px}#guser{padding-bottom:7px !important;text-align:right}.gbh,.gbd{border-top:1px solid #c9d7f1;font-size:1px}.gbh{height:0;position:absolute;top:24px;width:100%}#gbs,.gbm{background:#fff;left:0;position:absolute;text-align:left;visibility:hidden;z-index:1000}.gbm{border:1px solid;border-color:#c9d7f1 #36c #36c #a2bae7;z-index:1001}.gb1{margin-right:.5em}.gb1,.gb3{zoom:1}.gb2{display:block;padding:.2em .5em}.gb2,.gb3{text-decoration:none !important;border-bottom:none}a.gb1,a.gb4{text-decoration:underline !important}a.gb1,a.gb2,a.gb3,a.gb4{color:#00c !important}.gbi .gb3,.gbi .gb2,.gbi .gb4{color:#dd8e27 !important}.gbf .gb3,.gbf .gb2,.gbf .gb4{color:#900 !important}a.gb2:hover{background:#36c;color:#fff !important}#gbar .gbz0l{color:#000 !important;cursor:default;font-weight:bold;text-decoration:none !important}
#gbar { padding:.3em .6em !important;}</style></head><body  topmargin="3" marginheight="3"><div id=gbar><nobr><a onclick=gbar.qs(this);gbar.logger.il(1,{t:1}); class=gb1 id=gb_1 href="https://www.google.com/search?sa=N&tab=tw">Search</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:2}); class=gb1 id=gb_2 href="http://www.google.com/search?hl=en&tbm=isch&source=og&sa=N&tab=ti">Images</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:8}); class=gb1 id=gb_8 href="http://maps.google.com/maps?hl=en&sa=N&tab=tl">Maps</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:78}); class=gb1 id=gb_78 href="https://play.google.com/?hl=en&sa=N&tab=t8">Play</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:36}); class=gb1 id=gb_36 href="http://www.youtube.com/results?sa=N&tab=t1">YouTube</a> <a onclick=gbar.logger.il(1,{t:5}); class=gb1 id=gb_5 href="http://news.google.com/nwshp?hl=en&tab=tn">News</a> <a onclick=gbar.logger.il(1,{t:23}); class=gb1 id=gb_23 href="https://mail.google.com/mail/?tab=tm">Gmail</a> <a onclick=gbar.logger.il(1,{t:25}); class=gb1 id=gb_25 href="https://drive.google.com/?tab=to">Drive</a> <a class=gb3 href="http://www.google.com/intl/en/options/" onclick="this.blur();gbar.tg(event);return !1" aria-haspopup=true><u>More</u> <small>&#9660;</small></a><div class=gbm id=gbi><a onclick=gbar.logger.il(1,{t:24}); class=gb2 id=gb_24 href="https://www.google.com/calendar?tab=tc">Calendar</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:51}); class=gb2 id=gb_51 href="http://translate.google.com/?hl=en&sa=N&tab=tT">Translate</a><a onclick=gbar.logger.il(1,{t:17}); class=gb2 id=gb_17 href="http://www.google.com/mobile/?hl=en&tab=tD">Mobile</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:10}); class=gb2 id=gb_10 href="http://www.google.com/search?hl=en&tbo=u&tbm=bks&source=og&sa=N&tab=tp">Books</a><a onclick=gbar.logger.il(1,{t:212}); class=gb2 id=gb_212 href="https://wallet.google.com/manage/?tab=ta">Wallet</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:6}); class=gb2 id=gb_6 href="http://www.google.com/search?hl=en&tbo=u&tbm=shop&source=og&sa=N&tab=tf">Shopping</a><a onclick=gbar.logger.il(1,{t:30}); class=gb2 id=gb_30 href="http://www.blogger.com/?tab=tj">Blogger</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:27}); class=gb2 id=gb_27 href="http://www.google.com/finance?sa=N&tab=te">Finance</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:31}); class=gb2 id=gb_31 href="https://plus.google.com/photos?sa=N&tab=tq">Photos</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:12}); class=gb2 id=gb_12 href="http://www.google.com/search?hl=en&tbo=u&tbm=vid&source=og&sa=N&tab=tv">Videos</a><div class=gb2><div class=gbd></div></div><a onclick=gbar.logger.il(1,{t:66}); href="http://www.google.com/intl/en/options/" class=gb2>Even more &raquo;</a></div></nobr></div><div id=guser width=100%><nobr><span id=gbn class=gbi></span><span id=gbf class=gbf></span><span id=gbe></span><a target=_top id=gb_70 href="https://www.google.com/accounts/Login?service=&continue=http://www.google.com/patents%3Fhl%3Den&hl=en" class=gb4>Sign in</a><div style="display: none"><div class=gbm id=gbd5 aria-owner=gbg5><div class=gbmc><ol id=gbom class=gbmcc></ol></div></div></div></nobr></div><div class=gbh style=left:0></div><div class=gbh style=right:0></div><div role="alert" style="position: absolute; left: 0; right: 0;"><a href="http://www.google.com/patents/us6455275?hl=en&amp;output=html_text" title="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."><img border="0" src="http://www.google.com/images/cleardot.gif"alt="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."></a></div><div id="guser"><nobr></nobr></div><div style="clear:both;"></div><div id="gb-top-search-box" class="gb-top-search-box-small gb-reset"><table><tr><td class="logo"><a href="http://www.google.com/patents" class="logo-link"><img class="logo-img" src="/intl/en/images/logos/google_logo_41.png" alt="Go to Google Books Home" height="41"/></a></td><td><form action="http://www.google.com/search" name="f" id="vheadf" method="get"><span id="hf"></span><input type="hidden" name="tbm" value="pts"/><input type="hidden" name="tbo" value="1"/><input type="hidden" name="hl" value="en"/><table><tr><td><div class="inputs"><table><tr><td><div class="text-input"><input type="text" name="q" id="vheadq" class="text" maxlength="2048" size="31" value="" title="Search Patents" accesskey="s" autocomplete="off"/><script>window._OC_autoDir &&window._OC_autoDir('vheadq', 'tia-vheadq');</script></div></td><td><div class="submit-input"><input name="btnG" class="submit" type="submit" value=""/></div></td></tr></table></div></td><td class="col-ext-links"><div class="ext-links"><a href="http://www.google.com/advanced_patent_search">&lt;nobr&gt;Advanced Patent Search&lt;/nobr&gt;</a></div></td></tr></table></form></td></tr></table></div><div class="kd-appbar"><h2 class="kd-appname"><a href="/patents">Patents</a></h2><div class="kd-buttonbar left" id="left-toolbar-buttons"><a id="appbar-write-review-link" href=""></a><a id="appbar-view-print-sample-link" href=""></a><a id="appbar-view-ebook-sample-link" href=""></a><a id="appbar-patents-prior-art-finder-link" href="https://www.google.com/patents/related/US6455275"></a><a id="appbar-patents-discuss-this-link" href="http://www.google.com/url?id=s1BcBAABERAJ&amp;q=http://patents.stackexchange.com/redirect/google-patents%3Fpatent%3DUS6455275&amp;usg=AFQjCNFXW7KnRgko6cmc5GDnvDExZz8g_Q" data-is-grant="true"></a><a id="appbar-read-patent-link" href="//docs.google.com/viewer?url=patentimages.storage.googleapis.com/pdfs/US6455275.pdf"></a><a id="appbar-download-pdf-link" href="//patentimages.storage.googleapis.com/pdfs/US6455275.pdf"></a></div><div class="kd-buttonbar right" id="right-toolbar-buttons"></div></div><div id="books-microdata" itemscope=""itemtype="http://schema.org/Book"itemid="http://www.google.com/patents/US6455275" style="display:none"><span itemprop="description">The invention relates to processes for inserting into eucaryotic cells a multiplicity of DNA molecules which includes genes coding for desired proteinaceous materials. The insertion of multiple copies of desired genes is accomplished by cotransformation with the desired genes and with amplifiable genes...</span><span itemprop="url">http://www.google.com/patents/US6455275?utm_source=gb-gplus-share</span><span class="main-title" itemprop="name">Patent US6455275 - DNA construct for producing proteinaceous materials in eucaryotic cells</span><img itemprop="image" src="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"alt="Patent US6455275 - DNA construct for producing proteinaceous materials in eucaryotic cells" title="Patent US6455275 - DNA construct for producing proteinaceous materials in eucaryotic cells"></div><div style="display: none"><ol id="ofe-gear-menu-contents" class="gbmcc"><li class="gbe gbmtc"><a class="gbmt goog-menuitem-content" id="" href="http://www.google.com/advanced_patent_search">Advanced Patent Search</a></li></ol></div><table id="viewport_table" cellpadding="0" style="clear:both" cellspacing="0"><tr><td id="viewport_td"><div class=vertical_module_list_row><div id=intl_patents class=about_content><div id=intl_patents_v><table class="patent-bibdata"><tr><td class="patent-bibdata-heading">Publication number</td><td class="single-patent-bibdata">US6455275 B1</td></tr><tr><td class="patent-bibdata-heading">Publication type</td><td class="single-patent-bibdata">Grant</td></tr><tr><td class="patent-bibdata-heading">Application number</td><td class="single-patent-bibdata">US 08/484,136</td></tr><tr><td class="patent-bibdata-heading">Publication date</td><td class="single-patent-bibdata">Sep 24, 2002</td></tr><tr><td class="patent-bibdata-heading">Filing date</td><td class="single-patent-bibdata">Jun 7, 1995</td></tr><tr><td class="patent-bibdata-heading">Priority date<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The priority date is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the date listed."></span></td><td class="single-patent-bibdata">Feb 25, 1980</td></tr><tr class="patent-bibdata-list-row alternate-patent-number"><td class="patent-bibdata-heading">Publication number</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value">08484136, </span><span class="patent-bibdata-value">484136, </span><span class="patent-bibdata-value">US 6455275 B1, </span><span class="patent-bibdata-value">US 6455275B1, </span><span class="patent-bibdata-value">US-B1-6455275, </span><span class="patent-bibdata-value">US6455275 B1, </span><span class="patent-bibdata-value">US6455275B1</span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Inventors</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Richard+Axel%22">Richard Axel</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Michael+H.+Wigler%22">Michael H. Wigler</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Saul+J.+Silverstein%22">Saul J. Silverstein</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Original Assignee</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=inassignee:%22The+Trustees+Of+Columbia+University+In+The+City+Of+New+York%22">The Trustees Of Columbia University In The City Of New York</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Export Citation</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/US6455275.bibtex">BiBTeX</a>, </span><span class="patent-bibdata-value"><a href="/patents/US6455275.enw">EndNote</a>, </span><span class="patent-bibdata-value"><a href="/patents/US6455275.ris">RefMan</a></span></span></td></tr><tr class="patent-internal-links"><td colspan=2><span class="patent-bibdata-value"><a href="#backward-citations">Patent Citations</a> (20),</span> <span class="patent-bibdata-value"><a href="#npl-citations">Non-Patent Citations</a> (70),</span> <span class="patent-bibdata-value"><a href="#forward-citations">Referenced by</a> (10),</span> <span class="patent-bibdata-value"><a href="#classifications">Classifications</a> (12),</span> <span class="patent-bibdata-value"><a href="#legal-events">Legal Events</a> (7)</span> </td></tr><tr><td colspan=2 class="patent-bibdata-external-link-spacer-top"></td></tr><tr class="patent-bibdata-external-link-spacer-bottom"></tr><tr><td colspan=2><span class="patent-bibdata-heading">External Links:&nbsp;</span><span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=s1BcBAABERAJ&q=http://patft.uspto.gov/netacgi/nph-Parser%3FSect2%3DPTO1%26Sect2%3DHITOFF%26p%3D1%26u%3D/netahtml/PTO/search-bool.html%26r%3D1%26f%3DG%26l%3D50%26d%3DPALL%26RefSrch%3Dyes%26Query%3DPN/6455275&usg=AFQjCNGc-SXfLWT0QN_Ut5ydwoWOdVvYfw">USPTO</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=s1BcBAABERAJ&q=http://assignments.uspto.gov/assignments/q%3Fdb%3Dpat%26pat%3D6455275&usg=AFQjCNF-pMdToyRd2aVz_LyWhmZj6b3Y7w">USPTO Assignment</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=s1BcBAABERAJ&q=http://worldwide.espacenet.com/publicationDetails/biblio%3FCC%3DUS%26NR%3D6455275B1%26KC%3DB1%26FT%3DD&usg=AFQjCNGqu_axsR3CkpqOyAWA-_U7vh-JnA">Espacenet</a></span></span></td></tr><tr class="patent-bibdata-group-spacer"></tr></table><div class="number-and-title"><span class="patent-title"><invention-title mxw-id="PT54986363" lang="EN" load-source="patent-office">DNA construct for producing proteinaceous materials in eucaryotic cells</invention-title></span><br><span class="patent-number">US 6455275 B1</span></div><div class="patent-section patent-abstract-section"><div class="patent-section-header"><span class="patent-section-title">Abstract</span></div><div class="patent-text"><abstract mxw-id="PA50384654" lang="EN" load-source="patent-office"> <div class="abstract">The invention relates to processes for inserting into eucaryotic cells a multiplicity of DNA molecules which includes genes coding for desired proteinaceous materials. The insertion of multiple copies of desired genes is accomplished by cotransformation with the desired genes and with amplifiable genes for a dominant selectable marker in the presence of successively higher amounts of an inhibitor. Alternatively, the insertion of multiple copies of desired genes is accomplished by transformation using DNA molecules formed by ligating a DNA molecule including the desired gene to a DNA molecule which includes an amplifiable gene coding for a dominant selectable phenotype such as a gene associated with resistance to a drug in the presence of successively higher amounts of an agent such as a drug against which the gene confers resistance so that only those eucaryotic cells into which multiple copies of the amplifiable gene have been inserted survive. Eucaryotic cells into which multiple copies of the amplifiable gene have been inserted additionally include multiple copies of the desired gene and may be used to produce multiple copies of proteinaceous molecules. In this way otherwise rare proteinaceous materials may be obtained in higher concentrations than are obtainable using conventional techniques.</div>
  </abstract></div></div><div class="patent-section patent-drawings-section"><div class="patent-section-header"><span class="patent-section-title">Images<span class="patent-section-count">(2)</span></span></div><div class="patent-drawings-body"><div class="patent-drawings-carousel"><div class="drawings"><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6455275B1/US06455275-20020924-D00001.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6455275B1/US06455275-20020924-D00001.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6455275B1/US06455275-20020924-D00002.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6455275B1/US06455275-20020924-D00002.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div></div></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img" alt="Previous page"src="/googlebooks/images/kennedy/page_left.png"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img" alt="Next page"src="/googlebooks/images/kennedy/page_right.png"width="21" height="21" /></div></div></div><div class="patent-post-drawings"></div><div class="patent-section patent-claims-section"><div class="patent-section-header"><span class="patent-section-title">Claims<span class="patent-section-count">(20)</span></span></div><div class="patent-text"><div mxw-id="PCLM8364060" lang="EN" load-source="patent-office" class="claims">
    <claim-statement>What is claimed is: </claim-statement> <div class="claim"> <div num="1" id="US-6455275-B1-CLM-00001" class="claim">
      <div class="claim-text">1. A transformed Chinese Hamster Ovary cell comprising a DNA construct comprising DNA I encoding a proteinaceous material foreign to the Chinese Hamster Ovary cell and linked thereto DNA II encoding an amplifiable dominant selectable phenotype not expressed by such Chinese Hamster Ovary cell prior to transformation with the construct, the construct being effective for producing the proteinaceous material when the construct is introduced into the Chinese Hamster Ovary cell, wherein the construct is stably incorporated into the chromosomal DNA of the transformed Chinese Hamster Ovary cell.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="2" id="US-6455275-B1-CLM-00002" class="claim">
      <div class="claim-text">2. A method of producing a proteinaceous material which comprises culturing transformed Chinese Hamster Ovary cells according to <claim-ref idref="US-6455275-B1-CLM-00001">claim 1</claim-ref> under suitable conditions to produce the proteinaceous material and recovering the proteinaceous material so produced.</div>
    </div>
    </div> <div class="claim"> <div num="3" id="US-6455275-B1-CLM-00003" class="claim">
      <div class="claim-text">3. A transformed Chinese Hamster Ovary (CHO) cell which comprises amplified foreign DNA I encoding a proteinaceous material and amplified DNA II encoding a dihydrofolate reductase not expressed by the transformed CHO cell prior to transformation, both DNA I and DNA II being stably incorporated into the chromosomal DNA of the transformed CHO cell.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="4" id="US-6455275-B1-CLM-00004" class="claim">
      <div class="claim-text">4. The transformed Chinese Hamster Ovary cell of <claim-ref idref="US-6455275-B1-CLM-00003">claim 3</claim-ref>, wherein the proteinaceous material is a glycoprotein.</div>
    </div>
    </div> <div class="claim"> <div num="5" id="US-6455275-B1-CLM-00005" class="claim">
      <div class="claim-text">5. A transformed Chinese Hamster Ovary (CHO) cell which comprises amplified foreign DNA I corresponding to a gene of interest which encodes a proteinaceous material and amplified DNA II encoding a dominant selectable phenotype not expressed by the transformed cell prior to transformation, DNA I or DNA II or both being attached to bacterial plasmid DNA or phage DNA, and both DNA I and DNA II being stably incorporated into the chromosomal DNA of the transformed cell.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="6" id="US-6455275-B1-CLM-00006" class="claim">
      <div class="claim-text">6. The transformed CHO cell of <claim-ref idref="US-6455275-B1-CLM-00005">claim 5</claim-ref>, wherein DNA II encodes a dihydrofolate reductase which renders the transformed CHO cell resistant to methotrexate.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="7" id="US-6455275-B1-CLM-00007" class="claim">
      <div class="claim-text">7. The transformed Chinese Hamster Ovary cell of <claim-ref idref="US-6455275-B1-CLM-00005">claim 5</claim-ref>, wherein the DNA I is attached to bacterial plasmid DNA.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="8" id="US-6455275-B1-CLM-00008" class="claim">
      <div class="claim-text">8. The transformed Chinese Hamster Ovary cell of <claim-ref idref="US-6455275-B1-CLM-00005">claim 5</claim-ref>, wherein the DNA II is attached to bacterial plasmid DNA.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="9" id="US-6455275-B1-CLM-00009" class="claim">
      <div class="claim-text">9. The transformed Chinese Hamster Ovary cell of <claim-ref idref="US-6455275-B1-CLM-00005">claim 5</claim-ref>, wherein both DNA I and DNA II is attached to bacterial plasmid DNA.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="10" id="US-6455275-B1-CLM-00010" class="claim">
      <div class="claim-text">10. The transformed Chinese Hamster Ovary cell of <claim-ref idref="US-6455275-B1-CLM-00005">claim 5</claim-ref>, wherein the DNA I is attached to phage DNA.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="11" id="US-6455275-B1-CLM-00011" class="claim">
      <div class="claim-text">11. The transformed Chinese Hamster Ovary cell of <claim-ref idref="US-6455275-B1-CLM-00005">claim 5</claim-ref>, wherein the DNA II is attached to phage DNA.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="12" id="US-6455275-B1-CLM-00012" class="claim">
      <div class="claim-text">12. The transformed Chinese Hamster Ovary cell of <claim-ref idref="US-6455275-B1-CLM-00005">claim 5</claim-ref>, wherein both DNA I and DNA II is attached to phage DNA.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="13" id="US-6455275-B1-CLM-00013" class="claim">
      <div class="claim-text">13. The transformed Chinese Hamster Ovary cell of any of claims <b>5</b>-<b>12</b>, further comprising the proteinaceous material.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="14" id="US-6455275-B1-CLM-00014" class="claim">
      <div class="claim-text">14. A method of producing a proteinaceous protein which comprises culturing transformed CHO cells of <claim-ref idref="US-6455275-B1-CLM-00005">claim 5</claim-ref> under suitable conditions to produce the proteinaceous material and recovering the proteinaceous material so produced.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="15" id="US-6455275-B1-CLM-00015" class="claim">
      <div class="claim-text">15. The method of <claim-ref idref="US-6455275-B1-CLM-00014">claim 14</claim-ref>, wherein the proteinaceous material is glycoprotein.</div>
    </div>
    </div> <div class="claim"> <div num="16" id="US-6455275-B1-CLM-00016" class="claim">
      <div class="claim-text">16. A transformed Chinese Hamster Ovary (CHO) cell which comprises amplified foreign DNA I corresponding to a gene encoding a glycoprotein of interest and amplified DNA II encoding a dominant selectable phenotype not expressed by the transformed CHO cell prior to transformation, and both DNA I and DNA II being stably incorporated into the chromosomal DNA of the transformed Chinese Hamster Ovary cell.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="17" id="US-6455275-B1-CLM-00017" class="claim">
      <div class="claim-text">17. The transformed Chinese Hamster Ovary cell of <claim-ref idref="US-6455275-B1-CLM-00016">claim 16</claim-ref>, wherein DNA II encodes a dihydrofolate reductase which renders the transformed cell resistant to methotrexate.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="18" id="US-6455275-B1-CLM-00018" class="claim">
      <div class="claim-text">18. The transformed Chinese Hamster Ovary cell of <claim-ref idref="US-6455275-B1-CLM-00016">claim 16</claim-ref>, wherein DNA I or DNA II or both DNA I and DNA II is attached to bacterial plasmid DNA or phage DNA.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="19" id="US-6455275-B1-CLM-00019" class="claim">
      <div class="claim-text">19. The transformed Chinese Hamster Ovary cell of any of claims <b>16</b>-<b>18</b>, further comprising the glycoprotein of interest.</div>
    </div>
    </div> <div class="claim"> <div num="20" id="US-6455275-B1-CLM-00020" class="claim">
      <div class="claim-text">20. A DNA construct for expression in Chinese Hamster Ovary <u>(CHO)</u> cells comprising DNA I encoding a proteinaceous material foreign to such <u>CHO</u> cells and linked thereto DNA II encoding a dihydrofolate reductase which is not expressed by such CHO cells and renders such CHO cells resistant to methotrexate when the CHO cells are transformed with the construct, the construct being effective for producing the proteinaceous material when the construct is introduced into such CHO cells.</div>
    </div>
  </div> </div></div></div><div class="patent-section patent-description-section"><div class="patent-section-header"><span class="patent-section-title">Description</span></div><div class="patent-text"><div mxw-id="PDES53640435" lang="EN" load-source="patent-office" class="description">
    <p>This application is a continuation of now abandoned U.S. Ser. No. 08/395,520, filed Feb. 27, 1995; which is a continuation of U.S. Ser. No. 08/217,007, filed Mar. 23, 1994, now abandoned; which is a continuation of U.S. Ser. No. 07/866,800, filed Jun. 26, 1992, now abandoned; which is a continuation of U.S. Ser. No. 07/716,915, filed Jun. 18, 1991, now U.S. Pat. No. 5,179,017, issued Jan. 12, 1993; which is a divisional of U.S. Ser. No. 07/346,089, filed May 2, 1989, now abandoned; which is a continuation of U.S. Ser. No. 06/915,273, filed Oct. 3, 1986, now abandoned; which is a divisional of U.S. Ser. No. 06/522,408, filed Aug. 11, 1983, now U.S. Pat. No. 4,634,665, issued Jan. 6, 1987; which is a divisional of U.S. Ser. No. 06/124,513, filed Feb. 25, 1980, now U.S. Pat. No. 4,399,216, issued Aug. 16, 1983.</p>
    <heading>FIELD OF THE INVENTION</heading> <p>This invention concerns the introduction and expression of genetic informational material, i.e., DNA which includes genes coding for proteinaceous materials and/or genes regulating or otherwise influencing the production thereof, into eucaryotic cells, that is, cells of organisms classified under the Superkingdom Eucaryotes including organisms of the Plant and Animal Kingdoms. Such genetic intervention is commonly referred to as genetic engineering. and in certain aspects involves the use of recombinant DNA technology. The invention disclosed is to be distinguished from the introduction of DNA into organisms of the Superkingdom Procaryotes including particularly bacteria. This distinction is based in part upon the basic differences between eucaryotic and procaryotic cells, the former being characterized by true nuclei formed by nuclear envelopes and by meiosis and the latter being characterized by the absence of well-defined nuclei and the absence of melosis. Moreover, at the genetic level many genes in eucaryotes are split by non-coding sequences which are not continuously colinear, whereas in procaryotes, the genes are continuously colinear.</p>
    <heading>BACKGROUND OF THE INVENTION</heading> <p>Although advances in the understanding of procarvotic organisms, particularly bacteria, having for the most part proceeded independently of advances in the understanding of eucaryotic organisms,it may be helpful to an appreciation of the present invention to set forth certain developments involving procaryotes.</p>
    <p>In 1944, Avery reported the transformation of a procaryotic cell using DNA-mediated transfer of a cellular gene. Avery, O. T., et al., J. Exp. Med. 79: 137-158 (1944). Thereafter, reports of procaryotic transformation occurred in the literature. In 1975, Cohen and others reported results involving first transformation, then cotransformation of the procaryote <i>Escherichia coli. </i>Kretschmer, P. J., et al., J. Bacteriology 124: 225-231 (1975). In the experiments reported therein the authors disclosed the cotransformation of procaryotic cells using plasmid DNA, that is, extrachromosomal DNA which occurs naturally in many strains of Enterobacteriacae. In these experiments it was found that particular cells in a CaCl<sub>2 </sub>treated bacterial population are preferentially competent for transformation. However, the frequency of transformation and the stability of the transformants obtained was low, possibly because the plasmid is not incorporated into the chromosomal DNA. As a result, cotransformants lost acquired traits after several generations. In addition, these experiments with bacteria required the addition of a gene promoter to the transforming DNA in order to obtain expression.</p>
    <p>Meanwhile, experiments with eucaryotic cells proceeded substantially independently of those with procaryotic cells.</p>
    <p>In 1962, Szybalska, E. H. and Szybalski, W. PNAS 48: 2026 (1962) reported the transformation of mammalian cells but with such low frequency of transformation that it was not possible to distinguish transformants from cells which had merely undergone spontaneous reversion. Again, as with procaryotic cells, further reports of eucaryotic transformation occurred in the literature, but such results were oftentimes not reproducible by others. In addition, low frequencies of transformation, lack of understanding of the molecular basis for gene expression and the lack of molecular hybridization probes contributed to the lack of progress in this area. As a result, studies on the transformation of eucaryotic cells were essentially restricted to viral genes. Graham, F. L., et al., Cold Spring Harbor Symp. Quant. Biol. 39: 637-650 (1975) and McCutchen, J. H. and Pagano. J. S., Journal National Cancer Institute, 41: 351-357 (1968,).</p>
    <p>More recently, however, eucaryotic cells, specifically mammalian cells, were transformed with foreign DNA coding for a selectable phenotype. Wigler, M., et al., Cell 11: 223-232 (1977). This work has been extended and has resulted in the present invention wherein it has been discovered inter alia that eucaryotic cells can be cotransformed to yield transformants having foreign DNA integrated into the chromosomal DNA of the eucaryotic cell nucleus. Moreover, it has unexpectedly been discovered that such foreign DNA can be expressed by the cotransformants to generate functional proteins. In addition, by contrast with procaryotic transformants, the foreign DNA is stably expressed through hundreds of generations, a result that may be attributable to integration of the foreign DNA into the chromosomal DNA.</p>
    <p>The present invention provides major advances over bacterial systems for future use in the commercial preparation of proteinaceous materials particularly proteins of eucaryotic origin such as interferon protein, antibodies, insulin, and the like. Such advantages include the ability to use unaltered genes coding for precursors for such proteinaceous materials. After cellular synthesis, the precursor can be further processed or converted within the eucaryotic cell to produce the desired molecules of biological significance. This phenomenon is well known for insulin which is initially produced in the eucaryotic cell as preproinsulin which is then converted to active insulin within the cell by appropriate peptide cleavage. Since procaryotic cells lack the requisite cellular machinery for converting preproinsulin to insulin, the insertion into a procaryotic cell of the eucaryotic gene associated with insulin will result in the production of preproinsulin, not insulin. Although, in the case of insulin, a relatively small and well characterized protein, this difficulty can be overcome by chemical synthesis of the appropriate gene, such an approach is inherently limited by the level of understanding of the amino acid sequence of the desired protein. Thus, for interferon protein, clotting factors, antibodies and uncharacterized enzymes, for which the exact amino acid sequence is not yet known, a procaryotic system will likely not prove satisfactory. By contrast, a eucaryotic system is not associated with such disadvantages since the eucaryotic cell possesses the necessary processing machinery. It is thus one important object of the present invention to provide a process for producing desired proteinaceous materials such as interferon protein, insulin, antibodies and the like which does not require a detailed molecular understanding of amino acid sequence.</p>
    <p>In addition to the problem of precursors having additional amino acids which must be removed to produce active protein, important biological materials may be modified by chemical additions after synthesis and cleavage. Thus, for example, human-produced interferon is a glycoprotein containing sugar molecules in addition to protein. If produced in a bacterial cell, the interferon lacks the sugar molecules which are added when interferon is produced in a human cell. Moreover, proteinaceous materials produced within bacteria may include endotoxins which can cause inflammation if the proteinaceous material is administered to a mammal without significant purification. By contrast, interferon produced in a eucaryotic cell would be free of endotoxins.</p>
    <p>It is therefore another important object of this invention to provide a process for producing compounds which include both non-proteinaceous and proteinaceous moieties such as glycoproteins which cannot be produced in bacterial cell.</p>
    <heading>SUMMARY OF THE INVENTION</heading> <p>This invention provides a process for inserting foreign DNA into eucaryotic cells by cotransforming the cells with this foreign DNA and with unlinked DNA which codes for proteinaceous material associated with a selectable phenotype not otherwise expressed by the cell. The cotransformation is carried out in a suitable medium. and in the presence of selective conditions permitting survival and/or identification of eucaryotic cells which have acquired the selectable phenotype. The process of this invention is particularly suited for the insertion into eucaryotic cells of DNA which codes for proteinaceous materials which are not associated with a selectable phenotype such as interferon protein, insulin, growth hormones, clotting factors, viral antigens, antibodies and certain enzymes.</p>
    <p>By use of the cotransformation process of the present invention is it possible to produce eucaryotic cells which synthesize desired proteinaceous and other materials and which can be grown in culture to produce these materials in quantities not obtainable with conventional technology.</p>
    <p>In one embodiment of the invention, the cotransformation process can be used to insert multiple copies of genes coding for desired materials into eucaryotic cells. Alternatively, a multiplicity of foreign DNA molecules corresponding to multiple copies of a desired gene can be inserted into eucaryotic cells by transformation with molecules each of which is formed by linking a foreign DNA molecule to a second DNA molecule corresponding to an amplifiable gene for a dominant selectable phenotype not otherwise expressed by the cell. The transformation is then carried out in the presence of successively elevated concentrations of an agent permitting survival and/or identification of eucaryotic cells which have acquired multiple copies of the amplifiable gene. This approach is particularly useful when the dominant selectable phenotype is resistance to a drug which is lethal, unless multiple copies of the drug resistant gene are present and the agent is the drug.</p>
    <p>By inserting multiple copies of genes coding for desired materials into eucaryotic cells according to either of these approaches it is possible to produce eucaryotic cells which yield desired materials in high concentrations and which can be grown in culture to produce such materials in quantities not obtainable with conventional. technology.</p>
    <heading>BRIEF DESCRIPTION OF THE DRAWINGS</heading> <p>FIG. 1 is a schematic flow diagram illustrating the cotransformation process in accordance with the present invention.</p>
    <p>FIG. 2 is a schematic flow diagram illustrating a process for recovering foreign DNA I from cotransformed cultured cells using double selection techniques.</p>
    <heading>DETAILED DESCRIPTION OF THE INVENTION</heading> <p>Prior to setting forth the invention, it may be helpful to an understanding thereof to set forth definitions of certain terms to be used hereinafter.</p>
    <p>Transformation means the process for changing the genotype of a recipient cell mediated by the introduction of purified DNA. Transformation is typically detected by a stable and heritable change in the phenotype of the recipient cell that results from an alteration in either the biochemical or morphological properties of the recipient cell.</p>
    <p>Cotransformation means the process for carrying out transformations of a recipient cell with more than one different gene. Cotransformation includes both simultaneous and sequential changes in the genotype of a recipient cell mediated by the introduction of DNA corresponding to either unlinked or linked genes.</p>
    <p>Proteinaceous material means any biopolymer formed from amino acids.</p>
    <p>Genotype means the genetic constitution of an organism as distinguished from its physical appearance.</p>
    <p>Phenotype means the observable properties of an organism as produced by the genotype in conjunction with the environment.</p>
    <p>Selectable phenotype is a phenotype which confers upon an organism the ability to exist under conditions which kill off all organisms not possessing the phenotype. Examples include drug resistance or the ability to synthesize some molecule necessary to cell metabolism in a given growth medium. As used herein, selectable phenotypes also include identifiable phenotypes such as the production of materials which pass from or are secreted by the cell and can be detected as new phenotypes either by functional, immunologic or biochemical assays.</p>
    <p>Interferon protein means the proteinaceous part of the glycoprotein interferon, that is, the portion remaining after removal of the sugar portion. It includes the protein portion of interferon derived from human leukocyte, fibroblast or lymphoblastoid cells.</p>
    <p>Chromosomal DNA means the DNA normally associated with histone in the form of chromosomes residing in the nucleus of a eucaryotic cell.</p>
    <p>Transcription means the formation of a RNA chain in accordance with the genetic information contained in the DNA.</p>
    <p>Translation means the process whereby the genetic information in an mRNA molecule directs the order of specific amino acids during protein synthesis.</p>
    <p>In accordance with the present invention, foreign DNA I can be inserted into any eucaryotic cell by cotransforming the cell with DNA I and with unlinked foreign DNA II which includes a gene coding for a selectable phenotype not expressed by the cell unless acquired by transformation. The cotransformation is carried out in a suitable growth medium and in the presence of selective conditions such that the only cells which survive or are otherwise altered are those which have required the selectable phenotype. See FIG. <b>1</b>.</p>
    <p>Although the experiments discussed hereinafter concern cultured eucaryotic cells of mammalian origin such as human blood cells, mouse fibroblast cells, chinese hamster ovary cells and mouse teratocarcinoma cells, it is clear that the process described is generally applicable to all eucaryotic cells including, for example, cells from birds such as chickens cells from yeast and fungi, and cells from plants including grains and flowers. Therefore, it is to be understood that the invention encompasses all eucaryotic cells even though the invention may ultimately be most useful in cotransforming mammalian cells.</p>
    <p>The present invention is especially useful in connection with the insertion into eucaryotic cells of foreign DNA which includes genes which code for proteinaceous materials not associated with selectable phenotypes. Since such proteinaceous materials are characterized by the fact that they are not associated with a selectable phenotype, cells which contain DNA coding therefore cannot be identified except by destruction of the transformed cell and examination of its contents.</p>
    <p>Examples of proteinaceous materials, the genes for which may be inserted into and expressed by eucaryotic cells using the cotransformation process include interferon protein, insulin, growth hormones, clotting factors, viral antigens, enzymes and antibodies.</p>
    <p>Although in some cases the DNA I and DNA II may not need to be purified to obtain integration and expression, it is oftentimes preferable that the DNAs be purified prior to use in cotransforming cells. Such purification limits the possibility of spurious results due to the presence of contaminants and increases the probability that cotransformed cells can be identified and stably cultured. Also, although not essential, it is sometimes desirable that DNA I and/or DNA II have been obtained by restriction endonuclease cleavage of chromosomal donor DNAs, such as, for example, restriction endonuclease cleavage of eucaryotic chromosomal DNA. Additionally, it is preferable that DNA I and DNA II be treated with calcium phosphate prior to use in cotransforming eucaryotic cells. The procedure for so treating DNA with calcium phosphate is set forth more fully hereinafter. Finally, it is preferable that the foreign DNA I be present during cotransformation in an amount relative to DNA II coding for a selectable phenotype which constitutes an excess of the former, such as an amount in the range from about 1:1 to about 100,000:1.</p>
    <p>In a preferred embodiment of the invention, the foreign DNA I and/or the foreign DNA II are attached to bacterial plasmid or phage DNA prior to use in cotransforming eucaryotic cells. In a particularly promising embodiment, foreign DNA I and/or DNA II are attached to phage DNA and then encapsidated in phage particles prior to cotransformation.</p>
    <p>Although any DNA II coding for a selectable phenotype would be useful in the cotransformation process of the present invention, the experimental details set forth particularly concern the use of a gene for thymidine kinase obtained from herpes simplex virus and the use of a gene for adenine phosphoribosyl transferase. In addition, a DNA II which includes a gene coding for a selectable phenotype associated with drug resistance, e.g., a mutant dihydrofolate reductase gene which renders cells resistant to methotrexate greatly extends the applicability of the process.</p>
    <p>In accordance with a preferred embodiment, the cotransformation involves DNA I which is physically and chemically unlinked to DNA II, and the DNA I is stably integrated into the chromosomal DNA within the nucleus of the cotransformed eucaryotic cell.</p>
    <p>Cotransformation in accordance with this invention may be carried out in any suitable medium limited only in that cotransformed cells be capable of survival and/or identification on the medium. Merely by way of example, a suitable medium for mouse fibroblast cells which have acquired the thymidine kinase gene is HAT described more fully hereinafter. Also, the cotransformation is carried out in the presence of selective conditions which permit survival and/or identification of those cells which have acquired the selectable phenotype. Such conditions may include the presence of nutrients, drug or other chemical antagonists, temperature and the like.</p>
    <p>Eucaryotic cells cotransformed in accordance with this invention contain foreign DNA I coding for desired materials which can be recovered from the cells using techniques well known in the art. Additionally, the cells can be permitted to transcribe DNA I to form mRNA which in turn is translated to form protein or other desired material which may be recovered, again using well known techniques. Finally, the cells can be grown in culture, harvested and protein or other desired material recovered therefrom.</p>
    <p>Although the desired proteinaceous materials identified hereinabove are natural materials, the process can be equally useful in the production of synthetic biopolymers for which synthetic genes are constructed. Thus, the instant invention provides a process for producing novel proteins not yet in existence. Additionally, it provides a process for producing proteins which, although they presently exist, do so in such minute quantities or in such impure form that their isolation and/or identification cannot otherwise be effected. Finally, the invention provides a process for producing partially proteinaceous products such as the glycoproteins and other products, the synthesis of which is genetically directed.</p>
    <p>Another aspect of the invention involves processes for inserting multiple copies of genes into eucaryotic cells in order to increase the amount of gene product formed within the cell. One process for inserting a multiplicity of foreign DNA I molecules into a eucaryotic cell comprises cotransforming the cell with multiple DNA I molecules and with multiple, unlinked foreign DNA II molecules corresponding to multiple copies of an amplifiable gene for a dominant selectable phenotype not otherwise expressed by the cell. This cotransformation process is carried out in a suitable medium and in the presence of an agent permitting survival and/or identification of cells which acquire the dominant selectable phenotype. Preferably, this is done in the presence of successively higher concentrations of such an agent so that only those cells acquiring the highest number of amplifiable dominant genes (DNA II) survive and/or are identified. These cells then also contain multiple copies of DNA I. This approach is particularly appropriate for the insertion of multiple copies of amplifiable genes which confer drug resistance upon the cell, e.g., the mutant dihydrofolate reductase gene which renders cells resistant to methotrexate.</p>
    <p>Cotransformed eucaryotic cells which have acquired multiple copies of DNA I may then be used to produce increased amounts of the gene product for which DNA I codes in the same manner as described hereinabove.</p>
    <p>Alternatively, multiple copies of foreign genes can be generated in and ultimately expressed by eucaryotic cells by transforming the eucaryotic cells with DNA molecules, each of which has been formed by linking a foreign DNA I to a foreign DNA II which corresponds to an amplifiable gene for a dominant selectable phenotype not normally expressed by the eucaryotic cell. The linkage between DNA I and DNA II is preferably in the form of a chemical bond, particularly a bond formed as a result of enzymatic treatment with a ligase. Transformation with such hybrid DNA molecules so formed is then carried out in a suitable growth medium and in the presence of successively elevated concentrations, e.g., amounts ranging from 1:1 to 10,000:1 on a molarity basis, of an agent which permits survival and/or identification of those eucaryotic cells which have acquired a sufficiently high number of copies of the amplifiable gene. Using this approach, eucaryotic cells which have acquired multiple copies of the amplifiable gene for a dominant selectable phenotype not otherwise expressed by the cell survive and/or are identifiable in the presence of elevated concentrations of an agent complementary to the amplifiable gene which would otherwise result in death or inability to identify the cells.</p>
    <p>Although various amplifiable genes for dominant selectable phenotypes are useful in the practices of this invention, genes associated with drug resistance, e.g., the gene for dihydrofolate reductase which renders cells resistant to methotrexate, are particularly suitable.</p>
    <p>By using either of the two approaches just described, multiple copies of proteinaceous or other desired molecules can be produced within eucaryotic cells. Thus, for example, multiple molecules of interferon protein, insulin, growth hormone, clotting factor, viral antigen or antibody or of interferon per se can be produced by eucaryotic cells, particularly mammalian cells, which have been transformed using hybrid DNA or cotransformed using purified DNA which has been treated, with calcium phosphate in the manner described hereinafter. Thus, this invention provides a process for producing highly desired, rare and costly proteinaceous and other biological materials in concentrations not obtainble using conventional techniques.</p>
    <p>Still another aspect of the present invention involves the preparation of materials normally produced within eucaryotic cells in minute amounts such as glycoproteins including interferon, which are in part protein but additionally include other chemical species such as sugars, ribonucleic acids, histones and the like. Although the method or methods by which cells synthesize complicated cellular materials such as the glycoproteins are poorly understood, it is anticipated that by using the process of the present invention it will be possible to synthesize such materials in commercially useful quantities. Specifically, it is anticipated that after inserting a gene or genes for the protein portion of a cellular material such as a glycoprotein, which includes a non-protein portion, into a eucaryotic cell of the type, which normally produces such material, the cell will not only produce the corresponding proteinaceous material but will utilize already existing cellular mechanisms to process the proteinaceous materials, if and to the extent necessary, and will also add the appropriate non-proteinaceous material to form the complete, biologically active material. Thus, for example, the complete biologically active glyprotein, interferon, could be prepared by first synthesizing interferon protein in the manner described and additionally permitting the cell to produce the non-proteinaceous or sugar portion of interferon and to synthesize or assemble true interferon therefrom. The interferon so prepared could then be recovered using conventional techniques.</p>
    <p>In accordance with. the present invention and as described more fully hereinafter, eucaryotic cells have been stably transformed with precisely defined procaryotic and eucaryotic genes for which no selective criteria exist.</p>
    <p>The addition of a purified viral thymidine kinase (tk) gene to mouse cells lacking this enzyme results in the appearance of stable transformants which can be selected by their ability to grow in HAT medium. Since these biochemical transformants might represent a subpopulation of competent cells which are likely to integrate other unlinked genes at frequencies higher than the general population; cotransformation experiments were performed with the viral tk gene and bacteriophage ΦX174, plasmid pBR 322 or cloned chromosomal human or rabbit β-globin gene sequences. Tk transformants were cloned and analyzed for cotransfer of additional DNA sequences by blot hybridization. In this manner, mouse cell lines were identified which contain multiple copies of ΦX, pBR 322, or human and rabbit β-globin sequences. From one to more than 50 cotransformed sequences are integrated into high molecular weight DNA isolated from independent clones. Analysis of subclones demonstrates that the cotransformed DNA is stable through many generations in culture. This cotransformation system allows the introduction and stable integration of virtually any defined gene into cultured eucaryotic cells. Ligation to either viral vectors or selectable biochemical markers is not required.</p>
    <p>Cotransformation with dominant-acting markers should in principle permit the introduction of virtually any cloned genetic element into wild-type cultured eucaryotic cells. To this end, a dominant-acting, methotrexate resistant, dihydrofolate reducatse gene from CHO A29 cells was transferred to wild-type cultured mouse cells. By demonstrating the presence of CHO DHFR sequences in transformants, definitive evidence for gene transfer was provided. Exposure of these cells to elevated levels of methotrexate results in enhanced resistance to this drug, accompanied by amplification of the newly transferred gene. The mutant DHFR gene, therefore, has been used as a eucaryotic vector, by ligating CHO A29 cell DNA to pBR 322 sequences prior to transformation. Amplification of the DHFR sequences results in amplification of the pBR 322 sequences. The use of this gene as a dominant-acting vector in eucaryotic cells will expand the repetoire of potentially transformable cells, no longer restricting these sort of studies to available mutants.</p>
    <p>Using the techniques described, the cloned chromosomal rabbit β-globin gene has been introduced into mouse fibroblasts by DNA-mediated gene transfer. The cotransformed mouse fibroblast containing this gene provides a unique opportunity to study the expression and subsequent processing of these sequences in a heterologous host. Solution hybridization experiments in concert with RNA blotting techniques indicate that in at least one transformed cell line rabbit globin sequences are expressed in the cytoplasm as a polyadenylated 9S species. These 9S sequences result from perfect splicing and removal of the two intervening sequences. These results therefor suggest that nonerythroid cells from heterologous species contain the enzymes necessary to correctly process the intervening sequences of a rabbit gene whose expression is usually restricted to erythroid cells. Surprisingly, however, 45 nucleotides present at the 5′ terminus of mature rabbit mRNA are absent from the globin mRNA sequence detected in the cytoplasm of the transformants examine. These studies indicate the potential value of cotransformation systems in the analysis of eucaryotic gene expression. The introduction of wild type genes along with native and in vitro constructed mutant genes into cultured cells provides an assay for the functional significance of sequence organization.</p>
    <p>Recombinant DNA technology has facilitated the isolation of several higher eucaryotic genes for which hybridization probes are available. Genes expressed at exceedingly low levels, with mRNA transcripts present at from one to 20 copies per cell, such as those genes coding for essential metabolic functions, cannot be simply isolated by conventinal techniques involving construction of cDNA clones and the ultimate screening of recombinant libraries. An alternative approach for the isolation of such rarely expressed genes has therefore been developed utilizing transformation in concert with a procedure known as plasmid rescue. This schema which is currently underway in the laboratory is outlined below. The aprt gene of the chicken is not cleaved by the enzyme, Hin III or Xba, and transformation of aprt<sup>−</sup> mouse cells with cellular DNA digested with these enzymes results in the generation of aprt<sup>+</sup> clonies which express the chicken aprt genes. Ligation of Hin III-cleaved chicken DNA with Hin III-cleaved plasmid pBR 322 results in the formation of hybrid DNA molecules in which the aprt gene is now adjacent to plasmid sequences. Transformation of aprt<sup>−</sup> cells is now performed with this DNA. Transformants should contain the aprt gene covalently linked to pBR 322 with this entire complex integrated into high molecular weight DNA in the mouse cell. This initial cellular transformation serves to remove the chicken aprt gene from the vast majority of other chick sequences. This transformed cell DNA is now treated with an enzyme, Xba I, which does not cleave either pBR 322 or the aprt gene. The resultant fragments are then circularized with ligase.</p>
    <p>One such fragment should contain the aprt gene covalently linked to pBR 322 sequences coding for an origin of replication and the ampicillin resistant marker. Transformation of a bacterium such as <i>E. coli </i>with these circular markers selects for plasmid sequences from eucaryotic DNA which are now linked to chicken aprt sequences. This double selection technique should permit the isolation of genes expressed at low levels in eucaryotic cells for which hybridization probes are not readily obtained.</p>
    <p>In order to assist in a better understanding of the present invention, the results of various experiments are now set forth.</p>
    <heading>EXPERIMENTAL DETAILS FIRST SERIES OF EXPERIMENTS</heading> <p>The identification and isolation of cells transformed with genes which do not code for selectable. markers is problematic since current transformation procedures are highly inefficient. Thus, experiments were undertaken to determine the feasibility of cotransforming cells with two physically unlinked genes. In these experiments it was determined that cotransformed cells could be identified and isolated when one of the genes codes for a selectable marker. Viral thymidine kinase gene was used as a selectable marker to isolate mouse cell lines which contain the tk gene along with either bacteriphage ΦX 174, plasmid pBR 322 or cloned rabbit β-globin gene sequences stably integrated into. cellular DNA. The results of these experiments are also set forth in Wigler, M., et al., Cell 16: 777-785 (1979) and Wold, B. et al., Proc. Nat'l. Acad. Sci. 76: 5684-5688 (1979) are as follows:</p>
    <heading>Experimental Design</heading> <p>The addition of the purified thymidine kinase (tk). gene from herpes simplex virus to mutant mouse cells lacking tk results in the appearance of stable transformants expressing the viral gene which can be selected by their ability to grow in HAT. Maitland, N. J. and. McDougall J. K. Cell, 11: 233-241 (1977); Wigler, M. et al., Cell 11: 223-232 (1977). To obtain cotransformants, cultures are exposed to the tk gene in the presence of an excess of a well-defined DNA sequence for which hybridization probes are available. Tk transformants are isolated and scored for the cotransfer of additional DNA sequences by molecular hybridization.</p>
    <heading>Cotransformation of Mouse Cells with ΦX174 DNA</heading> <p>ΦX174 DNA was initially used in cotransformation experiments with the tk gene as the selectable marker. ΦX replicative form DNA was cleaved with Pst 1, which recognizes a single site in the circular genome. Sanger, F. et al., Nature 265: 687-695 (1977). 500 ρg of the purified tk gene were mixed with 1-10 μg of Pst-cleaved ΦX replicative form DNA. This DNA was then added to mouse Ltk<sup>−</sup> cells using the transformation conditions described under Methods and Materials hereinafter. After 2 weeks in selective medium (HAT), tk<sup>+</sup> transformants were observed at a frequency of one colony per 10 cells per 20 ρg of purified gene. Clones were picked and grown to mass culture.</p>
    <p>It was then asked whether tk<sup>+</sup> transformants also contained ΦX DNA sequences. High molecular weight DNA from the transformants was cleaved with the restriction endonuclease Eco RI, which recognizes no sites in the ΦX genome. The DNA was fractionated by agarose gel electrophoresis and transferred to nitrocellulose filters, and these filters were then annealed with nick-translated <sup>32</sup>P-ΦX DNA (blot hybridization). Southern, E. M., J. Mol. Biol. 98: 503-517 (1975); Botchan, M., et al., Cell 9: 269-287 (1976); Pellicer, A., et al. Cell 14: 133-141 (1978). These annealing experiments demonstrate that six of the seven transformants had acquired bacteriophage sequences. Since the ΦX genome is not cut by the enzyme Eco RI, the number of bands observed reflects the minimum number of eucaryotic DNA fragments containing information homologous to ΦX. The clones contain variable amounts of ΦX sequences. Clones ΦX1 and ΦX2 reveal a single annealing fragment which is smaller than the ΦX genome. In these clones, therefore, only a portion of the transforming sequences persist. There was also observed a tk<sup>+</sup> transformant (clone ΦX3) with no detectable ΦX sequences. Clones ΦX4, 5, 6, and 7 reveal numerous high molecular weight bands which are too closely spaced to count, indicating that these clones contain multiple ΦX-specific fragments. These experiments demonstrate cotransformation of cultured mammalian cells with the viral tk gene and ΦX DNA.</p>
    <heading>Selection Is Necessary to Identify ΦX Transformants</heading> <p>It was next asked whether transformants with ΦX DNA was restricted to the population of tk<sup>+</sup> cells or whether a significant proportion of the original culture now contained ΦX sequences. Cultures were exposed to a mixture of the tk gene and ΦX DNA in a molar ratio of 1:2000 or 1:20,000. Half of the cultures were plated under selective conditions, while the other half were plated in neutral media at low density to facilitate cloning. Both selected (tk<sup>+</sup>) and unselected (tk<sup>−</sup>) colonies were picked, grown into mass culture and scored for the presence of ΦX sequences. In this series of experiments, eight of the nine selected colonies contained phage information. As in the previous experiments, the clones contained varying amounts of ΦX DNA. In contrast, none of fifteen clones picked at random from neutral medium contained any ΦX information. Thus, the addition of a selectable marker facilitates the identification of those cells which contain ΦX DNA.</p>
    <heading>ΦX Sequences Are Integrated into Cellular DNA</heading> <p>Cleavage of DNA from ΦX transformants with Eco RI generates a series of fragments which contain ΦX DNA sequences. These fragments may reflect multiple integration events. Alternatively, these fragments could result from tandem arrays of complete or partial ΦX sequences which are not integrated into cellular DNA. To distinguish between these possibilities, transformed cell DNA was cut with BAM HI or Eco RI, neither of which cleaves the ΦX genome. If the ΦX DNA sequences were not integrated, neither of these enzymes would cleave the ΦX fragments. If the ΦX DNA sequences were not integrated, neither of these enzymes would cleave the ΦX fragments. Identical patterns would be generated from undigested DNA and from DNA cleaved with either of these enzymes. If the sequences are integrated, then BAM HI and Eco RI should recognize different sites in the flanking cellular DNA and generate unique restriction patterns. DNA from clones ΦX4 and ΦX5 was cleaved with BAM III or Eco RI and analyzed by Southern hybridization. In each instance, the annealing pattern with Eco RI fragments differed from that observed with the BAM HI fragments. Furthermore, the profile obtained with undigested DNA reveals annealing only in very high molecular weight regions with no discrete fragments observed. Similar observations were made on clone ΦX1. Thus, the most of the ΦX sequences in these three clones are integrated into cellular DNA.</p>
    <p>Intracellular Localization of the ΦX Sequences</p>
    <p>The location of ΦX sequences in transformed cells was determined by subcellular fractionation. Nuclear and cytoplasmic fractions was prepared, and the ΦX DNA sequence content of each was assayed by blot hybridization. The data indicate that 95% of the ΦX sequences are located in the nucleus. High and low molecular weight nuclear DNA was prepared by Hirt fractionation. Hirt, B. J., Mol. Biol. 26: 365-369 (1967). Hybridization with DNA from these two fractions indicates that more than 95% of the ΦX information co-purifies with the high molecular weight DNA fraction. The small amount of hybridization observed in the supernatant fraction reveals a profile identical to that of the high molecular weight DNA, suggesting contamination of this fraction with high molecular weight DNA.</p>
    <heading>Extent of Sequence Representation of the ΦX Genome</heading> <p>The annealing profiles of DNA from transformed clones digested with enzymes that do not cleave the ΦX genome provide evidence that integration of ΦX sequences has occurred and allow us to estimate the number of ΦX sequences integrated. Annealing profiles of DNA from transformed clones digested with enzymes which cleave within the ΦX genome allow us to determine what proportion of the genome is present and how these sequences are arranged following integration. Cleavage of ΦX with the enzyme Hpa I generates three fragments for each integration event: two “internal” fragments of 3.7 and 1.3 kb which together comprise 90% of the ΦX genome, and one “bridge” fragment of 0.5 kb which spans the Pst I cleavage site. In the annealing profile observed when clone ΦX4 is digested with Hpa I, two intense bands are observed at 3.7 and 1.3 kb. A less intense series of bands of higher molecular weight is also observed, some of which probably represent (X sequences adjacent to cellular DNA. These results indicate that at least 90% of the ΦX genome is present in these cells. It is worth noting that the internal 1.3 kb Hpa I fragment is bounded by an Hpa I site only 30 bp from the Pst I cleavage site. Comparison of the intensities of the internal bands with known quantities of Hpa I-cleaved ΦX DNA suggests that this clone contains approximately 100 copies of the 4X genome. The annealing pattern of clone 5 DNA cleaved with Hpa I is more complex. If internal fragments are present, they are markedly reduced in intensity; instead, multiple bands of varying molecular weight are observed. The 0.5 kb Hpa I fragment which bridges the Pst 1 cleavage site is not observed for either clone ΦX4 or clone ΦX5.</p>
    <p>A similar analysis of clone ΦX4 and ΦX5 was performed with the enzyme Hpa II. This enzyme cleaves the ΦX genome five times, thus generating four, “internal” fragments of 1.7, 0.5, 0.5 and 0.2 kb, and a 2.6 kb “bridge” fragment which spans the Pst I cleavage site. The annealing patterns for Hpa II-cleaved DNA from ΦX clones 4 and 5 each show an intense 1.7 kb band, consistent with the retention of at least two internal Hpa II sites. The 0.5 kb internal fragments can also be observed, but they are not shown on this gel. Many additional fragments, mostly of high molecular weight, are also present in each clone. These presumably reflect the multiple integration sites of ΦX DNA in the cellular genome. The 2.6 kb fragment bridging the Pst I cleavage site, however, is absent from clone ΦX4. Reduced amounts of annealing fragments which co-migrate with the 2.6 kb Hpa II bridge fragment are observed in clone ΦX5. Similar observations were made in experiments with the enzyme Hae III. The annealing pattern of Hae III-digested DNA from these clones was determined. In accord with previous data, the 0.87 kb Hae III bridge fragment spanning the Pst site is absent or present in reduced amount in transformed cell DNA. Thus, in general, “internal” fragments of ΦX are found in these transformants, while “bridge” fragments which span the Pst I cleavage site are reduced or absent.</p>
    <heading>Stability of the Transformed Genotype</heading> <p>Previous observations on the transfer of selectable biochemical markers indicate that the transformed pheno-type remains stable for hundreds of generations if cells are maintained under selective pressure. If maintained in neutral medium, the transformed phenotype is lost at frequencies which range from 0.1 to as high as 30% per generation. Wigler, M., et al., Cell 11: 223-232 (1977); Wigler, M. et al., PNAS 76: 5684-5688 (1979). The use of transformation to study the expression of foreign genes depends upon the stability of the transformed genotype. This is an important consideration with genes for which no selective criteria are available. It was assumed that the presence of ΦX DNA in transformants confers no selective advantage on the recipient cell. Therefore, the stability of the ΦX genotype was examined in the descendants of two clones after numerous generations in culture. Clone ΦX4 and ΦX5, both containing multiple-copies of ΦX DNA, were subcloned and six independent subclones from each clone were picked and grown into mass culture. DNA from each of these subclones from each original clone were picked and grown into mass culture. DNA from each of these subclones was then digested with either Eco RI or Hpa I, and the annealing profiles of ΦX-containing fragments were compared with those of the original parental clone. The annealing pattern observed for four of the six ΦX4 subclones is virtually identical to that of the parent. In two subclones, an additional Eco RI fragment appeared which is of identical molecular weight in both. This may have resulted from genotypic heterogeneity in the parental clone prior to subcloning. The patterns obtained for the subclones of ΦX5 are again virtually identical to the parental annealing profile. These data indicate that ΦX DNA is maintained within the ten subclones examined for numerous generations without significant loss or translocation or information.</p>
    <heading>Integration of pBR322 into Mouse Cells</heading> <p>The observations in cotransformation have been extended to the EK2-approved bacterial vector, plasmid pBR322. pBR322 linearized with BAM HI was mixed with the purified viral tk gene in a molar ratio of 1000:1. Tk<sup>+</sup> transformants were selected and scored for the presence of pBR322 sequences. Cleavage of BAM HI linearized pBR322 DNA with Bgl I generates two internal fragments of 2.4 and 0.3 kb. The sequence content of the pBR322 transformants was determined by digestion of transformed cell DNA with Bgl I followed by annealing with <sup>32</sup>P-labeled plasmid DNA. Four of five clones screened contained the 2.4 kb internal fragment. The 0.3 kb fragment would not be detected on these gels. From the intensity of the 2.4 kb band in comparison with controls, we conclude that multiple copies of this fragment are present in these transformants. Other bands are observed which presumably represent the segments of pBR322 attached to cellular DNA.</p>
    <heading>Transformation of Mouse Cells with the Rabbit β-Globin Gene</heading> <p>Transformation with purified eucaryotic genes may provide a means for studying the expression of cloned genes in a heterologous host. Cotransformation experiments were therefore performed with the rabbits major globin gene which was isolated from a cloned library of rabbit chromosomal DNA (Maniatis, T., et al., Cell 15: 687-701 (1978). One β-globin clone designated RβG-1 consists of a 15 kb rabbit DNA fragment carried on the bacteriophage cloning vector Charon 4a. Intact DNA from this clone (RβG-1) was mixed with the viral tk DNA at a molar ratio of 100:1, and tk<sup>+</sup> transformants were isolated and examined for the presence of rabbit globin sequences. Cleavage of RβG-1 with the enzyme Kpn I generates a 4.7 kb fragment which contains the entire rabbit β-globin gene. This fragment was purified by gel electrophoresis and nick-translated to generate a probe for subsequent annealing experiments. The β-globin genes of mouse and rabbit are partially homologous, although we do not observe annealing of the rabbit β-globin probe with Kpn-cleaved mouse DNA under our experimental conditions. In contrast, cleavage of rabbit liver DNA with Kpn I generates the expected 4.7 kb globin band. Cleavage of transformed cell DNA with the enzyme Kpn I generates a 4.7 kb fragment containing globin-specific information in six of the eight tk<sup>+</sup> transformants examined. In two of the clones, additional rabbit globin bands are observed which probably result from the loss of at least one of the Kpn sites during transformation. The number of rabbit globin genes integrated in these transformants is variable. In comparison with controls, some clones contain a single copy of the gene, while others contain multiple copies of this heterologous gene. These results demonstrate that cloned eucaryotic genes can be introduced into cultured mammalian cells by cotransformation.</p>
    <heading>Transformation Competence is not Stably Inherited</heading> <p>Our data suggest the existence of a subpopulation of transformation-competent cells within the total cell population. If competence is a stably inherited trait, then cells selected for transformation should be better recipients in subsequent gene transfer experiments than their parental cells. Two results indicate that as in procaryotes, competence is not stably heritable. In the first series of experiments, a double mutant, Ltk aprt<sup>−</sup> (deficient in both tk and aprt), was transformed to either the tk<sup>+</sup> aprt<sup>−</sup> or the tk<sup>−</sup> aprt<sup>+</sup> phenotype using cellular DNA as donor. Wigler, M. et al., Cell 14: 725-731 (1978); Wigler, M. et al., PNAS 76: 5684-5688 (1979). These clones were then transformed to the tk<sup>+</sup> aprt<sup>+</sup> phenotype. The frequency of the second transformation was not significantly higher than the first. In another series of experiments, clones ΦX4 and ΦX5 were used as recipients for the transfer of a mutant folate reductase gene which renders recipient cells resistant to methotrexate (mtx). The cell line A29 Mtx<sup>RIII </sup>contains a mutation in the structural gene for dihydrofolate reductase, reducing the affinity of this enzyme for methotrexate. Flintoff, W. F. et al., Somatic Cell Genetic 2: 245-261 (1976). Genomic DNA from this line was used to transform clones ΦX4 and ΦX5 and Ltk<sup>−</sup> cells. The frequency of transformation to mtx resistance for the ΦX clones was identical to that observed with the parental Ltk<sup>−</sup> cells. It is therefore concluded that competence is not a stably heritable trait and may be a transient property of cells.</p>
    <heading>Discussion</heading> <p>In these studies, we have stably transformed mammalian cells with precisely defined procaryotic and eucaryotic genes for which no selective criteria exist. Our chosen design derives from studies of transformation in bacteria which indicate that a small but selectable subpopulation of cells is competent in transformation. Thomas, R. Biochim. Biophys. Acta 18: 467-481 (1955); Hotchkiss, R. PNAS 40: 49-55 (19591; Thomasz, A. and Hotchkiss R. PNAS 51: 480-487 (.1964); Spizizen, J. et al., Ann Rev. Microbiol. 20: 371-400 (1966). If this is also true for animal cells, then biochemical transformants will represent a subpopulation of competent cells which are likely to integrate other unlinked genes at frequencies higher than the general population. Thus, to identify transformants containing genes which provide no selectable trait, cultures were cotransformed with a physically unlinked gene which provided a selectable marker. This cotransformation system should allow the introduction and stable integration of virtually any defined gene into cultured cells. Ligation to either viral vectors or selectable biochemical markers is not required.</p>
    <p>Cotransformation experiments were performed using the HSV tk gene as the selectable biochemical marker. The addition of this purified tk gene to mouse cells lacking thymidine kinase results in the appearance of stable transformants which can be selected by their ability to grow in HAT. Tk<sup>+</sup> transformants were cloned and analyzed by blot hybridization for cotransfer of additional DNA sequences. In this manner, we have constructed mouse cell lines which contain multiple copies of ΦX, pBR322 and rabbit β-globin gene sequences.</p>
    <p>The suggestion that these observations could result from contaminating procaryotic cells in our cultures is highly improbable. At least one of the rabbit β-globin mouse transformants expresses polyadenylated rabbit β-globin RNA sequences as a discrete 9S cytoplasmic species. The elaborate processing events required to generate 9S globin RNA correctly are unlikely to occur in procarvotes.</p>
    <p>The ΦX cotransformants were studied in greatest detail. The frequency of cotransformation is high: 14 of 16 tk<sup>+</sup> transformants contain ΦX sequences. The ΦX sequences are integrated into high molecular weight nuclear DNA. The number of integration events varies from one to more than fifty in independent clones. The extent of the bacteriophage genome present within a given transformant is also variable; while some clones have lost up to half the genome, other clones contain over 90% of the ΦX sequences. Analysis of subclones demonstrates that the ΦX genotype is stable through many generations in culture. Similar conclusions are emerging from the characterization of the pBR322 and globin gene cotransformants.</p>
    <p>Hybridization analysis of restriction endonuclease-cleaved transformed cell DNA allows one to make some preliminary statements on the nature of the integration intermediate. Only two ΦX clones have been examined in detail. In both clones, the donor DNA was Pst I-linearized ΦX DNA. Attempts were made to distinguish between the integration of a linear or circular intermediate. If either precise circularization or the formation of linear concatamers had occurred at the Pst I cleavage site, and if integration occurred at random points along this DNA, one would expect cleavage maps of transformed cell. DNA to mirror the circular ΦX map. The bridge fragment, however, is not observed or is present in reduced amounts in digests of transformed cell DNA with three different restriction endonucleases. The fragments observed are in accord with a model in which ΦX DNA integrates as a linear molecule. Alternatively, it is possible that intramolecular recombination of ΦX DNA occurs, resulting in circularization with deletions at the Pst termini. Lai, C. J. and Nathans, D. Cold Spring Harbor Symp. Quant. Biol. 39: 53-60 (1974).</p>
    <p>Random integration of this circular molecule would generate a restriction map similar to that observed for clones ΦX4 and ΦX5. Other more complex models of events occurring before, during or after integration can also be considered. Although variable amounts of DNA may be deleted from termini during transformation, most copies of integrated ΦX sequences in clone ΦX4 retain the Hpa I site, which is only 30 bp from the Pst I cleavage site. Whatever the mode of integration, it appears that cells can be stably transformed with long stretches of donor DNA. Transformants have been observed containing continuous stretches of donor DNA 50 kb long.</p>
    <p>There have been attempts to identify cells transformed with ΦX sequences in the absence of selective pressure. Cultures were exposed to ΦX and tk DNA and cells were cloned under nonselective conditions. ΦX sequences were absent from all fifteen clones picked. In contrast, 14 of 16 clones selected for the tk<sup>+</sup> phenotype contained ΦX DNA. The simplest interpretation is that a subpopulation of cells within the culture is competent in the uptake and integration of DNA. In this, subpopulation of cells, two physically unlinked genes can be introduced into the same cell with high frequency. At present one can only speculate on the biological basis of competency. Competent cells may be genetic variants within the culture; however, these studies indicate that the competent phenotype is not stably inherited. If one can extrapolate from studies in procaryotes, the phenomenon of competence is likely to be a complex and transient property reflecting the metabolic state of the cell.</p>
    <p>Cotransformants contain at least one copy of the tk gene and variable amounts of ΦX DNA. Although transformation was performed with ΦX and tk sequences at a molar ratio of 1000:1, the sequence ratio observed in the transformants never exceeded 100:1. There may be an upper limit to the number of integration events that a cell can tolerate, beyond which lethal mutations occur. Alternatively, it is possible that the efficiency of transformation may depend upon the nature of the transforming fragment. The tk gene may therefore represent a more efficient transforming agent than phage DNA.</p>
    <p>In other studies there has been demonstrated the co-transfer of plasmid pBR322 DNA into Ltk<sup>−</sup> aprt<sup>−</sup> cells using aprt<sup>+</sup> cellular DNA as donor and aprt as selectable marker. Furthermore, the use of dominant acting mutant genes which can confer drug resistance will extend the host range for cotransformation to virtually any cultured cell.</p>
    <p>The stable transfer of ΦX DNA sequences to mammalian cells serves as a model system for the introduction of defined genes for which no selective criteria exist. The tk cotransformation system has been used to transform cells with the bacterial plasmid pBR322 and the cloned rabbit β-globin gene. Experiments which indicate that several of the pBR transformants contain an uninterrupted sequence which includes the replicative origin and the gene coding for ampicillin resistance (β-lactamase), suggest that DNA from pBR transformants may transfer ampicillin resistance to <i>E. coli. </i>Although preliminary, these studies indicate the potential value of cotransformation in the analysis of eucaryotic gene expression.</p>
    <heading>SECOND SERIES OF EXPERIMENTS</heading> <p>Cotransformed mouse fibroblasts containing the rabbit β-globin gene provide an opportunity to study the expression and subsequent processing of these sequences in a heterologous host. In these experiments, we demonstrate the expression of the transformed rabbit β-globin gene generating a discrete polyadenylated 9S species of globin RNA. This RNA results from correct processing of both intervening sequences, but lacks approximately 48 nucleotides present at the 5′ terminus. of mature rabbit β-globin mRNA.</p>
    <heading>Transformation of Mouse Cells with the Rabbit β-Globin Gene</heading> <p>We have performed cotransformation experiments with the chromosomal adult rabbit β-globin gene, using the purified herpes virus tk gene as a biochemical marker. The addition of the tk gene to mutant Ltk<sup>−</sup> mouse fibroblasts results in the appearance of stable transformants that be selected by their ability to grow in hypoxanthine/aminopterin/thymidine (HAT) medium. Cells were cotransformed with a-β-globin gene clone designated RβG1, which consists of a 15.5-kbp insert of rabbit DNA carried in the bacteriophage λcloning vector Charon 4A. The purified tk gene was mixed with a 100-fold molar excess of intact recombinant DNA from clone RβG1. This DNA was then exposed to mouse Ltk<sup>−</sup> cells under transformation conditions described herein under Methods and Materials. After 2 weeks in selective medium, tk<sup>+</sup> transformants were observed at a frequency of one colony per 10<sup>6 </sup>cells per 20 pg of tk gene. Clones were picked and grown into mass culture.</p>
    <p>It was then asked if the tk<sup>+</sup> transformants also contain rabbit β-globin sequences. High molecular weight DNA from eight transformants was cleaved with the restriction endonuclease Knp I. The DNA was fractionated by agarose gel electrophoresis and transferred to nitocellulose filters, and these filters were then annealed with nick-translated globin [32P] DNA blot hybridization. Southern, E. M., J. Mol. Biol. 98: 503-517 (1975). Cleavage of this recombinant phage with the enzyme Kpn I generates a 4.7-kpb fragment that contains the entire adult β-globin gene, along with 1.4 kbp of 5′ flanking information and 2.0 kbp of 3′ flanking information. This fragment was purified by gel electrophoresis and nick translated to generate a hybridization probe. Blot hybridization experiments showed that the 4.7-kbp Kpn I fragment containing the globin gene was present in the DNA of six of the eight tk<sup>+</sup> transformants. In three of the clones additional rabbit globin bands were observed, which probably resulted from the loss of at least one of the Kpn I sites during transformation. The number of rabbit globin genes integrated in these transformants was variable: some clones contained a single copy of the gene, whereas others contained up to 20 copies of the heterologous gene. It should be noted that the β-globin genes of mouse and rabbit are partially homologous. However, we do not observe hybridization of the rabbit β-globin probe to Kpn-cleaved mouse DNA, presumably because Kpn cleaveage of mouse DNA leaves the β-gene cluster in exceedingly high molecular weight fragments not readily detected in these experiments. These results demonstrate the introduction of the cloned chromosomal rabbit β-globin transfer.</p>
    <heading>Rabbit β-Globin Sequences are Transcribed in Mouse Transformants</heading> <p>The cotransformation system we have developed may provide a functional assay for cloned eucaryotic genes if these genes are expressed in the heterologous recipient cell. Six transformed cell clones were therefore analyzed for the presence of rabbit β-globin RNA sequences. In initial experiments, solution hybridization reactions were performed to determine the cellular concentration of rabbit globin transcripts in our transformants. A radioactive cDNA copy of purified rabbit α- and β-globin mRNA was annealed with the vast excess of cellular RNA. Because homology exists between the mouse and rabbit globin sequences, it was necessary to determine experimental conditions such that the rabbit globin cDNAs did not form stable hybrids with mouse globin mRNA but did react completely with homologous rabbit sequences. At 75° C. in the presence of 0.4 M NaCl, over 80% hybridization was observed with the rabbit globin mRNA, whereas the heterologous reaction with purified mouse globin mRNA did not exceed 10% hybridization. The R<sub>0</sub>t<sub>1/2 </sub>of the homologous hybridization reaction was 6×10<sup>−4</sup>, a value consistent with a complexity of 1250 nucleotides contributed by the α- plus β-globin sequences in our cDNA probe. Axel, R., et al., Cell 7: 247-254 (1976).</p>
    <p>This rabbit globin cDNA was used as a probe in hybridization reactions with total RNA isolated from six transformed cell lines. Total RNA from transformed clone 6 protected 44% of the rabbit cDNA at completion, the value expected if only β-gene transcripts were present. This reaction displayed pseudo-first-order kinetics with R<sub>0</sub>t<sub>1/2 </sub>of 2×10<sup>3</sup>. A second transformant reacted with an R<sub>0</sub>t<sub>1/2 </sub>of 8×10<sup>3</sup>. No significant hybridization was observed at R<sub>0</sub>ts≧10<sup>4 </sup>with total RNA preparations from the four additional transformants.</p>
    <p>We have characterized the RNA from clone 6 in greatest detail. RNA from this transformant was fractionated into nuclear and cytoplasmic populations to determine the intracellular localization of the rabbit globin RNA. The cytoplasmic RNA was further fractionated by oligo. (dT)-cellulose chromatography into poly (A)<sup>+</sup> and poly (A)<sup>−</sup> RNA. Poly (A)<sup>+</sup> cytoplasmic RNA from clone 6 hybridizes with the rabbit cDNA with an R<sub>0</sub>t<sub>1/2 </sub>of 25. This value is {fraction (1/80)}th of the R<sub>0</sub>t<sub>1/2 </sub>served with total cellular RNA, consistent with the observation that poly (A)<sup>+</sup> cytoplasmic RNA is 1-2% of the total RNA in a mouse cell. Hybridization is not detectable with either nuclear RNA or cytoplasmic poly (A)<sup>−</sup> RNA at R<sub>0</sub>t values of 1×10<sup>4 </sup>and 2×10<sup>4</sup>, respectively. The steady-state concentration of rabbit β-globin RNA present in our transformant can be calculated from the R<sub>0</sub>t<sub>1/2 </sub>to be about five copies per cell, with greater than 0% localized in the cytoplasm.</p>
    <p>Several independent experiments argue that the globin RNA detected derives from transcription of the rabbit DNA sequences present in this transformant: (i) cDNA was prepared from purified 9S mouse globin RNA. This cDNA does not hybridize with poly (A)<sup>+</sup> RNA from clone 6 at R<sub>0</sub>t values at which the reaction with rabbit globin cDNA is complete. (ii) Rabbit globin cDNA does not hybridine with total cellular RNA obtained with tk<sup>+</sup> globin transformants at R<sub>0</sub>t values exceeding 10<sup>4</sup>. (iii) The hybridization observed does not result from duplex formation with rabbit globin DNA possibly contaminating the RNA Preparations. Rabbit cDNA was annealed with total cellular RNA from clone 6, the reaction product was treated with S1 nucelase, and the duplex was subjected to equilibrium density centrifugation in cesium sulfate under conditions that separate DNA-RNA hybrids from duplex DNA. The Sl-resistant cDNA banded at a density of 1.54 g/ml, as expected for DNA-RNA hybrid structures. These data, along with the observation that globin RNA is polyadenylated, demonstrate that the hybridization observed with RNA preparations does not result from contaminating DNA sequences.</p>
    <heading>Characterization of Rabbit Globin Transcripts in Transformed Cells</heading> <p>In rabbit erythroblast nuclei, the β-globin gene sequences are detected as a 14S precursor RNA that reflects transcription of two intervening sequences that are subsequently removed from this molecule to generate a 9S messenger RNA. It was therefore of interest to determine whether the globin transcripts detected exist at a discrete 9S species, which is likely to reflect appropriate splicing of the rabbit gene transcript by the mouse fibroblast. Cytoplasmic poly (A)-containing RNA from clone 6 was electrophoresed on a methyl-mercury/agarose gel, Bailey, J. &amp; Davidson, N., Anal. Biochem. 70: 75-85 (1976), and transferred to diazotized cellulose paper. Alwine, J. C. et al., Proc. Natl. Acad. Sci. USA 74: 5340-5454 (1977). After transfer, the RNA on the filters was hybridized with DNA from the plasmid pβG1, which contains rabbit β-globin cDNA sequences. Maniatis, T., et al., Cell 8: 163-182 (1976). Using this <sup>32</sup>P-labeled probe, a discrete 9S species of RNA was observed in the cytoplasm of the transformant, which comigrated with rabbit globin mRNA isolated from rabbit erythroblasts. Hybridization to 9S RNA species was not observed in parallel lanes containing either purified mouse 9S globin RNA or poly (A)-containing cytoplasmic RNA from a tk<sup>+</sup> transformant containing no rabbit globin genes.</p>
    <p>In these experiments, it was not possible to detect the presence of a 14S precursor in nuclear RNA populations from the transformants. This is not surprising, because the levels expected in nuclear RNA, given the observed cytoplasmic concentration, are likely to be below the limits of detection of this techniques. The 5′ and 3′ boundaries of the rabbit globin sequences expressed in transformed fibroblasts along with the internal processing sites can be defined more accurately by hybridizing this RNA with cloned DNAs, followed by S<b>1</b> nuclease digestion and subsequent gel analysis of the DNA products. Berk, A. J. &amp; Sharp, P. A., Cell 12: 721-732 (1977). When β-globin mRNA from rabbit erythroid cells was hybridized with cDNA clone pβG1 under appropriate conditions, the entire 576-base pair insert of cDNA was protected from S<b>1</b> nuclease attack. When the cDNA clone. was hybridized with RNA from our transformant, surprisingly, a discrete DNA band was observed at 525 base pairs, but not at 576 base pairs. These results suggest that, in this transformant, rabbit globin RNA molecules are present that have a deletion in a portion of the globin mRNA sequence at the 5′ or 3′ terminal. To distinguish between these possibilities, DNA of the λ clone, RβG1, containing the chromosomal rabbit β- globin sequence hybridized with transformed fibroblast RNA. The hybrid formed was treated with S<b>1</b> nuclease, and the protected DNA fragments were analyzed by alkaline agarose gel electrophoresis and identified by Southern blotting procedures. Southern, E. M., J. Mol. Biol. 98: 503-517 (1975). Because the rabbit β-globin gene is interrupted by two intervening sequences, the hybridization of mature rabbit mRNA to RβG1 DNA generates three DNA fragments in this sort of analysis: a 146-base pair fragment spanning the 5′ terminus to the junction of the small intervening sequence, a 222-base pair internal fragment bridging the small and large intervening sequences, and a 221-base pair fragment spanning the 3′ junction of the large intervening sequence to the 3′ terminus of the mRNA molecule. When transformant RNA was analyzed in this fashion, a 222-base pair fragment was observed as well as an aberrant fragment of 100 base pairs but no 146-base pair fragment. Hybridization with a specific 5′ probe showed that the internal 222 base pair fragment was present. The sum of the protected lengths equaled the length of the DNA fragment protected by using the cDNA clone. Taken together, these results indicate that although the intervening sequences expressed in transformed mouse fibroblast are removed from the RNA transcripts precisely, the 5′ termini of the cytoplasmic transcripts observed do not contain about 48<sup>±</sup>5 nucleotides present in mature 9S RNA of rabbit erythroblasts.</p>
    <heading>DISCUSSION</heading> <p>In these studies, mouse cell lines have been constructed that contain the rabbit β-globin gene. The ability of the mouse fibroblast recipient to transcribe and process this heterologous gene has then been analyzed. Solution hybridization experiments in concert with RNA blotting techniques indicate that, in at least one transformed cell line, rabbit globin sequences are expressed in the cytoplasm as a polyadenylylated 9S species. Correct processing of the rabbit β-globin gene has also been observed in tk<sup>+</sup> mouse cell transformants in which the globin and tk plasmids have been ligated prior to transformation. Mantei, N., et al., Nature (London) 281: 40-46 (1970). Similar results have been obtained by using a viral vector to introduce the rabbit globin gene into monkey cells. Hamer, D. H. &amp; Leder, P., Nature (London), 281: 35-39 (1979); Mulligan, R. C., et al., Nature (London) 277: 108-114 (1979). Taken together, these results suggest that nonerythroid cells from heterologous species contain the enzymes necessary to correctly process the intervening sequences of a rabbit gene whose expression usually is restricted to erythroid cells.</p>
    <p>The level of expression of rabbit globin sequences in the transformant is low: five copies of globin RNA are present in the cytoplasm of each cell. The results indicate that the two intervening sequences present in the original globin transcript are processed and removed at loci indistinguishable from those observed in rabbit erythroid cells. Surprisingly, 45 nucleotides present at the 5′ terminus of mature rabbit mRNA are absent from the β-globin RNA sequence detected in the cytoplasm of the transformant examined. It is possible that incorrect initiation of transcription occurs about the globin gene in this mouse cell line. Alternatively, the globin sequences detected may result from transcription of a long precursor that ultimately must undergo 5′ processing to generate the mature 9S species. Incorrect processing at the 5′ terminus in the mouse fibroblast could be responsible for the results. At present, it is difficult to distinguish among these alternatives. Because the analysis is restricted to a single transformant, it is not known whether these observations are common to all transformants expressing the globin gene or reflect a rare, but interesting abberation. It should be noted, however, that in similar experiments by Weissman and his colleagues, Mantei, N., et al., Nature (London) 281: 40-46 (1979), at least a portion of the rabbit globin RNA molecules transcribed in-transformed mouse fibroblasts retain the correct 5′ terminus.</p>
    <p>Several alternative explanations can be offered for the expression of globin sequences in transformed fibroblasts. It is possible that constitutive synthesis of globin RNA occurs in cultured fibroblasts, Humphries, S., et al., Cell 7: 267-277 (1976), at levels five to six orders of magnitude below the level observed in erythroblasts. The introduction of 20 additional globin DNA templates may simply increase this constitutive transcription to the levels observed in the transformant. Alternatively, it is possible that the homologous globin gene is repressed by factors that are partially overcome by a gene dosage effect provided by the introduction of 20 additional globin genes. Finally, normal repression of the globin gene in a fibro-blast may depend upon the position of these sequences in the chromosome. At least some of the newly introduced genes are likely to reside at loci distant from the resident mouse globin genes. Some of these ectopic sites may support low level transcription. Present data do not permit one to distinguish among these and other alternatives.</p>
    <p>Although the number of rabbit globin genes within a given transformant remains stable for over a hundred generations of culture in hypoxanthine/aminopterin/thymidine, it has not been possible to prove that these sequences are covalently integrated into recipient cell DNA. In previous studies, however, it has been demonstrated that cotransformation of either ΦX174 or plasmid pBR322 results in the stable integration of these sequences into high molecular nuclear DNA. In the present study, the globin gene represents a small internal segment of the high molecular weight concatenated phage DNA used in the transformation. Analysis of integration sites covalently linked to donor DNA is therefore difficult. Preliminary studies using radioactive λ sequences as a probe in DNA blotting experiments indicate that, in some cell lines, a contiguous stretch of recombinant phage DNA with a minimum length of 50 kbp has been introduced.</p>
    <p>The presence of 9S globin RNA in the cytoplasm of transformants suggests that this RNA may be translated to give rabbit β-globin polypeptide. Attempts to detect this protein in cell lysates using a purified anti-rabbit β-globin antibody have thus far been unsuccessful. It is possible that the globin RNAs in the transformant are not translated or are translated with very low efficiency due to the absence of a functional ribosomal binding site. The cytoplasmic globin transcripts in the transformant lack about nucleotides of untranslated 5′ sequence, which includes 13 nucleotides known to interact with the 40S ribosomal subunit in nuclease protection studies. Efstratiadis, A., et al., Cell 10: 571-585 (1977); Legon, S., J. Mol. Biol. 106: 37-53 (1976). Even if translation did occur with normal efficiency, it is probable that the protein would exist at levels below the limits of detection of the immunologic assay due to the low level of globin RNA, and the observation that the half-life of β-globin in the absence of heme and globin may be less than 30 min. Mulligan, R. C., et al., Nature (London) 277: 108-114 (1979).</p>
    <p>These studies indicate the potential value of cotransformation systems in the analysis of eucaryotic gene expression. The introduction of wild-type genes along with native and in vitro-constructed mutant genes into cultured cells provides an assay for the functional significance of sequence organization. It is obvious from these studies that this analysis will be facilitated by the ability to extend the generality of cotransformation to recipient cell lines, such as murine erythroleukemia cells, that provide a more appropriate environment for the study of heterologous globin gene expression.</p>
    <heading>THIRD SERIES OF EXPERIMENTS</heading> <p>The cotransformation experiments involving transformation of mouse cells with rabbit β-globin and with plasmid pBR322 and ΦX-174 DNA were continued and extended with the following results.</p>
    <p>ΦX DNA was used in cotransformation experiments with the tk gene as the selectable marker. ΦX replicative form DNA was cleaved with Pst I, which recognizes a single site in the circular genome, Sanger, F. et al., Nature 265: 687-695 (1977). Purified tk gene (500 pg) was mixed with 1-10 μg of Pst-cleaved ΦX replicative form DNA. This DNA was then added to mouse Ltk<sup>−</sup> cells using the transformation conditions described herein and in Wigler, M., et al., Cell 16:777-785 (1979). After two weeks in selective medium (HAT), tk<sup>+</sup> transformants were observed at a frequency of one colony per 10<sup>6 </sup>cells per 20 pg of purified gene. Clones were picked and grown into mass culture.</p>
    <p>It was then asked whether tk<sup>+</sup> transformants contained ΦX DNA sequences. High molecular weight DNA from the transformants was cleaved with the restriction endonuclease Eco RI, which recognizes no sites in the ΦX genome. The DNA was fractionated by agarose gel electrophoresis and transferred to nitrocellulose filters, and these filters were then annealed with nick-translated <sup>32</sup>P-ΦX DNA (blot hybridization).</p>
    <p>These annealing experiments indicated that 15 of 16 transformants acquired bacteriophage sequences. Since the ΦX genome is not cut with the enzyme Eco RI, the number of bands observed reflects the minimum number of eucaryotic DNA fragments containing information homologous to ΦX. The clones contain variable amounts of ΦX sequences: 4 of the 15 positive clones reveal only a single annealing fragment while others reveal at least fifty ΦX-specific fragments.</p>
    <p>It should be noted that none of 15 clones picked at random from neutral medium, following exposure to tk and ΦX DNA, contain ΦX information. Transformation with ΦX therefore is restricted to a subpopulation of tk<sup>+</sup> transformants. The addition of a selectable marker therefore facilitates the identification of cotransformants.</p>
    <heading>Transformation of Mouse Cells with the Rabbit β-Globin Gene</heading> <p>Transformation with purified eucaryotic genes provides a means for studying the expression of cloned genes in a heterologous host. Cotransformation experiments were performed with the rabbit s major globin gene which was isolated from a cloned library of rabbit chromosomal DNA. One β-globin clone, designated R G-1 consists of a 15 kb rabbit DNA fragment carried on the bacteriophage λ cloning vector Charon 4A. Intact DNA from this clone (RβG-1) was mixed with the viral tk DNA at a molar ratio of 100:1, and tk<sup>+</sup> transformants were isolated and examined for the presence of rabbit globin sequences. Cleavage of RβG-1 with the enzyme Kpn I generates a 4.7 kb fragment which contains the entire rabbit β-globin gene. This fragment was purified by gel electrophoresis and nick-translated to generate a probe for subsequent annealing experiments. The β-globin genes of mouse and rabbit are partially homologous, although we do not observe annealing of the rabbit β-globin probe with Kpn-cleaved mouse DNA, presumably because Kpn generates very large globin-specific fragments. In contrast, cleavage of rabbit liver DNA with Kpn I generates the expected 4.7 kb globin band. Cleavage of transformed cell DNA with the enzyme Kpn I generates a 4.7 kb fragment containing globin-specific information in six of the eight tk<sup>+</sup> transformants examined. The number of rabbit globin genes present in these transformants is variable. In comparison with controls, some of the clones contain a single copy of the gene, while others may contain as many as 20 copies of this heterologous gene.</p>
    <heading>Rabbit β-Globin Sequences are Transcribed in Mouse Transformants</heading> <p>The cotransformation system developed provides a functional assay for cloned eucaryotic genes if these genes are expressed in the heterologous recipient cell. Six transformed cell clones were analyzed for the presence of rabbit β-globin RNA sequences. Initial experiments, solution hybridization reactions were performed to determine the cellular concentration of rabbit globin transcripts in transformants.</p>
    <p>A radioactive cDNA copy of purified rabbit α and β-globin mRNA was annealed with a vast excess of total cellular RNA from transformants under experimental conditions such that rabbit globin cDNA does not form a stable hybrid with mouse sequences. Total RNA from transformed clone 6 protects 44% of the rabbit cDNA at completion, the value expected if only β gene transcripts are present. This reaction displays pseudo-first-order kinetics with an R<sub>0</sub>t<sub>1/2 </sub>of 2×10<sup>3</sup>. A second transformant (clone 2) reacts with an R<sub>0</sub>t<sub>1/2 </sub>of 8×10<sup>3</sup>. No significant hybridization was observed with total RNA preparations from our other transformants. Further analysis of clone 6 demonstrates that virtually all of the rabbit β-globin RNA detected in this transformant is polyadenylated and exists at a steady state concentration of about five copies per cell with greater then 90% of the sequences localized in the cytoplasm.</p>
    <heading>Globin Sequences Exist as a Discrete 9S Species in Transformed Cells</heading> <p>In rabbit erythroblast nuclei, the β-globin gene sequences are detected as a 14S precursor RNA which reflects transcription of two intervening sequences which are subsequently spliced from this molecule to generate a 9S messenger RNA. Our solution hybridization experiments only indicate that polyadenylated rabbit globin RNA sequences are present in the mouse transformant. It was therefore of interest to determine whether the globin transcripts we detected exist as a discrete 9S species, which is likely to reflect appropriate splicing of the rabbit gene transcript by the mouse fibroblast. Cytoplasmic poly A-containing RNA from clone 6 was denatured by treatment with 6M urea at 70° C., and electrophoresed on a 1% acid-urea-agarose gel and transferred to diazotized cellulose paper. Following transfer, the RNA filters were hybridized with DNA from the plasmid RβG-1 containing rabbit β-globin cDNA sequences. Using this <sup>32</sup>P-labeled probe, a discrete 9S species of cytoplasmic RNA is seen which co-migrates with rabbit globin mRNA isolated from rabbit erythroblasts. Hybridization to 9S RNA species is not observed in parallel lanes containing either purified mouse 9S globin RNA or polyadenylated cytoplasmic RNA from a tk<sup>+</sup> transformant containing no rabbit globin genes.</p>
    <p>One is unable in these experiments to detect the presence of a 14S precursor in nuclear RNA populations from the transformant. This is not surprising, since the levels expected in nuclear RNA, given the observed cytoplasmic concentration, are likely to be below the limits of detection of this technique. Nevertheless, the results with cytoplasmic RNA strongly suggest that the mouse fibroblast is capable of processing a transcript of the rabbit β-globin gene to generate a 9S polyadenylated species which is indistinguishable from the β-globin mRNA in rabbit erythroblasts.</p>
    <heading>Rescue of pBR 322 DNA from Transformed Mouse Cells</heading> <p>Observations on cotransformation were extended to the EK-2 approved bacterial vector, plasmid pBR 322. Using the cotransformation scheme outlined herein, cell lines were constructed containing multiple copies of the pBR 322 genome. Blot hybridization analyses indicate that the pBR 322 sequences integrate into cellular DNA without significant loss of plasmid DNA. pBR 322 DNA linearized with either Hind III or Bam HI, which destroys the tetracycline resistance gene, integrates into mouse DNA with retention of both the plasmid replication origin and the ampicillin resistance (β-lactamase) gene. It was therefore asked whether these plasmid sequences could be rescued from the mouse genome by a second transformation of bacterial cells.</p>
    <p>The experimental approach chosen is outlined in FIG. <b>2</b>. Linearized pBR 322 DNA is introduced into mouse Ltk<sup>−</sup> cells via cotransformation using the tk gene as a selectable marker. DNA is isolated from transformants and screened for the presence of pBR 322 sequences. Since the donor plasmid is linearized, interrupting the tetracycline resistant gene, transformed cell DNA contains a linear stretch of plasmid DNA consisting of the replication origin and the β-lactamase gene covalently linked to mouse cellular DNA. This DNA is cleaved with an enzyme such as Xho I, which does not digest the plasmid genome. The resulting fragments are circularized at low DNA concentrations in the presence of ligase. Circular molecules containing plasmid DNA are selected from the vast excess of eucaryotic circles by transformation of <i>E. coli </i>strain x1776.</p>
    <p>This series of experiments has been carried out and a recombinant plasmid isolated from transformed mouse cell DNA which displays the following properties: 1) The rescued plasmid is ampicillin resistant, but tetracycline sensitive consistent with the fact that the donor pBR 322 was linearized by cleavage within the tetracycline resistance gene. 2) The rescued plasmid is 1.9 kb larger than pBR 322 and therefore contains additional DNA. 3) The rescued plasmid anneals to a single band in blot hybridizations to Eco RI-cleaved mouse liver DNA, suggesting that the plasmid contains an insert of single copy mouse DNA. These observations demonstrate that bacterial plasmids stably integrated into the mouse genome via transformation, can be rescued from this unnatural environment, and retain their ability to function in bacterial hosts.</p>
    <p>This result immediately suggests modified schemes utilizing plasmid rescue to isolate virtually any cellular gene for which selective growth criteria are available. The aprt gene of the chicken is not cleaved by Hind III or Xho I and transformation of aprt<sup>−</sup> mouse cells with cellular DNA digested with these enzymes results in the generation of aprt<sup>+</sup> colonies which express the chicken aprt gene. Ligation of Hind III cleaved chicken DNA with Hind III cleaved pBR 322 results in the formation of hybrid DNA molecules, in which the aprt gene is now adjacent to plasmid sequences. Transformation of aprt<sup>−</sup> cells is now performed with this DNA. Transformants should contain the aprt gene covalently linked to pBR 322, integrated into the mouse genome. This transformed cell DNA is now treated with an enzyme which does not cleave either pBR 322 or the aprt gene, and the resultant fragments are circularized with ligase. Transformation of <i>E. coli </i>with these circular molecules should select for plasmid sequences from eucaryotic DNA and enormously enrich for chicken aprt sequences. This double selection technique permits the isolation of genes expressed at low levels in eucaryotic cells, for which hybridization probes are not readily obtained.</p>
    <heading>Discussion</heading> <p>The frequency with which DNA is stably introduced into competent cells is high. Furthermore, the cotransformed sequences appear to be integrated into high molecular weight nuclear DNA. The number of integration events varies from one to greater than fifty in independent transformed clones. At present, precise statements cannot be made concerning the nature of the integration intermediate. Although data with ΦX are in accord with the model in which ΦX DNA integrates as a linear molecule, it is possible that more complete intramolecular recombination events generating circular intermediates may have occurred prior to or during the integration process. Whatever the mode of integration, it appears that cells can be stably transformed with long stretches of donor DNA. It has been observed that transformants contain contiguous stretches of donor DNA 50 kb long. Furthermore, the frequency of competent cells in culture is also high. At least one percent of the mouse Ltk<sup>−</sup> cell recipients can be transformed to the tk<sup>+</sup> phenotype. Although the frequency of transformation in nature is not known, this process could have profound physiologic and evolutionary consequences.</p>
    <p>The introduction of cloned eucaryotic genes into animal cells provides an in vivo system to study the functional significance of various features of DNA sequence organization. In these studies, stable mouse cell lines have been constructed which contain up to 20 copies of the rabbit β-globin gene. The ability of the mouse fibroblast recipient to transcribe and process this heterologous gene has been analyzed. Solution hybridization experiments in concert with RNA blotting techniques indicate that in at least one transformed cell line, rabbit globin sequences are expressed in the cytoplasm as a 9S species indistinguishable from the mature messenger RNA of rabbit erythroblasts. These results suggest that the mouse fibroblast contains the enzymes necessary to transcribe and correctly process a rabbit gene whose expression is normally restricted to erythroid cells. Similar observations have been made by others using a viral vector to introduce the rabbit globin gene into monkey cells.</p>
    <p>These studies indicate the potential value of cotransformation systems in the analysis of eucaryotic gene expression. The introduction of wild type genes along with native and in vitro constructed mutant genes into cultured cells provides an assay for the functional significance of sequence organization. It is obvious from these studies that this analysis will be facilitated by the ability to extend the generality of cotransformation to recipient cell lines, such as murine erythroleukemia cells, which may provide a more appropriate environment for the study of heterologous globin gene expression.</p>
    <heading>FOURTH SERIES OF EXPERIMENTS</heading> <p>The ability to transfer purified genes into cultured cells provides the unique opportunity to study the function and physical state of exogenous genes in the transformed host. The development of a system for DNA-mediated transfer of the HSV thymidine kinase (tk) gene to mutant mouse cells, Wigler., M.,, et al., Cell 11:223-232 (1977), has permitted extension of these studies to unique cellular genes. Wigler, M., et al., Cell 14:725-731 (1979). It has been found that high molecular weight DNA obtained from tk<sup>+</sup> tissues and cultured cells from a variety of eucaryotic organisms can be used to transfer tk activity to mutant mouse cells deficient in this enzyme. The generality of the transformation process has been demonstrated by the successful transfer of the cellular adenine phosphoribosyl transferase (aprt) gene and the hypoxanthine phosphoribosyl transferase (hprt) gene. Wigler, M., et al., Proc. Nat. Acad. Sci. USA 76: 1373-1376 (1979); Willicke, K., et al., Molec. Gen. Genet. 170:179-185 (1979); Graf, L. Y., et al., Somatic Cell Genetics, in press (1979).</p>
    <p>More recently, it has been demonstrated that cells transformed with genes coding for selectable biochemical markers also integrate other physically unlinked DNA fragments at high frequency. In this manner, the tk gene has been used as a marker to identify mammalian cells cotransformed with defined procaryotic and eucaryotic genes into cultured mammalian cells. Wigler, M., et al., Cell 16:777-785 (1979).</p>
    <p>Detection of gene transfer has in the past relied extensively on the use of appropriate mutant cell lines. In some cases, cells resistant to metabolic inhibitors contain dominant acting mutant genes. Cotransformation with such dominant acting markers should in principle permit the introduction of virtually any cloned genetic element into wild type cultured cells. In this study, cells were transformed with the gene coding for a mutant dihydrofolate reductase (dhfr) gene which renders cells resistant to high concentrations of methotrexate (mtx). Flintoff, W. F., et al., Cell 2:245-262 (1976).</p>
    <p>Cultured mammalian cells are exquisitely sensitive to the folate antagonist, methotrexate. Mtx resistant cell lines have been identified which fall into three categories: 1) cells with decreased transport of this drug. Fischer, G. A. Biochem. Pharmacol. 11:1233-1237 (1962); Sirotnak, F. M., et al., Cancer Res. 28:75-80 (1968); 2) cells with structural mutations which lower the affinity of dhfr for methotrexate. Flintoff, W. F., et al., Cell 2:245-262 (1976); and 3) cells which produce inordinately high levels of dhfr. Biedler, J. L., et al., Cancer Res. 32: 153-161 (1972); Chang, S. E., and Littlefield, J. W., Cell 7:391-396 (1976). Where they have been examined, cells producing high levels of dhfr have been found to contain elevated levels of the dhfr gene (gene amplification). Schimke, R. T., et al, Science 202:1051-1055 (1978).</p>
    <p>An interesting methotrexate resistant variant cell line (A29) has been identified that synthesizes elevated levels of a mutant dihydrofolate reductase with reduced affinity for methotrexate. Wigler, M., et al., Cell 16:777-785 (1979). Genomic DNA from this cell line has been used as donor in experiments to transfer the mutant dhfr gene to mtx sensitive cells. Exposure of mtx resistant transformed cells to increasing levels of mtx selects for cells which have amplified the transferred gene. In this way, it is possible to transfer and amplify virtually any genetic element in cultured mammalian cells.</p>
    <heading>Transfer of the Mutant Hamster Dihydrofolate Reductase Gene to Mouse Cells</heading> <p>High molecular weight cellular DNA was prepared from wild-type mtx sensitive CHO cells and from A29 cells, an mtx resistant CHO derivative synthesizing increased levels of a mutant dhfr. Flintoff, W. F., et al., Cell 2:245-262 (19761. The ability of these DNA preparations to transfer either the dhfr gene or the tk gene to tk<sup>−</sup> mouse L cells (Ltk<sup>−</sup> aprt<sup>−</sup>) was tested using a modification of the calcium phosphate coprecipitation method. Wigler, M.,. et al., Proc. Nat. Acad. Sci. USA 76:1373-1376 (19791. DNA from both mutant A29 and wild-type CHO cells was competent in transferring the tk<sup>−</sup> gene to Ltk<sup>−</sup> aprt<sup>−</sup> cells. Methotrexate resistant colonies were observed only following treatment of cells with DNA from A29. The data obtained suggest that treatment of methotrexate sensitive cells with A29 DNA resulted in the transfer and expression of a mutant dhfr gene, thus rendering these cells insensitive to elevated levels of methotrexate.</p>
    <p>In order to test this hypothesis directly, molecular hybridization studies were performed to demonstrate the presence of the hamster dhfr gene in DNA from presumed transformants. A mouse dhfr cDNA clone (pdfr-21), Chang, A. C. Y., et al., Nature 275: 617-624 (19781, that shares homology with the structural gene sequences of the hamster dhfr gene was used to detect the presence of this gene in our transformants. Restriction analysis of the dhfr gene from A29, from presumed transformants, and from amplified mouse cells, was performed by blot hybridization. Southern, E. M., J. Mol. Biol. 98:503-517 (19751. DNA was cleaved with restriction endonuclease Hind III, electrophoresed in agarose gels, and transferred to nitrocellulose filters. These filters were then hybridized with high specific activity, <sup>32</sup>P-labeled nick-translated pdhfr-21 and developed by autoradiography. This procedure visualizes restriction fragments of genomic DNA homologous to the dhfr probe. Prominent bands are observed at 15 kb, 3.5 kb and 3 kb for mouse DNA and 17 kb, 7.9 kb, 3.7 kb and 1.4 kb for hamster DNA. The restriction profiles between these two species are sufficiently different to permit one to distinguish the hamster gene in the presence of an endogenous mouse gene. Five L cell transformants resistant to methotrexate were therefore examined by blot hybridization. In each transformed cell line, one observed the expected profile of bands resulting from cleavage of the endogenous mouse dhfr gene and a series of additional bands whose molecular weights are identical to those observed upon cleavage of hamster DNA. The 17.9 kb, 7.9 kb and 1.4 kb bands observed in hamster DNA are diagnostic for the presence of the hamster dhfr gene and are present in all transformants.</p>
    <p>In initial experiments, the lowest concentration of methotrexate (0.1 μg per ml). was chosen which would decrease survival of Ltk<sup>−</sup> aprt<sup>−</sup> cells to less than 10<sup>−</sup>. Previous studies, Flintoff, W. F., et al., Cell 2:245-262 (1976), suggested that the presence of a single mutant dhfr gene can render cells resistant to this concentration of methotrexate. Comparison of the intensity of the hamster dhfr gene fragments of transformed cell DNA with those of wild-type hamster DNA suggest that our transformants contain one or at most a few methotrexate resistant hamster genes. By contrast, donor A29 cells, which have been shown to produce elevated levels of the mutant dhfr, Flintoff, W. F., et al., Cell 2:245-262 (1976), appear to contain multiple copies of this gene.</p>
    <heading>Amplification of the Transferred dhfr Gene</heading> <p>Initial transformants were selected for resistance to relatively low levels of mtx (0.1 μg/ml). For every clone, however, it was possible to select cells resistant to elevated levels of mtx by exposing mass cultures to successively increasing concentrations of this drug. In this manner, we isolated cultures resistant to up to 40 μg/ml of methotrexate starting from clones that were initially resistant to 0.μg/ml. We next asked if increased resistance to methotrexate in these transformants was associated with amplification of a dhfr gene and, if so, whether the endogenous mouse or the newly transferred hamster gene was amplified. DNA from four independent isolates and their resistant derivatives was examined by blot hybridization. In each instance, enhanced resistance to methotrexate was accompanied by an increase in the copy number of the hamster gene. This is most readily seen by comparing the intensities of the 1.5 kb band. In no instance have we detected amplification of the endogenous mouse dhfr gene. Lastly, it is noted that not all lines selected at equivalent methotrexate concentrations appear to have the same dhfr gene copy number.</p>
    <heading>The dhfr Gene as a Generalized Transformation Vector</heading> <p>Selectable genes can be used as vectors for the introduction of other genetic elements into cultured cells. In previous studies, it has been demonstrated that cells transformed with the tk gene are likely to incorporate other unlinked genes. Wigler, M., et al., Cell 16:777-785 (1979). The generality of this approach was tested for the selectable marker, the mutant dhfr gene. 20 μg of total cellular DNA from A29 was mixed with 1 μg of Hind III-linearized pBR 322 DNA. Recipient cells were exposed to this DNA mixture and, after two weeks, methotrexate resistant colonies were picked. Genomic DNA from transformants was isolated, cleaved with Hind III and analyzed for the presence of pBR322 sequences. Two independent isolates were examined in this way and in both cases multiple copies of pBR322 sequences were present in these methotrexate transformants.</p>
    <p>An alternate approach to generalized transformation involves ligation of a nonselectable DNA sequence to a selectable gene. Since the muant dhfr gene is a dominant acting drug resistance factor, this gene is an ideal vector. Furthermore, it should be possible to amplify any genetic element ligated to this vector by selecting cells resistant to elevated levels of mtx. To explore this possibility, restriction endonucleases that do not destroy the dhfr gene of A29 were identified by transformation assay. One such restriction endonuclease, Sal I, does not destroy the transformation potential of A29 DNA. Sal I-cleaved A29 DNA was therefore ligated to an equal mass of Sal I-linearized pBR322. This ligation product was subsequently used in transformation experiments. Methotrexate resistant colonies were picked and grown into mass culture at 0.1 μg methotrexate/ml. Mass cultures were subsequently exposed to increasing concentrations of methotrexate.</p>
    <p>DNAs were obtained from mass cultures resistant to 0.1, 2, 10 and 40 μg/ml methotrexate, and the copy number of pBR322 and dhfr sequences was determined by blot hybridization. Six independent transformed lines were examined in this fashion. Five of these lines exhibited multiple bands homologous to pBR322 sequences. In four of these transformed clones, at least one of the pBR 322-specific bands increased in intensity upon amplification of dhfr. In SS-1, two pBR322-specific bands are observed in DNA from cells resistant to 0.1 μg/ml methotrexate. These bands increase several-fold in intensity in cells resistant to 2 μg/ml. No further increase in intensity is observed, however, in cells selected for resistance to 40 μg/ml. In a second line, SS-6, all pBR 322 bands present at 0.1 μg/ml continue to. increase in intensity as cells are selected first at 2 μg/ml and then at 40 μg/ml methotrexate. Curiously, new pBR322-specific bands appear after selection at higher methotrexate concentrations. It was estimated that there is at least a fifty-fold increase in copy number for pBR322 sequences in this cell line. In a third cell line, HH-1, two pBR322-specific bands increase in intensity upon amplification, others remain constant or decrease in intensity. Thus, the pattern of amplification of pBR322 sequences observed in these cells can be quite varied. Nevertheless, it appears that the mutant dhfr gene can be used as vector for the introduction and amplification of defined DNA sequences into cultured animal cells.</p>
    <heading>Discussion</heading> <p>The potential usefulness of DNA-mediated transformation in the study of eucaryotic gene expression depends to a large extent on its generality. Cellular genes coding for selectable biochemical functions have previously been introducted into mutant cultured cells, Wigler, M., et al., Cell 14:725-731 (1979); Wigler, M., et al., Proc. Nat. Acad. Sci. USA 76:1373-1376 (1979); Willecke, K., et al., Molec. Gen. Genet. 170:179-185 (19791); Graf, L. H., et al., Somatic Cell Genetics, in press (1979). In the present study, a dominant acting, methotrexate resistant dhfr gene has been transferred to wild-type cultured cells. The use of this gene as a vector in cotransformation systems may now permit the introduction of virtually any genetic element into a host of new cellular environments.</p>
    <p>In initial experiments, DNA from A29 cells, a methotrexate resistant CHO derivative synthesizing a mutant dhfr was added to cultures of mouse L cells. Methotrexate resistant colonies appeared at a frequency of one to ten colonies/5×10<sup>5 </sup>cells/20 μg cellular DNA. No colonies were observed upon transformation with DNA obtained from wild-type, methotrexate sensitive cells, although this DNA was a competent donor of the thymidine kinase gene. Definitive evidence that we have effected transfer of a mutant hamster dhfr gene was obtained by demonstrating the presence of the hamster gene in mouse transformants. The restriction maps of the mouse and hamster dhfr genes are significantly different and permit one to distinguish these genes in blot hybridization experiments. In all transformants examined, one observes two sets of restriction fragments homologous to a mouse dhfr cDNA clone: a series of bands characteristic of the endogenous mouse gene and a second series characteristic of the donor hamster gene.</p>
    <p>The utility of transformation of the dhfr locus is a function of the relative frequencies both of transformation and of spontaneous resistance to mtx. The demonstration that all mtx resistant L cells picked result from transformation rather than amplification of endogenous genes suggests that amplification of dhfr is a rare event in this cell line. Attempts were made to transform other cell lines, including mouse teratoma and rat liver cells and, in these instances, hybridization studies reveal that the acquisition of mtx resistance results from amplification of endogenous dhfr genes. The use of a purified dhfr gene is likely to overcome these difficulties by enormously increasing the frequency of transformation.</p>
    <p>The dhfr copy number observed in initial transformants is low. This observation is consistent with previous studies suggesting that a single mutant dhfr gene is capable of rendering cells mtx resistant under selective criteria (0.1 μg/ml mtx). Flintoff, W. F., et al., Cell 2: 245-262 (1976). Exposure of these initial mtx resistant transformants to stepwise increases in drug concentration results in the selection of cells with enhanced mtx resistance resulting from amplification of the newly, transferred mutant hamster dhfr gene. In no transformants has amplification of the endogenous mouse gene been observed in response to selective pressure. It is likely that a single mutant gene affords significantly greater resistance to a given concentration of mtx than a single wild-type gene. If the frequency of the amplification is low, one is merely selecting resistance variants having the minimum number of amplification events. It is also possible that newly transferred genes may be amplified more readily than endogenous genes.</p>
    <p>The mutant dhfr gene has been used as a dominant transfer vector to introduce nonselectable genetic elements into cultured cells. One experimental approach exploits the observation made previously, Wigler, M., et al.; Cell 16: 777-785 (1979), that competent cells integrate other physically unlinked genes at high frequency. Cultures exposed to pBR322 DNA, along with the genomic DNA containing the mutant dhfr gene give rise to mtx resistant cell lines containing multiple copies of the bacterial plasmid.</p>
    <p>An alternative approach to genetic vectoring involves ligation of pBR322 sequences to the selectable dhfr gene prior to transformations. This procedure also generates transformants containing multiple pBR322 sequences. Amplification of dhfr genes results in amplification of of pBR322 sequences, but the patterns of amplification differ among cell lines. In one instance, all pBR322 sequences amplify with increasing mtx concentrations. In other lines, only a subset of the sequences amplify.</p>
    <p>In yet other lines, sequences appear to have been lost or rearranged. In some lines, amplification proceeds. with increasing mtx concentrations up to 40 μg/ml, whereas in others, amplification ceases at 2 μg/ml. At present, the amplification process is not understood nor has the amplification unit been defined. Whatever the mechanisms responsible for these complex events, it is apparent that they can be expolited to control the dosage of virtually any gene introduced into cultured cells.</p>
    <heading>FIFTH SERIES OF EXPERIMENTS</heading> <p>Mouse teratocarcinoma (TCC) stem cells provide a unique vector for the introduction of specific, predetermined, genetic changes into mice.Mintz, B. &amp; Illmensee, K., Proc. Natl. Acad. Sci. 72: 3585-3589 (1975); Mintz, B., Brookhaven Symp. Biol. 29:. 82-85 (1977). These cells lose their neoplastic properties and undergo normal differentiation when placed in the environment of the early embryo. There they can contribute to formation of all somatic tissues in a mosaic animal comprising both donor and host-derived cells, and also to the germ line, from which the progeny have genes of the tumor strain in all their cells. Thus, during initial propagation of TCC stem cells in culture, clones with experimentally selected nuclear, Dewey, M. J., et al., Proc.Natl. Acad. Sci., 74 : 5564-368 (1977), and cytoplasmic, Watanabe, T., et al., Proc. Natl. Acad. Sci., 75: 5113-5117 (1978), gene mutations have been obtained and the cells have proved capable of participating in embryogenesis.</p>
    <p>The effective application of this system in probing the control of gene expression during differentiation would be greatly enhanced if, as proposed, Mintz, B., Differentiation 13: 25-27 (1979), precisely defined genes, either in native or modified form, with known associated sequences, could be introduced into developmentally totipotent TCC cells prior to their development in vivo. DNA-mediated gene transfer into cultured mouse cells has now been reported for a variety of viral and cellular genes coding for selectable biochemical functions., The purified viral thymidine kinase (tk; ATP: thymidine 5′-phosphotransferase, EC 2.7.1.21) gene has provided a model system for gene transfer, Wigler, M. et al., Cell 11: 223-232 (1977), and has been followed by the DNA-mediated transfer of the cellular genes coding for thymidine kinase, Wigler, M., et al., Cell 14: 725-731 (1978), hypoxanthine phosphoribosyltransferase, Willecke, K., et al., Molec. Gen. Genet. 170: 179-185 (1979); Graf, L. H., et al., Somat. Cell Genet., in press (1979), adenine phosphoribosyltransferase, Wigler, M., et al., Proc. Natl. Acad.Sci. USA, 76:.1373-1376 (1979), and dihydrofolate reductase, Wigler, M., et al., Proc. Natl. Acad. Sci, in press (1980); Lewis, W. H., et al., Somat. Cell. Genet., in press (1979). In this report is d monstrated the contransfer of the cloned Herpes simplex (HSV) thymidine kinase gene along with the human β-globin gene into mutant (tk<sup>−</sup>) teratocarcinoma stem cells in culture. These transformed cells, when tested by subcutaneous inoculation into mice, retain their developmental capacities in the tumors that are produced, and exhibit the viral-specific tk enzymatic activity for numerous cell generations in vivo.</p>
    <heading>Transformation of tk<sup>−</sup> Teratocarcinoma Cells.</heading> <p>The addition of plasmid DNA containing the HSV thymidine kinase gene to cultures of attached mouse L tk<sup>−</sup> cells yields L tk<sup>+</sup> transformants in HAT at a frequency of one colony per 100 pg of DNA per 5×10<sup>5 </sup>cells. Under identical transformation procedures,, tk<sup>−</sup> teratocarcinoma cells showed a strikingly lower transformation efficiency. Based on the average of three independent experiments, one surviving colony was obtained per 4 μg of plasmid DNA per 5×10<sup>5 </sup>cells, a value four to five orders of magnitude below that of the L tk<sup>−</sup> cells. This relatively low efficiency was confirmed when the DNA was added to TCC tk<sup>−</sup> cells in suspension. Addition of 10 μg of Bam H1-restricted ptk-1 DNA to 7×10<sup>6 </sup>cells resulted in only four transformants in HAT. With identical transformation conditions, L tk<sup>−</sup> cells gave 3×10<sup>3 </sup>tk<sup>+</sup> colonies per 10<sup>7 </sup>cells per 1.5 μg of ptk-1 DNA. While high concentrations of gene are thus required to effect transformation in this teratocarcinoma cell line, the availability of cloned DNA nonetheless allows numerous tk<sup>+</sup> transformants to be obtained.</p>
    <heading>Expression of HSV tk Activity in Transformed Teratocarcinoma Cells</heading> <p>To ascertain whether the tk<sup>+</sup> phenotypes of the TCC clones were indeed attributable to expression of the viral tk gene, seven colonies were picked from independent culture dishes and grown into mass cultures for testing. The activity of five clones were characterized by serological, and of two by biochemical, techniques. The Herpes-type antigenic identity of tk was verified by assaying the ability of HSV- tk-specific antibody to neutralize enzymatic activity. Over 90% inhibition of tk activity was in fact observed when immune serum was reacted with extracts of each of the five transformed clones chosen (Table I). The low residual activity remaining after neutralization of transformed-cell extracts may represent mitochondrial tk activity, which by itself is unable to afford survival in HAT. Cell extracts from the other two TCC tk<sup>+</sup> clones chosen were tested for tk electrophoretic mobility because of the marked difference between the mouse and HSV tk enzymes. While the TCC tk<sup>−</sup> control, as expected, shows no major peak of activity, the transformants have the HSV tk characteristic peak migrating wit an R<sub>f </sub>of 0.45, as shown for one of the clones.</p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 1</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Specific neutralization of Herpes thymidine kinase</td>
              </tr> <tr class="description-tr"> <td class="description-td">in transformants</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="5"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="35pt" align="left"> </colspec> <colspec colname="2" colwidth="70pt" align="center"> </colspec> <colspec colname="3" colwidth="91pt" align="center"> </colspec> <colspec colname="4" colwidth="7pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">Activity with</td>
                <td class="description-td">Activity with</td>
                <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Cell line</td>
                <td class="description-td">preimmune serum</td>
                <td class="description-td">antiserum</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="5"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="35pt" align="left"> </colspec> <colspec colname="2" colwidth="70pt" align="center"> </colspec> <colspec colname="3" colwidth="42pt" align="center"> </colspec> <colspec colname="4" colwidth="56pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">source of</td>
                <td class="description-td">Units × 10<sup>−3</sup> </td>
                <td class="description-td">Units × 10<sup>−3</sup> </td>
                <td class="description-td">% Residual</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">extract</td>
                <td class="description-td">per ml</td>
                <td class="description-td">per ml</td>
                <td class="description-td">activity</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="OFFSET" nameend="4" align="center" rowsep="1" class="description-td" colspan="5"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="5"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="35pt" align="left"> </colspec> <colspec colname="2" colwidth="70pt" align="char" char="."> </colspec> <colspec colname="3" colwidth="42pt" align="char" char="."> </colspec> <colspec colname="4" colwidth="56pt" align="char" char="."> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">TCC wt*</td>
                <td class="description-td">2.8</td>
                <td class="description-td">3.0</td>
                <td class="description-td">107.0</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">TCC tk<sup>−†</sup> </td>
                <td class="description-td">0.05</td>
                <td class="description-td">0.06</td>
                <td class="description-td">100.0</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">LHB 2b‡</td>
                <td class="description-td">3.4</td>
                <td class="description-td">0.06</td>
                <td class="description-td">2.0</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">TCC tk-1§</td>
                <td class="description-td">2.1</td>
                <td class="description-td">0.17</td>
                <td class="description-td">8.0</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">TCC tk-3</td>
                <td class="description-td">5.5</td>
                <td class="description-td">0.43</td>
                <td class="description-td">8.0</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">TCC tk-4</td>
                <td class="description-td">6.1</td>
                <td class="description-td">0.15</td>
                <td class="description-td">2.5</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">TCC tk-5</td>
                <td class="description-td">3.7</td>
                <td class="description-td">0.21</td>
                <td class="description-td">6.0</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="OFFSET" nameend="4" align="center" rowsep="1" class="description-td" colspan="5"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="OFFSET" nameend="4" align="left" class="description-td" colspan="5">*TCC wt is a mouse teratocarcinoma feeder-independent cell line (6050P) with tk<sup>+</sup> (wild-type) phenotype. </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="OFFSET" nameend="4" align="left" class="description-td" colspan="5"> <sup>†</sup>TCC tk<sup>−</sup> is a derivative of TCC wt that is resistant to BrdUrd and is tk-deficient. </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="OFFSET" nameend="4" align="left" class="description-td" colspan="5">‡LHB 2b is a mouse L tk<sup>−</sup> cell line transformed to the tk<sup>+</sup> phenotype with the Herpes thymidine kinase gene. </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="OFFSET" nameend="4" align="left" class="description-td" colspan="5">§TCC tk-1, -3, -4, and -5 are HAT-resistant teratocarcinoma clones derived from TCC tk<sup>−</sup> after transformation with the Herpes thymidine kinase gene. </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p>30,000×g supernatants of homogenates (S-30) from the indicated cell lines were mixed with preimmune serum or antiserum to purified HSV-1 tk, and tk activity was assayed as described in Materials and Method. Activity is expressed as units per ml of the S-30 fraction.</p>
    <heading>The Physical State of the tk Gene in Transformed Teratocarcinoma Cells</heading> <p>The number of viral tk gene fragments and the location of these fragments in independent transformants were examined utilizing the blot hybridization technique of Southern, Southern, E. M., J. Mol. Biol., 98: 503-517 (1975). The donor DNA was the recombinant plasmid, ptk-1, digested to completion with Bam Hl. This plasmid contains a 3.4 kb fragment with the viral tk gene inserted at the single Bam Hl site within the tetracycline resistance gene of pBR322. Transformation with Bam-cleaved tk DNA results in integration with loss of the Bam sites at the termini of the 3.4 kb fragment. High molecular weight DNA from transformants was cleaved with Bam Hl, fractionated by agarose gel electrophoresis, and transferred to nitrocellulose filters; the filters were then annealed with nick-translated <sup>32</sup>P-tk DNA. In each cell clone, a single annealing fragment was seen; therefore, each clone contains at least one viral tk gene. As expected, each clone reveals a band of molecular weight greater than 3.4 kb. The molecular weights of the annealing fragments differ among the transformed clones, a result suggesting that integration has occurred at different sites within the DNA of the respective transformants.</p>
    <heading>Stability of the Transformed Phenotype in Culture</heading> <p>To test the capacity of the TCC transformants to retain expression of the donor tk gene in culture in the absence of selective pressure, individual clones grown into mass culture in HAT selective medium were subcultured for various periods in the absence of the selective agent. The fraction of cells that retained the tk<sup>+</sup> phenotype was determined by measuring cloning efficiencies in selective and nonselective media. Wide differences among clones became apparent (Table II). Some cell lines,</p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE II</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">In vitro stability of the transformed phenotype</td>
              </tr> <tr class="description-tr"> <td class="description-td">in teratocarcinoma cells.</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="5"> <colspec colname="1" colwidth="35pt" align="left"> </colspec> <colspec colname="2" colwidth="28pt" align="center"> </colspec> <colspec colname="3" colwidth="42pt" align="center"> </colspec> <colspec colname="4" colwidth="56pt" align="center"> </colspec> <colspec colname="5" colwidth="56pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">Generations</td>
                <td class="description-td">Relative cloning</td>
                <td class="description-td">Rate of loss</td>
              </tr> <tr class="description-tr"> <td class="description-td">Clonal</td>
                <td class="description-td"> </td>
                <td class="description-td">in</td>
                <td class="description-td">efficiency in</td>
                <td class="description-td">of tk<sup>+</sup> </td>
              </tr> <tr class="description-tr"> <td class="description-td">cell</td>
                <td class="description-td">Experi-</td>
                <td class="description-td">nonselective</td>
                <td class="description-td">selective</td>
                <td class="description-td">phenotype per</td>
              </tr> <tr class="description-tr"> <td class="description-td">line</td>
                <td class="description-td">ment</td>
                <td class="description-td">medium*</td>
                <td class="description-td">medium†</td>
                <td class="description-td">generation</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="5" align="center" rowsep="1" class="description-td" colspan="5"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="5"> <colspec colname="1" colwidth="35pt" align="left"> </colspec> <colspec colname="2" colwidth="28pt" align="center"> </colspec> <colspec colname="3" colwidth="42pt" align="char" char="."> </colspec> <colspec colname="4" colwidth="56pt" align="center"> </colspec> <colspec colname="5" colwidth="56pt" align="char" char="."> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">TCC tk-1</td>
                <td class="description-td">1</td>
                <td class="description-td">28</td>
                <td class="description-td">0.45</td>
                <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">2</td>
                <td class="description-td">150</td>
                <td class="description-td">0.50</td>
                <td class="description-td">&lt;0.001</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCC tk-2</td>
                <td class="description-td">1</td>
                <td class="description-td">28</td>
                <td class="description-td">0.23</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">2</td>
                <td class="description-td">150</td>
                <td class="description-td">0.02</td>
                <td class="description-td">0.017</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCC tk-3</td>
                <td class="description-td">1</td>
                <td class="description-td">28</td>
                <td class="description-td">0.47</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">2</td>
                <td class="description-td">150</td>
                <td class="description-td">0.27</td>
                <td class="description-td">0.002</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCC tk-4</td>
                <td class="description-td">1</td>
                <td class="description-td">28</td>
                <td class="description-td">0.26</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">2</td>
                <td class="description-td">150</td>
                <td class="description-td">0.16</td>
                <td class="description-td">0.003</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCC tk-5</td>
                <td class="description-td">1</td>
                <td class="description-td">28</td>
                <td class="description-td">0.14</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">2</td>
                <td class="description-td">150</td>
                <td class="description-td">0.01</td>
                <td class="description-td">0.021</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="5" align="center" rowsep="1" class="description-td" colspan="5"> </td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="5" align="left" class="description-td" colspan="5">*Clones were picked and grown in HAT selective medium for 40 cell generations. Cells were then grown in nonselective medium for 28 or 150 generations prior to determining their cloning efficiencies under selective and nonselective conditions. </td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="5" align="left" class="description-td" colspan="5">†One hundred cells were plated in triplicate into HAT selective and nonselective media. The relative cloning efficiency in selective medium is defined as the ratio of the cloning efficiency under selective conditions to the cloning efficiency under nonselective conditions (50-70%). </td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="5" align="left" class="description-td" colspan="5">‡In these calculations it is assumed that for any given cell line the rate of loss of the tk phenotype is constant in each cell generation. The rate of loss per generation may then be calculated from the formula F<sub>M </sub>(1 − X)<sup>N−M </sup>+ F<sub>N</sub>, in which F<sub>M </sub>is the relative cloning efficiency in selective medium after M generations in non-selective medium; F<sub>N </sub>is similarly defined for N generations; and X is the rate of loss per cell generation. </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <heading>Maintenance and Expression of the HSV tk Gene in Vivo During Tissue Differentiation in Tumors</heading> <p>The more critical question of retention of the foreign gene and of its expression during TCC cell differentiation in vivo in the absence of selection was examined in solid tumors. Tumors were formed by inoculating syngeneic hosts (usually two hosts per clone) subcutaneously with 10<sup>7 </sup>cells from each of the same five transformed clones. DNA from these tumors was analyzed by blot hybridization. Neutralization assays and electrophoretic mobility tests of the tk enzyme were also carried out to identify expression of the viral gene. In addition, samples of the same tumors were fixed and examined histologically for evidence of differentiation.</p>
    <p>The restriction fragment profiles of the viral tk gene demonstrated that the gene was retained in all nine tumors analyzed. When each tumor (grown without HAT selection) was compared with its cell line of origin (cultured under HAT selective pressure), the number and location of the annealing fragments in seven of the tumors was identical to that of the corresponding cell line. Thus, the introduced tk gene was, in most cases, maintained for many cell generations spanning at least three weeks in vivo without significant loss or translocation. In two instances, however, a gene rearrangement had occurred, resulting from the loss of the original tk-containing fragment and the appearance of a new fragment of different molecular weight. It is of interest that these two tumors were produced from the two TCC clones that lost the tk<sup>+</sup> phenotype in vitro at highest frequencies (Table II).</p>
    <p>The results of neutralization tests with HSV-tk-specific antiserum demonstrated that at least three of the nine tumors (including one from the TCC tk-1 clone) had viral-type tk activity. (The presence of host cells in the tumors probably contributed substantial amounts of non-neutralized mouse tk in the remaining cases.) Another sample of the tumor derived from the TCC tk-1 line was also analyzed electrophoretically for HSV tk activity; a predominant peak migrating with an R<sub>f </sub>of 0.45, characteristic of the viral enzyme, was observed.</p>
    <p>Histological speciments from each of the tumors were prepared and examined. In addition to the TCC stem cells, tumors contained an array of differentiated tissues similar to those in tumors from the untransformed TCC wt and TCC tk<sup>−</sup> cell lines of origin. Included were muscle, neural formations, adipose tissue, some bone, squamous keratinizing epithelium, and other epithelia, ducts, and tubules.</p>
    <heading>Cotransformation of Teratocarcinoma Cells with the Human β-Globin Gene</heading> <p>Biochemical transformants of mouse L may constitute a competent subpopulation in which an unselectable gene can be introduced, along with an unlinked selectable gene, at frequencies higher than in the general population, Wigler, M., et al., Cell 16: 777-785 (1979). Cotransformation experiments have therefore been carried out in which the Herpes viral tk gene was used as a selectable marker to introduce the human β-globin gene into tk<sup>−</sup> TCC cells. A cloned Hind III restriction endonuclease fragment of human chromosomal DNA containing the β-globin gene (plasmid phβ-8) was cleaved with the enzyme Hind III and mixed with Hind III-linearized ptk-1. After TCC tk<sup>−</sup> cells were exposed to these genes, they were grown for two weeks in HAT selection medium and tk<sup>+</sup> transformants were cloned and analyzed by blot hybridization for presence of human β-globin sequences. A 4.3 kb Bql II restriction fragment containing the intact human β-globin gene is entirely contained within the donor pH -8 plasmid. High molecular weight DNA from the transformants was therefore cleaved with the Bgl II enzyme and analyzed in blot hybridization using the <sup>32</sup>P-labeled 4.3 kb Bgl II fragment as an annealing probe.</p>
    <p>In two of the ten TCC transformants examined, human β-globin sequences were detected. One of the transformants contains one to three copies of the 4.3 kb Bgl II fragment; in this cell line, therefore, the globin gene is evidently intact. The other TCC isolate containing the human β-globin gene displays an aberrant high molecular weight annealing fragment, a result suggesting that cleavage and integration have occurred within the Bgl II fragment. These data demonstrate that those TCC cells that are competent for uptake and expression of the tk gene also integrate another unlinked and unselectable gene at high frequency.</p>
    <heading>Discussion</heading> <p>The experimental introduction of foreign DNA into early mammalian embryos, and its persistence and augmentation during development, were first reported some six years ago, Jaenisch, R. &amp; Mintz, B., Proc. Natl. Acad. Sci. 71: 1250-1254 (19741). Purified (nonrecombinant) SV 40 viral DNA was microinjected into mouse blastocysts; they gave rise to healthy adults whose tissue DNA contained SV 40 gene sequences. Newer technologies such as described herein should allow a wide range of specific genes to be incorporated into the genome of the embryo for in vivo analyses of control of gene expression during differentiation. With the advent of recombinant DNA, quantities of particular genes in native or specifically modified form can be obtained. In the biological sphere, the malignant stem cells of mouse teratocarcinomas have contributed a novel avenue of intervention. These cells can be grown in culture, selected for specific mutations, and microinjected into blastocysts, where they lose their neoplastic properties and participate in development, Dewey, M., J. et al., Proc. Natl. Acad, Sci. USA, 74: 5564-5568 (1977); Watanabe, T., et al., Proc. Natl. Acad. Sci., 75: 5113-5117 (1978). The cultured TCC cells have therefore been viewed as vehicles for transmitting predetermined genetic changes to mice, Mintz, B., Brookhaven Symp., Bio., 29: 82-85, (1977); Mintz, B., Differentiation 13: 25-27 (1979). Such changes obviously might include genes acquired by uptake of DNA.</p>
    <p>DNA-mediated gene transfer into cells of fibroblast lines has been accomplished in culture, Wigler, M., et al., Cell 11: 223-232 (1977); Wigler, M., et al., Cell 14: 725-731 (1978); Willecke, K., et al., Molec. Gen. Genet. 170: 179-185 (1979), Graf, L. H., et al., Somat. Cell Genet., in press (1979); Wigler; M., et al., Proc. Natl. Acad. Sci. USA, 76: 1373-1376 (1979); Wigler, M., et al. Proc. Natl. Acad. Sci., in press (1980); Lewis, W. H. et al., Somat. Cell Genet., in press (1979), and furnished the basis for similar attempts here with teratocarcinoma lines. The TCC-cell route for gene transfer into embryos, as compared with embryo injection of DNA, offers the advantage that transformants, i.e., cell clones in which the specific gene has been retained, can be identified and isolated by selection or screening. In the case of unselectable genes, cotransfer with a selectable one has been found to occur with relatively high frequency, Wigler, M., et al., Cell 16: 777-785 (1979).</p>
    <p>In the present study, tk<sup>−</sup> teratocarcinoma cells have been treated with the cloned thymidine kinase gene of Herpes simplex and a number of HAT-resistant tk<sup>+</sup> clones have been obtained with a frequency of about one transformant per μg of DNA. The reason for the markedly lower frequency of TCC transformants than of L-cell transformants, Wigler, M., et al., Cell 14: 725-731 (1978), is obscure since the basis for transformation competence in eucaryotic cells remains unknown. The donor origin of the tk<sup>+</sup> phenotype in the TCC transformants was demonstrated by the HSV-type electrophoretic mobility of their tk enzyme, and also by neutralization of the tk activity by specific antiserum raised against HSV-1 tk (Table I). Furthermore, blot hybridization tests indicated that at least one intact copy of the viral tk gene was present and integrated into other DNA in the transformed cells. These data support the conclusion that the tk activity in the transformed clones is indeed attributable to presence and expression of the viral gene.</p>
    <p>A requirement for experiments involving the introduction of genes is that they remain stable in vivo, even in the absence of selective pressure, during many cell generations. Stability of the tk<sup>+</sup> transformed phenotype was in fact not only in culture (Table II), but also in tumors arising after subcutaneous inoculation of the stem cells into mice. These tumors exhibited various types of tissue differentiation, similar to the range observed in the untransformed parent TCC line. Hybridization experiments comparing each tumor with its transformed cell line of origin indicated that the donor tk gene was maintained without significant loss or rearrangement in seven of nine tumors examined.</p>
    <p>Many genes of interest in a developmental context are not selectable. An example is the globin gene. As in related experiments with L-cells, Wigler, M., et al., Cell 16: 777-785 (1979), a fragment of human genomic DNA containing an intact β-globin gene was administered to TCC tk<sup>−</sup> cells along with the unlinked HSV tk gene. This proved to be an effective method to obtain TCC tk<sup>+</sup> clones in which, from hybridization evidence, the human β-globin gene was present.</p>
    <p>The experiments described herein therefore demonstrate that cultured TCC stem cells can accept exogenous genes and that such genes can be stably retained as well as expressed during in vivo differentiation in tumors. On this basis, experiments with a euploid TCC cell line can proceed, for the purpose of creating in vivo. markers appropriate for analyses of gene regulation during embryogenesis.</p>
    <heading>MATERIALS AND METHODS</heading> <heading>Cell Cultures</heading> <p>Ltk aprt, a derivative of Ltk clone D, Kit, S. et al., Esp. Cell Res. 31:291-312 (1963), was maintained in Dulbecco's modified Eagle's medium (DME) containing 10% calf serum (Flow Laboratories, Rockville, Md.) and 50 μg/ml of diaminopurine (DAP). Prior to transformation, cells were washed and grown for three generations in the absence of DAP. A Chinese hamster cell line containing an altered dihydrofolate reductase (rendering it resistant to methoxtrexate) A29 Mtx<sup>RIII</sup>, Flintoff, W. F., et al., Somatic Cell Genetics 2:245-261 (1976), was propagated in DME supplemented with 3× non-essential amino acids, 10% calf serum and 1 μg/ml amethopterin. For the amplification experiments, the medium was additionally supplemented with 20 μg/ml of methotrexate.</p>
    <p>Murine Ltk<sup>−</sup> aprt<sup>−</sup> cells are adenine phosphoribosyltransferase-negative derivatives of Ltk<sup>−</sup> clone D cells. Cells were maintained in growth medium and prepared for transformation as described, wigler, M., et al., PNAS 76:1373 1376 (1979).</p>
    <p>HEp-2(human), HeLa(human), CHO (Chinese hamster ovary), and Ltk<sup>−</sup> cells were grown in growth medium. LH2b, a derivative of Ltk<sup>−</sup> transformed with herpes simples. virus tk DNA, was maintained in growth medium containing hypoxanthine at 15 μg/ml, aminopterin at 0.2 μg/ml, and thymidine at 5.0 μg/ml (HAT), Wigler, M., et al., Cell 1:223-232 (1977). All culture dishes were Nunclon (Vanguard International, Neptune, N.J.) plastic.</p>
    <p>The feeder-independent mouse teratocarcinoma cell culture line 6050P, Watanabe, T., et al., PNAS 75:5113-5117 (1978), obtained from a tumor of the OTT 6050 transplant line, was used as the wild-type, or tk<sup>+</sup>, parent and is here designated TCC wt. This line is of the X/O sex chromosome type and has a modal number of 39 chromosomes with characteristics described in Watanabe, T., et al., (1978). The cells were grown in Dulbecco's modified Eagle's medium with 10% fetal calf serum. After 3 hr of exposure to 3 μg/ml of the mutagen N-methyl-N′-nitro-N-nitrosoguanidine, the cells were allowed to recover for two days and were then transferred to medium with 80 μg/ml of BrdUrd. A series of resistant clones were isolated; one supplied the clonal line (TCC tk<sup>−</sup>) used in the present transformation experiments. This line had a reversion frequency to wild-type of less then 10<sup>−8</sup>. The cells were maintained in medium with 30 μg/ml of BrdUrd and, prior to transformation, were washed and grown for three generations in the absence of the drug. Transformation efficiency was compared with that of a tk-deficient line, Kit, S., et al., Exp. Cell. Res. 31:297-312 (1963) of mouse L-cells (L tk<sup>−</sup>).</p>
    <heading>Extraction and Restriction Endonuclease Cleavage of Genomic DNA</heading> <p>High molecular weight DNA was obtained from cultured cells (CHO, LH2b, and HeLa) or from frozen rabbit livers as previously described. Wigler, M., et al., Cell 14:725-731 (1978). High molecular weight salmon sperm DNA was obtained from Worthington. Restriction endonuclease cleavage (Bam I, HindIII, Kpn I, and Xba I) was performed in a buffer containing 50 mM NaCl, 10 mM Tris.HCL, 5 mM MgCl<sub>2</sub>, 7 mM mercapto-ethanol, and bovine serum albumin at 100 μg/ml (pH 7.9). The enzyme-to-DNA ratio was at least two units/μg of DNA, and reaction mixtures were incubated at 37° C. for at least 2 hrs (one unit is the amount of enzyme that digests 1 μg of DNA in 1 hr). To monitor the completeness of digestion, 1 μl of nick-translated adenovirus-2 [<sup>32</sup>P]DNA was incubated with 5 μl of reaction volume for at least 2 hr, cleavage products were separated by electrophoresis in 1% agarose gels, and digestion was monitored by exposing the dried gel to Cronex 2DC x-ray film.</p>
    <p>Intact herpes simplex virus (HSV) DNA was isolated from CV-1-infected cells as previously described. Pellicer, A., et al., Cell 14:133-141 (1978). DNA was digested to comletion with Kpn I (New England Biolabs) in a buffer containing 6 mM Tris (pH 7.9), 6mM MgCl<sub>2</sub>, 6 mM 2-mercapto-ethanol, 6 mM NaCl and 200 μg/ml bovine serum albumin. The restricted DNA was fractionated by electrophoresis through 0.5% agarose gels (17×20×0.5 cm) for 24 hr at 70 V, and the 5.1 kb tk-containing fragment was extracted from the gel as described by Maxam, A. M. and Gilbert, W. PNAS 74:560-564 (1977) and Wigler, M., et al., Cell 14:725-731 (1978)</p>
    <p>ΦX174 am3 RFI DNA was purchased from Bethesda Research Laboratories. Plasmid pBR322 DNA was grown in <i>E. coli </i>HB 101 and purified according to the method of Clewell, D. B., J. Bacteriol. 110:667-676 (1972). The cloned rabbit S major globin gene in the λ Charon 4A derivative (RβG-1) was identified and isolated as previously described. Maniatis, T., et al., Cell 15:687-701(1978).</p>
    <p>In the amplification experiments, the size of the high molecular weight DNA was determined by electrophoresis in 0.3% agarose gels using herpes simplex virus DNA and its Xba I fragments as markers. Only DNA whose average size was larger than 75 kb was found to possess transforming activity in the amplification-experiments. In these experiments, plasmid DNAs were isolated from chloramphenicol amplified cultures by isopycnic centrifugation in CsCl gradients containing 300 μg/ml ethidium bromide.</p>
    <heading>Transformation and Selection</heading> <p>The transformation protocol was as described in Graham, F. L. and Van der Eb, A. J., Virology, 52:456-457 (1973) with the following modifications. One day prior to transformation, cells were seeded at 0.7×10<sup>6 </sup>cells per dish. The medium was changed 4 hr prior to transformation. Sterile, ethanol-precipitated high molecular weight or restriction endonuclease-cleaved eucaryotic DNA dissolved in 1 mM Tris (pH 7.9)/0.1 mM EDTA was used to prepare DNA/CaCl<sub>2 </sub>which contains DNA at 40 μg/ml and 250 mM CaCl<sub>2 </sub>(Mallinkrodt). Twice-concentrated Hepes-buffered saline (2× HBS) was prepared: it contains 280 mM NaCl, 50 mM Hepes, and 1.5 mM sodium phosphate, pH adjusted to 7.10±0.05. DNA/CaCl<sub>2 </sub>solution was added dropwise to an equal volume of sterile 2× HBS. A 1-ml sterile plastic pipette with a cotton plug was inserted into the mixing tube containing 2× HBS, and bubbles were introduced by blowing while the DNA was being added. The calcium phosphate/DNA precipitate was allowed to form without agitation for30-45 min at room temperature. The precipitate was then mixed by gentle pipetting with a plastic pipette, and 1 ml of precipitate was added per plate, directly to the 10 ml of growth medium that covered the recipient cells. After 4-hr incubation at 37° C., the medium was replaced and the cells were allowed to incubate for an additional 20 hr. At that time, selective pressure was applied. For tk<sup>+ </sup>selection, medium was changed to growth medium containing HAT. For aprt selection, cells were trypsinized and replated at lower density (about 0.5×10<sup>6 </sup>cells per 10-cm dish) in medium containing 0.05 mM azaserine and 0.1 mM adenine, For both tk<sup>+</sup> and aprt<sup>+</sup> selection, selective media were changed the next day, 2 days after that, and subsequently every 3 days for 2-3 weeks while transformant clones arose. Colonies were picked by using cloning cylinders and the remainder of the colonies were scored after formaldehyde fixation and staining with Giemsa. For characterization, clones were grown into mass culture under continued selective pressure. A record was kept of the apparent number of cell doublings for each clone isolated.</p>
    <p>Methotrexate-resistant transformants of Ltk<sup>−</sup> aprt<sup>−</sup> cells were obtained following transformation with 20 μg of high molecular weight DNA from A29 Mtx<sup>RIII </sup>cells and selection in DME containing 10% calf serum and 0.2 μg/ml amethopterin.</p>
    <p>For tk<sup>+</sup> selection, cells were grown in HAT medium; for resistance to methotrexate, cells were selected in medium supplemented with 0.1 μg/ml of methotrexate. Colonies were cloned from individual dishes to assure that each transformant arose from an independent event. Ligates between A29 DNA and linearized pBR322 DNA were prepared by incubating a 1:1 ration(w/w) of Sal I-cleaved DNAs with T<sub>4 </sub>ligase (Bethesda Research Laboratories) under the conditions recommended by the supplier. A calcium phosphate precipitate was prepared using 2 μg ligate and 18 μg carrier/ml, and added to recipient cells (the amount of ligate was limited because of the observation that plasmid inhibits transformation). The DNA was allowed to remain in contact with the cells for 4-12 hr and the medium was then aspirated and replaced with fresh DME. Selective pressure was applied 24 hr following exposure to DNA. After 2-3 weeks, colonies were isolated using cloning cylinders.</p>
    <p>In the mouse teratocarcinoma cell experiments, transformation was performed as described previously except that the TCC tk<sup>−</sup> cells were seeded at 3×10<sup>5 </sup>cells/plate one day prior to transformation. To each plate of attached cells was added a calcium phosphate/DNA precipitate prepared with 4 μg of the recombinant plasmid, Ptk-1, digested with Bam Hl, in the presence of 20 μg of high molecular weight DNA obtained from L tk<sup>−</sup> aprt<sup>−</sup> cells.</p>
    <p>In addition, some cells were treated in suspension, Willecke, K. et al., Molec. Gen. Genet. 170:179-185 (1979). 7×10<sup>6 </sup>freshly trypsinized TCC tk<sup>−</sup> cells were mixed with a calcium phosphate/DNA precipitate prepared with 10 μg of DNA from the Bam Hl-cleaved plasmid Ptk-1 and 150 μg of high molecular weight DNA from salmon sperm. Following centrifugation, resuspension, and shaking, as described in Willecke, K. et al. (1979), the cells were again plated in growth medium. After three days, the medium was replaced with HAT medium and colonies of transformants were isolated after two weeks.</p>
    <p>Cotransformation experiments were performed with 4 μg of Bam Hl-digested Ptk-1 DNA along with 4 μg of Hind III-cleaved plasmid pHβ-8 containing the chromosomal adult human β-globin gene, Lawn, R. M., et al., Cell 15:1157-1174 (1978). Tk<sup>+</sup> transformants were selected in growth medium containing 0.1 mM hypoxanthine/0.4 μM aminopterin/16 μM thymidine (HAT). Colonies were picked with cloning cylinders and were grown into mass cultures.</p>
    <heading>Cotransformation of Defined DNA Sequences and the HSV tk Gene</heading> <p>Ltk<sup>−</sup> aprt<sup>−</sup> mouse cells were transformed with either 1-10 μg of ΦX174, 1 μg of pBR322 or 1 μg of RβG-1 DNA in the presence of 1 ng of HSV-1 tk gene and 10-20 μg of salmon sperm carrier DNA, as previously described. Wigler, M. et al., PNAS 76:1373-1376 (1979). Tk<sup>+</sup> transformants were selected in DME containing hypoxanthine, aminopterin and thymidine (HAT) and 10% calf serum. Isolated colonies were picked using cloning cylinders and grown into mass cultures.</p>
    <heading>Enzyme Assays</heading> <p>Extracts were prepared by resuspending washed cell pellets (approximately 10<sup>7 </sup>cells) in 0.1 ml of 0.02 M potassium phosphate, pH 7, containing 0.5% Triton X-100. The supernatant (cytoplasm) obtained after 25 min of 700×g centrifugation was used for the quantitation of enzymatic activity and for electrophoresis. aprt and protein were assayed as previously described. Chasin, L. A., Cell 2:37-41 (1974). Inclusion of 3 mM thymidine triphosphate, an inhibitor of 5′-nucleotidase, Murray, A. W. and Friedrichs, B., Biochem, J. 111:83-89 (1969), in the reaction mixture did not increase AMP recovery, indicating that the nucleotidase was not interfering with the measurement of aprt activity. Iso-electric focusing of aprt was carried out essentially as described for hypoxanthine. phosphoribosyltransferase, Chasin, L. A. and Urlaub, G. Somat. Cell Genet. 2:453-467 (1976), with the following exceptions: The polyacrylamide gel contained an Ampholine (LKB) mixture of 0.8% pH 2.5-4, 0.8% pH 4-6, and 0.4% pH 5-7. For assaying enzymatic activity, [2-<sup>3</sup>H] adenine [0.04 mM, 1 Ci/mmol, New England Nuclear (1 Ci=3.7×10<sup>10 </sup>becquerels)] was substituted for hypoxanthine.</p>
    <heading>Assays of Thymidine Kinase Activity</heading> <p>For specific activity measurements, cells from monolayer cultures were scraped into phosphate buffered saline and washed. The cell pellet was suspended in 5 volumes of extraction buffer (0.01 M Tris.HCl, pH 7.5, 0.01 M KCl, 1 mM MgCl<sub>2</sub>, 1 mM 2-mercaptoethanol, and 50 μM thymidine). The cell suspension was frozen and thawed three times and the KCl concentration was then adjusted to 0.15 M. After sonication, the cytoplasmic extract was obtained by centrifugation at 30,000×g for 30 min, and the supernatant was used for tk assays as described in Wigler, M. et al. Cell 16:777-785 (1979). Cytoplasmic extracts from tumors were obtained after disruption of the cells in a Potter-Elvejehm homogenizer. They were then treated as described above for cultured cells. One unit of thymidine kinase is defined as the amount of enzyme which converts one nanomole of thymidine into thymidine monophosphate per minute.</p>
    <p>In enzyme neutralization studies, anti-HSV-1 tk anti-serum or preimmune serum was mixed with an equal volume of cytoplasmic extract, and ATP and magnesium were added to 6.7 mM. The enzyme-antibody mixture was incubated for 30 min at room temperature, centrifuged at 2,000×g for 10 min, and the supernatant was assayed for tk activity.</p>
    <p>In an additional biochemical assay, 30,000×g supernatants of homogenates from cell cultures and from solid tumors were electrophoresed on 5% polyacrylamide gels which were then cut into 1.6 mm slices and assayed for tk activity as described. Lee, L. S. and Cheng, Y. C., J. Biol. Chem., 251:2600-2604 (1976).</p>
    <heading>RNA Isolation</heading> <p>Total RNA was isolated from logarithmic-phase cultures of transformed L cells by successive extractions with phenol at pH 5.1, phenol/chloroform/isoamyl alcohol (25:24:1, vol/vol), and chloroform/isoamyl alcohol (24:1, vol/vol). After ethanol precipitation, the RNA was digested with DNase, Maxwell, I. H., et al., Nucleic Acids Res. 4:241-246 (1977) and precipitated with ethanol. Nuclear and cytoplasmic fractions were isolated as described in Wigler, M. et al., PNAS 76:1373-1376 (1979) and RNAs were extracted as described above. Cytoplasmic polyadenylylated RNA was isolated by oligo(dT)-cellulose chromatography. Axel, R. et al., Cell 7:247-254 (1976).</p>
    <heading>cDNA Synthesis</heading> <p>Rabbit and mouse cDNAs were prepared by using avian myeloblastosis virus reverse transcriptase (RNA-dependent DNA polymerase) as described in Myers, J. C. and Spiegelman, S., PNAS 75:5329-5333 (1978).</p>
    <heading>Isolation of Transformed Cell DNA</heading> <p>Cells were harvested by scraping into PBS and centrifuging at 1000×g for 10 min. The pellet was resuspended in 40 vol of TNE [10 mM Tris-HCl (ph 8.0), 150 mM NaCl, 10 mM EDTA], and SDS and proteinase K were added to 0.2% and 100 μg/ml, respectively. The lysate was incubated at 37° C. for 5-10 hr and then extracted sequentially with buffer-saturated phenol and CHCl<sub>3</sub>. High molecular weight DNA isolated by mixing the aqueous phase with 2 vol of cold ethanol and immediately removing the precipitate that formed. The DNA was washed with 70% ethanol and dissolved in 1 mM Tris, 0.1 EDTA.</p>
    <p>Nuclei and cytoplasm from clones ΦX4 and ΦX5 were prepared as described by Ringold, G. M., et al. Cell 10:19-26 (1977). The nuclear fraction was further fractionated into high and low molecular weight DNA as described by Hirt, B., J. Mol. Biol. 26:365-369 (1967).</p>
    <heading>DNA Filter Hybridizations</heading> <p>Cellular DNA was digested with restriction endonucleases, electrophoresed on agarose slab gels, transferred to nitro-cellulose filter sheets, and hybridized with <sup>32</sup>P-labeled DNA probes as described by Wigler, M. et al., PNAS 76:1373-1376 (1979).</p>
    <p>DNA from transformed cells was digested with various restriction endonucleases using the conditions specified by the supplier (New England Biolabs or Bethesda Research Laboratories). Digestions were performed at an enzyme to DNA ratio of 1.5 U/μg for 2 hr at 37° C. Reactions were terminated by the addition of EDTA, and the product was electrophoresed on horizontal agarose slab gels in 36 mM Tris, 30 MM NaH<sub>2</sub>PO<sub>4</sub>, 1 mM EDTA (pH 7.7). DNA fragments were transferred to nitrocellulose sheets, hybridized and washed as previously described. Weinstock, R., et al., PNAS 75:1299-1303 (1978) with two modifications. Two nitrocellulose filters were used during transfer. Jeffreys, A. J. and Flavell,. R. A., Cell 12:1097-1108 (1977). The lower filter was discarded, and following hybridization the filter was washed 4 times for 20 min in 2×SSC, 25 mM sodium phosphate, 1.5 mM Na<sub>4</sub>P<sub>2</sub>O<sub>7</sub>, 0.05% SDS at 65° C. and then successively in 1:1 and 1:5 dilutions of this buffer. Jeffreys, A. J. and Flavell, R. A., Cell 12:429-439 (1977).</p>
    <p>In the amplification experiments the-probes were either <sup>32</sup>P-nick translated pBR322 or pdhfr-21, a cDNA copy of mouse dhfr mRNA. Chang, A. C. Y., et al., Nature 275:617-624 (1978).</p>
    <heading>Solution Hybridizations</heading> <p> <sup>32</sup>P-Labeled globin cDNAs (specific activities of 2-9×10<sup>8 </sup>cpm/μg) were hybridized with excess RNA in 0.4 M NaCl/25 -mM 1,4-piperazinediethanesulfonic acid (Pipes), pH 6.5/5 mM EDTA at 75° C. Incubation times did not exceed 70 hr. R<sub>0</sub>ts were calculated as moles of RNA nucleotides per liter times time in seconds. The fraction of cDNA rendered resistant to the single-strand nuclease S<b>1</b> in hybridization was determined as described. Axel, R. et al., Cell 7:247-254 (1976).</p>
    <heading>RNA Filter Hybridization</heading> <p>RNA was electrophoresed through 1% agarose slab gels (17×20×0.4 cm) containing 5 mM methylmercury hydroxide as described by Bailey, J. and Davidson, N., Anal. Biochem. 70:75-85 (1976). The concentration of RNA in each slot was 0.5 μg/μl. Electrophoresis was at 110 V for 12 hr at room temperature.</p>
    <p>RNA was transferred from the gel to diazotized cellulose paper as described by Alwine, J. C., et al., PNAS 74:5350-5354 (1979) by using pH 4.0 citrate transfer buffer. After transfer, the RNA filter was incubated for 1 hr with transfer buffer containing carrier RNA at 500 μg/ml. The RNA on the filters was hybridized with cloned DNA probe at 50 ng/ml labeled by <sup>32</sup>P-nick translation, Weinstock, R., et al., PNAS 75:1299-1303 (1978) to specific activities of 2-8×10<sup>8 </sup>cpm/μg. Reaction volumes were 25 μl/cm<sup>2 </sup>of filter. Hybridization was in 4× standard saline citrate (0.15 M NaCl/0.015 M sodium citrate)/50% formamide at 57° C. for 36-48 hr.</p>
    <p>After hybridization, filters were soaked in two changes of 2× standard saline citrate/25 mM sodium phosphate/1.5 mM sodium pyrophosphate/0.1% sodium dodecyl sulfate/5 mM EDTA at 37° C. for 30 min with shaking to remove formamide. Successive washes were at 68° C. with 1× and 0.1× standard saline citrate containing 5 mM EDTA and 0.1% sodium dodecyl sulfate for 30 min each.</p>
    <heading>Berk Sharp Analysis of Rabbit β-Globin RNA in Transformed Mouse L Cells</heading> <p>The hybridizations were carried out in 80% (vol/vol) formamide (Eastman)/0.4 M Pipes, pH 6.5/0.1 mM EDTA/0.4 M NaCl, Casey, J. and Davidson, N., Nucleic Acid Res., 4:1539-1552 (1977); Berk, A. J. and Sharp, P. A., Cell 12: 721-732 (1977) for 18 hr at 51° C. for the 1.8 kbp Hha I fragment and 49° C. for the Pst 1 fragment. The hybrids were treated with S<b>1</b> nuclease and analyzed essentially by the procedure described by Berk, A. J. and Sharp, P. A. (1977).</p>
    <p>Although the instant disclosure sets forth all essential information in connection with the invention, the numerous publications cited herein may be of assistance in understanding the background of the invention and the state of the art. Accordingly, all of the publications cited are hereby incorporated by reference into the present disclosure.</p>
    </div></div></div><div class="patent-section patent-tabular-section"><a id="backward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Cited Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US3800035">US3800035</a></td><td class="patent-data-table-td patent-date-value">Dec 7, 1971</td><td class="patent-data-table-td patent-date-value">Mar 26, 1974</td><td class="patent-data-table-td ">Smithkline Corp</td><td class="patent-data-table-td ">Production of interferon from human leukocytes in the absence of serum</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4195125">US4195125</a></td><td class="patent-data-table-td patent-date-value">Dec 20, 1978</td><td class="patent-data-table-td patent-date-value">Mar 25, 1980</td><td class="patent-data-table-td ">Hoechst Aktiengesellschaft</td><td class="patent-data-table-td ">Process for obtaining insulin-producing animal cells</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4237224">US4237224</a></td><td class="patent-data-table-td patent-date-value">Jan 4, 1979</td><td class="patent-data-table-td patent-date-value">Dec 2, 1980</td><td class="patent-data-table-td ">Board Of Trustees Of The Leland Stanford Jr. University</td><td class="patent-data-table-td ">Genetic engineering</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4264731">US4264731</a></td><td class="patent-data-table-td patent-date-value">Apr 21, 1978</td><td class="patent-data-table-td patent-date-value">Apr 28, 1981</td><td class="patent-data-table-td ">The Regents Of The University Of California</td><td class="patent-data-table-td ">DNA Joining method</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4342832">US4342832</a></td><td class="patent-data-table-td patent-date-value">Jul 5, 1979</td><td class="patent-data-table-td patent-date-value">Aug 3, 1982</td><td class="patent-data-table-td ">Genentech, Inc.</td><td class="patent-data-table-td ">Method of constructing a replicable cloning vehicle having quasi-synthetic genes</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4356270">US4356270</a></td><td class="patent-data-table-td patent-date-value">Nov 5, 1979</td><td class="patent-data-table-td patent-date-value">Oct 26, 1982</td><td class="patent-data-table-td ">Genentech, Inc.</td><td class="patent-data-table-td ">Recombinant DNA cloning vehicle</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4363877">US4363877</a></td><td class="patent-data-table-td patent-date-value">Apr 19, 1978</td><td class="patent-data-table-td patent-date-value">May 26, 1998</td><td class="patent-data-table-td ">Univ California</td><td class="patent-data-table-td ">Recombinant dna transfer vectors</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4366246">US4366246</a></td><td class="patent-data-table-td patent-date-value">Nov 5, 1979</td><td class="patent-data-table-td patent-date-value">Dec 28, 1982</td><td class="patent-data-table-td ">Genentech, Inc.</td><td class="patent-data-table-td ">Method for microbial polypeptide expression</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4399216">US4399216</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Feb 25, 1980</td><td class="patent-data-table-td patent-date-value">Aug 16, 1983</td><td class="patent-data-table-td ">The Trustees Of Columbia University</td><td class="patent-data-table-td ">Processes for inserting DNA into eucaryotic cells and for producing proteinaceous materials</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4411994">US4411994</a></td><td class="patent-data-table-td patent-date-value">Jun 8, 1978</td><td class="patent-data-table-td patent-date-value">Oct 25, 1983</td><td class="patent-data-table-td ">The President And Fellows Of Harvard College</td><td class="patent-data-table-td ">Genetic engineering, plasmids, hybrids, bacteria, cloning, dna</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4634665">US4634665</a></td><td class="patent-data-table-td patent-date-value">Aug 11, 1983</td><td class="patent-data-table-td patent-date-value">Jan 6, 1987</td><td class="patent-data-table-td ">The Trustees Of Columbia University In The City Of New York</td><td class="patent-data-table-td ">Insulin, interferon, protein and growth hormone</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5179017">US5179017</a></td><td class="patent-data-table-td patent-date-value">Jun 18, 1991</td><td class="patent-data-table-td patent-date-value">Jan 12, 1993</td><td class="patent-data-table-td ">The Trustees Of Columbia University In The City Of New York</td><td class="patent-data-table-td ">Genetic engineering</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP0022685A1?cl=en">EP0022685A1</a></td><td class="patent-data-table-td patent-date-value">Jun 9, 1980</td><td class="patent-data-table-td patent-date-value">Jan 21, 1981</td><td class="patent-data-table-td ">Institut Pasteur</td><td class="patent-data-table-td ">Vectors for the transfer and the expression of genetic material and their uses</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP0037723A2?cl=en">EP0037723A2</a></td><td class="patent-data-table-td patent-date-value">Apr 3, 1981</td><td class="patent-data-table-td patent-date-value">Oct 14, 1981</td><td class="patent-data-table-td ">The Regents Of The University Of California</td><td class="patent-data-table-td ">Expression of hormone genomic clones</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP0038765B1?cl=en">EP0038765B1</a></td><td class="patent-data-table-td patent-date-value">Apr 22, 1981</td><td class="patent-data-table-td patent-date-value">Sep 2, 1987</td><td class="patent-data-table-td ">Institut Pasteur</td><td class="patent-data-table-td ">Vaccine against viral hepatitis b, method and transformed eucaryotic cells for the preparation of this vaccine</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="http://www.google.com/url?id=s1BcBAABERAJ&amp;q=http://worldwide.espacenet.com/publicationDetails/biblio%3FCC%3DGB%26NR%3D1521032A%26KC%3DA%26FT%3DD&amp;usg=AFQjCNHj5-MbII3Yaw85CyY1v3FLOEKxiw">GB1521032A</a></td><td class="patent-data-table-td patent-date-value"></td><td class="patent-data-table-td patent-date-value"></td><td class="patent-data-table-td "> </td><td class="patent-data-table-td citation-no-title">Title not available</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="http://www.google.com/url?id=s1BcBAABERAJ&amp;q=http://worldwide.espacenet.com/publicationDetails/biblio%3FCC%3DGB%26NR%3D2007675A%26KC%3DA%26FT%3DD&amp;usg=AFQjCNGlL68IfTZAhorYObb1qEwi-6K0IA">GB2007675A</a></td><td class="patent-data-table-td patent-date-value"></td><td class="patent-data-table-td patent-date-value"></td><td class="patent-data-table-td "> </td><td class="patent-data-table-td citation-no-title">Title not available</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="http://www.google.com/url?id=s1BcBAABERAJ&amp;q=http://worldwide.espacenet.com/publicationDetails/biblio%3FCC%3DGB%26NR%3D2007676A%26KC%3DA%26FT%3DD&amp;usg=AFQjCNFbGzoeHDXnzAxO_wj7SrlvVo5lNQ">GB2007676A</a></td><td class="patent-data-table-td patent-date-value"></td><td class="patent-data-table-td patent-date-value"></td><td class="patent-data-table-td "> </td><td class="patent-data-table-td citation-no-title">Title not available</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="http://www.google.com/url?id=s1BcBAABERAJ&amp;q=http://worldwide.espacenet.com/publicationDetails/biblio%3FCC%3DGB%26NR%3D2008123A%26KC%3DA%26FT%3DD&amp;usg=AFQjCNGh1dkY6osPCJXJoyJ8i5CtKZxoog">GB2008123A</a></td><td class="patent-data-table-td patent-date-value"></td><td class="patent-data-table-td patent-date-value"></td><td class="patent-data-table-td "> </td><td class="patent-data-table-td citation-no-title">Title not available</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="http://www.google.com/url?id=s1BcBAABERAJ&amp;q=http://worldwide.espacenet.com/publicationDetails/biblio%3FCC%3DGB%26NR%3D2010847A%26KC%3DA%26FT%3DD&amp;usg=AFQjCNFTZ-g2ldOPBAf-GthjG4_DCuLjbw">GB2010847A</a></td><td class="patent-data-table-td patent-date-value"></td><td class="patent-data-table-td patent-date-value"></td><td class="patent-data-table-td "> </td><td class="patent-data-table-td citation-no-title">Title not available</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="npl-citations"></a><div class="patent-section-header"><span class="patent-section-title">Non-Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th colspan="3"class="patent-data-table-th">Reference</th></tr></thead><tr><td class="patent-data-table-td ">1</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Alt, F.W. et al. (1978) "<a href='http://scholar.google.com/scholar?q="Selective+multiplication+of+dihydrofolate+reductase+genes+in+methotrexate-resistant+variant+of+cultured+murine+cells."'>Selective multiplication of dihydrofolate reductase genes in methotrexate-resistant variant of cultured murine cells.</a>" The Journal of Biological Chemistry, 253:1357-1370 (Exhibit 22).</td></tr><tr><td class="patent-data-table-td ">2</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Anderson, R.P. and Roth, J.R. (1977) "<a href='http://scholar.google.com/scholar?q="Tandem+genetic+duplications+in+phage+and+bacteria."'>Tandem genetic duplications in phage and bacteria.</a>" Ann. Rev. Microbiol., 31:473-505 (Exhibit 24).</td></tr><tr><td class="patent-data-table-td ">3</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Anderson, R.P. et al. (1976) "<a href='http://scholar.google.com/scholar?q="Tandem+duplications+of+the+histidine+operon+observed+following+generalized+transduction+in+salmonella+typhimurium."'>Tandem duplications of the histidine operon observed following generalized transduction in salmonella typhimurium.</a>" J. Mol. Biol., 105:201-218 (Exhibit 23).</td></tr><tr><td class="patent-data-table-td ">4</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Anderson, W.F. et al. (1981) "<a href='http://scholar.google.com/scholar?q="Genetic+engineering+in+mammalian+cells."'>Genetic engineering in mammalian cells.</a>" Scientific American, 245:106-121.</td></tr><tr><td class="patent-data-table-td ">5</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Anonymous (1974) "<a href='http://scholar.google.com/scholar?q="Cellular+transformation+by+D.N.A."'>Cellular transformation by D.N.A.</a>" The Lancet 1205-1206 (Exhibit 25).</td></tr><tr><td class="patent-data-table-td ">6</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Biedler J.L. and Spengler, B.A. (1976b) "<a href='http://scholar.google.com/scholar?q="Quantitive+relationship+between+a+chromosome+abnormality+%28HSR%29+and+antifolate+resistance+associated+with+enzyme+overproduction."'>Quantitive relationship between a chromosome abnormality (HSR) and antifolate resistance associated with enzyme overproduction.</a>" Abs. Int'l Congress on Cell Biol., 117a (Exhibit 28).</td></tr><tr><td class="patent-data-table-td ">7</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Biedler, J.L. and Spengler, B.A. (1976a) "<a href='http://scholar.google.com/scholar?q="Metaphase+chromosome+anomaly%3A+Association+with+drug+resistance+and+cell-specific+products."'>Metaphase chromosome anomaly: Association with drug resistance and cell-specific products.</a>" Science, 191:185-187 (Exhibit 27).</td></tr><tr><td class="patent-data-table-td ">8</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Biedler, J.L. et al. (1972) "<a href='http://scholar.google.com/scholar?q="Drug+response%2C+dihydrofolate+reductase%2C+and+cytogenetics+of+amethopterin-resistant+Chinese+hamster+cells+in+vitro."'>Drug response, dihydrofolate reductase, and cytogenetics of amethopterin-resistant Chinese hamster cells in vitro.</a>" Cancer Research, 32:153-161.</td></tr><tr><td class="patent-data-table-td ">9</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Bostock, C.J. and Clark, E.M. (1980) "<a href='http://scholar.google.com/scholar?q="Satellite+DNA+in+large+marker+chromosomes+of+methotrexate-resistant+mouse+cells."'>Satellite DNA in large marker chromosomes of methotrexate-resistant mouse cells.</a>" Cell, 19:709-715 (Exhibit 30).</td></tr><tr><td class="patent-data-table-td ">10</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Breathnach, R. et al. (1980) "<a href='http://scholar.google.com/scholar?q="Correct+splicing+of+a+chicken+ovalbumin+gene+transcript+in+mouse+L+cells."'>Correct splicing of a chicken ovalbumin gene transcript in mouse L cells.</a>" Proc. Natl. Acad. Sci., 77:740-744 (Exhibit 31).</td></tr><tr><td class="patent-data-table-td ">11</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Burch, J.W. and McBride, O.W. (1975) "<a href='http://scholar.google.com/scholar?q="Human+gene+expression+in+rodent+cells+after+uptake+of+isolated+metaphase+chromosomes."'>Human gene expression in rodent cells after uptake of isolated metaphase chromosomes.</a>" Proc. Nat. Acad. Sci. 72:1797-1801 (Exhibit 32).</td></tr><tr><td class="patent-data-table-td ">12</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Camacho, A. and Spear, P.G. (1978) "<a href='http://scholar.google.com/scholar?q="Transformation+of+hamster+embryo+fibroblasts+by+a+specific+fragment+of+the+herpes+simplex+virus+genome."'>Transformation of hamster embryo fibroblasts by a specific fragment of the herpes simplex virus genome.</a>" Cell, 15:993-1002 (Exhibit 33).</td></tr><tr><td class="patent-data-table-td ">13</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Chang, A., et al. (1978) "<a href='http://scholar.google.com/scholar?q="Phenotypic+expression+in+E.+coli+of+a+DNA+sequence+coding+for+mouse+dihydrofolate+reductase."'>Phenotypic expression in E. coli of a DNA sequence coding for mouse dihydrofolate reductase.</a>" Nature, 275:617-624 (Exhibit 34).</td></tr><tr><td class="patent-data-table-td ">14</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Chang, S.E. and Littlefield, J.W. (1976) "<a href='http://scholar.google.com/scholar?q="Elevated+dihydrofolate+reductase+messenger+RNA+levels+in+methotrexate-resistant+BHK+cells."'>Elevated dihydrofolate reductase messenger RNA levels in methotrexate-resistant BHK cells.</a>" Cell, 7:391-396 (Exhibit 35).</td></tr><tr><td class="patent-data-table-td ">15</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Chasin et al., "<a href='http://scholar.google.com/scholar?q="Isolation+of+Chinese+hamster+cell+mutants+deficient+in+dihydrofolate+reductase+activity"'>Isolation of Chinese hamster cell mutants deficient in dihydrofolate reductase activity</a>" Proc. Natl. Acad. Sci., Jul. 1980 p. 4216-4220 (Exhibit 1).</td></tr><tr><td class="patent-data-table-td ">16</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Coon, H.G. and Ho, C. (1977) "<a href='http://scholar.google.com/scholar?q="Transformation+of+cultured+cells+to+chloramphenicol+resistance+by+purified+mammalian+mitochondrial+DNA."'>Transformation of cultured cells to chloramphenicol resistance by purified mammalian mitochondrial DNA.</a>" Brookhaven Symp. Biol., 29:166-177 (Exhibit 36).</td></tr><tr><td class="patent-data-table-td ">17</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Davis, B.D. (1980) "<a href='http://scholar.google.com/scholar?q="Gene+transfer+in+bacteria."'>Gene transfer in bacteria.</a>" Microbiology: Including Immunology and Molecular Genetics, Third Edition 138-151, Harper &amp; Row, Publishers, Inc. (Exhibit 37).</td></tr><tr><td class="patent-data-table-td ">18</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Degnen, G.E. et al. (1977) "<a href='http://scholar.google.com/scholar?q="Overexpression+of+an+unstably+inherited+gene+in+cultured+mouse+cells."'>Overexpression of an unstably inherited gene in cultured mouse cells.</a>" Proc. Natl. Acad. Sci., 74:3956-3959 (Exhibit 38).</td></tr><tr><td class="patent-data-table-td ">19</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Dolnick, B.J. et al. (1979) "<a href='http://scholar.google.com/scholar?q="Correlation+of+dihydrofolate+reductase+elevation+with+gene+amplification+in+a+homogeneously+staining+chromosomal+region+in+L5178Y+cells."'>Correlation of dihydrofolate reductase elevation with gene amplification in a homogeneously staining chromosomal region in L5178Y cells.</a>" J. Cell Biology, 83:394-402 (Exhibit 39).</td></tr><tr><td class="patent-data-table-td ">20</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Ehrlich, M. et al. (1976) "<a href='http://scholar.google.com/scholar?q="Interaction+of+microbial+DNA+with+cultured+mammalian+cells.+Binding+of+the+donor+DNA+to+the+cell+surface."'>Interaction of microbial DNA with cultured mammalian cells. Binding of the donor DNA to the cell surface.</a>" Biochima et Biophysica Acta, 454:397-409 (Exhibit 40).*</td></tr><tr><td class="patent-data-table-td ">21</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Erickson, R.J. and Braun, W. (1968) "<a href='http://scholar.google.com/scholar?q="Apparent+dependence+of+transformation+on+the+stage+of+deoxyribonucleic+acid+replication+of+recipient+cells."'>Apparent dependence of transformation on the stage of deoxyribonucleic acid replication of recipient cells.</a>" Bacteriological Reviews, 32:291-296 (Exhibit 41).*</td></tr><tr><td class="patent-data-table-td ">22</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Fischer, G.A. (1962) "<a href='http://scholar.google.com/scholar?q="Defective+transport+of+amethopterin+%28methotrexate%29+as+a+mechanism+of+resistance+to+the+antimetabolite+in+L5178Y+leukemic+cells."'>Defective transport of amethopterin (methotrexate) as a mechanism of resistance to the antimetabolite in L5178Y leukemic cells.</a>" Biochemical Pharmacology, 11:1233-1234 (Exhibit 42).*</td></tr><tr><td class="patent-data-table-td ">23</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Fling, M. et al. (1979) "<a href='http://scholar.google.com/scholar?q="Cloning+and+amplification+of+DNA+sequences+encoding+trimethoprim-resistant+dihydrofolate+reductase+gene."'>Cloning and amplification of DNA sequences encoding trimethoprim-resistant dihydrofolate reductase gene.</a>" Chemical Abstracts, published by The American Chemical Society, 90:51265f (Exhibit 43).*</td></tr><tr><td class="patent-data-table-td ">24</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Flintoff, W.F. et al., (1976) "<a href='http://scholar.google.com/scholar?q="Isolation+and+partial+characterization+of+three+methotrexate-resistant+phenotypes+from+chinese+hamster+ovary+cells."'>Isolation and partial characterization of three methotrexate-resistant phenotypes from chinese hamster ovary cells.</a>" Somatic Cell Genetics, 2:245-261 (Exhibit 44).</td></tr><tr><td class="patent-data-table-td ">25</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Fournier, R.E.K. and Ruddle, F.H. (1977a) "<a href='http://scholar.google.com/scholar?q="Microcell-mediated+transfer+of+murine+chromosomes+into+mouse%2C+Chinese+hamster%2C+and+human+somatic+cells."'>Microcell-mediated transfer of murine chromosomes into mouse, Chinese hamster, and human somatic cells.</a>" Proc. Natl. Acad. Sci., 74:319-323 (Exhibit 45).</td></tr><tr><td class="patent-data-table-td ">26</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Fournier, R.E.K. and Ruddle, F.H. (1977b) "<a href='http://scholar.google.com/scholar?q="Stable+association+of+the+human+transgenome+and+host+murine+chromosomes+demonstrated+with+trispecific+microcell+hybrids."'>Stable association of the human transgenome and host murine chromosomes demonstrated with trispecific microcell hybrids.</a>" Proc. Natl. Acad. Sci., 74:3937-3941 (Exhibit 46).</td></tr><tr><td class="patent-data-table-td ">27</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Fox, J. (1980) "<a href='http://scholar.google.com/scholar?q="Questions+muddy+gene+transplant+successes."'>Questions muddy gene transplant successes.</a>" C &amp; E News, pp. 35, 36 and 42.</td></tr><tr><td class="patent-data-table-td ">28</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Graham, F.L. and Van Der Eb, A.J. (1973) "<a href='http://scholar.google.com/scholar?q="A+new+technique+for+the+assay+of+infectivity+of+human+adenovirus+5+DNA."'>A new technique for the assay of infectivity of human adenovirus 5 DNA.</a>" Virology 52:456-467 (Exhibit 47).</td></tr><tr><td class="patent-data-table-td ">29</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Graham, F.L. et al. (1975) "<a href='http://scholar.google.com/scholar?q="Studies+on+in+vitro+transformation+by+DNA+and+DNA+fragments+of+human+adenoviruses+and+simian+virus+40."'>Studies on in vitro transformation by DNA and DNA fragments of human adenoviruses and simian virus 40.</a>" Cold Spring Harbor Symp. Quant. Biol., 39:637-649 (Exhibit 48).</td></tr><tr><td class="patent-data-table-td ">30</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Kaufman, R.J. et al. (1979) "<a href='http://scholar.google.com/scholar?q="Amplified+dihydrofolate+reductase+genes+in+unstably+methotrexate-resistant+cells+are+associated+with+double+minute+chromosomes."'>Amplified dihydrofolate reductase genes in unstably methotrexate-resistant cells are associated with double minute chromosomes.</a>" Proc. Natl. Acad. Sci., 76:5669-5673 (Exhibit 49).</td></tr><tr><td class="patent-data-table-td ">31</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Lehninger, A.L. (1975) "<a href='http://scholar.google.com/scholar?q="The+molecular+basis+of+cell+structure+and+function."'>The molecular basis of cell structure and function.</a>" Biochemistry, Second Edition 118-119 (Exhibit 51).</td></tr><tr><td class="patent-data-table-td ">32</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Lewis, W.H. (1980) "<a href='http://scholar.google.com/scholar?q="Parameters+governing+the+transfer+of+the+genes+for+thymidine+kinase+and+dihydrofolate+reductase+into+mouse+cells+using+metaphase+chromosomes+or+DNA."'>Parameters governing the transfer of the genes for thymidine kinase and dihydrofolate reductase into mouse cells using metaphase chromosomes or DNA.</a>" Somatic Cell and Molecular Genetics, 6:333-348 (Exhibit 52).</td></tr><tr><td class="patent-data-table-td ">33</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Maitland, N.J. and McDougall, J.K. (1977) "<a href='http://scholar.google.com/scholar?q="Biochemical+transformation+of+mouse+cells+by+fragments+of+herpes+simplex+virus+DNA."'>Biochemical transformation of mouse cells by fragments of herpes simplex virus DNA.</a>" Cell, 11:233-241 (Exhibit 53).</td></tr><tr><td class="patent-data-table-td ">34</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Majumdar, A. and Bose, S.K. (1968) "<a href='http://scholar.google.com/scholar?q="DNA+mediated+genetic+transformation+of+a+human+cancerous+cell+line+cultured+in+vitro."'>DNA mediated genetic transformation of a human cancerous cell line cultured in vitro.</a>" Br. J. Cancer, 22:603-613 (Exhibit 54).</td></tr><tr><td class="patent-data-table-td ">35</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">McBride, O.W. and Ozer, H.L. (1973) "<a href='http://scholar.google.com/scholar?q="Transfer+of+genetic+information+by+purified+metaphase+chromosomes."'>Transfer of genetic information by purified metaphase chromosomes.</a>" Proc. Nat. Acad. Sci., 70:1258-1262 (Exhibit 56).</td></tr><tr><td class="patent-data-table-td ">36</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">McBride, O.W. and Peterson, J.L. (1980) "<a href='http://scholar.google.com/scholar?q="Chromosome-mediated+gene+transfer+in+mammalian+cells."'>Chromosome-mediated gene transfer in mammalian cells.</a>" Ann. Rev. Genet., 14:321-345 (Exhibit 57).</td></tr><tr><td class="patent-data-table-td ">37</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Merck &amp; Co., Inc. (1968) The Merck Index, p. 568 (Exhibit 58).</td></tr><tr><td class="patent-data-table-td ">38</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Minson, A.C. et al. (1978) "<a href='http://scholar.google.com/scholar?q="Introduction+of+the+herpes+simplex+virus+thymidine+kinase+gene+into+mouse+cells+using+virus+DNA+or+transformed+cell+DNA."'>Introduction of the herpes simplex virus thymidine kinase gene into mouse cells using virus DNA or transformed cell DNA.</a>" Cell, 13:581-587 (Exhibit 59).</td></tr><tr><td class="patent-data-table-td ">39</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Mulligan, R.C. et al. (1979) "<a href='http://scholar.google.com/scholar?q="Synthesis+of+rabbit+beta-globin+in+cultured+monkey+kidney+cells+following+infection+with+a+SV40+beta-globin+recombinant+genome."'>Synthesis of rabbit beta-globin in cultured monkey kidney cells following infection with a SV40 beta-globin recombinant genome.</a>" Nature, 277:108-114 (Exhibit 60).</td></tr><tr><td class="patent-data-table-td ">40</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Mulligan, R.C. et al. (1979) "<a href='http://scholar.google.com/scholar?q="Synthesis+of+rabbit+%CE%B2-globin+in+cultured+monkey+kidney+cells+following+infection+with+a+SV40+%CE%B2-globin+recombinant+genome."'>Synthesis of rabbit β-globin in cultured monkey kidney cells following infection with a SV40 β-globin recombinant genome.</a>" Nature, 277:108-114 (Exhibit 60).</td></tr><tr><td class="patent-data-table-td ">41</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Nunberg, J.H. et al. (1978) "<a href='http://scholar.google.com/scholar?q="Amplified+dihydrofolate+reductase+genes+are+localized+to+a+homogeneously+staining+region+of+a+single+chromosome+in+a+methotrexate-resistant+chinese+hamster+ovary+cell+line."'>Amplified dihydrofolate reductase genes are localized to a homogeneously staining region of a single chromosome in a methotrexate-resistant chinese hamster ovary cell line.</a>" PNAS, 75:5553-5556.</td></tr><tr><td class="patent-data-table-td ">42</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Ottolenghi-Nightingale, E. (1974) "<a href='http://scholar.google.com/scholar?q="DNA-mediated+transformation+in+mammalian+cells."'>DNA-mediated transformation in mammalian cells.</a>" Cell Commun., 233-254 (Exhibit 63).</td></tr><tr><td class="patent-data-table-td ">43</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Perlman, D. et al. (1975) "<a href='http://scholar.google.com/scholar?q="Denaturation+mapping+of+R+factor+deoxyribonucleic+acid."'>Denaturation mapping of R factor deoxyribonucleic acid.</a>" J. Bacteriology, 123:1035-1042 (Exhibit 64).</td></tr><tr><td class="patent-data-table-td ">44</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Peterson, J.L. and McBride, O.W. (1980) "<a href='http://scholar.google.com/scholar?q="Cotransfer+of+linked+eukaryotic+genes+and+efficient+transfer+of+hypoxanthine+phosphoribosyltransferase+by+DNA-mediated+gene+transfer."'>Cotransfer of linked eukaryotic genes and efficient transfer of hypoxanthine phosphoribosyltransferase by DNA-mediated gene transfer.</a>" PNAS, 77:1583-1587 (Exhibit 65).</td></tr><tr><td class="patent-data-table-td ">45</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Roosa, R.A. and Bailey, E. (1970) "<a href='http://scholar.google.com/scholar?q="DNA-mediated+transformation+of+mammalian+cells+in+culture.+Increased+transforming+efficiency+following+sonication."'>DNA-mediated transformation of mammalian cells in culture. Increased transforming efficiency following sonication.</a>" J. Cell. Physiol., 75:137-150 (Exhibit 68).</td></tr><tr><td class="patent-data-table-td ">46</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Scangos, G.A. et al. (1979) "<a href='http://scholar.google.com/scholar?q="Molecular+analysis+of+chromosome-mediated+gene+transfer."'>Molecular analysis of chromosome-mediated gene transfer.</a>" Proc. Natl. Acad. Sci., 76:3987-3990 (Exhibit 69).</td></tr><tr><td class="patent-data-table-td ">47</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Schimke, R.T. et al. (1978a) "<a href='http://scholar.google.com/scholar?q="Gene+amplification+and+drug+resistance+in+cultured+murine+cells."'>Gene amplification and drug resistance in cultured murine cells.</a>" Science 202:1051-1055 (Exhibit 70).</td></tr><tr><td class="patent-data-table-td ">48</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Schimke, R.T. et al. (1978b) "<a href='http://scholar.google.com/scholar?q="Amplification+of+dihydrofolate+reductase+genes+in+methotrexate-resistant+cultured+mouse+cells."'>Amplification of dihydrofolate reductase genes in methotrexate-resistant cultured mouse cells.</a>" Cold Spring Harbor Symp. Quant. Biol., 42:649-657 (Exhibit 71).</td></tr><tr><td class="patent-data-table-td ">49</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Schimke, R.T. et al. (1979a) "<a href='http://scholar.google.com/scholar?q="Studies+on+the+amplification+of+dihydrofolate+reductase+genes+in+methotrexate-resistant+cultured+mouse+cells."'>Studies on the amplification of dihydrofolate reductase genes in methotrexate-resistant cultured mouse cells.</a>" Cold Spring Harbor Symp. Quant. Biol., 43:1297-1303 (Exhibit 72).</td></tr><tr><td class="patent-data-table-td ">50</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Schimke, R.T. et al. (1979b) "<a href='http://scholar.google.com/scholar?q="Structure+and+localization+of+dihydrofolate+reductase+genes+in+methotrexate-resistant+cultured+cells."'>Structure and localization of dihydrofolate reductase genes in methotrexate-resistant cultured cells.</a>" Eucaryotic Gene Regulation, ICN-UCLA Symposia, R. Axel and T. Maniatis, Editors, Academic Press, 499-510 (Exhibit 73).</td></tr><tr><td class="patent-data-table-td ">51</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Sirotnak, F.M. et al. (1968) "<a href='http://scholar.google.com/scholar?q="On+the+nature+of+a+transport+alteration+determining+resistance+to+amethopterin+in+the+L1210+leukemia."'>On the nature of a transport alteration determining resistance to amethopterin in the L1210 leukemia.</a>" Cancer Research, 28:75-80 (Exhibit 74).</td></tr><tr><td class="patent-data-table-td ">52</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Spandidos, D.A. and Siminovitch, L. (1977a) "<a href='http://scholar.google.com/scholar?q="Linkage+of+markers+controlling+consecutive+biochemical+steps+in+CHO+cells+as+demonstrated+by+chromosome+transfer."'>Linkage of markers controlling consecutive biochemical steps in CHO cells as demonstrated by chromosome transfer.</a>" Cell, 12:235-242 (Exhibit 75).</td></tr><tr><td class="patent-data-table-td ">53</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Spandidos, D.A. and Siminovitch, L. (1977b) "<a href='http://scholar.google.com/scholar?q="Transfer+of+codominant+markers+by+isolated+metaphase+chromosomes+in+Chinese+hamster+ovary+cells."'>Transfer of codominant markers by isolated metaphase chromosomes in Chinese hamster ovary cells.</a>" Proc. Natl. Acad. Sci., 74:3480-3484 (Exhibit 76).</td></tr><tr><td class="patent-data-table-td ">54</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Spradling, A.C. and Mahowald, A.P. (1980) "<a href='http://scholar.google.com/scholar?q="Amplification+of+genes+for+chorion+proteins+during+oogenesis+in+Drosophila+melanogaster."'>Amplification of genes for chorion proteins during oogenesis in Drosophila melanogaster.</a>" Proc. Natl. Acad. Sci., 77:1096-1100.</td></tr><tr><td class="patent-data-table-td ">55</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Stebbing, N. (1979) "<a href='http://scholar.google.com/scholar?q="Cellular+uptake+and+in+vivo+fate+of+polynucleotides."'>Cellular uptake and in vivo fate of polynucleotides.</a>" Cell Biology International Reports, 3:485-502 (Exhibit 78).</td></tr><tr><td class="patent-data-table-td ">56</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Sugden, B. et al. (1979) "<a href='http://scholar.google.com/scholar?q="Epstein-Barr+virus+DNA+is+amplified+in+transformed+lymphocytes."'>Epstein-Barr virus DNA is amplified in transformed lymphocytes.</a>" Journal of Virology, 31:590-595 (Exhibit 79).</td></tr><tr><td class="patent-data-table-td ">57</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Sullivan, D. et al. (1973) "<a href='http://scholar.google.com/scholar?q="Synthesis+of+a+deoxyribonucleic+acid+sequence+complementary+to+ovalbumin+messenger+ribonucleic+acid+and+quantification+of+ovalbumin+genes."'>Synthesis of a deoxyribonucleic acid sequence complementary to ovalbumin messenger ribonucleic acid and quantification of ovalbumin genes.</a>" J. Biological Chemistry 248: 7530-7539 (Exhibit 80).</td></tr><tr><td class="patent-data-table-td ">58</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Trust. Of Columbia Univ. v. Roche Diagnostics GMBH, 150 F. Supp.2d 191 (D. Mass 2001) (Exhibit 3).</td></tr><tr><td class="patent-data-table-td ">59</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Trust. Of Columbia Univ. v. Roche Diagnostics, 126 F. Supp.2d 16 (D. Mass 2000) (Exhibit 4).</td></tr><tr><td class="patent-data-table-td ">60</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">U.S. patent application Ser. No. 08/477159, Axel et al., filed Jun. 5, 1995.</td></tr><tr><td class="patent-data-table-td ">61</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Van den Berg, J. et al. (1978) "<a href='http://scholar.google.com/scholar?q="Comparison+of+cloned+rabbit+and+mouse+beta-globin+genes+showing+strong+evolutionary+divergence+of+two+homologous+pairs+of+introns."'>Comparison of cloned rabbit and mouse beta-globin genes showing strong evolutionary divergence of two homologous pairs of introns.</a>" Nature 276:37-44 (Exhibit 81).</td></tr><tr><td class="patent-data-table-td ">62</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Van den Berg, J. et al. (1978) "<a href='http://scholar.google.com/scholar?q="Comparison+of+cloned+rabbit+and+mouse+%CE%B2-globin+genes+showing+strong+evolutionary+divergence+of+two+homologous+pairs+of+introns."'>Comparison of cloned rabbit and mouse β-globin genes showing strong evolutionary divergence of two homologous pairs of introns.</a>" Nature 276:37-44 (Exhibit 81).</td></tr><tr><td class="patent-data-table-td ">63</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Werkheiser, W.C. (1961) "<a href='http://scholar.google.com/scholar?q="Specific+binding+of+4-amino+folic+acid+analogues+by+folic+acid+reductase."'>Specific binding of 4-amino folic acid analogues by folic acid reductase.</a>" The Journal of Biological Chemistry, 236:888-893 (Exhibit 83).</td></tr><tr><td class="patent-data-table-td ">64</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Wiessmann, C. et al. (1979) "<a href='http://scholar.google.com/scholar?q="Expression+of+cloned+viral+and+chromosomal+plasmid-linked+DNA+in+cognate+host+cells."'>Expression of cloned viral and chromosomal plasmid-linked DNA in cognate host cells.</a>" From Gene to Protein Information Transfer in Normal and Abnormal Cells: Miami Winter Symposium, 16:99-132 (Russell, T.R. et al. eds., Academic Press) (Exhibit 82).</td></tr><tr><td class="patent-data-table-td ">65</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Wigler, M. et al. (1977) "<a href='http://scholar.google.com/scholar?q="Transfer+of+purified+herpes+virus+thymidine+kinase+gene+to+cultured+mouse+cells."'>Transfer of purified herpes virus thymidine kinase gene to cultured mouse cells.</a>" Cell, 11:223-232.</td></tr><tr><td class="patent-data-table-td ">66</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Wigler, M. et al. (1978) "<a href='http://scholar.google.com/scholar?q="Biochemical+transfer+of+single-copy+eucaryotic+genes+using+total+cellular+DNA+as+donor."'>Biochemical transfer of single-copy eucaryotic genes using total cellular DNA as donor.</a>" Cell, 14:725-731.</td></tr><tr><td class="patent-data-table-td ">67</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Wilkie, N.M. et al. (1979) "<a href='http://scholar.google.com/scholar?q="Hybrid+plasmids+containing+an+active+thymidine+kinase+gene+of+herpes+simplex+virus+1."'>Hybrid plasmids containing an active thymidine kinase gene of herpes simplex virus 1.</a>" Nucleic Acids Research 7:859-877 (Exhibit 87).</td></tr><tr><td class="patent-data-table-td ">68</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Willecke, K. (1978) "<a href='http://scholar.google.com/scholar?q="Results+and+prospects+of+chromosomal+gene+transfer+between+cultured+mammalian+cells."'>Results and prospects of chromosomal gene transfer between cultured mammalian cells.</a>" Theor. Appl. Genet. 52:97-104 (Exhibit 89).</td></tr><tr><td class="patent-data-table-td ">69</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Willecke, K. and Ruddle, F.H. (1975) "<a href='http://scholar.google.com/scholar?q="Transfer+of+the+human+gene+for+hypoxanthine-guanine+phosphoribosyltransferase+via+isolated+human+metaphase+chromosomes+into+mouse+L-Cells."'>Transfer of the human gene for hypoxanthine-guanine phosphoribosyltransferase via isolated human metaphase chromosomes into mouse L-Cells.</a>" Proc. Nat. Acad. Sci., 72:1792-1796 (Exhibit 88).</td></tr><tr><td class="patent-data-table-td ">70</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Zakai, N. et al. (1977) "<a href='http://scholar.google.com/scholar?q="Membrane+ultrastructural+changes+during+calcium+phosphate-induced+fusion+of+human+erythrocyte+ghosts."'>Membrane ultrastructural changes during calcium phosphate-induced fusion of human erythrocyte ghosts.</a>" Proc. Natl. Acad. Sci., 74:2417-2421 (Exhibit 91).</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="forward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Referenced by</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Citing Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6933136">US6933136</a></td><td class="patent-data-table-td patent-date-value">Sep 17, 2003</td><td class="patent-data-table-td patent-date-value">Aug 23, 2005</td><td class="patent-data-table-td ">Novo Nordisk A/S</td><td class="patent-data-table-td ">Method for making recombinant proteins</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7011974">US7011974</a></td><td class="patent-data-table-td patent-date-value">Sep 17, 2002</td><td class="patent-data-table-td patent-date-value">Mar 14, 2006</td><td class="patent-data-table-td ">Wyeth</td><td class="patent-data-table-td ">Using hybridomas cells; gene expression; monoclonal antibodies</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7247475">US7247475</a></td><td class="patent-data-table-td patent-date-value">Jan 4, 2006</td><td class="patent-data-table-td patent-date-value">Jul 24, 2007</td><td class="patent-data-table-td ">Wyeth</td><td class="patent-data-table-td ">Optimizing the expression level of an antibody producing eukaryotic host cells capable of expressing a first DNA sequence encoding an antibody heavy chain or an antigen binding portion, culturing host cells, amplifying, choosing a host cell that produces a desired amount of antibody</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7879322">US7879322</a></td><td class="patent-data-table-td patent-date-value">Oct 10, 2008</td><td class="patent-data-table-td patent-date-value">Feb 1, 2011</td><td class="patent-data-table-td ">Novartis Ag</td><td class="patent-data-table-td ">Compositions and methods for use for antibodies against sclerostin</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7923221">US7923221</a></td><td class="patent-data-table-td patent-date-value">Apr 13, 1995</td><td class="patent-data-table-td patent-date-value">Apr 12, 2011</td><td class="patent-data-table-td ">Genentech, Inc</td><td class="patent-data-table-td ">Methods of making antibody heavy and light chains having specificity for a desired antigen</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8246953">US8246953</a></td><td class="patent-data-table-td patent-date-value">Nov 11, 2010</td><td class="patent-data-table-td patent-date-value">Aug 21, 2012</td><td class="patent-data-table-td ">Novartis Ag</td><td class="patent-data-table-td ">Compositions and methods for use for antibodies against sclerostin</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8486661">US8486661</a></td><td class="patent-data-table-td patent-date-value">Jul 12, 2012</td><td class="patent-data-table-td patent-date-value">Jul 16, 2013</td><td class="patent-data-table-td ">Novartis Ag</td><td class="patent-data-table-td ">Polynucleotides encoding antibodies against sclerostin and uses therefor</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2468768A2?cl=en">EP2468768A2</a></td><td class="patent-data-table-td patent-date-value">Jul 21, 2006</td><td class="patent-data-table-td patent-date-value">Jun 27, 2012</td><td class="patent-data-table-td ">Abbott Laboratories</td><td class="patent-data-table-td ">Multiple gene expression including sorf contructs and methods with polyproteins, pro-proteins, and proteolysis</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2468881A2?cl=en">EP2468881A2</a></td><td class="patent-data-table-td patent-date-value">Jul 21, 2006</td><td class="patent-data-table-td patent-date-value">Jun 27, 2012</td><td class="patent-data-table-td ">Abbott Laboratories</td><td class="patent-data-table-td ">Multiple gene expression including sorf contructs and methods with polyproteins, pro-proteins, and proteolysis</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP2484774A2?cl=en">EP2484774A2</a></td><td class="patent-data-table-td patent-date-value">Jul 21, 2006</td><td class="patent-data-table-td patent-date-value">Aug 8, 2012</td><td class="patent-data-table-td ">Abbott Laboratories</td><td class="patent-data-table-td ">Multiple gene expression including sorf contructs and methods with polyproteins, pro-proteins, and proteolysis</td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="classifications"></a><div class="patent-section-header"><span class="patent-section-title">Classifications</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th"> </th><th class="patent-data-table-th"> </th></tr></thead><tr><td class="patent-data-table-td ">U.S. Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=s1BcBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S069100">435/69.1</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=s1BcBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S358000">435/358</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=s1BcBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S320100">435/320.1</a></span></td></tr><tr><td class="patent-data-table-td ">International Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=s1BcBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0015870000">C12N15/87</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=s1BcBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0015850000">C12N15/85</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=s1BcBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12P0021020000">C12P21/02</a></span></td></tr><tr><td class="patent-data-table-td ">Cooperative Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=s1BcBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N15/87">C12N15/87</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=s1BcBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12P21/02">C12P21/02</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=s1BcBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N15/85">C12N15/85</a></span></td></tr><tr><td class="patent-data-table-td ">European Classification</td><td class="patent-data-table-td "><span class="nested-value">C12P21/02</span>, <span class="nested-value">C12N15/85</span>, <span class="nested-value">C12N15/87</span></td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="legal-events"></a><div class="patent-section-header"><span class="patent-section-title">Legal Events</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Date</th><th class="patent-data-table-th">Code</th><th class="patent-data-table-th">Event</th><th class="patent-data-table-th">Description</th></tr></thead><tr><td class="patent-data-table-td patent-date-value">May 2, 2014</td><td class="patent-data-table-td ">REMI</td><td class="patent-data-table-td ">Maintenance fee reminder mailed</td><td class="patent-data-table-td "></td></tr><tr><td class="patent-data-table-td patent-date-value">Feb 26, 2013</td><td class="patent-data-table-td ">FPB1</td><td class="patent-data-table-td ">Expired due to reexamination which canceled all claims</td><td class="patent-data-table-td "></td></tr><tr><td class="patent-data-table-td patent-date-value">Feb 4, 2010</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">8</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Dec 20, 2005</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">4</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Nov 30, 2004</td><td class="patent-data-table-td ">RF</td><td class="patent-data-table-td ">Reissue application filed</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20040618</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Apr 6, 2004</td><td class="patent-data-table-td ">RR</td><td class="patent-data-table-td ">Request for reexamination filed</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20040227</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Jun 10, 2003</td><td class="patent-data-table-td ">CC</td><td class="patent-data-table-td ">Certificate of correction</td><td class="patent-data-table-td "></td></tr></table><div class="patent-section-footer"></div></div><div class="modal-dialog" id="patent-images-lightbox"><div class="patent-lightbox-controls"><div class="patent-lightbox-rotate-controls"><div class="patent-lightbox-rotation-text">Rotate</div><div class="rotate-icon rotate-ccw-icon"></div><div class="rotate-icon rotate-cw-icon"></div></div><div class="patent-lightbox-index-counter"></div><a class="patent-lightbox-fullsize-link" target="_blank">Original Image</a><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_right.png" alt="Next page"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_left.png" alt="Previous page"width="21" height="21" /></div></div><div class="modal-dialog-content"><div class="patent-lightbox-image-holder"><div class="patent-lightbox-placeholder"></div></div></div></div><script>_OC_initPatentsAtb({image_not_available_html: " Image not available"});</script></div></div></div></td></tr></table><script>(function() {var href = window.location.href;if (href.indexOf('?') !== -1) {var parameters = href.split('?')[1].split('&');for (var i = 0; i < parameters.length; i++) {var param = parameters[i].split('=');if (param[0] == 'focus') {var elem = document.getElementById(param[1]);if (elem) {elem.focus();}}}}})();</script><script>_OC_addFlags({LockSrc:"/books/javascript/lock_8a2b04e7bf975d5171d8e4c0b6365c7a.js", Host:"http://www.google.com/", IsBooksRentalEnabled:1, IsWebstoreDisplayCaseEnabled:1, IsObfuscationEnabled:1, IsBrowsingHistoryEnabled:1, IsWebReaderSvgEnabled:0, IsGeoLayerEnabled:1, IsImageModeNotesEnabled:1, IsCopyMenuItemEnabled:1, IsGiftingEnabled:0, IsWebReaderUniversalPaginatorEnabled:0, IsOfflineBubbleEnabled:1, IsReaderEnabledForPlayRequests:1, IsFutureOnSaleVolumesEnabled:1, IsOfflineRestrictedCopyEnabled:1, IsBooksUnifiedLeftNavEnabled:1, IsRestrictedCopyEnabled:1, IsZipitFolderCollectionEnabled:1, IsEndOfSampleRecommendationsEnabled:1, IsRatingsOnBookcardsEnabled:1, IsAdsDisabled:0, IsIframePageDisplayEnabled:0, IsEmbeddedMediaEnabled:1, IsImageModeAnnotationsEnabled:1, IsMyLibraryGooglePlusEnabled:1, IsImagePageProviderEnabled:0, IsBookcardListPriceSmall:0, IsInternalUser:0, IsBooksShareButtonEnabled:0, IsPreOrdersEnabled:0, IsDisabledRandomBookshelves:0, WebstoreDisplayCasePosition:3});_OC_Run({"enable_p13n":false,"add_vol_to_collection_base_url":"http://www.google.com/patents?op=add\u0026sig=ACfU3U3-4m8OtmauQOW0hzTqCUhlEYTgnQ\u0026id=s1BcBAABERAJ","remove_vol_from_collection_base_url":"http://www.google.com/patents?op=remove\u0026sig=ACfU3U1w7zM2YIMxl8MF48HLtfUtpT9RyA\u0026id=s1BcBAABERAJ","logged_in":false,"p13n_save_user_settings_url":"http://www.google.com/patents?op=edit_user_settings\u0026sig=ACfU3U0uqd2fmDtT_7DC-Cz5OzZ1hpgEjA","is_cobrand":false,"sign_in_url":"https://www.google.com/accounts/Login?service=\u0026continue=http://www.google.com/patents%3Fhl%3Den\u0026hl=en","is_play_enabled":true}, {"volume_id":"","is_ebook":true,"volumeresult":{"has_flowing_text":false,"has_scanned_text":true,"can_download_pdf":false,"can_download_epub":false,"is_pdf_drm_enabled":false,"is_epub_drm_enabled":false,"download_pdf_url":"http://www.google.com/patents/download/DNA_construct_for_producing_proteinaceou.pdf?id=s1BcBAABERAJ\u0026output=pdf\u0026sig=ACfU3U3J0YOjV5nEGgbEreDVggLemv4oDQ"},"sample_url":"http://www.google.com/patents/reader?id=s1BcBAABERAJ\u0026printsec=frontcover\u0026output=reader\u0026source=gbs_atb_hover","is_browsable":true,"is_public_domain":true}, {});</script><div id="footer_table" style="font-size:83%;text-align:center;position:relative;top:20px;height:4.5em;margin-top:2em"><div style="margin-bottom:8px"><a href=http://www.google.com/><nobr>Google&nbsp;Home</nobr></a> - <a href=//www.google.com/patents/sitemap/><nobr>Sitemap</nobr></a> - <a href=http://www.google.com/googlebooks/uspto.html><nobr>USPTO Bulk Downloads</nobr></a> - <a href=/intl/en/privacy/><nobr>Privacy Policy</nobr></a> - <a href=/intl/en/policies/terms/><nobr>Terms of Service</nobr></a> - <a href=https://support.google.com/faqs/answer/2539193?hl=en><nobr>About Google Patents</nobr></a> - <a href="http://www.google.com/tools/feedback/intl/en/error.html" onclick="try{_OC_startFeedback({productId: '72792',locale: 'en'});return false;}catch(e){}"><nobr>Send Feedback</nobr></a></div><span>Data provided by IFI CLAIMS Patent Services</span><br><span >&copy;2012 Google</span></div> <script type="text/javascript">var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));</script><script type="text/javascript">var pageTracker = _gat._getTracker("UA-27188110-1");pageTracker._setCookiePath("/patents/");pageTracker._trackPageview();</script> </body></html>