<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN""http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html><head><title>Patent US6472164 - Method for evaluating the ability of a compound to inhibit the ... - Google Patents</title><script>(function(){(function(){function e(a){this.t={};this.tick=function(a,c,b){var d=void 0!=b?b:(new Date).getTime();this.t[a]=[d,c];if(void 0==b)try{window.console.timeStamp("CSI/"+a)}catch(e){}};this.tick("start",null,a)}var a;window.performance&&(a=window.performance.timing);var f=a?new e(a.responseStart):new e;window.jstiming={Timer:e,load:f};if(a){var c=a.navigationStart,d=a.responseStart;0<c&&d>=c&&(window.jstiming.srt=d-c)}if(a){var b=window.jstiming.load;0<c&&d>=c&&(b.tick("_wtsrt",void 0,c),b.tick("wtsrt_",
"_wtsrt",d),b.tick("tbsd_","wtsrt_"))}try{a=null,window.chrome&&window.chrome.csi&&(a=Math.floor(window.chrome.csi().pageT),b&&0<c&&(b.tick("_tbnd",void 0,window.chrome.csi().startE),b.tick("tbnd_","_tbnd",c))),null==a&&window.gtbExternal&&(a=window.gtbExternal.pageT()),null==a&&window.external&&(a=window.external.pageT,b&&0<c&&(b.tick("_tbnd",void 0,window.external.startE),b.tick("tbnd_","_tbnd",c))),a&&(window.jstiming.pt=a)}catch(g){}})();})();
</script><link rel="stylesheet" href="/patents/css/_8a2b04e7bf975d5171d8e4c0b6365c7a/kl_intl_patents_bundle.css" type="text/css" /><script src="/books/javascript/atb_8a2b04e7bf975d5171d8e4c0b6365c7a__en.js"></script><script>function googleTranslateElementInit() {new google.translate.TranslateElement({pageLanguage: "en",gaTrack: true,gaId: "UA-27188110-1",multilanguagePage: true});}</script><script src="//translate.google.com/translate_a/element.js?cb=googleTranslateElementInit"></script><meta name="DC.type" content="Patent"><meta name="DC.title" content="Method for evaluating the ability of a compound to inhibit the protoporphyrinogen oxidase activity"><meta name="DC.contributor" content="Yasutaka Shimokawatoko" scheme="inventor"><meta name="DC.contributor" content="Shoichi Nishio" scheme="inventor"><meta name="DC.contributor" content="Sumitomo Chemical Company, Limited" scheme="assignee"><meta name="DC.date" content="1999-4-9" scheme="dateSubmitted"><meta name="DC.description" content="The present invention provides a method for evaluating the ability of a compound to inhibit protoporphyrinogen oxidase activity with a host cell deficient in protoporphyrinogen oxidase production which has been transformed with a vector comprising a DNA fragment encoding a protoporphyrinogen oxidase. Transformed cells are grown in protoheme-free medium in the presence and absence of test compounds and growth rates measured under these condition are compared to determine if inhibition has occurred."><meta name="DC.date" content="2002-10-29" scheme="issued"><meta name="DC.relation" content="US:5767373" scheme="references"><meta name="DC.relation" content="US:5939602" scheme="references"><meta name="DC.relation" content="US:6018105" scheme="references"><meta name="DC.relation" content="US:6023012" scheme="references"><meta name="DC.relation" content="WO:1997004089:A2" scheme="references"><meta name="DC.relation" content="WO:1998029554:A1" scheme="references"><meta name="citation_reference" content="Fumiaki Yamao et al., Jpn. J. Genet., vol. 63, 1988, pp. 237-249."><meta name="citation_reference" content="Jean-Michel Camadro et al., Biochemical and Biophysical Research Communications, vol. 106, No. 3, 1982, pp. 724-730."><meta name="citation_reference" content="Kazumasa Miyamoto et al., J. Mol. Biol., vol. 219, 1991, pp. 393-398."><meta name="citation_reference" content="Koichi Nishimura et al., Gene, vol. 133, 1993, pp. 109-113."><meta name="citation_patent_number" content="US:6472164"><meta name="citation_patent_application_number" content="US:09/289,180"><link rel="canonical" href="http://www.google.com/patents/US6472164"/><meta property="og:url" content="http://www.google.com/patents/US6472164"/><meta name="title" content="Patent US6472164 - Method for evaluating the ability of a compound to inhibit the protoporphyrinogen oxidase activity"/><meta name="description" content="The present invention provides a method for evaluating the ability of a compound to inhibit protoporphyrinogen oxidase activity with a host cell deficient in protoporphyrinogen oxidase production which has been transformed with a vector comprising a DNA fragment encoding a protoporphyrinogen oxidase. Transformed cells are grown in protoheme-free medium in the presence and absence of test compounds and growth rates measured under these condition are compared to determine if inhibition has occurred."/><meta property="og:title" content="Patent US6472164 - Method for evaluating the ability of a compound to inhibit the protoporphyrinogen oxidase activity"/><meta property="og:type" content="book"/><meta property="og:site_name" content="Google Books"/><meta property="og:image" content="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><link rel="image_src" href="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><script>(function(){try{var aa=function(a,b,c,d){d=d||{};d._sn=["cfg",b,c].join(".");window.gbar.logger.ml(a,d)};var g=window.gbar=window.gbar||{},l=window.gbar.i=window.gbar.i||{},m={},n;function _tvn(a,b){var c=parseInt(a,10);return isNaN(c)?b:c}function _tvf(a,b){var c=parseFloat(a);return isNaN(c)?b:c}function _tvv(a){return!!a}function p(a,b,c){(c||g)[a]=b}g.bv={n:_tvn("2",0),r:"",f:".67.",e:"0",m:_tvn("0",1)};
function q(a,b,c){var d="on"+b;if(a.addEventListener)a.addEventListener(b,c,!1);else if(a.attachEvent)a.attachEvent(d,c);else{var f=a[d];a[d]=function(){var a=f.apply(this,arguments),b=c.apply(this,arguments);return void 0==a?b:void 0==b?a:b&&a}}}var s=function(a){return function(){return g.bv.m==a}},ba=s(1),ca=s(2);p("sb",ba);p("kn",ca);l.a=_tvv;l.b=_tvf;l.c=_tvn;l.i=aa;var da=window.gbar.i.i;var t,u,v,w;function ea(a){v=a}function fa(a){var b;if(b=v&&window.encodeURIComponent)b=a.href,b=!b.match(/^http[s]?:\/\/accounts\.google\.[^/]*\/ClearSID/i)&&!b.match(/^http[s]?:\/\/[^/]*\/accounts\/ClearSID/i);if(b=b&&encodeURIComponent(v()))a.href=a.href.replace(/([?&]continue=)[^&]*/,"$1"+b)}function ga(a){window.gApplication&&(a.href=window.gApplication.getTabUrl(a.href))}
function ha(a){var b=document.forms[0].q,c=window.encodeURIComponent&&b&&b.value,b=b&&b.placeholder;c&&c!=b&&(a.href=a.href.replace(/([?&])q=[^&]*|$/,function(a,b){return(b||"&")+"q="+encodeURIComponent(c)}))}n=l.a("")?ga:ha;
function x(a,b,c,d,f,e){var h=document.getElementById(a);if(h){var k=h.style;k.left=d?"auto":b+"px";k.right=d?b+"px":"auto";k.top=c+"px";k.visibility=u?"hidden":"visible";f&&e?(k.width=f+"px",k.height=e+"px"):(x(t,b,c,d,h.offsetWidth,h.offsetHeight),u=u?"":a)}}
var y=[],ia=function(a,b){y.push(b)},ja=function(a){a=a||window.event;var b=a.target||a.srcElement;a.cancelBubble=!0;null==t&&(a=document.createElement(Array.every||window.createPopup?"iframe":"div"),a.frameBorder="0",t=a.id="gbs",a.src="javascript:''",b.parentNode.appendChild(a),q(document,"click",z));var c=b,b=0;"gb3"!=c.className&&(c=c.parentNode);a=c.getAttribute("aria-owns")||"gbi";var d=c.offsetWidth,f=20<c.offsetTop?46:24;document.getElementById("tphdr")&&(f-=3);var e=!1;do b+=c.offsetLeft||
0;while(c=c.offsetParent);var c=(document.documentElement.clientWidth||document.body.clientWidth)-b-d,h,d=document.body,k=document.defaultView;k&&k.getComputedStyle?(d=k.getComputedStyle(d,""))&&(h=d.direction):h=d.currentStyle?d.currentStyle.direction:d.style.direction;h="rtl"==h;if("gbi"==a){for(d=0;k=y[d++];)k();A(null,window.navExtra);h&&(b=c,e=!0)}else h||(b=c,e=!0);u!=a&&z();x(a,b,f,e)},z=function(){u&&x(u,0,0)},A=function(a,b){var c,d=document.getElementById("gbi"),f=a;f||(f=d.firstChild);
for(;b&&(c=b.pop());){var e=d,h=c,k=f;w||(w="gb2");e.insertBefore(h,k).className=w}},ka=function(a,b,c){if((b=document.getElementById(b))&&a){a.className="gb4";var d=document.createElement("span");d.appendChild(a);d.appendChild(document.createTextNode(" | "));d.id=c;b.appendChild(d)}},la=function(){return document.getElementById("gb_70")},ma=function(){return!!u};p("qs",n);p("setContinueCb",ea);p("pc",fa);p("tg",ja);p("close",z);p("addLink",ka);p("almm",A);p("si",la);p("adh",ia);p("op",ma);var B=function(){},C=function(){},F=function(a){var b=new Image,c=D;b.onerror=b.onload=b.onabort=function(){try{delete E[c]}catch(a){}};E[c]=b;b.src=a;D=c+1},E=[],D=0;p("logger",{il:C,ml:B,log:F});var G=window.gbar.logger;var H={},na={},I=[],oa=l.b("0.1",.1),pa=l.a("1",!0),qa=function(a,b){I.push([a,b])},ra=function(a,b){H[a]=b},sa=function(a){return a in H},J={},K=function(a,b){J[a]||(J[a]=[]);J[a].push(b)},ta=function(a){K("m",a)},L=function(a,b){var c=document.createElement("script");c.src=a;c.async=pa;Math.random()<oa&&(c.onerror=function(){c.onerror=null;B(Error("Bundle load failed: name="+(b||"UNK")+" url="+a))});(document.getElementById("xjsc")||document.getElementsByTagName("body")[0]||
document.getElementsByTagName("head")[0]).appendChild(c)},N=function(a){for(var b=0,c;(c=I[b])&&c[0]!=a;++b);!c||c[1].l||c[1].s||(c[1].s=!0,M(2,a),c[1].url&&L(c[1].url,a),c[1].libs&&m.d&&m.d(c[1].libs))},O=function(a){K("gc",a)},P=null,ua=function(a){P=a},M=function(a,b,c){if(P){a={t:a,b:b};if(c)for(var d in c)a[d]=c[d];try{P(a)}catch(f){}}};p("mdc",H);p("mdi",na);p("bnc",I);p("qGC",O);p("qm",ta);p("qd",J);p("lb",N);p("mcf",ra);p("bcf",qa);p("aq",K);p("mdd","");p("has",sa);
p("trh",ua);p("tev",M);var Q=l.b("0.1",.001),R=0;
function _mlToken(a,b){try{if(1>R){R++;var c,d=a,f=b||{},e=encodeURIComponent,h=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&jexpid=",e("17483"),"&srcpg=",e("prop=22"),"&jsr=",Math.round(1/Q),"&ogev=",e("A6rtU6q2J8bjsASd-oCQCw"),"&ogf=",g.bv.f,"&ogrp=",e("1"),"&ogv=",e("1407723702.0"),"&oggv="+e("es_plusone_gc_20140723.0_p0"),"&ogd=",e("com"),"&ogc=",e("NOR"),"&ogl=",e("en")];f._sn&&(f._sn="og."+
f._sn);for(var k in f)h.push("&"),h.push(e(k)),h.push("="),h.push(e(f[k]));h.push("&emsg=");h.push(e(d.name+":"+d.message));var r=h.join("");S(r)&&(r=r.substr(0,2E3));c=r;var Aa=window.gbar.logger._aem(a,c);F(Aa)}}catch(Na){}}var S=function(a){return 2E3<=a.length},va=function(a,b){return b};function T(a){B=a;p("_itl",S,G);p("_aem",va,G);p("ml",B,G);a={};H.er=a}l.a("")?T(function(a){throw a;}):l.a("1")&&Math.random()<Q&&T(_mlToken);I.push(["m",{url:"//ssl.gstatic.com/gb/js/scm_7385cc5883250b43a39405734c1bea59.js"}]);g.mcf("c",{});g.sg={c:""};if(l.a("1")){var wa=l.a("");I.push(["gc",{auto:wa,url:"//ssl.gstatic.com/gb/js/abc/gci_91f30755d6a6b787dcc2a4062e6e9824.js",libs:"googleapis.client:plusone:gapi.iframes"}]);var xa={version:"gci_91f30755d6a6b787dcc2a4062e6e9824.js",index:"",lang:"en"};H.gc=xa;var U=function(a){window.googleapis&&window.iframes?a&&a():(a&&O(a),N("gc"))};p("lGC",U);l.a("1")&&p("lPWF",U)};window.__PVT="";if(l.a("1")&&l.a("1")){var V=function(a){U(function(){K("pw",a);N("pw")})};p("lPW",V);I.push(["pw",{url:"//ssl.gstatic.com/gb/js/abc/pwm_45f73e4df07a0e388b0fa1f3d30e7280.js"}]);var W=[],ya=function(a){W[0]=a},za=function(a,b){var c=b||{};c._sn="pw";B(a,c)},Ba={signed:W,elog:za,base:"https://plusone.google.com/u/0",loadTime:(new Date).getTime()};H.pw=Ba;var X=function(a,b){for(var c=b.split("."),d=function(){var b=arguments;a(function(){for(var a=g,d=0,e=c.length-1;d<e;++d)a=a[c[d]];a[c[d]].apply(a,b)})},f=g,e=0,h=c.length-1;e<h;++e)f=
f[c[e]]=f[c[e]]||{};return f[c[e]]=d};X(V,"pw.clk");X(V,"pw.hvr");p("su",ya,g.pw)};function Ca(){function a(){for(var b;(b=e[h++])&&"m"!=b[0]&&!b[1].auto;);b&&(M(2,b[0]),b[1].url&&L(b[1].url,b[0]),b[1].libs&&m.d&&m.d(b[1].libs));h<e.length&&setTimeout(a,0)}function b(){0<f--?setTimeout(b,0):a()}var c=l.a("1"),d=l.a(""),f=3,e=I,h=0,k=window.gbarOnReady;if(k)try{k()}catch(r){da(r,"ml","or")}d?p("ldb",a):c?q(window,"load",b):b()}p("rdl",Ca);var Da={D:1,H:2,da:3,p:4,W:5,M:6,F:7,g:8,ha:9,U:10,L:11,T:12,S:13,N:14,Q:15,P:16,fa:17,w:18,O:19,ga:20,ea:21,u:22,G:23,ja:24,ka:25,ia:26,A:27,j:28,o:29,k:30,ca:31,Z:32,$:33,J:34,K:35,ba:36,aa:37,Y:38,B:39,R:40,v:41,X:42,V:43,h:48,C:49,I:500},Y=[1,2,3,4,5,6,9,10,11,13,14,28,29,30,34,35,37,38,39,40,41,42,43,48,49,500];var Z=l.b("0.001",1E-4),Ea=l.b("1",1),Fa=!1,Ga=!1;if(l.a("1")){var Ha=Math.random();Ha<=Z&&(Fa=!0);Ha<=Ea&&(Ga=!0)}var Ia=Da,$=null;function Ja(){var a=0,b=function(b,d){l.a(d)&&(a|=b)};b(1,"");b(2,"");b(4,"");b(8,"");return a}
function Ka(a,b){var c=Z,d=Fa,f;f=a;if(!$){$={};for(var e=0;e<Y.length;e++){var h=Y[e];$[h]=!0}}if(f=!!$[f])c=Ea,d=Ga;if(d){d=encodeURIComponent;g.rp?(f=g.rp(),f="-1"!=f?f:"1"):f="1";c=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&oge=",a,"&ogex=",d("17483"),"&ogev=",d("A6rtU6q2J8bjsASd-oCQCw"),"&ogf=",g.bv.f,"&ogp=",d("22"),"&ogrp=",d(f),"&ogsr=",Math.round(1/c),"&ogv=",d("1407723702.0"),"&oggv="+
d("es_plusone_gc_20140723.0_p0"),"&ogd=",d("com"),"&ogl=",d("en"),"&ogc=",d("NOR"),"&ogus=",Ja()];if(b){"ogw"in b&&(c.push("&ogw="+b.ogw),delete b.ogw);var k;f=b;e=[];for(k in f)0!=e.length&&e.push(","),e.push(La(k)),e.push("."),e.push(La(f[k]));k=e.join("");""!=k&&(c.push("&ogad="),c.push(d(k)))}F(c.join(""))}}function La(a){"number"==typeof a&&(a+="");return"string"==typeof a?a.replace(".","%2E").replace(",","%2C"):a}C=Ka;p("il",C,G);var Ma={};H.il=Ma;setTimeout(function(){C(Ia.g)},0);}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var b=window.gbar.i.i;var c=window.gbar;var f=function(d){try{var a=document.getElementById("gbom");a&&d.appendChild(a.cloneNode(!0))}catch(e){b(e,"omas","aomc")}};c.aomc=f;}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var a=window.gbar;a.mcf("pm",{p:""});}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{window.gbar.rdl();}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
if (window['_OC_timingAction']) {window['_OC_timingAction']('patents_refpage');}</script><style>#gbar,#guser{font-size:13px;padding-top:1px !important;}#gbar{float:left;height:22px}#guser{padding-bottom:7px !important;text-align:right}.gbh,.gbd{border-top:1px solid #c9d7f1;font-size:1px}.gbh{height:0;position:absolute;top:24px;width:100%}#gbs,.gbm{background:#fff;left:0;position:absolute;text-align:left;visibility:hidden;z-index:1000}.gbm{border:1px solid;border-color:#c9d7f1 #36c #36c #a2bae7;z-index:1001}.gb1{margin-right:.5em}.gb1,.gb3{zoom:1}.gb2{display:block;padding:.2em .5em}.gb2,.gb3{text-decoration:none !important;border-bottom:none}a.gb1,a.gb4{text-decoration:underline !important}a.gb1,a.gb2,a.gb3,a.gb4{color:#00c !important}.gbi .gb3,.gbi .gb2,.gbi .gb4{color:#dd8e27 !important}.gbf .gb3,.gbf .gb2,.gbf .gb4{color:#900 !important}a.gb2:hover{background:#36c;color:#fff !important}#gbar .gbz0l{color:#000 !important;cursor:default;font-weight:bold;text-decoration:none !important}
#gbar { padding:.3em .6em !important;}</style></head><body  topmargin="3" marginheight="3"><div id=gbar><nobr><a onclick=gbar.qs(this);gbar.logger.il(1,{t:1}); class=gb1 id=gb_1 href="https://www.google.com/search?sa=N&tab=tw">Search</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:2}); class=gb1 id=gb_2 href="http://www.google.com/search?hl=en&tbm=isch&source=og&sa=N&tab=ti">Images</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:8}); class=gb1 id=gb_8 href="http://maps.google.com/maps?hl=en&sa=N&tab=tl">Maps</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:78}); class=gb1 id=gb_78 href="https://play.google.com/?hl=en&sa=N&tab=t8">Play</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:36}); class=gb1 id=gb_36 href="http://www.youtube.com/results?sa=N&tab=t1">YouTube</a> <a onclick=gbar.logger.il(1,{t:5}); class=gb1 id=gb_5 href="http://news.google.com/nwshp?hl=en&tab=tn">News</a> <a onclick=gbar.logger.il(1,{t:23}); class=gb1 id=gb_23 href="https://mail.google.com/mail/?tab=tm">Gmail</a> <a onclick=gbar.logger.il(1,{t:25}); class=gb1 id=gb_25 href="https://drive.google.com/?tab=to">Drive</a> <a class=gb3 href="http://www.google.com/intl/en/options/" onclick="this.blur();gbar.tg(event);return !1" aria-haspopup=true><u>More</u> <small>&#9660;</small></a><div class=gbm id=gbi><a onclick=gbar.logger.il(1,{t:24}); class=gb2 id=gb_24 href="https://www.google.com/calendar?tab=tc">Calendar</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:51}); class=gb2 id=gb_51 href="http://translate.google.com/?hl=en&sa=N&tab=tT">Translate</a><a onclick=gbar.logger.il(1,{t:17}); class=gb2 id=gb_17 href="http://www.google.com/mobile/?hl=en&tab=tD">Mobile</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:10}); class=gb2 id=gb_10 href="http://www.google.com/search?hl=en&tbo=u&tbm=bks&source=og&sa=N&tab=tp">Books</a><a onclick=gbar.logger.il(1,{t:212}); class=gb2 id=gb_212 href="https://wallet.google.com/manage/?tab=ta">Wallet</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:6}); class=gb2 id=gb_6 href="http://www.google.com/search?hl=en&tbo=u&tbm=shop&source=og&sa=N&tab=tf">Shopping</a><a onclick=gbar.logger.il(1,{t:30}); class=gb2 id=gb_30 href="http://www.blogger.com/?tab=tj">Blogger</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:27}); class=gb2 id=gb_27 href="http://www.google.com/finance?sa=N&tab=te">Finance</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:31}); class=gb2 id=gb_31 href="https://plus.google.com/photos?sa=N&tab=tq">Photos</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:12}); class=gb2 id=gb_12 href="http://www.google.com/search?hl=en&tbo=u&tbm=vid&source=og&sa=N&tab=tv">Videos</a><div class=gb2><div class=gbd></div></div><a onclick=gbar.logger.il(1,{t:66}); href="http://www.google.com/intl/en/options/" class=gb2>Even more &raquo;</a></div></nobr></div><div id=guser width=100%><nobr><span id=gbn class=gbi></span><span id=gbf class=gbf></span><span id=gbe></span><a target=_top id=gb_70 href="https://www.google.com/accounts/Login?service=&continue=http://www.google.com/patents%3Fhl%3Den&hl=en" class=gb4>Sign in</a><div style="display: none"><div class=gbm id=gbd5 aria-owner=gbg5><div class=gbmc><ol id=gbom class=gbmcc></ol></div></div></div></nobr></div><div class=gbh style=left:0></div><div class=gbh style=right:0></div><div role="alert" style="position: absolute; left: 0; right: 0;"><a href="http://www.google.com/patents/us6472164?hl=en&amp;output=html_text" title="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."><img border="0" src="http://www.google.com/images/cleardot.gif"alt="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."></a></div><div id="guser"><nobr></nobr></div><div style="clear:both;"></div><div id="gb-top-search-box" class="gb-top-search-box-small gb-reset"><table><tr><td class="logo"><a href="http://www.google.com/patents" class="logo-link"><img class="logo-img" src="/intl/en/images/logos/google_logo_41.png" alt="Go to Google Books Home" height="41"/></a></td><td><form action="http://www.google.com/search" name="f" id="vheadf" method="get"><span id="hf"></span><input type="hidden" name="tbm" value="pts"/><input type="hidden" name="tbo" value="1"/><input type="hidden" name="hl" value="en"/><table><tr><td><div class="inputs"><table><tr><td><div class="text-input"><input type="text" name="q" id="vheadq" class="text" maxlength="2048" size="31" value="" title="Search Patents" accesskey="s" autocomplete="off"/><script>window._OC_autoDir &&window._OC_autoDir('vheadq', 'tia-vheadq');</script></div></td><td><div class="submit-input"><input name="btnG" class="submit" type="submit" value=""/></div></td></tr></table></div></td><td class="col-ext-links"><div class="ext-links"><a href="http://www.google.com/advanced_patent_search">&lt;nobr&gt;Advanced Patent Search&lt;/nobr&gt;</a></div></td></tr></table></form></td></tr></table></div><div class="kd-appbar"><h2 class="kd-appname"><a href="/patents">Patents</a></h2><div class="kd-buttonbar left" id="left-toolbar-buttons"><a id="appbar-write-review-link" href=""></a><a id="appbar-view-print-sample-link" href=""></a><a id="appbar-view-ebook-sample-link" href=""></a><a id="appbar-patents-prior-art-finder-link" href="https://www.google.com/patents/related/US6472164"></a><a id="appbar-patents-discuss-this-link" href="http://www.google.com/url?id=aJVcBAABERAJ&amp;q=http://patents.stackexchange.com/redirect/google-patents%3Fpatent%3DUS6472164&amp;usg=AFQjCNGGc9BIYfNNuZryuisXwOIktM2ZFQ" data-is-grant="true"></a><a id="appbar-read-patent-link" href="//docs.google.com/viewer?url=patentimages.storage.googleapis.com/pdfs/US6472164.pdf"></a><a id="appbar-download-pdf-link" href="//patentimages.storage.googleapis.com/pdfs/US6472164.pdf"></a></div><div class="kd-buttonbar right" id="right-toolbar-buttons"></div></div><div id="books-microdata" itemscope=""itemtype="http://schema.org/Book"itemid="http://www.google.com/patents/US6472164" style="display:none"><span itemprop="description">The present invention provides a method for evaluating the ability of a compound to inhibit protoporphyrinogen oxidase activity with a host cell deficient in protoporphyrinogen oxidase production which has been transformed with a vector comprising a DNA fragment encoding a protoporphyrinogen oxidase....</span><span itemprop="url">http://www.google.com/patents/US6472164?utm_source=gb-gplus-share</span><span class="main-title" itemprop="name">Patent US6472164 - Method for evaluating the ability of a compound to inhibit the protoporphyrinogen oxidase activity</span><img itemprop="image" src="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"alt="Patent US6472164 - Method for evaluating the ability of a compound to inhibit the protoporphyrinogen oxidase activity" title="Patent US6472164 - Method for evaluating the ability of a compound to inhibit the protoporphyrinogen oxidase activity"></div><div style="display: none"><ol id="ofe-gear-menu-contents" class="gbmcc"><li class="gbe gbmtc"><a class="gbmt goog-menuitem-content" id="" href="http://www.google.com/advanced_patent_search">Advanced Patent Search</a></li></ol></div><table id="viewport_table" cellpadding="0" style="clear:both" cellspacing="0"><tr><td id="viewport_td"><div class=vertical_module_list_row><div id=intl_patents class=about_content><div id=intl_patents_v><table class="patent-bibdata"><tr><td class="patent-bibdata-heading">Publication number</td><td class="single-patent-bibdata">US6472164 B1</td></tr><tr><td class="patent-bibdata-heading">Publication type</td><td class="single-patent-bibdata">Grant</td></tr><tr><td class="patent-bibdata-heading">Application number</td><td class="single-patent-bibdata">US 09/289,180</td></tr><tr><td class="patent-bibdata-heading">Publication date</td><td class="single-patent-bibdata">Oct 29, 2002</td></tr><tr><td class="patent-bibdata-heading">Filing date</td><td class="single-patent-bibdata">Apr 9, 1999</td></tr><tr><td class="patent-bibdata-heading">Priority date<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The priority date is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the date listed."></span></td><td class="single-patent-bibdata">Apr 10, 1998</td></tr><tr><td class="patent-bibdata-heading">Fee status<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The fee status is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the status or dates listed."></span></td><td class="single-patent-bibdata">Paid</td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Also published as</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/CA2266807A1">CA2266807A1</a>, </span><span class="patent-bibdata-value"><a href="/patents/CA2266807C">CA2266807C</a>, </span><span class="patent-bibdata-value"><a href="/patents/DE69937423D1">DE69937423D1</a>, </span><span class="patent-bibdata-value"><a href="/patents/DE69937423T2">DE69937423T2</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP0955380A2">EP0955380A2</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP0955380A3">EP0955380A3</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP0955380B1">EP0955380B1</a>, </span><span class="patent-bibdata-value"><a href="/patents/US6830926">US6830926</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20020086395">US20020086395</a></span></span></td></tr><tr class="patent-bibdata-list-row alternate-patent-number"><td class="patent-bibdata-heading">Publication number</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value">09289180, </span><span class="patent-bibdata-value">289180, </span><span class="patent-bibdata-value">US 6472164 B1, </span><span class="patent-bibdata-value">US 6472164B1, </span><span class="patent-bibdata-value">US-B1-6472164, </span><span class="patent-bibdata-value">US6472164 B1, </span><span class="patent-bibdata-value">US6472164B1</span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Inventors</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Yasutaka+Shimokawatoko%22">Yasutaka Shimokawatoko</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Shoichi+Nishio%22">Shoichi Nishio</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Original Assignee</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=inassignee:%22Sumitomo+Chemical+Company,+Limited%22">Sumitomo Chemical Company, Limited</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Export Citation</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/US6472164.bibtex">BiBTeX</a>, </span><span class="patent-bibdata-value"><a href="/patents/US6472164.enw">EndNote</a>, </span><span class="patent-bibdata-value"><a href="/patents/US6472164.ris">RefMan</a></span></span></td></tr><tr class="patent-internal-links"><td colspan=2><span class="patent-bibdata-value"><a href="#backward-citations">Patent Citations</a> (6),</span> <span class="patent-bibdata-value"><a href="#npl-citations">Non-Patent Citations</a> (4),</span> <span class="patent-bibdata-value"><a href="#classifications">Classifications</a> (12),</span> <span class="patent-bibdata-value"><a href="#legal-events">Legal Events</a> (6)</span> </td></tr><tr><td colspan=2 class="patent-bibdata-external-link-spacer-top"></td></tr><tr class="patent-bibdata-external-link-spacer-bottom"></tr><tr><td colspan=2><span class="patent-bibdata-heading">External Links:&nbsp;</span><span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=aJVcBAABERAJ&q=http://patft.uspto.gov/netacgi/nph-Parser%3FSect2%3DPTO1%26Sect2%3DHITOFF%26p%3D1%26u%3D/netahtml/PTO/search-bool.html%26r%3D1%26f%3DG%26l%3D50%26d%3DPALL%26RefSrch%3Dyes%26Query%3DPN/6472164&usg=AFQjCNE3_w2HPbZ9N9dPx2-y9SeuUT44iQ">USPTO</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=aJVcBAABERAJ&q=http://assignments.uspto.gov/assignments/q%3Fdb%3Dpat%26pat%3D6472164&usg=AFQjCNHS59lkfYCjgQELWA8G6VdnmPcOhg">USPTO Assignment</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=aJVcBAABERAJ&q=http://worldwide.espacenet.com/publicationDetails/biblio%3FCC%3DUS%26NR%3D6472164B1%26KC%3DB1%26FT%3DD&usg=AFQjCNFbZ3sLsY7rJXXYMo4w38SHVPV0nQ">Espacenet</a></span></span></td></tr><tr class="patent-bibdata-group-spacer"></tr></table><div class="number-and-title"><span class="patent-title"><invention-title mxw-id="PT54990794" lang="EN" load-source="patent-office">Method for evaluating the ability of a compound to inhibit the protoporphyrinogen oxidase activity</invention-title></span><br><span class="patent-number">US 6472164 B1</span></div><div class="patent-section patent-abstract-section"><div class="patent-section-header"><span class="patent-section-title">Abstract</span></div><div class="patent-text"><abstract mxw-id="PA50400465" lang="EN" load-source="patent-office"> <div class="abstract">The present invention provides a method for evaluating the ability of a compound to inhibit protoporphyrinogen oxidase activity with a host cell deficient in protoporphyrinogen oxidase production which has been transformed with a vector comprising a DNA fragment encoding a protoporphyrinogen oxidase. Transformed cells are grown in protoheme-free medium in the presence and absence of test compounds and growth rates measured under these condition are compared to determine if inhibition has occurred.</div>
  </abstract></div></div><div class="patent-section patent-drawings-section"><div class="patent-section-header"><span class="patent-section-title">Images<span class="patent-section-count">(4)</span></span></div><div class="patent-drawings-body"><div class="patent-drawings-carousel"><div class="drawings"><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6472164B1/US06472164-20021029-D00000.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6472164B1/US06472164-20021029-D00000.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6472164B1/US06472164-20021029-D00001.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6472164B1/US06472164-20021029-D00001.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6472164B1/US06472164-20021029-D00002.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6472164B1/US06472164-20021029-D00002.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6472164B1/US06472164-20021029-D00003.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6472164B1/US06472164-20021029-D00003.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div></div></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img" alt="Previous page"src="/googlebooks/images/kennedy/page_left.png"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img" alt="Next page"src="/googlebooks/images/kennedy/page_right.png"width="21" height="21" /></div></div></div><div class="patent-post-drawings"></div><div class="patent-section patent-claims-section"><div class="patent-section-header"><span class="patent-section-title">Claims<span class="patent-section-count">(6)</span></span></div><div class="patent-text"><div mxw-id="PCLM8381297" lang="EN" load-source="patent-office" class="claims">
    <claim-statement>What is claimed is: </claim-statement> <div class="claim"> <div num="1" id="US-6472164-B1-CLM-00001" class="claim">
      <div class="claim-text">1. A method for evaluating the ability of a compound to inhibit protoporphyrinogen oxidase activity, comprising the steps of:</div>
      <div class="claim-text">(1) transforming with a vector a host cell deficient in growing ability based on protoporphyrinogen oxidase activity, said vector comprising a DNA fragment coding for enzyme protoporphyrinogen oxidase which is capable of oxidizing protoporphyrinogen into protoporphyrin and which confers growth ability, wherein said DNA fragment is operably linked to a promoter functional in said host cell; </div>
      <div class="claim-text">(2) culturing said transformant expressing said protoporphyrinogen oxidase DNA in a medium containing substantially no protoheme compounds, wherein in a first comparative system there is a presence of a test compound to measure a growth rate of the transformant and in a second comparative system there is an absence of said test compound; and </div>
      <div class="claim-text">(3) determining the ability of the test compound to inhibit the protoporphyrinogen oxidase activity by comparing the growth rates of the first comparative system to the second comparative system, wherein an inhibition of the growth rate is indicative of an inhibition of protoporphyrinogen oxidase activity by said test compound. </div>
    </div>
    </div> <div class="claim"> <div num="2" id="US-6472164-B1-CLM-00002" class="claim">
      <div class="claim-text">2. A method for evaluating the ability of a compound to inhibit protoporphyrinogen oxidase activity, comprising the steps of:</div>
      <div class="claim-text">(1) transforming with a vector a host cell deficient in growing ability based on protoporphyrinogen oxidase activity, said vector comprising a DNA fragment coding for enzyme protoporphyrinogen oxidase which is capable of oxidizing protoporphyrinogen into protoporphyrin and which confers growth ability, wherein said DNA fragment is operably linked to a promoter functional in said host cell, and a terminator functional in the host cell; </div>
      <div class="claim-text">(2) culturing said transformant expressing said protoporphyrinogen oxidase DNA in a medium containing substantially no protoheme compounds, wherein in a first comparative system there is a presence of a test compound to measure a growth rate of the transformant and in a second comparative system there is an absence of said test compound; and </div>
      <div class="claim-text">(3) determining the ability of the test compound to inhibit the protoporphyrinogen oxidase activity by comparing the growth rates of the first comparative system to the second comparative system, wherein an inhibition of the growth rate is indicative of an inhibition of protoporphyrinogen oxidase activity by said test compound. </div>
    </div>
    </div> <div class="claim"> <div num="3" id="US-6472164-B1-CLM-00003" class="claim">
      <div class="claim-text">3. A method for evaluating the ability of a compound to inhibit protoporphyrinogen oxidase activity, comprising the steps of:</div>
      <div class="claim-text">(1) transforming with a vector a host cell deficient in growing ability based on protoporphyrinogen oxidase activity, said vector comprising a DNA fragment coding for enzyme protoporphyrinogen oxidase which is capable of oxidizing protoporphyrinogen into protoporphyrin and which confers growth ability, wherein said DNA fragment is operably linked to a promoter functional in said host cell, wherein said promoter is inducible, and a second vector comprising a second DNA fragment which is a DNA capable of inducing the promoter of the first DNA fragment, and a promoter, wherein said promoter is not induced by the second DNA fragment but is functional in the host cell, are operatively linked; </div>
      <div class="claim-text">(2) culturing said transformant expressing said protoporphyrinogen oxidase DNA in a medium containing substantially no protoheme compounds, wherein in a first comparative system there is a presence of a test compound to measure a growth rate of the transformant and in a second comparative system there is an absence of said test compound; and </div>
      <div class="claim-text">(3) determining the ability of the test compound to inhibit the protoporphyrinogen oxidase activity by comparing the growth rates of the first comparative system to the second comparative system, wherein an inhibition of the growth rate is indicative of an inhibition of protoporphyrinogen oxidase activity by said test compound. </div>
    </div>
    </div> <div class="claim"> <div num="4" id="US-6472164-B1-CLM-00004" class="claim">
      <div class="claim-text">4. A method for evaluating the ability of a compound to inhibit protoporphyrinogen oxidase activity, comprising the steps of:</div>
      <div class="claim-text">(1) transforming with a vector a host cell deficient in growing ability based on protoporphyrinogen oxidase activity, said vector comprising a DNA fragment coding for enzyme protoporphyrinogen oxidase which is capable of oxidizing protoporphyrinogen into protoporphyrin and which confers growth ability, wherein said DNA fragment is operably linked to a promoter functional in said host cell, and a terminator functional in the host cell, wherein said promoter is inducible, and a second vector comprising a second DNA fragment in which a DNA being capable of inducing the promoter of the first DNA fragment, a promoter, wherein said promoter is not induced by the DNA fragment but is functional in the host cell, and a terminator functionable in the host cell are operatively linked; </div>
      <div class="claim-text">(2) culturing said transformant expressing said protoporphyrinogen oxidase DNA in a medium containing substantially no protoheme compounds, wherein in a first comparative system there is a presence of a test compound to measure a growth rate of the transformant and in a second comparative system there is an absence of said test compound; and </div>
      <div class="claim-text">(3) determining the ability of the test compound to inhibit the protoporphyrinogen oxidase activity by comparing the growth rates of the first comparative system to the second comparative system, wherein an inhibition of the growth rate is indicative of an inhibition of protoporphyrinogen oxidase activity by said test compound. </div>
    </div>
    </div> <div class="claim-dependent"> <div num="5" id="US-6472164-B1-CLM-00005" class="claim">
      <div class="claim-text">5. The method according to <claim-ref idref="US-6472164-B1-CLM-00001">claim 1</claim-ref> or <claim-ref idref="US-6472164-B1-CLM-00003">3</claim-ref>, wherein the host cell is an <i>E. coli </i>or a yeast cell.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="6" id="US-6472164-B1-CLM-00006" class="claim">
      <div class="claim-text">6. The method of <claim-ref idref="US-6472164-B1-CLM-00001">claim 1</claim-ref>, <b>2</b>, <b>3</b>, or <b>4</b>, wherein said DNA fragment coding for enzyme protoporphyrinogen oxidase comprises SEQ ID NO:2 or SEQ ID NO:10.</div>
    </div>
  </div> </div></div></div><div class="patent-section patent-description-section"><div class="patent-section-header"><span class="patent-section-title">Description</span></div><div class="patent-text"><div mxw-id="PDES53657673" lang="EN" load-source="patent-office" class="description">
    <heading>BACKGROUND OF THE INVENTION</heading> <p>1. Field of the Invention</p>
    <p>The present invention relates to a method for evaluating the ability of a compound to inhibit the protoporphyrinogen oxidase activity.</p>
    <p>2. Description of the Related Art</p>
    <p>Plants, animals and microorganisms possess the porphyrin biosynthetic system starting with 5-aminolevulinic acid and produce protoporphyrin which is a precursor for the heme biosynthesis and the like. An enzyme which catalyses a reaction oxidizing protoporphyrinogen at the final stage of the porphyrin biosynthesis to produce protoporphyrin is protoporphyrinogen oxidase (protoporphyrinogen IX oxidase; EC 1.3.3.4)(hereinafter referred to as “PPO”)(R. J. Porra and J. E. Falk, (1964) Biochem. J., Vol.90, pp.69-75).</p>
    <p>The PPO activity has the influence on the growth of plants, animals and microorganisms and it is known that compounds which inhibit the plant-derived PPO activity have generally the herbicidal activity. Thus, in order to effectively develop PPO inhibiting-type herbicides, there has been a need for a method for evaluating the ability of a compound to inhibit the PPO activity with simplicity.</p>
    <heading>SUMMARY OF THE INVENTION</heading> <p>Under the aforementioned circumstances, the present inventors studied hard, which resulted in completion of the present invention.</p>
    <p>The present invention provides:</p>
    <p>1. A method for evaluating the ability of a compound to inhibit the protoporphyrinogen oxidase activity, which comprises the steps of:</p>
    <p>(1) culturing a transformant expressing a protoporphyrinogen oxidase gene present in a DNA fragment in a medium containing substantially no protoheme compounds in each comparative system of the presence and absence of a test compound to measure a growth rate of the transformant under each condition, said transformant being resulted from a host cell deficient in the growing ability based on the protoporphyrinogen oxidase activity transformed with the DNA fragment in which a promoter functionable in the host cell and a protoporphyrinogen oxidase gene are operatively linked, and</p>
    <p>(2) determining the ability of the compound to inhibit the protoporphyrinogen oxidase activity by comparing the growth rates (hereinafter refered to as “the present method”).</p>
    <p>2. A method for evaluating the ability of a compound to inhibit the protoporphyrinogen oxidase activity, which comprises the steps of:</p>
    <p>(1) culturing a transformant expressing a protoporphyrinogen oxidase gene present in a DNA fragment in a medium containing substantially no protoheme compounds in each comparative system of the presence and absence of a test compound to measure a growth rate of the transformant under each condition, said transformant being resulted from a host cell deficient in the growing ability based on the protoporphyrinogen oxidase activity transformed with the DNA fragment in which a promoter functionable in the host cell, a protoporphyrinogen oxidase gene and a terminator functionable in the host cell are operatively linked, and</p>
    <p>(2) determining the ability of the compound to inhibit the protoporphyrinogen oxidase activity by comparing the growth rates.</p>
    <p>3. A method for evaluating the ability of a compound to inhibit the protoporphyrinogen oxidase activity, which comprises the steps of:</p>
    <p>(1) culturing a transformant expressing a protoporphyrinogen oxidase gene present in the following (a) DNA fragment in a medium containing substantially no protoheme compounds in each comparative system of the presence and absence of a test compound to measure a growth rate of the transformant under each condition, said transformant being resulted from a host cell deficient in the growing ability based on protoporphyrinogen oxidase activity transformed with</p>
    <p>(a) the DNA fragment in which a promoter functionable in the host cell and controllable in its transcriptional activity, and a protoporphyrinogen oxidase gene are operatively linked, and</p>
    <p>(b) a DNA fragment in which a gene being capable of controlling the transcriptional activity of the promoter in the above DNA fragment and a promoter having the transcriptional activity not controllable by the gene and functionable in the host cell are operatively linked, and</p>
    <p>(2) determining the ability of the compound to inhibit the protoporphyrinogen oxidase activity by comparing the growth rates.</p>
    <p>4. A method for evaluating the ability of a compound to inhibit the protoporphyrinogen oxidase activity, which comprises the steps of:</p>
    <p>(1) culturing a transformant expressing a protoporphyrinogen oxidase gene present in the following (a) DNA fragment in a medium containing substantially no protoheme compounds in each comparative system of the presence and absence of a test compound to measure a growth rate of the transformant under each condition, said transformant being resulted from a host cell deficient in the growing ability based on the protoporphyrinogen oxidase activity transformed with</p>
    <p>(a) the DNA fragment in which a promoter functionable in the host cell and controllable in its transcriptional activity, a protoporphyrinogen oxidase gene and a terminator functionable in the host cell are operatively linked, and</p>
    <p>(b) a DNA fragment in which a gene being capable of controlling the transcriptional activity of the promoter in the above DNA fragment, a promoter having the transcriptional activity not controllable by the gene and functionable in the host cell, and a terminator functionable in the host cell are operatively linked, and</p>
    <p>(2) determining the ability of the compound to inhibit the protoporphyrinogen oxidase activity by comparing the growth rates.</p>
    <p>5. The method according to 1 or 3, which is characterized in that the protoporphyrinogen oxidase gene is a protoporphyrinogen oxidase gene derived from an animal or a plant.</p>
    <p>6. The method according to 1 or 3, which is characterized in that the protoporphyrinogen oxidase gene is a protoporphyrinogen oxidase gene derived from a rat or <i>Chlamydomonas reinhardtii. </i> </p>
    <p>7. The method according to 1 or 3, which is characterized in that the host cell is a microorganism.</p>
    <p>8. A rat-derived gene encoding a protein having the protoporphyrinogen oxidase activity.</p>
    <p>9. A protoporphyrinogen oxidase gene encoding a protein having the amino acid sequence shown by SEQ ID: No.1.</p>
    <p>10. A gene encoding a protein having the protoporphyrinogen oxidase activity and having the amino acid sequence in which one or several amino acids are deleted, substituted, modified or added in the amino acid sequence shown by SEQ ID: No.1.</p>
    <p>11. A protoporphyrinogen oxidase gene having the nucleotide sequence encoding the amino acid sequence shown by SEQ ID: No.1.</p>
    <p>12. A protoporphyrinogen oxidase gene having the nucleotide sequence shown by SEQ ID: No.2.</p>
    <p>13. A DNA fragment having a partial nucleotide sequence of the protoporphyrinogen oxidase gene of any one of 8 to 12.</p>
    <p>14. A vector which comprises the protoporphyrinogen oxidase gene of any one of 8 to 12.</p>
    <p>15. A transformant which is characterized in that the vector of 14 is introduced in to a host cell.</p>
    <p>16. The transformant according to 15, wherein the host cell is a microorganism.</p>
    <p>17. The transformant according to 15, wherein the host cell is a plant.</p>
    <p>18. A <i>Chlamydomonas reinhardtii</i>-derived gene encoding a protein having the protoporphyrinogen oxidase activity.</p>
    <p>19. A protoporphyrinogen oxidase gene encoding a protein having the amino acid sequence shown by SEQ ID: No.9.</p>
    <p>20. A gene encoding a protein having the protoporphyrinogen oxidase activity and having the amino acid sequence in which one or several amino acids are deleted, substituted, modified or added in the amino acid sequence shown by SEQ ID: No.9.</p>
    <p>21. A protoporphyrinogen oxidase gene having the nucleotide sequence encoding the amino acid sequence shown by SEQ ID: No.9.</p>
    <p>22. A protoporphyrinogen oxidase gene having the nucleotide sequence shown by SEQ ID: No.10.</p>
    <p>23. A DNA fragment having a partial nucleotide sequence of the protoporphyrinogen oxidase gene of any one of 18 to 22.</p>
    <p>24. A vector which comprises the protoporphyrinogen oxidase gene of any one of 18 to 22.</p>
    <p>25. A transformant which is characterized in that the vector of 24 is introduced in a host cell.</p>
    <p>26. The transformant according to 25, wherein the host cell is a microorganism.</p>
    <p>27. The transformant according to 25, wherein the host cell is a plant.</p>
    <heading>BRIEF DESCRIPTION OF THE DRAWINGS</heading> <p>FIG. 1 shows a method for constructing a vector containing a full length rat-derived protoporphyrinogen oxidase cDNA.</p>
    <p>[PPO fragment 1] shows a DNA fragment obtained by a polymerase chain reaction of Example 2, [PPO fragment 2] shows a DNA fragment harbored by a clone obtained in Example 1, [PPO] shows a rat-derived protoporphyrinogen oxidase cDNA, and other symbols show a restriction enzyme recognition site.</p>
    <p>FIG. 2 shows a method for constructing a protoporphyrinogen expression vector which is constructed for expressing a rat-derived protoporphyrinogen oxidase by introducing it into <i>Escherichia coli. </i> </p>
    <p>[PPO] shows a rat-derived protoporphyrinogen oxidase cDNA, [lacZ] shows a beta-galactosidase gene, and other symbols show a restriction enzyme recognition site.</p>
    <p>FIG. 3 shows a method for constructing a lac repressor expression vector which is constructed for expressing a lac repressor by introducing it into <i>Escherichia coli. </i> </p>
    <p>FIG. 4 shows the structure of a compound which was subjected to a test on the ability of a compound to inhibit the rat-derived protoporphyrinogen oxidase activity.</p>
    <p>FIG. 5 shows a method for constructing a protoporphyrinogen oxidase expression vector for direct introduction which is constructed for expressing a rat-derived protoporphyrinogen oxidase by introducing it into a plant cell.</p>
    <p>[35S] shows the cauliflower mosaic virus-derived 35S promoter, [NOS] shows the Agrobacterium-derived nopaline synthase terminator, [PPO] shows a rat-derived protoporphyrinogen oxidase cDNA, and other symbols show a restriction enzyme recognition site.</p>
    <p>FIG. 6 shows a method for constructing a protoporphyrinogen oxidase expression vector for indirect introduction which is constructed for a rat-derived protoporphyrinogen oxidase by introducing it into a plant cell.</p>
    <p>[35S] shows the cauliflower mosaic virus-derived 35S promoter, [NOS] shows the Agrobacteirum-derived nopaline synthase terminator, [PPO] shows a rat-derived protoporphyrinogen oxidase cDNA, [NTPII] shows a kanamycin-resistant gene, [NOSp] shows the Agrobacteirum-derived nopaline synthase promoter, [LB] and [RB] show the Agrobacteirum T-DNA-derived left border nucleotide sequence and right border nucleotide sequence, respectively, and other symbols show a restriction enzyme recognition site.</p>
    <heading>DETAILED DESCRIPTION OF THE PREFERRED EMBODIMENT</heading> <p>The present invention will be described in detail.</p>
    <p>First, the present method will be described.</p>
    <p>A PPO gene used in the present method may be any ones encoding a protein having the PPO activity upon expression in the host cell and, for example, may be selected from PPO genes derived from plants and animals. More particularly, there are PPO genes derived from Dicotyledonous plants such as Arabidopsis, soybean, oil seed rape, sugar beat, potato and tobacco, Monocotyledonous plants such as corn, rice, wheat, barley, oat, rye, sugar cane and sorghum, algae such as <i>Chlamydomonas reinhardtii </i>and Chlorella, mammals such as mouse, rat and human, fishes sich as trout, bluegill, carp, cyprinodont, guppy, zebra fish and fathead minnow, insects such as fly, mosquito, cockroach, greasy grind, dragonfly and silkworm moth.</p>
    <p>As used herein, “a promoter functionable in a host cell” used in the present method is a DNA fragment having the transcriptional activity in a host cell being transformed and includes, for example, <i>Escherichia coli </i>lactose operon promoter (lacP), tryptophan operon promoter (trpP), arginine operon promoter (argP), galactose operon promoter (galP), tac promoter, T7 promoter, T3 promoter, λ phage promoter (λ-pL, λ-pR), yeast alcohol dehydrogenase gene (ADH) promoter, adenovirus major late (Ad.ML) promoter, SV40 early promoter, baculovirus promoter, nopaline synthase gene (NOS) promoter, octopine synthase gene (OCT) promoter, cauliflower mosaic virus (CaMV)-derived 19S and 35S promoters, phenylalanine ammonia lyase (PAL) gene promoter, chalcone synthase (CHS) gene promoter and the like.</p>
    <p>In addition, “a terminator functionable in a host cell” used if necessary includes any DNA fragments having the transcription terminating activity in a host cell being transformed. Examples thereof include <i>Escherichia coli </i>lactose operon terminator, arginine operon terminator, galactose operon terminator, yeast HIS terminator, ADH1 terminator, SV40 early splicing region, nopaline synthase gene (NOS) terminator, garlic virus GV1 and GV2 terminators and the like.</p>
    <p>As used herein, “a DNA fragment are operatively linked” means a DNA fragment in which these promoter, and, if necessary, these terminator, are linked so that the above PPO gene is expressed under the control of the above promoter, and, if necessary, the above terminator, in a host cell into which they are to be introduced (hereinafter referred to as “expression DNA fragment”).</p>
    <p>The above expression DNA fragment is introduced into a host cell deficient in the growing ability based on the PPO activity to obtain a transformant expressing the introduced PPO gene.</p>
    <p>A host cell being transformed, that is, deficient in the growing ability based on the PPO activity may be any host cells deficient in the PPO activity necessary for growth and, from a viewpoint of simple culturing, microorganisms are preferable. Examples of microorganisms deficient in the PPO activity necessary for growth include PPO gene (hemG locus)-deficient mutant <i>Escherichia coli </i>strain VSR 751 described in K. Miyamoto, K. Nakahigashi, K. Nishimura, T. Nakayashiki and H. Inokuchi, (1991) Journal of Molecular Biology, vol.219, pp393-398 and K. Nishimura, T. Nakayashiki and H. Inokuchi, (1993) Gene, vol.133, pp109-113, PPO gene (hemG locus)-deficient mutant <i>Escherichia coli </i>strain BT3 described in F. Yamamoto, H. Inokuchi and H. Ozeki, (1988) Jpn. J. Genet., vol.63, pp237-249, PPO gene (hem14-1 locus)-deficient mutant yeast strain hem14-1 described in J. M. Gamadro, D. Urban-Grimal and P. Labbe (1982) Biochem. Biophys. Res. Commun., vol.106, pp724-730, and the like.</p>
    <p>The above expression DNA fragment incorporated into a vector may be introduced into a host cell. The vector containing the expression DNA fragment may be introduced into a host cell according to the conventional genetic engineering method. For example, when the host cell is a microorganism, the vector may be introduced into a cell of the microorganism with a calcium chloride method, an electroporation method (Methods in Electroporation: Gene Pulser/<i>E. coli </i>Pulser System, Bio-Rad Laboratories, 1993) and the like. In addition, when the host cell is a plant, the vector may be introduced into a cell of the plant with an Agrobacterium infection method (JP-B-2-58917 (examined) and JP-A-60-70080 (laid-opened)), an electroporation into a protoplast method (JP-A-60-251887 (laid-opened) and JP-A-5-68575 (laid-opened)), a particle gun method (JP-A-5-508316 (laid-opened) and JP-A-63-258525 (laid-opened)) and the like. A transformant expressing the introduced PPO gene may be obtained by culturing the host cell with the introduced vector in a medium containing a growth inhibitor corresponding to a selectable marker lucus-linked on the vector and a medium containing substantially no protoheme compounds, and isolating a clone which can grow in the medium.</p>
    <p>The transformant thus obtained is first cultured in a medium containing substantially no protoheme compounds in each comparative system of the presence at a variety of concentrations and absence of a test compound to measure a growth rate of the transformant under each condition. Next, the PPO activity inhibiting ability of the test compound to inhibit the PPO activity is determined by comparing the growth rates based on the each comparative system of contact and non-contact of the transformant and the test compound. The comparison of the growth rates is performed, for example, by obtaining the growth inhibiting rate by calculating a ratio of the growth rate in a system of the presence of the test compound relative to the growth rate in a system of the absence of the test compound, and comparing the growth inhibiting rates regarding a plurality of test compounds and, based on the results thereof, the higher or lower ability for inhibiting PPO activity may be determined. More particularly, for example, a concentration of a test compound at which the growth is inhibited by 50% is obtained from the growth rate of the transformant in the system of the addition of a compound (that is, of the presence of the test compound) under the conditions that the growth rate of the transformant is 100% in the system of no addition of a compound (that is, of the absence of the test compound). The test compound having the lower concentration may be determined to be higher in the ability for inhibiting PPO activity than the test compound having the higher concentration. Alternatively, a compound known to inhibit the PPO activity (positive control) is simultaneously subjected to an experiment as one of test compounds, which may serve as an index for the growth inhibiting rate of each test compound (for example, a compound concentration at which the growth is inhibited by 50%).</p>
    <p>As used herein, “a medium containing substantially no protoheme compounds” means a medium not containing any protoheme compounds or a medium which may contain protoheme compounds at such an amount that a host cell deficient in the growing ability based on the PPO activity can not recover the growing ability to the same level as that of the corresponding non-deficient host cell (that is, a host cell not deficient in the growing ability based on the PPO activity) regardless of the presence of protoheme compounds in the medium. The examples thereof include an artificial medium not containing any protoheme compounds such as protoporphyrin, protohemin and the like at all, a medium containing natural extracts (for example, yeast extract, Malt extract and the like), and the like. From a viewpoint of the precision of the present method, it is preferable that an amount of the protoheme compounds present in a medium is zero or as low as possible. In addition, protoheme compounds include protoheme and its derivatives and these compounds mean a compound having the action of recovering the growing ability of a host cell deficient in the growing ability based on the PPO activity.</p>
    <p>In order to confirm that the growth inhibition by a test compound is due to the inhibition of PPO activity, it can be easily determined by performing the culturing of the above transformant in a medium containing substantially protoheme compounds such as protoporphyrin, protohemin and the like, and confirming the growth of the transformant. In addition, if needed, the growth rate of the transformant may be conveniently adjusted by the relationship between an expressed amount of PPO gene and a concentration of a test compound.</p>
    <p>In the present method, the expressed amount of PPO gene in the transformant may vary by using “a promoter functionable in a host cell and controllable in its transcriptional activity” as a promoter for the aforementioned expression DNA fragment, and introducing into the host cell “a DNA fragment in which a gene capable of controlling the transcriptional activity of a promoter in the expression DNA fragment and a promoter having the transcriptional activity not-controllable by the gene and functionable in the host cell, and an optionally used terminator functionable in the host cell are operatively linked” (hereinafter referred to as “regulatory DNA fragment”). For example, when evaluation of a test compound having the lower water-solubility and that of a small amount of a test compound are performed, by adjusting an expressed amount of PPO gene in the transformant to be small, evaluation may be performed with the higher precision.</p>
    <p>On the other hand, when evaluation of a test compound having the higher ability for inhibiting PPO activity are performed, by adjusting an expressed amount of PPO gene in the transformant to be increased, evaluation may be performed with the higher precision.</p>
    <p>As used herein, “a gene capable of controlling the transcriptional activity of a promoter in the expression DNA fragment”(hereinafter referred to as “a regulatory gene”) means a gene having the function of inhibiting or promoting the transcriptional activity of “a promoter functionable in the host cell and controllable in its transcriptional activity” as a promoter for the expression DNA fragment. When an expressed amount of PPO gene is to be decreased, a gene which inhibits the transcriptional activity of a promoter in the expression DNA fragment may be used, while when an expressed amount of PPO gene is to be increased, a gene which promotes the transcriptional activity of the promoter may be used. More particularly, examples of a combination for inhibiting the transcriptional activity include a combination of <i>Escherichia coli </i>lactose operon promoter and lacI, that of arginine operon promoter and argR, that of galactose operon promoter and GalR and the like. On the other hand, examples of a combination for promoting the transcriptional activity include a combination of <i>Escherichia coli </i>lactose operon promoter and crp, that of <i>Klebsiella pneumoniae </i>maltose regulon and malT and the like.</p>
    <p>“A promoter having the transcriptional activity not-controllable by the regulatory gene and functionable in a host cell”, that is, a promoter for the regulatory DNA fragment may be any promoters having no influence by a gene which they themselves control. For example, in a case of a promoer to be combined with a repressor gene for the above lactose operon, its own promoter, nopaline synthase enzyme gene (NOS) promoter and the like may be used.</p>
    <p>The above expression DNA fragment and the regulatory DNA fragment may be introduced into a host cell by incorporating into the same or different vector and these DNA fragments may be incorporated into chromosome(s) of the host cell.</p>
    <p>The present method may be utilized for screening medications having the ability for inhibiting the malignant cell growth in addition to evaluating the potency of a phototrophic herbicide, screening a compound having the hebicidal activity.</p>
    <p>When the present method is utilized for evaluating the potency of a phototrophic herbicide or screening a compound having the herbicidal activity, the present method is performed, for example, using PPO genes derived from higher plants such as Arabidopsis, PPO genes derived from algae such as <i>Chlamydomonas reinhardtii. </i>As a result, a compound inhibiting the growth of the transformant into which a PPO gene derived from higher plants or algae is introduced is potentially useful as an active ingredient of a herbicide.</p>
    <p>Next, a gene which may be used for the present method will be described in detail.</p>
    <p>Examples of a rat-derived gene encoding a protein having the PPO activity (hereinafter referred to as “the present rat PPO gene”) are (1) a rat-derived gene having about 1.64 kbp of length, for example, a PPO gene encoding a protein having the amino acid sequence shown by SEQ ID: No.1, (2) a gene encoding a protein having the PPO activity and having the amino acid sequence in which one or several amino acids are deleted, substituted, modified or added in the amino acid sequence shown in SEQ ID: No.1. More particularly, for example, mention be made of a PPO gene having the nucleotide sequence shown by SEQ ID: No.2.</p>
    <p>Examples of a <i>Chlamydomonas reinhardtii</i>-derived gene encoding a protein having the PPO activity (hereinafter referred to as “the present <i>Chlamydomonas reinhardtii </i>PPO gene”) are (1) a <i>Chlamydomonas reinhardtii</i>-derived gene having about 2 kbp of length, for example, a PPO gene encoding a protein having the amino acid sequence shown by SEQ ID: No.9, (2) a gene encoding a protein having the PPO activity and having the amino acid sequence in which one or several amino acids are deleted, substituted, modified or added in the amino acid sequence shown in SEQ ID: No.9. More particularly, for example, mention be made of a PPO gene having the nucleotide sequence shown by SEQ ID: No.10.</p>
    <p>The present rat PPO gene is obtained, for example, according to the following method.</p>
    <p>RNA is extracted from rat tissues such as liver, kidney and the like according to a method described in Labomanual Genetic Engineering, Suppl. Edition, edited by Matsumura, published by Maruzen K. K., pp76-77, 1990. The extracting procedures may be carried out utilizing a commercially available RNA extracting kit, for example, ISOGEN (Nippon Gene).</p>
    <p>The resultant RNA is subjected to the manipulation according to the attached manual using a commercially available poly(A) RNA fractionating kit such as BIOMAG mRNA Purification kit (Perceptive Biosystems) to prepare poly(A) RNA. The resulting poly(A) RNA is subjected to the manipulation according to a method described in Cloning and Sequence: Plant Biotechnology Experimental Manual, edited by Watanabe and Sugiura, published by Nosonbunka Company, pp74-103 (1989) to make cDNA library. This cDNA library making manipulation may utilize a commercially available cDNA library making kit such as ZAP-cDNA Gigapack Cloning Kit (STRATAGENE).</p>
    <p>From thus made cDNA library or a commercially available cDNA library such as a rat liver-derived cDNA liver manufactured by STRATAGENE, DNA may be prepared by a method described in Molecular Cloning 2nd edition (authors: J. Sambrook, E. F. Frisch, and T. Maniatis; Cold Spring Harbor Laboratory Press, 1989) 2.60-2.81.</p>
    <p>By using thus prepared DNA as a template, a polymerase chain reaction may be performed using an oligonucleotide having the nucleotide sequence shown by SEQ ID: No.3 and that having the nucleotide sequence shown in SEQ ID: No.4 as a primer to amplify a DNA fragment having a partial nucleotide sequence of PPO gene. This amplified DNA fragment having a partial nucleotide sequence of a PPO gene may be cloned into a plasmid using the conventional method described in Labomanual Genetic Engineering, Suppl. Edition, edited by Matsumura, published by Maruzen K. K., pp117-120 (1990) and a commercially available DNA cloning kit such as TA cloning kit (manufactured by Invitrogen) to serve for determining nucleotide sequence. Determination of the nucleotide sequence may be performed by a dideoxy method described, for example, in Molecular Cloning 2nd edition (authors: J. Sambrook, E. F. Frisch and T. Maniatis; Cold Spring Harbor Laboratory Press), 13.42-13.74 (1989). Determination of the nucleotide sequence according to dideoxy method may be performed utilizing a sequencing kit using fluorescently-labeled dideoxynucleotide (more particularly, for example, Dye terminator cycle sequencing kit (manufactured by PE Applied Biosytems) and using a commercially available autosequencer such as DNA Sequencer 373S (manufactured by PE Applied Biosytems).</p>
    <p>Thus, the DNA fragment having a partial nucleotide sequence of a rat PPO gene (hereinafter referred to as “the present rat PPO DNA fragment”) can be obtained. One example of the present rat PPO DNA fragment is a gene fragment having the nucleotide sequence shown in nucleotide Nos. 639 to 1333 shown by SEQ ID: No.2. The present rat PPO DNA fragment may be utilized in making a PPO gene-specific primer used in a polymerase chain reaction described below or used as a probe for detecting a PPO gene in a hybridization method described below.</p>
    <p>Based on the present rat PPO DNA fragment, the present rat PPO gene having entire nucleotide sequence encoding PPO may be obtained according to (1) polymerase chain reaction method or (2) hybridization method.</p>
    <p>In order to obtain a PPO gene by utilizing a polymerase chain reaction, a primer having about 15 bp to about 40 bp nucleotide sequence among the nucleotide sequence of the present rat PPO DNA fragment (primer for amplifying 3′ downstream region) and about 15 bp to about 40 bp nucleotide sequence among the nucleotide sequence complementary to the base sequence of the present rat PPO DNA fragment (primer for amplifying 5′ upstream region) are first prepared. Examples of the primer for amplifying 3′ downstream region are a gene fragment having the nucleotide sequence shown in nucleotide Nos.1175 to 1198 by SEQ ID: No.2 and the like, and examples of the primer for amplifying 5′ upstream region are a gene fragment having the nucleotide sequence complementary to the nucleotide sequence shown in nucleotide Nos.776 to 799 by SEQ ID: No.2 and the like.</p>
    <p>Then, a polymerase chain reaction is performed using as a template a DNA fragment having an adapter DNA added at an end of a rat-derived DNA or a DNA of a vector having rat-derived DNA fragment inserted therein and using a combination of a primer having a partial nucleotide sequence of an adapter DNA or a primer having a partial nucleotide sequence of a vector DNA and a primer for amplifying 5′ upstream region and a combination of a primer having a partial nucleotide sequence of an adapter DNA or a primer having a partial nucleotide sequence of a vector DNA and a primer for amplifying 3′ downstream region to amplify a DNA fragment containing 5′ upstream nucleotide sequence of the present rat PPO DNA fragment and a DNA fragment having 3′ downstream nucleotide sequence of the present rat PPO DNA fragment. The DNA fragment having an adapter DNA added at an end of a rat-derived DNA which is used as a template may be prepared by adding as an adapter DNA a polymer of a nucleotide such as cytosine to a rat-derived cDNA prepared above with terminal deoxynucleotidyl transferase or adding a commercially available adapter of a kit for RACE (rapid amplification of cDNA ends) reaction such as an adapter attached to Marathon cDNA amplification kit manufactured by Clontech to a rat-derived cDNA. In addition, one example of the primer having a partial nucleotide sequence of an adapter DNA is a primer specific for an adapter attached to a commercially available RACE (rapid amplification of cDNA ends) reaction kit (for example, AP-1 primer and AP-2 primer attached to Marathon cDNA amplification kit manufactured by Clontech). As the primer having a partial sequence of a vector DNA, when a vector is derived from λ phage, primers specific for an arm region of λ phage such as λ gtII reverse primer and λ gtII forward primer may be used.</p>
    <p>Two kinds of thus amplified DNA fragments may be cloned into a plasmid by using the conventional method described in Labomanual Genetic Engineering, Suppl. Edition, edited by Matsumura, published by Maruzen K. K., pp117-120 (1990) or a commercially available DNA cloning kit such as TA cloning kit (manufactured by Invitrogen) to serve for determination of the nucleotide sequence. Determination of the nucleotide sequence may be performed by a dideoxy method described in Molecular Cloning 2nd edition (authors: J. Sambrook, E. F. Frisch and T. Maniatis; Cold Spring Harbor Laboratory Press), 13.42-13.74 (1989). Determination of the nucleotide sequence by a dideoxy method may be performed by utilizing a sequencing kit using a fluorescently labeled dideoxynucleotide such as Dye terminator cycle sequencing kit (manufactured by PE Applied Biosystems) and using a commercially available autosequencer such as DNA Sequencer 373S (manufactured by PE Applied Biosystems). The regions between the thus determined nucleotide sequence and the nucleotide sequence of the present rat PPO DNA fragment which was referenced upon manufacturing the above primer are overlaid to ligate these nucleotide sequences to make one base sequence. Whether or not an entire region of open reading frame encoding PPO is contained in the thus made nucleotide sequence may be studied by investigating open reading frame using a commercially available gene analyzing software such as GENETYX (manufactured by SDC). When a 1.6 kbp or more entire open reading frame is not contained in the determined nucleotide sequence and translation initiation codon or translation termination codon are not contained therein, it is determined that the whole nucleotide sequence of the full length PPO gene is not contained in the nucleotide sequence, and steps for obtaining a gene fragment by a polymerase chain reaction is repeated until PPO gene having the whole nucleotide sequence encoding PPO is obtained. In addition, when open reading frame which is found in the nucleotide sequence determined as described above is missing in either in 5′ upstream or 3′ downstream side of the determined nucleotide sequence, that is, when a translation initiation codon or translation termination codon is not contained in the open reading frame, steps for obtaining a gene fragment by the polymerase chain reaction same as above may be repeated for obtaining a DNA fragment containing the nucleotide sequence on the missing's side.</p>
    <p>Based on the information of the nucleotide sequence thus determined, a primer containing the nucleotide sequence near translation initiation codon in the nucleotide sequence of a PPO gene (N-terminal primer) and a primer containing the nucleotide sequence complementary to the nucleotide sequence near translation termination codon (C-terminal primer) are made, and a polymerase chain reaction may be performed using a rat-derived DNA as a template and using the above both terminal primers to amplify the present rat PPO gene having the entire nucleotide sequence encoding PPO, which may be cloned. Examples of the N-terminal primer include a primer having the nucleotide sequence shown by SEQ ID: No. 5 and a primer having the nucleotide sequence shown by SEQ ID: No. 7, and examples of the C-terminal primer include a primer having the nucleotide sequence shown by SEQ ID: No. 6 and a primer having the nucleotide sequence shown by SEQ ID: No. 8. Alternatively, the DNA fragments obtained by the above steps for obtaining the gene fragment may be ligated with an restriction enzyme recognition site present in the overlapping regions to obtain the present rat PPO gene having the entire nucleotide sequence encoding PPO.</p>
    <p>(2) In order to obtain the present rat PPO gene by utilizing a hybridization method, a DNA fragment may be identified by hybridizing a probe of the present rat PPO DNA fragment to a rat-derived DNA and the identified DNA fragment may be isolated.</p>
    <p>A probe used for a hybridization method may be prepared by labeling the present rat PPO DNA fragment obtained as described above with a labeling compound by the conventional method described in Molecular Cloning 2nd edition (authors: J. Sambrook, E. F. Frisch and T. Maniatis; Cold Spring Harbor Laboratory Press), 10.6-10.26 (1989). A labeling compound such as a compound containing a radioactive element (hereinafter referred to as “RI”) and a fluorescent reagent and the like may be used. Labeling with a compound containing RI may be performed using [α-<sup>32</sup>P] dCTP (manufactured by Amersham) and a random prime labeling kit (manufactured by Boehringer Mannheim) according to the protocol attached to the kit. In addition, non-RI labeling may be performed using DIG-High Prime DNA Labeling and Detection Starter KitI (Boehringer Mannheim) and attached reagents according to the attached protocol.</p>
    <p>As a rat-derived DNA for hybridizing with a probe, a rat cDNA library and the like may be used which is obtained by ligating a cDNA derived from a rat to a vector with a suitable adaptor, derived from λ phage and the like and packaging the cDNA into phage particles. Such the gene library may be prepared according to a method, for example, described in Cloning and Sequencing: Plant Biotechnology Experimental Mannual, editted by Watanabe and Sugiura, published by Nosonbunka, pp74-103 (1989). Alternatively, the rat gene libraries sold by Clontech, STRATAGENE and the like may be used.</p>
    <p>In order to isolate the present gene, for example, colony hybridization, plaque hybridization and the like to the aforementioned rat gene library may be performed using the probe prepared by the aforementioned method. These may be performed according to a method described in Recombinant DNA Experiment (edited by J. R. Dillon, A. Nasim and E. R. Nestmann, published by Tokyo kagakudozin), pp98-101 (1987) or the conventional method described in Molecular Cloning 2nd edition (authors: J. Sambrook, E. F. Frisch and T. Maniatis; Cold Spring Harbor Laboratory Press), pp1.90-1.104 or 2.108-2.117 (1989). Determination of the nucleotide sequence of the DNA fragment incorporated in the resultant clone may be performed by a dideoxy method described in Molecular Cloning 2nd edition (authors: J. Sambrook, E. F. Frisch and T. Maniatis, Cold Spring Harbor Laboratory Press), pp13.42-13.74 (1989). The determination of the nucleotide sequence by a dideoxy method may be performed by utilizing a sequencing kit using fluorescently labeled dideoxynucleotide such as Dye terminator cycle sequencing kit (manufactured by PE Applied Biosystems) and using a commercially available autosequencer such as DNA Sequencer 373S (manufactured by PE Applied Biosystems). The determined nucleotide sequence may be analyzed using a commercially available gene analyzing software such as GENETYX (SDC) to reveal the entire nucleotide sequence of the present rat PPO gene.</p>
    <p>Next, the present <i>Chlamydomonas reinhardtii</i>-derived PPO gene is obtained by the following method. The fundamental manipulation may be performed according to the aforementioned description for obtaining the present rat PPO gene.</p>
    <p> <i>Chlamydomonas reinhardtii </i>is cultured to collect cells and RNA is extracted from the collected cells. A cDNA library is made from the resultant RNA according to a method for obtaining the present rat PPO gene to prepare its DNA.</p>
    <p>By using the DNA thus prepared as a template, a polymerase chain reaction may be performed using as a primer an oligonucleotide having the nucleotide sequence shown by SEQ ID: No.11 and that having the nucleotide sequence shown by SEQ ID: No.12 to amplify a DNA fragment having a <i>Chlamydomonas reinhardtii </i>PPO gene. The amplified DNA fragment having the nucleotide sequence of PPO gene may be cloned into a plasmid to serve for determination of the nucleotide sequence.</p>
    <p>Thus obtained cDNA of the present rat PPO gene or the present rat PPO DNA fragment, or the cDNA of the present <i>Chlamydomonas reinhardtii </i>PPO gene may be used to obtain a genome DNA clone of the present rat PPO gene or the present <i>Chlamydomonas reinhardtii </i>PPO gene, the nucleotide sequence of which may be determined. First, in the case of the genome DNA clone of the present rat PPO gene, for example, rat tissues such as liver, kidney and the like are taken and a genome DNA is extracted from the taken tissues by manipulations according to a method described in Labomanual Genetic Engineering, Suppl. Edition, edited by Matsumura, published by Maruzen K. K., pp76-77 (1990). On the other hand, in the case of a genome clone of the present <i>Chlamydomonas reinhardtii </i>PPO gene, for example, <i>Chlamydomonas reinhardtii </i>is cultured to collect cells and a genome DNA is extracted from the collected cells according to the similar procedures. The extracting procedures may be performed using a commercially available DNA extracting kit such as ISOTISSUE (Nippon Gene) according to the protocol attached to the kit to obtain a genome DNA. After the genome DNA is cut with a suitable restriction enzyme, the resultant genome DNA fragment is fractionated according to a NaCl density gradient centrifugation method described in Cloning and Sequencing: Plant Biotechnology Experimental Manual, editted by Watanabe and Sugiura, published by Nosonbunka, pp276-279 (1989). In general, conveniently, a fraction containing a genome DNA fragment having a size suitable for incorporating into a vector, particularly, a fraction containing 9 kbp to 23 kbp genome DNA fragment when a phage vector is used, a fraction containing 30 kbp to 42 kbp genome DNA fragment when a cosmid vector is used is selected from the fractionated respective genome DNA fragments. A genome library is made using the resultant genome DNA fragments and performing the procedures according to a method described in Cloning and Sequencing: Plant Biotechnology Experimental Manual, edited by Watanabe and Sugiura, published by Nosonbunka, pp96-103 and 280-284 (1989) or using a commercially available DNA library making kit such as Lambda EMBL3/Gigapack cloning kit (STRATAGENE) according to the attached protocol. The genome DNA library thus made, or a commercially available genome library such as rat genome library manufactured by STRATAGENE is screened by hybridization using as a probe a cDNA of the present rat PPO gene or the present rat PPO DNA fragment, or a cDNA of the present <i>Chlamydomonas reinhardtii </i>PPO gene according to a method described in Cloning and Sequence: Plant Biotechnology Experimental Manual, edited by Watanabe and Sugiura, published by Nosonbunka, pp106-147 (1989) to obtain a genome DNA clone having the nucleotide sequence of the present rat PPO gene, or the present <i>Chlamydomonas reinhardtii </i>PPO gene. The resultant genome DNA clone may be subcloned into a suitable vector for analyzing the nucleotide sequence such as a plasmid and the like according to a method described in Cloning and Sequence: Plant Biotechnology Experimental Mannual, edited by Watanabe and Sugiura, published by Nosonbunka, pp152-174 (1989), and sequenced by a dideoxy method and the like according to a primer extension method described in Molecular Cloning 2nd edition (authors; J. Sambrook, E. F. Frisch and T. Maniatis; Cold Spring Harbor Laboratory), pp13.15 (1989). Determination of the nucleotide sequence by dideoxy method may be performed using a commercially available kit such as Dye terminator cycle sequencing kit manufactured by PE Applied Biosystems and using a DNA sequencer such as Model 373S of PE Applied Biosystems.</p>
    <p>Further, by a primer extension method described in Bina-Stem Met et al., (1979) Proc. Natl. Acad. Sci.U.S.A.,vol.76, pp731 and Sollner-Webb and R. H. Reeder, (1979) Cell, vol.18, pp485 or a S1 mapping method described in A. J. Berk and P.A.Sharp, (1978) Proc. Natl. Acad. Sci. U.S.A., vol.75, pp1274 and the like, each transcription initial point of the genome DNA of the present rat PPO gene or the present <i>Chlamydomonas reinhardtii </i>PPO gene may be decided. At about 1 kb to about 10 kb upstream from this transcription initiation point, there is a promoter sequence responsible for controlling the gene expression.</p>
    <p>In addition, the present rat PPO gene and the present <i>Chlamydomonas reinhardtii </i>PPO gene can be utilized as a means for obtaining or generating a gene resistance to a PPO activity inhibiting agent known as a phototrophic herbicide. For example, it becomes possible to generate a new herbicide-resistant gene by introducing into a PPO gene a mutation which gives a change to the amino acid sequence of PPO encoded by the present rat PPO gene or the present <i>Chlamydomonas reinhardtii </i>PPO gene and screening a gene encoding the amino acid sequence of PPO showing phototrophic herbicide-resistance. More particularly, it becomes possible to generate a new phototrophic herbicide-resistant gene by inducing a random mutation in the nucleotide sequence of the present rat PPO gene or the present <i>Chlamydomonas reinhardtii </i>PPO gene to introduce a mutation into the amino acid sequence according to a method described in A. Greener, M. Callahan, Strategies, 1994, Vol.7, pp32 to 34, and the like. In addition, it also becomes possible to generate a new phototrophic herbicide-resistant gene by introducing a site-specific mutation into the nucleotide sequence of the present rat PPO gene or the present <i>Chlamydomonas reinhardtii </i>PPO gene to alter the amino acid sequence according to a gapped duplex method described in W. Kramer, et al., Nucleic Acids Research, 1984, vol.12, pp9441 or W. Kramer, H. J. Frits, Methods in Enzymology, 1987, vol.154, pp350, or a Kunkel method described in T. A. Kunkel, Proc. of Natl. Acad. Sci. U.S.A., 1985, vol.82, pp488 or T. A. Kunkel, et. al., Methods in Enzymology, 1987, vol. 154, pp367, and the like. Further, it also becomes possible to generate a new phototrophic herbicide-resistant gene by making a gene encoding a chimera protein in which one or several partial amino acid sequences among the amino acid sequence of PPO encoded by the present rat PPO gene or the present <i>Chlamydomonas reinhardtii </i>PPO gene are substituted with a part of the amino acid sequence of the present <i>Chlamydomonas reinhardtii </i>PPO gene, the present rat PPO gene or PPO gene derived from other organisms. The effects of the thus made phototrophic herbicide-resistant gene (expresasion of herbicide resistance) can be effectively confirmed by, for example, introducing a generated herbicide-resistant gene into a microorganism defective in PPO or a microorganism having PPO sensitive to a subject herbicide, selecting a transformed microorganism, and treating the transformed microorganism with the subject herbicide to reselect a herbicide-resistant colony.</p>
    <p>As a vector which comprises the present rat PPO gene or the present <i>Chlamydomonas reinhardtii </i>PPO gene, mention may be made of a plasmid made by cloning into pCR2.1 (Invitrogen) a PPO gene having the nucleotide sequence encoding the amino acid sequence shown by SEQ ID: No.1 or No.9 and the plasmid has the such the characteristics that a vector part is small and a copy number is large in <i>Escherichia coli </i>and, thus, is suitable for preparing DNA or analyzing the DNA structure.</p>
    <p>A vector which can cause the present rat PPO gene or the present <i>Chlamydomonas reinhardtii </i>PPO gene to be expressed in a host cell can be constructed by, for example, inserting into a vector a DNA fragment in which (1) a promoter functionable in a host cell, and (2) the present rat PPO gene or the present <i>Chlamydomonas reinhardtii </i>PPO gene and, if necessary, (3) a terminator functionable in a host cell are operatively linked in an operative manner in the above order. As used herein, “operatively” means that the present rat PPO gene or the present <i>Chlamydomonas reinhardtii </i>PPO gene are ligated to a promoter (and, if necessary, a terminator) so that they are expressed under the control of the promoter (and the terminator) upon introduction of the vector into the host cell to transform the host cell.</p>
    <p>As a “a promoter functionable in a host cell” is not limited to specified ones but may be any promoters having the transcriptional activity in a host cell to be transformed. For example, mention may be made of <i>Escherichia coli </i>lactose operon promoter, yeast alcohol dehydrogenase gene (ADH) promoter, adenovirus major late (Ad.ML) promoter, SV40 early promoter, baculovirus promoter, nopaline synthase gene (NOS) promoter, octopine synthase gene (OCT) promoter, cauliflower mosaic virus (CaMV)-derived 19S and 35S promoters, phenylalanine ammonia lyase (PAL) gene promoter, chalcone synthase (CHS) gene promoter and the like.</p>
    <p>“A terminator functionable in a host cell” is not limited to specified ones but may be any terminators having the transcription terminating activity in a host cell to be transformed. For example, mention may be made of <i>Escherichia coli </i>lactose operon terminator, yeast HIS terminator, ADH1 terminator, SV40 early splicing region, nopaline synthase gene (NOS) terminator, garlic virus GV1, GV2 terminators and the like.</p>
    <p>A host cell can be transformed by introducing the above vector into the host cell. For example, when a host cell is a microorganism, the above vector can be introduced into a cell of the microorganism by the known means such as a calcium chloride method, an electroporation method (Methods in Electroporation: Gene Pulser/E. coli Pulser System, Bio-Rad Laboratories, 1993) and the like. On the other hand, when a host cell is a plant, a transformed plant cell can be obtained by introducing the above present vector into a cell of the plant by the known means such as an Agrobacterium infection method (JP-B-2-58917 (examined) and JP-A-60-70080 (laid-opened)), an electroporation to a protoplast method (JP-A-60-251887 (laid-opened) and JP-A-5-68575 (laid-opened)), or a particle gun method (JP-A-5-508316 (laid-opened) and JP-A-63-258525 (laid-opened)) to select a cell into which the present rat PPO gene or the present <i>Chlamydomonas reinhardtii </i>PPO gene is introduced. From the resultant transformed plant cell, a transformed plant body can be obtained by regenerating a transformed plant by a plant cell culturing method described in, for example, Plant Genetic Manipulation Manual: A method for generating A Transgenic Plant (author: Uchimiya, Kodansha Scientific, 1990, pp.27 to 55).</p>
    <p>Furthermore, PPO activity of a host cell or sensitivity of a host cell to the inhibitor for PPO activity may be altered by transforming a host cell by introducing into the host cell a vector containing a DNA fragment in which (1) a promoter functionable in a host cell and (2) the present rat PPO gene or the present <i>Chlamydomonas reinhardtii </i>PPO gene and, if necessary, (3) a terminator functionable in a host cell using the present rat PPO gene or the present <i>Chlamydomonas reinhardtii </i>PPO gene.</p>
    <p>As used herein, “a promoter functionable in a host cell” is not limited to specific ones but may be any promoters having the transcriptional activity in a host cell to be transformed. For example, mention may be made of <i>Escherichia coli </i>lactose operon promoter, yeast alcohol dehydrogenase gene (ADH) promoter, adenovirus major late (Ad.ML) promoter, SV40 early promoter, baculovirus promoter, nopaline synthase gene (NOS) promoter, octopine synthase gene (OCT) promoter, cauliflower mosaic virus (CaMV)-derived 19S and 35S promoters, phenylalanine ammonia lyase (PAL) gene promoter, chalcone synthase (CHS) gene promoter and the like.</p>
    <p>“A terminator functionable in a host cell” is not limited to specified ones but may be any terminators having the transcription terminating activity in a host cell to be transformed. For example, mention may be made of <i>Escherichia coli </i>lactose operon terminator, yeast HIS terminator, ADH1 terminator, SV40 early splicing region, nopaline synthase gene (NOS) terminator, garlic virus GV1, GV2 terminators and the like.</p>
    <p>By the above method for altering a host cell, for example, a resistance to a phototrophic herbicide targeting PPO can be imparted to a plant.</p>
    <p>Other PPO genes can be obtained and utilized by the aforementioned method for obtaining the present rat PPO gene. For example, in order to obtain a rabbit-derived PPO gene, a commercially available rabbit cDNA library, primers, etc shown by SEQ ID: No.3 and No.4 and the like may be used.</p>
    <heading>EXAMPLES</heading> <p>The following Examples illustrate the present invention in detail but do not limit the present invention.</p>
    <heading>Example 1</heading> <p>(Cloning of a Rat PPO DNA Fragment)</p>
    <p>An oligonucleotide having the nucleotide sequence shown by SEQ ID: No.3 and that having the nucleotide sequence shown by SEQ ID: No.4 were prepared. The oligonucleotides were synthesized using a DNA synthesizer (PE Applied Systems: Model 394 DNA/RNA Synthesizer), and using as a solvent for synthesis a solvent for Model 394 DNA/RNA Synthesizer (PE Applied Systems) and using as a DNA synthesis reagent phosphoamidite reagents (PE Applied Systems) corresponding to adenine, cytosine, guanine and thymine. The synthesized oligonucleotides were purified by an oligonucleotide purifying cartridge (PE Applied Systems: OPC cartridge) and dried under reduced pressure to prepare oligonucleotides.</p>
    <p>A library comprising λ ZAPII vector having inserted cDNA derived from a rat liver (manufactured by STRATAGENE) (hereinafter referred to as “rat cDNA library”) was spread over several plates in NZCYM agar medium to amplify according to a method described in Molecular Cloning 2<sup>nd </sup>edition (authors: J. Sambrook, E. F. Frisch, T. Maniatis; Cold Spring Harbor Laboratory Press, 1989), 2.60-2.65 and phage particles were eluted with an SM buffer from the agar medium per each plate (hereinafter referred to as “amplified library”). A DNA was extracted from the amplified library using a DNA extracting kit (Lambda-TRAP PLUS: manufactured by Clontech) to prepare a phage cloned DNA. A polymerase chain reaction was performed using this phage cloned DNA as a template to amplify the DNA fragment. A reaction solution for polymerase chain reaction was prepared by taking 10 pmol of an oligonucleotide having the nucleotide sequence shown by SEQ ID: No. 3, 10 pmol of an oligonucleotide having the nucleotide sequence shown by SEQ ID: No. 4, 5 μl 10×PCR buffer (manufactured by TAKARA SHUZO CO., LTD), 0.25 μl Taq DNA polymerase (TaKaRa Taq manufactured by TAKARA SHUZO CO., LTD), each 10 nmol of four kinds of nucleotides (dATP, dCTP, dGTP, dTTP: manufactured by TAKARA SHUZO CO., LTD), and 10 ng of the phage clone DNA in a 0.5 ml volume of a polymerase chain reaction tube and adding sterile distilled water to total 50 μl. Each step of polymerase chain reaction was carried out under the following conditions: The first cycle comprising a denaturing step holding a temperature at 95° C. for 1 minute, an annealing step holding a temperature at 55° C. for 2 minutes, and an extension step with a DNA polymerase holding a temperature at 72° C. for 3 minutes was performed once, and the second cycle comprising a denaturing step holding a temperature at 95° C. for 1 minute, an annealing step holding a temperature at 55° C. for 1.5 minutes, and an extentsion step with a polymerase holding a temperature at 72° C. for 2 minutes was performed 34 times. After completion of the polymerase chain reaction, the reaction solution was analyzed on an agarose gel electrophoresis to select an amplified library from which about 700 bp amplified DNA fragment is detected. the nucleotide sequence shown by SEQ ID: No.3, 10 pmol of an oligonucleotide having the nucleotide sequence shown by SEQ ID: No. 4, 5, μl 10×PCR buffer (manufactured by TAKARA SHUZO CO., LTD), 0.25, μl TaKaRa Taq (manufactured by TAKARA SHUZO CO., LTD), each 10 nmol of four kinds of nucleotides (dATP, dCTP, dGTP, dTTP: manufactured by TAKARA SHUZO CO., LTD), and 10 ng of the phage clone DNA in a 0.5 ml volume of a polymerase chain reaction tube and adding sterile distilled water to total 50 μl. Each step of polymerase chain reaction was carried out under the following conditions: The first cycle comprising a denaturing step holding a temperature at 95° C. for 1 minute, an annealing step holding a temperature at 55° C. for 2 minutes, and an extension step with a DNA polymerase holding a temperature at 72° C. for 3 minutes was performed once, and the second cycle comprising a denaturing step holding a temperature at 95° C. for 1 minute, an annealing step holding a temperature at 55° C. for 1.5 minutes, and an extension step with a polymerase holding a temperature at 72° C. for 2 minutes was performed 34 times. After completion of the polymerase chain reaction, the reaction solution was analyzed on an agarose gel electrophoresis to select an amplified library from which about 700 bp amplified DNA fragment is detected.</p>
    <p>Next, <i>Escherichia coli </i>was infected with the selected amplified library and pBluescript vector containing a rat cDNA was prepared according to the description of manual attached to the above rat cDNA library, which was transformed into <i>Escherichia coli </i>to make an <i>Escherichia coli </i>library. Then, this <i>Escherichia coli </i>library was used to make a membrane for hybridization according to a method described in Recombinant DNA Experiment (editted by J. R. Dillon, A Nasim, E. R. Nestmann, Tokyo Kagakudozin, 1987), pp98 to 100. Further, this membrane was used to perform hybridization using DIG-High Prime DNA Labeling and Detection Starter Kit I (manufactured by Boehringer Mannheim) and reagents attached thereto and according to the protocol attached thereto. As a probe, a DNA fragment having the nucleotide sequence shown by nucleotide Nos. 639 to 1333 in SEQ ID; No. 2. As a result, two clones which strongly hybridize with the probes were obtained.</p>
    <p>The nucleotide sequence of the DNA fragment harbored by the resultant clone was determined using Dye terminator cycle sequencing kit (manufactured by PE Applied Systems) and a DNA sequencer 373S (manufactured by PE Applied Systems). As a result, the nucleotide sequence shown by nucleotide Nos.270 to 1636 in SEQ ID: No. 2 was revealed and it was found that the resultant two clones harbor the DNA fragments having the same nucleotide sequence. In addition, since a translation initiation codon is missing in the determined base sequence, a PPO gene encoded in the cloned DNA fragment was found to lack a 5′ upstream region containing a translation initiation codon.</p>
    <heading>Example 2</heading> <p>(Cloning of a Full Length PPO Gene)</p>
    <p>In order to clone a 5′ upstream region of a PPO gene which is missing in the PPO DNA fragment obtained in Example 1, a polymerase chain reaction was performed using as a template a DNA extracted from a rat cDNA library to amplify the DNA fragment. A reaction solution for polymerase chain reaction was prepared by taking 10 pmol of an oligobase having the nucleotide sequence shown by SEQ ID: No. 4, 10 pmol T3 primer (manufactured by TAKARA SHUZO CO., LTD), 0.5 μl long amplifying Taq DNA polymerase (TaKaRa LA Taq manufactured by TAKARA SHUZO CO., LTD), 5 μl 10×LA PCR buffer (manufactured by TAKARA SHUZO CO., LTD), each 20 nmol of four kinds of nucleotides (dATP, dCTP, dGTP, dTTP: manufactured by Clontech), and 10 ng of phage cloned DNA in 0.5 ml volume of a polymerase chain reaction tube and adding sterile distilled water to total 50 μl. Each step of polymerase chain reaction was carried out under the following conditions: The first cycle comprising a denaturing step holding a temperature at 95° C. for 1 minute, an annealing step holding a temperature at 55° C. for 2 minutes, and an extension step with a DNA polymerase holding a temperature at 72° C. for 3 minutes was performed once, and the second cycle comprising a denaturing step holding a temperature at 95° C. for 1 minute, an annealing step holding a temperature at 55° C. for 1.5 minutes, and an extension step with a polymerase holding a temperature at 72° C. for 2 minutes was performed 34 times. After completion of the polymerase chain reaction, the reaction solution was filtered by a spin column (MicroSpin S-400HR manufactured by Pharmacia Biotech) to purify the DNA fragments amplified by the polymerase chain reaction. (manufactured by TAKARA SHUZO CO., LTD), 5 μl 10×LA PCR buffer (manufactured by TAKARA SHUZO CO., LTD), each 20 nmol of four kinds of nucleotides (dATP, dCTP, dGTP, dTTP: manufactured by Clontech), and 10 ng of phage cloned DNA in a 0.5 ml volume of a polymerase chain reaction tube and adding sterile distilled water to total 50 μl. Each step of polymerase chain reaction was carried out under the following conditions: The first cycle comprising a denaturing step holding a temperature at 95° C. for 1 minute, an annealing step holding a temperature at 55° C. for 2 minutes, and an extension step with a DNA polymerase holding a temperature at 72° C. for 3 minutes was performed once, and the second cycle comprising a denaturing step holding a temperature at 95° C. for 1 minute, an annealing step holding a temperature at 55° C. for 1.5 minutes, and an extension step with a polymerase holding a temperature at 72° C. for 2 minutes was performed 34 times. After completion of the polymerase chain reaction, the reaction solution was filtered by MicroSpin S-400HR (manufactured by Pharmacia Biotech) to purify the DNA fragments amplified by the polymerase chain reaction.</p>
    <p>Next, 2 μl of the purified solution of the DNA fragment amplified by the above polymerase chain reaction, 50 ng of linearized pCR2.1 (manufactured by Invitrogen), 1 μl of Ligation buffer (manufactured by Invitrogen), and 4 Weiss units of T4 DNA Ligase (manufactured by Invitrogen) were taken into 1.5 ml microtube, a sterile distilled water was added thereto to total 10 μl to mix, a DNA ligation reaction was performed by holding a temperature at 14° C. for 16 hours to ligate the DNA fragment amplified by the polymerase chain reaction to pCR2.1. After completion of ligation reaction, a competent cell of <i>Escherichia coli </i>strain INV α F(manufactured by Invitrogen) was transformed with 1 μl of the reaction solution to select a strain which became ampicillin-resistant. Further, plasmid DNA was extracted from the selected strain and they were cut with restriction enzymes and analyzed on an agarose electrophoresis to select a plasmid in which about 1.4 kbp DNA fragment amplified by polymerase chain reaction was cloned.</p>
    <p>The nucleotide sequence of the DNA fragment harbored by the selected plasmid was determined using Dye terminator cycle sequencing kit (PE Applied Bio Systems) and a DNA sequencer 373S (manufactured by PE Applied Bio Systems). The determined nucleotide sequence was analyzed and found that the cloned 1.4 kbp DNA fragment overlap with the DNA fragment obtained in Example 1 by about 1.1 kbp and contain about 150 bp upstream from translation initiation point.</p>
    <p>The above plasmid having about 1.4 kbp DNA fragment was cut with a restriction enzyme, EcoRI and HincII (both manufactured by TAKARA SHUZO CO., LTD) to recover about 550 bp fragment. On the other hand, the plasmid harbored by the clone obtained in Example 1 was cut with a restriction enzyme, EcoRI and HincII (both manufactured by TAKARA SHUZO CO., LTD) and its 5′ terminal was dephosphorylated with calf intestine alkaline phosphatase (manufactured by TAKARA SHUZO CO., LTD).</p>
    <p>The aforementioned two DNA fragments were taken in a microtube, and ligated using a DNA ligation kit (manufactured by TAKARA SHUZO CO., LTD). <i>Escherichia coli </i>HB101 competent cell (manufactured by TAKARA SHUZO CO., LTD) was transformed with the resultant reaction solution and selected a strain which became ampicillin-resistant. The plasmid harbored by the selected strain was cut with restriction enzymes EcoRI and XhoI (both manufactured by TAKARA SHUZO CO., LTD), and analyzed on an agarose gel electrophoresis to select a plasmid into which about 1.7 kbp DNA fragment is cloned.</p>
    <p>The nucleotide sequence of the DNA fragment harbored by the resultant clone was determined using Dye terminator cycle sequencing kit (manufactured by PE Applied Bio Systems) and a DNA sequencer 373S (manufactured by PE Applied Bio Systems). As a result, the nucleotide sequence shown by SEQ ID: No.2 was revealed and the cloned DNA fragment was found to contain a full length structural gene (1431 bp) encoding a rat-derived PPO.</p>
    <heading>Example 3</heading> <p>(Analysis of PPO Amino Acid Sequence)</p>
    <p>The nucleotide sequence of the rat-derived PPO gene cDNA determined in Example 2 was analyzed to translate the amino acid sequence using genetic analyzing software (GENETYX: manufactured by SDC). As a result, a protein encoded by the cloned rat-derived PPO gene cDNA was found to be composed of 477 amino acid residues and its amino acid sequence was that shown by SEQ ID: No. 1.</p>
    <heading>Example 4</heading> <p>(Construction of a Vector for Expressing a Rat PPO Gene in <i>Escherichia coli</i>)</p>
    <p>An oligonucleotide having the nucleotide sequence shown by SEQ ID: No. 5 and that having the nucleotide sequence shown by SEQ ID: No. 6 were prepared. The oligonucleotides were synthesized using a DNA synthesizer (PE Applied Systems: Model 394 DNA/RNA Synthesizer) and using a solvent for Model 394 DNA/RNA Synthesizer as a solvent (PE Applied Systems) and using as a DNA synthesizing reagent phosphoamidite reagents corresponding to adenine, cyctosine, guanine and thymidine (PE Applied Systems). The synthesized oligonucleotides were purified by an oligonucleotide purifying cartridge (PE Applied Systems: OPC cartridge) and dried under reduced pressure to prepare oligonucleotides.</p>
    <p>A polymerase chain reaction was performed using as a template a full length gene cDNA encoding the rat-derived PPO obtained in Example 2 and as a primer an oligonucleotide having the nucleotide sequence shown by SEQ ID: No. 5 and that having the nucleotide sequence shown by SEQ ID: No. 6 to amplify the about 1.5 kbp DNA fragment encoding PPO. A reaction solution in the polymerase chain reaction was prepared by taking 10 pmol of an oligonucleotide having the nucleotide sequence shown by SEQ ID; No. 5, 10 pmol of an oligonucleotide having the nucleotide sequence shown by SEQ ID: No. 6, 0.5 μl of long-amplifying Taq DNA polymerase (TaKaRa LA Taq manufactured by TAKARA SHUZO CO., LTD.), 5.0 μl 10×LA PCR buffer (manufactured by TAKARA SHUZO CO., LTD), each 20 nmol of four kinds of nucleotides (dATP, dCTP, dGTP and dTTP; manufactured by Clontech), and 10 ng of a plasmid containing full length cDNA of the rat PPO obtained in Example 2 in a 0.5 ml volume of a polymerase chain reaction tube and adding sterile distilled water to total 50 μl. Each step in the polymerase chain reaction was performed under the following conditions: The first cycle comprising a denaturing step holding a temperature at 95° C. for 1 minute, an annealing step holding a temperature at 55° C. for two minutes, and extension step with a DNA polymerase holding a temperature at 72° C. for three minutes was performed once, and the second cycle comprising a denaturing step holding a temperature at 95° C. for 1 minute, an annealing step holding a temperature at 55° C. for 1.5 minutes, and an extension step holding a temperature at 72° C. for 2 minutes was performed 34 times.</p>
    <p>After the polymerase chain reaction, the DNA fragment amplified by the polymerase chain reaction was purified by filtering the reaction solution with a spin column (MicroSpin S-400HR manufactured by Pharmacia Biotech). A terminus of this DNA fragment was cut with restriction enzymes SacII and SmaI. On the other hand, pBluescript II SK+(manufactured by Stratagene) was cut with restriction enzymes SacII and SmaI (both manufactured by TAKARA SHUZO CO., LTD) and the 5′ end was dephosphorylated with calf intestine alkaline phosphatase (manufactured by TAKARA SHUZO CO., LTD).</p>
    <p>The aforementioned two DNA fragments were taken in a microtube, and ligated using a DNA ligation kit (manufactured by TAKARA SHUZO CO., LTD). A competent cell of <i>Escherichia coli </i>JM109 (manufactured by TAKARA SHUZO CO., LTD) was transformed with the resultant reaction solution to select a strain which became ampicillin-resistant and a plasmid DNA was prepared from the ampicillin-resistant strain to obtain a vector for expressing rat PPO gene in <i>Escherichia coli. </i> </p>
    <heading>Example 5</heading> <p>(Complementation Experiment of <i>Escherichia coli </i>hemG-deficient Strain with Rat PPO Gene)</p>
    <p>The expression vector obtained in Example 4 was introduced into <i>Escherichia coli </i>PPO gene (hemG locus)-deficient mutant strain BT3, which was spread-inoculated on LB agar medium containing kanamycin at final concentration of 10 μg/ml and ampicillin at final concentration of 50 μg/ml to culture at 37° C. for 2 days. BT3 strain became less proliferative and formed only small colonies, while a strain having the introduced vector for expressing rat PPO gene formed relatively large colonies on the LB plate.</p>
    <heading>Example 6</heading> <p>(Test on the Ability of a Compound to Inhibit the Rat PPO Activity)</p>
    <p>An overnight cultured bacterial solution of <i>Escherichia coli </i>strain BT3 into which a introduced vector for expressing the rat PPO gene obtained in Example 5 is diluted to turbidity(OD<sub>600</sub>) of 1.0. 0.05% medium volume equivalent amount of this diluted bacterial solution is inoculated on YPT liquid medium containing no various test compounds and also is inoculated on YPT liquid medium containing various compounds at a variety of concentrations to culture by shaking at 37° C. and turbidity (OD<sub>600</sub>) 16 hours after culturing is measured. The concentration at which 50% growth inhibition by various test compounds is seen is calculated by adopting turbidity of the system having no addtion of test compound as 100%. By comparing the concentrations in various test compounds, the PPO inhibiting abilities of test compounds are determined.</p>
    <heading>Example 7</heading> <p>(Obtaining of <i>Escherichia coli </i>for Expressing Rat PPO Gene, into which lac Repressor Gene is Introduced)</p>
    <p>pBR322 was cut with restriction enzymes HindIII and AvaI (both manufactured by TAKARA SHUZO CO., LTD) to recover the about 1.4 kb fragment. Separately, pREP4 was cut with restriction enzymes HindIII and AvaI (both manufactured by TAKARA SHUZO CO., LTD) to recover the about 2.4 kb fragment and its 5′end was dephosphorylated with calf intestine alkaline phosphatase (manufactured by TAKARA SHUZO CO., LTD).</p>
    <p>The aforementioned two DNA fragments were taken in a microtube and ligated using a DNA ligation kit (manufactured by TAKARA SHUZO CO., LTD). A competent cell (manufactured by TAKARA SHUZO CO., LTD) of <i>Escherichia coli </i>strain JM109 was transformed with the resultant reaction solution to select a strain which became kanamycin and tetracyclin-resistant and a plasmid DNA was prepared from the kanamycin and tetracyclin-resistant strain to obtain a vector for expressing lac repressor. The vector was introduced into <i>Escherichia coli </i>strain BT3 to select a strain which became kanamycin , tetracyclin and ampicillin-resistant. Further, the expression vector obtained in Example 4 was introduced into the tetracyclin-resistant strain to select a strain which became kanamycin and tetracyclin- and ampicillin-resistant to obtain <i>Escherichia coli </i>strain BT3 into which the rat PPO gene and lac repressor gene were introduced.</p>
    <heading>Example 8</heading> <p>(Assaying of the Ability of a Compound to Inhibit the PPO Activity Using <i>Escherichia coli </i>Strain BT3 for Expressing Rat PPO Gene, into which lac Repressor is Introduced)</p>
    <p>An overnight cultured bacterial solution of <i>Escherichia coli </i>strain BT3 for expressing a rat PPO gene, into which lac repressor was introduced, obtained in Example 7 was diluted to turbidity (OD<sub>600</sub>) of 0.5. 0.1% medium volume equivalent amount of this diluted bacterial solution was inoculated on YPT liquid medium containing no test compound and also was inoculated on YPT liquid medium containing various test compounds at various concentrations to culture by shaking at 37° C. and turbidity (OD<sub>600</sub>) 20 hours after culturing was measured. The concentration at which 50% growth inhibition by a compound having the structure shown in FIG. 4 was seen was calculated and found to be 0.11 ppm under the condition that the growth rate of the transformant is 100% in the system of no addition of test compound.</p>
    <heading>Example 9</heading> <p>(Cloning of <i>Chlamydomonas reinhardtii </i>PPO DNA Fragment)</p>
    <p> <i>Chlamydomonas reinhardtii </i>strain CC407 was obtained from Chlamydomonas Genetics Center (address: DCMB Group, Department of Botany, Box 91000, Duke University, Durham, N.C. 27708-1000, USA), and cultured in TAP liquid medium comprising 7 mM NH<sub>4</sub>Cl, 0.4 mM MgSO<sub>4</sub>.7H<sub>2</sub>O, 0.34 mM CaCl<sub>2</sub>, 25 mM potassium calcium, 0.5 mM Tris (pH 7.5), 1 ml/L Hutner trace elements, and 1 ml/L glacial acetic acid (E. H. Harris, The Chlamydomonas Sourcebook, Academic Press, San Diego, 1989, pp576-577) under the light (200 μE/m<sup>2</sup>/s<sup>1</sup>) for 5 days to obtain 200 ml of culture containing cells at early stationary proliferation stage (1.0×10<sup>6 </sup>cells/ml).</p>
    <p>The whole RNA was prepared from the cells by performing the manipulation using ISOGEN (Nippon gene) according to the attached manual. Further, poly(A) RNA was fractionated using BioMag mRNA Purification Kit (Perceptive Biosystems) by performing the manipulation according to the attached manual. From the resultant poly(A)RNA, a cDNA was synthesized using Marathon cDNA Amplification Kit (Clontech) by performing the manipulation according to the attached manual and this was used as a template for polymerase chain reaction.</p>
    <p>An oligonucleotide having the nucleotide sequence shown by SEQ ID: No. 11 and that having the nucleotide sequence shown by SEQ ID: 12 were prepared as a primer for polymerase chain reaction. The oligonucleotides were synthesized using a DNA synthesizer (PE Applied Systems: Model 394 DNA/RNA Synthesizer) and using as a solvent for synthesis a solvent for Model 394 DNA/RNA Synthesizer (PE Applied Systems) and using as a reagent for synthesizing DNA phosphoamidite reagents corresponding to adenine, cytosine, guanine and thymine (PE Applied Systems). Synthesized oligonucleotides were purified with oligonucleotide purifying cartridge (PE Applied Systems: OPC cartridge) and dried under reduced pressure to prepare an oligonucleotide.</p>
    <p>A polymerase chain reaction was performed by preparing a reaction solution using Advantage cDNA PCR kit (Clontech) according to the attached manual and repeating once a cycle comprising 94° C. for 1 minute and 70° C. for 4 minutes, four times a cycle comprising 94° C. for 10 seconds, then 70° C. for 4 minutes, five times a cycle comprising 94° C. for 10 seconds, then 68° C. for 4 minutes, and 25 times a cycle comprising 94° C. for 10 seconds, then 65° C. for 5 minutes, and an aliquot of the reaction solution was subjected to agarose gel electrophoresis to confirm that about 2 kbp amplified fragment is obtained. Further, the excess primers in the reaction solution were removed by performing the manipulation using a spin column (MicroSpin S400HR Pharmacia Biotech) according to the attached manual and the manipulation was performed using TA Cloning Kit (Invitrogen) according to the attached manual to clone the amplified fragment into pCR2.1 plasmid.</p>
    <p>The nucleotide sequence of the DNA fragment harbored by the resultant recombined plasmid was determined using Dye terminator cycle sequencing kit (manufactured by PE Applied Bio Systems) and a DNA sequencer 373S (manufactured by PE Applied Bio Systems). As a result, the nucleotide sequence shown by SEQ ID; No. 10 was revealed and it was found to be full length cDNA encoding the amino acid sequence shown by SEQ ID: No. 9 by the analysis using Genetic Analyzing software GENETYX (SDC).</p>
    <heading>Example 10</heading> <p>(Construction of a Vector for Expressing a <i>Chlamydomonas reinhardtii </i>PPO Gene in <i>Escherichia coli</i>)</p>
    <p>An oligonucleotide having the nucleotide sequence shown by SEQ ID: No. 13, in which a restriction enzyme SacI recognition sequence is introduced at an end, and an oligonucleotide having the nucleotide sequence shown by SEQ ID: No. 14, in which a restriction enzyme SalI recognition sequence is introduced at an end were prepared. The oligonucleotides were synthesized using a DNA synthesizer (PE Applied Systems: Model 394 DNA/RNA Synthesizer) and as a solvent for synthesis a solvent for Model 394 DNA/RNA Synthesizer (PE Applied Systems) and using as a reagent for DNA synthesis a phosphoamidite reagent corresponding to adenine, cytosine, guanine, and thymine (PE Applied Systems). The synthesized oligonucleotides were purified by an oligonucleotide purifying cartridge (PE Applied Systems: OPC cartridge) and dried under reduced pressure to prepare an oligonucleotide.</p>
    <p>A polymerase chain reaction was performed using as a template a full length gene cDNA encoding PPO derived from <i>Chlamydomonas reinhardtii </i>obtained in Example 9 and as a primer an oligonucleotide having the nucleotide sequence shown by SEQ ID: No. 13 and that having the nucleotide sequence shown by SEQ ID: No 14 to amplify the about 2 kbp DNA fragment encoding PPO. A reaction solution in the polymerase chian reaction was prepared by taking 10 pmol of an oligonucleotide having the nucleotide sequence shown by SEQ ID: No. 13, 10 pmol of an oligonucleotide having the nucleotide sequence shown by SEQ ID: No. 14, 0.5 μl of long-amplifying Taq DNA polymerase (TaKaRa LA Taq manufactured by TAKARA SHUZO CO., LTD), 5.0 μl of 10×LA PCR buffer (manufactured by TAKARA SHUZO CO., LTD), each 20 nmol of four kinds of nucleotides (dATP, dCTP, dGTP, dTTP: manufactured by Clontech), 10 ng of a plasmid containing full length cDNA of <i>Chlamydomonas reinhardtii </i>PPO obtained in Example 9 in 0.5 ml volume of a polymerase chain reaction tube and adding sterile distilled water to total 50 μl. Each step in the polymerase chain reaction was performed under the following conditions: After the first cycle comprising a denaturing step holding a temperature at 95° C. for 1 minute, an annealing step holding a temperature at 55° C. for 2 minutes, and an extension step with a DNA polymerase holding a temperature 72° C. for 3 minutes was performed once, the second cycle comprising a denaturing step holding a temperature at 95° C. for 1 minute, an annealing step holding a temperature at 55° C. for 15 minutes, and an extension step holding a temperature at 72° C. for 2 minutes was performed 34 times. (manufactured by TAKARA SHUZO CO., LTD), 5.0 μl of 10×LA PCR buffer (manufactured by TAKARA SHUZO CO., LTD), each 20 nmol of four kinds of nucleotides (dATP, dCTP, dGTP, dTTP: manufactured by: Clontech), 10 ng of a plasmid containing full length cDNA of <i>Chlamydomonas reinhardtii </i>PPO obtained in Example 9 in 0.5 ml volume of a polymerase chain reaction tube and adding sterile distilled water to total 50 μl. Each step in the polymerase chain reaction was performed under the following conditions; After the first cycle comprising a denaturing step holding a temperature at 95° C. for 1 minute, an annealing step holding a temperature at 55° C. for 2 minutes, and an extension step with a DNA polymerase holding a temperature at 72° C. for 3 minutes was performed once, the second cycle comprising a denaturing step holding a temperature at 95° C. for 1 minute, an annealing step holding a temperature at 55° C. for 15 minutes, and an extension step holding a temperature at 72° C. for 2 minutes was performed 34 times.</p>
    <p>After the polymerase chain reaction, the DNA fragment amplified by the polymerase chain reaction was purified by filtering the reaction solution with MicroSpin S-400HR (manufactured by Pharmacia Biotech). The end of this DNA fragment was cut with restriction enzymes SacI and SalI. On the other hand, the plasmid pUC118 (manufactured by TAKARA SHUZO CO., LTD) was cut with restriction enzymes SacI and SalI (both manufactured by TAKARA SHUZO CO., LTD) and 5′end was dephosphorylated with calf intestine alkaline phosphatase (manufactured by TAKARA SHUZO CO., LTD).</p>
    <p>The aforementioned two DNA fragments were taken in a microtube and ligated using a DNA ligation kit (manufactured by TAKARA SHUZO CO., LTD). A competent cell (manufactured by TAKARA SHUZO CO., LTD) of <i>Escherichia coli </i>JM109 was transformed with the resultant reaction solution to select a strain which became ampicillin-resistant and a plasmid DNA was prepared from the ampicillin-resistant strain to obtain a vector for expressing <i>Chlamydomonas reinhardtii </i>PPO gene.</p>
    <heading>Example 11</heading> <p>(Complementation Experiment of <i>Escherichia coli </i>hemG-deficient Strain with <i>Chlamydomonas reinhardtii </i>PPO Gene)</p>
    <p>The expression vector obtained in Example 10 was introduced into PPO gene (hemG locus)-deficient mutant <i>Escherichia coli </i>strain BT3 described in F. Yamamoto, H. Inokuchi, H. Ozeki, (1988) Japanese Journal of Genetics, vol.63, pp237 to 249, which was spread-inoculated on LB agar medium containing kanamycin to the final concentration of 10 μg/ml and ampicillin to the final concentration of 50 μg/ml to culture at 37° C. for 2 days. <i>Escherichia coli </i>BT3 strain became less proliferative and formed small colonies, while a strain having introduced vector for expressing <i>Chlamydomonas reinhardtii </i>PPO gene formed relatively large colonies on the LB plate.</p>
    <heading>Example 12</heading> <p>(Construction of a PPO Gene Expressing Vector for Direct Introduction)</p>
    <p>In order to express a rat-derived PPO gene in a plant cell, a PPO gene expressing vector for directly introducing a plant is constructed.</p>
    <p>An oligonucleotide having the nucleotide sequence shown by SEQ ID: No. 7 and that having the nucleotide sequence shown by SEQ ID: No.8 are prepared. The oligonucleotides are synthesized using a DNA synthesizer (PE Applied Systems: Model 394 DNA/RNA Synthesizer) and using as a solvent for DNA synthesis a solvent for Model 394 DMA/RNA Synthesizer (PE Applied Systems) and as a reagent for synthesis phosphoramidite reagents corresponding to adenine, cytosine, guanine and thymine (PE Applied Systems). The synthesized oligonucleotides are purified by an oligonucleotide purifying cartridge (PE Applied Systems: OPC cartridge) and dried under reduced pressure to prepare an oligonucleotide.</p>
    <p>A polymerase chain reaction is performed using as a template a full length cDNA of the rat-derived PPO obtained in Example 2, and using as a primer an oligonucleotide having the nucleotide sequence shown in SEQ ID: No. 7 and that having the nucleotide sequence shown in SEQ ID: No. 8 to amplify the about 1.5 kbp DNA fragment encoding the rat PPO. The polymerase chain reaction is performed by adding 10 pmol of an oligonucleotide having the nucleotide sequence shown in SEQ ID: No. 7, 10 pmol of an oligonucleotide having the nucleotide sequence shown in SEQ ID: No. 8, 0.5 μl of Advantage KlenTaq Polymerase Mix (manufactured by Clontech), 2.5 μl of 10×KlenTaq PCR reaction buffer (manufactured by Clontech), each 5 nmol of four kinds of nucleotides (dATP, dCTP, dGTP, dTTP: manufactured by Clontech), and 10 ng of the full length of cDNA of the rat-derived PPO obtained in Example 2 in a 0.2 ml volume of a polymerase chain reaction tube to total amount of 25 μl. Each step in the polymerase chain reaction is performed under the following conditions: After the first cycle comprising a denaturing step holding a temperature at 94° C. for 1 minute, and an annealing step and extension step with a DNA polymerase holding a temperature at 65° C. for 4 minutes is performed once, the second cycle comprising a denaturing step holding a temperature at 94° C. for 30 seconds, and an annealing step and extension step with a DNA polymerase holding a temperature at 65° C. for 4 minutes is performed 15 times. After completion of the polymerase chain reaction, the DNA fragment amplified in the polymerase chain reation is purified by filtering the reaction solution with a spin column (MicroSpin S-400HR manufactured by Pharmacia Biotech). After the end of this DNA fragment is made blunt with a DNA blunting kit (manufactured by TAKARA SHUZO CO., LTD), a phosphate group is added to the 5′ end with T4 polynucleotide kinase (manufactured by TAKARA SHUZO CO., LTD). times. After completion of the polymerase chain reaction, the DNA fragment amplified in the polymerase chain reaction is purified by filtering the reaction solution with MicroSpin S-400HR (manufactured by Pharmacia Biotech). After the end of this DNA fragment is made blunt with a DNA blunting kit (manufactured by TAKARA SHUZO CO., LTD), a phosphate group is added to the 5′end with T4 polynucleotide kinase (manufactured by TAKARA SHUZO CO., LTD).</p>
    <p>On the other hand, after the GUS expression vector pBI221 derived from pUC19 (manufactured by Clontech) is cut with restriction enzymes SmaI and SacI (both manufactured by TAKARA SHUZO CO., LTD) to recover the 2.8 kbp DNA fragment from which the GUS structural gene has been removed and its end is made blunt using a DNA blunting kit (manufactured by TAKARA SHUZO CO., LTD), the DNA fragment is dephosphorylated with bacterial alkaline phosphatase (manufactured by TAKARA SHUZO CO., LTD).</p>
    <p>The aforementioned two DNA fragments are taken in a microtube and ligated using a DNA ligation kit (manufactured by TAKARA SHUZO CO., LTD). A competent cell of <i>Escherichia coli </i>strain HB101 (manufactured by TAKARA SHUZO CO., LTD) is transformed with the resultant reaction solution to select a strain which has become ampicillin-resistant. Further, a plasmid in which a coding region for the rat-derived PPO is inserted in a forward direction relative to the cauliflower mosaic virus-derived 35S promoter and the nopaline synthase-derived terminator is selected from plasmids contained in the selected strain, to obtain a rat PPO gene expression vector for directly introducing into a plant.</p>
    <heading>Example 13</heading> <p>(Construction of a PPO Gene Expression Vector for Indirect Introduction)</p>
    <p>In order to express a rat-derived PPO gene to in a plant cell, a PPO gene expression vector for indirect introduction into a plant is constructed.</p>
    <p>An oligonucleotide having the nucleotide sequence shown by SEQ ID: No.7 and an oligonucleotide having the nucleotide sequence shown by SEQ ID: No.8 are prepared. The oligonucleotides are synthesized using a DNA synthesizer (PE Applied Systems: Model 394 DNA/RNA Synthesizer), using a solvent for synthesis a solvent for Model 394 DNA/RNA Synthesizer (PE Applied Systems) and using as a reagent for DNA synthesis phosphoamidite reagents corresponding to adenine, cytosine, guanine and thymine (PE Applied Systems). After the synthesized oligonucleotides are purified with an oligonucleotide purifying cartridge (PE Applied Systems: OPC cartridge), they are dried under reduced pressure to prepare the oligonucleotides.</p>
    <p>A polymerase chain reaction is performed using as a template the full length cDNA for the rat-derived PPO obtained in Example 2, and using an oligonucleotide having the nucleotide sequence shown in SEQ ID: No. 7 and that having the base sequence shown in SEQ ID: No. 8 to amplify the about 1.5 kbp DNA fragment encoding the rat-derived PPO. The polymerase chain reaction is performed by adding 10 pmol of an oligonucleotide having the nucleotide sequence shown by SEQ ID: No. 7, 10 pmol of an oligonucleotide having the nucleotide sequence shown by SEQ ID: No. 8, 0.5 μl of Advantage KlenTaq Polymerase Mix (manufactured by Clontech), 2.5 μl 10×KlenTaq PCR reaction buffer (manufactured by Clontech),each 5 n mol of four kinds of nucleotides (dATP, dCTP, dGTP, dTTP, manufactured by Clontech), and 10 ng of a DNA containing the full length cDNA for the rat-derived PPO obtained in Example 2 to 0.2 μl volume of a polymerase chain reaction tube to total volume of 25 μ. Each step of the polymerase chain reaction is performed as follows: After the first cycle comprising a denaturing step holding a temperature at 94° C. for 1 minute, and an annealing step and an extension step with a DNA polymerase holding a temperature at 65° C. for 4 minutes is performed once, the second step comprising a denaturing step holding a temperature at 94° C. for 30 seconds, an annealing step and an extension step with a DNA polymerase holding a temperature at 65° C. for 4 minutes is performed 15 times. After completion of the polymerase chain reaction, the DNA fragment amplified by polymerase chain reaction is purified by filtering the reaction solution with MicroSpin S-400HR (manufactured by Pharmacia Biotech). After the end of this DNA fragment is made blunt with a DNA blunting kit (manufactured by TAKARA SHUZO CO., LTD), a phosphate group is added to the 5′end with the T4 polynucleotide kinase (manufactured by TAKARA SHUZO CO., LTD).</p>
    <p>On the other hand, after the pBIN19-derived GUS expressing binary vector pBI121 (manufactured by Clontech) is cut with restriction enzymes SmaI and SacI (both manufactured by TAKARA SHUZO CO., LTD) to recover the DNA fragment from which the GUS structural gene has been removed and its end is made blunt using a DNA blunting kit (manufactured by TAKARA SHUZO CO., LTD), it is dephosphorylated with a bacterial alkaline phosphatase (manufactured by TAKARA SHUZO CO., LTD).</p>
    <p>The aforementioned two DNA fragments are taken in a microtube and ligated using a DNA ligation kit (manufactured by TAKARA SHUZO CO., LTD). A competent cell of <i>Escherichia coli </i>strain HB101 (manufactured by TAKARA SHUZO CO., LTD) is transformed with the resultant reaction solution to select a strain which has become kanamycin-resistant. Further, a plasmid in which a coding region for a rat PPO is inserted in a forward direction relative to the cauliflower mosaic virus-derived 35S promoter and the nopaline synthase-derived terminator from plasmids contained in the selected strain, to obtain the rat PPO gene expression vector for indirect introduction into a plant.</p>
    <heading>Example 14</heading> <p>(Construction of a Vector for Expressing a <i>Chlamydomonas reinhardtii </i>PPO Gene)</p>
    <p>In order to express a <i>Chlamydomonas reinhardtii</i>-derived PPO in a plant cell, a PPO gene expression vector for direct introduction into a plant is constructed.</p>
    <p>The pCR2.1 plasmid harboring the amplified fragment of the cDNA for the <i>Chlamydomonas reinhardtii</i>-derived PPO gene obtained in Example 9 is digested with restriction enzymes NotI and SpeI (both manufactured by TAKARA SHUZO CO., LTD) to obtain the about 2 kbp <i>Chlamydomonas reinhardtii</i>-derived PPO gene cDNA fragment having cohesive ends for NotI and SpeI at its end. On the other hand, pBluescriptII KS+ (manufactured by Stratagene) is cut with restriction enzymes NotI and SpeI (both manufactured by TAKARA SHUZO CO., LTD) and the 5′end is dephosphorylated with calf intestine alkaline phosphatase (manufactured by TAKARA SHUZO CO., LTD). The two DNA fragments are taken in a microtube and ligated using a DNA ligation kit (manufactured by TAKARA SHUZO CO., LTD). A competent cell of Escherichia coli strain HB101 (manufactured by TAKARA SHUZO CO., LTD) is transformed with the resultant reaction solution to select a strain which has become ampicillin-resistant. Further, a plasmid contained in the selected strain is selected to obtain a clone in which the <i>Chlamydomonas reinhardtii</i>-derived PPO gene cDNA is inserted into the plasmid pBluescriptII KS+.</p>
    <p>The resultant plasmid is digested with restriction enzymes BamHI and SacI (both manufactured by TAKARA SHUZO CO., LTD) to obtain the about 2 kbp <i>Chlamydomonas reinhardtii</i>-derived PPO gene cDNA fragment having cohesive ends for BamHI and SacI at its ends. On the other hand, pBI221 and pBI121 (both manufactured by Clontech) are cut with restriction enzymes BamHI and SacI (both manufactured by TAKARA SHUZO CO., LTD), respectively, to obtain a vector fragment from which a β-glucuronidase gene has been removed and the 5′end is dephosphorylated with the calf intestine alkaline phosphatase (manufactured by TAKARA SHUZO CO., LTD). The <i>Chlamydomonas reinhardtii</i>-derived PPO gene cDNA fragment and the two DNA fragments of the vector fragment of pBI221 or pBI121 are taken in a microtube and ligated using a DNA ligation kit (manufactured by TAKARA SHUZO CO., LTD). A competent cell of <i>Escherichia coli </i>strain HB101 (manufactured by TAKARA SHUZO CO., LTD) is transformed with the resultant reaction solution and a strain which has become ampicillin-resistant, in the case of ligation with the pBI221 vector fragment and, a plasmid which has become kanamycin-resistant, in the case of ligation with pBI121 vector fragment are selected. Further, a plasmid contained in the selected strain is select to obtain a plasmid for direct introduction in which the <i>Chlamydomonas reinhardtii</i>-derived PPO gene cDNA fragment is inserted into the pBI221 vector fragment or a plasmid for indirect introduction in which the <i>Chlamydomonas reinhardtii</i>-derived PPO gene cDNA fragment is inserted into the pBI121 vector fragment.</p>
    <heading>Example 15</heading> <p>(Production of a Transformed Plant in which a Rat or <i>Chlamydomonas reinhardtii</i>-Derived PPO Expression Vector is Introduced)</p>
    <p>The rat PPO gene expression vector for indirect introduction or the <i>Chlamydomonas reinhardtii </i>PPO gene expression vector for indirect induction obtained in Example 13 or Example 14 is transferred into <i>Agrobacterium tumefaciens </i>LBA4404 by a binary vector method (manufactured by Clontech: GUS Gene Fusion Ststem). A sterile-cultured tobacco leaf is infected with this bacterial strain according to Plane Gene Manipulation Manual (author: Uchimiya, Kodansha Scientific, 1990) to obtain a transformed tobacco. Similarly, a sterile-cultured carrot seedling petiole is infected therewith according to a method described in N. Pawlicki et al., (1992) Plant Cell, Tissue and Organ Culture, vol.31, pp.129 to 139 to obtain a transformed carrot.</p>
    <p>Similarly, a sterile-cultured pea seedling epicotyl or cotyledon is infected therewith according to a method described in J. Puonti-Kaerlas et al., (1990) Theoretical and Applied Genetics, vol.80, pp.246 to 252 to obtain a transformed pea.</p>
    <p>Further, the rat PPO gene expression vector for direct introduction or <i>Chlamydomonas reinhardtii </i>PPO gene expression vector for direct introduction obtained in Example 12 or Example 15 is introduced into soybean adventitious embryo with a particle gun according to a method described in JP-A-3-291501 (laid-opened) to obtain a transformed soybean. Similarly, it is introduced into a sterile-cultured rice immature scutellum with a particle gun according to a method described in Ikushugakkaishi, vol.44, suppl.No.1, pp.66, (1994)(author: Shimada, et al.) to obtain a transformed rice. Similarly, it is introduced into a sterile-cultured wheat immature scutellum with a particle gun according to the conventional method described in Ikushugakkaishi, vol.44, suppl. No.1, pp.57 (1995) (author: Takumi, et al.) to obtain a transformed wheat. Similarly, it is introduced into a sterile-cultured barley immature scutellum with a particle gun according to a method described in Ikushugakkaishi, vol.44, suppl. No.1, pp.67 (1995)(author: Hagio, et al.) to obtain a transformed barley. Similarly, it is introduced into a corn adventitious embryo with a particle gun according to a method described in M. E. Fromm et al., (1990) BIO/TECHNOLOGY, vol.8, pp.833 to 839 to obtain a transformed corn.</p>
    <p>The present invention makes it possible to provide a method for evaluating the ability of a compound to inhibit the PPO activity with simplicity.</p>
    <p>Brief Explanation of the Sequence</p>
    <p>1. SEQ ID: No.1</p>
    <p>Shows the amino acid sequence of a mitochondrial-type PPO encoded in a cDNA for a rat-derived PPO gene.</p>
    <p>2. SEQ ID: No.2</p>
    <p>Shows the nucleotide sequence of a cDNA clone for a rat-derived PPO gene.</p>
    <p>3. SEQ ID: No.3</p>
    <p>Shows the nucleotide sequence of an oligonucleotide used for amplifying a DNA fragment containing a partial nucleotide sequence of a rat-derived PPO gene.</p>
    <p>4. SEQ ID: No.4</p>
    <p>Shows the nucleotide sequence of an oligonucleotide used for amplifying a DNA fragment containing a partial nucleotide sequence of a rat-derived PPO gene.</p>
    <p>5. SEQ ID: No. 5</p>
    <p>Shows the nucleotide sequence of a primer used for constructing a vector expressing a rat-derived PPO gene in <i>Escherichia coli. </i> </p>
    <p>6. SEQ ID: No. 6</p>
    <p>Shows the nucleotide sequence of a primer used for constructing a vector expressing a rat-derived PPO gene in <i>Escherichia coli. </i> </p>
    <p>7. SEQ ID: No. 7</p>
    <p>Shows the nucleotide sequence of a primer used for constructing a rat-derived PPO gene expression vector for direct introduction and a rat-derived PPO expression vector for indirect introduction.</p>
    <p>8. SEQ ID: No.8</p>
    <p>Shows the nucleotide sequence of a primer used for constructing a rat-derived PPO gene expression vector for direct introduction and a rat-derived PPO expression vector for indirect introduction.</p>
    <p>9. SEQ ID: No. 9</p>
    <p>Shows the amino acid sequence of PPO encoded in a cDNA for <i>Chlamydomonas reinhardtii</i>-derived PPO gene.</p>
    <p>10. SEQ ID: No.10</p>
    <p>Shows the nucleotide sequence of a cDNA clone for <i>Chlamydomonas reinhardtii</i>-derived PPO gene.</p>
    <p>11. SEQ ID: No.11</p>
    <p>Shows the nucleotide sequence of an oligonucleotide used for amplifying a DNA fragment containing a <i>Chlamydomonas reinhardtii</i>-derived PPO gene.</p>
    <p>12. SEQ ID: No.12</p>
    <p>Shows the nucleotide sequence of an oligonucleotide used for amplifying a DNA fragment containing a <i>Chlamydomonas reinhardtii</i>-derived PPO gene.</p>
    <p>13. SEQ ID: No. 13</p>
    <p>Shows the nucleotide sequence of a primer used for constructing a vector expressing a <i>Chlamydomonas reinhardtii</i>-derived PPO gene in <i>Escherichia coli. </i> </p>
    <p>14. SEQ ID: No.14</p>
    <p>Shows the nucleotide sequence of a primer used for constructing a vector for expressing a <i>Chlamydomonas reinhardtii</i>-derived PPO gene in <i>Escherichia coli. </i> </p>
    <p>
      <sequence-cwu id="SEQLST-1"> <sequence-list-new-rules> <s100> <s160>14</s160> </s100> <s200> <s210>1</s210> <s211>477</s211> <s212>PRT</s212> <s213>Rattus norvegicus</s213> </s200> <s400> 1
Met Ala Arg Thr Val Ile Val Leu Gly Gly Gly Ile Ser Gly Leu Ala
  1               5                  10                  15
Ala Ser Tyr His Leu Thr Arg Ser Pro Ser Pro Pro Lys Val Ile Leu
             20                  25                  30
Val Glu Gly Ser Lys Arg Leu Gly Gly Trp Ile Arg Ser Val Arg Gly
         35                  40                  45
Ser Asp Gly Ala Ile Phe Glu Leu Gly Pro Arg Gly Ile Arg Pro Ala
     50                  55                  60
Gly Ala Leu Gly Ala Arg Thr Leu Leu Leu Val Ser Glu Leu Gly Leu
 65                  70                  75                  80
Glu Ser Glu Val Leu Pro Val Arg Gly Asp His Pro Ala Ala Gln Asn
                 85                  90                  95
Arg Phe Leu Tyr Val Gly Gly Ala Leu His Pro Leu Pro Ser Gly Leu
            100                 105                 110
Arg Gly Leu Leu Arg Pro Ser Pro Pro Phe Ser Lys Pro Leu Phe Trp
        115                 120                 125
Ala Gly Leu Arg Glu Leu Thr Lys Pro Arg Gly Lys Glu Pro Asp Glu
    130                 135                 140
Thr Val His Ser Phe Ala Gln Arg Arg Leu Gly Pro Glu Val Ala Ser
145                 150                 155                 160
Leu Ala Met Asp Ser Leu Cys Arg Gly Val Phe Ala Gly Asn Ser Gln
                165                 170                 175
Glu Leu Ser Ile Arg Ser Cys Phe Pro Ser Leu Phe Gln Ala Glu Gln
            180                 185                 190
Thr His Gly Ser Met Leu Leu Gly Leu Leu Leu Gly Ala Gly Gln Thr
        195                 200                 205
Pro Gln Pro Asn Ser Ser Leu Ile Arg Gln Ala Arg Ala Glu Arg Trp
    210                 215                 220
Ser Gln Trp Ser Leu Arg Gly Gly Leu Glu Met Leu Pro Gln Ala Leu
225                 230                 235                 240
His Asn Tyr Leu Thr Ser Lys Gly Val Thr Ile Leu Ser Gly Gln Pro
                245                 250                 255
Ala Cys Gly Leu Ser Leu Gln Pro Glu Gly His Trp Lys Val Ser Leu
            260                 265                 270
Gly Asp Ser Ser Leu Glu Ala Asp His Ile Ile Ser Thr Ile Pro Ala
        275                 280                 285
Ser Val Leu Ser Lys Leu Leu Pro Ala Glu Ala Ala Pro Leu Ala His
    290                 295                 300
Ile Leu Ser Thr Ile Gln Ala Val Ser Val Ala Val Val Asn Leu Gln
305                 310                 315                 320
Tyr Lys Gly Ala Cys Leu Pro Val Gln Gly Phe Gly His Leu Val Pro
                325                 330                 335
Ser Ser Glu Asp Pro Thr Val Leu Gly Ile Val Tyr Asp Ser Val Ala
            340                 345                 350
Phe Pro Glu Gln Asp Gly Asn Pro Pro Gly Leu Arg Leu Thr Val Met
        355                 360                 365
Leu Gly Gly Tyr Trp Leu Gln Lys Leu Lys Ala Asn Gly His Glu Leu
    370                 375                 380
Ser Pro Glu Leu Phe Gln Arg Ala Ala Gln Glu Ala Ala Ala Thr Gln
385                 390                 395                 400
Leu Gly Leu Lys Glu Gln Pro Ser His Cys Leu Val His Leu His Lys
                405                 410                 415
Asn Cys Ile Pro Gln Tyr Thr Leu Gly His Trp Gln Lys Leu Asp Ser
            420                 425                 430
Ala Leu Gln Phe Leu Thr Ala Gln Arg Leu Pro Leu Thr Leu Ala Gly
        435                 440                 445
Ala Ser Tyr Glu Gly Val Ala Val Asn Asp Cys Ile Glu Ser Gly Arg
    450                 455                 460
Gln Ala Ala Ile Ala Val Leu Gly Thr Glu Ser Asn Ser
465                 470                 475
</s400> <s200> <s210>2</s210> <s211>1638</s211> <s212>DNA</s212> <s213>Rattus norvegicus</s213> <s220> <s222>(143)...(1576)</s222> </s220> </s200> <s400> 2
cgtacacgcg cgttttgcat tagttgctca ttaatcagta agtgcccaga ggtggggtac     60
gggacccgtg gggtttctgc agttgtaaag cagggtgcct cccgttctcc tggggtatct    120
cgactttccc ccaggcctta cg atg gcc cgg act gtg ata gtg ctt ggc gga     172
                         Met Ala Arg Thr Val Ile Val Leu Gly Gly
                           1               5                  10
ggt atc agc gga ttg gcc gca agt tat cat ctg acc cga agc ccc agt      220
Gly Ile Ser Gly Leu Ala Ala Ser Tyr His Leu Thr Arg Ser Pro Ser
                 15                  20                  25
cct cct aag gtg atc tta gtg gag ggc agc aaa cgt ttg gga ggc tgg      268
Pro Pro Lys Val Ile Leu Val Glu Gly Ser Lys Arg Leu Gly Gly Trp
             30                  35                  40
atc cgt tca gtc cga gga tca gat ggt gcg atc ttt gaa ctt gga cct      316
Ile Arg Ser Val Arg Gly Ser Asp Gly Ala Ile Phe Glu Leu Gly Pro
         45                  50                  55
cga gga att agg ccg gct gga gcc ctg gga gcc cgg acc ctg ctc ctg      364
Arg Gly Ile Arg Pro Ala Gly Ala Leu Gly Ala Arg Thr Leu Leu Leu
     60                  65                  70
gtt tct gaa ctt ggc ttg gaa tcc gaa gtc ttg cct gtc cga ggg gat      412
Val Ser Glu Leu Gly Leu Glu Ser Glu Val Leu Pro Val Arg Gly Asp
 75                  80                  85                  90
cat cca gct gcc cag aac cgg ttc ctg tat gta ggc ggt gcc ctg cac      460
His Pro Ala Ala Gln Asn Arg Phe Leu Tyr Val Gly Gly Ala Leu His
                 95                 100                 105
ccc cta ccc tct ggc ctc agg ggg cta ctt cgt cct tca ccc ccc ttc      508
Pro Leu Pro Ser Gly Leu Arg Gly Leu Leu Arg Pro Ser Pro Pro Phe
            110                 115                 120
tca aaa cct cta ttt tgg gct gga ctg agg gag ttg acg aag ccc agg      556
Ser Lys Pro Leu Phe Trp Ala Gly Leu Arg Glu Leu Thr Lys Pro Arg
        125                 130                 135
ggc aaa gag cct gat gag act gtg cac agt ttt gcc cag cgc cgc ctt      604
Gly Lys Glu Pro Asp Glu Thr Val His Ser Phe Ala Gln Arg Arg Leu
    140                 145                 150
gga cct gag gtg gcg tct ctg gct atg gac agc ctt tgc aga gga gtg      652
Gly Pro Glu Val Ala Ser Leu Ala Met Asp Ser Leu Cys Arg Gly Val
155                 160                 165                 170
ttt gct ggc aac agc caa gag ctc agc atc cgg tcc tgc ttt ccc agt      700
Phe Ala Gly Asn Ser Gln Glu Leu Ser Ile Arg Ser Cys Phe Pro Ser
                175                 180                 185
ctc ttc caa gct gaa caa acc cac ggg tcc atg tta ctg ggg ctg ctg      748
Leu Phe Gln Ala Glu Gln Thr His Gly Ser Met Leu Leu Gly Leu Leu
            190                 195                 200
ctg ggg gca ggg caa act cca cag ccc aat tcc tca tta att cgt cag      796
Leu Gly Ala Gly Gln Thr Pro Gln Pro Asn Ser Ser Leu Ile Arg Gln
        205                 210                 215
gcc cgc gct gag cga tgg agt cag tgg tca ctc cgt gga ggg ctg gag      844
Ala Arg Ala Glu Arg Trp Ser Gln Trp Ser Leu Arg Gly Gly Leu Glu
    220                 225                 230
atg ttg ccc cag gcc ctt cat aac tac cta aca agt aaa ggg gtc act      892
Met Leu Pro Gln Ala Leu His Asn Tyr Leu Thr Ser Lys Gly Val Thr
235                 240                 245                 250
atc ctc agt ggt cag cca gcc tgc ggg ctc agc ctt cag cca gaa ggg      940
Ile Leu Ser Gly Gln Pro Ala Cys Gly Leu Ser Leu Gln Pro Glu Gly
                255                 260                 265
cac tgg aag gtg tct cta ggg gac agc agt ctg gag gct gac cac att      988
His Trp Lys Val Ser Leu Gly Asp Ser Ser Leu Glu Ala Asp His Ile
            270                 275                 280
ata agc acc att cca gct tca gtg ctc agc aag ctg ctc cct gcc gag     1036
Ile Ser Thr Ile Pro Ala Ser Val Leu Ser Lys Leu Leu Pro Ala Glu
        285                 290                 295
gct gca cct ctg gct cac atc ctg agt acc atc caa gct gtg tct gtg     1084
Ala Ala Pro Leu Ala His Ile Leu Ser Thr Ile Gln Ala Val Ser Val
    300                 305                 310
gcc gtg gtg aat ctg cag tac aaa gga gct tgt ctg cct gtg cag gga     1132
Ala Val Val Asn Leu Gln Tyr Lys Gly Ala Cys Leu Pro Val Gln Gly
315                 320                 325                 330
ttt gga cat ctg gtg cca tcc tca gaa gac ccg acc gtc ctg gga atc     1180
Phe Gly His Leu Val Pro Ser Ser Glu Asp Pro Thr Val Leu Gly Ile
                335                 340                 345
gtg tat gac tcg gtt gct ttt cct gag cag gat ggg aac ccc cca ggc     1228
Val Tyr Asp Ser Val Ala Phe Pro Glu Gln Asp Gly Asn Pro Pro Gly
            350                 355                 360
ctc aga ctg act gtg atg ttg gga ggt tac tgg tta cag aag ctg aaa     1276
Leu Arg Leu Thr Val Met Leu Gly Gly Tyr Trp Leu Gln Lys Leu Lys
        365                 370                 375
gcc aat ggc cat gaa ttg tct cca gag cta ttc caa cga gca gca cag     1324
Ala Asn Gly His Glu Leu Ser Pro Glu Leu Phe Gln Arg Ala Ala Gln
    380                 385                 390
gaa gcg gct gcc aca cag tta gga ctg aaa gag caa cca agc cat tgc     1372
Glu Ala Ala Ala Thr Gln Leu Gly Leu Lys Glu Gln Pro Ser His Cys
395                 400                 405                 410
ttg gtc cat cta cac aaa aac tgt atc cct cag tat aca cta ggc cac     1420
Leu Val His Leu His Lys Asn Cys Ile Pro Gln Tyr Thr Leu Gly His
                415                 420                 425
tgg caa aaa cta gac tca gct ctg caa ttc ctg acg gcc cag agg ttg     1468
Trp Gln Lys Leu Asp Ser Ala Leu Gln Phe Leu Thr Ala Gln Arg Leu
            430                 435                 440
ccc ctg act ttg gct ggg gcc tcc tat gag ggg gta gct gtc aat gac     1516
Pro Leu Thr Leu Ala Gly Ala Ser Tyr Glu Gly Val Ala Val Asn Asp
        445                 450                 455
tgt ata gag agt ggg cgc cag gca gca att gct gtc ctg ggc aca gaa     1564
Cys Ile Glu Ser Gly Arg Gln Ala Ala Ile Ala Val Leu Gly Thr Glu
    460                 465                 470
tcg aac agc tga cccccactct cctactcatg aaagtaaaag ttgatggagc         1616
Ser Asn Ser
475
ttgaaaaaaa aaaaaaaaaa aa                                            1638
</s400> <s200> <s210>3</s210> <s211>28</s211> <s212>DNA</s212> <s213>Artificial Sequence</s213> <s220> <s223>Designed oligonucleotide primer used for
      amplifying a DNA</s223> </s220> </s200> <s400> 3
tttgcagagg agtgtttgct ggcaacag                                        28
</s400> <s200> <s210>4</s210> <s211>29</s211> <s212>DNA</s212> <s213>Artificial Sequence</s213> <s220> <s223>Designed oligonucleotide primer used for
      amplifying a DNA</s223> </s220> </s200> <s400> 4
agccgcttcc tgtgctgctc gttggaata                                       29
</s400> <s200> <s210>5</s210> <s211>25</s211> <s212>DNA</s212> <s213>Artificial Sequence</s213> <s220> <s223>Designed oligonucleotide primer used for
      constructing a</s223> </s220> </s200> <s400> 5
aggccttacc gcggcccgga ctgtg                                           25
</s400> <s200> <s210>6</s210> <s211>25</s211> <s212>DNA</s212> <s213>Artificial Sequence</s213> <s220> <s223>Designed oligonucleotide primer used for
      constructing a</s223> </s220> </s200> <s400> 6
taggagagcc cgggtcagat gttcg                                           25
</s400> <s200> <s210>7</s210> <s211>25</s211> <s212>DNA</s212> <s213>Artificial Sequence</s213> <s220> <s223>Designed oligonucleotide primer used for
      constructing a</s223> </s220> </s200> <s400> 7
atggcccgga ctgtgatagt gcttg                                           25
</s400> <s200> <s210>8</s210> <s211>25</s211> <s212>DNA</s212> <s213>Artificial Sequence</s213> <s220> <s223>Designed oligonucleotide primer used for
      constructing a</s223> </s220> </s200> <s400> 8
ttcatgagta ggagagtggg ggtca                                           25
</s400> <s200> <s210>9</s210> <s211>563</s211> <s212>PRT</s212> <s213>Chlamydomonas reinhardtii CC-407</s213> </s200> <s400> 9
Met Met Leu Thr Gln Thr Pro Gly Thr Ala Thr Ala Ser Ser Arg Arg
  1               5                  10                  15
Ser Gln Ile Arg Ser Ala Ala His Val Ser Ala Lys Val Ala Pro Arg
             20                  25                  30
Pro Thr Pro Phe Ser Val Ala Ser Pro Ala Thr Ala Ala Ser Pro Ala
         35                  40                  45
Thr Ala Ala Ala Arg Arg Thr Leu His Arg Thr Ala Ala Ala Ala Thr
     50                  55                  60
Gly Ala Pro Thr Ala Ser Gly Ala Gly Val Ala Lys Thr Leu Asp Asn
 65                  70                  75                  80
Val Tyr Asp Val Ile Val Val Gly Gly Gly Leu Ser Gly Leu Val Thr
                 85                  90                  95
Gly Gln Ala Leu Ala Ala Gln His Lys Ile Gln Asn Phe Leu Val Thr
            100                 105                 110
Glu Ala Arg Glu Arg Val Gly Gly Asn Ile Thr Ser Met Ser Gly Asp
        115                 120                 125
Gly Tyr Val Trp Glu Glu Gly Pro Asn Ser Phe Gln Pro Asn Asp Ser
    130                 135                 140
Met Leu Gln Ile Ala Val Asp Ser Gly Cys Glu Lys Asp Leu Val Phe
145                 150                 155                 160
Gly Asp Pro Thr Ala Pro Arg Phe Val Trp Trp Glu Gly Lys Leu Arg
                165                 170                 175
Pro Val Pro Ser Gly Leu Asp Ala Phe Thr Phe Asp Leu Met Ser Ile
            180                 185                 190
Pro Gly Lys Ile Arg Ala Gly Leu Gly Ala Ile Gly Leu Ile Asn Gly
        195                 200                 205
Ala Met Pro Ser Phe Glu Glu Ser Val Glu Gln Phe Ile Arg Arg Asn
    210                 215                 220
Leu Gly Asp Glu Val Phe Phe Arg Leu Ile Glu Pro Phe Cys Ser Gly
225                 230                 235                 240
Val Tyr Ala Gly Asp Pro Ser Lys Leu Ser Met Lys Ala Ala Phe Asn
                245                 250                 255
Arg Ile Trp Ile Leu Glu Lys Asn Gly Gly Ser Leu Val Gly Gly Ala
            260                 265                 270
Ile Lys Leu Phe Gln Glu Arg Gln Ser Asn Pro Ala Pro Pro Arg Asp
        275                 280                 285
Pro Arg Leu Pro Pro Lys Pro Lys Gly Gln Thr Val Gly Ser Phe Arg
    290                 295                 300
Lys Gly Leu Lys Met Leu Pro Asp Ala Ile Glu Arg Asn Ile Pro Asp
305                 310                 315                 320
Lys Ile Arg Val Asn Trp Lys Leu Val Ser Leu Gly Arg Glu Ala Asp
                325                 330                 335
Gly Arg Tyr Gly Leu Val Tyr Asp Thr Pro Glu Gly Arg Val Lys Val
            340                 345                 350
Phe Ala Arg Ala Val Ala Leu Thr Ala Pro Ser Tyr Val Val Ala Asp
        355                 360                 365
Leu Val Lys Glu Gln Ala Pro Ala Ala Ala Glu Ala Leu Gly Ser Phe
    370                 375                 380
Asp Tyr Pro Pro Val Gly Ala Val Thr Leu Ser Tyr Pro Leu Ser Ala
385                 390                 395                 400
Val Arg Glu Glu Arg Lys Ala Ser Asp Gly Ser Val Pro Gly Phe Gly
                405                 410                 415
Gln Leu His Pro Arg Thr Gln Gly Ile Thr Thr Leu Gly Thr Ile Tyr
            420                 425                 430
Ser Ser Ser Leu Phe Pro Gly Arg Ala Pro Glu Gly His Met Leu Leu
        435                 440                 445
Leu Asn Tyr Ile Gly Gly Thr Thr Asn Arg Gly Ile Val Asn Gln Thr
    450                 455                 460
Thr Glu Gln Leu Val Glu Gln Val Asp Lys Asp Leu Arg Asn Met Val
465                 470                 475                 480
Ile Lys Pro Asp Ala Pro Lys Pro Arg Val Val Gly Val Arg Val Trp
                485                 490                 495
Pro Arg Ala Ile Pro Gln Phe Asn Leu Gly His Leu Glu Gln Leu Asp
            500                 505                 510
Lys Ala Arg Lys Ala Leu Asp Ala Ala Gly Leu Gln Gly Val His Leu
        515                 520                 525
Gly Gly Asn Tyr Val Ser Gly Val Ala Leu Gly Lys Val Val Glu His
    530                 535                 540
Gly Tyr Glu Ser Ala Ala Asn Leu Ala Lys Ser Val Ser Lys Ala Ala
545                 550                 555                 560
Val Lys Ala
        563
</s400> <s200> <s210>10</s210> <s211>1838</s211> <s212>DNA</s212> <s213>Chlamydomonas reinhardtii CC-407</s213> <s220> <s222>(2)...(1793)</s222> </s220> </s200> <s400> 10
a atg atg ttg acc cag act cct ggg acc gcc acg gct tct agc cgg         46
  Met Met Leu Thr Gln Thr Pro Gly Thr Ala Thr Ala Ser Ser Arg
    1               5                  10                  15
cgg tcg cag atc cgc tcg gct gcg cac gtc tcc gcc aag gtc gcg cct       94
Arg Ser Gln Ile Arg Ser Ala Ala His Val Ser Ala Lys Val Ala Pro
                 20                  25                  30
cgg ccc acg cca ttc tcg gtc gcg agc ccc gcg acc gct gcg agc ccc      142
Arg Pro Thr Pro Phe Ser Val Ala Ser Pro Ala Thr Ala Ala Ser Pro
             35                  40                  45
gcg acc gcg gcg gcc cgc cgc aca ctc cac cgc act gct gcg gcg gcc      190
Ala Thr Ala Ala Ala Arg Arg Thr Leu His Arg Thr Ala Ala Ala Ala
         50                  55                  60
act ggt gct ccc acg gcg tcc gga gcc ggc gtc gcc aag acg ctc gac      238
Thr Gly Ala Pro Thr Ala Ser Gly Ala Gly Val Ala Lys Thr Leu Asp
     65                  70                  75
aat gtg tat gac gtg atc gtg gtc ggt gga ggt ctc tcg ggc ctg gtg      286
Asn Val Tyr Asp Val Ile Val Val Gly Gly Gly Leu Ser Gly Leu Val
 80                  85                  90                  95
acc ggc cag gcc ctg gcg gct cag cac aaa att cag aac ttc ctt gtt      334
Thr Gly Gln Ala Leu Ala Ala Gln His Lys Ile Gln Asn Phe Leu Val
                100                 105                 110
acg gag gct cgc gag cgc gtc ggc ggc aac att acg tcc atg tcg ggc      382
Thr Glu Ala Arg Glu Arg Val Gly Gly Asn Ile Thr Ser Met Ser Gly
            115                 120                 125
gat ggc tac gtg tgg gag gag ggc ccg aac agc ttc cag ccc aac gat      430
Asp Gly Tyr Val Trp Glu Glu Gly Pro Asn Ser Phe Gln Pro Asn Asp
        130                 135                 140
agc atg ctg cag att gcg gtg gac tct ggc tgc gag aag gac ctt gtg      478
Ser Met Leu Gln Ile Ala Val Asp Ser Gly Cys Glu Lys Asp Leu Val
    145                 150                 155
ttc ggt gac ccc acg gct ccc cgc ttc gtg tgg tgg gag ggc aag ctg      526
Phe Gly Asp Pro Thr Ala Pro Arg Phe Val Trp Trp Glu Gly Lys Leu
160                 165                 170                 175
cgc ccc gtg ccc tcg ggc ctg gac gcc ttc acc ttc gac ctc atg tcc      574
Arg Pro Val Pro Ser Gly Leu Asp Ala Phe Thr Phe Asp Leu Met Ser
                180                 185                 190
atc ccc ggc aag atc cgc gcc ggg ctg ggc gcc atc ggc ctc atc aac      622
Ile Pro Gly Lys Ile Arg Ala Gly Leu Gly Ala Ile Gly Leu Ile Asn
            195                 200                 205
gga gcc atg ccc tcc ttc gag gag agt gtg gag cag ttc atc cgc cgc      670
Gly Ala Met Pro Ser Phe Glu Glu Ser Val Glu Gln Phe Ile Arg Arg
        210                 215                 220
aac ctg ggc gat gag gtg ttc ttc cgc ctg atc gag ccc ttc tgc tcc      718
Asn Leu Gly Asp Glu Val Phe Phe Arg Leu Ile Glu Pro Phe Cys Ser
    225                 230                 235
ggc gtg tac gcg ggc gac ccc tcc aag ctg tcc atg aag gcg gcc ttc      766
Gly Val Tyr Ala Gly Asp Pro Ser Lys Leu Ser Met Lys Ala Ala Phe
240                 245                 250                 255
aac agg atc tgg att ctg gag aag aac ggc ggc agc ctg gtg gga ggt      814
Asn Arg Ile Trp Ile Leu Glu Lys Asn Gly Gly Ser Leu Val Gly Gly
                260                 265                 270
gcc atc aag ctg ttc cag gaa cgc cag tcc aac ccg gcc ccg ccg cgg      862
Ala Ile Lys Leu Phe Gln Glu Arg Gln Ser Asn Pro Ala Pro Pro Arg
            275                 280                 285
gac ccg cgc ctg ccg ccc aag ccc aag ggc cag acg gtg ggc tcg ttc      910
Asp Pro Arg Leu Pro Pro Lys Pro Lys Gly Gln Thr Val Gly Ser Phe
        290                 295                 300
cgc aag ggc ctg aag atg ctg ccg gac gcc att gag cgc aac atc ccc      958
Arg Lys Gly Leu Lys Met Leu Pro Asp Ala Ile Glu Arg Asn Ile Pro
    305                 310                 315
gac aag atc cgc gtg aac tgg aag ctg gtg tct ctg ggc cgc gag gcg     1006
Asp Lys Ile Arg Val Asn Trp Lys Leu Val Ser Leu Gly Arg Glu Ala
320                 325                 330                 335
gac ggg cgg tac ggg ctg gtg tac gac acg ccc gag ggc cgt gtc aag     1054
Asp Gly Arg Tyr Gly Leu Val Tyr Asp Thr Pro Glu Gly Arg Val Lys
                340                 345                 350
gtg ttt gcc cgc gcc gtg gct ctg acc gcg ccc agc tac gtg gtg gcg     1102
Val Phe Ala Arg Ala Val Ala Leu Thr Ala Pro Ser Tyr Val Val Ala
            355                 360                 365
gac ctg gtc aag gag cag gcg ccc gcc gcc gcc gag gcc ctg ggc tcc     1150
Asp Leu Val Lys Glu Gln Ala Pro Ala Ala Ala Glu Ala Leu Gly Ser
        370                 375                 380
ttc gac tac ccg ccg gtg ggc gcc gtg acg ctg tcg tac ccg ctg agc     1198
Phe Asp Tyr Pro Pro Val Gly Ala Val Thr Leu Ser Tyr Pro Leu Ser
    385                 390                 395
gcc gtg cgg gag gag cgc aag gcc tcg gac ggg tcc gtg ccg ggc ttc     1246
Ala Val Arg Glu Glu Arg Lys Ala Ser Asp Gly Ser Val Pro Gly Phe
400                 405                 410                 415
ggt cag ctg cac ccg cgc acg cag ggc atc acc act ctg ggc acc atc     1294
Gly Gln Leu His Pro Arg Thr Gln Gly Ile Thr Thr Leu Gly Thr Ile
                420                 425                 430
tac agc tcc agc ctg ttc ccc ggc cgc gcg ccc gag ggc cac atg ctg     1342
Tyr Ser Ser Ser Leu Phe Pro Gly Arg Ala Pro Glu Gly His Met Leu
            435                 440                 445
ctg ctc aac tac atc ggc ggc acc acc aac cgc ggc atc gtc aac cag     1390
Leu Leu Asn Tyr Ile Gly Gly Thr Thr Asn Arg Gly Ile Val Asn Gln
        450                 455                 460
acc acc gag cag ctg gtg gag cag gtg gac aag gac ctg cgc aac atg     1438
Thr Thr Glu Gln Leu Val Glu Gln Val Asp Lys Asp Leu Arg Asn Met
    465                 470                 475
gtc atc aag ccc gac gcg ccc aag ccc cgt gtg gtg ggc gtg cgc gtg     1486
Val Ile Lys Pro Asp Ala Pro Lys Pro Arg Val Val Gly Val Arg Val
480                 485                 490                 495
tgg ccg cgc gcc atc ccg cag ttc aac ctg ggc cac ctg gag cag ctg     1534
Trp Pro Arg Ala Ile Pro Gln Phe Asn Leu Gly His Leu Glu Gln Leu
                500                 505                 510
gac aag gcg cgc aag gcg ctg gac gcg gcg ggg ctg cag ggc gtg cac     1582
Asp Lys Ala Arg Lys Ala Leu Asp Ala Ala Gly Leu Gln Gly Val His
            515                 520                 525
ctg ggg ggc aac tac gtc agc ggt gtg gcc ctg ggc aag gtg gtg gag     1630
Leu Gly Gly Asn Tyr Val Ser Gly Val Ala Leu Gly Lys Val Val Glu
        530                 535                 540
cac ggc tac gag tcc gca gcc aac ctg gcc aag agc gtg tcc aag gcc     1678
His Gly Tyr Glu Ser Ala Ala Asn Leu Ala Lys Ser Val Ser Lys Ala
    545                 550                 555
gca gtc aag gcc taa gcggctgcag cagtagcagc agcagcatcg ggctgtagct     1733
Ala Val Lys Ala
560         563
ggtaaatgcc gcagtggcac cggcagcagc aattggcaag cacttggggc aagcggagtg   1793
gaggcgaggg gggggctacc attggcgctt gctgggatgt gtagt                   1838</s400> <s200> <s210>11</s210> <s211>28</s211> <s212>DNA</s212> <s213>Artificial Sequence</s213> <s220> <s223>Designed oligonucleotide primer used for
      amplifying a DNA</s223> </s220> </s200> <s400> 11
aatgatgttg acccagactc ctgggacc                                        28
</s400> <s200> <s210>12</s210> <s211>27</s211> <s212>DNA</s212> <s213>Artificial Sequence</s213> <s220> <s223>Designed oligonucleotide primer used for
      constructing a</s223> </s220> </s200> <s400> 12
tactacacat cccagcaagc gccaatg                                         27
</s400> <s200> <s210>13</s210> <s211>32</s211> <s212>DNA</s212> <s213>Artificial Sequence</s213> <s220> <s223>Designed oligonucleotide primer used for
      constructing a</s223> </s220> </s200> <s400> 13
tcgagctcaa tgatgttgac ccagactcct gg                                   32
</s400> <s200> <s210>14</s210> <s211>32</s211> <s212>DNA</s212> <s213>Artificial Sequence</s213> <s220> <s223>Designed oligonucleotide primer used for
      constructing a</s223> </s220> </s200> <s400> 14
ttgtcgacta ctacacatcc cagcaagcgc ca                                   32
</s400> </sequence-list-new-rules> </sequence-cwu> </p>
    </div></div></div><div class="patent-section patent-tabular-section"><a id="backward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Cited Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5767373">US5767373</a></td><td class="patent-data-table-td patent-date-value">Jun 6, 1995</td><td class="patent-data-table-td patent-date-value">Jun 16, 1998</td><td class="patent-data-table-td ">Novartis Finance Corporation</td><td class="patent-data-table-td ">Manipulation of protoporphyrinogen oxidase enzyme activity in eukaryotic organisms</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5939602">US5939602</a></td><td class="patent-data-table-td patent-date-value">Feb 28, 1997</td><td class="patent-data-table-td patent-date-value">Aug 17, 1999</td><td class="patent-data-table-td ">Novartis Finance Corporation</td><td class="patent-data-table-td ">DNA molecules encoding plant protoporphyrinogen oxidase and inhibitor-resistant mutants thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6018105">US6018105</a></td><td class="patent-data-table-td patent-date-value">Feb 28, 1997</td><td class="patent-data-table-td patent-date-value">Jan 25, 2000</td><td class="patent-data-table-td ">Novartis Finance Corporation</td><td class="patent-data-table-td ">Nucleotide sequences; for expressing modified forms of herbicide target enzymes for herbicide resistant plants</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6023012">US6023012</a></td><td class="patent-data-table-td patent-date-value">Mar 30, 1998</td><td class="patent-data-table-td patent-date-value">Feb 8, 2000</td><td class="patent-data-table-td ">Novartis Finance Corporation</td><td class="patent-data-table-td ">DNA molecules encoding plant protoporphyrinogen oxidase</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1997004089A2?cl=en">WO1997004089A2</a></td><td class="patent-data-table-td patent-date-value">Jul 19, 1996</td><td class="patent-data-table-td patent-date-value">Feb 6, 1997</td><td class="patent-data-table-td ">John Boynton</td><td class="patent-data-table-td ">Porphyrin-accumulating type herbicide resistance gene</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1998029554A1?cl=en">WO1998029554A1</a></td><td class="patent-data-table-td patent-date-value">Dec 27, 1996</td><td class="patent-data-table-td patent-date-value">Jul 9, 1998</td><td class="patent-data-table-td ">John E Boynton</td><td class="patent-data-table-td ">Methods of conferring ppo-inhibiting herbicide resistance to plants by gene manipulation</td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="npl-citations"></a><div class="patent-section-header"><span class="patent-section-title">Non-Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th colspan="3"class="patent-data-table-th">Reference</th></tr></thead><tr><td class="patent-data-table-td ">1</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Fumiaki Yamao et al., Jpn. J. Genet., vol. 63, 1988, pp. 237-249.</td></tr><tr><td class="patent-data-table-td ">2</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Jean-Michel Camadro et al., Biochemical and Biophysical Research Communications, vol. 106, No. 3, 1982, pp. 724-730.</td></tr><tr><td class="patent-data-table-td ">3</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Kazumasa Miyamoto et al., J. Mol. Biol., vol. 219, 1991, pp. 393-398.</td></tr><tr><td class="patent-data-table-td ">4</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Koichi Nishimura et al., Gene, vol. 133, 1993, pp. 109-113.</td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="classifications"></a><div class="patent-section-header"><span class="patent-section-title">Classifications</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th"> </th><th class="patent-data-table-th"> </th></tr></thead><tr><td class="patent-data-table-td ">U.S. Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=aJVcBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S025000">435/25</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=aJVcBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S252300">435/252.3</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=aJVcBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S023200">536/23.2</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=aJVcBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S189000">435/189</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=aJVcBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S320100">435/320.1</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=aJVcBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S325000">435/325</a></span></td></tr><tr><td class="patent-data-table-td ">International Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=aJVcBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0009020000">C12N9/02</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=aJVcBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12Q0001680000">C12Q1/68</a></span></td></tr><tr><td class="patent-data-table-td ">Cooperative Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=aJVcBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N9/0004">C12N9/0004</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=aJVcBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12Q1/6897">C12Q1/6897</a></span></td></tr><tr><td class="patent-data-table-td ">European Classification</td><td class="patent-data-table-td "><span class="nested-value">C12N9/00P</span>, <span class="nested-value">C12Q1/68P</span></td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="legal-events"></a><div class="patent-section-header"><span class="patent-section-title">Legal Events</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Date</th><th class="patent-data-table-th">Code</th><th class="patent-data-table-th">Event</th><th class="patent-data-table-th">Description</th></tr></thead><tr><td class="patent-data-table-td patent-date-value">Jun 6, 2014</td><td class="patent-data-table-td ">REMI</td><td class="patent-data-table-td ">Maintenance fee reminder mailed</td><td class="patent-data-table-td "></td></tr><tr><td class="patent-data-table-td patent-date-value">Apr 21, 2010</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">8</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Dec 30, 2008</td><td class="patent-data-table-td ">B1</td><td class="patent-data-table-td ">Reexamination certificate first reexamination</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">THE PATENTABILITY OF CLAIMS 3-4 IS CONFIRMED. CLAIMS 1-2 ARE DETERMINED TO BE PATENTABLE AS AMENDED. CLAIMS 5-6, DEPENDENT ON AN AMENDED CLAIM, ARE DETERMINED TO BE PATENTABLE. NEW CLAIMS 7-8 ARE ADDED AND DETERMINED TO BE PATENTABLE.</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Jan 23, 2007</td><td class="patent-data-table-td ">RR</td><td class="patent-data-table-td ">Request for reexamination filed</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20060823</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Apr 7, 2006</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">4</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">May 11, 1999</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">SUMITOMO CHEMICAL COMPANY, LTD., JAPAN</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNORS:SHIMOKAWATOKO, YASUTAKA;NISHIO, SHOICHI;REEL/FRAME:009945/0391</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">19990419</span></div></td></tr></table><div class="patent-section-footer"></div></div><div class="modal-dialog" id="patent-images-lightbox"><div class="patent-lightbox-controls"><div class="patent-lightbox-rotate-controls"><div class="patent-lightbox-rotation-text">Rotate</div><div class="rotate-icon rotate-ccw-icon"></div><div class="rotate-icon rotate-cw-icon"></div></div><div class="patent-lightbox-index-counter"></div><a class="patent-lightbox-fullsize-link" target="_blank">Original Image</a><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_right.png" alt="Next page"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_left.png" alt="Previous page"width="21" height="21" /></div></div><div class="modal-dialog-content"><div class="patent-lightbox-image-holder"><div class="patent-lightbox-placeholder"></div></div></div></div><script>_OC_initPatentsAtb({image_not_available_html: " Image not available"});</script></div></div></div></td></tr></table><script>(function() {var href = window.location.href;if (href.indexOf('?') !== -1) {var parameters = href.split('?')[1].split('&');for (var i = 0; i < parameters.length; i++) {var param = parameters[i].split('=');if (param[0] == 'focus') {var elem = document.getElementById(param[1]);if (elem) {elem.focus();}}}}})();</script><script>_OC_addFlags({LockSrc:"/books/javascript/lock_8a2b04e7bf975d5171d8e4c0b6365c7a.js", Host:"http://www.google.com/", IsBooksRentalEnabled:1, IsWebstoreDisplayCaseEnabled:1, IsObfuscationEnabled:1, IsBrowsingHistoryEnabled:1, IsWebReaderSvgEnabled:0, IsGeoLayerEnabled:1, IsImageModeNotesEnabled:1, IsCopyMenuItemEnabled:1, IsGiftingEnabled:0, IsWebReaderUniversalPaginatorEnabled:0, IsOfflineBubbleEnabled:1, IsReaderEnabledForPlayRequests:1, IsFutureOnSaleVolumesEnabled:1, IsOfflineRestrictedCopyEnabled:1, IsBooksUnifiedLeftNavEnabled:1, IsRestrictedCopyEnabled:1, IsZipitFolderCollectionEnabled:1, IsEndOfSampleRecommendationsEnabled:1, IsRatingsOnBookcardsEnabled:1, IsAdsDisabled:0, IsIframePageDisplayEnabled:0, IsEmbeddedMediaEnabled:1, IsImageModeAnnotationsEnabled:1, IsMyLibraryGooglePlusEnabled:1, IsImagePageProviderEnabled:0, IsBookcardListPriceSmall:0, IsInternalUser:0, IsBooksShareButtonEnabled:0, IsPreOrdersEnabled:0, IsDisabledRandomBookshelves:0, WebstoreDisplayCasePosition:3});_OC_Run({"enable_p13n":false,"add_vol_to_collection_base_url":"http://www.google.com/patents?op=add\u0026sig=ACfU3U0I4UJpNIyGs_fRyaPrJ_ncswpsNA\u0026id=aJVcBAABERAJ","remove_vol_from_collection_base_url":"http://www.google.com/patents?op=remove\u0026sig=ACfU3U17x5zjZDoES48WL3YAHjtCfPYAYw\u0026id=aJVcBAABERAJ","logged_in":false,"p13n_save_user_settings_url":"http://www.google.com/patents?op=edit_user_settings\u0026sig=ACfU3U3o6sqC_hoh_8t90YQ1RXjlTu9OrA","is_cobrand":false,"sign_in_url":"https://www.google.com/accounts/Login?service=\u0026continue=http://www.google.com/patents%3Fhl%3Den\u0026hl=en","is_play_enabled":true}, {"volume_id":"","is_ebook":true,"volumeresult":{"has_flowing_text":false,"has_scanned_text":true,"can_download_pdf":false,"can_download_epub":false,"is_pdf_drm_enabled":false,"is_epub_drm_enabled":false,"download_pdf_url":"http://www.google.com/patents/download/Method_for_evaluating_the_ability_of_a_c.pdf?id=aJVcBAABERAJ\u0026output=pdf\u0026sig=ACfU3U1sCqesVlTbyC0BnTzVqno6XDPFSg"},"sample_url":"http://www.google.com/patents/reader?id=aJVcBAABERAJ\u0026printsec=frontcover\u0026output=reader\u0026source=gbs_atb_hover","is_browsable":true,"is_public_domain":true}, {});</script><div id="footer_table" style="font-size:83%;text-align:center;position:relative;top:20px;height:4.5em;margin-top:2em"><div style="margin-bottom:8px"><a href=http://www.google.com/><nobr>Google&nbsp;Home</nobr></a> - <a href=//www.google.com/patents/sitemap/><nobr>Sitemap</nobr></a> - <a href=http://www.google.com/googlebooks/uspto.html><nobr>USPTO Bulk Downloads</nobr></a> - <a href=/intl/en/privacy/><nobr>Privacy Policy</nobr></a> - <a href=/intl/en/policies/terms/><nobr>Terms of Service</nobr></a> - <a href=https://support.google.com/faqs/answer/2539193?hl=en><nobr>About Google Patents</nobr></a> - <a href="http://www.google.com/tools/feedback/intl/en/error.html" onclick="try{_OC_startFeedback({productId: '72792',locale: 'en'});return false;}catch(e){}"><nobr>Send Feedback</nobr></a></div><span>Data provided by IFI CLAIMS Patent Services</span><br><span >&copy;2012 Google</span></div> <script type="text/javascript">var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));</script><script type="text/javascript">var pageTracker = _gat._getTracker("UA-27188110-1");pageTracker._setCookiePath("/patents/");pageTracker._trackPageview();</script> </body></html>