<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN""http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html><head><title>Patent US6599502 - Intracellular delivery vehicles - Google Patents</title><script>(function(){(function(){function e(a){this.t={};this.tick=function(a,c,b){var d=void 0!=b?b:(new Date).getTime();this.t[a]=[d,c];if(void 0==b)try{window.console.timeStamp("CSI/"+a)}catch(e){}};this.tick("start",null,a)}var a;window.performance&&(a=window.performance.timing);var f=a?new e(a.responseStart):new e;window.jstiming={Timer:e,load:f};if(a){var c=a.navigationStart,d=a.responseStart;0<c&&d>=c&&(window.jstiming.srt=d-c)}if(a){var b=window.jstiming.load;0<c&&d>=c&&(b.tick("_wtsrt",void 0,c),b.tick("wtsrt_",
"_wtsrt",d),b.tick("tbsd_","wtsrt_"))}try{a=null,window.chrome&&window.chrome.csi&&(a=Math.floor(window.chrome.csi().pageT),b&&0<c&&(b.tick("_tbnd",void 0,window.chrome.csi().startE),b.tick("tbnd_","_tbnd",c))),null==a&&window.gtbExternal&&(a=window.gtbExternal.pageT()),null==a&&window.external&&(a=window.external.pageT,b&&0<c&&(b.tick("_tbnd",void 0,window.external.startE),b.tick("tbnd_","_tbnd",c))),a&&(window.jstiming.pt=a)}catch(g){}})();})();
</script><link rel="stylesheet" href="/patents/css/_8a2b04e7bf975d5171d8e4c0b6365c7a/kl_intl_patents_bundle.css" type="text/css" /><script src="/books/javascript/atb_8a2b04e7bf975d5171d8e4c0b6365c7a__en.js"></script><script>function googleTranslateElementInit() {new google.translate.TranslateElement({pageLanguage: "en",gaTrack: true,gaId: "UA-27188110-1",multilanguagePage: true});}</script><script src="//translate.google.com/translate_a/element.js?cb=googleTranslateElementInit"></script><meta name="DC.type" content="Patent"><meta name="DC.title" content="Intracellular delivery vehicles"><meta name="DC.contributor" content="Daniel A. Portnoy" scheme="inventor"><meta name="DC.contributor" content="Darren E. Higgins" scheme="inventor"><meta name="DC.contributor" content="The Regents Of The University Of California" scheme="assignee"><meta name="DC.date" content="2001-9-7" scheme="dateSubmitted"><meta name="DC.description" content="The invention provides methods and compositions relating to intracellular delivering of agents to eukaryotic cells. The compositions include microbial delivery vehicles such as nonvirulent bacteria comprising a first gene encoding a nonsecreted foreign cytolysin operably linked to a heterologous promoter and a second gene encoding a different foreign agent. The foreign agent may be a nucleic acid or protein, and is frequently bioactive in and therapeutic to the target eukaryote. In addition, the invention provides eukaryotic cells comprising the subject nonvirulent bacteria and nonhuman eukaryotic host organisms comprising such cells."><meta name="DC.date" content="2003-7-29" scheme="issued"><meta name="DC.relation" content="US:4888170" scheme="references"><meta name="DC.relation" content="US:5294441" scheme="references"><meta name="DC.relation" content="US:5387744" scheme="references"><meta name="DC.relation" content="US:5424065" scheme="references"><meta name="DC.relation" content="US:5824538" scheme="references"><meta name="DC.relation" content="US:5855879" scheme="references"><meta name="DC.relation" content="US:5855880" scheme="references"><meta name="DC.relation" content="US:5877159" scheme="references"><meta name="citation_reference" content="Anderson, Nature 392:25-30 Feb. 1998."><meta name="citation_reference" content="Branch, TIBS 23:45-50 Feb. 1998."><meta name="citation_reference" content="Darji, et al., Cell 91:765-75 1997."><meta name="citation_reference" content="Dietrich G. et al. &quot;Delivery of antigen-encoding plasmid DNA into the cytosol of macrophages by attenuated Listeria monocytogenes&quot; Nature Biotechnology vol. 16, Feb. 1998, pp. 181-185."><meta name="citation_reference" content="GenBank Accession U02451.*"><meta name="citation_reference" content="Higgins et al, Molecular Microbiology 31(6):1631-41 1999."><meta name="citation_reference" content="Higgins et al., Nature Biotechnology 16(2):138-9 1998."><meta name="citation_reference" content="Ikonomidis et al., Journal of Experimental Medicine 180(6):2209-18 1994."><meta name="citation_reference" content="Y Gholizadeh et al., Journal of Clinical Microbiology, &quot;Serodiagnosis of Listeriosis Based upon Detection of Antibodies against Recombinant Truncated Forms of Listeriolysin O,&quot; Jun. 1996, vol. 34, No. 6, pp. 1391-1395.*"><meta name="citation_patent_number" content="US:6599502"><meta name="citation_patent_application_number" content="US:09/949,109"><link rel="canonical" href="http://www.google.com/patents/US6599502"/><meta property="og:url" content="http://www.google.com/patents/US6599502"/><meta name="title" content="Patent US6599502 - Intracellular delivery vehicles"/><meta name="description" content="The invention provides methods and compositions relating to intracellular delivering of agents to eukaryotic cells. The compositions include microbial delivery vehicles such as nonvirulent bacteria comprising a first gene encoding a nonsecreted foreign cytolysin operably linked to a heterologous promoter and a second gene encoding a different foreign agent. The foreign agent may be a nucleic acid or protein, and is frequently bioactive in and therapeutic to the target eukaryote. In addition, the invention provides eukaryotic cells comprising the subject nonvirulent bacteria and nonhuman eukaryotic host organisms comprising such cells."/><meta property="og:title" content="Patent US6599502 - Intracellular delivery vehicles"/><meta property="og:type" content="book"/><meta property="og:site_name" content="Google Books"/><meta property="og:image" content="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><link rel="image_src" href="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><script>(function(){try{var aa=function(a,b,c,d){d=d||{};d._sn=["cfg",b,c].join(".");window.gbar.logger.ml(a,d)};var g=window.gbar=window.gbar||{},l=window.gbar.i=window.gbar.i||{},m={},n;function _tvn(a,b){var c=parseInt(a,10);return isNaN(c)?b:c}function _tvf(a,b){var c=parseFloat(a);return isNaN(c)?b:c}function _tvv(a){return!!a}function p(a,b,c){(c||g)[a]=b}g.bv={n:_tvn("2",0),r:"",f:".67.",e:"0",m:_tvn("0",1)};
function q(a,b,c){var d="on"+b;if(a.addEventListener)a.addEventListener(b,c,!1);else if(a.attachEvent)a.attachEvent(d,c);else{var f=a[d];a[d]=function(){var a=f.apply(this,arguments),b=c.apply(this,arguments);return void 0==a?b:void 0==b?a:b&&a}}}var s=function(a){return function(){return g.bv.m==a}},ba=s(1),ca=s(2);p("sb",ba);p("kn",ca);l.a=_tvv;l.b=_tvf;l.c=_tvn;l.i=aa;var da=window.gbar.i.i;var t,u,v,w;function ea(a){v=a}function fa(a){var b;if(b=v&&window.encodeURIComponent)b=a.href,b=!b.match(/^http[s]?:\/\/accounts\.google\.[^/]*\/ClearSID/i)&&!b.match(/^http[s]?:\/\/[^/]*\/accounts\/ClearSID/i);if(b=b&&encodeURIComponent(v()))a.href=a.href.replace(/([?&]continue=)[^&]*/,"$1"+b)}function ga(a){window.gApplication&&(a.href=window.gApplication.getTabUrl(a.href))}
function ha(a){var b=document.forms[0].q,c=window.encodeURIComponent&&b&&b.value,b=b&&b.placeholder;c&&c!=b&&(a.href=a.href.replace(/([?&])q=[^&]*|$/,function(a,b){return(b||"&")+"q="+encodeURIComponent(c)}))}n=l.a("")?ga:ha;
function x(a,b,c,d,f,e){var h=document.getElementById(a);if(h){var k=h.style;k.left=d?"auto":b+"px";k.right=d?b+"px":"auto";k.top=c+"px";k.visibility=u?"hidden":"visible";f&&e?(k.width=f+"px",k.height=e+"px"):(x(t,b,c,d,h.offsetWidth,h.offsetHeight),u=u?"":a)}}
var y=[],ia=function(a,b){y.push(b)},ja=function(a){a=a||window.event;var b=a.target||a.srcElement;a.cancelBubble=!0;null==t&&(a=document.createElement(Array.every||window.createPopup?"iframe":"div"),a.frameBorder="0",t=a.id="gbs",a.src="javascript:''",b.parentNode.appendChild(a),q(document,"click",z));var c=b,b=0;"gb3"!=c.className&&(c=c.parentNode);a=c.getAttribute("aria-owns")||"gbi";var d=c.offsetWidth,f=20<c.offsetTop?46:24;document.getElementById("tphdr")&&(f-=3);var e=!1;do b+=c.offsetLeft||
0;while(c=c.offsetParent);var c=(document.documentElement.clientWidth||document.body.clientWidth)-b-d,h,d=document.body,k=document.defaultView;k&&k.getComputedStyle?(d=k.getComputedStyle(d,""))&&(h=d.direction):h=d.currentStyle?d.currentStyle.direction:d.style.direction;h="rtl"==h;if("gbi"==a){for(d=0;k=y[d++];)k();A(null,window.navExtra);h&&(b=c,e=!0)}else h||(b=c,e=!0);u!=a&&z();x(a,b,f,e)},z=function(){u&&x(u,0,0)},A=function(a,b){var c,d=document.getElementById("gbi"),f=a;f||(f=d.firstChild);
for(;b&&(c=b.pop());){var e=d,h=c,k=f;w||(w="gb2");e.insertBefore(h,k).className=w}},ka=function(a,b,c){if((b=document.getElementById(b))&&a){a.className="gb4";var d=document.createElement("span");d.appendChild(a);d.appendChild(document.createTextNode(" | "));d.id=c;b.appendChild(d)}},la=function(){return document.getElementById("gb_70")},ma=function(){return!!u};p("qs",n);p("setContinueCb",ea);p("pc",fa);p("tg",ja);p("close",z);p("addLink",ka);p("almm",A);p("si",la);p("adh",ia);p("op",ma);var B=function(){},C=function(){},F=function(a){var b=new Image,c=D;b.onerror=b.onload=b.onabort=function(){try{delete E[c]}catch(a){}};E[c]=b;b.src=a;D=c+1},E=[],D=0;p("logger",{il:C,ml:B,log:F});var G=window.gbar.logger;var H={},na={},I=[],oa=l.b("0.1",.1),pa=l.a("1",!0),qa=function(a,b){I.push([a,b])},ra=function(a,b){H[a]=b},sa=function(a){return a in H},J={},K=function(a,b){J[a]||(J[a]=[]);J[a].push(b)},ta=function(a){K("m",a)},L=function(a,b){var c=document.createElement("script");c.src=a;c.async=pa;Math.random()<oa&&(c.onerror=function(){c.onerror=null;B(Error("Bundle load failed: name="+(b||"UNK")+" url="+a))});(document.getElementById("xjsc")||document.getElementsByTagName("body")[0]||
document.getElementsByTagName("head")[0]).appendChild(c)},N=function(a){for(var b=0,c;(c=I[b])&&c[0]!=a;++b);!c||c[1].l||c[1].s||(c[1].s=!0,M(2,a),c[1].url&&L(c[1].url,a),c[1].libs&&m.d&&m.d(c[1].libs))},O=function(a){K("gc",a)},P=null,ua=function(a){P=a},M=function(a,b,c){if(P){a={t:a,b:b};if(c)for(var d in c)a[d]=c[d];try{P(a)}catch(f){}}};p("mdc",H);p("mdi",na);p("bnc",I);p("qGC",O);p("qm",ta);p("qd",J);p("lb",N);p("mcf",ra);p("bcf",qa);p("aq",K);p("mdd","");p("has",sa);
p("trh",ua);p("tev",M);var Q=l.b("0.1",.001),R=0;
function _mlToken(a,b){try{if(1>R){R++;var c,d=a,f=b||{},e=encodeURIComponent,h=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&jexpid=",e("17483"),"&srcpg=",e("prop=22"),"&jsr=",Math.round(1/Q),"&ogev=",e("oKjtU8GEJNPQsQSo84GwDg"),"&ogf=",g.bv.f,"&ogrp=",e("1"),"&ogv=",e("1407723702.0"),"&oggv="+e("es_plusone_gc_20140723.0_p0"),"&ogd=",e("com"),"&ogc=",e("NOR"),"&ogl=",e("en")];f._sn&&(f._sn="og."+
f._sn);for(var k in f)h.push("&"),h.push(e(k)),h.push("="),h.push(e(f[k]));h.push("&emsg=");h.push(e(d.name+":"+d.message));var r=h.join("");S(r)&&(r=r.substr(0,2E3));c=r;var Aa=window.gbar.logger._aem(a,c);F(Aa)}}catch(Na){}}var S=function(a){return 2E3<=a.length},va=function(a,b){return b};function T(a){B=a;p("_itl",S,G);p("_aem",va,G);p("ml",B,G);a={};H.er=a}l.a("")?T(function(a){throw a;}):l.a("1")&&Math.random()<Q&&T(_mlToken);I.push(["m",{url:"//ssl.gstatic.com/gb/js/scm_7385cc5883250b43a39405734c1bea59.js"}]);g.mcf("c",{});g.sg={c:""};if(l.a("1")){var wa=l.a("");I.push(["gc",{auto:wa,url:"//ssl.gstatic.com/gb/js/abc/gci_91f30755d6a6b787dcc2a4062e6e9824.js",libs:"googleapis.client:plusone:gapi.iframes"}]);var xa={version:"gci_91f30755d6a6b787dcc2a4062e6e9824.js",index:"",lang:"en"};H.gc=xa;var U=function(a){window.googleapis&&window.iframes?a&&a():(a&&O(a),N("gc"))};p("lGC",U);l.a("1")&&p("lPWF",U)};window.__PVT="";if(l.a("1")&&l.a("1")){var V=function(a){U(function(){K("pw",a);N("pw")})};p("lPW",V);I.push(["pw",{url:"//ssl.gstatic.com/gb/js/abc/pwm_45f73e4df07a0e388b0fa1f3d30e7280.js"}]);var W=[],ya=function(a){W[0]=a},za=function(a,b){var c=b||{};c._sn="pw";B(a,c)},Ba={signed:W,elog:za,base:"https://plusone.google.com/u/0",loadTime:(new Date).getTime()};H.pw=Ba;var X=function(a,b){for(var c=b.split("."),d=function(){var b=arguments;a(function(){for(var a=g,d=0,e=c.length-1;d<e;++d)a=a[c[d]];a[c[d]].apply(a,b)})},f=g,e=0,h=c.length-1;e<h;++e)f=
f[c[e]]=f[c[e]]||{};return f[c[e]]=d};X(V,"pw.clk");X(V,"pw.hvr");p("su",ya,g.pw)};function Ca(){function a(){for(var b;(b=e[h++])&&"m"!=b[0]&&!b[1].auto;);b&&(M(2,b[0]),b[1].url&&L(b[1].url,b[0]),b[1].libs&&m.d&&m.d(b[1].libs));h<e.length&&setTimeout(a,0)}function b(){0<f--?setTimeout(b,0):a()}var c=l.a("1"),d=l.a(""),f=3,e=I,h=0,k=window.gbarOnReady;if(k)try{k()}catch(r){da(r,"ml","or")}d?p("ldb",a):c?q(window,"load",b):b()}p("rdl",Ca);var Da={D:1,H:2,da:3,p:4,W:5,M:6,F:7,g:8,ha:9,U:10,L:11,T:12,S:13,N:14,Q:15,P:16,fa:17,w:18,O:19,ga:20,ea:21,u:22,G:23,ja:24,ka:25,ia:26,A:27,j:28,o:29,k:30,ca:31,Z:32,$:33,J:34,K:35,ba:36,aa:37,Y:38,B:39,R:40,v:41,X:42,V:43,h:48,C:49,I:500},Y=[1,2,3,4,5,6,9,10,11,13,14,28,29,30,34,35,37,38,39,40,41,42,43,48,49,500];var Z=l.b("0.001",1E-4),Ea=l.b("1",1),Fa=!1,Ga=!1;if(l.a("1")){var Ha=Math.random();Ha<=Z&&(Fa=!0);Ha<=Ea&&(Ga=!0)}var Ia=Da,$=null;function Ja(){var a=0,b=function(b,d){l.a(d)&&(a|=b)};b(1,"");b(2,"");b(4,"");b(8,"");return a}
function Ka(a,b){var c=Z,d=Fa,f;f=a;if(!$){$={};for(var e=0;e<Y.length;e++){var h=Y[e];$[h]=!0}}if(f=!!$[f])c=Ea,d=Ga;if(d){d=encodeURIComponent;g.rp?(f=g.rp(),f="-1"!=f?f:"1"):f="1";c=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&oge=",a,"&ogex=",d("17483"),"&ogev=",d("oKjtU8GEJNPQsQSo84GwDg"),"&ogf=",g.bv.f,"&ogp=",d("22"),"&ogrp=",d(f),"&ogsr=",Math.round(1/c),"&ogv=",d("1407723702.0"),"&oggv="+
d("es_plusone_gc_20140723.0_p0"),"&ogd=",d("com"),"&ogl=",d("en"),"&ogc=",d("NOR"),"&ogus=",Ja()];if(b){"ogw"in b&&(c.push("&ogw="+b.ogw),delete b.ogw);var k;f=b;e=[];for(k in f)0!=e.length&&e.push(","),e.push(La(k)),e.push("."),e.push(La(f[k]));k=e.join("");""!=k&&(c.push("&ogad="),c.push(d(k)))}F(c.join(""))}}function La(a){"number"==typeof a&&(a+="");return"string"==typeof a?a.replace(".","%2E").replace(",","%2C"):a}C=Ka;p("il",C,G);var Ma={};H.il=Ma;setTimeout(function(){C(Ia.g)},0);}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var b=window.gbar.i.i;var c=window.gbar;var f=function(d){try{var a=document.getElementById("gbom");a&&d.appendChild(a.cloneNode(!0))}catch(e){b(e,"omas","aomc")}};c.aomc=f;}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var a=window.gbar;a.mcf("pm",{p:""});}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{window.gbar.rdl();}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
if (window['_OC_timingAction']) {window['_OC_timingAction']('patents_refpage');}</script><style>#gbar,#guser{font-size:13px;padding-top:1px !important;}#gbar{float:left;height:22px}#guser{padding-bottom:7px !important;text-align:right}.gbh,.gbd{border-top:1px solid #c9d7f1;font-size:1px}.gbh{height:0;position:absolute;top:24px;width:100%}#gbs,.gbm{background:#fff;left:0;position:absolute;text-align:left;visibility:hidden;z-index:1000}.gbm{border:1px solid;border-color:#c9d7f1 #36c #36c #a2bae7;z-index:1001}.gb1{margin-right:.5em}.gb1,.gb3{zoom:1}.gb2{display:block;padding:.2em .5em}.gb2,.gb3{text-decoration:none !important;border-bottom:none}a.gb1,a.gb4{text-decoration:underline !important}a.gb1,a.gb2,a.gb3,a.gb4{color:#00c !important}.gbi .gb3,.gbi .gb2,.gbi .gb4{color:#dd8e27 !important}.gbf .gb3,.gbf .gb2,.gbf .gb4{color:#900 !important}a.gb2:hover{background:#36c;color:#fff !important}#gbar .gbz0l{color:#000 !important;cursor:default;font-weight:bold;text-decoration:none !important}
#gbar { padding:.3em .6em !important;}</style></head><body  topmargin="3" marginheight="3"><div id=gbar><nobr><a onclick=gbar.qs(this);gbar.logger.il(1,{t:1}); class=gb1 id=gb_1 href="https://www.google.com/search?sa=N&tab=tw">Search</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:2}); class=gb1 id=gb_2 href="http://www.google.com/search?hl=en&tbm=isch&source=og&sa=N&tab=ti">Images</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:8}); class=gb1 id=gb_8 href="http://maps.google.com/maps?hl=en&sa=N&tab=tl">Maps</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:78}); class=gb1 id=gb_78 href="https://play.google.com/?hl=en&sa=N&tab=t8">Play</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:36}); class=gb1 id=gb_36 href="http://www.youtube.com/results?sa=N&tab=t1">YouTube</a> <a onclick=gbar.logger.il(1,{t:5}); class=gb1 id=gb_5 href="http://news.google.com/nwshp?hl=en&tab=tn">News</a> <a onclick=gbar.logger.il(1,{t:23}); class=gb1 id=gb_23 href="https://mail.google.com/mail/?tab=tm">Gmail</a> <a onclick=gbar.logger.il(1,{t:25}); class=gb1 id=gb_25 href="https://drive.google.com/?tab=to">Drive</a> <a class=gb3 href="http://www.google.com/intl/en/options/" onclick="this.blur();gbar.tg(event);return !1" aria-haspopup=true><u>More</u> <small>&#9660;</small></a><div class=gbm id=gbi><a onclick=gbar.logger.il(1,{t:24}); class=gb2 id=gb_24 href="https://www.google.com/calendar?tab=tc">Calendar</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:51}); class=gb2 id=gb_51 href="http://translate.google.com/?hl=en&sa=N&tab=tT">Translate</a><a onclick=gbar.logger.il(1,{t:17}); class=gb2 id=gb_17 href="http://www.google.com/mobile/?hl=en&tab=tD">Mobile</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:10}); class=gb2 id=gb_10 href="http://www.google.com/search?hl=en&tbo=u&tbm=bks&source=og&sa=N&tab=tp">Books</a><a onclick=gbar.logger.il(1,{t:212}); class=gb2 id=gb_212 href="https://wallet.google.com/manage/?tab=ta">Wallet</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:6}); class=gb2 id=gb_6 href="http://www.google.com/search?hl=en&tbo=u&tbm=shop&source=og&sa=N&tab=tf">Shopping</a><a onclick=gbar.logger.il(1,{t:30}); class=gb2 id=gb_30 href="http://www.blogger.com/?tab=tj">Blogger</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:27}); class=gb2 id=gb_27 href="http://www.google.com/finance?sa=N&tab=te">Finance</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:31}); class=gb2 id=gb_31 href="https://plus.google.com/photos?sa=N&tab=tq">Photos</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:12}); class=gb2 id=gb_12 href="http://www.google.com/search?hl=en&tbo=u&tbm=vid&source=og&sa=N&tab=tv">Videos</a><div class=gb2><div class=gbd></div></div><a onclick=gbar.logger.il(1,{t:66}); href="http://www.google.com/intl/en/options/" class=gb2>Even more &raquo;</a></div></nobr></div><div id=guser width=100%><nobr><span id=gbn class=gbi></span><span id=gbf class=gbf></span><span id=gbe></span><a target=_top id=gb_70 href="https://www.google.com/accounts/Login?service=&continue=http://www.google.com/patents%3Fhl%3Den&hl=en" class=gb4>Sign in</a><div style="display: none"><div class=gbm id=gbd5 aria-owner=gbg5><div class=gbmc><ol id=gbom class=gbmcc></ol></div></div></div></nobr></div><div class=gbh style=left:0></div><div class=gbh style=right:0></div><div role="alert" style="position: absolute; left: 0; right: 0;"><a href="http://www.google.com/patents/us6599502?hl=en&amp;output=html_text" title="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."><img border="0" src="http://www.google.com/images/cleardot.gif"alt="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."></a></div><div id="guser"><nobr></nobr></div><div style="clear:both;"></div><div id="gb-top-search-box" class="gb-top-search-box-small gb-reset"><table><tr><td class="logo"><a href="http://www.google.com/patents" class="logo-link"><img class="logo-img" src="/intl/en/images/logos/google_logo_41.png" alt="Go to Google Books Home" height="41"/></a></td><td><form action="http://www.google.com/search" name="f" id="vheadf" method="get"><span id="hf"></span><input type="hidden" name="tbm" value="pts"/><input type="hidden" name="tbo" value="1"/><input type="hidden" name="hl" value="en"/><table><tr><td><div class="inputs"><table><tr><td><div class="text-input"><input type="text" name="q" id="vheadq" class="text" maxlength="2048" size="31" value="" title="Search Patents" accesskey="s" autocomplete="off"/><script>window._OC_autoDir &&window._OC_autoDir('vheadq', 'tia-vheadq');</script></div></td><td><div class="submit-input"><input name="btnG" class="submit" type="submit" value=""/></div></td></tr></table></div></td><td class="col-ext-links"><div class="ext-links"><a href="http://www.google.com/advanced_patent_search">&lt;nobr&gt;Advanced Patent Search&lt;/nobr&gt;</a></div></td></tr></table></form></td></tr></table></div><div class="kd-appbar"><h2 class="kd-appname"><a href="/patents">Patents</a></h2><div class="kd-buttonbar left" id="left-toolbar-buttons"><a id="appbar-write-review-link" href=""></a><a id="appbar-view-print-sample-link" href=""></a><a id="appbar-view-ebook-sample-link" href=""></a><a id="appbar-patents-prior-art-finder-link" href="https://www.google.com/patents/related/US6599502"></a><a id="appbar-patents-discuss-this-link" href="http://www.google.com/url?id=SBhhBAABERAJ&amp;q=http://patents.stackexchange.com/redirect/google-patents%3Fpatent%3DUS6599502&amp;usg=AFQjCNGYNakLC__cwJGZ9y-pfkcrpNMAog" data-is-grant="true"></a><a id="appbar-read-patent-link" href="//docs.google.com/viewer?url=patentimages.storage.googleapis.com/pdfs/US6599502.pdf"></a><a id="appbar-download-pdf-link" href="//patentimages.storage.googleapis.com/pdfs/US6599502.pdf"></a><a class="appbar-application-grant-link" data-label="Application" href="/patents/US20020142007"></a><a class="appbar-application-grant-link" data-selected="true" data-label="Grant" href="/patents/US6599502"></a></div><div class="kd-buttonbar right" id="right-toolbar-buttons"></div></div><div id="books-microdata" itemscope=""itemtype="http://schema.org/Book"itemid="http://www.google.com/patents/US6599502" style="display:none"><span itemprop="description">The invention provides methods and compositions relating to intracellular delivering of agents to eukaryotic cells. The compositions include microbial delivery vehicles such as nonvirulent bacteria comprising a first gene encoding a nonsecreted foreign cytolysin operably linked to a heterologous promoter...</span><span itemprop="url">http://www.google.com/patents/US6599502?utm_source=gb-gplus-share</span><span class="main-title" itemprop="name">Patent US6599502 - Intracellular delivery vehicles</span><img itemprop="image" src="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"alt="Patent US6599502 - Intracellular delivery vehicles" title="Patent US6599502 - Intracellular delivery vehicles"></div><div style="display: none"><ol id="ofe-gear-menu-contents" class="gbmcc"><li class="gbe gbmtc"><a class="gbmt goog-menuitem-content" id="" href="http://www.google.com/advanced_patent_search">Advanced Patent Search</a></li></ol></div><table id="viewport_table" cellpadding="0" style="clear:both" cellspacing="0"><tr><td id="viewport_td"><div class=vertical_module_list_row><div id=intl_patents class=about_content><div id=intl_patents_v><table class="patent-bibdata"><tr><td class="patent-bibdata-heading">Publication number</td><td class="single-patent-bibdata">US6599502 B2</td></tr><tr><td class="patent-bibdata-heading">Publication type</td><td class="single-patent-bibdata">Grant</td></tr><tr><td class="patent-bibdata-heading">Application number</td><td class="single-patent-bibdata">US 09/949,109</td></tr><tr><td class="patent-bibdata-heading">Publication date</td><td class="single-patent-bibdata">Jul 29, 2003</td></tr><tr><td class="patent-bibdata-heading">Filing date</td><td class="single-patent-bibdata">Sep 7, 2001</td></tr><tr><td class="patent-bibdata-heading">Priority date<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The priority date is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the date listed."></span></td><td class="single-patent-bibdata">Aug 13, 1998</td></tr><tr><td class="patent-bibdata-heading">Fee status<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The fee status is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the status or dates listed."></span></td><td class="single-patent-bibdata">Paid</td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Also published as</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/EP1105512A1">EP1105512A1</a>, </span><span class="patent-bibdata-value"><a href="/patents/US6004815">US6004815</a>, </span><span class="patent-bibdata-value"><a href="/patents/US6287556">US6287556</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20020142007">US20020142007</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20040115221">US20040115221</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20080166366">US20080166366</a>, </span><span class="patent-bibdata-value"><a href="/patents/WO2000009733A1">WO2000009733A1</a></span></span></td></tr><tr class="patent-bibdata-list-row alternate-patent-number"><td class="patent-bibdata-heading">Publication number</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value">09949109, </span><span class="patent-bibdata-value">949109, </span><span class="patent-bibdata-value">US 6599502 B2, </span><span class="patent-bibdata-value">US 6599502B2, </span><span class="patent-bibdata-value">US-B2-6599502, </span><span class="patent-bibdata-value">US6599502 B2, </span><span class="patent-bibdata-value">US6599502B2</span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Inventors</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Daniel+A.+Portnoy%22">Daniel A. Portnoy</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Darren+E.+Higgins%22">Darren E. Higgins</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Original Assignee</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=inassignee:%22The+Regents+Of+The+University+Of+California%22">The Regents Of The University Of California</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Export Citation</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/US6599502.bibtex">BiBTeX</a>, </span><span class="patent-bibdata-value"><a href="/patents/US6599502.enw">EndNote</a>, </span><span class="patent-bibdata-value"><a href="/patents/US6599502.ris">RefMan</a></span></span></td></tr><tr class="patent-internal-links"><td colspan=2><span class="patent-bibdata-value"><a href="#backward-citations">Patent Citations</a> (8),</span> <span class="patent-bibdata-value"><a href="#npl-citations">Non-Patent Citations</a> (9),</span> <span class="patent-bibdata-value"><a href="#forward-citations">Referenced by</a> (4),</span> <span class="patent-bibdata-value"><a href="#classifications">Classifications</a> (18),</span> <span class="patent-bibdata-value"><a href="#legal-events">Legal Events</a> (6)</span> </td></tr><tr><td colspan=2 class="patent-bibdata-external-link-spacer-top"></td></tr><tr class="patent-bibdata-external-link-spacer-bottom"></tr><tr><td colspan=2><span class="patent-bibdata-heading">External Links:&nbsp;</span><span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=SBhhBAABERAJ&q=http://patft.uspto.gov/netacgi/nph-Parser%3FSect2%3DPTO1%26Sect2%3DHITOFF%26p%3D1%26u%3D/netahtml/PTO/search-bool.html%26r%3D1%26f%3DG%26l%3D50%26d%3DPALL%26RefSrch%3Dyes%26Query%3DPN/6599502&usg=AFQjCNEsE1Wxbl_r1S42PLHl3O8PXUU5lA">USPTO</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=SBhhBAABERAJ&q=http://assignments.uspto.gov/assignments/q%3Fdb%3Dpat%26pat%3D6599502&usg=AFQjCNG__3SWqMKnG5UotiEfYU_pJKiTUw">USPTO Assignment</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=SBhhBAABERAJ&q=http://worldwide.espacenet.com/publicationDetails/biblio%3FCC%3DUS%26NR%3D6599502B2%26KC%3DB2%26FT%3DD&usg=AFQjCNG3G4bBcetiJ276SEaZnBPU-mcvJA">Espacenet</a></span></span></td></tr><tr class="patent-bibdata-group-spacer"></tr></table><div class="number-and-title"><span class="patent-title"><invention-title mxw-id="PT55118918" lang="EN" load-source="patent-office">Intracellular delivery vehicles</invention-title></span><br><span class="patent-number">US 6599502 B2</span></div><div class="patent-section patent-abstract-section"><div class="patent-section-header"><span class="patent-section-title">Abstract</span></div><div class="patent-text"><abstract mxw-id="PA50529608" lang="EN" load-source="patent-office"> <div class="abstract">The invention provides methods and compositions relating to intracellular delivering of agents to eukaryotic cells. The compositions include microbial delivery vehicles such as nonvirulent bacteria comprising a first gene encoding a nonsecreted foreign cytolysin operably linked to a heterologous promoter and a second gene encoding a different foreign agent. The foreign agent may be a nucleic acid or protein, and is frequently bioactive in and therapeutic to the target eukaryote. In addition, the invention provides eukaryotic cells comprising the subject nonvirulent bacteria and nonhuman eukaryotic host organisms comprising such cells.</div>
  </abstract></div></div><div class="patent-section patent-drawings-section"><div class="patent-section-header"><span class="patent-section-title">Images<span class="patent-section-count">(2)</span></span></div><div class="patent-drawings-body"><div class="patent-drawings-carousel"><div class="drawings"><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6599502B2/US06599502-20030729-D00001.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6599502B2/US06599502-20030729-D00001.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6599502B2/US06599502-20030729-D00002.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6599502B2/US06599502-20030729-D00002.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div></div></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img" alt="Previous page"src="/googlebooks/images/kennedy/page_left.png"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img" alt="Next page"src="/googlebooks/images/kennedy/page_right.png"width="21" height="21" /></div></div></div><div class="patent-post-drawings"></div><div class="patent-section patent-claims-section"><div class="patent-section-header"><span class="patent-section-title">Claims<span class="patent-section-count">(18)</span></span></div><div class="patent-text"><div mxw-id="PCLM8523245" lang="EN" load-source="patent-office" class="claims">
    <claim-statement>What is claimed is: </claim-statement> <div class="claim"> <div num="1" id="US-6599502-B2-CLM-00001" class="claim">
      <div class="claim-text">1. A method of introducing a foreign agent into a eukaryotic cell comprising the step of:</div>
      <div class="claim-text">contacting the cell with a nonvirulent bacterium comprising a first gene encoding a nonsecreted foreign functional cytolysin operably linked to a heterologous promoter which expresses the cytolysin in the bacterium, and a second gene encoding a foreign therapeutic agent, different than the cytolysin, under conditions whereby the therapeutic agent enters the cell. </div>
    </div>
    </div> <div class="claim-dependent"> <div num="2" id="US-6599502-B2-CLM-00002" class="claim">
      <div class="claim-text">2. The method of <claim-ref idref="US-6599502-B2-CLM-00001">claim 1</claim-ref>, wherein the bacterium is endocytosed into a vacuole of the cell, the bacterium undergoes lysis and the cytolysin mediates transfer of the therapeutic agent from the vacuole to the cytosol of the cell.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="3" id="US-6599502-B2-CLM-00003" class="claim">
      <div class="claim-text">3. The method of <claim-ref idref="US-6599502-B2-CLM-00001">claim 1</claim-ref>, wherein the bacterium is dead or non-viable.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="4" id="US-6599502-B2-CLM-00004" class="claim">
      <div class="claim-text">4. The method of <claim-ref idref="US-6599502-B2-CLM-00001">claim 1</claim-ref>, wherein the bacterium comprises the cytolysin.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="5" id="US-6599502-B2-CLM-00005" class="claim">
      <div class="claim-text">5. The method of <claim-ref idref="US-6599502-B2-CLM-00001">claim 1</claim-ref>, wherein the therapeutic agent is synthesized by the bacterium.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="6" id="US-6599502-B2-CLM-00006" class="claim">
      <div class="claim-text">6. The method of <claim-ref idref="US-6599502-B2-CLM-00001">claim 1</claim-ref>, wherein the bacterium is nonreplicative and nonintegrative into the host cell genome.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="7" id="US-6599502-B2-CLM-00007" class="claim">
      <div class="claim-text">7. The method of <claim-ref idref="US-6599502-B2-CLM-00001">claim 1</claim-ref>, wherein the bacterium is a dead or nonviable laboratory strain of <i>E. coli. </i> </div>
    </div>
    </div> <div class="claim-dependent"> <div num="8" id="US-6599502-B2-CLM-00008" class="claim">
      <div class="claim-text">8. The method of <claim-ref idref="US-6599502-B2-CLM-00001">claim 1</claim-ref>, wherein the bacterium is a laboratory strain of <i>E. coli </i>and the bacterium comprises the cytolysin.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="9" id="US-6599502-B2-CLM-00009" class="claim">
      <div class="claim-text">9. The method of <claim-ref idref="US-6599502-B2-CLM-00001">claim 1</claim-ref>, wherein the bacterium is a dead or nonviable laboratory strain of <i>E. coli </i>and the bacterium comprises the cytolysin.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="10" id="US-6599502-B2-CLM-00010" class="claim">
      <div class="claim-text">10. The method of <claim-ref idref="US-6599502-B2-CLM-00001">claim 1</claim-ref>, wherein the bacterium is a dead or nonviable laboratory strain of <i>E. coli </i>and the bacterium comprises the cytolysin and the cytolysin is listeriolysin.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="11" id="US-6599502-B2-CLM-00011" class="claim">
      <div class="claim-text">11. The method of <claim-ref idref="US-6599502-B2-CLM-00001">claim 1</claim-ref>, wherein there is no growth or metabolism of the bacterium in the eukaryotic cell.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="12" id="US-6599502-B2-CLM-00012" class="claim">
      <div class="claim-text">12. The method of <claim-ref idref="US-6599502-B2-CLM-00001">claim 1</claim-ref>, wherein the therapeutic agent is selected from an antibiotic, insecticide, fungicide, anti-viral agent, anti-protozoan agent, enzyme, anti-cancer agent, antibody, anti-inflammatory peptide, and transcription factor.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="13" id="US-6599502-B2-CLM-00013" class="claim">
      <div class="claim-text">13. The method of <claim-ref idref="US-6599502-B2-CLM-00001">claim 1</claim-ref>, wherein the method is indicated by a disease selected from the group consisting of cancer, infection, degenerative disease, and diabetes.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="14" id="US-6599502-B2-CLM-00014" class="claim">
      <div class="claim-text">14. The method of <claim-ref idref="US-6599502-B2-CLM-00001">claim 1</claim-ref>, wherein the cell is a leukocyte.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="15" id="US-6599502-B2-CLM-00015" class="claim">
      <div class="claim-text">15. The method of <claim-ref idref="US-6599502-B2-CLM-00001">claim 1</claim-ref>, wherein the cell is a tumor cell.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="16" id="US-6599502-B2-CLM-00016" class="claim">
      <div class="claim-text">16. The method of <claim-ref idref="US-6599502-B2-CLM-00001">claim 1</claim-ref>, wherein the contacting step comprises administering a pharmaceutical composition comprising a therapeutically effective amount of the nonvirulent bacterium.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="17" id="US-6599502-B2-CLM-00017" class="claim">
      <div class="claim-text">17. The method of <claim-ref idref="US-6599502-B2-CLM-00016">claim 16</claim-ref>, wherein the administration is in vivo.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="18" id="US-6599502-B2-CLM-00018" class="claim">
      <div class="claim-text">18. The method of <claim-ref idref="US-6599502-B2-CLM-00016">claim 16</claim-ref>, wherein the administration is ex vivo.</div>
    </div>
  </div> </div></div></div><div class="patent-section patent-description-section"><div class="patent-section-header"><span class="patent-section-title">Description</span></div><div class="patent-text"><div mxw-id="PDES53964065" lang="EN" load-source="patent-office" class="description">
    <heading>CROSS-REFERENCE TO RELATED APPLICATION</heading> <p>This is a continuing application under 35 U.S.C. §120, of U.S. Ser. No. 09/469,197, filed Dec. 21, 1999, now U.S. Pat. No. 6,287,556, which is a continuing application of U.S. Ser. No. 09/133,914, filed Aug. 13, 1998, now U.S. Pat. No. 6,004,815, which are incorporated herein by reference.</p>
    <p>The disclosed inventions were made with Government support under Grant (Contract) Nos. A127655-10 BD37 awarded by the National Institutes of Health. The government may have rights in these inventions.</p>
    <heading>INTRODUCTION</heading> <p>1. Field of the Invention</p>
    <p>The field of this invention is microbial-based intracellular delivery of agents to eukaryotic cells.</p>
    <p>2. Background</p>
    <p>The efficient delivery of macromolecules to the cytosol of mammalian cells is of fundamental importance in such processes as the generation of transfected phenotypes and the study of protein function and localization. Furthermore, delivery of macromolecules to the cytosol is also important for the induction of cell-mediated immunity and is a significant challenge facing the rational design of vaccines to intracellular pathogens. Numerous methodologies currently exist for delivering macromolecules to mammalian cells. These include but are not limited to: mechanical techniques such as electroporation (1) and microinjection (2); fusion methodologies such as fusion with vesicles and liposomes (2); chemical treatments employing the use of ATP or EDTA (3) or the external addition of molecules mixed with pore-forming toxins such as α-toxin of <i>Staphylococcus aureus </i>(4). Many of these methods have a disadvantage in that the molecule to be delivered may require laborious purification (i.e. protein) or the delivery method is limited to use in vitro. In order to overcome these obstacles, investigators have sought to use biological vectors that can enter tissues, cells and specific cellular compartments for the delivery of macromolecules. These vectors are often derived from either retroviruses or bacteria that have evolved to invade and replicate in specific hosts, organs, or cell types. While retroviral vectors have been used for the delivery and subsequent expression of DNA in host cells (5-7), bacterial vectors have been exploited primarily for the delivery of antigenic proteins and more recently adapted for the delivery of DNA to mammalian cells (8-12).</p>
    <p>Relevant Literature</p>
    <p>Lee et al. (1997) U.S. Pat. No. 5,643,599 and Lee et al. (1996) <i>J.Biol.Chem. </i>271, 7249-7252 describe hemolysin loaded liposomes for intracellular delivery of macromolecules. Dietrich, G., et al. (1998) <i>Nature Biotech. </i>16, 138-139 and Ikonomidis, G., et al. (1997) <i>Vaccine </i>15, 433-440 describe the use of <i>Listeria monocytogenes </i>as a macrophage delivery vehicle. Sizemore, D. R., Branstrom, A. A., &amp; Sadoff, J. C. (1995) <i>Science </i>270, 299-302 and Courvalin, P., et al. (1995) <i>C. R. Acad. Sci. III </i>318, 1207-1212 describe the use of attenuated Shigella and invasive strains of <i>Shigella flexneri </i>and <i>E. coli</i>, respectively, as a DNA delivery vehicle. Hess, J., et al. (1998) <i>Proc. Natl. Acad. Sci. U.S.A. </i>95, 5299-5304; and Darji, A., et al. (1995) <i>J. Biotechnol. </i>43, 205-212 describe the expression of listeriolysin in several heterologous systems: an invasive <i>E. coli, Mycobacterium bovis </i>and <i>Listeria innocua, </i>respectively. Moriishi et al. (1996) <i>FEMS Immunol. Med. Microbiol. </i>16, 213-222, 217 describe the transformation of an <i>E.coli </i>with a plasmid encoding listeriolysin. Sanderson, S., Campbell, D. J., &amp; Shastri, N. (1995) <i>J. Exp. Med. </i>182, 1751-1757, describe the cloning of a <i>Listeria monocytogenes </i>genomic library in <i>E. coli</i>. Higgins and Portnoy (1998) <i>Nature Biotech. </i>16, 181-185 review bacterial delivery of DNA.</p>
    <heading>SUMMARY OF THE INVENTION</heading> <p>The invention provides methods and compositions relating to intracellular delivering of agents to eukaryotic cells. The compositions include microbial delivery vehicles such as nonvirulent bacteria comprising a first gene encoding a nonsecreted foreign cytolysin operably linked to a heterologous promoter and a second gene encoding a different foreign agent. In particular embodiments, the bacteria may be variously invasive to the target cell, autolysing within target cell endosomes and preferably, a laboratory strain of <i>E. coli</i>. The cytolysin may lack a functional signal sequence, and is preferably a listeriolysin. The foreign agent may be a nucleic acid or protein, and is frequently bioactive in and therapeutic to the target eukaryote. In addition, the invention provides eukaryotic cells comprising the subject nonvirulent bacteria and nonhuman eukaryotic host organisms comprising such cells. The invention also provides methods for introducing foreign agents into eukaryotic cells comprising the step of contacting the cell in vivo or in vitro with the subject bacteria under conditions whereby the agent enters the cell. In particular embodiments, the bacterium is endocytosed into a vacuole of the cell, undergoes lysis and the cytolysin mediates transfer of the agent from the vacuole to the cytosol of the cell.</p>
    <heading>BRIEF DESCRIPTION OF THE FIGURES</heading> <p>FIG. <b>1</b>. Presentation of SL8/K<sup>b </sup>complex to B3Z T-cells.</p>
    <p>FIG. 2A and 2B. Time requirement for presentation of SL8/K<sup>b </sup>complex.</p>
    <heading>DETAILED DESCRIPTION OF PREFERRED EMBODIMENTS OF THE INVENTION</heading> <p>The following preferred embodiments and examples are offered by way of illustration and not by way of limitation.</p>
    <p>The subject bacteria comprise a first gene encoding a nonsecreted foreign cytolysin operably linked to a heterologous promoter. A wide variety of foreign (i.e. not native to the microbial delivery vehicle) cytolysins may be used so long as the cytolysin is not significantly secreted by the microbe and facilitates cytosolic delivery of the foreign agent as determined by the assays described below. Exemplary cytolysins include phospholipases (see, e.g., Camilli, A., et al., <i>J. Exp. Med. </i>173:751-754 (1991)), pore-forming toxins (e.g., an alpha-toxin), natural cytolysins of gram-positive bacteria, such as listeriolysin O (LLO, e.g. Mengaud, J., et al., <i>Infect. Immun. </i>56:766-772 (1988) and Portnoy, et al., Infect. Immun. 60:2710-2717 (1992)), streptolysin O (SLO, e.g. Palmer M, et al., 1998, Biochemistry 37(8):2378-2383) and perfringolysin O (PFO, e.g. Rossjohn J, et al., Cell 89(5):685-692). Where the target cell is phagosomal, acid activated cytolysins may be advantageously used. For example, listeriolysin O exhibits greater pore-forming ability at mildly acidic pH (the pH conditions within the phagosome), thereby facilitating delivery of the liposome contents to the cytoplasm (see, e.g., Portnoy, et al., <i>Infect. Immun. </i>60:2710-2717 (1992)). Furthermore, natural cytolysins are readily modified to generate mutants which are screened in the assays described below or otherwise known in the art (e.g. Awad M M, et al., Microb Pathog. 1997, 22(5): 275-284) desired activity modifications. In general, the screening assays measure the ability of a candidate cytolysin to confer on a bacterium the ability to render a target cell vacuole permeable to a label (e.g., a fluorescent or radioactive label) that is contained in the vacuole. In a particular example, the invention provides mutations in natural cytolysin wherein highly conserved cysteine residues (e.g., cysteine 460 in PFO, cysteine 486 in LLO) are replaced by conservative amino acid substitutions which are not subject to reduction in order to prepare oxidation/reduction-insensitive cytolysin mutants which exhibit improved lytic activity. Alternatively, mutant cytolysins are selected from naturally occurring mutants by, for example, identifying bacteria which contain cytolysins that are capable of lysing cells over a narrow pH range, preferably the pH range which occurs in phagosomes (pH 5.0-6.0), or under other conditions (e.g., ionic strength) which occur in the targeted phagosomes. Nonsecreted cytolysins may be provided by various mechanisms, e.g. absence of a functional signal sequence, a secretion incompetent microbe, such as microbes having genetic lesions (e.g. a functional signal sequence mutation), or poisoned microbes, etc.</p>
    <p>The bacteria also comprise a second gene encoding a foreign agent different from the cytolysin, and the subject methods may be used to deliver a wide variety of such foreign agents for a variety of applications, including diagnosis, therapy including prophylactics such as immunizations (see, e.g. HIV vaccine, Table 1) and treatments such as gene therapy (especially of single gene disorders amenable to localized treatment, see Table 1, below), biosynthesis, etc.; essentially any agent that the microbial host can be engineered to produce. In a particular embodiment, the agent is largely retained by the microbe until lysis within the target cell vacuole. Note that the first and second genes may be the same, i.e. the same nucleic acid encodes both the cytolysin and the foreign agent. For example, in a particular embodiment, the foreign agent is expressed in frame with the cytolysin as a fusion protein. In other embodiments, the microbes are engineered to deliver libraries of agents for screening, e.g. Tenson T, et al., <i>J Biol Chem </i>1997 Jul 11;272(28):17425-17430.</p>
    <p>A wide variety of nucleic acid-based agents may be delivered, including expression vectors, probes, primers, antisense nucleic acids, knockout/in vectors, ribozymes, etc. For example, the subject bacteria are used to deliver nucleic acids which provide templates for transcription or translation or provide modulators of transcription and/or translation by hybridizing to selected endogenous templates, see, e.g. U.S. Pat. No. 5,399,346 for a non-limiting list of genes that can be administered using gene therapy and diseases that can be treated by gene therapy. For example, polynucleotide agents may provide a coding region operably linked to a transcriptional regulatory region functional in a target mammalian cell, e.g. a human cytomegalovirus (CMV) promoter. In particular, the polynucleotide may encode a transcription factor, whereby expression of the transcription factor in the target cell provides activation or de-activation of targeted gene expression in the target cell. In another example, RNA virus infected cells are targeted by microbes delivering viral RNA-specific ribozymes, e.g. HIV-infected T-cells, leukemia virus infected leukocytes, hepatitis C infected liver cells. In yet another embodiment, labeled probes are delivered which effect in situ hybridization-based diagnostics.</p>
    <p>A wide variety of polypeptide-based agents may also be delivered, including antibiotics, insecticides, fungicides, anti-viral agents, anti-protozoan agents, enzymes, anti-cancer agents (e.g. cyclin dependent kinase (CDK) inhibitors such as P16, P21 or P27), antibodies, anti-inflammatory peptides, transcription factors, antigenic peptides, etc. Exemplary therapeutically active polypeptides which can be delivered by the subject invention are described in Nature Biotech 16(2), entire issue, etc. In a particular embodiment, the invention provides for the delivery to antigen-presenting cells of antigenic polypeptides which are presented in association with MHC proteins. In another particular embodiment, both nucleic acids and proteins are delivered together contemporaneously, in the same administration or in the same microbe. In some such applications, the nucleic acids and proteins can act in concert, e.g. an integrating vector and an integrase, and RNA and a reverse transcriptase, a transposon and a transposase, etc.</p>
    <p>The subject methods may also be used to deliver a wide variety of other foreign agents that are synthesized by the host microbe. For example, microbes may be selected for, or engineered to contain, biosynthetic machinery to produce any microbiologically producible agent compatible with the subject methods (e.g. sufficiently microbe impermeant to provide effective delivery to the target cell). Preferred such agents are those that are contraindicated for convenient direct (e.g. oral) administration, because of, for example, gut inactivation, toxicity, intolerance, impermeability, etc. In fact, even agents providing significant toxicity to the microbial host find use so long as an effective amount of the agent may be loaded (by synthesis) or maintained in the microbe (see, e.g. LLO toxicity, below).</p>
    <p>A wide variety of nonvirulent, non-pathogenic bacteria may be used; preferred microbes are relatively well characterized strains, particularly laboratory strains of <i>E. coli</i>, such as MC4100, MC1061, DH5α, etc. Other bacteria that can be engineered for the invention include well-characterized, nonvirulent, non-pathogenic strains of <i>Listeria monocytogenes, Shigella flexneri, mycobacterium, </i>Salmonella, <i>Bacillus subtilis</i>, etc. In a particular embodiment, the bacteria are attenuated to be nonreplicative, nonintegrative into the host cell genome, and/or non-motile inter- or intra-cellularly. A wide variety of suitable means for microbial attenuation are known in the art. In another particular embodiment, the bacteria are dead or non-viable prior to endocytosis by the target cell or administration to the target organism, obviating any microbial growth or metabolism in the target cell. A wide variety of suitable means for killing or rendering the bacteria nonviable are known in the art, including fixation with organic solvents such as methanol, UV irradiation, heat, freeze-drying, etc. Preferred methods preserve the ability of the microbial membrane and/or wall to retain the cytolysin and the foreign agent. In this embodiment, the first and second genes are sufficiently expressed to load the microbe with an effective amount of the cytolysin and foreign agent prior to microbial cell death. Generally the bacteria contain (i.e. are loaded by expression within the bacteria with) with from about ten to one thousand, preferably from about one hundred to one thousand cytolysin molecules per bacterium.</p>
    <p>The microbes of the invention can be used to deliver the foreign agent to virtually any target cell capable of endocytosis of the subject microbe, including phagocytic, non-phagocytic, pathogenic or diseased cells. Exemplary target animal cells include epithelial cells, endothelial cells, muscle cells, liver cells, pancreatic cells, neural cells, fibroblasts, tumor cells, leukocytes such as macrophages, neutrophils, B-cells, T-cells, monocytes, etc., etc. The subject methods generally require microbial uptake by the target cell and subsequent lysis within the target cell vacuole (including phagosomes and endosomes). While phagocytic target cells generally provide for microbial uptake and lysis, for many cell types, it is necessary to provide the bacteria with an invasin to facilitate or mediate uptake by the target cell and an autolysin to facilitate or mediate autolysis of the bacteria within the target cell vacuole. A wide variety of suitable invasins and autolysins are known in the art. For example, both Sizemore et al. (<i>Science, </i>1995, 270:299-302) and Courvalin et al. (<i>C. R. Acad. Sci. Paris, </i>1995, 318: 1207-12) teach expression of an invasin to effect endocytosis of the bacterium by a target cell and suitable microbial autolysins are described by Cao et al., <i>Infect Immun </i>1998, 66(6): 2984-2986; Margot et al., <i>J. Bacteriol </i>1998, 180(3):749-752; Buist et al., <i>Appl Environ Microbiol, </i>1997, 63(7):2722-2728; Yamanaka et al., <i>FEMS Microbiol Lett, </i>1997, 150(2): 269-275; Romero et al., <i>FEMS Microbiol Lett, </i>1993, 108(1):87-92; Betzner and Keck, <i>Mol Gen Genet, </i>1989, 219(3): 489-491; Lubitz et al., <i>J. Bacteriol, </i>1984, 159(1):385-387; and Tomasz et al., <i>J. Bacteriol, </i>1988, 170(12): 5931-5934. Providing the advantage of delayed lysis are temperature-sensitive autolysins, time-sensitive autolysins (see, e.g. Chang et al., 1995, <i>J Bact </i>177, 3283-3294; Raab et al., 1985, <i>J Mol Biol </i>19, 95-105; Gerds et al., 1995, <i>Mol Microbiol </i>17, 205-210) and addiction (poison/antidote) autolysins, (see e.g. Magnuson R., et al., 1996, <i>J Biol Chem. </i>271(31), 18705-18710; Smith A S, et al., 1997, <i>Mol Microbiol. </i>26(5), 961-970).</p>
    <p>Administration of the microbe to target cells may be in vitro or in vivo according to conventional methodologies. In either case, the methods generally involve growing the microbes, inducing the expression of the first and second genes, and contacting the target cells with an effective amount of bacteria sufficient to effect the desired activity of the foreign agent in the target cell. Immunofluorescense may be used to image and track the contents of the bacteria upon administration to the cells in vivo or in vitro.</p>
    <p>In vitro or ex vivo administration generally involves contacting the target cell with an effective amount of the microbes of the invention. Exemplary in vitro administrations are described and/or cited by reference below. In vitro applications include protein delivery (e.g. for functional determinations, toxin delivery to targeted cells in culture, half-life, degradation and localization determinations), nucleic acid delivery (e.g. DNA to transfected cell lines, genomic libraries to screen and identify specific antigens, i.e. expression cloning, etc.)</p>
    <p>In vivo administration generally involves administering a pharmaceutical composition containing a therapeutically effective amount of the microbes of the invention. Generally, the therapeutically effective amount is between about 1 μg and 100 mg/kg, preferably between about 1 μg and 1 mg/kg. The microbes are formulated into a pharmaceutical composition by combination with an appropriate pharmaceutically acceptable excipient in accordance with routine procedures known to one of ordinary skill in the art. The microbes may be used alone or in appropriate association, as well as in combination with other pharmaceutically active compounds. The microbes may be formulated into preparations in solid, semisolid, or liquid form such as tablets, capsules, powders, granules, ointments, solutions, suppositories, and injections, in usual ways for topical, nasal, oral, parenteral, or surgical administration. Administration in vivo can be oral, mucosal, nasal, bronchial, parenteral, subcutaneous, intravenous, intra-arterial, intra-muscular, intra-organ, intra-tumoral, surgical or in general by any means typical of a gene therapy administration. Administration will be selected as is appropriate for the targeted host cells. Target cells may also be removed from the subject, treated ex vivo, and the cells then replaced into the subject. Exemplary methods for in vivo administration are described in Shen et al., Proc Natl Acad Sci USA 1995, 92(9):3987-3991; Jensen et al, Immunol Rev 1997, 158: 147-157; Szalay et al., Proc Natl Acad Sci USA 1995, 92(26):12389-12392; Belyi et al, FEMS Immunol Med Microbiol 1996, 13(3): 211-213; Frankel et al., J. Immunol 1995, 155(10):4775-4782; Goossens et al., Int Immunol 1995, 7(5):797-805; Schafer et al., J. Immunol 1992, 149(1):53-59; and Linde et al., Vaccine 1991, 9(2):101-105.</p>
    <p>The foregoing methods and compositions are demonstrated to be effective in a wide variety of exemplary applications. In one application, a K12 strain of <i>E.coli </i>is engineered with a signal sequence deficient LLO gene operably linked to the constitutive tet promoter for expressing the cytolysin in the bacterium and a second gene encoding a truncated BRCA1 cancer antigen, under regulatory control of a trc or tac promoter. The cytolysin and cancer antigen are expressed to maximum levels, the bacteria are then fixed with methanol, and the killed bacteria loaded with the cytolysin and cancer antigen are then injected into solid breast tumors in three weekly injections. At four weeks, a cancer antigen-specific cytotoxic T-cell response (CTL response) and tumor size reduction is detected. As shown in Table 1, analogous studies conducted in a variety of animals and animal cell types, both in vivo and in vitro, using a variety of agents, secretory deficient cytolysins, bacterial types and methods demonstrate consistent delivery of the agent to the target cell cytosol, as measured by agent activity, immunoassay, or other delivery monitoring assays described herein.</p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" pgwide="1" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="441pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 1</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Microbial-Based Delivery</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="7"> <colspec colname="1" colwidth="56pt" align="left"> </colspec> <colspec colname="2" colwidth="70pt" align="left"> </colspec> <colspec colname="3" colwidth="105pt" align="left"> </colspec> <colspec colname="4" colwidth="28pt" align="left"> </colspec> <colspec colname="5" colwidth="84pt" align="left"> </colspec> <colspec colname="6" colwidth="63pt" align="left"> </colspec> <colspec colname="7" colwidth="35pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">Cystosol</td>
              </tr> <tr class="description-tr"> <td class="description-td">Target Cell</td>
                <td class="description-td">Indication</td>
                <td class="description-td">Agent</td>
                <td class="description-td">Lysin</td>
                <td class="description-td">Bacteria</td>
                <td class="description-td">Administration</td>
                <td class="description-td">Delivery</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="7" align="center" rowsep="1" class="description-td" colspan="7"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">transformed</td>
                <td class="description-td">acites tumor in nude</td>
                <td class="description-td">human tumor antigen hTA1</td>
                <td class="description-td">LLO</td>
                <td class="description-td"> <i>E.coli</i>, JM109 (DE3)</td>
                <td class="description-td">intraperitoneal</td>
                <td class="description-td">+++</td>
              </tr> <tr class="description-tr"> <td class="description-td">human</td>
                <td class="description-td">mice</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">injection</td>
              </tr> <tr class="description-tr"> <td class="description-td">macrophage</td>
              </tr> <tr class="description-tr"> <td class="description-td">rat liver</td>
                <td class="description-td">hepatocellular</td>
                <td class="description-td">p51 tumor suppressor</td>
                <td class="description-td">LLO<sup>M1</sup> </td>
                <td class="description-td"> <i>E.coli</i>, DP-E3619,</td>
                <td class="description-td">in situ; intratumor</td>
                <td class="description-td">+++</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">carcinoma</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">invasin/autolysin</td>
                <td class="description-td">injection</td>
              </tr> <tr class="description-tr"> <td class="description-td">rat kidney</td>
                <td class="description-td">genetic nephopathy</td>
                <td class="description-td">angiotensin converting enzyme</td>
                <td class="description-td">LLO<sup>M2</sup> </td>
                <td class="description-td"> <i>S. typhimurium</i>, attenuated,</td>
                <td class="description-td">ex vivo</td>
                <td class="description-td">+++</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">invasin/autolysin</td>
              </tr> <tr class="description-tr"> <td class="description-td">mouse brain</td>
                <td class="description-td">neurodegeneration</td>
                <td class="description-td">cFos gene expression construct</td>
                <td class="description-td">LLO<sup>M3</sup> </td>
                <td class="description-td"> <i>S. typhimurium</i>, attenuated,</td>
                <td class="description-td">in situ; intracranial</td>
                <td class="description-td">+++</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">invasin/autolysin</td>
                <td class="description-td">implant</td>
              </tr> <tr class="description-tr"> <td class="description-td">mouse pancreas</td>
                <td class="description-td">transformation</td>
                <td class="description-td">γ-interferon expression construct</td>
                <td class="description-td">LLO<sup>M4</sup> </td>
                <td class="description-td"> <i>E.coli</i>, JM109 (DE3)</td>
                <td class="description-td">in vitro</td>
                <td class="description-td">+++</td>
              </tr> <tr class="description-tr"> <td class="description-td">pig muscle</td>
                <td class="description-td">muscular atrophy</td>
                <td class="description-td">insulin-like growth factor I (IGF-I)</td>
                <td class="description-td">PLO</td>
                <td class="description-td"> <i>E. coli</i>, DP-E3618,</td>
                <td class="description-td">in situ; i.m.</td>
                <td class="description-td">+++</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">invasin/autolysin</td>
                <td class="description-td">injection</td>
              </tr> <tr class="description-tr"> <td class="description-td">human breast</td>
                <td class="description-td">transformation</td>
                <td class="description-td">anti-estrogen receptor antibody</td>
                <td class="description-td">SLO</td>
                <td class="description-td"> <i>E. coli</i>, DP-3617,</td>
                <td class="description-td">in vitro</td>
                <td class="description-td">+++</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">expression construct</td>
                <td class="description-td"> </td>
                <td class="description-td">invasin/autolysin</td>
              </tr> <tr class="description-tr"> <td class="description-td">human prostate</td>
                <td class="description-td">localized prostatic</td>
                <td class="description-td">ribozyme or antisense against CDK</td>
                <td class="description-td">LLO</td>
                <td class="description-td"> <i>S. typhimurium</i>, attenuated</td>
                <td class="description-td">in situ; intratumor</td>
                <td class="description-td">+++</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">carcinoma</td>
                <td class="description-td">2 or CDK4</td>
                <td class="description-td"> </td>
                <td class="description-td">invasin/autolysin</td>
                <td class="description-td">injection</td>
              </tr> <tr class="description-tr"> <td class="description-td">human lymphoid</td>
                <td class="description-td">lymphoma</td>
                <td class="description-td">tumor necrosis factor (TNF)</td>
                <td class="description-td">LLO</td>
                <td class="description-td"> <i>E. coli</i>, DP-E3616</td>
                <td class="description-td">ex vivo</td>
                <td class="description-td">+++</td>
              </tr> <tr class="description-tr"> <td class="description-td">human bone</td>
                <td class="description-td">mylomoid leukemia</td>
                <td class="description-td">Hepatocyte growth factor/scatter</td>
                <td class="description-td">SLO</td>
                <td class="description-td"> <i>E. coli</i>, DP-E3615</td>
                <td class="description-td">ex vivo</td>
                <td class="description-td">+++</td>
              </tr> <tr class="description-tr"> <td class="description-td">marrow</td>
                <td class="description-td"> </td>
                <td class="description-td">factor (HGF/SF)</td>
              </tr> <tr class="description-tr"> <td class="description-td">human lymphoid</td>
                <td class="description-td">HIV infection</td>
                <td class="description-td">HIV RT gene-specific ribozyme</td>
                <td class="description-td">LLO</td>
                <td class="description-td"> <i>S. typhimurium</i>, attenuated</td>
                <td class="description-td">ex vivo</td>
                <td class="description-td">+++</td>
              </tr> <tr class="description-tr"> <td class="description-td">human hepatic</td>
                <td class="description-td">Hepatitis C infection</td>
                <td class="description-td">Hepatitis C virus-specific ribozyme</td>
                <td class="description-td">LLO</td>
                <td class="description-td"> <i>S. typhimurium</i>, attenuated,</td>
                <td class="description-td">in vivo; direct</td>
                <td class="description-td">+++</td>
              </tr> <tr class="description-tr"> <td class="description-td">cells</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">invasin/autolysin</td>
                <td class="description-td">injection</td>
              </tr> <tr class="description-tr"> <td class="description-td">human hepatic</td>
                <td class="description-td">diabetes</td>
                <td class="description-td">insulin receptor expression</td>
                <td class="description-td">LLO</td>
                <td class="description-td"> <i>S. typhimurium</i>, attenuated,</td>
                <td class="description-td">in vivo; direct</td>
                <td class="description-td">+++</td>
              </tr> <tr class="description-tr"> <td class="description-td">cells</td>
                <td class="description-td"> </td>
                <td class="description-td">construct</td>
                <td class="description-td"> </td>
                <td class="description-td">invasin/autolysin</td>
                <td class="description-td">injection</td>
              </tr> <tr class="description-tr"> <td class="description-td">human beta islet</td>
                <td class="description-td">diabetes</td>
                <td class="description-td">insulin expression construct</td>
                <td class="description-td">LLO</td>
                <td class="description-td"> <i>S. typhimurium</i>, attenuated,</td>
                <td class="description-td">in vivo; direct</td>
                <td class="description-td">+++</td>
              </tr> <tr class="description-tr"> <td class="description-td">cells</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">invasin/autolysin</td>
                <td class="description-td">injection</td>
              </tr> <tr class="description-tr"> <td class="description-td">murine fibroblast</td>
                <td class="description-td">fibroblastoma</td>
                <td class="description-td">diptheria toxin</td>
                <td class="description-td">LLO</td>
                <td class="description-td"> <i>S. typhimurium</i>, attenuated,</td>
                <td class="description-td">in vivo; direct</td>
                <td class="description-td">+++</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">invasin/autolysin</td>
                <td class="description-td">injection</td>
              </tr> <tr class="description-tr"> <td class="description-td">feline retinal</td>
                <td class="description-td">retinal degenerative</td>
                <td class="description-td">cGMP phosphodiesterase-beta</td>
                <td class="description-td">LLO</td>
                <td class="description-td"> <i>E. coli</i>, JM109 (DE3),</td>
                <td class="description-td">intraocular</td>
                <td class="description-td">+++</td>
              </tr> <tr class="description-tr"> <td class="description-td">cells</td>
                <td class="description-td">disease</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">invasin/autolysin</td>
                <td class="description-td">injection</td>
              </tr> <tr class="description-tr"> <td class="description-td">human cytotoxic</td>
                <td class="description-td">melanoma</td>
                <td class="description-td">melanosomal proteins</td>
                <td class="description-td">LLO</td>
                <td class="description-td"> <i>E.coli</i>, JM109 (DE3)</td>
                <td class="description-td">in vivo; IV</td>
                <td class="description-td">+++</td>
              </tr> <tr class="description-tr"> <td class="description-td">T-cells</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">injection</td>
              </tr> <tr class="description-tr"> <td class="description-td">human epithelium</td>
                <td class="description-td">Herpes infection</td>
                <td class="description-td">antisense RNAseP construct</td>
                <td class="description-td">LLO</td>
                <td class="description-td"> <i>S. typhimurium</i>, attenuated,</td>
                <td class="description-td">in vivo; oral, opical</td>
                <td class="description-td">+++</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">invasin/autolysin</td>
                <td class="description-td">abrasion</td>
              </tr> <tr class="description-tr"> <td class="description-td">murine</td>
                <td class="description-td">IL-2 production</td>
                <td class="description-td">NFAT</td>
                <td class="description-td">LLO</td>
                <td class="description-td">
                  <i>S. typhimurium</i>
                </td>
                <td class="description-td">in vitro</td>
                <td class="description-td">+++</td>
              </tr> <tr class="description-tr"> <td class="description-td">macrophages</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="7" align="center" rowsep="1" class="description-td" colspan="7"> </td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="7" align="left" class="description-td" colspan="7">LLO<sup>M1-M4 </sup>are LLO mutants M1(Cys486Ser), M2(Cys486Met), M3(Trp492Ala) and M4(del491-493), respectively. </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <heading>EXAMPLES</heading> <p>I. Delivery of Protein to the Cytosol of Macrophages Using <i>Escherichia coli </i>K-12 Expressing Listeriolysin O</p>
    <p> <i>Listeria monocytogenes </i>is a bacterial pathogen that replicates within the cytosol of mammalian cells. <i>L. monocytogenes </i>has been used extensively as a model for the study of cell-mediated immunity and as a model pathogen for understanding the basis of intracellular pathogenesis (13, 14). Following internalization into host cells, bacteria are initially contained within host vacuoles then subsequently lyse these vacuoles to gain access to the cytosol. The ability of <i>L. monocytogenes </i>to lyse the vacuole and enter the cytosol is primarily mediated by listeriolysin O (LLO). LLO is a member of a family of related pore-forming cytolysins secreted by diverse species of gram positive bacteria (15). LLO encapsulated into pH-sensitive liposomes has been used as a vehicle to deliver co-encapsulated protein to the cytosol of macrophages (16). Moreover, purified LLO when mixed with foreign proteins and added to mammalian cells can mediate the delivery of protein to the cytosol and has been exploited for delivery to host cells both in vitro and in vivo (17-19). However, both of these methods require the purification of LLO and the protein to be delivered. Here, we show that <i>Escherichia coli </i>expressing cytoplasmic LLO can be used to efficiently deliver co-expressed proteins to the cytosol of macrophages. The utility of this system to deliver a large active protein to the cytosol was demonstrated by the delivery of <i>E. coli </i>β-galactosidase (β-gal). Using chicken ovalbumin (OVA) we demonstrate the rapid delivery of protein to the cytosol of macrophages and the ability of the <i>E. coli</i>/LLO system to efficiently deliver OVA to the MHC class I pathway of antigen processing and presentation. Moreover, the time required for processing and presentation of an OVA-derived peptide to CD8<sup>+</sup>T cells, when OVA was delivered using this system, was equivalent to that previously reported when purified OVA was introduced into the cytosol by alternative methods such as scrape-loading or liposomes (16, 20, 21).</p>
    <p>Bacterial Strains and Plasmids. All bacterial strains and plasmids used in this report are listed in Table 2.</p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" pgwide="1" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="280pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 2</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> <i>E. coli </i>strains and plasmids used in this work</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="1" colwidth="56pt" align="left"> </colspec> <colspec colname="2" colwidth="154pt" align="left"> </colspec> <colspec colname="3" colwidth="70pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Strain or Plasmid</td>
                <td class="description-td">Description</td>
                <td class="description-td">Reference or source</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="3" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">pACYC184</td>
                <td class="description-td">cloning vector; Tc<sup>r </sup>Cm<sup>r</sup> </td>
                <td class="description-td">(23)</td>
              </tr> <tr class="description-tr"> <td class="description-td">pET28a</td>
                <td class="description-td">over-expression vector; Kan<sup>r</sup> </td>
                <td class="description-td">(Novagen, Inc.)</td>
              </tr> <tr class="description-tr"> <td class="description-td">pET29b</td>
                <td class="description-td">over-expression vector; Kan<sup>r</sup> </td>
                <td class="description-td">(Novagen, Inc.)</td>
              </tr> <tr class="description-tr"> <td class="description-td">pTL61T</td>
                <td class="description-td">lacZ transcriptional fusion vector; Ap<sup>r</sup> </td>
                <td class="description-td">(24)</td>
              </tr> <tr class="description-tr"> <td class="description-td">pBluescript SK-</td>
                <td class="description-td">cloning vector; Ap<sup>r</sup> </td>
                <td class="description-td">(Stratagene, Inc.)</td>
              </tr> <tr class="description-tr"> <td class="description-td">pTrcHisC/Ova</td>
                <td class="description-td">pTrcHisC::ova</td>
                <td class="description-td">(D. Campbell, UCB)</td>
              </tr> <tr class="description-tr"> <td class="description-td">pDP3615</td>
                <td class="description-td">pACYC184 tet::hly</td>
                <td class="description-td">Herein</td>
              </tr> <tr class="description-tr"> <td class="description-td">pDP3616</td>
                <td class="description-td">pET28a pT7::ova</td>
                <td class="description-td">Herein</td>
              </tr> <tr class="description-tr"> <td class="description-td">pDH70</td>
                <td class="description-td">pBluescript SK- pT7::lacZ</td>
                <td class="description-td">Herein</td>
              </tr> <tr class="description-tr"> <td class="description-td">MC4100(DE3)</td>
                <td class="description-td">F<sup>−</sup> araD 139 Δ(argF-lac)U169 rpsL150</td>
                <td class="description-td">(Schifferli, U. Penn)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">(Str<sup>r</sup>)relAl flbB530I deoCl ptsF25 rbsR with DE3,</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">a λ prophage carrying the T7 RNA polymerase gene</td>
              </tr> <tr class="description-tr"> <td class="description-td">JM109(DE3)</td>
                <td class="description-td">endAl recAl gyrA96 thi hsdR17 relAl supE44</td>
                <td class="description-td">(Promega, Co.)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Δ(lac-proAB) [F′ traD36 proAB lacl<sup>q</sup>ZΔM15] DE3</td>
              </tr> <tr class="description-tr"> <td class="description-td">DP-E3615</td>
                <td class="description-td">MC4100(DE3) harboring pDP3615</td>
                <td class="description-td">Herein</td>
              </tr> <tr class="description-tr"> <td class="description-td">DP-E3616</td>
                <td class="description-td">MC4100(DE3) harboring pDP3616</td>
                <td class="description-td">Herein</td>
              </tr> <tr class="description-tr"> <td class="description-td">DP-E3617</td>
                <td class="description-td">MC4100(DE3) harboring pDP3615 and pDP3616</td>
                <td class="description-td">Herein</td>
              </tr> <tr class="description-tr"> <td class="description-td">DP-E3618</td>
                <td class="description-td">JM109(DE3) harboring pDH70</td>
                <td class="description-td">Herein</td>
              </tr> <tr class="description-tr"> <td class="description-td">DP-E3619</td>
                <td class="description-td">JM1O9(DE3) harboring pDP3615 and pDH70</td>
                <td class="description-td">Herein</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="3" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="3" align="left" class="description-td" colspan="3">Ap<sup>r</sup>, ampicillin resistant; Tc<sup>r</sup>, tetracycline resistant; Kan<sup>r</sup>, kanamycin resistant; Cm<sup>r</sup>, chloramphenicol resistant; Str<sup>r</sup>, streptomycin resistant </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p>Plasmid pDP3615 was generated by PCR amplification of the hly gene encoding LLO lacking its secretion signal sequence (22). DNA sequences encoding mature LLO were first PCR amplified and cloned into pET29b (Novagen, Inc., Madison, Wis.) using oligonucleotide primer 5′-GGAATTCCATATGAAGGATGCATCTGCATTCAAT-3′ (SEQ ID NO:1) generating a NdeI restriction site at the 5′ end of the gene fragment and primer 5′-CGGGATCCTTATTATTCGATTGGATTATCTACT-3′ (SEQ ID NO:2) generating a BamHI restriction site at the 3′ end of the gene fragment. Following ligation into the pET29b vector, the DNA sequences encoding mature LLO along with the upstream translation initiation site found in pET29b were amplified using primer 5′-CGCGATATCCTCTAGAAATAATTTTG-3′ (SEQ ID NO:3) generating an EcoRI restriction site at the 5′ end of the gene fragment and the same primer used previously to generate a BamHI restriction site at the 3′ end of the gene fragment. The amplified fragment was ligated into pACYC 184 (23) placing transcription of the mature hly gene under control of the tet gene promoter. Plasmid pDP3616 was generated by subcloning a NcoI-HindIII fragment containing DNA sequences encoding truncated OVA from plasmid pTrcHisC/OVA. The DNA fragment was ligated into the over-expression vector pET28a (Novagen, Inc., Madison, Wis.). Plasmid pDH70 was generated by PCR amplification of the promoterless lacZ gene in plasmid pTL61T (24) using oligonucleotide primers 5′-AGGCGTCGACGGTTAATACGACCGGGATCGAG-3′ (SEQ ID NO:4) and 5′-AGGCGTCGACAGGCCTTACGCGAAATACGGGCAGACATGG-3′ (SEQ ID NO:5) generating SalI restriction sites at both the 5′ and 3′ ends of the fragment. The amplified fragment was ligated into pBluescript SK-(Stratagene, Inc., La Jolla, Calif.) placing transcription of the lacZ gene under control of a phage T7 promoter. Plasmid DNA was transferred to <i>E. coli </i>strains by transformation, using standard methods (25). <i>E. coli </i>strains were grown in Luria-Bertani (LB) medium. The strains were stored at −70° C. in LB medium plus 40% glycerol. Antibiotics were used at the following concentrations: ampicillin, 100 μg/ml; chloramphenicol, 40 μg/ml; and kanamycin, 30 μg/ml.</p>
    <p>Expression of Target Proteins. <i>E. coli </i>strains were inoculated from a LB agar plate into 2 mls of LB medium and grown overnight to stationary phase at 37° C. with aeration. Cultures were diluted 1:100 in 10 mls of LB medium in 250 ml flasks and grown 2 hours with aeration at 30° C. Target protein expression was induced by the addition of isopropyl-β-D-thiogalactopyranoside (IPTG) to 1 mM and growth continued until cultures reached an OD<sub>600 </sub>of 0.5. Equivalent numbers of bacteria were centrifuged (14,000×g) for 1 minute and washed once with phosphate buffered saline (PBS). Washed samples were suspended in Final Sample Buffer (0.0625M Tris pH 6.8, 2% SDS, 10% glycerol, 0.01% bromophenol blue) boiled for 5 minutes and total cellular protein analyzed by polyacrylamide gel electrophoresis followed by staining with Coomassie Brilliant Blue.</p>
    <p>Determination of Hemolytic Activity. Following bacterial growth and induction of target proteins, 1 ml aliquots of bacteria were centrifuged (14,000×g) for 1 minute and washed once with PBS. Samples were resuspended in 1 ml of PBS and lysed by sonication. Soluble extract fractions were obtained by centrifuging lysed samples for 10 minutes (14,000×g) at 4° C. and saving the supernatant. Hemolytic activity in the soluble fractions was determined as previously described (26) and is expressed as the reciprocal of the dilution of extracts required to lyse 50% of sheep erythrocytes.</p>
    <p>Cell Culture. Cell lines were maintained in RPMI 1640 medium or DMEM supplemented with 10% fetal bovine serum (Hyclone Laboratories, Inc., Logan, Utah), 2 mM glutamine, 1 mM pyruvate, 50 μM 2-mercaptoethanol, penicillin (200 units/ml), and streptomycin (200 μg/ml) at 37° C. in a 5% CO<sub>2</sub>/air atmosphere. The IC-21 and Raw 309 Cr.1 mouse macrophage cell lines were obtained from the American Type Culture Collection (ATCC, Rockford, Md.). The B3Z T-cell hybrid is a LacZ-inducible CD8<sup>+</sup> T-cell hybridoma specific for OVA residues 257-264, SIINFEKL (SL8)(SEQ ID NO:6), presented on the murine K<sup>b </sup>MHC class I molecule (27, 28).</p>
    <p>Delivery of Protein to the Cytosol of Macrophages. IC-21 cells were seeded onto 18 mm glass coverslips in 35 mm dishes in RPMI medium without antibiotics. One hour prior to addition of bacteria, dishes were placed at 4° C. Medium was removed from the dishes and <i>E. coli </i>were added in cold RPMI medium without antibiotics to obtain an infection ratio of one bacterium/macrophage. Samples were incubated at 4° C. for one hour to allow association of bacteria and macrophages. Samples were washed five times with 3 mls of cold PBS and 37° C. RPMI medium added. Samples were incubated at 37° C. in a 5% CO<sub>2</sub>/air atmosphere for up to one hour. At varying intervals during incubation of macrophages and <i>E. coli</i>, coverslips were removed and fixed for subsequent detection of protein.</p>
    <p>Detection of Target Protein in Macrophages. Following delivery of β-gal to macrophages, coverslips were fixed in cold 2% formaldehyde/0.2% glutaraldehyde for 5 minutes at 4° C. β-gal activity was detected by staining with 5-bromo-4-chloro-3-indolyl β-galactopyranoside (X-gal, Sigma Immunochemicals, St. Louis, Mo.) as previously described (27). For detection of OVA, coverslips were fixed in 3.2% electron microscopy grade paraformaldehyde (Electron Microscopy Sciences, Ft. Washington, Pa.) overnight at 4° C. in aluminum foil wrapped containers. OVA was detected by immunofluorescence as previously described (29) with the exception that polyclonal rabbit anti-OVA antibody (Calbiochem, San Diego, Calif.) and LRSC-conjugated donkey anti-rabbit IgG antibody (Jackson ImmunoResearch, West Grove, Pa.) were used for detection.</p>
    <p>Antigen Presentation Assays. Presentation of SL8/K<sup>b </sup>complex to B3Z cells was determined as previously described (30). Briefly, <i>E. coli </i>were added to 1×10<sup>5 </sup>antigen presenting cells (APCs) in each well of a 96-well microtiter plate. Following one hour of incubation at 37° C. in a 5% CO<sub>2</sub>/air atmosphere, extracellular bacteria were removed by washing three times with PBS and 1×10<sup>5 </sup>B3Z T cells were added to each well in medium containing 100 μg/ml gentamicin. Following 15 hours of incubation at 37° C. in a 5% CO<sub>2</sub>/air atmosphere, cultures were washed once with PBS and lysed by addition of 100 μl PBS buffer containing 100 μM 2-mercaptoethanol, 9 mM MgCl<sub>2</sub>, 0.125% NP40, and 0.15 mM chlorophenolred-β-galactoside (CPRG, Calbiochem, San Diego, Calif.). After 4-6 hours at 37° C., 50 μl of stop buffer (300 mM glycine and 15 mM EDTA in water) was added and the absorbance at 570 nm of each well was determined using a 96-well plate reader. Where appropriate, APCs were fixed with 1% paraformaldehyde prior to the addition of B3Z cells as described (31). Synthetic SL8 peptide was obtained from Research Genetics, Inc. (Huntsville, Ala.) and was incubated with APCs and B3Z cells at a saturating concentration of 90 nM in RPMI medium.</p>
    <p>Expression of Listeriolysin O and Target Proteins. To facilitate expression of mature cytoplasmic LLO in <i>E. coli, </i>the hly gene encoding LLO lacking its N-terminal signal sequence (22) was cloned into the plasmid vector pACYC184 to generate pDP3615 as described in Materials and Methods. Transcription of the truncated hly gene in pDP3615 is under the control of the constitutive tet gene promoter. Proteins to be delivered to the cytosol of macrophages were expressed from co-resident plasmids in <i>E. coli</i>. We chose chicken ovalbumin (OVA) as one of the representative proteins to deliver to the cytosol of macrophages. OVA is not toxic to <i>E. coli </i>and can be readily expressed to high levels (32). A plasmid encoding truncated (32 kD) OVA was transformed into <i>E. coli </i>along with pDP3615. In order to determine if a large protein with a measurable enzymatic activity could be delivered to the cytosol of macrophages, we expressed β-galactosidase (β-gal) along with LLO in <i>E. coli</i>. A plasmid containing the gene encoding β-gal, was transformed into <i>E. coli </i>along with plasmid pDP3615. Expression of both OVA and β-gal in these strains is under the control of IPTG-inducible phage T7 RNA polymerase. We next analyzed the hemolytic activity and protein expression profiles of these strains. Following IPTG induction, OVA and β-gal were expressed to approximately 20% of the total <i>E. coli </i>cellular protein as determined by SDS-PAGE. To verify expression of active LLO protein within <i>E. coli</i>, hemolytic activity contained in the soluble fraction of <i>E. Coli </i>extracts was determined as described above. All of the strains expressing LLO contained approximately 500-600 hemolytic units of activity in the soluble extracts. No measurable hemolytic activity was found in the culture medium in which the <i>E. coli </i>were grown. These data indicate that functional LLO protein was contained within the <i>E. coli </i>cells and not secreted to the extracellular environment.</p>
    <p>Delivery of Protein to the Cytosol of Macrophages. To examine the ability of <i>E. Coli </i>expressing LLO to deliver a co-expressed target protein to the cytosol of macrophages, <i>E. coli </i>expressing LLO and OVA were added to macrophages to obtain an infection ratio of approximately one bacterium/macrophage. The presence of OVA either in phagosomes or in the cytosol was determined by immunofluorescence microscopy. When bacteria expressing OVA in the absence of LLO were added to macrophages, protein was contained within phagosomes and no OVA could be detected in the cytosol of macrophages within one hour following phagocytosis. In contrast, when bacteria expressing both LLO and OVA were added to macrophages, OVA protein could be detected throughout the entire cytosolic compartment within 30 minutes of bacterial uptake. Moreover, at least 50% of macrophages that had phagocytosed a single bacterium demonstrated release of OVA into the cytosol within 30 minutes following bacterial uptake. In some instances, OVA protein could be detected leaking from phagosomes into the cytosol as early as 10 minutes post-phagocytosis of bacteria.</p>
    <p>Since release of OVA into the cytosol occurs subsequent to degradation of the <i>E. coli </i>within macrophage phagosomes, it was possible that proteins contained within the bacteria were also partially degraded or inactivated before release into the cytosol. To examine whether full-length β-gal could be delivered to the cytosol of macrophages and retain its biological activity, <i>E. coli </i>expressing LLO and β-gal were added to macrophages to obtain an infection ratio of approximately one bacterium/macrophage. β-gal activity in the cytosol was then determined by staining macrophages with X-gal. Our data indicate that β-gal activity could be detected throughout the cytosol within 30 minutes following bacterial uptake. Following phagocytosis of <i>E. coli </i>expressing β-gal in the absence of LLO, β-gal activity was detected sequestered within phagosomes. β-gal is a 116 kD protein that functions as a 465 kD tetramer (33). Whether β-gal is associated as a tetramer prior to release into the cytosol is unknown, but these data indicate that at a minimum a 116 kD protein can be delivered to the cytosol of macrophages using the <i>E. coli</i>/LLO delivery system and still retain its enzymatic activity.</p>
    <p>Delivery of OVA to the MHC Class I Pathway for Antigen Presentation. Immunity to intracellular bacterial pathogens and viruses often requires the generation of cytotoxic T-lymphocytes (CTLs) that recognize and kill infected cells (13, 34, 35). Efficient processing and presentation of antigens to CTLs typically requires the endogenous synthesis of the antigen within the cytosol of the infected cell or introduction of the antigenic protein into the cytosol of an antigen presenting cell (APC) (36, 37). Once in the cytosol, proteases process the antigen to peptide epitopes which are subsequently presented on the surface of the APC in association with major histocompatibility (MHC) class I molecules for recognition by CD8<sup>+</sup> CTLs (37-39).</p>
    <p>We wished to examine the ability of <i>E. coli </i>expressing LLO and an antigenic protein to deliver the antigen to the cytosol of macrophages for processing and presentation on MHC class I molecules. <i>E. coli </i>expressing LLO and OVA were added to macrophages and the processing and presentation of a peptide epitope derived from OVA was accessed using the B3Z T-cell hybrid. B3Z is a LacZ-inducible CD8<sup>+</sup> T-cell hybrid specific for OVA residues 257-264 (SIINFEKL, SEQ ID NO:6) presented on the murine K<sup>b </sup>MHC class I molecule (27, 28). The presentation of the SIINFEKL (SEQ ID NO:6) epitope (SL8) to B3Z cells results in the induction of β-gal synthesis by B3Z. The amount of β-gal produced can be measured by the hydrolysis of the chromogenic substrate CPRG and is an indication of the amount of SL8/K<sup>b </sup>complexes presented on the surface of APCs (27, 30). Bacteria were added to macrophages and phagocytosis allowed to proceed for one hour, followed by the removal of extracellular bacteria and addition of B3Z T-cells. As shown in FIG. 1, processing and presentation of SL8/K<sup>b </sup>to B3Z T-cells occurred when <i>E. coli </i>expressing both LLO and OVA were added to macrophages. In this figure, the indicated number of <i>E. coli </i>were added to 1×10<sup>5 </sup>Raw 309 Cr.1 APCs in each well of a 96-well plate. Following one hour of incubation at 37° C. in a 5% CO<sub>2</sub>/air atmosphere, extracellular bacteria were removed by washing with PBS and 1×10<sup>5 </sup>B3Z T-cells were added to each well in medium containing 100 μg/ml gentamicin. Following 15 hours of incubation at 37° C. in a 5% CO<sub>2</sub>/air atmosphere, presentation of SL8/K<sup>b </sup>to B3Z cells was assayed as described (27, 30) and indicated as an increase in the absorbance at 570 nm. <i>E. coli </i>strains added to APCs expressed LLO, DP-E3615 (∘); OVA, DP-E3616 (); or LLO and OVA, DP-E3617, (□). (▪) indicates the level of activation obtained when APCs were incubated with 90 nM synthetic SL8 and B3Z cells. Data presented is from triplicate groups of wells from one of several repeated experiments with identical results. As shown, antigen presentation could be detected with as few as 1 bacterium added/10 macrophages. At a ratio of 10 bacteria added/macrophage, which resulted in the phagocytosis of one to two bacteria/macrophage, the level of presentation was equivalent to the maximal activity achieved by incubating macrophages with a saturating dose (90 nM) of synthetic SL8 peptide. No presentation of SL8/K<sup>b </sup>complex could be detected when <i>E. coli </i>expressing OVA in the absence of LLO were added to macrophages. Equivalent results were obtained when primary bone marrow and peritoneal derived macrophages were used in antigen presentation experiments. The decrease in absorbance seen when 10<sup>7 </sup>and 10<sup>8 </sup>LLO expressing bacteria were added was due to visible damage to the macrophages.</p>
    <p>Processing and Presentation of SL8/K<sup>b </sup>Complex is Rapid. It has been previously demonstrated that OVA is efficiently processed and peptide/MHC complexes presented on the surface of APCs within two to four hours following delivery of OVA to the cytosol using alternative methods such as encapsulation in liposomes or scrape-loading (16, 20, 21). Data in FIG. 1 indicate that considerable processing and presentation of antigenic peptides can occur with as few as one bacterium added/macrophage. However, processing and presentation of antigen was allowed to occur for greater than 15 hours prior to measuring T-cell activation. It is possible that the time required for efficient processing of antigen has been altered by delivering protein to the cytosol using this method. We wished to examine the time necessary for antigen processing and presentation of peptide/MHC complexes when protein is delivered using the <i>E. coli</i>/LLO system.</p>
    <p>Paraformaldehyde fixation of macrophages prevents further phagocytosis of bacteria and has been shown to crosslink surface MHC class I molecules to associated β<sub>2</sub>-microglobulin (31). Thus, fixation stabilizes peptide/MHC complexes present on the cell surface and arrests any further processing and presentation of peptides. We examined the time required for antigen processing and presentation by fixing APCs with paraformaldehyde at varying intervals after addition of bacteria and measuring SL8/K<sup>b </sup>presentation to B3Z T-cells. First, the effect on antigen presentation of fixing macrophages prior to the addition of bacteria was addressed (FIG. <b>2</b>A). In FIG. 2A, 1×10<sup>6 </sup> <i>E. coli </i>strain DP-E3617 expressing LLO and OVA were added to 1×10<sup>5 </sup>IC-21 macrophages in each well of a 96-well plate. Processing and presentation of SL8/K<sup>b </sup>was assayed as described in FIG. <b>1</b>. Immediately prior to the addition of bacteria, APCs were either left untreated (Not Fixed) or fixed (Pre-Fixed) with 1% paraformaldehyde as described (31) Fixing APCs prior to the addition of bacteria completely abrogated the ability of macrophages to process and present SL8/K<sup>b </sup>complex to B3Z T-cells, as evident by an equivalent response as that seen when no bacteria were added to the macrophages (FIG. <b>1</b>).</p>
    <p>In FIG. 2B, the time requirement for processing and presentation of SL8/K<sup>b </sup>complex was addressed. Here, 1×10<sup>6 </sup> <i>E. coli </i>strain DP-E3617 expressing LLO and OVA were added to 1×10<sup>5 </sup>IC-21 APCs in each well of a 96-well plate. Following one hour of incubation at 37° C. in a 5% CO<sub>2</sub>/air atmosphere, extracellular bacteria were removed by washing with PBS and APCs were either immediately fixed with 1% paraformaldehyde (1 hour) or incubated further in media containing 100 μg/ml gentamicin. At one hour intervals, APCs were fixed with 1% paraformaldehyde until all time points were completed. Following completion of the four hour time interval, APCs were washed with PBS and 1×10<sup>5 </sup>B3Z T cells added to each well. Presentation of SL8/K<sup>b </sup>was assayed as described above. Labels indicate the time elapsed post addition of bacteria prior to fixation of APCs. Dark shaded bars indicate samples to which <i>E. coli </i>strain DP-E3617 were added. Light shaded bar indicates samples to which no bacteria but 90 nM synthetic SL8 was added with B3Z cells. The (Not Fixed) samples received no fixation prior to the addition of B3Z cells. Data presented is from triplicate groups of wells from one of three experiments with identical results.</p>
    <p>Fixing APCs at one hour following the addition of <i>E. coli </i>expressing LLO and OVA resulted in sufficient antigen presentation to yield activation of B3Z cells to a level slightly higher than those seen in the absence of fixing (compare 1 hour fixed to Not Fixed). The increased level of SL8/K<sup>b </sup>presentation following fixation can be attributed to crosslinking of surface MHC class I molecules resulting in increased stability of peptide-MHC complexes (31). Consistent with previous studies of OVA delivery to APCs (16, 20, 21), the maximal presentation of SL8/K<sup>b </sup>complex occurred when processing of OVA was allowed to continue for two hours prior to fixation. This level of SL8/K<sup>b </sup>presentation was equivalent to that seen with the addition of 90 nM synthetic SL8 in the absence of fixation (FIG. <b>2</b>B). Additional analysis indicated that fixing APCs later than two hours after addition of bacteria, resulted in a decreased level of SL8/K<sup>b </sup>presentation (FIG. 2B, 3 and 4 hour time points). This is consistent with dissociation of surface peptide/MHC complexes prior to crosslinking MHC class I molecules by fixation (31). These data indicate that no delay in the processing and presentation of SL8/K<sup>b </sup>complex occurred when OVA was delivered to the cytosol using the <i>E. coli</i>/LLO delivery system.</p>
    <p>The results of this example demonstrate that <i>E. coli </i>expressing cytoplasmic LLO can be used to deliver a co-expressed protein to the cytosol of macrophages. The delivery of protein to macrophages was rapid and efficient with protein first appearing in the cytosol within ten minutes following bacterial uptake. Moreover, large enzymatically active proteins can be introduced into the cytosol using this method as demonstrated by the delivery of active β-gal. The mechanism of delivery may be as follows. Subsequent or concomitant to phagocytosis, the <i>E. coli </i>are killed and degraded within phagosomes causing the release of LLO and the target protein from the bacteria. LLO acts by forming large pores in the phagosomal membrane thus releasing the target protein into the cytosol. In any event, any protein that can be synthesized in <i>E. coli </i>can be delivered to the macrophage cytosol.</p>
    <p>LLO is an essential determinant of pathogenesis whose role is to mediate release of <i>L. monocytogenes </i>from a phagosome. The biological properties of LLO make it well suited for use in our system. For example, LLO has an acidic pH optimum which facilitates its action in a phagosome (40, 41). However, it was unclear whether LLO released by degraded <i>E. coli </i>would retain its biological activity. The data presented here demonstrate that the amount of LLO expressed was sufficient to allow the rapid release of protein into the cytosol. Based on SDS-PAGE analysis of known quantities of purified LLO protein, we estimate approximately 1×10<sup>5 </sup>molecules of LLO per <i>E. coli </i>cell. Using our disclosure, one can now determine how many molecules of LLO are actually needed to introduce a pore into a phagosome as described (42), where it was determined to take only approximately 50 molecules of streptolysin O, a homologous pore-forming cytolysin, to form a pore in red cell membranes. A second property of LLO is its relative lack of toxicity thought to be due to its proteolysis in the cytosol of host cells (43). Indeed, secretion by <i>L. monocytogenes </i>of a related pore-forming hemolysin, perfringolysin O, resulted in death of the infected cells (44). Other facultative intracellular pathogens, Shigellae, Salmonellae, and Yersiniae all induce macrophage apoptosis (14, 45), yet infection with <i>L. monocytogenes </i>is relatively benign (46). In the current study, even though we estimate each recombinant <i>E. coli </i>contained approximately 1×10<sup>5 </sup>molecules of LLO, there was no evidence of toxicity until there were about 25 bacteria/macrophage.</p>
    <p>There are a number of advantages and applications of the <i>E. coli</i>/LLO delivery system. Many methodologies for delivering protein to the cytosol of macrophages require the prior purification of the protein to be delivered. With the <i>E. coli</i>/LLO mediated delivery of protein, no protein purification is required, only expression of the target protein in <i>E. coli </i>is necessary. Furthermore, with many alternative methods, delivery is restricted to minute amounts of protein or a limited number of cells and the in vivo delivery of protein can not be achieved (2). Using the <i>E. coli</i>/LLO system, high levels of protein can be delivered to the cytosol of virtually all of the cells in culture. In addition, by expressing protein under the control of inducible promoters, the level of protein produced and ultimately delivered to the cytosol of macrophages can be controlled. This system can be used in vivo and by expressing invasive determinants from other bacterial species, the <i>E. coli </i>may be modified to enter cells other than macrophages. Furthermore, this system has applications for the delivery of pathogen-specific protein antigens or DNA.</p>
    <p>The results of this example show that the <i>E. coli</i>/LLO system is particularly effective for the introduction of protein into the MHC class I pathway of antigen processing and presentation. We were able to detect antigen presentation with less than 1 bacterium/10 macrophages and observed a maximal response with as few as 1 to 2 bacteria/macrophage (FIG. <b>1</b>). In addition, efficient processing and surface presentation of peptide/MHC complexes occurred rapidly, within 1-2 hours following addition of bacteria to macrophages (FIG. <b>2</b>B). Delivery to the MHC class I pathway was enhanced greater than 4-logs compared to <i>E. coli </i>expressing OVA alone. This is a similar level of enhancement to that reported when OVA linked to beads was compared to soluble OVA for presentation with MHC class I molecules (47). It is clear from subsequent studies that the beads, like LLO, mediated disruption of the phagosome (21, 48). However, there was one report in which <i>E. Coli </i>expressing OVA was able to deliver OVA to the MHC class I pathway (49). Here, delivery was proposed to occur by a non-conventional pathway involving extracellular peptide regurgitation of phagosomal processed antigens instead of transfer of protein from the phagosomal compartment to the cytosol. Nonetheless, our data clearly show undetectable levels of antigen presentation when <i>E. coli </i>lacking LLO yet expressing OVA to 20% of the total cellular protein were used to deliver OVA to macrophages. Perhaps the T-cells used in our studies were unable to detect presentation of SL8/K<sup>b </sup>complexes, when <i>E. coli </i>expressing OVA alone were used, because of inefficient processing and presentation via the non-conventional pathway. In the previous study, OVA was generated as fusions to Crl or LamB proteins. The efficiency of processing and presentation of epitopes from OVA has been shown to be dependent on the protein context surrounding the epitope (50, 51). Therefore, it is possible that the fusion proteins used in the previous study are processed more efficiently than the truncated OVA used in this report.</p>
    <p>Recently, an <i>E. coli </i>expression cloning strategy for the identification of CD4<sup>−</sup> T cell-stimulating antigens has been reported (30). However, this method has not been successfully used to identify CD8<sup>+</sup> CTL-stimulating antigens since proteins expressed in <i>E. coli </i>do not gain efficient access to the MHC class I pathway for antigen presentation. The results of this study indicate that the <i>E. coli</i>/LLO delivery system provides an expression cloning strategy for the identification of pathogen-specific CD8<sup>+</sup> CTL-stimulating antigens. The identification of these pathogen-specific epitopes is an important step in the rational design of vaccine strategies against these infectious agents. These antigens are further characterized to determine the peptide epitopes recognized by CTLs, as well as the natural function the antigenic protein plays in the interaction of the pathogen and host cells.</p>
    <p>The efficiency of antigen delivery provides for the <i>E. coli</i>/LLO system to be used for the induction of CTLs in vivo. The efficient in vivo delivery of antigens to generate a protective immune response is a significant challenge in vaccine development. The use of bacterial vectors that have evolved to invade and replicate in mammalian cells such as Shigella (11, 12), Salmonella (8-10), and <i>Listeria monocytogenes </i>(52-55) are being explored as methods for the delivery of both protein antigens and DNA. Although these vehicles have had success in eliciting protective immune responses, the in vivo use of pathogenic bacteria has inherent risks. One strategy to overcome these obstacles has been to engineer attenuated <i>E. coli </i>that can invade and enter the cytosol of host cells for the delivery of macromolecules. <i>E. coli </i>deficient in the production of diaminopimelate (DAP), an essential cell wall component, undergo lysis during growth in the absence of DAP (56). DAP-minus <i>E. coli </i>carrying the 200 kb virulence plasmid pWR100 from <i>Shigella flexneri </i>have been engineered to deliver DNA to mammalian cells (11). These <i>E. coli </i>have the ability to invade cultured cells and enter the cytosol similar to <i>S. flexneri</i>, yet following brief replication, spontaneously lyse in the cytosol and allow for the delivery of DNA for subsequent expression in the host cell. However, the presence of the pWR100 virulence plasmid poses limitations on the suitability of this microbe for many applications. The rational design of safe delivery vectors is therefore of paramount importance when constructing new methodologies for in vivo delivery. Since the <i>E. coli</i>/LLO system does not contain any virulence associated determinants other than LLO, it is uniquely situated to safely deliver antigens to macrophages in vivo to generate a protective immune response.</p>
    <p>Numerical References</p>
    <p>1. Wong, T. K. &amp; Neumann, E. (1982) <i>Biochem. Biophys. Res. Commun. </i>107, 584-587.</p>
    <p>2. Celis, J. E., ed. (1998) in <i>Cell Biology: a laboratory handbook, </i>2nd. ed., Vol. 4. (Acad Press, San Diego, Calif. ).</p>
    <p>3. Rozengurt, E., et al. (1975) <i>Biochem. Biophys. Res. Commun. </i>67, 1581-1588.</p>
    <p>4. Grant, N. J., Aunis, D., &amp; Bader, M. F. (1987) <i>Neuroscience </i>23, 1143-1155.</p>
    <p>5. Vile, R. G., Tuszynski, A., &amp; Castleden, S. (1996) <i>Mol. Biotechnol </i>5, 139-158.</p>
    <p>6. Adams, S. E., &amp; Kingsman, A. J. (1995) <i>Pharm. Biotechnol </i>6, 769-786.</p>
    <p>7. Dachs, G. U., et al. (1997) <i>Oncol. Res. </i>9, 313-325.</p>
    <p>8. Darji, A., et al. (1997) <i>Cell </i>91, 765-775.</p>
    <p>9. Gentschev, I., et al. (1996) <i>Gene </i>179, 133-140.</p>
    <p>10. Hess, J., et al. (1996) <i>Proc. Natl. Acad. Sci. U.S.A. </i>93, 1458-1463.</p>
    <p>11. Courvalin, P., et al. (1995) <i>C. R. Acad. Sci. III </i>318, 1207-1212.</p>
    <p>12. Sizemore, D. R., Branstrom, A. A., &amp; Sadoff, J. C. (1995) <i>Science </i>270, 299-302.</p>
    <p>13. Kaufmann, S. H. (1993) <i>Annu. Rev. Immunol. </i>11, 129-163.</p>
    <p>14. Finlay, B. B., &amp; Cossart, P. (1997) <i>Science </i>276, 718-725.</p>
    <p>15. Portnoy, D. A., et al. (1992) <i>Infect. Immun. </i>60, 1263-1267.</p>
    <p>16. Lee, K. D., et al. (1996) <i>J. Biol. Chem. </i>271, 7249-7252.</p>
    <p>17. Darji, A., et al. (1997) <i>Eur. J. Immunol. </i>27, 1353-1359.</p>
    <p>18. Darji, A., et al. (1995) <i>Eur. J. Immunol. </i>25, 2967-2971.</p>
    <p>19. Mazzaccaro, R. J., et al. (1996) <i>Proc. Natl. Acad. Sci. U.S.A. </i>93, 11786-11791.</p>
    <p>20. Zhou, Watkins, S. C., &amp; Huang, L. (1994) <i>Immunobiology </i>190,35-52.</p>
    <p>21. Oh, Y. K., Harding, C. V., &amp; Swanson, J. A. (1997) <i>Vaccine </i>15, 511-518.</p>
    <p>22. Darji, A., et al. (1995) <i>J. Biotechnol. </i>43, 205-212.</p>
    <p>23. Chang, A. C. Y., &amp; Cohen, S. N. (1978) <i>J. Bacteriol. </i>134, 1141-1156.</p>
    <p>24. Linn, T., &amp; St. Pierre, R. (1990) <i>J. Bacteriol. </i>172, 1077-1084.</p>
    <p>25. Sambrook, J., et al. (1989) in <i>Molecular cloning: a laboratory manual, </i>eds. Ford, N., Nolan, C. &amp; Ferguson, M. (Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y.)</p>
    <p>26. Portnoy, D. A., Jacks, P. S., &amp; Hinrichs, D. J. (1988) <i>J. Exp. Med. </i>167, 1459-1471.</p>
    <p>27. Sanderson, S., &amp; Shastri, N. (1994) <i>Int. Immunol. </i>6, 369-376.</p>
    <p>28. Karttunen, J., et al. (1992) <i>Proc. Natl. Acad. Sci. U.S.A. </i>89, 6020-6024.</p>
    <p>29. Marquis, H., Goldfine, H., &amp; Portnoy, D. A. (1997) <i>J. Cell Biol. </i>137, 1381-1392.</p>
    <p>30. Sanderson, S., Campbell, D. J., &amp; Shastri, N. (1995)<i>J. Exp. Med. </i>182, 1751-1757.</p>
    <p>31. Rock, K. L., et al. (1992) <i>J. Immunol. </i>148, 1451-1457.</p>
    <p>32. Takahashi, N., Orita, T., &amp; Hirose, M. (1995) <i>Gene </i>161, 211-216.</p>
    <p>33. Jacobson, R. H., et al. (1994) <i>Nature </i>369, 761-766.</p>
    <p>34. Kagi, D., et al. (1996) <i>Annu. Rev. Immunol. </i>14, 207-232.</p>
    <p>35. Singh, N., Agrawal, S., &amp; Rastogi, A. K. (1997) <i>Emerg. Infect. Dis. </i>3, 41-49.</p>
    <p>36. Harding, C. V. (1996) <i>J. Clin. Immunol. </i>16, 90-96.</p>
    <p>37. York, I. A., &amp; Rock, K. L. (1996) <i>Annu. Rev. Immunol. </i>14, 369-396.</p>
    <p>38. Craiu, A., et al. (1997) <i>Proc. Natl. Acad. Sci. U.S.A. </i>94, 10850-10855.</p>
    <p>39. Rock, K. L., et al. (1994) <i>Cell </i>78, 761-771.</p>
    <p>40. Geoffroy, C., et al. (1987) <i>Infect. Immun. </i>55, 1641-1646.</p>
    <p>41. Beauregard, K. E., et al. (1997) <i>J. Exp. Med. </i>186, 1159-1163.</p>
    <p>42. Sekiya, K., et al. (1993) <i>J. Bacteriol. </i>175, 5953-5961.</p>
    <p>43. Villanueva, M. S., Sijts, A. J., &amp; Pamer, E. G. (1995) <i>J. Immunol. </i>155, 5227-5233.</p>
    <p>44. Jones, S., &amp; Portnoy, D. A. (1994) <i>Infect. Immun. </i>62, 5608-5613.</p>
    <p>45. Ruckdeschel, K., et al. (1997) <i>Infect. Immun. </i>65, 4813-4821.</p>
    <p>46. Barsig, J., &amp; Kaufmann, S. H. (1997) <i>Infect. Immun. </i>65, 4075-4081.</p>
    <p>47. Kovacsovics-Bankowski, M., et al. (1993) <i>Proc. Natl. Acad. Sci. USA </i>90, 4942-4946.</p>
    <p>48. Kovacsovics-Bankowski, M., &amp; Rock, K. L. (1995) <i>Science </i>267, 243-246.</p>
    <p>49. Pfeifer, J. D., et al. (1993) <i>Nature </i>361, 359-362.</p>
    <p>50. Svensson, M., et al. (1997) <i>Behring Inst. Mitt. </i>98, 197-211.</p>
    <p>51. Shastri, N., Serwold, T., &amp; Gonzalez, F. (1995) <i>J. Immunol. </i>155, 4339-4346.</p>
    <p>52. Frankel, F. R., et al. (1995) <i>J. Immunol. </i>155, 4775-4782.</p>
    <p>53. Ikonomidis, G., et al. (1997) <i>Vaccine </i>15, 433-440.</p>
    <p>54. Shen, H., et al. (1995) <i>Proc. Natl. Acad. Sci. U.S.A. </i>92, 3987-3991.</p>
    <p>55. Pan, Z. K., et al. (1995) <i>Nat. Med. </i>1, 471-477.</p>
    <p>56. Patte, J. C. (1996) in <i>Biosynthesis of Threonine and Lysine</i>, eds. Neidhardt, F. C., et al. (ASM Press, Washington, D.C.), pp. 528-541.</p>
    <p>All publications and patent applications cited in this specification are herein incorporated by reference as if each individual publication or patent application were specifically and individually indicated to be incorporated by reference. Although the foregoing invention has been described in some detail by way of illustration and example for purposes of clarity of understanding, it will be readily apparent to those of ordinary skill in the art in light of the teachings of this invention that certain changes and modifications may be made thereto without departing from the spirit or scope of the appended claims.</p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" pgwide="1" tabstyle="monospace" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="441pt" align="left"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">                  </td>
              </tr> <tr class="description-tr"> <td class="description-td">#             SEQUENCE LISTING </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;160&gt; NUMBER OF SEQ ID NOS: 6 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;210&gt; SEQ ID NO 1 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;211&gt; LENGTH: 34 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;212&gt; TYPE: DNA </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;213&gt; ORGANISM: Listeria monocytogenes </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;400&gt; SEQUENCE: 1 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">ggaattccat atgaaggatg catctgcatt caat       </td>
              </tr> <tr class="description-tr"> <td class="description-td">#                  </td>
              </tr> <tr class="description-tr"> <td class="description-td">#        34 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;210&gt; SEQ ID NO 2 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;211&gt; LENGTH: 33 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;212&gt; TYPE: DNA </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;213&gt; ORGANISM: Listeria monocytogenes </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;400&gt; SEQUENCE: 2 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">cgggatcctt attattcgat tggattatct act       </td>
              </tr> <tr class="description-tr"> <td class="description-td">#                  </td>
              </tr> <tr class="description-tr"> <td class="description-td">#         33 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;210&gt; SEQ ID NO 3 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;211&gt; LENGTH: 26 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;212&gt; TYPE: DNA </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;213&gt; ORGANISM: Listeria monocytogenes </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;400&gt; SEQUENCE: 3 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">cgcgatatcc tctagaaata attttg          </td>
              </tr> <tr class="description-tr"> <td class="description-td">#                  </td>
              </tr> <tr class="description-tr"> <td class="description-td">#              26 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;210&gt; SEQ ID NO 4 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;211&gt; LENGTH: 32 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;212&gt; TYPE: DNA </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;213&gt; ORGANISM: Listeria monocytogenes </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;400&gt; SEQUENCE: 4 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">aggcgtcgac ggttaatacg accgggatcg ag       </td>
              </tr> <tr class="description-tr"> <td class="description-td">#                  </td>
              </tr> <tr class="description-tr"> <td class="description-td">#          32 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;210&gt; SEQ ID NO 5 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;211&gt; LENGTH: 40 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;212&gt; TYPE: DNA </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;213&gt; ORGANISM: Listeria monocytogenes </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;400&gt; SEQUENCE: 5 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">aggcgtcgac aggccttacg cgaaatacgg gcagacatgg     </td>
              </tr> <tr class="description-tr"> <td class="description-td">#                  </td>
              </tr> <tr class="description-tr"> <td class="description-td">#    40 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;210&gt; SEQ ID NO 6 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;211&gt; LENGTH: 8 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;212&gt; TYPE: PRT </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;213&gt; ORGANISM: murine </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;400&gt; SEQUENCE: 6 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Ser Ile Ile Asn Phe Glu Lys Leu </td>
              </tr> <tr class="description-tr"> <td class="description-td"> 1               5</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    </div></div></div><div class="patent-section patent-tabular-section"><a id="backward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Cited Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4888170">US4888170</a></td><td class="patent-data-table-td patent-date-value">Sep 9, 1985</td><td class="patent-data-table-td patent-date-value">Dec 19, 1989</td><td class="patent-data-table-td ">Research Corporation</td><td class="patent-data-table-td ">Vaccines obtained from antigenic gene products of recombinant genes</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5294441">US5294441</a></td><td class="patent-data-table-td patent-date-value">Nov 7, 1991</td><td class="patent-data-table-td patent-date-value">Mar 15, 1994</td><td class="patent-data-table-td ">Washington University</td><td class="patent-data-table-td ">Immunization; obtained from a pathogenic S. typhi having the cya and crp genes deactivated by genetic mutation</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5387744">US5387744</a></td><td class="patent-data-table-td patent-date-value">Jul 7, 1993</td><td class="patent-data-table-td patent-date-value">Feb 7, 1995</td><td class="patent-data-table-td ">Washington University</td><td class="patent-data-table-td ">Avirulent microbes and uses therefor: Salmonella typhi</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5424065">US5424065</a></td><td class="patent-data-table-td patent-date-value">Nov 19, 1992</td><td class="patent-data-table-td patent-date-value">Jun 13, 1995</td><td class="patent-data-table-td ">Washington University</td><td class="patent-data-table-td ">Comprising mutant Salmonella</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5824538">US5824538</a></td><td class="patent-data-table-td patent-date-value">Sep 6, 1995</td><td class="patent-data-table-td patent-date-value">Oct 20, 1998</td><td class="patent-data-table-td ">The United States Of America As Represented By The Secretary Of The Army</td><td class="patent-data-table-td ">Shigella vector for delivering DNA to a mammalian cell</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5855879">US5855879</a></td><td class="patent-data-table-td patent-date-value">Mar 10, 1994</td><td class="patent-data-table-td patent-date-value">Jan 5, 1999</td><td class="patent-data-table-td ">Washington University</td><td class="patent-data-table-td ">Hybrid gramnegative bacteria</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5855880">US5855880</a></td><td class="patent-data-table-td patent-date-value">Feb 5, 1996</td><td class="patent-data-table-td patent-date-value">Jan 5, 1999</td><td class="patent-data-table-td ">Washington University</td><td class="patent-data-table-td ">Avirulent microbes and uses therefor</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5877159">US5877159</a></td><td class="patent-data-table-td patent-date-value">May 3, 1995</td><td class="patent-data-table-td patent-date-value">Mar 2, 1999</td><td class="patent-data-table-td ">University Of Maryland At Baltimore</td><td class="patent-data-table-td ">The gene may encode, e.g., a vaccine antigen, an therapeutic agent, an immunoregulatory agent or a anti-sense rna or a catalytic rna.</td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="npl-citations"></a><div class="patent-section-header"><span class="patent-section-title">Non-Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th colspan="3"class="patent-data-table-th">Reference</th></tr></thead><tr><td class="patent-data-table-td ">1</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Anderson, Nature 392:25-30 Feb. 1998.</td></tr><tr><td class="patent-data-table-td ">2</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Branch, TIBS 23:45-50 Feb. 1998.</td></tr><tr><td class="patent-data-table-td ">3</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Darji, et al., Cell 91:765-75 1997.</td></tr><tr><td class="patent-data-table-td ">4</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Dietrich G. et al. "<a href='http://scholar.google.com/scholar?q="Delivery+of+antigen-encoding+plasmid+DNA+into+the+cytosol+of+macrophages+by+attenuated+Listeria+monocytogenes"'>Delivery of antigen-encoding plasmid DNA into the cytosol of macrophages by attenuated Listeria monocytogenes</a>" Nature Biotechnology vol. 16, Feb. 1998, pp. 181-185.</td></tr><tr><td class="patent-data-table-td ">5</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">GenBank Accession U02451.*</td></tr><tr><td class="patent-data-table-td ">6</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Higgins et al, Molecular Microbiology 31(6):1631-41 1999.</td></tr><tr><td class="patent-data-table-td ">7</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Higgins et al., Nature Biotechnology 16(2):138-9 1998.</td></tr><tr><td class="patent-data-table-td ">8</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Ikonomidis et al., Journal of Experimental Medicine 180(6):2209-18 1994.</td></tr><tr><td class="patent-data-table-td ">9</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Y Gholizadeh et al., Journal of Clinical Microbiology, "<a href='http://scholar.google.com/scholar?q="Serodiagnosis+of+Listeriosis+Based+upon+Detection+of+Antibodies+against+Recombinant+Truncated+Forms+of+Listeriolysin+O%2C"'>Serodiagnosis of Listeriosis Based upon Detection of Antibodies against Recombinant Truncated Forms of Listeriolysin O,</a>" Jun. 1996, vol. 34, No. 6, pp. 1391-1395.*</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="forward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Referenced by</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Citing Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7794728">US7794728</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">May 29, 2003</td><td class="patent-data-table-td patent-date-value">Sep 14, 2010</td><td class="patent-data-table-td ">The Regents Of The University Of California</td><td class="patent-data-table-td ">Induction immunology response; vaccines</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7842289">US7842289</a></td><td class="patent-data-table-td patent-date-value">Dec 23, 2004</td><td class="patent-data-table-td patent-date-value">Nov 30, 2010</td><td class="patent-data-table-td ">Aduro Biotech</td><td class="patent-data-table-td ">Polypeptides; antigens; vaccines; fusion protein containing signal peptide and polypeptide; genetic engineered Listeria bacteria</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8277797">US8277797</a></td><td class="patent-data-table-td patent-date-value">Nov 21, 2007</td><td class="patent-data-table-td patent-date-value">Oct 2, 2012</td><td class="patent-data-table-td ">The Regents Of The University Of California</td><td class="patent-data-table-td ">Interferon-β production modulating Listeria strains and methods for using same</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8679476">US8679476</a></td><td class="patent-data-table-td patent-date-value">Aug 31, 2012</td><td class="patent-data-table-td patent-date-value">Mar 25, 2014</td><td class="patent-data-table-td ">The Regents Of The University Of California</td><td class="patent-data-table-td ">Interferon-β production modulating Listeria strains and methods for using same</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="classifications"></a><div class="patent-section-header"><span class="patent-section-title">Classifications</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th"> </th><th class="patent-data-table-th"> </th></tr></thead><tr><td class="patent-data-table-td ">U.S. Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=SBhhBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc424/defs424.htm&usg=AFQjCNH9_TEmt6LWxUKihruum311Bzz0Rg#C424S093100">424/93.1</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=SBhhBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc424/defs424.htm&usg=AFQjCNH9_TEmt6LWxUKihruum311Bzz0Rg#C424S093200">424/93.2</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=SBhhBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S252200">435/252.2</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=SBhhBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc424/defs424.htm&usg=AFQjCNH9_TEmt6LWxUKihruum311Bzz0Rg#C424S200100">424/200.1</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=SBhhBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S252100">435/252.1</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=SBhhBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc424/defs424.htm&usg=AFQjCNH9_TEmt6LWxUKihruum311Bzz0Rg#C424S184100">424/184.1</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=SBhhBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc424/defs424.htm&usg=AFQjCNH9_TEmt6LWxUKihruum311Bzz0Rg#C424S093400">424/93.4</a></span></td></tr><tr><td class="patent-data-table-td ">International Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=SBhhBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0001210000">C12N1/21</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=SBhhBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=A61P0035000000">A61P35/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=SBhhBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=A61P0037000000">A61P37/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=SBhhBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=A61K0039000000">A61K39/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=SBhhBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0015630000">C12N15/63</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=SBhhBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0015870000">C12N15/87</a></span></td></tr><tr><td class="patent-data-table-td ">Cooperative Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=SBhhBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N15/87">C12N15/87</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=SBhhBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=A61K39/00">A61K39/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=SBhhBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N15/63">C12N15/63</a></span></td></tr><tr><td class="patent-data-table-td ">European Classification</td><td class="patent-data-table-td "><span class="nested-value">C12N15/63</span>, <span class="nested-value">C12N15/87</span></td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="legal-events"></a><div class="patent-section-header"><span class="patent-section-title">Legal Events</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Date</th><th class="patent-data-table-th">Code</th><th class="patent-data-table-th">Event</th><th class="patent-data-table-th">Description</th></tr></thead><tr><td class="patent-data-table-td patent-date-value">Jan 31, 2011</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">8</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Jul 28, 2009</td><td class="patent-data-table-td ">CC</td><td class="patent-data-table-td ">Certificate of correction</td><td class="patent-data-table-td "></td></tr><tr><td class="patent-data-table-td patent-date-value">Oct 7, 2008</td><td class="patent-data-table-td ">B1</td><td class="patent-data-table-td ">Reexamination certificate first reexamination</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">CLAIM 1 IS DETERMINED TO BE PATENTABLE AS AMENDED. CLAIMS 2-18, DEPENDENT ON AN AMENDED CLAIM, ARE DETERMINED TO BE PATENTABLE.</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Feb 5, 2008</td><td class="patent-data-table-td ">RR</td><td class="patent-data-table-td ">Request for reexamination filed</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20071107</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Jan 29, 2007</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">4</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Sep 7, 2001</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">REGENTS OF THE UNIVERSITY OF CALIFORNIA, THE, CALI</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNORS:PORTNOY, DANIEL A.;HIGGINS, DARREN E.;REEL/FRAME:012160/0955</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">19980812</span></div><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">REGENTS OF THE UNIVERSITY OF CALIFORNIA, THE 1111</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNORS:PORTNOY, DANIEL A. /AR;REEL/FRAME:012160/0955</span></div></td></tr></table><div class="patent-section-footer"></div></div><div class="modal-dialog" id="patent-images-lightbox"><div class="patent-lightbox-controls"><div class="patent-lightbox-rotate-controls"><div class="patent-lightbox-rotation-text">Rotate</div><div class="rotate-icon rotate-ccw-icon"></div><div class="rotate-icon rotate-cw-icon"></div></div><div class="patent-lightbox-index-counter"></div><a class="patent-lightbox-fullsize-link" target="_blank">Original Image</a><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_right.png" alt="Next page"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_left.png" alt="Previous page"width="21" height="21" /></div></div><div class="modal-dialog-content"><div class="patent-lightbox-image-holder"><div class="patent-lightbox-placeholder"></div></div></div></div><script>_OC_initPatentsAtb({image_not_available_html: " Image not available"});</script></div></div></div></td></tr></table><script>(function() {var href = window.location.href;if (href.indexOf('?') !== -1) {var parameters = href.split('?')[1].split('&');for (var i = 0; i < parameters.length; i++) {var param = parameters[i].split('=');if (param[0] == 'focus') {var elem = document.getElementById(param[1]);if (elem) {elem.focus();}}}}})();</script><script>_OC_addFlags({LockSrc:"/books/javascript/lock_8a2b04e7bf975d5171d8e4c0b6365c7a.js", Host:"http://www.google.com/", IsBooksRentalEnabled:1, IsWebstoreDisplayCaseEnabled:1, IsObfuscationEnabled:1, IsBrowsingHistoryEnabled:1, IsWebReaderSvgEnabled:0, IsGeoLayerEnabled:1, IsImageModeNotesEnabled:1, IsCopyMenuItemEnabled:1, IsGiftingEnabled:0, IsWebReaderUniversalPaginatorEnabled:0, IsOfflineBubbleEnabled:1, IsReaderEnabledForPlayRequests:1, IsFutureOnSaleVolumesEnabled:1, IsOfflineRestrictedCopyEnabled:1, IsBooksUnifiedLeftNavEnabled:1, IsRestrictedCopyEnabled:1, IsZipitFolderCollectionEnabled:1, IsEndOfSampleRecommendationsEnabled:1, IsRatingsOnBookcardsEnabled:1, IsAdsDisabled:0, IsIframePageDisplayEnabled:0, IsEmbeddedMediaEnabled:1, IsImageModeAnnotationsEnabled:1, IsMyLibraryGooglePlusEnabled:1, IsImagePageProviderEnabled:0, IsBookcardListPriceSmall:0, IsInternalUser:0, IsBooksShareButtonEnabled:0, IsPreOrdersEnabled:0, IsDisabledRandomBookshelves:0, WebstoreDisplayCasePosition:3});_OC_Run({"enable_p13n":false,"add_vol_to_collection_base_url":"http://www.google.com/patents?op=add\u0026sig=ACfU3U2l7dG9Otix1AXIkwFQWvzxpUMqsQ\u0026id=SBhhBAABERAJ","remove_vol_from_collection_base_url":"http://www.google.com/patents?op=remove\u0026sig=ACfU3U3wCNbjjgWbahUysGEuzE1vk4l-NQ\u0026id=SBhhBAABERAJ","logged_in":false,"p13n_save_user_settings_url":"http://www.google.com/patents?op=edit_user_settings\u0026sig=ACfU3U2JZsLiNDSlR9kJkx3cE0awUFYagw","is_cobrand":false,"sign_in_url":"https://www.google.com/accounts/Login?service=\u0026continue=http://www.google.com/patents%3Fhl%3Den\u0026hl=en","is_play_enabled":true}, {"volume_id":"","is_ebook":true,"volumeresult":{"has_flowing_text":false,"has_scanned_text":true,"can_download_pdf":false,"can_download_epub":false,"is_pdf_drm_enabled":false,"is_epub_drm_enabled":false,"download_pdf_url":"http://www.google.com/patents/download/Intracellular_delivery_vehicles.pdf?id=SBhhBAABERAJ\u0026output=pdf\u0026sig=ACfU3U2aZiOTsrHSVKyuAAD5CQDtP9F_Og"},"sample_url":"http://www.google.com/patents/reader?id=SBhhBAABERAJ\u0026printsec=frontcover\u0026output=reader\u0026source=gbs_atb_hover","is_browsable":true,"is_public_domain":true}, {});</script><div id="footer_table" style="font-size:83%;text-align:center;position:relative;top:20px;height:4.5em;margin-top:2em"><div style="margin-bottom:8px"><a href=http://www.google.com/><nobr>Google&nbsp;Home</nobr></a> - <a href=//www.google.com/patents/sitemap/><nobr>Sitemap</nobr></a> - <a href=http://www.google.com/googlebooks/uspto.html><nobr>USPTO Bulk Downloads</nobr></a> - <a href=/intl/en/privacy/><nobr>Privacy Policy</nobr></a> - <a href=/intl/en/policies/terms/><nobr>Terms of Service</nobr></a> - <a href=https://support.google.com/faqs/answer/2539193?hl=en><nobr>About Google Patents</nobr></a> - <a href="http://www.google.com/tools/feedback/intl/en/error.html" onclick="try{_OC_startFeedback({productId: '72792',locale: 'en'});return false;}catch(e){}"><nobr>Send Feedback</nobr></a></div><span>Data provided by IFI CLAIMS Patent Services</span><br><span >&copy;2012 Google</span></div> <script type="text/javascript">var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));</script><script type="text/javascript">var pageTracker = _gat._getTracker("UA-27188110-1");pageTracker._setCookiePath("/patents/");pageTracker._trackPageview();</script> </body></html>