<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN""http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html><head><title>Patent US6617110 - Cells immortalized with telomerase reverse transcriptase for use in drug ... - Google Patents</title><script>(function(){(function(){function e(a){this.t={};this.tick=function(a,c,b){var d=void 0!=b?b:(new Date).getTime();this.t[a]=[d,c];if(void 0==b)try{window.console.timeStamp("CSI/"+a)}catch(e){}};this.tick("start",null,a)}var a;window.performance&&(a=window.performance.timing);var f=a?new e(a.responseStart):new e;window.jstiming={Timer:e,load:f};if(a){var c=a.navigationStart,d=a.responseStart;0<c&&d>=c&&(window.jstiming.srt=d-c)}if(a){var b=window.jstiming.load;0<c&&d>=c&&(b.tick("_wtsrt",void 0,c),b.tick("wtsrt_",
"_wtsrt",d),b.tick("tbsd_","wtsrt_"))}try{a=null,window.chrome&&window.chrome.csi&&(a=Math.floor(window.chrome.csi().pageT),b&&0<c&&(b.tick("_tbnd",void 0,window.chrome.csi().startE),b.tick("tbnd_","_tbnd",c))),null==a&&window.gtbExternal&&(a=window.gtbExternal.pageT()),null==a&&window.external&&(a=window.external.pageT,b&&0<c&&(b.tick("_tbnd",void 0,window.external.startE),b.tick("tbnd_","_tbnd",c))),a&&(window.jstiming.pt=a)}catch(g){}})();})();
</script><link rel="stylesheet" href="/patents/css/_8a2b04e7bf975d5171d8e4c0b6365c7a/kl_intl_patents_bundle.css" type="text/css" /><script src="/books/javascript/atb_8a2b04e7bf975d5171d8e4c0b6365c7a__en.js"></script><script>function googleTranslateElementInit() {new google.translate.TranslateElement({pageLanguage: "en",gaTrack: true,gaId: "UA-27188110-1",multilanguagePage: true});}</script><script src="//translate.google.com/translate_a/element.js?cb=googleTranslateElementInit"></script><meta name="DC.type" content="Patent"><meta name="DC.title" content="Cells immortalized with telomerase reverse transcriptase for use in drug screening"><meta name="DC.contributor" content="Thomas R. Cech" scheme="inventor"><meta name="DC.contributor" content="Joachim Lingner" scheme="inventor"><meta name="DC.contributor" content="Toru Nakamura" scheme="inventor"><meta name="DC.contributor" content="Karen B. Chapman" scheme="inventor"><meta name="DC.contributor" content="Gregg B. Morin" scheme="inventor"><meta name="DC.contributor" content="Calvin B. Harley" scheme="inventor"><meta name="DC.contributor" content="William H. Andrews" scheme="inventor"><meta name="DC.contributor" content="Geron Corporation" scheme="assignee"><meta name="DC.contributor" content="University Technology Corporation" scheme="assignee"><meta name="DC.date" content="2000-11-22" scheme="dateSubmitted"><meta name="DC.description" content="The invention provides compositions and methods related to human telomerase reverse transcriptase (hTRT), the catalytic protein subunit of human telomerase. The polynucleotides and polypeptides of the invention are useful for diagnosis, prognosis and treatment of human diseases, for changing the proliferative capacity of cells and organisms, and for identification and screening of compounds and treatments useful for treatment of diseases such as cancers."><meta name="DC.date" content="2003-9-9" scheme="issued"><meta name="DC.relation" content="JP:H09154575" scheme="references"><meta name="DC.relation" content="US:3817837" scheme="references"><meta name="DC.relation" content="US:3850752" scheme="references"><meta name="DC.relation" content="US:3939350" scheme="references"><meta name="DC.relation" content="US:3996345" scheme="references"><meta name="DC.relation" content="US:4275149" scheme="references"><meta name="DC.relation" content="US:4277437" scheme="references"><meta name="DC.relation" content="US:4366241" scheme="references"><meta name="DC.relation" content="US:4683195" scheme="references"><meta name="DC.relation" content="US:4683202" scheme="references"><meta name="DC.relation" content="US:4816567" scheme="references"><meta name="DC.relation" content="US:4965188" scheme="references"><meta name="DC.relation" content="US:5489508" scheme="references"><meta name="DC.relation" content="US:5583016" scheme="references"><meta name="DC.relation" content="US:5747317" scheme="references"><meta name="DC.relation" content="US:5770422" scheme="references"><meta name="DC.relation" content="US:6093809" scheme="references"><meta name="DC.relation" content="US:6258535" scheme="references"><meta name="DC.relation" content="US:6261556" scheme="references"><meta name="DC.relation" content="US:6261836" scheme="references"><meta name="DC.relation" content="WO:1984003584:A1" scheme="references"><meta name="DC.relation" content="WO:1995013382:A1" scheme="references"><meta name="DC.relation" content="WO:1996040868:A1" scheme="references"><meta name="DC.relation" content="WO:1998001542:A1" scheme="references"><meta name="DC.relation" content="WO:1998001543:A1" scheme="references"><meta name="DC.relation" content="WO:1998001835:A1" scheme="references"><meta name="DC.relation" content="WO:1998007838:A1" scheme="references"><meta name="DC.relation" content="WO:1998008938:A1" scheme="references"><meta name="DC.relation" content="WO:1998012811:A1" scheme="references"><meta name="DC.relation" content="WO:1998019580:A1" scheme="references"><meta name="DC.relation" content="WO:1998021343:A1" scheme="references"><meta name="DC.relation" content="WO:1998037181:A2" scheme="references"><meta name="DC.relation" content="WO:1998045450:A1" scheme="references"><meta name="DC.relation" content="WO:1998059040:A2" scheme="references"><meta name="DC.relation" content="WO:1999001560:A1" scheme="references"><meta name="citation_reference" content="1994 Genome Issue of Science (265:1981f)."><meta name="citation_reference" content="Anderson and Young, &quot;Quantitative Filter Hybridization&quot; in Nucleic Acid Hybridization pp. 73-111 (1985)."><meta name="citation_reference" content="Ausubel et al., Current Protocols In Molecular Biology, John Wiley &amp; Sons, New York NY (1989)."><meta name="citation_reference" content="Autexier et al., &quot;Reconstitution of human telomerase activity and Identification of a minimal functional region of the human telomease RNA,&quot; (1998) EMBO J, 15:5928."><meta name="citation_reference" content="Auxexier and Grelder, &quot;Functional reconstitution of wild-type and mutant Tetrahymena telomerase,&quot; (1994) Genes Develop., 8:583."><meta name="citation_reference" content="Berger and Kimmel, Guide to Molecualar Cloning Techniques, Meth. Enzymol., vol. 152, Academic Press, San Diego CA (1987)."><meta name="citation_reference" content="Biessman et al., &quot;Addition of Telomere-Associated HeT DNA Sequences &quot;Heals&quot; Broken Chromosome End is in Drosophila,&quot; (1990) Cell, 61:683."><meta name="citation_reference" content="Bitter et al., &quot;Expression and secretion vectors for yeast,&quot; Meth Enzymol., (1987) 153:516."><meta name="citation_reference" content="Blackburn and Chlou, &quot;Non-nucleosomal packaging of a tandemly repeated DNA sequence at termini of extrachromosomal DNA coding for rRNA in Tetrahymena,&quot; (1981) Proc. Natl. Acad. Sci., 78:2263."><meta name="citation_reference" content="Blackburn and Gall, &quot;A tandemly repeated sequence at the termini of the extrachromosomal ribosomal RNA genes In Tetrahymena,&quot; (1978) J. Mol. Biol., 120:33."><meta name="citation_reference" content="Blackburn, &quot;Telomerases,&quot; (1992) Ann. Rev. Biochem., 61:113."><meta name="citation_reference" content="Bodnar et al., &quot;Extension of Life-Span by Introduction of Telomerase into Normal Human Cells,&quot; (1998) Science, 279:349."><meta name="citation_reference" content="Bradford, &quot;A Rapid and Sensitive method for the Quantitation of Microgram Quantities of Protein Utilizing the Principle of Protein-Dye Binding,&quot; (1976) Anal. Biochem., 72:248."><meta name="citation_reference" content="Braunstein et al., &quot;Transcriptional silencing in yeast is associated with reduced nuclcosome acetylation,&quot; (1993) Genes Develop., 7:592."><meta name="citation_reference" content="Calvio et al., &quot;Identification of hnRNP P2 as TLS/FUS using electrospray mass spectromotry,&quot; (1995) RNA, 1:724."><meta name="citation_reference" content="Caruthers et al., &quot;New chemical methods for synthesizing polynucleotides,&quot; (1980) Nucleic Acids Res. Symp. Sor., 215-223."><meta name="citation_reference" content="Chan and Tye, &quot;Organization of DNA sequences and replication origins at yeast telomeres,&quot; (1983) Cell, 33:663."><meta name="citation_reference" content="Colbere-Gerapin et al., &quot;A new dominat hybrid selective marker for higher eukaryotic cells,&quot; (1981) J. Mol. Biol., 150:1."><meta name="citation_reference" content="Cole et al., &quot;Generation of human monoclonal antibodies reactive with cellular antigens,&quot; (1983) Proc. Natl. Acad. Sci., 80:2026."><meta name="citation_reference" content="Cole et al., &quot;The EBV-hybridoma technique and its application to human lung cancer,&quot; Monoclonal Antibodies and Cancer Therapy, Alan R. Liss Inc., New York NY 77-96 (1985)."><meta name="citation_reference" content="Collins et al., &quot;Purification of Tetrahymena telomerase and cloning of genes encoding the two protein components of the enzyme,&quot; (1995) Cell, 61:677."><meta name="citation_reference" content="Conrad et al., &quot;RAP1 protein interacts with yeast telomers In vivo: Overproduction allers telomers structure and decreases chromosome stability,&quot; (1990) Cell, 63-739."><meta name="citation_reference" content="Coombs, Dictionary of Biotechnology, Stockton Press, New York NY (1994)."><meta name="citation_reference" content="Counter et al., &quot;The calalytic subunit of yeast telomerase,&quot; (1997) Proc. Natl. Acad. Sci., 94:9202."><meta name="citation_reference" content="Counter et al., (1994) Proc. Natl. Acad. Sci., 91:2900."><meta name="citation_reference" content="Creighton, Proteins, Structures and Molecular Principles, WH Freeman and Co, New York, NY (1983)."><meta name="citation_reference" content="Dieffenbach and Dvekaler, PCR Primer, a Laboratory Manual, Cold Spring Harbor Press, Plainview NY (1995)."><meta name="citation_reference" content="Duplaa et al., &quot;Quantitative analysis of polymerase chain reaction products using biotinylated dUTP incorporation,&quot; (1993) Anal. Biochem., 212:229."><meta name="citation_reference" content="Fang et al., &quot;Oxytricha telomere-binding protein: separable DNA-binding and dimerization domains of the alpha-subunit,&quot; Genes Develop. 7:870 (1993) and Gray et al., (1991) Coli 67:807."><meta name="citation_reference" content="Fang et al., &quot;The RNA Componant of Human Telomerase,&quot; (1995) Science, 269:1236."><meta name="citation_reference" content="Fang et al., “Oxytricha telomere-binding protein: separable DNA-binding and dimerization domains of the α-subunit,” Genes Develop. 7:870 (1993) and Gray et al., (1991) Coli 67:807."><meta name="citation_reference" content="GenBank Accession No. AA281296 (Apr. 1997)."><meta name="citation_reference" content="GenBank accession No. AA299878 (Apr. 1997)."><meta name="citation_reference" content="Genbank accession No. AA311750 (Apr. 1997)."><meta name="citation_reference" content="Gilley et al., &quot;Altering specific telomerase RNA template residues affects active site function,&quot; (1995) Genes Develop., 9:2214."><meta name="citation_reference" content="Gottachling and Zaldan, &quot;Telomere proteins: specific recognition end protection of the natural termini of Oxytricha macronuclear DNA,&quot; (1986) Cell 47:195."><meta name="citation_reference" content="Gottschling and Cech, &quot;Chromatin Structure of the Molecular Ends of Oxytricha Mononuclear DNA: Phased Nucleosomes and a Telomeric Complex,&quot; (1984) Cell, 98:501."><meta name="citation_reference" content="Grant et al., Meth Enzymol., (1987) 153:516-544."><meta name="citation_reference" content="Greenwood et al., &quot;Phylogenetic relationships within the class oligohymenophores, phylum cillophora, inferred from the complete small subunit rRNA gene sequences of Colpidium campylum, Glaucoma chattoni, and Opisthonecta henneguyi,&quot; (1991) J. Mol. Evol., 3:163."><meta name="citation_reference" content="Greider and Blackburn, &quot;A telomeric sequence in the RNA of Tetrahymena telomerase required for telomere repeat synthesis,&quot; (1989) Nature, 337:331."><meta name="citation_reference" content="Greider, &quot;Telomere Length Regulation,&quot; (1996) Ann. Rev. Biochem., 65:337."><meta name="citation_reference" content="Grelder and Blackburn. &quot;Identification of a specific telomere terminal transferase activity in Tetrehymena extracts,&quot; (1985) Cell, 43:405."><meta name="citation_reference" content="Grelder, &quot;Telomerase is processive,&quot; (1991) Mol. Cell. Biol., 11:4572."><meta name="citation_reference" content="Hampton et al., Serological Methods a Laboratory Manual, APS Press, St Paul MN (1990)."><meta name="citation_reference" content="Harrington et al., &quot;A Mammallan Telomerase-Associated Protein,&quot; (1997) Science, 276:973."><meta name="citation_reference" content="Harrington et al., &quot;Human telomerase contains evolutionarily conserved catalytic and structural subunits,&quot; (1997) Genes Dev., 11:3109."><meta name="citation_reference" content="Hartman and Mulligan, &quot;Two dominant-acting selectable markets for gene transfer studies in mammalian cells,&quot; (1988) Proc. Acad. Sci., 85:8047."><meta name="citation_reference" content="Henderson and Blackburn, &quot;An overhanging 3&#39;&#39; terminus is a conserved feature of telomeres,&quot; (1989) Mol Cell. Biol., 9:345."><meta name="citation_reference" content="Henderson and Blackburn, “An overhanging 3″ terminus is a conserved feature of telomeres,” (1989) Mol Cell. Biol., 9:345."><meta name="citation_reference" content="Horn et al., &quot;Synthesis of oligonucleotides on cellulose. Part II: design and synthetic strategy to the synthesis of 22 oligodeoxynucleotides coding for gastric inhibitory polypeptide (GIP),&quot; (1980) Nucleic Acids Res. Symp. Ser., 225-232."><meta name="citation_reference" content="Hudson et al., &quot;An STS-based map of the human genome,&quot; (1995) Science, 270:1945."><meta name="citation_reference" content="Huse et al., &quot;Generation of a largo combinatorial library of the immunoglobulin repertoire in phage lambda,&quot; (1989) Science, 246:1275."><meta name="citation_reference" content="Johnson et al., (1981) Mol. Cell Biol. 11:1."><meta name="citation_reference" content="Kilian et al., &quot;Isolation of a candidate human telomerase catalytic subunit gene, which reveals complex splicing patterns in different cell types,&quot; (1997) Hum. Mol. Genet., 6:2011."><meta name="citation_reference" content="Kipling and Cooke, &quot;Hypervariable ultra-long telomeres in mice,&quot; (1990) Nature 347:400."><meta name="citation_reference" content="Klobutcher et al., &quot;All gene-sized DNA molecules in four species of hypotrichs have the same terminal sequence and an unusual 3&#39;&#39; terminus,&quot; (1981) Proc. Natl. Acad. Sci., 78:3015."><meta name="citation_reference" content="Klobutcher et al., “All gene-sized DNA molecules in four species of hypotrichs have the same terminal sequence and an unusual 3″ terminus,” (1981) Proc. Natl. Acad. Sci., 78:3015."><meta name="citation_reference" content="Koehler and Milstein, &quot;Continuous cultures of fused cells secreting antibody of predefined specificity,&quot; (1975) Nature 258:495."><meta name="citation_reference" content="Kosbor et al., &quot;The production of monocinal antibodies from human lymphocytes,&quot; (1983) Immunol. Today 4:72."><meta name="citation_reference" content="Lamond and Sproat, (1994) &quot;Isolation and Characterization of Ribonucleoprotein Complexes,&quot; pp. 103-140."><meta name="citation_reference" content="Lamond et al., &quot;Probing the structure and function of U2 an RNP with antisense oligonucleotides made of 2&#39;&#39;-OMe RNA,&quot; (1989) Cell, 58:383."><meta name="citation_reference" content="Lamond et al., “Probing the structure and function of U2 an RNP with antisense oligonucleotides made of 2″-OMe RNA,” (1989) Cell, 58:383."><meta name="citation_reference" content="Lendvey et al., &quot;Senescence mutants of Saccharomyces cerevisiae with a defect in telomere replication identify three additional EST genes,&quot; (1996) Genetics, 144."><meta name="citation_reference" content="Lingler et al., &quot;Purification of telomerase from Euplotes adeiculatus: requirement of a printer 3&#39;&#39; overhang,&quot; (1998) Proc. Natl. Acad. Sci., 93:10712."><meta name="citation_reference" content="Lingler et al., &quot;Reverse transcriptase motifs in the catalytic subunits of telomerase,&quot; (1997) Science, 278:561."><meta name="citation_reference" content="Lingler et al., “Purification of telomerase from Euplotes adeiculatus: requirement of a printer 3″ overhang,” (1998) Proc. Natl. Acad. Sci., 93:10712."><meta name="citation_reference" content="Lingner et al., &quot;Telomerase and DNA End Replication: No Longer a Lagging Strand Problem?,&quot; (1995) Science 289:1533."><meta name="citation_reference" content="Lingner et al., &quot;Telomerase RNAs of different ciliates have a common secondary structure and a permitted template,&quot; Genes Develop., 8:1984."><meta name="citation_reference" content="Lowy et al., &quot;Isolation of transforming DNA: Cloning the hamster aprt gene,&quot; (1980) Cell, 22:617."><meta name="citation_reference" content="Lustig and Peles, Identification of yeast mutants with altered telomere structure, (1986) Proc. Natl. Acad. Sci., 83:1398."><meta name="citation_reference" content="Maddox et al., &quot;Elevated serum levels in human pregnancy of a molecule immunochemically similar to eosinophil granule major basic protein,&quot; (1983) J. Exp. Med., 158:1211."><meta name="citation_reference" content="Malby et al., &quot;Quantitative measurement of human cytokine gene expression by polymerase chain reaction,&quot; (1993) J. Immunol. Meth., 159:235."><meta name="citation_reference" content="McEachern and Blackburn, &quot;runaway telomere elongation caused by telomerase RNA gene mutation,&quot; (1995) Nature, 376:403."><meta name="citation_reference" content="Mekerov et al., &quot;Nucleosomal Organization of Telomere-Specific Chromatin in Rat,&quot; (1993) Cell, 73:775."><meta name="citation_reference" content="Merrifield, &quot;Solid phase peptide synthesis, I. The synthesis of a tetrapeptide,&quot; (1983) J. Am. Chem. Soc., 65:2149."><meta name="citation_reference" content="Meyerson et al., &quot;hEST2, the Putative Human Telomerase Catalytic Subunit Gene, Is Up-Regulated In Tumor Cells and during Immortalization,&quot; (1997) Cell, 90:785."><meta name="citation_reference" content="Murray, In McGraw Hill Yearbook of Science and Technology, (1992) McGraw Hill, New York NY, pp 181-196."><meta name="citation_reference" content="Nakamura et al., &quot;Telomerase Catalytic Subunit Homologs from Fission Yeast and Human,&quot; (1997) Science, 277:955."><meta name="citation_reference" content="Nakayama et al., &quot;TLP1: A Gene Encoding a Protein Component of Mammalian Telomerase is a Novel Member of WD Repeats Family,&quot; (1997) Cell, 88:875."><meta name="citation_reference" content="Nielsen et al., (1993) &quot;Peptide nucleic acids (PNAs): Potential antisense and anti-gene agents,&quot; Anticancer Drug Des., 6:53."><meta name="citation_reference" content="Oka et al., &quot;Inverted terminal repeat sequence in the macronuclear DNA of Stylonychia pustulata,&quot; (1980) Gene, 10:301."><meta name="citation_reference" content="Olovnikov, &quot;A theory of marginotomy: The incomplete copying of template margin in enzymic synthesis of polynucleotides and biological significance of the phenomonon,&quot; (1973) J. Theor. Biol., 41:181."><meta name="citation_reference" content="Orlandl et al., &quot;Cloning immunoglobulin variable domains for expression by the polymerase chain reaction,&quot; (1989) Proc. Natl. Acad. Sci., 86:8833."><meta name="citation_reference" content="Prescott, &quot;The DNA of cilated protozea,&quot; (1994) Microbiol. Rev., 58:233."><meta name="citation_reference" content="Price, (1993) Blood Rev., 7:127."><meta name="citation_reference" content="Rhodes et al., &quot;Transformation of maize by electroporation of embryos,&quot; 1985) Meth. Mol. Biol., 55:121."><meta name="citation_reference" content="Roberge et al., &quot;A strategy for a convergent synthesis of N-linked glycopeptides on a solid support,&quot; (1995) Science, 269:202."><meta name="citation_reference" content="Romero and Blackburn, &quot;A conserved secondary structure for telomerase RNA,&quot; (1991) Cell, 67:343."><meta name="citation_reference" content="Sambrock et al., Molecular Cloning, A Laboratory Manual, Cold Spring Harbor Press, Plainview NY (1989)."><meta name="citation_reference" content="Sandell et al., &quot;Transcription of yeast telomere alleviates telomere position effect without affecting chromosome stability,&quot; (1994) Proc. Natl. Acad. Sci., 91:12081."><meta name="citation_reference" content="Sanger et al., &quot;DNA sequencing with chain-terminating Inhibitors,&quot; (1977) Proc. Natl. Acad. Sci., 74:5463."><meta name="citation_reference" content="Scharf et al., &quot;Heat stress promoters and transcription factors,&quot; (1994) Result Probl. Cell Differ. 20:125."><meta name="citation_reference" content="Shampay and Blackburn, &quot;Generation of telomere-length heterogeneity in Saccharomyces cerevisiae,&quot; (1988) Proc. Natl. Acad. Sci., 85:534."><meta name="citation_reference" content="Sheen and Levis, &quot;Transposition of the LINE-like retrotransposon TART to Drosophila chromosome termini,&quot; (1994) Proc. Natl. Acad. Sci., 91:12510."><meta name="citation_reference" content="Singer and Gottschling, &quot;TLC1: Template RNA Component of Saccharomyces cerevisiae Telomerase,&quot; (1994) Science 266:404."><meta name="citation_reference" content="Starling et al., &quot;Extensive telomere repeat arrays in mouse are hypervariable,&quot; (1990) Nucleic Acids Res., 18:6881."><meta name="citation_reference" content="Swanton et al., &quot;Arrangement of Cooling and Non-coding Sequence in the DNA Molecules Coding for rRNAs in Oxytricha sp.,&quot; (1980) Chromosoma 77:203."><meta name="citation_reference" content="Tommerup et al., &quot;Unusual chromatin in human telomeres,&quot; (1994) Mol. Cell. Biol., 14:5777."><meta name="citation_reference" content="Trask, &quot;Fluorescence in situ hybridization: application in cytogenetics and gene mapping,&quot; (1991) Trends Genet., 7:149."><meta name="citation_reference" content="U.S. patent application Ser. No. 08/751,189, Harrington et al., filed Nov. 15, 1996."><meta name="citation_reference" content="U.S. patent application Ser. No. 60/038,760, Counter et al., filed Feb. 20, 1997."><meta name="citation_reference" content="Verma et al., &quot;Human Chromosomes: A Manual of Basic Techniques,&quot; Pergamon Press, New York NY (1988)."><meta name="citation_reference" content="Watson, &quot;Origin of concatermeric T7 DNA,&quot; (1972) Nature New Biol., 239:197."><meta name="citation_reference" content="Weinrich et al., &quot;Reconstitution of human telomerase with the template RNA component hTR and the catalytic protein subunit hTRT,&quot; (1997) Nat. Genet., 17(4):498."><meta name="citation_reference" content="Wellinger et al., “Origin activation and formation of single-strand TG1-3 tails occur sequentially in late S phase on a Yeast linear plasmid,” (1993) Mol. Cell. Biol., 13:4057."><meta name="citation_reference" content="Wellinger et al., “Saccharomyces Telomeres Acquire Single-Strand TG1-3 Tails Late in S Phase,” (1993) Cell 72:51."><meta name="citation_reference" content="Whitehead Institute/MIT Center for Genome Research, Genetic Map of the Mouse, Database Release 10, Apr. 28, 1995."><meta name="citation_reference" content="Wigler et al., “Transfer of purified herpes virus thymidine kinase gene to cultured mouse cells,” (1977) Cell, 11:223."><meta name="citation_reference" content="Wigler et al., “Transformation of mammalian cells with an amplifiable dominant-acting gene,” (1980) Proc. Natl. Acad. Sci., 77:3567."><meta name="citation_reference" content="Winter and Milstein, “Man-made antibodies,” (1991) Nature, 349:293."><meta name="citation_reference" content="Wright et al., “Saccharomycas telomeres assume a non-nucleosomal chromatin structure,” (1992) Genes Develop., 6:197."><meta name="citation_reference" content="Yu et al., “In vivo alteration of telomere sequences and senescence caused by mutated Tetrahymena telomerase RNAs,” (1990) Nature, 344:126."><meta name="citation_reference" content="Zahler and Prescott, “Telomere terminal transferase activity in the hypotrichous ciliate Oxytricha nova and a model for replication of the ends of linear DNA molecules,” (1988) Nucleic Acids Res., 16:6953."><meta name="citation_reference" content="Zakian, Telomeres: Beginning to Understand the End, (1995) Science 270:1601."><meta name="citation_patent_number" content="US:6617110"><meta name="citation_patent_application_number" content="US:09/721,456"><link rel="canonical" href="http://www.google.com/patents/US6617110"/><meta property="og:url" content="http://www.google.com/patents/US6617110"/><meta name="title" content="Patent US6617110 - Cells immortalized with telomerase reverse transcriptase for use in drug screening"/><meta name="description" content="The invention provides compositions and methods related to human telomerase reverse transcriptase (hTRT), the catalytic protein subunit of human telomerase. The polynucleotides and polypeptides of the invention are useful for diagnosis, prognosis and treatment of human diseases, for changing the proliferative capacity of cells and organisms, and for identification and screening of compounds and treatments useful for treatment of diseases such as cancers."/><meta property="og:title" content="Patent US6617110 - Cells immortalized with telomerase reverse transcriptase for use in drug screening"/><meta property="og:type" content="book"/><meta property="og:site_name" content="Google Books"/><meta property="og:image" content="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><link rel="image_src" href="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><script>(function(){try{var aa=function(a,b,c,d){d=d||{};d._sn=["cfg",b,c].join(".");window.gbar.logger.ml(a,d)};var g=window.gbar=window.gbar||{},l=window.gbar.i=window.gbar.i||{},m={},n;function _tvn(a,b){var c=parseInt(a,10);return isNaN(c)?b:c}function _tvf(a,b){var c=parseFloat(a);return isNaN(c)?b:c}function _tvv(a){return!!a}function p(a,b,c){(c||g)[a]=b}g.bv={n:_tvn("2",0),r:"",f:".67.",e:"0",m:_tvn("0",1)};
function q(a,b,c){var d="on"+b;if(a.addEventListener)a.addEventListener(b,c,!1);else if(a.attachEvent)a.attachEvent(d,c);else{var f=a[d];a[d]=function(){var a=f.apply(this,arguments),b=c.apply(this,arguments);return void 0==a?b:void 0==b?a:b&&a}}}var s=function(a){return function(){return g.bv.m==a}},ba=s(1),ca=s(2);p("sb",ba);p("kn",ca);l.a=_tvv;l.b=_tvf;l.c=_tvn;l.i=aa;var da=window.gbar.i.i;var t,u,v,w;function ea(a){v=a}function fa(a){var b;if(b=v&&window.encodeURIComponent)b=a.href,b=!b.match(/^http[s]?:\/\/accounts\.google\.[^/]*\/ClearSID/i)&&!b.match(/^http[s]?:\/\/[^/]*\/accounts\/ClearSID/i);if(b=b&&encodeURIComponent(v()))a.href=a.href.replace(/([?&]continue=)[^&]*/,"$1"+b)}function ga(a){window.gApplication&&(a.href=window.gApplication.getTabUrl(a.href))}
function ha(a){var b=document.forms[0].q,c=window.encodeURIComponent&&b&&b.value,b=b&&b.placeholder;c&&c!=b&&(a.href=a.href.replace(/([?&])q=[^&]*|$/,function(a,b){return(b||"&")+"q="+encodeURIComponent(c)}))}n=l.a("")?ga:ha;
function x(a,b,c,d,f,e){var h=document.getElementById(a);if(h){var k=h.style;k.left=d?"auto":b+"px";k.right=d?b+"px":"auto";k.top=c+"px";k.visibility=u?"hidden":"visible";f&&e?(k.width=f+"px",k.height=e+"px"):(x(t,b,c,d,h.offsetWidth,h.offsetHeight),u=u?"":a)}}
var y=[],ia=function(a,b){y.push(b)},ja=function(a){a=a||window.event;var b=a.target||a.srcElement;a.cancelBubble=!0;null==t&&(a=document.createElement(Array.every||window.createPopup?"iframe":"div"),a.frameBorder="0",t=a.id="gbs",a.src="javascript:''",b.parentNode.appendChild(a),q(document,"click",z));var c=b,b=0;"gb3"!=c.className&&(c=c.parentNode);a=c.getAttribute("aria-owns")||"gbi";var d=c.offsetWidth,f=20<c.offsetTop?46:24;document.getElementById("tphdr")&&(f-=3);var e=!1;do b+=c.offsetLeft||
0;while(c=c.offsetParent);var c=(document.documentElement.clientWidth||document.body.clientWidth)-b-d,h,d=document.body,k=document.defaultView;k&&k.getComputedStyle?(d=k.getComputedStyle(d,""))&&(h=d.direction):h=d.currentStyle?d.currentStyle.direction:d.style.direction;h="rtl"==h;if("gbi"==a){for(d=0;k=y[d++];)k();A(null,window.navExtra);h&&(b=c,e=!0)}else h||(b=c,e=!0);u!=a&&z();x(a,b,f,e)},z=function(){u&&x(u,0,0)},A=function(a,b){var c,d=document.getElementById("gbi"),f=a;f||(f=d.firstChild);
for(;b&&(c=b.pop());){var e=d,h=c,k=f;w||(w="gb2");e.insertBefore(h,k).className=w}},ka=function(a,b,c){if((b=document.getElementById(b))&&a){a.className="gb4";var d=document.createElement("span");d.appendChild(a);d.appendChild(document.createTextNode(" | "));d.id=c;b.appendChild(d)}},la=function(){return document.getElementById("gb_70")},ma=function(){return!!u};p("qs",n);p("setContinueCb",ea);p("pc",fa);p("tg",ja);p("close",z);p("addLink",ka);p("almm",A);p("si",la);p("adh",ia);p("op",ma);var B=function(){},C=function(){},F=function(a){var b=new Image,c=D;b.onerror=b.onload=b.onabort=function(){try{delete E[c]}catch(a){}};E[c]=b;b.src=a;D=c+1},E=[],D=0;p("logger",{il:C,ml:B,log:F});var G=window.gbar.logger;var H={},na={},I=[],oa=l.b("0.1",.1),pa=l.a("1",!0),qa=function(a,b){I.push([a,b])},ra=function(a,b){H[a]=b},sa=function(a){return a in H},J={},K=function(a,b){J[a]||(J[a]=[]);J[a].push(b)},ta=function(a){K("m",a)},L=function(a,b){var c=document.createElement("script");c.src=a;c.async=pa;Math.random()<oa&&(c.onerror=function(){c.onerror=null;B(Error("Bundle load failed: name="+(b||"UNK")+" url="+a))});(document.getElementById("xjsc")||document.getElementsByTagName("body")[0]||
document.getElementsByTagName("head")[0]).appendChild(c)},N=function(a){for(var b=0,c;(c=I[b])&&c[0]!=a;++b);!c||c[1].l||c[1].s||(c[1].s=!0,M(2,a),c[1].url&&L(c[1].url,a),c[1].libs&&m.d&&m.d(c[1].libs))},O=function(a){K("gc",a)},P=null,ua=function(a){P=a},M=function(a,b,c){if(P){a={t:a,b:b};if(c)for(var d in c)a[d]=c[d];try{P(a)}catch(f){}}};p("mdc",H);p("mdi",na);p("bnc",I);p("qGC",O);p("qm",ta);p("qd",J);p("lb",N);p("mcf",ra);p("bcf",qa);p("aq",K);p("mdd","");p("has",sa);
p("trh",ua);p("tev",M);var Q=l.b("0.1",.001),R=0;
function _mlToken(a,b){try{if(1>R){R++;var c,d=a,f=b||{},e=encodeURIComponent,h=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&jexpid=",e("17483"),"&srcpg=",e("prop=22"),"&jsr=",Math.round(1/Q),"&ogev=",e("g57tU_C1O9TSsASYlIDICw"),"&ogf=",g.bv.f,"&ogrp=",e("1"),"&ogv=",e("1407723702.0"),"&oggv="+e("es_plusone_gc_20140723.0_p0"),"&ogd=",e("com"),"&ogc=",e("LUX"),"&ogl=",e("en")];f._sn&&(f._sn="og."+
f._sn);for(var k in f)h.push("&"),h.push(e(k)),h.push("="),h.push(e(f[k]));h.push("&emsg=");h.push(e(d.name+":"+d.message));var r=h.join("");S(r)&&(r=r.substr(0,2E3));c=r;var Aa=window.gbar.logger._aem(a,c);F(Aa)}}catch(Na){}}var S=function(a){return 2E3<=a.length},va=function(a,b){return b};function T(a){B=a;p("_itl",S,G);p("_aem",va,G);p("ml",B,G);a={};H.er=a}l.a("")?T(function(a){throw a;}):l.a("1")&&Math.random()<Q&&T(_mlToken);I.push(["m",{url:"//ssl.gstatic.com/gb/js/scm_7385cc5883250b43a39405734c1bea59.js"}]);g.mcf("c",{});g.sg={c:""};if(l.a("1")){var wa=l.a("");I.push(["gc",{auto:wa,url:"//ssl.gstatic.com/gb/js/abc/gci_91f30755d6a6b787dcc2a4062e6e9824.js",libs:"googleapis.client:plusone:gapi.iframes"}]);var xa={version:"gci_91f30755d6a6b787dcc2a4062e6e9824.js",index:"",lang:"en"};H.gc=xa;var U=function(a){window.googleapis&&window.iframes?a&&a():(a&&O(a),N("gc"))};p("lGC",U);l.a("1")&&p("lPWF",U)};window.__PVT="";if(l.a("1")&&l.a("1")){var V=function(a){U(function(){K("pw",a);N("pw")})};p("lPW",V);I.push(["pw",{url:"//ssl.gstatic.com/gb/js/abc/pwm_45f73e4df07a0e388b0fa1f3d30e7280.js"}]);var W=[],ya=function(a){W[0]=a},za=function(a,b){var c=b||{};c._sn="pw";B(a,c)},Ba={signed:W,elog:za,base:"https://plusone.google.com/u/0",loadTime:(new Date).getTime()};H.pw=Ba;var X=function(a,b){for(var c=b.split("."),d=function(){var b=arguments;a(function(){for(var a=g,d=0,e=c.length-1;d<e;++d)a=a[c[d]];a[c[d]].apply(a,b)})},f=g,e=0,h=c.length-1;e<h;++e)f=
f[c[e]]=f[c[e]]||{};return f[c[e]]=d};X(V,"pw.clk");X(V,"pw.hvr");p("su",ya,g.pw)};function Ca(){function a(){for(var b;(b=e[h++])&&"m"!=b[0]&&!b[1].auto;);b&&(M(2,b[0]),b[1].url&&L(b[1].url,b[0]),b[1].libs&&m.d&&m.d(b[1].libs));h<e.length&&setTimeout(a,0)}function b(){0<f--?setTimeout(b,0):a()}var c=l.a("1"),d=l.a(""),f=3,e=I,h=0,k=window.gbarOnReady;if(k)try{k()}catch(r){da(r,"ml","or")}d?p("ldb",a):c?q(window,"load",b):b()}p("rdl",Ca);var Da={D:1,H:2,da:3,p:4,W:5,M:6,F:7,g:8,ha:9,U:10,L:11,T:12,S:13,N:14,Q:15,P:16,fa:17,w:18,O:19,ga:20,ea:21,u:22,G:23,ja:24,ka:25,ia:26,A:27,j:28,o:29,k:30,ca:31,Z:32,$:33,J:34,K:35,ba:36,aa:37,Y:38,B:39,R:40,v:41,X:42,V:43,h:48,C:49,I:500},Y=[1,2,3,4,5,6,9,10,11,13,14,28,29,30,34,35,37,38,39,40,41,42,43,48,49,500];var Z=l.b("0.001",1E-4),Ea=l.b("1",1),Fa=!1,Ga=!1;if(l.a("1")){var Ha=Math.random();Ha<=Z&&(Fa=!0);Ha<=Ea&&(Ga=!0)}var Ia=Da,$=null;function Ja(){var a=0,b=function(b,d){l.a(d)&&(a|=b)};b(1,"");b(2,"");b(4,"");b(8,"");return a}
function Ka(a,b){var c=Z,d=Fa,f;f=a;if(!$){$={};for(var e=0;e<Y.length;e++){var h=Y[e];$[h]=!0}}if(f=!!$[f])c=Ea,d=Ga;if(d){d=encodeURIComponent;g.rp?(f=g.rp(),f="-1"!=f?f:"1"):f="1";c=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&oge=",a,"&ogex=",d("17483"),"&ogev=",d("g57tU_C1O9TSsASYlIDICw"),"&ogf=",g.bv.f,"&ogp=",d("22"),"&ogrp=",d(f),"&ogsr=",Math.round(1/c),"&ogv=",d("1407723702.0"),"&oggv="+
d("es_plusone_gc_20140723.0_p0"),"&ogd=",d("com"),"&ogl=",d("en"),"&ogc=",d("LUX"),"&ogus=",Ja()];if(b){"ogw"in b&&(c.push("&ogw="+b.ogw),delete b.ogw);var k;f=b;e=[];for(k in f)0!=e.length&&e.push(","),e.push(La(k)),e.push("."),e.push(La(f[k]));k=e.join("");""!=k&&(c.push("&ogad="),c.push(d(k)))}F(c.join(""))}}function La(a){"number"==typeof a&&(a+="");return"string"==typeof a?a.replace(".","%2E").replace(",","%2C"):a}C=Ka;p("il",C,G);var Ma={};H.il=Ma;setTimeout(function(){C(Ia.g)},0);}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var b=window.gbar.i.i;var c=window.gbar;var f=function(d){try{var a=document.getElementById("gbom");a&&d.appendChild(a.cloneNode(!0))}catch(e){b(e,"omas","aomc")}};c.aomc=f;}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var a=window.gbar;a.mcf("pm",{p:""});}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{window.gbar.rdl();}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
if (window['_OC_timingAction']) {window['_OC_timingAction']('patents_refpage');}</script><style>#gbar,#guser{font-size:13px;padding-top:1px !important;}#gbar{float:left;height:22px}#guser{padding-bottom:7px !important;text-align:right}.gbh,.gbd{border-top:1px solid #c9d7f1;font-size:1px}.gbh{height:0;position:absolute;top:24px;width:100%}#gbs,.gbm{background:#fff;left:0;position:absolute;text-align:left;visibility:hidden;z-index:1000}.gbm{border:1px solid;border-color:#c9d7f1 #36c #36c #a2bae7;z-index:1001}.gb1{margin-right:.5em}.gb1,.gb3{zoom:1}.gb2{display:block;padding:.2em .5em}.gb2,.gb3{text-decoration:none !important;border-bottom:none}a.gb1,a.gb4{text-decoration:underline !important}a.gb1,a.gb2,a.gb3,a.gb4{color:#00c !important}.gbi .gb3,.gbi .gb2,.gbi .gb4{color:#dd8e27 !important}.gbf .gb3,.gbf .gb2,.gbf .gb4{color:#900 !important}a.gb2:hover{background:#36c;color:#fff !important}#gbar .gbz0l{color:#000 !important;cursor:default;font-weight:bold;text-decoration:none !important}
#gbar { padding:.3em .6em !important;}</style></head><body  topmargin="3" marginheight="3"><div id=gbar><nobr><a onclick=gbar.qs(this);gbar.logger.il(1,{t:1}); class=gb1 id=gb_1 href="https://www.google.com/search?sa=N&tab=tw">Search</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:2}); class=gb1 id=gb_2 href="http://www.google.com/search?hl=en&tbm=isch&source=og&sa=N&tab=ti">Images</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:8}); class=gb1 id=gb_8 href="http://maps.google.com/maps?hl=en&sa=N&tab=tl">Maps</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:78}); class=gb1 id=gb_78 href="https://play.google.com/?hl=en&sa=N&tab=t8">Play</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:36}); class=gb1 id=gb_36 href="http://www.youtube.com/results?sa=N&tab=t1">YouTube</a> <a onclick=gbar.logger.il(1,{t:5}); class=gb1 id=gb_5 href="http://news.google.com/nwshp?hl=en&tab=tn">News</a> <a onclick=gbar.logger.il(1,{t:23}); class=gb1 id=gb_23 href="https://mail.google.com/mail/?tab=tm">Gmail</a> <a onclick=gbar.logger.il(1,{t:25}); class=gb1 id=gb_25 href="https://drive.google.com/?tab=to">Drive</a> <a class=gb3 href="http://www.google.com/intl/en/options/" onclick="this.blur();gbar.tg(event);return !1" aria-haspopup=true><u>More</u> <small>&#9660;</small></a><div class=gbm id=gbi><a onclick=gbar.logger.il(1,{t:24}); class=gb2 id=gb_24 href="https://www.google.com/calendar?tab=tc">Calendar</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:51}); class=gb2 id=gb_51 href="http://translate.google.com/?hl=en&sa=N&tab=tT">Translate</a><a onclick=gbar.logger.il(1,{t:17}); class=gb2 id=gb_17 href="http://www.google.com/mobile/?hl=en&tab=tD">Mobile</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:10}); class=gb2 id=gb_10 href="http://www.google.com/search?hl=en&tbo=u&tbm=bks&source=og&sa=N&tab=tp">Books</a><a onclick=gbar.logger.il(1,{t:212}); class=gb2 id=gb_212 href="https://wallet.google.com/manage/?tab=ta">Wallet</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:6}); class=gb2 id=gb_6 href="http://www.google.com/search?hl=en&tbo=u&tbm=shop&source=og&sa=N&tab=tf">Shopping</a><a onclick=gbar.logger.il(1,{t:30}); class=gb2 id=gb_30 href="http://www.blogger.com/?tab=tj">Blogger</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:27}); class=gb2 id=gb_27 href="http://www.google.com/finance?sa=N&tab=te">Finance</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:31}); class=gb2 id=gb_31 href="https://plus.google.com/photos?sa=N&tab=tq">Photos</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:12}); class=gb2 id=gb_12 href="http://www.google.com/search?hl=en&tbo=u&tbm=vid&source=og&sa=N&tab=tv">Videos</a><div class=gb2><div class=gbd></div></div><a onclick=gbar.logger.il(1,{t:66}); href="http://www.google.com/intl/en/options/" class=gb2>Even more &raquo;</a></div></nobr></div><div id=guser width=100%><nobr><span id=gbn class=gbi></span><span id=gbf class=gbf></span><span id=gbe></span><a target=_top id=gb_70 href="https://www.google.com/accounts/Login?service=&continue=http://www.google.com/patents%3Fhl%3Den&hl=en" class=gb4>Sign in</a><div style="display: none"><div class=gbm id=gbd5 aria-owner=gbg5><div class=gbmc><ol id=gbom class=gbmcc></ol></div></div></div></nobr></div><div class=gbh style=left:0></div><div class=gbh style=right:0></div><div role="alert" style="position: absolute; left: 0; right: 0;"><a href="http://www.google.com/patents/us6617110?hl=en&amp;output=html_text" title="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."><img border="0" src="http://www.google.com/images/cleardot.gif"alt="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."></a></div><div id="guser"><nobr></nobr></div><div style="clear:both;"></div><div id="gb-top-search-box" class="gb-top-search-box-small gb-reset"><table><tr><td class="logo"><a href="http://www.google.com/patents" class="logo-link"><img class="logo-img" src="/intl/en/images/logos/google_logo_41.png" alt="Go to Google Books Home" height="41"/></a></td><td><form action="http://www.google.com/search" name="f" id="vheadf" method="get"><span id="hf"></span><input type="hidden" name="tbm" value="pts"/><input type="hidden" name="tbo" value="1"/><input type="hidden" name="hl" value="en"/><table><tr><td><div class="inputs"><table><tr><td><div class="text-input"><input type="text" name="q" id="vheadq" class="text" maxlength="2048" size="31" value="" title="Search Patents" accesskey="s" autocomplete="off"/><script>window._OC_autoDir &&window._OC_autoDir('vheadq', 'tia-vheadq');</script></div></td><td><div class="submit-input"><input name="btnG" class="submit" type="submit" value=""/></div></td></tr></table></div></td><td class="col-ext-links"><div class="ext-links"><a href="http://www.google.com/advanced_patent_search">&lt;nobr&gt;Advanced Patent Search&lt;/nobr&gt;</a></div></td></tr></table></form></td></tr></table></div><div class="kd-appbar"><h2 class="kd-appname"><a href="/patents">Patents</a></h2><div class="kd-buttonbar left" id="left-toolbar-buttons"><a id="appbar-write-review-link" href=""></a><a id="appbar-view-print-sample-link" href=""></a><a id="appbar-view-ebook-sample-link" href=""></a><a id="appbar-patents-prior-art-finder-link" href="https://www.google.com/patents/related/US6617110"></a><a id="appbar-patents-discuss-this-link" href="http://www.google.com/url?id=JOlhBAABERAJ&amp;q=http://patents.stackexchange.com/redirect/google-patents%3Fpatent%3DUS6617110&amp;usg=AFQjCNEBvh2imC3aFwOD4YDgE4yzvqmk9g" data-is-grant="true"></a><a id="appbar-read-patent-link" href="//docs.google.com/viewer?url=patentimages.storage.googleapis.com/pdfs/US6617110.pdf"></a><a id="appbar-download-pdf-link" href="//patentimages.storage.googleapis.com/pdfs/US6617110.pdf"></a></div><div class="kd-buttonbar right" id="right-toolbar-buttons"></div></div><div id="books-microdata" itemscope=""itemtype="http://schema.org/Book"itemid="http://www.google.com/patents/US6617110" style="display:none"><span itemprop="description">The invention provides compositions and methods related to human telomerase reverse transcriptase (hTRT), the catalytic protein subunit of human telomerase. The polynucleotides and polypeptides of the invention are useful for diagnosis, prognosis and treatment of human diseases, for changing the proliferative...</span><span itemprop="url">http://www.google.com/patents/US6617110?utm_source=gb-gplus-share</span><span class="main-title" itemprop="name">Patent US6617110 - Cells immortalized with telomerase reverse transcriptase for use in drug screening</span><img itemprop="image" src="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"alt="Patent US6617110 - Cells immortalized with telomerase reverse transcriptase for use in drug screening" title="Patent US6617110 - Cells immortalized with telomerase reverse transcriptase for use in drug screening"></div><div style="display: none"><ol id="ofe-gear-menu-contents" class="gbmcc"><li class="gbe gbmtc"><a class="gbmt goog-menuitem-content" id="" href="http://www.google.com/advanced_patent_search">Advanced Patent Search</a></li></ol></div><table id="viewport_table" cellpadding="0" style="clear:both" cellspacing="0"><tr><td id="viewport_td"><div class=vertical_module_list_row><div id=intl_patents class=about_content><div id=intl_patents_v><table class="patent-bibdata"><tr><td class="patent-bibdata-heading">Publication number</td><td class="single-patent-bibdata">US6617110 B1</td></tr><tr><td class="patent-bibdata-heading">Publication type</td><td class="single-patent-bibdata">Grant</td></tr><tr><td class="patent-bibdata-heading">Application number</td><td class="single-patent-bibdata">US 09/721,456</td></tr><tr><td class="patent-bibdata-heading">Publication date</td><td class="single-patent-bibdata">Sep 9, 2003</td></tr><tr><td class="patent-bibdata-heading">Filing date</td><td class="single-patent-bibdata">Nov 22, 2000</td></tr><tr><td class="patent-bibdata-heading">Priority date<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The priority date is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the date listed."></span></td><td class="single-patent-bibdata">Oct 1, 1996</td></tr><tr><td class="patent-bibdata-heading">Fee status<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The fee status is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the status or dates listed."></span></td><td class="single-patent-bibdata">Paid</td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Also published as</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/CA2266752A1">CA2266752A1</a>, </span><span class="patent-bibdata-value"><a href="/patents/CA2267664A1">CA2267664A1</a>, </span><span class="patent-bibdata-value"><a href="/patents/CA2267664C">CA2267664C</a>, </span><span class="patent-bibdata-value"><a href="/patents/CA2645721A1">CA2645721A1</a>, </span><span class="patent-bibdata-value"><a href="/patents/CN1254376A">CN1254376A</a>, </span><span class="patent-bibdata-value"><a href="/patents/CN1291231A">CN1291231A</a>, </span><span class="patent-bibdata-value"><a href="/patents/CN1291231B">CN1291231B</a>, </span><span class="patent-bibdata-value"><a href="/patents/CN102021153A">CN102021153A</a>, </span><span class="patent-bibdata-value"><a href="/patents/DE841396T1">DE841396T1</a>, </span><span class="patent-bibdata-value"><a href="/patents/DE19743497A1">DE19743497A1</a>, </span><span class="patent-bibdata-value"><a href="/patents/DE69723531D1">DE69723531D1</a>, </span><span class="patent-bibdata-value"><a href="/patents/DE69723531T2">DE69723531T2</a>, </span><span class="patent-bibdata-value"><a href="/patents/DE69723531T3">DE69723531T3</a>, </span><span class="patent-bibdata-value"><a href="/patents/DE69739497D1">DE69739497D1</a>, </span><span class="patent-bibdata-value"><a href="/patents/DE69739675D1">DE69739675D1</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP0841396A1">EP0841396A1</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP0841396B1">EP0841396B1</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP0841396B2">EP0841396B2</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP0932686A2">EP0932686A2</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP0954585A2">EP0954585A2</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP0954585B1">EP0954585B1</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP1333094A2">EP1333094A2</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP1333094A3">EP1333094A3</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP1333094B1">EP1333094B1</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP1783139A2">EP1783139A2</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP1783139A3">EP1783139A3</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP1783139B1">EP1783139B1</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP1783139B8">EP1783139B8</a>, </span><span class="patent-bibdata-value"><a href="/patents/EP2213740A1">EP2213740A1</a>, </span><span class="patent-bibdata-value"><a href="/patents/US6166178">US6166178</a>, </span><span class="patent-bibdata-value"><a href="/patents/US7517971">US7517971</a>, </span><span class="patent-bibdata-value"><a href="/patents/US8222392">US8222392</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20090269739">US20090269739</a>, </span><span class="patent-bibdata-value"><a href="/patents/WO1998014592A2">WO1998014592A2</a>, </span><span class="patent-bibdata-value"><a href="/patents/WO1998014592A3">WO1998014592A3</a>, </span><span class="patent-bibdata-value"><a href="/patents/WO1998014593A2">WO1998014593A2</a>, </span><span class="patent-bibdata-value"><a href="/patents/WO1998014593A3">WO1998014593A3</a>, </span><span class="patent-bibdata-value"><a href="/patents/WO1998014593A9">WO1998014593A9</a></span></span></td></tr><tr class="patent-bibdata-list-row alternate-patent-number"><td class="patent-bibdata-heading">Publication number</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value">09721456, </span><span class="patent-bibdata-value">721456, </span><span class="patent-bibdata-value">US 6617110 B1, </span><span class="patent-bibdata-value">US 6617110B1, </span><span class="patent-bibdata-value">US-B1-6617110, </span><span class="patent-bibdata-value">US6617110 B1, </span><span class="patent-bibdata-value">US6617110B1</span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Inventors</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Thomas+R.+Cech%22">Thomas R. Cech</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Joachim+Lingner%22">Joachim Lingner</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Toru+Nakamura%22">Toru Nakamura</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Karen+B.+Chapman%22">Karen B. Chapman</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Gregg+B.+Morin%22">Gregg B. Morin</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Calvin+B.+Harley%22">Calvin B. Harley</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22William+H.+Andrews%22">William H. Andrews</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Original Assignee</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=inassignee:%22Geron+Corporation%22">Geron Corporation</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=inassignee:%22University+Technology+Corporation%22">University Technology Corporation</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Export Citation</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/US6617110.bibtex">BiBTeX</a>, </span><span class="patent-bibdata-value"><a href="/patents/US6617110.enw">EndNote</a>, </span><span class="patent-bibdata-value"><a href="/patents/US6617110.ris">RefMan</a></span></span></td></tr><tr class="patent-internal-links"><td colspan=2><span class="patent-bibdata-value"><a href="#backward-citations">Patent Citations</a> (35),</span> <span class="patent-bibdata-value"><a href="#npl-citations">Non-Patent Citations</a> (114),</span> <span class="patent-bibdata-value"><a href="#forward-citations">Referenced by</a> (23),</span> <span class="patent-bibdata-value"><a href="#classifications">Classifications</a> (72),</span> <span class="patent-bibdata-value"><a href="#legal-events">Legal Events</a> (6)</span> </td></tr><tr><td colspan=2 class="patent-bibdata-external-link-spacer-top"></td></tr><tr class="patent-bibdata-external-link-spacer-bottom"></tr><tr><td colspan=2><span class="patent-bibdata-heading">External Links:&nbsp;</span><span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=JOlhBAABERAJ&q=http://patft.uspto.gov/netacgi/nph-Parser%3FSect2%3DPTO1%26Sect2%3DHITOFF%26p%3D1%26u%3D/netahtml/PTO/search-bool.html%26r%3D1%26f%3DG%26l%3D50%26d%3DPALL%26RefSrch%3Dyes%26Query%3DPN/6617110&usg=AFQjCNHcGdyFJinSpGODNXxjopshVuI2CA">USPTO</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=JOlhBAABERAJ&q=http://assignments.uspto.gov/assignments/q%3Fdb%3Dpat%26pat%3D6617110&usg=AFQjCNFxy6ol6uVahD1G8xiWOxNv9F8aGg">USPTO Assignment</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=JOlhBAABERAJ&q=http://worldwide.espacenet.com/publicationDetails/biblio%3FCC%3DUS%26NR%3D6617110B1%26KC%3DB1%26FT%3DD&usg=AFQjCNFuEi62terX4f_dzuvU2lsbEoVUQg">Espacenet</a></span></span></td></tr><tr class="patent-bibdata-group-spacer"></tr></table><div class="number-and-title"><span class="patent-title"><invention-title mxw-id="PT55143302" lang="EN" load-source="patent-office">Cells immortalized with telomerase reverse transcriptase for use in drug screening</invention-title></span><br><span class="patent-number">US 6617110 B1</span></div><div class="patent-section patent-abstract-section"><div class="patent-section-header"><span class="patent-section-title">Abstract</span></div><div class="patent-text"><abstract mxw-id="PA50548045" lang="EN" load-source="patent-office"> <div class="abstract">The invention provides compositions and methods related to human telomerase reverse transcriptase (hTRT), the catalytic protein subunit of human telomerase. The polynucleotides and polypeptides of the invention are useful for diagnosis, prognosis and treatment of human diseases, for changing the proliferative capacity of cells and organisms, and for identification and screening of compounds and treatments useful for treatment of diseases such as cancers.</div>
  </abstract></div></div><div class="patent-section patent-drawings-section"><div class="patent-section-header"><span class="patent-section-title">Images<span class="patent-section-count">(103)</span></span></div><div class="patent-drawings-body"><div class="patent-drawings-carousel"><div class="drawings"><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00001.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00001.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00002.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00002.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00003.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00003.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00004.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00004.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00005.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00005.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00006.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00006.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00007.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00007.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00008.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00008.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00009.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00009.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00010.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00010.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00011.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00011.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00012.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00012.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00013.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00013.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00014.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00014.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00015.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00015.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00016.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00016.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00017.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00017.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00018.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00018.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00019.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00019.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00020.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00020.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00021.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00021.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00022.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00022.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00023.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00023.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00024.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00024.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00025.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00025.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00026.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00026.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00027.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00027.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00028.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00028.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00029.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00029.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00030.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00030.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00031.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00031.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00032.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00032.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00033.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00033.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00034.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00034.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00035.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00035.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00036.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00036.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00037.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00037.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00038.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00038.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00039.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00039.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00040.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00040.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00041.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00041.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00042.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00042.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00043.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00043.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00044.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00044.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00045.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00045.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00046.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00046.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00047.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00047.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00048.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00048.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00049.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00049.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00050.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00050.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00051.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00051.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00052.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00052.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00053.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00053.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00054.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00054.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00055.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00055.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00056.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00056.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00057.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00057.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00058.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00058.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00059.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00059.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00060.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00060.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00061.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00061.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00062.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00062.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00063.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00063.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00064.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00064.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00065.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00065.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00066.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00066.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00067.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00067.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00068.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00068.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00069.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00069.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00070.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00070.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00071.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00071.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00072.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00072.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00073.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00073.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00074.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00074.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00075.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00075.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00076.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00076.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00077.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00077.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00078.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00078.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00079.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00079.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00080.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00080.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00081.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00081.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00082.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00082.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00083.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00083.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00084.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00084.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00085.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00085.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00086.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00086.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00087.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00087.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00088.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00088.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00089.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00089.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00090.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00090.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00091.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00091.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00092.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00092.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00093.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00093.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00094.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00094.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00095.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00095.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00096.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00096.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00097.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00097.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00098.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00098.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00099.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00099.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00100.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00100.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00101.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00101.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00102.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00102.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6617110B1/US06617110-20030909-D00103.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6617110B1/US06617110-20030909-D00103.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div></div></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img" alt="Previous page"src="/googlebooks/images/kennedy/page_left.png"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img" alt="Next page"src="/googlebooks/images/kennedy/page_right.png"width="21" height="21" /></div></div></div><div class="patent-post-drawings"></div><div class="patent-section patent-claims-section"><div class="patent-section-header"><span class="patent-section-title">Claims<span class="patent-section-count">(39)</span></span></div><div class="patent-text"><div mxw-id="PCLM8543419" lang="EN" load-source="patent-office" class="claims">
    <claim-statement>What is claimed is: </claim-statement> <div class="claim"> <div num="1" id="US-6617110-B1-CLM-00001" class="claim">
      <div class="claim-text">1. A method of drug screening or validation of a drug, comprising:</div>
      <div class="claim-text">a) obtaining a drug or drug candidate, </div>
      <div class="claim-text">b) obtaining a cultured mammalian cell comprising a nucleic acid sequence that encodes a telomerase reverse transcriptase protein, variant, or fragment, wherein said variant, or fragment, has telomerase catalytic activity when complexed with a telomerase RNA, wherein the nucleic acid sequence hybridizes under stringent conditions to a polynucleotide having a sequence complementary to SEQ ID NO: 1, and wherein the expression of the protein, variant, or fragment in the cell increases the number of divisions the cell can undergo before senescence; </div>
      <div class="claim-text">c) administering the drug or drug candidate to the cultured cell; and </div>
      <div class="claim-text">d) determining if the drug or candidate has an effect on the cell. </div>
    </div>
    </div> <div class="claim-dependent"> <div num="2" id="US-6617110-B1-CLM-00002" class="claim">
      <div class="claim-text">2. The method of <claim-ref idref="US-6617110-B1-CLM-00001">claim 1</claim-ref>, wherein the cell is a human cell.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="3" id="US-6617110-B1-CLM-00003" class="claim">
      <div class="claim-text">3. The method cell of <claim-ref idref="US-6617110-B1-CLM-00002">claim 2</claim-ref>, wherein the cell further comprises a selectable marker gene.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="4" id="US-6617110-B1-CLM-00004" class="claim">
      <div class="claim-text">4. The method of <claim-ref idref="US-6617110-B1-CLM-00002">claim 2</claim-ref>, wherein the nucleic acid comprises a constitutive promoter.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="5" id="US-6617110-B1-CLM-00005" class="claim">
      <div class="claim-text">5. The method of <claim-ref idref="US-6617110-B1-CLM-00002">claim 2</claim-ref>, wherein the nucleic acid comprises an inducible promoter.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="6" id="US-6617110-B1-CLM-00006" class="claim">
      <div class="claim-text">6. The method of <claim-ref idref="US-6617110-B1-CLM-00002">claim 2</claim-ref>, wherein the cell is a liver cell.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="7" id="US-6617110-B1-CLM-00007" class="claim">
      <div class="claim-text">7. The method of <claim-ref idref="US-6617110-B1-CLM-00006">claim 6</claim-ref>, wherein the cell is a hepatocyte.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="8" id="US-6617110-B1-CLM-00008" class="claim">
      <div class="claim-text">8. The method of <claim-ref idref="US-6617110-B1-CLM-00002">claim 2</claim-ref>, wherein the cell is a nerve cell.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="9" id="US-6617110-B1-CLM-00009" class="claim">
      <div class="claim-text">9. The method of <claim-ref idref="US-6617110-B1-CLM-00008">claim 8</claim-ref>, wherein the cell is a glial cell, astrocyte, or oligodendrocyte.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="10" id="US-6617110-B1-CLM-00010" class="claim">
      <div class="claim-text">10. The method of <claim-ref idref="US-6617110-B1-CLM-00008">claim 8</claim-ref>, wherein the cell is a neuron of the central nervous system.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="11" id="US-6617110-B1-CLM-00011" class="claim">
      <div class="claim-text">11. The method of <claim-ref idref="US-6617110-B1-CLM-00010">claim 10</claim-ref>, wherein the cell is a cholinergic or adrenergic cell.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="12" id="US-6617110-B1-CLM-00012" class="claim">
      <div class="claim-text">12. The method of <claim-ref idref="US-6617110-B1-CLM-00002">claim 2</claim-ref>, wherein the cell is a retinal pigmented epithelial cell.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="13" id="US-6617110-B1-CLM-00013" class="claim">
      <div class="claim-text">13. The method of <claim-ref idref="US-6617110-B1-CLM-00002">claim 2</claim-ref>, wherein the cell is a contractile cell.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="14" id="US-6617110-B1-CLM-00014" class="claim">
      <div class="claim-text">14. The method of <claim-ref idref="US-6617110-B1-CLM-00013">claim 13</claim-ref>, wherein the cell is a heart muscle cell or smooth muscle cell.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="15" id="US-6617110-B1-CLM-00015" class="claim">
      <div class="claim-text">15. The method of <claim-ref idref="US-6617110-B1-CLM-00002">claim 2</claim-ref>, wherein the cell is a fat cell.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="16" id="US-6617110-B1-CLM-00016" class="claim">
      <div class="claim-text">16. The method of <claim-ref idref="US-6617110-B1-CLM-00002">claim 2</claim-ref>, wherein the cell is a fibroblast.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="17" id="US-6617110-B1-CLM-00017" class="claim">
      <div class="claim-text">17. The method of <claim-ref idref="US-6617110-B1-CLM-00002">claim 2</claim-ref>, wherein the cell is a vascular endothelial cell.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="18" id="US-6617110-B1-CLM-00018" class="claim">
      <div class="claim-text">18. The method of <claim-ref idref="US-6617110-B1-CLM-00002">claim 2</claim-ref>, wherein the cell is a hormone secreting cell.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="19" id="US-6617110-B1-CLM-00019" class="claim">
      <div class="claim-text">19. The method of <claim-ref idref="US-6617110-B1-CLM-00018">claim 18</claim-ref>, wherein the cell secretes insulin or glucagon.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="20" id="US-6617110-B1-CLM-00020" class="claim">
      <div class="claim-text">20. The method of <claim-ref idref="US-6617110-B1-CLM-00018">claim 18</claim-ref>, wherein the cell is a pituitary cell, thyroid hormone secreting cell, or adrenal cell.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="21" id="US-6617110-B1-CLM-00021" class="claim">
      <div class="claim-text">21. The method of <claim-ref idref="US-6617110-B1-CLM-00002">claim 2</claim-ref>, wherein the cell is a fat storing cell.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="22" id="US-6617110-B1-CLM-00022" class="claim">
      <div class="claim-text">22. The method of <claim-ref idref="US-6617110-B1-CLM-00002">claim 2</claim-ref>, wherein the cell is an epithelial or mucosal cell.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="23" id="US-6617110-B1-CLM-00023" class="claim">
      <div class="claim-text">23. The method of <claim-ref idref="US-6617110-B1-CLM-00022">claim 22</claim-ref>, wherein the cell is an oral cavity cell, stomach cell, or intestinal cell.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="24" id="US-6617110-B1-CLM-00024" class="claim">
      <div class="claim-text">24. The method of <claim-ref idref="US-6617110-B1-CLM-00022">claim 22</claim-ref>, wherein the cell is a mammary gland, uterus, or prostate cell.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="25" id="US-6617110-B1-CLM-00025" class="claim">
      <div class="claim-text">25. The method of <claim-ref idref="US-6617110-B1-CLM-00022">claim 22</claim-ref>, wherein the cell is an air space epithelial cell or the lung.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="26" id="US-6617110-B1-CLM-00026" class="claim">
      <div class="claim-text">26. The method of <claim-ref idref="US-6617110-B1-CLM-00002">claim 2</claim-ref>, wherein the cell is a tubular cell of the kidney.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="27" id="US-6617110-B1-CLM-00027" class="claim">
      <div class="claim-text">27. The method of <claim-ref idref="US-6617110-B1-CLM-00002">claim 2</claim-ref>, wherein the cell is a blood cell or a cell of the immune system,</div>
    </div>
    </div> <div class="claim-dependent"> <div num="28" id="US-6617110-B1-CLM-00028" class="claim">
      <div class="claim-text">28. The method of <claim-ref idref="US-6617110-B1-CLM-00027">claim 27</claim-ref>, wherein the cell is a T or B lymphocyte.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="29" id="US-6617110-B1-CLM-00029" class="claim">
      <div class="claim-text">29. The method of <claim-ref idref="US-6617110-B1-CLM-00027">claim 27</claim-ref>, wherein the cell is a mast cell or eosinophil.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="30" id="US-6617110-B1-CLM-00030" class="claim">
      <div class="claim-text">30. The method of <claim-ref idref="US-6617110-B1-CLM-00027">claim 27</claim-ref>, wherein the cell is a monocyte or macrophage.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="31" id="US-6617110-B1-CLM-00031" class="claim">
      <div class="claim-text">31. The method of <claim-ref idref="US-6617110-B1-CLM-00002">claim 2</claim-ref>, wherein the cell is an osteoblast, osteocyte, or osteoclast.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="32" id="US-6617110-B1-CLM-00032" class="claim">
      <div class="claim-text">32. The method of <claim-ref idref="US-6617110-B1-CLM-00002">claim 2</claim-ref>, wherein the cell is a chondrocyte or synovial cell.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="33" id="US-6617110-B1-CLM-00033" class="claim">
      <div class="claim-text">33. The method of <claim-ref idref="US-6617110-B1-CLM-00002">claim 2</claim-ref>, wherein the cell is a stem cell.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="34" id="US-6617110-B1-CLM-00034" class="claim">
      <div class="claim-text">34. The method of <claim-ref idref="US-6617110-B1-CLM-00033">claim 33</claim-ref>, wherein the cell is an embryonic stem cell.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="35" id="US-6617110-B1-CLM-00035" class="claim">
      <div class="claim-text">35. The method of <claim-ref idref="US-6617110-B1-CLM-00033">claim 33</claim-ref>, wherein the cell is an embryonic germ cell.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="36" id="US-6617110-B1-CLM-00036" class="claim">
      <div class="claim-text">36. The method of <claim-ref idref="US-6617110-B1-CLM-00033">claim 33</claim-ref>, wherein the cell is an adult stem cell.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="37" id="US-6617110-B1-CLM-00037" class="claim">
      <div class="claim-text">37. The method of <claim-ref idref="US-6617110-B1-CLM-00002">claim 2</claim-ref>, wherein the nucleic acid encodes a full-length, naturally occurring human telomerase reverse transcriptase.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="38" id="US-6617110-B1-CLM-00038" class="claim">
      <div class="claim-text">38. The method of <claim-ref idref="US-6617110-B1-CLM-00002">claim 2</claim-ref>, wherein the nucleic acid encodes a full-length, naturally occurring human telomerase reverse transcriptase having the amino acid sequence of SEQ ID NO: 2.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="39" id="US-6617110-B1-CLM-00039" class="claim">
      <div class="claim-text">39. The method of <claim-ref idref="US-6617110-B1-CLM-00001">claim 1</claim-ref>, comprising determining whether the drug or drug candidate is lethal to the cell.</div>
    </div>
  </div> </div></div></div><div class="patent-section patent-description-section"><div class="patent-section-header"><span class="patent-section-title">Description</span></div><div class="patent-text"><div mxw-id="PDES53984241" lang="EN" load-source="patent-office" class="description">
    <p>This application is a continuation of U.S. patent application Ser. No. 08/974,549 filed Nov. 19, 1997, U.S. Pat. No. 6,166,178, which is a continuation-in-part of U.S. patent application Ser. No. 08/915,503, filed Aug. 14, 1997, abandoned, which is a continuation-in-part of U.S. patent application Ser. No. 08/912,951, filed Aug. 14, 1997, U.S. Pat. No. 6,475,789 and a continuation-in-part of U.S. patent application Ser. No. 08/911,312, filed Aug. 14, 1997, abandoned, all three of which are continuations-in-part of U.S. patent application Ser. No. 08/854,050, filed May 9, 1997, U.S. Pat. No. 6,261,836, which is a continuation-in-part of U.S. patent application Ser. No. 08/851,843, filed May 6, 1997, U.S. Pat. No. 6,093,809, which is a continuation-in-part of U.S. patent application Ser. No. 08/846,017, filed Apr. 25, 1997, abandoned. This application also claims priority to Patent Convention Treaty Patent Application Serial No.: PCT/US97/17885 (published on Apr. 9, 1998 as WO 98/14593) and to Patent Convention Treaty Patent Application Serial No. PCT/US97/17618 (published on Apr. 9, 1998 as WO98/14592), both designating the U.S. and filed in the U.S. Receiving Office on Oct. 1, 1997. Each of the aforementioned applications is explicitly incorporated herein by reference in its entirety and for all purposes, This application also incorporates by reference copending U.S. patent application Ser. No. 08/974,584, filed Nov. 19, 1997, in its entirety and for all purposes.</p>
    <p>This invention was made with Government support under Grant No. GM28039, awarded by the National Institutes of Health. The Government has certain rights in this invention.</p>
    <heading>FIELD OF THE INVENTION</heading> <p>The present invention is related to novel nucleic acids encoding the catalytic subunit of telomerase and related polypeptides. In particular, the present invention is directed to the catalytic subunit of human telomerase. The invention provides methods and compositions relating to medicine, molecular biology, chemistry, pharmacology, and medical diagnostic and prognostic technology.</p>
    <heading>BACKGROUND OF THE INVENTION</heading> <p>The following discussion is intended to introduce the field of the present invention to the reader. The citation of various references in this section is not to be construed as an admission of prior invention.</p>
    <p>It has long been recognized that complete replication of the ends of eukaryotic chromosomes requires specialized cell components (Watson, 1972, <i>Nature New Biol., </i>239:197; Olovnikov, 1973, <i>J. Theor. Biol., </i>41:181). Replication of a linear DNA strand by conventional DNA polymerases requires an RNA primer, and can proceed only 5′ to 3′. When the RNA bound at the extreme 5′ ends of eukaryotic chromosomal DNA strands is removed, a gap is introduced, leading to a progressive shortening of daughter strands with each round of replication. This shortening of telomeres, the protein-DNA structures physically located on the ends of chromosomes, is thought to account for the phenomenon of cellular senescence or aging (see, e.g., Goldstein, 1990, <i>Science </i>249:1129; Martin et al., 1979, <i>Lab. Invest. </i>23:86; Goldstein et al., 1969, <i>Proc. Natl. Acad. Sci. USA </i>64:155; and Schneider and Mitsui, 1976, <i>Proc. Natl. Acad. Sci. USA, </i>73:3584) of normal human somatic cells in vitro and in vivo.</p>
    <p>The length and integrity of telomeres is thus related to entry of a cell into a senescent stage (i.e., loss of proliferative capacity). Moreover, the ability of a cell to maintain (or increase) telomere length may allow a cell to escape senescence, i.e., to become immortal.</p>
    <p>The structure of telomeres and telomeric DNA has been investigated in numerous systems (see, e.g, Harley and Villeponteau, 1995, <i>Curr. Opin. Genet. Dev. </i>5:249). In most organisms, telomeric DNA consists of a tandem array of very simple sequences; in humans and other vertebrates telomeric DNA consists of hundreds to thousands of tandem repeats of the sequence TTAGGG. Methods for determining and modulating telomere length in cells are described in PCT Publications WO 93/23572 and WO 96/41016.</p>
    <p>The maintenance of telomeres is a function of a telomere-specific DNA polymerase known as telomerase. Telomerase is a ribonucleoprotein (RNP) that uses a portion of its RNA moiety as a template for telomere repeat DNA synthesis (Morin, 1997, <i>Eur. J. Cancer </i>33:750; Yu et al., 1990, <i>Nature </i>344:126; Singer and Gottschling, 1994, <i>Science </i>266:404; Autexier and Greider, 1994, <i>Genes Develop., </i>8:563; Gilley et al., 1995, <i>Genes Develop., </i>9:2214; McEachern and Blackburn, 1995, <i>Nature </i>367:403; Blackburn, 1992, <i>Ann. Rev. Biochem., </i>61:113; Greider, 1996, <i>Ann. Rev. Biochem., </i>65:337). The RNA components of human and other telomerases have been cloned and characterized (see, PCT Publication WO 96/01835 and Feng et al., 1995, <i>Science </i>269:1236). However, the characterization of the protein components of telomerase has been difficult. In part, this is because it has proved difficult to purify the telomerase RNP, which is present in extremely low levels in cells in which it is expressed. For example, it has been estimated that human cells known to express high levels of telomerase activity may have only about one hundred molecules of the enzyme per cell.</p>
    <p>Consistent with the relationship of telomeres and telomerase to the proliferative capacity of a cell (i.e., the ability of the cell to divide indefinitely), telomerase activity is detected in immortal cell lines and an extraordinarily diverse set of tumor tissues, but is not detected (i.e., was absent or below the assay threshold) in normal somatic cell cultures or normal tissues adjacent to a tumor (see, U.S. Pat. Nos. 5,629,154; 5,489,508; 5,648,215; and 5,639,613; see also, Morin, 1989, <i>Cell </i>59: 521; Shay and Bacchetti 1997, <i>Eur. J. Cancer </i>33:787; Kim et al., 1994, <i>Science </i>266:2011; Counter et al., 1992, <i>EMBO J. </i>11:1921; Counter et al., 1994, <i>Proc. Natl. Acad. Sci. U.S.A. </i>91, 2900; Counter et al., 1994, <i>J. Virol. </i>68:3410). Moreover, a correlation between the level of telomerase activity in a tumor and the likely clinical outcome of the patient has been reported (e.g., U.S. Pat. No. 5,639,613, supra, Langford et al., 1997, <i>Hum. Pathol. </i>28:416). Telomerase activity has also been detected in human germ cells, proliferating stem or progenitor cells, and activated lymphocytes. In somatic stem or progenitor cells, and in activated lymphocytes, telomerase activity is typically either very low or only transiently expressed (see, Chiu et al., 1996, <i>Stem Cells </i>14:239; Bodnar et al., 1996, <i>Exp. Cell Res. </i>228:58; Taylor et al., 1996, <i>J. Invest. Dermatology </i>106: 759).</p>
    <p>Human telomerase is an ideal target for diagnosing and treating human diseases relating to cellular proliferation and senescence, such as cancer. Methods for diagnosing and treating cancer and other telomerase-related diseases in humans are described in U.S. Pat. Nos. 5,489,508, 5,639,613, and 5,645,986. Methods for predicting tumor progression by monitoring telomerase are described in U.S. Pat. No. 5,639,613. The discovery and characterization of the catalytic protein subunit of human telomerase would provide additional useful assays for telomerase and for disease diagnosis and therapy. Moreover, cloning and determination of the primary sequence of the catalytic protein subunit would allow more effective therapies for human cancers and other diseases related to cell proliferative capacity and senescence.</p>
    <heading>BRIEF SUMMARY OF THE INVENTION</heading> <p>The present invention provides an isolated, substantially pure, or recombinant protein preparation of a telomerase reverse transcriptase protein, or a variant thereof, or a fragment thereof. In one embodiment the protein is characterized as having a defined motif that has an amino acid sequence:</p>
    <p>Trp-R<sub>1</sub>-X<sub>7</sub>-R<sub>1</sub>-R<sub>1</sub>-R<sub>2</sub>-X-Phe-Phe-Tyr-X-Thr-Glu-X<sub>8-9</sub>-R<sub>3</sub>-R<sub>3</sub>-Arg-R<sub>4</sub>-X<sub>2</sub>-Trp (SEQ ID NOS:11 and 12)</p>
    <p>where X is any amino acid and a subscript refers to the number of consecutive residues, R<sub>1 </sub>is leucine or isoleucine, R<sub>2 </sub>is glutamine or arginine, R<sub>3 </sub>is phenylalanine or tyrosine, and R<sub>4 </sub>is lysine or histidine. In one embodiment the protein has a sequence of human TRT. In other embodiments, the invention relates to peptides and polypeptides sharing substantial sequence identity with a subsequence of such proteins.</p>
    <p>In a related embodiment the invention provides an isolated, substantially pure or recombinant nucleic acid that encodes a telomerase reverse transcriptase protein. In one embodiment the nucleic acid encodes a protein comprising an amino acid sequence (SEQ ID NOS:11 and 12):</p>
    <p>Trp-R<sub>1</sub>-X<sub>7</sub>-R<sub>1</sub>-R<sub>1</sub>-R<sub>2</sub>-X-Phe-Phe-Tyr-X-Thr-Glu-X<sub>8-9</sub>-R<sub>3</sub>-R<sub>3</sub>-Arg-R<sub>4</sub>-X<sub>2</sub>-Trp.</p>
    <p>In another embodiment, the nucleic acid has a sequence that encodes the human TRT protein. In other embodiments, the invention relates to oligonucleotides and polynucleotides sharing substantial sequence identity or complementarity with a subsequence of such nucleic acids.</p>
    <p>In one embodiment, the invention relates to human telomerase reverse transcriptase (hTRT) protein. Thus, in one embodiment, the invention provides an isolated, substantially pure, or recombinant protein preparation of an hTRT protein, or a variant thereof, or a fragment thereof. In one embodiment, the protein is characterized by having an amino acid sequence with at least about 75% or at least about 80% sequence identity to the hTRT protein of FIG. 17 (SEQ ID NO:2), or a variant thereof, or a fragment thereof. In a related aspect, the hTRT protein has the sequence of SEQ ID NO:2. In some embodiments, the protein has one or more telomerase activities, such as catalytic activity. In one embodiment, the hTRT protein fragment has at least 6 amino acid residues. In other embodiments, the hTRT protein fragment has at least 8, at least about 10, at least about 12, at least about 15 or at least about 20 contiguous amino acid residues of a naturally occurring hTRT polypeptide. In still other embodiments, the hTRT protein fragment has at least about 50 or at least about 100 amino acid residues.</p>
    <p>The invention also provides a composition comprising an hTRT protein and an RNA. The RNA may be a telomerase RNA, such as a human telomerase RNA. In one embodiment, the hTRT protein and the human telomerase RNA (hTR) form a ribonucleoprotein complex with a telomerase activity.</p>
    <p>In one embodiment, the invention provides isolated human telomerase comprising hTRT protein, such as a substantially pure human telomerase comprising hTRT protein and comprising hTR. In one embodiment, the telomerase is at least about 95% pure. The telomerase may be isolated from a cell, such as a recombinant host cell in or a cell that expresses telomerase activity.</p>
    <p>In another aspect, the invention provides an isolated, synthetic, substantially pure, or recombinant polynucleotide comprising a nucleic acid sequence that encodes an hTRT protein. In one embodiment, the polynucleotide has a nucleotide sequence encoding an hTRT protein that has an amino acid sequence as set forth in FIG. 17 (SEQ ID NO:2) or a sequence that comprises one or more conservative amino acid (or codon) substitutions or one or more activity-altering amino acid (or codon) substitutions in said amino acid sequence. In a related aspect, the polynucleotide hybridizes under stringent conditions to a polynucleotide having the sequence as set forth in FIG. 16 (SEQ ID NO:1). In another related aspect, the nucleotide sequence of the polynucleotide has a smallest sum probability of less than about 0.5 when compared to a nucleotide sequence as set forth in FIG. 16 (SEQ ID NO:1) using BLAST algorithm with default parameters.</p>
    <p>In another aspect, the invention provides a polynucleotide having a promoter sequence operably linked to the sequence encoding the hTRT protein. The promoter may be a promoter other than the naturally occurring hTRT promoter. In a related aspect, the invention provides an expression vector comprising the promoter of the hTRT.</p>
    <p>The invention also provides an isolated, synthetic, substantially pure, or recombinant polynucleotide that is at least ten nucleotides in length and comprises a contiguous sequence of at least ten nucleotides that is identical or exactly complementary to a contiguous sequence in a naturally occurring hTRT gene or hTRT mRNA. In some embodiments the polynucleotide is an RNA, a DNA, or contains one or more non-naturally occurring, synthetic nucleotides. In one aspect, the polynucleotide is identical or exactly complementary to the contiguous sequence of at least ten contiguous nucleotides in a naturally occurring hTRT gene or hTRT mRNA. For example, the polynucleotide may be an antisense polynucleotide. In one embodiment, the antisense polynucleotide comprises at least about 20 nucleotides.</p>
    <p>The invention further provides a method of preparing recombinant telomerase by contacting a recombinant hTRT protein with a telomerase RNA component under conditions such that said recombinant protein and said telomerase RNA component associate to form a telomerase enzyme capable of catalyzing the addition of nucleotides to a telomerase substrate. In one embodiment, the hTRT protein has a sequence as set forth in FIG. 17 (SEQ ID NO:2). The hTRT protein may be produced in an in vitro expression system and mixed with a telomerase RNA or, in another embodiment, the telomerase RNA can be co-expressed in the in vitro expression system. In one embodiment the telomerase RNA is hTR. In an alternative embodiment, the contacting occurs in a cell, such as a human cell. In one embodiment, the cell does not have telomerase activity prior to the contacting of the hTRT and the RNA, or the introduction, such as by transfection, of an hTRT polynucleotide. In one embodiment, the telomerase RNA is expressed naturally by said cell.</p>
    <p>The invention also provides a cell, such as a human, mouse, or yeast cell, containing the recombinant polynucleotides of the invention such as a polynucleotide with an hTRT protein coding sequence operably linked a promoter. In particular aspects, the cell is a vertebrate cell, such as a cell from a mammal, for example a human, and has an increased proliferative capacity relative to a cell that is otherwise identical but does not comprise the recombinant polynucleotide or has an increased telomerase activity level relative to a cell that is otherwise identical but does not comprise the recombinant polynucleotide. In some embodiments the cell is immortal.</p>
    <p>In related embodiments, the invention provides organisms and cells comprising a polynucleotide encoding a human telomerase reverse transcriptase polypeptide, such as a transgenic non-human organism such as a yeast, plant, bacterium, or a non-human animal, for example, a mouse. The invention also provides for transgenic animals and cells from which an hTRT gene has been deleted (knocked-out) or mutated such that the gene does not express a naturally occurring hTRT gene product. Thus, in alternative embodiments, the transgenic non-human animal has a mutated telomerase gene, is an animal deficient in a telomerase activity, is an animal whose TRT deficiency is a result of a mutated gene encoding a TRT having a reduced level of a telomerase activity compared to a wild-type TRT and is an animal having a mutated TRT gene with one or more mutations, including missense mutations nonsense mutations, insertions, or deletions.</p>
    <p>The invention also provides an isolated or recombinant antibody, or fragment thereof, that specifically binds to an hTRT protein. In one embodiment, the antibody binds with an affinity of at least about 10<sup>8 </sup>M<sup>−1</sup>. The antibody may be monoclonal or may be a polyclonal composition, such as a polyclonal antisera. In a related aspect, the invention provides a cell capable of secreting the antibody, such as a hybridoma.</p>
    <p>The invention also provides a method for determining whether a compound or treatment is a modulator of a telomerase reverse transcriptase activity or hTRT expression. The method involves detecting or monitoring a change in activity or expression in a cell, animal or composition comprising an hTRT protein or polynucleotide following administration of the compound or treatment. In one embodiment, the method includes the steps of providing a TRT composition, contacting the TRT with the test compound and measuring the activity of the TRT, where a change in TRT activity in the presence of the test compound is an indicator that the test compound modulates TRT activity. In certain embodiments, the composition is a cell, an organism, a transgenic organism or an in vitro system, such as an expression system, which contains a recombinant polynucleotide encoding an hTRT polypeptide. Thus, the hTRT of the method may be a product of in vitro expression. In various embodiments the detection of telomerase activity or expression may be by detecting a change in abundance of an hTRT gene product, monitoring incorporation of a nucleotide label into a substrate for telomerase, monitoring hybridization of a probe to an extended telomerase substrate, monitoring amplification of an extended telomerase substrate, monitoring telomere length of a cell exposed to the test compound, monitoring the loss of the ability of the telomerase to bind to a chromosome, or measuring the accumulation or loss of telomere structure.</p>
    <p>In one aspect, the invention provides a method of detecting an hTRT gene product in a biological sample by contacting the biological sample with a probe that specifically binds the gene product, wherein the probe and the gene product form a complex, and detecting the complex, where the presence of the complex is correlated with the presence of the hTRT gene product in the biological sample. The gene product may be RNA, DNA or a polypeptide. Examples of probes that may be used for detection include, but are not limited to, nucleic acids and antibodies.</p>
    <p>In one embodiment, the gene product is a nucleic acid which is detected by amplifying the gene and detecting the amplification product, where the presence of the complex or amplification product is correlated with the presence of the hTRT gene product in the biological sample.</p>
    <p>In one embodiment, the biological sample is from a patient, such as a human patient. In another embodiment the biological sample includes at least one cell from an in vitro cell culture, such as a human cell culture.</p>
    <p>The invention further provides a method of detecting the presence of at least one immortal or telomerase positive human cell in a biological sample comprising human cells by obtaining the biological sample comprising human cells; and detecting the presence in the sample of a cell having a high level of an hTRT gene product, where the presence of a cell having a high level of the hTRT gene product is correlated with the presence of immortal or telomerase positive cells in the biological sample.</p>
    <p>The invention also provides a method for diagnosing a telomerase-related condition in a patient by obtaining a cell or tissue sample from the patient, determining the amount of an hTRT gene product in the cell or tissue; and comparing the amount of hTRT gene product in the cell or tissue with the amount in a healthy cell or tissue of the same type, where a different amount of hTRT gene product in the sample from the patient and the healthy cell or tissue is diagnostic of a telomerase-related condition. In one embodiment the telomerase-related condition is cancer and a greater amount of hTRT gene product is detected in the sample.</p>
    <p>The invention further provides a method of diagnosing cancer in a patient by obtaining a biological sample from the patient, and detecting a hTRT gene product in the patient sample, where the detection of the hTRT gene product in the sample is correlated with a diagnosis of cancer.</p>
    <p>The invention further provides a method of diagnosing cancer in a patient by obtaining a patient sample, determining the amount of hTRT gene product in the patient sample; and comparing the amount of hTRT gene product with a normal or control value, where an amount of the hTRT gene product in the patient that is greater than the normal or control value is diagnostic of cancer.</p>
    <p>The invention also provides a method of diagnosing cancer in a patient, by obtaining a patient sample containing at least one cell; determining the amount of an hTRT gene product in a cell in the sample; and comparing the amount of hTRT gene product in the cell with a normal value for the cell, wherein an amount of the hTRT gene product greater than the normal value is diagnostic of cancer. In one embodiment, the sample is believed to contain at least one malignant cell.</p>
    <p>The invention also provides a method for a prognosing a cancer patient by determining the amount of hTRT gene product in a cancer cell obtained from the patient; and comparing the amount of hTRT in the cancer cell with a prognostic value of hTRT consistent with a prognosis for the cancer; where an amount of hTRT in the sample that is at the prognostic value provides the particular prognosis.</p>
    <p>The invention also provides a method for monitoring the ability of an anticancer treatment to reduce the proliferative capacity of cancer cells in a patient, by making a first measurement of the amount of an hTRT gene product in at least one cancer cell from the patient; making a second measurement of the level of the hTRT gene product in at least one cancer cell from the patient, wherein the anticancer treatment is administered to the patient before the second measurement; and comparing the first and second measurements, where a lower level of the hTRT gene product in the second measurement is correlated with the ability of an anticancer treatment to reduce the proliferative capacity of cancer cells in the patient.</p>
    <p>The invention also provides kits for the detection of an hTRT gene or gene product. In one embodiment, the kit includes a container including a molecule selected from an hTRT nucleic acid or subsequence thereof, an hTRT polypeptide or subsequence thereof, and an anti-hTRT antibody.</p>
    <p>The invention also provides methods of treating human diseases. In one embodiment, the invention provides a method for increasing the proliferative capacity of a vertebrate cell, such as a mammalian cell, by introducing a recombinant polynucleotide into the cell, wherein said polynucleotide comprises a sequence encoding an hTRT polypeptide. In one embodiment, the hTRT polypeptide has a sequence as shown in FIG. <b>17</b>. In one embodiment, the sequence is operably linked to a promoter. In one embodiment, the hTRT has telomerase catalytic activity. In one embodiment, the cell is human, such as a cell in a human patient. In an alternative embodiment, the cell is cultured in vitro. In a related embodiment, the cell is introduced into a human patient.</p>
    <p>The invention further provides a method for treating a human disease by introducing recombinant hTRT polynucleotide into at least one cell in a patient. In one embodiment, a gene therapy vector is used. In a related embodiment, the method further consists of introducing into the cell a polynucleotide comprising a sequence encoding hTR, for example, an hTR polynucleotide operably linked to a promoter.</p>
    <p>The invention also provides a method for increasing the proliferative capacity of a vertebrate cell, said method comprising introducing into the cell an effective amount of hTRT polypeptide. In one embodiment the hTRT polypeptide has telomerase catalytic activity. The invention further provides cells and cell progeny with increased proliferative capacity.</p>
    <p>The invention also provides a method for treating a condition associated with an elevated level of telomerase activity within a cell, comprising introducing into said cell a therapeutically effective amount of an inhibitor of said telomerase activity, wherein said inhibitor is an hTRT polypeptide or an hTRT polynucleotide. In one embodiment, the inhibitor is a polypeptide or polynucleotide comprising, e.g., at least a subsequence of a sequence shown in FIGS. 16, <b>17</b>, or <b>20</b>. In additional embodiments, the polypeptide or polynucleotide inhibits a TRT activity, such as binding of endogenous TRT to telomerase RNA.</p>
    <p>The invention also provides a vaccine comprising an hTRT polypeptide and an adjuvant. The invention also provides pharmacological compositions containing a pharmaceutically acceptable carrier and a molecule selected from: an hTRT polypeptide, a polynucleotide encoding an hTRT polypeptide, and an hTRT nucleic acid or subsequence thereof.</p>
    <heading>DESCRIPTION OF THE FIGURES</heading> <p>FIG. 1 shows highly conserved residues in TRT motifs from human (SEQ ID NO:13), <i>S. pombe </i>(tez1) (SEQ ID NO:14), <i>S. cerevisiae </i>(EST2) (SEQ ID NO:15) and <i>Euplotes aediculatus </i>(p123) (SEQ ID NO:16). Identical amino acids are indicated with an asterisk (*) [raised slightly], while the similar amino acid residues are indicated by a dot (•). Motif “0” in the figure is also called Motif T; Motif “3” is also called Motif A.</p>
    <p>FIG. 2 shows the location of telomerase-specific and RT-specific sequence motifs of telomerase proteins and other reverse transcriptases. Locations of telomerase-specific motif T and conserved RT motifs 1, 2 and A-E are indicated by boxes. The open rectangle labeled HIV-1 RT delineates the portion of this protein shown in FIG. <b>3</b>.</p>
    <p>FIG. 3 shows the crystal structure of the p66 subunit of HIV-1 reverse transcriptase (Brookhaven code 1HNV). The view is from the back of the right hand to enable all motifs to be shown.</p>
    <p>FIG. 4 shows multiple sequence alignment of telomerase RTs (Sp_Trt1p, <i>S. pombe </i>TRT (SEQ ID NOS:24-29) [also referred to herein as “tez1p”]; hTRT, human TRT (SEQ ID NOS:30-35); Ea_p123, Euplotes p123 (SEQ ID NOS:36-41); Sc_Est2p, <i>S. cerevisiae </i>Est2p) (SEQ ID NOS:42-48) and members of other RT families (Sc_al, cytochrome oxidase group II intron 1-encoded protein from <i>S. cerevisiae </i>mitochondria (SEQ ID NOS:51-56), Dm_TART, reverse transcriptase from <i>Drosophila melanogaster </i>TART non-LTR retrotransposable element) (SEQ ID NOS:57-63; HIV-1, human immunodeficiency virus reverse transcriptase (SEQ ID NOS:64-68)). TRT con (SEQ ID NOS:17-23) and RT con (SEQ ID NOS:49 and 50) represent consensus sequences for telomerase RTs and non-telomerase RTs. Amino acids are designated with an h, hydrophobic; p, polar; c, charged. Triangles show residues that are conserved among telomerase proteins but different in other RTs. The solid line below motif E highlights the primer grip region.</p>
    <p>FIG. 5 shows expression of hTRT RNA in telomerase-negative mortal cell strains and telomerase-positive immortal cell lines as described in Example 2.</p>
    <p>FIG. 6 shows a possible phylogenetic tree of telomerases and retroelements rooted with RNA-dependent RNA polymerases.</p>
    <p>FIG. 7 shows a restriction map of lambda clone Gφ5.</p>
    <p>FIG. 8 shows a map of chromosome 5p with the location of the STS marker D5S678 (located near the hTRT gene) indicated.</p>
    <p>FIG. 9 shows the construction of a hTRT promoter-reporter plasmid.</p>
    <p>FIGS. 10A and B, in two pages, shows coexpression in vitro of hTRT and hTR to produce catalytically active human telomerase.</p>
    <p>FIG. 11, in two pages, shows an alignment of sequences from four TRT proteins from human (hTRT; SEQ ID NOS:72-79), <i>S. pombe </i>Trt1 (spTRT; SEQ ID NOS:80-87), Euplotes p123 (Ea_p123; SEQ ID NOS:88-95), and <i>S. cerevisiae </i>EST2p TRT (Sc_Est2; SEQ ID NOS:96-104) and identifies motifs of interest. TRT con (SEQ ID NOS:69, 21, 70 and 71) shows a TRT consensus sequence. RT con (SEQ ID NOS:49 and 50) shows consensus residues for other reverse transcriptases. Consensus residues in upper case indicate absolute conservation in TRT proteins.</p>
    <p>FIG. 12 shows a Topoisomerase II cleavage site (SEQ ID NO: 108) and NFkB binding site motifs (NFkB_CS1=SEQ ID NO:105; NFkB-MHC-I.2=SEQ ID NO:106; NFkB_CS2=SEQ ID NO:107) in an hTRT intron, with the sequence shown corresponding to SEQ ID NO:7.</p>
    <p>FIG. 13, in two pages, shows the sequence of the DNA encoding the Euplotes 123 kDa telomerase protein subunit (Euplotes TRT; SEQ ID NO:109).</p>
    <p>FIG. 14 shows the amino acid sequence of the Euplotes 123 kDa telomerase protein subunit (Euplotes TRT protein; SEQ ID NO:110).</p>
    <p>FIG. 15, in five pages, shows the DNA (SEQ ID NO:111) and amino acid (SEQ ID NO:112) sequences of the <i>S. pombe </i>telomerase catalytic subunit (<i>S. pombe </i>TRT).</p>
    <p>FIG. 16, in two pages, shows the hTRT cDNA sequence, with the sequence shown corresponding to SEQ ID NO:1.</p>
    <p>FIG. 17 shows the hTRT protein encoded by the cDNA of FIG. <b>16</b>. The protein sequence shown corresponds to SEQ ID NO:2.</p>
    <p>FIG. 18 shows the sequence of clone 712562, with the sequence shown corresponding to SEQ ID NO:3.</p>
    <p>FIG. 19 shows a 259 residue protein encoded by clone 712562, with the sequence shown corresponding to SEQ ID NO:10.</p>
    <p>FIG. 20 shows, in seven pages, the sequence of a nucleic acid with an open reading frame encoding a Δ182 variant polypeptide, with the sequence shown corresponding to SEQ ID NO:4. This Figure also shows the amino acid sequence of this Δ182 variant polypeptide, with the amino acid sequence shown corresponding to SEQ ID NO:5.</p>
    <p>FIG. 21 shows, in six pages, sequence from an hTRT genomic clone, with the sequence shown corresponding to SEQ ID NO:6. Consensus motifs and elements are indicated, including sequences characteristic of a topoisomerase II cleavage site, NFκB binding sites, an Alu sequence. and other sequence elements.</p>
    <p>FIGS. 22A, B, C, D shows the effect of mutation of the TRT gene in yeast, as described in Example 1.</p>
    <p>FIG. 23 shows the sequence of EST AA281296, corresponding to SEQ ID NO:8.</p>
    <p>FIG. 24 shows the sequence of the 182 basepairs deleted in clone 712562, with the sequence shown corresponding to SEQ ID NO:9.</p>
    <p>FIG. 25 shows the results of an assay for telomerase activity from BJ cells transfected with an expression vector encoding an hTRT protein (pGRN133) or a control plasmid (pBBS212) as described in Example 13.</p>
    <p>FIGS. 26A and B is a schematic diagram of the affinity purification of telomerase showing the binding and displacement elution steps.</p>
    <p>FIG. 27 is a photograph of a Northern blot of telomerase preparations obtained during a purification protocol, as described in Example 1. Lane 1 contained 1.5 fmol telomerase RNA, lane 2 contained 4.6 fmol telomerase RNA, lane 3 contained 14 fmol telomerase RNA, lane 4 contained 41 fmol telomerase RNA, lane 5 contained nuclear extract (42 fmol telomerase), lane 6 contained Affi-Gel-heparin-purified telomerase (47 fmol telomerase), lane 7 contained affinity-purified telomerase (68 fmol), and lane 8 contained glycerol gradient-purified telomerase (35 fmol).</p>
    <p>FIG. 28 shows telomerase activity through a purification protocol.</p>
    <p>FIG. 29 is a photograph of a SDS-PAGE gel, showing the presence of an approximately 123 kDa polypeptide and an approximately 43 kDa doublet from <i>Euplotes aediculatus. </i> </p>
    <p>FIG. 30 is a graph showing the sedimentation coefficient of <i>Euplotes aediculatus </i>telomerase.</p>
    <p>FIG. 31 is a photograph of a polyacrylamide/urea gel with 36% formamide showing the substrate utilization of Euplotes telomerase.</p>
    <p>FIGS. 32A and B shows the putative alignments of telomerase RNA template (SEQ ID NO:113), and hairpin primers with telomerase RNA.</p>
    <p>FIG. 33 is a photograph of lanes 25-30 of the gel shown in FIG. 31, shown at a lighter exposure level (G<sub>4</sub>T<sub>4</sub>G<sub>4</sub>T<sub>4</sub>=SEQ ID NO:114).</p>
    <p>FIG. 34 shows the DNA sequence of the gene encoding the 43 kDa telomerase protein subunit from Euplotes (SEQ ID NO:115).</p>
    <p>FIG. 35 shows, in four pages, the DNA sequence (SEQ ID NO:115), as well as the amino acid sequences of all three open reading frames of the 43 kDa telomerase protein subunit from Euplotes (a=SEQ ID NOS:116-140; b=SEQ ID NOS:141-162; c=SEQ ID NOS:163-186).</p>
    <p>FIG. 36 shows a sequence comparison between the 123 kDa telomerase protein subunit of Euplotes (SEQ ID NO:187) (upper sequence) and the 80 kDa polypeptide subunit of <i>T. thermophila </i>(SEQ ID NO:188) (lower sequence).</p>
    <p>FIG. 37 shows a sequence comparison between the 123 kDa telomerase protein subunit of <i>E. aediculatus </i>(SEQ ID NO:189) (upper sequence) and the 95 kDa telomerase polypeptide of <i>T. thermophila </i>(SEQ ID NO:190) (lower sequence).</p>
    <p>FIG. 38 shows the best-fit alignment between a portion of the “La-domain” of the 43 kDa telomerase protein subunit of <i>E. aediculatus </i>(SEQ ID NO:191) (upper sequence) and a portion of the 95 kDa polypeptide subunit of <i>T. thermophila </i>(SEQ ID NO:192) (lower sequence).</p>
    <p>FIG. 39 shows the best-fit alignment between a portion of the “La-domain” of the 43 kDa telomerase protein subunit of <i>E. aediculatus </i>(SEQ ID NO:193) (upper sequence) and a portion of the 80 kDa polypeptide subunit of <i>T. thermophila </i>(SEQ ID NO:194) (lower sequence).</p>
    <p>FIG. 40 shows the alignment and motifs of the polymerase domain of the 123 kDa telomerase protein subunit of <i>E. aediculatus </i>(SEQ ID NOS:38-41) and the polymerase domains of various reverse transcriptases including a cytochrome oxidase group II intron 1-encoded protein from <i>S. cerevisiae </i>mitochondria (a1 S.c. (group II)) (SEQ ID NOS:204, 205, 54, 206, and 56), Dong (LINE) (SEQ ID NOS:200-203), and yeast ESTp (L8543.12) (SEQ ID NOS:45, 46, 211 and 212), HIV-RT (SEQ ID NOS:207-210) and consensus (SEQ ID NOS:195-199).</p>
    <p>FIG. 41 shows the alignment of a domain of the 43 kDa telomerase protein subunit (SEQ ID NO:213) with various La proteins (human La=SEQ ID NO:214; Xenopus LaA=SEQ ID NO:215; Drosophila La=SEQ ID NO:216; S.c. Lhplp=SEQ ID NO:217).</p>
    <p>FIG. 42 shows the nucleotide sequence encoding the <i>T. thermophila </i>80 kDa protein subunit.</p>
    <p>FIG. 43 shows the amino acid sequence of the <i>T. thermophila </i>80 kDa protein subunit (SEQ ID NO:219).</p>
    <p>FIG. 44 shows the nucleotide sequence encoding the <i>T. thermophila </i>95 kDa protein subunit (SEQ ID NO:220).</p>
    <p>FIG. 45 shows the amino acid sequence of the <i>T. thermophila </i>95 kDa protein subunit (SEQ ID NO:221).</p>
    <p>FIG. 46 shows the amino acid sequence of L8543.12 (“Est2p”) (SEQ ID NO:222).</p>
    <p>FIG. 47 shows the alignment of the amino acid sequence encoded by the Oxytricha PCR product (SEQ ID NO:223) with the Euplotes p123 sequence (SEQ ID NO:224).</p>
    <p>FIG. 48 shows the DNA sequence of Est2 (SEQ ID NO:225).</p>
    <p>FIG. 49 shows partial amino acid sequence from a cDNA clone encoding human telomerase peptide motifs (SEQ ID NO:13).</p>
    <p>FIG. 50 shows partial DNA sequence of a cDNA clone encoding human telomerase peptide motifs (SEQ ID NO:8).</p>
    <p>FIG. 51 shows the amino acid sequence of tez1, also called <i>S. pombe </i>trt (SEQ ID NO:112).</p>
    <p>FIG. 52 shows, in two pages, the DNA sequence of tez1 (SEQ ID NO:111). Intronic and other non-coding regions are shown in lower case and exons (i.e., coding regions) are shown in upper case.</p>
    <p>FIG. 53 shows the alignment of EST2p (SEQ ID NO:226), Euplotes (SEQ ID NO:227), and Tetrahymena SEQ ID NO:228) sequences, as well as consensus sequence (SEQ ID NOS:229-231).</p>
    <p>FIG. 54 shows the sequences of peptides (SEQ ID NOS:232-237) useful for production of anti-hTRT antibodies.</p>
    <p>FIGS. 55A and B is a schematic summary of the tez1<sup>+</sup> sequencing experiments.</p>
    <p>FIG. 56 shows two degenerate primers (SEQ ID NOS:238 and 241) used in PCR to identify the <i>S. pombe </i>homolog of the <i>E. aediculatus </i>p123 sequences (SEQ ID NOS:239 and 240).</p>
    <p>FIG. 57 shows the four major bands produced in PCR using degenerate primers to identify the <i>S. pombe </i>homolog of the <i>E. aediculatus </i>p123 sequences (SEQ ID NOS:239 and 240).</p>
    <p>FIG. 58 shows the alignment of the M2 PCR product (SEQ ID NO:243) with <i>E. aediculatus </i>p123 (SEQ ID NO:242), <i>S. cerevisiae </i>(SEQ ID NO:244), and Oxytricha (SEQ ID NO:223) telomerase protein sequences. Also shown are the actual genomic sequences (SEQ ID NOS:246 and 249) and the peptides encoded (SEQ ID NOS:245 and 250), degenerate primers Poly4 (SEQ ID NO:238) and Poly 1 (SEQ ID NO:244), and homologous regions of the M2 PCR product (SEQ ID NO:247) and its encoded peptide region (SEQ ID NO:248).</p>
    <p>FIG. 59 is a schematic showing the 3′ RT PCR strategy for identifying the <i>S. pombe </i>homolog of the <i>E. aediculatus </i>p123.</p>
    <p>FIGS. 60A, B, C and D shows characteristics of the libraries used to screen for <i>S. pombe </i>telomerase protein sequences and shows the results of screening the libraries for <i>S. pombe </i>telomerase protein sequences.</p>
    <p>FIG. 61 shows the positive results obtained with the HindIII-digested positive genomic clones containing <i>S. pombe </i>telomerase sequence.</p>
    <p>FIG. 62 is a schematic showing the 5′ RT PCR strategy used to obtain a full length <i>S. pombe </i>TRT clone.</p>
    <p>FIG. 63 shows the alignment of RT domains from telomerase catalytic subunits for <i>S. pombe </i>(S.p.) (SEQ ID NOS:251-255), <i>S. cerevisiae </i>(S.c.) (SEQ ID NOS:256-260) and <i>E. aediculatus </i>(E.a.) (SEQ ID NOS:261-265). Consensus sequences=SEQ ID NOS:49 and 50.</p>
    <p>FIG. 64 shows the alignment of the sequences from Euplotes (“Ea_p123”) (SEQ ID NO:110), <i>S. cerevisiae </i>(“Sc_Est2p”) (SEQ ID NO:222), and <i>S. pombe </i>(“SP_Tlp1p”) (SEQ ID NO:112). In Panel A, the shaded areas indicate residues shared between two sequences. In Panel B, the shaded areas indicate residues shared between all three sequences.</p>
    <p>FIG. 65 shows the disruption strategy used with the telomerase genes in <i>S. pombe. </i> </p>
    <p>FIG. 66 shows the experimental results confirming disruption of tez1.</p>
    <p>FIG. 67 shows the progressive shortening of telomeres in <i>S. pombe </i>due to tez1 disruption.</p>
    <p>FIG. 68 shows, in four pages, the DNA (SEQ ID NO:266) and amino acid (SEQ ID NO:267) of the ORF encoding an approximately 63 kDa telomerase protein encoded by the EcoRI-NotI insert of clone 712562.</p>
    <p>FIG. 69 shows an alignment of reverse transcriptase motifs from various sources, <i>E aediculatus </i>p123 (SEQ ID NOS:268-273), <i>S pombe </i>tez1 (SEQ ID NOS:274-279), <i>S. cerevisiae </i>EST2 (SEQ ID NOS:280-285), and human Hs TCP1 (SEQ ID NOS:286-291), with various consensus residues and motif sequences (SEQ ID NOS:49 and 50) indicated.</p>
    <p>FIG. 70 provides a restriction and function map of plasmid pGRN121.</p>
    <p>FIG. 71 shows, in two pages, the results of preliminary nucleic acid sequencing analysis of a hTRT cDNA sequence (SEQ ID NO:292).</p>
    <p>FIG. 72 shows, in ten pages, the preliminary nucleic acid sequence of hTRT (SEQ ID NO:292) and deduced ORF sequences in three reading frames (a=SEQ ID NOS:293-320; b=SEQ ID NOS:321-333; c=SEQ ID NOS:334-342).</p>
    <p>FIG. 73 provides a restriction and function map of plasmid pGRN121.</p>
    <p>FIG. 74 shows, in eight pages, refined nucleic acid sequence (SEQ ID NO:343) and deduced ORF sequences (SEQ ID NO:344) of hTRT</p>
    <p>FIG. 75 shows a restriction map of lambda clone 25-1.1.</p>
    <heading>DETAILED DESCRIPTION OF THE INVENTION</heading> <heading>I. Introduction</heading> <p>Telomerase is a ribonucleoprotein complex (RNP) comprising an RNA component and a catalytic protein component. The present invention relates to the cloning and characterization of the catalytic protein component of telomerase, hereinafter referred to as “TRT” (telomerase reverse transcriptase). TRT is so named because this protein acts as an RNA-dependent DNA polymerase (reverse transcriptase), using the telomerase RNA component (hereinafter, “TR”) to direct synthesis of telomere DNA repeat sequences. Moreover, TRT is evolutionarily related to other reverse transcriptases (see Example 12).</p>
    <p>In one aspect, the present invention relates to the cloning and characterization of the catalytic protein component of human telomerase, hereinafter referred to as “hTRT.” Human TRT is of extraordinary interest and value because, as noted supra, telomerase activity in human (and other mammalian cells) correlates with cell proliferative capacity, cell immortality, and the development of a neoplastic phenotype. For example, telomerase activity, and, as demonstrated in Example 2, infra, levels of human TRT gene products and are elevated in immortal human cells (such as malignant tumor cells and immortal cell lines) relative to mortal cells (such as most human somatic cells).</p>
    <p>The present invention further provides methods and compositions valuable for diagnosis, prognosis, and treatment of human diseases and disease conditions, as described in some detail infra. Also provided are methods and reagents useful for immortalizing cells (in vivo and ex vivo), producing transgenic animals with desirable characteristics, and numerous other uses, many of which are described infra. The invention also provides methods and reagents useful for preparing, cloning, or re-cloning TRT genes and proteins from ciliates, fungi, vertebrates, such as mammals, and other organisms.</p>
    <p>As described in detail infra, TRT was initially characterized following purification of telomerase from the ciliate <i>Euplotes aediculatus</i>. Extensive purification of <i>E. aediculatus </i>telomerase, using RNA-affinity chromatography and other methods, yielded the protein Ap123″. Surprisingly, p123 is unrelated to proteins previously believed to constitute the protein subunits of the telomerase holoenzyme (i.e., the p80 and p95 proteins of <i>Tetrahymena thermophila</i>). Analysis of the p123 DNA and protein sequences (Genbank Accession No. U95964; FIGS. 13 and 14) revealed reverse transcriptase (RT) motifs consistent with the role of p123 as the catalytic subunit of telomerase (see, e.g., FIGS. 1, <b>4</b> and <b>11</b>). Moreover, p123 is related to a <i>S. cerevisiae </i>(yeast) protein, Est2p, which was known to play a role in maintenance of telomeres in <i>S. cerevisiae </i>(Genbank Accession No. S5396), but prior to the present invention was not recognized as encoding a telomerase catalytic subunit protein (see, e.g., Lendvay et al., 1996, <i>Genetics, </i>144:1399).</p>
    <p>In one aspect, the present invention provides reagents and methods for identifying and cloning novel TRTs using: nucleic acid probes and primers generated or derived from the TRT polynucleotides disclosed (e.g., for cloning TRT genes and cDNAs); antibodies that specifically recognize the motifs or motif sequences or other TRT epitopes (e.g., for expression cloning TRT genes or purification of TRT proteins); by screening computer databases; or other means. For example, as described in Example 1, PCR (polymerase chain reaction) amplification of <i>S. pombe </i>DNA was carried out with degenerate-sequence primers designed from the Euplotes p123 RT motifs B′ and C. Of four prominent products generated, one encoded a peptide sequence homologous to Euplotes p123 and <i>S. cerevisiae </i>Est2p. Using this PCR product as a probe, the complete sequence of the <i>S. pombe </i>TRT homologue was obtained by screening of <i>S. pombe </i>cDNA and genomic libraries and amplifying <i>S. pombe </i>RNA by reverse transcription and PCR (RT-PCR). The complete sequence of the <i>S. pombe </i>gene (“trt1”; GenBank Accession No. AF015783; FIG. 15) revealed that homology with p123 and Est2p was especially high in the reverse transcriptase motifs. <i>S. pombe </i>trt1 is also referred to as tez1.</p>
    <p>Amplification using degenerate primers derived from the telomerase RT motifs was also used to obtain TRT gene sequences in <i>Oxytricha trifallax </i>and <i>Tetrahymena thermophila</i>, as described in Example 1.</p>
    <p>The Euplotes p123, <i>S. pombe </i>trt1, and <i>S. cerevisiae </i>Est2p nucleic acid sequences of the invention were used in a search of a computerized database of human expressed sequence tags (ESTs) using the program BLAST (Altschul et al. al, 1990, <i>J. Mol. Biol. </i>215:403). Searching this database with the Est2p sequence did not indicate a match, but searching with p123 and trt1 sequences identified a human EST (Genbank accession no. AA281296; see SEQ ID NO:8), as described in Example 1, putatively encoding a homologous protein. Complete sequencing of the cDNA clone containing the EST (hereinafter, “clone 712562”; see SEQ ID NO:3) showed that seven RT motifs were present. However, this clone did not encode a contiguous human TRT with all seven motifs, because motifs B′, C, D, and E were contained in a different open reading frame (ORF) than the more NH<sub>2</sub>-terminal motifs. In addition, the distance between motifs A and B′ was substantially shorter than that of the three previously characterized TRTs. Clone 712562 was obtained from the I.M.A.G.E. Consortium; Lennon et al., 1996, <i>Genomics </i>33:151.</p>
    <p>A cDNA clone, pGRN121, encoding a functional hTRT (see FIG. 16, SEQ ID NO:1) was isolated from a cDNA library derived from the human 293 cell line as described in Example 1. Comparing clone 712562 with pGRN121 showed that clone 712562 has a 182 base pair (see FIG. 24, SEQ ID NO:9) deletion between motifs A and B′. The additional 182 base pairs present in pGRN121 place all of the TRT motifs in a single open reading frame, and increase the spacing between the motif A and motif B′ regions to a distance consistent with the other known TRTs. As is described infra in the Examples (e.g., Example 7), SEQ ID NO:1 encodes a catalytically active telomerase protein having the sequence of SEQ ID NO:2. The polypeptide of SEQ ID NO:2 has 1132 residues and a calculated molecular weight of about 127 kilodaltons (kD).</p>
    <p>As is discussed infra, and described in Example 9, infra, TRT cDNAs possessing the 182 basepair deletion characteristic of the clone 712562 are detected following reverse transcription of RNA from telomerase-positive cells (e.g., testis and 293 cells). hTRT RNAs lacking this 182 base pair sequence are referred to generally as “Δ182 variants” and may represent one, two, or several species. Although the hTRT variants lacking the 182 basepair sequence found in the pGRN121 cDNA are unlikely to encode a fully active telomerase catalytic enzyme, they may play a role in telomerase regulation, as discussed infra, and/or have partial telomerase activity, such as telomere binding or hTR binding activity, as discussed infra.</p>
    <p>Thus, in one aspect, the present invention provides an isolated polynucleotide with a sequence of a naturally occurring human TRT gene or mRNA including, but not limited to, a polynucleotide having the sequence as set forth in FIG. 16 (SEQ ID NO:1). In a related aspect, the invention provides a polynucleotide encoding an hTRT protein, fragment, variant or derivative. In another related aspect, the invention provides sense and antisense nucleic acids that bind to an hTRT gene or mRNA. The invention further provides hTRT proteins, whether synthesized or purified from natural sources, as well as antibodies and other agents that specifically bind an hTRT protein or a fragment thereof. The present invention also provides many novel methods, including methods that employ the aforementioned compositions, for example, by providing diagnostic and prognostic assays for human diseases, methods for developing therapeutics and methods of therapy, identification of telomerase-associated proteins, and methods for screening for agents capable of activating or inhibiting telomerase activity. Numerous other aspects and embodiments of the invention are provided infra.</p>
    <p>One aspect of the invention is the use of a polynucleotide that is at least ten nucleotides to about 10 kb or more in length and comprises a contiguous sequence of at least ten nucleotides that is identical or exactly complementary to a contiguous sequence in a naturally occurring hTRT gene or hTRT mRNA in assaying or screening for an hTRT gene sequence or hTRT mRNA, or in preparing a recombinant host cell.</p>
    <p>A further aspect of the invention is the use of an agent increasing expression of hTRT in the manufacture of a medicament for the treatment of a condition addressed by increasing proliferative capacity of a vertebrate cell, optionally the medicament being for inhibiting the effects of aging.</p>
    <p>Yet a further aspect of the invention is the use of an inhibitor of telomerase activity in the manufacture of a medicament for the treatment of a condition associated with an elevated level of telomerase activity within a human cell.</p>
    <p>The proteins, variants and fragments of the invention, and the encoding polynucleotides or fragments, are also each provided in a further aspect of this invention for use as a pharmaceutical.</p>
    <p>The invention further includes the use of a protein, variant or fragment, or of a polynucleotide or fragment, in each case as defined herein, in the manufacture of a medicament, for example in the manufacture of a medicament for inhibiting an effect of aging or cancer.</p>
    <p>Another aspect of the invention is a polynucleotide selected from:</p>
    <p>(a) the DNA having a sequence as set forth in FIG. 16;</p>
    <p>(b) a polynucleotide of at least 10 nucleotides which hybridizes to the foregoing DNA and which codes for an hTRT protein or variant or which hybridizes to a coding sequence for such a variant; and,</p>
    <p>(c) DNA sequences which are degenerate as a result of the genetic code to the DNA sequences defined in (a) and (b) and which code for an hTRT polypeptide or variant.</p>
    <p>In certain embodiments of the present invention, the hTRT polynucleotides are other than the 389 nucleotide polynucleotide of SEQ ID NO:8 and/or other than clone 712562, the plasmid containing an insert, the sequence of which insert is shown in FIG. 18 (SEQ ID NO:3).</p>
    <p>The description below is organized by topic. Part II further describes amino acid motifs characteristic of TRT proteins, as well as TRT genes encoding proteins having such motifs. Parts III-VI describe, inter alia, nucleic acids, proteins, antibodies and purified compositions of the invention with particular focus on human TRT related compositions. Part VII describes, inter alia, methods and compositions of the invention useful for treatment of human disease. Part VIII describes production and identification of immortalized human cell lines. Part IX describes, inter alia, uses of the nucleic acids, polynucleotides, and other compositions of the invention for diagnosis of human diseases. Part X describes, inter alia, methods and compositions of the invention useful for screening and identifying agents and treatments that modulate (e.g., inhibit or promote) telomerase activity or expression. Part XI describes, inter alia, transgenic animals (e.g., telomerase knockout animals and cells). Part XII is a glossary of terms used in Parts I-XI. Part XIII describes examples relating to specific embodiments of the invention. The organization of the description of the invention by topic and subtopic is to provide clarity, and not to be limiting in any way.</p>
    <heading>II. TRT Genes and Proteins</heading> <p>The present invention provides isolated and/or recombinant genes and proteins having a sequence of a telomerase catalytic subunit protein (i.e., telomerase reverse transcriptase), including, but not limited to, the naturally occurring forms of such genes and proteins in isolated or recombinant form. Typically, TRTs are large, basic, proteins having reverse transcriptase (RT) and telomerase-specific (T) amino acid motifs, as disclosed herein. Because these motifs are conserved across diverse organisms, TRT genes of numerous organisms may be obtained using the methods of the invention or identified using primers, nucleic acid probes, and antibodies of the invention, such as those specific for one or more of the motif sequences.</p>
    <p>The seven RT motifs found in TRTs, while similar to those found in other reverse transcriptases, have particular hallmarks. For example, as shown in FIG. 4, within the TRT RT motifs there are a number of amino acid substitutions (marked with arrows) in residues highly conserved among the other RTs. For example, in motif C the two aspartic acid residues (DD) that coordinate active site metal ions (see, Kohlstaedt et al., 1992, <i>Science </i>256:1783; Jacobo-Molina et al., 1993, <i>Proc. Natl. Acad Sci. U.S.A. </i>90:6320; Patel et al., 1995, <i>Biochemistry </i>34:5351) occur in the context hxDD(F/Y) (SEQ ID NO:345) in the telomerase RTs compared to (F/Y)xDDh (SEQ ID NO:346) in the other RTs (where “h” is a hydrophobic amino acid, and “x” is any amino acid; see Xiong et al., 1990, <i>EMBO J. </i>9:3353; Eickbush, in <i>The Evolutionary Biology of Viruses</i>, (S. Morse, Ed., Raven Press, NY, p. 121, 1994)). Another systematic change characteristic of the telomerase subgroup occurs in motif E, where WxGxSx (SEQ ID NO:347) is a consensus sequence or is conserved among the telomerase proteins, whereas hLGxxh (SEQ ID NO:348) is characteristic of other RTs (Xiong et al., supra; Eickbush supra). This motif E is called the “primer grip”, and mutations in this region have been reported to affect RNA priming but not DNA priming (Powell et al., 1997, <i>J. Biol. Chem. </i>272:13262). Because telomerase requires a DNA primer (e.g., the chromosome 3′ end), it is not unexpected that telomerase should differ from other RTs in the primer grip region. In addition, the distance between motifs A and B′ is longer in the TRTs than is typical for other RTs, which may represent an insertion within the “fingers” region of the structure which resembles a right hand (FIG. 3; see Kohlstaedt et al., supra; Jacobo-Molina et al., supra; and Patel et al., supra).</p>
    <p>Moreover, as noted supra, Motif T is an additional hallmark of TRT proteins. This Motif T, as shown, for example in FIG. 4 (W-L-X-Y-X-X-h-h-X-h-h-X-p-F-F-Y-X-T-E-X-p-X-X-X-p-X-X-X-Y-X-R-K-X-X-W (SEQ ID NO:349) [X is any amino acid, h is hydrophobic, p is polar]), comprises a sequence that can be described using the formula:</p>
    <p>
      <maths> <formula-text>Trp-R<sub>1</sub>-X<sub>7</sub>-R<sub>1</sub>-R<sub>1</sub>-R<sub>2</sub>-X-Phe-Phe-Tyr-X-Thr-Glu-X<sub>8-9</sub>-R<sub>3</sub>-R<sub>3</sub>-Arg-R<sub>4</sub>-X<sub>2</sub>-Trp (SEQ ID NOS:11 and 12)</formula-text> </maths> </p>
    <p>where X is any amino acid and the subscript refers to the number of consecutive residues, R<sub>1 </sub>is leucine or isoleucine, R<sub>2 </sub>is glutamine or arginine, R<sub>3 </sub>is phenylalanine or tyrosine, and R<sub>4 </sub>is lysine or histidine.</p>
    <p>The T motif can also be described using the formula:</p>
    <p>
      <maths> <formula-text>Trp-R<sub>1</sub>-X<sub>4</sub>-h-h-X-h-h-R<sub>2</sub>-p-Phe-Phe-Tyr-X-Thr-Glu-X-p-X<sub>3</sub>-p-X<sub>2-3</sub>-R<sub>3</sub>-R<sub>3</sub>-Arg-R<sub>4</sub>-X<sub>2</sub>-Trp (SEQ ID NOS:350 and 351)</formula-text> </maths> </p>
    <p>where X is any amino acid and a subscript refers to the number of consecutive residues, R<sub>1 </sub>is leucine or isoleucine, R<sub>2 </sub>is glutamine or arginine, R<sub>3 </sub>is phenylalanine or tyrosine, R<sub>4 </sub>is lysine or histidine, h is a hydrophobic amino acid selected from Ala, Leu, lie, Val, Pro, Phe, Trp, and Met, and p is a polar amino acid selected from Gly, Ser, Thr, Tyr, Cys, Asn and Gln.</p>
    <p>In one embodiment, the present invention provides isolated naturally occurring and recombinant TRT proteins comprising one or more of the motifs illustrated in FIG. 11, e.g.,</p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="1" colwidth="28pt" align="left"> </colspec> <colspec colname="2" colwidth="189pt" align="left"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="2" align="center" rowsep="1" class="description-td" colspan="2"> </td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Motif T</td>
                <td class="description-td">W-X<sub>12</sub>-FFY-X-TE-X<sub>10-11</sub>-R-X<sub>3</sub>-W-X<sub>7</sub>-I (SEQ ID NOS:352 and</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">353)</td>
              </tr> <tr class="description-tr"> <td class="description-td">Motif T′</td>
                <td class="description-td">E-X<sub>2</sub>-V-X (SEQ ID NO:354)</td>
              </tr> <tr class="description-tr"> <td class="description-td">Motif 1</td>
                <td class="description-td">X<sub>3</sub>-R-X<sub>2</sub>-P-K-X<sub>3</sub>, (SEQ D NO:355) or, alternatively,</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">h-R-h-X-P-K (SEQ ID NO:633)</td>
              </tr> <tr class="description-tr"> <td class="description-td">Motif 2</td>
                <td class="description-td">X-R-X-I-X (SEQ ID NO:356) or, alternatively,</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">(F/L)-R-h-I-X<sub>2</sub>-h (SEQ ID NO:634)</td>
              </tr> <tr class="description-tr"> <td class="description-td">Motif A</td>
                <td class="description-td">X4-F-X<sub>3</sub>-D-X4-YD-X<sub>2</sub>, (SEQ ID NO:357) or, alternatively,</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">P-X-L-Y-F-h-X-h-D-h-X<sub>2</sub>-C-Y-D-X-I (SEQ ID NO:635)</td>
              </tr> <tr class="description-tr"> <td class="description-td">Motif B′</td>
                <td class="description-td">Y-X<sub>4</sub>-G-X<sub>2</sub>-QG-X<sub>3</sub>-S-X<sub>8 </sub> (SEQ ID NO:358) or, alternatively,</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">K-X-Y-X-Q-X<sub>2</sub>-G-I-P-Q-G-S-X-L-S-X-h-L (SEQ ID NO:636)</td>
              </tr> <tr class="description-tr"> <td class="description-td">Motif C</td>
                <td class="description-td">X<sub>6</sub>-DD-X-L-X<sub>3</sub>, (SEQ ID NO:359) or, altematively,</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">L-L-R-L-X-D-D-X-L-h-I-T (SEQ ID NO:637)</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="2" align="center" rowsep="1" class="description-td" colspan="2"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p>When the TRT protein shown contains more than one TRT motif, the order (NH2 -&gt;COOH) is as shown in FIG. <b>11</b>.</p>
    <p>In one embodiment, the present invention provides isolated naturally occurring TRT proteins comprising the following supermotif: (NH<sub>2</sub>)-X<sub>300-600</sub>-W-X<sub>12</sub>-FFY-X-TE-X<sub>10-11</sub>-R-X<sub>3</sub>-W-X<sub>7</sub>-I-X<sub>5-20</sub>-E-X<sub>2</sub>-V-X-X<sub>5-20</sub>-X<sub>3</sub>-R-X<sub>2</sub>-PK-X<sub>4-10</sub>-R-X-I-X-X<sub>60-80</sub>-X<sub>4</sub>-F-X<sub>3</sub>-D-X<sub>4</sub>-YD-X<sub>2</sub>-X<sub>80-130</sub>-Y-X<sub>4</sub>-G-X<sub>2</sub>-QG-X<sub>3</sub>-S-X<sub>8</sub>-X<sub>5-35</sub>-X<sub>6</sub>-DD-X-L-X<sub>3</sub>-X<sub>10-20</sub>-X<sub>12</sub>-K (SEQ ID NO:727)</p>
    <p>It will be apparent to one of skill that, provided with the reagents, including the TRT sequences disclosed herein for those reagents and the methods and guidance provided herein (including specific methodologies described infra), TRT genes and proteins can be obtained, isolated and produced in recombinant form by one of ordinary skill. For example, primers (e.g., degenerate amplification primers) are provided that hybridize to gene sequences encoding RT and T motifs characteristic of TRT. For example, one or more primers or degenerate primers that hybridize to sequences encoding the FFYXTE (SEQ ID NO:360) region of the T motif, other TRT motifs (as discussed infra), or combinations of motifs or consensus sequences, can be prepared based on the codon usage of the target organism, and used to amplify the TRT gene sequence from genomic DNA or cDNA prepared from the target organism. Use of degenerate primers is well known in the art and entails use of sets of primers that hybridize to the set of nucleic acid sequences that can potentially encode the amino acids of the target motif, taking into account codon preferences and usage of the target organism, and by using amplification (e.g., PCR) conditions appropriate for allowing base mismatches in the annealing steps of PCR. Typically two primer sets are used; however, single primer (or, in this case, a single degenerate primer set) amplification systems are well known and may be used to obtain TRT genes.</p>
    <p>Table 1 provides illustrative primers of the invention that may be used to amplify novel TRT nucleic acids, particularly those from vertebrates (e.g., humans and other mammals). “N” is an equimolar mixture of all four nucleotides, and nucleotides within parentheses are equimolar mixtures of the specified nucleotides.</p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" pgwide="1" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="329pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 1</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">ILLUSTRATIVE DEGENERATE PRIMERS FOR AMPLIFICATION OF TRT NUCLEIC ACIDS</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="6"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="42pt" align="left"> </colspec> <colspec colname="2" colwidth="42pt" align="center"> </colspec> <colspec colname="3" colwidth="35pt" align="left"> </colspec> <colspec colname="4" colwidth="154pt" align="left"> </colspec> <colspec colname="5" colwidth="42pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">motif</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">primer</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">motif</td>
                <td class="description-td">SEQ ID NO:</td>
                <td class="description-td">direction</td>
                <td class="description-td">5′ sequence-3′</td>
                <td class="description-td">SEQ ID NO:</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="5" align="center" rowsep="1" class="description-td" colspan="6"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="6"> <colspec colname="1" colwidth="14pt" align="left"> </colspec> <colspec colname="2" colwidth="42pt" align="left"> </colspec> <colspec colname="3" colwidth="42pt" align="char" char="."> </colspec> <colspec colname="4" colwidth="35pt" align="left"> </colspec> <colspec colname="5" colwidth="154pt" align="left"> </colspec> <colspec colname="6" colwidth="42pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">a</td>
                <td class="description-td"> <u>FFY</u>V<u>TE</u> </td>
                <td class="description-td">361</td>
                <td class="description-td">Forward</td>
                <td class="description-td">TT(CT)TT(CT)TA(CT)GTNACNGA</td>
                <td class="description-td">362</td>
              </tr> <tr class="description-tr"> <td class="description-td">b</td>
                <td class="description-td"> <u>FFY</u>V<u>TE</u> </td>
                <td class="description-td">361</td>
                <td class="description-td">Reverse</td>
                <td class="description-td">TCNGTNAC(GA)TA(GA)AA(GA)AA</td>
                <td class="description-td">363</td>
              </tr> <tr class="description-tr"> <td class="description-td">c</td>
                <td class="description-td"> <u>R</u>FI<u>PK</u>P</td>
                <td class="description-td">364</td>
                <td class="description-td">Forward</td>
                <td class="description-td">(CA)GNTT(CT)AT(ACT)CCNAA(AG)CC</td>
                <td class="description-td">365</td>
              </tr> <tr class="description-tr"> <td class="description-td">d</td>
                <td class="description-td"> <u>R</u>FI<u>PKP</u> </td>
                <td class="description-td">364</td>
                <td class="description-td">Reverse</td>
                <td class="description-td">GG(TC)TTNGG(TGA)AT(GA)AANC</td>
                <td class="description-td">366</td>
              </tr> <tr class="description-tr"> <td class="description-td">e</td>
                <td class="description-td">A<u>YD</u>TI</td>
                <td class="description-td">367</td>
                <td class="description-td">Forward</td>
                <td class="description-td">GCNTA(CT)GA(CT)ACNAT</td>
                <td class="description-td">365</td>
              </tr> <tr class="description-tr"> <td class="description-td">f</td>
                <td class="description-td">A<u>YD</u>TI</td>
                <td class="description-td">367</td>
                <td class="description-td">Reverse</td>
                <td class="description-td">TANGT(GA)TC(GA)TANGC</td>
                <td class="description-td">369</td>
              </tr> <tr class="description-tr"> <td class="description-td">g</td>
                <td class="description-td"> <u>G</u>IP<u>QG</u> </td>
                <td class="description-td">370</td>
                <td class="description-td">Forward</td>
                <td class="description-td">GGNAT(ACT)CCNCA(AG)GG</td>
                <td class="description-td">371</td>
              </tr> <tr class="description-tr"> <td class="description-td">h</td>
                <td class="description-td"> <u>G</u>IP<u>QGS</u> </td>
                <td class="description-td">21</td>
                <td class="description-td">Reverse</td>
                <td class="description-td">(GC)(AT)NCC(TC)TGNGG(TGA)ATNCC</td>
                <td class="description-td">372</td>
              </tr> <tr class="description-tr"> <td class="description-td">i</td>
                <td class="description-td">LV<u>DD</u>F<u>L</u> </td>
                <td class="description-td">373</td>
                <td class="description-td">Forward</td>
                <td class="description-td">(CT)TNGTNGA(CT)GA(CT)TT(CT)(CT)T</td>
                <td class="description-td">374</td>
              </tr> <tr class="description-tr"> <td class="description-td">j</td>
                <td class="description-td"> <u>DD</u>F<u>L</u>LVT</td>
                <td class="description-td">375</td>
                <td class="description-td">Reverse</td>
                <td class="description-td">GTNACNA(GA)NA(GA)(GA)AA(GA)TC(GA)TC</td>
                <td class="description-td">376</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="6" align="center" rowsep="1" class="description-td" colspan="6"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="329pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Preferred primer combinations (y = yes, n = no)</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="offset" colwidth="35pt" align="left"> </colspec> <colspec colname="1" colwidth="63pt" align="center"> </colspec> <colspec colname="2" colwidth="231pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Reverse</td>
                <td class="description-td"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="7"> <colspec colname="1" colwidth="35pt" align="center"> </colspec> <colspec colname="2" colwidth="14pt" align="left"> </colspec> <colspec colname="3" colwidth="14pt" align="left"> </colspec> <colspec colname="4" colwidth="14pt" align="left"> </colspec> <colspec colname="5" colwidth="14pt" align="left"> </colspec> <colspec colname="6" colwidth="14pt" align="left"> </colspec> <colspec colname="7" colwidth="224pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Forward</td>
                <td class="description-td">  b</td>
                <td class="description-td"> d</td>
                <td class="description-td"> f</td>
                <td class="description-td">  h</td>
                <td class="description-td">i</td>
                <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">a-</td>
                <td class="description-td">n</td>
                <td class="description-td">y</td>
                <td class="description-td">y</td>
                <td class="description-td">y</td>
                <td class="description-td">y</td>
              </tr> <tr class="description-tr"> <td class="description-td">c-</td>
                <td class="description-td">n</td>
                <td class="description-td">n</td>
                <td class="description-td">y</td>
                <td class="description-td">y</td>
                <td class="description-td">y</td>
              </tr> <tr class="description-tr"> <td class="description-td">e-</td>
                <td class="description-td">n</td>
                <td class="description-td">n</td>
                <td class="description-td">n</td>
                <td class="description-td">y</td>
                <td class="description-td">y</td>
              </tr> <tr class="description-tr"> <td class="description-td">g-</td>
                <td class="description-td">n</td>
                <td class="description-td">n</td>
                <td class="description-td">n</td>
                <td class="description-td">n</td>
                <td class="description-td">y</td>
              </tr> <tr class="description-tr"> <td class="description-td">i-</td>
                <td class="description-td">n</td>
                <td class="description-td">n</td>
                <td class="description-td">n</td>
                <td class="description-td">n</td>
                <td class="description-td">n</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="7" align="center" rowsep="1" class="description-td" colspan="7"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p>In one embodiment, an amplified TRT nucleic acid is used as a hybridization probe for colony hybridization to a library (e.g., cDNA library) made from the target organism, such that a nucleic acid having the entire TRT protein coding sequence, or a substantial portion thereof, is identified and isolated or cloned. Reagents and methods such as those just described were used in accordance with the methods described herein to obtain TRT gene sequences of <i>Oxytricha trifallax </i>and <i>Tetrahymena thermophila</i>, as described in detail infra. It will be recognized that following cloning of a previously uncharacterized TRT gene, the sequence can be determined by routine methods and the encoded polypeptide synthesized and assayed for a TRT activity, such as telomerase catalytic activity (as described herein and/or by telomerase assays known in the art).</p>
    <p>It will also be apparent to those of skill that TRT genes may be cloned using any of a variety of cloning methods of the invention because the TRT motif sequences and the nucleic acids of the invention comprising such sequences can be used in a wide variety of such methods. For example, hybridization using a probe based on the sequence of a known TRT to DNA or other nucleic acid libraries from the target organism, as described in Example 1 can be used. It will be appreciated that degenerate PCR primers or their amplification products such as those described supra, may themselves be labeled and used as hybridization probes. In another embodiment, expression cloning methods are used. For example, one or more antibodies that specifically bind peptides that span a TRT motif or other TRT epitope, such as the FFYXTE (SEQ ID NO:360) motif can be employed to isolate a ribosomal complex comprising a TRT protein and the mRNA that encodes it. For generating such antibodies of the invention, the peptide immunogens are typically between 6 and 30 amino acids in length, more often about 10 to 20 amino acids in length. The antibodies may also be used to probe a cDNA expression library derived from the organism of interest to identify a clone encoding a TRT sequence. In another embodiment, computer searches of DNA databases for DNAs containing sequences conserved with known TRTs can also be used to identify a clone comprising TRT sequence.</p>
    <p>In one aspect, the present invention provides compositions comprising an isolated or recombinant polypeptide having the amino acid sequence of a naturally occurring TRT protein. Usually the naturally occurring TRT has a molecular weight of between about 80,000 daltons (D) and about 150,000 D, most often between about 95,000 D and about 130,000 D. Typically, the naturally occurring TRT has a net positive charge at pH 7 (calculated pI typically greater than 9). In one embodiment, the polypeptide exhibits a telomerase activity as defined herein. In a related embodiment, the polypeptide has a TRT-specific region (T motif) sequence and exhibits a telomerase activity. The invention further provides fragments of such polypeptides. The present invention also provides isolated or recombinant polynucleotide having the sequence of a naturally occurring gene encoding a TRT protein. The invention provides regents useful for isolating sequence of a TRT from nonvertebrate (such as a yeast) and vertebrates, such as mammals (e.g., murine or human). The isolated polynucleotide may be associated with other naturally occurring or recombinant or synthetic vector nucleic acid sequences. Typically, the isolated nucleic acid is smaller than about 300 kb, often less than about 50 kb, more often less than about 20 kb, frequently less than about 10 kb and sometimes less than about 5 kb or 2 kb in length. In some embodiments the isolated TRT polynucleotide is even smaller, such as a gene fragment, primer, or probe of less than about 1 kb or less than 0.1 kb.</p>
    <heading>III. Nucleic Acids</heading> <p>A) Generally</p>
    <p>The present invention provides isolated and recombinant nucleic acids having a sequence of a polynucleotide encoding a telomerase catalytic subunit protein (TRT), such as a recombinant TRT gene from Euplotes, Tetrahymena, <i>S. pombe </i>or humans. Exemplary polynucleotides are provided in FIG. 13 (Euplotes); FIG. 15 (<i>S. pombe</i>) and FIG. 16 (human, GenBank Accession No. AF015950). The present invention provides sense and anti-sense polynucleotides having a TRT gene sequence, including probes, primers, TRT-protein-encoding polynucleotides, and the like.</p>
    <p>B) Human TRT</p>
    <p>The present invention provides nucleic acids having a sequence of a telomerase catalytic subunit from humans (i.e., hTRT).</p>
    <p>In one aspect, the invention provides a polynucleotide having a sequence or subsequence of a human TRT gene or RNA. In one embodiment, the polynucleotide of the invention has a sequence of SEQ ID NO:1 shown in FIG. 16 or a subsequence thereof. In another embodiment, the polynucleotide has a sequence of SEQ ID NO:3 (FIG. <b>18</b>), SEQ ID NO:4 (FIG. <b>20</b>), or subsequences thereof. The invention also provides polynucleotides with substantial sequence identity to the hTRT nucleic acid sequences disclosed herein, e.g., including but not limited to SEQ ID NOS:1 [FIG. <b>16</b>], 4 [FIG. <b>20</b>], 6 [FIG. <b>21</b>], and 7 [FIG. <b>12</b>]). Thus, the invention provides naturally occurring alleles of human TRT genes and variant polynucleotide sequences having one or more nucleotide deletions, insertions or substitutions relative to an hTRT nucleic acid sequence disclosed herein. As described infra, variant nucleic acids may be produced using the recombinant or synthetic methods described below or by other means.</p>
    <p>The invention also provides isolated and recombinant polynucleotides having a sequence from a flanking region of a human TRT gene. Such polynucleotides include those derived from genomic sequences of untranslated regions of the hTRT mRNA. An exemplary genomic sequence is shown in FIG. 21 (SEQ ID NO:6). As described in Example 4, SEQ ID NO:6 was obtained by sequencing a clone, λGΦ5 isolated from a human genomic library. Lambda GΦ5 contains a 15 kilobasepair (kbp) insert including approximately 13,000 bases 5′ to the hTRT coding sequences. This clone contains hTRT promoter sequences and other hTRT gene regulatory sequences (e.g., enhancers).</p>
    <p>The invention also provides isolated and recombinant polynucleotides having a sequence from an intronic region of a human TRT gene. An exemplary intronic sequence is shown in FIG. 12 (SEQ ID NO:7; see Example 3). In some embodiments, hTRT introns are included in “minigenes” for improved expression of hTRT proteins in eukaryotic cells.</p>
    <p>In a related aspect, the present invention provides polynucleotides that encode hTRT proteins or protein fragments, including modified, altered and variant hTRT polypeptides. In one embodiment, the encoded hTRT protein or fragment has an amino acid sequence as set forth in FIG. 17 (SEQ ID NO:2), or with conservative substitutions of SEQ ID NO:2. In one embodiment, the encoded hTRT protein or fragment has substitutions that change an activity of the protein (e.g., telomerase catalytic activity).</p>
    <p>It will be appreciated that, as a result of the degeneracy of the genetic code, the nucleic acid encoding the hTRT protein need not have the sequence of a naturally occurring hTRT gene, but that a multitude of polynucleotides can encode an hTRT polypeptide having an amino acid sequence of SEQ ID NO:2. The present invention provides each and every possible variation of nucleotide sequence that could be made by selecting combinations based on possible codon choices made in accordance with known triplet genetic codes, and all such variations are specifically disclosed hereby. Thus, although in some cases hTRT polypeptide-encoding nucleotide sequences that are capable of hybridizing to the nucleotide sequence of the naturally occurring sequence (under appropriately selected conditions of stringency) are preferred, it may be advantageous in other cases to produce nucleotide sequences encoding hTRT that employ a substantially different codon usage and so perhaps do not hybridize to nucleic acids with the naturally occurring sequence.</p>
    <p>In particular embodiments, the invention provides hTRT oligo- and polynucleotides that comprise a subsequence of an hTRT nucleic acid disclosed herein (e.g., SEQ ID NOS:1 and 6). The nucleic acids of the invention typically comprise at least about 10, more often at least about 12 or about 15 consecutive bases of the exemplified hTRT polynucleotide. Often, the nucleic acid of the invention will comprise a longer sequence, such as at least about 25, about 50, about 100, about 200, or at least about 500 to 3000 bases in length, for example when expression of a polypeptide, or full length hTRT protein is intended.</p>
    <p>In still other embodiments, the present invention provides “Δ182 Htrt” polynucleotides having a sequence identical or complementary to naturally occurring or non-naturally occurring hTRT polynucleotides such as SEQ ID NO:3 or SEQ ID NO:4, which do not contain the 182 nucleotide sequence (SEQ ID NO:9) found in pGRN121 (and also absent in clone 712562). These polynucleotides are of interest, in part, because they encode polypeptides that contain different combinations or arrangements of TRT motifs than found in the “full-length” hTRT polypeptide (SEQ ID NO:2) such as is encoded by pGRN121. As discussed infra, it is contemplated that these polypeptides may play a biological role in nature (e.g., in regulation of telomerase expression in cells) and/or find use as therapeutics (e.g., as dominant-negative products that inhibit function of wild-type proteins), or have other roles and uses, e.g. as described herein.</p>
    <p>For example, in contrast to the polypeptide encoded by pGRN121, clone 712562 encodes a 259 residue protein with a calculated molecular weight of approximately 30 kD (hereinafter, “712562 hTRT”). The 712562 hTRT polypeptide (SEQ ID NO:10 [FIG. <b>19</b>]) contains motifs T, 1, 2, and A, but not motifs B′, C, D and E (See FIG. <b>4</b>). Similarly, a variant hTRT polypeptide with therapeutic and other activities may be expressed from a nucleic acid similar to the pGRN121 cDNA but lacking the 182 basepairs missing in clone 712562, e.g., having the sequence shown in FIG. 20 (SEQ ID NO:4). This nucleic acid (hereinafter, “pro90 hTRT”), which may be synthesized using routine synthetic or recombinant methods as described herein, encodes a protein of 807 residues (calculated molecular weight of approximately 90 kD) that shares the same amino terminal sequence as the hTRT protein encoded by SEQ ID NO:1, but diverges at the carboxy-terminal region (the first 763 residues are common, the last 44 residues of pro90 hTRT are different than “full-length” hTRT). The pro90 hTRT polypeptide contains motifs T, 1, 2, and A, but not motifs B, C, D, E, and thus may have some, but not likely all telomerase activities.</p>
    <p>C) Production of Human TRT Nucleic Acids</p>
    <p>The polynucleotides of the invention have numerous uses including, but not limited to, expression of polypeptides encoding hTRT or fragments thereof, use as sense or antisense probes or primers for hybridization and/or amplification of naturally occurring hTRT genes or RNAs (e.g. for diagnostic or prognostic applications), and as therapeutic agents (e.g., in antisense, triplex, or ribozyme compositions). As will be apparent upon review of the disclosure, these uses will have enormous impact on the diagnosis and treatment of human diseases relating to aging, cancer, and fertility as well as the growth, reproduction, and manufacture of cell-based products. As described in the following sections, the hTRT nucleic acids of the invention may be made (e.g., cloned, synthesized, or amplified) using techniques well known in the art.</p>
    <p>1) Cloning, Amplification, and Recombinant Production</p>
    <p>In one embodiment, hTRT genes or cDNAs are cloned using a nucleic acid probe that specifically hybridizes to an hTRT mRNA, cDNA, or genomic DNA. One suitable probe for this purpose is a polynucleotide having all or part of the sequence provided in FIG. 16 (SEQ ID NO:1), such as a probe comprising a subsequence thereof. Typically, the target hTRT genomic DNA or cDNA is ligated into a vector (e.g., a plasmid, phage, virus, yeast artificial chromosome, or the like) and may be isolated from a genomic or cDNA library (e.g., a human placental cDNA library). Once an hTRT nucleic acid is identified, it can be isolated according to standard methods known to those of skill in the art. An illustrative example of screening a human cDNA library for the hTRT gene is provided in Example 1; similarly, an example of screening a human genomic library is found in Examples 3 and 4. Cloning methods are well known and are described, for example, in Sambrook et al., (1989) Molecular Cloning: A Laboratory Manual, 2nd ed., Vols. 1-3, Cold Spring Harbor Laboratory hereinafter, “Sambrook”); Berger and Kimmel, (1987) Methods in Enzymology, Vol. 152: Guide to Molecular Cloning Techniques, San Diego: Academic Press, Inc.; Ausubel et al., Current Protocols in Molecular Biology, Greene Publishing and Wiley-Interscience, New York (1997); Cashion et al., U.S. Pat. No. 5,017,478; and Carr, European Patent No. 0,246,864.</p>
    <p>The invention also provides hTRT genomic or cDNA nucleic acids isolated by amplification methods such as the polymerase chain reaction (PCR). In one embodiment, hTRT protein coding sequence is amplified from an RNA or cDNA sample (e.g., double stranded placental cDNA (Clontech, Palo Alto Calif.)) using the primers 5′-GTGAAGGCACTGTTCAGCG-3′ (“TCP1.1”) (SEQ ID NO:377) and 5′-CGCGTGGGTGAGGTGAGGTG-3 (“TCP 1.15”) (SEQ ID NO:378). In some embodiments a third primer or second pair of primers may be used, e.g., for “nested PCR”, to increase specificity. One example of a second pair of primers is 5′-CTGTGCTGGGCCTGGACGATA-3′ (“TCP1.14”) (SEQ ID NO:379) and 5′-AGCTTGTTCTCCATGTCGCCGTAG-3′ (“billTCP6”) (SEQ ID NO:380). It will be apparent to those of skill that numerous other primers and primer combinations, useful for amplification of hTRT nucleic acids are provided by the present invention.</p>
    <p>Moreover, the invention provides primers that amplify any specific region (e.g., coding regions, promoter regions, and/or introns) or subsequence of hTRT genomic DNA, cDNA or RNA. For example, the hTRT intron at position 274/275 of SEQ ID NO:1 (see Example 3) may be amplified (e.g., for detection of genomic clones) using primers TCP1.57 and TCP1.52 (primer pair 1) or primers TCP1.49 and TCP1.50 (primer pair 2). (Primer names refer to primers listed in Table 2, infra.) The primer pairs can be used individually or in a nested PCR where primer set 1 is used first. Another illustrative example relates to primers that specifically amplify and so detect the 5′ end of the hTRT mRNA or the exon encoding the 5′ end of hTRT gene (e.g., to assess the size or completeness of a cDNA clone). The following primer pairs are useful for amplifying the 5′ end of hTRT: primers K320 and K321 (primer pair 3); primers K320 and TCP1.61 (primer pair 4); primers K320 and K322 (primer pair 5). The primer sets can be used in a nested PCR in the order set 5, then set 4 or set 3, or set 4 or set 5, then set 3. Yet another illustrative example involves primers chosen to amplify or detect specifically the conserved hTRT TRT motif region comprising approximately the middle third of the mRNA (e.g., for use as a hybridization probe to identify TRT clones from, for example, nonhuman organisms). The following primer pairs are useful for amplifying the TRT motif region of hTRT nucleic acids: primers K304 and TCP1.8 (primer pair 6), or primers Lt1 and TCP1.15 (primer pair 7). The primer sets can be used in a nested PCR experiment in the order set 6 then set 7.</p>
    <p>Suitable PCR amplification conditions are known to those of skill and include (but are not limited to) 1 unit Taq polymerase (Perkin Elmer, Norwalk Conn.), 100 μM each dNTP (dATP, dCTP, dGTP, dTTP), 1×PCR buffer (50 mM KCl, 10 mM Tris, pH 8.3 at room temperature, 1.5 mM MgCl<sub>2</sub>, 0.01% gelatin) and 0.5 μM primers, with the amplification run for about 30 cycles at 94° for 45 sec, 55° for 45 sec and 72° for 90 sec. It will be recognized by those of skill in the art that other thermostable DNA polymerases, reaction conditions, and cycling parameters will also provide suitable amplification. Other suitable in vitro amplification methods that can be used to obtain hTRT nucleic acids include, but are not limited to, those herein, infra. Once amplified, the hTRT nucleic acids can be cloned, if desired, into any of a variety of vectors using routine molecular biological methods or detected or otherwise utilized in accordance with the methods of the invention.</p>
    <p>One of skill will appreciate that the cloned or amplified hTRT nucleic acids obtained as described above can be prepared or propagated using other methods, such as chemical synthesis or replication by transformation into bacterial systems, such as <i>E. coli </i>(see, e.g., Ausubel et al., supra), or eukaryotic, such as mammalian, expression systems. Similarly, hTRT RNA can be expressed in accordance with the present in vitro methods, or in bacterial systems such as <i>E. coli </i>using, for example, commercially available vectors containing promoters recognized by an RNA polymerase such as T7, T3 or SP6, or transcription of DNA generated by PCR amplification using primers containing an RNA polymerase promoter.</p>
    <p>The present invention further provides altered or modified hTRT nucleic acids. It will be recognized by one of skill that the cloned or amplified hTRT nucleic acids obtained can be modified (e.g., truncated, derivatized, altered) by methods well known in the art (e.g., site-directed mutagenesis, linker scanning mutagenesis) or simply synthesized de novo as described below. The altered or modified hTRT nucleic acids are useful for a variety of applications, including, but not limited to, facilitating cloning or manipulation of an hTRT gene or gene product, or expressing a variant hTRT gene product. For example, in one embodiment, the hTRT gene sequence is altered such that it encodes an hTRT polypeptide with altered properties or activities, as discussed in detail in infra, for example, by mutation in a conserved motif of hTRT. In another illustrative example, the mutations in the protein coding region of an hTRT nucleic acid may be introduced to alter glycosylation patterns, to change codon preference, to produce splice variants, remove protease-sensitive sites, create antigenic domains, modify specific activity, and the like. In other embodiments, the nucleotide sequence encoding hTRT and its derivatives is changed without altering the encoded amino acid sequences, for example, the production of RNA transcripts having more desirable properties, such as increased translation efficiency or a greater or a shorter half-life, compared to transcripts produced from the naturally occurring sequence. In yet another embodiment, altered codons are selected to increase the rate at which expression of the peptide occurs in a particular prokaryotic or eukaryotic expression host in accordance with the frequency with which particular codons are utilized by the host. Useful in vitro and in vivo recombinant techniques that can be used to prepare variant hTRT polynucleotides of the invention are found in Sambrook et al. and Ausubel et al., both supra.</p>
    <p>As noted supra, the present invention provides nucleic acids having flanking (5′ or 3′) and intronic sequences of the hTRT gene. The nucleic acids are of interest, inter alia, because they contain promoter and other regulatory elements involved in hTRT regulation and useful for expression of hTRT and other recombinant proteins or RNA gene products. It will be apparent that, in addition to the nucleic acid sequences provided in SEQ ID NOS:6 and 7, additional hTRT intron and flanking sequences may be readily obtained using routine molecular biological techniques. For example, additional hTRT genomic sequence may be obtained from Lambda clone GΦ5 (ATCC Accession No. 209024), described supra and in Example 4. Still other hTRT genomic clones and sequences may be obtained by screening a human genomic library using an hTRT nucleic acid probe having a sequence or subsequence from SEQ ID NO:1. Additional clones and sequences (e.g., still further upstream) may be obtained by using labeled sequences or subclones derived from λGΦ5 to probe appropriate libraries. Other useful methods for further characterization of hTRT flanking sequences include those general methods described by Gobinda et al., 1993, <i>PCR Meth. Applic. </i>2:318; Triglia et al., 1988, <i>Nucleic Acids Res. </i>16:8186; Lagerstrom et al., 1991, <i>PCR Methods Applic. </i>1:111; and Parker et al., 1991, <i>Nucleic Acids Res. </i>19:3055.</p>
    <p>Intronic sequences can be identified by routine means such as by comparing the hTRT genomic sequence with hTRT cDNA sequences (see, e.g., Example 3), by S1 analysis (see Ausubel et al., supra, at Chapter 4), or various other means known in the art. Intronic sequences can also be found in pre-mRNA (i.e., unspliced or incompletely spliced mRNA precursors), which may be amplified or cloned following reverse transcription of cellular RNA.</p>
    <p>When desired, the sequence of the cloned, amplified, or otherwise synthesized hTRT or other TRT nucleic acid can be determined or verified using DNA sequencing methods well known in the art (see, e.g., Ausubel et al., supra). Useful methods of sequencing employ such enzymes as the Klenow fragment of DNA polymerase I, Sequenase (US Biochemical Corp, Cleveland Ohio), Taq DNA polymerase (Perkin Elmer, Norwalk Conn.), thermostable T7 polymerase (Amersham, Chicago Ill.), or combinations of recombinant polymerases and proofreading exonucleases such as the ELONGASE Amplification System marketed by Gibco BRL (Gaithersburg Md.). When sequencing or verifying the sequence of oligonucleotides (such as oligonucleotide made de novo by chemical synthesis), the method of Maxam and Gilbert may be preferred (Maxam and Gilbert, 1980, <i>Meth. Enz. </i>65:499; Ausubel et al., supra, Ch. 7).</p>
    <p>The 5′ untranslated sequences of hTRT or other TRT mRNAs can be determined directly by cloning a “full-length” hTRT or other cDNA using standard methods such as reverse transcription of mRNA, followed by cloning and sequencing the resulting cDNA. Preferred oligo(dT)-primed libraries for screening or amplifying full length cDNAs that have been size-selected to include larger cDNAs may be preferred. Random primed libraries are also suitable and often include a larger proportion of clones that contain the 5′ regions of genes. Other well known methods for obtaining 5′ RNA sequences, such as the RACE protocol described by Frohman et al., 1988, <i>Proc. Nat. Acad. Sci USA </i>85:8998, may also be used. If desired, the transcription start site of an hTRT or other TRT mRNA can be determined by routine methods using the nucleic acids provided herein (e.g., having a sequence of SEQ ID NO:1). One method is S1 nuclease analysis (Ausubel et al., supra) using a labeled DNA having a sequence from the 5′ region of SEQ ID NO:1.</p>
    <p>2) Chemical Synthesis of Nucleic Acids</p>
    <p>The present invention also provides hTRT polynucleotides (RNA, DNA or modified) that are produced by direct chemical synthesis. Chemical synthesis is generally preferred for the production of oligonucleotides or for oligonucleotides and polynucleotides containing nonstandard nucleotides (e.g., probes, primers and antisense oligonucleotides). Direct chemical synthesis of nucleic acids can be accomplished by methods known in the art, such as the phosphotriester method of Narang et al., 1979, <i>Meth. Enzymol. </i>68:90; the phosphodiester method of Brown et al., <i>Meth. Enzymol. </i>68:109 (1979); the diethylphosphoramidite method of Beaucage et al., <i>Tetra. Lett., </i>22:1859 (1981); and the solid support method of U.S. Pat. No. 4,458,066. Chemical synthesis typically produces a single stranded oligonucleotide, which may be converted into double stranded DNA by hybridization with a complementary sequence, or by polymerization with a DNA polymerase and an oligonucleotide primer using the single strand as a template. One of skill will recognize that while chemical synthesis of DNA is often limited to sequences of about 100 or 150 bases, longer sequences may be obtained by the ligation of shorter sequences or by more elaborate synthetic methods.</p>
    <p>It will be appreciated that the hTRT (or hTR or other) polynucleotides and oligonucleotides of the invention can be made using nonstandard bases (e.g., other than adenine, cytidine, guanine, thymine, and uridine) or nonstandard backbone structures to provides desirable properties (e.g., increased nuclease-resistance, tighter-binding, stability or a desired T<sub>M</sub>). Techniques for rendering oligonucleotides nuclease-resistant include those described in PCT publication WO 94/12633. A wide variety of useful modified oligonucleotides may be produced, including oligonucleotides having a peptide-nucleic acid (PNA) backbone (Nielsen et al., 1991, <i>Science </i>254:1497) or incorporating 2′-O-methyl ribonucleotides, phosphorothioate nucleotides, methyl phosphonate nucleotides, phosphotriester nucleotides, phosphorothioate nucleotides, phosphoramidates. Still other useful oligonucleotides may contain alkyl and halogen-substituted sugar moieties comprising one of the following at the 2′ position: OH, SH, SCH<sub>3</sub>, F, OCN, OCH<sub>3</sub>OCH<sub>3</sub>, OCH<sub>3</sub>O(CH<sub>2</sub>)<sub>n</sub>CH<sub>3</sub>, O(CH<sub>2</sub>)<sub>n</sub>NH<sub>2 </sub>or O(CH<sub>2</sub>)<sub>n</sub>CH<sub>3 </sub>where n is from 1 to about 10; C<sub>1 </sub>to C<sub>10 </sub>lower alkyl, substituted lower alkyl, alkaryl or aralkyl; Cl; Br; CN; CF<sub>3</sub>; OCF<sub>3</sub>; O—, S—, or N-alkyl; O—, S—, or N-alkenyl; SOCH<sub>3</sub>; SO<sub>2</sub>CH<sub>3</sub>; ONO<sub>2</sub>; NO<sub>2</sub>; N<sub>3</sub>; NH<sub>2</sub>; heterocycloalkyl; heterocycloalkaryl; aminoalkylamino; polyalkylamino; substituted silyl; an RNA cleaving group; a cholesteryl group; a folate group; a reporter group; an intercalator; a group for improving the pharmacokinetic properties of an oligonucleotide; or a group for improving the pharmacodynamic properties of an oligonucleotide and other substituents having similar properties. Folate, cholesterol or other groups which facilitate oligonucleotide uptake, such as lipid analogs, may be conjugated directly or via a linker at the 2′ position of any nucleoside or at the 3′ or 5′ position of the 3′-terminal or 5′-terminal nucleoside, respectively. One or more such conjugates may be used. Oligonucleotides may also have sugar mimetics such as cyclobutyls in place of the pentofuranosyl group. Other embodiments may include at least one modified base form or “universal base” such as inosine, or inclusion of other nonstandard bases such as queosine and wybutosine as well as acetyl-, methyl-, thio- and similarly modified forms of adenine, cytidine, guanine, thymine, and uridine which are not as easily recognized by endogenous endonucleases. The invention further provides oligonucleotides having backbone analogues such as phosphodiester, phosphorothioate, phosphorodithioate, methylphosphonate, phosphoramidate, alkyl phosphotriester, sulfamate, 3′-thioacetal, methylene(methylimino), 3′-N-carbamate, morpholino carbamate, chiral-methyl phosphonates, nucleotides with short chain alkyl or cycloalkyl intersugar linkages, short chain heteroatomic or heterocyclic intersugar (“backbone”) linkages, or CH<sub>2</sub>—NH—O—CH<sub>2</sub>, CH<sub>2</sub>—N(CH<sub>3</sub>)—OCH<sub>2</sub>, CH<sub>2</sub>—O—N(CH<sub>3</sub>)—CH<sub>2</sub>, CH<sub>2</sub>—N(CH<sub>3</sub>)—N(CH<sub>3</sub>)—CH<sub>2 </sub>and O—N(CH<sub>3</sub>)—CH<sub>2</sub>—CH<sub>2 </sub>backbones (where phosphodiester is O—P—O—CH<sub>2</sub>), or mixtures of the same. Also useful are oligonucleotides having morpholino backbone structures (U.S. Pat. No. 5,034,506).</p>
    <p>Useful references include Oligonucleotides and Analogues, A Practical Approach, edited by F. Eckstein, IRL Press at Oxford University Press (1991); Antisense Strategies, Annals of the New York Academy of Sciences, Volume 600, Eds. Baserga and Denhardt (NYAS 1992); Milligan et al., Jul. 9, 1993, J. Med. Chem. 36(14):1923-1937; Antisense Research and Applications (1993, CRC Press), in its entirety and specifically Chapter 15, by Sanghvi, entitled “Heterocyclic base modifications in nucleic acids and their applications in antisense oligonucleotides.” Antisense Therapeutics, ed. Sudhir Agrawal (Humana Press, Totowa, N.J., 1996).</p>
    <p>D) Labeling Nucleic Acids</p>
    <p>It is often useful to label the nucleic acids of the invention, for example, when the hTRT or other oligonucleotides or polynucleotides are to be used as nucleic acid probes. The labels (see infra) may be incorporated by any of a number of means well known to those of skill in the art. In one embodiment, an unamplified nucleic acid (e.g., mRNA, polyA mRNA, cDNA) is labeled. Means of producing labeled nucleic acids are well known to those of skill in the art and include, for example, nick-translation, random primer labeling, end-labeling (e.g. using a kinase), and chemical conjugation (e.g., photobiotinylation) or synthesis. In another embodiment, the label is simultaneously incorporated during an amplification step in the preparation of the sample nucleic acids. Thus, for example, polymerase chain reaction (PCR) or other nucleic acid amplification method with labeled primers or labeled nucleotides will provide a labeled amplification product. In another embodiment, transcription amplification using a labeled nucleotide (e.g. fluorescein-labeled UTP and/or CTP) incorporates a label into the transcribed nucleic acids. An amplification product may also, or alternatively, be labeled after the amplification is completed.</p>
    <p>E) Illustrative Oligonucleotides</p>
    <p>As noted supra and discussed in detail infra, oligonucleotides are used for a variety of uses including as primers, probes, therapeutic or other antisense oligonucleotides, triplex oligonucleotides, and numerous other uses as apparent from this disclosure. Table 2 provides certain illustrative specific oligonucleotides that may be used in the practice of the invention. It will be appreciated that numerous other useful oligonucleotides of the invention may be synthesized by one of skill, following the guidance provided herein.</p>
    <p>In Table 2, “seq” means that the primer has been used, or is useful, for sequencing; “PCR” means that the primer has been used, or is useful, for PCR; “AS” means that means that the primer has been used, or is useful for antisense inhibition of telomerase activity; “CL” means that the primer has been used, or is useful in cloning regions of hTRT genes or RNA, “mut” means that the primer has been used, or is useful for constructing mutants of hTRT genes or gene products. “UC,” means “upper case,” and “lc” means “lower case.” Mismatches and insertions (relative to SEQ ID NO:1) are indicated by underlining; deletions are indicated by a “−”. It will be appreciated that nothing in Table 2 is intended to limit the use of any particular oligonucleotide to any single use or set of uses.</p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" orient="land" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="595pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 2</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">USEFUL OLIGONUCLEOTIDES</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="10"> <colspec colname="1" colwidth="42pt" align="center"> </colspec> <colspec colname="2" colwidth="224pt" align="left"> </colspec> <colspec colname="3" colwidth="154pt" align="left"> </colspec> <colspec colname="4" colwidth="42pt" align="center"> </colspec> <colspec colname="5" colwidth="21pt" align="center"> </colspec> <colspec colname="6" colwidth="21pt" align="center"> </colspec> <colspec colname="7" colwidth="21pt" align="center"> </colspec> <colspec colname="8" colwidth="21pt" align="center"> </colspec> <colspec colname="9" colwidth="21pt" align="center"> </colspec> <colspec colname="10" colwidth="28pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">USE</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">primer</td>
                <td class="description-td">5′-sequence-3′ *</td>
                <td class="description-td">Notes</td>
                <td class="description-td">mismatch? *</td>
                <td class="description-td">seq</td>
                <td class="description-td">PCR</td>
                <td class="description-td">AS</td>
                <td class="description-td">CL</td>
                <td class="description-td">MUT</td>
                <td class="description-td">SEQ ID</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="10" align="center" rowsep="1" class="description-td" colspan="10"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.1</td>
                <td class="description-td">GTGAAGGCACTGTTCAGCG</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">377</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.2</td>
                <td class="description-td">GTGGATGATTTCTTGTTGG</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">X</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">381</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.4</td>
                <td class="description-td">CTGGACACTCAGCCCTTGG</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">X</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">382</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.5</td>
                <td class="description-td">GGCAGGTGTGCTGGACACT</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">383</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.6</td>
                <td class="description-td">TTTGATGATGCTGGCGATG</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">384</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.7</td>
                <td class="description-td">GGG<u>G</u>CTCGTCTTCTACAGG</td>
                <td class="description-td"> </td>
                <td class="description-td">Y</td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">385</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.8</td>
                <td class="description-td">CAGCAGGAGGATCTTGTAG</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">386</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.9</td>
                <td class="description-td">TGACCCCAGGAGTGGCACG</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">387</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.10</td>
                <td class="description-td">TCAAGCTGACTCGACACCG</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">388</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.11</td>
                <td class="description-td">CGGCGTGACAGGGCTGC</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">389</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.12</td>
                <td class="description-td">GCTGAAGGCTGAGTGTCC</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">390</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.13</td>
                <td class="description-td">TAGTCCATGTTCACAATCG</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">391</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.14</td>
                <td class="description-td">CTGTGCTGGGCCTGGACGATA</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">379</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.15</td>
                <td class="description-td">CGCGTGGGTGAGGTGAGGTG</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">378</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.16</td>
                <td class="description-td">TTTCCGTGTTGAGTGTTTC</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">392</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.17</td>
                <td class="description-td">GTCACCGTGTTGGGCAGG</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">393</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.19</td>
                <td class="description-td">GCTACCTGCCCAACACGG</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">394</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.20</td>
                <td class="description-td">GCGCGAAGAACGTGCTGG</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">395</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.21</td>
                <td class="description-td">CA-CTGCTCCTTGTCGCCTG</td>
                <td class="description-td"> </td>
                <td class="description-td">Y</td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">396</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.22</td>
                <td class="description-td">TTCCCAAGGACTTTGTTGC</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">397</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.24</td>
                <td class="description-td">TG<u>T</u>TCCTCAAGACGCACTG</td>
                <td class="description-td"> </td>
                <td class="description-td">Y</td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">398</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.25</td>
                <td class="description-td">TACTGCGTGCGTCGGTATG</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">399</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.26</td>
                <td class="description-td">GGTCTTGCGGCTGAAGTGT</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">400</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.27</td>
                <td class="description-td">TGGTTCACCTGCTGGCACG</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">40)</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.28</td>
                <td class="description-td">GTGGTTTCTGTGTGGTGTC</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">402</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.29</td>
                <td class="description-td">GACACCACACAGAAACCAC</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">403</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.30</td>
                <td class="description-td">GTGCCAGCAGGTGAACCAG</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">404</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.32B</td>
                <td class="description-td">GCAGTGCGTCTTGAGGAGC</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">405</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.33</td>
                <td class="description-td">TGGAACCATAGCGTCAGGGAG</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">406</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.34</td>
                <td class="description-td">GGCCTCCCTGACGCTATGGTT</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">407</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.35</td>
                <td class="description-td">GC(GT)CGGCGCTGCCACTCAGG</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">408</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.35t</td>
                <td class="description-td">GCTCGGCGCTGCCACTCAGG</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">409</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.36</td>
                <td class="description-td">ACGCCGAGACCAAGCACTTC</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">410</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.38</td>
                <td class="description-td">CCAAAGAGGTGGCTTCTTCG</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">4)1</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.39</td>
                <td class="description-td">AAGGCCAGCACGTTCTTCGC</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">412</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.40</td>
                <td class="description-td">CACGTTCGTGCGGCGCCTG</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">413</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.41</td>
                <td class="description-td">CCTTCACCACCAGCGTGCG</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">414</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.42</td>
                <td class="description-td">GGCGACGACGTGCTGGTTC</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">415</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.43</td>
                <td class="description-td">GGCTCAGGGGCAGCGCCAC</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">416</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.44</td>
                <td class="description-td">CTGGCAGGTGTACGGCTTC</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">417</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.45</td>
                <td class="description-td">GCGTGGACCGAGTGACCGTGGTTTC</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">418</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.46</td>
                <td class="description-td">GACGTGGTGGCCGCGATGTGG</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">419</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.47</td>
                <td class="description-td">GAAGTCTGCCGTTGCCCAAGAG</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">420</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.48</td>
                <td class="description-td">GACACCACACAGAAACCACGGTCAC</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">421</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.49</td>
                <td class="description-td">CGCCCCCTCCTTCCGCCAGGT</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">422</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.50</td>
                <td class="description-td">CGAAGCCGAAGGCCAGCACGTTCTT</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">423</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.51</td>
                <td class="description-td">GGTGGCCCGAGTGCTGCAGAGG</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">424</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.52</td>
                <td class="description-td">GTAGCTGCGCACGCTGGTGGTGAAG</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">425</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.53</td>
                <td class="description-td">TGGGCGACGACGTGCTGGTTCA</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">426</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.54</td>
                <td class="description-td">TATGGTTCCAGGCCCGTTCGCATCC</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">427</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.55</td>
                <td class="description-td">CCAGCTGCGCCTACCAGGTGTGC</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">428</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.56</td>
                <td class="description-td">GGCCTCCCTGACGCTATGGTTCCAG</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">429</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.57</td>
                <td class="description-td">GGTGCTGCCGCTGGCCACGTTCG</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">430</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.58</td>
                <td class="description-td">TCCCAGGGCACGCACACCAGGCACT</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">431</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.59</td>
                <td class="description-td">GTACAGGGCACACCTTTGGTCACTC</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">432</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.60</td>
                <td class="description-td">TCGACGACGTACACACTCATCAGCC</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">433</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.61</td>
                <td class="description-td">AGCGGCAGCACCTCGCGGTAGTGGC</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">434</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.62</td>
                <td class="description-td">CCACCAGCTCCTTCAGGCAGGACAC</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">435</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.63</td>
                <td class="description-td">CCAGGGCTTCCCACGTGCGCAGCAG</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">436</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1:64</td>
                <td class="description-td">CGCACGAACGTGGCCAGCGGCAGCA</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">437</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.65</td>
                <td class="description-td">TGACCGTGGTTTCTGTGTGGTGT</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">438</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.66</td>
                <td class="description-td">CCCTCTTCAAGTGCTGTCTGATTCC</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">439</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.67</td>
                <td class="description-td">ATCGCGGCCACCACGTCCCT</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">440</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.68</td>
                <td class="description-td">TGCTCCAGACACTCGGCCGGTAGAA</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">441</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.69</td>
                <td class="description-td">ACGAAGCCGTACACCTGCC</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">442</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.72</td>
                <td class="description-td">CGACATCCCTGCGTTCTTGGCTTTC</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">443</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.73</td>
                <td class="description-td">CACTGCTGGCCTCATTCAGGG</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">444</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.74</td>
                <td class="description-td">GCGACATGGAGAACAAGC</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">445</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.75</td>
                <td class="description-td">GCAGCCATACTTCAGGGACAC</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">446</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.76</td>
                <td class="description-td">CCATCCTCTCCACGCTGCTC</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">447</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.77</td>
                <td class="description-td">GCGATGACCTCCGTGAGCCTG</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">448</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.78</td>
                <td class="description-td">CCCAGGACAGGCTCACGGA</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">449</td>
              </tr> <tr class="description-tr"> <td class="description-td">billTCP1</td>
                <td class="description-td">CCTCTTCAAGTGCTGTCTGATTCC</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">450</td>
              </tr> <tr class="description-tr"> <td class="description-td">billTCP2</td>
                <td class="description-td">CAGCTCGACGACGTACACACTCATC</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">451</td>
              </tr> <tr class="description-tr"> <td class="description-td">billTCP4</td>
                <td class="description-td">CTGACGTCCAGACTCCGCTTCAT</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">452</td>
              </tr> <tr class="description-tr"> <td class="description-td">billTCP6</td>
                <td class="description-td">AGCTTGTTCTCCATGTCGCCGTAG</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">380</td>
              </tr> <tr class="description-tr"> <td class="description-td">rpprim0l</td>
                <td class="description-td">GACCTGAGCAGCTCGACGACGTACACAC</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">453</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">TCATC</td>
              </tr> <tr class="description-tr"> <td class="description-td">Lt1</td>
                <td class="description-td">GTCGTCGAGCTGCTCAGGTC</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">454</td>
              </tr> <tr class="description-tr"> <td class="description-td">Lt2</td>
                <td class="description-td">AGCACGCTGAACAGTGCCTT</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">455</td>
              </tr> <tr class="description-tr"> <td class="description-td">Lt3</td>
                <td class="description-td">GACCTGAGCAGCTCGACGAC</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">456</td>
              </tr> <tr class="description-tr"> <td class="description-td">Lt4</td>
                <td class="description-td">AAGGCACTGTTCAGCGTGCT</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">457</td>
              </tr> <tr class="description-tr"> <td class="description-td">Lt5</td>
                <td class="description-td">CGG<u>CC</u>GAGTGTCTGGAGCAA</td>
                <td class="description-td"> </td>
                <td class="description-td">Y</td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">458</td>
              </tr> <tr class="description-tr"> <td class="description-td">Lt6</td>
                <td class="description-td">GGATGAAGCGGAGTCTGGA</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">459</td>
              </tr> <tr class="description-tr"> <td class="description-td">BamHI Lt7</td>
                <td class="description-td"> <u>ATGGATC</u>CGTCGTCGAGCTGCTCAGGTCT</td>
                <td class="description-td">BamHI Site</td>
                <td class="description-td">Y</td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">460</td>
              </tr> <tr class="description-tr"> <td class="description-td">Salt Lt8</td>
                <td class="description-td">
                  <u>ATCAGC</u>
                </td>
                <td class="description-td">Pvu II site (not Sal I)</td>
                <td class="description-td">Y</td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">461</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">TGAGCACGCTGAACAGTGCCTTC</td>
              </tr> <tr class="description-tr"> <td class="description-td">K303</td>
                <td class="description-td">GTCTCCGTGACATAAAAGAAAGAC</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">462</td>
              </tr> <tr class="description-tr"> <td class="description-td">K304</td>
                <td class="description-td">GCCAAGTTCCTGCACTGGCT</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">463</td>
              </tr> <tr class="description-tr"> <td class="description-td">K305</td>
                <td class="description-td">GCCTGTTCTTITGAAACGTGGTCT</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">464</td>
              </tr> <tr class="description-tr"> <td class="description-td">K306</td>
                <td class="description-td">XXGCCTGTTCTTTTGAAACGTGGTCT</td>
                <td class="description-td">X = biotin, = K305</td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">465</td>
              </tr> <tr class="description-tr"> <td class="description-td">K311</td>
                <td class="description-td">GTCAAGATGCCTGAGATAGAAC</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">466</td>
              </tr> <tr class="description-tr"> <td class="description-td">K312</td>
                <td class="description-td">TGCTTAGCTTGTGGGGGTGTCA</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">467</td>
              </tr> <tr class="description-tr"> <td class="description-td">K313</td>
                <td class="description-td">TGCTTAGCTTGTGGGGGTGTCA</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">467</td>
              </tr> <tr class="description-tr"> <td class="description-td">K320</td>
                <td class="description-td">GCTGCGTCCTGCTGCGCACGT</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">468</td>
              </tr> <tr class="description-tr"> <td class="description-td">K321</td>
                <td class="description-td">CAGCGGGGAGCGCGCGGCATC</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">469</td>
              </tr> <tr class="description-tr"> <td class="description-td">K322</td>
                <td class="description-td">TGGGCCACCAGCGCGCGGAAA</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">470</td>
              </tr> <tr class="description-tr"> <td class="description-td">slanti.1</td>
                <td class="description-td">CGGCGCAGCCCGTCAGGCTTGGGG</td>
                <td class="description-td"> </td>
                <td class="description-td">Y</td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">471</td>
              </tr> <tr class="description-tr"> <td class="description-td">slanti.2</td>
                <td class="description-td">CCGACA<u> GC</u>TCCCGCAGCTGCACCC</td>
                <td class="description-td"> </td>
                <td class="description-td">Y</td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">472</td>
              </tr> <tr class="description-tr"> <td class="description-td">slanti.3</td>
                <td class="description-td">CGTACACACTCATCAGCCAGTGCAGGAA</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">473</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">CTTGGC</td>
              </tr> <tr class="description-tr"> <td class="description-td">slanti.4</td>
                <td class="description-td">CGCGCCCGCTCGTAGTTGAGCACGCTGA</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">474</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">ACAGTGCCTTC</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">ACCCTCG</td>
              </tr> <tr class="description-tr"> <td class="description-td">slanti.5</td>
                <td class="description-td">GCGGAGTCTGGACGTCAGCAGGGCGGGC</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">475</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">CTGGCTTCCCG</td>
              </tr> <tr class="description-tr"> <td class="description-td">UTR2</td>
                <td class="description-td">ATTTGACCCACAGGGACCCCCATCCAG</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">476</td>
              </tr> <tr class="description-tr"> <td class="description-td">FW5</td>
                <td class="description-td">ATGACCGCCCTCCTCGTGAG</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">477</td>
              </tr> <tr class="description-tr"> <td class="description-td">Nam1</td>
                <td class="description-td">GCCACCCCCGCGATGCC</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">478</td>
              </tr> <tr class="description-tr"> <td class="description-td">Nam2</td>
                <td class="description-td">AGCCCTGGCCCCGGCCA</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">479</td>
              </tr> <tr class="description-tr"> <td class="description-td">Nam3</td>
                <td class="description-td">TCCCACGTGCGCAGCAG</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">480</td>
              </tr> <tr class="description-tr"> <td class="description-td">Nam4</td>
                <td class="description-td">AGCAGGACGCAGCGCTG</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">481</td>
              </tr> <tr class="description-tr"> <td class="description-td">PE01</td>
                <td class="description-td">CGCGGTAGTGGCTGCGCAGCAGGGAGCG</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">482</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">CACGGC</td>
              </tr> <tr class="description-tr"> <td class="description-td">PE02</td>
                <td class="description-td">CCAGGGCTTCCCACGTGCGCAGCAGGAC</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">483</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">GCAGCGC</td>
              </tr> <tr class="description-tr"> <td class="description-td">LM101</td>
                <td class="description-td">CTAGTCTAGATCA /</td>
                <td class="description-td">Xba I site / HA tag / hTRT</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">484</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">GCTAGCGTAATCTGGAACATCGTA</td>
                <td class="description-td">into pGRN121</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">TGGGTA / GTCCAGGATGGTCTTGAAGTC</td>
              </tr> <tr class="description-tr"> <td class="description-td">LM103</td>
                <td class="description-td">TACCATGGGCTACCCATACGACGTTCCAG</td>
                <td class="description-td">inserts HA tag into a Nde I site</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">485</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">ATTACGCTCA</td>
                <td class="description-td">at 5′ end of hTRT</td>
              </tr> <tr class="description-tr"> <td class="description-td">LM104</td>
                <td class="description-td">TATGAGCGTAATCTGGAACGTCGTATGGG</td>
                <td class="description-td">anneals to LM103</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">486</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">TAGCCCATGG</td>
              </tr> <tr class="description-tr"> <td class="description-td">LM105</td>
                <td class="description-td">GTGTACGTCGTCGAGCTCCTCAGGTCTGC</td>
                <td class="description-td">change = F560A</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">487</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">CTTTT</td>
                <td class="description-td">(phe &gt; ala)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">ATGTCACGGAG</td>
              </tr> <tr class="description-tr"> <td class="description-td">LM106</td>
                <td class="description-td">GTGTACGTCGTCGAGCTCCTCAGGTCTTT</td>
                <td class="description-td">change = F561A</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">488</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">CGCTTATGTC</td>
                <td class="description-td">(phe &gt; ala)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">ACGGAGACC</td>
              </tr> <tr class="description-tr"> <td class="description-td">LM107</td>
                <td class="description-td">CCTCAGGTCTTTCTTTTGCTGTCACGGAGA</td>
                <td class="description-td">change = Y562A</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">489</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">CAACGTTT</td>
                <td class="description-td">(tyr &gt; ala)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">CAAAAGAACAG</td>
              </tr> <tr class="description-tr"> <td class="description-td">LM108</td>
                <td class="description-td">GGTCTTTCTTTTATGTCGCGGAGACAACG</td>
                <td class="description-td">change = T564A</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">490</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">TTT</td>
                <td class="description-td">(tyr &gt; ala)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">CAAAAGAACAG</td>
              </tr> <tr class="description-tr"> <td class="description-td">LM109</td>
                <td class="description-td">CTTTCTTTTATGTCACGGCGACAACGTTT</td>
                <td class="description-td">change = E565A</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">491</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">CAAAAGAACA</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">G</td>
              </tr> <tr class="description-tr"> <td class="description-td">LM_FFYTE</td>
                <td class="description-td">ATGAGTGTGTACGTCGTCGAGCTCCTCAG</td>
                <td class="description-td">deletion of FFYVTE</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">492</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">GTCTACCACG</td>
                <td class="description-td">(aa560-565)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">TTTCAAAAGAACAGGCTCTTTTTC</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP061:</td>
                <td class="description-td">GGCTGATGAGTGTGTACGTCGTCGA</td>
                <td class="description-td">complement to TCP1.61</td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">493</td>
              </tr> <tr class="description-tr"> <td class="description-td">HUMO1:</td>
                <td class="description-td">ACGTGGTCTCCGTGACATAAAAGAA</td>
                <td class="description-td">to DD motif, designed to possibly anneal to mTRT</td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td">494</td>
              </tr> <tr class="description-tr"> <td class="description-td">HUMO2:</td>
                <td class="description-td">AGGTCTTTCTTTTATGTCACGGA</td>
                <td class="description-td">to DD motif, designed to possibly anneal to mTRT</td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td">495</td>
              </tr> <tr class="description-tr"> <td class="description-td">HUMO3:</td>
                <td class="description-td">CACAGACCCCCGTCGCCTGGTC</td>
                <td class="description-td">designed to possibly anneal to mTRT</td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td">496</td>
              </tr> <tr class="description-tr"> <td class="description-td">HUMO4:</td>
                <td class="description-td">CGGAGTCTGGACGTCAGCAGGGC</td>
                <td class="description-td">designed to possibly anneal to mTRT</td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td">497</td>
              </tr> <tr class="description-tr"> <td class="description-td">SLW F1N</td>
                <td class="description-td"> <u>cgcggatccgtaactaaa</u>ATGCCGCGCGCTCCCCG</td>
                <td class="description-td">for GST fusion construct (782 to 1636)</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td">498</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">CTGC</td>
                <td class="description-td">UC = hTRT seq, lc = Bam HI site + 2 stop codons</td>
              </tr> <tr class="description-tr"> <td class="description-td">SLW F1C</td>
                <td class="description-td"> <u>ccggaattcgttagttactta</u>CAAAGAGGTGGCTTCTT</td>
                <td class="description-td">for GST fusion construct (782 to 1636)</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td">499</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">CGGC</td>
                <td class="description-td">UC = hTRT seq, lc = EcoR I site + 3 stop codons</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">SLW F1N / SLW F1C amplify a 893 nt piece of pGRN121 (782 to 1636)</td>
                <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">SLW F2N</td>
                <td class="description-td"> <u> cgcggatccgtaactaaa</u>GCCACCTCTTTGGAGGGTGCG</td>
                <td class="description-td">for GST fusion construct (1625 to 2458)</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td">500</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">UC = hTRT seq, lc = EcoR I site + 3 stop codons</td>
              </tr> <tr class="description-tr"> <td class="description-td">SLW F2C</td>
                <td class="description-td"> <u> ccgtaattctttagttactta</u>GACCTGAGCAGCTCGACGAC</td>
                <td class="description-td">for GST fusion construct (1625 to 2458)</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td">501</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">UC = hTRT seq, lc = EcoR I site + 3 stop codons</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">SLW F2N / SLW F2C amplify a 872 nt piece of pGRN121 (1625 to 2458)</td>
                <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">SLW F3N</td>
                <td class="description-td"> <u> cgcggatccgtaactaaa</u>ATGAGTGTGTACGTCGTCGAG</td>
                <td class="description-td">for GST fusion construct (2426 to 3274)</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td">502</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">UC = hTRT seq, lc = Bam HI site + 2 stop codons</td>
              </tr> <tr class="description-tr"> <td class="description-td">SLW F3C</td>
                <td class="description-td"> <u> ccttgaattcgttagttactta</u>GATCCCCTGGCACTGGACG</td>
                <td class="description-td">for GST fusion construct (2426 to 3274)</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td">503</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">UC = hTRT seq, lc = EcoR I site + 3 stop codons</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">SLW F3N / SLW F3C amplify a 887 nt piece of pGRN121 (2426 to 3274)</td>
                <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">SLW F4N</td>
                <td class="description-td"> <u> cgcggatccgtaactaaaa</u>ATCCCGCAGGGCTCCATCCTC</td>
                <td class="description-td">for GST fusion construct (3272 to 4177)</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td">504</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">UC = hTRT seq, lc = Bam HI site + 2 stop codons</td>
              </tr> <tr class="description-tr"> <td class="description-td">SLW F4C</td>
                <td class="description-td"> <u> ccggaattcgttattttactta</u>GTCCAGGATGGTCTTGAAGTC</td>
                <td class="description-td">for GST fusion construct (3272 to 4177)</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td">505</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">UC = hTRT seq, lc = EcoR I site + 3 stop codons</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">SLW F4N / SLW F4C amplify a 944 nt piece of pGRN121 (3272 to 4177)</td>
                <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">40-60</td>
                <td class="description-td">GGCATCGCGGGGGTGGCCGGG</td>
                <td class="description-td">phosphorothioatc</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">506</td>
              </tr> <tr class="description-tr"> <td class="description-td">260-280</td>
                <td class="description-td">GGACACCTGGCGGAAGGAGGG</td>
                <td class="description-td">phosphorothioatc</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">507</td>
              </tr> <tr class="description-tr"> <td class="description-td">500-520</td>
                <td class="description-td">GCGTGCCAGCAGGTGAACCAG</td>
                <td class="description-td">phosphorothioatc</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">508</td>
              </tr> <tr class="description-tr"> <td class="description-td">770-790</td>
                <td class="description-td">CTCAGGGGCAGCGCCACGCCT</td>
                <td class="description-td">phosphorothioate</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">509</td>
              </tr> <tr class="description-tr"> <td class="description-td">885-905</td>
                <td class="description-td">AGGTGGCTTCTTCGGCGGGTC</td>
                <td class="description-td">phosphorothioate</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">510</td>
              </tr> <tr class="description-tr"> <td class="description-td">1000-1020</td>
                <td class="description-td">GGACAAGGCGTGTCCCAGGGA</td>
                <td class="description-td">phosphorothioate</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">5I1</td>
              </tr> <tr class="description-tr"> <td class="description-td">1300-1320</td>
                <td class="description-td">GCTGGGGTGACCGCAGCTCGC</td>
                <td class="description-td">phosphorothioate</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">512</td>
              </tr> <tr class="description-tr"> <td class="description-td">1520-1540</td>
                <td class="description-td">GATGAACTTCTTGGTGTTCCT</td>
                <td class="description-td">phosphorothioate</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">513</td>
              </tr> <tr class="description-tr"> <td class="description-td">2110-2130</td>
                <td class="description-td">GTGCGCCAGGCCCTGTGGATA</td>
                <td class="description-td">phosphorothioate</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">514</td>
              </tr> <tr class="description-tr"> <td class="description-td">2295-2315</td>
                <td class="description-td">GCCCATGGGCGGCCTTCTGGA</td>
                <td class="description-td">phosphorothioate</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">515</td>
              </tr> <tr class="description-tr"> <td class="description-td">2450-2470</td>
                <td class="description-td">GAGGCCACTGCTGGCCTCATT</td>
                <td class="description-td">phosphorothioate</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">516</td>
              </tr> <tr class="description-tr"> <td class="description-td">2670-2690</td>
                <td class="description-td">GGGTGAGGTGAGGTGTCACCA</td>
                <td class="description-td">phosphorothioate</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">517</td>
              </tr> <tr class="description-tr"> <td class="description-td">3080-3110</td>
                <td class="description-td">GCTGCAGCACACATGCGTGAAACCTGTACGC</td>
                <td class="description-td">phosphorothioate</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">518</td>
              </tr> <tr class="description-tr"> <td class="description-td">3140-3160</td>
                <td class="description-td">GACGCGCAGGAAAAATGTGGG</td>
                <td class="description-td">phosphorothioate</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">519</td>
              </tr> <tr class="description-tr"> <td class="description-td">3690-3710</td>
                <td class="description-td">CCGAGCGCCAGCCTGTGGGGA</td>
                <td class="description-td">phosphorothioate</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">520</td>
              </tr> <tr class="description-tr"> <td class="description-td">55-75</td>
                <td class="description-td">CAGCGGGGAGCGCGCGGCATC</td>
                <td class="description-td">phosphorothioatc</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">521</td>
              </tr> <tr class="description-tr"> <td class="description-td">151-171</td>
                <td class="description-td">CAGCACCTCGCGGTAGTGGCT</td>
                <td class="description-td">phosphorothioate</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">522</td>
              </tr> <tr class="description-tr"> <td class="description-td">TP1.1</td>
                <td class="description-td">TCAAGCCAAACCTGAATCTGAG</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">523</td>
              </tr> <tr class="description-tr"> <td class="description-td">TP1.2</td>
                <td class="description-td">CCCGAGTGAATCTTTCTACGC</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">524</td>
              </tr> <tr class="description-tr"> <td class="description-td">TP1.3</td>
                <td class="description-td">GTCTCTGGCAGTTTCCTCATCCC</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">525</td>
              </tr> <tr class="description-tr"> <td class="description-td">TP1.4</td>
                <td class="description-td">TTTAGGCATCCTCCCAAGCACA</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">x</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">526</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="10" align="center" rowsep="1" class="description-td" colspan="10"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <heading>IV. TRT Proteins and Peptides</heading> <p>A) Generally</p>
    <p>The invention provides a wide variety of hTRT proteins useful for, inter alia, production of telomerase activity, inhibition of telomerase activity in a cell, induction of an anti-hTRT immune response, as a therapeutic reagent, as a standard or control in a diagnostic assay, as a target in a screen for compounds capable of activation or inhibition of an activity of hTRT or telomerase, and numerous other uses that will be apparent to one of skill or are otherwise described herein. The hTRT of the invention include functionally active proteins (useful for e.g., conferring telomerase activity in a telomerase-negative cell) and variants, inactive variants (useful for e.g., inhibiting telomerase activity in a cell), hTRT polypeptides, and telomerase RNPs (e.g., ribonucleoprotein complexes comprising the proteins) that exhibit one, several, or all of the functional activities of naturally occurring hTRT and telomerase, as discussed in greater detail for illustrative purposes, below.</p>
    <p>In one embodiment, the hTRT protein of the invention is a polypeptide having a sequence as set forth in FIG. 17 (SEQ ID NO:2), or a fragment thereof. In another embodiment, the hTRT polypeptide differs from SEQ ID NO:2 by internal deletions, insertions, or conservative substitutions of amino acid residues. In a related embodiment, the invention provides hTRT polypeptides with substantial similarity to SEQ ID NO:2. The invention further provides hTRT polypeptides that are modified, relative to the amino acid sequence of SEQ ID NO:2, in some manner, e.g., truncated, mutated, derivatized, or fused to other sequences (e.g., to form a fusion protein). Moreover, the present invention provides telomerase RNPs comprising an hTRT protein of the invention complexed with a template RNA (e.g., hTR). In other embodiments, one or more telomerase-associated proteins is associated with hTRT protein and/or hTR.</p>
    <p>The invention also provides other naturally occurring hTRT species or nonnaturally occurring variants, such as proteins having the sequence of, or substantial similarity to SEQ ID NO:5 [[FIG. <b>20</b>], SEQ ID NO:10 [FIG. <b>19</b>], and fragments, variants, or derivatives thereof.</p>
    <p>The invention provides still other hTRT species and variants. One example of an hTRT variant may result from ribosome frameshifting of mRNA encoded by the clone 712562 (SEQ ID NO:3 [FIG. <b>18</b>]) or the pro90 variant hTRT shown in SEQ ID NO:4 [FIG. <b>20</b>] and so result in the synthesis of hTRT polypeptides containing all the TRT motifs (for a general example, see, e.g., Tsuchihashi et al., 1990, <i>Proc. Natl. Acad. Sci. USA </i>87:2516; Craigengen et al., 1987, <i>Cell </i>50:1; Weiss, 1990, <i>Cell </i>62:117). Ribosome frameshifting can occur when specific mRNA sequences or secondary structures cause the ribosome to “stall” and jump one nucleotide forwards or back in the sequence. Thus, a ribosome frameshift event on the 712562 mRNA could cause the synthesis of an approximately 523 amino acid residue polypeptide. A ribosome frameshift event on the pro90 sequence could result in a protein with approximately 1071 residues. It will be appreciated that proteins resulting from ribosome frameshifting can also be expressed by synthetic or recombinant techniques provided by the invention.</p>
    <p>Human TRT proteins, peptides, and functionally equivalent proteins may be obtained by purification, chemical synthesis, or recombinant production, as discussed in greater detail below.</p>
    <p>B) TRT Protein Activities</p>
    <p>The TRT polypeptides of the invention (including fragments, variants, products of alternative alleles, and fusion proteins) can have one or more, or all of the functional activities associated with native hTRT. Except as noted, as used herein, an hTRT or other TRT polypeptide is considered to have a specified activity if the activity is exhibited by either the hTRT protein without an associated RNA (e.g., hTR) or in an hTRT-associated RNA (e.g., hTR) complex. The hTR-binding activity of hTRT is one example of an activity associated with the hTRT protein. Methods for producing complexes of nucleic acids (e.g., hTR) and the hTRT polypeptides of the invention are described infra.</p>
    <p>Modification of the hTRT protein (e.g., by chemical or recombinant means, including mutation or modification of a polynucleotide encoding the hTRT polypeptide or chemical synthesis of a polynucleotide that has a sequence different than a native polynucleotide sequence) to have a different complement of activities than native hTRT can be useful in therapeutic applications or in screening for specific modulators of hTRT or telomerase activity. In addition, assays for various hTRT activities can be particularly useful for identification of agents (e.g., activity modulating agents) that interact with hTRT or telomerase to change telomerase activity.</p>
    <p>The activities of native hTRT, as discussed infra, include telomerase catalytic activity (which may be either processive or non-processive activity); telomerase processivity; conventional reverse transcriptase activity; nucleolytic activity; primer or substrate (telomere or synthetic telomerase substrate or primer) binding activity; dNTP binding activity; RNA (i.e., hTR) binding activity; and protein binding activity (e.g., binding to telomerase-associated proteins, telomere-binding proteins, or to a protein-telomeric DNA complex). It will be understood, however, that present invention also provides hTRT compositions without any particular hTRT activity but with some useful activity related to the hTRT or other TRT proteins (e.g., certain typically short immunogenic peptides, inhibitory peptides).</p>
    <p>1) Telomerase Catalytic Activity</p>
    <p>As used herein, a polypeptide of the invention has “telomerase catalytic activity,” when the polypeptide is capable of extending a DNA primer that functions as a telomerase substrate by adding a partial, one, or more than one repeat of a sequence (e.g., TTAGGG) encoded by a template nucleic acid (e.g., hTR). This activity may be processive or nonprocessive. Processive activity occurs when a telomerase RNP adds multiple repeats to a primer or telomerase before the DNA is released by the enzyme complex. Non-processive activity occurs when telomerase adds a partial, or only one, repeat to a primer and is then released. In vivo, however, a non-processive reaction could add multiple repeats by successive rounds of association, extension, and dissociation. This can occur in vitro as well, but it is not typically observed in standard assays due to the vastly large molar excess of primer over telomerase in standard assay conditions.</p>
    <p>To characterize an hTRT polypeptide as having non-processive activity, a conventional telomerase reaction is performed using conditions that favor a non-processive reaction, for example high temperatures (i.e., 35-40 EC, typically 37 EC), low dGTP concentrations (1 μM or less), high primer concentrations (5 μM or higher), and high dATP/TTP concentrations (2 mM or higher), with the temperature and dGTP typically having the greatest effect. To characterize an hTRT polypeptide as having processive activity, a conventional telomerase reaction is performed using conditions that favor a processive reaction (for example, 27-34° C., typically 30° C.), high dGTP concentration (10 μM or higher), low primer concentration (1 μM or lower), and/or low dATP and TTP concentrations (0.3-1 mM) with temperature and dGTP typically concentration being the most critical. Alternatively, a TRAP assay (for processive or moderately processive activity) or the dot-blot and gel blot assays (for processive activity) may be used. The hTRT polypeptide of the invention can possess a non-processive activity, but not a processive activity (e.g., if an alteration of the hTRT polypeptide reduces or eliminates the ability to translocate), can be solely processive, or can possess both activities.</p>
    <p>a) Non-processive Activity</p>
    <p>A non-processive telomerase catalytic activity can extend the DNA primer from the position where the 3′ end anneals to the RNA template to the 5′ end of the template sequence, typically terminating with the addition of the first G residue (as, for example, when the template is hTR). As shown below, the exact number of nucleotides added is dependent on the position of the 3′ terminal nucleotide of the primer in the TTAGGG repeat sequence.</p>
    <heading>Nonprocessive Activity</heading> <p>i) - - - TTAGGGttag (DNA)</p>
    <p>3′ - - - AUCCCAAUC - - - 5′ (RNA) SEQ ID NO:527</p>
    <p>ii) - - - TTAGggttag (DNA)</p>
    <p>3′ - - - AUCCCAAUC - - - 5′ (RNA) SEQ ID NO:527</p>
    <p>In DNA, UC,=primer, lc=added nucleotides</p>
    <p>Thus, 4 nucleotides are added to the - - - TTAGGG primer (i) while 6 nucleotides are added to the - - - TTAG primer (ii). The first repeat added by telomerase in a processive reaction is equivalent to this step; however, in a processive reaction telomerase performs a translocation step where the 3′ end is released and re-bound at the 3′ region of the template in a position sufficient to prime addition of another repeat (see Morin, 1997, <i>Eur. J. Cancer </i>33:750).</p>
    <p>A fully non-processive reaction produces only one band in a conventional assay using a single synthetic primer. Because this result could also be produced by other enzymes, such as a terminal transferase activity, it may be desirable in some applications to verify that the product is a result of a telomerase catalytic activity. A telomerase (comprising hTRT) generated band can be distinguished by several additional characteristics. The number of nucleotides added to the end of the primer should be consistent with the position of the primer 3′ end. Thus, a - - - TTAGGG primer should have 4 nucleotides added and a - - - TTAG primer should have 6 nucleotides added (see above). In practice, two or more sequence permuted primers can be used which have the same overall length but different 5′ and 3′ endpoints. As an illustrative example, the non-processive extension of primers 5′-TTAGGGTTAGGGTTAGGG (SEQ ID NO:528) and 5′-GTTAGGGTTAGGGTTAGG (SEQ ID NO:529) will generate products whose absolute length will be one nucleotide different (4 added to 5′-TTAGGGTTAGGGTTAGGG (SEQ ID NO:528) for a 22 nt total length, and 5 added to 5′-GTTAGGGTTAGGGTTAGG (SEQ ID NO:529) for a 23 nt total length). The nucleotide dependence of the reaction should be consistent with the position of the primer terminus. Thus, a - - - TTAGGG primer product should require dGTP, TTP, and dATP, but not dCTP, and a - - - AGGGTT primer product should require dGTP and DATP, but not TTP or dCTP. The activity should be sensitive to RNAase or micrococcal nuclease pre-treatment (see Morin, 1989, <i>Cell </i>59: 521) under conditions that will degrade hTR and so eliminate the template.</p>
    <p>b) Processive Activity</p>
    <p>In practice, a processive activity is easily observed by the appearance of a six nucleotide ladder in a conventional assay, TRAP assay, or gel-blot assay. A dot-blot assay can also be used, but no ladder is detected in such a method. The conventional assay is described in Morin, 1989, <i>Cell </i>59:521, which is incorporated herein in its entirety and for all purposes. The TRAP assay is described in U.S. Pat. No. 5,629,154; see also, PCT publication WO 97/15687, PCT publication WO 95/13381; Krupp et al. <i>Nucleic Acids Res., </i>1997, 25: 919; and Wright et al., 1995, <i>Nuc. Acids Res. </i>23:3794, each of which is incorporated herein in its entirety and for all purposes. The dot blot immunoassay is described in detail in co-pending U.S. patent application Ser. No. 08/833,377, filed Apr. 14, 1997, which is incorporated herein by reference in its entirety and for all purposes. The dot blot assay can be used in a format in which a non-processive activity, which does not add the 3 or more repeats required for stable hybridization of the (CCCUAA)n probe used to detect the activity, is tested with compounds or hTRT variants to determine if the same generates processivity, i.e., if the probe detects an expected telomerase substrate, then the compound or mutant is able to change the non-processive activity to a processive activity. Other assays for processive telomerase catalytic activity can also be used, e.g., the stretch PCR assay of Tatematsu et al., 1996, <i>Oncogene </i>13:2265. The gel-blot assay, a combination of the conventional and dot blot assays can also be used. In this variation a conventional assay is performed with no radiolabeled nucleotide and with high dGTP concentrations (e.g., 0.1-2 mM). After performing the conventional assay, the synthesized DNA is separated by denaturing PAGE and transferred to a membrane (e.g., nitrocellulose). Telomeric DNA (the product of telomerase—an extended telomerase primer or substrate) can then be detected by methods such as hybridization using labeled telomeric DNA probes (e.g., probes containing the CCCTAA sequence, as used in the dot blot assay, supra). An advantage of this technique is that it is more sensitive than the conventional assay and provides information about the size of the synthesized fragments and processivity of the reaction.</p>
    <p>c) Activity Determinations</p>
    <p>The telomerase activity of an hTRT polypeptide can be determined using an unpurified, partially purified or substantially purified hTRT polypeptide (e.g., in association with hTR), in vitro, or after expression in vivo. For example, telomerase activity in a cell (e.g., a cell expressing a recombinant hTRT polypeptide of the invention) can be assayed by detecting an increase or decrease in the length of telomeres. Typically assays for telomerase catalytic activity are carried out using an hTRT complexed with hTR; however, alternative telomerase template RNAs may be substituted, or one may conduct assays to measure another activity, such as telomerase-primer binding. Assays to determine the length of telomeres are known in the art and include hybridization of probes to telomeric DNA (an amplification step can be included) and TRF analysis i.e., the analysis of telomeric DNA restriction fragments [TRFs] following restriction endonuclease digestion, see PCT publications WO 93/23572 and WO 96/41016; Counter et al., 1992, <i>EMBO J. </i>11:1921; Allsopp et al., 1992, <i>Proc. Nat'l. Acad. Sci. USA </i>89:10114; Sanno, 1996, <i>Am J Clin Pathol </i>106:16 and Sanno, 1997, <i>Neuroendocrinology </i>65:299.</p>
    <p>The telomerase catalytic activity of an hTRT polypeptide may be determined in a number of ways using the assays supra and other telomerase catalytic activity assays. According to one method, the hTRT protein is expressed (e.g., as described infra) in a telomerase negative human cell in which hTR is expressed (i.e., either normally in the cell or through recombinant expression), and the presence or absence of telomerase activity in the cell or cell lysate is determined. Examples of suitable telomerase-negative cells are IMR 90 (ATCC, #CCL-186) or BJ cells (human foreskin fibroblast line; see, e.g., Feng et al., 1995, <i>Science </i>269:1236). Other examples include retinal pigmented epithelial cells (RPE), human umbilical vein endothelial cells (HUVEC; ATCC #CRL-1730), human aortic endothelial cells (HAEC; Clonetics Corp, #CC-2535), and human mammary epithelial cells (HME; Hammond et al., 1984, <i>Proc. Nat'l. Acad. Sci. USA </i>81:5435; Stampfer, 1985, <i>J. Tissue Culture Methods </i>9:107). In an alternative embodiment, the hTRT polypeptide is expressed (e.g., by transfection with an hTRT expression vector) in a telomerase positive cell, and an increase in telomerase activity in the cell compared to an untransfected control cell is detected if the polypeptide has telomerase catalytic activity. Usually the telomerase catalytic activity in a cell transfected with a suitable expression vector expressing hTRT will be significantly increased, such as at least about 2-fold, at least about 5-fold, or even at least about 10-fold to 100-fold or even 1000-fold higher than in untransfected (control) cells.</p>
    <p>In an alternative embodiment, the hTRT protein is expressed in a cell (e.g., a telomerase negative cell in which hTR is expressed) as a fusion protein (see infra) having a label or an “epitope tag” to aid in purification. In one embodiment, the RNP is recovered from the cell using an antibody that specifically recognizes the tag. Preferred tags are typically short or small and may include a cleavage site or other property that allows the tag to be removed from the hTRT polypeptide. Examples of suitable tags include the Xpress™ epitope (Invitrogen, Inc., San Diego Calif.), and other moieties that can be specifically bound by an antibody or nucleic acid or other equivalent method such as those described in Example 6. Alternative tags include those encoded by sequences inserted, e.g., into SEQ ID NO:1 upstream of the ATG codon that initiates translation of the protein of SEQ ID NO:2, which may include insertion of a (new) methionine initiation codon into the upstream sequence.</p>
    <p>It will be appreciated that when an hTRT variant is expressed in a cell (e.g., as a fusion protein) and subsequently isolated (e.g., as a ribonucleoprotein complex), other cell proteins (i.e., telomerase-associated proteins) may be associated with (directly or indirectly bound to) the isolated complex. In such cases, it will sometimes be desirable to assay telomerase activity for the complex containing hTRT, hTR and the associated proteins.</p>
    <p>2) Other Telomerase or TRT Protein Activities</p>
    <p>The hTRT polypeptides of the invention include variants that lack telomerase catalytic activity but retain one or more other activities of telomerase. These other activities and the methods of the invention for measuring such activities include (but are not limited to) those discussed in the following sections.</p>
    <p>a) Conventional Reverse Transcriptase Activity</p>
    <p>Telomerase conventional reverse transcriptase activity is described in, e.g., Morin, 1997, supra, and Spence et al., 1995, <i>Science </i>267:988. Because hTRT contains conserved amino acid motifs that are required for reverse transcriptase catalytic activity, hTRT has the ability to transcribe certain exogenous (e.g., non-hTR) RNAs. A conventional RT assay measures the ability of the enzyme to transcribe an RNA template by extending an annealed DNA primer. Reverse transcriptase activity can be measured in numerous ways known in the art, for example, by monitoring the size increase of a labeled nucleic acid primer (e.g., RNA or DNA), or incorporation of a labeled dNTP. See, e.g., Ausubel et al., supra.</p>
    <p>Because hTRT specifically associates with hTR, it can be appreciated that the DNA primer/RNA template for a conventional RT assay can be modified to have characteristics related to hTR and/or a telomeric DNA primer. For example, the RNA can have the sequence (CCCTAA)<sub>n</sub>, where n is at least 1, or at least 3, or at least 10 or more (SEQ ID NO:530). In one embodiment, the (CCCTAA)<sub>n </sub>region is at or near the 5′ terminus of the RNA (similar to the 5′ locations of template regions in telomerase RNAs). Similarly, the DNA primer may have a 3′ terminus that contains portions of the TTAGGG telomere sequence, for example X<sub>n</sub>TTAG (SEQ ID NO:531), X<sub>n</sub>AGGG (SEQ ID NO:532), X<sub>n</sub>(TTAGGG)<sub>q</sub>TTAG (SEQ ID NOS:533-536), etc., where X is a non-telomeric sequence and n is 8-20, or 6-30, and q is 1-4. In another embodiment, the DNA primer has a 5′ terminus that is non-complementary to the RNA template, such that when the primer is annealed to the RNA, the 5′ terminus of the primer remains unbound. Additional modifications of standard reverse transcription assays that may be applied to the methods of the invention are known in the art.</p>
    <p>b) Nucleolytic Activity</p>
    <p>Telomerase nucleolytic activity is described in e.g., Morin, 1997, supra; Collins and Grieder, 1993, <i>Genes and Development </i>7:1364. Telomerase possesses a nucleolytic activity (Joyce and Steitz, 1987, <i>Trends Biochem. Sci. </i>12:288); however, telomerase activity has defining characteristics. Telomerase preferentially removes nucleotides, usually only one, from the 3′ end of an oligonucleotide when the 3′ end of the DNA is positioned at the 5′ boundary of the DNA template sequence, in humans and Tetrahymena, this nucleotide is the first G of the telomeric repeat (TTAGG in humans). Telomerase preferentially removes G residues but has nucleolytic activity against other nucleotides. This activity can be monitored. Two different methods are described here for illustrative purposes. One method involves a conventional telomerase reaction with a primer that binds the entire template sequence (i.e., terminating at the template boundary; 5′-TAGGGATTAG (SEQ ID NO:537) in humans). Nucleolytic activity is observed by monitoring the replacement of the last dG residue with a radiolabeled dGTP provided in the assay. The replacement is monitored by the appearance of a band at the size of the starting primer as shown by gel electrophoresis and autoradiography.</p>
    <p>A preferred method uses a DNA primer that has a “blocked” 3′ terminus that cannot be extended by telomerase. The 3′-blocked primer can be used in a standard telomerase assay but will not be extended unless the 3′ nucleotide is removed by the nucleolytic activity of telomerase. The advantage of this method is that telomerase activity can be monitored by any of several standard means, and the signal is strong and easy to quantify. The blocking of the 3′ terminus of the primer can be accomplished in several ways. One method is the addition of a 3′-deoxy-dNTP residue at the 3′ terminus of the primer using standard oligonucleotide synthesis techniques. This terminus has a 2′ OH but not the 3′ OH required for telomerase. Other means of blocking the 3′ terminus exist, for instance, a 3′ dideoxy terminus, a 3′-amine terminus, and others. An example of a primer for an hTRT nucleolytic assay is 5′-TTAGGGTTAGGGTTA (G<sub>3′H</sub>) (SEQ ID NO:538) where the last residue denotes a 3′-deoxy-guanosine residue (Glen Research, Sterling, Va.). Numerous other variations for a suitable primer based on the disclosure are known to those of skill in the art.</p>
    <p>c) Primer (Telomere) Binding Activity</p>
    <p>Telomerase primer (telomere) binding activity is described in e.g., Morin, 1997, supra; Collins et al., 1995, <i>Cell </i>81:677; Harrington et al, 1995, <i>J. Biol. Chem. </i>270:8893. Telomerase is believed to have two sites which bind a telomeric DNA primer. The RT motifs associated with primer binding indicate hTRT and/or hTRT/hTR possesses DNA primer binding activity. There are several ways of assaying primer binding activity; however, a step common to most methods is incubation of a labeled DNA primer with hTRT or hTRT/hTR or other TRT/TR combinations under appropriate binding conditions. Also, most methods employ a means of separating unbound DNA from protein-bound DNA; those methods include the following.</p>
    <p>i) Gel-shift assays (also called electrophoretic/mobility shift assays) are those in which unbound DNA primer is separated from protein-bound DNA primer by electrophoresis on a nondenaturing gel (Ausubel et al., supra).</p>
    <p>ii) Matrix binding assays include several variations to the basic technique, which involves binding the hTRT or hTRT/hTR complex to a matrix (e.g., nitrocellulose), either before or after incubation with the labeled primer. By binding the hTRT to a matrix, the unbound primer can be mechanically separated from bound primer. Residual unbound DNA can be removed by washing the membrane prior to quantitation. Those of skill recognize there are several means of coupling proteins to such matrices, solid supports, and membranes, including chemical, photochemical, UV cross-linking, antibody/epitope, and non-covalent (hydrophobic, electrostatic, etc.) interactions.</p>
    <p>The DNA primer can be any DNA with an affinity for telomerase, such as, for example, a telomeric DNA primer like (TTAGGG)<sub>n</sub>, where n could be 1-10 and is typically 3-5 (SEQ ID NO:539). The 3′ and 5′ termini can end in any location of the repeat sequence. The primer can also have 5′ or 3′ extensions of non-telomeric DNA that could facilitate labeling or detection. The primer can also be derivatized, e.g., to facilitate detection or isolation.</p>
    <p>d) dNTP Binding Activity</p>
    <p>Telomerase dNTP binding activity is described in e.g., Morin, 1997, supra; Spence et al., supra. Telomerase requires dNTPs to synthesize DNA. The hTRT protein has a nucleotide binding activity and can be assayed for dNTP binding in a manner similar to other nucleotide binding proteins (Kantrowitz et al., 1980, <i>Trends Biochem. Sci. </i>5:124). Typically, binding of a labeled dNTP or dNTP analog can be monitored as is known in the art for non-telomerase RT proteins.</p>
    <p>e) RNA (i.e., hTR) Binding Activity</p>
    <p>Telomerase RNA (i.e., hTR) binding activity is described in e.g., Morin, 1997, supra; Harrington et al., 1997, <i>Science </i>275:973; Collins et al., 1995, <i>Cell </i>81:677. The RNA binding activity of a TRT protein of the invention may be assayed in a manner similar to the DNA primer binding assay described supra, using a labeled RNA probe. Methods for separating bound and unbound RNA and for detecting RNA are well known in the art and can be applied to the activity assays of the invention in a manner similar to that described for the DNA primer binding assay. The RNA can be full length hTR, fragments of hTR or other RNAs demonstrated to have an affinity for telomerase or hTRT. See U.S. Pat. No. 5,583,016 and PCT Pub. No. 96/40868.</p>
    <p>3) Telomerase Motifs as Targets</p>
    <p>The present invention, as noted supra, provides in addition to recombinant hTRT with a full complement (as described supra) of activities, hTRT polypeptides having less than the full complement of the telomerase activities of naturally occurring telomerase or hTRT or other TRT proteins. It will be appreciated that, in view of the disclosure herein of the RT and telomerase-specific motifs of TRT, alteration or mutation of conserved amino acid residues, such as are found in the motif sequences discussed supra, will result in loss-of activity mutants useful for therapeutic, drug screening and characterization, and other uses. For example, as described in Example 1, deletion of motifs B through D in the RT domains of the endogenous TRT gene in <i>S. pombe </i>resulted in haploid cells in which telomere progressively shortened to the point where hybridization of a telomere probe to telomeric repeats became almost undetectable, indicating a loss of telomerase catalytic activity. Similarly, alterations in the WxGxS (SEQ ID NO:540) site of motif E can affect telomerase DNA primer binding or function. Additionally, alterations of the amino acids in the motifs A, B′, and C can affect the catalytic activity of telomerase. Mutation of the DD motif of hTRT can significantly reduce or abolish telomerase activity (see Example 16).</p>
    <p>C) Synthesis of HTRT and Other TRT Polypeptides</p>
    <p>The invention provides a variety of methods for making the hTRT and other TRT polypeptides disclosed herein. In the following sections, chemical synthesis and recombinant expression of hTRT proteins, including fusion proteins, is described in some detail.</p>
    <p>1) Chemical Synthesis</p>
    <p>The invention provides hTRT polypeptides synthesized, entirely or in part, using general chemical methods well known in the art (see e.g., Caruthers et al., 1980, <i>Nucleic Acids Res. Symp. Ser., </i>215-223; and Horn et al., 1980, <i>Nucleic Acids Res. Symp. Ser., </i>225-232). For example, peptide synthesis can be performed using various solid-phase techniques (Roberge, et al., 1995, <i>Science </i>269:202), including automated synthesis (e.g., using the Perkin Elmer ABI 431A Peptide Synthesizer in accordance with the instructions provided by the manufacturer). When full length protein is desired, shorter polypeptides may be fused by condensation of the amino terminus of one molecule with the carboxyl terminus of the other molecule to form a peptide bond.</p>
    <p>The newly synthesized peptide can be substantially purified, for example, by preparative high performance liquid chromatography (e.g., Creighton, Proteins, Structures and Molecular Principles, WH Freeman and Co, New York N.Y. [1983]). The composition of the synthetic peptides (or any other peptides or polypeptides of the invention) may be confirmed by amino acid analysis or sequencing (e.g., the Edman degradation procedure; Creighton, supra). Importantly, the amino acid sequence of hTRT, or any part thereof, may be altered during direct synthesis and/or combined using chemical methods with sequences from other proteins or otherwise, or any part thereof or for any purpose, to produce a variant polypeptide of the invention.</p>
    <p>2) Recombinant Expression of hTRT and Other TRT Proteins</p>
    <p>The present invention provides methods, reagents, vectors, and cells useful for expression of hTRT polypeptides and nucleic acids using in vitro (cell-free), ex vivo or in vivo (cell or organism-based) recombinant expression systems. In one embodiment, expression of the hTRT protein, or fragment thereof, comprises inserting the coding sequence into an appropriate expression vector (i.e., a vector that contains the necessary elements for the transcription and translation of the inserted coding sequence required for the expression system employed). Thus, in one aspect, the invention provides for a polynucleotide substantially identical in sequence to an hTRT gene coding sequence at least 25 nucleotides, and preferably for many applications 50 to 100 nucleotides or more, of the hTRT cDNAs or genes of the invention, which is operably linked to a promoter to form a transcription unit capable of expressing an hTRT polypeptide. Methods well known to those skilled in the art can be used to construct the expression vectors containing an hTRT sequence and appropriate transcriptional or translational controls provided by the present invention (see, e.g., Sambrook et al., supra, Ausubel et al. supra, and this disclosure).</p>
    <p>The hTRT polypeptides provided by the invention include fusion proteins that contain hTRT polypeptides or fragments of the hTRT protein. The fusion proteins are typically produced by recombinant means, although they may also be made by chemical synthesis. Fusion proteins can be useful in providing enhanced expression of the hTRT polypeptide constructs, or in producing hTRT polypeptides having other desirable properties, for example, comprising a label (such as an enzymatic reporter group), binding group, or antibody epitope. An exemplary fusion protein, comprising hTRT and enhanced green fluorescent protein (EGFP) sequences is described in Example 15, infra. It will be apparent to one of skill that the uses and applications discussed in Example 15 and elsewhere herein are not limited to the particular fusion protein, but are illustrative of the uses of various fusion constructs.</p>
    <p>The fusion protein systems of the invention can also be used to facilitate efficient production and isolation of hTRT proteins or peptides. For example, in some embodiments, the non-hTRT sequence portion of the fusion protein comprises a short peptide that can be specifically bound to an immobilized molecule such that the fusion protein can be separated from unbound components (such as unrelated proteins in a cell lysate). One example is a peptide sequence that is bound by a specific antibody. Another example is a peptide comprising polyhistidine tracts e.g. (His)<sub>6 </sub>or histidine-tryptophan sequences that can be bound by a resin containing nickel or copper ions (i.e., metal-chelate affinity chromatography). Other examples include Protein A domains or fragments, which allow purification on immobilized immunoglobulin, and the domain utilized in the FLAGS, extension/affinity purification system (Immunex Corp, Seattle Wash.). In some embodiments, the fusion protein includes a cleavage site so that the hTRT or other TRT polypeptide sequence can be easily separated from the non-hTRT peptide or protein sequence. In this case, cleavage may be chemical (e.g., cyanogen bromide, 2-(2-nitrophenylsulphenyl)-3-methyl-3′-bromoindolene, hydroxylamine, or low pH) or enzymatic (e.g., Factor Xa, enterokinase). The choice of the fusion and cleavage systems may depend, in part, on the portion (i.e., sequence) of the hTRT polypeptide being expressed. Fusion proteins generally are described in Ausubel et al., supra, Ch. 16, Kroll et al., 1993, <i>DNA Cell. Biol. </i>12:441, and the Invitrogen 1997 Catalog (Invitrogen Inc, San Diego Calif.). Other exemplary fusion proteins of the invention with epitope tags or tags and cleavage sites are provided in Example 6, infra.</p>
    <p>It will be appreciated by those of skill that, although the expression systems discussed in this section are focused on expression of hTRT polypeptides, the same or similar cells, vectors and methods may be used to express hTRT polynucleotides of the invention, including sense and antisense polynucleotides without necessarily desiring production of hTRT polypeptides. Typically, expression of a polypeptide requires a suitable initiation codon (e.g., methionine), open reading frame, and translational regulatory signals (e.g., a ribosome binding site, a termination codon) which may be omitted when translation of a nucleic acid sequence to produce a protein is not desired.</p>
    <p>Expression of hTRT polypeptides and polynucleotides may be carried out to accomplish any of several related benefits provided by the present invention. One illustrative benefit is expression of hTRT polypeptides that are subsequently isolated from the cell in which they are expressed (for example for production of large amounts of hTRT for use as a vaccine or in screening applications to identify compounds that modulate telomerase activity). A second illustrative benefit is expression of hTRT in a cell to change the phenotype of the cell (as in gene therapy applications). Nonmammalian cells can be used for expression of hTRT for purification, while eukaryotic especially mammalian cells (e.g., human cells) can be used not only for isolation and purification of hTRT but also for expression of hTRT when a change in phenotype in a cell is desired (e.g., to effect a change in proliferative capacity as in gene therapy applications). By way of illustration and not limitation, hTRT polypeptides having one or more telomerase activities (e.g. telomerase catalytic activity) can be expressed in a host cell to increase the proliferative capacity of a cell (e.g., immortalize a cell) and, conversely, hTRT antisense polynucleotides or inhibitory polypeptides typically can be expressed to reduce the proliferative capacity of a cell (e.g., of a telomerase positive malignant tumor cell). Numerous specific applications are described herein, e.g., in the discussion of uses of the reagents and methods of the invention for therapeutic applications, below.</p>
    <p>Illustrative useful expression systems (cells, regulatory elements, vectors and expression) of the present invention include a number of cell-free systems such as reticulocyte lysate and wheat germ systems using hTRT polynucleotides in accordance with general methods well known in the art (see, e.g., Ausubel et al. supra at Ch. 10). In alternative embodiments, the invention provides reagents and methods for expressing hTRT in prokaryotic or eukaryotic cells. Thus, the present invention provides nucleic acids encoding hTRT polynucleotides, proteins, protein subsequences, or fusion proteins that can be expressed in bacteria, fungi, plant, insect, and animal, including human cell expression systems known in the art, including isolated cells, cell lines, cell cultures, tissues, and whole organisms. As will be understood by those of skill, the hTRT polynucleotides introduced into a host cell or cell free expression system will usually be operably linked to appropriate expression control sequences for each host or cell free system.</p>
    <p>Useful bacterial expression systems include <i>E. coli</i>, bacilli (such as <i>Bacillus subtilus</i>), other enterobacteriaceae (such as Salmonella, Serratia, and various Pseudomonas species) or other bacterial hosts (e.g., <i>Streptococcus cremoris, Streptococcus lactis, Streptococcus thermophilus, Leuconostoc citrovorum, Leuconostoc mesenteroides, Lactobacillus acidophilus, Lactobacillus lactis, Bifidobacterium bifidum, Bifidobacteriu breve</i>, and <i>Bifidobacterium longum</i>). The hTRT expression constructs useful in prokaryotes include recombinant bacteriophage, plasmid or cosmid DNA expression vectors, or the like, and typically include promoter sequences. Illustrative promoters include inducible promoters, such as the lac promoter, the hybrid lacZ promoter of the Bluescript7 phagemid [Stratagene, La Jolla Calif.] or pSport1 [Gibco BRL]; phage lambda promoter systems; a tryptophan (trp) promoter system; and ptrp-lac hybrids and the like. Bacterial expression constructs optionally include a ribosome binding site and transcription termination signal regulatory sequences. Illustrative examples of specific vectors useful for expression include, for example, pTrcHis2, (Invitrogen, San Diego Calif.), pThioHis A, B &amp; C, and numerous others known in the art or that may be developed (see, e.g. Ausubel). Useful vectors for bacteria include those that facilitate production of hTRT-fusion proteins. Useful vectors for high level expression of fusion proteins in bacterial cells include, but are not limited to, the multifunctional <i>E. coli </i>cloning and expression vectors such as Bluescript7 (Stratagene), noted above, in which the sequence encoding hTRT protein, an hTRT fusion protein or an hTRT fragment may be ligated into the vector in-frame with sequences for the amino-terminal Met and the subsequent 7 residues of β-galactosidase so that a hybrid protein is produced (e.g., pIN vectors; Van Heeke and Schuster, 1989, <i>J. Biol. Chem., </i>264:5503). Vectors such as pGEX vectors (e.g., pGEX-2TK; Pharmacia Biotech) may also be used to express foreign polypeptides, such as hTRT protein, as fusion proteins with glutathione S-transferase (GST). Such fusion proteins may be purified from lysed cells by adsorption to glutathione-agarose beads followed by elution in the presence of free glutathione. Proteins made in such systems often include enterokinase, thrombin or factor Xa protease cleavage sites so that the cloned polypeptide of interest can be released from the GST moiety at will, as may be useful in purification or other applications. Other examples are fusion proteins comprising hTRT and the <i>E. coli </i>Maltose Binding Protein (MBP) or <i>E. Coli </i>thioredoxin. Illustrative examples of hTRT expression constructs useful in bacterial cells are provided in Example 6, infra.</p>
    <p>The invention further provides hTRT polypeptides expressed in fungal systems, such as Dictyostelium and, preferably, yeast, such as <i>Saccharomyces cerevisiae, Pichia pastoris, Torulopsis holmil, Saccharomyces fragilis, Saccharomyces lactis, Hansenula polymorpha </i>and <i>Candida pseudotropicalis</i>. When hTRT is expressed in yeast, a number of suitable vectors are available, including plasmid and yeast artificial chromosomes (YACs) vectors. The vectors typically include expression control sequences, such as constitutive or inducible promoters (e.g., such as alpha factor, alcohol oxidase, PGH, and 3-phosphoglycerate kinase or other glycolytic enzymes), and an origin of replication, termination sequences and the like, as desired. Suitable vectors for use in Pichia include pPICZ, His6/pPICZB, pPICZalpha, pPIC3.5K, pPIC9K, pA0815, pGAP2A, B &amp; C, pGAP2alpha A, B, and C (Invitrogen, San Diego, Calif.) and numerous others known in the art or to be developed. In one embodiment, the vector His6/pPICZB (Invitrogen, San Diego, Calif.) is used to express a His<sub>6</sub>-hTRT fusion protein in the yeast <i>Pichia pastoris</i>. An example of a vector useful in Saccharomyces is pYES2 (Invitrogen, San Diego, Calif.). Illustrative examples of hTRT expression constructs useful in yeast are provided in Example 6, infra.</p>
    <p>The hTRT polypeptides of the invention may also be expressed in plant cell systems transfected with plant or plant virus expression vectors (e.g., cauliflower mosaic virus, CaMV; tobacco mosaic virus, TMV) or transformed with bacterial expression vectors (e.g., Ti or pBR<sub>322 </sub>plasmid). In cases where plant virus expression vectors are used, the expression of an hTRT-encoding sequence may be driven by any of a number of promoters. For example, viral promoters such as the 35S and 19S promoters of CaMV (Brisson et al., 1984, <i>Nature </i>310:511-514) may be used alone or in combination with the omega leader sequence from TMV (Takamatsu et al., 1987, <i>EMBO J., </i>6:307-311). Alternatively, plant promoters such as that from the small subunit gene of RUBISCO (Coruzzi et al., 1984, <i>EMBO J., </i>3:1671-1680; Broglie et al., 1984, <i>Science </i>224:838-843) or heat shock promoters (Winter and Sinibaldi, 1991, <i>Results Probl. Cell Differ., </i>17:85), or storage protein gene promoters may be used. These constructs can be introduced into plant cells by direct DNA transformation or pathogen-mediated transfection (for reviews of such techniques, see Hobbs or Murry, 1992, in McGraw Hill Yearbook of Science and Technology McGraw Hill New York N.Y., pp. 191-196 [1992]; or Weissbach and Weissbach, 1988, Methods for Plant Molecular Biology, Academic Press, New York N.Y., pp. 421-463).</p>
    <p>Another expression system provided by the invention for expression of hTRT protein is an insect system. A preferred system uses a baculovirus polyhedrin promoter. In one such system, <i>Autographa californica </i>nuclear polyhedrosis virus (AcNPV) is used as a vector to express foreign genes in <i>Spodoptera frugiperda </i>cells or in Trichoplusia larvae. The sequence encoding the gene of interest may be cloned into a nonessential region of the virus, such as the polyhedrin gene, and placed under control of the polyhedrin promoter. Successful insertion of the sequence, e.g., encoding the hTRT protein, will render the polyhedrin gene inactive and produce recombinant virus lacking coat protein. The recombinant viruses are then used to infect <i>S. frugiperda </i>cells or Trichoplusia larvae, in which the hTRT sequence is then expressed (see, for general methods, Smith et al., <i>J. Virol., </i>46:584 [1983]; Engelhard et al., <i>Proc. Natl. Acad. Sci. </i>91:3224-7 [1994]). Useful vectors for baculovirus expression include pBlueBacHis2 A, B &amp; C, pBlueBac4.5, pMelBacB and numerous others known in the art or to be developed. Illustrative examples of hTRT expression constructs useful in insect cells are provided in Example 6, infra.</p>
    <p>The present invention also provides expression systems in mammals and mammalian cells. As noted supra, hTRT polynucleotides may be expressed in mammalian cells (e.g., human cells) for production of significant quantities of hTRT polypeptides (e.g., for purification) or to change the phenotype of a target cell (e.g., for purposes of gene therapy, cell immortalization, or other). In the latter case, the hTRT polynucleotide expressed may or may not encode a polypeptide with a telomerase catalytic activity. That is, expression may be of a sense or antisense polynucleotide, an inhibitory or stimulatory polypeptide, a polypeptide with zero, one or more telomerase activities, and other combinations and variants disclosed herein or apparent to one of skill upon review of this disclosure.</p>
    <p>Suitable mammalian host tissue culture cells for expressing the nucleic acids of the invention include any normal mortal or normal or abnormal immortal animal or human cell, including: monkey kidney CV1 line transformed by SV40 (COS-7, ATCC CRL 1651); human embryonic kidney line (293; Graham et al., <i>J. Gen. Virol. </i>36:59 (1977)); baby hamster kidney cells (BHK, ATCC CCL 10); CHO (ATCC CCL 61 and CRL 9618); mouse sertoli cells (TM4, Mather, <i>Biol. Reprod. </i>23:243-251 (1980)); monkey kidney cells (CV1 ATCC CCL 70); African green monkey kidney cells (VERO-76, ATCC CRL 1587); human cervical carcinoma cells (HeLa, ATCC CCL 2); canine kidney cells (MDCK, ATCC CCL 34); buffalo rat liver cells (BRL 3A, ATCC CRL 1442); human lung cells (W138, ATCC CCL 75); human liver cells (Hep G2, HB 8065); mouse mammary tumor (MMT 060562, ATCC CCL51); TRI cells (Mather, et al., <i>Annals N.Y. Acad. Sci. </i>383:44-46 (1982); MDCK cells (ATCC CCL 34 and CRL 6253); HEK 293 cells (ATCC CRL 1573); and WI-38 cells (ATCC CCL 75; ATCC: American Type Culture Collection, Rockville, Md.). The use of mammalian tissue cell culture to express polypeptides is discussed generally in Winnacker, From Genes to Clones (VCH Publishers, N.Y., N.Y., 1987).</p>
    <p>For mammalian host cells, viral-based and nonviral expression systems are provided. Nonviral vectors and systems include plasmids and episomal vectors, typically with an expression cassette for expressing a protein or RNA, and human artificial chromosomes (see, e.g., Harrington et al., 1997, <i>Nat Genet </i>15:345). For example, nonviral vectors useful for expression of hTRT polynucleotides and polypeptides in mammalian (e.g., human) cells include pcDNA3.1/His, pEBVHis A, B &amp; C, (Invitrogen, San Diego Calif.), MPSV vectors, others described in the Invitrogen 1997 Catalog (Invitrogen Inc, San Diego Calif.), which is incorporated in its entirety herein, and numerous others known in the art for other proteins. Illustrative examples of hTRT expression constructs useful in mammalian cells are provided in Example 6, infra.</p>
    <p>Useful viral vectors include vectors based on retroviruses, adenoviruses, adenoassociated viruses, herpes viruses, vectors based on SV40, papilloma virus, HBP Epstein Barr virus, vaccinia virus vectors and Semliki Forest virus (SFV). SFV and vaccinia vectors are discussed generally in Ausubel et al., supra, Ch 16. These vectors are often made up of two components, a modified viral genome and a coat structure surrounding it (see generally Smith, 1995, <i>Annu. Rev. Microbiol. </i>49: 807), although sometimes viral vectors are introduced in naked form or coated with proteins other than viral proteins. However, the viral nucleic acid in a vector may be changed in many ways, for example, when designed for gene therapy. The goals of these changes are to disable growth of the virus in target cells while maintaining its ability to grow in vector form in available packaging or helper cells, to provide space within the viral genome for insertion of exogenous DNA sequences, and to incorporate new sequences that encode and enable appropriate expression of the gene of interest. Thus, vector nucleic acids generally comprise two components: essential cis-acting viral sequences for replication and packaging in a helper line and the transcription unit for the exogenous gene. Other viral functions are expressed in trans in a specific packaging or helper cell line. Adenoviral vectors (e.g., for use in human gene therapy) are described in, e.g., Rosenfeld et al., 1992, <i>Cell </i>68: 143; PCT publications WO 94/12650; 94/12649; and 94/12629. In cases where an adenovirus is used as an expression vector, a sequence encoding hTRT may be ligated into an adenovirus transcription/translation complex consisting of the late promoter and tripartite leader sequence. Insertion in a nonessential E1 or E3 region of the viral genome will result in a viable virus capable of expressing in infected host cells (Logan and Shenk, 1984, <i>Proc. Natl. Acad. Sci., </i>81:3655). Replication-defective retroviral vectors harboring a therapeutic polynucleotide sequence as part of the retroviral genome are described in, e.g., Miller et al., 1990, <i>Mol. Cell. Biol. </i>10: 4239; Kolberg, 1992, <i>J. NIH Res. </i>4: 43; and Cornetta et al., 1991, <i>Hum. Gene Ther. </i>2: 215.</p>
    <p>In mammalian cell systems, promoters from mammalian genes or from mammalian viruses are often appropriate. Suitable promoters may be constitutive, cell type-specific, stage-specific, and/or modulatable or regulatable (e.g., by hormones such as glucocorticoids). Useful promoters include, but are not limited to, the metallothionein promoter, the constitutive adenovirus major late promoter, the dexamethasone-inducible MMTV promoter, the SV40 promoter, the MRP polIII promoter, the constitutive MPSV promoter, the tetracycline-inducible CMV promoter (such as the human immediate-early CMV promoter), the constitutive CMV promoter, and promoter-enhancer combinations known in the art.</p>
    <p>Other regulatory elements may also be required or desired for efficient expression of an hTRT polynucleotide and/or translation of a sequence encoding hTRT proteins. For translation, these elements typically include an ATG initiation codon and adjacent ribosome binding site or other sequences. For sequences encoding the hTRT protein, provided its initiation codon and upstream promoter sequences are inserted into an expression vector, no additional translational or other control signals may be needed. However, in cases where only coding sequence, or a portion thereof, is inserted, exogenous transcriptional and/or translational control signals (e.g., the promoter, ribosome-binding site, and ATG initiation codon) must often be provided. Furthermore, the initiation codon must typically be in the correct reading frame to ensure translation of the desired protein. Exogenous transcriptional elements and initiation codons can be of various origins, both natural and synthetic. In addition, the efficiency of expression may be enhanced by the inclusion of enhancers appropriate to the cell system in use (Scharf et al., 1994, <i>Results Probl. Cell Differ. </i>20:125; and Bittner et al. 1987, <i>Meth. Enzymol., </i>153:516). For example, the SV40 enhancer or CMV enhancer may be used to increase expression in mammalian host cells.</p>
    <p>Expression of hTRT gene products can also by effected (increased) by activation of an hTRT promoter or enhancer in a cell such as a human cell, e.g., a telomerase-negative cell line. Activation can be carried out in a variety of ways, including administration of an exogenous promoter activating agent, or inhibition of a cellular component that suppresses expression of the hTRT gene. It will be appreciated that, conversely, inhibition of promoter function, as described infra, will reduce hTRT gene expression.</p>
    <p>The invention provides inducible and repressible expression of hTRT polypeptides using such system as the Ecdysone-Inducible Expression System (Invitrogen), and the Tet-On and Tet-off tetracycline regulated systems from Clontech. The ecdysone-inducible expression system uses the steroid hormone ecdysone analog, muristerone A, to activate expression of a recombinant protein via a heterodimeric nuclear receptor (No et al., 1996, <i>Proc. Natl. Acad. Sci. USA </i>93:3346). In one embodiment of the invention, hTRT is cloned in the pIND vector (Clontech), which contains five modified ecdysone response elements (E/GREs) upstream of a minimal heat shock promoter and the multiple cloning site. The construct is then transfected in cell lines stably expressing the ecdysone receptor. After transfection, cells are treated with muristerone A to induce intracellular expression from pIND. In another embodiment of the invention, hTRT polypeptide is expressed using the Tet-on and Tet-off expression systems (Clontech) to provide regulated, high-level gene expression (Gossen et al., 1992, <i>Proc. Natl. Acad. Sci. USA </i>89:5547; Gossen et al., 1995, <i>Science </i>268:1766).</p>
    <p>The hTRT vectors of the invention may be introduced into a cell, tissue, organ, patient or animal by a variety of methods. The nucleic acid expression vectors (typically dsDNA) of the invention can be transferred into the chosen host cell by well-known methods such as calcium chloride transformation (for bacterial systems), electroporation, calcium phosphate treatment, liposome-mediated transformation, injection and microinjection, ballistic methods, virosomes, immunoliposomes, polycation:nucleic acid conjugates, naked DNA, artificial virions, fusion to the herpes virus structural protein VP22 (Elliot and O'Hare, <i>Cell </i>88:223), agent-enhanced uptake of DNA, and ex vivo transduction. Useful liposome-mediated DNA transfer methods are described in U.S. Pat. Nos. 5,049,386, 4,946,787; and 4,897,355; PCT publications WO 91/17424, WO 91/16024; Wang and Huang, 1987, <i>Biochem. Biophys. Res. Commun. </i>147: 980; Wang and Huang, 1989, Biochemistry 28: 9508; Litzinger and Huang, 1992, <i>Biochem. Biophys. Acta </i>1113:201; Gao and Huang, 1991, <i>Biochem. Biophys. Res. Commun. </i>179: 280. Immunoliposomes have been described as carriers of exogenous polynucleotides (Wang and Huang, 1987, <i>Proc. Natl. Acad. Sci. U.S.A. </i>84:7851; Trubetskoy et al., 1992, <i>Biochem. Biophys. Acta </i>1131:311) and may have improved cell type specificity as compared to liposomes by virtue of the inclusion of specific antibodies which presumably bind to surface antigens on specific cell types. Behr et al., 1989, <i>Proc. Natl. Acad. Sci. U.S.A. </i>86:6982 report using lipopolyamine as a reagent to mediate transfection itself, without the necessity of any additional phospholipid to form liposomes. Suitable delivery methods will be selected by practitioners in view of acceptable practices and regulatory requirements (e.g., for gene therapy or production of cell lines for expression of recombinant proteins). It will be appreciated that the delivery methods listed above may be used for transfer of nucleic acids into cells for purposes of gene therapy, transfer into tissue culture cells, and the like.</p>
    <p>For long-term, high-yield production of recombinant proteins, stable expression will often be desired. For example, cell lines which stably express hTRT can be prepared using expression vectors of the invention which contain viral origins of replication or endogenous expression elements and a selectable marker gene. Following the introduction of the vector, cells may be allowed to grow for 1-2 days in an enriched media before they are switched to selective media. The purpose of the selectable marker is to confer resistance to selection, and its presence allows growth of cells which successfully express the introduced sequences in selective media. Resistant, stably transfected cells can be proliferated using tissue culture techniques appropriate to the cell type. An amplification step, e.g., by administration of methyltrexate to cells transfected with a DHFR gene according to methods well known in the art, can be included.</p>
    <p>In addition, a host cell strain may be chosen for its ability to modulate the expression of the inserted sequences or to process the expressed protein in the desired fashion. Such modifications of the polypeptide include, but are not limited to, acetylation, carboxylation, phosphorylation, lipidation and acylation. Post-translational processing may also be important for correct insertion, folding and/or function. Different host cells have cellular machinery and characteristic mechanisms specific for each cell for such post-translational activities and so a particular cell may be chosen to ensure the correct modification and processing of the introduced, foreign protein.</p>
    <p>As noted supra, when expressing an hTRT protein (including variants) in cells or organisms it is sometimes desirable to use an hTRT protein-encoding polynucleotide that employs a codon distribution other than that found in a naturally occurring hTRT gene. hTRT protein-encoding polynucleotides with alternative codons throughout, or at specific sites, in the coding sequence are used to optimize (e.g., increase) expression of the hTRT protein in cells, especially non-human cells (e.g., bacterial, plant, fungal, and non-human animal cells) which have different preferential codon usage than human cells. Codon changes may also be used to facilitate manipulation of the hTRT polynucleotide (e.g., by engineering useful tags or restriction sites into the coding sequence), and for other reasons. When the goal is to optimize expression (e.g., by increasing translational efficiency), tables of preferred codon usage, which are publicly available and are well known to those of skill, are used to design a suitable polynucleotide by “reverse translation” of the desired (e.g., hTRT) amino acid sequence. Alternatively, preferred codon usage can be determined for a particular organism (e.g., <i>Pichia pastoris</i>) or class of genes (e.g., highly expressed genes of a particular organism) by comparison of published gene sequences for the target organism or gene class.</p>
    <p>Illustrative hTRT-encoding polynucleotide sequences are provided in Table 9 (A-E), infra. All of the sequences in Table 9 are in the 5′→3′. Table 9A shows an hTRT protein encoding polynucleotide that uses a codon distribution preferentially employed in the bacterium <i>E. coli</i>. Table 9B shows a second polynucleotide sequence particularly useful for expression in <i>E. coli </i>(and other enteric bacteria) using codons preferentially used in highly expressed genes in enteric bacteria. Table 4C shows an hTRT protein encoding polynucleotide that uses a codon distribution preferentially employed in yeast (i.e., <i>S. cerevisiae</i>). Table 4D shows an hTRT protein encoding polynucleotide that uses a codon distribution preferentially used in highly expressed genes in yeast. Table 4E shows an hTRT protein encoding polynucleotide that uses a “generic” codon distribution that should be efficiently expressed in both bacteria (e.g., <i>E. coli</i>) and yeast (e.g., <i>S. pombe, S. cerevisiae, P. pastoris</i>) and some insect (e.g., <i>S. frugiperda</i>) cells. Such “generic” polynucleotide sequences (optimized for more than one organism) are useful for, for example, comparative studies, screening in different organisms of hTRT binding or modulatory agents, creation of shuttle vectors, and other uses. In this “generic” sequence, the codon TCT (serine) may not be optimal for expression in Drosophila cells. Therefore, in an alternative embodiment the sequence in Table 4E is modified to replace TCT with TCC for efficient expression in Drosophila as well as bacteria and yeast.</p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" pgwide="1" tabstyle="monospace" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="308pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 9</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">hTRT-ENCODING POLYNUCLEOTIDE SEQUENCES EMPLOYING</td>
              </tr> <tr class="description-tr"> <td class="description-td">ALTERNATIVE CODON DISTRIBUTIONS</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">TABLE 9A</td>
              </tr> <tr class="description-tr"> <td class="description-td"> <i>E. coli </i>(all genes) (SEQ ID NO:638) </td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">ATG CCG CGC GCG CCG CGC TGC CGC GCG GTG CGC AGC CTG CTG CGC AGC CAT TAT</td>
              </tr> <tr class="description-tr"> <td class="description-td">CGC GAA GTG CTG CCG CTG GCG ACC TTT GTG CGC CGC CTG GGC CCG CAG GGC TGG</td>
              </tr> <tr class="description-tr"> <td class="description-td">CGd CTG GTG CAG CGC GGC GAT CCG GCG GCG TTT CGC GCG CTG GTG GCG CAG TGC</td>
              </tr> <tr class="description-tr"> <td class="description-td">CTG GTG TGC GTG CCG TGG GAT GCG CGC CCG CCG CCG GCG GCG CCG AGC TTT CGC</td>
              </tr> <tr class="description-tr"> <td class="description-td">CAG GTG AGC TGC CTG AAA GAA CTG GTG GCG CGC GTG CTG CAG CGC CTG TGC GAA</td>
              </tr> <tr class="description-tr"> <td class="description-td">CGC GGC GCG AAA AAC GTG CTG GCG TTT GGC TTT GCG CTG CTG GAT GGC GCG CGC</td>
              </tr> <tr class="description-tr"> <td class="description-td">GGC GGC CCG CCG GAA GCG TTT ACC ACC AGC GTG CGC AGC TAT CTG CCG AAC ACC</td>
              </tr> <tr class="description-tr"> <td class="description-td">GTG ACC GAT GCG CTG CGC GGC AGC GGC GCG TGG GGC CTG CTG CTG CGC CGC GTG</td>
              </tr> <tr class="description-tr"> <td class="description-td">GGC GAT GAT GTG CTG GTG CAT CTG CTG GCG CGC TGC GCG CTG TTT GTG CTG GTG</td>
              </tr> <tr class="description-tr"> <td class="description-td">GCG CCG AGC TGC GCG TAT CAG GTG TGC GGC CCG CCG CTG TAT CAG CTG GGC GCG</td>
              </tr> <tr class="description-tr"> <td class="description-td">GCG ACC CAG GCG CGC CCG CCG CCG CAT GCG AGC GGC CCG CGC CGC CGC CTG GGC</td>
              </tr> <tr class="description-tr"> <td class="description-td">TGC GAA CGC GCG TGG AAC CAT AGC GTG CGC GAA GCG GGC GTG CCG CTG GGC CTG</td>
              </tr> <tr class="description-tr"> <td class="description-td">CCG GCG CCG GGC GCG CGC CGC CGC GGC GGC AGC GCG AGC CGC AGC CTG CCG CTG</td>
              </tr> <tr class="description-tr"> <td class="description-td">CCG AAA CGC CCG CGC CGC GGC GCG GCG CCG GAA CCG GAA CGC ACC CCG GTG GGC</td>
              </tr> <tr class="description-tr"> <td class="description-td">CAG GGC AGC TGG GCG CAT CCG GGC CGC ACC CGC GGC CCG AGC GAT CGC GGC TTT</td>
              </tr> <tr class="description-tr"> <td class="description-td">TGC GTG GTG AGC CCG GCG CGC CCG GCG GAA GAA GCG ACC AGC CTG GAA GGC GCG</td>
              </tr> <tr class="description-tr"> <td class="description-td">CTG AGC GGC ACC CGC CAT AGC CAT CCG AGC GTG GGC CGC CAG CAT CAT GCG GGC</td>
              </tr> <tr class="description-tr"> <td class="description-td">CCG CCG AGC ACC AGC CGC CCG CCG CGC CCG TGG GAT ACC CCG TGC CCG CCG GTG</td>
              </tr> <tr class="description-tr"> <td class="description-td">TAT GCG GAA ACC AAA CAT TTT CTG TAT AGC AGC GGC GAT AAA GAA CAG CTG CGC</td>
              </tr> <tr class="description-tr"> <td class="description-td">CCG AGC TTT CTG CTG AGC AGC CTG CGC CCG AGC CTG ACC GGC GCG CGC CGC CTG</td>
              </tr> <tr class="description-tr"> <td class="description-td">GTG GAA ACC ATT TTT CTG GGC AGC CGC CCG TGG ATG CCG GGC ACC CCG CGC CGC</td>
              </tr> <tr class="description-tr"> <td class="description-td">CTG CCG CGC CTG CCG CAG CGC TAT TGG CAG ATG CGC CCG CTG TTT CTG GAA CTG</td>
              </tr> <tr class="description-tr"> <td class="description-td">CTG GGC AAC CAT GCG CAG TGC CCG TAT GGC GTG CTG CTG AAA ACC CAT TGC CCG</td>
              </tr> <tr class="description-tr"> <td class="description-td">CTG CGC GCG GCG GTG ACC CCG GCG GCG GGC GTG TGC GCG CGC GAA AAA CCG CAG</td>
              </tr> <tr class="description-tr"> <td class="description-td">GGC AGC GTG GCG GCG CCG GAA GAA GAA GAT ACC GAT CCG CGC CGC CTG GTG CAG</td>
              </tr> <tr class="description-tr"> <td class="description-td">CTG CTG CGC CAG CAT AGC AGC CCG TGG CAG GTG TAT GGC TTT GTG CGC GCG TGC</td>
              </tr> <tr class="description-tr"> <td class="description-td">CTG CGC CGC CTG GTG CCG CCG GGC CTG TGG GGC AGC CGC CAT AAC GAA CGC CGC</td>
              </tr> <tr class="description-tr"> <td class="description-td">TTT CTG CGC AAC ACC AAA AAA TTT ATT AGC CTG GGC AAA CAT GCG AAA CTG AGC</td>
              </tr> <tr class="description-tr"> <td class="description-td">CTG CAG GAA CTG ACC TGG AAA ATG AGC GTG CGC GAT TGC GCG TGG CTG CGC CGC</td>
              </tr> <tr class="description-tr"> <td class="description-td">AGC CCG GGC GTG GGC TGC GTG CCG GCG GCG GAA CAT CGC CTG CGC GAA GAA ATT</td>
              </tr> <tr class="description-tr"> <td class="description-td">CTG GCG AAA TTT CTG CAT TGG CTG ATG AGC GTG TAT GTG GTG GAA CTG CTG CGC</td>
              </tr> <tr class="description-tr"> <td class="description-td">AGC TTT TTT TAT GTG ACC GAA ACC ACC TTT CAG AAA AAC CGC CTG TTT TTT TAT</td>
              </tr> <tr class="description-tr"> <td class="description-td">CGC AAA AGC GTG TGG AGC AAA CTG CAG AGC ATT GGC ATT CGC CAG CAT CTG AAA</td>
              </tr> <tr class="description-tr"> <td class="description-td">CGC GTG CAG CTG CGC GAA CTG AGC GAA GCG GAA GTG CGC CAG CAT CGC GAA GCG</td>
              </tr> <tr class="description-tr"> <td class="description-td">CGC CCG GCG CTG CTG ACC AGC CGC CTG CGC TTT ATT CCG AAA CCG GAT GGC CTG</td>
              </tr> <tr class="description-tr"> <td class="description-td">CGC CCG ATT GTG AAC ATG GAT TAT GTG GTG GGC GCG CGC ACC TTT CGC CGC GAA</td>
              </tr> <tr class="description-tr"> <td class="description-td">AAA CGC GCG GAA CGC CTG ACC AGC CGC GTG AAA GCG CTG TTT AGC GTG CTG AAC</td>
              </tr> <tr class="description-tr"> <td class="description-td">TAT GAA CGC GCG CGC CGC CCG GGC CTG CTG GGC GCG AGC GTG CTG GGC CTG GAT</td>
              </tr> <tr class="description-tr"> <td class="description-td">GAT ATT CAT CGC GCG TGG CGC ACC TTT GTG CTG CGC GTG CGC GCG CAG GAT CCG</td>
              </tr> <tr class="description-tr"> <td class="description-td">CCG CCG GAA CTG TAT TTT GTG AAA GTG GAT GTG ACC GGC GCG TAT GAT ACC ATT</td>
              </tr> <tr class="description-tr"> <td class="description-td">CCG CAG GAT CGC CTG ACC GAA GTG ATT GCG AGC ATT ATT AAA CCG CAG AAC ACC</td>
              </tr> <tr class="description-tr"> <td class="description-td">TAT TGC GTG CGC CGC TAT GCG GTG GTG CAG AAA GCG GCG CAT GGC CAT GTG CGC</td>
              </tr> <tr class="description-tr"> <td class="description-td">AAA GCG TTT AAA AGC CAT GTG AGC ACC CTG ACC GAT CTG CAG CCG TAT ATG CGC</td>
              </tr> <tr class="description-tr"> <td class="description-td">CAG TTT GTG GCG CAT CTG CAG GAA ACC AGC CCG CTG CGC GAT GCG GTG GTG ATT</td>
              </tr> <tr class="description-tr"> <td class="description-td">GAA CAG AGC AGC AGC CTG AAC GAA GCG AGC AGC GGC CTG TTT GAT GTG TTT CTG</td>
              </tr> <tr class="description-tr"> <td class="description-td">CGC TTT ATG TGC CAT CAT GCG GTG CGC ATT CGC GGC AAA AGC TAT GTG CAG TGC</td>
              </tr> <tr class="description-tr"> <td class="description-td">CAG GGC ATT CCG CAG GGC AGC ATT CTG AGC ACC CTG CTG TGC AGC CTG TGC TAT</td>
              </tr> <tr class="description-tr"> <td class="description-td">GGC GAT ATG GAA AAC AAA CTG TTT GCG GGC ATT CGC CGC GAT GGC CTG CTG CTG</td>
              </tr> <tr class="description-tr"> <td class="description-td">CGC CTG GTG GAT GAT TTT CTG CTG GTG ACC CCG CAT CTG ACC CAT GCG AAA ACC</td>
              </tr> <tr class="description-tr"> <td class="description-td">TTT CTG CGC ACC CTG GTG CGC GGC GTG CCG GAA TAT GGC TGC GTG GTG AAC CTG</td>
              </tr> <tr class="description-tr"> <td class="description-td">CGC AAA ACC GTG GTG AAC TTT CCG GTG GAA GAT GAA GCG CTG GGC GGC ACC GCG</td>
              </tr> <tr class="description-tr"> <td class="description-td">TTT GTG CAG ATG CCG GCG CAT GGC CTG TTT CCG TGG TGC GGC CTG CTG CTG GAT</td>
              </tr> <tr class="description-tr"> <td class="description-td">ACC CGC ACC CTG GAA GTG CAG AGC GAT TAT AGC AGC TAT GCG CGC ACC AGC ATT</td>
              </tr> <tr class="description-tr"> <td class="description-td">CGC GCG AGC CTG ACC TTT AAC CGC GGC TTT AAA GCG GGC CGC AAC ATG CGC CGC</td>
              </tr> <tr class="description-tr"> <td class="description-td">AAA CTG TTT GGC GTG CTG CGC CTG AAA TGC CAT AGC CTG TTT CTG GAT CTG CAG</td>
              </tr> <tr class="description-tr"> <td class="description-td">GTG AAC AGC CTG CAG ACC GTG TGC ACC AAC ATT TAT AAA ATT CTG CTG CTG CAG</td>
              </tr> <tr class="description-tr"> <td class="description-td">GCG TAT CGC TTT CAT GCG TGC GTG CTG CAG CTG CCG TTT CAT CAG CAG GTG TGG</td>
              </tr> <tr class="description-tr"> <td class="description-td">AAA AAC CCG ACC TTT TTT CTG CGC GTG ATT AGC GAT ACC GCG AGC CTG TGC TAT</td>
              </tr> <tr class="description-tr"> <td class="description-td">AGC ATT CTG AAA GCG AAA AAC GCG GGC ATG AGC CTG GGC GCG AAA GGC GCG GCG</td>
              </tr> <tr class="description-tr"> <td class="description-td">GGC CCG CTG CCG AGC GAA GCG GTG CAG TGG CTG TGC CAT CAG GCG TTT CTG CTG</td>
              </tr> <tr class="description-tr"> <td class="description-td">AAA CTG ACC CGC CAT CGC GTG ACC TAT GTG CCG CTG CTG GGC AGC CTG CGC ACC</td>
              </tr> <tr class="description-tr"> <td class="description-td">GCG CAG ACC CAG CTG AGC CGC AAA CTG CCG GGC ACC ACC CTG ACC GCG CTG GAA</td>
              </tr> <tr class="description-tr"> <td class="description-td">GCG GCG GCG AAC CCG GCG CTG CCG AGC GAT TTT AAA ACC ATT CTG GAT </td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">TABLE 9B</td>
              </tr> <tr class="description-tr"> <td class="description-td">Enteric Bacteria (High Expressing Genes) (SEQ ID NO:639) </td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">   1</td>
                <td class="description-td">ATGCCGCGTG CTCCGCGTTG CCGTGCTGTT CGTTCCCTGC TGCGTTCCCA</td>
              </tr> <tr class="description-tr"> <td class="description-td">  51</td>
                <td class="description-td">CTACCGTGAA GTTCTGCCGC TGGCTACCTT CGTTCGTCGT CTGGGTCCGC</td>
              </tr> <tr class="description-tr"> <td class="description-td"> 101</td>
                <td class="description-td">AGGGTTGGCG TCTGGTTCAG CGTGGTGACC CGGCTGCTTT CCGTGCTCTG</td>
              </tr> <tr class="description-tr"> <td class="description-td"> 151</td>
                <td class="description-td">GTTGCTCAGT GCCTGGTTTG CGTTCCGTGG GACGCTCGTC CGCCGCCGGC</td>
              </tr> <tr class="description-tr"> <td class="description-td"> 201</td>
                <td class="description-td">TGCTCCGTCC TTCCGTCAGG TTTCCTGCCT GAAAGAACTG GTTGCTCGTG</td>
              </tr> <tr class="description-tr"> <td class="description-td"> 251</td>
                <td class="description-td">TTCTGCAGCG TCTGTGCGAA CGTGGTGCTA AAAACGTTCT GGCTTTCGGT</td>
              </tr> <tr class="description-tr"> <td class="description-td"> 301</td>
                <td class="description-td">TTCGCTCTGC TGGACGGTGC TCGTGGTGGT CCGCCGGAAG CTTTCACCAC</td>
              </tr> <tr class="description-tr"> <td class="description-td"> 351</td>
                <td class="description-td">CTCCGTTCGT TCCTACCTGC CGAACACCGT TACCGACGCT CTGCGTGGTT</td>
              </tr> <tr class="description-tr"> <td class="description-td"> 401</td>
                <td class="description-td">CCGGTGCTTG GGGTCTGCTG CTGCGTCGTG TTGGTGACGA CGTTCTGGTT</td>
              </tr> <tr class="description-tr"> <td class="description-td"> 451</td>
                <td class="description-td">CACCTGCTGG CTCGTTGCGC TCTGTTCGTT CTGGTTGCTC CGTCCTGCGC</td>
              </tr> <tr class="description-tr"> <td class="description-td"> 501</td>
                <td class="description-td">TTACCAGGTT TGCGGTCCGC CGCTGTACCA GCTGGGTGCT GCTACCCAGG</td>
              </tr> <tr class="description-tr"> <td class="description-td"> 551</td>
                <td class="description-td">CTCGTCCGCC GCCGCACGCT TCCGGTCCGC GTCGTCGTCT GGGTTGCGAA</td>
              </tr> <tr class="description-tr"> <td class="description-td"> 601</td>
                <td class="description-td">CGTGCTTGGA ACCACTCCGT TCGTGAAGCT GGTGTTCCGC TGGGTCTGCC</td>
              </tr> <tr class="description-tr"> <td class="description-td"> 651</td>
                <td class="description-td">GGCTCCGGGT GCTCGTCGTC GTGGTGGTTC CGCTTCCCGT TCCCTGCCGC</td>
              </tr> <tr class="description-tr"> <td class="description-td"> 701</td>
                <td class="description-td">TGCCGAAACG TCCGCGTCGT GGTGCTGCTC CGGAACCGGA ACGTACCCCG</td>
              </tr> <tr class="description-tr"> <td class="description-td"> 751</td>
                <td class="description-td">GTTGGTCAGG GTTCCTGGGC TCACCCGGGT CGTACCCGTG GTCCGTCCGA</td>
              </tr> <tr class="description-tr"> <td class="description-td"> 801</td>
                <td class="description-td">CCGTGGTTTC TGCGTTGTTT CCCCGGCTCG TCCGGCTGAA GAAGCTACCT</td>
              </tr> <tr class="description-tr"> <td class="description-td"> 851</td>
                <td class="description-td">CCCTGGAAGG TGCTCTGTCC GGTACCCGTC ACTCCCACCC GTCCGTTGGT</td>
              </tr> <tr class="description-tr"> <td class="description-td"> 901</td>
                <td class="description-td">CGTCAGCACC ACGCTGGTCC GCCGTCCACC TCCCGTCCGC CGCGTCCGTG</td>
              </tr> <tr class="description-tr"> <td class="description-td"> 951</td>
                <td class="description-td">GGACACCCCG TGCCCGCCGG TTTACGCTGA AACCAAACAC TTCCTGTACT</td>
              </tr> <tr class="description-tr"> <td class="description-td">1001</td>
                <td class="description-td">CCTCCGGTGA CAAAGAACAG CTGCGTCCGT CCTTCCTGCT GTCCTCCCTG</td>
              </tr> <tr class="description-tr"> <td class="description-td">1051</td>
                <td class="description-td">CGTCCGTCCC TGACCGGTGC TCGTCGTCTG GTTGAAACCA TCTTCCTGGG</td>
              </tr> <tr class="description-tr"> <td class="description-td">1101</td>
                <td class="description-td">TTCCCGTCCG TGGATGCCGG GTACCCCGCG TCGTCTGCCG CGTCTGCCGC</td>
              </tr> <tr class="description-tr"> <td class="description-td">1151</td>
                <td class="description-td">AGCGTTACTG GCAGATGCGT CCGCTGTTCC TGGAACTGCT GGGTAACCAC</td>
              </tr> <tr class="description-tr"> <td class="description-td">1201</td>
                <td class="description-td">GCTCAGTGCC CGTACGGTGT TCTGCTGAAA ACCCACTGCC CGCTGCGTGC</td>
              </tr> <tr class="description-tr"> <td class="description-td">1251</td>
                <td class="description-td">TGCTGTTACC CCGGCTGCTG GTGTTTGCGC TCGTGAAAAA CCGCAGGGTT</td>
              </tr> <tr class="description-tr"> <td class="description-td">1301</td>
                <td class="description-td">CCGTTGCTGC TCCGGAAGAA GAAGACACCG ACCCGCGTCG TCTGGTTCAG</td>
              </tr> <tr class="description-tr"> <td class="description-td">1351</td>
                <td class="description-td">CTGCTGCGTC AGCACTCCTC CCCGTGGCAG GTTTACGGTT TCGTTCGTGC</td>
              </tr> <tr class="description-tr"> <td class="description-td">1401</td>
                <td class="description-td">TTGCCTGCGT CGTCTGGTTC CGCCGGGTCT GTGGGGTTCC CGTCACAACG</td>
              </tr> <tr class="description-tr"> <td class="description-td">1451</td>
                <td class="description-td">AACGTCGTTT CCTGCGTAAC ACCAAAAAAT TCATCTCCCT GGGTAAACAC</td>
              </tr> <tr class="description-tr"> <td class="description-td">1501</td>
                <td class="description-td">GCTAAACTGT CCCTGCAGGA ACTGACCTGG AAAATGTCCG TTCGTGACTG</td>
              </tr> <tr class="description-tr"> <td class="description-td">1551</td>
                <td class="description-td">CGCTTGGCTG CGTCGTTCCC CGGGTGTTGG TTGCGTTCCG GCTGCTGAAC</td>
              </tr> <tr class="description-tr"> <td class="description-td">1601</td>
                <td class="description-td">ACCGTCTGCG TGAAGAAATC CTGGCTAAAT TCCTGCACTG GCTGATGTCC</td>
              </tr> <tr class="description-tr"> <td class="description-td">1651</td>
                <td class="description-td">GTTTACGTTG TTGAACTGCT GCGTTCCTTC TTCTACGTTA CCGAAACCAC</td>
              </tr> <tr class="description-tr"> <td class="description-td">1701</td>
                <td class="description-td">CTTCCAGAAA AACCGTCTGT TCTTCTACCG TAAATCCGTT TGGTCCAAAC</td>
              </tr> <tr class="description-tr"> <td class="description-td">1751</td>
                <td class="description-td">TGCAGTCCAT CGGTATCCGT CAGCACCTGA AACGTGTTCA GCTGCGTGAA</td>
              </tr> <tr class="description-tr"> <td class="description-td">1801</td>
                <td class="description-td">CTGTCCGAAG CTGAAGTTCG TCAGCACCGT GAAGCTCGTC CGGCTCTGCT</td>
              </tr> <tr class="description-tr"> <td class="description-td">1851</td>
                <td class="description-td">GACCTCCCGT CTGCGTTTCA TCCCGAAACC GGACGGTCTG CGTCCGATCG</td>
              </tr> <tr class="description-tr"> <td class="description-td">1901</td>
                <td class="description-td">TTAACATGGA CTACGTTGTT GGTGCTCGTA CCTTCCGTCG TGAAAAACGT</td>
              </tr> <tr class="description-tr"> <td class="description-td">1951</td>
                <td class="description-td">GCTGAACGTC TGACCTCCCG TGTTAAAGCT CTGTTCTCCG TTCTGAACTA</td>
              </tr> <tr class="description-tr"> <td class="description-td">2001</td>
                <td class="description-td">CGAACGTGCT CGTCGTCCGG GTCTGCTGGG TGCTTCCGTT CTGGGTCTGG</td>
              </tr> <tr class="description-tr"> <td class="description-td">2051</td>
                <td class="description-td">ACGACATCCA CCGTGCTTGG CGTACCTTCG TTCTGCGTGT TCGTGCTCAG</td>
              </tr> <tr class="description-tr"> <td class="description-td">2101</td>
                <td class="description-td">GACCCGCCGC CGGAACTGTA CTTCGTTAAA GTTGACGTTA CCGGTGCTTA</td>
              </tr> <tr class="description-tr"> <td class="description-td">2151</td>
                <td class="description-td">CGACACCATC CCGCAGGACC GTCTGACCGA AGTTATCGCT TCCATCATCA</td>
              </tr> <tr class="description-tr"> <td class="description-td">2201</td>
                <td class="description-td">AACCGCAGAA CACCTACTGC GTTCGTCGTT ACGCTGTTGT TCAGAAAGCT</td>
              </tr> <tr class="description-tr"> <td class="description-td">2251</td>
                <td class="description-td">GCTCACGGTC ACGTTCGTAA AGCTTTCAAA TCCCACGTTT CCACCCTGAC</td>
              </tr> <tr class="description-tr"> <td class="description-td">2301</td>
                <td class="description-td">CGACCTGCAG CCGTACATGC GTCAGTTCGT TGCTCACCTG CAGGAAACCT</td>
              </tr> <tr class="description-tr"> <td class="description-td">2351</td>
                <td class="description-td">CCCCGCTGCG TGACGCTGTT GTTATCGAAC AGTCCTCCTC CCTGAACGAA</td>
              </tr> <tr class="description-tr"> <td class="description-td">2401</td>
                <td class="description-td">GCTTCCTCCG GTCTGTTCGA CGTTTTCCTG CGTTTCATGT GCCACCACGC</td>
              </tr> <tr class="description-tr"> <td class="description-td">2451</td>
                <td class="description-td">TGTTCGTATC CGTGGTAAAT CCTACGTTCA GTGCCAGGGT ATCCCGCAGG</td>
              </tr> <tr class="description-tr"> <td class="description-td">2501</td>
                <td class="description-td">GTTCCATCCT GTCCACCCTG CTGTGCTCCC TGTGCTACGG TGACATGGAA</td>
              </tr> <tr class="description-tr"> <td class="description-td">2551</td>
                <td class="description-td">AACAAACTGT TCGCTGGTAT CCGTCGTGAC GGTCTGCTGC TGCGTCTGGT</td>
              </tr> <tr class="description-tr"> <td class="description-td">2601</td>
                <td class="description-td">TGACGACTTC CTGCTGGTTA CCCCGCACCT GACCCACGCT AAAACCTTCC</td>
              </tr> <tr class="description-tr"> <td class="description-td">2651</td>
                <td class="description-td">TGCGTACCCT GGTTCGTGGT GTTCCGGAAT ACGGTTGCGT TGTTAACCTG</td>
              </tr> <tr class="description-tr"> <td class="description-td">2701</td>
                <td class="description-td">CGTAAAACCG TTGTTAACTT CCCGGTTGAA GACGAAGCTC TGGGTGGTAC</td>
              </tr> <tr class="description-tr"> <td class="description-td">2751</td>
                <td class="description-td">CGCTTTCGTT CAGATGCCGG CTCACGGTCT GTTCCCGTGG TGCGGTCTGC</td>
              </tr> <tr class="description-tr"> <td class="description-td">2801</td>
                <td class="description-td">TGCTGGACAC CCGTACCCTG GAAGTTCAGT CCGACTACTC CTCCTACGCT</td>
              </tr> <tr class="description-tr"> <td class="description-td">2851</td>
                <td class="description-td">CGTACCTCCA TCCGTGCTTC CCTGACCTTC AACCGTGGTT TCAAAGCTGG</td>
              </tr> <tr class="description-tr"> <td class="description-td">2901</td>
                <td class="description-td">TCGTAACATG CGTCGTAAAC TGTTCGGTGT TCTGCGTCTG AAATGCCACT</td>
              </tr> <tr class="description-tr"> <td class="description-td">2951</td>
                <td class="description-td">CCCTGTTCCT GGACCTGCAG GTTAACTCCC TGCAGACCGT TTGCACCAAC</td>
              </tr> <tr class="description-tr"> <td class="description-td">3001</td>
                <td class="description-td">ATCTACAAAA TCCTGCTGCT GCAGGCTTAC CGTTTCCACG CTTGCGTTCT</td>
              </tr> <tr class="description-tr"> <td class="description-td">3051</td>
                <td class="description-td">GCAGCTGCCG TTCCACCAGC AGGTTTGGAA AAACCCGACC TTCTTCCTGC</td>
              </tr> <tr class="description-tr"> <td class="description-td">3101</td>
                <td class="description-td">GTGTTATCTC CGACACCGCT TCCCTGTGCT ACTCCATCCT GAAAGCTAAA</td>
              </tr> <tr class="description-tr"> <td class="description-td">3151</td>
                <td class="description-td">AACGCTGGTA TGTCCCTGGG TGCTAAAGGT GCTGCTGGTC CGCTGCCGTC</td>
              </tr> <tr class="description-tr"> <td class="description-td">3201</td>
                <td class="description-td">CGAAGCTGTT CAGTGGCTGT GCCACCAGGC TTTCCTGCTG AAACTGACCC</td>
              </tr> <tr class="description-tr"> <td class="description-td">3251</td>
                <td class="description-td">GTCACCGTGT TACCTACGTT CCGCTGCTGG GTTCCCTGCG TACCGCTCAG</td>
              </tr> <tr class="description-tr"> <td class="description-td">3301</td>
                <td class="description-td">ACCCAGCTGT CCCGTAAACT GCCGGGTACC ACCCTGACCG CTCTGGAAGC</td>
              </tr> <tr class="description-tr"> <td class="description-td">3351</td>
                <td class="description-td">TGCTGCTAAC CCGGCTCTGC CGTCCGACTT CAAAACCATC CTGGAC     </td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">TABLE 9C</td>
              </tr> <tr class="description-tr"> <td class="description-td">Yeast (All Genes) (SEQ ID NO:640) </td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">ATG CCA AGA GCT CCA AGA TGT AGA GCT GTT AGA TCT TTG TTG AGA TCT CAT TAT</td>
              </tr> <tr class="description-tr"> <td class="description-td">AGA GAA GTT TTG CCA TTG GCT ACT TTT GTT AGA AGA TTG GGT CCA CAA GGT TGG</td>
              </tr> <tr class="description-tr"> <td class="description-td">AGA TTG GTT CAA AGA GGT GAT CCA GCT GCT TTT AGA GCT TTG GTT GCT CAA TGT</td>
              </tr> <tr class="description-tr"> <td class="description-td">TTG GTT TGT GTT CCA TGG GAT GCT AGA CCA CCA CCA GCT GCT CCA TCT TTT AGA</td>
              </tr> <tr class="description-tr"> <td class="description-td">CAA GTT TCT TGT TTG AAA GAA TTG GTT GCT AGA GTT TTG CAA AGA TTG TGT GAA</td>
              </tr> <tr class="description-tr"> <td class="description-td">AGA GGT GCT AAA AAT GTT TTG GCT TTT GGT TTT GCT TTG TTG GAT GGT GCT AGA</td>
              </tr> <tr class="description-tr"> <td class="description-td">GGT GGT CCA CCA GAA GCT TTT ACT ACT TCT GTT AGA TCT TAT TTG CCA AAT ACT</td>
              </tr> <tr class="description-tr"> <td class="description-td">GTT ACT GAT GCT TTG AGA GGT TCT GGT GCT TGG GGT TTG TTG TTG AGA AGA GTT</td>
              </tr> <tr class="description-tr"> <td class="description-td">GGT GAT GAT GTT TTG GTT CAT TTG TTG GCT AGA TGT GCT TTG TTT GTT TTG GTT</td>
              </tr> <tr class="description-tr"> <td class="description-td">GCT CCA TCT TGT GCT TAT CAA GTT TGT GGT CCA CCA TTG TAT CAA TTG GGT GCT</td>
              </tr> <tr class="description-tr"> <td class="description-td">GCT ACT CAA GCT AGA CCA CCA CCA CAT GCT TCT GGT CCA AGA AGA AGA TTG GGT</td>
              </tr> <tr class="description-tr"> <td class="description-td">TGT GAA AGA GCT TGG AAT CAT TCT GTT AGA GAA GCT GGT GTT CCA TTG GGT TTG</td>
              </tr> <tr class="description-tr"> <td class="description-td">CCA GCT CCA GGT GCT AGA AGA AGA GGT GGT TCT GCT TCT AGA TCT TTG CCA TTG</td>
              </tr> <tr class="description-tr"> <td class="description-td">CCA AAA AGA CCA AGA AGA GGT GCT GCT CCA GAA CCA GAA AGA ACT CCA GTT GGT</td>
              </tr> <tr class="description-tr"> <td class="description-td">CAA GGT TCT TGG GCT CAT CCA GGT AGA ACT AGA GGT CCA TCT GAT AGA GGT TTT</td>
              </tr> <tr class="description-tr"> <td class="description-td">TGT GTT GTT TCT CCA GCT AGA CCA GCT GAA GAA GCT ACT TCT TTG GAA GGT GCT</td>
              </tr> <tr class="description-tr"> <td class="description-td">TTG TCT GGT ACT AGA CAT TCT CAT CCA TCT GTT GGT AGA CAA CAT CAT GCT GGT</td>
              </tr> <tr class="description-tr"> <td class="description-td">CCA CCA TCT ACT TCT AGA CCA CCA AGA CCA TGG GAT ACT CCA TGT CCA CCA GTT</td>
              </tr> <tr class="description-tr"> <td class="description-td">TAT GCT GAA ACT AAA CAT TTT TTG TAT TCT TCT GGT GAT AAA GAA CAA TTG AGA</td>
              </tr> <tr class="description-tr"> <td class="description-td">CCA TCT TTT TTG TTG TCT TCT TTG AGA CCA TCT TTG ACT GGT GCT AGA AGA TTG</td>
              </tr> <tr class="description-tr"> <td class="description-td">GTT GAA ACT ATT TTT TTG GGT TCT AGA CCA TGG ATG CCA GGT ACT CCA AGA AGA</td>
              </tr> <tr class="description-tr"> <td class="description-td">TTG CCA AGA TTG CCA CAA AGA TAT TGG CAA ATG AGA CCA TTG TTT TTG GAA TTG</td>
              </tr> <tr class="description-tr"> <td class="description-td">TTG GGT AAT CAT GCT CAA TGT CCA TAT GGT GTT TTG TTG AAA ACT CAT TGT CCA</td>
              </tr> <tr class="description-tr"> <td class="description-td">TTG AGA GCT GCT GTT ACT CCA GCT GCT GGT GTT TGT GCT AGA GAA AAA CCA CAA</td>
              </tr> <tr class="description-tr"> <td class="description-td">GGT TCT GTT GCT GCT CCA GAA GAA GAA GAT ACT GAT CCA AGA AGA TTG GTT CAA</td>
              </tr> <tr class="description-tr"> <td class="description-td">TTG TTG AGA CAA CAT TCT TCT CCA TGG CAA GTT TAT GGT TTT GTT AGA GCT TGT</td>
              </tr> <tr class="description-tr"> <td class="description-td">TTG AGA AGA TTG GTT CCA CCA GGT TTG TGG GGT TCT AGA CAT AAT GAA AGA AGA</td>
              </tr> <tr class="description-tr"> <td class="description-td">TTT TTG AGA AAT ACT AAA AAA TTT ATT TCT TTG GGT AAA CAT GCT AAA TTG TCT</td>
              </tr> <tr class="description-tr"> <td class="description-td">TTG CAA GAA TTG ACT TGG AAA ATG TCT GTT AGA GAT TGT GCT TGG TTG AGA AGA</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCT CCA GGT GTT GGT TGT GTT CCA GCT GCT GAA CAT AGA TTG AGA GAA GAA ATT</td>
              </tr> <tr class="description-tr"> <td class="description-td">TTG GCT AAA TTT TTG CAT TGG TTG ATG TCT GTT TAT GTT GTT GAA TTG TTG AGA</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCT TTT TTT TAT GTT ACT GAA ACT ACT TTT CAA AAA AAT AGA TTG TTT TTT TAT</td>
              </tr> <tr class="description-tr"> <td class="description-td">AGA AAA TCT GTT TGG TCT AAA TTG CAA TCT ATT GGT ATT AGA CAA CAT TTG AAA</td>
              </tr> <tr class="description-tr"> <td class="description-td">AGA GTT CAA TTG AGA GAA TTG TCT GAA GCT GAA GTT AGA CAA CAT AGA GAA GCT</td>
              </tr> <tr class="description-tr"> <td class="description-td">AGA CCA GCT TTG TTG ACT TCT AGA TTG AGA TTT ATT CCA AAA CCA GAT GGT TTG</td>
              </tr> <tr class="description-tr"> <td class="description-td">AGA CCA ATT GTT AAT ATG GAT TAT GTT GTT GGT GCT AGA ACT TTT AGA AGA GAA</td>
              </tr> <tr class="description-tr"> <td class="description-td">AAA AGA GCT GAA AGA TTG ACT TCT AGA GTT AAA GCT TTG TTT TCT GTT TTG AAT</td>
              </tr> <tr class="description-tr"> <td class="description-td">TAT GAA AGA GCT AGA AGA CCA GGT TTG TTG GGT GCT TCT GTT TTG GGT TTG GAT</td>
              </tr> <tr class="description-tr"> <td class="description-td">GAT ATT CAT AGA GCT TGG AGA ACT TTT GTT TTG AGA GTT AGA GCT CAA GAT CCA</td>
              </tr> <tr class="description-tr"> <td class="description-td">CCA CCA GAA TTG TAT TTT GTT AAA GTT GAT GTT ACT GGT GCT TAT GAT ACT ATT</td>
              </tr> <tr class="description-tr"> <td class="description-td">CCA CAA GAT AGA TTG ACT GAA GTT ATT GCT TCT ATT ATT AAA CCA CAA AAT ACT</td>
              </tr> <tr class="description-tr"> <td class="description-td">TAT TGT GTT AGA AGA TAT GCT GTT GTT CAA AAA GCT GCT CAT GGT CAT GTT AGA</td>
              </tr> <tr class="description-tr"> <td class="description-td">AAA GCT TTT AAA TCT CAT GTT TCT ACT TTG ACT GAT TTG CAA CCA TAT ATG AGA</td>
              </tr> <tr class="description-tr"> <td class="description-td">CAA TTT GTT GCT CAT TTG CAA GAA ACT TCT CCA TTG AGA GAT GCT GTT GTT ATT</td>
              </tr> <tr class="description-tr"> <td class="description-td">GAA CAA TCT TCT TCT TTG AAT GAA GCT TCT TCT GGT TTG TTT GAT GTT TTT TTG</td>
              </tr> <tr class="description-tr"> <td class="description-td">AGA TTT ATG TGT CAT CAT GCT GTT AGA ATT AGA GGT AAA TCT TAT GTT CAA TGT</td>
              </tr> <tr class="description-tr"> <td class="description-td">CAA GGT ATT CCA CAA GGT TCT ATT TTG TCT ACT TTG TTG TGT TCT TTG TGT TAT</td>
              </tr> <tr class="description-tr"> <td class="description-td">GGT GAT ATG GAA AAT AAA TTG TTT GCT GGT ATT AGA AGA GAT GGT TTG TTG TTG</td>
              </tr> <tr class="description-tr"> <td class="description-td">AGA TTG GTT GAT GAT TTT TTG TTG GTT ACT CCA CAT TTG ACT CAT GCT AAA ACT</td>
              </tr> <tr class="description-tr"> <td class="description-td">TTT TTG AGA ACT TTG GTT AGA GGT GTT CCA GAA TAT GGT TGT GTT GTT AAT TTG</td>
              </tr> <tr class="description-tr"> <td class="description-td">AGA AAA ACT GTT GTT AAT TTT CCA GTT GAA GAT GAA GCT TTG GGT GGT ACT GCT</td>
              </tr> <tr class="description-tr"> <td class="description-td">TTT GTT CAA ATG CCA GCT CAT GGT TTG TTT CCA TGG TGT GGT TTG TTG TTG GAT</td>
              </tr> <tr class="description-tr"> <td class="description-td">ACT AGA ACT TTG GAA GTT CAA TCT GAT TAT TCT TCT TAT GCT AGA ACT TCT ATT</td>
              </tr> <tr class="description-tr"> <td class="description-td">AGA GCT TCT TTG ACT TTT AAT AGA GGT TTT AAA GCT GGT AGA AAT ATG AGA AGA</td>
              </tr> <tr class="description-tr"> <td class="description-td">AAA TTG TTT GGT GTT TTG AGA TTG AAA TGT CAT TCT TTG TTT TTG GAT TTG CAA</td>
              </tr> <tr class="description-tr"> <td class="description-td">GTT AAT TCT TTG CAA ACT GTT TGT ACT AAT ATT TAT AAA ATT TTG TTG TTG CAA</td>
              </tr> <tr class="description-tr"> <td class="description-td">GCT TAT AGA TTT CAT GCT TGT GTT TTG CAA TTG CCA TTT CAT CAA CAA GTT TGG</td>
              </tr> <tr class="description-tr"> <td class="description-td">AAA AAT CCA ACT TTT TTT TTG AGA GTT ATT TCT GAT ACT GCT TCT TTG TGT TAT</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCT ATT TTG AAA GCT AAA AAT GCT GGT ATG TCT TTG GGT GCT AAA GGT GCT GCT</td>
              </tr> <tr class="description-tr"> <td class="description-td">GGT CCA TTG CCA TCT GAA GCT GTT CAA TGG TTG TGT CAT CAA GCT TTT TTG TTG</td>
              </tr> <tr class="description-tr"> <td class="description-td">AAA TTG ACT AGA CAT AGA GTT ACT TAT GTT CCA TTG TTG GGT TCT TTG AGA ACT</td>
              </tr> <tr class="description-tr"> <td class="description-td">GCT CAA ACT CAA TTG TCT AGA AAA TTG CCA GGT ACT ACT TTG ACT GCT TTG GAA</td>
              </tr> <tr class="description-tr"> <td class="description-td">GCT GCT GCT AAT CCA GCT TTG CCA TCT GAT TTT AAA ACT ATT TTG GAT         </td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">TABLE 9D</td>
              </tr> <tr class="description-tr"> <td class="description-td">Yeast (High Expressing Genes) (SEQ ID NO:641) </td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">ATG CCA AGA GCT CCA AGA TGT AGA GCT GTT AGA TCT TTG TTG AGA TCT CAC TAC</td>
              </tr> <tr class="description-tr"> <td class="description-td">AGA GAA GTT TTG CCA TTG GCT ACT TTC GTT AGA AGA TTG GGT CCA CAA GGT TGG</td>
              </tr> <tr class="description-tr"> <td class="description-td">AGA TTG GTT CAA AGA GGT GAC CCA GCT GCT TTC AGA GCT TTG GTT GCT CAA TGT</td>
              </tr> <tr class="description-tr"> <td class="description-td">TTG GTT TGT GTT CCA TGG GAC GCT AGA CCA CCA CCA GCT GCT CCA TCT TTC AGA</td>
              </tr> <tr class="description-tr"> <td class="description-td">CAA GTT TCT TGT TTG AAG GAA TTG GTT GCT AGA GTT TTG CAA AGA TTG TGT GAA</td>
              </tr> <tr class="description-tr"> <td class="description-td">AGA GGT GCT AAG AAC GTT TTG GCT TTC GGT TTC GCT TTG TTG GAC GGT GCT AGA</td>
              </tr> <tr class="description-tr"> <td class="description-td">GGT GGT CCA CCA GAA GCT TTC ACT ACT TCT GTT AGA TCT TAC TTG CCA AAC ACT</td>
              </tr> <tr class="description-tr"> <td class="description-td">GTT ACT GAC GCT TTG AGA GGT TCT GGT GCT TGG GGT TTG TTG TTG AGA AGA GTT</td>
              </tr> <tr class="description-tr"> <td class="description-td">GGT GAC GAC GTT TTG GTT CAC TTG TTG GCT AGA TGT GCT TTG TTC GTT TTG GTT</td>
              </tr> <tr class="description-tr"> <td class="description-td">GCT CCA TCT TGT GCT TAC CAA GTT TGT GGT CCA CCA TTG TAC CAA TTG GGT GCT</td>
              </tr> <tr class="description-tr"> <td class="description-td">GCT ACT CAA GCT AGA CCA CCA CCA CAC GCT TCT GGT CCA AGA AGA AGA TTG GGT</td>
              </tr> <tr class="description-tr"> <td class="description-td">TGT GAA AGA GCT TGG AAC CAC TCT GTT AGA GAA GCT GGT GTT CCA TTG GGT TTG</td>
              </tr> <tr class="description-tr"> <td class="description-td">CCA GCT CCA GGT GCT AGA AGA AGA GGT GGT TCT GCT TCT AGA TCT TTG CCA TTG</td>
              </tr> <tr class="description-tr"> <td class="description-td">CCA AAG AGA CCA AGA AGA GGT GCT GCT CCA GAA CCA GAA AGA ACT CCA GTT GGT</td>
              </tr> <tr class="description-tr"> <td class="description-td">CAA GGT TCT TGG GCT CAC CCA GGT AGA ACT AGA GGT CCA TCT GAC AGA GGT TTC</td>
              </tr> <tr class="description-tr"> <td class="description-td">TGT GTT GTT TCT CCA GCT AGA CCA GCT GAA GAA GCT ACT TCT TTG GAA GGT GCT</td>
              </tr> <tr class="description-tr"> <td class="description-td">TTG TCT GGT ACT AGA CAC TCT CAC CCA TCT GTT GGT AGA CAA CAC CAC GCT GGT</td>
              </tr> <tr class="description-tr"> <td class="description-td">CCA CCA TCT ACT TCT AGA CCA CCA AGA CCA TGG GAC ACT CCA TGT CCA CCA GTT</td>
              </tr> <tr class="description-tr"> <td class="description-td">TAC GCT GAA ACT AAG CAC TTC TTG TAC TCT TCT GGT GAC AAG GAA CAA TTG AGA</td>
              </tr> <tr class="description-tr"> <td class="description-td">CCA TCT TTC TTG TTG TCT TCT TTG AGA CCA TCT TTG ACT GGT GCT AGA AGA TTG</td>
              </tr> <tr class="description-tr"> <td class="description-td">GTT GAA ACT ATT TTC TTG GGT TCT AGA CCA TGG ATG CCA GGT ACT CCA AGA AGA</td>
              </tr> <tr class="description-tr"> <td class="description-td">TTG CCA AGA TTG CCA CAA AGA TAC TGG CAA ATG AGA CCA TTG TTC TTG GAA TTG</td>
              </tr> <tr class="description-tr"> <td class="description-td">TTG GGT AAC CAC GCT CAA TGT CCA TAC GGT GTT TTG TTG AAG ACT CAC TGT CCA</td>
              </tr> <tr class="description-tr"> <td class="description-td">TTG AGA GCT GCT GTT ACT CCA GCT GCT GGT GTT TGT GCT AGA GAA AAG CCA CAA</td>
              </tr> <tr class="description-tr"> <td class="description-td">GGT TCT GTT GCT GCT CCA GAA GAA GAA GAC ACT GAC CCA AGA AGA TTG GTT CAA</td>
              </tr> <tr class="description-tr"> <td class="description-td">TTG TTG AGA CAA CAC TCT TCT CCA TGG CAA GTT TAC GGT TTC GTT AGA GCT TGT</td>
              </tr> <tr class="description-tr"> <td class="description-td">TTG AGA AGA TTG GTT CCA CCA GGT TTG TGG GGT TCT AGA CAC AAC GAA AGA AGA</td>
              </tr> <tr class="description-tr"> <td class="description-td">TTC TTG AGA AAC ACT AAG AAG TTC ATT TCT TTG GGT AAG CAC GCT AAG TTG TCT</td>
              </tr> <tr class="description-tr"> <td class="description-td">TTG CAA GAA TTG ACT TGG AAG ATG TCT GTT AGA GAC TGT GCT TGG TTG AGA AGA</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCT CCA GGT GTT GGT TGT GTT CCA GCT GCT GAA CAC AGA TTG AGA GAA GAA ATT</td>
              </tr> <tr class="description-tr"> <td class="description-td">TTG GCT AAG TTC TTG CAC TGG TTG ATG TCT GTT TAC GTT GTT GAA TTG TTG AGA</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCT TTC TTC TAC GTT ACT GAA ACT ACT TTC CAA AAG AAC AGA TTG TTC TTC TAC</td>
              </tr> <tr class="description-tr"> <td class="description-td">AGA AAG TCT GTT TGG TCT AAG TTG CAA TCT ATT GGT ATT AGA CAA CAC TTG AAG</td>
              </tr> <tr class="description-tr"> <td class="description-td">AGA GTT CAA TTG AGA GAA TTG TCT GAA GCT GAA GTT AGA CAA CAC AGA GAA GCT</td>
              </tr> <tr class="description-tr"> <td class="description-td">AGA CCA GCT TTG TTG ACT TCT AGA TTG AGA TTC ATT CCA AAG CCA GAC GGT TTG</td>
              </tr> <tr class="description-tr"> <td class="description-td">AGA CCA ATT GTT AAC ATG GAC TAC GTT GTT GGT GCT AGA ACT TTC AGA AGA GAA</td>
              </tr> <tr class="description-tr"> <td class="description-td">AAG AGA GCT GAA AGA TTG ACT TCT AGA GTT AAG GCT TTG TTC TCT GTT TTG AAC</td>
              </tr> <tr class="description-tr"> <td class="description-td">TAC GAA AGA GCT AGA AGA CCA GGT TTG TTG GGT GCT TCT GTT TTG GGT TTG GAC</td>
              </tr> <tr class="description-tr"> <td class="description-td">GAC ATT CAC AGA GCT TGG AGA ACT TTC GTT TTG AGA GTT AGA GCT CAA GAC CCA</td>
              </tr> <tr class="description-tr"> <td class="description-td">CCA CCA GAA TTG TAC TTC GTT AAG GTT GAC GTT ACT GGT GCT TAC GAC ACT ATT</td>
              </tr> <tr class="description-tr"> <td class="description-td">CCA CAA GAC AGA TTG ACT GAA GTT ATT GCT TCT ATT ATT AAG CCA CAA AAC ACT</td>
              </tr> <tr class="description-tr"> <td class="description-td">TAC TGT GTT AGA AGA TAC GCT GTT GTT CAA AAG GCT GCT CAC GGT CAC GTT AGA</td>
              </tr> <tr class="description-tr"> <td class="description-td">AAG GCT TTC AAG TCT CAC GTT TCT ACT TTG ACT GAC TTG CAA CCA TAC ATG AGA</td>
              </tr> <tr class="description-tr"> <td class="description-td">CAA TTC GTT GCT CAC TTG CAA GAA ACT TCT CCA TTG AGA GAC GCT GTT GTT ATT</td>
              </tr> <tr class="description-tr"> <td class="description-td">GAA CAA TCT TCT TCT TTG AAC GAA GCT TCT TCT GGT TTG TTC GAC GTT TTC TTG</td>
              </tr> <tr class="description-tr"> <td class="description-td">AGA TTC ATG TGT CAC CAC GCT GTT AGA ATT AGA GGT AAG TCT TAC GTT CAA TGT</td>
              </tr> <tr class="description-tr"> <td class="description-td">CAA GGT ATT CCA CAA GGT TCT ATT TTG TCT ACT TTG TTG TGT TCT TTG TGT TAC</td>
              </tr> <tr class="description-tr"> <td class="description-td">GGT GAC ATG GAA AAC AAG TTG TTC GCT GGT ATT AGA AGA GAC GGT TTG TTG TTG</td>
              </tr> <tr class="description-tr"> <td class="description-td">AGA TTG GTT GAC GAC TTC TTG TTG GTT ACT CCA CAC TTG ACT CAC GCT AAG ACT</td>
              </tr> <tr class="description-tr"> <td class="description-td">TTC TTG AGA ACT TTG GTT AGA GGT GTT CCA GAA TAC GGT TGT GTT GTT AAC TTG</td>
              </tr> <tr class="description-tr"> <td class="description-td">AGA AAG ACT GTT GTT AAC TTC CCA GTT GAA GAC GAA GCT TTG GGT GGT ACT GCT</td>
              </tr> <tr class="description-tr"> <td class="description-td">TTC GTT CAA ATG CCA GCT CAC GGT TTG TTC CCA TGG TGT GGT TTG TTG TTG GAC</td>
              </tr> <tr class="description-tr"> <td class="description-td">ACT AGA ACT TTG GAA GTT CAA TCT GAC TAC TCT TCT TAC GCT AGA ACT TCT ATT</td>
              </tr> <tr class="description-tr"> <td class="description-td">AGA GCT TCT TTG ACT TTC AAC AGA GGT TTC AAG GCT GGT AGA AAC ATG AGA AGA</td>
              </tr> <tr class="description-tr"> <td class="description-td">AAG TTG TTC GGT GTT TTG AGA TTG AAG TGT CAC TCT TTG TTC TTG GAC TTG CAA</td>
              </tr> <tr class="description-tr"> <td class="description-td">GTT AAC TCT TTG CAA ACT GTT TGT ACT AAC ATT TAC AAG ATT TTG TTG TTG CAA</td>
              </tr> <tr class="description-tr"> <td class="description-td">GCT TAC AGA TTC CAC GCT TGT GTT TTG CAA TTG CCA TTC CAC CAA CAA GTT TGG</td>
              </tr> <tr class="description-tr"> <td class="description-td">AAG AAC CCA ACT TTC TTC TTG AGA GTT ATT TCT GAC ACT GCT TCT TTG TGT TAC</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCT ATT TTG AAG GCT AAG AAC GCT GGT ATG TCT TTG GGT GCT AAG GGT GCT GCT</td>
              </tr> <tr class="description-tr"> <td class="description-td">GGT CCA TTG CCA TCT GAA GCT GTT CAA TGG TTG TGT CAC CAA GCT TTC TTG TTG</td>
              </tr> <tr class="description-tr"> <td class="description-td">AAG TTG ACT AGA CAC AGA GTT ACT TAC GTT CCA TTG TTG GGT TCT TTG AGA ACT</td>
              </tr> <tr class="description-tr"> <td class="description-td">GCT CAA ACT CAA TTG TCT AGA AAG TTG CCA GGT ACT ACT TTG ACT GCT TTG GAA</td>
              </tr> <tr class="description-tr"> <td class="description-td">GCT GCT GCT AAC CCA GCT TTG CCA TCT GAC TTC AAG ACT ATT TTG GAC         </td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">TABLE 9E</td>
              </tr> <tr class="description-tr"> <td class="description-td">“Generic” hTRT Protein Encoding Sequence (SEQ ID NO:642) </td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">ATG CCA CGT GCC CCA CGT TGT CGT GCC GTT CGT TCT TTG TTG CGT TCT CAC TAC CGT</td>
              </tr> <tr class="description-tr"> <td class="description-td">GAA GTT TTG CCA TTG GCC ACC TTC GTT CGT CGT TTG GGT CCA CAA GGT TGG CGT TTG</td>
              </tr> <tr class="description-tr"> <td class="description-td">GTT CAA CGT GGT GAT CCA GCC GCC TTC CGT GCC TTG GTT GCC CAA TGT TTG GTT TGT</td>
              </tr> <tr class="description-tr"> <td class="description-td">GTT CCA TGG GAT GCC CGT CCA CCA CCA GCC GCC CCA TCT TTC CGT CAA GTT TCT TGT</td>
              </tr> <tr class="description-tr"> <td class="description-td">TTG AAA GAA TTG GTT GCC CGT GTT TTG CAA CGT TTG TGT GAA CGT GGT GCC AAA AAC</td>
              </tr> <tr class="description-tr"> <td class="description-td">GTT TTG GCC TTC GGT TTC GCC TTG TTG GAT GGT GCC CGT GGT GGT CCA CCA GAA GCC</td>
              </tr> <tr class="description-tr"> <td class="description-td">TTC ACC ACC TCT GTT CGT TCT TAC TTG CCA AAC ACC GTT ACC GAT GCC TTG CGT GGT</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCT GGT GCC TGG GGT TTG TTG TTG CGT CGT GTT GGT GAT GAT GTT TTG GTT CAC TTG</td>
              </tr> <tr class="description-tr"> <td class="description-td">TTG GCC CGT TGT GCC TTG TTC GTT TTG GTT GCC CCA TCT TGT GCC TAC CAA GTT TGT</td>
              </tr> <tr class="description-tr"> <td class="description-td">GGT CCA CCA TTG TAC CAA TTG GGT GCC GCC ACC CAA GCC CGT CCA CCA CCA CAC GCC</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCT GGT CCA CGT CGT CGT TTG GGT TGT GAA CGT GCC TGG AAC CAC TCT GTT CGT GAA</td>
              </tr> <tr class="description-tr"> <td class="description-td">GCC GGT GTT CCA TTG GGT TTG CCA GCC CCA GGT GCC CGT CGT CGT GGT GGT TCT GCC</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCT CGT TCT TTG CCA TTG CCA AAA CGT CCA CGT CGT GGT GCC GCC CCA GAA CCA GAA</td>
              </tr> <tr class="description-tr"> <td class="description-td">CGT ACC CCA GTT GGT CAA GGT TCT TGG GCC CAC CCA GGT CGT ACC CGT GGT CCA TCT</td>
              </tr> <tr class="description-tr"> <td class="description-td">GAT CGT GGT TTC TGT GTT GTT TCT CCA GCC CGT CCA GCC GAA GAA GCC ACC TCT TTG</td>
              </tr> <tr class="description-tr"> <td class="description-td">GAA GGT GCC TTG TCT GGT ACC CGT CAC TCT CAC CCA TCT GTT GGT CGT CAA CAC CAC</td>
              </tr> <tr class="description-tr"> <td class="description-td">GCC GGT CCA CCA TCT ACC TCT CGT CCA CCA CGT CCA TGG GAT ACC CCA TGT CCA CCA</td>
              </tr> <tr class="description-tr"> <td class="description-td">GTT TAC GCC GAA ACC AAA CAC TTC TTG TAC TCT TCT GGT GAT AAA GAA CAA TTG CGT</td>
              </tr> <tr class="description-tr"> <td class="description-td">CCA TCT TTC TTG TTG TCT TCT TTG CGT CCA TCT TTG ACC GQT GCC CGT CGT TTG GTT</td>
              </tr> <tr class="description-tr"> <td class="description-td">GAA ACC ATT TTC TTG GGT TCT CGT CCA TGG ATG CCA GGT ACC CCA CGT CGT TTG CCA</td>
              </tr> <tr class="description-tr"> <td class="description-td">CGT TTG CCA CAA CGT TAC TGG CAA ATG CGT CCA TTG TTC TTG GAA TTG TTG GGT AAC</td>
              </tr> <tr class="description-tr"> <td class="description-td">CAC GCC CAA TGT CCA TAC GGT GTT TTG TTG AAA ACC CAC TGT CCA TTG CGT GCC GCC</td>
              </tr> <tr class="description-tr"> <td class="description-td">GTT ACC CCA GCC GCC GGT GTT TGT GCC CGT GAA AAA CCA CAA GGT TCT GTT GCC GCC</td>
              </tr> <tr class="description-tr"> <td class="description-td">CCA GAA GAA GAA GAT ACC GAT CCA CGT CGT TTG GTT CAA TTG TTG CGT CAA CAC TCT</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCT CCA TGG CAA GTT TAC GGT TTC GTT CGT GCC TGT TTG CGT CGT TTG GTT CCA CCA</td>
              </tr> <tr class="description-tr"> <td class="description-td">GGT TTG TGG GGT TCT CGT CAC AAC GAA CGT CGT TTC TTG CGT AAC ACC AAA AAA TTC</td>
              </tr> <tr class="description-tr"> <td class="description-td">ATT TCT TTG GGT AAA CAC GCC AAA TTG TCT TTG CAA GAA TTG ACC TGG AAA ATG TCT</td>
              </tr> <tr class="description-tr"> <td class="description-td">GTT CGT GAT TGT GCC TGG TTG CGT CGT TCT CCA GGT GTT GGT TGT GTT CCA GCC GCC</td>
              </tr> <tr class="description-tr"> <td class="description-td">GAA CAC CGT TTG CGT GAA GAA ATT TTG GCC AAA TTC TTG CAC TGG TTG ATG TCT GTT</td>
              </tr> <tr class="description-tr"> <td class="description-td">TAC GTT GTT GAA TTG TTG CGT TCT TTC TTC TAC GTT ACC GAA ACC ACC TTC CAA AAA</td>
              </tr> <tr class="description-tr"> <td class="description-td">AAC CGT TTG TTC TTC TAC CGT AAA TCT GTT TGG TCT AAA TTG CAA TCT ATT GGT ATT</td>
              </tr> <tr class="description-tr"> <td class="description-td">CGT CAA CAC TTG AAA CGT GTT CAA TTG CGT GAA TTG TCT GAA GCC GAA GTT CGT CAA</td>
              </tr> <tr class="description-tr"> <td class="description-td">CAC CGT GAA GCC CGT CCA GCC TTG TTG ACC TCT CGT TTG CGT TTC ATT CCA AAA CCA</td>
              </tr> <tr class="description-tr"> <td class="description-td">GAT GGT TTG CGT CCA ATT GTT AAC ATG GAT TAC GTT GTT GGT GCC CGT ACC TTC CGT</td>
              </tr> <tr class="description-tr"> <td class="description-td">CGT GAA AAA CGT GCC GAA CGT TTG ACC TCT CGT GTT AAA GCC TTG TTC TCT GTT TTG</td>
              </tr> <tr class="description-tr"> <td class="description-td">AAC TAC GAA CGT GCC CGT CGT CCA GGT TTG TTG GGT GCC TCT GTT TTG GGT TTG GAT</td>
              </tr> <tr class="description-tr"> <td class="description-td">GAT ATT CAC CGT GCC TGG CGT ACC TTC GTT TTG CGT GTT CGT GCC CAA GAT CCA CCA</td>
              </tr> <tr class="description-tr"> <td class="description-td">CCA GAA TTG TAC TTC GTT AAA GTT GAT GTT ACC GGT GCC TAC GAT ACC ATT CCA CAA</td>
              </tr> <tr class="description-tr"> <td class="description-td">GAT CGT TTG ACC GAA GTT ATT GCC TCT ATT ATT AAA CCA CAA AAC ACC TAC TGT GTT</td>
              </tr> <tr class="description-tr"> <td class="description-td">CGT CGT TAC GCC GTT GTT CAA AAA GCC GCC CAC GGT CAC GTT CGT AAA GCC TTC AAA</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCT CAC GTT TCT ACC TTG ACC GAT TTG CAA CCA TAC ATG CGT CAA TTC GTT GCC CAC</td>
              </tr> <tr class="description-tr"> <td class="description-td">TTG CAA GAA ACC TCT CCA TTG CGT GAT GCC GTT GTT ATT GAA CAA TCT TCT TCT TTG</td>
              </tr> <tr class="description-tr"> <td class="description-td">AAC GAA GCC TCT TCT GGT TTG TTC GAT GTT TTC TTG CGT TTC ATG TGT CAC CAC GCC</td>
              </tr> <tr class="description-tr"> <td class="description-td">GTT CGT ATT CGT GGT AAA TCT TAC GTT CAA TGT CAA GGT ATT CCA CAA GGT TCT ATT</td>
              </tr> <tr class="description-tr"> <td class="description-td">TTG TCT ACC TTG TTG TGT TCT TTG TGT TAC GGT GAT ATG GAA AAC AAA TTG TTC GCC</td>
              </tr> <tr class="description-tr"> <td class="description-td">GGT ATT CGT CGT GAT GGT TTG TTG TTG CGT TTG GTT GAT GAT TTC TTG TTG GTT ACC</td>
              </tr> <tr class="description-tr"> <td class="description-td">CCA CAC TTG ACC CAC GCC AAA ACC TTC TTG CGT ACC TTG GTT CGT GGT GTT CCA GAA</td>
              </tr> <tr class="description-tr"> <td class="description-td">TAC GGT TGT GTT GTT AAC TTG CGT AAA ACC GTT GTT AAC TTC CCA GTT GAA GAT GAA</td>
              </tr> <tr class="description-tr"> <td class="description-td">GCC TTG GGT GGT ACC GCC TTC GTT CAA ATG CCA GCC CAC GGT TTG TTC CCA TGG TGT</td>
              </tr> <tr class="description-tr"> <td class="description-td">GGT TTG TTG TTG GAT ACC CGT ACC TTG GAA GTT CAA TCT GAT TAC TCT TCT TAC GCC</td>
              </tr> <tr class="description-tr"> <td class="description-td">CGT ACC TCT ATT CGT GCC TCT TTG ACC TTC AAC CGT GGT TTC AAA GCC GGT CGT AAC</td>
              </tr> <tr class="description-tr"> <td class="description-td">ATG CGT CGT AAA TTG TTC GGT GTT TTG CGT TTG AAA TGT CAC TCT TTG TTC TTG GAT</td>
              </tr> <tr class="description-tr"> <td class="description-td">TTG CAA GTT AAC TCT TTG CAA ACC GTT TGT ACC AAC ATT TAC AAA ATT TTG TTG TTG</td>
              </tr> <tr class="description-tr"> <td class="description-td">CAA GCC TAC CGT TTC CAC GCC TGT GTT TTG CAA TTG CCA TTC CAC CAA CAA GTT TGG</td>
              </tr> <tr class="description-tr"> <td class="description-td">AAA AAC CCA ACC TTC TTC TTG CGT GTT ATT TCT GAT ACC GCC TCT TTG TGT TAC TCT</td>
              </tr> <tr class="description-tr"> <td class="description-td">ATT TTG AAA GCC AAA AAC GCC GGT ATG TCT TTG GGT GCC AAA GGT GCC GCC GGT CCA</td>
              </tr> <tr class="description-tr"> <td class="description-td">TTG CCA TCT GAA GCC GTT CAA TGG TTG TGT CAC CAA GCC TTC TTG TTG AAA TTG ACC</td>
              </tr> <tr class="description-tr"> <td class="description-td">CGT CAC CGT GTT ACC TAC GTT CCA TTG TTG GGT TCT TTG CGT ACC GCC CAA ACC CAA</td>
              </tr> <tr class="description-tr"> <td class="description-td">TTG TCT CGT AAA TTG CCA GGT ACC ACC TTG ACC GCC TTG GAA GCC GCC GCC AAC CCA</td>
              </tr> <tr class="description-tr"> <td class="description-td">GCC TTG CCA TCT GAT TTC AAA ACC ATT TTG GAT                                </td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p>Following determination of the desired nucleotide sequence for the hTRT protein-encoding polynucleotide, the polynucleotide can be made by any suitable method including de novo chemical synthesis, directed mutagenesis of a synthetic or naturally occurring TRT gene or cDNA, or a combination of these methods. In one exemplary embodiment, oligonucleotides (typically 50-100 bases in length) are synthesized with a 5′ phosphate group and include approximately 10-base overhangs (relative to adjacent oligonucleotides in the assembled gene) to direct subsequent ligations. Following purification and desalting, each oligonucleotide is annealed to its complement (e.g., by combining pairs of oligonucleotides in equimolar amounts in a neutral pH buffer with 50-200 mM NaCl and 0.5 mM MgCl<sub>2</sub>). Annealing may be monitored by native PAGE. The resulting double-stranded oligonucleotides are ligated to their neighbors in pairs. After each ligation the products are gel-purified, then ligated to the appropriate (neighboring) double-stranded DNAs. In this manner, fragments of approximately 600-800 basepairs are built up. These intermediate fragments are then cloned into vectors and sequenced. The fragments are then combined into a single vector (resulting in a vector containing a polynucleotide with the desired hTRT protein-encoding sequence). This step is facilitated by using restriction sites present in, or engineered into, the polynucleotide sequence. Alternatively, the fragments can be built up by ligation until the complete cDNA is assembled and the assembled sequence cloned into a vector. Numerous other alternative methods and approaches will be apparent to those of skill in the art.</p>
    <p>Table 10A shows an exemplary set of oligonucleotides that can be used to produce a polynucleotide, shown in Table 10B, that employs a codon distribution preferentially used by highly expressed genes in <i>E. coli</i>. The sequence in Table 5B contains silent changes to some codons to introduce useful restriction sites spaced every 300-800 base pairs, to facilitate subcloning and modification. Oligonucleotide pairs for the initial annealing steps are indicated by the labels “T” (top strand) and “B” (bottom strand). The full-length polynucleotide (Table 10B) encodes the hTRT protein (with the start codon at nucleotides 28-30) and contains Sac I and Xho I sites at the termini flanking the open reading frame, which are useful for cloning into a variety of vectors (e.g., pBluescript II KS, Stratagene Inc., San Diego Calif.). Once cloned into an appropriate vector, the hTRT sequence may be expressed, modified (e.g., by site directed or cassette mutagenesis), subcloned, or otherwise used or manipulated. In one embodiment, the polynucleotide is subcloned into a pET vector containing a T7 RNA polymerase promoter (Novagen Inc., Madison, Wis.) and introduced into an <i>E. coli </i>strain having an inducible T7 polymerase (Novagen Inc., Madison, Wis.). One advantage to the pET system is that the <i>E. coli </i>culture may be grown before the T7 RNA polymerase gene is induced, resulting in very high levels of transcription and minimizing the effect of any potential detrimental effect of the expressed protein on the cells.</p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" pgwide="1" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="322pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 10</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">SYNTHESIS OF hTRT POLYNUCLEOTIDE HAVING ALTERNATIVE</td>
              </tr> <tr class="description-tr"> <td class="description-td">CODON DISTRIBUTION</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="322pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Table 10A: Oligonucleotides (SEQ ID NOS:643-721)</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="1" colwidth="21pt" align="left"> </colspec> <colspec colname="2" colwidth="301pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">  1B</td>
                <td class="description-td">CCAGCGGCAGAACTTCGCGATAGTGGGAACGCAGCAGGGAACGAACAGCACGGCAACGCG</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">GAGCACGCGGCATATGGTCGACTCTAGAGCTCCCGCGTGC</td>
              </tr> <tr class="description-tr"> <td class="description-td">1T</td>
                <td class="description-td">GCACGCGGGAGCTCTAGAGTCGACCATATGCCGCGTGCTCCGCGTTGCCGTGCTGTTCGTTC</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">CCTGCTGCGTTCCCACTATCGCGAAGTT</td>
              </tr> <tr class="description-tr"> <td class="description-td">2B</td>
                <td class="description-td">GGCACTGAGCAACCAGAGCACGGAAAGCAGCCGGGTCACCACGCTGAACCAGACGCCAAC</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">CCTGCGGGCCCAGACGACGAACGAAGGTAG</td>
              </tr> <tr class="description-tr"> <td class="description-td">2T</td>
                <td class="description-td">CTGCCGCTGGCTACCTTCGTTCGTCGTCTGGGCCCGCAGGGTTGGCGTCTGGTTCAGCGTGG</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">TGACCCGGCTGCTTTCCGTGCTCTGGTT</td>
              </tr> <tr class="description-tr"> <td class="description-td">3B</td>
                <td class="description-td">GAACACGAGCAACCAGTTCTTTCAGGCAGGAAACCTGACGGAAGGACGGAGCAGCCGGCG</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">GCGGACGAGCGTCCCACGGAACGCAAACCA</td>
              </tr> <tr class="description-tr"> <td class="description-td">3T</td>
                <td class="description-td">GCTCAGTGCCTGGTTTGCGTTCCGTGGGACGCTCGTCCGCCGCCGGCTGCTCCGTCCTTCCGT</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">CAGGTTTCCTGCCTGAAAGAACTGGTT</td>
              </tr> <tr class="description-tr"> <td class="description-td">4B</td>
                <td class="description-td">ATGCTTCCGGCGGACCACCACGAGCACCGTCCAGCAGAGCGAAACCGAAAGCCAGAACGTT</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">TTTAGCACCACGTTCGCACAGACGCTGCA</td>
              </tr> <tr class="description-tr"> <td class="description-td">4T</td>
                <td class="description-td">GCTCGTGTTCTGCAGCGTCTGTGCGAACGTGGTGCTAAAAACGTTCTGGCTTTCGGTTTCGC</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">TCTGCTGGACGGTGCTCGTGGTGGTCCG</td>
              </tr> <tr class="description-tr"> <td class="description-td">5B</td>
                <td class="description-td">CAACACGACGCAGCAGCAGACCCCAAGCACCGGAACCACGCAGAGCGTCGGTAACGGTGTT</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">CGGCAGGTAGGAACGAACGGAGGTGGTGA</td>
              </tr> <tr class="description-tr"> <td class="description-td">5T</td>
                <td class="description-td">CCGGAAGCATTCACCACCTCCGTTCGTTCCTACCTGCCGAACACCGTTACCGACGCTCTGCG</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">TGGTTCCGGTGCTTGGGGTCTGCTGCTG</td>
              </tr> <tr class="description-tr"> <td class="description-td">6B</td>
                <td class="description-td">GCGGCGGACCACAAACCTGGTAAGCGCAGGACGGAGCAACCAGAACGAACAGAGCGCAAC</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">GAGCCAGCAGGTGAACCAGAACGTCGTCAC</td>
              </tr> <tr class="description-tr"> <td class="description-td">6T</td>
                <td class="description-td">CGTCGTGTTGGTGACGACGTTCTGGTTCACCTGCTGGCTCGTTGCGCTCTGTTCGTTCTGGTT</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">GCTCCGTCCTGCGCTTACCAGGTTTGT</td>
              </tr> <tr class="description-tr"> <td class="description-td">7B</td>
                <td class="description-td">GGTTCCAAGCACGTTCGCAACCCAGACGACGACGCGGACCGGAAGCGTGCGGCGGCGGAC</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">GAGCCTGGGTAGCAGCACCCAGCTGGTACA</td>
              </tr> <tr class="description-tr"> <td class="description-td">7T</td>
                <td class="description-td">GGTCCGCCGCTGTACCAGCTGGGTGCTGCTACCCAGGCTCGTCCGCCGCCGCACGCTTCCGG</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">TCCGCGTCGTCGTCTGGGTTGCGAACGT</td>
              </tr> <tr class="description-tr"> <td class="description-td">8B</td>
                <td class="description-td">GCAGCGGCAGGGAACGGGAAGCGGAACCACCACGACGACGAGCACCCGGAGCCGGCAGAC</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">CCAGCGGAACACCAGCTTCACGAACGGAGT</td>
              </tr> <tr class="description-tr"> <td class="description-td">8T</td>
                <td class="description-td">GCTTGGAACCACTCCGTTCGTGAAGCTGGTGTTCCGCTGGGTCTGCCGGCTCCGGGTGCTCG</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">TCGTCGTGGTGGTTCCGCTTCCCGTTCC</td>
              </tr> <tr class="description-tr"> <td class="description-td">9B</td>
                <td class="description-td">GACCACGGGTACGACCCGGGTGAGCCCAGGAACCCTGACCAACCGGGGTACGTTCCGGTTC</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">CGGAGCAGCACCACGACGCGGACGTTTCG</td>
              </tr> <tr class="description-tr"> <td class="description-td">9T</td>
                <td class="description-td">CTGCCGCTGCCGAAACGTCCGCGTCGTGGTGCTGCTCCGGAACCGGAACGTACCCCGGTTGG</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">TCAGGGTTCCTGGGCTCACCCGGGTCGT</td>
              </tr> <tr class="description-tr"> <td class="description-td">10B</td>
                <td class="description-td">AGTGACGGGTGCCGGACAGAGCACCTTCCAGGGAGGTAGCTTCTTCAGCCGGACGAGCCGG</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">GGAAACAACGCAGAAACCACGGTCGGACG</td>
              </tr> <tr class="description-tr"> <td class="description-td">10T</td>
                <td class="description-td">ACCCGTGGTCCGTCCGACCGTGGTTTCTGCGTTGTTTCCCCGGCTCGTCCGGCTGAAGAAGC</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">TACCTCCCTGGAAGGTGCTCTGTCCGGC</td>
              </tr> <tr class="description-tr"> <td class="description-td">11B</td>
                <td class="description-td">AAACCGGCGGGCACGGGGTGTCCCACGGACGCGGCGGACGGGAGGTGGACGGCGGACCAG</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">CGTGGTGCTGACGACCAACGGACGGGTGGG</td>
              </tr> <tr class="description-tr"> <td class="description-td">11T</td>
                <td class="description-td">ACCCGTCACTCCCACCCGTCCGTTGGTCGTCAGCACCACGCTGGTCCGCCGTCCACCTCCCG</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">TCCGCCGCGTCCGTGGGACACCCCGTGC</td>
              </tr> <tr class="description-tr"> <td class="description-td">12B</td>
                <td class="description-td">TCAGGGACGGACGCAGGGAGGACAGCAGGAAGGACGGACGCAGCTGTTCTTTGTCACCGG</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">AGGAGTACAGGAAGTGTTTGGTTTCAGCGT</td>
              </tr> <tr class="description-tr"> <td class="description-td">12T</td>
                <td class="description-td">CCGCCGGTTTACGCTGAAACCAAACACTTCCTGTACTCCTCCGGTGACAAAGAACAGCTGCG</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">TCCGTCCTTCCTGCTGTCCTCCCTGCGT</td>
              </tr> <tr class="description-tr"> <td class="description-td">13B</td>
                <td class="description-td">GCTGCGGCAGACGCGGCAGACGACGCGGGGTGCCCGGCATCCACGGACGGGAACCCAGGA</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">AGATAGTTTCAACCAGACGACGAGCACCGG</td>
              </tr> <tr class="description-tr"> <td class="description-td">13T</td>
                <td class="description-td">CCGTCCCTGACCGGTGCTCGTCGTCTGGTTGAAACTATCTTCCTGGGTTCCCGTCCGTGGA</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">TGCCGGGCACCCCGCGTCGTCTGCCGCGT</td>
              </tr> <tr class="description-tr"> <td class="description-td">14B</td>
                <td class="description-td">GCGGGCAGTGGGTTTTCAGCAGAACACCATACGGGCACTGAGCGTGGTTGCCCAGCAGTTC</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">CAGGAACAGCGGACGCATCTGCCAGTAAC</td>
              </tr> <tr class="description-tr"> <td class="description-td">14T</td>
                <td class="description-td">CTGCCGCAGCGTTACTGGCAGATGCGTCCGCTGTTCCTGGAACTGCTGGGCAACCACGCTCA</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">GTGCCCGTATGGTGTTCTGCTGAAAACC</td>
              </tr> <tr class="description-tr"> <td class="description-td">15B</td>
                <td class="description-td">GGTCGGTATCTTCTTCTTCCGGAGCAGCAACGGAACCCTGCGGTTTTTCACGAGCGCAAACA</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">CCAGCAGCCGGGGTAACAGCAGCACGCA</td>
              </tr> <tr class="description-tr"> <td class="description-td">15T</td>
                <td class="description-td">CACTGCCCGCTGCGTGCTGCTGTTACCCCGGCTGCTGGTGTTTGCGCTCGTGAAAAACCGCA</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">GGGTTCCGTTGCTGCTCCGGAAGAAGAA</td>
              </tr> <tr class="description-tr"> <td class="description-td">16B</td>
                <td class="description-td">GCGGAACCAGACGACGCAGGCATGCACGAACGAAACCGTAAACCTGCCACGGGGAGGAGT</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">GCTGACGCAGCAGCTGAACCAGACGACGCG</td>
              </tr> <tr class="description-tr"> <td class="description-td">16T</td>
                <td class="description-td">GATACCGACCCGCGTCGTCTGGTTCAGCTGCTGCGTCAGCACTCCTCCCCGTGGCAGGTTTA</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">CGGTTTCGTTCGTGCATGCCTGCGTCGT</td>
              </tr> <tr class="description-tr"> <td class="description-td">17B</td>
                <td class="description-td">GGGACAGTTTAGCGTGTTTACCCAGGGAGATGAATTTTTTGGTGTTACGCAGGAAACGACGT</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">TCGTTGTGACGGGAACCCCACAGACCCG</td>
              </tr> <tr class="description-tr"> <td class="description-td">17T</td>
                <td class="description-td">CTGGTTCCGCCGGGTCTGTGGGGTTCCCGTCACAACGAACGTCGTTTCCTGCGTAACACCAA</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">AAAATTCATCTCCCTGGGTAAACACGCT</td>
              </tr> <tr class="description-tr"> <td class="description-td">18B</td>
                <td class="description-td">GGTGTTCAGCAGCCGGAACGCAACCAACACCCGGAGAACGACGCAGCCAAGCGCAGTCAC</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">AACGGACATTTTCCAGGTCAGTTCCTGCA</td>
              </tr> <tr class="description-tr"> <td class="description-td">18T</td>
                <td class="description-td">AAACTGTCCCTGCAGGAACTGACCTGGAAAATGTCCGTTCGTGACTGCGCTTGGCTGCGTCG</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">TTCTCCGGGTGTTGGTTGCGTTCCGGCT</td>
              </tr> <tr class="description-tr"> <td class="description-td">19B</td>
                <td class="description-td">CGGTAACGTAGAAGAAGGAACGCAGCAGTTCAACAACGTATACGGACATCAGCCAGTGCA</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">GGAATTTAGCCAGGATTTCTTCACGCAGAC</td>
              </tr> <tr class="description-tr"> <td class="description-td">19T</td>
                <td class="description-td">GCTGAACACCGTCTGCGTGAAGAAATCCTGGCTAAATTCCTGCACTGGCTGATGTCCGTATA</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">CGTTGTTGAACTGCTGCGTTCCTTCTTC</td>
              </tr> <tr class="description-tr"> <td class="description-td">20B</td>
                <td class="description-td">GTTTCAGGTGCTGACGGATACCGATGGACTGCAGTTTGGACCAAACGGATTTACGGTAGAA</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">GAACAGACGGTTTTTCTGGAAGGTGGTTT</td>
              </tr> <tr class="description-tr"> <td class="description-td">20T</td>
                <td class="description-td">TACGTTACCGAAACCACCTTCCAGAAAAACCGTCTGTTCTTCTACCGTAAATCCGTTTGGTC</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">CAAACTGCAGTCCATCGGTATCCGTCAG</td>
              </tr> <tr class="description-tr"> <td class="description-td">21B</td>
                <td class="description-td">GATGAAACGCAGACGGGAGGTCAGCAGAGCCGGACGAGCTTCACGGTGCTGACGAACTTCA</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">GCTTCGGACAGTTCACGCAGCTGAACAC</td>
              </tr> <tr class="description-tr"> <td class="description-td">21T</td>
                <td class="description-td">CACCTGAAACGTGTTCAGCTGCGTGAACTGTCCGAAGCTGAAGTTCGTCAGCACCGTGAA</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">CTCGTCCGGCTCTGCTGACCTCCCGTCTG</td>
              </tr> <tr class="description-tr"> <td class="description-td">22B</td>
                <td class="description-td">TCAGACGCTCAGCACGTTTTTCACGACGGAAGGTACGAGCACCAACAACGTAGTCCATGTTT</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">ACGATCGGACGCAGACCGTCCGGTTTCG</td>
              </tr> <tr class="description-tr"> <td class="description-td">22T</td>
                <td class="description-td">CGTTTCATCCCGAAACCGGACGGTCTGCGTCCGATCGTAAACATGGACTACGTTGTTGGTGC</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">TCGTACCTTCCGTCGTGAAAAACGTGCT</td>
              </tr> <tr class="description-tr"> <td class="description-td">23B</td>
                <td class="description-td">CGTCCAGACCCAGAACGGAAGCACCCAGCAGACCCGGACGACGAGCACGTTCGTAGTTCAG</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">AACGGAGAACAGAGCTTTAACACGGGAGG</td>
              </tr> <tr class="description-tr"> <td class="description-td">23T</td>
                <td class="description-td">GAGCGTCTGACCTCCCGTGTTAAAGCTCTGTTCTCCGTTCTGAACTACGAACGTGCTCGTCG</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">TCCGGGTCTGCTGGGTGCTTCCGTTCTG</td>
              </tr> <tr class="description-tr"> <td class="description-td">24B</td>
                <td class="description-td">CGGTAACGTCAACTTTAACGAAGTACAGTTCCGGCGGCGGGTCCTGAGCACGAACACGCAG</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">AACGAAGGTACGCCAAGCACGGTGGATGT</td>
              </tr> <tr class="description-tr"> <td class="description-td">24T</td>
                <td class="description-td">GGTCTGGACGACATCCACCGTGCTTGGCGTACCTTCGTTCTGCGTGTTCGTGCTCAGGACCC</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">GCCGCCGGAACTGTACTTCGTTAAAGTT</td>
              </tr> <tr class="description-tr"> <td class="description-td">25B</td>
                <td class="description-td">CGTAACGACGAACGCAGTAGGTGTTCTGCGGTTTGATGATGGAAGCGATAACTTCGGTCAG</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">ACGGTCCTGCGGGATGGTGTCGTACGCGC</td>
              </tr> <tr class="description-tr"> <td class="description-td">25T</td>
                <td class="description-td">GACGTTACCGGCGCGTACGACACCATCCCGCAGGACCGTCTGACCGAAGTTATCGCTTCCAT</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">CATCAAACCGCAGAACACCTACTGCGTT</td>
              </tr> <tr class="description-tr"> <td class="description-td">26B</td>
                <td class="description-td">GACGCATGTACGGCTGCAGGTCGGTCAGGGTGGAAACGTGGGATTTGAATGCTTTACGAAC</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">GTGACCGTGAGCAGCTTTCTGAACAACAG</td>
              </tr> <tr class="description-tr"> <td class="description-td">26T</td>
                <td class="description-td">CGTCGTTACGCTGTTGTTCAGAAAGCTGCTCACGGTCACGTTCGTAAAGCATTCAAATCCCA</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">CGTTTCCACCCTGACCGACCTGCAGCCG</td>
              </tr> <tr class="description-tr"> <td class="description-td">27B</td>
                <td class="description-td">GACCGGAGGAAGCTTCGTTCAGGGAGGAGGACTGTTCGATAACAACAGCGTCACGCAGCGG</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">GGAGGTTTCCTGCAGGTGAGCAACGAACT</td>
              </tr> <tr class="description-tr"> <td class="description-td">27T</td>
                <td class="description-td">TACATGCGTCAGTTCGTTGCTCACCTGCAGGAAACCTCCCCGCTGCGTGACGCTGTTGTTAT</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">CGAACAGTCCTCCTCCCTGAACGAAGCT</td>
              </tr> <tr class="description-tr"> <td class="description-td">28B</td>
                <td class="description-td">AACCCTGCGGGATACCCTGGCACTGAACGTAGGATTTACCACGGATACGAACAGCGTGGTG</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">GCACATGAAACGCAGGAAAACGTCGAACA</td>
              </tr> <tr class="description-tr"> <td class="description-td">28T</td>
                <td class="description-td">TCCTCCGGTCTGTTCGACGTTTTCCTGCGTTTCATGTGCCACCACGCTGTTCGTATCCGTGGT</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">AAATCCTACGTTCAGTGCCAGGGTATC</td>
              </tr> <tr class="description-tr"> <td class="description-td">29B</td>
                <td class="description-td">GCAGCAGCAGACCGTCACGACGGATACCAGCGAACAGTTTGTTTTCCATGTCACCGTAGCA</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">CAGGGAGCACAGCAGGGTGGACAGGATGG</td>
              </tr> <tr class="description-tr"> <td class="description-td">29T</td>
                <td class="description-td">CCGCAGGGTTCCATCCTGTCCACCCTGCTGTGCTCCCTGTGCTACGGTGACATGGAAAACAA</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">ACTGTTCGCTGGTATCCGTCGTGACGGT</td>
              </tr> <tr class="description-tr"> <td class="description-td">30B</td>
                <td class="description-td">CGTATTCCGGAACACCACGAACCAGGGTACGCAGGAAGGTTTTAGCGTGGGTCAGGTGCGG</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">AGTAACCAGCAGGAAGTCGTCAACCAGAC</td>
              </tr> <tr class="description-tr"> <td class="description-td">30T</td>
                <td class="description-td">CTGCTGCTGCGTCTGGTTGACGACTTCCTGCTGGTTACTCCGCACCTGACCCACGCTAAAAC</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">CTTCCTGCGTACCCTGGTTCGTGGTGTT</td>
              </tr> <tr class="description-tr"> <td class="description-td">31B</td>
                <td class="description-td">GAGCCGGCATCTGAACGAAAGCGGTGCCACCCAGAGCTTCGTCTTCAACCGGGAAGTTAAC</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">AACGGTTTTACGCAGGTTTACAACGCAAC</td>
              </tr> <tr class="description-tr"> <td class="description-td">31T</td>
                <td class="description-td">CCGGAATACGGTTGCGTTGTAAACCTGCGTAAAACCGTTGTTAACTTCCCGGTTGAAGACGA</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">AGCTCTGGGTGGCACCGCTTTCGTTCAG</td>
              </tr> <tr class="description-tr"> <td class="description-td">32B</td>
                <td class="description-td">GGATGGAGGTACGAGCGTAGGAGGAGTAGTCGGACTGAACTTCCAGGGTACGGGTGTCCAG</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">CAGCAGACCGCACCACGGGAACAGACCGT</td>
              </tr> <tr class="description-tr"> <td class="description-td">32T</td>
                <td class="description-td">ATGCCGGCTCACGGTCTGTTCCCGTGGTGCGGTCTGCTGCTGGACACCCGTACCCTGGAAGT</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">TCAGTCCGACTACTCCTCCTACGCTCGT</td>
              </tr> <tr class="description-tr"> <td class="description-td">33B</td>
                <td class="description-td">GGGAGTGGCATTTCAGACGCAGAACACCGAACAGTTTACGACGCATGTTACGACCAGCTTT</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">GAAACCACGGTTGAAGGTCAGGGAAGCAC</td>
              </tr> <tr class="description-tr"> <td class="description-td">33T</td>
                <td class="description-td">ACCTCCATCCGTGCTTCCCTGACCTTCAACCGTGGTTTCAAAGCTGGTCGTAACATGCGTCGT</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">AAACTGTTCGGTGTTCTGCGTCTGAAA</td>
              </tr> <tr class="description-tr"> <td class="description-td">34B</td>
                <td class="description-td">ACGCGTGGAAACGGTAAGCCTGCAGCAGCAGGATTTTGTAGATGTTGGTGCAAACGGTCTG</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">CAGGGAGTTTACCTGCAGGTCCAGGAACA</td>
              </tr> <tr class="description-tr"> <td class="description-td">34T</td>
                <td class="description-td">TGCCACTCCCTGTTCCTGGACCTGCAGGTAAACTCCCTGCAGACCGTTTGCACCAACATCTA</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">CAAAATCCTGCTGCTGCAGGCTTACCGT</td>
              </tr> <tr class="description-tr"> <td class="description-td">35B</td>
                <td class="description-td">AGTAGCACAGGGAAGCGGTGTCGGAGATAACACGCAGGAAGAAGGTCGGGTTTTTCCAAA</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">CCTGCTGGTGGAACGGCAGCTGCAGAACGC</td>
              </tr> <tr class="description-tr"> <td class="description-td">35T</td>
                <td class="description-td">TTCCACGCGTGCGTTCTGCAGCTGCCGTTCCACCAGCAGGTTTGGAAAAACCCGACCTTCTT</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">CCTGCGTGTTATCTCCGACACCGCTTCC</td>
              </tr> <tr class="description-tr"> <td class="description-td">36B</td>
                <td class="description-td">GGCACAGCCACTGAACAGCTTCGGACGGCAGCGGACCAGCAGCACCTTTAGCACCCAGGGA</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">CATACCAGCGTTTTTAGCTTTCAGGATGG</td>
              </tr> <tr class="description-tr"> <td class="description-td">36T</td>
                <td class="description-td">CTGTGCTACTCCATCCTGAAAGCTAAAAACGCTGGTATGTCCCTGGGTGCTAAAGGTGCTGC</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">TGGTCCGCTGCCGTCCGAAGCTGTTCAG</td>
              </tr> <tr class="description-tr"> <td class="description-td">37B</td>
                <td class="description-td">ACAGCTGGGTCTGAGCGGTACGCAGGGAACCCAGCAGCGGAACGTAGGTAACACGGTGAC</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">GGGTCAGTTTCAGCAGGAAAGCCTGGT</td>
              </tr> <tr class="description-tr"> <td class="description-td">37T</td>
                <td class="description-td">TGGCTGTGCCACCAGGCTTTCCTGCTGAAACTGACCCGTCACCGTGTTACCTACGTTCCGCT</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">GCTGGGTTCCCTGCGTACCGCTCAG</td>
              </tr> <tr class="description-tr"> <td class="description-td">38B</td>
                <td class="description-td">ACGGCAGAGCCGGGTTAGCAGCAGCTTCCAGAGCGGTCAGGGTGGTACCCGGCAGTTTACG</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">GG</td>
              </tr> <tr class="description-tr"> <td class="description-td">38T</td>
                <td class="description-td">ACCCAGCTGTCCCGTAAACTGCCGGGTACCACCCTGACCGCTCTGGAAGCTGCTGCTAACCC</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">GG</td>
              </tr> <tr class="description-tr"> <td class="description-td">39B</td>
                <td class="description-td">GCGTGCCTCGAGGAATTCGGATCCATTAGTCCAGGATGGTTTTGAAGTCG</td>
              </tr> <tr class="description-tr"> <td class="description-td">39T</td>
                <td class="description-td">CTCTGCCGTCCGACTTCAAAACCATCCTGGACTAATGGATCCGAATTCCTCGAGGCACGC</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="2" align="center" rowsep="1" class="description-td" colspan="2"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="322pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Table 10B (SEQ ID NO:721)</td>
              </tr> <tr class="description-tr"> <td class="description-td">GCACGCGGGAGCTCTAGAGTCGACCATATGCCGCGTGCTCCGCGTTGCCGTGCTGTTCGTT</td>
              </tr> <tr class="description-tr"> <td class="description-td">CCCTGCTGCGTTCCCACTATCGCGAAGTTCTGCCGCTGGCTACCTTCGTTCGTCGTCTGGG</td>
              </tr> <tr class="description-tr"> <td class="description-td">CCCGCAGGGTTGGCGTCTGGTTCAGCGTGGTGACCCGGCTGCTTTCCGTGCTCTGGTTGCT</td>
              </tr> <tr class="description-tr"> <td class="description-td">CAGTGCCTGGTTTGCGTTCCGTGGGACGCTCGTCCGCCGCCGGCTGCTCCGTCCTTCCGTC</td>
              </tr> <tr class="description-tr"> <td class="description-td">AGGTTTCCTGCCTGAAAGAACTGGTTGCTCGTGTTCTGCAGCGTCTGTGCGAACGTGGTGC</td>
              </tr> <tr class="description-tr"> <td class="description-td">TAAAAACGTTCTGGCTTTCGGTTTCGCTCTGCTGGACGGTGCTCGTGGTGGTCCGCCGGAA</td>
              </tr> <tr class="description-tr"> <td class="description-td">GCATTCACCACCTCCGTTCGTTCCTACCTGCCGAACACCGTTACCGACGCTCTGCGTGGTT</td>
              </tr> <tr class="description-tr"> <td class="description-td">CCGGTGCTTGGGGTCTGCTGCTGCGTCGTGTTGGTGACGACGTTCTGGTTCACCTGCTGGC</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCGTTGCGCTCTGTTCGTTCTGGTTGCTCCGTCCTGCGCTTACCAGGTTTGTGGTCCGCCG</td>
              </tr> <tr class="description-tr"> <td class="description-td">CTGTACCAGCTGGGTGCTGCTACCCAGGCTCGTCCGCCGCCGCACGCTTCCGGTCCGCGTC</td>
              </tr> <tr class="description-tr"> <td class="description-td">GTCGTCTGGGTTGCGAACGTGCTTGGAACCACTCCGTTCGTGAAGCTGGTGTTCCGCTGGG</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCTGCCGGCTCCGGGTGCTCGTCGTCGTGGTGGTTCCGCTTCCCGTTCCCTGCCGCTGCCG</td>
              </tr> <tr class="description-tr"> <td class="description-td">AAACGTCCGCGTCGTGGTGCTGCTCCGGAACCGGAACGTACCCCGGTTGGTCAGGGTTCCT</td>
              </tr> <tr class="description-tr"> <td class="description-td">GGGCTCACCCGGGTCGTACCCGTGGTCCGTCCGACCGTGGTTTCTGCGTTGTTTCCCCGGC</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCGTCCGGCTGAAGAAGCTACCTCCCTGGAAGGTGCTCTGTCCGGCACCCGTCACTCCCAC</td>
              </tr> <tr class="description-tr"> <td class="description-td">CCGTCCGTTGGTCGTCAGCACCACGCTGGTCCGCCGTCCACCTCCCGTCCGCCGCGTCCGT</td>
              </tr> <tr class="description-tr"> <td class="description-td">GGGACACCCCGTGCCCGCCGGTTTACGCTGAAACCAAACACTTCCTGTACTCCTCCGGTGA</td>
              </tr> <tr class="description-tr"> <td class="description-td">CAAAGAACAGCTGCGTCCGTCCTTCCTGCTGTCCTCCCTGCGTCCGTCCCTGACCGGTGCT</td>
              </tr> <tr class="description-tr"> <td class="description-td">CGTCGTCTGGTTGAAACTATCTTCCTGGGTTCCCGTCCGTGGATGCCGGGCACCCCGCGTC</td>
              </tr> <tr class="description-tr"> <td class="description-td">GTCTGCCGCGTCTGCCGCAGCGTTACTGGCAGATGCGTCCGCTGTTCCTGGAACTGCTGGG</td>
              </tr> <tr class="description-tr"> <td class="description-td">CAACCACGCTCAGTGCCCGTATGGTGTTCTGCTGAAAACCCACTGCCCGCTGCGTGCTGCT</td>
              </tr> <tr class="description-tr"> <td class="description-td">GTTACCCCGGCTGCTGGTGTTTGCGCTCGTGAAAAACCGCAGGGTTCCGTTGCTGCTCCGG</td>
              </tr> <tr class="description-tr"> <td class="description-td">AAGAAGAAGATACCGACCCGCGTCGTCTGGTTCAGCTGCTGCGTCAGCACTCCTCCCCGTG</td>
              </tr> <tr class="description-tr"> <td class="description-td">GCAGGTTTACGGTTTCGTTCGTGCATGCCTGCGTCGTCTGGTTCCGCCGGGTCTGTGGGGT</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCCCGTCACAACGAACGTCGTTTCCTGCGTAACACCAAAAAATTCATCTCCCTGGGTAAAC</td>
              </tr> <tr class="description-tr"> <td class="description-td">ACGCTAAACTGTCCCTGCAGGAACTGACCTGGAAAATGTCCGTTCGTGACTGCGCTTGGCT</td>
              </tr> <tr class="description-tr"> <td class="description-td">GCGTCGTTCTCCGGGTGTTGGTTGCGTTCCGGCTGCTGAACACCGTCTGCGTGAAGAAATC</td>
              </tr> <tr class="description-tr"> <td class="description-td">CTGGCTAAATTCCTGCACTGGCTGATGTCCGTATACGTTGTTGAACTGCTGCGTTCCTTCT</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCTACGTTACCGAAACCACCTTCCAGAAAAACCGTCTGTTCTTCTACCGTAAATCCGTTTG</td>
              </tr> <tr class="description-tr"> <td class="description-td">GTCCAAACTGCAGTCCATCGGTATCCGTCAGCACCTGAAACGTGTTCAGCTGCGTGAACTG</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCCGAAGCTGAAGTTCGTCAGCACCGTGAAGCTCGTCCGGCTCTGCTGACCTCCCGTCTGC</td>
              </tr> <tr class="description-tr"> <td class="description-td">GTTTCATCCCGAAACCGGACGGTCTGCGTCCGATCGTAAACATGGACTACGTTGTTGGTGC</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCGTACCTTCCGTCGTGAAAAACGTGCTGAGCGTCTGACCTCCCGTGTTAAAGCTCTGTTC</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCCGTTCTGAACTACGAACGTGCTCGTCGTCCGGGTCTGCTGGGTGCTTCCGTTCTGGGTC</td>
              </tr> <tr class="description-tr"> <td class="description-td">TGGACGACATCCACCGTGCTTGGCGTACCTTCGTTCTGCGTGTTCGTGCTCAGGACCCGCC</td>
              </tr> <tr class="description-tr"> <td class="description-td">GCCGGAACTGTACTTCGTTAAAGTTGACGTTACCGGCGCGTACGACACCATCCCGCAGGAC</td>
              </tr> <tr class="description-tr"> <td class="description-td">CGTCTGACCGAAGTTATCGCTTCCATCATCAAACCGCAGAACACCTACTGCGTTCGTCGTT</td>
              </tr> <tr class="description-tr"> <td class="description-td">ACGCTGTTGTTCAGAAAGCTGCTCACGGTCACGTTCGTAAAGCATTCAAATCCCACGTTTC</td>
              </tr> <tr class="description-tr"> <td class="description-td">CACCCTGACCGACCTGCAGCCGTACATGCGTCAGTTCGTTGCTCACCTGCAGGAAACCTCC</td>
              </tr> <tr class="description-tr"> <td class="description-td">CCGCTGCGTGACGCTGTTGTTATCGAACAGTCCTCCTCCCTGAACGAAGCTTCCTCCGGTC</td>
              </tr> <tr class="description-tr"> <td class="description-td">TGTTCGACGTTTTCCTGCGTTTCATGTGCCACCACGCTGTTCGTATCCGTGGTAAATCCTA</td>
              </tr> <tr class="description-tr"> <td class="description-td">CGTTCAGTGCCAGGGTATCCCGCAGGGTTCCATCCTGTCCACCCTGCTGTGCTCCCTGTGC</td>
              </tr> <tr class="description-tr"> <td class="description-td">TACGGTGACATGGAAAACAAACTGTTCGCTGGTATCCGTCGTGACGGTCTGCTGCTGCGTC</td>
              </tr> <tr class="description-tr"> <td class="description-td">TGGTTGACGACTTCCTGCTGGTTACTCCGCACCTGACCCACGCTAAAACCTTCCTGCGTAC</td>
              </tr> <tr class="description-tr"> <td class="description-td">CCTGGTTCGTGGTGTTCCGGAATACGGTTGCGTTGTAAACCTGCGTAAAACCGTTGTTAAC</td>
              </tr> <tr class="description-tr"> <td class="description-td">TTCCCGGTTGAAGACGAAGCTCTGGGTGGCACCGCTTTCGTTCAGATGCCGGCTCACGGTC</td>
              </tr> <tr class="description-tr"> <td class="description-td">TGTTCCCGTGGTGCGGTCTGCTGCTGGACACCCGTACCCTGGAAGTTCAGTCCGACTACTC</td>
              </tr> <tr class="description-tr"> <td class="description-td">CTCCTACGCTCGTACCTCCATCCGTGCTTCCCTGACCTTCAACCGTGGTTTCAAAGCTGGT</td>
              </tr> <tr class="description-tr"> <td class="description-td">CGTAACATGCGTCGTAAACTGTTCGGTGTTCTGCGTCTGAAATGCCACTCCCTGTTCCTGG</td>
              </tr> <tr class="description-tr"> <td class="description-td">ACCTGCAGGTAAACTCCCTGCAGACCGTTTGCACCAACATCTACAAAATCCTGCTGCTGCA</td>
              </tr> <tr class="description-tr"> <td class="description-td">GGCTTACCGTTTCCACGCGTGCGTTCTGCAGCTGCCGTTCCACCAGCAGGTTTGGAAAAAC</td>
              </tr> <tr class="description-tr"> <td class="description-td">CCGACCTTCTTCCTGCGTGTTATCTCCGACACCGCTTCCCTGTGCTACTCCATCCTGAAAG</td>
              </tr> <tr class="description-tr"> <td class="description-td">CTAAAAACGCTGGTATGTCCCTGGGTGCTAAAGGTGCTGCTGGTCCGCTGCCGTCCGAAGC</td>
              </tr> <tr class="description-tr"> <td class="description-td">TGTTCAGTGGCTGTGCCACCAGGCTTTCCTGCTGAAACTGACCCGTCACCGTGTTACCTAC</td>
              </tr> <tr class="description-tr"> <td class="description-td">GTTCCGCTGCTGGGTTCCCTGCGTACCGCTCAGACCCAGCTGTCCCGTAAACTGCCGGGTA</td>
              </tr> <tr class="description-tr"> <td class="description-td">CCACCCTGACCGCTCTGGAAGCTGCTGCTAACCCGGCTCTGCCGTCCGACTTCAAAACCAT</td>
              </tr> <tr class="description-tr"> <td class="description-td">CCTGGACTAATGGATCCGAATTCCTCGAGGCACGC</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p>The present invention also provides transgenic animals (i.e., mammals transgenic for a human or other TRT gene sequence) expressing an hTRT or other TRT polynucleotide or polypeptide. In one embodiment, hTRT is secreted into the milk of a transgenic mammal such as a transgenic bovine, goat, or rabbit. Methods for production of such animals are found, e.g., in Heyneker et al., PCT WO 91/08216.</p>
    <p>The hTRT proteins and complexes of the invention, including those made using the expression systems disclosed herein supra, may be purified using a variety of general methods known in the art in accordance with the specific methods provided by the present invention (e.g., infra). One of skill in the art will recognize that after chemical synthesis, biological expression, or purification, the hTRT protein may possess a conformation different than a native conformation of naturally occurring telomerase. In some instances, it may be helpful or even necessary to denature (e.g., including reduction of disulfide or other linkages) the polypeptide and then to cause the polypeptide to re-fold into the preferred conformation. Productive refolding may also require the presence of hTR (or hTR fragments). Methods of reducing and denaturing proteins and inducing re-folding are well known to those of skill in the art (see, e.g., Debinski et al., 1993, <i>J. Biol. Chem., </i>268:14065; Kreitman and Pastan, 1993, <i>Bioconjug. Chem., </i>4:581; and Buchner et al., 1992, <i>Anal. Biochem., </i>205:263; and McCaman et al., 1985, <i>J. Biotech. </i>2:177). See also PCT Publication WO 96/40868, supra.</p>
    <p>D) Complexes of Human TRT and Human Telomerase RNA, Telomerase-Associated Proteins, and Other Biomolecules Produced by Coexpression and Other Means</p>
    <p>hTRT polypeptides of the invention can associate in vivo and in vitro with other biomolecules, including RNAs (e.g., hTR), proteins (e.g., telomerase-associated proteins), DNA (e.g., telomeric DNA, [T<sub>2</sub>AG<sub>3</sub>]<sub>N</sub>), and nucleotides, such as (deoxy)ribonucleotide triphosphates. These associations can be exploited to assay hTRT presence or function, to identify or purify hTRT or telomerase-associated molecules, and to analyze hTRT or telomerase structure or function in accordance with the methods of the present invention.</p>
    <p>In one embodiment, the present invention provides hTRT complexed with (e.g., associated with or bound to) a nucleic acid, usually an RNA, for example to produce a telomerase holoenzyme. In one embodiment, the bound RNA is capable of acting as a template for telomerase-mediated DNA synthesis. Examples of RNAs that may be complexed with the hTRT polypeptide include a naturally occurring host cell telomerase RNA, a human telomerase RNA (e.g., hTR; U.S. Pat. No. 5,583,016), an hTR subsequence or domain, a synthetic RNA, or other RNAs. The RNA-hTRT protein complex (an RNP) typically exhibits one or more telomerase activities, such as telomerase catalytic activities. These hTRT-hTR RNPs (or other hTRT-RNA complexes) can be produced by a variety of methods, as described infra for illustrative purposes, including in vitro reconstitution, by co-expression of hTRT and hTR (or other RNA) in vitro (i.e., in a cell free system), in vivo reconstitution, or ex vivo reconstitution.</p>
    <p>Thus, the present invention provides, in one embodiment, an hTRT-hTR complex (or other hTRT-RNA complex) formed in vitro by mixing separately purified components (“in vitro reconstitution;” see, e.g., U.S. Pat. No. 5,583,016 for a description of reconstitution; also see Autexier et al., <i>EMBO J. </i>15:5928). In one embodiment the hTRT protein is produced by recombinant expression in human or non-human cells, e.g., as described supra, and subsequently purified using protein purification methods (e.g., chromatography, affinity purification). In a particular embodiment, the recombinant hTRT protein is purified to homogeneity. The purified hTRT protein is combined with separately purified hTR, which may be produced using an in vitro transcription system, by chemical synthesis, or by other methods and purified using standard RNA purification techniques (see Melton et al., 1984, <i>Nucl. Acids Res. </i>12:7035; Studier et al., 1986, <i>J. Mol. Biol. </i>189:113).</p>
    <p>In an alternative embodiment, the invention provides telomerase RNPs produced by coexpression of the hTRT polypeptide and an RNA (e.g., hTR) in vitro in a cell-free transcription-translation system (e.g. wheat germ or rabbit reticulocyte lysate). As shown in Example 7, in vitro co-expression of a recombinant hTRT polypeptide and hTR results in production of telomerase catalytic activity (as measured by a TRAP assay).</p>
    <p>Further provided by the present invention are telomerase RNPs produced by expression of the hTRT polypeptide in a cell, e.g., a mammalian cell, in which hTR is naturally expressed or in which hTR (or another RNA capable of forming a complex with the hTRT protein) is introduced or expressed by recombinant means. Thus, in one embodiment, hTRT is expressed in a telomerase negative human cell in which hTR is present (e.g., BJ or IMP90 cells), allowing the two molecules to assemble into an RNP. In another embodiment, hTRT is expressed in a human or non-human cell in which hTR is recombinantly expressed. Methods for expression of hTR in a cell are found in U.S. Pat. No. 5,583,016. Further, a clone containing a cDNA encoding the RNA component of telomerase has been placed on deposit as pGRN33 (ATCC 75926). Genomic sequences encoding the RNA component of human telomerase are also on deposit in the ˜15 kb SauIIIA1 to HindIII insert of lambda clone 28-1 (ATCC 75925). For expression in eukaryotic cells the hTRT sequence will typically be operably linked to a transcription initiation sequence (RNA polymerase binding site) and transcription terminator sequences (see, e.g., PCT Publication WO 96/01835; Feng et al., 1995, <i>Science </i>269:1236).</p>
    <p>The present invention further provides recombinantly produced or substantially purified hTRT polypeptides coexpressed and/or associated with so-called “telomerase-associated proteins.” Thus, the present invention provides hTRT coexpressed with, or complexed with, other proteins (e.g., telomerase-associated proteins). Telomerase-associated proteins are those proteins that copurify with human telomerase and/or that may play a role in modulating telomerase function or activity, for example by participating in the association of telomerase with telomeric DNA. Examples of telomerase-associated proteins include (but are not limited to) the following proteins and/or their human homologs: nucleolin (see, Srivastava et al., 1989, <i>FEBS Letts. </i>250:99); EF2H (elongation factor 2 homolog; see Nomura et al. 1994, <i>DNA Res. </i>(<i>Japan</i>) 1:27, GENBANK accession #D21163); TP1/TLP1 (Harrington et al., 1997, <i>Science </i>275:973; Nakayama, 1997, <i>Cell </i>88:875); the human homologue of the Tetrahymena p95 or p95 itself (Collins et al., 1995, <i>Cell </i>81:677); TPC2 (a telomere length regulatory protein; ATCC accession number 97708; TPC3 (also a telomere length regulatory protein; ATCC accession number 97707; DNA-binding protein B (dbpB; Horwitz et al., 1994, <i>J. Biol. Chem. </i>269:14130; and Telomere Repeat Binding Factors (TRF 1 &amp; 2; Chang et al., 1995, <i>Science </i>270:1663; Chong et al., 1997, <i>Hum Mol Genet </i>6:69); EST1, 3 and 4 (Lendvay et al., 1996, <i>Genetics </i>144:1399, Nugent et al., 1996, <i>Science </i>274:249, Lundblad et al., 1989, <i>Cell </i>57:633); and End-capping factor (Cardenas et al., 1993, <i>Genes Dev. </i>7:883).</p>
    <p>Telomerase associated proteins can be identified on the basis of co-purification with, or binding to, hTRT protein or the hTRT-hTR RNP. Alternatively, they can be identified on the basis of binding to an hTRT fusion protein, e.g., a GST-hTRT fusion protein or the like, as determined by affinity purification (see, Ausubel et al. Ch 20). A particularly useful technique for assessing protein-protein interactions, which is applicable to identifying hTRT-associated proteins, is the two hybrid screen method of Chien et al. (<i>Proc. Natl. Acad. Sci. USA </i>88:9578 [1991]; see also Ausubel et al., supra, at Ch. 20). This screen identifies protein-protein interactions in vivo through reconstitution of a transcriptional activator, the yeast Gal4 transcription protein (see, Fields and Song, 1989, <i>Nature </i>340:245). The method is based on the properties of the yeast Gal4 protein, which consists of separable domains responsible for DNA-binding and transcriptional activation. Polynucleotides, usually expression vectors, encoding two hybrid proteins are constructed. One polynucleotide comprises the yeast Gal4 DNA-binding domain fused to a polypeptide sequence of a protein to be tested for an hTRT interaction (e.g., nucleolin or EF2H). Alternatively the yeast Gal4 DNA-binding domain is fused to cDNAs from a human cell, thus creating a library of human proteins fused to the Gal4 DNA binding domain for screening for telomerase associated proteins. The other polynucleotide comprises the Gal4 activation domain fused to an hTRT polypeptide sequence. The constructs are introduced into a yeast host cell. Upon expression, intermolecular binding between hTRT and the test protein can reconstitute the Gal4 DNA-binding domain with the Gal4 activation domain. This leads to the transcriptional activation of a reporter gene (e.g., lacZ, HIS3) operably linked to a Gal4 binding site. By selecting for, or by assaying the reporter, gene colonies of cells that contain an hTRT interacting protein or telomerase associated protein can be identified. Those of skill will appreciate that there are numerous variations of the 2-hybrid screen, e.g., the LexA system (Bartel et al, 1993, in Cellular Interactions in Development: A Practical Approach Ed. Hartley, D. A. (Oxford Univ. Press) pp. 153-79).</p>
    <p>Another useful method for identifying telomerase-associated proteins is a three-hybrid system (see, e.g., Zhang et al., 1996, <i>Anal. Biochem. </i>242:68; Licitra et al., 1996, <i>Proc. Natl. Acad. Sci. USA </i>93:12817). The telomerase RNA component can be utilized in this system with the TRT or hTRT protein and a test protein. Another useful method for identifying interacting proteins, particularly (i.e., proteins that heterodimerize or form higher order heteromultimers), is the <i>E. coli</i>/BCCP interactive screening system (see, Germino et al. (1993) <i>Proc. Natl. Acad. Sci. U.S.A. </i>90:933; Guarente (1993) <i>Proc. Natl. Acad. Sci. </i>(<i>U.S.A.</i>) 90:1639).</p>
    <p>The present invention also provides complexes of telomere binding proteins (which may or may not be telomerase associated proteins) and hTRT (which may or may not be complexed with hTR, other RNAs, or one or more telomerase associated proteins). Examples of telomere binding proteins include TRF1 and TRF2 (supra); rnpA1, rnpA2, RAP1 (Buchman et al., 1988, <i>Mol. Cell. Biol. </i>8:210, Buchman et al., 1988, <i>Mol. Cell. Biol. </i>8:5086), SIR3 and SIR4 (Aparicio et al, 1991, Cell 66:1279), TEL1 (Greenwell et al., 1995, <i>Cell </i>82:823; Morrow et al., 1995, <i>Cell </i>82:831); ATM (Savitsky et al., 1995, Science 268:1749), end-capping factor (Cardenas et al., 1993, <i>Genes Dev. </i>7:883), and corresponding human homologs. The aforementioned complexes may be produced generally as described supra for complexes of hTRT and hTR or telomerase associated proteins, e.g., by mixing or co-expression in vitro or in vivo.</p>
    <heading>V. Antibodies and Other Binding Agents</heading> <p>In a related aspect, the present invention provides antibodies that are specifically immunoreactive with hTRT, including polyclonal and monoclonal antibodies, antibody fragments, single chain antibodies, human and chimeric antibodies, including antibodies or antibody fragments fused to phage coat or cell surface proteins, and others known in the art and described herein. The antibodies of the invention can specifically recognize and bind polypeptides that have an amino acid sequence that is substantially identical to the amino acid sequence set forth in FIG. 17 SEQ ID NO:2, or an immunogenic fragment thereof or epitope on the protein defined thereby. The antibodies of the invention can exhibit a specific binding affinity for hTRT of at least about 10<sup>7</sup>, 10<sup>8</sup>, 10<sup>9</sup>, or 10<sup>10 </sup>M<sup>−1</sup>, and may be polyclonal, monoclonal, recombinant or otherwise produced. The invention also provides anti-hTRT antibodies that recognize an hTRT conformational epitope (e.g., an epitope on the surface of the hTRT protein or a telomerase RNP). Likely conformational epitopes can be identified, if desired, by computer-assisted analysis of the hTRT protein sequence, comparison to the conformation of related reverse transcriptases, such as the p66 subunit of HIV-1 (see, e.g., FIG. <b>3</b>), or empirically. Anti-hTRT antibodies that recognize conformational epitopes have utility, inter alia, in detection and purification of human telomerase and in the diagnosis and treatment of human disease.</p>
    <p>For the production of anti-hTRT antibodies, hosts such as goats, sheep, cows, guinea pigs, rabbits, rats, or mice, may be immunized by injection with hTRT protein or any portion, fragment or oligopeptide thereof which retains immunogenic properties. In selecting hTRT polypeptides for antibody induction, one need not retain biological activity; however, the protein fragment, or oligopeptide must be immunogenic, and preferably antigenic. Immunogenicity can be determined by injecting a polypeptide and adjuvant into an animal (e.g., a rabbit) and assaying for the appearance of antibodies directed against the injected polypeptide (see, e.g., Harlow and Lane, Antibodies: A Laboratory Manual, Cold Spring Harbor Laboratory, New York (1988), which is incorporated in its entirety and for all purposes, e.g., at Chapter 5). Peptides used to induce specific antibodies typically have an amino acid sequence consisting of at least five amino acids, preferably at least 8 amino acids, more preferably at least 10 amino acids. Usually they will mimic or have substantial sequence identity to all or a contiguous portion of the amino acid sequence of the protein of SEQ ID NO:2. Short stretches of hTRT protein amino acids may be fused with those of another protein, such as keyhole limpet hemocyanin, and an anti-hTRT antibody produced against the chimeric molecule. Depending on the host species, various adjuvants may be used to increase immunological response.</p>
    <p>The antigen is presented to the immune system in a fashion determined by methods appropriate for the animal. These and other parameters are generally well known to immunologists. Typically, injections are given in the footpads, intramuscularly, intradermally, perilymph nodally or intraperitoneally. The immunoglobulins produced by the host can be precipitated, isolated and purified by routine methods, including affinity purification.</p>
    <p>Illustrative examples of immunogenic hTRT peptides include are provided in Example 8. In addition, Example 8 describes the production and use of anti-hTRT polyclonal antibodies.</p>
    <p>A) Monoclonal Antibodies</p>
    <p>Monoclonal antibodies to hTRT proteins and peptides may be prepared in accordance with the methods of the invention using any technique which provides for the production of antibody molecules by continuous cell lines in culture. These include, but are not limited to, the hybridoma technique originally described by Koehler and Milstein (<i>Nature </i>256:495 [1975]), the human B-cell hybridoma technique (Kosbor et al., 1983, <i>Immunol. Today </i>4:72; Cote et al., 1983, <i>Proc. Natl. Acad. Sci. USA, </i>80:2026), and the EBV-hybridoma technique (Cole et al., Monoclonal Antibodies and Cancer Therapy, Alan R Liss Inc, New York N.Y., pp 77-96 [1985]).</p>
    <p>In one embodiment, appropriate animals are selected and the appropriate immunization protocol followed. The production of non-human monoclonal antibodies, e.g., murine, lagomorpha, equine, is well known and can be accomplished by, for example, immunizing an animal with a preparation containing hTRT or fragments thereof. In one method, after the appropriate period of time, the spleens of the animals are excised and individual spleen cells are fused, typically, to immortalized myeloma cells under appropriate selection conditions. Thereafter, the cells are clonally separated and the supernatants of each clone (e.g., hybridoma) are tested for the production of an appropriate antibody specific for the desired region of the antigen. Techniques for producing antibodies are well known in the art. See, e.g., Goding et al., Monoclonal Antibodies: Principles and Practice (2d ed.) Acad. Press, N.Y., and Harlow and Lane, supra, each of which is incorporated in its entirety and for all purposes. Other suitable techniques involve the in vitro exposure of lymphocytes to the antigenic polypeptides or alternatively, to selection of libraries of antibodies in phage or similar vectors (see, infra).</p>
    <p>B) Human Antibodies</p>
    <p>In another aspect of the invention, human antibodies against an hTRT polypeptide are provided. Human monoclonal antibodies against a known antigen can also be made using transgenic animals having elements of a human immune system (see, e.g., U.S. Pat. Nos. 5,569,825 and 5,545,806, both of which are incorporated by reference in their entirety for all purposes) or using human peripheral blood cells (Casali et al., 1986, <i>Science </i>234:476). Some human antibodies are selected by competitive binding experiments, or otherwise, to have the same epitope specificity as a particular mouse antibody.</p>
    <p>In an alternative embodiment, human antibodies to an hTRT polypeptide can be produced by screening a DNA library from human B cells according to the general protocol outlined by Huse et al., 1989, <i>Science </i>246:1275, which is incorporated by reference. Antibodies binding to the hTRT polypeptide are selected. Sequences encoding such antibodies (or binding fragments) are then cloned and amplified. The protocol described by Huse is often used with phage-display technology.</p>
    <p>C) Humanized or Chimeric Antibodies</p>
    <p>The invention also provides anti-hTRT antibodies that are made chimeric, human-like or humanized, to reduce their potential antigenicity, without reducing their affinity for their target. Preparation of chimeric, human-like and humanized antibodies have been described in the art (see, e.g., U.S. Pat. Nos. 5,585,089 and 5,530,101; Queen, et al., 1989, <i>Proc. Nat'l Acad. Sci. USA </i>86:10029; and Verhoeyan et al., 1988, <i>Science </i>239:1534; each of which is incorporated by reference in their entirety and for all purposes). Humanized immunoglobulins have variable framework regions substantially from a human immunoglobulin (termed an acceptor immunoglobulin) and complementarity determining regions substantially from a non-human (e.g., mouse) immunoglobulin (referred to as the donor immunoglobulin). The constant region(s), if present, are also substantially from a human immunoglobulin.</p>
    <p>In some applications, such as administration to human patients, the humanized (as well as human) anti-hTRT antibodies of the present invention offer several advantages over antibodies from murine or other species: (1) the human immune system should not recognize the framework or constant region of the humanized antibody as foreign, and therefore the antibody response against such an injected antibody should be less than against a totally foreign mouse antibody or a partially foreign chimeric antibody; (2) because the effector portion of the humanized antibody is human, it may interact better with other parts of the human immune system; and (3) injected humanized antibodies have a half-life essentially equivalent to naturally occurring human antibodies, allowing smaller and less frequent doses than antibodies of other species. As implicit from the foregoing, anti hTRT antibodies have application in the treatment of disease, i.e., to target telomerase-positive cells.</p>
    <p>D) Phage Display</p>
    <p>The present invention also provides anti-hTRT antibodies (or binding compositions) produced by phage display methods (see, e.g., Dower et al., WO 91/17271 and McCafferty et al., WO 92/01047; and Vaughan et al., 1996, <i>Nature Biotechnology, </i>14: 309; each of which is incorporated by reference in its entirety for all purposes). In these methods, libraries of phage are produced in which members display different antibodies on their outer surfaces. Antibodies are usually displayed as Fv or Fab fragments. Phage displaying antibodies with a desired specificity are selected by affinity enrichment to an hTRT polypeptide.</p>
    <p>In a variation of the phage-display method, humanized antibodies having the binding specificity of a selected murine antibody can be produced. In this method, either the heavy or light chain variable region of the selected murine antibody is used as a starting material. If, for example, a light chain variable region is selected as the starting material, a phage library is constructed in which members display the same light chain variable region (i.e., the murine starting material) and a different heavy chain variable region. The heavy chain variable regions are obtained from a library of rearranged human heavy chain variable regions. A phage showing strong specific binding for the hTRT polypeptide (e.g., at least 10<sup>8 </sup>and preferably at least 10<sup>9 </sup>M<sup>−1</sup>) is selected. The human heavy chain variable region from this phage then serves as a starting material for constructing a further phage library. In this library, each phage displays the same heavy chain variable region (i.e., the region identified from the first display library) and a different light chain variable region. The light chain variable regions are obtained from a library of rearranged human variable light chain regions. Again, phage showing strong specific binding are selected. These phage display the variable regions of completely human anti-hTRT antibodies. These antibodies usually have the same or similar epitope specificity as the murine starting material.</p>
    <p>E) Hybrid Antibodies</p>
    <p>The invention also provides hybrid antibodies that share the specificity of antibodies against an hTRT polypeptide but are also capable of specific binding to a second moiety. In such hybrid antibodies, one heavy and light chain pair is usually from an anti-hTRT antibody and the other pair from an antibody raised against another epitope or protein. This results in the property of multi-functional valency, i.e., ability to bind at least two different epitopes simultaneously, where at least one epitope is the epitope to which the anti-complex antibody binds. Such hybrids can be formed by fusion of hybridomas producing the respective component antibodies, or by recombinant techniques. Such hybrids can be used to carry a compound (i.e., drug) to a telomerase-positive cell (i.e., a cytotoxic agent is delivered to a cancer cell).</p>
    <p>Immunoglobulins of the present invention can also be fused to functional regions from other genes (e.g., enzymes) to produce fusion proteins (e.g., immunotoxins) having useful properties.</p>
    <p>F) Anti-Idiotypic Antibodies</p>
    <p>Also useful are anti-idiotype antibodies which can be isolated by the above procedures. Anti-idiotypic antibodies may be prepared by, for example, immunization of an animal with the primary antibody (i.e., anti-hTRT antibodies or hTRT-binding fragments thereof). For anti-hTRT antibodies, anti-idiotype antibodies whose binding to the primary antibody is inhibited by an hTRT polypeptide or fragments thereof are selected. Because both the anti-idiotypic antibody and the hTRT polypeptide or fragments thereof bind the primary immunoglobulin, the anti-idiotypic immunoglobulin can represent the “internal image” of an epitope and thus can substitute for the hTRT polypeptide in assays or can be used to bind (i.e., inactivate) anti-hTRT antibodies, e.g., in a patient. Anti-idiotype antibodies can also interact with telomerase associated proteins. Administration of such antibodies can affect telomerase function by titrating out or competing with hTRT in binding to hTRT-associated proteins.</p>
    <p>G) General</p>
    <p>The antibodies of the invention may be of any isotype, e.g., IgM, IgD, IgG, IgA, and IgE, with IgG, IgA and IgM often preferred. Humanized antibodies may comprise sequences from more than one class or isotype.</p>
    <p>In another embodiment of the invention, fragments of the intact antibodies described above are provided. Typically, these fragments can compete with the intact antibody from which they were derived for specific binding to the hTRT polypeptide, and bind with an affinity of at least 10<sup>7</sup>, 10<sup>8</sup>, 10<sup>9 </sup>M<sup>−1</sup>, or 10<sup>10 </sup>M<sup>−1</sup>. Antibody fragments include separate heavy chains, light chains, Fab, Fab′ F(ab′)<sub>2</sub>, Fabc, and Fv. Fragments can be produced by enzymatic or chemical separation of intact immunoglobulins. For example, a F(ab′)<sub>2 </sub>fragment can be obtained from an IgG molecule by proteolytic digestion with pepsin at pH 3.0-3.5 using standard methods such as those described in Harlow and Lane, supra. Fab fragments may be obtained from F(ab′)<sub>2 </sub>fragments by limited reduction, or from whole antibody by digestion with papain in the presence of reducing agents (see generally, Paul, W., ed Fundamental Immunology 2nd Raven Press, N.Y., 1989, Ch. 7, incorporated by reference in its entirety for all purposes). Fragments can also be produced by recombinant DNA techniques. Segments of nucleic acids encoding selected fragments are produced by digestion of full-length coding sequences with restriction enzymes, or by de novo synthesis. Often fragments are expressed in the form of phage-coat fusion proteins.</p>
    <p>Many of the immunoglobulins described above can undergo non-critical amino-acid substitutions, additions or deletions in both the variable and constant regions without loss of binding specificity or effector functions, or intolerable reduction of binding affinity (i.e., below about 10<sup>7 </sup>M<sup>−1</sup>). Usually, immunoglobulins incorporating such alterations exhibit substantial sequence identity to a reference immunoglobulin from which they were derived. A mutated immunoglobulin can be selected having the same specificity and increased affinity compared with a reference immunoglobulin from which it was derived. Phage-display technology offers useful techniques for selecting such immunoglobulins. See, e.g., Dower et al., WO 91/17271 McCafferty et al., WO 92/01047; and Huse, WO 92/06204.</p>
    <p>The antibodies of the present invention can be used with or without modification. Frequently, the antibodies will be labeled by joining, either covalently or non-covalently, a detectable label. As labeled binding entities, the antibodies of the invention are particularly useful in diagnostic applications.</p>
    <p>The anti-hTRT antibodies of the invention can be purified using well known methods. The whole antibodies, their dimers, individual light and heavy chains, or other immunoglobulin forms of the present invention can be purified using the methods and reagents of the present invention in accordance with standard procedures of the art, including ammonium sulfate precipitation, affinity columns, column chromatography, gel electrophoresis and the like (see generally Scopes, Protein Purification: Principles and Practice 3rd Edition (Springer-Verlag, N.Y., 1994)). Substantially pure immunoglobulins of at least about 90 to 95%, or even 98 to 99% or more homogeneity are preferred.</p>
    <heading>VI. Purification of Human Telomerase</heading> <p>The present invention provides isolated human telomerase of unprecedented purity. In particular, the present invention provides: purified hTRT of recombinant or nonrecombinant origin; purified hTRT-hTR complexes (i.e., RNPs) of recombinant, nonrecombinant, or mixed origin, optionally comprising one or more telomerase-associated proteins; purified naturally occurring human telomerase; and the like. Moreover, the invention provides methods and reagents for partially, substantially or highly purifying the above-molecules and complexes, including variants, fusion proteins, naturally occurring proteins, and the like (collectively referred to as “hTRT and/or hTRT complexes”).</p>
    <p>Prior to the present disclosure, attempts had been made to purify the telomerase enzyme complex to homogeneity had met with limited success. The methods provided in the aforelisted applications provide purification of telomerase by approximately up to 60,000-fold or more compared to crude cell extracts. The present invention provides hTRT and hTRT complexes of even greater purity, in part by virtue of the novel immunoaffinity reagents (e.g., anti-hTRT antibodies) of the present invention, and/or the reagents, cells, and methods provided herein for recombinant expression of hTRT. Recombinant expression of hTRT and hTRT complexes facilitates purification because the desired molecules can be produced at much higher levels than found in most expressing cells occurring in nature, and/or because the recombinant hTRT molecule can be modified (e.g., by fusion with an epitope tag) such that it may be easily purified.</p>
    <p>It will be recognized that naturally occurring telomerase can be purified from any telomerase-positive cell, and recombinant hTRT and hTRT complexes can be expressed and purified, inter alia, using any of the in vitro, in vivo, ex vivo, or plant or animal expression systems disclosed supra, or others/systems known in the art.</p>
    <p>In one embodiment, the hTRT, telomerase and other compositions of the invention are purified using an immunoaffinity step, alone or in combination with other purification steps. Typically, an immobilized or immobilizable anti-hTRT antibody, as provided by the present invention, is contacted with a sample, such as a cell lysate, that contains the desired hTRT or hTRT-containing complex under conditions in which anti-hTRT antibody binds the hTRT antigen. After removal of the unbound components of the sample by methods well known in the art, the hTRT composition may be eluted, if desired, from the antibody, in substantially pure form. In one embodiment, immunoaffinity chromatography methods well known in the art are used (see, e.g., Harlow and Lane, supra; and Ausubel, supra; Hermansan et al., 1992, Immobilized Affinity Ligand Techniques (Academic Press, San Diego)) in accordance with the methods of the invention. In another illustrative embodiment, immunoprecipitation of anti-hTRT-immunoglobulin-hTRT complexes is carried out using immobilized Protein A. Numerous variations and alternative imnmunoaffinity purification protocols suitable for use in accordance with the methods and reagents of the invention are well-known to those of skill.</p>
    <p>In another embodiment, recombinant hTRT proteins can, as a consequence of their high level of expression, be purified using routine protein purification methods, such as ammonium sulfate precipitation, affinity columns (e.g., immunoaffinity), size-exclusion, anion and cation exchange chromatography, gel electrophoresis and the like (see, generally, R. Scopes, Protein Purification, Springer-Verlag, N.Y. (1982) and Deutscher, Methods in Enzymology Vol. 182: Guide to Protein Purification, Academic Press, Inc. N.Y. (1990)) instead of, or in addition to, immunoaffinity methods. Cation exchange methods can be particularly useful due to the basic pI of the hTRT protein. For example, immobilized phosphate may be used as a cation exchange functional group (e.g., P-11 Phosphocellulose, Whatman catalog #4071 or Cellulose Phosphate, Sigma catalog #C 3145). Immobilized phosphate has two advantageous features for hTRT purification—it is a cation exchange resin, and it shows physical resemblance to the phosphate backbone of nucleic acid. This can allow for affinity chromatography because hTRT binds hTR and telomeric DNA. Other non-specific and specific nucleic acid affinity chromatography methods are also useful for purification (e.g., Alberts et al., 1971, <i>Methods Enzymol. </i>21:198; Arnt-Jovin et al., 1975, <i>Eur. J. Biochem. </i>54:411; Pharmacia catalog #27-5575-02). Further exploitation of this binding function of hTRT could include the use of specific nucleic acid (e.g., telomerase primer or hTR) affinity chromatography for purification (Chodosh et al., 1986, <i>Mol. Cell. Biol. </i>6:4723; Wu et al., 1987, <i>Science </i>238:1247; Kadonaga, 1991, <i>Methods Enzymol. </i>208:10); immobilized Cibricon Blue Dye, which shows physical resemblance to nucleotides, is another useful resin for hTRT purification (Pharmacia catalog #17-0948-01 or Sigma catalog #C 1285), due to hTRT binding of nucleotides (e.g., as substrates for DNA synthesis).</p>
    <p>In one embodiment, hTRT proteins are isolated directly from an in vitro or in vivo expression system in which other telomerase components are not coexpressed. It will be recognized that isolated hTRT protein may also be readily obtained from purified human telomerase or hTRT complexes, for example, by disrupting the telomerase RNP (e.g., by exposure to a mild or other denaturant) and separating the RNP components (e.g., by routine means such as chromatography or immunoaffinity chromatography).</p>
    <p>Telomerase purification may be monitored using a telomerase activity assay (e.g., the TRAP assay, conventional assay, or primer-binding assay), by measuring the enrichment of hTRT (e.g., by ELISA), by measuring the enrichment of hTR, or other methods known in the art.</p>
    <p>The purified human telomerase, hTRT proteins, and hTRT complexes provided by the present invention are, in one embodiment, highly purified (i.e., at least about 90% homogeneous, more often at least about 95% homogeneous). Homogeneity can be determined by standard means such as SDS-polyacrylamide gel electrophoresis and other means known in the art (see, e.g., Ausubel et al, supra). It will be understood that, although highly purified human telomerase, hTRT protein, or hTRT complexes are sometimes desired, substantially purified (e.g., at least about 75% homogeneous) or partially purified (e.g., at least about 20% homogeneous) human telomerase, hTRT protein, or hTRT complexes are useful in many applications, and are also provided by the present invention. For example, partially purified telomerase is useful for screening test compounds for telomerase modulatory activity, and other uses (see, infra and supra; see U.S. Pat. No. 5,645,986).</p>
    <heading>VII. Treatment of Telomerase-Related Disease</heading> <p>A) Introduction</p>
    <p>The present invention provides hTRT polynucleotides, polypeptides, and antibodies useful for the treatment of human diseases and disease conditions. The recombinant and synthetic hTRT gene products (protein and mRNA) of the invention can be used to create or elevate telomerase activity in a cell, as well as to inhibit telomerase activity in cells in which it is not desired. Thus, inhibiting, activating or otherwise altering a telomerase activity (e.g., telomerase catalytic activity, fidelity, processivity, telomere binding, etc.) in a cell can be used to change the proliferative capacity of the cell. For example, reduction of telomerase activity in an immortal cell, such as a malignant tumor cell, can render the cell mortal. Conversely, increasing the telomerase activity in a mortal cell (e.g., most human somatic cells) can increase the proliferative capacity of the cell. For example, expression of hTRT protein in dermal fibroblasts, thereby increasing telomere length, will result in increased fibroblast proliferative capacity; such expression can slow or reverse the age-dependent slowing of wound closure (see, e.g., West, 1994, <i>Arch. Derm. </i>130:87).</p>
    <p>Thus, in one aspect, the present invention provides reagents and methods useful for treating diseases and conditions characterized by the presence, absence, or amount of human telomerase activity in a cell and that are susceptible to treatment using the compositions and methods disclosed herein. These diseases include, as described more fully below, cancers, other diseases of cell proliferation (particularly diseases of aging), immunological disorders, infertility (or fertility), and others.</p>
    <p>B) Treatment of Cancer</p>
    <p>The present invention provides methods and compositions for reducing telomerase activity in tumor cells and for treating cancer. Compositions include antisense oligonucleotides, peptides, gene therapy vectors encoding antisense oligonucleotides or activity altering proteins, and anti-hTRT antibodies. Cancer cells (e.g., malignant tumor cells) that express telomerase activity (telomerase-positive cells) can be mortalized by decreasing or inhibiting the endogenous telomerase activity. Moreover, because telomerase levels correlate with disease characteristics such as metastatic potential (e.g., U.S. Pat. Nos. 5,639,613; 5,648,215; 5,489,508; Pandita et al., 1996, <i>Proc. Am. Ass. Cancer Res. </i>37:559), any reduction in telomerase activity could reduce the aggressive nature of a cancer to a more manageable disease state (increasing the efficacy of traditional interventions).</p>
    <p>The invention provides compositions and methods useful for treatment of cancers of any of a wide variety of types, including solid tumors and leukemias. Types of cancer that may be treated include (but are not limited to): adenocarcinoma of the breast, prostate, and colon; all forms of bronchogenic carcinoma of the lung; myeloid; melanoma; hepatoma; neuroblastoma; papilloma; apudoma; choristoma; branchioma; malignant carcinoid syndrome; carcinoid heart disease; carcinoma (e.g., Walker, basal cell, basosquamous, Brown-Pearce, ductal, Ehrlich tumor, in situ, Krebs 2, merkel cell, mucinous, non-small cell lung, oat cell, papillary, scirrhous, bronchiolar, bronchogenic, squamous cell, and transitional cell), histiocytic disorders; leukemia (e.g., B-cell, mixed-cell, null-cell, T-cell, T-cell chronic, HTLV-II-associated, lyphocytic acute, lymphocytic chronic, mast-cell, and myeloid); histiocytosis malignant; Hodgkin's disease; immunoproliferative small; non-Hodgkin's lymphoma; plasmacytoma; reticuloendotheliosis; melanoma; chondroblastoma; chondroma; chondrosarcoma; fibroma; fibrosarcoma; giant cell tumors; histiocytoma; lipoma; liposarcoma; mesothelioma; myxoma; myxosarcoma; osteoma; osteosarcoma; Ewing's sarcoma; synovioma; adenofibroma; adenolymphoma; carcinosarcoma; chordoma; craniopharyngioma; dysgerminoma; hamartoma; mesenchymoma; mesonephroma; myosarcoma; ameloblastoma; cementoma; odontoma; teratoma; thymoma; trophoblastic tumor; adenocarcinoma; adenoma; cholangioma; cholesteatoma; cylindroma; cystadenocarcinoma; cystadenoma; granulosa cell tumor; gynandroblastoma; hepatoma; hidradenoma; islet cell tumor; leydig cell tumor; papilloma; sertoli cell tumor; theca cell tumor; leiomyoma; leiomyosarcoma; myoblastoma; myoma; myosarcoma; rhabdomyoma; rhabdomyosarcoma; ependymoma; ganglioneuroma; glioma; medulloblastoma; meningioma; neurilemmoma; neuroblastoma; neuroepithelioma; neurofibroma; neuroma; paraganglioma; paraganglioma nonchromaffin; angiokeratoma; angiolymphoid hyperplasia with eosinophilia; angioma sclerosing; angiomatosis; glomangioma; hemangioendothelioma; hemangioma; hemangiopericytoma; hemangiosarcoma; lymphangioma; lymphangiomyoma; lymphangiosarcoma; pinealoma; carcinosarcoma; chondrosarcoma; cystosarcoma phyllodes; fibrosarcoma; hemangiosarcoma; leiomyosarcoma; leukosarcoma; liposarcoma; lymphangiosarcoma; myosarcoma; myxosarcoma; ovarian carcinoma; rhabdomyosarcoma; sarcoma (e.g., Ewing's, experimental, Kaposi's, and mast-cell); neoplasms (e.g., bone, breast, digestive system, colorectal, liver, pancreatic, pituitary, testicular, orbital, head and neck, central nervous system, acoustic, pelvic, respiratory tract, and urogenital); neurofibromatosis, and cervical dysplasia). The invention provides compositions and methods useful for treatment of other conditions in which cells have become immortalized or hyperproliferative, e.g., by disregulation (e.g., abnormally high expression) of hTRT, telomerase enzyme, or telomerase activity.</p>
    <p>The present invention further provides compositions and methods for prevention of cancers, including anti-hTRT vaccines, gene therapy vectors that prevent telomerase activation, and gene therapy vectors that result in specific death of telomerase-positive cells. In a related aspect, the gene replacement therapy methods described below may be used for “treating” a genetic predilection for cancers.</p>
    <p>C) Treatment of Other Conditions</p>
    <p>The present invention also provides compositions and methods useful for treatment of diseases and disease conditions (in addition to cancers) characterized by under- or over-expression of telomerase or hTRT gene products. Examples include: diseases of cell proliferation, diseases resulting from cell senescence (particularly diseases of aging), immunological disorders, infertility, diseases of immune dysfunction, and others.</p>
    <p>Certain diseases of aging are characterized by cell senescence-associated changes due to reduced telomere length (compared to younger cells), resulting from the absence (or much lower levels) of telomerase activity in the cell. Decreased telomere length and decreased replicative capacity contribute to diseases such as those described below. Telomerase activity and telomere length can be increased by, for example, increasing levels of hTRT gene products (protein and mRNA) in the cell. A partial listing of conditions associated with cellular senescence in which hTRT expression can be therapeutic includes Alzheimer's disease, Parkinson's disease, Huntington's disease, and stroke; age-related diseases of the integument such as dermal atrophy, elastolysis and skin wrinkling, sebaceous gland hyperplasia, senile lentigo, graying of hair and hair loss, chronic skin ulcers, and age-related impairment of wound healing; degenerative joint disease; osteoporosis; age-related immune system impairment (e.g., involving cells such as B and T lymphocytes, monocytes, neutrophils, eosinophils, basophils, NK cells and their respective progenitors); age-related diseases of the vascular system including atherosclerosis, calcification, thrombosis, and aneurysms; diabetes, muscle atrophy, respiratory diseases, diseases of the liver and GI tract, metabolic diseases, endocrine diseases (e.g., disorders of the pituitary and adrenal gland), reproductive diseases, and age-related macular degeneration. These diseases and conditions can be treated by increasing the levels of hTRT gene products in the cell to increase telomere length, thereby restoring or imparting greater replicative capacity to the cell. Such methods can be carried out on cells cultured ex vivo or cells in vivo. In one embodiment, the cells are first treated to activate telomerase and lengthen telomeres, and then treated to inactivate the hTRT gene and telomerase activity. In a preferred embodiment, telomerase activity is generated by a vector of the invention in an embryonic germ or stem cell prior to or during differentiation.</p>
    <p>The present invention also provides methods and composition useful for treating infertility. Human germline cells (e.g., spermatogonia cells, their progenitors or descendants) are capable of indefinite proliferation and characterized by high telomerase activity. Abnormal or diminished levels of hTRT gene products can result, for example, in inadequate or abnormal production of spermatozoa, leading to infertility or disorders of reproduction. Accordingly, “telomerase-based” infertility can be treated using the methods and compositions described herein to increase telomerase levels. Similarly, because inhibition of telomerase may negatively impact spermatogenesis, oogenesis, and sperm and egg viability, the telomerase inhibitory compositions of the invention can have contraceptive effects when used to reduce hTRT gene product levels in germline cells.</p>
    <p>Further, the invention provides methods and composition useful for decreasing the proliferative potential of telomerase-positive cells such as activated lymphocytes and hematopoietic stem cells by reducing telomerase activity. Thus, the invention provide means for effecting immunosuppression. Conversely, the methods and reagents of the invention are useful for increasing telomerase activity and proliferative potential in cells, such as stem cells, that express a low level of telomerase or no telomerase prior to therapeutic intervention.</p>
    <p>D) Modes of Intervention</p>
    <p>As is clear from the foregoing discussion, modulation of the level of telomerase or telomerase activity of a cell can have a profound effect on the proliferative potential of the cell, and so has great utility in treatment of disease. As is also clear, this modulation may be either a decrease in telomerase activity or an increase in activity. The telomerase modulatory molecules of the invention can act through a number of mechanisms; some of these are described in this and the following subsections to aid the practitioner in selecting therapeutic agents. However, applicants do not intend to be limited to any particular mechanism of action for the novel therapeutic compounds, compositions and methods described herein.</p>
    <p>Telomerase activity may be decreased through any of several mechanisms or combinations of mechanisms. One mechanism is the reduction of hTRT gene expression to reduce telomerase activity. This reduction can be at the level of transcription of the hTRT gene into mRNA, processing (e.g., splicing), nuclear transport or stability of mRNA, translation of mRNA to produce hTRT protein, or stability and function of hTRT protein. Another mechanism is interference with one or more activities of telomerase (e.g., the reverse transcriptase catalytic activity, or the hTR-binding activity) using inhibitory nucleic acids, polypeptides, or other agents (e.g., mimetics, small molecules, drugs and pro-drugs) that can be identified using the methods, or are provided by compositions, disclosed herein. Other mechanisms include sequestration of hTR and/or telomerase associated proteins, and interference with the assembly of the telomerase RNP from its component subunits. In a related mechanism, an hTRT promoter sequence is operably linked to a gene encoding a toxin and introduced into a cell; if or when hTRT transcriptional activators are expressed or activated in the cell, the toxin will be expressed, resulting in specific cell killing.</p>
    <p>A related method for reducing the proliferative capacity of a cell involves introducing an hTRT variant with low fidelity (i.e., one with a high, e.g., greater than 1%, error rate) such that aberrant telomeric repeats are synthesized. These aberrant repeats affect telomere protein binding and lead to chromosomal rearrangements and aberrations and/or lead to cell death.</p>
    <p>Similarly, telomerase activity may be increased through any of several mechanisms, or a combination of mechanisms. These include increasing the amount of hTRT in a cell. Usually this is carried out by introducing an hTRT polypeptide-encoding polynucleotide into the cell (e.g., a recombinantly produced polypeptide comprising an hTRT DNA sequence operably linked to a promoter, or a stable hTRT mRNA). Alternatively, a catalytically active hTRT polypeptide can itself be introduced into a cell or tissue, e.g., by microinjection or other means known in the art. In other mechanisms, expression from the endogenous hTRT gene or the stability of hTRT gene products in the cell can be increased. Telomerase activity in a cell can also be increased by interfering with the interaction of endogenous telomerase inhibitors and the telomerase RNP, or endogenous hTRT transcription repressors and the hTRT gene; by increasing expression or activity of hTRT transcription activators; and other means apparent to those of skill upon review of this disclosure.</p>
    <p>E) Intervention Agents</p>
    <p>1) TRT Proteins &amp; Peptides</p>
    <p>In one embodiment, the invention provides telomerase modulatory polypeptides (i.e., proteins, polypeptides, and peptides) that increase or reduce telomerase activity which can be introduced into a target cell directly (e.g., by injection, liposome-mediated fusion, application of a hydrogel to the tumor [e.g., melanoma] surface, fusion or attachment to herpes virus structural protein VP22, and other means described herein and known in the art). In a second embodiment, telomerase modulatory proteins and peptides of the invention are expressed in a cell by introducing a nucleic acid (e.g., a DNA expression vector or mRNA) encoding the desired protein or peptide into the cell. Expression may be either constitutive or inducible depending on the vector and choice of promoter (see discussion below). Messenger RNA preparations encoding hTRT are especially useful when only transient expression (e.g., transient activation of telomerase) is desired. Methods for introduction and expression of nucleic acids into a cell are well known in the art (also, see elsewhere in this specification, e.g., sections on oligonucleotides, gene therapy methods).</p>
    <p>In one aspect of the invention, a telomerase modulatory polypeptide that increases telomerase activity in a cell is provided. In one embodiment, the polypeptide is a catalytically active hTRT polypeptide capable of directing the synthesis (in conjunction with an RNA template such as hTR) of human telomeric DNA. This activity can be measured, as discussed above, e.g., using a telomerase activity assay such as a TRAP assay. In one embodiment, the polypeptide is a full-length hTRT protein, having a sequence of, or substantially identical to, the sequence of 1132 residues of SEQ ID NO:2. In another embodiment, the polypeptide is a variant of the hTRT protein of SEQ ID NO:2, such as a fusion polypeptide, derivatized polypeptide, truncated polypeptide, conservatively substituted-polypeptide, activity-modified polypeptide, or the like. A fusion or derivatized protein may include a targeting moiety that increases the ability of the polypeptide to traverse a cell membrane or causes the polypeptide to be delivered to a specified cell type (e.g., liver cells or tumor cells) preferentially or cell compartment (e.g., nuclear compartment) preferentially. Examples of targeting moieties include lipid tails, amino acid sequences such as antennapoedia peptide or a nuclear localization signal (NLS; e.g., Xenopus nucleoplasmin Robbins et al., 1991, <i>Cell </i>64:615). Naturally occurring hTRT protein (e.g., having a sequence of, or substantially identical to, SEQ ID NO:2) acts in the cell nucleus. Thus, it is likely that one or more subsequences of SEQ ID NO:2, such as residues 193-196 (PRRR SEQ ID NO:541) and residues 235-240 (PKRPRR SEQ ID NO:542) act as a nuclear localization signal. The small regions are likely NLSs based on the observation that many NLSs comprise a 4 residue pattern composed of basic amino acids (K or R), or composed of three basic amino acids (K or R) and H or P; a pattern starting with P and followed within 3 residues by a basic segment containing 3 K or R residues out of 4 residues (see, e.g., Nakai et al., 1992, <i>Genomics </i>14:897). Deletion of one or both of these sequences and/or additional localization sequences is expected to interfere with hTRT transport to the nucleus and/or increase hTRT turnover, and is useful for preventing access of telomerase to its nuclear substrates and decreasing proliferative potential. Moreover, a variant hTRT polypeptide lacking NLS may assemble into an RNP that will not be able to maintain telomere length, because the resulting enzyme cannot enter the nucleus.</p>
    <p>The hTRT polypeptides of the invention will typically be associated in the target cell with a telomerase RNA, such as hTR, especially when they are used to increase telomerase activity in a cell. In one embodiment, an introduced hTRT polypeptide associates with an endogenous hTR to form a catalytically active RNP (e.g., an RNP comprising the hTR and a full-length polypeptide having a sequence of SEQ ID NO:2). The RNP so-formed may also associate with other, e.g., telomerase-associated, proteins. In other embodiments, telomerase RNP (containing hTRT protein, hTR and optionally other components) is introduced as a complex to the target cell.</p>
    <p>In a related embodiment, an hTRT expression vector is introduced into a cell (or progeny of a cell) into which a telomerase RNA (e.g., hTR) expression vector is simultaneously, subsequently or has been previously introduced. In this embodiment, hTRT protein and telomerase RNA are coexpressed in the cell and assemble to form a telomerase RNP. A preferred telomerase RNA is hTR. An expression vector useful for expression of hTR in a cell is described supra (see U.S. Pat. No. 5,583,016). In yet another embodiment, the hTRT polypeptide and hTR RNA (or equivalent) are associated in vitro to form a complex, which is then introduced into the target cells, e.g., by liposome mediated transfer.</p>
    <p>In another aspect, the invention provides hTRT polypeptides useful for reducing telomerase activity in a cell. As above, these “inhibitory” polypeptides can be introduced directly, or by expression of recombinant nucleic acids in the cell. It will be recognized that peptide mimetics or polypeptides comprising nonstandard amino acids (i.e., other than the 20 amino acids encoded by the genetic code or their normal derivatives) will typically be introduced directly.</p>
    <p>In one embodiment, inhibition of telomerase activity results from the sequestration of a component required for accurate telomere elongation. Examples of such components are hTRT and hTR. Thus, administration of a polypeptide that binds hTR, but which does not have telomerase catalytic activity, can reduce endogenous telomerase activity in the cell. In a related embodiment, the hTRT polypeptide may bind a cell component other than hTR, such as one or more telomerase-associated proteins, thereby interfering with telomerase activity in the cell.</p>
    <p>In another embodiment, hTRT polypeptides of the invention interfere (e.g., by competition) with the interaction of endogenously expressed hTRT protein and another cellular component required for telomerase function, such as hTR, telomeric DNA, telomerase-associated proteins, telomere-associated proteins, telomeres, cell cycle control proteins, DNA repair enzymes, histone or non-histone chromosomal proteins, or others.</p>
    <p>In selecting molecules (e.g., polypeptides) of the invention that affect the interaction of endogenously expressed hTRT protein and other cellular components, one may prefer molecules that include one or more of the conserved motifs of the hTRT protein, as described herein. The evolutionary conservation of these regions indicates the important function in the proper functioning of human telomerase contributed by these motifs, and the motifs are thus generally useful sites for changing hTRT protein function to create variant hTRT proteins of the invention. Thus, variant hTRT polypeptides having mutations in conserved motifs will be particularly useful for some applications of the invention.</p>
    <p>In another embodiment, expression of the endogenous hTRT gene is repressed by introduction into the cell of a large amount of hTRT polypeptide (e.g., typically at least about 2-fold more than the endogenous level, more often at least about 10- to about 100-fold) which acts via a feedback loop to inhibit transcription of the hTRT gene, processing of the hTRT pre-mRNA, translation of the hTRT mRNA, or assembly and transport of the telomerase RNP.</p>
    <p>2) Oligonucleotides</p>
    <p>a) Antisense Constructs</p>
    <p>The invention provides methods and antisense oligonucleotide or polynucleotide reagents which can be used to reduce expression of hTRT gene products in vitro or in vivo. Administration of the antisense reagents of the invention to a target cell results in reduced telomerase activity, and is particularly useful for treatment of diseases characterized by high telomerase activity (e.g., cancers). Without intending to be limited to any particular mechanism, it is believed that antisense oligonucleotides bind to, and interfere with the translation of, the sense hTRT mRNA. Alternatively, the antisense molecule may render the hTRT mRNA susceptible to nuclease digestion, interfere with transcription, interfere with processing, localization or otherwise with RNA precursors (“pre-mRNA”), repress transcription of mRNA from the hTRT gene, or act through some other mechanism. However, the particular mechanism by which the antisense molecule reduces hTRT expression is not critical.</p>
    <p>The antisense polynucleotides of the invention comprise an antisense sequence of at least 7 to 10 to typically 20 or more nucleotides that specifically hybridize to a sequence from mRNA encoding hTRT or mRNA transcribed from the hTRT gene. More often, the antisense polynucleotide of the invention is from about 10 to about 50 nucleotides in length or from about 14 to about 35 nucleotides in length. In other embodiments, antisense polynucleotides are polynucleotides of less than about 100 nucleotides or less than about 200 nucleotides. In general, the antisense polynucleotide should be long enough to form a stable duplex but short enough, depending on the mode of delivery, to administer in vivo, if desired. The minimum length of a polynucleotide required for specific hybridization to a target sequence depends on several factors, such as G/C content, positioning of mismatched bases (if any), degree of uniqueness of the sequence as compared to the population of target polynucleotides, and chemical nature of the polynucleotide (e.g., methylphosphonate backbone, peptide nucleic acid, phosphorothioate), among other factors.</p>
    <p>Generally, to assure specific hybridization, the antisense sequence is substantially complementary to the target hTRT mRNA sequence In certain embodiments, the antisense sequence is exactly complementary to the target sequence. The antisense polynucleotides may also include, however, nucleotide substitutions, additions, deletions, transitions, transpositions, or modifications, or other nucleic acid sequences or non-nucleic acid moieties so long as specific binding to the relevant target sequence corresponding to hTRT RNA or its gene is retained as a functional property of the polynucleotide.</p>
    <p>In one embodiment, the antisense sequence is complementary to relatively accessible sequences of the hTRT mRNA (e.g., relatively devoid of secondary structure). This can be determined by analyzing predicted RNA secondary structures using, for example, the MFOLD program (Genetics Computer Group, Madison Wis.) and testing in vitro or in vivo as is known in the art. Examples of oligonucleotides that may be tested in cells for antisense suppression of hTRT function are those capable of hybridizing to (i.e., substantially complementary to) the following positions from SEQ ID NO:1: 40-60; 260-280; 500-520; 770-790; 885-905; 1000-1020; 1300-1320; 1520-1540; 2110-2130; 2295-2315; 2450-2470; 2670-2690; 3080-3110; 3140-3160; and 3690-3710. Another useful method for identifying effective antisense compositions uses combinatorial arrays of oligonucleotides (see, e.g., Milner et al., 1997, <i>Nature Biotechnology </i>15:537).</p>
    <p>The invention also provides an antisense polynucleotide that has sequences in addition to the antisense sequence (i.e., in addition to anti-hTRT-sense sequence). In this case, the antisense sequence is contained within a polynucleotide of longer sequence. In another embodiment, the sequence of the polynucleotide consists essentially of, or is, the antisense sequence.</p>
    <p>The antisense nucleic acids (DNA, RNA, modified, analogues, and the like) can be made using any suitable method for producing a nucleic acid, such as the chemical synthesis and recombinant methods disclosed herein. In one embodiment, for example, antisense RNA molecules of the invention may be prepared by de novo chemical synthesis or by cloning. For example, an antisense RNA that hybridizes to hTRT mRNA can be made by inserting (ligating) an hTRT DNA sequence (e.g., SEQ ID NO:1, or fragment thereof) in reverse orientation operably linked to a promoter in a vector (e.g., plasmid). Provided that the promoter and, preferably termination and polyadenylation signals, are properly positioned, the strand of the inserted sequence corresponding to the noncoding strand will be transcribed and act as an antisense oligonucleotide of the invention.</p>
    <p>The antisense oligonucleotides of the invention can be used to inhibit telomerase activity in cell-free extracts, cells, and animals, including mammals and humans. For example, the phosphorothioate antisense oligonucleotides:</p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="1" colwidth="28pt" align="left"> </colspec> <colspec colname="2" colwidth="119pt" align="left"> </colspec> <colspec colname="3" colwidth="70pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="3" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">A)</td>
                <td class="description-td">5″-GGCATCGCGGGGGTGGCCGGG</td>
                <td class="description-td">SEQ ID NO:506</td>
              </tr> <tr class="description-tr"> <td class="description-td">B)</td>
                <td class="description-td">5″-CAGCGGGGAGCGCGCGGCATC</td>
                <td class="description-td">SEQ ID NO:521</td>
              </tr> <tr class="description-tr"> <td class="description-td">C)</td>
                <td class="description-td">5″-CAGCACCTCGCGGTAGTGGCT</td>
                <td class="description-td">SEQ ID NO:522</td>
              </tr> <tr class="description-tr"> <td class="description-td">D)</td>
                <td class="description-td">5″-GGACACCTGGCGGAAGGAGGG</td>
                <td class="description-td">SEQ ID NO:507</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="3" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p>can be used to inhibit telomerase activity. At 10 micromolar concentration each oligonucleotide, mixtures of oligonucleotides A and B; A, B, C, and D; and A, C, and D inhibited telomerase activity in 293 cells when treated once per day for seven days. Inhibition was also observed when an antisense hTR molecule (5′-GCTCTAGAATGAAGGGTG-3′; 3′ SEQ ID NO:543) was used in combination with oligonucleotides A, B, and C; A, B, and D; and A and C. Useful control oligonucleotides in such experiments include:</p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="1" colwidth="28pt" align="center"> </colspec> <colspec colname="2" colwidth="119pt" align="left"> </colspec> <colspec colname="3" colwidth="70pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="3" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">        S1)</td>
                <td class="description-td">5″-GCGACGACTGACATTGGCCGG</td>
                <td class="description-td">SEQ ID NO:544</td>
              </tr> <tr class="description-tr"> <td class="description-td">S2)</td>
                <td class="description-td">5″-GGCTCGAAGTAGCACCGGTGC</td>
                <td class="description-td">SEQ ID NO:545</td>
              </tr> <tr class="description-tr"> <td class="description-td">S3)</td>
                <td class="description-td">5″-GTGGGAACAGGCCGATGTCCC</td>
                <td class="description-td">SEQ ID NO:546</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="3" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p>To determine the optimum antisense oligonucleotide of the invention for the particular application of interest, one can perform a scan using antisense oligonucleotide sets of the invention. One illustrative set is the set of 30-mer oligonucleotides that span the hTRT mRNA and are offset one from the next by fifteen nucleotides (i.e., ON1 corresponds to positions 1-30 and is TCCCACGTGCGCAGCAGGACGCAGCGCTGC (SEQ ID NO:547), ON2 corresponds to positions 16-45 and is GCCGGGGCCAGGGCTTCCCACGTGCGCAGC (SEQ ID NO:548), and ON3 corresponds to positions 31-60 and is GGCATCGCGGGGGTGGCCGGGGCCAGGGCT (SEQ ID NO:549), and so on to the end of the mRNA). Each member of this set can be tested for inhibitory activity as disclosed herein. Those oligonucleotides that show inhibitory activity under the conditions of interest then identify a region of interest, and other oligonucleotides of the invention corresponding to the region of interest (i.e., 8-mers, 10-mers, 15-mers, and so on) can be tested to identify the oligonucleotide with the preferred activity for the application.</p>
    <p>Exemplary antisense oligonucleotides include 5′-GGCATCGCGGGGGTG GCCGGGGCCAGGGCT-3′ (SEQ ID NO:722) (corresponding to nucleotide positions 31-60 of hTRT); 5′-GCGCA GCGTGCCAGCAGGTGAACCAGCACG-3′ (SEQ ID NO:723) (corresponding to positions 496-525); 5′-GCCCGTTCGCATCCCAGACGCCTTCGGGGT-3′ (SEQ ID NO:724) (corresponding to positions 631-660); and 5′-ACGCTATGGTTCCAGGCCCGTTCGCATCCC-3′ (SEQ ID NO:725) (corresponding to positions 646-675). When ACHN cells (NCI #503755) or 293 cells were treated for three days with 10 μM of phosphorothioate oligonucleotides with any of the four sequences supra, inhibition of telomerase activity by about 50%-90% (compared to control untreated cells) as measured by a TRAP assay, was observed.</p>
    <p>For general methods relating to antisense polynucleotides, see Antisense RNA and DNA, (1988), D. A. Melton, Ed., Cold Spring Harbor Laboratory, Cold Spring Harbor, N.Y.). See also, Dagle et al., 1991, <i>Nucleic Acids Research, </i>19:1805. For a review of antisense therapy, see, e.g., Uhlmann et al., <i>Chem. Reviews, </i>90:543-584 (1990).</p>
    <p>b) Triplex Oligo- and Polynucleotides</p>
    <p>The present invention provides oligo- and polynucleotides (e.g., DNA, RNA, PNA or the like) that bind to double-stranded or duplex hTRT nucleic acids (e.g., in a folded region of the hTRT RNA or in the hTRT gene), forming a triple helix-containing, or “triplex” nucleic acid. Triple helix formation results in inhibition of hTRT expression by, for example, preventing transcription of the hTRT gene, thus reducing or eliminating telomerase activity in a cell. Without intending to be bound by any particular mechanism, it is believed that triple helix pairing compromises the ability of the double helix to open sufficiently for the binding of polymerases, transcription factors, or regulatory molecules to occur.</p>
    <p>Triplex oligo- and polynucleotides of the invention are constructed using the base-pairing rules of triple helix formation (see, e.g., Cheng et al., 1988, <i>J. Biol. Chem. </i>263: 15110; Ferrin and Camerini-Otero, 1991, <i>Science </i>354:1494; Ramdas et al., 1989, <i>J. Biol. Chem. </i>264:17395; Strobel et al., 1991, <i>Science </i>254:1639; and Rigas et al., 1986, <i>Proc. Natl. Acad. Sci. U.S.A. </i>83: 9591; each of which is incorporated herein by reference) and the hTRT mRNA and/or gene sequence. Typically, the triplex-forming oligonucleotides of the invention comprise a specific sequence of from about 10 to at least about 25 nucleotides or longer “complementary” to a specific sequence in the hTRT RNA or gene (i.e., large enough to form a stable triple helix, but small enough, depending on the mode of delivery, to administer in vivo, if desired). In this context, “complementary” means able to form a stable triple helix. In one embodiment, oligonucleotides are designed to bind specifically to the regulatory regions of the hTRT gene (e.g., the hTRT 5′-flanking sequence, promoters, and enhancers) or to the transcription initiation site, (e.g., between −10 and +10 from the transcription initiation site). For a review of recent therapeutic advances using triplex DNA, see Gee et al., in Huber and Carr, 1994, Molecular and Immunologic Approaches, Futura Publishing Co, Mt Kisco N.Y. and Rininsland et al., 1997, <i>Proc. Natl. Acad. Sci. USA </i>94:5854, which are both incorporated herein by reference.</p>
    <p>c) Ribozymes</p>
    <p>The present invention also provides ribozymes useful for inhibition of telomerase activity. The ribozymes of the invention bind and specifically cleave and inactivate hTRT mRNA. Useful ribozymes can comprise 5′- and 3′-terminal sequences complementary to the hTRT mRNA and can be engineered by one of skill on the basis of the hTRT mRNA sequence disclosed herein (see PCT publication WO 93/23572, supra). Ribozymes of the invention include those having characteristics of group I intron ribozymes (Cech, 1995, <i>Biotechnology </i>13:323) and others of hammerhead ribozymes (Edgington, 1992, <i>Biotechnology </i>10:256).</p>
    <p>Ribozymes of the invention include those having cleavage sites such as GUA, GUU) and GUC. Other optimum cleavage sites for ribozyme-mediated inhibition of telomerase activity in accordance with the present invention include those described in PCT publications WO 94/02595 and WO 93/23569, both incorporated herein by reference. Short RNA oligonucleotides between 15 and 20 ribonucleotides in length corresponding to the region of the target hTRT gene containing the cleavage site can be evaluated for secondary structural features that may render the oligonucleotide more desirable. The suitability of cleavage sites may also be evaluated by testing accessibility to hybridization with complementary oligonucleotides using ribonuclease protection assays, or by testing for in vitro ribozyme activity in accordance with standard procedures known in the art.</p>
    <p>As described by Hu et al., PCT publication WO 94/03596, incorporated herein by reference, antisense and ribozyme functions can be combined in a single oligonucleotide. Moreover, ribozymes can comprise one or more modified nucleotides or modified linkages between nucleotides, as described above in conjunction with the description of illustrative antisense oligonucleotides of the invention.</p>
    <p>In one embodiment, the ribozymes of the invention are generated in vitro and introduced into a cell or patient. In another embodiment, gene therapy methods are used for expression of ribozymes in a target cell ex vivo or in vivo.</p>
    <p>d) Administration of Oligonucleotides</p>
    <p>Typically, the therapeutic methods of the invention involve the administration of an oligonucleotide that functions to inhibit or stimulate telomerase activity under in vivo physiological conditions, and is relatively stable under those conditions for a period of time sufficient for a therapeutic effect. As noted above, modified nucleic acids may be useful in imparting such stability, as well as for targeting delivery of the oligonucleotide to the desired tissue, organ, or cell.</p>
    <p>Oligo- and poly-nucleotides can be delivered directly as a drug in a suitable pharmaceutical formulation, or indirectly by means of introducing a nucleic acid into a cell, including liposomes, immunoliposomes, ballistics, direct uptake into cells, and the like as described herein. For treatment of disease, the oligonucleotides of the invention will be administered to a patient in a therapeutically effective amount. A therapeutically effective amount is an amount sufficient to ameliorate the symptoms of the disease or modulate telomerase activity in the target cell, e.g., as can be measured using a TRAP assay or other suitable assay of telomerase biological function. Methods useful for delivery of oligonucleotides for therapeutic purposes are described in U.S. Pat. No. 5,272,065, incorporated herein by reference. Other details of administration of pharmaceutically active compounds are provided below. In another embodiment, oligo- and poly-nucleotides can be delivered using gene therapy and recombinant DNA expression plasmids of the invention.</p>
    <p>3) Gene Therapy</p>
    <p>Gene therapy refers to the introduction of an otherwise exogenous polynucleotide which produces a medically useful phenotypic effect upon the (typically) mammalian cell(s) into which it is transferred. In one aspect, the present invention provides gene therapy methods and compositions for treatment of telomerase-associated conditions. In illustrative embodiments, gene therapy involves introducing into a cell a vector that expresses an hTRT gene product (such as an hTRT protein substantially similar to the hTRT polypeptide having a sequence of SEQUENCE ID NO:2, e.g., to increase telomerase activity, or an inhibitory hTRT polypeptide to reduce activity), expresses a nucleic acid having an hTRT gene or mRNA sequence (such as an antisense RNA, e.g., to reduce telomerase activity), expresses a polypeptide or polynucleotide that otherwise affects expression of hTRT gene products (e.g., a ribozyme directed to hTRT mRNA to reduce telomerase activity), or replaces or disrupts an endogenous hTRT sequence (e.g., gene replacement and “gene knockout,” respectively). Numerous other embodiments will be evident to one of skill upon review of the disclosure herein. In one embodiment, a vector encoding hTR is also introduced. In another embodiment, vectors encoding telomerase-associated proteins are also introduced with or without a vector for hTR.</p>
    <p>Vectors useful in hTRT gene therapy can be viral or nonviral, and include those described supra in relation to the hTRT expression systems of the invention. It will be understood by those of skill in the art that gene therapy vectors may comprise promoters and other regulatory or processing sequences, such as are described in this disclosure. Usually the vector will comprise a promoter and, optionally, an enhancer (separate from any contained within the promoter sequences) that serve to drive transcription of an oligoribonucleotide, as well as other regulatory elements that provide for episomal maintenance or chromosomal integration and for high-level transcription, if desired. A plasmid useful for gene therapy can comprise other functional elements, such as selectable markers, identification regions, and other sequences. The additional sequences can have roles in conferring stability both outside and within a cell, targeting delivery of hTRT nucleotide sequences (sense or antisense) to a specified organ, tissue, or cell population, mediating entry into a cell, mediating entry into the nucleus of a cell and/or mediating integration within nuclear DNA. For example, aptamer-like DNA structures, or other protein binding moieties sites can be used to mediate binding of a vector to cell surface receptors or to serum proteins that bind to a receptor thereby increasing the efficiency of DNA transfer into the cell. Other DNA sites and structures can directly or indirectly bind to receptors in the nuclear membrane or to other proteins that go into the nucleus, thereby facilitating nuclear uptake of a vector. Other DNA sequences can directly or indirectly affect the efficiency of integration.</p>
    <p>Suitable gene therapy vectors may, or may not, have an origin of replication. For example, it is useful to include an origin of replication in a vector for propagation of the vector prior to administration to a patient. However, the origin of replication can often be removed before administration if the vector is designed to integrate into host chromosomal DNA or bind to host mRNA or DNA. In some situations (e.g., tumor cells) it may not be necessary for the exogenous DNA to integrate stably into the transduced cell, because transient expression may suffice to kill the tumor cells.</p>
    <p>As noted, the present invention also provides methods and reagents for gene replacement therapy (i.e., replacement by homologous recombination of an endogenous hTRT gene with a recombinant gene). Vectors specifically designed for integration by homologous recombination may be used. Important factors for optimizing homologous recombination include the degree of sequence identity and length of homology to chromosomal sequences. The specific sequence mediating homologous recombination is also important, because integration occurs much more easily in transcriptionally active DNA. Methods and materials for constructing homologous targeting constructs are described by e.g., Mansour et al., 1988, <i>Nature </i>336: 348; Bradley et al., 1992, <i>Bio/Technology </i>10: 534. See also, U.S. Pat. Nos. 5,627,059; 5,487,992; 5,631,153; and 5,464,764. In one embodiment, gene replacement therapy involves altering or replacing all or a portion of the regulatory sequences controlling expression of the hTRT gene that is to be regulated. For example, the hTRT promoter sequences (e.g., such as are found in SEQ ID NO:6) may be disrupted (to decrease hTRT expression or to abolish a transcriptional control site) or an exogenous promoter (e.g., to increase hTRT expression) substituted.</p>
    <p>The invention also provides methods and reagents for hTRT “gene knockout” (i.e., deletion or disruption by homologous recombination of an endogenous hTRT gene using a recombinantly produced vector). In gene knockout, the targeted sequences can be regulatory sequences (e.g., the hTRT promoter), or RNA or protein coding sequences. The use of homologous recombination to alter expression of endogenous genes is described in detail in U.S. Pat. No. 5,272,071 (and the U.S. Patents cited supra), WO 91/09955, WO 93/09222, WO 96/29411, WO 95/31560, and WO 91/12650. See also, Moynahan et al., 1996, <i>Hum. Mol. Genet. </i>5:875.</p>
    <p>The invention further provides methods for specifically killing telomerase-positive cells, or preventing transformation of telomerase negative cells to a telomerase positive state, using the hTRT gene promoter to regulate expression of a protein toxic to the cell. As shown in Example 14, an hTRT promoter sequence may be operably linked to a reporter gene such that activation of the promoter results in expression of the protein encoded by the reporter gene. If, instead of a reporter protein, the encoded protein is toxic to the cell, activation of the promoter leads to cell morbidity or death. In one embodiment of the present invention, a vector comprising an hTRT promoter operably linked to a gene encoding a toxic protein is introduced into cells, such as human cells, e.g., cells in a human patient, resulting in cell death of cells in which hTRT promoter activating factors are expressed, such as cancer cells. In a related embodiment, the encoded protein is not itself toxic to a cell, but encodes an activity that renders the cell sensitive to an otherwise nontoxic drug. For example, tumors can be treated by introducing an hTRT-promoter-Herpes thymidine kinase (TK) gene fusion construct into tumor cells, and administering gancyclovir or the equivalent (see, e.g., Moolton and Wells, 1990, <i>J. Nat'l Canc. Inst. </i>82:297). The art knows of numerous other suitable toxic or potentially toxic proteins and systems (using promoter sequences other that hTRT) that may be modified and applied in accordance with the present invention by one of skill in the art upon review of this disclosure.</p>
    <p>Gene therapy vectors may be introduced into cells or tissues in vivo, in vitro or ex vivo. For ex vivo therapy, vectors may be introduced into cells, e.g., stem cells, taken from the patient and clonally propagated for autologous transplant back into the same patient (see, e.g., U.S. Pat. Nos. 5,399,493 and 5,437,994, the disclosures of which are herein incorporated by reference). Cells that can be targeted for hTRT gene therapy aimed at increasing the telomerase activity of a target cell include, but are not limited to, embryonic stem or germ cells, particularly primate or human cells, as noted supra, hematopoietic stem cells (AIDS and post-chemotherapy), vascular endothelial cells (cardiac and cerebral vascular disease), skin fibroblasts and basal skin keratinocytes (wound healing and burns), chondrocytes (arthritis), brain astrocytes and microglial cells (Alzheimer's Disease), osteoblasts (osteoporosis), retinal cells (eye diseases), and pancreatic islet cells (Type I diabetes) and any of the cells listed in Table 3, infra, as well as any other cell types known to divide.</p>
    <p>In one embodiment of the invention, an inducible promoter operably linked to a TRT, such as HTRT, coding sequence (or variant) is used to modulate the proliferative capacity of cells in vivo or in vitro. In a particular embodiment, for example, insulin-producing pancreatic cells transfected with an hTRT expression vector under the control of an inducible promoter are introduced into a patient. The proliferative capacity of the cells can then be controlled by administration to the patient of the promoter activating agent (e.g., tetracycline) to enable the cells to multiply more than otherwise would have been possible. Cell proliferation can then be terminated, continued, or reinitiated as desired by the treating physician.</p>
    <p>4) Vaccines and Antibodies</p>
    <p>Immuogenic peptides or polypeptides having an hTRT sequence can be used to elicit an anti-hTRT immune response in a patient (i.e., act as a vaccine). Exemplary immunogenic hTRT peptides and polypeptides are described infra in Examples 6 and 8. An immune response can also be raised by delivery of plasmid vectors encoding the polypeptide of interest (i.e., administration of “naked DNA”). The nucleic acids of interest can be delivered by injection, liposomes, or other means of administration. In one embodiment, immunization modes that elicit in the subject a Class I MHC restricted cytotoxic lymphocyte response against telomerase expressing cells are chosen. Once immunized, the individual or animal will elicit a heightened immune response against cells expressing high levels of telomerase (e.g., malignant cells).</p>
    <p>Anti-hTRT antibodies, e.g., murine, human, or humanized monoclonal antibodies may also be administered to a patient (e.g., passive immunization) to effect an immune response against telomerase-expressing cells.</p>
    <p>F) Pharmaceutical Compositions</p>
    <p>In related aspects, the invention provides pharmaceutical compositions that comprise hTRT oligo- and poly-nucleotides, polypeptides, and antibodies, agonists, antagonists, or inhibitors, alone or in combination with at least one other agent, such as a stabilizing compound, diluent, carrier, or another active ingredient or agent.</p>
    <p>The therapeutic agents of the invention may be administered in any sterile, biocompatible pharmaceutical carrier, including, but not limited to, saline, buffered saline, dextrose, and water. Any of these molecules can be administered to a patient alone, or in combination with other agents, drugs or hormones, in pharmaceutical compositions where it is mixed with suitable excipient(s), adjuvants, and/or pharmaceutically acceptable carriers. In one embodiment of the present invention, the pharmaceutically acceptable carrier is pharmaceutically inert.</p>
    <p>Administration of pharmaceutical compositions is accomplished orally or parenterally. Methods of parenteral delivery include topical, intra-arterial (e.g., directly to the tumor), intramuscular, subcutaneous, intramedullary, intrathecal, intraventricular, intravenous, intraperitoneal, or intranasal administration. In addition to the active ingredients, these pharmaceutical compositions may contain suitable pharmaceutically acceptable carriers comprising excipients and other compounds that facilitate processing of the active compounds into preparations which can be used pharmaceutically. Further details on techniques for formulation and administration may be found in the latest edition of “Remington's Pharmaceutical Sciences” (Maack Publishing Co, Easton Pa.).</p>
    <p>Pharmaceutical compositions for oral administration can be formulated using pharmaceutically acceptable carriers well known in the art in dosages suitable for oral administration. Such carriers enable the pharmaceutical compositions to be formulated as tablets, pills, dragees, capsules, liquids, gels, syrups, slurries, suspensions, etc., suitable for ingestion by the patient. See PCT publication WO 93/23572.</p>
    <p>Pharmaceutical preparations for oral use can be obtained through combination of active compounds with solid excipient, optionally grinding a resulting mixture, and processing the mixture of granules, after adding suitable additional compounds, if desired, to obtain tablets or dragee cores. Suitable excipients are carbohydrate or protein fillers include, but are not limited to sugars, including lactose, sucrose, mannitol, or sorbitol; starch from corn, wheat, rice, potato, or other plants; cellulose such as methyl cellulose, hydroxypropylmethyl-cellulose, or sodium carboxymethylcellulose; and gums including arabic and tragacanth; as well as proteins such as gelatin and collagen. If desired, disintegrating or solubilizing agents may be added, such as the cross-linked polyvinyl pyrrolidone, agar, alginic acid, or a salt thereof, such as sodium alginate.</p>
    <p>Dragee cores are provided with suitable coatings such as concentrated sugar solutions, which may also contain gum arabic, talc, polyvinylpyrrolidone, carbopol gel, polyethylene glycol, and/or titanium dioxide, lacquer solutions, and suitable organic solvents or solvent mixtures. Dyestuffs or pigments may be added to the tablets or dragee coatings for product identification or to characterize the quantity of active compound (i.e., dosage).</p>
    <p>Pharmaceutical preparations which can be used orally include push-fit capsules made of gelatin, as well as soft, sealed capsules made of gelatin and a coating such as glycerol or sorbitol. Push-fit capsules can contain active ingredients mixed with a filler or binders such as lactose or starches, lubricants such as talc or magnesium stearate, and, optionally, stabilizers. In soft capsules, the active compounds may be dissolved or suspended in suitable liquids, such as fatty oils, liquid paraffin, or liquid polyethylene glycol with or without stabilizers.</p>
    <p>Pharmaceutical formulations for parenteral administration include aqueous solutions of active compounds. For injection, the pharmaceutical compositions of the invention may be formulated in aqueous solutions, preferably in physiologically compatible buffers such as Hank's solution, Ringer's solution, or physiologically buffered saline. Aqueous injection suspensions may contain substances which increase the viscosity of the suspension, such as sodium carboxymethyl cellulose, sorbitol, or dextran. Additionally, suspensions of the active compounds may be prepared as appropriate oily injection suspensions. Suitable lipophilic solvents or vehicles include fatty oils such as sesame oil, or synthetic fatty acid esters, such as ethyl oleate or triglycerides, or liposomes. Optionally, the suspension may also contain suitable stabilizers or agents which increase the solubility of the compounds to allow for the preparation of highly concentrated solutions.</p>
    <p>For topical or nasal administration, penetrants appropriate to the particular barrier to be permeated are used in the formulation. Such penetrants are generally known in the art.</p>
    <p>The pharmaceutical compositions of the present invention may be manufactured in a manner similar to that known in the art (e.g., by means of conventional mixing, dissolving, granulating, dragee-making, levigating, emulsifying, encapsulating, entrapping or lyophilizing processes).</p>
    <p>The pharmaceutical composition may be provided as a salt and can be formed with many acids, including but not limited to hydrochloric, sulfuric, acetic, lactic, tartaric, malic, succinic, etc. Salts tend to be more soluble in aqueous or other protonic solvents that are the corresponding free base forms. In other cases, the preferred preparation may be a lyophilized powder in 1 mM-50 mM histidine, 0.1%-2% sucrose, 2%-7% mannitol at a pH range of 4.5 to 5.5, that is combined with buffer prior to use.</p>
    <p>After pharmaceutical compositions comprising a compound of the invention formulated in a acceptable carrier have been prepared, they can be placed in an appropriate container and labeled for treatment of an indicated condition. For administration of human telomerase proteins and nucleic acids, such labeling would include amount, frequency and method of administration.</p>
    <p>Pharmaceutical compositions suitable for use in the present invention include compositions wherein the active ingredients are contained in an effective amount to achieve the intended purpose. “Therapeutically effective amount” or “pharmacologically effective amount” are well recognized phrases and refer to that amount of an agent effective to produce the intended pharmacological result. Thus, a therapeutically effective amount is an amount sufficient to ameliorate the symptoms of the disease being treated. One useful assay in ascertaining an effective amount for a given application (e.g., a therapeutically effective amount) is measuring the effect on telomerase activity in a target cell. The amount actually administered will be dependent upon the individual to which treatment is to be applied, and will preferably be an optimized amount such that the desired effect is achieved without significant side-effects. The determination of a therapeutically effective dose is well within the capability of those skilled in the art.</p>
    <p>For any compound, the therapeutically effective dose can be estimated initially either in cell culture assays or in any appropriate animal model. The animal model is also used to achieve a desirable concentration range and route of administration. Such information can then be used to determine useful doses and routes for administration in humans.</p>
    <p>A therapeutically effective amount refers to that amount of protein, polypeptide, peptide, antibody, oligo- or polynucleotide, agonist or antagonists which ameliorates the symptoms or condition. Therapeutic efficacy and toxicity of such compounds can be determined by standard pharmaceutical procedures in cell cultures or experimental animals (e.g., ED<sub>50</sub>, the dose therapeutically effective in 50% of the population; and LD<sub>50</sub>, the dose lethal to 50% of the population). The dose ratio between therapeutic and toxic effects is the therapeutic index, and it can be expressed as the ratio, ED<sub>50</sub>/LD<sub>50</sub>. Pharmaceutical compositions which exhibit large therapeutic indices are preferred. The data obtained from cell culture assays and animal studies is used in formulating a range of dosage for human use. The dosage of such compounds lies preferably within a range of circulating concentrations that include the ED<sub>50 </sub>with little or no toxicity. The dosage varies within this range depending upon the dosage form employed, sensitivity of the patient, and the route of administration.</p>
    <p>The exact dosage is chosen by the individual physician in view of the patient to be treated. Dosage and administration are adjusted to provide sufficient levels of the active moiety or to maintain the desired effect. Additional factors which may be taken into account include the severity of the disease state (e.g., tumor size and location; age, weight and gender of the patient; diet, time and frequency of administration, drug combination(s), reaction sensitivities and tolerance/response to therapy). Long acting pharmaceutical compositions might be administered every 3 to 4 days, every week, or once every two weeks depending on half-life and clearance rate of the particular formulation. Guidance as to particular dosages and methods of delivery is provided in the literature (see, U.S. Pat. Nos. 4,657,760; 5,206,344; and 5,225,212, herein incorporated by reference). Those skilled in the art will typically employ different formulations for nucleotides than for proteins or their inhibitors. Similarly, delivery of polynucleotides or polypeptides can be specific to particular cells, conditions, locations, and the like.</p>
    <heading>VIII. Increasing Proliferative Capacity and Production of Immortalized Cells, Cell Lines, and Animals</heading> <p>As discussed above, most vertebrate cells senesce after a finite number of divisions in culture (e.g., 50 to 100 divisions). Certain variant cells, however, are able to divide indefinitely in culture (e.g., HeLa cells, 293 cells) and, for this reason, are useful for research and industrial applications. Usually these immortal cell lines are derived from spontaneously arising tumors, or by transformation by exposure to radiation or a tumor-inducing virus or chemical. Unfortunately, a limited selection of cell lines, especially human cell lines representing differentiated cell function, is available. Moreover, the immortal cell lines presently available are characterized by chromosomal abnormalities (e.g., aneuploidy, gene rearrangements, or mutations). Further, many long-established cell lines are relatively undifferentiated (e.g., they do not produce highly specialized products of the sort that uniquely characterize particular tissues or organs). Thus, there is a need for new methods of generating immortal cells, especially human cells. As used herein, the term “immortalized cells” is not limited to cells that proliferate indefinitely, but may also include cells with increased proliferative capacity compared to similar wild-type cells. Depending on the cell type, increased proliferative capacity may mean proliferation for at least about 100, about 150, about 200, or about 400 or more generations, or for at least about 6, about 12, about 18, about 24 or about 36 or more months in in vitro culture. One use for immortalized cells is in production of natural proteins and recombinant proteins (e.g., therapeutic polypeptides such as erytbropoietin, human growth hormone, insulin, and the like), or antibodies, for which a stable, genetically normal cell line is preferred. For production of some recombinant proteins, specialized cell types may also be preferred (e.g., pancreatic cells for the production of human insulin). Another use for immortalized cells or even mortal cells with increased proliferative capacity (relative to unmodified cells) is for introduction into a patient for gene therapy, or for replacement of diseased or damaged cells or tissue. For example, autologous immune cells containing or expressing a, e.g., recombinant hTRT gene or polypeptide of the invention can be used for cell replacement in a patient after aggressive cancer therapy, e.g., whole body irradiation. Another use for immortalized cells is for ex vivo production of “artificial” tissues or organs (e.g., skin) for therapeutic use. Another use for such cells is for screening or validation of drugs, such as telomerase-inhibiting drugs, or for use in production of vaccines or biological reagents. Additional uses of the cells of the invention will be apparent to those of skill.</p>
    <p>The immortalized cells and cell lines, as well as those of merely increased replicative capacity, of the invention are made by increasing telomerase activity in the cell. Any method disclosed herein for increasing telomerase activity can be used. Thus, in one embodiment, cells are immortalized by increasing the amount of an hTRT polypeptide in the cell. In one embodiment, hTRT levels are increased by introducing an hTRT expression vector into the cell (with stable transfection sometimes preferred). As discussed above, the hTRT coding sequence is usually operably linked to a promoter, which may be inducible or constitutively active in the cell.</p>
    <p>In one embodiment, a polynucleotide comprising a sequence encoding a polypeptide of SEQ ID NO:2, which sequence is operably linked to a promoter (e.g., a constitutively expressed promoter, e.g., a sequence of SEQ ID NO:6, is introduced into the cell. In one embodiment the polynucleotide comprises a sequence of SEQ ID NO:1. Preferably the polynucleotide includes polyadenylation and termination signals. In other embodiments, additional elements such as enhancers or others discussed supra are included. In an alternative embodiment, the polynucleotide does not include a promoter sequence, such sequence being provided by the target cell endogenous genome following integration (e.g., recombination, e.g., homologous recombination) of the introduced polynucleotide. The polynucleotide may be introduced into the target cell by any method, including any method disclosed herein, such as lipofection, electroporation, virosomes, liposomes, immnunoliposomes, polycation:nucleic acid conjugates, naked DNA).</p>
    <p>Using the methods of the invention, any vertebrate cell can be caused to have an increased proliferative capacity or even be immortalized and sustained indefinitely in culture. In one embodiment the cells are mammalian, with human cells preferred for many applications. Examples of human cells that can be immortalized include those listed in Table 3.</p>
    <p>It will be recognized that the “diagnostic” assays of the invention described infra may be used to identify and characterize the immortalized cells of the invention.</p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 3</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">HUMAN CELLS IN WHICH hTRT EXPRESSION</td>
              </tr> <tr class="description-tr"> <td class="description-td">MAY BE INCREASED</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Keratinizing Epithelial Cells</td>
              </tr> <tr class="description-tr"> <td class="description-td">keratinocyte of epidermis (differentiating epidermal cell)</td>
              </tr> <tr class="description-tr"> <td class="description-td">basal cell of epidermis (stem cell)</td>
              </tr> <tr class="description-tr"> <td class="description-td">keratinocyte of fingernails and toenails</td>
              </tr> <tr class="description-tr"> <td class="description-td">basal cell of nail bed (stem cell)</td>
              </tr> <tr class="description-tr"> <td class="description-td">hair shaft cells</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="203pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">medullary, cortical, cuticular; hair-root sheath cells,</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">cuticular, of Huxley's layer, of Henle's layer external;</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">hair matrix cell (stem cell)</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Cells of Wet Stratified Barrier Epithelia</td>
              </tr> <tr class="description-tr"> <td class="description-td">surface epithelial cell of stratified squamous epithelium of</td>
              </tr> <tr class="description-tr"> <td class="description-td">tongue, oral cavity, esophagus, anal canal, distal urethra,</td>
              </tr> <tr class="description-tr"> <td class="description-td">vagina</td>
              </tr> <tr class="description-tr"> <td class="description-td">basal cell of these epithelia (stem cell)</td>
              </tr> <tr class="description-tr"> <td class="description-td">cell of external corneal epithelium</td>
              </tr> <tr class="description-tr"> <td class="description-td">cell of urinary epithelium (lining bladder and urinary ducts)</td>
              </tr> <tr class="description-tr"> <td class="description-td">Epithelial Cells Specialized for Exocrine Secretion</td>
              </tr> <tr class="description-tr"> <td class="description-td">cells of salivary gland</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="203pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">mucous cell (secretion rich in polysaccharide)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">serous cell (secretion rich in glycoprotein enzymes) cell</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">of von Ebner's gland in tongue (secretion to wash over</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">taste buds)</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">cell of mammary gland, secreting milk</td>
              </tr> <tr class="description-tr"> <td class="description-td">cell of lacrimal gland, secreting tears</td>
              </tr> <tr class="description-tr"> <td class="description-td">cell of ceruminous gland of ear, secreting wax</td>
              </tr> <tr class="description-tr"> <td class="description-td">cell of eccrine sweat gland, secreting glycoproteins (dark</td>
              </tr> <tr class="description-tr"> <td class="description-td">cell)</td>
              </tr> <tr class="description-tr"> <td class="description-td">cell of eccrine sweat gland, secreting small molecules (clear</td>
              </tr> <tr class="description-tr"> <td class="description-td">cell)</td>
              </tr> <tr class="description-tr"> <td class="description-td">cell of apocrine sweat gland (odoriferous secretion, sex-</td>
              </tr> <tr class="description-tr"> <td class="description-td">hormone sensitive)</td>
              </tr> <tr class="description-tr"> <td class="description-td">cell of gland of Moll in eyelid (specialized sweat gland)</td>
              </tr> <tr class="description-tr"> <td class="description-td">cell of sebaceous gland, secreting lipid-rich sebum</td>
              </tr> <tr class="description-tr"> <td class="description-td">cell of Bowman's gland in nose (secretion to wash over</td>
              </tr> <tr class="description-tr"> <td class="description-td">olfactory epithelium)</td>
              </tr> <tr class="description-tr"> <td class="description-td">cell of Brunner's gland in duodenum, secreting alkaline</td>
              </tr> <tr class="description-tr"> <td class="description-td">solution of mucus and enzymes</td>
              </tr> <tr class="description-tr"> <td class="description-td">cell of seminal vesicle, secreting components of seminal</td>
              </tr> <tr class="description-tr"> <td class="description-td">fluid, including fructose (as fuel for swimming sperm)</td>
              </tr> <tr class="description-tr"> <td class="description-td">cell of prostate gland, secreting other components of seminal</td>
              </tr> <tr class="description-tr"> <td class="description-td">fluid</td>
              </tr> <tr class="description-tr"> <td class="description-td">cell of bulbourethral gland, secreting mucus</td>
              </tr> <tr class="description-tr"> <td class="description-td">cell of Bartholin's gland, secreting vaginal lubricant</td>
              </tr> <tr class="description-tr"> <td class="description-td">cell of gland of Littré, secreting mucus</td>
              </tr> <tr class="description-tr"> <td class="description-td">cell of endometrium of uterus, secreting mainly carbohydrates</td>
              </tr> <tr class="description-tr"> <td class="description-td">isolated goblet cell of respiratory and digestive tracts,</td>
              </tr> <tr class="description-tr"> <td class="description-td">secreting mucus</td>
              </tr> <tr class="description-tr"> <td class="description-td">mucous cell of lining of stomach</td>
              </tr> <tr class="description-tr"> <td class="description-td">zymogenic cell of gastric gland, secreting pepsinogen</td>
              </tr> <tr class="description-tr"> <td class="description-td">oxyntic cell of gastric gland, secreting HCl</td>
              </tr> <tr class="description-tr"> <td class="description-td">acinar cell of pancreas, secreting digestive enzymes and</td>
              </tr> <tr class="description-tr"> <td class="description-td">bicarbonate</td>
              </tr> <tr class="description-tr"> <td class="description-td">Paneth cell of small intestine, secreting lysozyme</td>
              </tr> <tr class="description-tr"> <td class="description-td">type II pneumocyte of lung, secreting surfactant</td>
              </tr> <tr class="description-tr"> <td class="description-td">Clara cell of lung</td>
              </tr> <tr class="description-tr"> <td class="description-td">Cells specialized for Secretion of Hormones</td>
              </tr> <tr class="description-tr"> <td class="description-td">cells of anterior pituitary, secreting</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="203pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">growth hormone, follicle-stimulating hormone,</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">luteinizing hormone, prolactin, adrenocorticotropic</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">hormone, and thyroid-stimulating hormone,</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">cell of intermediate pituitary, secreting</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="203pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">melanocyte-stimulating hormone</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">cells of posterior pituitary, secreting</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="203pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">oxytocin, vasopressin</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">cells of gut, secreting</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="203pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">serotonin, endorphin, somatostatin, gastrin, secretin,</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">cholecystokinin, insulin and glucagon</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">cells of thyroid gland, secreting</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="203pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">thyroid hormone, calcitonin</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">cells of parathyroid gland, secreting</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="203pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">parathyroid hormone, oxyphil cell</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">cells of adrenal gland, secreting</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="203pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">epinephrine, norepinephrine, and steroid hormones;</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="189pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">mineralocorticoids</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">glucocorticoids</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">cells of gonads, secreting</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="203pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">testosterone (Leydig cell of testis)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">estrogen (theca interna cell of ovarian follicle)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">progesterone (corpus luteum cell of ruptured ovarian</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">follicle)</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">cells of juxtaglomerular apparatus of kidney</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="203pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">juxtaglomerular cell (secreting renin)</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">macula densa cell</td>
              </tr> <tr class="description-tr"> <td class="description-td">peripolar cell</td>
              </tr> <tr class="description-tr"> <td class="description-td">mesangial cell</td>
              </tr> <tr class="description-tr"> <td class="description-td">Epithelial Absorptive Cells in Gut, Exocrine Glands, and</td>
              </tr> <tr class="description-tr"> <td class="description-td">Urogenital Tract</td>
              </tr> <tr class="description-tr"> <td class="description-td">brush border cell of intestine (with microvilli)</td>
              </tr> <tr class="description-tr"> <td class="description-td">striated duct cell of exocrine glands</td>
              </tr> <tr class="description-tr"> <td class="description-td">gall bladder epithelial cell</td>
              </tr> <tr class="description-tr"> <td class="description-td">brush border cell of proximal tubule of kidney</td>
              </tr> <tr class="description-tr"> <td class="description-td">distal tubule cell of kidney</td>
              </tr> <tr class="description-tr"> <td class="description-td">nonciliated cell of ductulus efferens</td>
              </tr> <tr class="description-tr"> <td class="description-td">epididymal principal cell</td>
              </tr> <tr class="description-tr"> <td class="description-td">epididymal basal cell</td>
              </tr> <tr class="description-tr"> <td class="description-td">Cells Specialized for Metabolism and Storage</td>
              </tr> <tr class="description-tr"> <td class="description-td">hepatocyte (liver cell)</td>
              </tr> <tr class="description-tr"> <td class="description-td">fat cells</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="203pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">white fat</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">brown fat</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">lipocyte of liver</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Epithelial Cells Serving Primarily a Barrier Function, Lining</td>
              </tr> <tr class="description-tr"> <td class="description-td">the Lung, Gut, Exocrine Glands, and Urogenital Tract</td>
              </tr> <tr class="description-tr"> <td class="description-td">type I pneumocyte (lining air space of lung)</td>
              </tr> <tr class="description-tr"> <td class="description-td">pancreatic duct cell (centroacinar cell)</td>
              </tr> <tr class="description-tr"> <td class="description-td">nonstriated duct cell of sweat gland, salivary gland, mammary</td>
              </tr> <tr class="description-tr"> <td class="description-td">gland</td>
              </tr> <tr class="description-tr"> <td class="description-td">parietal cell of kidney glomerulus</td>
              </tr> <tr class="description-tr"> <td class="description-td">podocyte of kidney glomerulus</td>
              </tr> <tr class="description-tr"> <td class="description-td">cell of thin segment of loop of Henle (in kidney)</td>
              </tr> <tr class="description-tr"> <td class="description-td">collecting duct cell (in kidney)</td>
              </tr> <tr class="description-tr"> <td class="description-td">duct cell of seminal vesicle, prostate gland</td>
              </tr> <tr class="description-tr"> <td class="description-td">Epithelial Cells Lining Closed Internal Body Cavities</td>
              </tr> <tr class="description-tr"> <td class="description-td">vascular endothelial cells of blood vessels and lymphatics</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="203pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">fenestrated</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">continuous</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">splenic</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">synovial cell (lining joint cavities, secreting largely</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="203pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">hyaluronic acid)</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">serosal cell (lining peritoneal, pleural, and pericardial</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="203pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">cavities)</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">squamous cell lining perilymphatic space of ear</td>
              </tr> <tr class="description-tr"> <td class="description-td">cells lining endolymphatic space of ear</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="203pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">squamous cell</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">columnar cells of endolymphatic sac</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="189pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">with microvilli</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">without microvilli</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="203pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">“dark” cell</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">vestibular membrane cell (resembling choroid plexus cell)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">stria vascularis basal cell</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">stria vascularis marginal cell</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">cell of Claudius</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">cell of Boettcher</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">choroid plexus cell (secreting cerebrospinal fluid)</td>
              </tr> <tr class="description-tr"> <td class="description-td">squamous cell of pia-arachnoid</td>
              </tr> <tr class="description-tr"> <td class="description-td">cells of ciliary epithelium of eye</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="203pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">pigmented</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">nonpigmented</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">corneal “endothelial” cell</td>
              </tr> <tr class="description-tr"> <td class="description-td">Ciliated Cells with Propulsive Function</td>
              </tr> <tr class="description-tr"> <td class="description-td">of respiratory tract</td>
              </tr> <tr class="description-tr"> <td class="description-td">of oviduct and of endometrium of uterus (in female)</td>
              </tr> <tr class="description-tr"> <td class="description-td">of rete testis and ductulus efferens (in male)</td>
              </tr> <tr class="description-tr"> <td class="description-td">of central nervous system (ependymal cell lining brain</td>
              </tr> <tr class="description-tr"> <td class="description-td">cavities)</td>
              </tr> <tr class="description-tr"> <td class="description-td">Cells Specialized for Secretion of Extracellular Matrix</td>
              </tr> <tr class="description-tr"> <td class="description-td">epithelial:</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="203pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">ameloblast (secreting enamel of tooth)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">planum semilunatum cell of vestibular apparatus of ear</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="189pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">(secreting proteoglycan)</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="203pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">interdental cell of organ of Corti (secreting tectorial</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="189pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">“membrane” covering hair cells of organ of Corti)</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">nonepithelial (connective tissue)</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="203pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">fibroblasts (various-of loose connective tissue, of cornea,</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">of tendon, of reticular tissue of bone marrow, etc.)</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="203pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">pericyte of blood capillary</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">nucleus pulposus cell of intervertebral disc</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">cementoblast/cementocyte (secreting bonelike cementum of</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">root of tooth)</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="203pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">odontoblast/odontocyte (secreting dentin of tooth)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">chondrocytes</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="189pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">of hyaline cartilage, of fibrocartilage, of elastic</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">cartilage</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="203pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">osteoblast/osteocyte</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">osteoprogenitor cell (stem cell of osteoblasts)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">hyalocyte of vitreous body of eye</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">stellate cell of perilymphatic space of ear</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Contractile Cells</td>
              </tr> <tr class="description-tr"> <td class="description-td">skeletal muscle cells</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="203pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">red (slow)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">white (fast)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">intermediate</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">muscle spindleXXnuclear bag</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">muscle spindleXXnuclear chain</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">satellite cell (stem cell)</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">heart muscle cells</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="203pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">ordinary</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">nodal</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Purkinje fiber</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">smooth muscle cells</td>
              </tr> <tr class="description-tr"> <td class="description-td">myoepithelial cells</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="203pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">of iris</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">of exocrine glands</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Cells of Blood and Immune System</td>
              </tr> <tr class="description-tr"> <td class="description-td">red blood cell</td>
              </tr> <tr class="description-tr"> <td class="description-td">megakaryocyte</td>
              </tr> <tr class="description-tr"> <td class="description-td">macrophages</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="203pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">monocyte</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">connective tissue macrophage (various)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Langerhans cell (in epidermis)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">osteoclast (in bone)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">dendritic cell (in lymphoid tissues)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">microglial cell (in central nervous system)</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">neutrophil</td>
              </tr> <tr class="description-tr"> <td class="description-td">eosinophil</td>
              </tr> <tr class="description-tr"> <td class="description-td">basophil</td>
              </tr> <tr class="description-tr"> <td class="description-td">mast cell</td>
              </tr> <tr class="description-tr"> <td class="description-td">T lymphocyte</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="203pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">helper T cell</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">suppressor T cell</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">killer T cell</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">B lymphocyte</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="203pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">IgM</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">IgG</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">IgA</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">IgE</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">killer cell</td>
              </tr> <tr class="description-tr"> <td class="description-td">stem cells for the blood and immune system (various)</td>
              </tr> <tr class="description-tr"> <td class="description-td">Sensory Transducers</td>
              </tr> <tr class="description-tr"> <td class="description-td">photoreceptors</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="203pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">rod</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">cones</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="189pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">blue sensitive</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">green sensitive</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">red sensitive</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">hearing</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="203pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">inner hair cell of organ of Corti</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">outer hair cell of organ of Corti</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">acceleration and gravity</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="203pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">type I hair cell of vestibular apparatus of ear</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">type II hair cell of vestibular apparatus of ear</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">taste</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="203pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">type 11 taste bud cell</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">smell</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="203pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">olfactory neuron</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">basal cell of olfactory epithelium (stem cell for olfactory</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="189pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">neurons)</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">blood Ph</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="203pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">carotid body cell</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="189pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">type I</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">type II</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">touch</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="203pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Merkel cell of epidermis</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">primary sensory neurons specialized for touch temperature</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">primary sensory neurons specialized for temperature</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="28pt" align="left"> </colspec> <colspec colname="1" colwidth="189pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">cold sensitive</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">heat sensitive</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">pain</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="203pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">primary sensory neurons specialized for pain configurations</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">and forces in musculoskeletal system</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="203pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">proprioceptive primary sensory neurons</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Autonomic Neurons</td>
              </tr> <tr class="description-tr"> <td class="description-td">cholinergic</td>
              </tr> <tr class="description-tr"> <td class="description-td">adrenergic</td>
              </tr> <tr class="description-tr"> <td class="description-td">peptidergic</td>
              </tr> <tr class="description-tr"> <td class="description-td">Supporting Cells of Sense Organs and of Peripheral Neurons</td>
              </tr> <tr class="description-tr"> <td class="description-td">supporting cells of organ of Corti</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="203pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">inner pillar cell</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">outer pillar cell</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">inner phalangeal cell</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">outer phalangeal cell</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">border cell</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Hensen cell</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">supporting cell of vestibular apparatus</td>
              </tr> <tr class="description-tr"> <td class="description-td">supporting cell of taste bud (type I taste bud cell)</td>
              </tr> <tr class="description-tr"> <td class="description-td">supporting cell of olfactory epithelium</td>
              </tr> <tr class="description-tr"> <td class="description-td">Schwann cell</td>
              </tr> <tr class="description-tr"> <td class="description-td">satellite cell (encapsulating peripheral nerve cell bodies)</td>
              </tr> <tr class="description-tr"> <td class="description-td">enteric glial cell</td>
              </tr> <tr class="description-tr"> <td class="description-td">Neurons and Glial Cells of Central Nervous System</td>
              </tr> <tr class="description-tr"> <td class="description-td">neurons</td>
              </tr> <tr class="description-tr"> <td class="description-td">glial cells</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="203pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">astrocyte</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">oligodendrocyte</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Lens Cells</td>
              </tr> <tr class="description-tr"> <td class="description-td">anterior lens epithelial cell</td>
              </tr> <tr class="description-tr"> <td class="description-td">lens fiber (crystallin-containing cell)</td>
              </tr> <tr class="description-tr"> <td class="description-td">Pigment Cells</td>
              </tr> <tr class="description-tr"> <td class="description-td">melanocyte, retinal pigmented epithelial cell</td>
              </tr> <tr class="description-tr"> <td class="description-td">Germ Cells</td>
              </tr> <tr class="description-tr"> <td class="description-td">oogonium/oocyte</td>
              </tr> <tr class="description-tr"> <td class="description-td">spermatocyte</td>
              </tr> <tr class="description-tr"> <td class="description-td">spermatogonium (stem cell for spermatocyte)</td>
              </tr> <tr class="description-tr"> <td class="description-td">Nurse Cells</td>
              </tr> <tr class="description-tr"> <td class="description-td">ovarian follicle cell</td>
              </tr> <tr class="description-tr"> <td class="description-td">Sertoli cell (in testis)</td>
              </tr> <tr class="description-tr"> <td class="description-td">thymus epithelial cell</td>
              </tr> <tr class="description-tr"> <td class="description-td">Stem Cells</td>
              </tr> <tr class="description-tr"> <td class="description-td">embryonic stem cell</td>
              </tr> <tr class="description-tr"> <td class="description-td">embryonic germ cell</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="2"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="203pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">adult stem cell</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">fetal stem cell</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="1" align="center" rowsep="1" class="description-td" colspan="2"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <heading>IX. Diagnostic Assays</heading> <p>A) Introduction</p>
    <p>1) TRT Assays</p>
    <p>The present invention provides a wide variety of assays for TRT, preferably hTRT, and telomerase. These assays provide, inter alia, the basis for sensitive, inexpensive, convenient, and widely applicable assays for diagnosis and prognosis of a number of human diseases, of which cancer is an illustrative example. As noted supra, hTRT gene products (protein and mRNA) are usually elevated in immortal human cells relative to most normal mortal cells (i.e., telomerase-negative cells and most telomerase-positive normal adult somatic cells). Thus, in one aspect, the invention provides assays useful for detecting or measuring the presence, absence, or quantity of an hTRT gene product in a sample from, or containing, human or other mammalian or eukayotic cells to characterize the cells as immortal (such as a malignant tumor cell) or mortal (such as most normal somatic cells in adults) or as telomerase positive or negative.</p>
    <p>Any condition characterized by the presence or absence of an hTRT gene product (i.e., protein or RNA) may be diagnosed using the methods and materials described herein. These include, as described more fully below, cancers, other diseases of accelerated cell proliferation, immunological disorders, fertility, infertility, and others. Moreover, because the degree to which telomerase activity is elevated in cancer cells is correlated with characteristics of the tumor, such as metastatic potential, monitoring hTRT, mRNA or protein levels can be used to estimate and predict the likely future progression of a tumor.</p>
    <p>In one aspect, the diagnostic and prognostic methods of the invention entail determining whether a human TRT gene product is present in a biological sample (e.g., from a patient). In a second aspect, the abundance of hTRT gene product in a biological sample (e.g., from a patient) is determined and compared to the abundance in a control sample (e.g., normal cells or tissues). In a third aspect, the cellular or intracellular localization of an hTRT gene product is determined in a cell or tissue sample. In a fourth aspect, host (e.g., patient) cells are assayed to identify nucleic acids with sequences characteristic of a heritable propensity for abnormal hTRT gene expression (abnormal quantity, regulation, or product), such as is useful in genetic screening or genetic counseling. In a fifth aspect, the assays of the invention are used detect the presence of anti-hTRT antibodies (e.g., in patient serum). The methods described below in some detail are indicative of useful assays that can be carried out using the sequences and relationships disclosed herein. However, numerous variations or other applications of these assays will be apparent to those of ordinary skill in the art in view of this disclosure.</p>
    <p>It will be recognized that, although the assays below are presented in terms of diagnostic and prognostic methods, they may be used whenever an hTRT gene, gene product, or variant is to be detected, quantified, or characterized. Thus, for example, the “diagnostic” methods described infra are useful for assays of hTRT or telomerase during production and purification of hTRT or human telomerase, for characterization of cell lines derived from human cells (e.g., to identify immortal lines), for characterization of cells, non-human animals, plants, fungi, bacteria or other organisms that comprise a human TRT gene or gene product (or fragments thereof).</p>
    <p>As used herein, the term “diagnostic” has its usual meaning of identifying the presence or nature of a disease (e.g., cancer), condition (e.g., infertile, activated), or status (e.g., fertile), and the term “prognostic” has its usual meaning of predicting the probable development and/or outcome of a disease or condition. Although these two terms are used in somewhat different ways in a clinical setting, it will be understood that any of the assays or assay formats disclosed below in reference to “diagnosis” are equally suitable for determination of prognosis because it is well established that higher telomerase activity levels are associated with poorer prognoses for cancer patients, and because the present invention provides detection methods specific for hTRT, which is expressed at levels that closely correlate with telomerase activity in a cell.</p>
    <p>2) Diagnosis and Prognosis of Cancer</p>
    <p>The determination of an hTRT gene, mRNA or protein level above normal or standard range is indicative of the presence of telomerase-positive cells, or immortal, of which certain tumor cells are examples. Because certain embryonic and fetal cells, as well as certain adult stem cells, express telomerase, the present invention also provides methods for determining other conditions, such as pregnancy, by the detection or isolation of telomerase positive fetal cells from maternal blood. These values can be used to make, or aid in making, a diagnosis, even when the cells would not have been classified as cancerous or otherwise detected or classified using traditional methods. Thus, the methods of the present invention permit detection or verification of cancerous or other conditions associated with telomerase with increased confidence, and at least in some instances at an earlier stage. The assays of the invention allow discrimination between different classes and grades of human tumors or other cell-proliferative diseases by providing quantitative assays for the hTRT gene and gene products and thereby facilitate the selection of appropriate treatment regimens and accurate diagnoses. Moreover, because levels of telomerase activity can be used to distinguish between benign and malignant tumors (e.g., U.S. Pat. No. 5,489,508; Hiyama et al., 1997, <i>Proc. Am Ass. Cancer Res. </i>38:637), to predict immanence of invasion (e.g., U.S. Pat. No. 5,639,613; Yashima et al., 1997, <i>Proc. Am Ass. Cancer Res. </i>38:326), and to correlate with metastatic potential (e.g., U.S. Pat. No. 5,648,215; Pandita et al, 1996, <i>Proc. Am Ass. Cancer Res. </i>37:559), these assays will be useful for prophylaxis, detection, and treatment of a wide variety of human cancers.</p>
    <p>For prognosis of cancers (or other diseases or conditions characterized by elevated telomerase), a prognostic value of hTRT gene product (mRNA or protein) or activity for a particular tumor type, class or grade, is determined as described infra. hTRT protein or mRNA levels or telomerase activity in a patient can also be determined (e.g., using the assays disclosed herein) and compared to the prognostic level.</p>
    <p>Depending on the assay used, in some cases the abundance of an hTRT gene product in a sample will be considered elevated whenever it is detectable by the assay. Due to the low abundance of hTRT mRNA and protein even in telomerase-positive cells, and the rarity or non-existence of these gene products in normal or telomerase-negative cells, sensitive assays are required to detect the hTRT gene product if present at all in normal cells. If less sensitive assays are selected, hTRT gene products will be undetectable in healthy tissue but will be detectable in telomerase-positive cancer or other telomerase-positive cells. Typically, the amount of hTRT gene product in an elevated sample is at least about five, frequently at least about ten, more often at least about 50, and very often at least about 100 to 1000 times higher than the levels in telomerase-negative control cells or cells from healthy tissues in an adult, where the percentage of telomerase-positive normal cells is very low.</p>
    <p>The diagnostic and prognostic methods of the present invention can be employed with any cell or tissue type of any origin and can be used to detect an immortal or neoplastic cell, or tumor tissue, or cancer, of any origin. Types of cancer that may be detected include, but are not limited to, all those listed supra in the discussion of therapeutic applications of hTRT.</p>
    <p>The assays of the invention are also useful for monitoring the efficacy of therapeutic intervention in patients being treated with anticancer regimens. Anticancer regimens that can be monitored include all presently approved treatments (including chemotherapy, radiation therapy, and surgery) and also includes treatments to be approved in the future, such as telomerase inhibition or activation therapies as described herein. (See, e.g., See PCT Publication Nos. 96/01835 and 96/40868 and U.S. Pat. No. 5,583,016; all of which are incorporated by reference in their entirety).</p>
    <p>In another aspect, the assays described below are useful for detecting certain variations in hTRT gene sequence (mutations and heritable hTRT alleles) that are indicative of a predilection for cancers or other conditions associated with abnormal regulation of telomerase activity (infertility, premature aging).</p>
    <p>3) Diagnosis of Conditions Other than Cancer</p>
    <p>In addition to diagnosis of cancers, the assays of the present invention have numerous other applications. The present invention provides reagents and methods/diagnosis of conditions or diseases characterized by under- or over-expression of telomerase or hTRT gene products in cells. In adults, a low level of telomerase activity is normally found in a limited complement of normal human somatic cells, e.g., stem cells, activated lymphocytes and germ cells, and is absent from other somatic cells. Thus, the detection of hTRT or telomerase activity in cells in which it is normally absent or inactive, or detection at abnormal (i.e., higher or lower than normal) levels in cells in which hTRT is normally present at a low level (such as stem cells, activated lymphocytes and germ cells), can be diagnostic of a telomerase-related disease or condition or used to identify or isolate a specific cell type (i.e., to isolate stem cells). Examples of such diseases and conditions include: diseases of cell proliferation, immunological disorders, infertility, diseases of immune cell function, pregnancy, fetal abnormalities, premature aging, and others. Moreover, the assays of the invention are useful for monitoring the effectiveness of therapeutic intervention (including but not limited to drugs that modulate telomerase activity) in a patient or in a cell- or animal-based assay.</p>
    <p>In one aspect, the invention provides assays useful for diagnosing infertility. Human germ cells (e.g., spermatogonia cells, their progenitors or descendants) are capable of indefinite proliferation and characterized by high telomerase activity. Abnormal levels or products or diminished levels of hTRT gene products can result in inadequate or abnormal production of spermatozoa, leading to infertility or disorders of reproduction. Accordingly, the invention provides assays (methods and reagents) for diagnosis and treatment of “telomerase-based” reproductive disorders. Similarly, the assays can be used to monitor the efficacy of contraceptives (e.g., male contraceptives) that target or indirectly affect sperm production (and which would reduce hTRT levels or telomerase activity).</p>
    <p>In another aspect, the invention provides assays for analysis of telomerase and hTRT levels and function in stem cells, fetal cells, embryonic cells, activated lymphocytes and hematopoietic stem cells. For example, assays for hTRT gene product detection can be used to monitor immune function generally (e.g., by monitoring the prevalence of activated lymphocytes or abundance of progenitor stem cells), to identify or select or isolate activated lymphocytes or stem cells (based on elevated hTRT levels), and to monitor the efficacy of therapeutic interventions targeting these tissues (e.g., immunosuppressive agents or therapeutic attempt to expand a stem cell population).</p>
    <p>The invention also provides assays useful for identification of anti-telomerase and anti-TRT immunoglobulins (found in serum from a patient). The materials and assays described herein can be used to identify patients in which such autoimmune antibodies are found, permitting diagnosis and treatment of the condition associated with the immunoglobulins.</p>
    <p>4) Monitoring Cells in Culture</p>
    <p>The assays described herein are also useful for monitoring the expression of hTRT gene products and characterization of hTRT genes in cells ex vivo or in vitro. Because elevated hTRT levels are characteristic of immortalized cells, the assays of the invention can be used, for example, to screen for, or identify, immortalized cells or to identify an agent capable of mortalizing immortalized cells by inhibiting hTRT expression or function. For example, the assay will be useful for identifying cells immortalized by increased expression of hTRT in the cell, e.g., by the expression of a recombinant hTRT or by increased expression of an endogenously coded hTRT (e.g., by promoter activation).</p>
    <p>Similarly, these assays may be used to monitor hTRT expression in transgenic animals or cells (e.g., yeast or human cells containing an hTRT gene). In particular, the effects of certain treatments (e.g., application of known or putative telomerase antagonists) on the hTRT levels in human and nonhuman cells expressing the hTRT of the invention can be used for identifying useful drugs and drug candidates (e.g., telomerase activity-modulating drugs).</p>
    <p>B) Normal, Diagnostic, and Prognostic Values</p>
    <p>Assays for the presence or quantity of hTRT gene products may be carried out and the results interpreted in a variety of ways, depending on the assay format, the nature of the sample being assayed, and the information sought. For example, the steady state abundance of hTRT gene products is so low in most human somatic tissues that they are undetectable by certain assays. Moreover, there is generally no telomerase activity in the cells of these tissues, making verification of activity quite easy. Conversely, hTRT protein and/or hTRT mRNA or telomerase is sufficiently abundant in other telomerase-positive tissues, e.g., malignant tumors, so that the same can be detected using the same assays. Even in those somatic cell types in which low levels of telomerase activity can normally be detected (e.g., stem cells, and certain activated hematopoietic system cells), the levels of hTRT mRNA and telomerase activity are a small fraction (e.g., estimated at about 1% or less) of the levels in immortal cells; thus, immortal and mortal cells may be easily distinguished by the methods of the present invention. It will be appreciated that, when a “less sensitive” assay is used, the mere detection of the hTRT gene product in a biological sample can itself be diagnostic, without the requirement for additional analysis. Moreover, although the assays described below can be made exquisitely sensitive, they may also, if desired, be made less sensitive (e.g., through judicious choice of buffers, wash conditions, numbers of rounds of amplification, reagents, and/or choice of signal amplifiers). Thus, virtually any assay can be designed so that it detects hTRT gene products only in biological samples in which they are present at a particular concentration, e.g. a higher concentration than in healthy or other control tissue. In this case, any detectable level of hTRT mRNA or protein will be considered elevated in cells from post-natal human somatic tissue (other than hematopoietic cells and other stem cells).</p>
    <p>In some cases, however, it will be desirable to establish normal or baseline values (or ranges) for hTRT gene product expression levels, particularly when very sensitive assays capable of detecting very low levels of hTRT gene products that may be present in normal somatic cells are used. Normal levels of expression or normal expression products can be determined for any particular population, subpopulation, or group of organisms according to standard methods well known to those of skill in the art and employing the methods and reagents of the invention. Generally, baseline (normal) levels of hTRT protein or hTRT mRNA are determined by quantitating the amount of hTRT protein and/or mRNA in biological samples (e.g., fluids, cells or tissues) obtained from normal (healthy) subjects, e.g., a human subject. For certain samples and purposes, one may desire to quantitate the amount of hTRT gene product on a per cell, or per tumor cell, basis. To determine the cellularity of a sample, one may measure the level of a constitutively expressed gene product or other gene product expressed at known levels in cells of the type from which the sample was taken. Alternatively, normal values of hTRT protein or hTRT mRNA can be determined by quantitating the amount of hTRT protein/RNA in cells or tissues known to be healthy, which are obtained from the same patient from whom diseased (or possibly diseased) cells are collected or from a healthy individual. Alternatively, baseline levels can be defined in some cases as the level present in non-immortal human somatic cells in culture. It is possible that normal (baseline) values may differ somewhat between different cell types (for example, hTRT mRNA levels will be higher in testis than kidney), or according to the age, sex, or physical condition of a patient. Thus, for example, when an assay is used to determine changes in hTRT levels associated with cancer, the cells used to determine the normal range of hTRT gene product expression can be cells from persons of the same or a different age, depending on the nature of the inquiry. Application of standard statistical methods used in molecular genetics permits determination of baseline levels of expression, as well as permits identification of significant deviations from such baseline levels.</p>
    <p>In carrying out the diagnostic and prognostic methods of the invention, as described above, it will sometimes be useful to refer to “diagnostic” and “prognostic values” As used herein, “diagnostic value” refers to a value that is determined for the hTRT gene product detected in a sample which, when compared to a normal (or “baseline”) range of the hTRT gene product is indicative of the presence of a disease. The disease may be characterized by high telomerase activity (e.g., cancer), the absence of telomerase activity (e.g., infertility), or some intermediate value. “Prognostic value” refers to an amount of the hTRT gene product detected in a given cell type (e.g., malignant tumor cell) that is consistent with a particular diagnosis and prognosis for the disease (e.g., cancer). The amount (including a zero amount) of the hTRT gene product detected in a sample is compared to the prognostic value for the cell such that the relative comparison of the values indicates the presence of disease or the likely outcome of the disease (e.g., cancer) progression. In one embodiment, for example, to assess tumor prognosis, data are collected to obtain a statistically significant correlation of hTRT levels with different tumor classes or grades. A predetermined range of hTRT levels is established for the same cell or tissue sample obtained from subjects having known clinical outcomes. A sufficient number of measurements is made to produce a statistically significant value (or range of values) to which a comparison will be made. The predetermined range of hTRT levels or activity for a given cell or tissue sample can then be used to determine a value or range for the level of hTRT gene product that would correlate to favorable (or less unfavorable) prognosis (e.g., a “low level” in the case of cancer). A range corresponding to a “high level” correlated to an (or a more) unfavorable prognosis in the case of cancer can similarly be determined. The level of hTRT gene product from a biological sample (e.g., a patient sample) can then be determined and compared to the low and high ranges and used to predict a clinical outcome.</p>
    <p>Although the discussion above refers to cancer for illustration, it will be understood that diagnostic and prognostic values can also be determined for other diseases (e.g., diseases of cell proliferation) and conditions and that, for diseases or conditions other than cancer, a “high” level may be correlated with the desired outcome and a “low” level correlated with an unfavorable outcome. For example, some diseases may be characterized by a deficiency (e.g., low level) of telomerase activity in stem cells, activated lymphocytes, or germline cells. In such cases, “high” levels of hTRT gene products relative to cells of similar age and/or type (e.g., from other patients or other tissues in a particular patient) may be correlated with a favorable outcome.</p>
    <p>It will be appreciated that the assay methods do not necessarily require measurement of absolute values of hTRT, unless it is so desired, because relative values are sufficient for many applications of the methods of the present invention. Where quantitation is desirable, the present invention provides reagents such that virtually any known method for quantitating gene products can be used.</p>
    <p>The assays of the invention may also be used to evaluate the efficacy of a particular therapeutic treatment regime in animal studies, in clinical trials, or in monitoring the treatment of an individual patient. In these cases, it may be desirable to establish the baseline for the patient prior to commencing therapy and to repeat the assays one or more times through the course of treatment, usually on a regular basis, to evaluate whether hTRT levels are moving toward the desired endpoint (e.g., reduced expression of hTRT when the assay is for cancer) as a result of the treatment.</p>
    <p>One of skill will appreciate that, in addition to the quantity or abundance of hTRT gene products, variant or abnormal expression patterns (e.g., abnormal amounts of RNA splicing variants) or variant or abnormal expression products (e.g., mutated transcripts, truncated or non-sense polypeptides) may also be identified by comparison to normal expression levels and normal expression products. In these cases determination of“normal” or “baseline” involves identifying healthy organisms and/or tissues (i.e. organisms and/or tissues without hTRT expression disregulation or neoplastic growth) and measuring expression levels of the variant hTRT gene products (e.g., splicing variants), or sequencing or detecting the hTRT gene, mRNA, or reverse transcribed cDNA to obtain or detect typical (normal) sequence variations. Application of standard statistical methods used in molecular genetics permits determination of significant deviations from such baseline levels.</p>
    <p>C) Detection and Quantitation of TRT Gene Products</p>
    <p>As has been emphasized herein, hTRT gene products are usually found in most normal somatic cells at extremely low levels. For example, the mRNA encoding hTRT protein is extremely rare or absent in all telomerase-negative cell types studied thus far. In immortal cells, such as 293 cells, HTRT mRNA may be present at only about 100 copies per cell, while normal somatic cells may have as few as one or zero copies per cell. It will thus be apparent that, when highly sensitive assays for hTRT gene products are desired, it will sometimes be advantageous to incorporate signal or target amplification technologies into the assay format. See, for example, Plenat et al., 1997, <i>Ann. Pathol. </i>17:17 (fluoresceinyl-tyramide signal amplification); Zehbe et al., 1997, <i>J. Pathol. </i>150:1553 (catalyzed reporter deposition); other references listed herein (e.g., for bDNA signal amplification, for PCR and other target amplification formats); and other techniques known in the art.</p>
    <p>As noted above, it is often unnecessary to quantitate the hTRT mRNA or protein in the assays disclosed herein, because the detection of an hTRT gene product (under assay conditions in which the product is not detectable in control, e.g., telomerase-negative cells) is in itself sufficient for a diagnosis. As another example, when the levels of product found in a test (e.g., tumor) and control (e.g., healthy cell) samples are directly compared, quantitation may be superfluous.</p>
    <p>When desired, however, quantities of hTRT gene product measured in the assays described herein may be described in a variety of ways, depending on the method of measurement and convenience. Thus, normal, diagnostic, prognostic, high or low quantities of hTRT protein/mRNA may be expressed as standard units of weight per quantity of biological sample (e.g., picograms per gram tissue, picograms per 10<sup>12 </sup>cells), as a number of molecules per quantity of biological sample (e.g., transcripts/cell, moles/cell), as units of activity per cell or per other unit quantity, or by similar methods. The quantity of hTRT gene product can also be expressed in relation to the quantity of another molecule; examples include: number of hTRT transcripts in sample/number of 28S rRNA transcripts in sample; nanograms of hTRT protein/ nanograms of total protein; and the like.</p>
    <p>When measuring hTRT gene products in two (or more) different samples, it will sometimes be useful to have a common basis of comparison for the two samples. For example, when comparing a sample of normal tissue and a sample of cancerous tissue, equal amounts of tissue (by weight, volume, number of cells, etc.) can be compared. Alternatively, equivalents of a marker molecule (e.g., 28S rRNA, hTR, telomerase activity, telomere length, actin) may be used. For example, the amount of hTRT protein in a healthy tissue sample containing 10 picograms of 28S rRNA can be compared to a sample of diseased tissue containing the same amount of 28S rRNA.</p>
    <p>It will also be recognized by those of skill that virtually any of the assays described herein can be designed to be quantitative. Typically, a known quantity or source of an hTRT gene product (e.g., produced using the methods and compositions of the invention) is used to calibrate the assay.</p>
    <p>In certain embodiments, assay formats are chosen that detect the presence, absence, or abundance of an hTRT allele or gene product in each cell in a sample (or in a representative sampling). Examples of such formats include those that detect a signal by histology (e.g., immunohistochemistry with signal-enhancing or target-enhancing amplification steps) or fluorescence-activated cell analysis or cell sorting (FACS). These formats are particularly advantageous when dealing with a highly heterogeneous cell population (e.g., containing multiple cells types in which only one or a few types have elevated hTRT levels, or a population of similar cells expressing telomerase at different levels).</p>
    <p>D) Sample Collection</p>
    <p>The hTRT gene or gene product (i.e., mRNA or polypeptide) is preferably detected and/or quantified in a biological sample. Such samples include, but are not limited to, cells (including whole cells, cell fractions, cell extracts, and cultured cells or cell lines), tissues (including blood, blood cells (e.g., white cells), and tissue samples such as fine needle biopsy samples (e.g., from prostate, breast, thyroid, etc.)), body fluids (e.g., urine, sputum, amniotic fluid, blood, peritoneal fluid, pleural fluid, semen) or cells collected therefrom (e.g., bladder cells from urine, lymphocytes from blood), media (from cultured cells or cell lines), and washes (e.g., of bladder and lung). Biological samples may also include sections of tissues such as frozen sections taken for histological purposes. For cancer diagnosis and prognosis, a sample will be obtained from a cancerous or precancerous or suspected cancerous tissue or tumor. It will sometimes be desirable to freeze a biological sample for later analysis (e.g., when monitoring efficacy of drug treatments).</p>
    <p>In some cases, the cells or tissues may be fractionated before analysis. For example, in a tissue biopsy from a patient, a cell sorter (e.g., a fluorescence-activated cell sorter) may be used to sort cells according to characteristics such as expression of a surface antigen (e.g., a tumor specific antigen) according to well known methods.</p>
    <p>Although the sample is typically taken from a human patient or cell line, the assays can be used to detect hTRT homolog genes or gene products in samples from other animals. Alternatively, hTRT genes and gene products can be assayed in transgenic animals or organisms expressing a human TRT protein or nucleic acid sequence.</p>
    <p>The sample may be pretreated as necessary by dilution in an appropriate buffer solution or concentrated, if desired. Any of a number of standard aqueous buffer solutions, employing one of a variety of buffers, such as phosphate, Tris-buffer, or the like, at physiological pH can be used.</p>
    <p>A “biological sample” obtained from a patient can be referred to either as a “biological sample” or a “patient sample.” It will be appreciated that analysis of a “patient sample” need not necessarily require removal of cells or tissue from the patient. For example, appropriately labeled hTRT-binding agents (e.g., antibodies or nucleic acids) can be injected into a patient and visualized (when bound to the target) using standard imaging technology (e.g., CAT, NMR, and the like.)</p>
    <p>E) Nucleic Acid Assays</p>
    <p>In one embodiment, this invention provides for methods of detecting and/or quantifying expression of hTRT mRNAs (including splicing or sequence variants and alternative alleles). In an alternative embodiment, the invention provides methods for detecting and analyzing normal or abnormal hTRT genes (or fragments thereof). The form of such qualitative or quantitative assays may include, but is not limited to, amplification-based assays with or without signal amplification, hybridization based assays, and combination amplification-hybridization assays. It will be appreciated by those of skill that the distinction between hybridization and amplification is for convenience only: as illustrated in the examples below, many assay formats involve elements of both hybridization and amplification, so that the categorization is somewhat arbitrary in some cases.</p>
    <p>1) Preparation of Nucleic Acids</p>
    <p>In some embodiments, nucleic acid assays are performed with a sample of nucleic acid isolated from the cell, tissue, organism, or cell line to be tested. The nucleic acid (e.g., genomic DNA, RNA or cDNA) may be “isolated” from the sample according to any of a number of methods well known to those of skill in the art. In this context, “isolated” refers to any separation of the species or target to be detected from any other substance in the mixture, but does not necessarily indicate a significant degree of purification of the target. One of skill will appreciate that, where alterations in the copy number of the hTRT gene are to be detected, genomic DNA is the target to be detected. Conversely, where expression levels of a gene or genes are to be detected, RNA is the target to be detected in a nucleic acid-based assay. In one preferred embodiment, the nucleic acid sample is the total mRNA (i.e., poly(A)<sup>+</sup> RNA) in a biological sample. Methods for isolating nucleic acids are well known to those of skill in the art and are described, for example, Tijssen, P. ed. of Laboratory Techniques in Biochemistry and Molecular Biology: Hybridization with Nucleic Acid Probes, Part I. Theory and Nucleic Acid Preparation, Elsevier, N.Y. (1993) Chapt. 3, which is incorporated herein by reference. In one embodiment, the total nucleic acid is isolated from a given sample using an acid guanidinium-phenol-chloroform extraction method and poly(A)+ mRNA is isolated by oligo-dT column chromatography or by using (dT)n magnetic beads (see, e.g., Sambrook et al., and Ausubel et al., supra).</p>
    <p>In alternative embodiments, it is not necessary to isolate nucleic acids (e.g., total or polyA<sup>+</sup> RNA) from the biological sample prior to carrying out amplification, hybridization or other assays. These embodiments have certain advantages when hTRT RNA is to be measured, because they reduce the possibility of loss of hTRT mRNA during isolation and handling. For example, many amplification techniques such as PCR and RT-PCR defined above can be carried out using permeabilized cells (histological specimens and FACS analyses), whole lysed cells, or crude cell fractions such as certain cell extracts. Preferably, steps are taken to preserve the integrity of the target nucleic acid (e.g., mRNA) if necessary (e.g., addition of RNAase inhibitors). Amplification and hybridization assays can also be carried out in situ, for example, in thin tissue sections from a biopsy sample or from a cell monolayer (e.g., blood cells or disagregated tissue culture cells). Amplification can also be carried out in an intact whole cell or fixed cells. For example, PCR, RT-PCR, or LCR amplification methods may be carrier out, as is well known in the art, in situ, e.g., using a polymerase or ligase, a primer or primer(s), and (deoxy)ribonucleoside triphosphates (if a polymerase is employed), and reverse transcriptase and primer (if RNA is to be transcribed and the cDNA is to be detected) on fixed, permeabilized, or microinjected cells to amplify target hTRT RNA or DNA. Cells containing hTRT RNA (e.g., telomerase positive cells) or an hTRT DNA sequence of interest can then be detected. This method is often useful when fluorescently-labeled dNTPs, primers, or other components are used in conjunction with microscopy, FACS analysis or the equivalent.</p>
    <p>2) Amplification Based Assays</p>
    <p>In one embodiment, the assays of the present invention are amplification-based assays for detection of an hTRT gene or gene product. In an amplification based assay, all or part of an hTRT gene or transcript (e.g., mRNA or cDNA; hereinafter also referred to as “target”) is amplified, and the amplification product is then detected directly or indirectly. When there is no underlying gene or gene product to act as a template, no amplification product is produced (e.g., of the expected size), or amplification is non-specific and typically there is no single amplification product. In contrast, when the underlying gene or gene product is present, the target sequence is amplified, providing an indication of the presence and/or quantity of the underlying gene or mRNA. Target amplification-based assays are well known to those of skill in the art.</p>
    <p>The present invention provides a wide variety of primers and probes for detecting hTRT genes and gene products. Such primers and probes are sufficiently complementary to the hTRT gene or gene product to hybridize to the target nucleic acid. Primers are typically at least 6 bases in length, usually between about 10 and about 100 bases, typically between about 12 and about 50 bases, and often between about 14 and about 25 bases in length. One of skill, having reviewed the present disclosure, will be able, using routine methods, to select primers to amplify all, or any portion, of the hTRT gene or gene product, or to distinguish between variant gene products, hTRT alleles, and the like. Table 2 lists illustrative primers useful for PCR amplification of the hTRT, or specific hTRT gene products or regions. As is known in the art, single oligomers (e.g., U.S. Pat. No. 5,545,522), nested sets of oligomers, or even a degenerate pool of oligomers may be employed for amplification, e.g., as illustrated by the amplification of the Tetrahymena TRT cDNA as described infra.</p>
    <p>The invention provides a variety of methods for amplifying and detecting an hTRT gene or gene product, including the polymerase chain reaction (including all variants, e.g., reverse-transcriptase-PCR; the Sunrise Amplification System (Oncor, Inc, Gaithersburg Md.); and numerous others known in the art). In one illustrative embodiment, PCR amplification is carried out in a 50 μl solution containing the nucleic acid sample (e.g., cDNA obtained through reverse transcription of hTRT RNA), 100 μM in each dNTP (dATP, dCTP, dGTP and dTTP; Pharmacia LKB Biotechnology, N.J.), the hTRT-specific PCR primer(s), 1 unit/Taq polymerase (Perkin Elmer, Norwalk Conn.), 1×PCR buffer (50 mM KCl, 10 mM Tris, pH 8.3 at room temperature, 1.5 mM MgCl<sub>2</sub>, 0.01% gelatin) with the amplification ran for about 30 cycles at 94° for 45 sec, 55° for 45 sec and 72° for 90 sec. However, as will be appreciated, numerous variations may be made to optimize the PCR amplification for any particular reaction.</p>
    <p>Other suitable target amplification methods include the ligase chain reaction (LCR; e.g., Wu and Wallace, 1989, <i>Genomics </i>4:560; Landegren et al., 1988, <i>Science, </i>241: 1077, Barany, 1991, <i>Proc. Natl. Acad. Sci. USA </i>88:189 and Barringer et al., 1990, <i>Gene, </i>89: 117); strand displacement amplification (SDA; e.g., Walker et al., 1992, <i>Proc. Natl. Acad. Sci. U.S.A. </i>89:392-396); transcription amplification (e.g., Kwoh et al., 1989, <i>Proc. Natl. Acad. Sci. USA, </i>86: 1173); self-sustained sequence replication (3SR; e.g., Fahy et al., 1992, <i>PCR Methods Appl. </i>1:25, Guatelli et al., 1990, <i>Proc. Nat. Acad. Sci. USA, </i>87: 1874); the nucleic acid sequence based amplification (NASBA, Cangene, Mississauga, Ontario; e.g., Compton, 1991, <i>Nature </i>350:91); the transcription-based amplification system (TAS); and the self-sustained sequence replication system (SSR). Each of the aforementioned publications is incorporated herein by reference. One useful variant of PCR is PCR ELISA (e.g., Boehringer Mannheim Cat. No. 1 636 111) in which digoxigenin-dUTP is incorporated into the PCR product. The PCR reaction mixture is denatured and hybridized with a biotin-labeled oligonucleotide designed to anneal to an internal sequence of the PCR product. The hybridization products are immobilized on streptavidin coated plates and detected using anti-digoxigenin antibodies. Examples of techniques sufficient to direct persons of skill through in vitro amplification methods are found in PCR Technology: Principles and Applications for DNA Amplification, H. Erlich, Ed. Freeman Press, New York, N.Y. (1992); PCR Protocols: A Guide to Methods and Applications, eds. Innis, Gelfland, Snisky, and White, Academic Press, San Diego, Calif. (1990); Mattila et al., 1991, <i>Nucleic Acids Res. </i>19: 4967; Eckert and Kunkel, (1991) PCR Methods and Applications 1: 17; PCR, eds. McPherson, Quirkes, and Taylor, IRL Press, Oxford; U.S. Pat. Nos. 4,683,195, 4,683,202, and 4,965,188; Barringer et al., 1990, <i>Gene, </i>89:117; Lomell et al., 1989, <i>J. Clin. Chem., </i>35:1826, each of which is incorporated herein for all purposes.</p>
    <p>Amplified products may be directly analyzed, e.g., by size as determined by gel electrophoresis; by hybridization to a target nucleic acid immobilized on a solid support such as a bead, membrane, slide, or chip; by sequencing; immunologically, e.g., by PCR-ELISA, by detection of a fluorescent, phosphorescent, or radioactive signal; or by any of a variety of other well-known means. For example, an illustrative example of a detection method uses PCR primers augmented with hairpin loops linked to fluorescein and a benzoic acid derivative that serves as a quencher, such that fluorescence is emitted only when the primers unfold to bind their targets and replication occurs.</p>
    <p>Because hTRT mRNA is typically expressed as an extremely rare transcript, present at very low levels even in telomerase positive cells, it is often desirable to optimize or increase the signal resulting from the amplification step. One way to do this is to increase the number of cycles of amplification. For example, although 20-25 cycles are adequate for amplification of most mRNAs using the polymerase chain reaction under standard reaction conditions, detection of hTRT mRNA in many samples can require as many as 30 to 35 cycles of amplification, depending on detection format and efficiency of amplification. It will be recognized that judicious choice of the amplification conditions including the number of amplification cycles can be used to design an assay that results in an amplification product only when there is a threshold amount of target in the test sample (i.e., so that only samples with a high level of hTRT mRNA give a “positive” result). In addition, methods are known to increase signal produced by amplification of the target sequence. Methods for augmenting the ability to detect the amplified target include signal amplification system such as: branched DNA signal amplification (e.g., U.S. Pat. No. 5,124,246; Urdea, 1994, <i>Bio/Tech. </i>12:926); tyramide signal amplification (TSA) system (Du Pont); catalytic signal amplification (CSA; Dako); Q Beta Replicase systems (Tyagi et al., 1996, <i>Proc. Nat. Acad. Sci. USA, </i>93: 5395); or the like.</p>
    <p>One of skill in the art will appreciate that whatever amplification method is used, a variety of quantitative methods known in the art can be used if quantitation is desired. For example, when desired, two or more polynucleotides can be co-amplified in a single sample. This method can be used as a convenient method of quantitating the amount of hTRT mRNA in a sample, because the reverse transcription and amplification reactions are carried out in the same reaction for a target and control polynucleotide. The co-amplification of the control polynucleotide (usually present at a known concentration or copy number) can be used for normalization to the cell number in the sample as compared to the amount of hTRT in the sample. Suitable control polynucleotides for co-amplification reactions include DNA, RNA expressed from housekeeping genes, constitutively expressed genes, and in vitro synthesized RNAs or DNAs added to the reaction mixture. Endogenous control polynucleotides are those that are already present in the sample, while exogenous control polynucleotides are added to a sample, creating a “spiked” reaction. Illustrative control RNAs include β-actin RNA, GAPDH RNA, snRNAs, hTR, and endogenously expressed 28S rRNA (see Khan et al., 1992, <i>Neurosci. Lett. </i>147:114). Exogenous control polynucleotides include a synthetic AW106 cRNA, which may be synthesized as a sense strand from pAW106 by T7 polymerase. It will be appreciated that for the co-amplification method to be useful for quantitation, the control and target polynucleotides must typically both be amplified in a linear range. Detailed protocols for quantitative PCR may be found in PCR Protocols, A Guide to Methods and Applications, Innis et al., Academic Press, Inc. N.Y., (1990) and Ausubel et al., supra (Unit 15) and Diaco, R. (1995) <i>Practical Considerations for the Design of Quantitative PCR Assays</i>, in PCR Strategies, pg. 84-108, Innis et al. eds, Academic Press, New York.</p>
    <p>Depending on the sequence of the endogenous or exogenous standard, different primer sets may be used for the co-amplification reaction. In one method, called competitive amplification, quantitative PCR involves simultaneously co-amplifying a known quantity of a control sequence using the same primers used for amplification of the target nucleic acid (one pair of 2 primers). In an alternative embodiment, known as non-competitive competition, the control sequence and the target sequence (e.g., hTRT cDNA) are amplified using different primers (i.e., 2 pairs of 2 primers). In another alternative embodiment, called semi-competitive amplification, three primers are used, one of which is hTRT-specific, one of which is control specific, and one of which is capable of annealing to both the target and control sequences. Semi-competitive amplification is described in U.S. Pat. No. 5,629,154, which is incorporated herein by reference.</p>
    <p>3) Hybridization-Based Assays</p>
    <p>a) Generally</p>
    <p>A variety of methods for specific DNA and RNA measurement using nucleic acid hybridization techniques are known to those of skill in the art (see Sambrook et al., supra). Hybridization based assays refer to assays in which a probe nucleic acid is hybridized to a target nucleic acid. Usually the nucleic acid hybridization probes of the invention are entirely or substantially identical to a contiguous sequence of the hTRT gene or RNA sequence. Preferably, nucleic acid probes are at least about 10 bases, often at least about 20 bases, and sometimes at least about 200 bases or more in length. Methods of selecting nucleic acid probe sequences for use in nucleic acid hybridization are discussed in Sambrook et al., supra. In some formats, at least one of the target and probe is immobilized. The immobilized nucleic acid may be DNA, RNA, or another oligo- or poly-nucleotide, and may comprise natural or non-naturally occurring nucleotides, nucleotide analogs, or backbones. Such assays may be in any of several formats including: Southern, Northern, dot and slot blots, high-density polynucleotide or oligonucleotide arrays (e.g., GeneChips™ Affymetrix), dip sticks, pins, chips, or beads. All of these techniques are well known in the art and are the basis of many commercially available diagnostic kits. Hybridization techniques are generally described in Hames et al., ed., Nucleic Acid Hybridization, A Practical Approach IRL Press, (1985); Gall and Pardue <i>Proc. Natl. Acad. Sci., U.S.A., </i>63: 378-383 (1969); and John et al., <i>Nature, </i>223: 582-587 (1969).</p>
    <p>A variety of nucleic acid hybridization formats are known to those skilled in the art. For example, one common format is direct hybridization, in which a target nucleic acid is hybridized to a labeled, complementary probe. Typically, labeled nucleic acids are used for hybridization, with the label providing the detectable signal. One method for evaluating the presence, absence, or quantity of hTRT mRNA is carrying out a Northern transfer of RNA from a sample and hybridization of a labeled hTRT specific nucleic acid probe, as illustrated in Example 2. As was noted supra, hTRT mRNA, when present at all, is present in very low quantities in most cells. Therefore, when Northern hybridization is used, it will often be desirable to use an amplification step (or, alternatively, large amounts of starting RNA). A useful method for evaluating the presence, absence, or quantity of DNA encoding hTRT proteins in a sample involves a Southern transfer of DNA from a sample and hybridization of a labeled hTRT specific nucleic acid probe.</p>
    <p>Other common hybridization formats include sandwich assays and competition or displacement assays. Sandwich assays are commercially useful hybridization assays for detecting or isolating nucleic acid sequences. Such assays utilize a “capture” nucleic acid covalently immobilized to a solid support and a labeled “signal” nucleic acid in solution. The biological or clinical sample will provide the target nucleic acid. The “capture” nucleic acid and “signal” nucleic acid probe hybridize with the target nucleic acid to form a “sandwich” hybridization complex. To be effective, the signal nucleic acid cannot hybridize with the capture nucleic acid.</p>
    <p>b) Chip-Based and Slide-Based Assays</p>
    <p>The present invention also provides probe-based hybridization assays for hTRT gene products employing arrays of immobilized oligonucleotide or polynucleotides to which an hTRT nucleic acid can hybridize (i.e., to some, but usually not all or even most, of the immobilized oligo- or poly-nucleotides). High density oligonucleotide arrays or polynucleotide arrays provide a means for efficiently detecting the presence and characteristics (e.g., sequence) of a target nucleic acid (e.g., hTRT gene, mRNA, or cDNA). Techniques are known for producing arrays containing thousands of oligonucleotides complementary to defined sequences, at defined locations on a surface using photolithographic techniques for synthesis in situ (see, e.g., U.S. Pat. Nos. 5,578,832; 5,556,752; and 5,510,270; Fodor et al., 1991, <i>Science </i>251:767; Pease et al., 1994, <i>Proc. Natl. Acad. Sci. USA </i>91:5022; and Lockhart et al., 1996, <i>Nature Biotech </i>14:1675) or other methods for rapid synthesis and deposition of defined oligonucleotides (Blanchard et al., 1996, <i>Biosensors </i>&amp; <i>Bioelectronics </i>11:687). When these methods are used, oligonucleotides (e.g., 20-mers) of known sequence are synthesized directly on a surface such as a derivatized glass slide. Usually, the array produced is redundant, having several oligonucleotide probes on the chip specific for the hTRT polynucleotide to be detected.</p>
    <p>Combinations of oligonucleotide probes can be designed to detect alternatively spliced mRNAs, or to identify which of various hTRT alleles is expressed in a particular sample.</p>
    <p>In one illustrative embodiment, cDNA prepared by reverse transcription of total RNA from a test cell is amplified (e.g., using PCR). Typically the amplification product is labeled, e.g., by incorporation of a fluorescently labeled dNTP. The labeled cDNAs are then hybridized to a chip comprising oligonucleotide probes complementary to various subsequences of the hTRT gene. The positions of hybridization are determined (e.g., in accordance with the general methods of Shalon et al., 1996, <i>Genome Research </i>6:639 or Schena et al., 1996, <i>Genome Res. </i>6:639), and sequence (or other information) deduced from the hybridization pattern, by means well known in the art.</p>
    <p>In one embodiment, two cDNA samples, each labeled with a different fluorescent group, are hybridized to the same chip. The ratio of the hybridization of each labeled sample to sites complementary to the hTRT gene are then assayed. If both samples contain the same amount of hTRT mRNA, the ratio of the two fluors will be 1:1 (it will be appreciated that the signal from the fluors may need to be adjusted to account for any difference in the molar sensitivity of the fluors). In contrast, if one sample is from a healthy (or control) tissue and the second sample is from a cancerous tissue the fluor used in the second sample will predominate.</p>
    <p>c) In Situ Hybridization</p>
    <p>An alternative means for detecting expression of a gene encoding an hTRT protein is in situ hybridization. In situ hybridization assays are well known and are generally described in Angerer et al., Methods Enzymol., 152: 649-660 (1987) and Ausubel et al., supra. In an in situ hybridization assay, cells or tissue specimens are fixed to a solid support, typically in a permeablilized state, typically on a glass slide. The cells are then contacted with a hybridization solution at a moderate temperature to permit annealing of labeled nucleic acid probes (e.g., <sup>35</sup>S-labeled riboprobes, fluorescently labeled probes) completely or substantially complementary to hTRT. Free probe is removed by washing and/or nuclease digestion, and bound probe is visualized directly on the slide by autoradiography or an appropriate imaging techniques, as is known in the art.</p>
    <p>4) Specific Detection of Variants</p>
    <p>As noted supra and illustrated in the Examples (e.g., Example 9), amplification primers or probes can be selected to provide amplification products that span specific deletions, truncations, and insertions, thereby facilitating the detection of specific variants or abnormalities in the hTRT mRNA.</p>
    <p>One example of an hTRT variant gene product that may be detected is an hTRT RNA such as a product (SEQ ID NO:4) described supra and in Example 9. The biological function, if any, of the Δ182 variant(s) is not known; however, the truncated hTRT protein putatively encoded by the variant may be involved in regulation of telomerase activity, e.g., by assembling a non-functional telomerase RNP that titrates telomerase components. Alternatively, negative regulation of telomerase activity could be accomplished by directing hTRT pre-mRNA (nascent mRNA) processing in a manner leading to elimination of the full length mRNA and reducing hTRT mRNA levels and increasing Δ182 hTRT RNA levels. For these and other reasons, the ability to detect Δ182 variants is useful. In addition, it will sometimes be desirable, in samples in which two species of hTRT RNA are present (such as a Δ182 hTRT RNA and hTRT RNA encoding the full-length hTRT protein) to compare their relative and/or absolute abundance.</p>
    <p>The invention provides a variety of methods for detection of Δ182 variants. For example, amplification using primer pairs spanning the 182 basepair deletion will result in different sized products corresponding to the deleted and undeleted hTRT RNAs, if both are present, which can be distinguished on the basis of size (e.g., by gel electrophoresis). Examples of primer pairs useful for amplifying the region spanning the 182 bp deletion include TCP1.14 and TCP1.15 (primer set 1), or TCP1.25 and billTCP6 (primer set 2) (see Table 2). These primer pairs can be used individually or in a nested PCR experiment where primer set 1 is used first. It will also be apparent to one of skill that hybridization methods (e.g., Northern hybridization) or RNAse protection assays using an hTRT nucleic acid probe of the invention can be used to detect and distinguish hTRT RNA variants.</p>
    <p>Another suitable method entails PCR amplification (or the equivalent) using three primers. Analogous to the semi-competitive quantitative PCR method described in greater detail supra, one primer is specific to each of the hTRT RNA species (e.g., as illustrated in Table 4) and one primer is complementary to both species (e.g., TCP1.25 (2270-2288)). An example of a primer specific to SEQ ID NO:1 is one that anneals within the 182 nucleotide sequence (i.e., nucleotides 2345 to 2526 of SEQ ID NO:1), e.g., TCP1.73 (2465-2445). For example, a primer specific to SEQ ID NO:4 (a Δ182 variant) is one that anneals at nucleotides 2358 to 2339 of SEQ ID NO:4 (i.e., the site corresponding to the 182 nucleotide insertion in SEQ ID NO:1). The absolute abundance of the Δ182 hTRT mRNA species or its relative abundance compared to the species encoding the full-length hTRT protein can be analyzed for correlation to cell state (e.g., capacity for indefinite proliferation). It will be appreciated that numerous other primers or amplification or detection methods can be selected based on the present disclosure.</p>
    <heading>TABLE 4</heading> <heading>ILLUSTRATIVE PRIMERS</heading> <p>Δ182 species (e.g., SEQ ID NO:4) specific primer:</p>
    <p>5′-GGCACTGGACGTAGGACGTG-3 (SEQ ID NO:550)</p>
    <p>hTRT (SEQ ID NO:1) specific primer (TCP1.73):</p>
    <p>5′-CACTGCTGGCCTCATTCAGGG-3 (SEQ ID NO:445)</p>
    <p>Common (forward) primer (TCP1.25):</p>
    <p>5′-TACTGCGTGCGTCGGTATG-3′ (SEQ ID NO:399)</p>
    <p>Other variant hTRT genes or gene products that can be detected include those characterized by premature stop codons, deletions, substitutions or insertions. Deletions can be detected by the decreased size of the gene, mRNA transcript, or cDNA. Similarly, insertions can be detected by the increased size of the gene, mRNA transcript, or cDNA insertions and deletions could also cause shifts in the reading frame that lead to premature stop codons or longer open reading frames. Substitutions, deletions, and insertions can also be detected by probe hybridization. Alterations can also be detected by observing changes in the size of the variant hTRT polypeptide (e.g., by Western analysis) or by hybridization or specific amplification as appropriate. Alternatively, mutations can be determined by sequencing of the gene or gene product according to standard methods. In addition, and as noted above, amplification assays and hybridization probes can be selected to target particular abnormalities specifically. For example, nucleic acid probes or amplification primers can be selected that specifically hybridize to or amplify, respectively, the region encompassing the deletion, substitution, or insertion. Where the hTRT gene harbors such a mutation, the probe will either (1) fail to hybridize or the amplification reaction will fail to provide specific amplification or cause a change in the size of the amplification product or hybridization signal; or (2) the probe or amplification reaction encompasses the entire deletion or either end of the deletion (deletion junction); or (3) similarly, probes and amplification primers can be selected that specifically target point mutations or insertions.</p>
    <p>5) Detection of Mutant hTRT Alleles</p>
    <p>Mutations in the hTRT gene can be responsible for disease initiation or can contribute to a disease condition. Alterations of the genomic DNA of hTRT can affect levels of gene transcription, change amino acid residues in the hTRT protein, cause truncated hTRT polypeptides to be produced, alter pre-mRNA processing pathways (which can alter hTRT mRNA levels), and cause other consequences as well.</p>
    <p>Alterations of genomic DNA in non-hTRT loci can also affect expression of hTRT or telomerase by altering the enzymes or cellular processes that are responsible for regulating hTRT, hTR, and telomerase-associated protein expression and processing and RNP assembly and transport. Alterations which affect hTRT expression, processing, or RNP assembly could be important for cancer progression, for diseases of aging, for DNA damage diseases, and others.</p>
    <p>Detection of mutations in hTRT mRNA or its gene and gene control elements can be accomplished in accordance with the methods herein in multiple ways. Illustrative examples include the following: A technique termed primer screening can be employed; PCR primers are designed whose 3′ termini anneal to nucleotides in a sample DNA (or RNA) that are possibly mutated. If the DNA (or RNA) is amplified by the primers, then the 3′ termini matched the nucleotides in the gene; if the DNA is not amplified, then one or both termini did not match the nucleotides in the gene, indicating a mutation was present. Similar primer design can be used to assay for point mutations using the Ligase Chain Reaction (LCR, described supra). Restriction fragment length polymorphism, RFLP (Pourzand, C., Cerutti, P. (1993) Mutat. Res 288: 113-121), is another technique that can be applied in the present method. A Southern blot of human genomic DNA digested with various restriction enzymes is probed with an hTRT specific probe. Differences in the fragment number or sizes between the sample and a control indicate an alteration of the experimental sample, usually an insertion or deletion. Single strand conformation polymorphism, SSCP (Orrita, M., et al. (1989) PNAS USA 86:2766-70), is another technique that can be applied in the present method. SSCP is based on the differential migration of denatured wild-type and mutant single-stranded DNA (usually generated by PCR). Single-stranded DNA will take on a three-dimensional conformation that is sequence-specific. Sequence differences as small as a single base change can result in a mobility shift on a nondenaturing gel. SSCP is one of the most widely used mutation screening methods because of its simplicity. Denaturing Gradient Gel Electrophoresis, DGGE (Myers, R. M., Maniatis, T. and Lerman, L., (1987) Methods in Enzymology, 155: 501-527), is another technique that can be applied in the present method. DGGE identifies mutations based on the melting behavior of double-stranded DNA. Specialized denaturing electrophoresis equipment is utilized to observe the melting profile of experimental and control DNAs: a DNA containing a mutation will have a different mobility compared to the control in these gel systems. The examples discussed illustrate commonly employed methodology; many other techniques exist which are known by those skilled in the art and can be applied in accordance with the teachings herein.</p>
    <p>F. Karyotype Analysis</p>
    <p>The present invention further provides methods and reagents for karyotype or other chromosomal analysis using hTRT-sequence probes and/or detecting or locating hTRT gene sequences in chromosomes from a human patient, human cell line, or non-human cell. In one embodiment, amplification (i.e., change in copy number), deletion (i.e., partial deletion), insertion, substitution, or changes in the chromosomal location (e.g., translocation) of an hTRT gene may be correlated with the presence of a pathological condition or a predisposition to developing a pathological condition (e.g., cancer).</p>
    <p>It has been determined by the present inventors that, in normal human cells, the hTRT gene maps close to the telomere of chromosome 5p (see Example 5, infra). The closest STS marker is D5S678 (see FIG. <b>8</b>). The location can be used to identify markers that are closely linked to the hTRT gene. The markers can be used to identify YACs, STSs, cosmids, BACs, lambda or P1 phage, or other clones which contain hTRT genomic sequences or control elements. The markers or the gene location can be used to scan human tissue samples for alterations in the normal hTRT gene location, organization or sequence that is associated with the occurrence of a type of cancer or disease. This information can be used in a diagnostic or prognostic manner for the disease or cancer involved. Moreover, the nature of any alterations to the hTRT gene can be informative as to the nature by which cells become immortal. For instance, a translocation event could indicate that activation of hTRT expression occurs in some cases by replacing the hTRT promoter with another promoter which directs hTRT transcription in an inappropriate manner. Methods and reagents of the invention of this type can be used to inhibit hTRT activation. The location may also be useful for determining the nature of hTRT gene repression in normal somatic cells, for instance, whether the location is part of non-expressing heterochromatin. Nuclease hypersensitivity assays for distinguishing heterochromatin and euchromatin are described, for example, in Wu et al., 1979, <i>Cell </i>16:797; Groudine and Weintraub, 1982, <i>Cell </i>30:131 Gross and Garrard, 1988, <i>Ann. Rev. Biochem. </i>57:159.</p>
    <p>In one embodiment, alterations to the hTRT gene are identified by karyotype analysis, using any of a variety of methods known in the art. One useful technique is in situ hybridization (ISH). Typically, when in situ hybridization techniques are used for karyotype analysis, a detectable or detectably-labeled probe is hybridized to a chromosomal sample in situ to locate an hTRT gene sequence. Generally, ISH comprises one or more of the following steps: (1) fixation of the tissue, cell or other biological structure to be analyzed; (2) prehybridization treatment of the biological structure to increase accessibility of target DNA (e.g., denaturation with heat or alkali), and to reduce nonspecific binding (e.g., by blocking the hybridization capacity of repetitive sequences, e.g., using human genomic DNA); (3) hybridization of one or more nucleic acid probes (e.g., conventional nucleic acids, PNAs, or probes containing other nucleic acid analogs) to the nucleic acid in the biological structure or tissue; (4) posthybridization washes to remove nucleic acid fragments not bound in the hybridization; and, (5) detection of the hybridized nucleic acid fragments. The reagents used in each of these steps and conditions for their use vary depending on the particular application. It will be appreciated that these steps can be modified in a variety of ways well known to those of skill in the art.</p>
    <p>In one embodiment of ISH, the hTRT probe is labeled with a fluorescent label (fluorescent in situ hybridization; “FISH”). Typically, it is desirable to use dual color fluorescent in situ hybridization, in which two probes are utilized, each labeled by a different fluorescent dye. A test probe that hybridizes to the hTRT sequence of interest is labeled with one dye, and a control probe that hybridizes to a different region is labeled with a second dye. A nucleic acid that hybridizes to a stable portion of the chromosome of interest, such as the centromere region, can be used as the control probe. In this way, one can account for differences between efficiency of hybridization from sample to sample.</p>
    <p>The ISH methods for detecting chromosomal abnormalities (e.g., FISH) can be performed on nanogram quantities of the subject nucleic acids. Paraffin embedded normal tissue or tumor sections can be used, as can fresh or frozen material, tissues, or sections. Because FISH can be applied to limited material, touch preparations prepared from uncultured primary tumors can also be used (see, e.g., Kallioniemi et al., 1992, <i>Cytogenet. Cell Genet. </i>60:190). For instance, small biopsy tissue samples from tumors can be used for touch preparations (see, e.g., Kallioniemi et al., supra). Small numbers of cells obtained from aspiration biopsy or cells in bodily fluids (e.g., blood, urine, sputum and the like) can also be analyzed. For prenatal diagnosis, appropriate samples will include amniotic fluid, maternal blood, and the like. Useful hybridization protocols applicable to the methods and reagents disclosed here are described in Pinkel et al., 1988, <i>Proc. Natl. Acad. Sci. USA, </i>85:9138; EPO Pub. No. 430,402; Choo, ed., Methods in Molecular Biology Vol. 33: In Situ Hybridization Protocols, Humana Press, Totowa, N.J., (1994); and Kallioniemi et al., supra.</p>
    <p>Other techniques useful for karyotype analysis include, for example, techniques such as quantitative Southern blotting, quantitative PCR, or comparative genomic hybridization (Kallioniemi et al., 1992, <i>Science, </i>258:818), using the hTRT probes and primers of the invention which may be used to identify amplification, deletion, insertion, substitution or other rearrangement of hTRT sequences in chromosomes in a biological sample.</p>
    <p>G. TRT Polypeptide Assays</p>
    <p>1) Generally</p>
    <p>The present invention provides methods and reagents for detecting and quantitating hTRT polypeptides. These methods include analytical biochemical methods such as electrophoresis, mass spectroscopy, gel shift, capillary electrophoresis, chromatographic methods such as size exclusion chromatography, high performance liquid chromatography (HPLC), thin layer chromatography (TLC), hyperdiffusion chromatography, and the like, or various immunological methods such as fluid or gel precipitin reactions, immunodiffusion (single or double), immunoelectrophoresis, radioimmunoassay (RIA), enzyme-linked immunosorbent assays (ELISAs), immunofluorescent assays, western blotting, mass spectrometry, and others described below and apparent to those of skill in the art upon review of this disclosure.</p>
    <p>2) Electrophoretic Assays</p>
    <p>In one embodiment, the hTRT polypeptides are detected in an electrophoretic protein separation; in one aspect, a two-dimensional electrophoresis system is employed. Means of detecting proteins using electrophoretic techniques are well known to those of skill in the art (see generally, R. Scopes (1982) Protein Purification, Springer-Verlag, N.Y.; Deutscher, (1990) Methods in Enzymology Vol. 182: Guide to Protein Purification, Academic Press, Inc., N.Y.).</p>
    <p>In a related embodiment, a mobility shift assay (see, e.g., Ausubel et al., supra) is used. For example, labeled-hTR will associate with hTRT and migrate with altered mobility upon electrophoresis in a nondenaturing polyacrylamide gel or the like. Thus, for example, if an (optionally labeled) hTR probe or a (optionally labeled) telomerase primer is mixed with a sample containing hTRT, or coexpressed with hTRT (e.g., in a cell-free expression system) the presence of hTRT protein (or a polynucleotide encoding hTRT) in the sample will result in a detectable alteration of hTR mobility.</p>
    <p>3) Immunoassays</p>
    <p>a) Generally</p>
    <p>The present invention also provides methods for detection of hTRT polypeptides employing one or more antibody reagents of the invention (i.e., immunoassays). As used herein, an immunoassay is an assay that utilizes an antibody (as broadly defined herein and specifically includes fragments, chimeras and other binding agents) that specifically binds an hTRT polypeptide or epitope. Antibodies of the invention may be made by a variety of means well known to those of skill in the art, e.g., as described supra.</p>
    <p>A number of well established immunological binding assay formats suitable for the practice of the invention are known (see, e.g., U.S. Pat. Nos. 4,366,241; 4,376,110; 4,517,288; and 4,837,168). See, e.g., Methods in Cell Biology Volume 37: Antibodies in Cell Biology, Asai, ed. Academic Press, Inc. New York (1993); Basic and Clinical Immunology 7th Edition, Stites &amp; Terr, eds. (1991); Harlow and Lane, supra [e.g., Chapter 14], and Ausubel et al., suptra, [e.g., Chapter 11], each of which is incorporated by reference in its entirety and for all purposes. Typically, immunological binding assays (or immunoassays) utilize a “capture agent” to specifically bind to and, often, immobilize the analyte. In one embodiment, the capture agent is a moiety that specifically binds to an hTRT polypeptide or subsequence, such as an anti-hTRT antibody. In an alternative embodiment, the capture agent may bind an hTRT-associated protein or RNA under conditions in which the hTRT-associated molecule remains bound to the hTRT (such that if the hTRT-associated molecule is immobilized the hTRT protein is similarly immobilized). It will be understood that in assays in which an hTRT-associated molecule is captured the associated hTRT protein will usually be present and so can be detected, e.g., using an anti-hTRT antibody or the like. Immunoassays for detecting protein complexes are known in the art (see, e.g., Harlow and Lane, supra, at page 583).</p>
    <p>Usually the hTRT gene product being assayed is detected directly or indirectly using a detectable label. The particular label or detectable group used in the assay is usually not a critical aspect of the invention, so long as it does not significantly interfere with the specific binding of the antibody or antibodies used in the assay. The label may be covalently attached to the capture agent (e.g., an anti-TRT antibody), or may be attached to a third moiety, such as another antibody, that specifically binds to, e.g.,: the hTRT polypeptide (at a different epitope than recognized by the capture agent), the capture agent (e.g., an anti-(first antibody) immunoglobulin); an anti-TRT antibody; an antibody that binds an anti-TRT antibody; or, an antibody/telomerase complex (e.g., via binding to an associated molecule such as a telomerase-associated protein). Other proteins capable of binding an antibody used in the assay, such as protein A or protein G, may also be labeled. In some embodiments, it will be useful to use more than one labeled molecule (i.e., ones that can be distinguished from one another). In addition, when the target bound (e.g., immobilized) by the capture agent (e.g., anti-hTRT antibody) is a complex (i.e., a complex of hTRT and a TRT-associated protein, hTR, or other TRT associated molecule), a labeled antibody that recognizes the protein or RNA associated with the hTRT protein can be used. When the complex is a protein-nucleic acid complex (e.g., TRT-hTR), the reporter molecule can be a polynucleotide or other molecule (e.g., enzyme) that recognizes the RNA component of the complex.</p>
    <p>Some immunoassay formats do not require the use of labeled components. For instance, agglutination assays can be used to detect the presence of the target antibodies. In this case, antigen-coated particles are agglutinated by samples comprising the target antibodies. In this format, the components do not need to be labeled, and the presence of the target antibody can be detected by simple visual inspection.</p>
    <p>b) Non-Competitive Assay Formats</p>
    <p>The present invention provides methods and reagents for competitive and noncompetitive immunoassays for detecting hTRT polypeptides. Noncompetitive immunoassays are assays in which the amount of captured analyte (in this case hTRT) is directly measured. One such assay is a two-site, monoclonal-based immunoassay utilizing monoclonal antibodies reactive to two non-interfering epitopes on the hTRT protein. See, e.g., Maddox et al., 1983, <i>J. Exp. Med., </i>158:1211 for background information. In one preferred “sandwich” assay, the capture agent (e.g., an anti-TRT antibody) is bound directly to a solid substrate where it is immobilized. These immobilized antibodies then capture any hTRT protein present in the test sample. The hTRT thus immobilized can then be labeled, i.e., by binding to a second anti-hTRT antibody bearing a label. Alternatively, the second anti-hTRT antibody may lack a label, but be bound by a labeled third antibody specific to antibodies of the species from which the second antibody is derived. The second antibody alternatively can be modified with a detectable moiety, such as biotin, to which a third labeled molecule can specifically bind, such as enzyme-labeled streptavidin.</p>
    <p>c) Competitive Assay Formats</p>
    <p>In competitive assays, the amount of hTRT protein present in the sample is measured indirectly by measuring the amount of an added (exogenous) hTRT displaced (or competed away) from a capture agent (e.g., anti-TRT antibody) by the hTRT protein present in the sample. In one competitive assay, a known amount of labeled hTRT protein is added to the sample and the sample is then contacted with a capture agent (e.g., an antibody that specifically binds hTRT protein). The amount of exogenous (labeled) hTRT protein bound to the antibody is inversely proportional to the concentration of hTRT protein present in the sample. In one embodiment, the antibody is immobilized on a solid substrate. The amount of hTRT protein bound to the antibody may be determined either by measuring the amount of hTRT protein present in a TRT/antibody complex, or alternatively by measuring the amount of remaining uncomplexed TRT protein. The amount of hTRT protein may be detected by providing a labeled hTRT molecule.</p>
    <p>A hapten inhibition assay is another example of a competitive assay. In this assay hTRT protein is immobilized on a solid substrate. A known amount of anti-TRT antibody is added to the sample, and the sample is then contacted with the immobilized hTRT protein. In this case, the amount of anti-TRT antibody bound to the immobilized hTRT protein is inversely proportional to the amount of hTRT protein present in the sample. The amount of immobilized antibody may be detected by detecting either the immobilized fraction of antibody or the fraction of the antibody that remains in solution. In this aspect, detection may be direct, where the antibody is labeled, or indirect where the label is bound to a molecule that specifically binds to the antibody as described above.</p>
    <p>d) Other Assay Formats</p>
    <p>The invention also provides reagents and methods for detecting and quantifying the presence of hTRT in the sample by using an immunoblot (Western blot) format. In this format, hTRT polypeptides in a sample are separated from other sample components by gel electrophoresis (e.g., on the basis of molecular weight), the separated proteins are transferred to a suitable solid support (such as a nitrocellulose filter, a nylon filter, derivatized nylon filter, or the like), and the support is incubated with anti-TRT antibodies of the invention. The anti-TRT antibodies specifically bind to hTRT or other TRT on the solid support. These antibodies may be directly labeled or alternatively may be subsequently detected using labeled antibodies (e.g., labeled sheep anti-mouse antibodies) or other labeling reagents that specifically bind to the anti-TRT antibody.</p>
    <p>Other assay formats include liposome immunoassays (LIA), which use liposomes designed to bind specific molecules (e.g., antibodies) and release encapsulated reagents or markers. The released chemicals can then be detected according to standard techniques (see, Monroe et al., 1986, <i>Amer. Clin. Prod. Rev. </i>5:34).</p>
    <p>As noted supra, assay formats using FACS (and equivalent instruments or methods) have advantages when measuring hTRT gene products in a heterogeneous sample (such as a biopsy sample containing both normal and malignant cells).</p>
    <p>e) Substrates, Solid Supports, Membranes, Filters</p>
    <p>As noted supra, depending upon the assay, various components, including the antigen, target antibody, or anti-hTRT antibody, may be bound to a solid surface or support (i.e., a substrate, membrane, or filter paper). Many methods for immobilizing biomolecules to a variety of solid surfaces are known in the art. For instance, the solid surface may be a membrane (e.g., nitrocellulose), a microtiter dish (e.g., PVC, polypropylene, or polystyrene), a test tube (glass or plastic), a dipstick (e.g. glass, PVC, polypropylene, polystyrene, latex, and the like), a microcentrifuge tube, or a glass or plastic bead. The desired component may be covalently bound or noncovalently attached through nonspecific bonding.</p>
    <p>A wide variety of organic and inorganic polymers, both natural and synthetic may be employed as the material for the solid surface. Illustrative polymers include polyethylene, polypropylene, poly(4-methylbutene), polystyrene, polymethacrylate, poly(ethylene terephthalate), rayon, nylon, poly(vinyl butyrate), polyvinylidene difluoride (PVDF), silicones, polyformaldehyde, cellulose, cellulose acetate, nitrocellulose, and the like. Other materials which may be employed, include paper, glasses, ceramics, metals, metalloids, semiconductive materials, cements or the like. In addition, substances that form gels, such as proteins (e.g., gelatins), lipopolysaccharides, silicates, agarose and polyacrylamides can be used. Polymers which form several aqueous phases, such as dextrans, polyalkylene glycols or surfactants, such as phospholipids, long chain (12-24 carbon atoms) alkyl ammonium salts and the like are also suitable. Where the solid surface is porous, various pore sizes may be employed depending upon the nature of the system.</p>
    <p>In preparing the surface, a plurality of different materials may be employed, particularly as laminates, to obtain various properties. For example, protein coatings, such as gelatin can be used to avoid non-specific binding, simplify covalent conjugation, enhance signal detection or the like.</p>
    <p>If covalent bonding between a compound and the surface is desired, the surface will usually be polyfunctional or be capable of being polyfunctionalized. Functional groups which may be present on the surface and used for linking can include carboxylic acids, aldehydes, amino groups, cyano groups, ethylenic groups, hydroxyl groups, mercapto groups and the like. The manner of linking a wide variety of compounds to various surfaces is well known and is amply illustrated in the literature. See, for example, <i>Immobilized Enzymes, </i>Ichiro Chibata, Halsted Press, New York, 1978, and Cuatrecasas (1970) <i>J. Biol. Chem. </i>245 3059).</p>
    <p>In addition to covalent bonding, various methods for noncovalently binding an assay component can be used. Noncovalent binding is typically nonspecific absorption of a compound to the surface.</p>
    <p>One of skill in the art will appreciate that it is often desirable to reduce non-specific binding in immunoassays. Particularly, where the assay involves an antigen or antibody immobilized on a solid substrate it is desirable to minimize the amount of non-specific binding to the substrate. Means of reducing such non-specific binding are well known to those of skill in the art. Typically, this involves coating the substrate with a proteinaceous composition. In particular, protein compositions such as bovine serum albumin (BSA), nonfat powdered milk, and gelatin are widely used with powdered milk sometimes preferred. Alternatively, the surface is designed such that it nonspecifically binds one component but does not significantly bind another. For example, a surface bearing a lectin such as Concanavalin A will bind a carbohydrate containing compound but not a labeled protein that lacks glycosylation. Various solid surfaces for use in noncovalent attachment of assay components are reviewed in U.S. Pat. Nos. 4,447,576 and 4,254,082.</p>
    <p>H) Assays for Anti-TRT Antibodies</p>
    <p>The present invention also provides reagents and assays for detecting hTRT-specific immunoglobulins. In one embodiment, immobilized hTRT (e.g., recombinant hTRT bound to a microassay plate well) is incubated with serum from a patient under conditions in which anti-hTRT antibodies, if present, bind the immobilized hTRT. After washing to remove nonspecifically bound immunoglobulin, bound serum antibodies can be detected, if they are present, by adding detectably labeled anti-(human Ig) antibodies (alternative embodiments and variations are well known to those of skill in the art; see, e.g., Harlow, supra, at Ch. 14). These assays are useful for detecting anti-hTRT antibodies in any source including animal or human serum or a carrier such as saline. In one embodiment, the assays are used to detect or monitor an immune response to hTRT proteins in a patient, particularly an autoimmune (e.g., anti-telomerase) response. Anti-hTRT antibodies may be present in the serum or other tissues or fluids from a patient suffering from an autoimmune disease or other condition.</p>
    <p>I) Assay Combinations</p>
    <p>The diagnostic and prognostic assays described herein can be carried out in various combinations and can also be carried out in conjunction with other diagnostic or prognostic tests. For example, when the present methods are used to detect the presence of cancer cells in patient sample, the presence of hTRT can be used to determine the stage of the disease, whether a particular tumor is likely to invade adjoining tissue or metastasize to a distant location, and whether a recurrence of the cancer is likely. Tests that may provide additional information include microscopic analysis of biopsy samples, detection of antigens (e.g., cell-surface markers) associated with tumorigenicity (e.g., using histocytochemistry, FACS, or the like), imaging methods (e.g., upon administration to a patient of labeled anti-tumor antibodies), telomerase activity assays, telomere length assays, hTR assays, or the like. Such combination tests can provide useful information regarding the progression of a disease.</p>
    <p>It will also be recognized that combinations of assays can provide useful information. For example, and as noted above, assays for hTRT mRNA can be combined with assays for hTR (human telomerase RNA) or telomerase activity (i.e., TRAP) assays to provide information about telomerase assembly and function.</p>
    <p>J) Kits</p>
    <p>The present invention also provides kits useful for the screening, monitoring, diagnosis and prognosis of patients with a telomerase-related condition, or for determination of the level of expression of hTRT in cells or cell lines. The kits include one or more reagents for determining the presence or absence of an hTRT gene product (RNA or protein) or for quantifying expression of the hTRT gene. Preferred reagents include nucleic acid primers and probes that specifically bind to the hTRT gene, RNA, cDNA, or portions thereof, along with proteins, peptides, antibodies, and control primers, probes, oligonucleotides, proteins, peptides and antibodies. Other materials, including enzymes (e.g., reverse transcriptases, DNA polymerases, ligases), buffers, reagents (labels, dNTPs), may be included.</p>
    <p>The kits may include alternatively, or in combination with any of the other components described herein, an antibody that specifically binds to hTRT polypeptides or subsequences thereof. The antibody can be monoclonal or polyclonal. The antibody can be conjugated to another moiety such as a label and/or it can be immobilized on a solid support (substrate). The kit(s) may also contain a second antibody for detection of hTRT polypeptide/antibody complexes or for detection of hybridized nucleic acid probes, as well as one or more hTRT peptides or proteins for use as control or other reagents.</p>
    <p>The antibody or hybridization probe may be free or immobilized on a solid support such as a test tube, a microtiter plate, a dipstick and the like. The kit may also contain instructional materials teaching the use of the antibody or hybridization probe in an assay for the detection of TRT. The kit may contain appropriate reagents for detection of labels, or for labeling positive and negative controls, washing solutions, dilution buffers and the like.</p>
    <p>In one embodiment, the kit includes a primer pair for amplifying hTRT mRNA. Such a kit may also include a probe for hTRT amplified DNA and/or a polymerase, buffer, dNTPs, and the like. In another, the kit comprises a probe, optionally a labeled probe. In another, the kit comprises an antibody.</p>
    <heading>X. Identification of Modulators of Telomerase Activity</heading> <p>A. Generally</p>
    <p>The invention provides compounds and treatments that modulate the activity or expression of a telomerase or telomerase component (e.g., hTRT protein). The invention also provides assays and screening methods (including high-throughput screens) for identification of compounds and treatments that modulate telomerase activity or expression. These modulators of telomerase activity and expression (hereinafter referred to as “modulators”) include telomerase agonists (which increase telomerase activity and/or expression) and telomerase antagonists (which decrease telomerase activity and/or expression).</p>
    <p>The modulators of the invention have a wide variety of uses. For example, it is contemplated that telomerase modulators will be effective therapeutic agents for treatment of human diseases. Screening for agonist activity and transcriptional or translational activators provides for compositions that increase telomerase activity in a cell (including a telomere dependent replicative capacity, or a “partial” telomerase activity). Such agonist compositions provide for methods of immortalizing otherwise normal untransformed cells, including cells which can express useful proteins. Such agonists can also provide for methods of controlling cellular senescence. Conversely, screening for antagonist activity provides for compositions that decrease telomere dependent replicative capacity, thereby mortalizing otherwise immortal cells, such as cancer cells. Screening for antagonist activity provides for compositions that decrease telomerase activity, thereby preventing unlimited cell division of cells exhibiting unregulated cell growth, such as cancer cells. Illustrative diseases and conditions that may be treated using modulators are listed herein, e.g., in Sections VIII and IX, supra. In general, the modulators of the invention can be used whenever it is desired to increase or decrease a telomerase activity in a cell or organism. Thus, in addition to use in treatment of disease, a modulator that increases hTRT expression levels can be used to produce a cultured human cell line having properties as generally described in Section VIII, supra, and various other uses that will be apparent to one of skill.</p>
    <p>A compound or treatment modulates “expression” of telomerase or a telomerase component when administration of the compound or treatment changes the rate or level of transcription of the gene encoding a telomerase component (e.g., the gene encoding hTRT mRNA), affects stability or post-transcriptional processing of RNA encoding a telomerase component (e.g., transport, splicing, polyadenylation, or other modification), affects translation, stability, post-translational processing or modification of an encoded protein (e.g., hTRT), or otherwise changes the level of functional (e.g., catalytically active) telomerase RNP. A compound or treatment affects a telomerase “activity” when administration of the compound or treatment changes a telomerase activity such as any activity described in Section IV(B), supra (e.g., including processive or non-processive telomerase catalytic activity; telomerase processivity; conventional reverse transcriptase activity; nucleolytic activity; primer or substrate binding activity; dNTP binding activity; RNA binding activity; telomerase RNP assembly; and protein binding activity). It will be appreciated that there is not necessarily a sharp delineation between changes in “activity” and changes in “expression,” and that these terms are used for ease of discussion and not for limitation. It will also be appreciated that the modulators of the invention should specifically affect telomerase activity or expression (e.g., without generally changing the expression of housekeeping proteins such as actin) rather than, for example, reducing expression of a telomerase component by nonspecific poisoning of a target cell.</p>
    <p>B. Assays for Identification of Telomerase Modulators</p>
    <p>The invention provides methods and reagents to screen for compositions or compounds capable of affecting expression of a telomerase or telomerase component, capable of modifying the DNA replicative capacity of telomerase, or otherwise modifying the ability of the telomerase enzyme and TRT protein to synthesize telomeric DNA (“full activity”). The invention also provides screens for modulators of any or all of hTRT's “partial activities.” Thus, the present invention provides assays that can be used to screen for agents that increase the activity of telomerase, for example, by causing hTRT protein or telomerase to be expressed in a cell in which it normally is not expressed or by increasing telomerase activity levels in telomerase positive cells.</p>
    <p>Telomerase or telomerase subunit proteins or their catalytic or immunogenic fragments or oligopeptides thereof, can be used for screening therapeutic compounds in any of a variety of drug screening techniques. The fragment employed in such a test may be free in solution, affixed to a solid support, borne on a cell surface, or located intracellularly. The formation of binding complexes, between telomerase or the subunit protein and the agent being tested, may be measured.</p>
    <p>In various embodiments, the invention includes methods for screening for antagonists that: bind to the enzyme's active site; inhibit the association of its RNA moiety, telomerase-associated proteins, nucleotides, or telomeric DNA to telomerase or hTRT protein; promote the disassociation of the enzyme complex; interfere with transcription of the telomerase RNA moiety (e.g., hTR); or inhibit any of the “partial activities” described herein. The invention provides methods for screening for compositions that inhibit the association of nucleic acid and/or telomerase-associated compositions with hTRT, such as the association of hTR with hTRT or the association of hTRT with the human homologs of p80 or p95 or another associated protein, or association of hTRT with a telomere or a nucleotide; screening for compositions that promote the disassociation or promote the association (i.e., assembly) of the enzyme complex, such as an antibody directed to hTR or hTRT; screening for agents that effect the processivity of the enzyme; and screening for nucleic acids and other compositions that bind to telomerase, such as a nucleic acid complementary to hTR. The invention further contemplates screening for compositions that increase or decrease the transcription of the hTRT gene and/or translation of the hTRT gene product. The invention also contemplates a method of screening for telomerase modulators in animals, in one embodiment, by reconstituting a telomerase activity, or an anti-telomerase activity, in an animal, such as a transgenic animal. The invention provides for in vivo assays systems that include “knockout” models, in which one or several units of the endogenous telomerase, telomerase RNA moiety and/or telomerase-associated proteins have been deleted or inhibited. The endogenous telomerase activity, full or partial, can remain or be absent. In one embodiment, an exogenous telomerase activity, full or partial, is reconstituted.</p>
    <p>In one embodiment of the invention, a variety of partial activity telomerase assays are provided to identify a variety of different classes of modulators of telomerase activity. The “partial activity” assays of the invention allow identification of classes of telomerase activity modulators that might otherwise not be detected in a “full activity” telomerase assay. One partial activity assay involves the non-processive activity of TRT and telomerase. The processive nature of telomerase is described by Morin (1989) <i>Cell </i>59:521-529; see also Prowse (1993) “Identification of a nonprocessive telomerase activity from mouse cells” <i>Proc. Natl. Acad. Sci. USA </i>90:1493-1497. Another partial activity assay of the invention exploits the “reverse-transcriptase-like” activity of telomerase. In these assays, one assays the reverse transcriptase activity of the hTRT protein. See Lingner (1997) “Reverse transcriptase motifs in the catalytic subunit of telomerase” <i>Science </i>276:561-567. Another partial activity assay of the invention exploits the “nucleolytic activity” of hTRT and telomerase, involving the enzyme's removing of at least one nucleotide, typically guanosine, from the 3′ strand of a primer. This nucleolytic activity has been observed in Tetrahymena telomerase by Collins (1993) “Tetrahymena telomerase catalyzes nucleolytic cleavage and nonprocessive elongation” <i>Genes Dev </i>7:1364-1376. Another partial activity assay of the invention involves analyzing hTRT's and telomerase's ability to bind nucleotides as part of its enzymatic processive DNA polymerization activity. Another partial activity assay of the invention involves analyzing hTRT's or telomerase's ability to bind its RNA moiety, i.e., hTR for human cells, used as a template for telomere synthesis. Additional partial activity assays of the invention involve analyzing hTRT's and telomerase's ability to bind chromosomes in vivo, or to bind oligonucleotide primers in vitro or in reconstituted systems, or to bind proteins associated with chromosomal structure (see, for an example of such a protein, Harrington (1995) <i>J Biol Chem </i>270: 8893-8901). Chromosomal structures which bind hTRT include, for example, telomeric repeat DNA, telomere proteins, histones, nuclear matrix protein, cell division/ cell cycle control proteins and the like.</p>
    <p>In one embodiment, an assay for identification of modulators comprises contacting one or more cells (i.e., “test cells”) with a test compound, and determining whether the test compound affects expression or activity of a telomerase (or telomerase component) in the cell. Usually this determination comprises comparing the activity or expression in the test cell compared to a similar cell or cells (i.e., control cells) that have not been contacted with the test compound. Alternatively, cell extracts may be used in place of intact cells. In a related embodiment, the test compound is administered to a multicellular organism (e.g., a plant or animal). The telomerase or telomerase component may be wholly endogenous to the cell or multicellular organism (i.e., encoded by naturally occurring endogenous genes), or may be a recombinant cell or transgenic organism comprising one or more recombinantly expressed telomerase components (e.g., hTRT, hTR, telomerase-associated proteins), or may have both endogenous and recombinant components. Thus, in one embodiment, telomerase-activity-modulators are administered to mortal cells. In another embodiment, telomerase-activity-modulators are administered to immortal cells. For example, antagonists of telomerase-mediated DNA replication can be identified by administering the putative inhibitory composition to a cell that is known to exhibit significant amounts of telomerase activity, such as cancer cells, and measuring whether a decrease in telomerase activity, telomere length, or proliferative capacity is observed, all of which are indicative of a compound with antagonist activity.</p>
    <p>In another embodiment, a modulator is identified by monitoring a change in a telomerase activity of a ribonucleoprotein complex (RNP) comprising a TRT (e.g., hTRT) and a template RNA (e.g., hTR), which RNP is reconstituted in vitro (e.g., as described in Example 7, infra).</p>
    <p>In yet another embodiment, the modulator is identified by monitoring a change in expression of a TRT gene product (e.g., RNA or protein) in a cell, animal, in vitro expression system, or other expression system.</p>
    <p>In still another embodiment, the modulator is identified by changing the expression of a reporter gene, such as that described in Example 15, whose expression is regulated, in whole or part, by a naturally occurring TRT regulatory element such as a promoter or enhancer. In a related embodiment, the ability of a test compound to bind to a telomerase component (e.g., hTRT), RNA, or gene regulatory sequence (e.g., the TRT gene promoter) is assayed.</p>
    <p>In another embodiment, the modulator is identified by observing changes in hTRT pre-mRNA processing, for example, alternatively spliced products, alternative poly-adenylation events, RNA cleavage, and the like. In a related embodiment the activity of the modulator can be observed by monitoring the production of variant hTRT polypeptides, some of which may possess dominant-negative telomerase regulation activity.</p>
    <p>Assay formats for identification of compounds that affect expression and activity of proteins are well known in the biotechnological and pharmaceutical industries, and numerous additional assays and variations of the illustrative assays provided supra will be apparent to those of skill.</p>
    <p>Changes in telomerase activity or expression can be measured by any suitable method. Changes in levels of expression of a telomerase component (e.g., hTRT protein) or precursor (e.g., hTRT mRNA) can be assayed using methods well known to those of skill, some of which are described hereinabove, e.g., in Section IX and including monitoring levels of TRT gene products (e.g., protein and RNAs) by hybridization (e.g., using the TRT probes and primers of the invention), immunoassays (e.g., using the anti-TRT antibodies of the invention), RNAse protection assays, amplification assays, or any other suitable detection means described herein or known in the art. Quantitating amounts of nucleic acid in a sample (e.g., evaluating levels of RNA, e.g., hTR or hTRT mRNA) is also useful in evaluating cis- or trans-transcriptional regulators.</p>
    <p>Similarly, changes in telomerase activity can be measured using methods such as those described herein (e.g., in Section IV(B), supra) or other assays of telomerase function. Quantitation of telomerase activity, when desired, may be carried out by any method, including those disclosed herein. Telomerase antagonists that can cause or accelerate loss of telomeric structure can be identified by monitoring and measuring their effect on telomerase activity in vivo, ex vivo, or in vitro, or by their effects on telomere length (as measured or detected through staining, use of tagged hybridization probes or other means) or, simply, by the inhibition of cell division of telomerase positive cancer cells (critical shortening of telomeres leads to a phenomenon termed “crisis” or M2 senescence (Shay, 1991) <i>Biochem. Biophys. Acta </i>1072:1-7), which cancer cells have bypassed by the activation of telomerase, but which, in the absence of telomerase, will lead to their senescence or death through chromosomal deletion and rearrangement). The in vivo human telomerase activity reconstitution provides for a method of screening for telomerase modulators in cells or animals from any origin. Such agonists can be identified in an activity assay of the invention, including measurements of changes in telomere length. Other examples of assays measuring telomerase activity in cells include assays for the accumulation or loss of telomere structure, the TRAP assay or a quantitative polymerase chain reaction assay.</p>
    <p>In one embodiment, the assays of the invention also include a method where the test compound produces a statistically significant decrease in the activity of hTRT as measured by the incorporation of a labeled nucleotide into a substrate compared to the relative amount of incorporated label in a parallel reaction lacking the test compound, thereby determining that the test compound is a telomerase inhibitor.</p>
    <p>The methods of the invention are amenable to adaptations from protocols described in the scientific and patent literature and known in the art. For example, when a telomerase or TRT protein of this invention is used to identify compositions which act as modulators of telomerase activities, large numbers of potentially useful molecules can be screened in a single test. The modulators can have an inhibitory (antagonist) or potentiating (agonist) effect on telomerase activity. For example, if a panel of 1,000 inhibitors is to be screened, all 1,000 inhibitors can potentially be placed into one microtiter well and tested simultaneously. If such an inhibitor is discovered, then the pool of 1,000 can be subdivided into 10 pools of 100 and the process repeated until an individual inhibitor is identified.</p>
    <p>In drug screening large numbers of compounds are examined for their ability to act as telomerase modulators, a process greatly accelerated by the techniques of high throughput screening. The assays for telomerase activity, full or partial, described herein may be adapted to be used in a high throughput technique. Those skilled in the art appreciate that there are numerous methods for accomplishing this purpose.</p>
    <p>Another technique for drug screening which may be applied for high throughput screening of compounds having suitable binding affinity to the telomerase or telomerase protein subunit is described in detail in “Determination of Amino Acid Sequence Antigenicity” by Geysen, (Geysen, WO Application 84/03564, published on Sep. 13, 1984, incorporated herein by reference). In summary, large numbers of different small peptide test compounds are synthesized on a solid substrate, such as plastic pins or some other surface. The peptide test compounds are reacted with fragments of telomerase or telomerase protein subunits and washed. Bound telomerase or telomerase protein subunit is then detected by methods well known in the art. Substantially purified telomerase or telomerase protein subunit can also be coated directly onto plates for use in the aforementioned drug screening techniques. Alternatively, non-neutralizing antibodies can be used to capture the peptide and immobilize it on a solid support.</p>
    <p>This invention also contemplates the use of competitive drug screening assays in which neutralizing antibodies capable of binding telomerase or subunit protein(s) specifically compete with a test compound for binding telomerase or the subunit protein. Antibodies can also be used to detect the presence of any peptide which shares one or more antigenic determinants with the telomerase or subunit protein.</p>
    <p>Additional methods for identifying modulators of a telomerase activity have been described in U.S. Pat. No. 5,645,986, which is incorporated herein by reference. It will be appreciated that the present invention provides improvements to previously known methods, in part by providing reagents such as hTRT polynucleotides, probes and primers, highly purified hTR, hTRT and telomerase, as well as anti-telomerase and anti-TRT antibodies, all of which may be used in assays, e.g., as controls, standards, binding or hybridization agents, or otherwise.</p>
    <p>It will be recognized that the recombinantly produced telomerase and TRT (e.g., hTRT) of the invention will be useful in assays for identification of modulators. The screening assay can utilize telomerase or hTRT derived by a full or partial reconstitution of telomerase activity, or by an augmentation of existing activity. The assay or screens provided by the invention can be used to test for the ability of telomerase to synthesize telomeric DNA or to test for any one or all or of the “partial activities” of hTRT and TRTs generally, as described above. The assay can incorporate ex vivo modification of cells which have been manipulated to express telomerase with or without its RNA moiety or associated proteins, and these can be re-implanted into an animal, which can be used for in vivo testing. Thus, this invention provides in vivo assays and transgenic animals useful therein. These in vivo assays systems can employ “knockout” cells, in which one or several units of the endogenous telomerase enzyme complex have been deleted or inhibited, as well as cells in which an exogenous or endogenous telomerase activity is reconstituted or activated.</p>
    <p>Telomerases and TRT proteins that have been modified in a site-specific manner (by site-specific mutation) to modify or delete any or all functions of the telomerase enzyme or the TRT protein can also be employed in the screens of the invention to discover therapeutic agents. For example, the TRT can be engineered to lose its ability to bind substrate DNA, to bind its RNA moiety (as hTR), to catalyze the addition of telomeric DNA, to bind deoxynucleotide substrate, to have nucleolytic activity, to bind telomere-associated proteins or chromosomal structures, and the like. The resulting “mutant proteins” or “muteens” can be used to identify compounds that specifically modulate one, several, or all functions or activities of the TRT protein or telomerase.</p>
    <p>C. Exemplary Telomerase Modulators</p>
    <p>1) Generally</p>
    <p>The test compounds referred to supra may be any of a large variety of compounds, both naturally occurring and synthetic, organic and inorganic, and including polymers (e.g., oligopeptides, polypeptides, oligonucleotides, and polynucleotides), small molecules, antibodies (as broadly defined herein), sugars, fatty acids, nucleotides and nucleotide analogs, analogs of naturally occurring structures (e.g., peptide mimetics, nucleic acid analogs, and the like), and numerous other compounds.</p>
    <p>The invention provides modulators of all types, without limitation to any particular mechanism of action. For illustrative purposes, examples of modulators include compounds or treatments that:</p>
    <p>(i) bind to the hTRT polypeptide (e.g., the active site of the enzyme) or other telomerase component, and affect a telomerase activity;</p>
    <p>(ii) inhibit or promote association, or inhibit or promote disassociation, of a telomerase component (e.g., hTRT or the hTRT-hTR RNP) with or from a telomerase-associated protein (e.g., including those described in Section IV(D), supra);</p>
    <p>(iii) inhibit or promote association, or inhibit or promote disassociation, of telomerase polypeptides (e.g., hTRT) with or from a telomerase RNA (e.g., hTR);</p>
    <p>(iv) inhibit or promote association, or inhibit or promote disassociation, of telomerase polypeptides (e.g., hTRT) with or from chromosomes (e.g., telomeres) or chromosomal DNA (e.g. telomeric DNA);</p>
    <p>(v) increase or decrease expression of a telomerase component gene product (e.g., products of the hTRT gene), including change the rate or level of transcription of the TRT gene, or translation, transport or stability of a gene product, or the like, by binding to the gene or gene product (e.g., by interacting with a factor (e.g., a transcription regulatory protein) that affects transcription of the hTRT gene or another telomerase component).</p>
    <p>2) Peptide Modulators</p>
    <p>Potential modulators of telomerase activity also include peptides (e.g., inhibitory (antagonist) and activator (agonist) peptide modulators). For example, oligopeptides with randomly generated sequences can be screened to discover peptide modulators (agonists or inhibitors) of telomerase activity. Such peptides can be used directly as drugs or to find the orientation or position of a functional group that can inhibit telomerase activity that, in turn, leads to design and testing of a small molecule inhibitor, or becomes the backbone for chemical modifications that increase pharmacological utility. Peptides can be structural mimetics, and one can use molecular modeling programs to design mimetics based on the characteristic secondary structure and/or tertiary structure of telomerase enzyme and hTRT protein. Such structural mimetics can also be used therapeutically, in vivo, as modulators of telomerase activity (agonists and antagonists). Structural mimetics can also be used as immunogens to elicit anti-telomerase or anti-TRT protein antibodies.</p>
    <p>3) Inhibitory Natural Compounds as Modulators of Telomerase Activity</p>
    <p>In addition, a large number of potentially useful activity-modifying compounds can be screened in extracts from natural products as a source material. Sources of such extracts can be from a large number of species of fungi, actinomyces, algae, insects, protozoa, plants, and bacteria. Those extracts showing inhibitory activity can then be analyzed to isolate the active molecule. See for example, Turner (1996) <i>J. Ethnopharmacol </i>51(1-3):39-43; Suh (1995) <i>Anticancer Res. </i>15:233-239.</p>
    <p>4) Inhibitory Oligonucleotides</p>
    <p>One particularly useful set of inhibitors provided by the present invention includes oligonucleotides which are able to either bind mRNA encoding hTRT protein or to the hTRT gene, in either case preventing or inhibiting the production of functional hTRT protein. Other oligonucleotides of the invention interact with telomerase's RNA moiety, such as hTR, or are able to prevent binding of telomerase or hTRT to its DNA target, or one telomerase component to another, or to a substrate. Such oligonucleotides can also bind the telomerase enzyme, hTRT protein, or both protein and RNA and inhibit a partial activity as described above (such as its processive activity, its reverse transcriptase activity, its nucleolytic activity, and the like). The association can be through sequence specific hybridization to another nucleic acid or by general binding, as in an aptamer, or both.</p>
    <p>Telomerase activity can be inhibited by targeting the hTRT mRNA with antisense oligonucleotides capable of binding the hTRT mRNA.</p>
    <p>Another useful class of inhibitors includes oligonucleotides which cause inactivation or cleavage of hTRT mRNA or hTR. That is, the oligonucleotide is chemically modified, or has enzyme activity, which causes such cleavage, such as is the case for a ribozyme, an EDTA-tethered oligonucleotide, or a covalently bound oligonucleotide, such as a psoralen or other cross-linking reagent bound oligonucleotide. As noted above, one may screen a pool of many different such oligonucleotides for those with the desired activity.</p>
    <p>Another useful class of inhibitors includes oligonucleotides which bind polypeptides. Double- or single-stranded DNA or double- or single-stranded RNA molecules that bind to specific polypeptides targets are called “aptamers.” The specific oligonucleotide-polypeptide association may be mediated by electrostatic interactions. For example, aptamers specifically bind to anion-binding exosites on thrombin, which physiologically binds to the polyanionic heparin (Bock (1992) <i>Nature </i>355:564-566). Because hTRT protein binds both hTR and its DNA substrate, and because the present invention provides hTRT and other TRT proteins in purified form in large quantities, those of skill in the art can readily screen for TRT-binding aptamers using the methods of the invention.</p>
    <p>Oligonucleotides (e.g., RNA oligonucleotides) that bind telomerase, hTRT, hTR, or portions thereof, can be generated using the techniques of SELEX (Tuerk, 1997, <i>Methods Mol Biol </i>67, 2190). In this technique a very large pool (106-109) of random sequence nucleic acids is bound to the target (e.g. hTRT) using conditions that cause a large amount of discrimination between molecules with high affinity and low affinity for binding the target. The bound molecules are separated from unbound, and the bound molecules are amplified by virtue of a specific nucleic acid sequence included at their termini and suitable amplification reagents. This process is reiterated several times until a relatively small number of molecules remain that possess high binding affinity for the target. These molecules can then be tested for their ability to modulate telomerase activity as described herein.</p>
    <p>Antagonists of telomerase-mediated DNA replication can also be based on inhibition of hTR (Norton (1996) <i>Nature Biotechnology </i>14:615-619) through complementary sequence recognition or cleavage, as through ribozymes.</p>
    <p>The inhibitory oligonucleotides of the invention can be transferred into the cell using a variety of techniques well known in the art. For example, oligonucleotides can be delivered into the cytoplasm without specific modification. Alternatively, they can be delivered by the use of liposomes which fuse with the cellular membrane or are endocytosed, i.e., by employing ligands attached to the liposome or directly to the oligonucleotide, that bind to surface membrane protein receptors of the cell resulting in endocytosis. Alternatively, the cells may be permeabilized to enhance transport of the oligonucleotides into the cell, without injuring the host cells. One can use a DNA binding protein, e.g., HBGF-1, known to transport an oligonucleotide into a cell.</p>
    <p>5) Inhibitory Ribozymes</p>
    <p>Ribozymes act by binding to a target RNA through the target RNA binding portion of a ribozyme which is held in close proximity to an enzymatic portion of the ribozyme that cleaves the target RNA. Thus, the ribozyme recognizes and binds a target RNA usually through complementary base-pairing, and once bound to the correct site, acts enzymatically to cleave and inactivate the target RNA. Cleavage of a target RNA in such a manner will destroy its ability to direct synthesis of an encoded protein if the cleavage occurs in the coding sequence. After a ribozyme has bound and cleaved its RNA target, it is typically released from that RNA and so can bind and cleave new targets repeatedly.</p>
    <p>6) Identifying Telomerase-Associated Proteins for Use as Modulators</p>
    <p>In one embodiment of the invention, telomerase is used to identify telomerase-associated proteins, i.e., telomerase accessory proteins which modulate or otherwise complement telomerase activity. As noted above, these proteins or fragments thereof can modulate function by causing the dissociation or preventing the association of the telomerase enzyme complex, preventing the assembly of the telomerase complex, preventing hTRT from binding to its nucleic acid complement or to its DNA template, preventing hTRT from binding nucleotides, or preventing, augmenting, or inhibiting any one, several or all of the partial activities of the telomerase enzyme or hTRT protein, as described above.</p>
    <p>One of skill in the art can use the methods of the invention to identify which portions (e.g., domains) of these telomerase-associating proteins contact telomerase. In one embodiment of the invention, these telomerase-associating proteins or fragments thereof are used as modulators of telomerase activity.</p>
    <p>7) Telomerase-Associated Proteins as Dominant Negative Mutants</p>
    <p>In one embodiment of the invention, telomerase-associated proteins are used as modulators of telomerase activity. Telomerase-associated proteins include chromosomal structures, such as histones, nuclear matrix proteins, cell division and cell cycle control proteins, and the like. Other telomerase-associated proteins which can be used as modulators for the purpose of the invention include the p80 and p95 proteins and their human homologs, such as TP1 and TRF-1 (Chong, 1995, <i>Science </i>270:1663-1667). In addition, fragments of these telomerase-associated proteins can be identified by the skilled artisan in accordance with the methods of the invention and used as modulators of telomerase activity.</p>
    <p>8) Dominant Negative Mutants</p>
    <p>Eight highly conserved motifs have been identified between TRTs of different non-human species, as described above (see also Lingner (1997) <i>Science </i>276:561-567). FIG. 4 shows a schematic of the human TRT amino acid sequence (from pGRN121) and RT motifs as compared to <i>S. pombe </i>Trt1p, Euplotes p123 and <i>S. cerevisiae </i>Est2p. The present invention provides recombinant and synthetic nucleic acids in which the codons for the conserved amino acid residues in each, alone or in conjunction with one or more additional codons, of all eight of these motifs has been a changed to each of the other codons. A variety of the resulting coding sequences express a non-functional hTRT. See, for instance, Example 16. Thus, the present invention provides, for example, a wide variety of “mutated” telomerase enzymes and TRT proteins which have a partial activity but not full activity of telomerase. For example, one such telomerase is able to bind telomeric structures, but not bind telomerase-associated RNA (i.e., hTR). If expressed at high enough levels, such a telomerase mutant can deplete a necessary telomerase component (e.g., hTR) and thereby function as an inhibitor of wild-type telomerase activity. A mutated telomerase acting in this manner is an antagonist or a so-called “dominant-negative” mutant.</p>
    <p>9) Antibodies</p>
    <p>In general, the antibodies of the invention can be used to identify, purify, or inhibit any or all activity of telomerase enzyme and hTRT protein. Antibodies can act as antagonists of telomerase activity in a variety of ways, for example, by preventing the telomerase complex or nucleotide from binding to its DNA substrates, by preventing the components of telomerase from forming an active complex, by maintaining a functional (telomerase complex) quaternary structure or by binding to one of the enzyme's active sites or other sites that have allosteric effects on activity (the different partial activities of telomerase are described in detail elsewhere in this specification).</p>
    <p>D) Modulator Synthesis</p>
    <p>It is contemplated that the telomerase modulators of the invention will be made using methods well known in the pharmaceutical arts, including combinatorial methods and rational drug design techniques.</p>
    <p>1) Combinatorial Chemistry Methodology</p>
    <p>The creation and simultaneous screening of large libraries of synthetic molecules can be carried out using well-known techniques in combinatorial chemistry, for example, see van Breemen (1997) <i>Anal Chem </i>69:2159-2164; Lam (1997) <i>Anticancer Drug Des </i>12:145-167 (1997).</p>
    <p>As noted above, combinatorial chemistry methodology can be used to create vast numbers of oligonucleotides (or other compounds) that can be rapidly screened for specific oligonucleotides (or compounds) that have appropriate binding affinities and specificities toward any target, such as the TRT proteins of the invention, can be utilized (for general background information Gold (1995) <i>J. of Biol. Chem. </i>270:13581-13584).</p>
    <p>2) Rational Drug Design</p>
    <p>Rational drug design involves an integrated set of methodologies that include structural analysis of target molecules, synthetic chemistries, and advanced computational tools. When used to design modulators, such as antagonists/inhibitors of protein targets, such as telomerase enzyme and hTRT protein, the objective of rational drug design is to understand a molecule's three-dimensional shape and chemistry. Rational drug design is aided by X-ray crystallographic data or NMR data, which can now be determined for the hTRT protein and telomerase enzyme in accordance with the methods and using the reagents provided by the invention. Calculations on electrostatics, hydrophobicities and solvent accessibility is also helpful. See, for example, Coldren (1997) <i>Proc. Natl. Acad. Sci. USA </i>94:6635-6640.</p>
    <p>E) Kits</p>
    <p>The invention also provides kits that can be used to aid in determining whether a test compound is a modulator of a TRT activity. The kit will typically include one or more of the following components: a substantially purified TRT polypeptide or polynucleotide (including probes and primers); a plasmid capable of expressing a TRT (e.g., hTRT) when introduced into a cell or cell-free expression system; a plasmid capable of expressing a TR (e.g., hTR) when introduced into a cell or cell-free expression system; cells or cell lines; a composition to detect a change in TRT activity; and, an instructional material teaching a means to detect and measure a change in the TRT activity, indicating that a change in the telomerase activity in the presence of the test compound is an indicator that the test compound modulates the telomerase activity, and one or more containers. The kit can also include means, such as TRAP assay reagents or reagents for a quantitative polymerase chain reaction assay, to measure a change in TRT activity. The kit may also include instructional material teaching a means to detect and measure a change in the TRT activity, indicating that a change in the telomerase activity in the presence of the test compound is an indicator that the test compound modulates the telomerase activity.</p>
    <heading>XI. Transgenic Organisms (Telomerase Knockout Cells and Animal Models)</heading> <p>The invention also provides transgenic non-human multicellular organisms (e.g., plants and non-human animals) or unicellular organisms (e.g., yeast) comprising an exogenous TRT gene sequence, which may be a coding sequence or a regulatory (e.g., promoter) sequence. In one embodiment, the organism expresses an exogenous TRT polypeptide, having a sequence of a human TRT protein. In a related embodiment, the organism also expresses a telomerase RNA component (e.g., hTR).</p>
    <p>The invention also provides unicellular and multicellular organisms (or cells therefrom) in which at least one gene encoding a telomerase component (e.g., TRT or TR) or telomerase-associated protein is mutated or deleted (i.e., in a coding or regulatory region) such that native telomerase is not expressed, or is expressed at reduced levels or with different activities when compared to wild-type cells or organisms. Such cells and organisms are often referred to as “gene knock-out” cells or organisms.</p>
    <p>The invention further provides cells and organisms in which an endogenous telomerase gene (e.g., murine TRT) is either present or optionally mutated or deleted and an exogenous telomerase gene or variant (e.g., human TRT) is introduced and expressed. Cells and organisms of this type will be useful, for example, as model systems for identifying modulators of hTRT activity or expression; determining the effects of mutations in telomerase component genes, and other uses such as determining the developmental timing and tissue location of telomerase activity (e.g., for assessing when to administer a telomerase modulator and for assessing any potential side effects).</p>
    <p>Examples of multicellular organisms include plants, insects, and nonhuman animals such as mice, rats, rabbits, monkeys, apes, pigs, and other nonhuman mammals. An example of a unicellular organism is a yeast.</p>
    <p>Methods for alteration or disruption of specific genes (e.g., endogenous TRT genes) are well known to those of skill, see, e.g., Baudin et al., 1993, <i>Nucl. Acids Res. </i>21:3329; Wach et al., 1994, <i>Yeast </i>10:1793; Rothstein, 1991, <i>Methods Enzymol. </i>194:281; Anderson, 1995, <i>Methods Cell Biol. </i>48:31; Pettitt et al., 1996, <i>Development </i>122:4149-4157; Ramirez-Solis et al., 1993, <i>Methods Enzymol. </i>225:855; and Thomas et al., 1987, <i>Cell </i>51:503, each of which is incorporated herein by reference in its entirety for all purposes.</p>
    <p>The “knockout” cells and animals of the invention include cells and animals in which one or several units of the endogenous telomerase enzyme complex have been deleted or inhibited. Reconstitution of telomerase activity will save the cell or animal from senescence or, for cancer cells, cell death caused by its inability to maintain telomeres. Methods of altering the expression of endogenous genes are well known to those of skill in the art. Typically, such methods involve altering or replacing all or a portion of the regulatory sequences controlling expression of the particular gene to be regulated. The regulatory sequences, e.g., the native promoter can be altered. The conventional technique for targeted mutation of genes involves placing a genomic DNA fragment containing the gene of interest into a vector, followed by cloning of the two genomic arms associated with the targeted gene around a selectable neomycin-resistance cassette in a vector containing thymidine kinase. This “knock-out” construct is then transfected into the appropriate host cell, i.e., a mouse embryonic stem (ES) cell, which is subsequently subjected to positive selection (using G418, for example, to select for neomycin-resistance) and negative selection (using, for example, FIAU to exclude cells lacking thymidine kinase), allowing the selection of cells which have undergone homologous recombination with the knockout vector. This approach leads to inactivation of the gene of interest. See, e.g., U.S. Pat. Nos. 5,464,764; 5,631,153; 5,487,992; and, 5,627,059.</p>
    <p>“Knocking out” expression of an endogenous gene can also be accomplished by the use of homologous recombination to introduce a heterologous nucleic acid into the regulatory sequences (e.g., promoter) of the gene of interest. To prevent expression of functional enzyme or product, simple mutations that either alter the reading frame or disrupt the promoter can be suitable. To up-regulate expression, a native promoter can be substituted with a heterologous promoter that induces higher levels of transcription. Also, “gene trap insertion” can be used to disrupt a host gene, and mouse ES cells can be used to produce knockout transgenic animals, as described for example, in Holzschu (1997) <i>Transgenic Res </i>6: 97-106.</p>
    <p>Altering the expression of endogenous genes by homologous recombination can also be accomplished by using nucleic acid sequences comprising the structural gene in question. Upstream sequences are utilized for targeting heterologous recombination constructs. Utilizing TRT structural gene sequence information, such as SEQ ID NO:1, one of skill in the art can create homologous recombination constructs with only routine experimentation. Homologous recombination to alter expression of endogenous genes is described in U.S. Pat. No. 5,272,071, and WO 91/09955, WO 93/09222, WO 96/29411, WO 95/31560, and WO 91/12650. Homologous recombination in mycobacteria is described by Azad (1996) Proc. Natl. Acad. Sci. USA 93:4787; Baulard (1996) J. Bacteriol.178:3091; and Pelicic (1996) Mol. Microbiol. 20:919. Homologous recombination in animals has been described by Moynahan (1996) Hum. Mol. Genet. 5:875, and in plants by Offringa (1990) EMBO J. 9:3077.</p>
    <heading>XII. GLOSSARY</heading> <p>The following terms are defined infra to provide additional guidance to one of skill in the practice of the invention: adjuvant, allele (&amp; allelic sequence), amino acids (including hydrophobic, polar, charged), conservative substitution, control elements (&amp; regulatory sequences), derivatized, detectable label, elevated level, epitope, favorable and unfavorable prognosis, fusion protein, gene product, hTR, immortal, immunogen and immunogenic, isolated, modulator, motif, nucleic acid (&amp; polynucleotide), oligonucleotides (&amp; oligomers), operably linked, polypeptide, probe (including nucleic acid probes &amp; antibody probes), recombinant, selection system, sequence, specific binding, stringent hybridization conditions (&amp; stringency), substantial identity (&amp; substantial similarity), substantially pure (&amp; substantially purified), telomerase-negative and telomerase-positive cells, telomerase catalytic activity, telomerase-related, and test compound.</p>
    <p>As used herein, the term “adjuvant” refers to its ordinary meaning of any substance that enhances the immune response to an antigen with which it is mixed. Adjuvants useful in the present invention include, but are not limited to, Freund's, mineral gels such as aluminum hydroxide, and surface active substances such as lysolecithin, pluronic polyols, polyanions, peptides, oil emulsions, keyhole limpet hemocyanin, and dinitrophenol. BCG (Bacillus Calmette-Guerin) and <i>Corynebacterium parvum </i>are potentially useful adjuvants.</p>
    <p>As used herein, the terms “allele” or “allelic sequence” refer to an alternative form of a nucleic acid sequence (i.e., a nucleic acid encoding hTRT protein). Alleles result from mutations (i.e., changes in the nucleic acid sequence), and generally produce altered and/or differently regulated mRNAs or polypeptides whose structure and/or function may or may not be altered. Common mutational changes that give rise to alleles are generally ascribed to natural deletions, additions, or substitutions of nucleotides that may or may not affect the encoded amino acids. Each of these types of changes may occur alone, in combination with the others, or one or more times within a given gene, chromosome or other cellular nucleic acid. Any given gene may have no, one or many allelic forms. As used herein, the term “allele” refers to either or both a gene or an mRNA transcribed from the gene.</p>
    <p>As used herein, “amino acids” are sometimes specified using the standard one letter code: Alanine (A), Serine (S), Threonine (T), Aspartic acid (D), Glutamic acid (E) Asparagine (N), Glutamine (Q), Arginine (R), Lysine (K), Isoleucine (I), Leucine (L), Methionine (M), Valine (V), Phenylalanine (F), Tyrosine (Y), Tryptophan (W), Proline (P), Glycine (G), Histidine (H), Cysteine (C). Synthetic and non-naturally occurring amino acid analogues (and/or peptide linkages) are included.</p>
    <p>As used herein, “Hydrophobic amino acids” refers to A, L, I, V, P, F, W, and M. As used herein, “polar amino acids” refers to G, S, T, Y, C, N, and Q. As used herein, “charged amino acids” refers to D, E, H, K, and R.</p>
    <p>As used herein, “conservative substitution”, when describing a protein refers to a change in the amino acid composition of the protein that does not substantially alter the protein's activity. Thus, “conservatively modified variations” of a particular amino acid sequence refers to amino acid substitutions of those amino acids that are not critical for protein activity or substitution of amino acids with other amino acids having similar properties (e.g., acidic, basic, positively or negatively charged, polar or non-polar, etc.) such that the substitutions of even critical amino acids does not substantially alter activity. Conservative substitution tables providing functionally similar amino acids are well known in the art. The following six groups each contain amino acids that are conservative substitutions for one another: 1) Alanine (A), Serine (S), Threonine (T); 2) Aspartic acid (D), Glutamic acid (E); 3) Asparagine (N), Glutamine (Q); 4) Arginine (R), Lysine (K); 5) Isoleucine (I), Leucine (L), Methionine (M), Valine (V); and 6) Phenylalanine (F), Tyrosine (Y), Tryptophan (W) (see also, Creighton (1984) <i>Proteins, </i>W.H. Freeman and Company). One of skill in the art will appreciate that the above-identified substitutions are not the only possible conservative substitutions. For example, one may regard all charged amino acids as conservative substitutions for each other whether they are positive or negative. In addition, individual substitutions, deletions or additions which alter, add or delete a single amino/acid or a small percentage of amino acids in an encoded sequence can also be “conservatively modified variations”. One can also make a “conservative substitution” in a recombinant protein by utilizing one or more codons that differ from the codons employed by the native or wild-type gene. In this instance, a conservative substitution also includes substituting a codon for an amino acid with a different codon for the same amino acid.</p>
    <p>As used herein, “control elements” or “regulatory sequences” include enhancers, promoters, transcription terminators, origins of replication, chromosomal integration sequences, 5′ and 3′ untranslated regions, with which proteins or other biomolecules interact to carry out transcription and translation. For eukaryotic cells, the control sequences will include a promoter and preferably an enhancer, e.g., derived from immunoglobulin genes, SV40, cytomegalovirus, and a polyadenylation sequence, and may include splice donor and acceptor sequences. Depending on the vector system and host utilized, any number of suitable transcription and translation elements, including constitutive and inducible promoters, may be used.</p>
    <p>As used herein, a “derivatized” polynucleotide, oligonucleotide, or nucleic acid refers to oligo- and polynucleotides that comprise a derivatized substituent. In some embodiments, the substituent is substantially non-interfering with respect to hybridization to complementary polynucleotides. Derivatized oligo- or polynucleotides that have been modified with appended chemical substituents (e.g., by modification of an already synthesized oligo- or poly-nucleotide, or by incorporation of a modified base or backbone analog during synthesis) may be introduced into a metabolically active eukaryotic cell to hybridize with an hTRT DNA, RNA, or protein where they produce an alteration or chemical modification to a local DNA, RNA, or protein. Alternatively, the derivatized oligo or polynucleotides may interact with and alter hTRT polypeptides, telomerase-associated proteins, or other factors that interact with hTRT DNA or hTRT gene products, or alter or modulate expression or function of hTRT DNA, RNA or protein. Illustrative attached chemical substituents include: europium (m) texaphyrin, cross-linking agents, psoralen, metal chelates (e.g., iron/EDTA chelate for iron catalyzed cleavage), topoisomerases, endonucleases, exonucleases, ligases, phosphodiesterases, photodynamic porphyrins, chemotherapeutic drugs (e.g., adriamycin, doxirubicin), intercalating agents, base-modification agents, immunoglobulin chains, and oligonucleotides. Iron/EDTA chelates are chemical substituents often used where local cleavage of a polynucleotide sequence is desired (Hertzberg et al., 1982, <i>J. Am. Chem. Soc. </i>104: 313; Hertzberg and Dervan, 1984, <i>Biochemistry </i>23: 3934; Taylor et al., 1984, <i>Tetrahedron </i>40: 457; Dervan, 1986, <i>Science </i>232: 464. Illustrative attachment chemistries include: direct linkage, e.g., via an appended reactive amino group (Corey and Schultz (1988) <i>Science </i>238: 1401, which is incorporated herein by reference) and other direct linkage chemistries, although streptavidin/biotin and digoxigenin/anti-digoxigenin antibody linkage methods can also be used. Methods for linking chemical substituents are provided in U.S. Pat. Nos. 5,135,720, 5,093,245, and 5,055,556, which are incorporated herein by reference. Other linkage chemistries may be used at the discretion of the practitioner.</p>
    <p>As used herein, a “detectable label” has the ordinary meaning in the art and refers to an atom (e.g., radionuclide), molecule (e.g., fluorescein), or complex, that is or can be used to detect (e.g., due to a physical or chemical property), indicate the presence of a molecule or to enable binding of another molecule to which it is covalently bound or otherwise associated. The term “label” also refers to covalently bound or otherwise associated molecules (e.g., a biomolecule such as an enzyme) that act on a substrate to produce a detectable atom, molecule or complex. Detectable labels suitable for use in the present invention include any composition detectable by spectroscopic, photochemical, biochemical, immunochemical, electrical, optical or chemical means. Labels useful in the present invention include biotin for staining with labeled streptavidin conjugate, magnetic beads (e.g., Dynabeads™), fluorescent dyes (e.g., fluorescein, Texas red, rhodamine, green fluorescent protein, enhanced green fluorescent protein, lissamine, phycoerythrin, Cy2, Cy3, Cy3.5, Cy5, Cy5.5, Cy7, FluorX [Amersham], SyBR Green I &amp; II [Molecular Probes], and the like), radiolabels (e.g.,<sup>3</sup>H, <sup>125</sup>I, <sup>35</sup>S, <sup>14</sup>C, or <sup>32</sup>P), enzymes (e.g., hydrolases, particularly phosphatases such as alkaline phosphatase, esterases and glycosidases, or oxidoreductases, particularly peroxidases such as horse radish peroxidase, and others commonly used in ELISAs), substrates, cofactors, inhibitors, chemiluminescent groups, chromogenic agents, and colorimetric labels such as colloidal gold or colored glass or plastic (e.g., polystyrene, polypropylene, latex, etc.) beads. Patents teaching the use of such labels include U.S. Pat. Nos. 3,817,837; 3,850,752; 3,939,350; 3,996,345; 4,277,437; 4,275,149; and 4,366,241. Means of detecting such labels are well known to those of skill in the art. Thus, for example, radiolabels and chemiluminescent labels may be detected using photographic film or scintillation counters, fluorescent markers may be detected using a photodetector to detect emitted light (e.g., as in fluorescence-activated cell sorting). Enzymatic labels are typically detected by providing the enzyme with a substrate and detecting the reaction product produced by the action of the enzyme on the substrate, and colorimetric labels are detected by simply visualizing the colored label. Thus, a label is any composition detectable by spectroscopic, photochemical, biochemical, immunochemical, electrical, optical or chemical means. The label may be coupled directly or indirectly to the desired component of the assay according to methods well known in the art. Non-radioactive labels are often attached by indirect means. Generally, a ligand molecule (e.g., biotin) is covalently bound to the molecule. The ligand then binds to an anti-ligand (e.g., streptavidin) molecule which is either inherently detectable or covalently bound to a signal generating system, such as a detectable enzyme, a fluorescent compound, or a chemiluminescent compound. A number of ligands and anti-ligands can be used. Where a ligand has a natural anti-ligand, for example, biotin, thyroxine, and cortisol, it can be used in conjunction with the labeled, naturally occurring anti-ligands. Alternatively, any haptenic or antigenic compound can be used in combination with an antibody. The molecules can also be conjugated directly to signal generating compounds, e.g., by conjugation with an enzyme or fluorophore. Means of detecting labels are well known to those of skill in the art. Thus, for example, where the label is a radioactive label, means for detection include a scintillation counter, photographic film as in autoradiography, or storage phosphor imaging. Where the label is a fluorescent label, it may be detected by exciting the fluorochrome with the appropriate wavelength of light and detecting the resulting fluorescence. The fluorescence may be detected visually, by means of photographic film, by the use of electronic detectors such as charge coupled devices (CCDs) or photomultipliers and the like. Similarly, enzymatic labels may be detected by providing the appropriate substrates for the enzyme and detecting the resulting reaction product. Also, simple colorimetric labels may be detected by observing the color associated with the label. It will be appreciated that when pairs of fluorophores are used in an assay, it is often preferred that the they have distinct emission patterns (wavelengths) so that they can be easily distinguished.</p>
    <p>The phrase “elevated level” refers to an amount of hTRT gene product (or other specified substance or activity) in a cell that is elevated or higher than the level in a reference standard, e.g., for diagnosis, the level in normal, telomerase-negative cells in an individual or in other individuals not suffering from the condition, and for prognosis, the level in tumor cells from a variety of grades or classes of, e.g., tumors.</p>
    <p>As used herein, the term “epitope” has its ordinary meaning of a site on an antigen recognized by an antibody. Epitopes are typically segments of amino acids which are a small portion of the whole protein. Epitopes may be conformational (i.e., discontinuous). That is, they may be formed from amino acids encoded by noncontiguous parts of a primary sequence that have been juxtaposed by protein folding.</p>
    <p>The terms “favorable prognosis” and “unfavorable prognosis” are known in the art. In the context of cancers, “favorable prognosis” means that there is a likelihood of tumor regression or longer survival times for patients with a favorable prognosis relative to those with unfavorable prognosis, whereas “unfavorable prognosis” means that the tumor is likely to be more aggressive, i.e., grow faster and/or metastasize, resulting in a poor outcome or a more rapid course of disease progression for the patient.</p>
    <p>As used herein, the term “fusion protein,” refers to a composite protein, i.e., a single contiguous amino acid sequence, made up of two (or more) distinct, heterologous polypeptides which are not normally fused together in a single amino acid sequence. Thus, a fusion protein may include a single amino acid sequence that contains two entirely distinct amino acid sequences or two similar or identical polypeptide sequences, provided that these sequences are not normally found together in the same configuration in a single amino acid sequence found in nature. Fusion proteins may generally be prepared using either recombinant nucleic acid methods, i.e., as a result of transcription and translation of a recombinant gene fusion product, which fusion comprises a segment encoding a polypeptide of the invention and a segment encoding a heterologous protein, or by chemical synthesis methods well known in the art. The non-hTRT region(s) of the fusion protein can be fused to the amino terminus of the hTRT polypeptide or the carboxyl terminus, or both or the non-hTRT region can be inserted into the interior of the protein sequence (by moiety inserting or by replacing amino acids) or combinations of the foregoing can be performed.</p>
    <p>As used herein, the term “gene product” refers to an RNA molecule transcribed from a gene, or a protein encoded by the gene or translated from the RNA.</p>
    <p>As used herein, “hTR” (human telomerase RNA) refers to the RNA component of human telomerase and any naturally occurring alleles and variants or recombinant variants. hTR is described in detail in U.S. Pat. No. 5,583,016 which is incorporated herein by reference in its entirety and for all purposes.</p>
    <p>As used herein, the term “immortal,” when referring to a cell, has its normal meaning in the telomerase art and refers to cells that have apparently unlimited replicative potential. Immortal can also refer to cells with increased proliferative capacity relative to their unmodified counterparts. Examples of immortal human cells are malignant tumor cells, germ line cells, and certain transformed human cell lines cultured in vitro (e.g., cells that have become immortal following transformation by viral oncogenes or otherwise). In contrast, most normal human somatic cells are mortal, i.e., have limited replicative potential and become senescent after a finite number of cell divisions.</p>
    <p>As used herein, the terms “immunogen” and “immunogenic” have their ordinary meaning in the art, i.e, an immunogen is a molecule, such as a protein or other antigen, that can elicit an adaptive immune response upon injection into a person or an animal.</p>
    <p>As used herein, “isolated,” when referring to a molecule or composition, such as, for example, an RNP (e.g., at least one protein and at least one RNA), means that the molecule or composition is separated from at least one other compound, such as a protein, other RNAs, or other contaminants with which it is associated in vivo or in its naturally occurring state. Thus, an RNP is considered isolated when the RNP has been isolated from any other component with which it is naturally associated, e.g., cell membrane, as in a cell extract. An isolated composition can, however, also be substantially pure.</p>
    <p>As used herein, “modulator” refers to any synthetic or natural compound or composition that can change in any way either or both the “full” or any “partial activity” of a telomerase reverse transcriptase (TRT). A modulator can be an agonist or an antagonist. A modulator can be any organic and inorganic compound; including, but not limited to, for example, small molecules, peptides, proteins, sugars, nucleic acids, fatty acids and the like.</p>
    <p>As used herein, “motif” refers to a sequence of contiguous amino acids (or to a nucleic acid sequence that encodes a sequence of contiguous amino acids) that defines a feature or structure in a protein that is common to or conserved in all proteins of a defined class or type. The motif or consensus sequence may include both conserved and non-conserved residues. The conserved residues in the motif sequence indicate that the conserved residue or class (i.e., hydrophobic, polar, non-polar, or other class) of residues is typically present at the indicated location in each protein (or gene or mRNA) of the class of proteins defined by the motif. Motifs can differ in accordance with the class of proteins. Thus, for example, the reverse transcriptase enzymes form a class of proteins than can be defined by one or more motifs, and this class includes telomerase enzymes. However, the telomerase enzymes can also be defined as the class of enzymes with motifs characteristic for that class. Those of skill recognize that the identification of a residue as a conserved residue in a motif does not mean that every member of the class defined by the motif has the indicated residue (or class of residues) at the indicated position, and that one or more members of the class may have a different residue at the conserved position.</p>
    <p>As used herein, the terms “nucleic acid” and “polynucleotide” are used interchangeably. Use of the term “polynucleotide” is not intended to exclude oligonucleotides (i.e., short polynucleotides) and can also refer to synthetic and/or non-naturally occurring nucleic acids (i.e., comprising nucleic acid analogues or modified backbone residues or linkages).</p>
    <p>As used herein “oligonucleotides” or “oligomers” refer to a nucleic acid sequence of approximately 7 nucleotides or greater, and as many as approximately 100 nucleotides, which can be used as a primer, probe or amplimer. Oligonucleotides are often between about 10 and about 50 nucleotides in length, more often between about 14 and about 35 nucleotides, very often between about 15 and about 25 nucleotides, and the terms oligonucleotides or oligomers can also refer to synthetic and/or non-naturally occurring nucleic acids (i.e., comprising nucleic acid analogues or modified backbone residues or linkages).</p>
    <p>As used herein, the term “operably linked,” refers to a functional relationship between two or more nucleic acid (e.g., DNA) segments: for example, a promoter or enhancer is operably linked to a coding sequence if it stimulates the transcription of the sequence in an appropriate host cell or other expression system. Generally, sequences that are operably linked are contiguous, and in the case of a signal sequence both contiguous and in reading phase. However, enhancers need not be located in close proximity to the coding sequences whose transcription they enhance.</p>
    <p>As used herein, the term “polypeptide” is used interchangeably herein with the term “protein,” and refers to a polymer composed of amino acid residues linked by amide linkages, including synthetic, naturally-occurring and non-naturally occurring analogs thereof (amino acids and linkages). Peptides are examples of polypeptides.</p>
    <p>As used herein, a “probe” refers to a molecule that specifically binds another molecule. One example of a probe is a “nucleic acid probe” that specifically binds (i.e., anneals or hybridizes) to a substantially complementary nucleic acid. Another example of a probe is an “antibody probe” that specifically binds to a corresponding antigen or epitope.</p>
    <p>As used herein, “recombinant” refers to a polynucleotide synthesized or otherwise manipulated in vitro (e.g., “recombinant polynucleotide”), to methods of using recombinant polynucleotides to produce gene products in cells or other biological systems, or to a polypeptide (“recombinant protein”) encoded by a recombinant polynucleotide.</p>
    <p>As used herein, a “selection system,” in the context of stably transformed cell lines, refers to a method for identifying and/or selecting cells containing a recombinant nucleic acid of interest. A large variety of selection systems are known for identification of transformed cells and are suitable for use with the present invention. For example, cells transformed by plasmids or other vectors can be selected by resistance to antibiotics conferred by genes contained on the plasmids, such as the well known amp, gpt, neo and hyg genes, or other genes such as the herpes simplex virus thymidine kinase (Wigler et al., Cell 11:223-32 [1977]) and adenine phosphoribosyltransferase (Lowy et al., Cell 22:817 [1980]) genes which can be employed in tk− or aprt− cells, respectively. Also, antimetabolite, antibiotic or herbicide resistance can be used as the basis for selection; for example, dhfr which confers resistance to methotrexate and is also useful for gene amplification (Wigler et al., <i>Proc. Natl. Acad. Sci., </i>77:3567 [1980]); npt, which confers resistance to the aminoglycosides neomycin and G-418 (Colbere-Garapin et al., <i>J. Mol. Biol., </i>150:1 [1981]) and als or pat, which confer resistance to chlorsulfuron and phosphinotricin acetyltransferase, respectively (Murry, in McGraw Hill Yearbook of Science and Technology, McGraw Hill, New York N.Y., pp 191-196, [1992]). Additional selectable genes have been described, for example, hygromycin resistance-conferring genes, trpB, which allows cells to utilize indole in place of tryptophan, or hisD, which allows cells to utilize histinol in place of histidine (Hartman and Mulligan, <i>Proc. Natl. Acad. Sci., </i>85:8047 [1988]). Recently, the use of visible markers has gained popularity with such markers as anthocyanins, beta-glucuronidase and its substrate, GUS, and luciferase and its substrate, luciferin, being widely used not only to identify transformants, but also to quantify the amount of transient or stable protein expression attributable to a specific vector system (Rhodes et al., <i>Meth. Mol. Biol., </i>55:121 [1995]).</p>
    <p>As used herein, the “sequence” of a gene (unless specifically stated otherwise), nucleic acid, protein, or peptide refers to the order of nucleotides in either or both strands of a double-stranded DNA molecule, e.g., the sequence of both the coding strand and its complement, or of a single-stranded nucleic acid molecule, or to the order of amino acids in a peptide or protein.</p>
    <p>As used herein, “specific binding” refers to the ability of one molecule, typically an antibody or polynucleotide, to contact and associate with another specific molecule even in the presence of many other diverse molecules. For example, a single-stranded polynucleotide can specifically bind to a single-stranded polynucleotide that is complementary in sequence, and an antibody specifically binds to (or “is specifically immunoreactive with”) its corresponding antigen.</p>
    <p>As used herein, “stringent hybridization conditions” or “stringency” refers to conditions in a range from about 5° C. to about 20° C. or 25° C. below the melting temperature (T<sub>m</sub>) of the target sequence and a probe with exact or nearly exact complementarity to the target. As used herein, the melting temperature is the temperature at which a population of double-stranded nucleic acid molecules becomes half-dissociated into single strands. Methods for calculating the T<sub>m </sub>of nucleic acids are well known in the art (see, e.g., Berger and Kimmel (1987) Methods in Enzymology, Vol. 152: Guide to Molecular Cloning Techniques, San Diego: Academic Press, Inc. and Sambrook et al. (1989) Molecular Cloning: A Laboratory Manual, 2nd Ed., Vols. 1-3, Cold Spring Harbor Laboratory hereinafter, “Sambrook”), both incorporated herein by reference). As indicated by standard references, a simple estimate of the T<sub>m </sub>value may be calculated by the equation: T<sub>m</sub>=81.5+0.41(% G+C), when a nucleic acid is in aqueous solution at 1 M NaCl (see e.g., Anderson and Young, <i>Quantitative Filter Hybridization </i>in Nucleic Acid Hybridization (1985)). Other references include more sophisticated computations which take structural as well as sequence characteristics into account for the calculation of T<sub>m</sub>. The melting temperature of a hybrid (and thus the conditions for stringent hybridization) is affected by various factors such as the length and nature (DNA, RNA, base composition) of the probe and nature of the target (DNA, RNA, base composition, present in solution or immobilized, and the like), and the concentration of salts and other components (e.g., the presence or absence of formamide, dextran sulfate, polyethylene glycol). The effects of these factors are well known and are discussed in standard references in the art, e.g., Sambrook, supra and Ausubel et al. supra. Typically, stringent hybridization conditions are salt concentrations less than about 1.0 M sodium ion, typically about 0.01 to 1.0 M sodium ion at pH 7.0 to 8.3, and temperatures at least about 30° C. for short probes (e.g., 10 to 50 nucleotides) and at least about 60° C. for long probes (e.g., greater than 50 nucleotides). As noted, stringent conditions may also be achieved with the addition of destabilizing agents such as formamide, in which case lower temperatures may be employed.</p>
    <p>As used herein, the term “substantial identity,” “substantial sequence identity,” or “substantial similarity” in the context of nucleic acids, refers to a measure of sequence similarity between two polynucleotides. Substantial sequence identity can be determined by hybridization under stringent conditions, by direct comparison, or other means. For example, two polynucleotides can be identified as having substantial sequence identity if they are capable of specifically hybridizing to each other under stringent hybridization conditions. Other degrees of sequence identity (e.g., less than “substantial”) can be characterized by hybridization under different conditions of stringency. Alternatively, substantial sequence identity can be described as a percentage identity between two nucleotide (or polypeptide) sequences. Two sequences are considered substantially identical when they are at least about 60% identical, preferably at least about 70% identical, or at least about 80% identical, or at least about 90% identical, or at least about 95% or 98% to 100% identical. Percentage sequence (nucleotide or amino acid) identity is typically calculated by determining the optimal alignment between two sequences and comparing the two sequences. For example an exogenous transcript used for protein expression can be described as having a certain percentage of identity or similarity compared to a reference sequence (e.g., the corresponding endogenous sequence). Optimal alignment of sequences may be conducted using the local homology algorithm of Smith and Waterman (1981) <i>Adv. Appl. Math. </i>2: 482, by the homology alignment algorithm of Needleman and Wunsch (1970) <i>J. Mol. Biol. </i>48: 443, by the search for similarity method of Pearson and Lipman (1988) <i>Proc. Natl. Acad. Sci. U.S.A. </i>85: 2444, by computerized implementations of these algorithms (GAP, BESTFIT, FASTA, and TFASTA in the Wisconsin Genetics Software Package, Genetics Computer Group, 575 Science Dr., Madison, Wis.), or by inspection. The best alignment (i.e., resulting in the highest percentage of identity) generated by the various methods is selected. Typically these algorithms compare the two sequences over a “comparison window” (usually at least 18 nucleotides in length) to identify and compare local regions of sequence similarity, thus allowing for small additions or deletions (i.e., gaps). Additions and deletions are typically 20 percent or less of the length of the sequence relative to the reference sequence, which does not comprise additions or deletions. It is sometimes desirable to describe sequence identity between two sequences in reference to a particular length or region (e.g., two sequences may be described as having at least 95% identity over a length of at least 500 basepairs). Usually the length will be at least about 50, 100, 200, 300, 400 or 500 basepairs, amino acids, or other residues. The percentage of sequence identity is calculated by comparing two optimally aligned sequences over the region of comparison, determining the number of positions at which the identical nucleic acid base (e.g., A, T, C, G, or U) occurs in both sequences to yield the number of matched positions, and determining the number (or percentage) of matched positions as compared to the total number of bases in the reference sequence or region of comparison. An additional algorithm that is suitable for determining sequence similarity is the BLAST algorithm, which is described in Altschul (1990) <i>J. Mol. Biol. </i>215: 403-410; and Shpaer (1996) <i>Genomics </i>38:179-191. Software for performing BLAST analyses is publicly available at the National Center for Biotechnology Information (http://www.ncbi.nlm.nih.gov/). This algorithm involves first identifying high scoring sequence pairs (HSPs) by identifying short words of length W in the query sequence that either match or satisfy some positive-valued threshold score T when aligned with a word of the same length in a database sequence. T is referred to as the neighborhood word score threshold (Altschul et al, supra.). These initial neighborhood word hits act as seeds for initiating searches to find longer HSPs containing them. The word hits are extended in both directions along each sequence for as far as the cumulative alignment score can be increased. Extension of the word hits in each direction are halted when: the cumulative alignment score falls off by the quantity X from its maximum achieved value; the cumulative score goes to zero or below, due to the accumulation of one or more negative-scoring residue alignments; or the end of either sequence is reached. The BLAST algorithm parameters W, T and X determine the sensitivity and speed of the alignment. The BLAST program uses as defaults a wordlength (W) of 11, the BLOSUM62 scoring matrix (see Henikoff (1992) <i>Proc. Natl. Acad. Sci. USA </i>89: 10915-10919) alignments (B) of 50, expectation (E) of 10, M=5, N=−4, and a comparison of both strands. The term BLAST refers to the BLAST algorithm which performs a statistical analysis of the similarity between two sequences; see, e.g., Karlin (1993) <i>Proc. Natl. Acad. Sci. USA </i>90:5873-5787. One measure of similarity provided by the BLAST algorithm is the smallest sum probability (P(N)), which provides an indication of the probability by which a match between two nucleotide or amino acid sequences would occur by chance. For example, a nucleic acid can be considered similar to a TRT nucleic acid if the smallest sum probability in a comparison of the test nucleic acid to an TRT nucleic acid is less than about 0.5, 0.2, 0.1, 0.01, or 0.001. Alternatively, another indication that two nucleic acid sequences are similar is that the polypeptide which the first nucleic acid encodes is immunologically cross reactive with the polypeptide encoded by the second nucleic acid. It will be recognized that homologous non-human TRT polynucleotides may have less that “substantial” nucleotide identity in certain regions, as the term “substantial identity” is defined herein. For example, Euplotes TRT is substantially less than about 60% identical to the hTRT polynucleotide of SEQ ID NO:1 in certain regions, although the two genes are homologs.</p>
    <p>As used herein, the terms “substantial identity,” “substantial sequence identity,” or “substantial similarity” in the context of a polypeptide, refers to a degree of similarity between two polypeptides in which a polypeptides comprises a sequence with at least 70% sequence identity to a reference sequence, or 80%, or 85% or up to 100% sequence identity to the reference sequence, or most preferably 90% identity over a comparison window of about 10-20 amino acid residues. Amino acid sequence similarity, or sequence identity, is determined by optimizing residue matches, if necessary, by introducing gaps as required. See Needleham et al. (1970) J. Mol. Biol. 48: 443-453; and Sankoff et al., 1983, <i>Time Warps, String Edits, and Macromolecules, The Theory and Practice of Sequence Comparison, </i>Chapter One, Addison-Wesley, Reading, Mass.; and software packages from IntelliGenetics, Mountain View, Calif., and the University of Wisconsin Genetics Computer Group, Madison, Wis. As will be apparent to one of skill, the terms “substantial identity”, “substantial similarity” and “substantial sequence identity” can be used interchangeably with regard to polypeptides or polynucleotides. It will be recognized that homologous non-human TRT polypeptides may have less that “substantial” sequence identity in certain regions, as the term “substantial identity” is defined herein. For example, Euplotes TRT protein is substantially less than about 60% identical to the hTRT polynucleotide of SEQ ID NO:2 in certain regions, although the two genes are homologs. In the context of TRT polypeptides from different species, for example, “significant homology” at the amino acid sequence means at least about 20% sequence identity in region of about 20 to about 40 residues, or at least about 40% sequence identity in region of at least about 20% sequence identity.</p>
    <p>As used herein, the term “substantially pure,” or “substantially purified,” when referring to a composition comprising a specified reagent, such as an antibody (e.g. an anti-hTRT antibody), means that the specified reagent is at least about 75%, or at least about 90%, or at least about 95%, or at least about 99% or more of the composition (not including, e.g., solvent or buffer). Thus, for example, a preferred immunoglobulin preparation of the invention that specifically binds an hTRT polypeptide is substantially purified.</p>
    <p>As used herein, a “telomerase negative” cell is one in which telomerase is not expressed, i.e., no telomerase catalytic activity can be detected using a conventional assay or a TRAP assay for telomerase catalytic activity. As used herein, a “telomerase positive” cell is a cell in which telomerase is expressed (i.e. telomerase activity can be detected).</p>
    <p>As used herein, a “telomerase-related” disease or condition is a disease or condition in a subject that is correlated with an abnormally high level of telomerase activity in cells of the individual, which can include any telomerase activity at all for most normal somatic cells, or which is correlated with a low level of telomerase activity that results in impairment of a normal cell function. Examples of telomerase-related conditions include, e.g., cancer (high telomerase activity in malignant cells) and infertility (low telomerase activity in germ-line cells).</p>
    <p>As used herein, “test compound” or “agent” refers to any synthetic or natural compound or composition. The term includes all organic and inorganic compounds; including, for example, small molecules, peptides, proteins, sugars, nucleic acids, fatty acids and the like.</p>
    <heading>XIII. EXAMPLES</heading> <p>The following examples are provided to illustrate the present invention, and not by way of limitation.</p>
    <p>In the following sections, the following abbreviations apply: eq (equivalents); M (Molar); μM (micromolar); N (Normal); mol (moles); mmol (millimoles); μmol (micromoles); nmol (nanomoles); g (grams); mg (milligrams); μg (micrograms); ng (nanograms); l or L (liters); ml (milliliters); μl (microliters); cm (centimeters); mm (millimeters); μm (micrometers); nm (nanometers); ° C. (degrees Centigrade); RPN (ribonucleoprotein); mreN (2′-O-methylribonucleotides); dNTP (deoxyribonucleotide); dH<sub>2</sub>O (distilled water); DDT (dithiothreitol); PMSF (phenylmethylsulfonyl fluoride); TE (10 mM Tris HCl, 1 mM EDTA, approximately pH 7.2); KGlu (potassium glutamate); SSC (salt and sodium citrate buffer); SDS (sodium dodecyl sulfate); PAGE (polyacrylamide gel electrophoresis); Novex (Novex, San Diego, Calif.); BioRad (Bio-Rad Laboratories, Hercules, Calif.); Pharmacia (Pharmacia Biotech, Piscataway, N.J.); Boehringer-Mannheim (Boehringer-Mannheim Corp., Concord, Calif.); Amersham (Amersham, Inc., Chicago, Ill.); Stratagene (Stratagene Cloning Systems, La Jolla, Calif.); NEB (New England Biolabs, Beverly, Mass.); Pierce (Pierce Chemical Co., Rockford, Ill.); Beckman (Beckman Instruments, Fullerton, Calif.); Lab Industries (Lab Industries, Inc., Berkeley, Calif.); Eppendorf (Eppendorf Scientific, Madison, Wis.); and Molecular Dynamics (Molecular Dynamics, Sunnyvale, Calif.).</p>
    <heading>Example 1</heading> <heading>Isolation of Telomerase Proteins and Clones</heading> <p>The following example details the isolation of telomerase proteins and clones from various organisms, including the euplotes p. 123, hTRT, TRT and <i>S. pombe </i>TRT telomerase cDNA clones.</p>
    <p>A. Background</p>
    <p>i) Introduction</p>
    <p>This section provides an overview of the purification and cloning of TRT genes, which is described in greater detail in subsequent sections of this Example. While telomerase RNA subunits have been identified in ciliates, yeast and mammals, protein subunits of the enzyme have not been identified as such prior to the present invention. Purification of telomerase from the ciliated protozoan <i>Euplotes aediculatus </i>yielded two proteins, termed p123 and p43 (see infra; Lingner (1996) <i>Proc. Natl. Acad. Sci. U.S.A. </i>93:10712). <i>Euplotes aediculatus </i>is a hypotrichous ciliate having a macronucleus containing about 8×10<sup>7 </sup>telomeres and about 3×10<sup>5 </sup>molecules of telomerase. After purification, the active telomerase complex had a molecular mass of about 230 kD, corresponding to a 66 kD RNA subunit and two proteins of about 123 kD and 43 kD (Lingner (1996) supra). Photocross-linking experiments indicated that the larger p123 protein was involved in specific binding of the telomeric DNA substrate (Lingner, (1996) supra).</p>
    <p>The p123 and p43 proteins were sequenced and the cDNA clones which encoded these proteins were isolated. These Euplotes sequences were found to be unrelated to the Tetrahymena telomerase-associated proteins p80 and p95. Sequence analysis of the Euplotes p123 revealed reverse transcriptase (RT) motifs. Furthermore, sequence analysis of the Euplotes p123 by comparison to other sequences revealed a yeast homolog, termed Est2 protein (Lingner (1997) <i>Science </i>276:561). Yeast Est2 had previously been shown to be essential for telomere maintenance in vivo (Lendvay (1996) <i>Genetics </i>144:1399) but had not been identified as a telomerase catalytic protein. Site-specific mutagenesis demonstrated that the RT motifs of yeast Est2 are essential for telomeric DNA synthesis in vivo and in vitro (Lingner (1997) supra).</p>
    <p>ii) Identifying and Characterizing <i>S. pombe </i>Telomerase</p>
    <p>PCR amplification of <i>S. pombe </i>DNA was carried out with degenerate sequence primers designed from the Euplotes p123 RT motifs as described below. Of the four prominent PCR products generated, a 120 base pair band encoded a peptide sequence homologous to p123 and Est2. This PCR product was used as a probe in colony hybridization and identified two overlapping clones from an <i>S. pombe </i>genomic library and three from an <i>S. pombe </i>cDNA library. Sequence analysis revealed that none of the three <i>S. pombe </i>cDNA clones was full length, so RT-PCR was used to obtain the sequences encoding the protein's N-terminus.</p>
    <p>Complete sequencing of these clones revealed a putative <i>S. pombe </i>telomerase RT gene, trt1. The complete nucleotide sequence of trt1 has been deposited in GenBank, accession number AF015783 (see FIG. <b>15</b>).</p>
    <p>To test <i>S. pombe </i>trt1 (as a catalytic subunit, two deletion constructs were created. Analysis of the sequence showed that trt1 encoded a basic protein with a predicted molecular mass of 116 kD. It was found that homology with p123 and Est2 was especially high in the seven reverse transcriptase motifs, underlined and designated as motifs 1, 2, A, B, C, D, and E (see FIG. <b>63</b>). An additional telomerase-specific motif, designated the T-motif, was also found. Fifteen introns, ranging in size from 36 to 71 base pairs, interrupted the coding sequence.</p>
    <p>To test <i>S. pombe </i>trt1 as a catalytic subunit, two deletion constructs were created. One removed only motifs B through D in the RT domains. The second removed 99% of the open reading frame.</p>
    <p>Haploid cells grown from <i>S. pombe </i>spores of both mutants showed progressive telomere shortening to the point where hybridization to telomeric repeats became almost undetectable. A trt1<sup>+</sup>/trt1<sup>−</sup> diploid was sporulated and the resulting tetrads were dissected and germinated on a yeast extract medium supplemented with amino acids (a YES plate, Alfa (1993) <i>Experiments with Fission Yeast, </i>Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y.). Colonies derived from each spore were grown at 32° C. for three days, and streaked successively to fresh YES plates every three days. A colony from each round was placed in six ml of YES liquid culture at 32° C. and grown to stationary phase. Genomic DNA was prepared. After digestion with ApaI, DNA was subjected to electrophoresis on a 2.3% agarose gel, stained with ethidium bromide to confirm approximately equal loading in each lane, then transferred to a nylon membrane and hybridized to a telomeric DNA probe.</p>
    <p>Senescence was indicated by the delayed onset of growth or failure to grow on agar (typically at the fourth streak-out after germination) and by colonies with increasingly ragged edges (colony morphology shown in FIG. 22C) and by increasingly high fractions of elongated cells (as shown in FIG. <b>22</b>D). Cells were plated on Minimal Medium (Alfa (1993) supra) with glutamic acid substituted for ammonium chloride for two days at 32° C. prior to photography.</p>
    <p>When individual enlarged cells were separated on the dissecting microscope, the majority were found to undergo no further division. The same telomerase negative (trt1<sup>−</sup>) cell population always contained normal-sized cells which continued to divide, but which frequently produced non-dividing progeny. The telomerase-negative survivors may use a recombinational mode of telomere maintenance as documented in budding yeast strains that have various telomere-replication genes deleted (Lendvay (1996) supra, Lundblad (1993) <i>Cell </i>73:347).</p>
    <p>iii) Identifying and Characterizing Human Telomerase</p>
    <p>An EST (expressed sequence tag) derived from human telomerase reverse transcriptase (hTRT) cDNA was identified by a BLAST search of the dbEST (expressed sequence tag) Genbank database using the Euplotes 123 kDa peptide and nucleic acid sequences, as well as the Schizosaccharomyces protein and corresponding cDNA (tez1) sequences. The EST, designated Genbank AA28196, is 389 nucleotides long and it corresponds to positions 1679 to 2076 of clone 712562 (FIG. <b>18</b>), was obtained from the I.M.A.G.E. Consortium (Human Genome Center, DOE, Lawrence Livermore National Laboratory, Livermore, Calif.). This clone was obtained from a cDNA library of germinal B cells derived by flow sorting of tonsil cells. Complete sequencing of this hTRT cDNA clone showed all eight telomerase RT (TRT) motifs. However, this hTRT clone did not encode a contiguous portion of a TRT because RT motifs B′, C, D, and E, were contained in a different open reading frame than the more N-terminal RT motifs. In addition, the distance between RT motifs A and B was substantially shorter than that of the three previously known (non-human) TRTs.</p>
    <p>To isolate a full length cDNA clone, a cDNA library derived form the human 293 cell line (described above) which expresses high levels of telomerase activity, was screened. A lambda cDNA library from the 293 cell line was partitioned into 25 pools containing about 200,000 plaques each. Each pool was screened by PCR with the primer pair 5′-CGGAAGAGTGTCTGGAGCAA-3′ (SEQ ID NO:551) and 5′-GGATGAAGCGGAGTCTGGA-3′ (SEQ ID NO:459). Six subpools of one positive primary pool were further screened by PCR using this same primer pair. For both the primary and the secondary subpool screening, hTRT was amplified for a total of 31 cycles at: 94° C., 45 seconds; 60° C., 45 seconds; and 72° C., 90 seconds. As a control, RNA of the house-keeping enzyme GAPDH was amplified using the primer pair 5′-CTCAGACACCATGGGGAAGGTGA-3′ (SEQ ID NO:552) and 5′-ATGATCTTGAGGCTGTTGTCATA-3′ (SEQ ID NO:553) for a total of 16 cycles at 94° C., 45 seconds; 55° C., 45 seconds; and 72° C., 90 seconds.</p>
    <p>One hTRT positive subpool from the secondary screening was then screened by plaque hybridization with a probe from the 5′ region of clone #712562. One phage was positively identified (designated Lambda phage 25-1.1, ATCC 209024, deposited May 12, 1997). It contained an approximately four kilobase insert, which was excised and subcloned into the EcoRI site of pBluescript II SK+ vector (Stratagene, San Diego, Calif.) as an EcoRI fragment. This cDNA clone-containing plasmid was designated pGRN121. The cDNA insert totals approximately 4 kilobasepairs. The complete nucleotide sequence of the human hTRT cDNA (pGRN121) has been deposited in Genbank (accession AF015950) and the plasmid has been deposited with the ATCC (ATCC 209016, deposited May 6, 1997).</p>
    <p>B. Growth of <i>Euplotes aediculatus </i> </p>
    <p>In this Example, cultures of <i>E. aediculatus </i>were obtained from Dr. David Prescott, MCDB, University of Colorado. Dr. Prescott originally isolated this culture from pond water, although this organism is also available from the ATCC (ATCC #30859). Cultures were grown as described by Swanton et al., (Swanton et al., Chromosoma 77:203 [1980]), under non-sterile conditions, in 15-liter glass containers containing Chlorogonium as a food source. Organisms were harvested from the cultures when the density reached approximately 10<sup>4 </sup>cells/ml.</p>
    <p>C. Preparation of Nuclear Extracts</p>
    <p>In this Example, nuclear extracts of <i>E. aediculatus </i>were prepared using the method of Lingner et al., (Lingner et al., Genes Develop., 8:1984 [1994]), with minor modifications, as indicated below. Briefly, cells grown as described in Part B were concentrated with 15 μm Nytex filters and cooled on ice. The cell pellet was resuspended in a final volume of 110 ml TMS/PMSF/spermidine phosphate buffer. The stock TMS/PMSF/spermidine phosphate buffer was prepared by adding 0.075 g spermidine phosphate (USB) and 0.75 ml PMSF (from 100 mM stock prepared in ethanol) to 150 ml TMS. TMS comprised 10 mM Tris-acetate, 10 mM MgCl<sub>2</sub>, 85.5752 g sucrose/liter, and 0.33297 g CaCl<sub>2</sub>/liter, pH 7.5.</p>
    <p>After resuspension in TMS/PMSF/spermidine phosphate buffer, 8.8 ml 10% NP-40 and 94.1 g sucrose were added and the mixture placed in a siliconized glass beaker with a stainless steel stirring rod attached to an overhead motor. The mixture was stirred until the cells were completely lysed (approximately 20 minutes). The mixture was then centrifuged for 10 minutes at 7500 rpm (8950×g), at 4EC, using a Beckman JS-13 swing-out rotor. The supernatant was removed and nuclei pellet was resuspended in TMS/PMSF/spermidine phosphate buffer, and centrifuged again, for 5 minutes at 7500 rpm (8950×g), at 4EC, using a Beckman JS-13 swing-out rotor.</p>
    <p>The supernatant was removed and the nuclei pellet was resuspended in a buffer comprised of 50 mM Tris-acetate, 10 mM MgCl<sub>2</sub>, 10% glycerol, 0.1% NP-40, 0.4 M KGlu, 0.5 mM PMSF, pH 7.5, at a volume of 0.5 ml buffer per 10 g of harvested cells. The resuspended nuclei were then dounced in a glass homogenizer with approximately 50 strokes, and then centrifuged for 25 minutes at 14,000 rpm at 4° C., in an Eppendorf centrifuge. The supernatant containing the nuclear extract was collected, frozen in liquid nitrogen, and stored at −80° C. until used.</p>
    <p>D. Purification of Telomerase</p>
    <p>In this Example, nuclear extracts prepared as described in Part C were used to purify <i>E. aediculatus </i>telomerase. In this purification protocol, telomerase was first enriched by chromatography on an Affi-Gel-heparin column, and then extensively purified by affinity purification with an antisense oligonucleotide. As the template region of telomerase RNA is accessible to hybridization in the telomerase RNP particle, an antisense oligonucleotide (i.e., the “affinity oligonucleotide”) was synthesized that was complementary to this template region as an affinity bait for the telomerase. A biotin residue was included at the 5′ end of the oligonucleotide to immobilize it to an avidin column.</p>
    <p>Following the binding of the telomerase to the oligonucleotide, and extensive washing, the telomerase was eluted by use of a displacement oligonucleotide. The affinity oligonucleotide included DNA bases that were not complementary to the telomerase RNA 5′ to the telomerase-specific sequence. As the displacement oligonucleotide was complementary to the affinity oligonucleotide for its entire length, it was able to form a more thermodynamically stable duplex than the telomerase bound to the affinity oligonucleotide. Thus, addition of the displacement oligonucleotide resulted in the elution of the telomerase from the column.</p>
    <p>The nuclear extracts prepared from 45 liter cultures were frozen until a total of 34 ml of nuclear extract was collected. This corresponded to 630 liters of culture (i.e., approximately 4×10<sup>9 </sup>cells). The nuclear extract was diluted with a buffer to 410 ml, to provide final concentrations of 20 mM Tris-acetate, 1 mM MgCl<sub>2</sub>, 0.1 mM EDTA, 33 mM KGlu, 10% (vol/vol) glycerol, 1 mM dithiothreitol (DTT), and 0.5 mM phenylmethylsulfonyl fluoride (PMSF), at a pH of 7.5.</p>
    <p>The diluted nuclear extract was applied to an Affi-Gel-heparin gel column (Bio-Rad), with a 230 ml bed volume and 5 cm diameter, equilibrated in the same buffer and eluted with a 2-liter gradient from 33 to 450 mM KGlu. The column was run at 4° C., at a flow rate of 1 column volume/hour. Fractions of 50 mls each were collected and assayed for telomerase activity as described in Part E. Telomerase was eluted from the column at approximately 170 mM KGlu. Fractions containing telomerase (approximately 440 ml) were pooled and adjusted to 20 mM Tris-acetate, 10 mM MgCl<sub>2</sub>, 1 mM EDTA, 300 mM KGlu, 10% glycerol, 1 mM DTT, and 1% Nonidet P-40. This buffer was designated as “WB.”</p>
    <p>To this preparation, 1.5 nmol of each of two competitor DNA oligonucleotides (5′-TAGACCTGTTAGTGTACATTTGAATTGAAGC-3′; SEQ ID NO:554 and 5′-TAGACCTGTTAGGTTGGATTTGTGGCATCA-3′; SEQ ID NO:552), 50 μg yeast RNA (Sigma), and 0.3 nmol of biotin-labeled telomerase-specific oligonucleotide (5′-biotin-TAGACCTGTTA-(rmeG)<sub>2</sub>-(rmeU)<sub>4</sub>-(rmeG)<sub>4</sub>-(rmeU)<sub>4</sub>-rmeG-3′; SEQ ID NO:556), were added per ml of the pool. The 2-O-methyribonucleotides of the telomerase specific oligonucleotides were complementary to the telomerase RNA; template region; the deoxyribonucleotides were not complementary. The inclusion of competitor, non-specific DNA oligonucleotides increased the efficiency of the purification, as the effects of nucleic acid binding proteins and other components in the mixture that would either bind to the affinity oligonucleotide or remove the telomerase from the mixture were minimized.</p>
    <p>This material was then added to Ultralink immobilized neutravidin plus (Pierce) column material, at a volume of 60 μl of suspension per ml of pool. The column material was pre-blocked twice for 15 minutes each blocking, with a preparation of WB containing 0.01% Nonidet P-40, 0.5 mg BSA, 0.5 mg/ml lysozyme, 0.05 mg/ml glycogen, and 0.1 mg/ml yeast RNA. The blocking was conducted at 4° C., using a rotating wheel to block the column material thoroughly. After the first blocking step, and before the second blocking step, the column material was centrifuged at 200×g for 2 minutes to pellet the matrix.</p>
    <p>The pool-column mixture was incubated for 8 minutes at 30° C., and then for an additional 2 hours at 4E° C., on a rotating wheel (approximately 10 rpm; Labindustries) to allow binding. The pool-column mixture was then centrifuged 200×g for 2 minutes, and the supernatant containing unbound material was removed. The pool-column mixture was then washed. This washing process included the steps of rinsing the pool-column mixture with WB at 4° C., washing the mixture for 15 minutes with WB at 4° C., rinsing with WB, washing for 5 minutes at 30° C., with WB containing 0.6 M KGlu, and no Nonidet P-40, washing 5 minutes at 25° C. with WB, and finally, rinsing again with WB. The volume remaining after the final wash was kept small, in order to yield a ratio of buffer to column material of approximately 1:1.</p>
    <p>Telomerase was eluted from the column material by adding 1 nmol of displacement deoxyoligonucleotide (5′-CA<sub>4</sub>C<sub>4</sub>A<sub>4</sub>C<sub>2</sub>TA<sub>2</sub>CAG<sub>2</sub>TCTA-3′; SEQ ID NO:557), per ml of column material and incubating at 25° C. for 30 minutes. The material was centrifuged for 2 minutes at 14,000 rpm in a microcentrifuge (Eppendorf), and the eluate collected. The elution procedure was repeated twice more, using fresh displacement oligonucleotide each time. As mentioned above, because the displacement oligonucleotide was complementary to the affinity oligonucleotide, it formed a more thermodynamically stable complex with the affinity oligonucleotide than P-40. Thus, addition of the displacement oligonucleotide to an affinity-bound telomerase resulted in efficient elution of telomerase under native conditions. The telomerase appeared to be approximately 50% pure at this stage, as judged by analysis on a protein gel. The affinity purification of telomerase and elution with a displacement oligonucleotide is shown in FIG. 26 (panels A and B, respectively). In this Figure, the 2′-O-methyl sugars of the affinity oligonucleotide are indicated by the bold line. The black and shaded oval shapes in this figure are intended to represent graphically the protein subunits of the present invention.</p>
    <p>The protein concentrations of the extract and material obtained following Affi-Gel-heparin column chromatography were determined using the method of Bradford (Bradford, Anal. Biochem., 72:248 [1976]), using BSA as the standard. Only a fraction of the telomerase preparation was further purified on a glycerol gradient.</p>
    <p>The sedimentation coefficient of telomerase was determined by glycerol gradient centrifugation, as described in Part I.</p>
    <p>Table 5 below is a purification table for telomerase purified according to the methods of this Example. The telomerase was enriched 12-fold in nuclear extracts, as compared to whole cell extracts, with a recovery of 80%; 85% of telomerase was solubilized from nuclei upon extraction.</p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 5</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Purification of Telomerase</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="6"> <colspec colname="1" colwidth="28pt" align="left"> </colspec> <colspec colname="2" colwidth="35pt" align="center"> </colspec> <colspec colname="3" colwidth="35pt" align="center"> </colspec> <colspec colname="4" colwidth="42pt" align="center"> </colspec> <colspec colname="5" colwidth="35pt" align="center"> </colspec> <colspec colname="6" colwidth="42pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">Telomer-</td>
                <td class="description-td">Telomerase/</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Protein</td>
                <td class="description-td">ase (pmol</td>
                <td class="description-td">Protein/pmol</td>
                <td class="description-td">Recovery</td>
                <td class="description-td">Purification</td>
              </tr> <tr class="description-tr"> <td class="description-td">Fraction</td>
                <td class="description-td">(mg)</td>
                <td class="description-td">of RNP)</td>
                <td class="description-td">of RNP/mg</td>
                <td class="description-td">(%)</td>
                <td class="description-td">Factor</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="6" align="center" rowsep="1" class="description-td" colspan="6"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="6"> <colspec colname="1" colwidth="28pt" align="left"> </colspec> <colspec colname="2" colwidth="35pt" align="char" char="."> </colspec> <colspec colname="3" colwidth="35pt" align="center"> </colspec> <colspec colname="4" colwidth="42pt" align="char" char="."> </colspec> <colspec colname="5" colwidth="35pt" align="char" char="."> </colspec> <colspec colname="6" colwidth="42pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Nuclear</td>
                <td class="description-td">2020</td>
                <td class="description-td">1720</td>
                <td class="description-td">0.9</td>
                <td class="description-td">100</td>
                <td class="description-td">  1</td>
              </tr> <tr class="description-tr"> <td class="description-td">Extract</td>
              </tr> <tr class="description-tr"> <td class="description-td">Heparin</td>
                <td class="description-td">125</td>
                <td class="description-td">1040</td>
                <td class="description-td">8.3</td>
                <td class="description-td">60</td>
                <td class="description-td"> 10</td>
              </tr> <tr class="description-tr"> <td class="description-td">Affinity</td>
                <td class="description-td">0.3**</td>
                <td class="description-td"> 680</td>
                <td class="description-td">2270</td>
                <td class="description-td">40</td>
                <td class="description-td">2670</td>
              </tr> <tr class="description-tr"> <td class="description-td">Glyc-</td>
                <td class="description-td">NA*</td>
                <td class="description-td">NA*</td>
                <td class="description-td">NA*</td>
                <td class="description-td">25</td>
                <td class="description-td">NA*</td>
              </tr> <tr class="description-tr"> <td class="description-td">erol</td>
              </tr> <tr class="description-tr"> <td class="description-td">Gra-</td>
              </tr> <tr class="description-tr"> <td class="description-td">dient</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="6" align="center" rowsep="1" class="description-td" colspan="6"> </td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="6" align="left" class="description-td" colspan="6">*NA = Not available </td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="6" align="left" class="description-td" colspan="6">**This value was calculated from the measured amount of telomerase (680 pmol), by assuming a purity of 50% (based on a protein gel). </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p>E. Telomerase Activity</p>
    <p>At each step in the purification of telomerase, the preparation was analyzed by three separate assays, one of which was activity, as described in this Example. In general, telomerase assays were done in 40 μl containing 0.003-0.3 μl of nuclear extract, 50 mM Tris-Cl (pH 7.5), 50 mM KGlu, 10 mM MgCl<sub>2</sub>, 1 mM DTT, 125 μM dTTP, 125 μM dGTP, and approximately 0.2 pmoles of 5′-<sup>32</sup>P-labelled oligonucleotide substrate (i.e., approximately 400,000 cpm). Oligonucleotide primers were heat-denatured prior to their addition to the reaction mixture. Reactions were assembled on ice and incubated for 30 minutes at 25EC. The reactions were stopped by addition of 200 μl of 10 mM Tris-Cl (pH 7.5), 15 mM EDTA, 0.6% SDS, and 0.05 mg/ml proteinase K, and incubated for at least 30 minutes at 45EC. After ethanol precipitation, the products were analyzed on denaturing 8% PAGE gels, as known in the art (See e.g., Sambrook et al., 1989).</p>
    <p>F. Quantitation of Telomerase Activity</p>
    <p>In this Example, quantitation of telomerase activity through the purification procedure is described. Quantitation was accomplished by assaying the elongation of oligonucleotide primers in the presence of dGTP and [α-<sup>32</sup>P]dTTP. Briefly, 1 μM 5′-(G<sub>4</sub>T<sub>4</sub>)<sub>2</sub>-3′ oligonucleotide was extended in a 20 μl reaction mixture in the presence of 2 μl of [α-<sup>32</sup>P]dTTP (10 mCi/ml, 400 Ci/mmol; 1 Ci=37 GBq), and 125 μM dGTP as described (Lingner et al., Genes Develop., 8:1984 [1994]) and loaded onto an 8% PAGE sequencing gel as described.</p>
    <p>The results of this study are shown in FIG. <b>28</b>. In lane 1, there is no telomerase present (i.e., a negative control); lanes 2, 5, 8, and 11 contained 0.14 fmol telomerase; lanes 3, 6, 9, and 12 contained 0.42 fmol telomerase; and lanes 4, 7, 10, and 13 contained 1.3 fmol telomerase. Activity was quantitation using a PhosphorImager (Molecular Dynamics) using the manufacturer's instructions. It was determined that under these conditions, 1 fmol of affinity-purified telomerase incorporated 21 fmol of dTTP in 30 minutes.</p>
    <p>As shown in FIG. 28, the specific activity of the telomerase did not change significantly through the purification procedure. Affinity-purified telomerase was fully active. However, it was determined that at high concentrations, an inhibitory activity was detected and the activity of crude extracts was not linear. Thus, in the assay shown in FIG. <b>28</b>, the crude extract was diluted 700-7000-fold. Upon purification, this inhibitory activity was removed and no inhibitory effect was detected in the purified telomerase preparations, even at high enzyme concentrations.</p>
    <p>G. Gel Electrophoresis and Northern Blots</p>
    <p>As stated in Part E, at each step in the purification of telomerase, the preparation was analyzed by three separate assays. This Example describes the gel electrophoresis and blotting procedures used to quantify telomerase RNA present in fractions and analyze the integrity of the telomerase ribonucleoprotein particle.</p>
    <p>i) Denaturing Gels and Northern Blots</p>
    <p>In this Example, synthetic T7-transcribed telomerase RNA of known concentration served as the standard. Throughout this investigation, the RNA component was used as a measure of telomerase.</p>
    <p>A construct for phage T7 RNA polymerase transcription of <i>E. aediculatus </i>telomerase RNA was produced, using (PCR). The telomerase RNA gene was amplified with primers that annealed to either end of the gene. The primer that annealed at the 5′ end also encoded a hammerhead ribozyme sequence to generate the natural 5′ end upon cleavage of the transcribed RNA, a T7-promoter sequence, and an EcoRI site for subcloning. The sequence of this 5′ primer was 5′-GCGGGAATTCTAATACGACTCACTATAGGGAAGAAACTCTGATGAGGCCGAAAGGCCGAAACTCCACGAAAGTGGAGTAAGTTTCTCGATAATTGATCTGTAG-3′ (SEQ ID NO:558). The 3′ primer included an EarI site for termination of transcription at the natural 3′ end, and a BamHI site for cloning. The sequence of this 3′ primer was 5′-CGGGGATCCTCTTCAAAAGATGAGAGGACAGCAAAC-3′ (SEQ ID NO:559). The PCR amplification product was cleaved with EcoRI and BamHI, and subcloned into the respective sites of pUC19 (NEB), to give “pEaT7.” The correctness of this insert was confirmed by DNA sequencing. T7 transcription was performed as described by Zaug et al., Biochemistry 33:14935 [1994], with EarI-linearized plasmid. RNA was gel-purified and the concentration was determined (an A<sub>260 </sub>of 1=40 μg/ml). This RNA was used as a standard to determine the telomerase RNA present in various preparations of telomerase.</p>
    <p>The signal of hybridization was proportional to the amount of telomerase RNA, and the derived RNA concentrations were consistent with, but slightly higher than those obtained by native gel electrophoresis. Comparison of the amount of whole telomerase RNA in whole cell RNA to serial dilutions of known T7 RNA transcript concentrations indicated that each <i>E. aediculatus </i>cell contained approximately 300,000 telomerase molecules.</p>
    <p>Visualization of the telomerase was accomplished by Northern blot hybridization to its RNA component, using methods as described (Linger et al., Genes Develop., 8:1984 [1994]). Briefly, RNA (less than or equal to 0.5 μg/lane) was resolved on an 8% PAGE and electroblotted onto a Hybond-N membrane (Amersham), as known in the art (see e.g., Sambrook et al., 1989). The blot was hybridized overnight in 10 ml of 4×SSC, 10×Denhardt's solution, 0.1% SDS, and 50 μg/ml denatured herring sperm DNA. After pre-hybridizing for 3 hours, 2×10<sup>6 </sup>cpm probe/ml hybridization solution was added. The randomly labelled probe was a PCR-product that covered the entire telomerase RNA gene. The blot was washed with several buffer changes for 30 minutes in 2×SSC, 0.1% SDS, and then washed for 1 hour in 0.1×SSC and 0.1% SDS at 45° C.</p>
    <p>ii) Native Gels and Northern Blots</p>
    <p>In this experiment, the purified telomerase preparation was run on native (i.e., non-denaturing) gels of 3.5% polyacrylamide and 0.33% agarose, as known in the art and described (Lamond and Sproat, [1994], supra). The telomerase comigrated approximately with the xylene cyanol dye.</p>
    <p>The native gel results indicated that telomerase was maintained as an RNP throughout the purification protocol. FIG. 27 is a photograph of a Northern blot showing the mobility of the telomerase in different fractions on a non-denaturing gel as well as in vitro transcribed telomerase. In this figure, lane 1 contained 1.5 fmol telomerase RNA, lane 2 contained 4.6 fmol telomerase RNA, lane 3 contained 14 fmol telomerase RNA, lane 4 contained 41 fmol telomerase RNA, lane 5 contained nuclear extract (42 fmol telomerase), lane 6 contained Affi-Gel-heparin-purified telomerase (47 fmol telomerase), lane 7 contained affinity-purified telomerase (68 fmol), and lane 8 contained glycerol gradient-purified telomerase (35 fmol).</p>
    <p>As shown in FIG. 27, in nuclear extracts, the telomerase was assembled into an RNP particle that migrated slower than unassembled telomerase RNA. Less than 1% free RNA was detected by this method. However, a slower migrating telomerase RNP complex was also sometimes detected in extracts. Upon purification on the Affi-Gel-heparin column, the telomerase RNP particle did not change in mobility (FIG. 27, lane 6). However, upon affinity purification the mobility of the RNA particle slightly increased (FIG. 27, lane 7), perhaps indicating that a protein subunit or fragment had been lost. On glycerol gradients, the affinity-purified telomerase did not change in size, but approximately 2% free telomerase RNA was detectable (FIG. 27, lane 8), suggesting that a small amount of disassembly of the RNP particle had occurred.</p>
    <p>H. Telomerase Protein Composition</p>
    <p>In this Example, the analysis of the purified telomerase protein composition are described.</p>
    <p>Glycerol gradient fractions obtained as described in Part D, were separated on a 4-20% polyacrylamide gel (Novex). Following electrophoresis, the gel was stained with Coomassie brilliant blue. FIG. 29 shows a photograph of the gel. Lanes 1 and 2 contained molecular mass markers (Pharmacia) as indicated on the left side of the gel shown in FIG. <b>29</b>. Lanes 3-5 contained glycerol gradient fraction pools as indicated on the top of the gel (i.e., lane 3 contained fractions 9-14, lane 4 contained fractions 15-22, and lane 5 contained fractions 23-32). Lane 4 contained the pool with 1 pmol of telomerase RNA. In lanes 6-9 BSA standards were run at concentrations indicated at the top of the gel in FIG. 29 (i.e., lane 6 contained 0.5 pmol BSA, lane 7 contained 1.5 pmol BSA, lane 8 contained 4.5 BSA, and lane 9 contained 15 pmol BSA).</p>
    <p>As shown in FIG. 29, polypeptides with molecular masses of 120 and 43 kDa co-purified with the telomerase. The 43 kDa polypeptide was observed as a doublet. It was noted that the polypeptide of approximately 43 kDa in lane 3 migrated differently than the doublet in lane 4; it may be an unrelated protein. The 120 kDa and 43 kDa doublet each stained with Coomassie brilliant blue at approximately the level of 1 pmol, when compared with BSA standards. Because this fraction contained 1 pmol of telomerase RNA, all of which was assembled into an RNP particle (See, FIG. 27, lane 8), there appear to be two polypeptide subunits that are stoichiometric with the telomerase RNA. However, it is also possible that the two proteins around 43 kDa are separate enzyme subunits.</p>
    <p>Affinity-purified telomerase that was not subjected to fractionation on a glycerol gradient contained additional polypeptides with apparent molecular masses of 35 and 37 kDa, respectively. This latter fraction was estimated to be at least 50% pure. However, the 35 kDa and 37 kDa polypeptides that were present in the affinity-purified material were not reproducibly separated by glycerol gradient centrifugation. These polypeptides may be contaminants, as they were not visible in all activity-containing preparations.</p>
    <p>I. Sedimentation Coefficient</p>
    <p>The sedimentation coefficient for telomerase was determined by glycerol gradient centrifugation. In this Example, nuclear extract and affinity-purified telomerase were fractionated on 15-40% glycerol gradients containing 20 mM Tris-acetate, with 1 mM MgCl<sub>2</sub>, 0.1 mM EDTA, 300 mM KGlu, and 1 mM DTT, at pH 7.5. Glycerol gradients were poured in 5 ml (13×51 mm) tubes, and centrifuged using an SW55Ti rotor (Beckman) at 55,000 rpm for 14 hours at 4° C.</p>
    <p>Marker proteins were run in a parallel gradient and had a sedimentation coefficient of 7.6 S for alcohol dehydrogenase (ADH), 113 S for catalase, 17.3 S for apoferritin, and 19.3 S for thyroglobulin. The telomerase peak was identified by native gel electrophoresis of gradient fractions followed by blot hybridization to its RNA component.</p>
    <p>FIG. 30 is a graph showing the sedimentation coefficient for telomerase. As shown in this Figure, affinity-purified telomerase co-sedimented with catalase at 11.5 S, while telomerase in nuclear extracts sedimented slightly faster, peaking around 12.5 S. Therefore, consistent with the mobility of the enzyme in native gels, purified telomerase appears to have lost a proteolytic fragment or a loosely associated subunit.</p>
    <p>The calculated molecular mass for telomerase, if it is assumed to consist of one 120 kDa protein subunit, one 43 kDa subunit, and one RNA subunit of 66 kDa, adds up to a total of 229 kDa. This is in close agreement with the 232 kDa molecular mass of catalase. However, the sedimentation coefficient is a function of the molecular mass, as well as the partial specific volume and the frictional coefficient of the molecule, both of which are unknown for the Euplotes telomerase RNP.</p>
    <p>J. Substrate Utilization</p>
    <p>In this Example, the substrate requirements of Euplotes telomerase were investigated. One simple model for DNA end replication predicts that after semi-conservative DNA replication, telomerase extends double-stranded, blunt-ended DNA molecules. In a variation of this model, a single-stranded 3′ end is created by a helicase or nuclease after replication. This 3′ end is then used by telomerase for binding and extension.</p>
    <p>To determine whether telomerase is capable of elongating blunt-ended molecules, model hairpins were synthesized with telomeric repeats positioned at their 3′ ends. These primer substrates were gel-purified, 5′-end labelled with polynucleotide kinase, heated at 0.4 μM to 80° C. for 5 minutes, and then slowly cooled to room temperature in a heating block, to allow renaturation and helix formation of the hairpins. Substrate mobility on a non-denaturing gel indicated that very efficient hairpin formation was present, as compared to dimerization.</p>
    <p>Assays were performed with unlabelled 125 μM dGTP, 125 μM dTTP, and 0.02 μM 5′-end-labelled primer (5′-<sup>32</sup>P-labelled oligonucleotide substrate) in 10 μl reaction mixtures that contained 20 mM Tris-acetate, with 10 mM MgCl<sub>2</sub>, 50 mM KGlu, and 1 mM DTT, at pH 7.5. These mixtures were incubated at 25° C. for 30 minutes. Reactions were stopped by adding formamide loading buffer (i.e., TBE, formamide, bromthymol blue, and cyanol, Sambrook, 1989, supra).</p>
    <p>Primers were incubated without telomerase (“−”), with 5.9 fmol of affinity-purified telomerase (“+”), or with 17.6 fmol of affinity-purified telomerase (“+++”). Affinity-purified telomerase used in this assay was dialyzed with a membrane having a molecular cut-off of 100 kDa, in order to remove the displacement oligonucleotide. Reaction products were separated on an 8% PAGE/urea gel containing 36% formamide, to denature the hairpins. The sequences of the primers used in this study, as well as their lane assignments are shown in Table 6.</p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 6</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Primer Sequences</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="1" colwidth="56pt" align="center"> </colspec> <colspec colname="2" colwidth="98pt" align="left"> </colspec> <colspec colname="3" colwidth="63pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Lane</td>
                <td class="description-td">Primer Sequence: (5′ to 3′)</td>
                <td class="description-td">SEQ ID NO:</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="3" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> 1-3</td>
                <td class="description-td">C<sub>4</sub>(A<sub>4</sub>C<sub>4</sub>)<sub>3</sub>CACA(G<sub>4</sub>T<sub>4</sub>)<sub>3</sub>G<sub>4</sub> </td>
                <td class="description-td">560</td>
              </tr> <tr class="description-tr"> <td class="description-td">4-6</td>
                <td class="description-td">C<sub>2</sub>(A<sub>4</sub>C<sub>4</sub>)<sub>3</sub>CACA(G<sub>4</sub>T<sub>4</sub>)<sub>3</sub>G<sub>4</sub> </td>
                <td class="description-td">561</td>
              </tr> <tr class="description-tr"> <td class="description-td">7-9</td>
                <td class="description-td">(A<sub>4</sub>C<sub>4</sub>)<sub>3</sub>CACA(G<sub>4</sub>T<sub>4</sub>)<sub>3</sub>G<sub>4</sub> </td>
                <td class="description-td">562</td>
              </tr> <tr class="description-tr"> <td class="description-td">10-12</td>
                <td class="description-td">A<sub>2</sub>C<sub>4</sub>(A<sub>4</sub>C<sub>4</sub>)<sub>2</sub>CACA(G<sub>4</sub>T<sub>4</sub>)<sub>3</sub>G<sub>4</sub> </td>
                <td class="description-td">563</td>
              </tr> <tr class="description-tr"> <td class="description-td">13-15</td>
                <td class="description-td">C<sub>4</sub>(A<sub>4</sub>C<sub>4</sub>)<sub>2</sub>CACA(G<sub>4</sub>T<sub>4</sub>)<sub>3</sub> </td>
                <td class="description-td">564</td>
              </tr> <tr class="description-tr"> <td class="description-td">16-18</td>
                <td class="description-td">(A<sub>4</sub>C<sub>4</sub>)<sub>3</sub>CACA(G<sub>4</sub>T<sub>4</sub>)<sub>3</sub> </td>
                <td class="description-td">565</td>
              </tr> <tr class="description-tr"> <td class="description-td">19-21</td>
                <td class="description-td">A<sub>2</sub>C<sub>4</sub>(A<sub>4</sub>C<sub>4</sub>)<sub>2</sub>CACA(G<sub>4</sub>T<sub>4</sub>)<sub>3</sub> </td>
                <td class="description-td">566</td>
              </tr> <tr class="description-tr"> <td class="description-td">22-24</td>
                <td class="description-td">C<sub>4</sub>(A<sub>4</sub>C<sub>4</sub>)<sub>2</sub>CACA(G<sub>4</sub>T<sub>4</sub>)<sub>3</sub> </td>
                <td class="description-td">564</td>
              </tr> <tr class="description-tr"> <td class="description-td">25-27</td>
                <td class="description-td">C<sub>2</sub>(A<sub>4</sub>C<sub>4</sub>)<sub>2</sub>CACA(G<sub>4</sub>T<sub>4</sub>)<sub>3</sub> </td>
                <td class="description-td">567</td>
              </tr> <tr class="description-tr"> <td class="description-td">28-30</td>
                <td class="description-td">(A<sub>4</sub>C<sub>4</sub>)<sub>2</sub>CACA(G<sub>4</sub>T<sub>4</sub>)<sub>3</sub> </td>
                <td class="description-td">568</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="3" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p>The gel results are shown in FIG. <b>31</b>. Lanes 1-15 contained substrates with telomeric repeats ending with four G residues. Lanes 16-30 contained substrates with telomeric repeats ending with four T residues. The putative alignment on the telomerase RNA template is indicated in FIG. <b>32</b>. It was assumed that the primer sets anneal at two very different positions in the template shown in FIG. 32 (i.e., Panel A and Panel B, respectively). This may have affected their binding and/or elongation rate.</p>
    <p>FIG. 33 shows a lighter exposure of lanes 25-30 in FIG. <b>31</b>. The lighter exposure of FIG. 33 was taken to permit visualization of the nucleotides that are added and the positions of pausing in elongated products. Percent of substrate elongated for the third lane in each set was quantified on a PhosphorImager, as indicated on the bottom of FIG. <b>31</b>.</p>
    <p>The substrate efficiencies for these hairpins were compared with double-stranded telomere-like substrates with overhangs of differing lengths. A model substrate that ended with four G residues (see lanes 1-15 of FIG. 31) was not elongated when it was blunt ended (see lanes 1-3). However, slight extension was observed with an overhang length of two bases; elongation became efficient when the overhang was at least 4 bases in length. The telomerase acted in a similar manner with a double-stranded substrate that ended with four T residues, with a 6-base overhang required for highly efficient elongation. In FIG. 31, the faint bands below the primers in lanes 10-15 that are independent of telomerase represent shorter oligonucleotides in the primer preparations.</p>
    <p>The lighter exposure of lanes 25-30 in FIG. 33 shows a ladder of elongated products, with the darkest bands correlating with the putative 5′ boundary of the template (as described by Lingner et al., Genes Develop., 8:1984 [1994]). The abundance of products that correspond to other positions in the template suggested that pausing and/or dissociation occurs at sites other than the site of translocation with the purified telomerase.</p>
    <p>As shown in FIG. 31, double-stranded, blunt-ended oligonucleotides were not substrates for telomerase. To determine whether these molecules would bind to telomerase, a competition experiment was performed. In this experiment, 2 nM of 5′-end labeled substrate with the sequence (G<sub>4</sub>T<sub>4</sub>)<sub>2 </sub>(SEQ ID NO:114), or a hairpin substrate with a six base overhang were extended with 0.125 nM telomerase (FIG. 31, lanes 25-27). Although the same unlabeled oligonucleotide substrates competed efficiently with labeled substrate for extension, no reduction of activity was observed when the double-stranded blunt-ended hairpin oligonucleotides were used as competitors, even in the presence of 100-fold excess hairpins.</p>
    <p>These results indicated that double-stranded, blunt-ended oligonucleotides cannot bind to telomerase at the concentrations and conditions tested in this Example. Rather, a single-stranded 3′ end is required for binding. It is likely that this 3′ end is required to base pair with the telomerase RNA template.</p>
    <p>K. Cloning &amp; Sequencing of the 123 kDa Polypeptide</p>
    <p>In this Example, the cloning of the 123 kDa polypeptide of Euplotes telomerase (i.e., the 123 kDa protein subunit) is described. In this study, an internal fragment of the telomerase gene was amplified by PCR, with oligonucleotide primers designed to match peptide sequences that were obtained from the purified polypeptide obtained in Part D, above. The polypeptide sequence was determined using the nanoES tandem mass spectroscopy methods known in the art and described by Calvio et al., RNA 1:724-733 [1995]. The oligonucleotide primers used in this Example had the following sequences, with positions that were degenerate shown in parentheses - - - 5′-TCT(G/A)AA(G/A)TA(G/A)TG(T/G/A)GT(G/A/T/C)A(T/G/A)(G/A)TT(G/A)TTCAT-3′ (SEQ ID NO:569), and 5′-GCGGATCCATGAA(T/C)CC(A/T)GA(G/A)AA(T/C)CC(A/T)AA(T/C)GT-3′ (SEQ ID NO:570).</p>
    <p>A 50 μl reaction contained 0.2 mM dNTPs, 0.15 μg <i>E. aediculatus </i>chromosomal DNA, 0.5 μl Taq (Boehringer-Mannheim), 0.8 μg of each primer, and 1×reaction buffer (Boehringer-Mannheim). The reaction was incubated in a thermocycler (Perkin-Elmer), using the following—5 minutes at 95° C., followed by 30 cycles of 1 minute at 94° C., 1 minute at 52° C., and 2 minutes at 72° C. The reaction was completed by a 10 minute incubation at 72EC.</p>
    <p>A genomic DNA library was prepared from the chromosomal <i>E. aediculatus </i>DNA by cloning blunt-ended DNA into the SmaI site of pCR-Script plasmid vector FIG. 14 (Stratagene). This library was screened by colony hybridization, with the radiolabelled, gel-purified PCR product. Plasmid DNA of positive clones was prepared and sequenced by the dideoxy method (Sanger et al., Proc. Natl. Acad. Sci., 74:5463 [1977]) or manually, through use of an automated sequencer (ABI). The DNA sequence of the gene encoding this polypeptide is shown in FIG. <b>13</b>. The start codon in this sequence inferred from the DNA sequence, is located at nucleotide position 101, and the open reading frame ends at position 3193. The genetic code of Euplotes differs from other organisms in that the “UGA” codon encodes a cysteine residue. The amino acid sequence of the polypeptide inferred from the DNA sequence is shown in FIG. 14, and assumes that no unusual amino acids are inserted during translation and no post-translational modification occurs.</p>
    <p>L. Cloning &amp; Sequencing of the 43 kDa Polypeptide</p>
    <p>In this Example, the cloning of the 43 kDa polypeptide of telomerase (i.e., the 43 kDa protein subunit) is described. In this study, an internal fragment of the corresponding telomerase gene was amplified by PCR, with oligonucleotide primers designed to match peptide sequences that were obtained from the purified polypeptide obtained in Part D, above. The polypeptide sequence was determined using the nanoES tandem mass spectroscopy methods known in the art and described by Calvio et at., supra. The oligonucleotide primers used in this Example had the following sequences - - - 5′-NNNGTNAC(C/T/A)GG(C/T/A)AT(C/T/A)AA(C/T)AA-3′ (SEQ ID NO :571), and 5′-(T/G/A)GC(T/G/A)GT(C/T)TC(T/C)TG(G/A)TC(G/A)TT(G/A)TA-3′ (SEQ ID NO:572). In this sequence, “N” indicates the presence of any of the four nucleotides (i.e., A, T, G, or C).</p>
    <p>The PCR was performed as described in Part K.</p>
    <p>A genomic DNA library was prepared and screened as described in Part K. The DNA sequence of the gene encoding this polypeptide is shown in FIG. <b>34</b>. Three potential reading frames are shown for this sequence, as shown in FIG. <b>35</b>. For clarity, the amino acid sequence is indicated below the nucleotide sequence in all three reading frames. These reading frames are designated as “a,” “b,” and “c”. A possible start codon is encoded at nucleotide position 84 in reading frame “c.” The coding region could end at position 1501 in reading frame “b.” Early stop codons, indicated by asterisks in this figure, occur in all three reading frames between nucleotide position 337-350.</p>
    <p>Further downstream, the protein sequence appears to be encoded by different reading frames, as none of the three frames is uninterrupted by stop codons. Furthermore, peptide sequences from purified protein are encoded in all three frames. Therefore, this gene appears to contain intervening sequences, or in the alternative, the RNA is edited. Other possibilities include ribosomal frame-shifting or sequence errors. However, the homology to the La-protein sequence remains of significant interest. Again, in Euplotes, the “UGA” codon encodes a cysteine residue.</p>
    <p>M. Amino Acid and Nucleic Acid Comparisons</p>
    <p>In this Example, comparisons between various reported sequences and the sequences of the 123 kDa and 43 kDa telomerase subunit polypeptides were made.</p>
    <p>i) Comparisons with the 123 kDa <i>E. aediculatus </i>Telomerase Subunit</p>
    <p>The amino acid sequence of the 123 kDa <i>Euplotes aediculatus </i>polypeptide was compared with the sequence of the 80 kDa telomerase protein subunit of <i>Tetrahymena thermophila </i>(GenBank accession #U25641) to investigate their similarity. The nucleotide sequence as obtained from GenBank encoding this protein is shown in FIG. <b>42</b>. The amino acid sequence of this protein as obtained from GenBank is shown in FIG. <b>43</b>. The sequence comparison between the 123 kDa <i>E. aedicutlatus </i>and 80 kDa <i>T. thermophila </i>is shown in FIG. <b>36</b>. In this figure, the <i>E. aediculatus </i>sequence is the upper sequence, while the <i>T. thermophila </i>sequence is the lower sequence. The observed identity was determined to be approximately 19%, while the percent similarity was approximately 45%, values similar to what would be observed with any random protein sequence. In FIGS. 36-39, identities are indicated by vertical bars, while single dots between the sequences indicate somewhat similar amino acids, and double dots between the sequences indicate more similar amino acids.</p>
    <p>The amino acid sequence of the 123 kDa <i>Euplotes aediculatus </i>polypeptide was also compared with the sequence of the 95 kDa telomerase protein subunit of <i>Tetrahymena thermophila </i>(GenBank accession #U25642), to investigate their similarity. The nucleotide sequence as obtained from GenBank encoding this protein is shown in FIG. <b>44</b>. The amino acid sequence of this protein as obtained from GenBank is shown in FIG. <b>45</b>. This sequence comparison is shown in FIG. <b>37</b>. In this figure, the <i>E. aediculatus </i>sequence is the upper sequence), while the <i>T. thermophila </i>sequence is the lower sequence. The observed identity was determined to be approximately 20%, while the percent similarity was approximately 43%, values similar to what would be observed with any random protein sequence.</p>
    <p>Significantly, the amino acid sequence of the 123 kDa <i>E. aediculatus </i>polypeptide contains the five motifs characteristic of reverse transcriptases. The 123 kDa polypeptide was also compared with the polymerase domains of various reverse transcriptases. FIG. 40 shows the alignment of the 123 kDa polypeptide with the putative yeast homolog (L8543.12 or ESTp). The amino acid sequence of L8543.12 obtained from GenBank is shown in FIG. <b>46</b>.</p>
    <p>Four motifs (A, B, C, and D) were included in this comparison. In this FIG. 40, highly conserved residues are indicated by white letters on a black background. Residues of the <i>E. aediculatus </i>sequences that are conserved in the other sequence are indicated in bold; the “h” indicates the presence of a hydrophobic amino acid. The numerals located between amino acid residues of the motifs indicates the length of gaps in the sequences. For example, the “100” shown between motifs A and B reflects a 100 amino acid gap in the sequence between the motifs.</p>
    <p>As noted above, Genbank searches identified a yeast protein (Genbank accession #u20618), and gene L8543.12 (Est2) containing or encoding amino acid sequence that shows some homology to the <i>E. aediculatus </i>123 kDa telomerase subunit. Based on the observations that both proteins contain reverse transcriptase motifs in their C-terminal regions; both proteins share similarity in regions outside the reverse transcriptase motif; the proteins are similarly basic (pI=10.1 for <i>E. aediculatus </i>and pI=10.0 for the yeast); and both proteins are large (123 kDa for <i>E. aediculatus </i>and 103 kDa for the yeast), these sequences comprise the catalytic core of their respective telomerases. It was contemplated based on this observation of homology in two phylogenetically distinct organisms as <i>E. aediculatus </i>and yeast, that human telomerase would contain a protein that has the same characteristics (i.e., reverse transcriptase motifs, is basic, and large [&gt;100 kDa]).</p>
    <p>ii) Comparisons with the 43 kDa <i>E. aediculatus </i>Telomerase Subunit</p>
    <p>The amino acid sequence of the “La-domain” of the 43 kDa <i>Euplotes aediculatus </i>polypeptide was compared with the sequence of the 95 kDa telomerase protein subunit of <i>Tetrahymena thermophila </i>(described above) to investigate their similarity. This sequence comparison is shown in FIG. 38, while the <i>T. thermophila </i>sequence is the lower sequence. The observed identity was determined to be approximately 23%, while the percent similarity was approximately 46%, values similar to what would be observed with any random protein sequence.</p>
    <p>The amino acid sequence of the “La-domain” of the 43 kDa <i>Euplotes aediculatus </i>polypeptide was compared with the sequence of the 80 kDa telomerase protein subunit of <i>Tetrahymena thermophila </i>(described above) to investigate their similarity. This sequence comparison is shown in FIG. <b>39</b>. In this figure, the <i>E. aediculatus </i>sequence is the upper sequence, while the <i>T. thermophila </i>sequence is the lower sequence. The observed identity was determined to be approximately 26%, while the percent similarity was approximately 49%, values similar to what would be observed with any random protein sequence.</p>
    <p>The amino acid sequence of a domain of the 43 kDa <i>E. aediculatus </i>polypeptide was also compared with La proteins from various other organisms. These comparisons are shown in FIG. <b>41</b>. In this Figure, highly conserved residues are indicated by white letters on a black background. Residues of the <i>E. aediculatus </i>sequences that are conserved in the other sequence are indicated in bold.</p>
    <p>N. Identification of Telomerase Protein Subunits in Another Organism</p>
    <p>In this Example, the sequences identified in the previous Examples above were used to identify the telomerase protein subunits of <i>Oxytricha trifallax, </i>a ciliate that is very distantly related to <i>E. aediculatus. </i>Primers were chosen based on the conserved region of the <i>E. aediculatus </i>123 kDa polypeptide which comprised the reverse transcriptase domain motifs. Suitable primers were synthesized and used in a PCR reaction with total DNA from Oxytricha. The Oxytricha DNA was prepared according to methods known in the art. The PCR products were then cloned and sequenced using methods known in the art.</p>
    <p>The oligonucleotide sequences used as the primers were as follows:</p>
    <p>5′-(T/C)A(A/G)AC(T/A/C)AA(G/A)GG(T/A/C)AT(T/C)CC(C/T/A)(C/T)A(G/A)GG-3′ (SEQ ID NO:573) and 5′-(G/A/T)GT(G/A/T)ATNA(G/A)NA(G/A)(G/A)TA(G/A)TC(G/A)TC-3′ (SEQ ID NO:574).</p>
    <p>Positions that were degenerate are shown in parentheses, with the alternative bases shown within the parenthesis. “N” represents any of the four nucleotides.</p>
    <p>In the PCR reaction, a 50 μl reaction contained 0.2 mM dNTPs, 0.3 μg <i>Oxytricha trifallax </i>chromosomal DNA, 1 μl Taq polymerase (Boehringer-Mannheim), 2 micromolar of each primer, 1×reaction buffer (Boehringer-Mannheim). The reaction was incubated in a thermocycler (Perkin-Elmer) under the following conditions: 5 min at 95° C., 30 cycles consisting of 1 min at 94° C., 1 min at 53° C, and 1 min at 72° C., followed by a 10 min incubation at 72° C. The PCR-product was gel-purified and sequenced by the dideoxy-method (e.g., Sanger et al., Proc. Natl. Acad. Sci. 74, 5463-5467 (1977).</p>
    <p>The deduced amino acid sequence of the PCR product was determined and compared with the <i>E. aediculatus </i>sequence. FIG. 47 shows the alignment of these sequences, with the <i>O. trifallax </i>sequence shown in the top row, and the <i>E. aediculatus </i>sequence shown in the bottom row. As can be seen from this figure, there is a great deal of homology between the <i>O. trifallax </i>polypeptide sequence identified in this Example with the <i>E. aediculatus </i>polypeptide sequence. Thus, it is clear that the sequences identified in the present invention are useful for the identification of homologous telomerase protein subunits in other eukaryotic organisms. Indeed, development of the present invention has identified homologous telomerase sequences in multiple, diverse species, as described herein.</p>
    <p>O. Identification of Tetrahymena Telomerase Sequences</p>
    <p>In this Example, a Tetrahymena clone was produced that shares homology with the Euplotes sequences, and EST2p.</p>
    <p>This experiment utilized PCR with degenerate oligonucleotide primers directed against conserved motifs to identify regions of homology between Tetrahymena, Euplotes, and EST2p sequences. The PCR method used in this Example is a novel method designed to amplify specifically rare DNA sequences from complex mixtures. This method avoids the problem of amplification of DNA products with the same PCR primer at both ends (i.e., single primer products) commonly encountered in PCR cloning methods. These single primer products produce unwanted background and can often obscure the amplification and detection of the desired two-primer product. The method used in this experiment preferentially selects for two-primer products. In particular, one primer is biotinylated and the other is not. After several rounds of PCR amplification, the products are purified using streptavidin magnetic beads and two primer products are specifically eluted using heat denaturation. This method finds use in settings other than the experiments described in this Example. Indeed, this method finds use in application in which it is desired to specifically amplify rare DNA sequences, including the preliminary steps in cloning methods such as 5′ and 3′RACE, and any method that uses degenerate primers in PCR.</p>
    <p>A first PCR run was conducted using Tetrahymena template macronuclear DNA isolated using methods known in the art, and the 24-mer forward primer with the sequence 5′biotin-GCCTATTT(TC)TT(TC)TA(TC)(GATC)(GATC)(GATC)AC(GATC)GA-3′ (SEQ ID NO:575) designated as “K231,” corresponding to the FFYXTE SEQ ID NO:360 region, and the 23-mer reverse primer with the sequence 5′-CCAGATAT(GATC)A (TGA)(GATC)A(AG)(AG)AA(AG)TC(AG)TC-3′ (SEQ ID NO:576), designated as “K220,” corresponding to the DDFL(FIL)I (SEQ ID NO:577) region. This PCR reaction contained 2.5 μl DNA (50 ng), 4 μl of each primer (20 μM), 3 μl 10×PCR buffer, 3 μl 10×dNTPs, 2 μl Mg, 0.3 μl Taq, and 11.2 μl dH<sub>2</sub>O. The mixture was cycled for 8 cycles of 94° C. for 45 seconds, 37° C. for 45 seconds, and 72° C. for 1 minute.</p>
    <p>This PCR reaction was bound to 200 μl streptavidin magnetic beads, washed with 200 μl TE, resuspended in 20 μl dH<sub>2</sub>O and then heat-denatured by boiling at 100° C. for 2 minutes. The beads were pulled down and the eluate removed. Then, 2.5 μl of this eluate was subsequently reamplified using the above conditions, with the exception being that 0.3 μl of α-<sup>32</sup>P dATP was included, and the PCR was carried out for 33 cycles. This reaction was run a 5% denaturing polyacrylamide gel, and the appropriate region was cut out of the gel. These products were then reamplified for an additional 34 cycles, under the conditions listed above, with the exception being that a 42° C. annealing temperature was used.</p>
    <p>A second PCR run was conducted using Tetrahymena macronuclear DNA template isolated using methods known in the art, and the 23-mer forward primer with the sequence 5′-ACAATG(CA)G(GATC)(TCA)T(GATC)(TCA)T(GATC)CC (GATC)AA(AG)AA-3′ (SEQ ID NO:578), designated as “K228,” corresponding to the region R(LI)(LI)PKK (SEQ ID NO:579), and a reverse primer with the sequence 5′-ACGAATC(GT)(GATC)G(TAG)AT(GATC)(GC)(TA)(AG)TC(AG)TA(AG)CA 3′ (SEQ ID NO:580), designated “K224,” corresponding to the CYDSIPR (SEQ ID NO:581) region. This PCR reaction contained 2.5 μl DNA (50 ng), 4 μl of each primer (20 μM), 3 μl 10×PCR buffer, 3 μl 10×dNTPs, 2 μl Mg, 0.3 μl α-<sup>32</sup>P dATP, 0.3 μl Taq, and 10.9 μl dH<sub>2</sub>O. This reaction was run on a 5% denaturing polyacrylamide gel, and the appropriate region was cut out of the gel. These products were reamplified for an additional 34 cycles, under the conditions listed above, with the exception being that a 42° C. annealing temperature was used.</p>
    <p>Ten μl of the reaction product from run 1 were bound to streptavidin-coated magnetic beads in 200 μl TE. The beads were washed with 200 μl TE, and then resuspended in 20 μl of dH<sub>2</sub>O, heat denatured, and the eluate was removed. The reaction product from run 2 was then added to the beads and diluted with 30 μl 0.5×SSC. The mixture was heated from 94° C. to 50° C. The eluate was removed and the beads were washed three times in 0.5×SSC at 55EC. The beads were then resuspended in 20 μl dH<sub>2</sub>O, heat denatured, and the eluate was removed, designated as “round 1 eluate” and saved.</p>
    <p>To isolate the Tetrahymena band, the round 1 eluate was reamplified with the forward primer K228 and reverse primer K227 with the sequence 5′-CAATTCTC(AG)TA(AG)CA(GATC)(CG)(TA)(CT)TT(AGT)AT(GA)TC-3′ (SEQ ID NO:582), corresponding to the DIKSCYD (SEQ ID NO:583) region. The PCR reactions were conducted as described above. The reaction products were run on a 5% polyacrylamide gel; the band corresponding to approximately 295 nucleotides was cut from the gel and sequenced.</p>
    <p>The clone designated as 168-3 was sequenced. The DNA sequence (including the primer sequences) was found to be:</p>
    <p>GATTACTCCCGAAGAAAGGATCTTTCCGTCCAATCATGACTTTCTTAAGAAAGGACAAGCAAAAAAATATTAAGTTAAATCTAAATTAAATTCTAATGGATAGCCAACTTGTGTTTAGGAATTTAAAAGACATGCTGGGATAAAAGATAGGATACTCAGTCTTTGATAATAAACAAATTTCAGAAAAATTTGCCTAATTCATAGAGAAATGGAAAAATAAAGGAAGACCTCAGCTATATTATGTCACTCTAGACATAAAGACTTGCTAC (SEQ ID NO:584).</p>
    <p>Additional sequence of this gene was obtained by PCR using one unique primer designed to match the sequence from 168-3 (“K297” with the sequence 5′-GAGTGACATAATATACGTGA-3′ (SEQ ID NO:585); and the K231 (FFYXTE; SEQ ID NO:360) primer. The sequence of the fragment obtained from this reaction, together with 168-3 is as follows (without the primer sequences):</p>
    <p>AAACACAAGGAAGGAAGTCAAATATTCTATTACCGTAAACCAATATGGAAATTAGTGAGTAAATTAACTATTGTCAAAGTAAGAATTTAGTTTTCTGAAAAGAATAAATAAATGAAAAATAATTTTTATCAAAAAATTTAGCTTGAAGAGGAGAATTTGGAAAAAGTTGAAGAAAAATTGATACCAGAAGATTCATTTTAGAAATACCCTCAAGGAAAGCTAAGGATTATACCTAAAAAAGGATCTTTCCGTCCAATCATGACTTTCTTAAGAAAGGACAAGCAAAAAAATATTAAGTTAAATCTAAATTAAATTCTAATGGATAGCCAACTTGTGTTTAGGAATTTAAAAGACATGCTGGGATAAAAGATAGGATACTCAGTCTTTGATAATAAACAAATTTCAGAAAAATTTGCCTAATTCATAGAGAAATGGAAAAATAAAGGAAGACCTCAGCTATATTATGTCACTCTA (SEQ ID NO:586).</p>
    <p>The amino acid sequence corresponding to this DNA fragment was found to be:</p>
    <p>KHKEGSQIFYYRKPIWKLVSKLTIVKVRIQFSEKNKQMKNNFYQKIQLEEENLEKVEEKLIPEDSFQKYPQGKLRIIPKKGSFRPIMTFLRKDKQKNIKLNLNQILMDSQLVFRNLKDMLGQKIGYSVFDNKQISEKFAQFIEKWKNKGRPQLYYVTL (SEQ ID NO:228).</p>
    <p>This amino acid sequence was then aligned with other telomerase genes (EST2p, and Euplotes). The alignment is shown in FIG. 53. A consensus sequence is also shown in this Figure.</p>
    <p>P. Identification of <i>Schizosaccharomyces pombe </i>Telomerase Sequences</p>
    <p>In this Example, the tez1 sequence of <i>S. pombe </i>was identified as a homolog of the <i>E. aediculatus </i>p123, and <i>S. cerevisiae </i>Est2p.</p>
    <p>FIG. 55 provides an overall summary of these experiments. In this Figure, the top portion (Panel A) shows the relationship of two overlapping genomic clones, and the 5825 bp portion that was sequenced. The region designated at “tez1<sup>+</sup>” is the protein coding region, with the flanking sequences indicated as well, the box underneath the 5825 bp region is an approximately 2 kb HindIII fragment that was used to make the tez1 disruption construct, as described below.</p>
    <p>The bottom half of FIG. 55 (Panel B) is a “close-up” schematic of this same region of DNA. The sequence designated as “original PCR” is the original degenerate PCR fragment that was generated with a degenerate oligonucleotide primer pair designed based on Euplotes sequence motif 4 (B′) and motif 5 (C), as described.</p>
    <p>i) PCR With Degenerate Primers</p>
    <p>PCR using degenerate primers was used to find the homolog of the <i>E. aediculatus </i>p123 in <i>S. pombe. </i>FIG. 56 shows the sequences of the degenerate primers (designated as “poly 4” and “poly 1”) used in this reaction. The PCR runs were conducted using the same buffer as described in previous Examples (See e.g., Part K, above), with a 5 minute ramp time at 94° C., followed by 30 cycles of 94° C. for 30 seconds, 50° C. for 45 seconds, and 72° C. for 30 seconds, and 7 minutes at 72° C., followed by storage at 4° C. PCR runs were conducted using varied conditions, (i.e., various concentrations of <i>S. pombe </i>DNA and MgCl<sub>2 </sub>concentrations). The PCR products were run on agarose gels and stained with ethidium bromide as described above. Several PCR runs resulted in the production of three bands (designated as “T,” “M,” and “B”). These bands were re-amplified and run on gels using the same conditions as described above. Four bands were observed following this re-amplification (“T,” “M1,” “M2,” and “B”), as shown in FIG. <b>57</b>. These four bands were then re-amplified using the same conditions as described above. The third band from the top of the lane in FIG. 57 was identified as containing the correct sequence for a telomerase protein. The PCR product designated as M2 was found to show a reasonable match with other telomerase proteins, as indicated in FIG. <b>58</b>. In addition to the alignment shown, this Figure also shows the actual sequence of tez1. In this Figure, the asterisks indicate residues shared with all four sequences (Oxytricha “Ot”; <i>E. aediculatus </i>“Ea_p123<i>”; S. cerevisiae </i>“Sc_p103”; and M2), while the circles (i.e., dots) indicate similar amino acid residues.</p>
    <p>ii) 3′ RT PCR</p>
    <p>To obtain additional sequence information, 3′ and 5′ RT PCR were conducted on the telomerase candidate identified in FIG. <b>58</b>. FIG. 59 provides a schematic of the 3′ RT PCR strategy used. First, cDNA was prepared from mRNA using the oligonucleotide primer “Q<sup>T</sup>,” (5′-CCA GTG AGC AGA GTG ACG AGG ACT CGA GCT CAA GCT TTT TTT TTT TTT TT-3′; SEQ ID NO:587), then using this cDNA as a template for PCR with “Q<sub>O</sub>” (5′-CCA GTG AGC AGA GTG ACG-3′; SEQ ID NO:588), and a primer designed based on the original degenerated PCR reaction (i.e., “M2-T” with the sequence 5′-G TGT CAT TTC TAT ATG GAA GAT TTG ATT GAT G-3′; SEQ ID NO:589). The second PCR reaction (i.e., nested PCR) with “Q<sub>1</sub>” (5′-GAG GAC TCG AGC TCA AGC-3′; SEQ ID NO:590), and another PCR primer designed with sequence derived from the original degenerate PCR reaction or “M2-T2” (5′-AC CTA TCG TTT ACG AAA AAG AAA GGA TCA GTG-3′; SEQ ID NO:591). The buffers used in this PCR were the same as described above, with amplification conducted beginning with a ramp up of 940 for 5 min, followed by 30 cycles of 94° for 30 sec, 55° C. for 30 sec, and 72° C. for 3 min, followed by 72° C. The reaction products were stored at 4° C. until use.</p>
    <p>iii) Screening of Genomic and cDNA Libraries</p>
    <p>After obtaining this additional sequence information, several genomic and cDNA libraries were screened to identify any libraries that contain this telomerase candidate gene. The approach used, as well as the libraries and results are shown in FIG. <b>60</b>. In this Figure, Panel A lists the libraries tested in this experiment; Panel B shows the regions used; Panels C and D show the dot blot hybridization results obtained with these libraries. Positive libraries were then screened by colony hybridization to obtain genomic and cDNA version of tez1 gene. In this experiment, approximately 3×10<sup>4 </sup>colonies from the HindIII genomic library were screened and six positive clones were identified (approximately 0.01%). DNA was then prepared from two independent clones (A5 and B2). FIG. 61 shows the results obtained with the HindIII-digested A5 and B2 positive genomic clones.</p>
    <p>In addition, cDNA REP libraries were used. Approximately 3×10<sup>5 </sup>colonies were screened, and 5 positive clones were identified (0.002%). DNA was prepared from three independent clones (2-3, 4-1, and 5-20). In later experiments, it was determined that clones 2-3 and 5-20 contained identical inserts.</p>
    <p>iv) 5′ RT PCR</p>
    <p>As the cDNA version of gene produced to this point was not complete, 5′ RT-PCR was conducted to obtain a full length clone. The strategy is schematically shown in FIG. <b>62</b>. In this experiment, cDNA was prepared using DNA oligonucleotide primer “M2-B” (5′-CAC TGA TCC TTT CTT TTT CGT AAA CGA TAG GT-3′; SEQ ID NO:592) and “M2-B2” (5′-C ATC AAT CAA ATC TTC CAT ATA GAA ATG ACA-3′; SEQ ID NO:593), designed from known regions of tez1 identified previously. An oligonucleotide linker PCR Adapt SfiI with a phosphorylated 5′ end (“P”) (P-GGG CCG TGT TGG CCT AGT TCT CTG CTC-3′ SEQ ID NO:594; was then ligated at the 3′ end of this cDNA, and this construct was used as the template for nested PCR. In the first round of PCR, PCR Adapt SFI and M2-B were used as the primers; while PCR Adapt SfiI (5′-GAG GAG GAG AAG AGC AGA GAA CTA GGC CAA CAC GCC CC-3′; SEQ ID NO:595), and M2-B2 were used as primers in the second round. Nested PCR was used to increase specificity of reaction.</p>
    <p>v) Sequence Alignments</p>
    <p>Once the sequence of tez1 was identified, it was compared with sequences previously described. FIG. 63 shows the alignment of RT domains from telomerase catalytic subunits of <i>S. pombe </i>(“S.p. Tez1p”), <i>S. cerevisiae </i>(“S.c. Est2p”), and <i>E. aediculatus </i>p123 (“E.a. p123”). In this Figure, “h” indicates hydrophobic residues, while “p” indicates small polar residues, and “c” indicates charged residues. The amino acid residues indicated above the alignment show a known consensus RT motif of Y. Xiong and T. H. Eickbush (Y. Xiong and T. H. Eickbush, EMBO J., 9: 3353-3362 [1990]). The asterisks indicate the residues that are conserved for all three proteins. “Motif O” is identified herein and in FIG. 63 as a motif specific to this telomerase subunit and not found in reverse transcriptases in general. It is therefore valuable in identifying other amino acid sequences as telomerase catalytic subunits.</p>
    <p>FIG. 64 shows the alignment of entire sequences from Euplotes (“Ea_p123”), <i>S. cerevisiae </i>(“Sc_Est2p”), and <i>S. pombe </i>(“Sp_Tez1p”). In Panel A, the shaded areas indicate residues shared between two sequences. In Panel B, the shaded areas indicate residues shared between all three sequences.</p>
    <p>vi) Genetic Disruption of tez1</p>
    <p>In this Example, the effects of disruption of tez1 were investigated. As telomerase is involved in telomere maintenance, it was hypothesized that if tez1 were indeed a telomerase component, disruption of tez1 would cause gradual telomere shortening.</p>
    <p>In these experiments, homologous recombination was used to disrupt the tez1 gene in <i>S. pombe </i>specifically. This approach is schematically illustrated in FIG. <b>65</b>. As indicated in FIG. 65, wild type tez1 was replaced with a fragment containing the ura4 or LEU2 marker.</p>
    <p>The disruption of tez1 gene was confirmed by PCR (FIG. <b>66</b>), and a Southern blot was performed to check for telomere length. FIG. 67 shows the Southern blot results for this experiment. Because an ApaI restriction enzyme site is present immediately adjacent to telomeric sequence in <i>S. pombe, </i>ApaI digestion of <i>S. pombe </i>genomic DNA preparations permits analysis of telomere length. Thus, DNA from <i>S. pombe </i>was digested with ApaI and the digestion products were run on an agarose gel and probed with a telomeric sequence-specific probe to determine whether the telomeres of disrupted <i>S. pombe </i>cells were shortened. The results are shown in FIG. <b>67</b>. From these results, it was clear that disruption of the tez1 gene caused a shortening of the telomeres.</p>
    <p>Q. Cloning and Characterization of Human Telomerase Protein and cDNA</p>
    <p>In this Example, the nucleic and amino acid sequence information for human telomerase was determined. Partial homologous sequences were first identified in a BLAST search conducted using the Euplotes 123 kDa peptide and nucleic acid sequences, as well as Schizosaccharomyces protein and corresponding cDNA (tez1) sequences. The human sequences (also referred to as “hTCP1.1”) were identified from a partial cDNA clone (clone 712562). Sequences from this clone were aligned with the sequences determined as described in previous Examples.</p>
    <p>FIG. 1 shows the sequence alignment of the Euplotes (“p123”), Schizosaccharomyces (“tez1”), Est2p (i.e., the <i>S. cerevisiae </i>protein encoded by the Est2 nucleic acid sequence, and also referred to herein as “L8543.12”), and the human homolog identified in this comparison search. FIG. 51 shows the amino acid sequence of tez1, while FIG. 52 shows the DNA sequence of tez1. In FIG. 52, the introns and other non-coding regions, are shown in lower case, while the exons (i.e., coding regions) are shown in upper case.</p>
    <p>As shown in the Figures, there are regions that are highly conserved among these proteins. For example, as shown in FIG. 1, there are regions of identity in “Motif 0,” “Motif 1,” “Motif 2,” and “Motif 3.” The identical amino acids are indicated with an asterisk (*), while the similar amino acid residues are indicated by a circle (•). This indicates that there are regions within the telomerase motifs that are conserved among a wide variety of eukaryotes, ranging from yeast to ciliates to humans. It is contemplated that additional organisms will likewise contain such conserved regions of sequence. FIG. 49 shows the partial amino acid sequence of the human telomerase motifs, while FIG. 50 shows the corresponding DNA sequence.</p>
    <p>Sanger dideoxy sequencing and other methods were used, as known in the art to obtain complete sequence information of clone 712562. Some of the primers used in the sequencing are shown in Table 7. These primers were designed to hybridize to the clone, based on sequence complementarity to either plasmid backbone sequence or the sequence of the human cDNA insert in the clone.</p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 7</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Primers</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="1" colwidth="56pt" align="center"> </colspec> <colspec colname="2" colwidth="98pt" align="left"> </colspec> <colspec colname="3" colwidth="63pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">     Primer</td>
                <td class="description-td">        Sequence</td>
                <td class="description-td">SEQ ID NO:</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="3" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="1" colwidth="56pt" align="center"> </colspec> <colspec colname="2" colwidth="98pt" align="left"> </colspec> <colspec colname="3" colwidth="63pt" align="char" char="."> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">  TCP1.1</td>
                <td class="description-td">GTGAAGGCACTGTTCAGCG</td>
                <td class="description-td">377</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.2</td>
                <td class="description-td">GTGGATGATTTCTTGTTGG</td>
                <td class="description-td">381</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.3</td>
                <td class="description-td">ATGCTCCTGCGTTTGGTGG</td>
                <td class="description-td">596</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.4</td>
                <td class="description-td">CTGGACACTCAGCCCTTGG</td>
                <td class="description-td">382</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.5</td>
                <td class="description-td">GGCAGGTGTGCTGGACACT</td>
                <td class="description-td">383</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.6</td>
                <td class="description-td">TTTGATGATGCTGGCGATG</td>
                <td class="description-td">384</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.7</td>
                <td class="description-td">GGGGCTCGTCTTCTACAGG</td>
                <td class="description-td">385</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.8</td>
                <td class="description-td">CAGCAGGAGGATCTTGTAG</td>
                <td class="description-td">386</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.9</td>
                <td class="description-td">TGACCCCAGGAGTGGCACG</td>
                <td class="description-td">387</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.10</td>
                <td class="description-td">TCAAGCTGACTCGACACCG</td>
                <td class="description-td">388</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.11</td>
                <td class="description-td">CGGCGTGACAGGGCTGC</td>
                <td class="description-td">389</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.12</td>
                <td class="description-td">GCTGAAGGCTGAGTGTCC</td>
                <td class="description-td">390</td>
              </tr> <tr class="description-tr"> <td class="description-td">TCP1.13</td>
                <td class="description-td">TAGTCCATGTTCACAATCG</td>
                <td class="description-td">391</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="3" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p>From these experiments, it was determined that the EcoRI-NotI insert of clone 712562 contains only a partial open reading frame for the human telomerase protein, although it may encode an active fragment of that protein. The open reading frame in the clone encodes an approximately 63 kD protein. The sequence of the longest open reading frame identified is shown in FIG. <b>68</b>. The ORF begins at the ATG codon with the “met” indicated in the Figure. The poly A tail at the 3′ end of the sequence is also shown. FIG. 69 shows a tentative, preliminary alignment of telomerase reverse transcriptase proteins from the human sequence (human Telomerase Core Protein 1, “Hs TCP1”), <i>E. aediculatus </i>p123 (“Ep p123”), <i>S. pombe </i>tez1 (“Sp Tez1”), <i>S. cerevisiae </i>EST2 (“Sc Est2”), and consensus sequence. In this Figure various motifs are indicated.</p>
    <p>To obtain a full-length clone, probing of a cDNA library and 5′-RACE were used to obtain clones encoding portions of the previously uncloned regions. In these experiments, RACE (Rapid Amplification of cDNA Ends; See e.g., M. A. Frohman, “RACE: Rapid Amplification of cDNA Ends,” in Innis et al. (eds), <i>PCR Protocols: A Guide to Methods and Applications [</i>1990], pp. 28-38; and Frohman et al., Proc. Natl. Acad. Sci., 85:8998-9002 [1988]) was used to generate material for sequence analysis. Four such clones were generated and used to provide additional 5′ sequence information (pFWRP5, 6, 19, and 20).</p>
    <p>In addition, human cDNA libraries (inserted into lambda) were probed with the EcoRI-NotI fragment of the clone. One lambda clone, designated “lambda 25-1.1” (ATCC accession #209024), was identified as containing complementary sequences. FIG. 75 shows a restriction map of this lambda clone. The human cDNA insert from this clone was subcloned as an EcoRI restriction fragment into the EcoRI site of commercially available phagemid pBluescriptIISK+ (Stratagene), to create the plasmid “pGRN121,” which was deposited with the ATCC (ATCC accession #209016). Preliminary results indicated that plasmid pGRN121 contains the entire open reading frame (ORF) sequence encoding the human telomerase protein.</p>
    <p>The cDNA insert of plasmid pGRN121 was sequenced using techniques known in the art. FIG. 70 provides a restriction site and function map of plasmid pGRN121 identified based on this preliminary work. The results of this preliminary sequence analysis are shown in FIG. <b>71</b>. From this analysis, and as shown in FIG. 70, a putative start site for the coding region was identified at approximately 50 nucleotides from the EcoRI site (located at position 707), and the location of the telomerase-specific motifs, “FFYVTE” (SEQ ID NO:361), “PKP,” “AYD,” “QG”, and “DD,” were identified, in addition to a putative stop site at nucleotide #3571 (See, FIG. 72, which shows the DNA and corresponding amino acid sequences for the open reading frames in the sequence (“a”, “b”, and “c”)). However, due to the preliminary nature of the early sequencing work, the reading frames for the various motifs were found not to be in alignment.</p>
    <p>Additional analysis conducted on the pGRN121 indicated that the plasmid contained significant portions from the 5′-end of the coding sequence not present on clone 712562. Furthermore, pGRN121 was found to contain a variant coding sequence that includes an insert of approximately 182 nucleotides. This insert was found to be absent from the clone. As with the <i>E. aediculatus </i>sequences, such variants can be tested in functional assays, such as telomerase assays to detect the presence of functional telomerase in a sample.</p>
    <p>Further sequence analysis resolved the cDNA sequence of pGRN121 to provide a contiguous open reading frame that encodes a protein of molecular weight of approximately 127,000 daltons, and 1132 amino acids as shown in FIG. 74. A refined map of pGRN121 based on this analysis, is provided in FIG. <b>73</b>. The results of additional sequence analysis of the hTRT cDNA are presented in FIG. 16 SEQ ID NO:1.</p>
    <heading>Example 2</heading> <heading>Correlation of hTRT Abundance and Cell Immortality</heading> <p>The relative abundance of hTRT mRNA was assessed in six telomerase-negative mortal cell strains and six telomerase-positive immortal cell lines (FIG. <b>5</b>). The steady state level of hTRT mRNA was significantly increased in immortal cell lines that had previously been shown to have active telomerase. Lower levels of the hTRT mRNA were detected in some telomerase-negative cell strains.</p>
    <p>RT-PCR for hTRT, hTR, TP1 (telomerase-associated protein related to Tetrahymena p80 [Harrington et al., 1997, <i>Science </i>275:973; Nakayama et al., 1997, <i>Cell </i>88:875]) and GAPDH (to normalize for equal amounts of RNA template) was carried out on RNA derived from the following cells: (1) human fetal lung fibroblasts GFL, (2) human fetal skin fibroblasts GFS, (3) adult prostate stromal fibroblasts 31 YO, (4) human fetal knee synovial fibroblasts HSF, (5) neonatal foreskin fibroblasts BJ, (6) human fetal lung fibroblasts IMR90, and immortalized cell lines (7) melanoma LOX IMVI, (8) leukemia U251, (9) NCI H23 lung carcinoma, (10) colon adenocarcinoma SW620, (11) breast tumor MCF7, (12) 293 adenovirus E1 transformed human embryonic kidney cell line.</p>
    <p>hTRT nucleic acid was amplified from cDNA using oligonucleotide primers Lt5 and Lt6 (Table 2) for a total of 31 cycles (94° C. 45s, 60° C. 45s, 72° C. 90s). GAPDH was amplified using primers K136 (5′-CTCAGACACCATGGGGAA GGTGA; SEQ ID NO:552) and K137 (5′-ATGATCTTGAGGCTGTTGTCATA; SEQ ID NO:553) for a total of 16 cycles (94° C. 45 s, 55° C. 45 s, 72° C. 90 s). hTR was amplified using primers F3b (5′-TCTAACCCTAACTGAGAAGGGCGTAG; SEQ ID NO:597) and R3c (5′-GTTTGCTCTAGAATGAACGGTGGAAG; SEQ ID NO:598) for a total of 22 cycles (94° C. 45s, 55° C. 45 s,72° C. 90s). TP1 mRNA was amplified using primers TP1.1 and TP1.2 for 28 cycles (cycles the same as hTRT). Reaction products were resolved on an 8% polyacrylamide gel, stained with SYBR Green (Molecular Probes) and visualized by scanning on a Storm 860 (Molecular Dynamics). The results, shown in FIG. 5, demonstrate that hTRT mRNA levels correlate directly with telomerase activity levels in the cells tested.</p>
    <heading>Example 3</heading> <heading>Characterization of an hTRT Intronic Sequence</heading> <p>A putative intron was first identified by PCR amplification of human genomic DNA, as described in this example, and subsequently confirmed by sequencing the genomic clone λGφ5 (see Example 4). PCR amplification was carried out using the forward primer TCP1.57 paired individually with the reverse primers TCP1.46, TCP1.48, TCP1.50, TCP1.52, TCP1.54, TCP1.56, and TCP1.58 (see Table 2). The products from genomic DNA of the TCP1.57/TCP1.46, TCP1.48, TCP1.50, TCP1.52, TCP1.54, or TCP1.56 amplifications were approximately 100 basepairs larger than the products of the pGRN121 amplifications. The TCP1.57/TCP1.58 amplification was the same on either genomic or pGRN121 DNA. This indicated the genomic DNA contained an insertion between the sites for TCP1.58 and TCP1.50. The PCR products of TCP1.57/TCP1.50 and TCP1.57/TCP1.52 were sequenced directly, without subcloning, using the primers TCP1.39, TCP1.57, and TCP1.49.</p>
    <p>As shown below, the 104-base intronic sequence SEQ ID NO:7 is inserted in the hTRT mRNA (shown in bold) at the junction corresponding to bases 274 and 275 of FIG. <b>16</b>:</p>
    <p>CCCCCCGCCGCCCCCTCCTTCCGCCAG/GTGGGCCTCCCCGGGGTCGGCGTCCGGCTGGGGTTGAGGGCGGCCGGGGGGAACCAGCGACATGCGGAGAGCAGCGCAGGCGACTCAGGGCGCTTCCCCCGCAG/GTGTCCTGCCTGAAGGAGCTGGTGGCCCGAGTGCTGCAG (SEQ ID NO:599)</p>
    <p>The “/” indicates the splice junctions; the sequence shows good matches to consensus 5′ and 3′ splice site sequences typical for human introns.</p>
    <p>This intron contains motifs characteristic of a topoisomerase II cleavage site and a NFκB binding site (see FIG. <b>21</b>). These motifs are of interest, in part, because expression of topoisomerase II is up regulated in most tumors. It functions to relax DNA by cutting and rewinding the DNA, thus increasing expression of particular genes. Inhibitors of topoisomerase II have been shown to work as anti-tumor agents. In the case of NFκB, this transcription factor may play a role in regulation of telomerase during terminal differentiation, such as in early repression of telomerase during development and so is another target for therapeutic intervention to regulate telomerase activity in cells.</p>
    <heading>Example 4</heading> <heading>Cloning of Lambda Phage GΦ5 and Characterization of hTRT Genomic Sequences</heading> <p>A. Lambda GΦ5</p>
    <p>A human genomic DNA library was screened by PCR and hybridization to identify a genomic clone containing hTRT RNA coding sequences. The library was a human fibroblast genomic library made using DNA from WI38 lung fibroblast cells (Stratagene, Cat # 946204). In this library, partial Sau3AI fragments are ligated into the XhoI site of Lambda FIX711 Vector (Stratagene), with an insert size of 9-22 kb.</p>
    <p>The genomic library was divided into pools of 150,000 phage each, and each pool screened by nested PCR (outer primer pair TCP1.52 &amp; TCP1.57; inner pair TCP1.49 &amp; TCP1.50, see Table 1). These primer pairs span a putative intron (see Example 3, supra) in the genomic DNA of hTRT and ensured the PCR product was derived from a genomic source and not from contamination by the hTRT cDNA clone. Positive pools were further subdivided until a pool of 2000 phage was obtained. This pool was plated at low density and screened via hybridization with a DNA fragment encompassing basepairs 1552-2108 of FIG. 16 (restriction sites SphI and EcoRV, respectively).</p>
    <p>Two positive clones were isolated and rescreened via nested PCR as described above; both clones were positive by PCR. One of the clones (λGΦ5) was digested with NotI, revealing an insert size of approximately 20 kb. Subsequent mapping (see below) indicated the insert size was 15 kb and that phage GΦ5 contains approximately 13 kb of DNA upstream from the start site of the cDNA sequence.</p>
    <p>Phage GΦ5 was mapped by restriction enzyme digestion and DNA sequencing. The resulting map is shown in FIG. <b>7</b>. The phage DNA was digested with NcoI and the fragments cloned into pBBS167. The resulting subclones were screened by PCR to identify those containing sequence corresponding to the 5′ region of the hTRT cDNA. A subclone (pGRN140) containing a 9 kb NcoI fragment (with hTRT gene sequence and 4-5 kb of lambda vector sequence) was partially sequenced to determine the orientation of the insert. pGRN 140 was digested using SalI to remove lambda vector sequences, resulting in pGRN144. pGRN144 was then sequenced. The results of the sequencing are provided in FIG. <b>21</b>. The 5′ end of the hTRT mRNA corresponds to base 2441 of FIG. <b>21</b>. As indicated in FIG. 7, two Alu sequence elements are located 1700 base pairs upstream of the hTRT cDNA 5′ end and provide a likely upstream limit to the promoter region of hTRT. The sequence also reveals an intron positioned at bases 4173 in FIG. 21, 3′ to the intron described in Example 3, supra.</p>
    <p>B. Additional Genomic Clones</p>
    <p>In addition to the genomic clone described above, two P1 bacteriophage clones and one human BAC clone are provided as illustrative embodiments of the invention. P1 inserts are usually 75-100 kb, and BAC inserts are usually over 100 Kb.</p>
    <p>The P1 clones (DMPC-HFF#1-477(F6)-GS #15371 and DMPC-HEF#1-1103(H6)-GS #15372) were obtained by PCR screening of a human P1 library derived from human foreskin fibroblast cells (Shepherd et al., 1994, PNAS USA 91:2629) using primers TCP1.12 and UTR2 which amplify the 3′ end of hTRT. These clones were both negative (failed to amplify) with primers that amplify the 5′ end of hTRT.</p>
    <p>The human BAC clone (326 E 20) was obtained with a hybridization screen of a BAC human genomic library using an 1143 bp Sph1/Xmn1 fragment of pGRN121 (FIG. 16; bases 1552-2695) that encompasses the RT motif region. The clone is believed to include the 5′ end of the gene. The hTRT genomic clones in this example are believed to encompass the entire hTRT gene.</p>
    <heading>Example 5</heading> <heading>Chromosomal Location of hTRT Gene</heading> <p>The hTRT gene was localized to chromosome 5p by radiation hybrid mapping (Boehnke et al., 1991, <i>Am J Hum Genet </i>49:1174; Walter et al., 1994, <i>Nature Genet </i>7:22) using the medium resolution Stanford G3 panel of 83 RH clones of the whole human genome (created at the Stanford Human Genome Center). A human lymphoblastoid cell line (donor; rM) was exposed to 10,000 rad of x-rays and was then fused with nonirradiated hamster recipient cells (A3). Eighty-three independent somatic cell hybrid clones were isolated, and each represents a fusion event between an irradiated donor cell and a recipient hamster cell. The panel of G3 DNA was used for ordering markers in the region of interest as well as establishing the distance between these markers.</p>
    <p>The primers used for the RH mapping were TCP1.12 and UTR2 with amplification conditions of 94° C. 45 sec, 55° C. 45 sec, 72° C. 45 sec, for 45 cycles using Boehringer Mannheim Taq buffer and Perkin-Elmer Taq. The 83 pools were amplified independently and 14 (17%) scored positive for hTRT (by appearance of a 346 bp band). The amplification results were submitted to Stanford RH server, which then provided the map location, 5p, and the closest marker, STS D5S678.</p>
    <p>By querying the Genethon genome mapping web site, the map location identified a YAC that contains the STS marker D5S678: CEPH YAC 780_C<sub>—</sub>3 Size: 390,660 kb. This YAC also contained chromosome 17 markers. This result indicated that the hTRT gene is on chromosome 5, near the telomeric end. There are increased copy numbers of 5p in a number of tumors. Cri-du-chat syndrome also has been mapped to deletions in this region.</p>
    <heading>Example 6</heading> <heading>Design and Construction of Vectors for Expression of hTRT Proteins and Polynucleotides</heading> <p>Expression of hTRT in Bacteria</p>
    <p>The following portion of this example details the design of hTRT-expressing bacterial and eukaryotic cell expression vectors to produce large quantities of full-length, biologically active hTRT. Generation of biologically active hTRT protein in this manner is useful for telomerase reconstitution assays, assaying for telomerase activity modulators, analysis of the activity of newly isolated species of hTRT, identifying and isolating compounds which specifically associate with hTRT, analysis of the activity of an hTRT variant protein that has been site-specifically mutated, and as an immunogen, as a few examples.</p>
    <p>pThioHis A/hTRT Bacterial Expression Vector</p>
    <p>To produce large quantities of full-length hTRT, the bacterial expression vector pThioHis A (Invitrogen, San Diego, Calif.) was selected as an expression system. The hTRT-coding insert includes nucleotides 707 to 4776 of the hTRT insert in the plasmid pGRN121. This nucleotide sequence includes the complete coding sequence for the hTRT protein.</p>
    <p>This expression vector of the invention is designed for inducible expression in bacteria. The vector can be induced to express, in <i>E. coli, </i>high levels of a fusion protein composed of a cleavable, HIS tagged thioredoxin moiety and the full length hTRT protein. The use of the expression system was in substantial accordance with the manufacturer's instructions. The amino acid sequence of the fusion protein encoded by the resulting vector of the invention is shown below; (-*-) denotes an enterokinase cleavage site:</p>
    <p>MSDKIIHLTDDSFDTDVLKADGAILVDFWAHWCGPCKMIAPILDEIADEYQGKLTVAKLRIDHNPGTAPKYGIRGIPTLLLFKNGEVAATKVGALSKGQLKEFLDANLAGSGSGDDDDK-*-VPMHELEIFEFAAASTQRCVLLRTWEALAPATPAMPRAPRCRAVRSLLRSHYREVLPLATFVRRLGPQGWRLVQRGDPAAFRALVAQCLVCVPWDARPPPAAPSFRQVSCLKELVARVLQRLCERGAKNVLAFGFALLDGARGGPPEAFTTSVRSYLPNTVTDALRGSGAWGLLLRRVGDDVLVHLLARCALFVLVAPSCAYQVCGPPLYQLGAATQARPPPHASGPRRRLGCERAWNHSVREAGVPLGLPAPGARRRGGSASRSLPLPKRPRRGAAPEPERTPVGQGSWAHPGRTRGPSDRGFCVVSPARPAEEATSLEGALSGTRHSHPSVGRQHHAGPPSTSRPPRPWDTPCPPVYAETKHFLYSSGDKEQLRPSFLLSSLRPSLTGARRLVETIFLGSRPWMPGTPRRLPRLPQRYWQMRPLFLELLGNHAQCPYGVLLKTHCPLRAAVTPAAGVCAREKPQGSVAAPEEEDTDPRRLVQLLRQHSSPWQVYGFVRACLRRLVPPGLWGSRHNERRFLRNTKKFISLGKHAKLSLQELTWKMSVRDCAWLRRSPGVGCVPAAEHRLREEILAKFLHWLMSVYVVELLRSFFYVTETTFQKNRLFFYRKSVWSKLQSIGIRQHLKRVQLRELSEAEVRQHREARPALLTSRLRFIPKPDGLRPIVNMDYVVGARTFRREKRAERLTSRVKALFSVLNYERARRPGLLGASVLGLDDIHRAWRTFVLRVRAQDPPPELYFVKVDVTGAYDTIPQDRLTEVIASIIKPQNTYCVRRYAVVQKAAHGHVRKAFKSHVSTLTDLQPYMRQFVAHLQETSPLRDAVVIEQSSSLNEASSGLFDVFLRFMCHHAVRIRGKSYVQCQGIPQGSILSTLLCSLCYGDMENKLFAGIRRDGLLLRLVDDFLLVTPHLTHAKTFLRTLVRGVPEYGCVVNLRKTVVNFPVEDEALGGTAFVQMPAHGLFPWCGLLLDTRTLEVQSDYSSYARTSIRASLTFNRGFKAGRNMRRKLFGVLRLKCHSLFLDLQVNSLQTVCTNIYKILLLQAYRFHACVLQLPFHQQVWKNPTFFLRVISDTASLCYSILKAKNAGMSLGAKGAAGPLPSEAVQWLCHQAFLLKLTRHRVTYVPLLGSLRTAQTQLSRKLPGTTLTALEAAANPALPSDFKTILD (SEQ ID NO:600)</p>
    <p>pGEX−2TK with hTRT Nucleotides 3272 to 4177 of pGRN121</p>
    <p>This construct of the invention is used to produce fusion protein for, e.g., the purpose of raising polyclonal and monoclonal antibodies to hTRT protein. Fragments of hTRT can also be used for other purposes, such as to modulate telomerase activity, for example, as a dominant-negative mutant or to prevent the association of a telomerase component with other proteins or nucleic acids.</p>
    <p>To produce large quantities of an hTRT protein fragment, the <i>E. coli </i>expression vector pGEX-2TK (Pharmacia Biotech, Piscataway N.J.) was selected, and used essentially according to manufacturer's instructions to make an expression vector of the invention. The resulting construct contains an insert derived from nucleotides 3272 to 4177 of the hTRT insert in the plasmid pGRN121. The vector directs expression in <i>E. coli </i>of high levels of a fusion protein composed of glutathione-S-transferase sequence (underlined below), thrombin cleavage sequence (double underlined), recognition sequence for heart muscle protein kinase (italicized), residues introduced by cloning in brackets ([GSVTK]; SEQ ID NO:601) and hTRT protein fragment (in bold) as shown below:</p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" pgwide="1" tabstyle="monospace" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="1" colwidth="280pt" align="left"> </colspec> <colspec colname="2" colwidth="70pt" align="left"> </colspec> <colspec colname="3" colwidth="91pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">
                  <u>MSPILGYWKIKGLVQPTRLLLEYLEEKYEEHLYERDEGDKWRNKKFELGLEFPNLPYYIDGD</u>
                </td>
                <td class="description-td">(SEQ ID NO:602)</td>
                <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <u>VKLTQSMAIIRYIADKHNMLGGCPKERAEISMLEGAVLDIRYGVSRIAYSKDFETLKVDFLS</u>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <u>KLPEMLKMFEDRLCHKTYLNGDHVTHPDFMLYDALDVVLYMDPMCLDAFPKLVCFKKRIEAI</u>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> <u>PQIDKYLKSSKYIAWPLQGWQATFGGGDHPPKSD</u> <u style="double">LVPRGS</u> <i>RRASV</i>[GSVTK]<b>IPQGSILSTL</b> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <b>LCSLCYGDMENKLFAGIRRDGLLLRLVDDFLLVTPHLTHAKTFLRTLVRGVPEYGCVVNLRK</b>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <b>TVVNFPVEDEALGGTAFVQMPAHGLFPWCGLLLDTRTLEVQSDYSSYARTSIRASVTFNRGF</b>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <b>KAGRNMRRKLFGVLRLKCHSLFLDLQVNSLQTVCTNIYKILLLQAYRFHACVLQLPFHQQVW</b>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <b>KNPTFFLRVISDTASLCYSILKAKNAGMSLGAKGAAGPLPSEAVQWLCHQAFLLKLTRHRVT</b>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <b>YVPLLGSLRTAQTQLSRKLPGTTLTALEAAANPALPSDFKTILD</b>
                </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p>When this fusion protein was expressed, it formed insoluble aggregates. It was treated generally as described above, in the section entitled purification of proteins from inclusion bodies. Specifically, induced cells were suspended in PBS (20 mM sodium phosphate, pH 7.4, 150 mM NaCl) and disrupted by sonication. NP-40 was added to 0.1%, and the mixture was incubated for 30 minutes at 4° C. with gentle mixing. The insoluble material was collected by centrifugation at 25,000 g for 30 minutes at 4° C. The insoluble material was washed once in 4M urea in PBS, collected by centrifugation, then washed again in PBS. The collected pellet was estimated to contain greater than 75% fusion protein. This material was dried in a speed vacuum, then suspended in adjuvant for injection into mice and rabbits for the generation of antibodies. Separation of the recombinant protein from the glutathione S-transferase moiety is accomplished by site-specific proteolysis using thrombin according to manufacturer's instructions.</p>
    <p>pGEX-2TK with hTRT Nucleotides 2426 to 3274 of pGRN121 with HIS-8 Tag</p>
    <p>To produce large quantities of a fragment of hTRT, another <i>E. coli </i>expression vector pGEX-2TK construct was prepared. This construct contains an insert derived from nucleotides 2426 to 3274 of the hTRT insert in the plasmid pGRN121 and a sequence encoding eight consecutive histidine residues (HIS-8 Tag). To insert the HIS-8 TAG, the pGEX-2TK vector with hTRT nucleotides 2426 to 3274 of pGRN 121 was linearized with BamH1. This opened the plasmid at the junction between the GST-thrombin-heart muscle protein kinase and the hTRT coding sequence. A double stranded oligonucleotide with BamH1 compatible ends was ligated to the linearized plasmid resulting in the in-frame introduction of eight histidine residues upstream of the hTRT sequence.</p>
    <p>The vector directs expression in <i>E coli </i>of high levels of a fusion protein composed of glutathione-S-transferase sequence (underlined); thrombin cleavage sequence (double underlined); recognition sequence for heart muscle protein kinase (italicized); a set of three and a set of five residues introduced by cloning are in brackets ([GSV] and [GSVTK] SEQ ID NO:601); eight consecutive histidines (also double underlined); and hTRT protein fragment (in bold):</p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" pgwide="1" tabstyle="monospace" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="1" colwidth="245pt" align="left"> </colspec> <colspec colname="2" colwidth="70pt" align="left"> </colspec> <colspec colname="3" colwidth="126pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">
                  <u>MSPILGYWKIKGLVQPTRLLLEYLEEKYEEHLYERDEGDKWRNKKFELGLEFPNLPY</u>
                </td>
                <td class="description-td">(SEQ ID NO:603)</td>
                <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <u>YIDGDVKLTQSMAIIRYIADKHNMLGGCPKERAEISMLEGAVLDIRYGVSRIAYSKDF</u>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <u>ETLKVDFLSKLPEMLKMFEDRLCHKTYLNGDHVTHPDFMLYDALDVVLYMDPMCL</u>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <u>DAFPKLVCFKKRIEAIPQIDKYLKSSKYIAWPLQGWQATFGGGDHPPKSD</u>
                  <u style="double">LVPRGS</u>
                  <i>RR</i>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> <i>ASV</i>[GSV]<u style="double">HHHHHHHH</u>[GSVTK]<b>MSVYVVELLRSFFYVTETTFQKNRLFFYRPSVWS</b> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <b>KLQSIGIRQHLKRVQLRELSEAEVRQHREARPALLTSRLRFIPKPDGLRPIVNMD</b>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <b>YVVGARTFRREKRAERLTSRVKALFSVLNYERARRPGLLGASVLGLDDIHRAW</b>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <b>RTFVLRVRAQDPPPELYFVKVDVTGAYDTIPQDRLTEVIASIIKPQNTYCVRRYA</b>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <b>VVQKAAHGHVRKAFKSHVSTLTDLQPYMRQFVAHLQETSPLRDAVVIEQSSSLN</b>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <b>EASSGLFDVFLRFMCHHAVRIRGKSYVQCQGI</b>
                </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p>Each of the pGEX-2TK vectors of the invention can be used to produce fusion protein for the purpose of raising polyclonal and monoclonal antibodies to hTRT protein. Additionally, this fusion protein can be used to affinity purify antibodies raised to hTRT peptides that are encompassed within the fusion protein. Separation of the recombinant protein from the glutathione S-transferase moiety can be accomplished by site-specific proteolysis using thrombin according to manufacturer's instructions.</p>
    <p>pGEX-2TK with hTRT Nucleotides 2426 to 3274 of pGRN121, no HIS-8 Tag</p>
    <p>To produce large quantities of a fragment of hTRT, another <i>E. coli </i>expression vector pGEX-2TK construct was prepared.</p>
    <p>This construct contains an insert derived from nucleotides 2426 to 3274 of the hTRT insert in the plasmid pGRN121, but without the HIS-8 tag of the construct described above. The vector directs expression in <i>E coli </i>of high levels of a fusion protein composed of glutathione-S-transferase (underlined), thrombin cleavage sequence (double underlined), recognition sequence for heart muscle protein kinase (italicized), residues introduced by cloning in brackets ([GSVTK]; SEQ ID NO:601) and hTRT protein fragment (in bold):</p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" pgwide="1" tabstyle="monospace" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="1" colwidth="266pt" align="left"> </colspec> <colspec colname="2" colwidth="70pt" align="left"> </colspec> <colspec colname="3" colwidth="105pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">
                  <u>MSPILGYWKIKGLVQPTRLLLEYLEEKYEEHLYERDEGDKWRNKKFELGLEFPNLPYYIDGD</u>
                </td>
                <td class="description-td">(SEQ ID NO:604)</td>
                <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <u>VKLTQSMAIIRYIADKHNMLGGCPKERAEISMLEGAVLDIRYGVSRIAYSKDFETLKVDFLS</u>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <u>KLPEMLKMFEDRLCHKTYLNGDHVTHPDFMLYDALDVVLYMDPMCLDAFPKLVCFKKRIEAI</u>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> <u>PQIDKYLKSSKYIAWPLQGWQATFGGGDHPPKSD</u> <u style="double">LVPRGS</u> <i>RRASV</i>[GSVTK]<b>MSVYVVELLR</b> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <b>SFFYVTETTFQKNRLFFYRPSVWSKLQSIGIRQHLKRVQLRELSEAEVRQHREARPALLTSR</b>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <b>LRFIPKPDGLRPIVNMDYVVGARTFRREKRAERLTSRKALFSVLNYERARRPGLLGASVLGL</b>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <b>DDIHRAWRTFVLRVRAQDPPPEYFVKVDVTGAYDTIPQDRLTEVIASIIKPQNTYCVRRYAV</b>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <b>VQKAAHGVRKAFKSHVSTLTDLQPYMRQFVAHLQETSPLRDAVVIEQSSSLNEASGLFDVFL</b>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <b>RFMCHHAVRIRGKSYVQCQGI</b>
                </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p>pGEX-2TK with hTRT Nucleotides 1625 to 2458 of pGRN121</p>
    <p>To produce large quantities of a fragment of hTRT protein, another <i>E. coli </i>expression vector pGEX-2TK construct was prepared.</p>
    <p>This construct contains an insert derived from nucleotides 1625 to 2458 of the hTRT insert in the plasmid pGRN121. The vector directs expression in <i>E coli </i>of high levels of a fusion protein composed of glutathione-S-transferase, (underlined), thrombin cleavage sequence (double underlined), recognition sequence for heart muscle protein kinase (italicized) residues introduced by cloning in brackets ([GSVTK]; SEQ ID NO:601) and hTRT protein fragment (in bold):</p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" pgwide="1" tabstyle="monospace" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="1" colwidth="245pt" align="left"> </colspec> <colspec colname="2" colwidth="70pt" align="left"> </colspec> <colspec colname="3" colwidth="126pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">
                  <u>
                    <b>MSPILGYWKIKGLVQPTRLLLEYLEEKYEEHLYERDEGDKWRNKKFELGLEFP</b>
                  </u>
                </td>
                <td class="description-td">(SEQ ID NO:605)</td>
                <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <u>
                    <b>NLPYYIDGDVKLTQSMAIIRYIADKHNMLGGCPKERAEISMLEGAVLDIRYGVS</b>
                  </u>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <u>
                    <b>RIAYSKDFETLKVDFLSKLPEMLKMFEDRLCHKTYLNGDHVTHPDFMLYDALD</b>
                  </u>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <u>
                    <b>VVLYMDPMCLDAFPKLVCFKKRIEAIPQIDKYLKSSKYIAWPLQGWQATFGGG</b>
                  </u>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <u>
                    <b>DHPPKSDL</b>
                  </u>
                  <u style="double">
                    <b>LVPRGS</b>
                  </u>
                  <b>[GSVTK]ATSLEGALSGTRHSHPSVGRQHHAGPPSTSR</b>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <b>PPRPWDTPCPPVYAETKHFLYSSGDKEQLRPSFLLSSLRPSLTGARRLVETIFLGS</b>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <b>RPWMPGTPRRLPRLPQRYWQMRPLFLELLGNHAQCPYGVLLKTHCPLRAAVT</b>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <b>PAAGVCAREKPQGSVAAPEEEDTDPRRLVQLLRQHSSPWQVYGFVRACLRRLV</b>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <b>PPGLWGSRHNERRFLRNTKKFISLGKHAKLSLQELTWKMSVRDCAWLRRSPGV</b>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <b>GCVPAAEHRLREEILAKFLHWLMSVYVVELLRS</b>
                </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p>pGEX-2TK with hTRT Nucleotides 782 to 1636 of pGRN121</p>
    <p>To produce large quantities of a fragment of hTRT protein, another <i>E. coli </i>expression vector pGEX-2TK construct was prepared.</p>
    <p>This construct contains an insert derived from nucleotides 782 to 1636 of the hTRT insert in the plasmid pGRN121. The vector directs expression in <i>E coli </i>of high levels of a fusion protein composed of glutathione-S-transferase, (underlined), thrombin cleavage sequence (double underlined), recognition sequence for heart muscle protein kinase (italicized) residues introduced by cloning in brackets ([GSVTK]; SEQ ID NO:601) and hTRT protein fragment (in bold):</p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" pgwide="1" tabstyle="monospace" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="1" colwidth="266pt" align="left"> </colspec> <colspec colname="2" colwidth="70pt" align="left"> </colspec> <colspec colname="3" colwidth="105pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">
                  <u>MSPILGYWKIKGLVQPTRLLLEYLEEKYEEHLYERDEGDKWRNKKFELGLEFPNLPYYIDGD</u>
                </td>
                <td class="description-td">(SEQ ID NO:606)</td>
                <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <u>VKLTQSMAIIRYIADKHNMLGGCPKERAEISMLEGAVLDIRYGVSRIAYSKDFETLKVDFLS</u>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <u>KLPEMLKMFEDRLCHKTYLNGDHVTHPDFMLYDALDVVLYMDPMCLDAFPKLVCFKKRIEAI</u>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> <u>PQIDKYLKSSKYIAWPLQGWQATFGGGDHPPKSD</u> <u style="double">LVPRGS</u> <i>RRASV</i>[GSVTK]<b>MPRAPRCRAV</b> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <b>RSLLSHYREVLPLATFVRRLGPQGWRLVQRGDPAAFRALVAQCLVCVPWDARPPAAPSFRQV</b>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <b>SCLKELVARVLQRLCERGAKNVLAFGFALLDGARGGPPEATTSVRSYLPNTVTDALRGSGAW</b>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <b>GLLLRRVGDDVLVHLLARCALFVLVAPCAYQVCGPPLYQLGAATQARPPPHASGPRRRLGCE</b>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <b>RAWNHSVREAGVPLGLPAPGARRRGGSASRSLPLPKRPRRGAAPEPERTPVGQGSWAHPGRT</b>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <b>RGPSDRGFCVVSPARPAEEATSL</b>
                </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p>pT7FLhTRT with hTRT cDNA Lacking 5′-Non-Coding Sequence</p>
    <p>As described above, in one embodiment, the invention provides for an hTRT that is modified in a site-specific manner to facilitate cloning into bacterial, mammalian, yeast and insect expression vectors without any 5′ untranslated hTRT sequence. In some circumstances, minimizing the amount of non-protein encoding sequence allows for improved protein production (yield) and increased mRNA stability. In this embodiment of the invention, the hTRT gene's 5′ non-coding region was removed before cloning into a bacterial expression vector.</p>
    <p>This was effected by engineering an additional restriction endonuclease site just upstream (5′) to the start (ATG) codon of the hTRT coding sequence (FIG. <b>16</b>). The creation of a restriction site just 5′ to the coding region of the protein allows for efficient production of a wide variety of vectors that encode fusion proteins, such as fusion proteins comprising labels and peptide TAGs, for immunodetection and purification.</p>
    <p>Specifically, the oligonucleotide 5′- CCGGCCACCCCCCATATGCCGCGCGCTCCC-3′ (SEQ ID NO:607) was used as described above to modify hTRT cDNA nucleotides 779 to 781 of the hTRT cDNA (FIG. 16) from GCG to CAT. These 3 nucleotides are the last nucleotides before the ATG start codon so they do not modify the protein sequence. The change in sequence results in the creation of a unique NdeI restriction site in the hTRT cDNA. Single-stranded hTRT DNA was used as a DNA source for the site directed mutagenesis. The resulting plasmid was sequenced to confirm the success of the mutagenesis.</p>
    <p>This modification allowed the construction of the following plasmid of the invention, designated pT7FLhTRT. The site-specifically modified hTRT sequence (addition of the NdeI restriction site) was digested with NdeI and NotI (and filled in with Klenow enzyme to generate blunt ended DNA) to generate an hTRT encoding nucleic acid fragment. The fragment was then cloned into a pSL3418 plasmid previously restriction digested with NdeI and SmaI (also a blunt ended cutter). pSL 3418 is a modified pAED4 plasmid into which a FLAG sequence (Immunex Corp, Seattle Wash.) and an enterokinase sequence are inserted just upstream from the above-referenced NdeI site. This plasmid, designated pT7FLhTR, allows the expression of full length hTRT (with a Flag-Tag at its 5′ end) in an <i>E. coli </i>strain expressing the T7 RNA polymerase.</p>
    <p>Plasmids with hTRT cDNA Lacking 3′-Non-Coding Sequence</p>
    <p>As discussed above, the invention provides for expression vectors containing TRT-encoding nucleic acids in which some or all non-coding sequences have been deleted. In some circumstances, minimizing the amount of non-protein encoding sequence allows for improved protein production (yield) and increases mRNA stability. In this embodiment of the invention, the 3′ untranslated region of hTRT is deleted before cloning into a bacterial expression plasmid.</p>
    <p>The plasmid pGRN121, containing the full length hTRT cDNA,: as discussed above, was first deleted of all Apa1 sites. This was followed by deletion of the MscI-HincII hTRT restriction digest enzyme fragment containing the 3′UTR. The NcoI-XbaI restriction digest fragment containing the stop codon of hTRT was then inserted into the NcoI-XbaI site of pGRN121 to make a plasmid equivalent to pGRN121, designated pGRN124, except lacking the 3′UTR.</p>
    <p>Bacterial Expression Vectors Using Antibiotic Selection Markers</p>
    <p>The invention also provides for bacterial expression vectors that can contain selection markers to confer a selectable phenotype on transformed cells and sequences coding for episomal maintenance and replication such that integration into the host genome is not required. For example, the marker may encode antibiotic resistance, particularly resistance to chloramphenicol (see Harrod (1997) <i>Nucleic Acids Res. </i>25: 1720-1726), kanamycin, G418, bleomycin and hygromycin, to permit selection of those cells transformed with the desired DNA sequences, see for example, Blondelet-Rouault (1997) Gene 190:315-317; and Mahan (1995) <i>Proc Natl Acad Sci USA </i>92:669-673.</p>
    <p>In one embodiment of the invention, the full length hTRT was cloned into a modified BlueScript plasmid vector (Stratagene, San Diego, Calif.), designated pBBS235, into which a chloramphenicol antibiotic resistence gene had been inserted. The NotI fragment from pGRN124 (discussed above) containing the hTRT ORF into the NotI site of pBBS235 so that the TRT ORF is in the opposite orientation of the vector's Lac promoter. This makes a plasmid that is suitable for mutageneis of plasmid inserts, such as TRT nucleic acids of the invention. This plasmid construct, designated pGRN125, can be used in the methods of the invention involving mutagenesis of telomerase enzyme and TRT protein coding sequences and for in vitro transcription of hTRT using the T7 promoter (and in vitro transcription of antisense hTRT using the T3 promoter).</p>
    <p>In another embodiment of the invention, NotI restriction digest fragments from pGRN124 containing the hTRT ORF were subcloned into the NotI site of pBBS235 (described above) so the TRT ORF is in the same orientation as the vector's Lac promoter. This makes a plasmid, designated pGRN126, that can be used for expression of full length hTRT in <i>E. coli. </i>The expressed product will contain 29 amino acids encoded by the vector pBBS235, followed by 18 amino acids encoded by the 5′UTR of hTRT, followed by the full length hTRT protein.</p>
    <p>In a further embodiment of the invention, in vitro mutagenesis of pGRN125 was done to convert the hTRT initiating ATG codon into a Kozak consensus and create EcoRI and BglII restriction digest sites to facilitate cloning into expression vectors. The oligonucleotide 5′-TGCGCACGTGGGAAGCCCTGGCagatctgAattCcaCcATGCCGCGCGCTCCCCGCTG-3′ (SEQ ID NO:608) (altered nucleotides in lower case) was used in the mutagenesis procedure. The resulting expression vector was designated pGRN127.</p>
    <p>In another embodiment of the invention, the second Asp of the TRT “DD motif” was converted to an alanine to create a non-functional telomerse enzyme, thus creating a mutant TRT protein for use as a dominant/negative mutant. The hTRT coding sequence was mutagenized in vitro using the oligonucleotide 5′-CGGGACGGGCTGCTCCTGCGTTTGGTGGAcGcgTTCTTGTTGGTGACACCTCACCTCACC-3′ (SEQ ID NO:609) to convert the asparagine codon for residue 869 (Asp869) to an alanine (Ala) codon. This also created an MluI restriction enzyme site. The resulting expression plasmid was designated pGRN130, which also contains the Kozak consensus sequence as described for pGRN127.</p>
    <p>The invention also provides a vector designed to express an antisense sequence fragment of hTRT. The pGRN126 plasmid was cut to completion with MscI and SmaI restriction enzymes and religated to delete over 95% of the hTRT ORF. One SmaI-MscI fragment was re-inserted during the process to recreate CAT activity. This unpurified plasmid was then redigested with SalI and EcoRI and the fragment containing the initiating codon of the hTRT ORF was inserted into the SalI-EcoRI sites of pBBS212 to make an antisense expression plasmid expressing the antisense sequence spanning the 5′UTR and 73 bases pair residues of the hTRT ORF (in mammalian cells). This plasmid was designated pGRN135.</p>
    <p>Expression of hTRT Telomerase in Yeast</p>
    <p>The present invention also provides hTRT-expressing yeast expression vectors to produce large quantities of full-length, biologically active hTRT.</p>
    <p> <i>Pichia pastoris </i>Expression Vector pPICZ B and Full Length hTRT</p>
    <p>To produce large quantities of full-length, biologically active hTRT, the <i>Picha pastoris </i>expression vector PPICZ B (Invitrogen, San Diego, Calif.) was selected. The hTRT-coding sequence insert was derived from nucleotides 659 to 4801 of the hTRT insert in plasmid pGRN121. This nucleotide sequence includes the full-length sequence encoding hTRT. This expression vector is designed for inducible expression in <i>P. pastoris </i>of high levels of full-length, unmodified hTRT protein. Expression is driven by a yeast promoter, but the expressed sequence utilizes the hTRT initiation and termination codons. No exogenous codons were introduced by the cloning. The resulting pPICZ B/hTRT vector was used to transform the yeast.</p>
    <p> <i>Pichia pastoris </i>Expression Vector hTRT-His6/pPICZ B</p>
    <p>A second <i>Picha pastoris </i>expression vector of the invention derived from pPICZ B, also contains the full-length sequence encoding hTRT derived from nucleotides 659 to 4801 of the hTRT insert in the plasmid pGRN121. This hTRT-His6/pPICZ B expression vector encodes full length hTRT protein fused at its C-terminus to the Myc epitope and His6 reporter tag sequences. The hTRT stop codon has been removed and replaced by vector sequences encoding the Myc epitope and the His6 reporter tag as well as a stop codon. This vector is designed to direct high-level inducible expression in yeast of the following fusion protein, which consists of hTRT sequence (underlined), vector sequences in brackets ([L] and [NSAVD]; SEQ ID NO:610) the Myc epitope (double underlined), and the His6 tag (italicized):</p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" pgwide="1" tabstyle="monospace" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="1" colwidth="259pt" align="left"> </colspec> <colspec colname="2" colwidth="70pt" align="left"> </colspec> <colspec colname="3" colwidth="112pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">
                  <u>MPRAPRCRAVRSLLRSHYREVLPLATFVRRLGPQGWRLVQRGDPAAFRALVAQCLV</u>
                </td>
                <td class="description-td">(SEQ ID NO:611)</td>
                <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <u>CVPWDARPPPAAPSFRQVSCLKELVARVLQRLCERGAKNVLAFGFALLDGARGGPP</u>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <u>EAFTTSVRSYLPNTVTDALRGSGAWGLLLRRVGDDVLVHLLARCALFVLVAPSCAY</u>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <u>QVCGPPLYQLGAATQARPPPHASGPRRRLGCERAWNHSVREAGVPLGLPAPGARRR</u>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <u>GGSASRSLPLPKRPRRGAAPEPERTPVGQGSWAHPGRTRGPSDRGFCVVSPARPAEE</u>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <u>ATSLEGALSGTRHSHPSVGRQHHAGPPSTSRPPRPWDTPCPPVYAETKHFLYSSGDKE</u>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <u>QLRPSFLLSSLRPSLTGARRLVETIFLGSRPWMPGTPRRLPRLPQRYWQMRPLFLELLG</u>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <u>NHAQCPYGVLLKTHCPLRAAVTPAAGVCAREKPQGSVAAPEEEDTDPRRLVQLLRQ</u>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <u>HSSPWQVYGFVRACLRRLVPPGLWGSRHNERRFLRNTKKFISLGKHAKLSLQELTW</u>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <u>KMSVRDCAWLRRSPGVGCVPAAEHRLREEILAKFLHWLMSVYVVELLRSFFYVTET</u>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <u>TFQKNRLFFYRKSVWSKLQSIGIRQHLKRVQLRELSEAEVRQHREARPALLTSRLRFIP</u>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <u>KPDGLRPIVNMDYVVGARTFRREKRAERLTSRVKALFSVLNYERARRPGLLGASVLG</u>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <u>LDDIHRAWRTFVLRVRAQDPPPELYFVKVDVTGAYDTIPQDRLTEVIASIIKPQNTYC</u>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <u>VRRYAVVQKAAHGHVRKAFKSHVSTLTDLQPYMRQFVAHLQETSPLRDAVVIEQSS</u>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <u>SLNEASSGLFDVFLRFMCHHAVRIRGKSYVQCQGIPQGSILSTLLCSLCYGDMENKLF</u>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <u>AGIRRDGLLLRLVDDFLLVTPHLTHAKTFLRTLVRGVPEYGCVVNLRKTVVNFPVED</u>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <u>EALGGTAFVQMPAHGLFPWCGLLLDTRTLEVQSDYSSYARTSIRASLTFNRGFKAGR</u>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <u>NMRRKLFGVLRLKCHSLFLDLQVNSLQTVCTNIYKILLLQAYRFHACVLQLPFHQQV</u>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">
                  <u>WKNPTFFLRVISDTASLCYSILKAKNAGMSLGAKGAAGPLPSEAVQWLCHQAFLLKL</u>
                </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> <u>TRHRVTYVPLLGSLRTAQTQLSRKLPGTTLTALEAAANPALPSDFKTILD</u>[L]<u style="double">EQKLISE</u> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> <u style="double">EDL</u>[<u style="double">N</u>SAVD]<i>HHHHHH</i> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p>Expression of hTRT in Insect Cells</p>
    <p>The present invention also provides hTRT telomerase-expressing insect cell expression vectors that produce large quantities of full-length, biologically active hTRT.</p>
    <p>Baculovirus Expression Vector pVL1393 and Full Length hTRT</p>
    <p>The telomerase coding sequence of interest was cloned into the baculovirus expression vector pVL1393 (Invitrogen, San Diego, Calif.). This construct was subsequently cotransfected into <i>Spodoptera fungupeida </i>(sf-9) cells with linearized DNA from <i>Autograph california </i>nuclear polyhedrosis virus (Baculogold-AcMNPV). The recombinant baculoviruses obtained were subsequently plaque purified and expanded following standard protocols.</p>
    <p>This expression vector provides for expression in insect cells of high levels of full-length hTRT protein. Expression is driven by a baculoviral polyhedrin gene promoter. No exogenous codons were introduced by the cloning.</p>
    <p>Baculovirus Expression Vector pBlueBacHis2 B and Full Length hTRT</p>
    <p>To produce large quantities of full-length, biologically active hTRT, the baculovirus expression vector pBlueBacHis2 B (Invitrogen, San Diego, Calif.) was selected as a source of control elements. The hTRT-coding insert consisted of nucleotides 707 to 4776 of the hTRT insert in plasmid pGRN121.</p>
    <p>A full length hTRT with a His6 and Anti-Xpress tags (Invitrogen) was also constructed. This vector also contains an insert consisting of nucleotides 707 to 4776 of the hTRT insert from the plasmid pGRN121. The vector directs expression in insect cells of high levels of full length hTRT protein fused to a cleavable 6-histidine and Anti-Xpress tags, and the amino acid sequence of the fusion protein is shown below; (-*-) denotes enterokinase cleavage site:</p>
    <p>MPRGSHHHHHHGMASMTGGQQMGRDLYDDDDL-*-DPSSRSAAGTMEFAAASTQRCVLLRTWEALAPATPAMPRAPRCRAVRSLLRSHYREVLPLATFVRRLGPQGWRLVQRGDPAAFRALVAQCLVCVPWDARPPPAAPSFRQVSCLKELVARVLQRLCERGAKNVLAFGFALLDGARGGPPEAFTTSVRSYLPNTVTDALRGSGAWGLLLRRVGDDVLVHLLARCALFVLVAPSCAYQVCGPPLYQLGAATQARPPPHASGPRRRLGCERAWNHSVREAGVPLGLPAPGARRRGGSASRSLPLPKRPRRGAAPEPERTPVGQGSWAHPGRTRGPSDRGFCVVSPARPAEEATSLEGALSGTRHSHPSVGRQHHAGPPSTSRPPRPWDTPCPPVYAETKHFLYSSGDKEQLRPSFLLSSLRPSLTGARRLVETIFLGSRPWMPGTPRRLPRLPQRYWQMRPLFLELLGNHAQCPYGVLLKTHCPLRAAVTPAAGVCAREKPQGSVAAPEEEDTDPRRLVQLLRQHSSPWQVYGFVRACLRRLVPPGLWGSRHNERRFLRNTKKFISLGKHAKLSLQELTWKMSVRDCAWLRRSPGVGCVPAAEHRLREEILAKFLHWLMSVYVVELLRSFFYVTETTFQKNRLFFYRKSVWSKLQSIGIRQHLKRVQLRELSEAEVRQHREARPALLTSRLRFIPKPDGLRPIVNMDYVVGARTFRREKRAERLTSRVKALFSVLNYERARRPGLLGASVLGLDDIHPRAWRTFVLRVRAQDPPPELYFVKVDVTGAYDTIPQDRLTEVIASIIKPQNTYCVRRYAVVQKAAHGHVRKAFKSHVSTLTDLQPYMRQFVAHLQETSPLRDAVVIEQSSSLNEASSGLFDVFLRFMCHHAVRIRGKSYVQCQGIPQGSILSTLLCSLCYGDMENKLFAGIRRDGLLLRLVDDFLLVTPHLTHAKTFLRTLVRGVPEYGCVVNLRKTVVNFPVEDEALGGTAFVQMPAHGLFPWCGLLLDTRTLEVQSDYSSYARTSIRASLTFNRGFKAGRNMRRKLFGVLRLKCHSLFLDLQVNSLQTVCTNIYKILLLQAYRFHACVLQLPFHQQVWKNPTFFLRVISDTASLCYSILKAKNAGMSLGAKGAAGPLPSEAVQWLCHQAFLLKLTRHRVTYVPLLGSLRTAQTQLSRKLPGTTLTALEAAANPALPSDFKTILD (SEQ ID NO:612)</p>
    <p>Baculovirus Expression Vector pBlueBac4.5 and Full Length hTRT Protein</p>
    <p>To produce large quantities of full-length, biologically active hTRT, a second baculovirus expression vector, pBlueBac4.5 (Invitrogen, San Diego, Calif.) was constructed. The hTRT-coding insert also consisted of nucleotides 707 to 4776 of the hTRT from the plasmid pGRN121.</p>
    <p>Baculovirus Expression Vector pMelBacB and Full Length hTRT Protein</p>
    <p>To produce large quantities of full-length, biologically active hTRT, a third baculovirus expression vector, pMelBacB (Invitrogen, San Diego, Calif.) was constructed. The hTRT-coding insert also consists of nucleotides 707 to 4776 of the hTRT insert from the plasmid pGRN121.</p>
    <p>pMelBacB directs expression of full length hTRT in insect cells to the extracellular medium through the secretory pathway using the melittin signal sequence. High levels of full length hTRT are thus secreted. The melittin signal sequence is cleaved upon excretion, but is part of the protein pool that remains intracellularly. For that reason, it is indicated in parentheses in the following sequence. The sequence of the fusion protein encoded by the vector is shown below:</p>
    <p>(MKFLVNVALVFMVVYISYIYA)-*-DPS SRSAAGTMEFAAASTQRCVLLRTWEALAPATPAMPRAPRCRAVRSLLRSHYREVLPLATFVRRLGPQGWRLVQRGDPAAFRALVAQCLVCVPWDARPPPAAPSFRQVSCLKELVARVLQRLCERGAKNVLAFGFALLDGARGGPPEAFTTSVRSYLPNTVTDALRGSGAWGLLLRRVGDDVLVHLLARCALFVLVAPSCAYQVCGPPLYQLGAATQARPPPHASGPRRRLGCERAWNHSVREAGVPLGLPAPGARRRGGSASRSLPLPKRPRRGAAPEPERTPVGQGSWAHPGRTRGPSDRGFCVVSPARPAEEATSLEGALSGTRHSHPSVGRQHHAGPPSTSRPPRPWDTPCPPVYAETKHFLYSSGDKEQLRPSFLLSSLRPSLTGARRLVETIFLGSRPWMPGTPRRLPRLPQRYWQMRPLFLELLGNHAQCPYGVLLKTHCPLRAAVTPAAGVCAREKPQGSVAAPEEEDTDPRRLVQLLRQHSSPWQVYGFVRACLRRLVPPGLWGSRHNERRFLRNTKKFISLGKHAKLSLQELTWKMSVRDCAWLRRSPGVGCVPAAEHRLREEILAKFLHWLMSVYVVELLRSFFYVTETTFQKNRLFFYRKSVWSKLQSIGIRQHLKRVQLRELSEAEVRQHREARPALLTSRLRFFPKPDGLRPIVNMDYVVGARTFRREKRAERLTSRVKALFSVLNYERARRPGLLGASVLGLDDIHRAWRTFVLRVRAQDPPPELYFVKVDVTGAYDTIPQDRLTEVIASIIKPQNTYCVRRYAVVQKAAHGHVRKAFKSHVSTLTDLQPYMRQFVAHLQETSPLRDAVVIEQSSSLNEASSGLFDVFLRFMCHHAVRIRGKSYVQCQGIPQGSILSTLLCSLCYGDMENKLFAGIRRDGLLLRLVDDFLLVTPHLTHAKTFLRTLVRGVPEYGCVVNLRKTVVNFPVEDEALGGTAFVQMPAHGLFPWCGLLLDTRTLEVQSDYSSYARTSIRASLTFNRGFKAGRNMRRKLFGVLRLKCHSLFLDLQVNSLQTVCTNIYKILLLQAYRFHACVLQLPFHQQVWKNPTFFLRVISDTASLCYSILKAKNAGMSLGAKGAAGPLPSEAVQWLCHQAFLLKLTRHRVTYVPLLGSLRTAQTQLSRKLPGTTLTALEAAANPALPSDFKTILD (SEQ ID NO:613)</p>
    <p>Expression of hTRT in Mammalian Cells</p>
    <p>The present invention also provides vectors to produce hTRT in large quantities as full-length, biologically active protein in a variety of mammalian cell lines, which is useful in many embodiments of the invention, as discussed above.</p>
    <p>MPSV-hTRT Expression Plasmids</p>
    <p>The invention also provides for an expression system for use in mammalian cells that gives the highest possible expression of recombinant protein, such as telomerase, without actually modifying the coding sequence (e.g. optimizing codon usage). In one embodiment, the invention provides MPSV mammalian expression plasmids (from plasmid pBBS212, described as pMPSV-TM in Lin J-H (1994) <i>Gene </i>47:287-292) capable of expressing the TRTs of the invention. The MPSV plasmids can be expressed either as stable or transient clones.</p>
    <p>In this expression system, while the hTRT coding sequence itself is unchanged, exogenous transcriptional control elements are incorporated into the vector. The myeloproliferative sarcoma virus (MPSV) LTR (MPSV-LTR) promoter, enhanced by the cytomegalovirus (CMV) enhancer, is incorporated for transcriptional initiation. This promoter consistently shows higher expression levels in cell lines (see Lin J-H (1994) supra). A Kozak consensus sequence can be incorporated for translation initiation (see Kozak (1996) <i>Mamm. Genome </i>7:563-574). All extraneous 5′ and 3′ untranslated hTRT sequences can be removed to insure that these sequences do not interfere with expression, as discussed above. The MPSV plasmid containing the complete hTRT coding sequence, with all extraneous sequences included, is designated pGRN133. A control, hTRT “antisense” plasmid was also constructed. This vector is identical to pGRN133 except that the TRT insert is the antisense sequence of hTRT (the antisense, which control can be used as a vector is designated pGRN134). The MPSV plasmid containing the complete hTRT coding sequence with all other extraneous sequences removed and containing the Kozak consensus sequence is designated pGRN145.</p>
    <p>Two selection markers, PAC (Puromycin-N-acetyl-transferase=Puromycin resistance) and HygB (Hygromycin B=Hygromycin resistance) are present for selection of the plasmids after transfection (see discussion referring to selectable markers, above). Double selection using markers on both sides of the vector polylinker should increase the stability of the hTRT coding sequence. A DHFR (dihydrofolate reductase) encoding sequence is included to allow amplification of the expression cassette after stable clones are made. Other means of gene amplification can also be used to increase recombinant protein yields.</p>
    <p>The invention also provides for MPSV mammalian expression plasmids containing hTRT fusion proteins. In one embodiment, the hTRT sequence, while retaining its 5′ untranslated region, is linked to an epitope flag, such as the IBI FLAG (International Biotechnologies Inc. (IBI), Kodak, New Haven, Conn.) and inserted into the MPSV expression plasmid (designated pGRN147). This particular constuct contains a Kozak translation initiation site. The expressed fusion protein can be purified using the M−1 anti-FLAG octapeptide monoclonal antibody (IBI, Kodak, supra).</p>
    <p>In another embodiment, hTRT is site-specifically altered. One amino acid residue codon is mutagenized, changing the aspartic acid at position 869 to an alanine. This Asp869-&gt;Ala hTRT mutant, retaining its 5′ untranslated region and incorporating a Kozak sequence, was inserted into an MPSV expression plasmid, and designated pGRN146. The Asp869-&gt;Ala hTRT mutant was further engineered to contain the FLAG sequence, as described above, and the insert cloned into an MPSV expression plasmid. One such expression plasmid is designated pGRN154-I. Specifically, for pGRN154-I, an Eam1105I restriction digest fragment from pGRN146 containing the Kozak sequence-containing “front end” (5′ segment) of hTRT is cloned into the Eam1105I sites of pGRN147 (see above) to make an MPSV expression plasmid capable of expressing hTRT with a Kozak sequence, the above-described D869-&gt;A mutation, and the IBI flag.</p>
    <p>Another embodiment of the invention is an expression plasmid derived from pGRN146. The mammalian expression plasmid, designated pGRN152, was generated by excising the EcoRI fragment from plasmid pGRN146 (containing the hTRT ORF) and cloned into the EcoRI site of pBBS212 to remove the 5′UTR of hTRT. The hTRT is oriented so that its expression is controlled by the MPSV promoter. This makes a mammalian expression plasmid that expresses hTRT with a Kozak consensus sequence and the D869-&gt;A mutation, and uses the MPSV promoter.</p>
    <p>The invention provides for a mammalian expression vector in which hTRT is oriented so that the hTRT coding sequence is driven by the MPSV promoter. For example, an EcoR1 restriction digest fragment from pGRN137 containing the hTRT open reading frame (ORF) was cloned into the EcoR1 site of pBBS212 (see below), thus removing the 5′ untranslated region (5′-UTR) of hTRT. pGRN137 was constructed by excising a SalI-Sse8387I fragment from pGRN130, described below, containing the Kozak mutation of hTRT into the Sal 1-SSE 8387I sites of pGRN136, making a mammalian expression plasmid expressing hTRT containing a Kozak consensus sequence off the MPSV promoter. Plasmid pGRN136 was constructed by excising a HindIII SalI fragment from pGRN126 containing the hTRT ORF and cloning it into the HindIII SalI sites of plasmid, pBBS242, making a mammalian expression plasmid expressing hTRT off the MPSV promoter). This makes a mammalian expression plasmid, designated pGRN145, that expresses hTRT with a Kozak consensus sequence using the MPSV promoter. See also the pGRN152 MPSV promoter-driven mammalian expression vector described below.</p>
    <p>hTRT Expressed in 293 Cells using Episomal Vector pEBVHis</p>
    <p>An episomal vector, pEBVHis (Invitrogen, San Diego, Calif.) was engineered to express an hTRT fusion protein comprising hTRT fused to an N-terminal extension epitope tag, the Xpress epitope (Invitrogen, San Diego, Calif.) (designated pGRN122). The NotI hTRT fragment from pGRN121 containing the hTRT ORF was cloned into the NotI site of pEBVHisA so that the hTRT ORF is in the same orientation as the vector's Rous Sarcoma Virus (RSV) promoter. In this orientation the His6 flag was relatively closer to the N-terminus of hTRT.</p>
    <p>A vector was also constructed containing as an insert the antisense sequence of hTRT and the epitope tag (the plasmid designated pGRN123, which can be used as a control). The vector was transfected into 293 cells and translated hTRT identified and isolated using an antibody specific for the Xpress epitope. pEBVHis is a hygromycin resistant EBV episomal vector that expresses the protein of interest fused to a N-terminal peptide. Cells carrying the vector are selected and expanded, then nuclear and cytoplasmic extracts prepared. These and control extracts are immunoprecipitated with anti-Xpress antibody, and the immunoprecipitated beads are tested for telomerase activity by conventional assay.</p>
    <p>Expression of Recombinant hTRT in Mortal, Normal Diploid Human Cells</p>
    <p>In one embodiment of the invention, recombinant hTRT and necessary telomerase enzyme complex components can be expressed in normal, diploid mortal cells to increase their proliferative capacity or to immortalize them, or to facilitate immortalizing them. This allows one to obtain diploid immortal cells with an otherwise normal phenotype and karotype. As discussed above, this use of telomerase has enormous commercial utility.</p>
    <p>Sense hTRT (FIG. 16) and antisense hTRT were cloned into a CMV vector. These vectors were purified and transiently transfected into two normal, mortal, diploid human cell clones. The human clones were young passage diploid human BJ and IMR90 cell strains.</p>
    <p>Analysis of telomerase activity using a TRAP assay utilizing the TRAPeze™ Kit (Oncor, Inc., Gaithersburg, Md.) showed that transfection of sense hTRT—but not antisense hTRT—generated telomerase activity in both the BJ and IMR90 cell strains.</p>
    <p>Expression of Recombinant hTRT in Immoralized IMR90 Human Cells</p>
    <p>Using the same hTRT sense construct cloned into CMV vectors used in the above described diploid human BJ and IMR90 cell strains studies, immortalized SW13 ALT pathway cell line (an IMR90 cell immortalized with SV40 antigen) was transiently transfected. A TRAP assay (TRAPeze, Oncor, Inc, Gaithersburg, Md.) demonstrated that telomerase activity was generated in the sense construct transfected cells.</p>
    <p>Vectors for Regulated Expression of hTRT in Mammalian Cells: Inducible and Repressible Expression of hTRT</p>
    <p>The invention provides vectors that can be manipulated to induce or repress the expression of the TRTs of the invention, such as hTRT. For example, the hTRT coding sequence can be cloned into the Ecdysone-Inducible Expression System from Invitrogen (San Diego, Calif.) and the Tet-On and Tet-off tetracycline regulated systems from Clontech Laboratories, Inc. (Palo Alto, Calif.). Such inducible expression systems are provided for use in the methods of the invention where it is important to control the level or rate of transcription of transfected TRT. For example, the invention provides for cell lines immortalized through the expression of hTRT; such cells can be rendered “mortal” by inhibition of hTRT expression by the vector through transcriptional controls, such as those provided by the Tet-Off system. The invention also provides for methods of expressing TRT only transiently to avoid the constitutive expression of hTRT, which may lead to unwanted “immortalization” of the transfected cells, as discussed above.</p>
    <p>The Ecdysone-Inducible Mammalian Expression System is designed to allow regulated expression of the gene of interest in mammalian cells. The system is distinguished by its tightly regulated mechanism that allows almost no detectable basal expression and greater than 200-fold inducibility in mammalian cells. The expression system is based on the heterodimeric ecdysone receptor of Drosophila. The Ecdysone-Inducible Expression System uses a steroid hormone ecdysone analog, muristerone A, to activate expression of hTRT via a heterodimeric nuclear receptor. Expression levels have been reported to exceed 200-fold over basal levels with no effect on mammalian cell physiology “Ecdysone-Inducible Gene Expression in Mammalian Cells and Transgenic Mice” (1996) <i>Proc. Natl. Acad. Sci. USA </i>93, 3346-3351). Once the receptor binds ecdysone or muristerone, an analog of ecdysone, the receptor activates an ecdysone-responsive promoter to give controlled expression of the gene of interest. In the Ecdysone-Inducible Mammalian Expression System, both monomers of the heterodimeric receptor are constitutively expressed from the same vector, pVgRXR. The ecdysone-responsive promoter, which ultimately drives expression of the gene of interest, is located on a second vector, pIND, which drives the transcription of the gene of interest.</p>
    <p>The hTRT coding sequence is cloned in the pIND vector (Clontech Laboratories, Inc, Palo Alto, Calif.), which contains 5 modified ecdysone response elements (E/GREs) upstream of a minimal heat shock promoter and the multiple cloning site. The construct is then transfected in cell lines which have been pre-engineered to stably express the ecdysone receptor. After transfection, cells are treated with muristerone A to induce intracellular expression from pIND.</p>
    <p>The Tet-on and Tet-off expression systems (Clontech, Palo Alto, Calif.) give access to the regulated, high-level gene expression systems described by Gossen (1992) “Tight control of gene expression in mammalian cells by tetracycline responsive promoters” <i>Proc. Natl. Acad. Sci. USA </i>89:5547-5551, for the Tet-Off transcription repression system; and Gossen (1995) “Transcriptional activation by tetracycline in mammalian cells” <i>Science </i>268:1766-1769, for the Tet-On inducible transcriptional system. In “Tet-Off” transformed cell lines, gene expression is turned on when tetracycline (Tc) or doxycycline (“Dox;” a Tc derivative) is removed from the culture medium. In contrast, expression is turned on in Tet-On cell lines by the addition of Tc or Dox to the medium. Both systems permit expression of cloned genes to be regulated closely in response to varying concentrations of Tc or Dox.</p>
    <p>This system uses the “pTRE” as a response plasmid that can be used to express a gene of interest. Plasmid pTRE contains a multiple cloning site (MCS) immediately downstream of the Tet-responsive PhCMV*−1 promoter. Genes or cDNAs of interest inserted into one of the sites in the MCS will be responsive to the tTA and rtTA regulatory proteins in the Tet-Off and Tet-On systems, respectively. PhCMV*−1 contains the Tet-responsive element (TRE), which consists of seven copies of the 42-bp tet operator sequence (tetO). The TRE element is just upstream of the minimal CMV promoter (PminCMV), which lacks the enhancer that is part of the complete CMV promoter in the pTet plasmids. Consequently, PhCMV*−1 is silent in the absence of binding of regulatory proteins to the tetO sequences. The cloned insert must have an initiation codon. In some cases, addition of a Kozak consensus ribosome binding site may improve expression levels; however, many cDNAs have been efficiently expressed in Tet systems without the addition of a Kozak sequence. pTRE-Gene X plasmids are cotransfected with pTK-Hyg to permit selection of stable transfectants.</p>
    <p>Setting up a Tet-Off or Tet-On expression system generally requires two consecutive stable transfections to create a “double-stable” cell line that contains integrated copies of genes encoding the appropriate regulatory protein and TRT under the control of a TRE. In the first transfection, the appropriate regulatory protein is introduced into the cell line of choice by transfection of a “regulator plasmid” such as pTet-Off or pTet-On vector, which expresses the appropriate regulatory proteins. The hTRT cloned in the pTRE “response plasmid” is then introduced in the second transfection to create the double-stable Tet-Off or Tet-On cell line. Both systems give very tight on/off control of gene expression, regulated dose-dependent induction, and high absolute levels of gene expression.</p>
    <p>Expression Recombinant hTRT With DHFR and Adenovirus Sequences</p>
    <p>The pGRN155 plasmid construct was designed for transient expression of hTRT cDNA in mammalian cells. A Kozak consensus is inserted at the 5′ end of the hTRT sequence. The hTRT insert contains no 3′ or 5′ UTR. The hTRT cDNA is inserted into the EcoRI site of p91023(B) (Wong (1985) <i>Science </i>228:810-815). The hTRT insert is in the same orientation as the DHFR ORF.</p>
    <p>Plasmid pGRN155 contains the SV40 origin and enhancer just upstream of an adenovirus promoter, a tetracycline resistance gene, an <i>E. coli </i>origin and an adenovirus VAI and VAII gene region. This expression cassette contains, in the following order: the adenovirus major late promoter; the adenovirus tripartite leader; a hybrid intron consisting of a 5′ splice site from the first exon of the tripartite leader and a 3′ splice site from the mouse immunoglobulin gene; the hTRT cDNA; the mouse DHFR coding sequence; and, the SV40 polyadenylation signal.</p>
    <p>The adenovirus tripartite leader and the VA RNAs have been reported to increase the efficiency with which polycistronic mRNAs are translated. DHFR sequences have been reported to enhance the stability of hybrid mRNA. DHFR sequences also can provide a marker for selection and amplification of vector sequences. See Logan (1984) <i>Proc. Natl Acad. Sci. USA </i>81:3655); Kaufman (1985) Proc. Natl. Acad. Sci. USA 82: 689; and Kaufman (1988) Focus (Life Technologies, Inc.), Vol.10, no. 3). This makes the expression vector particularly useful for transient expression.</p>
    <p>Other expression plamids of the invention are described for illustrative purposes.</p>
    <p>pGRN121</p>
    <p>The EcoRI fragment from lambda clone 25-1.1.6 containing the entire cDNA encoding hTRT protein was inserted into the EcoRI site of pBluescriptIISK+such that the 5′ end of the cDNA is near the T7 promoter in the vector. The selectable marker that is used with this vector is ampicillin.</p>
    <p>pGRN122</p>
    <p>The NotI fragment from pGRN121 containing the hTRT ORF was inserted into the NotI site of pEBVHisA so that the coding sequence is operably linked to the RSV promoter. This plasmid expresses a fusion protein composed of a His6 flag fused to the N-terminal of the hTRT protein. The selectable marker that is used with this vector is ampicillin or hygromycin.</p>
    <p>pGRN123</p>
    <p>The NotI fragment from pGRN121 containing the hTRT ORF was inserted into the NotI site of pEBVHisA so that the coding sequence is in the opposite orientation as the RSV promoter, thus expressing antisense hTRT.</p>
    <p>pGRN124</p>
    <p>Plasmid pGRN121 was deleted of all ApaI sites followed by deletion of the MscI-HincII fragment containing the 3′UTR. The Nco-XbaI fragment containing the stop codon of the hTRT coding sequence was then inserted into the Nco-XbaI sites of pGRN121 to make a plasmid equivalent to pGRN121 except lacking the 3′UTR, which may be preferred for increased expression levels in some cells.</p>
    <p>pGRN125</p>
    <p>The NotI fragment from pGRN124 containing the hTRT coding sequence was inserted into the NotI site of pBBS235 so that the open reading frame is in the opposite orientation of the Lac promoter. The selectable marker that is used with this vector is chloramphenicol.</p>
    <p>pGRN126</p>
    <p>The NotI fragment from pGRN124 containing the hTRT coding sequence was inserted into the NotI site of pBBS235 so that the hTRT coding sequence inserted is in the same orientation as the Lac promoter.</p>
    <p>pGRN127</p>
    <p>The oligonucleotide 5′-TGCGCACGTGGGAAGCCCTGGCagatctgAattCcaCcATGCCGCGCGCTCCCCGCTG-3′ (SEQ ID NO:608) was used in in vitro mutagenesis of pGRN125 to convert the initiating ATG codon of the hTRT coding sequence into a Kozak consensus sequence and create EcoRI and BglII sites for cloning. Also, oligonucleotide COD2866 was used to convert AmpS to AmpR (ampicillin resistant) and oligonucleotide COD1941 was used to convert CatR (chloramphenicol resistant) to CatS (chloramphenicol sensitive).</p>
    <p>pGRN128</p>
    <p>The oligonucleotide 5′-TGCGCACGTGGGAAGCCCTGGCagatctgAattCcaCcATGCCGCGCGCTCCCCGCTG-3′ (SEQ ID NO:608) is used in in vitro mutagenesis to convert the initiating ATG codon of hTRT into a Kozak consensus and create EcoRI and BglII sites for cloning. Also, oligo 5′-CTGCCCTCAGACTTCAAGACCATCCTGGACTACAAGGACGACGATGACAAATGAATTCAGATCTGCGGCCGCCACCGCGGTGGAGCTCCAGC-3′ (SEQ ID NO:614) is used to insert the IBI Flag (International Biotechnologies Inc. (131), Kodak, New Haven, Conn.) at the C-terminus and create EcoRI and BglII sites for cloning. Also, COD2866 is used to convert AmpS to AmpR and COD1941 is used to convert CatR to CatS.</p>
    <p>pGRN129</p>
    <p>The oligonucleotide 5′-CGGGACGGGCTGCTCCTGCGTTTGGTGGAcGcgTTCTTGTTGGTGACACCTCACCTCACC-3′ (SEQ ID NO:609) was used by in vitro mutagenesis to convert Asp869 to an Ala codon (i.e. the second Asp of the DD motif was converted to an Alanine to create a dominant/negative hTRT mutant). This also created a MluI site. Also, oligonucleotide 5′-CTGCCCTCAGACTTCAAGACCATCCTGGACTACAAGGACGACGATGACAAATGAATTCAGATCTGCGGCCGCCACCGCGGTGGAGCTCCAGC-3′ SEQ ID NO:614) was used to insert the IBI Flag at the C-terminus and create EcoRI and BglII sites for cloning. Also, COD2866 was used to convert AmpS to AmpR and COD1941 was used to convert CatR to CatS.</p>
    <p>pGRN130</p>
    <p>The oligonucleotide 5′-CGGGACGGGCTGCTCCTGCGTTTGGTGGAcGcgTTCTTGTTGGTGACACCTCACCTCACC-3′ (SEQ ID NO:609) was used in in vitro mutagenesis to convert the Asp869 codon into an Ala codon (i.e. the second Asp of the DD motif was converted to an Alanine to make a dominant/negative variant protein). This also created an MluI site. Also, the oligonucleotide 5′-TGCGCACGTGGGAAGCCCTGGCagatctgAattCcaCcATGCCGCGCGCTCCCCGCTG-3′ (SEQ ID NO:608) was used in in vitro mutagenesis to convert the initiating ATG codon of the hTRT coding sequence into a Kozak consensus sequence and create EcoRI and BglII sites for cloning. Also, COD2866 was used to convert AmpS to AmpR and COD1941 was used to convert CatR.</p>
    <p>pGRN131</p>
    <p>The EcoRI fragment from pGRN128 containing the hTRT ORF with Kozak sequence and IBI Flag mutations is inserted into the EcoRI site of pBBS212 so that the hTRT ORF is expressed off the MPSV promoter. Plasmid pBSS212 contains a MPSV promoter, the CMV enhancer, and the SV40 polyadenylation site.</p>
    <p>pGRN132</p>
    <p>The EcoRI fragment from pGRN128 containing the hTRT ORF with Kozak sequence and 1131 Flag mutations is inserted into the EcoRI site of pBBS212 so that the antisense of the hTRT ORF is expressed off the MPSV promoter.</p>
    <p>pGRN133</p>
    <p>The EcoRI fragment from pGRN121 containing the hTRT coding sequence was inserted into the EcoRI site of pBBS212 so that the hTRT protein is expressed under the control of the MPSV promoter.</p>
    <p>pGRN134</p>
    <p>The EcoRI fragment from pGRN121 containing the hTRT coding sequence was inserted into the EcoRI site of pBBS212 so that the antisense of the hTRT coding sequence is expressed under the control of the MPSV promoter. The selectable markers used with this vector are Chlor/HygB/PAC.</p>
    <p>pGRN135</p>
    <p>Plasmid pGRN126 was digested to completion with MscI and SmaI and religated to delete over 95% of the hTRT coding sequence inserted. One SmaI-MscI fragment was re-inserted during the process to recreate the Cat activity for selection. This unpurified plasmid was then redigested with SalI and EcoRI and the fragment containing the initiating codon of the hTRT coding sequence was inserted into the SalI-EcoRI sites of pBBS212. This makes an antisense expression plasmid expressing the antisense of the 5′UTR and 73 bases of the coding sequence. The selectable markers used with this vector are Chlor/HygB/PAC.</p>
    <p>pGRN136</p>
    <p>The HindIII-SalI fragment from pGRN126 containing the hTRT coding sequence was inserted into the HindIII-SalI sites of pBBS242.</p>
    <p>pGRN137</p>
    <p>The SalI-Sse8387I fragment from pGRN130 containing the Kozak sequence was inserted into the SalI-Sse8387I sites of pGRN136.</p>
    <p>pGRN138</p>
    <p>The EcoRI fragment from pGRN124 containing hTRT minus the 3′UTR was inserted into the EcoRI site of pEGFP-C2 such that the orientation of the hTRT is the same as the EGFP domain.</p>
    <p>pGRN139</p>
    <p>The oligonucleotide 5′-CTGCCCTCAGACTTCAAGACCATCCTGGACTACAAGGACGACGATGACAAATGAATTCAGATCTGCGGCCGCCACCGCGGTGGAGCTCCAGC-3′ (SEQ ID NO:614) was used to insert the IBI Flag at the C-terminus of hTRT in pGRN125 and create EcoRI and BglII sites for cloning. Also, COD2866 was used to convert AmpS to AmpR and COD 1941 was used to convert CatR to CatS.</p>
    <p>pGRN140</p>
    <p>The NcoI fragment containing the upstream sequences of genomic hTRT and the first intron of hTRT from lambdaG55 was inserted into the NcoI site of pBBS167. The fragment is oriented so that hTRT is in the same direction as the Lac promoter.</p>
    <p>pGRN141</p>
    <p>The NcoI fragment containing the upstream sequences of genomic hTRT and the first intron of hTRT from lambdaG55 was inserted into the NcoI site of pBBS167. The fragment is oriented so that hTRT is in the opposite direction as the Lac promoter.</p>
    <p>pGRN142</p>
    <p>The NotI fragment from lambdaGphi5 containing the complete˜15 kbp genomic insert including the hTRT gene promoter region was inserted in the NotI site of plasmid pBBS 185. The fragment is oriented so that the hTRT ORF is in the opposite orientation as the Lac promoter.</p>
    <p>pGRN143</p>
    <p>The NotI fragment from lambdaGphi5 containing the complete˜15 kbp genomic insert including the hTRT gene promoter region was inserted in the NotI site of plasmid pBBS 185. The fragment is oriented so that the hTRT ORF is in the same orientation as the Lac promoter.</p>
    <p>pGRN144</p>
    <p>SAL1 deletion of pGRN140 to remove lambda sequences.</p>
    <p>pGRN145</p>
    <p>This vector was constructed for the expression of hTRT sequences in mammalian cells. The EcoRI fragment from pGRN137 containing the hTRT coding sequence was inserted into the EcoRI site of pBBS212 to remove the portion of the sequence corresponding to the 5′UTR of hTRT mRNA. The hTRT coding sequence is oriented so that it is expressed under the control of the MPSV promoter. The selectable markers used with this vector are Chlor/HygB/PAC.</p>
    <p>pGRN146</p>
    <p>This vector was constructed for the expression of hTRT sequences in mammalian cells. The Sse8387I-NotI fragment from pGRN130 containing the D869A mutation of hTRT was inserted into the Sse8387I-NotI sites of pGRN137. The selectable markers used with this vector are Ampicillin/HygB/PAC.</p>
    <p>pGRN147</p>
    <p>The Sse8387I-NotI fragment from pGRN139 containing the IBI Flag was inserted into the Sse8387I-NotI sites of pGRN137.</p>
    <p>pGRN148</p>
    <p>The BglII-Eco47III fragment from pGRN144 containing the promoter region of hTRT was inserted into the BglII-NruI sites of pSEAP2 to make an hTRT promoter/reporter construct.</p>
    <p>pGRN149</p>
    <p>This vector is an intermediate vector for constructing a hTRT fusion protein expression vector. The mutagenic oligo 5′-cttcaagaccatcctggactttcgaaacgcggccgccaccgcggtggagctcc-3′ (SEQ ID NO:615) was used to add a CSP45I site at the C-terminus of hTRT by in vitro mutagenesis of pGRN125. The “stop” codon of hTRT was deleted and replaced with a Csp45I site. The selectable marker that is used with this vector is ampicillin.</p>
    <p>pGRN150</p>
    <p>The BglII-FspI fragment from pGRN144 containing the promoter region of hTRT was inserted into the BglII-NruI sites of pSEAP2 to make an hTRT promoter/reporter construct.</p>
    <p>pGRN151</p>
    <p>This vector was constructed for the expression of hTRT sequences in mammalian cells. The EcoRI fragment from pGRN147 containing the hTRT coding sequence was inserted into the EcoRI site of pBBS212 to remove the portion of the sequence corresponding to the 5′UTR of the hTRT mRNA. The hTRT coding sequence is oriented so that it is expressed under the control of the MPSV promoter. The selectable markers used with this vector are Chlor/HygB/PAC.</p>
    <p>pGRN152</p>
    <p>The EcoRI fragment from pGRN146 containing the hTRT coding sequence was inserted into the EcoRI site of pBBS212 to remove the portion of the sequence corresponding to the 5′UTR of the hTRT. The hTRT coding sequence is oriented so that it is expressed under the control of the MPSV promoter.</p>
    <p>pGRN153</p>
    <p>The StyI fragment from pGRN130 containing the D869→A mutation of hTRT (hTRT variant coding sequence) was inserted into the StyI sites of pGRN158 to make a plasmid containing the hTRT coding sequence with a Kozak consensus sequence at its 5′-end, an IBI FLAG sequence at its 3′-end (the C-terminus encoding region), and the D869--&gt;A mutation.</p>
    <p>pGRN154</p>
    <p>The EcoRI fragment of pGRN153 containing the hTRT gene was inserted into the EcoRI site of plasmid pBS212 in an orientation such that the hTRT ORF is oriented in the same direction as the MPSV promoter. This makes an MPSV-directed expression plasmid that expresses the hTRT protein with a Kozak consensus sequence at its amino-terminal end, an IBI FLAG at its carboxy-terminal end, and the D869→A mutation</p>
    <p>pGRN155</p>
    <p>This vector was constructed for the expression of hTRT sequences in mammalian cells. The insert included full length cDNA of hTRT minus 5′ and 3′ UTR, and Kozak sequences. The EcoRI fragment from pGRN145 containing the hTRT cDNA with the Kozak consensus and no 3′ or 5′ UTR was inserted into the EcoRI site of p91023(B) such that the hTRT is in the same orientation as the DHFR ORF. This makes a transient expression vector for hTRT. The selectable marker used with this vector is tetracycline.</p>
    <p>pGRN156</p>
    <p>This vector was constructed for the expression of hTRT sequences in mammalian cells. The EcoRI fragment from pGRN 146 containing the D869A mutation of the hTRT cDNA with the Kozak consensus and no 3′ or 5′ UTR was inserted into the EcoRI site of p91023(B) such that the hTRT is in the same orientation as the DHFR ORF. This makes a transient expression vector for hTRT. The insert included full length cDNA of hTRT minus 5′ and 3′ UTR, D869A, and Kozak sequences. The selectable marker used with this vector is tetracycline.</p>
    <p>pGRN157</p>
    <p>This vector was constructed for the expression of hTRT sequences in mammalian cells. The EcoRI fragment from pGRN147 containing the hTRT cDNA with the IBI FLAG at the C-terminus; the Kozak consensus and no 3′ or 5′ UTR into the EcoRI site of p91023(B) such that the hTRT is in the same orientation as the DHFR ORF. This makes a transient expression vector for hTRT. The insert included full length cDNA of hTRT minus 5′ and 3′ UTR, the IBI FLAG sequence, and Kozak sequences. The selectable marker used with this vector is tetracycline.</p>
    <p>pGRN158</p>
    <p>This vector was constructed for the expression and mutagenesis of TRT sequences in <i>E. coli. </i>The EcoRI fragment from pGRN151 containing the hTRT ORF was inserted into the EcoRI site of pBBS183 so that the hTRT ORF is oriented in the opposite direction as the Lac promoter. The insert included full length cDNA of hTRT minus 5′ and 3′ UTR, IBI FLAG sequence, and Kozak sequences. The hTRT coding sequence is driven by a T7 promoter. The selectable marker used with this vector is amphicillin.</p>
    <p>pGRN159</p>
    <p>This vector was constructed for the expression and mutagenesis of TRT sequences in <i>E. coli. </i>The NheI-KpnI fragment from pGRN138 containing the EGFP to hTRT fusion was inserted into the XbaI-KpnI sites of pBluescriptIIKS+. This makes a T7 expression vector for the fusion protein (the coding sequence is driven by a T7 promoter). The insert included full length cDNA of hTRT minus the 3′ UTR as a fusion protein with EGFP. The selectable marker used with this vector is amphicillin.</p>
    <p>pGRN160</p>
    <p>This vector was constructed for the expression of antisense hTR sequences in mammalian cells. The coding sequence is operably linked to an MPSV promoter. The XhoI-NsiI fragment from pGRN90 containing the full length hTR ORF was inserted into the SalI-Sse8387I sites of pBBS295. This makes a transient/stable vector expressing hTR antisense RNA. A GPT marker was incorporated into the vector. The selectable markers used with this vector are Chlor/gpt/PAC.</p>
    <p>pGRN161</p>
    <p>This vector was constructed for the expression of sense hTR sequences in mammalian cells. The XhoI-NniI fragment from pGRN89 containing the full length hTR ORF was inserted into the SalI-Sse8387I sites of pBBS295. This makes a transient/stable vector expressing hTR in the sense orientation. The coding sequence is driven by an MPSV promoter. A GPT marker was incorporated into the vector. The selectable markers used with this vector are Chlor/gpt/PAC.</p>
    <p>pGRN162</p>
    <p>The XhoI-NsiI fragment from pGRN87 containing the full length hTR ORF was inserted into the SalI-Sse8387I sites of pBBS295. This makes a transient/stable vector expressing truncated hTR (from position +108 to +435) in the sense orientation.</p>
    <p>pGRN163</p>
    <p>This vector was constructed for the expression and mutagenesis of TRT sequences in <i>E. coli. </i>The coding sequence is driven by a T7 promoter. Oligonucleotide RA45 (5′-GCCACCCCCGCGCTGCCTCGAGCTCCCCGCTGC-3′; SEQ ID NO:616) is used in in vitro mutagenesis to change the initiating met in hTRT to Leu and introduce an XhoI site in the next two codons after the Leu. Also COD 1941 was used to change CatR to CatS, and introduces a BSPH1 site, and COD 2866 was used to change AmpS to AmpR, introducing an FSP1 site. The selectable marker used with this vector is amphicillin.</p>
    <p>pGRN164</p>
    <p>This vector was constructed for the expression of hTR sequences in <i>E. coli. </i>Primers hTR+1 5′-GGGGAAGCTTTAATACGACTCACTATAGGGTTGCGGAGGGTGG GCCTG-3′ (SEQ ID NO:617) and hTR+445 5′-CCCCGGATCCTGCGCATGTGTGAGCCGAGTCCTGGG-3′ (SEQ ID NO:618) were used to amplify by PCR a fragment from pGRN33 containing the full length hTR with the T7 promoter on the 5′ end (as in hTR+1). A BamHI-HindIII digest of the PCR product was put into the BamHI-HindIII sites of pUC119. The coding sequence operably linked to a T7 promoter. The selectable marker used with this vector is amphicillin. pGRN164 is also called phTR+1.</p>
    <p>pGRN165</p>
    <p>This vector was constructed for the expression and mutagenesis of hTRT sequences in <i>E. coli. </i>The coding sequence is operably linked to a T7 promoter. The EcoRI fragment from pGRN145 containing the hTRT ORF with a Kozak front end was inserted into the EcoRI site of pBluescriptIISK+ so that the hTRT is oriented in the same direction as the T7 promoter. The selectable marker used with this vector is amphicillin.</p>
    <p>pGRN166</p>
    <p>This vector was constructed for the expression and mutagenesis of TRT sequences in mammalian cells. The coding sequence is operably linked to a T7 promoter. The EcoRI fragment from pGRN151 containing the hTRT ORF with a Kozak front end and IBI flag at the back end was inserted into the EcoRI site of pBluescriptIISK+ so that the hTRT ORF is oriented in the same direction as the T7 promoter. The insert included full length cDNA of hTRT minus 5′ and 3′ UTR, FLAG sequence (Immunex Corp, Seattle Wash.), and Kozak sequences. The selectable marker used with this vector is amphicillin.</p>
    <p>pGRN167</p>
    <p>AvRII-StuI fragment from pGRN144 containing the 5′ end of the hTRT ORF was inserted into the XbaI-StuI sites of pBBS161.</p>
    <p>pGRN168</p>
    <p>The EcoRI fragment from pGRN145 containing the optimized hTRT expression cassette was inserted into the EcoRI site of pIND such that the hTRT coding sequence is in the same orientation as the miniCMV promoter.</p>
    <p>pGRN169</p>
    <p>The EcoRI fragment from pGRN145 containing the optimized hTRT expression cassette was inserted into the EcoRI site of pIND such that the hTRT is in the reverse orientation from the miniCMV promoter.</p>
    <p>pGRN170</p>
    <p>The EcoRI fragment from pGRN145 containing the optimized hTRT expression cassette was inserted into the EcoRI site of pIND(sp1) such that the hTRT is in the opposite orientation from the miniCMV promoter.</p>
    <p>pGRN171</p>
    <p>The Eco47III-NarI fragment from pGRN163 was inserted into the Eco47III-NarI sites of pGRN167, putting the M1L mutation into a fragment of the hTRT genomic DNA.</p>
    <p>pGRN172</p>
    <p>The BamHI-StuI fragment from pGRN171 containing the Met to Leu mutation in the hTRT ORF was inserted into the BglII-NruI sites of pSEAP2-Basic.</p>
    <p>pGRN173</p>
    <p>The EcoRV-ECO47III fragment from pGRN144 containing the 5′ end of the hTRT promoter region was inserted into the SrfI-Eco47III sites of pGRN172. This makes a promoter reporter plasmid that contains the promoter region of hTRT from approximately 2.3 kb upstream from the start of the hTRT ORF to just after the first intron in the coding region, with the Met1→Leu mutation.</p>
    <p>pGRN174</p>
    <p>The EcoRI fragment from pGRN145 containing the “optimized” hTRT expression cassette was inserted into the EcoRI site of pIND(sp1) such that the hTRT is in the same orientation as the miniCMV promoter.</p>
    <heading>Example 7</heading> <heading>Reconstitution of Telomerase Activity</heading> <p>A. Co-Expression of hTRT and hTR in vitro</p>
    <p>In this example, the coexpression of hTRT and hTR using an in vitro cell-free expression system is described. These results demonstrate that the hTRT polypeptide encoded by pGRN121 encodes a catalytically active telomerase protein and that in vitro reconstitution (IVR) of the telomerase RNP can be accomplished using recombinantly expressed hTRT and hTR.</p>
    <p>Telomerase activity was reconstituted by adding linearized plasmids of hTRT (pGRN121; 1 μg DNA digested with Xba I) and hTR (phTR+1; 1 μg digested with FspI) to a coupled transcription-translation reticulocyte lysate system (Promega TNT™). phTR+1 is a plasmid which, when linearized with FspI and then transcribed by T7 RNA polymerase, generates a 445 nucleotide transcript beginning with nucleotide+1 and extending to nucleotide 446 of hTR (Autexier et al., 1996, <i>EMBO J </i>15:5928). For a 50 μl reaction the following components were added: 2 μl TNT™ buffer, 1 μl TNT™ T7 RNA polymerase, 1 μl 1 mM amino acid mixture, 40 units Rnasin™ RNase inhibitor, 1 μg each linearized template DNA, and 25 μl TNT™ reticulocyte lysate. Components were added in the ratio recommended by the manufacturer and were incubated for 90 min at 30° C. Transcription was under the direction of the T7 promoter and could also be carried out prior to the addition of reticulocyte lysate with similar results. After incubation, 5 and 10 μl of the programmed transcription-translation reaction were assayed for telomerase activity by TRAP as previously described (Autexier et al., supra) using 20 cycles of PCR to amplify the signal.</p>
    <p>The results of the reconstitution are shown in FIG. <b>10</b>. For each transcription!translation reaction assayed there are 3 lanes: The first 2 lanes are duplicate assays and the third lane is a duplicate sample heat denatured (95° C., 5 min) prior to the TRAP phase to rule out PCR generated artifacts.</p>
    <p>As shown in FIG. 10, reticulocyte lysate alone has no detectable telomerase activity (lane 6). Similarly, no detectable activity is observed when either hTR alone (lane 1) or full length hTRT gene (lane 4) are added to the lysate. When both components are added (lane 2), telomerase activity is generated as demonstrated by the characteristic repeat ladder pattern. When the carboxyl-terminal region of the hTRT gene is removed by digestion of the vector with NcoI (“truncated hTRT”) telomerase activity is abolished (lane 3). Lane 5 shows that translation of the truncated hTRT alone does not generate telomerase activity. Lane “R8” shows a positive control for a telomerase product ladder generated by TRAP of TSR8, a synthetic telomerase product having a nucleotide sequence of 5′-ATTCCGTCGAGCAGAGTTAG[GGTTAG]<sub>7</sub>-3′ (SEQ ID NO:619).</p>
    <p>It was also observed that purification of IVR telomerase resulted in a stronger signal and/or reduced background in certain telomerase activity assays. In some experiments, IVR telomerase activity from co-synthesized components was enriched by fractionation of TNT reactions over DEAE anion exchange membranes (Millipore Ultrafree-MC): 200 μl of the hTRT/hTR TNT reaction was passed through a single DEAE membrane. The membrane was washed with 400 μl of 0.2 M NaCl in buffer A (20 mM HEPES-KOH pH 7.9, 2 mM MgCl<sub>2</sub>, 1 mM EGTA, 10% glycerol, 0.1% Nonidet P-40, 0.1 mM phenylmethylsulfonyl fluoride) and IVR telomerase was eluted from the membrane with 80 μl of 1 M NaCl in buffer A. Alternatively, batch chromatography was used: 400 μl of the TNT reaction was partially purified by batch chromatography using 25 μl of Toso-Haas Q-650M resin. After binding telomerase to the resin, it was washed with 0.1 M NaCl in buffer A, followed by a second wash with 0.18 M NaCl in buffer A and eluted with 100 μl of 0.3 M NaCl in buffer A.</p>
    <p>B. Mixing of hTRT and hTR in vitro</p>
    <p>In vitro reconstitution of telomerase activity was also accomplished by mixing. hTRT was transcribed and translated as described supra, but without the addition of the hTR plasmid. Reconstitution of the telomerase RNP was then accomplished by mixing the hTRT translation mixture with hTR (previously generated by T7 RNA polymerase transcription from phTR+1-Fsp) in the ratio of 2 μl of hTRT translation mix to 2 μl of hTR (1 ug) then incubated for 90 minutes at 30° C. The reaction conditions were adjusted to a KCl concentration of about 0.2 M. (The presence of KCl at a concentration of about 0.1 M to about 1.0 M may enhance telomerase activity or telomerase reconstitution in IVR). This method of hTRT/hTR reconstitution is referred to as “linked reconstitution” or “linked IVR.” Telomerase activity is present (i.e., can be detected) in this mixture. Improved signal was observed following partial purification of the activity by DEAE chromatography. In this case Millipore Ultrafree-MC DEAE Centrifugal Filter Devices were used according to the manufacturer's directions). The buffers used were hypo0.1, hypo0.2, and hypo1.0, where hypo is 20 mM Hepes-KOH, pH 7.9, 2 mM MgCl<sub>2</sub>, 1 mM EGTA, 10% glycerol, 0.1% NP-40, 1 mM DTT, 1 mM Na-metabisulfite, 1 mM benzamidine, and 0.2 mM phenylmethylsulfonylflouride (PMSF), and where 0.1, 0.2 and 1.0 refers to 0.1, 0.2 or 1.0 M KCL. The filters were pre-conditioned with hypo1.0, washed with hypo0.1, the reconstituted telomerase was loaded, the column was washed with hypo0.1 then hypo0.2, and the reconstituted telomerase was eluted with hypo1.0 at half the volume as was loaded. This formulation could be stored frozen at −70° C. and retains activity.</p>
    <p>Telomerase activity was assayed in a two step procedure. In step one, a conventional telomerase assay was performed as described in Morin, 1989, <i>Cell </i>59: 521, except no radiolabel was used. In step two, an aliquot was assayed by the TRAP procedure for 20-30 cycles as described supra. The conventional assay was performed by assaying 1-10 μl of reconstituted telomerase in 40-50 μl final volume of 25 mM Tris-HCl, pH 8.3, 50 mM K-acetate, 1 mM EGTA, 1 mM MgCl<sub>2</sub>, 2 mM dATP, 2 mM TTP, 10 uM dGTP, and 1 uM primer (usually M2, 5′-AATCCGTCGAGCAGAGTT; SEQ ID NO:620) at 30° C. for 60-180 minutes. The reaction was stopped by heating to 95° C. for 5 minutes and 1-10 μl of the first step mixture was carried onto the step two TRAP reaction (50 ul).</p>
    <p>In additional experiments, the synthesis of hTRT and hTR during in vitro reconstitution was monitored by <sup>35</sup>S-methionine incorporation and Northern blotting, respectively. Proteins of approximately the predicted size were synthesized for hTRT (127 kD), hTRT-Nco (85 kD), and pro90hTRT (90 kD) in approximately equal molar amounts relative to each other. The Northern analysis indicated hTR synthesis was the correct size (445 nucleotides) and predominantly intact.</p>
    <p>High levels of reconstitution and telomerase activity were also obtained with 2 μg of linearized pGRN121 in a 50 μl TnT reaction as described supra (Example 7A) except that in place of the hTR template, 4 pmol (0.6 μg) of hTR RNA (previously generated by T7 RNA polymerase transcription from phTR+1-Fsp) was added at the beginning of the TnT reaction and the reaction was incubated at 30EC for 90-120 minutes. Slightly greater (2-5 times) activity was achieved using 1 μg of supercoiled XhTRT- and 16 pmol (2.4 μg) of pre-synthesized hTR RNA set up in a 50 μl TnT reaction, as described supra, with incubation at 30EC for 90-120 minutes. XhTRT-E is an hTRT construct in the pcDNA3.1/His Xpress vector (Invitrogen) in which an optimized ribosome recognition site (Kozak consensus), six histidine residues, and an epitope tag are fused with the hTRT open reading frame.</p>
    <p>Variations of the reconstitution protocols, supra, will be apparent to those of skill. For example, the time and temperature of reconstitution, and presence or concentration of components such as monovalent salt (e.g. NaCl, KCl, potassium acetate, potassium glutamate, and the like), divalent salt (MgCl<sub>2</sub>, MnCl<sub>2</sub>, MgSO<sub>4</sub>, and the like), denaturants (urea, formamide, and the like), detergents (NP-40, Tween, CHAPS, and the like), and alternative improved purification procedures (such as immunoprecipitation, affinity or standard chromatography) can be employed. These and other parameters can be varied in a systematic way to optimize conditions for particular assays or other reconstitution protocols.</p>
    <p>C. Reconstitution Using hTRT Variants and Fusion Proteins</p>
    <p>Reconstitution of telomerase catalytic activity occurred when EGFP-hTRT, a fusion of the enhanced green fluorescent protein to hTRT (see Examples 6 and 15), or epitope-tagged hTRT (IBI FLAG, see Example 6) both reconstituted telomerase activity at approximately wild-type levels were coexpressed with hTR.</p>
    <p>In contrast, telomerase activity was not reconstituted when a variant hTRT, pro90hTRT (missing RT motifs B′, C, D, and E) was used. This demonstrates that pro90hTRT does not possess full telomerase catalytic activity, although it may have other partial activities (e.g., RNA [i.e. hTR] binding ability and function as dominant-negative regulator of telomerase in vivo as described supra).</p>
    <p>D. Assay of in vitro Reconstituted Telomerase Activity Using the Gel Blot and Conventional Telomerase Assay</p>
    <p>The following example demonstrates that in vitro reconstituted (IVR) telomerase can be assayed using conventional telomerase assays in addition to amplification-based assays (i.e., TRAP). IVR telomerase as described in part (B), supra (the “linked reconstitution method”) followed by DEAE purification, as described supra was assayed using the gel blot assay using the following reaction conditions; 1-10 μl of linked IVR telomerase in 40 μl final volume of 25 mM Tris-HCl, pH 8.3, 50 mM K-acetate, 1 mM EGTA, 1 mM MgCl<sub>2</sub>, 0.8 mM dATP, 0.8 mM TTP, 1.0 mM dGTP, and 1 uM primer ( M2, supra; or H3.03, 5′-TTAGGGTTAGGGTTAGGG; SEQ ID NO:621) at 30° C. for 180 minutes. The telomeric DNA synthesized was isolated by standard procedures, separated on a 8% polyacrylamide, 8 M urea gel, transferred to a nylon membrane, and probed using the <sup>32</sup>P-(CCCTAA)n riboprobe used in the dot-blot assay. The probe identified a six nucleotide ladder in the lane representing 10 μl of IVR telomerase that was equivalent to the ladder observed for 5 μl of native nuclear telomerase purified by mono Q and heparin chromatography. The results show that IVR telomerase possesses processive telomerase catalytic activity equivalent to native telomerase.</p>
    <p>Linked IVR telomerase was also assayed by the conventional <sup>32</sup>P-dGTP incorporation telomerase assay. IVR telomerase prepared by the linked reconstitution method followed by DEAE purification, as described above, was assayed under both processive and non-processive reaction conditions. Assay conditions were 5-10 μl of linked IVR telomerase in 40 μl final volume of 25 mM Tris-HCl, pH 8.3, 50 mM K-acetate, 1 mM EGTA, 1 mM MgCl<sub>2</sub>, 2 mM dATP, 2 mM TTP, with 10 uM <sup>32</sup>P-dGTP (72 Ci/mmol) [for assay of processive conditions] or 1 uM <sup>32</sup>P-dGTP (720 Ci/mmol) [for non-processive], and 1 uM primer (i.e., H3.03, supra) at 30° C. [for the processive reaction] or 37° C. [for the non-processive reaction] for 180 minutes. The telomeric DNA synthesized was isolated by standard procedures and separated on a 8% polyacrylamide, 8 M urea gel sequencing gel. The processive reaction showed a weak six nucleotide ladder consistent with a processive telomerase reaction, and the non-processive reaction added one repeat, a pattern equivalent to a control reaction with a native telomerase preparation. Conventional assays using IVR telomerase are useful in screens for telomerase modulators, as described herein, as well as other uses such as elucidation of the structural and functional properties of telomerase.</p>
    <p>E. In vitro Reconstituted Telomerase Recognizes Primer 3′ Termini</p>
    <p>This experiment demonstrates that IVR telomerase recognizes primer 3′ termini equivalently to native (purified) telomerase. Telomerase forms a base-paired duplex between the primer 3′ end and the template region of hTR and adds the next specified nucleotide (Morin, 1989, supra). To verify that IVR (recombinant) telomerase has the same property, the reactions of primers with - - - GGG or - - - TAG 3′ termini (AATCCGTCGAGCAGAGGG; SEQ ID NO:622 and AATCCGTCGAGCAGATAG; SEQ ID NO:623) were compared to a primer having a - - - GTT 3′ terminus (M2 supra) using IVR and native telomerase assayed by the two step conventional/TRAP assay detailed above. The product ladders of the - - - GGG and - - - TAG primers were shifted +4 and +2, respectively, when compared to the standard primer ( - - - GTT 3′ end), the same effect as was observed with native telomerase. This experiment demonstrates IVR and native telomerases recognize primer termini in a similar manner.</p>
    <p>These results (along with the results supra showing that IVR telomerase possesses both processive and non-processive catalytic activity) indicate that IVR telomerase has similar structure and properties compared to native or purified telomerase.</p>
    <heading>Example 8</heading> <heading>Production of Anti-hTRT Antibodies</heading> <p>A. Production of Anti-hTRT Antibodies Against hTRT Peptides</p>
    <p>To produce anti-hTRT antibodies, the following peptides from hTRT were synthesized with the addition of C (cysteine) as the amino terminal residue (see FIG. <b>54</b>).</p>
    <p>S-1: FFY VTE TTF QKN RLF FYR KSV WSK SEQ ID NO:232</p>
    <p>S-2: RQH LKR VQL RDV SEA EVR QHR EA SEQ ID NO:233</p>
    <p>S-3: ART FRR EKR AER LTS RVK ALF SVL NYE SEQ ID NO:234</p>
    <p>A-3: PAL LTS RLR FIP KPD GLR PIV NMD YVV SEQ ID NO:237</p>
    <p>The cysteine moiety was used to immobilize (i.e., covalently link) the peptides to BSA and KLH [keyhole limpet hemocyanin] carrier proteins. The KLH-peptides were used as antigen. The BSA-peptide conjugates served as material for ELISAs for testing the specificity of immune antisera.</p>
    <p>The KLH-peptide conjugates were injected into New Zealand White rabbits. The initial injections are made by placing the injectant proximal to the axillary and inguinal lymph nodes. Subsequent injections were made intramuscularly. For initial injections, the antigen was emulsified with Freund's complete adjuvant; for subsequent injections, Freund's incomplete adjuvant was used. Rabbits follow a three week boost cycle, in which 50 ml of blood yielding 20-25 ml of serum is taken 10 days after each boost. Antisera against each of the four peptides recognized the hTRT moiety of recombinant hTRT fusion protein (GST-HIS<sub>8</sub>-hTRT-fragment 2426 to 3274); see Example 6) on western blots.</p>
    <p>Using a partially purified telomerase fraction from human 293 cells (approximately 1000-fold purification compared to a crude nuclear extract) that was produced as described in PCT application No. 97/06012 and affinity purified anti-S-2 antibodies, a 130 kd protein doublet could be detected on a western blot. A sensitive chemiluminescence detection method was employed (SuperSignal chemiluminescence substrates, Pierce) but the signal on the blot was weak, suggesting that hTRT is present in low or very low abundance in these immortal cells. The observation of a doublet is consistent with a post-translational modification of hTRT, i.e., phosphorylation or glycosylation.</p>
    <p>For affinity purification, the S-2 peptide was immobilized to SulfoLink (Pierce, Rockford Ill.) through its N-terminal Cysteine residue according to the manufacturer's protocol. First bleed serum from a rabbit immunized with the KLH-S-2 peptide antigen was loaded over a the S-2-SulfoLink and antibodies specifically bound to the S-2 peptide were eluted.</p>
    <p>B. Production of Anti-hTRT Antibodies against hTRT Fusion Proteins</p>
    <p>GST-hTRT fusion proteins were expressed in <i>E. coli </i>as the GST-hTRT fragment #4 (nucleotides 3272-4177) and the GST-HIS8-hTRT fragment #3 (nucleotides 2426 to 3274) proteins described in Example 6. The fusion proteins were purified as insoluble protein, and the purity of the antigens was assayed by SDS polyacrylamide gels and estimated to be about 75% pure for the GST-hTRT fragment #4 recombinant protein and more than 75% pure for GST-HIS8-hTRT. fragment #3 recombinant protein. Routine methods may be used to obtain these and other fusion proteins at a purity of greater than 90%. These recombinant proteins were used to immunize both rabbits and mice, as described above.</p>
    <p>The first and second bleeds from both the mice and rabbits were tested for the presence of anti-hTRT antibodies after removal of anti-GST antibodies using a matrix containing immobilized GST. The antisera were tested for anti-hTRT antibodies by Western blotting using immobilized recombinant GST-hTRT fusion protein, and by immunoprecipitation using partially purified native telomerase enzyme. While no signal was observed in these early bleeds, titers of anti-hTRT antibodies, as expected, increased in subsequent bleeds.</p>
    <heading>Example 9</heading> <heading>Detection of an hTRT mRNA Corresponding to Δ182 RNA Variant</heading> <p>Poly A<sup>+</sup> RNA from human testis and the 293 cell line was analyzed for hTRT mRNA using RT-PCR and nested primers. The first primer set was TCP1.1 and TCP1.15; the second primer set was TCP1.14 and BTCP6. Amplification from each gave two products differing by 182 bp; the larger and smaller products from testis RNA were sequenced and found to correspond exactly to pGRN121(FIG. 16) and the 712562 clone (FIG. <b>18</b>), respectively. The variant hTRT RNA product has been observed in mRNA from SW39i, OVCAR4, 293, and Testes.</p>
    <p>Additional experiments were carried out to demonstrate that the Δ182 cDNA was not an artifact of reverse transcription. Briefly, full-length hTRT RNA (i.e., without the deletion) was produced by in vitro transcription of pGRN121 for use as a template for RT-PCR. Separate cDNA synthesis reactions were carried out using Superscript7 reverse transcriptase (Bethesda Research Laboratories, Bethesda Md.) at 42° or 50° C., and with random-primers or a specific primer. After 15 PCR cycles the longer product was detectable; however, the smaller product (i.e., corresponding to the deletion) was not detectable even after 30 or more cycles. This indicates that the RT-PCR product is not an artifact.</p>
    <heading>Example 10</heading> <heading>Sequencing of Test is hTRT mRNA</heading> <p>The sequence of the testis form of hTRT RNA was determined by direct manual sequencing of DNA fragments generated by PCR from testis cDNA (Marathon Testes cDNA, Clontech, San Diego Calif.) using a ThermoSequenase radiolabeled terminator cycle sequencing kit (Amersham Life Science). The PCR step was performed by a nested PCR, as shown in Table 8. In all cases a negative control reaction with primers but no cDNA was performed. The absence of product in the control reaction demonstrated that the products derived from the reaction with cDNA present were not due to contamination of hTRT from pGRN121 or other cell sources (e.g., 293 cells). The DNA fragments were excised from agarose gels to purify the DNA prior to sequencing.</p>
    <p>The testis mRNA sequence corresponding to bases 27 to 3553 of the pGRN121 insert sequence, and containing the entire hTRT ORF (bases 56 to 3451) was obtained. There were no differences between the testis and the pGRN121 sequences in this region.</p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" pgwide="1" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="5"> <colspec colname="1" colwidth="35pt" align="center"> </colspec> <colspec colname="2" colwidth="63pt" align="left"> </colspec> <colspec colname="3" colwidth="63pt" align="left"> </colspec> <colspec colname="4" colwidth="21pt" align="center"> </colspec> <colspec colname="5" colwidth="210pt" align="left"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="5" rowsep="1" class="description-td" colspan="5">TABLE 8</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="5" align="center" rowsep="1" class="description-td" colspan="5"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">Final</td>
                <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Fragment</td>
                <td class="description-td">Primer Set 1</td>
                <td class="description-td">Primer Set 2</td>
                <td class="description-td">Size</td>
                <td class="description-td">Primers for Seq</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="5" align="center" rowsep="1" class="description-td" colspan="5"> </td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">OA</td>
                <td class="description-td">na</td>
                <td class="description-td">K320/K322</td>
                <td class="description-td">208</td>
                <td class="description-td">K320, K322</td>
              </tr> <tr class="description-tr"> <td class="description-td">A</td>
                <td class="description-td">K320/TCP1.43</td>
                <td class="description-td">TCP1.40/TCP1.34</td>
                <td class="description-td">556</td>
                <td class="description-td">TCP1.52, TCP1.39, K322, TCP1.40, TCP1.41, TCP1.30, TCP1.34,</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">TCP1.49</td>
              </tr> <tr class="description-tr"> <td class="description-td">B</td>
                <td class="description-td">TCP1.42/TCP1.32B</td>
                <td class="description-td">TCP1.35/TCP1.21</td>
                <td class="description-td">492</td>
                <td class="description-td">TCP1.35, TCP1.28, TCP1.38, TCP1.21, TCP1.46, TCP1.33, TCP1.48</td>
              </tr> <tr class="description-tr"> <td class="description-td">C</td>
                <td class="description-td">TCP1.65/TCP1.66</td>
                <td class="description-td">TCP1.67/TCP1.68</td>
                <td class="description-td">818</td>
                <td class="description-td">TCP1.67, TCP1.32, TCP1.69, TCP1.68, TCP1.24, TCP1.44, K303</td>
              </tr> <tr class="description-tr"> <td class="description-td">D2</td>
                <td class="description-td">K304/billTCP6</td>
                <td class="description-td">Lt1/TCP1.6</td>
                <td class="description-td">546</td>
                <td class="description-td">Lt2, Lt1, TCP1.6, billTCP4, TCP1.13, TCP1.77, TCP1.1</td>
              </tr> <tr class="description-tr"> <td class="description-td">D3</td>
                <td class="description-td">TCP1.12/TCP1.7</td>
                <td class="description-td">TCP1.14/TCP1.15</td>
                <td class="description-td">604</td>
                <td class="description-td">TCP1.6, TCP1.14, TCP1.73, TCP1.78, TCP1.25, TCP1.15, TCP1.76</td>
              </tr> <tr class="description-tr"> <td class="description-td">EF</td>
                <td class="description-td">na</td>
                <td class="description-td">TCP1.74/TCP1.7</td>
                <td class="description-td">201</td>
                <td class="description-td">TCP1.74, TCP1.7, TCP1.75, TCP1.15, TCP1.3</td>
              </tr> <tr class="description-tr"> <td class="description-td">E</td>
                <td class="description-td">TCP1.3/TCP1.4</td>
                <td class="description-td">TCP1.2/TCP1.9</td>
                <td class="description-td">687</td>
                <td class="description-td">TCP1.2, TCP1.8, TCP1.9, TCP1.26</td>
              </tr> <tr class="description-tr"> <td class="description-td">F</td>
                <td class="description-td">TCP1.26/UTR2</td>
                <td class="description-td">TCP1.10 TCP1.4</td>
                <td class="description-td">377</td>
                <td class="description-td">TCP1.4, TCP1.10, TCP1.11</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="5" align="center" rowsep="1" class="description-td" colspan="5"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <heading>Example 11</heading> <heading>Detection of hTRT mRNA by RNase Protection</heading> <p>RNase protection assays can be used to detect, monitor, or diagnose the presence of an hTRT mRNA or variant mRNA. One illustrative RNAse protection probe is an in vitro synthesized RNA comprised of sequences complementary to hTRT mRNA sequences and additional, non-complementary sequences. The latter sequences are included to distinguish the full-length probe from the fragment of the probe that results from a positive result in the assay: in a positive assay, the complementary sequences of the probe are protected from RNase digestion, because they are hybridized to hTRT mRNA. The non-complementary sequences are digested away from the probe in the presence of RNase and target complementary nucleic acid.</p>
    <p>Two RNAse protection probes are described for illustrative purposes; either can be used in the assay. The probes differ in their sequences complementary to hTRT, but contain identical non-complementary sequences, in this embodiment, derived from the SV40 late mRNA leader sequence. From 5′-3′, one probe is comprised of 33 nucleotides of non-complementary sequence and 194 nucleotides of sequence complementary to hTRT nucleotides 2513-2707 for a full length probe size of 227 nucleotides. From 5′-3′, the second probe is comprised of 33 nucleotides of non-complementary sequence and 198 nucleotides of sequence complementary to hTRT nucleotides 2837-3035 for a full length probe size of 231 nucleotides. To conduct the assay, either probe can be hybridized to RNA, i.e., polyA+ RNA, from a test sample, and T1 ribonuclease and RNase A are then added. After digestion, probe RNA is purified and analyzed by gel electrophoresis. Detection of a 194 nucleotide fragment of the 227 nucleotide probe or a 198 nucleotide fragment of the 231 nucleotide probe is indicative of hTRT mRNA in the sample.</p>
    <p>The illustrative RNAse protection probes described in this example can be generated by in vitro transcription using T7 RNA polymerase. Radioactive or otherwise labeled ribonucleotides can be included for synthesis of labeled probes. The templates for the in vitro transcription reaction to produce the RNA probes are PCR products. These illustrative probes can be synthesized using T7 polymerase following PCR amplification of pGRN121 DNA using primers that span the corresponding complementary region of the hTRT gene or mRNA. In addition, the downstream primer contains T7 RNA polymerase promoter sequences and the non-complementary sequences.</p>
    <p>For generation of the first RNAse protection probe, the PCR product from the following primer pair (T701 and reverse01) is used:</p>
    <p>T701 5′-GGGAGATCT TAATACGACTCACTATAG ATTCA GGCCATGGTGCTGCGCCGGC TGTCA GGCTCCC ACGACGTAGT CCATGTTCAC-3′ (SEQ ID NO:624); and reverse01 5′-GGGTCTAGAT CCGGAAGAGTGT CTGGAGCAAG-3′ (SEQ ID NO:625).</p>
    <p>For generation of the second RNase protection probe, the PCR product from the following primer pair (T702 and reverse02) is used:</p>
    <p>T702 5′-GGGAGATCT TAATACGACTCACTATAG ATTCA GGCCATGGTGCTGCGCCGGC TGTCA GGGCG GCCTTCTGGA CCACGGCATA CC-3′ (SEQ ID NO:626); and reverse02 5′-G GTCTAGA CGATATCC ACAGGGCCTG GCGC-3′ (SEQ ID NO:672).</p>
    <heading>Example 12</heading> <heading>Construction of a Phylogenetic Tree Comparing hTRT and Other Reverse Transcriptases</heading> <p>A phylogenetic tree (FIG. 6) was constructed by comparison of the seven RT domains defined by Xiong and Eickbush (1990, <i>EMBO J. </i>9:3353). After sequence alignment of motifs 1, 2, and A-E from 4 TRTs, 67 RTs, and 3 RNA polymerases, the tree was constructed using the NJ (Neighbor Joining) method (Saitou and Nei, 1987, <i>Mol. Biol. Evol. </i>4:406). Elements from the same class that are located on the same branch of the tree are simplified as a box. The length of each box corresponds to the most divergent element within that box.</p>
    <p>The TRTs appear to be more closely related to RTs associated with msDNA, group II introns, and non-LTR (Long Terminal Repeat) retrotransposons than to the LTR-retrotransposon and viral RTs. The relationship of the telomerase RTs to the non-LTR branch of retroelements is intriguing, given that these latter elements have replaced telomerase for telomere maintenance in Drosophila. However, the most striking finding is that the TRTs form a discrete subgroup, almost as closely related to the RNA-dependent RNA polymerases of plus-stranded RNA viruses such as poliovirus as to any of the previously known RTs. Considering that the four telomerase genes come from evolutionarily distant organisms—protozoan, fungi, and mammal—this separate grouping cannot be explained by lack of phylogenetic diversity in the data set. Instead, this deep bifurcation suggests that the telomerase RTs are an ancient group, perhaps originating with the first eukaryote.</p>
    <p>GenBank protein identification or accession numbers used in the phylogenetic analysis were: msDNAs (94535, 134069, 134074, 134075, 134078), group II introns (483039, 101880, 1332208, 1334433, 1334435, 133345, 1353081), mitochondrial plasmid/RTL (903835, 134084), non-LTR retrotransposons (140023, 84806, 103221, 103353, 134083, 435415, 103015, 1335673, 85020, 141475, 106903, 130402, U0551, 903695, 940390, 2055276, L08889), LTR retrotransposons (74599, 85105, 130582, 99712, 83589, 84126, 479443, 224319, 130398, 130583, 1335652, 173088, 226407, 101042, 1078824), hepadnaviruses (I 18876, 1706510, 118894), caulimoviruses (331554, 130600, 130593, 93553), retroviruses (130601, 325465, 74601, 130587, 130671, 130607, 130629, 130589, 130631, 1346746, 130651, 130635, 1780973, 130646). Alignment was analyzed using ClustalW 1.5 [J. D. Thompson, D. G. Higgins, T. J. Gibson, <i>Nucleic Acids Res. </i>22, 4673 (1994)] and PHYLIP 3.5 [J. Felsenstein, Cladisfics 5, 164 (1989)].</p>
    <heading>Example 13</heading> <heading>Transfection of Cultured Human Fibroblasts (BJ) with Control Plasmid and Plasmid Encoding hTRT</heading> <p>This example demonstrates that expression of recombinant hTRT protein in a mammalian cell results in the generation of an active telomerase.</p>
    <p>Subconfluent BJ fibroblasts were trypsinized and resuspended in fresh medium (DMEM/199 containing 10% Fetal Calf Serum) at a concentration of 4×10<sup>6 </sup>cells/ml. The cells were transfected using electroporation with the BioRad Gene Pulser™ electroporator. Optionally, one may also transfect cells using Superfect™ reagent (Qiagen) in accordance with the manufacturer's instructions. For electroporation, 500 μl of the cell suspension were placed in an electroporation cuvette (BioRad, 0.4 cm electrode gap). Plasmid DNA (2 μg) was added to the cuvettes and the suspension was gently mixed and incubated on ice for 5 minutes. The control plasmid (pBBS212) contained no insert behind the MPSV promoter and the experimental plasmid (pGRN133) expressed hTRT from the MPSV promoter. The cells were electroporated at 300 Volts and 960 μFD. After the pulse was delivered, the cuvettes were placed on ice for approximately 5 minutes prior to plating on 100 mm tissue culture dishes in medium. After 6 hours, the medium was replaced with fresh medium. 72 hours after the transfection, the cells were trypsinized, washed once with PBS, pelleted and stored frozen at −80° C. Cell extracts were prepared at a concentration of 25,000 cells/μl by a modified detergent lysis method (see Bodnar et al., 1996, <i>Exp. Cell Res. </i>228:58; Kim et al., 1994, <i>Science </i>266:2011, and as described in patents and publications relating to the TRAP assay, supra) and telomerase activity in the cell extracts was determined using a modified PCR-based TRAP assay (Kim et al., 1994, Bodnar et al., 1996). Briefly, 5×10<sup>4 </sup>cell equivalents were used in the telomerase primer extension portion of the reaction. While the extract is typically taken directly from the telomerase extension reaction to the PCR amplification, one may also extract once with phenol/chloroform and once with chloroform prior to the PCR amplification. One-fifth of the material was used in the PCR amplification portion of the TRAP reaction (approximately 10,000 cell equivalents). Onehalf of the TRAP reaction was loaded onto the gel for analysis, such that each lane in FIG. 25 represents reaction products from 5,000 cell equivalents. Extracts from cells transfected with pGRN133 were positive for telomerase activity while extracts from untransfected (not shown) or control plasmid transfected cells showed no telomerase activity. Similar experiments using RPE cells gave the same result.</p>
    <p>Reconstitution in BJ cells was also carried out using other hTRT constructs (i.e., pGRN145, pGRN155 and pGRN138). Reconstitution using these constructs appeared to result in more telomerase activity than in the pGRN133 transfected cells.</p>
    <p>The highest level of telomerase activity was achieved using pGRN155. As discussed supra, pGRN155 is a vector containing the adenovirus major late promoter as a controlling element for the expression of hTRT and was shown to reconstitute telomerase activity when transfected into BJ cells.</p>
    <p>Notably, when reconstitution using the hTRT-GFP fusion protein pGRN138 (which localizes to the nucleus, see Example 15, infra) was performed either in vitro (see Example 7) or in vivo (transfection into BJ cells) telomerase activity resulted. By transfection into BJ cells, for example, as described supra, telomerase activity was comparable to that resulting from reconstitution in vitro using pGRN133 or pGRN145.</p>
    <p>Similar results were obtained upon transfection of normal human retinal pigmented epithelial (RPE) with the hTRT expression vectors of the invention. The senescence of RPE cells is believed to contribute to or cause the disease of age-related macular degeneration. RPE cells treated in accordance with the methods of the invention using the hTRT expression vectors of the invention should exhibit delayed senescence, as compared to untreated cells, and so be useful in transplantation therapies to treat or prevent age-related macular degeneration.</p>
    <heading>Example 14</heading> <heading>Promoter Reporter Construct</heading> <p>This example describes the construction of plasmids in which reporter genes are operably linked to hTRT upstream sequences containing promoter elements. The vectors have numerous uses, including identification of cis and trans transcriptional regulatory factors in vivo and for screening of agents capable of modulating (e.g., activating or inhibiting) hTRT expression (e.g., drug screening). Although a number of reporters may be used (e.g., firefly luciferase, β-glucuronidase, β-galactosidase, chloramphenicol acetyl transferase, and GFP and the like), the human secreted alkaline phosphatase (SEAP; CloneTech) was used for initial experiments. The SEAP reporter gene encodes a truncated form of the placental enzyme which lacks the membrane anchoring domain, thereby allowing the protein to be secreted efficiently from transfected cells. Levels of SEAP activity detected in the culture medium have been shown to be directly proportional to changes in intracellular concentrations of SEAP mRNA and protein (Berger et al., 1988, <i>Gene </i>66:1; Cullen et al., 1992, <i>Meth. Enzymol. </i>216:362).</p>
    <p>Four constructs (pGRN148, pGRN150, “pSEAP2 basic” (no promoter sequences=negative control) and “pSEAP2 control” (contains the SV40 early promoter and enhancer) were transfected in triplicate into mortal and immortal cells.</p>
    <p>Plasmid pGRN148 was constructed as illustrated in FIG. <b>9</b>. Briefly, a Bgl2-Eco47III fragment from pGRN144 was digested and cloned into the BglII-NruI site of pSeap2Basic (Clontech, San Diego, Calif.). A second reporter-promoter, plasmid pGRN150, includes sequences from the hTRT intron described in Example 3, to employ regulatory sequences that may be present in the intron. The initiating Met is mutated to Leu, so that the second ATG following the promoter region will be the initiating ATG of the SEAP ORF.</p>
    <p>The pGRN148 and pGRN150 constructs (which include the hTRT promoter) were transfected into mortal (BJ cells) and immortal (293) cells. All transfections were done in parallel with two control plasmids: one negative control plasmid (pSEAP basic) and one positive control plasmid (pSEAP control which contains the SV40 early promoter and the SV40 enhancer).</p>
    <p>In immortal cells, pGRN148 and pGRN150 constructs appear to drive SEAP expression as efficiently as the pSEAP2 positive control (containing the SV40 early promoter and enhancer). In contrast, in mortal cells only the pSEAP2 control gave detectable activity. These results indicate that, as expected, hTRT promoter sequences are active in tumor cells but not in mortal cells.</p>
    <p>Similar results were obtained using another normal cell line (RPE, or retinal pigmental epithelial cells). In RPE cells transfected with pGRN150 (containing 2.2 KB of upstream genomic sequence), the hTRT promoter region was inactive while the pSEAP2 control plasmid was active.</p>
    <p>As noted supra, plasmids in which reporter genes are operably linked to hTRT upstream sequences containing promoter elements are extremely useful for identification and screening of telomerase activity modulatory agents, using both transient and stable transfection techniques. In one approach, for example, stable transformants of pGRN148 are made in telomerase negative and telomerase positive cells by cotransfection with a eukaryotic selectable marker (such as neo) according to Ausubel et al., 1997, supra. The resulting cell lines are used for screening of putative telomerase modulatory agents, for example, by comparing hTRT-promoter-driven expression in the presence and absence of a test compound.</p>
    <p>The promoter-reporter (and other) vectors of the invention are also used to identify trans- and cis-acting transcriptional and translational regulatory elements. Examples of cis-acting transcriptional regulatory elements include promoters and enhancers of the telomerase gene. The identification and isolation of cis- and trans-acting regulatory agents provide for further methods and reagents for identifying agents that modulate transcription and translation of telomerase.</p>
    <p>To identify sequences or elements that play a role in hTRT expression, expression was tested using promoter-reporter constructs with varying amounts of the upstream region (5′ to the transcription initiation site) of the hTRT gene. Experiments were conducted using pGRN 150 [which contains approximately 2405 bp of genomic sequence upstream of the most 5′ nucleotide present in the hTRT cDNA], pGRN 176 [which contains approximately 186 bp of genomic sequence upstream of the most 5′ nucleotide present in the hTRT cDNA] and pGRN 175 [which contains approximately 77 bp of genomic sequence upstream of the most 5′ nucleotide present in the hTRT cDNA]. The following sequence is present in pGRN 176 but not pGRN 175: 5′-GTGGCGGAGGGACTGGGGACCCGGGCACCGGTCCTGCCCCTTCACCTTCCAGCTCCGCCTCGTCCGCGCGGAACCCCGCCCCGTCCCGAACCCTTCCCGGGTCCCCGGCCCAGCCCCTTCCGGG-3′ (SEQ ID NO:726).</p>
    <p>When transfected into mortal cells (RPE and BJ), the pGRN 175 promoter was active, while the pGRN 176 and pGRN 150 promoters were not active. These results demonstrate that the approximately 120 basepair region present in pGRN 176 but not pGRN 175 includes sequences that play a role in the mortal-cell specific repression of hTRT gene expression is achieved. It will be recognized that less than the entire approximately 120 basepair sequence may be required for this effect, and that other sequences not in the approximately 120 base pair region may also play a role (independently or in combination with the approximately 120 base pair region) in regulation of hTRT expression. Thus, the approximately 120 base pair region includes all or part of one or more cis-acting elements.</p>
    <p>Without intending to be bound by any particular mechanism, the approximately 120 base pair sequence includes a binding site for a repressor (e.g., a trans acting repressor) which upon binding prevents initiation of transcription of the hTRT gene. Such a repressor may be the product of an anti-oncogene (e.g., a novel anti-oncogene), which can be identified and cloned in accordance with the teachings herein and the use of the novel reagents disclosed herein. In normal cells, repressor binding or interaction with hTRT regulatory sequences (e.g., including or within the approximately 120 base pair sequence) results in the absence of hTRT protein and therefore of telomerase activity. Activation of telomerase in cancer cells can result from the loss of hTRT repressor activity.</p>
    <p>A number of applications of the “approximately 120 base pair region” described above will be immediately apparent upon review of this disclosure, including for treatment or diagnosis of telomerase related diseases and identification of agents with telomerase modulatory activity. For example, using standard techniques, the sequence may be used to identify agents or proteins (e.g. naturally occurring repressor proteins) that specifically bind to the approximately 120 base pair sequence or a subsequence thereof. In addition, synthetic or naturally occurring agents that increase or stabilize repression (e.g., by binding or otherwise interacting with the sequence, by stabilizing binding by a naturally occurring repressor, or by other means) will be useful for reducing telomerase activity in a cell (e.g., for treatment of malignancy). Similarly, agents that reduce repression (e.g., by inhibiting repressor binding, or by other means) will be useful for increasing telomerase expression (e.g., by controlled activation), for example to increase the proliferative capacity of normal cells).</p>
    <heading>Example 15</heading> <heading>Subcellular Localization of hTRT</heading> <p>A fusion protein having hTRT and enhanced green fluorescent protein (EGFP; Cormack et al., 1996, <i>Gene </i>173:33) regions was constructed as described below. The EGFP moiety provides a detectable tag or signal so that the presence or location of the fusion protein can be easily determined. Because EGFP-fusion proteins localize in the correct cellular compartments, this construct may be used to determine the subcellular location of hTRT protein.</p>
    <p>A. Construction of pGRN138</p>
    <p>A vector for expression of an hTRT-EGFP fusion protein in mammalian cells was constructed by placing the EcoRI insert from pGRN 124 (see Example 6) into the EcoRI site of pEGFP-C2 (Clontech, San Diego, Calif.). The amino acid sequence of the fusion protein is provided below. EGFP residues are in bold, residues encoded by the 5′ untranslated region of hTRT mRNA are underlined, and the hTRT protein sequence is in normal font.</p>
    <p>MVSKGEELFTGWPILVELDGDVNGHKFSVSGEGEGDATYGKLTLKFICTTGKLPVPWPTLVTTLTYGVQCFSRYPDHMKQHDFFKSAMPEGYVQERTIFFKDDGNYKTRAEVKFEGDTLVNRIELKGIDFKEDGNILGHKLEYNYNSHNVYIMADKQKNGIKVNFKIRHNIEDGSVQLADHYQQNTPIGDGPVLLPDNHYLSTQSALSKDPNEKRDHMVLLEFVTAAGITLGMDELYKSGRTQISSSSFEFAAAST<u>QRCVLLRTWEALAPATPA</u>MPRAPRCRAVRSLLRSHYREVLPLATFVRRLGPQGWRLVQRGDPAAFRALVAQCLVCVPWDARPPPAAPSFRQVSCLKELVARVLQRLCERGAKNVLAFGFALLDGARGGPPEAFTTSVRSYLPNTVTDALRGSGAWGLLLRRVGDDVLVHLLARCALFVLVAPSCAYQVCGPPLYQLGAATQARPPPHASGPRRRLGCERAWNHSVREAGVPLGLPAPGARRRGGSASRSLPLPKRPRRGAAPEPERTPVGQGSWAHPGRTRGPSDRGFCVVSPARPAEEATSLEGALSGTRHSHPSVGRQHHAGPPSTSRPPRPWDTPCPPVYAETKHFLYSSGDKEQLRPSFLLSSLRPSLTGARRLVETIFLGSRPWMPGTPRRLPRLPQRYWQMRPLFLELLGNHAQCPYGVLLKTHCPLRAAVTPAAGVCAREKPQGSVAAPEEEDTDPRRLVQLLRQHSSPWQVYGFVRACLRRLVPPGLWGSRHNERRFLRNTKKFISLGKHAKLSLQELTWKMSVRDCAWLRRSPGVGCVPAAEHRLREEILAKFLHWLMSVYWVELLRSFFYVTETTFQKNRLFFYRPSVWSKLQSIGIRQHLKRVQLRELSEAEVRQHREARPALLTSRLRFIPKPDGLRPIVNMDYVVGARTFRREKRAERLTSRVKALFSVLNYERARRPGLLGASVLGLDDIHRAWRTFVLRVRAQDPPPELYFVKVDVTGAYDTIPQDRLTEVIASIIKPQNTYCVRRYAVVQKAAHGHVRKAFKSHVSTLTDLQPYMRQFVAHLQETSPLRDAVVIEQSSSLNEASSGLFDVFLRFMCHHAVRIRGKSYVQCQGIPQGSILSTLLCSLCYGDMENKLFAGIRRDGLLLRLVDDFLLVTPHLTHAKTFLRTLVRGVPEYGCVVNLRKTVVNFPVEDEALGGTAFVQMPAHGLFPWCGLLLDTRTLEVQSDYSSYARTSIRASVTFNRGFKAGRNMRRKLFGVLRLKCHSLFLDLQVNSLQTVCTNIYKILLLQAYRFHACVLQLPFHQQVWKNPTFFLRVISDTASLCYSILKAKNAGMSLGAKGAAGPLPSEAVQWLCHQAFLLKLTRHRVTYVPLLGSLRTAQTQLSRKLPGTTLTALEAAANPALPSDFKTILD (SEQ ID NO:628)</p>
    <p>Other EGFP fusion constructs can be made using partial (e.g., truncated) hTRT coding sequence and used, as described infra, to identify activities of particular regions of the hTRT polypeptide.</p>
    <p>B. Nuclear Localization and Uses of pGRN138</p>
    <p>Transfection of NIH 293 and BJ cells with pGRN138 confirmed the nuclear localization of recombinantly expressed hTRT. Cells were transfected with pGRN138 (EGFP-hTRT) and with a control construct (expressing EGFP only). Nuclear localization of the EGFP-hTRT is apparent in both cell types by fluorescence microscopy. As noted supra, the pGRN138 hTRT-GFP fusion protein supports reconstitution of telomerase activity in both an in vitro transcription translation system and in vivo when transfected into BJ cells.</p>
    <p>The hTRT-EGFP fusion proteins (or similar detectable fusion proteins) can be used in a variety of applications. For example, the fusion construct described in this example, or a construct of EGFP and a truncated form of hTRT, can be used to assess the ability of hTRT and variants to enter a cell nucleus and/or localize at the chromosome ends. In addition, cells stably or transiently transfected with pGRN138 are used for screening compounds to identify telomerase modulatory drugs or compounds. Agents that interfere with nuclear localization or telomere localization can be identified as telomerase inhibitors. Tumor cell lines stably expressing EGFP-hTRT can be useful for this purpose. Potential modulators of telomerase will be administered to these transfected cells and the localization of the EGFP-hTRT will be assessed. In addition, FACS or other fluorescence-based methods can be used to select cells expressing hTRT to provide homogeneous populations for drug screening, particularly when transient transfection of cells is employed.</p>
    <p>In other applications, regions of the hTRT can be mutagenized to identify regions (e.g., residues 193-196 (PRRR; SEQ ID NO:541) and 235-240 (PKRPRR; SEQ ID NO:542)) required for nuclear localization, which are targets for anti-telomerase drugs (telomerase activity modulators). Other applications include:</p>
    <p>use of the fusion protein as a fluorescent marker of efficient cell transfection for both transient transfection experiments and when establishing stable cell lines expressing EGFP-hTRT;</p>
    <p>expression of an hTRT-EGFP fusion with mutated nuclear localization signals (deficient for nuclear localization) in immortal cells so that the hTRT mutant-EGFP scavenges all the hTR of the immortal cells, retaining it in the cytoplasm and preventing telomere maintenance; and</p>
    <p>use as a tagged protein for immunoprecipitation.</p>
    <heading>Example 16</heading> <heading>Effect of Mutation on Telomerase Catalytic Activity</heading> <p>This example describes hTRT variant proteins having altered amino acids and altered telomerase catalytic activity. Amino acid substitutions followed by functional analysis is a standard means of assessing the importance and function of a polypeptide sequence. This example demonstrates that changes in the reverse transcriptase (RT) and telomerase (T) motifs affect telomerase catalytic activity.</p>
    <p>Conventional nomenclature is used to describe mutants: the target residue in the native molecule (hTRT) is identified by one-letter code and position, and the corresponding residue in the mutant protein is indicated by one-letter code. Thus, for example, “K626A” specifies a mutant in which the lysine at position 626 (i.e., in motif 1) of hTRT is changed to an alanine.</p>
    <p>A. Mutation of hTRT FFYxTE (SEQ ID NO:360) Motif</p>
    <p>In initial experiments, a vector encoding an hTRT mutant protein, “F560A,” was produced in which amino acid 560 of hTRT was changed from phenylalanine (F) to alanine (A) by site directed mutagenesis of pGRN121 using standard techniques. This mutation disrupts the TRT FFYxTE (SEQ ID NO:360) motif. The resulting F560A mutant polynucleotide was shown to direct synthesis of a full length hTRT protein as assessed using a cell-free reticulocyte lysate transcription/translation system in the presence of <sup>35</sup>S-methionine.</p>
    <p>When the mutant polypeptide was co-translated with hTR, as described in Example 7, no telomerase activity was detected as observed by TRAP using 20 cycles of PCR, while a control hTRT/hTR cotranslation did reconstitute activity. With 30 cycles of PCR in the TRAP assay, telomerase activity was observable with the mutant hTRT, but was considerably lower than the control (wild-type) hTRT.</p>
    <p>B. Additional Site-Directed Mutagenesis of hTRT Amino Acid Residues</p>
    <p>Conserved amino acids in six RT motifs were changed to alanine using standard site directed mutagenesis techniques (see, e.g., Ausubel, supra) to assess their contribution to catalytic activity. The mutants were assayed using IVR telomerase using the two step conventional/TRAP assay detailed in example 7.</p>
    <p>The K626A (motif 1), R631A (motif 2), D712A (motif A), Y717A (motif A), D868A (motif C) mutants had greatly reduced or undetectable telomerase activity (&lt;1% of wild-type), while the Q833A (motif B) and G932A (motif E) mutants exhibited low/intermediate levels of activity (&lt;10% of wild-type). Two mutations outside the RT motifs, R688A and D897A, had activity equivalent to wild type hTRT. These results were consistent with analogous mutations in reverse transcriptases (Joyce et al., 1994, <i>Ann. Rev. Biochem. </i>63:777) and are similar to results obtained with Est2p (see Lingner, 1997, <i>Science </i>276:561). The experiments identify residues in the RT motifs critical and not critical for enzymatic activity and demonstrate that hTRT is the catalytic protein of human telomerase. The mutations provide variant hTRT polypeptides that have utility, e.g., as dominant/negative regulators of telomerase activity.</p>
    <p>Amino acid alignment of the known TRTs identified a telomerase-specific motif, motif T (see supra). To determine the catalytic role of this motif in hTRT, a six amino acid deletion in this motif (Δ560-565; FFYxTE; SEQ ID NO:360), was constructed using standard site directed mutagenesis techniques (Ausubel, supra). The deletion was assayed using IVR telomerase using the two step conventional/TRAP assay detailed in Example 7. The Δ560-565 mutant had no observable telomerase activity after 25 cycles of PCR whereas wild type hTRT IVR telomerase produced a strong signal. Each amino acid in each residue in motif T was examined independently in a similar manner; mutants F560A, Y562A, T564A, and E565A retained intermediate levels of telomerase activity, while a control mutant, F487A, had minimal affect on activity. Notably, mutant F561A had greatly reduced or undetectable telomerase activity, while activity was fully restored in its “revertant”, F561A561F. F561A561Fchanges the mutated position back to its original phenylalanine. This is a control that demonstrates that no other amino acid changes occurred to the plasmid that could account for the decreased activity observed. Thus, the T motif is the first non-RT motif shown to be absolutely required for telomerase activity.</p>
    <p>Motif T can be used for identification of TRTs from other organisms and hTRT proteins comprising variants of this motif can be used as a dominant/negative regulator of telomerase activity. Unlike most other RTs, telomerase stably associates with and processively copies a small portion of a single RNA (ie. hTR), thus motif T can be involved in mediating hTR binding, the processivity of the reaction, or other functions unique to the telomerase RT.</p>
    <p>In other experiments, it was observed that the deletion variant encoded by pro90hTRT described herein, did not reconstitute telomerase activity when co-synthesized with hTR, as measured using a modified TRAP assay (Autexier et al., 1996, <i>EMBO Journal </i>15:5928, which is incorporated herein by reference).</p>
    <heading>Example 17</heading> <heading>Screening for Telomerase Activity Modulators Using Recombinantly Expressed Telomerase Components</heading> <p>This example describes the use of in vitro reconstituted telomerase for screening and identifying telomerase activity modulators. The assay described is easily adapted to high-through-put methods (e.g., using multiple well plates and/or robotic systems). Numerous variations on the steps of the assay will be apparent to one of skill in the art after review of this disclosure.</p>
    <p>Recombinant clones for telomerase components (e.g., hTRT and hTR) are transcribed and translated (hTRT only) in an in vitro reaction as follows and as described in Example 7 supra, using the TNT7 T7 Coupled Reticulocyte lysate system (Promega), which is described in U.S. Pat. No. 5,324,637, following the manufacturer's instructions:</p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="112pt" align="left"> </colspec> <colspec colname="2" colwidth="91pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="2" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Reagent</td>
                <td class="description-td">Amount per reaction (μL)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="2" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">TNT Rabbit Reticulocyte lysate</td>
                <td class="description-td">25 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">TNT reaction buffer</td>
                <td class="description-td">2</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">TNT T7 RNA Pol.</td>
                <td class="description-td">1</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">AA mixture (complete)</td>
                <td class="description-td">1</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Prime RNase inhibitor</td>
                <td class="description-td">1</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Nuclease-free water</td>
                <td class="description-td">16 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Xba1 cut pGRN121 [hTRT] (0.5 μg)</td>
                <td class="description-td">2</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Fsp1 cut pGRN164 [hTR] (0.5 μg)</td>
                <td class="description-td">2</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="2" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p>The reaction is incubated at 30° C. for 2 hours. The product is then purified on an ultrafree-MC DEAE filter (Millipore).</p>
    <p>The recombinant telomerase product (IVRP) is assayed in the presence and absence of multiple concentrations of test compounds which are solubilized in DMSO (e.g. 10 μM-100 μM). Test compounds are preincubated in a total volume of 25 μL for 30 minutes at room temperature in the presence of 2.5 μL IVRP, 2.5% DMSO, and 1×TRAP Buffer (20 mM Tris-HCl, pH 8.3, 1.5 mM MgCl<sub>2</sub>, 63 mM KCl, 0.05% Tween20, 1.0 mM EGTA, 0.1 mg/ml Bovine serum albumin). Following the preincubation, 25 μL of the TRAP assay reaction mixture is added to each sample. The TRAP assay reaction mixture is composed of 1×TRAP buffer, 50 μL dNTP, 2.0 μg/ml primer ACX, 4 μg/ml primer U2, 0.8 attomol/ml TSU2, 2 units/50 μl Taq polymerase (Perkin Elmer), and 2 μg/ml [<sup>32</sup>P]5′end-labeled primer TS (3000 Ci/mmol). The reaction tubes are then placed in the PCR thermocycler (MJ Research) and PCR is performed as follows: 60 min at 30° C., 20 cycles of {30 sec at 94° C., 30 sec. at 60° C., 30 sec. at 72° C.}, 1 min at 72° C., cool down to 10° C. The TRAP assay is described, as noted supra, in U.S. Pat. No. 5,629,154. The primers and substrate used have the sequences: TS Primer (5′-AATCCGTCGAGCAGAGTT-3′; SEQ ID NO:629); ACX Primer (5′-GCGCGG[CTTACC]<sub>3</sub>CTAACC-3′; SEQ ID NO:630); U2 primer (5′-ATCGCTTCTCGGCCTTTT-3′; SEQ ID NO:631); TSU2 (5′-AATCCGTCGAGCAGAGTTAAAAGGCCGAGAAGCGAT-3′; SEQ ID NO:632).</p>
    <p>After completion of the PCR step, 4 μl of 10×loading buffer containing bromophenol blue is added to each reaction tube and products (20 μl) are run on a 12.5% non-denaturing PAGE in 0.5×TBE at 400 V. The completed gel is subsequently dried and the TRAP products are visualized by Phosphorimager or by autoradiography. The telomerase activity in the presence of the test compound is measured by comparing the incorporation of label in reaction product to a parallel reaction lacking the agent.</p>
    <p>The following clones described in the Examples have been deposited with the American Type Culture Collection, Rockville, Md. 20852, USA:</p>
    <p>Lambda phage λ25-1.1 ATCC accession number 209024</p>
    <p>pGRN121 ATCC accession number 209016</p>
    <p>Lambda phage λGΦ5 ATCC accession number 98505</p>
    <p>The present invention provides novel methods and materials relating to hTRT and diagnosis and treatment of telomerase-related diseases. While specific examples have been provided, the above description is illustrative and not restrictive. Many variations of the invention will become apparent to those of skill in the art upon review of this specification. The scope of the invention should, therefore, be determined not with reference to the above description, but instead should be determined with reference to the appended claims along with their full scope of equivalents.</p>
    <p>All publications and patent documents cited in this application are incorporated by reference in their entirety for all purposes to the same extent as if each individual publication or patent document were so individually denoted.</p>
    <p>
      <sequence-cwu id="SEQLST-0"> <table frame="none" colsep="0" rowsep="0" tabstyle="monospace" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="441pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">SEQUENCE LISTING</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="441pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">The patent contains a lengthy “Sequence Listing” section. A copy of the “Sequence Listing” is available in electronic form from the USPTO</td>
              </tr> <tr class="description-tr"> <td class="description-td">web site (http://seqdata.uspto.gov/sequence.html?DocID=06617110B1). An electronic copy of the “Sequence Listing” will also be available from the</td>
              </tr> <tr class="description-tr"> <td class="description-td">USPTO upon request and payment of the fee set forth in 37 CFR 1.19(b)(3).</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> </tbody> </tgroup> </table> </sequence-cwu> </p>
    </div></div></div><div class="patent-section patent-tabular-section"><a id="backward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Cited Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US3817837">US3817837</a></td><td class="patent-data-table-td patent-date-value">Nov 6, 1972</td><td class="patent-data-table-td patent-date-value">Jun 18, 1974</td><td class="patent-data-table-td ">Syva Corp</td><td class="patent-data-table-td ">Enzyme amplification assay</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US3850752">US3850752</a></td><td class="patent-data-table-td patent-date-value">Oct 29, 1971</td><td class="patent-data-table-td patent-date-value">Nov 26, 1974</td><td class="patent-data-table-td ">Akzona Inc</td><td class="patent-data-table-td ">Process for the demonstration and determination of low molecular compounds and of proteins capable of binding these compounds specifically</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US3939350">US3939350</a></td><td class="patent-data-table-td patent-date-value">Apr 29, 1974</td><td class="patent-data-table-td patent-date-value">Feb 17, 1976</td><td class="patent-data-table-td ">Board Of Trustees Of The Leland Stanford Junior University</td><td class="patent-data-table-td ">Fluorescent immunoassay employing total reflection for activation</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US3996345">US3996345</a></td><td class="patent-data-table-td patent-date-value">Jun 30, 1975</td><td class="patent-data-table-td patent-date-value">Dec 7, 1976</td><td class="patent-data-table-td ">Syva Company</td><td class="patent-data-table-td ">Fluorescence quenching with immunological pairs in immunoassays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4275149">US4275149</a></td><td class="patent-data-table-td patent-date-value">Nov 24, 1978</td><td class="patent-data-table-td patent-date-value">Jun 23, 1981</td><td class="patent-data-table-td ">Syva Company</td><td class="patent-data-table-td ">Immunoassay</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4277437">US4277437</a></td><td class="patent-data-table-td patent-date-value">Dec 10, 1979</td><td class="patent-data-table-td patent-date-value">Jul 7, 1981</td><td class="patent-data-table-td ">Syva Company</td><td class="patent-data-table-td ">Chemiluminescent source, quencher molecule</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4366241">US4366241</a></td><td class="patent-data-table-td patent-date-value">Aug 7, 1980</td><td class="patent-data-table-td patent-date-value">Dec 28, 1982</td><td class="patent-data-table-td ">Syva Company</td><td class="patent-data-table-td ">Concentrating zone method in heterogeneous immunoassays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4683195">US4683195</a></td><td class="patent-data-table-td patent-date-value">Feb 7, 1986</td><td class="patent-data-table-td patent-date-value">Jul 28, 1987</td><td class="patent-data-table-td ">Cetus Corporation</td><td class="patent-data-table-td ">One of the first Polymerase Chain Reaction (PCR) patents</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4683202">US4683202</a></td><td class="patent-data-table-td patent-date-value">Oct 25, 1985</td><td class="patent-data-table-td patent-date-value">Nov 27, 1990</td><td class="patent-data-table-td ">Cetus Corp</td><td class="patent-data-table-td citation-no-title">Title not available</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4816567">US4816567</a></td><td class="patent-data-table-td patent-date-value">Apr 8, 1983</td><td class="patent-data-table-td patent-date-value">Mar 28, 1989</td><td class="patent-data-table-td ">Genentech, Inc.</td><td class="patent-data-table-td ">Immunology, vectors, dna</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4965188">US4965188</a></td><td class="patent-data-table-td patent-date-value">Jun 17, 1987</td><td class="patent-data-table-td patent-date-value">Oct 23, 1990</td><td class="patent-data-table-td ">Cetus Corporation</td><td class="patent-data-table-td ">Reacting nucleic acid with oligonucleotide primer in presence of catalytic enzyme dna polymerase; polymerase chain reaction patent</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5489508">US5489508</a></td><td class="patent-data-table-td patent-date-value">Mar 24, 1993</td><td class="patent-data-table-td patent-date-value">Feb 6, 1996</td><td class="patent-data-table-td ">University Of Texas System Board Of Regents</td><td class="patent-data-table-td ">Therapy and diagnosis of conditions related to telomere length and/or telomerase activity</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5583016">US5583016</a></td><td class="patent-data-table-td patent-date-value">Oct 27, 1994</td><td class="patent-data-table-td patent-date-value">Dec 10, 1996</td><td class="patent-data-table-td ">Geron Corporation</td><td class="patent-data-table-td ">Isolated, purified recombinant nucleic acid fragment comprising oligonucleotide having sequence complementary or identical to human genomic dna sequence encoding rna component of human telomerase</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5747317">US5747317</a></td><td class="patent-data-table-td patent-date-value">Jul 8, 1996</td><td class="patent-data-table-td patent-date-value">May 5, 1998</td><td class="patent-data-table-td ">Tularik Inc.</td><td class="patent-data-table-td ">Human telomerase RNA interacting protein gene</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5770422">US5770422</a></td><td class="patent-data-table-td patent-date-value">Jul 8, 1996</td><td class="patent-data-table-td patent-date-value">Jun 23, 1998</td><td class="patent-data-table-td ">The Regents Of The University Of California</td><td class="patent-data-table-td ">Proteins, useful in diagnosis, therapy and in biopharmaceutical industry</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6093809">US6093809</a></td><td class="patent-data-table-td patent-date-value">May 6, 1997</td><td class="patent-data-table-td patent-date-value">Jul 25, 2000</td><td class="patent-data-table-td ">University Technology Corporation</td><td class="patent-data-table-td ">Nucleotide sequence; anticarcinogenic and antiaging agents; diagnostic applications; evolutionary (e.g., phylogenetic) investigations</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6258535">US6258535</a></td><td class="patent-data-table-td patent-date-value">Apr 14, 1998</td><td class="patent-data-table-td patent-date-value">Jul 10, 2001</td><td class="patent-data-table-td ">Geron Corporation</td><td class="patent-data-table-td ">Mammalian telomerase</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6261556">US6261556</a></td><td class="patent-data-table-td patent-date-value">Oct 18, 1999</td><td class="patent-data-table-td patent-date-value">Jul 17, 2001</td><td class="patent-data-table-td ">Geron Corporation</td><td class="patent-data-table-td ">Binding to matrices; separation</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6261836">US6261836</a></td><td class="patent-data-table-td patent-date-value">May 9, 1997</td><td class="patent-data-table-td patent-date-value">Jul 17, 2001</td><td class="patent-data-table-td ">Geron Corporation</td><td class="patent-data-table-td ">Telomerase</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="http://www.google.com/url?id=JOlhBAABERAJ&amp;q=http://worldwide.espacenet.com/publicationDetails/biblio%3FCC%3DJP%26NR%3DH09154575A%26KC%3DA%26FT%3DD&amp;usg=AFQjCNFbm0-tFFO9oF393OCm8uH9YhskGg">JPH09154575A</a></td><td class="patent-data-table-td patent-date-value"></td><td class="patent-data-table-td patent-date-value"></td><td class="patent-data-table-td "> </td><td class="patent-data-table-td citation-no-title">Title not available</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1984003584A1?cl=en">WO1984003584A1</a></td><td class="patent-data-table-td patent-date-value">Mar 8, 1984</td><td class="patent-data-table-td patent-date-value">Sep 13, 1984</td><td class="patent-data-table-td ">Sevylor International</td><td class="patent-data-table-td ">Protection suit, particularly usable in irradiated atmosphere</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1995013382A1?cl=en">WO1995013382A1</a></td><td class="patent-data-table-td patent-date-value">Nov 14, 1994</td><td class="patent-data-table-td patent-date-value">May 18, 1995</td><td class="patent-data-table-td ">Geron Corp</td><td class="patent-data-table-td ">Therapy and diagnosis of conditions related to telomere length and/or telomerase activity</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1996040868A1?cl=en">WO1996040868A1</a></td><td class="patent-data-table-td patent-date-value">Jun 6, 1996</td><td class="patent-data-table-td patent-date-value">Dec 19, 1996</td><td class="patent-data-table-td ">Chantal Autexier</td><td class="patent-data-table-td ">Essential oligonucleotides of vertebrate telomerase</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1998001542A1?cl=en">WO1998001542A1</a></td><td class="patent-data-table-td patent-date-value">Jul 8, 1997</td><td class="patent-data-table-td patent-date-value">Jan 15, 1998</td><td class="patent-data-table-td ">Univ California</td><td class="patent-data-table-td ">Human telomerase</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1998001543A1?cl=en">WO1998001543A1</a></td><td class="patent-data-table-td patent-date-value">Jul 8, 1997</td><td class="patent-data-table-td patent-date-value">Jan 15, 1998</td><td class="patent-data-table-td ">Tularik Inc</td><td class="patent-data-table-td ">Human telomerase gene</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1998001835A1?cl=en">WO1998001835A1</a></td><td class="patent-data-table-td patent-date-value">Jul 7, 1997</td><td class="patent-data-table-td patent-date-value">Jan 15, 1998</td><td class="patent-data-table-td ">Perrone Stephen C</td><td class="patent-data-table-td ">Self-contained, programmable, time interval alarm reminder device for eyedrop medication administration and a means for affixing such to eyedrop/medication container</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1998007838A1?cl=en">WO1998007838A1</a></td><td class="patent-data-table-td patent-date-value">Aug 21, 1997</td><td class="patent-data-table-td patent-date-value">Feb 26, 1998</td><td class="patent-data-table-td ">Yasuhiro Fujino</td><td class="patent-data-table-td ">Higher animal telomerase protein and gene encoding the same</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1998008938A1?cl=en">WO1998008938A1</a></td><td class="patent-data-table-td patent-date-value">Aug 27, 1997</td><td class="patent-data-table-td patent-date-value">Mar 5, 1998</td><td class="patent-data-table-td ">Shonen Yoshida</td><td class="patent-data-table-td ">Proteins having telomerase activity</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1998012811A1?cl=en">WO1998012811A1</a></td><td class="patent-data-table-td patent-date-value">Sep 3, 1997</td><td class="patent-data-table-td patent-date-value">Mar 26, 1998</td><td class="patent-data-table-td ">Bosch Gmbh Robert</td><td class="patent-data-table-td ">Method of selecting a transmit frequency</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1998019580A1?cl=en">WO1998019580A1</a></td><td class="patent-data-table-td patent-date-value">Nov 5, 1997</td><td class="patent-data-table-td patent-date-value">May 14, 1998</td><td class="patent-data-table-td ">Hill Rom Co Inc</td><td class="patent-data-table-td ">Head wall for a hospital room</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1998021343A1?cl=en">WO1998021343A1</a></td><td class="patent-data-table-td patent-date-value">Nov 13, 1997</td><td class="patent-data-table-td patent-date-value">May 22, 1998</td><td class="patent-data-table-td ">Amgen Canada Inc</td><td class="patent-data-table-td ">Genes encoding telomerase proteins</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1998037181A2?cl=en">WO1998037181A2</a></td><td class="patent-data-table-td patent-date-value">Feb 20, 1998</td><td class="patent-data-table-td patent-date-value">Aug 27, 1998</td><td class="patent-data-table-td ">Christopher M Counter</td><td class="patent-data-table-td ">Telomerase catalytic subunit gene and encoded protein</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1998045450A1?cl=en">WO1998045450A1</a></td><td class="patent-data-table-td patent-date-value">Apr 4, 1997</td><td class="patent-data-table-td patent-date-value">Oct 15, 1998</td><td class="patent-data-table-td ">Geron Corp</td><td class="patent-data-table-td ">Purified telomerase</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1998059040A2?cl=en">WO1998059040A2</a></td><td class="patent-data-table-td patent-date-value">Jun 9, 1998</td><td class="patent-data-table-td patent-date-value">Dec 30, 1998</td><td class="patent-data-table-td ">Bayer Ag</td><td class="patent-data-table-td ">Human catalytic telomerase sub-unit and its diagnostic and therapeutic use</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1999001560A1?cl=en">WO1999001560A1</a></td><td class="patent-data-table-td patent-date-value">Jul 1, 1998</td><td class="patent-data-table-td patent-date-value">Jan 14, 1999</td><td class="patent-data-table-td ">David Bowtell</td><td class="patent-data-table-td ">Vertebrate telomerase genes and proteins and uses thereof</td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="npl-citations"></a><div class="patent-section-header"><span class="patent-section-title">Non-Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th colspan="3"class="patent-data-table-th">Reference</th></tr></thead><tr><td class="patent-data-table-td ">1</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">1994 Genome Issue of Science (265:1981f).</td></tr><tr><td class="patent-data-table-td ">2</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Anderson and Young, "<a href='http://scholar.google.com/scholar?q="Quantitative+Filter+Hybridization"'>Quantitative Filter Hybridization</a>" in Nucleic Acid Hybridization pp. 73-111 (1985).</td></tr><tr><td class="patent-data-table-td ">3</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Ausubel et al., Current Protocols In Molecular Biology, John Wiley &amp; Sons, New York NY (1989).</td></tr><tr><td class="patent-data-table-td ">4</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Autexier et al., "<a href='http://scholar.google.com/scholar?q="Reconstitution+of+human+telomerase+activity+and+Identification+of+a+minimal+functional+region+of+the+human+telomease+RNA%2C"'>Reconstitution of human telomerase activity and Identification of a minimal functional region of the human telomease RNA,</a>" (1998) EMBO J, 15:5928.</td></tr><tr><td class="patent-data-table-td ">5</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Auxexier and Grelder, "<a href='http://scholar.google.com/scholar?q="Functional+reconstitution+of+wild-type+and+mutant+Tetrahymena+telomerase%2C"'>Functional reconstitution of wild-type and mutant Tetrahymena telomerase,</a>" (1994) Genes Develop., 8:583.</td></tr><tr><td class="patent-data-table-td ">6</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Berger and Kimmel, Guide to Molecualar Cloning Techniques, Meth. Enzymol., vol. 152, Academic Press, San Diego CA (1987).</td></tr><tr><td class="patent-data-table-td ">7</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Biessman et al., "<a href='http://scholar.google.com/scholar?q="Addition+of+Telomere-Associated+HeT+DNA+Sequences+%22Heals%22+Broken+Chromosome+End+is+in+Drosophila%2C"'>Addition of Telomere-Associated HeT DNA Sequences "Heals" Broken Chromosome End is in Drosophila,</a>" (1990) Cell, 61:683.</td></tr><tr><td class="patent-data-table-td ">8</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Bitter et al., "<a href='http://scholar.google.com/scholar?q="Expression+and+secretion+vectors+for+yeast%2C"'>Expression and secretion vectors for yeast,</a>" Meth Enzymol., (1987) 153:516.</td></tr><tr><td class="patent-data-table-td ">9</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Blackburn and Chlou, "<a href='http://scholar.google.com/scholar?q="Non-nucleosomal+packaging+of+a+tandemly+repeated+DNA+sequence+at+termini+of+extrachromosomal+DNA+coding+for+rRNA+in+Tetrahymena%2C"'>Non-nucleosomal packaging of a tandemly repeated DNA sequence at termini of extrachromosomal DNA coding for rRNA in Tetrahymena,</a>" (1981) Proc. Natl. Acad. Sci., 78:2263.</td></tr><tr><td class="patent-data-table-td ">10</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Blackburn and Gall, "<a href='http://scholar.google.com/scholar?q="A+tandemly+repeated+sequence+at+the+termini+of+the+extrachromosomal+ribosomal+RNA+genes+In+Tetrahymena%2C"'>A tandemly repeated sequence at the termini of the extrachromosomal ribosomal RNA genes In Tetrahymena,</a>" (1978) J. Mol. Biol., 120:33.</td></tr><tr><td class="patent-data-table-td ">11</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Blackburn, "<a href='http://scholar.google.com/scholar?q="Telomerases%2C"'>Telomerases,</a>" (1992) Ann. Rev. Biochem., 61:113.</td></tr><tr><td class="patent-data-table-td ">12</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Bodnar et al., "<a href='http://scholar.google.com/scholar?q="Extension+of+Life-Span+by+Introduction+of+Telomerase+into+Normal+Human+Cells%2C"'>Extension of Life-Span by Introduction of Telomerase into Normal Human Cells,</a>" (1998) Science, 279:349.</td></tr><tr><td class="patent-data-table-td ">13</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Bradford, "<a href='http://scholar.google.com/scholar?q="A+Rapid+and+Sensitive+method+for+the+Quantitation+of+Microgram+Quantities+of+Protein+Utilizing+the+Principle+of+Protein-Dye+Binding%2C"'>A Rapid and Sensitive method for the Quantitation of Microgram Quantities of Protein Utilizing the Principle of Protein-Dye Binding,</a>" (1976) Anal. Biochem., 72:248.</td></tr><tr><td class="patent-data-table-td ">14</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Braunstein et al., "<a href='http://scholar.google.com/scholar?q="Transcriptional+silencing+in+yeast+is+associated+with+reduced+nuclcosome+acetylation%2C"'>Transcriptional silencing in yeast is associated with reduced nuclcosome acetylation,</a>" (1993) Genes Develop., 7:592.</td></tr><tr><td class="patent-data-table-td ">15</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Calvio et al., "<a href='http://scholar.google.com/scholar?q="Identification+of+hnRNP+P2+as+TLS%2FFUS+using+electrospray+mass+spectromotry%2C"'>Identification of hnRNP P2 as TLS/FUS using electrospray mass spectromotry,</a>" (1995) RNA, 1:724.</td></tr><tr><td class="patent-data-table-td ">16</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Caruthers et al., "<a href='http://scholar.google.com/scholar?q="New+chemical+methods+for+synthesizing+polynucleotides%2C"'>New chemical methods for synthesizing polynucleotides,</a>" (1980) Nucleic Acids Res. Symp. Sor., 215-223.</td></tr><tr><td class="patent-data-table-td ">17</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Chan and Tye, "<a href='http://scholar.google.com/scholar?q="Organization+of+DNA+sequences+and+replication+origins+at+yeast+telomeres%2C"'>Organization of DNA sequences and replication origins at yeast telomeres,</a>" (1983) Cell, 33:663.</td></tr><tr><td class="patent-data-table-td ">18</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Colbere-Gerapin et al., "<a href='http://scholar.google.com/scholar?q="A+new+dominat+hybrid+selective+marker+for+higher+eukaryotic+cells%2C"'>A new dominat hybrid selective marker for higher eukaryotic cells,</a>" (1981) J. Mol. Biol., 150:1.</td></tr><tr><td class="patent-data-table-td ">19</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Cole et al., "<a href='http://scholar.google.com/scholar?q="Generation+of+human+monoclonal+antibodies+reactive+with+cellular+antigens%2C"'>Generation of human monoclonal antibodies reactive with cellular antigens,</a>" (1983) Proc. Natl. Acad. Sci., 80:2026.</td></tr><tr><td class="patent-data-table-td ">20</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Cole et al., "<a href='http://scholar.google.com/scholar?q="The+EBV-hybridoma+technique+and+its+application+to+human+lung+cancer%2C"'>The EBV-hybridoma technique and its application to human lung cancer,</a>" Monoclonal Antibodies and Cancer Therapy, Alan R. Liss Inc., New York NY 77-96 (1985).</td></tr><tr><td class="patent-data-table-td ">21</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Collins et al., "<a href='http://scholar.google.com/scholar?q="Purification+of+Tetrahymena+telomerase+and+cloning+of+genes+encoding+the+two+protein+components+of+the+enzyme%2C"'>Purification of Tetrahymena telomerase and cloning of genes encoding the two protein components of the enzyme,</a>" (1995) Cell, 61:677.</td></tr><tr><td class="patent-data-table-td ">22</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Conrad et al., "<a href='http://scholar.google.com/scholar?q="RAP1+protein+interacts+with+yeast+telomers+In+vivo%3A+Overproduction+allers+telomers+structure+and+decreases+chromosome+stability%2C"'>RAP1 protein interacts with yeast telomers In vivo: Overproduction allers telomers structure and decreases chromosome stability,</a>" (1990) Cell, 63-739.</td></tr><tr><td class="patent-data-table-td ">23</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Coombs, Dictionary of Biotechnology, Stockton Press, New York NY (1994).</td></tr><tr><td class="patent-data-table-td ">24</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Counter et al., "<a href='http://scholar.google.com/scholar?q="The+calalytic+subunit+of+yeast+telomerase%2C"'>The calalytic subunit of yeast telomerase,</a>" (1997) Proc. Natl. Acad. Sci., 94:9202.</td></tr><tr><td class="patent-data-table-td ">25</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Counter et al., (1994) Proc. Natl. Acad. Sci., 91:2900.</td></tr><tr><td class="patent-data-table-td ">26</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Creighton, Proteins, Structures and Molecular Principles, WH Freeman and Co, New York, NY (1983).</td></tr><tr><td class="patent-data-table-td ">27</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Dieffenbach and Dvekaler, PCR Primer, a Laboratory Manual, Cold Spring Harbor Press, Plainview NY (1995).</td></tr><tr><td class="patent-data-table-td ">28</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Duplaa et al., "<a href='http://scholar.google.com/scholar?q="Quantitative+analysis+of+polymerase+chain+reaction+products+using+biotinylated+dUTP+incorporation%2C"'>Quantitative analysis of polymerase chain reaction products using biotinylated dUTP incorporation,</a>" (1993) Anal. Biochem., 212:229.</td></tr><tr><td class="patent-data-table-td ">29</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Fang et al., "<a href='http://scholar.google.com/scholar?q="Oxytricha+telomere-binding+protein%3A+separable+DNA-binding+and+dimerization+domains+of+the+alpha-subunit%2C"'>Oxytricha telomere-binding protein: separable DNA-binding and dimerization domains of the alpha-subunit,</a>" Genes Develop. 7:870 (1993) and Gray et al., (1991) Coli 67:807.</td></tr><tr><td class="patent-data-table-td ">30</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Fang et al., "<a href='http://scholar.google.com/scholar?q="The+RNA+Componant+of+Human+Telomerase%2C"'>The RNA Componant of Human Telomerase,</a>" (1995) Science, 269:1236.</td></tr><tr><td class="patent-data-table-td ">31</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Fang et al., "<a href='http://scholar.google.com/scholar?q="Oxytricha+telomere-binding+protein%3A+separable+DNA-binding+and+dimerization+domains+of+the+%CE%B1-subunit%2C"'>Oxytricha telomere-binding protein: separable DNA-binding and dimerization domains of the α-subunit,</a>" Genes Develop. 7:870 (1993) and Gray et al., (1991) Coli 67:807.</td></tr><tr><td class="patent-data-table-td ">32</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">GenBank Accession No. AA281296 (Apr. 1997).</td></tr><tr><td class="patent-data-table-td ">33</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">GenBank accession No. AA299878 (Apr. 1997).</td></tr><tr><td class="patent-data-table-td ">34</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Genbank accession No. AA311750 (Apr. 1997).</td></tr><tr><td class="patent-data-table-td ">35</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Gilley et al., "<a href='http://scholar.google.com/scholar?q="Altering+specific+telomerase+RNA+template+residues+affects+active+site+function%2C"'>Altering specific telomerase RNA template residues affects active site function,</a>" (1995) Genes Develop., 9:2214.</td></tr><tr><td class="patent-data-table-td ">36</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Gottachling and Zaldan, "<a href='http://scholar.google.com/scholar?q="Telomere+proteins%3A+specific+recognition+end+protection+of+the+natural+termini+of+Oxytricha+macronuclear+DNA%2C"'>Telomere proteins: specific recognition end protection of the natural termini of Oxytricha macronuclear DNA,</a>" (1986) Cell 47:195.</td></tr><tr><td class="patent-data-table-td ">37</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Gottschling and Cech, "<a href='http://scholar.google.com/scholar?q="Chromatin+Structure+of+the+Molecular+Ends+of+Oxytricha+Mononuclear+DNA%3A+Phased+Nucleosomes+and+a+Telomeric+Complex%2C"'>Chromatin Structure of the Molecular Ends of Oxytricha Mononuclear DNA: Phased Nucleosomes and a Telomeric Complex,</a>" (1984) Cell, 98:501.</td></tr><tr><td class="patent-data-table-td ">38</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Grant et al., Meth Enzymol., (1987) 153:516-544.</td></tr><tr><td class="patent-data-table-td ">39</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Greenwood et al., "<a href='http://scholar.google.com/scholar?q="Phylogenetic+relationships+within+the+class+oligohymenophores%2C+phylum+cillophora%2C+inferred+from+the+complete+small+subunit+rRNA+gene+sequences+of+Colpidium+campylum%2C+Glaucoma+chattoni%2C+and+Opisthonecta+henneguyi%2C"'>Phylogenetic relationships within the class oligohymenophores, phylum cillophora, inferred from the complete small subunit rRNA gene sequences of Colpidium campylum, Glaucoma chattoni, and Opisthonecta henneguyi,</a>" (1991) J. Mol. Evol., 3:163.</td></tr><tr><td class="patent-data-table-td ">40</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Greider and Blackburn, "<a href='http://scholar.google.com/scholar?q="A+telomeric+sequence+in+the+RNA+of+Tetrahymena+telomerase+required+for+telomere+repeat+synthesis%2C"'>A telomeric sequence in the RNA of Tetrahymena telomerase required for telomere repeat synthesis,</a>" (1989) Nature, 337:331.</td></tr><tr><td class="patent-data-table-td ">41</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Greider, "<a href='http://scholar.google.com/scholar?q="Telomere+Length+Regulation%2C"'>Telomere Length Regulation,</a>" (1996) Ann. Rev. Biochem., 65:337.</td></tr><tr><td class="patent-data-table-td ">42</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Grelder and Blackburn. "<a href='http://scholar.google.com/scholar?q="Identification+of+a+specific+telomere+terminal+transferase+activity+in+Tetrehymena+extracts%2C"'>Identification of a specific telomere terminal transferase activity in Tetrehymena extracts,</a>" (1985) Cell, 43:405.</td></tr><tr><td class="patent-data-table-td ">43</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Grelder, "<a href='http://scholar.google.com/scholar?q="Telomerase+is+processive%2C"'>Telomerase is processive,</a>" (1991) Mol. Cell. Biol., 11:4572.</td></tr><tr><td class="patent-data-table-td ">44</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Hampton et al., Serological Methods a Laboratory Manual, APS Press, St Paul MN (1990).</td></tr><tr><td class="patent-data-table-td ">45</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Harrington et al., "<a href='http://scholar.google.com/scholar?q="A+Mammallan+Telomerase-Associated+Protein%2C"'>A Mammallan Telomerase-Associated Protein,</a>" (1997) Science, 276:973.</td></tr><tr><td class="patent-data-table-td ">46</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Harrington et al., "<a href='http://scholar.google.com/scholar?q="Human+telomerase+contains+evolutionarily+conserved+catalytic+and+structural+subunits%2C"'>Human telomerase contains evolutionarily conserved catalytic and structural subunits,</a>" (1997) Genes Dev., 11:3109.</td></tr><tr><td class="patent-data-table-td ">47</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Hartman and Mulligan, "<a href='http://scholar.google.com/scholar?q="Two+dominant-acting+selectable+markets+for+gene+transfer+studies+in+mammalian+cells%2C"'>Two dominant-acting selectable markets for gene transfer studies in mammalian cells,</a>" (1988) Proc. Acad. Sci., 85:8047.</td></tr><tr><td class="patent-data-table-td ">48</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Henderson and Blackburn, "<a href='http://scholar.google.com/scholar?q="An+overhanging+3%27%27+terminus+is+a+conserved+feature+of+telomeres%2C"'>An overhanging 3'' terminus is a conserved feature of telomeres,</a>" (1989) Mol Cell. Biol., 9:345.</td></tr><tr><td class="patent-data-table-td ">49</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Henderson and Blackburn, "<a href='http://scholar.google.com/scholar?q="An+overhanging+3%E2%80%B3+terminus+is+a+conserved+feature+of+telomeres%2C"'>An overhanging 3″ terminus is a conserved feature of telomeres,</a>" (1989) Mol Cell. Biol., 9:345.</td></tr><tr><td class="patent-data-table-td ">50</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Horn et al., "<a href='http://scholar.google.com/scholar?q="Synthesis+of+oligonucleotides+on+cellulose.+Part+II%3A+design+and+synthetic+strategy+to+the+synthesis+of+22+oligodeoxynucleotides+coding+for+gastric+inhibitory+polypeptide+%28GIP%29%2C"'>Synthesis of oligonucleotides on cellulose. Part II: design and synthetic strategy to the synthesis of 22 oligodeoxynucleotides coding for gastric inhibitory polypeptide (GIP),</a>" (1980) Nucleic Acids Res. Symp. Ser., 225-232.</td></tr><tr><td class="patent-data-table-td ">51</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Hudson et al., "<a href='http://scholar.google.com/scholar?q="An+STS-based+map+of+the+human+genome%2C"'>An STS-based map of the human genome,</a>" (1995) Science, 270:1945.</td></tr><tr><td class="patent-data-table-td ">52</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Huse et al., "<a href='http://scholar.google.com/scholar?q="Generation+of+a+largo+combinatorial+library+of+the+immunoglobulin+repertoire+in+phage+lambda%2C"'>Generation of a largo combinatorial library of the immunoglobulin repertoire in phage lambda,</a>" (1989) Science, 246:1275.</td></tr><tr><td class="patent-data-table-td ">53</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Johnson et al., (1981) Mol. Cell Biol. 11:1.</td></tr><tr><td class="patent-data-table-td ">54</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Kilian et al., "<a href='http://scholar.google.com/scholar?q="Isolation+of+a+candidate+human+telomerase+catalytic+subunit+gene%2C+which+reveals+complex+splicing+patterns+in+different+cell+types%2C"'>Isolation of a candidate human telomerase catalytic subunit gene, which reveals complex splicing patterns in different cell types,</a>" (1997) Hum. Mol. Genet., 6:2011.</td></tr><tr><td class="patent-data-table-td ">55</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Kipling and Cooke, "<a href='http://scholar.google.com/scholar?q="Hypervariable+ultra-long+telomeres+in+mice%2C"'>Hypervariable ultra-long telomeres in mice,</a>" (1990) Nature 347:400.</td></tr><tr><td class="patent-data-table-td ">56</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Klobutcher et al., "<a href='http://scholar.google.com/scholar?q="All+gene-sized+DNA+molecules+in+four+species+of+hypotrichs+have+the+same+terminal+sequence+and+an+unusual+3%27%27+terminus%2C"'>All gene-sized DNA molecules in four species of hypotrichs have the same terminal sequence and an unusual 3'' terminus,</a>" (1981) Proc. Natl. Acad. Sci., 78:3015.</td></tr><tr><td class="patent-data-table-td ">57</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Klobutcher et al., "<a href='http://scholar.google.com/scholar?q="All+gene-sized+DNA+molecules+in+four+species+of+hypotrichs+have+the+same+terminal+sequence+and+an+unusual+3%E2%80%B3+terminus%2C"'>All gene-sized DNA molecules in four species of hypotrichs have the same terminal sequence and an unusual 3″ terminus,</a>" (1981) Proc. Natl. Acad. Sci., 78:3015.</td></tr><tr><td class="patent-data-table-td ">58</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Koehler and Milstein, "<a href='http://scholar.google.com/scholar?q="Continuous+cultures+of+fused+cells+secreting+antibody+of+predefined+specificity%2C"'>Continuous cultures of fused cells secreting antibody of predefined specificity,</a>" (1975) Nature 258:495.</td></tr><tr><td class="patent-data-table-td ">59</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Kosbor et al., "<a href='http://scholar.google.com/scholar?q="The+production+of+monocinal+antibodies+from+human+lymphocytes%2C"'>The production of monocinal antibodies from human lymphocytes,</a>" (1983) Immunol. Today 4:72.</td></tr><tr><td class="patent-data-table-td ">60</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Lamond and Sproat, (1994) "<a href='http://scholar.google.com/scholar?q="Isolation+and+Characterization+of+Ribonucleoprotein+Complexes%2C"'>Isolation and Characterization of Ribonucleoprotein Complexes,</a>" pp. 103-140.</td></tr><tr><td class="patent-data-table-td ">61</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Lamond et al., "<a href='http://scholar.google.com/scholar?q="Probing+the+structure+and+function+of+U2+an+RNP+with+antisense+oligonucleotides+made+of+2%27%27-OMe+RNA%2C"'>Probing the structure and function of U2 an RNP with antisense oligonucleotides made of 2''-OMe RNA,</a>" (1989) Cell, 58:383.</td></tr><tr><td class="patent-data-table-td ">62</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Lamond et al., "<a href='http://scholar.google.com/scholar?q="Probing+the+structure+and+function+of+U2+an+RNP+with+antisense+oligonucleotides+made+of+2%E2%80%B3-OMe+RNA%2C"'>Probing the structure and function of U2 an RNP with antisense oligonucleotides made of 2″-OMe RNA,</a>" (1989) Cell, 58:383.</td></tr><tr><td class="patent-data-table-td ">63</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Lendvey et al., "<a href='http://scholar.google.com/scholar?q="Senescence+mutants+of+Saccharomyces+cerevisiae+with+a+defect+in+telomere+replication+identify+three+additional+EST+genes%2C"'>Senescence mutants of Saccharomyces cerevisiae with a defect in telomere replication identify three additional EST genes,</a>" (1996) Genetics, 144.</td></tr><tr><td class="patent-data-table-td ">64</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Lingler et al., "<a href='http://scholar.google.com/scholar?q="Purification+of+telomerase+from+Euplotes+adeiculatus%3A+requirement+of+a+printer+3%27%27+overhang%2C"'>Purification of telomerase from Euplotes adeiculatus: requirement of a printer 3'' overhang,</a>" (1998) Proc. Natl. Acad. Sci., 93:10712.</td></tr><tr><td class="patent-data-table-td ">65</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Lingler et al., "<a href='http://scholar.google.com/scholar?q="Reverse+transcriptase+motifs+in+the+catalytic+subunits+of+telomerase%2C"'>Reverse transcriptase motifs in the catalytic subunits of telomerase,</a>" (1997) Science, 278:561.</td></tr><tr><td class="patent-data-table-td ">66</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Lingler et al., "<a href='http://scholar.google.com/scholar?q="Purification+of+telomerase+from+Euplotes+adeiculatus%3A+requirement+of+a+printer+3%E2%80%B3+overhang%2C"'>Purification of telomerase from Euplotes adeiculatus: requirement of a printer 3″ overhang,</a>" (1998) Proc. Natl. Acad. Sci., 93:10712.</td></tr><tr><td class="patent-data-table-td ">67</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Lingner et al., "<a href='http://scholar.google.com/scholar?q="Telomerase+and+DNA+End+Replication%3A+No+Longer+a+Lagging+Strand+Problem%3F%2C"'>Telomerase and DNA End Replication: No Longer a Lagging Strand Problem?,</a>" (1995) Science 289:1533.</td></tr><tr><td class="patent-data-table-td ">68</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Lingner et al., "<a href='http://scholar.google.com/scholar?q="Telomerase+RNAs+of+different+ciliates+have+a+common+secondary+structure+and+a+permitted+template%2C"'>Telomerase RNAs of different ciliates have a common secondary structure and a permitted template,</a>" Genes Develop., 8:1984.</td></tr><tr><td class="patent-data-table-td ">69</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Lowy et al., "<a href='http://scholar.google.com/scholar?q="Isolation+of+transforming+DNA%3A+Cloning+the+hamster+aprt+gene%2C"'>Isolation of transforming DNA: Cloning the hamster aprt gene,</a>" (1980) Cell, 22:617.</td></tr><tr><td class="patent-data-table-td ">70</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Lustig and Peles, Identification of yeast mutants with altered telomere structure, (1986) Proc. Natl. Acad. Sci., 83:1398.</td></tr><tr><td class="patent-data-table-td ">71</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Maddox et al., "<a href='http://scholar.google.com/scholar?q="Elevated+serum+levels+in+human+pregnancy+of+a+molecule+immunochemically+similar+to+eosinophil+granule+major+basic+protein%2C"'>Elevated serum levels in human pregnancy of a molecule immunochemically similar to eosinophil granule major basic protein,</a>" (1983) J. Exp. Med., 158:1211.</td></tr><tr><td class="patent-data-table-td ">72</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Malby et al., "<a href='http://scholar.google.com/scholar?q="Quantitative+measurement+of+human+cytokine+gene+expression+by+polymerase+chain+reaction%2C"'>Quantitative measurement of human cytokine gene expression by polymerase chain reaction,</a>" (1993) J. Immunol. Meth., 159:235.</td></tr><tr><td class="patent-data-table-td ">73</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">McEachern and Blackburn, "<a href='http://scholar.google.com/scholar?q="runaway+telomere+elongation+caused+by+telomerase+RNA+gene+mutation%2C"'>runaway telomere elongation caused by telomerase RNA gene mutation,</a>" (1995) Nature, 376:403.</td></tr><tr><td class="patent-data-table-td ">74</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Mekerov et al., "<a href='http://scholar.google.com/scholar?q="Nucleosomal+Organization+of+Telomere-Specific+Chromatin+in+Rat%2C"'>Nucleosomal Organization of Telomere-Specific Chromatin in Rat,</a>" (1993) Cell, 73:775.</td></tr><tr><td class="patent-data-table-td ">75</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Merrifield, "<a href='http://scholar.google.com/scholar?q="Solid+phase+peptide+synthesis%2C+I.+The+synthesis+of+a+tetrapeptide%2C"'>Solid phase peptide synthesis, I. The synthesis of a tetrapeptide,</a>" (1983) J. Am. Chem. Soc., 65:2149.</td></tr><tr><td class="patent-data-table-td ">76</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Meyerson et al., "<a href='http://scholar.google.com/scholar?q="hEST2%2C+the+Putative+Human+Telomerase+Catalytic+Subunit+Gene%2C+Is+Up-Regulated+In+Tumor+Cells+and+during+Immortalization%2C"'>hEST2, the Putative Human Telomerase Catalytic Subunit Gene, Is Up-Regulated In Tumor Cells and during Immortalization,</a>" (1997) Cell, 90:785.</td></tr><tr><td class="patent-data-table-td ">77</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Murray, In McGraw Hill Yearbook of Science and Technology, (1992) McGraw Hill, New York NY, pp 181-196.</td></tr><tr><td class="patent-data-table-td ">78</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Nakamura et al., "<a href='http://scholar.google.com/scholar?q="Telomerase+Catalytic+Subunit+Homologs+from+Fission+Yeast+and+Human%2C"'>Telomerase Catalytic Subunit Homologs from Fission Yeast and Human,</a>" (1997) Science, 277:955.</td></tr><tr><td class="patent-data-table-td ">79</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Nakayama et al., "<a href='http://scholar.google.com/scholar?q="TLP1%3A+A+Gene+Encoding+a+Protein+Component+of+Mammalian+Telomerase+is+a+Novel+Member+of+WD+Repeats+Family%2C"'>TLP1: A Gene Encoding a Protein Component of Mammalian Telomerase is a Novel Member of WD Repeats Family,</a>" (1997) Cell, 88:875.</td></tr><tr><td class="patent-data-table-td ">80</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Nielsen et al., (1993) "<a href='http://scholar.google.com/scholar?q="Peptide+nucleic+acids+%28PNAs%29%3A+Potential+antisense+and+anti-gene+agents%2C"'>Peptide nucleic acids (PNAs): Potential antisense and anti-gene agents,</a>" Anticancer Drug Des., 6:53.</td></tr><tr><td class="patent-data-table-td ">81</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Oka et al., "<a href='http://scholar.google.com/scholar?q="Inverted+terminal+repeat+sequence+in+the+macronuclear+DNA+of+Stylonychia+pustulata%2C"'>Inverted terminal repeat sequence in the macronuclear DNA of Stylonychia pustulata,</a>" (1980) Gene, 10:301.</td></tr><tr><td class="patent-data-table-td ">82</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Olovnikov, "<a href='http://scholar.google.com/scholar?q="A+theory+of+marginotomy%3A+The+incomplete+copying+of+template+margin+in+enzymic+synthesis+of+polynucleotides+and+biological+significance+of+the+phenomonon%2C"'>A theory of marginotomy: The incomplete copying of template margin in enzymic synthesis of polynucleotides and biological significance of the phenomonon,</a>" (1973) J. Theor. Biol., 41:181.</td></tr><tr><td class="patent-data-table-td ">83</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Orlandl et al., "<a href='http://scholar.google.com/scholar?q="Cloning+immunoglobulin+variable+domains+for+expression+by+the+polymerase+chain+reaction%2C"'>Cloning immunoglobulin variable domains for expression by the polymerase chain reaction,</a>" (1989) Proc. Natl. Acad. Sci., 86:8833.</td></tr><tr><td class="patent-data-table-td ">84</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Prescott, "<a href='http://scholar.google.com/scholar?q="The+DNA+of+cilated+protozea%2C"'>The DNA of cilated protozea,</a>" (1994) Microbiol. Rev., 58:233.</td></tr><tr><td class="patent-data-table-td ">85</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Price, (1993) Blood Rev., 7:127.</td></tr><tr><td class="patent-data-table-td ">86</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Rhodes et al., "<a href='http://scholar.google.com/scholar?q="Transformation+of+maize+by+electroporation+of+embryos%2C"'>Transformation of maize by electroporation of embryos,</a>" 1985) Meth. Mol. Biol., 55:121.</td></tr><tr><td class="patent-data-table-td ">87</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Roberge et al., "<a href='http://scholar.google.com/scholar?q="A+strategy+for+a+convergent+synthesis+of+N-linked+glycopeptides+on+a+solid+support%2C"'>A strategy for a convergent synthesis of N-linked glycopeptides on a solid support,</a>" (1995) Science, 269:202.</td></tr><tr><td class="patent-data-table-td ">88</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Romero and Blackburn, "<a href='http://scholar.google.com/scholar?q="A+conserved+secondary+structure+for+telomerase+RNA%2C"'>A conserved secondary structure for telomerase RNA,</a>" (1991) Cell, 67:343.</td></tr><tr><td class="patent-data-table-td ">89</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Sambrock et al., Molecular Cloning, A Laboratory Manual, Cold Spring Harbor Press, Plainview NY (1989).</td></tr><tr><td class="patent-data-table-td ">90</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Sandell et al., "<a href='http://scholar.google.com/scholar?q="Transcription+of+yeast+telomere+alleviates+telomere+position+effect+without+affecting+chromosome+stability%2C"'>Transcription of yeast telomere alleviates telomere position effect without affecting chromosome stability,</a>" (1994) Proc. Natl. Acad. Sci., 91:12081.</td></tr><tr><td class="patent-data-table-td ">91</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Sanger et al., "<a href='http://scholar.google.com/scholar?q="DNA+sequencing+with+chain-terminating+Inhibitors%2C"'>DNA sequencing with chain-terminating Inhibitors,</a>" (1977) Proc. Natl. Acad. Sci., 74:5463.</td></tr><tr><td class="patent-data-table-td ">92</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Scharf et al., "<a href='http://scholar.google.com/scholar?q="Heat+stress+promoters+and+transcription+factors%2C"'>Heat stress promoters and transcription factors,</a>" (1994) Result Probl. Cell Differ. 20:125.</td></tr><tr><td class="patent-data-table-td ">93</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Shampay and Blackburn, "<a href='http://scholar.google.com/scholar?q="Generation+of+telomere-length+heterogeneity+in+Saccharomyces+cerevisiae%2C"'>Generation of telomere-length heterogeneity in Saccharomyces cerevisiae,</a>" (1988) Proc. Natl. Acad. Sci., 85:534.</td></tr><tr><td class="patent-data-table-td ">94</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Sheen and Levis, "<a href='http://scholar.google.com/scholar?q="Transposition+of+the+LINE-like+retrotransposon+TART+to+Drosophila+chromosome+termini%2C"'>Transposition of the LINE-like retrotransposon TART to Drosophila chromosome termini,</a>" (1994) Proc. Natl. Acad. Sci., 91:12510.</td></tr><tr><td class="patent-data-table-td ">95</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Singer and Gottschling, "<a href='http://scholar.google.com/scholar?q="TLC1%3A+Template+RNA+Component+of+Saccharomyces+cerevisiae+Telomerase%2C"'>TLC1: Template RNA Component of Saccharomyces cerevisiae Telomerase,</a>" (1994) Science 266:404.</td></tr><tr><td class="patent-data-table-td ">96</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Starling et al., "<a href='http://scholar.google.com/scholar?q="Extensive+telomere+repeat+arrays+in+mouse+are+hypervariable%2C"'>Extensive telomere repeat arrays in mouse are hypervariable,</a>" (1990) Nucleic Acids Res., 18:6881.</td></tr><tr><td class="patent-data-table-td ">97</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Swanton et al., "<a href='http://scholar.google.com/scholar?q="Arrangement+of+Cooling+and+Non-coding+Sequence+in+the+DNA+Molecules+Coding+for+rRNAs+in+Oxytricha+sp.%2C"'>Arrangement of Cooling and Non-coding Sequence in the DNA Molecules Coding for rRNAs in Oxytricha sp.,</a>" (1980) Chromosoma 77:203.</td></tr><tr><td class="patent-data-table-td ">98</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Tommerup et al., "<a href='http://scholar.google.com/scholar?q="Unusual+chromatin+in+human+telomeres%2C"'>Unusual chromatin in human telomeres,</a>" (1994) Mol. Cell. Biol., 14:5777.</td></tr><tr><td class="patent-data-table-td ">99</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Trask, "<a href='http://scholar.google.com/scholar?q="Fluorescence+in+situ+hybridization%3A+application+in+cytogenetics+and+gene+mapping%2C"'>Fluorescence in situ hybridization: application in cytogenetics and gene mapping,</a>" (1991) Trends Genet., 7:149.</td></tr><tr><td class="patent-data-table-td ">100</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">U.S. patent application Ser. No. 08/751,189, Harrington et al., filed Nov. 15, 1996.</td></tr><tr><td class="patent-data-table-td ">101</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">U.S. patent application Ser. No. 60/038,760, Counter et al., filed Feb. 20, 1997.</td></tr><tr><td class="patent-data-table-td ">102</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Verma et al., "<a href='http://scholar.google.com/scholar?q="Human+Chromosomes%3A+A+Manual+of+Basic+Techniques%2C"'>Human Chromosomes: A Manual of Basic Techniques,</a>" Pergamon Press, New York NY (1988).</td></tr><tr><td class="patent-data-table-td ">103</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Watson, "<a href='http://scholar.google.com/scholar?q="Origin+of+concatermeric+T7+DNA%2C"'>Origin of concatermeric T7 DNA,</a>" (1972) Nature New Biol., 239:197.</td></tr><tr><td class="patent-data-table-td ">104</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Weinrich et al., "<a href='http://scholar.google.com/scholar?q="Reconstitution+of+human+telomerase+with+the+template+RNA+component+hTR+and+the+catalytic+protein+subunit+hTRT%2C"'>Reconstitution of human telomerase with the template RNA component hTR and the catalytic protein subunit hTRT,</a>" (1997) Nat. Genet., 17(4):498.</td></tr><tr><td class="patent-data-table-td ">105</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Wellinger et al., "<a href='http://scholar.google.com/scholar?q="Origin+activation+and+formation+of+single-strand+TG1-3+tails+occur+sequentially+in+late+S+phase+on+a+Yeast+linear+plasmid%2C"'>Origin activation and formation of single-strand TG1-3 tails occur sequentially in late S phase on a Yeast linear plasmid,</a>" (1993) Mol. Cell. Biol., 13:4057.</td></tr><tr><td class="patent-data-table-td ">106</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Wellinger et al., "<a href='http://scholar.google.com/scholar?q="Saccharomyces+Telomeres+Acquire+Single-Strand+TG1-3+Tails+Late+in+S+Phase%2C"'>Saccharomyces Telomeres Acquire Single-Strand TG1-3 Tails Late in S Phase,</a>" (1993) Cell 72:51.</td></tr><tr><td class="patent-data-table-td ">107</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Whitehead Institute/MIT Center for Genome Research, Genetic Map of the Mouse, Database Release 10, Apr. 28, 1995.</td></tr><tr><td class="patent-data-table-td ">108</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Wigler et al., "<a href='http://scholar.google.com/scholar?q="Transfer+of+purified+herpes+virus+thymidine+kinase+gene+to+cultured+mouse+cells%2C"'>Transfer of purified herpes virus thymidine kinase gene to cultured mouse cells,</a>" (1977) Cell, 11:223.</td></tr><tr><td class="patent-data-table-td ">109</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Wigler et al., "<a href='http://scholar.google.com/scholar?q="Transformation+of+mammalian+cells+with+an+amplifiable+dominant-acting+gene%2C"'>Transformation of mammalian cells with an amplifiable dominant-acting gene,</a>" (1980) Proc. Natl. Acad. Sci., 77:3567.</td></tr><tr><td class="patent-data-table-td ">110</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Winter and Milstein, "<a href='http://scholar.google.com/scholar?q="Man-made+antibodies%2C"'>Man-made antibodies,</a>" (1991) Nature, 349:293.</td></tr><tr><td class="patent-data-table-td ">111</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Wright et al., "<a href='http://scholar.google.com/scholar?q="Saccharomycas+telomeres+assume+a+non-nucleosomal+chromatin+structure%2C"'>Saccharomycas telomeres assume a non-nucleosomal chromatin structure,</a>" (1992) Genes Develop., 6:197.</td></tr><tr><td class="patent-data-table-td ">112</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Yu et al., "<a href='http://scholar.google.com/scholar?q="In+vivo+alteration+of+telomere+sequences+and+senescence+caused+by+mutated+Tetrahymena+telomerase+RNAs%2C"'>In vivo alteration of telomere sequences and senescence caused by mutated Tetrahymena telomerase RNAs,</a>" (1990) Nature, 344:126.</td></tr><tr><td class="patent-data-table-td ">113</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Zahler and Prescott, "<a href='http://scholar.google.com/scholar?q="Telomere+terminal+transferase+activity+in+the+hypotrichous+ciliate+Oxytricha+nova+and+a+model+for+replication+of+the+ends+of+linear+DNA+molecules%2C"'>Telomere terminal transferase activity in the hypotrichous ciliate Oxytricha nova and a model for replication of the ends of linear DNA molecules,</a>" (1988) Nucleic Acids Res., 16:6953.</td></tr><tr><td class="patent-data-table-td ">114</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Zakian, Telomeres: Beginning to Understand the End, (1995) Science 270:1601.</td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="forward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Referenced by</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Citing Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6808880">US6808880</a></td><td class="patent-data-table-td patent-date-value">Jan 19, 2001</td><td class="patent-data-table-td patent-date-value">Oct 26, 2004</td><td class="patent-data-table-td ">Geron Corporation</td><td class="patent-data-table-td ">Method for detecting polynucleotides encoding telomerase</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6921664">US6921664</a></td><td class="patent-data-table-td patent-date-value">Jan 18, 2002</td><td class="patent-data-table-td patent-date-value">Jul 26, 2005</td><td class="patent-data-table-td ">Regents Of The University Of Colorado</td><td class="patent-data-table-td ">Telomerase</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6927285">US6927285</a></td><td class="patent-data-table-td patent-date-value">Nov 12, 1999</td><td class="patent-data-table-td patent-date-value">Aug 9, 2005</td><td class="patent-data-table-td ">Geron Corporation</td><td class="patent-data-table-td ">Genes for human telomerase reverse transcriptase and telomerase variants</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7005262">US7005262</a></td><td class="patent-data-table-td patent-date-value">Jan 18, 2002</td><td class="patent-data-table-td patent-date-value">Feb 28, 2006</td><td class="patent-data-table-td ">Geron Corporation</td><td class="patent-data-table-td ">Methods for detecting nucleic acids encoding human telomerase reverse transcriptase</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7056513">US7056513</a></td><td class="patent-data-table-td patent-date-value">Apr 26, 2001</td><td class="patent-data-table-td patent-date-value">Jun 6, 2006</td><td class="patent-data-table-td ">Geron Corporation</td><td class="patent-data-table-td ">Telomerase</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7195911">US7195911</a></td><td class="patent-data-table-td patent-date-value">Jan 11, 2002</td><td class="patent-data-table-td patent-date-value">Mar 27, 2007</td><td class="patent-data-table-td ">Geron Corporation</td><td class="patent-data-table-td ">Mammalian cells that have increased proliferative capacity</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7262174">US7262174</a></td><td class="patent-data-table-td patent-date-value">May 9, 2002</td><td class="patent-data-table-td patent-date-value">Aug 28, 2007</td><td class="patent-data-table-td ">Geron Corporation</td><td class="patent-data-table-td ">Causing increased telomerase activity in cells at the wound site</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7262288">US7262288</a></td><td class="patent-data-table-td patent-date-value">Nov 22, 2000</td><td class="patent-data-table-td patent-date-value">Aug 28, 2007</td><td class="patent-data-table-td ">Geron Corporation</td><td class="patent-data-table-td ">Nucleic acids encoding human telomerase reverse transcriptase and related homologs</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7285639">US7285639</a></td><td class="patent-data-table-td patent-date-value">Jan 18, 2002</td><td class="patent-data-table-td patent-date-value">Oct 23, 2007</td><td class="patent-data-table-td ">Geron Corporation</td><td class="patent-data-table-td ">Antibody to telomerase reverse transcriptase</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7297488">US7297488</a></td><td class="patent-data-table-td patent-date-value">Aug 8, 2003</td><td class="patent-data-table-td patent-date-value">Nov 20, 2007</td><td class="patent-data-table-td ">Geron Corporation</td><td class="patent-data-table-td ">Identifying and testing antisense oligonucleotides that inhibit telomerase reverse transcriptase</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7413864">US7413864</a></td><td class="patent-data-table-td patent-date-value">Jun 24, 2004</td><td class="patent-data-table-td patent-date-value">Aug 19, 2008</td><td class="patent-data-table-td ">Geron Corporation</td><td class="patent-data-table-td ">A nucleic acid expression vector encodes the catalytic subunit of human telomerase reverse transcriptase (hTRT); immunostimulants; anticarcinogenic agents; gene therapy</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7517971">US7517971</a></td><td class="patent-data-table-td patent-date-value">Nov 22, 2000</td><td class="patent-data-table-td patent-date-value">Apr 14, 2009</td><td class="patent-data-table-td ">Geron Corporation</td><td class="patent-data-table-td ">recombinant polynucleotide encoding activity-modified polypeptides, useful for medical diagnosis and prognosis, changing the proliferative capacity of cells and organisms, drug screening and treatment of diseases such as cancers</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7560437">US7560437</a></td><td class="patent-data-table-td patent-date-value">Jan 11, 2002</td><td class="patent-data-table-td patent-date-value">Jul 14, 2009</td><td class="patent-data-table-td ">Geron Corporation</td><td class="patent-data-table-td ">Therapeutic agent for use in the diagnosis and treatment of cancer and cell proliferative disorders</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7585622">US7585622</a></td><td class="patent-data-table-td patent-date-value">Nov 2, 1999</td><td class="patent-data-table-td patent-date-value">Sep 8, 2009</td><td class="patent-data-table-td ">Geron Corporation</td><td class="patent-data-table-td ">Increasing the proliferative capacity of cells using telomerase reverse transcriptase</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7622549">US7622549</a></td><td class="patent-data-table-td patent-date-value">Jun 24, 2004</td><td class="patent-data-table-td patent-date-value">Nov 24, 2009</td><td class="patent-data-table-td ">Geron Corporation</td><td class="patent-data-table-td ">Human telomerase reverse transcriptase polypeptides</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7750121">US7750121</a></td><td class="patent-data-table-td patent-date-value">Aug 17, 2005</td><td class="patent-data-table-td patent-date-value">Jul 6, 2010</td><td class="patent-data-table-td ">Geron Corporation</td><td class="patent-data-table-td ">A recombinant human telomerase reverse transcriptase (hTRT) protein; drug screening for antiproliferative agents that inhibit hTRT activity; cancer diagnosis and prognosis; anticarcinogenic agents</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7879609">US7879609</a></td><td class="patent-data-table-td patent-date-value">Mar 26, 2007</td><td class="patent-data-table-td patent-date-value">Feb 1, 2011</td><td class="patent-data-table-td ">Geron Corporation</td><td class="patent-data-table-td ">Regulatory segments of the human gene for telomerase reverse transcriptase</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8003375">US8003375</a></td><td class="patent-data-table-td patent-date-value">Nov 19, 2009</td><td class="patent-data-table-td patent-date-value">Aug 23, 2011</td><td class="patent-data-table-td ">Exonhit Therapeutics S.A.</td><td class="patent-data-table-td ">Qualitative differential screening</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8222392">US8222392</a></td><td class="patent-data-table-td patent-date-value">Aug 20, 2007</td><td class="patent-data-table-td patent-date-value">Jul 17, 2012</td><td class="patent-data-table-td ">Geron Corporation</td><td class="patent-data-table-td ">Kit for detection of telomerase reverse transcriptase nucleic acids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8236774">US8236774</a></td><td class="patent-data-table-td patent-date-value">Aug 21, 2009</td><td class="patent-data-table-td patent-date-value">Aug 7, 2012</td><td class="patent-data-table-td ">Geron Corporation</td><td class="patent-data-table-td ">Human telomerase catalytic subunit</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8658778">US8658778</a></td><td class="patent-data-table-td patent-date-value">Mar 9, 2006</td><td class="patent-data-table-td patent-date-value">Feb 25, 2014</td><td class="patent-data-table-td ">Board Of Regents, The University Of Texas System</td><td class="patent-data-table-td ">hTMC promoter and vectors for the tumor-selective and high-efficient expression of cancer therapeutic genes</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8709995">US8709995</a></td><td class="patent-data-table-td patent-date-value">Aug 20, 2007</td><td class="patent-data-table-td patent-date-value">Apr 29, 2014</td><td class="patent-data-table-td ">Geron Corporation</td><td class="patent-data-table-td ">Method for eliciting an immune response to human telomerase reverse transcriptase</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2011005930A1?cl=en">WO2011005930A1</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jul 8, 2010</td><td class="patent-data-table-td patent-date-value">Jan 13, 2011</td><td class="patent-data-table-td ">Centocor Ortho Biotech Inc.</td><td class="patent-data-table-td ">Cardiac tissue-derived cells</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="classifications"></a><div class="patent-section-header"><span class="patent-section-title">Classifications</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th"> </th><th class="patent-data-table-th"> </th></tr></thead><tr><td class="patent-data-table-td ">U.S. Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=JOlhBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S006130">435/6.13</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JOlhBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S029000">435/29</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JOlhBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S325000">435/325</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JOlhBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S194000">435/194</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JOlhBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S069200">435/69.2</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JOlhBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S023200">536/23.2</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JOlhBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S070100">435/70.1</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JOlhBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S455000">435/455</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JOlhBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S006180">435/6.18</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JOlhBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S006150">435/6.15</a></span></td></tr><tr><td class="patent-data-table-td ">International Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=JOlhBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=G01N0033480000">G01N33/48</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JOlhBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=A61P0035000000">A61P35/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JOlhBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12Q0001680000">C12Q1/68</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JOlhBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C07K0016400000">C07K16/40</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JOlhBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0015090000">C12N15/09</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JOlhBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=A61K0038450000">A61K38/45</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JOlhBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=G01N0033150000">G01N33/15</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JOlhBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C07K0005100000">C07K5/10</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JOlhBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0015120000">C12N15/12</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JOlhBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C07K0005117000">C07K5/117</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JOlhBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=A61K0038550000">A61K38/55</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JOlhBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0015850000">C12N15/85</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JOlhBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12R0001840000">C12R1/84</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JOlhBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=A61K0031700000">A61K31/70</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JOlhBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=G01N0033530000">G01N33/53</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JOlhBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=A61K0035120000">A61K35/12</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JOlhBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12P0021080000">C12P21/08</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JOlhBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=G01N0033566000">G01N33/566</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JOlhBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=A01K0067027000">A01K67/027</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JOlhBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=A61K0048000000">A61K48/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JOlhBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C07K0005107000">C07K5/107</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JOlhBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0001190000">C12N1/19</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JOlhBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0001210000">C12N1/21</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JOlhBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12R0001190000">C12R1/19</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JOlhBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=A61K0039000000">A61K39/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JOlhBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=A61K0039395000">A61K39/395</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JOlhBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C07K0014470000">C07K14/47</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JOlhBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=A61K0031708800">A61K31/7088</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JOlhBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C07K0007080000">C07K7/08</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JOlhBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=G01N0033573000">G01N33/573</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JOlhBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=G01N0033500000">G01N33/50</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JOlhBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0001150000">C12N1/15</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JOlhBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12R0001910000">C12R1/91</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JOlhBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=A61P0043000000">A61P43/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JOlhBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12Q0001020000">C12Q1/02</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JOlhBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0015540000">C12N15/54</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JOlhBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0005100000">C12N5/10</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JOlhBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C07K0007060000">C07K7/06</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JOlhBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=A61K0038000000">A61K38/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JOlhBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12Q0001480000">C12Q1/48</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JOlhBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0009120000">C12N9/12</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JOlhBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0015113000">C12N15/113</a></span></td></tr><tr><td class="patent-data-table-td ">Cooperative Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=JOlhBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=Y10S530/827">Y10S530/827</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JOlhBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=Y10S530/828">Y10S530/828</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JOlhBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=A61K39/00">A61K39/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JOlhBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N9/1276">C12N9/1276</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JOlhBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=A61K38/00">A61K38/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JOlhBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N2310/15">C12N2310/15</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JOlhBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N15/113">C12N15/113</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JOlhBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=A01K2217/05">A01K2217/05</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JOlhBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N15/1137">C12N15/1137</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JOlhBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12Y207/07049">C12Y207/07049</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JOlhBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N2310/315">C12N2310/315</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JOlhBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C07K2319/00">C07K2319/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JOlhBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=A61K48/00">A61K48/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JOlhBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N9/1241">C12N9/1241</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=JOlhBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C07K2319/003">C07K2319/003</a></span></td></tr><tr><td class="patent-data-table-td ">European Classification</td><td class="patent-data-table-td "><span class="nested-value">C12Y207/07049</span>, <span class="nested-value">C12N9/12J49</span>, <span class="nested-value">C12N9/12J</span>, <span class="nested-value">C12N15/113</span>, <span class="nested-value">C12N15/113D</span></td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="legal-events"></a><div class="patent-section-header"><span class="patent-section-title">Legal Events</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Date</th><th class="patent-data-table-th">Code</th><th class="patent-data-table-th">Event</th><th class="patent-data-table-th">Description</th></tr></thead><tr><td class="patent-data-table-td patent-date-value">Feb 10, 2011</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">8</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Jan 12, 2010</td><td class="patent-data-table-td ">B1</td><td class="patent-data-table-td ">Reexamination certificate first reexamination</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">CLAIM 1 IS DETERMINED TO BE PATENTABLE AS AMENDED. CLAIMS 2, AND 37-38, DEPENDENT ON AN AMENDED CLAIM, ARE DETERMINED TO BE PATENTABLE. CLAIMS 3-36, AND 39 WERE NOT REEXAMINED.</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Jan 13, 2009</td><td class="patent-data-table-td ">RR</td><td class="patent-data-table-td ">Request for reexamination filed</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20081113</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Feb 9, 2007</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">4</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Sep 6, 2005</td><td class="patent-data-table-td ">CC</td><td class="patent-data-table-td ">Certificate of correction</td><td class="patent-data-table-td "></td></tr><tr><td class="patent-data-table-td patent-date-value">Jun 7, 2005</td><td class="patent-data-table-td ">CC</td><td class="patent-data-table-td ">Certificate of correction</td><td class="patent-data-table-td "></td></tr></table><div class="patent-section-footer"></div></div><div class="modal-dialog" id="patent-images-lightbox"><div class="patent-lightbox-controls"><div class="patent-lightbox-rotate-controls"><div class="patent-lightbox-rotation-text">Rotate</div><div class="rotate-icon rotate-ccw-icon"></div><div class="rotate-icon rotate-cw-icon"></div></div><div class="patent-lightbox-index-counter"></div><a class="patent-lightbox-fullsize-link" target="_blank">Original Image</a><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_right.png" alt="Next page"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_left.png" alt="Previous page"width="21" height="21" /></div></div><div class="modal-dialog-content"><div class="patent-lightbox-image-holder"><div class="patent-lightbox-placeholder"></div></div></div></div><script>_OC_initPatentsAtb({image_not_available_html: " Image not available"});</script></div></div></div></td></tr></table><script>(function() {var href = window.location.href;if (href.indexOf('?') !== -1) {var parameters = href.split('?')[1].split('&');for (var i = 0; i < parameters.length; i++) {var param = parameters[i].split('=');if (param[0] == 'focus') {var elem = document.getElementById(param[1]);if (elem) {elem.focus();}}}}})();</script><script>_OC_addFlags({LockSrc:"/books/javascript/lock_8a2b04e7bf975d5171d8e4c0b6365c7a.js", Host:"http://www.google.com/", IsBooksRentalEnabled:1, IsWebstoreDisplayCaseEnabled:1, IsObfuscationEnabled:1, IsBrowsingHistoryEnabled:1, IsWebReaderSvgEnabled:0, IsGeoLayerEnabled:1, IsImageModeNotesEnabled:1, IsCopyMenuItemEnabled:1, IsGiftingEnabled:0, IsWebReaderUniversalPaginatorEnabled:0, IsOfflineBubbleEnabled:1, IsReaderEnabledForPlayRequests:1, IsFutureOnSaleVolumesEnabled:1, IsOfflineRestrictedCopyEnabled:1, IsBooksUnifiedLeftNavEnabled:1, IsRestrictedCopyEnabled:1, IsZipitFolderCollectionEnabled:1, IsEndOfSampleRecommendationsEnabled:1, IsRatingsOnBookcardsEnabled:1, IsAdsDisabled:0, IsIframePageDisplayEnabled:0, IsEmbeddedMediaEnabled:1, IsImageModeAnnotationsEnabled:1, IsMyLibraryGooglePlusEnabled:1, IsImagePageProviderEnabled:0, IsBookcardListPriceSmall:0, IsInternalUser:0, IsBooksShareButtonEnabled:0, IsPreOrdersEnabled:0, IsDisabledRandomBookshelves:0, WebstoreDisplayCasePosition:3});_OC_Run({"enable_p13n":false,"add_vol_to_collection_base_url":"http://www.google.com/patents?op=add\u0026sig=ACfU3U2DVhR5i5IWwQZXBU4rx3gtqJnzoA\u0026id=JOlhBAABERAJ","remove_vol_from_collection_base_url":"http://www.google.com/patents?op=remove\u0026sig=ACfU3U3wOO12gLUDW7ZjiS94PhYUCXV4kQ\u0026id=JOlhBAABERAJ","logged_in":false,"p13n_save_user_settings_url":"http://www.google.com/patents?op=edit_user_settings\u0026sig=ACfU3U0mPPKIECpNEhfehJ0je2ye0Unt0w","is_cobrand":false,"sign_in_url":"https://www.google.com/accounts/Login?service=\u0026continue=http://www.google.com/patents%3Fhl%3Den\u0026hl=en","is_play_enabled":true}, {"volume_id":"","is_ebook":true,"volumeresult":{"has_flowing_text":false,"has_scanned_text":true,"can_download_pdf":false,"can_download_epub":false,"is_pdf_drm_enabled":false,"is_epub_drm_enabled":false,"download_pdf_url":"http://www.google.com/patents/download/Cells_immortalized_with_telomerase_rever.pdf?id=JOlhBAABERAJ\u0026output=pdf\u0026sig=ACfU3U2lM34vKzjlqOeOsxbTqcGn35Av6w"},"sample_url":"http://www.google.com/patents/reader?id=JOlhBAABERAJ\u0026printsec=frontcover\u0026output=reader\u0026source=gbs_atb_hover","is_browsable":true,"is_public_domain":true}, {});</script><div id="footer_table" style="font-size:83%;text-align:center;position:relative;top:20px;height:4.5em;margin-top:2em"><div style="margin-bottom:8px"><a href=http://www.google.com/><nobr>Google&nbsp;Home</nobr></a> - <a href=//www.google.com/patents/sitemap/><nobr>Sitemap</nobr></a> - <a href=http://www.google.com/googlebooks/uspto.html><nobr>USPTO Bulk Downloads</nobr></a> - <a href=/intl/en/privacy/><nobr>Privacy Policy</nobr></a> - <a href=/intl/en/policies/terms/><nobr>Terms of Service</nobr></a> - <a href=https://support.google.com/faqs/answer/2539193?hl=en><nobr>About Google Patents</nobr></a> - <a href="http://www.google.com/tools/feedback/intl/en/error.html" onclick="try{_OC_startFeedback({productId: '72792',locale: 'en'});return false;}catch(e){}"><nobr>Send Feedback</nobr></a></div><span>Data provided by IFI CLAIMS Patent Services</span><br><span >&copy;2012 Google</span></div> <script type="text/javascript">var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));</script><script type="text/javascript">var pageTracker = _gat._getTracker("UA-27188110-1");pageTracker._setCookiePath("/patents/");pageTracker._trackPageview();</script> </body></html>