<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN""http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html><head><title>Patent US6635418 - Assay methods for nucleic acid in a sample - Google Patents</title><script>(function(){(function(){function e(a){this.t={};this.tick=function(a,c,b){var d=void 0!=b?b:(new Date).getTime();this.t[a]=[d,c];if(void 0==b)try{window.console.timeStamp("CSI/"+a)}catch(e){}};this.tick("start",null,a)}var a;window.performance&&(a=window.performance.timing);var f=a?new e(a.responseStart):new e;window.jstiming={Timer:e,load:f};if(a){var c=a.navigationStart,d=a.responseStart;0<c&&d>=c&&(window.jstiming.srt=d-c)}if(a){var b=window.jstiming.load;0<c&&d>=c&&(b.tick("_wtsrt",void 0,c),b.tick("wtsrt_",
"_wtsrt",d),b.tick("tbsd_","wtsrt_"))}try{a=null,window.chrome&&window.chrome.csi&&(a=Math.floor(window.chrome.csi().pageT),b&&0<c&&(b.tick("_tbnd",void 0,window.chrome.csi().startE),b.tick("tbnd_","_tbnd",c))),null==a&&window.gtbExternal&&(a=window.gtbExternal.pageT()),null==a&&window.external&&(a=window.external.pageT,b&&0<c&&(b.tick("_tbnd",void 0,window.external.startE),b.tick("tbnd_","_tbnd",c))),a&&(window.jstiming.pt=a)}catch(g){}})();})();
</script><link rel="stylesheet" href="/patents/css/_8a2b04e7bf975d5171d8e4c0b6365c7a/kl_intl_patents_bundle.css" type="text/css" /><script src="/books/javascript/atb_8a2b04e7bf975d5171d8e4c0b6365c7a__en.js"></script><script>function googleTranslateElementInit() {new google.translate.TranslateElement({pageLanguage: "en",gaTrack: true,gaId: "UA-27188110-1",multilanguagePage: true});}</script><script src="//translate.google.com/translate_a/element.js?cb=googleTranslateElementInit"></script><meta name="DC.type" content="Patent"><meta name="DC.title" content="Assay methods for nucleic acid in a sample"><meta name="DC.contributor" content="Jeffrey A. Heroux" scheme="inventor"><meta name="DC.contributor" content="Marta L. Corcoran" scheme="inventor"><meta name="DC.contributor" content="Savitha M. Rao" scheme="inventor"><meta name="DC.contributor" content="Igen International, Inc." scheme="assignee"><meta name="DC.date" content="1998-2-13" scheme="dateSubmitted"><meta name="DC.description" content="The present invention relates to a method for the determination of the presence and amount of DNA in a sample. The method is based on the use of a nucleic acid template dependent enzyme in combination with a random primer to generate an enzymatic product which incorporates a binding species and a detectable species covalently linked."><meta name="DC.date" content="2003-10-21" scheme="issued"><meta name="DC.relation" content="US:5043272" scheme="references"><meta name="DC.relation" content="US:5364759" scheme="references"><meta name="DC.relation" content="US:5413906" scheme="references"><meta name="DC.relation" content="US:5599662" scheme="references"><meta name="DC.relation" content="US:6096499" scheme="references"><meta name="citation_reference" content="Briggs, Jonathan, et al, Anal. Chem., 1991, 63, 850-859, &quot;Quantitation of DNA and Protein Impurities in Biopharmaceuticals&quot;."><meta name="citation_reference" content="Feinberg, Andrew P., et al, Analytical Biochemistry, 132, 6-13 (1983), &quot;A Technique for Radiolabeling DNA Restriction Endonuclease Fragments to High Specific Activity&quot;."><meta name="citation_reference" content="Merrick et al. Threshold, a complete system for quantitative analysis of total DNA, protein impurities and relevant proteins, 1992, Biotech Forum Europe, vol. 9(6), pp. 398-403.*"><meta name="citation_reference" content="Wu et al. The ligation amplification reaction (LAR)-amplification of specific DNA sequences using sequential rounds of template-dependent ligation. Genomics , vol. 4, p. 560-569.*"><meta name="citation_reference" content="Wu et al. The ligation amplification reaction (LAR)—amplification of specific DNA sequences using sequential rounds of template-dependent ligation. Genomics , vol. 4, p. 560-569.*"><meta name="citation_patent_number" content="US:6635418"><meta name="citation_patent_application_number" content="US:09/023,483"><link rel="canonical" href="http://www.google.com/patents/US6635418"/><meta property="og:url" content="http://www.google.com/patents/US6635418"/><meta name="title" content="Patent US6635418 - Assay methods for nucleic acid in a sample"/><meta name="description" content="The present invention relates to a method for the determination of the presence and amount of DNA in a sample. The method is based on the use of a nucleic acid template dependent enzyme in combination with a random primer to generate an enzymatic product which incorporates a binding species and a detectable species covalently linked."/><meta property="og:title" content="Patent US6635418 - Assay methods for nucleic acid in a sample"/><meta property="og:type" content="book"/><meta property="og:site_name" content="Google Books"/><meta property="og:image" content="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><link rel="image_src" href="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><script>(function(){try{var aa=function(a,b,c,d){d=d||{};d._sn=["cfg",b,c].join(".");window.gbar.logger.ml(a,d)};var g=window.gbar=window.gbar||{},l=window.gbar.i=window.gbar.i||{},m={},n;function _tvn(a,b){var c=parseInt(a,10);return isNaN(c)?b:c}function _tvf(a,b){var c=parseFloat(a);return isNaN(c)?b:c}function _tvv(a){return!!a}function p(a,b,c){(c||g)[a]=b}g.bv={n:_tvn("2",0),r:"",f:".67.",e:"0",m:_tvn("0",1)};
function q(a,b,c){var d="on"+b;if(a.addEventListener)a.addEventListener(b,c,!1);else if(a.attachEvent)a.attachEvent(d,c);else{var f=a[d];a[d]=function(){var a=f.apply(this,arguments),b=c.apply(this,arguments);return void 0==a?b:void 0==b?a:b&&a}}}var s=function(a){return function(){return g.bv.m==a}},ba=s(1),ca=s(2);p("sb",ba);p("kn",ca);l.a=_tvv;l.b=_tvf;l.c=_tvn;l.i=aa;var da=window.gbar.i.i;var t,u,v,w;function ea(a){v=a}function fa(a){var b;if(b=v&&window.encodeURIComponent)b=a.href,b=!b.match(/^http[s]?:\/\/accounts\.google\.[^/]*\/ClearSID/i)&&!b.match(/^http[s]?:\/\/[^/]*\/accounts\/ClearSID/i);if(b=b&&encodeURIComponent(v()))a.href=a.href.replace(/([?&]continue=)[^&]*/,"$1"+b)}function ga(a){window.gApplication&&(a.href=window.gApplication.getTabUrl(a.href))}
function ha(a){var b=document.forms[0].q,c=window.encodeURIComponent&&b&&b.value,b=b&&b.placeholder;c&&c!=b&&(a.href=a.href.replace(/([?&])q=[^&]*|$/,function(a,b){return(b||"&")+"q="+encodeURIComponent(c)}))}n=l.a("")?ga:ha;
function x(a,b,c,d,f,e){var h=document.getElementById(a);if(h){var k=h.style;k.left=d?"auto":b+"px";k.right=d?b+"px":"auto";k.top=c+"px";k.visibility=u?"hidden":"visible";f&&e?(k.width=f+"px",k.height=e+"px"):(x(t,b,c,d,h.offsetWidth,h.offsetHeight),u=u?"":a)}}
var y=[],ia=function(a,b){y.push(b)},ja=function(a){a=a||window.event;var b=a.target||a.srcElement;a.cancelBubble=!0;null==t&&(a=document.createElement(Array.every||window.createPopup?"iframe":"div"),a.frameBorder="0",t=a.id="gbs",a.src="javascript:''",b.parentNode.appendChild(a),q(document,"click",z));var c=b,b=0;"gb3"!=c.className&&(c=c.parentNode);a=c.getAttribute("aria-owns")||"gbi";var d=c.offsetWidth,f=20<c.offsetTop?46:24;document.getElementById("tphdr")&&(f-=3);var e=!1;do b+=c.offsetLeft||
0;while(c=c.offsetParent);var c=(document.documentElement.clientWidth||document.body.clientWidth)-b-d,h,d=document.body,k=document.defaultView;k&&k.getComputedStyle?(d=k.getComputedStyle(d,""))&&(h=d.direction):h=d.currentStyle?d.currentStyle.direction:d.style.direction;h="rtl"==h;if("gbi"==a){for(d=0;k=y[d++];)k();A(null,window.navExtra);h&&(b=c,e=!0)}else h||(b=c,e=!0);u!=a&&z();x(a,b,f,e)},z=function(){u&&x(u,0,0)},A=function(a,b){var c,d=document.getElementById("gbi"),f=a;f||(f=d.firstChild);
for(;b&&(c=b.pop());){var e=d,h=c,k=f;w||(w="gb2");e.insertBefore(h,k).className=w}},ka=function(a,b,c){if((b=document.getElementById(b))&&a){a.className="gb4";var d=document.createElement("span");d.appendChild(a);d.appendChild(document.createTextNode(" | "));d.id=c;b.appendChild(d)}},la=function(){return document.getElementById("gb_70")},ma=function(){return!!u};p("qs",n);p("setContinueCb",ea);p("pc",fa);p("tg",ja);p("close",z);p("addLink",ka);p("almm",A);p("si",la);p("adh",ia);p("op",ma);var B=function(){},C=function(){},F=function(a){var b=new Image,c=D;b.onerror=b.onload=b.onabort=function(){try{delete E[c]}catch(a){}};E[c]=b;b.src=a;D=c+1},E=[],D=0;p("logger",{il:C,ml:B,log:F});var G=window.gbar.logger;var H={},na={},I=[],oa=l.b("0.1",.1),pa=l.a("1",!0),qa=function(a,b){I.push([a,b])},ra=function(a,b){H[a]=b},sa=function(a){return a in H},J={},K=function(a,b){J[a]||(J[a]=[]);J[a].push(b)},ta=function(a){K("m",a)},L=function(a,b){var c=document.createElement("script");c.src=a;c.async=pa;Math.random()<oa&&(c.onerror=function(){c.onerror=null;B(Error("Bundle load failed: name="+(b||"UNK")+" url="+a))});(document.getElementById("xjsc")||document.getElementsByTagName("body")[0]||
document.getElementsByTagName("head")[0]).appendChild(c)},N=function(a){for(var b=0,c;(c=I[b])&&c[0]!=a;++b);!c||c[1].l||c[1].s||(c[1].s=!0,M(2,a),c[1].url&&L(c[1].url,a),c[1].libs&&m.d&&m.d(c[1].libs))},O=function(a){K("gc",a)},P=null,ua=function(a){P=a},M=function(a,b,c){if(P){a={t:a,b:b};if(c)for(var d in c)a[d]=c[d];try{P(a)}catch(f){}}};p("mdc",H);p("mdi",na);p("bnc",I);p("qGC",O);p("qm",ta);p("qd",J);p("lb",N);p("mcf",ra);p("bcf",qa);p("aq",K);p("mdd","");p("has",sa);
p("trh",ua);p("tev",M);var Q=l.b("0.1",.001),R=0;
function _mlToken(a,b){try{if(1>R){R++;var c,d=a,f=b||{},e=encodeURIComponent,h=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&jexpid=",e("17483"),"&srcpg=",e("prop=22"),"&jsr=",Math.round(1/Q),"&ogev=",e("5antU8D-KcrBsQSz1YCYCQ"),"&ogf=",g.bv.f,"&ogrp=",e("1"),"&ogv=",e("1407723702.0"),"&oggv="+e("es_plusone_gc_20140723.0_p0"),"&ogd=",e("com"),"&ogc=",e("NOR"),"&ogl=",e("en")];f._sn&&(f._sn="og."+
f._sn);for(var k in f)h.push("&"),h.push(e(k)),h.push("="),h.push(e(f[k]));h.push("&emsg=");h.push(e(d.name+":"+d.message));var r=h.join("");S(r)&&(r=r.substr(0,2E3));c=r;var Aa=window.gbar.logger._aem(a,c);F(Aa)}}catch(Na){}}var S=function(a){return 2E3<=a.length},va=function(a,b){return b};function T(a){B=a;p("_itl",S,G);p("_aem",va,G);p("ml",B,G);a={};H.er=a}l.a("")?T(function(a){throw a;}):l.a("1")&&Math.random()<Q&&T(_mlToken);I.push(["m",{url:"//ssl.gstatic.com/gb/js/scm_7385cc5883250b43a39405734c1bea59.js"}]);g.mcf("c",{});g.sg={c:""};if(l.a("1")){var wa=l.a("");I.push(["gc",{auto:wa,url:"//ssl.gstatic.com/gb/js/abc/gci_91f30755d6a6b787dcc2a4062e6e9824.js",libs:"googleapis.client:plusone:gapi.iframes"}]);var xa={version:"gci_91f30755d6a6b787dcc2a4062e6e9824.js",index:"",lang:"en"};H.gc=xa;var U=function(a){window.googleapis&&window.iframes?a&&a():(a&&O(a),N("gc"))};p("lGC",U);l.a("1")&&p("lPWF",U)};window.__PVT="";if(l.a("1")&&l.a("1")){var V=function(a){U(function(){K("pw",a);N("pw")})};p("lPW",V);I.push(["pw",{url:"//ssl.gstatic.com/gb/js/abc/pwm_45f73e4df07a0e388b0fa1f3d30e7280.js"}]);var W=[],ya=function(a){W[0]=a},za=function(a,b){var c=b||{};c._sn="pw";B(a,c)},Ba={signed:W,elog:za,base:"https://plusone.google.com/u/0",loadTime:(new Date).getTime()};H.pw=Ba;var X=function(a,b){for(var c=b.split("."),d=function(){var b=arguments;a(function(){for(var a=g,d=0,e=c.length-1;d<e;++d)a=a[c[d]];a[c[d]].apply(a,b)})},f=g,e=0,h=c.length-1;e<h;++e)f=
f[c[e]]=f[c[e]]||{};return f[c[e]]=d};X(V,"pw.clk");X(V,"pw.hvr");p("su",ya,g.pw)};function Ca(){function a(){for(var b;(b=e[h++])&&"m"!=b[0]&&!b[1].auto;);b&&(M(2,b[0]),b[1].url&&L(b[1].url,b[0]),b[1].libs&&m.d&&m.d(b[1].libs));h<e.length&&setTimeout(a,0)}function b(){0<f--?setTimeout(b,0):a()}var c=l.a("1"),d=l.a(""),f=3,e=I,h=0,k=window.gbarOnReady;if(k)try{k()}catch(r){da(r,"ml","or")}d?p("ldb",a):c?q(window,"load",b):b()}p("rdl",Ca);var Da={D:1,H:2,da:3,p:4,W:5,M:6,F:7,g:8,ha:9,U:10,L:11,T:12,S:13,N:14,Q:15,P:16,fa:17,w:18,O:19,ga:20,ea:21,u:22,G:23,ja:24,ka:25,ia:26,A:27,j:28,o:29,k:30,ca:31,Z:32,$:33,J:34,K:35,ba:36,aa:37,Y:38,B:39,R:40,v:41,X:42,V:43,h:48,C:49,I:500},Y=[1,2,3,4,5,6,9,10,11,13,14,28,29,30,34,35,37,38,39,40,41,42,43,48,49,500];var Z=l.b("0.001",1E-4),Ea=l.b("1",1),Fa=!1,Ga=!1;if(l.a("1")){var Ha=Math.random();Ha<=Z&&(Fa=!0);Ha<=Ea&&(Ga=!0)}var Ia=Da,$=null;function Ja(){var a=0,b=function(b,d){l.a(d)&&(a|=b)};b(1,"");b(2,"");b(4,"");b(8,"");return a}
function Ka(a,b){var c=Z,d=Fa,f;f=a;if(!$){$={};for(var e=0;e<Y.length;e++){var h=Y[e];$[h]=!0}}if(f=!!$[f])c=Ea,d=Ga;if(d){d=encodeURIComponent;g.rp?(f=g.rp(),f="-1"!=f?f:"1"):f="1";c=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&oge=",a,"&ogex=",d("17483"),"&ogev=",d("5antU8D-KcrBsQSz1YCYCQ"),"&ogf=",g.bv.f,"&ogp=",d("22"),"&ogrp=",d(f),"&ogsr=",Math.round(1/c),"&ogv=",d("1407723702.0"),"&oggv="+
d("es_plusone_gc_20140723.0_p0"),"&ogd=",d("com"),"&ogl=",d("en"),"&ogc=",d("NOR"),"&ogus=",Ja()];if(b){"ogw"in b&&(c.push("&ogw="+b.ogw),delete b.ogw);var k;f=b;e=[];for(k in f)0!=e.length&&e.push(","),e.push(La(k)),e.push("."),e.push(La(f[k]));k=e.join("");""!=k&&(c.push("&ogad="),c.push(d(k)))}F(c.join(""))}}function La(a){"number"==typeof a&&(a+="");return"string"==typeof a?a.replace(".","%2E").replace(",","%2C"):a}C=Ka;p("il",C,G);var Ma={};H.il=Ma;setTimeout(function(){C(Ia.g)},0);}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var b=window.gbar.i.i;var c=window.gbar;var f=function(d){try{var a=document.getElementById("gbom");a&&d.appendChild(a.cloneNode(!0))}catch(e){b(e,"omas","aomc")}};c.aomc=f;}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var a=window.gbar;a.mcf("pm",{p:""});}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{window.gbar.rdl();}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
if (window['_OC_timingAction']) {window['_OC_timingAction']('patents_refpage');}</script><style>#gbar,#guser{font-size:13px;padding-top:1px !important;}#gbar{float:left;height:22px}#guser{padding-bottom:7px !important;text-align:right}.gbh,.gbd{border-top:1px solid #c9d7f1;font-size:1px}.gbh{height:0;position:absolute;top:24px;width:100%}#gbs,.gbm{background:#fff;left:0;position:absolute;text-align:left;visibility:hidden;z-index:1000}.gbm{border:1px solid;border-color:#c9d7f1 #36c #36c #a2bae7;z-index:1001}.gb1{margin-right:.5em}.gb1,.gb3{zoom:1}.gb2{display:block;padding:.2em .5em}.gb2,.gb3{text-decoration:none !important;border-bottom:none}a.gb1,a.gb4{text-decoration:underline !important}a.gb1,a.gb2,a.gb3,a.gb4{color:#00c !important}.gbi .gb3,.gbi .gb2,.gbi .gb4{color:#dd8e27 !important}.gbf .gb3,.gbf .gb2,.gbf .gb4{color:#900 !important}a.gb2:hover{background:#36c;color:#fff !important}#gbar .gbz0l{color:#000 !important;cursor:default;font-weight:bold;text-decoration:none !important}
#gbar { padding:.3em .6em !important;}</style></head><body  topmargin="3" marginheight="3"><div id=gbar><nobr><a onclick=gbar.qs(this);gbar.logger.il(1,{t:1}); class=gb1 id=gb_1 href="https://www.google.com/search?sa=N&tab=tw">Search</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:2}); class=gb1 id=gb_2 href="http://www.google.com/search?hl=en&tbm=isch&source=og&sa=N&tab=ti">Images</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:8}); class=gb1 id=gb_8 href="http://maps.google.com/maps?hl=en&sa=N&tab=tl">Maps</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:78}); class=gb1 id=gb_78 href="https://play.google.com/?hl=en&sa=N&tab=t8">Play</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:36}); class=gb1 id=gb_36 href="http://www.youtube.com/results?sa=N&tab=t1">YouTube</a> <a onclick=gbar.logger.il(1,{t:5}); class=gb1 id=gb_5 href="http://news.google.com/nwshp?hl=en&tab=tn">News</a> <a onclick=gbar.logger.il(1,{t:23}); class=gb1 id=gb_23 href="https://mail.google.com/mail/?tab=tm">Gmail</a> <a onclick=gbar.logger.il(1,{t:25}); class=gb1 id=gb_25 href="https://drive.google.com/?tab=to">Drive</a> <a class=gb3 href="http://www.google.com/intl/en/options/" onclick="this.blur();gbar.tg(event);return !1" aria-haspopup=true><u>More</u> <small>&#9660;</small></a><div class=gbm id=gbi><a onclick=gbar.logger.il(1,{t:24}); class=gb2 id=gb_24 href="https://www.google.com/calendar?tab=tc">Calendar</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:51}); class=gb2 id=gb_51 href="http://translate.google.com/?hl=en&sa=N&tab=tT">Translate</a><a onclick=gbar.logger.il(1,{t:17}); class=gb2 id=gb_17 href="http://www.google.com/mobile/?hl=en&tab=tD">Mobile</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:10}); class=gb2 id=gb_10 href="http://www.google.com/search?hl=en&tbo=u&tbm=bks&source=og&sa=N&tab=tp">Books</a><a onclick=gbar.logger.il(1,{t:212}); class=gb2 id=gb_212 href="https://wallet.google.com/manage/?tab=ta">Wallet</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:6}); class=gb2 id=gb_6 href="http://www.google.com/search?hl=en&tbo=u&tbm=shop&source=og&sa=N&tab=tf">Shopping</a><a onclick=gbar.logger.il(1,{t:30}); class=gb2 id=gb_30 href="http://www.blogger.com/?tab=tj">Blogger</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:27}); class=gb2 id=gb_27 href="http://www.google.com/finance?sa=N&tab=te">Finance</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:31}); class=gb2 id=gb_31 href="https://plus.google.com/photos?sa=N&tab=tq">Photos</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:12}); class=gb2 id=gb_12 href="http://www.google.com/search?hl=en&tbo=u&tbm=vid&source=og&sa=N&tab=tv">Videos</a><div class=gb2><div class=gbd></div></div><a onclick=gbar.logger.il(1,{t:66}); href="http://www.google.com/intl/en/options/" class=gb2>Even more &raquo;</a></div></nobr></div><div id=guser width=100%><nobr><span id=gbn class=gbi></span><span id=gbf class=gbf></span><span id=gbe></span><a target=_top id=gb_70 href="https://www.google.com/accounts/Login?service=&continue=http://www.google.com/patents%3Fhl%3Den&hl=en" class=gb4>Sign in</a><div style="display: none"><div class=gbm id=gbd5 aria-owner=gbg5><div class=gbmc><ol id=gbom class=gbmcc></ol></div></div></div></nobr></div><div class=gbh style=left:0></div><div class=gbh style=right:0></div><div role="alert" style="position: absolute; left: 0; right: 0;"><a href="http://www.google.com/patents/us6635418?hl=en&amp;output=html_text" title="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."><img border="0" src="http://www.google.com/images/cleardot.gif"alt="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."></a></div><div id="guser"><nobr></nobr></div><div style="clear:both;"></div><div id="gb-top-search-box" class="gb-top-search-box-small gb-reset"><table><tr><td class="logo"><a href="http://www.google.com/patents" class="logo-link"><img class="logo-img" src="/intl/en/images/logos/google_logo_41.png" alt="Go to Google Books Home" height="41"/></a></td><td><form action="http://www.google.com/search" name="f" id="vheadf" method="get"><span id="hf"></span><input type="hidden" name="tbm" value="pts"/><input type="hidden" name="tbo" value="1"/><input type="hidden" name="hl" value="en"/><table><tr><td><div class="inputs"><table><tr><td><div class="text-input"><input type="text" name="q" id="vheadq" class="text" maxlength="2048" size="31" value="" title="Search Patents" accesskey="s" autocomplete="off"/><script>window._OC_autoDir &&window._OC_autoDir('vheadq', 'tia-vheadq');</script></div></td><td><div class="submit-input"><input name="btnG" class="submit" type="submit" value=""/></div></td></tr></table></div></td><td class="col-ext-links"><div class="ext-links"><a href="http://www.google.com/advanced_patent_search">&lt;nobr&gt;Advanced Patent Search&lt;/nobr&gt;</a></div></td></tr></table></form></td></tr></table></div><div class="kd-appbar"><h2 class="kd-appname"><a href="/patents">Patents</a></h2><div class="kd-buttonbar left" id="left-toolbar-buttons"><a id="appbar-write-review-link" href=""></a><a id="appbar-view-print-sample-link" href=""></a><a id="appbar-view-ebook-sample-link" href=""></a><a id="appbar-patents-prior-art-finder-link" href="https://www.google.com/patents/related/US6635418"></a><a id="appbar-patents-discuss-this-link" href="http://www.google.com/url?id=-rdiBAABERAJ&amp;q=http://patents.stackexchange.com/redirect/google-patents%3Fpatent%3DUS6635418&amp;usg=AFQjCNGoC5mFQGICAXLf_3DFXuHPXYTUgw" data-is-grant="true"></a><a id="appbar-read-patent-link" href="//docs.google.com/viewer?url=patentimages.storage.googleapis.com/pdfs/US6635418.pdf"></a><a id="appbar-download-pdf-link" href="//patentimages.storage.googleapis.com/pdfs/US6635418.pdf"></a><a class="appbar-application-grant-link" data-label="Application" href="/patents/US20010014446"></a><a class="appbar-application-grant-link" data-selected="true" data-label="Grant" href="/patents/US6635418"></a></div><div class="kd-buttonbar right" id="right-toolbar-buttons"></div></div><div id="books-microdata" itemscope=""itemtype="http://schema.org/Book"itemid="http://www.google.com/patents/US6635418" style="display:none"><span itemprop="description">The present invention relates to a method for the determination of the presence and amount of DNA in a sample. The method is based on the use of a nucleic acid template dependent enzyme in combination with a random primer to generate an enzymatic product which incorporates a binding species and a detectable...</span><span itemprop="url">http://www.google.com/patents/US6635418?utm_source=gb-gplus-share</span><span class="main-title" itemprop="name">Patent US6635418 - Assay methods for nucleic acid in a sample</span><img itemprop="image" src="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"alt="Patent US6635418 - Assay methods for nucleic acid in a sample" title="Patent US6635418 - Assay methods for nucleic acid in a sample"></div><div style="display: none"><ol id="ofe-gear-menu-contents" class="gbmcc"><li class="gbe gbmtc"><a class="gbmt goog-menuitem-content" id="" href="http://www.google.com/advanced_patent_search">Advanced Patent Search</a></li></ol></div><table id="viewport_table" cellpadding="0" style="clear:both" cellspacing="0"><tr><td id="viewport_td"><div class=vertical_module_list_row><div id=intl_patents class=about_content><div id=intl_patents_v><table class="patent-bibdata"><tr><td class="patent-bibdata-heading">Publication number</td><td class="single-patent-bibdata">US6635418 B2</td></tr><tr><td class="patent-bibdata-heading">Publication type</td><td class="single-patent-bibdata">Grant</td></tr><tr><td class="patent-bibdata-heading">Application number</td><td class="single-patent-bibdata">US 09/023,483</td></tr><tr><td class="patent-bibdata-heading">Publication date</td><td class="single-patent-bibdata">Oct 21, 2003</td></tr><tr><td class="patent-bibdata-heading">Filing date</td><td class="single-patent-bibdata">Feb 13, 1998</td></tr><tr><td class="patent-bibdata-heading">Priority date<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The priority date is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the date listed."></span></td><td class="single-patent-bibdata">Feb 13, 1998</td></tr><tr><td class="patent-bibdata-heading">Fee status<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The fee status is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the status or dates listed."></span></td><td class="single-patent-bibdata">Lapsed</td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Also published as</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/US20010014446">US20010014446</a></span></span></td></tr><tr class="patent-bibdata-list-row alternate-patent-number"><td class="patent-bibdata-heading">Publication number</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value">023483, </span><span class="patent-bibdata-value">09023483, </span><span class="patent-bibdata-value">US 6635418 B2, </span><span class="patent-bibdata-value">US 6635418B2, </span><span class="patent-bibdata-value">US-B2-6635418, </span><span class="patent-bibdata-value">US6635418 B2, </span><span class="patent-bibdata-value">US6635418B2</span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Inventors</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Jeffrey+A.+Heroux%22">Jeffrey A. Heroux</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Marta+L.+Corcoran%22">Marta L. Corcoran</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Savitha+M.+Rao%22">Savitha M. Rao</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Original Assignee</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=inassignee:%22Igen+International,+Inc.%22">Igen International, Inc.</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Export Citation</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/US6635418.bibtex">BiBTeX</a>, </span><span class="patent-bibdata-value"><a href="/patents/US6635418.enw">EndNote</a>, </span><span class="patent-bibdata-value"><a href="/patents/US6635418.ris">RefMan</a></span></span></td></tr><tr class="patent-internal-links"><td colspan=2><span class="patent-bibdata-value"><a href="#backward-citations">Patent Citations</a> (5),</span> <span class="patent-bibdata-value"><a href="#npl-citations">Non-Patent Citations</a> (5),</span> <span class="patent-bibdata-value"><a href="#forward-citations">Referenced by</a> (2),</span> <span class="patent-bibdata-value"><a href="#classifications">Classifications</a> (10),</span> <span class="patent-bibdata-value"><a href="#legal-events">Legal Events</a> (9)</span> </td></tr><tr><td colspan=2 class="patent-bibdata-external-link-spacer-top"></td></tr><tr class="patent-bibdata-external-link-spacer-bottom"></tr><tr><td colspan=2><span class="patent-bibdata-heading">External Links:&nbsp;</span><span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=-rdiBAABERAJ&q=http://patft.uspto.gov/netacgi/nph-Parser%3FSect2%3DPTO1%26Sect2%3DHITOFF%26p%3D1%26u%3D/netahtml/PTO/search-bool.html%26r%3D1%26f%3DG%26l%3D50%26d%3DPALL%26RefSrch%3Dyes%26Query%3DPN/6635418&usg=AFQjCNFe3A4DxlXDOsD8igld69MHE9FJZQ">USPTO</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=-rdiBAABERAJ&q=http://assignments.uspto.gov/assignments/q%3Fdb%3Dpat%26pat%3D6635418&usg=AFQjCNGkfunJGSZwliQ8PZCGdav_ke2upg">USPTO Assignment</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=-rdiBAABERAJ&q=http://worldwide.espacenet.com/publicationDetails/biblio%3FCC%3DUS%26NR%3D6635418B2%26KC%3DB2%26FT%3DD&usg=AFQjCNEEV8qlwoCWCtBe6JvbP66simsHtQ">Espacenet</a></span></span></td></tr><tr class="patent-bibdata-group-spacer"></tr></table><div class="number-and-title"><span class="patent-title"><invention-title mxw-id="PT55166445" lang="EN" load-source="patent-office">Assay methods for nucleic acid in a sample</invention-title></span><br><span class="patent-number">US 6635418 B2</span></div><div class="patent-section patent-abstract-section"><div class="patent-section-header"><span class="patent-section-title">Abstract</span></div><div class="patent-text"><abstract mxw-id="PA50565987" lang="EN" load-source="patent-office"> <div class="abstract">The present invention relates to a method for the determination of the presence and amount of DNA in a sample. The method is based on the use of a nucleic acid template dependent enzyme in combination with a random primer to generate an enzymatic product which incorporates a binding species and a detectable species covalently linked.</div>
  </abstract></div></div><div class="patent-section patent-drawings-section"><div class="patent-section-header"><span class="patent-section-title">Images<span class="patent-section-count">(20)</span></span></div><div class="patent-drawings-body"><div class="patent-drawings-carousel"><div class="drawings"><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6635418-0.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6635418-0.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6635418-1.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6635418-1.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6635418-2.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6635418-2.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6635418-3.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6635418-3.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6635418-4.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6635418-4.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6635418-5.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6635418-5.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6635418-6.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6635418-6.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6635418-7.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6635418-7.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6635418-8.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6635418-8.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6635418-9.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6635418-9.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6635418-10.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6635418-10.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6635418-11.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6635418-11.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6635418-12.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6635418-12.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6635418-13.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6635418-13.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6635418-14.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6635418-14.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6635418-15.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6635418-15.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6635418-16.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6635418-16.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6635418-17.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6635418-17.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6635418-18.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6635418-18.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/pages/US6635418-19.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/pages/US6635418-19.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div></div></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img" alt="Previous page"src="/googlebooks/images/kennedy/page_left.png"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img" alt="Next page"src="/googlebooks/images/kennedy/page_right.png"width="21" height="21" /></div></div></div><div class="patent-post-drawings"></div><div class="patent-section patent-claims-section"><div class="patent-section-header"><span class="patent-section-title">Claims<span class="patent-section-count">(62)</span></span></div><div class="patent-text"><div mxw-id="PCLM8562906" lang="EN" load-source="patent-office" class="claims">
    <claim-statement>What is claimed is: </claim-statement> <div class="claim"> <div num="1" id="US-6635418-B2-CLM-00001" class="claim">
      <div class="claim-text">1. A method for determining total nucleic acid in a sample, which comprises:</div>
      <div class="claim-text">a) mixing at least one random primer at least 4 nucleotides in length, having at least one detectable species with a sample which may contain nucleic acid, </div>
      <div class="claim-text">b) adding at least one nucleotide triphosphate having at least one binding species and optionally at least one second nucleotide triphosphate, </div>
      <div class="claim-text">c) adding at least one nucleic acid polymerase, </div>
      <div class="claim-text">d) incubating the mixture of step c), under conditions which allow said at least one nucleic acid polymerase to be active, </div>
      <div class="claim-text">e) contacting the mixture of step d) with at least one solid phase, </div>
      <div class="claim-text">f) measuring total nucleic acid in said sample by measuring the total amount of said at least one detectable species bound to said solid phase, and </div>
      <div class="claim-text">g) determining whether the total nucleic acid in said sample is higher or lower than a threshold amount of contamination, wherein the threshold amount of contamination is equal to or less than 100 pg. </div>
    </div>
    </div> <div class="claim"> <div num="2" id="US-6635418-B2-CLM-00002" class="claim">
      <div class="claim-text">2. A method for determining total nucleic acid in a sample, which comprises:</div>
      <div class="claim-text">a) mixing at least one random primer at least 4 nucleotides in length, having at least one binding species with a sample which may contain nucleic acid </div>
      <div class="claim-text">b) adding at least one nucleotide triphosphate having at least one detectable species and optionally at least one second nucleotide triphosphate, </div>
      <div class="claim-text">c) adding at least one nucleic acid polymerase, </div>
      <div class="claim-text">d) incubating the mixture of step c), under conditions which allow said at least one nucleic acid polymerase to be active, </div>
      <div class="claim-text">e) contacting the mixture of step d) with at least one solid phase, </div>
      <div class="claim-text">f) measuring total nucleic acid in said sample by measuring the total amount of said at least one detectable species bound to said solid phase, and </div>
      <div class="claim-text">g) determining whether the total nucleic acid in said sample is higher or lower than a threshold amount of contamination, wherein the threshold amount of contamination is equal to or less than 100 pg. </div>
    </div>
    </div> <div class="claim"> <div num="3" id="US-6635418-B2-CLM-00003" class="claim">
      <div class="claim-text">3. A method for determining total nucleic acid in a sample, which comprises:</div>
      <div class="claim-text">a) mixing at least one random primer at least 4 nucleotides in length with a sample which may contain nucleic acid, </div>
      <div class="claim-text">b) adding at least one nucleotide triphosphate having at least one binding species and optionally at least one nucleotide triphosphate having at least one detectable species and optionally at least one second nucleotide triphosphate, </div>
      <div class="claim-text">c) adding at least one nucleic acid polymerase, </div>
      <div class="claim-text">d) incubating the mixture of step c), under conditions which allow said at least one nucleic acid polymerase to be active, </div>
      <div class="claim-text">e) contacting the mixture of step d) with at least one solid phase, </div>
      <div class="claim-text">f) measuring total nucleic acid in said sample by measuring the total amount of said at least one detectable species or the amount of said at least one binding species bound to said solid phase, and </div>
      <div class="claim-text">g) determining whether the total nucleic acid in said sample is higher or lower than a threshold amount of contamination, wherein the threshold amount of contamination is equal to or less than 100 pg. </div>
    </div>
    </div> <div class="claim"> <div num="4" id="US-6635418-B2-CLM-00004" class="claim">
      <div class="claim-text">4. A method for determining total nucleic acid in a sample, which comprises:</div>
      <div class="claim-text">a) mixing at least one first labeled random primer at least 4 nucleotides in length having at least one binding species and at least one second random primer at least 4 nucleotides in length having at least one detectable species, with a sample which may contain nucleic acid, </div>
      <div class="claim-text">b) adding at least one nucleic acid ligase, </div>
      <div class="claim-text">c) incubating the mixture of step b), under conditions which allow said at least one nucleic acid ligase to be active, </div>
      <div class="claim-text">d) contacting the mixture of step c) with at least one solid phase, </div>
      <div class="claim-text">e) measuring total nucleic acid in said sample by measuring the total amount of said at least one detectable species or the amount of said at least one binding species bound to said solid phase, and </div>
      <div class="claim-text">f) determining whether the total nucleic acid in said sample is higher or lower than a threshold amount of contamination, wherein the threshold amount of contamination is equal to or less than 100 pg. </div>
    </div>
    </div> <div class="claim"> <div num="5" id="US-6635418-B2-CLM-00005" class="claim">
      <div class="claim-text">5. A method for determining total nucleic acid in a sample, which comprises:</div>
      <div class="claim-text">a) mixing at least one first labeled random primer at least 4 nucleotides in length having at least one binding species and at least one second random primer at least 4 nucleotides in length having at least one detectable species, with a sample which may contain nucleic acid, </div>
      <div class="claim-text">b) adding at least one nucleic acid ligase and at least one nucleic acid polymerase, </div>
      <div class="claim-text">c) incubating the mixture of step b), under conditions which allow said at least one nucleic acid ligase and at least one nucleic acid polymerase to be active, </div>
      <div class="claim-text">d) contacting the mixture of step c) with at least one solid phase, </div>
      <div class="claim-text">c) measuring total nucleic acid in said sample by measuring the total amount of said at least one detectable species or the amount of said at least one binding bound to said solid phase, and </div>
      <div class="claim-text">f) determining whether the total nucleic acid in said sample is higher or lower than a threshold amount of contamination, wherein the threshold amount of contamination is equal to or less than 100 pg. </div>
    </div>
    </div> <div class="claim-dependent"> <div num="6" id="US-6635418-B2-CLM-00006" class="claim">
      <div class="claim-text">6. A method as in <claim-ref idref="US-6635418-B2-CLM-00001">claim 1</claim-ref>, wherein said at least one binding species is selected from the group consisting of biotin, avidin, streptavidin, antibody, antigen, lectin, receptor, ligand, hormone, nucleic acid sequence, mimitope and nucleic acid base pairing polymer.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="7" id="US-6635418-B2-CLM-00007" class="claim">
      <div class="claim-text">7. A method as in <claim-ref idref="US-6635418-B2-CLM-00001">claim 1</claim-ref>, wherein said at least one detectable species is selected from the group consisting of biotin, avidin, streptavidin, antibody, antigen, lectin, receptor, ligand, hormone, nucleic acid sequence, mimitope, nucleic acid base pairing polymer, fluorescent molecule, electrochemiluminescent molecule, radioactive molecule, colored molecule, peroxidase, alkaline phosphatase and enzymes capable of producing a detectable species.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="8" id="US-6635418-B2-CLM-00008" class="claim">
      <div class="claim-text">8. A method as in <claim-ref idref="US-6635418-B2-CLM-00001">claim 1</claim-ref>, wherein said at least one nucleic acid polymerase is selected from the group consisting of Taq DNA polymerase, T4 DNA polymerase, Klenow fragment, Pfu DNA polymerase, Exo-Pfu DNA polymerase, <i>E. coli </i>DNA polymerase I, Klenow fragment of DNA polymerase I, MMLV reverse transcriptase and AMV reverse transcriptase.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="9" id="US-6635418-B2-CLM-00009" class="claim">
      <div class="claim-text">9. A method as in <claim-ref idref="US-6635418-B2-CLM-00001">claim 1</claim-ref>, wherein said at least one solid phase is selected from the group consisting of fiber, fibril, plastic surface, plastic bead, magnetic bead, plastic tube, gold surface, metal surface, metal bead and colloids.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="10" id="US-6635418-B2-CLM-00010" class="claim">
      <div class="claim-text">10. A method as in <claim-ref idref="US-6635418-B2-CLM-00001">claim 1</claim-ref>, wherein said at least one nucleotide triphosphate is selected from the group consisting of dATP, dGTP, dCTP, dUTP, dTTP, ATP, CTP, GTP, TTP, UTP inosineTP, propyne dCTP, propyne dUTP, 5-bromo dCTP, 5-iodo dUTP, 5-fluoro dUTP, O-6 methyl dGTP, 7-deaza dGTP, N-6 methyl-2′-dATP, biotin-dATP, biotin-dCTP, biotin-dUTP, digoxigenin dUTP, digoxigenin UTP and biotin ddUTP.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="11" id="US-6635418-B2-CLM-00011" class="claim">
      <div class="claim-text">11. A method as in <claim-ref idref="US-6635418-B2-CLM-00001">claim 1</claim-ref>, wherein said at least one random primer is 4-70 nucleotides.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="12" id="US-6635418-B2-CLM-00012" class="claim">
      <div class="claim-text">12. A method as in <claim-ref idref="US-6635418-B2-CLM-00001">claim 1</claim-ref>, wherein said conditions comprise a solution with a pH between 5.5 and 9.5, a nucleotide triphosphate concentration between 1 pM and 10 mM, a Mg2+ concentration between 0.05 mM and 500 mM, and a reducing agent concentration between 0 and 500 mM, wherein the sum of the molarities is between 1 mM and 500 mM.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="13" id="US-6635418-B2-CLM-00013" class="claim">
      <div class="claim-text">13. A method as in <claim-ref idref="US-6635418-B2-CLM-00004">claim 4</claim-ref>, wherein said at least one ligase is selected from the group consisting of Pfu DNA ligase, T4 DNA ligase, Taq DNA ligase, T4 RNA ligase, and <i>E. coli </i>DNA ligase.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="14" id="US-6635418-B2-CLM-00014" class="claim">
      <div class="claim-text">14. A method as in <claim-ref idref="US-6635418-B2-CLM-00001">claim 1</claim-ref>, wherein said random primer is from 4 to 20 nucleotides in length.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="15" id="US-6635418-B2-CLM-00015" class="claim">
      <div class="claim-text">15. A method as in <claim-ref idref="US-6635418-B2-CLM-00014">claim 14</claim-ref>, wherein said at least one detectable species is selected from the group consisting of biotin, nucleic acid sequence, nucleic acid base pairing linear polymer, fluorescent molecule, electrochemiluminescent molecule, radioactive molecule, peroxidase and alkaline phosphatase.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="16" id="US-6635418-B2-CLM-00016" class="claim">
      <div class="claim-text">16. A method as in <claim-ref idref="US-6635418-B2-CLM-00014">claim 14</claim-ref>, wherein said at least one binding species is selected from the group consisting of biotin, antigen, lectin, ligand, hormone, nucleic acid sequence, mimitope and nucleic acid base pairing linear polymer.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="17" id="US-6635418-B2-CLM-00017" class="claim">
      <div class="claim-text">17. A method as in <claim-ref idref="US-6635418-B2-CLM-00014">claim 14</claim-ref>, wherein said at least one nucleic acid polymerase is selected from the group consisting of Taq DNA polymerase, Klenow fragment (3′-5′) of <i>E. coli </i>DNA polymerase I and Klenow fragment of DNA polymerase I.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="18" id="US-6635418-B2-CLM-00018" class="claim">
      <div class="claim-text">18. A method as in <claim-ref idref="US-6635418-B2-CLM-00014">claim 14</claim-ref>, wherein said at least one solid phase is selected from the group consisting of magnetic bead, plastic plate and polymer bead.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="19" id="US-6635418-B2-CLM-00019" class="claim">
      <div class="claim-text">19. A method as in <claim-ref idref="US-6635418-B2-CLM-00014">claim 14</claim-ref>, wherein said at least one nucleotide triphosphate is selected from the group consisting of dATP, dGTP, dCTP, dUTP, dTTP, 7-deaza dGTP, biotin-dATP, biotin-dCTP, biotin-dUTP, digoxigenin dUTP, digoxigenin UTP and biotin ddUTP.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="20" id="US-6635418-B2-CLM-00020" class="claim">
      <div class="claim-text">20. A method as in <claim-ref idref="US-6635418-B2-CLM-00014">claim 14</claim-ref>, wherein said random primer is 6-10 nucleotides in length.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="21" id="US-6635418-B2-CLM-00021" class="claim">
      <div class="claim-text">21. A method as in <claim-ref idref="US-6635418-B2-CLM-00014">claim 14</claim-ref>, wherein said conditions comprise those optimal for Klenow fragment of DNA polymerase I to synthesize DNA.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="22" id="US-6635418-B2-CLM-00022" class="claim">
      <div class="claim-text">22. A method as in <claim-ref idref="US-6635418-B2-CLM-00020">claim 20</claim-ref>, wherein said NTP is a dNTP.</div>
    </div>
    </div> <div class="claim"> <div num="23" id="US-6635418-B2-CLM-00023" class="claim">
      <div class="claim-text">23. A method for determining total nucleic acid in a sample, which comprises:</div>
      <div class="claim-text">a) mixing at least one random primer at least 4 nucleotides in length having at least one detectable species, with a sample which may contain nucleic acid, </div>
      <div class="claim-text">b) adding at least one nucleotide triphosphate having at least one binding species and optionally at least one second nucleotide triphosphate, </div>
      <div class="claim-text">c) adding at least one nucleic acid polymerase, </div>
      <div class="claim-text">d) incubating the mixture of step c), under conditions which allow said at least one nucleic acid polymerase to be active, </div>
      <div class="claim-text">e) measuring total nucleic acid in said sample by measuring the total amount of said at least one detectable species or the amount of said at least one binding species, and </div>
      <div class="claim-text">f) determining whether the total nucleic acid in said sample is higher or lower than a threshold amount of contamination, wherein the threshold amount of contamination is equal to or less than 100 pg. </div>
    </div>
    </div> <div class="claim"> <div num="24" id="US-6635418-B2-CLM-00024" class="claim">
      <div class="claim-text">24. A method for determining total nucleic acid in a sample, which comprises:</div>
      <div class="claim-text">a) mixing at least one random primer at least 4 nucleotides in length, having at least one binding species, with a sample which may contain nucleic acid, </div>
      <div class="claim-text">b) adding at least one nucleotide triphosphate having at least one detectable species and optionally at least one second nucleotide triphosphate, </div>
      <div class="claim-text">c) adding at least one nucleic acid polymerase, </div>
      <div class="claim-text">d) incubating the mixture of step c), under conditions which allow said at least one nucleic acid polymerase to be active </div>
      <div class="claim-text">e) measuring total nucleic acid in said sample by measuring the total amount of said at least one detectable species or the amount of said at least one binding species, and </div>
      <div class="claim-text">f) determining whether the total nucleic acid in said sample is higher or lower than a threshold amount of contamination, wherein the threshold amount of contamination is equal to or less than 100 pg. </div>
    </div>
    </div> <div class="claim"> <div num="25" id="US-6635418-B2-CLM-00025" class="claim">
      <div class="claim-text">25. A method for determining total nucleic acid in a sample, which comprises:</div>
      <div class="claim-text">a) mixing at least one random primer at least 4 nucleotides in length with a sample which may contain nucleic acid, </div>
      <div class="claim-text">b) adding at least one nucleotide triphosphate having at least one binding moiety and optionally at least one second nucleotide triphosphate having at least one label and optionally at least one nucleotide triphosphate, </div>
      <div class="claim-text">c) adding at least one nucleic acid polymerase, </div>
      <div class="claim-text">d) incubating the mixture of step c), under conditions which allow said at least one nucleic acid polymerase to be active, </div>
      <div class="claim-text">e) measuring total nucleic acid in said sample by measuring the total amount of said at least one label or the amount of said at least one binding moiety, and </div>
      <div class="claim-text">f) determining whether the total nucleic acid in said sample is higher or lower than a threshold amount of contamination, wherein the threshold amount of contamination is equal to or less than 100 pg. </div>
    </div>
    </div> <div class="claim"> <div num="26" id="US-6635418-B2-CLM-00026" class="claim">
      <div class="claim-text">26. A method for determining total nucleic acid in a sample, which comprises:</div>
      <div class="claim-text">a) mixing at least one labeled random primer at least 4 nucleotides in length having at least one binding species and optionally at least one second random primer at least 4 nucleotides in length having at least one detectable species, with a sample nucleic acid, </div>
      <div class="claim-text">b) adding at least one nucleic acid ligase, </div>
      <div class="claim-text">c) incubating the mixture of step b), under conditions which allow said at least one nucleic acid ligase to be active, </div>
      <div class="claim-text">d) measuring total nucleic acid in said sample by measuring the total amount of said at least one detectable species or the amount of said at least one binding species, and </div>
      <div class="claim-text">e) determining whether the total nucleic acid in said sample is higher or lower than a threshold amount of contamination, wherein the threshold amount of contamination is equal to or less than 100 pg. </div>
    </div>
    </div> <div class="claim"> <div num="27" id="US-6635418-B2-CLM-00027" class="claim">
      <div class="claim-text">27. A method for determining total nucleic acid in a sample, which comprises:</div>
      <div class="claim-text">a) mixing at least one labeled random primer at least 4 nucleotides in length having at least one binding species and optionally at least one second random primer at least 4 nucleotides in length having at least one detectable species, with a sample which may contain nucleic acid, </div>
      <div class="claim-text">b) adding at least one nucleic acid ligase and at least one nucleic acid polymerase, </div>
      <div class="claim-text">c) adding at least one nucleotide triphosphate, </div>
      <div class="claim-text">d) incubating the mixture of step c), under conditions which allow said at least one nucleic acid ligase and at least one nucleic acid polymerase to be active, </div>
      <div class="claim-text">e) measuring total nucleic acid in said sample by measuring the total amount of said at least one detectable species or the amount of said at least one binding species, and </div>
      <div class="claim-text">f) determining whether the total nucleic acid in said sample is higher or lower than a threshold amount of contamination, wherein the threshold amount of contamination is equal to or less than 100 pg. </div>
    </div>
    </div> <div class="claim-dependent"> <div num="28" id="US-6635418-B2-CLM-00028" class="claim">
      <div class="claim-text">28. A method as in <claim-ref idref="US-6635418-B2-CLM-00023">claim 23</claim-ref>, wherein said at least one detectable species is selected from the group consisting of biotin, avidin, streptavidin, antibody, antigen, lectin, receptor, ligand, hormone, nucleic acid sequence, mimitope, nucleic acid base pairing linear polymer, fluorescent molecule, electrochemiluminescent molecule, radioactive molecule, colored molecule, peroxidase, alkaline phosphatase and enzymes capable of producing a detectable species.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="29" id="US-6635418-B2-CLM-00029" class="claim">
      <div class="claim-text">29. A method as in <claim-ref idref="US-6635418-B2-CLM-00001">claim 1</claim-ref>, wherein said at least one nucleic acid polymerase is selected from the group consisting of Taq DNA polymerase, T4 DNA polymerase, Klenow fragment, Pfu DNA polymerase, Exo-Pfu DNA polymerase, <i>E. coli </i>DNA polymerase I, Klenow fragment of DNA polymerase I, MMLV reverse transcriptase and AMV reverse transcriptase.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="30" id="US-6635418-B2-CLM-00030" class="claim">
      <div class="claim-text">30. A method as in <claim-ref idref="US-6635418-B2-CLM-00023">claim 23</claim-ref>, wherein said at least one nucleotide triphosphate is selected from the group consisting of dATP, dGTP, dCTP, dUTP, dTTP, ATP, CTP, GTP, TTP, UTP inosineTP, propyne dCTP, propyne dUTP, 5-bromo dCTP, 5-iodo dUTP, 5-fluoro dUTP, O-6 methyl dGTP, 7-deaza dGTP, N-6 methyl-2′-dATP, biotin-dATP, biotin-dCTP, biotin-dUTP, digoxigenin dUTP, digoxigenin UTP and biotin ddUTP.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="31" id="US-6635418-B2-CLM-00031" class="claim">
      <div class="claim-text">31. A method as in <claim-ref idref="US-6635418-B2-CLM-00023">claim 23</claim-ref>, wherein said at least one random primer is 4-70 nucleotides.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="32" id="US-6635418-B2-CLM-00032" class="claim">
      <div class="claim-text">32. A method as in <claim-ref idref="US-6635418-B2-CLM-00023">claim 23</claim-ref>, wherein said conditions comprise a solution with a pH between 5.5 and 9.5, a nucleotide triphosphate concentration between 1 pM and 10 mM, a Mg2+ concentration between 0.05 mM and 500 mM, and a reducing agent concentration between 0 and 500 mM, where the sum of the molarities is between 1 M and 500 mM.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="33" id="US-6635418-B2-CLM-00033" class="claim">
      <div class="claim-text">33. A method as in <claim-ref idref="US-6635418-B2-CLM-00026">claim 26</claim-ref>, wherein said at least one ligase is selected from the group consisting of Pfu DNA ligase, T4 DNA ligase, Taq DNA ligase, T4 RNA ligase, and <i>E. coli </i>DNA ligase.</div>
    </div>
    </div> <div class="claim"> <div num="34" id="US-6635418-B2-CLM-00034" class="claim">
      <div class="claim-text">34. A method for determining total nucleic acid in a sample, which comprises:</div>
      <div class="claim-text">a) mixing at least one random primer at least 4 nucleotides in length having at least one first label, with a sample nucleic acid, </div>
      <div class="claim-text">b) adding at least one nucleotide triphosphate having at least one second label and optionally at least one second nucleotide triphosphate, </div>
      <div class="claim-text">c) adding at least one nucleic acid polymerase, </div>
      <div class="claim-text">d) incubating the mixture of step c), under conditions which allow said at least one nucleic acid polymerase to be active, </div>
      <div class="claim-text">e) measuring total nucleic acid in said sample by measuring the total amount of said at least one first label or the amount of said at least one second label, and </div>
      <div class="claim-text">f) determining whether the total nucleic acid in said sample is higher or lower than a threshold amount of contamination, wherein the threshold amount of contamination is equal to or less than 100 pg. </div>
    </div>
    </div> <div class="claim"> <div num="35" id="US-6635418-B2-CLM-00035" class="claim">
      <div class="claim-text">35. A method for determining total nucleic acid in a sample, which comprises:</div>
      <div class="claim-text">a) mixing at least one labeled random primer at least 4 nucleotides in length having at least one first label species and optionally at least one second random primer at least 4 nucleotides in length having at least one second label, with a sample which may contain nucleic acid, </div>
      <div class="claim-text">b) adding at least one nucleic acid ligase, </div>
      <div class="claim-text">c) incubating the mixture of step b), under conditions which allow said at least one nucleic acid ligase to be active, </div>
      <div class="claim-text">d) measuring total nucleic acid in said sample by measuring the total amount of said at least one first label or the amount of said at least one second label, and </div>
      <div class="claim-text">e) determining whether the total nucleic acid in said sample is higher or lower than a threshold amount of contamination, wherein the threshold amount of contamination is equal to or less than 100 pg. </div>
    </div>
    </div> <div class="claim"> <div num="36" id="US-6635418-B2-CLM-00036" class="claim">
      <div class="claim-text">36. A method for determining total nucleic acid in a sample, which comprises:</div>
      <div class="claim-text">a) mixing at least one labeled random primer at least 4 nucleotides in length having at least one first label and optionally at least one second random primer at least 4 nucleotides in length having at least one second label, with a sample which may contain nucleic acid </div>
      <div class="claim-text">b) adding at least one nucleic acid ligase and at least one nucleic acid polymerase, </div>
      <div class="claim-text">c) adding at least one nucleotide triphosphate, </div>
      <div class="claim-text">d) incubating the mixture of step c), under conditions which allow said at least one nucleic acid ligase and at least one nucleic acid polymerase to be active, </div>
      <div class="claim-text">e) measuring total nucleic acid in said sample by measuring the total amount of said at least one first label or the amount of said at least one second label, and </div>
      <div class="claim-text">f) determining whether the total nucleic acid in said sample is higher or lower than a threshold amount of contamination, wherein the threshold amount of contamination is equal to or less than 100 pg. </div>
    </div>
    </div> <div class="claim-dependent"> <div num="37" id="US-6635418-B2-CLM-00037" class="claim">
      <div class="claim-text">37. The method of <claim-ref idref="US-6635418-B2-CLM-00001">claim 1</claim-ref>, wherein said method is capable of detecting total nucleic acid in amounts as low as 5 pg.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="38" id="US-6635418-B2-CLM-00038" class="claim">
      <div class="claim-text">38. The method of <claim-ref idref="US-6635418-B2-CLM-00002">claim 2</claim-ref>, wherein said method is capable of detecting total nucleic acid in amounts as low as 5 pg.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="39" id="US-6635418-B2-CLM-00039" class="claim">
      <div class="claim-text">39. The method of <claim-ref idref="US-6635418-B2-CLM-00003">claim 3</claim-ref>, wherein said method is capable of detecting total nucleic acid in amounts as low as 5 pg.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="40" id="US-6635418-B2-CLM-00040" class="claim">
      <div class="claim-text">40. The method of <claim-ref idref="US-6635418-B2-CLM-00004">claim 4</claim-ref>, wherein said method is capable of detecting total nucleic acid in amounts as low as 5 pg.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="41" id="US-6635418-B2-CLM-00041" class="claim">
      <div class="claim-text">41. The method of <claim-ref idref="US-6635418-B2-CLM-00005">claim 5</claim-ref>, wherein said method is capable of detecting total nucleic acid in amounts as low as 5 pg.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="42" id="US-6635418-B2-CLM-00042" class="claim">
      <div class="claim-text">42. The method of <claim-ref idref="US-6635418-B2-CLM-00023">claim 23</claim-ref>, wherein said method is capable of detecting total nucleic acid in amounts as low as 5 pg.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="43" id="US-6635418-B2-CLM-00043" class="claim">
      <div class="claim-text">43. The method of <claim-ref idref="US-6635418-B2-CLM-00024">claim 24</claim-ref>, wherein said method is capable of detecting total nucleic acid in amounts as low as 5 pg.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="44" id="US-6635418-B2-CLM-00044" class="claim">
      <div class="claim-text">44. The method of <claim-ref idref="US-6635418-B2-CLM-00025">claim 25</claim-ref>, wherein said method is capable of detecting total nucleic acid in amounts as low as 5 pg.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="45" id="US-6635418-B2-CLM-00045" class="claim">
      <div class="claim-text">45. The method of <claim-ref idref="US-6635418-B2-CLM-00026">claim 26</claim-ref>, wherein said method is capable of detecting total nucleic acid in amounts as low as 5 pg.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="46" id="US-6635418-B2-CLM-00046" class="claim">
      <div class="claim-text">46. The method of <claim-ref idref="US-6635418-B2-CLM-00027">claim 27</claim-ref>, wherein said method is capable of detecting total nucleic acid in amounts as low as 5 pg.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="47" id="US-6635418-B2-CLM-00047" class="claim">
      <div class="claim-text">47. The method of <claim-ref idref="US-6635418-B2-CLM-00034">claim 34</claim-ref>, wherein said method is capable of detecting total nucleic acid in amounts as low as 5 pg.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="48" id="US-6635418-B2-CLM-00048" class="claim">
      <div class="claim-text">48. The method of <claim-ref idref="US-6635418-B2-CLM-00035">claim 35</claim-ref>, wherein said method is capable of detecting total nucleic acid in amounts as low as 5 pg.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="49" id="US-6635418-B2-CLM-00049" class="claim">
      <div class="claim-text">49. The method of <claim-ref idref="US-6635418-B2-CLM-00036">claim 36</claim-ref>, wherein said method is capable of detecting total nucleic acid in amounts as low 5 pg.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="50" id="US-6635418-B2-CLM-00050" class="claim">
      <div class="claim-text">50. The method of <claim-ref idref="US-6635418-B2-CLM-00001">claim 1</claim-ref>, wherein said method is capable of detecting DNA fragments shorter than 800 basepairs.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="51" id="US-6635418-B2-CLM-00051" class="claim">
      <div class="claim-text">51. The method of <claim-ref idref="US-6635418-B2-CLM-00002">claim 2</claim-ref>, wherein said method is capable of detecting total DNA fragments shorter than 800 basepairs.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="52" id="US-6635418-B2-CLM-00052" class="claim">
      <div class="claim-text">52. The method of <claim-ref idref="US-6635418-B2-CLM-00003">claim 3</claim-ref>, wherein said method is capable of detecting total DNA fragments shorter than 800 basepairs.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="53" id="US-6635418-B2-CLM-00053" class="claim">
      <div class="claim-text">53. The method of <claim-ref idref="US-6635418-B2-CLM-00004">claim 4</claim-ref>, wherein said method is capable of detecting total DNA fragments shorter than 800 basepairs.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="54" id="US-6635418-B2-CLM-00054" class="claim">
      <div class="claim-text">54. The method of <claim-ref idref="US-6635418-B2-CLM-00005">claim 5</claim-ref>, wherein said method is capable of detecting total DNA fragments shorter than 800 basepairs.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="55" id="US-6635418-B2-CLM-00055" class="claim">
      <div class="claim-text">55. The method of <claim-ref idref="US-6635418-B2-CLM-00023">claim 23</claim-ref>, wherein said method is capable of detecting DNA fragments shorter than 800 basepairs.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="56" id="US-6635418-B2-CLM-00056" class="claim">
      <div class="claim-text">56. The method of <claim-ref idref="US-6635418-B2-CLM-00024">claim 24</claim-ref>, wherein said method is capable of detecting DNA fragments shorter than 800 basepairs.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="57" id="US-6635418-B2-CLM-00057" class="claim">
      <div class="claim-text">57. The method of <claim-ref idref="US-6635418-B2-CLM-00025">claim 25</claim-ref>, wherein said method is capable of detecting DNA fragments shorter than 800 basepairs.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="58" id="US-6635418-B2-CLM-00058" class="claim">
      <div class="claim-text">58. The method of <claim-ref idref="US-6635418-B2-CLM-00026">claim 26</claim-ref>, wherein said method is capable of detecting DNA fragments shorter than 800 basepairs.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="59" id="US-6635418-B2-CLM-00059" class="claim">
      <div class="claim-text">59. The method of <claim-ref idref="US-6635418-B2-CLM-00027">claim 27</claim-ref>, wherein said method is capable of detecting DNA fragments shorter than 800 basepairs.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="60" id="US-6635418-B2-CLM-00060" class="claim">
      <div class="claim-text">60. The method of <claim-ref idref="US-6635418-B2-CLM-00034">claim 34</claim-ref>, wherein said method is capable of detecting DNA fragments shorter than 800 basepairs.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="61" id="US-6635418-B2-CLM-00061" class="claim">
      <div class="claim-text">61. The method of <claim-ref idref="US-6635418-B2-CLM-00035">claim 35</claim-ref>, wherein said method is capable of detecting DNA fragments shorter than 800 basepairs.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="62" id="US-6635418-B2-CLM-00062" class="claim">
      <div class="claim-text">62. The method of <claim-ref idref="US-6635418-B2-CLM-00036">claim 36</claim-ref>, wherein said method is capable of detecting DNA fragments shorter than 800 basepairs.</div>
    </div>
  </div> </div></div></div><div class="patent-section patent-description-section"><div class="patent-section-header"><span class="patent-section-title">Description</span></div><div class="patent-text"><div mxw-id="PDES54003728" lang="EN" load-source="patent-office" class="description">
    <heading>FIELD OF INVENTION</heading> <p>The present invention relates to a method for the determination of the presence and amount of DNA in a sample. The method is based on the use of a nucleic acid template dependent enzyme in combination with a random primer to generate an enzymatic product which incorporates a binding species and a detectable species covalently linked.</p>
    <heading>BACKGROUND OF INVENTION</heading> <p>The development of therapeutic biopharmaceuticals for human injection generated through recombinant deoxyribonucleic acid (DNA) technology has resulted in new standards for product purity. Viral contamination of several vaccines initially demonstrated the safety risk associated with the products generated from recombinant technology. Thus, contamination of products by host cell DNA could be a biological hazard for the recipient. The primary concern of regulatory agencies is the potential contamination of product with oncogenes or infectious viral DNA. DNA in high amounts (1-10 μg) has been shown to cause tumors in mice. Because the risk associated with exposure of 100 pg of DNA per dose is negligible, the WHO (World Health Organization) currently requires the DNA in all biopharmaceuticals to be below 100 pg per dose. Thus, methodologies that accurately determine picogram amounts of DNA have become a requisite in the biopharmaceutical field. Several different methodologies exist for quantitation of DNA. Each method has limitations with respect to dynamic range for accurate DNA quantitation and biases due to base composition. Because of the multitude of manipulations during the purification process of a typical protein therapeutic sample, the DNA in the final protein therapeutic is most likely sheared (less than 1,000 basepairs) and exists in both single and/or double stranded forms.</p>
    <p>1. Current Methods for Determination of DNA</p>
    <p>The most widely accepted method for quantitation of DNA is absorbance spectroscopy. At 260 nm, each base has a specific absorption spectra. Using a spectrophotometer, an average absorbance is obtained which directly reflects the base composition of the DNA. This reading is based on the base composition of the DNA because at an OD of 260, the extinction coefficient of adenosine and guanosine bases is more than cytosine or thymine. The discrimination on base composition and the low level of sensitivity (1 ug DNA) are several limitations of this method. Another significant drawback of this method is the interference caused by impurities such as organic compounds and protein.</p>
    <p>A second method for quantitation of DNA utilizes fluorescence spectroscopy.</p>
    <p>Upon binding DNA, the fluorescence characteristics of flourochromes change such that energy emitted at a particular wavelength is proportional to the amount of DNA in the sample. Typically, unknowns are extrapolated from a standard curve of known calibrated DNA standards. Several dyes currently exist that have the capacity to bind single or double stranded DNA, as well as mRNA. Ethidium bromide is one dye that binds to DNA without base discrimination and can be used to accurately quantitate at least 100 ng of DNA. Although the Hoechst dye (33258, Molecular Probes Eugene, Oreg.) quantitates 10 ng of DNA accurately (DyNA Quant; Hoefer Pharmacia Biotech), it preferentially binds AT rich regions of DNA. The Beacon DNA dye (PanVera Corp., Madison, Wis.) detects 25 ng/ml of double stranded DNA only. Molecular Probes manufactures two dyes that the company claims to accurately detect 25 pg/ml of double stranded DNA (PicoGreen) and 100 pg/ml of single stranded DNA (OliGreen). If both forms of DNA are represented in a given sample, two separate determinations must be performed. A PCR based method exists that amplifies the DNA and measures the relative amounts of DNA by changes in fluorescence polarization. All these fluorochromes have limitations with respect to the ability to detect low picogram amounts of DNA.</p>
    <p>Another method for quantitation of DNA utilizes a specific labeled probe to quantitate a unique sequence of host DNA. Currently, the Southern slot-blot method is accepted by the FDA for the final determination of total DNA in protein therapeutic samples. Specifically, a labeled probe generated from purified host DNA is used to quantitate the presence of specific sequences of DNA in the final protein therapeutic sample. Although slot-blot detects low picogram quantities of DNA, the results represent only the host DNA. Therefore, contaminating DNA from other genes of the host or from other contaminants (such as bacterial DNA) are not detected. Major disadvantages of slot-blot include a relatively narrow dynamic range, a technically challenging protocol, a time consuming, and difficult protocol to troubleshoot. Additionally, the semi-quantitative results obtained from this method are directly dependent on the specific activity of the probe which can be highly variable.</p>
    <p>Another commonly used methodology (for quantitation of 2-200 picograms of DNA) is an assay that captures DNA by binding to a single stranded DNA binding protein and detects the DNA via a enzymatic amplification of signal from a enzyme labeled anti-DNA antibody. Disadvantages to the total DNA assay performed on a threshold system (Molecular Devices) include the narrow dynamic range, the technically challenging and time consuming protocol. In addition the inability to accurately detect fragments of DNA below 872 basepairs severely limits the overall sensitivity of the assay. In fact, decreasing the size of DNA below 872 basepairs inhibits the detection of fragments greater than 872 basepairs. The lack of sensitivity for smaller DNA fragments is a major disadvantage of this methodology because the significant quantity of DNA in protein therapeutic samples is fragmented.</p>
    <p>Chemical modification of DNA bases followed by detection via an enzyme labeled antibody that recognizes specific modified base is another technique that claims to detect pg amounts of DNA in samples. However, this method is complicated, extremely technically challenging, requires handing and requires subsequent disposal of hazardous waste, and extensive data manipulation.</p>
    <p>2. Current Technology Using Random Oligonucleotide Priming</p>
    <p>DNA dependent DNA polymerases incorporate deoxyribonucleotide to the 3′ hydroxyl terminus of a double stranded primed DNA molecule. The synthesis of the new strand of DNA occurs in a 5′ to 3′ direction with respect to the synthesized strand. Each nucleotide that is incorporated during the polymerization is complementary to the one opposite to it in the template(dA pairs with dT, dC with dG). The reaction requires four deoxyribonucleotide triphosphates (dNTPs) and magnesium ions. Many of the polymerases have a 3′-5′ exonuclease inherently associated with the polymerase activity. During DNA synthesis, the exonuclease activity performs a proofreading function by removing mismatched and modified nucleotides. One property of DNA polymerases that becomes important for the random priming reactions is the ability to continuously incorporate nucleotides without dissociating from the primer template (processivity). The low processivity associated with the klenow fragment (C-terminus) of DNA polymerase allows incorporation of less than 10 nucleotides into the primer template and thus maximizes the generation of signal. An outline of the properties of some polynucleotide dependent polymerases are provided in the table below.</p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="4"> <colspec colname="1" colwidth="70pt" align="left"> </colspec> <colspec colname="2" colwidth="42pt" align="left"> </colspec> <colspec colname="3" colwidth="56pt" align="left"> </colspec> <colspec colname="4" colwidth="49pt" align="left"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="4" align="center" rowsep="1" class="description-td" colspan="4"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">3′-5′</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Enzyme</td>
                <td class="description-td">exonuclease</td>
                <td class="description-td">Polymerase rate</td>
                <td class="description-td">Processivity</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="4" align="center" rowsep="1" class="description-td" colspan="4"> </td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">klenow fragment</td>
                <td class="description-td">low</td>
                <td class="description-td">intermediate</td>
                <td class="description-td">low</td>
              </tr> <tr class="description-tr"> <td class="description-td">Reverse transcriptase</td>
                <td class="description-td">none</td>
                <td class="description-td">slow</td>
                <td class="description-td">intermediate</td>
              </tr> <tr class="description-tr"> <td class="description-td">Taq DNA polymerase</td>
                <td class="description-td">none</td>
                <td class="description-td">fast</td>
                <td class="description-td">high</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="4" align="center" rowsep="1" class="description-td" colspan="4"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p>Several kits have been generated based on the properties of these polymerases. These protocols have become widely acceptable methodologies for labeling DNA and generating probes. Included in these methods are nick-translation of double stranded DNA and random priming of DNA using hexamer oligonucleotides. Both these protocols utilize DNA polymerases to incorporate the signal moiety (labeled-dNTP) into the DNA. In addition to DNA polymerase, terminal transferase can also be used to generate a labeled DNA. The detection of DNA using these protocols is dependent upon hybridization of a labeled DNA probe to a complementary strand of DNA/RNA. Many commercially available kits use these acceptable methodologies for labeling DNA. One kit format utilizes a “biotinylated hexamer mix” in conjunction with a biotinylated dNTP mix to generate a biotinylated DNA probe using a random priming methodology. This probe is hybridized to target DNA and the signal generated via an enzyme mediated chemiluminescence that is sensitive to low picogram amounts of DNA. Another kit utilizes a random nanomer, instead of a hexamer, and incorporates a fluorescein-dNTP into the DNA. An alternative format to random priming utilizes incorporation of digoxigenin-dUTP into the DNA which is detected by labeled anti-digoxigenin antibody. See also U.S. Pat. No. 5,043,272 which describes the use of random sequences for amplification.</p>
    <p>In addition, a method of polymerase activity by incubating the polymerase with a template nucleic acid, one detectable mononucleoside triphosphate and one immobilizable nucleoside triphosphate, binding the immobilizable nucleotides to a solid phase and detecting the bound detectable nucleotides and tests based thereon U.S. Pat. No. 5,413,906 has been described.</p>
    <heading>OBJECTS OF THE INVENTION</heading> <p>It is an object of this invention to provide methods and kits for determining the presence and amount of nucleic acid in a sample using random primers and nucleic acid template dependent enzymes.</p>
    <heading>SUMMARY OF THE INVENTION</heading> <p>This invention includes a method for the detection and/or quantitation of nucleic acid in a sample, which comprises; mixing at least one random primer (advantageously one or two primers) at least 4 nucleotides in length (advantageously 6-10 in length), having at least one detectable species (advantageously 1-3 detectable species) with a sample nucleic acid, adding at least one NTP (advantageously 1-3 NTPs) having at least one binding species (advantageously 1-3 binding species) and optionally at least one NTP (advantageously 3-5 NTPs), adding at least one nucleic acid polymerase (advantageously 1-3 polymerases), incubating this mixture under conditions which allow the at least one nucleic acid polymerase to be active, contacting this mixture with at least one solid phase (advantageously 1-3 solid phases) and detecting and/or quantitating the amount of nucleic acid in the sample by detecting and/or quantitating the amount of the at least one detectable species bound to said solid phase.</p>
    <p>This invention also includes a method for the detection and/or quantitation of nucleic acid in a sample, which comprises; mixing at least one random primer (advantageously one or two primers) at least 4 nucleotides in length (advantageously 6-10 in length), having at least one binding species (advantageously 1-3 binding species) with a sample nucleic acid, adding at least one NTP (advantageously 1-3 NTPs) having at least one detectable species (advantageously 1-3 detectable species) and optionally at least one NTP (advantageously 3-5 NTPs), adding at least one nucleic acid polymerase (advantageously 1-3 polymerases), incubating this mixture under conditions which allow the at least one nucleic acid polymerase to be active, contacting this mixture with at least one solid phase (advantageously 1-3 solid phases) and detecting and/or quantitating the amount of nucleic acid in the sample by detecting and/or quantitating the amount of the at least one detectable species bound to said solid phase.</p>
    <p>This invention also includes a method for the detection and/or quantitation of nucleic acid in a sample, which comprises; mixing at least one random primer (advantageously one or two primers) at least 4 nucleotides in length (advantageously 6-10 in length), with a sample nucleic acid, adding at least one NTP (advantageously 1-3 NTPs) having at least one binding species (advantageously 1-3 binding species) and optionally at least one NTP (advantageously 1-3 NTPs), having at least one detectable species (advantageously 1-3 detectable species) and optionally at least one NTP (advantageously 3-5 NTPs), adding at least one nucleic acid polymerase (advantageously 1-3 polymerases), incubating this mixture under conditions which allow the at least one nucleic acid polymerase to be active, contacting this mixture with at least one solid phase (advantageously 1-3 solid phases), detecting and/or quantitating the amount of nucleic acid in the sample by detecting and/or quantitating the amount of the at least one detectable species or the amount of the at least one binding species bound to said solid phase.</p>
    <p>This invention also includes a method for the detection and/or quantitation of nucleic acid in a sample, which comprises: mixing at least one labeled random primer (advantageously one or two primers) at least 4 nucleotides in length (advantageously 6-10 in length), having at least one binding species (advantageously 1-3 binding species) and optionally at least one detectable species (advantageously 1-3 detectable species), with a sample nucleic acid, adding at least one nucleic acid ligase (advantageously 1-3 ligase), adding at least one NTP (advantageously 1-3 NTPs), incubating this mixture under conditions which allow said at least one nucleic acid ligase to be active, contacting this mixture with at least one solid phase (advantageously 1-3 solid phases), and detecting and/or quantitating the amount of nucleic acid in the sample by detecting and/or quantitating the amount of the at least one detectable species or the amount of the at least one binding species bound to the solid phase.</p>
    <p>This invention also includes a method for the detection and/or quantitation of nucleic acid in a sample, which comprises; mixing at least one labeled random primer (advantageously one or two primers) at least 4 nucleotides in length (advantageously 6-10 in length), having at least one binding species (advantageously 1-3 binding species) and optionally at least one detectable species (advantageously 1-3 detectable species), with a sample nucleic acid, adding at least one nucleic acid ligase (advantageously 1-3 ligase), and at least one nucleic acid polymerase (advantageously 1-3 polymerase), adding at least one NTP (advantageously 3-5 NTPs), incubating this mixture conditions which allow the at least one nucleic acid ligase and at least one nucleic acid polymerase to be active, contacting this mixture with at least one solid phase (advantageously 1-3 solid phases), and detecting and/or quantitating the amount of nucleic acid in the sample by detecting and/or quantitating the amount of the at least one detectable species or the amount of the at least one binding species bound to the solid phase.</p>
    <p>This invention also includes a method for the detection and/or quantitation of nucleic acid in a sample, which comprises: mixing at least one random primer (advantageously one or two primers) at least 4 nucleotides in length (advantageously 6-10 in length), having at least one detectable species (advantageously 1-3 detectable species), with a sample nucleic acid, adding at least one NTP (advantageously 1-3 NTPs), having at least one binding species (advantageously 1-3 binding species) and optionally at least one NTP (advantageously 3-5 NTPs), adding at least one nucleic acid polymerase (advantageously 1-3 polymerase), incubating this mixture under conditions which allow said at least one nucleic acid polymerase to be active and quantitating the amount of nucleic acid in the sample by detecting and/or quantitating the amount of the at least one detectable species or the amount of the at least one binding species.</p>
    <p>This invention also includes a method for the detection and/or quantitation of nucleic acid in a sample, which comprises; mixing at least one random primer (advantageously one or two primers) at least 4 nucleotides in length (advantageously 6-10 in length), having at least one binding species (advantageously 1-3 binding species), with a sample nucleic acid, adding at least one NTP (advantageously 1-3 NTPs), having at least one detectable species (advantageously 1-3 detectable species), and optionally at least one NTP (advantageously 3-5 NTPs), adding at least one nucleic acid polymerase (advantageously 1-3 polymerase), incubating this mixture under conditions which allow said at least one nucleic acid polymerase to be active and quantitating the amount of nucleic acid in the sample by detecting and/or quantitating the amount of the at least one detectable species or the amount of the at least one binding species.</p>
    <p>This invention also includes a method for the detection and/or quantitation of nucleic acid in a sample, which comprises; mixing at least one random primer (advantageously one or two primers) at least 4 nucleotides in length (advantageously 6-10 in length), with a sample nucleic acid, adding at least one NTP (advantageously 1-3 NTPs) having at least one binding species (advantageously 1-3 binding species), and optionally at least one NTP (advantageously 1-3 NTPs) having at least one label (advantageously 1-3 labels), and optionally at least one NTP (advantageously 3-5 NTPs), adding at least one nucleic acid polymerase (advantageously 1-3 polymerase), incubating this mixture under conditions which allow the at least one nucleic acid polymerase to be active and quantitating the amount of nucleic acid in the sample by detecting and/or quantitating the amount of said at least one label or the amount of the at least one binding species.</p>
    <p>This invention also includes a method for the detection and/or quantitation of nucleic acid in a sample, which comprises: mixing at least one labeled random primer (advantageously one or two primers) at least 4 nucleotides in length (advantageously 6-10 in length), having at least one binding species (advantageously 1-3 binding species), and optionally at least one detectable species (advantageously 1-3 detectable species), with a sample nucleic acid, adding at least one nucleic acid ligase (advantageously 1-3 ligase), adding at least one NTP (advantageously 1-3 NTPs), incubating this mixture under conditions which allow the at least one nucleic acid ligase to be active and quantitating the amount of nucleic acid in the sample by detecting and/or quantitating the amount of the at least one detectable species or the amount of the at least one binding species.</p>
    <p>This invention also includes a method for the detection and/or quantitation of nucleic acid in a sample, which comprises; mixing at least one labeled random primer (advantageously one or two primers) at least 4 nucleotides in length (advantageously 6-10 in length) having at least one binding species (advantageously 1-3 binding species) and optionally at least one detectable species (advantageously 1-3 detectable species), with a sample nucleic acid, adding at least one nucleic acid ligase (advantageously 1-3 ligase), and at least one nucleic acid polymerase (advantageously 1-3 polymerase), adding at least one NTP (advantageously 3-5 NTPs), incubating this mixture under conditions which allow the at least one nucleic acid ligase and at least one nucleic acid polymerase to be active and quantitating the amount of nucleic acid in the sample by detecting and/or quantitating the amount of the at least one detectable species or the amount of the at least one binding species.</p>
    <p>This invention also includes a kit comprising; a vial containing at least one random primer (advantageously one or two primers) at least 4 nucleotides in length (advantageously 6-10 in length) having at least one detectable species (advantageously 1-3 detectable species), and containing at least one NTP (advantageously 1-3 NTPs), having at least one binding species (advantageously 1-3 binding species) and optionally at least one NTP (advantageously 3-5 NTPs), a vial containing at least one nucleic acid polymerase and a vial containing at least one solid phase (advantageously 1-3 solid phases).</p>
    <p>This invention also includes a kit comprising; a vial containing at least one random primer (advantageously one or two primers) at least 4 nucleotides in length (advantageously 6-10 in length) having at least one detectable species (advantageously 1-3 detectable species), a vial containing at least one NTP (advantageously 1-3 NTPs), having at least one binding species (advantageously 1-3 binding species) and optionally at least one NTP (advantageously 3-5 NTPs), a vial containing at least one nucleic acid polymerase and a vial containing at least one solid phase (advantageously 1-3 solid phases).</p>
    <p>This invention also includes a kit comprising; a vial containing at least one random primer (advantageously one or two primers) at least 4 nucleotides in length (advantageously 6-10 in length) having at least one binding species (advantageously 1-3 binding species), and containing at least one NTP (advantageously 1-3 NTPs), having at least one detectable species (advantageously 1-3 detectable species), and optionally at least one NTP (advantageously 3-5 NTPs), a vial containing at least one nucleic acid polymerase and a vial containing at least one solid phase (advantageously 1-3 solid phases).</p>
    <p>This invention also includes a kit comprising; a vial containing at least one random primer (advantageously one or two primers) at least 4 nucleotides in length (advantageously 6-10 in length) having at least one binding species (advantageously 1-3 binding species), a vial containing at least one NTP (advantageously 1-3 NTPs), having at least one detectable species (advantageously 1-3 detectable species), and optionally at least one NTP (advantageously 3-5 NTPs), a vial containing at least one nucleic acid polymerase and a vial containing at least one solid phase (advantageously 1-3 solid phases).</p>
    <heading>DETAILED DESCRIPTION OF THE INVENTION</heading> <p>The Tag-DNA assay of the subject invention improves on current methodology in several aspects. Tag-DNA assay is not as technically challenging or labor intensive compared to the assays currently used for detection of 10 pg amounts of DNA. Results from the assays that can accurately detect 10 pg of DNA have lengthy protocols (8 hrs-5 days) as compared to Tag-DNA assay (4 hrs). The Tag-DNA assay has a greater dynamic range for detection of DNA (5-10,000 pg) at a 60 minute reaction time than any of the current assays. The range of detecting greater amounts of DNA may be tailored to the requirements of the users by altering the enzyme incubation time. The subject assay can detect DNA larger than 100 base pairs regardless of base sequence or species of DNA. This is in contrast to slot-blot, which detects sequences that bind to the probe, or the threshold system which can only accurately detect DNA greater that 872 base pairs. Additionally, the generation of the signal in the Tag-DNA assay does not require radioactivity or incubation with a signal amplifier or enzyme substrate. Another advantage of the subject method is based on the ability to recount samples initially determined to be off the range of the standard curve. A sample will need to be re-run on slot-blot and threshold if a sample is out of the quantitation range of the standard curve. Thus, Tag-DNA improves on the speed and accuracy for determining the amount of DNA in the sample. Additionally, the results are generated faster due to the ease of the protocol and the short incubation time of the enzyme reaction.</p>
    <p>The basic format for the assay of the subject invention is unique because the protocol utilizes an oligonucleotide having a detectable species (eg ruthenylated hexamer) as a primer for incorporating NTP(s) having a binding species (eg biotin-dNTP) as a capture moiety in a DNA template dependent manner. Specifically, random oligonucleotide hexamers covalently labeled with ruthenium at the 5′ end are boiled and snapped-cooled in the presence of template DNA. DNA polymerase I (exonuclease Klenow fragment) is used to covalently incorporate biotin-dNTP, the capture moiety, directly to the ruthenylated-labeled hexamer. Free hexamers and biotin-dNTP are eliminated by a column purification step. The labeled DNA product is then captured using streptavidin coated magnetic beads and analyzed (eg using an ORIGEN analyzer, IGEN Inc.). Unknown concentrations of DNA are extrapolated from a standard curve generated by the Multicalc Wallac program using a 4 or 5 parameter logistic algorithm. Thus the assay of the subject invention not only detects low picogram amounts of DNA but also overcomes the disadvantages of current methodologies.</p>
    <p>In addition to residual DNA detection, this assay format has a multitude of other potential application uses. One application is for the general quantitation of total DNA in the field of molecular biology. The methods currently available for determination of DNA are biased for specific bases. The assay of the subject invention does not discriminate between bases nor the origin or nature of the DNA (single vs. double stranded). Another format for random priming that has resulted in a signal differential involves use of polymerase to incorporate digoxygenin-dUTP onto a biotinylated random oligonucleotide hexamer which is detected by ruthenylated anti-digoxigenin antibody. Thus, the format described in this application can be adapted to a variety of molecular biology applications.</p>
    <p>Detectable Species and Binding Species</p>
    <p>A moiety or label is a binding species and/or a detectable species. Any binding species is also a detectable moiety as in the case of biotin or digoxigenin which can be a binding species and a detectable species. This is due to the ability of the binding species to be detected by binding another species which itself contains either a signal generating system or a directly detectable species. In certain cases the detectable species is not a binding species as in the case of a radioactive label, ie <sup>32</sup>p labeled at the 5′ of the random primers. Thus, all binding species can be detectable species but not all detectable can be binding species. Those of ordinary skill in the art recognize that numerous possible detectable and binding species can be used in the present invention. Examples of binding species are biotin, avidin, streptavidin, antibodies, antigens, lectins, receptors, ligands, hormones, nucleic acid sequences, mimitopes, nucleic acid base pairing linear polymers (such as peptide nucleic acid, PNA). Examples of directly detectable species are radioactive molecules, fluorescent molecules, enzymes, electrochemiluminescent molecules, electrochemically active molecules, Ru(bpy)<sub>3</sub> <sup>2+</sup>, luminescent molecules, colored molecules, particles such as latex and colloidal gold.</p>
    <p>Primers</p>
    <p>Primers are polymer sequences capable of binding to nucleic acid sequences and directing the activity of nucleic acid template dependent enzymes such as polymerases, ligases, integrases, nucleases and recombinases in combination with nucleic acid templates. Primers are typically made synthetically to contain natural chemistries of the backbone and bases of nucleic acids. In addition to these synthetically made but natural nucleic acid sequences, modifications are commonly made to introduce moieties such as binding species for example biotin, digoxigenin, and also detectable moieties such as fluorescein, rhodamine, acridine, and Ru(bpy)<sub>3</sub> <sup>2+</sup>. Also the introduction of unnatural bases such as inosine, propyne dC, propyne dU, 5-bromo dC, 5-iodo dU, 5-fluoro dU, O-6 methyl dG, 7-deaza dG, N-6 methyl-2′-dA and additional modifications can be made which allow further chemistry such as introduction of amino and thiol groups within the sequence and at the ends of the sequences. It is also known that the primers need not be made only of naturally occurring bases and or backbone chemistries and that a number of unnatural chemistries have been used to make polymeric sequences which are able to bind to natural nucleic acid sequences. Some of these unnatural sequences are able to bind with greater affinity than the natural sequence as is the case of PNA (Perseptive biosystems, Framingham, Mass.) which is a peptide nucleic acid backbone. Typically in the case of these unnatural sequences they are not able to function as templates or primers in nucleic acid template dependent enzymatic reactions. They are able to function as primers when extended with a single natural base of nucleic acid. Thus these hybrid sequences of natural and unnatural elements are able to function as primers when combined with template and polymerases or ligase or other nucleic acid template dependent enzymes. It is also known that certain unnatural chemistries do not affect the ability of the primers to allow nucleic acid template dependent enzyme activities such as in the case of the phosphorothioate backbone chemistries which are able to activate RNases on hybridization to a RNA template. Examples of other alternative chemistries are available from various companies such as Oligo's etc, Wilsonville, Oreg. and Clontech, San Diego, Calif. Primers can also be made or isolated from biological sources such as calf thymus which is used as a source of DNA. Typically natural DNA is subjected to a fragmentation to generate a mixture of fragment lengths which are then fractionated to yield the desired primer lengths. One example of a fragmentation method is the use of a DNase I enzymatic reaction. Primers made in this way are considered to be random primers. These random primers can be from 4-400 bases long, advantageously 4-40 and most advantageously 6-10 bases long. Typically these random primers are from 4-100 bases long and may or may not include binding and/or detectable species. It is a advantageous embodiment of the present invention that the primer sequence is 5′ Ru(bpy)<sub>3</sub> <sup>2+</sup>-(N)n, where n is 4-40 bases. Typically the primers of the present invention is 5′ Ru(bpy)<sub>3</sub> <sup>2+</sup>-(N)n, where n is 6-10 bases.</p>
    <p>Random primers are typically made synthetically by the use of mixtures of the four bases A, C, G, T (or U in place of T) during synthesis. This protocol results in a sequence of the formula 5′(N)n, where n=4-400, where N is A, C, G, T (or U in place of T). The random primer can contain elements in the linear polymer which are not random such as a labeling moiety ie Ru(bpy)<sub>3</sub> <sup>2+</sup>. Numerous modifications to the basic random primer core sequence can also be considered to be random primers. The types of modifications which can be considered are labeling with moieties as described above, the inclusion of unnatural bases as described above and also the use of un-natural backbone chemistries.</p>
    <p>Solid Phases</p>
    <p>Solid phases are used in numerous assay formats and are used as a means of capturing or binding complexes formed by the various binding species in the assay. This capture on to a solid phase is typically used as a means for separation of the specific complexes formed in an assay from the excess label species. This separation allows the detection of the specifically formed complexes in an assay. In a typical assay; for example an ELISA, the complex is captured directly on to the surface of a coated plastic tube and this tube is then washed to remove any excess enzyme label. Following this wash step the substrates are then added and the specific complex is detected and quantitated by the level of enzyme activity which is related to the level of color, fluorescence or luminescence generated by the action of the enzyme (label) used in this assay format. It is known that many solid phases can be used typically fibers, fibrils, plastic surfaces, plastic beads, magnetic beads, plastic tubes, gold surfaces, metal surfaces, metal beads, metal colloids, and other colloids.</p>
    <p>Proximity Formats</p>
    <p>In addition to the detection formats and assay formats described above which are based on the use of a solid phase to enable the analysis of formed complexes. Methods also exist for the detection of formed complexes which make use of the enhanced proximity of labels and binding species caused by the action of the template depended nucleic acid synthase enzymes of the present invention. Methods which make use of proximity to allow detection and quantitation are fluorescent energy transfer, scintillation proximity assay systems (Amersham, Dover, Del.), enzyme channeling assay systems (Litman et al (1980) Anal Chem, 106, 223), luminescent oxygen channeling assay systems (Ullman et al Clin Chem (1996) 42, 1518). These detection methods are also contemplated for the detection of nucleic acid using the methods of the invention to incorporate labels or binding species which are able to generate signals dependent on proximity of the incorporated groups into the products of the template dependent nucleic acid synthase enzymes.</p>
    <p>Enzymes</p>
    <p>The subject invention requires an enzyme which is active and activated by the presence of a primer and a template nucleic acid. Examples of such enzymes include, Taq DNA polymerase, T4 DNA polymerase, Klenow fragment (3′-5′), Pfu DNA polymerase, Exo-Pfu DNA polymerase, <i>E. coli </i>DNA polymerase I, Klenow fragment of DNA polymerase I, MMLV reverse transcriptase, AMV reverse transcriptase, Pfu DNA ligase, T4 DNA ligase, Taq DNA ligase, T4 RNA ligase, and <i>E. coli </i>DNA ligase. The conditions which are required for the optimal activity of these enzymes is well known to the art and buffers which support the activity of the these enzymes is typically supplied with the enzymes from the suppliers. In this invention activity of the polymerase enzymes is the incorporation of NTP's in a template dependent fashion not any of the other enzymatic activities of the polymerases such as various nuclease like activities such as 3′-5′ exo-nuclease and 5′-3′ exonuclease. Also in this invention activity of the ligase enzymes is the linkage of oligonucleotides in a template dependent fashion.</p>
    <p>Buffers</p>
    <p>The buffer conditions for the desired enzyme activities of the subject invention have an expected wide range. This is due to the wide variety of organisms with enzymes which have the desired properties of nucleic acid template dependent activity. In addition the use of polymerases, transcriptases and ligases with differing enzymatic systems and substrates also allows for a great variety of buffer conditions within the scope of the subject invention. These enzymes are well documented by the various vendors and in the literature regarding the potential scope of buffer conditions under which the enzymes of the subject inventions are active. For example the pH may be between 5.5 and 9.5, the NTP concentrations may vary from 1 pM to 10 mM. In the case of the Mg<sup>2+ </sup>ion concentration the range is also wide from 0.05 mM and 500 mM and with certain enzymes the Mg ion can be replaced with Mn. Many enzymes like to have a reducing agent (eg DTT and/or 2-mercaptoethanol) present to maintain full activity for longer incubations the concentration can range from 0 to 500 mM. Also the sum of the molarities for the nucleic acid template dependent enzymes can be between 1 mM and 600 mM this can be made up only of the pH buffer, Mg salts, NTPs and reducing agents but may typically contain other salts (eg NaCl and/or KCl) to raise the ionic strength of the buffer.</p>
    <p>Test Samples</p>
    <p>The present invention provides a method for the detection and quantitation of nucleic acids using random primers. The nucleic acid of the present invention includes DNA, RNA in both single stranded from and double stranded forms. In an advantageous embodiment, the nucleic acid is DNA. These nucleic acids can be synthetic or natural, ie derived from a living organisms and viruses. The nucleic acid detectable by these methods are from 4 nucleotides or longer. There is no upper limit to the size of nucleic acids detectable using the present invention. In an advantageous embodiment, the size of the nucleic acid is at least 50 nucleotides.</p>
    <p>Typically the nucleic acid to be detected is present in a biological samples or material derived from a biological source such as an organism, a fermentation, a culture, a plant, an animal, cells. Other examples of biological samples are cells, organs, tissues, body parts, to biological fluids. Examples of biological fluids are culture media, a fermentation, blood, mucous, urine, plasma, and saliva. In an advantageous embodiment, the sample is a purified biological sample intended for use as a pharmaceutical for example human growth hormone, insulin, G-CSF etc.</p>
    <p>Assay Formats</p>
    <p>Detection of DNA Using Primers with a Detectable Species</p>
    <p>In one embodiment of the present invention random primers containing a detectable species, are added to purified samples with unknown levels of DNA. This mixture is then heated to denature the DNA followed by an annealing step which allows the primers to hybridize to the sample DNA. Following this step, the polymerase buffer is added which includes nucleotide triphosphates at least one of which is labeled with a moiety which is a binding species. A DNA dependent polymerase is then added. This mixture is then incubated to allow the polymerase to extend the primers in a template dependent manner which results in the incorporation of labeled NTPs into a primer extended product which has copied the sample DNA in a template dependent fashion. The resultant product of the polymerase extension of the primer has now covalently linked the detectable moiety of the primer to the binding moiety linked to the NTPs. This product of the primer, NTP polymerase reaction is then captured on to a solid phase and the detectable moiety is then detected. The level of the detectable moiety is then used to determine the level of DNA in the samples.</p>
    <p>Detection of RNA Using Primers with a Detectable Species and Reverse Transcriptases</p>
    <p>In another embodiment of the present invention, random primers containing a detectable species, is added to purified samples with unknown levels of RNA. This mixture is heated to denature the RNA followed by an annealing step which allows the primers to hybridize to the sample RNA. Following this step, reverse transcriptase buffer is added which includes nucleotide triphosphates at least one of which is labeled with a moiety which is a binding species. A reverse transcriptase is then added. This mixture is then incubated to allow the polymerase to extend the primers in a template dependent manner which results in the incorporation of labeled NTPs into a primer extended product which has copied the sample RNA in a template dependent fashion. The resultant product of the reverse transcriptase extension of the primer has now covalently linked the detectable moiety of the primer to the binding moiety linked to the NTPs. This product of the primer, NTP reverse transcriptase reaction is then captured on to a solid phase and the detectable moiety is then detected. The level of the detectable moiety is then used to determine the level of RNA in the samples.</p>
    <p>Detection of DNA Using Primers with a Binding Species and DNA Polymerases</p>
    <p>In another embodiment of the present invention, random primers containing a binding species, is added to purified samples with unknown levels of DNA. This mixture is heated to denature the DNA followed by an annealing step which allows the primers to hybridize to the sample DNA. Following this step, the polymerase buffer is added which includes nucleotide triphosphates at least one of which is labeled with a moiety which is detectable. A DNA dependent polymerase is then added. This mixture is then incubated to allow the polymerase to extend the primers in a template dependent manner which results in the incorporation of labeled NTPs into a primer extended product which has copied the sample DNA in a template dependent fashion. The resultant product of the polymerase extension of the primer has now covalently linked the binding moiety of the primer to the detectable moiety linked to the NTPs. This product of the primer, NTP polymerase reaction is then captured on to a solid phase and the detectable moiety is then detected. The level of the detectable moiety is then used to determine the level of DNA in the samples.</p>
    <p>Detection of DNA Using Primers and DNA Polymerases</p>
    <p>In another embodiment of the present invention, random unlabeled primers are added to purified samples with unknown levels of DNA. This mixture is heated to denature the DNA followed by an annealing step which allows the primers to hybridize to the sample DNA. Following this step the polymerase buffer is added which includes nucleotide triphosphates at least one of which is labeled with a moiety which is detectable and at least one of which is labeled with a moiety which is a binding species and a DNA dependent polymerase. This mixture is then incubated to allow the polymerase to extend the primers in a template dependent manner which results in the incorporation of labeled NTPs into a primer extended product which has copied the sample DNA in a template dependent fashion. The resultant product of the polymerase extension of the primer has now covalently linked the binding moiety of the NTPs to the detectable moiety linked to the NTPs. This product of the primer, NTPs and polymerase reaction is then captured on to a solid phase and the detectable moiety is then detected. The level of the detectable moiety is then used to determine the level of DNA in the samples.</p>
    <p>Detection of DNA Using Primers with a Detectable Species and DNA Ligases</p>
    <p>In another embodiment of the present invention, random primers labeled with a binding species at the 5′ end of the sequence, and random primers labeled with a moiety which is a detectable species at the 3′ end of the sequence are added to purified samples with unknown levels of DNA. This mixture is heated to denature the DNA followed by an annealing step which allows the primers to hybridize to the sample DNA. Following this step the ligase buffer and a DNA dependent ligase are added. This mixture is then incubated to allow the ligase to ligate the primers in a template dependent manner which results in the covalent linkage of the labeled primers in a template dependent fashion. The resultant product of the ligase reaction covalently linked the binding moiety of one primer to the detectable moiety linked to the other primer. This product of the primer, ligase reaction is then captured on to a solid phase and the detectable moiety is then detected. The level of the detectable moiety is then used to determine the level of DNA in the samples. In this format DNA polymerase may also be used with the DNA ligases.</p>
    <p>Detection of DNA Using Primers with Ru(bpy)<sub>3</sub> <sup>2+</sup> </p>
    <p>In an advantageous embodiment of the present invention, random hexamer primers labeled with an ECL label of the formula 5′ Ru(bpy)<sub>3</sub> <sup>2+</sup>-(N)6, is added to purified samples with unknown levels of DNA. This mixture is heated to denature the DNA followed by an annealing step which allows the primers to hybridize to the sample DNA. Following this step the polymerase buffer is added which includes a mixture of dNTPs, including at least one biotin-dNTP and klenow polymerase. This mixture is then incubated to allow the klenow polymerase to extend the primers in a template dependent manner which results in the incorporation of biotin-dNTPs into a primer extended product which has copied the sample DNA in a template dependent fashion. The resultant product of the polymerase extension of the primer has now covalently linked the 5′ Ru(bpy)<sub>3</sub> <sup>2+</sup>-(N)n, where n=6-36, primer to the biotin-NTPs. This product of the primer, NTP polymerase reaction is then captured on to a streptavidin bead and analyzed in an ORIGEN analyzer (IGEN, Gaithersburg, Md.). The signal detected is then used to determine the level of DNA in the samples based on a standard curve generated using known amounts of DNA in the above assay protocol.</p>
    <p>It will be understood by those skilled in the art that these embodiments can be extended to other methods which still fall within the scope of the invention which is based on the use of a nucleic acid template dependent enzyme in combination with a primer to generate a molecular species via the action of the enzyme which is dependent on the levels of nucleic acid in the sample.</p>
    <heading>EXAMPLES</heading> <heading>Example 1</heading> <p>Random Oligonucleotide Priming Using Poly A or Biotin Labeled Hexamers</p>
    <p>Random hexamers are oligonucleotides (NNNNNN) that are synthesized in a random fashion with respect to sequence order. The last nucleotide placed on by the synthesizer can be a dNTP that is modified. Modifications include a fluoresceinated, biotinylated or ruthenylated containing moiety. Several different forms of random hexamers were evaluated in order to determine the best format for hexamers for detecting low picogram amounts of DNA.</p>
    <p>A. Random oligonucleotide hexamers containing a poly A sequence—Random oligonucleotide hexamers containing a poly A sequence ((A)<sub>n</sub>-NNNNNN) were utilized as template for DNA polymerase to incorporate biotin-dNTP into DNA. The product was captured by oligo-dT magnetic beads and the signal generated by a ruthenylated anti-biotin antibody or ruthenylated streptavidin. Alternatively, digoxigenin-dNTP was incorporated and the signal generated by a ruthenylated anti-digoxigenin antibody. Both permutations generated a signal that detects nanogram amounts of DNA.</p>
    <p>B. Random oligonucleotide hexamers labeled with biotin were utilized as a primer for DNA polymerase to incorporate digoxigenin-dNTP into the template DNA. The labeled DNA was captured by streptavidin beads and the signal generated by a ruthenylated anti-digoxigenin antibody. These experiments showed that 100 ng DNA yielded a 100 fold increase in signal over background. A significant signal differential was evident by this random priming format.</p>
    <p>C.</p>
    <heading>Example 2</heading> <p>Random Oligonucleotide Priming Using Ruthenylated Labeled Hexamers.</p>
    <p>The ruthenium moiety is added to the hexamer by means of phosphoramidite chemistry. The ruthenium is the moiety that generates the electrochemoluminescent signal detected by the ORIGEN analyzer.</p>
    <p>A. Annealing Conditions. Random oligonucleotide priming with ruthenylated hexamers was evaluated using different conditions for annealing hexamers to DNA. Using assay format 1 and 2 did not result in a significant difference between positive and negative control.</p>
    <p>1. Target DNA was boiled and snap-cooled on ice. Biotin-ddUTP and the hexamer was then added and extended with exonuclease containing klenow fragment.</p>
    <p>2. Target DNA was boiled with hexamers then slowly cooled to room temperature. Biotin-ddUTP was incorporated into the annealed hybrid during the polymerase reaction.</p>
    <p>3. DNA diluted in water was boiled and snap-cooled with the ruthenylated hexamers. Biotin-dNTPs and biotin-ddUTP's were incorporated by klenow as the capture moiety. This assay format resulted in 100 fold signal differential between 4 ng and negative control. Although inclusion of manganese increases the polymerase affinity for ddNTP, the signal differential was significantly reduced. Therefore, use of ddUTP was eliminated and the biotin-dNTP in the reaction mix was the only biotin species used for the assay. This assay format resulted in 200 fold differential between negative control and 10 ng of target DNA. A standard curve was generated from target DNA in order to determine the sensitivity of this assay format. Picogram sensitivity was obtained only when the ruthenylated hexamers are added directly to each standard.</p>
    <p>4. Taq-polymerase was tested using 10 uM dNTP (30% biotin-ATP). Use of PCR amplification of DNA with taq-polymerase resulted in a 1.5X signal differential with 10 pg of DNA.</p>
    <p>Out of the four different permutations, # 3 was selected because of the potential for detecting low picogram amounts of target DNA. In addition to ruthenylated hexamers, ruthenylated random oligonucleotide decamers and octamers were also evaluated using this assay format. The results from these experiments did not yield a signal differential comparable to the hexamer.</p>
    <p>B. Determination of the Concentration of Ruthenylated Hexamers. The ruthenylated hexamers were procured from Midland Certified Reagent Company (Midland, Tex.). The concentration of ruthenylated hexamer is determined using the extinction coefficient of ruthenium at 455 nm. (13,700). The stock hexamer was aliquoted at 10 ug/ul and stored at −20° C. Prior to use, the ruthenylated hexamers were diluted in 1 mM Tris-HCL (pH8) to a final concentration of 2 ug/ul.</p>
    <p>C. Titration of the Ruthenylated Hexamers. Several experiments were performed to decrease the background signal. The concentration of hexamer was tested at three different concentrations (0.5,1, 1.5 and 2 ug/reaction) in order to determine the lowest concentration of hexamer that would not interfere with the signal differential. Out of all the different lots of hexamers used for this assay, lug of hexamer per reaction has been consistently showing the best signal differential. Hexamers that were not HPLC purified were compared to HPLC purified hexamers. These experiments showed that HPLC purification altered the characteristics of the assay and therefore non-HPLC purified hexamers were selected for the assay.</p>
    <heading>Example 3</heading> <p>Decreasing the Annealing Stringency Between Hexamer and DNA</p>
    <p>The annealing of template DNA to the hexamer was initially performed in water. In order to increase the binding of hexamer to the template DNA, several salts were tested including 2X klenow buffer, NaCl, LiCl, 100 mM Tris-HCl (pH 7.5 or 9), 1 mM Hepes (pH6.6) and dextran sulfate. Compared to water, Tris-HCl (1 mM, pH 8) yielded the most consistent differential between picogram amounts of DNA and the negative control.</p>
    <heading>Example 4</heading> <p>Selection of DNA Polymerase, Optimal Concentration of dNTP and Percentage of Biotin-dNTP</p>
    <p>Normal random oligonucleotide priming methodologies function by extending the hexamers and copying the template DNA in the presence of labeled dNTPs. Several polymerases were tested for use in the Tag-DNA assay:</p>
    <p>A. Taq-polymerase was tested using 200 uM dNTP (30% biotin-ATP). PCR was implemented to amplify target DNA with taq-polymerase. These results yielded a 1.5X signal differential with 10 pg of DNA.</p>
    <p>B. In the early stages of assay development, the exonuclease containing and deficient klenow DNA polymerase enzymes were tested. As stated in the background, this activity is responsible for replacing the incorporated biotin-dNTP with unmodified dNTP. The best signal differential was generated using the 3′-5′ exonuclease deficient klenow fragment because of the lack of proofreading activity. The signal differential with exonuclease containing enzyme was 24 fold whereas the exonuclease deficient enzyme gave a 66 fold differential with 5 ng of target DNA. Therefore, exonuclease deficient enzyme was selected for further use in the Tag-DNA assay.</p>
    <p>C. The optimal dNTP concentration for the klenow fragment was selected (8 μM). This was determined on experimental evaluation of several different concentrations of biotin-dNTP. The percent biotin-dNTP represents both biotin-dATP and biotin-dCTP. Using 5 ng of target DNA yielded the following results:</p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="offset" colwidth="42pt" align="left"> </colspec> <colspec colname="1" colwidth="56pt" align="center"> </colspec> <colspec colname="2" colwidth="119pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="2" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Condition</td>
                <td class="description-td">ECL Counts</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="2" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Negative control</td>
                <td class="description-td">  4393</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">50%</td>
                <td class="description-td">1411475</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">30%</td>
                <td class="description-td">1820840</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">10%</td>
                <td class="description-td">1280735</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> 5%</td>
                <td class="description-td"> 551261</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="2" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p>A. The percent of biotin-dNTP was determined as the concentration that gave the best signal differential between negative control and 10 pg of target DNA. The concentrations for the 10X dNTP mix are as follows:</p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="5"> <colspec colname="offset" colwidth="42pt" align="left"> </colspec> <colspec colname="1" colwidth="42pt" align="center"> </colspec> <colspec colname="2" colwidth="28pt" align="center"> </colspec> <colspec colname="3" colwidth="42pt" align="center"> </colspec> <colspec colname="4" colwidth="63pt" align="left"> </colspec> <thead> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="4" align="center" rowsep="1" class="description-td" colspan="5"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Stock</td>
                <td class="description-td">μL</td>
                <td class="description-td">Final (μM)</td>
                <td class="description-td">Source</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="4" align="center" rowsep="1" class="description-td" colspan="5"> </td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="6"> <colspec colname="1" colwidth="42pt" align="left"> </colspec> <colspec colname="2" colwidth="21pt" align="right"> </colspec> <colspec colname="3" colwidth="21pt" align="left"> </colspec> <colspec colname="4" colwidth="28pt" align="char" char="."> </colspec> <colspec colname="5" colwidth="42pt" align="center"> </colspec> <colspec colname="6" colwidth="63pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">dATP</td>
                <td class="description-td">10</td>
                <td class="description-td">mM</td>
                <td class="description-td">5.6</td>
                <td class="description-td">56</td>
                <td class="description-td">Gibco/BRL</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">(Gaithersburg, MD)</td>
              </tr> <tr class="description-tr"> <td class="description-td">dGTP</td>
                <td class="description-td">10</td>
                <td class="description-td">mM</td>
                <td class="description-td">8</td>
                <td class="description-td">80</td>
                <td class="description-td">Gibco/BRL</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">(Gaithersburg, MD)</td>
              </tr> <tr class="description-tr"> <td class="description-td">dTTP</td>
                <td class="description-td">10</td>
                <td class="description-td">mM</td>
                <td class="description-td">8</td>
                <td class="description-td">80</td>
                <td class="description-td">Gibco/BRL</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">(Gaithersburg, MD)</td>
              </tr> <tr class="description-tr"> <td class="description-td">dCTP</td>
                <td class="description-td">10</td>
                <td class="description-td">mM</td>
                <td class="description-td">5.6</td>
                <td class="description-td">56</td>
                <td class="description-td">Gibco/BRL</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">(Gaithersburg, MD)</td>
              </tr> <tr class="description-tr"> <td class="description-td">biotin-dATP</td>
                <td class="description-td">400</td>
                <td class="description-td">μM</td>
                <td class="description-td">60</td>
                <td class="description-td">24</td>
                <td class="description-td">Gibco/BRL</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">(Gaithersburg, MD)</td>
              </tr> <tr class="description-tr"> <td class="description-td">biotin-dCTP</td>
                <td class="description-td">400</td>
                <td class="description-td">μM</td>
                <td class="description-td">60</td>
                <td class="description-td">24</td>
                <td class="description-td">Gibco/BRL</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">(Gaithersburg, MD)</td>
              </tr> <tr class="description-tr"> <td class="description-td">DEPC water</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">852.8</td>
                <td class="description-td"> </td>
                <td class="description-td">Ambion</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="6" align="center" rowsep="1" class="description-td" colspan="6"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <heading>Example 5</heading> <p>Kinetic Properties of Exonuclease Deficient Klenow DNA Polymerase in Random Priming Reactions With Ruthenylated Hexamers</p>
    <p>In order to determine the highest sensitivity for quantitating picogram amounts of DNA, kinetic analysis was performed. These results suggested that the signal differential between negative control and 5 pg of DNA increases with longer incubation times. Furthermore, these results further suggest that higher amounts of DNA can be quantitated using this format. Because the goal was to develop an assay that detects low picogram amounts of DNA, the assay detailed in this protocol focuses on characterization of the 60 minute assay.</p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Mean Counts from ORIGEN Analyzer</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="5"> <colspec colname="1" colwidth="42pt" align="center"> </colspec> <colspec colname="2" colwidth="42pt" align="center"> </colspec> <colspec colname="3" colwidth="42pt" align="center"> </colspec> <colspec colname="4" colwidth="49pt" align="center"> </colspec> <colspec colname="5" colwidth="42pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">pg DNA</td>
                <td class="description-td">30 minutes</td>
                <td class="description-td">60 minutes</td>
                <td class="description-td">120 minutes</td>
                <td class="description-td">240 minutes</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="5" align="center" rowsep="1" class="description-td" colspan="5"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="5"> <colspec colname="1" colwidth="42pt" align="char" char="."> </colspec> <colspec colname="2" colwidth="42pt" align="char" char="."> </colspec> <colspec colname="3" colwidth="42pt" align="center"> </colspec> <colspec colname="4" colwidth="49pt" align="center"> </colspec> <colspec colname="5" colwidth="42pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">500,000</td>
                <td class="description-td">13294384</td>
                <td class="description-td">out of range</td>
                <td class="description-td">out of range</td>
                <td class="description-td">out of range</td>
              </tr> <tr class="description-tr"> <td class="description-td">250,000</td>
                <td class="description-td">16394352</td>
                <td class="description-td">out of range</td>
                <td class="description-td">out of range</td>
                <td class="description-td">out of range</td>
              </tr> <tr class="description-tr"> <td class="description-td">125,000</td>
                <td class="description-td">15241295</td>
                <td class="description-td">out of range</td>
                <td class="description-td">out of range</td>
                <td class="description-td">out of range</td>
              </tr> <tr class="description-tr"> <td class="description-td">31,000</td>
                <td class="description-td">12908362</td>
                <td class="description-td">17215729 </td>
                <td class="description-td">out of range</td>
                <td class="description-td">out of range</td>
              </tr> <tr class="description-tr"> <td class="description-td">16,000</td>
                <td class="description-td">6764104</td>
                <td class="description-td">10647597 </td>
                <td class="description-td">13249103  </td>
                <td class="description-td">out of range</td>
              </tr> <tr class="description-tr"> <td class="description-td">5,000</td>
                <td class="description-td">4394376</td>
                <td class="description-td">6886864 </td>
                <td class="description-td">9635308 </td>
                <td class="description-td">14839843 </td>
              </tr> <tr class="description-tr"> <td class="description-td">500</td>
                <td class="description-td">397842</td>
                <td class="description-td">922998</td>
                <td class="description-td">1560462 </td>
                <td class="description-td">3672890 </td>
              </tr> <tr class="description-tr"> <td class="description-td">100</td>
                <td class="description-td">120605</td>
                <td class="description-td">210523</td>
                <td class="description-td">394444 </td>
                <td class="description-td">945501</td>
              </tr> <tr class="description-tr"> <td class="description-td">50</td>
                <td class="description-td">60912</td>
                <td class="description-td">100712</td>
                <td class="description-td">159976 </td>
                <td class="description-td">416124</td>
              </tr> <tr class="description-tr"> <td class="description-td">10</td>
                <td class="description-td">22628</td>
                <td class="description-td"> 36665</td>
                <td class="description-td">62114</td>
                <td class="description-td">129247</td>
              </tr> <tr class="description-tr"> <td class="description-td">5</td>
                <td class="description-td">19005</td>
                <td class="description-td"> 27164</td>
                <td class="description-td">43696</td>
                <td class="description-td"> 89136</td>
              </tr> <tr class="description-tr"> <td class="description-td">0</td>
                <td class="description-td">14881</td>
                <td class="description-td"> 21477</td>
                <td class="description-td">26819</td>
                <td class="description-td"> 18982</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="5" align="center" rowsep="1" class="description-td" colspan="5"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <heading>Example 6</heading> <p>Elimination of Ruthenylated Hexamers Not Extended by the DNA Polymerase</p>
    <p>In order to maximize the signal to noise ratio between low picogram amounts of DNA and the negative control, there is a requirement for elimination of the ruthenylated hexamers. Elimination of the column step increases the signal from the negative control and therefore decreases the assay sensitivity. Several blocking agents such as (Denhart's, BSA, CHAPS, Tween-20 and assay diluent) did not efficiently block the background contributed by the ruthenylated hexamers. These results confirmed the requirement for a column purification step in the quantitation of low picogram amounts of target DNA. Quantitation of higher amounts of target DNA worked well using this format. Experiments also showed that Boeringer-Mannheim columns were advantageous for determining low picogram amounts of DNA as compared to other commercially available columns.</p>
    <heading>Example 7</heading> <p>Preparation of Samples for Tag-DNA Assay</p>
    <p>In order to identify the most efficient protocol for extracting small quantities of DNA from high protein samples, two DNA extraction kits were evaluated; WAKO and Puregene. Using several different mock buffers containing 70 mg/ml of BSA, the performance of these extraction protocols in the Tag-DNA assay was determined. In addition to glycogen, another co-precipitant (pellet paint; Novagen) to increase the yield of low picogram quantities of DNA was utilized. An additional wash step with 70% ethanol was also implemented to eliminate matrix interferences in the assay. The results from these experiments suggested that the WAKO kit works best not only to eliminate protein from sample but also to yield close to 100% recovery of the DNA from samples with low picogram amounts of DNA. Using this protocol, a large white precipitate that is not eliminated during wash steps and/or ethanol in sample prior to testing in the assay was observed to interfere with the generation of the signal.</p>
    <heading>Example 8</heading> <p>Tag-DNA Assay Protocol</p>
    <p>The protocol described in this example is for duplicate determinations made as a 3X solution. Hexamers at 2 μg/μL (0.5 μL/reaction or 1.5 μL/3X reactions; example 2) were aliquoted into siliconized tubes. Negative control [1 mM Tris-HCl (pH8)], DNA standards and samples (5 μL/reaction or 15 μL/3X reactions) were aliquoted into separate siliconized tubes, immediately capped, vortexed and centrifuged. The DNA standards are aliquoted from the lowest to highest concentration. The hexamer was annealed to the DNA by boiling the tubes at 100° C. for 3 minutes, and immediately followed by snap cooling on ice for 5 minutes.</p>
    <p>A master mix of the following components was prepared according to the number of reactions in the experiment as detailed in the following table:</p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Master Mix (1 X)</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="4"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="70pt" align="left"> </colspec> <colspec colname="2" colwidth="49pt" align="center"> </colspec> <colspec colname="3" colwidth="84pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">DEPC Water</td>
                <td class="description-td">1.5 μL</td>
                <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">dNTP Mix</td>
                <td class="description-td">1.0 μL</td>
                <td class="description-td">Described example 3</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">10X Klenow Buffer</td>
                <td class="description-td">1.0 μL</td>
                <td class="description-td">New England Biolabs</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Klenow (exonuclease-)</td>
                <td class="description-td">1.0 μL</td>
                <td class="description-td">New England Biolabs</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="3" align="center" rowsep="1" class="description-td" colspan="4"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p>The master mix (4.5 μL) is aliquoted into a new set of reaction tubes. The negative control, DNA standards, QC standards and samples (5.5 μL/reaction) were aliquoted into the PCR tubes containing the master mix and the reaction was allowed to incubate 60 minutes at 25° C. After incubation the biotin-dNTPs and the unextended and &lt;100 basepair extended tag-hexamers were eliminated using the Boehringer Mannheim High Pure PCR purification columns according to manufacturers instructions (see example 7). Following incubation, 200 μL of column binding buffer was added to each tube and applied to the column. The column was washed with 500 ul of wash buffer and the sample was eluted using 200 ul of elution buffer or 1XTE. The sample was then transferred to analyzer tubes and the product bound to streptavidin beads (100 ul of 0.1 mg/ml diluted in PBS-1; IGEN, Inc.) by a 30 minute incubation at room temperature while vortexing. The relative ECL counts were subsequently obtained on an ORIGEN analyzer.</p>
    <heading>Example 9</heading> <p>Quantitation of DNA and Preparation of DNA Standards</p>
    <p>A. Preparation of DNA standards: The concentration of the standard sheared calf thymus DNA (Gibco/BRL) was determined by optical density to be 9.3 ug/ul. This concentration was used to generate the standard curve by diluting DNA in 1 mM Tris (pH8) (Ambion). The dilution scheme is as follows:</p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="6"> <colspec colname="1" colwidth="28pt" align="center"> </colspec> <colspec colname="2" colwidth="35pt" align="center"> </colspec> <colspec colname="3" colwidth="35pt" align="center"> </colspec> <colspec colname="4" colwidth="35pt" align="center"> </colspec> <colspec colname="5" colwidth="42pt" align="center"> </colspec> <colspec colname="6" colwidth="42pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="6" align="center" rowsep="1" class="description-td" colspan="6"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">DNA</td>
                <td class="description-td">Vol. DNA</td>
                <td class="description-td">Vol. Tris</td>
                <td class="description-td"> </td>
                <td class="description-td">[DNA]</td>
              </tr> <tr class="description-tr"> <td class="description-td">Tube #</td>
                <td class="description-td">Source</td>
                <td class="description-td">(μL)</td>
                <td class="description-td">(μL)</td>
                <td class="description-td">Final [DNA]</td>
                <td class="description-td">in Std.</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="6" align="center" rowsep="1" class="description-td" colspan="6"> </td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="8"> <colspec colname="1" colwidth="28pt" align="center"> </colspec> <colspec colname="2" colwidth="35pt" align="center"> </colspec> <colspec colname="3" colwidth="35pt" align="char" char="."> </colspec> <colspec colname="4" colwidth="35pt" align="center"> </colspec> <colspec colname="5" colwidth="21pt" align="right"> </colspec> <colspec colname="6" colwidth="21pt" align="left"> </colspec> <colspec colname="7" colwidth="21pt" align="right"> </colspec> <colspec colname="8" colwidth="21pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">B</td>
                <td class="description-td">100</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">100</td>
                <td class="description-td">ng/μL</td>
                <td class="description-td">500</td>
                <td class="description-td">ng</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">ng/μL</td>
              </tr> <tr class="description-tr"> <td class="description-td">2</td>
                <td class="description-td">100</td>
                <td class="description-td">10</td>
                <td class="description-td">490</td>
                <td class="description-td">2</td>
                <td class="description-td">ng/μL</td>
                <td class="description-td">10</td>
                <td class="description-td">ng</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">ng/μL</td>
              </tr> <tr class="description-tr"> <td class="description-td">3</td>
                <td class="description-td">2 ng/μL</td>
                <td class="description-td">250</td>
                <td class="description-td">250</td>
                <td class="description-td">1</td>
                <td class="description-td">n/μL</td>
                <td class="description-td">5</td>
                <td class="description-td">ng</td>
              </tr> <tr class="description-tr"> <td class="description-td">4</td>
                <td class="description-td">1 ng/μL</td>
                <td class="description-td">100</td>
                <td class="description-td">400</td>
                <td class="description-td">200</td>
                <td class="description-td">pg/μL</td>
                <td class="description-td">1</td>
                <td class="description-td">ng</td>
              </tr> <tr class="description-tr"> <td class="description-td">5</td>
                <td class="description-td">200</td>
                <td class="description-td">250</td>
                <td class="description-td">250</td>
                <td class="description-td">100</td>
                <td class="description-td">pg/μL</td>
                <td class="description-td">500</td>
                <td class="description-td">pg</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">pg/μL</td>
              </tr> <tr class="description-tr"> <td class="description-td">6</td>
                <td class="description-td">100</td>
                <td class="description-td">100</td>
                <td class="description-td">400</td>
                <td class="description-td">20</td>
                <td class="description-td">pg/μL</td>
                <td class="description-td">100</td>
                <td class="description-td">pg</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">pg/μL</td>
              </tr> <tr class="description-tr"> <td class="description-td">7</td>
                <td class="description-td">20</td>
                <td class="description-td">250</td>
                <td class="description-td">250</td>
                <td class="description-td">10</td>
                <td class="description-td">pg/μL</td>
                <td class="description-td">50</td>
                <td class="description-td">pg</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">pg/μL</td>
              </tr> <tr class="description-tr"> <td class="description-td">8</td>
                <td class="description-td">10</td>
                <td class="description-td">100</td>
                <td class="description-td">400</td>
                <td class="description-td">2</td>
                <td class="description-td">pg/μL</td>
                <td class="description-td">10</td>
                <td class="description-td">pg</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">pg/μL</td>
              </tr> <tr class="description-tr"> <td class="description-td">9</td>
                <td class="description-td">2 pg/μL</td>
                <td class="description-td">250</td>
                <td class="description-td">250</td>
                <td class="description-td">1</td>
                <td class="description-td">pg/μL</td>
                <td class="description-td">5</td>
                <td class="description-td">pg</td>
              </tr> <tr class="description-tr"> <td class="description-td">10 </td>
                <td class="description-td">1 pg/μL</td>
                <td class="description-td">100</td>
                <td class="description-td">400</td>
                <td class="description-td">0.2</td>
                <td class="description-td">pg/μL</td>
                <td class="description-td">1</td>
                <td class="description-td">pg</td>
              </tr> <tr class="description-tr"> <td class="description-td">blank</td>
                <td class="description-td">none</td>
                <td class="description-td">0</td>
                <td class="description-td">500</td>
                <td class="description-td">0</td>
                <td class="description-td"> </td>
                <td class="description-td">0</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="8" align="center" rowsep="1" class="description-td" colspan="8"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">DNA Quality Controls</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="6"> <colspec colname="1" colwidth="28pt" align="center"> </colspec> <colspec colname="2" colwidth="42pt" align="center"> </colspec> <colspec colname="3" colwidth="35pt" align="center"> </colspec> <colspec colname="4" colwidth="35pt" align="center"> </colspec> <colspec colname="5" colwidth="49pt" align="center"> </colspec> <colspec colname="6" colwidth="28pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">1 mM</td>
                <td class="description-td">DNA</td>
                <td class="description-td">DNA/</td>
              </tr> <tr class="description-tr"> <td class="description-td">Tube</td>
                <td class="description-td">DNA Source</td>
                <td class="description-td">DNA (ul)</td>
                <td class="description-td">Tris (ul)</td>
                <td class="description-td">concentration</td>
                <td class="description-td">reaction</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="6" align="center" rowsep="1" class="description-td" colspan="6"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">QC1</td>
                <td class="description-td">6000 pg/ul</td>
                <td class="description-td">10</td>
                <td class="description-td">90</td>
                <td class="description-td">600 pg/ul</td>
                <td class="description-td">3000 pg</td>
              </tr> <tr class="description-tr"> <td class="description-td">QC2</td>
                <td class="description-td"> 600 pg/ul</td>
                <td class="description-td">10</td>
                <td class="description-td">90</td>
                <td class="description-td"> 60 pg/ul</td>
                <td class="description-td"> 300 pg</td>
              </tr> <tr class="description-tr"> <td class="description-td">QC3</td>
                <td class="description-td"> 60 pg/ul</td>
                <td class="description-td">10</td>
                <td class="description-td">490 </td>
                <td class="description-td"> 1.2 pg/ul</td>
                <td class="description-td">  6 pg</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="6" align="center" rowsep="1" class="description-td" colspan="6"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p>B. Determination of DNA. In order to eliminate any bias due to base composition, the fluorescence of ethidium bromide was selected as the means to accurately determine the concentration of all DNAs studied including DNA fragments. The relative amount of DNA in the sample is extrapolated from a standard curve generated by using sheared calf thymus DNA standard curve. The basic protocol used for these determinations follows the protocol outlined in the current protocols.</p>
    <p>Standards generated from calf thymus DNA resulted in the following values. The curve generated from these values was back-fitted to determine the goodness of fit of the curve. It is evident that the curve generated from these data can accurately determine DNA concentrations as low as 5 pg of DNA. This assay has also been carried out using Dynal M280 beads coated with streptavidin with similar results.</p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="1" colwidth="70pt" align="center"> </colspec> <colspec colname="2" colwidth="56pt" align="center"> </colspec> <colspec colname="3" colwidth="91pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="3" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">pg DNA</td>
                <td class="description-td">Back-fitted value</td>
                <td class="description-td">Mean ECL counts</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="3" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="1" colwidth="70pt" align="char" char="."> </colspec> <colspec colname="2" colwidth="56pt" align="char" char="."> </colspec> <colspec colname="3" colwidth="91pt" align="char" char="."> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">5</td>
                <td class="description-td">5.0</td>
                <td class="description-td">23921</td>
              </tr> <tr class="description-tr"> <td class="description-td">10</td>
                <td class="description-td">8.981</td>
                <td class="description-td">33021</td>
              </tr> <tr class="description-tr"> <td class="description-td">50</td>
                <td class="description-td">41.45</td>
                <td class="description-td">96404</td>
              </tr> <tr class="description-tr"> <td class="description-td">100</td>
                <td class="description-td">101.1</td>
                <td class="description-td">196644</td>
              </tr> <tr class="description-tr"> <td class="description-td">500</td>
                <td class="description-td">453</td>
                <td class="description-td">682223</td>
              </tr> <tr class="description-tr"> <td class="description-td">1000</td>
                <td class="description-td">8553</td>
                <td class="description-td">1148437</td>
              </tr> <tr class="description-tr"> <td class="description-td">5000</td>
                <td class="description-td">4929</td>
                <td class="description-td">4323320</td>
              </tr> <tr class="description-tr"> <td class="description-td">10000</td>
                <td class="description-td">10012</td>
                <td class="description-td">6721160</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="3" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <heading>Example 10</heading> <p>Comparison to Slot-Blot and Threshold Methodologies</p>
    <p>Several samples were obtained and extracted using Wako extraction. Comparative assay results are shown below:</p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="5"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="28pt" align="center"> </colspec> <colspec colname="2" colwidth="63pt" align="center"> </colspec> <colspec colname="3" colwidth="42pt" align="center"> </colspec> <colspec colname="4" colwidth="70pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="4" align="center" rowsep="1" class="description-td" colspan="5"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">Threshold</td>
                <td class="description-td">Slot-blot</td>
                <td class="description-td">Tag-DNA</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Sample</td>
                <td class="description-td">pg DNA/ml</td>
                <td class="description-td">pg DNA/ml</td>
                <td class="description-td">pg DNA/ml</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="4" align="center" rowsep="1" class="description-td" colspan="5"> </td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">1</td>
                <td class="description-td">1,300,000</td>
                <td class="description-td">—</td>
                <td class="description-td">1,271,000</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">2</td>
                <td class="description-td">  20,000</td>
                <td class="description-td">—</td>
                <td class="description-td">  59,000</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">3</td>
                <td class="description-td">  736,000</td>
                <td class="description-td">—</td>
                <td class="description-td">1,247,000</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">4</td>
                <td class="description-td">—</td>
                <td class="description-td">380</td>
                <td class="description-td">   660</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">5</td>
                <td class="description-td">—</td>
                <td class="description-td">400</td>
                <td class="description-td">   720</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">6</td>
                <td class="description-td">—</td>
                <td class="description-td">440</td>
                <td class="description-td">   670</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">7</td>
                <td class="description-td">—</td>
                <td class="description-td">620</td>
                <td class="description-td">   1040</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="4" align="center" rowsep="1" class="description-td" colspan="5"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <heading>Example 11</heading> <p>Sensitivity of the Tag-DNA Assay for all DNAs Regardless of Size and Species</p>
    <p>Whether DNA from other species behaved similar to that observed with calf thymus DNA was evaluated. The genomic DNA from all species studied have a unique DNA size and composition. It is evident that mouse DNA generates a standard curve similar to that seen with the calf thymus DNA:</p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="1" colwidth="77pt" align="center"> </colspec> <colspec colname="2" colwidth="56pt" align="center"> </colspec> <colspec colname="3" colwidth="84pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="3" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">pg mouse DNA</td>
                <td class="description-td">Back-fitted value</td>
                <td class="description-td">Mean ECL counts</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="3" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="1" colwidth="77pt" align="char" char="."> </colspec> <colspec colname="2" colwidth="56pt" align="char" char="."> </colspec> <colspec colname="3" colwidth="84pt" align="char" char="."> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">5</td>
                <td class="description-td">6</td>
                <td class="description-td">28599</td>
              </tr> <tr class="description-tr"> <td class="description-td">10</td>
                <td class="description-td">10</td>
                <td class="description-td">36123</td>
              </tr> <tr class="description-tr"> <td class="description-td">50</td>
                <td class="description-td">47</td>
                <td class="description-td">102272</td>
              </tr> <tr class="description-tr"> <td class="description-td">100</td>
                <td class="description-td">116</td>
                <td class="description-td">218254</td>
              </tr> <tr class="description-tr"> <td class="description-td">500</td>
                <td class="description-td">491</td>
                <td class="description-td">778042</td>
              </tr> <tr class="description-tr"> <td class="description-td">1000</td>
                <td class="description-td">955</td>
                <td class="description-td">1369470</td>
              </tr> <tr class="description-tr"> <td class="description-td">5000</td>
                <td class="description-td">4708</td>
                <td class="description-td">4269472</td>
              </tr> <tr class="description-tr"> <td class="description-td">10000</td>
                <td class="description-td">11722</td>
                <td class="description-td">6507011</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="3" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p>The concentration of <i>E. coli </i>DNA (average size 5 kb) was determined as described in Example 2 using ethidium bromide fluorescence spectroscopy and tested in the tag-DNA assay. This experiment shows that DNA from species other than mammalian are efficiently detected in the Tag-DNA assay.</p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="5"> <colspec colname="1" colwidth="56pt" align="center"> </colspec> <colspec colname="2" colwidth="28pt" align="center"> </colspec> <colspec colname="3" colwidth="63pt" align="center"> </colspec> <colspec colname="4" colwidth="28pt" align="center"> </colspec> <colspec colname="5" colwidth="42pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="5" align="center" rowsep="1" class="description-td" colspan="5"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Calf</td>
                <td class="description-td">S. Cervicie</td>
                <td class="description-td"> </td>
                <td class="description-td">E. coli</td>
              </tr> <tr class="description-tr"> <td class="description-td">DNA (pg)</td>
                <td class="description-td">Thymus</td>
                <td class="description-td">(50 kb)</td>
                <td class="description-td">Mouse</td>
                <td class="description-td">(5 kb)</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="5" align="center" rowsep="1" class="description-td" colspan="5"> </td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">  5</td>
                <td class="description-td">  5</td>
                <td class="description-td">  4</td>
                <td class="description-td">  7</td>
                <td class="description-td"> 4</td>
              </tr> <tr class="description-tr"> <td class="description-td">  10</td>
                <td class="description-td">  10</td>
                <td class="description-td">  7</td>
                <td class="description-td"> 10</td>
                <td class="description-td"> 11</td>
              </tr> <tr class="description-tr"> <td class="description-td">  50</td>
                <td class="description-td">  50</td>
                <td class="description-td"> 70</td>
                <td class="description-td"> 45</td>
                <td class="description-td"> 56</td>
              </tr> <tr class="description-tr"> <td class="description-td"> 100</td>
                <td class="description-td">  97</td>
                <td class="description-td"> 135</td>
                <td class="description-td"> 115</td>
                <td class="description-td">118</td>
              </tr> <tr class="description-tr"> <td class="description-td"> 500</td>
                <td class="description-td"> 506</td>
                <td class="description-td"> 553</td>
                <td class="description-td"> 533</td>
                <td class="description-td">512</td>
              </tr> <tr class="description-tr"> <td class="description-td"> 1000</td>
                <td class="description-td"> 995</td>
                <td class="description-td">1007</td>
                <td class="description-td">1068</td>
                <td class="description-td">979</td>
              </tr> <tr class="description-tr"> <td class="description-td"> 5000</td>
                <td class="description-td"> 498</td>
                <td class="description-td">3522</td>
                <td class="description-td">4897</td>
                <td class="description-td">—</td>
              </tr> <tr class="description-tr"> <td class="description-td">10000</td>
                <td class="description-td">10376</td>
                <td class="description-td">—</td>
                <td class="description-td">9481</td>
                <td class="description-td">—</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="5" align="center" rowsep="1" class="description-td" colspan="5"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p>Whether this assay detected smaller fragments of DNA was evaluated. Specific PCR generated products were quantitated by ethidium bromide fluorescence and tested in the assay. The relative concentration of DNA in the PCR products are extrapolated from the calf thymus standard curve.</p>
    <p>A range of DNA fragments isolated from an agarose gel was tested in the assay.</p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="offset" colwidth="42pt" align="left"> </colspec> <colspec colname="1" colwidth="63pt" align="center"> </colspec> <colspec colname="2" colwidth="112pt" align="left"> </colspec> <thead> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="2" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Range of basepairs</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="2" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="offset" colwidth="42pt" align="left"> </colspec> <colspec colname="1" colwidth="63pt" align="center"> </colspec> <colspec colname="2" colwidth="112pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">200-300</td>
                <td class="description-td">117</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">300-400</td>
                <td class="description-td">127</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">400-500</td>
                <td class="description-td"> 96</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Cald Thymus</td>
                <td class="description-td">110</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="2" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p>The back-fitted values obtained from calf thymus DNA standard curve suggests that the assay does not discriminate based on size or sequence.</p>
    <p>While the present invention has been described in terms of specific examples and compositions, it is understood that based on the information contained within this invention variations and modifications will occur to those skilled in the art.</p>
    </div></div></div><div class="patent-section patent-tabular-section"><a id="backward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Cited Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5043272">US5043272</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Apr 27, 1989</td><td class="patent-data-table-td patent-date-value">Aug 27, 1991</td><td class="patent-data-table-td ">Life Technologies, Incorporated</td><td class="patent-data-table-td ">Amplification of nucleic acid sequences using oligonucleotides of random sequence as primers</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5364759">US5364759</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jan 31, 1991</td><td class="patent-data-table-td patent-date-value">Nov 15, 1994</td><td class="patent-data-table-td ">Baylor College Of Medicine</td><td class="patent-data-table-td ">DNA typing with short tandem repeat polymorphisms and identification of polymorphic short tandem repeats</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5413906">US5413906</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">May 31, 1991</td><td class="patent-data-table-td patent-date-value">May 9, 1995</td><td class="patent-data-table-td ">Boehringer Mannheim Gmbh</td><td class="patent-data-table-td ">Method of determining polymerase activity</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5599662">US5599662</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Feb 17, 1995</td><td class="patent-data-table-td patent-date-value">Feb 4, 1997</td><td class="patent-data-table-td ">Hoffmann-La Roche Inc.</td><td class="patent-data-table-td ">Oliconucleotide primers and probes for the detection of HIV-1</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6096499">US6096499</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Mar 21, 1997</td><td class="patent-data-table-td patent-date-value">Aug 1, 2000</td><td class="patent-data-table-td ">Geron Corporation</td><td class="patent-data-table-td ">Mammalian DNA primase screen and activity modulating agents</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="npl-citations"></a><div class="patent-section-header"><span class="patent-section-title">Non-Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th colspan="3"class="patent-data-table-th">Reference</th></tr></thead><tr><td class="patent-data-table-td ">1</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Briggs, Jonathan, et al, Anal. Chem., 1991, 63, 850-859, "<a href='http://scholar.google.com/scholar?q="Quantitation+of+DNA+and+Protein+Impurities+in+Biopharmaceuticals"'>Quantitation of DNA and Protein Impurities in Biopharmaceuticals</a>".</td></tr><tr><td class="patent-data-table-td ">2</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Feinberg, Andrew P., et al, Analytical Biochemistry, 132, 6-13 (1983), "<a href='http://scholar.google.com/scholar?q="A+Technique+for+Radiolabeling+DNA+Restriction+Endonuclease+Fragments+to+High+Specific+Activity"'>A Technique for Radiolabeling DNA Restriction Endonuclease Fragments to High Specific Activity</a>".</td></tr><tr><td class="patent-data-table-td ">3</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Merrick et al. Threshold, a complete system for quantitative analysis of total DNA, protein impurities and relevant proteins, 1992, Biotech Forum Europe, vol. 9(6), pp. 398-403.*</td></tr><tr><td class="patent-data-table-td ">4</td><td class="patent-data-table-td "><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td ">Wu et al. The ligation amplification reaction (LAR)-amplification of specific DNA sequences using sequential rounds of template-dependent ligation. Genomics , vol. 4, p. 560-569.*</td></tr><tr><td class="patent-data-table-td ">5</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Wu et al. The ligation amplification reaction (LAR)—amplification of specific DNA sequences using sequential rounds of template-dependent ligation. Genomics , vol. 4, p. 560-569.*</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="forward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Referenced by</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Citing Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8372652">US8372652</a></td><td class="patent-data-table-td patent-date-value">Apr 24, 2009</td><td class="patent-data-table-td patent-date-value">Feb 12, 2013</td><td class="patent-data-table-td ">Board Of Regents Of The University Of Texas System</td><td class="patent-data-table-td ">Luminescent nanostructured materials for use in electrogenerated chemiluminescence</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8586378">US8586378</a></td><td class="patent-data-table-td patent-date-value">Apr 7, 2009</td><td class="patent-data-table-td patent-date-value">Nov 19, 2013</td><td class="patent-data-table-td ">Board Of Regents, The University Of Texas System</td><td class="patent-data-table-td ">Method and apparatus for nanoparticle electrogenerated chemiluminescence amplification</td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="classifications"></a><div class="patent-section-header"><span class="patent-section-title">Classifications</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th"> </th><th class="patent-data-table-th"> </th></tr></thead><tr><td class="patent-data-table-td ">U.S. Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=-rdiBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S006100">435/6.1</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=-rdiBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S025600">536/25.6</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=-rdiBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S025320">536/25.32</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=-rdiBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S091200">435/91.2</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=-rdiBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S091520">435/91.52</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=-rdiBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S091510">435/91.51</a></span></td></tr><tr><td class="patent-data-table-td ">International Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=-rdiBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12Q0001680000">C12Q1/68</a></span></td></tr><tr><td class="patent-data-table-td ">Cooperative Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=-rdiBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12Q1/6851">C12Q1/6851</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=-rdiBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12Q1/6834">C12Q1/6834</a></span></td></tr><tr><td class="patent-data-table-td ">European Classification</td><td class="patent-data-table-td "><span class="nested-value">C12Q1/68B10</span></td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="legal-events"></a><div class="patent-section-header"><span class="patent-section-title">Legal Events</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Date</th><th class="patent-data-table-th">Code</th><th class="patent-data-table-th">Event</th><th class="patent-data-table-th">Description</th></tr></thead><tr><td class="patent-data-table-td patent-date-value">Dec 13, 2011</td><td class="patent-data-table-td ">FP</td><td class="patent-data-table-td ">Expired due to failure to pay maintenance fee</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20111021</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Oct 21, 2011</td><td class="patent-data-table-td ">LAPS</td><td class="patent-data-table-td ">Lapse for failure to pay maintenance fees</td><td class="patent-data-table-td "></td></tr><tr><td class="patent-data-table-td patent-date-value">May 30, 2011</td><td class="patent-data-table-td ">REMI</td><td class="patent-data-table-td ">Maintenance fee reminder mailed</td><td class="patent-data-table-td "></td></tr><tr><td class="patent-data-table-td patent-date-value">Dec 23, 2008</td><td class="patent-data-table-td ">B1</td><td class="patent-data-table-td ">Reexamination certificate first reexamination</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">CLAIMS 1-49 ARE DETERMINED TO BE PATENTABLE AS AMENDED. CLAIMS 50-62, DEPENDENT ON AN AMENDED CLAIM, ARE DETERMINED TO BE PATENTABLE. NEW CLAIMS 63-97 ARE ADDED AND DETERMINED TO BE PATENTABLE.</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Apr 23, 2007</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">4</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Jan 16, 2007</td><td class="patent-data-table-td ">RR</td><td class="patent-data-table-td ">Request for reexamination filed</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20060815</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Jan 9, 2007</td><td class="patent-data-table-td ">RR</td><td class="patent-data-table-td ">Request for reexamination filed</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20060815</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Apr 27, 2004</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">BIOVERIS CORPORATION, MARYLAND</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNOR:IGEN INTERNATIONAL, INC.;REEL/FRAME:015232/0563</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20040212</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNOR:IGEN INTERNATIONAL, INC.;REEL/FRAME:015232/0070</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNOR:IGEN INTERNATIONAL, INC.;REEL/FRAME:015232/0080</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNOR:IGEN INTERNATIONAL, INC.;REEL/FRAME:015232/0232</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNOR:IGEN INTERNATIONAL, INC.;REEL/FRAME:015232/0375</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNOR:IGEN INTERNATIONAL, INC.;REEL/FRAME:015232/0727</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNOR:IGEN INTERNATIONAL, INC.;REEL/FRAME:015242/0799</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNOR:IGEN INTERNATIONAL, INC.;REEL/FRAME:015248/0731</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNOR:IGEN INTERNATIONAL, INC.;REEL/FRAME:015249/0001</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNOR:IGEN INTERNATIONL, INC.;REEL/FRAME:015232/0435</span></div><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">BIOVERIS CORPORATION 16020 INDUSTRIAL DRIVEGAITHER</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNOR:IGEN INTERNATIONAL, INC. /AR;REEL/FRAME:015232/0232</span></div><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">BIOVERIS CORPORATION 16020 INDUSTRIAL DRIVEGAITHER</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNOR:IGEN INTERNATIONAL, INC. /AR;REEL/FRAME:015232/0727</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNOR:IGEN INTERNATIONAL, INC. /AR;REEL/FRAME:015232/0070</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNOR:IGEN INTERNATIONAL, INC. /AR;REEL/FRAME:015232/0080</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNOR:IGEN INTERNATIONAL, INC. /AR;REEL/FRAME:015232/0375</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNOR:IGEN INTERNATIONAL, INC. /AR;REEL/FRAME:015232/0563</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNOR:IGEN INTERNATIONAL, INC. /AR;REEL/FRAME:015242/0799</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNOR:IGEN INTERNATIONAL, INC. /AR;REEL/FRAME:015248/0731</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNOR:IGEN INTERNATIONAL, INC. /AR;REEL/FRAME:015249/0001</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNOR:IGEN INTERNATIONL, INC. /AR;REEL/FRAME:015232/0435</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Jun 24, 1998</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">IGEN INTERNATIONAL, INC., MARYLAND</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNORS:HEROUX, JEFFREY A.;CORCORAN, MARTA L.;RAO, SAVITHA M.;REEL/FRAME:009295/0535;SIGNING DATES FROM 19980311 TO 19980511</span></div></td></tr></table><div class="patent-section-footer"></div></div><div class="modal-dialog" id="patent-images-lightbox"><div class="patent-lightbox-controls"><div class="patent-lightbox-rotate-controls"><div class="patent-lightbox-rotation-text">Rotate</div><div class="rotate-icon rotate-ccw-icon"></div><div class="rotate-icon rotate-cw-icon"></div></div><div class="patent-lightbox-index-counter"></div><a class="patent-lightbox-fullsize-link" target="_blank">Original Image</a><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_right.png" alt="Next page"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_left.png" alt="Previous page"width="21" height="21" /></div></div><div class="modal-dialog-content"><div class="patent-lightbox-image-holder"><div class="patent-lightbox-placeholder"></div></div></div></div><script>_OC_initPatentsAtb({image_not_available_html: " Image not available"});</script></div></div></div></td></tr></table><script>(function() {var href = window.location.href;if (href.indexOf('?') !== -1) {var parameters = href.split('?')[1].split('&');for (var i = 0; i < parameters.length; i++) {var param = parameters[i].split('=');if (param[0] == 'focus') {var elem = document.getElementById(param[1]);if (elem) {elem.focus();}}}}})();</script><script>_OC_addFlags({LockSrc:"/books/javascript/lock_8a2b04e7bf975d5171d8e4c0b6365c7a.js", Host:"http://www.google.com/", IsBooksRentalEnabled:1, IsWebstoreDisplayCaseEnabled:1, IsObfuscationEnabled:1, IsBrowsingHistoryEnabled:1, IsWebReaderSvgEnabled:0, IsGeoLayerEnabled:1, IsImageModeNotesEnabled:1, IsCopyMenuItemEnabled:1, IsGiftingEnabled:0, IsWebReaderUniversalPaginatorEnabled:0, IsOfflineBubbleEnabled:1, IsReaderEnabledForPlayRequests:1, IsFutureOnSaleVolumesEnabled:1, IsOfflineRestrictedCopyEnabled:1, IsBooksUnifiedLeftNavEnabled:1, IsRestrictedCopyEnabled:1, IsZipitFolderCollectionEnabled:1, IsEndOfSampleRecommendationsEnabled:1, IsRatingsOnBookcardsEnabled:1, IsAdsDisabled:0, IsIframePageDisplayEnabled:0, IsEmbeddedMediaEnabled:1, IsImageModeAnnotationsEnabled:1, IsMyLibraryGooglePlusEnabled:1, IsImagePageProviderEnabled:0, IsBookcardListPriceSmall:0, IsInternalUser:0, IsBooksShareButtonEnabled:0, IsPreOrdersEnabled:0, IsDisabledRandomBookshelves:0, WebstoreDisplayCasePosition:3});_OC_Run({"enable_p13n":false,"add_vol_to_collection_base_url":"http://www.google.com/patents?op=add\u0026sig=ACfU3U2xeWwxJcr_1oLKrzEZuP3NW4XlEg\u0026id=-rdiBAABERAJ","remove_vol_from_collection_base_url":"http://www.google.com/patents?op=remove\u0026sig=ACfU3U2rLjj2isgvr4V3Z93p0xoGrtUEAg\u0026id=-rdiBAABERAJ","logged_in":false,"p13n_save_user_settings_url":"http://www.google.com/patents?op=edit_user_settings\u0026sig=ACfU3U3cJf_-kPGNcwAhlUP9AN7UK6kNAw","is_cobrand":false,"sign_in_url":"https://www.google.com/accounts/Login?service=\u0026continue=http://www.google.com/patents%3Fhl%3Den\u0026hl=en","is_play_enabled":true}, {"volume_id":"","is_ebook":true,"volumeresult":{"has_flowing_text":false,"has_scanned_text":true,"can_download_pdf":false,"can_download_epub":false,"is_pdf_drm_enabled":false,"is_epub_drm_enabled":false,"download_pdf_url":"http://www.google.com/patents/download/Assay_methods_for_nucleic_acid_in_a_samp.pdf?id=-rdiBAABERAJ\u0026output=pdf\u0026sig=ACfU3U0tXVaadwgOiM4Pgyk7biT_wHJhuw"},"sample_url":"http://www.google.com/patents/reader?id=-rdiBAABERAJ\u0026printsec=frontcover\u0026output=reader\u0026source=gbs_atb_hover","is_browsable":true,"is_public_domain":true}, {});</script><div id="footer_table" style="font-size:83%;text-align:center;position:relative;top:20px;height:4.5em;margin-top:2em"><div style="margin-bottom:8px"><a href=http://www.google.com/><nobr>Google&nbsp;Home</nobr></a> - <a href=//www.google.com/patents/sitemap/><nobr>Sitemap</nobr></a> - <a href=http://www.google.com/googlebooks/uspto.html><nobr>USPTO Bulk Downloads</nobr></a> - <a href=/intl/en/privacy/><nobr>Privacy Policy</nobr></a> - <a href=/intl/en/policies/terms/><nobr>Terms of Service</nobr></a> - <a href=https://support.google.com/faqs/answer/2539193?hl=en><nobr>About Google Patents</nobr></a> - <a href="http://www.google.com/tools/feedback/intl/en/error.html" onclick="try{_OC_startFeedback({productId: '72792',locale: 'en'});return false;}catch(e){}"><nobr>Send Feedback</nobr></a></div><span>Data provided by IFI CLAIMS Patent Services</span><br><span >&copy;2012 Google</span></div> <script type="text/javascript">var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));</script><script type="text/javascript">var pageTracker = _gat._getTracker("UA-27188110-1");pageTracker._setCookiePath("/patents/");pageTracker._trackPageview();</script> </body></html>