<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN""http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html><head><title>Patent US6730778 - Nucleotide sequences coding polypeptide for use in the treatment of nervous ... - Google Patents</title><script>(function(){(function(){function e(a){this.t={};this.tick=function(a,c,b){var d=void 0!=b?b:(new Date).getTime();this.t[a]=[d,c];if(void 0==b)try{window.console.timeStamp("CSI/"+a)}catch(e){}};this.tick("start",null,a)}var a;window.performance&&(a=window.performance.timing);var f=a?new e(a.responseStart):new e;window.jstiming={Timer:e,load:f};if(a){var c=a.navigationStart,d=a.responseStart;0<c&&d>=c&&(window.jstiming.srt=d-c)}if(a){var b=window.jstiming.load;0<c&&d>=c&&(b.tick("_wtsrt",void 0,c),b.tick("wtsrt_",
"_wtsrt",d),b.tick("tbsd_","wtsrt_"))}try{a=null,window.chrome&&window.chrome.csi&&(a=Math.floor(window.chrome.csi().pageT),b&&0<c&&(b.tick("_tbnd",void 0,window.chrome.csi().startE),b.tick("tbnd_","_tbnd",c))),null==a&&window.gtbExternal&&(a=window.gtbExternal.pageT()),null==a&&window.external&&(a=window.external.pageT,b&&0<c&&(b.tick("_tbnd",void 0,window.external.startE),b.tick("tbnd_","_tbnd",c))),a&&(window.jstiming.pt=a)}catch(g){}})();})();
</script><link rel="stylesheet" href="/patents/css/_8a2b04e7bf975d5171d8e4c0b6365c7a/kl_intl_patents_bundle.css" type="text/css" /><script src="/books/javascript/atb_8a2b04e7bf975d5171d8e4c0b6365c7a__en.js"></script><script>function googleTranslateElementInit() {new google.translate.TranslateElement({pageLanguage: "en",gaTrack: true,gaId: "UA-27188110-1",multilanguagePage: true});}</script><script src="//translate.google.com/translate_a/element.js?cb=googleTranslateElementInit"></script><meta name="DC.type" content="Patent"><meta name="DC.title" content="Nucleotide sequences coding polypeptide for use in the treatment of nervous system disorders"><meta name="DC.contributor" content="Mark E. Gurney" scheme="inventor"><meta name="DC.contributor" content="Adele M. Pauley" scheme="inventor"><meta name="DC.contributor" content="Jinhe Li" scheme="inventor"><meta name="DC.contributor" content="Pharmacia And Upjohn Company" scheme="assignee"><meta name="DC.date" content="1999-6-9" scheme="dateSubmitted"><meta name="DC.description" content="The present invention provides isolated nucleic acid molecules comprising a polynucleotide encoding either of two alternative splice variants of human sel-10, one of which is expressed in hippocampal cells, and one of which is expressed in mammary cells. The invention also provides isolated sel-10 polypeptides and cell lines which express them in which Aβ processing is altered."><meta name="DC.date" content="2004-5-4" scheme="issued"><meta name="DC.relation" content="WO:1997011956:A1" scheme="references"><meta name="citation_reference" content="A. Doan, G. Thinakaran, D.R. Borchelt, H.H. Slunt, T. Ratovitsky, M. Podlisny, D.J. Selkoe, M. Seeger, S.E. Gandy, D.L. Price, S.S. Sisodia, &quot;Protein Topology of Presenilin 1,&quot; Neuron, 1996, 17:1023-1030."><meta name="citation_reference" content="D. Levitan, I. Greenwald, &quot;Facilitation of lin-12-mediated signalling by sel-12, a Caenorhabditis elegans S182 Alzheimer&#39;s disease gene,&quot; Nature, 1995, 377:351-4."><meta name="citation_reference" content="D. Levitan, T.G. Doyle, D. Brousseau, M.K. Lee, G. Thinakaran, H.H. Slunt, S.S. Sidodia, I. Greenwald, &quot;Assessment of normal and mutant human presenilin function in Caenorgabditis elegans,&quot; Proc. Natl. Acad. Sci. USA, 1996, 93: 14940-4."><meta name="citation_reference" content="D. Scheuner, C. Eckman, M. Jensen, X. Song, M. Citron, N. Suzukik, T.D. Bird, J. Hardy, M. Hutton, W. Kukull, E. Larson, E. Levy-Lahad, M. Viitanen, E. Peskind, P. Poorkaj, G. Schellenberg, R. Tanzi, W. Wasco, L. Lannfelt, D. Selkoe, S. Younkin, &quot;Secreted amyloid beta-protein similar to that in the senile plaques of Alzheimer&#39;s disease is increased in vivo by the presenilin 1 and 2 and APP mutations linked to familial Alzheimer&#39;s disease,&quot; Nat. Med., 1996, 2:864-870."><meta name="citation_reference" content="D. Scheuner, C. Eckman, M. Jensen, X. Song, M. Citron, N. Suzukik, T.D. Bird, J. Hardy, M. Hutton, W. Kukull, E. Larson, E. Levy-Lahad, M. Viitanen, E. Peskind, P. Poorkaj, G. Schellenberg, R. Tanzi, W. Wasco, L. Lannfelt, D. Selkoe, S. Younkin, “Secreted amyloid β-protein similar to that in the senile plaques of Alzheimer&#39;s disease is increased in vivo by the presenilin 1 and 2 and APP mutations linked to familial Alzheimer&#39;s disease,” Nat. Med., 1996, 2:864-870."><meta name="citation_reference" content="D.M. Kovacs, H.J. Fausett, K.J. Page, T.-W. Kim, R.D. Moir, D.E.Merriam, R.D. Hollister, O.G. Hallmark, R. Mancini, K.M. Felsenstein, B.T. Hyman, R.E. Tanzi, W. Wasco,&quot;Alzheimer-associated presenilins 1 and 2: Neronal expression in brain and localization to intracellular membranes in mammalian cells,&quot; Nat. Med., 1996, 2:224-229."><meta name="citation_reference" content="D.R. Borchelt, G. Thinakaran, C.B. Eckman, M.K. Lee, F. Davenport, T. Ratovitosky, C.-M. Prada, G. Kim, S. Seekins, D. Yager, H.H. Slunt, R. Wang, M. M. Seeger, A.I. Levey, S.E. Gandy, N.G. Copeland, N.A. Jenkins, D.L. Price, S.G. Younkin, S.S. Sisodia, &quot;Familial Alzheimer&#39;s Disease-Linked Presenilin 1 Variants Elevate Abeta1-42/1-40 Ratio In Vitro and In Vivo,&quot; Neuron, 1996, 17:1005-1013."><meta name="citation_reference" content="D.R. Borchelt, G. Thinakaran, C.B. Eckman, M.K. Lee, F. Davenport, T. Ratovitosky, C.-M. Prada, G. Kim, S. Seekins, D. Yager, H.H. Slunt, R. Wang, M. M. Seeger, A.I. Levey, S.E. Gandy, N.G. Copeland, N.A. Jenkins, D.L. Price, S.G. Younkin, S.S. Sisodia, “Familial Alzheimer&#39;s Disease-Linked Presenilin 1 Variants Elevate Aβ1-42/1-40 Ratio In Vitro and In Vivo,” Neuron, 1996, 17:1005-1013."><meta name="citation_reference" content="D.R. Borchelt, T. Ratovitski, J. van Lare, M.K. Lee, V. Gonzales, N.A. Jenkins, N.G. Copeland, D.L. Price, S.S. Sisodia, &quot;Accelerated Amyloid Deposition in the Brains of Transgenic Mice Coexpressing Mutant Presenilin 1 and Amyloid PrecursorProteins,&quot; Neuron, 1997, 19:939-945."><meta name="citation_reference" content="E. Levy-Lahad, W. Wasco, P. Poorkaj, D.M. Ramano, J. Oshima, W.H. Pettingell, C.-E. Yu, P.D. Jondro, S.D. Schmidt, K. Wang, A.C. Crowley, Y.-H. Fu, S.Y. Guenette, D. Galas, E. Nemens, E.M. Wijsman, T.D. Bird, G.D. Schellenberg, R.E. Tanzi, &quot;Candidate Gene for the Chromosome 1 Familial Alzheimer&#39;s Disease Locus,&quot; Science, 1995, 269: 973-977."><meta name="citation_reference" content="E.I. Rogaev, R. Sherrington, E.A. Rogaeva, G. Levesque, M. Ideda, Y. Liang, H. Chi, C. Lin, K. Holman, T. Tsuda, L. Mar, S. Sorbi, B. Nacmais, S. Piacentini, L. Amaducci, I. Chumakov, D. Cohen, L. Lannfelt, P.E. Fraser, J.M. Rommens, P.H. St George-Hyslop, &quot;Familial Alzheimer&#39;s disease in kindreds with missense mutations in a gene on chromosome 1 related to the Alzheimer&#39;s disease type 3 gene,&quot; Nature, 1995, 376: 775-778."><meta name="citation_reference" content="E.J.A. Hubbard, G. Wu, J. Kitajewski, I. Greenwald, &quot;sel-10, a negative regulator of lin-12 activity in Caenorhabditis elegans, encodes a member of the CDC4 family of proteins,&quot; Genes &amp; Development, 1997, 11:3182-3193."><meta name="citation_reference" content="Genbank H22962 Jul. 6, 1995."><meta name="citation_reference" content="Genbank H34371 Jul. 19, 1995."><meta name="citation_reference" content="I. Greenwald, G. Seydoux, &quot;Analysis of gain-of-function mutations of the lin-12 gene of Caenorhabditis elegans,&quot; Nature, 1990, 346: 197-9."><meta name="citation_reference" content="J. Li, J. Ma, H. Potter, &quot;Identification and expression analysis of a potential familial Alzheimer disease gene on chromosome 1 related to AD3,&quot; Proc. Natl. Acad. Sci. U.S.A., 1995, 92:12180-12184."><meta name="citation_reference" content="J. Perez-Tur, S. Froelich, G. Prihar, R. Crook, M. Baker, K. Duff, M. Wragg, F. Busfield, C. Lendon, R.F. Clark, P. Roques, R.A. Fuldner, J. Johnston, R. Cowburn, C. Forsell, K. Axelman, L. Lilius, H. Houlden, E. Karran, G.W. Roberts, M. Rossor, M.D. Adams, J. Hardy, A. Goate, L. Lannfelt, M. Hutton, &quot;A Mutation in Alzheimer&#39;s disease destroying a splice acceptor site in the presenilin-1 gene,&quot; Neuroreport, 1995, 7:297-301."><meta name="citation_reference" content="K. Duff, C. Eckman, C. Zehr, X. Yu, C.-M. Prada, J. Perez-Tur, M. Hutton, L. Buee, Y. Harigaya, D. Yager, D. Morgan, M.N. Gordon, L. Holcomb, L. Refolo, B. Zenk, J. Hardy, S. Younkin, &quot;Increased amyloid-beta42(42)in brains of mice expressing mutant presenilin 1,&quot; Nature, 1996, 383:710-713."><meta name="citation_reference" content="K. Duff, C. Eckman, C. Zehr, X. Yu, C.-M. Prada, J. Perez-Tur, M. Hutton, L. Buee, Y. Harigaya, D. Yager, D. Morgan, M.N. Gordon, L. Holcomb, L. Refolo, B. Zenk, J. Hardy, S. Younkin, “Increased amyloid-β42(42)in brains of mice expressing mutant presenilin 1,” Nature, 1996, 383:710-713."><meta name="citation_reference" content="M. Citron, D. Westaway, W. Xia, G. Carlson, T. Diehl, G. Levesque, K. Johnson-Wood, M. Lee, P. Seubert, A. Davis, D. Kholodenko, R. Motter, R. Sherrington,l B. Perry, H. Yao, R. Strome, I. Lieberburg, J. Rommens, S. Kim, D. Schenk, P. Fraser, P. St George Hyslop, D.J. Selkoe, &quot;Mutant presenilins of Alzheimer&#39;s disease increase production of 42-residue amyloid beta-protein in both transfected cells and transgenic mice,&quot; Nat. Med., 1997, 3:67-72."><meta name="citation_reference" content="M. Citron, D. Westaway, W. Xia, G. Carlson, T. Diehl, G. Levesque, K. Johnson-Wood, M. Lee, P. Seubert, A. Davis, D. Kholodenko, R. Motter, R. Sherrington,l B. Perry, H. Yao, R. Strome, I. Lieberburg, J. Rommens, S. Kim, D. Schenk, P. Fraser, P. St George Hyslop, D.J. Selkoe, “Mutant presenilins of Alzheimer&#39;s disease increase production of 42-residue amyloid β-protein in both transfected cells and transgenic mice,” Nat. Med., 1997, 3:67-72."><meta name="citation_reference" content="M. Hochstrasser, &quot;Ubiquitin-dependent protein degradation,&quot; Annual Review of Genetics, 1996, 30:405-439."><meta name="citation_reference" content="M. Mercken, H. Takahashi, T. Honda, Z. Sato, M. Murayama, Y. Nakazato, K. Noguchi, K. Imahori, A. Takashima, &quot;Characterization of human presenilin 1 using N-terminal specific monoclonal antibodies: Evidence that Alzheimer mutations affect proteolytic processing,&quot; FEBS Lett., 1996, 389:297-303."><meta name="citation_reference" content="M. Mullan, F. Crawford, K. Axelman, H. Houlden, L. Lilius, B. Winblad, L. Lannfelt, &quot;A pathogenic mutation for probable Alzheimer&#39;s disease in the APP gene at the N-terminus of beta-amyloid,&quot; Nat Genet, Aug., 1992 1(5):345-7."><meta name="citation_reference" content="M. Mullan, F. Crawford, K. Axelman, H. Houlden, L. Lilius, B. Winblad, L. Lannfelt, “A pathogenic mutation for probable Alzheimer&#39;s disease in the APP gene at the N-terminus of β-amyloid,” Nat Genet, Aug., 1992 1(5):345-7."><meta name="citation_reference" content="M. Sundaram, I. Greenwald, &quot;Genetic and Phenotypic Studies of Hypomorphic lin-12 Mutants in Caenorhabditis elegans,&quot; Genetics, 1993, 135: 755-763."><meta name="citation_reference" content="M. Sundaram, I. Greenwald, &quot;Suppressors of a lin-12 Hypomorph Define Genes That Interact With Both lin-12 and glp-1 in Caenorhabditis elegans,&quot; Genetics, 1993, 135: 765-83."><meta name="citation_reference" content="R. Sherrington, E.I. Rogaev, Y. Liang, E.A. Rogaeva, G. Levesque, M. Ikeda, H. Chi, C. Lin, G. Li, K. Holman, T. Tsuda, L. Mar, J.-F. Foncin, A.C. Bruni, M.P. Montesi, S. Sorbi, I. Rainero, L. Pinessi, L. Nee, I. Chumakov, D. Pollen, A. Brookes, P. Sanseau, R.J. Polinsky, W. Wasco, H.A.R. Da Silva, J.L. Haines, M.A. Pericak-Vance, R.E. Tanzi, A.D. Roses, P.E. Fraser, J.M. Rommens, P.H. St George-Hyslop, &quot;Cloning of a gene bearing missense mutations in early-onset familial Alzheimer&#39;s disease,&quot; Nature, 1995, 375: 754-760."><meta name="citation_reference" content="T.W. Kim, W.H. Pettingell, O.G. Hallmark, R.D. Moir, W. Wasco, R.E. Tanzi, &quot;Endoproteoltic Cleavage and Proteasomal Degradation of Presenilin 2 in Transfected Cells,&quot; Journal of Biological Chemistry, 1997, 272: 11006-11010."><meta name="citation_patent_number" content="US:6730778"><meta name="citation_patent_application_number" content="US:09/328,877"><link rel="canonical" href="http://www.google.com/patents/US6730778"/><meta property="og:url" content="http://www.google.com/patents/US6730778"/><meta name="title" content="Patent US6730778 - Nucleotide sequences coding polypeptide for use in the treatment of nervous system disorders"/><meta name="description" content="The present invention provides isolated nucleic acid molecules comprising a polynucleotide encoding either of two alternative splice variants of human sel-10, one of which is expressed in hippocampal cells, and one of which is expressed in mammary cells. The invention also provides isolated sel-10 polypeptides and cell lines which express them in which Aβ processing is altered."/><meta property="og:title" content="Patent US6730778 - Nucleotide sequences coding polypeptide for use in the treatment of nervous system disorders"/><meta property="og:type" content="book"/><meta property="og:site_name" content="Google Books"/><meta property="og:image" content="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><link rel="image_src" href="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><script>(function(){try{var aa=function(a,b,c,d){d=d||{};d._sn=["cfg",b,c].join(".");window.gbar.logger.ml(a,d)};var g=window.gbar=window.gbar||{},l=window.gbar.i=window.gbar.i||{},m={},n;function _tvn(a,b){var c=parseInt(a,10);return isNaN(c)?b:c}function _tvf(a,b){var c=parseFloat(a);return isNaN(c)?b:c}function _tvv(a){return!!a}function p(a,b,c){(c||g)[a]=b}g.bv={n:_tvn("2",0),r:"",f:".67.",e:"0",m:_tvn("0",1)};
function q(a,b,c){var d="on"+b;if(a.addEventListener)a.addEventListener(b,c,!1);else if(a.attachEvent)a.attachEvent(d,c);else{var f=a[d];a[d]=function(){var a=f.apply(this,arguments),b=c.apply(this,arguments);return void 0==a?b:void 0==b?a:b&&a}}}var s=function(a){return function(){return g.bv.m==a}},ba=s(1),ca=s(2);p("sb",ba);p("kn",ca);l.a=_tvv;l.b=_tvf;l.c=_tvn;l.i=aa;var da=window.gbar.i.i;var t,u,v,w;function ea(a){v=a}function fa(a){var b;if(b=v&&window.encodeURIComponent)b=a.href,b=!b.match(/^http[s]?:\/\/accounts\.google\.[^/]*\/ClearSID/i)&&!b.match(/^http[s]?:\/\/[^/]*\/accounts\/ClearSID/i);if(b=b&&encodeURIComponent(v()))a.href=a.href.replace(/([?&]continue=)[^&]*/,"$1"+b)}function ga(a){window.gApplication&&(a.href=window.gApplication.getTabUrl(a.href))}
function ha(a){var b=document.forms[0].q,c=window.encodeURIComponent&&b&&b.value,b=b&&b.placeholder;c&&c!=b&&(a.href=a.href.replace(/([?&])q=[^&]*|$/,function(a,b){return(b||"&")+"q="+encodeURIComponent(c)}))}n=l.a("")?ga:ha;
function x(a,b,c,d,f,e){var h=document.getElementById(a);if(h){var k=h.style;k.left=d?"auto":b+"px";k.right=d?b+"px":"auto";k.top=c+"px";k.visibility=u?"hidden":"visible";f&&e?(k.width=f+"px",k.height=e+"px"):(x(t,b,c,d,h.offsetWidth,h.offsetHeight),u=u?"":a)}}
var y=[],ia=function(a,b){y.push(b)},ja=function(a){a=a||window.event;var b=a.target||a.srcElement;a.cancelBubble=!0;null==t&&(a=document.createElement(Array.every||window.createPopup?"iframe":"div"),a.frameBorder="0",t=a.id="gbs",a.src="javascript:''",b.parentNode.appendChild(a),q(document,"click",z));var c=b,b=0;"gb3"!=c.className&&(c=c.parentNode);a=c.getAttribute("aria-owns")||"gbi";var d=c.offsetWidth,f=20<c.offsetTop?46:24;document.getElementById("tphdr")&&(f-=3);var e=!1;do b+=c.offsetLeft||
0;while(c=c.offsetParent);var c=(document.documentElement.clientWidth||document.body.clientWidth)-b-d,h,d=document.body,k=document.defaultView;k&&k.getComputedStyle?(d=k.getComputedStyle(d,""))&&(h=d.direction):h=d.currentStyle?d.currentStyle.direction:d.style.direction;h="rtl"==h;if("gbi"==a){for(d=0;k=y[d++];)k();A(null,window.navExtra);h&&(b=c,e=!0)}else h||(b=c,e=!0);u!=a&&z();x(a,b,f,e)},z=function(){u&&x(u,0,0)},A=function(a,b){var c,d=document.getElementById("gbi"),f=a;f||(f=d.firstChild);
for(;b&&(c=b.pop());){var e=d,h=c,k=f;w||(w="gb2");e.insertBefore(h,k).className=w}},ka=function(a,b,c){if((b=document.getElementById(b))&&a){a.className="gb4";var d=document.createElement("span");d.appendChild(a);d.appendChild(document.createTextNode(" | "));d.id=c;b.appendChild(d)}},la=function(){return document.getElementById("gb_70")},ma=function(){return!!u};p("qs",n);p("setContinueCb",ea);p("pc",fa);p("tg",ja);p("close",z);p("addLink",ka);p("almm",A);p("si",la);p("adh",ia);p("op",ma);var B=function(){},C=function(){},F=function(a){var b=new Image,c=D;b.onerror=b.onload=b.onabort=function(){try{delete E[c]}catch(a){}};E[c]=b;b.src=a;D=c+1},E=[],D=0;p("logger",{il:C,ml:B,log:F});var G=window.gbar.logger;var H={},na={},I=[],oa=l.b("0.1",.1),pa=l.a("1",!0),qa=function(a,b){I.push([a,b])},ra=function(a,b){H[a]=b},sa=function(a){return a in H},J={},K=function(a,b){J[a]||(J[a]=[]);J[a].push(b)},ta=function(a){K("m",a)},L=function(a,b){var c=document.createElement("script");c.src=a;c.async=pa;Math.random()<oa&&(c.onerror=function(){c.onerror=null;B(Error("Bundle load failed: name="+(b||"UNK")+" url="+a))});(document.getElementById("xjsc")||document.getElementsByTagName("body")[0]||
document.getElementsByTagName("head")[0]).appendChild(c)},N=function(a){for(var b=0,c;(c=I[b])&&c[0]!=a;++b);!c||c[1].l||c[1].s||(c[1].s=!0,M(2,a),c[1].url&&L(c[1].url,a),c[1].libs&&m.d&&m.d(c[1].libs))},O=function(a){K("gc",a)},P=null,ua=function(a){P=a},M=function(a,b,c){if(P){a={t:a,b:b};if(c)for(var d in c)a[d]=c[d];try{P(a)}catch(f){}}};p("mdc",H);p("mdi",na);p("bnc",I);p("qGC",O);p("qm",ta);p("qd",J);p("lb",N);p("mcf",ra);p("bcf",qa);p("aq",K);p("mdd","");p("has",sa);
p("trh",ua);p("tev",M);var Q=l.b("0.1",.001),R=0;
function _mlToken(a,b){try{if(1>R){R++;var c,d=a,f=b||{},e=encodeURIComponent,h=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&jexpid=",e("17483"),"&srcpg=",e("prop=22"),"&jsr=",Math.round(1/Q),"&ogev=",e("lS7uU5KROoa7sQTklYLgAQ"),"&ogf=",g.bv.f,"&ogrp=",e("1"),"&ogv=",e("1407723702.0"),"&oggv="+e("es_plusone_gc_20140723.0_p0"),"&ogd=",e("com"),"&ogc=",e("DEU"),"&ogl=",e("en")];f._sn&&(f._sn="og."+
f._sn);for(var k in f)h.push("&"),h.push(e(k)),h.push("="),h.push(e(f[k]));h.push("&emsg=");h.push(e(d.name+":"+d.message));var r=h.join("");S(r)&&(r=r.substr(0,2E3));c=r;var Aa=window.gbar.logger._aem(a,c);F(Aa)}}catch(Na){}}var S=function(a){return 2E3<=a.length},va=function(a,b){return b};function T(a){B=a;p("_itl",S,G);p("_aem",va,G);p("ml",B,G);a={};H.er=a}l.a("")?T(function(a){throw a;}):l.a("1")&&Math.random()<Q&&T(_mlToken);I.push(["m",{url:"//ssl.gstatic.com/gb/js/scm_7385cc5883250b43a39405734c1bea59.js"}]);g.mcf("c",{});g.sg={c:""};if(l.a("1")){var wa=l.a("");I.push(["gc",{auto:wa,url:"//ssl.gstatic.com/gb/js/abc/gci_91f30755d6a6b787dcc2a4062e6e9824.js",libs:"googleapis.client:plusone:gapi.iframes"}]);var xa={version:"gci_91f30755d6a6b787dcc2a4062e6e9824.js",index:"",lang:"en"};H.gc=xa;var U=function(a){window.googleapis&&window.iframes?a&&a():(a&&O(a),N("gc"))};p("lGC",U);l.a("1")&&p("lPWF",U)};window.__PVT="";if(l.a("1")&&l.a("1")){var V=function(a){U(function(){K("pw",a);N("pw")})};p("lPW",V);I.push(["pw",{url:"//ssl.gstatic.com/gb/js/abc/pwm_45f73e4df07a0e388b0fa1f3d30e7280.js"}]);var W=[],ya=function(a){W[0]=a},za=function(a,b){var c=b||{};c._sn="pw";B(a,c)},Ba={signed:W,elog:za,base:"https://plusone.google.com/u/0",loadTime:(new Date).getTime()};H.pw=Ba;var X=function(a,b){for(var c=b.split("."),d=function(){var b=arguments;a(function(){for(var a=g,d=0,e=c.length-1;d<e;++d)a=a[c[d]];a[c[d]].apply(a,b)})},f=g,e=0,h=c.length-1;e<h;++e)f=
f[c[e]]=f[c[e]]||{};return f[c[e]]=d};X(V,"pw.clk");X(V,"pw.hvr");p("su",ya,g.pw)};function Ca(){function a(){for(var b;(b=e[h++])&&"m"!=b[0]&&!b[1].auto;);b&&(M(2,b[0]),b[1].url&&L(b[1].url,b[0]),b[1].libs&&m.d&&m.d(b[1].libs));h<e.length&&setTimeout(a,0)}function b(){0<f--?setTimeout(b,0):a()}var c=l.a("1"),d=l.a(""),f=3,e=I,h=0,k=window.gbarOnReady;if(k)try{k()}catch(r){da(r,"ml","or")}d?p("ldb",a):c?q(window,"load",b):b()}p("rdl",Ca);var Da={D:1,H:2,da:3,p:4,W:5,M:6,F:7,g:8,ha:9,U:10,L:11,T:12,S:13,N:14,Q:15,P:16,fa:17,w:18,O:19,ga:20,ea:21,u:22,G:23,ja:24,ka:25,ia:26,A:27,j:28,o:29,k:30,ca:31,Z:32,$:33,J:34,K:35,ba:36,aa:37,Y:38,B:39,R:40,v:41,X:42,V:43,h:48,C:49,I:500},Y=[1,2,3,4,5,6,9,10,11,13,14,28,29,30,34,35,37,38,39,40,41,42,43,48,49,500];var Z=l.b("0.001",1E-4),Ea=l.b("1",1),Fa=!1,Ga=!1;if(l.a("1")){var Ha=Math.random();Ha<=Z&&(Fa=!0);Ha<=Ea&&(Ga=!0)}var Ia=Da,$=null;function Ja(){var a=0,b=function(b,d){l.a(d)&&(a|=b)};b(1,"");b(2,"");b(4,"");b(8,"");return a}
function Ka(a,b){var c=Z,d=Fa,f;f=a;if(!$){$={};for(var e=0;e<Y.length;e++){var h=Y[e];$[h]=!0}}if(f=!!$[f])c=Ea,d=Ga;if(d){d=encodeURIComponent;g.rp?(f=g.rp(),f="-1"!=f?f:"1"):f="1";c=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&oge=",a,"&ogex=",d("17483"),"&ogev=",d("lS7uU5KROoa7sQTklYLgAQ"),"&ogf=",g.bv.f,"&ogp=",d("22"),"&ogrp=",d(f),"&ogsr=",Math.round(1/c),"&ogv=",d("1407723702.0"),"&oggv="+
d("es_plusone_gc_20140723.0_p0"),"&ogd=",d("com"),"&ogl=",d("en"),"&ogc=",d("DEU"),"&ogus=",Ja()];if(b){"ogw"in b&&(c.push("&ogw="+b.ogw),delete b.ogw);var k;f=b;e=[];for(k in f)0!=e.length&&e.push(","),e.push(La(k)),e.push("."),e.push(La(f[k]));k=e.join("");""!=k&&(c.push("&ogad="),c.push(d(k)))}F(c.join(""))}}function La(a){"number"==typeof a&&(a+="");return"string"==typeof a?a.replace(".","%2E").replace(",","%2C"):a}C=Ka;p("il",C,G);var Ma={};H.il=Ma;setTimeout(function(){C(Ia.g)},0);}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var b=window.gbar.i.i;var c=window.gbar;var f=function(d){try{var a=document.getElementById("gbom");a&&d.appendChild(a.cloneNode(!0))}catch(e){b(e,"omas","aomc")}};c.aomc=f;}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var a=window.gbar;a.mcf("pm",{p:""});}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{window.gbar.rdl();}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
if (window['_OC_timingAction']) {window['_OC_timingAction']('patents_refpage');}</script><style>#gbar,#guser{font-size:13px;padding-top:1px !important;}#gbar{float:left;height:22px}#guser{padding-bottom:7px !important;text-align:right}.gbh,.gbd{border-top:1px solid #c9d7f1;font-size:1px}.gbh{height:0;position:absolute;top:24px;width:100%}#gbs,.gbm{background:#fff;left:0;position:absolute;text-align:left;visibility:hidden;z-index:1000}.gbm{border:1px solid;border-color:#c9d7f1 #36c #36c #a2bae7;z-index:1001}.gb1{margin-right:.5em}.gb1,.gb3{zoom:1}.gb2{display:block;padding:.2em .5em}.gb2,.gb3{text-decoration:none !important;border-bottom:none}a.gb1,a.gb4{text-decoration:underline !important}a.gb1,a.gb2,a.gb3,a.gb4{color:#00c !important}.gbi .gb3,.gbi .gb2,.gbi .gb4{color:#dd8e27 !important}.gbf .gb3,.gbf .gb2,.gbf .gb4{color:#900 !important}a.gb2:hover{background:#36c;color:#fff !important}#gbar .gbz0l{color:#000 !important;cursor:default;font-weight:bold;text-decoration:none !important}
#gbar { padding:.3em .6em !important;}</style></head><body  topmargin="3" marginheight="3"><div id=gbar><nobr><a onclick=gbar.qs(this);gbar.logger.il(1,{t:1}); class=gb1 id=gb_1 href="https://www.google.com/search?sa=N&tab=tw">Search</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:2}); class=gb1 id=gb_2 href="http://www.google.com/search?hl=en&tbm=isch&source=og&sa=N&tab=ti">Images</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:8}); class=gb1 id=gb_8 href="http://maps.google.com/maps?hl=en&sa=N&tab=tl">Maps</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:78}); class=gb1 id=gb_78 href="https://play.google.com/?hl=en&sa=N&tab=t8">Play</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:36}); class=gb1 id=gb_36 href="http://www.youtube.com/results?sa=N&tab=t1">YouTube</a> <a onclick=gbar.logger.il(1,{t:5}); class=gb1 id=gb_5 href="http://news.google.com/nwshp?hl=en&tab=tn">News</a> <a onclick=gbar.logger.il(1,{t:23}); class=gb1 id=gb_23 href="https://mail.google.com/mail/?tab=tm">Gmail</a> <a onclick=gbar.logger.il(1,{t:25}); class=gb1 id=gb_25 href="https://drive.google.com/?tab=to">Drive</a> <a class=gb3 href="http://www.google.com/intl/en/options/" onclick="this.blur();gbar.tg(event);return !1" aria-haspopup=true><u>More</u> <small>&#9660;</small></a><div class=gbm id=gbi><a onclick=gbar.logger.il(1,{t:24}); class=gb2 id=gb_24 href="https://www.google.com/calendar?tab=tc">Calendar</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:51}); class=gb2 id=gb_51 href="http://translate.google.com/?hl=en&sa=N&tab=tT">Translate</a><a onclick=gbar.logger.il(1,{t:17}); class=gb2 id=gb_17 href="http://www.google.com/mobile/?hl=en&tab=tD">Mobile</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:10}); class=gb2 id=gb_10 href="http://www.google.com/search?hl=en&tbo=u&tbm=bks&source=og&sa=N&tab=tp">Books</a><a onclick=gbar.logger.il(1,{t:212}); class=gb2 id=gb_212 href="https://wallet.google.com/manage/?tab=ta">Wallet</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:6}); class=gb2 id=gb_6 href="http://www.google.com/search?hl=en&tbo=u&tbm=shop&source=og&sa=N&tab=tf">Shopping</a><a onclick=gbar.logger.il(1,{t:30}); class=gb2 id=gb_30 href="http://www.blogger.com/?tab=tj">Blogger</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:27}); class=gb2 id=gb_27 href="http://www.google.com/finance?sa=N&tab=te">Finance</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:31}); class=gb2 id=gb_31 href="https://plus.google.com/photos?sa=N&tab=tq">Photos</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:12}); class=gb2 id=gb_12 href="http://www.google.com/search?hl=en&tbo=u&tbm=vid&source=og&sa=N&tab=tv">Videos</a><div class=gb2><div class=gbd></div></div><a onclick=gbar.logger.il(1,{t:66}); href="http://www.google.com/intl/en/options/" class=gb2>Even more &raquo;</a></div></nobr></div><div id=guser width=100%><nobr><span id=gbn class=gbi></span><span id=gbf class=gbf></span><span id=gbe></span><a target=_top id=gb_70 href="https://www.google.com/accounts/Login?service=&continue=http://www.google.com/patents%3Fhl%3Den&hl=en" class=gb4>Sign in</a><div style="display: none"><div class=gbm id=gbd5 aria-owner=gbg5><div class=gbmc><ol id=gbom class=gbmcc></ol></div></div></div></nobr></div><div class=gbh style=left:0></div><div class=gbh style=right:0></div><div role="alert" style="position: absolute; left: 0; right: 0;"><a href="http://www.google.com/patents/us6730778?hl=en&amp;output=html_text" title="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."><img border="0" src="http://www.google.com/images/cleardot.gif"alt="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."></a></div><div id="guser"><nobr></nobr></div><div style="clear:both;"></div><div id="gb-top-search-box" class="gb-top-search-box-small gb-reset"><table><tr><td class="logo"><a href="http://www.google.com/patents" class="logo-link"><img class="logo-img" src="/intl/en/images/logos/google_logo_41.png" alt="Go to Google Books Home" height="41"/></a></td><td><form action="http://www.google.com/search" name="f" id="vheadf" method="get"><span id="hf"></span><input type="hidden" name="tbm" value="pts"/><input type="hidden" name="tbo" value="1"/><input type="hidden" name="hl" value="en"/><table><tr><td><div class="inputs"><table><tr><td><div class="text-input"><input type="text" name="q" id="vheadq" class="text" maxlength="2048" size="31" value="" title="Search Patents" accesskey="s" autocomplete="off"/><script>window._OC_autoDir &&window._OC_autoDir('vheadq', 'tia-vheadq');</script></div></td><td><div class="submit-input"><input name="btnG" class="submit" type="submit" value=""/></div></td></tr></table></div></td><td class="col-ext-links"><div class="ext-links"><a href="http://www.google.com/advanced_patent_search">&lt;nobr&gt;Advanced Patent Search&lt;/nobr&gt;</a></div></td></tr></table></form></td></tr></table></div><div class="kd-appbar"><h2 class="kd-appname"><a href="/patents">Patents</a></h2><div class="kd-buttonbar left" id="left-toolbar-buttons"><a id="appbar-write-review-link" href=""></a><a id="appbar-view-print-sample-link" href=""></a><a id="appbar-view-ebook-sample-link" href=""></a><a id="appbar-patents-prior-art-finder-link" href="https://www.google.com/patents/related/US6730778"></a><a id="appbar-patents-discuss-this-link" href="http://www.google.com/url?id=XkhmBAABERAJ&amp;q=http://patents.stackexchange.com/redirect/google-patents%3Fpatent%3DUS6730778&amp;usg=AFQjCNGcv8o6wrOHHEDc-KGbnq_CEjCf7Q" data-is-grant="true"></a><a id="appbar-read-patent-link" href="//docs.google.com/viewer?url=patentimages.storage.googleapis.com/pdfs/US6730778.pdf"></a><a id="appbar-download-pdf-link" href="//patentimages.storage.googleapis.com/pdfs/US6730778.pdf"></a><a class="appbar-application-grant-link" data-label="Application" href="/patents/US20020177187"></a><a class="appbar-application-grant-link" data-selected="true" data-label="Grant" href="/patents/US6730778"></a></div><div class="kd-buttonbar right" id="right-toolbar-buttons"></div></div><div id="books-microdata" itemscope=""itemtype="http://schema.org/Book"itemid="http://www.google.com/patents/US6730778" style="display:none"><span itemprop="description">The present invention provides isolated nucleic acid molecules comprising a polynucleotide encoding either of two alternative splice variants of human sel-10, one of which is expressed in hippocampal cells, and one of which is expressed in mammary cells. The invention also provides isolated sel-10 polypeptides...</span><span itemprop="url">http://www.google.com/patents/US6730778?utm_source=gb-gplus-share</span><span class="main-title" itemprop="name">Patent US6730778 - Nucleotide sequences coding polypeptide for use in the treatment of nervous system disorders</span><img itemprop="image" src="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"alt="Patent US6730778 - Nucleotide sequences coding polypeptide for use in the treatment of nervous system disorders" title="Patent US6730778 - Nucleotide sequences coding polypeptide for use in the treatment of nervous system disorders"></div><div style="display: none"><ol id="ofe-gear-menu-contents" class="gbmcc"><li class="gbe gbmtc"><a class="gbmt goog-menuitem-content" id="" href="http://www.google.com/advanced_patent_search">Advanced Patent Search</a></li></ol></div><table id="viewport_table" cellpadding="0" style="clear:both" cellspacing="0"><tr><td id="viewport_td"><div class=vertical_module_list_row><div id=intl_patents class=about_content><div id=intl_patents_v><table class="patent-bibdata"><tr><td class="patent-bibdata-heading">Publication number</td><td class="single-patent-bibdata">US6730778 B2</td></tr><tr><td class="patent-bibdata-heading">Publication type</td><td class="single-patent-bibdata">Grant</td></tr><tr><td class="patent-bibdata-heading">Application number</td><td class="single-patent-bibdata">US 09/328,877</td></tr><tr><td class="patent-bibdata-heading">Publication date</td><td class="single-patent-bibdata">May 4, 2004</td></tr><tr><td class="patent-bibdata-heading">Filing date</td><td class="single-patent-bibdata">Jun 9, 1999</td></tr><tr><td class="patent-bibdata-heading">Priority date<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The priority date is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the date listed."></span></td><td class="single-patent-bibdata">Dec 19, 1997</td></tr><tr><td class="patent-bibdata-heading">Fee status<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The fee status is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the status or dates listed."></span></td><td class="single-patent-bibdata">Lapsed</td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Also published as</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/EP1192251A1">EP1192251A1</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20020177187">US20020177187</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20040116672">US20040116672</a>, </span><span class="patent-bibdata-value"><a href="/patents/WO2000075328A1">WO2000075328A1</a></span></span></td></tr><tr class="patent-bibdata-list-row alternate-patent-number"><td class="patent-bibdata-heading">Publication number</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value">09328877, </span><span class="patent-bibdata-value">328877, </span><span class="patent-bibdata-value">US 6730778 B2, </span><span class="patent-bibdata-value">US 6730778B2, </span><span class="patent-bibdata-value">US-B2-6730778, </span><span class="patent-bibdata-value">US6730778 B2, </span><span class="patent-bibdata-value">US6730778B2</span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Inventors</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Mark+E.+Gurney%22">Mark E. Gurney</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Adele+M.+Pauley%22">Adele M. Pauley</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Jinhe+Li%22">Jinhe Li</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Original Assignee</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=inassignee:%22Pharmacia+And+Upjohn+Company%22">Pharmacia And Upjohn Company</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Export Citation</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/US6730778.bibtex">BiBTeX</a>, </span><span class="patent-bibdata-value"><a href="/patents/US6730778.enw">EndNote</a>, </span><span class="patent-bibdata-value"><a href="/patents/US6730778.ris">RefMan</a></span></span></td></tr><tr class="patent-internal-links"><td colspan=2><span class="patent-bibdata-value"><a href="#backward-citations">Patent Citations</a> (1),</span> <span class="patent-bibdata-value"><a href="#npl-citations">Non-Patent Citations</a> (29),</span> <span class="patent-bibdata-value"><a href="#classifications">Classifications</a> (22),</span> <span class="patent-bibdata-value"><a href="#legal-events">Legal Events</a> (6)</span> </td></tr><tr><td colspan=2 class="patent-bibdata-external-link-spacer-top"></td></tr><tr class="patent-bibdata-external-link-spacer-bottom"></tr><tr><td colspan=2><span class="patent-bibdata-heading">External Links:&nbsp;</span><span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=XkhmBAABERAJ&q=http://patft.uspto.gov/netacgi/nph-Parser%3FSect2%3DPTO1%26Sect2%3DHITOFF%26p%3D1%26u%3D/netahtml/PTO/search-bool.html%26r%3D1%26f%3DG%26l%3D50%26d%3DPALL%26RefSrch%3Dyes%26Query%3DPN/6730778&usg=AFQjCNFCZXNg7_nvKWMwLY5_LJjB923_8Q">USPTO</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=XkhmBAABERAJ&q=http://assignments.uspto.gov/assignments/q%3Fdb%3Dpat%26pat%3D6730778&usg=AFQjCNEs3avNCWfoZKGw1K3zAWnrC6HWcg">USPTO Assignment</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=XkhmBAABERAJ&q=http://worldwide.espacenet.com/publicationDetails/biblio%3FCC%3DUS%26NR%3D6730778B2%26KC%3DB2%26FT%3DD&usg=AFQjCNHzJDgvTGbEf4dxO-twjskdTj2YDA">Espacenet</a></span></span></td></tr><tr class="patent-bibdata-group-spacer"></tr></table><div class="number-and-title"><span class="patent-title"><invention-title mxw-id="PT129227615" lang="EN" load-source="ifi">Nucleotide sequences coding polypeptide for use in the treatment of nervous system disorders</invention-title></span><br><span class="patent-number">US 6730778 B2</span></div><div class="patent-section patent-abstract-section"><div class="patent-section-header"><span class="patent-section-title">Abstract</span></div><div class="patent-text"><abstract mxw-id="PA50661524" lang="EN" load-source="patent-office"> <div class="abstract">The present invention provides isolated nucleic acid molecules comprising a polynucleotide encoding either of two alternative splice variants of human sel-10, one of which is expressed in hippocampal cells, and one of which is expressed in mammary cells. The invention also provides isolated sel-10 polypeptides and cell lines which express them in which Aβ processing is altered.</div>
  </abstract></div></div><div class="patent-section patent-drawings-section"><div class="patent-section-header"><span class="patent-section-title">Images<span class="patent-section-count">(7)</span></span></div><div class="patent-drawings-body"><div class="patent-drawings-carousel"><div class="drawings"><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6730778B2/US06730778-20040504-D00001.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6730778B2/US06730778-20040504-D00001.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6730778B2/US06730778-20040504-D00002.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6730778B2/US06730778-20040504-D00002.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6730778B2/US06730778-20040504-D00003.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6730778B2/US06730778-20040504-D00003.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6730778B2/US06730778-20040504-D00004.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6730778B2/US06730778-20040504-D00004.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6730778B2/US06730778-20040504-D00005.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6730778B2/US06730778-20040504-D00005.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6730778B2/US06730778-20040504-D00006.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6730778B2/US06730778-20040504-D00006.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6730778B2/US06730778-20040504-D00007.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6730778B2/US06730778-20040504-D00007.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div></div></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img" alt="Previous page"src="/googlebooks/images/kennedy/page_left.png"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img" alt="Next page"src="/googlebooks/images/kennedy/page_right.png"width="21" height="21" /></div></div></div><div class="patent-post-drawings"></div><div class="patent-section patent-claims-section"><div class="patent-section-header"><span class="patent-section-title">Claims<span class="patent-section-count">(10)</span></span></div><div class="patent-text"><div mxw-id="PCLM8666754" lang="EN" load-source="patent-office" class="claims">
    <claim-statement>What is claimed is: </claim-statement> <div class="claim"> <div num="1" id="US-6730778-B2-CLM-00001" class="claim">
      <div class="claim-text">1. An isolated human sel-10 polypeptide comprising SEQ ID NO:7.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="2" id="US-6730778-B2-CLM-00002" class="claim">
      <div class="claim-text">2. The isolated sel-10 polypeptide of <claim-ref idref="US-6730778-B2-CLM-00001">claim 1</claim-ref>, wherein said polypeptide comprises the amino acid sequence of SEQ ID NO:3.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="3" id="US-6730778-B2-CLM-00003" class="claim">
      <div class="claim-text">3. The isolated sel-10 polypeptide of <claim-ref idref="US-6730778-B2-CLM-00001">claim 1</claim-ref>, wherein said polypeptide comprises the amino acid sequence of SEQ ID NO:4.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="4" id="US-6730778-B2-CLM-00004" class="claim">
      <div class="claim-text">4. The isolated sel-10 polypeptide of <claim-ref idref="US-6730778-B2-CLM-00001">claim 1</claim-ref>, wherein said polypeptide comprises the amino acid sequence of SEQ ID NO:5.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="5" id="US-6730778-B2-CLM-00005" class="claim">
      <div class="claim-text">5. The isolated sel-10 polypeptide of <claim-ref idref="US-6730778-B2-CLM-00001">claim 1</claim-ref>, wherein said polypeptide comprises the amino acid sequence of SEQ ID NO:6.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="6" id="US-6730778-B2-CLM-00006" class="claim">
      <div class="claim-text">6. The isolated sel-10 polypeptide of <claim-ref idref="US-6730778-B2-CLM-00001">claim 1</claim-ref>, wherein said polypeptide comprises the amino acid sequence of SEQ ID NO:8.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="7" id="US-6730778-B2-CLM-00007" class="claim">
      <div class="claim-text">7. The isolated sel-10 polypeptide of <claim-ref idref="US-6730778-B2-CLM-00001">claim 1</claim-ref>, wherein said polypeptide comprises the amino acid sequence of SEQ ID NO:9.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="8" id="US-6730778-B2-CLM-00008" class="claim">
      <div class="claim-text">8. The isolated sel-10 polypeptide of <claim-ref idref="US-6730778-B2-CLM-00001">claim 1</claim-ref>, wherein said polypeptide comprises the amino acid sequence of SEQ ID NO:21.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="9" id="US-6730778-B2-CLM-00009" class="claim">
      <div class="claim-text">9. The isolated sel-10 polypeptide of <claim-ref idref="US-6730778-B2-CLM-00001">claim 1</claim-ref>, wherein said polypeptide comprises the amino acid sequence of SEQ ID NO:25.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="10" id="US-6730778-B2-CLM-00010" class="claim">
      <div class="claim-text">10. The isolated sel-10 polypeptide of <claim-ref idref="US-6730778-B2-CLM-00001">claim 1</claim-ref>, wherein said polypeptide comprises the amino acid sequence of SEQ ID NO:27.</div>
    </div>
  </div> </div></div></div><div class="patent-section patent-description-section"><div class="patent-section-header"><span class="patent-section-title">Description</span></div><div class="patent-text"><div mxw-id="PDES54174476" lang="EN" load-source="patent-office" class="description">
    <heading>CROSS-REFERENCE TO RELATED APPLICATIONS</heading> <p>This application claims the benefit of the following provisional application: U.S. Ser. No. 60/068,243, now abandoned, filed Dec. 19, 1997, under 35 USC 119(e)(1). This application is a continuation-in-part of the following application: U.S. Ser. No. 09/213,888, filed Dec. 17, 1998, under 35 USC 120.</p>
    <heading>FIELD OF THE INVENTION</heading> <p>The present invention provides isolated nucleic acid molecules comprising a polynucleotide encoding either of two alternative splice variants of human sel-10, one of which is expressed in hippocampal cells, and one of which is expressed in mammary cells. The invention also provides isolated sel-10 polypeptides.</p>
    <heading>BACKGROUND OF THE INVENTION</heading> <p>Alzheimer's disease (AD) is a degenerative disorder of the central nervous system which causes progressive memory and cognitive decline during mid to late adult life. The disease is accompanied by a wide range of neuropathologic features including extracellular amyloid plaques and intra-neuronal neurofibrillary tangles. (Sherrington, R., et al.; <i>Nature </i>375: 754-60 (1995)). Although the pathogenic pathway leading to AD is not well understood, several genetic loci are known to be involved in the development of the disease.</p>
    <p>Genes associated with early onset Alzheimer's disease (AD) have been identified by the use of mapping studies in families with early-onset AD. These studies have shown that genetic loci on chromosomes 1 and 14 were likely to be involved in AD. Positional cloning of the chromosome 14 locus identified a novel mutant gene encoding an eight-transmembrane domain protein which subsequently was named presenilin-1 (PS-1). (Sherrington, R., et al.; <i>Nature </i>375: 754-60 (1995)). Blast search of the human EST database revealed a single EST exhibiting homology to PS-1, designated presenilin-2 (PS-2) which was shown to be the gene associated with AD on chromosome 1. (Levy-Lahad, E. et al., <i>Science </i>269:973-977 (1995); Rogaev, E. I., et al., <i>Nature </i>376: 775-8 (1995); Li, J. et al., <i>Proc. Natl. Acad. Sci. U.S.A</i>. 92: 12180-12184 (1995)).</p>
    <p>Mutations in PS-1 and PS-2 that are associated with Alzheimer's disease are primarily missense mutations. Both PS-1 and PS-2 undergo proteolytic processing, which can be altered by the point mutations found in familial Alzheimer's disease [Perez-Tur, J. et al., <i>Neuroreport </i>7: 297-301 (1995); Mercken, M. et al., <i>FEBS Lett</i>. 389: 297-303 (1996)]. PS-1 gene expression is widely distributed across tissues, while the highest levels of PS-2 mRNA are found in pancreas and skeletal muscle. (Li, J. et al., <i>Proc. Natl. Acad. Sci. U.S.A</i>. 92: 12180-12184 (1995); Jinhe Li, personal communication). The highest levels of PS-2 protein, however, are found in brain (Jinhe Li, personal communication). Both PS-1 and PS-2 proteins have been localized to the endoplasmic reticulum, the Golgi apparatus, and the nuclear envelope. (Jinhe Li, personal communication; Kovacs, D. M. et al., <i>Nat. Med</i>. 2:224-229 (1996); Doan, A. et al., <i>Neuron </i>17: 1023-1030 (1996)). Mutations in either the PS-1 gene or the PS-2 gene alter the processing of the amyloid protein precursor (APP) such that the ratio of A-beta<sub>1-42 </sub>is increased relative to A-beta<sub>1-40 </sub>(Scheuner, D. et al., <i>Nat. Med</i>. 2: 864-870 (1996)). When coexpressed in transgenic mice with human APP, a similar increase in the ratio of A-beta<sub>1-42 </sub>as compared to A-beta<sub>1-40 </sub>is observed (Borchelt, D. R. et al., <i>Neuron </i>17: 1005-1013 (1996); Citron, M. et al., <i>Nat. Med</i>. 3: 67-72 (1997); Duff, K. et al., <i>Nature </i>383: 710-713 (1996)), together with an acceleration of the deposition of A-beta in amyloid plaques (Borchelt et al., <i>Neuron </i>19: 939 (1997).</p>
    <p>Despite the above-described observations made with respect to the role of PS-1 and PS-2 in AD, their biological function remains unknown, placing them alongside a large number of human disease genes having an unknown biological function. Where the function of a gene or its product is unknown, genetic analysis in model organisms can be useful in placing such genes in known biochemical or genetic pathways. This is done by screening for extragenic mutations that either suppress or enhance the effect of mutations in the gene under analysis. For example, extragenic suppressors of loss-of-function mutations in a disease gene may turn on the affected genetic or biochemical pathway downstream of the mutant gene, while suppressers of gain-of-function mutations will probably turn the pathway off.</p>
    <p>One model organism that can be used in the elucidation of the function of the presenilin genes is <i>C. elegans</i>, which contains three genes having homology to PS-1 and PS-2, with sel-12 having the highest degree of homology to the genes encoding the human presenilins. Sel-12 was discovered in a screen for genetic suppressers of an activated notch receptor, lin-12(d) (Levitan, D. et al., <i>Nature </i>377: 351-354 (1995)). Lin-12 functions in development to pattern cell lineages. Hypermorphic mutations such as lin-12(d), which increase lin-12 activity, cause a “multi-vulval” phenotype, while hypomorphic mutations which decrease activity cause eversion of the vulva, as well as homeotic changes in several other cell lineages (Greenwald, I., et al., <i>Nature </i>346: 197-199 (1990); Sundaram, M. et al., <i>Genetics </i>135: 755-763 (1993)). Sel-12 mutations suppress hypermorphic lin-12(d) mutations, but only if the lin-12(d) mutations activate signaling by the intact lin-12(d) receptor (Levitan, D. et al., <i>Nature </i>377: 351-354 (1995)). Lin-12 mutations that truncate the cytoplasmic domain of the receptor also activate signaling (Greenwald, I., et al., <i>Nature </i>346: 197-199 (1990)), but are not suppressed by mutations of sel-12 (Levitan, D. et al., <i>Nature </i>377: 351-354 (1995)). This implies that sel-12 mutations act upstream of the lin-12 signaling pathway, perhaps by decreasing the amount of functional lin-12 receptor present in the plasma membrane. In addition to suppressing certain lin-12 hypermorphic mutations, mutations to sel-12 cause a loss-of-function for egg laying, and thus internal accumulation of eggs, although the mutants otherwise appear anatomically normal (Levitan, D. et al., <i>Nature </i>377: 351-354 (1995)). Sel-12 mutants can be rescued by either human PS-1 or PS-2, indicating that sel-12, PS-1 and PS-2 are functional homologues (Levitan, D., et al., <i>Proc. Natl. Acad. Sci. U.S.A</i>. 93: 14940-14944 (1996)).</p>
    <p>A second gene, sel-10, has been identified in a separate genetic screen for suppressors of lin-12 hypomorphic mutations. Loss-of-function mutations in sel-10 restore signaling by lin-12 hypomorphic mutants. As the lowering of sel-10 activity elevates lin-12 activity, it can be concluded that sel-10 acts as a negative regulator of lin-12 signaling. Sel-10 also acts as a negative regulator of sel-12, the <i>C. elegans </i>presenilin homologue (Levy-Lahad, E. et al., <i>Science </i>269:973-977 (1995)). Loss of sel-10 activity suppresses the egg laying defect associated with hypomorphic mutations in sel-12 (Iva Greenwald, personal communication). The effect of loss-of-function mutations to sel-10 on lin-12 and sel-12 mutations indicates that sel-10 acts as a negative regulator of both lin-12/notch and presenilin activity. Thus, a human homologue of <i>C. elegans </i>sel-10 would be expected to interact genetically and/or physiologically with human presenilin genes in ways relevant to the pathogenesis of Alzheimer's Disease.</p>
    <p>In view of the foregoing, it will be clear that there is a continuing need for the identification of genes related to AD, and for the development of assays for the identification of agents capable of interfering with the biological pathways that lead to AD.</p>
    <heading>INFORMATION DISCLOSURE</heading> <p>Hubbard E J A, Wu G, Kitajewski J, and Greenwald I (1997) Sel-10, a negative regulator of lin-12 activity in <i>Caenorhabditis elegans</i>, encodes a member of the CDC4 family of proteins. Genes &amp; Dev 11:3182-3193.</p>
    <p>Greenwald-I; Seydoux-G (1990) Analysis of gain-of-function mutations of the lin-12 gene of <i>Caenorhabditis elegans. Nature</i>. 346: 197-9</p>
    <p>Kim T-W, Pettingell W H, Hallmark O G, Moir R D, Wasco W, Tanzi R (1997) Endoproteolytic cleavage and proteasomal degradation of presenilin 2 in transfected cells. J Biol Chem 272:11006-11010.</p>
    <p>Levitan-D; Greenwald-I (1995) Facilitation of lin-12-mediated signalling by sel-12, a <i>Caenorhabditis elegans </i>S182 Alzheimer's disease gene. <i>Nature</i>. 377: 351-4.</p>
    <p>Levitan-D; Doyle-T G; Brousseau-D; Lee-M K; Thinakaran-G; Slunt-H H; Sisodia-S S; Greenwald-I (1996) Assessment of normal and mutant human presenilin function in <i>Caenorhabditis elegans. Proc. Natl. Acad. Sci. U.S.A</i>. 93: 14940-4.</p>
    <p>Sundaram-M; Greenwald-I (1993) Suppressors of a lin-12 hypomorph define genes that interact with both lin-12 and glp-1 in <i>Caenorhabditis elegans. Genetics</i>. 135: 765-83.</p>
    <p>Sundaram-M; Greenwald-I (1993) Genetic and phenotypic studies of hypomorphic lin-12 mutants in <i>Caenorhabditis elegans. Genetics</i>. 135: 755-63.</p>
    <p>F55B12.3 GenPep Report (WMBL locus CEF55B12, accession z79757).</p>
    <p>WO 97/11956</p>
    <heading>SUMMARY OF THE INVENTION</heading> <p>The present invention provides isolated nucleic acid molecules comprising a polynucleotide encoding human sel-10, which is expressed in hippocampal cells and in mammary cells. Unless otherwise noted, any reference herein to sel-10 will be understood to refer to human sel-10, and to encompass both hippocampal and mammary sel-10. Fragments of hippocampal sel-10 and mammary sel-10 are also provided.</p>
    <p>In a preferred embodiment, the invention provides an isolated nucleic acid molecule comprising a polynucleotide having a sequence at least 95% identical to a sequence selected from the group consisting of:</p>
    <p>(a) a nucleotide sequence encoding a human sel-10 polypeptide having the complete amino acid sequence selected from the group consisting of SEQ ID NO:3, SEQ ID NO:4, SEQ ID NO:5, SEQ ID NO:6, and SEQ ID NO:7, or as encoded by the cDNA clone contained in ATCC Deposit No.98978;</p>
    <p>(b) a nucleotide sequence encoding a human sel-10 polypeptide having the complete amino acid sequence selected from the group consisting of SEQ ID NO:8, SEQ ID NO:9, and SEQ ID NO:10, or as encoded by the cDNA clone contained in ATCC Deposit No. 98979; and</p>
    <p>(c) a nucleotide sequence complementary to the nucleotide sequence of (a) or (b).</p>
    <p>In another aspect, the invention provides an isolated nucleic acid molecule comprising a polynucleotide which hybridizes under stringent conditions to a polynucleotide encoding sel-10, or fragments thereof.</p>
    <p>The present invention also provides vectors comprising the isolated nucleic acid molecules of the invention, host cells into which such vectors have been introduced, and recombinant methods of obtaining a sel-10 polypeptide comprising culturing the above-described host cell and isolating the sel-10 polypeptide.</p>
    <p>In another aspect, the invention provides isolated sel-10 polypeptides, as well as fragments thereof. In a preferred embodiment, the sel-10 polypeptides have an amino acid sequence selected from the group consisting of SEQ ID NO:3, 4, 5, 6, 7, 8, 9, and 10. Isolated antibodies, both polyclonal and monoclonal, that bind specifically to sel-10 polypeptides are also provided.</p>
    <heading>BRIEF DESCRIPTION OF THE FIGURES</heading> <p>FIGS. <b>1</b>A and <b>1</b>B: FIGS. 1A and 1B are western blots showing protein expression in HEK293 cells transfected with PS1-C-FLAG, 6-myc-N-sel-10, and APP695NL-KK cDNAs.</p>
    <p>FIGS. <b>2</b>A and <b>2</b>B: FIGS. 2A and 2B are northern blots. FIG. 2A is a multiple tissue northern blot probed with the common, mammary sel-10 mRNA showing ubiquitous expression of the 6.5- and 4.5-Kb transcripts. FIG. 2B is a multiple tissue northern blot showing limited expression of the hippocampal sel-10 mRNA only in brain.</p>
    <p>FIGS. 3A, <b>3</b>B and <b>3</b>C. FIGS. 3A, <b>3</b>B and <b>3</b>C are western blots. FIG. 3A demonstrates that SEL-10-myc forms a complex with PS1 as shown by immunoprecipitation with anti-PS1 loop antibody. HEK293 cells were transfected with constructs containing SEL-10-myc or PS1, cotransfected with both constructs, transfected with the corresponding vector only control (pcDNA3 or pCS, respectively), or mock transfected without DNA. Cultures were treated with lactacystin (12 μM) to inhibit proteasome function as indicated. The immunoprecipitates were loaded on the left side of the gel and cell lysates on the right. Controls for immunoprecipitation were a nonspecific IgG and just anti-PS1 loop antibody with protein G beads with no cell lysate (“no protein”). The immunoblot was probed with anti-myc antibody to detect SEL-10-myc in the immunoprecipitates and cell lysates. SEL-10-myc is expressed in lysates from cells transfected with SEL-10-myc or cotransfected with SEL-10-myc and PS1, but was detected only as a complex with PS1 in cells cotransfected with the SEL-10-myc and PS1 constructs. FIG. 3B demonstrates that the SEL-10-myc/PS1 complex could be detected only when proteasome degradation was inhibited with lactacystin. FIG. 3C demonstrates that immunoprecipitation of the SEL-10-myc/PS1 complex with anti-myc antibody indicates that SEL-10-myc associates primarily with full length and high molecular weight forms of PS1. Immunoprecipitates were probed with either anti-PS1 loop or anti-myc antibodies. Longer exposures of the ECL developed Western blot reveal very low levels of PS1 -NTF in the immunoprecipitate with SEL-10-myc, but fail to detect PS1-CTF.</p>
    <p>FIGS. 4A and 4B. FIGS. 4A and 4B are western blots. FIG. 4A demonstrates that SEL-10-myc stimulates ubiquitination of PS1. An HEK293 cell line with stable expression of PS1 was transfected with SEL-10-myc. Immunoprecipitation with an anti-PS1 loop antibody was followed by detection with an anti-ubiquitin antibody. FIG. 4B demonstrates that increased expression of SEL-10-myc leads to a decrease in PS1-NTF and PS1-CTF with a corresponding increase in the amount of full length PS1. Cell lysates were immunoprecipitated with anti-PS1 loop antibody and then probed on the Western blot with the same antibody to detect PS1 and its processing products.</p>
    <p>FIGS. 5A, <b>5</b>B and <b>5</b>C. FIGS. 5A, <b>5</b>B and <b>5</b>C are graphs. FIG. 5A demonstrates that transient co-expression of SEL-10-myc with APP increases production of Aβ1-40 and Aβ1-42. The effect is additive with co-expressed PS1. FIG. 5B demonstrates that stable expression of SEL-10-myc or PS1 increases endogenous Aβ production. FIG. 5C demonstrates that transient expression of APP in the stable cell lines increases exogenous Aβ production over the level seen in a cell line transformed with the pcDNA3.1 vector control.</p>
    <heading>DETAILED DESCRIPTION OF THE INVENTION</heading> <p>The present invention provides isolated nucleic acid molecules comprising a polynucleotide encoding human sel-10. The nucleotide sequence of human hippocampal sel-10 (hhsel-10), which sequence is given in SEQ ID NO:1, encodes five hhsel-10 polypeptides (hhsel-10-(1), hhsel-10-(2), hhsel-10-(3), hhsel-10-(4), and hhsel-10-(5), referred to collectively herein as hhsel-10). The nucleotide sequence of human mammary sel-10 (hmsel-10), which sequence is given in SEQ ID NO:2, encodes three hmsel-10 polypeptides (hmSel-10-(1), hmSel-10-(2), and hmsel-10-(3), referred to collectively herein as hmsel-10). The nucleotide sequences of the hhsel-10 polynucleotides are given in SEQ ID NO. 1, where nucleotide residues 45-1928 of SEQ ID NO. 1 correspond to hhsel-10-(1), nucleotide residues 150-1928 of SEQ ID NO. 1 correspond to hhSel-10-(2), nucleotide residues 267-1928 of SEQ ID NO. 1 correspond to hhSel-10-(3), nucleotide residues 291-1928 of SEQ ID NO. 1 correspond to hhSel-10-(4), and nucleotide residues 306-1928 of SEQ ID NO. 1 correspond to hhSel-10-(5). The nucleotide sequences of the hmSel-10 polynucleotides are given in SEQ ID NO. 2, where nucleotide residues 180-1949 of SEQ ID NO. 2 correspond to hmSel-10-(1), nucleotide residues 270-1949 of SEQ ID NO. 2 correspond to hmSel-10-(2), and nucleotide residues 327-1949 of SEQ ID NO. 2 correspond to hmSel-10-(3). The amino acid sequences of the polypeptides encoded by the hhSel-10 and hm-Sel-10 nucleic acid molecules are given as follows: SEQ ID NOS: 3, 4, 5, 6, and 7 correspond to the hhSel-10-(1), hhSel-10-(2), hhSel-10-(3). hhSel-10-(4), and hhSel-10-(5) polypeptides, respectively, and SEQ ID NOS: 8, 9, and 10 correspond to the hmSel-10-(1), hmSel-10-(2), and hmSel-10-(3) polypeptides, respectively. Unless otherwise noted, any reference herein to sel-10 will be understood to refer to human sel-10, and to encompass all of the hippocampal and mammary sel-10 nucleic acid molecules (in the case of reference to sel-10 nucleic acid, polynucleotide, DNA, RNA, or gene) or polypeptides (in the case of reference to sel-10 protein, polypeptide, amino acid sequence). Fragments of hippocampal sel-10 and mammary sel-10 nucleic acid molecules and polypeptides are also provided.</p>
    <p>The nucleotide sequence of SEQ ID NO:1 was obtained as described in Example 1, and is contained in cDNA clone PNV 102-1, which was deposited on Nov. 9, 1998, at the American Type Culture Collection, 10801 University Blvd., Manassas, Va. 20110, and given accession number 98978. The nucleotide sequence of SEQ ID NO:2 was obtained as described in Example 1, and is contained in cDNA clone PNV 108-2, which was deposited on Nov. 9, 1998, at the American Type Culture Collection, 10801 University Blvd., Manassas, Va. 20110, and given accession number 98979.</p>
    <p>The human sel-10 polypeptides of the invention share homology with <i>C. elegans </i>sel-10, as well as with members of the β-transducin protein family, including yeast CDC4, and human LIS-1. This family is characterized by the presence of an F-box and multiple WD-40 repeats (Li, J., et al., <i>Proc. Natl. Acad. Sci. U.S.A</i>. 92:12180-12184 (1995)). The repeats are 20-40 amino acids long and are bounded by gly-his (GH) and trp-asp (WD) residues. The three dimensional structure of β-transducin indicates that the WD40 repeats form the arms of a seven-bladed propeller like structure (Sondek, J., et al., <i>Nature </i>379:369-374 (1996)). Each blade is formed by four alternating pleats of beta-sheet with a pair of the conserved aspartic acid residues in the protein motif forming the limits of one internal beta strand. WD40 repeats are found in over 27 different proteins which represent diverse functional classes (Neer, E. J., et al., <i>Nature </i>371:297-300 (1994)). These regulate cellular functions including cell division, cell fate determination, gene transcription, signal transduction, protein degradation, mRNA modification and vesicle fusion. This diversity in function has led to the hypothesis that β-transducin family members provide a common scaffolding upon which multiprotein complexes can be assembled.</p>
    <p>The nucleotide sequence given in SEQ ID NO:1 corresponds to the nucleotide sequence encoding hhsel-10, while the nucleotide sequence given in SEQ ID NO:2 corresponds to the nucleotide sequence encoding hmsel-10. The isolation and sequencing of DNA encoding sel-10 is described below in Examples 1 and 2.</p>
    <p>As is described in Examples 1 and 2, automated sequencing methods were used to obtain the nucleotide sequence of sel-10. The sel-10 nucleotide sequences of the present invention were obtained for both DNA strands, and are believed to be 100% accurate. However, as is known in the art, nucleotide sequence obtained by such automated methods may contain some errors. Nucleotide sequences determined by automation are typically at least about 90%, more typically at least about 95% to at least about 99.9% identical to the actual nucleotide sequence of a given nucleic acid molecule. The actual sequence may be more precisely determined using manual sequencing methods, which are well known in the art. An error in sequence which results in an insertion or deletion of one or more nucleotides may result in a frame shift in translation such that the predicted amino acid sequence will differ from that which would be predicted from the actual nucleotide sequence of the nucleic acid molecule, starting at the point of the mutation. The sel-10 DNA of the present invention includes cDNA, chemically synthesized DNA, DNA isolated by PCR, genomic DNA, and combinations thereof. Genomic sel-10 DNA may be obtained by screening a genomic library with the sel-10 cDNA described herein, using methods that are well known in the art. RNA transcribed from sel-10 DNA is also encompassed by the present invention.</p>
    <p>Due to the degeneracy of the genetic code, two DNA sequences may differ and yet encode identical amino acid sequences. The present invention thus provides isolated nucleic acid molecules having a polynucleotide sequence encoding any of the sel-10 polypeptides of the invention, wherein said polynucleotide sequence encodes a sel-10 polypeptide having the complete amino acid sequence of SEQ ID NOs:3-10, or fragments thereof.</p>
    <p>Also provided herein are purified sel-10 polypeptides, both recombinant and non-recombinant. Variants and derivatives of native sel-10 proteins that retain any of the biological activities of sel-10 are also within the scope of the present invention. As is described above, the sel-10 polypeptides of the present invention share homology with yeast CDC4. As CDC4 is known to catalyze ubiquitination of specific cellular proteins (Feldman et al., <i>Cell </i>91:221 (1997)), it may be inferred that sel-10 will also have this activity. Assay procedures for demonstrating such activity are well known, and involve reconstitution of the ubiquitinating system using purified human sel-10 protein together with the yeast proteins Cdc4p, Cdc53p and Skp1p, or their human orthologs, and an E1 enzyme, the E2 enzyme Cdc34p or its human ortholog, ubiquitin, a target protein and an ATP regenerating system (Feldman et al., 1997). Skp1p associates with Cdc4p through a protein domain called an F-box (Bai et al., <i>Cell </i>86:263 (1996)). The F-box protein motif is found in yeast CDC4<i>, C. elegans </i>sel-10, mouse sel-10 and human sel-10. The sel-10 ubiquitination system may be reconstituted with the <i>C. elegans </i>counterparts of the yeast components, e.g., cul-1 (also known as lin-19) protein substituting for Cdc53p (Kipreos et al., <i>Cell </i>85:829 (1996)) and the protein F46A9 substituting for Skp1p, or with their mammalian counterparts, e.g., Cul-2 protein substituting for Cdc53p (Kipreos et al., 1996) and mammalian Skp1p substituting for yeast Skp1p. A phosphorylation system provided by a protein kinase is also included in the assay system as per Feldman et al., 1997.</p>
    <p>Sel-10 variants may be obtained by mutation of native sel-10-encoding nucleotide sequences, for example. A sel-10 variant, as referred to herein, is a polypeptide substantially homologous to a native sel-10 but which has an amino acid sequence different from that of native sel-10 because of one or more deletions, insertions, or substitutions in the amino acid sequence. The variant amino acid or nucleotide sequence is preferably at least about 80% identical, more preferably at least about 90% identical, and most preferably at least about 95% identical, to a native sel-10 sequence. Thus, a variant nucleotide sequence which contains, for example, 5 point mutations for every one hundred nucleotides, as compared to a native sel-10 gene, will be 95% identical to the native protein. The percentage of sequence identity, also termed homology, between a native and a variant sel-10 sequence may also be determined, for example, by comparing the two sequences using any of the computer programs commonly employed for this purpose, such as the Gap program (Wisconsin Sequence Analysis Package, Version 8 for Unix, Genetics Computer Group, University Research Park, Madison Wis.), which uses the algorithm of Smith and Waterman (<i>Adv. Appl. Math</i>. 2: 482-489 (1981)).</p>
    <p>Alterations of the native amino acid sequence may be accomplished by any of a number of known techniques. For example, mutations may be introduced at particular locations by procedures well known to the skilled artisan, such as oligonucleotide-directed mutagenesis, which is described by Walder et al. (<i>Gene </i>42:133 (1986)); Bauer et al. (<i>Gene </i>37:73 (1985)); Craik (<i>BioTechniques</i>, January 1985, pp. 12-19); Smith et al. (<i>Genetic Engineering: Principles and Methods</i>, Plenum Press (1981)); and U.S. Pat. Nos. 4,518,584 and 4,737,462.</p>
    <p>Sel-10 variants within the scope of the invention may comprise conservatively substituted sequences, meaning that one or more amino acid residues of a sel-10 polypeptide are replaced by different residues that do not alter the secondary and/or tertiary structure of the sel-10 polypeptide. Such substitutions may include the replacement of an amino acid by a residue having similar physicochemical properties, such as substituting one aliphatic residue (Ile, Val, Leu or Ala) for another, or substitution between basic residues Lys and Arg, acidic residues Glu and Asp, amide residues Gln and Asn, hydroxyl residues Ser and Tyr, or aromatic residues Phe and Tyr. Further information regarding making phenotypically silent amino acid exchanges may be found in Bowie et al., <i>Science </i>247:1306-1310 (1990). Other sel-10 variants which might retain substantially the biological activities of sel-10 are those where amino acid substitutions have been made in areas outside functional regions of the protein.</p>
    <p>In another aspect, the invention provides an isolated nucleic acid molecule comprising a polynucleotide which hybridizes under stringent conditions to a portion of the nucleic acid molecules described above, e.g., to at least about 15 nucleotides, preferably to at least about 20 nucleotides, more preferably to at least about 30 nucleotides, and still more preferably to at least about from 30 to at least about 100 nucleotides, of one of the previously described nucleic acid molecules. Such portions of nucleic acid molecules having the described lengths refer to, e.g., at least about 15 contiguous nucleotides of the reference nucleic acid molecule. By stringent hybridization conditions is intended overnight incubation at about 42/C for about 2.5 hours in 6×SSC/0.1% SDS, followed by washing of the filters in 1.0×SSC at 65/C, 0.1% SDS.</p>
    <p>Fragments of the sel-10-encoding nucleic acid molecules described herein, as well as polynucleotides capable of hybridizing to such nucleic acid molecules may be used as a probe or as primers in a polymerase chain reaction (PCR). Such probes may be used, e.g., to detect the presence of sel-10 nucleic acids in in vitro assays, as well as in Southern and northern blots. Cell types expressing sel-10 may also be identified by the use of such probes. Such procedures are well known, and the skilled artisan will be able to choose a probe of a length suitable to the particular application. For PCR, 5′ and 3′ primers corresponding to the termini of a desired sel-10 nucleic acid molecule are employed to isolate and amplify that sequence using conventional techniques.</p>
    <p>Other useful fragments of the sel-10 nucleic acid molecules are antisense or sense oligonucleotides comprising a single-stranded nucleic acid sequence capable of binding to a target sel-10 mRNA (using a sense strand), or sel-10 DNA (using an antisense strand) sequence.</p>
    <p>In another aspect, the invention includes sel-10 polypeptides with or without associated native pattern glycosylation. Sel-10 expressed in yeast or mammalian expression systems (discussed below) may be similar to or significantly different from a native sel-10 polypeptide in molecular weight and glycosylation pattern. Expression of sel-10 in bacterial expression systems will provide non-glycosylated sel-10.</p>
    <p>The polypeptides of the present invention are preferably provided in an isolated form, and preferably are substantially purified. Sel-10 polypeptides may be recovered and purified from recombinant cell cultures by well-known methods, including ammonium sulfate or ethanol precipitation, anion or cation exchange chromatography, phosphocellulose chromatography, hydrophobic interaction chromatography, affinity chromatography, hydroxylapatite chromatography and lectin chromatography. In a preferred embodiment, high performance liquid chromatography (HPLC) is employed for purification.</p>
    <p>The present invention also relates to vectors comprising the polynucleotide molecules of the invention, as well as host cell transformed with such vectors. Any of the polynucleotide molecules of the invention may be joined to a vector, which generally includes a selectable marker and an origin of replication, for propagation in a host. Because the invention also provides sel-10 polypeptides expressed from the polynucleotide molecules described above, vectors for the expression of sel-10 are preferred. The vectors include DNA encoding any of the sel-10 polypeptides described above or below, operably linked to suitable transcriptional or translational regulatory sequences, such as those derived from a mammalian, microbial, viral, or insect gene. Examples of regulatory sequences include transcriptional promoters, operators, or enhancers, mRNA ribosomal binding sites, and appropriate sequences which control transcription and translation. Nucleotide sequences are operably linked when the regulatory sequence functionally relates to the DNA encoding sel-10. Thus, a promoter nucleotide sequence is operably linked to a sel-10 DNA sequence if the promoter nucleotide sequence directs the transcription of the sel-10 sequence.</p>
    <p>Selection of suitable vectors to be used for the cloning of polynucleotide molecules encoding sel-10, or for the expression of sel-10 polypeptides, will of course depend upon the host cell in which the vector will be transformed, and, where applicable, the host cell from which the sel-10 polypeptide is to be expressed. Suitable host cells for expression of sel-10 polypeptides include prokaryotes, yeast, and higher eukaryotic cells, each of which is discussed below.</p>
    <p>The sel-10 polypeptides to be expressed in such host cells may also be fusion proteins which include regions from heterologous proteins. Such regions may be included to allow, e.g., secretion, improved stability, or facilitated purification of the polypeptide. For example, a sequence encoding an appropriate signal peptide can be incorporated into expression vectors. A DNA sequence for a signal peptide (secretory leader) may be fused in-frame to the sel-10 sequence so that sel-10 is translated as a fusion protein comprising the signal peptide. A signal peptide that is functional in the intended host cell promotes extracellular secretion of the sel-10 polypeptide. Preferably, the signal sequence will be cleaved from the sel-10 polypeptide upon secretion of sel-10 from the cell. Non-limiting examples of signal sequences that can be used in practicing the invention include the yeast I-factor and the honeybee melatin leader in sf9 insect cells.</p>
    <p>In a preferred embodiment, the sel-10 polypeptide will be a fusion protein which includes a heterologous region used to facilitate purification of the polypeptide. Many of the available peptides used for such a function allow selective binding of the fusion protein to a binding partner. For example, the sel-10 polypeptide may be modified to comprise a peptide to form a fusion protein which specifically binds to a binding partner, or peptide tag. Non-limiting examples of such peptide tags include the 6-His tag, thioredoxin tag, FLAG tag, hemaglutinin tag, GST tag, and OmpA signal sequence tag. As will be understood by one of skill in the art, the binding partner which recognizes and binds to the peptide may be any molecule or compound including metal ions (e.g., metal affinity columns), antibodies, or fragments thereof, and any protein or peptide which binds the peptide. These tags may be recognized by fluorescein or rhodamine labeled antibodies that react specifically with each type of tag</p>
    <p>Suitable host cells for expression of sel-10 polypeptides include prokaryotes, yeast, and higher eukaryotic cells. Suitable prokaryotic hosts to be used for the expression of sel-10 include bacteria of the genera Escherichia, Bacillus, and Salmonella, as well as members of the genera Pseudomonas, Streptomyces, and Staphylococcus. For expression in, e.g., <i>E. coli</i>, a sel-10 polypeptide may include an N-terminal methionine residue to facilitate expression of the recombinant polypeptide in a prokaryotic host. The N-terminal Met may optionally then be cleaved from the expressed sel-10 polypeptide.</p>
    <p>Expression vectors for use in prokaryotic hosts generally comprise one or more phenotypic selectable marker genes. Such genes generally encode, e.g., a protein that confers antibiotic resistance or that supplies an auxotrophic requirement. A wide variety of such vectors are readily available from commercial sources. Examples include pSPORT vectors, pGEM vectors (Promega), pPROEX vectors (LTI, Bethesda, Md.), Bluescript vectors (Stratagene), and pQE vectors (Qiagen).</p>
    <p>Sel-10 may also be expressed in yeast host cells from genera including Saccharomyces, Pichia, and Kluveromyces. Preferred yeast hosts are <i>S. cerevisiae </i>and <i>P. pastoris</i>. Yeast vectors will often contain an origin of replication sequence from a 2T yeast plasmid, an autonomously replicating sequence (ARS), a promoter region, sequences for polyadenylation, sequences for transcription termination, and a selectable marker gene.</p>
    <p>Vectors replicable in both yeast and <i>E. coli </i>(termed shuttle vectors) may also be used. In addition to the above-mentioned features of yeast vectors, a shuttle vector will also include sequences for replication and selection in <i>E. coli</i>. Direct secretion of sel-10 polypeptides expressed in yeast hosts may be accomplished by the inclusion of nucleotide sequence encoding the yeast I-factor leader sequence at the 5′ end of the sel-10-encoding nucleotide sequence.</p>
    <p>Insect host cell culture systems may also be used for the expression of Sel-10 polypeptides. In a preferred embodiment, the sel-10 polypeptides of the invention are expressed using a baculovirus expression system. Further information regarding the use of baculovirus systems for the expression of heterologous proteins in insect cells are reviewed by Luckow and Summers, <i>Bio/Technology </i>6:47 (1988).</p>
    <p>In another preferred embodiment, the sel-10 polypeptide is expressed in mammalian host cells. Non-limiting examples of suitable mammalian cell lines include the COS-7 line of monkey kidney cells (Gluzman et al., <i>Cell </i>23:175 (1981)) and Chinese hamster ovary (CHO) cells.</p>
    <p>The choice of a suitable expression vector for expression of the sel-10 polypeptides of the invention will of course depend upon the specific mammalian host cell to be used, and is within the skill of the ordinary artisan. Examples of suitable expression vectors include pcDNA3 (Invitrogen) and pSVL (Pharmacia Biotech). Expression vectors for use in mammalian host cells may include transcriptional and translational control sequences derived from viral genomes. Commonly used promoter sequences and enhancer sequences which may be used in the present invention include, but are not limited to, those derived from human cytomegalovirus (CMV), Adenovirus 2, Polyoma virus, and Simian virus 40 (SV40). Methods for the construction of mammalian expression vectors are disclosed, for example, in Okayama and Berg (<i>Mol. Cell. Biol</i>. 3:280 (1983)); Cosman et al. (<i>Mol. Immunol</i>. 23:935 (1986)); Cosman et al. (<i>Nature </i>312:768 (1984)); EP-A-0367566; and WO 91/18982.</p>
    <p>The polypeptides of the present invention may also be used to raise polyclonal and monoclonal antibodies, which are useful in diagnostic assays for detecting sel-10 polypeptide expression. Such antibodies may be prepared by conventional techniques. See, for example, <i>Antibodies: A Laboratory Manual</i>, Harlow and Land (eds.), Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y., (1988); <i>Monoclonal Antibodies, Hybridomas: A New Dimension in Biological Analyses</i>, Kennet et al. (eds.), Plenum Press, New York (1980).</p>
    <p>The sel-10 nucleic acid molecules of the present invention are also valuable for chromosome identification, as they can hybridize with a specific location on a human chromosome. There is a current need for identifying particular sites on the chromosome, as few chromosome marking reagents based on actual sequence data (repeat polymorphisms) are presently available for marking chromosomal location. Once a sequence has been mapped to a precise chromosomal location, the physical position of the sequence on the chromosome can be correlated with genetic map data. The relationship between genes and diseases that have been mapped to the same chromosomal region can then be identified through linkage analysis, wherein the coinheritance of physically adjacent genes is determined. Whether a gene appearing to be related to a particular disease is in fact the cause of the disease can then be determined by comparing the nucleic acid sequence between affected and unaffected individuals.</p>
    <p>The sel-10 polypeptides of the invention, and the DNA encoding them, may also be used to further elucidate the biological mechanism of AD, and may ultimately lead to the identification of compounds that can be used to alter such mechanisms. The sel-10 polypeptides of the invention are 47.6% identical and 56.7% similar to <i>C. elegans </i>sel-10. As is described above, mutations to <i>C. elegans </i>sel-10 are known to suppress mutations to sel-12 that result in a loss-of-function for egg laying, and also to suppress certain hypomorphic mutations to lin-12. Mutations to <i>C. elegans </i>sel-12 can also be rescued by either of the human AD-linked genes PS-1 (42.7% identical to sel-12) or PS-2 (43.4% identical to sel-12). However, human PS-1 with a familial AD-linked mutant has a reduced ability to rescue sel-12 mutants (Levitan, D. et al., <i>Proc. Natl. Acad. Sci. USA </i>93: 14940-14944 (1996)).</p>
    <p>This demonstrated interchangeability of human and <i>C. elegans </i>genes in the notch signaling pathway makes it reasonable to predict that mutations of human sel-10 will suppress mutations to PS-1 or PS-2 that lead to AD, especially in light of the predicted structure of sel-10. As described above, PS-1 and PS-2 mutations that lead to AD are those which interfere with the proteolytic processing of PS-1 or PS-2. The sel-10 polypeptides of the invention are members of the β-transducin protein family, which includes yeast CDC4, a component of an enzyme which functions in the ubiquitin-dependent protein degradation pathway. Thus, human sel-10 may regulate presenilin degradation via the ubiquitin-proteasome pathway. Alternatively, or in addition, human sel-10 may alter presenilin function by targeting for degradation through ubiquitination a modulator of presenilin activity, e.g., a negative regulator. Therefore, mutations to sel-10 may reverse the faulty proteolytic processing of PS-1 or PS-2 which occurs as a result of mutation to PS-1 or PS-2 or otherwise increase presenilin function. For the same reason, inhibition of sel-10 activity may also act to reverse PS-1 or PS-2 mutations. Thus, it may be hypothesized that compounds which inhibit either the expression or the activity of the human sel-10 polypeptides of the invention may reverse the effects of mutations to PS-1 or PS-2, and thus be useful for the prevention or treatment of AD.</p>
    <p>Thus, <i>C. elegans </i>may be used as a genetic system for the identification of agents capable of inhibiting the activity or expression of the human sel-10 polypeptides of the invention. A suitable <i>C. elegans </i>strain for use in such assays lacks a gene encoding active <i>C. elegans </i>sel-10, and exhibits a loss-of-function for egg-laying resulting from an inactivated sel-12 gene. Construction of <i>C. elegans </i>strains having a loss-of-function for egg-laying due to mutation of sel-12 may be accomplished using routine methods, as both the sequence of sel-12 (Genebank accession number U35660) and mutations to sel-12 resulting in a loss-of-function for egg laying are known (see Levitan et al., <i>Nature </i>377: 351-354 (1995), which describes construction of <i>C. elegans </i>sel-12(ar171)). An example of how to make such a strain is also given in Levitan et al. (<i>Nature </i>377: 351-354 (1995)). Wild-type <i>C. elegans </i>sel-10 in the <i>C. elegans </i>sel-12(ar171)) , is also mutagenized using routine methods, such as the technique used for sel-12 mutagenesis in Levitan et al., supra.</p>
    <p>In order to identify compounds inhibiting human sel-10 activity, a DNA vector containing a human sel-10 gene encoding any of the wild-type human sel-10 proteins of the invention is introduced into the above-described <i>C. elegans </i>strain. In a preferred embodiment, the heterologous human sel-10 gene is integrated into the <i>C. elegans </i>genome. The gene is then expressed, using techniques described in Levitan et al. (<i>Proc. Natl. Acad. Sci. USA </i>93: 14940-14944 (1996)). Test compounds are then administered to this strain in order to determine whether a given agent is capable of inhibiting sel-10 activity so as to suppress mutations to sel-12 or lin-12 that result in egg-laying defects. Egg-laying in this strain is then determined, e.g. by the assay described in Levitan et al. (<i>Proc. Natl. Acad. Sci. USA </i>93: 14940-14944 (1996)). To confirm that the compound's effect on egg-laying is due to inhibition of sel-10 activity, the action of the compound can be tested in a second biochemical or genetic pathway that is known to be affected by loss-of-function mutations in sel-10 (e.g., further elevation of lin-12 activity in lin-12(d) hypomorphic strains). Such assays may be performed as described in Sundarem and Greenwald (<i>Genetics </i>135: 765-783 (1993)).</p>
    <p>Alternatively, compounds are tested for their ability to inhibit the E3 Ubiquitin Ligating Enzyme. Assays procedures for demonstrating such activity are well known, and involve reconstitution of the ubiquitinating system using purified human sel-10 protein together with the yeast proteins Cdc4p, Cdc53p and Skp1p and an E1 enzyme, the E2 enzyme Cdc34p, ubiquitin, a target protein and an ATP regenerating system (Feldman et al., 1997). The sel-10 ubiquitination system may also be reconstituted with the <i>C. elegans </i>counterparts of the yeast components, e.g., cul-1 (also known as lin-19) protein substituting for Cdc53p (Kipreos et al., <i>Cell </i>85:829 (1996)) and the protein F46A9 substituting for Skp1p, or with their mammalian counterparts, e.g., Cul-2 protein substituting for Cdc53p (Kipreos et al., ibid.) and mammalian Skp1p substituting for yeast Skp1p. A phosphorylation system provided by a protein kinase is also to be included in the assay system as per Feldman et al., 1997.</p>
    <p>Alternatively, cell lines which express human sel-10 due to transformation with a human sel-10 cDNA and which as a consequence have elevated APP processing and formation of Aβ<sub>1-40 </sub>or Aβ<sub>1-42 </sub>may also be used for such assays as in Example 3. Compounds may be tested for their ability to reduce the elevated Aβ processing seen in the sel-10 transformed cell line.</p>
    <p>Compounds that rescue the egg-laying defect or that inhibit E3 Ubiquitin Ligating Enzyme are then screened for their ability to cause a reduction in the production of A-beta<sub>1-40 </sub>or A-beta<sub>1-42 </sub>in a human cell line. Test compounds are used to expose IMR-32 or other human cell lines known to produce A-beta<sub>1-40 </sub>or A-beta<sub>1-42 </sub>(Asami-Okada et al., <i>Biochemistry </i>34: 10272-10278 (1995)), or in human cell lines engineered to express human APP at high levels. In these assays, A-beta<sub>1-40</sub>or A-beta<sub>1-42 </sub>is measured in cell extracts or after release into the medium by ELISA or other assays which are known in the art (Borchelt et al., i Neuron 17: 1005-1013 (1996); Citron et al., <i>Nat. Med</i>. 3: 67-72 (1997)).</p>
    <p>Having generally described the invention, the same will be more readily understood by reference to the following examples, which are provided by way of illustration and are not intended as limiting.</p>
    <heading>EXAMPLES</heading> <heading>Example 1</heading> <p>Identification of a Human Homologue to <i>C. elegans </i>sel-10 Results</p>
    <p>Identification of sel-10 in ACEDB</p>
    <p>Sel-10 maps between the cloned polymorphisms arP3 and TCPARI just to the left of him-5 [ACEDB entry wm95p536]. Three phage lambda clones have been sequenced across the interval, F53C11, F09F3, and F55B12. Sel-10 is reported to have homology to yeast cdc4 [ACEDB entry wm97ab259]. Blast search revealed a single ORF with homology to yeast cdc4 (CC4_YST) within the interval defined by arP3 and TCPARI corresponding to the GenPep entry F55B12.3. F55B12.3, like yeast cdc4, is a member of the β-transducin protein family. This family is characterized by the presence of multiple WD40 repeats [Neer, E. J. et al., <i>Nature </i>371: 297-300 (1994)].</p>
    <p>Identification of a Human sel-10 homologue, Incyte 028971</p>
    <p>The GenPep entry F55B12.3 was used to search the LifeSeq, LifeSeq FL and EMBL data bases using tblastn. The search revealed multiple homologies to β-transducin family members including LIS-1 (S36113 and P43035), a gene implicated in Miller-Dieker lissencephaly, a <i>Xenopus laevis </i>gene, TRCPXEN (U63921), and a human contig in LifeSeq FL, 028971. Since there also are multiple β-transducin family members within the <i>C. elegans </i>genome, these were collected using multiple blast searches and then clustered with the sel-10 candidate genes. Multiple alignments were performed with the DNAStar program Megalign using the Clustal method. This revealed that LIS-1 clustered with T03F6.F, a different β-transducin family member and thus excluded it as a candidate sel-10 homologue. TRCPXEN clustered with K10 B2.1, a gene which also clusters with F55B12.3 and CC4YST, while Incyte 028971 clustered with sel-10. Thus, Incyte 028971 appears to encode the human homologue of <i>C. elegans </i>sel-10. Sequence homology between sel-10 and 028971 is strongest in the region of the protein containing 7 repeats of the WD40 motif. The Incyte 028971 contig contains 44 ESTs from multiple libraries including pancreas, lung, T-lymphocytes, fibroblasts, breast, hippocampus, cardiac muscle, colon, and others.</p>
    <p>Public EST</p>
    <p>Blastx searches with the DNA sequence 028971 against the TREMBLP dataset identified a single homologous mouse EST (W85144) from the IMAGE Library, Soares mouse embryo NbME13.5-14.5. The blastx alignment of 028971 with W85144 and then with F55B12.3 revealed a change in reading frame in 028971 which probably is due to a sequencing error.</p>
    <p>Blastn searches of the EMBL EST database with the 028971 DNA sequence revealed in addition to W85144, three human EST that align with the coding sequence of 028971 and six EST that align with the 3′ untranslated region of the 028971 sequence.</p>
    <p>Protein Motifs</p>
    <p>Two protein motifs were identified in F55B12.3 which are shared with yeast cdc4, mouse w85144 and human 028971. These are an F-box in the N-terminal domain and seven β-transducin repeats in the C-terminal domain.</p>
    <p>Discussion</p>
    <p>The sel-10 gene encodes a member of the β-transducin protein family. These are characterized by the presence of multiple WD40 repeats [Neer, E. J. et al., <i>Nature </i>371: 297-300 (1994)]. The repeats are 20-40 amino acids long and are bounded by gly-his (GH) and trp-asp (WD) residues. Solution of the three dimensional structure of β-transducin indicates that the WD40 repeats form the arms of a seven-bladed propeller like structure [Sondek, J. et al., <i>Nature </i>379: 369-74 (1996)]. Each blade is formed by four alternating pleats of beta-sheet with a pair of the conserved aspartic acid residues in the protein motif forming the limits of one internal beta strand. WD40 repeats are found in over 27 different proteins which represent diverse functional classes [Neer, E. J. et al., <i>Nature </i>371: 297-300 (1994)]. These regulate cellular functions including cell division, cell fate determination, gene transcription, signal transduction, protein degradation, mRNA modification and vesicle fusion. This diversity in function has led to the hypothesis that β-transducin family members provide a common scaffolding upon which multiprotein complexes can be assembled.</p>
    <p>The homology of sel-10, 28971 and W85144 to the yeast cdc4 gene suggests a functional role in the ubiquitin-proteasome pathway for intracellular degradation of protein. Mutations of the yeast cdc4 gene cause cell cycle arrest by blocking degradation of Sic1, an inhibitor of S-phase cyclin/cdk complexes [King, R. W. et al., <i>Science </i>274: 1652-9 (1996)]. Phosphorylation of Sic1 targets it for destruction through the ubiquitin-proteasome pathway. This pathway consists of three linked enzyme reactions that are catalyzed by multiprotein complexes [Ciechanover, A., <i>Cell </i>79: 13-21 (1994); Ciechanover, A. and A. L. Schwartz, <i>FASEB J</i>. 8: 182-91 (1994)]. Initially, the C-terminal glycine of ubiquitin is activated by ATP to form a high energy thiol ester intermediate in a reaction catalyzed by the ubiquitin-activating enzyme, E1. Following activation, an E2 enzyme (ubiquitin conjugating enzyme) transfers ubiquitin from E1 to the protein target. In some cases, E2 acts alone. In others, it acts in concert with an E3 ubiquitin-ligating enzyme which binds the protein substrate and recruits an E2 to catalyze ubiquitination. E2 ubiquitin-conjugating enzymes comprise a fairly conserved gene family, while E3 enzymes are divergent in sequence [Ciechanover, A., <i>Cell </i>79: 13-21 (1994); Ciechanover, A. and A. L. Schwartz, <i>FASEB J</i>. 8: 182-91 (1994)].</p>
    <p>In yeast, mutation of the E2 ubiquitin-conjugating enzyme, cdc34, causes cell cycle arrest through failure to degrade the Sic1 inhibitor of the S-phase cyclin/cdk complex [King, R. W. et al., Science 274: 1652-9 (1996)]. Sic1 normally is degraded as cells enter the G1-S phase transition, but in the absence of cdc34, Sic1 escapes degradation and its accumulation causes cell cycle arrest. Besides cdc34, cdc4 is one of three other proteins required for the G1-S phase transition. The other two are cdc53 and Skp1. As discussed above, cdc4 contains two structural motifs, seven WD40 repeats (which suggests that the protein forms a beta-propeller) and a structural motif shared with cyclin F which is an interaction domain for Skp1 [Bai, C. et al., <i>Cell </i>86: 263-74 (1996)]. Insect cell lysates containing cdc53, cdc4 and skp1 (and also ubiquitin, cdc34 and El) can transfer ubiquitin to Sic1 suggesting that one or more of these components functions as an E3 ubiquitin-ligating enzyme [King, R. W. et al., <i>Science </i>274: 1652-9 (1996)]. Increased expression of either cdc4 or Skp1 partially rescues loss of the other.</p>
    <p>In <i>C. elegans</i>, mutation of sel-10 has no visible phenotype indicating that sel-10 does not play a role in regulation of the cell-cycle. A closely related, <i>C. elegans </i>β-transducin family member, K10B2.6 may play that role as it clusters with the gene TRCP_XEN from Xenopus laevis which rescues yeast cell cycle mutants arrested in late anaphase due to a failure to degrade cyclin B [Spevak, W. et al., <i>Mol. Cell. Biol</i>. 13: 4953-66 (1993)]. If sel-10 does encode a component of an E3-ubiquitin ligating enzyme, how might it suppress sel-12 and enhance lin-12 mutations? The simplest hypothesis is that sel-10 regulates degradation of both proteins via the ubiquitin-proteasome pathway. Both sel-12 and lin-12 are transmembrane proteins. Sel-12 crosses the membrane 8 times such that its N- and C-termini face the cytosol [Kim, T. W. et al., <i>J. Biol. Chem</i>. 272: 11006-10 (1997)], while lin-12 is a type 1 transmembrane protein (Greenwald, I. and G. Seydoux, <i>Nature </i>346: 197-9 (1990)). Both are ubiquitinated, and in the case of human PS2, steady state levels increase in cells treated with an inhibitor of the proteasome, N-acetyl-L-leucinal-L-norleucinal and lactacystin (Li, X. and I. Greenwald, <i>Neuron</i>. 17: 1015-21 (1996)). Alternatively, sel-10 may target for degradation of a negative regulator of presenilin function.</p>
    <p>The genetic analysis and protein function suggested by homology to cdc4 implies that drug inhibitors of human sel-10 may increase steady state levels of human presenilins. This could potentiate activity of the presenilin pathway and provide a means for therapeutic intervention in Alzheimer's disease.</p>
    <heading>Example 2</heading> <p>5′RACE Cloning of a Human cDNA Encoding Sel-10, an Extragenic Suppressor of Presenilin Mutations in <i>C. elegans </i> </p>
    <p>Materials and Methods</p>
    <p>Oligonucleotide primers for the amplification of the sel-10 coding sequence from <i>C. elegans </i>cDNA were prepared based on the sequence of F55B12.3, identified in Example 1 as the coding sequence for <i>C. elegans </i>sel-10. The primers prepared were: 5′-CGGGATCCACCATGGATGATGGATCGATGACACC-3′ (SEQ ID NO:11) and 5′-GGAATTCCTTAAGGGTATACAGCATCAAAGTCG-3′ (SEQ ID NO:12). <i>C. elegans </i>mRNA was converted to cDNA using a BRL Superscript II Preamplification kit. The PCR product was digested with restriction enzymes BamHI and EcoRI (LTI, Gaithersberg, Md.) and cloned into pcDNA3.1 (Invitrogen). Two isolates were sequenced (ABI, Perkin-Elmer Corp).</p>
    <p>The sequence of Incyte clone 028971 (encoding a portion of the human homologue of <i>C. elegans </i>sel-10), was used to design four antisense oligonucleotide primers:5′-TCACTTCATGTCCACATCAAAGTCC-3′ (SEQ ID NO:13), 5′-GGTAA-TTACAAGTTCTTGTTGAACTG (SEQ ID NO:14), 5′-CCCTGCAACGTGTGT-AGACAGG-3′ (SEQ ID NO:15), and 5′-CCAGTCTCTGCATTCCACACTTTG-3′ (SEQ ID NO:16) to amplify the missing 5′ end of human sel-10. The Incyte LifeSeq “Electronic Northern” analysis was used to identify tissues in which sel-10 was expressed. Two of these, hippocampus and mammary gland, were chosen for 5′ RACE cloning using a CloneTech Marathon kit and prepared Marathon-ready cDNA from hippocampus and mammary gland. PCR products were cloned into the TA vector pCR3.1 (Invitrogen), and isolates were sequenced. An alternate 5′ oligonucleotide primer was also designed based on Incyte clones which have 5′ ends that differ from the hippocampal sel-10 sequence: 5′-CTCAGACAGGTCAGGACATTTGG-3′ (SEQ ID NO:17).</p>
    <p>Blastn was used to search Incyte databases LifeSeq and LifeSeqFL. Gap alignments and translations were performed with GCG programs (Wisconsin Sequence Analysis Package, Version 8 for Unix, Genetics Computer Group, University Research Park, Madison Wis.).</p>
    <p>Results</p>
    <p>The Coding Sequence of the <i>C. elegans </i>sel-10</p>
    <p>The predicted coding sequence of the <i>C. elegans </i>sel-10, F55B12.3, had originally been determined at the Genome Sequencing Center, Washington University, St. Louis, by using the computer program GeneFinder to predict introns and exons in the genomic cosmid F55B12. The hypothetical cDNA sequence was confirmed by amplifying this region from <i>C. elegans </i>cDNA, cloning, and sequencing it .</p>
    <p>The Coding Sequence of the Human sel-10 Gene Homologue</p>
    <p>All of the 028971 antisense oligonucleotides amplified a 5′ RACE product from human hippocampal and mammary cDNA. The longest PCR product from the hippocampal reactions was cloned and sequenced. This PCR reaction was designed to generate products which end at the predicted stop codon. Two isolates contained identical sequence which begins 880 bases before the beginning of the 028971 sequence. This sequence was confirmed by comparison with spanning Incyte cDNA clones. The Incyte clones that spanned the 5′ end of the human sel-10 homologue were not annotated as F55B12.3, as the homology in this region between the human and <i>C. elegans </i>genes is low, and as the overlap between these clones and the annotated clones happened to be too small for them to be clustered in the Incyte database or uncovered by our blasting the Incyte database with the 028971 sequence.</p>
    <p>The predicted protein sequences of human sel-10 have 47.6% identity and 56.7% similarity to <i>C. elegans </i>sel-10. The N-terminus of the human sel-10 sequence begins with 4 in-frame methionines. In addition to the WD40 repeats described above, the human sequence also contains a region homologous to the CDC4 F-box for binding Skp1, as expected for a sel-10 homologue.</p>
    <p>Different Human sel-10 mRNAs Expressed in Mammary and Hippocampal Tissues</p>
    <p>Several additional human sel-10 ESTs which differ from the hippocampal sequence were identified. These are an exact match, which indicates that the alternative transcript is probably real. Comparison of these sequences with the human hippocampal sel-10 sequence shows divergence prior to the 4th in-frame methionine and then exact sequence match thereafter. An oligonucleotide primer specific for the 5′ end of this alternative transcript was found to amplify a product from mammary but not hippocampal cDNA. This indicates either that the human sel-10 transcript undergoes differential splicing in a tissue-specific fashion or that the gene contains multiple, tissue specific promoters.</p>
    <p>Discussion</p>
    <p>5′RACE and PCR amplification were used to clone a full-length cDNA encoding the human homologue of the <i>C. elegans </i>gene, sel-10. Sequence analysis confirms the earlier prediction that sel-10 is a member of the CDC4 family of proteins containing F-Box and WD40 Repeat domains. Two variants of the human sel-10 cDNA were cloned from hippocampus and mammary gland which differed in 5′ sequence preceding the apparent site of translation initiation. This implies that the gene may have two or more start sites for transcription initiation which are tissue-specific or that the pattern of exon splicing is tissue-specific.</p>
    <heading>Example 3</heading> <p>Expression of Epitope-Tagged Sel-10 in Human Cells, and Perturbation of Amyloid β Peptide Processing by Human Sel-10</p>
    <p>Materials and Methods</p>
    <p>Construction of Epitope-Tagged Sel-10: Subcloning, Cell Growth and Transfection</p>
    <p>An EcoR1 site was introduced in-frame into the human sel-10 cDNA using a polymerase chain reaction (PCR) primed with the oligonucleotides 237 (5′-GGAATTC-CATGAAAAGATTGGACCATGGTTCTG-3′) (SEQ ID NO:18) and 206 (5′-GGA-ATTCCTCACTTCATGT-CACATCAAAGTCCAG-3′) (SEQ ID NO:19). The resulting PCR product was cloned into the EcoR1 site of the vector pCS2+MT. This fused a 5′ 6-myc epitope tag in-frame to the fifth methionine of the hippocampal sel-10 cDNA, i.e., upstream of nucleotide 306 of the sequence given in SEQ ID NO: 1. The nucleotide sequence of this construct, designated 6myc-N-sel-10, is given in SEQ ID NO: 20, while the amino acid sequence of the polypeptide encoded thereby is given in SEQ ID NO: 21. The hippocampal and mammary sel-10 cDNA diverge upstream of this methionine. A PS1 cDNA with a 3′-FLAG tag (PS1-C-FLAG) was subcloned into the pcDNA3.1 vector. An APP cDNA containing the Swedish NL mutation and an attenuated ER retention sequence consisting of the addition of a di-lysyl motif to the C-terminus of APP695 (APP695NL-KK) was cloned into vector pIRES-EGFP (Mullan et al., Nat Genet August 1992; 1(5):345-7). HEK293 and IMR32 cells were grown to 80% confluence in DMEM with 10% FBS and transfected with the above cDNA. A total of 10 mg total DNA/6×10<sup>6 </sup>cells was used for transfection with a single plasmid. For cotransfections of multiple plasmids, an equal amount of each plasmid was used for a total of 10 mg DNA using LipofectAmine (BRL).</p>
    <p>In order to construct C-term V5 his tagged sel-10 and the C-term mychis tagged sel-10, the coding sequence of human hippocampal sel-10 was amplified using oligonucleotides primers containing a KpnI restriction site on the 5′ primer: 5′-GGGTA-CCCCTCATTATTCCCTCGAGTTCTTC-3′ (SEQ ID NO:22) and an EcoRI site on the 3′ primer: 5′-GGAATTCCTTCATGTCCACATCAAAGTCC-3′ (SEQ ID NO:23), using the original human sel-10 RACE per product as template. The product was digested with both KpnI and EcoRI and cloned into either the vector pcDNA6/V5-His A or pcDNA3.1/Myc-His(+) A (Invitrogen). The nucleotide sequence of independent isolates was confirmed by dideoxy sequencing. The nucleotide sequence of the C-term V5 his tagged sel-10 is given in SEQ ID NO: 24, while the amino acid sequence of the polypeptide encoded thereby is given in SEQ ID NO: 25. The nucleotide sequence of independent isolates was confirmed by dideoxy sequencing. The nucleotide sequence of the C-term mychis tagged sel-10 is given in SEQ ID NO: 26, while the amino acid sequence of the polypeptide encoded thereby is given in SEQ ID NO: 27.</p>
    <p>Clonal Selection of Transformed Cells by FACS</p>
    <p>Cell samples were analyzed on an EPICS Elite ESP flow cytometer (Coulter, Hialeah, Fla.) equipped with a 488 nm excitation line supplied by an air-cooled argon laser. EGFP emission was measured through a 525 nm band-pass filter and fluorescence intensity was displayed on a 4-decade log scale after gating on viable cells as determined by forward and right angle light scatter. Single green cells were separated into each well of one 96 well plate containing growth medium without G418. After a four day recovery period, G418 was added to the medium to a final concentration of 400 mg/ml. Wells with clones were expanded from the 96 well plate to a 24 well plate and then to a 6 well plate with the fastest growing colonies chosen for expansion at each passage.</p>
    <p>Immunofluorescence</p>
    <p>Cells grown on slides were fixed 48 hrs after transfection with 4% formaldehyde and 0.1% Triton X-100 in PBS for 30 min on ice and blocked with 10% Goat serum in PBS (blocking solution) 1 hr RT (i.e., 25° C.), followed by incubation with mouse anti-myc (10 mg/ml) or rabbit anti-FLAG (0.5 mg/ml) antibody 4° C. O/N and then fluorescein-labeled goat anti-mouse or anti-rabbit antibody (5mg/ml) in blocking solution 1 hr at 25° C.</p>
    <p>Western Blotting</p>
    <p>Cell lysates were made 48 hrs after transfection by incubating 10<sup>5 </sup>cells with 100 ml TENT (50 mM Tris-HCl pH 8.0,2 mM EDTA, 150 mM NaCl, 1% Triton X-100, 1× protease inhibitor cocktail) 10 min on ice followed by centrifugation at 14,000 g. The supernatant was loaded on 4-12% NuPage gels (50 mg protein/lane) and electrophoresis and transfer were conducted using an Xcell II Mini-Cell system (Novex). The blot was blocked with 5% milk in PBS 1 hr RT and incubated with anti-myc or anti-FLAG antibody (described in “Immunofluorescence” above) 4° C. O/N, then sheep anti-mouse or anti-rabbit antibody-HRP (0.1 mg/ml) 1 hr RT, followed by Supersignal (Pierce) detection.</p>
    <p>ELISA</p>
    <p>Cell culture supernatant or cell lysates (100 ml formic acid/10<sup>6 </sup>cells) were assayed in the following double antibody sandwich ELISA, which is capable of detecting levels of Aβ<sub>1-40 </sub>and Aβ<sub>1-42 </sub>peptide in culture supernatant.</p>
    <p>Human Aβ 1-40 or 1-42 was measured using monoclonal antibody (mAb) 6E10 (Senetek, St. Louis, Mo.) and biotinylated rabbit antiserum 162 or 164 (NYS Institute for Basic Research, Staten Island, N.Y.) in a double antibody sandwich ELISA. The capture antibody 6E10 is specific to an epitope present on the N-terminal amino acid residues 1-16 of hAβ. The conjugated detecting antibodies 162 and 164 are specific for hAβ 1-40 and 1-42, respectively. The sandwich ELISA was performed according to the method of Pirttila et al. (<i>Neurobiology of Aging </i>18: 121-7 (1997)). Briefly, a Nunc Maxisorp 96 well immunoplate was coated with 100 μl/well of mAb 6E10 (5 μg/ml) diluted in 0.1M carbonate-bicarbonate buffer, pH 9.6 and incubated at 4° C. overnight. After washing the plate 3× with 0.01M DPBS (Modified Dulbecco's Phosphate Buffered Saline (0.008M sodium phosphate, 0.002M potassium phosphate, 0.14M sodium chloride, 0.01 M potassium chloride, pH 7.4) from Pierce, Rockford, Ill.) containing 0.05% of Tween-20 (DPBST), the plate was blocked for 60 min with 200 μl of 10% normal sheep serum (Sigma) in 0.01M DPBS to avoid non-specific binding. Human Aβ 1-40 or 1-42 standards 100 μl/well (Bachem, Torrance, Calif.) diluted, from a 1 mg/ml stock solution in DMSO, in non transfected conditioned cell medium was added after washing the plate, as well as 100 μl /well of sample i.e. filtered conditioned medium of transfected cells. The plate was incubated for 2 hours at room temperature and 4° C. overnight. The next day, after washing the plate, 100 μl/well biotinylated rabbit antiserum 162 1:400 or 164 1:50 diluted in DPBST+0.5% BSA was added and incubated at room temperature for 1 hr 15 min. Following washes, 100 μl/well neutravidin-horseradish peroxidase (Pierce, Rockford, Ill.) diluted 1:10,000 in DPBST was applied and incubated for 1 hr at room temperature. After the last washes 100 μl well of o-phenylnediamine dihydrochloride (Sigma Chemicals, St. Louis, Mo.) in 50 mM citric acid/100 mM sodium phosphate buffer (Sigma Chemicals, St. Louis, Mo.), pH 5.0, was added as substrate and the color development was monitored at 450 nm in a kinetic microplate reader for 20 min. using Soft max Pro software.</p>
    <p>Results</p>
    <p>Transfection of HEK293 Cells</p>
    <p>Transfection efficiency was monitored through the use of vectors that express green fluorescent protein (GFP) or by immunofluorescent detection of epitope-tagged sel-10 or PS1. An N-terminal 6-myc epitope was used to tag human sel-10 (6 myc-N-sel-10), while PS1 was tagged with a C-terminal FLAG epitope (PS1-C-FLAG). APP695 was modified by inclusion of the Swedish NL mutation to increase Aβ processing and an attenuated endoplasmic reticulum (ER) retention signal consisting of a C-terminal di-lysine motif (APP695NL-KK). The di-lysine motif increases Aβ processing about two fold. The APP695NL-KK construct was inserted into the first cistron of a bicistronic vector containing GFP (pIRES-EGFP, Invitrogen) to allow us to monitor transfection efficiency. Transfection efficiency in HEK293 cells was about 50% for transfections with a single plasmid DNA. For cotransfections with two plasmids, about 30-40% of the cells expressed both proteins as detected by double immunofluorescence.</p>
    <p>Expression of recombinant protein in transfected HEK293 cells was confirmed by Western blot as illustrated for PS1-C-FLAG and 6myc-N-sel-10 (FIG. <b>1</b>A). In the case of cotransfections with three plasmids (PS 1-C-FLAG+6myc-N-sel-10+APP), all three proteins were detected in the same cell lysate by Western blot (FIG. 1B) using appropriate antibodies.</p>
    <p>Effect of 6myc-N-sel-10 and PS1-C-FLAG on Aβ Processing</p>
    <p>Cotransfection of APP695NL-KK with 6myc-N-sel-10 or PS1-C-FLAG into HEK293 cells increased the release of Ab1-40 and Ab1-42 peptide into the culture supernatant by 2- to 3-fold over transfections with just APP695NL-KK (Table 1). Cotransfection of APP695NL-KK with both 6myc-N-sel-10 and PS1-C-FLAG increased Ab release still further (i.e., 4- to 6-fold increase). In contrast, the ratio of Ab1-42/(Ab1-40+Ab1-42) released into the supernatant decreased about 50%. The subtle decrease in the ratio of Ab1-42/(Ab1-40+Ab1-42) reflects the larger increase in Ab 1-40 relative to Ab 1-42. Neither 6myc-N-sel-10 nor PS1-C-FLAG affected endogenous Ab production in HEK293 cells. Similar observations were also obtained in IMR32 cells (Table 2). However, IMR32 cells transfected less well than HEK293 cells, so the stimulation of APP695NL-KK processing by cotransfection with 6myc-N-sel-10 or PS1-C-FLAG was lower.</p>
    <p>Levels of Ab 1-40 expressed in HEK293 cells transfected with APP695NL-KK were sufficient to measure Ab peptide in both the culture supernatant and cell pellet. Considerably more Ab 1-40 is detected in the HEK293 cell pellet than in the supernatant in cells transfected with just APP695NL-KK. Cotransfection with 6myc-N-sel-10 or PS1-C-FLAG proportionately decreased Ab 1-40 in the cell pellet and increased Ab in the culture supernatant. This implies that 6myc-N-sel-10 and PS1-C-FLAG alter processing or trafficking of APP such that proportionately more Ab is released from the cell.</p>
    <p>Effect of 6myc-N-sel-10 and PS1-C-FLAG Expression on endogenous Aβ Processing</p>
    <p>The effect of 6myc-N-sel-10 on the processing of endogenous APP by human cells was assessed by creating stably transformed HEK293 cell lines expressing these proteins. Two cell lines expressing 6myc-N-sel-10 were derived (sel-10/2 &amp; sel-10/6) as well as a control cell line transformed with pcDNA3.1 vector DNA. Both 6myc-N-sel-10 cell lines expressed the protein as shown by Western blot analysis. Endogenous production of Ab 1-40 was increased in both 6myc-N-sel-10 cell lines in contrast to vector DNA transformed cells Table 3). In addition, stable expression of 6myc-N-sel-10 significantly increased Ab production after transfection with APP695NL-KK plasmid DNA (Table 3). Similar results were obtained with 6 stable cell lines expressing PS1-C-FLAG. All 6 cell lines showed significant elevation of endogenous Aβ processing and all also showed enhanced processing of Ab after transfection with APP695NL-KK (Table 3). In addition, the increase of Aβ processing seen with 6myc-N-sel-10 was also seen with sel-10 tagged at the C-terminus with either mychis or v5his (See Table 4). Both C-terminal and N-terminal tags resulted in an increase in Aβ processing.</p>
    <p>Discussion</p>
    <p>These data suggest that , when over expressed, 6myc-N-sel-10 as well as PS1-C-FLAG alter Aβ processing in both transient and stable expression systems. A 6-myc epitope tag was used in these experiments to allow detection of sel-10 protein expression by Western blot analysis. If as its sequence homology to yeast CDC4 suggests, sel-10 is an E2-E3 ubiquitin ligase, it should be possible to identify the proteins it targets for ubiquitination. Since the presenilins are degraded via the ubiquitin-proteasome pathway, PS1 &amp; PS2 are logical targets of sel-10 catalyzed ubiquitination [Kim et al., <i>J. Biol. Chem</i>. 272:11006-11010 (1997)]. How sel-10 affects Aβ processing is not understood at this point. In the future, it will be necessary to determine if sel-10 &amp; PS1 increase Aβ processing by altering production, processing, transport, or turn-over of APP, and whether the effect of PS1 is mediated or regulated by sel-10.</p>
    <p>These experiments suggest that sel-10 is a potential drug target for decreasing Ab levels in the treatment of AD. They also show that <i>C. elegans </i>is an excellent model system in which to investigate presenilin biology in the context of AD. Thus, as is shown by cotransfection experiments, as well as in stable transformants, expression of 6myc-N-sel10 or PS1-C-FLAG increases Aβ processing. An increase in Aβ processing was seen in both HEK293 cells and IMR32 cells after cotransfection of 6myc-N-sel10 or PS1-C-FLAG with APP695NL-KK. In stable transformants of HEK293 cells expressing 6myc-Sel10 or PS1-C-FLAG, an increase in endogenous Aβ processing was observed, as well as an increase in Aβ processing after transfection with APP695NL-KK. This suggests that inhibitors of either sel-10 and/or PS1, may decrease Aβ processing, and could have therapeutic potential for Alzheimer's disease.</p>
    <heading>Example 4</heading> <p>SEL-10 Interacts with Presenilin 1, Facilitates its Ubiquitination, and Alters Aβ Production</p>
    <p>Mutations in the presenilin genes (PS1 or PS2) in man cause autosomal dominant early onset Alzheimer's disease. These have been linked to alterations in the processing of the amyloid protein precursor (APP)<sup>1,2</sup>. The presenilins are membrane proteins with 6-8 transmembrane domains<sup>3,4</sup>, which localize to the endoplasmic reticulum, Golgi complex, nuclear envelope, kinetochore and centrosome<sup>5,6</sup>. Mutations in sel-12, a <i>Caenorhabditis elegans </i>presenilin cause a defect in egg laying<sup>7</sup>. Sel-12 is one of three nematode presenilin homologous (sel-12, hop-1 and spe-4)<sup>7-9</sup>. The sel-12 mutant phenotype can be rescued by human PS1 or PS2<sup>10</sup>, indicating that PS1, PS2 and sel-12 are functional homologues. Mutations in sel-12 cause a defect in egg laying by altering signaling through the Notch/lin-12 pathway. The sel-12 mutant phenotype can be suppressed by loss of function mutations in a second gene, sel-10<sup>11</sup>, which probably results in rescue of the egg laying defect by increasing the activity of a functionally redundant presenilin, hop-1<sup>8,9</sup>. SEL-10 is a homologue of yeast Cdc4, a member of the SCF (Skp1-Cdc53/CUL1-F-box protein) E2-E3 ubiquitin ligase family<sup>12</sup>. In this study, we show that human SEL-10 interacts with PS1 and enhances PS1 ubiquitination, thus altering cellular levels of unprocessed PS1 and its N- and C-terminal fragments. This leads to an alteration in the metabolism of APP and to an increase in the production of amyloid β-peptide, the principal component of amyloid plaque in Alzheimer's disease<sup>13</sup>.</p>
    <p>The SCF E2-E3 ubiquitin ligases contain a catalytic core consisting of Skp1, Rbx1, Cdc53/CUL-1 and an E2 ubiquitin transferase, Cdc34<sup>14</sup>. These are targeted to substrates for ubiquitination by adapter proteins (e.g., Cdc4, Grr1, Met30, β-TrCP) containing an F-box motif and WD40 repeats<sup>15</sup>. There is evidence that presenilins are ubiquitinated and undergo degradation through the ubiquitin-proteasome pathway<sup>16,17</sup>. Physical interaction between <i>C. elegans </i>SEL-10 and SEL-12 has been shown previously<sup>11</sup>. Thus, SEL-10 may be the F-box adaptor protein that recruits presenilins for ubiquitination and subsequent degradation.</p>
    <p>Both human and mouse orthologs of <i>C. elegans </i>sel-10 have been identified in EST databases, although the sequence information is incomplete<sup>12</sup>. We used rapid amplification of cDNA ends (RACE) to clone amplification products containing the full coding sequences for human and mouse sel-10. There are two variants of human sel-10 cDNA from hippocampus (hippocampus form) and mammary gland (common form) which differ in 5′ sequence upstream of a common translation initiation site. This may indicate that the human sel-10 transcript undergoes differential splicing in a tissue-specific fashion or that the gene contains multiple, tissue specific promoters. The hippocampal form contains four in-frame methionines upstream of the common initiation site and the mammary form contains three. Whether or not these encode proteins with different N-termini is not known. The common transcript is ubiquitously expressed in all tissues tested (FIG. <b>2</b>A), while the hippocampal form is present only in brain (FIG. <b>2</b>B). The human sel-10 gene was localized to chromosome 4q31.2-31.3 by in situ hybridization (data not shown). The predicted protein sequences of human and <i>C. elegans </i>SEL-10 have 47.6% amino acid identity and 56.7% similarity. Human SEL-10 contains an F-box domain as found in other SCF family members. It also contains seven WD40/β-transducin repeats<sup>18 </sup>as seen in yeast Cdc4p and <i>C. elegans </i>SEL-10, suggesting that the protein forms a seven-bladed propeller structure<sup>19</sup>.</p>
    <p>We first assessed the physical interaction of human SEL-10 and PS1. SEL-10 tagged with an N-terminal 6-Myc epitope and PS1 were transiently co-expressed in human embryonic kidney cells (HEK293) and their interaction was assessed by immuno-precipitation. Complexes between SEL-10 and PS1 could only be detected in the presence of a proteasome inhibitor, lactacystin, which was added to the cultures at the time of transfection. When immunoprecipitated with anti-PS1 loop antibody, only the immuno-precipitate from cotransfected cells contained SEL-10-myc (FIG. <b>3</b>A), indicating that SEL-10 can interact with PS1. Since co-immunoprecipitation was not observed in cells without lactacystin treatment (FIG. <b>3</b>B), this suggests that the complex can only be captured by blocking the entry of PS1 into the proteasome degradation pathway. The interaction between SEL-10 and PS1 was confirmed in the reverse experiment by using anti-myc antibody for immunoprecipitation of SEL-10 (FIG. <b>3</b>C). PS1 is cleaved within the cytoplasmic loop between transmembrane domains six and seven by an unknown protease which generates N- and C-terminal fragments (PS1-NTF and PS1-CTF, respectively). The SEL-10-myc immunoprecipitates contained primarily full length and high molecular weight forms of PS1, but only very low or undetectable amounts of PS1-NTF and no PS1-CTF. This suggests that SEL-10 may bind primarily to unprocessed PS1.</p>
    <p>Next, we examined the effect of SEL-10 co-transfection on PS1 ubiquitination. PS1 immunoprecipitated from cotransfected cells contains a higher level of ubiquitination compared to cells transfected with PS1 alone as shown by probing with anti-ubiquitin antibody (FIG. <b>4</b>A). This result demonstrates that complex formation between SEL-10 and PS1 facilitates ubiquitination as implied previously by the need for lactacystin to demonstrate SEL-10/PS1 complex accumulation.</p>
    <p>We then investigated how ubiquitination affects PS1 protein level. HEK293 cells were cotransfected with either SEL-10 or PS1 alone or in combination and immunoprecipitates were probed with anti-PS1 loop antibody (FIG. <b>4</b>B). In cells co-transfected with PS1 and SEL-10 in comparison to cells transfected with PS1 alone, PS1-NTF and PS1 -CTF were decreased as were the high molecular weight forms of PS1. However, the amount of unprocessed PS1 appeared to be slightly increased. There are several possible interpretations of this complex cellular effect. The first is that SEL-10 mediated ubiquitination of PS1-CTF and PS1-NTF leads to their degradation. However, the fragments of PS1 are not immunoprecipitated with SEL-10 suggesting that SEL-10 does not directly facilitate ubiquitination of PS1 -NTF and PS1 -CTF. The slight increase in unprocessed PS1 seen in cells cotransfected with SEL-10 may suggest instead that SEL-10 binding to PS1 or SEL-10 mediated ubiquitination of PS1 inhibits proteolytic processing of PS1-NTF and PS1-CTF leading to accumulation of the unprocessed precursor.</p>
    <p>In order to examine the impact of SEL-10 mediated ubiquitination of PS1 on amyloid β-peptide (Aβ) production, SEL-10 was expressed by either transient or stable transfection in HEK293 cells with or without wild-type PS1. Aβ peptide production was measured by enzyme immunoassays that could distinguish the 1-40 and 1-42 forms of the peptide. In transient expression experiments (FIG. <b>5</b>A), coexpression of SEL-10 with APP increased production and release of Aβ1-40 and Aβ1-42 by more than 2-fold in comparison to APP expression alone. Transient coexpression of PS1 with APP also increased Aβ1-40 and Aβ1-42 levels by more than 3-fold, similar to previous reports <sup>20 </sup>although this is not a consistent finding<sup>2</sup>. Coexpression of SEL-10 and PS1 with APP had an additive effect with an increase in Aβ production of approximately 7-fold. No effect on the ratio of Aβ1-42/total Aβ was observed with all values falling in the range 8.5% to 13.5%. The stimulation of Aβ processing due to expression of SEL-10 was consistently observed across experiments, however, the degree of stimulation did vary. To confirm and extend the result, a series of HEK293 cell lines were derived with stable expression of SEL-10 or PS1 (FIG. <b>5</b>B). The two SEL-10 cell lines obtained showed 4- and 7-fold increases in endogenous Aβ1-40 peptide production compared to HEK293 cells transfected with just the pcDNA3.1 vector. Similarly, six cell lines with stable expression of PS1 showed a 2- to 6-fold increase in endogenous Aβ1-40 peptide production. Transient transfection of the stable cell lines with APP cDNA was performed to confirm and extend this result (FIG. <b>5</b>C). Production of Aβ was greater in the SEL-10 and PS1 cell lines in comparison to a control HEK293 cell line that had been transformed with just the pcDNA3.1 vector. Since SEL-10 decreases the cellular level of PS1-NTF and PS1 -CTF fragments while slightly increasing unprocessed PS1, the increase in unprocessed PS1 may be associated with the increase in Aβ peptide processing observed. Transient expression of PS1 has a similar effect. It increases the cellular level of unprocessed PS1 while having relatively little effect on the levels of PS1-NTF and PS1 -CTF fragments.</p>
    <p>Our data indicate that SEL-10 interacts with PS1, stimulates PS1 ubiquitination, and recruits it into the proteasome pathway for protein degradation. SEL-10 is likely to function as an adaptor protein that assembles the core catalytic complex of an SCF E2-E3 ubiquitin ligase<sup>14</sup>. Recognition of most SCF substrates by F-box/WD40 repeat adaptor proteins is phosphorylation dependent<sup>15</sup>, suggesting that this may be an additional level of cellular regulation of presenilin levels. To demonstrate complex formation between human SEL-10 and PS1 in HEK293 cells required inhibition of proteasome function by lactacystin. In contrast, the <i>C. elegans </i>SEL-10/SEL-12 complex accumulates in human HEK293 cells in the absence of proteasome inhibitors suggesting that nematode SEL-10 is unable to assemble the human core catalytic complex<sup>11</sup>. Degradation of an eight-pass transmembrane protein such as presenilin presents a topological hurdle since the presenilin protein must be extracted from the membrane and delivered to the proteasome. Like presenilins, a number of other multipass integral membrane proteins with large cytoplasmic domains such as the cystic fibrosis transmembrane conductance regulator (CFTR) are degraded through the ubiquitin-proteasome pathway<sup>21,22</sup>. This pathway also is important for quality control within the endoplasmic reticulum<sup>23 </sup>and conceivably could impact on intracellular production of Aβ peptide.</p>
    <p>Transfection of PS1 into human cells generally is observed to increase cellular levels of unprocessed PS1, but causes little change in levels of PS1-CTF and PS1-NTF suggesting that processing is under tight cellular control<sup>17</sup>, perhaps due to their involvement in apoptotic pathways<sup>24</sup>. As a consequence of increased transient or stable expression of PS1, we see an increase in Aβ peptide production that probably is due to the accumulation of unprocessed PS1. A similar effect on Aβ peptide production and accumulation of unprocessed PS1 is caused by increased expression of SEL-10. SEL-10 may directly regulate PS1 processing by binding to the cleavage site in the large cytoplasmic loop of PS1, or SEL-10 mediated ubiquitination may block PS1 processing. Similarly, a number of PS1 mutations associated with familial Alzheimer's disease have been shown to decrease processing of PS1 into N- and C-terminal fragments, although that is not a consistent finding<sup>25</sup>. However, mutant forms of PS1 are associated with a shift in the ratio of Aβ1-42 and Aβ1-40, whereas expression of wild-type PS1 or SEL-10 has no effect on the ratio of Aβ1-42/total Aβ<sup>1,2</sup>.</p>
    <p>The genetic data indicates that SEL-10 is a negative regulator of presenilin activity in <i>C. elegans</i>. Loss of SEL-10 function presumably rescues the egg laying defect in sel-12 mutant worms through facilitation of HOP-1 presenilin activity, perhaps by allowing the increased accumulation of processed N- and C-terminal fragments of HOP-1. Sel-10 was identified in a screen for mutations that increase presenilin activity<sup>11</sup>. In principle, genetic screens in model organisms such as <i>C. elegans </i>or Drosophila can be used to find mutations that decrease presenilin activity, the desired therapeutic goal in Alzheimer's disease<sup>26</sup>. Such screens have the potential to identify novel therapeutic targets for this devastating disease.</p>
    <p>Methods</p>
    <p>Cloning</p>
    <p>Incyte clone (028971) was identified as the human homologue of <i>C. elegans </i>sel-10 and its sequence was used to design four antisense oligonucleotide primers 5′-TCACT-TCATGTCCACATCAAAGTCC-3′ (SEQ ID NO: 28), 5′-GGTAATTACAAAGTTCTTG-TTGAACTG-3′ (SEQ ID NO: 29), 5′-CCCTGCAACGTGTGTAGACAGG-3′ (SEQ ID NO: 30), and 5′-CCAGTCTCTGCATTCCACACTTTG-3′ (SEQ ID NO: 31), to amplify the remainder of the human sel-10 sequence. “Electronic Northern” analysis revealed expression of sel-10 in hippocampus and mammary gland so these tissues were chosen for 5′RACE cloning using Marathon kit (CloneTech). Marathon-ready cDNA from hippocampus and mammary gland were prepared as directed in the kit. PCR products were cloned into the TA vector pCR3.1 (Invitrogen), and isolates were sequenced. An alternate 5′oligonucleotide primer was also designed based on Incyte clones that have 5′ ends that differ from the hippocampal sel-10 sequence (5′-CTCAGACAGGTCAGGACATTTGG-3′ (SEQ ID NO: 32). Blastn was used to search the Incyte databases LifeSeq and LifeSeqFL. Gap alignments and translations were performed with GCG programs.</p>
    <p>Plasmids and Transfections</p>
    <p>The human sel-10 cDNA was inserted into the EcoR1 site of the vector pCS2+MT (gift of Jan Kitajewski, Columbia University College of Physicians and Surgeons). This fused a 5′ 6-myc epitope tag in-frame to the fifth methionine of the hippocampal sel-10 cDNA. The hippocampal and mammary sel-10 cDNA diverge upstream of this methionine. A PS1 cDNA with a 3′-FLAG tag (PS1 -C-FLAG) was inserted into the pcDNA3.1 vector. An APP cDNA containing the Swedish KM→NL mutation and an attenuated ER retention sequence consisting of a C-terminal di-lysine motif (APP695Sw-KK) was inserted into the pIRES-EGFP vector (Clontech). HEK293 cells were grown to 80% confluence in DMEM with 10% FBS and transfected with the above cDNAs. A total of 10 μg DNA/6×10<sup>6 </sup>cells was used for transfection with a single plasmid. For cotransfections of multiple plasmids, an equal amount of each plasmid was used for a total of 10 μg DNA using LipofectAmine (BRL). Single cells were sorted into each well of one 96 well plate containing growth medium without G418 by FACS using an EPICS Elite ESP flow cytometer (Coulter, Hialeah, Fla.) equipped with a 488 nm excitation line supplied by an air-cooled argon laser. After a four day recovery period, G418 was added to the medium to a final concentration of 400 μg/ml for the selection of stably transfected cell lines. Stable expression of a cDNA was confirmed by detection of the specific protein or sequence tag by Western blot.</p>
    <p>Immunoprecipitation, Western Blot, and ELISA</p>
    <p>Aβ and PS1 antibodies have been characterized<sup>6,27</sup>. Cell lysates with equal amount of protein were precipitated with antibody and protein G-Sepharose at 4° C. for 2 hr and beads were washed three times with TENT buffer<sup>11</sup>. Immunoprecipitates were analyzed by Western blot using 4-12% NuPage Bis-Tris gel and PVDF membrane (Novex), peroxidase-conjugated secondary antibody (Vector) and SuperSignal West Pico Luminol/Enhancer (Pierce). The ELISA for Aβ 1-40 and Aβ 1-42 was performed as described<sup>27</sup>.</p>
    <p>References</p>
    <p>1. Borchelt, D. R. et al. Familial Alzheimer's disease-linked presenilin 1 variants elevate Abeta 1-42/1-40 ratio in vitro and in vivo. Neuron 17, 1005-1013 (1996).</p>
    <p>2. Citron, M. et al. Mutant presenilins of Alzheimer's disease increase production of 42-residue amyloid beta-protein in both transfected cells and transgenic mice. Nat. Med. 3, 67-72 (1997).</p>
    <p>3. Doan, A. et al. Protein topology of presenilin 1. Neuron. 17, 1023-1030 (1996).</p>
    <p>4. Li, X. &amp; Greenwald, I. Membrane topology of the <i>C. elegans </i>SEL-12 presenilin. Neuron 17, 1015-1021 (1996).</p>
    <p>5. De Strooper, B. et al. Phosphorylation, subcellular localization, and membrane orientation of the Alzheimer's disease-associated presenilins. J. Biol. Chem. 272, 3590-3598 (1997).</p>
    <p>6. Li, J. et al., Alzheimer presenilins in the nuclear membrane, interphase kinetochores, and centrosomes suggest a role in chromosome segregation. Cell 90, 917-927 (1997).</p>
    <p>7. Levitan, D. &amp; Greenwald, I. Facilitation of lin, 12, mediated signalling by sel, 12, a <i>Caenorhabditis elegans </i>S182 Alzheimer's disease gene. Nature 377, 351-4 (1995).</p>
    <p>8. Li, X. &amp; Greenwald, I. HOP, 1, a <i>Caenorhabditis elegans </i>presenilin, appears to be functionally redundant with SEL, 12 presenilin and to facilitate LIN, 12 and GLP, 1 signaling. Proc. Natl. Acad. Sci. U.S.A. 94,12204-9 (1997).</p>
    <p>9. Westlund, B. et al. Reverse genetic analysis of <i>Caenorhabditis elegans </i>presenilins reveals redundant but unequal roles for sel-12 and hop-1 in Notch-pathway signaling. Proc. Natl. Acad. Sci. U.S.A. 96, 2497-2502 (1999).</p>
    <p>10. Levitan, D. et al. Assessment of normal and mutant human presenilin function in <i>Caenorhabditis elegans</i>. Proc. Natl. Acad. Sci. U.S.A. 93, 14940-1494 (1996).</p>
    <p>11. Wu, G. et al. Evidence for functional and physical association between <i>Caenorhabditis elegans </i>SEL, 10, a Cdc4p, related protein, and SEL, 12 presenilin. Proc. Natl. Acad. Sci. U.S.A. 95, 15787-1 (1998).</p>
    <p>12. Hubbard, E. J. et al. Sel, 10, a negative regulator of lin, 12 activity in <i>Caenorhabditis elegans</i>, encodes a member of the CDC4 family of proteins. Genes. Dev. 11, 3182-93 (1997).</p>
    <p>13. Selkoe, D. J. Physiological production of the b-amyloid protein and the mechanism of Alzheimer's disease. Trends Neurosci. 16, 403-409 (1993).</p>
    <p>14. Tyers, M. &amp; Willems, A. R. One ring to rule a superfamily of E3 ubiquitin ligases. Science 284, 601-604 (1999).</p>
    <p>15. Patton, E. E. et al., Combinatorial control in ubiquitin-dependent proteolysis: don't Skp the F-box hypothesis. Trends Genet. 14, 236-243 (1998).</p>
    <p>16. Steiner, H. et al. Expression of Alzheimer's disease-associated presenilin-1 is controlled by proteolytic degradation and complex formation. J. Biol. Chem. 273, 32322-32331 (1998).</p>
    <p>17. Kim, T. W. et al. Endoproteolytic cleavage and proteasomal degradation of presenilin 2 in transfected cells. J. Biol. Chem. 272, 11006-11010 (1997).</p>
    <p>18. Neer E. J. et al. The ancient regulatory-protein family of WD-repeat proteins. Nature 371, 297-300, (1994).</p>
    <p>19. Sondek, J. et al. Crystal structure of a G-protein beta gamma dimmer at 2.1A resolution. Nature 379, 369-374 (1996).</p>
    <p>20. Ancolio, K. et al. Alpha-secretase-derived product of beta-amyloid precursor protein is decreased by presenilin 1 mutations linked to familial Alzheimer's disease. J. Neurochem. 69, 2494-2499 (1997).</p>
    <p>21. Ward, C. L. et al. Degradation of CFTR by the ubiquitin-proteasome pathway. Cell 83, 121-127 (1995).</p>
    <p>22. Johnston, J. A. et al. Aggresomes: a cellular response to misfolded proteins. J. Cell Biol. 143: 1883-1898 (1998).</p>
    <p>23. Kopito, R. R. ER quality control: the cytoplasmic connection. Cell. 88, 427-430 (1997).</p>
    <p>24. Zhang, Z. et al. Destabilization of beta-catenin by mutations in presenilin-1 potentiates neuronal apoptosis. Nature. 395: 698-702 (1998).</p>
    <p>25. Shirotani, K. et al. Effects of presenilin N-terminal fragments on production of amyloid b peptide and accumulation of endogenous presenilins. Neurosci. Let. 262, 37-40 (1999).</p>
    <p>26. De Strooper, B. et al. Deficiency of presenilin-1 inhibits the normal cleavage of amyloid precursor protein. Nature 391, 387-390 (1998).</p>
    <p>27. Mehta, P. D. et al. Increased plasma amyloid beta protein <b>1-42 </b>levels in Down syndrome. Neurosci Lett 241, 13-16 (1998).</p>
    <p>It will be clear that the invention may be practiced otherwise than as particularly described in the foregoing description and examples.</p>
    <p>Numerous modifications and variations of the present invention are possible in light of the above teachings and, therefore, are within the scope of the invention.</p>
    <p>The entire disclosure of all publications cited herein are hereby incorporated by reference.</p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 1</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Effect of 6myc-N-sel-10 and PS1-C-FLAG transient</td>
              </tr> <tr class="description-tr"> <td class="description-td">transfection on Ab levels in HEK293 cell supernatants.</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="4"> <colspec colname="1" colwidth="63pt" align="left"> </colspec> <colspec colname="2" colwidth="42pt" align="left"> </colspec> <colspec colname="3" colwidth="49pt" align="center"> </colspec> <colspec colname="4" colwidth="63pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Plasmids</td>
                <td class="description-td">Ab1-42</td>
                <td class="description-td">Ab1-40</td>
                <td class="description-td">Ab1-42/total Ab</td>
              </tr> <tr class="description-tr"> <td class="description-td">Transfected</td>
                <td class="description-td">ng/ml</td>
                <td class="description-td">ng/ml</td>
                <td class="description-td">ng/ml</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="4" align="center" rowsep="1" class="description-td" colspan="4"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">pcDNA3</td>
                <td class="description-td"> 81 ± 20</td>
                <td class="description-td">231 ± 50</td>
                <td class="description-td">0.26 ± 0.05</td>
              </tr> <tr class="description-tr"> <td class="description-td">6myc-N-sel-10</td>
                <td class="description-td"> 67 ± 7</td>
                <td class="description-td">246 ± 34</td>
                <td class="description-td">0.21 ± 0.03</td>
              </tr> <tr class="description-tr"> <td class="description-td">PS1-C-FLAG</td>
                <td class="description-td"> 75 ± 18</td>
                <td class="description-td">227 ± 45</td>
                <td class="description-td">0.25 ± 0.03</td>
              </tr> <tr class="description-tr"> <td class="description-td">PS1-C-FLAG +</td>
                <td class="description-td"> 77 ± 21</td>
                <td class="description-td">220 ± 26</td>
                <td class="description-td">0.25 ± 0.03</td>
              </tr> <tr class="description-tr"> <td class="description-td">6myc-N-sel-10</td>
              </tr> <tr class="description-tr"> <td class="description-td">APP695NL-KK</td>
                <td class="description-td">141 ± 27</td>
                <td class="description-td"> 896 ± 103</td>
                <td class="description-td">0.14 ± 0.02</td>
              </tr> <tr class="description-tr"> <td class="description-td">APP695NL-KK +</td>
                <td class="description-td">308 ± 17</td>
                <td class="description-td">2576 ± 190</td>
                <td class="description-td">0.11 ± 0.00</td>
              </tr> <tr class="description-tr"> <td class="description-td">6myc-N-sel-10</td>
              </tr> <tr class="description-tr"> <td class="description-td">APP695NL-KK +</td>
                <td class="description-td">364 ± 39</td>
                <td class="description-td">3334 ± 337</td>
                <td class="description-td">0.09 ± 0.00</td>
              </tr> <tr class="description-tr"> <td class="description-td">PS1-C-FLAG</td>
              </tr> <tr class="description-tr"> <td class="description-td">APP695NL-KK +</td>
                <td class="description-td">550 ± 20</td>
                <td class="description-td">5897 ± 388</td>
                <td class="description-td">0.09 ± 0.00</td>
              </tr> <tr class="description-tr"> <td class="description-td">PS1-C-FLAG +</td>
              </tr> <tr class="description-tr"> <td class="description-td">6myc-N-sel-10</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="4" align="center" rowsep="1" class="description-td" colspan="4"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 2</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Effect of 6myc-N-sel-10 and PS1-C-FLAG transient</td>
              </tr> <tr class="description-tr"> <td class="description-td">transfected on Ab levels in IMR32 cell supernatants.</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="4"> <colspec colname="1" colwidth="63pt" align="left"> </colspec> <colspec colname="2" colwidth="42pt" align="left"> </colspec> <colspec colname="3" colwidth="49pt" align="left"> </colspec> <colspec colname="4" colwidth="63pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">Plasmids</td>
                <td class="description-td">Ab1-42</td>
                <td class="description-td">Ab1-40</td>
                <td class="description-td">Ab1-42/total Ab</td>
              </tr> <tr class="description-tr"> <td class="description-td">Transfected</td>
                <td class="description-td">ng/ml</td>
                <td class="description-td">ng/ml</td>
                <td class="description-td">ng/ml</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="4" align="center" rowsep="1" class="description-td" colspan="4"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">pcDNA3</td>
                <td class="description-td">65 ± 3</td>
                <td class="description-td">319 ± 146</td>
                <td class="description-td">0.19 ± 0.06</td>
              </tr> <tr class="description-tr"> <td class="description-td">6myc-N-sel-10</td>
                <td class="description-td">63 ± 0</td>
                <td class="description-td">246 ± 53</td>
                <td class="description-td">0.21 ± 0.04</td>
              </tr> <tr class="description-tr"> <td class="description-td">PS1-C-FLAG</td>
                <td class="description-td">67 ± 6</td>
                <td class="description-td">307 ± 79</td>
                <td class="description-td">0.18 ± 0.04</td>
              </tr> <tr class="description-tr"> <td class="description-td">PS1-C-FLAG +</td>
                <td class="description-td">67 ± 6</td>
                <td class="description-td">302 ± 94</td>
                <td class="description-td">0.20 ± 0.08</td>
              </tr> <tr class="description-tr"> <td class="description-td">6myc-N-sel-10</td>
              </tr> <tr class="description-tr"> <td class="description-td">APP695NL-KK</td>
                <td class="description-td">66 ± 5</td>
                <td class="description-td">348 ± 110</td>
                <td class="description-td">0.17 ± 0.05</td>
              </tr> <tr class="description-tr"> <td class="description-td">APP695NL-KK +</td>
                <td class="description-td">75 ± 18</td>
                <td class="description-td">448 ± 141</td>
                <td class="description-td">0.15 ± 0.03</td>
              </tr> <tr class="description-tr"> <td class="description-td">6myc-N-sel-10</td>
              </tr> <tr class="description-tr"> <td class="description-td">APP695NL-KK +</td>
                <td class="description-td">63 ± 26</td>
                <td class="description-td">446 ± 72</td>
                <td class="description-td">0.12 ± 0.02</td>
              </tr> <tr class="description-tr"> <td class="description-td">PS1-C-FLAG</td>
              </tr> <tr class="description-tr"> <td class="description-td">APP695NL-KK +</td>
                <td class="description-td">81 ± 26</td>
                <td class="description-td">565 ± 179</td>
                <td class="description-td">0.12 ± 0.01</td>
              </tr> <tr class="description-tr"> <td class="description-td">PS1-C-FLAG +</td>
              </tr> <tr class="description-tr"> <td class="description-td">6myc-N-sel-10</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="4" align="center" rowsep="1" class="description-td" colspan="4"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 3</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Endogenous and exogenous Ab1-40 and Ab1-42 levels in</td>
              </tr> <tr class="description-tr"> <td class="description-td">supernatants from stable transformants of HEK293 cells.</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="1" colwidth="56pt" align="left"> </colspec> <colspec colname="2" colwidth="77pt" align="center"> </colspec> <colspec colname="3" colwidth="84pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">GFP</td>
                <td class="description-td">APP695NL-KK</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Transfection</td>
                <td class="description-td">Transfection</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="5"> <colspec colname="1" colwidth="56pt" align="left"> </colspec> <colspec colname="2" colwidth="35pt" align="left"> </colspec> <colspec colname="3" colwidth="42pt" align="center"> </colspec> <colspec colname="4" colwidth="42pt" align="left"> </colspec> <colspec colname="5" colwidth="42pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Ab1-40</td>
                <td class="description-td">Ab1-42</td>
                <td class="description-td">Ab1-40</td>
                <td class="description-td">Ab1-42</td>
              </tr> <tr class="description-tr"> <td class="description-td">Stable Line</td>
                <td class="description-td">ng/ml</td>
                <td class="description-td">ng/ml</td>
                <td class="description-td">ng/ml</td>
                <td class="description-td">ng/ml</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="5" align="center" rowsep="1" class="description-td" colspan="5"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">6myc-N-sel10/2</td>
                <td class="description-td">297 ± 29</td>
                <td class="description-td">109 ± 17 </td>
                <td class="description-td">4877 ± 547</td>
                <td class="description-td">750 ± 32</td>
              </tr> <tr class="description-tr"> <td class="description-td">6myc-N-sel10/6</td>
                <td class="description-td">168 ± 18</td>
                <td class="description-td">85 ± 11</td>
                <td class="description-td">8310 ± 308</td>
                <td class="description-td">1913 ± 19 </td>
              </tr> <tr class="description-tr"> <td class="description-td">PS1-C-FLAG/2</td>
                <td class="description-td"> 97 ± 6</td>
                <td class="description-td">68 ± 8 </td>
                <td class="description-td">3348 ± 68</td>
                <td class="description-td">493 ± 21</td>
              </tr> <tr class="description-tr"> <td class="description-td">PS1-C-FLAG/8</td>
                <td class="description-td">118 ± 11</td>
                <td class="description-td">85 ± 17</td>
                <td class="description-td">3516 ± 364</td>
                <td class="description-td">515 ± 36</td>
              </tr> <tr class="description-tr"> <td class="description-td">PS1-C-FLAG/9</td>
                <td class="description-td"> 83 ± 20</td>
                <td class="description-td">67 ± 16</td>
                <td class="description-td">2369 ± 73</td>
                <td class="description-td">350 ± 12</td>
              </tr> <tr class="description-tr"> <td class="description-td">PS1-C-FLAG/11</td>
                <td class="description-td">152 ± 17</td>
                <td class="description-td">68 ± 13</td>
                <td class="description-td">4771 ± 325</td>
                <td class="description-td">599 ± 25</td>
              </tr> <tr class="description-tr"> <td class="description-td">PS1-C-FLAG/12</td>
                <td class="description-td">141 ± 12</td>
                <td class="description-td">50 ± 10</td>
                <td class="description-td">4095 ± 210</td>
                <td class="description-td">449 ± 21</td>
              </tr> <tr class="description-tr"> <td class="description-td">PS1-C-FLAG/13</td>
                <td class="description-td">270 ± 139</td>
                <td class="description-td">61 ± 28</td>
                <td class="description-td">6983 ± 304</td>
                <td class="description-td">745 ± 41</td>
              </tr> <tr class="description-tr"> <td class="description-td">pcDNA3/1</td>
                <td class="description-td"> 43 ± 13</td>
                <td class="description-td">75 ± 15</td>
                <td class="description-td">1960 ± 234</td>
                <td class="description-td">61 ± 6</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="5" align="center" rowsep="1" class="description-td" colspan="5"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" pgwide="1" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="280pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 4</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Sel-10 constructs with epitope tags at the N or C terminus increase Aβ 1-40 and Aβ 1-42.</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="7"> <colspec colname="1" colwidth="56pt" align="left"> </colspec> <colspec colname="2" colwidth="42pt" align="center"> </colspec> <colspec colname="3" colwidth="35pt" align="center"> </colspec> <colspec colname="4" colwidth="35pt" align="center"> </colspec> <colspec colname="5" colwidth="42pt" align="center"> </colspec> <colspec colname="6" colwidth="35pt" align="center"> </colspec> <colspec colname="7" colwidth="35pt" align="center"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">construct</td>
                <td class="description-td">Aβ 1-40</td>
                <td class="description-td">% increase</td>
                <td class="description-td">P-value</td>
                <td class="description-td">Aβ 1-42</td>
                <td class="description-td">% increase</td>
                <td class="description-td">P-value</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="7" align="center" rowsep="1" class="description-td" colspan="7"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">pcDNA</td>
                <td class="description-td">4240 ± 102</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">614 ± 10</td>
                <td class="description-td"> </td>
                <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">6myc-N-sel-10</td>
                <td class="description-td">7631 ± 465</td>
                <td class="description-td">80%</td>
                <td class="description-td">3.7 × 10<sup>−6</sup> </td>
                <td class="description-td">1136 ± 73 </td>
                <td class="description-td">46%</td>
                <td class="description-td">7.9 × 10<sup>−6</sup> </td>
              </tr> <tr class="description-tr"> <td class="description-td">sel-10-C-mychis</td>
                <td class="description-td">5485 ± 329</td>
                <td class="description-td">29%</td>
                <td class="description-td">1.8 × 10<sup>−4</sup> </td>
                <td class="description-td">795 ± 50</td>
                <td class="description-td">29%</td>
                <td class="description-td">4.0 × 10<sup>−4</sup> </td>
              </tr> <tr class="description-tr"> <td class="description-td">sel-10-C-V5his</td>
                <td class="description-td">6210 ± 498</td>
                <td class="description-td">46%</td>
                <td class="description-td">1.2 × 10<sup>−4</sup> </td>
                <td class="description-td">906 ± 73</td>
                <td class="description-td">48%</td>
                <td class="description-td">2.1 × 10<sup>−4</sup> </td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="7" align="center" rowsep="1" class="description-td" colspan="7"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p>
      <sequence-cwu id="SEQLST-1"> <sequence-list-new-rules> <s100> <s160>32</s160> </s100> <s200> <s210>1</s210> <s211>3550</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> <s220> <s221>misc_feature</s221> <s222>(2485)..(2485)</s222> <s223>residue uncertain</s223> </s220> </s200> <s400> 1
ctcattattc cctcgagttc ttctcagtca agctgcatgt atgtatgtgt gtcccgagaa     60
gcggtttgat actgagctgc atttgccttt actgtggagt tttgttgccg gttctgctcc    120
ctaatcttcc ttttctgacg tgcctgagca tgtccacatt agaatctgtg acatacctac    180
ctgaaaaagg tttatattgt cagagactgc caagcagccg gacacacggg ggcacagaat    240
cactgaaggg gaaaaataca gaaaatatgg gtttctacgg cacattaaaa atgatttttt    300
acaaaatgaa aagaaagttg gaccatggtt ctgaggtccg ctctttttct ttgggaaaga    360
aaccatgcaa agtctcagaa tatacaagta ccactgggct tgtaccatgt tcagcaacac    420
caacaacttt tggggacctc agagcagcca atggccaagg gcaacaacga cgccgaatta    480
catctgtcca gccacctaca ggcctccagg aatggctaaa aatgtttcag agctggagtg    540
gaccagagaa attgcttgct ttagatgaac tcattgatag ttgtgaacca acacaagtaa    600
aacatatgat gcaagtgata gaaccccagt ttcaacgaga cttcatttca ttgctcccta    660
aagagttggc actctatgtg ctttcattcc tggaacccaa agacctgcta caagcagctc    720
agacatgtcg ctactggaga attttggctg aagacaacct tctctggaga gagaaatgca    780
aagaagaggg gattgatgaa ccattgcaca tcaagagaag aaaagtaata aaaccaggtt    840
tcatacacag tccatggaaa agtgcataca tcagacagca cagaattgat actaactgga    900
ggcgaggaga actcaaatct cctaaggtgc tgaaaggaca tgatgatcat gtgatcacat    960
gcttacagtt ttgtggtaac cgaatagtta gtggttctga tgacaacact ttaaaagttt   1020
ggtcagcagt cacaggcaaa tgtctgagaa cattagtggg acatacaggt ggagtatggt   1080
catcacaaat gagagacaac atcatcatta gtggatctac agatcggaca ctcaaagtgt   1140
ggaatgcaga gactggagaa tgtatacaca ccttatatgg gcatacttcc actgtgcgtt   1200
gtatgcatct tcatgaaaaa agagttgtta gcggttctcg agatgccact cttagggttt   1260
gggatattga gacaggccag tgtttacatg ttttgatggg tcatgttgca gcagtccgct   1320
gtgttcaata tgatggcagg agggttgtta gtggagcata tgattttatg gtaaaggtgt   1380
gggatccaga gactgaaacc tgtctacaca cgttgcaggg gcatactaat agagtctatt   1440
cattacagtt tgatggtatc catgtggtga gtggatctct tgatacatca atccgtgttt   1500
gggatgtgga gacagggaat tgcattcaca cgttaacagg gcaccagtcg ttaacaagtg   1560
gaatggaact caaagacaat attcttgtct ctgggaatgc agattctaca gttaaaatct   1620
gggatatcaa aacaggacag tgtttacaaa cattgcaagg tcccaacaag catcagagtg   1680
ctgtgacctg tttacagttc aacaagaact ttgtaattac cagctcagat gatggaactg   1740
taaaactatg ggacttgaaa acgggtgaat ttattcgaaa cctagtcaca ttggagagtg   1800
gggggagtgg gggagttgtg tggcggatca gagcctcaaa cacaaagctg gtgtgtgcag   1860
ttgggagtcg gaatgggact gaagaaacca agctgctggt gctggacttt gatgtggaca   1920
tgaagtgaag agcagaaaag atgaatttgt ccaattgtgt agacgatata ctccctgccc   1980
ttccccctgc aaaaagaaaa aaagaaaaga aaaagaaaaa aatcccttgt tctcagtggt   2040
gcaggatgtt ggcttggggc aacagattga aaagacctac agactaagaa ggaaaagaag   2100
aagagatgac aaaccataac tgacaagaga ggcgtctgct gtctcatcac ataaaaggct   2160
tcacttttga ctgagggcag ctttgcaaaa tgagactttc taaatcaaac caggtgcaat   2220
tatttcttta ttttcttctc cagtggtcat tggggcagtg ttaatgctga aacatcatta   2280
cagattctgc tagcctgttc ttttaccact gacagctaga cacctagaaa ggaactgcaa   2340
taatatcaaa acaagtactg gttgactttc taattagaga gcatctgcaa caaaaagtca   2400
tttttctgga gtggaaaagc ttaaaaaaat tactgtgaat tgtttttgta cagttatcat   2460
gaaaagcttt tttttttatt ttttngccaa ccattgccaa tgtcaatcaa tcacagtatt   2520
agcctctgtt aatctattta ctgttgcttc catatacatt cttcaatgca tatgttgctc   2580
aaaggtggca agttgtcctg ggttctgtga gtcctgagat ggatttaatt cttgatgctg   2640
gtgctagaag taggtcttca aatatgggat tgttgtccca accctgtact gtactcccag   2700
tggccaaact tatttatgct gctaaatgaa agaaagaaaa aagcaaatta ttttttttat   2760
tttttttctg ctgtgacgtt ttagtcccag actgaattcc aaatttgctc tagtttggtt   2820
atggaaaaaa gactttttgc cactgaaact tgagccatct gtgcctctaa gaggctgaga   2880
atggaagagt ttcagataat aaagagtgaa gtttgcctgc aagtaaagaa ttgagagtgt   2940
gtgcaaagct tattttcttt tatctgggca aaaattaaaa cacattcctt ggaacagagc   3000
tattacttgc ctgttctgtg gagaaacttt tctttttgag ggctgtggtg aatggatgaa   3060
cgtacatcgt aaaactgaca aaatatttta aaaatatata aaacacaaaa ttaaaataaa   3120
gttgctggtc agtcttagtg ttttacagta tttgggaaaa caactgttac agttttattg   3180
ctctgagtaa ctgacaaagc agaaactatt cagtttttgt agtaaaggcg tcacatgcaa   3240
acaaacaaaa tgaatgaaac agtcaaatgg tttgcctcat tctccaagag ccacaactca   3300
agctgaactg tgaaagtggt ttaacactgt atcctaggcg atcttttttc ctccttctgt   3360
ttattttttt gnttgtttta tttatagtct gatttaaaac aatcagattc aagttggtta   3420
attttagtta tgtaacaacc tgacatgatg gaggaaaaca acctttaaag ggattgtgtc   3480
tatggtttga ttcacttaga aattttattt tcttataact taagtgcaat aaaatgtgtt   3540
ttttcatgtt                                                          3550
</s400> <s200> <s210>2</s210> <s211>3571</s211> <s212>DNA</s212> <s213>Homo sapiens</s213> <s220> <s221>misc_feature</s221> <s222>(2506)..(2506)</s222> <s223>residue uncertain</s223> </s220> </s200> <s400> 2
ctcagcaggt caggacattt ggtaggggaa ggttgaaaga caaaagcagc aggccttggg     60
ttctcagcct tttaaaaact attattaaat atatattttt aaaatttagt ggttagagct    120
tttagtaatg tgcctgtatt acatgtagag agtattcgtc aaccaagagg agttttaaaa    180
tgtcaaaacc gggaaaacct actctaaacc atggcttggt tcctgttgat cttaaaagtg    240
caaaagagcc tctaccacat caaaccgtga tgaagatatt tagcattagc atcattgccc    300
aaggcctccc tttttgtcga agacggatga aaagaaagtt ggaccatggt tctgaggtcc    360
gctctttttc tttgggaaag aaaccatgca aagtctcaga atatacaagt accactgggc    420
ttgtaccatg ttcagcaaca ccaacaactt ttggggacct cagagcagcc aatggccaag    480
ggcaacaacg acgccgaatt acatctgtcc agccacctac aggcctccag gaatggctaa    540
aaatgtttca gagctggagt ggaccagaga aattgcttgc tttagatgaa ctcattgata    600
gttgtgaacc aacacaagta aaacatatga tgcaagtgat agaaccccag tttcaacgag    660
acttcatttc attgctccct aaagagttgg cactctatgt gctttcattc ctggaaccca    720
aagacctgct acaagcagct cagacatgtc gctactggag aattttggct gaagacaacc    780
ttctctggag agagaaatgc aaagaagagg ggattgatga accattgcac atcaagagaa    840
gaaaagtaat aaaaccaggt ttcatacaca gtccatggaa aagtgcatac atcagacagc    900
acagaattga tactaactgg aggcgaggag aactcaaatc tcctaaggtg ctgaaaggac    960
atgatgatca tgtgatcaca tgcttacagt tttgtggtaa ccgaatagtt agtggttctg   1020
atgacaacac tttaaaagtt tggtcagcag tcacaggcaa atgtctgaga acattagtgg   1080
gacatacagg tggagtatgg tcatcacaaa tgagagacaa catcatcatt agtggatcta   1140
cagatcggac actcaaagtg tggaatgcag agactggaga atgtatacac accttatatg   1200
ggcatacttc cactgtgcgt tgtatgcatc ttcatgaaaa aagagttgtt agcggttctc   1260
gagatgccac tcttagggtt tgggatattg agacaggcca gtgtttacat gttttgatgg   1320
gtcatgttgc agcagtccgc tgtgttcaat atgatggcag gagggttgtt agtggagcat   1380
atgattttat ggtaaaggtg tgggatccag agactgaaac ctgtctacac acgttgcagg   1440
ggcatactaa tagagtctat tcattacagt ttgatggtat ccatgtggtg agtggatctc   1500
ttgatacatc aatccgtgtt tgggatgtgg agacagggaa ttgcattcac acgttaacag   1560
ggcaccagtc gttaacaagt ggaatggaac tcaaagacaa tattcttgtc tctgggaatg   1620
cagattctac agttaaaatc tgggatatca aaacaggaca gtgtttacaa acattgcaag   1680
gtcccaacaa gcatcagagt gctgtgacct gtttacagtt caacaagaac tttgtaatta   1740
ccagctcaga tgatggaact gtaaaactat gggacttgaa aacgggtgaa tttattcgaa   1800
acctagtcac attggagagt ggggggagtg ggggagttgt gtggcggatc agagcctcaa   1860
acacaaagct ggtgtgtgca gttgggagtc ggaatgggac tgaagaaacc aagctgctgg   1920
tgctggactt tgatgtggac atgaagtgaa gagcagaaaa gatgaatttg tccaattgtg   1980
tagacgatat actccctgcc cttccccctg caaaaagaaa aaaagaaaag aaaaagaaaa   2040
aaatcccttg ttctcagtgg tgcaggatgt tggcttgggg caacagattg aaaagaccta   2100
cagactaaga aggaaaagaa gaagagatga caaaccataa ctgacaagag aggcgtctgc   2160
tgtctcatca cataaaaggc ttcacttttg actgagggca gctttgcaaa atgagacttt   2220
ctaaatcaaa ccaggtgcaa ttatttcttt attttcttct ccagtggtca ttggggcagt   2280
gttaatgctg aaacatcatt acagattctg ctagcctgtt cttttaccac tgacagctag   2340
acacctagaa aggaactgca ataatatcaa aacaagtact ggttgacttt ctaattagag   2400
agcatctgca acaaaaagtc atttttctgg agtggaaaag cttaaaaaaa ttactgtgaa   2460
ttgtttttgt acagttatca tgaaaagctt ttttttttat tttttngcca accattgcca   2520
atgtcaatca atcacagtat tagcctctgt taatctattt actgttgctt ccatatacat   2580
tcttcaatgc atatgttgct caaaggtggc aagttgtcct gggttctgtg agtcctgaga   2640
tggatttaat tcttgatgct ggtgctagaa gtaggtcttc aaatatggga ttgttgtccc   2700
aaccctgtac tgtactccca gtggccaaac ttatttatgc tgctaaatga aagaaagaaa   2760
aaagcaaatt atttttttta ttttttttct gctgtgacgt tttagtccca gactgaattc   2820
caaatttgct ctagtttggt tatggaaaaa agactttttg ccactgaaac ttgagccatc   2880
tgtgcctcta agaggctgag aatggaagag tttcagataa taaagagtga agtttgcctg   2940
caagtaaaga attgagagtg tgtgcaaagc ttattttctt ttatctgggc aaaaattaaa   3000
acacattcct tggaacagag ctattacttg cctgttctgt ggagaaactt ttctttttga   3060
gggctgtggt gaatggatga acgtacatcg taaaactgac aaaatatttt aaaaatatat   3120
aaaacacaaa attaaaataa agttgctggt cagtcttagt gttttacagt atttgggaaa   3180
acaactgtta cagttttatt gctctgagta actgacaaag cagaaactat tcagtttttg   3240
tagtaaaggc gtcacatgca aacaaacaaa atgaatgaaa cagtcaaatg gtttgcctca   3300
ttctccaaga gccacaactc aagctgaact gtgaaagtgg tttaacactg tatcctaggc   3360
gatctttttt cctccttctg tttatttttt tgnttgtttt atttatagtc tgatttaaaa   3420
caatcagatt caagttggtt aattttagtt atgtaacaac ctgacatgat ggaggaaaac   3480
aacctttaaa gggattgtgt ctatggtttg attcacttag aaattttatt ttcttataac   3540
ttaagtgcaa taaaatgtgt tttttcatgt t                                  3571
</s400> <s200> <s210>3</s210> <s211>627</s211> <s212>PRT</s212> <s213>Homo sapiens</s213> </s200> <s400> 3
Met Cys Val Pro Arg Ser Gly Leu Ile Leu Ser Cys Ile Cys Leu Tyr
  1               5                  10                  15
Cys Gly Val Leu Leu Pro Val Leu Leu Pro Asn Leu Pro Phe Leu Thr
             20                  25                  30
Cys Leu Ser Met Ser Thr Leu Glu Ser Val Thr Tyr Leu Pro Glu Lys
         35                  40                  45
Gly Leu Tyr Cys Gln Arg Leu Pro Ser Ser Arg Thr His Gly Gly Thr
     50                  55                  60
Glu Ser Leu Lys Gly Lys Asn Thr Glu Asn Met Gly Phe Tyr Gly Thr
 65                  70                  75                  80
Leu Lys Met Ile Phe Tyr Lys Met Lys Arg Lys Leu Asp His Gly Ser
                 85                  90                  95
Glu Val Arg Ser Phe Ser Leu Gly Lys Lys Pro Cys Lys Val Ser Glu
            100                 105                 110
Tyr Thr Ser Thr Thr Gly Leu Val Pro Cys Ser Ala Thr Pro Thr Thr
        115                 120                 125
Phe Gly Asp Leu Arg Ala Ala Asn Gly Gln Gly Gln Gln Arg Arg Arg
    130                 135                 140
Ile Thr Ser Val Gln Pro Pro Thr Gly Leu Gln Glu Trp Leu Lys Met
145                 150                 155                 160
Phe Gln Ser Trp Ser Gly Pro Glu Lys Leu Leu Ala Leu Asp Glu Leu
                165                 170                 175
Ile Asp Ser Cys Glu Pro Thr Gln Val Lys His Met Met Gln Val Ile
            180                 185                 190
Glu Pro Gln Phe Gln Arg Asp Phe Ile Ser Leu Leu Pro Lys Glu Leu
        195                 200                 205
Ala Leu Tyr Val Leu Ser Phe Leu Glu Pro Lys Asp Leu Leu Gln Ala
    210                 215                 220
Ala Gln Thr Cys Arg Tyr Trp Arg Ile Leu Ala Glu Asp Asn Leu Leu
225                 230                 235                 240
Trp Arg Glu Lys Cys Lys Glu Glu Gly Ile Asp Glu Pro Leu His Ile
                245                 250                 255
Lys Arg Arg Lys Val Ile Lys Pro Gly Phe Ile His Ser Pro Trp Lys
            260                 265                 270
Ser Ala Tyr Ile Arg Gln His Arg Ile Asp Thr Asn Trp Arg Arg Gly
        275                 280                 285
Glu Leu Lys Ser Pro Lys Val Leu Lys Gly His Asp Asp His Val Ile
    290                 295                 300
Thr Cys Leu Gln Phe Cys Gly Asn Arg Ile Val Ser Gly Ser Asp Asp
305                 310                 315                 320
Asn Thr Leu Lys Val Trp Ser Ala Val Thr Gly Lys Cys Leu Arg Thr
                325                 330                 335
Leu Val Gly His Thr Gly Gly Val Trp Ser Ser Gln Met Arg Asp Asn
            340                 345                 350
Ile Ile Ile Ser Gly Ser Thr Asp Arg Thr Leu Lys Val Trp Asn Ala
        355                 360                 365
Glu Thr Gly Glu Cys Ile His Thr Leu Tyr Gly His Thr Ser Thr Val
    370                 375                 380
Arg Cys Met His Leu His Glu Lys Arg Val Val Ser Gly Ser Arg Asp
385                 390                 395                 400
Ala Thr Leu Arg Val Trp Asp Ile Glu Thr Gly Gln Cys Leu His Val
                405                 410                 415
Leu Met Gly His Val Ala Ala Val Arg Cys Val Gln Tyr Asp Gly Arg
            420                 425                 430
Arg Val Val Ser Gly Ala Tyr Asp Phe Met Val Lys Val Trp Asp Pro
        435                 440                 445
Glu Thr Glu Thr Cys Leu His Thr Leu Gln Gly His Thr Asn Arg Val
    450                 455                 460
Tyr Ser Leu Gln Phe Asp Gly Ile His Val Val Ser Gly Ser Leu Asp
465                 470                 475                 480
Thr Ser Ile Arg Val Trp Asp Val Glu Thr Gly Asn Cys Ile His Thr
                485                 490                 495
Leu Thr Gly His Gln Ser Leu Thr Ser Gly Met Glu Leu Lys Asp Asn
            500                 505                 510
Ile Leu Val Ser Gly Asn Ala Asp Ser Thr Val Lys Ile Trp Asp Ile
        515                 520                 525
Lys Thr Gly Gln Cys Leu Gln Thr Leu Gln Gly Pro Asn Lys His Gln
    530                 535                 540
Ser Ala Val Thr Cys Leu Gln Phe Asn Lys Asn Phe Val Ile Thr Ser
545                 550                 555                 560
Ser Asp Asp Gly Thr Val Lys Leu Trp Asp Leu Lys Thr Gly Glu Phe
                565                 570                 575
Ile Arg Asn Leu Val Thr Leu Glu Ser Gly Gly Ser Gly Gly Val Val
            580                 585                 590
Trp Arg Ile Arg Ala Ser Asn Thr Lys Leu Val Cys Ala Val Gly Ser
        595                 600                 605
Arg Asn Gly Thr Glu Glu Thr Lys Leu Leu Val Leu Asp Phe Asp Val
    610                 615                 620
Asp Met Lys
625
</s400> <s200> <s210>4</s210> <s211>592</s211> <s212>PRT</s212> <s213>Homo sapiens</s213> </s200> <s400> 4
Met Ser Thr Leu Glu Ser Val Thr Tyr Leu Pro Glu Lys Gly Leu Tyr
  1               5                  10                  15
Cys Gln Arg Leu Pro Ser Ser Arg Thr His Gly Gly Thr Glu Ser Leu
             20                  25                  30
Lys Gly Lys Asn Thr Glu Asn Met Gly Phe Tyr Gly Thr Leu Lys Met
         35                  40                  45
Ile Phe Tyr Lys Met Lys Arg Lys Leu Asp His Gly Ser Glu Val Arg
     50                  55                  60
Ser Phe Ser Leu Gly Lys Lys Pro Cys Lys Val Ser Glu Tyr Thr Ser
 65                  70                  75                  80
Thr Thr Gly Leu Val Pro Cys Ser Ala Thr Pro Thr Thr Phe Gly Asp
                 85                  90                  95
Leu Arg Ala Ala Asn Gly Gln Gly Gln Gln Arg Arg Arg Ile Thr Ser
            100                 105                 110
Val Gln Pro Pro Thr Gly Leu Gln Glu Trp Leu Lys Met Phe Gln Ser
        115                 120                 125
Trp Ser Gly Pro Glu Lys Leu Leu Ala Leu Asp Glu Leu Ile Asp Ser
    130                 135                 140
Cys Glu Pro Thr Gln Val Lys His Met Met Gln Val Ile Glu Pro Gln
145                 150                 155                 160
Phe Gln Arg Asp Phe Ile Ser Leu Leu Pro Lys Glu Leu Ala Leu Tyr
                165                 170                 175
Val Leu Ser Phe Leu Glu Pro Lys Asp Leu Leu Gln Ala Ala Gln Thr
            180                 185                 190
Cys Arg Tyr Trp Arg Ile Leu Ala Glu Asp Asn Leu Leu Trp Arg Glu
        195                 200                 205
Lys Cys Lys Glu Glu Gly Ile Asp Glu Pro Leu His Ile Lys Arg Arg
    210                 215                 220
Lys Val Ile Lys Pro Gly Phe Ile His Ser Pro Trp Lys Ser Ala Tyr
225                 230                 235                 240
Ile Arg Gln His Arg Ile Asp Thr Asn Trp Arg Arg Gly Glu Leu Lys
                245                 250                 255
Ser Pro Lys Val Leu Lys Gly His Asp Asp His Val Ile Thr Cys Leu
            260                 265                 270
Gln Phe Cys Gly Asn Arg Ile Val Ser Gly Ser Asp Asp Asn Thr Leu
        275                 280                 285
Lys Val Trp Ser Ala Val Thr Gly Lys Cys Leu Arg Thr Leu Val Gly
    290                 295                 300
His Thr Gly Gly Val Trp Ser Ser Gln Met Arg Asp Asn Ile Ile Ile
305                 310                 315                 320
Ser Gly Ser Thr Asp Arg Thr Leu Lys Val Trp Asn Ala Glu Thr Gly
                325                 330                 335
Glu Cys Ile His Thr Leu Tyr Gly His Thr Ser Thr Val Arg Cys Met
            340                 345                 350
His Leu His Glu Lys Arg Val Val Ser Gly Ser Arg Asp Ala Thr Leu
        355                 360                 365
Arg Val Trp Asp Ile Glu Thr Gly Gln Cys Leu His Val Leu Met Gly
    370                 375                 380
His Val Ala Ala Val Arg Cys Val Gln Tyr Asp Gly Arg Arg Val Val
385                 390                 395                 400
Ser Gly Ala Tyr Asp Phe Met Val Lys Val Trp Asp Pro Glu Thr Glu
                405                 410                 415
Thr Cys Leu His Thr Leu Gln Gly His Thr Asn Arg Val Tyr Ser Leu
            420                 425                 430
Gln Phe Asp Gly Ile His Val Val Ser Gly Ser Leu Asp Thr Ser Ile
        435                 440                 445
Arg Val Trp Asp Val Glu Thr Gly Asn Cys Ile His Thr Leu Thr Gly
    450                 455                 460
His Gln Ser Leu Thr Ser Gly Met Glu Leu Lys Asp Asn Ile Leu Val
465                 470                 475                 480
Ser Gly Asn Ala Asp Ser Thr Val Lys Ile Trp Asp Ile Lys Thr Gly
                485                 490                 495
Gln Cys Leu Gln Thr Leu Gln Gly Pro Asn Lys His Gln Ser Ala Val
            500                 505                 510
Thr Cys Leu Gln Phe Asn Lys Asn Phe Val Ile Thr Ser Ser Asp Asp
        515                 520                 525
Gly Thr Val Lys Leu Trp Asp Leu Lys Thr Gly Glu Phe Ile Arg Asn
    530                 535                 540
Leu Val Thr Leu Glu Ser Gly Gly Ser Gly Gly Val Val Trp Arg Ile
545                 550                 555                 560
Arg Ala Ser Asn Thr Lys Leu Val Cys Ala Val Gly Ser Arg Asn Gly
                565                 570                 575
Thr Glu Glu Thr Lys Leu Leu Val Leu Asp Phe Asp Val Asp Met Lys
            580                 585                 590
</s400> <s200> <s210>5</s210> <s211>553</s211> <s212>PRT</s212> <s213>Homo sapiens</s213> </s200> <s400> 5
Met Gly Phe Tyr Gly Thr Leu Lys Met Ile Phe Tyr Lys Met Lys Arg
  1               5                  10                  15
Lys Leu Asp His Gly Ser Glu Val Arg Ser Phe Ser Leu Gly Lys Lys
             20                  25                  30
Pro Cys Lys Val Ser Glu Tyr Thr Ser Thr Thr Gly Leu Val Pro Cys
         35                  40                  45
Ser Ala Thr Pro Thr Thr Phe Gly Asp Leu Arg Ala Ala Asn Gly Gln
     50                  55                  60
Gly Gln Gln Arg Arg Arg Ile Thr Ser Val Gln Pro Pro Thr Gly Leu
 65                  70                  75                  80
Gln Glu Trp Leu Lys Met Phe Gln Ser Trp Ser Gly Pro Glu Lys Leu
                 85                  90                  95
Leu Ala Leu Asp Glu Leu Ile Asp Ser Cys Glu Pro Thr Gln Val Lys
            100                 105                 110
His Met Met Gln Val Ile Glu Pro Gln Phe Gln Arg Asp Phe Ile Ser
        115                 120                 125
Leu Leu Pro Lys Glu Leu Ala Leu Tyr Val Leu Ser Phe Leu Glu Pro
    130                 135                 140
Lys Asp Leu Leu Gln Ala Ala Gln Thr Cys Arg Tyr Trp Arg Ile Leu
145                 150                 155                 160
Ala Glu Asp Asn Leu Leu Trp Arg Glu Lys Cys Lys Glu Glu Gly Ile
                165                 170                 175
Asp Glu Pro Leu His Ile Lys Arg Arg Lys Val Ile Lys Pro Gly Phe
            180                 185                 190
Ile His Ser Pro Trp Lys Ser Ala Tyr Ile Arg Gln His Arg Ile Asp
        195                 200                 205
Thr Asn Trp Arg Arg Gly Glu Leu Lys Ser Pro Lys Val Leu Lys Gly
    210                 215                 220
His Asp Asp His Val Ile Thr Cys Leu Gln Phe Cys Gly Asn Arg Ile
225                 230                 235                 240
Val Ser Gly Ser Asp Asp Asn Thr Leu Lys Val Trp Ser Ala Val Thr
                245                 250                 255
Gly Lys Cys Leu Arg Thr Leu Val Gly His Thr Gly Gly Val Trp Ser
            260                 265                 270
Ser Gln Met Arg Asp Asn Ile Ile Ile Ser Gly Ser Thr Asp Arg Thr
        275                 280                 285
Leu Lys Val Trp Asn Ala Glu Thr Gly Glu Cys Ile His Thr Leu Tyr
    290                 295                 300
Gly His Thr Ser Thr Val Arg Cys Met His Leu His Glu Lys Arg Val
305                 310                 315                 320
Val Ser Gly Ser Arg Asp Ala Thr Leu Arg Val Trp Asp Ile Glu Thr
                325                 330                 335
Gly Gln Cys Leu His Val Leu Met Gly His Val Ala Ala Val Arg Cys
            340                 345                 350
Val Gln Tyr Asp Gly Arg Arg Val Val Ser Gly Ala Tyr Asp Phe Met
        355                 360                 365
Val Lys Val Trp Asp Pro Glu Thr Glu Thr Cys Leu His Thr Leu Gln
    370                 375                 380
Gly His Thr Asn Arg Val Tyr Ser Leu Gln Phe Asp Gly Ile His Val
385                 390                 395                 400
Val Ser Gly Ser Leu Asp Thr Ser Ile Arg Val Trp Asp Val Glu Thr
                405                 410                 415
Gly Asn Cys Ile His Thr Leu Thr Gly His Gln Ser Leu Thr Ser Gly
            420                 425                 430
Met Glu Leu Lys Asp Asn Ile Leu Val Ser Gly Asn Ala Asp Ser Thr
        435                 440                 445
Val Lys Ile Trp Asp Ile Lys Thr Gly Gln Cys Leu Gln Thr Leu Gln
    450                 455                 460
Gly Pro Asn Lys His Gln Ser Ala Val Thr Cys Leu Gln Phe Asn Lys
465                 470                 475                 480
Asn Phe Val Ile Thr Ser Ser Asp Asp Gly Thr Val Lys Leu Trp Asp
                485                 490                 495
Leu Lys Thr Gly Glu Phe Ile Arg Asn Leu Val Thr Leu Glu Ser Gly
            500                 505                 510
Gly Ser Gly Gly Val Val Trp Arg Ile Arg Ala Ser Asn Thr Lys Leu
        515                 520                 525
Val Cys Ala Val Gly Ser Arg Asn Gly Thr Glu Glu Thr Lys Leu Leu
    530                 535                 540
Val Leu Asp Phe Asp Val Asp Met Lys
545                 550
</s400> <s200> <s210>6</s210> <s211>545</s211> <s212>PRT</s212> <s213>Homo sapiens</s213> </s200> <s400> 6
Met Ile Phe Tyr Lys Met Lys Arg Lys Leu Asp His Gly Ser Glu Val
  1               5                  10                  15
Arg Ser Phe Ser Leu Gly Lys Lys Pro Cys Lys Val Ser Glu Tyr Thr
             20                  25                  30
Ser Thr Thr Gly Leu Val Pro Cys Ser Ala Thr Pro Thr Thr Phe Gly
         35                  40                  45
Asp Leu Arg Ala Ala Asn Gly Gln Gly Gln Gln Arg Arg Arg Ile Thr
     50                  55                  60
Ser Val Gln Pro Pro Thr Gly Leu Gln Glu Trp Leu Lys Met Phe Gln
 65                  70                  75                  80
Ser Trp Ser Gly Pro Glu Lys Leu Leu Ala Leu Asp Glu Leu Ile Asp
                 85                  90                  95
Ser Cys Glu Pro Thr Gln Val Lys His Met Met Gln Val Ile Glu Pro
            100                 105                 110
Gln Phe Gln Arg Asp Phe Ile Ser Leu Leu Pro Lys Glu Leu Ala Leu
        115                 120                 125
Tyr Val Leu Ser Phe Leu Glu Pro Lys Asp Leu Leu Gln Ala Ala Gln
    130                 135                 140
Thr Cys Arg Tyr Trp Arg Ile Leu Ala Glu Asp Asn Leu Leu Trp Arg
145                 150                 155                 160
Glu Lys Cys Lys Glu Glu Gly Ile Asp Glu Pro Leu His Ile Lys Arg
                165                 170                 175
Arg Lys Val Ile Lys Pro Gly Phe Ile His Ser Pro Trp Lys Ser Ala
            180                 185                 190
Tyr Ile Arg Gln His Arg Ile Asp Thr Asn Trp Arg Arg Gly Glu Leu
        195                 200                 205
Lys Ser Pro Lys Val Leu Lys Gly His Asp Asp His Val Ile Thr Cys
    210                 215                 220
Leu Gln Phe Cys Gly Asn Arg Ile Val Ser Gly Ser Asp Asp Asn Thr
225                 230                 235                 240
Leu Lys Val Trp Ser Ala Val Thr Gly Lys Cys Leu Arg Thr Leu Val
                245                 250                 255
Gly His Thr Gly Gly Val Trp Ser Ser Gln Met Arg Asp Asn Ile Ile
            260                 265                 270
Ile Ser Gly Ser Thr Asp Arg Thr Leu Lys Val Trp Asn Ala Glu Thr
        275                 280                 285
Gly Glu Cys Ile His Thr Leu Tyr Gly His Thr Ser Thr Val Arg Cys
    290                 295                 300
Met His Leu His Glu Lys Arg Val Val Ser Gly Ser Arg Asp Ala Thr
305                 310                 315                 320
Leu Arg Val Trp Asp Ile Glu Thr Gly Gln Cys Leu His Val Leu Met
                325                 330                 335
Gly His Val Ala Ala Val Arg Cys Val Gln Tyr Asp Gly Arg Arg Val
            340                 345                 350
Val Ser Gly Ala Tyr Asp Phe Met Val Lys Val Trp Asp Pro Glu Thr
        355                 360                 365
Glu Thr Cys Leu His Thr Leu Gln Gly His Thr Asn Arg Val Tyr Ser
    370                 375                 380
Leu Gln Phe Asp Gly Ile His Val Val Ser Gly Ser Leu Asp Thr Ser
385                 390                 395                 400
Ile Arg Val Trp Asp Val Glu Thr Gly Asn Cys Ile His Thr Leu Thr
                405                 410                 415
Gly His Gln Ser Leu Thr Ser Gly Met Glu Leu Lys Asp Asn Ile Leu
            420                 425                 430
Val Ser Gly Asn Ala Asp Ser Thr Val Lys Ile Trp Asp Ile Lys Thr
        435                 440                 445
Gly Gln Cys Leu Gln Thr Leu Gln Gly Pro Asn Lys His Gln Ser Ala
    450                 455                 460
Val Thr Cys Leu Gln Phe Asn Lys Asn Phe Val Ile Thr Ser Ser Asp
465                 470                 475                 480
Asp Gly Thr Val Lys Leu Trp Asp Leu Lys Thr Gly Glu Phe Ile Arg
                485                 490                 495
Asn Leu Val Thr Leu Glu Ser Gly Gly Ser Gly Gly Val Val Trp Arg
            500                 505                 510
Ile Arg Ala Ser Asn Thr Lys Leu Val Cys Ala Val Gly Ser Arg Asn
        515                 520                 525
Gly Thr Glu Glu Thr Lys Leu Leu Val Leu Asp Phe Asp Val Asp Met
    530                 535                 540
Lys
545
</s400> <s200> <s210>7</s210> <s211>540</s211> <s212>PRT</s212> <s213>Homo sapiens</s213> </s200> <s400> 7
Met Lys Arg Lys Leu Asp His Gly Ser Glu Val Arg Ser Phe Ser Leu
  1               5                  10                  15
Gly Lys Lys Pro Cys Lys Val Ser Glu Tyr Thr Ser Thr Thr Gly Leu
             20                  25                  30
Val Pro Cys Ser Ala Thr Pro Thr Thr Phe Gly Asp Leu Arg Ala Ala
         35                  40                  45
Asn Gly Gln Gly Gln Gln Arg Arg Arg Ile Thr Ser Val Gln Pro Pro
     50                  55                  60
Thr Gly Leu Gln Glu Trp Leu Lys Met Phe Gln Ser Trp Ser Gly Pro
 65                  70                  75                  80
Glu Lys Leu Leu Ala Leu Asp Glu Leu Ile Asp Ser Cys Glu Pro Thr
                 85                  90                  95
Gln Val Lys His Met Met Gln Val Ile Glu Pro Gln Phe Gln Arg Asp
            100                 105                 110
Phe Ile Ser Leu Leu Pro Lys Glu Leu Ala Leu Tyr Val Leu Ser Phe
        115                 120                 125
Leu Glu Pro Lys Asp Leu Leu Gln Ala Ala Gln Thr Cys Arg Tyr Trp
    130                 135                 140
Arg Ile Leu Ala Glu Asp Asn Leu Leu Trp Arg Glu Lys Cys Lys Glu
145                 150                 155                 160
Glu Gly Ile Asp Glu Pro Leu His Ile Lys Arg Arg Lys Val Ile Lys
                165                 170                 175
Pro Gly Phe Ile His Ser Pro Trp Lys Ser Ala Tyr Ile Arg Gln His
            180                 185                 190
Arg Ile Asp Thr Asn Trp Arg Arg Gly Glu Leu Lys Ser Pro Lys Val
        195                 200                 205
Leu Lys Gly His Asp Asp His Val Ile Thr Cys Leu Gln Phe Cys Gly
    210                 215                 220
Asn Arg Ile Val Ser Gly Ser Asp Asp Asn Thr Leu Lys Val Trp Ser
225                 230                 235                 240
Ala Val Thr Gly Lys Cys Leu Arg Thr Leu Val Gly His Thr Gly Gly
                245                 250                 255
Val Trp Ser Ser Gln Met Arg Asp Asn Ile Ile Ile Ser Gly Ser Thr
            260                 265                 270
Asp Arg Thr Leu Lys Val Trp Asn Ala Glu Thr Gly Glu Cys Ile His
        275                 280                 285
Thr Leu Tyr Gly His Thr Ser Thr Val Arg Cys Met His Leu His Glu
    290                 295                 300
Lys Arg Val Val Ser Gly Ser Arg Asp Ala Thr Leu Arg Val Trp Asp
305                 310                 315                 320
Ile Glu Thr Gly Gln Cys Leu His Val Leu Met Gly His Val Ala Ala
                325                 330                 335
Val Arg Cys Val Gln Tyr Asp Gly Arg Arg Val Val Ser Gly Ala Tyr
            340                 345                 350
Asp Phe Met Val Lys Val Trp Asp Pro Glu Thr Glu Thr Cys Leu His
        355                 360                 365
Thr Leu Gln Gly His Thr Asn Arg Val Tyr Ser Leu Gln Phe Asp Gly
    370                 375                 380
Ile His Val Val Ser Gly Ser Leu Asp Thr Ser Ile Arg Val Trp Asp
385                 390                 395                 400
Val Glu Thr Gly Asn Cys Ile His Thr Leu Thr Gly His Gln Ser Leu
                405                 410                 415
Thr Ser Gly Met Glu Leu Lys Asp Asn Ile Leu Val Ser Gly Asn Ala
            420                 425                 430
Asp Ser Thr Val Lys Ile Trp Asp Ile Lys Thr Gly Gln Cys Leu Gln
        435                 440                 445
Thr Leu Gln Gly Pro Asn Lys His Gln Ser Ala Val Thr Cys Leu Gln
    450                 455                 460
Phe Asn Lys Asn Phe Val Ile Thr Ser Ser Asp Asp Gly Thr Val Lys
465                 470                 475                 480
Leu Trp Asp Leu Lys Thr Gly Glu Phe Ile Arg Asn Leu Val Thr Leu
                485                 490                 495
Glu Ser Gly Gly Ser Gly Gly Val Val Trp Arg Ile Arg Ala Ser Asn
            500                 505                 510
Thr Lys Leu Val Cys Ala Val Gly Ser Arg Asn Gly Thr Glu Glu Thr
        515                 520                 525
Lys Leu Leu Val Leu Asp Phe Asp Val Asp Met Lys
    530                 535                 540
</s400> <s200> <s210>8</s210> <s211>589</s211> <s212>PRT</s212> <s213>Homo sapiens</s213> </s200> <s400> 8
Met Ser Lys Pro Gly Lys Pro Thr Leu Asn His Gly Leu Val Pro Val
  1               5                  10                  15
Asp Leu Lys Ser Ala Lys Glu Pro Leu Pro His Gln Thr Val Met Lys
             20                  25                  30
Ile Phe Ser Ile Ser Ile Ile Ala Gln Gly Leu Pro Phe Cys Arg Arg
         35                  40                  45
Arg Met Lys Arg Lys Leu Asp His Gly Ser Glu Val Arg Ser Phe Ser
     50                  55                  60
Leu Gly Lys Lys Pro Cys Lys Val Ser Glu Tyr Thr Ser Thr Thr Gly
 65                  70                  75                  80
Leu Val Pro Cys Ser Ala Thr Pro Thr Thr Phe Gly Asp Leu Arg Ala
                 85                  90                  95
Ala Asn Gly Gln Gly Gln Gln Arg Arg Arg Ile Thr Ser Val Gln Pro
            100                 105                 110
Pro Thr Gly Leu Gln Glu Trp Leu Lys Met Phe Gln Ser Trp Ser Gly
        115                 120                 125
Pro Glu Lys Leu Leu Ala Leu Asp Glu Leu Ile Asp Ser Cys Glu Pro
    130                 135                 140
Thr Gln Val Lys His Met Met Gln Val Ile Glu Pro Gln Phe Gln Arg
145                 150                 155                 160
Asp Phe Ile Ser Leu Leu Pro Lys Glu Leu Ala Leu Tyr Val Leu Ser
                165                 170                 175
Phe Leu Glu Pro Lys Asp Leu Leu Gln Ala Ala Gln Thr Cys Arg Tyr
            180                 185                 190
Trp Arg Ile Leu Ala Glu Asp Asn Leu Leu Trp Arg Glu Lys Cys Lys
        195                 200                 205
Glu Glu Gly Ile Asp Glu Pro Leu His Ile Lys Arg Arg Lys Val Ile
    210                 215                 220
Lys Pro Gly Phe Ile His Ser Pro Trp Lys Ser Ala Tyr Ile Arg Gln
225                 230                 235                 240
His Arg Ile Asp Thr Asn Trp Arg Arg Gly Glu Leu Lys Ser Pro Lys
                245                 250                 255
Val Leu Lys Gly His Asp Asp His Val Ile Thr Cys Leu Gln Phe Cys
            260                 265                 270
Gly Asn Arg Ile Val Ser Gly Ser Asp Asp Asn Thr Leu Lys Val Trp
        275                 280                 285
Ser Ala Val Thr Gly Lys Cys Leu Arg Thr Leu Val Gly His Thr Gly
    290                 295                 300
Gly Val Trp Ser Ser Gln Met Arg Asp Asn Ile Ile Ile Ser Gly Ser
305                 310                 315                 320
Thr Asp Arg Thr Leu Lys Val Trp Asn Ala Glu Thr Gly Glu Cys Ile
                325                 330                 335
His Thr Leu Tyr Gly His Thr Ser Thr Val Arg Cys Met His Leu His
            340                 345                 350
Glu Lys Arg Val Val Ser Gly Ser Arg Asp Ala Thr Leu Arg Val Trp
        355                 360                 365
Asp Ile Glu Thr Gly Gln Cys Leu His Val Leu Met Gly His Val Ala
    370                 375                 380
Ala Val Arg Cys Val Gln Tyr Asp Gly Arg Arg Val Val Ser Gly Ala
385                 390                 395                 400
Tyr Asp Phe Met Val Lys Val Trp Asp Pro Glu Thr Glu Thr Cys Leu
                405                 410                 415
His Thr Leu Gln Gly His Thr Asn Arg Val Tyr Ser Leu Gln Phe Asp
            420                 425                 430
Gly Ile His Val Val Ser Gly Ser Leu Asp Thr Ser Ile Arg Val Trp
        435                 440                 445
Asp Val Glu Thr Gly Asn Cys Ile His Thr Leu Thr Gly His Gln Ser
    450                 455                 460
Leu Thr Ser Gly Met Glu Leu Lys Asp Asn Ile Leu Val Ser Gly Asn
465                 470                 475                 480
Ala Asp Ser Thr Val Lys Ile Trp Asp Ile Lys Thr Gly Gln Cys Leu
                485                 490                 495
Gln Thr Leu Gln Gly Pro Asn Lys His Gln Ser Ala Val Thr Cys Leu
            500                 505                 510
Gln Phe Asn Lys Asn Phe Val Ile Thr Ser Ser Asp Asp Gly Thr Val
        515                 520                 525
Lys Leu Trp Asp Leu Lys Thr Gly Glu Phe Ile Arg Asn Leu Val Thr
    530                 535                 540
Leu Glu Ser Gly Gly Ser Gly Gly Val Val Trp Arg Ile Arg Ala Ser
545                 550                 555                 560
Asn Thr Lys Leu Val Cys Ala Val Gly Ser Arg Asn Gly Thr Glu Glu
                565                 570                 575
Thr Lys Leu Leu Val Leu Asp Phe Asp Val Asp Met Lys
            580                 585
</s400> <s200> <s210>9</s210> <s211>559</s211> <s212>PRT</s212> <s213>Homo sapiens</s213> </s200> <s400> 9
Met Lys Ile Phe Ser Ile Ser Ile Ile Ala Gln Gly Leu Pro Phe Cys
  1               5                  10                  15
Arg Arg Arg Met Lys Arg Lys Leu Asp His Gly Ser Glu Val Arg Ser
             20                  25                  30
Phe Ser Leu Gly Lys Lys Pro Cys Lys Val Ser Glu Tyr Thr Ser Thr
         35                  40                  45
Thr Gly Leu Val Pro Cys Ser Ala Thr Pro Thr Thr Phe Gly Asp Leu
     50                  55                  60
Arg Ala Ala Asn Gly Gln Gly Gln Gln Arg Arg Arg Ile Thr Ser Val
 65                  70                  75                  80
Gln Pro Pro Thr Gly Leu Gln Glu Trp Leu Lys Met Phe Gln Ser Trp
                 85                  90                  95
Ser Gly Pro Glu Lys Leu Leu Ala Leu Asp Glu Leu Ile Asp Ser Cys
            100                 105                 110
Glu Pro Thr Gln Val Lys His Met Met Gln Val Ile Glu Pro Gln Phe
        115                 120                 125
Gln Arg Asp Phe Ile Ser Leu Leu Pro Lys Glu Leu Ala Leu Tyr Val
    130                 135                 140
Leu Ser Phe Leu Glu Pro Lys Asp Leu Leu Gln Ala Ala Gln Thr Cys
145                 150                 155                 160
Arg Tyr Trp Arg Ile Leu Ala Glu Asp Asn Leu Leu Trp Arg Glu Lys
                165                 170                 175
Cys Lys Glu Glu Gly Ile Asp Glu Pro Leu His Ile Lys Arg Arg Lys
            180                 185                 190
Val Ile Lys Pro Gly Phe Ile His Ser Pro Trp Lys Ser Ala Tyr Ile
        195                 200                 205
Arg Gln His Arg Ile Asp Thr Asn Trp Arg Arg Gly Glu Leu Lys Ser
    210                 215                 220
Pro Lys Val Leu Lys Gly His Asp Asp His Val Ile Thr Cys Leu Gln
225                 230                 235                 240
Phe Cys Gly Asn Arg Ile Val Ser Gly Ser Asp Asp Asn Thr Leu Lys
                245                 250                 255
Val Trp Ser Ala Val Thr Gly Lys Cys Leu Arg Thr Leu Val Gly His
            260                 265                 270
Thr Gly Gly Val Trp Ser Ser Gln Met Arg Asp Asn Ile Ile Ile Ser
        275                 280                 285
Gly Ser Thr Asp Arg Thr Leu Lys Val Trp Asn Ala Glu Thr Gly Glu
    290                 295                 300
Cys Ile His Thr Leu Tyr Gly His Thr Ser Thr Val Arg Cys Met His
305                 310                 315                 320
Leu His Glu Lys Arg Val Val Ser Gly Ser Arg Asp Ala Thr Leu Arg
                325                 330                 335
Val Trp Asp Ile Glu Thr Gly Gln Cys Leu His Val Leu Met Gly His
            340                 345                 350
Val Ala Ala Val Arg Cys Val Gln Tyr Asp Gly Arg Arg Val Val Ser
        355                 360                 365
Gly Ala Tyr Asp Phe Met Val Lys Val Trp Asp Pro Glu Thr Glu Thr
    370                 375                 380
Cys Leu His Thr Leu Gln Gly His Thr Asn Arg Val Tyr Ser Leu Gln
385                 390                 395                 400
Phe Asp Gly Ile His Val Val Ser Gly Ser Leu Asp Thr Ser Ile Arg
                405                 410                 415
Val Trp Asp Val Glu Thr Gly Asn Cys Ile His Thr Leu Thr Gly His
            420                 425                 430
Gln Ser Leu Thr Ser Gly Met Glu Leu Lys Asp Asn Ile Leu Val Ser
        435                 440                 445
Gly Asn Ala Asp Ser Thr Val Lys Ile Trp Asp Ile Lys Thr Gly Gln
    450                 455                 460
Cys Leu Gln Thr Leu Gln Gly Pro Asn Lys His Gln Ser Ala Val Thr
465                 470                 475                 480
Cys Leu Gln Phe Asn Lys Asn Phe Val Ile Thr Ser Ser Asp Asp Gly
                485                 490                 495
Thr Val Lys Leu Trp Asp Leu Lys Thr Gly Glu Phe Ile Arg Asn Leu
            500                 505                 510
Val Thr Leu Glu Ser Gly Gly Ser Gly Gly Val Val Trp Arg Ile Arg
        515                 520                 525
Ala Ser Asn Thr Lys Leu Val Cys Ala Val Gly Ser Arg Asn Gly Thr
    530                 535                 540
Glu Glu Thr Lys Leu Leu Val Leu Asp Phe Asp Val Asp Met Lys
545                 550                 555
</s400> <s200> <s210>10</s210> <s211>540</s211> <s212>PRT</s212> <s213>Homo sapiens</s213> </s200> <s400> 10
Met Lys Arg Lys Leu Asp His Gly Ser Glu Val Arg Ser Phe Ser Leu
  1               5                  10                  15
Gly Lys Lys Pro Cys Lys Val Ser Glu Tyr Thr Ser Thr Thr Gly Leu
             20                  25                  30
Val Pro Cys Ser Ala Thr Pro Thr Thr Phe Gly Asp Leu Arg Ala Ala
         35                  40                  45
Asn Gly Gln Gly Gln Gln Arg Arg Arg Ile Thr Ser Val Gln Pro Pro
     50                  55                  60
Thr Gly Leu Gln Glu Trp Leu Lys Met Phe Gln Ser Trp Ser Gly Pro
 65                  70                  75                  80
Glu Lys Leu Leu Ala Leu Asp Glu Leu Ile Asp Ser Cys Glu Pro Thr
                 85                  90                  95
Gln Val Lys His Met Met Gln Val Ile Glu Pro Gln Phe Gln Arg Asp
            100                 105                 110
Phe Ile Ser Leu Leu Pro Lys Glu Leu Ala Leu Tyr Val Leu Ser Phe
        115                 120                 125
Leu Glu Pro Lys Asp Leu Leu Gln Ala Ala Gln Thr Cys Arg Tyr Trp
    130                 135                 140
Arg Ile Leu Ala Glu Asp Asn Leu Leu Trp Arg Glu Lys Cys Lys Glu
145                 150                 155                 160
Glu Gly Ile Asp Glu Pro Leu His Ile Lys Arg Arg Lys Val Ile Lys
                165                 170                 175
Pro Gly Phe Ile His Ser Pro Trp Lys Ser Ala Tyr Ile Arg Gln His
            180                 185                 190
Arg Ile Asp Thr Asn Trp Arg Arg Gly Glu Leu Lys Ser Pro Lys Val
        195                 200                 205
Leu Lys Gly His Asp Asp His Val Ile Thr Cys Leu Gln Phe Cys Gly
    210                 215                 220
Asn Arg Ile Val Ser Gly Ser Asp Asp Asn Thr Leu Lys Val Trp Ser
225                 230                 235                 240
Ala Val Thr Gly Lys Cys Leu Arg Thr Leu Val Gly His Thr Gly Gly
                245                 250                 255
Val Trp Ser Ser Gln Met Arg Asp Asn Ile Ile Ile Ser Gly Ser Thr
            260                 265                 270
Asp Arg Thr Leu Lys Val Trp Asn Ala Glu Thr Gly Glu Cys Ile His
        275                 280                 285
Thr Leu Tyr Gly His Thr Ser Thr Val Arg Cys Met His Leu His Glu
    290                 295                 300
Lys Arg Val Val Ser Gly Ser Arg Asp Ala Thr Leu Arg Val Trp Asp
305                 310                 315                 320
Ile Glu Thr Gly Gln Cys Leu His Val Leu Met Gly His Val Ala Ala
                325                 330                 335
Val Arg Cys Val Gln Tyr Asp Gly Arg Arg Val Val Ser Gly Ala Tyr
            340                 345                 350
Asp Phe Met Val Lys Val Trp Asp Pro Glu Thr Glu Thr Cys Leu His
        355                 360                 365
Thr Leu Gln Gly His Thr Asn Arg Val Tyr Ser Leu Gln Phe Asp Gly
    370                 375                 380
Ile His Val Val Ser Gly Ser Leu Asp Thr Ser Ile Arg Val Trp Asp
385                 390                 395                 400
Val Glu Thr Gly Asn Cys Ile His Thr Leu Thr Gly His Gln Ser Leu
                405                 410                 415
Thr Ser Gly Met Glu Leu Lys Asp Asn Ile Leu Val Ser Gly Asn Ala
            420                 425                 430
Asp Ser Thr Val Lys Ile Trp Asp Ile Lys Thr Gly Gln Cys Leu Gln
        435                 440                 445
Thr Leu Gln Gly Pro Asn Lys His Gln Ser Ala Val Thr Cys Leu Gln
    450                 455                 460
Phe Asn Lys Asn Phe Val Ile Thr Ser Ser Asp Asp Gly Thr Val Lys
465                 470                 475                 480
Leu Trp Asp Leu Lys Thr Gly Glu Phe Ile Arg Asn Leu Val Thr Leu
                485                 490                 495
Glu Ser Gly Gly Ser Gly Gly Val Val Trp Arg Ile Arg Ala Ser Asn
            500                 505                 510
Thr Lys Leu Val Cys Ala Val Gly Ser Arg Asn Gly Thr Glu Glu Thr
        515                 520                 525
Lys Leu Leu Val Leu Asp Phe Asp Val Asp Met Lys
    530                 535                 540
</s400> <s200> <s210>11</s210> <s211>34</s211> <s212>DNA</s212> <s213>Artificial Sequence</s213> <s220> <s223>Description of Artificial Sequence
      Oligonucleotide primer</s223> </s220> </s200> <s400> 11
cgggatccac catggatgat ggatcgatga cacc                                 34
</s400> <s200> <s210>12</s210> <s211>33</s211> <s212>DNA</s212> <s213>Artificial Sequence</s213> <s220> <s223>Description of Artificial Sequence
      Oligonucleotide primer</s223> </s220> </s200> <s400> 12
ggaattcctt aagggtatac agcatcaaag tcg                                  33
</s400> <s200> <s210>13</s210> <s211>25</s211> <s212>DNA</s212> <s213>Artificial Sequence</s213> <s220> <s223>Description of Artificial Sequence
      Oligonucleotide primer</s223> </s220> </s200> <s400> 13
tcacttcatg tccacatcaa agtcc                                           25
</s400> <s200> <s210>14</s210> <s211>26</s211> <s212>DNA</s212> <s213>Artificial Sequence</s213> <s220> <s223>Description of Artificial Sequence
      Oligonucleotide primer</s223> </s220> </s200> <s400> 14
ggtaattaca agttcttgtt gaactg                                          26
</s400> <s200> <s210>15</s210> <s211>22</s211> <s212>DNA</s212> <s213>Artificial Sequence</s213> <s220> <s223>Description of Artificial Sequence
      Oligonucleotide primer</s223> </s220> </s200> <s400> 15
ccctgcaacg tgtgtagaca gg                                              22
</s400> <s200> <s210>16</s210> <s211>24</s211> <s212>DNA</s212> <s213>Artificial Sequence</s213> <s220> <s223>Description of Artificial Sequence
      Oligonucleotide primer</s223> </s220> </s200> <s400> 16
ccagtctctg cattccacac tttg                                            24
</s400> <s200> <s210>17</s210> <s211>23</s211> <s212>DNA</s212> <s213>Artificial Sequence</s213> <s220> <s223>Description of Artificial Sequence
      Oligonucleotide primer</s223> </s220> </s200> <s400> 17
ctcagacagg tcaggacatt tgg                                             23
</s400> <s200> <s210>18</s210> <s211>33</s211> <s212>DNA</s212> <s213>Artificial Sequence</s213> <s220> <s223>Description of Artificial Sequence
      Oligonucleotide primer</s223> </s220> </s200> <s400> 18
ggaattccat gaaaagattg gaccatggtt ctg                                  33
</s400> <s200> <s210>19</s210> <s211>34</s211> <s212>DNA</s212> <s213>Artificial Sequence</s213> <s220> <s223>Description of Artificial Sequence
      Oligonucleotide primer</s223> </s220> </s200> <s400> 19
ggaattcctc acttcatgtc acatcaaagt ccag                                 34
</s400> <s200> <s210>20</s210> <s211>1881</s211> <s212>DNA</s212> <s213>Artificial Sequence</s213> <s220> <s223>Description of Artificial Sequence  6 myc
      tagged homo sapiens</s223> </s220> </s200> <s400> 20
atggagcaaa agctcatttc tgaagaggac ttgaatgaaa tggagcaaaa gctcatttct     60
gaagaggact tgaatgaaat ggagcaaaag ctcatttctg aagaggactt gaatgaaatg    120
gagcaaaagc tcatttctga agaggacttg aatgaaatgg agcaaaagct catttctgaa    180
gaggacttga atgaaatgga gagcttgggc gacctcacca tggagcaaaa gctcatttct    240
gaagaggact tgaattccat gaaaagaaag ttggaccatg gttctgaggt ccgctctttt    300
tctttgggaa agaaaccatg caaagtctca gaatatacaa gtaccactgg gcttgtacca    360
tgttcagcaa caccaacaac ttttggggac ctcagagcag ccaatggcca agggcaacaa    420
cgacgccgaa ttacatctgt ccagccacct acaggcctcc aggaatggct aaaaatgttt    480
cagagctgga gtggaccaga gaaattgctt gctttagatg aactcattga tagttgtgaa    540
ccaacacaag taaaacatat gatgcaagtg atagaacccc agtttcaacg agacttcatt    600
tcattgctcc ctaaagagtt ggcactctat gtgctttcat tcctggaacc caaagacctg    660
ctacaagcag ctcagacatg tcgctactgg agaattttgg ctgaagacaa ccttctctgg    720
agagagaaat gcaaagaaga ggggattgat gaaccattgc acatcaagag aagaaaagta    780
ataaaaccag gtttcataca cagtccatgg aaaagtgcat acatcagaca gcacagaatt    840
gatactaact ggaggcgagg agaactcaaa tctcctaagg tgctgaaagg acatgatgat    900
catgtgatca catgcttaca gttttgtggt aaccgaatag ttagtggttc tgatgacaac    960
actttaaaag tttggtcagc agtcacaggc aaatgtctga gaacattagt gggacataca   1020
ggtggagtat ggtcatcaca aatgagggac aacatcatca ttagtggatc tacagatcgg   1080
acactcaaag tgtggaatgc agagactgga gaatgtatac acaccttata tgggcatact   1140
tccactgtgc gttgtatgca tcttcatgaa aaaagagttg ttagcggttc tcgagatgcc   1200
actcttaggg tttgggatat tgagacaggc cagtgtttac atgttttgat gggtcatgtt   1260
gcagcagtcc gctgtgttca atatgatggc aggagggttg ttagtggagc atatgatttt   1320
atggtaaagg tgtgggatcc agagactgaa acctgtctac acacgttgca ggggcatact   1380
aatagagtct attcattaca gtttgatggt atccatgtgg tgagtggatc tcttgataca   1440
tccatccgtg tttgggatgt ggagacaggg aattgcattc acacgttaac agggcaccag   1500
tcgttaacaa gtggaatgga actcaaagac aatattcttg tctctgggaa tgcagattct   1560
acagttaaaa tctgggatat caaaacagga cagtgtttac aaacattgca aggtcccaac   1620
aagcatcaga gtgctgtgac ctgtttacag ttcaacaaga actttgtaat taccagctca   1680
gatgatggaa ctgtaaaact atgggacttg aaaacgggtg aatttattcg aaacctagtc   1740
acattggaga gtggggggag tgggggagtt gtgtggcgga tcagagcctc aaacacaaag   1800
ctggtgtgtg cagttgggag tcggaatggg actgaagaaa ccaagctgct ggtgctggac   1860
tttgatgtgg acatgaagtg a                                             1881
</s400> <s200> <s210>21</s210> <s211>626</s211> <s212>PRT</s212> <s213>Artificial Sequence</s213> <s220> <s223>Description of Artificial Sequence  6 myc
      tagged homo sapien</s223> </s220> </s200> <s400> 21
Met Glu Gln Lys Leu Ile Ser Glu Glu Asp Leu Asn Glu Met Glu Gln
  1               5                  10                  15
Lys Leu Ile Ser Glu Glu Asp Leu Asn Glu Met Glu Gln Lys Leu Ile
             20                  25                  30
Ser Glu Glu Asp Leu Asn Glu Met Glu Gln Lys Leu Ile Ser Glu Glu
         35                  40                  45
Asp Leu Asn Glu Met Glu Gln Lys Leu Ile Ser Glu Glu Asp Leu Asn
     50                  55                  60
Glu Met Glu Ser Leu Gly Asp Leu Thr Met Glu Gln Lys Leu Ile Ser
 65                  70                  75                  80
Glu Glu Asp Leu Asn Ser Met Lys Arg Lys Leu Asp His Gly Ser Glu
                 85                  90                  95
Val Arg Ser Phe Ser Leu Gly Lys Lys Pro Cys Lys Val Ser Glu Tyr
            100                 105                 110
Thr Ser Thr Thr Gly Leu Val Pro Cys Ser Ala Thr Pro Thr Thr Phe
        115                 120                 125
Gly Asp Leu Arg Ala Ala Asn Gly Gln Gly Gln Gln Arg Arg Arg Ile
    130                 135                 140
Thr Ser Val Gln Pro Pro Thr Gly Leu Gln Glu Trp Leu Lys Met Phe
145                 150                 155                 160
Gln Ser Trp Ser Gly Pro Glu Lys Leu Leu Ala Leu Asp Glu Leu Ile
                165                 170                 175
Asp Ser Cys Glu Pro Thr Gln Val Lys His Met Met Gln Val Ile Glu
            180                 185                 190
Pro Gln Phe Gln Arg Asp Phe Ile Ser Leu Leu Pro Lys Glu Leu Ala
        195                 200                 205
Leu Tyr Val Leu Ser Phe Leu Glu Pro Lys Asp Leu Leu Gln Ala Ala
    210                 215                 220
Gln Thr Cys Arg Tyr Trp Arg Ile Leu Ala Glu Asp Asn Leu Leu Trp
225                 230                 235                 240
Arg Glu Lys Cys Lys Glu Glu Gly Ile Asp Glu Pro Leu His Ile Lys
                245                 250                 255
Arg Arg Lys Val Ile Lys Pro Gly Phe Ile His Ser Pro Trp Lys Ser
            260                 265                 270
Ala Tyr Ile Arg Gln His Arg Ile Asp Thr Asn Trp Arg Arg Gly Glu
        275                 280                 285
Leu Lys Ser Pro Lys Val Leu Lys Gly His Asp Asp His Val Ile Thr
    290                 295                 300
Cys Leu Gln Phe Cys Gly Asn Arg Ile Val Ser Gly Ser Asp Asp Asn
305                 310                 315                 320
Thr Leu Lys Val Trp Ser Ala Val Thr Gly Lys Cys Leu Arg Thr Leu
                325                 330                 335
Val Gly His Thr Gly Gly Val Trp Ser Ser Gln Met Arg Asp Asn Ile
            340                 345                 350
Ile Ile Ser Gly Ser Thr Asp Arg Thr Leu Lys Val Trp Asn Ala Glu
        355                 360                 365
Thr Gly Glu Cys Ile His Thr Leu Tyr Gly His Thr Ser Thr Val Arg
    370                 375                 380
Cys Met His Leu His Glu Lys Arg Val Val Ser Gly Ser Arg Asp Ala
385                 390                 395                 400
Thr Leu Arg Val Trp Asp Ile Glu Thr Gly Gln Cys Leu His Val Leu
                405                 410                 415
Met Gly His Val Ala Ala Val Arg Cys Val Gln Tyr Asp Gly Arg Arg
            420                 425                 430
Val Val Ser Gly Ala Tyr Asp Phe Met Val Lys Val Trp Asp Pro Glu
        435                 440                 445
Thr Glu Thr Cys Leu His Thr Leu Gln Gly His Thr Asn Arg Val Tyr
    450                 455                 460
Ser Leu Gln Phe Asp Gly Ile His Val Val Ser Gly Ser Leu Asp Thr
465                 470                 475                 480
Ser Ile Arg Val Trp Asp Val Glu Thr Gly Asn Cys Ile His Thr Leu
                485                 490                 495
Thr Gly His Gln Ser Leu Thr Ser Gly Met Glu Leu Lys Asp Asn Ile
            500                 505                 510
Leu Val Ser Gly Asn Ala Asp Ser Thr Val Lys Ile Trp Asp Ile Lys
        515                 520                 525
Thr Gly Gln Cys Leu Gln Thr Leu Gln Gly Pro Asn Lys His Gln Ser
    530                 535                 540
Ala Val Thr Cys Leu Gln Phe Asn Lys Asn Phe Val Ile Thr Ser Ser
545                 550                 555                 560
Asp Asp Gly Thr Val Lys Leu Trp Asp Leu Lys Thr Gly Glu Phe Ile
                565                 570                 575
Arg Asn Leu Val Thr Leu Glu Ser Gly Gly Ser Gly Gly Val Val Trp
            580                 585                 590
Arg Ile Arg Ala Ser Asn Thr Lys Leu Val Cys Ala Val Gly Ser Arg
        595                 600                 605
Asn Gly Thr Glu Glu Thr Lys Leu Leu Val Leu Asp Phe Asp Val Asp
    610                 615                 620
Met Lys
625
</s400> <s200> <s210>22</s210> <s211>31</s211> <s212>DNA</s212> <s213>Artificial Sequence</s213> <s220> <s223>Description of Artificial Sequence
      Oligonucleotide primer</s223> </s220> </s200> <s400> 22
gggtacccct cattattccc tcgagttctt c                                    31
</s400> <s200> <s210>23</s210> <s211>29</s211> <s212>DNA</s212> <s213>Artificial Sequence</s213> <s220> <s223>Description of Artificial Sequence
      Oligonucleotide primer</s223> </s220> </s200> <s400> 23
ggaattcctt catgtccaca tcaaagtcc                                       29
</s400> <s200> <s210>24</s210> <s211>2010</s211> <s212>DNA</s212> <s213>Artificial Sequence</s213> <s220> <s223>Description of Artificial Sequence  V5HIS
      tagged homo sapien</s223> </s220> </s200> <s400> 24
atgtgtgtcc cgagaagcgg tttgatactg agctgcattt gcctttactg tggagttttg     60
ttgccggttc tgctccctaa tcttcctttt ctgacgtgcc tgagcatgtc cacattagaa    120
tctgtgacat acctacctga aaaaggttta tattgtcaga gactgccaag cagccggaca    180
cacgggggca cagaatcact gaaggggaaa aatacagaaa atatgggttt ctacggcaca    240
ttaaaaatga ttttttacaa aatgaaaaga aagttggacc atggttctga ggtccgctct    300
ttttctttgg gaaagaaacc atgcaaagtc tcagaatata caagtaccac tgggcttgta    360
ccatgttcag caacaccaac aacttttggg gacctcagag cagccaatgg ccaagggcaa    420
caacgacgcc gaattacatc tgtccagcca cctacaggcc tccaggaatg gctaaaaatg    480
tttcagagct ggagtggacc agagaaattg cttgctttag atgaactcat tgatagttgt    540
gaaccaacac aagtaaaaca tatgatgcaa gtgatagaac cccagtttca acgagacttc    600
atttcattgc tccctaaaga gttggcactc tatgtgcttt cattcctgga acccaaagac    660
ctgctacaag cagctcagac atgtcgctac tggagaattt tggctgaaga caaccttctc    720
tggagagaga aatgcaaaga agaggggatt gatgaaccat tgcacatcaa gagaagaaaa    780
gtaataaaac caggtttcat acacagtcca tggaaaagtg catacatcag acagcacaga    840
attgatacta actggaggcg aggagaactc aaatctccta aggtgctgaa aggacatgat    900
gatcatgtga tcacatgctt acagttttgt ggtaaccgaa tagttagtgg ttctgatgac    960
aacactttaa aagtttggtc agcagtcaca ggcaaatgtc tgagaacatt agtgggacat   1020
acaggtggag tatggtcatc acaaatgaga gacaacatca tcattagtgg atctacagat   1080
cggacactca aagtgtggaa tgcagagact ggagaatgta tacacacctt atatgggcat   1140
acttccactg tgcgttgtat gcatcttcat gaaaaaagag ttgttagcgg ttctcgagat   1200
gccactctta gggtttggga tattgagaca ggccagtgtt tacatgtttt gatgggtcat   1260
gttgcagcag tccgctgtgt tcaatatgat ggcaggaggg ttgttagtgg agcatatgat   1320
tttatggtaa aggtgtggga tccagagact gaaacctgtc tacacacgtt gcaggggcat   1380
actaatagag tctattcatt acagtttgat ggtatccatg tggtgagtgg atctcttgat   1440
acatcaatcc gtgtttggga tgtggagaca gggaattgca ttcacacgtt aacagggcac   1500
cagtcgttaa caagtggaat ggaactcaaa gacaatattc ttgtctctgg gaatgcagat   1560
tctacagtta aaatctggga tatcaaaaca ggacagtgtt tacaaacatt gcaaggtccc   1620
aacaagcatc agagtgctgt gacctgttta cagttcaaca agaactttgt aattaccagc   1680
tcagatgatg gaactgtaaa actatgggac ttgaaaacgg gtgaatttat tcgaaaccta   1740
gtcacattgg agagtggggg gagtggggga gttgtgtggc ggatcagagc ctcaaacaca   1800
aagctggtgt gtgcagttgg gagtcggaat gggactgaag aaaccaagct gctggtgctg   1860
gactttgatg tggacatgaa ggaattctgc agatatccag cacagtggcg gccgctcgag   1920
tctagagggc ccttcgaagg taagcctatc cctaaccctc tcctcggtct cgattctacg   1980
cgtaccggtc atcatcacca tcaccattga                                    2010
</s400> <s200> <s210>25</s210> <s211>669</s211> <s212>PRT</s212> <s213>Artificial Sequence</s213> <s220> <s223>Description of Artificial Sequence  V5HIS
      tagged homo sapien</s223> </s220> </s200> <s400> 25
Met Cys Val Pro Arg Ser Gly Leu Ile Leu Ser Cys Ile Cys Leu Tyr
  1               5                  10                  15
Cys Gly Val Leu Leu Pro Val Leu Leu Pro Asn Leu Pro Phe Leu Thr
             20                  25                  30
Cys Leu Ser Met Ser Thr Leu Glu Ser Val Thr Tyr Leu Pro Glu Lys
         35                  40                  45
Gly Leu Tyr Cys Gln Arg Leu Pro Ser Ser Arg Thr His Gly Gly Thr
     50                  55                  60
Glu Ser Leu Lys Gly Lys Asn Thr Glu Asn Met Gly Phe Tyr Gly Thr
 65                  70                  75                  80
Leu Lys Met Ile Phe Tyr Lys Met Lys Arg Lys Leu Asp His Gly Ser
                 85                  90                  95
Glu Val Arg Ser Phe Ser Leu Gly Lys Lys Pro Cys Lys Val Ser Glu
            100                 105                 110
Tyr Thr Ser Thr Thr Gly Leu Val Pro Cys Ser Ala Thr Pro Thr Thr
        115                 120                 125
Phe Gly Asp Leu Arg Ala Ala Asn Gly Gln Gly Gln Gln Arg Arg Arg
    130                 135                 140
Ile Thr Ser Val Gln Pro Pro Thr Gly Leu Gln Glu Trp Leu Lys Met
145                 150                 155                 160
Phe Gln Ser Trp Ser Gly Pro Glu Lys Leu Leu Ala Leu Asp Glu Leu
                165                 170                 175
Ile Asp Ser Cys Glu Pro Thr Gln Val Lys His Met Met Gln Val Ile
            180                 185                 190
Glu Pro Gln Phe Gln Arg Asp Phe Ile Ser Leu Leu Pro Lys Glu Leu
        195                 200                 205
Ala Leu Tyr Val Leu Ser Phe Leu Glu Pro Lys Asp Leu Leu Gln Ala
    210                 215                 220
Ala Gln Thr Cys Arg Tyr Trp Arg Ile Leu Ala Glu Asp Asn Leu Leu
225                 230                 235                 240
Trp Arg Glu Lys Cys Lys Glu Glu Gly Ile Asp Glu Pro Leu His Ile
                245                 250                 255
Lys Arg Arg Lys Val Ile Lys Pro Gly Phe Ile His Ser Pro Trp Lys
            260                 265                 270
Ser Ala Tyr Ile Arg Gln His Arg Ile Asp Thr Asn Trp Arg Arg Gly
        275                 280                 285
Glu Leu Lys Ser Pro Lys Val Leu Lys Gly His Asp Asp His Val Ile
    290                 295                 300
Thr Cys Leu Gln Phe Cys Gly Asn Arg Ile Val Ser Gly Ser Asp Asp
305                 310                 315                 320
Asn Thr Leu Lys Val Trp Ser Ala Val Thr Gly Lys Cys Leu Arg Thr
                325                 330                 335
Leu Val Gly His Thr Gly Gly Val Trp Ser Ser Gln Met Arg Asp Asn
            340                 345                 350
Ile Ile Ile Ser Gly Ser Thr Asp Arg Thr Leu Lys Val Trp Asn Ala
        355                 360                 365
Glu Thr Gly Glu Cys Ile His Thr Leu Tyr Gly His Thr Ser Thr Val
    370                 375                 380
Arg Cys Met His Leu His Glu Lys Arg Val Val Ser Gly Ser Arg Asp
385                 390                 395                 400
Ala Thr Leu Arg Val Trp Asp Ile Glu Thr Gly Gln Cys Leu His Val
                405                 410                 415
Leu Met Gly His Val Ala Ala Val Arg Cys Val Gln Tyr Asp Gly Arg
            420                 425                 430
Arg Val Val Ser Gly Ala Tyr Asp Phe Met Val Lys Val Trp Asp Pro
        435                 440                 445
Glu Thr Glu Thr Cys Leu His Thr Leu Gln Gly His Thr Asn Arg Val
    450                 455                 460
Tyr Ser Leu Gln Phe Asp Gly Ile His Val Val Ser Gly Ser Leu Asp
465                 470                 475                 480
Thr Ser Ile Arg Val Trp Asp Val Glu Thr Gly Asn Cys Ile His Thr
                485                 490                 495
Leu Thr Gly His Gln Ser Leu Thr Ser Gly Met Glu Leu Lys Asp Asn
            500                 505                 510
Ile Leu Val Ser Gly Asn Ala Asp Ser Thr Val Lys Ile Trp Asp Ile
        515                 520                 525
Lys Thr Gly Gln Cys Leu Gln Thr Leu Gln Gly Pro Asn Lys His Gln
    530                 535                 540
Ser Ala Val Thr Cys Leu Gln Phe Asn Lys Asn Phe Val Ile Thr Ser
545                 550                 555                 560
Ser Asp Asp Gly Thr Val Lys Leu Trp Asp Leu Lys Thr Gly Glu Phe
                565                 570                 575
Ile Arg Asn Leu Val Thr Leu Glu Ser Gly Gly Ser Gly Gly Val Val
            580                 585                 590
Trp Arg Ile Arg Ala Ser Asn Thr Lys Leu Val Cys Ala Val Gly Ser
        595                 600                 605
Arg Asn Gly Thr Glu Glu Thr Lys Leu Leu Val Leu Asp Phe Asp Val
    610                 615                 620
Asp Met Lys Glu Phe Cys Arg Tyr Pro Ala Gln Trp Arg Pro Leu Glu
625                 630                 635                 640
Ser Arg Gly Pro Phe Glu Gly Lys Pro Ile Pro Asn Pro Leu Leu Gly
                645                 650                 655
Leu Asp Ser Thr Arg Thr Gly His His His His His His
            660                 665
</s400> <s200> <s210>26</s210> <s211>2001</s211> <s212>DNA</s212> <s213>Artificial Sequence</s213> <s220> <s223>Description of Artificial Sequence  MYCHIS
      tagged homo sapiens</s223> </s220> </s200> <s400> 26
atgtgtgtcc cgagaagcgg tttgatactg agctgcattt gcctttactg tggagttttg     60
ttgccggttc tgctccctaa tcttcctttt ctgacgtgcc tgagcatgtc cacattagaa    120
tctgtgacat acctacctga aaaaggttta tattgtcaga gactgccaag cagccggaca    180
cacgggggca cagaatcact gaaggggaaa aatacagaaa atatgggttt ctacggcaca    240
ttaaaaatga ttttttacaa aatgaaaaga aagttggacc atggttctga ggtccgctct    300
ttttctttgg gaaagaaacc atgcaaagtc tcagaatata caagtaccac tgggcttgta    360
ccatgttcag caacaccaac aacttttggg gacctcagag cagccaatgg ccaagggcaa    420
caacgacgcc gaattacatc tgtccagcca cctacaggcc tccaggaatg gctaaaaatg    480
tttcagagct ggagtggacc agagaaattg cttgctttag atgaactcat tgatagttgt    540
gaaccaacac aagtaaaaca tatgatgcaa gtgatagaac cccagtttca acgagacttc    600
atttcattgc tccctaaaga gttggcactc tatgtgcttt cattcctgga acccaaagac    660
ctgctacaag cagctcagac atgtcgctac tggagaattt tggctgaaga caaccttctc    720
tggagagaga aatgcaaaga agaggggatt gatgaaccat tgcacatcaa gagaagaaaa    780
gtaataaaac caggtttcat acacagtcca tggaaaagtg catacatcag acagcacaga    840
attgatacta actggaggcg aggagaactc aaatctccta aggtgctgaa aggacatgat    900
gatcatgtga tcacatgctt acagttttgt ggtaaccgaa tagttagtgg ttctgatgac    960
aacactttaa aagtttggtc agcagtcaca ggcaaatgtc tgagaacatt agtgggacat   1020
acaggtggag tatggtcatc acaaatgaga gacaacatca tcattagtgg atctacagat   1080
cggacactca aagtgtggaa tgcagagact ggagaatgta tacacacctt atatgggcat   1140
acttccactg tgcgttgtat gcatcttcat gaaaaaagag ttgttagcgg ttctcgagat   1200
gccactctta gggtttggga tattgagaca ggccagtgtt tacatgtttt gatgggtcat   1260
gttgcagcag tccgctgtgt tcaatatgat ggcaggaggg ttgttagtgg agcatatgat   1320
tttatggtaa aggtgtggga tccagagact gaaacctgtc tacacacgtt gcaggggcat   1380
actaatagag tctattcatt acagtttgat ggtatccatg tggtgagtgg atctcttgat   1440
acatcaatcc gtgtttggga tgtggagaca gggaattgca ttcacacgtt aacagggcac   1500
cagtcgttaa caagtggaat ggaactcaaa gacaatattc ttgtctctgg gaatgcagat   1560
tctacagtta aaatctggga tatcaaaaca ggacagtgtt tacaaacatt gcaaggtccc   1620
aacaagcatc agagtgctgt gacctgttta cagttcaaca agaactttgt aattaccagc   1680
tcagatgatg gaactgtaaa actatgggac ttgaaaacgg gtgaatttat tcgaaaccta   1740
gtcacattgg agagtggggg gagtggggga gttgtgtggc ggatcagagc ctcaaacaca   1800
aagctggtgt gtgcagttgg gagtcggaat gggactgaag aaaccaagct gctggtgctg   1860
gactttgatg tggacatgaa ggaattctgc agatatccag cacagtggcg gccgctcgag   1920
tctagagggc ccttcgaaca aaaactcatc tcagaagagg atctgaatat gcataccggt   1980
catcatcacc atcaccattg a                                             2001
</s400> <s200> <s210>27</s210> <s211>666</s211> <s212>PRT</s212> <s213>Artificial Sequence</s213> <s220> <s223>Description of Artificial Sequence  MYCHIS
      tagged homo sapiens</s223> </s220> </s200> <s400> 27
Met Cys Val Pro Arg Ser Gly Leu Ile Leu Ser Cys Ile Cys Leu Tyr
  1               5                  10                  15
Cys Gly Val Leu Leu Pro Val Leu Leu Pro Asn Leu Pro Phe Leu Thr
             20                  25                  30
Cys Leu Ser Met Ser Thr Leu Glu Ser Val Thr Tyr Leu Pro Glu Lys
         35                  40                  45
Gly Leu Tyr Cys Gln Arg Leu Pro Ser Ser Arg Thr His Gly Gly Thr
     50                  55                  60
Glu Ser Leu Lys Gly Lys Asn Thr Glu Asn Met Gly Phe Tyr Gly Thr
 65                  70                  75                  80
Leu Lys Met Ile Phe Tyr Lys Met Lys Arg Lys Leu Asp His Gly Ser
                 85                  90                  95
Glu Val Arg Ser Phe Ser Leu Gly Lys Lys Pro Cys Lys Val Ser Glu
            100                 105                 110
Tyr Thr Ser Thr Thr Gly Leu Val Pro Cys Ser Ala Thr Pro Thr Thr
        115                 120                 125
Phe Gly Asp Leu Arg Ala Ala Asn Gly Gln Gly Gln Gln Arg Arg Arg
    130                 135                 140
Ile Thr Ser Val Gln Pro Pro Thr Gly Leu Gln Glu Trp Leu Lys Met
145                 150                 155                 160
Phe Gln Ser Trp Ser Gly Pro Glu Lys Leu Leu Ala Leu Asp Glu Leu
                165                 170                 175
Ile Asp Ser Cys Glu Pro Thr Gln Val Lys His Met Met Gln Val Ile
            180                 185                 190
Glu Pro Gln Phe Gln Arg Asp Phe Ile Ser Leu Leu Pro Lys Glu Leu
        195                 200                 205
Ala Leu Tyr Val Leu Ser Phe Leu Glu Pro Lys Asp Leu Leu Gln Ala
    210                 215                 220
Ala Gln Thr Cys Arg Tyr Trp Arg Ile Leu Ala Glu Asp Asn Leu Leu
225                 230                 235                 240
Trp Arg Glu Lys Cys Lys Glu Glu Gly Ile Asp Glu Pro Leu His Ile
                245                 250                 255
Lys Arg Arg Lys Val Ile Lys Pro Gly Phe Ile His Ser Pro Trp Lys
            260                 265                 270
Ser Ala Tyr Ile Arg Gln His Arg Ile Asp Thr Asn Trp Arg Arg Gly
        275                 280                 285
Glu Leu Lys Ser Pro Lys Val Leu Lys Gly His Asp Asp His Val Ile
    290                 295                 300
Thr Cys Leu Gln Phe Cys Gly Asn Arg Ile Val Ser Gly Ser Asp Asp
305                 310                 315                 320
Asn Thr Leu Lys Val Trp Ser Ala Val Thr Gly Lys Cys Leu Arg Thr
                325                 330                 335
Leu Val Gly His Thr Gly Gly Val Trp Ser Ser Gln Met Arg Asp Asn
            340                 345                 350
Ile Ile Ile Ser Gly Ser Thr Asp Arg Thr Leu Lys Val Trp Asn Ala
        355                 360                 365
Glu Thr Gly Glu Cys Ile His Thr Leu Tyr Gly His Thr Ser Thr Val
    370                 375                 380
Arg Cys Met His Leu His Glu Lys Arg Val Val Ser Gly Ser Arg Asp
385                 390                 395                 400
Ala Thr Leu Arg Val Trp Asp Ile Glu Thr Gly Gln Cys Leu His Val
                405                 410                 415
Leu Met Gly His Val Ala Ala Val Arg Cys Val Gln Tyr Asp Gly Arg
            420                 425                 430
Arg Val Val Ser Gly Ala Tyr Asp Phe Met Val Lys Val Trp Asp Pro
        435                 440                 445
Glu Thr Glu Thr Cys Leu His Thr Leu Gln Gly His Thr Asn Arg Val
    450                 455                 460
Tyr Ser Leu Gln Phe Asp Gly Ile His Val Val Ser Gly Ser Leu Asp
465                 470                 475                 480
Thr Ser Ile Arg Val Trp Asp Val Glu Thr Gly Asn Cys Ile His Thr
                485                 490                 495
Leu Thr Gly His Gln Ser Leu Thr Ser Gly Met Glu Leu Lys Asp Asn
            500                 505                 510
Ile Leu Val Ser Gly Asn Ala Asp Ser Thr Val Lys Ile Trp Asp Ile
        515                 520                 525
Lys Thr Gly Gln Cys Leu Gln Thr Leu Gln Gly Pro Asn Lys His Gln
    530                 535                 540
Ser Ala Val Thr Cys Leu Gln Phe Asn Lys Asn Phe Val Ile Thr Ser
545                 550                 555                 560
Ser Asp Asp Gly Thr Val Lys Leu Trp Asp Leu Lys Thr Gly Glu Phe
                565                 570                 575
Ile Arg Asn Leu Val Thr Leu Glu Ser Gly Gly Ser Gly Gly Val Val
            580                 585                 590
Trp Arg Ile Arg Ala Ser Asn Thr Lys Leu Val Cys Ala Val Gly Ser
        595                 600                 605
Arg Asn Gly Thr Glu Glu Thr Lys Leu Leu Val Leu Asp Phe Asp Val
    610                 615                 620
Asp Met Lys Glu Phe Cys Arg Tyr Pro Ala Gln Trp Arg Pro Leu Glu
625                 630                 635                 640
Ser Arg Gly Pro Phe Glu Gln Lys Leu Ile Ser Glu Glu Asp Leu Asn
                645                 650                 655
Met His Thr Gly His His His His His His
            660                 665
</s400> <s200> <s210>28</s210> <s211>25</s211> <s212>DNA</s212> <s213>Artificial Sequence</s213> <s220> <s223>Description of Artificial Sequence
      Oligonucleotide primer</s223> </s220> </s200> <s400> 28
tcacttcatgtccacatcaaagtcc                                             25
</s400> <s200> <s210>29</s210> <s211>27</s211> <s212>DNA</s212> <s213>Artificial Sequence</s213> <s220> <s223>Description of Artificial Sequence
      Oligonucleotide primer</s223> </s220> </s200> <s400> 29
ggtaattacaaagttcttgttgaactg                                           27
</s400> <s200> <s210>30</s210> <s211>22</s211> <s212>DNA</s212> <s213>Artificial Sequence</s213> <s220> <s223>Description of Artificial Sequence
      Oligonucleotide primer</s223> </s220> </s200> <s400> 30
ccctgcaacgtgtgtagacagg                                                22
</s400> <s200> <s210>31</s210> <s211>24</s211> <s212>DNA</s212> <s213>Artificial Sequence</s213> <s220> <s223>Description of Artificial Sequence
      Oligonucleotide primer</s223> </s220> </s200> <s400> 31
ccagtctctgcattccacactttg                                              24
</s400> <s200> <s210>32</s210> <s211>23</s211> <s212>DNA</s212> <s213>Artificial Sequence</s213> <s220> <s223>Description of Artificial Sequence
      Oligonucleotide primer</s223> </s220> </s200> <s400> 32
ctcagacaggtcaggacatttgg                                               23
</s400> </sequence-list-new-rules> </sequence-cwu> </p>
    </div></div></div><div class="patent-section patent-tabular-section"><a id="backward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Cited Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1997011956A1?cl=en">WO1997011956A1</a></td><td class="patent-data-table-td patent-date-value">Sep 27, 1996</td><td class="patent-data-table-td patent-date-value">Apr 3, 1997</td><td class="patent-data-table-td ">Iva Greenwald</td><td class="patent-data-table-td ">IDENTIFICATION OF sel-12 AND USES THEREOF</td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="npl-citations"></a><div class="patent-section-header"><span class="patent-section-title">Non-Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th colspan="3"class="patent-data-table-th">Reference</th></tr></thead><tr><td class="patent-data-table-td ">1</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">A. Doan, G. Thinakaran, D.R. Borchelt, H.H. Slunt, T. Ratovitsky, M. Podlisny, D.J. Selkoe, M. Seeger, S.E. Gandy, D.L. Price, S.S. Sisodia, "<a href='http://scholar.google.com/scholar?q="Protein+Topology+of+Presenilin+1%2C"'>Protein Topology of Presenilin 1,</a>" Neuron, 1996, 17:1023-1030.</td></tr><tr><td class="patent-data-table-td ">2</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">D. Levitan, I. Greenwald, "<a href='http://scholar.google.com/scholar?q="Facilitation+of+lin-12-mediated+signalling+by+sel-12%2C+a+Caenorhabditis+elegans+S182+Alzheimer%27s+disease+gene%2C"'>Facilitation of lin-12-mediated signalling by sel-12, a Caenorhabditis elegans S182 Alzheimer's disease gene,</a>" Nature, 1995, 377:351-4.</td></tr><tr><td class="patent-data-table-td ">3</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">D. Levitan, T.G. Doyle, D. Brousseau, M.K. Lee, G. Thinakaran, H.H. Slunt, S.S. Sidodia, I. Greenwald, "<a href='http://scholar.google.com/scholar?q="Assessment+of+normal+and+mutant+human+presenilin+function+in+Caenorgabditis+elegans%2C"'>Assessment of normal and mutant human presenilin function in Caenorgabditis elegans,</a>" Proc. Natl. Acad. Sci. USA, 1996, 93: 14940-4.</td></tr><tr><td class="patent-data-table-td ">4</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">D. Scheuner, C. Eckman, M. Jensen, X. Song, M. Citron, N. Suzukik, T.D. Bird, J. Hardy, M. Hutton, W. Kukull, E. Larson, E. Levy-Lahad, M. Viitanen, E. Peskind, P. Poorkaj, G. Schellenberg, R. Tanzi, W. Wasco, L. Lannfelt, D. Selkoe, S. Younkin, "<a href='http://scholar.google.com/scholar?q="Secreted+amyloid+beta-protein+similar+to+that+in+the+senile+plaques+of+Alzheimer%27s+disease+is+increased+in+vivo+by+the+presenilin+1+and+2+and+APP+mutations+linked+to+familial+Alzheimer%27s+disease%2C"'>Secreted amyloid beta-protein similar to that in the senile plaques of Alzheimer's disease is increased in vivo by the presenilin 1 and 2 and APP mutations linked to familial Alzheimer's disease,</a>" Nat. Med., 1996, 2:864-870.</td></tr><tr><td class="patent-data-table-td ">5</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">D. Scheuner, C. Eckman, M. Jensen, X. Song, M. Citron, N. Suzukik, T.D. Bird, J. Hardy, M. Hutton, W. Kukull, E. Larson, E. Levy-Lahad, M. Viitanen, E. Peskind, P. Poorkaj, G. Schellenberg, R. Tanzi, W. Wasco, L. Lannfelt, D. Selkoe, S. Younkin, "<a href='http://scholar.google.com/scholar?q="Secreted+amyloid+%CE%B2-protein+similar+to+that+in+the+senile+plaques+of+Alzheimer%27s+disease+is+increased+in+vivo+by+the+presenilin+1+and+2+and+APP+mutations+linked+to+familial+Alzheimer%27s+disease%2C"'>Secreted amyloid β-protein similar to that in the senile plaques of Alzheimer's disease is increased in vivo by the presenilin 1 and 2 and APP mutations linked to familial Alzheimer's disease,</a>" Nat. Med., 1996, 2:864-870.</td></tr><tr><td class="patent-data-table-td ">6</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">D.M. Kovacs, H.J. Fausett, K.J. Page, T.-W. Kim, R.D. Moir, D.E.Merriam, R.D. Hollister, O.G. Hallmark, R. Mancini, K.M. Felsenstein, B.T. Hyman, R.E. Tanzi, W. Wasco,"<a href='http://scholar.google.com/scholar?q="Alzheimer-associated+presenilins+1+and+2%3A+Neronal+expression+in+brain+and+localization+to+intracellular+membranes+in+mammalian+cells%2C"'>Alzheimer-associated presenilins 1 and 2: Neronal expression in brain and localization to intracellular membranes in mammalian cells,</a>" Nat. Med., 1996, 2:224-229.</td></tr><tr><td class="patent-data-table-td ">7</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">D.R. Borchelt, G. Thinakaran, C.B. Eckman, M.K. Lee, F. Davenport, T. Ratovitosky, C.-M. Prada, G. Kim, S. Seekins, D. Yager, H.H. Slunt, R. Wang, M. M. Seeger, A.I. Levey, S.E. Gandy, N.G. Copeland, N.A. Jenkins, D.L. Price, S.G. Younkin, S.S. Sisodia, "<a href='http://scholar.google.com/scholar?q="Familial+Alzheimer%27s+Disease-Linked+Presenilin+1+Variants+Elevate+Abeta1-42%2F1-40+Ratio+In+Vitro+and+In+Vivo%2C"'>Familial Alzheimer's Disease-Linked Presenilin 1 Variants Elevate Abeta1-42/1-40 Ratio In Vitro and In Vivo,</a>" Neuron, 1996, 17:1005-1013.</td></tr><tr><td class="patent-data-table-td ">8</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">D.R. Borchelt, G. Thinakaran, C.B. Eckman, M.K. Lee, F. Davenport, T. Ratovitosky, C.-M. Prada, G. Kim, S. Seekins, D. Yager, H.H. Slunt, R. Wang, M. M. Seeger, A.I. Levey, S.E. Gandy, N.G. Copeland, N.A. Jenkins, D.L. Price, S.G. Younkin, S.S. Sisodia, "<a href='http://scholar.google.com/scholar?q="Familial+Alzheimer%27s+Disease-Linked+Presenilin+1+Variants+Elevate+A%CE%B21-42%2F1-40+Ratio+In+Vitro+and+In+Vivo%2C"'>Familial Alzheimer's Disease-Linked Presenilin 1 Variants Elevate Aβ1-42/1-40 Ratio In Vitro and In Vivo,</a>" Neuron, 1996, 17:1005-1013.</td></tr><tr><td class="patent-data-table-td ">9</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">D.R. Borchelt, T. Ratovitski, J. van Lare, M.K. Lee, V. Gonzales, N.A. Jenkins, N.G. Copeland, D.L. Price, S.S. Sisodia, "<a href='http://scholar.google.com/scholar?q="Accelerated+Amyloid+Deposition+in+the+Brains+of+Transgenic+Mice+Coexpressing+Mutant+Presenilin+1+and+Amyloid+PrecursorProteins%2C"'>Accelerated Amyloid Deposition in the Brains of Transgenic Mice Coexpressing Mutant Presenilin 1 and Amyloid PrecursorProteins,</a>" Neuron, 1997, 19:939-945.</td></tr><tr><td class="patent-data-table-td ">10</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">E. Levy-Lahad, W. Wasco, P. Poorkaj, D.M. Ramano, J. Oshima, W.H. Pettingell, C.-E. Yu, P.D. Jondro, S.D. Schmidt, K. Wang, A.C. Crowley, Y.-H. Fu, S.Y. Guenette, D. Galas, E. Nemens, E.M. Wijsman, T.D. Bird, G.D. Schellenberg, R.E. Tanzi, "<a href='http://scholar.google.com/scholar?q="Candidate+Gene+for+the+Chromosome+1+Familial+Alzheimer%27s+Disease+Locus%2C"'>Candidate Gene for the Chromosome 1 Familial Alzheimer's Disease Locus,</a>" Science, 1995, 269: 973-977.</td></tr><tr><td class="patent-data-table-td ">11</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">E.I. Rogaev, R. Sherrington, E.A. Rogaeva, G. Levesque, M. Ideda, Y. Liang, H. Chi, C. Lin, K. Holman, T. Tsuda, L. Mar, S. Sorbi, B. Nacmais, S. Piacentini, L. Amaducci, I. Chumakov, D. Cohen, L. Lannfelt, P.E. Fraser, J.M. Rommens, P.H. St George-Hyslop, "<a href='http://scholar.google.com/scholar?q="Familial+Alzheimer%27s+disease+in+kindreds+with+missense+mutations+in+a+gene+on+chromosome+1+related+to+the+Alzheimer%27s+disease+type+3+gene%2C"'>Familial Alzheimer's disease in kindreds with missense mutations in a gene on chromosome 1 related to the Alzheimer's disease type 3 gene,</a>" Nature, 1995, 376: 775-778.</td></tr><tr><td class="patent-data-table-td ">12</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">E.J.A. Hubbard, G. Wu, J. Kitajewski, I. Greenwald, "<a href='http://scholar.google.com/scholar?q="sel-10%2C+a+negative+regulator+of+lin-12+activity+in+Caenorhabditis+elegans%2C+encodes+a+member+of+the+CDC4+family+of+proteins%2C"'>sel-10, a negative regulator of lin-12 activity in Caenorhabditis elegans, encodes a member of the CDC4 family of proteins,</a>" Genes &amp; Development, 1997, 11:3182-3193.</td></tr><tr><td class="patent-data-table-td ">13</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Genbank H22962 Jul. 6, 1995.</td></tr><tr><td class="patent-data-table-td ">14</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Genbank H34371 Jul. 19, 1995.</td></tr><tr><td class="patent-data-table-td ">15</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">I. Greenwald, G. Seydoux, "<a href='http://scholar.google.com/scholar?q="Analysis+of+gain-of-function+mutations+of+the+lin-12+gene+of+Caenorhabditis+elegans%2C"'>Analysis of gain-of-function mutations of the lin-12 gene of Caenorhabditis elegans,</a>" Nature, 1990, 346: 197-9.</td></tr><tr><td class="patent-data-table-td ">16</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">J. Li, J. Ma, H. Potter, "<a href='http://scholar.google.com/scholar?q="Identification+and+expression+analysis+of+a+potential+familial+Alzheimer+disease+gene+on+chromosome+1+related+to+AD3%2C"'>Identification and expression analysis of a potential familial Alzheimer disease gene on chromosome 1 related to AD3,</a>" Proc. Natl. Acad. Sci. U.S.A., 1995, 92:12180-12184.</td></tr><tr><td class="patent-data-table-td ">17</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">J. Perez-Tur, S. Froelich, G. Prihar, R. Crook, M. Baker, K. Duff, M. Wragg, F. Busfield, C. Lendon, R.F. Clark, P. Roques, R.A. Fuldner, J. Johnston, R. Cowburn, C. Forsell, K. Axelman, L. Lilius, H. Houlden, E. Karran, G.W. Roberts, M. Rossor, M.D. Adams, J. Hardy, A. Goate, L. Lannfelt, M. Hutton, "<a href='http://scholar.google.com/scholar?q="A+Mutation+in+Alzheimer%27s+disease+destroying+a+splice+acceptor+site+in+the+presenilin-1+gene%2C"'>A Mutation in Alzheimer's disease destroying a splice acceptor site in the presenilin-1 gene,</a>" Neuroreport, 1995, 7:297-301.</td></tr><tr><td class="patent-data-table-td ">18</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">K. Duff, C. Eckman, C. Zehr, X. Yu, C.-M. Prada, J. Perez-Tur, M. Hutton, L. Buee, Y. Harigaya, D. Yager, D. Morgan, M.N. Gordon, L. Holcomb, L. Refolo, B. Zenk, J. Hardy, S. Younkin, "<a href='http://scholar.google.com/scholar?q="Increased+amyloid-beta42%2842%29in+brains+of+mice+expressing+mutant+presenilin+1%2C"'>Increased amyloid-beta42(42)in brains of mice expressing mutant presenilin 1,</a>" Nature, 1996, 383:710-713.</td></tr><tr><td class="patent-data-table-td ">19</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">K. Duff, C. Eckman, C. Zehr, X. Yu, C.-M. Prada, J. Perez-Tur, M. Hutton, L. Buee, Y. Harigaya, D. Yager, D. Morgan, M.N. Gordon, L. Holcomb, L. Refolo, B. Zenk, J. Hardy, S. Younkin, "<a href='http://scholar.google.com/scholar?q="Increased+amyloid-%CE%B242%2842%29in+brains+of+mice+expressing+mutant+presenilin+1%2C"'>Increased amyloid-β42(42)in brains of mice expressing mutant presenilin 1,</a>" Nature, 1996, 383:710-713.</td></tr><tr><td class="patent-data-table-td ">20</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">M. Citron, D. Westaway, W. Xia, G. Carlson, T. Diehl, G. Levesque, K. Johnson-Wood, M. Lee, P. Seubert, A. Davis, D. Kholodenko, R. Motter, R. Sherrington,l B. Perry, H. Yao, R. Strome, I. Lieberburg, J. Rommens, S. Kim, D. Schenk, P. Fraser, P. St George Hyslop, D.J. Selkoe, "<a href='http://scholar.google.com/scholar?q="Mutant+presenilins+of+Alzheimer%27s+disease+increase+production+of+42-residue+amyloid+beta-protein+in+both+transfected+cells+and+transgenic+mice%2C"'>Mutant presenilins of Alzheimer's disease increase production of 42-residue amyloid beta-protein in both transfected cells and transgenic mice,</a>" Nat. Med., 1997, 3:67-72.</td></tr><tr><td class="patent-data-table-td ">21</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">M. Citron, D. Westaway, W. Xia, G. Carlson, T. Diehl, G. Levesque, K. Johnson-Wood, M. Lee, P. Seubert, A. Davis, D. Kholodenko, R. Motter, R. Sherrington,l B. Perry, H. Yao, R. Strome, I. Lieberburg, J. Rommens, S. Kim, D. Schenk, P. Fraser, P. St George Hyslop, D.J. Selkoe, "<a href='http://scholar.google.com/scholar?q="Mutant+presenilins+of+Alzheimer%27s+disease+increase+production+of+42-residue+amyloid+%CE%B2-protein+in+both+transfected+cells+and+transgenic+mice%2C"'>Mutant presenilins of Alzheimer's disease increase production of 42-residue amyloid β-protein in both transfected cells and transgenic mice,</a>" Nat. Med., 1997, 3:67-72.</td></tr><tr><td class="patent-data-table-td ">22</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">M. Hochstrasser, "<a href='http://scholar.google.com/scholar?q="Ubiquitin-dependent+protein+degradation%2C"'>Ubiquitin-dependent protein degradation,</a>" Annual Review of Genetics, 1996, 30:405-439.</td></tr><tr><td class="patent-data-table-td ">23</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">M. Mercken, H. Takahashi, T. Honda, Z. Sato, M. Murayama, Y. Nakazato, K. Noguchi, K. Imahori, A. Takashima, "<a href='http://scholar.google.com/scholar?q="Characterization+of+human+presenilin+1+using+N-terminal+specific+monoclonal+antibodies%3A+Evidence+that+Alzheimer+mutations+affect+proteolytic+processing%2C"'>Characterization of human presenilin 1 using N-terminal specific monoclonal antibodies: Evidence that Alzheimer mutations affect proteolytic processing,</a>" FEBS Lett., 1996, 389:297-303.</td></tr><tr><td class="patent-data-table-td ">24</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">M. Mullan, F. Crawford, K. Axelman, H. Houlden, L. Lilius, B. Winblad, L. Lannfelt, "<a href='http://scholar.google.com/scholar?q="A+pathogenic+mutation+for+probable+Alzheimer%27s+disease+in+the+APP+gene+at+the+N-terminus+of+beta-amyloid%2C"'>A pathogenic mutation for probable Alzheimer's disease in the APP gene at the N-terminus of beta-amyloid,</a>" Nat Genet, Aug., 1992 1(5):345-7.</td></tr><tr><td class="patent-data-table-td ">25</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">M. Mullan, F. Crawford, K. Axelman, H. Houlden, L. Lilius, B. Winblad, L. Lannfelt, "<a href='http://scholar.google.com/scholar?q="A+pathogenic+mutation+for+probable+Alzheimer%27s+disease+in+the+APP+gene+at+the+N-terminus+of+%CE%B2-amyloid%2C"'>A pathogenic mutation for probable Alzheimer's disease in the APP gene at the N-terminus of β-amyloid,</a>" Nat Genet, Aug., 1992 1(5):345-7.</td></tr><tr><td class="patent-data-table-td ">26</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">M. Sundaram, I. Greenwald, "<a href='http://scholar.google.com/scholar?q="Genetic+and+Phenotypic+Studies+of+Hypomorphic+lin-12+Mutants+in+Caenorhabditis+elegans%2C"'>Genetic and Phenotypic Studies of Hypomorphic lin-12 Mutants in Caenorhabditis elegans,</a>" Genetics, 1993, 135: 755-763.</td></tr><tr><td class="patent-data-table-td ">27</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">M. Sundaram, I. Greenwald, "<a href='http://scholar.google.com/scholar?q="Suppressors+of+a+lin-12+Hypomorph+Define+Genes+That+Interact+With+Both+lin-12+and+glp-1+in+Caenorhabditis+elegans%2C"'>Suppressors of a lin-12 Hypomorph Define Genes That Interact With Both lin-12 and glp-1 in Caenorhabditis elegans,</a>" Genetics, 1993, 135: 765-83.</td></tr><tr><td class="patent-data-table-td ">28</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">R. Sherrington, E.I. Rogaev, Y. Liang, E.A. Rogaeva, G. Levesque, M. Ikeda, H. Chi, C. Lin, G. Li, K. Holman, T. Tsuda, L. Mar, J.-F. Foncin, A.C. Bruni, M.P. Montesi, S. Sorbi, I. Rainero, L. Pinessi, L. Nee, I. Chumakov, D. Pollen, A. Brookes, P. Sanseau, R.J. Polinsky, W. Wasco, H.A.R. Da Silva, J.L. Haines, M.A. Pericak-Vance, R.E. Tanzi, A.D. Roses, P.E. Fraser, J.M. Rommens, P.H. St George-Hyslop, "<a href='http://scholar.google.com/scholar?q="Cloning+of+a+gene+bearing+missense+mutations+in+early-onset+familial+Alzheimer%27s+disease%2C"'>Cloning of a gene bearing missense mutations in early-onset familial Alzheimer's disease,</a>" Nature, 1995, 375: 754-760.</td></tr><tr><td class="patent-data-table-td ">29</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">T.W. Kim, W.H. Pettingell, O.G. Hallmark, R.D. Moir, W. Wasco, R.E. Tanzi, "<a href='http://scholar.google.com/scholar?q="Endoproteoltic+Cleavage+and+Proteasomal+Degradation+of+Presenilin+2+in+Transfected+Cells%2C"'>Endoproteoltic Cleavage and Proteasomal Degradation of Presenilin 2 in Transfected Cells,</a>" Journal of Biological Chemistry, 1997, 272: 11006-11010.</td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="classifications"></a><div class="patent-section-header"><span class="patent-section-title">Classifications</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th"> </th><th class="patent-data-table-th"> </th></tr></thead><tr><td class="patent-data-table-td ">U.S. Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=XkhmBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc530/defs530.htm&usg=AFQjCNEOd3srpZKCSH0W93D5fiJ4_0S25w#C530S350000">530/350</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=XkhmBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc530/defs530.htm&usg=AFQjCNEOd3srpZKCSH0W93D5fiJ4_0S25w#C530S300000">530/300</a></span></td></tr><tr><td class="patent-data-table-td ">International Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=XkhmBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=G01N0033500000">G01N33/50</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=XkhmBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12Q0001020000">C12Q1/02</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=XkhmBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0001150000">C12N1/15</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=XkhmBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=A61P0025280000">A61P25/28</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=XkhmBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=G01N0033150000">G01N33/15</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=XkhmBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0005100000">C12N5/10</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=XkhmBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12P0021020000">C12P21/02</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=XkhmBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=G01N0033530000">G01N33/53</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=XkhmBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C07K0016180000">C07K16/18</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=XkhmBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0015090000">C12N15/09</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=XkhmBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=A61P0043000000">A61P43/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=XkhmBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=G01N0033566000">G01N33/566</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=XkhmBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=A61K0045000000">A61K45/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=XkhmBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0015120000">C12N15/12</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=XkhmBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0001190000">C12N1/19</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=XkhmBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0005020000">C12N5/02</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=XkhmBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C07K0014470000">C07K14/47</a></span></td></tr><tr><td class="patent-data-table-td ">Cooperative Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=XkhmBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C07K14/4702">C07K14/4702</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=XkhmBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C07K2319/00">C07K2319/00</a></span></td></tr><tr><td class="patent-data-table-td ">European Classification</td><td class="patent-data-table-td "><span class="nested-value">C07K14/47A1</span></td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="legal-events"></a><div class="patent-section-header"><span class="patent-section-title">Legal Events</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Date</th><th class="patent-data-table-th">Code</th><th class="patent-data-table-th">Event</th><th class="patent-data-table-th">Description</th></tr></thead><tr><td class="patent-data-table-td patent-date-value">Jun 24, 2008</td><td class="patent-data-table-td ">FP</td><td class="patent-data-table-td ">Expired due to failure to pay maintenance fee</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20080504</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">May 4, 2008</td><td class="patent-data-table-td ">LAPS</td><td class="patent-data-table-td ">Lapse for failure to pay maintenance fees</td><td class="patent-data-table-td "></td></tr><tr><td class="patent-data-table-td patent-date-value">Nov 12, 2007</td><td class="patent-data-table-td ">REMI</td><td class="patent-data-table-td ">Maintenance fee reminder mailed</td><td class="patent-data-table-td "></td></tr><tr><td class="patent-data-table-td patent-date-value">Apr 4, 2006</td><td class="patent-data-table-td ">B1</td><td class="patent-data-table-td ">Reexamination certificate first reexamination</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">THE PATENTABILITY OF CLAIMS 1-10 IS CONFIRMED.</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Oct 5, 2004</td><td class="patent-data-table-td ">RR</td><td class="patent-data-table-td ">Request for reexamination filed</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20040823</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Dec 15, 1999</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">PHARMACIA &amp; UPJOHN COMPANY, MICHIGAN</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNORS:GURNEY, MARK E.;PAULEY, ADELE M.;LI, JINHE;REEL/FRAME:010454/0650;SIGNING DATES FROM 19990728 TO 19990823</span></div></td></tr></table><div class="patent-section-footer"></div></div><div class="modal-dialog" id="patent-images-lightbox"><div class="patent-lightbox-controls"><div class="patent-lightbox-rotate-controls"><div class="patent-lightbox-rotation-text">Rotate</div><div class="rotate-icon rotate-ccw-icon"></div><div class="rotate-icon rotate-cw-icon"></div></div><div class="patent-lightbox-index-counter"></div><a class="patent-lightbox-fullsize-link" target="_blank">Original Image</a><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_right.png" alt="Next page"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_left.png" alt="Previous page"width="21" height="21" /></div></div><div class="modal-dialog-content"><div class="patent-lightbox-image-holder"><div class="patent-lightbox-placeholder"></div></div></div></div><script>_OC_initPatentsAtb({image_not_available_html: " Image not available"});</script></div></div></div></td></tr></table><script>(function() {var href = window.location.href;if (href.indexOf('?') !== -1) {var parameters = href.split('?')[1].split('&');for (var i = 0; i < parameters.length; i++) {var param = parameters[i].split('=');if (param[0] == 'focus') {var elem = document.getElementById(param[1]);if (elem) {elem.focus();}}}}})();</script><script>_OC_addFlags({LockSrc:"/books/javascript/lock_8a2b04e7bf975d5171d8e4c0b6365c7a.js", Host:"http://www.google.com/", IsBooksRentalEnabled:1, IsWebstoreDisplayCaseEnabled:1, IsObfuscationEnabled:1, IsBrowsingHistoryEnabled:1, IsWebReaderSvgEnabled:0, IsGeoLayerEnabled:1, IsImageModeNotesEnabled:1, IsCopyMenuItemEnabled:1, IsGiftingEnabled:0, IsWebReaderUniversalPaginatorEnabled:0, IsOfflineBubbleEnabled:1, IsReaderEnabledForPlayRequests:1, IsFutureOnSaleVolumesEnabled:1, IsOfflineRestrictedCopyEnabled:1, IsBooksUnifiedLeftNavEnabled:1, IsRestrictedCopyEnabled:1, IsZipitFolderCollectionEnabled:1, IsEndOfSampleRecommendationsEnabled:1, IsRatingsOnBookcardsEnabled:1, IsAdsDisabled:0, IsIframePageDisplayEnabled:0, IsEmbeddedMediaEnabled:1, IsImageModeAnnotationsEnabled:1, IsMyLibraryGooglePlusEnabled:1, IsImagePageProviderEnabled:0, IsBookcardListPriceSmall:0, IsInternalUser:0, IsBooksShareButtonEnabled:0, IsPreOrdersEnabled:0, IsDisabledRandomBookshelves:0, WebstoreDisplayCasePosition:3});_OC_Run({"enable_p13n":false,"add_vol_to_collection_base_url":"http://www.google.com/patents?op=add\u0026sig=ACfU3U12ikFTi9jHqrQP0XgALNq0T4qF3A\u0026id=XkhmBAABERAJ","remove_vol_from_collection_base_url":"http://www.google.com/patents?op=remove\u0026sig=ACfU3U0wSm9jh3CLEOuIw7JPgvoim4r2hA\u0026id=XkhmBAABERAJ","logged_in":false,"p13n_save_user_settings_url":"http://www.google.com/patents?op=edit_user_settings\u0026sig=ACfU3U0yPRW_-xRQNyyetnfJ8CGpDxl6QQ","is_cobrand":false,"sign_in_url":"https://www.google.com/accounts/Login?service=\u0026continue=http://www.google.com/patents%3Fhl%3Den\u0026hl=en","is_play_enabled":true}, {"volume_id":"","is_ebook":true,"volumeresult":{"has_flowing_text":false,"has_scanned_text":true,"can_download_pdf":false,"can_download_epub":false,"is_pdf_drm_enabled":false,"is_epub_drm_enabled":false,"download_pdf_url":"http://www.google.com/patents/download/Nucleotide_sequences_coding_polypeptide.pdf?id=XkhmBAABERAJ\u0026output=pdf\u0026sig=ACfU3U2xkVXLSvxTtGzinQgnplrhLFq8Lw"},"sample_url":"http://www.google.com/patents/reader?id=XkhmBAABERAJ\u0026printsec=frontcover\u0026output=reader\u0026source=gbs_atb_hover","is_browsable":true,"is_public_domain":true}, {});</script><div id="footer_table" style="font-size:83%;text-align:center;position:relative;top:20px;height:4.5em;margin-top:2em"><div style="margin-bottom:8px"><a href=http://www.google.com/><nobr>Google&nbsp;Home</nobr></a> - <a href=//www.google.com/patents/sitemap/><nobr>Sitemap</nobr></a> - <a href=http://www.google.com/googlebooks/uspto.html><nobr>USPTO Bulk Downloads</nobr></a> - <a href=/intl/en/privacy/><nobr>Privacy Policy</nobr></a> - <a href=/intl/en/policies/terms/><nobr>Terms of Service</nobr></a> - <a href=https://support.google.com/faqs/answer/2539193?hl=en><nobr>About Google Patents</nobr></a> - <a href="http://www.google.com/tools/feedback/intl/en/error.html" onclick="try{_OC_startFeedback({productId: '72792',locale: 'en'});return false;}catch(e){}"><nobr>Send Feedback</nobr></a></div><span>Data provided by IFI CLAIMS Patent Services</span><br><span >&copy;2012 Google</span></div> <script type="text/javascript">var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));</script><script type="text/javascript">var pageTracker = _gat._getTracker("UA-27188110-1");pageTracker._setCookiePath("/patents/");pageTracker._trackPageview();</script> </body></html>