<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN""http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html><head><title>Patent US6759243 - High affinity TCR proteins and methods - Google Patents</title><script>(function(){(function(){function e(a){this.t={};this.tick=function(a,c,b){var d=void 0!=b?b:(new Date).getTime();this.t[a]=[d,c];if(void 0==b)try{window.console.timeStamp("CSI/"+a)}catch(e){}};this.tick("start",null,a)}var a;window.performance&&(a=window.performance.timing);var f=a?new e(a.responseStart):new e;window.jstiming={Timer:e,load:f};if(a){var c=a.navigationStart,d=a.responseStart;0<c&&d>=c&&(window.jstiming.srt=d-c)}if(a){var b=window.jstiming.load;0<c&&d>=c&&(b.tick("_wtsrt",void 0,c),b.tick("wtsrt_",
"_wtsrt",d),b.tick("tbsd_","wtsrt_"))}try{a=null,window.chrome&&window.chrome.csi&&(a=Math.floor(window.chrome.csi().pageT),b&&0<c&&(b.tick("_tbnd",void 0,window.chrome.csi().startE),b.tick("tbnd_","_tbnd",c))),null==a&&window.gtbExternal&&(a=window.gtbExternal.pageT()),null==a&&window.external&&(a=window.external.pageT,b&&0<c&&(b.tick("_tbnd",void 0,window.external.startE),b.tick("tbnd_","_tbnd",c))),a&&(window.jstiming.pt=a)}catch(g){}})();})();
</script><link rel="stylesheet" href="/patents/css/_4ff636b3d23669b7103f3b3a3a18b4cd/kl_intl_patents_bundle.css" type="text/css" /><script src="/books/javascript/atb_4ff636b3d23669b7103f3b3a3a18b4cd__en.js"></script><script>function googleTranslateElementInit() {new google.translate.TranslateElement({pageLanguage: "en",gaTrack: true,gaId: "UA-27188110-1",multilanguagePage: true});}</script><script src="//translate.google.com/translate_a/element.js?cb=googleTranslateElementInit"></script><meta name="DC.type" content="Patent"><meta name="DC.title" content="High affinity TCR proteins and methods"><meta name="DC.contributor" content="David M. Kranz" scheme="inventor"><meta name="DC.contributor" content="K. Dane Wittrup" scheme="inventor"><meta name="DC.contributor" content="Phillip D. Holler" scheme="inventor"><meta name="DC.contributor" content="Board Of Trustees Of The University Of Illinois" scheme="assignee"><meta name="DC.date" content="2000-12-6" scheme="dateSubmitted"><meta name="DC.description" content="T cell receptors (TCRs) that have higher affinity for a ligand than wild type TCRs are provided. These high affinity TCRs are formed by mutagenizing a T cell receptor protein coding sequence to generate a variegated population of mutants of the T cell receptor protein coding sequence; transforming the T cell receptor mutant coding sequence into yeast cells; inducing expression of the T cell receptor mutant coding sequence on the surface of yeast cells; and selecting those cells expressing T cell receptor mutants that have higher affinity for the peptide/MHC ligand than the wild type T cell receptor protein. The high affinity TCRs can be used in place of an antibody or single chain antibody."><meta name="DC.date" content="2004-7-6" scheme="issued"><meta name="DC.relation" content="AU:195191" scheme="references"><meta name="DC.relation" content="EP:0067553:A2" scheme="references"><meta name="DC.relation" content="EP:0174759:A1" scheme="references"><meta name="DC.relation" content="EP:0194809" scheme="references"><meta name="DC.relation" content="EP:0278667:A2" scheme="references"><meta name="DC.relation" content="EP:0425004:A2" scheme="references"><meta name="DC.relation" content="EP:0479180:A2" scheme="references"><meta name="DC.relation" content="EP:0672754:A1" scheme="references"><meta name="DC.relation" content="US:3817837" scheme="references"><meta name="DC.relation" content="US:3850752" scheme="references"><meta name="DC.relation" content="US:3927193" scheme="references"><meta name="DC.relation" content="US:3939350" scheme="references"><meta name="DC.relation" content="US:3996345" scheme="references"><meta name="DC.relation" content="US:4275149" scheme="references"><meta name="DC.relation" content="US:4277437" scheme="references"><meta name="DC.relation" content="US:4331647" scheme="references"><meta name="DC.relation" content="US:4348376" scheme="references"><meta name="DC.relation" content="US:4361544" scheme="references"><meta name="DC.relation" content="US:4366241" scheme="references"><meta name="DC.relation" content="US:4444744" scheme="references"><meta name="DC.relation" content="US:4468457" scheme="references"><meta name="DC.relation" content="US:4640561" scheme="references"><meta name="DC.relation" content="US:4713332" scheme="references"><meta name="DC.relation" content="US:4831122" scheme="references"><meta name="DC.relation" content="US:4873190" scheme="references"><meta name="DC.relation" content="US:4874845" scheme="references"><meta name="DC.relation" content="US:4885248" scheme="references"><meta name="DC.relation" content="US:4923799" scheme="references"><meta name="DC.relation" content="US:4946778" scheme="references"><meta name="DC.relation" content="US:4970296" scheme="references"><meta name="DC.relation" content="US:5013650" scheme="references"><meta name="DC.relation" content="US:5024940" scheme="references"><meta name="DC.relation" content="US:5059413" scheme="references"><meta name="DC.relation" content="US:5101827" scheme="references"><meta name="DC.relation" content="US:5173410" scheme="references"><meta name="DC.relation" content="US:5185250" scheme="references"><meta name="DC.relation" content="US:5223409" scheme="references"><meta name="DC.relation" content="US:5225539" scheme="references"><meta name="DC.relation" content="US:5258289" scheme="references"><meta name="DC.relation" content="US:5258498" scheme="references"><meta name="DC.relation" content="US:5260203" scheme="references"><meta name="DC.relation" content="US:5299253" scheme="references"><meta name="DC.relation" content="US:5316922" scheme="references"><meta name="DC.relation" content="US:5316925" scheme="references"><meta name="DC.relation" content="US:5340921" scheme="references"><meta name="DC.relation" content="US:5348867" scheme="references"><meta name="DC.relation" content="US:5403484" scheme="references"><meta name="DC.relation" content="US:5411873" scheme="references"><meta name="DC.relation" content="US:5427908" scheme="references"><meta name="DC.relation" content="US:5466788" scheme="references"><meta name="DC.relation" content="US:5482858" scheme="references"><meta name="DC.relation" content="US:5498530" scheme="references"><meta name="DC.relation" content="US:5500360" scheme="references"><meta name="DC.relation" content="US:5510240" scheme="references"><meta name="DC.relation" content="US:5571698" scheme="references"><meta name="DC.relation" content="US:5580717" scheme="references"><meta name="DC.relation" content="US:5580961" scheme="references"><meta name="DC.relation" content="US:5601822" scheme="references"><meta name="DC.relation" content="US:5602242" scheme="references"><meta name="DC.relation" content="US:5614192" scheme="references"><meta name="DC.relation" content="US:5624817" scheme="references"><meta name="DC.relation" content="US:5627060" scheme="references"><meta name="DC.relation" content="US:5633447" scheme="references"><meta name="DC.relation" content="US:5635363" scheme="references"><meta name="DC.relation" content="US:5670353" scheme="references"><meta name="DC.relation" content="US:5723286" scheme="references"><meta name="DC.relation" content="US:5723309" scheme="references"><meta name="DC.relation" content="US:5723323" scheme="references"><meta name="DC.relation" content="US:5733743" scheme="references"><meta name="DC.relation" content="US:5738996" scheme="references"><meta name="DC.relation" content="US:5763192" scheme="references"><meta name="DC.relation" content="US:5763733" scheme="references"><meta name="DC.relation" content="US:5767260" scheme="references"><meta name="DC.relation" content="US:5780225" scheme="references"><meta name="DC.relation" content="US:5789208" scheme="references"><meta name="DC.relation" content="US:5814476" scheme="references"><meta name="DC.relation" content="US:5817483" scheme="references"><meta name="DC.relation" content="US:5824483" scheme="references"><meta name="DC.relation" content="US:5824514" scheme="references"><meta name="DC.relation" content="US:5837477" scheme="references"><meta name="DC.relation" content="US:5837500" scheme="references"><meta name="DC.relation" content="US:5840304" scheme="references"><meta name="DC.relation" content="US:5858657" scheme="references"><meta name="DC.relation" content="US:5861156" scheme="references"><meta name="DC.relation" content="US:5866344" scheme="references"><meta name="DC.relation" content="US:5866363" scheme="references"><meta name="DC.relation" content="US:5869620" scheme="references"><meta name="DC.relation" content="US:5871907" scheme="references"><meta name="DC.relation" content="US:5871974" scheme="references"><meta name="DC.relation" content="US:5882945" scheme="references"><meta name="DC.relation" content="US:5900476" scheme="references"><meta name="DC.relation" content="US:5948409" scheme="references"><meta name="DC.relation" content="US:5969108" scheme="references"><meta name="DC.relation" content="US:5977321" scheme="references"><meta name="DC.relation" content="US:6017732" scheme="references"><meta name="DC.relation" content="US:6027910" scheme="references"><meta name="DC.relation" content="US:6087096" scheme="references"><meta name="DC.relation" content="US:6114147" scheme="references"><meta name="DC.relation" content="US:6232099" scheme="references"><meta name="DC.relation" content="US:6319713" scheme="references"><meta name="DC.relation" content="US:6416971" scheme="references"><meta name="DC.relation" content="US:RE35500" scheme="references"><meta name="citation_reference" content="Ahlquist and Pacha, Physiol. Plant. 79:163-167 (1990)."><meta name="citation_reference" content="Ahlquist et al., &quot;Viral Vectors,&quot; Cold Spring Harbor Laboratory, New York 183-189 (1988)."><meta name="citation_reference" content="Alam et al., (Jun. 1996), &quot;T-cell-receptor Affinity and Thymocyte Positive Selection,&quot; Nature 381:616-620."><meta name="citation_reference" content="Alam, S.M. et al., (Feb. 1999), “Qualitative and Quantitative Differences in T Cell Receptor Binding of Agonist and Antagonist Ligands,” Immunity 10:227-237."><meta name="citation_reference" content="Alberti, S., (1996), “A high affinity T cell receptor?,” Immunol. Cell Biol. 74:292-297."><meta name="citation_reference" content="Al-Ramadi BK, et al., (1995)Lack of strict correlation of functional sensitation with the apparent affinity of MHC/peptide complexes for the TCR. J. Immunol.155: 662-673."><meta name="citation_reference" content="Altschul, S.F. et al., “ Gapped BLAST and PSI-BLAST: a new generation of protein database search programs” (Sep. 1997) Nucleic Acids Research 25(17):2278-3402."><meta name="citation_reference" content="Anand, R. et al. (1992), &quot;Progress in developing methylotrophic yeasts as expression systems.&quot; TIBTECH 10:413-417."><meta name="citation_reference" content="Baldwin, R.W. and Byers, V.S., (Editors) (1985), Monoclonal Antibodies for Cancer Detection and Therapy,London Academic Press pp. 159-179."><meta name="citation_reference" content="Barton et al., Plant Physiol. 85:1103-1109 (1987)."><meta name="citation_reference" content="Beeson, C. et al., (Aug. 1996), “Early Biochemical Signals Arise from Low Affinity TCR-Ligand Reactions at the Cell-Cell Interface,” J. Exp. Med. 184:777-782."><meta name="citation_reference" content="Bellio M, et al., (1994), The &amp;b complementarity determining region 1 of a major histocompatibility complex (MHC) class 1-restricted T cell receptor is involved in the recognition of peptide/MHC I and superantigen/MHC complex. J. Exp. Med.179: 1087-1089."><meta name="citation_reference" content="Bentley, G.A. and Mariuzza, R.A., (1996), &quot;The Structure of the T Cell Antigen Receptor,&quot; Annu. Rev. Immunol. 14:563-590."><meta name="citation_reference" content="Bevan, M.J., “In Thymic Selection, Peptide Diversity Gives and Takes Away” (Aug. 1997), Immunity 7:175-178."><meta name="citation_reference" content="Bird, RE, et al., (1988), Single-chain antigen-binding proteins. Science. 242: 423-426."><meta name="citation_reference" content="Bjorkman, P.J., (Apr. 1997), &quot;MHC Restriction in Three Dimensions: A View of T Cell Receptor/Ligand Interactions,&quot; Cell 89:167-170."><meta name="citation_reference" content="Boder, E.T. and Wittrup, K.D. (1995), &quot;A Yeast Surface Display System for in vitro Affinity Maturation of Antibodies,&quot; Protein Interactions, Jun. 1-4, 1995, Beckman Institute, University of Illinois, Urbana (Abstract Only)."><meta name="citation_reference" content="Boder, E.T. and Wittrup, K.D., (1997), &quot;Yeast surface display for screening combinatorial ploypeptide libraries.&quot; Nature Biotech,. 15(6):553-557."><meta name="citation_reference" content="Boder, E.T. and Wittrup, K.D., (Feb. 1998), “Optimal Screening of Surface-Displayed Polypeptide Libraries,” Biotech. Progress 14(1):55-62."><meta name="citation_reference" content="Boder, E.T. et al., “ Directed Evolution of Antibody Fragments with Monovalent Femtomolar Antigen-binding Affinity” (Sep. 2000) Proc. Nat&#39;l. Acad. Sci. USA 97(20):10701-10705."><meta name="citation_reference" content="Boder, E.T., et al., (2000), Yeast surface display for directed evolution of protein expression, affinity, and stability. Methods Enzymol328, 430-444."><meta name="citation_reference" content="Boniface, J.J. et al., (Sep. 1999), “Thermodynamics of T cell receptor binding to peptide-MHC: Evidence for a general mechanism of molecular scanning” Proc. Natl. Acad. Sci. USA 96:11446-11451."><meta name="citation_reference" content="Brodnicki, TC., (1996), Reactivity and epitope mapping of single-chain T cell receptors with monoclonal antibodies. Mol. Immunol.33:253-263."><meta name="citation_reference" content="Bruening, G., &quot;Comovirus group, C.M.I./A.A.B. Descriptions of plant viruses,&quot; No. 199. Wm. Culross and Son Ltd., Coupar Angus, Perthshire, Scotland. (1978)."><meta name="citation_reference" content="Buchwalder, A. et al., (1994), “Immunochemical and Molecular Analysis of Antigen Binding to Lipid Anchored and Soluble Forms of an MHC Independent Human α/β T Cell Receptor,” Mol. Immunol. 31(11):857-872."><meta name="citation_reference" content="Buckholz, R.G. and Gleeson, M.A.G. (1991), &quot;Yeast Systems for the Commercial Production of Heterologous Proteins,&quot; Bio/Technol. 9:1067-1072."><meta name="citation_reference" content="Butler and Mayo, &quot;Molecular architecture and assembly of tobacco mosaic virus particles,&quot; The molecular biology of the positive strand RNA Viruses, Academic Press, London:237-257 (1987)."><meta name="citation_reference" content="Callan, M.F. et al., (1995), “Selection of T cell receptor variable gene-encoded amino acids on the third binding site loop: a factor influencing variable chain selection in a T cell response,” Eur. J. Immunol. 25:1529-1534."><meta name="citation_reference" content="Cassidy and Nelson, Phytopathology 80:1037 (1990)."><meta name="citation_reference" content="Chapman et al., Plant Journal 2:549 (1992)."><meta name="citation_reference" content="Charoenvit et al., &quot;Inability of malaria vaccine to induce antibodies to a protective epitope within its sequence,&quot; Science 251:668-671 (1991)."><meta name="citation_reference" content="Charoenvit et al., &quot;Monoclonal, but not polyclonal, antibodies protect against Plasmodium yoeli sporozoites,&quot; J. Immunol. 146:1020-1025 (1991)."><meta name="citation_reference" content="Cheng, Y.C., “Relationship Between The Inhibition Constant (K1) and the Concentration of Inhibitor Which Causes 50 Per Cent Inhibition (I50) of an Enzymatic Reaction” (1973), Biochem. Pharm. 22:3099-3108."><meta name="citation_reference" content="Cho, B.K. et al., “A yeast surface display system for the discovery of ligands that trigger cell activation.” (Nov. 1988) J. Immunol Methods (Netherlands) 220:179-88."><meta name="citation_reference" content="Cho, BK, et al., (1995), Characterization of a single-chain antibody to the &amp;b-chain of the T cell receptor. J. Biol. Chem.270: 25819-25826."><meta name="citation_reference" content="Chung. S. et al., (Dec. 1994), “Functional three-domain single-chain T-cell receptors,” Proc. Natl. Acad. Sci. USA 91:12654-12658."><meta name="citation_reference" content="Citovsky and Zambryski, BioEssays 13:373-379 (1991)."><meta name="citation_reference" content="Clackson et al., (Apr. 1991), &quot;Making Antibody Fragments Using Phage Display Libraries,&quot; Nature 352:624-628."><meta name="citation_reference" content="Clackson, T. et al., (Aug. 1991) “Making antibody fragments using phage display libraries” Nature 352:624-628."><meta name="citation_reference" content="Cochran, et al., (2000), A diverse of oligomeric class II MHC-peptide complexes for probing T-cell receptor interactions. Chemistry &amp; Biology, vol. 7:683-696."><meta name="citation_reference" content="Corr M, et al., (1994), T cell receptor-MHC class I peptide interations: affinity, kinetics, and specificity. Science265: 946-949."><meta name="citation_reference" content="Cregg, J.M. et al. (1993), &quot;Recent Advances in the Expression of Foreign Genes in Pichia pastoris,&quot; Bio/Technol. 11:905-910."><meta name="citation_reference" content="Culver et al., in press, Virology."><meta name="citation_reference" content="Dal Porto, J. et al., (1993), “A soluble divalent class I major histocompatibility complex molecule inhibits alloreactive T cells at nanomolar concentrations” Proc. Natl. Acad. Sci. USA 90:6671-6675."><meta name="citation_reference" content="Davis et al., (Annual-1998), “Ligand Recognition by αβ T Cell Receptors,” Annu. Rev. Immunol. 16:523-544."><meta name="citation_reference" content="Davis, M.M. and Bjorkman, D.J., (Aug. 1988), “T-cell antigen receptor genes and T-cell recognition,” Nature 334:395-402."><meta name="citation_reference" content="Davis, M.M. and Chien, Y., (1993), “Topology and affinity of T-cell receptor mediated recognition of peptide-MHC complexes,” Curr. Opin. Immunol. 5:45-49."><meta name="citation_reference" content="Dawson and Hilf, Ann. Rev. Plant Physiol. Plant Mol. Biol. 43:527-555 (1992)."><meta name="citation_reference" content="Dawson et al., &quot;A tobacco mosaic virus-hybrid expresses and loses an added gene,&quot; Virology 172:285-292 (1989)."><meta name="citation_reference" content="Dawson et al., &quot;cDNA cloning of the complete genome of tobacco mosaic virus and production of infectious transcripts,&quot; Proc. Natl. Acad. Sci. USA 83:1832-1836 (1986)."><meta name="citation_reference" content="Dawson et al., &quot;Modifications of the tobacco mosaic virus coat protein gene affecting replication, movement, and symptomatology,&quot; Phytopathol. 78:783-789 (1988)."><meta name="citation_reference" content="Dawson, Adv. Virus Res. 38:307-342 (1990)."><meta name="citation_reference" content="Dawson, Virology 186:359-367 (1992)."><meta name="citation_reference" content="de Kruif, J. and Logtenberg, T., (Mar. 1996), “Leucine Zipper Dimerized Bivalent and Bispecific scFv Antibodies from a Semi-synthetic Antibody Phage Display Library,” Amer. Soc. Biochem. Mol. Biol. 271(13):7630-7634."><meta name="citation_reference" content="Deom et al., &quot;Plant Virus Movement Proteins,&quot; Cell 69:221-224 (1992)."><meta name="citation_reference" content="Deom et al., Science 237:389-394 (1987)."><meta name="citation_reference" content="Dolja et al., Proc. Natl. Acad. Sci. USA 89:10208 (1992)."><meta name="citation_reference" content="Donson et al., &quot;Systemic expression of a bacterial gene by a tobacco mosaic virus-based vector,&quot; Proc. Natl. Acad. Sci. USA 88:7204-7208 (1991)."><meta name="citation_reference" content="Dunsmuir et al., &quot;Stability of introduced genes and stability of expression,&quot; Plant Molecular Biology Manual, Kluwer Academic Publishers, Dordrecht, The Netherlands:C1:1-17 (1988)."><meta name="citation_reference" content="Eisen, H. N. et al., (1996),“ Antigen-Specific T-Cell Receptors And Their Reactions With Complexes Formed By Peptides With Major Histocompatibility Complex Proteins” Adv. Protein Chem. 49:1-56."><meta name="citation_reference" content="Engel I, et al., (1988), Site-directed mutations in the VDJ junctional region of a T cell receptor &amp;b chain cause changes in antigenic peptide recognition. Cell54: 473-484."><meta name="citation_reference" content="Faber, K.N. et al. (1995), &quot;Review: Methylotrophic Yeasts as Factories for the Production of Foreign Proteins,&quot; Yeast 11:1331-1344."><meta name="citation_reference" content="Faber, K.N. et al., (1995), “Review: Methylotrophic Yeasts as Factories for the Production of Foreign Proteins” Yeast 11:1331-1344."><meta name="citation_reference" content="Fitchen et al., &quot;Plant virus expressing hybrid coat protein with added murine epitope elicits autoantibody response,&quot; Vaccine 13:1051-1057 (1995)."><meta name="citation_reference" content="Foote, J. and Eisen, H.N., (Sep. 2000) “Breaking the affinity ceiling for antibodies and T cell receptors” PNAS 97(20):10679-10681."><meta name="citation_reference" content="Fremont et al., (1996), &quot;Biophysical Studies of T-cell Receptors and Their Ligands,&quot; Curr. Opin. Immunol. 8:93-100."><meta name="citation_reference" content="French et al., &quot;Bacterial ene inserted in an engineered RNA virus: Efficient expression in monocotyledonous plant cells,&quot; Science 231:1294-1297 (1986)."><meta name="citation_reference" content="Furukawa, k. et al., “Junctional Amino Acids Determine the Maturation Pathway of an Antibody” (Sep. 1999), Immunity 11:329-338."><meta name="citation_reference" content="Ganju, R.K. et al., (Dec. 1992), “Similarity between fluorescein-specific T-cell receptor and antibody in chemical details of antigen recognition,” Proc. Natl. Acad. Sci. USA 89:11552-11556."><meta name="citation_reference" content="Garcia, K.C. et al. “An αB T Cell Receptor Structure at 2.5 Åand Its Orientation in the TCR-MHC Complex” (Oct. 1996), Science 274:209-219."><meta name="citation_reference" content="Garcia, K.C. et al., (Dec. 1997), “αβ T cell receptor interactions with syngeneic and allogeneic ligands: Affinity measurements and crystallization,” Proc. Natl. Acad. Sci. USA 94:13838-13843."><meta name="citation_reference" content="Garcia, K.C. et al., “CD8 enhances formation of stable T-cell receptor/MHC class I molecule complexes” (Dec. 1996), Nature 384:577-584."><meta name="citation_reference" content="Garcia, K.C. et al., “Structural Basis of Plasticity in T Cell Receptor Recognition of a Self Peptide-MHC Antigen” (Feb. 1998) Science 279:1166-1172."><meta name="citation_reference" content="Gascoigne, N.R. et al., (May 1987), “Secretion of a chimeric T-cell receptor-immunoglobulin protein,” Proc. Natl. Acad. Sci. USA 84:2936-2940."><meta name="citation_reference" content="Geitz, R.D. et al., (1995), “Studies on the Transformation of Intact Yeast Cells by the LiAc/SS-DNA/PEG Procedure” Yeast 11:355-360."><meta name="citation_reference" content="Gellissen, G. et al., (1992) “Progress in developing methylotrophic yeasts as expression systems” Tibtech 10:413-417."><meta name="citation_reference" content="Gibbs, A.J., &quot;Tobamovirus group, C.M.I./A.A.B. Descriptions of plant viruses,&quot; No. 184. Wm. Culross and Son Ltd., Coupar Angus, Perthshire, Scotland (1977)."><meta name="citation_reference" content="Goelet et al., &quot;Nucleotide sequence of tobacco mosaic virus RNA,&quot; Proc. Natl. Acad. Sci. USA 79:5818-5822 (1982)."><meta name="citation_reference" content="Gooding, Jr., G.V., and Hebert, T.T., &quot;A simple technique for purification of tobacco mosaic virus in large quantities,&quot; Phytopathology 57:1285 (1967)."><meta name="citation_reference" content="Hamamoto et al., &quot;A new tobacco mosaic virus vector and its use for the systemic production of angiotensin-I-converting enzyme inhibitor in transgenic tobacco and tomato,&quot; Bio/Technology 11:930-932 (1993)."><meta name="citation_reference" content="Hare, B.J. et al., (Jun. 1999), “Structure, specificity and CDR mobility of a class II restricted single-chain T-cell receptor” Nat. Struct. Biol. 6:574-581."><meta name="citation_reference" content="Hawkins, R.E. et al. (1992), &quot;Selections of Phage Antibodies by Binding Affinity Mimicking Affinity Maturation,&quot; J. Mol. Biol. 226:889-896."><meta name="citation_reference" content="Hawkins, R.E. et al. (1993), &quot;The Contribution of Contact and Non-contact Residues of Antibody in the Affinity of Binding to Antigen,&quot; J. Mol. Biol. 234:958-964."><meta name="citation_reference" content="Haynes et al., &quot;Development of a genetically-engineered, candidate polio vaccine employing the self-assembling properties of the tobacco mosaic virus coat protein,&quot; Bio/Technology 4:637-641 (1986)."><meta name="citation_reference" content="Hilyard, K.L. et al., (Sep. 1994), “Binding of soluble natural ligands to a soluble human T-cell receptor fragment produced in Escherichia coli,” Pro. Natl. Acad. Sci. USA 91:9057-9061."><meta name="citation_reference" content="Holler, et al., (2000) In vitroevolution of a T cell receptor with high affinity for peptide/MHC, PNAS, 97:5387-5392."><meta name="citation_reference" content="Holler, et al., (2002) TCRs with high affinity for foreign pMHC show self-reactivity, Nature Immunology, Published online Dec. 9, 2002; doi:10.1038/ni863."><meta name="citation_reference" content="Holler, et al., (2003) Quantitative Analysis of the Contribution of TCR/pepMHC Affinity and CD8 to T Cell Activation, Immunity, 18:255-264."><meta name="citation_reference" content="Holler, P.D. et al., (May 2000), “In vitro evolution of a T cell receptor with high affinity for peptide/MHC,”PNAS 97(10):5387-5392."><meta name="citation_reference" content="Holler, Philip D., et al., (2001), CD8-T Cell Transfectants that Express a High Affinity T Cell Receptor Exhibit Enhanced Peptide-dependent Activation. J. Exp. Med.194: 1043-1052."><meta name="citation_reference" content="Hoogenboom, Hennie R., (1997) Designing and optimizing library selection strategies for generating high-affinity antibodies, Tibtech, 15:62-70."><meta name="citation_reference" content="Horsch et al., &quot;Leaf disc transformation,&quot; Plant Molecular Biology Manual, Kluwer Academic Publishers, Dordrecht, The Netherlands:A5:1-9 (1988)."><meta name="citation_reference" content="Huse, W. D. et al., (Dec. 1989), “Generation of a Large Combinatorial Library of the Immunoglobulin Repertoire in Phage Lambda,” Science 246:1275-1281."><meta name="citation_reference" content="Jagadish et al., &quot;High Level Production of Hybrid otybirus-like Particles Carrying Repetitive Copies of Foreign Antigens in Escherichia coli,&quot; Bio/Technology 11:1166-1170 (1993)."><meta name="citation_reference" content="Jorgensen, J.L. et al., (Jan. 1992), “Mapping T-cell receptor-peptide contacts by variant peptide immunization of single-chain transgenics,” Nature 355:224-230."><meta name="citation_reference" content="Joshi and Joshi, FEBS Letters 281:1-8 (1991)."><meta name="citation_reference" content="Joshi et al., &quot;BSMV genome mediated expression of a foreign gene in dicot and monoclonal plant cells,&quot; EMBO J. 9:2663-2669 (1990)."><meta name="citation_reference" content="Jung, S. and Plückthun (1997), &quot;Improving in vivo folding and stability of a single-chain Fv antibody fragment by loop grafting,&quot; Protein Eng. 10(8):959-966."><meta name="citation_reference" content="Jupin et al., Virology 178:273-280 (1990)."><meta name="citation_reference" content="Kappler, J. et al., (Aug. 1994), “Binding of a soluble αβ T-cell receptor to superantigen/major histocompatibility complex ligands,” Proc. Natl. Acad. Sci. USA 91:8462-8466."><meta name="citation_reference" content="Kasibhatla S. et al., (1993) Simultaneous involvement of all six predicted antigen binding loops of the T cell receptor in recognition of the MHC/antigenic peptide complex. J. Immunol.151:3140-51."><meta name="citation_reference" content="Kearny et al., Virology 192:000-000 (in press) (1993)."><meta name="citation_reference" content="Kieke, M.C. et al. (1997), &quot;Isolation of anti-T cell receptor scFv mutants by yeast surface display,&quot;Protein Eng. 10(11):1303-1310."><meta name="citation_reference" content="Kieke, M.C. et al., (May 1999), “Selection of functional T cell receptor mutants from a yeast surface-display library,” Proc. Natl. Acad. Sci. USA 96(10):5651-5656."><meta name="citation_reference" content="Kieke, M.C., et al., (2001), High affinity T cell receptors from yeast display libraries block T cell activation by superantigens. J Mol Biol 307:1305-1315."><meta name="citation_reference" content="Kipriyanov, S. M. et al., (Apr. 1997), “Two amino acid mutations in an anti-human CD3 single chain Fv antibody fragment that affect the yield on bacterial secretion but not the affinity,” Protein Eng. 10(4):445-453."><meta name="citation_reference" content="Klis, F.M. (1994). &quot;Review: Cell Wall Assembly in Yeast,&quot; Yeast 10:851-869."><meta name="citation_reference" content="Knappik, A. and Plückthun, A. (1995), &quot;Engineered turns of a recombinant antibody improve its in vivo folding,&quot; Protein Eng. 8(1):81-89."><meta name="citation_reference" content="Kowalski, J. M. et al., (Feb. 1998), “Secretion efficiency in Saccharomyces cerevisiae of bovine pancreatic trypsin inhibitor mutants lacking disulfide bonds is correlated with thermodynamic stability,” Biochemistry 37(5):1264-1273."><meta name="citation_reference" content="Kowalski, J.M. et al., (Jul. 1998), “Protein folding stability can determine the efficiency of escape from endoplasmic reticulum quality control,” J. Biol. Chem. 273(31):19453-19458."><meta name="citation_reference" content="Krebbers et al., &quot;Prospects and progress in the production of foreign proteins and peptides in plants,&quot; Plant Protein Engineering. (P.R. Shewry and S. Gutteridge, eds.), Cambridge University Press, Cambridge, pp. 315-325 (1992)."><meta name="citation_reference" content="Kumagai et al., &quot;Rapid, high level expression of biologically active alpha-trichosanthin in transfected plants by a novel RNA viral vector,&quot; Proc. Natl. Acad. Sci USA 90:427-430 (1993)."><meta name="citation_reference" content="Kumagai et al., “Rapid, high level expression of biologically active α-trichosanthin in transfected plants by a novel RNA viral vector,” Proc. Natl. Acad. Sci USA 90:427-430 (1993)."><meta name="citation_reference" content="Kumar, V. et al. (1997) &quot;Recombinant T Cell Receptor Molecules Can Prevent and Reverse Experimental Autoimmune Encephalomyelitis&quot; The Journal of Immunology 159:5150-5156."><meta name="citation_reference" content="Lake, D.F. et al. (Jan. 1999) &quot;Construction and binding analysis of recombinant single-chain TCR derived from tumor-infiltrating lymphocytes and a cytotoxic T lymphocyte clone directed against MAGE-1&quot; International Immunology 11:745-751."><meta name="citation_reference" content="Larkins et al., J. Cell. Biochem. Suppl. 0(9 Part C):264 (1985)."><meta name="citation_reference" content="Letourneur, F. and Malissen, B., (1989), “Derivation of a T cell hybridoma variant deprived of functional T cell receptor a and b chain transcripts reveals a nonfunctional a-mRNA of BW5147 origin” Eur. J. Immunol. 19(12):2269-2274."><meta name="citation_reference" content="Lipke, P.N. and Kurjan, J., (Mar. 1992), &quot;Sexual Agglutination in Budding Yeasts: Structure, Function, and Regulation of Adhesion Glycoproteins,&quot; Microbiological Reviews pp. 180-194."><meta name="citation_reference" content="Lyons et al., (Jul. 1996), &quot;A TCR Binds to Antagonist Ligands with Lower Affinities and Faster Dissociation Rates Than to Agonists,&quot; Immunity 5:53-61."><meta name="citation_reference" content="Malchiodi EL, (1995), Superantigen binding to a T cell receptor βchain of known three-dimensional structure. J. Exp. Med.182:1833-1845."><meta name="citation_reference" content="Manning et al., (Feb. 1999), “Effects of complementarity determining region mutations on the affinity of alpha/beta T cell receptor: measuring the energy associated with CD4/CD8 repertoire skewing,” J. Exp. Med. 189(3):461-470."><meta name="citation_reference" content="Manning, T. C. et al., (Apr. 1998), “Alanine Scanning Mutagenesis of an αβ T cell Receptor: Mapping the Energy of Antigen Recognition,” Immunity 8:413-425."><meta name="citation_reference" content="Margulies, D.H., (Jun. 1996), &quot;An Affinity for Learning,&quot; Nature 381:558-559."><meta name="citation_reference" content="Marks, J.D. et al., (1991), “By-passing Immunization. Human Antibodies from V-gene Libraries Displayed on Phage,” J. Mol. Biol. 222:581-597."><meta name="citation_reference" content="Marks, J.D. et al., (Aug. 1992), “Molecular Evolution of Proteins on Filamentous Phage,” J. Biol. Chem. 267(23):16007-16010."><meta name="citation_reference" content="Martelli, Plant Disease 76:436 (1992)."><meta name="citation_reference" content="Martineau, P. et al., (Jul. 1998), “Expression of an antibody fragment at high levels in the bacterial cytoplasm,” J. Mol. Biol. 280(1):117-127."><meta name="citation_reference" content="Marx, J. (Jan. 1995), &quot;The T Cell Receptor Begins to Reveal Its Many Facets,&quot; Science 267:459-460."><meta name="citation_reference" content="Mason et al., &quot;Expression of hepatitis B surface antigen in transgenic plants,&quot; Proc. Natl. Acad. Sci. USA 89:11745-11749 (1992)."><meta name="citation_reference" content="Matsui et al., (Dec. 1991), &quot;Low Affinity Interaction of Peptide-MHC Complexes with T Cell Receptors,&quot; Science 254:1788-1791."><meta name="citation_reference" content="Matsui et al., (Dec. 1994), &quot;Kinetics of T-cell Receptor Binding to Peptide/I-E&lt;k &gt;Complexes : Correlation of the Dissociation Rate with T-cell Responsiveness,&quot; Proc. Natl. Acad. Sci. USA 91:12862-12866."><meta name="citation_reference" content="Matsui et al., (Dec. 1994), “Kinetics of T-cell Receptor Binding to Peptide/I-Ek Complexes : Correlation of the Dissociation Rate with T-cell Responsiveness,” Proc. Natl. Acad. Sci. USA 91:12862-12866."><meta name="citation_reference" content="McCafferty, J. et al., (Dec. 1990), “Phage antibodies: filamentous phage displaying antibody variable domains,” Nature 348:552-554."><meta name="citation_reference" content="Nieba, L. et al. (1997). &quot;Disrupting the hydrophobic patches at the antibody variable/constant domain interface: improved in vivo folding and physical characterization of an engineered scFv fragment,&quot; Protein Eng. 10(4):435-444."><meta name="citation_reference" content="Nieba, L. et al., (Apr. 1997), “Disrupting the hydrophobic patches at the antibody variable/constant domain interface: improved in vivo folding and physical characterization of an engineered scFv fragment,” Protein Eng. 10(4):435-444."><meta name="citation_reference" content="Novotny, J. et al., (Oct. 1991), “A soluble, single-chain T-cell receptor fragment endowed with antigen-combining properties,” Proc. Natl. Acad. Sci. USA 88:8646-8650."><meta name="citation_reference" content="Ogawa et al., Virology 185:580-584 (1991)."><meta name="citation_reference" content="O&#39;Herrin et al., (Oct. 1997), “Analysis of the Expression of Peptide-Major Histocompatibility Complexes Using High Affinity Soluble Divalent T Cell Receptors,” J. Exp. Med 186:1333-1345."><meta name="citation_reference" content="O&#39;Herrin, S.M. et al., (Oct. 1997), “Analysis of the Expression of Peptide-Major Histocompatibility Complexes Using High Affinity Soluble Divalent T Cell Receptors,” J. Exp. Med 186:1333-1345."><meta name="citation_reference" content="Olsnes, S. and Pihl, A., “Chimeric Toxins” (1982), Pharmac. Ther. 25:355-381."><meta name="citation_reference" content="Ow et al., Science 234:856 (1986)."><meta name="citation_reference" content="Pelham, H.R.B., &quot;Leaky UAG termination codon in tobacco mosaic virus RNA,&quot; Nature 272:469-471 (1978)."><meta name="citation_reference" content="Porta et al., &quot;Development of Cowpea Mosaic Virus as a High-Yielding System for the Presentation of Foreign Peptides,&quot; Virology 202:949-955 (1994)."><meta name="citation_reference" content="Potrykus, Ann. Rev. Plant Physiol. Plant Mol. Biol. 42:205-225 (1991)."><meta name="citation_reference" content="Rabinowitz et al., (Feb. 1996), Kinetic discrimination in T-cell activation Proc. Natl. Acad. Sci. USA 93:1401-1405."><meta name="citation_reference" content="Raffo and Dawson, &quot;Construction of Tobacco Mosaic Virus Subgenomic Replicons that are Replicated and Spread Systemically in Tobacco Plants,&quot; Virology 184:277-289 (1991)."><meta name="citation_reference" content="Reich et al., (Jun. 1997), “Ligand-specific Oligomerization of T-cell Receptor Molecules,” Nature 387:617-620."><meta name="citation_reference" content="Ridder, R. et al. (1995), “Generation of Rabbit Monoclonal Antibody Fragments from a Combinatorial Phage Display Library and Their Production in the Yeast Pichia pastoris,” Bio/Technol. 13:255-259."><meta name="citation_reference" content="Romanos et al., (1992), “Foreign Gene Expression in Yeast: a Review,” Yeast 8:423-488."><meta name="citation_reference" content="Romanos, M. (1995), “Advances in the use of Pichia pastoris for high-level gene expression,” Curr. Opinion in Biotechnol. 6:527-533."><meta name="citation_reference" content="Rowlands et al., eds., American Press, London, pp. 237-257 (1987)."><meta name="citation_reference" content="Saito et al., Virology 176:329-336 (1990)."><meta name="citation_reference" content="Sant&#39; Angelo, D.B. et al., (Apr. 1996), “The Specificity and Orientation of a TCR to its Peptide-MHC Class II Ligands,” Immunity 4:367-376."><meta name="citation_reference" content="Schlueter et al., (1996), “Specificity and Binding Properties of a Single-chain T Cell Receptor,” J. Mol. Biol. 256:859-869."><meta name="citation_reference" content="Schlueter, C.J. et al., (1996), “A Residue in the Center of Peptide QL9 Affects Binding to Both Ld and the T Cell Receptor1,” J. Immunol. 157:4478-4485."><meta name="citation_reference" content="Schneck, J. et al., (Jan. 1989), “Inhibition of an Allospecific T Cell Hybridoma by Soluble Class I Proteins and Peptides: Estimation of the Affinity of a T Cell Receptor for MHC,” Cell 56:47-55."><meta name="citation_reference" content="Schodin et al., (1996), “Binding Properties and Solubility of Single-Chain T Cell Receptors Expressed in E. coli,” Molec. Immunol. 33(9):819-829."><meta name="citation_reference" content="Schodin, B.A. and Kranz, D.M., (Dec. 1993), “Binding Affinity and Inhibitory Properties of a Single-Chain Anti-T Cell Receptor Antibody,” J. Biol. Chem. 268(34):25722-25727."><meta name="citation_reference" content="Schodin, B.A. et al., (1996) “Binding Properties and solubility of single-chain T cell receptors expressed in E. coli” Immun.33(9):819-829."><meta name="citation_reference" content="Schreuder et al., (Apr. 1996), “Immobilizing Proteins on the Surface of Yeast Cells,” TIBTECH 14:115-120."><meta name="citation_reference" content="Seibel, J.L. et al., (Jun. 1997), “Influence of the NH2-terminal Amino Acid of the T Cell Receptor α Chain on Major Histocompatibility Complex (MHC) Class II + Peptide Recognition,” J. Exp. Med. 185(11):1919-1927."><meta name="citation_reference" content="Seth, A. et al., (May 1994), “Binary and ternary complexes between T-cell receptor, class II MHC and superantigen in vitro,” Nature 369:324-327."><meta name="citation_reference" content="Shaw, &quot;Chlorampheniol acetyltransferase from chloramphenicol-resistant bacteria,&quot; Methods in Enzymology 53:737-755 (1975)."><meta name="citation_reference" content="Sheets, M.D. et al., (May 1998), “Efficient construction of a large nonimmune phage antibody library: The production of high-affinity human single-chain antibodies to protein antigens,” Cell Biology 95(11):6157-6162."><meta name="citation_reference" content="Shusta, E.V. et al. (1999) &quot;Yeast Polypeptide Fusion Surface Display Levels Predict Thermal Stability and Soluble Secretion Efficiency&quot; Academic Press 292:949-956."><meta name="citation_reference" content="Shusta, E.V. et al., (Apr. 1999), “Biosynthetic polypeptide libraries,” Curr. Opin. Biotechnol. 10:117-122."><meta name="citation_reference" content="Shusta, E.V. et al., (Oct. 1999), “Yeast Polypeptide Fusion Surface Display Levels Predict Thermal Stability and Soluble Secretion Efficiency,” J. Mol. Biol. 292:949-956."><meta name="citation_reference" content="Shusta, E.V., et al., (2000), Directed evolution of a stable scaffold for T-cell receptor engineering. Nat Biotechnol 18:754-759."><meta name="citation_reference" content="Skuzeski et al., &quot;The signal for a leaky UAG stop codon in several plant viruses includes the two downstream codons,&quot; J. Mol. Biol. 218:365-373 (1991)."><meta name="citation_reference" content="Slanetz, A.E. and Bothwell, A.L., (1991), “Heterodimeric, disulfide-linked a / b T cell receptors in solution” Eur. J. Immunol. 21:179-183."><meta name="citation_reference" content="Soo Hoo, W.F. et al., (May 1992), “Characterization of a single-chain T-cell receptor expressed in Escherichia coli,” Proc. Natl. Acad. Sci. USA 89:4759-4763."><meta name="citation_reference" content="Speir, J.A. et al., (May 1998), “Structural Basis of 2C TCR Allorecognition of H-2Ld Peptide Complexes,” Immunity 8:553-562."><meta name="citation_reference" content="Sudbery, P.E. (1994), “The Non-Saccharomyces Yeasts,” Yeast 10:1707-1726."><meta name="citation_reference" content="Sykulev, Y. et al., (Dec. 1995) “The Law of mass action governs antigen-stimulated cytolytic activity of CD8+ cytotoxic T lymphocytes” Proc. Natl. Acad. Sci. USA 92:11990-11992."><meta name="citation_reference" content="Sykulev, Y. et al., (Nov. 1994), “High-affinity reactions between antigen-specific T-cell receptors and peptides associated with allogeneic and syngeneic major histocompatibility complex class I proteins,” Proc. Natl. Acad. Sci. USA 91:11487-11491."><meta name="citation_reference" content="Sykuley, Y. et al., “Kinetics and Affinity of Reactions between an Antigen-Specific T Cell Receptor and Peptide-MHC Complexes” (1994), Immunity 1:15-22."><meta name="citation_reference" content="Syrulev et al., (Dec. 1995), “The Law of Mass Action Governs Antigen-stimulated Cytolytic Activity of CD8+ cytotoxic T Lymphocytes,” Proc. Natl. Acad. Sci. USA92:11990-11992."><meta name="citation_reference" content="Takamatsu et al., &quot;Expression of bacterial chloramphenicol acetyltransferase gene in tobacco plants mediated by TMV-RNA,&quot; EMBO J. 6:307-311 (1987)."><meta name="citation_reference" content="Takamatsu et al., &quot;Production of enkephalin in tobacco protoplasts using tobacco mosaic virus RNA vector,&quot; FEBS Lett. 269:73-76 (1990)."><meta name="citation_reference" content="Takamatsu et al., J. Virol. 64:3686-3693 (1990)."><meta name="citation_reference" content="Takamatsu et al., J. Virol. 65:1619-1622 (1991)."><meta name="citation_reference" content="Tjoa, B.and Kranz, D.M., (Jul. 1992) “Diversity of T cell receptor-alpha chain transcripts from hyperimmune alloreactive T cells.” J Immunol (United States) 149(1) 253-259."><meta name="citation_reference" content="Turpen and Dawson, &quot;Amplification, movement and expression of genes in plants by viral-based vectors,&quot; Marcel Dekkar, New York, pp. 195-217."><meta name="citation_reference" content="Turpen and Dawson, &quot;Transgenic Plants, Fundamentals and Applications,&quot; Marcel Dekkar, New York, pp. 195-217 (1992)."><meta name="citation_reference" content="Turpen et al., &quot;Malarial Epitopes Expressed on the Surface of Recombinant Tobacco Mosaic Virus,&quot; Bio/Technology 10:53-57 (1995)."><meta name="citation_reference" content="Turpen, &quot;Ph.D. Dissertation,&quot; University of California, Riverside, pp. 106-132 (1992)."><meta name="citation_reference" content="Turpen, &quot;Ph.D. Dissertation,&quot; University of California, Riverside, pp. 72-87 (1992)."><meta name="citation_reference" content="Turpen, &quot;Ph.D. Dissertation,&quot; University of California, Riverside, pp. 85-105 (1992)."><meta name="citation_reference" content="Turpen, T.H., &quot;a Molecular Genetic Analysis of Host/Viral Interactions, Implications for the Use of Plant RNA Viruses as Gene Vectors,&quot; Chem. Abstracts 120(9):97427 (1992)."><meta name="citation_reference" content="Udaka, K. et al., (1993), “A ubiquitous protein is the source of naturally occurring peptides that are recognized by a CD8+ T-cell clone” Proc. Natl. Acad. Sci. USA 90:11272-11276."><meta name="citation_reference" content="Ulrich et al. (Dec. 1995), “Expression Studies of Catalytic Antibodies,” Proceed. Natl. Acad. Sci. 92:11907-11911."><meta name="citation_reference" content="Usha et al., &quot;Expression of an animal virus antigenic site on the surface of a plantvirus particle,&quot; Virology 197:366-374 (1993)."><meta name="citation_reference" content="Valitutti, S.et al., “Serial triggering of many T-cell receptors by a few peptide-MHC complexes” (May 1995), Nature 375:148-151."><meta name="citation_reference" content="van der Vaart (Sep. 1965), “Identification and Characterization of Cell Wall Proteins of Saccharomyces ceevisiae,” Thesis. ISBN 90-393-1498-5 pp. 1-138."><meta name="citation_reference" content="Van Haute et al., EMBO J. 2:411-417 (1983)."><meta name="citation_reference" content="Venton and Schell, NAR 13:6981 (1985)."><meta name="citation_reference" content="Von Kammen et al., &quot;Cowpea mosaic virus, C.M.I./A.A.B. Descriptions of plant viruses,&quot; No. 197, Wm. Culross and Son Ltd., Coupar Angus, Perthshire, Scotland, pp. 1-5 (1978)."><meta name="citation_reference" content="Walden and Schell, Eur. J. Biochem. 192:563-576 (1990)."><meta name="citation_reference" content="Ward, E.S., (1991), “Expression and Secretion of T-Cell Receptor Vα and Vβ Domains using Escherichia coli as a Host,” Scand. J. Immunol. 34:215-220."><meta name="citation_reference" content="Ward, E.S., (1992), “Secretion of T Cell Receptor Fragments From Recombinant Escherichia coli Cells,” J. Mol. Biol. 224:885-890."><meta name="citation_reference" content="Weber et al., (Apr. 1992), “Specific Low-affinity Recognition of Major Histocompatibility Complex Plus Peptide by Soluble T-cell Receptor,” Nature 356:793-796."><meta name="citation_reference" content="Wedemayer, G.J.et al., “Structural Insights into the Evolution of an Antibody Combining Site” (Jun. 1997), Science 276:1665-1669."><meta name="citation_reference" content="Weidanz, J.A. et al., (Dec. 1998), “Display of functional αβ single-chain T-cell receptor molecules on the surface of bacteriophage,” J. Immunol. Meth. 221:59-76."><meta name="citation_reference" content="Weidanz, Jon A. et al. (Aug. 1998), &quot;Display of functional alphabeta single-chain T-cell receptor molecules on the surface of bacteriophage&quot; Journal of Immunological Methods 221:59-76."><meta name="citation_reference" content="Weidanz, Jon A. et al. (Aug. 1998), “Display of functional αβ single-chain T-cell receptor molecules on the surface of bacteriophage” Journal of Immunological Methods 221:59-76."><meta name="citation_reference" content="Weiss et al., &quot;A T cell clone directed at the circumsporozoite protein which protects mice against both Plasmodium yoelli and Plasmodium berghei,&quot; J. Immunol. 149:2103-2109 (1992)."><meta name="citation_reference" content="Willcox, B.E. et al., “TCR Binding to Peptide-MHC Stabilizes a Flexible Recognition Interface” (Mar. 1999), Immunity 10:357-365."><meta name="citation_reference" content="Winter, G. et al., “Making Antibodies By Phage Display Technology” (1994) Annu. Rev. Immunol. 12:433-455."><meta name="citation_reference" content="Wittrup, D.K., “Phage on Display” (Nov. 1999), Trends in Biotech. 17(11):423-424."><meta name="citation_reference" content="Wittrup, K.D., (2000), The single cell as a microplate well. Nat Biotechnol 18:1039-1040."><meta name="citation_reference" content="Wolfe, M.S. et al., (Jan. 1998), A Substrate-Based Diffusion Ketone Selectively Inhibits Alzheimer&#39;s γ-Secretase Activity, J. Med. Chem. 41:6-9."><meta name="citation_reference" content="Yamaya et al., Mol. Gen. Genet. 211:520-525 (1988)."><meta name="citation_reference" content="Yoon, ST., (1994), Both high and low avidiry antibodies to the T cell receptor can have agonist or antagonist activity. Immunity1:563-569."><meta name="citation_reference" content="Zaitlin and Hull, Ann. Rev. Physiol. 38:291-315 (1987)."><meta name="citation_reference" content="Zaitlin et al., &quot;Tobacco mosaic virus (type strain), C.M.I./A.A.B. Descriptions of plant viruses,&quot; No. 151, Wm. Culross and Son Ltd., Coupar Angus, Perthshire, Scotland, pp. 1-6 (1975)."><meta name="citation_reference" content="Zambryski et al., EMBO J. 2:2143-2150 (1983)."><meta name="citation_patent_number" content="US:6759243"><meta name="citation_patent_application_number" content="US:09/731,242"><link rel="canonical" href="http://www.google.com/patents/US6759243"/><meta property="og:url" content="http://www.google.com/patents/US6759243"/><meta name="title" content="Patent US6759243 - High affinity TCR proteins and methods"/><meta name="description" content="T cell receptors (TCRs) that have higher affinity for a ligand than wild type TCRs are provided. These high affinity TCRs are formed by mutagenizing a T cell receptor protein coding sequence to generate a variegated population of mutants of the T cell receptor protein coding sequence; transforming the T cell receptor mutant coding sequence into yeast cells; inducing expression of the T cell receptor mutant coding sequence on the surface of yeast cells; and selecting those cells expressing T cell receptor mutants that have higher affinity for the peptide/MHC ligand than the wild type T cell receptor protein. The high affinity TCRs can be used in place of an antibody or single chain antibody."/><meta property="og:title" content="Patent US6759243 - High affinity TCR proteins and methods"/><meta property="og:type" content="book"/><meta property="og:site_name" content="Google Books"/><meta property="og:image" content="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><link rel="image_src" href="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><script>(function(){try{var aa=function(a,b,c,d){d=d||{};d._sn=["cfg",b,c].join(".");window.gbar.logger.ml(a,d)};var g=window.gbar=window.gbar||{},l=window.gbar.i=window.gbar.i||{},m={},n;function _tvn(a,b){var c=parseInt(a,10);return isNaN(c)?b:c}function _tvf(a,b){var c=parseFloat(a);return isNaN(c)?b:c}function _tvv(a){return!!a}function p(a,b,c){(c||g)[a]=b}g.bv={n:_tvn("2",0),r:"",f:".67.",e:"0",m:_tvn("0",1)};
function q(a,b,c){var d="on"+b;if(a.addEventListener)a.addEventListener(b,c,!1);else if(a.attachEvent)a.attachEvent(d,c);else{var f=a[d];a[d]=function(){var a=f.apply(this,arguments),b=c.apply(this,arguments);return void 0==a?b:void 0==b?a:b&&a}}}var s=function(a){return function(){return g.bv.m==a}},ba=s(1),ca=s(2);p("sb",ba);p("kn",ca);l.a=_tvv;l.b=_tvf;l.c=_tvn;l.i=aa;var da=window.gbar.i.i;var t,u,v,w;function ea(a){v=a}function fa(a){var b;if(b=v&&window.encodeURIComponent)b=a.href,b=!b.match(/^http[s]?:\/\/accounts\.google\.[^/]*\/ClearSID/i)&&!b.match(/^http[s]?:\/\/[^/]*\/accounts\/ClearSID/i);if(b=b&&encodeURIComponent(v()))a.href=a.href.replace(/([?&]continue=)[^&]*/,"$1"+b)}function ga(a){window.gApplication&&(a.href=window.gApplication.getTabUrl(a.href))}
function ha(a){var b=document.forms[0].q,c=window.encodeURIComponent&&b&&b.value,b=b&&b.placeholder;c&&c!=b&&(a.href=a.href.replace(/([?&])q=[^&]*|$/,function(a,b){return(b||"&")+"q="+encodeURIComponent(c)}))}n=l.a("")?ga:ha;
function x(a,b,c,d,f,e){var h=document.getElementById(a);if(h){var k=h.style;k.left=d?"auto":b+"px";k.right=d?b+"px":"auto";k.top=c+"px";k.visibility=u?"hidden":"visible";f&&e?(k.width=f+"px",k.height=e+"px"):(x(t,b,c,d,h.offsetWidth,h.offsetHeight),u=u?"":a)}}
var y=[],ia=function(a,b){y.push(b)},ja=function(a){a=a||window.event;var b=a.target||a.srcElement;a.cancelBubble=!0;null==t&&(a=document.createElement(Array.every||window.createPopup?"iframe":"div"),a.frameBorder="0",t=a.id="gbs",a.src="javascript:''",b.parentNode.appendChild(a),q(document,"click",z));var c=b,b=0;"gb3"!=c.className&&(c=c.parentNode);a=c.getAttribute("aria-owns")||"gbi";var d=c.offsetWidth,f=20<c.offsetTop?46:24;document.getElementById("tphdr")&&(f-=3);var e=!1;do b+=c.offsetLeft||
0;while(c=c.offsetParent);var c=(document.documentElement.clientWidth||document.body.clientWidth)-b-d,h,d=document.body,k=document.defaultView;k&&k.getComputedStyle?(d=k.getComputedStyle(d,""))&&(h=d.direction):h=d.currentStyle?d.currentStyle.direction:d.style.direction;h="rtl"==h;if("gbi"==a){for(d=0;k=y[d++];)k();A(null,window.navExtra);h&&(b=c,e=!0)}else h||(b=c,e=!0);u!=a&&z();x(a,b,f,e)},z=function(){u&&x(u,0,0)},A=function(a,b){var c,d=document.getElementById("gbi"),f=a;f||(f=d.firstChild);
for(;b&&(c=b.pop());){var e=d,h=c,k=f;w||(w="gb2");e.insertBefore(h,k).className=w}},ka=function(a,b,c){if((b=document.getElementById(b))&&a){a.className="gb4";var d=document.createElement("span");d.appendChild(a);d.appendChild(document.createTextNode(" | "));d.id=c;b.appendChild(d)}},la=function(){return document.getElementById("gb_70")},ma=function(){return!!u};p("qs",n);p("setContinueCb",ea);p("pc",fa);p("tg",ja);p("close",z);p("addLink",ka);p("almm",A);p("si",la);p("adh",ia);p("op",ma);var B=function(){},C=function(){},F=function(a){var b=new Image,c=D;b.onerror=b.onload=b.onabort=function(){try{delete E[c]}catch(a){}};E[c]=b;b.src=a;D=c+1},E=[],D=0;p("logger",{il:C,ml:B,log:F});var G=window.gbar.logger;var H={},na={},I=[],oa=l.b("0.1",.1),pa=l.a("1",!0),qa=function(a,b){I.push([a,b])},ra=function(a,b){H[a]=b},sa=function(a){return a in H},J={},K=function(a,b){J[a]||(J[a]=[]);J[a].push(b)},ta=function(a){K("m",a)},L=function(a,b){var c=document.createElement("script");c.src=a;c.async=pa;Math.random()<oa&&(c.onerror=function(){c.onerror=null;B(Error("Bundle load failed: name="+(b||"UNK")+" url="+a))});(document.getElementById("xjsc")||document.getElementsByTagName("body")[0]||
document.getElementsByTagName("head")[0]).appendChild(c)},N=function(a){for(var b=0,c;(c=I[b])&&c[0]!=a;++b);!c||c[1].l||c[1].s||(c[1].s=!0,M(2,a),c[1].url&&L(c[1].url,a),c[1].libs&&m.d&&m.d(c[1].libs))},O=function(a){K("gc",a)},P=null,ua=function(a){P=a},M=function(a,b,c){if(P){a={t:a,b:b};if(c)for(var d in c)a[d]=c[d];try{P(a)}catch(f){}}};p("mdc",H);p("mdi",na);p("bnc",I);p("qGC",O);p("qm",ta);p("qd",J);p("lb",N);p("mcf",ra);p("bcf",qa);p("aq",K);p("mdd","");p("has",sa);
p("trh",ua);p("tev",M);var Q=l.b("0.1",.001),R=0;
function _mlToken(a,b){try{if(1>R){R++;var c,d=a,f=b||{},e=encodeURIComponent,h=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&jexpid=",e("17483"),"&srcpg=",e("prop=22"),"&jsr=",Math.round(1/Q),"&ogev=",e("eGvoU7n9II62sQT5sYHYDA"),"&ogf=",g.bv.f,"&ogrp=",e("1"),"&ogv=",e("1407291699.0"),"&oggv="+e("es_plusone_gc_20140723.0_p0"),"&ogd=",e("com"),"&ogc=",e("NLD"),"&ogl=",e("en")];f._sn&&(f._sn="og."+
f._sn);for(var k in f)h.push("&"),h.push(e(k)),h.push("="),h.push(e(f[k]));h.push("&emsg=");h.push(e(d.name+":"+d.message));var r=h.join("");S(r)&&(r=r.substr(0,2E3));c=r;var Aa=window.gbar.logger._aem(a,c);F(Aa)}}catch(Na){}}var S=function(a){return 2E3<=a.length},va=function(a,b){return b};function T(a){B=a;p("_itl",S,G);p("_aem",va,G);p("ml",B,G);a={};H.er=a}l.a("")?T(function(a){throw a;}):l.a("1")&&Math.random()<Q&&T(_mlToken);I.push(["m",{url:"//ssl.gstatic.com/gb/js/scm_7385cc5883250b43a39405734c1bea59.js"}]);g.mcf("c",{});g.sg={c:""};if(l.a("1")){var wa=l.a("");I.push(["gc",{auto:wa,url:"//ssl.gstatic.com/gb/js/abc/gci_91f30755d6a6b787dcc2a4062e6e9824.js",libs:"googleapis.client:plusone:gapi.iframes"}]);var xa={version:"gci_91f30755d6a6b787dcc2a4062e6e9824.js",index:"",lang:"en"};H.gc=xa;var U=function(a){window.googleapis&&window.iframes?a&&a():(a&&O(a),N("gc"))};p("lGC",U);l.a("1")&&p("lPWF",U)};window.__PVT="";if(l.a("1")&&l.a("1")){var V=function(a){U(function(){K("pw",a);N("pw")})};p("lPW",V);I.push(["pw",{url:"//ssl.gstatic.com/gb/js/abc/pwm_45f73e4df07a0e388b0fa1f3d30e7280.js"}]);var W=[],ya=function(a){W[0]=a},za=function(a,b){var c=b||{};c._sn="pw";B(a,c)},Ba={signed:W,elog:za,base:"https://plusone.google.com/u/0",loadTime:(new Date).getTime()};H.pw=Ba;var X=function(a,b){for(var c=b.split("."),d=function(){var b=arguments;a(function(){for(var a=g,d=0,e=c.length-1;d<e;++d)a=a[c[d]];a[c[d]].apply(a,b)})},f=g,e=0,h=c.length-1;e<h;++e)f=
f[c[e]]=f[c[e]]||{};return f[c[e]]=d};X(V,"pw.clk");X(V,"pw.hvr");p("su",ya,g.pw)};function Ca(){function a(){for(var b;(b=e[h++])&&"m"!=b[0]&&!b[1].auto;);b&&(M(2,b[0]),b[1].url&&L(b[1].url,b[0]),b[1].libs&&m.d&&m.d(b[1].libs));h<e.length&&setTimeout(a,0)}function b(){0<f--?setTimeout(b,0):a()}var c=l.a("1"),d=l.a(""),f=3,e=I,h=0,k=window.gbarOnReady;if(k)try{k()}catch(r){da(r,"ml","or")}d?p("ldb",a):c?q(window,"load",b):b()}p("rdl",Ca);var Da={D:1,H:2,da:3,p:4,W:5,M:6,F:7,g:8,ha:9,U:10,L:11,T:12,S:13,N:14,Q:15,P:16,fa:17,w:18,O:19,ga:20,ea:21,u:22,G:23,ja:24,ka:25,ia:26,A:27,j:28,o:29,k:30,ca:31,Z:32,$:33,J:34,K:35,ba:36,aa:37,Y:38,B:39,R:40,v:41,X:42,V:43,h:48,C:49,I:500},Y=[1,2,3,4,5,6,9,10,11,13,14,28,29,30,34,35,37,38,39,40,41,42,43,48,49,500];var Z=l.b("0.001",1E-4),Ea=l.b("1",1),Fa=!1,Ga=!1;if(l.a("1")){var Ha=Math.random();Ha<=Z&&(Fa=!0);Ha<=Ea&&(Ga=!0)}var Ia=Da,$=null;function Ja(){var a=0,b=function(b,d){l.a(d)&&(a|=b)};b(1,"");b(2,"");b(4,"");b(8,"");return a}
function Ka(a,b){var c=Z,d=Fa,f;f=a;if(!$){$={};for(var e=0;e<Y.length;e++){var h=Y[e];$[h]=!0}}if(f=!!$[f])c=Ea,d=Ga;if(d){d=encodeURIComponent;g.rp?(f=g.rp(),f="-1"!=f?f:"1"):f="1";c=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&oge=",a,"&ogex=",d("17483"),"&ogev=",d("eGvoU7n9II62sQT5sYHYDA"),"&ogf=",g.bv.f,"&ogp=",d("22"),"&ogrp=",d(f),"&ogsr=",Math.round(1/c),"&ogv=",d("1407291699.0"),"&oggv="+
d("es_plusone_gc_20140723.0_p0"),"&ogd=",d("com"),"&ogl=",d("en"),"&ogc=",d("NLD"),"&ogus=",Ja()];if(b){"ogw"in b&&(c.push("&ogw="+b.ogw),delete b.ogw);var k;f=b;e=[];for(k in f)0!=e.length&&e.push(","),e.push(La(k)),e.push("."),e.push(La(f[k]));k=e.join("");""!=k&&(c.push("&ogad="),c.push(d(k)))}F(c.join(""))}}function La(a){"number"==typeof a&&(a+="");return"string"==typeof a?a.replace(".","%2E").replace(",","%2C"):a}C=Ka;p("il",C,G);var Ma={};H.il=Ma;setTimeout(function(){C(Ia.g)},0);}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var b=window.gbar.i.i;var c=window.gbar;var f=function(d){try{var a=document.getElementById("gbom");a&&d.appendChild(a.cloneNode(!0))}catch(e){b(e,"omas","aomc")}};c.aomc=f;}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var a=window.gbar;a.mcf("pm",{p:""});}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{window.gbar.rdl();}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
if (window['_OC_timingAction']) {window['_OC_timingAction']('patents_refpage');}</script><style>#gbar,#guser{font-size:13px;padding-top:1px !important;}#gbar{float:left;height:22px}#guser{padding-bottom:7px !important;text-align:right}.gbh,.gbd{border-top:1px solid #c9d7f1;font-size:1px}.gbh{height:0;position:absolute;top:24px;width:100%}#gbs,.gbm{background:#fff;left:0;position:absolute;text-align:left;visibility:hidden;z-index:1000}.gbm{border:1px solid;border-color:#c9d7f1 #36c #36c #a2bae7;z-index:1001}.gb1{margin-right:.5em}.gb1,.gb3{zoom:1}.gb2{display:block;padding:.2em .5em}.gb2,.gb3{text-decoration:none !important;border-bottom:none}a.gb1,a.gb4{text-decoration:underline !important}a.gb1,a.gb2,a.gb3,a.gb4{color:#00c !important}.gbi .gb3,.gbi .gb2,.gbi .gb4{color:#dd8e27 !important}.gbf .gb3,.gbf .gb2,.gbf .gb4{color:#900 !important}a.gb2:hover{background:#36c;color:#fff !important}#gbar .gbz0l{color:#000 !important;cursor:default;font-weight:bold;text-decoration:none !important}
#gbar { padding:.3em .6em !important;}</style></head><body  topmargin="3" marginheight="3"><div id=gbar><nobr><a onclick=gbar.qs(this);gbar.logger.il(1,{t:1}); class=gb1 id=gb_1 href="https://www.google.com/search?sa=N&tab=tw">Search</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:2}); class=gb1 id=gb_2 href="http://www.google.com/search?hl=en&tbm=isch&source=og&sa=N&tab=ti">Images</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:8}); class=gb1 id=gb_8 href="http://maps.google.com/maps?hl=en&sa=N&tab=tl">Maps</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:78}); class=gb1 id=gb_78 href="https://play.google.com/?hl=en&sa=N&tab=t8">Play</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:36}); class=gb1 id=gb_36 href="http://www.youtube.com/results?sa=N&tab=t1">YouTube</a> <a onclick=gbar.logger.il(1,{t:5}); class=gb1 id=gb_5 href="http://news.google.com/nwshp?hl=en&tab=tn">News</a> <a onclick=gbar.logger.il(1,{t:23}); class=gb1 id=gb_23 href="https://mail.google.com/mail/?tab=tm">Gmail</a> <a onclick=gbar.logger.il(1,{t:25}); class=gb1 id=gb_25 href="https://drive.google.com/?tab=to">Drive</a> <a class=gb3 href="http://www.google.com/intl/en/options/" onclick="this.blur();gbar.tg(event);return !1" aria-haspopup=true><u>More</u> <small>&#9660;</small></a><div class=gbm id=gbi><a onclick=gbar.logger.il(1,{t:24}); class=gb2 id=gb_24 href="https://www.google.com/calendar?tab=tc">Calendar</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:51}); class=gb2 id=gb_51 href="http://translate.google.com/?hl=en&sa=N&tab=tT">Translate</a><a onclick=gbar.logger.il(1,{t:17}); class=gb2 id=gb_17 href="http://www.google.com/mobile/?hl=en&tab=tD">Mobile</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:10}); class=gb2 id=gb_10 href="http://www.google.com/search?hl=en&tbo=u&tbm=bks&source=og&sa=N&tab=tp">Books</a><a onclick=gbar.logger.il(1,{t:212}); class=gb2 id=gb_212 href="https://wallet.google.com/manage/?tab=ta">Wallet</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:6}); class=gb2 id=gb_6 href="http://www.google.com/search?hl=en&tbo=u&tbm=shop&source=og&sa=N&tab=tf">Shopping</a><a onclick=gbar.logger.il(1,{t:30}); class=gb2 id=gb_30 href="http://www.blogger.com/?tab=tj">Blogger</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:27}); class=gb2 id=gb_27 href="http://www.google.com/finance?sa=N&tab=te">Finance</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:31}); class=gb2 id=gb_31 href="https://plus.google.com/photos?sa=N&tab=tq">Photos</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:12}); class=gb2 id=gb_12 href="http://www.google.com/search?hl=en&tbo=u&tbm=vid&source=og&sa=N&tab=tv">Videos</a><div class=gb2><div class=gbd></div></div><a onclick=gbar.logger.il(1,{t:66}); href="http://www.google.com/intl/en/options/" class=gb2>Even more &raquo;</a></div></nobr></div><div id=guser width=100%><nobr><span id=gbn class=gbi></span><span id=gbf class=gbf></span><span id=gbe></span><a target=_top id=gb_70 href="https://www.google.com/accounts/Login?service=&continue=http://www.google.com/patents%3Fhl%3Den&hl=en" class=gb4>Sign in</a><div style="display: none"><div class=gbm id=gbd5 aria-owner=gbg5><div class=gbmc><ol id=gbom class=gbmcc></ol></div></div></div></nobr></div><div class=gbh style=left:0></div><div class=gbh style=right:0></div><div role="alert" style="position: absolute; left: 0; right: 0;"><a href="http://www.google.com/patents/us6759243?hl=en&amp;output=html_text" title="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."><img border="0" src="http://www.google.com/images/cleardot.gif"alt="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."></a></div><div id="guser"><nobr></nobr></div><div style="clear:both;"></div><div id="gb-top-search-box" class="gb-top-search-box-small gb-reset"><table><tr><td class="logo"><a href="http://www.google.com/patents" class="logo-link"><img class="logo-img" src="/intl/en/images/logos/google_logo_41.png" alt="Go to Google Books Home" height="41"/></a></td><td><form action="http://www.google.com/search" name="f" id="vheadf" method="get"><span id="hf"></span><input type="hidden" name="tbm" value="pts"/><input type="hidden" name="tbo" value="1"/><input type="hidden" name="hl" value="en"/><table><tr><td><div class="inputs"><table><tr><td><div class="text-input"><input type="text" name="q" id="vheadq" class="text" maxlength="2048" size="31" value="" title="Search Patents" accesskey="s" autocomplete="off"/><script>window._OC_autoDir &&window._OC_autoDir('vheadq', 'tia-vheadq');</script></div></td><td><div class="submit-input"><input name="btnG" class="submit" type="submit" value=""/></div></td></tr></table></div></td><td class="col-ext-links"><div class="ext-links"><a href="http://www.google.com/advanced_patent_search">&lt;nobr&gt;Advanced Patent Search&lt;/nobr&gt;</a></div></td></tr></table></form></td></tr></table></div><div class="kd-appbar"><h2 class="kd-appname"><a href="/patents">Patents</a></h2><div class="kd-buttonbar left" id="left-toolbar-buttons"><a id="appbar-write-review-link" href=""></a><a id="appbar-view-print-sample-link" href=""></a><a id="appbar-view-ebook-sample-link" href=""></a><a id="appbar-patents-prior-art-finder-link" href="https://www.google.com/patents/related/US6759243"></a><a id="appbar-patents-discuss-this-link" href="http://www.google.com/url?id=W4xnBAABERAJ&amp;q=http://patents.stackexchange.com/redirect/google-patents%3Fpatent%3DUS6759243&amp;usg=AFQjCNEITFXlJB_LPnuG48vPRGmfdZzN2A" data-is-grant="true"></a><a id="appbar-read-patent-link" href="//docs.google.com/viewer?url=patentimages.storage.googleapis.com/pdfs/US6759243.pdf"></a><a id="appbar-download-pdf-link" href="//patentimages.storage.googleapis.com/pdfs/US6759243.pdf"></a><a class="appbar-application-grant-link" data-label="Application" href="/patents/US20020058253"></a><a class="appbar-application-grant-link" data-selected="true" data-label="Grant" href="/patents/US6759243"></a></div><div class="kd-buttonbar right" id="right-toolbar-buttons"></div></div><div id="books-microdata" itemscope=""itemtype="http://schema.org/Book"itemid="http://www.google.com/patents/US6759243" style="display:none"><span itemprop="description">T cell receptors (TCRs) that have higher affinity for a ligand than wild type TCRs are provided. These high affinity TCRs are formed by mutagenizing a T cell receptor protein coding sequence to generate a variegated population of mutants of the T cell receptor protein coding sequence; transforming the...</span><span itemprop="url">http://www.google.com/patents/US6759243?utm_source=gb-gplus-share</span><span class="main-title" itemprop="name">Patent US6759243 - High affinity TCR proteins and methods</span><img itemprop="image" src="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"alt="Patent US6759243 - High affinity TCR proteins and methods" title="Patent US6759243 - High affinity TCR proteins and methods"></div><div style="display: none"><ol id="ofe-gear-menu-contents" class="gbmcc"><li class="gbe gbmtc"><a class="gbmt goog-menuitem-content" id="" href="http://www.google.com/advanced_patent_search">Advanced Patent Search</a></li></ol></div><table id="viewport_table" cellpadding="0" style="clear:both" cellspacing="0"><tr><td id="viewport_td"><div class=vertical_module_list_row><div id=intl_patents class=about_content><div id=intl_patents_v><table class="patent-bibdata"><tr><td class="patent-bibdata-heading">Publication number</td><td class="single-patent-bibdata">US6759243 B2</td></tr><tr><td class="patent-bibdata-heading">Publication type</td><td class="single-patent-bibdata">Grant</td></tr><tr><td class="patent-bibdata-heading">Application number</td><td class="single-patent-bibdata">US 09/731,242</td></tr><tr><td class="patent-bibdata-heading">Publication date</td><td class="single-patent-bibdata">Jul 6, 2004</td></tr><tr><td class="patent-bibdata-heading">Filing date</td><td class="single-patent-bibdata">Dec 6, 2000</td></tr><tr><td class="patent-bibdata-heading">Priority date<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The priority date is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the date listed."></span></td><td class="single-patent-bibdata">Jan 20, 1998</td></tr><tr><td class="patent-bibdata-heading">Fee status<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The fee status is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the status or dates listed."></span></td><td class="single-patent-bibdata">Paid</td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Also published as</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/US7569357">US7569357</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20020058253">US20020058253</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20040146952">US20040146952</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20090275137">US20090275137</a></span></span></td></tr><tr class="patent-bibdata-list-row alternate-patent-number"><td class="patent-bibdata-heading">Publication number</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value">09731242, </span><span class="patent-bibdata-value">731242, </span><span class="patent-bibdata-value">US 6759243 B2, </span><span class="patent-bibdata-value">US 6759243B2, </span><span class="patent-bibdata-value">US-B2-6759243, </span><span class="patent-bibdata-value">US6759243 B2, </span><span class="patent-bibdata-value">US6759243B2</span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Inventors</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22David+M.+Kranz%22">David M. Kranz</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22K.+Dane+Wittrup%22">K. Dane Wittrup</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Phillip+D.+Holler%22">Phillip D. Holler</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Original Assignee</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=inassignee:%22Board+Of+Trustees+Of+The+University+Of+Illinois%22">Board Of Trustees Of The University Of Illinois</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Export Citation</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/US6759243.bibtex">BiBTeX</a>, </span><span class="patent-bibdata-value"><a href="/patents/US6759243.enw">EndNote</a>, </span><span class="patent-bibdata-value"><a href="/patents/US6759243.ris">RefMan</a></span></span></td></tr><tr class="patent-internal-links"><td colspan=2><span class="patent-bibdata-value"><a href="#backward-citations">Patent Citations</a> (102),</span> <span class="patent-bibdata-value"><a href="#npl-citations">Non-Patent Citations</a> (216),</span> <span class="patent-bibdata-value"><a href="#forward-citations">Referenced by</a> (5),</span> <span class="patent-bibdata-value"><a href="#classifications">Classifications</a> (24),</span> <span class="patent-bibdata-value"><a href="#legal-events">Legal Events</a> (7)</span> </td></tr><tr><td colspan=2 class="patent-bibdata-external-link-spacer-top"></td></tr><tr class="patent-bibdata-external-link-spacer-bottom"></tr><tr><td colspan=2><span class="patent-bibdata-heading">External Links:&nbsp;</span><span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=W4xnBAABERAJ&q=http://patft.uspto.gov/netacgi/nph-Parser%3FSect2%3DPTO1%26Sect2%3DHITOFF%26p%3D1%26u%3D/netahtml/PTO/search-bool.html%26r%3D1%26f%3DG%26l%3D50%26d%3DPALL%26RefSrch%3Dyes%26Query%3DPN/6759243&usg=AFQjCNFxmfK3mgxzRSIie3MJ_fZVEQkSvA">USPTO</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=W4xnBAABERAJ&q=http://assignments.uspto.gov/assignments/q%3Fdb%3Dpat%26pat%3D6759243&usg=AFQjCNHBkm4zk2AcDq9SV4bIUOEujY0c8A">USPTO Assignment</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=W4xnBAABERAJ&q=http://worldwide.espacenet.com/publicationDetails/biblio%3FCC%3DUS%26NR%3D6759243B2%26KC%3DB2%26FT%3DD&usg=AFQjCNFIW_MCoDCBg-Qm61KRQdD04egTBg">Espacenet</a></span></span></td></tr><tr class="patent-bibdata-group-spacer"></tr></table><div class="number-and-title"><span class="patent-title"><invention-title mxw-id="PT55295926" lang="EN" load-source="patent-office">High affinity TCR proteins and methods</invention-title></span><br><span class="patent-number">US 6759243 B2</span></div><div class="patent-section patent-abstract-section"><div class="patent-section-header"><span class="patent-section-title">Abstract</span></div><div class="patent-text"><abstract mxw-id="PA50691338" lang="EN" load-source="patent-office"> <div class="abstract">T cell receptors (TCRs) that have higher affinity for a ligand than wild type TCRs are provided. These high affinity TCRs are formed by mutagenizing a T cell receptor protein coding sequence to generate a variegated population of mutants of the T cell receptor protein coding sequence; transforming the T cell receptor mutant coding sequence into yeast cells; inducing expression of the T cell receptor mutant coding sequence on the surface of yeast cells; and selecting those cells expressing T cell receptor mutants that have higher affinity for the peptide/MHC ligand than the wild type T cell receptor protein. The high affinity TCRs can be used in place of an antibody or single chain antibody.</div>
  </abstract></div></div><div class="patent-section patent-drawings-section"><div class="patent-section-header"><span class="patent-section-title">Images<span class="patent-section-count">(15)</span></span></div><div class="patent-drawings-body"><div class="patent-drawings-carousel"><div class="drawings"><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6759243B2/US06759243-20040706-D00001.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6759243B2/US06759243-20040706-D00001.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6759243B2/US06759243-20040706-D00002.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6759243B2/US06759243-20040706-D00002.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6759243B2/US06759243-20040706-D00003.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6759243B2/US06759243-20040706-D00003.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6759243B2/US06759243-20040706-D00004.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6759243B2/US06759243-20040706-D00004.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6759243B2/US06759243-20040706-D00005.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6759243B2/US06759243-20040706-D00005.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6759243B2/US06759243-20040706-D00006.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6759243B2/US06759243-20040706-D00006.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6759243B2/US06759243-20040706-D00007.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6759243B2/US06759243-20040706-D00007.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6759243B2/US06759243-20040706-D00008.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6759243B2/US06759243-20040706-D00008.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6759243B2/US06759243-20040706-D00009.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6759243B2/US06759243-20040706-D00009.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6759243B2/US06759243-20040706-D00010.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6759243B2/US06759243-20040706-D00010.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6759243B2/US06759243-20040706-D00011.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6759243B2/US06759243-20040706-D00011.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6759243B2/US06759243-20040706-D00012.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6759243B2/US06759243-20040706-D00012.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6759243B2/US06759243-20040706-D00013.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6759243B2/US06759243-20040706-D00013.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6759243B2/US06759243-20040706-D00014.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6759243B2/US06759243-20040706-D00014.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6759243B2/US06759243-20040706-D00015.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6759243B2/US06759243-20040706-D00015.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div></div></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img" alt="Previous page"src="/googlebooks/images/kennedy/page_left.png"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img" alt="Next page"src="/googlebooks/images/kennedy/page_right.png"width="21" height="21" /></div></div></div><div class="patent-post-drawings"></div><div class="patent-section patent-claims-section"><div class="patent-section-header"><span class="patent-section-title">Claims<span class="patent-section-count">(23)</span></span></div><div class="patent-text"><div mxw-id="PCLM8699891" lang="EN" load-source="patent-office" class="claims">
    <claim-statement>We claim: </claim-statement> <div class="claim"> <div num="1" id="US-6759243-B2-CLM-00001" class="claim">
      <div class="claim-text">1. A soluble T cell receptor (TCR) having high affinity for a ligand exhibiting a dissociation constant for that ligand greater than about 10 <sup>7</sup>M<sup>−1</sup>, wherein said TCR is a mutant TCR carrying one or more mutations in a CDR.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="2" id="US-6759243-B2-CLM-00002" class="claim">
      <div class="claim-text">2. The soluble T cell receptor of <claim-ref idref="US-6759243-B2-CLM-00001">claim 1</claim-ref> wherein the TCR exhibits a dissociation constant for the ligand between about 10<sup>7 </sup>to 10<sup>10</sup>M<sup>−1</sup>.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="3" id="US-6759243-B2-CLM-00003" class="claim">
      <div class="claim-text">3. The soluble TCR of <claim-ref idref="US-6759243-B2-CLM-00001">claim 1</claim-ref> wherein the ligand is a peptide/MHC ligand.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="4" id="US-6759243-B2-CLM-00004" class="claim">
      <div class="claim-text">4. The soluble TCR of <claim-ref idref="US-6759243-B2-CLM-00001">claim 1</claim-ref> wherein the ligand is a superantigen.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="5" id="US-6759243-B2-CLM-00005" class="claim">
      <div class="claim-text">5. The soluble TCR of <claim-ref idref="US-6759243-B2-CLM-00001">claim 1</claim-ref> which is a mutant TCR carrying one or more mutations in a CDR.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="6" id="US-6759243-B2-CLM-00006" class="claim">
      <div class="claim-text">6. The soluble TCR of <claim-ref idref="US-6759243-B2-CLM-00005">claim 5</claim-ref> wherein the one or more mutations are in CDR3α or CDR3β.</div>
    </div>
    </div> <div class="claim"> <div num="7" id="US-6759243-B2-CLM-00007" class="claim">
      <div class="claim-text">7. A soluble mutant T cell receptor (TCR) having high affinity for a ligand and having one or more mutations in a CDR.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="8" id="US-6759243-B2-CLM-00008" class="claim">
      <div class="claim-text">8. The soluble TCR of <claim-ref idref="US-6759243-B2-CLM-00007">claim 7</claim-ref> wherein the one or more mutations are in CDR3α or CDR3β.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="9" id="US-6759243-B2-CLM-00009" class="claim">
      <div class="claim-text">9. The soluble TCR of <claim-ref idref="US-6759243-B2-CLM-00007">claim 7</claim-ref> wherein the ligand is a peptide/MHC ligand.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="10" id="US-6759243-B2-CLM-00010" class="claim">
      <div class="claim-text">10. The soluble TCR of <claim-ref idref="US-6759243-B2-CLM-00007">claim 7</claim-ref> wherein the ligand is a superantigen.</div>
    </div>
    </div> <div class="claim"> <div num="11" id="US-6759243-B2-CLM-00011" class="claim">
      <div class="claim-text">11. An isolated T cell expressing on its surface high affinity TCR's exhibiting a dissociation constant greater than 10<sup>7 </sup>M<sup>−1 </sup>for a selected ligand, wherein said TCR is a mutant TCR carrying one or more mutations in a CDR.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="12" id="US-6759243-B2-CLM-00012" class="claim">
      <div class="claim-text">12. The T cell of <claim-ref idref="US-6759243-B2-CLM-00011">claim 11</claim-ref> wherein the TCR exhibits a dissociation constant between about 10<sup>7 </sup>to 10<sup>10 </sup>M<sup>−1 </sup>for said ligand.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="13" id="US-6759243-B2-CLM-00013" class="claim">
      <div class="claim-text">13. The T cell of <claim-ref idref="US-6759243-B2-CLM-00011">claim 11</claim-ref> wherein the ligand is a peptide/MHC ligand.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="14" id="US-6759243-B2-CLM-00014" class="claim">
      <div class="claim-text">14. The T cell of <claim-ref idref="US-6759243-B2-CLM-00011">claim 11</claim-ref> wherein the ligand is a superantigen.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="15" id="US-6759243-B2-CLM-00015" class="claim">
      <div class="claim-text">15. The T cell of <claim-ref idref="US-6759243-B2-CLM-00011">claim 11</claim-ref> wherein the high affinity TCR is a mutant carrying one or more mutations in a CDR.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="16" id="US-6759243-B2-CLM-00016" class="claim">
      <div class="claim-text">16. The T cell of <claim-ref idref="US-6759243-B2-CLM-00011">claim 11</claim-ref> wherein the high affinity TCR is a mutant carrying one or more mutations in CDR3α or CDR3β.</div>
    </div>
    </div> <div class="claim"> <div num="17" id="US-6759243-B2-CLM-00017" class="claim">
      <div class="claim-text">17. An isolated T cell expressing on its surface a high affinity TCR carrying one or more mutations in a CDR.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="18" id="US-6759243-B2-CLM-00018" class="claim">
      <div class="claim-text">18. The T cell of <claim-ref idref="US-6759243-B2-CLM-00017">claim 17</claim-ref> wherein the high affinity TCR is a mutant carrying one or more mutations in a CDR3α or CDR3β.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="19" id="US-6759243-B2-CLM-00019" class="claim">
      <div class="claim-text">19. The T cell of <claim-ref idref="US-6759243-B2-CLM-00017">claim 17</claim-ref> wherein the ligand is a peptide/MHC ligand.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="20" id="US-6759243-B2-CLM-00020" class="claim">
      <div class="claim-text">20. The T cell of <claim-ref idref="US-6759243-B2-CLM-00017">claim 17</claim-ref> wherein the ligand is a superantigen.</div>
    </div>
    </div> <div class="claim"> <div num="21" id="US-6759243-B2-CLM-00021" class="claim">
      <div class="claim-text">21. A soluble mutant T cell receptor (TCR) having an high affinity for a ligand and having one or more mutations in a CDR where the TCR exhibits a dissociation constant for the ligand from about 10<sup>10</sup>M<sup>−1</sup>.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="22" id="US-6759243-B2-CLM-00022" class="claim">
      <div class="claim-text">22. The soluble TCR of <claim-ref idref="US-6759243-B2-CLM-00021">claim 21</claim-ref> wherein the one or ore mutations are in CDR3α or CDR3β.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="23" id="US-6759243-B2-CLM-00023" class="claim">
      <div class="claim-text">23. The soluble TCR of <claim-ref idref="US-6759243-B2-CLM-00021">claim 21</claim-ref> wherein the ligand is a peptide/HMC ligand.</div>
    </div>
  </div> </div></div></div><div class="patent-section patent-description-section"><div class="patent-section-header"><span class="patent-section-title">Description</span></div><div class="patent-text"><div mxw-id="PDES54264059" lang="EN" load-source="patent-office" class="description">
    <heading>CROSS REFERENCE TO RELATED APPLICATIONS</heading> <p>This application is a continuation-in-part of U.S. application Ser. No. 09/009,388, filed Jan. 20, 1998. This application claims the benefit of U.S. Provisional Application No. 60/169,179, filed Dec. 6, 1999.</p>
    <heading>ACKNOWLEDGMENT OF FEDERAL RESEARCH SUPPORT</heading> <p>This invention was made with Government support under Contract No.PHS-5-R01-GM55767-03 awarded by the National Institutes of Health (NIH). The Government has certain rights in this invention.</p>
    <heading>BACKGROUND OF THE INVENTION</heading> <p>The field of the present invention is molecular biology, in particular, as it is related to combinatorial libraries of immune cell receptors displayed on the cell surface of a recombinant host cell. More specifically, the present invention relates to a library of high affinity T cell receptor proteins displayed on the surfaces of recombinant yeast cells, to soluble high affinity TCR receptor proteins, to high affinity TCR proteins selected for high affinity binding to particular peptide/MHC pairs, to high affinity TCR proteins selected for binding to a particular antigen in the absence of an MHC determinant, and to the use of the selected high affinity TCR derivatives in diagnostic methods and imaging assays, among other applications.</p>
    <p>T cell receptors (TCRs) and antibodies have evolved to recognize different classes of ligands. Antibodies function as membrane-bound and soluble proteins that bind to soluble antigens, whereas in nature, TCRs function only as membrane-bound molecules that bind to cell-associated peptide/MHC antigens. All of the energy of the antibody:antigen interaction focuses on the foreign antigen, whereas a substantial fraction of the energy of the TCR peptide/MHC interaction seems to be directed at the self-MHC molecule [Manning et al. (1998) <i>Immunity </i>8:413:425]. In addition, antibodies can have ligand-binding affinities that are orders of magnitude higher than those of TCRs, largely because of the processes of somatic mutation and affinity maturation. In their normal cellular context, TCRs do not undergo somatic mutation, and the processes of thymic selection seem to operate by maintaining a narrow window of affinities [Alam et al. (1996) <i>Nature </i>381:616-620]. The association of TCRs at the cell surface with the accessory molecules CD4 or CD8 also may influence the functional affinity of TCRs [Garcia et al. (1996) <i>Nature </i>384:577-581]. Despite these differences, the three-dimensional structures of the two proteins are remarkably similar, with the hypervariable regions forming loops on a single face of the molecule that contacts the antigen.</p>
    <p>Based on their structural similarities, it is somewhat surprising that there have been significant differences in the success of producing soluble and surface-displayed forms of the extracellular domains of TCRs and antibodies in heterologous expression systems. Many antibodies have now been expressed at high yield and solubility as either intact or Fab-fragment forms or as single-chain (sc) fragment-variable (Fv) proteins. In addition, there are numerous antigen-binding Fv fragments that have been isolated de novo and/or improved through the use of phage-display technology and, more recently, with yeast-display technology [Boder and Wittrup (1997) <i>Nat. Biotechnol. </i>15:553-557; Kieke et al. (1997) <i>Prot. Eng. </i>10:1303-1310]. These expression systems for antibody fragments have been key in structural studies and in the design of diagnostic and therapeutic antibodies.</p>
    <p>In contrast, the three-dimensional structures of a few TCR molecules were determined only after considerable effort on the expression of soluble, properly folded TCRs [Bentley and Mariuzza (1996) <i>Ann. Rev. Immunol. </i>14:563-590]. One of the difficulties in exploring the basis of differences between Fab and TCR has been that the extensive sequence diversity in antibody and TCR variable (V) regions complicates efforts to discern what features of the V regions are important for functions other than antigen binding (e.g., V region pairing and association kinetics, stability, and folding). There have been relatively few studies that have compared the V regions of TCRs and antibodies in terms of these properties.</p>
    <p>Nevertheless, the TCR from the mouse T cell clone 2C has now been expressed as an sc V<sub>α</sub>V<sub>β</sub>(scTCR) in <i>Escherichia coli </i>[Soo Hoo et al. (1992) <i>Proc. Natl. Acad. Sci. USA </i>89:4759-4763], as a lipid-linked V<sub>α</sub>C<sub>α</sub>V<sub>β</sub>C<sub>β</sub> dimer from myeloma cells [Slanetz and Bothwell (1991) <i>Eur. J. Immunol. </i>21:179-183], and as a secreted V<sub>α</sub>C<sub>α</sub>V<sub>βC</sub> <sub>β</sub> dimer from insect cells [Garcia et al. (1996) <i>Science </i>274:209-219]. The 2C scTCR had relatively low solubility compared with most scFv, although its solubility is increased about 10-fold by fusion at the amino terminus to thioredoxin [Schodin et al. (1996) <i>Molec. Immunol. </i>33:819-829]. The difficulty in generating soluble, properly folded V<sub>α</sub>V<sub>β</sub> domains has extended to other TCRs [Udaka et al. (1993) supra; Sykulev et al. (1994) supra; Manning et al. (1998) supra]. The molecular explanation for the apparent differences between TCR and Fv in either solubility or surface-display capability has not been explored adequately. It has been shown that the 2C scTCR can be expressed in a yeast surface-display system after the selection, from a random library, of specific single-site mutations at the V<sub>α</sub>/V<sub>β</sub> interface or in a region of the V<sub>β</sub> framework suspected to interact with the CD3<sub>ε</sub> signal-transduction sub-unit. These mutations, several of which are found naturally in antibody V regions, reflect the significance of these positions in the TCR and provide a basis for further engineering of TCR-binding properties.</p>
    <heading>SUMMARY OF THE INVENTION</heading> <p>The invention provides a combinatorial library of immune T cell receptor polypeptides displayed on the surfaces of recombinant host cells, for example, yeast cells, desirably <i>Saccharomyces cerevisiae</i>. From such a library can be isolated high affinity TCR polypeptides (those that exhibit higher affinity than wild type for the cognate ligand: a complex of peptide bound to a protein of the major histocompatibility complex, pMHC). Desirably, the affinity of the TCR peptide for the pMHC is reflected in a dissociation constant of from about 10<sup>7 </sup>to about 10<sup>10</sup>, e.g., as measured by methods known to the art. A DNA library comprising nucleic acids encoding soluble high affinity TCRs, wherein said TCRs are made by the method of mutagenizing a TCR to create mutant TCR coding sequences; transforming DNA comprising the mutant TCR coding sequences for mutant TCRs into yeast cells; inducing expression of the mutant TCR coding sequences such that the mutant TCRs are displayed on the surface of yeast cells; contacting the yeast cells with a fluorescent label which binds to the peptide/MHC ligand to produce selected yeast cells; and isolating the yeast cells showing the highest fluorescence is provided. Also provided is a library of T cell receptor proteins displayed on the surface of yeast cells which have higher affinity for the peptide/MHC ligand than the wild type T cell receptor protein, wherein said library is formed by mutagenizing a T cell receptor protein coding sequence to generate a variegated population of mutants of the T cell receptor protein coding sequence; transforming the T cell receptor mutant coding sequence into yeast cells; inducing expression of the T cell receptor mutant coding sequence on the surface of yeast cells; and selecting those cells expressing T cell receptor mutants that have higher affinity for the peptide/MHC ligand than the wild type T cell receptor protein.</p>
    <p>The present invention further provides TCR proteins (in cell-bound or in soluble form) that exhibit high affinity binding for the cognate ligand. In the present invention the ligand bound by the TCR protein can be a peptide/MHC complex or because of the selection process, desirably an iterated selection process, it can be a ligand which does not include an MHC component, such as a superantigen. This ligand can be a peptide, a protein, a carbohydrate moiety, or a lipid moiety, among others. These soluble high affinity TCRs may be made by the method comprising: mutagenizing a TCR to create mutant TCR coding sequences; transforming DNA comprising the mutant TCR coding sequences for mutant TCRs into yeast cells; inducing expression of the mutant TCR coding sequences such that the mutant TCRs are displayed on the surface of yeast cells; contacting the yeast cells with a fluorescent label which binds to the peptide/MHC ligand to produce selected yeast cells; and isolating the yeast cells showing the highest fluorescence. The soluble high affinity TCRs are preferably isolated by yeast display.</p>
    <p>The present invention further provides methods for detecting the cognate ligand of a high affinity TCR protein, said methods comprising the step of binding the high affinity TCR protein with the cognate ligand, where the high affinity TCR protein is detectably labeled or where there is a secondary detectable protein added, such as an antibody specific for the TCR in a region other than the region which binds the cognate ligand. A preferred method for using high affinity TCRs to identify ligands comprises: labeling high affinity TCRs with a detectable label; contacting said labeled TCRs with ligands; identifying the ligand with which the labeled TCR is bound. Preferably the ligands are those peptide/MHC ligands to which the TCR binds with higher affinity than the wild type TCR. Methods of identifying the ligand are known to one of ordinary skill in the art. Suitable labels allowing for detection of the TCR protein, directly or indirectly, include but are not limited to fluorescent compounds, chemiluminescent compounds, radioisotopes, chromophores, and others.</p>
    <p>The high affinity TCR protein can be used in the laboratory as a tool for qualitative and quantitative measurements of a target ligand, in medical, veterinary or plant diagnostic setting or for tissue or plant material identification. Similarly, the high affinity TCRs of the present invention can be used as reagents for detecting and/or quantitating a target material or ligand. Also provided is a method of using high affinity TCRs to bind to a selected peptide/MHC ligand comprising: labeling said high affinity TCRs with a label that binds to the selected peptide/MHC ligand; contacting said labeled high affinity TCRs with cells containing MHC molecules. The high affinity protein of the present invention, where it specifically binds to a tumor cell antigen with high affinity and specificity can be used in diagnostic tests for the particular type of cancer or it can be used in an organism in imaging tests to locate and/or estimate size and number of tumors in an organism, preferably a mammal, and also preferably a human. Methods provided for using high affinity TCRs that bind to pMHCs for diagnostic tests comprise: labeling the high affinity TCR with a detectable label; contacting said high affinity TCR with cells containing the ligand to which the high affinity TCR has high affinity for; and detecting the label. In the method, the label may be chosen to bind to specific peptide/MHC ligands, whereby cells that express specific peptide/MHC ligands are targeted. Preferred methods for using high affinity TCRs as diagnostic probes for specific peptide/MHC molecules on surfaces of cells comprise: labeling high affinity TCRs with a detectable label that binds to specific peptide/MHC ligands; contacting said TCRs with cells; and detecting said label. The detectable label chosen for use depends on the particular use, and the choice of a suitable label is well within the ordinary skill of one in the relevant art. In general, the TCR proteins selected for high affinity binding to a ligand of interest can be used in methods in which antibodies specific for the ligand can be used, with procedural modifications made for the TCR vs. antibody protein, such modifications being known in the art.</p>
    <p>The high affinity TCR, desirably a soluble single chain (sc) TCR, can be used to block autoimmune destruction of cells or tissues in autoimmune disease, where the site recognized by the cytotoxic lymphocytes on the surface of the target cell is the same as the site bound by the high affinity TCR. Preferred methods for blocking autoimmune destruction of cells comprise contacting TCRs with high affinity for the site recognized by the T lymphocytes on the surface of a target cell with cells, whereby the autoimmune destruction of cells is blocked.</p>
    <p>A soluble, high affinity scTCR can be coupled to a therapeutic compound (e.g., an anticancer compound, a therapeutic radionuclide or a cytoxic protein) where the cognate ligand of the sc TCR is a neoplastic cell surface marker. Alternatively, the binding specificity of the high affinity soluble sc TCR can be a pathogen infected target cell (such as virus-, bacteria- or protozoan-infected) and a toxic molecule can be coupled so that the target cell can be eliminated without further replication of the infective agent. Provided methods of using high affinity TCRs to inactivate pathogens comprise: binding a molecule which is toxic to the pathogen to the high affinity TCR; and contacting said TCR with cells that express said pathogen. “Toxic” means that the pathogen prevents or inhibits replication of the pathogen.</p>
    <p>Also provided are methods for using high affinity TCRs to treat disease comprising: coupling a TCR having a high affinity for a neoplastic cell surface marker with a therapeutic compound; and contacting said TCR with cells. Any therapeutic compound that is useful in slowing the progress of the disease that can be coupled with the TCR may be used. Methods of coupling the therapeutic compound with the TCR are known in the art.</p>
    <p>Also provided is a method for cloning the gene for a high affinity TCR mutant into a system that allows expression of the mutant on the surface of T cells comprising: mutating TCRs to create high affinity TCR mutants; cloning said TCR mutants into a vector; transfecting the vector into T cells; expressing the high affinity TCR mutant on the surface of T cells. This method may further comprise: selecting those T cells that are activated to a greater extent than other T cells by a peptide/MHC ligand. The transfected/infected T cells may be used for recognition of selected peptide-bearing MHC cells. These transfected/infected T cells are useful in treating disease in patients where T cells from a patient are removed and transformed with the vector that expresses the high affinity TCR mutants and returned to the patient where they are activated to a greater extent by a peptide/MHC ligand than the patient's wild type T cells.</p>
    <p>A soluble, high affinity TCR molecule can be used in place of an antibody or single chain antibody for most applications, as will be readily apparent to one of skill in the relevant arts.</p>
    <heading>BRIEF DESCRIPTION OF THE DRAWINGS</heading> <p>FIG. <b>1</b>: Flow cytometric analysis of yeast cells that express wild-type and mutant 2C TCR on their surfaces. Yeast cells displaying wild-type (T7) and mutant (qL2, qL7) scTCR were stained with anti-Vβ8 antibody F23.2 (120 nM), the specific alloantigenic peptide-MHC, QL9/L<sup>d</sup>/Ig (40 nM), or a null peptide MCMV(SEQ ID NO:1)L<sup>d</sup>/Ig (40 nM). Binding was detected by FITC-conjugated goat anti-mouse IgG F(ab′)<sub>2 </sub>and analyzed by flow cytometry. The negative population (e.g. seen with F23.2 staining) has been observed for all yeast displayed-proteins and is thought to be due to cells at a stage of growth or induction that are incapable of expressing surface fusion protein (Kieke et al. (1999) <i>Proc. Natl. Acad. Sci. USA </i>96:5651-5656; Boder and Wittrup (1997) <i>Nat. Biotech. </i>15:553-557; Kieke et al. (1997) <i>Protein Engineering </i>10:1303-1310).</p>
    <p>FIG. <b>2</b>: Fine specificity analysis of mutant scTCR binding to different QL9 variant peptides bound to L<sup>d</sup>. The original T cell clone 2C and various yeast clones were analyzed by flow cytometry for binding to L<sup>d</sup>/Ig dimers loaded with wild type QL9 (P5F), position 5 variants of QL9 (P5Y, P5H, P5E) or MCMV (SEQ ID NO:1). Binding was detected with FITC-labeled goat anti-mouse IgG. Relative fluorescence was measured by two different approaches. For T cell clone 2C, the binding of the various peptide/L<sup>d </sup>Ig dimers was adjusted relative to the QL9/L<sup>d </sup>staining (MFU<sub>pMHC</sub>/MFU<sub>QL9-Ld</sub>). For yeast cells, the binding of each peptide/L<sup>d </sup>dimer was adjusted relative to binding by the anti-Vβ8 antibody F23.2 (MFU<sub>pMHC</sub>/MFU<sub>F23.2</sub>). The latter allowed different mutants to be compared relative to each other for binding to the wild type QL9/L<sup>d</sup>.</p>
    <p>FIG. <b>3</b>: QL9/L<sup>d </sup>binding by soluble scTCRs. T2-L<sup>d </sup>cells loaded with QL9 were incubated with <sup>125</sup>I-labeled anti-L<sup>d </sup>Fab fragments (30-5-7) and various concentrations of unlabeled Fab (♦), scTCR-T7 (▪), or mutant scTCR-qL2 (). Bound and unbound <sup>125</sup>I 30-5-7 Fab fragments were separated by centrifugation through olive oil/dibutyl phthalate. Binding of <sup>125</sup>I-labeled anti-L<sup>d </sup>Fab fragments to T2-L<sup>d </sup>cells loaded with the control peptide MCMV (SEQ ID NO:1) was not inhibited even at the highest concentrations of scTCRs (data not shown).</p>
    <p>FIGS. <b>4</b>A-C: Flow cytometric analysis of the binding of scTCR/biotin to cell surface peptide/MHC. Peptide-loaded T2-L<sup>d </sup>cells were incubated with biotinylated qL2 scTCR (˜0.3 μM) or T7 scTCR (˜1.6 μM) scTCR followed by streptavidin-PE and analyzed by flow cytometry. FIG. <b>4</b>A: Flow cytometry histograms of T2-L<sup>d </sup>cells loaded with QL9 (unshaded), p2Ca (light shade), or MCMV (SEQ ID NO:1) (dark shade) and stained with qL2 scTCR/biotin. FIG. <b>4</b>B: Mean fluorescent units (MFU) of T2-L<sup>d </sup>cells loaded with QL9, p2Ca, or MCMV (SEQ ID NO:1) and stained with either secondary SA-PE only, T7 scTCR/biotin+SA-PE, or qL2 scTCR/biotin+SA-PE. FIG. <b>4</b>C: A soluble, high affinity form of mutant qL2 expressed from insect cells can detect very low concentrations of a peptide-MHC complex. Ld complexes were up-regulated on the surface of T2-Ld cells (3×10<sup>6</sup>/ml) by incubation with various concentrations of QL9 peptide at 37° C. for 1.5 hr. Approximately 2×10<sup>5 </sup>cells were stained for 30 min on ice with TCRs derived from transfected <i>Drosophila melanogaster </i>(insect) SC2 cells (Garcia, K. C., et al. (1997) <i>Proc Natl Acad Sci USA </i>94(25), 13838-13843). Cells were then washed and stained with biotin-labeled anti-Vb IgG (F23.1) followed by streptavidin-PE and analyzed by flow cytometry.</p>
    <p>FIG. <b>5</b>: Flow cytometry histograms of yeast displaying a mutant scTCR (called 3SQ2) stained with biotinylated peptide/MHC complexes, OVA/K<sup>b</sup>, dEV8/K<sup>b </sup>or SIYR (SEQ ID NO:2)/K<sup>b</sup>, followed by streptavidin-PE. As a positive control for the presence of scTCR, yeast were stained with the Vβ-specific Ig, F23.2 followed by FITC goat-anti-mouse F(ab′)<sub>2</sub>.</p>
    <p>FIG. <b>6</b>: T2-K<sup>b </sup>tumor cells were incubated with specific peptides (OVA, dEV8 or SIYR (SEQ ID NO:2)) and analyzed by flow cytometry staining with biotinylated soluble scTCR, 3SQ2 followed by streptavidin-PE. As a positive control for the presence of K<sup>b</sup>, T2-K<sup>b </sup>cells were stained with biotinylated antibody B8.24.3, which recognizes K<sup>b </sup>irrespective of the bound peptide.</p>
    <p>FIG. <b>7</b>: After multiple rounds of sorting with dEV8/K<sup>b</sup>, the yeast VαCDR3 library was stained with biotinylated dEV8/K<sup>b </sup>followed by streptavidin-PE and analyzed by flow cytometry.</p>
    <p>FIG. <b>8</b>: Flow cytometry histograms of yeast displaying a mutant scTCR (called 4d1) stained with biotinylated peptide/MHC complexes, OVA/K<sup>b </sup>or dEV8/K<sup>b </sup>followed by staining with biotinylated streptavidin-PE. As positive controls the yeast were analyzed for the presence of scTCR Vβ with F23.2 Ig and for epitope tags with an anti-6His antibody or the anti-HA Ig, 12CA5.</p>
    <p>FIG. <b>9</b>: T cells transfected with the mutant T cell receptor qL2 can recognize and be stimulated by target cells that express the peptide-MHC at low concentrations. T-cell hybridoma cell line 58−/− (Letourneur, F., and B. Malissen. (1989) <i>Eur J Immunol </i>19(12), 2269-74) was transfected with the wild-type (2C) or mutant (qL2) TCRs and 7.5×10<sup>4 </sup>transfected cells/well were incubated at 37° C. with T2-Ld cells (7.5×10<sup>4</sup>/well) in the presence of QL9 peptide. After ˜30 hrs, supernatants were collected and assayed for IL-2 released by the T cells: Supernatants were incubated with the IL-2 dependent cell line, HT2 (5×10<sup>3</sup>/well) for 18 hrs at 37° C. Proliferation of HT2 cells was measured by the incorporation of 3 [H] thymidine. Mean CPM represents the average of triplicate wells. No IL-2 was released in the absence of the QL9 peptide (data not shown).</p>
    <p>FIG. <b>10</b>: Structure and sequences of the 2C TCR CDR3α. Alignment of amino acid sequences (SEQ ID NO: 8-22) of mutant scTCRs isolated by yeast display and selection with QL9/L<sup>d</sup>. Display plasmids were isolated from yeast clones after selection and sequenced to determine CDR3α sequences. Mutants m1, m2, m3, m4, m10 and m11 were isolated after the third round of sorting. All other mutants were isolated after the fourth round of sorting.</p>
    <p>FIG. <b>11</b>: SIYR/K<sup>b </sup>Binders (3SQ2, 3SQ5): CDR3α Sequences (SEQ ID NO: <b>23</b>-<b>39</b>).</p>
    <p>FIG. <b>12</b>: Alignment of VαCDR3 Mutant Sequences (SEQ ID NO: <b>40</b>-<b>47</b>) with High Affinity of dEV8/K<sup>b </sup>(4d1, 4d2, 3Sd3, 3dS6, 3dS2, 3d2) and SIYR/K<sup>b</sup>.</p>
    <p>FIG. <b>13</b>: Alignment of VβCDR3 Sequences of Mutant scTCRs (SEQ ID NO: <b>48</b>-<b>53</b>) Selected for High Affinity for QL9/L<sup>D </sup>from a CDR3α Yeast Library. All have qL2 CDR3α (SHQGRYL (SEQ ID NO:13)). QB/5=wt; QB3 not sequenced.</p>
    <heading>DETAILED DESCRIPTION OF THE PREFERRED EMBODIMENTS</heading> <p>In order to provide a clear and consistent understanding of the specification and claims, including the scope to be given to such terms, the following definitions are provided.</p>
    <p>A coding sequence is the part of a gene or cDNA which codes for the amino acid sequence of a protein, or for a functional RNA such as a tRNA or rRNA.</p>
    <p>Complement or complementary sequence means a sequence of nucleotides which forms a hydrogen-bonded duplex with another sequence of nucleotides according to Watson-Crick base-pairing rules. For example, the complementary base sequence for 5′-AAGGCT-3' is 3′-TTCCGA-5′.</p>
    <p>Downstream means on the 3′ side of any site in DNA or RNA.</p>
    <p>Expression refers to the transcription of a gene into structural RNA (rRNA, tRNA) or messenger RNA (mRNA) and subsequent translation of a mRNA into a protein.</p>
    <p>An amino acid sequence that is functionally equivalent to a specifically exemplified TCR sequence is an amino acid sequence that has been modified by single or multiple amino acid substitutions, by addition and/or deletion of amino acids, or where one or more amino acids have been chemically modified, but which nevertheless retains the binding specificity and high affinity binding activity of a cell-bound or a soluble TCR protein of the present invention. Functionally equivalent nucleotide sequences are those that encode polypeptides having substantially the same biological activity as a specifically exemplified cell-bound or soluble TCR protein. In the context of the present invention, a soluble TCR protein lacks the portions of a native cell-bound TCR and is stable in solution (i.e., it does not generally aggregate in solution when handled as described herein and under standard conditions for protein solutions).</p>
    <p>Two nucleic acid sequences are heterologous to one another if the sequences are derived from separate organisms, whether or not such organisms are of different species, as long as the sequences do not naturally occur together in the same arrangement in the same organism.</p>
    <p>Homology refers to the extent of identity between two nucleotide or amino acid sequences.</p>
    <p>Isolated means altered by the hand of man from the natural state. If an “isolated” composition or substance occurs in nature, it has been changed or removed from its original environment, or both. For example, a polynucleotide or a polypeptide naturally present in a living animal is not isolated, but the same polynucleotide or polypeptide separated from the coexisting materials of its natural state is isolated, as the term is employed herein.</p>
    <p>A linker region is an amino acid sequence that operably links two functional or structural domains of a protein.</p>
    <p>A nucleic acid construct is a nucleic acid molecule which is isolated from a naturally occurring gene or which has been modified to contain segments of nucleic acid which are combined and juxtaposed in a manner which would not otherwise exist in nature.</p>
    <p>Nucleic acid molecule means a single- or double-stranded linear polynucleotide containing either deoxyribonucleotides or ribonucleotides that are linked by 3′-5′-phosphodiester bonds.</p>
    <p>Two DNA sequences are operably linked if the nature of the linkage does not interfere with the ability of the sequences to effect their normal functions relative to each other. For instance, a promoter region would be operably linked to a coding sequence if the promoter were capable of effecting transcription of that coding sequence.</p>
    <p>A polypeptide is a linear polymer of amino acids that are linked by peptide bonds.</p>
    <p>Promoter means a cis-acting DNA sequence, generally 80-120 base pairs long and located upstream of the initiation site of a gene, to which RNA polymerase may bind and initiate correct transcription. There can be associated additional transcription regulatory sequences which provide on/off regulation of transcription and/or which enhance (increase) expression of the downstream coding sequence.</p>
    <p>A recombinant nucleic acid molecule, for instance a recombinant DNA molecule, is a novel nucleic acid sequence formed in vitro through the ligation of two or more nonhomologous DNA molecules (for example a recombinant plasmid containing one or more inserts of foreign DNA cloned into at least one cloning site).</p>
    <p>Transformation means the directed modification of the genome of a cell by the external application of purified recombinant DNA from another cell of different genotype, leading to its uptake and integration into the subject cell's genome. In bacteria, the recombinant DNA is not typically integrated into the bacterial chromosome, but instead replicates autonomously as a plasmid.</p>
    <p>Upstream means on the 5′ side of any site in DNA or RNA.</p>
    <p>A vector is a nucleic acid molecule that is able to replicate autonomously in a host cell and can accept foreign DNA. A vector carries its own origin of replication, one or more unique recognition sites for restriction endonucleases which can be used for the insertion of foreign DNA, and usually selectable markers such as genes coding for antibiotic resistance, and often recognition sequences (e.g. promoter) for the expression of the inserted DNA. Common vectors include plasmid vectors and phage vectors.</p>
    <p>High affinity T cell receptor (TCR) means an engineered TCR with stronger binding to a target ligand than the wild type TCR.</p>
    <p>T cells recognize a foreign peptide bound to the MHC product through the αβ heterodimeric T cell receptor (TCR). The TCR repertoire has extensive diversity created by the same gene rearrangement mechanisms used in antibody heavy and light chain genes [Tonegawa, S. (1988) <i>Biosci. Rep. </i>8:3-26]. Most of the diversity is generated at the junctions of variable (V) and joining (J) (or diversity, D) regions that encode the complementarity determining region 3 (CDR3) of the α and β chains [Davis and Bjorkman (1988) <i>Nature </i>334:395-402]. However, TCRs do not undergo somatic point mutations as do antibodies and, perhaps not coincidentally. TCRs also do not undergo the same extent of affinity maturation as antibodies. TCRs as they occur in nature appear to have affinities that range from 10<sup>5 </sup>to 10<sup>6 </sup>M<sup>−1 </sup>whereas antibodies typically have affinities that range from 10<sup>5 </sup>to 10<sup>9 </sup>M<sup>−1 </sup>[Davis et al. (1998) <i>Annu. Rev. Immunol. </i>16:523-544; Eisen et al. (1996) <i>Adv. Protein Chem. </i>49:1-56]. While the absence of somatic mutation in TCRs may be associated with lower affinities, it has also been argued that there is not a selective advantage for a TCR to have higher affinity. In fact, the serial-triggering [Valitutti et al. (1995) <i>Nature </i>375:148-151] and kinetic proofreading [Rabinowitz et al. (1996) <i>Proc. Natl. Acad. Sci. USA </i>93:1401-1405] models of T cell activation both suggest that longer off-rates (associated with higher affinity) would be detrimental to the signaling process. It is also possible that higher affinity TCRs might not maintain the peptide specificity required for T cell responses. For example, peptides bound within the MHC groove display limited accessible surface [Bjorkman, P. J. (1997) <i>Cell </i>89:167-170], which may in turn limit the amount of energy that can be generated in the interaction. On the other hand, raising the affinity of a TCR by directing the energy toward the MHC helices would presumably lead to thymic deletion during negative selection [Bevan, M. J. (1997) <i>Immunity </i>7:175-178].</p>
    <p>We show that there is not an inherent structural property or genetic limitation on higher affinity of T cell receptor proteins. Higher affinity TCR variants were generated in the absence of in vivo selection pressures by using yeast display combinatorial technology and TCR mutants (e.g., Vα and Vβ CDR3 mutants). Mutants selected for relatively strong binding to the target ligand (a particular p/MHC complex) can have greater than 100-fold higher affinity, i.e., a K<sub>d </sub>of about 10 nM for the p/MHC, and these mutants retained a high degree of peptide specificity. A strong preference for TCR proteins with conserved CDR3 motifs that were rich in proline or glycine were also evident. A soluble monomeric form of a high affinity TCR was capable of directly detecting p/MHC complexes on antigen-presenting cells. These findings prove that affinity maturation of TCRs is possible, at least in vitro. Thus, engineered TCR proteins can be used for targeting specific ligands, including particular p/MHC complexes and peptides, proteins or other ligands in the absence of a MHC component.</p>
    <p>To examine if it is possible to generate higher affinity TCR that retain peptide specificity, we subjected a characterized TCR to a process of directed in vitro evolution. Phage display technology [Clackson et al. (1991) <i>Nature </i>352:624-628] has not yet proven successful in the engineering of single-chain TCRs (scTCRs, Vβ-linker-Vα), despite the extensive structural similarity between antibody and TCR V regions. However, we recently showed that a scTCR could be displayed on the surface of yeast [Kieke et al. (1999) <i>Proc. Natl. Acad. Sci. USA </i>96:5651-5656], in a system that has proven successful in antibody engineering [Boder and Wittrup (1997) supra; Kieke et al. (1997) supra]. A temperature-stabilized variant (called T7) [Shusta et al. (1999) <i>J. Mol. Biol. </i>292:949-956] of the scTCR from the CTL clone 2C was used in the present study. CTL clone 2C recognizes the alloantigen L<sup>d </sup>with a bound octamer peptide called p2Ca, derived from the enzyme 2-oxoglutarate dehydrogenase [Udaka et al. (1993) <i>Proc. Natl. Acad. Sci. USA </i>90:11272-11276]. The nonameric variant QL9 is also recognized by CTL 2C, but with 10-fold higher affinity by the 2C TCR [Sykulev et al. (1994) <i>Proc. Natl. Acad. Sci. USA </i>91:11487-11491]. Alanine scanning mutagenesis shows that the CDR3α loop contributed minimal energy to the binding interaction [Manning (1998) supra], even though structural studies have shown that CDR3α of the 2C TCR is near the peptide and it undergoes a conformational change in order to accommodate the pMHC complex [Garcia (1998) <i>Science </i>279:1166-1172]. Thus, we focused our mutagenesis efforts on the five residues that form the tip of CDR3α.</p>
    <p>A library of 10<sup>5 </sup>independent TCR-CDR3α yeast mutants was subjected to selection by flow cytometry with a fluorescently-labeled QL9/L<sup>d </sup>ligand [Dal Porto et al. (1993) <i>Proc. Natl. Acad. Sci. USA </i>90:6671-6675]. After four rounds of sorting and growth, fifteen different yeast colonies were examined for their ability to bind the ligand, in comparison to the scTCR variant T7, which bears the wt CRD3α sequence (FIG. <b>1</b>). The anti-Vβ8.2 antibody F23.2 which recognizes residues in the CDR1 and CDR2 regions of the protein was used as a control to show that wt scTCR-T7 and scTCR mutants (qL2 and qL7 in FIG. <b>1</b> and others) each had approximately equivalent surface levels of the scTCR (FIG. <b>1</b>). In contrast, the soluble QL9/L<sup>d </sup>ligand bound very well to each mutant yeast clone but not to wt scTCR-T7. The MCMV (SEQ ID NO:1)/L<sup>d </sup>complex, which is not recognized by CTL clone 2C, did not bind to the scTCR mutants or to the wt scTCR-T7, indicating that the scTCR mutants retained peptide specificity. The relative affinities of the mutant TCR also appeared to vary among clones, based on differences in signals observed with the QL9/L<sup>d </sup>ligand at constant concentrations.</p>
    <p>CDR3α sequences of the fifteen mutants all differed from the starting 2C TCR sequence FIG. <b>10</b>. Comparison by a BLAST alignment algorithm aligned the sequences into two motifs. One motif contained glycine in the middle of the 5 residue stretch whereas the other motif contained three tandem prolines. Evidence that all three prolines are important in generating the highest affinity site is suggested by results with mutant q3r. Mutant q3r contained only two of the three prolines and exhibited reduced binding compared to the triple-proline mutants. The glycine-containing mutants appeared to have preferences for positive-charged residues among the two residues to the carboxy side (7/9) and aromatic and/or positive-charged residues among the two residues to the amino side (4/9 and 5/9). Without wishing to be bound by theory, it is believed that the selection for a glycine residue at position 102 in the motif indicates that the CDR3α loop requires conformational flexibility around this residue in order to achieve increased affinity. This is consistent with the large (6 Å) conformational difference observed between the CDR3α loops of the liganded and unliganded TCR [Garcia et al. (1998) supra]. It is also interesting to note that glycine is the most common residue at the V(D)J junctions of antibodies and that the presence of a glycine has recently been associated with increased affinity in the response to the (4-hydroxy-3-nitrophenyl) acetyl hapten [Furukawa et al. (1999) <i>Immunity </i>11:329-338].</p>
    <p>In contrast to the isolates that contain glycine, the selection for a proline-rich sequence at the tip of the CDR3α loop is believed, without wishing to be bound by any particular theory, to indicate that these TCR molecules exhibit a more rigid conformation that confers higher affinity. The X-ray crystallographic structures of a germ line antibody of low affinity compared to its affinity-matured derivative showed that the high affinity state may have been due to the stabilization of the antibody in a configuration that accommodated the hapten [Wedemayer et al. (1997) <i>Science </i>276:1665-1669]. Similarly, the NMR solution structure of a scTCR that may be analogous to the germline antibody showed that the CDR3α and β loops both exhibited significant mobility [Hare et al. (1999) <i>Nat. Struct. Biol. </i>6:574-581]. Recent thermodynamic studies of TCR:pMHC interactions have also suggested the importance of conformational changes in binding [Willcox et al. (1999) <i>Immunity </i>10:357-365; Boniface et al. (1999) <i>Proc. Natl. Acad. Sci. USA </i>96:11446-11451]. Structural and thermodynamic studies of the TCR mutants discussed herein allowed us to examine if the two CDR3α motifs (Gly versus Pro-rich) differ in the mechanism by which they confer higher affinity.</p>
    <p>Although the scTCR mutants did not bind the null (irrelevant) peptide/L<sup>d </sup>complex MCMV (SEQ ID NO:1)/L<sup>d</sup>, it remained possible that the increase in affinity was accompanied by a change in fine specificity. To examine this question, we used QL9 position 5 (Phe) peptide variants which have been shown previously to exhibit significant differences in their binding affinity for the wt 2C TCR [Schlueter (1996) <i>J. Immunol. </i>157:4478-4485]. The binding of these pMHC to various TCR mutants on the yeast surface and to clone 2C was measured by flow cytometry. As shown in FIG. 2, the native TCR on 2C is capable of binding QL9 variants that contain either tyrosine or histidine at position 5 but not those containing glutamic acid. Each of the higher affinity TCR mutants retained the ability to recognize the conservative tyrosine-substituted peptide, and they were likewise incapable of recognizing the glutamic acid-substituted peptide. However, several of the TCR mutants (qL2, qL5, and qL7) bound to the histidine-substituted peptide (albeit to different extents) whereas other mutants (qL 1, qL3, and qL8) did not bind this peptide (within the detection limits of this assay). Thus, the CDR3α loop can influence the peptide fine specificity of recognition, but it is not the only region of the TCR involved. The effect on peptide specificity could be through direct interaction of CDR3α residues with the variant peptide, as suggested from earlier studies involving CDR3-directed selections [Sant' Angelo et al. (1996) <i>Immunity </i>4:367-376; Jorgensen et al. (1992) <i>Nature </i>355:224-230]. Alternatively, binding energy may be directed at peptide-induced changes in the L<sup>d </sup>molecule itself. The latter possibility is perhaps more likely in the case of the 2C TCR:QL9/L<sup>d </sup>interaction, as position 5 of QL9 has been predicted to point toward the L<sup>d </sup>groove [Schlueter et al. (1996) supra; Speir et al. (1998) <i>Immunity </i>8:553-562]. The fine-specificity analysis also shows that it is possible to engineer TCR with increased, or at least altered, specificity for cognate peptides. Thus, directed evolution of only a short region (CDR3α) of a single TCR allows the isolation of many TCR variants with desirable peptide-binding specificities and/or increased binding affinities.</p>
    <p>In order to determine the magnitude of the affinity increases associated with a selected CDR3α mutant, the wild type T7 scTCR and the qL2 scTCR were expressed as soluble forms in a yeast secretion system. Purified scTCR preparations were compared for their ability to block the binding of a <sup>125</sup>I-labeled anti-L<sup>d </sup>Fab fragments to QL9 or MCMV (SEQ ID NO:1) loaded onto L<sup>d </sup>on the surface of T2-L<sup>d </sup>cells [Manning (1998) supra; Sykulev et al. (1994) <i>Immunity </i>1:15-22]. As expected, neither T7 nor qL2 scTCR were capable of inhibiting the binding of <sup>125</sup>I-Fab fragments to T2-L<sup>d </sup>cells upregulated with the MCMV (SEQ ID NO:1) peptide. However, both T7 and qL2 were capable of inhibiting the binding of anti-L<sup>d </sup>Fab fragments to QL9/L<sup>d </sup>(FIG. <b>3</b>). The qL2 scTCR variant was as effective as unlabeled Fab fragments in inhibiting binding, whereas the T7 scTCR was 160-fold less effective (average of 140-fold difference among four independent titrations). The K<sub>D </sub>values of the scTCR for the QL9/L<sup>d </sup>were calculated from the inhibition curves to be 1.5 μM for T7 and 9.0 nM for qL2. The value for T7 is in close agreement with the 3.2 μM K<sub>D </sub>previously reported for the 2C scTCR [Manning et al. (1999) <i>J. Exp. Med. </i>189:461-470]. These findings show that the yeast system, combined with CDR3α-directed mutagenesis, allows selection of mutants with at least about 100-fold higher intrinsic binding affinities for a particular pMHC ligand.</p>
    <p>If the soluble scTCR has a high affinity for its pMHC ligand, then it is useful, like antibodies, as a specific probe for cell-surface bound antigen. To confirm this, the soluble T7 and qL2 scTCR were biotinylated, and the labeled-scTCR molecules were incubated with T2-L<sup>d </sup>cells loaded with QL9, p2Ca, or MCMV (SEQ ID NO:1). The qL2 scTCR, but not the T7 scTCR, yielded easily detectable staining of the T2 cells that had been incubated with QL9 or p2Ca (FIGS. <b>4</b>A-<b>4</b>B). It is significant that p2Ca-upregulated cells were also readily detected by qL2 scTCR, as p2Ca is the naturally processed form of the peptide recognized by the alloreactive clone 2C and it has an even lower affinity than the QL9/L<sup>d </sup>complex for the 2C TCR [Sykulev et al. (1994) supra].</p>
    <p>The high affinity receptors described in our study were derived by variation at the VJ junction, the same process that operates very effectively in vivo through gene rearrangements in T cells (Davis and Bjorkman (1988) <i>Nature </i>334:395-402). The fact that we could readily isolate a diverse set of high affinity TCR in vitro indicates that there is not a genetic or structural limitation to high affinity receptors. This supports the view that inherently low affinities of TCRs found in vivo are due to a lack of selection for higher affinity and perhaps a selection for lower affinity (Sykulev et al. (1995) <i>Proc. Natl. Acad. Sci. USA </i>92:11990-11992; Valitutti et al. (1995) <i>Nature </i>375:148-151; Rabinowitz et al. (1996) <i>Proc. Natl. Acad. Sci. USA </i>93:1401-1405). In this respect, the higher affinity TCRs of the present invention now provide the reagents for directly testing hypotheses about the effects of affinity on T cell responses (Davis et al. (1998) <i>Ann. Rev. Immunol. </i>16:523-544; Sykulev et al. (1995) supra; Valitutti et al. (1995) supra; Rabinowitz et al. 1996) supra).</p>
    <p>In summary, we have shown that T cell receptors, which represent a class of proteins as diverse as antibodies, can be engineered like antibodies to yield high affinity, antigen-specific probes. Furthermore, a soluble version of the high affinity receptor can directly detect specific peptide/MHC complexes on cells. Thus, these engineered proteins are useful as diagnostics, for tumor cells, for example. Soluble derivatives of the high affinity TCRs are useful or can be further engineered as high affinity, antigen-specific probes. The soluble TCR derivatives when appropriately labeled (or bound by a detectable ligand for that soluble TCR) can serve as a probe for specific peptide/MCHC complexes on cells, for example, derived surfaces of tumor cells or other neoplastic cells, or antigens diagnostic of virus-infected cells or other diseased cells. Other applications for high affinity TCR cell bound proteins or soluble derivatives include use in diagnosis or study of certain autoimmune diseases. Where a characteristic peptide/MHC or other marker surface antigen is known or can be identified, a high affinity, soluble TCR can be isolated for specific binding to that cell surface moiety and used in diagnosis or in therapy. The high affinity TCR proteins, desirably the soluble derivatives, can be used bound to cytotoxic agents as therapeutics in cancer treatment or other disorders where cells to be desirably destroyed have a characteristic and specific cell surface moiety which is recognized by a high affinity TCR (desirably a soluble TCR protein). Similarly, a soluble high affinity TCR as described herein can be coupled to an imaging agent and used to identify sites within the body where tumor cells reside where the TCR specifically binds a tumor cell marker with high affinity and specificity. A high affinity TCR bound to the surface of a cell or tissue which has been inappropriately targeted for autoimmune destruction can reduce autoimmune tissue destruction by cytotoxic lymphocytes by competing with those cytotoxic lymphocytes for binding to the cell surface of the targeted cells or tissue.</p>
    <p>These results can also be considered in the context of an important, basic question in T cell responses. Are the low affinities previously observed for T cell receptors due to the absence of somatic mutations or due to in vivo selective pressures that act on the T cell? The high affinity receptors described in our study were derived by variation at the VJ junction, the same process that operates very effectively in T cells [Davis and Bjorkman (1988) supra]. The fact that we could readily isolate a diverse set of high affinity TCR in vitro indicates that there is no structural or genetic limitation to high affinity receptors. This supports the view that inherently low affinities of TCRs found in vivo are due to a lack of selection for higher affinity and perhaps a selection for lower affinity [Sykulev et al. (1995) <i>Proc. Natl. Acad. Sci. USA </i>92:11990-11992; Rabinowitz et al. (1996) supra]. In this respect, the higher affinity TCRs now provide the reagents for directly testing hypotheses about the effects of affinity on T cell responses [Davis et al. (1998) supra; Sykulev et al. (1995) supra; Valitutti et al. (1995) supra; Rabinowitz et al. (1996) supra].</p>
    <p>It will be appreciated by those of skill in the art that, due to the degeneracy of the genetic code, numerous functionally equivalent nucleotide sequences encode the same amino acid sequence.</p>
    <p>Additionally, those of skill in the art, through standard mutagenesis techniques, in conjunction with the antigen-finding activity assays described herein, can obtain altered TCR sequences and test them for the expression of polypeptides having particular binding activity. Useful mutagenesis techniques known in the art include, without limitation, oligonucleotide-directed mutagenesis, region-specific mutagenesis, linker-scanning mutagenesis, and site-directed mutagenesis by PCR [see e.g. Sambrook et al. (1989) and Ausubel et al. (1999)].</p>
    <p>In obtaining variant TCR coding sequences, those of ordinary skill in the art will recognize that TCR-derived proteins may be modified by certain amino acid substitutions, additions, deletions, and post-translational modifications, without loss or reduction of biological activity. In particular, it is well-known that conservative amino acid substitutions, that is, substitution of one amino acid for another amino acid of similar size, charge, polarity and conformation, are unlikely to significantly alter protein function. The 20 standard amino acids that are the constituents of proteins can be broadly categorized into four groups of conservative amino acids as follows: the nonpolar (hydrophobic) group includes alanine, isoleucine, leucine, methionine, phenylalanine, proline, tryptophan and valine; the polar (uncharged, neutral) group includes asparagine, cysteine, glutamine, glycine, serine, threonine and tyrosine; the positively charged (basic) group contains arginine, histidine and lysine; and the negatively charged (acidic) group contains aspartic acid and glutamic acid. Substitution in a protein of one amino acid for another within the same group is unlikely to have an adverse effect on the biological activity of the protein.</p>
    <p>Homology between nucleotide sequences can be determined by DNA hybridization analysis, wherein the stability of the double-stranded DNA hybrid is dependent on the extent of base pairing that occurs. Conditions of high temperature and/or low salt content reduce the stability of the hybrid, and can be varied to prevent annealing of sequences having less than a selected degree of homology. For instance, for sequences with about 55% G-C content, hybridization and wash conditions of 40-50° C., 6×SSC (sodium chloride/sodium citrate buffer) and 0.1% SDS (sodium dodecyl sulfate) indicate about 60-70% homology, hybridization and wash conditions of 50-65° C., 1×SSC and 0.1% SDS indicate about 82-97% homology, and hybridization and wash conditions of 52° C., 0.1×SSC and 0.1% SDS indicate about 99-100% homology. A wide range of computer programs for comparing nucleotide and amino acid sequences (and measuring the degree of homology) are also available, and a list providing sources of both commercially available and free software is found in Ausubel et al. (1999). Readily available sequence comparison and multiple sequence alignment algorithms are, respectively, the Basic Local Alignment Search Tool (BLAST) (Altschul et al., 1997) and ClustalW programs. BLAST is available on the Internet at .ncbi.nlm.nih.gov and a version of ClustalW is available at 2.ebi.ac.uk.</p>
    <p>Industrial strains of microorganisms (e.g., <i>Aspergillus niger, Aspergillus ficuum, Aspergillus awamori, Aspergillus oryzae, Trichoderma reesei, Mucor miehei, Kluyveromyces lactis, Pichia pastoris, Saccharomyces cerevisiae, Escherichia coli, Bacillus subtilis </i>or <i>Bacillus licheniformis</i>) or plant species (e.g., canola, soybean, corn, potato, barley, rye, wheat) may be used as host cells for the recombinant production of the TCR peptides. As the first step in the heterologous expression of a high affinity TCR protein or soluble protein, an expression construct is assembled to include the TCR or soluble TCR coding sequence and control sequences such as promoters, enhancers and terminators. Other sequences such as signal sequences and selectable markers may also be included. To achieve extracellular expression of the scTCR, the expression construct may include a secretory signal sequence. The signal sequence is not included on the expression construct if cytoplasmic expression is desired. The promoter and signal sequence are functional in the host cell and provide for expression and secretion of the TCR or soluble TCR protein. Transcriptional terminators are included to ensure efficient transcription. Ancillary sequences enhancing expression or protein purification may also be included in the expression construct.</p>
    <p>Various promoters (transcriptional initiation regulatory region) may be used according to the invention. The selection of the appropriate promoter is dependent upon the proposed expression host. Promoters from heterologous sources may be used as long as they are functional in the chosen host.</p>
    <p>Promoter selection is also dependent upon the desired efficiency and level of peptide or protein production. Inducible promoters such as tac are often employed in order to dramatically increase the level of protein expression in <i>E. coli</i>. Overexpression of proteins may be harmful to the host cells. Consequently, host cell growth may be limited. The use of inducible promoter systems allows the host cells to be cultivated to acceptable densities prior to induction of gene expression, thereby facilitating higher product yields.</p>
    <p>Various signal sequences may be used according to the invention. A signal sequence which is homologous to the TCR coding sequence may be used. Alternatively, a signal sequence which has been selected or designed for efficient secretion and processing in the expression host may also be used. For example, suitable signal sequence/host cell pairs include the <i>B. subtilis </i>sacB signal sequence for secretion in <i>B. subtilis</i>, and the <i>Saccharomyces cerevisiae </i>α-mating factor or <i>P. pastoris </i>acid phosphatase phoI signal sequences for <i>P. pastoris </i>secretion. The signal sequence may be joined directly through the sequence encoding the signal peptidase cleavage site to the protein coding sequence, or through a short nucleotide bridge consisting of usually fewer than ten codons, where the bridge ensures correct reading frame of the downstream TCR sequence.</p>
    <p>Elements for enhancing transcription and translation have been identified for eukaryotic protein expression systems. For example, positioning the cauliflower mosaic virus (CaMV) promoter 1000 bp on either side of a heterologous promoter may elevate transcriptional levels by 10- to 400-fold in plant cells. The expression construct should also include the appropriate translational initiation sequences. Modification of the expression construct to include a Kozak consensus sequence for proper translational initiation may increase the level of translation by 10 fold.</p>
    <p>A selective marker is often employed, which may be part of the expression construct or separate from it (e.g., carried by the expression vector), so that the marker may integrate at a site different from the gene of interest. Examples include markers that confer resistance to antibiotics (e.g., bla confers resistance to ampicillin for <i>E. coli </i>host cells, nptII confers kanamycin resistance to a wide variety of prokaryotic and eukaryotic cells) or that permit the host to grow on minimal medium (e.g., HIS4 enables <i>P. pastoris </i>or His<sup>− </sup> <i>S. cerevisiae </i>to grow in the absence of histidine). The selectable marker has its own transcriptional and translational initiation and termination regulatory regions to allow for independent expression of the marker. If antibiotic resistance is employed as a marker, the concentration of the antibiotic for selection will vary depending upon the antibiotic, generally ranging from 10 to 600 μg of the antibiotic/mL of medium.</p>
    <p>The expression construct is assembled by employing known recombinant DNA techniques (Sambrook et al., 1989; Ausubel et al., 1999). Restriction enzyme digestion and ligation are the basic steps employed to join two fragments of DNA. The ends of the DNA fragment may require modification prior to ligation, and this may be accomplished by filling in overhangs, deleting terminal portions of the fragment(s) with nucleases (e.g., ExoIII), site directed mutagenesis, or by adding new base pairs by PCR. Polylinkers and adaptors may be employed to facilitate joining of selected fragments. The expression construct is typically assembled in stages employing rounds of restriction, ligation, and transformation of <i>E. coli</i>. Numerous cloning vectors suitable for construction of the expression construct are known in the art (λZAP and pBLUESCRIPT SK-1, Stratagene, LaJolla, Calif., pET, Novagen Inc., Madison, Wis. —cited in Ausubel et al., 1999) and the particular choice is not critical to the invention. The selection of cloning vector will be influenced by the gene transfer system selected for introduction of the expression construct into the host cell. At the end of each stage, the resulting construct may be analyzed by restriction, DNA sequence, hybridization and PCR analyses.</p>
    <p>The expression construct may be transformed into the host as the cloning vector construct, either linear or circular, or may be removed from the cloning vector and used as is or introduced onto a delivery vector. The delivery vector facilitates the introduction and maintenance of the expression construct in the selected host cell type. The expression construct is introduced into the host cells by any of a number of known gene transfer systems (e.g., natural competence, chemically mediated transformation, protoplast transformation, electroporation, biolistic transformation, transfection, or conjugation) (Ausubel et al., 1999; Sambrook et al., 1989). The gene transfer system selected depends upon the host cells and vector systems used.</p>
    <p>For instance, the expression construct can be introduced into <i>S. cerevisiae </i>cells by protoplast transformation or electroporation. Electroporation of <i>S. cerevisiae </i>is readily accomplished, and yields transformation efficiencies comparable to spheroplast transformation.</p>
    <p>Monoclonal or polyclonal antibodies, preferably monoclonal, specifically reacting with a TCR protein at a site other than the ligand binding site may be made by methods known in the art. See, e.g., Harlow and Lane (1988) <i>Antibodies: A Laboratory Manual</i>, Cold Spring Harbor Laboratories; Goding (1986) <i>Monoclonal Antibodies: Principles and Practice, </i>2d ed., Academic Press, New York; and Ausubel et al. (1999) <i>Current Protocols in Molecular Biology</i>, John Wiley &amp; Sons, Inc., New York.</p>
    <p>High affinity TCR proteins in cell-bound or soluble form which are specific for a particular pMHC are useful, for example, as diagnostic probes for screening biological samples (such as cells, tissue samples, biopsy material, bodily fluids and the like) or for detecting the presence of the cognate pMHC in a test sample. Frequently, the high affinity TCR proteins are labeled by joining, either covalently or noncovalently, a substance which provides a detectable signal. Suitable labels include but are not limited to radionuclides, enzymes, substrates, cofactors, inhibitors, fluorescent agents, chemiluminescent agents, magnetic particles and the like. Additionally the TCR protein can be coupled to a ligand for a second binding molecules: for example, the TCR protein can be biotinylated. Detection of the TCR bound to a target cell or molecule can then be effected by binding of a detectable streptavidin (a streptavidin to which a fluorescent, radioactive, chemiluminescent, or other detectable molecule is attached or to which an enzyme for which there is a chromophoric substrate available). U.S. patents describing the use of such labels and/or toxic compounds to be covalently bound to the scTCR protein include but are not limited to U.S. Pat. Nos. 3,817,837; 3,850,752; 3,927,193; 3,939,350; 3,996,345; 4,277,437; 4,275,149; 4,331,647; 4,348,376; 4,361,544; 4,468,457; 4,444,744; 4,640,561; 4,366,241; RE 35,500; 5,299,253; 5,101,827; 5,059,413. Labeled TCR proteins can be detected using a monitoring device or method appropriate to the label used. Fluorescence microscopy or fluorescence activated cell sorting can be used where the label is a fluorescent moiety, and where the label is a radionuclide, gamma counting, autoradiography or liquid scintillation counting, for example, can be used with the proviso that the method is appropriate to the sample being analyzed and the radionuclide used. In addition, there can be secondary detection molecules or particle employed where there is a detectable molecule or particle which recognized the portion of the TCR protein which is not part of the binding site for the cognate pMHC ligand or other ligand in the absence of a MHC component as noted herein. The art knows useful compounds for diagnostic imaging in situ; see, e.g., U.S. Pat. Nos. 5,101,827; 5,059,413. Radionuclides useful for therapy and/or imaging in vivo include <sup>111</sup>Indium, <sup>97</sup>Rubidium, <sup>125</sup>Iodine, <sup>131</sup>Iodine, <sup>123</sup>Iodine, <sup>67</sup>Gallium, <sup>99</sup>Technetium. Toxins include diphtheria toxin, ricin and castor bean toxin, among others, with the proviso that once the TCR-toxin complex is bound to the cell, the toxic moiety is internalized so that it can exert its cytotoxic effect. Immunotoxin technology is well known to the art, and suitable toxic molecules include, without limitation, chemotherapeutic drugs such as vindesine, antifolates, e.g. methotrexate, cisplatin, mitomycin, anthrocyclines such as daunomycin, daunorubicin or adriamycin, and cytotoxic proteins such as ribosome inactivating proteins (e.g., diphtheria toxin, pokeweed antiviral protein, abrin, ricin, pseudomonas exotoxin A or their recombinant derivatives. See, generally, e.g., Olsnes and Pihl (1982) <i>Pharmac. Ther. </i>25:355-381 and <i>Monoclonal Antibodies for Cancer Detection and Therapy</i>, Eds. Baldwin and Byers, pp. 159-179, Academic Press, 1985.</p>
    <p>High affinity TCR proteins specific for a particular pMHC ligand are useful in diagnosing animals, including humans believed to be suffering from a disease associated with the particular pMHC. The sc TCR molecules of the present invention are useful for detecting essentially any antigen, including but not limited to, those associated with a neoplastic condition, an abnormal protein, or an infection or infestation with a bacterium, a fungus, a virus, a protozoan, a yeast, a nematode or other parasite. The high affinity sc TCR proteins can also be used in the diagnosis of certain genetic disorders in which there is an abnormal protein produced. Exemplary applications for these high affinity proteins is in the treatment of autoimmune diseases in which there is a known pMHC. Type I diabetes is relatively well characterized with respect to the autoantigens which attract immune destruction. Multiple sclerosis, celiac disease, inflammatory bowel disease, Crohn's disease and rheumatoid arthritis are additional candidate diseases for such application. High affinity TCR (soluble) proteins with binding specificity for the p/MHC complex on the surface of cells or tissues which are improperly targeted for autoimmune destruction can serve as antagonists of the autoimmune destruction by competing for binding to the target cells by cytotoxic lymphocytes. By contrast, high affinity TCR proteins, desirably soluble single chain TCR proteins, which specifically bind to an antigen or to a p/MHC on the surface of a cell for which destruction is beneficial, can be coupled to toxic compounds (e.g., toxins or radionuclides) so that binding to the target cell results in subsequent binding and destruction by cytotoxic lymphocytes. The cell targeted for destruction can be a neoplastic cell (such as a tumor cell), a cell infected with a virus, bacterium or protozoan or other disease-causing agent or parasite, or it can be a bacterium, yeast, fungus, protozoan or other undesirable cell. Such high affinity sc TCR proteins can be obtained by the methods described herein and subsequently used for screening for a particular ligand of interest.</p>
    <p>The high affinity TCR compositions can be formulated by any of the means known in the art. They can be typically prepared as injectables, especially for intravenous, intraperitoneal or synovial administration (with the route determined by the particular disease) or as formulations for intranasal or oral administration, either as liquid solutions or suspensions. Solid forms suitable for solution in, or suspension in, liquid prior to injection or other administration may also be prepared. The preparation may also, for example, be emulsified, or the protein(s)/peptide(s) encapsulated in liposomes.</p>
    <p>The active ingredients are often mixed with excipients or carriers which are pharmaceutically acceptable and compatible with the active ingredient. Suitable excipients include but are not limited to water, saline, dextrose, glycerol, ethanol, or the like and combinations thereof. The concentration of the high affinity TCR protein in injectable, aerosol or nasal formulations is usually in the range of 0.05 to 5 mg/ml. The selection of the particular effective dosages is known and performed without undue experimentation by one of ordinary skill in the art. Similar dosages can be administered to other mucosal surfaces.</p>
    <p>In addition, if desired, vaccines may contain minor amounts of auxiliary substances such as wetting or emulsifying agents, pH buffering agents, and/or adjuvants which enhance the effectiveness of the vaccine. Examples of adjuvants which may be effective include but are not limited to: aluminum hydroxide; N-acetyl-muramyl-L-threonyl-D-isoglutamine (thr-MDP); N-acetyl-nor-muramyl-L-alanyl-D-isoglutamine (CGP 11637, referred to as nor-MDP); N-acetylmuramyl-L-alanyl-D-isoglutaminyl-L-alanine-2-(1′-2′-dipalmitoyl-sn-glycero-3hydroxyphosphoryloxy)-ethylamine (CGP 19835A, referred to as MTP-PE); and RIBI, which contains three components extracted from bacteria: monophosphoryl lipid A, trehalose dimycolate and cell wall skeleton (MPL+TDM+CWS) in a 2% squalene/Tween 80 emulsion. Such additional formulations and modes of administration as are known in the art may also be used.</p>
    <p>The high affinity TCR proteins of the present invention and/or pMHC-binding fragments having primary structure similar (more than 90% identity) to the high affinity TCR proteins and which maintain the high affinity for the cognate ligand may be formulated into vaccines as neutral or salt forms. Pharmaceutically acceptable salts include but are not limited to the acid addition salts (formed with free amino groups of the peptide) which are formed with inorganic acids, e.g., hydrochloric acid or phosphoric acids; and organic acids, e.g., acetic, oxalic, tartaric, or maleic acid. Salts formed with the free carboxyl groups may also be derived from inorganic bases, e.g., sodium, potassium, ammonium, calcium, or ferric hydroxides, and organic bases, e.g., isopropylamine, trimethylamine, 2-ethylamino-ethanol, histidine, and procaine.</p>
    <p>High affinity TCR proteins for therapeutic use, e.g., those conjugated to cytotoxic compounds are administered in a manner compatible with the dosage formulation, and in such amount and manner as are prophylactically and/or therapeutically effective, according to what is known to the art. The quantity to be administered, which is generally in the range of about 100 to 20,000 μg of protein per dose, more generally in the range of about 1000 to 10,000 μg of protein per dose. Similar compositions can be administered in similar ways using labeled high affinity TCR proteins for use in imaging, for example, to detect tissue under autoimmune attack and containing the cognate pMHCs or to detect cancer cells bearing a cognate pMHC on their surfaces. Precise amounts of the active ingredient required to be administered may depend on the judgment of the physician or veterinarian and may be peculiar to each individual, but such a determination is within the skill of such a practitioner.</p>
    <p>The vaccine or other immunogenic composition may be given in a single dose; two dose schedule, for example two to eight weeks apart; or a multiple dose schedule. A multiple dose schedule is one in which a primary course of vaccination may include 1 to 10 or more separate doses, followed by other doses administered at subsequent time intervals as required to maintain and/or reinforce the immune response, e.g., at 1 to 4 months for a second dose, and if needed, a subsequent dose(s) after several months. Humans (or other animals) immunized with the retrovirus-like particles of the present invention are protected from infection by the cognate retrovirus.</p>
    <p>Standard techniques for cloning, DNA isolation, amplification and purification, for enzymatic reactions involving DNA ligase, DNA polymerase, restriction endonucleases and the like, and various separation techniques are those known and commonly employed by those skilled in the art. A number of standard techniques are described in Sambrook et al. (1989) <i>Molecular Cloning</i>, Second Edition, Cold Spring Harbor Laboratory, Plainview, N.Y.; Maniatis et al. (1982) <i>Molecular Cloning</i>, Cold Spring Harbor Laboratory, Plainview, N.Y.; Wu (ed.) (1993) <i>Meth. Enzymol. </i>218, Part I; Wu (ed.) (1979) <i>Meth Enzymol. </i>68; Wu et al. (eds.) (1983) <i>Meth. Enzymol. </i>100 and 101; Grossman and Moldave (eds.) <i>Meth. Enzymol. </i>65; Miller (ed.) (1972) <i>Experiments in Molecular Genetics</i>, Cold Spring Harbor Laboratory, Cold Spring Harbor, N.Y.; Old and Primrose (1981) <i>Principles of Gene Manipulation</i>, University of California Press, Berkeley; Schleif and Wensink (1982) <i>Practical Methods in Molecular Biology</i>; Glover (ed.) (1985) <i>DNA Cloning </i>Vol. I and II, IRL Press, Oxford, UK; Hames and Higgins (eds.) (1985) <i>Nucleic Acid Hybridization</i>, IRL Press, Oxford, UK; and Setlow and Hollaender (1979) <i>Genetic Engineering: Principles and Methods</i>, Vols. 1-4, Plenum Press, New York. Abbreviations and nomenclature, where employed, are deemed standard in the field and commonly used in professional journals such as those cited herein.</p>
    <p>All references cited in the present application are incorporated by reference herein to supplement the disclosure and experimental procedures provided in the present Specification to the extent that there is no inconsistency with the present disclosure.</p>
    <p>The following examples are provided for illustrative purposes, and are not intended to limit the scope of the invention as claimed herein. Any variations in the exemplified articles and/or methods which occur to the skilled artisan are intended to fall within the scope of the present invention.</p>
    <heading>EXAMPLES</heading> <p>General methods for making high affinity TCRs are given in U.S. patent application Ser. No. 09/009,388, filed Jan. 20, 1998, and WO99/36569, filed Jan. 20, 1999, which are hereby incorporated by reference to the extent not inconsistent with the disclosure herewith.</p>
    <heading>Example 1</heading> <heading>Library Construction</heading> <p>The 2C scTCR used as the scaffold for directed evolution (T7) contained six mutations (βG17E, βG42E βL81S, αL43P, αW82R, and αI118N) that have been shown to increase the stability of the TCR but still allow pMHC binding [see, e.g., Shusta, E. V. et al. (1999) <i>J. Mol. Biol. </i>292, 949-956].</p>
    <p>Mutagenic PCR of the T7 scTCR VαCDR3 was performed using an AGA2-specific upstream primer (GGCAGCCCCATAAACACACAGTAT (SEQ ID NO:3)) and a degenerate downstream primer CTTTTGT GCC<u>G</u>GATCCAAATGTCAG(SNN)<sub>5</sub>GCTCACAGCACAGAAGTACAC<u>G</u>GCCGAGTCGCTC (SEQ ID NO:4). Underlined bases indicate the positions of silent mutations introducing unique BamHI and EagI restriction sites. The purified PCR product was digested with NdeI and BamHI and ligated to NdeI-BamHI digested T7/pCT302 [Boder and Wittrup (1997) supra; Kieke et al. (1999) supra; Shusta et al. (1999) supra]. The ligation mixture was transformed into DH10B electro-competent <i>E. coli </i>(Gibco BRI, Gaithersburg, Md.), and transformants were pooled into 250 ml LB supplemented with ampicillin at 100 μg/ml and grown overnight at 37° C. Plasmid DNA was transformed into the yeast, (<i>Saccharomyces cerevisiae</i>) strain EBY100 by the method of Gietz and Schiestl [Geitz et al. (1995) <i>Yeast </i>11:355-360].</p>
    <heading>Example 2</heading> <heading>Cell Sorting</heading> <p>The yeast library [Shusta et al. (1999) <i>Curr. Opin. Biotechnol. </i>10:117-122] was grown in SD-CAA (2% dextrose, 0.67% yeast nitrogen base, 1% casamino acids (Difco, Livonia, Mich.)) at 30° C. to an OD<sub>600</sub>=4.0. To induce surface scTCR expression, yeast were pelleted by centrifugation, resuspended to an OD<sub>600</sub>=1.0 in SG-CAA (2% galactose, 0.67% yeast nitrogen base, 1% casamino acids), and incubated at 20° C. for about 24 hr. In general, about 10<sup>7 </sup>cells/tube were incubated on ice for 1 hr with 50 μl of QL9/L<sup>d</sup>/IgG dimers [Dal Porto et al. (1993) supra] diluted in phosphate buffered saline, pH 7.4 supplemented with 0.5 mg/ml BSA (PBS-BSA). After incubation, cells were washed and labeled for 30 min with FITC-conjugated goat anti-mouse IgG F(ab<sup>7</sup>)<sub>2 </sub>(Kirkegaard &amp; Perry, Gaithersburg, Md.) in PBS-BSA. Yeast were then washed and resuspended in PBS-BSA immediately prior to sorting. Cells exhibiting the highest fluorescence were isolated by FACS sorting with a Coulter 753 bench. After isolation, sorted cells were expanded in SD-CAA and induced in SG-CAA for subsequent rounds of selection. A total of four sequential sorts were performed. The concentrations of QL9/L<sup>d</sup>/IgG dimers used for staining were 50 μg/ml for sorts 1-3 and 0.5 μg/ml for the final sort. The percentages of total cells isolated from each sort were 5.55, 2.68, 2.56, and 0.58%, respectively. Aliquots of sorts 3 and 4 were plated on SD-CAA to isolate individual clones which were analyzed by flow cytometry using a Coulter Epics XL instrument.</p>
    <heading>Example 3</heading> <heading>Soluble scTCR Production</heading> <p>The T7 and qL2 open reading frames were excised from pCT302 NheI-XhoI and ligated into NheI-XhoI digested pRSGALT, a yeast expression plasmid [Shusta et al. (1999) supra]. Ligated plasmids were transformed into DH10B electro-competent <i>E. coli </i>(Gibco BRL). Plasmid DNA was isolated from bacterial cultures and transformed into <i>Saccharomyces cerevisiae </i>BJ5464 (α ura3-52 trp1 leu2 1 his3 200pep4::HIS3 prb1 1.6R can1 GAL) [Shusta et al. (1999) supra]. Yeast clones were grown in one liter SD-CAA/Trp (20 mg/L tryptophan) for 48 hr at 30° C. To induce scTCR secretion, cells were pelleted by centrifugation at 4000×g, resuspended in one liter SG-CAA/Trp supplemented with 1 mg/ml BSA, and incubated for 72 hr at 20° C. Culture supernatants were harvested by centrifugation at 4000×g, concentrated to about 50 ml, and dialyzed against PBS, pH 8.0. The 6His-tagged scTCRs were purified by native nickel affinity chromatography (Ni-NTA Superflow, Qiagen, Valencia, Calif.; 5 mM and 20 mM imidazole, pH 8.0 wash; 250 mM imidazole elution) [Shusta et al. (1999) supra].</p>
    <heading>Example 4</heading> <heading>Cell-Binding Assays</heading> <p>The binding of soluble scTCRs to QL9/L<sup>d </sup>was monitored in a competition format as described previously [Manning et al. (1998) supra; Sykulev et al. (1994) supra]. Peptide-upregulated T2-L<sup>d </sup>cells (3×10<sup>5</sup>/well) were incubated for one hour on ice in the presence of <sup>125</sup>I-labeled anti-L<sup>d </sup>Fabs (30-5-7) and various concentrations of scTCRs. Bound and unbound <sup>125</sup>I 30-5-7 Fabs were separated by centrifugation through olive oil/dibutyl phthalate. Inhibition curves were constructed to determine inhibitor concentrations yielding 50% of maximal inhibition. Dissociation constants were calculated using the formula of Cheng and Pursoff [Cheng (1973) <i>Biochem. Pharm. </i>22:3099-3108]. To monitor direct binding of scTCRs to cell-bound pMHC, peptide-upregulated T2-L<sup>d </sup>cells (5×10<sup>5</sup>/tube) were incubated for 40 min on ice with biotinylated soluble scTCRs followed by staining for 30 min with streptavidin-phycoerythrin (PharMingen, San Diego, Calif.). Cellular fluorescence was detected by flow cytometry.</p>
    <heading>Example 5</heading> <heading>Identification of High Affinity TCRs that are Specific for a Different Peptide and a Different MHC Molecule (K<sup>b</sup>)</heading> <p>Using the same library of yeast-displayed mutants of the CDR3α region of the TCR, it was possible to select for higher affinity TCRs that are specific for yet a different peptide bound to a different MHC molecule. In this case the peptide called SIYR (SIYRYYGL (SEQ ID NO:5)) was bound to the MHC molecule called K<sup>b</sup>, and this ligand complex was used in fluorescent form to select by flow cytometry. Sixteen clones expressing high affinity TCR were sequenced, each showing a different sequence in the CDR3α region FIG. <b>11</b>.</p>
    <p>As an example of the specificity of these TCRs, the mutant 3SQ2 was stained with various agents, including the secondary reagent alone (SA:PE), the anti-Vβ antibody F23.2, and three peptide/K<sup>b </sup>complexes (OVA/K<sup>b</sup>, dEV8/K<sup>b</sup>, and SIYR/K<sup>b</sup>). As shown in FIG. 5, only the pMHC (SIYR/K<sup>b</sup>) used in the original selection had sufficient affinity to bind to the mutant TCR. Wild-type TCR did not bind the SIYR/K<sup>b </sup>ligand at any concentration tested (data not shown).</p>
    <p>The mutant TCR 3SQ2 was also expressed as a soluble protein in the yeast secretion system and tested after biotinylation for its ability to bind directly to pMHC on the surface of tumor cells. As shown in FIG. 6, the labeled 3SQ2 bound very well to tumor cells that expressed only the appropriate peptide SIYR. The staining was nearly as strong as the high affinity anti-K<sup>b </sup>monoclonal antibody B8.24.3, that binds to any K<sup>b </sup>molecule (FIG. <b>6</b>), regardless of the peptide present.</p>
    <heading>Example 6</heading> <heading>Identification of High Affinity TCRs that are Specific for a Different Peptide Bound to the Same MHC Molecule (K<sup>b</sup>)</heading> <p>To determine if the same TCR scaffold could be used to isolate higher affinity forms against yet a different peptide bound to the same MHC, we screened the TCR CDR3α library with the peptide called dEV8 (EQYKFYSV (SEQ ID NO:6)), bound to K<sup>b</sup>. After several sorts by flow cytometry with the biotinylated dEV<sub>8</sub>/K<sup>b </sup>ligand (followed by phycoerythrin-streptavidin, PE-SA), there was a significant enrichment of yeast cells that bound to the dEV8/K<sup>b </sup>(as indicated by PE levels in FIG. <b>7</b>).</p>
    <p>Six of the clones that were isolated by selection with dEV8/K<sup>b </sup>were sequenced and the CDR3 sequences all differed FIG. <b>12</b>. These sequences were similar in sequence, but different from, those isolated by selection with SIYR/K<sup>b </sup>(two examples, 3SQ2 and 3SQ5, are also shown in FIG. <b>12</b>. It can be concluded that it is possible to isolate higher affinity TCRs against different antigens, even using the same TCR library of mutants.</p>
    <p>To prove the antigen specificity of the isolated clones, one of the dEV<sub>8</sub>/K<sup>b </sup>selected clones (4d1) was examined with a panel of different antibodies and ligands (FIG. <b>6</b>). As expected, this TCR reacted with the three appropriate antibodies (anti-Vβ8 antibody F23.2, anti-HA tag antibody, and anti-His tag antibody) and the dEV8/K<sup>b </sup>antigen, but not with another antigen, OVA/K<sup>b</sup>. Wild type TCR did not bind to either peptide/K<sup>b </sup>complex (data not shown). Thus, the high affinity TCR was specific for the selected antigen.</p>
    <heading>Example 7</heading> <heading>Identification of High Affinity TCRs by Creating a Random TCR Library in a Different Region of the TCR (Complementarity-Determining Region 3 of the β Chain)</heading> <p>The examples described above used a library of TCR that were mutated within a region of the α chain called CDR3. In order to show that other regions of the TCR could also be mutated to yield higher affinity TCR, a library of random mutants within five contiguous amino acid residues of the CDR3 region of the β chain was produced, using the qL2 TCR mutant as the starting material. This library was then selected with the QL9/L<sup>d </sup>ligand at concentrations below that detected with the qL2 mutant. Five yeast clones, selected by flow cytometry, were sequenced and each showed a different nucleotide and amino acid sequence (FIG. <b>8</b>). There was remarkable conservation of sequence within the five amino acid region that was mutated, suggesting that this sequence motif has been optimized for high affinity. We conclude that it is possible to mutate different regions of the TCR to yield derivatives having higher affinity for a particular pMHC.</p>
    <p>Although the description above contains many specificities, these should not be construed to limit the scope of the invention but as merely providing illustrations of some of the presently-preferred embodiments of this invention. For example, ligands other than those specifically illustrated may be used. Thus the scope of the invention should be determined by the appended claims and their legal equivalents, rather than by the examples given. All references cited herein are incorporated to the extent not inconsistent with the disclosure herewith.</p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" pgwide="1" tabstyle="monospace" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="441pt" align="left"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">                  </td>
              </tr> <tr class="description-tr"> <td class="description-td">#             SEQUENCE LISTING </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;160&gt; NUMBER OF SEQ ID NOS: 53 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;210&gt; SEQ ID NO 1 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;211&gt; LENGTH: 4 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;212&gt; TYPE: PRT </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;213&gt; ORGANISM: ARTIFICIAL SEQUENCE </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;220&gt; FEATURE: </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;221&gt; NAME/KEY: misc_feature </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;222&gt; LOCATION: ()..() </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;223&gt; OTHER INFORMATION: null peptide </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;400&gt; SEQUENCE: 1 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Met Cys Met Val </td>
              </tr> <tr class="description-tr"> <td class="description-td">1 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;210&gt; SEQ ID NO 2 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;211&gt; LENGTH: 4 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;212&gt; TYPE: PRT </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;213&gt; ORGANISM: ARTIFICIAL SEQUENCE </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;220&gt; FEATURE: </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;221&gt; NAME/KEY: misc_feature </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;222&gt; LOCATION: ()..() </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;223&gt; OTHER INFORMATION: incubation peptide </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;400&gt; SEQUENCE: 2 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Ser Ile Tyr Arg </td>
              </tr> <tr class="description-tr"> <td class="description-td">1 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;210&gt; SEQ ID NO 3 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;211&gt; LENGTH: 24 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;212&gt; TYPE: PRT </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;213&gt; ORGANISM: ARTIFICIAL SEQUENCE </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;220&gt; FEATURE: </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;221&gt; NAME/KEY: misc_feature </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;222&gt; LOCATION: ()..() </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;223&gt; OTHER INFORMATION: upstream primer </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;400&gt; SEQUENCE: 3 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Gly Gly Cys Ala Gly Cys Cys Cys Cys Ala Th </td>
              </tr> <tr class="description-tr"> <td class="description-td">#r Ala Ala Ala Cys Ala </td>
              </tr> <tr class="description-tr"> <td class="description-td">1               5   </td>
              </tr> <tr class="description-tr"> <td class="description-td">#                10  </td>
              </tr> <tr class="description-tr"> <td class="description-td">#                15 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Cys Ala Cys Ala Gly Thr Ala Thr </td>
              </tr> <tr class="description-tr"> <td class="description-td">            20 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;210&gt; SEQ ID NO 4 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;211&gt; LENGTH: 74 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;212&gt; TYPE: PRT </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;213&gt; ORGANISM: ARTIFICIAL SEQUENCE </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;220&gt; FEATURE: </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;221&gt; NAME/KEY: misc_feature </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;222&gt; LOCATION: ()..() </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;223&gt; OTHER INFORMATION: downstream primer </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;400&gt; SEQUENCE: 4 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Cys Thr Thr Thr Thr Gly Thr Gly Cys Cys Gl </td>
              </tr> <tr class="description-tr"> <td class="description-td">#y Gly Ala Thr Cys Cys </td>
              </tr> <tr class="description-tr"> <td class="description-td">1               5   </td>
              </tr> <tr class="description-tr"> <td class="description-td">#                10  </td>
              </tr> <tr class="description-tr"> <td class="description-td">#                15 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Ala Ala Ala Thr Gly Thr Cys Ala Gly Ser As </td>
              </tr> <tr class="description-tr"> <td class="description-td">#n Asn Ser Asn Asn Ser </td>
              </tr> <tr class="description-tr"> <td class="description-td">            20      </td>
              </tr> <tr class="description-tr"> <td class="description-td">#            25      </td>
              </tr> <tr class="description-tr"> <td class="description-td">#            30 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Asn Asn Ser Asn Asn Ser Asn Asn Gly Cys Th </td>
              </tr> <tr class="description-tr"> <td class="description-td">#r Cys Ala Cys Ala Gly </td>
              </tr> <tr class="description-tr"> <td class="description-td">        35          </td>
              </tr> <tr class="description-tr"> <td class="description-td">#        40          </td>
              </tr> <tr class="description-tr"> <td class="description-td">#        45 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Cys Ala Cys Ala Gly Ala Ala Gly Thr Ala Cy </td>
              </tr> <tr class="description-tr"> <td class="description-td">#s Ala Cys Gly Gly Cys </td>
              </tr> <tr class="description-tr"> <td class="description-td">    50              </td>
              </tr> <tr class="description-tr"> <td class="description-td">#    55              </td>
              </tr> <tr class="description-tr"> <td class="description-td">#    60 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Cys Gly Ala Gly Thr Cys Gly Cys Thr Cys </td>
              </tr> <tr class="description-tr"> <td class="description-td">65                  </td>
              </tr> <tr class="description-tr"> <td class="description-td">#70 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;210&gt; SEQ ID NO 5 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;211&gt; LENGTH: 8 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;212&gt; TYPE: PRT </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;213&gt; ORGANISM: ARTIFICIAL SEQUENCE </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;220&gt; FEATURE: </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;221&gt; NAME/KEY: misc_feature </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;222&gt; LOCATION: ()..() </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;223&gt; OTHER INFORMATION: binding peptide </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;400&gt; SEQUENCE: 5 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Ser Ile Tyr Arg Tyr Tyr Gly Leu </td>
              </tr> <tr class="description-tr"> <td class="description-td">1               5 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;210&gt; SEQ ID NO 6 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;211&gt; LENGTH: 8 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;212&gt; TYPE: PRT </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;213&gt; ORGANISM: ARTIFICIAL SEQUENCE </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;220&gt; FEATURE: </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;221&gt; NAME/KEY: misc_feature </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;222&gt; LOCATION: ()..() </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;223&gt; OTHER INFORMATION: screening peptide </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;400&gt; SEQUENCE: 6 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Glu Gln Tyr Lys Phe Tyr Ser Val </td>
              </tr> <tr class="description-tr"> <td class="description-td">1               5 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;210&gt; SEQ ID NO 7 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;211&gt; LENGTH: 7 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;212&gt; TYPE: PRT </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;213&gt; ORGANISM: ARTIFICIAL SEQUENCE </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;220&gt; FEATURE: </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;221&gt; NAME/KEY: misc_feature </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;222&gt; LOCATION: ()..() </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;223&gt; OTHER INFORMATION: CDR3alpha sequence </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;400&gt; SEQUENCE: 7 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Ser Gly Phe Ala Ser Ala Leu </td>
              </tr> <tr class="description-tr"> <td class="description-td">1               5 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;210&gt; SEQ ID NO 8 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;211&gt; LENGTH: 7 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;212&gt; TYPE: PRT </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;213&gt; ORGANISM: ARTIFICIAL SEQUENCE </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;220&gt; FEATURE: </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;221&gt; NAME/KEY: misc_feature </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;222&gt; LOCATION: ()..() </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;223&gt; OTHER INFORMATION: CDR3alpha sequence </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;400&gt; SEQUENCE: 8 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Ser Ser Tyr Gly Asn Tyr Leu </td>
              </tr> <tr class="description-tr"> <td class="description-td">1               5 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;210&gt; SEQ ID NO 9 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;211&gt; LENGTH: 7 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;212&gt; TYPE: PRT </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;213&gt; ORGANISM: ARTIFICIAL SEQUENCE </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;220&gt; FEATURE: </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;221&gt; NAME/KEY: misc_feature </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;222&gt; LOCATION: ()..() </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;223&gt; OTHER INFORMATION: CDR3alpha sequence </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;400&gt; SEQUENCE: 9 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Ser Arg Arg Gly His Ala Leu </td>
              </tr> <tr class="description-tr"> <td class="description-td">1               5 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;210&gt; SEQ ID NO 10 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;211&gt; LENGTH: 7 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;212&gt; TYPE: PRT </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;213&gt; ORGANISM: ARTIFICIAL SEQUENCE </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;220&gt; FEATURE: </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;221&gt; NAME/KEY: misc_feature </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;222&gt; LOCATION: ()..() </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;223&gt; OTHER INFORMATION: CDR3alpha sequence </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;400&gt; SEQUENCE: 10 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Ser Ser Arg Gly Thr Ala Leu </td>
              </tr> <tr class="description-tr"> <td class="description-td">1               5 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;210&gt; SEQ ID NO 11 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;211&gt; LENGTH: 7 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;212&gt; TYPE: PRT </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;213&gt; ORGANISM: ARTIFICIAL SEQUENCE </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;220&gt; FEATURE: </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;221&gt; NAME/KEY: misc_feature </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;222&gt; LOCATION: ()..() </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;223&gt; OTHER INFORMATION: CDR3alpha sequence </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;400&gt; SEQUENCE: 11 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Ser His Phe Gly Thr Arg Leu </td>
              </tr> <tr class="description-tr"> <td class="description-td">1               5 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;210&gt; SEQ ID NO 12 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;211&gt; LENGTH: 7 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;212&gt; TYPE: PRT </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;213&gt; ORGANISM: ARTIFICIAL SEQUENCE </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;220&gt; FEATURE: </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;221&gt; NAME/KEY: misc_feature </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;222&gt; LOCATION: ()..() </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;223&gt; OTHER INFORMATION: CDR3alpha sequence </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;400&gt; SEQUENCE: 12 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Ser Met Phe Gly Thr Arg Leu </td>
              </tr> <tr class="description-tr"> <td class="description-td">1               5 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;210&gt; SEQ ID NO 13 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;211&gt; LENGTH: 7 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;212&gt; TYPE: PRT </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;213&gt; ORGANISM: ARTIFICIAL SEQUENCE </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;220&gt; FEATURE: </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;221&gt; NAME/KEY: misc_feature </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;222&gt; LOCATION: ()..() </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;223&gt; OTHER INFORMATION: CDR3alpha sequence </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;400&gt; SEQUENCE: 13 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Ser His Gln Gly Arg Tyr Leu </td>
              </tr> <tr class="description-tr"> <td class="description-td">1               5 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;210&gt; SEQ ID NO 14 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;211&gt; LENGTH: 7 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;212&gt; TYPE: PRT </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;213&gt; ORGANISM: ARTIFICIAL SEQUENCE </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;220&gt; FEATURE: </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;221&gt; NAME/KEY: misc_feature </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;222&gt; LOCATION: ()..() </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;223&gt; OTHER INFORMATION: CDR3alpha sequence </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;400&gt; SEQUENCE: 14 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Ser Tyr Leu Gly Leu Arg Leu </td>
              </tr> <tr class="description-tr"> <td class="description-td">1               5 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;210&gt; SEQ ID NO 15 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;211&gt; LENGTH: 7 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;212&gt; TYPE: PRT </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;213&gt; ORGANISM: ARTIFICIAL SEQUENCE </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;220&gt; FEATURE: </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;221&gt; NAME/KEY: misc_feature </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;222&gt; LOCATION: ()..() </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;223&gt; OTHER INFORMATION: CDR3alpha sequence </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;400&gt; SEQUENCE: 15 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Ser Lys His Gly Ile His Leu </td>
              </tr> <tr class="description-tr"> <td class="description-td">1               5 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;210&gt; SEQ ID NO 16 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;211&gt; LENGTH: 7 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;212&gt; TYPE: PRT </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;213&gt; ORGANISM: ARTIFICIAL SEQUENCE </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;220&gt; FEATURE: </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;221&gt; NAME/KEY: misc_feature </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;222&gt; LOCATION: ()..() </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;223&gt; OTHER INFORMATION: CDR3alpha sequence </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;400&gt; SEQUENCE: 16 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Ser Leu Thr Gly Arg Tyr Leu </td>
              </tr> <tr class="description-tr"> <td class="description-td">1               5 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;210&gt; SEQ ID NO 17 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;211&gt; LENGTH: 7 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;212&gt; TYPE: PRT </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;213&gt; ORGANISM: ARTIFICIAL SEQUENCE </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;220&gt; FEATURE: </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;221&gt; NAME/KEY: misc_feature </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;222&gt; LOCATION: ()..() </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;223&gt; OTHER INFORMATION: CDR3alpha sequence </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;400&gt; SEQUENCE: 17 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Ser Leu Pro Pro Pro Leu Leu </td>
              </tr> <tr class="description-tr"> <td class="description-td">1               5 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;210&gt; SEQ ID NO 18 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;211&gt; LENGTH: 7 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;212&gt; TYPE: PRT </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;213&gt; ORGANISM: ARTIFICIAL SEQUENCE </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;220&gt; FEATURE: </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;221&gt; NAME/KEY: misc_feature </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;222&gt; LOCATION: ()..() </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;223&gt; OTHER INFORMATION: CDR3alpha sequence </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;400&gt; SEQUENCE: 18 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Ser Ile Pro Thr Pro Ser Leu </td>
              </tr> <tr class="description-tr"> <td class="description-td">1               5 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;210&gt; SEQ ID NO 19 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;211&gt; LENGTH: 7 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;212&gt; TYPE: PRT </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;213&gt; ORGANISM: ARTIFICIAL SEQUENCE </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;220&gt; FEATURE: </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;221&gt; NAME/KEY: misc_feature </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;222&gt; LOCATION: ()..() </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;223&gt; OTHER INFORMATION: CDR3alpha sequence </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;400&gt; SEQUENCE: 19 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Ser Asn Pro Pro Pro Leu Leu </td>
              </tr> <tr class="description-tr"> <td class="description-td">1               5 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;210&gt; SEQ ID NO 20 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;211&gt; LENGTH: 7 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;212&gt; TYPE: PRT </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;213&gt; ORGANISM: ARTIFICIAL SEQUENCE </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;220&gt; FEATURE: </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;221&gt; NAME/KEY: misc_feature </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;222&gt; LOCATION: ()..() </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;223&gt; OTHER INFORMATION: CDR3alpha sequence </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;400&gt; SEQUENCE: 20 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Ser Asp Pro Pro Pro Leu Leu </td>
              </tr> <tr class="description-tr"> <td class="description-td">1               5 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;210&gt; SEQ ID NO 21 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;211&gt; LENGTH: 7 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;212&gt; TYPE: PRT </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;213&gt; ORGANISM: ARTIFICIAL SEQUENCE </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;220&gt; FEATURE: </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;221&gt; NAME/KEY: misc_feature </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;222&gt; LOCATION: ()..() </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;223&gt; OTHER INFORMATION: CDR3alpha sequence </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;400&gt; SEQUENCE: 21 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Ser Ser Pro Pro Pro Arg Leu </td>
              </tr> <tr class="description-tr"> <td class="description-td">1               5 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;210&gt; SEQ ID NO 22 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;211&gt; LENGTH: 7 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;212&gt; TYPE: PRT </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;213&gt; ORGANISM: ARTIFICIAL SEQUENCE </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;220&gt; FEATURE: </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;221&gt; NAME/KEY: misc_feature </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;222&gt; LOCATION: ()..() </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;223&gt; OTHER INFORMATION: CDR3alpha sequence </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;400&gt; SEQUENCE: 22 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Ser Ala Pro Pro Pro Ile Leu </td>
              </tr> <tr class="description-tr"> <td class="description-td">1               5 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;210&gt; SEQ ID NO 23 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;211&gt; LENGTH: 7 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;212&gt; TYPE: PRT </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;213&gt; ORGANISM: ARTIFICIAL SEQUENCE </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;220&gt; FEATURE: </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;221&gt; NAME/KEY: misc_feature </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;222&gt; LOCATION: ()..() </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;223&gt; OTHER INFORMATION: CDR3alpha sequence </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;400&gt; SEQUENCE: 23 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Ser Gly Thr His Pro Phe Leu </td>
              </tr> <tr class="description-tr"> <td class="description-td">1               5 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;210&gt; SEQ ID NO 24 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;211&gt; LENGTH: 7 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;212&gt; TYPE: PRT </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;213&gt; ORGANISM: ARTIFICIAL SEQUENCE </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;220&gt; FEATURE: </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;221&gt; NAME/KEY: misc_feature </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;222&gt; LOCATION: ()..() </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;223&gt; OTHER INFORMATION: CDR3alpha sequence </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;400&gt; SEQUENCE: 24 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Ser Gly His Leu Pro Phe Leu </td>
              </tr> <tr class="description-tr"> <td class="description-td">1               5 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;210&gt; SEQ ID NO 25 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;211&gt; LENGTH: 7 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;212&gt; TYPE: PRT </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;213&gt; ORGANISM: ARTIFICIAL SEQUENCE </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;220&gt; FEATURE: </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;221&gt; NAME/KEY: misc_feature </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;222&gt; LOCATION: ()..() </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;223&gt; OTHER INFORMATION: CDR3alpha sequence </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;400&gt; SEQUENCE: 25 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Ser Asp Ser Lys Pro Phe Leu </td>
              </tr> <tr class="description-tr"> <td class="description-td">1               5 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;210&gt; SEQ ID NO 26 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;211&gt; LENGTH: 7 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;212&gt; TYPE: PRT </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;213&gt; ORGANISM: ARTIFICIAL SEQUENCE </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;220&gt; FEATURE: </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;221&gt; NAME/KEY: misc_feature </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;222&gt; LOCATION: ()..() </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;223&gt; OTHER INFORMATION: CDR3alpha sequence </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;400&gt; SEQUENCE: 26 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Ser Ser Asp Arg Pro Tyr Leu </td>
              </tr> <tr class="description-tr"> <td class="description-td">1               5 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;210&gt; SEQ ID NO 27 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;211&gt; LENGTH: 7 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;212&gt; TYPE: PRT </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;213&gt; ORGANISM: ARTIFICIAL SEQUENCE </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;220&gt; FEATURE: </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;221&gt; NAME/KEY: misc_feature </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;222&gt; LOCATION: ()..() </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;223&gt; OTHER INFORMATION: CDR3alpha sequence </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;400&gt; SEQUENCE: 27 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Ser Leu Glu Arg Pro Tyr Leu </td>
              </tr> <tr class="description-tr"> <td class="description-td">1               5 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;210&gt; SEQ ID NO 28 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;211&gt; LENGTH: 7 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;212&gt; TYPE: PRT </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;213&gt; ORGANISM: ARTIFICIAL SEQUENCE </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;220&gt; FEATURE: </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;221&gt; NAME/KEY: misc_feature </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;222&gt; LOCATION: ()..() </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;223&gt; OTHER INFORMATION: CDR3alpha sequence </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;400&gt; SEQUENCE: 28 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Ser Arg Glu Ala Pro Tyr Leu </td>
              </tr> <tr class="description-tr"> <td class="description-td">1               5 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;210&gt; SEQ ID NO 29 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;211&gt; LENGTH: 7 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;212&gt; TYPE: PRT </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;213&gt; ORGANISM: ARTIFICIAL SEQUENCE </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;220&gt; FEATURE: </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;221&gt; NAME/KEY: misc_feature </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;222&gt; LOCATION: ()..() </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;223&gt; OTHER INFORMATION: CDR3alpha sequence </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;400&gt; SEQUENCE: 29 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Ser Leu His Arg Pro Ala Leu </td>
              </tr> <tr class="description-tr"> <td class="description-td">1               5 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;210&gt; SEQ ID NO 30 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;211&gt; LENGTH: 7 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;212&gt; TYPE: PRT </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;213&gt; ORGANISM: ARTIFICIAL SEQUENCE </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;220&gt; FEATURE: </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;221&gt; NAME/KEY: misc_feature </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;222&gt; LOCATION: ()..() </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;223&gt; OTHER INFORMATION: CDR3alpha sequence </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;400&gt; SEQUENCE: 30 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Ser Leu His Arg Pro Ala Leu </td>
              </tr> <tr class="description-tr"> <td class="description-td">1               5 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;210&gt; SEQ ID NO 31 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;211&gt; LENGTH: 7 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;212&gt; TYPE: PRT </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;213&gt; ORGANISM: ARTIFICIAL SEQUENCE </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;220&gt; FEATURE: </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;221&gt; NAME/KEY: misc_feature </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;222&gt; LOCATION: ()..() </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;223&gt; OTHER INFORMATION: CDR3alpha sequence </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;400&gt; SEQUENCE: 31 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Ser Ser Asn Arg Pro Ala Leu </td>
              </tr> <tr class="description-tr"> <td class="description-td">1               5 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;210&gt; SEQ ID NO 32 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;211&gt; LENGTH: 7 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;212&gt; TYPE: PRT </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;213&gt; ORGANISM: ARTIFICIAL SEQUENCE </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;220&gt; FEATURE: </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;221&gt; NAME/KEY: misc_feature </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;222&gt; LOCATION: ()..() </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;223&gt; OTHER INFORMATION: CDR3alpha sequence </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;400&gt; SEQUENCE: 32 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Ser Thr Asp Arg Pro Ser Leu </td>
              </tr> <tr class="description-tr"> <td class="description-td">1               5 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;210&gt; SEQ ID NO 33 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;211&gt; LENGTH: 7 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;212&gt; TYPE: PRT </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;213&gt; ORGANISM: ARTIFICIAL SEQUENCE </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;220&gt; FEATURE: </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;221&gt; NAME/KEY: misc_feature </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;222&gt; LOCATION: ()..() </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;223&gt; OTHER INFORMATION: CDR3alpha sequence </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;400&gt; SEQUENCE: 33 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Ser Gly Ser Arg Pro Thr Leu </td>
              </tr> <tr class="description-tr"> <td class="description-td">1               5 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;210&gt; SEQ ID NO 34 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;211&gt; LENGTH: 7 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;212&gt; TYPE: PRT </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;213&gt; ORGANISM: ARTIFICIAL SEQUENCE </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;220&gt; FEATURE: </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;221&gt; NAME/KEY: misc_feature </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;222&gt; LOCATION: ()..() </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;223&gt; OTHER INFORMATION: CDR3alpha sequence </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;400&gt; SEQUENCE: 34 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Ser Leu Val Thr Pro Ala Leu </td>
              </tr> <tr class="description-tr"> <td class="description-td">1               5 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;210&gt; SEQ ID NO 35 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;211&gt; LENGTH: 7 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;212&gt; TYPE: PRT </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;213&gt; ORGANISM: ARTIFICIAL SEQUENCE </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;220&gt; FEATURE: </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;221&gt; NAME/KEY: misc_feature </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;222&gt; LOCATION: ()..() </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;223&gt; OTHER INFORMATION: CDR3alpha sequence </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;400&gt; SEQUENCE: 35 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Ser Ala Thr Ser Pro Ala Leu </td>
              </tr> <tr class="description-tr"> <td class="description-td">1               5 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;210&gt; SEQ ID NO 36 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;211&gt; LENGTH: 7 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;212&gt; TYPE: PRT </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;213&gt; ORGANISM: ARTIFICIAL SEQUENCE </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;220&gt; FEATURE: </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;221&gt; NAME/KEY: misc_feature </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;222&gt; LOCATION: ()..() </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;223&gt; OTHER INFORMATION: CDR3alpha sequence </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;400&gt; SEQUENCE: 36 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Ser Ser Ile Asn Pro Ala Leu </td>
              </tr> <tr class="description-tr"> <td class="description-td">1               5 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;210&gt; SEQ ID NO 37 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;211&gt; LENGTH: 7 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;212&gt; TYPE: PRT </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;213&gt; ORGANISM: ARTIFICIAL SEQUENCE </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;220&gt; FEATURE: </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;221&gt; NAME/KEY: misc_feature </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;222&gt; LOCATION: ()..() </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;223&gt; OTHER INFORMATION: CDR3alpha sequence </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;400&gt; SEQUENCE: 37 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Ser Ala Ser Tyr Pro Ser Leu </td>
              </tr> <tr class="description-tr"> <td class="description-td">1               5 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;210&gt; SEQ ID NO 38 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;211&gt; LENGTH: 7 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;212&gt; TYPE: PRT </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;213&gt; ORGANISM: ARTIFICIAL SEQUENCE </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;220&gt; FEATURE: </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;221&gt; NAME/KEY: misc_feature </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;222&gt; LOCATION: ()..() </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;223&gt; OTHER INFORMATION: CDR3alpha sequence </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;400&gt; SEQUENCE: 38 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Ser Arg Trp Thr Ser Gly Leu </td>
              </tr> <tr class="description-tr"> <td class="description-td">1               5 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;210&gt; SEQ ID NO 39 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;211&gt; LENGTH: 7 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;212&gt; TYPE: PRT </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;213&gt; ORGANISM: ARTIFICIAL SEQUENCE </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;220&gt; FEATURE: </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;221&gt; NAME/KEY: misc_feature </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;222&gt; LOCATION: ()..() </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;223&gt; OTHER INFORMATION: CDR3alpha sequence </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;400&gt; SEQUENCE: 39 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Ser Gly Ser Arg Pro Ala Leu </td>
              </tr> <tr class="description-tr"> <td class="description-td">1               5 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;210&gt; SEQ ID NO 40 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;211&gt; LENGTH: 7 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;212&gt; TYPE: PRT </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;213&gt; ORGANISM: ARTIFICIAL SEQUENCE </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;220&gt; FEATURE: </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;221&gt; NAME/KEY: misc_feature </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;222&gt; LOCATION: ()..() </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;223&gt; OTHER INFORMATION: CDR3alpha sequence </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;400&gt; SEQUENCE: 40 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Ser Leu Thr His His Phe Leu </td>
              </tr> <tr class="description-tr"> <td class="description-td">1               5 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;210&gt; SEQ ID NO 41 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;211&gt; LENGTH: 7 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;212&gt; TYPE: PRT </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;213&gt; ORGANISM: ARTIFICIAL SEQUENCE </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;220&gt; FEATURE: </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;221&gt; NAME/KEY: misc_feature </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;222&gt; LOCATION: ()..() </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;223&gt; OTHER INFORMATION: CDR3alpha sequence </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;400&gt; SEQUENCE: 41 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Ser Met Thr His His Phe Leu </td>
              </tr> <tr class="description-tr"> <td class="description-td">1               5 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;210&gt; SEQ ID NO 42 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;211&gt; LENGTH: 7 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;212&gt; TYPE: PRT </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;213&gt; ORGANISM: ARTIFICIAL SEQUENCE </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;220&gt; FEATURE: </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;221&gt; NAME/KEY: misc_feature </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;222&gt; LOCATION: ()..() </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;223&gt; OTHER INFORMATION: CDR3alpha sequence </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;400&gt; SEQUENCE: 42 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Ser Leu Ser Arg Pro Tyr Leu </td>
              </tr> <tr class="description-tr"> <td class="description-td">1               5 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;210&gt; SEQ ID NO 43 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;211&gt; LENGTH: 7 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;212&gt; TYPE: PRT </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;213&gt; ORGANISM: ARTIFICIAL SEQUENCE </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;220&gt; FEATURE: </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;221&gt; NAME/KEY: misc_feature </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;222&gt; LOCATION: ()..() </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;223&gt; OTHER INFORMATION: CDR3alpha sequence </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;400&gt; SEQUENCE: 43 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Ser Leu Thr Arg Pro Tyr Leu </td>
              </tr> <tr class="description-tr"> <td class="description-td">1               5 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;210&gt; SEQ ID NO 44 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;211&gt; LENGTH: 7 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;212&gt; TYPE: PRT </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;213&gt; ORGANISM: ARTIFICIAL SEQUENCE </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;220&gt; FEATURE: </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;221&gt; NAME/KEY: misc_feature </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;222&gt; LOCATION: ()..() </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;223&gt; OTHER INFORMATION: CDR3alpha sequence </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;400&gt; SEQUENCE: 44 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Ser Thr Tyr Arg His Tyr Leu </td>
              </tr> <tr class="description-tr"> <td class="description-td">1               5 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;210&gt; SEQ ID NO 45 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;211&gt; LENGTH: 7 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;212&gt; TYPE: PRT </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;213&gt; ORGANISM: ARTIFICIAL SEQUENCE </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;220&gt; FEATURE: </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;221&gt; NAME/KEY: misc_feature </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;222&gt; LOCATION: ()..() </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;223&gt; OTHER INFORMATION: CDR3alpha sequence </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;400&gt; SEQUENCE: 45 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Ser Gly Leu Ala Arg Pro Leu </td>
              </tr> <tr class="description-tr"> <td class="description-td">1               5 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;210&gt; SEQ ID NO 46 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;211&gt; LENGTH: 7 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;212&gt; TYPE: PRT </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;213&gt; ORGANISM: ARTIFICIAL SEQUENCE </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;220&gt; FEATURE: </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;221&gt; NAME/KEY: misc_feature </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;222&gt; LOCATION: ()..() </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;223&gt; OTHER INFORMATION: CDR3alpha sequence </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;400&gt; SEQUENCE: 46 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Ser Leu His Arg Pro Ala Leu </td>
              </tr> <tr class="description-tr"> <td class="description-td">1               5 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;210&gt; SEQ ID NO 47 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;211&gt; LENGTH: 7 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;212&gt; TYPE: PRT </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;213&gt; ORGANISM: ARTIFICIAL SEQUENCE </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;220&gt; FEATURE: </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;221&gt; NAME/KEY: misc_feature </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;222&gt; LOCATION: ()..() </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;223&gt; OTHER INFORMATION: CDR3alpha sequence </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;400&gt; SEQUENCE: 47 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Ser Gly Thr His Pro Phe Leu </td>
              </tr> <tr class="description-tr"> <td class="description-td">1               5 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;210&gt; SEQ ID NO 48 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;211&gt; LENGTH: 7 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;212&gt; TYPE: PRT </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;213&gt; ORGANISM: ARTIFICIAL SEQUENCE </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;220&gt; FEATURE: </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;221&gt; NAME/KEY: misc_feature </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;222&gt; LOCATION: ()..() </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;223&gt; OTHER INFORMATION: CDR3beta sequence </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;400&gt; SEQUENCE: 48 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Gly Gly Gly Gly Thr Leu Tyr </td>
              </tr> <tr class="description-tr"> <td class="description-td">1               5 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;210&gt; SEQ ID NO 49 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;211&gt; LENGTH: 7 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;212&gt; TYPE: PRT </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;213&gt; ORGANISM: ARTIFICIAL SEQUENCE </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;220&gt; FEATURE: </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;221&gt; NAME/KEY: misc_feature </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;222&gt; LOCATION: ()..() </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;223&gt; OTHER INFORMATION: CDR3beta sequence </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;400&gt; SEQUENCE: 49 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Gly Gly Gly Gly Val Leu Tyr </td>
              </tr> <tr class="description-tr"> <td class="description-td">1               5 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;210&gt; SEQ ID NO 50 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;211&gt; LENGTH: 7 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;212&gt; TYPE: PRT </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;213&gt; ORGANISM: ARTIFICIAL SEQUENCE </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;220&gt; FEATURE: </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;221&gt; NAME/KEY: misc_feature </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;222&gt; LOCATION: ()..() </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;223&gt; OTHER INFORMATION: CDR3beta sequence </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;400&gt; SEQUENCE: 50 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Gly Leu Gly Gly Ile Leu Tyr </td>
              </tr> <tr class="description-tr"> <td class="description-td">1               5 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;210&gt; SEQ ID NO 51 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;211&gt; LENGTH: 7 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;212&gt; TYPE: PRT </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;213&gt; ORGANISM: ARTIFICIAL SEQUENCE </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;220&gt; FEATURE: </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;221&gt; NAME/KEY: misc_feature </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;222&gt; LOCATION: ()..() </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;223&gt; OTHER INFORMATION: CDR3beta sequence </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;400&gt; SEQUENCE: 51 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Gly Gln Gly Gly Val Leu Tyr </td>
              </tr> <tr class="description-tr"> <td class="description-td">1               5 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;210&gt; SEQ ID NO 52 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;211&gt; LENGTH: 7 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;212&gt; TYPE: PRT </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;213&gt; ORGANISM: ARTIFICIAL SEQUENCE </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;220&gt; FEATURE: </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;221&gt; NAME/KEY: misc_feature </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;222&gt; LOCATION: ()..() </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;223&gt; OTHER INFORMATION: CDR3beta sequence </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;400&gt; SEQUENCE: 52 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Gly Ser Gly Gly Ile Ile Tyr </td>
              </tr> <tr class="description-tr"> <td class="description-td">1               5 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;210&gt; SEQ ID NO 53 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;211&gt; LENGTH: 7 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;212&gt; TYPE: PRT </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;213&gt; ORGANISM: ARTIFICIAL SEQUENCE </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;220&gt; FEATURE: </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;221&gt; NAME/KEY: misc_feature </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;222&gt; LOCATION: ()..() </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;223&gt; OTHER INFORMATION: CDR3beta sequence </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;400&gt; SEQUENCE: 53 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Gly Gly Gly Gly Ile Leu Tyr </td>
              </tr> <tr class="description-tr"> <td class="description-td">1               5</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    </div></div></div><div class="patent-section patent-tabular-section"><a id="backward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Cited Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US3817837">US3817837</a></td><td class="patent-data-table-td patent-date-value">Nov 6, 1972</td><td class="patent-data-table-td patent-date-value">Jun 18, 1974</td><td class="patent-data-table-td ">Syva Corp</td><td class="patent-data-table-td ">Enzyme amplification assay</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US3850752">US3850752</a></td><td class="patent-data-table-td patent-date-value">Oct 29, 1971</td><td class="patent-data-table-td patent-date-value">Nov 26, 1974</td><td class="patent-data-table-td ">Akzona Inc</td><td class="patent-data-table-td ">Process for the demonstration and determination of low molecular compounds and of proteins capable of binding these compounds specifically</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US3927193">US3927193</a></td><td class="patent-data-table-td patent-date-value">May 18, 1973</td><td class="patent-data-table-td patent-date-value">Dec 16, 1975</td><td class="patent-data-table-td ">Hoffmann La Roche</td><td class="patent-data-table-td ">Localization of tumors by radiolabelled antibodies</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US3939350">US3939350</a></td><td class="patent-data-table-td patent-date-value">Apr 29, 1974</td><td class="patent-data-table-td patent-date-value">Feb 17, 1976</td><td class="patent-data-table-td ">Board Of Trustees Of The Leland Stanford Junior University</td><td class="patent-data-table-td ">Fluorescent immunoassay employing total reflection for activation</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US3996345">US3996345</a></td><td class="patent-data-table-td patent-date-value">Jun 30, 1975</td><td class="patent-data-table-td patent-date-value">Dec 7, 1976</td><td class="patent-data-table-td ">Syva Company</td><td class="patent-data-table-td ">Fluorescence quenching with immunological pairs in immunoassays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4275149">US4275149</a></td><td class="patent-data-table-td patent-date-value">Nov 24, 1978</td><td class="patent-data-table-td patent-date-value">Jun 23, 1981</td><td class="patent-data-table-td ">Syva Company</td><td class="patent-data-table-td ">Immunoassay</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4277437">US4277437</a></td><td class="patent-data-table-td patent-date-value">Dec 10, 1979</td><td class="patent-data-table-td patent-date-value">Jul 7, 1981</td><td class="patent-data-table-td ">Syva Company</td><td class="patent-data-table-td ">Chemiluminescent source, quencher molecule</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4331647">US4331647</a></td><td class="patent-data-table-td patent-date-value">Mar 3, 1980</td><td class="patent-data-table-td patent-date-value">May 25, 1982</td><td class="patent-data-table-td ">Goldenberg Milton David</td><td class="patent-data-table-td ">Tumor localization and therapy with labeled antibody fragments specific to tumor-associated markers</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4348376">US4348376</a></td><td class="patent-data-table-td patent-date-value">Mar 3, 1980</td><td class="patent-data-table-td patent-date-value">Sep 7, 1982</td><td class="patent-data-table-td ">Goldenberg Milton David</td><td class="patent-data-table-td ">Tumor localization and therapy with labeled anti-CEA antibody</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4361544">US4361544</a></td><td class="patent-data-table-td patent-date-value">Mar 3, 1980</td><td class="patent-data-table-td patent-date-value">Nov 30, 1982</td><td class="patent-data-table-td ">Goldenberg Milton David</td><td class="patent-data-table-td ">Tumor localization and therapy with labeled antibodies specific to intracellular tumor-associated markers</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4366241">US4366241</a></td><td class="patent-data-table-td patent-date-value">Aug 7, 1980</td><td class="patent-data-table-td patent-date-value">Dec 28, 1982</td><td class="patent-data-table-td ">Syva Company</td><td class="patent-data-table-td ">Concentrating zone method in heterogeneous immunoassays</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4444744">US4444744</a></td><td class="patent-data-table-td patent-date-value">Sep 3, 1982</td><td class="patent-data-table-td patent-date-value">Apr 24, 1984</td><td class="patent-data-table-td ">Goldenberg Milton David</td><td class="patent-data-table-td ">Tumor localization and therapy with labeled antibodies to cell surface antigens</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4468457">US4468457</a></td><td class="patent-data-table-td patent-date-value">Jun 1, 1981</td><td class="patent-data-table-td patent-date-value">Aug 28, 1984</td><td class="patent-data-table-td ">David M. Goldenberg</td><td class="patent-data-table-td ">Method for producing a CSAp tryptic peptide and anti-CSAp antibodies</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4640561">US4640561</a></td><td class="patent-data-table-td patent-date-value">Nov 15, 1985</td><td class="patent-data-table-td patent-date-value">Feb 3, 1987</td><td class="patent-data-table-td ">Ford Motor Company</td><td class="patent-data-table-td ">Flexible printed circuit connector</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4713332">US4713332</a></td><td class="patent-data-table-td patent-date-value">Feb 6, 1984</td><td class="patent-data-table-td patent-date-value">Dec 15, 1987</td><td class="patent-data-table-td ">The Ontario Cancer Institute</td><td class="patent-data-table-td ">T cell specific CDNA clone</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4831122">US4831122</a></td><td class="patent-data-table-td patent-date-value">Jan 9, 1986</td><td class="patent-data-table-td patent-date-value">May 16, 1989</td><td class="patent-data-table-td ">Regents Of The University Of Minnesota</td><td class="patent-data-table-td ">Radioimmunotoxins</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4873190">US4873190</a></td><td class="patent-data-table-td patent-date-value">Oct 31, 1984</td><td class="patent-data-table-td patent-date-value">Oct 10, 1989</td><td class="patent-data-table-td ">Massachusetts Institute Of Technology</td><td class="patent-data-table-td ">Heterodimeric T lymphocyte receptor</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4874845">US4874845</a></td><td class="patent-data-table-td patent-date-value">Jun 13, 1984</td><td class="patent-data-table-td patent-date-value">Oct 17, 1989</td><td class="patent-data-table-td ">Massachusetts Institute Of Technology</td><td class="patent-data-table-td ">T lymphocyte receptor subunit</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4885248">US4885248</a></td><td class="patent-data-table-td patent-date-value">Mar 9, 1987</td><td class="patent-data-table-td patent-date-value">Dec 5, 1989</td><td class="patent-data-table-td ">Lubrizol Genetics, Inc.</td><td class="patent-data-table-td ">Restriction endonuclease site at transcription initiation point</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4923799">US4923799</a></td><td class="patent-data-table-td patent-date-value">Nov 9, 1987</td><td class="patent-data-table-td patent-date-value">May 8, 1990</td><td class="patent-data-table-td ">The Ontario Cancer Institute</td><td class="patent-data-table-td ">T cell specific cDNA clone</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4946778">US4946778</a></td><td class="patent-data-table-td patent-date-value">Jan 19, 1989</td><td class="patent-data-table-td patent-date-value">Aug 7, 1990</td><td class="patent-data-table-td ">Genex Corporation</td><td class="patent-data-table-td ">Single polypeptide chain binding molecules</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US4970296">US4970296</a></td><td class="patent-data-table-td patent-date-value">Jul 27, 1989</td><td class="patent-data-table-td patent-date-value">Nov 13, 1990</td><td class="patent-data-table-td ">Massachusetts Institute Of Technology</td><td class="patent-data-table-td ">Units composed of signal peptide, transmembrane, and cytoplasmic segments</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5013650">US5013650</a></td><td class="patent-data-table-td patent-date-value">Aug 25, 1988</td><td class="patent-data-table-td patent-date-value">May 7, 1991</td><td class="patent-data-table-td ">Merck &amp; Co., Inc.</td><td class="patent-data-table-td ">Increased fermentation yield of expressed protein</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5024940">US5024940</a></td><td class="patent-data-table-td patent-date-value">Oct 29, 1987</td><td class="patent-data-table-td patent-date-value">Jun 18, 1991</td><td class="patent-data-table-td ">T Cell Sciences, Inc.</td><td class="patent-data-table-td ">Nucleic acids encoding the delta chain of the T cell antigen receptor</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5059413">US5059413</a></td><td class="patent-data-table-td patent-date-value">Apr 18, 1988</td><td class="patent-data-table-td patent-date-value">Oct 22, 1991</td><td class="patent-data-table-td ">Xoma Corporation</td><td class="patent-data-table-td ">Scintigraphic monitoring of immunotoxins using radionuclides and heterobifunctional chelators</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5101827">US5101827</a></td><td class="patent-data-table-td patent-date-value">Mar 11, 1988</td><td class="patent-data-table-td patent-date-value">Apr 7, 1992</td><td class="patent-data-table-td ">Immunomedics, Inc.</td><td class="patent-data-table-td ">Lymphographic and organ imaging method and kit</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5173410">US5173410</a></td><td class="patent-data-table-td patent-date-value">Oct 3, 1989</td><td class="patent-data-table-td patent-date-value">Dec 22, 1992</td><td class="patent-data-table-td ">Lubrizol Genetics Inc.</td><td class="patent-data-table-td ">Providing a complementary DNA sequence corresponding to the RNA transscript desired attached to a promoter sequence, replicating DNA, placing replicated DNA in a vitro environment comprising RNA polymerase and isolating RNA transcripts</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5185250">US5185250</a></td><td class="patent-data-table-td patent-date-value">Jan 13, 1989</td><td class="patent-data-table-td patent-date-value">Feb 9, 1993</td><td class="patent-data-table-td ">T Cell Sciences, Inc.</td><td class="patent-data-table-td ">Human γ, δT cell antigen receptor polypeptides and nucleic acids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5223409">US5223409</a></td><td class="patent-data-table-td patent-date-value">Mar 1, 1991</td><td class="patent-data-table-td patent-date-value">Jun 29, 1993</td><td class="patent-data-table-td ">Protein Engineering Corp.</td><td class="patent-data-table-td ">Directed evolution of novel binding proteins</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5225539">US5225539</a></td><td class="patent-data-table-td patent-date-value">Oct 25, 1991</td><td class="patent-data-table-td patent-date-value">Jul 6, 1993</td><td class="patent-data-table-td ">Medical Research Council</td><td class="patent-data-table-td ">Recombinant altered antibodies and methods of making altered antibodies</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5258289">US5258289</a></td><td class="patent-data-table-td patent-date-value">Oct 22, 1991</td><td class="patent-data-table-td patent-date-value">Nov 2, 1993</td><td class="patent-data-table-td ">Davis Claude G</td><td class="patent-data-table-td ">Specific cleavage of immunoglobulin E class peptides, allergy treatment</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5258498">US5258498</a></td><td class="patent-data-table-td patent-date-value">May 19, 1988</td><td class="patent-data-table-td patent-date-value">Nov 2, 1993</td><td class="patent-data-table-td ">Creative Biomolecules, Inc.</td><td class="patent-data-table-td ">Targeted multifunctional proteins, immunological treatment of cancer and infection</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5260203">US5260203</a></td><td class="patent-data-table-td patent-date-value">Apr 25, 1990</td><td class="patent-data-table-td patent-date-value">Nov 9, 1993</td><td class="patent-data-table-td ">Enzon, Inc.</td><td class="patent-data-table-td ">Computer based system determined chemical structure for converting two naturally aggregated but chemically separated light and heavy polpeptide chain into single chain having three dimensional structure</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5299253">US5299253</a></td><td class="patent-data-table-td patent-date-value">Apr 10, 1992</td><td class="patent-data-table-td patent-date-value">Mar 29, 1994</td><td class="patent-data-table-td ">Akzo N.V.</td><td class="patent-data-table-td ">Alignment system to overlay abdominal computer aided tomography and magnetic resonance anatomy with single photon emission tomography</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5316922">US5316922</a></td><td class="patent-data-table-td patent-date-value">Apr 15, 1992</td><td class="patent-data-table-td patent-date-value">May 31, 1994</td><td class="patent-data-table-td ">The United States Of America As Represented By Department Of Health And Human Services</td><td class="patent-data-table-td ">Binding proteins, ligation polynucleotides and insertion gene coding</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5316925">US5316925</a></td><td class="patent-data-table-td patent-date-value">Dec 3, 1992</td><td class="patent-data-table-td patent-date-value">May 31, 1994</td><td class="patent-data-table-td ">Davis Mark M</td><td class="patent-data-table-td ">T-cell receptor specific for antigen polypeptides and related polynucleotides</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5340921">US5340921</a></td><td class="patent-data-table-td patent-date-value">Feb 19, 1987</td><td class="patent-data-table-td patent-date-value">Aug 23, 1994</td><td class="patent-data-table-td ">T Cell Sciences, Inc.</td><td class="patent-data-table-td ">Γ, δT cell receptor and methods and detection</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5348867">US5348867</a></td><td class="patent-data-table-td patent-date-value">Nov 15, 1991</td><td class="patent-data-table-td patent-date-value">Sep 20, 1994</td><td class="patent-data-table-td ">George Georgiou</td><td class="patent-data-table-td ">Expression of proteins on bacterial surface</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5403484">US5403484</a></td><td class="patent-data-table-td patent-date-value">Jan 26, 1993</td><td class="patent-data-table-td patent-date-value">Apr 4, 1995</td><td class="patent-data-table-td ">Protein Engineering Corporation</td><td class="patent-data-table-td ">Viruses expressing chimeric binding proteins</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5411873">US5411873</a></td><td class="patent-data-table-td patent-date-value">Aug 11, 1992</td><td class="patent-data-table-td patent-date-value">May 2, 1995</td><td class="patent-data-table-td ">Genencor, Inc.</td><td class="patent-data-table-td ">Process for producing heterologous polypeptides</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5427908">US5427908</a></td><td class="patent-data-table-td patent-date-value">May 1, 1990</td><td class="patent-data-table-td patent-date-value">Jun 27, 1995</td><td class="patent-data-table-td ">Affymax Technologies N.V.</td><td class="patent-data-table-td ">Transforming cell with filamentous bacteriophage expression vector, culturing for expression and assembly of particle with antibody Fab fragment on coat, selecting encoding particle by binding to specific ligand, removing nonbinding particles</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5466788">US5466788</a></td><td class="patent-data-table-td patent-date-value">Aug 25, 1994</td><td class="patent-data-table-td patent-date-value">Nov 14, 1995</td><td class="patent-data-table-td ">Mycogen Plant Science, Inc.</td><td class="patent-data-table-td ">Subgenomic promoter</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5482858">US5482858</a></td><td class="patent-data-table-td patent-date-value">Oct 19, 1993</td><td class="patent-data-table-td patent-date-value">Jan 9, 1996</td><td class="patent-data-table-td ">Creative Biomolecules, Inc.</td><td class="patent-data-table-td ">Polypeptide linkers for production of biosynthetic proteins</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5498530">US5498530</a></td><td class="patent-data-table-td patent-date-value">Aug 15, 1994</td><td class="patent-data-table-td patent-date-value">Mar 12, 1996</td><td class="patent-data-table-td ">Affymax Technologies, N.V.</td><td class="patent-data-table-td ">Dna binding peptides inserting coding sequence and transformation of host cells and culturing cells</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5500360">US5500360</a></td><td class="patent-data-table-td patent-date-value">Mar 14, 1994</td><td class="patent-data-table-td patent-date-value">Mar 19, 1996</td><td class="patent-data-table-td ">Mycogen Plant Science, Inc.</td><td class="patent-data-table-td ">RNA transformation vector</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5510240">US5510240</a></td><td class="patent-data-table-td patent-date-value">Feb 8, 1993</td><td class="patent-data-table-td patent-date-value">Apr 23, 1996</td><td class="patent-data-table-td ">The Arizona Board Of Regents</td><td class="patent-data-table-td ">Method of screening a peptide library</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5571698">US5571698</a></td><td class="patent-data-table-td patent-date-value">Jun 18, 1993</td><td class="patent-data-table-td patent-date-value">Nov 5, 1996</td><td class="patent-data-table-td ">Protein Engineering Corporation</td><td class="patent-data-table-td ">Genetic package, chimeric</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5580717">US5580717</a></td><td class="patent-data-table-td patent-date-value">Jan 20, 1995</td><td class="patent-data-table-td patent-date-value">Dec 3, 1996</td><td class="patent-data-table-td ">Affymax Technologies N.V.</td><td class="patent-data-table-td ">Recombinant library screening methods</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5580961">US5580961</a></td><td class="patent-data-table-td patent-date-value">Mar 18, 1994</td><td class="patent-data-table-td patent-date-value">Dec 3, 1996</td><td class="patent-data-table-td ">Massachusetts Institute Of Technology</td><td class="patent-data-table-td ">Heterodimeric T lymphocyte receptor subunit</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5601822">US5601822</a></td><td class="patent-data-table-td patent-date-value">Nov 19, 1993</td><td class="patent-data-table-td patent-date-value">Feb 11, 1997</td><td class="patent-data-table-td ">T Cell Sciences, Inc.</td><td class="patent-data-table-td ">γ, δ T cell receptor and methods for detection</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5602242">US5602242</a></td><td class="patent-data-table-td patent-date-value">May 22, 1995</td><td class="patent-data-table-td patent-date-value">Feb 11, 1997</td><td class="patent-data-table-td ">Mycogen Plant Science, Inc.</td><td class="patent-data-table-td ">Genetic engineered, encapsidated protein shell transfecting the plant cells</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5614192">US5614192</a></td><td class="patent-data-table-td patent-date-value">Mar 16, 1993</td><td class="patent-data-table-td patent-date-value">Mar 25, 1997</td><td class="patent-data-table-td ">Connective Therapeutics, Inc.</td><td class="patent-data-table-td ">T cell receptor peptides as therapeutics for immune-related disease</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5624817">US5624817</a></td><td class="patent-data-table-td patent-date-value">Apr 28, 1994</td><td class="patent-data-table-td patent-date-value">Apr 29, 1997</td><td class="patent-data-table-td ">The Research Foundation Of State University Of New York</td><td class="patent-data-table-td ">Mutations in the gene encoding the alpha chain of platelet glycoprotein Ib</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5627060">US5627060</a></td><td class="patent-data-table-td patent-date-value">Jun 7, 1995</td><td class="patent-data-table-td patent-date-value">May 6, 1997</td><td class="patent-data-table-td ">Mycogen Plant Science, Inc.</td><td class="patent-data-table-td ">Hybrid RNA virus</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5633447">US5633447</a></td><td class="patent-data-table-td patent-date-value">Jun 2, 1995</td><td class="patent-data-table-td patent-date-value">May 27, 1997</td><td class="patent-data-table-td ">Mycogen Plant Science, Inc.</td><td class="patent-data-table-td ">Plant tissue comprising a subgenomic promoter</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5635363">US5635363</a></td><td class="patent-data-table-td patent-date-value">Feb 28, 1995</td><td class="patent-data-table-td patent-date-value">Jun 3, 1997</td><td class="patent-data-table-td ">The Board Of Trustees Of The Leland Stanford Junior University</td><td class="patent-data-table-td ">Compositions and methods for the detection, quantitation and purification of antigen-specific T cells</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5670353">US5670353</a></td><td class="patent-data-table-td patent-date-value">Jun 2, 1995</td><td class="patent-data-table-td patent-date-value">Sep 23, 1997</td><td class="patent-data-table-td ">Mycogen Plant Science, Inc.</td><td class="patent-data-table-td ">Directs the amplified expression of a structural gene in plant tissue</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5723286">US5723286</a></td><td class="patent-data-table-td patent-date-value">Jun 20, 1990</td><td class="patent-data-table-td patent-date-value">Mar 3, 1998</td><td class="patent-data-table-td ">Affymax Technologies N.V.</td><td class="patent-data-table-td ">Peptide library and screening systems</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5723309">US5723309</a></td><td class="patent-data-table-td patent-date-value">Nov 25, 1993</td><td class="patent-data-table-td patent-date-value">Mar 3, 1998</td><td class="patent-data-table-td ">Institut National De La Sante Et De La Recherche Medicale (Inserm)</td><td class="patent-data-table-td ">Production of subunits of soluble T cell receptors by co-transfection</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5723323">US5723323</a></td><td class="patent-data-table-td patent-date-value">Dec 2, 1994</td><td class="patent-data-table-td patent-date-value">Mar 3, 1998</td><td class="patent-data-table-td ">Kauffman; Stuart Alan</td><td class="patent-data-table-td ">Method of identifying a stochastically-generated peptide, polypeptide, or protein having ligand binding property and compositions thereof</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5733743">US5733743</a></td><td class="patent-data-table-td patent-date-value">Mar 24, 1993</td><td class="patent-data-table-td patent-date-value">Mar 31, 1998</td><td class="patent-data-table-td ">Cambridge Antibody Technology Limited</td><td class="patent-data-table-td ">Genetic engineering</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5738996">US5738996</a></td><td class="patent-data-table-td patent-date-value">Jun 15, 1994</td><td class="patent-data-table-td patent-date-value">Apr 14, 1998</td><td class="patent-data-table-td ">Pence, Inc.</td><td class="patent-data-table-td ">Used in selecting an oligomer compound capable of interacting specifically with a selected macromolecular receptor</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5763192">US5763192</a></td><td class="patent-data-table-td patent-date-value">Jun 5, 1995</td><td class="patent-data-table-td patent-date-value">Jun 9, 1998</td><td class="patent-data-table-td ">Ixsys, Incorporated</td><td class="patent-data-table-td ">Process for obtaining DNA, RNA, peptides, polypeptides, or protein, by recombinant DNA technique</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5763733">US5763733</a></td><td class="patent-data-table-td patent-date-value">Oct 13, 1994</td><td class="patent-data-table-td patent-date-value">Jun 9, 1998</td><td class="patent-data-table-td ">Enzon, Inc.</td><td class="patent-data-table-td ">Antigen-binding fusion proteins</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5767260">US5767260</a></td><td class="patent-data-table-td patent-date-value">Aug 16, 1995</td><td class="patent-data-table-td patent-date-value">Jun 16, 1998</td><td class="patent-data-table-td ">Enzon Inc.</td><td class="patent-data-table-td ">A nucleic acid coding for two epitope-containing polypeptide joined by a peptide linker and attached to phospholipase a activating protein or tumor necrosis factor; antiinflammatory agents</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5780225">US5780225</a></td><td class="patent-data-table-td patent-date-value">Sep 29, 1994</td><td class="patent-data-table-td patent-date-value">Jul 14, 1998</td><td class="patent-data-table-td ">Stratagene</td><td class="patent-data-table-td ">Method for generating libaries of antibody genes comprising amplification of diverse antibody DNAs and methods for using these libraries for the production of diverse antigen combining molecules</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5789208">US5789208</a></td><td class="patent-data-table-td patent-date-value">Feb 19, 1997</td><td class="patent-data-table-td patent-date-value">Aug 4, 1998</td><td class="patent-data-table-td ">The Trustees Of Boston University</td><td class="patent-data-table-td ">Polyclonal antibody libraries</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5814476">US5814476</a></td><td class="patent-data-table-td patent-date-value">Jun 5, 1995</td><td class="patent-data-table-td patent-date-value">Sep 29, 1998</td><td class="patent-data-table-td ">Stuart Kauffman</td><td class="patent-data-table-td ">Process for the production of stochastically-generated transcription or translation products</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5817483">US5817483</a></td><td class="patent-data-table-td patent-date-value">Jun 5, 1995</td><td class="patent-data-table-td patent-date-value">Oct 6, 1998</td><td class="patent-data-table-td ">Stuart Kauffman</td><td class="patent-data-table-td ">Process for the production of stochastically-generated peptides,polypeptides or proteins having a predetermined property</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5824483">US5824483</a></td><td class="patent-data-table-td patent-date-value">Jun 16, 1995</td><td class="patent-data-table-td patent-date-value">Oct 20, 1998</td><td class="patent-data-table-td ">Pence Inc.</td><td class="patent-data-table-td ">Conformationally-restricted combinatiorial library composition and method</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5824514">US5824514</a></td><td class="patent-data-table-td patent-date-value">Jun 5, 1995</td><td class="patent-data-table-td patent-date-value">Oct 20, 1998</td><td class="patent-data-table-td ">Stuart A. Kauffman</td><td class="patent-data-table-td ">Process for obtaining dna, rna, peptides, polypeptides, or proteins, through use of transformed host cells containing genes capable of expressing these products</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5837477">US5837477</a></td><td class="patent-data-table-td patent-date-value">Jan 15, 1993</td><td class="patent-data-table-td patent-date-value">Nov 17, 1998</td><td class="patent-data-table-td ">The United States Of America As Represented By The Department Of Health And Human Services</td><td class="patent-data-table-td ">Hybrid; immunomodulatory properties</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5837500">US5837500</a></td><td class="patent-data-table-td patent-date-value">Apr 3, 1995</td><td class="patent-data-table-td patent-date-value">Nov 17, 1998</td><td class="patent-data-table-td ">Dyax, Corp.</td><td class="patent-data-table-td ">Gene expression; genetic engineering; mutation</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5840304">US5840304</a></td><td class="patent-data-table-td patent-date-value">Apr 29, 1994</td><td class="patent-data-table-td patent-date-value">Nov 24, 1998</td><td class="patent-data-table-td ">Bd. Of Trustees Of The Leland Stanford Junior University</td><td class="patent-data-table-td ">Amino acid sequences and codes for t-cells</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5858657">US5858657</a></td><td class="patent-data-table-td patent-date-value">Jun 7, 1995</td><td class="patent-data-table-td patent-date-value">Jan 12, 1999</td><td class="patent-data-table-td ">Medical Research Council</td><td class="patent-data-table-td ">Introducing into host cells vectors containing nucleic acids coding for polypeptide chains (e.g. immunoglobulins) which are displayed on surface of secreted replicable genetic display package (e.g. phage) forming library</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5861156">US5861156</a></td><td class="patent-data-table-td patent-date-value">Jan 8, 1993</td><td class="patent-data-table-td patent-date-value">Jan 19, 1999</td><td class="patent-data-table-td ">Creative Biomolecules</td><td class="patent-data-table-td ">Methods of delivering agents to target cells</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5866344">US5866344</a></td><td class="patent-data-table-td patent-date-value">May 23, 1995</td><td class="patent-data-table-td patent-date-value">Feb 2, 1999</td><td class="patent-data-table-td ">Board Of Regents, The University Of Texas System</td><td class="patent-data-table-td ">Antibody selection methods using cell surface expressed libraries</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5866363">US5866363</a></td><td class="patent-data-table-td patent-date-value">Feb 28, 1991</td><td class="patent-data-table-td patent-date-value">Feb 2, 1999</td><td class="patent-data-table-td ">Pieczenik; George</td><td class="patent-data-table-td ">Method and means for sorting and identifying biological information</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5869620">US5869620</a></td><td class="patent-data-table-td patent-date-value">Feb 22, 1995</td><td class="patent-data-table-td patent-date-value">Feb 9, 1999</td><td class="patent-data-table-td ">Enzon, Inc.</td><td class="patent-data-table-td ">Multivalent antigen-binding proteins</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5871907">US5871907</a></td><td class="patent-data-table-td patent-date-value">May 15, 1992</td><td class="patent-data-table-td patent-date-value">Feb 16, 1999</td><td class="patent-data-table-td ">Medical Research Council</td><td class="patent-data-table-td ">Methods for producing members of specific binding pairs</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5871974">US5871974</a></td><td class="patent-data-table-td patent-date-value">Dec 2, 1994</td><td class="patent-data-table-td patent-date-value">Feb 16, 1999</td><td class="patent-data-table-td ">Ixsys Inc.</td><td class="patent-data-table-td ">Recombinant expression of heteromeric receptors on the surface of filamentous bacteriophage</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5882945">US5882945</a></td><td class="patent-data-table-td patent-date-value">Jun 21, 1994</td><td class="patent-data-table-td patent-date-value">Mar 16, 1999</td><td class="patent-data-table-td ">Massachusetts Institute Of Technology</td><td class="patent-data-table-td ">Isolation, expression, immunizing and antibodies</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5900476">US5900476</a></td><td class="patent-data-table-td patent-date-value">Feb 26, 1992</td><td class="patent-data-table-td patent-date-value">May 4, 1999</td><td class="patent-data-table-td ">The Scripps Research Institute</td><td class="patent-data-table-td ">Mutant polypeptide without cysteines, or modified to prevent disulfide bond formation suppresses binding of factor to platelets; antithrombosis agents</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5948409">US5948409</a></td><td class="patent-data-table-td patent-date-value">May 19, 1997</td><td class="patent-data-table-td patent-date-value">Sep 7, 1999</td><td class="patent-data-table-td ">The United States Of America As Represented By The Department Of Health And Human Services</td><td class="patent-data-table-td ">T cell receptor ligands and methods of using same</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5969108">US5969108</a></td><td class="patent-data-table-td patent-date-value">Jul 10, 1991</td><td class="patent-data-table-td patent-date-value">Oct 19, 1999</td><td class="patent-data-table-td ">Medical Research Council</td><td class="patent-data-table-td ">Methods for producing members of specific binding pairs</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5977321">US5977321</a></td><td class="patent-data-table-td patent-date-value">Aug 8, 1991</td><td class="patent-data-table-td patent-date-value">Nov 2, 1999</td><td class="patent-data-table-td ">Massachusetts Institute Of Technology</td><td class="patent-data-table-td ">The subunit consists of variable, joining, constant, transmembrane, and cytoplasmic regions.</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6017732">US6017732</a></td><td class="patent-data-table-td patent-date-value">Jun 30, 1994</td><td class="patent-data-table-td patent-date-value">Jan 25, 2000</td><td class="patent-data-table-td ">Medical Research Council</td><td class="patent-data-table-td ">To screen for enhanced specfic binding pair binding</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6027910">US6027910</a></td><td class="patent-data-table-td patent-date-value">Jul 7, 1993</td><td class="patent-data-table-td patent-date-value">Feb 22, 2000</td><td class="patent-data-table-td ">Unilever Patent Holdings B.V.</td><td class="patent-data-table-td ">Process for immobilizing enzymes to the cell wall of a microbial cell by producing a fusion protein</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6087096">US6087096</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Nov 13, 1995</td><td class="patent-data-table-td patent-date-value">Jul 11, 2000</td><td class="patent-data-table-td ">Dau; Peter C.</td><td class="patent-data-table-td ">Screening lymphocyte propagation in human; obtaining cells, generation of genetic map, comparing control genetic map to the genetic map of sample</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6114147">US6114147</a></td><td class="patent-data-table-td patent-date-value">Feb 10, 1994</td><td class="patent-data-table-td patent-date-value">Sep 5, 2000</td><td class="patent-data-table-td ">Unilever Patent Holdings</td><td class="patent-data-table-td ">Immobilizing protein by modifying a fungus to produce a chimeric protein composed of a binding protein, a secretory signal peptide, and an anchoring portion; producing protein; transporting to cell wall; for separating target compounds</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6232099">US6232099</a></td><td class="patent-data-table-td patent-date-value">Apr 18, 1997</td><td class="patent-data-table-td patent-date-value">May 15, 2001</td><td class="patent-data-table-td ">Scottish Crop Research Institute</td><td class="patent-data-table-td ">Method of producing a chimeric protein</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6319713">US6319713</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Jun 25, 1999</td><td class="patent-data-table-td patent-date-value">Nov 20, 2001</td><td class="patent-data-table-td ">Maxygen, Inc.</td><td class="patent-data-table-td ">Methods and compositions for polypeptide engineering</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6416971">US6416971</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">May 8, 1991</td><td class="patent-data-table-td patent-date-value">Jul 9, 2002</td><td class="patent-data-table-td ">E.R. Squibb &amp; Sons, Inc.</td><td class="patent-data-table-td ">Soluble, single chain t cell receptor encoded by specified nucleic acid sequence</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/USRE35500">USRE35500</a></td><td class="patent-data-table-td patent-date-value">Feb 16, 1995</td><td class="patent-data-table-td patent-date-value">May 6, 1997</td><td class="patent-data-table-td ">Aberlyn Capital Management Limited Partnership</td><td class="patent-data-table-td ">Direct radiolabeling of antibodies and other proteins with technetium or rhenium</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="http://www.google.com/url?id=W4xnBAABERAJ&amp;q=http://worldwide.espacenet.com/publicationDetails/biblio%3FCC%3DAU%26NR%3D195191B7%26KC%3DB7%26FT%3DD&amp;usg=AFQjCNHfnx0xJzC9pEIwhdnlNSZeqv_bfA">AU195191B7</a></td><td class="patent-data-table-td patent-date-value"></td><td class="patent-data-table-td patent-date-value"></td><td class="patent-data-table-td "> </td><td class="patent-data-table-td citation-no-title">Title not available</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP0067553A2?cl=en">EP0067553A2</a></td><td class="patent-data-table-td patent-date-value">May 20, 1982</td><td class="patent-data-table-td patent-date-value">Dec 22, 1982</td><td class="patent-data-table-td ">National Research Council Of Canada</td><td class="patent-data-table-td ">An RNA plant virus vector or portion thereof, a method of construction thereof, and a method of producing a gene derived product therefrom</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP0174759A1?cl=en">EP0174759A1</a></td><td class="patent-data-table-td patent-date-value">Aug 22, 1985</td><td class="patent-data-table-td patent-date-value">Mar 19, 1986</td><td class="patent-data-table-td ">Connaught Laboratories Limited</td><td class="patent-data-table-td ">Multispecific immunogenic proteins</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP0194809B1?cl=en">EP0194809B1</a></td><td class="patent-data-table-td patent-date-value">Mar 6, 1986</td><td class="patent-data-table-td patent-date-value">Mar 13, 1991</td><td class="patent-data-table-td ">Lubrizol Genetics Inc.</td><td class="patent-data-table-td ">Rna transformation vector</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP0278667A2?cl=en">EP0278667A2</a></td><td class="patent-data-table-td patent-date-value">Feb 3, 1988</td><td class="patent-data-table-td patent-date-value">Aug 17, 1988</td><td class="patent-data-table-td ">Mycogen Plant Science, Inc.</td><td class="patent-data-table-td ">Hybrid RNA virus</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP0425004A2?cl=en">EP0425004A2</a></td><td class="patent-data-table-td patent-date-value">Oct 3, 1990</td><td class="patent-data-table-td patent-date-value">May 2, 1991</td><td class="patent-data-table-td ">Aveve N.V.</td><td class="patent-data-table-td ">Genetic manipulations with recombinant DNA comprising sequences derived from RNA virus</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP0479180A2?cl=en">EP0479180A2</a></td><td class="patent-data-table-td patent-date-value">Sep 30, 1991</td><td class="patent-data-table-td patent-date-value">Apr 8, 1992</td><td class="patent-data-table-td ">Hoechst Aktiengesellschaft</td><td class="patent-data-table-td ">Virus resistant plants, method for their production</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP0672754A1?cl=en">EP0672754A1</a></td><td class="patent-data-table-td patent-date-value">Mar 31, 1993</td><td class="patent-data-table-td patent-date-value">Sep 20, 1995</td><td class="patent-data-table-td ">Kanebo Ltd.</td><td class="patent-data-table-td ">Plant virus vector, plasmid, method of expressing foreign gene, and method of acquiring foreign gene product</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="npl-citations"></a><div class="patent-section-header"><span class="patent-section-title">Non-Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th colspan="3"class="patent-data-table-th">Reference</th></tr></thead><tr><td class="patent-data-table-td ">1</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Ahlquist and Pacha, Physiol. Plant. 79:163-167 (1990).</td></tr><tr><td class="patent-data-table-td ">2</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Ahlquist et al., "<a href='http://scholar.google.com/scholar?q="Viral+Vectors%2C"'>Viral Vectors,</a>" Cold Spring Harbor Laboratory, New York 183-189 (1988).</td></tr><tr><td class="patent-data-table-td ">3</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Alam et al., (Jun. 1996), "<a href='http://scholar.google.com/scholar?q="T-cell-receptor+Affinity+and+Thymocyte+Positive+Selection%2C"'>T-cell-receptor Affinity and Thymocyte Positive Selection,</a>" Nature 381:616-620.</td></tr><tr><td class="patent-data-table-td ">4</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Alam, S.M. et al., (Feb. 1999), "<a href='http://scholar.google.com/scholar?q="Qualitative+and+Quantitative+Differences+in+T+Cell+Receptor+Binding+of+Agonist+and+Antagonist+Ligands%2C"'>Qualitative and Quantitative Differences in T Cell Receptor Binding of Agonist and Antagonist Ligands,</a>" Immunity 10:227-237.</td></tr><tr><td class="patent-data-table-td ">5</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Alberti, S., (1996), "<a href='http://scholar.google.com/scholar?q="A+high+affinity+T+cell+receptor%3F%2C"'>A high affinity T cell receptor?,</a>" Immunol. Cell Biol. 74:292-297.</td></tr><tr><td class="patent-data-table-td ">6</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Al-Ramadi BK, et al., (1995)Lack of strict correlation of functional sensitation with the apparent affinity of MHC/peptide complexes for the TCR. J. Immunol.155: 662-673.</td></tr><tr><td class="patent-data-table-td ">7</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Altschul, S.F. et al., "<a href='http://scholar.google.com/scholar?q="+Gapped+BLAST+and+PSI-BLAST%3A+a+new+generation+of+protein+database+search+programs"'> Gapped BLAST and PSI-BLAST: a new generation of protein database search programs</a>" (Sep. 1997) Nucleic Acids Research 25(17):2278-3402.</td></tr><tr><td class="patent-data-table-td ">8</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Anand, R. et al. (1992), "<a href='http://scholar.google.com/scholar?q="Progress+in+developing+methylotrophic+yeasts+as+expression+systems."'>Progress in developing methylotrophic yeasts as expression systems.</a>" TIBTECH 10:413-417.</td></tr><tr><td class="patent-data-table-td ">9</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Baldwin, R.W. and Byers, V.S., (Editors) (1985), Monoclonal Antibodies for Cancer Detection and Therapy,London Academic Press pp. 159-179.</td></tr><tr><td class="patent-data-table-td ">10</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Barton et al., Plant Physiol. 85:1103-1109 (1987).</td></tr><tr><td class="patent-data-table-td ">11</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Beeson, C. et al., (Aug. 1996), "<a href='http://scholar.google.com/scholar?q="Early+Biochemical+Signals+Arise+from+Low+Affinity+TCR-Ligand+Reactions+at+the+Cell-Cell+Interface%2C"'>Early Biochemical Signals Arise from Low Affinity TCR-Ligand Reactions at the Cell-Cell Interface,</a>" J. Exp. Med. 184:777-782.</td></tr><tr><td class="patent-data-table-td ">12</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Bellio M, et al., (1994), The &amp;b complementarity determining region 1 of a major histocompatibility complex (MHC) class 1-restricted T cell receptor is involved in the recognition of peptide/MHC I and superantigen/MHC complex. J. Exp. Med.179: 1087-1089.</td></tr><tr><td class="patent-data-table-td ">13</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Bentley, G.A. and Mariuzza, R.A., (1996), "<a href='http://scholar.google.com/scholar?q="The+Structure+of+the+T+Cell+Antigen+Receptor%2C"'>The Structure of the T Cell Antigen Receptor,</a>" Annu. Rev. Immunol. 14:563-590.</td></tr><tr><td class="patent-data-table-td ">14</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Bevan, M.J., "<a href='http://scholar.google.com/scholar?q="In+Thymic+Selection%2C+Peptide+Diversity+Gives+and+Takes+Away"'>In Thymic Selection, Peptide Diversity Gives and Takes Away</a>" (Aug. 1997), Immunity 7:175-178.</td></tr><tr><td class="patent-data-table-td ">15</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Bird, RE, et al., (1988), Single-chain antigen-binding proteins. Science. 242: 423-426.</td></tr><tr><td class="patent-data-table-td ">16</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Bjorkman, P.J., (Apr. 1997), "<a href='http://scholar.google.com/scholar?q="MHC+Restriction+in+Three+Dimensions%3A+A+View+of+T+Cell+Receptor%2FLigand+Interactions%2C"'>MHC Restriction in Three Dimensions: A View of T Cell Receptor/Ligand Interactions,</a>" Cell 89:167-170.</td></tr><tr><td class="patent-data-table-td ">17</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Boder, E.T. and Wittrup, K.D. (1995), "<a href='http://scholar.google.com/scholar?q="A+Yeast+Surface+Display+System+for+in+vitro+Affinity+Maturation+of+Antibodies%2C"'>A Yeast Surface Display System for in vitro Affinity Maturation of Antibodies,</a>" Protein Interactions, Jun. 1-4, 1995, Beckman Institute, University of Illinois, Urbana (Abstract Only).</td></tr><tr><td class="patent-data-table-td ">18</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Boder, E.T. and Wittrup, K.D., (1997), "<a href='http://scholar.google.com/scholar?q="Yeast+surface+display+for+screening+combinatorial+ploypeptide+libraries."'>Yeast surface display for screening combinatorial ploypeptide libraries.</a>" Nature Biotech,. 15(6):553-557.</td></tr><tr><td class="patent-data-table-td ">19</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Boder, E.T. and Wittrup, K.D., (Feb. 1998), "<a href='http://scholar.google.com/scholar?q="Optimal+Screening+of+Surface-Displayed+Polypeptide+Libraries%2C"'>Optimal Screening of Surface-Displayed Polypeptide Libraries,</a>" Biotech. Progress 14(1):55-62.</td></tr><tr><td class="patent-data-table-td ">20</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Boder, E.T. et al., "<a href='http://scholar.google.com/scholar?q="+Directed+Evolution+of+Antibody+Fragments+with+Monovalent+Femtomolar+Antigen-binding+Affinity"'> Directed Evolution of Antibody Fragments with Monovalent Femtomolar Antigen-binding Affinity</a>" (Sep. 2000) Proc. Nat'l. Acad. Sci. USA 97(20):10701-10705.</td></tr><tr><td class="patent-data-table-td ">21</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Boder, E.T., et al., (2000), Yeast surface display for directed evolution of protein expression, affinity, and stability. Methods Enzymol328, 430-444.</td></tr><tr><td class="patent-data-table-td ">22</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Boniface, J.J. et al., (Sep. 1999), "<a href='http://scholar.google.com/scholar?q="Thermodynamics+of+T+cell+receptor+binding+to+peptide-MHC%3A+Evidence+for+a+general+mechanism+of+molecular+scanning"'>Thermodynamics of T cell receptor binding to peptide-MHC: Evidence for a general mechanism of molecular scanning</a>" Proc. Natl. Acad. Sci. USA 96:11446-11451.</td></tr><tr><td class="patent-data-table-td ">23</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Brodnicki, TC., (1996), Reactivity and epitope mapping of single-chain T cell receptors with monoclonal antibodies. Mol. Immunol.33:253-263.</td></tr><tr><td class="patent-data-table-td ">24</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Bruening, G., "<a href='http://scholar.google.com/scholar?q="Comovirus+group%2C+C.M.I.%2FA.A.B.+Descriptions+of+plant+viruses%2C"'>Comovirus group, C.M.I./A.A.B. Descriptions of plant viruses,</a>" No. 199. Wm. Culross and Son Ltd., Coupar Angus, Perthshire, Scotland. (1978).</td></tr><tr><td class="patent-data-table-td ">25</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Buchwalder, A. et al., (1994), "<a href='http://scholar.google.com/scholar?q="Immunochemical+and+Molecular+Analysis+of+Antigen+Binding+to+Lipid+Anchored+and+Soluble+Forms+of+an+MHC+Independent+Human+%CE%B1%2F%CE%B2+T+Cell+Receptor%2C"'>Immunochemical and Molecular Analysis of Antigen Binding to Lipid Anchored and Soluble Forms of an MHC Independent Human α/β T Cell Receptor,</a>" Mol. Immunol. 31(11):857-872.</td></tr><tr><td class="patent-data-table-td ">26</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Buckholz, R.G. and Gleeson, M.A.G. (1991), "<a href='http://scholar.google.com/scholar?q="Yeast+Systems+for+the+Commercial+Production+of+Heterologous+Proteins%2C"'>Yeast Systems for the Commercial Production of Heterologous Proteins,</a>" Bio/Technol. 9:1067-1072.</td></tr><tr><td class="patent-data-table-td ">27</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Butler and Mayo, "<a href='http://scholar.google.com/scholar?q="Molecular+architecture+and+assembly+of+tobacco+mosaic+virus+particles%2C"'>Molecular architecture and assembly of tobacco mosaic virus particles,</a>" The molecular biology of the positive strand RNA Viruses, Academic Press, London:237-257 (1987).</td></tr><tr><td class="patent-data-table-td ">28</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Callan, M.F. et al., (1995), "<a href='http://scholar.google.com/scholar?q="Selection+of+T+cell+receptor+variable+gene-encoded+amino+acids+on+the+third+binding+site+loop%3A+a+factor+influencing+variable+chain+selection+in+a+T+cell+response%2C"'>Selection of T cell receptor variable gene-encoded amino acids on the third binding site loop: a factor influencing variable chain selection in a T cell response,</a>" Eur. J. Immunol. 25:1529-1534.</td></tr><tr><td class="patent-data-table-td ">29</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Cassidy and Nelson, Phytopathology 80:1037 (1990).</td></tr><tr><td class="patent-data-table-td ">30</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Chapman et al., Plant Journal 2:549 (1992).</td></tr><tr><td class="patent-data-table-td ">31</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Charoenvit et al., "<a href='http://scholar.google.com/scholar?q="Inability+of+malaria+vaccine+to+induce+antibodies+to+a+protective+epitope+within+its+sequence%2C"'>Inability of malaria vaccine to induce antibodies to a protective epitope within its sequence,</a>" Science 251:668-671 (1991).</td></tr><tr><td class="patent-data-table-td ">32</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Charoenvit et al., "<a href='http://scholar.google.com/scholar?q="Monoclonal%2C+but+not+polyclonal%2C+antibodies+protect+against+Plasmodium+yoeli+sporozoites%2C"'>Monoclonal, but not polyclonal, antibodies protect against Plasmodium yoeli sporozoites,</a>" J. Immunol. 146:1020-1025 (1991).</td></tr><tr><td class="patent-data-table-td ">33</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Cheng, Y.C., "<a href='http://scholar.google.com/scholar?q="Relationship+Between+The+Inhibition+Constant+%28K1%29+and+the+Concentration+of+Inhibitor+Which+Causes+50+Per+Cent+Inhibition+%28I50%29+of+an+Enzymatic+Reaction"'>Relationship Between The Inhibition Constant (K1) and the Concentration of Inhibitor Which Causes 50 Per Cent Inhibition (I50) of an Enzymatic Reaction</a>" (1973), Biochem. Pharm. 22:3099-3108.</td></tr><tr><td class="patent-data-table-td ">34</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Cho, B.K. et al., "<a href='http://scholar.google.com/scholar?q="A+yeast+surface+display+system+for+the+discovery+of+ligands+that+trigger+cell+activation."'>A yeast surface display system for the discovery of ligands that trigger cell activation.</a>" (Nov. 1988) J. Immunol Methods (Netherlands) 220:179-88.</td></tr><tr><td class="patent-data-table-td ">35</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Cho, BK, et al., (1995), Characterization of a single-chain antibody to the &amp;b-chain of the T cell receptor. J. Biol. Chem.270: 25819-25826.</td></tr><tr><td class="patent-data-table-td ">36</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Chung. S. et al., (Dec. 1994), "<a href='http://scholar.google.com/scholar?q="Functional+three-domain+single-chain+T-cell+receptors%2C"'>Functional three-domain single-chain T-cell receptors,</a>" Proc. Natl. Acad. Sci. USA 91:12654-12658.</td></tr><tr><td class="patent-data-table-td ">37</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Citovsky and Zambryski, BioEssays 13:373-379 (1991).</td></tr><tr><td class="patent-data-table-td ">38</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Clackson et al., (Apr. 1991), "<a href='http://scholar.google.com/scholar?q="Making+Antibody+Fragments+Using+Phage+Display+Libraries%2C"'>Making Antibody Fragments Using Phage Display Libraries,</a>" Nature 352:624-628.</td></tr><tr><td class="patent-data-table-td ">39</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Clackson, T. et al., (Aug. 1991) "<a href='http://scholar.google.com/scholar?q="Making+antibody+fragments+using+phage+display+libraries"'>Making antibody fragments using phage display libraries</a>" Nature 352:624-628.</td></tr><tr><td class="patent-data-table-td ">40</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Cochran, et al., (2000), A diverse of oligomeric class II MHC-peptide complexes for probing T-cell receptor interactions. Chemistry &amp; Biology, vol. 7:683-696.</td></tr><tr><td class="patent-data-table-td ">41</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Corr M, et al., (1994), T cell receptor-MHC class I peptide interations: affinity, kinetics, and specificity. Science265: 946-949.</td></tr><tr><td class="patent-data-table-td ">42</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Cregg, J.M. et al. (1993), "<a href='http://scholar.google.com/scholar?q="Recent+Advances+in+the+Expression+of+Foreign+Genes+in+Pichia+pastoris%2C"'>Recent Advances in the Expression of Foreign Genes in Pichia pastoris,</a>" Bio/Technol. 11:905-910.</td></tr><tr><td class="patent-data-table-td ">43</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Culver et al., in press, Virology.</td></tr><tr><td class="patent-data-table-td ">44</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Dal Porto, J. et al., (1993), "<a href='http://scholar.google.com/scholar?q="A+soluble+divalent+class+I+major+histocompatibility+complex+molecule+inhibits+alloreactive+T+cells+at+nanomolar+concentrations"'>A soluble divalent class I major histocompatibility complex molecule inhibits alloreactive T cells at nanomolar concentrations</a>" Proc. Natl. Acad. Sci. USA 90:6671-6675.</td></tr><tr><td class="patent-data-table-td ">45</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Davis et al., (Annual-1998), "<a href='http://scholar.google.com/scholar?q="Ligand+Recognition+by+%CE%B1%CE%B2+T+Cell+Receptors%2C"'>Ligand Recognition by αβ T Cell Receptors,</a>" Annu. Rev. Immunol. 16:523-544.</td></tr><tr><td class="patent-data-table-td ">46</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Davis, M.M. and Bjorkman, D.J., (Aug. 1988), "<a href='http://scholar.google.com/scholar?q="T-cell+antigen+receptor+genes+and+T-cell+recognition%2C"'>T-cell antigen receptor genes and T-cell recognition,</a>" Nature 334:395-402.</td></tr><tr><td class="patent-data-table-td ">47</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Davis, M.M. and Chien, Y., (1993), "<a href='http://scholar.google.com/scholar?q="Topology+and+affinity+of+T-cell+receptor+mediated+recognition+of+peptide-MHC+complexes%2C"'>Topology and affinity of T-cell receptor mediated recognition of peptide-MHC complexes,</a>" Curr. Opin. Immunol. 5:45-49.</td></tr><tr><td class="patent-data-table-td ">48</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Dawson and Hilf, Ann. Rev. Plant Physiol. Plant Mol. Biol. 43:527-555 (1992).</td></tr><tr><td class="patent-data-table-td ">49</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Dawson et al., "<a href='http://scholar.google.com/scholar?q="A+tobacco+mosaic+virus-hybrid+expresses+and+loses+an+added+gene%2C"'>A tobacco mosaic virus-hybrid expresses and loses an added gene,</a>" Virology 172:285-292 (1989).</td></tr><tr><td class="patent-data-table-td ">50</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Dawson et al., "<a href='http://scholar.google.com/scholar?q="cDNA+cloning+of+the+complete+genome+of+tobacco+mosaic+virus+and+production+of+infectious+transcripts%2C"'>cDNA cloning of the complete genome of tobacco mosaic virus and production of infectious transcripts,</a>" Proc. Natl. Acad. Sci. USA 83:1832-1836 (1986).</td></tr><tr><td class="patent-data-table-td ">51</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Dawson et al., "<a href='http://scholar.google.com/scholar?q="Modifications+of+the+tobacco+mosaic+virus+coat+protein+gene+affecting+replication%2C+movement%2C+and+symptomatology%2C"'>Modifications of the tobacco mosaic virus coat protein gene affecting replication, movement, and symptomatology,</a>" Phytopathol. 78:783-789 (1988).</td></tr><tr><td class="patent-data-table-td ">52</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Dawson, Adv. Virus Res. 38:307-342 (1990).</td></tr><tr><td class="patent-data-table-td ">53</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Dawson, Virology 186:359-367 (1992).</td></tr><tr><td class="patent-data-table-td ">54</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">de Kruif, J. and Logtenberg, T., (Mar. 1996), "<a href='http://scholar.google.com/scholar?q="Leucine+Zipper+Dimerized+Bivalent+and+Bispecific+scFv+Antibodies+from+a+Semi-synthetic+Antibody+Phage+Display+Library%2C"'>Leucine Zipper Dimerized Bivalent and Bispecific scFv Antibodies from a Semi-synthetic Antibody Phage Display Library,</a>" Amer. Soc. Biochem. Mol. Biol. 271(13):7630-7634.</td></tr><tr><td class="patent-data-table-td ">55</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Deom et al., "<a href='http://scholar.google.com/scholar?q="Plant+Virus+Movement+Proteins%2C"'>Plant Virus Movement Proteins,</a>" Cell 69:221-224 (1992).</td></tr><tr><td class="patent-data-table-td ">56</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Deom et al., Science 237:389-394 (1987).</td></tr><tr><td class="patent-data-table-td ">57</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Dolja et al., Proc. Natl. Acad. Sci. USA 89:10208 (1992).</td></tr><tr><td class="patent-data-table-td ">58</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Donson et al., "<a href='http://scholar.google.com/scholar?q="Systemic+expression+of+a+bacterial+gene+by+a+tobacco+mosaic+virus-based+vector%2C"'>Systemic expression of a bacterial gene by a tobacco mosaic virus-based vector,</a>" Proc. Natl. Acad. Sci. USA 88:7204-7208 (1991).</td></tr><tr><td class="patent-data-table-td ">59</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Dunsmuir et al., "<a href='http://scholar.google.com/scholar?q="Stability+of+introduced+genes+and+stability+of+expression%2C"'>Stability of introduced genes and stability of expression,</a>" Plant Molecular Biology Manual, Kluwer Academic Publishers, Dordrecht, The Netherlands:C1:1-17 (1988).</td></tr><tr><td class="patent-data-table-td ">60</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Eisen, H. N. et al., (1996),"<a href='http://scholar.google.com/scholar?q="+Antigen-Specific+T-Cell+Receptors+And+Their+Reactions+With+Complexes+Formed+By+Peptides+With+Major+Histocompatibility+Complex+Proteins"'> Antigen-Specific T-Cell Receptors And Their Reactions With Complexes Formed By Peptides With Major Histocompatibility Complex Proteins</a>" Adv. Protein Chem. 49:1-56.</td></tr><tr><td class="patent-data-table-td ">61</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Engel I, et al., (1988), Site-directed mutations in the VDJ junctional region of a T cell receptor &amp;b chain cause changes in antigenic peptide recognition. Cell54: 473-484.</td></tr><tr><td class="patent-data-table-td ">62</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Faber, K.N. et al. (1995), "<a href='http://scholar.google.com/scholar?q="Review%3A+Methylotrophic+Yeasts+as+Factories+for+the+Production+of+Foreign+Proteins%2C"'>Review: Methylotrophic Yeasts as Factories for the Production of Foreign Proteins,</a>" Yeast 11:1331-1344.</td></tr><tr><td class="patent-data-table-td ">63</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Faber, K.N. et al., (1995), "<a href='http://scholar.google.com/scholar?q="Review%3A+Methylotrophic+Yeasts+as+Factories+for+the+Production+of+Foreign+Proteins"'>Review: Methylotrophic Yeasts as Factories for the Production of Foreign Proteins</a>" Yeast 11:1331-1344.</td></tr><tr><td class="patent-data-table-td ">64</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Fitchen et al., "<a href='http://scholar.google.com/scholar?q="Plant+virus+expressing+hybrid+coat+protein+with+added+murine+epitope+elicits+autoantibody+response%2C"'>Plant virus expressing hybrid coat protein with added murine epitope elicits autoantibody response,</a>" Vaccine 13:1051-1057 (1995).</td></tr><tr><td class="patent-data-table-td ">65</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Foote, J. and Eisen, H.N., (Sep. 2000) "<a href='http://scholar.google.com/scholar?q="Breaking+the+affinity+ceiling+for+antibodies+and+T+cell+receptors"'>Breaking the affinity ceiling for antibodies and T cell receptors</a>" PNAS 97(20):10679-10681.</td></tr><tr><td class="patent-data-table-td ">66</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Fremont et al., (1996), "<a href='http://scholar.google.com/scholar?q="Biophysical+Studies+of+T-cell+Receptors+and+Their+Ligands%2C"'>Biophysical Studies of T-cell Receptors and Their Ligands,</a>" Curr. Opin. Immunol. 8:93-100.</td></tr><tr><td class="patent-data-table-td ">67</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">French et al., "<a href='http://scholar.google.com/scholar?q="Bacterial+ene+inserted+in+an+engineered+RNA+virus%3A+Efficient+expression+in+monocotyledonous+plant+cells%2C"'>Bacterial ene inserted in an engineered RNA virus: Efficient expression in monocotyledonous plant cells,</a>" Science 231:1294-1297 (1986).</td></tr><tr><td class="patent-data-table-td ">68</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Furukawa, k. et al., "<a href='http://scholar.google.com/scholar?q="Junctional+Amino+Acids+Determine+the+Maturation+Pathway+of+an+Antibody"'>Junctional Amino Acids Determine the Maturation Pathway of an Antibody</a>" (Sep. 1999), Immunity 11:329-338.</td></tr><tr><td class="patent-data-table-td ">69</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Ganju, R.K. et al., (Dec. 1992), "<a href='http://scholar.google.com/scholar?q="Similarity+between+fluorescein-specific+T-cell+receptor+and+antibody+in+chemical+details+of+antigen+recognition%2C"'>Similarity between fluorescein-specific T-cell receptor and antibody in chemical details of antigen recognition,</a>" Proc. Natl. Acad. Sci. USA 89:11552-11556.</td></tr><tr><td class="patent-data-table-td ">70</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Garcia, K.C. et al. "<a href='http://scholar.google.com/scholar?q="An+%CE%B1B+T+Cell+Receptor+Structure+at+2.5+%E2%84%ABand+Its+Orientation+in+the+TCR-MHC+Complex"'>An αB T Cell Receptor Structure at 2.5 Åand Its Orientation in the TCR-MHC Complex</a>" (Oct. 1996), Science 274:209-219.</td></tr><tr><td class="patent-data-table-td ">71</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Garcia, K.C. et al., (Dec. 1997), "<a href='http://scholar.google.com/scholar?q="%CE%B1%CE%B2+T+cell+receptor+interactions+with+syngeneic+and+allogeneic+ligands%3A+Affinity+measurements+and+crystallization%2C"'>αβ T cell receptor interactions with syngeneic and allogeneic ligands: Affinity measurements and crystallization,</a>" Proc. Natl. Acad. Sci. USA 94:13838-13843.</td></tr><tr><td class="patent-data-table-td ">72</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Garcia, K.C. et al., "<a href='http://scholar.google.com/scholar?q="CD8+enhances+formation+of+stable+T-cell+receptor%2FMHC+class+I+molecule+complexes"'>CD8 enhances formation of stable T-cell receptor/MHC class I molecule complexes</a>" (Dec. 1996), Nature 384:577-584.</td></tr><tr><td class="patent-data-table-td ">73</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Garcia, K.C. et al., "<a href='http://scholar.google.com/scholar?q="Structural+Basis+of+Plasticity+in+T+Cell+Receptor+Recognition+of+a+Self+Peptide-MHC+Antigen"'>Structural Basis of Plasticity in T Cell Receptor Recognition of a Self Peptide-MHC Antigen</a>" (Feb. 1998) Science 279:1166-1172.</td></tr><tr><td class="patent-data-table-td ">74</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Gascoigne, N.R. et al., (May 1987), "<a href='http://scholar.google.com/scholar?q="Secretion+of+a+chimeric+T-cell+receptor-immunoglobulin+protein%2C"'>Secretion of a chimeric T-cell receptor-immunoglobulin protein,</a>" Proc. Natl. Acad. Sci. USA 84:2936-2940.</td></tr><tr><td class="patent-data-table-td ">75</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Geitz, R.D. et al., (1995), "<a href='http://scholar.google.com/scholar?q="Studies+on+the+Transformation+of+Intact+Yeast+Cells+by+the+LiAc%2FSS-DNA%2FPEG+Procedure"'>Studies on the Transformation of Intact Yeast Cells by the LiAc/SS-DNA/PEG Procedure</a>" Yeast 11:355-360.</td></tr><tr><td class="patent-data-table-td ">76</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Gellissen, G. et al., (1992) "<a href='http://scholar.google.com/scholar?q="Progress+in+developing+methylotrophic+yeasts+as+expression+systems"'>Progress in developing methylotrophic yeasts as expression systems</a>" Tibtech 10:413-417.</td></tr><tr><td class="patent-data-table-td ">77</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Gibbs, A.J., "<a href='http://scholar.google.com/scholar?q="Tobamovirus+group%2C+C.M.I.%2FA.A.B.+Descriptions+of+plant+viruses%2C"'>Tobamovirus group, C.M.I./A.A.B. Descriptions of plant viruses,</a>" No. 184. Wm. Culross and Son Ltd., Coupar Angus, Perthshire, Scotland (1977).</td></tr><tr><td class="patent-data-table-td ">78</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Goelet et al., "<a href='http://scholar.google.com/scholar?q="Nucleotide+sequence+of+tobacco+mosaic+virus+RNA%2C"'>Nucleotide sequence of tobacco mosaic virus RNA,</a>" Proc. Natl. Acad. Sci. USA 79:5818-5822 (1982).</td></tr><tr><td class="patent-data-table-td ">79</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Gooding, Jr., G.V., and Hebert, T.T., "<a href='http://scholar.google.com/scholar?q="A+simple+technique+for+purification+of+tobacco+mosaic+virus+in+large+quantities%2C"'>A simple technique for purification of tobacco mosaic virus in large quantities,</a>" Phytopathology 57:1285 (1967).</td></tr><tr><td class="patent-data-table-td ">80</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Hamamoto et al., "<a href='http://scholar.google.com/scholar?q="A+new+tobacco+mosaic+virus+vector+and+its+use+for+the+systemic+production+of+angiotensin-I-converting+enzyme+inhibitor+in+transgenic+tobacco+and+tomato%2C"'>A new tobacco mosaic virus vector and its use for the systemic production of angiotensin-I-converting enzyme inhibitor in transgenic tobacco and tomato,</a>" Bio/Technology 11:930-932 (1993).</td></tr><tr><td class="patent-data-table-td ">81</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Hare, B.J. et al., (Jun. 1999), "<a href='http://scholar.google.com/scholar?q="Structure%2C+specificity+and+CDR+mobility+of+a+class+II+restricted+single-chain+T-cell+receptor"'>Structure, specificity and CDR mobility of a class II restricted single-chain T-cell receptor</a>" Nat. Struct. Biol. 6:574-581.</td></tr><tr><td class="patent-data-table-td ">82</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Hawkins, R.E. et al. (1992), "<a href='http://scholar.google.com/scholar?q="Selections+of+Phage+Antibodies+by+Binding+Affinity+Mimicking+Affinity+Maturation%2C"'>Selections of Phage Antibodies by Binding Affinity Mimicking Affinity Maturation,</a>" J. Mol. Biol. 226:889-896.</td></tr><tr><td class="patent-data-table-td ">83</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Hawkins, R.E. et al. (1993), "<a href='http://scholar.google.com/scholar?q="The+Contribution+of+Contact+and+Non-contact+Residues+of+Antibody+in+the+Affinity+of+Binding+to+Antigen%2C"'>The Contribution of Contact and Non-contact Residues of Antibody in the Affinity of Binding to Antigen,</a>" J. Mol. Biol. 234:958-964.</td></tr><tr><td class="patent-data-table-td ">84</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Haynes et al., "<a href='http://scholar.google.com/scholar?q="Development+of+a+genetically-engineered%2C+candidate+polio+vaccine+employing+the+self-assembling+properties+of+the+tobacco+mosaic+virus+coat+protein%2C"'>Development of a genetically-engineered, candidate polio vaccine employing the self-assembling properties of the tobacco mosaic virus coat protein,</a>" Bio/Technology 4:637-641 (1986).</td></tr><tr><td class="patent-data-table-td ">85</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Hilyard, K.L. et al., (Sep. 1994), "<a href='http://scholar.google.com/scholar?q="Binding+of+soluble+natural+ligands+to+a+soluble+human+T-cell+receptor+fragment+produced+in+Escherichia+coli%2C"'>Binding of soluble natural ligands to a soluble human T-cell receptor fragment produced in Escherichia coli,</a>" Pro. Natl. Acad. Sci. USA 91:9057-9061.</td></tr><tr><td class="patent-data-table-td ">86</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Holler, et al., (2000) In vitroevolution of a T cell receptor with high affinity for peptide/MHC, PNAS, 97:5387-5392.</td></tr><tr><td class="patent-data-table-td ">87</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Holler, et al., (2002) TCRs with high affinity for foreign pMHC show self-reactivity, Nature Immunology, Published online Dec. 9, 2002; doi:10.1038/ni863.</td></tr><tr><td class="patent-data-table-td ">88</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Holler, et al., (2003) Quantitative Analysis of the Contribution of TCR/pepMHC Affinity and CD8 to T Cell Activation, Immunity, 18:255-264.</td></tr><tr><td class="patent-data-table-td ">89</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Holler, P.D. et al., (May 2000), "<a href='http://scholar.google.com/scholar?q="In+vitro+evolution+of+a+T+cell+receptor+with+high+affinity+for+peptide%2FMHC%2C"'>In vitro evolution of a T cell receptor with high affinity for peptide/MHC,</a>"PNAS 97(10):5387-5392.</td></tr><tr><td class="patent-data-table-td ">90</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Holler, Philip D., et al., (2001), CD8-T Cell Transfectants that Express a High Affinity T Cell Receptor Exhibit Enhanced Peptide-dependent Activation. J. Exp. Med.194: 1043-1052.</td></tr><tr><td class="patent-data-table-td ">91</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Hoogenboom, Hennie R., (1997) Designing and optimizing library selection strategies for generating high-affinity antibodies, Tibtech, 15:62-70.</td></tr><tr><td class="patent-data-table-td ">92</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Horsch et al., "<a href='http://scholar.google.com/scholar?q="Leaf+disc+transformation%2C"'>Leaf disc transformation,</a>" Plant Molecular Biology Manual, Kluwer Academic Publishers, Dordrecht, The Netherlands:A5:1-9 (1988).</td></tr><tr><td class="patent-data-table-td ">93</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Huse, W. D. et al., (Dec. 1989), "<a href='http://scholar.google.com/scholar?q="Generation+of+a+Large+Combinatorial+Library+of+the+Immunoglobulin+Repertoire+in+Phage+Lambda%2C"'>Generation of a Large Combinatorial Library of the Immunoglobulin Repertoire in Phage Lambda,</a>" Science 246:1275-1281.</td></tr><tr><td class="patent-data-table-td ">94</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Jagadish et al., "<a href='http://scholar.google.com/scholar?q="High+Level+Production+of+Hybrid+otybirus-like+Particles+Carrying+Repetitive+Copies+of+Foreign+Antigens+in+Escherichia+coli%2C"'>High Level Production of Hybrid otybirus-like Particles Carrying Repetitive Copies of Foreign Antigens in Escherichia coli,</a>" Bio/Technology 11:1166-1170 (1993).</td></tr><tr><td class="patent-data-table-td ">95</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Jorgensen, J.L. et al., (Jan. 1992), "<a href='http://scholar.google.com/scholar?q="Mapping+T-cell+receptor-peptide+contacts+by+variant+peptide+immunization+of+single-chain+transgenics%2C"'>Mapping T-cell receptor-peptide contacts by variant peptide immunization of single-chain transgenics,</a>" Nature 355:224-230.</td></tr><tr><td class="patent-data-table-td ">96</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Joshi and Joshi, FEBS Letters 281:1-8 (1991).</td></tr><tr><td class="patent-data-table-td ">97</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Joshi et al., "<a href='http://scholar.google.com/scholar?q="BSMV+genome+mediated+expression+of+a+foreign+gene+in+dicot+and+monoclonal+plant+cells%2C"'>BSMV genome mediated expression of a foreign gene in dicot and monoclonal plant cells,</a>" EMBO J. 9:2663-2669 (1990).</td></tr><tr><td class="patent-data-table-td ">98</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Jung, S. and Plückthun (1997), "<a href='http://scholar.google.com/scholar?q="Improving+in+vivo+folding+and+stability+of+a+single-chain+Fv+antibody+fragment+by+loop+grafting%2C"'>Improving in vivo folding and stability of a single-chain Fv antibody fragment by loop grafting,</a>" Protein Eng. 10(8):959-966.</td></tr><tr><td class="patent-data-table-td ">99</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Jupin et al., Virology 178:273-280 (1990).</td></tr><tr><td class="patent-data-table-td ">100</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Kappler, J. et al., (Aug. 1994), "<a href='http://scholar.google.com/scholar?q="Binding+of+a+soluble+%CE%B1%CE%B2+T-cell+receptor+to+superantigen%2Fmajor+histocompatibility+complex+ligands%2C"'>Binding of a soluble αβ T-cell receptor to superantigen/major histocompatibility complex ligands,</a>" Proc. Natl. Acad. Sci. USA 91:8462-8466.</td></tr><tr><td class="patent-data-table-td ">101</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Kasibhatla S. et al., (1993) Simultaneous involvement of all six predicted antigen binding loops of the T cell receptor in recognition of the MHC/antigenic peptide complex. J. Immunol.151:3140-51.</td></tr><tr><td class="patent-data-table-td ">102</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Kearny et al., Virology 192:000-000 (in press) (1993).</td></tr><tr><td class="patent-data-table-td ">103</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Kieke, M.C. et al. (1997), "<a href='http://scholar.google.com/scholar?q="Isolation+of+anti-T+cell+receptor+scFv+mutants+by+yeast+surface+display%2C"'>Isolation of anti-T cell receptor scFv mutants by yeast surface display,</a>"Protein Eng. 10(11):1303-1310.</td></tr><tr><td class="patent-data-table-td ">104</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Kieke, M.C. et al., (May 1999), "<a href='http://scholar.google.com/scholar?q="Selection+of+functional+T+cell+receptor+mutants+from+a+yeast+surface-display+library%2C"'>Selection of functional T cell receptor mutants from a yeast surface-display library,</a>" Proc. Natl. Acad. Sci. USA 96(10):5651-5656.</td></tr><tr><td class="patent-data-table-td ">105</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Kieke, M.C., et al., (2001), High affinity T cell receptors from yeast display libraries block T cell activation by superantigens. J Mol Biol 307:1305-1315.</td></tr><tr><td class="patent-data-table-td ">106</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Kipriyanov, S. M. et al., (Apr. 1997), "<a href='http://scholar.google.com/scholar?q="Two+amino+acid+mutations+in+an+anti-human+CD3+single+chain+Fv+antibody+fragment+that+affect+the+yield+on+bacterial+secretion+but+not+the+affinity%2C"'>Two amino acid mutations in an anti-human CD3 single chain Fv antibody fragment that affect the yield on bacterial secretion but not the affinity,</a>" Protein Eng. 10(4):445-453.</td></tr><tr><td class="patent-data-table-td ">107</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Klis, F.M. (1994). "<a href='http://scholar.google.com/scholar?q="Review%3A+Cell+Wall+Assembly+in+Yeast%2C"'>Review: Cell Wall Assembly in Yeast,</a>" Yeast 10:851-869.</td></tr><tr><td class="patent-data-table-td ">108</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Knappik, A. and Plückthun, A. (1995), "<a href='http://scholar.google.com/scholar?q="Engineered+turns+of+a+recombinant+antibody+improve+its+in+vivo+folding%2C"'>Engineered turns of a recombinant antibody improve its in vivo folding,</a>" Protein Eng. 8(1):81-89.</td></tr><tr><td class="patent-data-table-td ">109</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Kowalski, J. M. et al., (Feb. 1998), "<a href='http://scholar.google.com/scholar?q="Secretion+efficiency+in+Saccharomyces+cerevisiae+of+bovine+pancreatic+trypsin+inhibitor+mutants+lacking+disulfide+bonds+is+correlated+with+thermodynamic+stability%2C"'>Secretion efficiency in Saccharomyces cerevisiae of bovine pancreatic trypsin inhibitor mutants lacking disulfide bonds is correlated with thermodynamic stability,</a>" Biochemistry 37(5):1264-1273.</td></tr><tr><td class="patent-data-table-td ">110</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Kowalski, J.M. et al., (Jul. 1998), "<a href='http://scholar.google.com/scholar?q="Protein+folding+stability+can+determine+the+efficiency+of+escape+from+endoplasmic+reticulum+quality+control%2C"'>Protein folding stability can determine the efficiency of escape from endoplasmic reticulum quality control,</a>" J. Biol. Chem. 273(31):19453-19458.</td></tr><tr><td class="patent-data-table-td ">111</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Krebbers et al., "<a href='http://scholar.google.com/scholar?q="Prospects+and+progress+in+the+production+of+foreign+proteins+and+peptides+in+plants%2C"'>Prospects and progress in the production of foreign proteins and peptides in plants,</a>" Plant Protein Engineering. (P.R. Shewry and S. Gutteridge, eds.), Cambridge University Press, Cambridge, pp. 315-325 (1992).</td></tr><tr><td class="patent-data-table-td ">112</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Kumagai et al., "<a href='http://scholar.google.com/scholar?q="Rapid%2C+high+level+expression+of+biologically+active+alpha-trichosanthin+in+transfected+plants+by+a+novel+RNA+viral+vector%2C"'>Rapid, high level expression of biologically active alpha-trichosanthin in transfected plants by a novel RNA viral vector,</a>" Proc. Natl. Acad. Sci USA 90:427-430 (1993).</td></tr><tr><td class="patent-data-table-td ">113</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Kumagai et al., "<a href='http://scholar.google.com/scholar?q="Rapid%2C+high+level+expression+of+biologically+active+%CE%B1-trichosanthin+in+transfected+plants+by+a+novel+RNA+viral+vector%2C"'>Rapid, high level expression of biologically active α-trichosanthin in transfected plants by a novel RNA viral vector,</a>" Proc. Natl. Acad. Sci USA 90:427-430 (1993).</td></tr><tr><td class="patent-data-table-td ">114</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Kumar, V. et al. (1997) "<a href='http://scholar.google.com/scholar?q="Recombinant+T+Cell+Receptor+Molecules+Can+Prevent+and+Reverse+Experimental+Autoimmune+Encephalomyelitis"'>Recombinant T Cell Receptor Molecules Can Prevent and Reverse Experimental Autoimmune Encephalomyelitis</a>" The Journal of Immunology 159:5150-5156.</td></tr><tr><td class="patent-data-table-td ">115</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Lake, D.F. et al. (Jan. 1999) "<a href='http://scholar.google.com/scholar?q="Construction+and+binding+analysis+of+recombinant+single-chain+TCR+derived+from+tumor-infiltrating+lymphocytes+and+a+cytotoxic+T+lymphocyte+clone+directed+against+MAGE-1"'>Construction and binding analysis of recombinant single-chain TCR derived from tumor-infiltrating lymphocytes and a cytotoxic T lymphocyte clone directed against MAGE-1</a>" International Immunology 11:745-751.</td></tr><tr><td class="patent-data-table-td ">116</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Larkins et al., J. Cell. Biochem. Suppl. 0(9 Part C):264 (1985).</td></tr><tr><td class="patent-data-table-td ">117</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Letourneur, F. and Malissen, B., (1989), "<a href='http://scholar.google.com/scholar?q="Derivation+of+a+T+cell+hybridoma+variant+deprived+of+functional+T+cell+receptor+a+and+b+chain+transcripts+reveals+a+nonfunctional+a-mRNA+of+BW5147+origin"'>Derivation of a T cell hybridoma variant deprived of functional T cell receptor a and b chain transcripts reveals a nonfunctional a-mRNA of BW5147 origin</a>" Eur. J. Immunol. 19(12):2269-2274.</td></tr><tr><td class="patent-data-table-td ">118</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Lipke, P.N. and Kurjan, J., (Mar. 1992), "<a href='http://scholar.google.com/scholar?q="Sexual+Agglutination+in+Budding+Yeasts%3A+Structure%2C+Function%2C+and+Regulation+of+Adhesion+Glycoproteins%2C"'>Sexual Agglutination in Budding Yeasts: Structure, Function, and Regulation of Adhesion Glycoproteins,</a>" Microbiological Reviews pp. 180-194.</td></tr><tr><td class="patent-data-table-td ">119</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Lyons et al., (Jul. 1996), "<a href='http://scholar.google.com/scholar?q="A+TCR+Binds+to+Antagonist+Ligands+with+Lower+Affinities+and+Faster+Dissociation+Rates+Than+to+Agonists%2C"'>A TCR Binds to Antagonist Ligands with Lower Affinities and Faster Dissociation Rates Than to Agonists,</a>" Immunity 5:53-61.</td></tr><tr><td class="patent-data-table-td ">120</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Malchiodi EL, (1995), Superantigen binding to a T cell receptor βchain of known three-dimensional structure. J. Exp. Med.182:1833-1845.</td></tr><tr><td class="patent-data-table-td ">121</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Manning et al., (Feb. 1999), "<a href='http://scholar.google.com/scholar?q="Effects+of+complementarity+determining+region+mutations+on+the+affinity+of+alpha%2Fbeta+T+cell+receptor%3A+measuring+the+energy+associated+with+CD4%2FCD8+repertoire+skewing%2C"'>Effects of complementarity determining region mutations on the affinity of alpha/beta T cell receptor: measuring the energy associated with CD4/CD8 repertoire skewing,</a>" J. Exp. Med. 189(3):461-470.</td></tr><tr><td class="patent-data-table-td ">122</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Manning, T. C. et al., (Apr. 1998), "<a href='http://scholar.google.com/scholar?q="Alanine+Scanning+Mutagenesis+of+an+%CE%B1%CE%B2+T+cell+Receptor%3A+Mapping+the+Energy+of+Antigen+Recognition%2C"'>Alanine Scanning Mutagenesis of an αβ T cell Receptor: Mapping the Energy of Antigen Recognition,</a>" Immunity 8:413-425.</td></tr><tr><td class="patent-data-table-td ">123</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Margulies, D.H., (Jun. 1996), "<a href='http://scholar.google.com/scholar?q="An+Affinity+for+Learning%2C"'>An Affinity for Learning,</a>" Nature 381:558-559.</td></tr><tr><td class="patent-data-table-td ">124</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Marks, J.D. et al., (1991), "<a href='http://scholar.google.com/scholar?q="By-passing+Immunization.+Human+Antibodies+from+V-gene+Libraries+Displayed+on+Phage%2C"'>By-passing Immunization. Human Antibodies from V-gene Libraries Displayed on Phage,</a>" J. Mol. Biol. 222:581-597.</td></tr><tr><td class="patent-data-table-td ">125</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Marks, J.D. et al., (Aug. 1992), "<a href='http://scholar.google.com/scholar?q="Molecular+Evolution+of+Proteins+on+Filamentous+Phage%2C"'>Molecular Evolution of Proteins on Filamentous Phage,</a>" J. Biol. Chem. 267(23):16007-16010.</td></tr><tr><td class="patent-data-table-td ">126</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Martelli, Plant Disease 76:436 (1992).</td></tr><tr><td class="patent-data-table-td ">127</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Martineau, P. et al., (Jul. 1998), "<a href='http://scholar.google.com/scholar?q="Expression+of+an+antibody+fragment+at+high+levels+in+the+bacterial+cytoplasm%2C"'>Expression of an antibody fragment at high levels in the bacterial cytoplasm,</a>" J. Mol. Biol. 280(1):117-127.</td></tr><tr><td class="patent-data-table-td ">128</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Marx, J. (Jan. 1995), "<a href='http://scholar.google.com/scholar?q="The+T+Cell+Receptor+Begins+to+Reveal+Its+Many+Facets%2C"'>The T Cell Receptor Begins to Reveal Its Many Facets,</a>" Science 267:459-460.</td></tr><tr><td class="patent-data-table-td ">129</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Mason et al., "<a href='http://scholar.google.com/scholar?q="Expression+of+hepatitis+B+surface+antigen+in+transgenic+plants%2C"'>Expression of hepatitis B surface antigen in transgenic plants,</a>" Proc. Natl. Acad. Sci. USA 89:11745-11749 (1992).</td></tr><tr><td class="patent-data-table-td ">130</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Matsui et al., (Dec. 1991), "<a href='http://scholar.google.com/scholar?q="Low+Affinity+Interaction+of+Peptide-MHC+Complexes+with+T+Cell+Receptors%2C"'>Low Affinity Interaction of Peptide-MHC Complexes with T Cell Receptors,</a>" Science 254:1788-1791.</td></tr><tr><td class="patent-data-table-td ">131</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Matsui et al., (Dec. 1994), "<a href='http://scholar.google.com/scholar?q="Kinetics+of+T-cell+Receptor+Binding+to+Peptide%2FI-E%26lt%3Bk+%26gt%3BComplexes+%3A+Correlation+of+the+Dissociation+Rate+with+T-cell+Responsiveness%2C"'>Kinetics of T-cell Receptor Binding to Peptide/I-E&lt;k &gt;Complexes : Correlation of the Dissociation Rate with T-cell Responsiveness,</a>" Proc. Natl. Acad. Sci. USA 91:12862-12866.</td></tr><tr><td class="patent-data-table-td ">132</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Matsui et al., (Dec. 1994), "<a href='http://scholar.google.com/scholar?q="Kinetics+of+T-cell+Receptor+Binding+to+Peptide%2FI-Ek+Complexes+%3A+Correlation+of+the+Dissociation+Rate+with+T-cell+Responsiveness%2C"'>Kinetics of T-cell Receptor Binding to Peptide/I-Ek Complexes : Correlation of the Dissociation Rate with T-cell Responsiveness,</a>" Proc. Natl. Acad. Sci. USA 91:12862-12866.</td></tr><tr><td class="patent-data-table-td ">133</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">McCafferty, J. et al., (Dec. 1990), "<a href='http://scholar.google.com/scholar?q="Phage+antibodies%3A+filamentous+phage+displaying+antibody+variable+domains%2C"'>Phage antibodies: filamentous phage displaying antibody variable domains,</a>" Nature 348:552-554.</td></tr><tr><td class="patent-data-table-td ">134</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Nieba, L. et al. (1997). "<a href='http://scholar.google.com/scholar?q="Disrupting+the+hydrophobic+patches+at+the+antibody+variable%2Fconstant+domain+interface%3A+improved+in+vivo+folding+and+physical+characterization+of+an+engineered+scFv+fragment%2C"'>Disrupting the hydrophobic patches at the antibody variable/constant domain interface: improved in vivo folding and physical characterization of an engineered scFv fragment,</a>" Protein Eng. 10(4):435-444.</td></tr><tr><td class="patent-data-table-td ">135</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Nieba, L. et al., (Apr. 1997), "<a href='http://scholar.google.com/scholar?q="Disrupting+the+hydrophobic+patches+at+the+antibody+variable%2Fconstant+domain+interface%3A+improved+in+vivo+folding+and+physical+characterization+of+an+engineered+scFv+fragment%2C"'>Disrupting the hydrophobic patches at the antibody variable/constant domain interface: improved in vivo folding and physical characterization of an engineered scFv fragment,</a>" Protein Eng. 10(4):435-444.</td></tr><tr><td class="patent-data-table-td ">136</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Novotny, J. et al., (Oct. 1991), "<a href='http://scholar.google.com/scholar?q="A+soluble%2C+single-chain+T-cell+receptor+fragment+endowed+with+antigen-combining+properties%2C"'>A soluble, single-chain T-cell receptor fragment endowed with antigen-combining properties,</a>" Proc. Natl. Acad. Sci. USA 88:8646-8650.</td></tr><tr><td class="patent-data-table-td ">137</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Ogawa et al., Virology 185:580-584 (1991).</td></tr><tr><td class="patent-data-table-td ">138</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">O'Herrin et al., (Oct. 1997), "<a href='http://scholar.google.com/scholar?q="Analysis+of+the+Expression+of+Peptide-Major+Histocompatibility+Complexes+Using+High+Affinity+Soluble+Divalent+T+Cell+Receptors%2C"'>Analysis of the Expression of Peptide-Major Histocompatibility Complexes Using High Affinity Soluble Divalent T Cell Receptors,</a>" J. Exp. Med 186:1333-1345.</td></tr><tr><td class="patent-data-table-td ">139</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">O'Herrin, S.M. et al., (Oct. 1997), "<a href='http://scholar.google.com/scholar?q="Analysis+of+the+Expression+of+Peptide-Major+Histocompatibility+Complexes+Using+High+Affinity+Soluble+Divalent+T+Cell+Receptors%2C"'>Analysis of the Expression of Peptide-Major Histocompatibility Complexes Using High Affinity Soluble Divalent T Cell Receptors,</a>" J. Exp. Med 186:1333-1345.</td></tr><tr><td class="patent-data-table-td ">140</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Olsnes, S. and Pihl, A., "<a href='http://scholar.google.com/scholar?q="Chimeric+Toxins"'>Chimeric Toxins</a>" (1982), Pharmac. Ther. 25:355-381.</td></tr><tr><td class="patent-data-table-td ">141</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Ow et al., Science 234:856 (1986).</td></tr><tr><td class="patent-data-table-td ">142</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Pelham, H.R.B., "<a href='http://scholar.google.com/scholar?q="Leaky+UAG+termination+codon+in+tobacco+mosaic+virus+RNA%2C"'>Leaky UAG termination codon in tobacco mosaic virus RNA,</a>" Nature 272:469-471 (1978).</td></tr><tr><td class="patent-data-table-td ">143</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Porta et al., "<a href='http://scholar.google.com/scholar?q="Development+of+Cowpea+Mosaic+Virus+as+a+High-Yielding+System+for+the+Presentation+of+Foreign+Peptides%2C"'>Development of Cowpea Mosaic Virus as a High-Yielding System for the Presentation of Foreign Peptides,</a>" Virology 202:949-955 (1994).</td></tr><tr><td class="patent-data-table-td ">144</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Potrykus, Ann. Rev. Plant Physiol. Plant Mol. Biol. 42:205-225 (1991).</td></tr><tr><td class="patent-data-table-td ">145</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Rabinowitz et al., (Feb. 1996), Kinetic discrimination in T-cell activation Proc. Natl. Acad. Sci. USA 93:1401-1405.</td></tr><tr><td class="patent-data-table-td ">146</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Raffo and Dawson, "<a href='http://scholar.google.com/scholar?q="Construction+of+Tobacco+Mosaic+Virus+Subgenomic+Replicons+that+are+Replicated+and+Spread+Systemically+in+Tobacco+Plants%2C"'>Construction of Tobacco Mosaic Virus Subgenomic Replicons that are Replicated and Spread Systemically in Tobacco Plants,</a>" Virology 184:277-289 (1991).</td></tr><tr><td class="patent-data-table-td ">147</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Reich et al., (Jun. 1997), "<a href='http://scholar.google.com/scholar?q="Ligand-specific+Oligomerization+of+T-cell+Receptor+Molecules%2C"'>Ligand-specific Oligomerization of T-cell Receptor Molecules,</a>" Nature 387:617-620.</td></tr><tr><td class="patent-data-table-td ">148</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Ridder, R. et al. (1995), "<a href='http://scholar.google.com/scholar?q="Generation+of+Rabbit+Monoclonal+Antibody+Fragments+from+a+Combinatorial+Phage+Display+Library+and+Their+Production+in+the+Yeast+Pichia+pastoris%2C"'>Generation of Rabbit Monoclonal Antibody Fragments from a Combinatorial Phage Display Library and Their Production in the Yeast Pichia pastoris,</a>" Bio/Technol. 13:255-259.</td></tr><tr><td class="patent-data-table-td ">149</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Romanos et al., (1992), "<a href='http://scholar.google.com/scholar?q="Foreign+Gene+Expression+in+Yeast%3A+a+Review%2C"'>Foreign Gene Expression in Yeast: a Review,</a>" Yeast 8:423-488.</td></tr><tr><td class="patent-data-table-td ">150</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Romanos, M. (1995), "<a href='http://scholar.google.com/scholar?q="Advances+in+the+use+of+Pichia+pastoris+for+high-level+gene+expression%2C"'>Advances in the use of Pichia pastoris for high-level gene expression,</a>" Curr. Opinion in Biotechnol. 6:527-533.</td></tr><tr><td class="patent-data-table-td ">151</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Rowlands et al., eds., American Press, London, pp. 237-257 (1987).</td></tr><tr><td class="patent-data-table-td ">152</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Saito et al., Virology 176:329-336 (1990).</td></tr><tr><td class="patent-data-table-td ">153</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Sant' Angelo, D.B. et al., (Apr. 1996), "<a href='http://scholar.google.com/scholar?q="The+Specificity+and+Orientation+of+a+TCR+to+its+Peptide-MHC+Class+II+Ligands%2C"'>The Specificity and Orientation of a TCR to its Peptide-MHC Class II Ligands,</a>" Immunity 4:367-376.</td></tr><tr><td class="patent-data-table-td ">154</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Schlueter et al., (1996), "<a href='http://scholar.google.com/scholar?q="Specificity+and+Binding+Properties+of+a+Single-chain+T+Cell+Receptor%2C"'>Specificity and Binding Properties of a Single-chain T Cell Receptor,</a>" J. Mol. Biol. 256:859-869.</td></tr><tr><td class="patent-data-table-td ">155</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Schlueter, C.J. et al., (1996), "<a href='http://scholar.google.com/scholar?q="A+Residue+in+the+Center+of+Peptide+QL9+Affects+Binding+to+Both+Ld+and+the+T+Cell+Receptor1%2C"'>A Residue in the Center of Peptide QL9 Affects Binding to Both Ld and the T Cell Receptor1,</a>" J. Immunol. 157:4478-4485.</td></tr><tr><td class="patent-data-table-td ">156</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Schneck, J. et al., (Jan. 1989), "<a href='http://scholar.google.com/scholar?q="Inhibition+of+an+Allospecific+T+Cell+Hybridoma+by+Soluble+Class+I+Proteins+and+Peptides%3A+Estimation+of+the+Affinity+of+a+T+Cell+Receptor+for+MHC%2C"'>Inhibition of an Allospecific T Cell Hybridoma by Soluble Class I Proteins and Peptides: Estimation of the Affinity of a T Cell Receptor for MHC,</a>" Cell 56:47-55.</td></tr><tr><td class="patent-data-table-td ">157</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Schodin et al., (1996), "<a href='http://scholar.google.com/scholar?q="Binding+Properties+and+Solubility+of+Single-Chain+T+Cell+Receptors+Expressed+in+E.+coli%2C"'>Binding Properties and Solubility of Single-Chain T Cell Receptors Expressed in E. coli,</a>" Molec. Immunol. 33(9):819-829.</td></tr><tr><td class="patent-data-table-td ">158</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Schodin, B.A. and Kranz, D.M., (Dec. 1993), "<a href='http://scholar.google.com/scholar?q="Binding+Affinity+and+Inhibitory+Properties+of+a+Single-Chain+Anti-T+Cell+Receptor+Antibody%2C"'>Binding Affinity and Inhibitory Properties of a Single-Chain Anti-T Cell Receptor Antibody,</a>" J. Biol. Chem. 268(34):25722-25727.</td></tr><tr><td class="patent-data-table-td ">159</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Schodin, B.A. et al., (1996) "<a href='http://scholar.google.com/scholar?q="Binding+Properties+and+solubility+of+single-chain+T+cell+receptors+expressed+in+E.+coli"'>Binding Properties and solubility of single-chain T cell receptors expressed in E. coli</a>" Immun.33(9):819-829.</td></tr><tr><td class="patent-data-table-td ">160</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Schreuder et al., (Apr. 1996), "<a href='http://scholar.google.com/scholar?q="Immobilizing+Proteins+on+the+Surface+of+Yeast+Cells%2C"'>Immobilizing Proteins on the Surface of Yeast Cells,</a>" TIBTECH 14:115-120.</td></tr><tr><td class="patent-data-table-td ">161</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Seibel, J.L. et al., (Jun. 1997), "<a href='http://scholar.google.com/scholar?q="Influence+of+the+NH2-terminal+Amino+Acid+of+the+T+Cell+Receptor+%CE%B1+Chain+on+Major+Histocompatibility+Complex+%28MHC%29+Class+II+%2B+Peptide+Recognition%2C"'>Influence of the NH2-terminal Amino Acid of the T Cell Receptor α Chain on Major Histocompatibility Complex (MHC) Class II + Peptide Recognition,</a>" J. Exp. Med. 185(11):1919-1927.</td></tr><tr><td class="patent-data-table-td ">162</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Seth, A. et al., (May 1994), "<a href='http://scholar.google.com/scholar?q="Binary+and+ternary+complexes+between+T-cell+receptor%2C+class+II+MHC+and+superantigen+in+vitro%2C"'>Binary and ternary complexes between T-cell receptor, class II MHC and superantigen in vitro,</a>" Nature 369:324-327.</td></tr><tr><td class="patent-data-table-td ">163</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Shaw, "<a href='http://scholar.google.com/scholar?q="Chlorampheniol+acetyltransferase+from+chloramphenicol-resistant+bacteria%2C"'>Chlorampheniol acetyltransferase from chloramphenicol-resistant bacteria,</a>" Methods in Enzymology 53:737-755 (1975).</td></tr><tr><td class="patent-data-table-td ">164</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Sheets, M.D. et al., (May 1998), "<a href='http://scholar.google.com/scholar?q="Efficient+construction+of+a+large+nonimmune+phage+antibody+library%3A+The+production+of+high-affinity+human+single-chain+antibodies+to+protein+antigens%2C"'>Efficient construction of a large nonimmune phage antibody library: The production of high-affinity human single-chain antibodies to protein antigens,</a>" Cell Biology 95(11):6157-6162.</td></tr><tr><td class="patent-data-table-td ">165</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Shusta, E.V. et al. (1999) "<a href='http://scholar.google.com/scholar?q="Yeast+Polypeptide+Fusion+Surface+Display+Levels+Predict+Thermal+Stability+and+Soluble+Secretion+Efficiency"'>Yeast Polypeptide Fusion Surface Display Levels Predict Thermal Stability and Soluble Secretion Efficiency</a>" Academic Press 292:949-956.</td></tr><tr><td class="patent-data-table-td ">166</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Shusta, E.V. et al., (Apr. 1999), "<a href='http://scholar.google.com/scholar?q="Biosynthetic+polypeptide+libraries%2C"'>Biosynthetic polypeptide libraries,</a>" Curr. Opin. Biotechnol. 10:117-122.</td></tr><tr><td class="patent-data-table-td ">167</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Shusta, E.V. et al., (Oct. 1999), "<a href='http://scholar.google.com/scholar?q="Yeast+Polypeptide+Fusion+Surface+Display+Levels+Predict+Thermal+Stability+and+Soluble+Secretion+Efficiency%2C"'>Yeast Polypeptide Fusion Surface Display Levels Predict Thermal Stability and Soluble Secretion Efficiency,</a>" J. Mol. Biol. 292:949-956.</td></tr><tr><td class="patent-data-table-td ">168</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Shusta, E.V., et al., (2000), Directed evolution of a stable scaffold for T-cell receptor engineering. Nat Biotechnol 18:754-759.</td></tr><tr><td class="patent-data-table-td ">169</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Skuzeski et al., "<a href='http://scholar.google.com/scholar?q="The+signal+for+a+leaky+UAG+stop+codon+in+several+plant+viruses+includes+the+two+downstream+codons%2C"'>The signal for a leaky UAG stop codon in several plant viruses includes the two downstream codons,</a>" J. Mol. Biol. 218:365-373 (1991).</td></tr><tr><td class="patent-data-table-td ">170</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Slanetz, A.E. and Bothwell, A.L., (1991), "<a href='http://scholar.google.com/scholar?q="Heterodimeric%2C+disulfide-linked+a+%2F+b+T+cell+receptors+in+solution"'>Heterodimeric, disulfide-linked a / b T cell receptors in solution</a>" Eur. J. Immunol. 21:179-183.</td></tr><tr><td class="patent-data-table-td ">171</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Soo Hoo, W.F. et al., (May 1992), "<a href='http://scholar.google.com/scholar?q="Characterization+of+a+single-chain+T-cell+receptor+expressed+in+Escherichia+coli%2C"'>Characterization of a single-chain T-cell receptor expressed in Escherichia coli,</a>" Proc. Natl. Acad. Sci. USA 89:4759-4763.</td></tr><tr><td class="patent-data-table-td ">172</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Speir, J.A. et al., (May 1998), "<a href='http://scholar.google.com/scholar?q="Structural+Basis+of+2C+TCR+Allorecognition+of+H-2Ld+Peptide+Complexes%2C"'>Structural Basis of 2C TCR Allorecognition of H-2Ld Peptide Complexes,</a>" Immunity 8:553-562.</td></tr><tr><td class="patent-data-table-td ">173</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Sudbery, P.E. (1994), "<a href='http://scholar.google.com/scholar?q="The+Non-Saccharomyces+Yeasts%2C"'>The Non-Saccharomyces Yeasts,</a>" Yeast 10:1707-1726.</td></tr><tr><td class="patent-data-table-td ">174</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Sykulev, Y. et al., (Dec. 1995) "<a href='http://scholar.google.com/scholar?q="The+Law+of+mass+action+governs+antigen-stimulated+cytolytic+activity+of+CD8%2B+cytotoxic+T+lymphocytes"'>The Law of mass action governs antigen-stimulated cytolytic activity of CD8+ cytotoxic T lymphocytes</a>" Proc. Natl. Acad. Sci. USA 92:11990-11992.</td></tr><tr><td class="patent-data-table-td ">175</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Sykulev, Y. et al., (Nov. 1994), "<a href='http://scholar.google.com/scholar?q="High-affinity+reactions+between+antigen-specific+T-cell+receptors+and+peptides+associated+with+allogeneic+and+syngeneic+major+histocompatibility+complex+class+I+proteins%2C"'>High-affinity reactions between antigen-specific T-cell receptors and peptides associated with allogeneic and syngeneic major histocompatibility complex class I proteins,</a>" Proc. Natl. Acad. Sci. USA 91:11487-11491.</td></tr><tr><td class="patent-data-table-td ">176</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Sykuley, Y. et al., "<a href='http://scholar.google.com/scholar?q="Kinetics+and+Affinity+of+Reactions+between+an+Antigen-Specific+T+Cell+Receptor+and+Peptide-MHC+Complexes"'>Kinetics and Affinity of Reactions between an Antigen-Specific T Cell Receptor and Peptide-MHC Complexes</a>" (1994), Immunity 1:15-22.</td></tr><tr><td class="patent-data-table-td ">177</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Syrulev et al., (Dec. 1995), "<a href='http://scholar.google.com/scholar?q="The+Law+of+Mass+Action+Governs+Antigen-stimulated+Cytolytic+Activity+of+CD8%2B+cytotoxic+T+Lymphocytes%2C"'>The Law of Mass Action Governs Antigen-stimulated Cytolytic Activity of CD8+ cytotoxic T Lymphocytes,</a>" Proc. Natl. Acad. Sci. USA92:11990-11992.</td></tr><tr><td class="patent-data-table-td ">178</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Takamatsu et al., "<a href='http://scholar.google.com/scholar?q="Expression+of+bacterial+chloramphenicol+acetyltransferase+gene+in+tobacco+plants+mediated+by+TMV-RNA%2C"'>Expression of bacterial chloramphenicol acetyltransferase gene in tobacco plants mediated by TMV-RNA,</a>" EMBO J. 6:307-311 (1987).</td></tr><tr><td class="patent-data-table-td ">179</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Takamatsu et al., "<a href='http://scholar.google.com/scholar?q="Production+of+enkephalin+in+tobacco+protoplasts+using+tobacco+mosaic+virus+RNA+vector%2C"'>Production of enkephalin in tobacco protoplasts using tobacco mosaic virus RNA vector,</a>" FEBS Lett. 269:73-76 (1990).</td></tr><tr><td class="patent-data-table-td ">180</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Takamatsu et al., J. Virol. 64:3686-3693 (1990).</td></tr><tr><td class="patent-data-table-td ">181</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Takamatsu et al., J. Virol. 65:1619-1622 (1991).</td></tr><tr><td class="patent-data-table-td ">182</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Tjoa, B.and Kranz, D.M., (Jul. 1992) "<a href='http://scholar.google.com/scholar?q="Diversity+of+T+cell+receptor-alpha+chain+transcripts+from+hyperimmune+alloreactive+T+cells."'>Diversity of T cell receptor-alpha chain transcripts from hyperimmune alloreactive T cells.</a>" J Immunol (United States) 149(1) 253-259.</td></tr><tr><td class="patent-data-table-td ">183</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Turpen and Dawson, "<a href='http://scholar.google.com/scholar?q="Amplification%2C+movement+and+expression+of+genes+in+plants+by+viral-based+vectors%2C"'>Amplification, movement and expression of genes in plants by viral-based vectors,</a>" Marcel Dekkar, New York, pp. 195-217.</td></tr><tr><td class="patent-data-table-td ">184</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Turpen and Dawson, "<a href='http://scholar.google.com/scholar?q="Transgenic+Plants%2C+Fundamentals+and+Applications%2C"'>Transgenic Plants, Fundamentals and Applications,</a>" Marcel Dekkar, New York, pp. 195-217 (1992).</td></tr><tr><td class="patent-data-table-td ">185</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Turpen et al., "<a href='http://scholar.google.com/scholar?q="Malarial+Epitopes+Expressed+on+the+Surface+of+Recombinant+Tobacco+Mosaic+Virus%2C"'>Malarial Epitopes Expressed on the Surface of Recombinant Tobacco Mosaic Virus,</a>" Bio/Technology 10:53-57 (1995).</td></tr><tr><td class="patent-data-table-td ">186</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Turpen, "<a href='http://scholar.google.com/scholar?q="Ph.D.+Dissertation%2C"'>Ph.D. Dissertation,</a>" University of California, Riverside, pp. 106-132 (1992).</td></tr><tr><td class="patent-data-table-td ">187</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Turpen, "<a href='http://scholar.google.com/scholar?q="Ph.D.+Dissertation%2C"'>Ph.D. Dissertation,</a>" University of California, Riverside, pp. 72-87 (1992).</td></tr><tr><td class="patent-data-table-td ">188</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Turpen, "<a href='http://scholar.google.com/scholar?q="Ph.D.+Dissertation%2C"'>Ph.D. Dissertation,</a>" University of California, Riverside, pp. 85-105 (1992).</td></tr><tr><td class="patent-data-table-td ">189</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Turpen, T.H., "<a href='http://scholar.google.com/scholar?q="a+Molecular+Genetic+Analysis+of+Host%2FViral+Interactions%2C+Implications+for+the+Use+of+Plant+RNA+Viruses+as+Gene+Vectors%2C"'>a Molecular Genetic Analysis of Host/Viral Interactions, Implications for the Use of Plant RNA Viruses as Gene Vectors,</a>" Chem. Abstracts 120(9):97427 (1992).</td></tr><tr><td class="patent-data-table-td ">190</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Udaka, K. et al., (1993), "<a href='http://scholar.google.com/scholar?q="A+ubiquitous+protein+is+the+source+of+naturally+occurring+peptides+that+are+recognized+by+a+CD8%2B+T-cell+clone"'>A ubiquitous protein is the source of naturally occurring peptides that are recognized by a CD8+ T-cell clone</a>" Proc. Natl. Acad. Sci. USA 90:11272-11276.</td></tr><tr><td class="patent-data-table-td ">191</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Ulrich et al. (Dec. 1995), "<a href='http://scholar.google.com/scholar?q="Expression+Studies+of+Catalytic+Antibodies%2C"'>Expression Studies of Catalytic Antibodies,</a>" Proceed. Natl. Acad. Sci. 92:11907-11911.</td></tr><tr><td class="patent-data-table-td ">192</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Usha et al., "<a href='http://scholar.google.com/scholar?q="Expression+of+an+animal+virus+antigenic+site+on+the+surface+of+a+plantvirus+particle%2C"'>Expression of an animal virus antigenic site on the surface of a plantvirus particle,</a>" Virology 197:366-374 (1993).</td></tr><tr><td class="patent-data-table-td ">193</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Valitutti, S.et al., "<a href='http://scholar.google.com/scholar?q="Serial+triggering+of+many+T-cell+receptors+by+a+few+peptide-MHC+complexes"'>Serial triggering of many T-cell receptors by a few peptide-MHC complexes</a>" (May 1995), Nature 375:148-151.</td></tr><tr><td class="patent-data-table-td ">194</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">van der Vaart (Sep. 1965), "<a href='http://scholar.google.com/scholar?q="Identification+and+Characterization+of+Cell+Wall+Proteins+of+Saccharomyces+ceevisiae%2C"'>Identification and Characterization of Cell Wall Proteins of Saccharomyces ceevisiae,</a>" Thesis. ISBN 90-393-1498-5 pp. 1-138.</td></tr><tr><td class="patent-data-table-td ">195</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Van Haute et al., EMBO J. 2:411-417 (1983).</td></tr><tr><td class="patent-data-table-td ">196</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Venton and Schell, NAR 13:6981 (1985).</td></tr><tr><td class="patent-data-table-td ">197</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Von Kammen et al., "<a href='http://scholar.google.com/scholar?q="Cowpea+mosaic+virus%2C+C.M.I.%2FA.A.B.+Descriptions+of+plant+viruses%2C"'>Cowpea mosaic virus, C.M.I./A.A.B. Descriptions of plant viruses,</a>" No. 197, Wm. Culross and Son Ltd., Coupar Angus, Perthshire, Scotland, pp. 1-5 (1978).</td></tr><tr><td class="patent-data-table-td ">198</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Walden and Schell, Eur. J. Biochem. 192:563-576 (1990).</td></tr><tr><td class="patent-data-table-td ">199</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Ward, E.S., (1991), "<a href='http://scholar.google.com/scholar?q="Expression+and+Secretion+of+T-Cell+Receptor+V%CE%B1+and+V%CE%B2+Domains+using+Escherichia+coli+as+a+Host%2C"'>Expression and Secretion of T-Cell Receptor Vα and Vβ Domains using Escherichia coli as a Host,</a>" Scand. J. Immunol. 34:215-220.</td></tr><tr><td class="patent-data-table-td ">200</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Ward, E.S., (1992), "<a href='http://scholar.google.com/scholar?q="Secretion+of+T+Cell+Receptor+Fragments+From+Recombinant+Escherichia+coli+Cells%2C"'>Secretion of T Cell Receptor Fragments From Recombinant Escherichia coli Cells,</a>" J. Mol. Biol. 224:885-890.</td></tr><tr><td class="patent-data-table-td ">201</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Weber et al., (Apr. 1992), "<a href='http://scholar.google.com/scholar?q="Specific+Low-affinity+Recognition+of+Major+Histocompatibility+Complex+Plus+Peptide+by+Soluble+T-cell+Receptor%2C"'>Specific Low-affinity Recognition of Major Histocompatibility Complex Plus Peptide by Soluble T-cell Receptor,</a>" Nature 356:793-796.</td></tr><tr><td class="patent-data-table-td ">202</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Wedemayer, G.J.et al., "<a href='http://scholar.google.com/scholar?q="Structural+Insights+into+the+Evolution+of+an+Antibody+Combining+Site"'>Structural Insights into the Evolution of an Antibody Combining Site</a>" (Jun. 1997), Science 276:1665-1669.</td></tr><tr><td class="patent-data-table-td ">203</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Weidanz, J.A. et al., (Dec. 1998), "<a href='http://scholar.google.com/scholar?q="Display+of+functional+%CE%B1%CE%B2+single-chain+T-cell+receptor+molecules+on+the+surface+of+bacteriophage%2C"'>Display of functional αβ single-chain T-cell receptor molecules on the surface of bacteriophage,</a>" J. Immunol. Meth. 221:59-76.</td></tr><tr><td class="patent-data-table-td ">204</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Weidanz, Jon A. et al. (Aug. 1998), "<a href='http://scholar.google.com/scholar?q="Display+of+functional+alphabeta+single-chain+T-cell+receptor+molecules+on+the+surface+of+bacteriophage"'>Display of functional alphabeta single-chain T-cell receptor molecules on the surface of bacteriophage</a>" Journal of Immunological Methods 221:59-76.</td></tr><tr><td class="patent-data-table-td ">205</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Weidanz, Jon A. et al. (Aug. 1998), "<a href='http://scholar.google.com/scholar?q="Display+of+functional+%CE%B1%CE%B2+single-chain+T-cell+receptor+molecules+on+the+surface+of+bacteriophage"'>Display of functional αβ single-chain T-cell receptor molecules on the surface of bacteriophage</a>" Journal of Immunological Methods 221:59-76.</td></tr><tr><td class="patent-data-table-td ">206</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Weiss et al., "<a href='http://scholar.google.com/scholar?q="A+T+cell+clone+directed+at+the+circumsporozoite+protein+which+protects+mice+against+both+Plasmodium+yoelli+and+Plasmodium+berghei%2C"'>A T cell clone directed at the circumsporozoite protein which protects mice against both Plasmodium yoelli and Plasmodium berghei,</a>" J. Immunol. 149:2103-2109 (1992).</td></tr><tr><td class="patent-data-table-td ">207</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Willcox, B.E. et al., "<a href='http://scholar.google.com/scholar?q="TCR+Binding+to+Peptide-MHC+Stabilizes+a+Flexible+Recognition+Interface"'>TCR Binding to Peptide-MHC Stabilizes a Flexible Recognition Interface</a>" (Mar. 1999), Immunity 10:357-365.</td></tr><tr><td class="patent-data-table-td ">208</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Winter, G. et al., "<a href='http://scholar.google.com/scholar?q="Making+Antibodies+By+Phage+Display+Technology"'>Making Antibodies By Phage Display Technology</a>" (1994) Annu. Rev. Immunol. 12:433-455.</td></tr><tr><td class="patent-data-table-td ">209</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Wittrup, D.K., "<a href='http://scholar.google.com/scholar?q="Phage+on+Display"'>Phage on Display</a>" (Nov. 1999), Trends in Biotech. 17(11):423-424.</td></tr><tr><td class="patent-data-table-td ">210</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Wittrup, K.D., (2000), The single cell as a microplate well. Nat Biotechnol 18:1039-1040.</td></tr><tr><td class="patent-data-table-td ">211</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Wolfe, M.S. et al., (Jan. 1998), A Substrate-Based Diffusion Ketone Selectively Inhibits Alzheimer's γ-Secretase Activity, J. Med. Chem. 41:6-9.</td></tr><tr><td class="patent-data-table-td ">212</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Yamaya et al., Mol. Gen. Genet. 211:520-525 (1988).</td></tr><tr><td class="patent-data-table-td ">213</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Yoon, ST., (1994), Both high and low avidiry antibodies to the T cell receptor can have agonist or antagonist activity. Immunity1:563-569.</td></tr><tr><td class="patent-data-table-td ">214</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Zaitlin and Hull, Ann. Rev. Physiol. 38:291-315 (1987).</td></tr><tr><td class="patent-data-table-td ">215</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Zaitlin et al., "<a href='http://scholar.google.com/scholar?q="Tobacco+mosaic+virus+%28type+strain%29%2C+C.M.I.%2FA.A.B.+Descriptions+of+plant+viruses%2C"'>Tobacco mosaic virus (type strain), C.M.I./A.A.B. Descriptions of plant viruses,</a>" No. 151, Wm. Culross and Son Ltd., Coupar Angus, Perthshire, Scotland, pp. 1-6 (1975).</td></tr><tr><td class="patent-data-table-td ">216</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Zambryski et al., EMBO J. 2:2143-2150 (1983).</td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="forward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Referenced by</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Citing Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7465787">US7465787</a></td><td class="patent-data-table-td patent-date-value">Dec 16, 2003</td><td class="patent-data-table-td patent-date-value">Dec 16, 2008</td><td class="patent-data-table-td ">The Board Of Trustees Of The University Of Illinois</td><td class="patent-data-table-td ">Transforming microorganism and monitoring expression of heterologous proteins on cell surface; monitoring protein-protein interactions and identifying modulators for treating autoimmune and cell proliferative diseases</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7608410">US7608410</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Oct 7, 2008</td><td class="patent-data-table-td patent-date-value">Oct 27, 2009</td><td class="patent-data-table-td ">Immunocore Limited</td><td class="patent-data-table-td ">Method of improving T cell receptors</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8022188">US8022188</a></td><td class="patent-data-table-td patent-date-value">Apr 23, 2007</td><td class="patent-data-table-td patent-date-value">Sep 20, 2011</td><td class="patent-data-table-td ">Abbott Laboratories</td><td class="patent-data-table-td ">Immunosuppressant binding antibodies and methods of obtaining and using same</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8034607">US8034607</a></td><td class="patent-data-table-td patent-date-value">Sep 12, 2008</td><td class="patent-data-table-td patent-date-value">Oct 11, 2011</td><td class="patent-data-table-td ">Wisconsin Alumni Research Foundation</td><td class="patent-data-table-td ">Methods of enhanced heterologous protein secretion</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8372636">US8372636</a></td><td class="patent-data-table-td patent-date-value">Dec 16, 2008</td><td class="patent-data-table-td patent-date-value">Feb 12, 2013</td><td class="patent-data-table-td ">Board Of Trustees Of The University Of Illinois</td><td class="patent-data-table-td ">Yeast cell surface display of proteins and uses thereof</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="classifications"></a><div class="patent-section-header"><span class="patent-section-title">Classifications</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th"> </th><th class="patent-data-table-th"> </th></tr></thead><tr><td class="patent-data-table-td ">U.S. Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=W4xnBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S372300">435/372.3</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=W4xnBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc530/defs530.htm&usg=AFQjCNEOd3srpZKCSH0W93D5fiJ4_0S25w#C530S350000">530/350</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=W4xnBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S372000">435/372</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=W4xnBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S366000">435/366</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=W4xnBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S325000">435/325</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=W4xnBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S320100">435/320.1</a></span></td></tr><tr><td class="patent-data-table-td ">International Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=W4xnBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C40B0040020000">C40B40/02</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=W4xnBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0015100000">C12N15/10</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=W4xnBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=G01N0033680000">G01N33/68</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=W4xnBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12N0005080000">C12N5/08</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=W4xnBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C07K0014725000">C07K14/725</a></span></td></tr><tr><td class="patent-data-table-td ">Cooperative Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=W4xnBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12N15/1037">C12N15/1037</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=W4xnBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=G01N2333/39">G01N2333/39</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=W4xnBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C40B40/02">C40B40/02</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=W4xnBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=G01N33/6854">G01N33/6854</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=W4xnBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=G01N33/6842">G01N33/6842</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=W4xnBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=G01N33/68">G01N33/68</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=W4xnBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C07K14/7051">C07K14/7051</a></span></td></tr><tr><td class="patent-data-table-td ">European Classification</td><td class="patent-data-table-td "><span class="nested-value">C07K14/705B12</span>, <span class="nested-value">G01N33/68</span>, <span class="nested-value">G01N33/68B</span>, <span class="nested-value">C40B40/02</span>, <span class="nested-value">C12N15/10C1</span>, <span class="nested-value">G01N33/68A8</span></td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="legal-events"></a><div class="patent-section-header"><span class="patent-section-title">Legal Events</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Date</th><th class="patent-data-table-th">Code</th><th class="patent-data-table-th">Event</th><th class="patent-data-table-th">Description</th></tr></thead><tr><td class="patent-data-table-td patent-date-value">May 20, 2014</td><td class="patent-data-table-td ">B1</td><td class="patent-data-table-td ">Reexamination certificate first reexamination</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">THE PATENTABILITY OF CLAIMS 7-10 AND 17-20 IS CONFIRMED.CLAIMS 5 AND 15 ARE CANCELLED.CLAIMS 1, 11 AND 21 ARE DETERMINED TO BE PATENTABLE AS AMENDED.CLAIMS 2-4, 6, 12-14, 16, 22 AND 23, DEPENDENT ON AN AMENDED CLAIM, ARE DETERMINED TO BE PATENTABLE.</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Apr 30, 2013</td><td class="patent-data-table-td ">RR</td><td class="patent-data-table-td ">Request for reexamination filed</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20130308</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Jan 6, 2012</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">8</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Oct 29, 2008</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">NATIONAL INSTITUTES OF HEALTH (NIH), U.S. DEPT. OF</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">CONFIRMATORY LICENSE;ASSIGNOR:UNIVERSITY OF ILLINOIS URBANA-CHAMPAIGN;REEL/FRAME:021759/0198</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20021203</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Dec 14, 2007</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">4</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Apr 17, 2007</td><td class="patent-data-table-td ">CC</td><td class="patent-data-table-td ">Certificate of correction</td><td class="patent-data-table-td "></td></tr><tr><td class="patent-data-table-td patent-date-value">May 14, 2001</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">BOARD OF TRUSTEES OF THE UNIVERSITY OF ILLINOIS, I</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNORS:KRANZ, DAVID M.;WITTRUP, K. DANE;HOLLER, PHILLIP D.;REEL/FRAME:011802/0962;SIGNING DATES FROM 20010410 TO 20010503</span></div><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">BOARD OF TRUSTEES OF THE UNIVERSITY OF ILLINOIS 50</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNORS:KRANZ, DAVID M. /AR;REEL/FRAME:011802/0962;SIGNING DATESFROM 20010410 TO 20010503</span></div></td></tr></table><div class="patent-section-footer"></div></div><div class="modal-dialog" id="patent-images-lightbox"><div class="patent-lightbox-controls"><div class="patent-lightbox-rotate-controls"><div class="patent-lightbox-rotation-text">Rotate</div><div class="rotate-icon rotate-ccw-icon"></div><div class="rotate-icon rotate-cw-icon"></div></div><div class="patent-lightbox-index-counter"></div><a class="patent-lightbox-fullsize-link" target="_blank">Original Image</a><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_right.png" alt="Next page"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_left.png" alt="Previous page"width="21" height="21" /></div></div><div class="modal-dialog-content"><div class="patent-lightbox-image-holder"><div class="patent-lightbox-placeholder"></div></div></div></div><script>_OC_initPatentsAtb({image_not_available_html: " Image not available"});</script></div></div></div></td></tr></table><script>(function() {var href = window.location.href;if (href.indexOf('?') !== -1) {var parameters = href.split('?')[1].split('&');for (var i = 0; i < parameters.length; i++) {var param = parameters[i].split('=');if (param[0] == 'focus') {var elem = document.getElementById(param[1]);if (elem) {elem.focus();}}}}})();</script><script>_OC_addFlags({LockSrc:"/books/javascript/lock_4ff636b3d23669b7103f3b3a3a18b4cd.js", Host:"http://www.google.com/", IsBooksRentalEnabled:1, IsWebstoreDisplayCaseEnabled:1, IsObfuscationEnabled:1, IsBrowsingHistoryEnabled:1, IsWebReaderSvgEnabled:0, IsGeoLayerEnabled:1, IsImageModeNotesEnabled:1, IsCopyMenuItemEnabled:1, IsGiftingEnabled:0, IsWebReaderUniversalPaginatorEnabled:0, IsOfflineBubbleEnabled:1, IsReaderEnabledForPlayRequests:1, IsFutureOnSaleVolumesEnabled:1, IsOfflineRestrictedCopyEnabled:1, IsBooksUnifiedLeftNavEnabled:1, IsRestrictedCopyEnabled:1, IsZipitFolderCollectionEnabled:1, IsEndOfSampleRecommendationsEnabled:1, IsRatingsOnBookcardsEnabled:1, IsAdsDisabled:0, IsIframePageDisplayEnabled:0, IsEmbeddedMediaEnabled:1, IsImageModeAnnotationsEnabled:1, IsMyLibraryGooglePlusEnabled:1, IsImagePageProviderEnabled:0, IsBookcardListPriceSmall:0, IsInternalUser:0, IsBooksShareButtonEnabled:0, IsPreOrdersEnabled:0, IsDisabledRandomBookshelves:0, WebstoreDisplayCasePosition:3});_OC_Run({"enable_p13n":false,"add_vol_to_collection_base_url":"http://www.google.com/patents?op=add\u0026sig=ACfU3U2XZi_D-KM99pblYh-msnnl1qcPVQ\u0026id=W4xnBAABERAJ","remove_vol_from_collection_base_url":"http://www.google.com/patents?op=remove\u0026sig=ACfU3U27YxmGTSs9PJ79fhjmvEA7iq_3fw\u0026id=W4xnBAABERAJ","logged_in":false,"p13n_save_user_settings_url":"http://www.google.com/patents?op=edit_user_settings\u0026sig=ACfU3U3f1Bjlr6eM9TrqiBhCeBTDj-4xAQ","is_cobrand":false,"sign_in_url":"https://www.google.com/accounts/Login?service=\u0026continue=http://www.google.com/patents%3Fhl%3Den\u0026hl=en","is_play_enabled":true}, {"volume_id":"","is_ebook":true,"volumeresult":{"has_flowing_text":false,"has_scanned_text":true,"can_download_pdf":false,"can_download_epub":false,"is_pdf_drm_enabled":false,"is_epub_drm_enabled":false,"download_pdf_url":"http://www.google.com/patents/download/High_affinity_TCR_proteins_and_methods.pdf?id=W4xnBAABERAJ\u0026output=pdf\u0026sig=ACfU3U1MZ1mtGTSN1CHQ5R5rh66aXqMF_Q"},"sample_url":"http://www.google.com/patents/reader?id=W4xnBAABERAJ\u0026printsec=frontcover\u0026output=reader\u0026source=gbs_atb_hover","is_browsable":true,"is_public_domain":true}, {});</script><div id="footer_table" style="font-size:83%;text-align:center;position:relative;top:20px;height:4.5em;margin-top:2em"><div style="margin-bottom:8px"><a href=http://www.google.com/><nobr>Google&nbsp;Home</nobr></a> - <a href=//www.google.com/patents/sitemap/><nobr>Sitemap</nobr></a> - <a href=http://www.google.com/googlebooks/uspto.html><nobr>USPTO Bulk Downloads</nobr></a> - <a href=/intl/en/privacy/><nobr>Privacy Policy</nobr></a> - <a href=/intl/en/policies/terms/><nobr>Terms of Service</nobr></a> - <a href=https://support.google.com/faqs/answer/2539193?hl=en><nobr>About Google Patents</nobr></a> - <a href="http://www.google.com/tools/feedback/intl/en/error.html" onclick="try{_OC_startFeedback({productId: '72792',locale: 'en'});return false;}catch(e){}"><nobr>Send Feedback</nobr></a></div><span>Data provided by IFI CLAIMS Patent Services</span><br><span >&copy;2012 Google</span></div> <script type="text/javascript">var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));</script><script type="text/javascript">var pageTracker = _gat._getTracker("UA-27188110-1");pageTracker._setCookiePath("/patents/");pageTracker._trackPageview();</script> </body></html>