<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN""http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html><head><title>Patent US6808888 - Amplificaton of nucleotide sequences present in sample; provide sample ... - Google Patents</title><script>(function(){(function(){function e(a){this.t={};this.tick=function(a,c,b){var d=void 0!=b?b:(new Date).getTime();this.t[a]=[d,c];if(void 0==b)try{window.console.timeStamp("CSI/"+a)}catch(e){}};this.tick("start",null,a)}var a;window.performance&&(a=window.performance.timing);var f=a?new e(a.responseStart):new e;window.jstiming={Timer:e,load:f};if(a){var c=a.navigationStart,d=a.responseStart;0<c&&d>=c&&(window.jstiming.srt=d-c)}if(a){var b=window.jstiming.load;0<c&&d>=c&&(b.tick("_wtsrt",void 0,c),b.tick("wtsrt_",
"_wtsrt",d),b.tick("tbsd_","wtsrt_"))}try{a=null,window.chrome&&window.chrome.csi&&(a=Math.floor(window.chrome.csi().pageT),b&&0<c&&(b.tick("_tbnd",void 0,window.chrome.csi().startE),b.tick("tbnd_","_tbnd",c))),null==a&&window.gtbExternal&&(a=window.gtbExternal.pageT()),null==a&&window.external&&(a=window.external.pageT,b&&0<c&&(b.tick("_tbnd",void 0,window.external.startE),b.tick("tbnd_","_tbnd",c))),a&&(window.jstiming.pt=a)}catch(g){}})();})();
</script><link rel="stylesheet" href="/patents/css/_8a2b04e7bf975d5171d8e4c0b6365c7a/kl_intl_patents_bundle.css" type="text/css" /><script src="/books/javascript/atb_8a2b04e7bf975d5171d8e4c0b6365c7a__en.js"></script><script>function googleTranslateElementInit() {new google.translate.TranslateElement({pageLanguage: "en",gaTrack: true,gaId: "UA-27188110-1",multilanguagePage: true});}</script><script src="//translate.google.com/translate_a/element.js?cb=googleTranslateElementInit"></script><meta name="DC.type" content="Patent"><meta name="DC.title" content="Amplificaton of nucleotide sequences present in sample; provide sample containing nucleotide sequnces, incubate with primers, amplify sequences, detect amplified product"><meta name="DC.contributor" content="Jing Zhang" scheme="inventor"><meta name="DC.contributor" content="John Dekker" scheme="inventor"><meta name="DC.contributor" content="Antoinette C. van der Kuyl" scheme="inventor"><meta name="DC.contributor" content="Jolanda Maas" scheme="inventor"><meta name="DC.contributor" content="Bob Van Gemen" scheme="inventor"><meta name="DC.contributor" content="Primagen Holding B.V." scheme="assignee"><meta name="DC.date" content="2002-7-10" scheme="dateSubmitted"><meta name="DC.description" content="Methods for amplifying nucleic acid in a sample comprising providing the sample with a set of primers to enable synthesis of at least one nucleic acid strand complementary to at least part of the nucleic acid, wherein the set of primers comprises between 3-8 random bases, preferably clustered near the 3′ end of each primer in said set of primers. The methods of the invention are useful, for example, for determining whether samples derived from humans, mammals, poultry, or fish comprise nucleic acid of a pathogen. The methods are further suited for typing the pathogen and typing particular variants of said pathogen. The methods are also suited for the elucidation of the gene expression profile or genetic profile of cells."><meta name="DC.date" content="2004-10-26" scheme="issued"><meta name="DC.relation" content="EP:0466520:A1" scheme="references"><meta name="DC.relation" content="US:20010021518:A1" scheme="references"><meta name="DC.relation" content="US:20030022194:A1" scheme="references"><meta name="DC.relation" content="US:5104792" scheme="references"><meta name="DC.relation" content="US:5106727" scheme="references"><meta name="DC.relation" content="US:5948649" scheme="references"><meta name="DC.relation" content="US:6521428" scheme="references"><meta name="DC.relation" content="WO:1996015264:A2" scheme="references"><meta name="DC.relation" content="WO:1997030062:A1" scheme="references"><meta name="DC.relation" content="WO:1998002575:A1" scheme="references"><meta name="DC.relation" content="WO:2001051661:A2" scheme="references"><meta name="DC.relation" content="WO:2002020571:A2" scheme="references"><meta name="citation_reference" content="McGrath et al., Sequence analysis of DNA randomly amplified from the Saccharomyces cerevisiae genome, Molecular and Cellular Probes, 1998, pp. 397-405. vol. 12."><meta name="citation_reference" content="PCT International Preliminary Examination Report, PCT NL01 00020, dated Mar. 25, 2003, 2 pages."><meta name="citation_reference" content="PCT International Search, PCT-NL01 00020, dated Jul. 18, 2002, 2 pages."><meta name="citation_reference" content="Rose et al., Consensus-degenerate hybrid oligonucleotide primers for amplification of distantly related sequences. Nucleic Acids Research, 1998, pp. 1628-1635, vol. 26, No. 7."><meta name="citation_reference" content="Smith et al , Automated differential display using a flourescently labeled universal primer, Biotechniques, 1997, pp. 274-279, vol. 23, No. 2."><meta name="citation_patent_number" content="US:6808888"><meta name="citation_patent_application_number" content="US:10/192,786"><link rel="canonical" href="http://www.google.com/patents/US6808888"/><meta property="og:url" content="http://www.google.com/patents/US6808888"/><meta name="title" content="Patent US6808888 - Amplificaton of nucleotide sequences present in sample; provide sample containing nucleotide sequnces, incubate with primers, amplify sequences, detect amplified product"/><meta name="description" content="Methods for amplifying nucleic acid in a sample comprising providing the sample with a set of primers to enable synthesis of at least one nucleic acid strand complementary to at least part of the nucleic acid, wherein the set of primers comprises between 3-8 random bases, preferably clustered near the 3′ end of each primer in said set of primers. The methods of the invention are useful, for example, for determining whether samples derived from humans, mammals, poultry, or fish comprise nucleic acid of a pathogen. The methods are further suited for typing the pathogen and typing particular variants of said pathogen. The methods are also suited for the elucidation of the gene expression profile or genetic profile of cells."/><meta property="og:title" content="Patent US6808888 - Amplificaton of nucleotide sequences present in sample; provide sample containing nucleotide sequnces, incubate with primers, amplify sequences, detect amplified product"/><meta property="og:type" content="book"/><meta property="og:site_name" content="Google Books"/><meta property="og:image" content="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><link rel="image_src" href="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"/><script>(function(){try{var aa=function(a,b,c,d){d=d||{};d._sn=["cfg",b,c].join(".");window.gbar.logger.ml(a,d)};var g=window.gbar=window.gbar||{},l=window.gbar.i=window.gbar.i||{},m={},n;function _tvn(a,b){var c=parseInt(a,10);return isNaN(c)?b:c}function _tvf(a,b){var c=parseFloat(a);return isNaN(c)?b:c}function _tvv(a){return!!a}function p(a,b,c){(c||g)[a]=b}g.bv={n:_tvn("2",0),r:"",f:".67.",e:"0",m:_tvn("0",1)};
function q(a,b,c){var d="on"+b;if(a.addEventListener)a.addEventListener(b,c,!1);else if(a.attachEvent)a.attachEvent(d,c);else{var f=a[d];a[d]=function(){var a=f.apply(this,arguments),b=c.apply(this,arguments);return void 0==a?b:void 0==b?a:b&&a}}}var s=function(a){return function(){return g.bv.m==a}},ba=s(1),ca=s(2);p("sb",ba);p("kn",ca);l.a=_tvv;l.b=_tvf;l.c=_tvn;l.i=aa;var da=window.gbar.i.i;var t,u,v,w;function ea(a){v=a}function fa(a){var b;if(b=v&&window.encodeURIComponent)b=a.href,b=!b.match(/^http[s]?:\/\/accounts\.google\.[^/]*\/ClearSID/i)&&!b.match(/^http[s]?:\/\/[^/]*\/accounts\/ClearSID/i);if(b=b&&encodeURIComponent(v()))a.href=a.href.replace(/([?&]continue=)[^&]*/,"$1"+b)}function ga(a){window.gApplication&&(a.href=window.gApplication.getTabUrl(a.href))}
function ha(a){var b=document.forms[0].q,c=window.encodeURIComponent&&b&&b.value,b=b&&b.placeholder;c&&c!=b&&(a.href=a.href.replace(/([?&])q=[^&]*|$/,function(a,b){return(b||"&")+"q="+encodeURIComponent(c)}))}n=l.a("")?ga:ha;
function x(a,b,c,d,f,e){var h=document.getElementById(a);if(h){var k=h.style;k.left=d?"auto":b+"px";k.right=d?b+"px":"auto";k.top=c+"px";k.visibility=u?"hidden":"visible";f&&e?(k.width=f+"px",k.height=e+"px"):(x(t,b,c,d,h.offsetWidth,h.offsetHeight),u=u?"":a)}}
var y=[],ia=function(a,b){y.push(b)},ja=function(a){a=a||window.event;var b=a.target||a.srcElement;a.cancelBubble=!0;null==t&&(a=document.createElement(Array.every||window.createPopup?"iframe":"div"),a.frameBorder="0",t=a.id="gbs",a.src="javascript:''",b.parentNode.appendChild(a),q(document,"click",z));var c=b,b=0;"gb3"!=c.className&&(c=c.parentNode);a=c.getAttribute("aria-owns")||"gbi";var d=c.offsetWidth,f=20<c.offsetTop?46:24;document.getElementById("tphdr")&&(f-=3);var e=!1;do b+=c.offsetLeft||
0;while(c=c.offsetParent);var c=(document.documentElement.clientWidth||document.body.clientWidth)-b-d,h,d=document.body,k=document.defaultView;k&&k.getComputedStyle?(d=k.getComputedStyle(d,""))&&(h=d.direction):h=d.currentStyle?d.currentStyle.direction:d.style.direction;h="rtl"==h;if("gbi"==a){for(d=0;k=y[d++];)k();A(null,window.navExtra);h&&(b=c,e=!0)}else h||(b=c,e=!0);u!=a&&z();x(a,b,f,e)},z=function(){u&&x(u,0,0)},A=function(a,b){var c,d=document.getElementById("gbi"),f=a;f||(f=d.firstChild);
for(;b&&(c=b.pop());){var e=d,h=c,k=f;w||(w="gb2");e.insertBefore(h,k).className=w}},ka=function(a,b,c){if((b=document.getElementById(b))&&a){a.className="gb4";var d=document.createElement("span");d.appendChild(a);d.appendChild(document.createTextNode(" | "));d.id=c;b.appendChild(d)}},la=function(){return document.getElementById("gb_70")},ma=function(){return!!u};p("qs",n);p("setContinueCb",ea);p("pc",fa);p("tg",ja);p("close",z);p("addLink",ka);p("almm",A);p("si",la);p("adh",ia);p("op",ma);var B=function(){},C=function(){},F=function(a){var b=new Image,c=D;b.onerror=b.onload=b.onabort=function(){try{delete E[c]}catch(a){}};E[c]=b;b.src=a;D=c+1},E=[],D=0;p("logger",{il:C,ml:B,log:F});var G=window.gbar.logger;var H={},na={},I=[],oa=l.b("0.1",.1),pa=l.a("1",!0),qa=function(a,b){I.push([a,b])},ra=function(a,b){H[a]=b},sa=function(a){return a in H},J={},K=function(a,b){J[a]||(J[a]=[]);J[a].push(b)},ta=function(a){K("m",a)},L=function(a,b){var c=document.createElement("script");c.src=a;c.async=pa;Math.random()<oa&&(c.onerror=function(){c.onerror=null;B(Error("Bundle load failed: name="+(b||"UNK")+" url="+a))});(document.getElementById("xjsc")||document.getElementsByTagName("body")[0]||
document.getElementsByTagName("head")[0]).appendChild(c)},N=function(a){for(var b=0,c;(c=I[b])&&c[0]!=a;++b);!c||c[1].l||c[1].s||(c[1].s=!0,M(2,a),c[1].url&&L(c[1].url,a),c[1].libs&&m.d&&m.d(c[1].libs))},O=function(a){K("gc",a)},P=null,ua=function(a){P=a},M=function(a,b,c){if(P){a={t:a,b:b};if(c)for(var d in c)a[d]=c[d];try{P(a)}catch(f){}}};p("mdc",H);p("mdi",na);p("bnc",I);p("qGC",O);p("qm",ta);p("qd",J);p("lb",N);p("mcf",ra);p("bcf",qa);p("aq",K);p("mdd","");p("has",sa);
p("trh",ua);p("tev",M);var Q=l.b("0.1",.001),R=0;
function _mlToken(a,b){try{if(1>R){R++;var c,d=a,f=b||{},e=encodeURIComponent,h=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&jexpid=",e("17483"),"&srcpg=",e("prop=22"),"&jsr=",Math.round(1/Q),"&ogev=",e("zKztU43SKrW6sQSYvIDIBA"),"&ogf=",g.bv.f,"&ogrp=",e("1"),"&ogv=",e("1407723702.0"),"&oggv="+e("es_plusone_gc_20140723.0_p0"),"&ogd=",e("com"),"&ogc=",e("PAN"),"&ogl=",e("en")];f._sn&&(f._sn="og."+
f._sn);for(var k in f)h.push("&"),h.push(e(k)),h.push("="),h.push(e(f[k]));h.push("&emsg=");h.push(e(d.name+":"+d.message));var r=h.join("");S(r)&&(r=r.substr(0,2E3));c=r;var Aa=window.gbar.logger._aem(a,c);F(Aa)}}catch(Na){}}var S=function(a){return 2E3<=a.length},va=function(a,b){return b};function T(a){B=a;p("_itl",S,G);p("_aem",va,G);p("ml",B,G);a={};H.er=a}l.a("")?T(function(a){throw a;}):l.a("1")&&Math.random()<Q&&T(_mlToken);I.push(["m",{url:"//ssl.gstatic.com/gb/js/scm_7385cc5883250b43a39405734c1bea59.js"}]);g.mcf("c",{});g.sg={c:""};if(l.a("1")){var wa=l.a("");I.push(["gc",{auto:wa,url:"//ssl.gstatic.com/gb/js/abc/gci_91f30755d6a6b787dcc2a4062e6e9824.js",libs:"googleapis.client:plusone:gapi.iframes"}]);var xa={version:"gci_91f30755d6a6b787dcc2a4062e6e9824.js",index:"",lang:"en"};H.gc=xa;var U=function(a){window.googleapis&&window.iframes?a&&a():(a&&O(a),N("gc"))};p("lGC",U);l.a("1")&&p("lPWF",U)};window.__PVT="";if(l.a("1")&&l.a("1")){var V=function(a){U(function(){K("pw",a);N("pw")})};p("lPW",V);I.push(["pw",{url:"//ssl.gstatic.com/gb/js/abc/pwm_45f73e4df07a0e388b0fa1f3d30e7280.js"}]);var W=[],ya=function(a){W[0]=a},za=function(a,b){var c=b||{};c._sn="pw";B(a,c)},Ba={signed:W,elog:za,base:"https://plusone.google.com/u/0",loadTime:(new Date).getTime()};H.pw=Ba;var X=function(a,b){for(var c=b.split("."),d=function(){var b=arguments;a(function(){for(var a=g,d=0,e=c.length-1;d<e;++d)a=a[c[d]];a[c[d]].apply(a,b)})},f=g,e=0,h=c.length-1;e<h;++e)f=
f[c[e]]=f[c[e]]||{};return f[c[e]]=d};X(V,"pw.clk");X(V,"pw.hvr");p("su",ya,g.pw)};function Ca(){function a(){for(var b;(b=e[h++])&&"m"!=b[0]&&!b[1].auto;);b&&(M(2,b[0]),b[1].url&&L(b[1].url,b[0]),b[1].libs&&m.d&&m.d(b[1].libs));h<e.length&&setTimeout(a,0)}function b(){0<f--?setTimeout(b,0):a()}var c=l.a("1"),d=l.a(""),f=3,e=I,h=0,k=window.gbarOnReady;if(k)try{k()}catch(r){da(r,"ml","or")}d?p("ldb",a):c?q(window,"load",b):b()}p("rdl",Ca);var Da={D:1,H:2,da:3,p:4,W:5,M:6,F:7,g:8,ha:9,U:10,L:11,T:12,S:13,N:14,Q:15,P:16,fa:17,w:18,O:19,ga:20,ea:21,u:22,G:23,ja:24,ka:25,ia:26,A:27,j:28,o:29,k:30,ca:31,Z:32,$:33,J:34,K:35,ba:36,aa:37,Y:38,B:39,R:40,v:41,X:42,V:43,h:48,C:49,I:500},Y=[1,2,3,4,5,6,9,10,11,13,14,28,29,30,34,35,37,38,39,40,41,42,43,48,49,500];var Z=l.b("0.001",1E-4),Ea=l.b("1",1),Fa=!1,Ga=!1;if(l.a("1")){var Ha=Math.random();Ha<=Z&&(Fa=!0);Ha<=Ea&&(Ga=!0)}var Ia=Da,$=null;function Ja(){var a=0,b=function(b,d){l.a(d)&&(a|=b)};b(1,"");b(2,"");b(4,"");b(8,"");return a}
function Ka(a,b){var c=Z,d=Fa,f;f=a;if(!$){$={};for(var e=0;e<Y.length;e++){var h=Y[e];$[h]=!0}}if(f=!!$[f])c=Ea,d=Ga;if(d){d=encodeURIComponent;g.rp?(f=g.rp(),f="-1"!=f?f:"1"):f="1";c=["//www.google.com/gen_204?atyp=i&zx=",(new Date).getTime(),"&oge=",a,"&ogex=",d("17483"),"&ogev=",d("zKztU43SKrW6sQSYvIDIBA"),"&ogf=",g.bv.f,"&ogp=",d("22"),"&ogrp=",d(f),"&ogsr=",Math.round(1/c),"&ogv=",d("1407723702.0"),"&oggv="+
d("es_plusone_gc_20140723.0_p0"),"&ogd=",d("com"),"&ogl=",d("en"),"&ogc=",d("PAN"),"&ogus=",Ja()];if(b){"ogw"in b&&(c.push("&ogw="+b.ogw),delete b.ogw);var k;f=b;e=[];for(k in f)0!=e.length&&e.push(","),e.push(La(k)),e.push("."),e.push(La(f[k]));k=e.join("");""!=k&&(c.push("&ogad="),c.push(d(k)))}F(c.join(""))}}function La(a){"number"==typeof a&&(a+="");return"string"==typeof a?a.replace(".","%2E").replace(",","%2C"):a}C=Ka;p("il",C,G);var Ma={};H.il=Ma;setTimeout(function(){C(Ia.g)},0);}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var b=window.gbar.i.i;var c=window.gbar;var f=function(d){try{var a=document.getElementById("gbom");a&&d.appendChild(a.cloneNode(!0))}catch(e){b(e,"omas","aomc")}};c.aomc=f;}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{var a=window.gbar;a.mcf("pm",{p:""});}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
(function(){try{window.gbar.rdl();}catch(e){window.gbar&&gbar.logger&&gbar.logger.ml(e,{"_sn":"cfg.init"});}})();
if (window['_OC_timingAction']) {window['_OC_timingAction']('patents_refpage');}</script><style>#gbar,#guser{font-size:13px;padding-top:1px !important;}#gbar{float:left;height:22px}#guser{padding-bottom:7px !important;text-align:right}.gbh,.gbd{border-top:1px solid #c9d7f1;font-size:1px}.gbh{height:0;position:absolute;top:24px;width:100%}#gbs,.gbm{background:#fff;left:0;position:absolute;text-align:left;visibility:hidden;z-index:1000}.gbm{border:1px solid;border-color:#c9d7f1 #36c #36c #a2bae7;z-index:1001}.gb1{margin-right:.5em}.gb1,.gb3{zoom:1}.gb2{display:block;padding:.2em .5em}.gb2,.gb3{text-decoration:none !important;border-bottom:none}a.gb1,a.gb4{text-decoration:underline !important}a.gb1,a.gb2,a.gb3,a.gb4{color:#00c !important}.gbi .gb3,.gbi .gb2,.gbi .gb4{color:#dd8e27 !important}.gbf .gb3,.gbf .gb2,.gbf .gb4{color:#900 !important}a.gb2:hover{background:#36c;color:#fff !important}#gbar .gbz0l{color:#000 !important;cursor:default;font-weight:bold;text-decoration:none !important}
#gbar { padding:.3em .6em !important;}</style></head><body  topmargin="3" marginheight="3"><div id=gbar><nobr><a onclick=gbar.qs(this);gbar.logger.il(1,{t:1}); class=gb1 id=gb_1 href="https://www.google.com/search?sa=N&tab=tw">Search</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:2}); class=gb1 id=gb_2 href="http://www.google.com/search?hl=en&tbm=isch&source=og&sa=N&tab=ti">Images</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:8}); class=gb1 id=gb_8 href="http://maps.google.com/maps?hl=en&sa=N&tab=tl">Maps</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:78}); class=gb1 id=gb_78 href="https://play.google.com/?hl=en&sa=N&tab=t8">Play</a> <a onclick=gbar.qs(this);gbar.logger.il(1,{t:36}); class=gb1 id=gb_36 href="http://www.youtube.com/results?sa=N&tab=t1">YouTube</a> <a onclick=gbar.logger.il(1,{t:5}); class=gb1 id=gb_5 href="http://news.google.com/nwshp?hl=en&tab=tn">News</a> <a onclick=gbar.logger.il(1,{t:23}); class=gb1 id=gb_23 href="https://mail.google.com/mail/?tab=tm">Gmail</a> <a onclick=gbar.logger.il(1,{t:25}); class=gb1 id=gb_25 href="https://drive.google.com/?tab=to">Drive</a> <a class=gb3 href="http://www.google.com/intl/en/options/" onclick="this.blur();gbar.tg(event);return !1" aria-haspopup=true><u>More</u> <small>&#9660;</small></a><div class=gbm id=gbi><a onclick=gbar.logger.il(1,{t:24}); class=gb2 id=gb_24 href="https://www.google.com/calendar?tab=tc">Calendar</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:51}); class=gb2 id=gb_51 href="http://translate.google.com/?hl=en&sa=N&tab=tT">Translate</a><a onclick=gbar.logger.il(1,{t:17}); class=gb2 id=gb_17 href="http://www.google.com/mobile/?hl=en&tab=tD">Mobile</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:10}); class=gb2 id=gb_10 href="http://www.google.com/search?hl=en&tbo=u&tbm=bks&source=og&sa=N&tab=tp">Books</a><a onclick=gbar.logger.il(1,{t:212}); class=gb2 id=gb_212 href="https://wallet.google.com/manage/?tab=ta">Wallet</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:6}); class=gb2 id=gb_6 href="http://www.google.com/search?hl=en&tbo=u&tbm=shop&source=og&sa=N&tab=tf">Shopping</a><a onclick=gbar.logger.il(1,{t:30}); class=gb2 id=gb_30 href="http://www.blogger.com/?tab=tj">Blogger</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:27}); class=gb2 id=gb_27 href="http://www.google.com/finance?sa=N&tab=te">Finance</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:31}); class=gb2 id=gb_31 href="https://plus.google.com/photos?sa=N&tab=tq">Photos</a><a onclick=gbar.qs(this);gbar.logger.il(1,{t:12}); class=gb2 id=gb_12 href="http://www.google.com/search?hl=en&tbo=u&tbm=vid&source=og&sa=N&tab=tv">Videos</a><div class=gb2><div class=gbd></div></div><a onclick=gbar.logger.il(1,{t:66}); href="http://www.google.com/intl/en/options/" class=gb2>Even more &raquo;</a></div></nobr></div><div id=guser width=100%><nobr><span id=gbn class=gbi></span><span id=gbf class=gbf></span><span id=gbe></span><a target=_top id=gb_70 href="https://www.google.com/accounts/Login?service=&continue=http://www.google.com/patents%3Fhl%3Den&hl=en" class=gb4>Sign in</a><div style="display: none"><div class=gbm id=gbd5 aria-owner=gbg5><div class=gbmc><ol id=gbom class=gbmcc></ol></div></div></div></nobr></div><div class=gbh style=left:0></div><div class=gbh style=right:0></div><div role="alert" style="position: absolute; left: 0; right: 0;"><a href="http://www.google.com/patents/us6808888?hl=en&amp;output=html_text" title="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."><img border="0" src="http://www.google.com/images/cleardot.gif"alt="Screen reader users: click this link for accessible mode. Accessible mode has the same essential features but works better with your reader."></a></div><div id="guser"><nobr></nobr></div><div style="clear:both;"></div><div id="gb-top-search-box" class="gb-top-search-box-small gb-reset"><table><tr><td class="logo"><a href="http://www.google.com/patents" class="logo-link"><img class="logo-img" src="/intl/en/images/logos/google_logo_41.png" alt="Go to Google Books Home" height="41"/></a></td><td><form action="http://www.google.com/search" name="f" id="vheadf" method="get"><span id="hf"></span><input type="hidden" name="tbm" value="pts"/><input type="hidden" name="tbo" value="1"/><input type="hidden" name="hl" value="en"/><table><tr><td><div class="inputs"><table><tr><td><div class="text-input"><input type="text" name="q" id="vheadq" class="text" maxlength="2048" size="31" value="" title="Search Patents" accesskey="s" autocomplete="off"/><script>window._OC_autoDir &&window._OC_autoDir('vheadq', 'tia-vheadq');</script></div></td><td><div class="submit-input"><input name="btnG" class="submit" type="submit" value=""/></div></td></tr></table></div></td><td class="col-ext-links"><div class="ext-links"><a href="http://www.google.com/advanced_patent_search">&lt;nobr&gt;Advanced Patent Search&lt;/nobr&gt;</a></div></td></tr></table></form></td></tr></table></div><div class="kd-appbar"><h2 class="kd-appname"><a href="/patents">Patents</a></h2><div class="kd-buttonbar left" id="left-toolbar-buttons"><a id="appbar-write-review-link" href=""></a><a id="appbar-view-print-sample-link" href=""></a><a id="appbar-view-ebook-sample-link" href=""></a><a id="appbar-patents-prior-art-finder-link" href="https://www.google.com/patents/related/US6808888"></a><a id="appbar-patents-discuss-this-link" href="http://www.google.com/url?id=hRtpBAABERAJ&amp;q=http://patents.stackexchange.com/redirect/google-patents%3Fpatent%3DUS6808888&amp;usg=AFQjCNF46bXUicRY0M89TrHUrRH7iBQ-jg" data-is-grant="true"></a><a id="appbar-read-patent-link" href="//docs.google.com/viewer?url=patentimages.storage.googleapis.com/pdfs/US6808888.pdf"></a><a id="appbar-download-pdf-link" href="//patentimages.storage.googleapis.com/pdfs/US6808888.pdf"></a><a class="appbar-application-grant-link" data-label="Application" href="/patents/US20030073112"></a><a class="appbar-application-grant-link" data-selected="true" data-label="Grant" href="/patents/US6808888"></a></div><div class="kd-buttonbar right" id="right-toolbar-buttons"></div></div><div id="books-microdata" itemscope=""itemtype="http://schema.org/Book"itemid="http://www.google.com/patents/US6808888" style="display:none"><span itemprop="description">Methods for amplifying nucleic acid in a sample comprising providing the sample with a set of primers to enable synthesis of at least one nucleic acid strand complementary to at least part of the nucleic acid, wherein the set of primers comprises between 3-8 random bases, preferably clustered near the...</span><span itemprop="url">http://www.google.com/patents/US6808888?utm_source=gb-gplus-share</span><span class="main-title" itemprop="name">Patent US6808888 - Amplificaton of nucleotide sequences present in sample; provide sample containing nucleotide sequnces, incubate with primers, amplify sequences, detect amplified product</span><img itemprop="image" src="http://www.google.com/patents?id=&amp;printsec=frontcover&amp;img=1&amp;zoom=1"alt="Patent US6808888 - Amplificaton of nucleotide sequences present in sample; provide sample containing nucleotide sequnces, incubate with primers, amplify sequences, detect amplified product" title="Patent US6808888 - Amplificaton of nucleotide sequences present in sample; provide sample containing nucleotide sequnces, incubate with primers, amplify sequences, detect amplified product"></div><div style="display: none"><ol id="ofe-gear-menu-contents" class="gbmcc"><li class="gbe gbmtc"><a class="gbmt goog-menuitem-content" id="" href="http://www.google.com/advanced_patent_search">Advanced Patent Search</a></li></ol></div><table id="viewport_table" cellpadding="0" style="clear:both" cellspacing="0"><tr><td id="viewport_td"><div class=vertical_module_list_row><div id=intl_patents class=about_content><div id=intl_patents_v><table class="patent-bibdata"><tr><td class="patent-bibdata-heading">Publication number</td><td class="single-patent-bibdata">US6808888 B2</td></tr><tr><td class="patent-bibdata-heading">Publication type</td><td class="single-patent-bibdata">Grant</td></tr><tr><td class="patent-bibdata-heading">Application number</td><td class="single-patent-bibdata">US 10/192,786</td></tr><tr><td class="patent-bibdata-heading">Publication date</td><td class="single-patent-bibdata">Oct 26, 2004</td></tr><tr><td class="patent-bibdata-heading">Filing date</td><td class="single-patent-bibdata">Jul 10, 2002</td></tr><tr><td class="patent-bibdata-heading">Priority date<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The priority date is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the date listed."></span></td><td class="single-patent-bibdata">Jan 13, 2000</td></tr><tr><td class="patent-bibdata-heading">Fee status<span class="patent-tooltip-anchor patent-question-icon"data-tooltip-text="The fee status is an assumption and is not a legal conclusion. Google has not performed a legal analysis and makes no representation as to the accuracy of the status or dates listed."></span></td><td class="single-patent-bibdata">Lapsed</td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Also published as</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/EP1276901A2">EP1276901A2</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20030073112">US20030073112</a>, </span><span class="patent-bibdata-value"><a href="/patents/US20050064478">US20050064478</a>, </span><span class="patent-bibdata-value"><a href="/patents/WO2001051661A2">WO2001051661A2</a>, </span><span class="patent-bibdata-value"><a href="/patents/WO2001051661A3">WO2001051661A3</a></span></span></td></tr><tr class="patent-bibdata-list-row alternate-patent-number"><td class="patent-bibdata-heading">Publication number</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value">10192786, </span><span class="patent-bibdata-value">192786, </span><span class="patent-bibdata-value">US 6808888 B2, </span><span class="patent-bibdata-value">US 6808888B2, </span><span class="patent-bibdata-value">US-B2-6808888, </span><span class="patent-bibdata-value">US6808888 B2, </span><span class="patent-bibdata-value">US6808888B2</span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Inventors</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Jing+Zhang%22">Jing Zhang</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22John+Dekker%22">John Dekker</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Antoinette+C.+van+der+Kuyl%22">Antoinette C. van der Kuyl</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Jolanda+Maas%22">Jolanda Maas</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=ininventor:%22Bob+Van+Gemen%22">Bob Van Gemen</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Original Assignee</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="http://www.google.com/search?tbo=p&tbm=pts&hl=en&q=inassignee:%22Primagen+Holding+B.V.%22">Primagen Holding B.V.</a></span></span></td></tr><tr class="patent-bibdata-list-row "><td class="patent-bibdata-heading">Export Citation</td><td><span class="patent-bibdata-value-list"><span class="patent-bibdata-value"><a href="/patents/US6808888.bibtex">BiBTeX</a>, </span><span class="patent-bibdata-value"><a href="/patents/US6808888.enw">EndNote</a>, </span><span class="patent-bibdata-value"><a href="/patents/US6808888.ris">RefMan</a></span></span></td></tr><tr class="patent-internal-links"><td colspan=2><span class="patent-bibdata-value"><a href="#backward-citations">Patent Citations</a> (12),</span> <span class="patent-bibdata-value"><a href="#npl-citations">Non-Patent Citations</a> (5),</span> <span class="patent-bibdata-value"><a href="#forward-citations">Referenced by</a> (19),</span> <span class="patent-bibdata-value"><a href="#classifications">Classifications</a> (18),</span> <span class="patent-bibdata-value"><a href="#legal-events">Legal Events</a> (7)</span> </td></tr><tr><td colspan=2 class="patent-bibdata-external-link-spacer-top"></td></tr><tr class="patent-bibdata-external-link-spacer-bottom"></tr><tr><td colspan=2><span class="patent-bibdata-heading">External Links:&nbsp;</span><span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=hRtpBAABERAJ&q=http://patft.uspto.gov/netacgi/nph-Parser%3FSect2%3DPTO1%26Sect2%3DHITOFF%26p%3D1%26u%3D/netahtml/PTO/search-bool.html%26r%3D1%26f%3DG%26l%3D50%26d%3DPALL%26RefSrch%3Dyes%26Query%3DPN/6808888&usg=AFQjCNEVwZAHxInt5Og5OxyVIHwodqghxA">USPTO</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=hRtpBAABERAJ&q=http://assignments.uspto.gov/assignments/q%3Fdb%3Dpat%26pat%3D6808888&usg=AFQjCNH-aOmonHCA44YKDbR1_TBjNJkXdg">USPTO Assignment</a>, </span><span class="patent-bibdata-value"><a href="http://www.google.com/url?id=hRtpBAABERAJ&q=http://worldwide.espacenet.com/publicationDetails/biblio%3FCC%3DUS%26NR%3D6808888B2%26KC%3DB2%26FT%3DD&usg=AFQjCNEGVH9r98EhS3-FZhJQ3xSJg7X36w">Espacenet</a></span></span></td></tr><tr class="patent-bibdata-group-spacer"></tr></table><div class="number-and-title"><span class="patent-title"><invention-title mxw-id="PT129210800" lang="EN" load-source="ifi">Amplificaton of nucleotide sequences present in sample; provide sample containing nucleotide sequnces, incubate with primers, amplify sequences, detect amplified product</invention-title></span><br><span class="patent-number">US 6808888 B2</span></div><div class="patent-section patent-abstract-section"><div class="patent-section-header"><span class="patent-section-title">Abstract</span></div><div class="patent-text"><abstract mxw-id="PA50740544" lang="EN" load-source="patent-office"> <div class="abstract">Methods for amplifying nucleic acid in a sample comprising providing the sample with a set of primers to enable synthesis of at least one nucleic acid strand complementary to at least part of the nucleic acid, wherein the set of primers comprises between 3-8 random bases, preferably clustered near the 3′ end of each primer in said set of primers. The methods of the invention are useful, for example, for determining whether samples derived from humans, mammals, poultry, or fish comprise nucleic acid of a pathogen. The methods are further suited for typing the pathogen and typing particular variants of said pathogen. The methods are also suited for the elucidation of the gene expression profile or genetic profile of cells.</div>
  </abstract></div></div><div class="patent-section patent-drawings-section"><div class="patent-section-header"><span class="patent-section-title">Images<span class="patent-section-count">(7)</span></span></div><div class="patent-drawings-body"><div class="patent-drawings-carousel"><div class="drawings"><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6808888B2/US06808888-20041026-D00000.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6808888B2/US06808888-20041026-D00000.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6808888B2/US06808888-20041026-D00001.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6808888B2/US06808888-20041026-D00001.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6808888B2/US06808888-20041026-D00002.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6808888B2/US06808888-20041026-D00002.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6808888B2/US06808888-20041026-D00003.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6808888B2/US06808888-20041026-D00003.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6808888B2/US06808888-20041026-D00004.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6808888B2/US06808888-20041026-D00004.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6808888B2/US06808888-20041026-D00005.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6808888B2/US06808888-20041026-D00005.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div><div class="patent-image"><div class="patent-thumbnail"><a href="//patentimages.storage.googleapis.com/US6808888B2/US06808888-20041026-D00006.png"><img class="patent-thumbnail-image" alt="Patent Drawing"src="//patentimages.storage.googleapis.com/thumbnails/US6808888B2/US06808888-20041026-D00006.png" /></a></div><div class="patent-thumbnail-caption">&nbsp;</div></div></div></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img" alt="Previous page"src="/googlebooks/images/kennedy/page_left.png"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img" alt="Next page"src="/googlebooks/images/kennedy/page_right.png"width="21" height="21" /></div></div></div><div class="patent-post-drawings"></div><div class="patent-section patent-claims-section"><div class="patent-section-header"><span class="patent-section-title">Claims<span class="patent-section-count">(26)</span></span></div><div class="patent-text"><div mxw-id="PCLM8753939" lang="EN" load-source="patent-office" class="claims">
    <claim-statement>What is claimed is: </claim-statement> <div class="claim"> <div num="1" id="US-6808888-B2-CLM-00001" class="claim">
      <div class="claim-text">1. A method for amplifying nucleic acids in a sample, said method comprising:</div>
      <div class="claim-text">providing said sample with a first set of primers, each member of said first set of primers comprising between 3 and 8 random bases and at least 8 essentially non-random bases; </div>
      <div class="claim-text">subjecting said sample to a first nucleic acid amplification reaction; </div>
      <div class="claim-text">providing said sample with at least one second primer comprising at least 8 bases essentially identical to said at least 8 essentially non-random bases; </div>
      <div class="claim-text">subjecting said sample to a second amplification reaction; and </div>
      <div class="claim-text">detecting amplified nucleic acid in said sample. </div>
    </div>
    </div> <div class="claim-dependent"> <div num="2" id="US-6808888-B2-CLM-00002" class="claim">
      <div class="claim-text">2. The method according to <claim-ref idref="US-6808888-B2-CLM-00001">claim 1</claim-ref>, wherein said first set of primers has between 17 and 22 essentially non-random bases.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="3" id="US-6808888-B2-CLM-00003" class="claim">
      <div class="claim-text">3. The method according to <claim-ref idref="US-6808888-B2-CLM-00001">claim 1</claim-ref>, wherein said first set of primers has between 4 and 7 random bases.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="4" id="US-6808888-B2-CLM-00004" class="claim">
      <div class="claim-text">4. The method according to <claim-ref idref="US-6808888-B2-CLM-00001">claim 1</claim-ref>, wherein said first set of primers has 5 or 6 random bases.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="5" id="US-6808888-B2-CLM-00005" class="claim">
      <div class="claim-text">5. The method according to <claim-ref idref="US-6808888-B2-CLM-00001">claim 1</claim-ref>, wherein said random bases are clustered at the 3′ end of each member of said first set of primers.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="6" id="US-6808888-B2-CLM-00006" class="claim">
      <div class="claim-text">6. The method according to <claim-ref idref="US-6808888-B2-CLM-00001">claim 1</claim-ref>, wherein each member of said first set of primers comprises a G at the 3′ end.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="7" id="US-6808888-B2-CLM-00007" class="claim">
      <div class="claim-text">7. The method according to <claim-ref idref="US-6808888-B2-CLM-00001">claim 1</claim-ref>, wherein said at least 8 essentially non-random bases comprise a sequence enabling non-nucleic acid-primed nucleic acid synthesis.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="8" id="US-6808888-B2-CLM-00008" class="claim">
      <div class="claim-text">8. The method according to <claim-ref idref="US-6808888-B2-CLM-00007">claim 7</claim-ref>, wherein said non-nucleic acid-primed nucleic acid synthesis comprises transcription.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="9" id="US-6808888-B2-CLM-00009" class="claim">
      <div class="claim-text">9. The method according to <claim-ref idref="US-6808888-B2-CLM-00001">claim 1</claim-ref>, wherein said set of primers comprises SEQ ID NO:1.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="10" id="US-6808888-B2-CLM-00010" class="claim">
      <div class="claim-text">10. The method according to <claim-ref idref="US-6808888-B2-CLM-00001">claim 1</claim-ref>, wherein said detecting amplified nucleic acid comprises determining the sequence of at least part of said amplified nucleic acid.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="11" id="US-6808888-B2-CLM-00011" class="claim">
      <div class="claim-text">11. The method according to <claim-ref idref="US-6808888-B2-CLM-00001">claim 1</claim-ref>, wherein the detection of amplified nucleic acid comprises:</div>
      <div class="claim-text">subjecting at least part of said amplified nucleic acid to a hybridization reaction with a plurality of nucleic acids; and </div>
      <div class="claim-text">detecting whether said amplified nucleic acid hybridizes with one or more of said plurality of nucleic acids. </div>
    </div>
    </div> <div class="claim-dependent"> <div num="12" id="US-6808888-B2-CLM-00012" class="claim">
      <div class="claim-text">12. The method according to <claim-ref idref="US-6808888-B2-CLM-00011">claim 11</claim-ref>, wherein said plurality of nucleic acids is present in a microarray or DNA-chip.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="13" id="US-6808888-B2-CLM-00013" class="claim">
      <div class="claim-text">13. The method according to <claim-ref idref="US-6808888-B2-CLM-00001">claim 1</claim-ref>, further comprising subjecting said sample to a third amplification reaction using at least one third primer comprising essentially non-random bases.</div>
    </div>
    </div> <div class="claim"> <div num="14" id="US-6808888-B2-CLM-00014" class="claim">
      <div class="claim-text">14. A set of oligonucleotides, said set of oligonucleotides comprising at least one of SEQ ID NO:1 and SEQ ID NO:2.</div>
    </div>
    </div> <div class="claim"> <div num="15" id="US-6808888-B2-CLM-00015" class="claim">
      <div class="claim-text">15. A method for preferentially amplifying at least part of a viral nucleic acid, said method comprising:</div>
      <div class="claim-text">providing a sample comprising said viral nucleic acid; </div>
      <div class="claim-text">providing said sample with a set of oligonucleotides comprising SEQ ID NO:1; </div>
      <div class="claim-text">subjecting said sample to a nucleic acid amplification reaction; and </div>
      <div class="claim-text">detecting amplified viral nucleic acid in said sample. </div>
    </div>
    </div> <div class="claim"> <div num="16" id="US-6808888-B2-CLM-00016" class="claim">
      <div class="claim-text">16. A nucleic acid amplification reaction of the type providing one or more primers with a nucleic acid sequence enabling further amplification and/or detection of complementary nucleic acid generated in said nucleic acid amplification reaction, wherein the improvement comprises using a set of primers, each member of said set of primers comprising between 3 and 8 random bases clustered around the 3′ end and one or more essentially constant sequences clustered at essentially the 5′ ends.</div>
    </div>
    </div> <div class="claim-dependent"> <div num="17" id="US-6808888-B2-CLM-00017" class="claim">
      <div class="claim-text">17. The nucleic acid amplification reaction of <claim-ref idref="US-6808888-B2-CLM-00016">claim 16</claim-ref>, wherein said set of primers provides an essentially constant template for detection and/or further amplification of said complementary nucleic acid.</div>
    </div>
    </div> <div class="claim"> <div num="18" id="US-6808888-B2-CLM-00018" class="claim">
      <div class="claim-text">18. A kit of parts for implementing a nucleic acid amplification reaction on a sample comprising nucleic acid, said kit of parts comprising:</div>
      <div class="claim-text">a set of primers comprising between 3 and 8 random bases, at least 8 essentially non-random bases, and at least one second primer comprising at least 8 bases essentially identical to said at least 8 essentially non-random bases. </div>
    </div>
    </div> <div class="claim-dependent"> <div num="19" id="US-6808888-B2-CLM-00019" class="claim">
      <div class="claim-text">19. The kit of parts of <claim-ref idref="US-6808888-B2-CLM-00018">claim 18</claim-ref>, further comprising:</div>
      <div class="claim-text">means for implementing a nucleic acid amplification reaction on said sample using said set of primers. </div>
    </div>
    </div> <div class="claim-dependent"> <div num="20" id="US-6808888-B2-CLM-00020" class="claim">
      <div class="claim-text">20. The kit of parts of <claim-ref idref="US-6808888-B2-CLM-00018">claim 18</claim-ref>, wherein said nucleic acid comprises nucleic acid from a microorganism.</div>
    </div>
    </div> <div class="claim"> <div num="21" id="US-6808888-B2-CLM-00021" class="claim">
      <div class="claim-text">21. A kit of parts for implementing a nucleic acid amplification reaction on a sample comprising nucleic acid, said kit of parts comprising:</div>
      <div class="claim-text">a set of primers comprising between 3 and 8 random bases, wherein said set of primers comprises SEQ ID NO:1. </div>
    </div>
    </div> <div class="claim"> <div num="22" id="US-6808888-B2-CLM-00022" class="claim">
      <div class="claim-text">22. A method for producing a positive or a negative diagnosis of pathogenic infection in a subject, comprising:</div>
      <div class="claim-text">obtaining a sample from the subject; </div>
      <div class="claim-text">providing said sample with a set of primers, each member of said set of primers comprising between 3 and 8 random bases and at least 8 essentially non-random bases; </div>
      <div class="claim-text">subjecting said sample to a first nucleic acid amplification reaction; </div>
      <div class="claim-text">providing said sample with at least one second primer comprising at least 8 bases essentially identical to said non-random bases; </div>
      <div class="claim-text">subjecting said sample to a second amplification reaction; </div>
      <div class="claim-text">detecting amplified nucleic acid in said sample, wherein the presence of amplified nucleic acid indicates said positive diagnosis and the absence of amplified nucleic acid indicates said negative diagnosis; and </div>
      <div class="claim-text">providing said positive or negative diagnosis to the subject. </div>
    </div>
    </div> <div class="claim-dependent"> <div num="23" id="US-6808888-B2-CLM-00023" class="claim">
      <div class="claim-text">23. The method according to <claim-ref idref="US-6808888-B2-CLM-00001">claim 1</claim-ref>, wherein said set of primers comprises SEQ ID NO:2.</div>
    </div>
    </div> <div class="claim"> <div num="24" id="US-6808888-B2-CLM-00024" class="claim">
      <div class="claim-text">24. A set of oligonucleotides, said set of oligonucleotides comprising SEQ ID NO:2.</div>
    </div>
    </div> <div class="claim"> <div num="25" id="US-6808888-B2-CLM-00025" class="claim">
      <div class="claim-text">25. A method for preferentially amplifying at least part of a viral nucleic acid, said method comprising:</div>
      <div class="claim-text">providing a sample comprising said viral nucleic acid; </div>
      <div class="claim-text">providing said sample with a set of oligonucleotides comprising SEQ ID NO:2; </div>
      <div class="claim-text">subjecting said sample to a nucleic acid amplification reaction; and </div>
      <div class="claim-text">detecting amplified viral nucleic acid in said sample. </div>
    </div>
    </div> <div class="claim"> <div num="26" id="US-6808888-B2-CLM-00026" class="claim">
      <div class="claim-text">26. A kit of parts for implementing a nucleic acid amplification reaction on a sample comprising nucleic acid, said kit of parts comprising:</div>
      <div class="claim-text">a set of primers comprising between 3 and 8 random bases, wherein said set of primers comprises SEQ ID NO:2.</div>
    </div>
  </div> </div></div></div><div class="patent-section patent-description-section"><div class="patent-section-header"><span class="patent-section-title">Description</span></div><div class="patent-text"><div mxw-id="PDES54324172" lang="EN" load-source="patent-office" class="description">
    <heading>CROSS REFERENCE TO RELATED APPLICATIONS</heading> <p>This application is a continuation of PCT International Application No. PCT/NL01/00020, filed Jan. 15, 2001, designating the United States of America, published in English on Jul. 19, 2001, as International Publication No. WO 01/51661, content thereof being incorporated herein by this reference.</p>
    <heading>TECHNICAL FIELD</heading> <p>The invention lies in the field of diagnostics. More in particular, the invention lies in the field of molecular diagnostics.</p>
    <heading>BACKGROUND</heading> <p>The increased knowledge of the molecular basis for disease has generated an increasing demand for more and more sophisticated diagnostic methods that can help identify the exact molecular cause of disease. In particular, for infectious diseases, clinicians want to be able to rapidly identify the pathogen. Importantly, concurrent accurate typing and discrimination of different strains of a pathogen is desired. This is important, for instance, in cases where certain strains have a particular unfavorable phenotype. It is also important, for instance, in the case where the pathogen is capable of rapid mutation of its genome to counteract selective pressures induced by the patient and/or the treatment. One non-limiting example of such a pathogen is, of course, Human Immunodeficiency Virus (“HIV”). HIV is, for instance, capable of evading selective pressure induced by nucleotide analogues through mutation of the reverse transcriptase enzyme. To be able to predict which nucleotide analogue, if any, would benefit the patient, it is desired to know in advance, i.e. before treatment starts, which genotype(s) of HIV prevail in the patient.</p>
    <p>One possibility to find the pathogen causing the disease is to harvest a sample from the patient comprising the pathogen and culturing the pathogen on suitable media in the case of a bacterial pathogen or in a suitable marker cell line for a viral and/or mycobacterium pathogen. The pathogen may be typed following and/or during culturing. This culture process can be combined with, for instance, antibiotics and/or other medicines to find the relative resistance/sensitivity of the pathogen to said medicine. This so-called culture driven testing has several advantages and is indeed routinely applied for a number of diseases.</p>
    <p>However, generally, the considerable amount of time involved with the culture process necessitates that a treatment schedule be started prior to the identification of the causative agent. This is not desired, since the treatment started may prove to be ineffective. Moreover, for many pathogens, a culture system is as yet not available. Another problem with the culture system is the inherent variability of the procedure. Not all pathogens are equally well cultured outside the body of a patient. In addition, since viability of the pathogen is essential, differences in the handling of the sample outside the body will result in variability of the result. Moreover, the costs involved in the initiation of a screen with the culture system for a wide variety of different possible causative agents in any clinical sample are considerable.</p>
    <p>For this reason there is a need for a rapid system for the typing of a pathogen that is versatile, reliable and at least partially able to discriminate between different variants of the pathogen. A number of different strategies have been tried. One such strategy relies on the detection of pathogen-derived nucleic acid in a sample. To be able to rapidly detect such nucleic acid, a nucleic acid amplification step is usually required.</p>
    <p>Many nucleic acid amplification methods have been devised that are able to specifically detect a certain pathogen and possibly even a number of different strains of said pathogen. However, such methods usually require the clinician to have at least some idea of the kind of pathogen that may cause the disease in the patient. This is frequently not the case.</p>
    <heading>BRIEF SUMMARY OF THE INVENTION</heading> <p>The present invention provides a method for detecting, quantifying and/or simultaneous typing of a variety of different nucleic acid sequences in a sample. Preferably, said nucleic acid sequences comprise nucleic acid from a microorganism and/or derivative thereof. The microorganism can be a bacterium, a phage and/or a virus.</p>
    <p>Preferably, said microorganism is a pathogenic microorganism. The present invention further allows discrimination between different strains of a microorganism or other sequences. The present invention is not only useful for the typing of a pathogen in a sample of a patient, but it is also applicable for the typing of a pathogen in a sample derived from an animal. Preferably, said animal has a commercial and/or emotional value to a human, such as a pet, a farm animal and/or an animal living in a natural reserve. The invention is also suitable for application in poultry and fish. The method of the invention is, of course, not only suited for the typing and/or detection of a pathogen. The method is generally suited for the typing and/or detection of nucleic acid in a sample. For instance, in the case of cellular DNA or RNA, the method can be used for creating a genetic expression profile, respectively, of the nucleic acid in the sample. Knowing the origin of the nucleic acid in the sample then allows the correlation of the profile with the origin.</p>
    <p>In case the origin is nucleic acid of (a specific part of) an individual, the profile, or a part thereof, can be correlated to, for instance, a database of profiles, or parts thereof, of other individuals. Matching of the profile (or part thereof) to known profiles(or parts thereof) allows the correlation of the profile (or part thereof) of the individual with the phenotypes of individuals with matching profiles(or parts thereof) displayed by these other individuals. Thus, the method of the invention can be used generally for the typing and/or detection of nucleic acid in a sample.</p>
    <p>In one aspect, the invention provides a method for amplifying nucleic acid in a sample comprising providing said sample with a set of primers comprising between 3 and 8 random bases and at least 8 essentially non-random bases, subjecting said sample to a first nucleic acid amplification reaction, providing said sample with at least one second primer comprising at least 8 bases essentially identical to said non-random bases, subjecting said sample to a second amplification reaction and detecting nucleic acid amplified in said sample. Typically only a limited amplification of nucleic acid will occur in said first nucleic acid amplification reaction. Said first and second amplification reactions are preferably performed separately, optionally including a step to remove any unused primer in said first amplification reaction. This way, the reproducibility of the method is best controlled. However, the first and the second amplification reactions may also be performed simultaneously.</p>
    <p>Nucleic acid in said sample may be DNA and/or RNA. A double stranded nucleic acid can be denatured into essentially single stranded nucleic acid prior to the priming of synthesis of a complementary strand of nucleic acid. The complementary strand may be DNA and/or RNA. Synthesis of said complementary nucleic acid is performed under conditions and using enzymes that are known in the art, such as, for instance, conditions and enzymes commonly used for polymerase chain reaction and/or NASBA.</p>
    <p>The number of nucleic acids amplified with the method of the invention is dependent on the amount and the complexity of the nucleic acid in the sample. When the complexity, i.e. the number of different sequences in the nucleic acid(s) is low, a small number of nucleic acids will be amplified with the method of the invention. In this case, some nucleic acids will be dominant in the amplificate, resulting in a banding pattern when the amplificate is run on a gel. On the contrary, when the complexity of the nucleic acid in the sample is high, many nucleic acids will be amplified, resulting in a smear when the amplificate is run on a gel. An example of nucleic acid with a particularly low complexity is nucleic acid derived from a small virus and/or plasmid (typically smaller than 10 kb). An example of nucleic acid with a particularly high complexity is cellular DNA (typically comprising &gt;10<sup>8 </sup>kb). It is clear that the mentioned examples are non-limiting. Many different complexities are possible, and additionally mixtures of low and high complexity nucleic acid can be used for the present invention. However, samples comprising only nucleic acid of low complexity that is smaller than 1 kb are not suited for the present invention. As mentioned above, the number of nucleic acids effectively amplified with the method of the invention is also dependent on the amount of nucleic acid in the sample. When the sample comprises particularly low amounts of nucleic acid, some nucleic acids will be dominant in the amplificate, resulting in a banding pattern when the amplificate is run on a gel. On the other hand, when the amount of nucleic acid in the sample is high, many nucleic acids will be amplified, resulting in a smear when the amplificate is run on a gel.</p>
    <p>The dependencies on the complexity and the amount of nucleic acid in the sample are intertwined. For example, when the complexity of the nucleic acid in the sample is high, but the amount of nucleic acid in the sample is low, some nucleic acids will be dominant in the amplificate. As will be discussed in more detail later in the description, amplificates comprising dominant nucleic acids and amplificates comprising many different nucleic acids and amplificates comprising both dominant and many different nucleic acids are useful in the present invention. However, the present invention is only useful when at least two nucleic acids are amplified. Preferably, at least 5 nucleic acids are amplified. More preferably, at least 50 nucleic acids are amplified. Typically, the method is used to amplify approximately 10,000 different nucleic acids, for instance, in samples comprising a relatively large amount of complex nucleic acid.</p>
    <p>Although the dependencies mentioned above are intertwined, the person skilled in the art will be able to determine what amount of nucleic acid is required to obtain a specific resulting amplificate.</p>
    <p>Preferably, said sample is provided with a set of primers comprising at least three or more different primers. Preferably, said set comprises at least ten different primers.</p>
    <p>A nucleotide may be an A, T, C, G, or a U and/or a functional equivalent thereof. A functional equivalent of a nucleotide is any substance capable of mimicking, at least in part, an A, T, C, G, or a U in a nucleic acid. For instance, known nucleotide analogues are suitable substances. Also, substances that can mimic a couple of nucleotides, such as, for instance, inosine are suitable substances. Preferably, said nucleotide analogues allow continued synthesis of the nascent strand.</p>
    <p>As used herein, the term “random base” means that between any two primers in said set of primers, there is at least one nucleotide or a functional equivalent thereof at a certain position that is different between the two primers.</p>
    <p>Apart from the random bases, primers in said set of primers further comprise an essentially non-random number of bases. This has the advantage that for subsequent amplification and/or detection of synthesized nucleic acid, an essentially known template is provided such that one or more new primers can be devised that can be used for the subsequent amplification and or detection of said synthesized nucleic acid. Preferably, said essentially non-random number of bases comprises between 17 and 22 nucleotides. Subsequent amplification is typically performed with at least one primer comprising a sequence essentially identical to a sequence formed by non-random bases in said at least one primer. There can, of course, be more than one second primer. Furthermore, a second primer can comprise nucleotides in addition, to the nucleotides required to create identity to a non-random sequence of a primer in the first set of primers. Additional nucleotides at the 3′ end can be advantageously used in applications wherein additional specificity is required in the amplified product. Additional nucleotides at the 5′ end can be advantageously used for the introduction of restriction enzyme sites that can be utilized to clone amplified nucleic acid. Cloning of amplified nucleic acid is often desired when amplified nucleic acid needs to be sequenced.</p>
    <p>In the present invention, the number of random bases in the set of primers has been observed to be of crucial importance to the practical application of the method of the invention, for instance, for the detection, quantification and/or typing of nucleic acid in the sample. This is especially crucial to the detection of nucleic acid from a wide variety of different pathogens. When less then 3 random bases are used in the set of primers, the subsequent amplification is not sufficiently versatile to detect a wide variety of different nucleic acids (nucleic acid with different sequences), such as from a wide variety of different microorganisms. Presumably, this is due to a lack of hybridization capability among the various nucleic acids. When more than 8 random bases are used in the set of primers, the signal detected is too specific for particular nucleic acids. When the method is used for the detection of a microorganism, such as a pathogen, this leads to the situation that nucleic acid of the microorganism present in the sample may not be detected with sufficient sensitivity. This is presumably due to the fact that not all possible combinations of 9-mers can be included in a practical way in the amount of primer that can be used in the method of the invention. Without being bound by theory, it is the observation in the present invention that for the capability to detect a wide variety of different nucleic acids, it is necessary to have in the set of primers between 3 and 8 random bases. Preferably, said set of primers comprises between 4 and 7 random bases. More preferably, said set of primers comprises 5 or 6 random bases.</p>
    <p>To increase the specificity of the reaction, said random bases are preferably clustered at the 3′ end of the primer. In the present invention, it has been observed useful for optimal yield of amplificate to include a G at the extreme 3′ end of the oligonucleotides of the set of primers. A set of primers of the invention therefore preferably comprises a G at the extreme 3′ end of at least most, and preferably all, of the oligonucleotides contained in the set of primers.</p>
    <p>In a preferred embodiment of the invention, the non-random bases in the set of primers comprise a sequence enabling non-nucleic acid-primed nucleic acid synthesis. Such a sequence may be used to obtain further amplification of complementary nucleic acid, which further amplification strengthens the signal obtained from the method of the invention. Moreover, the further amplification may be used in a method tbr determining at least part of a sequence of amplified nucleic acid such that amplified nucleic acid may be typed and/or variants of different nucleic acids, such as different variants and/or strains of a microorganism, may be determined. Preferably the non-nucleic acid-primed nucleic acid synthesis comprises transcription.</p>
    <p>In a preferred embodiment, said set of primers comprises the sequence:</p>
    <p>5′-GCT ATC ATC ACA ATG GAC NNN NNG-3′ (SEQ ID NO:1), and/or</p>
    <p>5′-AAT TCT AAT ACG ACT CAC TAT AGG GNN NNN G-3′ (SEQ ID NO:2),</p>
    <p>wherein N can be any nucleotide or functional equivalent thereof.</p>
    <p>For the detection of a wide variety of different nucleic acids, such as from different microorganisms, pathogens and/or different variants of a particular microorganism, it is essential that the amplificate of the amplification reaction be scrutinized. This can be done through detecting the amplificate with a probe specific for amplified nucleic acid, for instance, a probe specific for nucleic acid of a microorganism, such as nucleic acid from a pathogen and/or variant of said pathogen. Alternatively, the amplificate is at least in part sequenced, wherein the resolved sequence is specific for nucleic acid of said pathogen and/or variant of said pathogen.</p>
    <p>Sequencing of at least part of the amplificate is particularly favorable when the complexity of the nucleic acid in said sample is relatively small, particularly when said sample comprises essentially one type of nucleic acid, such as nucleic acid from one microorganism. However, sequencing of at least part of the amplificate is also possible when the sample comprises two, three or more types of nucleic acid. In this embodiment of the invention, however, the sample preferably does not comprise more than 5 different types of nucleic acid in a substantial amount. In one embodiment of the invention, it is possible with this method to obtain a complete, or at least nearly complete, sequence of a particular nucleic acid present in said sample. A low complexity of the nucleic acid in the sample can be obtained in various ways, for instance, in applications wherein the nucleic acid of, for instance, a microorganism, preferably a virus and/or a phage, is collected into an enriched fraction. For instance, a sample of cell free serum obtained from an HIV-infected patient will be enriched for nucleic acid of HIV viruses. Such samples, or parts thereof, may be used in a method of the present invention. A sequence of an HIV virus present in said sample can then be determined by sequencing of the amplificate obtained with the method of the invention. Furthermore, sequencing of the amplificate will also enable the typing of at least the dominant HIV variants in the sample.</p>
    <p>Alternatively, a sequence may be generated representing a gross average of the various variants of HIV in said sample. For this embodiment of the invention, a sample comprising a low complexity of nucleic acid is preferred. A low complexity of nucleic acid in the sample does not mean that said sample may not contain complex nucleic acid, such as cellular DNA. It can contain complex nucleic acid as long as the amount (by weight) of complex nucleic acid does not exceed the amount (by weight) of said low complexity nucleic acid. Preferably, the amount of complex nucleic acid does not comprise more than 25% of the nucleic acid in the sample. More preferably, the amount of complex nucleic acid does not comprise more than 10% of the nucleic acid in the sample. Of course, it is clear to the person skilled in the art that this feature of the present invention is not only useful for the sequencing and/or typing of different HIV variants, but it is also generally applicable for the typing of nucleic acid in said sample.</p>
    <p>In another embodiment of the invention, said detecting of amplified nucleic acid comprises subjecting at least part of said amplified nucleic acid to a hybridization reaction with a multiplicity of nucleic acids preferably present in a microarray and/or DNA-chip and detecting whether amplified nucleic acid hybridized with one or more nucleic acids of said multiplicity of nucleic acids. This embodiment is particularly useful when the complexity of the nucleic acid in the sample is relatively large. This embodiment is also very useful when the type of nucleic acid present in said sample is not known. Preferably, the multiplicity of nucleic acids comprise microorganism nucleic acid or nucleic acid that is a reflection of nucleic acid expressed by a cell. The cell may be any type of cell. When the multiplicity of nucleic acid comprises a reflection of nucleic acid expressed by a cell, it is preferred that the nucleic acid in said sample comprises RNA that is or was expressed by a cell. In such a case, the RNA is preferably first transcribed into DNA with, for instance, a primer capable of recognizing the poly-A tail of mRNA.</p>
    <p>The method of the invention can further comprise one or more additional amplification reactions using one or more other primers. Such an additional amplification reaction can be advantageously used to pre-amplify “certain” nucleic acid in the sample. Alternatively, an additional amplification reaction can be used to further amplify at least part of the amplificate of said first and/or second amplification reactions. In this embodiment, therefore, a method of the invention is provided, further comprising an additional nucleic acid amplification of nucleic acid in said sample using at least one primer comprising essentially non-random bases.</p>
    <p>In one embodiment, the invention provides a set of oligonucleotides comprising a sequence:</p>
    <p>5′-GCT ATC ATC ACA ATG GAC NNN NNG-3′ (SEQ ID NO:1), and/or</p>
    <p>5′-AAT TCT AAT ACG ACT CAC TAT AGG GNN NNN G-3′ (SEQ ID NO:2),</p>
    <p>wherein N can be any nucleotide or functional equivalent thereof. In these sets of oligonucleotides, N delineates the position of a random base and C, A, T and G the position of a non-random base.</p>
    <p>In another aspect, the invention provides the use of a set of oligonucleotides and/or primers of the invention for the preferred amplification of at least part of a viral nucleic acid. Preferably said set of oligonucleotides and/or primers comprises between 3 to 8 random bases clustered around the 3′ end of said oligonucleotides and/or primers and an essentially constant sequence at essentially the 5′ end of said oligonucleotides and/or primers for priming the synthesis of a complementary nucleic acid in a nucleic acid amplification method. Typically, said set of oligonucleotides and/or primers provide one or more essentially constant templates for detection and/or further amplification of said complementary nucleic acid. Preferably, said set of primers and/or oligonucleotides comprises a sequence:</p>
    <p>5′-GCT ATC ATC ACA ATG GAC NNN NNG-3′ (SEQ ID NO:1), and/or</p>
    <p>5′-AAT TCT AAT ACG ACT CAC TAT AGG GNN NNN G-3′ (SEQ ID NO:2),</p>
    <p>wherein N can be any nucleotide or functional equivalent thereof. In these sets of oligonucleotides, N delineates the position of a random base and C, A, T and G the position of a non-random base. Preferably, said set of primers provides one essentially constant template for detection and/or further amplification of said complementary nucleic acid.</p>
    <p>In yet another aspect, the invention provides the use of a set of primers comprising between 3 and 8 random bases clustered around the 3′ end and one or more essentially constant sequences clustered at essentially the 5′ end of each primer in said set of primers in a nucleic acid amplification reaction comprising nucleic acid for providing complementary nucleic acid generated with said set of primers in said amplification reaction with one or more tags enabling further amplification and/or detection of said complementary nucleic acid.</p>
    <p>In yet another aspect, the invention provides a kit for the amplification of nucleic acid in a sample comprising at least one random primer comprising between 3 and 8 random bases. Preferably, said kit comprises at least one set of oligonucleotides and/or primers of the invention. Preferably, said set of primers and/or oligonucleotides further comprise one or more essentially constant sequences clustered at essentially the 5′ end of each primer in said set of primers. Preferably, said nucleic acid in a sample comprises nucleic acid from a microorganism or a derivative thereof.</p>
    <p>In yet another aspect, the invention provides the use of a kit of the invention in a method of the invention.</p>
    <heading>BRIEF DESCRIPTION OF THE DRAWINGS</heading> <p>FIG. 1 is a schematic representation of one embodiment of the invention. The primer JZHI corresponds to SEQ ID NO:3</p>
    <p>FIG. 2 is an ethidium bromide-stained agarose gel showing the amplification products of the PCR reaction: Lanes 2-7: reactions performed with primer TAG7 (N6); Lanes 9-13: reactions performed with primer TAG7 (N5G); Lanes 14-19: reactions performed with primer TAG20; Lanes 21-26: reactions performed with primer JZH2R (SEQ ID NO:1); Lanes 8 and 20 are markers. Input concentration of 4.0×10<sup>9 </sup>MS2 RNA copies was used for reactions loaded in lanes 2, 9, 14 and 21. Input concentration of 4.0×10<sup>7 </sup>MS2 RNA copies was used for reactions loaded in lanes 3, 10, 15 and 22. Input concentration of 4.0×10<sup>5 </sup>MS2 RNA copies was used for reactions loaded in lanes 4, 11, 16 and 23. Input concentration of 4.0×10<sup>3 </sup>MS2 RNA copies was used for reactions loaded in lanes 5, 12, 17 and 24. Input concentration of 40 MS2 RNA copies was used for reactions loaded in lanes 6, 13, 18 and 25. Negative control reactions are shown in lanes 7, 19 and 26.</p>
    <p>FIG. 3 is an ethidium bromide-stained agarose gel (left half) and HBV DIG-labeled probe hybridized blot (right half) of the 10 independent PCR reactions per second strand synthesis in this example. Panel A: non-treated serum; Panel B: DNase I-treated serum; Panel C: RNase A-treated serum. Lanes marked with M are marker lanes.</p>
    <p>FIG. 4 is a schematic presentation of the sequences obtained in this example (the arrows) on the HBV genome sequence (the red filled bar). The numbers in each arrow represent the lane number on the sequence gel (left of the slash) and the number of the clone (right of the slash).</p>
    <p>FIG. 5 is an ethidium bromide-stained agarose gel of the amplification results using a dilution of the second strand cDNA synthesis as input for the PCR amplification. Lanes 1-2 is 10-fold dilution, lanes 3-4 is 100-fold dilution, lanes 5-6 is 1,000-fold dilution lanes 7-8 is 10,000-fold dilution, and lanes 9-10 is 100,000-fold dilution.</p>
    <p>FIG. 6 depicts a conventional microarray and/or DNA-chip.</p>
    <heading>DETAILED DESCRIPTION OF THE INVENTION</heading> <p>Nucleotide sequence analysis has become an important tool in modern molecular biology. Recent technological advancements have enabled high throughput sequencing protocols that generate multiple sequences of 200-800 nucleotides in length. On the most modern systems, 96 of such sequences can be determined simultaneously.</p>
    <p>In order to make optimal use of the high throughput capabilities of the modern sequencing methods, a good strategy is important. In particular, sequence analysis of long stretches of nucleotides (&gt;2 kb) and small sample sizes (or a combination of both) is a challenge. In one embodiment, the present invention discloses a method that enables nonspecific amplification of nucleic acid in a sample and relative simple sequence analysis of long stretches of nucleotides.</p>
    <p>The method consists of a number of steps that can be described as follows:</p>
    <p>1) First strand synthesis. The sample nucleic acid is used to copy in a first strand synthesis (i.e. cDNA synthesis) using reverse transcriptase (if RNA or DNA is the nucleic acid being sequenced) or DNA-dependent DNA polymerase (if DNA is the nucleic acid being sequenced), see FIG. <b>1</b>. The oligos that are used to prime the first strand synthesis consist of a specific sequence at the 5′ part of the oligo and a random sequence at the 3′ part of the oligo. An example of such an oligo is JZH2R, which has the following sequence: 5′-GCT ATC ATC ACA ATG GAC NNN NNG-3′ (SEQ ID NO:1). Typical of JZH2R (SEQ ID NO:1) is that the oligo has a G residue at its 3′ end flanking the random sequence. The length of the cDNA will vary depending on the exact location where the random part of the oligo was hybridized and the length of the elongation by the polymerase.</p>
    <p>2) Stand separation or degradation. Following the first strand synthesis, the newly made DNA strand is separated from its template strand by denaturation or by degradation of the template strand with RNase H in case the template strand was RNA, see FIG. <b>1</b>.</p>
    <p>3) Second strand synthesis. Using oligo JZH2R (SEQ ID NO:1) in combination with a DNA-dependent DNA polymerase, a second DNA strand is synthesized complementary to the first strand synthesized (i.e. the cDNA), see FIG. <b>1</b>.</p>
    <p>4) PCR amplification. The newly formed double stranded DNA molecules of variable length are amplified by PCR using an oligo primer that fits the specific part of the oligo that was used in the first and second strand protocol described above. In the case of JZH2R (SEQ ID NO:1), that oligo primer would be 5′-GCT ATC ATC ACA ATG GAC-3′ (SEQ ID NO:3), which is named JZH1, see FIG. <b>1</b>.</p>
    <p>5) Cloning and sequencing. The PCR fragments that display a wide variety in length are cloned into a plasmid and used for transformation of <i>E. coli </i>cells using standard protocols known by persons skilled in the art. After growing individual colonies overnight, the plasmids of the individual colonies are isolated and used for sequence analysis of both strands of the DNA, obtaining a sequence of 400-1000 nucleotides per clone.</p>
    <p>In addition, the above mentioned method can be used for non-biased amplification of all RNAs or DNA from a sample from a small amount to amounts sufficient for analysis of labeling.</p>
    <p>With the incorporation of an RNA polymerase promoter sequence instead of the specific sequence part of the primers, the method can be coupled to a transcription reaction. An example of such a primer would be: 5′-AAT TCT AAT ACG ACT CAC TAT AGG GNN NNN G-3′ (SEQ ID NO:2) containing a T7 RNA polymerase promoter sequence. Following the PCR step (see flow chart in FIG. 1) a transcription reaction can be performed using T7 RNA polymerase translating the PCR fragments into RNA. Approximately 100-1000 copies of RNA are made per DNA template in a transcription reaction with T7 RNA polymerase. During the transcription reaction, the RNA may be labeled, enabling subsequent analysis of the RNA, for instance, via hybridization on a microarray or DNA-chip.</p>
    <p>In another embodiment, the method can be used for the non-biased amplification of mRNA. In that case, the first strand cDNA synthesis will be performed with an oligo that consists of a poly-T stretch (10 to 30 T residues) at the 3′ end and a specific sequence at the 5′ end (e.g., 5′-GCT ATC ATC ACA ATG GAC T<sub>(10-30)</sub>-3′ (SEQ ID NO:4)). For the second strand synthesis, an oligo with a random sequence can be used, e.g. JZH2R (5′-GCT ATC ATC ACA ATG GAC NNN NNG-3′ (SEQ ID NO:1)). Following the second strand synthesis, the double stranded DNA can be amplified with a PCR or other nucleic acid amplification reaction (e.g., NASBA, TMA, rolling circle amplification).</p>
    <p>In another embodiment, the method used for the non-biased amplification of mRNA described above may be coupled to a transcription reaction by incorporation of an RNA polymerase promoter sequence in the primer used for the first strand synthesis. An example of such a primer would be: 5′-AAT TCT AAT ACG ACT CAC TAT AGG GT<sub>(10-30)</sub>-3′ (SEQ ID NO:5). Following the second strand synthesis, the double stranded DNA can be used as a template in a PCR amplification, and the amplified PCR fragments can be used as templates in a transcription reaction with T7 RNA polymerase in case the primer described above is used that contains the T7 RNA polymerase promoter sequence. Following the second strand synthesis, the double stranded DNA can also be directly used as a template in a transcription reaction with T7 RNA polymerase in case the primer described above is used that contains the T7 RNA polymerase promoter sequence.</p>
    <p>During the transcription reactions, the RNA that is made can be labeled with a detectable moiety, e.g., a radioactive or fluorescent label. Subsequently, the RNA can be used for analysis on microarrays or DNA-chips, for instance, to elucidate the expression profile of the cells in the sample that the RNA was isolated from.</p>
    <heading>EXAMPLES</heading> <p>Materials and Ingredients</p>
    <p>All basic ingredients for buffers TRIS™, MgCl<sub>2</sub>, KCl. etc.) were purchased from Merck Nederland BV. Postbox 8198,1005 AD Amsterdam, The Netherlands, or Sigma-Aldrich Chemie BV, Stationsplein. Postbox 27,3330 AA Zwijndrecht, The Netherlands. RNasin, MMLV reverse transcriptase and AMPLITAO™ DNA polymerase were purchased from PE Applied Biosystems, Benelux, Hoogeveenenweg 100, Postbox 305,2910 AH Nieuwerkerk a/d Ussel, The Netherlands. RNase H was purchased from Roche Diagnostics Nederland BV, Post Box 1007,1300 BA Almere. The Netherlands. SEQUENASE™ DNA polymerase and dNTPs were purchased from Amersham Pharmacia Biotech, 800 Centennial Avenue. P0 Box 1327, Piscataway, N.J. 08855, USA. The TOPO-TA™ cloning kit (containing the pCR2.1-TOPO plasmid vector) was purchased from Invitrogen BV, Dc Schelp 12,9351 NV Leek, The Netherlands. Oligonucleotides were purchased from different oligonucleotide suppliers and were usually purified by the supplier and tested for functionality in a PCR reaction with a known amount of input. Sequence analysis was-preformed using the ABI Prism kits purchased from PE Applied Biosystems, Benelux, Hoogeveenenweg 100, Postbox 305,2910 AH Nieuwerkerk a/d lissel, The Netherlands.</p>
    <p>Input for a method of the invention is purified nucleic acid that can be isolated from (clinical) samples by a wide variety of methods. One of the methods that is very suitable for this purpose is the method described in Boom R, Sol C J, Salimans M M, Jansen C L, Wertheim-van Dillen P M, van der Noordaa J (1990), “Rapid and simple method for purification of nucleic acids,” <i>J. Clin. Microbiology </i>28(3): 495-503.</p>
    <heading>Example 1</heading> <p>In this example, we compared different primers to amplify MS2-phage RNA. Four different primers were tested (see Table 1). After isolation and purification, 4.0×10<sup>9</sup>, 4.0×10<sup>7</sup>, 4.0×10<sup>5</sup>, 4.0×10<sup>3 </sup>and 4.0×10 copies of MS-2 phage RNA were used as input to do this experiment.</p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" tabstyle="monospace" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="217pt" align="center"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" rowsep="1" class="description-td" colspan="1">TABLE 1</td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">Primers used in this example</td>
              </tr> </tbody> </tgroup> <tgroup align="left" colsep="0" rowsep="0" cols="3"> <colspec colname="offset" colwidth="14pt" align="left"> </colspec> <colspec colname="1" colwidth="35pt" align="left"> </colspec> <colspec colname="2" colwidth="168pt" align="left"> </colspec> <tbody valign="top"> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">Name</td>
                <td class="description-td">Sequence</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="2" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">JZH2R</td>
                <td class="description-td">5′-GCT ATC ATC ACA ATG GAC NNN NNG-3′</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">(SEQ ID NO:1)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td">JZH1</td>
                <td class="description-td">5′-GCT ATC ATC ACA ATG GAC-3′</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td class="description-td"> </td>
                <td class="description-td">(SEQ ID NO:3)</td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
                <td namest="offset" nameend="2" align="center" rowsep="1" class="description-td" colspan="3"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    <p>The experiment was performed using 10 μl of MS2 phage RNA at the amounts mentioned above. The RNA was heated 5 minutes to 80° C. and subsequently cooled on ice to denature any double stranded nucleic acid and structures in the RNA. Subsequently, 10 μl of mixture A (100 mM KCl, 20 mM Tris, pH=8.3, 10 mM MgCl<sub>2</sub>, 2 μM dATP, 2 μM dTTP, 2 μM dCTP, 2 μM dGTP, 50 ng primer (see Table 1), 0.5 μl RNasin and 25 units MMLV reverse transcriptase) was added to the RNA and incubated for 10 minutes at ambient temperature, followed by an incubation at 42° C. for 30 minutes. Subsequently, the reaction was incubated at 80° C. for 5 minutes and cooled down to ambient temperature, followed by the addition of 0.5 μl RNase H (0.5 units) and further incubation at 37° C. for 30 minutes. After this incubation, the reactions are placed on ice to stop the reaction of RNase H.</p>
    <p>The second strand synthesis was performed by using 20 μl of the first strand synthesis (on ice) and adding 20 μl of mixture B (70 mM Tris, pH=7.5, 50 mM NaCl, 35 mM MgCl<sub>2</sub>, 2 μM dATP, 2 μM dTTP, 2 μM dCTP, 2 μM dGTP, 100 ng primer with random sequence (see Table 1), and 2.6 units Sequenase DNA polymerase) and incubating on ice for 10 minutes, subsequently incubating at ambient temperature for 10 minutes, followed by incubation at 37° C. for 30 minutes. After the reaction, the tubes are placed on ice and 2 μl is used for subsequent PCR amplification.</p>
    <p>The PCR reactions were performed by adding to the 2 μl second strand synthesis reactions 48 μl PCR mix (50 mM Tris, pH=8.3, 20 mM KCl, 0.1 mg/ml BSA, 1.8 mM MgCl<sub>2</sub>, 0.1 μM dATP, 0.1 μM dTTP, 0.1 μM dCTP, 0.1 μM dGTP, 100 ng PCR primer (see Table 1)) and incubating at 95° C. for 5 minutes, followed by 45 cycles of 20 seconds at 95° C., 30 seconds at 55° C., and 2 minutes 72° C. Following the last cycle, the reactions are incubated for 10 minutes at 72° C., subsequently 10 minutes at 40° C., and stored at 4° C. or −20° C. until further use.</p>
    <p>In the PCR reactions, primer JZH1 (SEQ ID NO:3) was used for PCR amplification of reactions that were made with primer JZH2R (SEQ ID NO:1). In every experiment H<sub>2</sub>O was used as a negative control.</p>
    <p>Of each PCR reaction, 15 μl was run on an agarose gel and stained with ethidium bromide (standard protocols known to persons skilled in the art), the results of which are depicted in FIG. <b>2</b>.</p>
    <p>From the data in FIG. 2 it is clear that the JZH2R (SEQ ID NO:1) gives the best results with products visible on the agarose gel after PCR amplification when the method was started with only 4.0×10<sup>5 </sup>copies of MS2 phage RNA. Furthermore this example shows the applicability of the method to analyze RNA sequences.</p>
    <heading>Example 2</heading> <p>In this example we show the ability of a method of the invention to analyze DNA sequences. A Hepatitis B virus (HBV)-positive serum was used as input to do this experiment. Nucleic acid isolated and purified from the serum were subjected to DNase I and RNase A treatment, respectively, to show the difference in analysis when either the DNA or RNA had been degraded. As a control, a non-treated serum sample was analyzed. In addition to direct analysis of a serum sample, the same set of experiments were done on the supernatants of a serum sample that was briefly centrifuged (10 minutes at 3,000 g) to remove any cellular debris in the serum sample.</p>
    <p>The method used in this example was identical to the method used in Example 1, with the following minor changes: (1) no RNasin was used for the analysis of DNA samples; (2) in the case of a DNA template, a DNA-dependent DNA polymerase may also be used instead of the reverse transcriptase for the first strand synthesis; in this particular experiment, reverse transcriptase was used; and (3) the second strand synthesis was amplified 10 times in 10 independent PCR reactions.</p>
    <p>The results are depicted in FIG. <b>3</b>. The PCR fragments (see flowchart in FIG. 1) were analyzed on an agarose gel and visualized with ethidium, bromide. Subsequently, the bands from the gel were transferred to a filter with a standard blotting procedure known to persons skilled in the art. The bands on the filter were interrogated (i.e. hybridized) with a specific digoxygenin (DIG)-labeled probe covering the whole HBV genome of 3 kb (see FIG. <b>3</b>).</p>
    <p>The results clearly show that without any treatment of the serum sample (Panel A in FIG. 3) the most bands in the 10 independent PCR reactions are observed. DNase treatment decreases the number of bands to virtually zero (Panel B in FIG. <b>3</b>), while RNase treatment still enables the amplification of some bands (Panel C in FIG. <b>3</b>). The DNase I-treated nucleic acids contained no HBV positive bands (Panel 3, the right half of FIG. <b>3</b>), while the RNase A-treated nucleic acids still contained HBV positive bands confirmed by hybridization with DIG-labeled HBV probe bands (Panel C, the right half of FIG. <b>3</b>). This result demonstrates that this method can randomly amplify not only RNA, as in Example 1, but also DNA.</p>
    <heading>Example 3</heading> <p>In this example, the serum of a patient suspected of infection with HIV-1 (antibody positive for HIV-1), HHV8, HBV and HGV was analyzed with a method of the invention. Nucleic acid was isolated and purified from 100 μl serum of this patient, and 10 μl and 30 μl of nucleic acid solution (total 100 μl) was used for two independent experiments (the protocol as described in Example 1 was applied). Per second strand synthesis (see flow chart in FIG. <b>1</b>), 10 PCR reactions were performed. The PCR fragments were cloned in the TOPO-TA plasmid vector according to the manufacturer's instructions (Invitrogen BV, De Schelp 12, 9351 NV Leek, The Netherlands). After transformation of <i>E. coli </i>cells, a total of 198 different inserts in the plasmid were sequenced, and of the 198 sequences, 2 sequences were from HBV and 2 sequences were from HGV. The remaining 194 sequences were all of human origin or unknown (i.e. no homology found in the GenBank and EMBL nucleic acid databases). These results show that the method of the invention is capable of simultaneously analyzing both DNA (HBV genome) and RNA (HGV genome) sequences. The probable reason for not finding any HIV-1 or HHV8 sequences in the analysis in this example is most likely the low number of copies of the HIV-1 and HHV8 viruses in the sample used for the analysis.</p>
    <heading>Example 4</heading> <p>The same serum sample as used in Example 3 was used in this example. We used 0.5 ml of serum to load on a 10 ml continuous sucrose gradient (10% to 60% sucrose w/v), which was centrifuged for 18 hours at 30,000 rpm at 4° C. in a Beckman SW41Ti swing-out rotor in a Beckman ultracentrifuge. After the centrifugation, 0.25 ml fractions were collected from the bottom of the tube. The density of the sucrose solution in each fraction was determined using a refractometer, and each fraction was tested for the presence of HBV virus particles by an HBV-specific PCR reaction detection HBV DNA. The HBV DNA peak was found in fractions with a density of 1.186 g/ml to 1.205 g/ml. This peak fraction was pooled and nucleic acid isolated and purified. The nucleic acid was analyzed using the method as described in Example 1. After the PCR reactions, 14 different discrete bands were cloned in the TOPO-TA plasmid vector according to the manufacturer's instructions (Invitrogen BV, De Schelp 12,9351 NV Leek, The Netherlands) and sequenced. All 14 inserts in the plasmid were HBV sequences, and together they covered 2.8 kb of the HBV genome that is in total 3.2 kb.</p>
    <p>A schematic presentation of the location of part of the sequences on the HBV genome is shown in FIG. <b>4</b>. The data clearly show the applicability of the methods of the invention to obtain the nucleotide sequence of a long genome in a homogenous solution. The same method can, of course, be applied to the sequence analysis of long inserts in cloning systems like plasmids, phage lambda or yeast systems. The methods of the invention are very suitable, for instance, to determine the nucleotide sequences of inserts larger than 10 kb in phage lambda.</p>
    <heading>Example 5</heading> <p>The experiment in this example was performed with the protocol as described in Example 1. The input material for the first strand synthesis was nucleic acid isolated from the culture supernatant of an HIV-1 culture. The primer that was used for the first strand synthesis was primer JZH2R (5′-GCT ATC ATC ACA ATG GAC NNN NNG-3′ (SEQ ID NO:1)). After the second strand synthesis (see Example 1), the products were diluted in water and the serial dilutions used as input for the PCR amplification reaction. The PCR reaction (see Example 1) was performed with primer JZH1 (5′-GCT ATC ATC ACA ATG GAC-3′ (SEQ ID NO:3)). Dilutions of 10, 100, 1,000, 10,000 and 100,000 times of the second strand synthesis were used for amplification.</p>
    <p>The results are shown in FIG. <b>5</b>. The results clearly show that input with a high complexity (i.e. many different sequences) will result in a smear of products after the amplification, illustrating the non-biased nature of the amplification. The complexity of sequences consists of cellular and viral sequences in the culture supernatant that is the result of cell lysis and non-adherent cells in the culture.</p>
    <p>When the complexity of the input sequences is decreased by dilution, the chances for a particular sequence to be part of the pool decreases, and not all sequences are represented in the input material. Furthermore, the decrease in absolute copy number of the target sequences also allows only a few amplicons per target sequence to be made. Both of these phenomena result in only a few bands per amplification (lanes 7-10). After cloning, these bands are suited for sequence analysis, for instance, of the HIV-1 genome. If the bands also contain human cellular sequences, it may be necessary to purify the viral particles before application of the GAT procedure if the goal is sequence analysis of the HIV-1 genome. Such purification can be achieved by spinning down of cells and removal of background nucleic acid by DNase and RNase treatment prior to nucleic acid isolation. Dilution of the nucleic acid after second strand synthesis would then cause a few (10-100) bands to be the result of the amplification. These bands can then be sequenced, showing the HIV-1 genomic sequence.</p>
    <heading>Example 6</heading> <p>First strand synthesis is performed as described in Example 1 using total poly-A mRNA isolated from cells as the input for the first strand synthesis (1-5 mg). The primer used for the first strand synthesis is GAT01 (5′-AAT TCT AAT ACG ACT CAC TAT AGG GAG AGA AGG ATA CCA CTA GCT AGC GTT TTT TTT TTT TTT TTT TTT TTT T-3′ (SEQ ID NO:6), the T7 promoter sequence is shown in italics). The second strand synthesis is performed as described in Example 1 using primer JZH2R (5′-GCT ATC ATC ACA ATG GAC NNN NNG-3′ (SEQ ID NO:1)).</p>
    <p>The second-strand synthesis (or part thereof) is used as input for amplification in a NASBA reaction. The NASBA reaction (Tris-HCl 40 mM, pH=8.5, MgCl<sub>2 </sub>12 mM, KCl 70 mM, DTT 5 mM, dNTPs (each) 1 mM, rATP 2 mM, rUTP 2 mM, rCTP 2 mM, rGTP 1.5 mM, ITP 0.5 mM, EDTA 0.75 mM, DMSO 15% v/v, oligonucleotide P1 (GAT02: 5′-AAT TCT AAT ACG ACT CAC TAT AGG GAG AGA AGG ATA CCA CTA GCT AGC GT-3′ (SEQ ID NO:7)) 0.2 μM, oligonucleotide P2 (JZHI: 5′-GCT ATC ATC ACA ATG GAC-3′ (SEQ ID NO:3)) 0.2 μM, and Sorbitol 0.375 M) was incubated at 65° C. for 5 minutes and subsequently at 41° C. for 5 minutes. Then the enzyme mix was added (BSA 2.1 mg, RNase H 0.01 units, T7 RNA Polymerase 37 units, AMV-RT 7.5 units) and after gentle mixing by tapping the reactions were incubated at 41° C. in a water bath for 90 minutes.</p>
    <p>The analysis of the amplification on an ethidium bromide-stained agarose gel shows smears, indicating the non-biased amplification of all poly-A mRNAs present in the input sample.</p>
    <heading>Example 7</heading> <p>The experiment in this example was performed with the protocol as described in Example 1. The input material for the first strand synthesis was poly-A+mRNA provided as a control by CLONTECH array systems. The amount of poly-A+mRNA used in subsequent reactions with diluted nucleic acid was equivalent to the amount of poly-A+mRNA present in 10,000, 1000, 100. 10 or 1 cell(s). First and second strand syntheses were performed as described in Example 1. The complete cDNA product was subsequently used as input for the PCR amplification reaction. The PCR reaction (seer Example 1) was performed with primer JZHI (5′GCT ATC ATC ACA ATO GAC-3′(SEQ ID NO:3)), and after the amplification step, the amplification products were purified using a NUCLEOSPIN™ column, supplied with arrays (CLONTECH Inc., 1020 East Meadow Circle, Palo Alto, Calif. 94303, USA, Subsequently, the amplification products were radiolabeled with alfa-32P-dATP in a primer extension reaction with random hexamers or a collection of specific oligonucleotides supplied with the ATLAS array's (CLONTECH Inc.</p>
    <p>The labeled products of the amplification were hybridized onto an ATLAS mouse array filter containing probes for approximately 600 genes of the mouse genome (CLONTECH Inc., 1020 East Meadow Circle, Palo Alto, Calif. 94303, USA, www.clontech.com). Hybridizations were performed according to the protocol CLONTECH supplies with the ATLAS arrays.</p>
    <p>Results</p>
    <p>All dilutions of the poly-A+ mRNA gave smears on gels, indicating the generic, non-biased amplification of all mRNAs in the sample. After hybridization of the amplified products generated with a poly-A+ mRNA input equivalent to 1000 cells, approximately 70 genes lighted up on the autoradiograph with some clear differences in the expression level of the genes. This result clearly shows the suitability of the GAT method described in this invention for preparing mRNA for analysis on (micro) arrays.</p>
    <p>
      <tables> <table frame="none" colsep="0" rowsep="0" pgwide="1" tabstyle="monospace" class="description-table"> <tgroup align="left" colsep="0" rowsep="0" cols="1"> <colspec colname="1" colwidth="441pt" align="left"> </colspec> <thead> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> </thead> <tbody valign="top"> <tr class="description-tr"> <td class="description-td">                  </td>
              </tr> <tr class="description-tr"> <td class="description-td">#             SEQUENCE LISTING </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;160&gt; NUMBER OF SEQ ID NOS: 7 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;210&gt; SEQ ID NO 1 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;211&gt; LENGTH: 24 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;212&gt; TYPE: DNA </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;213&gt; ORGANISM: Artificial Sequence </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;220&gt; FEATURE: </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;223&gt; OTHER INFORMATION: Primer JZH2R </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;220&gt; FEATURE: </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;221&gt; NAME/KEY: misc_feature </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;222&gt; LOCATION: (19)..(23) </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;223&gt; OTHER INFORMATION: n can be any nucleotid </td>
              </tr> <tr class="description-tr"> <td class="description-td">#e or functional </td>
              </tr> <tr class="description-tr"> <td class="description-td">      equivalent thereof </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;400&gt; SEQUENCE: 1 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">gctatcatca caatggacnn nnng          </td>
              </tr> <tr class="description-tr"> <td class="description-td">#                  </td>
              </tr> <tr class="description-tr"> <td class="description-td">#                24 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;210&gt; SEQ ID NO 2 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;211&gt; LENGTH: 31 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;212&gt; TYPE: DNA </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;213&gt; ORGANISM: Artificial Sequence </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;220&gt; FEATURE: </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;223&gt; OTHER INFORMATION: Primer </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;220&gt; FEATURE: </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;221&gt; NAME/KEY: misc_feature </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;222&gt; LOCATION: (26)..(30) </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;223&gt; OTHER INFORMATION: n can be any nucleotid </td>
              </tr> <tr class="description-tr"> <td class="description-td">#e or functional </td>
              </tr> <tr class="description-tr"> <td class="description-td">      equivalent thereof </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;220&gt; FEATURE: </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;221&gt; NAME/KEY: promoter </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;222&gt; LOCATION: (1)..(25) </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;223&gt; OTHER INFORMATION: T7 promoter sequence </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;400&gt; SEQUENCE: 2 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">aattctaata cgactcacta tagggnnnnn g        </td>
              </tr> <tr class="description-tr"> <td class="description-td">#                  </td>
              </tr> <tr class="description-tr"> <td class="description-td">#          31 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;210&gt; SEQ ID NO 3 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;211&gt; LENGTH: 18 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;212&gt; TYPE: DNA </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;213&gt; ORGANISM: Artificial Sequence </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;220&gt; FEATURE: </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;223&gt; OTHER INFORMATION: Primer JZH1 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;400&gt; SEQUENCE: 3 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">gctatcatca caatggac             </td>
              </tr> <tr class="description-tr"> <td class="description-td">#                  </td>
              </tr> <tr class="description-tr"> <td class="description-td">#                  </td>
              </tr> <tr class="description-tr"> <td class="description-td">#  18 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;210&gt; SEQ ID NO 4 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;211&gt; LENGTH: 19 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;212&gt; TYPE: DNA </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;213&gt; ORGANISM: Artificial Sequence </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;220&gt; FEATURE: </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;223&gt; OTHER INFORMATION: Primer </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;220&gt; FEATURE: </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;221&gt; NAME/KEY: misc_feature </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;222&gt; LOCATION: (19)..(19) </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;223&gt; OTHER INFORMATION: n represents 10-30 T r </td>
              </tr> <tr class="description-tr"> <td class="description-td">#esidues </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;400&gt; SEQUENCE: 4 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">gctatcatca caatggacn             </td>
              </tr> <tr class="description-tr"> <td class="description-td">#                  </td>
              </tr> <tr class="description-tr"> <td class="description-td">#                  </td>
              </tr> <tr class="description-tr"> <td class="description-td"># 19 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;210&gt; SEQ ID NO 5 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;211&gt; LENGTH: 26 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;212&gt; TYPE: DNA </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;213&gt; ORGANISM: Artificial Sequence </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;220&gt; FEATURE: </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;223&gt; OTHER INFORMATION: Primer </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;220&gt; FEATURE: </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;221&gt; NAME/KEY: misc_feature </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;222&gt; LOCATION: (26)..(26) </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;223&gt; OTHER INFORMATION: n represents 10-30 T r </td>
              </tr> <tr class="description-tr"> <td class="description-td">#esidues </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;220&gt; FEATURE: </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;221&gt; NAME/KEY: promoter </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;222&gt; LOCATION: (1)..(25) </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;223&gt; OTHER INFORMATION: T7 promoter sequence </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;400&gt; SEQUENCE: 5 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">aattctaata cgactcacta tagggn          </td>
              </tr> <tr class="description-tr"> <td class="description-td">#                  </td>
              </tr> <tr class="description-tr"> <td class="description-td">#              26 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;210&gt; SEQ ID NO 6 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;211&gt; LENGTH: 73 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;212&gt; TYPE: DNA </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;213&gt; ORGANISM: Artificial Sequence </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;220&gt; FEATURE: </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;223&gt; OTHER INFORMATION: Primer GAT01 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;220&gt; FEATURE: </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;221&gt; NAME/KEY: promoter </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;222&gt; LOCATION: (1)..(25) </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;223&gt; OTHER INFORMATION: T7 promoter sequence </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;400&gt; SEQUENCE: 6 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">aattctaata cgactcacta tagggagaga aggataccac tagctagcgt tt </td>
              </tr> <tr class="description-tr"> <td class="description-td">#tttttttt     60 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">tttttttttt ttt              </td>
              </tr> <tr class="description-tr"> <td class="description-td">#                  </td>
              </tr> <tr class="description-tr"> <td class="description-td">#                  </td>
              </tr> <tr class="description-tr"> <td class="description-td">#      73 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;210&gt; SEQ ID NO 7 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;211&gt; LENGTH: 50 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;212&gt; TYPE: DNA </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;213&gt; ORGANISM: Artificial Sequence </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;220&gt; FEATURE: </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;223&gt; OTHER INFORMATION: Primer GAT02 </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;220&gt; FEATURE: </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;221&gt; NAME/KEY: promoter </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;222&gt; LOCATION: (1)..(25) </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;223&gt; OTHER INFORMATION: T7 promoter sequence </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">&lt;400&gt; SEQUENCE: 7 </td>
              </tr> <tr class="description-tr"> <td class="description-td"> </td>
              </tr> <tr class="description-tr"> <td class="description-td">aattctaata cgactcacta tagggagaga aggataccac tagctagcgt  </td>
              </tr> <tr class="description-tr"> <td class="description-td">#              50</td>
              </tr> <tr class="description-tr"> <td namest="1" nameend="1" align="center" rowsep="1" class="description-td" colspan="1"> </td>
              </tr> </tbody> </tgroup> </table> </tables> </p>
    </div></div></div><div class="patent-section patent-tabular-section"><a id="backward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Cited Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5104792">US5104792</a></td><td class="patent-data-table-td patent-date-value">Dec 21, 1989</td><td class="patent-data-table-td patent-date-value">Apr 14, 1992</td><td class="patent-data-table-td ">The United States Of America As Represented By The Department Of Health And Human Services</td><td class="patent-data-table-td ">Using a primer</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5106727">US5106727</a></td><td class="patent-data-table-td patent-date-value">Jul 13, 1990</td><td class="patent-data-table-td patent-date-value">Apr 21, 1992</td><td class="patent-data-table-td ">Life Technologies, Inc.</td><td class="patent-data-table-td ">Priming with random oligonuceotides, incubating with inducing and displacement agents and nucleoside triphosphates substrates</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US5948649">US5948649</a></td><td class="patent-data-table-td patent-date-value">Nov 13, 1995</td><td class="patent-data-table-td patent-date-value">Sep 7, 1999</td><td class="patent-data-table-td ">Biotec Laboratories Limited</td><td class="patent-data-table-td ">Method for identifying nucleic acid sequences from different biological sources through amplification of the same</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US6521428">US6521428</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Nov 4, 1999</td><td class="patent-data-table-td patent-date-value">Feb 18, 2003</td><td class="patent-data-table-td ">Genome Technologies, Llc</td><td class="patent-data-table-td ">Shot-gun sequencing and amplification without cloning</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20010021518">US20010021518</a></td><td class="patent-data-table-td patent-date-value">Jan 12, 2001</td><td class="patent-data-table-td patent-date-value">Sep 13, 2001</td><td class="patent-data-table-td ">Jaap Goudsmit</td><td class="patent-data-table-td ">Contacting a mixture of single stranded mRNA and double stranded nucleic acid material with a liquid comprising a chaotropic agent and a nucleic acid binding solid phase, solid phase binds double-stranded nucleic acid, not single</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US20030022194">US20030022194</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Oct 25, 2001</td><td class="patent-data-table-td patent-date-value">Jan 30, 2003</td><td class="patent-data-table-td ">Erlander Mark G.</td><td class="patent-data-table-td ">Amplification of preferential nucleotide sequences; obtain nucleotide sequences, denature, incubate with primer, transcribe, recover and amplify nucleotide sequences</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/EP0466520A1?cl=en">EP0466520A1</a></td><td class="patent-data-table-td patent-date-value">Jul 15, 1991</td><td class="patent-data-table-td patent-date-value">Jan 15, 1992</td><td class="patent-data-table-td ">Life Technologies Inc.</td><td class="patent-data-table-td ">Amplification of nucleic acid sequences using oligonucleotides of random sequence as primers</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1996015264A2?cl=en">WO1996015264A2</a></td><td class="patent-data-table-td patent-date-value">Nov 13, 1995</td><td class="patent-data-table-td patent-date-value">May 23, 1996</td><td class="patent-data-table-td ">Kevin Patrick Francis</td><td class="patent-data-table-td ">Method for identifying nucleic acid sequences from different biological sources through amplification of the same</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1997030062A1?cl=en">WO1997030062A1</a></td><td class="patent-data-table-td patent-date-value">Feb 14, 1997</td><td class="patent-data-table-td patent-date-value">Aug 21, 1997</td><td class="patent-data-table-td ">Amsterdam Support Diagnostics</td><td class="patent-data-table-td ">Isolation and amplification of nucleic acid materials</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO1998002575A1?cl=en">WO1998002575A1</a></td><td class="patent-data-table-td patent-date-value">Jul 16, 1996</td><td class="patent-data-table-td patent-date-value">Jan 22, 1998</td><td class="patent-data-table-td ">Senapathy Periannan</td><td class="patent-data-table-td ">Method for contiguous genome sequencing</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2001051661A2?cl=en">WO2001051661A2</a></td><td class="patent-data-table-td patent-date-value">Jan 15, 2001</td><td class="patent-data-table-td patent-date-value">Jul 19, 2001</td><td class="patent-data-table-td ">Amsterdam Support Diagnostics</td><td class="patent-data-table-td ">A universal nucleic acid amplification system for nucleic acids in a sample</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/WO2002020571A2?cl=en">WO2002020571A2</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Sep 5, 2001</td><td class="patent-data-table-td patent-date-value">Mar 14, 2002</td><td class="patent-data-table-td ">Marion Cornelissen</td><td class="patent-data-table-td ">Attenuated hiv strains and use thereof</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="npl-citations"></a><div class="patent-section-header"><span class="patent-section-title">Non-Patent Citations</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th colspan="3"class="patent-data-table-th">Reference</th></tr></thead><tr><td class="patent-data-table-td ">1</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">McGrath et al., Sequence analysis of DNA randomly amplified from the Saccharomyces cerevisiae genome, Molecular and Cellular Probes, 1998, pp. 397-405. vol. 12.</td></tr><tr><td class="patent-data-table-td ">2</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">PCT International Preliminary Examination Report, PCT NL01 00020, dated Mar. 25, 2003, 2 pages.</td></tr><tr><td class="patent-data-table-td ">3</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">PCT International Search, PCT-NL01 00020, dated Jul. 18, 2002, 2 pages.</td></tr><tr><td class="patent-data-table-td ">4</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Rose et al., Consensus-degenerate hybrid oligonucleotide primers for amplification of distantly related sequences. Nucleic Acids Research, 1998, pp. 1628-1635, vol. 26, No. 7.</td></tr><tr><td class="patent-data-table-td ">5</td><td class="patent-data-table-td "></td><td class="patent-data-table-td ">Smith et al , Automated differential display using a flourescently labeled universal primer, Biotechniques, 1997, pp. 274-279, vol. 23, No. 2.</td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="forward-citations"></a><div class="patent-section-header"><span class="patent-section-title">Referenced by</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Citing Patent</th><th class="patent-data-table-th">Filing date</th><th class="patent-data-table-th">Publication date</th><th class="patent-data-table-th">Applicant</th><th class="patent-data-table-th">Title</th></tr></thead><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7171622">US7171622</a></td><td class="patent-data-table-td patent-date-value">Jul 18, 2002</td><td class="patent-data-table-td patent-date-value">Jan 30, 2007</td><td class="patent-data-table-td ">International Business Machines Corporation</td><td class="patent-data-table-td ">Method, apparatus and computer program product for projecting objects in a display unit</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7718403">US7718403</a><span class='patent-tooltip-anchor' data-tooltip-text="Cited by examiner"> *</span></td><td class="patent-data-table-td patent-date-value">Mar 8, 2004</td><td class="patent-data-table-td patent-date-value">May 18, 2010</td><td class="patent-data-table-td ">Rubicon Genomics, Inc.</td><td class="patent-data-table-td ">Amplification and analysis of whole genome and whole transcriptome libraries generated by a DNA polymerization process</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7803550">US7803550</a></td><td class="patent-data-table-td patent-date-value">Nov 13, 2008</td><td class="patent-data-table-td patent-date-value">Sep 28, 2010</td><td class="patent-data-table-td ">Rubicon Genomics, Inc.</td><td class="patent-data-table-td ">Attaching one strand of a stem-loop oligonucleotide having an inverted repeat and a loop to a double stranded nucleic acid molecule, and displacing one strand of the oligonucleotide from the oligonucleotide-attached nucleic acid molecule by strand displacement or by nick translation polymerization</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US7833716">US7833716</a></td><td class="patent-data-table-td patent-date-value">Jun 6, 2007</td><td class="patent-data-table-td patent-date-value">Nov 16, 2010</td><td class="patent-data-table-td ">Gen-Probe Incorporated</td><td class="patent-data-table-td ">Tagged oligonucleotides and their use in nucleic acid amplification methods</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8034570">US8034570</a></td><td class="patent-data-table-td patent-date-value">Sep 28, 2010</td><td class="patent-data-table-td patent-date-value">Oct 11, 2011</td><td class="patent-data-table-td ">Gen-Probe Incorporated</td><td class="patent-data-table-td ">Tagged oligonucleotides and their use in nucleic acid amplification methods</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8039214">US8039214</a></td><td class="patent-data-table-td patent-date-value">Jun 30, 2008</td><td class="patent-data-table-td patent-date-value">Oct 18, 2011</td><td class="patent-data-table-td ">Cellscript, Inc.</td><td class="patent-data-table-td ">Synthesis of tagged nucleic acids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8071312">US8071312</a></td><td class="patent-data-table-td patent-date-value">Sep 28, 2010</td><td class="patent-data-table-td patent-date-value">Dec 6, 2011</td><td class="patent-data-table-td ">Rubicon Genomics, Inc.</td><td class="patent-data-table-td ">Methods for producing and using stem-loop oligonucleotides</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8198027">US8198027</a></td><td class="patent-data-table-td patent-date-value">Dec 20, 2007</td><td class="patent-data-table-td patent-date-value">Jun 12, 2012</td><td class="patent-data-table-td ">Gen-Probe Incorporated</td><td class="patent-data-table-td ">Methods and compositions for nucleic acid amplification</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8206913">US8206913</a></td><td class="patent-data-table-td patent-date-value">Mar 3, 2010</td><td class="patent-data-table-td patent-date-value">Jun 26, 2012</td><td class="patent-data-table-td ">Rubicon Genomics, Inc.</td><td class="patent-data-table-td ">Amplification and analysis of whole genome and whole transcriptome libraries generated by a DNA polymerization process</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8278052">US8278052</a></td><td class="patent-data-table-td patent-date-value">Sep 13, 2011</td><td class="patent-data-table-td patent-date-value">Oct 2, 2012</td><td class="patent-data-table-td ">Gen-Probe Incorporated</td><td class="patent-data-table-td ">Tagged oligonucleotides and their use in nucleic acid amplification methods</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8329887">US8329887</a></td><td class="patent-data-table-td patent-date-value">Sep 20, 2011</td><td class="patent-data-table-td patent-date-value">Dec 11, 2012</td><td class="patent-data-table-td ">Cellscript, Inc.</td><td class="patent-data-table-td ">Synthesis of tagged nucleic acids</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8399199">US8399199</a></td><td class="patent-data-table-td patent-date-value">Nov 1, 2011</td><td class="patent-data-table-td patent-date-value">Mar 19, 2013</td><td class="patent-data-table-td ">Rubicon Genomics</td><td class="patent-data-table-td ">Use of stem-loop oligonucleotides in the preparation of nucleic acid molecules</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8409804">US8409804</a></td><td class="patent-data-table-td patent-date-value">Mar 2, 2006</td><td class="patent-data-table-td patent-date-value">Apr 2, 2013</td><td class="patent-data-table-td ">Rubicon Genomics, Inc.</td><td class="patent-data-table-td ">Isolation of CpG islands by thermal segregation and enzymatic selection-amplification method</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8440404">US8440404</a></td><td class="patent-data-table-td patent-date-value">Mar 3, 2005</td><td class="patent-data-table-td patent-date-value">May 14, 2013</td><td class="patent-data-table-td ">Rubicon Genomics</td><td class="patent-data-table-td ">Methods and compositions for generating and amplifying DNA libraries for sensitive detection and analysis of DNA methylation</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8512955">US8512955</a></td><td class="patent-data-table-td patent-date-value">Jul 1, 2010</td><td class="patent-data-table-td patent-date-value">Aug 20, 2013</td><td class="patent-data-table-td ">Gen-Probe Incorporated</td><td class="patent-data-table-td ">Methods and compositions for nucleic acid amplification</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8580510">US8580510</a></td><td class="patent-data-table-td patent-date-value">Sep 12, 2012</td><td class="patent-data-table-td patent-date-value">Nov 12, 2013</td><td class="patent-data-table-td ">Gen-Probe Incorporated</td><td class="patent-data-table-td ">Tagged oligonucleotides and their use in nucleic acid amplification methods</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8642268">US8642268</a></td><td class="patent-data-table-td patent-date-value">Apr 30, 2012</td><td class="patent-data-table-td patent-date-value">Feb 4, 2014</td><td class="patent-data-table-td ">Gen-Probe Incorporated</td><td class="patent-data-table-td ">Methods and compositions for nucleic acid amplification</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8728737">US8728737</a></td><td class="patent-data-table-td patent-date-value">Feb 13, 2013</td><td class="patent-data-table-td patent-date-value">May 20, 2014</td><td class="patent-data-table-td ">Rubicon Genomics, Inc.</td><td class="patent-data-table-td ">Attaching a stem-loop oligonucleotide to a double stranded DNA molecule</td></tr><tr><td class="patent-data-table-td citation-patent"><a href="/patents/US8778610">US8778610</a></td><td class="patent-data-table-td patent-date-value">Feb 27, 2013</td><td class="patent-data-table-td patent-date-value">Jul 15, 2014</td><td class="patent-data-table-td ">Rubicon Genomics, Inc.</td><td class="patent-data-table-td ">Methods for preparing amplifiable DNA molecules</td></tr></table><div class="patent-section-footer">* Cited by examiner</div></div><div class="patent-section patent-tabular-section"><a id="classifications"></a><div class="patent-section-header"><span class="patent-section-title">Classifications</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th"> </th><th class="patent-data-table-th"> </th></tr></thead><tr><td class="patent-data-table-td ">U.S. Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=hRtpBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S006100">435/6.1</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=hRtpBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S091200">435/91.2</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=hRtpBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S091100">435/91.1</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=hRtpBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S024330">536/24.33</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=hRtpBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S023100">536/23.1</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=hRtpBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S022100">536/22.1</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=hRtpBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc435/defs435.htm&usg=AFQjCNFivWgFr8pvtdk7qYLW9R7yxFP7Dg#C435S007100">435/7.1</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=hRtpBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S024300">536/24.3</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=hRtpBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S024320">536/24.32</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=hRtpBAABERAJ&q=http://www.uspto.gov/web/patents/classification/uspc536/defs536.htm&usg=AFQjCNHex3TR9tihVxBKQnsQg6n8YmdVYg#C536S024310">536/24.31</a></span></td></tr><tr><td class="patent-data-table-td ">International Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=hRtpBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12Q0001700000">C12Q1/70</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=hRtpBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12P0019340000">C12P19/34</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=hRtpBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C07H0021000000">C07H21/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=hRtpBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C07H0021040000">C07H21/04</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=hRtpBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C07H0019000000">C07H19/00</a></span>, <span class="nested-value"><a href="http://www.google.com/url?id=hRtpBAABERAJ&q=http://web2.wipo.int/ipcpub/&usg=AFQjCNER44F5jlVoswCkvW3YEcB5lW4moA#refresh=page&notion=scheme&version=20130101&symbol=C12Q0001680000">C12Q1/68</a></span></td></tr><tr><td class="patent-data-table-td ">Cooperative Classification</td><td class="patent-data-table-td "><span class="nested-value"><a href="http://www.google.com/url?id=hRtpBAABERAJ&q=http://worldwide.espacenet.com/classification&usg=AFQjCNGs5WqSrPE3A4ZP63zGuM6PRNfEFA#!/CPC=C12Q1/6853">C12Q1/6853</a></span></td></tr><tr><td class="patent-data-table-td ">European Classification</td><td class="patent-data-table-td "><span class="nested-value">C12Q1/68D2E</span></td></tr></table><div class="patent-section-footer"></div></div><div class="patent-section patent-tabular-section"><a id="legal-events"></a><div class="patent-section-header"><span class="patent-section-title">Legal Events</span></div><table class="patent-data-table"><thead class="patent-data-table-thead"><tr class="patent-data-table"><th class="patent-data-table-th">Date</th><th class="patent-data-table-th">Code</th><th class="patent-data-table-th">Event</th><th class="patent-data-table-th">Description</th></tr></thead><tr><td class="patent-data-table-td patent-date-value">Dec 18, 2012</td><td class="patent-data-table-td ">FP</td><td class="patent-data-table-td ">Expired due to failure to pay maintenance fee</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20121026</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Oct 26, 2012</td><td class="patent-data-table-td ">LAPS</td><td class="patent-data-table-td ">Lapse for failure to pay maintenance fees</td><td class="patent-data-table-td "></td></tr><tr><td class="patent-data-table-td patent-date-value">Jun 11, 2012</td><td class="patent-data-table-td ">REMI</td><td class="patent-data-table-td ">Maintenance fee reminder mailed</td><td class="patent-data-table-td "></td></tr><tr><td class="patent-data-table-td patent-date-value">Mar 27, 2008</td><td class="patent-data-table-td ">FPAY</td><td class="patent-data-table-td ">Fee payment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Year of fee payment: </span><span class="nested-value">4</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Apr 25, 2006</td><td class="patent-data-table-td ">RR</td><td class="patent-data-table-td ">Request for reexamination filed</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20060303</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Mar 3, 2003</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">PRIMAGEN HOLDING B.V., NETHERLANDS</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">CORRECTIVE CHANGE OF NAME TO CORRECT THE NAME OF THE ASSIGNEE.  DOCUMENT PREVIOUSLY RECORDED AT REEL 013402 FRAME 0520.;ASSIGNOR:AMSTERDAM SUPPORT DIAGNOSTICS B.V.;REEL/FRAME:013797/0683</span></div><div class="nested-key-value"><span class="nested-key">Effective date: </span><span class="nested-value">20030224</span></div><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">PRIMAGEN HOLDING B.V. MEIBERGDREEF 591105 BA AMSTE</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">CORRECTIVE CHANGE OF NAME TO CORRECT THE NAME OF THE ASSIGNEE. DOCUMENT PREVIOUSLY RECORDED AT REEL013402 FRAME 0520.;ASSIGNOR:AMSTERDAM SUPPORT DIAGNOSTICS B.V. /AR;REEL/FRAME:013797/0683</span></div></td></tr><tr><td class="patent-data-table-td patent-date-value">Oct 21, 2002</td><td class="patent-data-table-td ">AS</td><td class="patent-data-table-td ">Assignment</td><td class="patent-data-table-td "><div class="nested-key-value"><span class="nested-key">Owner name: </span><span class="nested-value">AMSTERDAM SUPPORT DIAGNOSTICS B.V., NETHERLANDS</span></div><div class="nested-key-value"><span class="nested-key">Free format text: </span><span class="nested-value">ASSIGNMENT OF ASSIGNORS INTEREST;ASSIGNORS:ZHANG, JING;DEKKER, JOHN;VAN DER KUYL, ANTOINETTE C.;ANDOTHERS;REEL/FRAME:013402/0520;SIGNING DATES FROM 20020710 TO 20020808</span></div></td></tr></table><div class="patent-section-footer"></div></div><div class="modal-dialog" id="patent-images-lightbox"><div class="patent-lightbox-controls"><div class="patent-lightbox-rotate-controls"><div class="patent-lightbox-rotation-text">Rotate</div><div class="rotate-icon rotate-ccw-icon"></div><div class="rotate-icon rotate-cw-icon"></div></div><div class="patent-lightbox-index-counter"></div><a class="patent-lightbox-fullsize-link" target="_blank">Original Image</a><div class="patent-drawings-control patent-drawings-next"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_right.png" alt="Next page"width="21" height="21" /></div><div class="patent-drawings-control patent-drawings-prev"><img class="patent-drawings-button-img"src="/googlebooks/images/kennedy/page_left.png" alt="Previous page"width="21" height="21" /></div></div><div class="modal-dialog-content"><div class="patent-lightbox-image-holder"><div class="patent-lightbox-placeholder"></div></div></div></div><script>_OC_initPatentsAtb({image_not_available_html: " Image not available"});</script></div></div></div></td></tr></table><script>(function() {var href = window.location.href;if (href.indexOf('?') !== -1) {var parameters = href.split('?')[1].split('&');for (var i = 0; i < parameters.length; i++) {var param = parameters[i].split('=');if (param[0] == 'focus') {var elem = document.getElementById(param[1]);if (elem) {elem.focus();}}}}})();</script><script>_OC_addFlags({LockSrc:"/books/javascript/lock_8a2b04e7bf975d5171d8e4c0b6365c7a.js", Host:"http://www.google.com/", IsBooksRentalEnabled:1, IsWebstoreDisplayCaseEnabled:1, IsObfuscationEnabled:1, IsBrowsingHistoryEnabled:1, IsWebReaderSvgEnabled:0, IsGeoLayerEnabled:1, IsImageModeNotesEnabled:1, IsCopyMenuItemEnabled:1, IsGiftingEnabled:0, IsWebReaderUniversalPaginatorEnabled:0, IsOfflineBubbleEnabled:1, IsReaderEnabledForPlayRequests:1, IsFutureOnSaleVolumesEnabled:1, IsOfflineRestrictedCopyEnabled:1, IsBooksUnifiedLeftNavEnabled:1, IsRestrictedCopyEnabled:1, IsZipitFolderCollectionEnabled:1, IsEndOfSampleRecommendationsEnabled:1, IsRatingsOnBookcardsEnabled:1, IsAdsDisabled:0, IsIframePageDisplayEnabled:0, IsEmbeddedMediaEnabled:1, IsImageModeAnnotationsEnabled:1, IsMyLibraryGooglePlusEnabled:1, IsImagePageProviderEnabled:0, IsBookcardListPriceSmall:0, IsInternalUser:0, IsBooksShareButtonEnabled:0, IsPreOrdersEnabled:0, IsDisabledRandomBookshelves:0, WebstoreDisplayCasePosition:3});_OC_Run({"enable_p13n":false,"add_vol_to_collection_base_url":"http://www.google.com/patents?op=add\u0026sig=ACfU3U0NbxX5DHtXD2qMCkylHWXC4tBBxQ\u0026id=hRtpBAABERAJ","remove_vol_from_collection_base_url":"http://www.google.com/patents?op=remove\u0026sig=ACfU3U0RN2U7Jb4T2llU9jBDamflnGcgvg\u0026id=hRtpBAABERAJ","logged_in":false,"p13n_save_user_settings_url":"http://www.google.com/patents?op=edit_user_settings\u0026sig=ACfU3U2W-BFef1gM5cTenWii8mC8_kpeDw","is_cobrand":false,"sign_in_url":"https://www.google.com/accounts/Login?service=\u0026continue=http://www.google.com/patents%3Fhl%3Den\u0026hl=en","is_play_enabled":true}, {"volume_id":"","is_ebook":true,"volumeresult":{"has_flowing_text":false,"has_scanned_text":true,"can_download_pdf":false,"can_download_epub":false,"is_pdf_drm_enabled":false,"is_epub_drm_enabled":false,"download_pdf_url":"http://www.google.com/patents/download/Amplificaton_of_nucleotide_sequences_pre.pdf?id=hRtpBAABERAJ\u0026output=pdf\u0026sig=ACfU3U29HMyiHqQ5fFmlC6z5lG7ZhcJ6yw"},"sample_url":"http://www.google.com/patents/reader?id=hRtpBAABERAJ\u0026printsec=frontcover\u0026output=reader\u0026source=gbs_atb_hover","is_browsable":true,"is_public_domain":true}, {});</script><div id="footer_table" style="font-size:83%;text-align:center;position:relative;top:20px;height:4.5em;margin-top:2em"><div style="margin-bottom:8px"><a href=http://www.google.com/><nobr>Google&nbsp;Home</nobr></a> - <a href=//www.google.com/patents/sitemap/><nobr>Sitemap</nobr></a> - <a href=http://www.google.com/googlebooks/uspto.html><nobr>USPTO Bulk Downloads</nobr></a> - <a href=/intl/en/privacy/><nobr>Privacy Policy</nobr></a> - <a href=/intl/en/policies/terms/><nobr>Terms of Service</nobr></a> - <a href=https://support.google.com/faqs/answer/2539193?hl=en><nobr>About Google Patents</nobr></a> - <a href="http://www.google.com/tools/feedback/intl/en/error.html" onclick="try{_OC_startFeedback({productId: '72792',locale: 'en'});return false;}catch(e){}"><nobr>Send Feedback</nobr></a></div><span>Data provided by IFI CLAIMS Patent Services</span><br><span >&copy;2012 Google</span></div> <script type="text/javascript">var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));</script><script type="text/javascript">var pageTracker = _gat._getTracker("UA-27188110-1");pageTracker._setCookiePath("/patents/");pageTracker._trackPageview();</script> </body></html>